<article>
<section>
<h1 id="header">Isolation of spermatogenic cells from the <i>cynomolgus</i> macaque testis with flow cytometry</h1>
<p><time datetime="2021-01-22">Published: January 22, 2021</time></p>
<p>Xianzhong Lau,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Pearly Jean Ai Yong,<sup><a href="#aff2">2</a></sup> Charles Kang Liang Lou,<sup><a href="#aff3">3</a></sup> Prabhakaran Munusamy,<sup><a href="#aff1">1</a></sup> and Mahesh Sangrithi<sup><a href="#aff1">1</a>,<a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>KK Women’s and Children’s Hospital, Division of Obstetrics and Gynaecology, 100 Bukit Timah Road, Singapore 229899, Singapore</p>
<p id="aff2"><sup>2</sup>KK Women’s and Children’s Hospital, Department of Pathology and Laboratory Medicine, 100 Bukit Timah Road, Singapore 229899, Singapore</p>
<p id="aff3"><sup>3</sup>Singhealth, Flow Cytometry Core Platform, 20 College Road, Singapore 169856, Singapore</p>
<p id="aff4"><sup>4</sup>Duke-NUS Graduate Medical School, Cancer Stem Cell Biology/OBGYN ACP, 8 College Road, Singapore 169857, Singapore</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:lau.xianzhong@kkh.com.sg">lau.xianzhong@kkh.com.sg</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:mahesh.sangrithi@duke-nus.edu.sg">mahesh.sangrithi@duke-nus.edu.sg</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100294">10.1016/j.xpro.2021.100294</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe a detailed protocol for the isolation of purified populations of viable spermatogenic cells derived from the non-human primate model organism <i>Macaca fascicularis</i> (<i>cynomolgus</i>). Using fluorescence-activated cell sorting (FACS), we describe methods to isolate spermatogonia and primary spermatocytes ranging across the sub-stages of meiosis prophase I. These cell populations can be used with a variety of downstream assays, including single-cell approaches such as RNA sequencing, chromatin immunoprecipitation, quantitative RT-PCR, and immunocytochemistry.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib5">Lau et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Dissection of macaque testis and dissociation of tissue to single cells</li>
<li>Isolation of primary spermatocytes with FACS using Hoechst 33342 fluorescence</li>
<li>Sample preparation and gating strategy for obtaining SSEA-4-positive spermatogonia</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/428-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Preparation of BD FACS Aria III Cell Sorter for analysis and sorting</h3>
<ol>
<li>Flow cytometric analysis and fluorescence-activated cell sorting were performed on a BD FACS Aria III Cell Sorter equipped with a 488 nm blue, 561 nm yellow green, 640 nm red, and 355 nm UV laser.</li>
<li>Warm the lasers on the BD FACS Aria III Cell Sorter for a minimum of 20 min before calibrating the system with BD FACS Diva Cytometer Setup &amp; Tracking (CS &amp; T) beads. This ensures that time delay calibration was properly setup prior to all flow cytometry experiments.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Time delay calibration must be done prior to all experiments to ensure that laser delay values are updated for accurate generation of flow cytometry results.</div>
<ol start="3">
<li>Quality control for cell sorting experiments was performed using the BD FACS Accudrop beads to ensure that at least 95% beads were deflected to the left stream before the initiation of all cell sorting experiments.</li>
<li>Unless otherwise stated, PI was excited using the 561 nm yellow green laser and its emission captured using a 610/20 bandpass (BP) and 600 long pass (LP) filter. Emission from APC off the 640 nm red laser was detected using a 670/14BP filter. Hoechst blue and Hoechst red fluorescence emitted from excitation of Hoechst 33342 was detected off the 450/50BP coupled with a 410LP filter and 670 LP/630 LP filter respectively.</li>
<li>Cells were sorted at 70 psi with a 70μm nozzle. Sort streams were aligned and sorted into tubes using the 4-way purity precision mode.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sort precision modes on the sorter may differ between different brands of instruments. If in doubt, select the precision mode that allows you to obtain the highest purity that your experimental needs require without compromising yield.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-SSEA-4 APC-conjugated; dilution: 10 μL/10<sup>6</sup> cells</td>
<td>R&amp;D Systems</td>
<td>Cat# FAB1435A, RRID:AB_494994</td>
</tr>
<tr>
<td>Mouse monoclonal anti-γH2A.X (Phospho S139); dilution: 1:100</td>
<td>Abcam</td>
<td>Cat# ab26350, RRID:AB_470861</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-SYCP3; dilution: 1:100</td>
<td>Abcam</td>
<td>Cat# ab15093, RRID:AB_301639</td>
</tr>
<tr>
<td>Goat polyclonal anti-mouse IgG H&amp;L (Alexa Fluor 488); dilution: 1:500</td>
<td>Abcam</td>
<td>Cat# ab150113, RRID:AB_2576208</td>
</tr>
<tr>
<td>Goat polyclonal anti-rabbit IgG H&amp;L (Alexa Fluor 594); dilution: 1:500</td>
<td>Abcam</td>
<td>Cat# ab150080, RRID:AB_2650602</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Adult testis; &gt;4 year-old <i>Macaca fascicularis</i>
</td>
<td>National Large Animal Research Facility</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Collagenase type I</td>
<td>Gibco</td>
<td>Cat# 17100017</td>
</tr>
<tr>
<td>Dispase II</td>
<td>Gibco</td>
<td>Cat# 17105041</td>
</tr>
<tr>
<td>Calcium chloride dihydrate</td>
<td>Sigma-Aldrich</td>
<td>Cat# 21097-250G</td>
</tr>
<tr>
<td>Hanks’ balanced salt solution</td>
<td>Sigma-Aldrich</td>
<td>Cat# H6648-500ML</td>
</tr>
<tr>
<td>DMEM, high glucose, HEPES</td>
<td>Gibco</td>
<td>Cat# 12430054</td>
</tr>
<tr>
<td>Dulbecco’s phosphate-buffered saline</td>
<td>Gibco</td>
<td>Cat# 14190144</td>
</tr>
<tr>
<td>Phosphate-buffered saline</td>
<td>Gibco</td>
<td>Cat# 10010023</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>Sigma-Aldrich</td>
<td>Cat# F2442-100ML</td>
</tr>
<tr>
<td>Trypan blue solution</td>
<td>Sigma-Aldrich</td>
<td>Cat# T8154-20ML</td>
</tr>
<tr>
<td>Hoechst 33342, trihydrochloride, trihydrate, 10 mg/mL</td>
<td>Invitrogen</td>
<td>Cat# H3570</td>
</tr>
<tr>
<td>Propidium iodide (PI), 1.0 mg/mL</td>
<td>Invitrogen</td>
<td>Cat# P3566</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat# S0389-500G</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Millipore</td>
<td>Cat# 9410-OP</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma-Aldrich</td>
<td>Cat# P9416-100ML</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate</td>
<td>Sigma-Aldrich</td>
<td>Cat# L3771-100G</td>
</tr>
<tr>
<td>16% paraformaldehyde aqueous solution</td>
<td>Electron Microscopy Sciences</td>
<td>Cat# 15710</td>
</tr>
<tr>
<td>SlowFade Gold Antifade Mountant with DAPI</td>
<td>Invitrogen</td>
<td>Cat# S36938</td>
</tr>
<tr>
<td>BD FACS Diva CS&amp;T Research Beads</td>
<td>BD</td>
<td>Cat# 655051</td>
</tr>
<tr>
<td>BD FACSAccudrop Beads</td>
<td>BD</td>
<td>Cat# 345249</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td><i>Macaca fascicularis</i></td>
<td>National Large Animal Research Facility</td>
<td>IACUC protocol Ref. 2019/SHS/1465</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BD FACS Diva 8.0.1 Software, Firmware Version 1.3</td>
<td>BD</td>
<td>NA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Collagenase type 1 stock</h3>
<p>Reconstitute 1 g of collagenase powder in vial with 1 mL of HBSS. Vortex to dissolve the powder and transfer to a fresh tube. Dilute the collagenase solution with HBSS, bringing the concentration to 100 U/μL (500× stock solution). Filter the stock solution with a 0.45 μm filter. Dispense into aliquots and store in −20°C. Avoid multiple freeze/thaw cycles.</p>
<h3 id="sec2.2">Dispase II stock</h3>
<p>Reconstitute 5 g of dispase powder in vial with DPBS to 10 mg/mL. Further dilute with PBS to a concentration of 240 U/mL (100× stock solution). Filter the stock solution with a 0.45 μm filter. Dispense into aliquots and store in −20°C. Avoid multiple freeze/thaw cycles.</p>
<table id="undtbl1">
<caption>Tissue digest buffer A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>(500×) Collagenase Type 1 stock</td>
<td>200 U/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>HBSS</td>
<td>n/a</td>
<td>24.95 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the buffer fresh and keep at 22°C.</p>
<table id="undtbl2">
<caption>Tissue digest buffer B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>(500×) Collagenase Type 1 stock solution</td>
<td>200 U/mL</td>
<td>50 μL</td>
</tr>
<tr>
<td>(100×) Dispase II stock solution</td>
<td>2.4 U/mL</td>
<td>250 μL</td>
</tr>
<tr>
<td>1 M CaCl<sub>2</sub>
</td>
<td>1 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>HBSS</td>
<td>n/a</td>
<td>24.675 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the buffer fresh and keep at 22°C.</p>
<table id="undtbl3">
<caption>Unstained control</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>1%</td>
<td>10 μL</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>990 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the solution fresh and keep at 22°C.</p>
<table id="undtbl4">
<caption>PI only control</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1.0 mg/mL PI</td>
<td>1 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>FBS</td>
<td>1%</td>
<td>10 μL</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>989 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the solution fresh and keep at 22°C protected from light.</p>
<table id="undtbl5">
<caption>Hoechst 33342 only control</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 mg/mL Hoechst 33342</td>
<td>5 μg/mL</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>FBS</td>
<td>1%</td>
<td>10 μL</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>989.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the solution fresh and keep at 22°C protected from light.</p>
<table id="undtbl6">
<caption>PI + Hoechst 33342 sample</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1.0 mg/mL PI</td>
<td>1 μg/mL</td>
<td>10 μL</td>
</tr>
<tr>
<td>10 mg/mL Hoechst 33342</td>
<td>5 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>FBS</td>
<td>1%</td>
<td>100 μL</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>9.885 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the solution fresh and keep at 22°C protected from light.</p>
<table id="undtbl7">
<caption>SSEA-4 APC-conjugated Ab stain solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mouse monoclonal anti-SSEA-4 APC-conjugated Antibody</td>
<td>10 μL/10<sup>6</sup> cells</td>
<td>Dependent on cell concentration</td>
</tr>
<tr>
<td>FBS</td>
<td>1%</td>
<td>10 μL</td>
</tr>
<tr>
<td>DPBS</td>
<td>n/a</td>
<td>990 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the solution fresh and keep at 22°C protected from light.</p>
<table id="undtbl8">
<caption>Collection media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>10%</td>
<td>2 mL</td>
</tr>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>18 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the media and store at 4°C.</p>
<table id="undtbl9">
<caption>Freezing media</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>20%</td>
<td>4 mL</td>
</tr>
<tr>
<td>DMSO</td>
<td>10%</td>
<td>2 mL</td>
</tr>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>14 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the media fresh and keep at 22°C.</p>
<table id="undtbl10">
<caption>PFA fixative solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>16% PFA</td>
<td>2%</td>
<td>250 μL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.02%</td>
<td>4 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>1.746 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the fixative solution and store at 22°C.</p>
<table id="undtbl11">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10% BSA</td>
<td>0.15%</td>
<td>30 μL</td>
</tr>
<tr>
<td>10% Tween-20</td>
<td>0.2%</td>
<td>40 μL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>1.93 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the buffer and store at 4°C. Warm it to 22°C before use.</p>
<table id="undtbl12">
<caption>Primary antibody mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Mouse monoclonal anti-γH2AFX (Phospho S139)</td>
<td>1:100</td>
<td>10 μL</td>
</tr>
<tr>
<td>Rabbit polyclonal anti-SYCP3</td>
<td>1:100</td>
<td>10 μL</td>
</tr>
<tr>
<td>Blocking Buffer</td>
<td>n/a</td>
<td>980 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the antibody mix fresh and keep at 4°C.</p>
<table id="undtbl13">
<caption>Secondary antibody mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM or μM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Goat polyclonal anti-mouse IgG H&amp;L (Alexa Fluor 488)</td>
<td>1:500</td>
<td>2 μL</td>
</tr>
<tr>
<td>Goat polyclonal anti-rabbit IgG H&amp;L (Alexa Fluor 594)</td>
<td>1:500</td>
<td>2 μL</td>
</tr>
<tr>
<td>Blocking Buffer</td>
<td>n/a</td>
<td>996 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<p>Prepare the antibody mix fresh and keep at 4°C protected from light.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Testis dissection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>The following steps are carried out to isolate seminiferous tubules from the macaque testis for enzymatic digestion.</p>
<ol>
<li>Perform testis dissection is on a 10 cm petri dish (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/428-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. C<i>ynomolgus</i> macaque testis dissection</div>
<p>(A) Image of an adult <i>cynomolgus</i> macaque testis.</p>
<p>(B) Detachment of the epididymis from the testis.</p>
<p>(C) Small incision of the tunica albuginea.</p>
<p>(D) Dissection of tunica albuginea and separation of the seminiferous tubules from the tunica albuginea.</p>
<p>(E) Segmentation of testis tubule tissue into 1–2 g pieces.</p>
</figcaption>
</figure>
<ol type="a">
<li>Remove the outer layers, e.g., skin and loose connective tissue.</li>
<li>Detach the epididymis.</li>
</ol>
</li>
<li>Weigh the testis.</li>
<li>Remove the tunica albuginea.
<ol type="a">
<li>Make a small incision of the tunica albuginea.</li>
<li>Dissect under it to create a dissection plane, separating the seminiferous tubules underneath from the tunica.</li>
</ol>
</li>
<li>Cut the testis tubules into 1 - 2 g pieces, and further cut each piece into smaller pieces to facilitate subsequent digestion.</li>
</ol>
<h3 id="sec3.2">Dissociation of testis samples</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>The subsequent steps are performed to obtain a single-cell suspension of unselected cells from seminiferous tubules using enzymatic digestion.</p>
<ol start="5">
<li>Submerge testis tubule pieces in 25 mL of Tissue Digest Buffer A and leave the samples to digest at 37°C in a water bath for 5 min.
<ol type="a">
<li>Shake the tube vigorously every 2 min until loosened tubules become visible.</li>
</ol>
</li>
<li>Remove and discard the supernatant.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Cells in the supernatant, i.e., primarily interstitial cells, may also be kept and processed separately if this is of interest.</div>
<ol start="7">
<li>Add 25 mL of Tissue Digest Buffer B and allow the digest to occur for 30 min at 37°C in a water bath.
<ol type="a">
<li>Shake the tube vigorously every 2 min until dissociation of the tubules into a single-cell suspension of cells.</li>
</ol>
</li>
<li>Add 2.5 mL of FBS to stop digestion.</li>
<li>Centrifuge the cell suspension at 200 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend the cell pellet in 25 mL of PBS</li>
<li>Repeat steps 9 and 10.</li>
<li>Centrifuge the cell suspension at 200 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend the cells in a desired volume of PBS.</li>
<li>Filter the cell suspension through a 40 μm cell strainer into a fresh tube.</li>
<li>Count the cells to determine cell concentration.</li>
</ol>
<h3 id="sec3.3">Sample preparation for spermatocyte fluorescence-activated cell sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min</div>
<p>Steps 16–20 describe live cells staining with the Hoechst 33342<i>.</i></p>
<ol start="16">
<li>Aliquot the appropriate number of cells into four tubes.
<ol type="a">
<li>Unstained control: 1 × 10<sup>6</sup> cells</li>
<li>PI only control: 1 × 10<sup>6</sup> cells</li>
<li>Hoechst 33342 only control: 1 × 10<sup>6</sup> cells</li>
<li>PI + Hoechst 33342 sample: 5 × 10<sup>6</sup> to 1 × 10<sup>7</sup> cells</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Tubes <i>a</i> - <i>c</i> serve as single stained controls used for gating and assessing fluorescence spectral overlap purposes. Assessment of the spectrum of spermatogenic cells can then be gated and analyzed using Tube d.</div>
<ol start="17">
<li>Centrifuge the cell suspension at 200 × <i>g</i> for 10 min at 4°C and discard the supernatant.</li>
<li>Resuspend the matching cell pellet tube with 1 mL of prepared stain solution.</li>
<li>Incubate tubes at 37°C for 30 min in the dark. Agitate tubes every 10 min to resuspend cells thus ensuring uniform staining.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Incubation with Hoechst 33342 stain has to be performed for at least 30 min at 37°C for optimal resolution.</div>
<ol start="20">
<li>Tubes are kept on ice, protected from light and filtered using a 40μm cell strainer prior to analysis.</li>
</ol>
<h3 id="sec3.4">Sample preparation for spermatogonia fluorescence-activated cell sorting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45 min</div>
<p>Steps 21–28 describe live cell staining with SSEA-4 APC-conjugated antibody for the isolation of spermatogonia.</p>
<ol start="21">
<li>Aliquot the appropriate number of cells into three tubes.
<ol type="a">
<li>Unstained control: 1 × 10<sup>6</sup> cells</li>
<li>PI only control: 1 × 10<sup>6</sup> cells</li>
<li>PI + SSEA-4 APC-conjugated Ab sample: 1 × 10<sup>7</sup> cells</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Tubes a and b serve as single stained controls used for gating and assessing fluorescence spectral overlap purposes. Tube c contains all fluorochromes of interest involved in the cell sorting assay.</div>
<ol start="22">
<li>Centrifuge the cell suspension at 200 × <i>g</i> for 10 min at 4°C and discard the supernatant.</li>
<li>Resuspend the matching cell pellet tube with 1 mL of prepared SSEA-4 APC-conjugated Ab stain solution. Incubate tubes at 37°C for 30 min in the dark. Agitate tubes every 10 min to resuspend cells.</li>
<li>Centrifuge the cell suspension at 200 × <i>g</i> for 10 min at 4°C and discard the supernatant.</li>
<li>Resuspend cell pellet and wash twice with DPBS to remove unbound SSEA-4-APC-conjugated antibodies.</li>
<li>Resuspend cells with DPBS + 10% FBS at a density of 2 to 3 million cells/mL for the sort.</li>
<li>Add 1 μg/mL of PI and filter the stained cell suspension using a 40μm cell strainer prior to cell sorting.</li>
<li>Tubes are kept on ice and protected from light before cell sorting.</li>
</ol>
<h3 id="sec3.5">Gating strategy for isolating primary spermatocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>Steps 29–34 describe the gating strategy to use on the BD FACS Diva software for isolation of primary spermatocytes at various meiotic sub-stages of prophase I.</p>
<ol start="29">
<li>Create an FSC-A versus SSC-A contour plot (linear scale). Draw a “Scatter” gate to include cells of interest while excluding debris.</li>
<li>On the FSC-A versus FSC-H contour plot (linear scale), create a sub-gate under the “Scatter” gate to encompass single cells. This step helps to omit doublets which could otherwise contaminate purity of the sorted cells.</li>
<li>Next, gate on PI negative live cells under the Singlets Gate using a PI-A (log scale) versus FSC-A (linear scale) contour.</li>
<li>Display all live viable events on a DNA histogram with Hoechst blue on the x axis and count on the y axis. Low Hoechst blue intensity events are excluded to omit debris and unstained cells. Draw a “DNA” gate to only include cells containing DNA content of 1C and greater. Data are collected in linear scale.</li>
<li>View gated “DNA” events on the Hoechst red-A/Hoechst blue-A contour plot (linear scale). A spectrum of cells in varying stages of spermatogenesis can be identified and gated based on their DNA content and Hoechst red/Hoechst blue fluorescence profile (please refer to <a href="#fig2">Figure 2</a>F for more details on the profile obtained).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/428-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Flow cytometry analysis and gating of primary spermatocytes</div>
<p>(A) Gating strategy for isolating primary spermatocytes.</p>
<p>(B) Scatter gate drawn from unstained control to include cells of interest while excluding debris. Baseline voltages for the sample and level of autofluorescence in sample can be determined without the addition of fluorochromes.</p>
<p>(C) Singlet gate drawn to omit doublets which may contribute to false positives.</p>
<p>(D) Gate on PI negative live cells drawn from PI only control to determine cell viability.</p>
<p>(E) DNA gate to determine spread of cells based on nucleic acid content from Hoechst only control, and exclude cells less than 1C.</p>
<p>(F) Gates drawn for the collection of enriched primary spermatocytes based on their stages of meiotic prophase I.</p>
</figcaption>
</figure>
</li>
<li>Create 4 gates and label them A, B, C, and D. These gates represent cell populations that are enriched in primary spermatocytes at pre-leptotene, leptotene/zygotene, early pachytene, late pachytene meiotic stages respectively (<a href="#fig2">Figure 2</a>A).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PI is a nucleic acid stain that enters dead cells with disrupted cell membranes, and is excluded from live ones with intact membranes. The live cell dye Hoechst 33342 on the other hand is able to enter cells through diffusion and binds to the nucleic acid residues on the minor groove of DNA. Hoechst allows the detection variation in DNA content and chromatin, and in meiotic cells Hoechst red/blue fluorescence can be used to discriminate subpopulation of cells undergoing meiosis I (<a href="#bib1">Bastos et al., 2005</a>). Similarly, in this protocol, PI is used to reliably discriminate live cells (i.e., intact PI negative cells) versus non-viable (PI positive) cells, and Hoechst fluorescence is used to discriminate on the DNA content of meiotic cells.</div>
<div class="note">
<span class="note-title">Note:</span> Similar Hoechst 33342 staining profiles can be obtained using different brands of flow cytometers equipped with the laser lines described above. Filter configurations can differ between different systems.</div>
<h3 id="sec3.6">Flow cytometry analysis and fluorescence-activated cell sorting for primary spermatocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 40 min</div>
<p>In this section, testis cells are sorted according to their Hoechst blue/Hoechst red fluorescence. Gated primary spermatocytes at various meiotic prophase I sub-stages are collected from the sort.</p>
<ol start="35">
<li>Using the gating template created above, acquire samples and optimize photomultiplier tube (PMT) voltages using the single stain controls in the following sequence (unstained, PI only, Hoechst 33342 only).
<div class="note">
<span class="note-title">Note:</span> This step ensures that signals captured are on scale and also allows us to access fluorochrome interactions with one another (spectral overlap).</div>
<ol type="a">
<li>Unstained tube
<ol type="i">
<li>Sets baseline voltages for the sample.</li>
<li>Determines the level of autofluorescence in the sample without adding any fluorochromes (<a href="#fig2">Figure 2</a>B).</li>
<li>Select and gate for singlets (<a href="#fig2">Figure 2</a>C). Omit doublets which may contribute to false positive signals.</li>
</ol>
</li>
<li>PI only control tube
<ol type="i">
<li>Allows for the assessment of PI negative live and PI positive dead cells. Cell viability can be determined from the percentages of PI negative cells that did not stain with propidium iodide (<a href="#fig2">Figure 2</a>D).</li>
</ol>
</li>
<li>Hoechst 33342 only control tube
<ol type="i">
<li>Determines the spread of cells based on their nucleic acid content on the Hoechst blue histogram (<a href="#fig2">Figure 2</a>E).</li>
<li>Bi-parameter display of Hoechst blue/Hoechst red fluorescence signals allows for the assessment of cells in varying stages of spermatogenesis.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="36">
<li>Acquire and record 10,000 to 50,000 events in these tubes.</li>
<li>Check the Hoechst profile of viable (PI negative) cells using the Hoechst 33342 + PI-stained tube using the optimized PMT settings above.
<ol type="a">
<li>Record a minimum of 500,000 events before establishing gates for primary spermatocytes.</li>
<li>Define and gate cells based on their <i>Hoechst</i> fluorescence pattern, which corresponds to their DNA content (<a href="#fig2">Figure 2</a>F). The shift in Hoechst red and Hoechst blue corresponds to the meiotic progression of primary spermatocytes, which allows for the discrimination of sub-stages of meiotic prophase I.
<ol type="i">
<li>Gate 4N primary spermatocytes, based on their distinct sub-stages, following the contours of the cell density.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="38">
<li>Program to collect and sort cells.
<ol type="a">
<li>Sort cells using a low flow rate with 70 μm nozzle at 70 psi. Maintain event rate around 2,000–3,500 events/s.
<div class="note">
<span class="note-title">Note:</span> High flow rates can increase the speed of cell sorting but compromise sort purity of the isolated fractions.</div>
</li>
<li>Maintain a minimum of &gt;80% sort efficiency during the sort. Monitor discard rate throughout to ensure that is it preferably not &gt;200 events/s.</li>
</ol>
</li>
</ol>
<ol start="39">
<li>Collect sorted cells in round bottom FACS tubes pre-loaded with collection media.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is important to pre-coat the interior of collection tubes with collection media as this reduces the chances of sorted cells sticking to the interior of the tube during collection. 10% FBS in collection media enhances the survival of sorted fractions.</div>
<ol start="40">
<li>To store cells, centrifuge to pellet collected cells at 200 × <i>g</i> for 10 min at 4°C.</li>
<li>Discard supernatant and resuspend cells in Freezing Media to a concentration of 1000 cells/μL.</li>
<li>Aliquot 100–200 μL of cell suspension in cryovials and store at −80°C.</li>
</ol>
<h3 id="sec3.7">Gating strategy for isolating spermatogonia</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<p>Steps 43–46 describe the gating strategy to use on the BD FACS Diva software for isolation of SSEA-4 positive spermatogonia.</p>
<ol start="43">
<li>Create an FSC-A versus SSC-A contour plot (linear scale). Draw a “Scatter” gate to include cells of interest while excluding debris.</li>
<li>On the FSC-A versus FSC-H contour plot (linear scale), create a sub-gate under the “Scatter” gate to encompass single cells. This step helps to omit doublets which could otherwise contaminate purity of the sorted cells.</li>
<li>Next, gate on PI negative live cells under the Singlets Gate using a PI-A (log scale) versus FSC-A (linear scale) contour.</li>
<li>Create 2 gates and label them SSEA4- and SSEA4+. These gates represent cells that are negatively or positively stained with anti-SSEA-4 APC-conjugated antibody (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/428-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Flow cytometry analysis and gating of spermatogonia</div>
<p>(A) Gating strategy for isolating spermatogonia.</p>
<p>(B) Scatter gate drawn from unstained control to include cells of interest while excluding debris. Baseline voltages for the sample and level of autofluorescence in sample can be determined without the addition of fluorochromes.</p>
<p>(C) Singlet gate drawn to omit doublets which may contribute to false positives.</p>
<p>(D) Gate on PI negative live cells drawn from PI only control to determine cell viability.</p>
<p>(E) Gates drawn for the collection of SSEA-4 negative and positive cells.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.8">Fluorescence-activated cell sorting for spermatogonia</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 40 min</div>
<p>In this section, testis cells are sorted based on the presence of APC fluorescence and SSEA-4 positive spermatogonia are collected.</p>
<ol start="47">
<li>Using the gating template created above, acquire samples and optimize photomultiplier tube (PMT) voltages using the single stain controls in the following sequence (unstained, PI only, Hoechst 33342 only).
<div class="note">
<span class="note-title">Note:</span> This step ensures that signals captured are on scale and also allows us to access fluorochrome interactions with one another (spectral overlap).</div>
<ol type="a">
<li>Unstained tube
<ol type="i">
<li>Sets baseline voltages for the sample</li>
<li>Determines the level of autofluorescence in the sample without adding any fluorochromes (<a href="#fig3">Figure 3</a>B).</li>
<li>Select and gate for singlets. Omit doublets which may contribute to false positive signals (<a href="#fig3">Figure 3</a>C).</li>
</ol>
</li>
<li>PI only control tube. Allows for the assessment of viable PI negative and PI positive dead cells. Cell viability can be determined from the percentages of PI negative cells that did not take up propidium iodide (<a href="#fig3">Figure 3</a>D).</li>
</ol>
</li>
</ol>
<ol start="48">
<li>Acquire and analyze the PI + SSEA-4 APC-conjugated Ab sample tube.
<ol type="a">
<li>Record a minimum of 100,000 events before establishing gates.</li>
<li>Cells are gated and selected for SSEA-4-positivity (<a href="#fig3">Figure 3</a>E).</li>
</ol>
</li>
</ol>
<ol start="49">
<li>Program to collect and sort PI negative SSEA-4-APC positive cells.
<ol type="a">
<li>Sort cells using a low flow rate with 70 μm nozzle at 70 psi. Maintain event rate around 2,000–3,500 events/s.
<div class="note">
<span class="note-title">Note:</span> High flow rates can increase the speed of cell sorting but compromise sort purity of the isolated fractions.</div>
</li>
<li>Maintain a minimum of &gt;80% sort efficiency during the sort. Monitor discard rate throughout to ensure that is it preferably not &gt;200 events/s.</li>
<li>Collect sorted cells in round bottom FACS tubes pre-loaded with collection media.
<div class="note">
<span class="note-title">Note:</span> It is important to pre-coat the interior of collection tubes with collection media as this reduces the chances of sorted cells sticking to the interior of the tube during collection. 10% FBS in collection media enhances the survival of sorted fractions.</div>
</li>
</ol>
</li>
</ol>
<ol start="50">
<li>To store cells, centrifuge to pellet collected cells at 200 × <i>g</i> for 10 min at 4°C.</li>
<li>Discard supernatant and resuspend cells in Freezing Media to a concentration of 1000 cells/μL.</li>
<li>Aliquot 100–200 μL of cell suspension in cryovials and store at −80°C.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec4.1">Fluorescence-activated cell sorting for primary spermatocytes</h3>
<p>Hoechst blue fluorescence reflects variations in chromatin content/DNA content. Increasing Hoechst blue fluorescence intensity allows visualization of testicular cells in various stages of spermatogenesis based on their DNA content. Haploid spermatids containing 1N DNA content are located at the lowest part of the Hoechst blue scale nearest 50. Hoechst blue/Hoechst red contour plot allows distinction of cellular populations A, B, C, and D. Back gating of these individual clusters reveals that A corresponds to regions enriched in 2N to 4N early meiotic prophase I cells such as pre-leptotene, and B corresponds to 4N leptotene and zygotene spermatocytes. Region C and D corresponds to early and late pachytene spermatocytes containing 4N DNA content respectively (<a href="#fig2">Figure 2</a>F).</p>
<p>The expected percentage of live cells and numbers of collected events from the various gated samples are shown in <a href="#tbl1">Table 1</a>.</p>
<table id="tbl1">
<caption>Table 1. The expected percentage of live cells and numbers of collected events from the various gated samples</caption>
<thead>
<tr>
<th> </th>
<th>Events (rounded to nearest 10,000)</th>
<th>% total</th>
</tr>
</thead>
<tbody>
<tr>
<td>Total unsorted cells</td>
<td>10,000,000</td>
<td> </td>
</tr>
<tr>
<td>Live cells</td>
<td>6,260,000</td>
<td>62.6</td>
</tr>
<tr>
<td>A</td>
<td>120,000</td>
<td>1.2</td>
</tr>
<tr>
<td>B</td>
<td>250,000</td>
<td>2.5</td>
</tr>
<tr>
<td>C</td>
<td>330,000</td>
<td>3.3</td>
</tr>
<tr>
<td>D</td>
<td>140,000</td>
<td>1.4</td>
</tr>
</tbody>
</table>
<h3 id="sec4.2">Fluorescence-activated cell sorting for spermatogonia</h3>
<p>SSEA-4 is a surface marker of primate spermatogonial stem cells (SSC) (<a href="#bib4">Guo et al., 2018</a>; <a href="#bib8">Maki et al., 2009</a>). Typically, SSEA-4 positive cells comprised only 1%–2% of unselected cells from the adult testis, with 70%–80% of these being spermatogonia (<a href="#bib5">Lau et al., 2020</a>).</p>
<p>The expected percentage of live cells and number of collected events from the SSEA4+ gate are shown in <a href="#tbl2">Table 2</a>.</p>
<table id="tbl2">
<caption>Table 2. The expected percentage of live cells and number of collected events from the SSEA4-/+ gates</caption>
<thead>
<tr>
<th> </th>
<th>Events</th>
<th>% Total</th>
</tr>
</thead>
<tbody>
<tr>
<td>Total unsorted cells</td>
<td>10,000,000</td>
<td> </td>
</tr>
<tr>
<td>Live cells</td>
<td>5,500,000</td>
<td>55.0</td>
</tr>
<tr>
<td>SSEA4-</td>
<td>5,330,000</td>
<td>53.3</td>
</tr>
<tr>
<td>SSEA4+</td>
<td>140,000</td>
<td>1.4</td>
</tr>
</tbody>
</table>
<h3 id="sec4.3">Quality control of sorted spermatocytes using immunofluorescence</h3>
<p>Spermatocytes isolated may be further analyzed and classified into sub-stages of prophase I of meiosis, based on their immunostaining patterns for γ-<i>H2AFX</i> (green) and <i>SYCP3</i> staining patterns (<a href="#fig4">Figure 4</a>A). Cells undergoing leptonema exhibit <i>SYCP3</i> aggregates and abundant intense γ-<i>H2AFX</i> staining (<a href="#bib3">Gaysinskaya and Bortvin, 2015</a>). During zygonema, the formation of long <i>SYCP3</i> stretches are observed in the cell, with decreased levels of γ-<i>H2AFX</i>, as it becomes restricted to chromatin of the remaining axial elements to synapse and start to associate with the condensing X and Y chromatin (<a href="#bib3">Gaysinskaya and Bortvin, 2015</a>; <a href="#bib6">Mahadevaiah et al., 2001</a>). For pachytene cells, γ-<i>H2AFX</i> is only present on the condensed X and Y chromatin, as referred to as the sex body. This transcriptionally silenced sex chromatin is typically observed at the nuclear periphery (<a href="#bib2">Chi et al., 2011</a>; <a href="#bib6">Mahadevaiah et al., 2001</a>, <a href="#bib7">2008</a>). Finally, during diplonema, γ-<i>H2AFX</i> remains on the sex body as it observed toward the middle of the nucleus (<a href="#bib6">Mahadevaiah et al., 2001</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/428-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Quality control of sorted spermatocytes using immunofluorescence</div>
<p>(A) Immunofluorescence for γ-<i>H2AFX</i> (green) and <i>SYCP3</i> (red). A, representative image of a leptotene cell from gate A; B, representative image of a zygotene cell from gate B; C, representative image of a pachytene cell from gate C; and D, representative image of a diplotene cell from gate D. DNA was visualized using DAPI (blue).</p>
<p>(B) Stacked percentage plot representing spermatocytes at varying stages of meiosis prophase I from each gate.</p>
</figcaption>
</figure>
<p>An example of the percentages of cells undergoing leptonema, zygonema, pachynema, and diplonema for each gate obtained from a previous experiment are shown in <a href="#tbl3">Table 3</a> and visualized in <a href="#fig4">Figure 4</a>B. Percentages are based on 100 cells scored and counted for each gate.</p>
<table id="tbl3">
<caption>Table 3. The expected percentages of cells undergoing leptonema, zygonema, pachynema, and diplonema for each gate</caption>
<thead>
<tr>
<th> </th>
<th colspan="5">%</th>
</tr>
<tr>
<th>Gate</th>
<th>Leptonema</th>
<th>Zygonema</th>
<th>Pachynema</th>
<th>Diplonema</th>
<th>Other cells</th>
</tr>
</thead>
<tbody>
<tr>
<td>A</td>
<td>76.1</td>
<td>5.3</td>
<td>0</td>
<td>0</td>
<td>18.6</td>
</tr>
<tr>
<td>B</td>
<td>1.6</td>
<td>64.5</td>
<td>21.8</td>
<td>2.4</td>
<td>9.7</td>
</tr>
<tr>
<td>C</td>
<td>2.2</td>
<td>18.4</td>
<td>63.7</td>
<td>15.6</td>
<td>0</td>
</tr>
<tr>
<td>D</td>
<td>0</td>
<td>17.1</td>
<td>54.9</td>
<td>28.0</td>
<td>0</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Hoechst 33342 staining is extremely sensitive to deviations from the intended protocol. Slight variations in the amount of Hoechst 33342 dye added and incubation timings of the dye with dissociated testicular cells may influence the overall Hoechst profile derived. We also recommend that a minimum incubation timing of 30 min at 37°C with the dye to obtain the Hoechst blue/red profile described above. Additionally, previous studies have shown that the Hoechst incubation timing and profile can differ between different species of animals (<a href="#bib1">Bastos et al., 2005</a>; <a href="#bib3">Gaysinskaya and Bortvin, 2015</a>). Hence, the protocol for Hoechst staining presented should be used as a reference in species other than an adult <i>cynomolgus</i> macaque.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<p>The authors report no problems encountered.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bastos, H., Lassalle, B., Chicheportiche, A., Riou, L., Testart, J., Allemand, I., and Fouchet, P. (2005). Flow cytometric characterization of viable meiotic and postmeiotic cells by Hoechst 33342 in mouse spermatogenesis. Cytom. Part A <i>65</i>, 40-49. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref1">View at publisher</a></p>
<p id="bib2">Chi, M.N., Auriol, J., Jégou, B., Kontoyiannis, D.L., Turner, J.M.A., De Rooij, D.G., and Morello, D. (2011). The RNA-binding protein ELAVL1/HuR is essential for mouse spermatogenesis, acting both at meiotic and postmeiotic stages. Mol. Biol. Cell <i>22</i>, 2875-2885. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref2">View at publisher</a></p>
<p id="bib3">Gaysinskaya, V. and Bortvin, A. (2015). Flow cytometry of murine spermatocytes. Curr. Protoc. Cytom. <i>7</i>, 44.1-47.44. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref3">View at publisher</a></p>
<p id="bib4">Guo, J., Grow, E.J., Mlcochova, H., Maher, G.J., Lindskog, C., Nie, X., Guo, Y., Takei, Y., Yun, J., Cai, L., et al. (2018). The adult human testis transcriptional cell atlas. Cell Res. <i>28</i>, 1141-1157. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref4">View at publisher</a></p>
<p id="bib5">Lau, X., Munusamy, P., Ng, M.J., and Sangrithi, M. (2020). Single-cell RNA sequencing of the cynomolgus macaque testis reveals conserved transcriptional profiles during Mammalian Spermatogenesis. Dev. Cell <i>54</i>, 548-566.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref5">View at publisher</a></p>
<p id="bib6">Mahadevaiah, S.K., Turner, J.M.A., Baudat, F., Rogakou, E.P., De Boer, P., Blanco-Rodríguez, J., Jasin, M., Keeney, S., Bonner, W.M., and Burgoyne, P.S. (2001). Recombinational DNA double-strand breaks in mice precede synapsis. Nat. Genet. <i>27</i>, 271-276. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref6">View at publisher</a></p>
<p id="bib7">Mahadevaiah, S.K., Bourc’his, D., De Rooij, D.G., Bestor, T.H., Turner, J.M.A., and Burgoyne, P.S. (2008). Extensive meiotic asynapsis in mice antagonises meiotic silencing of unsynapsed chromatin and consequently disrupts meiotic sex chromosome inactivation. Source J. Cell Biol. <i>182</i>, 263-276. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref7">View at publisher</a></p>
<p id="bib8">Maki, C.B., Pacchiarotti, J., Ramos, T., Pascual, M., Pham, J., Kinjo, J., Anorve, S., and Izadyar, F. (2009). Phenotypic and molecular characterization of spermatogonial stem cells in adult primate testes. Hum. Reprod. <i>24</i>, 1480-1491. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00001-0/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Mahesh Sangrithi (<a href="mailto:mahesh.sangrithi@duke-nus.edu.sg">mahesh.sangrithi@duke-nus.edu.sg</a>).</p>
<h4>Materials Availability</h4>
<p>There are restrictions to the availability of primate derived cells due to permits (e.g., CITES).</p>
<h4>Data and Code Availability</h4>
<p>The study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>We thank SingHealth Experimental Medicine Centre (SEMC) and SingHealth National Large Animal Research Facility (NLARF) (Dr. Edgar Pena and staff) for their assistance with animal husbandry and procedures. We also thank the SingHealth Flow Cytometry Core Platform for help with the flow cytometry experiments. M.S. is supported by the <a href="https://doi.org/10.13039/501100001349">National Medical Research Council</a>, Singapore Clinician Scientist Investigator Award.</p>
<h3>Author Contributions</h3>
<p>Experiments were conceived by M.S. and X.L.; sample collection, processing, and laboratory procedures were performed by X.L. and M.S.; flow cytometry was performed by P.J.A.Y. and C.L.K.L; and M.S., P.J.A.Y., and X.L. wrote the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>