<article>
<section>
<h1 id="header">Optimized electroporation of CRISPR-Cas9/gRNA ribonucleoprotein complex for selection-free homologous recombination in human pluripotent stem cells</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Huaigeng Xu,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a></sup> Yuto Kita,<sup><a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Uikyu Bang,<sup><a href="#aff2">2</a></sup> Peter Gee,<sup><a href="#aff3">3</a>,<a href="#fn2">5</a></sup> and Akitsu Hotta<sup><a href="#aff2">2</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Urology, University of California, San Francisco, San Francisco, CA 94143, USA</p>
<p id="aff2"><sup>2</sup>Center for iPS cell Research and Application, Kyoto University, Kyoto 606-8507, Japan</p>
<p id="aff3"><sup>3</sup>MaxCyte Inc., Gaithersburg, USA</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:akitsu.hotta@cira.kyoto-u.ac.jp">akitsu.hotta@cira.kyoto-u.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100965">10.1016/j.xpro.2021.100965</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Selection-free, scarless genome editing in human pluripotent stem cells (PSCs) by utilizing ribonucleoprotein (RNP) of CRISPR-Cas9 is a useful tool for a variety of applications. However, the process can be hampered by time-consuming subcloning steps and inefficient delivery of the RNP complex and ssDNA template. Here, we describe the optimized protocol to introduce a single nucleotide change or a loxP site insertion in feeder-free, xeno-free iPSCs by utilizing MaxCyte and 4D-Nucleofector electroporators.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.02.013">Kagita et al. (2021)</a> and <a class="external-link" href="https://doi.org/10.1016/j.stem.2019.02.005">Xu et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A detailed protocol for efficient genome editing with Cas9/gRNA RNP electroporation</li>
<li>Single-strand DNA template-mediated substitution of single nucleotide biallelically</li>
<li>Targeted insertion of a short sequence, such as loxP site, without drug selection</li>
<li>Establishment of knockin or knockout clones by seamless genotyping and subcloning</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1179-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The discovery of CRISPR-Cas9 has made tremendous progress in various research areas especially for genetic functions, disease modeling, and gene therapy (<a href="#bib8">Dastidar et al., 2018</a>; <a href="#bib24">Ran et al., 2013</a>; <a href="#bib28">Soldner and Jaenisch, 2018</a>; <a href="#bib34">Zhong et al., 2020</a>). This technology is widely applied to human PSCs, which have a great potential for regenerative medicine and cell therapies. With this in mind, the protocol utilizes feeder-free, xeno-free human PSCs cultured on recombinant Laminin-511 (iMatrix-511) with StemFit AK02N (or AK03N) media.</p>
<p>For successful genome editing, the method to introduce Cas9/gRNA complex into cells is critical. Ribonucleoprotein (RNP) form of delivery has several advantages over the DNA/RNA form (<a href="#bib10">Gee et al., 2020</a>; <a href="#bib15">Kim et al., 2014</a>; <a href="#bib23">Paquet et al., 2016</a>; <a href="#bib25">Richardson et al., 2016</a>; <a href="#bib35">Zuris et al., 2015</a>), since the RNP functions immediately, degrades quickly, and is relatively smaller than plasmid DNA or mRNA. Furthermore, the pre-formed Cas9/gRNA RNP complex can evade the inhibition by cellular mRNA (<a href="#bib14">Kagita et al., 2021</a>).</p>
<p>In addition to Cas9/gRNA, single-stranded oligodeoxynucleotides (ssODNs) must be available at the DNA repair site as a template for homology directed repair (HDR) pathway. In this protocol, we introduce Cas9/gRNA RNP complex and ssODN together into human PSCs by utilizing MaxCyte or 4D-Nucleofector electroporators, which were superior for inducing HDR.</p>
<p>Also, this protocol can be applied for single nucleotide editing for disease modeling and correcting mutations in PSCs, since more than half of disease-associated genetic changes are single nucleotide polymorphisms (<a href="#bib1">Bamshad et al., 2011</a>; <a href="#bib6">Collins et al., 1997</a>). Moreover, a 34 bp loxP sequence can be inserted for further genomic engineering by Cre recombination (<a href="#bib4">Branda and Dymecki, 2004</a>; <a href="#bib27">Sauer and Henderson, 1988</a>). Additionally, by simply omitting an ssODN, a small (∼10 nt) indel may be introduced for knocking out a target gene.</p>
<h3 id="sec1.1">Preparation of human iPSCs (feeder-free, Xeno-free condition)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼8 days</div>
<p>Prepare human iPSCs cultured in a well of 6-well plate with iMatrix-511 coating and StemFit AK02N (or AK03N) media as below. The original protocol developed by Dr. Nakagawa can be found in the below CiRA’s website.</p>
<p>(<a href="https://www.cira.kyoto-u.ac.jp/j/research/img/protocol/Ff-iPSC-culture_protocol_E_v140311.pdf">https://www.cira.kyoto-u.ac.jp/j/research/img/protocol/Ff-iPSC-culture_protocol_E_v140311.pdf</a>)</p>
<div class="critical">
<span class="critical-title">Critical:</span> During culturing, iPSCs should be in the logarithmic growth-phase with minimal sign of differentiation or cell death. Passage at least two times after thawing before use. The condition and health of iPSCs are critical for this protocol because the HDR event only occurs in dividing cells at the S-G2 cell cycle phase.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Although we have not tested alternative culture conditions in our hands, other feeder-free media (mTeSR, Essential 8, or DEF-CS) and coating materials (such as Matrigel, Vitronectin, or Geltrex) might be utilized. However, all the conditions, including seeding cell density and electroporation conditions, must be optimized for each culture system.</div>
<ol>
<li>Plate coating.
<ol type="a">
<li>For coating an entire plate of 6-well plate, we add 50 μL (30–60 μL) of iMatrix into 12 mL PBS and mix them, and add 2 mL into each well of 6-well plate (final concentration: 0.25–0.5 μg/cm<sup>2</sup>). Required volume for different multi-well plates is summarized below.
<table id="undtbl1">
<thead>
<tr>
<th>Plate</th>
<th>Amount of coating for each well</th>
</tr>
</thead>
<tbody>
<tr>
<td>6-well plate</td>
<td>2 mL</td>
</tr>
<tr>
<td>12-well plate</td>
<td>1 mL</td>
</tr>
<tr>
<td>24-well plate</td>
<td>500 μL</td>
</tr>
<tr>
<td>96-well plate</td>
<td>100 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Incubate at 37°C in a 5% CO<sub>2</sub> incubator for 1–18 h.
<div class="note">
<span class="note-title">Note:</span> Longer coating (such as overnight) is better for tight adhesion of iPSCs, and shorter coating (1 h) can be used for gentle dissociation of iPSCs before electroporation.</div>
</li>
<li>Aspirate the coating buffer and add 2 mL of StemFit media + 2 μL of 10 mM Y-27632 (final 10 μM) into each well, and incubate in a CO<sub>2</sub> incubator at 37°C for pre-warming the media until the dissociation of iPSCs is finished.</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Passage of iPSCs from a well of 6-well plate
<ol type="a">
<li>Aspirate media of cultured iPSCs.</li>
<li>Wash the cells by adding 1–2 mL of PBS, gently rock the plate, and aspirate PBS.</li>
<li>Add 300 μL of 0.5 × TrypLE Select into iPSCs.
<div class="note">
<span class="note-title">Note:</span> See “<a href="#materials-and-equipment">materials and equipment</a>” section for making 0.5 × TrypLE Select. The half concentration of TrypLE Select solution provides better control and gentle detachment of iPSCs. Alternatively, accutase can be used as an animal product free cell dissociation reagent. Accutase contains collagenases, in addition to proteases, to facilitate detachment of cell types with rich extracellular matrix, such as iPSCs.</div>
</li>
<li>Put the plate into a 37°C incubator and incubate for 10 min. Gently shaking the plate every 3–4 min is helpful for detaching the cells completely.</li>
<li>Add 1 mL of StemFit media with 10 μM Y-27632 and gently pipet cells until iPSCs are dissociated completely.</li>
<li>Count cells, and seed 1.0 × 10<sup>4</sup>–1.5 × 10<sup>4</sup> cells into the iMatrix-coated 6-well plate which contains 2 mL of StemFit media with 10 μM Y-27632, as described in the Step c.</li>
<li>Incubate cells in a CO<sub>2</sub> incubator at 37°C for overnight.</li>
<li>The next day, change media with 2 mL of StemFit media. If there are a lot of dead cells floating, keep adding Y-27632 to the media at the final 10 μM concentration.</li>
<li>Change media every 2 days while culturing.</li>
<li>iPSCs will be semi-confluent on day 6–8. Never let them become over-confluent. “Semi-confluent” means that iPSC colonies are less than 2 mm in diameter and there are still some spaces between iPSC colonies. The speed of growth depends on iPSC lines, so the timing of semi-confluency should be determined experimentally.
<div class="note">
<span class="note-title">Note:</span> The addition of Y-27632 is important to improve the survival rate of iPSCs suspended in single cells (<a href="#bib31">Watanabe et al., 2007</a>). Keep adding Y-27632 during the culture in case there are many dead cells after seeding. If iPSCs do not reach semi-confluency after 8 days of seeding due to slow growth, passage the cells again with higher seeding cell density. As there are significant line-to-line variations among iPSC lines, we recommend switching the iPSC line if a certain cell line of iPSCs is difficult for culturing, electroporating, or genome editing.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">gRNA design for <i>Streptococcus pyogenes</i> CRISPR-Cas9 and preparation by <i>in vitro</i> transcription</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="3">
<li>Planning genome editing strategy and design of gRNA.
<ol type="a">
<li>Plan your genome editing strategy. Locate the nucleotide position and obtain the surrounding genomic sequence (i.e., from USCS genome browser, NCBI Nucleotide database, or IGV software).</li>
<li>Search for a PAM sequence around the targeting site, ideally within a 10–20 bp window from the target site. SpCas9 utilize “5′-NGG-3′” PAM sequence (or “5′-CCN-3′” sequence for targeting reverse strand) and the expected cutting site is at 3 bp upstream (5′) from the PAM sequence (<a href="#fig1">Figure 1</a>). Choose several gRNAs if multiple candidate PAM sequences are found.
<div class="note">
<span class="note-title">Note:</span> Several gRNA design tools such as CRISPick or Benchling are useful to identify the PAM sequence and gRNA site near the targeting site.</div>
</li>
<li>Check the number of potential off-target sites by a web tool, such as CRISPRdirect or CRISPOR. Try to avoid gRNAs that target multiple potential off-target sites (less than 3 bp mismatch.) in the human genome. If multiple candidate gRNAs are found, choose gRNAs with fewer off-target sites. In addition, such mismatches are better located at the seed region (within 10 bp from PAM) since sequence recognition by gRNA is more precise at the seed region. Many gRNA design web tools provide a simple score that implement such mismatch preferences, so choose gRNAs with good score. If possible, we recommend designing several gRNAs for each target locus, as it is difficult to predict gRNAs’ activity and specificity until those are tested in cells.</li>
<li>To prepare gRNA with <i>in vitro</i> transcription (IVT) reaction using T7 promoter, it is preferred that the target sequence starts from GG. This is because the first base of gRNA is the transcriptional start site for the T7 promoter, and gRNA transcription is more efficient if there are one or two Gs at the beginning of gRNA. Add two Gs in front of your gRNA sequence, or replace the first one or two nucleotides at the beginning of gRNA. Such additional Gs may decrease on-target efficiency and/or increase off-target cleavage risk. To avoid this complication associated with IVT reaction, order chemical synthesis of gRNA to completely match your gRNA sequence with the target site.
<div class="note">
<span class="note-title">Note:</span> Design several gRNAs as described below to target the site of interest where you would like to alter the nucleic acid sequence or insert a small fragment such as the loxP site. When creating a conditional allele franked by two loxP sequence, make sure the insertion of the two loxP sites do not interrupt the genomic function, such as transcription and splicing. Stay away from such genomic motifs, if possible.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If no proper PAM sequence is found within 10–20 bp window from the target site, consider utilization of engineered type II SpCas9 variants with unconstrained PAM sequence (i.e., SpCas9-VQR, VRQR, VRER, SpCas9-NG, xCas9, SpG, or SpRY), orthologs of SpCas9 (i.e., SaCas9 from S<i>taphylococcus aureus</i>), or other types of CRISPR (i.e., type V Cas12a/Cpf1 (<a href="#bib33">Zetsche et al., 2015</a>)). Note, only limited kinds of the engineered Cas9 variants are commercially available as RNP form (Cas12a RNP can be purchased from IDT). For this, try plasmid DNA transfection or protein purification.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the purpose of the experiment is gene knockout through NHEJ, there are multiple ways to prevent protein synthesis. The most major approach is to induce a small indel at the protein coding region (i.e. within exon) to disrupt the protein coding flame. In this case, be aware of the position of the indel if there are no alternative ATG start site(s) downstream or alternative isoforms that might skip the indel site. It is also better to make sure the expressed partial protein loses its functionality by deleting an important domain or residue.</div>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Preparation of <i>in vitro</i> transcribed gRNA (IVT-gRNA)
<ol type="a">
<li>Design an oligo DNA as a template for IVT reaction. Insert your 20 bp-long gRNA target sequence (without PAM sequence, Green X letters) into the following primer. If your target sequence does not start from “G” or “GG”, then insert “<b>GG</b>” in front of the target sequence. If your target sequence starts from a “G”, then it is recommended to insert an additional “<b>G</b>” in front of the target sequence.<br> T7-sgRNA forward primer (If the target sequence does not start from “G” or “GG”)<br> 5′-GAAATTAATACGACTCACTATA<b>GG</b>X<sub>1</sub>X<sub>2</sub>X<sub>3</sub>X<sub>4</sub>X<sub>5</sub>X<sub>6</sub>X<sub>7</sub>X<sub>8</sub>X<sub>9</sub>X<sub>10</sub>X<sub>11</sub>X<sub>12</sub>X<sub>13</sub>X<sub>14</sub>X<sub>15</sub>X<sub>16</sub>X<sub>17</sub>X<sub>18</sub>X<sub>19</sub>X<sub>20</sub>GTTTTAGAGCTAGAAATAGCAAG-3′<br> T7-sgRNA forward primer (If the target sequence starts from a single “G”)<br> 5′-GAAATTAATACGACTCACTATA<b>G</b>X<sub>1</sub>X<sub>2</sub>X<sub>3</sub>X<sub>4</sub>X<sub>5</sub>X<sub>6</sub>X<sub>7</sub>X<sub>8</sub>X<sub>9</sub>X<sub>10</sub>X<sub>11</sub>X<sub>12</sub>X<sub>13</sub>X<sub>14</sub>X<sub>15</sub>X<sub>16</sub>X<sub>17</sub>X<sub>18</sub>X<sub>19</sub>X<sub>20</sub>GTTTTAGAGCTAGAAATAGCAAG-3′<br> T7-sgRNA forward primer (If the target sequence starts from “GG”) 5′-GAAATTAATACGACTCACTATAX<sub>1</sub>X<sub>2</sub>X<sub>3</sub>X<sub>4</sub>X<sub>5</sub>X<sub>6</sub>X<sub>7</sub>X<sub>8</sub>X<sub>9</sub>X<sub>10</sub>X<sub>11</sub>X<sub>12</sub>X<sub>13</sub>X<sub>14</sub>X<sub>15</sub>X<sub>16</sub>X<sub>17</sub>X<sub>18</sub>X<sub>19</sub>X<sub>20</sub>GTTTTAGAGCTAGAAATAGCAAG-3′
<div class="note">
<span class="note-title">Note:</span> the gRNA target sequence in the forward primer needs to be put into 5′→3′ direction. DO NOT include PAM sequence (NGG) (i.e. “TGTACTACACACACTGACGG” is the sequence without PAM in <a href="#fig1">Figure 1</a>A).</div>
</li>
<li>PCR amplify the gRNA template by the T7-fwd and T7-rev primers<br> T7-rev primer: 5′-AAAGCACCGACTCGGTGCCACTTTTTCAAGTTGATAACGGACTAGCCTTATTTTAACTTGCTATTTCTAGCTCTAAAAC-3′ PCR reaction
<table id="undtbl2">
<thead>
<tr>
<th>PCR reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ultrapure H<sub>2</sub>O</td>
<td> </td>
<td>33 μL</td>
</tr>
<tr>
<td>10 × buffer</td>
<td>1 ×</td>
<td>5 μL</td>
</tr>
<tr>
<td>2 mM dNTPs Mix</td>
<td>0.3 mM</td>
<td>5 μL</td>
</tr>
<tr>
<td>25 mM MgSO<sub>4</sub>
</td>
<td>1.5 mM</td>
<td>3 μL</td>
</tr>
<tr>
<td>T7-sgRNA forward primer (10 μM)</td>
<td>0.3 μM</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>T7-sgRNA reverse primer (10 μM)</td>
<td>0.3 μM</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>KOD-Plus-Neo (1.0 U/μL)</td>
<td>1.0 U/50 μL</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>
<b>50</b> μL</td>
</tr>
</tbody>
</table>
PCR cycling conditions
<table id="undtbl3">
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">30 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>50°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">∞</td>
</tr>
</tbody>
</table>
</li>
<li>Run the PCR product on a 2% agarose gel, and cut out the band with 121–123 bp in size.</li>
<li>Extract the PCR band from the gel, and elute the DNA in 20–30 μL of RNase-free ultrapure water.</li>
<li>Measure the DNA concentration by NanoDrop. The DNA concentration should be 20–50 ng/μL or higher.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>
<i>In vitro</i> transcription reaction
<ol type="a">
<li>Use MEGAshortscript™ T7 Transcription Kit for IVT reaction and mix the reagents as follow.
<table id="undtbl4">
<thead>
<tr>
<th>In vitro transcription reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ultrapure H<sub>2</sub>O</td>
<td> </td>
<td>Up to 20 μL</td>
</tr>
<tr>
<td>10 × reaction buffer</td>
<td>1 ×</td>
<td>2 μL</td>
</tr>
<tr>
<td>75 mM GTP solution</td>
<td>7.5 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>75 mM ATP solution</td>
<td>7.5 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>75 mM CTP solution</td>
<td>7.5 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>75 mM UTP solution</td>
<td>7.5 mM</td>
<td>2 μL</td>
</tr>
<tr>
<td>T7 Enzyme Mix</td>
<td> </td>
<td>2 μL</td>
</tr>
<tr>
<td>Template DNA</td>
<td>75–150 ng/20 μL</td>
<td>&lt; 4 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>
<b>20</b> μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The template DNA volume is better with &lt; 4 μL for minimizing carry-over of impurities from the PCR product.</div>
</li>
<li>Incubate at 37°C for overnight (12–18 h)</li>
<li>After the incubation, add 1 μL of TURBO DNase (attached in the kit) and incubate at 37°C for 15 min, for removing the template DNA.</li>
<li>Extract gRNA by using the RNeasy MinElute Cleanup Kit. After the purification, we recommend eluting in relatively smaller volume of RNase-free ultrapure water (i.e., 30–40 μL) to obtain concentrated IVT-gRNAs. For better electroporation results, the concentration of the gRNA is better to be 1,000 ng/μL or greater.</li>
<li>Aliquot IVT-gRNAs into 5–10 μL (5–10 μg) in each tube and store at −80°C. We recommend minimizing freeze-thaw cycles as much as possible. Thaw on ice right before use.
<div class="optional">
<span class="optional-title">Optional:</span> If desired, gRNA can be synthesized by a company for nucleotide synthesis, such as IDT.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Preparation of single-stranded oligodeoxynucleotide (ssODN) for a template of homologous recombination</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<ol start="6">
<li>Design ssODN for homologous recombination (<a href="#fig1">Figure 1</a>).
<ol type="a">
<li>For replacing a single nucleotide in iPSCs, place the target nucleotide in the middle of ssODN and add 30–60-nt (typically, 50 nt) of homology arms in both sides. The position of the intended alteration site should be located at the 5′ end from the Cas9 cutting site (<a href="#bib22">Paix et al., 2017</a>). With this design, either sense or antisense strand ssODN can be utilized. An example of ssODN design for single nucleotide alteration is shown in <a href="#fig1">Figure 1</a>A. If a restriction enzyme site is designed to appear after ssODN-mediated HR, the frequency of ssODN-mediated HR can be estimated by restriction digestion. Also, a silent mutation should be added at the PAM site to avoid recutting after successful genome editing. To avoid introducing unnecessary amino acid mutations, refer to the codon table. If silent mutations cannot be added at the PAM site, introduce a silent mutation at the seed region of gRNA or near the Cas9 cleavage site. To insert a loxP site (5′- <u>ATAACTTCGTATAGCATACATTATACGAAGTTAT</u>-3′, 34 bp), the ssODN sequence can be designed as <a href="#fig1">Figure 1</a>B. Similar to above, the addition of two nucleotides “TC” would generate an XmnI restriction enzyme site (5′-GAANNNNTTC-3′), which can be used to assess the knock-in efficiency of the loxP sequence afterword. No silent mutation is needed in this case as insertion of the loxP sequence is sufficient to prevent recutting after recombination.</li>
<li>Obtain ssODNs from a company for nucleotide synthesis (such as Fasmac, IDT, Hokkaido System Science) as a freeze-dried product. For most genome editing experiments, a basic purification method (i.e., reverse-phase cartridge purification) should be sufficient. If a higher efficiency is desired, select HPLC as a purification method.</li>
<li>Dissolve the ssODN in ultrapure water at 10 μg/μL concentration. A high concentration of DNA is critical for efficient electroporation outcomes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> TE buffer may be used to reconstitute ssODN for 4D-nucleofector, but reconstitution in water is recommended for MaxCyte electroporation.</div>
<div class="note">
<span class="note-title">Note:</span> If homozygous targeting is necessary, choose your Cas9-cleavage site to be as close to the target base as possible. If heterozygous targeting is desired, and if current homozygous editing efficiency is too high, try designing your gRNA (Cas9 cleavage sight) slightly distal (5–15 bp) from the target base (<a href="#bib23">Paquet et al., 2016</a>).</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1179-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. gRNA and ssODN design for single nucleotide substitution and loxP sequence knockin</div>
<p>(A) An example of gRNA and ssODN design for single nucleotide alteration. First, the Cas9 cleavage site should be as close as the target SNP, ideally within 5 bp. ssODN template contains around 30–60 nt homology arms on both ends. In this case, the “T” base will be altered into “C” at 3-nt upstream (5′ side) from the Cas9 cleavage site. This conversion can be detected by the appearance of the Hpy99I restriction enzyme recognition site (5′-CGWCG-3′) as shown in red letters. Any other restriction enzyme site can be utilized, so long as the cleavage pattern by the restriction enzyme can be distinguished by gel electrophoresis. If it is difficult to design an appropriate restriction enzyme site after a single nucleotide alteration, the efficiency of substitution can be detected by Sanger sequencing. It is recommended to introduce a (silent) mutation at PAM or at the seed region of the target sequence to avoid recutting after recombination.</p>
<p>(B) gRNA targeting site should be adjusted so that the Cas9 cleavage site is located as close to the loxP insertion site as possible. When two additional nucleotide sequence (TC) is added to the 3′ end of the loxP sequence, XmnI restriction enzyme site (5′-GAANN|NNTTC-3′) can be generated, which is later utilized to assess knock-in efficiency, as described in <a href="#fig5">Figure 5</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>iMatrix-511 silk (Laminin-511 E8)</td>
<td>Takara Bio</td>
<td>Cat#T311</td>
</tr>
<tr>
<td>Y-27632</td>
<td>Fujifilm Wako Pure Chemical Corporation</td>
<td>Cat#034-24024</td>
</tr>
<tr>
<td>StemFit® AK02N (or AK03N) media</td>
<td>Takara Bio</td>
<td>Cat#AK02N; AJ100</td>
</tr>
<tr>
<td>PBS (−)</td>
<td>Nacalai Tesque</td>
<td>Cat#14249-24</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>Nacalai Tesque</td>
<td>Cat#06894-14</td>
</tr>
<tr>
<td>Accutase</td>
<td>Sigma-Aldrich</td>
<td>Cat#A6964-500ML</td>
</tr>
<tr>
<td>TrypLE™ Select Enzyme (1×), no phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12563029</td>
</tr>
<tr>
<td>STEM-CELLBANKER® GMP grade</td>
<td>ZENOAQ</td>
<td>Cat#CB045</td>
</tr>
<tr>
<td>HyClone™ MaxCyte® Electroporation buffer</td>
<td>KIKO TECH</td>
<td>Cat#EPB-1</td>
</tr>
<tr>
<td>Recombinant SpCas9 protein</td>
<td>IDT, or Thermo Fisher Scientific</td>
<td>Cat#1081058, A36498</td>
</tr>
<tr>
<td>Lipofectamine Stem Transfection Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#STEM00015</td>
</tr>
<tr>
<td>4-hydroxytamoxifen</td>
<td>Sigma-Aldrich</td>
<td>Cat#H7904</td>
</tr>
<tr>
<td>KOD-plus-Neo</td>
<td>Toyobo</td>
<td>Cat#KOD-401</td>
</tr>
<tr>
<td>PrimeSTAR® GXL DNA Polymerase</td>
<td>Takara bio</td>
<td>Cat#R050B</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>MEGAshortscript™ T7 Transcription Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM1354</td>
</tr>
<tr>
<td>RNeasy MinElute Cleanup Kit</td>
<td>QIAGEN</td>
<td>Cat#74204</td>
</tr>
<tr>
<td>Wizard® SV Gel and PCR Clean-Up System</td>
<td>Promega</td>
<td>Cat#A9281</td>
</tr>
<tr>
<td>P4 Primary Cell 4D-Nucleofector™ X Kit S (32 RCT)</td>
<td>Lonza</td>
<td>Cat#V4XP-4032</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>1383D2 Human iPSCs, reprogrammed PBMC (LP_53, donor #40) isolated from a 36-years-old male donor by episomal vectors</td>
<td><a href="#bib20">Nakagawa et al. (2014)</a></td>
<td>From Dr. Nakagawa</td>
</tr>
<tr>
<td>DMD-iPSCs (CiRA00111 clone) from a male patient with Duchenne muscular dystrophy with exon 44 deletion (The sampling age is 1–9 years)</td>
<td><a href="#bib16">Li et al. (2015)</a></td>
<td>From Riken BRC <a href="https://cellbank.brc.riken.jp/cell_bank/CellInfo/?cellNo=HPS0383">https://cellbank.brc.riken.jp/cell_bank/CellInfo/?cellNo=HPS0383</a>
</td>
</tr>
<tr>
<td>Ff-XT28s05 iPSCs were generated from a healthy male donor homozygous for the 3rd frequent HLA haplotypes in Japan (A<sup>∗</sup>24:02, B<sup>∗</sup>07:02, C<sup>∗</sup>07:02, DRB1<sup>∗</sup>01:01)</td>
<td>CiRA Foundation, <a href="#bib36">Xu et al. (2019)</a>
</td>
<td><a href="https://www.cira-foundation.or.jp/e/project/genome-edited.html">https://www.cira-foundation.or.jp/e/project/genome-edited.html</a></td>
</tr>
<tr>
<td>404C2 Human iPSCs, reprogrammed skin fibroblasts (HDF1388) isolated from a 36-years-old female donor by episomal vectors.</td>
<td><a href="#bib21">Okita et al. (2011)</a></td>
<td>From Dr. Okita, RRID:CVCL_DP92</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>T7-DMDsgRNA1 forward primer: GAAATTAATACGACTCACTATAgggtatcttacaggaactccGTTTTAGAGCTAGAAATA GCAAG</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td>T7-DMD-in55- g3-IVT forward primer: GAAATTAATACGACTCACTATAggactttatagatatctcccaGTTTTAGAGCTAGAAATA GCAAG</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td>T7-DYSFgRNA2-IVT forward primer: GAAATTAATACGACTCACTATAggtactacacacactgacggGTTTTAGAGCTAGAAAT AGCAAG</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td>T7-sgRNA reverse primer: AAAGCACCGACTCGGTGCCACTTTTTCAAGTTGATAACGGACT AGCCTTATTTTAACTTGCTATTTCTAGCTCTAAAAC</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td>DMD1+loxPssODN (134-mer): AAGACATGGGGCTTCATTTTTGTTTTGCCTTTTTGGTATCTTACAGGAACATAACTTCGTATAGCATACATTATACGAAGTTATTCCAGGATGGCATTGGGCAGCGGCAAACTGTTGTCAGAACATTGAATGCA</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td>DMD-in55- g3+loxP-ssODN (134-mer): TCATTTGGAGGTAATTTGTTTGGAACAGTATCAGACTTTATAGATATCTCATAACTTCGTATAGCATACATTATACGAAGTTATCCATGGCTTGTGATAGAATATAAGGGCAATGCAAATGTAGAGTTTTTTGC</td>
<td><a href="#bib14">Kagita et al. (2021)</a></td>
<td>N/A</td>
</tr>
<tr>
<td>DMDexon55(45-55)check_dir5 primer:CCTCGGGTACACTGAAAGTTATGTG</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td>DMDexon45(45-55)check_rev5 primer:CACCACAGGCTTTAACTTCTGCCG</td>
<td>This Paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>CRISPRdirect</td>
<td><a href="#bib19">Naito et al. (2015)</a></td>
<td><a href="https://crispr.dbcls.jp">https://crispr.dbcls.jp</a></td>
</tr>
<tr>
<td>CRISPOR</td>
<td><a href="#bib7">Concordet and Haeussler (2018)</a></td>
<td><a href="http://crispor.tefor.net">http://crispor.tefor.net</a></td>
</tr>
<tr>
<td>ICE</td>
<td>SYNTHEGO</td>
<td><a href="https://www.synthego.com/products/bioinformatics/crispr-analysis">https://www.synthego.com/products/bioinformatics/crispr-analysis</a></td>
</tr>
<tr>
<td>TIDE</td>
<td><a href="#bib5">Brinkman et al. (2014)</a></td>
<td><a href="http://shinyapps.datacurators.nl/tide/">http://shinyapps.datacurators.nl/tide/</a></td>
</tr>
<tr>
<td>Primer3</td>
<td><a href="#bib29">Untergasser et al. (2012)</a></td>
<td><a href="https://bioinfo.ut.ee/primer3-0.4.0/">https://bioinfo.ut.ee/primer3-0.4.0/</a></td>
</tr>
<tr>
<td>Primer-BLAST</td>
<td><a href="#bib32">Ye et al. (2012)</a></td>
<td><a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>6-well plate</td>
<td>Greiner BIO-ONE</td>
<td>Cat#657160</td>
</tr>
<tr>
<td>12-well plate</td>
<td>Greiner BIO-ONE</td>
<td>Cat#665180</td>
</tr>
<tr>
<td>24-well plate</td>
<td>Greiner BIO-ONE</td>
<td>Cat#662160</td>
</tr>
<tr>
<td>96-well plate</td>
<td>Greiner BIO-ONE</td>
<td>Cat#655180</td>
</tr>
<tr>
<td>10 cm Cell Culture Dishes</td>
<td>Greiner BIO-ONE</td>
<td>Cat#664160-013</td>
</tr>
<tr>
<td>Falcon 5 mL Round Bottom Polystyrene Test Tube, with Cell Strainer Snap Cap</td>
<td>Corning</td>
<td>Cat#352235</td>
</tr>
<tr>
<td>MaxCyte® OC-100 cuvette</td>
<td>Kiko tech</td>
<td>Cat#SOC-1</td>
</tr>
<tr>
<td>MaxCyte ATx</td>
<td>Kiko Tech</td>
<td>Cat#E-ATx</td>
</tr>
<tr>
<td>4D-Nucelofector™ Core Unit</td>
<td>Lonza</td>
<td>Cat#AAF-1002B</td>
</tr>
<tr>
<td>4D-Nucelofector™ X Unit</td>
<td>Lonza</td>
<td>Cat#AAF-1002X</td>
</tr>
<tr>
<td>Applied Biosystems™ Veriti™ 96-Well Thermal Cycler</td>
<td>Applied Biosystems</td>
<td>N/A</td>
</tr>
<tr>
<td>Genetic Analyzer Trade-in for 3500 Series System</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>NanoDrop™ 2000c Spectrophotometers</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#ND-2000C</td>
</tr>
<tr>
<td>Agilent 2200 TapeStation</td>
<td>Agilent Technologies</td>
<td>Cat#G2965A</td>
</tr>
<tr>
<td>BD FACS Aria™ II cell sorter</td>
<td>Becton, Dickinson and company</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Make 0.5 TrypLE by adding an equal amount of 1 × TrypLE Select and 0.5 mM EDTA. An example is below.</p>
<table id="undtbl5">
<caption>0.5 × TrypLE solution</caption>
<thead>
<tr>
<th>Solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 × TrypLE Select</td>
<td>0.5 ×</td>
<td>5 mL</td>
</tr>
<tr>
<td>0.5 mM EDTA</td>
<td>0.25 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for daily use. For long-term (&gt;1 week) storage, store in −30°C.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Accutase can be used instead of 0.5 × TrypLE solution.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Electroporation of iPSCs by MaxCyte ATx electroporator (option 1)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<div class="note">
<span class="note-title">Note:</span> If you use 4D-nucleofector as transfection method, skip steps 1–4 and go to step 5.</div>
<p>MaxCyte electroporation of iPSCs enables higher efficiency and viability compared with other electroporation platforms under the conditions we have tested in the laboratory. We believe higher cell viability may enable better HDR efficiency when performing genome editing with CRISPR-Cas9 RNP and an ssODN template.</p>
<ol>
<li>Day-1 or Day-2: Passage iPSCs one or two days before electroporation at a high cell density
<ol type="a">
<li>For pre-coating of a 6-well plate with 0.25 μg/cm<sup>2</sup> of iMatrix-511 (half of the usual concentration), prepare 2.5 μg of iMatrix-511 diluted in 2 mL of PBS. Then, coat wells at 37°C for at least 2 h in a humidified CO<sub>2</sub> incubator. If more than one well is being coated, a master mix of the PBS and iMatrix-511 can be made in a 50 mL conical centrifuge tube and then added to each well.</li>
<li>Wash iPSCs with 2 mL of PBS once and remove PBS.</li>
<li>Add 0.5 mL of Accutase to the well.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> 0.5 × TrypLE solution can be used instead of Accutase. Accutase may be less toxic for iPSCs than 0.5 × TrypLE solution, especially at the timing of electroporation.</div>
</li>
<li>Rock the plate every 3–4 min to ensure the Accutase covers the cells homogenously in the well.</li>
<li>Incubate the plate at 37°C for 10 min. The cells should become rounded in the plate.</li>
<li>Pipette the cells up and down in the Accutase solution with a 1000 μL pipette tip to detach the cells from the plate. Suspend the cells by pipetting while in the Accutase mixture to make sure the cells are dissociated well. Be careful not to pipette the cells too quickly to avoid shear stress, which will lower cell viability.</li>
<li>Transfer the cell suspension to a new 1.5 mL microcentrifuge tube containing 0.9 mL of StemFit media.</li>
<li>Centrifuge the cell suspension at 120 × <i>g</i> for 5 min.</li>
<li>Aspirate the supernatant.</li>
<li>Resuspend cells with 1 mL of StemFit media containing 10 μM Y-27632 by gentle pipetting.</li>
<li>Count cells and check cell viability by trypan blue staining. The ideal cell viability is greater than 90%. If the cell viability is lower than 80%, it may indicate that the overall cell health is not optimal and more sensitive to electroporation damage.</li>
<li>Plate 1.5 × 10<sup>6</sup> cells of live iPSCs in the pre-coated iMatrix-511 plate in 2 mL of StemFit media containing 10 μM Y-27632. Incubate in a CO<sub>2</sub> incubator at 37°C for one day.
<div class="critical">
<span class="critical-title">Critical:</span> The initial cell condition is critical. In our experience, passaging the iPSCs one or two days before electroporation has resulted in high HDR efficiencies. There may be some iPSC lines that are more fragile and may not respond well to passaging one day before electroporation. If this is the case, then the cells could be passaged two or three days before the above cell dissociation at a high cell density (i.e., 1.5–2.0 × 10<sup>6</sup> cells/well in case 6 well plate).</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Day 0: Preparation of the MaxCyte ATx electroporation instrument.
<ol type="a">
<li>Turn on the laptop PC connected to the MaxCyte ATx electroporation instrument at least 10 min before electroporation.</li>
<li>After logging into the laptop PC, turn on the ATx instrument.</li>
<li>Click the ATx shortcut icon on the Windows desktop.</li>
<li>Select desired protocol for the experiment. Optimization 8 was the electroporation protocol established in our laboratory with several cell lines for efficient delivery of a variety of loading agents including DNA plasmids, mRNA, and CRISPR-Cas9 RNP.</li>
<li>Select the OC-100 processing assembly from the processing assembly dropdown menu.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Day 0: Preparation of iPSCs for electroporation (<a href="#fig2">Figure 2</a>).
<ol type="a">
<li>6-well plate with 0.25 μg/cm<sup>2</sup> of iMatrix-511 (half of usual concentration) at 37°C for at least 1 h in a humidified CO<sub>2</sub> incubator.</li>
<li>Aspirate the coating buffer and add 2 mL of StemFit media containing 10 μM Y-27632 into each well, and incubate in a CO<sub>2</sub> incubator at 37°C for pre-warming the media until finish the dissociation of iPSCs</li>
<li>Perform the same procedures 1.b–i.</li>
<li>Add 1 mL of MaxCyte HyClone Buffer to the cell pellet and pipet up and down gently to dissociate and wash the pellet.</li>
<li>Count the cells and check for cell viability by trypan blue staining. The cell viability is suggested to be &gt;80 %. Lower cell viability may indicate cells are not in optimal condition.</li>
<li>Centrifuge at 120 × <i>g</i> for 5 min (at room temperature).</li>
<li>Aspirate MaxCyte HyClone Buffer.</li>
<li>Resuspend iPSC pellet in MaxCyte HyClone buffer at a cell density of 1 × 10<sup>7</sup> cells/mL. Take into account the volume of the cell pellet itself when adding the MaxCyte HyClone buffer. For example, if there is the cell pellet with 1 × 10<sup>6</sup> cells with around 10 μL of volume, add 90 μL of MaxCyte HyClone buffer to be the total 100 μL. For one electroporation condition, 50 μL of cell suspension is needed.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Higher cell densities up to 1 × 10<sup>8</sup> cells/mL can also be used without a loss in efficiency. Higher cell densities during electroporation can be considered in order to obtain a larger number of edited cells.</div>
</li>
<li>Prepare CRISPR-Cas9 RNP mixture by mixing 10 μg of CRISPR-Cas9 protein with 2.5 μg of IVT gRNA and incubate at room temperature for at least 10 min in a 1.5 mL microcentrifuge tube.
<div class="note">
<span class="note-title">Note:</span> The gRNA amount can be increased to 2–3 fold (5–7.5 μg). A higher ratio of gRNA to Cas9 protein ensures that free Cas9 is in a complex with gRNA, for improving knockout and knock-in efficiency.</div>
</li>
<li>Then add 10 μg of ssODN into the Cas9/gRNA complex.
<div class="critical">
<span class="critical-title">Critical:</span> If this concentration of ssODN is causing low cell viability after electroporation, the amount of the ssODN can be titrated down in order to achieve better viability. The amount of ssODN can be optimized between 5 and 12 μg. Note, we found that high ssODN concentration tends to give a higher HDR rate, but is more toxic for cells.</div>
</li>
<li>Add 50 μL of the iPSC cell suspension to the microcentrifuge tube containing the Cas9/gRNA complex and ssODN solution. Pipette up and down five times gently and avoid introducing air bubbles.</li>
<li>Transfer the cell mixture to the OC-100 processing assembly slowly but steadily, only going to the first stop of the pipette in order to avoid the introduction of air bubbles. The OC-100 could be tapped gently to ensure the solution spreads out in the processing assembly. The mixture of single cells and CRISPR-Cas9/gRNA complex is not required to be incubated on ice before the electroporation step.
<div class="critical">
<span class="critical-title">Critical:</span> An air bubble in the cuvette may lead to an abnormal electric current. Make sure to avoid the addition of air bubbles. If found, make sure to remove it by using a 10 μL pipette. Also, make sure your cell suspension solution is evenly distributed (importantly, touching both electrodes on the side walls) in the well of the cuvette, so that an electric field will be evenly applied.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not leave the iPSCs sitting in the MaxCyte buffer for an extended period of time before electroporation. This may result in sensitizing the cells to more damage during electroporation.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the purpose of the experiment is gene knockout, omit the step ”j” and proceed to the next step.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1179-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. MaxCyte ATx mediated electroporation of RNP complex into iPSCs</div>
<p>(A) Prepare a mixture of MaxCyte HyClone buffer with cells and RNP complex, and ssODN if applicable. Transfer 50 μL of the mixture into the well of OC-100 Processing Assembly chamber, after opening the lid. Then close the lid after the sample has been added to the OC-100.</p>
<p>(B) Operation of the instrument. Select the desired electroporation protocol (Optimization Energy 8), and processing assembly (OC-100). Finally insert the OC-100 processing assembly and start electroporation.</p>
</figcaption>
</figure>
<ol start="4">
<li>Day 0: Electroporation of iPSCs
<ol type="a">
<li>Insert the OC-100 into the MaxCyte ATx electroporation port.</li>
<li>Press “Start” on the instrument software.</li>
<li>The first electroporation takes about 40 s for the machine to charge up. The actual electroporation is around 2–3 s.</li>
<li>When you have more than 2 samples for electroporation, the electroporation from the 2<sup>nd</sup> sample will take only 2–3 s as you do not have to wait for the charge</li>
<li>After electroporation is completed, place the OC-100 into a 37°C humidified CO<sub>2</sub> incubator for 20–30 min to allow for cell membrane recovery.</li>
<li>After the recovery step, the cells can be transferred from the OC-100 to a well of 6-well plate with 2 mL of StemFit media containing 10 μM Y-27632 which is pre-coated with iMatrix-511 as described above.</li>
<li>(Optional step) Rinse the OC-100 processing assembly with 50 μL of StemFit media in order to recover residual cells and add them to the culture vessel.
<div class="note">
<span class="note-title">Note:</span> Cell viability can be affected by the seeding density. If you find significant cell death, try to use a smaller plate, such as 12-well plate for seeding.</div>
</li>
<li>Incubate the cells in 37°C for culturing. Y-27632 can be removed after 1–2 days of transfection.
<div class="optional">
<span class="optional-title">Optional:</span> if there are a lot of dead cells even after 2 days of transfection, keep adding Y-27632 in culturing media until cell recovery.</div>
</li>
<li>After 3–5 days of transfection when cells become semi-confluent, dissociate cells. Expand a part of cells when it is necessary to maintain the cells, make a cell stock or extract genomic DNA.
<div class="optional">
<span class="optional-title">Optional:</span> If there are enough cells, we recommend making a freeze stock of cells by STEM-CELLBANKER.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once genomic DNA was extracted and frozen cell stocks were made, you may pause at this point.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Electroporation of iPSCs by 4D-Nucleofector (option 2)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<div class="note">
<span class="note-title">Note:</span> If you use MaxCyte as transfection method, skip step 5–7 and go to step 8.</div>
<p>4D-nucleofector electroporation also enables high knockout and knock-in efficiency as the MaxCyte electroporator described above.</p>
<ol start="5">
<li>Day-1 - 0: Plate coating.
<ol type="a">
<li>Prepare a 6-well plate where each well contains 2 mL of StemFit media containing 10 μM Y-27632 as described in MaxCyte part 3. a, b.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Day 0: Set up the 4D-nucleofector program for transfection (<a href="#fig3">Figure 3</a>)
<div class="note">
<span class="note-title">Note:</span> This section can be done in parallel to “<a href="#olist0090">3. Preparation of iPSCs for electroporation</a>”</div>
<ol type="a">
<li>Turn on the 4D-nucleofector instrument.</li>
<li>Chose X unit.</li>
<li>Chose X-16 Strip (or X-single if you are using Single Nucleocuvette)</li>
<li>Select the wells you are going to transfect.</li>
<li>Set program as CA-137 and P4 primary buffer (or the buffer you are using).
<div class="optional">
<span class="optional-title">Optional:</span> If necessary, other programs (CM-138, CM137, CM-150, DS-137, DS-138, and DS-130) also work for electroporating human iPSCs.</div>
</li>
<li>Press “OK” for opening the tray port and proceed the cell preparation.</li>
</ol>
</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1179-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. 4D-Nucleofector mediated electroporation of RNP complex into iPSCs</div>
<p>(A) Prepare mixture of P4 primary buffer with cells and RNP complex, and ssODN if applicable. Transfer 20 μL of the mixture into one well of 16 well strips.</p>
<p>(B) Operation of the instrument. Select wells where your samples are, and then input pulse code (CA-137) and select solution (Primary Cell P4). Finally insert your 16 well strips and start electroporation.</p>
</figcaption>
</figure>
<ol start="7">
<li>Day 0: Preparation of iPSCs for electroporation
<ol type="a">
<li>Perform the cell dissociation procedures as described in “<a href="#olist0020">Passage of iPSCs</a>” section 2. b–d.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> For cell dissociation by Accutase, see the MaxCyte electroporation section.</div>
</li>
<li>Perform cell counting using a hemocytometer and trypan blue.</li>
<li>For one electroporation condition, add 3.0 × 10<sup>5</sup> cells per condition in a 1.5 mL tube. If there are 5 conditions, multiple by 5 (i.e., 1.5 × 10<sup>6</sup> cells).</li>
<li>Centrifuge at 120 × <i>g</i> for 5 min (at room temperature).</li>
<li>Aspirate supernatant.</li>
<li>Add 20 μL of P4 Primary Cell Solution per condition into the cell pellet and gently suspend at room temperature. If you have 5 conditions, add 100 μL of P4 Primary Cell Solution.
<div class="note">
<span class="note-title">Note:</span> P3 Primary Cell Solution can also be used, instead of P4 Primary Cell Solution.</div>
</li>
<li>Prepare CRISPR-Cas9 RNP mixture by mixing 5 μg of CRISPR-Cas9 protein with 1.25 μg of IVT gRNA and incubate at room temperature in a 1.5 mL microcentrifuge tube.
<div class="note">
<span class="note-title">Note:</span> 1.25 μg gRNA should be sufficient to 5 μg Cas9 in terms of molar ratio (roughly 1:1 molar ratio), however increasing gRNA to 2.5–3.75 μg may improve genome editing efficiency sometimes.</div>
</li>
<li>Add 10 (or 6–12) μg of ssODN as a template for single nucleotide alteration or loxP insertion.
<div class="critical">
<span class="critical-title">Critical:</span> If this concentration of ssODN causes low cell viability after electroporation, the amount of the ssODN can be titrated down in order to achieve better viability. However, if the ssODN amount is too low, it may reduce knock-in efficiency.</div>
</li>
<li>Transfer 20 μL of Cas9/gRNA and ssODN mixture into a well of Nucleocuvette Strip at room temperature. DO NOT make bubbles.
<div class="critical">
<span class="critical-title">Critical:</span> An air bubble in the cuvette causes an electric spark when high voltage is applied. Make sure to avoid air bubbles as possible. If you find one, make sure to remove it by using a P-10 pipette. Also, make sure your cell suspension solution is evenly distributed (especially, both sides of the electrode wall) in the well of the cuvette, so that electric field will be evenly applied.</div>
</li>
<li>Set Nucleocuvette strip into 4D-Nucleofector.</li>
<li>Press “Start” for starting electroporation.</li>
<li>After electroporation, add 50–80 μL of StemFit media with Y-27632 into each well of Nucleocuvette Strip, pipet up and down several times gently, and transfer all cells into one well of the 6-well plate with 2 mL of StemFit media containing 10 μM Y-27632 which pre-coated with iMatrix-511 as described in the “<a href="#o0010">Plate coating</a>” section.
<div class="note">
<span class="note-title">Note:</span> Cell viability can be affected by the seeding density. If you find significant cell death, try to use smaller plate, such as 12-well or 24-well plate for seeding.</div>
</li>
<li>Incubate the cells at 37°C for culturing. Y-27632 can be removed after 1–2 days of transfection.
<div class="optional">
<span class="optional-title">Optional:</span> if you see a lot of dead cells even after 2 days of transfection, you can keep adding Y-27632 in culturing media until cell recovery.</div>
</li>
<li>After 4–7 days of transfection when cells become semi-confluent, dissociate cells. Expand a part of cells when it is necessary to maintain the cells, make a cell stock or extract genomic DNA.
<div class="optional">
<span class="optional-title">Optional:</span> If there is a sufficient cell number, we recommend to make a freeze stock of cells by STEM-CELLBANKER.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the purpose of the experiment is gene knockout, omit the step “h” and proceed to the next step.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once genomic DNA was extracted and frozen cell stocks were made, you may pause at this point.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Assessment of genome editing efficiency in bulk iPSCs by Sanger sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 day(s)</div>
<p>Assess overall genome editing efficiency in the electroporated iPSCs. This is an important step to validate your genome editing procedure worked or not, and to estimate how many subclones will be necessary to genotype from the bulk cells.</p>
<ol start="8">
<li>Genomic extraction
<ol type="a">
<li>Extract genomic DNA from the cell pellet by using MonoFas gDNA Cultured Cells Extraction Kit VI. This will be used for analyzing knockout efficiency in the next section. As a negative control, make sure to prepare genomic DNA from non-transfected iPSCs as well.</li>
</ol>
</li>
</ol>
<ol start="9">
<li>PCR and sequencing
<ol type="a">
<li>Design a pair of PCR primers to amplify the target locus by using Primer-blast or Primer3 software. Preferred PCR band size is around 500–800 bp for Sanger sequencing, and 150–200 bp for gel electrophoresis analysis for loxP insertion. The gRNA target locus should be located in the middle part of the amplified region.</li>
<li>Amplify the target locus from the genomic sample(s) by PCR. Make sure to amplify from the non-genome edited sample as a negative control. PCR Reaction:
<table id="undtbl6">
<thead>
<tr>
<th>PCR reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ultrapure H<sub>2</sub>O</td>
<td> </td>
<td>Up to 50 μL</td>
</tr>
<tr>
<td>5 × PrimeSTAR GXL buffer</td>
<td>1 ×</td>
<td>10 μL</td>
</tr>
<tr>
<td>dNTPs Mix (2.5mM each)</td>
<td>0.2 mM</td>
<td>4 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>0.3 μM</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>0.3 μM</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>Genomic DNA</td>
<td>100–500 ng/50 μL</td>
<td>X μL</td>
</tr>
<tr>
<td>PrimeSTAR GXL DNA Polymerase</td>
<td>1.25 U/50 μL</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td>
<b>50</b> μL</td>
</tr>
</tbody>
</table>
PCR cycling conditions
<table id="undtbl7">
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>X°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>1 min/kb</td>
</tr>
<tr>
<td>Final extension</td>
<td>68°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">∞</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Try a higher amount (0.5–2 μg) of template genomic DNA if PCR amplification didn’t happen.</div>
</li>
<li>Run on a 2% agarose gel to confirm a specific amplification. If a loxP sequence is inserted, you may see an additional band with 34-bp larger size. If you see non-specific amplification, optimize the PCR condition (i.e., raise annealing temperature). If desired, you may perform gel extraction to exclude the non-specific bands.</li>
<li>Send the PCR sample for Sanger sequencing service. (i.e., GENEWIZ)</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Decomposition of Sanger sequence result by using ICE (Inference of CRISPR Edits) CRISPR Analysis tool.
<ol type="a">
<li>After you received Sangar sequencing result as “ab1” file, access to ICE web site and upload a control sample file and genome edited sample file. Alternatively, TIDE website can also be utilized.</li>
<li>An example of ICE analysis for detecting NHEJ events is shown in <a href="#fig4">Figure 4</a>.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1179-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Indel pattern analysis by Sanger sequencing and ICE analysis</div>
<p>(A) Cas9 and gRNA (DMD-in55-g3-gRNA: gactttatagatatctccca tgg) complex was electroporated into Ff-XT28s05 iPSCs by using 4D-nucleofector and P4 primary cell buffer. Sequencing results of the control sample (non-transfection) and experiment sample (transfected) were shown in upper panel. In the experiment sample, a mixture of spectrum can be observed after the Cas9 cleavage site.</p>
<p>(B) Analysis result by ICE software. Indel percentage of the above Sanger sequencing data analyzed by ICE was 92 %, and knockout-score (exclude in-frame indels) was 83 %.</p>
</figcaption>
</figure>
</li>
<li>Select the best condition (i.e., choice of gRNA or amount of ssODN) which shows the highest knock-in efficiency. The overall rate of expected genome editing outcomes is critical. If the efficiency is greater than 20–30%, proceed to the next subcloning step. If the efficiency is less than 10%, we recommend optimizing the experimental design or electroporation condition. Otherwise, it is necessary to screen hundreds of subclones for identifying a genome-edited clone."</li>
<li>Expand the iPSCs that were confirmed for successful genome editing as described in the previous section. Make some frozen stocks as necessary.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Isolation of subclones of iPSCs and genotyping</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 weeks</div>
<ol start="11">
<li>Day-7 - 0: Preparation of subcloning
<ol type="a">
<li>Harvest the semi-confluent iPSCs from the above section by 0.5 × TrypLE Select.</li>
<li>Coat three 96-well plates with iMatrix-511 at least 1–24 h before cell seeding.</li>
<li>If a cell sorter is available, proceed to single-cell sorting as described in step <b>12.</b> If not, proceed to limit dilution as described in step <b>13.</b>
</li>
</ol>
</li>
<li>Day 0: Sub-cloning by single cell sorting
<ol type="a">
<li>Perform the cell dissociation procedures as described in “Preparation of human iPSCs” section 2. b–d.</li>
<li>Perform cell counting using a hemocytometer and trypan blue.</li>
<li>Add 1.0–10.0 × 10<sup>5</sup> of live cells in a 1.5 mL tube.</li>
<li>Centrifuge at 120 × <i>g</i> for 5 min (at room temperature).</li>
<li>Remove supernatant.</li>
<li>Gently resuspend cells in 300 μL of FACS buffer (2% FBS in PBS). Transfer into a 5 mL flow tube with cell strainer to remove cell clumps.</li>
<li>Use a cell sorter (i.e., BD FACS Aria II) to sort 1 cell/well of 96-well plate which prepared at part d. of “Day-7–0: Preparation of sub cloning”.</li>
<li>Incubate the sorted 96-well plate at 37°C in a CO<sub>2</sub> incubator.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Day 0: Sub-cloning by limit dilution <b>(Optional</b>)
<ol type="a">
<li>Perform the cell dissociation procedures as described in “Preparation of human iPSCs” section 2. b–d.</li>
<li>Perform cell counting using a hemocytometer and trypan blue.</li>
<li>Transfer 1.0 × 10<sup>4</sup> of cells into a 15 mL tube with 10 mL of StemFit media containing 10 μM Y-27632.</li>
<li>Further dilute the cell suspension by transferring 10 μL, 30 μL, and 100 μL into three 15 mL tubes with 10 mL of StemFit media and 10 μM Y-27632, so that the final concentration of the cells should be 10 cells, 30 cells, and 100 cells per 10 mL.
<div class="note">
<span class="note-title">Note:</span> The number of single cell-derived subclones varies greatly from experiment to experiment, so we recommend to prepare three 96-well plates with different seeding densities (Suggested final seeding density for each plate could be 0.25 cells / 100 μL, 0.5 cells / 100 μL, and 1 cell / 100 μL, respectively). If the genome editing efficiency at the bulk stage was less than 10%–20%, preparation of more 96-well plates is recommended.</div>
</li>
<li>Remove iMatrix coating buffer from the plate prepared in “Day-7–0: Preparation of sub cloning”.</li>
<li>Add 100 μL of the cell suspension into one well of 96-well plate by using a multi-channel pipette. Make sure to suspend the cells frequently, to avoid uneven cell seeding.</li>
<li>Incubate the plates at 37°C in a 5% CO<sub>2</sub> incubator.</li>
<li>The next day, observe each well of the 96-well plates under a microscope and mark the wells with single-cell seeding. A well with more than one cell should be discarded.</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Day 2–14: After cell sorting or limit dilution (common step)
<ol type="a">
<li>After the single-cell seeding into a 96-well plate, there is a risk of cell loss during medium change. Instead of complete removal of media, add 50 μL of fresh StemFit media with 10 μM Y-27632 into a well of 96-well plate every 2 or 3 days.</li>
<li>On day 6 or day 7, change media with StemFit (without Y-27632).</li>
<li>Colonies will be visible at around day 7–10. Mark the wells with a single colony of iPS. If more than two colonies were observed, discard such wells.</li>
<li>Add media for the wells with a single iPSC colony every 2 days (add 50 μL for 96-well plate)</li>
<li>Around day 9-11, once the iPSC colony will become 100–300 μm in diameter, passage all the cells from one well of 96-well plate into a well of 24-well plate.</li>
<li>Change media every 2 days until iPSCs become semi-confluent.</li>
</ol>
</li>
<li>Day 14–21: After passaging cells from a single-cell derived colony
<ol type="a">
<li>When passaged cells become semi-confluent, extract genomic DNA from half of the cells by using MonoFas gDNA Cultured Cells Extraction Kit VI or a similar genome DNA extraction kit (i.e., Direct PCR). As the starting cell number is low, elute genomic DNA in a smaller volume (i.e., 40 μL) of elution buffer. Make a vial of freeze stock from the other half the cells using STEM-CELLBANKER.</li>
<li>Repeat step a. for all single clones.</li>
<li>When the number of samples has accumulated to some extent, perform PCR amplification of the target region and Sanger sequencing as described in the “<a href="#sec3.3">assessment of genome editing efficiency in bulk iPSCs by Sanger sequencing</a>” section. Repeat this step for all the isolated subclones or until desired subclones are identified.</li>
<li>Choose the subclones which contain the expected knockout or knock-in sequence. More than 3 subclones are recommended for further downstream analysis (i.e., check for protein expression, cellular phenotype, pluripotency test, differentiation capacity, and genomic integrity).</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We examined if it is possible to induce a large (&gt;300 kb) deletion using Cre-loxP reaction without antibiotic selection in iPSCs. As a proof-of-concept, we thought to introduce a 342 kb deletion between exon 45 and 55 of the dystrophin gene. We performed targeted insertion of two loxP sites by two rounds of ssODN mediated HDR. To insert two loxP sequences around exon 45 and exon 55, we designed two gRNAs to target each site, respectively. The loxP insertion site is adjusted to the Cas9 cleavage site, which is located at 3 bp upstream of the PAM sequence as shown in <a href="#fig1">Figure 1</a>B. Donor template ssODN for loxP insertion contains a 34 bp loxP site on the center and 50 bp homology arms on each side, that are homologous to the target locus on the exon 45 or 55.</p>
<p>We electroporated Cas9 protein and IVT-sgRNA (DMD-sgRNA1) complex together with the ssODN template (DMD1 + loxP-ssODN) into male iPSCs (1383D2 line) using MaxCyte with Optimization Energy 8. The first loxP knock-in was confirmed in bulk electroporated cells by genomic DNA PCR and subsequent restriction enzyme digestion by XmnI (<a href="#fig5">Figure 5</a>A). The insertion efficiency of loxP was more than 20% (<a href="#fig5">Figure 5</a>B) without antibiotic selection.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1179-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Insertion of loxP sequence using ssODN</div>
<p>(A) Schematic to insert a loxP sequence at the DMD gene locus in 1383D2 iPSCs. Successful insertion of a loxP site generates an extra XmnI restriction enzyme site, which can be detected by DNA electrophoresis.</p>
<p>(B) Cas9, gRNA, and ssODN were electroporated into 1383D2 iPSCs and genomic DNA was extracted for PCR amplification. TapeStation analysis of the PCR products with or without XmnI digestion. With the RNP and ssODN electroporation condition, extra bands with expected sizes (229 bp and 67 bp) were observed, suggesting the loxP sequence is inserted at the target site with around 24% of efficiency. This figure is adapted from <a href="#bib14">Kagita et al. (2021)</a>.</p>
</figcaption>
</figure>
<p>After confirmation of the 1st loxP insertion in bulk cells by restriction enzyme digest, we isolated subclones by single-cell cloning into a 96-well plate using BD FACS Aria cell sorter. Sanger sequencing of 20 subclones revealed that 8 subclones (40%) showed the intact loxP insertion, 8 subclones (40%) had unwanted indels, and 4 subclones (20%) were unedited. After finishing the first round of genome editing, we inserted the second loxP site around the exon 55 region by ssODN-mediated knock-in using the gRNA (DMD-in55-g3) and ssODN template (DMD-in55-g3+loxP-ssODN). Out of 14 subclones we analyzed by Sanger sequencing, 2 subclones showed successful insertion of the second loxP sequence.</p>
<p>To induce the excision of the 342 kb region flanked by the two loxP sites, 2 floxed clones were seeded into a 24-well plate (1.5–3.0 × 10<sup>6</sup> cells/well) and 500 ng of tamoxifen-inducible Cre expressing plasmid (pCAG-IP-MerCreMer) were transfected with 1.5 μL of Lipofectamine Stem Transfection Reagent. 4–6 h after the transfection, 1 μM 4-hydroxytamoxifen was added to the media to induce nuclear translocation of Cre recombinase fused with a modified estrogen receptor (Mer). The excision of the floxed site (in this case, DMD exon 45–55 region) was confirmed by genomic DNA PCR in the bulk edited cells using “DMDexon55(45-55)check_dir5” and “DMDexon45(45–55)check_rev5” primers. In this PCR, the specific amplification appears only when the floxed site is excised. Then, subclones were isolated into several 96-well plates by a cell sorter and genotyped by the same PCR primers. We also performed Sanger sequencing of the PCR products and found that 4 subclones out of 11 clones showed excision of the DMD exon 45–55 region.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Our protocol is not necessarily applicable for all the genomic DNA sites because the CRISPR-Cas9 target site is restricted by PAM sequence specificity (NGG: in the case of SpCas9 gRNA). When gRNA activity is low, it is difficult to perform knock-in at a high frequency.</p>
<p>Since ssODN-mediated knock-in utilizes the HDR DNA repair pathway, the HDR pathway is required to be active (<a href="#bib23">Paquet et al., 2016</a>; <a href="#bib25">Richardson et al., 2016</a>). The HDR pathway is only active during the S phase and the G2 phase, hence the target cells should be actively proliferating (<a href="#bib11">Gutschner et al., 2016</a>; <a href="#bib12">Howden et al., 2016</a>; <a href="#bib17">Mao et al., 2008</a>) and our protocol is not applicable for non-proliferating cells.</p>
<p>Genome editing efficiency highly depends on the condition of the iPSC line to be utilized. Some iPSC lines might be difficult to handle due to slow growth, low survival, or spontaneous differentiation. In general, such iPSC lines are difficult to transfect and to edit by genome editing.</p>
<p>If there is a Single nucleotide polymorphism (SNP) in your gRNA target sequence, the SNP most likely interferes with the gRNA binding and subsequent Cas9 cleavage. Therefore, it is recommended to check the target sequence before designing gRNA.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No indel detected by Cas9/gRNA, without ssODN.</p>
<p>Step 10b at "<a href="#sec3.3">assessment of genome editing efficiency in bulk iPSCs by Sanger sequencing</a>" section.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Use a positive control gRNA which has been shown to have cleavage activity in previous studies.</p>
<p>If your gRNA does not show cleavage activity despite the fact that the control gRNA does, check the sequence used in IVT and check the target sequence by Sanger sequencing.</p>
<p>Because Cas9/gRNA cleavage depends not only on the target sequence but also on the epigenetic state and many other factors. Therefore, no matter how carefully you design gRNA, some gRNAs have little cleavage activity. In that case, gRNA should be re-designed away from the original target site. To minimize the time burden, we recommend designing several gRNAs for each target site from the beginning.</p>
<p>In addition, previous studies have reported that gRNAs with extremely high or low GC content have low cleavage activity, so it is desirable that GC content of gRNA is within 40%–60% (<a href="#bib9">Doench et al., 2014</a>; <a href="#bib30">Wang et al., 2014</a>). It is necessary to avoid poly T (“TTTTT”) sequence from gRNA because it is a transcription termination signal when transcribed from polymerase III promoter.</p>
<p>If even a positive control gRNA does not show cleavage activity, re-check the condition of iPSCs and the RNP transduction step. Highly efficient genome editing is feasible only when iPSCs grow steadily in an undifferentiated state. In the RNP transduction step, it is important to mix Cas9 protein and gRNA well, because Cas9 protein is unstable until it binds to gRNA.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Error in ICE or TIDE analysis and gRNA activity cannot be assessed.</p>
<p>Step 10b at "<a href="#sec3.3">assessment of genome editing efficiency in bulk iPSCs by Sanger sequencing</a>" section</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Although ICE or TIDE are useful because they analyze indels using Sanger sequence data, high quality Sanger electrogram data are necessary for analysis and the results are not reliable if the quality of the data is low (<a href="#bib5">Brinkman et al., 2014</a>). Nonspecific amplification in PCR causes a background noise. To avoid this, we recommend optimizing the PCR condition for specific amplification and performing agarose gel extraction of the PCR product.</p>
<p>If a sequence primer is designed close to the target site, it is not feasible to analyze the target sequence. Therefore, a sequence primer should be designed approximately 100 bp away from the target site. When analyzing a region that is difficult to sequence, such as a GC-rich region or a repeat region, re-consider a sequencing reaction or change the position of the primer.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Although indels were actively induced by NHEJ, single nucleotide conversion/insertion via HDR was not detected.</p>
<p>Step 10b at "<a href="#sec3.3">assessment of genome editing efficiency in bulk iPSCs by Sanger sequencing</a>" section</p>
<h3 id="sec6.6">Potential solution</h3>
<p>If the HDR mediated conversion/insertion does not occur even though double strand breaks are induced sufficiently, check the sequence of ssODN whether the homology arm matches the genomic DNA. Also,check the concentration of ssODN and the condition of the electroporation are appropriate. The optimal range of ssODN amount (typically 6–12 μg) varies depending on targeting locus and iPSC lines (<a href="#bib13">Ishida et al., 2018</a>). Optimize the amount of ssODN for each locus, each iPSC line.</p>
<p>It is important to use iPSCs in a logarithmic growth phase, because HDR occurs only in the late S and the G2 phase, and ssODN can reach the nucleus to serve as a template for HDR only when nuclear envelope disappears in the M phase (<a href="#bib12">Howden et al., 2016</a>; <a href="#bib17">Mao et al., 2008</a>).</p>
<p>The purity of synthesized ssODN affects electroporation efficiency and cell viability. To boost the HDR efficiency, it is worth trying high-purity purification methods such as HPLC.</p>
<p>HDR rate over NHEJ event is the locus- and cell type-dependent (<a href="#bib18">Miyaoka et al., 2016</a>). If you found it difficult to achieve HDR despite successful NHEJ events, consider changing the target site, or using a long, circular DNA template (i.e., plasmid DNA) with antibiotic resistance gene, to perform antibiotic selection of rare knock-in cells.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>After transfection, the cell condition is poor (such as low viability, differentiated, or do not grow).</p>
<p>Step 4i at the "<a href="#sec3.1">electroporation of iPSCs by MaxCyte ATx electroporator (option 1)</a>" section or Step 7n at the "<a href="#sec3.2">electroporation of iPSCs by 4D-Nucleofector (option 2)</a>" section</p>
<h3 id="sec6.8">Potential solution</h3>
<p>It is inevitable that 20%–80% of cells die after transfection, but usually the cells recover from the damage within several days. If the cells do not grow at all or differentiate, it is possible that the transfection condition is too strong, therefore try a milder electric program or reduce ssODN amount. After electroporation, seeding into a smaller well might help as well to keep cell confluency.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>After sub-cloning, the cell condition is poor (such as low viability, differentiation, or loss of proliferation).</p>
<p>Steps at the "<a href="#sec3.4">isolation of subclones of iPSCs and genotyping</a>" section</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If the cells stop proliferation or get differentiated with the target gRNA sample, but not with the control gRNA, it is possible that your target is an essential gene of PSCs. If the problem is caused by the target gene after considering several gRNAs, it is necessary to use an inducible CRISPR system (<a href="#bib13">Ishida et al., 2018</a>) or create floxed allele to perform conditional knockout.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bamshad, M.J., Ng, S.B., Bigham, A.W., Tabor, H.K., Emond, M.J., Nickerson, D.A., and Shendure, J. (2011). Exome sequencing as a tool for Mendelian disease gene discovery. Nat. Rev. Genet. <i>12</i>, 745-755. <a class="external-link" href="https://doi.org/10.1038/nrg3031">https://doi.org/10.1038/nrg3031</a></p>
<p id="bib4">Branda, C.S. and Dymecki, S.M. (2004). Talking about a revolution: the impact of site-specific recombinases on genetic analyses in mice. Dev. Cell <i>6</i>, 7-28. <a class="external-link" href="https://doi.org/10.1016/s1534-5807(03)00399-x">https://doi.org/10.1016/s1534-5807(03)00399-x</a></p>
<p id="bib5">Brinkman, E.K., Chen, T., Amendola, M., and van Steensel, B. (2014). Easy quantitative assessment of genome editing by sequence trace decomposition. Nucleic Acids Res. <i>42</i>, e168. <a class="external-link" href="https://doi.org/10.1093/nar/gku936">https://doi.org/10.1093/nar/gku936</a></p>
<p id="bib6">Collins, F.S., Guyer, M.S., and Chakravarti, A. (1997). Variations on a theme: cataloging human DNA sequence variation. Science <i>278</i>, 1580-1581. <a class="external-link" href="https://doi.org/10.1126/science.278.5343.1580">https://doi.org/10.1126/science.278.5343.1580</a></p>
<p id="bib7">Concordet, J.P. and Haeussler, M. (2018). CRISPOR: intuitive guide selection for CRISPR/Cas9 genome editing experiments and screens. Nucleic Acids Res. <i>46</i>, W242-W245. <a class="external-link" href="https://doi.org/10.1093/nar/gky354">https://doi.org/10.1093/nar/gky354</a></p>
<p id="bib8">Dastidar, S., Ardui, S., Singh, K., Majumdar, D., Nair, N., Fu, Y., Reyon, D., Samara, E., Gerli, M.F.M., Klein, A.F., et al. (2018). Efficient CRISPR/Cas9-mediated editing of trinucleotide repeat expansion in myotonic dystrophy patient-derived iPS and myogenic cells. Nucleic Acids Res. <i>46</i>, 8275-8298. <a class="external-link" href="https://doi.org/10.1093/nar/gky548">https://doi.org/10.1093/nar/gky548</a></p>
<p id="bib9">Doench, J.G., Hartenian, E., Graham, D.B., Tothova, Z., Hegde, M., Smith, I., Sullender, M., Ebert, B.L., Xavier, R.J., and Root, D.E. (2014). Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation. Nat. Biotechnol. <i>32</i>, 1262-1267. <a class="external-link" href="https://doi.org/10.1038/nbt.3026">https://doi.org/10.1038/nbt.3026</a></p>
<p id="bib10">Gee, P., Lung, M.S.Y., Okuzaki, Y., Sasakawa, N., Iguchi, T., Makita, Y., Hozumi, H., Miura, Y., Yang, L.F., Iwasaki, M., et al. (2020). Extracellular nanovesicles for packaging of CRISPR-Cas9 protein and sgRNA to induce therapeutic exon skipping. Nat. Commun. <i>11</i>, 1334. <a class="external-link" href="https://doi.org/10.1038/s41467-020-14957-y">https://doi.org/10.1038/s41467-020-14957-y</a></p>
<p id="bib11">Gutschner, T., Haemmerle, M., Genovese, G., Draetta, G.F., and Chin, L. (2016). Post-translational regulation of Cas9 during G1 enhances homology-directed repair. Cell Rep. <i>14</i>, 1555-1566. <a class="external-link" href="https://doi.org/10.1016/j.celrep.2016.01.019">https://doi.org/10.1016/j.celrep.2016.01.019</a></p>
<p id="bib12">Howden, S.E., McColl, B., Glaser, A., Vadolas, J., Petrou, S., Little, M.H., Elefanty, A.G., and Stanley, E.G. (2016). A Cas9 variant for efficient generation of indel-free knockin or gene-corrected human pluripotent stem cells. Stem Cell Reports <i>7</i>, 508-517. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2016.07.001">https://doi.org/10.1016/j.stemcr.2016.07.001</a></p>
<p id="bib13">Ishida, K., Xu, H., Sasakawa, N., Lung, M.S.Y., Kudryashev, J.A., Gee, P., and Hotta, A. (2018). Site-specific randomization of the endogenous genome by a regulatable CRISPR-Cas9 piggyBac system in human cells. Sci. Rep. <i>8</i>, 310. <a class="external-link" href="https://doi.org/10.1038/s41598-017-18568-4">https://doi.org/10.1038/s41598-017-18568-4</a></p>
<p id="bib14">Kagita, A., Lung, M.S.Y., Xu, H., Kita, Y., Sasakawa, N., Iguchi, T., Ono, M., Wang, X.H., Gee, P., and Hotta, A. (2021). Efficient ssODN-mediated targeting by avoiding cellular inhibitory RNAs through precomplexed CRISPR-Cas9/sgRNA ribonucleoprotein. Stem Cell Reports <i>16</i>, 985-996. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.02.013">https://doi.org/10.1016/j.stemcr.2021.02.013</a></p>
<p id="bib15">Kim, S., Kim, D., Cho, S.W., Kim, J., and Kim, J.S. (2014). Highly efficient RNA-guided genome editing in human cells via delivery of purified Cas9 ribonucleoproteins. Genome Res. <i>24</i>, 1012-1019. <a class="external-link" href="https://doi.org/10.1101/gr.171322.113">https://doi.org/10.1101/gr.171322.113</a></p>
<p id="bib16">Li, H.L., Fujimoto, N., Sasakawa, N., Shirai, S., Ohkame, T., Sakuma, T., Tanaka, M., Amano, N., Watanabe, A., Sakurai, H., et al. (2015). Precise correction of the dystrophin gene in duchenne muscular dystrophy patient induced pluripotent stem cells by TALEN and CRISPR-Cas9. Stem Cell Reports <i>4</i>, 143-154. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2014.10.013">https://doi.org/10.1016/j.stemcr.2014.10.013</a></p>
<p id="bib17">Mao, Z., Bozzella, M., Seluanov, A., and Gorbunova, V. (2008). DNA repair by nonhomologous end joining and homologous recombination during cell cycle in human cells. Cell Cycle <i>7</i>, 2902-2906. <a class="external-link" href="https://doi.org/10.4161/cc.7.18.6679">https://doi.org/10.4161/cc.7.18.6679</a></p>
<p id="bib18">Miyaoka, Y., Berman, J.R., Cooper, S.B., Mayerl, S.J., Chan, A.H., Zhang, B., Karlin-Neumann, G.A., and Conklin, B.R. (2016). Systematic quantification of HDR and NHEJ reveals effects of locus, nuclease, and cell type on genome-editing. Sci. Rep. <i>6</i>, 23549. <a class="external-link" href="https://doi.org/10.1038/srep23549">https://doi.org/10.1038/srep23549</a></p>
<p id="bib19">Naito, Y., Hino, K., Bono, H., and Ui-Tei, K. (2015). CRISPRdirect: software for designing CRISPR/Cas guide RNA with reduced off-target sites. Bioinformatics <i>31</i>, 1120-1123. <a class="external-link" href="https://doi.org/10.1093/bioinformatics/btu743">https://doi.org/10.1093/bioinformatics/btu743</a></p>
<p id="bib20">Nakagawa, M., Taniguchi, Y., Senda, S., Takizawa, N., Ichisaka, T., Asano, K., Morizane, A., Doi, D., Takahashi, J., Nishizawa, M., et al. (2014). A novel efficient feeder-free culture system for the derivation of human induced pluripotent stem cells. Sci. Rep. <i>4</i>, 3594. <a class="external-link" href="https://doi.org/10.1038/srep03594">https://doi.org/10.1038/srep03594</a></p>
<p id="bib21">Okita, K., Matsumura, Y., Sato, Y., Okada, A., Morizane, A., Okamoto, S., Hong, H., Nakagawa, M., Tanabe, K., Tezuka, K., et al. (2011). A more efficient method to generate integration-free human iPS cells. Nat. Methods <i>8</i>, 409-412. <a class="external-link" href="https://doi.org/10.1038/nmeth.1591">https://doi.org/10.1038/nmeth.1591</a></p>
<p id="bib22">Paix, A., Folkmann, A., Goldman, D.H., Kulaga, H., Grzelak, M.J., Rasoloson, D., Paidemarry, S., Green, R., Reed, R.R., and Seydoux, G. (2017). Precision genome editing using synthesis-dependent repair of Cas9-induced DNA breaks. Proc. Natl. Acad. Sci. U S A <i>114</i>, E10745-E10754. <a class="external-link" href="https://doi.org/10.1073/pnas.1711979114">https://doi.org/10.1073/pnas.1711979114</a></p>
<p id="bib23">Paquet, D., Kwart, D., Chen, A., Sproul, A., Jacob, S., Teo, S., Olsen, K.M., Gregg, A., Noggle, S., and Tessier-Lavigne, M. (2016). Efficient introduction of specific homozygous and heterozygous mutations using CRISPR/Cas9. Nature <i>533</i>, 125-129. <a class="external-link" href="https://doi.org/10.1038/nature17664">https://doi.org/10.1038/nature17664</a></p>
<p id="bib24">Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F. (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. <i>8</i>, 2281-2308. <a class="external-link" href="https://doi.org/10.1038/nprot.2013.143">https://doi.org/10.1038/nprot.2013.143</a></p>
<p id="bib25">Richardson, C.D., Ray, G.J., DeWitt, M.A., Curie, G.L., and Corn, J.E. (2016). Enhancing homology-directed genome editing by catalytically active and inactive CRISPR-Cas9 using asymmetric donor DNA. Nat. Biotechnol. <i>34</i>, 339-344. <a class="external-link" href="https://doi.org/10.1038/nbt.3481">https://doi.org/10.1038/nbt.3481</a></p>
<p id="bib27">Sauer, B. and Henderson, N. (1988). Site-specific DNA recombination in mammalian cells by the Cre recombinase of bacteriophage P1. Proc. Natl. Acad. Sci. U S A <i>85</i>, 5166-5170. <a class="external-link" href="https://doi.org/10.1073/pnas.85.14.5166">https://doi.org/10.1073/pnas.85.14.5166</a></p>
<p id="bib28">Soldner, F. and Jaenisch, R. (2018). Stem cells, genome editing, and the path to translational medicine. Cell <i>175</i>, 615-632. <a class="external-link" href="https://doi.org/10.1016/j.cell.2018.09.010">https://doi.org/10.1016/j.cell.2018.09.010</a></p>
<p id="bib29">Untergasser, A., Cutcutache, I., Koressaar, T., Ye, J., Faircloth, B.C., Remm, M., and Rozen, S.G. (2012). Primer3—new capabilities and interfaces. Nucleic Acids Res. <i>40</i>, e115. <a class="external-link" href="https://doi.org/10.1093/nar/gks596">https://doi.org/10.1093/nar/gks596</a></p>
<p id="bib30">Wang, T., Wei, J.J., Sabatini, D.M., and Lander, E.S. (2014). Genetic screens in human cells using the CRISPR-Cas9 system. Science <i>343</i>, 80-84. <a class="external-link" href="https://doi.org/10.1126/science.1246981">https://doi.org/10.1126/science.1246981</a></p>
<p id="bib31">Watanabe, K., Ueno, M., Kamiya, D., Nishiyama, A., Matsumura, M., Wataya, T., Takahashi, J.B., Nishikawa, S., Nishikawa, S., Muguruma, K., and Sasai, Y. (2007). A ROCK inhibitor permits survival of dissociated human embryonic stem cells. Nat. Biotechnol. <i>25</i>, 681-686. <a class="external-link" href="https://doi.org/10.1038/nbt1310">https://doi.org/10.1038/nbt1310</a></p>
<p id="bib36">Xu, H., Wang, B., Ono, M., Kagita, A., Fujii, K., Sasakawa, N., Ueda, T., Gee, P., Nishikawa, M., Nomura, M., et al. (2019). Targeted Disruption of HLA Genes via CRISPR-Cas9 Generates iPSCs with Enhanced Immune Compatibility. Cell Stem Cell <i>24</i>, 566-578. <a class="external-link" href="https://doi.org/10.1016/j.stem.2019.02.005">https://doi.org/10.1016/j.stem.2019.02.005</a></p>
<p id="bib32">Ye, J., Coulouris, G., Zaretskaya, I., Cutcutache, I., Rozen, S., and Madden, T.L. (2012). Primer-BLAST: a tool to design target-specific primers for polymerase chain reaction. BMC Bioinformatics <i>13</i>, 134. <a class="external-link" href="https://doi.org/10.1186/1471-2105-13-134">https://doi.org/10.1186/1471-2105-13-134</a></p>
<p id="bib33">Zetsche, B., Gootenberg, J.S., Abudayyeh, O.O., Slaymaker, I.M., Makarova, K.S., Essletzbichler, P., Volz, S.E., Joung, J., van der Oost, J., Regev, A., et al. (2015). Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR-Cas system. Cell <i>163</i>, 759-771. <a class="external-link" href="https://doi.org/10.1016/j.cell.2015.09.038">https://doi.org/10.1016/j.cell.2015.09.038</a></p>
<p id="bib34">Zhong, A., Li, M., and Zhou, T. (2020). Protocol for the generation of human pluripotent reporter cell lines using CRISPR/Cas9. STAR Protoc. <i>1</i>, 100052. <a class="external-link" href="https://doi.org/10.1016/j.xpro.2020.100052">https://doi.org/10.1016/j.xpro.2020.100052</a></p>
<p id="bib35">Zuris, J.A., Thompson, D.B., Shu, Y., Guilinger, J.P., Bessen, J.L., Hu, J.H., Maeder, M.L., Joung, J.K., Chen, Z.Y., and Liu, D.R. (2015). Cationic lipid-mediated delivery of proteins enables efficient protein-based genome editing in vitro and in vivo. Nat. Biotechnol. <i>33</i>, 73-80. <a class="external-link" href="https://doi.org/10.1038/nbt.3081">https://doi.org/10.1038/nbt.3081</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Akitsu Hotta (<a href="mailto:akitsu.hotta@cira.kyoto-u.ac.jp">akitsu.hotta@cira.kyoto-u.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>Genome edited iPSCs described in <a href="#bib14">Kagita et al. (2021)</a> will be available upon reasonable request, under MTA and ethical committee approval to use iPSCs derived from volunteers.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We thank Dr. Yusuke Kojima and Akane Ichiki for critical reading of the manuscript, Kanae Mitsunaga for FACS sorting assistance, and MaxCyte Inc. for technical assistance. This study is in part funded by <a href="https://doi.org/10.13039/100009619">Japan Agency for Medical Research and Development</a> (AMED) grants (JP21bm0104001, JP21bm0804005, JP21ek0109293, JP21bm0404037), a Japan Society for the Promotion of Science (JSPS) KAKENHI grant (19H02924), and a <a href="https://doi.org/10.13039/501100009438">National Center of Neurology and Psychiatry</a> (NCNP) grant (no.28-6). Y.K. is supported by <a href="https://doi.org/10.13039/501100001691">JSPS DC2</a> fellowship (21J13593).</p>
<h3>Author contributions</h3>
<p>H.X., Y.K., and P.G. performed the experiments and analyzed the data. All the authors wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>P.G. is an employee of MaxCyte Inc. The other authors declare no competing interests.</p>
</section>
</article>