<article>
<section>
<h1 id="header">Isolation of high-quality total RNA and RNA sequencing of single bovine oocytes</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Fernando H. Biase<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Animal and Poultry Sciences, Virginia Polytechnic Institute and State University, Blacksburg, VA 24061, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:fbiase@vt.edu">fbiase@vt.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100895">10.1016/j.xpro.2021.100895</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Studying individual mammalian oocytes has been extremely valuable for the understanding of the molecular composition of oocytes including RNA storage. Here, a detailed protocol for isolation of oocytes, extraction of total RNA from single oocytes followed by full-length cDNA amplification, and library preparation is presented. The procedure permits the production of cost-effective and high-quality sequencing libraries. This protocol can be adapted for transcriptome analysis of oocytes from other species and be used to generate high-quality data from single embryos.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1186/s12864-018-4738-2">Biase and Kimble (2018)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of high-quality total RNA from single bovine oocytes</li>
<li>Detailed procedures for amplification of complementary DNA for library preparation</li>
<li>A bioinformatic pipeline for the quantitation of transcript abundance</li>
<li>The protocol also enables high-quality data production from single embryos</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for working with bovine oocytes, all the way from obtaining and preserving oocyte samples to the quantification of transcript abundances (see <a href="#fig1">Figure 1</a>A for overview of the procedures). However, we have also used this protocol for production of transcriptome data from single embryos. This protocol can be applied to other species such as human and mouse.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Single-cell RNA sequencing of oocytes</div>
<p>(A) Schematic diagram of the workflow following the deposit of the oocyte in the microcentrifuge tube.</p>
<p>(B) Representative images of the samples processed. Following the removal of the cumulus cells, single oocytes are deposited in 0.2 mL tubes with minimal volume (∼ 1 μL) of PBS containing RNAse inhibitor (0.2 IU/μL) and bovine serum albumin (0.2%). Images were obtained with a 10<b>×</b> objective. Scale bar for COCs is 400 μm, and for oocytes is 200 μm.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Confirm that all reagents are available before beginning the protocol.</div>
<div class="note">
<span class="note-title">Note:</span> Wipe all the surfaces with an RNase decontamination solution (i.e. RNAseZap or an alternative), which includes but is not limited to: bench, centrifuge, pipettors, pipette tip boxes.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use sterile and nuclease-free tubes and nuclease-free pipette tips containing a filter barrier. Always change pipette tips to avoid cross contamination, or contamination of reagents.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Water and all solutions and tubes should be free of nucleases.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Ampure XP</td>
<td>Beckman Coulter</td>
<td>A63881</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Millipore Sigma</td>
<td>126609-5GM</td>
</tr>
<tr>
<td>Chloroform</td>
<td>Fisher Scientific</td>
<td>AAJ67241AP</td>
</tr>
<tr>
<td>dNTPs</td>
<td>Thermo Fisher Scientific</td>
<td>10297018</td>
</tr>
<tr>
<td>Ethanol</td>
<td>VWR</td>
<td>BP2818500</td>
</tr>
<tr>
<td>Gibco TrypLE Express Enzyme</td>
<td>Fisher Scientific</td>
<td>12604-013</td>
</tr>
<tr>
<td>Glycoblue</td>
<td>Thermo Fisher Scientific</td>
<td>AM9516</td>
</tr>
<tr>
<td>Hyaluronidase</td>
<td>Millipore Sigma</td>
<td>H4272</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>VWR</td>
<td>32727-0010</td>
</tr>
<tr>
<td>Maxima H Minus RT</td>
<td>Thermo Fisher Scientific</td>
<td>EP0751</td>
</tr>
<tr>
<td>Mineral Oil</td>
<td>Sigma</td>
<td>M5310-500mL</td>
</tr>
<tr>
<td>PBS, 10X Solution</td>
<td>Thermo Fisher Scientific</td>
<td>193871</td>
</tr>
<tr>
<td>PEG 8000</td>
<td>VWR</td>
<td>101443-878</td>
</tr>
<tr>
<td>Phasemaker Tubes</td>
<td>Thermo Fisher</td>
<td>A33248</td>
</tr>
<tr>
<td>Phosphate buffer saline</td>
<td>Thermo Fisher Scientific</td>
<td>193871</td>
</tr>
<tr>
<td>RNAlater Stabilization Solution</td>
<td>Thermo Fisher Scientific</td>
<td>AM7021</td>
</tr>
<tr>
<td>RNAseZap</td>
<td>Thermo Fisher Scientific</td>
<td>AM9780</td>
</tr>
<tr>
<td>RNAse inhibitor</td>
<td>Promega</td>
<td>N2611</td>
</tr>
<tr>
<td>Terra PCR Direct Polymerase Mix</td>
<td>Takara</td>
<td>639271</td>
</tr>
<tr>
<td>TripLe Express</td>
<td>Gibco</td>
<td>12604-013</td>
</tr>
<tr>
<td>Tris EDTA buffer</td>
<td>VWR</td>
<td>BP24731</td>
</tr>
<tr>
<td>Trizol reagent</td>
<td>Thermo Fisher Scientific</td>
<td>15596026</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Agilent High Sensitivity DNA Kit</td>
<td>Agilent</td>
<td>5067-4626</td>
</tr>
<tr>
<td>Agilent RNA 6000 Pico Kit</td>
<td>Agilent</td>
<td>5067-1513</td>
</tr>
<tr>
<td>Nextera DNA Flex Library Preparation Kit</td>
<td>Illumina, Inc</td>
<td>20015826</td>
</tr>
<tr>
<td>Qubit 1X dsDNA HS Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Q33230</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>5′-AAGCAGTGGTATCAACGCAGAGT-3′</td>
<td>IDT</td>
<td>n/a</td>
</tr>
<tr>
<td>5′-AAGCAGTGGTATCAACGCAGAGTACATrGrGrG-3′</td>
<td>IDT</td>
<td>n/a</td>
</tr>
<tr>
<td>5′-AAGCAGTGGTATCAACGCAGAGTACT30VN-3′</td>
<td>IDT</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Hisat2</td>
<td>(<a href="#bib21">Kim et al., 2019</a>)</td>
<td><a href="http://daehwankimlab.github.io/hisat2/">http://daehwankimlab.github.io/hisat2/</a></td>
</tr>
<tr>
<td>Samtools</td>
<td>(<a href="#bib23">Li et al., 2009</a>; <a href="#bib25">Morgan and Pagés, 2012</a>)</td>
<td><a href="http://www.htslib.org/">http://www.htslib.org/</a></td>
</tr>
<tr>
<td>Biobambam</td>
<td>(<a href="#bib33">Tischler and Leonard, 2014</a>)</td>
<td><a href="https://github.com/gt1/biobambam">https://github.com/gt1/biobambam</a></td>
</tr>
<tr>
<td>Picard</td>
<td>n/a</td>
<td><a href="https://broadinstitute.github.io/picard/">https://broadinstitute.github.io/picard/</a></td>
</tr>
<tr>
<td>Featurecounts</td>
<td>(<a href="#bib24">Liao et al., 2014</a>)</td>
<td><a href="http://subread.sourceforge.net/">http://subread.sourceforge.net/</a></td>
</tr>
<tr>
<td>R software</td>
<td>(<a href="#bib18">Ihaka and Gentleman, 1996</a>)</td>
<td><a href="https://www.r-project.org/">https://www.r-project.org/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Bioanalyzer</td>
<td>Agilent</td>
<td>G2939B</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>Eppendorf</td>
<td>6331000025</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Eppendorf</td>
<td>022623508</td>
</tr>
<tr>
<td>Rotor</td>
<td>Eppendorf</td>
<td>FA-45-30-11</td>
</tr>
<tr>
<td>Qubit</td>
<td>Thermo Fisher Scientific</td>
<td>Q33238</td>
</tr>
<tr>
<td>Stripper pipette</td>
<td>CooperSurgical</td>
<td>MXL3-STR</td>
</tr>
<tr>
<td>Stripper tips</td>
<td>CooperSurgical</td>
<td>MXL3-175</td>
</tr>
<tr>
<td>BRAND® PCR Mini-cooler with transparent lid</td>
<td>Sigma</td>
<td>BR781260-2EA</td>
</tr>
<tr>
<td>DynaMag – 96 side magnet</td>
<td>Thermo Fisher Scientific</td>
<td>12331D</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>RNA elution mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Oligo-dT 5′-AAGCAGTGGTATCAACGCAGAGTACT30VN-3′ (100 μM)</td>
<td>10 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>RNase inhibitor (40 U/μL)</td>
<td>0.6 U/μL</td>
<td>0.15 μL</td>
</tr>
<tr>
<td>Water</td>
<td>n/a</td>
<td>3.75 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare sufficient mix for two extra reactions, one will serve as a non-template control. Keep it on ice until used and do not store for future use in this protocol.</p>
<table id="undtbl2">
<caption>Reverse transcription mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PEG 8000 (50%)</td>
<td>7.5 <b>×</b>
</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>Maxima RT Buffer (5<b>×</b>)</td>
<td>1 <b>×</b>
</td>
<td>2 μL</td>
</tr>
<tr>
<td>dNTPs (10mM)</td>
<td>1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Template switching oligonucleotide 5′-AAGCAGTGGTATCAACGCAGAGTACATrGrGrG-3′ (100 μM)</td>
<td>2 mM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>Maxima H Minus RT (200 U/μL)</td>
<td>5 U/μL</td>
<td>0.25 μL</td>
</tr>
<tr>
<td>RNAse inhibitor (40 U/μL)</td>
<td>0.5 U/μL</td>
<td>0.13 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare sufficient mix for two extra reactions, one will serve as a non-template control. Keep it on ice until used and do not store for future use in this protocol.</p>
<table id="undtbl3">
<caption>PCR reaction mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Terra direct Buffer</td>
<td>2<b>×</b>
</td>
<td>10 μL</td>
</tr>
<tr>
<td>IsPCR (oligonucleotide)5′-AAGCAGTGGTATCAACGCAGAGT-3′</td>
<td>10 μM</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>Terra polymerase</td>
<td>1.25 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>11.2 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare sufficient mix for two extra reactions, one will serve as a non-template control. Keep it on ice until used and do not store for future use in this protocol.</p>
<table id="undtbl4">
<caption>Other solutions</caption>
<thead>
<tr>
<th>Name</th>
<th>Reagents</th>
</tr>
</thead>
<tbody>
<tr>
<td>Denuding solution</td>
<td>1<b>×</b> Trypsin, RNase inhibitor (0.2U/μL)</td>
</tr>
<tr>
<td>Oocyte wash solution</td>
<td>1<b>×</b> PBS, RNase inhibitor (0.2U/μL), BSA (0.2%)</td>
</tr>
<tr>
<td>Isopropanol 50%</td>
<td>50% Isopropanol (v/v)</td>
</tr>
<tr>
<td>Ethanol 75%</td>
<td>75% Ethanol (v/v)</td>
</tr>
<tr>
<td>Ethanol 80%</td>
<td>80% Ethanol (v/v)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Ethanol 80% must be done on the same day of its use. It is also recommended that RNase inhibitor is added to these solutions on the day of their use. Do not store for future use in this protocol.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol uses a Bioanalyzer to assess amplification and library quality, which can be substituted by a TapeStation.</div>
<p>Other brands of thermocycler can also be used if it has heated lid and holds 0.2mL centrifuge tubes.</p>
<p>Alternative brands of centrifuge can also be used if they are refrigerated.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Oocyte collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2–3 h]</div>
<p>Cattle (Bos taurus) oocytes can be collected from multiple sources depending on the hypothesis or the biological questions addressed in a specific study. Potential sources of oocytes are in vivo aspiration of follicles using ultrasonography-guided ovum pick up (<a href="#bib5">Bo et al., 2019</a>) or ex vivo aspiration of follicles from ovaries obtained from an abattoir (<a href="#bib34">Tribulo et al., 2019</a>). All oocytes used in this protocol were obtained from ex vivo aspiration of follicles from ovaries obtained from an abattoir. Ovaries were obtained postmortem, and no animal was handled or euthanized for this study. Thus, this work was carried out in compliance with the Institutional Animal Care and Use Committee of Virginia Tech. We note that the user should follow the appropriate regulations when obtaining biological samples from vertebrates. Cumulus oocytes complexes (COCs) can be obtained from multiple sources.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol starts with the COCs (<a href="#fig1">Figure 1</a>B) after their isolation and selection for further work. In a cumulus-oocyte complex, the oocyte will be identified as a large cell, often &gt;100 μm in diameter, enclosed in a thick layer of glycoproteins, the zona pellucida (<a href="#bib15">Gupta, 2018</a>; <a href="#bib17">Hyttel et al., 1986</a>; <a href="#bib36">Wassarman and Litscher, 2018</a>). The cumulus cells are the small cells surrounding the zona pellucida (<a href="#fig1">Figure 1</a>B).</div>
<div class="note">
<span class="note-title">Note:</span> The procedures described for the handling of COCs and oocytes are carried out with the aid of a stereoscope.</div>
<ol>
<li>Stripping of cumulus cells from oocytes.
<ol type="a">
<li>Add three drops (50 μL) of Denuding solution on a 35mm dish and cover with oil.</li>
<li>Add three drops (50 μL) of Oocyte wash solution on a 35mm dish and cover with oil.</li>
</ol>
</li>
<li>Transfer the oocytes into one of the drops with Denuding solution and pipette (20 or 100 μL pipettor) to remove the cumulus cells.</li>
<li>Using a stripper pipette (175 μm tip) aspirate the oocytes to the next drop of Denuding solution. Repeat the removal of the cumulus cells with the (20 or 100 μL pipettor) or the stripper pipette.
<ol type="a">
<li>Repeat the washing once again using the third drop of denuding solution. Aspirate as minimal volume as possible.</li>
</ol>
</li>
<li>Using a stripper pipette (175 μm tip) aspirate the oocytes to the drop containing the Oocyte washing solution. Aspirate as minimal volume as possible.
<ol type="a">
<li>Transfer the oocytes to the next drop containing the Oocyte washing solution and repeat the procedure until no cumulus cells are visible under the stereoscope.</li>
</ol>
</li>
<li>Using a stripper pipette (175 μm tip) aspirate one oocyte with minimal volume of washing solution and transfer into a 0.2 mL microcentrifuge tube.</li>
<li>Immerse the tube immediately into liquid nitrogen (snap freezing). Preserve the material at -80°C until used for RNA extraction.
<ol type="a">
<li>Alternatively, if liquid nitrogen is not available, transfer the oocytes to tubes containing 5 μL of an RNA stabilization solution (<a href="#bib7">Camacho-Sanchez et al., 2013</a>). Preserve the material at -80°C until used for RNA extraction.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cells can be maintained at -80°C for long-term storage</div>
<h3 id="sec3.2">Total RNA extraction from single oocytes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2–3 <b>h]</b>
</div>
<p>This step involves the extraction of total RNA from single oocytes using Trizol reagent. The procedures described here have been adapted from the manufacturer’s protocol to minimize the loss of total RNA obtained from single oocytes and embryos. A schematic of the procedure with pictures of critical steps is depicted in <a href="#fig2">Figure 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic of the total RNA extraction from single oocytes</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Pre-spin the Phasemaker Tubes for 30 sec at 12,000 <b>×</b> <i>g</i>.</div>
<div class="note">
<span class="note-title">Note:</span> In our laboratory we work with up to 12 tubes containing single oocytes in a single batch.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is particularly important that the tubes containing the oocyte lysates do not thaw before the addition of the Trizol reagent to the tube.</div>
<ol start="7">
<li>Transfer the 0.2 mL microcentrifuge tubes containing the single oocytes to a precooled PCR rack.</li>
<li>Before the solution containing the cell lysate thaws, add 150 μL of a monophasic solution of Trizol (<a href="#bib9">Chomczynski and Sacchi, 1987</a>, <a href="#bib10">2006</a>; <a href="#bib31">Rio et al., 2010</a>) to the 0.2 mL microcentrifuge tube. Add Trizol reagent to all tubes you are working with, then proceed to the next step.</li>
<li>Remove the tube from the cold rack onto a rack at ∼24°C.</li>
<li>Assuming that a minimum volume (∼ 1 μL) was added to the tube with the oocyte, add 10 μL of water to the solution, and mix the solution gently by pipetting up and down (∼5–10 times). Add water to all tubes you are working with, then proceed to the next step.</li>
<li>Transfer the homogenate solution to into a 2 mL centrifuge tube containing a gel polymer capable of separating the aqueous phase from the organic phase (<a href="#bib26">Murphy and Hellwig, 1996</a>).</li>
<li>Let the solution stand at ∼24°C for 5 min.</li>
<li>Add 30 μL of chloroform to the solution and mix the solution vigorously by shaking the tube for 15 s.</li>
<li>Let the solution stand for 3 min at ∼24°C.</li>
<li>Centrifuge at 12,000 <b>×</b> <i>g</i> for 5 min at 4°C.</li>
<li>Add 20 μL of chloroform to the solution and mix the solution vigorously by shaking the tube for 15 s,</li>
<li>Centrifuge at 12,000 <b>×</b> <i>g</i> for 5 min at 4°C.
<ol type="a">
<li>In the meantime add 1μL (15 μg) of glycoblue to a new 0.2 mL microcentrifuge tube.</li>
</ol>
</li>
<li>Remove all aqueous solution from the phase maker tube and place it in the 0.2 mL microcentrifuge tube containing the glycoblue.
<ol type="a">
<li>Mix the aqueous solution with the glycoblue by pipetting very gently (5 times).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not to touch the gel with the pipette tip. If the gel is touched, dispense the aqueous solution onto the gel gently and use a new pipette tip.</div>
<div class="note">
<span class="note-title">Note:</span> This step can also be executed by transferring the aqueous solution to all tubes, followed by mixing of the solution with glycoblue, which can be done with a multi-channel pipette.</div>
<ol start="19">
<li>Add 100 μL of isopropanol to the solution and mix by gentle pipetting (5 times).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step can also be executed by adding isopropanol to all tubes, followed by mixing of the solution with a multi-channel pipette.</div>
<ol start="20">
<li>Let the tube stand at ∼24°C for 10 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Extended precipitation may be used to increase the yield of total RNA. In this case store samples for 12–18h at -20°C and resume the protocol to pellet the RNA.</div>
<ol start="21">
<li>Centrifuge at 15,000 <b>×</b> <i>g</i> for 10 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the centrifuge rotor accommodates only 2 mL tubes. Add the 0.2 mL tubes into a 0.5 mL tube, which can be placed in an adaptor or add the 0.5 mL tube into a 2mL tube. Cut the lids of the 0.5 and 2 mL tubes to facilitate the handling.</div>
<ol start="22">
<li>Remove the supernatant gently and discard. Do not disrupt the pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is easier to remove most of the liquid with a 200 μL pipette and remove the remainder with a 10 μL or 20 μL pipette, which allows a more precise removal of the liquid without touching the pellet.</div>
<div class="note">
<span class="note-title">Note:</span> If RNALater or equivalent high salt RNA stabilization solution was used there will a formation of a bubble-like blue precipitate (<a href="#bib7">Camacho-Sanchez et al., 2013</a>). Remove the isopropanol without disrupting this bubble. Add 150 μL of isopropanol 50% to the microcentrifuge tube, and repeat steps 21 and 22.</div>
<ol start="23">
<li>Add 150 μL of Ethanol 75% to the 0.2 mL tube.</li>
<li>Centrifuge at 15,000 <b>×</b> <i>g</i> for 2 min at 4°C.</li>
<li>Remove the supernatant gently and discard. Do not disrupt the pellet.</li>
<li>Add 150 μL of Ethanol 75% to the 0.2 mL tube.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The pelleted RNA can be stored long term in Ethanol 75% at -80°C without cause RNA degradation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Only continue the protocol if the reverse transcription and PCR amplification will be executed without interruption or storage of nucleic acids synthesized.</div>
<div class="note">
<span class="note-title">Note:</span> If the material was stored resume the protocol on step 27.</div>
<ol start="27">
<li>Centrifuge at 15,000 <b>×</b> <i>g</i> for 2 min at 4°C.</li>
<li>Remove the supernatant gently and discard. Do not disrupt the pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is easier to remove most of the liquid with a 200 μL pipette and remove the remainder with a 10 μL or 20 μL pipette, which allows a more precise removal of the liquid. Remove ethanol 75% as much as possible without touching the pellet.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Have the RNA elution mix ready before starting air drying the pellet.</div>
<ol start="29">
<li>Air-dry the pellet, which takes about one minute if all the liquid is removed. Proceed immediately to the desired assay using the RNA.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Proceed immediately to the reverse transcription using the RNA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid extensive air drying because it will make the elution of the pellet very difficult. The pipetting required for the dissolution of a dry pellet will increase the breaking of the RNA strands.</div>
<div class="note">
<span class="note-title">Note:</span> Since the RNA used for sequencing is limited in quantity, assessment of quality prior to reverse transcription is not possible. Alternatively, assess the RNA quality of some samples that are collected and processed exclusively for quality control. Add 1 μL of water directly to the RNA pellet and wait for the pellet to dissolve in approximately 5–10 min. on ice. Proceed with the automated capillary electrophoresis.</div>
<h3 id="sec3.3">Reverse transcription</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h]</div>
<p>This protocol for synthesis of complementary DNA was adapted from the molecular crowding single-cell RNA barcoding sequencing (mcSCRB-seq) (<a href="#bib1">Bagnoli et al., 2018</a>) and the Smart-Seq2 (<a href="#bib28">Picelli et al., 2013</a>, <a href="#bib29">2014</a>). The procedure described here rely on manual liquid handling and transfer, however the protocol can be adapted for robotic liquid handlers (<a href="#bib19">Jaeger et al., 2020</a>).</p>
<div class="critical">
<span class="critical-title">Critical:</span> Spin down all microtubes after thawing reagents (in the case of enzymes, spin down the tubes in a refrigerated centrifuge immediately before pipetting the necessary volume), before and after incubations.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use sterile and nuclease-free pipette tips containing a filter barrier. Always change pipette tips to avoid cross contamination.</div>
<div class="note">
<span class="note-title">Note:</span> Have all the programs set up on the thermocycler prior to starting the protocol.</div>
<ol start="30">
<li>Add 5 μL the RNA elution mix directly onto the pellet and allow the total RNA pellet to dissolve in this solution. Maintain the tubes on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step usually takes approximately 5 min. Do not speed up the elution by pipetting. Only proceed once the pellet disappears.</div>
<ol start="31">
<li>Using a thermocycler, incubate the solution at 72°C for 3 min.</li>
<li>Immediately after the conclusion of the previous step, immerse the tubes on ice. Keep the tubes on ice for 3–5 min.</li>
<li>While the tubes are on ice, prepare the Reverse transcription mix, keep tubes on ice.</li>
<li>Add 5 μL of Reverse transcription mix to the solution containing RNA, mix gently by pipetting (∼5 times).</li>
<li>Using a thermocycler, incubate the solution at 42°C for 90 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Due to the presence of PEG 8000, the Reverse transcription mix is viscous. Start preparation of the reverse transcription mix by adding the buffer to help the dispensing of the PEG 8000. Mix this solution very gently. On step 34 mix the solution very gently.</div>
<h3 id="sec3.4">Cleanup of the DNA:RNA hybrids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<div class="note">
<span class="note-title">Note:</span> Remove the magnetic beads from the refrigerator 30 minutes prior to using it. At the time of use, vortex the magnetic beads to mix the solution well. In our laboratory we prepare aliquots of the beads solution to prevent contamination of the bottle.</div>
<div class="note">
<span class="note-title">Note:</span> All procedures for DNA:RNA clean-up are carried out at ∼24°C. Magnetic beads are magnetic nano spheres coated with a biopolymer exhibiting high affinity to nucleic acids (see review by (<a href="#bib2">Berensmeier, 2006</a>))</div>
<ol start="36">
<li>Add 10 μL of magnetic beads solution to each tube. After the addition of magnetic beads to all tubes, mix the solution with gentle pipetting (∼ 10 times). Let the tubes stand for 5 min.</li>
<li>Transfer the tubes with magnetic beads to a magnetic rack. Let the tubes stand for 8 min.</li>
<li>Remove the solution gently without disrupting the beads and discard the solution. Use a 100 μL pipette set at &gt; 20 μL to remove the ∼ 20 μL of volume.</li>
<li>With the tubes still on the magnetic rack, gently add 200 μL of ethanol 80% to avoid disrupting the magnetic beads.
<ol type="a">
<li>Wait for 30 s</li>
</ol>
</li>
<li>Remove the solution carefully to avoid disrupting the magnetic beads</li>
<li>Gently add 200 μL of ethanol 80% to avoid disrupting the magnetic beads.
<ol type="a">
<li>Wait for 30 s</li>
</ol>
</li>
<li>Remove the solution carefully to avoid disrupting the magnetic beads</li>
<li>Remove the tubes from the rack and let the tubes standing on a regular rack or the beads to dry.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Usually, it takes ∼5 min for “cracks” to form on the beads, which indicates their dryness.</div>
<ol start="44">
<li>Add 9.5 μL of water to each tube, making sure that the water is deposited on the magnetic beads that are on the wall of the microcentrifuge tube.
<ol type="a">
<li>Wait for ∼2 min for the magnetic beads to begin absorbing water.</li>
</ol>
</li>
<li>Mix the solution gently with a 10 μL pipette (∼ 5 times).
<ol type="a">
<li>Let the tubes stand for 2 min.</li>
</ol>
</li>
<li>Transfer the tubes to the magnetic rack.
<ol type="a">
<li>Let the tubes stand for 5 min.</li>
</ol>
</li>
<li>Very gently, aspirate 9 μL of the solution containing the DNA:RNA hybrids into a new 0.2 mL tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid aspirating beads. If beads are seen in the pipette tip, dispense the liquid gently, wait for 2 min and repeat the process.</div>
<h3 id="sec3.5">Full-length transcript polymerase chain reaction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2 h]</div>
<ol start="48">
<li>Prepare a PCR reaction mix. Add 11 μL to each tube and mix by gentle pipetting (∼5 times).</li>
<li>Carry out the amplification using the following cycles.</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>15 s</td>
<td rowspan="3">8–11 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>66°C</td>
<td>15 s</td>
</tr>
<tr>
<td>Extension</td>
<td>68°C</td>
<td>4 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>4 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Ideally, the number of cycles should be determined by assaying the RNA from few oocytes at different cycle numbers to avoid over-amplification, which can be observed by the presence of peaks when assaying the amplified complementary DNA on the automated capillary electrophoresis.</div>
<ol start="50">
<li>Remove the tubes from the thermocycler and proceed with the clean-up.</li>
</ol>
<h3 id="sec3.6">Amplified DNA cleanup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<div class="note">
<span class="note-title">Note:</span> Remove the magnetic beads from the refrigerator 30 minutes prior to using it. At the time of use, vortex the magnetic beads to mix the solution well.</div>
<div class="note">
<span class="note-title">Note:</span> All procedures for DNA clean-up are carried out at ∼24°C</div>
<ol start="51">
<li>Add 20 μL of magnetic beads solution to each tube. After the addition of magnetic beads to all tubes, mix the solution with gentle pipetting (∼ 10 times). Let the tubes stand for 5 min.</li>
<li>Transfer the tubes with magnetic beads to a magnetic rack. Let the tubes stand for 8 min.</li>
<li>Remove the solution gently without disrupting the beads and discard the solution. Use a 100 μL pipette set at &gt; 40 μL to remove the ∼40 μL of volume.</li>
<li>With the tubes still on the magnetic rack, gently add 200 μL of ethanol 80% to avoid disrupting the magnetic beads.
<ol type="a">
<li>Wait for 30 s</li>
</ol>
</li>
<li>Remove the solution carefully to avoid disrupting the magnetic beads</li>
<li>Gently add 200 μL of ethanol 80% to avoid disrupting the magnetic beads.
<ol type="a">
<li>Wait for 30 s</li>
</ol>
</li>
<li>Remove the solution carefully to avoid disrupting the magnetic beads</li>
<li>Remove the tubes from the rack and let the tubes stand on a regular rack for the beads to dry.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Usually, it takes ∼5 min for “cracks” to form on the beads, which indicates their dryness.</div>
<ol start="59">
<li>Add 10.5 μL of Tris EDTA buffer to each tube, making sure that the buffer is deposited on the magnetic beads that are on the wall of the microcentrifuge tube.
<ol type="a">
<li>Wait for ∼2 min for the magnetic beads to begin absorbing water.</li>
</ol>
</li>
<li>Mix the solution gently with a 10 μL pipette (∼ 5 times).
<ol type="a">
<li>Let the tubes stand for 2 min.</li>
</ol>
</li>
<li>Transfer the tubes to the magnetic rack.
<ol type="a">
<li>Let the tubes stand for 5 min.</li>
</ol>
</li>
<li>Very gently, aspirate 10 μL of the solution containing the DNA:RNA hybrids into a new 0.2 mL tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid aspirating beads. If beads are seen in the pipette tip, dispense the liquid gently, wait for 2 min and repeat the process.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> DNA can be stored at -80°C for at least six months.</div>
<ol start="63">
<li>Assess the yield of the amplified complementary DNA using fluorometry instrument and assess the DNA profile using capillary automated electrophoresis.</li>
</ol>
<h3 id="sec3.7">Preparation of DNA library for high-throughput sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [1 h]</div>
<p>These steps will describe the production of DNA library for next generation sequencing based on bead-linked transposome technology (<a href="#bib6">Bruinsma et al., 2018</a>; <a href="#bib8">Caruccio, 2011</a>) to produce fragments of DNA within a narrow range around 450 base pairs long.</p>
<ol start="64">
<li>Assess the yield of the amplified complementary DNA using fluorometry instrument (e.g., Qubit) and assess the DNA profile using capillary automated electrophoresis (e.g., Bioanalyzer).</li>
<li>Prepare a diluted sample of amplified cDNA at 1 ng in 30 μL of water.</li>
<li>Follow the protocol described in the Nextera DNA flex Library preparation kit reference guide until the elution of the cleaned library.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Libraries are barcoded in one the specific steps of library construction. In this kit, during the PCR amplification. In our laboratory we use the dual barcode scheme.</div>
<div class="note">
<span class="note-title">Note:</span> Do not carry out the normalization steps for analysis of gene expression.</div>
<div class="note">
<span class="note-title">Note:</span> Carry out 12 cycles of PCR.</div>
<ol start="67">
<li>Assess the yield of the amplified complementary DNA using fluorometry instrument (e.g., Qubit) and assess the DNA profile using capillary automated electrophoresis (e.g., Bioanalyzer). The automated electrophoresis will also be important for the estimation of average fragment length of the library</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The data presented in this protocol was prepared with the Nextera DNA flex Library preparation kit, however alternatives to this protocol exist. Since the production of the representative data, Illumina Inc. has released another kit for library production based on the transposon technology. It is also possible to quantify the library DNA using quantitative PCR reagents and the appropriate analytical procedures (<a href="#bib16">Hawkins and Guest, 2018</a>).</div>
<h3 id="sec3.8">Pooling of libraries for high-throughput sequencing</h3>
<ol start="68">
<li>Calculate the molarity of each library in nanomolar using the standard equation: <math> <mrow> <mfrac> <mrow> <mo>(</mo> <mrow> <mi>c</mi> <mi>o</mi> <mi>n</mi> <mi>c</mi> <mi>e</mi> <mi>n</mi> <mi>t</mi> <mi>r</mi> <mi>a</mi> <mi>t</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mspace></mspace> <mrow> <mo>(</mo> <mi>n</mi> <mi>g</mi> <mo>/</mo> <mi>μ</mi> <mi>l</mi> <mo>)</mo> </mrow> <mo>)</mo> </mrow> </mrow> <mrow> <mo>(</mo> <mrow> <mn>660</mn> <mfrac> <mi>g</mi> <mrow> <mi>m</mi> <mi>o</mi> <mi>l</mi> </mrow> </mfrac> <mspace></mspace> <mo>×</mo> <mspace></mspace> <mi>a</mi> <mi>v</mi> <mi>e</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>e</mi> <mspace></mspace> <mi>l</mi> <mi>i</mi> <mi>b</mi> <mi>r</mi> <mi>a</mi> <mi>r</mi> <mi>y</mi> <mspace></mspace> <mi>f</mi> <mi>r</mi> <mi>a</mi> <mi>g</mi> <mi>m</mi> <mi>e</mi> <mi>n</mi> <mi>t</mi> <mspace></mspace> <mi>l</mi> <mi>e</mi> <mi>n</mi> <mi>g</mi> <mi>t</mi> <mi>h</mi> <mspace></mspace> <mrow> <mo>(</mo> <mi>b</mi> <mi>p</mi> <mo>)</mo> </mrow> <mo>)</mo> </mrow> </mrow> </mfrac> <mspace></mspace> <mo>×</mo> <msup> <mn>10</mn> <mn>6</mn> </msup> </mrow> </math>
</li>
<li>Combine samples maintaining equimolar quantities of material from each library.</li>
<li>Submit samples for sequencing according to the sequencing facility’s guidelines.</li>
</ol>
<h3 id="sec3.9">Basic bioinformatic procedures</h3>
<p>The processing of the RNA-sequencing data often follows a basic flow of alignment of the reads to the reference genome and counting of the reads according to gene annotation, but there are many variations of this workflow (<a href="#bib11">Conesa et al., 2016</a>; <a href="#bib3">Van den Berge et al., 2019</a>). To produce the representative results, sequences were aligned with Hisat2 (<a href="#bib20">Kim et al., 2015</a>, <a href="#bib21">2019</a>; <a href="#bib27">Pertea et al., 2016</a>), followed by filtering with samtools (<a href="#bib23">Li et al., 2009</a>) (remove unmapped reads, secondary alignments, failing of platform quality checks, PCR or optical duplicate) and removal of duplicates with the function ‘bammarkduplicates’ from biobambam (<a href="#bib33">Tischler and Leonard, 2014</a>). Sorting and indexing were done with Picard (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>). Finally, fragments were counted with ‘featurecounts’ (<a href="#bib24">Liao et al., 2014</a>) using the Ensembl (<a href="#bib14">Flicek et al., 2014</a>; <a href="#bib22">Kinsella et al., 2011</a>) bovine annotation as a guide.</p>
<p>The code used to produce representative results is described below.</p>
<ol start="71">
<li>Pipeline for processing raw reads.
<ol type="a">
<li>Align the reads to the reference genome.
<div class="textbox">
<p><code>#!/bin/bash</code></p>
<p><code>##Paths:</code></p>
<p><code>-</code></p>
<p><code>path_to_index= .../Bos_taurus.ARS-UCD1.2.99</code></p>
<p><code>path_to_gtf=.../Bos_taurus.ARS-UCD1.2.98.gtf</code></p>
<p><code>splice_sites=.../hisat_splice.txt</code></p>
<p><code>path_to_picard=.../bioinfo</code></p>
<p><code>merged_fastq=.../merged_fastq</code></p>
<p><code>sample=70o</code></p>
<p><code>output_alignment= .../alignment/$sample</code></p>
<p><code>mkdir $output_alignment</code></p>
<p><code>read1=$merged_fastq/sample_51_R1_001.fastq.gz</code></p>
<p><code>read2=$merged_fastq/sample_51_R2_001.fastq.gz</code></p>
<p><code>/home/fbiase/bioinfo/hisat2-2.2.0/hisat2 -p 30 -k 1 -x $path_to</code></p>
<p><code>index -1 $read1 -2 $read2 --known-splicesite-infile $splice_sit</code></p>
<p><code>es --no-mixed -S $output_alignment/$sample.alignment.sam --sum</code></p>
<p><code>mary-file $output_alignment/$sample.summary.txt</code></p>
<p><code>sample=72o</code></p>
<p><code>output_alignment= .../alignment/$sample</code></p>
<p><code>mkdir $output_alignment</code></p>
<p><code>read1=$merged_fastq/sample_55_R1_001.fastq.gz</code></p>
<p><code>read2=$merged_fastq/sample_55_R2_001.fastq.gz</code></p>
<p><code>/home/fbiase/bioinfo/hisat2-2.2.0/hisat2 -p 30 -k 1 -x $path_to</code></p>
<p><code>index -1 $read1 -2 $read2 --known-splicesit</code></p>
<p><code>e-infile $splice_sites --no-mixed -S $output_alignment/$sample.alignment.sam --sum</code></p>
<p><code>mary-file $output_alignment/$sample.summary.txt</code></p>
<p><code>sample=87o</code></p>
<p><code>output_alignment= .../alignment/$sample</code></p>
<p><code>mkdir $output_alignment</code></p>
<p><code>read1=$merged_fastq/sample_67_R1_001.fastq.gz</code></p>
<p><code>read2=$merged_fastq/sample_67_R2_001.fastq.gz</code></p>
<p><code>/home/fbiase/bioinfo/hisat2-2.2.0/hisat2 -p 30 -k 1 -x $path_to</code></p>
<p><code>index -1 $read1 -2 $read2 --known-splicesite-infile $splice_sit</code></p>
<p><code>es --no-mixed -S $output_alignment/$sample.alignment.sam --sum</code></p>
<p><code>mary-file $output_alignment/$sample.summary.txt</code></p>
<p><code>sample=89o</code></p>
<p><code>output_alignment= .../alignment/$sample</code></p>
<p><code>mkdir $output_alignment</code></p>
<p><code>read1=$merged_fastq/sample_71_R1_001.fastq.gz</code></p>
<p><code>read2=$merged_fastq/sample_71_R2_001.fastq.gz</code></p>
<p><code>/home/fbiase/bioinfo/hisat2-2.2.0/hisat2 -p 30 -k 1 -x $path_to</code></p>
<p><code>_index -1 $read1 -2 $read2 --known-splicesite-infile $splice_sit</code></p>
<p><code>es --no-mixed -S $output_alignment/$sample.alignment.sam --sum</code></p>
<p><code>mary-file $output_alignment/$sample.summary.txt</code></p>
<p><code>sample=91o</code></p>
<p><code>output_alignment= .../alignment/$sample</code></p>
<p><code>mkdir $output_alignment</code></p>
<p><code>read1=$merged_fastq/sample_75_R1_001.fastq.gz</code></p>
<p><code>read2=$merged_fastq/sample_75_R2_001.fastq.gz</code></p>
<p><code>/home/fbiase/bioinfo/hisat2-2.2.0/hisat2 -p 30 -k 1 -x $path_to</code></p>
<p><code>_index -1 $read1 -2 $read2 --known-splicesite-infile $splice_sit</code></p>
<p><code>es --no-mixed -S $output_alignment/$sample.alignment.sam --sum</code></p>
<p><code>mary-file $output_alignment/$sample.summary.txt</code></p>
</div>
</li>
<li>Filter low-quality alignments.
<div class="textbox">
<p><code>Folder= .../alignment</code></p>
<p><code>samtools view -b -h -F 1796 $folder/70o/70o.alignment.sam -o $folder</code></p>
<p><code>/70o/70o.alignment.filtered.bam</code></p>
<p><code>samtools view -b -h -F 1796 $folder/89o/89o.alignment.sam -o $folder</code></p>
<p><code>/89o/89o.alignment.filtered.bam</code></p>
<p><code>samtools view -b -h -F 1796 $folder/87o/87o.alignment.sam -o $folder</code></p>
<p><code>/87o/87o.alignment.filtered.bam</code></p>
<p><code>samtools view -b -h -F 1796 $folder/72o/72o.alignment.sam -o $folder</code></p>
<p><code>/72o/72o.alignment.filtered.bam</code></p>
<p><code>samtools view -b -h -F 1796 $folder/91o/91o.alignment.sam -o $folder</code></p>
<p><code>/91o/91o.alignment.filtered.bam</code></p>
</div>
</li>
<li>Sort alignments by coordinate.
<div class="textbox">
<p><code>java -jar $path_to_picard/picard.jar SortSam INPUT=$folder</code></p>
<p><code>/70o/70o.alignment.filtered.bam OUTPUT=$folder/70o/70o.align</code></p>
<p><code>ment.filtered.sorted.bam SORT_ORDER=coordinate</code></p>
<p><code>java -jar $path_to_picard/picard.jar SortSam INPUT=$folder/</code></p>
<p><code>89o/89o.alignment.filtered.bam OUTPUT=$folder/89o/89o.align</code></p>
<p><code>ment.filtered.sorted.bam SORT_ORDER=coordinate</code></p>
<p><code>java -jar $path_to_picard/picard.jar SortSam INPUT=$folder/</code></p>
<p><code>87o/87o.alignment.filtered.bam OUTPUT=$folder/87o/87o.align</code></p>
<p><code>ment.filtered.sorted.bam SORT_ORDER=coordinate</code></p>
<p><code>java -jar $path_to_picard/picard.jar SortSam INPUT=$folder</code></p>
<p><code>/72o/72o.alignment.filtered.bam OUTPUT=$folder/72o/72o.align</code></p>
<p><code>ment.filtered.sorted.bam SORT_ORDER=coordinate</code></p>
<p><code>java -jar $path_to_picard/picard.jar SortSam INPUT=$folder</code></p>
<p><code>/91o/91o.alignment.filtered.bam OUTPUT=$folder/91o/91o.align</code></p>
<p><code>ment.filtered.sorted.bam SORT_ORDER=coordinate</code></p>
</div>
</li>
<li>Index and create index file.
<div class="textbox">
<p><code>java -jar $path_to_picard/picard.jar BuildBamIndex INPUT=$folder</code></p>
<p><code>/70o/70o.alignment.filtered.sorted.bam</code></p>
<p><code>java -jar $path_to_picard/picard.jar BuildBamIndex INPUT=$folder</code></p>
<p><code>/89o/89o.alignment.filtered.sorted.bam</code></p>
<p><code>java -jar $path_to_picard/picard.jar BuildBamIndex INPUT=$folder</code></p>
<p><code>/87o/87o.alignment.filtered.sorted.bam</code></p>
<p><code>java -jar $path_to_picard/picard.jar BuildBamIndex INPUT=$folder</code></p>
<p><code>/72o/72o.alignment.filtered.sorted.bam</code></p>
<p><code>java -jar $path_to_picard/picard.jar BuildBamIndex INPUT=$folder</code></p>
<p><code>/91o/91o.alignment.filtered.sorted.bam</code></p>
</div>
</li>
<li>Remove duplicates.
<div class="textbox">
<p><code>bammarkduplicates I=$folder/70o/70o.alignment.filtered.sorted.bam</code></p>
<p><code>O=$folder/70o/70o.alignment.filtered.sorted.undup.bam level=9</code></p>
<p><code>markthreads=5 rmdup=1 index=1 dupindex=0 verbose=0</code></p>
<p><code>bammarkduplicates I=$folder/89o/89o.alignment.filtered.sorted.bam</code></p>
<p><code>O=$folder/89o/89o.alignment.filtered.sorted.undup.bam level=9</code></p>
<p><code>markthreads=5 rmdup=1 index=1 dupindex=0 verbose=0</code></p>
<p><code>bammarkduplicates I=$folder/87o/87o.alignment.filtered.sorted.bam</code></p>
<p><code>O=$folder/87o/87o.alignment.filtered.sorted.undup.bam level=9</code></p>
<p><code>markthreads=5 rmdup=1 index=1 dupindex=0 verbose=0</code></p>
<p><code>bammarkduplicates I=$folder/72o/72o.alignment.filtered.sorted.bam</code></p>
<p><code>O=$folder/72o/72o.alignment.filtered.sorted.undup.bam level=9</code></p>
<p><code>markthreads=5 rmdup=1 index=1 dupindex=0 verbose=0</code></p>
<p><code>bammarkduplicates I=$folder/91o/91o.alignment.filtered.sorted.bam</code></p>
<p><code>O=$folder/91o/91o.alignment.filtered.sorted.undup.bam level=9</code></p>
<p><code>markthreads=5 rmdup=1 index=1 dupindex=0 verbose=0</code></p>
</div>
</li>
<li>Count fragments relative to genome annotation.
<div class="textbox">
<p><code>path_to_gtf=.../Bos_taurus.ARS-UCD1.2.98.gtf</code></p>
<p><code>folder=.../alignment</code></p>
<p><code>folder1=.../counting</code></p>
<p><code>/subread-2.0.0-Linux-x86_64/bin/featureCounts -s 0 -a $pa</code></p>
<p><code>th_to_gtf -o $folder1/70o/70o.count -F 'GTF' -t 'exon' -g</code></p>
<p><code>'gene_id' --ignoreDup -p -T 5 $folder/70o/70o.alignment</code></p>
<p><code>.filtered.sorted.undup.bam</code></p>
<p><code>/subread-2.0.0-Linux-x86_64/bin/featureCounts -s 0 -a $pa</code></p>
<p><code>th_to_gtf -o $folder1/89o/89o.count -F 'GTF' -t 'exon' -g</code></p>
<p><code>'gene_id' --ignoreDup -p -T 5 $folder/89o/89o.alignment</code></p>
<p><code>.filtered.sorted.undup.bam</code></p>
<p><code>/subread-2.0.0-Linux-x86_64/bin/featureCounts -s 0 -a $pa</code></p>
<p><code>th_to_gtf -o $folder1/87o/87o.count -F 'GTF' -t 'exon' -g</code></p>
<p><code>'gene_id' --ignoreDup -p -T 5 $folder/87o/87o.alignment</code></p>
<p><code>.filtered.sorted.undup.bam</code></p>
<p><code>/subread-2.0.0-Linux-x86_64/bin/featureCounts -s 0 -a $pa</code></p>
<p><code>th_to_gtf -o $folder1/72o/72o.count -F 'GTF' -t 'exon' -g</code></p>
<p><code>'gene_id' --ignoreDup -p -T 5 $folder/72o/72o.alignment</code></p>
<p><code>.filtered.sorted.undup.bam</code></p>
<p><code>/subread-2.0.0-Linux-x86_64/bin/featureCounts -s 0 -a $pa</code></p>
<p><code>th_to_gtf -o $folder1/91o/91o.count -F 'GTF' -t 'exon' -g</code></p>
<p><code>'gene_id' --ignoreDup -p -T 5 $folder/91o/91o.alignment</code></p>
<p><code>.filtered.sorted.undup.bam</code></p>
</div>
</li>
</ol>
</li>
</ol>
<ol start="72">
<li>In R software, combine the counts from all files into one matrix.
<div class="textbox">
<p><code>files&lt;-list.files("/mnt/storage/lab_folder/oocyte_project/oocyte_</code></p>
<p><code>CC_BCB/counting_JOVE_paper", recursive=T, pattern=".count", full.</code></p>
<p><code>names = TRUE)</code></p>
<p><code>files&lt;-files[grep("summary", files, invert = TRUE)]</code></p>
<p><code>length(files)</code></p>
<p><code>count_data&lt;-data.frame(matrix(nrow=27607))</code></p>
<p><code>for (n in 1:length(files)) {</code></p>
<p><code>  count&lt;-read.delim(files[n], header=TRUE, sep= "\t", stringsAsFa</code></p>
<p><code>ctors = FALSE, comment.char= "#")</code></p>
<p><code>  count&lt;-count[,c(1,7)]</code></p>
<p><code>  count_data&lt;-cbind(count_data,count)</code></p>
<p><code>}</code></p>
<p><code>rownames(count_data)&lt;-count_data[,2]</code></p>
<p><code>count_data&lt;-count_data[,seq(from = 3, to = 11, by = 2)]</code></p>
<p><code>colnames(count_data)&lt;- paste("oocyte" , substr(colnames(count_dat</code></p>
<p><code>a), 80, 81), sep="")</code></p>
</div>
</li>
</ol>
<ol start="73">
<li>Subset the data to retain genes with &gt; 50 reads
<div class="textbox">
<p><code>count_data_a&lt;-count_data[rowSums(count_data)&gt;50,]</code></p>
<p><code>annotation.ensembl.symbol&lt;-read.delim("/resources/2020_05_29</code></p>
<p><code>_annotation.ensembl.symbol.txt.bz2", header=TRUE, sep= "\t",</code></p>
<p><code>row.names=1, stringsAsFactors = FALSE)</code></p>
<p><code>count_data_annotated&lt;-merge(count_data_a,annotation.ensembl.</code></p>
<p><code>symbol, by.x="row.names", by.y="ensembl_gene_id", all.x=TRUE</code>,</p>
<p><code>all.y=FALSE)</code></p>
<p><code>count_data_annotated&lt;-count_data_annota</code></p>
<p><code>ted[count_data_annotated$gene_biotype %in% c('protein_coding', 'lncRNA','pseudoge</code></p>
<p><code>ne'),]</code></p>
<p><code>count_data_annotated_length&lt;-count_data_annotated$transcript_length</code></p>
</div>
</li>
</ol>
<ol start="74">
<li>Load the necessary libraries
<div class="textbox">
<p><code>libPaths("/usr/lib/R/site-library")</code></p>
<p><code>library('dplyr', lib.loc="/usr/lib/R/site-library")</code></p>
<p><code>library('ggplot2', lib.loc="/usr/lib/R/site-library")</code></p>
<p><code>library('edgeR', lib.loc="/usr/lib/R/site-library")</code></p>
<p><code>library("cowplot", lib.loc="/usr/lib/R/site-library")</code></p>
<p><code>library("GGally",lib.loc="/usr/lib/R/site-library")</code></p>
<p><code>library("DESeq2",lib.loc="/usr/lib/R/site-library")</code></p>
</div>
<ol type="a">
<li>Calculate fragments per kilobase and counts per million
<div class="textbox">
<p><code>oocyte_fpkm&lt;-rpkm(count_data_annotated[,c(2:6)], count_data_annotated_length)</code></p>
<p><code>oocyte_cpm&lt;-edgeR::cpm(count_data_annotated[,c(2:6)])</code></p>
</div>
</li>
<li>Code to generate <a href="#fig4">Figure 4</a>.
<div class="textbox">
<p><code>table_N_genes_oocyte&lt;-data.frame()</code></p>
<p><code>      for (i in seq(0.1,1,0.1)){</code></p>
<p><code>        for (j in seq(1,5,1)){</code></p>
<p><code>          oocyte_fpkm_a&lt;-oocyte_fpkm[rowSums(oocyte_fp</code></p>
<p><code>km &gt;= i) &gt;= j,]</code></p>
<p><code>          dim(oocyte_fpkm_a)[1]</code></p>
<p><code>          table_N_genes_oocyte&lt;-rbind(table_N_genes_oo</code></p>
<p><code>cyte, data.frame("treshold"=i,"n_samples"=j, "n_genes"=di</code></p>
<p><code>m(oocyte_fpkm_a)[1]))</code></p>
<p><code>        }</code></p>
<p><code>      }</code></p>
<p><code>font_size&lt;-12</code></p>
<p><code>plot2&lt;-ggplot(table_N_genes_oocyte, aes(treshold, n_genes</code>,</p>
<p><code>color = n_samples)) +</code></p>
<p><code>geom_point() +</code></p>
<p><code>scale_x_continuous("FPKM",breaks = seq(0,1,0.1))+</code></p>
<p><code>scale_y_continuous("Number of genes \n equal or above FPK</code></p>
<p><code>M threshold", breaks = seq(11000,15000,500), limits=c(110</code></p>
<p><code>00,15000))+</code></p>
<p><code>scale_color_continuous(name = "Sample (N)",breaks = seq(1</code>,</p>
<p><code>5,1) ,high = "#132B43", low = "#56B1F7")+</code></p>
<p><code>guides(colour = guide_legend(reverse=T))+</code></p>
<p><code>theme_bw(base_size = font_size)+</code></p>
<p><code>theme(panel.grid.major = element_blank(),</code></p>
<p><code>    #panel.grid.minor = element_blank(),</code></p>
<p><code>    panel.background = element_blank(),</code></p>
<p><code>    axis.title=element_text(color="black"),</code></p>
<p><code>    axis.text=element_text(color="black"),</code></p>
<p><code>    legend.position = c(0.1, 0.25),</code></p>
<p><code>    legend.key.size = unit(.1, "cm"),</code></p>
<p><code>    legend.text = element_text(size=10),</code></p>
<p><code>    legend.title = element_text(size=10))</code></p>
<p><code>table_N_genes_oocyte&lt;-data.frame()</code></p>
<p><code>      for (i in seq(0.1,1,0.1)){</code></p>
<p><code>        for (j in seq(1,5,1)){</code></p>
<p><code>          oocyte_cpm_a&lt;-oocyte_cpm[rowSums(oocyte_cpm</code></p>
<p><code>&gt;= i) &gt;= j,]</code></p>
<p><code>          dim(oocyte_cpm_a)[1]</code></p>
<p><code>          table_N_genes_oocyte&lt;-rbind(table_N_genes_oo</code></p>
<p><code>cyte, data.frame("treshold"=i,"n_samples"=j, "n_genes"=di</code></p>
<p><code>m(oocyte_cpm_a)[1]))</code></p>
<p><code>        }</code></p>
<p><code>      }</code></p>
<p><code>plot3&lt;-ggplot(table_N_genes_oocyte, aes(treshold, n_genes</code>,</p>
<p><code>color = n_samples)) +</code></p>
<p><code>geom_point() +</code></p>
<p><code>scale_x_continuous("CPM",breaks = seq(0,1,0.1))+</code></p>
<p><code>scale_y_continuous("Number of genes \n equal or above CPM</code></p>
<p><code>threshold", breaks = seq(12000,15000,500), limits=c(1200</code></p>
<p><code>0,15000))+</code></p>
<p><code>scale_color_continuous(name = "Sample (N)",breaks = seq(1</code>,</p>
<p><code>5,1) ,high = "#132B43", low = "#56B1F7")+</code></p>
<p><code>guides(colour = guide_legend(reverse=T))+</code></p>
<p><code>theme_bw(base_size = font_size)+</code></p>
<p><code>theme(panel.grid.major = element_blank(),</code></p>
<p><code>    #panel.grid.minor = element_blank(),</code></p>
<p><code>    panel.background = element_blank(),</code></p>
<p><code>    axis.title=element_text(color="black"),</code></p>
<p><code>    axis.text=element_text(color="black"),</code></p>
<p><code>    legend.position = c(0.1, 0.25),</code></p>
<p><code>    legend.key.size = unit(.1, "cm"),</code></p>
<p><code>    legend.text = element_text(size=10),</code></p>
<p><code>    legend.title = element_text(size=10))</code></p>
<p><code>plot_grid(plot2,plot3, nrow=2)</code></p>
</div>
</li>
<li>Transform the counts using the rlog approach.
<div class="textbox">
<p><code>oocyte_count_b&lt;-count_data_annotated[,c(2:6)]</code></p>
<p><code>colnames(oocyte_count_b)&lt;-c("oocyte_A", "oocyte_B" ,"oocyte_C" ,"oocyte_D", "o</code></p>
<p><code>ocyte_E")</code></p>
<p><code>deseq2_oocyte&lt;-DESeqDataSetFromMatrix(oocyte_count_b, colData=DataFrame(c("ooc</code></p>
<p><code>yte_A", "oocyte_B" ,"oocyte_C" ,"oocyte_D", "oocyte_E")), design = ∼1 )</code></p>
<p><code>deseq2_oocyte_RLOG&lt;-rlog(deseq2_oocyte, blind=TRUE)</code></p>
</div>
</li>
<li>Code to generate <a href="#fig5">Figure 5</a>
<div class="textbox">
<p><code>ggpairs(as.data.frame(assay(deseq2_oocyte_RLOG)), xlab = "Regularized Log2 tra</code></p>
<p><code>nsformed counts",ylab = "Regularized Log2 transformed counts", diag=list(conti</code></p>
<p><code>nuous='blank'), lower = list(continuous = wrap("smooth", alpha = 0.3, size=0.</code></p>
<p><code>5)))</code></p>
</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The protocol described here was used to produce RNA sequencing data from single bovine oocytes. The wet lab protocol consists of four main procedures: 1- RNA extraction; 2- reverse transcription; 3 - amplification of complementary DNA; and 4- library preparation. Following sequencing assay, the process generally involves, 1-the alignment to the genome; 2- filtering of poor-quality alignment and 3- counting of the reads for quantification of gene transcript abundance (<a href="#fig1">Figure 1</a>A).</p>
<p>Using the protocol described for RNA extraction, the RNA can be indirectly assessed by the amplification of complementary DNA. On average, eight cycles of PCR produce 4.4 ng of amplified complementary DNA (0.44 ± 0.2 ng/μL, n = 26). The amplification is very homogeneous across cells. Fragment length often averages 2600 nucleotides long, ranging from 700 to ∼8000 nucleotides long (<a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative electrophoretic profiles of the DNA produced from two samples using this protocol</div>
<p>(A and B) (A) Amplified complementary DNA, and (B) Amplified DNA containing the insert and adapters for sequencing. The x-axis shows base pairs (bp), and the y-axis shows fluorescence units (FU).</p>
</figcaption>
</figure>
<p>The amplification of complementary DNA often yields material for multiple library preparations, if necessary. After library preparation, the amplified fragments often range from 250 - 1000 nucleotides long, averaging 457 nucleotides long. On average, ∼1 μg of library (37.9 ± 13.7 ng/μL, n = 26) is produced (<a href="#fig3">Figure 3</a>B).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The sequencing of five representative single oocytes collected at the GV stage yielded an average depth of 18,629,853 ± 1,713,461 non-duplicated pair of reads mapped to the bovine genome built ARS-UCD1.2 (<a href="#bib12">Elsik et al., 2009</a>) at one location and assigned to the bovine Ensembl gene annotation (<a href="#bib14">Flicek et al., 2014</a>; <a href="#bib22">Kinsella et al., 2011</a>). In total, 99.19% of the fragments mapped to protein-coding genes. With simple filtering of the genes that have at least 50 fragments across all five cells, it is possible to identify the detection of 14,204 protein-coding genes, 396 long non-coding RNA, and 82 pseudogenes, which will vary based on the different criterion of filtering and quantification metric (<a href="#fig4">Figure 4</a>). Following data normalization, the average correlation across the gene quantification was 0.99 between samples (p&lt;0.001, <a href="#fig5">Figure 5</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The number of genes detected based on fragments per kilobase per million (FPKM) or counts per million (CPM)</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Pair-wise sample correlation of five single-cell RNA sequencing produced from oocytes</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Sensitivity is one of the limitations of single-cell RNA-sequencing. The detection of up to 14,204 protein-coding genes presented here is similar to other results from single cattle oocytes obtained with different procedures (<a href="#bib4">Biase and Kimble, 2018</a>; <a href="#bib30">Reyes et al., 2015</a>). It is also very similar to the 14,647 genes detected from a compilation of bulk data from cattle oocytes (<a href="#bib35">Walker and Biase, 2020</a>). Even when applying a stringent threshold based on the number of samples and expression levels, it was possible to detect &gt; 11,000 genes. Thus, the protocol presented here detects a large majority of transcripts present in single oocytes, but not necessarily all transcripts.</p>
<p>Technical reproducibility is a serious concern of procedures involving single-cell RNA-sequencing (<a href="#bib32">Stegle et al., 2015</a>). While automation of this protocol is a possibility for somatic cells and spermatozoids, it is not an easy alternative for oocytes. First, obtaining cumulus-oocyte complexes followed by the removal of the cumulus from the oocytes are not trivial tasks that can be executed in hundreds of oocytes at a time. Oocytes must be manipulated carefully and placed in a microtube, which must be immediately snap-frozen. Practicing in several samples prior to working with the research samples should reduce the introduction of technical variability.</p>
<p>Non-polyadenylated (poly(A)-) transcripts are not quantified. Approximately ∼1% of the protein-coding genes in the mammalian genome produce poly(A)- transcripts, among which, genes in the histone family are classic examples (<a href="#bib37">Yang et al., 2011</a>; <a href="#bib38">Zhang et al., 2014</a>). Alternative approaches for producing sequencing libraries use random priming and oligo-dT for reverse transcription have been developed (<a href="#bib13">Fang and Akinci-Tolun, 2016</a>), but their efficacy for producing transcriptome from single-cells remain uncertain. This protocol is effective in producing transcriptome data from poly(A)+ transcripts from single oocytes.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Contamination of the reactions.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>It is common for primer dimer and excess of oligonucleotides to be present in non-template negative control. However, should high molecular weight DNA appear in the electrophoresis, all reagents should be replaced with unused reagents to prevent contamination of samples.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>No oocyte is deposited in the tube.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Researchers will gain experience with the pipetting of small volumes and use of the striper pipette. Diligent and careful pipetting will maximize the chances of depositing the oocyte at the bottom of the tube.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>No pellet is produced at the centrifugation with isopropanol.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If there is no pellet at the end of the protocol, it is possible that there was no oocyte deposited in the tube. It is also important that the ratio of sample/Trizol be maintained so that the sample volume does not exceed 10% of the volume of Trizol added to the tube. Refer to step 8.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Under amplification or over amplification of the amplified complementary DNA. Refer to step 49.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Although is it always important to understand that there will be variability of RNA abundance across oocytes, it is important to assess the appropriate number of PCR cycles used for amplification of complementary DNA that will yield a balance between sufficient template for library preparation and high quality amplified complementary DNA in test samples. Please, refer to <a href="#fig6">Figure 6</a> for examples of suitable amplification and over amplification.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1108-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Examples of profiles of amplified complimentary DNA (acDNA)</div>
<p>(A) Examples of acDNA that are not over amplified but did not produce sufficient template for the library preparation.</p>
<p>(B) Examples of acDNA that are not over amplified and produced sufficient template for the library preparation.</p>
<p>(C) Examples of acDNA that are over amplified. Notice the presence of peaks on this profile, which indicate overamplification of specific transcripts.</p>
</figcaption>
</figure>
<h3 id="sec7.9">Problem 5</h3>
<p>Library has fragments that are broad in range or are outside the expected range.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Set up the tagmentation reaction with tubes on a rack placed on ice. Add the neutralization buffer immediately after the tagmentation reaction is concluded. Refer to step 66.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bagnoli, J.W., Ziegenhain, C., Janjic, A., Wange, L.E., Vieth, B., Parekh, S., Geuder, J., Hellmann, I., and Enard, W. (2018). Sensitive and powerful single-cell RNA sequencing using mcSCRB-seq. Nat. Commun. <i>9</i>, 2937. <a class="external-link" href="https://doi.org/10.1038/s41467-018-05347-6">https://doi.org/10.1038/s41467-018-05347-6</a></p>
<p id="bib2">Berensmeier, S. (2006). Magnetic particles for the separation and purification of nucleic acids. Appl. Microbiol. Biotechnol. <i>73</i>, 495-504. <a class="external-link" href="https://doi.org/10.1007/s00253-006-0675-0">https://doi.org/10.1007/s00253-006-0675-0</a></p>
<p id="bib3">Van den Berge, K., Hembach, K.M., Soneson, C., Tiberi, S., Clement, L., Love, M.I., Patro, R., and Robinson, M.D. (2019). RNA sequencing data: Hitchhiker's guide to expression analysis. Annu. Rev. Biomed. Data Sci. <i>2</i>, 139-173. <a class="external-link" href="https://doi.org/10.1146/annurev-biodatasci-072018-021255">https://doi.org/10.1146/annurev-biodatasci-072018-021255</a></p>
<p id="bib4">Biase, F.H. and Kimble, K.M. (2018). Functional signaling and gene regulatory networks between the oocyte and the surrounding cumulus cells. BMC Genomics <i>19</i>, 351. <a class="external-link" href="https://doi.org/10.1186/s12864-018-4738-2">https://doi.org/10.1186/s12864-018-4738-2</a></p>
<p id="bib5">Bo, G.A., Cedeno, A., and Mapletoft, R.J. (2019). Strategies to increment in vivo and in vitro embryo production and transfer in cattle. Anim. Reprod. <i>16</i>, 411-422. <a class="external-link" href="https://doi.org/10.21451/1984-3143-AR2019-0042">https://doi.org/10.21451/1984-3143-AR2019-0042</a></p>
<p id="bib6">Bruinsma, S., Burgess, J., Schlingman, D., Czyz, A., Morrell, N., Ballenger, C., Meinholz, H., Brady, L., Khanna, A., Freeberg, L., et al. (2018). Bead-linked transposomes enable a normalization-free workflow for NGS library preparation. BMC Genomics <i>19</i>, ARTN 722. <a class="external-link" href="https://doi.org/10.1186/s12864-018-5096-9">https://doi.org/10.1186/s12864-018-5096-9</a></p>
<p id="bib7">Camacho-Sanchez, M., Burraco, P., Gomez-Mestre, I., and Leonard, J.A. (2013). Preservation of RNA and DNA from mammal samples under field conditions. Mol. Ecol. Resour. <i>13</i>, 663-673. <a class="external-link" href="https://doi.org/10.1111/1755-0998.12108">https://doi.org/10.1111/1755-0998.12108</a></p>
<p id="bib8">Caruccio, N. (2011). Preparation of next-generation sequencing libraries using Nextera technology: simultaneous DNA fragmentation and adaptor tagging by in vitro transposition. Methods Mol. Biol. <i>733</i>, 241-255. <a class="external-link" href="https://doi.org/10.1007/978-1-61779-089-8_17">https://doi.org/10.1007/978-1-61779-089-8_17</a></p>
<p id="bib9">Chomczynski, P. and Sacchi, N. (1987). Single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction. Anal. Biochem. <i>162</i>, 156-159. <a class="external-link" href="https://doi.org/10.1006/abio.1987.9999">https://doi.org/10.1006/abio.1987.9999</a></p>
<p id="bib10">Chomczynski, P. and Sacchi, N. (2006). The single-step method of RNA isolation by acid guanidinium thiocyanate-phenol-chloroform extraction: twenty-something years on. Nat. Protoc. <i>1</i>, 581-585. <a class="external-link" href="https://doi.org/10.1038/nprot.2006.83">https://doi.org/10.1038/nprot.2006.83</a></p>
<p id="bib11">Conesa, A., Madrigal, P., Tarazona, S., Gomez-Cabrero, D., Cervera, A., McPherson, A., Szcześniak, M.W., Gaffney, D.J., Elo, L.L., Zhang, X., and Mortazavi, A. (2016). A survey of best practices for RNA-seq data analysis. Genome Biol. <i>17</i>, 13. <a class="external-link" href="https://doi.org/10.1186/s13059-016-0881-8">https://doi.org/10.1186/s13059-016-0881-8</a></p>
<p id="bib12">Elsik, C.G., Tellam, R.L., Worley, K.C., Gibbs, R.A., Muzny, D.M., Weinstock, G.M., Adelson, D.L., Eichler, E.E., Elnitski, L., Guigó, R., et al. (2009). The genome sequence of taurine cattle: a window to ruminant biology and evolution. Science <i>324</i>, 522-528. <a class="external-link" href="https://doi.org/10.1126/science.1169588">https://doi.org/10.1126/science.1169588</a></p>
<p id="bib13">Fang, N. and Akinci-Tolun, R. (2016). Depletion of ribosomal RNA sequences from single-cell RNA-sequencing library. Curr. Protoc. Mol. Biol. <i>115</i>, 7 27 21-27 27 20. <a class="external-link" href="https://doi.org/10.1002/cpmb.11">https://doi.org/10.1002/cpmb.11</a></p>
<p id="bib14">Flicek, P., Amode, M.R., Barrell, D., Beal, K., Billis, K., Brent, S., Carvalho-Silva, D., Clapham, P., Coates, G., Fitzgerald, S., et al. (2014). Ensembl 2014. Nucleic Acids Res. <i>42</i>, D749-D755. <a class="external-link" href="https://doi.org/10.1093/nar/gkt1196">https://doi.org/10.1093/nar/gkt1196</a></p>
<p id="bib15">Gupta, S.K. (2018). The human egg's zona pellucida. Curr. Top. Dev. Biol. <i>130</i>, 379-411. <a class="external-link" href="https://doi.org/10.1016/bs.ctdb.2018.01.001">https://doi.org/10.1016/bs.ctdb.2018.01.001</a></p>
<p id="bib16">Hawkins, S.F.C. and Guest, P.C. (2018). Rapid and easy protocol for quantification of next-generation sequencing libraries. Methods Mol. Biol. <i>1735</i>, 343-350. <a class="external-link" href="https://doi.org/10.1007/978-1-4939-7614-0_23">https://doi.org/10.1007/978-1-4939-7614-0_23</a></p>
<p id="bib17">Hyttel, P., Xu, K.P., Smith, S., and Greve, T. (1986). Ultrastructure of in-vitro oocyte maturation in cattle. J. Reprod. Fertil. <i>78</i>, 615-625. <a class="external-link" href="https://doi.org/10.1530/jrf.0.0780615">https://doi.org/10.1530/jrf.0.0780615</a></p>
<p id="bib18">Ihaka, R. and Gentleman, R. (1996). R: a language for data analysis and graphics. J. Comput. Graph Stat. <i>5</i>, 299-314. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00601-8/sref18">View at publisher</a></p>
<p id="bib19">Jaeger, B.N., Yanguez, E., Gesuita, L., Denoth-Lippuner, A., Kruse, M., Karayannis, T., and Jessberger, S. (2020). Miniaturization of smart-seq2 for single-cell and single-nucleus RNA sequencing. STAR Protoc. <i>1</i>, 100081. <a class="external-link" href="https://doi.org/10.1016/j.xpro.2020.100081">https://doi.org/10.1016/j.xpro.2020.100081</a></p>
<p id="bib20">Kim, D., Langmead, B., and Salzberg, S.L. (2015). HISAT: a fast spliced aligner with low memory requirements. Nat. Methods <i>12</i>, 357-360. <a class="external-link" href="https://doi.org/10.1038/nmeth.3317">https://doi.org/10.1038/nmeth.3317</a></p>
<p id="bib21">Kim, D., Paggi, J.M., Park, C., Bennett, C., and Salzberg, S.L. (2019). Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat. Biotechnol. <i>37</i>, 907-915. <a class="external-link" href="https://doi.org/10.1038/s41587-019-0201-4">https://doi.org/10.1038/s41587-019-0201-4</a></p>
<p id="bib22">Kinsella, R.J., Kähäri, A., Haider, S., Zamora, J., Proctor, G., Spudich, G., Almeida-King, J., Staines, D., Derwent, P., Kerhornou, A., et al. (2011). Ensembl BioMarts: A hub for data retrieval across taxonomic space. Database <i>2011</i>, bar030. <a class="external-link" href="https://doi.org/10.1093/database/bar030">https://doi.org/10.1093/database/bar030</a></p>
<p id="bib23">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R., and Genome Project Data Processing, S. (2009). The sequence alignment/map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="https://doi.org/10.1093/bioinformatics/btp352">https://doi.org/10.1093/bioinformatics/btp352</a></p>
<p id="bib24">Liao, Y., Smyth, G.K., and Shi, W. (2014). featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics <i>30</i>, 923-930. <a class="external-link" href="https://doi.org/10.1093/bioinformatics/btt656">https://doi.org/10.1093/bioinformatics/btt656</a></p>
<p id="bib26">Murphy, N.R. and Hellwig, R.J. (1996). Improved nucleic acid organic extraction through use of a unique gel barrier material. Biotechniques <i>21</i>, 934. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00601-8/sref26">View at publisher</a></p>
<p id="bib27">Pertea, M., Kim, D., Pertea, G.M., Leek, J.T., and Salzberg, S.L. (2016). Transcript-level expression analysis of RNA-seq experiments with HISAT, StringTie and Ballgown. Nat. Protoc. <i>11</i>, 1650-1667. <a class="external-link" href="https://doi.org/10.1038/nprot.2016.095">https://doi.org/10.1038/nprot.2016.095</a></p>
<p id="bib28">Picelli, S., Björklund, Å.K., Faridani, O.R., Sagasser, S., Winberg, G., and Sandberg, R. (2013). Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Nat. Methods <i>10</i>, 1096-1098. <a class="external-link" href="https://doi.org/10.1038/nmeth.2639">https://doi.org/10.1038/nmeth.2639</a></p>
<p id="bib29">Picelli, S., Faridani, O.R., Björklund, A.K., Winberg, G., Sagasser, S., and Sandberg, R. (2014). Full-length RNA-seq from single cells using Smart-seq2. Nat. Protoc. <i>9</i>, 171-181. <a class="external-link" href="https://doi.org/10.1038/nprot.2014.006">https://doi.org/10.1038/nprot.2014.006</a></p>
<p id="bib30">Reyes, J.M., Chitwood, J.L., and Ross, P.J. (2015). RNA-Seq profiling of single bovine oocyte transcript abundance and its modulation by cytoplasmic polyadenylation. Mol. Reprod. Dev. <i>82</i>, 103-114. <a class="external-link" href="https://doi.org/10.1002/mrd.22445">https://doi.org/10.1002/mrd.22445</a></p>
<p id="bib31">Rio, D.C., Ares, M., Hannon, G.J., and Nilsen, T.W. (2010). Purification of RNA using TRIzol (TRI reagent). Cold Spring Harb. Protoc. <i>2010</i>. pdb prot5439. <a class="external-link" href="https://doi.org/10.1101/pdb.prot5439">https://doi.org/10.1101/pdb.prot5439</a></p>
<p id="bib32">Stegle, O., Teichmann, S.A., and Marioni, J.C. (2015). Computational and analytical challenges in single-cell transcriptomics. Nat. Rev. Genet. <i>16</i>, 133-145. <a class="external-link" href="https://doi.org/10.1038/nrg3833">https://doi.org/10.1038/nrg3833</a></p>
<p id="bib33">Tischler, G. and Leonard, S. (2014). biobambam: tools for read pair collation based algorithms on BAM files. Source Code Biol. Med. <i>9</i>, Artn 13. <a class="external-link" href="https://doi.org/10.1186/1751-0473-9-13">https://doi.org/10.1186/1751-0473-9-13</a></p>
<p id="bib34">Tribulo, P., Rivera, R.M., Ortega Obando, M.S., Jannaman, E.A., and Hansen, P.J. (2019). Production and Culture of the bovine embryo. Methods Mol. Biol. <i>2006</i>, 115-129. <a class="external-link" href="https://doi.org/10.1007/978-1-4939-9566-0_8">https://doi.org/10.1007/978-1-4939-9566-0_8</a></p>
<p id="bib35">Walker, B.N. and Biase, F.H. (2020). The blueprint of RNA storages relative to oocyte developmental competence in cattle (Bos taurus). Biol. Reprod. <i>102</i>, 784-794. <a class="external-link" href="https://doi.org/10.1093/biolre/ioaa015">https://doi.org/10.1093/biolre/ioaa015</a></p>
<p id="bib36">Wassarman, P.M. and Litscher, E.S. (2018). The mouse egg's zona pellucida. Curr. Top. Dev. Biol. <i>130</i>, 331-356. <a class="external-link" href="https://doi.org/10.1016/bs.ctdb.2018.01.003">https://doi.org/10.1016/bs.ctdb.2018.01.003</a></p>
<p id="bib37">Yang, L., Duff, M.O., Graveley, B.R., Carmichael, G.G., and Chen, L.L. (2011). Genomewide characterization of non-polyadenylated RNAs. Genome Biol. <i>12</i>, R16. <a class="external-link" href="https://doi.org/10.1186/gb-2011-12-2-r16">https://doi.org/10.1186/gb-2011-12-2-r16</a></p>
<p id="bib38">Zhang, X.O., Yin, Q.F., Chen, L.L., and Yang, L. (2014). Gene expression profiling of non-polyadenylated RNA-seq across species. Genomics Data <i>2</i>, 237-241. <a class="external-link" href="https://doi.org/10.1016/j.gdata.2014.07.005">https://doi.org/10.1016/j.gdata.2014.07.005</a></p>
<p id="bib25">Morgan, M., and Pagés, H. (2012). Rsamtools: Binary Alignment (BAM), Variant Call (BCF), or Tabix File Import. <a href="https://bioconductor.org/packages/Rsamtools">https://bioconductor.org/packages/Rsamtools</a>.</p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Fernando Biase, <a href="mailto:fbiase@vt.edu">fbiase@vt.edu</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The fastq files used as representative outcomes are available from the corresponding author on request. Count data have been deposited to Mendeley Data: <a href="http://dx.doi.org/10.17632/fwyk4tfrhg.1">http://dx.doi.org/10.17632/fwyk4tfrhg.1</a>. The code used to process raw files, obtain gene transcript abundances, and produce some of the charts presented is presented in steps 71, 72, 73 and 74.</p>
<h3>Acknowledgments</h3>
<p>The sequencing was conducted by the Vanderbilt Technologies for Advanced Genomics. Support for the publication fee was provided by the Virginia Tech Open Access Subvention Fund.</p>
<h3>Author contributions</h3>
<p>F.H.B. produced material submitted for sequencing, carried out analysis, and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The author declares no competing interests.</p>
</section>
</article>