<article>
<section>
<h1 id="header">Protocol for determining the regulation of lipid kinases and changes in phospholipids <i>in vitro</i>
</h1>
<p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
<p>Cansu Karabiyik<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and David C. Rubinsztein<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Medical Genetics, Cambridge Institute for Medical Research, University of Cambridge, Cambridge CB2 0XY, UK</p>
<p id="aff2"><sup>2</sup>UK Dementia Research Institute, Cambridge, UK</p>
<p id="aff3"><sup>3</sup>Mortimer B. Zuckerman Mind Brain and Behavior Institute, Columbia University, 3227 Broadway, Manhattan, NY 10027, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ck3086@columbia.edu">ck3086@columbia.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:dcr1000@cam.ac.uk">dcr1000@cam.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100926">10.1016/j.xpro.2021.100926</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The regulation of lipid kinases has remained elusive given the difficulties of assessing changes in lipid levels. Here, we describe the isolation of protein and lipid kinases to determine the regulation of lipid kinases <i>in vitro</i>. This can be followed by analysis of effects of regulators on lipid kinase-mediated changes in phospholipids without the use of radioactivity, with a specific focus on PI(5)P generation by the enzyme PIKfyve.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.devcel.2021.05.010">Karabiyik et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of proteins to determine the regulation of lipid kinases <i>in vitro</i>
</li>
<li>Assessing changes in lipid kinase activity by visualization of lipid products</li>
<li>Monitoring of lipid kinase activation by upstream effectors and the downstream effects</li>
<li>Non-radioactive assay to detect changes in phospholipid levels</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1143-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the specific steps that allow assessment of the regulation of the lipid kinase activity of PIKfyve. This protocol can be used to study the regulation of various lipid kinases. Phosphatidylinositol (PI) lipids are abundant components of eukaryotic cell membranes. As a response to upstream signals, lipid kinases phosphorylate PI on the inositol head group and generate an array of lipid second messengers. The phosphorylated lipids regulate the activity of a variety of proteins, including protein kinases, ion channels and adaptor proteins (<a href="#bib1">Di Paolo and De Camilli, 2006</a>). To study the enzymatic properties of lipid kinases, these are often purified and assayed using radiolabeled lipids (<a href="#bib6">Sbrissa et al., 2002</a>). Here, we present a method to study the regulation of lipid kinases in a non-radioactive manner. In our recent study (<a href="#bib4">Karabiyik et al., 2021</a>), we examined the regulation of the lipid kinase PIKfyve by an upstream mediator, the autophagy regulator ULK1.</p>
<p>PIKfyve generates PI(3,5)P<sub>2</sub> and PI(5)P lipids by phosphorylating the D-5 position in PI(3)P or PI (<a href="#bib9">Shisheva et al., 1999</a>), which play important roles in various cellular functions (<a href="#bib8">Shisheva et al., 2015</a>). We explored whether ULK1-mediated phosphorylation of PIKfyve would increase PI(5)P levels. First, we performed an <i>in vitro</i> kinase assay containing PIKfyve and ULK1 to allow phosphorylation of PIKfyve by ULK1 in the presence or absence of an ULK1 inhibitor or a PIKfyve inhibitor. Next, PI lipids were added to the reaction to allow phosphorylation of PIs by PIKfyve. The reaction was terminated by extracting phospholipids following an approach described by <a href="#bib7">Shisheva et al. (2001)</a>. The lipid extracts were spotted on a nitrocellulose membrane for a protein lipid overlay assay. To visualize changes in PI(5)P levels in the various reactions without the use of radioactive material, we exploited the fact that PI(5)P is bound by the plant homeodomain of the ING2 protein as a rationale for visualization of PI(5)P lipids (<a href="#bib3">Gozani et al., 2003</a>). The membrane spotted with lipids was incubated with recombinant GST-tagged ING2 protein following an approach previously described by <a href="#bib2">Dowler et al. (2002)</a>. PI(5)P present on the membrane should be bound by GST-ING2. By immunoblotting the membrane with an anti-GST antibody and secondary antibodies, changes in PI(5)P lipids in the various reactions were visualized, thus allowing us to infer how PIKfyve activity was regulated by ULK1. We have used this protocol to effectively study changes in PI(5)P levels by assessing GST-ING2 binding.</p>
<p>The same rationale can be applied to study the regulation of any lipid kinase whose lipid products are known, and which have suitable bio-probes. Before proceeding with this protocol, considerations should be made on downstream effectors, depending on the lipid kinase of interest (see below).</p>
<h3 id="sec1.1">Optimization of transfection</h3>
<p>To gain the highest yield of proteins, the use of HEK 293T cells is recommended. First, a control experiment should be performed to determine which day post transfection offers the highest protein yield. In this study, 4 wells of a 6 well plate were seeded with HEK 293T cells. The following day, all wells were transfected with HA-PIKfyve. We recommend transfecting one well with an empty backbone vector. Each day, the contents of one well were lysed with lysis buffer. After collection of all wells, the protein concentration in each lysate was determined with a BCA protein assay (as per manufacturer’s instructions). Lysates (20 ug protein/well) were resolved with SDS-PAGE. Our study found that the highest yield was achieved on day 2 post transfection of cells with a plasmid containing HA-PIKfyve (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1143-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Optimization of transfection</div>
<p>HEK 293T cells were transfected with HA-PIKfyve. Cells were lysed on day 1, 2, 3 or 4 post transfection. Immunoblots were blotted with anti-HA antibody. GAPDH was used as loading control.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Identification of lipid binding partners</h3>
<p>A protein lipid overlay assay will be performed to assess the effects of the regulation of a lipid kinase on its synthesis of given phospholipids. To this end, the product of the lipid kinase should be identified, and a binding protein selected for use in this protocol.</p>
<p>Many proteins contain a lipid binding domain, such as pleckstrin homology (PH) domain; the Fab1p, YOTB, Vac1p, EEA1 (FYVE) domain; the PhoX homology (PX) domain; the Epsin NH2-terminal homology (ENTH) domain; the glucosyltransferase, Rab-like GTPase activator and myotubularin (GRAM) domain; the cytosolic N-terminal polybasic domain of the mucolipin 1 channel (MLN1) (<a href="#bib1">Di Paolo and De Camilli, 2006</a>; <a href="#bib5">Li et al., 2013</a>). A list of phospholipids with their known interactors is shown in <a href="#tbl1">Table 1</a>.</p>
<table id="tbl1">
<caption>Table 1. Phosphoinositide-binding domains</caption>
<thead>
<tr>
<th>Phospholipid</th>
<th>Lipid kinase</th>
<th>Lipid-protein binding</th>
<th>Domain</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">PI(3)P</td>
<td rowspan="2">Class III PI(3)kinase hVPS34</td>
<td>EEA1, Hrs, SARA, PIKfyve</td>
<td>FYVE</td>
</tr>
<tr>
<td>SNX2, 3, 7, 13</td>
<td>PX</td>
</tr>
<tr>
<td rowspan="3">PI(3,5)P<sub>2</sub>
</td>
<td rowspan="3">PIKfyve (PIP kinase Type III)</td>
<td>Myotubularin</td>
<td>GRAM</td>
</tr>
<tr>
<td>Ent3p, Ent5p</td>
<td>A/ENTH</td>
</tr>
<tr>
<td>TRPML1</td>
<td>MLN1</td>
</tr>
<tr>
<td rowspan="2">PI(3,4)P<sub>2</sub>
</td>
<td rowspan="2">5-phosphatase SHIP2</td>
<td>AKT, TAPP1,2</td>
<td>PH</td>
</tr>
<tr>
<td>P47phox</td>
<td>PX</td>
</tr>
<tr>
<td rowspan="3">PI(3,4,5)P<sub>3</sub>
</td>
<td rowspan="3">Class I PI(3)kinase p110a</td>
<td>BTK, AKT, ARNO, GRP1</td>
<td>PH</td>
</tr>
<tr>
<td>SHC</td>
<td>PTB</td>
</tr>
<tr>
<td>CISK</td>
<td>PX</td>
</tr>
<tr>
<td rowspan="2">PI(4)P</td>
<td rowspan="2">PtdIns(4)kinase Type IIa,5-phosphatase OCRL, polyphosphoinositide phosphatase</td>
<td>EpsinR</td>
<td>A/ENTH</td>
</tr>
<tr>
<td>FAPP1/2, OSBP</td>
<td>PH</td>
</tr>
<tr>
<td rowspan="6">PI(4,5)P<sub>2</sub>
</td>
<td rowspan="6">PIP kinase Type IIa, 3-phosphatase PTEN</td>
<td>AP180, CALM, epsin, HIP</td>
<td>A/ENTH</td>
</tr>
<tr>
<td>Synaptotagmin</td>
<td>C2</td>
</tr>
<tr>
<td>Ezrin, moesin, radixin, talin</td>
<td>FERM</td>
</tr>
<tr>
<td>Syntenin</td>
<td>PDZ</td>
</tr>
<tr>
<td>PLC, dynamin</td>
<td>PH</td>
</tr>
<tr>
<td>Class II PI(3)K</td>
<td>PX</td>
</tr>
<tr>
<td rowspan="2">PI(5)P</td>
<td rowspan="2">PIKfyve, 3-phosphatase myotubularin-related protein (MTMR2)</td>
<td>ING2</td>
<td>PHD</td>
</tr>
<tr>
<td>SNX13</td>
<td>PX</td>
</tr>
</tbody>
</table>
<p class="table-legend">An overview of phospholipids, their known interactors, and the binding domains. PH, Pleckstrin homology; FYVE, Fab1p, YOTB, Vac1p, EEA1; PX, PhoX homology; ENTH, Epsin NH2-terminal homology; GRAM, glucosyltransferase, Rab-like GTPase activator and myotubularin; MLN1, cytosolic N-terminal polybasic domain of the mucolipin 1 channel.</p>
<p><a href="#tbl1">Table 1</a> should be used to select the appropriate lipid binding protein for visualization of changes in phospholipids. e.g., The FYVE domain specifically binds PI(3)P lipids and would be an appropriate domain to visualize changes in PI(3)P lipids. The PH or PX domains bind a variety of lipids and would therefore confound findings if used in this protocol.</p>
<p>In our study, we used binding to the ING2 protein to assess changes in PI(5)P levels. We tested whether GST-ING2 bound to PI, PI(3)P, PI(3,5)P<sub>2</sub> or PI(5)P lipids and found that it specifically bound PI(5)P (<a href="#fig2">Figure 2</a>). We suggest performing a similar test prior to employment of the lipid binding protein in this assay.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1143-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. ING2 binds PI(5)P</div>
<p>Exogenous PI, PI(3)P, PI(3,5)P<sub>2</sub> or PI(5)P (100 μM) were spotted on a membrane. The membrane was incubated with recombinant GST-ING2 (10 nM). Anti-GST antibody was used to visualize GST-ING2 binding to the lipids on the membrane. Where necessary, consider comparing signals from GST-tagged lipid probe to GST alone.</p>
</figcaption>
</figure>
<h3 id="sec1.3">Perform a preliminary <i>in vitro</i> kinase assay</h3>
<p>In our study, we assessed the ULK1-mediated effects on the lipid kinase activity of PIKfyve. Prior to this, we established that ULK1 phosphorylates PIKfyve by performing a malachite green-based <i>in vitro</i> kinase assay, determining the concentration and time required for ULK1 phosphorylation of PIKfyve. We suggest establishing these parameters for the protein kinase of interest prior to conducting the assay presented here.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>anti-GST antibody [3G10/1B3]</td>
<td>Abcam</td>
<td>#ab92RRID:AB_307067</td>
</tr>
<tr>
<td>anti-HA.11 clone 16B12</td>
<td>Covance</td>
<td>#2729SRRID:AB_1031062</td>
</tr>
<tr>
<td>Goat anti-Rabbit IgG (H+L) Secondary Antibody, DyLight 680</td>
<td>ThermoFisher</td>
<td>#35568</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>SBI-0206965</td>
<td>Sigma Aldrich</td>
<td>#SML1540</td>
</tr>
<tr>
<td>YM206136</td>
<td>Merck</td>
<td>#524611</td>
</tr>
<tr>
<td>PI(5)PdiC16</td>
<td>Echelon</td>
<td># P-5016</td>
</tr>
<tr>
<td>PIdiC16, PI(3)PdiC16, PI(3,5)P2diC16</td>
<td>Echelon</td>
<td>#P-0016#P-3016#P-3516</td>
</tr>
<tr>
<td>Recombinant Human ULK1</td>
<td>ThermoFisher</td>
<td>#PV6430</td>
</tr>
<tr>
<td>Recombinant Human-ING2</td>
<td>Creative Biomart</td>
<td>#5130H</td>
</tr>
<tr>
<td>ATP 100mM solution pH 7.5</td>
<td>Sigma-Aldrich</td>
<td>#GE27-2056-01</td>
</tr>
<tr>
<td>cOmplete Protease Inhibitor Cocktail</td>
<td>Sigma-Aldrich</td>
<td>#11697498001</td>
</tr>
<tr>
<td>Phosphatase Inhibitor Cocktail 2</td>
<td>Sigma-Aldrich</td>
<td>#P5726</td>
</tr>
<tr>
<td>Phosphatase Inhibitor Cocktail 3</td>
<td>Sigma-Aldrich</td>
<td>#P0044</td>
</tr>
<tr>
<td>InstantBlue Coomassie Protein Stain</td>
<td>Abcam</td>
<td>#ab119211</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Universal In Vitro Assay Kit</td>
<td>R&amp;D Systems</td>
<td>#EA004</td>
</tr>
<tr>
<td>Pierce HA magnetic beads</td>
<td>ThermoFisher</td>
<td>#88836</td>
</tr>
<tr>
<td>Pierce BCA Protein Assay Kit</td>
<td>ThermoFisher</td>
<td>#23225</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HEK 293T</td>
<td>ATCC</td>
<td>#CRL-3216; CVCL_0063</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pCMV5-HA-PIKfyve WT</td>
<td>A. Shisheva</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Image Studio Lite</td>
<td>LI-COR</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Hybond C-extra nitrocellulose membrane</td>
<td>GE Healthcare</td>
<td>#RPN1520N</td>
</tr>
<tr>
<td>Glass vials</td>
<td>Analytical-sales</td>
<td>#88120-CASE</td>
</tr>
<tr>
<td>Nonstick, RNase-free Microfuge Tubes, 1.5 mL</td>
<td>ThermoFischer</td>
<td>#AM12450</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NP-40 (10%)</td>
<td>0.5%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Tris pH 7.5 (1 M)</td>
<td>50 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.5 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Protease inhibitors 100×</td>
<td>1×</td>
<td>200 μL</td>
</tr>
<tr>
<td>Phosphatase inhibitors 100×</td>
<td>1×</td>
<td>200 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>16.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Keep buffer on ice. Buffer can be prepared and kept in −20°C. Phosphatase and protease inhibitors should be added immediately before use.</div>
<div class="note">
<span class="note-title">Note:</span> Additives such as 1mM DTT may be added to lysis buffer for optimal kinase stability.</div>
<table id="undtbl2">
<caption>Wash buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 7.5 (1 M)</td>
<td>50 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.5 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>18.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Buffer can be stored in 4°C for up to a month.</div>
<table id="undtbl3">
<caption>RIPA buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton-X (10%)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Tris pH 8.0 (1 M)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.5 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>sodium deoxycholate (10%)</td>
<td>1%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>SDS (sodium dodecyl sulfate) (20%)</td>
<td>0.1%</td>
<td>250 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>38.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Sodium deoxycholate stock must be protected from light. RIPA buffer stock can be stored in aliquots at −20°C.</div>
<table id="undtbl4">
<caption>Buffer A</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES pH 7.4 (1 M)</td>
<td>50 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>46 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Buffer can be stored in 4°C for up to 2 months.</div>
<table id="undtbl5">
<caption>Buffer B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 7.5 (1 M)</td>
<td>100 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>LiCl (5 M)</td>
<td>500 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">LiCl ensures effective removal of non-specific chromatin interactions with the agarose beads.</p>
<div class="note">
<span class="note-title">Note:</span> Buffer can be stored in 4°C for up to 2 months.</div>
<table id="undtbl6">
<caption>Buffer C</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 7.5 (1 M)</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>100 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>48.5</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Buffer can be stored in 4°C for up to 2 months.</div>
<table id="undtbl7">
<caption>Assay buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES pH 7.4 (1 M)</td>
<td>25 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>120 mM</td>
<td>1.2 mL</td>
</tr>
<tr>
<td>MnCl<sub>2</sub> (0.5 M)</td>
<td>2.5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (0.5 M)</td>
<td>2.5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>β-glycerophosphate (1 M)</td>
<td>5 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>DTT (100 M)</td>
<td>1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>47.4</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">β-glycerophosphate is added to inhibit potential immunoprecipitated phosphatases that may interfere with the assay.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare fresh before use.</div>
<table id="undtbl8">
<caption>Lipid buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES pH 7.4 (1 M)</td>
<td>20 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>1 mM</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>49 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Buffer can be stored in 4°C for up to 2 months.</div>
<table id="undtbl9">
<caption>TBS-T buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS (10×)</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>900 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<table id="undtbl10">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS-T 0.1%</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>2 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare fresh prior to use.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Precipitation of protein from mammalian cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days (depends on optimal transfection day)</div>
<ol>
<li>Purify proteins (kinase and/or substrate) using HEK 293T cells. This may be omitted if commercial recombinant proteins are available.
<ol type="a">
<li>Seed HEK 293T cells in 140 mm cell culture dishes.</li>
<li>Transfect cells with plasmid containing gene of interest with tag:
<ol type="i">
<li>One/two days later, wash cells with PBS three times and lyse in ice-cold lysis buffer.</li>
<li>Incubate lysates on ice for 30 min.</li>
<li>Centrifugate at 16,100g for 10 min at 4°C.</li>
<li>In the meantime, wash magnetic beads three times with wash buffer using a magnet.</li>
<li>Incubate post-nuclear supernatant with magnetic beads at 4°C on a rotating wheel for 1–2 h (time depends on the manufacturer’s instructions).</li>
<li>Centrifugate briefly and remove supernatant. Wash the beads three times with wash buffer. Mix by inverting tubes.
<div class="note">
<span class="note-title">Note:</span> If working with large quantities, proteins may be eluted using a peptide that competes with the bead-protein binding.</div>
Protein purity should be assessed using Coomassie or Silver stained gels, which can also be used to estimate protein concentrations by adding one or more lanes with defined amounts of a protein of known concentration as described below in c.</li>
</ol>
</li>
<li>Recommended. Determine protein concentration of the bead-bound protein:
<ol type="i">
<li>Prepare a stock solution of BSA (10 mg/mL).</li>
<li>Prepare 7 dilutions of BSA.</li>
<li>Add 2× Laemmli buffer with β-mercaptoethanol to 10 μL of each BSA solution and 10 μL of bead-bound protein. After boiling samples at 100°C for 5 min, run samples on SDS–PAGE at 100 V for 2 h.</li>
<li>Incubate the gel with Coomassie Blue stain to determine the relative protein amount bound by the beads and ensure that additional bands are not present.</li>
<li>Wash bead-bound protein with buffers as described below.
<div class="critical">
<span class="critical-title">Critical:</span> If large amounts of protein have been precipitated, aliquot and store at −80°C to avoid freeze and thaw. We suggest using magnetic beads. Agarose resin beads may be used if protein is eluted from the beads. However, we do not suggest using agarose resin-tagged protein for further steps, as the agarose resin cannot be washed off the membrane used in future steps.</div>
<div class="note">
<span class="note-title">Note:</span> While in this study, we used a precipitated substrate (HA-PIKfyve) and a recombinant commercial kinase (ULK1) without experiencing technical issues, this step may be skipped if both proteins are available as commercial products. Alternatively, both proteins may be purified using HEK 293T cells.</div>
<div class="note">
<span class="note-title">Note:</span> Since the relative ratio of substrate to kinase may affect the outcome of the kinase activity, we recommend determining the concentration of precipitated substrate and performing a kinase assay with varying concentrations of substrate to determine the optimal conditions.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">
<i>In vitro</i> lipid kinase assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>In this step an <i>in vitro</i> kinase assay was performed assessing whether the ULK1-mediated phosphorylation of PIKfyve leads to an increase in the lipid kinase activity of PIKfyve. Prior to this, we performed a malachite green-based <i>in vitro</i> kinase assay and found that ULK1 phosphorylates PIKfyve. Here, we assess whether ULK1 phosphorylation of PIKfyve leads to an increase in the lipid kinase activity of PIKfyve.</p>
<ol start="2">
<li>Prepare phosphoinositides for the kinase assay:
<ol type="a">
<li>Prepare a 1:1 solution of methanol and chloroform.</li>
<li>Add to glass vial containing PI or PI(5)P lipids for a 1 mM stock. Glass vials will minimize sticking that can occur to plastic tubes.</li>
<li>Add PI solution to empty tubes (n tubes depends on the number of reactions) for a final concentration of 100 μM.</li>
<li>Dry off the chloroform using an argon gas flux until all liquid has evaporated and a film is visible at the bottom of the tube.</li>
<li>Reconstitute lipid in lipid buffer.</li>
<li>Sonicate twice for 30 s on ice.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Prepare kinases:
<ol type="a">
<li>Wash the precipitated proteins three times with RIPA buffer, twice with buffer A, three times with buffer B, twice with buffer C and twice with assay buffer.</li>
<li>Incubate lipid kinase, protein kinase (10 nM) and ATP (50 μM) in assay buffer (total volume 50 μL) for 15 min at 37°C in a non-adherent tube.</li>
<li>Next, add the reaction mix to tube containing PIs (100 μM). Incubate for additional 15 min at 37°C.
<div class="note">
<span class="note-title">Note:</span> The time points are subject to change depending on the kinase.</div>
</li>
<li>Set up the reactions in <a href="#tbl2">Table 2</a>:</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Stop the reaction by adding 200 μL 1N HCl. The lipids should be used directly in the next step as phospholipids may be unstable when frozen. If required, we recommend storing lipids in an organic solvent such as chloroform in a tightly sealed glass vial and storing at −20°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The purified kinases should be as pure as possible. The wash steps will ensure dissociation of interactors. To limit confounding variables, SDS-PAGE may be performed for the precipitated kinases, assessing whether additional bands are visible after a Coomassie Blue stain as mentioned earlier.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If inhibitors of the protein kinase and/or the lipid kinase are available, these should be included in the assay as additional controls. Inhibitors and kinases should be incubated prior to initiation of the kinase assay.</div>
<div class="note">
<span class="note-title">Note:</span> If the phosphorylation sites are known, we suggest using phospho-mutant and phospho-mimic forms of the lipid kinase to further validate the effect of the protein kinase on the lipid kinase.</div>
<div class="note">
<span class="note-title">Note:</span> The SDS and sodium deoxycholate in the RIPA buffer may cause partial denaturation of some proteins. While this was not observed in our study, compromised activity of certain enzymes caused by these detergents in RIPA buffer may necessitate the use of non-ionic detergents instead. The use of negative and positive controls would help ensure that a lack in activity is not caused by buffer compositions.</div>
<div class="note">
<span class="note-title">Note:</span> EDTA may interfere with the activity of some kinases. If kinase activity is too low, adjust buffers accordingly.</div>
<div class="note">
<span class="note-title">Note:</span> Some kinases may depend on accessory proteins for activation of their kinase function. Orthogonal tools should be employed to assess the requirement for additional proteins.</div>
<table id="tbl2">
<caption>Table 2. Recommended reactions</caption>
<thead>
<tr>
<th> </th>
<th>Lipid kinase</th>
<th>Protein kinase</th>
<th>PI</th>
<th>ATP</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tube 1</td>
<td>+</td>
<td>-</td>
<td>+</td>
<td>+</td>
</tr>
<tr>
<td>Tube 2</td>
<td>+</td>
<td>+</td>
<td>+</td>
<td>+</td>
</tr>
<tr>
<td>Tube 3</td>
<td>-</td>
<td>+</td>
<td>+</td>
<td>+</td>
</tr>
<tr>
<td>Tube 4</td>
<td>+</td>
<td>+</td>
<td>-</td>
<td>+</td>
</tr>
</tbody>
</table>
<p class="table-legend">An overview of reactions that contain lipid kinases, protein kinases, phosphoinositides (PI) and ATP with appropriate controls.</p>
<h3 id="sec3.3">Lipid extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h, 20 min</div>
<p>If the lipid kinase has become activated in the <i>in vitro</i> kinase assay, this should lead to a generation of phospholipids.</p>
<ol start="5">
<li>Extract lipids from the kinase reaction:
<ol type="a">
<li>Add 160 μL 1:1 chloroform:methanol to the tube containing the reaction to create a biphasic system.</li>
<li>Vortex and centrifuge tube at 10,000g for 2 min.</li>
<li>Remove the upper methanol layer containing the kinases.</li>
<li>Wash the lower chloroform layer twice with 100 μL of 1:1 of methanol:1N HCl.</li>
<li>Use an argon gas flux to remove excess volume.</li>
<li>Apply the resulting lipid solution left in the tube to a nitrocellulose membrane.</li>
</ol>
</li>
<li>According to the lipid of interest, prepare the following steps:
<ol type="a">
<li>Dilute lipids in a 2:1:0.8 solution of methanol:chloroform:water in 5–8 serial dilutions starting with 100 μM.</li>
<li>Spot 5 μL of each dilution on the same nitrocellulose membrane as the lipids extracted from the kinase reactions.</li>
</ol>
</li>
<li>Air-dry the membrane at room temperature for 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To prevent the membrane from curling upon spotting with lipids, tape it firmly to a flat surface. Once dry, use a clean razor blade to cut within the edges of the tape to use it for the next steps.</div>
<h3 id="sec3.4">Protein lipid overlay assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The objective of this part of the protocol is to assess changes in lipid synthesis as a result of the lipid kinase activation by the protein kinase of interest. This step requires knowing whether the lipid of interest is bound by any given protein (see above). In our study, we used GST-tagged ING2, as this protein is known to bind PI(5)P.</p>
<ol start="8">
<li>Incubate the membrane containing lipid spots in blocking buffer for 1 h at room temperature.</li>
<li>Incubate the membrane overnight at 4°C with 10 nM GST-ING2 protein in blocking buffer in 4°C.
<ol type="a">
<li>The next day, wash the membrane 10 × 5 min with TBS-T.</li>
</ol>
</li>
<li>Incubate the membrane with a 1:2000 dilution of anti-GST monoclonal antibody in blocking buffer for 1 h at room temperature.
<ol type="a">
<li>Wash the membrane 10 × 5 min with TBS-T.</li>
</ol>
</li>
<li>Incubate the membrane with secondary antibody for 1 h to visualize the GST-tagged protein bound to your lipid of interest. We used of DyLight Fluors-conjugated secondary antibody in TBS-T at 1:5000 for 1 h.
<ol type="a">
<li>Wash the membrane 10 × 5 min with TBS-T and 1 × 5min with TBS.</li>
</ol>
</li>
<li>Visualize with direct infrared fluorescence detection using LICOR-Odyssey apparatus/Bio-Rad ChemiDoc.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the products of the lipid kinase in question have yet to be characterized, this tool allows detection of generated lipids by performing serial dilutions of other phospholipids and repeating the above assay with tools that enable their identification and quantification - e.g., <a href="#tbl1">Table 1</a>.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>If the protein kinase of interest leads to an activation of the lipid kinase activity, an increase in the generation of the relevant lipid should be observed. In our study, we found that the ULK1-mediated phosphorylation of PIKfyve led to an increased generation of PI(5)P lipids bound by GST-ING2. This was prevented with ULK1 inhibition (<a href="#fig3">Figure 3</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1143-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. ULK1 causes a PIKfyve-mediated increase in PI(5)P lipids</div>
<p>(A) Five serial dilutions of PI(5)P (100-0 μM) were spotted on a nitrocellulose membrane and incubated with GST-ING2 recombinant protein. PI(5)P-bound GST-ING2 was visualized by blotting membranes with anti-GST antibody.</p>
<p>(B) HA-PIKfyve precipitated from HEK 293T using magnetic HA beads were incubated with PI (100 μM) and ATP (50 μM) ± recombinant ULK1 (10 nM; 15 min, 37°C). Lipids were extracted and spotted on a nitrocellulose membrane, which was treated as in (A). PI or HA-PIKfyve were not added to controls reactions as indicated.</p>
<p>(C) HA-PIKfyve precipitated from HEK 293T using magnetic HA beads were incubated with PI (100 μM) and ATP (50 μM) ± recombinant ULK1 (10 nM; 15 min, 37°C) or following pre-treatment of ULK1 recombinant protein with the ULK1 inhibitor SBI-0206965 (SBI; 5 μM, 30 min). Lipids were extracted and spotted on a nitrocellulose membrane, which was treated as in (A). DMSO was used as vehicle control. Data show the relative levels of GST-ING2-bound PI(5)P represented as mean ± SEM (n = 3, ∗P &lt; 0.05; two-tailed, paired student’s t-test).</p>
</figcaption>
</figure>
<p>While we did not assess the concentration of lipids synthesized by PIKfyve, this assay offers the option of quantitative measurement. This requires first determining the concentration of the precipitated lipid kinase. A concentration curve can be created by a densitometric quantification of the PI(5)P standards. Subsequently, using varying concentrations of lipid kinase in multiple reactions will allow one to determine kinase activity.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Densitometric quantification of PI(5)P-containing spots can be performed for a quantitative analysis using FIJI/ImageJ or Image Studio Lite software. The relative amounts of GST-protein bound lipid in each reaction can be normalized to the reaction containing no protein kinase.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The success of this assay is determined by whether the conditions for protein kinase-mediated phosphorylation of the lipid kinase is established in preliminary assessments.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No protein is precipitated.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>The transfection efficiency may be too low and the transfection should therefore be optimized as described above.</p>
<p>The protein may not be correctly expressing the affinity tag. The plasmid should be sequenced to ensure that the sequence is in frame and in case of a C-terminal tag, that there are no stop codons in the sequence.</p>
<p>The lysis may have been inefficient. A number of reasons can affect the efficiency of lysis buffer. NP-40 in a 10% solution should not be kept for longer than a month. Ideally, fresh lysis buffer should be prepared each time.</p>
<p>If the protein-bead binding is too weak, the protein may be washed off the beads. The salt concentration may be adapted, so it is less stringent. Alternatively, a different affinity tag may be introduced by subcloning to ensure optimal binding.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Lipid synthesis is not observed.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Dissolve lipids prior to use. Do not keep aliquots of lipids that have been diluted in the 2:1:08 solution of methanol:chloroform:water in −20°C for longer than 2 weeks, as this may affect the assay.</p>
<p>If the lipid kinase used in the assay is bead-bound, it is possible that the beads may cause steric hindrance and prevent activation of the kinase. Elution of protein may resolve this issue (see above).</p>
<p>To ensure that the lack of lipid synthesis is not due to a technical issue, use a positive control. If an activator for the lipid kinase is available, it may be used to establish that the assay works.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low kinase activity.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>A time course should be performed to establish whether an increased reaction time is required for activation of the lipid kinase.</p>
<p>The effects of additives in the lysis buffer, such as DTT, may be tested to ensure optimal kinase activity as described above.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>High background.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Membrane washes with TBS-T should be increased to 10 × 10 min. The final wash may be extended overnight to limit background.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Di Paolo, G. and De Camilli, P. (2006). Phosphoinositides in cell regulation and membrane dynamics. Nature <i>443</i>, 651-657. <a class="external-link" href="https://doi.org/10.1038/nature05185">https://doi.org/10.1038/nature05185</a></p>
<p id="bib2">Dowler, S., Kular, G., and Alessi, D.R. (2002). Protein lipid overlay assay. Sci. STKE <i>2002</i>, 1-11. <a class="external-link" href="https://doi.org/10.1126/scisignal.1292002pl6">https://doi.org/10.1126/scisignal.1292002pl6</a></p>
<p id="bib3">Gozani, O., Karuman, P., Jones, D.R., Ivanov, D., Cha, J., Lugovskoy, A.A., Baird, C.L., Zhu, H., Field, S.J., Lessnick, S.L., et al. (2003). The PHD finger of the chromatin-associated protein ING2 functions as a nuclear phosphoinositide receptor. Cell <i>114</i>, 99-111. <a class="external-link" href="https://doi.org/10.1016/S0092-8674(03)00480-X">https://doi.org/10.1016/S0092-8674(03)00480-X</a></p>
<p id="bib4">Karabiyik, C., Vicinanza, M., Son, S.M., and Rubinsztein, D.C. (2021). Glucose starvation induces autophagy via ULK1-mediated activation of PIKfyve in an AMPK-dependent manner. Dev. Cell. 1-15. <a class="external-link" href="https://doi.org/10.1016/j.devcel.2021.05.010">https://doi.org/10.1016/j.devcel.2021.05.010</a></p>
<p id="bib5">Li, X., Wang, X., Zhang, X., Zhao, M., Tsang, W.L., Zhang, Y., Yau, R.G.W., Weisman, L.S., and Xu, H. (2013). Genetically encoded fluorescent probe to visualize intracellular phosphatidylinositol 3,5-bisphosphate localization and dynamics. Proc. Natl. Acad. Sci. U S A <i>110</i>, 21165-21170. <a class="external-link" href="https://doi.org/10.1073/pnas.1311864110">https://doi.org/10.1073/pnas.1311864110</a></p>
<p id="bib6">Sbrissa, D., Ikonomov, O.C., Deeb, R., and Shisheva, A. (2002). Phosphatidylinositol 5-phosphate biosynthesis is linked to PIKfyve and is involved in osmotic response pathway in mammalian cells. J. Biol. Chem. <i>277</i>, 47276-47284. <a class="external-link" href="https://doi.org/10.1074/jbc.M207576200">https://doi.org/10.1074/jbc.M207576200</a></p>
<p id="bib7">Shisheva, A., Rusin, B., Ikonomov, O.C., DeMarco, C., and Sbrissa, D. (2001). Localization and insulin-regulated relocation of phosphoinositide 5-kinase PIKfyve in 3T3-L1 adipocytes. J. Biol. Chem. <i>276</i>, 11859-11869. <a class="external-link" href="https://doi.org/10.1074/jbc.M008437200">https://doi.org/10.1074/jbc.M008437200</a></p>
<p id="bib8">Shisheva, A., Sbrissa, D., and Ikonomov, O. (2015). Plentiful PtdIns5P from scanty PtdIns(3,5)P2 or from ample PtdIns? PIKfyve-dependent models: Evidence and speculation (response to: DOI 10.1002/bies.201300012). BioEssays <i>37</i>, 267-277. <a class="external-link" href="https://doi.org/10.1002/bies.201400129">https://doi.org/10.1002/bies.201400129</a></p>
<p id="bib9">Shisheva, A., Sbrissa, D., and Ikonomov, O. (1999). Cloning, characterization, and expression of a novel Zn 2+-binding FYVE finger-containing phosphoinositide kinase in insulin-sensitive cells. Mol. Cell. Biol. <i>19</i>, 623-634. <a class="external-link" href="https://doi.org/10.1128/mcb.19.1.623">https://doi.org/10.1128/mcb.19.1.623</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, David C. Rubinsztein (<a href="mailto:dcr1000@cam.ac.uk">dcr1000@cam.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>Materials generated in this study are available upon request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new datasets or code.</p>
<h3>Acknowledgments</h3>
<p>We are grateful for funding from the UK Dementia Research Institute (funded by the <a href="https://doi.org/10.13039/501100000265">MRC</a>, <a href="https://doi.org/10.13039/501100002283">Alzheimer’s Research UK,</a> and the Alzheimer’s Society) (to D.C.R.), <a href="https://doi.org/10.13039/501100008236">Roger de Spoelberch Foundation</a> (to D.C.R.), and the Gates Cambridge Scholarship (to C.K.) for funding. We thank M. Vicinanza (Glaxo-Smith Kline) and J. Clarke for their advice on lipid kinase assays (ALBORADA Drug Discovery Institute) and A. Shisheva (Wayne State University School of Medicine) for PIKfyve constructs.</p>
<h3>Author contributions</h3>
<p>C.K. developed the protocol and performed experiments. C.K. and D.C.R. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>D.C.R. is a consultant for D.C.R. is a consultant for Alladdin Healthcare Technologies Ltd., Mindrank AI, Nido Biosciences, Drishti Discoveries, and PAQ Therapeutics.</p>
</section>
</article>