<article>
<section>
<h1 id="header">Fast-dissociating but highly specific antibodies are novel tools in biology, especially useful for multiplex super-resolution microscopy</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Takushi Miyoshi,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn1">5</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup> Thomas B. Friedman,<sup><a href="#aff3">3</a></sup> and Naoki Watanabe<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup></p>
<p id="aff1"><sup>1</sup>Laboratory of Single-Molecule Cell Biology, Graduate School of Biostudies, Kyoto University, Kyoto 606-8501, Japan</p>
<p id="aff2"><sup>2</sup>Department of Pharmacology, Graduate School of Medicine, Kyoto University, Kyoto 606-8501, Japan</p>
<p id="aff3"><sup>3</sup>Laboratory of Molecular Genetics, National Institute on Deafness and Other Communication Disorders, National Institutes of Health, Porter Neuroscience Research Center, Room 1F-143A, Bethesda, MD 20892, USA</p>
<p id="aff4"><sup>4</sup>Department of Otolaryngology - Head and Neck Surgery, Graduate School of Medicine, Kyoto University, Kyoto 606-8507, Japan</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:takushi.miyoshi@nih.gov">takushi.miyoshi@nih.gov</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100967">10.1016/j.xpro.2021.100967</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Fast-dissociating, highly specific monoclonal antibodies (FDSAs) are single-molecule imaging probes useful for many biological assays including consecutive, multiplexable super-resolution microscopy. We developed a screening assay to characterize the kinetics of antibody-antigen interactions using single-molecule microscopy and established a pipeline to identify FDSAs from thousands of monoclonal candidates. Provided here are detailed protocols to prepare multi-well glass-bottom plates necessary for our assay to identify hybridoma clones secreting FDSAs. Synthesis of fluorescently labeled Fab fragments (Fab probes) from FDSAs is also described.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref26">Miyoshi et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Identification of fast-dissociating, highly specific monoclonal antibodies (FDSAs)</li>
<li>Kinetic screen of antibodies using semi-automated single-molecule TIRF microscopy</li>
<li>Synthesis of maleimide-labeled Fab probes for multiplex super-resolution microscopy</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used previously to identify fast-dissociating, highly specific monoclonal antibodies (FDSAs) suitable for consecutive, multiplex super-resolution microscopy (<a href="#bib26">Miyoshi et al., 2021</a>). Fluorescently labeled Fab fragments (Fab probes) can be generated from the identified antibodies, including FDSAs, and employed as exchangeable imaging probes in super-resolution microscopy referred to as IRIS (integrating exchangeable single-molecule localization) (<a href="#bib18">Kiuchi et al., 2015</a>), which in principle is a technique related to DNA-PAINT and Exchange-PAINT (<a href="#bib16">Jungmann et al., 2014</a>; <a href="#bib35">Schueder et al., 2017</a>). The availability of FDSAs should also benefit immunoassays requiring the reversibility of antibody-antigen interactions, for example in Fab-based live endogenous modification labeling (FabLEM) and continuous monitoring of serum myoglobin concentration (<a href="#bib9">Hayashi-Takanaka et al., 2011</a>; <a href="#bib39">Song et al., 2015</a>). Theoretically, FDSAs are available through systematic kinetic screening of antibodies for mostly any antigen. However, antibodies characterized by existing bulk protein-protein interaction analyses, such as surface plasmon resonance (SPR) and biolayer interferometry (BLI), usually remain bound to their epitopes for minutes to hours (<a href="#bib2">Canziani et al., 2004</a>; <a href="#bib17">Kamat et al., 2020</a>; <a href="#bib20">Kumaraswamy and Tobias, 2015</a>; <a href="#bib21">Lad et al., 2015</a>; <a href="#bib28">Nikolovska-Coleska, 2015</a>; <a href="#bib32">Safsten et al., 2006</a>; <a href="#bib43">Ylera et al., 2013</a>). FDSAs that can dissociate from epitopes with a 1 second (s) half-life, such as for our anti-FLAG tag and anti-S tag antibodies (<a href="#bib26">Miyoshi et al., 2021</a>), would not be easily identified using current SPR or BLI although perhaps feasible with cutting-edge techniques (<a href="#bib10">Helmerhorst et al., 2012</a>; <a href="#bib42">Wallace, 2002</a>).</p>
<p>Our screening assay employed semi-automated single-molecule fluorescence microscopy to screen for FDSAs to detect weak, transient interactions of FDSAs with their epitopes, even for interactions occurring with a 100 millisecond (ms) half-life (<a href="#bib26">Miyoshi et al., 2021</a>). To improve the low throughput of single-molecule microscopy, we designed a multi-well plate assay and scanned the plates using a total internal reflection fluorescence (TIRF) microscope equipped with an autofocusing device and a programmable electric stage (<a href="#fig1">Figures 1</a>A and 1B). In our previous study, 1,000–2,000 candidate hybridoma supernatants were screened in 1–2 days while the viability of each hybridoma line was maintained. As demonstrated in our study, FDSAs are reagents available through systematic, kinetic screening of candidate antibodies (<a href="#bib26">Miyoshi et al., 2021</a>). FDSAs and their fragments should expand opportunities for assays using antibodies as reversibly-binding probes. Here, we describe our methods to identify FDSAs from thousands of candidate monoclonals using single-molecule microscopy and to proteolytically harvest from them fast-dissociating, highly specific Fab probes.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Design and details of the screening assay</div>
<p>(A) Overview of the screening assay. Amine-functionalized 96-well glass-bottom plates are activated using sulfo-SANPAH, a bifunctional crosslinker that has an <i>N</i>-hydroxysuccinimide (NHS) ester group and a nitrophenyl azide group. Both of these groups react with amine groups although UV illumination is required to activate the nitrophenyl azide group. Sulfo-SANPAH on the glass surface was crosslinked to Protein A/G under UV illumination in order to selectively immobilize antibodies in hybridoma culture supernatants. Immobilized antibodies are confronted with EGFP-antigen. Binding of EGFP-antigen molecules can be visualized in real-time using single-molecule TIRF microscopy. Diagrams are reproduced from our previous study with permission from Elsevier (<a href="#bib26">Miyoshi et al., 2021</a>).</p>
<p>(B) Components of our TIRF microscope. A programable electric stage accepting 96-well plates and an autofocusing device are equipped for semi-automated single-molecule microscopy of antibody-antigen interaction. Images are acquired using an EMCCD camera.</p>
<p>(C) Custom-made 96-well glass-bottom plates used in our assay. A glass bottom equivalent to a high-precision No. 1.5 coverslip is suitable for autofocusing. A low-bottom plastic frame is designed with dimensions so as to avoid interference with the typical wide barrel of a high-NA 100× objective.</p>
</figcaption>
</figure>
<h3 id="sec1.1">Epitope tags and protein immunogens</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 1 month</div>
<ol>
<li>Prepare immunogens to harvest sensitized B cells. Both chemically synthesized peptides and purified proteins are available as immunogens. Conjugate chemically synthesized peptides with keyhole limpet hemocyanin (KLH) to enhance immunogenicity. Emulsify purified proteins in Freund’s complete adjuvant before injecting into mice (<a href="#bib5">Freund et al., 1937</a>; <a href="#bib40">Stils, 2005</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Aliquots of purified proteins can be stored at −80°C until injection.</div>
<div class="note">
<span class="note-title">Note:</span> In the previous study, we developed hybridoma clones against three epitope tags, FLAG-tag, S-tag and V5-tag, and two mouse (m) F-actin crosslinking proteins, plastin1 (mPLS1) and espin isoform 1 (mESPN1). These two proteins are required for normal hearing in humans and mice (<a href="#bib19">Krey et al., 2016</a>; <a href="#bib25">Metlagel et al., 2019</a>; <a href="#bib36">Sekerkova et al., 2011</a>).</div>
<h3 id="sec1.2">Candidate hybridoma clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 1 month</div>
<ol start="2">
<li>Develop hybridoma clones using sensitized B cells as previously published (<a href="#bib12">Hnasko and Stanker, 2015</a>). We fused P3U1 myeloma cells (RIKEN BRC) to the sensitized B cells using Polyethylene Glycol (PEG).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The method used to harvest B cells affects the number of positive clones in the screening assay. We recommend the mouse iliac lymph node method to achieve a high yield of positive hybridoma clones, in which immunogens are injected at the root of tails and B cells are harvested from iliac lymph nodes (<a href="#bib30">Sado et al., 2006</a>; <a href="#bib31">Sado and Okigaki, 1996</a>). Our qualitative sense is that a method using spleen B cells produced fewer positive clones, approximately one fifth to one-tenth compared to the iliac lymph node method.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The culture medium used to develop hybridoma clones must contain a negligible concentration of immunoglobulin, which is normally present in fetal bovine serum (FBS) supplemented to the medium. Normal FBS can contain as much as 50–300 μg/mL of immunoglobulin G (IgG) (<a href="#bib38">Son et al., 2001</a>) and also other Ig subclasses. During the screening assay, such bovine immunoglobulins will compete with the binding of monoclonal candidates to the Protein A/G-coated glass surfaces and significantly decrease the sensitivity of the assay to detect antibody-antigen binding. To lower the concentration of unwanted immunoglobulins in the culture medium, we used Hyclone fetal bovine serum Super-Low IgG from Cytiva, which has an IgG concentration of less than 5 μg/mL. The company substantially removes immunoglobulins by performing Protein G affinity chromatography.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Institutional permission and oversight information for the animal study should be obtained to immunize mice and harvest B cells to develop hybridomas. These processes in our study were performed at the MEDICAL &amp; BIOLOGICAL LABORATORIES CO., LTD. (MBL) and at the Mediridge Co., Ltd. in Japan in accordance with the laws and regulations for animal studies.</div>
<ol start="3">
<li>For each antigen, prepare hybridoma clones seeded in a total 1,000–2,000 wells using 96-well tissue-culture plates after dilution with culture medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Hereafter, we use the term “clones” to refer to cells in each well. However, these “clones” may consist of different populations since two to three colonies often form in a single well. Thus, it is safer to cryopreserve these “clones” or to isolate monoclonal populations (described in the <a href="#step-by-step-method-details">step-by-step method details</a> section) as early as possible after the screening. These hybridoma “clones” were cultured until colonies grow to 1–2 mm in diameter. Fibroblasts found in some wells disappear after subculturing once or twice.</div>
<h3 id="sec1.3">EGFP-antigens for antibody screening</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 1 month</div>
<ol start="4">
<li>Synthesize EGFP-antigens using adherent or suspension HEK293 cells (HEK-293, ATCC; Expi293F, Thermo Fischer Scientific). To screen anti-FLAG tag, anti-S tag and anti-V5 tag hybridoma supernatants, we used lysates of adherent HEK293 cells expressing EGFP with epitope tags at the N-terminus, namely FLAG-EGFP, S-EGFP and V5-EGFP, respectively (<a href="#bib26">Miyoshi et al., 2021</a>). We prepared purified EGFP-mPLS1 and two EGFP-fused mESPN1 protein fragments using Expi293F cells to screen anti-mPLS1 and anti-mESPN1 monoclonal antibodies. See the <b>Note</b> below and the Expected Outcome section for the reason to use fragments of mESPN1. Aliquots of EGFP-antigen solutions can be stored at −80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our assay, the arrangement of antibodies and antigens is “flipped” from the typical designs of antibody screening assays, such as ELISA, western blotting and immunostaining, to avoid the necessity of labelling thousands of candidate monoclonal antibodies. This design is also advantageous for a more accurate determination of dissociation rates (<i>k</i><sub>off</sub>) between antibodies and antigens. If an antigen is immobilized and confronted with an antibody, some antibodies may simultaneously bind to antigens bivalently and thus show an apparent enhanced target residence time (slower dissociation), which is referred to as an avidity effect (<a href="#bib27">Myszka, 1999</a>; <a href="#bib41">Vauquelin and Charlton, 2013</a>).</div>
<div class="note">
<span class="note-title">Note:</span> The epitope recognition of some antibodies can be affected by the amino-acid residues linking EGFP to the antigens (<a href="#bib34">Schuchner et al., 2020</a>). Thus, it might be helpful to prepare several plasmid vectors encoding EGFP-antigens using different linkers and to test EGFP-antigens with commercial antibodies. In our previous study, EGFP is connected to the epitope tags with a short linker (Arg-Ser-Arg-Ala) and to the protein antigens with another short linker (Ser-Gly-Leu-Arg-Ser-Arg-Ala) both of which are encoded in the multiple cloning site of Clonetech pEGFP-C1 vector (<a href="#bib26">Miyoshi et al., 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Some EGFP-antigens, especially large proteins of 100 kDa or greater, may show non-specific binding to the glass surface of the 96 well assay plates even in the absence of immobilized antibodies. Non-specific binding of the EGFP-antigen to glass will not allow an investigator to distinguish whether or not immobilized antibodies truly interact with a confronted EGFP-antigen. One option to reduce non-specific binding is to fragment the antigen into domains, which worked well for mESPN1, a 94.5 kDa F-actin crosslinker (<a href="#bib26">Miyoshi et al., 2021</a>). Another solution is to synthesize peptides from the entire sequence of the antigen and screen antibodies using EGFP-fused peptides. EGFP-fused peptides rarely show non-specific binding to antibody-free glass surface at a concentration of 20–30 nM used in our screening assay. The longest synthetic peptide tested in our study is MKETAAAKFERQHMDS (17 residues of the S-tag including its first methionine) although longer peptides should work. However, this approach requires multiple scans of antibodies to cover as large a proportion of the antigen as possible and potentially overlooks antibodies that recognize epitopes not included in the set of peptides used for screening.</div>
<div class="note">
<span class="note-title">Note:</span> Other monomeric fluorescent proteins, such as mCherry and TagRFP, can be used if they are compatible with the fluorescence filters on the TIRF microscope (<a href="#bib3">Chudakov et al., 2010</a>). Organic fluorescent dyes are also available although labeling methods, especially those targeting residues such as lysine, may interfere with antibody-antigen binding by masking an epitope (<a href="#bib11">Hermanson, 2013</a>). Tags to conjugate organic fluorescent dyes, such as self-labeling tags (e.g., HaloTag, SNAP-tag and CLIP-tag), a tetracysteine tag and a histidine tag, are also available to take advantage of the brightness and robustness of organic dyes (<a href="#bib8">Griffin et al., 1998</a>; <a href="#bib22">Liss et al., 2015</a>; <a href="#bib29">Peciak et al., 2019</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Positive control antibodies, diluted to 0.1–10 μg/mL in PBS with 0.2% Triton X-100</td>
<td>any</td>
<td>any</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Hyclone Super-Low IgG FBS</td>
<td>Cytiva</td>
<td>Cat#SH30898.03</td>
</tr>
<tr>
<td>Sulfo-SANPH Crosslinker</td>
<td>ProteoChem</td>
<td>Cat#c1111-100mg</td>
</tr>
<tr>
<td>Protein A/G</td>
<td>ProSpec</td>
<td>Cat#pro-646</td>
</tr>
<tr>
<td>Glucose oxidase</td>
<td>Sigma</td>
<td>Cat#G6125-10KU</td>
</tr>
<tr>
<td>Catalase</td>
<td>SERVA</td>
<td>Cat#26910</td>
</tr>
<tr>
<td>Protein A Sepharose CL-4B beads</td>
<td>GE Healthcare</td>
<td>Cat#17096303</td>
</tr>
<tr>
<td>DyLight 488 Maleimide</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#46602</td>
</tr>
<tr>
<td>DyLight 550 Maleimide</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#62290</td>
</tr>
<tr>
<td>Papain from papaya latex</td>
<td>Sigma-Aldrich</td>
<td>Cat#P4762</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Nacalai Tesque, Inc.</td>
<td>Cat#35501-15</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HEK293 cells</td>
<td>ATCC</td>
<td>Cat#CRL-1573</td>
</tr>
<tr>
<td>Expi293F cells</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A14635</td>
</tr>
<tr>
<td>P3U1 mouse myeloma cells</td>
<td>RIKEN BRC</td>
<td>Cat#JCRB0708</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Adult female Balb/c mouse</td>
<td>The Jackson Laboratory</td>
<td>Stock No. 000651</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>MetaMorph Microscopy Automation and Image Analysis Software (to control microscope)</td>
<td>Molecular Devices</td>
<td>N/A</td>
</tr>
<tr>
<td>ImageJ (<a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a>)</td>
<td>(<a href="#bib33">Schneider et al., 2012</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad Software, Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Custom-made Python scripts</td>
<td><a href="https://github.com/takushim/tanitracer">https://github.com/takushim/tanitracer</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>96-well cell culture plate, non-treated</td>
<td>IWAKI</td>
<td>Cat#1860-096</td>
</tr>
<tr>
<td>24-well cell culture plate, non-treated</td>
<td>IWAKI</td>
<td>Cat#1860-024</td>
</tr>
<tr>
<td>96-well deep-well storage plates, 1 mL</td>
<td>Nunc</td>
<td>Cat#260251</td>
</tr>
<tr>
<td>Benchtop centrifuge with a 96-well deep-well plate adapter</td>
<td>KUBOTA</td>
<td>Model 4000 and Cat#PT-89M</td>
</tr>
<tr>
<td>96-well multichannel pipettor</td>
<td>Eppendorf</td>
<td>epMotion96; Cat#5069000004</td>
</tr>
<tr>
<td>Automated cell counter</td>
<td>NanoEnTek</td>
<td>EVE-MC</td>
</tr>
<tr>
<td>Bench-top shaker</td>
<td>BIO CRAFT</td>
<td>Model BS-740</td>
</tr>
<tr>
<td>Bench-top rotator</td>
<td>NISSIN</td>
<td>Model NRC-20D</td>
</tr>
<tr>
<td>Amine-functionalized glass bottom plates</td>
<td>Matsunami Glass Ind., Ltd.</td>
<td>Custom-made (see the next section)</td>
</tr>
<tr>
<td>UV transilluminator</td>
<td>NIPPON Genetics</td>
<td>MUV21-365</td>
</tr>
<tr>
<td>Inverted microscope</td>
<td>Olympus</td>
<td>IX83</td>
</tr>
<tr>
<td>Oil objective lens</td>
<td>Olympus</td>
<td>UPlansApo 100×, 1.40 NA</td>
</tr>
<tr>
<td>EMCCD (or CMOS) camera</td>
<td>Roper Scientific</td>
<td>Evolve 512</td>
</tr>
<tr>
<td>Illumination laser</td>
<td>Cobolt</td>
<td>Cobolt Blues 50 mW laser (473 nm)</td>
</tr>
<tr>
<td>Automated Stage</td>
<td>Applied Scientific Instrumentation</td>
<td>MS-2000 Flat-Top XY</td>
</tr>
<tr>
<td>Z-drift compensator</td>
<td>Olympus</td>
<td>IX3-ZDC2</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Multichannel pipettor and aspirator</h3>
<p>For this protocol, cells are cultured in 96-well plates. Multichannel pipettes and aspirators are recommended to shorten sample handling time, which is vital to maintain cell viability and reactivity of reagents, especially sulfo-SANPAH, a bifunctional crosslinker which has a half-life of approximately 10 min in aqueous solution, pH 8.6 at 4°C (<a href="#bib4">Cuatrecasas and Parikh, 1972</a>). We also used a 96-well multichannel Eppendorf epMotion96 robotic liquid handler to suspend and distribute hybridoma cells to be cultured in 96-well plates (<a href="#fig2">Figure 2</a>B).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Subculture of hybridoma clones during the screening assay</div>
<p>(A) Diagram describing the time course of subculture. On day 1, hybridoma clones seeded in one 96-well plate are subcultured into three 96-well plates. One 96-well plate is used for the next passage on day 3 while two plates are used to harvest culture supernatants for the screening assay performed on days 4 and 5. On day 3, hybridoma cells in one of the three plates is subcultured into three 96-well plates. Clones selected by the screening assay on days 4 and 5 are subcultured in 24-well plates and cryopreserved thereafter.</p>
<p>(B) The recommended method to subculture hybridoma clones using multiple 96-well plates. Hybridoma clones seeded in 96-well plates are detached by a gentle jet of culture medium from pipettes. We used a multichannel pipette to avoid repetitive hand movements. Cells are then gently suspended using a 96-well multichannel pipetting machine. Cell suspensions, which are ∼300 μL for each well, are aspirated, and 100 μL is dispensed into each well of three non-treated 96-well culture plates containing 200 μL of fresh medium in each well. In all passages except for the initial seeding of hybridoma clones, we used untreated culture plates to avoid adherence of hybridomas and allow for cells to be easily suspended.</p>
</figcaption>
</figure>
<h3 id="sec2.2">Amine-functionalized 96-well glass-bottom plates</h3>
<p>The 96-well glass bottom plates used in the screening assay are functionalized by amine groups and provided by several manufacturers, such as Corning and Thermo Fisher Scientific. We purchased custom-made glass-bottom plates from Matsunami Glass IND LTD (<a href="#fig1">Figure 1</a>C). The glass bottom of these 96-well plates is equivalent to that for a high-precision No. 1.5 coverslip, which is well-suited for autofocusing. Additionally, these 96 well plates have a low-bottom plastic frame designed with dimensions so as to avoid interference with the typical wide barrel of a high-NA 100× objective (<a href="#fig1">Figure 1</a>C).</p>
<h3 id="sec2.3">Cell culture medium</h3>
<table id="undtbl1">
<caption>Hybridoma culture medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI1640</td>
<td>N/A</td>
<td>500 mL</td>
</tr>
<tr>
<td>Hyclone Super-Low IgG FBS∗</td>
<td>15%</td>
<td>75 mL</td>
</tr>
<tr>
<td>50× HAT Supplement</td>
<td>1×</td>
<td>10 mL</td>
</tr>
<tr>
<td>200 mM L-glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>100 mM Sodium pyruvate</td>
<td>1 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>100× Penicillin-streptomycin</td>
<td>100 units/mL Penicillin, 100 μg/mL Streptomycin</td>
<td>5 mL</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>600 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare aseptically and store at 4°C.<br> ∗ Heat inactivated at 56°C for 30 min and aliquoted at −20°C.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Standard FBS contains a large amount of immunoglobulin, which will hinder immobilizing monoclonal antibodies of interest via Protein A/G in the screening assay. Use low IgG fetal bovine serum.</div>
<div class="note">
<span class="note-title">Note:</span> A different cell culture medium such as Hybridoma-SFM can be substituted for RPIM1640. Use a culture medium formulated for hybridoma clones.</div>
<h3 id="sec2.4">Buffers and solutions</h3>
<table id="undtbl2">
<caption>PBS, pH 7.0 for the screening assay</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>–</td>
<td>200 mL</td>
</tr>
<tr>
<td>HCl</td>
<td>–</td>
<td>Adjust to pH 7.0</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>∼200 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare before use and store at 20°C–25°C.</p>
<table id="undtbl3">
<caption>100× Glucose oxidase stock solution for the screening assay</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.1 M Pipes-KOH, pH 7.1</td>
<td>12 mM</td>
<td>1.2 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>2 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>0.5 M EGTA-KOH, pH 8.0</td>
<td>1 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Glucose oxidase</td>
<td>20 mg/mL</td>
<td>200 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C and use within three months.</p>
<table id="undtbl4">
<caption>100× Catalase stock solution for the screening assay</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.1 M Pipes-KOH, pH 7.1</td>
<td>12 mM</td>
<td>1.2 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>2 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>0.5 M EGTA-KOH, pH 8.0</td>
<td>1 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Catalase</td>
<td>3.5 mg/mL</td>
<td>35 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>-</td>
</tr>
<tr>
<td>Total</td>
<td>-</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −80°C and use within three months (<a href="#bib23">Lyubimova et al., 2013</a>).</p>
<table id="undtbl5">
<caption>Protein A/G stock solution for the screening assay</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>–</td>
<td>10 mL</td>
</tr>
<tr>
<td>Protein A/G</td>
<td>10 mg/mL</td>
<td>100 mg</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>∼10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C and use within three months.</p>
<table id="undtbl6">
<caption>HEPES-KCl-Tx buffer for the screening assay</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M HEPES-KOH</td>
<td>10 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>3 M KCl</td>
<td>90 mM</td>
<td>6 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>3 mM</td>
<td>600 μL</td>
</tr>
<tr>
<td>20% Triton X-100</td>
<td>0.2%</td>
<td>2 mL</td>
</tr>
<tr>
<td>0.1 M Dithiothreitol</td>
<td>0.1 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>200 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare just before use and, if necessary, store only briefly on ice.</p>
<table id="undtbl7">
<caption>EGFP-antigen solution for the screening assay</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES-KCl-Tx buffer</td>
<td>–</td>
<td>10 mL</td>
</tr>
<tr>
<td>100× Glucose oxidase stock solution</td>
<td>1×</td>
<td>100 μL</td>
</tr>
<tr>
<td>100× Catalase stock solution</td>
<td>1×</td>
<td>100 μL</td>
</tr>
<tr>
<td>20% Glucose</td>
<td>0.45%</td>
<td>225 μL</td>
</tr>
<tr>
<td>EGFP-antigen stock solution∗</td>
<td>5–30 nM∗∗</td>
<td>varies with the concentration of EGFP-antigen stock solution</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>∼10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare every time before scanning a glass-bottom plate using the TIRF microscope. Apply to the plate and start screening within 5 min.<br> ∗Concentration of EGFP-antigen can be determined using fluorescence intensity of EGFP measured with a spectrophotometer or a multi-well plate reader. Concentrations of purified proteins can be determined, for example, using a Bradford protein assay.<br> ∗∗The final concentration must be optimized for each EGFP-antigen. Test EGFP-antigen using antibody-free glass surface (i.e., wells only treated with culture medium) and glass-surface immobilizing positive control antibodies. Apply EGFP-antigen to antibody-free glass surface at various concentrations and determine the maximum concentration at which the EGFP-antigen does not show non-specific binding. Also apply an EGFP-antigen to the glass surface immobilizing a commercial antibody as a positive control, for example, an antibody developed against the antigen of interest. For EGFP-antigen with an epitope tag for purification, such as a FLAG-tag, an antibody recognizing the tag can be used as a positive control. Confirm that a bound EGFP-antigen molecule is visible against the background fluorescence of diffusing EGFP-antigen molecules using positive control antibodies.</p>
<div class="critical">
<span class="critical-title">Critical:</span> A high concentration of EGFP-antigen is advantageous to identify low-affinity antibodies although the greater background fluorescent of diffusing EGFP-antigen hinders detecting fluorescent spots of single EGFP-antigen molecules captured by immobilized antibodies. In addition, EGFP-antigen at a high concentration may show non-specific binding to the glass surface or may aggregate. We used epitope-tagged EGFP at 30 nM, the maximum concentration that our TIRF microscope can detect a signal against a fluorescent background of diffusing EGFP-antigen. We applied EGFP-fused proteins and fragments at 20 nM unless the EGFP-antigen at this concentration aggregates or binds non-specifically. See our previous study for the relationship between the sensitivity of our screening assay to antibody-antigen binding and the concentration of an EGFP-antigen (<a href="#bib26">Miyoshi et al., 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Prepare the solution of the EGFP-antigen immediately before using it in the scanning assay of the 96-well plates. The solution weakens gradually since glucose oxidase immediately begins to oxidize the glucose. The concentration of EGFP-antigen may also decrease if it is adsorbed to the container wall.</div>
<table id="undtbl8">
<caption>PBS-EDTA, pH 7.0 for the synthesis of Fab probes</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>–</td>
<td>100 mL</td>
</tr>
<tr>
<td>0.5M EDTA-NaOH, pH 8.0</td>
<td>10 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>HCl</td>
<td>–</td>
<td>Adjust to pH 7.0</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>∼100 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare before use and store at 20°C–25°C.</p>
<table id="undtbl9">
<caption>Papain solution for synthesis of Fab probes</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH 8.0</td>
<td>50 mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>100 mM Cysteine-HCl</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>0.5M EDTA-NaOH</td>
<td>2 mM</td>
<td>40 μL</td>
</tr>
<tr>
<td>1 mg/mL Papain in PBS</td>
<td>0.01 mg/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>–</td>
<td>–</td>
</tr>
<tr>
<td>Total</td>
<td>–</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare before use and store on ice.</p>
<h3 id="sec2.5">TIRF microscope</h3>
<p>An inverted microscope equipped for single-molecule microscopy is necessary for the screening assay described herein. Fully functional TIRF microscopes are commercially available from Olympus, Zeiss and Nikon. Alternatively, a custom-made TIRF microscope, which is prism-based and different from our microscope, can be constructed from commercially available microscope parts (<a href="#bib7">Gibbs et al., 2018</a>). The microscope stage needs to accept 96-well plates and be programmable to scan each well semi-automatically (<a href="#fig1">Figure 1</a>B). We controlled a programable electric stage and an autofocusing device using MetaMorph Microscopy Automation and Image Analysis Software.</p>
<h3 id="sec2.6">Software for image analyses (optional)</h3>
<p>Regression analyses of bound EGFP-antigen can be used to determine the <i>k</i><sub>off</sub> values between immobilized antibodies and EGFP-antigens. We used custom-made Python scripts to track fluorescence spots of bound EGFP-antigen molecules (<a href="https://github.com/takushim/tanitracer">https://github.com/takushim/tanitracer</a>). One-phase decay models were fit to the regression curves using GraphPad Prism software.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Day 1: Subculturing hybridoma clones #1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h, see <a href="#fig2">Figure 2</a> <b>for details</b>
</div>
<ol>
<li>The procedure to subculture hybridoma cells is described in <a href="#fig2">Figure 2</a>A. Prepare three non-treated 96-well tissue culture plates for each 96-well plate used to culture hybridoma clones. A total of thirty 96-well plates are necessary if hybridomas are initially seeded on ten 96-well plates.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our protocol, hybridomas are cultured in 300 μL of culture medium in each well of a 96-well plate.</div>
<div class="note">
<span class="note-title">Note:</span> Growth rates of hybridoma cells can vary among clones. In the previous study, we waited until a colony attained a size of 1–2 mm in diameter in one half to two-thirds of the wells with hybridoma cells. Ignore wells without hybridomas or with only fibroblasts. Two or three colonies usually form in each well. Overgrowth damages hybridomas. Start subcultures when the growth medium color is red to orange or when colonies grow in diameter to 1–2 mm. Do not let cells grow until the medium becomes yellow (pH &lt; 7.0), which indicates that the buffering capacity of the medium is exhausted.</div>
<ol start="2">
<li>Put 200 μL of fresh culture medium in each well of the new plates (shown in <a href="#fig2">Figure 2</a>B, right panel).</li>
<li>Detach hybridoma clones from the tissue culture plates by suspending culture medium in each well using a multichannel pipette (<a href="#fig2">Figure 2</a>B, left panel). The fluid jet from the pipette should be gentle enough to prevent splashing and contaminating the culture medium from one well to another. Hybridomas weakly adhere to the 96-well plate treated for tissue culturing. Cells can be detached by gently flowing the culture medium against the surface of wells. Detachment of cells from the well bottom can be seen with the naked eye.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use a new pipette tip for each well. Cross-contamination between hybridoma clones can result in the loss of an antibody-secreting clone, which can be outcompeted in growth by non-antibody secreting cells.</div>
<ol start="4">
<li>To avoid considerable repetitive hand movements, if available, use a multichannel automated pipettor, such as an Eppendorf epMotion96 (<a href="#fig2">Figure 2</a>B, center and right panels):
<ol type="a">
<li>Attached new 300-μL tips to the automated pipettor.</li>
<li>Suspended the detached hybridomas by aspirating and dispensing 250 μL of culture medium three times for each well.</li>
<li>Aspirate a total 300 μL of cell suspension from each original well and dispense 100 μl into each of three wells of three 96-well plates.</li>
</ol>
</li>
<li>Culture the hybridomas at 37°C for 2 days in a 5% CO<sub>2</sub> humidified atmosphere.</li>
</ol>
<h3 id="sec3.2">Day 2: Crosslinking of protein A/G to 96-well glass-bottom plates</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h, see <a href="#fig1">Figure 1</a> <b>for an overview</b>
</div>
<ol start="6">
<li>Prepare amine-functionalized 96-well glass-bottom plates, which are available from several manufacturers such as Matsunami Glass IND LTD and Corning. From Matsunami, we purchased custom requested 96-well plates with a glass bottom equivalent to a high-precision No. 1.5 coverslip and with a low-bottom plastic frame designed with dimensions so as to avoid interference with the typical wide barrel of a high NA 100× objective (<a href="#fig1">Figure 1</a>C).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare additional 96-well plates for positive control antibodies and antibody-free culture medium as a negative control. Commercial antibodies can be used as positive controls. If the EGFP-antigen has an epitope tag for purification, such as a FLAG-tag, an antibody recognizing the tag can be a positive control.</div>
<ol start="7">
<li>For one plate, prepare 10 mL of the 10 mM sulfo-SANPAH solution. Weigh 4.93 mg of sulfo-SANPAH in a 15 mL conical tube. Add 100 μL of DMSO and completely dissolve the sulfo-SANPAH using a vortex mixer for 15–30 s. Then, add 10 mL of PBS, pH 7.0 and vortex the solution once again for 15–30 s.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Sulfo-SANPAH is sensitive to moisture. Keep sulfo-SANPAH at −20°C and avoid humid air. Prepare the sulfo-SANPAH solution immediately before use.</div>
<div class="note">
<span class="note-title">Note:</span> Sulfo-SANPAH is poorly soluble in water. A small amount of sulfo-SANPAH may gradually precipitate after adding PBS at pH 7.0 and sometimes may appear on the glass surface of the 96-well assay plate. It is unnecessary to remove the precipitate since it is completely removed from the glass surface of the 96-well assay plate after the rinsing step below and does not affect the success of the methodology described here.</div>
<div class="note">
<span class="note-title">Note:</span> The pH of PBS was adjusted to 7.0 which appears to increase the stability of the N-hydroxysuccinimide (NHS) ester group of sulfo-SANPAH (<a href="#bib11">Hermanson, 2013</a>).</div>
<ol start="8">
<li>Apply 100 μL of sulfo-SANPAH solution to each well. Wrap the plates with an aluminum foil to shield from light and incubate the plates at 20°C–25°C for 2 hours (h) gently mixing the solution on a shaker at ∼10 rpm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sulfo-SANPAH is a bifunctional crosslinker. At this step, the NHS ester side of sulfo-SANPAH reacts with the amine groups on the glass surface of glass-bottom plates. Until the Protein A/G crosslinking is completed as described below, shield the sulfo-SANPAH and plate from light to protect the other side of sulfo-SANPAH, which is a photoactivatable nitrophenyl azide.</div>
<ol start="9">
<li>Remove the sulfo-SANPAH solution from each well with a multichannel aspirator, such as ASPIT-8S (OptoCode), and rinse wells three times with PBS pH 7.0. Incubate for 3 min at 20°C–25°C for each rinse.</li>
<li>Add 300 μL of 10 mg/mL Protein A/G stock solution to 10 mL PBS, pH 7.0 to prepare the 0.3 mg/mL solution. Mix by gently inverting the tube. Do not vortex. Apply 100 μL to each well.</li>
</ol>
<ol start="11">
<li>Place the plates on a UV transilluminator. For MUV21-365 (NIPPON Genetics), select “High” and incubate for 30 min. Do not expose your eyes to the UV light from the transilluminator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Optimal strength and duration of UV illumination differ between UV transilluminators. Thermo Fisher Scientific provides a technical tip to determine the UV parameters (<a href="https://assets.thermofisher.com/TFS-Assets/LSG/Application-Notes/TR0011-Photoactivate-aryl-azides.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/Application-Notes/TR0011-Photoactivate-aryl-azides.pdf</a>).</div>
<ol start="12">
<li>Gently rock the plates on a shaker for 30 min at 20°C–25°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Our transilluminator uses UV lamps that will warm the plates. Therefore, plates are cooled down during this step by incubating at 20°C–25°C while gently rocking. Recent transilluminators with LEDs that produce little heat may not require a cooling step.</div>
<ol start="13">
<li>Remove the Protein A/G solution from each well.</li>
<li>Block the non-specific protein binding sites in each well by adding 300 μL of 3% BSA dissolved in PBS and incubate the plate for 16–18 h at 4°C with gentle shaking.</li>
</ol>
<h3 id="sec3.3">Day 3: Subculture of hybridomas #2 and sampling of culture supernatants</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h, see <a href="#fig2">Figure 2</a> <b>for details</b>
</div>
<ol start="15">
<li>In this step, hybridoma clones cultured in one 96-well plate on day 1 is subcultured in a set of three plates (see <a href="#fig2">Figure 2</a>A). Prepare three new 96-well non-treated tissue culture plates for each 96-well plate of hybridomas to be subcultured. Aliquot 200 μL of culture medium to each well. In other words, hybridoma cells initially cultured in one well are now cultured in triplicate using three wells of three separate 96-well plates.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For example, hybridomas initially cultured in the wells of 10 plates are now cultured using 30 plates (10 sets of 3 plates). At this step, one plate in each set (for a total 10 plates) is subcultured into three fresh plates. The remaining twenty plates are used as a source of hybridoma culture supernatant containing the monoclonal antibodies and are not used to subculture hybridomas.</div>
<ol start="16">
<li>Using a 96-well multichannel pipetting instrument, suspend hybridoma clones in the 96-well plates and dispense 100 μL to each of three 96-well tissue culture plates containing 200 ul of fresh culture medium in each well (<a href="#fig2">Figure 2</a>B, center and right panels).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Hybridomas can be easily suspended by gentle pipetting because the cells are cultured using non-treated tissue culture plates. The procedure to detach cells, such as a fluid jet as described above and in the left panel of <a href="#fig2">Figure 2</a>B, is not necessary.</div>
<ol start="17">
<li>Culture the hybridomas at 37°C in 5% CO<sub>2</sub> while the antibodies are being screened over a period of 4 and 5 days.</li>
<li>After finishing the subculture steps described above, sample culture supernatants by suspending cells in the plates not used for subculture using a 96-well multichannel pipettor. Aspirate cell suspensions and transfer them into 96-1-mL deep-well plates. Centrifuge the plates at ∼180 × <i>g</i> for 2 min to pellet the cells.</li>
<li>Aspirate the culture supernatants and store in new 96-1-mL deep-well plates. To prevent evaporation of supernatants, apply a sealable lid and store the plates at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add 1/100 volume of 3% sodium azide (NaN<sub>3</sub>) to each well for long-term storage of culture supernatants. Handle powdered sodium azide carefully. It is odorless and is a potentially deadly chemical. Dispose of sodium azide according to the chemical waste requirements of your institution.</div>
<div class="note">
<span class="note-title">Note:</span> Cell pellets can be suspended in a cell cryopreservation medium for storage at –80°C. CELLBANKER1 (AMSBIO) worked well. Barcoded cryopreservation tubes (Thermo Fischer Scientific, Cat#3744) are a good way to keep track of your inventory of cells. A barcode reader for these tubes is also available from Thermo Fischer Scientific (Cat#3125).</div>
<ol start="20">
<li>Rinse each well of the 96-wells glass-bottom plates (prepared on day 2) using PBS. Incubate for 3 min at 20°C–25°C for each rinse.</li>
<li>Apply 300 μL of culture supernatant for each well of the glass-bottom plates and incubate for 16–18 h at 4°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare positive control wells immobilizing positive control antibodies and negative control wells treated with antibody-free culture medium. These control wells function as standards to evaluate the quality of the assay plates and to identify antibodies showing positive responses to confronted EGFP-antigens.</div>
<h3 id="sec3.4">Days 4 and 5: Screening monoclonal antibodies and selection of clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h per plate<b>,</b> <b>see</b> <a href="#fig3">Figure 3</a> <b>for representative images</b>
</div>
<ol start="22">
<li>Power up the TIRF microscope and adjust the angle of laser light to achieve TIRF illumination. Place a new 96-well glass-bottom plate and add a solution of calibration beads to a few wells. Tilt the laser angle stepwise from the epi-fluorescence position to find the narrow range of angles where the background fluorescence intensity suddenly decreases. Avoid an angle of TIRF illumination that results in interference fringes (<a href="#bib24">Mattheyses et al., 2010</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative images of screening</div>
<p>Binding of anti-FLAG tag antibodies with FLAG-EGFP antigens are shown (<a href="#bib26">Miyoshi et al., 2021</a>)</p>
<p>(A) Images from the initial screening showing positive responses of three anti-FLAG tag antibodies to FLAG-EGFP. Antibodies were numbered from FLAG-1 to FLAG-3, from high density to a low density of bound FLAG-EGFP molecules. Bound FLAG-EGFP molecules are visualized as fluorescent spots for FLAG-2 and FLAG-3 (magenta circles), while each molecule is hardly distinguishable for FLAG-1 due to the high density of bound FLAG-EGFP molecules. Antibody-free glass surface treated only with fresh culture medium did not show binding of FLAG-EGFP in this image. FLAG-EGFP, 30 nM. Exposure, 50 ms. Bar, 2 μm.</p>
<p>(B) Time-lapse images acquired during the initial screening indicated fast-dissociation of FLAG-2 and FLAG-3 antibodies. Bound FLAG-EGFP molecules are frequently exchanged. Bound FLAG-EGFP molecules are indicated by circles: red (new binding), blue (disappears in the next frame) and orange (remains bound). Molecules that remain bound for only one frame are indicated by circles composed of red and blue arcs. FLAG-EGFP, 30 nM. Time-lapse, every 50 ms. Bar, 2 μm.</p>
<p>(C) Time-lapse images used to determine the dissociation rates (<i>k</i><sub>off</sub>). Acquisition intervals are adjusted to visualize a 30%–50% exchange of bound FLAG-EGFP molecules in each time-lapse image. FLAG-EGFP, 0.1 nM for FLAG-1, 0.3 nM for FLAG-2 and 30 nM for FLAG-3. Bar, 2 μm.</p>
<p>(D) Fitting a one-phase decay model to the regression of bound FLAG-EGFP. The FLAG-2 and FLAG-3 antibodies show faster dissociations from epitopes than the FLAG-1 antibody consistent with data from the initial screening. Images and graphs are reproduced from our previous study with permission from Elsevier (<a href="#bib26">Miyoshi et al., 2021</a>). See <a href="#mmc1">Methods video S1</a>, <a href="#mmc2">S2</a>, <a href="#mmc3">S3</a> and <a href="#mmc4">S4</a>.</p>
</figcaption>
</figure>
</li>
<li>Remove the unbound components of culture supernatants by rinsing the wells of the 96-well glass-bottom three times for 3 min using 300 μL of HEPES-KCl-Tx buffer.</li>
<li>Prepare EGFP-antigen solutions as described in the materials and equipment section.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The concentration of EGFP-antigen directly affects the sensitivity of the screening assay. We used 20–30 nM of EGFP-antigen solution. However, some EGFP-antigens cannot be used at this concentration due to aggregation of the protein and/or to non-specific binding to the glass surface. Before developing hybridoma clones, we recommend testing (1) the EGFP-antigen using an antibody-free glass surface, specifically wells treated with culture medium, and (2) the glass-surface with an immobilized positive control antibody, for example, against the antigen of interest. If the EGFP-antigen has an epitope tag for purification, such as a FLAG-tag, antibodies against the tag, such as an anti-FLAG tag antibody, can be the positive control. Apply EGFP-antigen at various concentrations and determine the maximum concentration at which the EGFP-antigen does not show non-specific binding. Set up the imaging condition, such as laser power and exposure time, to detect bound single EGFP-antigens molecules using the positive control antibody.</div>
<ol start="25">
<li>Before applying EGFP-antigen solution, place the glass-bottom plate upside-down on a paper towel and thinly spread the immersion oil on the lens-side of the entire glass-bottom.</li>
<li>Without disturbing or accidentally blotting the immersion oil, place the glass-bottom plate in an upright position and add 100 μL of EGFP-antigen solution to each well.</li>
<li>Acquire short time-lapse movies sequentially for each well, for example, for 50 frames. We set the exposure time to 50 ms and the laser power (473 nm) to 4–6 mW, both of which are minimized to reduce photobleaching. The Evolve 512 EMCCD camera was used with EM Gain (EM = 300, Gain State 3, 10 MHz). We used a MetaMorph macroinstruction to control the automated stage and the autofocusing device.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Scan a plate treated with positive and negative controls before scanning antibodies of interest to exclude the possibility of troubles with the TIRF microscope, assay plates or EGFP-antigen.</div>
<div class="note">
<span class="note-title">Note:</span> The concentration of EGFP-antigen may gradually decrease by being adsorbed to the glass surface and walls of wells. We started scanning within 5 min after adding EGFP-antigen solution to each well and finished the screening within 1 hour. You can scan the 96 wells group-by-group, for example, with every 24 wells consisting of three columns and eight rows and apply EGFP-antigen solution before scanning each group of wells.</div>
<ol start="28">
<li>Compare the density of bound EGFP-antigen in each well with a negative control well that was treated with antibody-free culture medium. If the density of bound EGFP-antigen is larger than that in a negative control well, antibodies immobilized in the well can be identified as “showing a positive response” to the EGFP-antigen. See <a href="#fig3">Figures 3</a>A and 3B) and <a href="#mmc1">Methods video S1</a> for representative images.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Fast-dissociating antibodies usually show frequent exchange of bound EGFP-antigen molecules during the screening. Further evaluation using diluted EGFP-antigen is necessary if the density of bound EGFP-antigen is too high to observe the exchange of EGFP-antigen.</div>
<ol start="29">
<li>Identify and subculture “positive” clones (see the third and fourth panels in <a href="#fig2">Figure 2</a>A). For each clone, suspend cells in three wells in all of the three tissue culture plates (total ∼900 μL) and put them in a well of a non-treated, 24-well plate. Add 2 mL of fresh culture medium to the well and subculture the hybridomas by dispensing 1 mL of the total ∼3 mL into two unused wells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our protocol, hybridomas identified by the screening are cultured in ∼1 mL of culture medium using non-treated, 24-well tissue culture plates.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Fast-dissociating anti-FLAG tag antibodies identified in initial screen, related to step 28</div>
<p>Time-lapse movie corresponding to data in <a href="#fig3">Figure 3</a>B. FLAG-EGFP molecules show frequent binding and dissociation from immobilized FLAG-2 and FLAG-3 antibodies. Bound FLAG-EGFP molecules are indicated by the circles: red (new binding), blue (disappear in the next frame) and orange (remain bound). Molecules that remain bound for only one frame are indicated by circles composed of red and blue arcs. Time-lapse, every 50 ms. FLAG-EGFP, 30 nM. Bar, 2 μm. The movies, S1 and S3–S5, are reproduced from our previous study with permission from Elsevier (Miyoshi et al., 2021)</p>
</figcaption>
</figure>
<h3 id="sec3.5">Day 7 and thereafter: Cryopreservation and isolation of monoclonal populations</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="30">
<li>Each identified hybridoma clone is cultured using two or three wells of a 24-well tissue culture plate. Use cells in one well for subculture and cells in the remaining one or two wells for cryopreservation. Add 2 mL of culture medium to one well, suspend gently and subculture them in three wells.</li>
<li>Cryopreserve cells that were not used for subculture in the previous step. CELLBANKER1 (AMSBIO) worked effectively for us to preserve hybridomas by performing the following steps:
<ol type="a">
<li>Centrifuge the cell suspension at ∼180 × <i>g</i> for 2 min.</li>
<li>Remove the culture medium supernatant.</li>
<li>Add 1 mL of CELLBANKER1 to the cells from one well and gently suspend (i.e., use 2 mL of CELLBANKER1 for cells collected from two wells).</li>
<li>Transfer the cell suspension to cryotubes and freeze at −80°C in a microtube storage box.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> CELLBANKER1 does not require step-down freezing reducing the time to handle a large number of clones and eliminating the purchase of a device for stepping down the temperature at one degree per minute.</div>
<div class="note">
<span class="note-title">Note:</span> We usually collect culture supernatants after cells are centrifuged for cryopreservation and use them to evaluate the antibody secretion of cryopreserved clones and to synthesize Fab probes. Culture supernatants can be stored up to ∼1 month at 4°C after adding 1/100 volume of 3% sodium azide (NaN<sub>3</sub>). Aliquot and freeze hybridoma supernatants at −20°C or preferably at −80°C for longer-term storage. Some antibodies are sensitive to freeze-thaw cycles. Be sure to check a small quantity of each of your monoclonals for retention of function following a freeze thaw.</div>
<ol start="32">
<li>Repeat the previous step for subculture and additional cryopreservation of hybridoma clones. We usually prepare 4–6 tubes of cryopreserved cells for each hybridoma clone. Subsequently, to obtain monoclonal antibodies, hybridoma clones were subcultured until we obtained ∼12 mL of culture supernatant.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Hybridoma clones sometimes lose their antibody secretion abilities. Therefore, cryopreserve hybridoma clones as early as possible in their passage number and definitely avoid repeated passaging of hybridomas. It is safer to perform limiting dilution (described below) to isolate monoclonal populations while cells are being cryopreserved.</div>
<ol start="33">
<li>Isolate monoclonal populations using a limiting dilution technique. Both cells currently cultured and cells recovered from cryopreservation are available for isolating cells to grow into monoclonal populations.
<ol type="a">
<li>Suspend cells gently using a pipette and determine the density of cells. We used an EVE automatic cell counter (EVE-MC, NanoEnTek) after trypan blue staining. An inexpensive hemocytometer can also be used to count cells.</li>
<li>Dilute a cell suspension to achieve ∼1 cell per 100 μL of culture medium. For example, a cell suspension containing 1 × 10<sup>4</sup> cells/mL should be diluted by one to a hundred (i.e., add 10 μL of cell suspension to 1 mL of culture medium) to achieve 100 cells/mL.</li>
<li>Add 10 μL of diluted cell suspension to 10 mL of culture medium to achieve one cell per 100 μL (0.1 cell/mL).</li>
<li>Aliquot 100 μL of this suspension to each well of a 96-well tissue culture plate to start culturing from single cells. To increase the probability of having only one cell per well, adjust the dilution of cells for a target of ∼0.5 cells per well since the number of cells in a well follows a Poisson distribution.</li>
<li>Add 100 μL of fresh culture medium two days later and then again at four days after starting the limiting dilution. Incubate the plate until cells proliferate. Medium exchange is not necessary.</li>
<li>Identify wells in which cells proliferate and subculture the monoclonal populations. Cryopreserve the cells and obtain hybridoma culture supernatants for evaluation.</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Option #1: Determination of <i>k</i><sub>off</sub> between antibodies and epitopes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h (except for the preparation of Protein A/G-coated glass-bottom plates), see <a href="#fig3">Figure 3</a> <b>for representative images</b>
</div>
<ol start="34">
<li>Prepare a new Protein A/G-coated glass-bottom plate. Apply hybridoma culture supernatants containing antibodies and incubate for 16–18 h at 4°C to immobilize the antibodies.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Synthesis of fluorescently-labeled Fab fragments (Fab probes)</div>
<p>(A) Diagram illustrating the protocol to synthesize Fab probes. Antibodies in culture supernatants are captured on Protein-A beads and labeled via free cysteine residues using fluorescent dyes coupled with maleimide, such as DyLight488-maleimide and DyLight550-maleimide. Fab probes are obtained from antibodies by proteolytic cleavage with Papain. Specificity of Fab probes can be tested, for example, by preparing cultured cells ectopically expressing target antigens or epitopes. We used <i>Xenopus</i> XTC cells, which were fixed and permeabilized with 3.7% paraformaldehyde supplemented with 0.5% Triton X-100 for 20 min at 20°C–25°C, and then blocked with 3% BSA in PBS for 30 min at 20°C–25°C. Recognition of epitope tagged actin filaments are shown.</p>
<p>(B) Fast-dissociating anti-epitope tag Fab probes showing frequent binding and dissociation against epitope-tagged actin expressed in <i>Xenopus</i> XTC cells. Three Fab probes, named as FLAG-2, S-1 and V5-3 in our previous study, recognize FLAG-actin, S-actin and V5-actin, respectively. Time-lapse images, every 200 ms. Fab probes, 1 nM. Bars, 5 μm and 2 μm.</p>
<p>(C) Super-resolution images of XTC cells expressing epitope-tagged actin acquired using these Fab probes. Thin actin fibers are visualized at high-density labeling (between arrowheads) reflecting the frequent binding and dissociation of these Fab probes. Images are reconstructed from 160,000 frames acquired every 50 ms. Fab probes, 1 nM. Bars, 5 μm and 1 μm. Diagrams and images are reproduced from our previous study with permission from Elsevier (<a href="#bib26">Miyoshi et al., 2021</a>). See also <a href="#mmc5">Methods video S5</a>.</p>
</figcaption>
</figure>
</li>
<li>Remove the unbound components of culture supernatants by rinsing the wells using 300 μL of HEPES-KCl-Tx buffer. Incubate for 3 min at 20°C–25°C for each rinse.</li>
<li>Place the 96-well glass-bottom plate on the stage of the TIRF microscope. Just prior to starting acquisition of time-lapse images, remove the HEPES-KCl-Tx buffer and apply diluted EGFP-antigen. We usually dilute the EGFP-antigen to 0.1 nM–1 nM to decrease the density of bound EGFP-antigen molecules.</li>
<li>Acquire time-lapse images at appropriate intervals. Try several time-intervals, typically in the range of 50 ms to 10 s, and find an interval to achieve a 30%–50% exchange of bound EGFP-antigen molecules per frame. For fast-dissociating antibodies, we acquired images using the stream acquisition mode of MetaMorph at 50–300 ms intervals. For slow-dissociating antibodies, we acquired images every 500 ms to 10 s with a 50-ms exposure time using a time-lapse acquisition mode of MetaMorph. We set the laser power (473 nm) to 4–6 mW, which was adjusted depending on the exposure time. The Evolve 512 EMCCD camera was used with EM Gain (EM = 300, Gain State 3, 10 MHz). The appropriate interval of acquisition was determined by tracking the regression of bound EGFP-antigen molecules and fitting a one-phase decay model as described below. See <a href="#fig3">Figure 3</a>C and <a href="#mmc2">Methods video S2</a>, <a href="#mmc3">S3</a> and <a href="#mmc4">S4</a> for representative images.</li>
<li>Track the regression of bound EGFP-antigen molecules. Our custom-made Python scripts (<a href="https://github.com/takushim/tanitracer">https://github.com/takushim/tanitracer</a>) are available to automatically track the spots of single molecules.</li>
<li>Fit a one-phase decay model to the regression curve of bound EGFP-antigen molecules to determine the <i>k</i><sub>off</sub> between antibodies and epitopes. We used GraphPad Prism software for kinetic analyses. Multi-phase decay model may better fit the data for some antibodies although the physiological meaning of a more complex model would need to be considered. See <a href="#fig3">Figure 3</a>D for representative graphs.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Determine the photobleaching rate with the same illumination condition and minimize it by reducing the laser power and exposure time. We prepared weakly-fixed <i>Xenopus</i> XTC cells expressing EGFP-actin and immersed the cells in HEPES-KCl-Tx buffer supplemented with glucose, glucose oxidase and catalase as described in the preparation of EGFP-antigen solution (<a href="#bib26">Miyoshi et al., 2021</a>). We set the laser power and the exposure time to cause photobleaching at a rate approximately 10-fold slower than the <i>k</i><sub>off</sub>.</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Mmc2.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S2. Slow dissociation between FLAG-1 antibody and FLAG-EGFP, related to step 37</div>
<p>Time-lapse movie corresponding to data in <a href="#fig3">Figure 3</a>C. FLAG-EGFP is applied to FLAG-1 antibody immobilized from hybridoma culture supernatant. Images were acquired every 5 s using 0.1 nM of FLAG-EGFP. Exposure, 100 ms. Color key for circles is the same as in <a href="#mmc1">Methods video S1</a>. Bar, 2 μm</p>
</figcaption>
</figure>
<figure id="mmc3"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Mmc3.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S3. Fast dissociation between FLAG-2 antibody and FLAG-EGFP, related to step 37</div>
<p>Time-lapse movie corresponding to data in <a href="#fig3">Figure 3</a>C. FLAG-EGFP is applied to FLAG-2 antibody immobilized from hybridoma culture supernatant. Images were acquired every 300 ms using 0.3 nM of FLAG-EGFP. Color key for circles is the same as in <a href="#mmc1">Methods video S1</a>. Bar, 2 μm</p>
</figcaption>
</figure>
<figure id="mmc4"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Mmc4.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S4. Extremely fast dissociation of FLAG-3 antibody and FLAG-EGFP, related to step 37</div>
<p>Time-lapse movie corresponding to data in <a href="#fig3">Figure 3</a>C. FLAG-EGFP is applied to FLAG-3 antibody immobilized from a hybridoma culture supernatant. Images were acquired every 50 ms using 30 nM of FLAG-EGFP. Color key for circles is the same as in <a href="#mmc1">Methods video S1</a>. Bar, 2 μm</p>
</figcaption>
</figure>
<h3 id="sec3.7">Option #2: Synthesis of Fab probes labeled at free cysteine residues</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h (except for antibody collection using Protein A beads), see <a href="#fig4">Figure 4</a> <b>for the overview and representative images</b>
</div>
<ol start="40">
<li>For 2 mL of hybridoma culture supernatant, add 20 μL of Protein A beads (Cytiva) suspended in PBS (40 μL of the beads in PBS for a 50% slurry) and incubate for 16–18 h at 4°C with gentle rotation at 10–15 rpm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step assumes that 5–10 μg of antibody is contained in 1 mL of hybridoma culture supernatant. Increase the volume of culture supernatant if the antibody concentration is less than 5–10 μg/mL.</div>
<ol start="41">
<li>Move the beads to 500 μL microtubes and wash three times in 500 μL of PBS-EDTA pH 7.0. Centrifuge at 500 × <i>g</i> for 1 min. Save the pellet and resuspend in 70–80 μL of PBS-EDTA pH 7.0. The total volume will be ∼100 μL.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Maleimide-conjugated dyes, such as DyLight488-maleimide and DyLight550-maleimide, selectively react with cysteine residues at pH 7.0 (<a href="#bib11">Hermanson, 2013</a>). EDTA is supplemented to prevent metal-catalyzed oxidation of the sulfhydryl group of cysteine residues (<a href="#bib42">Wallace, 2002</a>). Under these conditions, the dyes are selectively conjugated to free cysteine residues in mouse IgG (<a href="#bib14">Huh et al., 2013</a>) although labeling of some amine groups may occur.</div>
<ol start="42">
<li>Dissolve 50 μg of DyLight488-maleimide or DyLight550-maleimide (Thermo Fischer Scientific) in 5 μL of DMSO to obtain 10 μg/μL dye solution. Mix well using a vortex machine for 30–60 s.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> DyLight488-maleimide or DyLight550-maleimide (Thermo Fischer Scientific) are currently supplied as 1 mg dry powder. To prepare aliquots, dissolve the dyes in 300 μL methanol, dispense the solution by 15 μL in Eppendorf tubes and evaporate the methanol under reduced pressure. Close the cap tightly and store in the dark at −20°C with desiccant.</div>
<ol start="43">
<li>Add 5 μL of dye in DMSO to the bead suspension. Shield from light and incubate the beads for 2 h at 20°C–25°C gently mixing using a rotator at 10–15 rpm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Maleimide dye reacts with Protein A and antibodies on beads. The amount of Protein A is ∼60 μg for each 20 μL of beads as indicated on the product sheet from Cytiva, which is 1.42 nmol using a molecular weight of 42 kDa for Protein A (<a href="#bib1">Bjork et al., 1972</a>). Antibodies should weigh 10–20 μg, which is 0.066–0.13 nmol using an IgG molecular weight of 150 kDa (<a href="#bib15">Janeway et al., 2001</a>). The molar quantities of dyes are 56 and 50 nmol for DyLight488-maleimide (888 Da) and DyLight550-maleimide (1,000 Da), respectively (product sheet, Thermo Fischer Scientific). The molar ratio between protein and dye is adjusted to be ∼1:30 although the exact ratio is hard to estimate due to the small amount of antibody compared with that of Protein A on the beads. Most antibodies are successfully labeled at a lower molar ratio of protein to dye, for example, at 1:10–1:15 in order to conserve expensive dyes.</div>
<div class="note">
<span class="note-title">Note:</span> NHS-ester dyes, such as DyLight488-NHS-ester or DyLight550-NHS-ester (Thermo Fischer Scientific), are available to label antibodies via lysine residues. Labeling with these dyes requires careful optimization of the amount of dye. An excess amount of dye increases the labeling ratio of a Fab probe and may decrease their specificity of interaction with the epitope.</div>
<ol start="44">
<li>Remove the unreacted dye with five washes of the beads in 500 μL of PBS.</li>
<li>Remove the supernatant. Add 20 μL of 0.01 mg/mL of Papain solution and incubate at 37°C in a water bath for 1 h. Shield from light and tap the tubes every 15 min to suspend the beads.</li>
<li>Centrifuge the beads to recover supernatant containing Fab probes. Add 2 μL of 0.1 mg/mL Leupeptin dissolved in PBS, which will inhibit the proteolytic activity of papain. Aliquot the Fab probe solution and store at −80°C. See <a href="#fig4">Figure 4</a> and <a href="#mmc5">Methods video S5</a> for fast exchange of Fab probes on epitopes and super-resolution images acquired using the Fab probes.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our super-resolution microscopy, we used Fab probes diluted at 1:200 to 1:2000. Although the amount of Papain in the working solution is low (5–50 ng/mL), Leupeptin is a competitive inhibitor of serine proteinases including Papain and not an irreversible inhibitor of Papain. Thus, we supplemented an additional 1 μg/mL of Leupeptin to the working solutions of Fab probes, for example, when we perform multiplex super-resolution microscopy (<a href="#bib26">Miyoshi et al., 2021</a>).</div>
<figure id="mmc5"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Mmc5.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S5. Fast exchange of fast-dissociating Fab probes, related to step 46</div>
<p>Time-lapse movie corresponding to data in <a href="#fig4">Figure 4</a>B. FLAG-2, S-1 and V5-3 Fab probes were applied to fixed XTC cells expressing FLAG-actin, S-actin and V5-actin, respectively. Images were acquired every 200 ms using 1 nM of Fab probes. Color key for circles is the same as in <a href="#mmc1">Methods video S1</a>. Bar, 2 μm</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We screened 1,000–2,000 hybridoma clones for three epitope tags, FLAG-tag, S-tag and V5-tag, and for two F-actin crosslinking proteins, mPLS1 (mouse plastin 1) and mESPN1 (mouse espin isoform 1) (<a href="#bib26">Miyoshi et al., 2021</a>). The pairs of immunogens and EGFP-antigens are summarized in <a href="#tbl1">Table 1</a>. The anti-epitope tag antibodies were screened using EGFP-fused peptides, FLAG-EGFP, S-EGFP and V5-EGFP, respectively. Anti-mPLS1 antibodies were screened using EGFP-mPLS1. However, EGFP-fused full-length mESPN1 was not used to screen anti-mESPN1 antibodies due to its non-specific binding to the Protein A/G-coating on the glass surface. We minimized the non-specific binding of mESPN1 by subdividing mESPN1 into N-terminal and C-terminal fragments (see <a href="#limitations">limitations</a> for additional details). Representative time-lapse images are shown for the anti-FLAG tag monoclonal antibodies, which were acquired during the screening of candidates, subsequent determination of <i>k</i><sub>off</sub> values and recognition of FLAG-actin using Fab probes synthesized from the identified antibodies (<a href="#fig3">Figures 3</a> and <a href="#fig4">4</a>).</p>
<table id="tbl1">
<caption>Table 1. Combinations of immunizing antigens and EGFP-antigens</caption>
<thead>
<tr>
<th> </th>
<th>Immunizing antigen</th>
<th>EGFP-antigen</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">Epitope tags</td>
<td>FLAG-tag (DYKDDDDK)</td>
<td>FLAG-EGFP</td>
</tr>
<tr>
<td>S-tag (KETAAAKFERQHMDS)</td>
<td>S-EGFP</td>
</tr>
<tr>
<td>V5-tag (GKPIPNPLLGLDST)</td>
<td>V5-EGFP</td>
</tr>
<tr>
<td rowspan="2">Proteins</td>
<td>Mouse plastin1 (mPLS1)</td>
<td>EGFP-mPLS1</td>
</tr>
<tr>
<td>Mouse espin1 (mESPN1)</td>
<td>
<ul>
<li>EGFP-mESPN1 fragments
<ul>
<li>N-terminus 534 residues<sup><a href="#tblfn1">a</a></sup>
</li>
<li>C-terminus 215 residues<sup><a href="#tblfn1">a</a></sup>
</li>
</ul>
</li>
</ul>
</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Among the 871 residues of full-length mESPN1 (Genbank: NP_997570.1), the residues from 535 to 656 were omitted to suppress non-specific binding to the Protein A/G-coated glass surface of assay plates.</p>
<p>As summarized in <a href="#tbl2">Table 2</a>, the results show a difference in outcome between the anti-epitope tag antibodies and the antibodies developed against recombinant proteins. The screens of anti-epitope tag antibodies were straightforward. Eleven antibodies were identified by the screening assay. Six of the eleven antibodies were fast-dissociating from epitopes, and three of the six fast-dissociating antibodies generated Fab probes that were sufficiently specific to recognize epitope-tagged actin expressed in <i>Xenopus</i> XTC cells. These three antibodies are considered to be FDSAs that we pursued in our study. In contrast, the screens of anti-mPLS1 and anti-mESPN1 to identify FDSAs were more challenging. First, a total 65 anti-mPLS1 antibodies and 111 anti-mESPN1 monoclonal antibodies were identified, which is a 10-fold larger frequency than outcome from screening for anti-epitope tag antibodies. This probably reflects the higher immunogenicity of a recombinant protein compared to a short peptide (<a href="#bib37">Skwarczynski and Toth, 2016</a>). Second, only one of the 111 anti-mESPN1 antibodies showed fast-dissociation kinetics from the epitopes suggesting that fast-dissociating antibodies are rarely available for some antigens. Finally, only four anti-mPLS1 and five anti-mESPN1 antibodies, even including slow-dissociating antibodies, generated specific Fab probes (i.e., those recognizing epitopes expressed in XTC cells). We speculate that many anti-mPLS1 and anti-mESPN1 Fab probes did not recognize their epitopes because (1) access to the epitopes is limited due to the three-dimensional structure of the target protein, (2) the epitope is covered by a macromolecular binding-partner, or (3) the Fab probe is fluorescently labeled at or near the epitope recognition site, which interfered with the interaction between the Fab probe and its epitope. We obtained one anti-mPLS1 FDSA to generate a fast-dissociating, specific Fab probe. Anti-mESPN1 FDSAs were not available in our previous study although two anti-mESPN1 antibodies increased their dissociation rates after being converted to Fab probes (marked with an arrow in <a href="#tbl2">Table 2</a>).</p>
<table id="tbl2">
<caption>Table 2. Summary of our screening</caption>
<tbody>
<tr>
<td><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1191-Fx4.jpg" alt="Fx4.jpg"></td>
</tr>
</tbody>
</table>
<p class="table-legend"><sup>a</sup> Summation of 91 monoclonal antibodies recognizing the N-terminal fragment and 20 antibodies recognizing the C-terminal fragment.<br> <sup>b</sup> Antibodies that showed an exchange of bound EGFP-antigen in the screening assay, which corresponds to half-lives less than a few seconds.<br> <sup>c</sup> Fab probes that recognized epitopes ectopically expressed in <i>Xenopus</i> XTC cells.<br> <sup>d</sup> Two antibodies increased their dissociation rates after being converted into Fab probes and showed half-lives from epitopes of 4.4 and 4.5 s. The fast-dissociating antibody in the cell above did not produce a specific Fab probe.</p>
<p>We were able to obtain fast-dissociating, highly specific Fab probes with half-lives of less than a few seconds for all of three epitope tags and the two F-actin binding proteins (<a href="#bib26">Miyoshi et al., 2021</a>). These Fab probes were sufficiently specific to be useful as monovalent, single-molecule imaging probes for multiplex super-resolution microscopy. An anti-FLAG tag Fab probe and an anti-mPLS1 Fab probe dissociated from their epitopes at <i>k</i><sub>off</sub> = 0.51 s<sup>−1</sup> and 0.39 s<sup>−1</sup> (half-life = 1.4 s and 1.8 s), respectively. These results indicate that fast-dissociation is not such a rare or incompatible property of highly specific antibodies. We hope that our protocol will help researchers who require probes that have readily reversible binding properties. Fast-dissociating antibodies and their fragments may complement the benefits of conventional custom and commercially available antibodies.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Some EGFP-antigens are difficult to use in our screening assay. For example, EGFP-tagged full-length mESPN1 protein showed non-specific binding to the antibody-free glass surface and could not be used as a screening antigen. To solve this problem, we subdivided full-length mESPN1 (871 residues; Genbank: NP_997570.1) into a N-terminal protein (534 residues) and a C-terminal protein (215 residues) and omitted residues from 535 to 656 to minimize non-specific binding. Even with this approach, the EGFP-fused N-terminus was only usable at 5 nM while other EGFP-tagged antigens could be used at 20–30 nM. We also found that the N-terminal 334 residues consisting of Ankyrin repeat show nonspecific binding at 1 nM due to aggregation (<a href="#bib26">Miyoshi et al., 2021</a>). For antigens showing non-specific binding and/or aggregation in our assay, one possible solution is to synthesize peptides from the entire sequence of the antigen and screen antibodies using EGFP-fused peptides. However, this approach requires multiple scans of antibodies to cover as large a proportion of the antigen as possible but will potentially overlooks antibodies that recognize epitopes not included in the set of peptides used for screening.</p>
<p>We also noticed that some antigens may not be suitable for identifying fast-dissociating antibodies. When we screened anti-mESPN1 antibodies, only one antibody showed an exchange of bound EGFP-antigen with a dissociation half-life of less than a few seconds (<a href="#bib26">Miyoshi et al., 2021</a>). Considering that two anti-mESPN1 antibodies increased their dissociation from epitopes after being converted to Fab probes (marked with daggers in <a href="#tbl2">Table 2</a>), labeling with fluorophores is a “worth-a-try” technique to obtain fast-dissociating, specific Fab probes. Different approaches, such as an amino acid substitution within the antigen recognition site of an existing antibody, may also be useful to produce FDSAs (<a href="#bib6">Fukunaga et al., 2018</a>; <a href="#bib13">Hugo et al., 2003</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>EGFP-antigen shows non-specific binding to Protein A/G-coated glass surface (step 4, <a href="#before-you-begin">before you begin</a>).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Some EGFP-antigens may show non-specific binding to the control, antibody-free glass surface that has been covered with Protein A/G. One solution to control non-specific binding is to lower the concentration of EGFP-antigen. However, identification of low-affinity antibodies, which may include some fast-dissociating antibodies, can be difficult with a low concentration of EGFP-antigen as we simulated in our reported study (<a href="#bib26">Miyoshi et al., 2021</a>). Another solution, as mentioned above, is to divide the antigen into domains or fragments. Use of protein domains is recommended to identify antibodies recognizing a native fold of a protein.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Only a limited number of antibodies recognize EGFP-antigen (step 28, <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Check the DNA sequences of plasmids used to synthesize immunogens and EGFP-antigens to exclude the possibility of human or PCR-associated errors in constructing the expression vectors. In addition, the method used to harvest B cells can directly affect the number of “positive” clones. We recommend using the mouse iliac lymph node method (<a href="#bib30">Sado et al., 2006</a>; <a href="#bib31">Sado and Okigaki, 1996</a>).</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Hybridoma clones lose antibody secreting ability (step 31, <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Crude hybridoma clones can lose their ability to secrete antibodies. Cryopreserve hybridoma clones as early as possible in their passage and definitely avoid long-term subculture. Perform limiting dilution to isolate monoclonal populations before cryopreservation of hybridomas. In addition, be sure to use new pipette tips for each clone to avoid cross contamination during the subculture.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bjork, I., Petersson, B.A., and Sjoquist, J. (1972). Some physiochemical properties of protein A from <i>Staphylococcus aureus</i>. Eur. J. Biochem. <i>29</i>, 579-584. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref1">View at publisher</a></p>
<p id="bib2">Canziani, G.A., Klakamp, S., and Myszka, D.G. (2004). Kinetic screening of antibodies from crude hybridoma samples using Biacore. Anal. Biochem. <i>325</i>, 301-307. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref2">View at publisher</a></p>
<p id="bib3">Chudakov, D.M., Matz, M.V., Lukyanov, S., and Lukyanov, K.A. (2010). Fluorescent proteins and their applications in imaging living cells and tissues. Physiol. Rev. <i>90</i>, 1103-1163. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref3">View at publisher</a></p>
<p id="bib4">Cuatrecasas, P. and Parikh, I. (1972). Adsorbents for affinity chromatography. Use of N-hydroxysuccinimide esters of agarose. Biochemistry <i>11</i>, 2291-2299. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref4">View at publisher</a></p>
<p id="bib5">Freund, J., Casals, J., and Hosmer, E.P. (1937). Sensitization and antibody formation after injection of tubercle bacilli and paraffin oil. Exp. Biol. Med. <i>37</i>, 509-513. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref5">View at publisher</a></p>
<p id="bib6">Fukunaga, A., Maeta, S., Reema, B., Nakakido, M., and Tsumoto, K. (2018). Improvement of antibody affinity by introduction of basic amino acid residues into the framework region. Biochem. Biophys. Rep. <i>15</i>, 81-85. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref6">View at publisher</a></p>
<p id="bib7">Gibbs, D.R., Kaur, A., Megalathan, A., Sapkota, K., and Dhakal, S. (2018). Build your own microscope: step-by-step guide for building a prism-based TIRF microscope. Methods Protoc. <i>1</i>, 40. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref7">View at publisher</a></p>
<p id="bib8">Griffin, B.A., Adams, S.R., and Tsien, R.Y. (1998). Specific covalent labeling of recombinant protein molecules inside live cells. Science <i>281</i>, 269-272. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref8">View at publisher</a></p>
<p id="bib9">Hayashi-Takanaka, Y., Yamagata, K., Wakayama, T., Stasevich, T.J., Kainuma, T., Tsurimoto, T., Tachibana, M., Shinkai, Y., Kurumizaka, H., Nozaki, N., et al. (2011). Tracking epigenetic histone modifications in single cells using Fab-based live endogenous modification labeling. Nucl. Acids Res. <i>39</i>, 6475-6488. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref9">View at publisher</a></p>
<p id="bib10">Helmerhorst, E., Chandler, D.J., Nussio, M., and Mamotte, C.D. (2012). Real-time and label-free bio-sensing of molecular interactions by surface plasmon resonance: a laboratory medicine perspective. Clin. Biochem. Rev. <i>33</i>, 161-173. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref10">View at publisher</a></p>
<p id="bib11">Hermanson, G.T. (2013). Chapter 3 - the reactions of bioconjugation. In Bioconjugate Techniques, Third Edition, G.T. Hermanson, ed. (Academic Press), pp. 229-258. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref11">View at publisher</a></p>
<p id="bib12">Hnasko, R.M. and Stanker, L.H. (2015). Hybridoma technology. In ELISA: Methods and Protocols, R. Hnasko, ed. (Springer New York), pp. 15-28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref12">View at publisher</a></p>
<p id="bib13">Hugo, N., Weidenhaupt, M., Beukes, M., Xu, B., Janson, J.C., Vernet, T., and Altschuh, D. (2003). VL position 34 is a key determinant for the engineering of stable antibodies with fast dissociation rates. Protein Eng. <i>16</i>, 381-386. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref13">View at publisher</a></p>
<p id="bib14">Huh, J.H., White, A.J., Brych, S.R., Franey, H., and Matsumura, M. (2013). The identification of free cysteine residues within antibodies and a potential role for free cysteine residues in covalent aggregation because of agitation stress. J. Pharm. Sci. <i>102</i>, 1701-1711. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref14">View at publisher</a></p>
<p id="bib15">Janeway, C.J., Travers, P., Walport, M., and MJ, S. (2001). Immunobiology, 5th Edition: The Immune System in Health and Disease (Garland Publishing). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref15">View at publisher</a></p>
<p id="bib16">Jungmann, R., Avendano, M.S., Woehrstein, J.B., Dai, M., Shih, W.M., and Yin, P. (2014). Multiplexed 3D cellular super-resolution imaging with DNA-PAINT and exchange-PAINT. Nat. Methods <i>11</i>, 313-318. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref16">View at publisher</a></p>
<p id="bib17">Kamat, V., Rafique, A., Huang, T., Olsen, O., and Olson, W. (2020). The impact of different human IgG capture molecules on the kinetics analysis of antibody-antigen interaction. Anal. Biochem. <i>593</i>, 113580. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref17">View at publisher</a></p>
<p id="bib18">Kiuchi, T., Higuchi, M., Takamura, A., Maruoka, M., and Watanabe, N. (2015). Multitarget super-resolution microscopy with high-density labeling by exchangeable probes. Nat. Methods <i>12</i>, 743-746. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref18">View at publisher</a></p>
<p id="bib19">Krey, J.F., Krystofiak, E.S., Dumont, R.A., Vijayakumar, S., Choi, D., Rivero, F., Kachar, B., Jones, S.M., and Barr-Gillespie, P.G. (2016). Plastin 1 widens stereocilia by transforming actin filament packing from hexagonal to liquid. J. Cell. Biol. <i>215</i>, 467-482. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref19">View at publisher</a></p>
<p id="bib20">Kumaraswamy, S. and Tobias, R. (2015). Label-free kinetic analysis of an antibody–antigen interaction using biolayer interferometry. In Protein-Protein Interactions: Methods and Applications, C.L. Meyerkord and H. Fu, eds. (Springer New York), pp. 165-182. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref20">View at publisher</a></p>
<p id="bib21">Lad, L., Clancy, S., Kovalenko, M., Liu, C., Hui, T., Smith, V., and Pagratis, N. (2015). High-throughput kinetic screening of hybridomas to identify high-affinity antibodies using bio-layer interferometry. J. Biomol. Screen <i>20</i>, 498-507. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref21">View at publisher</a></p>
<p id="bib22">Liss, V., Barlag, B., Nietschke, M., and Hensel, M. (2015). Self-labelling enzymes as universal tags for fluorescence microscopy, super-resolution microscopy and electron microscopy. Sci. Rep. <i>5</i>, 17740. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref22">View at publisher</a></p>
<p id="bib23">Lyubimova, A., Itzkovitz, S., Junker, J.P., Fan, Z.P., Wu, X., and van Oudenaarden, A. (2013). Single-molecule mRNA detection and counting in mammalian tissue. Nat. Protoc. <i>8</i>, 1743-1758. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref23">View at publisher</a></p>
<p id="bib24">Mattheyses, A.L., Simon, S.M., and Rappoport, J.Z. (2010). Imaging with total internal reflection fluorescence microscopy for the cell biologist. J. Cell. Sci. <i>123</i>, 3621-3628. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref24">View at publisher</a></p>
<p id="bib25">Metlagel, Z., Krey, J.F., Song, J., Swift, M.F., Tivol, W.J., Dumont, R.A., Thai, J., Chang, A., Seifikar, H., Volkmann, N., et al. (2019). Electron cryo-tomography of vestibular hair-cell stereocilia. J. Struct. Biol. <i>206</i>, 149-155. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref25">View at publisher</a></p>
<p id="bib26">Miyoshi, T., Zhang, Q., Miyake, T., Watanabe, S., Ohnishi, H., Chen, J., Vishwasrao, H.D., Chakraborty, O., Belyantseva, I.A., Perrin, B.J., et al. (2021). Semi-automated single-molecule microscopy screening of fast-dissociating specific antibodies directly from hybridoma cultures. Cell. Rep. <i>34</i>, 108708. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref26">View at publisher</a></p>
<p id="bib27">Myszka, D.G. (1999). Improving biosensor analysis. J. Mol. Recognit. <i>12</i>, 279-284. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref27">View at publisher</a></p>
<p id="bib28">Nikolovska-Coleska, Z. (2015). Studying protein-protein interactions using surface plasmon resonance. In Protein-Protein Interactions: Methods and Applications, C.L. Meyerkord and H. Fu, eds. (Springer New York), pp. 109-138. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref28">View at publisher</a></p>
<p id="bib29">Peciak, K., Laurine, E., Tommasi, R., Choi, J.W., and Brocchini, S. (2019). Site-selective protein conjugation at histidine. Chem. Sci. <i>10</i>, 427-439. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref29">View at publisher</a></p>
<p id="bib30">Sado, Y., Inoue, S., Tomono, Y., and Omori, H. (2006). Lymphocytes from enlarged iliac lymph nodes as fusion partners for the production of monoclonal antibodies after a single tail base immunization attempt. Acta Histochem. Cytochem. <i>39</i>, 89-94. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref30">View at publisher</a></p>
<p id="bib31">Sado, Y. and Okigaki, T. (1996). A novel method for production of monoclonal antibodies. Evaluation and expectation of the rat lymph node method in cell and molecular biology. Cell. Biol. Int. <i>20</i>, 7-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref31">View at publisher</a></p>
<p id="bib32">Safsten, P., Klakamp, S.L., Drake, A.W., Karlsson, R., and Myszka, D.G. (2006). Screening antibody-antigen interactions in parallel using Biacore A100. Anal. Biochem. <i>353</i>, 181-190. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref32">View at publisher</a></p>
<p id="bib33">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nat. Methods <i>9</i>, 671-675. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref33">View at publisher</a></p>
<p id="bib34">Schuchner, S., Behm, C., Mudrak, I., and Ogris, E. (2020). The Myc tag monoclonal antibody 9E10 displays highly variable epitope recognition dependent on neighboring sequence context. Sci. Signal <i>13</i>. . <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref34">View at publisher</a></p>
<p id="bib35">Schueder, F., Strauss, M.T., Hoerl, D., Schnitzbauer, J., Schlichthaerle, T., Strauss, S., Yin, P., Harz, H., Leonhardt, H., and Jungmann, R. (2017). Universal super-resolution multiplexing by DNA exchange. Angew. Chem. Int. Ed. Engl. <i>56</i>, 4052-4055. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref35">View at publisher</a></p>
<p id="bib36">Sekerkova, G., Richter, C.P., and Bartles, J.R. (2011). Roles of the espin actin-bundling proteins in the morphogenesis and stabilization of hair cell stereocilia revealed in CBA/CaJ congenic jerker mice. PLoS Genet. <i>7</i>, e1002032. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref36">View at publisher</a></p>
<p id="bib37">Skwarczynski, M. and Toth, I. (2016). Peptide-based synthetic vaccines. Chem. Sci. <i>7</i>, 842-854. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref37">View at publisher</a></p>
<p id="bib38">Son, K.K., Tkach, D., and Rosenblatt, J. (2001). Delipidated serum abolishes the inhibitory effect of serum on in vitro liposome-mediated transfection. Biochim. Biophys. Acta (Bba) - Biomembranes <i>1511</i>, 201-205. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref38">View at publisher</a></p>
<p id="bib39">Song, H.N., Kim, D.H., Park, S.G., Lee, M.K., Paek, S.H., and Woo, E.J. (2015). Purification and characterization of Fab fragments with rapid reaction kinetics against myoglobin. Biosci. Biotechnol. Biochem. <i>79</i>, 718-724. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref39">View at publisher</a></p>
<p id="bib40">Stils, H.F. (2005). Adjuvants and antibody production: dispelling the myths associated with freund's complete and other adjuvants. ILAR J. <i>46</i>, 280-293. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref40">View at publisher</a></p>
<p id="bib41">Vauquelin, G. and Charlton, S.J. (2013). Exploring avidity: understanding the potential gains in functional affinity and target residence time of bivalent and heterobivalent ligands. Br. J. Pharmacol. <i>168</i>, 1771-1785. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref41">View at publisher</a></p>
<p id="bib42">Wallace, T.J. (2002). Reactions of thiols with metals. I. Low-temperature oxidation by metal oxides1. J. Org. Chem. <i>31</i>, 1217-1221. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref42">View at publisher</a></p>
<p id="bib43">Ylera, F., Harth, S., Waldherr, D., Frisch, C., and Knappik, A. (2013). Off-rate screening for selection of high-affinity anti-drug antibodies. Anal Biochem. <i>441</i>, 208-213. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00673-0/sref43">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Takushi Miyoshi (<a href="mailto:takushi.miyoshi@nih.gov">takushi.miyoshi@nih.gov</a> or <a href="mailto:miyoshi.takushi.76e@kyoto-u.jp">miyoshi.takushi.76e@kyoto-u.jp</a>).</p>
<h4>Materials availability</h4>
<p>Materials (hybridomas, antibodies and plasmids) generated in our previous study (<a href="#bib26">Miyoshi et al., 2021</a>) are available upon request from the Lead Contact, Takushi Miyoshi (NIDCD, National Institutes of Health), or from the last author, Naoki Watanabe (Kyoto University) with a completed Material Transfer Agreement (MTA).</p>
<h4>Data and code availability</h4>
<p>Python scripts used in this protocol are available from a GitHub repository (<a href="https://github.com/takushim/tanitracer">https://github.com/takushim/tanitracer</a>).</p>
<h3>Acknowledgments</h3>
<p>We thank Dr. Akira Kakizuka, Mr. Kimiaki Tsukagami, and Mr. Seiji Suzuki (Kyoto University) for advice about this collaboration. We also thank Drs. Rafal Olszewski and Dennis Winkler for valuable comments; Ms. Erina He for her beautiful diagrams; Mr. Qianli Zhang, Dr. Shin Watanabe, and Dr. Makio Higuchi for technical support; Dr. Daisuke Taniguchi for python scripts providing the core algorithms for Gaussian fitting with subpixel correction; and Dr. Hiroe Ohnishi for instructing us in hybridoma culture. This work was supported by <a href="https://doi.org/10.13039/501100003382">CREST</a> grant number JPMJCR15G5 to N.W., <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI grant number JP19H01020 to N.W., and <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI grant numbers 16J09300 and 18K16884 to T.M. This research was also supported (in part) by the Intramural Research Programs of the <a href="https://doi.org/10.13039/100000002">NIH</a>, <a href="https://doi.org/10.13039/100000055">NIDCD</a>, DC000039 to T.B.F. and a JSPS Overseas Research Fellowships to T.M.</p>
<h3>Author contributions</h3>
<p>T.M. developed the assay and strategy to screen monoclonals and to produce Fab probes in N.W.’s laboratory. T.M. prepared a first draft of the manuscript and figures under the mentorship of T.B.F. N.W. evaluated the screening assay and strategy. T.M. and T.B.F. crafted this manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>