<article>
<section>
<h1 id="header">Protocol for rapid manipulation of mitochondrial morphology in living cells using inducible counter mitochondrial morphology (iCMM)</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Takafumi Miyamoto,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Song-Iee Han,<sup><a href="#aff1">1</a></sup> and Hitoshi Shimano<sup><a href="#aff1">1</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Internal Medicine (Endocrinology and Metabolism), Faculty of Medicine, University of Tsukuba, Tsukuba, Ibaraki 305-8575, Japan</p>
<p id="aff2"><sup>2</sup>Transborder Medical Research Center, University of Tsukuba, Tsukuba, Ibaraki 305-8577, Japan</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:takmi565@md.tsukuba.ac.jp">takmi565@md.tsukuba.ac.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hshimano@md.tsukuba.ac.jp">hshimano@md.tsukuba.ac.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100721">10.1016/j.xpro.2021.100721</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Disruption of mitochondrial morphology occurs during various diseases, but the biological significance is not entirely clear. Here, we describe a detailed step-by-step protocol for a chemically inducible dimerization system-based synthetic protein device, termed inducible counter mitochondrial morphology. This system allows artificial manipulation of mitochondrial morphology on a timescale of minutes in living mammalian cells. We also describe an AI-assisted imaging processing approach.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Miyamoto et al., 2021</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for rapid manipulation of mitochondrial morphology by iCMM</li>
<li>iCMM is a genetically encoded chemically inducible dimerization system</li>
<li>AI-assisted image processing for mitochondrial morphology</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>In this paper, we describe a step-by-step protocol for manipulating mitochondrial morphology in living cells at any given time point within a few minutes using iCMM, a Boolean YES logic gate-based synthetic protein device based on a Chemically Inducible Dimerization (CID) system. When this protocol is followed, mitochondrial morphological changes are observed within 1–5 min of actuating the iCMM. It employs the CID method with rapamycin as a chemical dimerizer; however, the fundamental experimental procedure is the same when other CID methods are used.</p>
<p>The iCMM consists of a functional iCMM effector (FiCE), which can induce changes in mitochondrial morphology, and a mitochondrial outer membrane-specific anchor protein that tethers the effector to the mitochondria in the presence of a chemical dimerizer (<a href="#fig1">Figure 1</a>A). In contrast, an effector that does not cause mitochondrial morphological changes is termed a control iCMM effector (CiCE), and a synthetic protein device consisting of a CiCE and a mitochondria-specific anchor is referred to as a negative iCMM (NiCMM; <a href="#fig1">Figure 1</a>A).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic diagram of iCMM and NiCMM</div>
<p>Reprinted with permission from <a href="#bib3">Miyamoto et al., 2021</a>.</p>
<p>(A) Schematic diagram of the iCMM and NiCMM system.</p>
<p>(B) Structural features of functional iCMM effectors (FiCEs), control iCMM effector (CiCE), and anchor.</p>
</figcaption>
</figure>
<p>To date, five FiCEs (YF, YF-Cav1s, mChF-Cav1s, mActZ-FY, and mActZ-FmCh), one CiCE (mYF), and one anchor (Tom20-CR) (<a href="#bib3">Miyamoto et al., 2021</a>) have been developed (<a href="#fig1">Figure 1</a>B). The effector (FiCE/CiCE) and anchor contain interaction domains (in this protocol, FKBP and FRB; <a href="#fig1">Figure 1</a>B) required to induce dimer formation through chemical dimerizers (<a href="#bib1">Derose et al., 2013</a>). It is possible to create various effectors and anchors with different characteristics by reorganizing protein structures according to the intended purpose.</p>
<p>The main characteristic of iCMM is that it can induce morphological changes in mitochondria on a minute timescale at any time point. Thus, changes that occur in the cell immediately after induction of mitochondrial morphological changes can be observed.</p>
<h3 id="sec1.1">Plasmid preparation for transfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol>
<li>Expression plasmids encoding effectors and anchor should be prepared in advance. These plasmids are available from Addgene (Watertown, MA, USA; <a href="https://www.addgene.org/">https://www.addgene.org/</a>).</li>
<li>Day 1: transform plasmids into competent cells as follows:
<ol type="a">
<li>Thaw ECOS Competent <i>E. coli</i> DH5α cells (NIPPON GENE, Tokyo, Japan) on ice.</li>
<li>Add 10 ng of plasmid to 20 μL competent cells in a 1.5-mL microcentrifuge tube and mix gently.</li>
<li>Incubate the mixture on ice for 20 min.</li>
<li>Heat shock at 42°C for 45 s.</li>
<li>Put the mixture on ice for 3 min.</li>
<li>Add 50 μL LB liquid medium without antibiotics to the mixture.</li>
<li>Incubate the mixture at 37°C for 60 min.</li>
<li>Warm LB agar plates containing 50 μg/mL kanamycin at 37°C for 60 min.</li>
<li>Spread all of the mixture onto the LB agar plates containing 50 μg/mL kanamycin.</li>
<li>Incubate the plates at 37°C overnight (16–18 h).</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other competent cells, such as the <i>E. coli</i> DH5α Competent Cells (TaKaRa, 9057), can be used instead of ECOS Competent <i>E. coli</i> DH5α cells.</div>
<ol start="3">
<li>Day 2: select a single clone and culture it in 100 mL LB medium containing 50 μg/mL kanamycin in a 200-mL flask.</li>
<li>Shake at 37°C and 160 rpm overnight (16–18 h).</li>
<li>Day 3: transfer the bacterial culture to two 50-mL conical tubes and centrifuge at 4,000 × <i>g</i> and 4°C for 10 min.</li>
<li>Remove the supernatant.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Bacterial pellets can be stored at −80°C until purification.</div>
<ol start="7">
<li>Purify the plasmid for transfection using the PureLink™ HiPure Plasmid Midiprep Kit (Thermo Fisher Scientific, Waltham, MA, USA) according to the manufacturer’s instructions (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fpurelink_hipure_plasmid_dna_purification_man.pdf&amp;title=VXNlciBHdWlkZTogUHVyZUxpbmsgSGlQdXJlIFBsYXNtaWQgRE5BIFB1cmlmaWNh">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fpurelink_hipure_plasmid_dna_purification_man.pdf&amp;title=VXNlciBHdWlkZTogUHVyZUxpbmsgSGlQdXJlIFBsYXNtaWQgRE5BIFB1cmlmaWNhdGlvbiBLaXRz</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Plasmid DNA at a A260/A280 ratio &lt; 1.8 should be used at a concentration of 1 mg/mL.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other plasmid purification kits, such as the QIAGEN Plasmid Midi Kit (QIAGEN, 12143) or NucleoBond™ Xtra Midi (TaKaRa, 740410.10), may be used to purify plasmids according to the manufacturer’s instructions (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits/?catno=12143">https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/plasmid-dna/qiagen-plasmid-kits/?catno=12143</a> or <a href="https://www.mn-net.com/media/pdf/ed/82/0f/Instruction-NucleoBond-Xtra.pdf">https://www.mn-net.com/media/pdf/ed/82/0f/Instruction-NucleoBond-Xtra.pdf</a>).</div>
<h3 id="sec1.2">Maintenance of HeLa cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Human cervical adenocarcinoma HeLa cells were purchased from the American Type Culture Collection (Manassas, VA, USA) and were cultured in Dulbecco’s modified Eagle’s medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 1% ZellShield (Minerva Biolabs, Berlin, Germany) at 37°C in 5% CO<sub>2</sub>.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> ZellShield (Minerva Biolabs) can be substituted for penicillin-streptomycin (final conc. 100 units penicillin and 0.1 mg streptomycin/mL).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> iCMM can be used with cells other than HeLa cells. We confirmed that iCMM can induce mitochondrial morphological changes in Hep 3 B human hepatoma and U-2 OS human osteogenic sarcoma cells (<a href="#bib3">Miyamoto et al., 2021</a>).</div>
<ol start="8">
<li>Day 1: thaw cells stored in a −80°C freezer or in liquid nitrogen were transfered to a 15-mL conical tube containing 10 mL prewarmed culture medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> CELLBANKER (Nippon Zenyaku Kogyo, Tokyo, Japan), a ready-to-use cell cryopreservation medium, may be used for cell stocks.</div>
<ol start="9">
<li>Centrifuge the tube with swing buckets at 161 × <i>g</i> and at room temperature (15°C–25°C) for 2 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Sorvall ST 8 (Thermo Fisher Scientific) may be used for centrifugation.</div>
<ol start="10">
<li>Remove the supernatant and resuspended precipitated cells in 10 mL prewarmed culture medium.</li>
<li>Plate the cell suspension in a 10-cm cell culture dish.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After plating cells in the culture dish, use a microscope to ensure that the cells are not dead. Furthermore, 24 h after plating verify that confluency is &lt; 50%; if not, then proceed to Step 13 (Day 3).</div>
<ol start="12">
<li>Incubate cells at 37°C and 5% CO<sub>2</sub> for 48 h.</li>
<li>Day 3: remove the culture medium.</li>
<li>Add 4 mL prewarmed phosphate-buffered saline (PBS) at the periphery of the dish and slowly tilt to gently wash the cells.</li>
<li>Remove PBS.</li>
<li>Add 1 mL 0.05% trypsin-EDTA solution to the dish.</li>
<li>Incubate the cells at 37°C for 4 min in a 5% CO<sub>2</sub> incubator.</li>
<li>Gently tap the dish to completely detach the cells.</li>
<li>Add 7 mL prewarmed culture medium and suspend the cells by gentle pipetting.</li>
<li>Transfer the cells to a sterilized 15-mL conical tube.</li>
<li>Centrifuge the tube with swing buckets at 161 × <i>g</i> and at room temperature (15°C–25°C) for 2 min.</li>
<li>Add 7 mL prewarmed culture medium and suspend the cells by gentle pipetting.</li>
<li>To count cells, mix 40 μL of the cell suspension with 40 μL trypan blue in a 1.5-mL microcentrifuge tube. Pipette up and down gently several times to mix.</li>
<li>Use an automated cell counter with 10 μL of the mixture.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A hemocytometer may be used for cell counts.</div>
<div class="note">
<span class="note-title">Note:</span> Verify, at least qualitatively, that cell viability, shape, and size are not atypical. If the cell counter used can measure quantitative information about viability, size, among others, then it is recommended to record such numerical information.</div>
<ol start="25">
<li>Plate 1 × 10<sup>6</sup> cells in a 10-cm dish and culture at 37°C under 5% CO<sub>2</sub>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this case (1 × 10<sup>6</sup> cells in a 10-cm dish), seeded HeLa cells need to be passaged after 48 h. To passage cells at 72-h intervals, plate 0.5 × 10<sup>6</sup> cells in a 10-cm dish. The number of cells to be plated should be changed according to the cells’ growth rate. Confluency of 80% is recommended as a standard for cell passaging. To prevent damage to the cells, we do not recommend passaging at 24-h intervals.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Before starting the experiment, infection of the cells with Mycoplasma should be excluded using PCR and Hoechst staining. To examine whether cells are infected with Mycoplasma by PCR, we recommend using ready-to-use kits such as e-MycoTM Mycoplasma PCR Detection Kit (iNtRON Biotechnology, Cat#25235), e-MycoTM VALiD Mycoplasma PCR Detection Kit (iNtRON Biotechnology, Cat#25239), TaKaRa PCR Mycoplasma Detection Set (Takara, Cat#6601), or LookOut® Mycoplasma PCR Detection Kit (Sigma-Aldrich, Cat#MP0035-1KT). For Hoechst staining-based Mycoplasma contamination test, please refer to the technical article at the following site: <a href="https://www.sigmaaldrich.com/JP/en/technical-documents/technical-article/microbiological-testing/mycoplasma-testing/testing-for-mycoplasma">https://www.sigmaaldrich.com/JP/en/technical-documents/technical-article/microbiological-testing/mycoplasma-testing/testing-for-mycoplasma</a>.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial strains</td>
</tr>
<tr>
<td>ECOS Competent <i>E. coli</i> DH5α</td>
<td>NIPPON GENE</td>
<td>Cat# 312-07031</td>
</tr>
<tr>
<td colspan="3">Chemicals</td>
</tr>
<tr>
<td>Dulbecco’s Modified Eagle’s Medium (DMEM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11965118</td>
</tr>
<tr>
<td>Phenol red-free DMEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 31053028</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10270-106</td>
</tr>
<tr>
<td>
<span class="small-caps">l</span>-glutamine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25030081</td>
</tr>
<tr>
<td>Phosphate-Buffered Saline (PBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 10010023</td>
</tr>
<tr>
<td>ZellShield</td>
<td>Minerva Biolabs GmbH</td>
<td>Cat# 13-0050</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Sigma-Aldrich</td>
<td>Cat# P4333</td>
</tr>
<tr>
<td>0.05% Trypsin-EDTA Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25300-054</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11058-021</td>
</tr>
<tr>
<td>FuGENE HD</td>
<td>Promega</td>
<td>Cat# E2311</td>
</tr>
<tr>
<td>MitoTracker Red CMXRos</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# M7513</td>
</tr>
<tr>
<td>Rapamycin</td>
<td>Calbiochem</td>
<td>Cat# 553211-1MGCN</td>
</tr>
<tr>
<td>Trypan Blue Stain</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15250-061</td>
</tr>
<tr>
<td>Hoechst 33342, Trihydrochloride, Trihydrate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# H1399</td>
</tr>
<tr>
<td>CELLBANKER</td>
<td>Nippon Zenyaku Kogyo</td>
<td>Cat# CB023</td>
</tr>
<tr>
<td>STAR Agar L-grade</td>
<td>Rikaken</td>
<td>Cat# RSU-AC01-500G</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Nacalai Tesque</td>
<td>Cat# 35640-95</td>
</tr>
<tr>
<td>Extract Yeast Dried</td>
<td>Nacalai Tesque</td>
<td>Cat# 15838-45</td>
</tr>
<tr>
<td>NaCl</td>
<td>Nacalai Tesque</td>
<td>Cat# 31320-05</td>
</tr>
<tr>
<td>Kanamycin Sulfate</td>
<td>FUJIFLIM</td>
<td>Cat# 117-00341</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PureLink™ HiPure Plasmid Midiprep Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# K210005</td>
</tr>
<tr>
<td>e-Myco<sup>TM</sup> Mycoplasma PCR Detection Kit (ver.2.0)</td>
<td>iNtRON Biotechnology</td>
<td>Cat# 25235</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>HeLa</td>
<td>ATCC</td>
<td>CCL-2</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>mYF</td>
<td><a href="#bib3">Miyamoto et al., 2021</a></td>
<td>Addgene Plasmid#171460</td>
</tr>
<tr>
<td>YF</td>
<td><a href="#bib3">Miyamoto et al., 2021</a></td>
<td>Addgene Plasmid#171459</td>
</tr>
<tr>
<td>YF-Cav1s</td>
<td><a href="#bib3">Miyamoto et al., 2021</a></td>
<td>Addgene Plasmid#171462</td>
</tr>
<tr>
<td>mActZ-YF</td>
<td><a href="#bib3">Miyamoto et al., 2021</a></td>
<td>Addgene Plasmid#171463</td>
</tr>
<tr>
<td>Tom20-CR</td>
<td><a href="#bib3">Miyamoto et al., 2021</a></td>
<td>Addgene Plasmid#171461</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>NIS-Elements AR 5.30</td>
<td>Nikon</td>
<td><a href="https://www.nsl.nikon.com/">https://www.nsl.nikon.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>3.5-cm poly-lysine-coated glass bottom dish</td>
<td>Matsunami</td>
<td>Cat# D1131H</td>
</tr>
<tr>
<td>10-cm cell culture dish</td>
<td>Corning</td>
<td>Cat# 430167</td>
</tr>
<tr>
<td>15-mL Conical Tube</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 239650</td>
</tr>
<tr>
<td>50-mL Conical Tube</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 339652</td>
</tr>
<tr>
<td>Sterilized petri dish</td>
<td>WiSM</td>
<td>Cat# WMS-500</td>
</tr>
<tr>
<td>1.5-mL microcentrifuge tube</td>
<td>BIO-BIK</td>
<td>Cat# CF-0150</td>
</tr>
<tr>
<td>5-mL polystyrene round-bottom tube</td>
<td>?</td>
<td>?</td>
</tr>
<tr>
<td>Toothpicks</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Water bath (thermo minder SD mini)</td>
<td>TAITEC</td>
<td>Cat# 0034942-000</td>
</tr>
<tr>
<td>CO<sub>2</sub> incubator (37°C, 5% CO<sub>2</sub>)</td>
<td>Panasonic</td>
<td>Cat# MCO-170AICUV-PJ</td>
</tr>
<tr>
<td>Luna<sup>TM</sup> automated cell counter</td>
<td>Shinkouseiki.Co.,Ltd.</td>
<td>Cat# L10001</td>
</tr>
<tr>
<td>NanoDrop<sup>TM</sup> (ND-1000 spectrophotometer)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# ND-1000</td>
</tr>
<tr>
<td>Eclipse Ti2-E microscope and Intensilight mercury-fiber illuminator</td>
<td>Nikon</td>
<td><a href="https://www.microscope.healthcare.nikon.com/products/inverted-microscopes/eclipse-ti2-series/eclipse-ti2-e">https://www.microscope.healthcare.nikon.com/products/inverted-microscopes/eclipse-ti2-series/eclipse-ti2-e</a></td>
</tr>
<tr>
<td>CFP-A-Basic-NTE filter</td>
<td>Semrock</td>
<td><a href="http://www.opto-line.co.jp/sem/detail_sets.html?pn=CFP-A-Basic">http://www.opto-line.co.jp/sem/detail_sets.html?pn=CFP-A-Basic</a></td>
</tr>
<tr>
<td>YFP-A-Basic-NTE filter</td>
<td>Semrock</td>
<td><a href="http://www.opto-line.co.jp/sem/detail_sets.html?pn=YFP-A-Basic">http://www.opto-line.co.jp/sem/detail_sets.html?pn=YFP-A-Basic</a></td>
</tr>
<tr>
<td>mCherry-B-NTE-ZERO filter</td>
<td>Semrock</td>
<td><a href="http://www.opto-line.co.jp/sem/detail_sets.html?pn=mCherry-C">http://www.opto-line.co.jp/sem/detail_sets.html?pn=mCherry-C</a></td>
</tr>
<tr>
<td>Plan Apochromat Lambda Series, 40× objective</td>
<td>Nikon</td>
<td><a href="https://www.microscope.healthcare.nikon.com/products/optics/cfi-plan-apochromat-lambda-series">https://www.microscope.healthcare.nikon.com/products/optics/cfi-plan-apochromat-lambda-series</a></td>
</tr>
<tr>
<td>Zyla 4.2 PLUS sCMOS camera</td>
<td>Oxford Instruments</td>
<td><a href="https://andor.oxinst.jp/products/scmos-camera-series/zyla-4-2-scmos">https://andor.oxinst.jp/products/scmos-camera-series/zyla-4-2-scmos</a></td>
</tr>
<tr>
<td>STX stage top incubator</td>
<td>Tokai Hit</td>
<td><a href="https://www.tokaihit.com/about-stage-top-incubator/">https://www.tokaihit.com/about-stage-top-incubator/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Plasmid preparation for transfection</h3>
<table id="undtbl1">
<caption>LB agar plates containing 50 μg/mL kanamycin</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>Dried yeast extract</td>
<td>0.5%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>STAR Agar L-grade</td>
<td>1.5%</td>
<td>1.5 g</td>
</tr>
<tr>
<td>MilliQ water</td>
<td>n/a</td>
<td>up to 100 mL</td>
</tr>
<tr>
<td>50 mg/mL kanamycin</td>
<td>50 μg/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Please refer to the addgene website (<a href="https://www.addgene.org/protocols/pouring-lb-agar-plates/">https://www.addgene.org/protocols/pouring-lb-agar-plates/</a>) for detailed preparation instructions.</div>
<div class="note">
<span class="note-title">Note:</span> The LB agar plates can be stored at 4°C away from light until use. They should be used within a month.</div>
<table id="undtbl2">
<caption>LB liquid medium containing 50 μg/mL kanamycin</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>Dried yeast extract</td>
<td>0.5%</td>
<td>0.5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>1%</td>
<td>1 g</td>
</tr>
<tr>
<td>MilliQ water</td>
<td>n/a</td>
<td>up to 100 mL</td>
</tr>
<tr>
<td>50 mg/mL kanamycin</td>
<td>50 μg/mL</td>
<td>100 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Please refer to the addgene website (<a href="https://www.addgene.org/protocols/inoculate-bacterial-culture/">https://www.addgene.org/protocols/inoculate-bacterial-culture/</a>) for detailed preparation instructions.</div>
<div class="note">
<span class="note-title">Note:</span> The medium can be stored at room temperature (15°C–25°C) without antibiotics until use. Thus, the LB liquid medium can be prepared a day before its intended use.</div>
<h3 id="sec2.2">Maintenance of HeLa cells</h3>
<div>Culture medium (DMEM supplemented with 10% FBS and 1% ZellShield)</div>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>445 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>ZellShield</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The culture medium can be stored at 4°C away from light until use.</div>
<h3 id="sec2.3">Live cell imaging</h3>
<ul>
<li>To prepare a stock solution of rapamycin, dissolve rapamycin in dimethyl sulfoxide to a final concentration of 100 μM. Stock solutions may be stored at −30°C until use.</li>
<li>For live cell imaging, prepare phenol red-free DMEM supplemented with 10% FBS, 4 mM L-glutamine, and 1% penicillin and streptomycin (referred to as imaging medium).</li>
</ul>
<table id="undtbl4">
<caption>Imaging medium</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Phenol red-free DMEM</td>
<td>n/a</td>
<td>435 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>
<span class="small-caps">l</span>-glutamine (200 mM)</td>
<td>4 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin</td>
<td>100 units penicillin and 0.1 mg streptomycin/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The imaging medium can be stored at 4°C away from light until use.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Seeding and culturing of the cells (day 1)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>Before starting this protocol, cells should be cultured for at least one week after thawing; additionally, the confluency of the cells should not exceed 80%.</p>
<ol>
<li>Add 4 mL prewarmed PBS at the periphery of the dish and slowly tilt to gently wash the cells.</li>
<li>Remove PBS.</li>
<li>Add 1 mL 0.05% trypsin-EDTA solution to the dish.</li>
<li>Incubate the cells at 37°C for 4 min in a 5% CO<sub>2</sub> incubator.</li>
<li>Gently tap the dish to completely detach the cells.</li>
<li>Add 5 mL prewarmed fresh culture medium and suspend the cells by gentle pipetting.</li>
<li>Transfer the cells to a sterilized 15-mL conical tube.</li>
<li>Centrifuge the tube with swing buckets at 161 × <i>g</i> and at room temperature (15°C–25°C) for 2 min.</li>
<li>Add 7 mL prewarmed culture medium and suspend the cells by gentle pipetting.</li>
<li>To count cells, mix 40 μL of the cell suspension with 40 μL trypan blue in a 1.5-mL microcentrifuge tube. Gently pipette up and down several times to mix.</li>
<li>Use an automated cell counter with 10 μL of the mixture.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A hemocytometer may be used for cell counts.</div>
<div class="note">
<span class="note-title">Note:</span> Verify, at least qualitatively, that cell viability, shape, and size are not atypical. If the cell counter used can measure quantitative information about viability and size, among others, it is recommended to record such numerical information.</div>
<ol start="12">
<li>Plate 2.4 × 10<sup>5</sup> cells in a 3.5-cm poly-lysine-coated glass bottom dish and culture at 37°C in 5% CO<sub>2</sub> (2 mL/dish).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A 3.5-cm poly-lysine-coated glass-bottom dish can be substituted for either glass cover slips coated with poly-<span class="small-caps">D</span>-lysine or a 35-mm imaging dish with a polymer coverslip bottom (e.g., iBidi, Fitchburg, WI, USA; cat# 81156). Notably, as changes in cell adhesion conditions may alter cellular responses to various factors, it is recommended that cell adhesion conditions be kept consistent.</div>
<ol start="13">
<li>After plating, culture cells at 37°C and 5% CO<sub>2</sub> in an incubator for 4 h (until cells attach well) and then transfect.</li>
</ol>
<h3 id="sec3.2">Transfection (day 1)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="14">
<li>Transfect as follows:
<ol type="a">
<li>Add 0.5 μg of each plasmid-encoding effector and anchor (1:1 ratio; total 1 μg) to 100 μL Opti-MEM (Thermo Fisher Scientific) in a 5-mL polystyrene round-bottom tube.</li>
<li>Add 3 μL FuGENE HD (Promega) and mix gently to avoid introducing bubbles.</li>
<li>Incubate the transfection mixture at room temperature (15°C–25°C) for 10 min.</li>
<li>Add 100 μL of the mixture to the cells that are plated on a 3.5-cm poly-lysine-coated glass-bottom dish and mix gently.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount and proportion of plasmids need to be optimized according to the used cell line, the transfection reagents, and the purpose of the experiment.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other transfection reagents such as Lipofectamine<sup>TM</sup> 3000 Transfection Reagent (Thermo Fisher Scientific, L3000001) may be used to transfect cells according to the manufacturer’s instructions (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Flipofectamine3000_protocol.pdf&amp;title=TGlwb2ZlY3RhbWluZSAzMDAwIFJlYWdlbnQgUHJvdG9jb2wgKEVuZ2xpc2gp">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Flipofectamine3000_protocol.pdf&amp;title=TGlwb2ZlY3RhbWluZSAzMDAwIFJlYWdlbnQgUHJvdG9jb2wgKEVuZ2xpc2gp</a>).</div>
<ol start="15">
<li>After transfection, culture cells under 5% CO<sub>2</sub> at 37°C for 20–32 h.</li>
</ol>
<h3 id="sec3.3">Live cell imaging using epifluorescence microscopy (day 2)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>Cells were viewed using a 40× objective (Plan Apochromat Lambda Series, Nikon) mounted on an inverted Eclipse Ti2-E microscope (Nikon) and imaged using a Zyla 4.2 PLUS sCMOS camera (Oxford Instruments). Imaging data were processed using the NIS-Elements AR imaging software (Nikon).</p>
<ol start="16">
<li>Two hours before live cell imaging, replace culture medium containing the transfection mixture with 2 mL prewarmed fresh culture medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To reduce cytotoxicity of transfection reagents, the culture medium should be replaced at least 12–16 h after transfection.</div>
<ol start="17">
<li>Activate the STX stage top incubator system (Tokai Hit, Fujinomiya, Japan) to maintain a temperature of 37°C in 5% CO<sub>2</sub> for 1 h before live cell imaging.</li>
<li>Add 1 μL 1 mM MitoTracker Red CMXRos (Thermo Fisher Scientific) into the glass-bottom dish (final concentration: 500 nM) 30 min before live cell imaging.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To stain the cells uniformly, 1 μL MitoTracker Red CMXRos may be aliquoted to a 1.5-mL microcentrifuge tube and suspended in 100 μL culture medium. The suspension may then be added to the dish followed by gentle agitation. Additionally, the concentration of MitoTracker Red CMXRos should be optimized for each cell line used according to the manufacturer’s protocol (<a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fmp07510.pdf&amp;title=VXNlciBHdWlkZTogTWl0b1RyYWNrZXIgTWl0b2Nob25kcmlvbi1TZWxlY3RpdmUgUHJvYmVz">https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fmp07510.pdf&amp;title=VXNlciBHdWlkZTogTWl0b1RyYWNrZXIgTWl0b2Nob25kcmlvbi1TZWxlY3RpdmUgUHJvYmVz</a>).</div>
<ol start="19">
<li>During the staining process, start the microscopy system according to the manufacturer’s instructions (<a href="https://www.microscope.healthcare.nikon.com/products/inverted-microscopes/eclipse-ti2-series/eclipse-ti2-e">https://www.microscope.healthcare.nikon.com/products/inverted-microscopes/eclipse-ti2-series/eclipse-ti2-e</a>).</li>
<li>Thirty minutes after adding MitoTracker Red CMXRos, remove the culture medium and wash twice using 1 mL prewarmed imaging medium.</li>
<li>Add 2 mL prewarmed imaging medium.</li>
<li>Find and focus cells expressing NiCMM/iCMM.</li>
<li>Select five positions for live cell imaging.</li>
<li>Set the imaging parameter of the NIS-Elements AR imaging software (Nikon) (<a href="#fig2">Figure 2</a>)
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. An example of acquisition setting in the NIS-Elements AR imaging software</div>
<p>Details of acquisition settings in the NIS-Elements AR imaging software are shown.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although we do not have strict criteria for determining the imaging positions, we believe it is important to select the imaging positions so that we can capture iCMM/NiCMM-expressing cells without bias as much as is possible.</div>
<div class="note">
<span class="note-title">Note:</span> Live cell imaging was performed under the same imaging conditions for each experiment. Representative imaging conditions are described below.</div>
<h4 id="sec3.3.1">General information</h4>
<p>Camera Name: Andor Zyla VSC-07685</p>
<p>Numerical Aperture: 0.95</p>
<p>No Z-stack imaging</p>
<h4 id="sec3.3.2">mCherry (for mitochondria imaging)</h4>
<p>Modality: Widefield Fluorescence</p>
<p>Filter: mCherry-B-NTE-ZERO filter (excitation: 542–582 nm, emission: 604–679 nm)</p>
<p>Binning: 1×1</p>
<p>Exposure: 100 ms</p>
<p>Perfect focus (PFS), state: On</p>
<p>Zoom: 1.00×</p>
<p>Intensilight, Illuminator(Illuminator (EPI)) NDFilter: 3.1</p>
<h4 id="sec3.3.3">ECFP (for anchor imaging)</h4>
<p>Modality: Widefield Fluorescence</p>
<p>Filter: CFP-A-Basic-NTE filter (excitation: 425.5–442.5 nm, emission: 459–499 nm)</p>
<p>Binning: 1×1</p>
<p>Exposure: 100 ms</p>
<p>PFS, state: On</p>
<p>Zoom: 1.00×</p>
<p>Intensilight, Illuminator (Illuminator (EPI)) NDFilter: 3.1</p>
<h4 id="sec3.3.4">EYFP (for effector imaging)</h4>
<p>Modality: Widefield Fluorescence</p>
<p>Filter: YFP-A-Basic-NTE filter (excitation: 489–505 nm, emission: 524–546 nm)</p>
<p>Binning: 1×1</p>
<p>Exposure: 100 ms</p>
<p>PFS, state: On</p>
<p>Zoom: 1.00×</p>
<p>Intensilight, Illuminator (Illuminator (EPI)) NDFilter: 3.1</p>
<ol start="25">
<li>Start time-lapse imaging.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The interval between image acquisition is changed depending on the purpose of the experiment. Two representative imaging schedules are shown below.</div>
<p>[Schedule 1] Interval: 2 min, Duration: 40 min, Adding chemical dimerizer: frame 6</p>
<p>[Schedule 2] Interval: 5 min, Duration: 60 min, Adding chemical dimerizer: frame 5</p>
<ol start="26">
<li>Add a chemical dimerizer (1 μL 100 μM rapamycin in this protocol) at the appropriate time point to induce mitochondrial morphology changes (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. How to add chemical dimerizer during time-lapse imaging</div>
<p>During time-lapse imaging, it is recommended that the samples are kept out of the light of the room. We create a simple dark room by shading the underside of the microscope and placing a kim towel on top of the stage-top incubator where light can enter. When adding rapamycin, the cover of the stage-top incubator is removed, and rapamycin is added. After the addition of rapamycin, the cover is closed, and the images are taken again in a simple darkroom. In this figure, the light is turned on to facilitate the photo shoot; however, in the actual experiment, the light in the room is switched off.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To treat the cells uniformly with rapamycin, aliquot 1 μL 100 μM rapamycin into a 1.5-mL microcentrifuge tube and suspend in 100 μl imaging medium. Add the suspension so that it is spread evenly over the entire culture dish and is not accumulated in only one place.</div>
<div class="note">
<span class="note-title">Note:</span> The addition of rapamycin does not necessarily have to be performed in the dark. However, it is recommended that imaging is performed in the dark.</div>
<div class="note">
<span class="note-title">Note:</span> It is essential to use imaging conditions that do not cause changes in mitochondrial morphology unless the iCMM is activated. To optimize imaging conditions, it is recommended to perform time-lapse imaging without adding rapamycin and confirm that mitochondrial morphology does not change (no fragmentation or aggregation of mitochondria).</div>
<ol start="27">
<li>Process the imaging data using NIS-Elements AR imaging software (Nikon).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For image processing, it is necessary to follow the Data and Image Processing Policy of each journal. We show in <a href="#fig4">Figure 4</a> the image processing to be performed for a representative mitochondrial image acquired by epifluorescence microscopy.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. An example of image processing for mitochondria</div>
<p>An image of the processing in NIS-Elements AR imaging software of a raw image of mitochondria in HeLa cells visualized by MitoTracker Red CMXRos dye is shown.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> When performing quantitative analysis of mitochondrial morphology, it is necessary to perform image processing according to the method used. Methods using Image J (<a href="#bib5">Valente et al., 2017</a>) and machine learning (<a href="#bib2">Leonard et al., 2015</a>; <a href="#bib6">Zahedi et al., 2018</a>) have been developed for mitochondrial morphology analysis.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>iCMM facilitates inducing mitochondrial morphology changes in a time scale of minutes. Mitochondrial morphology changes induced by iCMM depend on the used FiCE. Specifically, YF induces a large punctate structure of mitochondria (due to clustering rather than fusion of mitochondria), while YF-Cav1s and mActZ-FY induce a small punctate structure (<a href="#fig5">Figure 5</a>). In contrast, NiCMM, which is the technical counterpart of iCMM, does not induce any mitochondrial morphological changes, even when CiCE is translocated to the mitochondria (<a href="#fig5">Figure 5</a>). We have so far confirmed that the induction of mitochondrial morphological changes by iCMM occurs similarly in HeLa human cervix adenocarcinoma cells, Hep 3B human hepatoma cells, and U-2 OS human bone osteosarcoma cells (<a href="#bib3">Miyamoto et al., 2021</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. iCMM-induced changes in mitochondrial morphology</div>
<p>Reprinted with permission from <a href="#bib3">Miyamoto et al., 2021</a>.</p>
<p>(A–D) Time-lapse imaging (40 min/frame) of HeLa cells transiently expressing the indicated NiCMM/iCMM system was performed on confocal microscopy. Rapamycin (50 nM) was added after image acquisition at frame 1 (indicated as “0 min”). Mitochondria (coral) were stained with the MitoTracker Red CMXRos dye. Tom20-CR: slate blue, effector; yellow. Scale bar = 10 μm. Insets show a magnified view of the boxed area. Scale bar in inset = 1 μm. LUTs (Effector): 0–2000 [mYF, YF, and YF-Cav1s], 0–500 [mActZ-FY].</p>
</figcaption>
</figure>
<p>In our experiments, we confirmed that iCMM-induced mitochondrial morphology changes do not affect mitochondrial membrane potential or reactive oxygen species production over a 120-min observation window as judged by tetramethylrhodamine ethyl ester (Thermo Fisher, T669) and MitoSox (Thermo Fisher, M36008), respectively. In contrast, the oxygen consumption and extracellular acidification rates were decreased only when mitochondrial morphological changes were induced by mActZ-FY.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>It should be noted that as iCMM artificially induces changes in mitochondrial morphology, comparisons of the respective results with physiological mitochondrial morphological changes should be made with caution. However, detailed observation of cellular responses to iCMM-induced alterations may yield novel insights. Thus, an appropriate combination of iCMM and conventional genetic approaches (e.g., modification of genes involved in mitochondrial morphology) is required to elucidate the operating mechanism(s) of living organisms, with a focus on mitochondrial morphology.</p>
<p>The current protocol employs rapamycin; however, it is important to note that rapamycin is a potent inhibitor of mammalian/mechanistic targeting of rapamycin complex 1, a serine/threonine kinase. If the use of rapamycin is not suitable, then other CID methods or optogenetics may be used instead (<a href="#bib4">Stanton et al., 2018</a>). However, optimization of the effectors and anchor is necessary.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No or very few cells express iCMM/NiCMM (steps 14 and 22 in step-by-step method details).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>With respect to cell lines that are used for the first time, it is recommended to transfect only mYF in order to optimize the transfection protocol. The amount of plasmid to be transfected (0.1–5 μg), the ratio of plasmid to transfection reagent (1:1–1:5), and the time between transfection and experiment (12–72 h) are the three most crucial points to consider. It is also possible that performing transfection 18–24 h after seeding the cells may improve the results. If there are no cells expressing iCMM/NiCMM at all, we recommend changing the transfection reagent or the transfection method (e.g., electroporation and lentivirus transfection).</p>
<p>By contrast, if the efficiency of gene expression is low in a cell line that has been expressing genes successfully in the past, first check the quality of the plasmid: ensure that the A260/A280 ratio is &lt; 1.8, and that the A260/A230 ratio is between 1.8 and 2.2. In addition, run 500 ng plasmid on a 1% agarose gel by electrophoresis to ensure that the plasmid is not degraded. Next, check cell morphology and proliferation rate to determine whether cell characteristics are changed. If there are no respective issues, it is recommended to individually change the conditions and factors (Opti-MEM or transfection reagent) for stepwise examination. New reagents or media may also be tested.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The mitochondrial structure is aggregated in cells expressing iCMM/NiCMM before starting time-lapse imaging (<a href="#fig6">Figure 6</a>) (step 22 in step-by-step method details).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/901-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Representative images of aggregated mitochondria</div>
<p>HeLa cells were transfected with Tom20-CR. Live-cell imaging was performed at 18 h post-transfection. Raw mitochondrial images were processed by Clarify.ai. and Denoise.ai. Scale bar = 10 μm.</p>
</figcaption>
</figure>
<h3 id="sec6.4">Potential solution</h3>
<p>If expression levels of Tom20-CR are excessively high, mitochondria may not be able to form a normal network structure. This can be improved by reducing the amount of plasmid introduced into the cells or by reducing the time between transfection and experiment. In addition, this may also be improved by using a mitochondria-targeting signal (MTS) other than Tom20-derived MTS or by changing the length of the linker connecting the functional domains.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The number of cells adhering to the glass bottom dish is low (step 22 in step-by-step method details).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>It is possible that cells detach during washing. Although the proteins constituting iCMM/NiCMM do not induce cell death, transfection-induced cell death occurs at a certain rate. Optimization of cell confluence is also recommended as a potential solution.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The mitochondrial structure does not change after adding chemical dimerizer (step 26 in step-by-step method details).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Ensure that expression levels of FiCEs and of the anchor (Tom20-CR) are adequate. For example, YF may not induce mitochondrial morphological changes effectively at low expression levels. mActZ-FY has lower expression levels than other FiCEs, however, lower expression levels are more effective for inducing mitochondrial morphology changes.</p>
<p>If expression of FiCE and of the anchor is sufficient, add additional chemical dimerizers. If an additional chemical dimerizer induces mitochondrial morphological changes, it is possible that the previous chemical dimerizer was not added adequately. Further, ensure that the chemical dimerizer had been stored properly.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Fluorescence intensity of mitochondria stained with MitoTracker Red CMXRos decrease during time-lapse imaging (step 27 in step-by-step method details).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Change the exposure time and ND filter to achieve the lowest fluorescence intensity at which mitochondria can be observed or reduce the number of imaging positions. As MitoTracker Red CMXRos stains mitochondria in a mitochondrial membrane potential-dependent manner, it is also necessary to avoid imaging under conditions in which the membrane potential is reduced.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Derose, R., Miyamoto, T., and Inoue, T. (2013). Manipulating signaling at will: Chemically-inducible dimerization (CID) techniques resolve problems in cell biology. Pflugers Arch. Eur. J. Physiol. <i>465</i>, 409-417. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00428-7/sref1">View at publisher</a></p>
<p id="bib2">Leonard, A.P., Cameron, R.B., Speiser, J.L., Wolf, B.J., Peterson, Y.K., Schnellmann, R.G., Beeson, C.C., and Rohrer, B. (2015). Quantitative analysis of mitochondrial morphology and membrane potential in living cells using high-content imaging, machine learning, and morphological binning. Biochim. Biophys. Acta <i>1853</i>, 348-360. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00428-7/sref2">View at publisher</a></p>
<p id="bib3">Miyamoto, T., Uosaki, H., Mizunoe, Y., Han, S.I., Goto, S., Yamanaka, D., Masuda, M., Yoneyama, Y., Nakamura, H., Hattori, N., et al. (2021). Rapid manipulation of mitochondrial morphology in a living cell with iCMM. Cell Rep. Methods. <a class="external-link" href="https://doi.org/10.1016/j.crmeth.2021.100052">https://doi.org/10.1016/j.crmeth.2021.100052</a></p>
<p id="bib4">Stanton, B.Z., Chory, E.J., and Crabtree, G.R. (2018). Chemically induced proximity in biology and medicine. Science <i>359</i>, eaao5902. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00428-7/sref4">View at publisher</a></p>
<p id="bib5">Valente, A.J., Maddalena, L.A., Robb, E.L., Moradi, F., and Stuart, J.A. (2017). A simple ImageJ macro tool for analyzing mitochondrial network morphology in mammalian cell culture. Acta Histochem. <i>119</i>, 315-326. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00428-7/sref5">View at publisher</a></p>
<p id="bib6">Zahedi, A., On, V., Phandthong, R., Chaili, A., Remark, G., Bhanu, B., and Talbot, P. (2018). Deep analysis of mitochondria and cell health using machine learning. Sci. Rep. <i>8</i>, 1-15. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00428-7/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Takafumi Miyamoto (<a href="mailto:takmi565@md.tsukuba.ac.jp">takmi565@md.tsukuba.ac.jp</a>).</p>
<h4>Materials availability</h4>
<p>Plasmids generated in this study have been deposited to Addgene (see <a href="#key-resources-table">key resources table</a>).</p>
<h4>Data and code availability</h4>
<p>This study did not generate any datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This study was supported in part by a Grant-in-Aid for Scientific Research on Innovative Areas (18H04854), the Leading Initiative for Excellent Young Researchers, <a href="https://doi.org/10.13039/501100002241">JST</a> CREST (JPMJCR1927), JST COI-NEXT (JPMJPF2017), and Moonshot R&amp;D – MILLENNIA Program (20350956). We thank Katsuko Okubo and Yasuko Maruyama (University of Tsukuba) for generous assistance.</p>
<h3>Author contributions</h3>
<p>T.M. designed the experiments; T.M. and H.S. optimized the protocols and prepared the figures; and T.M. and S.-I.H. wrote the manuscript. All authors have reviewed the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>