<article>
<section>
<h1 id="header">Generating stable isolated mitochondrial recipient clones in mammalian cells using MitoPunch mitochondrial transfer</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Alexander J. Sercel,<sup><a href="#aff1">1</a></sup> Alexander J. Napior,<sup><a href="#aff2">2</a></sup> Alexander N. Patananan,<sup><a href="#aff2">2</a></sup> Ting-Hsiang Wu,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a></sup> Pei-Yu Chiou,<sup><a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a></sup> and Michael A. Teitell<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff6">6</a>,<a href="#aff7">7</a>,<a href="#aff8">8</a>,<a href="#aff9">9</a>,<a href="#aff10">10</a>,<a href="#fn1">11</a>,<a href="#fn2">12</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Molecular Biology Interdepartmental Doctoral Program, University of California, Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff2"><sup>2</sup>Department of Pathology and Laboratory Medicine, David Geffen School of Medicine, University of California, Los Angeles, CA 90095, USA</p>
<p id="aff3"><sup>3</sup>NanoCav, LLC, Culver City, CA 90230, USA</p>
<p id="aff4"><sup>4</sup>NantWorks, LLC, Culver City, CA 90230, USA</p>
<p id="aff5"><sup>5</sup>Department of Mechanical and Aerospace Engineering, University of California, Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff6"><sup>6</sup>California NanoSystems Institute, University of California, Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff7"><sup>7</sup>Department of Bioengineering, University of California, Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff8"><sup>8</sup>Eli and Edythe Broad Center of Regenerative Medicine and Stem Cell Research, University of California, Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff9"><sup>9</sup>Department of Pediatrics, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90095, USA</p>
<p id="aff10"><sup>10</sup>Jonsson Comprehensive Cancer Center, David Geffen School of Medicine, University of California, Los Angeles, Los Angeles, CA 90024, USA</p>
<p id="fn1"><sup>11</sup>Technical contact</p>
<p id="fn2"><sup>12</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:mteitell@mednet.ucla.edu">mteitell@mednet.ucla.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100850">10.1016/j.xpro.2021.100850</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes the assembly and use of MitoPunch to deliver mitochondria containing mitochondrial DNA (mtDNA) into cells lacking mtDNA (ρ0 cells). MitoPunch generates stable isolated mitochondrial recipient clones with restored mtDNA and recovered respiration, enabling investigation of mtDNA mutations and mtDNA-nuclear DNA interactions in a range of cell types.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref6">Sercel et al. (2021)</a> and <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref5">Patananan et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>High-throughput mitochondrial transfer into transformed and primary recipient cells</li>
<li>Selection scheme to isolate stable mtDNA transplant recipient cells</li>
<li>MitoPunch device is easily assembled with minimal engineering expertise</li>
<li>MitoPunch enables new combinations of mitochondrial and nuclear genomes</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1047-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the process of using MitoPunch to deliver isolated HEK293T mitochondria into 143BTK− ρ0 recipient mammalian cells. We have used MitoPunch to generate stable isolated mitochondrial recipient (SIMR) cell lines with a wide range of both human and mouse cells, of both malignant and primary, non-immortalized origins. Few alterations are necessary to adapt the protocol to different cell lines and cell types, and guidance for doing so is included in the text below.</p>
<h3 id="sec1.1">Construct MitoPunch apparatus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>Fabricate aluminum washer and 5 V solenoid housing.
<ol type="a">
<li>Aluminum washer dimensions: outer diameter, 25 mm; inner diameter, 10 mm; thickness, 1.4 mm.</li>
<li>Aluminum housing dimensions: rectangular prism of 19.13 mm length, 12.80 mm width, and 30.50 mm height with an internal vertical cutout of 12.65 mm length, 10.20 mm depth, and 19.13 mm height, including two 2 mm diameter screw holes in the bottom of the housing and one 2 mm screw hole on either side of the cutout for securing the 5 V solenoid (Spark Fun, Cat#ROB-11015).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Ensure the 5 V solenoid fits securely in the machined housing. This may require close collaboration with a machinist.</div>
<ol start="2">
<li>Connect mains power cord (Volex, Cat#1725010B1) to the power supply (MEAN WELL, Cat#RS-35-12) by removing the IEC 60320 C13 adapter, striping the wires, and attaching them appropriately to the load (L), neutral (N), and ground hookups.</li>
<li>Connect the power supply to the power supply mini board (Futurelec, Cat#MINIPOWER) using a barrel style DC connector with wire leads (McMaster-Carr, Cat#8320N111).</li>
<li>Insert hookup wires (Spark Fun, Cat#PRT-08867) into the positive and negative terminals of the mini board, then connect the mini board terminals to the breadboard (Spark Fun, Cat#12615).</li>
<li>Install the switch (Spark Fun, Cat#COM-09190) across the DIP ravine on the breadboard and install the jumper wires (Spark Fun, Cat#PRT-00124) to complete the circuit.</li>
<li>Connect the JST jumper 2 wire to the breadboard.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> See <a href="#fig1">Figure 1</a> for circuit diagram and image of fully assembled device to assist with the wiring of the apparatus. See <a href="#fig2">Figure 2</a> for step-by-step assembly of the device.</div>
<ol start="7">
<li>Place the 5 V solenoid into the custom metal casing ensuring that the top of the solenoid is level with the top of the casing, and tighten screws (diameter, 2 mm) to hold it in place.</li>
<li>Fit the 5 V solenoid in the casing onto a threaded plug with screws (Thor Labs, Cat#SM1PL) and screw plug into the bottom plate (Thor Labs, Cat#CP02T).</li>
<li>Insert 4 optomechanical assembly rods (Thor Labs, Cat#ER3) into the holes in the bottom plate.
<ol type="a">
<li>With rods flush with the bottom cage plate, secure rods by tightening the screws found next to each rod slot.</li>
</ol>
</li>
<li>Remove internal ring from middle cage plate using a spanner wrench (Thor Labs, Cat#SPW602) and slide the middle cage plate (Thor Labs, Cat#CP02) down the assembly rods until the top surface of the middle plate is flush with the top surface of the solenoid.
<ol type="a">
<li>Reinsert internal ring, tighten, and double check that the top of the middle cage plate is flush with the top of the solenoid.</li>
</ol>
</li>
<li>Mount the custom washer atop the middle cage plate and check that the ring stably supports it.</li>
<li>Insert the 2-pin JST connector from the breadboard into the MitoPunch 2-pin JST PH connector and test whether the piston actuates when the switch is depressed (see <a href="#sec6.1">troubleshooting 1</a>).</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1047-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. MitoPunch circuit diagram</div>
<p>(A) Circuit diagram of the MitoPunch apparatus. Numbered nodes indicate where wires are inserted into the corresponding breadboard ports.</p>
<p>(B) Image of fully wired and completed MitoPunch apparatus.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1047-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Assembling the MitoPunch apparatus</div>
<p>(A) Layout of all MitoPunch components prior to assembly.</p>
<p>(B) MitoPunch solenoid mounted on the aluminum plug.</p>
<p>(C) Mounted MitoPunch solenoid screwed into base plate.</p>
<p>(D) MitoPunch base plate with optomechanical assembly rods inserted and fixed with screws.</p>
<p>(E) Middle plate mounted flush with the solenoid using screws on the optomechanical assembly rods.</p>
<p>(F) Fully assembled MitoPunch apparatus with top plate lowered onto PDMS reservoir ready for MitoPunch transfer.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Assemble polydimethylsiloxane (PDMS) reservoirs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 h</div>
<p>PDMS reservoirs are essential to hold the mitochondrial suspension for delivery into cells using MitoPunch and for delivering the pressure that enables a mitochondrial transfer. PDMS is a biocompatible polymer that is easily fabricated with minimal equipment and expertise (<a href="#bib2">Berthier et al., 2012</a>).</p>
<ol start="13">
<li>Weigh and mix Sylgard 184 PDMS base and curing agent (Dow Corning, Cat#1673921) at a 10:1 ratio.
<ol type="a">
<li>Prepare a weigh boat on a scale. Tare the scale.</li>
<li>Pour 12 g PDMS base into the weigh boat.</li>
<li>Tare the scale.</li>
<li>Add 1.2 g curing agent into the weigh boat.</li>
<li>Mix thoroughly for about 1–2 min with a chemical spatula or similar implement.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PDMS should remain covered until cured to avoid the accumulation of dust and debris.</div>
<div class="note">
<span class="note-title">Note:</span> For visual reference of PDMS fabrication, see <a href="#fig3">Figure 3</a>.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1047-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Fabricating PDMS reservoirs for MitoPunch</div>
<p>(A) Uncured PDMS containing bubbles following thorough mixing of the PDMS base and curing agent.</p>
<p>(B) Uncured PDMS following degassing to remove all bubbles.</p>
<p>(C) Punching disks out of a cured layer of PDMS with a hole punch.</p>
<p>(D) Seven punched PDMS disks in a 10 cm dish.</p>
<p>(E) PDMS disks and rings on a 15 cm dish in an oxygen plasma cleaning chamber.</p>
<p>(F) Bonded PDMS reservoir after oxygen plasma treatment.</p>
</figcaption>
</figure>
<ol start="14">
<li>Degas PDMS.
<ol type="a">
<li>Transfer the mixed PDMS into a vacuum desiccator.</li>
<li>Connect a vacuum pump to the desiccator and turn on the pump.</li>
<li>Degas PDMS until all visible air bubbles in the PDMS have disappeared, approximately 30 min.</li>
</ol>
</li>
<li>Pour and cure PDMS.
<ol type="a">
<li>Prepare a clean 10 cm cell culture dish on a scale. Tare the scale.</li>
<li>Pour 6 g of PDMS into the dish. Be careful not to introduce bubbles. If bubbles are generated while pouring, use a 20 μL pipette tip to remove the bubbles or move the dish into the desiccator and repeat degas step.
<ol type="i">
<li>Repeat with a second 10 cm dish using the remaining PDMS.</li>
</ol>
</li>
<li>Allow PDMS to uniformly coat the dishes (target thickness ∼1 mm).</li>
<li>Put the dishes with PDMS into an oven at 65°C for 2–4 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> 10 cm dishes may melt if placed directly on a metal surface in the oven. Place dishes on autoclave safe plastic inside the oven to avoid melting.</div>
<ol start="16">
<li>Hole punch PDMS.
<ol type="a">
<li>Take the cured PDMS out from the oven and allow to cool to 20°C–25°C.</li>
<li>For the bottom layer, cut out circular PDMS disks with a hole puncher of 25 mm diameter (McMaster-Carr, Cat#3418A25).</li>
<li>For the top layer, cut out circular PDMS disks with a hole puncher of 25 mm diameter. Then use another hole puncher of 10 mm diameter (McMaster-Carr, Cat#3418A1) to remove the center portion and create a PDMS ring.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare an equal number of disks and rings for bonding.</div>
<ol start="17">
<li>Bond PDMS layers.
<ol type="a">
<li>Remove dust and debris from the PDMS disks and PDMS rings with a piece of removable adhesive tape.</li>
<li>Place the PDMS disks and PDMS rings into an expanded plasma cleaner (Harrick Plasma, Cat#PDC-001) with the clean bonding side face up.</li>
<li>Close the chamber and set the plasma cleaner to high.
<ol type="i">
<li>Maintain internal pressure between 300 mTorr and 500 mTorr with the RF level set on high for 1 min.</li>
</ol>
</li>
<li>Remove plasma treated PDMS disks and rings from the oxygen plasma chamber.</li>
<li>Carefully align and bond the disks and rings by applying firm pressure to ensure the two pieces are sealed together.
<div class="note">
<span class="note-title">Note:</span> Join the two PDMS pieces by pressing the oxygen plasma treated faces together to ensure strong bonding.</div>
</li>
<li>Heat the bonded PDMS reservoir in the oven at 65°C for at least 2 h to complete bonding.
<div class="critical">
<span class="critical-title">Critical:</span> Excess exposure of the oxygen plasma treated faces of the PDMS pieces to air and debris before bonding will result in poor adhesion between the two components. Join the two parts as quickly as possible after oxygen plasma treatment for optimal bonding strength.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Sterilization of MitoPunch components</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="18">
<li>Sterilize a number of PDMS reservoirs equal to the number of MitoPunch replicates to be performed.
<ol type="a">
<li>Wash reservoirs three times with a 10% bleach solution.</li>
<li>Rinse three times with DI water.</li>
<li>Wash reservoirs three times with a 70% ethanol in water solution.</li>
<li>Rinse three times with DI water.</li>
<li>Package PDMS reservoirs into disposable sterilization pouches.</li>
<li>Autoclave packaged PDMS at 121°C for 30 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The wash steps can be completed in a 15 cm dish or a 50 mL conical tube for convenience prior to autoclaving in a disposable sterilization pouch.</div>
<ol start="19">
<li>Sterilize stainless steel forceps.
<ol type="a">
<li>Wash forceps three times with a 10% bleach solution.</li>
<li>Rinse three times with DI water.</li>
<li>Wash forceps three times with a 70% ethanol in water solution.</li>
<li>Rinse three times with DI water.</li>
<li>Package forceps into a disposable sterilization pouch.</li>
<li>Autoclave packaged forceps at 121°C for 30 min.</li>
</ol>
</li>
<li>Sterilize high-vacuum grease.
<ol type="a">
<li>Apply approximately 4 mL high-vacuum grease to a 5 cm glass petri dish.</li>
<li>Close the glass dish and seal with autoclave tape.</li>
<li>Autoclave high-vacuum grease at 121°C for 30 min.</li>
</ol>
</li>
<li>Sterilize glass cloning cylinders.
<ol type="a">
<li>Place as many cloning cylinders as necessary into a 100 mL glass beaker.</li>
<li>Seal with aluminum foil and apply autoclave tape to the foil top.</li>
<li>Autoclave cloning cylinders at 121°C for 30 min.</li>
</ol>
</li>
</ol>
<h3 id="sec1.4">Thaw and expand mitochondrial donor and recipient cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="22">
<li>Thaw mitochondrial donor and recipient cells according to standard protocols for the chosen lines.
<ol type="a">
<li>Thaw at least 5 d prior to mitochondrial transfer to allow for expansion and mycoplasma testing of cell lines.</li>
</ol>
</li>
<li>Expand mitochondrial donor and recipient cell lines.
<ol type="a">
<li>Expand recipient lines to have 1 × 10<sup>5</sup> cells per planned MitoPunch replicate.</li>
<li>Expand donor cells on T-225 plates until you have one T-225 plate for every two planned MitoPunch replicates, or ∼1.5 × 10<sup>6</sup> mitochondrial donor cells for each replicate transfer.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Donor cells harvested at high confluence are more likely to cause needle clogging during the mitochondrial isolation. Aim to harvest donor cells at ∼80%–90% confluence.</div>
<h3 id="sec1.5">Test recipient cell sensitivity to different SIMR clone selection media conditions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 d<b>ays</b>
</div>
<p>Isolation of SIMR clones following a MitoPunch transfer requires a restrictive medium selection that selectively permits only cells with functional mitochondria to grow. This selection scheme takes advantage of the dependence on electron transport chain activity for <i>de novo</i> nucleotide synthesis in mammalian cells (<a href="#bib4">Gregoire et al., 1984</a>). Two selection medium formulations, SIMR selection medium and galactose SIMR selection medium, are used depending on the sensitivity of mitochondrial recipient cells to this selection. SIMR selection medium contains 4.5 g/L glucose, whereas the galactose SIMR selection medium is glucose-free and supplemented with galactose so that glycolysis generates no net ATP (<a href="#bib1">Aguer et al., 2011</a>). Below is a protocol to measure the sensitivity of a chosen ρ0 recipient cell line in these two restrictive mediums in preparation for post-MitoPunch selection.</p>
<ol start="24">
<li>Seed ∼2 × 10<sup>4</sup> cells in a 12-well plate with the recipient cell line in uridine-supplemented complete medium.
<ol type="a">
<li>Incubate ∼24 h to allow cells to adhere.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This seeding density depends upon the growth rate of the chosen recipient cell line. Alter the seeding density such that the cells would reach confluence ∼5 d post seeding under normal conditions.</div>
<ol start="25">
<li>Exchange the uridine-supplemented complete medium for 2 mL SIMR selection medium.
<ol type="a">
<li>Exchange medium with fresh SIMR selection medium daily.</li>
</ol>
</li>
<li>Monitor the wells twice daily for 7 d.
<ol type="a">
<li>Record the time point when cells begin to die.</li>
</ol>
</li>
<li>Continue to feed wells and monitor cell viability until all cells have died or 7 d have passed.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In the event that living cells remain after 7 d, re-plate the experiment as outlined in step 23, and begin with the following alternative protocol.</div>
<ol start="28">
<li>Exchange the uridine-supplemented complete medium for 2 mL galactose SIMR selection medium.
<ol type="a">
<li>Exchange medium with fresh galactose SIMR selection medium daily.</li>
</ol>
</li>
<li>Monitor the wells five times daily for 7 d.
<ol type="a">
<li>Record the time point when cells begin to die.</li>
</ol>
</li>
<li>Continue to feed wells and monitor cell viability until all cells have died or 7 d have passed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Galactose SIMR selection medium is far more restrictive than SIMR selection medium and cells can die within hours of first treatment. Careful monitoring of this process is key to timing the selection of MitoPunch experiments to successfully isolate SIMR clones. If cells do not die under either medium, see <a href="#sec6.3">troubleshooting 2</a>.</div>
<h3 id="sec1.6">Seed mitochondrial recipient cells on PET filters</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 d<b>ays</b>
</div>
<ol start="31">
<li>Collect and count mitochondrial recipient cells.
<ol type="a">
<li>Remove culture medium from recipient cell flasks.</li>
<li>Wash plate with 1X Dulbecco’s Phosphate-Buffered Salt Solution (DPBS) (Corning, Cat#21031CV).</li>
<li>Aspirate DPBS and apply StemPro Accutase Cell Dissociation Reagent (Accutase) (Thermo Fisher Scientific, Cat#A1110501).
<div class="note">
<span class="note-title">Note:</span> Use preferred dissociation reagent for the chosen recipient cell type.</div>
</li>
<li>Incubate in 37°C humidified incubator for 5 min.</li>
<li>Resuspend cells in 8.5 mL medium. Transfer cell suspension to 15 mL conical tube.</li>
<li>Centrifuge at 500 × <i>g</i> for 5 min, aspirate supernatant, and resuspend in culture medium.</li>
<li>Count cells using a hemocytometer.</li>
<li>Calculate volume of cell suspension needed to seed 1 × 10<sup>5</sup> recipient cells.</li>
</ol>
</li>
</ol>
<ol start="32">
<li>Seed recipient cells.
<ol type="a">
<li>Insert 3.0 μm PET filter inserts (Corning, Cat# 353181) into wells of a 12-well dish and dispense 1.5 mL medium to the well outside of the filter insert.</li>
<li>Pipette volume of cell suspension containing 1 × 10<sup>5</sup> cells into filter insert. Add volume of medium to filter insert to bring volume inside filter insert to 500 μL.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure even plating of recipient cells on the filter insert for consistent MitoPunch results. See <a href="#fig4">Figure 4</a>A for an example of an evenly seeded well 24 h post-seeding.</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1047-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Filter insert seeding and harvesting</div>
<p>(A) Filter insert without cells in DPBS.</p>
<p>(B) Filter insert seeded with 1 × 10<sup>5</sup> 143BTK− ρ0 cells imaged after washing with DPBS.</p>
<p>(C) Filter insert seeded with 1 × 10<sup>5</sup> 143BTK− ρ0 cells imaged after washing with DPBS and ∼7 min incubation with Accutase.</p>
<p>Scale bars represent 200 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM w/ 4.5 g/L glucose, sodium pyruvate, and L-glutamine</td>
<td>Corning</td>
<td>Cat#10013CM</td>
</tr>
<tr>
<td>DMEM, no glucose</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11966025</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Omega Scientific</td>
<td>Cat#FB-11</td>
</tr>
<tr>
<td>Fetal Bovine Serum, dialyzed, US origin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#26400-044</td>
</tr>
<tr>
<td>Penicillin-Streptomycin Solution, 100×</td>
<td>Corning</td>
<td>Cat#30-002-CI</td>
</tr>
<tr>
<td>GlutaMAX Supplement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#35050061</td>
</tr>
<tr>
<td>MEM Non-Essential Amino Acids Solution (100X)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11140050</td>
</tr>
<tr>
<td>Uridine</td>
<td>Acros Organics</td>
<td>Cat#140770250</td>
</tr>
<tr>
<td>D-(+)-Galactose</td>
<td>Millipore Sigma</td>
<td>Cat#G5388-100G</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate-Buffered Salt Solution 1X</td>
<td>Corning</td>
<td>Cat#21031CV</td>
</tr>
<tr>
<td>Sodium Pyruvate (100 mM)</td>
<td>Corning</td>
<td>Cat#25000Cl</td>
</tr>
<tr>
<td>Dulbecco's Phosphate-Buffered Salt Solution, w/ Calcium and Magnesium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#14040133</td>
</tr>
<tr>
<td>StemPro Accutase Cell Dissociation Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1110501</td>
</tr>
<tr>
<td>16% Formaldehyde</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#28906</td>
</tr>
<tr>
<td>Crystal Violet</td>
<td>Fisher Chemical</td>
<td>Cat#C581-25</td>
</tr>
<tr>
<td>Methanol</td>
<td>Fisher Chemical</td>
<td>Cat#A412-4</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Qproteome Mitochondria Isolation Kit</td>
<td>QIAGEN</td>
<td>Cat#37612</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>143 BTK– osteosarcoma ρ0; human, female</td>
<td>(<a href="#bib5">Patananan et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Primary dermal fibroblast normal ρ0; human, male, neonatal</td>
<td>(<a href="#bib5">Patananan et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>HEK293T dsRed; human, female</td>
<td><a href="#bib7">Miyata et al., 2014</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Breadboard - Full-Size (Bare)</td>
<td>SparkFun</td>
<td>Cat#12615</td>
</tr>
<tr>
<td>Solenoid - 5V</td>
<td>SparkFun</td>
<td>Cat#ROB-11015</td>
</tr>
<tr>
<td>Momentary Pushbutton Switch - 12mm Square</td>
<td>SparkFun</td>
<td>Cat#COM-09190</td>
</tr>
<tr>
<td>Hook-up Stranded Wire - Black (22 AWG)</td>
<td>SparkFun</td>
<td>Cat#PRT-08867</td>
</tr>
<tr>
<td>JST Jumper 2 Wire Assembly</td>
<td>SparkFun</td>
<td>Cat#PRT-09914</td>
</tr>
<tr>
<td>Jumper Wire Kit - 140pcs</td>
<td>SparkFun</td>
<td>Cat#PRT-00124</td>
</tr>
<tr>
<td>Power Supply Mini Board</td>
<td>Futurelec</td>
<td>Cat#MINIPOWER</td>
</tr>
<tr>
<td>35W Single Output Switching Power Supply</td>
<td>MEAN WELL</td>
<td>Cat#RS-35-12</td>
</tr>
<tr>
<td>Mains Power Cord, NEMA 5-15P to IEC 60320 C13, 2.3 m, 10 A, 125 VAC, Black</td>
<td>Volex</td>
<td>Cat#1725010B1</td>
</tr>
<tr>
<td>Push-In Plugs with Wire Leads, 2.1 mm End ID, 12V DC</td>
<td>McMaster-Carr</td>
<td>Cat#8320N111</td>
</tr>
<tr>
<td>Spanner Wrench for SM1-Threaded Retaining Rings</td>
<td>Thorlabs</td>
<td>Cat# SPW602</td>
</tr>
<tr>
<td>Externally SM1-Threaded Plug</td>
<td>Thorlabs</td>
<td>Cat#SM1PL</td>
</tr>
<tr>
<td>SM1-Threaded 30 mm Cage Plate, 0.50" Thick, 2 Retaining Rings, 8–32 Tap</td>
<td>Thorlabs</td>
<td>Cat#CP02T</td>
</tr>
<tr>
<td>Cage Assembly Rod, 3" Long, Ø6 mm</td>
<td>Thorlabs</td>
<td>Cat#ER3</td>
</tr>
<tr>
<td>SM1-Threaded 30 mm Cage Plate, 0.35" Thick, 2 Retaining Rings, 8–32 Tap</td>
<td>Thorlabs</td>
<td>Cat#CP02</td>
</tr>
<tr>
<td>Custom Aluminum Washer</td>
<td>University of California, Los Angeles, Machine Shop</td>
<td>N/A</td>
</tr>
<tr>
<td>Custom Aluminum Case</td>
<td>University of California, Los Angeles, Machine Shop</td>
<td>N/A</td>
</tr>
<tr>
<td>SYLGARD™ 184 Silicone Elastomer Kit 1.1 KG Kit</td>
<td>Dow Corning</td>
<td>Cat#1673921</td>
</tr>
<tr>
<td>Polypropylene Desiccator with Stopcock</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#53100250</td>
</tr>
<tr>
<td>230 mm Ceramic Metal Desiccator Plate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#53120230</td>
</tr>
<tr>
<td>10 mm Hole punch</td>
<td>McMaster-Carr</td>
<td>Cat#3418A1</td>
</tr>
<tr>
<td>25 mm Hole punch</td>
<td>McMaster-Carr</td>
<td>Cat#3418A25</td>
</tr>
<tr>
<td>Expanded Plasma Cleaner</td>
<td>Harrick Plasma</td>
<td>Cat#PDC-001</td>
</tr>
<tr>
<td>12–Well 3.0 μm Transparent PET Membrane Cell Culture Inserts</td>
<td>Corning</td>
<td>Cat#353181</td>
</tr>
<tr>
<td>26 G Blunt-ended needle</td>
<td>VWR</td>
<td>Cat#89134-164</td>
</tr>
<tr>
<td>3 mL Luer-Lok Syringe</td>
<td>BD Biosciences</td>
<td>Cat#309657</td>
</tr>
<tr>
<td>High-vacuum grease</td>
<td>Dow Corning</td>
<td>Cat#1597418</td>
</tr>
<tr>
<td>Cloning cylinders</td>
<td>Fisher Scientific</td>
<td>Cat# 09-552-21</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Preparation of complete medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM w/ 4.5 g/L glucose, sodium pyruvate, and L-glutamine (Corning, C#10013CM)</td>
<td>-</td>
<td>870 mL</td>
</tr>
<tr>
<td>Penicillin Streptomycin solution 100× (Corning, Cat#30-002-CI)</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>GlutaMAX Supplement (Thermo Fisher Scientific, Cat#35050061)</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acids 100× (Thermo Fisher Scientific, Cat#11140050)</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS) (Omega Scientific, Cat#FB-11)</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Medium can be stored at 4°C for up to 1 month.</p>
<table id="undtbl2">
<caption>Preparation of uridine-supplemented complete medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM w/ 4.5 g/L glucose, sodium pyruvate, and L-glutamine</td>
<td>-</td>
<td>869 mL</td>
</tr>
<tr>
<td>Uridine (Acros Organics, Cat#140770250)</td>
<td>50 mg/L</td>
<td>1 mL (from 50 mg/mL stock)</td>
</tr>
<tr>
<td>Penicillin Streptomycin solution</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>GlutaMAX Supplement</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acids</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Medium can be stored at 4°C for up to 1 month.</p>
<div class="note">
<span class="note-title">Note:</span> Uridine stocks are prepared and filtered through a 0.22 μm filter in advance then stored at –20°C for up to 1 year.</div>
<table id="undtbl3">
<caption>Preparation of SIMR selection medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM w/ 4.5 g/L glucose, sodium pyruvate, and L-glutamine</td>
<td>-</td>
<td>870 mL</td>
</tr>
<tr>
<td>Penicillin Streptomycin solution</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>GlutaMAX Supplement</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acids</td>
<td>1×</td>
<td>10 mL (from 100× stock)</td>
</tr>
<tr>
<td>Dialyzed Fetal Bovine Serum (FBS) (Thermo Fisher Scientific, Cat#26400-044)</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Medium can be stored at 4°C for up to 1 month.</p>
<table id="undtbl4">
<caption>Preparation of galactose SIMR selection medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM, no glucose (Thermo Fisher Scientific, Cat#11966025)</td>
<td>-</td>
<td>430 mL</td>
</tr>
<tr>
<td>Galactose (Millipore Sigma, Cat#G5388-100G)</td>
<td>4.5 g/L</td>
<td>2.25 g</td>
</tr>
<tr>
<td>Penicillin Streptomycin solution</td>
<td>1×</td>
<td>5 mL (from 100× stock)</td>
</tr>
<tr>
<td>GlutaMAX Supplement</td>
<td>1×</td>
<td>5 mL (from 100× stock)</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acids</td>
<td>1×</td>
<td>5 mL (from 100× stock)</td>
</tr>
<tr>
<td>Sodium Pyruvate (Corning, Cat#25000CI)</td>
<td>1×</td>
<td>5 mL (from 100× stock)</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Medium can be stored at 4°C for up to 1 month.</p>
<div class="note">
<span class="note-title">Note:</span> The galactose should be added by dissolving in 40 mL of DMEM and then adding back to the medium before filtering through a 0.22 μm PES filter.</div>
<table id="undtbl5">
<caption>Preparation of paraformaldehyde</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>16% Paraformaldehyde (Thermo Fisher Scientific, Cat#28906)</td>
<td>4%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Dulbecco's Phosphate-Buffered Salt Solution 1X</td>
<td>-</td>
<td>3 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>4%</b></td>
<td><b>4 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Paraformaldehyde solution should be made freshly before use.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde is a toxin and carcinogen that should be handled only in a chemical fume hood for proper ventilation with appropriate PPE including safety glasses, a lab coat, closed-toed shoes, and nitrile gloves. All waste containing paraformaldehyde should be labeled and disposed of through proper channels. Paraformaldehyde should be stored in a flammable storage cabinet.</div>
<table id="undtbl6">
<caption>Preparation of crystal violet staining solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Methanol</td>
<td>20%</td>
<td>20 mL</td>
</tr>
<tr>
<td>Deionized water</td>
<td>80%</td>
<td>80 mL</td>
</tr>
<tr>
<td>Crystal Violet (Fisher Chemical, Cat#C581-25)</td>
<td>0.5% w/v</td>
<td>5 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Crystal violet solution can be kept in the flammable chemical cabinet for up to 6 months.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Crystal violet staining solution contains methanol which is a flammable toxin and should be stored in a flammable storage cabinet. Appropriate PPE including safety glasses, a lab coat, closed-toed shoes, and nitrile gloves should be used when preparing and working with this solution. All waste containing crystal violet staining solution should be labeled and disposed of through proper channels.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Mitochondrial preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>MitoPunch requires a freshly isolated stock of cell-free mitochondria for transfer into recipient cells. This protocol uses the <a href="https://www.qiagen.com/us/resources/resourcedetail?id=b13c089e-17ec-4707-8efe-10d52d84349d&amp;lang=en">Qiagen Qproteome Mitochondrial Isolation Kit</a> (Qiagen, Cat#37612) with minor optimizations for efficiency and purity in this application; however, other mitochondrial isolation methods are also compatible with MitoPunch. Here we define purity as the absence of intact mitochondrial donor cells that escape through the disruption and mitochondrial isolation process. We have measured the purity of our mitochondrial isolates in <a href="#fig3">Figure 3</a>-figure supplement 1 of (<a href="#bib6">Sercel et al., 2021</a>).</p>
<ol>
<li>Harvest mitochondrial donor cells.
<ol type="a">
<li>Aspirate culture medium from T-225 flasks.</li>
<li>Pipette 20 mL DPBS into each flask.</li>
<li>Release cells from the flasks using a cell scraper.</li>
<li>Suspend scraped cells in DPBS and collect into 50 mL conical tubes.</li>
<li>Centrifuge tubes at 500 × <i>g</i> for 10 min at 4°C. Aspirate supernatant.</li>
<li>Wash cell pellets in 10 mL DPBS and centrifuge again at 500 × <i>g</i> for 10 min at 4°C. Aspirate supernatant.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep samples on ice when outside the centrifuge for the remainder of the mitochondrial isolation process.</div>
<ol start="2">
<li>Disrupt mitochondrial donor cells using the Qiagen Qproteome Mitochondrial Isolation Kit.
<div class="note">
<span class="note-title">Note:</span> The buffers used in the following steps are from the Qiagen Qproteome Mitochondrial Isolation Kit and must be handled according to manufacturer protocol.</div>
<ol type="a">
<li>Resuspend cell pellets in ice-cold Lysis Buffer at a concentration of ∼1 × 10<sup>7</sup> cells per 1 mL and transfer to 2 mL tubes.
<ol type="i">
<li>Prepare Lysis Buffer with 1:100 Protease Inhibitor Solution prior to resuspension.</li>
</ol>
</li>
<li>Incubate on end-over-end shaker for 10 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Ensure microcentrifuge is chilled to 4°C during this incubation in preparation for the next step.</div>
</li>
<li>Centrifuge the samples at 1000 × <i>g</i> for 10 min at 4°C and aspirate supernatant without disrupting the pellet.</li>
<li>Resuspend each cell pellet in 1.5 mL ice cold Disruption Buffer with a 1 mL pipette tip.
<ol type="i">
<li>Prepare Disruption Buffer with 1:100 Protease Inhibitor Solution prior to resuspension.</li>
</ol>
</li>
<li>Disrupt cells using a blunt-ended 26G (VWR, Cat#89134-164) needle attached to a 3 mL Luer-Lok syringe.
<ol type="i">
<li>Slowly pull the entire lysate into the syringe and expel the volume 10 times, avoiding the formation of bubbles in the suspension.
<div class="note">
<span class="note-title">Note:</span> If clogging of the disruption needle occurs, see <a href="#sec6.5">troubleshooting 3</a>.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Mitochondrial purification.
<ol type="a">
<li>Centrifuge lysate at 1000 × <i>g</i> for 10 min at 4°C and transfer supernatant to a new sterile 1.5 mL tube.
<div class="optional">
<span class="optional-title">Optional:</span> Repeat this step to reduce the probability of intact cells contaminating the mitochondrial preparation.</div>
<div class="note">
<span class="note-title">Note:</span> Pellet will be diffuse and easily disrupted. Take care in removing supernatant to minimize the mass of pellet that is carried over to the next step.</div>
<div class="note">
<span class="note-title">Note:</span> Supernatants should be combined at this point.</div>
</li>
<li>Centrifuge the supernatant at 6000 × <i>g</i> for 10 min at 4°C. Carefully aspirate supernatant.
<div class="note">
<span class="note-title">Note:</span> The pellet now contains isolated mitochondria.</div>
</li>
<li>Resuspend the pellet with 1 mL Mitochondrial Storage Buffer using a 1 mL pipette tip and centrifuge at 6000 × <i>g</i> for 20 min at 4°C.
<div class="critical">
<span class="critical-title">Critical:</span> Prepare the MitoPunch apparatus during this centrifugation step to minimize time between resuspension of mitochondrial isolate and MitoPunch transfer as MitoPunch efficiency is negatively impacted by older mitochondrial preparations.</div>
</li>
<li>Remove supernatant. The mitochondrial mass should appear as a compact, pin-head-sized pellet at the bottom of the 1.5 mL tube.</li>
<li>Resuspend the mitochondrial pellet in a volume of DPBS with calcium and magnesium (Thermo Fisher Scientific, Cat#14040133) equal to 120 μL times the number of MitoPunch replicates and keep on ice.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">MitoPunch mitochondrial transfer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>The following instructions describe the process of delivering freshly isolated mitochondria to recipient cells using the MitoPunch apparatus. Briefly, the mechanism by which MitoPunch delivers isolated mitochondria to recipient cells requires pressure generated by a solenoid-driven mechanical plunger. Isolated mitochondria are loaded into a PDMS reservoir to which a cell culture filter insert seeded with recipient cells is sealed. The solenoid is activated which drives the mechanical plunger into the underside of the PDMS reservoir, causing the internal pressure of the reservoir to increase. This increase in pressure forces mitochondrial suspension directly through the pores in the filter insert and into the recipient cell cytoplasm. We have previously quantified the transfer efficiency of MitoPunch by delivering fluorescently labeled isolated mitochondria to 143 BTK− ρ0 and BJ ρ0 recipient cells and measuring the fraction of the recipients positive for the fluorescent marker by imaging flow cytometry (<a href="#bib6">Sercel et al., 2021</a>). We observe ∼50% of 143 BTK− ρ0 recipient cells and ∼15% of BJ ρ0 recipients as positive for the fluorescent mitochondria. Additionally, we measured the mean and median number of fluorescent mitochondrial puncta present in recipient cells by imaging flow cytometry. We observed a mean of ∼5 and a median of ∼3 puncta for 143 BTK− ρ0 recipients and a mean of ∼2 and a median of ∼1 puncta for BJ ρ0 recipients. Finally, it is important to keep the isolated mitochondrial sample on ice while performing MitoPunch transfers and to minimize the time that isolated mitochondria are kept on ice prior to transfer.</p>
<ol start="4">
<li>Spray 70% ethanol into a tissue and wipe down the MitoPunch apparatus before entering it into a biosafety cabinet.</li>
<li>Connect MitoPunch apparatus to breadboard and connect breadboard to power supply (<a href="#fig1">Figure 1</a>B). Connect power supply to the 120 V wall outlet and ensure device has power by testing piston actuation.</li>
<li>Remove a sterile PDMS reservoir from an autoclave bag and place on MitoPunch apparatus.</li>
<li>Wash PDMS reservoir 3 times with 120 μL DPBS with calcium and magnesium, pipetting up and down 3 times with each wash, being sure to cover the entire surface area of the reservoir.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step is important to remove any traces of bleach or ethanol from pre-autoclave sterilization and to help the hydrophobic PDMS more readily hold the mitochondrial suspension evenly without pooling into a large droplet.</div>
<ol start="8">
<li>Pipette 120 μL mitochondrial suspension or DPBS control into the PDMS reservoir.</li>
<li>Remove 12-well dish of recipient-cell-seeded filter inserts from the incubator and enter it into the biosafety cabinet. Using sterile forceps, remove one filter insert and secure it into the top plate of the MitoPunch apparatus.
<ol type="a">
<li>To secure, remove the top internal ring from the upper plate of the MitoPunch and place the plastic wings of the filter insert on the remaining ring such that the filter insert is supported within the hole of the top plate. Carefully screw the top internal ring until tight against the filter wings.</li>
</ol>
</li>
<li>Lower the top plate along the optomechanical assembly rods such that the filter insert contacts the PDMS reservoir and ensure it is firmly pressed against the PDMS, creating a seal.</li>
<li>Actuate the MitoPunch piston using the switch on the breadboard. Leave the piston in the protracted position for 3 s, release the piston switch, remove the filter insert from the apparatus, and replace it in the 12-well dish in its original medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Apply gentle pressure to the top plate with a hand while actuating the solenoid to maintain a seal between the filter insert and the PDMS.</div>
<ol start="12">
<li>After completing all MitoPunch transfers, place the 12-well dish in a humidified 37°C incubator for 2 h prior to collecting the recipient cells.</li>
</ol>
<h3 id="sec3.3">Recipient cell collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Mitochondrial recipient cells need to be harvested and plated on 10 cm dishes after the 2 h incubation that follows mitochondrial transfer for expansion and selection.</p>
<ol start="13">
<li>Aspirate medium from the wells containing filter inserts.
<ol type="a">
<li>Carefully aspirate medium from within the filter insert, being mindful to not contact the cell layer on the PET membrane.</li>
<li>Aspirate medium from outside the filter insert, making sure to collect the medium retained at the contact point between the bottom of the filter insert and the well.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The PET filter can retain medium and reduce the efficacy of dissociation reagents if not handled properly.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Change aspirator tip between samples to avoid contamination of intact cells from the mitochondrial preparation that may be present on the outside of the filter insert.</div>
<ol start="14">
<li>Wash filter inserts with DPBS without calcium and magnesium.
<ol type="a">
<li>Apply 0.5 mL DPBS gently to the filter insert and 1.5 mL DPBS to the well outside the filter insert.</li>
<li>Carefully aspirate DPBS from within the filter insert and then from outside the insert, being sure to remove any residual liquid retained between the bottom of the filter and the well.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that DPBS reaches both sides of the filter insert.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Change aspirator tip between samples to avoid contamination of intact cells from the mitochondrial preparation that may be present on the outside of the filter insert.</div>
<ol start="15">
<li>Apply Accutase dissociation reagent to the filters and incubate for 5 min in a 37°C humidified incubator.
<ol type="a">
<li>Pipette 0.5 mL Accutase to the filter insert and 1.5 mL to the well outside the insert.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that Accutase reaches both sides of the filter insert.</div>
<div class="note">
<span class="note-title">Note:</span> Visually inspect the plate under an inverted microscope to verify that the cells have released from the filter. If dissociation is incomplete (<a href="#fig4">Figure 4</a>B), return plate to the incubator for 3 min and repeat until cells have released from the membrane (<a href="#fig4">Figure 4</a>C).</div>
<ol start="16">
<li>Once the cells are released from the membrane, apply 0.5 mL uridine-supplemented medium to the filter insert and gently disrupt the cell layer with a 1 mL pipette tip.
<ol type="a">
<li>Do not draw liquid from outside the filter to reduce the risk of intact mitochondrial donor cell contamination.</li>
</ol>
</li>
<li>Pipette the full volume from the filter insert to a 10 cm dish containing 10 mL warm uridine-supplemented complete medium. If few cells are observed in the dish, see <a href="#sec6.7">troubleshooting 4</a>.</li>
<li>Place 10 cm dishes in a 37°C humidified incubator.</li>
</ol>
<h3 id="sec3.4">SIMR clone selection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>Recipient cells need to be cultured in a dialyzed, nucleotide-free selection medium to isolate stable mitochondrial recipient cells. The selection medium is not permissive for ρ0 cells as they are incapable of generating nucleotides <i>de novo</i>. A negative ρ0 control is essential for determining the endpoint of the following selection protocol.</p>
<ol start="19">
<li>Incubate the 10 cm dishes of MitoPunch recipient cells in a 37°C humidified incubator with uridine-supplemented complete medium for 3 d following transfer.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Monitor confluence daily and do not allow cultures to exceed 80% confluence. If cultures reach &gt;80% confluence, proceed with step 20.</div>
<ol start="20">
<li>Exchange the complete medium for SIMR selection medium.
<ol type="a">
<li>Exchange the medium daily with 10 mL fresh SIMR selection medium and record the confluence of the ρ0 negative control.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Small colonies will form on the MitoPunch recipient plates and be visible by microscopy typically 3–5 d into SIMR selection medium treatment.</div>
<ol start="21">
<li>Continue daily feeding with SIMR selection medium until the ρ0 control plate is no longer viable, typically 5–7 d under SIMR selection medium treatment.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Proceed to either clone isolation or crystal violet staining as soon as the control plate is no longer viable.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> When the recipient cells used were determined to require galactose SIMR selection medium in Test recipient cell sensitivity to different SIMR clone selection media conditions, proceed to step 22.</div>
<ol start="22">
<li>At 5 d post-delivery, feed cells with galactose SIMR selection medium.
<div class="note">
<span class="note-title">Note:</span> Follow selection scheme developed in “Test recipient cell sensitivity to different SIMR clone selection media” for the specific recipient cells used.</div>
<ol type="a">
<li>Selection medium should be exchanged daily and condition of the cells noted.</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Continue galactose selection until the DPBS control transfer has completely died, up to 7 d post-transfer.
<ol type="a">
<li>After death of the control, exchange the selection medium for SIMR selection medium to allow clones to grow large enough to harvest.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When selection fails to slow cell growth and induce cell death, see <a href="#sec6.9">troubleshooting 5</a>. In the event no clones appear after selection, see <a href="#sec6.11">troubleshooting 6</a>.</div>
<h3 id="sec3.5">SIMR clone harvesting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>After the selection process, isolate SIMR clones from 10 cm dishes and passage them onto 6-well dishes for expansion. This must be done before clones overgrow as large clones can merge with neighboring clones or become internally over confluent and die.</p>
<ol start="24">
<li>Select clones for isolation that are not adjacent to or in contact with other clones.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When selecting multiple clones, mark the location of desired clones with a permanent marker or a microscope mounted object marker.</div>
<ol start="25">
<li>Aspirate the medium from the dish and wash with 5 mL of DPBS. Aspirate the DPBS.</li>
<li>For each clone to be isolated, coat one side of a cylinder in sterilized high-vacuum grease. Gently adhere each cylinder to the dish and fully encompass one clone within the cylinder.</li>
<li>Slowly add 200 μL of Accutase to each cylinder and incubate for 5 min in a 37°C humidified incubator.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Watch for any leaks from the bottom of the cylinder while adding dissociation reagent and immediately stop when one appears. Any leaked dissociation reagent can prematurely disrupt other clones adhered to the plate.</div>
<ol start="28">
<li>After incubation, verify that the cells have fully dissociated from the dish under an inverted microscope.
<ol type="a">
<li>When the cells have not fully dissociated, incubate for another 3 min.</li>
</ol>
</li>
<li>Once the cells have fully dissociated, use a micropipettor to move the cell suspension into a well of a 6-well dish containing 2 mL of warm complete medium.</li>
<li>Incubate the newly plated clones in a 37°C humidified incubator and continue to expand until ready for use in downstream analysis.</li>
</ol>
<h3 id="sec3.6">Crystal violet staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This protocol enables visualization of clone morphology, number, and distribution. Plates treated with this protocol can be stored indefinitely away from light. Additionally, when stain does fade with time, plates can be re-stained with additional crystal violet to increase color saturation for future imaging.</p>
<ol start="31">
<li>Aspirate medium from the plate, taking care to not disturb clones.</li>
<li>Apply 1 mL of freshly diluted 4% paraformaldehyde to the dish and tilt the dish to fully coat the surface.</li>
<li>Incubate at RT in a chemical fume hood for 15 min.</li>
<li>Aspirate the 4% paraformaldehyde from the dish and dispose of in the appropriate chemical waste container.</li>
<li>Apply 2 mL of crystal violet solution to each plate and tilt to fully coat. Incubate at RT in the chemical fume hood for 30 min.</li>
<li>Remove excess crystal violet using a serological pipette and dispose of it in the appropriate chemical waste container.</li>
<li>Rinse each plate twice with DI water to remove excess crystal violet solution.</li>
<li>Dry plates upside-down on an absorbent surface to allow any residual crystal violet solution to drip off the plate.</li>
<li>Store plates away from light until ready to analyze.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Mitochondrial preparations performed as written in this protocol typically yield 60–100 μg of isolated protein per T-225 plate of donor cells. MitoPunch transfer of HEK293T mitochondria into 143BTK− ρ0 cells can generate between dozens to hundreds of independent SIMR clones (<a href="#fig5">Figure 5</a>). Other cell lines used as donor or recipient cells, particularly transferring mutant mtDNA-bearing mitochondria, can reduce the SIMR clone yield (<a href="#bib5">Patananan et al., 2020</a>; <a href="#bib6">Sercel et al., 2021</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1047-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Crystal violet staining of SIMR clones</div>
<p>Images of Crystal Violet stained SIMR clones generated by MitoPunch transfer of HEK293T mitochondria or DPBS with calcium and magnesium vehicle into 143BTK− ρ0 and NDF ρ0 recipient cells on 10 cm dishes after SIMR clone selection.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Using this protocol, MitoPunch can only deliver isolated mitochondria into adherent cells that can attach to the PET filter membrane. Additionally, MitoPunch is unable to generate SIMR clones in recipient cells that already contain mtDNA unless a secondary antibiotic selection is applied simultaneously with the SIMR restrictive medium scheme (<a href="#bib3">Dawson et al., 2020</a>). The pressure generated by the MitoPunch apparatus as described in this protocol is fixed, and some cell types may require different levels of pressure to generate SIMR clones efficiently (<a href="#bib6">Sercel et al., 2021</a>). MitoPunch delivery into ρ0 primary fibroblasts yields fewer SIMR clones than transfers into transformed ρ0 cells, and SIMR clones isolated from MitoPunch transfer into ρ0 primary fibroblasts can be difficult to expand before the cells reach the Hayflick limit and senesce. Furthermore, mtDNA and nDNA of different haplotypic backgrounds may suffer from biochemical or genetic incompatibilities and reduce the SIMR generation efficiency of MitoPunch transfers using material from such sources. Combinations of mitochondria and recipient cells from different species face incompatibility that prohibits SIMR clone generation. Lastly, isolated donor mitochondria must be freshly isolated prior to every delivery, as extended time in suspension or flash-freezing prior to MitoPunch delivery have a negative effect on SIMR generation efficiency (data not shown).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Construct MitoPunch apparatus: MitoPunch piston is not actuating.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Ensure the electrical wiring of the MitoPunch device is properly connected and remains connected between samples. Always test fire the MitoPunch device before placing the PDMS reservoir on the platform. Additionally, verify that the piston is installed in the correct orientation such that the piston is driven up toward the PDMS reservoir when actuated.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Test recipient cell sensitivity to different SIMR clone selection media conditions: Restrictive medium selection does not reduce cell viability of recipient cells.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>When the recipient cells are not completely mtDNA deficient the restrictive media will not effectively induce cell cycle arrest and death. Verify that the recipient cells are ρ0 prior to beginning MitoPunch using qPCR to measure mtDNA copy number and respirometry to quantify mitochondrial electron transport chain activity.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Mitochondrial preparation: Needle clogs persistently during mechanical isolation step of mitochondrial isolation.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Mitochondrial donor cell cultures that reach 100% confluency cause more clogging than cells from plates at lower confluency. Only grow donor cell cultures to 80%–90% confluence before harvesting for mitochondrial isolation.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Recipient cell collection: Few cells are recovered after MitoPunch and collection from filter inserts.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Carefully remove all cell growth medium from inside, outside, and below the PET filter insert. Medium is easily trapped between the bottom surface of the insert and the 12-well culture dish, and this excess medium can impact the efficacy of the dissociation reagent used to collect cells. Additionally, ensure dissociation reagent is applied to both sides of the filter insert as MitoPunch recipient cells grown on the filters extend projections through the pores in the membrane and must be exposed to reagent from both sides for complete dissociation.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>SIMR clone selection: Restrictive medium selection does not reduce cell viability of recovered MitoPunch recipient cells.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>This problem can be caused by three main factors. The first is allowing the recipient cell cultures to reach 100% confluence before introducing restrictive medium. Begin medium selection before cultures reach 80% confluence. The second is not feeding recipient cell cultures with restrictive medium with sufficient frequency. Apoptotic cells on the dishes will release nucleotides into the medium which can be scavenged by nascent SIMR cells and ρ0 cells alike, leading to constant cell viability throughout the selection process. When daily feeding is insufficient, increase the frequency of media exchange to ensure selection occurs. Third, not all lines are sufficiently sensitive to SIMR selection medium and require galactose SIMR selection medium to induce selection. Follow the steps in Test recipient cell sensitivity to different SIMR clone selection media conditions to determine the sensitivity of the chosen recipient cell line to the selection scheme.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>SIMR clone selection: No SIMR clones are observed following MitoPunch and restrictive medium selection.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>When no SIMR clones are observed, verify that the mitochondrial isolation yielded coupled mitochondria. Perform a mitochondrial isolation and measure the respiration rate of the mitochondrial isolate using a Seahorse Extracellular Flux Analyzer. When the isolation does not yield coupled mitochondria, verify that the buffers used in the isolation protocol are stored at the appropriate temperatures and are supplemented with protease inhibitor.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aguer, C., Gambarotta, D., Mailloux, R.J., Moffat, C., Dent, R., McPherson, R., and Harper, M.E. (2011). Galactose enhances oxidative metabolism and reveals mitochondrial dysfunction in human primary muscle cells. PLoS One <i>6</i>, e28536. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref1">View at publisher</a></p>
<p id="bib2">Berthier, E., Young, E.W., and Beebe, D. (2012). Engineers are from PDMS-land, biologists are from polystyrenia. Lab. Chip <i>12</i>, 1224-1237. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref2">View at publisher</a></p>
<p id="bib3">Dawson, E.R., Patananan, A.N., Sercel, A.J., and Teitell, M.A. (2020). Stable retention of chloramphenicol-resistant mtDNA to rescue metabolically impaired cells. Sci. Rep. <i>10</i>, 14328. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref3">View at publisher</a></p>
<p id="bib4">Gregoire, M., Morais, R., Quilliam, M.A., and Gravel, D. (1984). On auxotrophy for pyrimidines of respiration-deficient chick embryo cells. Eur. J. Biochem. <i>142</i>, 49-55. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref4">View at publisher</a></p>
<p id="bib7">Miyata, N., Steffen, J., Johnson, M.E., Fargue, S., Danpure, C.J., and Koehler, C.M. (2014). Pharmacologic rescue of an enzyme-trafficking defect in primary hyperoxaluria 1. Proceedings of the National Academy of Sciences <i>111</i>, 14406-14411. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref7">View at publisher</a></p>
<p id="bib5">Patananan, A.N., Sercel, A.J., Wu, T.H., Ahsan, F.M., Torres, A., Kennedy, S.A.L., Vandiver, A., Collier, A.J., Mehrabi, A., Van Lew, J., et al. (2020). Pressure-driven mitochondrial transfer pipeline generates mammalian cells of desired genetic combinations and fates. Cell Rep. <i>33</i>, 108562. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref5">View at publisher</a></p>
<p id="bib6">Sercel, A.J., Patananan, A.N., Man, T., Wu, T.H., Yu, A.K., Guyot, G.W., Rabizadeh, S., Niazi, K.R., Chiou, P.Y., and Teitell, M.A. (2021). Stable transplantation of human mitochondrial DNA by high-throughput, pressurized isolated mitochondrial delivery. Elife <i>10</i>, e63102. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00556-6/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Michael A. Teitell (<a href="mailto:mteitell@mednet.ucla.edu">mteitell@mednet.ucla.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents. Dimensions of custom fabricated parts necessary to replicate this work are included in the text of this protocol.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze new code or datasets.</p>
<h3>Acknowledgments</h3>
<p>A.J.S. is supported by the <a href="https://doi.org/10.13039/100000002">NIH</a> (T32GM007185 and T32CA009120). A.N.P. is supported by the <a href="https://doi.org/10.13039/100000002">NIH</a> (T32CA009120) and <a href="https://doi.org/10.13039/100000968">American Heart Association</a> (18POST34080342). P.-Y.C. is supported by the <a href="https://doi.org/10.13039/100000001">NSF</a> (CBET 1404080), the <a href="https://doi.org/10.13039/100000002">NIH</a> (RO1GM114188), and the <a href="https://doi.org/10.13039/100000181">Air Force Office of Scientific Research</a> (FA9550-15-1-0406). M.A.T. is supported by the <a href="https://doi.org/10.13039/100000181">Air Force Office of Scientific Research</a> (FA9550-15-1-0406), the <a href="https://doi.org/10.13039/100000005">Department of Defense</a> (W81XWH2110139), the <a href="https://doi.org/10.13039/100000002">NIH</a> (R01GM114188, R01GM073981, R01CA185189, R21CA227480, R01GM127985, and P30CA016042), and <a href="https://doi.org/10.13039/100000900">CIRM</a> (RT3-07678). We thank Yen-Ju Lin, Xiang Zhang, and Marvin Tan for their assistance with polymer fabrication.</p>
<h3>Author contributions</h3>
<p>A.J.S. and A.J.N. drafted and edited the manuscript and figures. T.-H.W., P.-Y.C., and M.A.T. developed the core MitoPunch technology. A.J.S., A.N.P., and T.-H.W. optimized and implemented the protocol. P.-Y.C. provided funds and infrastructure to conduct the work. M.A.T. provided funds and infrastructure to conduct the work, edited the manuscript, and advised throughout the optimization of the protocol.</p>
<h3>Declaration of interests</h3>
<p>P.-Y.C. and M.A.T. are co-founders, board members, shareholders, and consultants for NanoCav, LLC. Patents related to this technology include (1) Efficient delivery of large cargos into cells on a porous substrate, issued Nov. 12, 2019, US 10,472,651 (UCLA); and (2) Mechanical transfection devices and methods, issued Sep. 1, 2020, US 10,760,040 (NanoCav). The other authors declare no competing interests.</p>
</section>
</article>