<article>
<section>
<h1 id="header">Population imaging of synaptically released glutamate in mouse hippocampal slices</h1>
<p><time datetime="2021-11-12">Published: November 12, 2021</time></p>
<p>Felix Unger,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Arthur Konnerth,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#cor1">*</a></sup> and Benedikt Zott<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Neuroscience, Technical University of Munich, 80802 Munich, Germany</p>
<p id="aff2"><sup>2</sup>Munich Cluster for Systems Neurology, Technical University of Munich, 80802 Munich, Germany</p>
<p id="aff3"><sup>3</sup>Department of Neuroradiology, Klinikum Rechts der Isar, Technical University of Munich, 81675 Munich, Germany</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:arthur.konnerth@tum.de">arthur.konnerth@tum.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:benedikt.zott@tum.de">benedikt.zott@tum.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100877">10.1016/j.xpro.2021.100877</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Glutamatergic neurotransmission is a widespread form of synaptic excitation in the mammalian brain. The development of genetically encoded fluorescent glutamate sensors allows monitoring synaptic signaling in living brain tissue in real time. Here, we describe single- and two-photon imaging of synaptically evoked glutamatergic population signals in acute hippocampal slices expressing the fluorescent glutamate sensor SF-iGluSnFR.A184S in CA1 or CA3 pyramidal neurons. The protocol can be readily used to study defective synaptic glutamate signaling in mouse models of neuropsychiatric disorders, such as Alzheimer disease.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref21">Zott et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Single- or two-photon imaging of synaptically evoked population glutamate transients</li>
<li>Instructions for pre- or post-synaptic targeting by in vivo viral transduction</li>
<li>Combination of glutamate imaging with electrophysiology and pharmacology</li>
<li>Application of population glutamate imaging in mouse models of Alzheimer disease</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Here we describe a protocol for fluorometric glutamate imaging at the Schaffer collateral - CA1 pyramidal neuron synapses in the stratum radiatum of acute mouse hippocampal slices. Most likely, these are the best studied synapses of the mammalian brain and a prime preparation for the investigation of learning-dependent synaptic plasticity (<a href="#bib14">Martin and Clark, 2007</a>; <a href="#bib17">Moser et al., 2008</a>) and its alterations in dementias such as Alzheimer’s disease. Our protocol provides a widely accessible tool for the study of synaptically-evoked glutamate transients in wild-type or transgenic mice. The stratum radiatum is particularly suitable for population imaging because of a dense and exclusive presence of similar synaptic connections. A direct comparison of post- and presynaptic sensing of glutamate is readily feasible through the selective targeting of glutamate sensor expression in postsynaptic CA1 pyramidal neurons or presynaptic CA3 pyramidal neurons, respectively. To enable the detection of small glutamate transients elicited by single action potentials, we based our approach on the recently developed fluorescent glutamate sensor SF-iGluSnFR.A184S with a high affinity for glutamate. We also demonstrate how to combine population glutamate imaging with simultaneous electrophysiological recordings. For example, the protocol described here is suitable for the detection of extracellular glutamate transients, which represent a direct correlate of electrophysiologically recorded field potentials. We illustrate the technique with example experiments using single- or two-photon imaging.</p>
<h3 id="sec1.1">Virus injection in the hippocampus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h, 20–55 days before the imaging experiment</div>
<p>In the first step, the viral construct of the extracellular, membrane-bound protein SF-iGluSnFR is delivered in vivo through a glass pipette under stereotactic control (<a href="#fig1">Figure 1</a>). This is a reliable and well-established procedure. For population glutamate imaging with a high signal-to-noise ratio, a dense labeling of the CA1 pyramidal neurons is necessary to collect the signal from as many spiny dendrites as possible. Details on the brain injection of viral constructs have been described extensively elsewhere (e.g. <a href="#bib13">Lowery and Majewska, 2010</a>). In case of failed injections, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.1">problem 1</a>, <a href="#sec7.3">2</a>, and <a href="#sec7.5">3</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Experimental timeline</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Experimental procedures should be performed in compliance with institutional animal welfare guidelines. The in vivo experiments described here were explicitly approved by the Government of Oberbayern and constantly overseen by a veterinarian.</div>
<ol>
<li>Preparations:
<ol type="a">
<li>Virus preparation: Thaw the vial with the viral construct and dilute it in PBS to the desired concentration.
<div class="note">
<span class="note-title">Note:</span> In our hands, concentrations of around 1–3 <b>×</b> 10<sup>12</sup> genomic copies (GC) of AAV1.hSyn.SF-iGluSnFR.A184S per ml yielded a reliable dense expression, but other concentrations might work equally well. Keep the stock solution at -80°C. The diluted viral construct solution can be stored at 4°C for up to six months.</div>
</li>
<li>Preparation of injection pipettes:<br> For injection of the viral constructs, use capillary pipettes with 1 μL ring marks originally used for blood collection (see <a href="#key-resources-table">key resources table</a>), pull these pipettes in a single step in a Narishige PC-10 vertical puller, and carefully trim the tip under a dissection microscope to the desired size using fine scissors. This procedure robustly produces pipettes with a fine tip (about 30 μm) and a long taper (2–3 cm).</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Surgical procedures:
<ol type="a">
<li>Preparation of the animal: Anesthetize a mouse with isoflurane in pure O<sub>2</sub> (2% for induction, 1.0%–1.4% during the surgical procedure) and place it in a stereotactic frame. Apply Metamizol (200 mg/kg body weight) and Metacam (1.5 mg/kg body weight) subcutaneously and Lidocain (2%, 50–100 μL) under the scalp for analgesia. Continuously monitor and adjust the mouse temperature (optimal: 37 ± 0.5°C) throughout the surgical procedure.
<div class="note">
<span class="note-title">Note:</span> Male and female mice of all ages can be used for this protocol. However, slices prepared from mice older than 2–3 months might require alternative slicing procedures to ensure viability (see <a href="#troubleshooting">troubleshooting</a> section, <a href="#sec7.7">problem 4</a>). Moreover, the stereotactic coordinates of CA1 may differ from those indicated here for mice of other ages or genetic backgrounds. We used 20 to 30 day-old wildtype C57Bl/6N mice (<a href="#fig1">Figure 1</a>). Animals of the APP23xPS45 mouse model of β-amyloidosis were used at the age of around 200 days.</div>
</li>
<li>Remove the hair from the surgical area using an electrical razor and make a midline incision using a scalpel. Push all connective tissue aside and mark the bregma and lambda coordinates (<a href="#fig2">Figure 2</a>A) with a surgical skin marker.
<div class="critical">
<span class="critical-title">Critical:</span> Optimal levelling of the skull is essential when targeting the narrowly confined hippocampal subregions. If you have difficulties at this step, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.3">problem 2</a>).</div>
</li>
<li>
<b>Mark</b> on the skull the entry point of the injection pipette with a surgical skin marker (anterior-posterior (AP): - 2.75 mm and medial-lateral (ML): 3.5 mm for CA1; AP: -2.35 mm, ML 2.70 mm for CA3). Cautiously perform a craniotomy (diameter about 300 μm). Use a small dental drill to thin the skull at the marked coordinates (<a href="#fig2">Figure 2</a>A) and carefully break the center with a syringe needle (e.g., 30G). Importantly, avoid penetrating the dura with the drill.</li>
<li>Viral construct injection at three depths:
<ol type="i">
<li>Fill 1.5 μl of viral construct into a pipette. Mount it to the manipulator attached to the stereotactic frame and insert the tip into the brain, thus penetrating the dura. <a href="#fig2">Figure 2</a>B indicates the depths of the viral construct injection sites for the CA1 and the CA3 regions, respectively.</li>
<li>Start by moving the pipette forward in z-direction 200 μm below the respective deepest injection site of 3.2 mm for CA1 or 3.3 mm for CA3 (<a href="#fig2">Figure 2</a>B) to create a small cavity for construct retention. Then, retract the tip back to the deepest injection site at 3.0 mm for CA1 or 3.1 mm for CA3. Inject slowly 100 nl viral construct solution (optimal speed: 10–20 nl/min) using a 20 ml syringe connected to the pipette via a silicon tube.
<div class="critical">
<span class="critical-title">Critical:</span> It will be difficult to maintain a constant injection speed. If you injected the target amount of 100 nl too quickly, wait for at least five minutes to ensure an average speed of 20 nl/min for sufficient diffusion of the viral construct solution into the tissue. If you accidentally injected larger amounts of fluid or air bubbles into the brain, abort the experiment and sacrifice the animal.</div>
</li>
<li>Retract the pipette sequentially to the other two injection sites and deliver at each site again 100 nl viral construct solution.
<div class="note">
<span class="note-title">Note:</span> Although a single injection can produce a sufficient level of expression in a small transverse hippocampal section, we recommend injections at the indicated three depths to increase the length of this section and, thus, the number of usable slices per experimental animal.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid any lateral movement of the pipette to minimize tissue damage.</div>
</li>
</ol>
</li>
<li>
<b>Surgery termination and recovery:</b> Slowly remove the pipette. If more than 300 μl viral solution is left in the pipette, it can be used for delivery to a second mouse. Close the skin above the craniotomy using surgical suture material and tissue adhesive. It is not necessary to seal the skull. Stop isoflurane delivery and place the mouse into a cage on a heating pad for postsurgical recovery. Monitor mouse vitals and fitness regularly until the day of the experiment.
<div class="optional">
<span class="optional-title">Optional:</span> To enable the identification of single dendrites or axons in order to visualize the location of the SF-iGluSnFR on the membrane of the respective neurons in CA1 or CA3, you can use a sparse labeling protocol (<a href="#bib15">Marvin et al., 2013</a>). For this purpose, mix AAV2/1.hSyn.FLEX-SF-iGluSnFR.A184S (diluted in PBS to a final concentration of 1.8 <b>×</b> 10<sup>12</sup> GC/ml) and AAV1.hSyn.Cre.WPRE.hGH (diluted in PBS to 1.5 <b>×</b> 10<sup>8</sup> GC/ml) at the ratio of 1:1 and co-apply them through the same pipette.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Expression of the SF-iGluSnFR may be detected at low levels as early as 7–10 days after the injection, but we recommend performing the imaging experiment at least 20–30 days after injection to ensure a high level of stable dense (<a href="#fig3">Figures 3</a>A and 3B) or sparse (<a href="#fig3">Figures 3</a>C and 3D) expression. We have not noticed major changes in the level of SF-iGluSnFR expression up to 55 days after injection.</div>
</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Coordinates for the injection of viral constructs in the mouse hippocampus</div>
<p>(A) Schematic of the exposed skull as seen from above. Bregma and lambda are indicated for orientation. The coordinates of the craniotomy for injection of viral constructs in CA1 (left, black) and CA3 (right, blue) are depicted (circles).</p>
<p>(B) Top: Schematic of the coronal view of the brain at the locations depicted in (A). The dashed lines indicate the pipette tracks in the tissue (black: CA1, blue: CA3). Release sites are marked with an x. Bottom left: Schematic of a hippocampal slice with the expected region of SF-iGluSnFR expression in CA1 depicted in green. Bottom right: Expected region of SF-iGluSnFR expression in CA3 (green). AP, anterior-posterior; CA1, cornu ammonis 1; CA3, cornu ammonis 3; ML: medio-lateral.</p>
<p>(C) Linearized sequence map for pAAV.hSyn.SF-iGluSnFR.A184S (6324 base pairs), generated from <a href="#bib16">Marvin et al. (2018)</a> and the data deposited at Addgene (#106174-AAV1). Green arrows indicate transcribed SF-iGluSnFR parts, small gray arrows indicate the open reading frames.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Expression of SF-iGluSnFR in CA1 or CA3</div>
<p>(A) Confocal image of a hippocampal slice expressing SF-iGluSnFR (green) in CA1 (dashed red line). NeuroTrace counterstaining (cyan).</p>
<p>(B) same as (A) for SF-iGluSnFR expression in CA3 with DAPI counterstaining (cyan). Inset: Confocal image from a different slice of SF-iGluSnFR-expressing axons in the hippocampal CA1 region.</p>
<p>(C) Confocal image of a hippocampal slice with sparse expression of SF-iGluSnFR (green) in CA1 with NeuroTrace counterstaining (cyan). The inset in the bottom right corner is a magnification of the area within the dashed white line.</p>
<p>(D) Same as (C) for sparse expression of SF-iGluSnFR in CA3.</p>
<p>All images are intensity-maximized z-projections. Scale bars in (A–D), 250 μm, inset scale bars, 50 μm. CA 1-3, cornu ammonis 1-3; DG, dentate gyrus.</p>
</figcaption>
</figure>
<h3 id="sec1.2">Reagent and pipette preparation on the day of experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="3">
<li>Prepare artificial cerebrospinal fluid (aCSF), sucrose-based cutting solution (CS) and slice holding solution (SHS) according to the recipes in the <a href="#materials-and-equipment">materials and equipment</a> section.</li>
<li>Keep aCSF and SHS at room temperature (20°C–25°C) and cool the CS to 0°C–4°C. Constantly bubble all three solutions with carbogen gas (95% O<sub>2</sub>, 5% CO<sub>2</sub>).</li>
<li>Prepare pipettes for stimulation and, if needed, recording of field excitatory postsynaptic potentials (fEPSP) or drug application. For all purposes, pipettes were pulled from glass capillaries using a Narishige PC10 vertical puller in two steps. When using 2 M NaCl as pipette solution, the resistance was &lt; 1 MΩ.</li>
</ol>
<h3 id="sec1.3">Preparation of acute brain slices containing SF-iGluSnFR-expressing neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Prepare acute hippocampal slices according to standard procedures (<a href="#bib4">Edwards et al., 1989</a>) from a virus-injected mouse 20–55 days after injection of the viral construct (<a href="#fig1">Figure 1</a>). We recommend the use of 300 μm thick horizontal slices.</p>
<ol start="6">
<li>Expose the experimental animal to CO<sub>2</sub> or isoflurane until the nociceptive reflexes extinguish.</li>
<li>Decapitate the animal. Extract the brain and drop into the cold CS.</li>
<li>Cut 300 μm horizontal slices in a vibratome (recommended cutting speed: 0.1 mm/s).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Other slice orientations should work equally well. However, if the injection is performed at three different depths as described above, it is advisable to cut the slices horizontally, i.e. perpendicular to the pipette tract. This way, multiple slices per animal can be used for imaging.</div>
<ol start="9">
<li>Use a pipette to transfer the slices into the SHS-filled slice holding chamber (a plastic mesh fixed on a ring made from the lid of a small petri dish, for specifications see (<a href="#bib4">Edwards et al., 1989</a>)).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Placing the slices in the holding chamber in a distinct order (e.g. clockwise, starting from the top) can facilitate slice identification and selection (see below).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Let the slices recover for at least one hour in the slice holding chamber at room temperature before beginning the imaging experiment. Make sure that the brain slices stay oxygenated.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>AAV1.hSyn.SF-iGluSnFR.A184S</td>
<td><a href="#bib16">Marvin et al., 2018</a></td>
<td>Gift from Loren LoogerPlasmid available at:Addgene Cat#106174-AAV1</td>
</tr>
<tr>
<td>AAV2/1.hSyn.FLEX.SF-iGluSnFR.A184S</td>
<td><a href="#bib16">Marvin et al., 2018</a></td>
<td>Gift from Loren LoogerPlasmid available at:Addgene Cat#106180-plasmid</td>
</tr>
<tr>
<td>AAV1.hSyn.Cre.WPRE.hGH</td>
<td>Penn Vector Core</td>
<td>Originally #p2676Now available at:Addgene Cat#105553-AAV1</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Isofluran CP 1 mL/mL</td>
<td>CP-Pharma</td>
<td>Cat#1214</td>
</tr>
<tr>
<td>Xylocain 2%</td>
<td>Covetrus</td>
<td>Cat#778-805</td>
</tr>
<tr>
<td>Meloxicam 5 mg/mL (Meloxidyl)</td>
<td>Covetrus</td>
<td>Cat#796672</td>
</tr>
<tr>
<td>Metamizol-Natrium 500 mg/mL (Vetalgin)</td>
<td>Covetrus</td>
<td>Cat#770618</td>
</tr>
<tr>
<td>Lidocainhydrochlorid (Xylocain) Gel 2%</td>
<td>Mycare.de</td>
<td>Cat#01138060</td>
</tr>
<tr>
<td>Dexpanthenol 5% (Bepanthen)</td>
<td>Mycare.de</td>
<td>Cat#01578675</td>
</tr>
<tr>
<td>Methoxy-X04 (optional)</td>
<td>Tocris</td>
<td>Cat#4920</td>
</tr>
<tr>
<td>S26C-β-amyloid (1-40) dimer (optional)</td>
<td>JPT Peptide Technologies GmbH</td>
<td>Cat#SP-Ab-24_0.1</td>
</tr>
<tr>
<td>Betaisadona Povidon-Iod (Mundipharma)</td>
<td>Mycare.de</td>
<td>Cat#01931491</td>
</tr>
<tr>
<td>Thiourea</td>
<td>Sigma-Aldrich</td>
<td>Cat#T7875</td>
</tr>
<tr>
<td>Na-Ascorbate</td>
<td>Sigma-Aldrich</td>
<td>Cat#A7631</td>
</tr>
<tr>
<td>Na-Pyruvate</td>
<td>Sigma-Aldrich</td>
<td>Cat#P2256</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat#84100</td>
</tr>
<tr>
<td>Ascorbic acid</td>
<td>VWR</td>
<td>Cat#83568.180</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse C57Bl/6N</td>
<td>Charles River</td>
<td>Strain code 027</td>
</tr>
<tr>
<td>Mouse APP23xPS45 (optional)</td>
<td><a href="#bib22">Busche et al., 2008</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>LabVIEW</td>
<td>National Instruments</td>
<td><a href="http://www.ni.com/download-labview">www.ni.com/download-labview</a></td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td><a href="http://www.mathworks.com/downloads">www.mathworks.com/downloads</a></td>
</tr>
<tr>
<td>Igor Pro</td>
<td>WaveMetrics</td>
<td><a href="http://www.wavemetrics.com/order/order_igordownloads.htm">www.wavemetrics.com/order/order_igordownloads.htm</a></td>
</tr>
<tr>
<td>Patcher’s Power Tools Igor Toolbox</td>
<td>Francisco Mendez, Frank Würriehausen</td>
<td><a href="http://www3.mpibpc.mpg.de/groups/neher/index.php?page=software">www3.mpibpc.mpg.de/groups/neher/index.php?page=software</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Capillary pipettes for virus injection</td>
<td>Hirschmann</td>
<td>Ringcaps 1–5 μL; Cat#9600105</td>
</tr>
<tr>
<td>Vertical pipette puller</td>
<td>NARISHIGE</td>
<td>Model PC-10</td>
</tr>
<tr>
<td>Micromotor dental drill system</td>
<td>Osada</td>
<td>Model EXL-M40</td>
</tr>
<tr>
<td>Dissection microscope</td>
<td>Leica</td>
<td>MZ6</td>
</tr>
<tr>
<td>Stereotactic alignment system</td>
<td>David Kopf Instruments</td>
<td>Model 1900</td>
</tr>
<tr>
<td>Centering scope for stereotactic system</td>
<td>David Kopf Instruments</td>
<td>Model 1915</td>
</tr>
<tr>
<td>Micromanipulator with digital display console</td>
<td>David Kopf Instruments</td>
<td>Anilam Wizard 211</td>
</tr>
<tr>
<td>Recovery heating pad</td>
<td>H. Suar Laborbedarf</td>
<td>Cat#13501</td>
</tr>
<tr>
<td>Silk suture (Ethicon Perma-Hand Silk 5-0)</td>
<td>Johnson &amp; Johnson Medical Ltd.</td>
<td>Cat#K890H</td>
</tr>
<tr>
<td>Drill bits</td>
<td>M+W Dental</td>
<td>Steel drill, Size 5, Shaft H; Cat#21800</td>
</tr>
<tr>
<td>Unsterile cotton buds</td>
<td>Diagonal/sBoettger GmbH &amp; Co KG</td>
<td>Cat#09-119-0100</td>
</tr>
<tr>
<td>Hot bead sterilizer</td>
<td>Fine Science Tools</td>
<td>Cat#18000-45</td>
</tr>
<tr>
<td>Dumont #5 forceps</td>
<td>Fine Science Tools</td>
<td>Cat#11251-20</td>
</tr>
<tr>
<td>Noyes spring scissors</td>
<td>Fine Science Tools</td>
<td>Cat#15012-12</td>
</tr>
<tr>
<td>Olsen-Hegar Needle Holders with Suture Cutters</td>
<td>Fine Science Tools</td>
<td>Cat#12502-12</td>
</tr>
<tr>
<td>Narrow pattern forceps</td>
<td>Fine Science Tools</td>
<td>Cat#11002-12</td>
</tr>
<tr>
<td>Scalpel handle</td>
<td>Fine Science Tools</td>
<td>Cat#10003-12</td>
</tr>
<tr>
<td>Scalpel blade (carbon steel, No. 11)</td>
<td>Brinkmann Medical</td>
<td>Cat#BB511</td>
</tr>
<tr>
<td>Tissue adhesive (3M Vetbond)</td>
<td>Fisher Scientific</td>
<td>Cat#10430774</td>
</tr>
<tr>
<td>Semiautomatic vibrating blade microtome (VT 1200 S)</td>
<td>Leica Biosystems</td>
<td>Cat#1491200S001</td>
</tr>
<tr>
<td>Rodent guillotine</td>
<td>World Precision Instruments</td>
<td>Cat#DCAP</td>
</tr>
<tr>
<td>Super glue</td>
<td>UHU</td>
<td>Super Glue Liquid</td>
</tr>
<tr>
<td>Slicing blade</td>
<td>Scherenprofi.com</td>
<td>Cat#9201A</td>
</tr>
<tr>
<td>Fluorescence lamp (X-Cite Series 120Q)</td>
<td>Excelitas Technologies</td>
<td>X-Cite Series model 120Q</td>
</tr>
<tr>
<td>Upright stereomicroscope</td>
<td>Olympus</td>
<td>Model SZX2</td>
</tr>
<tr>
<td>Stereomicroscope objective</td>
<td>Olympus</td>
<td>Model MVPLAPO 0.63<b>×</b>
</td>
</tr>
<tr>
<td>Fluorescence filter system</td>
<td>Olympus</td>
<td>Model U-MGFPHQ/XL</td>
</tr>
<tr>
<td>Borosilicate glass capillaries, filament, fire polished</td>
<td>Hilgenberg</td>
<td>Cat#1807516</td>
</tr>
<tr>
<td>Ti:sapphire laser</td>
<td>Spectra-Physics</td>
<td>Model Mai Tai HP DS</td>
</tr>
<tr>
<td>Transmitting light microscope</td>
<td>Olympus</td>
<td>Model BX51WI</td>
</tr>
<tr>
<td>Hybrid photo detector</td>
<td>Hamamatsu</td>
<td>R11322U-40, Cat#C12929</td>
</tr>
<tr>
<td>Amplifier for electrophysiology</td>
<td>HEKA Elektronik</td>
<td>Model EPC 9</td>
</tr>
<tr>
<td>Isolated pulse stimulator</td>
<td>A-M Systems</td>
<td>Model 2100, Cat#720005</td>
</tr>
<tr>
<td>Pressure application system</td>
<td>npi electronic</td>
<td>Model PDES-01AM</td>
</tr>
<tr>
<td>Water immersion objective (40<b>×</b>)</td>
<td>Nikon</td>
<td>Model NIR Apo 40<b>×</b>/0.8 DIC N2</td>
</tr>
<tr>
<td>Water immersion objective (20<b>×</b>)</td>
<td>Olympus</td>
<td>Model XLUMPPlanFl 20<b>×</b> 0.95 NA</td>
</tr>
<tr>
<td>sCMOS camera</td>
<td>Oxford Instruments</td>
<td>Andor Zyla 4.2; Cat# ZYLA-4.2P-CL10</td>
</tr>
<tr>
<td>Inverted camera objective (focal length, 50 mm) with 2<b>×</b> tele converter</td>
<td>SFE</td>
<td>Auto 2<b>×</b> Tele converter</td>
</tr>
<tr>
<td>488 nm Laser (20 mW)</td>
<td>Coherent</td>
<td>Model Sapphire LP 488</td>
</tr>
<tr>
<td>Bandpass filter 470/40 nm</td>
<td>AHF analysentechnik AG</td>
<td>Cat#F49-470</td>
</tr>
<tr>
<td>Beam splitter 505 nm LP</td>
<td>AHF analysentechnik AG</td>
<td>Cat#F48-504PCR</td>
</tr>
<tr>
<td>Longpass filter 510 nm</td>
<td>AHF analysentechnik AG</td>
<td>Cat#F47-511</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Artificial cerebrospinal fluid (aCSF)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub> (1 M)</td>
<td>2 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>KCl (1 M)</td>
<td>3.5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26 mM</td>
<td>2.1843 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>-monohydrate</td>
<td>1.25 mM</td>
<td>0.17245 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>125 mM</td>
<td>7.305 g</td>
</tr>
<tr>
<td>KCl</td>
<td>2.5 mM</td>
<td>0.1864 g</td>
</tr>
<tr>
<td>Glucose-monohydrate</td>
<td>20 mM</td>
<td>3.964 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Add 1000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">The solution without glucose can be stored 3–4 days in a refrigerator (4°C). Osmolarity without glucose should be 290–295 mosm/l. To prevent contamination with microorganisms, add glucose only on the day of the experiment.</p>
<table id="undtbl2">
<caption>Slice holding solution (SHS)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (2,5 M)</td>
<td>119 mM</td>
<td>4.76 mL</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub> (0,52 M)</td>
<td>26 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>KCl (1 M)</td>
<td>2.5 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub> (0,5 M)</td>
<td>1.25 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> (1 M)</td>
<td>2 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Thiourea (1 M)</td>
<td>2 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1 M)</td>
<td>2 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Na-Ascorbate</td>
<td>5 mM</td>
<td>100 mg</td>
</tr>
<tr>
<td>Na-Pyruvate</td>
<td>3 mM</td>
<td>33 mg</td>
</tr>
<tr>
<td>Glucose-monohydrate</td>
<td>12.5 mM</td>
<td>250 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Add 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Adjust pH to 7.4 with HCl (usually around 2.3 mL of 1 M HCl) before adding H<sub>2</sub>O. A 500 mL stock solution without glucose can be stored for 3–4 days in a refrigerator (4°C). Osmolarity without glucose should be around 290 mosm/L.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Adding 1 mM of glutathione monoethyl ester (Santa Cruz Biotechnology, Cat#sc-203974B) or 5 mM of N-acetyl-N-cysteine (Sigma-Aldrich, Cat#A7250) can protect neurons from oxidative stress (See <a href="#troubleshooting">troubleshooting</a> section, <a href="#sec7.7">problem 4</a>).</div>
<table id="undtbl3">
<caption>Cutting solution (CS)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>105 mM</td>
<td>36 g</td>
</tr>
<tr>
<td>Glucose monohydrate</td>
<td>25 mM</td>
<td>4.9 g</td>
</tr>
<tr>
<td>Ascorbic acid</td>
<td>1.7 mM</td>
<td>0.3 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub> (1 M)</td>
<td>0.5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>7 mM</td>
<td>7 mL</td>
</tr>
<tr>
<td>KCl</td>
<td>2.5 mM</td>
<td>0.185 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub> monohydrate</td>
<td>1.25 mM</td>
<td>0.173 g</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>26 mM</td>
<td>2.183 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>65.5 mM</td>
<td>3.826 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Add 1000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">A stock solution with 1000 mL should be stored for no longer than 3–4 days in a refrigerator (4°C). Osmolarity should be 330 mosm/l.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Critical: Preselection of brain slices for the experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5–10 min per slice</div>
<p>Before performing a population glutamate imaging experiment, it is important to identify slices containing SF-iGluSnFR expressing neurons. As identification of slices with fluorescently labeled neurons one by one is tedious under the high-magnification objective of the experimental setup, we recommend the following procedure:</p>
<ol>
<li>Arrange all slices in a holding chamber that you than place under an upright widefield epifluorescence microscope with a low magnification and a large working distance. Use white light for a first inspection (<a href="#fig4">Figure 4</a>A) and, then, illumination with a blue laser or a halogen lamp, equipped with a GFP filter set consisting of a 470 nm bandpass filter for the excitation light, a 505 nm beam splitter and a 510 nm longpass filter for the emitted fluorescence, to detect the SF-iGluSnFR fluorescence (<a href="#fig4">Figure 4</a>B). Select slices which show fluorescence in the target region (CA1 or CA3).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Preselection of brain slices</div>
<p>(A) Widefield image under white light illumination of the slice holding chamber containing all hippocampal slices from a mouse 20 days after virus injection.</p>
<p>(B) Image of the same slices as shown in (A) for excitation with a mercury lamp and a GFP filter set. Scale bars: 5 mm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The aCSF in the slice holding chamber needs to be bubbled constantly with carbogen. During the selection process, however, it may be necessary to briefly interrupt the gas flow for a better view of the slices. Keep such interruptions as brief as possible.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To avoid bleaching of the fluorescent glutamate sensor and phototoxicity, use minimal levels of fluorescence excitation light levels and short exposure times.</div>
<ol start="2">
<li>Gently collect one of the selected slices with a Pasteur’s pipette and transfer it to the experiment chamber. Secure the slice position in the chamber, for example, with a flattened platinum wire frame with a nylon fiber grid (<a href="#bib4">Edwards et al., 1989</a>).</li>
<li>Keep the remaining slices at room temperature in aCSF bubbled continuously with carbogen. A slice at a time can be collected for imaging for up to 8 h without any obvious decrease of recording quality.</li>
</ol>
<h3 id="sec3.2">Population glutamate imaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h per brain slice, up to four brain slices per day</div>
<p>Glutamate imaging of SF-iGluSnFR-expressing neurons works similarly well with single- or two-photon imaging. For two-photon population imaging (<a href="#fig5">Figures 5</a>A–5C), you can use any standard two-photon microscope which can achieve a high frame rate (see below). Set the excitation light wavelength to 920–940 nm and introduce a beam splitter (705 nm) into the light path to reflect the emitted light into the photomultiplier tube (<a href="#fig5">Figure 5</a>A). For standard imaging recordings, we recommend using a long working-distance 40<b>×</b> water-immersion objective. Our microscope is equipped with a 12 kHz resonance scanner and a 80 MHz Ti:Sapphire laser.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Experimental setups for two- or single-photon imaging of synaptically-evoked glutamate transients</div>
<p>(A) Schematic of the experimental setup for two-photon microscope-based population glutamate imaging in the hippocampal CA1 region.</p>
<p>(B) Representative low-magnification two-photon image of a hippocampal slice expressing SF-iGluSnFR in CA1. The stimulation (Stim) and fEPSP recording pipettes (visible as dark shadows) are positioned in the stratum radiatum. The dashed white line outlines the region of interest (ROI) from which population glutamate transients can be recorded.</p>
<p>(C) Two-photon glutamate population signals of SF-iGluSnFR expressing dendrites of CA1 pyramidal neurons at increasing stimulation strengths (each trace is an average of three transients).</p>
<p>(D) Schematic of the experimental setup for single-photon population glutamate imaging in the hippocampal CA1 region using a sCMOS camera and an inverted camera objective.</p>
<p>(E) Wide-field image of a hippocampal slice expressing SF-iGluSnFR in CA1, imaged through the camera objective. The stimulation pipette is barely visible as a dark shadow under these conditions and is outlined for clarity (white lines). The dashed white rectangle indicates the ROI from which population glutamate transients can be recorded. Inset: Wide field image of a hippocampal slice expressing SF-iGluSnFR in CA1, imaged through a 20<b>×</b> water immersion objective.</p>
<p>(F) Single-photon glutamate population signals of SF-iGluSnFR expressing dendrites of CA1 pyramidal neurons at increasing stimulation strengths.</p>
<p>Scale bars in (B and E), 50 μm, inset scale bar in (E), 20 μm. fEPSP: field excitatory postsynaptic potential, PMT: photo multiplier tube, ROI: region of interest, sCMOS: scientific complementary metal–oxide–semiconductor, Stim: stimulation pipette.</p>
</figcaption>
</figure>
<p>For single-photon imaging (<a href="#fig5">Figures 5</a>D–5F), equip an upright microscope with a 488 nm laser or laser diode for excitation and a fast and sensitive camera with a bit-depth of at least 12-Bit (<a href="#fig5">Figure 5</a>D). Introduce a 505 nm beam splitter into the light path to guide the excitation light onto the sample and a 520 nm longpass filter in front of the camera for the detection of the emitted light. For our recordings, we used a sCMOS ANDOR Zyla 4.2 camera mounted on an Olympus BW51WI scope. The microscope was either equipped with an inverted camera objective with a focal length of 50 mm or a 20× water immersion objective.</p>
<p>In both cases, we recommend imaging frame rates of 30–100 Hz to ensure optimal sampling of the synaptically-evoked glutamate transients. The recording setup has to be equipped with a slice perfusion system mounted on a small stage and a micromanipulator for the positioning of the micropipette used for synaptic stimulation. If needed, a second manipulator for electrophysiological recordings can be added.</p>
<h4 id="sec3.2.1">Preparations</h4>
<ol start="4">
<li>Mount the experiment chamber to the stage under the objective of the microscope, start perfusion with warmed (34°C), carbogen-saturated aCSF (0.5–2 ml/min). Illuminate the slice, for example with a white LED placed near the recording chamber and lower the water-immersion microscope objective into the aCSF. Adjust the focal plane until the slice surface becomes visible through the ocular of your microscope.</li>
<li>Locate the CA1 region by using the outline of the hippocampal slice and densely packed neuronal somata in the stratum pyramidale as guiding structures.</li>
<li>Turn on the single- or two-photon microscope. Inspect the SF-iGluSnFR expression level in the stratum radiatum under single- or two-photon excitation, respectively. Identify the region of interest with dense and homogenous expression of SF-iGluSnFR (<a href="#fig5">Figures 5</a>B and 5E).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For two-photon imaging, set the focal plane at least 50 μm below the slice surface, as superficial neurons may be damaged by the slicing procedure. In the widefield microscope, this is not necessary as you will also collect out-of-focus fluorescence from below and above your focal plane</div>
<div class="critical">
<span class="critical-title">Critical:</span> Try to keep the laser intensity low to prevent phototoxicity. Try to use a laser power below 15 mW (two-photon) and 1 mW (single-photon), when measured under the microscope’s objective.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Acquire an image series along the z-axis (z-stack) for orientation during the analysis of the experiment.</div>
<ol start="7">
<li>Use the micromanipulator to place a stimulation pipette filled with 2M NaCl solution below the objective. Carefully lower the pipette until it touches the slice surface (<a href="#fig5">Figure 5</a>B). Advance the pipette 50–100 μm below the slice surface with the tip in the stratum radiatum at a distance of 75–100 μm from the stratum pyramidale (<a href="#fig5">Figure 5</a>B).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The visualization of the stimulation pipette is facilitated by the addition of a fluorescent dye to the NaCl pipette solution. For example, if your system supports imaging at longer wavelengths (1040 nm and 561 nm for two- and single-photon imaging, respectively), you can add 5–20 μM Alexa 594 to the pipette solution. Alternatively, Alexa 488 (excited at 920 nm and 488 nm, respectively) can be used.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid any lateral movement of the pipette to prevent tissue damage.</div>
<ol start="8">
<li>For simultaneous recordings of fEPSPs, mount another pipette filled with 2M NaCl to the headstage of your extracellular amplifier connected to a second micromanipulator. Advance the pipette tip to a location 50–100 μm below the slice surface and 75–100 μm from the pyramidal layer (<a href="#fig5">Figure 5</a>B), as described in step 7 for the stimulation pipette. Typically, the fEPSP electrode is placed 100–150 μm away from the stimulation pipette towards CA3. Place the reference electrode in the bath solution.</li>
<li>Center the field of view (FOV) to the area surrounding the tip of the fEPSP recording electrode (<a href="#fig5">Figures 5</a>B and 5E). Adjust the focal plane to a depth of about 100–150 μm below the surface. We usually imaged a rectangular FOV with an edge length of about 60–120 μm.</li>
<li>Connect the stimulation pipette and the ground electrode to a stimulus isolator.</li>
<li>Deliver short pulses (80–100 μsec, 1 V) and monitor both the fluorescence changes in a region of interest covering the whole FOV and the electrophysiological responses. With this low level of stimulation strength (1 V), synaptically-evoked signals are generally not detectable.</li>
<li>Repeat step 11 with increasing voltage levels (in increments of 3–5 V) until you can detect a population glutamate signal and a corresponding fEPSP (typically at stimulation strengths of 30–50 V, <a href="#fig5">Figures 5</a>D and 5F). The stimulation settings are optimal if single pulses can evoke robust glutamate transients and fEPSPs without population spikes (<a href="#fig6">Figures 6</a>A and 6B).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Expected outcomes: two- or single-photon imaging of synaptically-evoked population glutamate transients in CA1</div>
<p>(A) Two-photon glutamate population imaging of SF-iGluSnFR expressing dendrites of CA1 pyramidal neurons. Top left: Schematic of the hippocampal slice depicting the locations of the stimulation (Stim) and fEPSP recording pipettes and the region of interest (ROI, dashed line) in the CA1 stratum radiatum containing the dendrites of the virus-expressing neurons (green). Bottom left: representative two-photon image of the stratum radiatum of a hippocampal slice expressing SF-iGluSnFR in CA1. Right: Single sweeps (gray) and average (green) of synaptically-evoked glutamate transients recorded from the ROI indicated in the bottom left. Single sweeps (gray) and average (red) of the evoked fEPSPs recorded in simultaneously with the population glutamate transients. The arrowhead indicates the time of stimulation.</p>
<p>(B) Same as (A) for two-photon glutamate population imaging of SF-iGluSnFR expressing axons from CA3 pyramidal neurons (Schaffer collaterals). The ROI in the stratum radiatum of CA1 (dashed line) contains the axons of SF-iGluSnFR expressing neurons in CA3 (green).</p>
<p>(C) Single-photon glutamate population imaging of SF-iGluSnFR expressing dendrites of CA1 pyramidal neurons. Top left: Schematic of the hippocampal slice. The ROI in the stratum radiatum contains the dendrites of SF-iGluSnFR-expressing neurons in CA1. Bottom left: representative wide-field image of the CA1 region of a SF-iGluSnFR expressing hippocampal slice imaged through a 20<b>×</b> water immersion objective. Dashed line: ROI. Right: Single sweeps (gray) and average (green) of synaptically-evoked glutamate transients recorded with a sCMOS camera from the ROI indicated in the bottom left. Scale bars in (A and B), 5 μm, in (C), 20 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To avoid tissue damage, do not exceed a stimulation voltage of 60 V and don’t increase the pulse duration. If you have difficulties with this step, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.7">problems 4</a> and <a href="#sec7.9">5</a>).</div>
<h4 id="sec3.2.2">Imaging experiment</h4>
<ol start="13">
<li>Record 10-20 image series of 3–4 sec with the stimulation pulse delivered after one second of baseline registration (<a href="#fig6">Figures 6</a>A–6C, <a href="#mmc1">Methods video S1</a>). Ideally, stimuli should be delivered at least 10 s apart to minimize activity-dependent plastic changes of synaptic responses.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For simultaneous recordings of fEPSPs, it will be necessary to synchronize the electrophysiological recordings with the imaging. If your imaging system can record voltage traces, you can record the trigger and the fEPSP directly in your imaging software. Otherwise, you can create a voltage pulse in your imaging software which you record with the fEPSP and align the imaging and electrophysiology with the trigger signal.</div>
<ol start="14">
<li>For local drug application through a glass pipette, you can use a third micromanipulator or replace the fEPSP recording pipette with a similar pipette containing the desired drug (e.g., TTX, 5 μM). Connect a pressure application device to the drug application pipette. Place the pipette tip near the site of imaging, 50–100 μm below the slice surface.</li>
<li>Record a series of sweeps under baseline conditions to make sure that the population glutamate signals remained stable.</li>
<li>Carefully eject a small volume of the drug by applying increasing pressure under visual control until you see a small movement artifact from the ejected substance (typically 10–50 mbar). Readjust the focal plane to keep all previously imaged structures in the FOV. For troubleshooting of typical errors, refer to the <a href="#troubleshooting">troubleshooting</a> section (<a href="#sec7.11">problem 6</a>).</li>
<li>Record a series of sweeps during the drug application, then stop pressure application. Wait several minutes for the complete washout of the applied substances and record a third set of 10–20 sweeps.</li>
<li>After the imaging experiment, retract all pipettes and remove the experiment chamber from the microscope table.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> After the completion of the optical recordings, the slice can be fixed in paraformaldehyde and used for additional anatomical and immunohistochemical analyses (<a href="#fig3">Figure 3</a>).</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Example of a synaptically-evoked population glutamate transient in the CA1 hippocampal region recorded with single-photon imaging, related to step 13</div>
<p>Wide-field image of a hippocampal slice expressing SF-iGluSnFR in CA1 pyramidal neurons acquired through a 50 mm focal length camera objective at 40 Hz framerate. The stimulus (40V, 100 μs) was delivered at timepoint 0 ms through a glass pipette barely visible as a shadow on the right side of the image. The fluorescence changes are overlayed in false colors.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>After a short training period, the success rate of the viral transfection should be very high and it is typically possible to perform glutamate imaging in at least four hippocampal slices per injected animal. We did not detect spontaneous glutamate transients under baseline conditions. Single stimuli (typically 40V, 100 μsec) reliably triggered glutamate transients in the stratum radiatum both for the dendritic (expression of SF-iGluSnFR in CA1 neurons) or axonal (expression of SF-iGluSnFR in CA3 neurons) glutamate imaging (<a href="#fig6">Figures 6</a>A and 6B). The amplitude of the axonal signals (2.2% ± 0.5% (mean ± std), n = 5) was slightly lower than that of the dendritic imaging (3.1% ± 0.6%, n=9; p = 0.019). However, both the time-to-peak (65 ± 18msec vs. 47 ± 2msec, p = 0.36) and the full width at half-maximum (174 ± 17 msec vs. 161 ± 14 msec, p = 0.36) were undistinguishable for both conditions. Bath application of TTX suppressed the glutamate signals, while blocking glutamate uptake through local TBOA-applications increased the amplitude of the glutamate transients (<a href="#bib21">Zott et al., 2019</a>). Other investigators reported similar results using slightly modified protocols (<a href="#bib2">Barnes et al., 2020</a>; <a href="#bib12">Kopach et al., 2020</a>).</p>
<p>Glutamate imaging and this protocol can be particularly useful for the study of synaptic glutamate signaling and its possible perturbations in the diseased brain, for example in mouse models of Alzheimer’s disease and other pathological conditions (<a href="#bib5">Hefendehl et al., 2016</a>; <a href="#bib18">Parsons et al., 2016</a>; <a href="#bib21">Zott et al., 2019</a>). We observed that SF-iGluSnFR expresses well also in hippocampal neurons of the APP23xPS45 mouse model of β-amyloidosis (<a href="#fig7">Figure 7</a>A). If needed, amyloid plaques can be identified post hoc or during the experiment by staining with Methoxy-X04 (10 mg/kg body weight) (<a href="#bib11">Klunk et al., 2002</a>), delivered intraperitoneally 24 h before the experiment (<a href="#fig7">Figure 7</a>A). Methoxy-X04 can be excited at 800 or 920 nm in two-photon imaging or at 405 nm under single-photon conditions. The emission wavelength of Methoxy-X04 is 430 nm. Alternatively, in the two-photon microscope, amyloid plaques can be detected by their autofluorescence at 920 nm without the need for any additional filters (<a href="#fig7">Figure 7</a>B). Synaptically-evoked glutamate transients can be evoked as explained above (<a href="#fig7">Figure 7</a>B). To study the acute effects of soluble β-amyloid (Aβ) on glutamate homeostasis, we locally applied Aβ dimers through a glass pipette (see above) to the hippocampal CA1 region of wildtype mice. As previously reported (<a href="#bib21">Zott et al., 2019</a>), synaptically-evoked glutamate transients were enhanced in the presence of Aβ, an effect that was fully reversible after 15 min of washout (<a href="#fig7">Figure 7</a>C).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1161-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Expected outcomes: Glutamate imaging in mouse models of Alzheimer’s disease</div>
<p>(A) Confocal image of the expression of SF-iGluSnFR (green) in the hippocampus of a 7-month-old APP23xPS45 mouse with Methoxy-X04 plaque staining (orange) and NeuroTrace counterstaining (cyan). Intensity-maximized z-projection.</p>
<p>(B) Two-photon glutamate population imaging of SF-iGluSnFR expressing dendrites of CA1 pyramidal neurons in an APP23xPS45 mouse. Top left: Cartoon of the hippocampal slice depicting the location of the stimulation pipette (Stim) and the region of interest (ROI, dashed line) in the stratum radiatum containing the dendrites of the virus-expressing neurons (green). Amyloid plaques are indicated in orange. Bottom left: representative two-photon image of the stratum radiatum of a hippocampal slice from an APP23xPS45 mouse expressing SF-iGluSnFR in CA1. Plaques can be detected by their autofluorescence (orange dashed line). Right: Single sweeps (gray) and average (orange) of synaptically evoked glutamate transients recorded from the ROI indicated in the bottom left. The arrowhead indicates the time of stimulation.</p>
<p>(C) Glutamate population imaging during the application of Aβ dimers in a wildtype mouse. Top left: Cartoon of the imaging setup, the recommended positions of the stimulation (Stim) and Aβ application pipettes are indicated. The ROI in the stratum radiatum (dashed line) contains the dendrites of SF-iGluSnFR expressing neurons in CA1 (green). Bottom left: representative two-photon image of the experiment. Right: Overlay of the population glutamate transients before (black, solid line) and during pressure application of Aβ S26C dimers (1 μM, orange line) and after a washout period of 15 min (black dashed line), average of 20 sweeps, respectively. Scale bar in (A), 200 μm, in (B), 10 μm, in (C), 5 μm.</p>
</figcaption>
</figure>
<p>Single-photon imaging of synaptically-evoked population glutamate transients is a promising low-cost alternative to the two-photon imaging recordings. The synaptically-evoked glutamate transients even had a higher signal-to-noise-ratio under single-photon (45.7 ± 28, n = 5) compared to two-photon (7.5 ± 1.5, n = 9; p = 0.004) postsynaptic imaging conditions (<a href="#fig5">Figures 5</a> and <a href="#fig6">6</a> and <a href="#mmc1">Methods video S1</a>). This was calculated as the signal amplitude divided by the noise level at baseline. Moreover, the kinetics of the synaptically-evoked glutamate transients were slower in the single-photon compared to two-photon imaging conditions (time-to-peak: 164 ± 40 msec vs. 47 ± 2 msec ; p&lt;0.0001; full width at half-maximum: 257 ± 60 msec vs. 161 ± 14 msec; p&lt;0.0001). This is likely due to the fact that signal from a larger volume is integrated under single-photon imaging conditions (<a href="#bib7">Helmchen and Denk, 2005</a>). However, when imaging small structures such as amyloid plaques, single-photon wide-field imaging is not appropriate. Moreover, we recommend to perform the targeted application of drugs through a patch pipette (steps 14–18) only under two-photon conditions, because for the titration of the pressure needed for drug ejection, it is necessary to carefully visualize the displacement of the tissue in front of the pipette tip.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>To analyze population glutamate transients, extract the fluorescence signal from the full field of view frame by frame. This can be easily done using freely available software solutions such as ImageJ. Fluorescence traces can be further analyzed using toolboxes in IgorPro, in Python or in Matlab. Calculate the relative fluorescence for each sweep ΔF(t) = (F(t)-F<sub>BL</sub>)/F<sub>BL</sub>, with F(t) being the measured fluorescence at time point t and F<sub>BL</sub> the mean fluorescence 500 ms before stimulus application (<a href="#bib9">Jia et al., 2011</a>) and create an average of all sweeps from the respective experiment. The electrophysiological data can be filtered, offset-corrected and averaged using the Patcher’s Power Tools toolbox in Igor Pro or similar toolboxes in Matlab or Python.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>We have presented a versatile protocol to study glutamate (dys-) homeostasis in acute hippocampal slices. We have also included a description on how to combine the method with electrophysiology and drug applications. We demonstrate electrical synaptic stimulation, but expect optogenetic stimulation to work equally well.</p>
<p>When deciding which glutamate sensor to use, one has to consider the tradeoff between kinetics and sensitivity. The high sensitivity of the SF-iGluSnFR.A184S variant comes at the cost of relatively slow kinetics. This makes it unsuitable to resolve glutamate transients of high-frequency trains of action potentials (<a href="#bib16">Marvin et al., 2018</a>). “Faster” variants of the SF-iGluSnFR are available from the Looger and Oertner groups (<a href="#bib6">Helassa et al., 2018</a>; <a href="#bib16">Marvin et al., 2018</a>). However, kinetics of even the fastest available sensor are at least one order of magnitude slower than the dwell time of glutamate in the synaptic cleft, so it is important to keep in mind that the kinetics of the imaged glutamate transients are shaped by the sensor rather than glutamate itself (<a href="#bib1">Armbruster et al., 2020</a>).</p>
<p>Another limitation that needs to be taken into account is that the expression of glutamate sensors in the membrane of neurons might interfere with glutamate metabolism by disrupting the function of glutamate transporters or buffering glutamate (<a href="#bib1">Armbruster et al., 2020</a>). In consequence, appropriate control experiments with complementary approaches such as electrophysiology have to be carried out.</p>
<p>Finally, our protocol is limited to the analysis of population glutamate signals. Protocols for the analysis of glutamate transients at single axons and dendrites or even boutons also exist (<a href="#bib3">Dürst et al., 2019</a>; <a href="#bib6">Helassa et al., 2018</a>; <a href="#bib8">Jensen et al., 2019</a>; <a href="#bib19">Rama et al., 2019</a>), but require more sophisticated equipment enabling spiral scanning of the two-photon laser across the axonal boutons and single-cell electrophysiology for the stimulation of individual cells.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Neurons in the target area do not express SF-iGluSnFR.</p>
<p>If no SF-iGluSnFR-expressing neurons can be found in CA1 or CA3, respectively (step 1), this typically results from (i) wrong injection coordinates or (ii) inadequate injection parameters causing the viral vectors to spread along the track of the pipette instead of diffusing into the tissue at the pipette tip.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>As the first step, check again all your slices for fluorescence. If you find labeled cells outside the target region, go to problem 2.</p>
<p>A slow release speed of the viral construct from the injection pipette is highly recommended for achieving its uniform diffusion and distribution in the brain tissue penetration. Therefore, it is important to carefully monitor the meniscus of the fluid in your pipette under your dissection microscope during the release of viral construct solution and ensure that it drops slowly. The average injection speed should not exceed 20 nl/min, i.e., injecting 100 nl per injection site should take at least five minutes. Also, after the last injection, you can leave the pipette in the brain for at least five minutes before removing it. This will further facilitate an even diffusion of the viral vector. If your pipette is clogged, you can try to clear the tip with PBS or carefully trim it using fine scissors.</p>
<p>If not a single neuron is labeled throughout the brain, the viral construct may be damaged or non-functional.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Wrong injection site.</p>
<p>It is possible that the fluorescently labeled neurons are located outside of the target region (step 1). This is the most common problem when targeting small subcortical structures such as the hippocampal subdivisions. It typically arises from incorrect injection coordinates or an inadequate positioning of the mouse head in the stereotactic frame during the injection of viral constructs.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Ensure that the skull is level during the injection of viral constructs. Bregma and lambda should be at the same height and the head should not be tilted to either side. Using a mechanical or inductive alignment indicator (e.g., eLeVeLeR, Sigmann Elektronik) can help with this problem.</p>
<p>When performing the craniotomy, make sure to fully remove the bone flap. Any residual bone can deflect the pipette tip, possibly causing great aberrations.</p>
<p>If you detect a systematic offset of the site of SF-iGluSnFR expression in several mice, it will be necessary to adapt the injection coordinates. This is especially necessary if you use animals of different ages or from other genetic backgrounds. It might be helpful to perform acute test injections with a bright fluorescence dye (<a href="#bib10">Kim et al., 2014</a>) to get immediate feedback on how the adaptation of the injection coordinates affected the outcome.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Wound adhesions.</p>
<p>Scar-tissue adhesions between dura, brain and skull or skull and scalp may be caused by a post-surgical infection or dural damage during the injection of viral constructs. These adhesions might damage the tissue when the brain is removed from the skull before the slicing procedure (see “<a href="#before-you-begin">before you begin</a>” section).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Keep the scalp incision as small as possible and make sure to remove all cut hair to prevent infection.</p>
<p>Only penetrate the dura with the injection pipette. Avoid any lateral movement of the pipette in the brain tissue. Perform a well-adapted and tension-free wound suture.</p>
<p>When removing the brain from the skull, make sure that there is no scar tissue holding the dura to the bone. If you find such adhesions, carefully clear them with a fine forceps.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Synaptic stimulation does not evoke glutamate transients.</p>
<p>It is possible that stimulation parameters indicated in step 12 do not trigger population glutamate transients. This is usually due to a noxious slicing or recovery procedure, inadequate stimulation parameters or a wrong positioning of the stimulation pipette.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>In order to avoid hypoxic brain damage during the slicing procedure, take great care to stick to the timing (<a href="#bib4">Edwards et al., 1989</a>). This may need some practice. Make sure all your solutions have the correct pH, osmolarity, electrolyte levels and temperature.</p>
<p>Brain slices need continuous saturation with carbogen gas. However, you will need to interrupt the gas supply during transfers of the slices between setups or slice selection (step 1). Make sure to keep such interruptions at a minimum. If using mice older than 2–3 months, it can be helpful to perform an intracardial perfusion with ice-cold CS and using a protective recovery protocol to prevent cellular damage during the slicing procedure (<a href="#bib20">Ting et al., 2018</a>). We used this protocol to obtain the data in <a href="#fig7">Figures 7</a>A and 7B.</p>
<p>For troubleshooting, we recommend simultaneous recording of fEPSPs. If no fEPSP can be recorded with the stimulation parameters indicated in step 12, reposition the stimulation electrode (step 7). If your stimulation evokes fEPSPs but no glutamate transients, it is possible that your imaging settings are not sensitive enough. Increase the laser power but take care not to induce bleaching. You may also adjust your FOV (i.e., decrease the zoom or use a lower magnification objective) to integrate signal from a larger number of SF-iGluSnFR-expressing neuronal processes. Finally, applying short bursts of stimuli (e.g., 10 pulses at 100 Hz) increases the amount of released glutamate and may help to unmask the fluorescent glutamate transients.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Complications caused by excessive stimulation.</p>
<p>Applying stimuli with the parameters indicated in step 12 should elicit glutamate transients with a short rise time and a longer decay time which return to baseline after less than one second. The transients may vary slightly in amplitude but not in kinetics from trial to trial (<a href="#fig6">Figures 6</a> and <a href="#fig7">7</a>). If the evoked transients do not return to baseline or if their amplitude decreases with each stimulus, this is typically a result of excessive stimulation.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Carefully follow steps 11 and 12 to determine the adequate stimulus voltage. Repeat these steps every time you change the slice or the stimulation pipette because the position of the tip of the stimulation pipette in relation to the Schaffer collaterals may be slightly different. Never exceed a stimulus duration of 100 μsec.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Difficulties with drug application.</p>
<p>Pressure application of drugs (steps 14–18) and especially of hydrophobic substances such as Aβ is an error-prone step and requires some practice. Typical problems include clogging of the pipette tip and the sudden release of an uncontrolled amount of fluid sometimes causing tissue damage.</p>
<h3 id="sec7.12">Potential solution</h3>
<p>To prevent clogging of the pipette, you can apply a low level of baseline holding pressure (2–3 mbar) before entering the tissue. This can be best achieved with a pressure application system with a fine scale. To ensure a stable efflux of controlled amounts of solution from your injection pipette , it is vital to identify the most appropriate pressure level. Increase the pressure level slowly (1 mbar/sec starting from 0 mbar or your holding pressure). If you do not achieve drug release (typically seen as a slight tissue displacement) with a pressure of 50 mbar, replace the pipette rather than increasing the pressure further. Replace the slice if you have accidentally ejected large volumes.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Armbruster, M., Dulla, C.G., and Diamond, J.S. (2020). Effects of fluorescent glutamate indicators on neurotransmitter diffusion and uptake. eLife <i>9</i>, e54441. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref1">View at publisher</a></p>
<p id="bib2">Barnes, J.R., Mukherjee, B., Rogers, B.C., Nafar, F., Gosse, M., and Parsons, M.P. (2020). The relationship between glutamate dynamics and activity-dependent synaptic plasticity. J. Neurosci. <i>40</i>, 2793-2807. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref2">View at publisher</a></p>
<p id="bib22">Busche, M.A., Eichhoff, G., Adelsberger, H., Abramowski, D., Wiederhold, K.H., Haass, C., Staufenbiel, M., Konnerth, A., and Garaschuk, O. (2008). Clusters of hyperactive neurons near amyloid plaques in a mouse model of Alzheimer’s disease. Science <i>321</i>, 1686-1689. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/opt0ah9Srosfl">View at publisher</a></p>
<p id="bib3">Dürst, C.D., Wiegert, J.S., Helassa, N., Kerruth, S., Coates, C., Schulze, C., Geeves, M.A., Török, K., and Oertner, T.G. (2019). High-speed imaging of glutamate release with genetically encoded sensors. Nat. Protoc. <i>14</i>, 1401-1424. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref3">View at publisher</a></p>
<p id="bib4">Edwards, F.A., Konnerth, A., Sakmann, B., and Takahashi, T. (1989). A thin slice preparation for patch clamp recordings from neurones of the mammalian central nervous system. Pflügers Arch. <i>414</i>, 600-612. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref4">View at publisher</a></p>
<p id="bib5">Hefendehl, J.K., LeDue, J., Ko, R.W., Mahler, J., Murphy, T.H., and MacVicar, B.A. (2016). Mapping synaptic glutamate transporter dysfunction in vivo to regions surrounding Abeta plaques by iGluSnFR two-photon imaging. Nat. Commun. <i>7</i>, 13441. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref5">View at publisher</a></p>
<p id="bib6">Helassa, N., Dürst, C.D., Coates, C., Kerruth, S., Arif, U., Schulze, C., Wiegert, J.S., Geeves, M., Oertner, T.G., and Török, K. (2018). Ultrafast glutamate sensors resolve high-frequency release at Schaffer collateral synapses. Proc. Natl. Acad. Sci. U S A <i>115</i>, 5594-5599. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref6">View at publisher</a></p>
<p id="bib7">Helmchen, F. and Denk, W. (2005). Deep tissue two-photon microscopy. Nat. Methods <i>2</i>, 932-940. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref7">View at publisher</a></p>
<p id="bib8">Jensen, T.P., Zheng, K., Cole, N., Marvin, J.S., Looger, L.L., and Rusakov, D.A. (2019). Multiplex imaging relates quantal glutamate release to presynaptic Ca2+ homeostasis at multiple synapses in situ. Nat. Commun. <i>10</i>, 1414. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref8">View at publisher</a></p>
<p id="bib9">Jia, H., Rochefort, N.L., Chen, X., and Konnerth, A. (2011). In vivo two-photon imaging of sensory-evoked dendritic calcium signals in cortical neurons. Nat. Protoc. <i>6</i>, 28-35. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref9">View at publisher</a></p>
<p id="bib10">Kim, J.Y., Grunke, S.D., Levites, Y., Golde, T.E., and Jankowsky, J.L. (2014). Intracerebroventricular viral injection of the neonatal mouse brain for persistent and widespread neuronal transduction. J. Vis. Exp. <i>91</i>, 51863. <a class="external-link" href="https://doi.org/10.3791/51863">https://doi.org/10.3791/51863</a></p>
<p id="bib11">Klunk, W.E., Bacskai, B.J., Mathis, C.A., Kajdasz, S.T., McLellan, M.E., Frosch, M.P., Debnath, M.L., Holt, D.P., Wang, Y., and Hyman, B.T. (2002). Imaging Abeta plaques in living transgenic mice with multiphoton microscopy and methoxy-X04, a systemically administered Congo red derivative. J. Neuropathol. Exp. Neurol. <i>61</i>, 797-805. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref11">View at publisher</a></p>
<p id="bib12">Kopach, O., Zheng, K., and Rusakov, D.A. (2020). Optical monitoring of glutamate release at multiple synapses in situ detects changes following LTP induction. Mol. Brain <i>13</i>, 39. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref12">View at publisher</a></p>
<p id="bib13">Lowery, R.L. and Majewska, A.K. (2010). Intracranial injection of Adeno-associated viral vectors. J. Vis. Exp. <i>45</i>, e2140. <a class="external-link" href="https://doi.org/10.3791/2140">https://doi.org/10.3791/2140</a></p>
<p id="bib14">Martin, S.J. and Clark, R.E. (2007). The rodent hippocampus and spatial memory: from synapses to systems. Cell. Mol. Life Sci. <i>64</i>, 401-431. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref14">View at publisher</a></p>
<p id="bib15">Marvin, J.S., Borghuis, B.G., Tian, L., Cichon, J., Harnett, M.T., Akerboom, J., Gordus, A., Renninger, S.L., Chen, T.-W., Bargmann, C.I., et al. (2013). An optimized fluorescent probe for visualizing glutamate neurotransmission. Nat. Methods <i>10</i>, 162-170. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref15">View at publisher</a></p>
<p id="bib16">Marvin, J.S., Scholl, B., Wilson, D.E., Podgorski, K., Kazemipour, A., Müller, J.A., Schoch, S., Quiroz, F.J.U., Rebola, N., Bao, H., et al. (2018). Stability, affinity, and chromatic variants of the glutamate sensor iGluSnFR. Nat. Methods <i>15</i>, 936-939. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref16">View at publisher</a></p>
<p id="bib17">Moser, E.I., Kropff, E., and Moser, M.B. (2008). Place cells, grid cells, and the brain's spatial representation system. Annu. Rev. Neurosci. <i>31</i>, 69-89. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref17">View at publisher</a></p>
<p id="bib18">Parsons, M.P., Vanni, M.P., Woodard, C.L., Kang, R., Murphy, T.H., and Raymond, L.A. (2016). Real-time imaging of glutamate clearance reveals normal striatal uptake in Huntington disease mouse models. Nat. Commun. <i>7</i>, 11251. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref18">View at publisher</a></p>
<p id="bib19">Rama, S., Jensen, T.P., and Rusakov, D.A. (2019). Glutamate imaging reveals multiple sites of stochastic release in the CA3 giant mossy fiber boutons. Front. Cell. Neurosci. <i>13</i>, 243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref19">View at publisher</a></p>
<p id="bib20">Ting, J.T., Lee, B.R., Chong, P., Soler-Llavina, G., Cobbs, C., Koch, C., Zeng, H., and Lein, E. (2018). Preparation of acute brain slices using an optimized N-Methyl-D-glucamine protective recovery method. J. Vis. Exp. <i>132</i>, e53825. <a class="external-link" href="https://doi.org/10.3791/53825">https://doi.org/10.3791/53825</a></p>
<p id="bib21">Zott, B., Simon, M.M., Hong, W., Unger, F., Chen-Engerer, H.-J., Frosch, M.P., Sakmann, B., Walsh, D.M., and Konnerth, A. (2019). A vicious cycle of β amyloid–dependent neuronal hyperactivation. Science <i>365</i>, 559-565. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00583-9/sref21">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Benedikt Zott (<a href="mailto:benedikt.zott@tum.de">benedikt.zott@tum.de</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new reagents.</p>
<h4>Data and code availability</h4>
<p>All data needed to evaluate the conclusions of this paper are presented here, additional data may be requested from the authors.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> (SFB 870) and the <a href="https://doi.org/10.13039/501100012319">Max-Planck-School of Cognition</a>. A.K. is a Hertie-Senior-Professor for Neuroscience. Additional support was provided by the Commission for Clinical Research (KKF) of the <a href="https://doi.org/10.13039/501100005713">Technical University of Munich</a> (B.Z.). We thank C. Karrer, P. Apostolopoulos, F. Beyer, and R. M. Karl for their technical support. We are grateful to L. Looger for providing the SF-iGlu-SnFR constructs.</p>
<h3>Author contributions</h3>
<p>Conceptualization, F.U., A.K., and B.Z.; methodology, F.U. and B.Z.; data acquisition and analysis, F.U. and B.Z.; writing – original draft, F.U., A.K., and B.Z.; writing – review &amp; editing, F.U., B.Z., and A.K.; funding acquisition, A.K.; supervision, A.K. and B.Z.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>