<article>
<section>
<h1 id="header">Collection of cells for single-cell RNA sequencing using high-resolution fluorescence microscopy</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Hendrika A. Segeren,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Kiki C. Andree,<sup><a href="#aff2">2</a></sup> Lisa Oomens,<sup><a href="#aff2">2</a></sup> and Bart Westendorp<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biomolecular Health Sciences, Faculty of Veterinary Medicine, Utrecht University, Utrecht, the Netherlands</p>
<p id="aff2"><sup>2</sup>VYCAP B.V., Enschede, the Netherlands</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:H.A.Segeren@uu.nl">H.A.Segeren@uu.nl</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:B.Westendorp@uu.nl">B.Westendorp@uu.nl</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100718">10.1016/j.xpro.2021.100718</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>FACS sorting followed by single-cell RNA-sequencing (SORT-Seq) is a popular procedure to select cells of interest for single-cell transcriptomics. However, FACS is not suitable for measurement of subcellular distribution of fluorescence or for small samples (&lt;1,000 cells). The VYCAP puncher system overcomes these limitations. Here, we describe a workflow to capture, image, and collect fluorescent human retina pigment epithelium cells for SORT-Seq using this system. The workflow can be used for any cell type with a diameter of ∼5–50 μm.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib5">Segeren et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Capture up to 6,400 living fluorescent-labeled single cells on a chip using air pressure</li>
<li>High-magnification fluorescence microscopy and selection of cells of interest</li>
<li>Punch single cells of interest into microwells using an ultrafine needle</li>
<li>Microwell plate processing for single-cell RNA-sequencing using automated platform</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/904-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The platform used in this protocol has three main components, which are described in detail elsewhere (<a href="#bib6">Stevens et al, 2018</a>). Briefly, the first component is a customized microscope equipped with 2 xy stages, a z stage containing a needle for punching, and dedicated software (<a href="#fig1">Figure 1</a>A). The second component is a disposable microwell chip placed in a two-compartment reservoir (<a href="#fig1">Figure 1</a>B). The chip contains a grid with 80 × 80 microwells (referred to as cups hereafter) and act as a sieve in which cells are collected (<a href="#fig1">Figure 1</a>C). Each cup contains a pore with a diameter of either 2 or 5 μm. Fluid can pass through, but single cells are trapped on these pores. The third component is a vacuum pump to pull the cell suspension through the chip (<a href="#fig1">Figure 1</a>B).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/904-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Collecting single cells for RNA-sequencing with the VYCAP needle puncher</div>
<p>(A) Customized microscope system with dedicated software to isolate, select and punch single cells. The chip is placed on xy-stage #1, and the microwell plate is placed on xy-stage #2.</p>
<p>(B) Pump system to apply a negative pressure on the chip to pull the single cell suspension from the collection reservoir towards the waste reservoir.</p>
<p>(C) The cell capture chip. Inset: microscopic overview of the grid, which contains 6400 cups that each contain a pore. The chip acts as a sieve; single cells cannot pass through the pores.</p>
<p>(D) Image showing cell with 53BP1 foci and nuclear expression of Geminin(1–120) captured on a VYCAP microwell chip. Images were taken with a 40× long working distance objective.</p>
</figcaption>
</figure>
<p>The protocol below describes the specific steps for human retina pigment epithelium (hTERT RPE-1) cells carrying the FUCCI-system (<a href="#bib3">Sakaue-Sawano et al, 2008</a>). However, the protocol is suitable for or can be adapted to other adherent cell cultures or single cell suspensions. For example, we have also used this protocol to create single cell RNA-sequencing data from primary hepatocytes, as well as from colorectal carcinoma (HT29) and prostate cancer (PC-3 and LNCAP) cell lines.</p>
<p>Cell selection in the current protocol is done based on overall fluorescence intensity per nucleus. However, the system is equipped with a 40× long working distance objective and therefore well suited to analyze subcellular fluorescent signals, such as formation of fluorescent tagged truncated 53BP1 foci in response to replication stress-induced DNA damage (<a href="#fig1">Figure 1</a>D).</p>
<h3 id="sec1.1">Prepare reporter-carrying cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 weeks</div>
<ol>
<li>To create reporter-carrying cell lines, different cell lines and reporters can be used. We routinely use hTERT RPE-1 cells transduced with lentivirus carrying the cell cycle reporters mAG-hGeminin(1-110), mKO2-hCDT1(30-120), and mTurquoise2-53BP1trunc (<a href="#bib5">Segeren et al, 2020</a>).
<div class="critical">
<span class="critical-title">Critical:</span> At least one fluorescent marker must be present in all cells to enable automated selection of cells. Hoechst33342 or another DNA dye able to pass membranes of intact cells can be used for this purpose.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Add a fluorescent lipophilic dye to allow simultaneous capture of two different conditions in one sequencing plate. For example drug versus control or knockout versus wild-type.</div>
<ol type="a">
<li>Add 1 μL of the fluorescent dye DiIC18(5); 1,1′-dioctadecyl-3,3,3′,3′- tetramethylindodicarbocyanine, 4-chlorobenzenesulfonate salt (DiD) per mL medium to one of the two conditions.</li>
<li>Gently swirl the culture dish, and place the cells back into the incubator for 1 h.</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Prepare the chip</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h</div>
<ol start="2">
<li>Filter 100% analysis-grade ethanol and PBS using a 0.2 μm filter.</li>
<li>Remove the chip from the holder, and immerse it completely in a Petri dish containing filtered 100% analysis-grade ethanol (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/904-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Preparing the cell capture chip</div>
<p>(A) Immerse chip in 100% ethanol in a petri dish to prepare the air bubble removal step.</p>
<p>(B) Chip viewed from above, before (upper image) and after (lower image) removal of air bubbles and re-mounting. The areas containing cups are marked by the yellow boxes. Note that the surface of the chip looks smooth and regular after bubble removal. After this step, make sure that the chip does not run dry again.</p>
<p>(C) Re-mount the chip in the holder and gently pipet 1 mL of PBS on top. Avoid air bubble formation at all times.</p>
<p>(D) Correct alignment of the microwell plate. The first well (A1) should be right underneath the tip of the needle.</p>
<p>(E) The plateholder must be calibrated in the z-stage such that the distance between microwell plate and chip holder is not more than 1–2 mm (lower panel, yellow arrow).</p>
<p>(F) Dry the backside of the chip by touching a corner with a piece of absorbing lens cleaning paper before mounting the chip on the microscope. Avoid touching the chip.</p>
<p>(G) Pipet roughly 150–200 μL of PBS or serum-free medium on the chip as soon as it is mounted on the microscope.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Chips with different pore sizes are available. Choose the chip with appropriate pore size based on cell size and compatibility with downstream analysis. Default chips have a pore size of 5 μm and suffice for regular sized cells and evaluation of nuclear reporters.</div>
<ol start="4">
<li>Place the Petri dish containing the chip immersed in ethanol in a vacuum chamber. Maintain the vacuum until the chip appears completely free from air bubbles (<a href="#fig2">Figure 2</a>B). This can take 0.5–3 h depending on the vacuum applied.</li>
<li>Place the chip back into the holder and gently pipet 1 mL of sterile, pre-filtered PBS on top of the chip (<a href="#fig2">Figure 2</a>C). Take care to produce no new bubbles.</li>
<li>To wash the chip with a 5 μm pore size, switch on the pump at 20 mbar negative pressure to pull the PBS through the pores of the chip. Increase the pressure to −100 mbar for chips with a pore size of 2 μm.</li>
<li>When the PBS is almost gone and only 100–200 μL of liquid still covers the chip, apply another 1 mL of PBS.</li>
<li>Repeat step 9.</li>
<li>Switch off pump when PBS is almost gone. Keep the chip covered with a small amount of PBS (∼200 μL).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid that the chip runs dry during steps 8–11. Otherwise new air bubbles will form, and block the pores of the microwell chip (<a href="#fig2">Figure 2</a>C). Don’t keep the pump running unattended and stop it as soon as the liquid surface is a few millimeters above the collection cups.</div>
<h3 id="sec1.3">Align the needle and the collection plate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="10">
<li>Start the Puncher and open the VYCAP Punch software.</li>
<li>Place the needle and wait at least 3 min to let it settle in its holder.</li>
<li>Select the desired objective and the type of collection device (384-well plate). Align the 384-well plate in xy-directions using the VYCAP software; the needle should be above the middle of well A1 (<a href="#fig2">Figure 2</a>D).</li>
<li>Align the 384-well plate in the z-direction using the VYCAP software; make sure that the distance between chip and plate is as short as possible (<a href="#fig2">Figure 2</a>E).</li>
<li>Calibrate the needle via the VYCAP software.</li>
<li>Adjust in the settings section of the VYCAP software the frequency of re-calibrations between the punches. The frequency of re-calibration is a trade-off between speed and accuracy, but we typically calibrate each 24 punches.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Check proper alignment by punching a few wells of a clean, dry chip.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Vybrant DiD Cell-Labeling Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# V22887</td>
</tr>
<tr>
<td>Hoechst 33342</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# H3570</td>
</tr>
<tr>
<td>Ethanol 100% analysis grade</td>
<td>Merck</td>
<td>Cat# 100983</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Single cell RNA-sequencing RPE-FUCCI cells</td>
<td>(<a href="#bib5">Segeren et al, 2020</a>)</td>
<td>GEO: GSE146759</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>hTERT RPE-1-FUCCI</td>
<td>(<a href="#bib1">Boekhout et al, 2016</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>hTERT RPE-1-FUCCI-<i>E2F7/8</i><sup><i>KO</i></sup>
</td>
<td>(<a href="#bib7">Yuan et al, 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FIJI (ImageJ)</td>
<td>http://fiji.sc</td>
<td>RRID:SCR_002285</td>
</tr>
<tr>
<td>Custom VYCAP image quantification macro</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Puncher system</td>
<td>VYCAP</td>
<td>Cat# Ti2-PS</td>
</tr>
<tr>
<td>Pump unit</td>
<td>VYCAP</td>
<td>Cat# PU-250</td>
</tr>
<tr>
<td>Punch needles</td>
<td>VYCAP</td>
<td>Cat# PN-70</td>
</tr>
<tr>
<td>Disposable single cell microwell chips</td>
<td>VYCAP</td>
<td>Cat# FM75-510</td>
</tr>
<tr>
<td>CEL-Seq2 collection plates (384)</td>
<td>Single Cell Discoveries</td>
<td>N/A</td>
</tr>
<tr>
<td>Microseal ‘F’ PCR plate seal foil</td>
<td>Bio-Rad</td>
<td>Cat# MSF1001</td>
</tr>
<tr>
<td>Nalgene Syringe Filters 0.22 μm</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 723-2520</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>For the cell capture, a VYCAP Puncher system is required. The system is equipped with dedicated software for imaging, cell selection and punching, and comes with a pump to capture single cells onto a chip. To remove air bubbles from the chip a vacuum chamber is required. If a vacuum chamber is unavailable, it is possible to order chips that are pretreated with a special coating to avoid bubble formation, and for which it is not necessary to remove bubbles in a vacuum chamber.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This section lists the major steps and provides step-by-step details and timing for each major step. Please use continuous number for this section.</p>
<h3 id="sec3.1">Trypsinize, count, and load cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>This step describes how to prepare a suspension containing living single cells for the punch procedure. Depending on cell line and cell counting method used, this step may require more time. We count the hTERT RPE-1 cells with an automated cell counter such as the Bio-Rad TC20.</p>
<ol>
<li>Filter serum-free medium using a 0.2 μm filter.</li>
<li>Trypsinize cells according to routine procedures.</li>
<li>Spin down cells to collect cells in a pellet (3 min, 400 rcf for hTERT RPE-1 cells).</li>
<li>Remove medium and reconstitute cell pellet in filtered serum-free medium.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> if serum was used to inactivate the trypsin, we recommend an extra washing step to remove traces of serum. This can be done by spinning down the cells a second time and reconstituting them again in fresh serum-free medium.</div>
<ol start="5">
<li>Count the cells.</li>
<li>Dilute ∼7500 cells in 200 μL of filtered serum-free medium.</li>
<li>If not all the cells harbor a fluorescent marker which can be used for automated selection, add Hoechst33342 (1:20.000 dilution) or another DNA dye to the cell suspension. This DNA staining is highly suitable for automated cell selection after the imaging step.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When counting the concentration of cells, include also dead cells in the calculation. Dead cells will also block the pores! The chip contains only 6400 pores and adding a large excess of cells will inevitably result in multiple cells settling into one well. A modest excess of 1000–1500 cells is recommended however, because there is always a percentage of cells that don’t get captured, for example because they stick to the walls of the collection reservoir.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Reconstitute the cell pellet in serum-free medium. Serum contains high amounts of proteins. These will stick to the needle, causing highly inefficient punching (see <a href="#sec7.1">troubleshooting 1</a>).</div>
<h3 id="sec3.2">Cell capture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>This step describes the capturing of single cells in the chip and mounting of the chip on the microscope.</p>
<ol start="8">
<li>Pipet the suspension containing the cells on the chip and apply a negative pressure of −20 mbar on chips with 5 μm pores, or −100 mbar on chips with 2 μm pores.</li>
<li>As soon as the chip runs almost dry (i.e., only a &lt; 1 mm thick film of liquid on top of the chip), immediately stop the pump and clean the bottom of the chip by pipetting 4–5 times 1 mL of sterile, filtered PBS on the backside of the chip.</li>
<li>After the last wash, remove the last droplets of PBS by holding some absorbing paper to the corner of the bottom of the chip (<a href="#fig2">Figure 2</a>F). Avoid touching the chip.</li>
<li>Mount the chip on the Puncher and immediately pipet 150–200 μL of serum-free medium on the chip to keep the cells in a wet environment (<a href="#fig2">Figure 2</a>G).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Carefully clean the backside of the chip, which faces the objective of the microscope. It should be completely dry and free of salt crystals or large dust particles. Contamination will result in poor quality of the reflection images. These images must be of high quality as they are essential for alignment and focusing of the chip, and evaluation of punch success.</div>
<h3 id="sec3.3">Image and select cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step describes the imaging and selection of captured cells.</p>
<ol start="12">
<li>Align the chip according to the instructions in the puncher software.</li>
<li>Focus the chip using one of the fluorescence filters or the reflection channel.</li>
<li>Scan the relevant fluorescent channels, as well as the reflection channel. The reflection channel will be used to evaluate punching success. Fluorescent reporters typically require exposure times of 20–100 milliseconds. A full microwell chip scan using a 20× objective typically generates 225 images per channel and takes 15–20 min when scanning 4 different channels. When using a 40× objective, scanning time can be reduced by imaging only a part of the chip.</li>
<li>Select cells of interest via automated selection. Automated selection can be performed by setting intensity ranges in up to 3 channels simultaneously.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Instead of selection based on microscopy images, a file with predefined cup numbers can be loaded to select cells of interest.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Alternative to automated selection, cells can be manually selected. However, to minimize time between preparation of cell suspension and collection of cells in multi-well plates, automated selection is preferred. Images of punched cells are stored and can be linked to transcriptomic data in downstream analysis.</div>
<ol start="16">
<li>Scroll through the list of selected cells to visually inspect their quality (<a href="#fig3">Figure 3</a>A). Doublets or dead cells can simply be removed from the list.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/904-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Cell selection and inspection of punch success</div>
<p>(A) Screenshot from the VYCAP punch software showing a list of automatically detected cells. The list can be manually inspected and unwanted cells can be manually deleted (click button indicated by dark red arrow).</p>
<p>(B) Screenshot from the VYCAP software showing the well assignment window. In this case the first 384 cells of interest were each assigned automatically to individual well positions.</p>
<p>(C) Reflection images before and after a successful punch. Scalebar indicates 20 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.4">Punch cells into microwell plate</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step describes how the cells are punched into 384-well plates compatible with an automated platform for CEL-Seq2 library preparation (<a href="#bib2">Muraro et al, 2016</a>). Wells of the collection plates contain primers and nucleotides in a volume of 100 nL, and 5 μL of mineral oil to avoid evaporation. Any desired alternative library preparation chemistry can in principle be used.</p>
<ol start="17">
<li>Take a ready-to-use 384-well plate containing CEL-Seq2 primers from the −20°C freezer, and briefly spin down the plate (1 min, 2000 rfc).</li>
<li>Remove the foil and place the plate on the puncher.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> a cooling block can be used to keep the 384-well plate cold to slow down potential RNA degradation.</div>
<ol start="19">
<li>Assign each selected cell to a specific well ID of the 384-well plate (A1 through P24) using the VYCAP software (<a href="#fig3">Figure 3</a>B).</li>
<li>Punch the cells.</li>
<li>After the last punch, immediately take the plate from the puncher, and seal it with adhesive foil.</li>
<li>Spin down the plate (1 min, 2000 rfc).</li>
<li>Immediately transfer the plate to a −80°C freezer.</li>
<li>Store at −80 degrees Celsius until library preparation.</li>
<li>Inspect the after images to confirm successful punching throughout the plate (<a href="#fig3">Figure 3</a>C). The number of correct punches should be ∼90% or higher; if not, please refer to <a href="#troubleshooting">troubleshooting</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Taking after-images takes a substantial amount of time: 30 minutes for a full 384-well plate. Hence it is important to spin down and freeze the plate immediately after the last punch, and not wait until the after-images are all taken.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Initially assign and punch a limited number of cells (5–10) and check the after-images to ensure successful punching. After this small test the entire plate can be punched. This allows the user to troubleshoot by quickly replacing or re-calibrate the needle if necessary.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>hTERT RPE-1 cells captured using this protocol, and further processed for single cell transcriptomics according to the CEL-Seq2 protocol (<a href="#bib2">Muraro et al, 2016</a>) typically yield libraries of 30.000 to 100.000 unique UMI counts, mapping to 4000–8000 different genes. This library complexity and sequencing depth is comparable to FACS sorted hTERT RPE-1 cells. We usually see that 100–150 of the wells from a fully punched 384-well plate contain sufficiently complex libraries for downstream RNA-seq analysis. This success rate is somewhat lower than FACS-sorted hTERT RPE-1 cells, which typically yields ∼200–250 high-quality libraries per 384-well plate.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Quantitative image analysis can be done with ImageJ or any other imaging analysis software. An example of a macro that recursively quantifies mAG-hGeminin(1–110) in nuclear regions detected with the Hoechst33342 signal is included in this paper (Supplementary Macro). ROIs are drawn around the nuclei in the DAPI channel. The ROIs are then projected on the FITC channel to measure the mean fluorescence intensity within each nucleus. This information on fluorescent intensities can now be coupled into a metadata table using for example basic R coding in Rstudio, and then integrated with the RNA count data using the R-package Seurat (<a href="#bib4">Satija et al, 2015</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>In principle, any type of cell can be captured for single cell RNA-sequencing using this procedure. However, the cells should have a diameter within the range of 2–70 μm. The pores in the chip suitable for small cells have a diameter of 2 μm, and very small cells can thus slip through and disappear. The diameter of the cups is 70 μm, meaning that very large cells such as adult cardiomyocytes may not fit in the cups.</p>
<p>Another limitation is that the cells must not be suspended in a protein- or polysaccharide-rich environment, as these molecules tend to contaminate the needle and make the punch process highly inefficient. For example, we unsuccessfully tried to punch primary cells isolated from synovial fluid, which is highly rich in hyaluronic acid, even after extensive washing by multiple rounds of centrifugation and resuspension in PBS or basic medium.</p>
<p>A final limitation is that cells are captured alive, and the transcriptomes of the captured cells can change between the moment of isolation and punching. We are currently exploring the possibility to capture fixed cells. However, it remains to be seen if the effect of fixation on mRNA quality for single cell sequencing will be problematic. Besides ruling out the effect of processing time, an additional advantage would be that the morphology of the cells may be better preserved with a mild fixation. This might allow imaging of more complex morphologic cell structures.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<p>The main problems we encountered are related to a low percentage of successfully punched cups. If this percentage is well below the expected 90%, inspect the after-images to analyze which of the problems described below is the most likely underlying cause.</p>
<h3 id="sec7.1">Problem 1</h3>
<p>After punching many well-bottoms stick to the chip, as can be seen in the after-images (<a href="#fig4">Figure 4</a>A).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/904-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Troubleshooting</div>
<p>(A) Filmstrip with reflection images from a failed experiment with a sticky needle. The bottoms do not fall, and stick to the chip when the needle returns to the start position. Scalebar: 20 μm.</p>
<p>(B) Brightfield images of an intact clean punch needle (upper panel) and an example of a needle tip heavily contaminated with protein and hyaluronan (lower panel). Arrowhead: glass microsphere that is glued to the tip of the metal. Scalebar: 10 μm.</p>
<p>(C) Filmstrip with reflection images from a failed experiment with a sharp punch needle; the wells get shattered, because the smooth glass microsphere has fallen off. Scalebar: 20 μm.</p>
<p>(D) Brightfield image of a needle lacking the microsphere after accidentally hitting the metal of the chip between the wells. Scalebar: 10 μm.</p>
<p>(E) Incorrect detection of the breaking edge of one of the corner cups during chip alignment. This is usually seen if the back-side of the chip is not properly cleaned (left panel), or if the contrast in the image is not high enough. The right panel shows correct detection.</p>
</figcaption>
</figure>
<p>This problem is most likely caused by contamination of either the needle or the chip with proteins (<a href="#fig4">Figure 4</a>B, lower panel). As a result, the tip of the punch needle becomes sticky. The needle breaks the bottom of the well, but the latter does not fall, and instead sticks to the chip as soon as the needle returns to the start position.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Avoid high amounts of protein in the single cell suspension, for example by washing the cells multiple times with PBS after trypsinization. Resuspend the cells in serum-free medium. Finally, make sure to carefully wash the back side of the chip at least 5 times with 1 mL PBS before placing it on the microscope. The surface of the back side should look dry and clean after visual inspection. Additionally, after mounting the chip on the microscope the reflection channel can be used to further inspect the surface.</p>
<p>See <a href="#step-by-step-method-details">step-by-step method details</a> (steps 4–6 and 25).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Many well-bottoms are shattered instead of punched, or not punched out at all (<a href="#fig4">Figure 4</a>D). This problem may occur when the round tip of the needle becomes sharp. We have for example seen this when the microsphere that is glued to the tip of the needle falls off. The tip is typically robust enough to punch a full 384-well plate. However, if the calibration was not done properly and the metal between the wells is hit during an unsuccessful punch, the microsphere will fall off (<a href="#fig4">Figure 4</a>G). The needle tip is now sharp metal, and it will not punch out the well correctly.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Cancel the punch process and immediately replace the needle when it accidentally punches the metal of the chip. This can be done while the experiment is still ongoing. To prevent mis-punching, regularly recalibrate the needle after a limited number of punches. The latter can be done by checking the option to recalibrate after any desired number of punches in the start settings.</p>
<p>See <a href="#before-you-begin">before you begin</a> (steps 11–15).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The cell suspension liquid stops flowing through the chip after a few minutes when the vacuum is applied. This typically occurs if a large excess of cells is loaded on the chip.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>We advise to load a small excess of cells, but when the far majority of the cups is filled, there are insufficient pores to pull the remaining liquid through. Carefully remove the remaining cell suspension from the top of the chip and continue at step 9 of the <a href="#step-by-step-method-details">step-by-step method details</a> section.</p>
<p>See <a href="#step-by-step-method-details">step-by-step method details</a> (steps 5-6 and 8-9).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>During chip alignment the VYCAP software cannot detect the breaking edge of the wells at the corners of the chip (<a href="#fig4">Figure 4</a>E, left panel). This will result in incorrect alignment of the chip relative to the punch needle.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This problem is typically seen if the back-side of the chip is not sufficiently not cleaned. The solution is to repeat steps 9 and 10 of the step-by-step protocol; in addition, try to carefully wipe the chip clean with a piece of lens-cleaning paper. The breaking edge can also be made more clearly visible by changing the exposure time, or by using the brightfield channel instead of the reflection channel for chip alignment.</p>
<p>See <a href="#step-by-step-method-details">step-by-step method details</a> (steps 9-10, and 12).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Low percentage of cells with high-quality RNA-seq data, and many cells with a high percentage of mitochondrial gene counts (&gt;30%). This is typically due to delays in the overall time between trypsinization and freezing down the plate with punched cells. The instability of mRNA makes it critical to work as fast as possible at all times.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Reduce scanning time by 1) scanning only a part of the chip and 2) reducing exposure time during imaging as much as possible. The visual inspection of the list of automatically selected cups to remove unwanted ones -such as cups containing doublets- can be omitted during the experiments, and done retrospectively. Although these cups will be punched, they can thus be excluded from the final data analysis.</p>
<p>See <a href="#step-by-step-method-details">step-by-step method details</a> (steps 14 and 16) and expected outcomes.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Boekhout, M., Yuan, R., Wondergem, A.P., Segeren, H.A., van Liere, E.A., Awol, N., Jansen, I., Wolthuis, R.M., de Bruin, A., and Westendorp, B. (2016). Feedback regulation between atypical E2Fs and APC/CCdh1 coordinates cell cycle progression. EMBO Rep. <i>3</i>, 414-427. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref1">View at publisher</a></p>
<p id="bib2">Muraro, M.J., Dharmadhikari, G., Grun, D., Groen, N., Dielen, T., Jansen, E., van Gurp, L., Engelse, M.A., Carlotti, F., de Koning, E.J., and van Oudenaarden, A. (2016). A Single-Cell Transcriptome Atlas of the Human Pancreas. Cell. Syst. <i>4</i>, 385-394.e3. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref2">View at publisher</a></p>
<p id="bib3">Sakaue-Sawano, A., Kurokawa, H., Morimura, T., Hanyu, A., Hama, H., Osawa, H., Kashiwagi, S., Fukami, K., Miyata, T., Miyoshi, H., et al. (2008). Visualizing spatiotemporal dynamics of multicellular cell-cycle progression. Cell <i>3</i>, 487-498. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref3">View at publisher</a></p>
<p id="bib4">Satija, R., Farrell, J.A., Gennert, D., Schier, A.F., and Regev, A. (2015). Spatial reconstruction of single-cell gene expression data. Nat. Biotechnol. <i>5</i>, 495-502. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref4">View at publisher</a></p>
<p id="bib5">Segeren, H.A., van Rijnberk, L.M., Moreno, E., Riemers, F.M., van Liere, E.A., Yuan, R., Wubbolts, R., de Bruin, A., and Westendorp, B. (2020). Excessive E2F Transcription in Single Cancer Cells Precludes Transient Cell-Cycle Exit after DNA Damage. Cell. Rep. <i>9</i>, 108449. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref5">View at publisher</a></p>
<p id="bib6">Stevens, M., Oomens, L., Broekmaat, J., Weersink, J., Abali, F., Swennenhuis, J., and Tibbe, A. (2018). VyCAP's puncher technology for single cell identification, isolation, and analysis. Cytometry A <i>12</i>, 1255-1259. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref6">View at publisher</a></p>
<p id="bib7">Yuan, R., Liu, Q., Segeren, H.A., Yuniati, L., Guardavaccaro, D., Lebbink, R.J., Westendorp, B., and de Bruin, A. (2019). Cyclin F-dependent degradation of E2F7 is critical for DNA repair and G2-phase progression. EMBO J. e101430. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00425-1/sref7">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Bart Westendorp (<a href="mailto:b.westendorp@uu.nl">b.westendorp@uu.nl</a>).</p>
<h4>Materials availability</h4>
<p>All materials newly generated in this study are available from the Lead Contact with appropriate regulatory clearances and a completed Materials Transfer Agreement.</p>
<h4>Data and code availability</h4>
<p>Raw and processed single cell RNA-sequencing data generated in this study are available on Gene Expression Omnibus under accession number GSE146759.</p>
<h3>Acknowledgments</h3>
<p>This work is financially supported by <a href="https://doi.org/10.13039/501100004622">KWF</a> Kankerbestrijding (Dutch Cancer Society, project grant 11941-2018-II) and <a href="https://doi.org/10.13039/501100001826">ZonMW</a> (grant 91116011). Further financial support was provided by research infrastructure grants from Utrecht Life Sciences to the Single Cell Analysis Center and the Center for Cell Imaging.</p>
<h3>Author contributions</h3>
<p>H.A.S. conceived and performed experiments, analyzed data, and wrote the manuscript. K.C.A. performed experiments and critically reviewed the manuscript. L.O. performed experiments and critically reviewed the manuscript. B.W. conceived and performed experiments, analyzed data, and wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>H.A.S. and B.W. declare no competing interests. L.O. and K.C.A. are employed by VYCAP B.V.</p>
</section>
</article>