<article>
<section>
<h1 id="header">Immunofluorescence microscopy-based detection of ssDNA foci by BrdU in mammalian cells</h1>
<p><time datetime="2021-11-26">Published: November 26, 2021</time></p>
<p>Susan Kilgas,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Anne E. Kiltie,<sup><a href="#aff2">2</a></sup> and Kristijan Ramadan<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>MRC Oxford Institute for Radiation Oncology, Department of Oncology, University of Oxford, Oxford OX3 7DQ, UK</p>
<p id="aff2"><sup>2</sup>Rowett Institute, University of Aberdeen, Aberdeen AB25 2ZD, UK</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kristijan.ramadan@oncology.ox.ac.uk">kristijan.ramadan@oncology.ox.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100978">10.1016/j.xpro.2021.100978</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>DNA end resection converts broken ends of double-stranded DNA (dsDNA) to 3′-single-stranded DNA (3′-ssDNA). The extent of resection regulates DNA double-strand break (DSB) repair pathway choice and thereby genomic stability. Here, we characterize an optimized immunofluorescence (IF) microscopy-based protocol for measuring ssDNA in mammalian cells by labeling genomic DNA with 5-bromo-2′-deoxyuridine (BrdU). BrdU foci can be detected under non-denaturing conditions by anti-BrdU antibody, providing an accurate and reliable readout of DNA end resection in most mammalian cell lines.</p>
<p>For complete details on the use and execution of this protocol, please refer to Kilgas et al. (2021).</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Single-stranded DNA (ssDNA) detection by BrdU labeling under non-denaturing conditions</li>
<li>Number of ssDNA foci measured by immunofluorescence (IF)-based confocal microscopy</li>
<li>Compatible with co-staining for cell cycle markers</li>
<li>Amenable to semi-quantitative image analysis for quantification</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1202-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the detailed steps for immunofluorescence microscopy (IF)-based detection and quantification of ssDNA foci upon BrdU labeling under native (non-denaturing) conditions. Cells are optionally exposed to ionising irradiation (IR) or hydroxyurea (HU) as a strategy to induce DNA damage/DSB and consequently initiate DNA end resection. This protocol is optimized for T24 human urinary bladder cancer cells, although variations of this protocol exist in several mammalian cell lines subjected to various DNA damaging agents <i>in vitro</i> (<a href="#bib10">He et al., 2018</a>; <a href="#bib5">Broderick et al., 2016</a>; <a href="#bib12">Kao et al., 2001</a>; <a href="#bib9">Halder et al., 2019</a>). Specifically, this protocol: (1) offers a simple cell culture-based technique to label ssDNA in cells exposed to DNA damage; (2) quantifies the number of ssDNA foci per cell using a semi-quantitative method; (3) involves simultaneous labeling of cell cycle markers to study any cell-cycle specific effects on DNA end resection.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>5-Bromo-2′-deoxyuridine (BrdU) Mouse monoclonal (concentration: 1:300)</td>
<td>BD Biosciences</td>
<td>Cat# 347580; RRID:AB_10015219</td>
</tr>
<tr>
<td>Cyclin A2 antibody (concentration: 1:500)</td>
<td>Novus Biologicals</td>
<td>Cat# NBP1-31330, RRID:AB_10003781</td>
</tr>
<tr>
<td>Alexa Fluor 488 Mouse polyclonal (concentration: 1:500)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A-21202; RRID:AB_141607</td>
</tr>
<tr>
<td>Alexa Fluor 568 Rabbit polyclonal (concentration: 1:500)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A-11011; RRID: AB_143157</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>BrdU</td>
<td>Sigma-Aldrich</td>
<td>Cat# B5002</td>
</tr>
<tr>
<td>5-Ethynyl-2′-deoxyuridine (EdU)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A10044</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>Cat# D2650</td>
</tr>
<tr>
<td>Sodium azide 0.1 M solution</td>
<td>Sigma-Aldrich</td>
<td>Cat# 08591</td>
</tr>
<tr>
<td>ProLong Diamond antifade with DAPI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# P36962</td>
</tr>
<tr>
<td>Hoechst 33342</td>
<td>Sigma-Aldrich</td>
<td>Cat# B2261</td>
</tr>
<tr>
<td>4′-6-Diamidino-2-Phenylindole, Dihydrocloride (DAPI)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #D1306</td>
</tr>
<tr>
<td>Immuno-Mount Aqueous Non-fluorescing Mounting Medium</td>
<td>Interchim</td>
<td>Cat# 983260</td>
</tr>
<tr>
<td>Bovine Serum Albumin (BSA)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 9048-46-8</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11573397</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.25%)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25200072</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS)</td>
<td>Gibco</td>
<td>Cat# 10010023</td>
</tr>
<tr>
<td>Hydroxyurea (HU)</td>
<td>Sigma</td>
<td>Cat# H8627</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat# T8787</td>
</tr>
<tr>
<td>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid(HEPES), pH 7.5</td>
<td>Sigma-Aldrich</td>
<td>Cat# H4034</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat# S0389</td>
</tr>
<tr>
<td>Magnesium chloride (MgCl<sub>2</sub>)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9530G</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>Sigma-Aldrich</td>
<td>Cat# S5150</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid (EDTA)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15575020</td>
</tr>
<tr>
<td>Paraformaldehyde (PFA) 36% (39% w/v)</td>
<td>VWR</td>
<td>Cat# 50-00-0</td>
</tr>
<tr>
<td>Click-IT EdU Alexa Fluor 555 imaging kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# C10338</td>
</tr>
<tr>
<td>McCoy’s 5A Medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 16600082</td>
</tr>
<tr>
<td>Trypan blue solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15250061</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Human: T-24 urinary bladder cancer, transitional cell carcinoma grade III</td>
<td>German Collection of Microorganisms and Cell Cultures (DSMZ)</td>
<td>DSMZ No 376; RRID:CVCL_0554</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism 8</td>
<td>GraphPad Software</td>
<td>
<a href="https://www.graphpad.com">https://www.graphpad.com</a>; RRID:SCR_002798</td>
</tr>
<tr>
<td>Fiji</td>
<td>NIH</td>
<td>
<a href="https://imagej.net/Fiji/Downloads">https://imagej.net/Fiji/Downloads</a>; RRID:SCR_002285</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Round cover glasses; 9 mm; thickness: No. 1</td>
<td>VWR</td>
<td>Cat# 631-0169</td>
</tr>
<tr>
<td>6-well plate, round</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 174901</td>
</tr>
<tr>
<td>T75 flasks</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 156499</td>
</tr>
<tr>
<td>Forceps</td>
<td>Sigma-Aldrich</td>
<td>Cat# F3767</td>
</tr>
<tr>
<td>Kimwipes</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 06-666</td>
</tr>
<tr>
<td>Superfrost Plus Microscope Slides</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #22-037-246</td>
</tr>
<tr>
<td>Caesium-137 GSR D1 gamma irradiator (1.5 Gy/min)</td>
<td>Gamma-Service Medical GmbH</td>
<td>N/A</td>
</tr>
<tr>
<td>Zeiss 710 confocal microscope</td>
<td>ZEISS</td>
<td>N/A</td>
</tr>
<tr>
<td>Nikon Ni-E epifluorescent microscope</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Preparation of BrdU stock solution</caption>
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BrdU</td>
<td>10 μM</td>
<td>3.071 g</td>
</tr>
<tr>
<td>DMSO</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>1 mL stock solution</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Final concentration of 10 μM BrdU was obtained from a 10 mM stock by dissolving 3.071 g of BrdU powder (MW= 307.10 g/mol) in 1 mL DMSO.</div>
<table id="undtbl2">
<caption>Preparation of EdU stock solution</caption>
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EdU</td>
<td>10 μM</td>
<td>2.5223 g</td>
</tr>
<tr>
<td>DMSO</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>1 mL stock solution</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Final concentration of 10 μM EdU was obtained from a 10 mM stock by dissolving 3.071 g of BrdU powder (MW= 252.23 g/mol) in 1 mL DMSO.</div>
<table id="undtbl3">
<caption>Fixing solution (prepare fresh, cool on ice before use)</caption>
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Paraformaldehyde (PFA) (36%)</td>
<td>4%</td>
<td>5.55 mL</td>
</tr>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>44.45 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde (PFA) is classified as a hazardous chemical by the 2012 OSHA Hazard Communication Standard (29 CFR 1910.1200). Harmful if swallowed or inhaled (H302+H332). Causes skin irritation (H315), serious eye damage (H318). May cause respiratory irritation (H335). Suspected carcinogen (H351). Avoid contact with eyes, skin, or on clothing. Avoid ingestion and inhalation. Ensure adequate ventilation in the room of handling. Keep away from sources of ignition and heat. Wear personal protective equipment (PPE) when handling.</div>
<div class="note">
<span class="note-title">Note:</span> Paraformaldehyde (4%) fixing solution should be prepared fresh on the day.</div>
<table id="undtbl4">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>5%</td>
<td>5 g</td>
</tr>
<tr>
<td>MilliQ water</td>
<td>n/a</td>
<td>95 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>100 mL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Bovine serum albumin (BSA) is harmful if swallowed (H302). May be irritating to the mucous membranes and upper respiratory tract. May be harmful by inhalation or absorption by the skin, and cause eye irritation. Wear Personal Protective Equipment (PPE) when handling.</div>
<div class="note">
<span class="note-title">Note:</span> Blocking buffer can be maintained for weeks at 4°C upon addition of 1 mM sodium azide.</div>
<table id="undtbl5">
<caption>Antibody dilution buffer</caption>
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5% BSA in PBS</td>
<td>2.5%</td>
<td>5 mL</td>
</tr>
<tr>
<td>1× PBS</td>
<td>1:1 ratio with 5% BSA/PBS</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The volume of actual antibody dilution buffer depends on the number of samples and experiments. Recommended to use fresh immediately after preparation. Keep on ice or at 4°C.</div>
<table id="undtbl6">
<caption>Pre-extraction buffer (prepare fresh, cool on ice before use)</caption>
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hepes pH 7.5 (1 M)</td>
<td>25 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>50 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>EDTA (500 mM)</td>
<td>1 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>3 mM</td>
<td>30 μL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>300 mM</td>
<td>1.026 g</td>
</tr>
<tr>
<td>Triton X-100 (10% in ddH<sub>2</sub>O)</td>
<td>0.5%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>9.1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Triton X-100 is harmful if swallowed (H302). May cause serious eye damage (H318) and skin irritation (H315). Wear PPE when handling.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ethylenediaminetetraacetic acid (EDTA) is harmful if inhaled (H332). Avoid contact with eyes, skin, or clothing. Avoid inhalation and wear PPE when handling.</div>
<div class="note">
<span class="note-title">Note:</span> Pre-extraction buffer stock solutions can be stored at RT (20°C–22°C) for at least 1 month.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Alternative cytoskeletal (CSK) pre-extraction buffer (<a href="#bib4">Britton et al., 2013</a>):</div>
<table id="undtbl7">
<thead>
<tr>
<th>Compound</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PIPES pH 7.5 (0.5 M)</td>
<td>10 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>100 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>3 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>300 mM</td>
<td>1 g</td>
</tr>
<tr>
<td>Triton X-100 (10% in ddH<sub>2</sub>O)</td>
<td>0.7%</td>
<td>700 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>8.8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Pre-extraction buffer stock solutions can be stored at RT (20°C–22°C) for at least 1 month. PIPES should be stored at −20°C.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cell culture and BrdU treatment (± IR)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol>
<li>Detach T24 cells from maintenance culture flasks (T75 flasks) using Trypsin-EDTA (0.25%) by incubating flasks for a few minutes at 37°C and 5% CO<sub>2</sub>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cell culture should be performed under a Class II Biological Safety Cabinet. Pre-warm cell culture medium and Trypsin-EDTA solution to avoid excessive thermal shifts in cells; Before starting any treatments, check whether cell morphology appears normal, and make sure there is no contamination present in any of the wells. To avoid contamination, make sure the forceps and the coverslips are sterilized before using them for cell culture. Do not open cell culture forceps outside the biological safety cabinet and always sterilize with 70% ethanol after use. Coverslip sterilization can be achieved by autoclaving.</div>
<div class="note">
<span class="note-title">Note:</span> Cell detachment time depends on the cell type and the presence of medium leftovers in the flask. Before proceeding with next steps, ensure that cells are detached from culture flasks.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Mechanical scraping using sterile scrapers and PBS or PBS-EDTA solution (3 mM EDTA) can be used instead of trypsin-EDTA.</div>
<ol start="2">
<li>Neutralize trypsin by adding at least 2 volumes of complete media (McCoy’s 5A Medium with 10% FBS).</li>
<li>Collect cells into a Falcon tube of appropriate size.</li>
<li>Centrifuge cells at 900–1,000 × <i>g</i> at 20°C–22°C for 3–4 min.</li>
<li>Gently aspirate media so as not to disturb the cell pellet at the bottom.</li>
<li>Resuspend cell pellets in complete culture medium.</li>
<li>Seed cells into 6-well plates so they are around 50%–60% confluent the next day. Typically, for T24 cells: 0.4 × 10<sup>6</sup> cells per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each well in a 6-well plate is a separate condition, so the number of coverslips needed per well depends on how many time points and technical replicates are required for a specific experiment</div>
<ol start="8">
<li>Count cells in trypan blue solution (1:1 ratio of cells and trypan blue reagent) using a standard or automated hemocytometer.</li>
<li>Add sterilized round cover glasses (9 mm; thickness No.1) to the bottom of the wells with fresh media in it. Gently press down with forceps to make sure that the cover glasses stick to the bottom of the wells. Then, add cell suspension to the wells (total volume to 2 mL).</li>
<li>The next day, add 10 μM BrdU to 50%–60% confluent cells and keep on for 24 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When using other chemicals/inhibitors to study their effects on ssDNA generation, add these to the media with BrdU in the last few hours of the 24-h BrdU treatment at a concentration needed according to your specific compound. For example, add HU (500 μM) for 1 h before sample processing.</div>
<ol start="11">
<li>Add 10 μM EdU to cells recovered with the culture media containing BrdU in the last 20 min prior to cell fixation to mark cells in S phase.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of using EdU and the associated “Click-IT” reaction kit for S-phase staining outlined later in this protocol (step 20), cells can be stained with Cyclin A antibody instead. The staining procedure will be outlined below in step 35.</div>
<ol start="12">
<li>Remove media with BrdU/EdU after 20-min EdU treatment, treat cells with DNA damage inducing agents, recover over time or fix immediately.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The recovery time between radiation/other DNA damaging agents and fixation may influence the amount of ssDNA detectable under the microscope (<a href="#bib14">Kilgas et al., 2021</a>). Radiation doses over 2–4 Gy can lead to exhaustion of the ssDNA-binding protein RPA involved in end resection (<a href="#bib18">Toledo et al., 2013</a>), and therefore we recommend limiting irradiation dose to no more than 4 Gy for ssDNA microscopy-based assays. Doses as high as 10 Gy will also result in more cell death and consequently fewer cells available for analysis.</div>
<ol start="13">
<li>Irradiate cells with 2 Gy IR. Non-irradiated cells can be kept in a separate 6-well plate and used as an internal control. During the recovery period after IR, control and irradiated cells are maintained in an incubator in complete media at 37°C and 5% CO<sub>2</sub> for the desired amount of time prior to fixing and processing for IF.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Irradiation time to achieve 2 Gy total dose depends on the specific dose-rate used and varies between different irradiators.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Both gamma- and X-ray based irradiators can be used. Alternatively, radiomimetic drugs such as bleomycin or neocarzinostatin can be used.</div>
<h3 id="sec3.2">Pre-extraction and cell fixation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Approximately 30 min</div>
<ol start="14">
<li>Transfer coverslips into individual wells in a 12-well plate and wash the media off with 1 mL 1× ice-cold PBS.</li>
<li>Aspirate PBS and add 1 mL of ice-cold pre-extraction buffer for 5 min over ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cells can become more likely to detach after pre-extraction. Handle coverslips gently and carefully after this step. Add all the buffers from the wall of the culture dish and not directly onto cells to avoid cell detachment. Also, make sure to keep coverslips on ice during pre-extraction, and do not exceed 5 min to preserve cells (see <a href="#sec7.5">troubleshooting 3</a>). Prepare a pre-extraction buffer fresh every time.</div>
<ol start="16">
<li>Aspirate pre-extraction buffer and wash once with 1 mL cold 1× PBS to remove pre-extraction buffer.</li>
<li>Add 1 mL of freshly prepared ice-cold 4% paraformaldehyde (PFA) in PBS to each well. Incubate for 10 min at 20°C–22°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare 4% PFA fixing solution fresh every time (see <a href="#sec7.1">troubleshooting 1</a> and <a href="#sec7.3">2</a>).</div>
<ol start="18">
<li>Wash cells twice with 1 mL 1× PBS at 20°C–22°C for 2–3 min each.</li>
</ol>
<h3 id="sec3.3">Blocking and EdU detection for S-phase cell detection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Approximately 1 h and 30 min</div>
<ol start="19">
<li>Add 1 mL of 5% BSA in PBS and incubate for 1 h at 20°C–22°C</li>
<li>Prepare Click-iT reaction mixture according to the manufacturer’s protocol (Thermo Fisher).</li>
<li>Add 30 μL droplets of Click-iT reaction solution on a parafilm in a wet chamber.</li>
<li>Lift coverslips from the 12-well plate with forceps, dry extra liquid on a tissue paper and place upside down on the Click-iT reaction buffer.</li>
<li>Incubate for 30 min in the dark.</li>
<li>Transfer coverslips back to the 12-well plate before washing the cells.</li>
<li>Wash cells twice for 2–3 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to keep coverslips in dark after the Click-IT reaction step, as the fluorescent azide dye is light-sensitive. When washing coverslips in the 12-well plate, cover the plates with aluminum foil during washes.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Coverslips can be kept in the blocking solution at 4°C overnight (approximately 16 h) or up to a few days.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Thermo Fisher Scientific <b>“</b>Click-iT” reaction kits come with a variety of different wavelengths for the fluorescent dye and can be chosen according to the choice of BrdU channel and DAPI channel, to avoid any potential cross-reactivity. Alternatively, “Click-IT” reaction kit can be replaced by staining cells with Cyclin A antibody, which would skip both the EdU labelling step (step 11) as well as the “Click-iT” step outlined in steps 20–25.</div>
<h3 id="sec3.4">Antibody staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h and 30 min (up to 4 h and 30 min when Cyclin A antibody is added instead of EdU labelling)</div>
<ol start="26">
<li>Dilute primary BrdU antibody in 2.5% BSA/PBS solution (BrdU antibody concentration: 1:300).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure the coverslip is fully covered in antibody solution (see <a href="#sec7.7">troubleshooting 4</a>).</div>
<ol start="27">
<li>Add 30 μL droplets of antibody solution on parafilm in a wet chamber.</li>
<li>Lift coverslips from the 12-well plate with forceps, dry extra liquid on a tissue paper and place upside down on the antibody droplet. Incubate for 1 h at 20°C–22°C in the dark.</li>
<li>Transfer the coverslips back to the 12-well plate, fixed cells facing up, and add 1 mL 1× PBS at 20°C–22°C into each well for 5 min, then gently remove PBS by using a vacuum pump. Repeat this step 2 additional times (washing step; a total of 3 washing steps).</li>
<li>Dilute secondary fluorescent antibody (Alexa Fluor 488) in antibody dilution buffer.</li>
<li>Add 30 μL droplets of antibody solution on parafilm in a wet chamber.</li>
<li>Lift coverslips from the 12-well plate using forceps as in (3), dry extra liquid and place coverslip upside down on antibody droplets.</li>
<li>Incubate for 1 h at 20°C–22°C in the dark.</li>
<li>Transfer the coverslips back to 12-well plate as in (4), and wash with 1 mL 1× PBS at 20°C–22°C for 3 times and 5 min each.</li>
<li>Dilute Cyclin A antibody in 2.5% BSA/PBS solution (Cyclin A antibody concentration: 1:500).</li>
<li>Repeat steps 27–29.</li>
<li>Dilute secondary fluorescent antibody (Alexa Fluor 594) in antibody dilution buffer.</li>
<li>Repeat steps 31–34.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Coverslips can be kept in primary antibody dilution at 4°C overnight (approximately 16 h) provided they are in a wet chamber so that the antibody solution will not dry out.</div>
<h3 id="sec3.5">Mounting and microscopy image acquisition</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Approximately 1–3 days</div>
<ol start="39">
<li>Use ProLong Diamond antifade with DAPI mounting solution to make individual droplets on microscopy slides and mount the coverslips (one droplet for each coverslip).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid bubbles when putting the mounting solution on microscopy slides as this will greatly decrease the quality of your imaging. Also, the mounting solution hardens relatively quickly, so make sure to add the mounting droplets just before adding coverslips on them (see <a href="#sec7.9">troubleshooting 5</a>).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of using the Prolong Diamond antifade DAPI mounting solution, cells can be incubated in DAPI (Thermo Fisher Scientific; Cat #D1306) with PBS (1 μg/mL final concentration) for 10 min in the dark and mounted using ProLong Glass Antifade Mountant (Thermo Fisher Scientific; Cat# P36980). After incubation with DAPI, wash cells once for 5 min with 1× PBS (Step 30); As an alternative to DAPI dye, nuclear staining can be done with Hoechst 33342 dye (Sigma-Aldrich; Cat# B2261) in PBS (5 μg/mL final concentration) for 10 min at 20°C–22°C in the dark, followed by mounting with ProLong Glass Antifade Mountant (Thermo Fisher Scientific; Cat# P36980).</div>
<ol start="40">
<li>Dip coverslips gently in ddH<sub>2</sub>O to remove salts.</li>
<li>Dry extra liquid on kimwipes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Drying extra liquid should be done superficially and quickly to avoid excessive dilution of the mounting medium. When allowed to completely dry, auto-fluorescence can interfere with sample staining (see <a href="#sec7.9">troubleshooting 5</a>).</div>
<ol start="42">
<li>Place coverslips sample-upper side down (fixed cells facing the mounting solution).</li>
<li>Incubate at 20°C–22°C in the dark for 30 min minimum (up to 24 h at 20°C–22°C to let the mounting medium cure).</li>
<li>Store at 4°C in the dark.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Microscopy slides mounted with ProLong Diamond antifade with DAPI mounting solution can be kept at 4°C in the dark (to avoid photobleaching) for at least a month without losing quality of the images. However, it is recommended to image the slides as soon as possible.</div>
<ol start="45">
<li>Capture images using an epifluorescent or confocal microscope using the Use DAPI channel with a 10× lens to locate cells. Then change the magnification to either 40× or 63× oil lens, focus and capture images with DAPI channel (excitation: 359 nm), FITC channel (BrdU antibody; excitation: 488 nm) and TRITC channel (EdU label/Cyclin A antibody; excitation: 555 nm).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Adjust laser power and gain without overexposing samples even in the brightest condition (see <a href="#sec7.11">troubleshooting 6</a> and <a href="#sec7.13">7</a>). Keep the same settings for each biological condition.</div>
<div class="optional">
<span class="optional-title">Optional:</span> The excitation wavelength can be different for BrdU and Edu-labelled cells, depending on which fluorescent antibody and dye you are using. There are several “Click” reaction kits with different wavelength dyes, and BrdU can be alternatively imaged at higher excitation wavelengths.</div>
<ol start="46">
<li>Save files as original multichannel/composite images because they contain the original pixels from the microscope. RGB images are good for display purposes as they have consistent appearance but have lost original pixel values, so are not suitable for accurate analysis.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Multichannel/composite images may not be readable in some software but can be opened in imageJ/Fiji.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Confocal images of T24 cells exposed to 24-h BrdU treatment and 2 Gy IR show that cells stain positive for BrdU foci/ssDNA foci (<a href="#fig1">Figure 1</a>A). Because stimulating DNA end resection above background threshold and subsequent generation of long stretches of ssDNA requires DSB formation (<a href="#bib19">Zhao et al., 2020</a>), non-treated control cells have no noticeable BrdU foci (<a href="#fig1">Figure 1</a>B). When compared with IR-treated conditions, the DMSO control and HU-treated cells have minimal cytoplasmic staining of BrdU foci compared to IR-treated cells. One of the potential reasons for this could be that IR itself causes ssDNA accumulation in the cytoplasm (<a href="#bib16">Ragu et al., 2020</a>), and the residual cytoplasm that remains after pre-extraction will therefore stain for some ssDNA foci outside the nucleus. Additionally, there could be background foci present depending on the endogenous levels of DNA damage in specific cell lines, but compared to irradiated and treated conditions, they should be significantly less (<a href="#bib14">Kilgas et al., 2021</a>; <a href="#bib15">Mukherjee et al., 2015</a>). A positive control should be used when comparing between treatments, which here is shown as HU in S-phase cells (<a href="#bib7">Feng et al., 2006</a>; <a href="#bib17">Sogo et al., 2002</a>). The replication stress-inducing agent HU causes an accumulation of ssDNA foci in cells in S-phase cells but not outside S-phase (<a href="#fig1">Figure 1</a>C) (<a href="#bib17">Sogo et al., 2002</a>). In addition to IR and replication stress, ssDNA accumulation can be observed with other agents of interest targeting the DDR (<a href="#bib14">Kilgas et al., 2021</a>; <a href="#bib10">He et al., 2018</a>; <a href="#bib13">Kijas et al., 2015</a>; <a href="#bib5">Broderick et al., 2016</a>; <a href="#bib11">Jachimowicz et al., 2019</a>; <a href="#bib2">Bai et al., 2019</a>). In addition, not only the presence of DNA damaging agents, but also the dose and duration of treatment as well as the recovery times will affect the intensity and/or number of BrdU foci being generated (<a href="#bib20">Zhou and Paull, 2021</a>; <a href="#bib6">Despras et al., 2010</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1202-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Confocal microscopy-based representative images showing BrdU foci during and outside S-phase</div>
<p>(A and B) T24 cells treated with 2 Gy IR and recovered over 2 h show generation of nuclear BrdU/ssDNA foci. (B) DMSO-treated control (no IR) T24 cells showing no discernible nuclear BrdU/ssDNA foci (FITC/green channel) being formed either during or outside S-phase (TRITC/red channel).</p>
<p>(C) HU-treated (500 μM for 1 h) positive control T24 cells showing representative formation of BrdU/ssDNA foci in S-phase cells only. Scale bar = 10 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<ul>
<li>The number of BrdU foci (ssDNA foci) is determined by Fiji/Image J with a semi-quantitative approach:</li>
</ul>
<ol>
<li>Open original microscopy images (the CZI file format for the Zeiss 710 confocal microscope used here) by drag and drop option in Fiji (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1202-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Individual steps for semi-automated assessment of BrdU/ssDNA foci</div>
<p>Steps and representative images that illustrate BrdU foci quantification using Fiji/ImageJ software</p>
<p>(A) Steps showing how to open original microscopy images in Fiji program.</p>
<p>(B) Steps showing how to choose individual nuclei in DAPI channel and add the chosen cells to the ROI Manager.</p>
<p>(C) Steps showing how to set tolerance levels to recognize individual foci and subsequently calculate the total number of BrdU foci in cells.</p>
</figcaption>
</figure>
</li>
<li>Select cells in the DAPI channel and click on the “wand tool”, set tolerance and enable thresholding to select individual nuclei. These can then be added to “Region of interest (ROI) Manager” by clicking “Add” or pressing “t” (<a href="#fig2">Figure 2</a>B).</li>
<li>Move to the channel of interest to count individual foci. For this example, it is the FITC channel (Ex 488 nm) for BrdU foci. Select process ▸ find maxima ▸ output type: “single points” ▸ choose “Prominence” (threshold levels of foci recognition) until individual single points are identified with minimal background noise (<a href="#fig2">Figure 2</a>C).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the same threshold levels across all conditions during analysis. Do not change image contrast or intensity (see <a href="#sec7.11">troubleshooting 6</a>).</div>
<ol start="4">
<li>When nuclei are selected on an image, click “Find maxima” and then “Measure” from the ROI Manager.</li>
<li>Number of foci can be determined by dividing the raw intensity (“RawIntDen”) value with the “Max” value.</li>
</ol>
<p>All statistical analysis was done using GraphPad software (PRISM 8). A value of p ≤ 0.05 was considered statistically significant.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The major limitation of this protocol is that it is a semi-quantitative way of analyzing the amount ssDNA foci generated. Despite being a useful way to easily and reliably estimate the number and/or brightness of ssDNA foci generated, when a more quantitative and precise method is required that measures the extent of resection from specific DSB sites, quantitative PCR (qPCR)-based assays might be the best option (<a href="#bib20">Zhou and Paull, 2021</a>). However, it is worth noting that qPCR-based methods do require the generation of stable cell lines, which is more labor-intensive than the current protocol. Additionally, other ways to quantitatively measure DNA end resection are in-gel assays with fluorophores/radioactive isotopes (<a href="#bib3">Bantele et al., 2019</a>), FACS-based methods to quantify ssDNA (<a href="#bib8">Forment and Jackson, 2015</a>) and the DNA fiber-based Single Molecule Analysis of Resection Tracks (SMART) assay. The SMART assay resembles the current protocol, but instead of analyzing BrdU foci, it is based on anti-BrdU antibody detection on DNA fibers, providing a quantitative readout of the length of DNA resection tracks in different conditions (<a href="#bib10">He et al., 2018</a>; <a href="#bib1">Altieri et al., 2020</a>). Finally, image acquisition and analysis using Fiji/Image J are relatively time-consuming and labor-intensive but depend on the number of samples being analyzed.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1: Insufficient fixation before BrdU staining</h3>
<p>When fixation time or the concentration of the fixative is not sufficient, the signal-to-noise ratio for BrdU will decrease, potentially compromising the quality of analysis. This will also lead to loss of sample.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Perform further fixation prior to staining. Altering the fixative (changing PFA to ice-cold methanol) or increasing fixation time could help. Please refer to protocol step 17 for further information.</p>
<h3 id="sec7.3">Problem 2: Cells are over fixed</h3>
<p>When fixation time or the concentration of the fixative is too high, antibody-binding efficiency will decrease due to over-crosslinking. Longer fixation times with PFA will also increase autofluorescence, hampering analysis of the samples.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Reduce the duration of fixation, or alternatively use a lower concentration of PFA. Please refer to protocol step 17 for further information.</p>
<h3 id="sec7.5">Problem 3: Cells were not properly permeabilized and cell detachment was observed</h3>
<p>When cells are not properly permeabilized, antibodies will not be able to access the intracellular components and there will be no discernible BrdU foci or nuclear staining of BrdU. Also, when the pre-extraction buffer is left on for longer than indicated or performed at 20°C–22°C, this will lead to loss of cells.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Fixing cells with methanol will permeabilize cells, so there is no need to add Triton X-100 in extraction buffers, which potentially can avoid problems around cell detachment. However, we recommend the use of PFA for cell fixation for this assay as PFA works better for signal clarity as well as preserving cell structure compared to methanol fixation in our protocol. When cells are fixed with PFA, cells need to be permeabilized with Triton X-100, which is included in the pre-extraction buffer recipes outlined above (see Materials and Equipment). Alternatively, if cell detachment is a persistent issue, the concentration of Triton X-100 could be lowered from 0.5% to 0.25%. Also, handle all coverslips carefully to avoid cell detachment after pre-extraction: keep coverslips on ice, add PBS gently to 12-wells and not directly onto the coverslip. Also, avoid scratching the top of the coverslip with forceps when moving samples around. Refer to protocol step 15 for further information</p>
<h3 id="sec7.7">Problem 4: Not enough primary antibody</h3>
<p>When primary antibody concentration or incubation time is too low, then not enough primary antibody can bind the protein of interest, causing faint samples and low signal-to-noise ratio.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Using a higher concentration of the primary antibody could eliminate this problem. Additionally, incubating for longer, such as 4°C overnight (approximately 16 h) in the dark, could result in better signal. Please refer to protocol step 26 for further information.</p>
<h3 id="sec7.9">Problem 5: High background</h3>
<p>High background fluorescence can result for various reasons: low signal-to-noise ratio from cross-reactivity between different fluorescent channels; primary antibody is not specific enough or not at the right concentration; not enough blocking or inappropriate blocking reagent; not enough washing in between staining steps; over and under fixation of cells; excessive drying of coverslips prior to mounting.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Use of specific antibodies at the recommended concentrations; using siRNA-mediated depletion of protein of interest to see loss of signal compared to siRNA control; checking any fluorescence in an unstained cell. If autofluorescence is present despite siRNA-mediated knockdown (KD) or in unstained control, reduce the concentration of primary and/or secondary antibody, increase duration of blocking time, or consider changing the blocking reagent. If imaging more than one fluorescent probe, choose excitation and emission spectra that do not overlap between different antibodies. Furthermore, dry coverslips very briefly before mounting and do not let the coverslips dry out during antibody incubation or washing times. Finally, make sure to wash coverslips at least three times and for 5 min each in between staining steps and prior to mounting. Please refer to protocol step 41 for further information.</p>
<h3 id="sec7.11">Problem 6: Incorrect intensity settings at the image acquisition step</h3>
<p>Incorrect intensity can lead to either signal being too low to visualize foci or signal being saturated with high background (<a href="#fig3">Figures 3</a>A and B).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1202-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative confocal microscopy images illustrating both overexposed and correctly exposed images</div>
<p>(A) T24 cells were treated with 500 μM HU for 1 h and stained with DAPI, BrdU antibody (FITC/green channel) and fluorescent dye (“Click-iT” Alexa Fluor 555) to mark S-phase cells (TRITC/red channel). All three channels show representative images of incorrect intensity, where images were taking when signal was too saturated with high background. Also note the high background cytoplasmic foci becoming visible in the green/BrdU foci channel (see also <a href="#sec7.5">troubleshooting 3</a>).</p>
<p>(B) Same conditions as in (A), but correctly exposed DAPI, green and red channels. Scale bar = 30 μm. See <a href="#sec7.11">troubleshooting 6</a> for further details.</p>
</figcaption>
</figure>
<h3 id="sec7.12">Potential solution</h3>
<p>Before acquiring images, test out a few controls and a few treated conditions and set the intensity at the right level. Signal should not be saturated even in the brightest condition, and signal should have just enough brightness so foci will be clearly visible without background noise. Please refer to protocol step 45 for further information. Please refer to protocol step 3 of “<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>”.</p>
<h3 id="sec7.13">Problem 7: Fluorescence bleaching</h3>
<p>When the laser is left on the sample for too long, sample bleaching and subsequent loss of signal will occur.</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Avoid overexposing samples to the laser by switching off the light source as soon as imaging is done. Also, store all your coverslips in the dark and avoid any light exposure in between image acquisition and sample storage. Please refer to protocol step 45 for further information.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Altieri, A., Dell'aquila, M., Pentimalli, F., Giordano, A., and Alfano, L. (2020). SMART (single molecule analysis of resection tracks) technique for assessing DNA end-resection in response to DNA damage. Bio Protoc. <i>10</i>, e3701. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref1">View at publisher</a></p>
<p id="bib2">Bai, Y., Wang, W., Li, S., Zhan, J., Li, H., Zhao, M., Zhou, X.A., Li, S., Li, X., Huo, Y., et al. (2019). C1QBP promotes homologous recombination by stabilizing MRE11 and controlling the assembly and activation of MRE11/RAD50/NBS1 complex. Mol. Cell <i>75</i>, 1299-1314.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref2">View at publisher</a></p>
<p id="bib3">Bantele, S.C.S., Lisby, M., and Pfander, B. (2019). Quantitative sensing and signalling of single-stranded DNA during the DNA damage response. Nat. Commun. <i>10</i>, 944. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref3">View at publisher</a></p>
<p id="bib4">Britton, S., Coates, J., and Jackson, S.P. (2013). A new method for high-resolution imaging of Ku foci to decipher mechanisms of DNA double-strand break repair. J. Cell Biol. <i>202</i>, 579-595. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref4">View at publisher</a></p>
<p id="bib5">Broderick, R., Nieminuszczy, J., Baddock, H.T., Deshpande, R., Gileadi, O., Paull, T.T., Mchugh, P.J., and Niedzwiedz, W. (2016). EXD2 promotes homologous recombination by facilitating DNA end resection. Nat. Cell Biol. <i>18</i>, 271-280. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref5">View at publisher</a></p>
<p id="bib6">Despras, E., Daboussi, F., Hyrien, O., Marheineke, K., and Kannouche, P.L. (2010). ATR/Chk1 pathway is essential for resumption of DNA synthesis and cell survival in UV-irradiated XP variant cells. Hum. Mol. Genet. <i>19</i>, 1690-1701. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref6">View at publisher</a></p>
<p id="bib7">Feng, W., Collingwood, D., Boeck, M.E., Fox, L.A., Alvino, G.M., Fangman, W.L., Raghuraman, M.K., and Brewer, B.J. (2006). Genomic mapping of single-stranded DNA in hydroxyurea-challenged yeasts identifies origins of replication. Nat. Cell Biol. <i>8</i>, 148-155. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref7">View at publisher</a></p>
<p id="bib8">Forment, J.V. and Jackson, S.P. (2015). A flow cytometry-based method to simplify the analysis and quantification of protein association to chromatin in mammalian cells. Nat. Protoc. <i>10</i>, 1297-1307. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref8">View at publisher</a></p>
<p id="bib9">Halder, S., Torrecilla, I., Burkhalter, M.D., Popovic, M., Fielden, J., Vaz, B., Oehler, J., Pilger, D., Lessel, D., Wiseman, K., et al. (2019). SPRTN protease and checkpoint kinase 1 cross-activation loop safeguards DNA replication. Nat. Commun. <i>10</i>, 3142. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref9">View at publisher</a></p>
<p id="bib10">He, Y.J., Meghani, K., Caron, M.-C., Yang, C., Ronato, D.A., Bian, J., Sharma, A., Moore, J., Niraj, J., Detappe, A., et al. (2018). DYNLL1 binds to MRE11 to limit DNA end resection in BRCA1-deficient cells. Nature <i>563</i>, 522-526. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref10">View at publisher</a></p>
<p id="bib11">Jachimowicz, R.D., Beleggia, F., Isensee, J., Velpula, B.B., Goergens, J., Bustos, M.A., Doll, M.A., Shenoy, A., Checa-Rodriguez, C., Wiederstein, J.L., et al. (2019). UBQLN4 represses homologous recombination and is overexpressed in aggressive tumors. Cell <i>176</i>, 505-519 e22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref11">View at publisher</a></p>
<p id="bib12">Kao, G.D., Mckenna, W.G., and Yen, T.J. (2001). Detection of repair activity during the DNA damage-induced G2 delay in human cancer cells. Oncogene <i>20</i>, 3486-3496. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref12">View at publisher</a></p>
<p id="bib13">Kijas, A.W., Lim, Y.C., Bolderson, E., Cerosaletti, K., Gatei, M., Jakob, B., Tobias, F., Taucher-Scholz, G., Gueven, N., Oakley, G., et al. (2015). ATM-dependent phosphorylation of MRE11 controls extent of resection during homology directed repair by signalling through Exonuclease 1. Nucleic Acids Res. <i>43</i>, 8352-8367. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref13">View at publisher</a></p>
<p id="bib14">Kilgas, S., Singh, A.N., Paillas, S., Then, C.K., Torrecilla, I., Nicholson, J., Browning, L., Vendrell, I., Konietzny, R., Kessler, B.M., et al. (2021). p97/VCP inhibition causes excessive MRE11-dependent DNA end resection promoting cell killing after ionizing radiation. Cell Rep. <i>35</i>, 109153. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref14">View at publisher</a></p>
<p id="bib15">Mukherjee, B., Tomimatsu, N., and Burma, S. (2015). Immunofluorescence-based methods to monitor DNA end resection. Methods Mol. Biol. <i>1292</i>, 67-75. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref15">View at publisher</a></p>
<p id="bib16">Ragu, S., Matos-Rodrigues, G., and Lopez, B.S. (2020). Replication stress, DNA damage, inflammatory cytokines and innate immune response. Genes <i>11</i>, 409. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref16">View at publisher</a></p>
<p id="bib17">Sogo, J.M., Lopes, M., and Foiani, M. (2002). Fork reversal and ssDNA accumulation at stalled replication forks owing to checkpoint defects. Science <i>297</i>, 599-602. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref17">View at publisher</a></p>
<p id="bib18">Toledo, L.I., Altmeyer, M., Rask, M.B., Lukas, C., Larsen, D.H., Povlsen, L.K., Bekker-Jensen, S., Mailand, N., Bartek, J., and Lukas, J. (2013). ATR prohibits replication catastrophe by preventing global exhaustion of RPA. Cell <i>155</i>, 1088-1103. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref18">View at publisher</a></p>
<p id="bib19">Zhao, F., Kim, W., Kloeber, J.A., and Lou, Z. (2020). DNA end resection and its role in DNA replication and DSB repair choice in mammalian cells. Exp. Mol. Med. <i>52</i>, 1705-1714. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref19">View at publisher</a></p>
<p id="bib20">Zhou, Y. and Paull, T.T. (2021). Quantifying DNA end resection in human cells. Methods Mol. Biol. <i>2153</i>, 59-69. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00684-5/sref20">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Kristijan Ramadan (<a href="mailto:kristijan.ramadan@oncology.ox.ac.uk">kristijan.ramadan@oncology.ox.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was funded by the MRC Program grant MC_PC 12001/1 and MC_UU_00001/1 to K.R., and S.K. was supported by the MRC Oxford Institute of Radiation Oncology (OIRO) <a href="https://doi.org/10.13039/501100000289">Cancer Research UK</a> (CRUK) Studentship. We thank Dr. Rhodri Wilson from the microscopy imaging core (University of Oxford, Department of Oncology) for his technical advice and assistance. Graphical abstract was created with BioRender.</p>
<h3>Author contributions</h3>
<p>Methodology, S.K.; validation, S.K.; writing – original draft, S.K.; writing – review &amp; editing, S.K., A.E.K., and K.R.; supervision, K.R. and A.E.K.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>