<article>
<section>
<h1 id="header">Optimized protocols for ChIP-seq and deletion mutant construction in <i>Pseudomonas syringae</i>
</h1>
<p><time datetime="2021-08-27">Published: August 27, 2021</time></p>
<p>Chunyan Yao,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Xiaolong Shao,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Jingwei Li,<sup><a href="#aff1">1</a></sup> and Xin Deng<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biomedical Science, City University of Hong Kong, Kowloon Tong, Hong Kong SAR, China</p>
<p id="aff2"><sup>2</sup>Shenzhen Research Institute, City University of Hong Kong, Shenzhen 518057, China</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:xindeng@cityu.edu.hk">xindeng@cityu.edu.hk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100776">10.1016/j.xpro.2021.100776</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Chromatin immunoprecipitation sequencing (ChIP-seq) is an efficient technique to identify the binding sites of transcription factors (TFs) in both eukaryotes and prokaryotes. However, its application in bacteria is very heterogeneous. In this protocol, we optimized the methods of ChIP-seq that can be widely applied to plant pathogens. We used homologous recombination to construct pK18mobsacB-<i>Psph</i> plasmid instead of restriction site ligation and replaced transconjugation with electroporation transformation in <i>Pseudomonas syringae</i> deletion mutant construction, which is more efficient and faster than previous methods.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref15">Shao et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>This approach can be used to construct TF-overexpressed <i>Pseudomonas syringae</i> strain</li>
<li>Protocols for ChIP-seq library construction of <i>Pseudomonas syringae</i>
</li>
<li>A simplified procedure to construct a deletion mutant of <i>Pseudomonas syringae</i>
</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/964-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Introduction</h3>
<p><i>Pseudomonas syringae</i> is a Gram-negative pathogenic bacterium, causing deadly diseases of more than 50 plants and huge economic lost in agriculture worldwide (<a href="#bib6">Gonzalez et al., 2000</a>; <a href="#bib7">Hirano and Upper, 2000</a>). <i>Pseudomonas syringae</i> relies on type III secretion system (T3SS) to invade hosts (<a href="#bib1">Cunnac et al., 2009</a>; <a href="#bib5">Galan and Collmer, 1999</a>). Transcription factors (TFs) control the rate of transcription by binding to specific downstream DNA sequences (<a href="#bib9">Karin, 1990</a>; <a href="#bib13">Latchman, 1997</a>), which are of great importance for regulation of virulence and metabolism in <i>Pseudomonas syringae</i>. We have identified the binding sites of nine T3SS TFs, RhpR, AefR, HrpS, Lon, OmpR, CbrB2, PhoP, PilR, and MgrA in <i>P. syringae</i> by ChIP-seq (<a href="#bib2">Deng et al., 2010</a>; <a href="#bib3">Deng et al., 2014</a>; <a href="#bib4">Deng et al., 2009</a>; <a href="#bib8">Hua et al., 2020</a>; <a href="#bib15">Shao et al., 2021</a>; <a href="#bib16">Wang et al., 2018</a>; <a href="#bib17">Xiao et al., 2007</a>; <a href="#bib18">Xie et al., 2019</a>; <a href="#bib19">Zhou et al., 2016</a>). However, the protocols of ChIP-seq are not unified in the study of plant pathogenic bacteria compared with eukaryotes.</p>
<p>In addition, phenotypes and gene expression level in deletion mutants are essential for determining the functions of TFs. In this protocol, we provided an optimized method for ChIP-seq in <i>P. syringae</i> and it can be used in other plant pathogenic bacteria. We also use an efficient homologous recombination strategy to construct the pK18mobsacB -<i>Psph</i> suicide plasmid instead of restriction site ligation, which simplifies the procedures in a previous method (<a href="#bib14">McDowell, 2011</a>).</p>
<h3 id="sec1.2">Primer design</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol>
<li>Primers design of the pHM1-TF (<a href="#bib12">Labes et al., 1990</a>) plasmids overexpressing each TF tagged by hemagglutinin (HA).</li>
</ol>
<p>This STAR protocol uses <i>cbrB2</i> (PSPPH_0857) as an example. The opening reading frame (ORF) of <i>cbrB2</i> is amplified from <i>P. syringae</i> pv. <i>phaeseolicola</i> (<i>Psph</i>) chromosomal DNA using forward and reverse primers (pHM1-<i>cbrB2</i>-F and pHM1-<i>cbrB2</i>-HA-R). Primers are designed as the operation instruction of ClonExpress II One Step Cloning Kit. As shown in <a href="#fig1">Figure 1</a>A, the left homology arm of pHM1 plasmid (5′ATGACCATGATTACGCCAAGCTT3′) is added to the 5′ end of the forward primer. Then, the right homology arm of pHM1 plasmid and HA-tag sequence (5′GACCTGCAGGCATGCAAGCTTTTAAGCGTAATCTGGAACATCGTATGGGTA3′) is added to the 3′ end of the reverse primer.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/964-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Primers design of pHM1-<i>cbrB2-</i>HA plasmid and <i>Psph</i> deletion mutant construction</div>
<p>(A) Primers design of pHM1-<i>cbrB2</i> plasmid.</p>
<p>(B) Primers design of <i>Psph</i> deletion mutant construction.</p>
</figcaption>
</figure>
<ol start="2">
<li>Primers design of <i>Psph</i> deletion mutant construction (<a href="#fig1">Figure 1</a>B).<br> Four pairs of primers are required for each mutant construction. The sizes of upstream and downstream fragments need to differ by 100–500-bp to differentiate them in agarose gel. Generally, these two fragments are about 1,000-bp and 1,500-bp, respectively.
<ol type="a">
<li>Upstream fragment primers. A pair of Upstream fragment primers are used to amplify the upstream fragment of <i>cbrB2.</i>The left homology arm of pK18mobsacB suicide plasmid (5′AAACAGCTATGACATGATTACGAATTC3′) is added to the 5′ end of the forward primer (pK18-<i>cbrB2</i>-Upstream fragment-F). The <i>Xba</i>I restriction site (CCTCTAGA) is added to the 5′ end of the reverse primer (pK18-<i>cbrB2</i>- Upstream fragment-R).</li>
<li>Downstream fragment primers. A pair of downstream fragment primers is used to amplify the downstream fragment of <i>cbrB2.</i> The <i>Xba</i>I restriction site (CCTCTAGA) is added to the 5′ end of the forward primer (pK18-<i>cbrB2</i>-Downstream fragment-F). The right homology arm of pK18mobsacB suicide plasmid (5′AACGACGGCCAGTGCCAAGCTT3′) was added to the 5′ end of the reverse primer (pK18-<i>cbrB2</i>-Downstream fragment-R).</li>
<li>A pair of verification primers (<i>cbrB2</i>-verify-F and <i>cbrB2</i>-verify-R) outside the upstream fragment reverse primer and the downstream fragment forward primer are designed to amplify the connected part of the upstream and downstream fragments.</li>
<li>A pair of qRT-PCR primers (<i>cbrB2</i>-RT-F and <i>cbrB2</i>-RT-R) designed to genomic location that has been deleted are used to verify the deletion.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>HA Tag Monoclonal Antibody (1:10000 diluted with PBST)</td>
<td>Invitrogen</td>
<td>Catalog # 26183-1MG</td>
</tr>
<tr>
<td>Monoclonal Anti-HA-Agarose antibody produced in mouse (1:10000 diluted with PBST)</td>
<td>Sigma-Aldrich</td>
<td>Catalog # A2095</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E.coli</i> DH5α (<i>endA hsdR17 supE44 thi-1 recA1 gyrA relA1</i> D (<i>lacZYA-argF</i>) U169 <i>deoR</i> (ɸ80<i>dlacD</i> (<i>lacZ</i>) M15))</td>
<td>Tiangen Biotech</td>
<td>Catalog # CB101</td>
</tr>
<tr>
<td>
<i>Pseudomonas syringae</i> pv. <i>phaseolicola</i> 1448A (<i>Psph</i>)</td>
<td>(<a href="#bib3">Deng et al., 2014</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>
<i>Xba</i>I</td>
<td>NEB</td>
<td>Catalog # R0145M</td>
</tr>
<tr>
<td>
<i>Eco</i>RI-HF</td>
<td>NEB</td>
<td>Catalog # R3101M</td>
</tr>
<tr>
<td>
<i>Hind</i>III-HF</td>
<td>NEB</td>
<td>Catalog # R3104M</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>Takara</td>
<td>Catalog # 2011A</td>
</tr>
<tr>
<td>Taq Master Mix for PAGE</td>
<td>Vazyme</td>
<td>Catalog # P115-01</td>
</tr>
<tr>
<td>Proteinase K stock</td>
<td>Tiangen</td>
<td>Catalog # RT403</td>
</tr>
<tr>
<td>dNTP Mix</td>
<td>NEB</td>
<td>Catalog # N0447S</td>
</tr>
<tr>
<td>T4 DNA Polymerase</td>
<td>NEB</td>
<td>Catalog # M0203L</td>
</tr>
<tr>
<td>Klenow DNA polymerase (1 U/μL)</td>
<td>NEB</td>
<td>Catalog # M0210L</td>
</tr>
<tr>
<td>T4 polynucleotide kinase</td>
<td>NEB</td>
<td>Catalog # M0201L</td>
</tr>
<tr>
<td>dATP</td>
<td>NEB</td>
<td>Catalog # N0440S</td>
</tr>
<tr>
<td>Klenow exo</td>
<td>NEB</td>
<td>Catalog # M0212L</td>
</tr>
<tr>
<td>Phusion High-Fidelity PCR Master Mix</td>
<td>NEB</td>
<td>Catalog # M05315</td>
</tr>
<tr>
<td>NEBNext® Multiplex Oligos for Illumina</td>
<td>NEB</td>
<td>Catalog # E6612A</td>
</tr>
<tr>
<td>Adaptor Dilution Buffer</td>
<td>NEB</td>
<td>Catalog # B1430S</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>DNA Purification Kit</td>
<td>Tiangen</td>
<td>Catalog # DP214-03</td>
</tr>
<tr>
<td>Rapid Mini Plasmid Kit</td>
<td>Tiangen</td>
<td>Catalog # DP105-03</td>
</tr>
<tr>
<td>ClonExpress II One Step Cloning Kit</td>
<td>Vazyme</td>
<td>Catalog # C112-01</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>QIAGEN</td>
<td>Catalog # 28106</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>Catalog # 28706</td>
</tr>
<tr>
<td>Oligo Clean &amp; Concentrator</td>
<td>Zymo</td>
<td>Catalog # D4061</td>
</tr>
<tr>
<td>Qubit™ dsDNA HS Assay Kit</td>
<td>Invitrogen</td>
<td>Catalog # Q32851</td>
</tr>
<tr>
<td>Quick Ligation Kit</td>
<td>NEB</td>
<td>Catalog # M2200L</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>pHM1-<i>cbrB2</i>-F:ATGACCATGATTACGCCAAGCTTACCGTCGAGAGAACTGAA</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>pHM1-<i>cbrB2-HA</i>-R:GACCTGCAGGCATGCAAGCTTTTAAGCGTAATCTGGAACATCGTATGGGTATCGCCAACGAGACGTAA</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>pK18-<i>cbrB2</i>-Upstream fragment-F:AAACAGCTATGACATGATTACGAATTCGGGTTGATGGGGCCAAGT</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>pK18-<i>cbrB2</i>-Upstream fragment-R(<i>Xba</i>I):AATCTAGA AGGCTGGCGTAACTGGTC</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>pK18-<i>cbrB2</i>-Downstream fragment-F(<i>Xba</i>I):AATCTAGA TTATGCCCGCAACAACCG</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>pK18-<i>cbrB2</i>-Downstream fragment-R:AACGACGGCCAGTGCCAAGCTTTCCTGGGGATGATTGGCG</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>cbrB2</i>-verify-F: AAGTCAGTGAAGCCGGCT</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>cbrB2</i>-verify-R: TGTTGTTCTCGGCCTGGT</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>cbrB2</i>-RT-F: ATCGCCCTGAAACTCCCC</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>cbrB2</i>-RT-R: AACTTCAGGTCGTCGCGA</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pK18mobsacB (suicide plasmid for deletion mutant construction, Kan<sup>r</sup>)</td>
<td>ATCC (<a href="#bib11">Kvitko and Collmer, 2011</a>)</td>
<td>ATCC87097</td>
</tr>
<tr>
<td>pK18mobsacB-<i>cbrB2</i> (suicide plasmid for <i>Psph cbrB2</i> deletion mutant, Kan<sup>r</sup>)</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>pHM1 plasmid</td>
<td>(<a href="#bib12">Labes et al., 1990</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pHM1-<i>CbrB2</i>-HA (overexpressing CbrB2 for ChIP-seq, Spe<sup>r</sup>)</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism 8</td>
<td>Software</td>
<td>RRID: SCR_002798 <a href="https://www.graphpad.com/">https://www.graphpad.com/</a>
</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>T100<sup>TM</sup> Thermal Cycle</td>
<td>Bio-Rad</td>
<td>Catalog # 1861096</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Thermo Fisher</td>
<td>Catalog # 75002435</td>
</tr>
<tr>
<td>Centrifuge (4°C)</td>
<td>Thermo Fisher</td>
<td>Catalog # 75002445</td>
</tr>
<tr>
<td>Ultrasonic Cell Crusher</td>
<td>Scientz</td>
<td>Catalog # JY96-IIN</td>
</tr>
<tr>
<td>Electrophoresis power supplies</td>
<td>Bio-Rad</td>
<td>Catalog # 1645070</td>
</tr>
<tr>
<td>MicroPulser Electroporator</td>
<td>Bio-Rad</td>
<td>Catalog # 1652100</td>
</tr>
<tr>
<td>Gene Pulser/MicroPulser Electroporation Cuvettes, 0.1 cm gap</td>
<td>Bio-Rad</td>
<td>Catalog # 1652083</td>
</tr>
<tr>
<td>Obital Shaking Incubator</td>
<td>Yihder</td>
<td>Catalog # LM-400D</td>
</tr>
<tr>
<td>HulaMixer Sample Mixer</td>
<td>Thermo Fisher</td>
<td>Catalog # 15920D</td>
</tr>
<tr>
<td>Qubit4 Fluorometer</td>
<td>Thermo Fisher</td>
<td>Catalog # Q33226</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>King’s B</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BactoTM Proteose peptone No.3</td>
<td>N/A</td>
<td>20 g</td>
</tr>
<tr>
<td>MgSO<sub>4</sub> 7H<sub>2</sub>O</td>
<td>N/A</td>
<td>1.5 g</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>N/A</td>
<td>1.5 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>N/A</td>
<td>15 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve in 1 L ddH<sub>2</sub>O and adjust the pH to 7.2. Add 15 g/L agar for solidified medium and autoclave. Prepare before use.</div>
<table id="undtbl2">
<caption>Luria-Bertani</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Trypton</td>
<td>N/A</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>N/A</td>
<td>5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>N/A</td>
<td>10 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve in 1 L ddH<sub>2</sub>O and adjust the pH to 7.2. Add 15 g/L agar for solidified medium and autoclave. Prepare before use.</div>
<table id="undtbl3">
<caption>lysis buffer basic</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5M)</td>
<td>0.14M</td>
<td>28 mL</td>
</tr>
<tr>
<td>HEPES (1M)</td>
<td>15mM</td>
<td>15 mL</td>
</tr>
<tr>
<td>EDTA pH8.0 (0.5M)</td>
<td>1mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>10% sodium deoxycholate</td>
<td>N/A</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>945 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>N/A</td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Stock needs to be autoclaved. Store at 4°C. Stable for 1 month.</div>
<table id="undtbl4">
<caption>lysis buffer complete=IP buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>lysis buffer basic</td>
<td>N/A</td>
<td>45 mL</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>DTT (1M)</td>
<td>0.5mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>cocktail-inhibitors (Roche)</td>
<td>N/A</td>
<td>1 tablet</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C. Stable for 1 month.</div>
<table id="undtbl5">
<caption>TE buffer:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1M Tris-HCl pH 8.0</td>
<td>10mM</td>
<td>500 μL</td>
</tr>
<tr>
<td>0.5M EDTA</td>
<td>1mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>49.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 25°C. Stable for 2–3 months.</div>
<table id="undtbl6">
<caption>Elution buffer 1</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (volume)</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.5M EDTA</td>
<td>10mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>20% SDS</td>
<td>1%</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>1 M Tris-HCl</td>
<td>50mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>44 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 25°C. Stable for 2–3 months.</div>
<table id="undtbl7">
<caption>Elution buffer 2</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (volume)</th>
</tr>
</thead>
<tbody>
<tr>
<td>TE</td>
<td>N/A</td>
<td>14.5 mL</td>
</tr>
<tr>
<td>20% SDS</td>
<td>0.67%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>15 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 25°C. Stable for 2–3 months.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This STAR protocol uses TF CbrB2 as an example. The flow diagram of overexpressed TF strain construction and ChIP are showed in <a href="#fig2">Figure 2</a>.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/964-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. The flow diagram of overexpression of <i>cbrB2</i>, ChIP, and library construction</div>
</figcaption>
</figure>
<h3 id="sec3.1">Overexpressed TF strain construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<h4 id="sec3.1.1">Construction of pHM1-<i>cbrB2-</i>HA plasmid</h4>
<ol>
<li>Amplify the ORF of <i>Psph cbrB2</i> gene from <i>Psph</i> chromosomal DNA using 20 μL PCR reaction (18 μL Taq premix, 0.5 μL pHM1-<i>cbrB2</i>-F primer, 0.5 μL pHM1-<i>cbrB2</i>-HA-R primer and 20 ng <i>Psph</i> chromosomal DNA). The concentration of primers is 10 μM.
<ol type="a">
<li>The following thermocycle program is recommended for PCR amplification.
<table id="undtbl8">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min/Kb</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>The PCR product is checked by 1% agarose gel. The band is specific and at the right size (1529 bp) (<a href="#fig3">Figure 3</a>A). The PCR product was purified by DNA purification kit.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/964-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. The experimental results of pHM1-<i>cbrB2</i>-HA plasmid, ChIP, and library construction</div>
<p>(A) PCR amplification of <i>Psph cbrB2</i> gene. The band is specific and at the right size (1529 bp).</p>
<p>(B) pHM1 is digested by <i>Hind</i>III. The plasmid is digested to a linear fragment.</p>
<p>(C) PCR verification of pHM1<i>-cbrB2</i>-HA construction. Two pHM1<i>-cbrB2</i>-HA plasmids are at the right size of 1529 bp.</p>
<p>(D) Detection of the expression of CbrB2-HA protein in the OX-<i>cbrB2</i> and <i>Psph</i> WT/pHM1 strain by Western blot.</p>
<p>(E) DNA bands after sonication.</p>
<p>(F) Size selection by 2% low melting point agarose gel.</p>
<p>(G) PCR amplification production of libraries.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Digest the pHM1 plasmid by <i>Hind</i>III. The pHM1 is extracted from the 1 mL mid-log phase liquid culture. Incubate 2 μg pHM1 plasmid with <i>Hind</i>III in a 50-μL reaction at 37°C for 1 h . The digested pHM1 plasmid is separated by 0.8% agarose gel. The band of linear plasmid is above the circular plasmid (<a href="#fig3">Figure 3</a>B). Extract the linear plasmid fragment from the gel by DNA purification kit. Elute with 30 μL ddH2O.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The <i>E.coli</i> DH5α strain with pHM1 should be recovered on agar plate. The overnight (12–16) colony is cultured in 1ml LB broth to mid-log phase for plasmid extraction. Old bacteria (OD<sub>600</sub> &gt;1.0) are not recommended for plasmid extraction.</div>
<ol start="3">
<li>Recombine <i>cbrB2</i> DNA fragment with linear pHM1 plasmid in a 10 μL reaction (50–100 ng <i>Hind</i>III-digested pHM1 plasmid fragment, 50 ng <i>cbrB2</i> DNA fragment, 2 μL 5× CE II buffer and 0.5 μL Exnase II). Incubate the mixture at 37°C for 30 min.</li>
<li>Heat-shock transform all 10 μL recombination product into <i>E. coli</i> DH5α competent cell.The transformants are screened by LB agar plates with 100 μg/mL spectinomycin at 37°C for 12–16 h.</li>
<li>Choose two colonies and culture them in 2 mL LB broth with 100 μg/mL spectinomycin to mid-log phase and extract plasmids using the Rapid Mini Plasmid kit.</li>
<li>Screen the correct plasmids.
<ol type="a">
<li>The <i>cbrB2</i> DNA fragment is amplified by PCR from the generated plasmids by corresponding primers (pHM1-<i>cbrB2</i>-F and pHM1-<i>cbrB2</i>-HA-R).The PCR product is checked by 1% agarose gel. The band is specific and at the right size (1529 bp) (<a href="#fig3">Figure 3</a>C).</li>
<li>Plasmid sequencing is used to ensure the plasmid is correct.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend to verify the recombined pHM1 plasmid using PCR verification, because of poor visualization of enzyme digestion verification.</div>
<h4 id="sec3.1.2">Transform pHM1-cbrB2-HA plasmid into <i>Psph</i> wild-type strain</h4>
<ol start="8">
<li>Prepare <i>Psph</i> competent cell.
<ol type="a">
<li>Culture <i>Psph</i> wild-type strain in 2 mL KB (King’B)(<a href="#bib10">King et al., 1954</a>) broth with 25 μg/mL rifampicin for 12–16 h with shaking at 220 rpm.</li>
<li>Gather bacterial cells by centrifugation at 2,400 × <i>g</i> for 5 min and dispose of the supernatant. Resuspend the pellets with 500 μL pre-cooled 10% aseptic glycerin. Centrifuge at 4°C (2,400 × <i>g</i>, 5 min) and dispose of the supernatant. Repeat at least three times.</li>
<li>Resuspend the pellets with 100 μL pre-cooled 10% aseptic glycerin and put it on ice.</li>
</ol>
</li>
<li>Transform 50–100 ng pHM1-<i>cbrB2</i>-HA plasmid into the <i>Psph</i> competent cells prepared above by electroporation transformation (Electroporation Cuvettes: 0.1 cm gap, E=1.8 KV) after 20 min incubation on ice.</li>
<li>Add 1 mL KB to the reaction tube and culture it at 28°C for 2 h with shaking at 220 rpm.</li>
<li>Collect bacterial cells by centrifugation at 3,500 × <i>g</i> for 5 min and plate them onto KB agar plates with 25 μg/mL rifampicin and 100 μg/mL spectinomycin. Incubate the plates at 28°C for 36 h.</li>
<li>Choose single colonies and culture it in 1 mL KB with 25 μg/mL rifampicin and 100 μg/mL spectinomycin for 12–16 h with shaking at 220 rpm.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> After mixing with 50% sterile glycerin by 1:1, the bacteria can be stored at −80°C.</div>
<ol start="13">
<li>Detect the expression of CbrB2-HA protein by Western blot.</li>
</ol>
<p>Collect bacterial cells (200 μL overnight (12–16 h) KB cultures) by centrifugation at 13,800 × <i>g</i> for 1 min. Resuspend the pellets with 20 μL 50% SDS and incubate at 95°C for 15 min. The total bacterial lysates with same amounts of protein (50 μg) are loaded and separated by 10% SDS-PAGE to detect the expression of CbrB2-HA protein in <i>Psph</i> wild-type and <i>cbrB2</i>-overexpressing (OX-<i>cbrB2</i>) strain. As shown in <a href="#fig3">Figure 3</a>D, a specific band of 53.7 KDa in OX-<i>cbrB2</i> was detected, while no band in <i>Psph</i> wild-type. Both strains are used for the subsequent ChIP-seq experiment.</p>
<h3 id="sec3.2">ChIP</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<h4 id="sec3.2.1">Formaldehyde cross-linking</h4>
<ol start="14">
<li>Cross-linking. Culture <i>Psph</i> strains (OX-<i>cbrB2</i> strain and WT/pHM1 strain) in 20 mL KB broth supplemented with rifampicin at 28°C for 12–16 h to OD<sub>600</sub>=0.6 (Two biological replicates are required). Add 500 μL 39% formaldehyde to make the final concentration of formaldehyde 1%. Incubate at 28°C for 10 min with shaking at 220 rpm.</li>
<li>Stop crosslinking. Add 1.5 mL 2 M glycine to the <i>Psph</i> culture. Incubate at 28°C for 5 min with shaking at 220 rpm.</li>
<li>Collect sample. Collect bacteria by centrifugation (6,000 × <i>g</i>, 5 min, 25°C). Wash the pellets with 20 mL Tris buffer. Centrifuge (6,000 × <i>g</i>, 5 min, 25°C) to remove the supernatant. Resuspend the pellets with 1 mL Tris buffer and transfer to a new 1.5 mL EP tubes. Centrifuge (6,000 × <i>g</i>, 5 min, 25°C) to remove the supernatant.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The pellets can be stored at −80°C before subsequent steps.</div>
<h4 id="sec3.2.2">Fragmentation by sonication</h4>
<ol start="17">
<li>Add 500 μL IP buffer to resuspend the pellets. Disrupt samples by sonication (sonicator manufacturer:SCIENTZ, model: JY96-IIN,power: 5%, mode: 5 s on, 5 s off, 3 min in total) in an ice bath.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The sonication conditions need to be optimized according to the amount of bacteria.</div>
<ol start="18">
<li>Centrifuge the lysate solution (13,800 × <i>g</i>, 10 min) at 4°C. Use 1 μL supernatant to check the size of DNA fragments. The bands are between 100–300-bp by agarose gel electrophoresis (<a href="#fig3">Figure 3</a>E).</li>
</ol>
<h4 id="sec3.2.3">Incubate with antibody</h4>
<ol start="19">
<li>Pretreat IgG1-HA-agarose-beads. Calculate the total amount of beads required (20 μL/sample). Centrifuge and remove the supernatant (13,800 × <i>g</i>, 1 min). Wash beads with 1 mL IP buffer at 4°C (13,800 × <i>g</i>, 1 min) for 5 times.</li>
<li>Add 20 μL beads to cell lysates and incubate overnight (or 6 h) with HulaMixer Sample Mixer at 4°C. Centrifuge and remove the supernatant (13,800 × <i>g</i>, 5 min, 4°C).</li>
<li>Wash beads.
<ol type="a">
<li>Wash beads with 1 mL lysis buffer (13,800 × <i>g</i>, 5 min) at 4°C for four times. Discard the supernatant.</li>
<li>Wash beads with 1 mL TE buffer (13,800 × <i>g</i>, 5 min) at 4°C for twice. Discard the supernatant.</li>
</ol>
</li>
<li>Elution.
<ol type="a">
<li>Add 100 μL elution buffer 1 to beads. Mix well and incubate at 65°C for 10 min. Centrifuge (13,800 × <i>g</i>, 1 min) at 25°C and transfer the supernatant to a new 1.5 mL tube.</li>
<li>Add 150 μL elution buffer 2 to beads and mix well. Centrifuge (13,800 × <i>g</i>, 1 min) at 25°C and collect supernatant to the tube in step 22a. (Total supernatant is 250 μL, named IP sample)</li>
</ol>
</li>
</ol>
<h4 id="sec3.2.4">Cross-linking reversal</h4>
<ol start="23">
<li>Incubate IP samples at 65°C for 6 h (or overnight).</li>
</ol>
<h4 id="sec3.2.5">Collect DNA</h4>
<ol start="24">
<li>Prepare proteinase K solution (0.5 μL 20 mg/mL glycogen, 5 μL 20 mg/mL proteinase K and 244.5 μL TE).</li>
<li>Add 250 μL proteinase K solution to IP sample and incubate the mixture at 58°C for 2 h.</li>
<li>Add 55 μL 4M LiCl and mix well at 25°C.</li>
<li>Add 500 μL phenol-chloroform and mix well.</li>
<li>Centrifuge (13,800 × <i>g</i>, 3 min) at 25°C and collect supernatant to a new 1.5 mL tube.</li>
<li>Add 1 mL pre-cooled absolute ethanol and incubate for 30 min at −80°C. Centrifuge (13,800 × <i>g</i>, 30 min) at 4°C and discard the supernatant. Wash the precipitation with 1 mL 70% ethanol (13,800 × <i>g</i>, 5 min) at 4°C. Air-dry the precipitation for 5–10 min and dissolve with 25μL nuclease-free water.</li>
<li>Measure the DNA concentration with Qubit (model: dsDNA high sensitivity, dye: Qubit dsDNA HS Assay Kit).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The DNA samples can be stored at −20°C.</div>
<h3 id="sec3.3">ChIP-seq library construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 days</div>
<h4 id="sec3.3.1">End repair</h4>
<ol start="31">
<li>Mix PCR reaction according to the following recipe. (5 μL T4 DNA ligase buffer with 10 mM ATP, 2 μL dNTP mix, 1 μL T4 DNA polymerase, 1 μL Klenow DNA polymerase (1 U/μL), 1 μL T4 polynucleotide kinase, 30 μL ChIP enriched DNA (10–30 ng, add H<sub>2</sub>O to 30 μL), 10 μL nuclease-free H<sub>2</sub>O).</li>
<li>Incubate at 20°C for 30 min.</li>
<li>Use QIAquick PCR purification kit to clean the PCR reaction. Elute with a 34 μL elution buffer.</li>
</ol>
<h4 id="sec3.3.2">Add “A” bases to the 3′ end of the DNA fragments</h4>
<ol start="34">
<li>Mix the PCR reaction according to the following recipe. (5 μL Klenow buffer, 10 μL dATP, 1 μL Klenow exo (3′ to 5; exo minus), 34 μL DNA sample form Step 33).</li>
<li>Incubate at 37°C for 30 min.</li>
<li>Use Zymo Oligo Clean &amp; Concentrator to concentrate the volume to 10 μL.</li>
</ol>
<h4 id="sec3.3.3">Ligate adapters to DNA fragment</h4>
<ol start="37">
<li>Mix reaction according to the following recipe. (15 μL Quick ligase buffer, 1 μL Diluted oligos for Illumina (10–30 ng, dilute with adapter dilution buffer), 4 μL Quick ligase, 4 μL DNA sample)</li>
<li>Incubate at 25°C for 15 min.</li>
<li>Add 1 μL User enzyme and incubate at 37°C for 15 min.</li>
<li>Use Zymo Oligo Clean &amp; Concentrator to concentrate the volume to 15 μL.</li>
</ol>
<h4 id="sec3.3.4">Size selection</h4>
<ol start="41">
<li>Mix DNA sample with 6× loading and run a 2% low melting point agarose gel electrophoresis (90 V, 25 min) to 100–200-bp separated.</li>
<li>Cut the gel in 150–250-bp size and extract the DNA by QIAquick Gel Extraction kit. Elute with 72 μL EB. DNA bands cannot be seen at this step (<a href="#fig3">Figure 3</a>F).</li>
</ol>
<h4 id="sec3.3.5">PCR amplification</h4>
<ol start="43">
<li>DNA fragments in step 42 are amplified by PCR.
<ol type="a">
<li>Mix PCR reaction according to the following recipe. (25 μL NEB Phusion high-fidelity PCR master mix, 15 μL cDNA, 3.1 μL Universal Primer (0.125 uM), 3.1 μL Index Primer (0.125 uM), 3.8 μL H<sub>2</sub>O)</li>
<li>The following thermocycle program is recommended for PCR amplification.</li>
</ol>
</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="2">17 cycles</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>forever</td>
<td> </td>
</tr>
</tbody>
</table>
<ol start="44">
<li>Mix DNA sample with 6× loading and run a 2% low melting point agarose gel electrophoresis (90 V, 25 min). The band is between 200−300-bp (<a href="#fig3">Figure 3</a>G).</li>
<li>Cut the band between 200−300-bp from the gel and extract the DNA by QIAquick Gel Extraction kit. Elute with 30 μL EB. Measure the DNA concentration with Qubit.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The DNA samples can be stored at −20°C. The samples will be sent for sequencing by Illumina HiSeq 2000 system.</div>
<h3 id="sec3.4">
<i>Psph</i> deletion mutant construction</h3>
<p>The flow diagram of mutant construction is showed in <a href="#fig4">Figure 4</a>.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/964-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The flow diagram of <i>P. syringae</i> deletion mutant construction</div>
</figcaption>
</figure>
<h4 id="sec3.4.1">pK18mobsacB-TF-upstream-downstream suicide plasmid construction</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="46">
<li>Amplify the <i>cbrB2</i> upstream and downstream fragment form <i>Psph</i> chromosomal DNA by PCR.
<ol type="a">
<li>Set up a PCR reaction in a 200 μL PCR tube according to the following recipe. (18 μL Taq premix, 0.5 μL <i>cbrB2</i>-Upstream/Downstream fragment-F primer, 0.5 μL <i>cbrB2</i>-Upstream/Downstream fragment-R primer, 20 ng <i>Psph</i> chromosomal DNA)</li>
<li>The following thermocycler program is recommended for PCR amplification.
<table id="undtbl10">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min/kb</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>The PCR product is checked by 1% agarose gel. The upstream and downstream fragments are specific and at the size of 1491 bp and 965 bp, respectively (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/964-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. The experimental results of <i>Psph cbrB2</i> mutant construction</div>
<p>(A) PCR amplification of the upstream (1491 bp) and downstream (965 bp) fragments of <i>cbrB2</i> open reading frame. The bands are at the right size.</p>
<p>(B) pk18mobsacB plasmid is digested by <i>Eco</i>RI and <i>Hind</i>III.</p>
<p>(C) PCR verification of pk18mobsacB-<i>cbrB2</i>-up-down plasmid by <i>cbrB2</i>-verify-F/R primers. All three plasmids are at the right size (411 bp).</p>
<p>(D) Verification of pk18mobsacB-<i>cbrB2</i>-up-down plasmid by <i>Xba</i>I, <i>Eco</i>RI and <i>Hind</i>III. All six plasmids are digested to linear fragments of right size.</p>
<p>(E) PCR amplification of chromosomal DNA of Δ<i>cbrB2</i> and WT strain using <i>cbrB2</i>-verify-F/R primers. All six mutants are at the right size (411 bp).</p>
<p>(F) qRT-PCR detection of the expression of <i>cbrB2</i> in three Δ<i>cbrB2</i> mutants and the WT strain.</p>
</figcaption>
</figure>
</li>
<li>Mix upstream and downstream fragments together. Purify upstream and downstream fragment mixture by DNA purification kit. Elute with 30 μL ddH<sub>2</sub>O.</li>
</ol>
</li>
</ol>
<ol start="47">
<li>Digest upstream and downstream fragment mixture in step 46d with the <i>Xba</i>I in 30 μL reaction (27 μL purified DNA, 2.6 μL 10×cutsmart buffer and 0.4 μL <i>Xba</i>I enzyme). Incubate the reaction for at least 2 h at 37°C.</li>
<li>Clean the digestion reaction using DNA purification kit. Elute with 30 μL ddH<sub>2</sub>O.</li>
<li>Ligate the purified DNA fragment with T4 ligase (2.5 μL DNA, 0.5 μL 10× T4 ligase buffer, 0.25 μL T4 ligase and 1.75 μL ddH2O). Incubate 3 h at 16°C.</li>
<li>Incubate 2 μg pK18mobsacB plasmid with <i>Eco</i>RI and <i>Hind</i>III in a 50 μL reaction at 37°C for 1 h. The digested pK18mobsacB plasmid is separated by 0.8% agarose gel. The band of linear plasmid is above the circular plasmid (<a href="#fig5">Figure 5</a>B). Extract the linear plasmid fragment from the gel by DNA purification kit. Elute with 30 μL ddH<sub>2</sub>O.</li>
<li>Recombine the ligated DNA with digested pK18mobsacB plasmid in a 10 μL reaction (100 ng lined pK18mobsacB plasmid, 3 μL ligated DNA, 2 μL 5× CE II buffer, 0.5 μL Exnase II). Incubate the reaction for 30 min at 37°C.</li>
<li>Transform the recombination products into <i>E. coli</i> DH5α competent cell by heat-shock transformation.</li>
<li>Transformants are screened by LB agar plates with 100 μg/mL kanamycin incubating the plates at 37°C for 12–16 h.</li>
<li>Choose two colonies and culture them in 2 mL LB broth with 100 μg/mL kanamycin culture at 37°C for 12–16 h shaking with 220 rpm.</li>
<li>Extract the plasmid DNA from the cultures by using the Rapid Mini Plasmid kit.</li>
<li>Screen the correct plasmids.
<ol type="a">
<li>PCR verification. The DNA sequence is amplified by PCR from the generated plasmids by corresponding primers (<i>cbrB2</i>-verify-F/R). The PCR product is checked by 1% agarose gel. The band is specific and at the correct size (411 bp) (<a href="#fig5">Figure5C</a>).</li>
<li>Enzyme digesting verification. Digest 2 μL of plasmid DNA by <i>Xba</i>I or <i>Xba</i>I, <i>Eco</i>RI, and <i>Hind</i>III. Incubate for at least 1 h at 37°C. The circular plasmid is digested to one linear fragment by <i>Xba</i>I and four linear fragments by <i>Xba</i>I, <i>Eco</i>RI and <i>Hind</i>III (<a href="#fig5">Figure 5</a>D).</li>
<li>Plasmid sequencing is used to ensure the plasmid is correct.</li>
</ol>
</li>
</ol>
<h4 id="sec3.4.2">
<i>Psph</i> deletion mutant construction</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol start="57">
<li>Prepare <i>Psph</i> competent cell as above mentioned in step 8 TF-overexpressing strain construction.</li>
<li>Transform 50–100 ng pK18mobsacB-up-down plasmid into <i>Psph</i>-W<i>T</i> competent cell prepared above by electroporation transformation (Electroporation Cuvettes: 0.1 cm gap, E=1.8 KV) after 20 min incubation.</li>
<li>Add 1 mL KB with 25 μg/mL rifampicin to the reaction tube and culture it at 37°C for 2 h shaking with 220 rpm.</li>
<li>Collect the bacterial cells by centrifugation at 3,500 × <i>g</i> for 5 min and plate them onto KB agar plates with 25 μg/mL rifampicin and 100 μg/mL kanamycin. Incubate the plates at 28°C for 36 h.</li>
<li>Choose single colony and culture it in 1 mL KB broth with 25 μg/mL rifampicin and 100 μg/mL kanamycin for 12–16 h shaking with 220 rpm.</li>
<li>Streak plate the mutants with KB 5% sucrose agar plates with 25 μg/mL rifampicin. Incubate the plates at 28°C for 36 h.</li>
<li>Spot several colonies on KB+25 μg/mL rifampicin and KB+25 μg/mL rifampicin +100 μg/mL kanamycin agar plates. Choose colonies which grow on KB+25 μg/mL rifampicin plates but not on KB+25 μg/mL rifampicin +100 μg/mL kanamycin plates to do subsequent PCR screen.</li>
<li>Screen the correct <i>cbrB2</i> mutants by PCR.
<ol type="a">
<li>The DNA fragments are amplified by PCR form the chromosomal DNA of <i>cbrB2</i> mutants and WT by corresponding primers (<i>cbrB2</i>-verify-F/R), respectively. The PCR product is checked by 1% agarose gel. As shown in <a href="#fig5">Figure 5</a>E, the PCR product of <i>cbrB2</i> mutant is 411 bp while the PCR product of WT is 1712 bp.</li>
<li>Use qRT-PCR to check the loss of <i>cbrB2</i> in the <i>cbrB2</i> mutant strain by corresponding primers (<i>cbrB2</i>-RT-F/R). As shown in <a href="#fig5">Figure 5</a>F, the expression of <i>cbrB2</i> is not detected in three deletion mutants of <i>cbrB2.</i>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>We have overexpressed HA-tagged <i>cbrB2</i> in <i>P. syringae</i>. Western blot shows CbrB2-HA expresses in the OX-<i>cbrB2</i> strain (<a href="#fig3">Figure 3</a>D). Then, we have done ChIP-seq of OX-<i>cbrB2</i> strain and <i>Psph</i>-WT/pHM1 strain. First, the DNA is between 100–300 bp size after sonication (<a href="#fig3">Figure 3</a>E). The DNA band is invisible in the DNA size selection (step 41) (<a href="#fig3">Figure 3</a>F), and DNA in 150–250 bp size is extracted. The final result indicates that ChIP-seq library DNA is between 200–300 bp for sequencing (<a href="#fig3">Figure 3</a>G). The final outcome DNA is 10–20 ng when the input DNA is 10–30 ng in step 31.</p>
<p>We have constructed <i>Psph cbrB2</i> deletion mutant following this protocol. First, the upstream and downstream fragment of <i>cbrB2</i> ORF are amplified (<a href="#fig5">Figure 5</a>B). After homologous recombination and heat-shock transformation, pK18mobsacB-<i>cbrB2</i>-upstream-downstream plasmid is constructed and the upstream and downstream connected part can be amplified using <i>cbrB2</i> verification primers (<a href="#fig5">Figure 5</a>C). At the same time, the <i>Xba</i>I, <i>Eco</i>RI and <i>Hind</i>III sites on pK18mobsacB-<i>cbrB2</i>-upstream-downstream plasmid can be digested (<a href="#fig5">Figure 5</a>D), suggesting that the ligated upstream and downstream fragments of <i>cbrB2</i> are successfully connected to the lined plasmids. In step 64. We have amplified six colonies and one <i>Psph</i>-WT strain using <i>cbrB2</i> verification primers, and all the bands of colonies show the correct size (accuracy: 100%) (<a href="#fig5">Figure 5</a>E). Then three of <i>cbrB2</i> mutants have been chosen to do qRT-PCR. As the result, the expression of <i>cbrB2</i> is not detected in the correct <i>cbrB2</i> mutant strain compared with the <i>Psph</i>-WT strain (accuracy: 100%) (<a href="#fig5">Figure 5</a>F).</p>
<p>In addition, compared with the previous method (<a href="#bib14">McDowell, 2011</a>), the present protocol improves the screen accuracy and simplify the experimental steps in the following ways: 1) We use the ligated DNA in step 49 directly and omit the gel purification of the ligated band (upstream-downstream fragment) in this protocol. 2) We use homologous recombination strategy to insert the ligated DNA into pK18mobsacB plasmid instead of restriction sites ligation to improve efficiency (step 51). 3) We construct clean mutants without antibiotic marker cassette. 4) Electroporation transformation is used to transform pK18mobsacB-TF-upstream-downstream plasmid to <i>Psph</i> instead of transconjugation. 5) Previous protocol takes 8 days to construct mutant by transconjugation, while 2 days are needed in electroporation transformation in our protocol. Besides, LM medium and <i>E.coli</i> S17-1 must be prepared. 6) Streak plate method should be used to isolate transconjugants instead of direct coating method.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is mainly for ChIP-seq and mutant construction of <i>P. syringae.</i> The experimental conditions and materials may need to be optimized for other bacteria, including medium, plasmid, antibodies, culture conditions and sonication conditions.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The DNA band is not at the correct size after sonication (step 18).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Optimize sonication conditions (power, number of cycle and working mode) according to different bacteria and bacterial densities.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The weight of DNA in the IP sample (&gt;10 ng) is not enough to do further library construction or below the detection limitation of Qubit Kit (step 30).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Increase the bacterial culture in step 14 or the amount of antibody in step 19. Besides, optimize sonication conditions to make the DNA band in 100–300 bp (step 17) can alleviate this problem.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Unexpected enrichment is observed in sequenced libraries (step 44).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Reduce the bacterial culture and the amount of antibody (step 14 and step 19). Reduce cross-linking time because over cross-linking can cause non-specific binding. Besides, please be care to cut the gel in 150–250-bp size (step 42).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The library complexity is low after sequencing.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Increase the amount of antibody in step 19 or use the fewest possible PCR amplification cycles during ChIP-seq library construction (step 43) to get a final DNA yield appropriate for sequencing.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The PCR bands are not single (step 1 and step 46).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Optimize PCR cycling program (increase annealing temperature) or gel purify the correct band.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Cunnac, S., Lindeberg, M., and Collmer, A. (2009). <i>Pseudomonas syringae</i> type III secretion system effectors: repertoires in search of functions. Curr. Opin. Microbiol. <i>12</i>, 53-60. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref1">View at publisher</a></p>
<p id="bib2">Deng, X., Lan, L., Xiao, Y., Kennelly, M., Zhou, J.M., and Tang, X. (2010). <i>Pseudomonas syringae</i> two-component response regulator RhpR regulates promoters carrying an inverted repeat element. Mol. Plant Microbe Interact. <i>23</i>, 927-939. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref2">View at publisher</a></p>
<p id="bib3">Deng, X., Liang, H., Chen, K., He, C., Lan, L., and Tang, X. (2014). Molecular mechanisms of two-component system RhpRS regulating type III secretion system in <i>Pseudomonas syringae</i>. Nucleic Acids Res. <i>42</i>, 11472-11486. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref3">View at publisher</a></p>
<p id="bib4">Deng, X., Xiao, Y., Lan, L., Zhou, J.M., and Tang, X. (2009). <i>Pseudomonas syringae</i> pv. <i>phaseolicola</i> mutants compromised for type III secretion system gene induction. Mol. Plant Microbe Interact. <i>22</i>, 964-976. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref4">View at publisher</a></p>
<p id="bib5">Galan, J.E. and Collmer, A. (1999). Type III secretion machines: bacterial devices for protein delivery into host cells. Science <i>284</i>, 1322-1328. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref5">View at publisher</a></p>
<p id="bib6">Gonzalez, A.J., Landeras, E., and Mendoza, M.C. (2000). Pathovars of <i>Pseudomonas syringae</i> causing bacterial brown spot and halo blight in phaseolus vulgaris L. are distinguishable by ribotyping. Appl. Environ. Microbiol. <i>66</i>, 850-854. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref6">View at publisher</a></p>
<p id="bib7">Hirano, S.S. and Upper, C.D. (2000). Bacteria in the leaf ecosystem with emphasis on <i>Pseudomonas syringae</i> pathogen, ice nucleus, and epiphyte. Microbiol. Mol. Biol. Rev. <i>64</i>, 624-653. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref7">View at publisher</a></p>
<p id="bib8">Hua, C.F., Wang, T.T., Shao, X.L., Xie, Y.P., Huang, H., Liu, J.G., Zhang, W.T., Zhang, Y.C., Ding, Y.Q., Jiang, L., et al. (2020). <i>Pseudomonas syringae</i> dual-function protein Lon switches between virulence and metabolism by acting as both DNA-binding transcriptional regulator and protease in different environments. Environ. Microbiol. <i>22</i>, 2968-2988. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref8">View at publisher</a></p>
<p id="bib9">Karin, M. (1990). Too many transcription factors: positive and negative interactions. New Biol. <i>2</i>, 126-131. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref9">View at publisher</a></p>
<p id="bib10">King, E.O., Ward, M.K., and Raney, D.E. (1954). Two simple media for the demonstration of pyocyanin and fluorescin. J. Lab. Clin. Med. <i>44</i>, 301-307. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref10">View at publisher</a></p>
<p id="bib11">Kvitko, B.H. and Collmer, A. (2011). Construction of <i>Pseudomonas syringae</i> pv. <i>tomato</i> DC3000 Mutant and Polymutant Strains. Methods Mol. Biol. <i>712</i>, 109-128. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref11">View at publisher</a></p>
<p id="bib12">Labes, M., Puhler, A., and Simon, R. (1990). A new family of Rsf1010-derived expression and lac-fusion broad-host-range vectors for gram-negative bacteria. Gene <i>89</i>, 37-46. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref12">View at publisher</a></p>
<p id="bib13">Latchman, D.S. (1997). Transcription factors: an overview. Int. J. Biochem. Cell Biol. <i>29</i>, 1305-1312. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref13">View at publisher</a></p>
<p id="bib14">McDowell, J.M. (2011). Plant Immunity : Methods and Protocols (Humana Press). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref14">View at publisher</a></p>
<p id="bib15">Shao, X.L., Tan, M.M., Xie, Y.P., Yao, C.Y., Wang, T.T., Huang, H., Zhang, Y.C., Ding, Y.Q., Liu, J.G., Han, L.L., et al. (2021). Integrated regulatory network in <i>Pseudomonas syringae</i> reveals dynamics of virulence. Cell Rep. <i>34</i>, 108920. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref15">View at publisher</a></p>
<p id="bib16">Wang, J.R., Shao, X.L., Zhang, Y.C., Zhu, Y.N., Yang, P., Yuan, J., Wang, T.T., Yin, C.Y., Wang, W., Chen, S., et al. (2018). HrpS is a global regulator on type III secretion system (T3SS) and non-T3SS genes in <i>Pseudomonas savastanoi</i> pv. <i>phaseolicola</i>. Mol. Plant Microbe Interact. <i>31</i>, 1232-1243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref16">View at publisher</a></p>
<p id="bib17">Xiao, Y., Lan, L., Yin, C., Deng, X., Baker, D., Zhou, J.M., and Tang, X. (2007). Two-component sensor RhpS promotes induction of <i>Pseudomonas syringae</i> type III secretion system by repressing negative regulator RhpR. Mol. Plant Microbe Interact. <i>20</i>, 223-234. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref17">View at publisher</a></p>
<p id="bib18">Xie, Y.P., Shao, X.L., Zhang, Y.C., Liu, J.G., Wang, T.T., Zhang, W.T., Hua, C.F., and Deng, X. (2019). <i>Pseudomonas savastanoi</i> two-component system RhpRS switches between virulence and metabolism by tuning phosphorylation state and sensing nutritional conditions. Mbio <i>10</i>, e02838-18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref18">View at publisher</a></p>
<p id="bib19">Zhou, T.H., Yin, C.Y., Zhang, Y.C., Shi, H., Wang, J.R., Sun, L.B., Shao, X.L., Gao, R.X., Wang, W., and Deng, X. (2016). Lon protease is involved in RhpRS-mediated regulation of type III secretion in <i>Pseudomonas syringae</i>. Mol. Plant Microbe Interact. <i>29</i>, 807-814. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00482-2/sref19">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Xin Deng (<a href="mailto:xindeng@cityu.edu.hk">xindeng@cityu.edu.hk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>This study was supported by grants from the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (31670127 and 31870116 to X.D.), <a href="https://doi.org/10.13039/501100012479">General Research Funds of Hong Kong</a> (21103018, 11101619, and 11102720 to X.D.). The funders had no role in study design, data collection, and interpretation or the decision to submit the work for public.</p>
<h3>Author contributions</h3>
<p>X.D. and X.S. supervised the study. C.Y., X.S., and J.L. carried out the experiments. C.Y., X.S., and X.D. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>