<article>
<section>
<h1 id="header">Purification of DNA repair protein complexes from mammalian cells</h1>
<p><time datetime="2021-02-19">Published: February 19, 2021</time></p>
<p>Chih-Chao Liang<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a></sup> and Martin A. Cohn<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biochemistry, Oxford OX1 3QU, UK</p>
<p id="fn1"><sup>2</sup>Current address: The Francis Crick Institute, London NW1 1AT, UK</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:martin.cohn@bioch.ox.ac.uk">martin.cohn@bioch.ox.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100348">10.1016/j.xpro.2021.100348</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cells possess multiple DNA repair pathways to tackle a variety of DNA lesions. Often, DNA repair proteins function as large protein complexes. Here, we describe a protocol to purify DNA repair protein complexes from nuclei of mammalian cells. The method permits purification of protein complexes containing stable as well as transiently associated proteins, which subsequently can be identified by mass-spectrometry analysis. This protocol can be applied to uncover the functions and mechanism of DNA repair pathways.</p>
<p>For complete information on the use and execution of this protocol, please refer to <a href="#bib11">Socha et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to purify DNA repair protein complexes from a mammalian cell line</li>
<li>Permits purification of both stable and transiently associated protein subunits</li>
<li>Protein complexes bound to DNA can also be purified</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/481-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<ol>
<li>Prepare plenty of buffers in advance. All buffers are filtered with 0.22 μm filters. 0.2 mM PMSF (protease inhibitor) and 2 mM 2-mercaptoethanol (reducing agent) are added freshly to the following buffers immediately before use: Hypotonic buffer, Low-salt nuclear extraction buffer (low-salt NE buffer), High-salt nuclear extraction buffer (high-salt NE buffer) and Buffer A.</li>
<li>All purifications are carried out at 4°C.</li>
<li>HeLa S3 cells are used in this protocol as they can be grown both as semi-adherent monolayer in dishes or as suspension culture in spinner flasks, enabling large scale culture. Other cell types can be used but will require additional optimization.</li>
<li>Methanol-free formaldehyde used is less than 3 months old, preferably less than 1 month.</li>
<li>FLAG and HA epitope tags are used for purification as effective antibodies, as well as peptides for elution, are available against both tags for both purification and for detection by immunoblotting analysis. Other tags can be used but will also require additional optimization.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Phoenix-A</td>
<td>ATCC</td>
<td>ATCC CRL- 3213</td>
</tr>
<tr>
<td>HeLa S3</td>
<td>ATCC</td>
<td>ATCC CCL-2.2</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>IL-2Rα (7G7/B6)</td>
<td>Merck</td>
<td>05-170</td>
</tr>
<tr>
<td>
<i>α</i>-Tubulin (DM1A) (1:1,000)</td>
<td>Merck</td>
<td>05-829</td>
</tr>
<tr>
<td>HA (12CA5) (1:1,000)</td>
<td>Sigma-Aldrich</td>
<td>SAB1305536</td>
</tr>
<tr>
<td>FANCD2 (FI17) (1:100)</td>
<td>Santa Cruz</td>
<td>sc-20022</td>
</tr>
<tr>
<td>Anti-mouse (horseradish peroxidase conjugated) (1:8,000)</td>
<td>GE Healthcare</td>
<td>NA9310V</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pGag+Pol</td>
<td>(<a href="#bib7">Morita et al., 2000</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pEnv</td>
<td>(<a href="#bib7">Morita et al., 2000</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pOZ-N</td>
<td>(<a href="#bib8">Nakatani and Ogryzko, 2003</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dulbecco’s modified Eagle’s medium, DMEM</td>
<td>Sigma-Aldrich</td>
<td>D5796</td>
</tr>
<tr>
<td>Opti-MEM I reduced serum medium, no phenol red</td>
<td>Thermo Fisher</td>
<td>11058021</td>
</tr>
<tr>
<td>Minimum essential medium Joklik’s modified</td>
<td>Sigma-Aldrich</td>
<td>M8028</td>
</tr>
<tr>
<td>Fetal bovine serum, FBS</td>
<td>Sigma-Aldrich</td>
<td>F7524</td>
</tr>
<tr>
<td>L-Glutamine solution</td>
<td>Sigma-Aldrich</td>
<td>G7513</td>
</tr>
<tr>
<td>MEM non-essential amino acid solution (100×)</td>
<td>Sigma-Aldrich</td>
<td>M7145</td>
</tr>
<tr>
<td>Penicillin-streptomycin (100×)</td>
<td>Sigma-Aldrich</td>
<td>P4333</td>
</tr>
<tr>
<td>Hexadimethrine bromide, Polybrene</td>
<td>Sigma-Aldrich</td>
<td>H9268</td>
</tr>
<tr>
<td>Fugene 6</td>
<td>Promega</td>
<td>E2691</td>
</tr>
<tr>
<td>Mitomycin C</td>
<td>Sigma-Aldrich</td>
<td>M0440</td>
</tr>
<tr>
<td>Benzonase nuclease, 99% purity</td>
<td>Merck</td>
<td>71,206</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Sigma-Aldrich</td>
<td>M6250</td>
</tr>
<tr>
<td>16% formaldehyde</td>
<td>Thermo Fisher</td>
<td>28906</td>
</tr>
<tr>
<td>Phenylmethanesulfonyl fluoride, PMSF</td>
<td>Sigma-Aldrich</td>
<td>P7626</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma-Aldrich</td>
<td>54457</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>G2025</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma-Aldrich</td>
<td>P7949</td>
</tr>
<tr>
<td>Sodium dodecyl sulfate, SDS</td>
<td>Sigma-Aldrich</td>
<td>71725</td>
</tr>
<tr>
<td>Ethylenediaminetetraacetic acid, EDTA</td>
<td>Sigma-Aldrich</td>
<td>EDS</td>
</tr>
<tr>
<td>Glycine</td>
<td>Sigma-Aldrich</td>
<td>50046</td>
</tr>
<tr>
<td>FLAG peptide</td>
<td>Sigma-Aldrich</td>
<td>F3290</td>
</tr>
<tr>
<td>HA peptide</td>
<td>Sigma-Aldrich</td>
<td>I2149</td>
</tr>
<tr>
<td>DL-Dithiothreitol, DTT</td>
<td>Sigma-Aldrich</td>
<td>43815</td>
</tr>
<tr>
<td>Iodoacetamide</td>
<td>Sigma-Aldrich</td>
<td>I1149</td>
</tr>
<tr>
<td>Trichloroacetic acid, TCA</td>
<td>Sigma-Aldrich</td>
<td>T6399</td>
</tr>
<tr>
<td>Dynabeads goat anti-mouse IgG magnetic beads</td>
<td>Invitrogen</td>
<td>11033</td>
</tr>
<tr>
<td>Anti-FLAG M2 agarose resin</td>
<td>Sigma-Aldrich</td>
<td>A2220</td>
</tr>
<tr>
<td>Protein A Sepharose Cl-4B</td>
<td>Thermo Fisher</td>
<td>GE17-0963-02</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Western Lightning Plus-ECL, enhanced chemiluminescence substrate</td>
<td>PerkinElmer</td>
<td>NEL105001EA</td>
</tr>
<tr>
<td>SilverQuest silver staining kit</td>
<td>Invitrogen</td>
<td>LC6070</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Steritop threaded bottle top filter (PES; 0.22 μm pore size)</td>
<td>Millipore</td>
<td>SCGPT02RE</td>
</tr>
<tr>
<td>Glass Dounce homogenizer, 15 mL</td>
<td>Bellco Glass</td>
<td>1984-10015</td>
</tr>
<tr>
<td>Econo-Pac chromatography columns</td>
<td>Bio-Rad</td>
<td>7321010</td>
</tr>
<tr>
<td>Poly-Prep chromatography columns</td>
<td>Bio-Rad</td>
<td>7311550</td>
</tr>
<tr>
<td>MagRack 6 magnetic stand</td>
<td>GE</td>
<td>28948964</td>
</tr>
<tr>
<td>Refrigerated centrifuges</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Sonicator with 3 mm probe</td>
<td>Sonics &amp; Materials</td>
<td>VC 130</td>
</tr>
<tr>
<td>Rotating wheel</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Tube roller</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Vertical electrophoresis systems</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Electroblotting systems</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Horizontal electrophoresis systems</td>
<td>N/A</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Buffers and other solutions</h3>
<table id="undtbl1">
<caption>
<b>Phosphate-buffered saline (PBS)</b> 1,000 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>137 mM</td>
<td>8 g</td>
</tr>
<tr>
<td>KCl</td>
<td>2.7 mM</td>
<td>0.2 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>10 mM</td>
<td>1.44 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>1.8 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>·2H<sub>2</sub>O</td>
<td>1 mM</td>
<td>0.133 g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>·6H<sub>2</sub>O</td>
<td>0.5 mM</td>
<td>0.1 g</td>
</tr>
<tr>
<td colspan="3">Adjust pH to 7.4 with HCl and add ddH<sub>2</sub>O to 1,000 mL. Store at 20–25°C for up to 1 year</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>
<b>Hypotonic buffer</b> 1,000 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M HEPES-KOH pH 7.5</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>4 M KCl</td>
<td>10 mM</td>
<td>2.5 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>1.5 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td colspan="3">Add ddH<sub>2</sub>O to 1,000 mL. Store at 4°C for up to 1 year</td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>
<b>Low-salt nuclear extraction buffer (low-salt NE buffer)</b> 1,000 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M HEPES-KOH pH 7.5</td>
<td>20 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>4 M KCl</td>
<td>50 mM</td>
<td>12.5 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>1.5 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>25%</td>
<td>250 mL</td>
</tr>
<tr>
<td colspan="3">Add ddH<sub>2</sub>O to 1,000 mL. Store at 4°C for up to 1 year</td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>
<b>High-salt nuclear extraction buffer (high-salt NE buffer)</b> 500 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M HEPES-KOH pH 7.5</td>
<td>20 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>1 M</td>
<td>100 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>1.5 mM</td>
<td>0.75 mL</td>
</tr>
<tr>
<td>25% Tween 20</td>
<td>1%</td>
<td>20 mL</td>
</tr>
<tr>
<td>10% SDS</td>
<td>0.15%</td>
<td>7.5 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>25%</td>
<td>125 mL</td>
</tr>
<tr>
<td colspan="3">Add ddH<sub>2</sub>O to 1,000 mL. Store at 4°C for up to 1 year</td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>
<b>Buffer A</b> 1,000 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 8.0</td>
<td>20 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>100 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>5 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>25% Tween 20</td>
<td>0.1%</td>
<td>4 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td colspan="3">Add ddH<sub>2</sub>O to 1,000 mL. Store at 4°C for up to 1 year</td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<thead>
<tr>
<th>Other solutions</th>
<th>Final concentration</th>
<th>Volume</th>
<th>Note</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hexadimethrine bromide, Polybrene (100×)</td>
<td>0.4 mg/mL in ddH<sub>2</sub>O</td>
<td>10 mL</td>
<td>Store at −20°C for up to 1 year</td>
</tr>
<tr>
<td>PMSF (100×)</td>
<td>200 mM phenylmethanesulfonyl fluoride (PMSF) in ethanol</td>
<td>20 mL</td>
<td>Heat to 37°C to dissolve, aliquot to 2 mL per tube and store at −20°C for up to 1 year</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Generation of a HeLa S3 cell line stably expressing FLAG and HA-tagged FANCD2</h3>
<p>This protocol is a modified version of a previously described method developed in the laboratory of Yoshihiro Nakatani (<a href="#bib8">Nakatani and Ogryzko, 2003</a>). As a proof-of-principle, we use human FANCD2 as a target to study its interacting proteins. The human <i>FANCD2</i> cDNA was cloned into the pOZ-N retroviral expression vector, which contains a Flag and HA tag at the N-terminus. The pOZ vector contains long terminal repeats to control the integration and expression of target proteins and also a bi-cistronic transcriptional unit for co-expression of interleukin-2 receptor α chain (IL-2Rα) as a selection marker. We normally select for cells or cell clones expressing levels of exogenous proteins comparable with those of endogenous the proteins, to decrease the likelihood of protein-protein interaction being driven by overexpression of the target protein of interest. It should be noted, that while transcription of many genes are cell cycle dependent or regulated in response to cell state or external stimuli, transcription via the described vehicle is more constitutive. Therefore, alternative methods for generating stable or inducible expression can be used. We have successfully used CRISPR/Cas9 to introduce the FLAG-HA tag into the <i>FANCD2</i> gene in its chromosomal location, an approach, which might be suitable for other genes (<a href="#bib4">Lopez-Martinez et al., 2019</a>).</p>
<p>A schematic of this protocol is shown in <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/481-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic of protocol</div>
</figcaption>
</figure>
<h3 id="sec3.2">Preparation of retrovirus with packaging cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>We use Phoenix-A cells (Phoenix-AMPHO or Phoenix-Amphotropic), a derivative of human 293T cells, for retrovirus packaging, yet supplemented with further Gag, Pol and Env viral proteins via transfection. Phoenix-A cells are grown in DMEM supplemented with 10% FBS in a 37°C incubator with 5% CO<sub>2</sub>. We normally keep the cells in culture for less than 2 months.</p>
<ol>
<li>For one transfection, seed 120,000 cells in one well of a 6-well plate or a 6-cm dish (roughly 10% confluency).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to plate an appropriate amount of cells. Too few cells will yield low titer virus stock, and too many cells will deplete the nutrition in the medium, which negatively affects the viral transduction to follow.</div>
<ol start="2">
<li>Prepare the transfection mixture in two separate microcentrifuge tubes. Tube No 1: Target plasmids and two helper plasmids</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>100 μL</th>
<th>DMEM medium (without serum and antibiotics) or opti-MEM</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 μg</td>
<td>pOZ-N-FANCD2 plasmids or desired plasmids</td>
</tr>
<tr>
<td>0.5 μg</td>
<td>pGag+Pol</td>
</tr>
<tr>
<td>0.5 μg</td>
<td>pEnv</td>
</tr>
</tbody>
</table>
<table id="undtbl8">
<caption>Tube No 2: Transfection agent</caption>
<thead>
<tr>
<th>100 μL</th>
<th>DMEM medium (without serum and antibiotics) or opti-MEM</th>
</tr>
</thead>
<tbody>
<tr>
<td>6 μL</td>
<td>Fugene 6</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>Pipette the DNA mixture in Tube No 1 into Tube No 2, invert 5 times, and incubate at 20–25°C for 15–30 min.</li>
<li>Pipette the whole transfection mixture drop-wise into the well with Phoenix-A cells.</li>
<li>Gently rock the plate to help the transfection mixture disperse into the medium.</li>
<li>Let the cells grow for 66–72 h.</li>
<li>Harvest the retrovirus and pass the supernatant through a 0.22 μm filter.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The color of the medium should be red/orange but not yellow. The Phoenix-A cells can be harvested and lysed for assessing expression level of protein of interest by SDS-PAGE and immunoblotting with specific antibodies.</div>
<h3 id="sec3.3">Preparation of IL-2Rα antibody conjugated magnetic beads</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min and 30 min</div>
<p>For using the IL-2Rα antibody conjugated magnetic beads in cell selection in the next section, the preparation is done in a sterile environment, e.g., laminar flow hood.</p>
<ol start="8">
<li>Prepare 30 mL of 0.1% FBS in PBS (referred to as PBS/FBS).</li>
<li>Aliquot 1.8 mL of Dynabeads Goat Anti-Mouse IgG Magnetic Beads into three 1.5 mL microcentrifuge tubes (600 μL each).</li>
<li>Place the tubes on a magnetic stand for 2 min and gently remove the supernatant. Remove the tubes from the magnetic stand, and gently resuspend the beads with 1 mL PBS/FBS.</li>
<li>Repeat the magnetic capturing and washing (step 10) twice. After the last wash, gently resuspend the beads with 600 μL PBS/FBS.</li>
<li>Dissolve 200 μg of lyophilized anti-IL-2Rα antibody (whole vial) with 600 μL of PBS/FBS. Transfer 20 μL to a separate tube, this is the “input” sample for later SDS- PAGE analysis of samples.</li>
<li>Add 200 μL of anti-IL-2Rα antibody to each tube of beads.</li>
<li>Incubate the tubes on a slow rotator in the cold room for 12–16 h.</li>
<li>Place the tubes on a magnetic stand for 2 min and gently transfer the supernatant to a 15-mL tube. This sample is considered “flow though.” Remove the tubes from the magnetic stand, and gently resuspend the beads with 1 mL PBS/FBS.</li>
<li>Repeat the magnetic capturing and washing (step 15) three times, though without keeping the supernatant.</li>
<li>For each tube, resuspend the beads in 1 mL PBS/FBS supplemented with 50% glycerol. These are the “coupled beads.” Aliquot the beads in 200 μL per 1.5 mL microcentrifuge tubes. Store the beads at −20°C.</li>
<li>Analyze “input,” “flow through” as well as “coupled beads” by SDS-PAGE followed by Coomassie stain. IgG should be present in “input” and “coupled beads” samples, but not in the “flow through” sample.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Excessive freeze-thaw cycles will lead to dissociation of the IL-2Rα antibody from the magnetic beads.</div>
<h3 id="sec3.4">Transducing HeLa S3 cells with retrovirus and sorting for transduced cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–15 min and 1.5 h</div>
<p>DMEM supplemented with 2.5%–10% FBS is used when HeLa S3 cells are grown as semi-adherent monolayer in cell culture dishes in an incubator at 37°C and 5% CO<sub>2</sub>. We normally keep the cells in culture for less than 2 months.</p>
<ol start="19">
<li>Supplement the retrovirus stock with 4 μg/mL of Hexadimethrine bromide (polybrene).</li>
<li>Detach the HeLa S3 cells by treating the cells with PBS supplemented with 2 mM EDTA, and pellet down roughly 200,000 cells in a 15 mL tube (188 × <i>g</i>, 20°C, 5 min).</li>
<li>Resuspend the cells with the virus stock and plate them in a 9 cm dish.</li>
<li>Let the cells grow in the virus stock overnight (less than 24 h).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Polybrene is toxic to the cells. It is important to not leave the cells in medium with retrovirus and polybrene for longer than 24 h. Longer incubation can lead to increased cell death.</div>
<ol start="23">
<li>Collect and centrifuge the supernatant (188 × <i>g</i>, 20°C, 5 min).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Normally there will be a portion of cells that remain partially adhered.</div>
<ol start="24">
<li>Discard the supernatant, resuspend the cell pellet with fresh supplemented DMEM, and plate the cells back in the 9 cm dish. Allow to grow in an incubator at 37°C and 5% CO<sub>2</sub> till the HeLa S3 cells reach 60%–80% confluency (66–72 h).</li>
<li>Detach the transduced HeLa S3 cells by treating the cells with PBS supplemented with 2 mM EDTA, and centrifuge (188 × <i>g</i>, 20°C, 5 min).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If trypsin was used, it is important not to over-trypsinize the cells. IL-2Rα, which is localized on the cell surface and used for sorting in the following step, could be damaged leading to ineffective sorting.</div>
<ol start="26">
<li>Aspirate the supernatant and resuspend the cells with 1 mL fresh supplemented DMEM in a 1.5 mL microtube.</li>
<li>Add 10 μL of 50% slurry IL-2Rα antibody conjugated magnetic beads (prepared in steps 8–18) to the cells and leave the tube on a roller at 20–25°C for 1 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The incubation is to allow the IL-2Rα antibody conjugated magnetic beads to properly bind to the recombinant IL-2Rα on the cell surface. Do not leave the tube rolling for more than 1 h.</div>
<ol start="28">
<li>Place the tube on a magnetic stand for 2 min, and gently remove the supernatant. Remove the tube from the magnetic stand, and gently resuspend the cells with the beads with 1 mL fresh DMEM.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not pipette vigorously or vortex the tube. The IL-2Rα antibody conjugated magnetic beads could detach from the cells due to shear force.</div>
<ol start="29">
<li>Repeat the magnetic capturing and washing (step 28) twice.</li>
<li>Resuspend the cells with 1 mL supplemented DMEM and plate them into a 9 cm dish containing 9 mL of supplemented DMEM.</li>
</ol>
<p>Alternatives: the cells can be diluted with conditioned medium and plated on 96-well plate as single cell in each well to select for individual clones.</p>
<div class="note">
<span class="note-title">Note:</span> Due to the bi-cistronic expression of both the target gene and IL-2Rα, the number of magnetic beads bound to the cell surface could be used to assess the expression level of target gene prior to immunoblot analysis. It should be noted, that the ideal level of expression of the FLAG-HA-tagged protein is typically such that the level is comparable to that of the endogenous protein (<a href="#fig2">Figure 2</a>). Overexpression of the tagged protein can lead to abnormal cellular localization, spurious protein-protein interaction, and other artifacts.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/481-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Expression of FLAG-HA-tagged FANCD2 in HeLa cells</div>
<p>FLAG-HA-tagged FANCD2 is expressed at levels comparable to endogenous FANCD2 in HeLa S3 cells. Immunoblot analysis.</p>
</figcaption>
</figure>
<h3 id="sec3.5">Preparation of nuclear extract from HeLa S3 cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4.5 h</div>
<p>This part of the protocol is a modified version of an original protocol of cellular fractionation developed by the laboratory of Robert Roeder (<a href="#bib2">Dignam et al., 1983</a>). We introduced a DNA digestion step with Benzonase nuclease to reduce chromatin mediated interaction, and to permit purification of complexes associated with chromosomes. Furthermore, we introduced a formaldehyde fixation step adapted from other cell systems (<a href="#bib1">Aparicio et al., 2004</a>) to preserve weak protein-protein interactions, which otherwise could be lost during the purification. HeLa S3 cells are grown as suspension cultures in spinner flasks in an incubator at 37°C (no CO<sub>2</sub> regulation required) using Joklik’s Modified MEM supplemented with 2.5% FBS, 2 mM L-glutamine, non-essential amino acids (100× dilution) and penicillin-streptomycin (100× dilution). HeLa S3 cells are normally grown in DMEM supplemented with 2.5% FBS in dishes as described in the previous section. To set up larger cultures in spinner flask, HeLa S3 cells from an 80% confluent 15 cm dish are detached and centrifuged as described in the previous section. The cells are resuspended in 50 mL supplemented Joklik’s medium and grown in a 125 mL spinner flask in a 37°C incubator. The color of the medium should change to orange (but not yellow) over time, and the culture is expanded by adding in equal volume of fresh supplemented Joklik’s medium to the same flask (or transferred to a larger spinner flask). The volume of the culture will double each day until the desired scale is reached. We optimized each step in the following nuclear extract preparation and purification of protein complexes with 1 L starting culture. Once optimal purification results were achieved, we scaled the experiment to 6 L culture. The following steps indicate volume used for 1 L starting culture.</p>
<div class="note">
<span class="note-title">Note:</span> The fractionation procedure is carried out at 4°C or on ice throughout. All the buffers are pre-chilled to 4°C and freshly supplemented with 0.2 mM PMSF and 2 mM β-ME.</div>
<ol start="31">
<li>Treat the cells with 160 ng/mL Mitomycin C for 16 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step is optional. The Mitomycin C treatment is to induce DNA interstrand crosslinks (ICLs) in the genome and to activate the ICL repair pathways. Alternative treatment, or using unperturbed cells, could be relevant.</div>
<ol start="32">
<li>Harvest the cells by centrifuging the culture (1,800 × <i>g</i>, 4°C, 5 min) and discard the supernatant.</li>
<li>Wash the cell pellet by resuspending it with 50 mL cold PBS, centrifuge (1,800 × <i>g</i>, 4°C, 5 min) and discard the supernatant.</li>
<li>Equilibrate the cell pellet with hypotonic buffer by resuspending the cell pellet with 2 mL of hypotonic buffer. Centrifuge (1,800 × <i>g</i>, 4°C, 5 min) and discard the supernatant.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not leave the cells incubated in the hypotonic buffer at this step. Otherwise, the cells will start swelling, which will affect the efficiency of fractionation.</div>
<ol start="35">
<li>Resuspend and incubate the pellet in 4 mL (twice pellet volume) of hypotonic buffer on ice for 10 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The cells will be swollen at the end of the incubation. It is important to observe the changes of cell size under a microscope to confirm the efficiency of hypotonic treatment.</div>
<ol start="36">
<li>Transfer the solution into a pre-chilled glass Dounce homogenizer on ice.</li>
<li>Use a B type, tight pestle, with a clearance of 20–56 μm, to press up and down promptly and steadily for 10–20 cycles on ice.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Dounce homogenizers can vary slightly, and different cell types will require varying amount of cycles. It is important to collect samples of cells before and after homogenization, and examine these under a microscope to evaluate the efficiency of cell membrane rupture, which should be above 90%. Continue homogenization until sufficient cell breakage is achieved. Insufficient rupture will lead to the cytoplasmic fraction from unbroken cells contaminating the nuclear fraction. On the other hand, excessive homogenization will result in breakdown of the nuclear envelope, leading to the loss of soluble nuclear proteins from nuclear fractions.</div>
<ol start="38">
<li>Centrifuge the solution to pellet nuclei (3,300 × <i>g</i>, 4°C, 15 min).</li>
<li>Remove the top lipid layer by inserting a clean pipette tip into the solution, which the lipid layer will adhere to.</li>
<li>
<b>Cytoplasmic fraction</b>: Collect the supernatant as the cytoplasmic fraction and supplement with 11% of the volume of 10× S-100 buffer.
<table id="undtbl22">
<caption>S-100 buffer (10× stock) 100 mL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl pH 7.3</td>
<td>300 mM</td>
<td>30 mL</td>
</tr>
<tr>
<td>4 M KCl</td>
<td>1.4 M</td>
<td>35 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>30 mM</td>
<td>3 mL</td>
</tr>
<tr>
<td colspan="3">Add ddH<sub>2</sub>O to 100 mL. Store at 4°C for up to 1 year</td>
</tr>
</tbody>
</table>
</li>
<li>Centrifuge the cytoplasmic fraction (17,000 × <i>g</i>, 4°C, 30 min) and remove the lipid layer as described above.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The cytoplasmic fraction can be dialyzed against the desired buffer if needed for subsequent purification. In this protocol, we focus on isolating nuclear protein complexes.</div>
<ol start="42">
<li>
<b>Nuclear fraction with Benzonase digestion and formaldehyde fixation</b>: Resuspend the nuclei in 1 mL (half of pellet volume) low-salt NE buffer supplemented with 0.01% Triton X-100 and 75 unit/mL Benzonase, and leave it incubating on ice for 30 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of Benzonase added and the incubation time can be adjusted if needed. Digestion of chromosomal DNA can be monitored by agarose gel electrophoresis.</div>
<ol start="43">
<li>Supplement the mixture with formaldehyde to final concentration of 1% and incubate at 20–25°C for 4 min to crosslink the nuclei.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The nuclei solution will turn opaque and slightly clumpy after crosslinking.</div>
<ol start="44">
<li>Quench the formaldehyde by adding 150 μL (1/20 volume) of 2.5 M Glycine pH 7.5 to the mixture.</li>
<li>Centrifuge the mixture (3,000 × <i>g</i>, 4°C, 10 min) and discard the supernatant.</li>
<li>Wash the crosslinked nuclei by resuspending the nuclei with 10 mL (10 times pellet volume) of PBS, centrifuge (3,000 × <i>g</i>, 4°C, 10 min) and discard the supernatant.</li>
<li>Repeat step 46 twice to wash the nuclei two more times.</li>
<li>Resuspend the crosslinked nuclei with 1 mL (same pellet volume) of high-salt NE buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The protein complexes are now crosslinked and stabilized, so we can use extraction condition with high stringency without disrupting protein complexes.</div>
<ol start="49">
<li>Sonicate the crosslinked nuclei for 20 min (with 1 s on/off intervals) at 4 W power in an ice/water slurry bath.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sample would normally turn clear after sonication, and the pellet from centrifugation (step 52) is small.</div>
<div class="note">
<span class="note-title">Note:</span> In addition to the Benzonase digestion, this sonication step renders chromatin DNA to smaller fragments.</div>
<ol start="50">
<li>Leave the sonicated nuclei incubating on ice for 30 min.</li>
<li>Dilute the lysate 5-fold with 8 mL low-salt NE buffer supplemented with 125 mM NaCl and 1% Tween 20.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This dilution step is to lower the concentration of SDS from 0.15% to 0.03% and NaCl from 1 M to 300 mM to make it compatible with anti-FLAG M2 agarose used in the next stage.</div>
<ol start="52">
<li>Centrifuge the solution (17,000 × <i>g</i>, 4°C, 20 min) and collect the supernatant as nuclear extract.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The protein complexes should be preserved by formaldehyde crosslinking (step 43). The nuclear extract can be snap-frozen using liquid nitrogen and stored at −80°C for purification later.</div>
<h3 id="sec3.6">Immunoaffinity purification</h3>
<p>Sequential affinity purifications usually yield proteins of higher purity or, in this case, more pure protein complexes. However, the handling during sequential purification usually leads to loss of some interacting proteins, especially those with weaker affinities. With our modified nuclear extraction protocol, we are able to perform the sequential affinity purifications with higher stringency to reduce non-specific binding to the antibodies and matrixes while retaining the specific protein complexes. In this protocol, we purify FANCD2 sequentially via the FLAG and HA tags engineered at its N-terminus. Though using this protocol, we typically observe negligible level of contaminants (<a href="#bib11">Socha et al., 2020</a>), we recommend performing a parallel “mock” purification from cells not expressing a FLAG-HA-tagged protein, to assess the level of background binding proteins.</p>
<p>The purification is carried out at 4°C or on ice throughout.</p>
<h3 id="sec3.7">FLAG tag affinity purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 h</div>
<ol start="53">
<li>Add 250 μL bed volume of anti-FLAG M2 agarose beads to the 10 mL nuclear extract, and incubate the tube on a rotator at 10 rpm for 1.5–2 h at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of anti-FLAG M2 agarose used in the purification is determined empirically by monitoring the depletion of FANCD2 from the nuclear extract during purification. We use 250 μL bed volume of anti-FLAG M2 agarose for a 1 L HeLa S3 culture. When targeting different proteins with distinct expression level, it will require optimization.</div>
<div class="note">
<span class="note-title">Note:</span> Prior to use, the anti-FLAG M2 agarose beads are washed with 5 column volume (c.v.) of 100 mM glycine pH 2.5, 10 c.v. of ddH<sub>2</sub>O and 10 c.v. of buffer A. The glycine wash is to remove antibodies not crosslinked to the agarose matrix.</div>
<div class="note">
<span class="note-title">Note:</span> Anti-FLAG purification is performed in batch mode to improve the binding efficiency.</div>
<ol start="54">
<li>Centrifuge the mixture (1,000 × <i>g</i>, 4°C, 5 min) and collect the supernatant as flow- through fraction of anti-FLAG purification step.</li>
<li>Resuspend the beads with 1 mL Buffer A and transfer the beads to a clean gravity-flow chromatography column.</li>
<li>Carefully apply 10 mL Buffer A to wash the beads and repeat the washing step two more times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The washing of the beads is more efficient if allowing most of the buffer to drip through before applying more buffer.</div>
<ol start="57">
<li>Cap the outlet and apply 250 μL (equal volume of the anti-FLAG beads) of Buffer A supplemented with 0.5 mg/mL FLAG peptide. Seal the top with parafilm and leave the column on a roller at 4°C for 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We perform the elution in batch because it allows FLAG peptide to compete away FLAG-tagged FANCD2 from anti-FLAG beads efficiently, helping to keep the volume of eluate low, thus more concentrated.</div>
<ol start="58">
<li>Collect the FLAG eluate and repeat the elution (step 57) once more.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> An extra elution can be performed, while we usually observe a decrease in concentration.</div>
<ol start="59">
<li>Pool all FLAG eluates.</li>
</ol>
<h3 id="sec3.8">HA tag affinity purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 h</div>
<p>Overall, the HA tag affinity purification is carried out similarly to the FLAG tag affinity purification described in the previous section with minor modifications. We generally use less anti-HA Sepharose beads comparing to the anti-FLAG M2 agarose used in the previous section. We use 0.5 mg/mL HA peptide for elution.</p>
<ol start="60">
<li>Add 60 μL bed volume anti-HA Sepharose beads to the FLAG eluate from the previous step, and incubate the tube on a rotator at 10 rpm for 2 h at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Similarly to step 53, the necessary amount of anti-HA Sepharose needs to be determined empirically.</div>
<div class="note">
<span class="note-title">Note:</span> Prior to use, the anti-HA beads are washed and equilibrated as described for the anti-FLAG M2 agarose beads in step 53, the second Note.</div>
<div class="note">
<span class="note-title">Note:</span> We perform the anti-HA purification in batch because it provides better binding efficiency.</div>
<ol start="61">
<li>Centrifuge the mixture (1,000 × <i>g</i>, 4°C, 5 min) and collect the supernatant as the flow-through fraction of the anti-HA purification step.</li>
<li>Resuspend the beads with 1 mL Buffer A and transfer the beads to a clean gravity-flow chromatography column.</li>
<li>Carefully apply 5 mL Buffer A to wash the beads and repeat the washing step two more times.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The washing of the beads is more efficient if allowing most of the buffer to drip through before applying more buffer.</div>
<ol start="64">
<li>Cap the outlet and apply 60 μL (equal volume to the anti-HA beads) of Buffer A supplemented with 0.5 mg/mL HA peptide. Seal the top with parafilm and leave the column on a roller at 4°C for 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We perform the elution in batch because it allows HA peptide to compete away HA-tagged FANCD2 from anti-HA beads efficiently, which helps to keep the volume of eluate low with good concentration.</div>
<p>Alternatives: the protein complexes can be eluted from the anti-HA Sepharose matrix with 0.1 M Glycine pH 2.5, followed by pH neutralization of the eluate by adding 1/5 volume of 1.5 M Tris pH 8.8 immediately after collecting the eluate. It is faster and more efficient using Glycine for elution. However, it could also elute proteins bound non-specifically to the matrix.</p>
<ol start="65">
<li>Collect the HA eluate and repeat the elution (step 64) once more.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> An extra elution can be performed, though we usually observe a decrease in the overall concentration as a result.</div>
<h3 id="sec3.9">Reversion of formaldehyde crosslinks and sample preparation for mass spectrometry analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5 h</div>
<p>We reverse the formaldehyde crosslinks and alkylate our sample with Iodoacetamide prior to mass spectrometry analysis. Different mass spectrometry processes may require samples being processed differently after reversal of formaldehyde crosslinks.</p>
<ol start="66">
<li>Supplement the HA eluate with 0.1% SDS and 5 mM DTT and incubate the sample at 95°C for 20 min to reverse the formaldehyde crosslinks.</li>
</ol>
<p>Alternatives: the HA eluate can be supplemented with SDS-PAGE loading buffer, incubate the samples at 95°C for 20 min, and load the samples onto SDS-PAGE followed by Coomassie or silver staining for analysis.</p>
<ol start="67">
<li>Dilute the sample ten-fold with milli-Q water supplemented with 5 mM DTT and cool the sample on ice for 5 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The dilution is to prevent SDS from precipitating on ice at higher concentration. If some precipitants are formed, one could slightly warm up the sample till the precipitants are dissolved again.</div>
<ol start="68">
<li>Supplement the sample with 15 mM Iodoacetamide and incubate the sample at 20–25°C protected from light for 20 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Iodoacetamide is light sensitive. It is important to protect the sample from light.</div>
<ol start="69">
<li>Supplement the sample with 10 mM of DTT and incubate the sample at 20–25°C protected from light for 15 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The concentration of DTT at this step is 15 mM.</div>
<ol start="70">
<li>Supplement the sample with 10% TCA by mixing equal volume of sample with 20% TCA and incubate the sample on ice 2–16 h, protected from light.</li>
<li>Centrifuge the sample (17,000 × <i>g</i>, 4°C, 30 min) and discard the supernatant.</li>
<li>Rinse the pellet with −20°C 100% acetone gently, centrifuge the sample (17,000 × <i>g</i>, 4°C, 10 min) and discard the supernatant carefully.</li>
<li>Let the pellet air-dry at 20–25°C.</li>
<li>Analyze the precipitated protein complexes by MS/MS.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol allows purification of FLAG- and HA-tagged protein containing complexes. The optimized formaldehyde crosslinking protocol allows co-purification of weakly interacting proteins. We typically analyze replicate purifications with SDS-PAGE followed by silver or Coomassie staining to test for reproducibility (step 66). An example of this is shown in <a href="#fig3">Figure 3</a>, which show two independent purifications of FANCD2 complexes. One of the two purifications is identical to a previously published purification (<a href="#bib11">Socha et al., 2020</a>). The gel image is shown again here, side-by-side with an independent purification, as an example of how robust and reproducible the method is.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/481-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Purification of FANCD2-containing protein complexes from HeLa cells is highly reproducible</div>
<p>FANCD2-containing complexes purified from two independent experiments were resolved by SDS-PAGE and visualized using silver stain. Lane 2, small scale purification (experiment A). Lane 3, large scale purification (experiment B). Comparable profiles of polypeptides are observed. A mock purification (lane 1) contains essentially no polypeptides. The clean mock purification and the reproducibility of purified complexes indicate the robustness of the protocol. Parts of this figure (marker lane and lanes 1 and 3) are identical to previously published data, and are reprinted with permission (<a href="#bib11">Socha et al., 2020</a>).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol was optimized to purify FANCD2 containing complexes. FANCD2 is a nuclear protein, which is recruited to chromatin during replication and/or upon DNA damage. The formaldehyde crosslinking step was introduced after nuclei isolation. If the protein of interest is cytoplasmic or soluble nuclear protein, the timing of formaldehyde crosslinking needs to be further optimized. The FLAG and HA tags on the N-terminus of FANCD2 permits purification of this protein, however other proteins might require C-terminal tagging to permit purification, and to preserve function of the fusion protein. We discuss this potential issue in the troubleshooting section problem 5.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low expression of target gene in HeLa S3 cells after retrovirus transduction (step 19).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>The quality of retrovirus packaging in Phoenix-A cells is very important. Good transfection efficiency is essential. Other transfection agents can also be used but require further optimization.</p>
<p>The color of medium, which reflects the nutrition level and the pH, on the day of harvesting retrovirus is critical. If the cellular density of the Phoenix-A cells for transfection is too high or the retrovirus is harvested too late, the nutrition in the medium would be depleted, which cannot support the HeLa S3 cells during the retrovirus transduction step. If the medium is yellow on the day of harvesting retrovirus, one can try to replace the retrovirus containing medium after 6 h or supplement with some fresh medium.</p>
<p>Aggregation of the HeLa S3 cells during incubation with anti-IL-2Rα antibodies conjugated magnetic beads can lead to contamination by non-transduced cells. Additional rounds of cell selection with anti-IL-2Rα antibodies conjugated magnetic beads can be applied a few days after the first round of sorting to select for true-positive cells.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Contamination of cytoplasmic proteins in nuclear fraction or vice versa (steps 37, 40, and 42).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>We routinely check the efficiency of our cellular fractionation by immunoblotting using antibodies against cytoplasmic proteins, e.g., α-Tubulin, and nuclear or chromatin proteins, e.g., Lamin B and histones.</p>
<p>The first wash with hypotonic buffer (step 34) is designed to increase the efficiency of the hypotonic treatment in step 35. However, it is important to centrifuge cells immediately after resuspension, follow the incubation times carefully, and to monitor the cells under a microscope. If different cell type is used, the time of incubation and number of strokes of the Dounce homogenizer should be optimized.</p>
<p>As stated in the step-by-step section, there can be variation between Dounce homogenizers. It is important to examine the before and after samples carefully under a light microscope. If the nuclear envelope is disrupted, try reducing the number of strokes. Likewise, if plasma membranes are not disrupted sufficiently, increase the number of strokes. When collecting the cytoplasmic fraction after centrifugation of nuclei, try to remove as much solution as possible without disturbing the nuclei pellet.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Inconsistent fragmentation of chromatin DNA by Benzonase digestion and sonication (step 42).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>This protocol relies on Benzonase to digest chromosomal DNA to smaller fragments. We supplemented the buffer with 0.01% Triton X-100 to improve the accessibility of chromatin to Benzonase. We tried to keep the concentration of detergent as low as possible at this step to avoid disrupting any protein-protein interaction. One could try to further increase the concentration of detergent or increase the amount of Benzonase used at this step.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Insufficient extraction of target protein (step 52).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Sonication (step 49) helps the rupture of nuclei and further shear the Benzonase digested DNA to smaller fragments. The crosslinked nuclei should be sonicated over a time-course to identify optimal condition for protein extraction. We monitor the extraction of target protein by immunoblotting. Both the power level and sonication time can be adjusted. While the nuclei are crosslinked and protein complexes are preserved at step 43, it is important to avoid warming up the solution during sonication.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Low recovery of target protein during immunoaffinity purification (steps 58 and 65).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>This protocol was developed to purify FANCD2-containing protein complexes. Different proteins of interest might provide different accessibility for purification. First, confirm that the tags used for purification are accessible during non-crosslinked native condition. It is possible that formaldehyde crosslinking affects the accessibility of the tags by stabilizing some interacting proteins near the tags. Extending the linker between tag and protein of interest or reducing the degree of crosslinking by lowering the formaldehyde concentration or shortening the time of crosslinking at step 43 can help. C-terminal tagging rather than N-terminal tagging can also be considered.</p>
<p>We optimized our protocol with the use of formaldehyde as the crosslinking agent. However, there are limitation due to its small size (about 2 Å) and amino group preferences (<a href="#bib3">Hoffman et al., 2015</a>; <a href="#bib5">Metz et al., 2004</a>), which could lead to loss of true interacting partners identified in the subsequent mass spectrometry analysis. Similar to conventional chromatin immunoprecipitation (ChIP), other crosslinking agents, such as EGS (ethylene glycol bis(succinimidyl succinate)) or DSG (di(N-succinimidyl glutarate)) can be used in substitute or in combination of formaldehyde to improve crosslinking efficiency (<a href="#bib10">Singh et al., 2019</a>; <a href="#bib12">Tian et al., 2012</a>; <a href="#bib13">Zeng et al., 2006</a>). If other crosslinkers are used, timing of crosslinking, quenching and extraction will require further optimization. It should be noted that not all crosslinks are reversible, which can affect the subsequent mass spectrometry analysis.</p>
<p>On the other hand, crosslinks between proteins can be used to successfully map direct protein-protein interactions via mass spectrometry (<a href="#bib6">Mintseris and Gygi, 2020</a>; <a href="#bib9">O'Reilly and Rappsilber, 2018</a>).</p>
<p>The affinity between different tags and their binding counterparts also varies. Reducing the stringency of binding and washing condition at step 53 and onwards are options. It is possible to extend the time of binding at step 53. Alternative tags can be considered.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aparicio, O., Geisberg, J.V., and Struhl, K. (2004). Chromatin immunoprecipitation for determining the association of proteins with specific genomic sequences in vivo. Curr. Protoc. Cell Biol. <i>Chapter 17</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref1">View at publisher</a></p>
<p id="bib2">Dignam, J.D., Martin, P.L., Shastry, B.S., and Roeder, R.G. (1983). Eukaryotic gene transcription with purified components. Methods Enzymol. <i>101</i>, 582-598. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref2">View at publisher</a></p>
<p id="bib3">Hoffman, E.A., Frey, B.L., Smith, L.M., and Auble, D.T. (2015). Formaldehyde crosslinking: a tool for the study of chromatin complexes. J. Biol. Chem. <i>290</i>, 26404-26411. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref3">View at publisher</a></p>
<p id="bib4">Lopez-Martinez, D., Kupculak, M., Yang, D., Yoshikawa, Y., Liang, C.C., Wu, R., Gygi, S.P., and Cohn, M.A. (2019). Phosphorylation of FANCD2 inhibits the FANCD2/FANCI complex and suppresses the Fanconi anemia pathway in the absence of DNA damage. Cell Rep. <i>27</i>, 2990-3005.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref4">View at publisher</a></p>
<p id="bib5">Metz, B., Kersten, G.F., Hoogerhout, P., Brugghe, H.F., Timmermans, H.A., de Jong, A., Meiring, H., ten Hove, J., Hennink, W.E., Crommelin, D.J., et al. (2004). Identification of formaldehyde-induced modifications in proteins: reactions with model peptides. J. Biol. Chem. <i>279</i>, 6235-6243. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref5">View at publisher</a></p>
<p id="bib6">Mintseris, J. and Gygi, S.P. (2020). High-density chemical cross-linking for modeling protein interactions. Proc. Natl. Acad. Sci. U S A <i>117</i>, 93-102. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref6">View at publisher</a></p>
<p id="bib7">Morita, S., Kojima, T., and Kitamura, T. (2000). Plat-E: an efficient and stable system for transient packaging of retroviruses. Gene Ther. <i>7</i>, 1063-1066. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref7">View at publisher</a></p>
<p id="bib8">Nakatani, Y. and Ogryzko, V. (2003). Immunoaffinity purification of mammalian protein complexes. Methods Enzymol. <i>370</i>, 430-444. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref8">View at publisher</a></p>
<p id="bib9">O'Reilly, F.J. and Rappsilber, J. (2018). Cross-linking mass spectrometry: methods and applications in structural, molecular and systems biology. Nat. Struct. Mol. Biol. <i>25</i>, 1000-1008. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref9">View at publisher</a></p>
<p id="bib10">Singh, A.A., Schuurman, K., Nevedomskaya, E., Stelloo, S., Linder, S., Droog, M., Kim, Y., Sanders, J., van der Poel, H., Bergman, A.M., et al. (2019). Optimized ChIP-seq method facilitates transcription factor profiling in human tumors. Life Sci. Alliance <i>2</i>, e201800115. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref10">View at publisher</a></p>
<p id="bib11">Socha, A., Yang, D., Bulsiewicz, A., Yaprianto, K., Kupculak, M., Liang, C.C., Hadjicharalambous, A., Wu, R., Gygi, S.P., and Cohn, M.A. (2020). WRNIP1 is recruited to DNA interstrand crosslinks and promotes repair. Cell Rep. <i>32</i>, 107850. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref11">View at publisher</a></p>
<p id="bib12">Tian, B., Yang, J., and Brasier, A.R. (2012). Two-step cross-linking for analysis of protein- chromatin interactions. Methods Mol Biol <i>809</i>, 105-120. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref12">View at publisher</a></p>
<p id="bib13">Zeng, P.Y., Vakoc, C.R., Chen, Z.C., Blobel, G.A., and Berger, S.L. (2006). In vivo dual cross- linking for identification of indirect DNA-associated proteins by chromatin immunoprecipitation. Biotechniques <i>41</i>, 694. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00055-1/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Martin A. Cohn (<a href="mailto:martin.cohn@bioch.ox.ac.uk">martin.cohn@bioch.ox.ac.uk</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new data or code.</p>
<h3>Acknowledgments</h3>
<p>The authors thank Luke Greenhough and Marcel Tuppi for reading and discussing the manuscript. We thank members of the Cohn laboratory for helpful discussions and comments. This work was supported by grants UF100717 and UF150651 from the <a href="https://doi.org/10.13039/501100000288">Royal Society</a> (to M.A.C.), grant 153/092 from The Fell Fund (to M.A.C.), grants OCRC 0213-MC and CRUKDF 0715-MC from the Oxford Cancer Research Centre (OCRC)/<a href="https://doi.org/10.13039/501100000289">Cancer Research UK</a> (CR-UK) (to M.A.C.), a grant from the Medical Research Fund (to M.A.C.), grant MR/N021002/1 from the <a href="https://doi.org/10.13039/501100000265">Medical Research Council</a> (MRC) (to M.A.C.), a <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> Senior Research Fellowship 210640/Z/18/Z (to M.A.C.), the Taiwanese Government (to C.-C.L.), and a Goodger scholarship (to C.-C.L.).</p>
<h3>Author Contributions</h3>
<p>C.-C.L. developed and wrote the method. M.A.C. oversaw the project.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>