<article>
<section>
<h1 id="header">Nuclear RNA purification by flow cytometry to study nuclear processes in plants</h1>
<p><time datetime="2021-02-12">Published: February 12, 2021</time></p>
<p>Belén Moro,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a></sup> Malgorzata Kisielow,<sup><a href="#aff2">2</a></sup> Veronica Barragan Borrero,<sup><a href="#aff3">3</a></sup> Antoine Bouet,<sup><a href="#aff1">1</a></sup> Christopher A. Brosnan,<sup><a href="#aff4">4</a></sup> and Nicolás G. Bologna<sup><a href="#aff1">1</a>,<a href="#fn2">6</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Centre for Research in Agricultural Genomics (CRAG), CSIC-IRTA-UAB-UB, Bellaterra, Barcelona 08193, Spain</p>
<p id="aff2"><sup>2</sup>ETH Zurich Flow Cytomtery Core Facility, Zurich 8093, Switzerland</p>
<p id="aff3"><sup>3</sup>GMI-Gregor Mendel Institute of Molecular Plant Biology (ÖEAW), Vienna 1030, Austria</p>
<p id="aff4"><sup>4</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, QLD 4072, Australia</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:nicolas.bologna@cragenomica.es">nicolas.bologna@cragenomica.es</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100320">10.1016/j.xpro.2021.100320</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The nature of plant tissues has continuously hampered understanding of the spatio-temporal and subcellular distribution of RNA-guided processes. Here, we describe a universal protocol based on <i>Arabidopsis</i> to investigate subcellular RNA distribution from virtually any plant species using flow cytometry sorting. This protocol includes all necessary control steps to assess the quality of the nuclear RNA purification. Moreover, it can be easily applied to different plant developmental stages, tissues, cell cycle phases, experimental growth conditions, and specific cell type(s).</p>
<p>For complete information on the use and execution of this protocol, please refer to <a href="#bib1">Bologna et al. (2018)</a> and <a href="#bib6">de Leone et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of highly pure nuclear RNA from essentially any plant species</li>
<li>Full procedure describing key steps for quality validation of nuclear RNA purification</li>
<li>Applicable to different tissues, cell cycle phases, stresses, or specific cell types</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/462-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Prepare all the necessary buffers, filter and store them according to the following instructions. Grow the plants from which you wish to purify the nuclear RNA. Order the necessary primers for the qPCR analyses (See <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Table S1</a> for a list of suggested primers.).</p>
<h3 id="sec1.1">Plant growth conditions</h3>
<p>To perform nuclear RNA extraction from <i>Arabidopsis</i> seedlings, 2 weeks before the experiment grow <i>Arabidopsis</i> seedlings on Murashige and Skoog (MS) media plates under continuous white light (100 mmol photons m<sup>−2</sup> s<sup>−1</sup>) at 22°C.</p>
<div class="note">
<span class="note-title">Note:</span> We recommend at least two biological replicates for every sample, using ∼100 <i>Arabidopsis</i> seedlings per sample. A larger number of biological replicates can be included depending on the experimental design.</div>
<div class="note">
<span class="note-title">Note:</span> Growing time should be adjusted depending plant tissues collected (e.g., leaves, inflorescences) and plant species used.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Long-day (16 h light/8 h dark) or short-day conditions (8 h light/16 h dark) can be used as well.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>cOmplete, Mini, EDTA-free tablets</td>
<td>Roche</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/roche/11836170001?lang=es&amp;region=ES">11836170001</a>
</td>
</tr>
<tr>
<td>Formaldehyde 37%</td>
<td>Sigma-Aldrich</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/sigma/f8775?lang=es&amp;region=ES&amp;cm_sp=Insite-_-caContent_prodMerch_gruCrossEntropy-_-prodMerch10-1">F8775</a>
</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma- Aldrich</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/sigma/d9542?lang=es&amp;region=ES&amp;gclid=CjwKCAjw0On8BRAgEiwAincsHEjoqz4Muj76fwM-RTz3MIRc4AvSFNrc_oHO3HJkcGGjRnA6vQxKERoCY5UQAvD_BwE">D9542</a>
</td>
</tr>
<tr>
<td>Glycogen</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#:<a href="https://www.thermofisher.com/order/catalog/product/R0561?SID=srch-hj-R0561">R0561</a>
</td>
</tr>
<tr>
<td>KAPA SYBR FAST for LightCycler480 qPCR Kit</td>
<td>Roche</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/roche/sflckb?lang=es&amp;region=ES">SFLCKB</a>
</td>
</tr>
<tr>
<td>Propidium iodide</td>
<td>Sigma-Aldrich</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/sigma/p4170?lang=es&amp;region=ES">P4170</a>
</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Roche</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/roche/rprotkro?lang=es&amp;region=ES">RPROTK-RO</a>
</td>
</tr>
<tr>
<td>RiboLock RNase inhibitor</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#:<a href="https://www.thermofisher.com/order/catalog/product/EO0381?SID=srch-hj-EO0381">EO0381</a>
</td>
</tr>
<tr>
<td>Spermine 97%</td>
<td>Sigma-Aldrich</td>
<td>Cat#:<a href="https://www.sigmaaldrich.com/catalog/product/sigma/s3256?lang=es&amp;region=ES&amp;gclid=CjwKCAjw0On8BRAgEiwAincsHFcD8HMBVQvt5wfY5MkZq_8UO8bMUBkm4aljMjHdUPyY4IUuKF7kpxoCRt8QAvD_BwE">S3256</a>
</td>
</tr>
<tr>
<td>Tris-HCl (1 M, pH 7.5)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#:<a href="https://www.thermofisher.com/order/catalog/product/15567027?SID=srch-hj-15567027">15567027</a>
</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Maxima First Strand cDNA Synthesis Kit for RT-qPCR, with dsDNase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#:<a href="https://www.fishersci.es/shop/products/thermo-scientific-fermentas-maxima-first-strand-cdna-synthesis-kit-rt-qpcr-dsdnase-2/15279064">K1671</a>
</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Processed data from Col-0 nuclear and total fractions</td>
<td>(<a href="#bib6">de Leone et al., 2020</a>)</td>
<td>GEO:<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120757">GSE120757</a>
</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Arabidopsis Col_ <i>pSHR</i>:NTF</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Primers for qPCR, see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Table S1</a>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo Software Version 10</td>
<td>Ashland, OR: Becton, Dickinson and Company; 2019</td>
<td>FlowJo</td>
</tr>
<tr>
<td>BD FACSDiva software Version 8.0.3</td>
<td>BD Biosciences</td>
<td>FACSDiva</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Vacuum pump</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Desiccator</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Razor blade double-sided 0.1 mm</td>
<td>Feather</td>
<td>Cat#:52-004180; <a href="https://www.labtech-em.com/em/feather-fa-10-carbon-steel-double-edge-razor-blades">https://www.labtech-em.com/em/feather-fa-10-carbon-steel-double-edge-razor-blades</a>
</td>
</tr>
<tr>
<td>5 mL polystyrene round-bottom tube with cell strainer cap</td>
<td>Falcon</td>
<td>Cat#:<a href="https://ecatalog.corning.com/life-sciences/b2c/US/en/General-Labware/Tubes/Tubes,-Round-Bottom/Falcon%C2%AE-Round-Bottom-Polystyrene-Tubes/p/352235">352235</a>
</td>
</tr>
<tr>
<td>4 laser BD FACS AriaIII cell sorter</td>
<td>Bioscience</td>
<td>N/A</td>
</tr>
<tr>
<td>1.5 mL screw-cap tubes</td>
<td>Sarstedt</td>
<td>Cat#:<a href="https://www.sarstedt.com/es/productos/laboratorio/microtubos-roscados-recipientes-de-reaccion/microtubos-roscados/producto/72.703.406/">72.703.406</a>
</td>
</tr>
<tr>
<td>Confocal microscope</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Mortar and pestle</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Bioruptor Plus sonication device</td>
<td>Diagenode</td>
<td>Cat#:<a href="https://www.diagenode.com/en/p/bioruptor-plus-sonication-device">B01020001</a>
</td>
</tr>
<tr>
<td>Refrigerated centrifuge</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Qubit fluorometer</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>LightCycler 480 Instrument II</td>
<td>Roche</td>
<td>Cat#:<a href="https://lifescience.roche.com/en_es/products/lightcycler14301-480-instrument-ii.html">05015243001</a>
</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="critical">
<span class="critical-title">Critical:</span> Use gloves and RNase-free material throughout the entire protocol including the preparation of the Buffers and solutions.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The Buffers and solutions described here were prepared in house. However, if after performing the protocol RNA degradation proves to be a problem, different options are available. In house buffers and solutions can be prepared using DEPC treated water or molecular biology grade solutions can be purchase ready to use. See <a href="#sec6.9">Problem 5</a> in the <a href="#troubleshooting">Troubleshooting</a> Section.</div>
<div class="note">
<span class="note-title">Note:</span> Once buffer and solutions are filtered they can be store at 4°C or −20°C for 4–6 months.</div>
<div class="note">
<span class="note-title">Note:</span> All solution and/or reagent filtrations were done using 0.22 μm filters.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare all solutions using ddH<sub>2</sub>O.</div>
<h3 id="sec2.1">Common stock buffers</h3>
<ul>
<li>100 mM Tris-HCl, pH 7.5. Make 1:10 dilution of 1 M Tris-HCl, pH 7.5 to a final volume of 500 mL.</li>
<li>100 mM Na<sub>2</sub>EDTA. Dissolve 16.75 g of Na<sub>2</sub>EDTA.2H<sub>2</sub>O in 450 mL.</li>
<li>5 M NaCl. Dissolve 29.2 g of NaCl in 100 mL. Use it to prepare NaCl 1 M (see next step) and filtrate and store the remaining ∼10 mL at 4°C.</li>
<li>1 M NaCl. Make 1:5 dilution of 5 M NaCl stock solution to a final volume of 450 mL.</li>
<li>1 M Glycine. Dissolve 22.52 g of Glycine in 300 mL. Filter and store at 4°C.</li>
<li>0.5 M Spermine. Dissolve 104.29 mg in 1 mL. Dispense into 100 μL aliquots and store at −20°C.</li>
<li>800 mM KCl. Dissolve 374.40 mg of KCl in 10 mL.</li>
<li>25% (v/v) Triton X-100. Make a 1:4 dilution of Triton X-100 to a final volume of 20 mL.</li>
<li>10% (w/v) SDS. Dissolve 1 g of SDS in 10 mL.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> If the solutions listed here are not going to be used straight away to prepare the Sample, NIB, and Lysis Buffer, they should be filtered and stored at 4°C or -20°C in the case of Spermine. The exceptions are the 5 M NaCl and Glycine that are not used to prepared any buffer and should be filtrated as indicated above.</div>
<h3 id="sec2.2">Sample buffer</h3>
<p>Approximately 100 mL is used per sample. We recommend preparing 4 L of the Sample Buffer. Mix the following components:</p>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (100 mM, pH 7.5)</td>
<td>10 mM</td>
<td>400 mL</td>
</tr>
<tr>
<td>Na<sub>2</sub>EDTA (100 mM)</td>
<td>10 mM</td>
<td>400 mL</td>
</tr>
<tr>
<td>NaCl (1 M)</td>
<td>100 mM</td>
<td>400 mL</td>
</tr>
<tr>
<td>Triton X-100 (25% (v/v))</td>
<td>0.1% (v/v)</td>
<td>16 mL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>N/A</td>
<td>2,784 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>4,000 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Filter and store at 4°C.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> Triton X-100 is a corrosive agent. Always handle with gloves under the hood and discard according to safety guidelines.</div>
<div class="note">
<span class="note-title">Note:</span> Part of the Sample Buffer is used to prepare the Cross-linking (CL) Buffer while the rest will be used as Sample Buffer during washing steps (see steps 1 and 5 respectively).</div>
<h3 id="sec2.3">Nuclei isolation buffer (NIB)</h3>
<p>Approximately 2 mL is used per sample. We recommend preparing 80 mL. Mix the following components:</p>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (100 mM, pH 7.5)</td>
<td>15 mM</td>
<td>12 mL</td>
</tr>
<tr>
<td>Na<sub>2</sub>EDTA (100 mM)</td>
<td>2 mM</td>
<td>1.60 mL</td>
</tr>
<tr>
<td>Spermine (0.5 M)</td>
<td>0.5 mM</td>
<td>80 μL</td>
</tr>
<tr>
<td>KCl (800 mM)</td>
<td>80 mM</td>
<td>8 mL</td>
</tr>
<tr>
<td>NaCl (1 M)</td>
<td>20 mM</td>
<td>1.60 mL</td>
</tr>
<tr>
<td>β-mercaptoethanol</td>
<td>15 mM</td>
<td>83.20 μL</td>
</tr>
<tr>
<td>Triton X-100 25% (v/v)</td>
<td>0.1% (v/v)</td>
<td>320 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>N/A</td>
<td>56.96 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>80 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Adjust pH to 7.5 with NaOH.</li>
<li>Filter and dispense into 10 mL aliquots.</li>
<li>Store these aliquots at −20°C.</li>
<li>Right before using it, add to a 10 mL aliquot one tablet of cOmplete Mini EDTA-free and RNase inhibitors to a final concentration of 160 U/mL to obtain the NIB buffer.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> β-mercaptoethanol causes severe eye irritation and may be fatal if absorbed through the skin. Always handle with gloves under the fume hood and discard according to safety guidelines.</div>
<h3 id="sec2.4">Lysis buffer</h3>
<p>Approximately 2 mL is used per sample. We recommend preparing 80 mL. Mix the following components:</p>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (100 mM, pH 7.5)</td>
<td>50 mM</td>
<td>40 mL</td>
</tr>
<tr>
<td>Na<sub>2</sub>EDTA (100 mM)</td>
<td>10 mM</td>
<td>8 mL</td>
</tr>
<tr>
<td>SDS (10% w/v)</td>
<td>1% (w/v)</td>
<td>8 mL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>N/A</td>
<td>24 mL</td>
</tr>
<tr>
<td>Total</td>
<td>N/A</td>
<td>80 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Filter and dispense into 10 mL aliquots.</li>
<li>Store it at 4°C.</li>
<li>Right before using it, add to 10 mL of Lysis Buffer: one tablet of cOmplete Mini EDTA-free and RNase inhibitors to a final concentration of 160 U/mL.</li>
</ul>
<div class="critical">
<span class="critical-title">Critical:</span> SDS causes skin and eye irritation. Always handle with gloves under the fume hood and discard according to safety guidelines.</div>
<h3 id="sec2.5">Nuclei dye stock solutions</h3>
<p>Prepare 4,6-diamidino-2-phenylindole (DAPI) stock solution: 0.1 mg/mL in deionized water (diH<sub>2</sub>0). Dispense into aliquots and store covered with tin aluminium foil at −20°C.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Propidium Iodide (PI) can also be used to stain the nuclei. PI stock solution 1 mg/mL must be prepared in diH<sub>2</sub>0. Dispense into aliquots and store covered with tin aluminium foil at −20°C.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>The protocol presented here provides a key approach to investigate subcellular RNA distribution. It is an assembly and adaptation of different existing methods optimized by introducing key control steps in order to obtain a highly pure nuclear (membrane-/chromatin-associated and nucleoplasmic) RNA from intact nuclei to study nuclear-cytoplasmic shuttling processes in plants (<a href="#bib8">Galbraith, 2014</a>; <a href="#bib12">Slane and Bayer, 2017</a>; <a href="#bib14">Weinhofer and Köhler, 2014</a>; <a href="#bib18">Zhang et al., 2008</a>). The procedure relies on the manual homogenization of fresh cross-linked tissue (using a razor blade), leading to the release of cellular content into a small volume of Nuclei Isolation Buffer. Filtering the crude homogenate through ∼35 μm filter removes large tissue debris, serves as an initial purification step, and avoids the blockage of the sample tubing and sorter nozzle. Crude nuclear preparations are stained with DAPI/PI or labeled using fluorescence proteins and flow-sorted (<a href="#bib13">Vrána et al., 2014</a>). RNA extraction followed by qPCR is used to validate nuclear RNA quality. Finally, the levels of particular RNA molecules in total and nuclear fractions are determine by qPCR/RNA sequencing (RNA-seq)(<a href="#bib1">Bologna et al., 2018</a>; <a href="#bib6">de Leone et al., 2020</a>) (<a href="#fig1">Figure 1</a>A).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/462-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation and isolation of plant nuclei by flow sorting</div>
<p>(A) Protocol work flow.</p>
<p>(B–E) Steps for sample preparation. (B) Cross-linking. (C) Samples before chopping. (D) Plant homogenate after chopping. (E) Samples after filtration with 35 μm filters.</p>
<p>(F and G) Microscope images during sample preparation. (F) DAPI stained nuclei before filtration. (G) DAPI stained nuclei after filtration. Top panels show DAPI fluorescence channel, bottom panels show bright light channel. White arrows indicate examples of small and large contaminations and/or debris present in the samples before sorting.</p>
<p>(H and I) Gating during FACS sorting. Top panels show dot plot of SSC-A versus DAPI fluorescence area. Populations of nuclei with different DAPI fluorescence intensities reflect the different levels of polyploidy. Bottom panel shows histogram of DAPI fluorescence of all particles. The gates show percentage of nuclei on total events before and after nuclear sorting. (H) Before sorting. (I) After sorting. See also <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Figure S1</a>A.</p>
<p>(J and K) Microscope images of sorted nuclei prior to and post-concentration. Top panels show the DAPI fluorescence channel, bottom panels show bright light channel. (J) Before nuclear concentration. The inset shows a 60× image of a purified nucleus where is possible to distinguish the nucleolus, depicted as an empty black hole (mark with a circular dotted line), and chromocenters showed as high-intensity dots inside the nucleus. (K) After nuclear concentration. Note that small and large contaminations and/or debris (thick white arrows) that were previously present in the sample (F and G) showed on the bright field, have been eliminated during the sorting (J and K).</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Use gloves and RNase-free material throughout the entire protocol.</div>
<h3 id="sec3.1">Tissue cross-linking: day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>This step helps preventing the loss of nucleoplasmic RNA prompted to diffuse out of the nucleus during nuclear isolation.</p>
<div class="critical">
<span class="critical-title">Critical:</span> All solutions should be ice-cold and all manipulations through to step 7 should be performed on ice to prevent sample degradation.</div>
<ol>
<li>Prepare fresh Cross-linking (CL) Buffer. Add 1.35 mL of 37% Formaldehyde to every 48.65 mL of freshly prepared Sample Buffer (final concentration: 1% (v/v)).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Formaldehyde is a corrosive and carcinogenic compound. Always handle with gloves under a fume hood and discard according to safety guidelines.</div>
<ol start="2">
<li>Place a chilled beaker containing ∼50 mL of CL Buffer on ice and add ∼100 seedlings including roots. Shake the beaker to allow as much contact as possible between the plant material and CL Buffer, avoiding air bubbles (<a href="#fig1">Figure 1</a>B). Keep the beaker on ice and transfer to a desiccator.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To extract nuclei from other plant tissues (e.g., leaves, inflorescences or isolated roots), maintain a CL Buffer/tissue ratio of ∼10:1.</div>
<ol start="3">
<li>Apply 600 atm vacuum for 7 min and then slowly release the vacuum. Shake the beaker and repeat this step one more time.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Check if seedlings have sunk from the interphase into the sine of the solution. The cross-linking starts only after CL buffer penetrates the tissue (<a href="#fig1">Figure 1</a>B). See <a href="#sec6.1">Problem 1</a> in the <a href="#troubleshooting">Troubleshooting</a> Section.</div>
<ol start="4">
<li>Add ∼7 mL of 1 M Glycine to obtain a final concentration of 0.125 M. Shake the beaker well and apply 600 atm vacuum for 7 min. This step stops the cross-linking.</li>
<li>Discard the content of the beaker and wash the seedlings twice with ∼15 mL of Sample Buffer.</li>
<li>Rapidly semi-dry the plant tissue in Whatman Paper. Separate 10% of the plant tissue in an Eppendorf tube, freeze it in liquid nitrogen and store it at −80°C. This will be the used later on as total fraction. See step 24 for further details.</li>
</ol>
<h3 id="sec3.2">Tissue homogenization</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min per sample</div>
<ol start="7">
<li>Transfer the remaining plant tissue (90%) from the Whatman Paper into a pre-cooled Petri dish. Add approximately 300 μL of NIB and chop it for ∼5 min with a sharp razor blade until getting a fine suspension (<a href="#fig1">Figure 1</a>C) (<a href="#mmc2">Methods Video S1</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the sample cold and monitor the blades of the homogenizer to ensure that fibrous debris does not accumulate and impairs homogenization. The quality of the blades is important, we used Feather double-sided 0.1 mm.</div>
<ol start="8">
<li>Depending on the amount of plant tissue, add an additional 300–700 μL of NIB and chop it for ∼5 more min (<a href="#fig1">Figure 1</a>D).</li>
<li>Filter the suspension through a 35 μm cell strainer caps into a 5 mL round-bottom polystyrene tube (<a href="#fig1">Figure 1</a>E).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Tilt the tube and pipet out gently the supernatant on the filter so that the drops run on the side of the tube and do not fall to the bottom. Repeating this step will help clarify the suspension. See <a href="#sec6.3">Problem 2</a> and <a href="#sec6.5">Problem 3</a> in the <a href="#troubleshooting">Troubleshooting</a> Section. Keep it on ice.</div>
<ol start="10">
<li>Stain the suspension by adding DAPI to a final concentration of 1–2 μg/mL and incubate it at 25°C for 5 min.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Nuclei can be stained with PI to a final concentration of 50 μg/mL.</div>
<div class="note">
<span class="note-title">Note:</span> The quantity of debris and/or contamination prior and after filtering the sample can be verified by observing an aliquot with DAPI/PI under the microscope (<a href="#fig1">Figures 1</a>F and 1G). Contamination can be spotted as particles on the bright field, which are not detected on the fluorescent channel (<a href="#fig1">Figures 1</a>F and 1G, upper and lower panels, white arrows). See step 22 for more details on how to set the microscope.</div>
<figure id="mmc2"><video controls="controls">
        <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/462-Mmc2.mp4" type="video/mp4">
        Your browser does not support HTML5 video.
      </source></video>
<figcaption>
<div class="figcaption-title">Methods Video S1. Sample homogenization using sharp razor blade, related to step 7</div>
</figcaption>
</figure>
<h3 id="sec3.3">Sorting plant nuclei using BD FACS AriaIII cell sorter</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 25 min per sample (plus 40–60 min sorter start-up time)</div>
<ol start="11">
<li>Switch on the cell sorter (including lasers) and the computer. Launch the BD FACS Diva Software. Perform fluidics start up, insert the 70 μm nozzle (70 psi), switch on the stream, and let it stabilize. Switch on the cooling water-bath. Set the temperature of the sample injection chamber to 4°C.</li>
<li>Run Cytometer Setup and Tracking research beads (CS&amp;T beads, BD Biosciences Cat. nr 655050) to check performance. Connect the 1.5 mL Eppendorf tube collection device to the cooling system; adjust the side streams so the sorted droplets fall in the middle of the collection tubes.</li>
<li>Run Accudrop Beads (BD Biosciences Cat. nr 345249) and determine the optimal drop delay according to BD FACS Aria III User’s Guide.</li>
<li>In the Settings, choose the appropriate channels to measure light scattering and fluorescence. Use the following laser excitation parameters and fluorescence detection filters: for DAPI: excitation at 405 nm (50 mW) and detection with a 450/40 Band Pass (BP) filter. Set up the trigger on DAPI fluorescence. For side scatter use 488 nm laser (20 mW or higher) and 488/10 BP.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> for PI use the following parameters. Excitation at 488 nm, 532 nm or 561 nm (50 mW, depending on laser line availability) and detection with a 620/10 nm BP filter. Set up the trigger on PI fluorescence.</div>
<ol start="15">
<li>Set up the experiment with 2 bivariate dot plots: log side scatter (SSC) A vs log DAPI-A, and log DAPI-A vs log DAPI-H; and one histogram: log DAPI-A.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Define the bivariate dot plots and histogram for PI, when using it to stain the nuclei, instead of DAPI.</div>
<ol start="16">
<li>Run the nuclei sample; select a proper threshold level that allows the display of the signals on an SSC-A plot versus DAPI/PI fluorescence area (both in log scale). Adjust the voltage of the photomultiplier for the DAPI/PI fluorescence to achieve a displaying of all ploidy level populations (<a href="#fig1">Figure 1</a>H-upper panel). Make sure the debris from preparation are also visible on this plot. Histogram of DAPI fluorescence area is shown on lower panel (<a href="#fig1">Figure 1</a>H).</li>
<li>Define a gate around the nuclei populations selected to be sorted. To exclude aggregates, we suggest also drawing additional diagonal gate on the DAPI-A vs DAPI-H plot (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Figure S1</a>A).</li>
<li>In the “Sort layout” choose the 4-way purity mode. Assign the sort location field to the collection tube. Run the sample at 1,000–3,000 events/s. Sort the required number of nuclei into 1.5 mL Sarstedt screw-cap tubes containing ∼100 μL of NIB and keep them on ice afterwards. For additional analysis export recorded data as FCS 3.0 file(s) and use FlowJo, LLC software.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The number of nuclei varies between 250,000 to 1,000,000; depending on the availability of the individual ploidy levels and the desired number of samples for RNA analysis. For example, for RT-qPCR or RNA sequencing analysis 250,000 nuclei should suffice.</div>
<ol start="19">
<li>Re-analyze sorted DAPI or PI-positive nuclei to check the sorting quality and verify sorting parameters (<a href="#fig1">Figure 1</a>I).</li>
</ol>
<h3 id="sec3.4">Microscopy, nuclei concentration, and reverse cross-linking</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60 min</div>
<p>We recommend following key steps of the protocol by confocal microscopy to check the presence of debris, prior and after sorting (<a href="#fig1">Figures 1</a>F, 1G, 1J, and 1K, steps 8–18), at least the first time the protocol is performed. Small and large contaminations and/or debris can be found in the samples before sorting (<a href="#fig1">Figures 1</a>F and 1G, white arrows bottom panels). The optimal outcome after sorting the samples is the lack of debris in the bright field when comparing the prior and post sorting (<a href="#fig1">Figures 1</a>F, 1G, 1J, and 1K).</p>
<ol start="20">
<li>Centrifuge 30 min 300 × <i>g</i> at 4°C, in a pre-cooled centrifuge.</li>
<li>Remove the supernatant leaving only ∼100 μL remaining. Re-suspend the nuclei in those 100 μL and place 2–5 μL to check nuclear integrity under the microscope. Use the bright field to look for small debris of contamination that could have passed through the sorter (<a href="#fig1">Figure 1</a>J).</li>
<li>Turn on the microscope, select the laser for DAPI or PI, and set the objective first to 20× and then to 40× for an overview of the nuclei. Once the nuclei are spotted, use the 60× objective to assess nuclear integrity in more detail.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The intact nuclei should look blue when stained with DAPI and red for PI. In both cases, a non-stained dark spot, the nucleolus, should be visible (<a href="#fig1">Figure 1</a>J, inset).</div>
<ol start="23">
<li>Add 200 μL of Lysis Buffer to the remaining re-suspended nuclei from step 21. The final volume will be ∼300 μL. Incubate on a spinning wheel 10 min at 4°C. Keep as nuclear fraction.</li>
<li>In parallel, during centrifugation time (step 20), process total fraction from step 6. Grind the sample in liquid Nitrogen with a mortar and pestle while adding 1 mL of Lysis Buffer and filter through a 35 μm mesh. Afterwards, incubate it on a spinning wheel for 10 min at 4°C. Take ∼300 μL of the total fraction for sonication.</li>
<li>Sonicate total and nuclear fractions using a Bioruptor. Use the following settings: Power: H position (High), Sonication cycle: 30 s ON, 30 s OFF, Total sonication time: five cycles on a refrigerated bath. The main purpose of the sonication step is not only to shear DNA but also ensure all nuclei are lysed after the Lysis Buffer incubation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pre-chilled Bioruptor before using it and keep the samples on ice and/or a refrigerated bath during sonication.</div>
<ol start="26">
<li>Add 12 μL of 5 M NaCl, to a final concentration of ∼200 mM, and 40 μg of proteinase K. First, incubate for 15 min at 50°C to allow digestion of cross-linked polypeptides by proteinase K, and then for 30 min at 65°C to reverse the formaldehyde cross-linking.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The protocol has been successfully applied to different plants species, as Rice, Sorghum, Tomato, Tobacco, Cucumber, Zucchini, and Melon, in addition to <i>Arabidopsis</i>. Samples have been chopped, filtered, DAPI labeled, the nuclei were gated and sorted by FACS. After sorting, isolated nuclei were analyzed by FACS and also microscopy confirming the quality of nuclear purification (<a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/462-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Nuclei isolation by flow sorting from the leaves of different plant species</div>
<p>Dot plot of SSC-A versus DAPI fluorescence area before and after sorting, and microscope images of sorted nuclei from different plant species. Right: <i>Arabidopsis thaliana</i>, <i>Cucumis sativus</i> (cucumber), <i>Cucumis melo</i> (LP T111, melon), and <i>Oryza sativa</i> (subsp. japonica, rice). Left: <i>Sorgum bicolor</i> (BTX623, sorgum), <i>Nicotiana bentamiana</i> (tobacco), <i>Solanum lycopersicum</i> (cv. Micro-Tom, tomato), and <i>Cucurbica pepo</i> (zucchini).</p>
</figcaption>
</figure>
<h3 id="sec3.5">RNA extraction and cDNA synthesis: days 1 and 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼<b>120 min</b>
</div>
<ol start="27">
<li>Add 300 μL of RotiAqua P/C/I (Phenol acid/Chloroform/Isoamilic) for RNA extraction.</li>
<li>Vortex strongly for 15 s, mix well.</li>
<li>Centrifuge at 12,000 × <i>g</i> for 15 min at 4°C</li>
<li>Transfer ∼250 μL of aqueous phase (upper phase) into a new 1.5 mL Eppendorf tube. Add 1 μL of glycogen , 30 μL of NaAc 3 M pH 5.2 and 750 μL Ethanol absolute. Mix by inverting and incubate at −80°C for at least 30 min or at −20°C for at least 8 h (recommended).</li>
<li>Centrifuge at 12,000 × <i>g</i> for 30 min at 4°C.</li>
<li>Discard the supernatant. The RNA should be seen as a white pellet.</li>
<li>Wash the pellet with ∼500 μL of 80% EtOH. Centrifuge at maximum speed for 5 min at 4°C.</li>
<li>Discard the supernatant. Repeat steps 33 and 34 once.</li>
<li>Dry the pellet for 5–10 min at 37°C. Re-suspend the pellet in 10 μL of ddH<sub>2</sub>O.</li>
<li>Measure RNA concentration using Qubit RNA HS Assay Kit. Nuclear RNA samples should have a concentration of ∼5–100 ng/μL, depending on the initial number of nuclei. See <a href="#sec6.7">Problem 4</a> in the <a href="#troubleshooting">Troubleshooting</a> Section.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> RNA integrity could be checked using Bioanalyzer High Sensitivity RNA Analysis (Agilent RNA 6000 Pico Kit-5067-1513).</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> RNA may be stored at −80°C.</div>
<ol start="37">
<li>Prepare DNase treatment by following the reaction.</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>3 μL</td>
</tr>
<tr>
<td>RNA (5–100 ng/μL)</td>
<td>5 μL</td>
</tr>
<tr>
<td>10× dsDNase buffer</td>
<td>1 μL</td>
</tr>
<tr>
<td>dsDNase</td>
<td>1 μL</td>
</tr>
<tr>
<td>Final volume:</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<ol start="38">
<li>Incubate 2 min at 37°C and place it on ice.</li>
<li>Take 1 μL of each DNase treated RNA to be used as a negative control for contaminations in the qPCR. See steps 43–44 for further details.</li>
<li>Add components listed to the same tube containing DNase-treated RNA to synthesize the cDNA.</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>5 μL</td>
</tr>
<tr>
<td>5× Reaction Mix</td>
<td>4 μL</td>
</tr>
<tr>
<td>Maxima Enzyme Mix</td>
<td>2 μL</td>
</tr>
<tr>
<td>Final volume:</td>
<td>20 μL</td>
</tr>
</tbody>
</table>
<ol start="41">
<li>Mix gently and spin down.</li>
<li>Use the cDNA program:</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Annealing</td>
<td>25°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Synthesis</td>
<td>50°C</td>
<td>15 min</td>
<td>1</td>
</tr>
<tr>
<td>Inactivation</td>
<td>85°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Cooling</td>
<td>4°C</td>
<td>2 min</td>
<td>1</td>
</tr>
</tbody>
</table>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> cDNA can be stored at −20°C.</div>
<h3 id="sec3.6">Nuclear RNA purification quality control step by qPCR: day 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 180 min</div>
<p>In order to analyze the quality of the nuclear purification, we have designed three specific pairs of oligos to amplify: <i>(i)</i> un-spliced nuclear-only mRNA of <i>ACT2</i> (primers UA-Fw and UA-Rv), <i>(ii)</i> spliced mRNA of <i>ACT2</i> (primers SA-Fw and SA-Rv) and <i>(iii)</i> total (un-spliced and spliced) mRNA of <i>ACT2</i> (primers TA-Fw and TA-Rv) (<a href="#fig3">Figure 3</a>A, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Table S1</a>). The purpose of these pairs of oligos, is to control the success of the nuclear enrichment. In the case of an efficient nuclear purification, the amount of un-spliced mRNA of <i>ACT2</i> (UA mRNA) in the nuclear fraction, should be higher than the amount of UA mRNA in the total fraction. Opposite to this, the amount of spliced mRNA of <i>ACT2</i> (SA mRNA) present in the total fraction, should surpass the amount of SA mRNA detected in the nuclear fraction. The relative accumulation levels will depend on several factors including the number of nuclei sorted, plants species, and plant tissue. We suggest a difference in the Ct of UA between the nuclear and total fraction of at least two cycles (See <a href="#fig3">Figure 3</a>B). Our experience showed that differences of four to five cycles showed better results. Additionally, a comparison of Ct values of the SA between the total and nuclear fraction higher than two will also allow discarding any possible cytoplasmic contaminations on nuclear fractions. Finally, the TA, will be used as reference gene for the Relative accumulation levels of mRNA measure by RT-qPCR in the nuclear and total fractions.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/462-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Molecular validation of nuclear RNA enrichment by RT-qPCR and RNA-seq</div>
<p>(A) Schematic representation of exon1, intron1, and exon2 of <i>ACT2</i>. Arrows indicate primers combination to amplify: un-spliced mRNA of <i>ACT2</i> (UA primers FW and RV), spliced mRNA of <i>ACT2</i> (SA primers FW and RV) and total (un-spliced and spliced) mRNA of <i>ACT2</i> (TA primers FW and RV).</p>
<p>(B) Threshold cycle of un-spliced and spliced mRNA of <i>ACT2</i> in total (T) and nuclear (N) fractions. Biological duplicates are shown as 1 and 2.</p>
<p>(C) Relative expression levels of mRNA in total and nuclear fractions of un-spliced <i>ACT2</i> (yellow) and <i>RHIP1</i> (purple) measured by RT-qPCR.</p>
<p>(D) Relative expression levels of <i>MIR159a</i> and <i>MIR167a</i> in total and nuclear fractions measured by RT-qPCR. Note that both miRNA precursors and un-spliced <i>ACT2</i> accumulate on the nuclear fraction, while the levels of <i>RHIP1</i> are similar in both fractions. (B–D) The errors bars show the standard deviation of the three technical replicates performed for each sample. For all RT-qPCR total <i>ACT2</i> (primers TA-Fw and TA-Rv) was used as reference RNA.</p>
<p>(E–G) RNA-seq analysis. (E) Nuclear and total fractions distribution of inter-genic (turquoise), introns (yellow) and exons (gray). Biological duplicates are shown as 1 and 2. Note the increase in detection of inter-genic and intronic regions in nuclear fractions. (F) Reads distribution in total (light purple) and nuclear (light green) fractions of exons corresponding to: protein coding, long-non-coding RNA (lnc), small nuclear (sn)/ small nucleolar (sno) RNA, and microRNA precursors (<i>MIRNA</i>). (G) Reads distribution in total and nuclear fractions of <i>MIR159b</i> and <i>MIR167b</i>. Note that over ~85% of the miRNA precursors reads are detected on the nuclear fraction.</p>
</figcaption>
</figure>
<ol start="43">
<li>Prepare a dilution of each cDNA (we recommend 1:4) to use as a template in the qPCR reaction.</li>
<li>Prepare a dilution of each DNase treated RNA (keeping same ratio used for cDNA synthesis) to use as DNAse treatment control.</li>
<li>Prepare the following qPCR reaction. Perform technical triplicates of each sample.</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>cDNA</td>
<td>4 μL</td>
</tr>
<tr>
<td>2× Mix</td>
<td>5 μL</td>
</tr>
<tr>
<td>Primer mix (50 μM)</td>
<td>0.1 μL</td>
</tr>
<tr>
<td>Nuclease-free H<sub>2</sub>O</td>
<td>0.9 μL</td>
</tr>
<tr>
<td>Final volume:</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<ol start="46">
<li>Set and run the qPCR program as shown below. See <a href="#sec6.9">Problem 5</a> and <a href="#sec6.11">Problem 6</a> in the <a href="#troubleshooting">Troubleshooting</a> Section.</li>
</ol>
<table id="undtbl8">
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="3">45</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>20 s + fluorescence measurement</td>
</tr>
<tr>
<td>Melting Curve</td>
<td>70°C to 95°C</td>
<td>increment 0.5°C + measure fluorescence</td>
<td>1</td>
</tr>
</tbody>
</table>
<ol start="47">
<li>Calculate relative accumulation levels by subtracting the average threshold cycle (Ct) values for the genes of interest from those of the housekeeping gene (to give ΔCt) and then calculating “ΔΔCt” (<a href="#fig3">Figures 3</a>B–3D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In the example depicted in <a href="#fig3">Figure 3</a>, we analyzed two biological replicates of wild-type (Col-0) plants total (T) and nuclear (N) fractions (1 and 2) by RT-qPCR. Both biological replicates showed a relative mRNA expression level of un-spliced <i>ACT2</i> of ∼100 times higher in the nuclear fraction than in total fractions (<a href="#fig3">Figure 3</a>C, left panel). Conversely, housekeeping control mRNA, <i>RHIP1</i> (<a href="#bib3">Czechowski et al., 2005</a>) showed an equal relative expression level in the nuclear and total fractions (<a href="#fig3">Figure 3</a>C, right panel). Finally, we decided to analyze relative accumulation level of <i>MIRNA</i> precursors (e.g., <i>MIR159a</i> and <i>MIR167a</i>). <i>MIRNA</i> precursors are processed in the nucleus, not being exported to the cytoplasm, this makes them a good example of nuclear-specific RNAs (<a href="#bib16">Yu et al., 2017</a>). These showed an enrichment on the nuclear fraction versus the total fraction (<a href="#fig3">Figure 3</a>D). These controls were used to verify the quality of the nuclear RNA purification.</div>
<div class="note">
<span class="note-title">Note:</span> In addition to be nuclear-specific RNAs, one of the advantages of using un-spliced <i>Actin</i>, or <i>MIRNA</i> precursors is that these genes were conserved during evolution and can be found in other plant species. Moreover, orthologs have been already described (<a href="#bib5">Chorostecki et al., 2017</a>) and/or they can be found through ortholog finders such as <a href="http://orthology.phylomedb.org/">MetaPhOrs</a> (<a href="#bib4">Chorostecki et al., 2020</a>), in case nuclear sorting is performed from other plant species different to <i>Arabidopsis</i>.</div>
<h3 id="sec3.7">RNA-seq: day 3</h3>
<p>RT-qPCRs can be a very effective approach to analyze the changes in a handful of genes but, when global analyses are desired, we recommend doing RNA-seq. After validating nuclear purification quality by qPCR, RNA libraries can be constructed from (i) total RNA, (ii) ribosomal-depleted RNA, and (iii) poly-A-enriched RNA. Alternatively, (iv) small RNA libraries can be constructed using the Illumina TruSeq sRNA kit. Additionally, other types of specific libraries, such as miRNA target detection or <i>MIRNA</i> processing intermediates, among others) can be constructed using nuclear sorted RNA (<a href="#bib11">Schapire et al., 2013</a>; <a href="#bib17">Zhai et al., 2013</a>).</p>
<ol start="48">
<li>After performing nuclear sorting, RNA libraries can be prepared according to TruSeq RNA Sample Preparation Guide (Illumina) as used in (<a href="#bib6">de Leone et al., 2020</a>). First, RNA is poly-A-enriched. This represents the case of lowest RNA input, but often the preferred method when constructing libraries for RNA-seq. cDNA is synthesized using random hexamers and multiplex indexed adapters ligated. Finally, an enrichment step of 12 PCR cycles is done prior to DNA sequencing.</li>
<li>Libraries are validated with the Agilent 2100 Bioanalyzer and the Agilent DNA1000 Kit (Agilent Technologies). The libraries are pooled and pair-end sequenced with an Illumina HiSeq1500, providing 100-bp single end reads.</li>
</ol>
<p>As an example, we analyzed RNA-seq results of total and nuclear fractions of two replicates of wild-type (Col-0) plants poly-A enriched (<a href="#fig3">Figures 3</a>E–3G) (<a href="#bib6">de Leone et al., 2020</a>). After sequencing, raw data processing and normalization of the reads, the quality of the nuclear purification was assessed. We quantify the presence of poly-A RNA molecules derived from exons, introns, and inter-genic regions in total and nuclear fractions. Exons represented ∼96.5% of the reads in the Total fraction and ∼89.0% of the reads in the nuclear fraction (<a href="#fig3">Figure 3</a>E). On the other hand, introns represented ∼2.7% of the reads in the total fraction and ∼8.7% in the nuclear fraction, consistent with the enrichment of nuclear RNA molecules (<a href="#fig3">Figure 3</a>E). Next, we analyzed the distribution of different RNA molecules in the total and nuclear fraction. While the same amount of protein-coding exon reads was found in the total and nuclear fractions, long-non-coding RNA (lnc), small nuclear (sn), nucleolar (sno) RNA, and <i>MIRNA</i> precursors were highly enriched in the nuclear fraction (82%–94% of the reads) (<a href="#fig3">Figure 3</a>F). Furthermore, ∼85%–90% of the reads of, previously RT-qPCR validated same <i>MIRNA</i> precursors families, <i>MIR159</i> and <i>MIR167,</i> were found in the nuclear fraction (<a href="#fig3">Figure 3</a>G). Throughout this analysis, we could verify the efficiency of the nuclear purification on a global scale as well as particular examples of well-known RNA molecules, such as snRNA, snoRNA, and <i>MIRNA</i> precursors.</p>
<h3 id="sec3.8">Cell-type-specific sorting: day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 180 min</div>
<p>The protocol described so far purifies the nucleus of every cell in the chopped plant tissue without distinction. However, the experimental design may occasionally require the analysis of subcellular RNA distribution of specific cell types. For this purpose, we use transgenic <i>Arabidopsis</i> lines expressing Nuclear Fluorescence Protein. In this protocol, we used a Nuclear Targeting Fusion (NTF), as an example. NTF consist on: the WPP domain of RanGAP1 (targeting the nuclear envelope) and GFP under the transcriptional control of the <i>SHR</i> promoter (<i>pSHR</i>:NTF), which is specifically expressed in the stele of <i>Arabidopsis</i> roots (<a href="#bib2">Brosnan et al., 2019</a>; <a href="#bib10">Helariutta et al., 2000</a>) (<a href="#fig4">Figures 4</a>A–4C). The translational fusion of WPP to GFP results in the GFP protein being inserted into the nuclear envelope (<a href="#fig4">Figure 4</a>B, white arrows). The advantage of this approach is that any plants expressing WPP-GFP or any other nuclear fluorescent tags can be used to purify nuclei by flow cytometry (<a href="#bib7">Deal and Henikoff, 2011</a>; <a href="#bib15">You et al., 2017</a>; <a href="#bib9">Gutzat and Scheid, 2020</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/462-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Cell-type-specific nuclear sorting</div>
<p>(A) Microscope image of <i>Arabidopsis</i> transgenic roots expressing <i>pSHR</i>:NTF (WPP:GFP:BLRP) stained with PI. PI and GFP channels are merged.</p>
<p>(B) Close up of <i>Arabidopsis</i> transgenic roots expressing <i>pSHR</i>:NTF (WPP:GFP:BLRP) stained with PI. White arrows indicate nuclei where the GFP is visually located on the nuclear envelope.</p>
<p>(C) Schematic representation of the expressed construct, <i>pSHR</i>:NTF.</p>
<p>(D) Plot of SSC-A versus DAPI fluorescence area from nuclear preparation of <i>pSHR</i>:NTF transgenic plants.</p>
<p>(E and F) Plot of GFP versus DAPI fluorescence area in control non-transgenic plants nuclei (E) and in <i>pSHR</i>:NTF transgenic plants nuclei (F). Note than non-transgenic plants nuclei could show some false positive GFP events (usually less than 2%) cause probably to some fiber leftover present on plants homogenate. See also <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Figure S1</a>B.</p>
<p>(G) Plots of GFP versus DAPI fluorescence on GFP-positive nuclei from <i>pSHR</i>:NTF transgenic <i>Arabidopsis</i> plants after sorting.</p>
<p>(H) Relative expression levels of mRNA in total and nuclear fractions of whole roots (DAPI staining defined) and cell-type-specific (GFP labeled) nuclear purifications measured by RT-qPCR. The mRNA levels of un-spliced nuclear <i>ACT2</i> (yellow) and <i>SHR</i> (blue) stele-specific transcription factor are shown in the upper and lower panels, respectively. The errors bars show the standard deviation of the three technical replicates performed for each sample. Total <i>ACT2</i> (primers TA-Fw and TA-Rv) was used as reference RNA.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Grow seedlings on vertical MS plates covered in nylon mesh with continuous white light (100 mmol photons m<sup>−2</sup> s<sup>−1</sup>) at 22°C. Use a Nº 10 scalpel blade to harvest 6-day-old roots by cutting several millimeters from the cotyledons of the seedlings. Cross-link, homogenize, and stain the samples as described on steps 1–10. To set up the flow sorter (BD FACS Aria III), follow steps 11–13.</div>
<ol start="50">
<li>Use the following laser excitation parameters and fluorescence detection filters. For DAPI: excitation at 405 nm (50 mW) and detection with 450/40 BP. Set the trigger on DAPI fluorescence. For GFP: excitation at 488 nm (20 mW or higher) and detection with a 530/30 BP.</li>
<li>Prepare the layout with two bivariate dot plots: log SSC-A vs log DAPI-A (<a href="#fig4">Figure 4</a>D) and log GFP-A versus log DAPI-A (<a href="#fig4">Figures 4</a>E and 4F).</li>
<li>Run the nuclei sample and select a proper threshold level that allows you to display the signals on a dot plot of SSC-A versus DAPI fluorescence area (both in log scale). Adjust the voltage of the photomultiplier for the DAPI fluorescence to achieve a displaying of all ploidy level populations.</li>
<li>Draw a gate around the nuclei populations (<a href="#fig4">Figure 4</a>D, black polygon). For GFP-positive nuclei sorting, draw the subsequent polygonal gate around GFP-positive events on the GFP-A vs DAPI-A plot (<a href="#fig4">Figure 4</a>F). To exclude aggregates, its optional to draw additional diagonal gate on the DAPI-A vs DAPI-H plot. (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Figure S1</a>B).</li>
<li>In the “Sort layout” choose the 4-way purity mode. Assign the sort location field for the collection tube. Sort the required number of nuclei into collection tubes with ∼100 μL of NIB and afterwards keep them on ice. For additional analysis export data as FCS 3.0 file(s) and use FlowJo, LLC software.</li>
<li>Re-analyze sorted GFP-positive nuclei to check purity and verify sorting conditions (<a href="#fig4">Figure 4</a>G).</li>
<li>Concentrate, reverse cross-link, extract, and quantify the RNA from the sorted samples (as was described before in steps 20-36).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> RNA may be stored at −80°C.</div>
<ol start="57">
<li>Perform DNase, cDNA synthesis, and qPCR (as was described before in steps 37-47).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For Nuclear RNA purification quality control step, we first used the same primers described in <a href="#fig3">Figure 3</a>A to quantify un-spliced nuclear mRNA of <i>ACT2</i> and the total mRNA of <i>ACT2</i> (See <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Table S1</a>). Initially, we analyzed the enrichment of nuclear <i>ACT2</i>. There was an enrichment of nuclear <i>ACT2</i> RNA levels in the nuclear fractions sorted either by DAPI (∼30 times) or GFP (∼20 times) when compared to the total fraction (<a href="#fig4">Figure 4</a>H, top panel). Then, to characterize the GFP-sorted nuclei, we designed a new set of primers to specifically amplify <i>SHR</i> mRNA (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/protocol-id-Mmc1.extension">Table S1</a>). Analysis of <i>SHR</i> mRNA levels showed a 5-fold enrichment on GFP-sorted nuclei in comparison to DAPI-sorted nuclei (<a href="#fig4">Figure 4</a>H, bottom panel). These results support the high effectiveness of nuclear sorting by GPF expression driven by cell type-specific promoters.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The above-described procedure provides a key approach to investigate subcellular RNA distribution using highly pure nuclei preparations from practically any plant tissue. The number of purified nuclei can vary between 100,000 and 1,000,000, yielding RNA samples of 5–100 ng/μL. The quality of the nuclear purification can be assessed by (i) re-analyzing the sorted DAPI/PI-positive nuclei by FACS to verify the sorting conditions (<a href="#fig1">Figures 1</a>I and <a href="#fig4">4</a>G), (ii) verifying the nuclear purity using confocal microscopy (<a href="#fig1">Figures 1</a>J and 1K) and (iii) quantifying the differential RNA accumulation by qPCR in order to determine nuclear enrichment (<a href="#fig3">Figures 3</a>B–3D and <a href="#fig4">4</a>H). The protocol can be performed in wild-type plants. Nonetheless it has also been used to compare the Nuclear/Total distribution of RNA molecules among different mutant backgrounds (<a href="#bib1">Bologna et al., 2018</a>; <a href="#bib6">de Leone et al., 2020</a>). In addition, the same approach can be used under different stress conditions, developmental stages, or cell cycle phases. Molecules obtained from purified nuclei are valuable not only to study subcellular distributions, but also understand nuclear processes such as DNA methylation, splicing events, nuclear RNA quality control, long non-coding RNA, or epigenetic and chromatin organization, among others. Finally, cell-specific nuclear sorting by GFP allows the study of the RNA subcellular distribution in specific cell types, thus enabling the elucidation of local processes and cell-to-cell movement.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>One theoretical drawback of the presented protocol is that the chopping method used to disintegrate the tissue and release the nuclei makes it difficult to automatize the sample preparation for high-throughput applications. Furthermore, samples cannot be stored prior to processing. Both caveats are, however, surmountable by meticulous experimental planning and outweighed by the quality of the final nuclear preparation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>During cross-linking step, seedling did not sink from the interphase into the sine of the solution.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Check and remove possible air bubbles located surrounding seedlings before apply vacuum. Evaluate CL Buffer/tissue ratio. Use bigger beakers.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>During the filtration of tissue suspension, chunks of tissue block the filter.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Perform more strokes with razor blade during a longer period of time. Monitor the blades of the homogenizer to ensure that fibrous debris does not accumulate and impairs homogenization. See <a href="#mmc2">Methods Video S1</a> where the chopping is being performed.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Nozzle clogs during sorting.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Nucleus and cell suspensions from plant tissue tend to contain numerous debris. If clogged, sonicate the nozzle and dilute and filter one more time the sample. Also, if needed, we recommend repeating the filtration step and diluting more the samples.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The quantity of nuclear RNA is low after RNA extraction.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Increase starting material and/or sort a higher number of nuclei. Check sorting quality under confocal microscope. Make sure RNA was not lost during RNA extraction by checking the presence of the pellet after precipitation.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>After the reverse transcription, the cDNA quality is poor.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The poor cDNA quality can be due to RNA degradation or contamination with phenol during sample preparation. To prevent RNA degradation, make sure to have added the RNase inhibitor to the buffers where is indicated and handle samples on ice at all times. Alternatively, if solutions have been prepared in house, they can be re-made using DEPC treated water or Molecular Biology grade commercial solutions can be purchased. Phenol can interfere with the cDNA synthesis. To prevent RNA contamination with phenol during RNA extraction, be particularly careful when taking the aqueous phase in step 30, not to drag the organic lower phase where phenol is. Also, repeat washing step with 80% EtOH.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>The volume of the different wells is inconsistent after PCR.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Make sure that the adhesive foil is tightly attached to the plate, paying special attention to the wells that are close to the edges of the plate.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bologna, N.G., Iselin, R., Abriata, L.A., Sarazin, A., Pumplin, N., Jay, F., Grentzinger, T., Dal Peraro, M., and Voinnet, O. (2018). Nucleo-cytosolic shuttling of ARGONAUTE1 prompts a revised model of the plant microRNA pathway. Mol. Cell <i>69</i>, 709-719.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref1">View at publisher</a></p>
<p id="bib2">Brosnan, C.A., Sarazin, A., Lim, P., Bologna, N.G., Hirsch-Hoffmann, M., and Voinnet, O. (2019). Genome-scale, single-cell-type resolution of micro RNA activities within a whole plant organ. EMBO J. <i>38</i>, e100754. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref2">View at publisher</a></p>
<p id="bib3">Czechowski, T., Stitt, M., Altmann, T., Udvardi, M.K., and Scheible, W.R. (2005). Genome-wide identification and testing of superior reference genes for transcript normalization in <i>Arabidopsis</i>. Plant Physiol. <i>139</i>, 5-17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref3">View at publisher</a></p>
<p id="bib4">Chorostecki, U., Molina, M., Pryszcz, L.P., and Gabaldón Estevan, J.A. (2020). MetaPhOrs 2.0: integrative, phylogeny-based inference of orthology and paralogy across the tree of life. Nucleic Acids Res. <i>48</i>, W553-W557. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref4">View at publisher</a></p>
<p id="bib5">Chorostecki, U., Moro, B., Rojas, A.M.L., Debernardi, J.M., Schapire, A.L., Notredame, C., and Palatnik, J.F. (2017). Evolutionary footprints reveal insights into plant MicroRNA biogenesis. Plant Cell <i>29</i>, 1248-1261. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref5">View at publisher</a></p>
<p id="bib6">de Leone, M.J., Hernando, C.E., Romanowski, A., Careno, D.A., Soverna, A.F., Sun, H., Bologna, N.G., Vázquez, M., Schneeberger, K., and Yanovsky, M.J. (2020). Bacterial infection disrupts clock gene expression to attenuate immune responses. Curr. Biol. <i>30</i>, 1740-1747.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref6">View at publisher</a></p>
<p id="bib7">Deal, R.B. and Henikoff, S. (2011). The INTACT method for cell type-specific gene expression and chromatin profiling in <i>Arabidopsis thaliana</i>. Nat. Protoc. <i>6</i>, 56-68. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref7">View at publisher</a></p>
<p id="bib8">Galbraith, D.W. (2014). Flow cytometry and sorting in <i>Arabidopsis</i>. In Arabidopsis Protocols (Springer), pp. 509-537. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref8">View at publisher</a></p>
<p id="bib9">Gutzat, R. and Scheid, O.M. (2020). Preparing chromatin and RNA from rare cell types with fluorescence-activated nuclear sorting (FANS). In Plant Epigenetics and Epigenomics (Springer), pp. 95-105. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref9">View at publisher</a></p>
<p id="bib10">Helariutta, Y., Fukaki, H., Wysocka-Diller, J., Nakajima, K., Jung, J., Sena, G., Hauser, M.T., and Benfey, P.N. (2000). The SHORT-ROOT gene controls radial patterning of the <i>Arabidopsis</i> root through radial signaling. Cell <i>101</i>, 555-567. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref10">View at publisher</a></p>
<p id="bib11">Schapire, A.L., Bologna, N.G., Moro, B., Zhai, J., Meyers, B.C., and Palatnik, J.F. (2013). Construction of Specific Parallel Amplification of RNA Ends (SPARE) libraries for the systematic identification of plant microRNA processing intermediates. Methods <i>64</i>, 283-291. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref11">View at publisher</a></p>
<p id="bib12">Slane, D. and Bayer, M. (2017). Cell type-specific gene expression profiling using fluorescence-activated nuclear sorting. In Plant Gene Regulatory Networks (Springer), pp. 27-35. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref12">View at publisher</a></p>
<p id="bib13">Vrána, J., Cápal, P., Bednářová, M., and Doležel, J. (2014). Flow cytometry in plant research: a success story. In Applied Plant Cell Biology (Springer), pp. 395-430. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref13">View at publisher</a></p>
<p id="bib14">Weinhofer, I. and Köhler, C. (2014). Endosperm-specific chromatin profiling by fluorescence-activated nuclei sorting and ChIP-on-chip. In Plant Epigenetics and Epigenomics (Springer), pp. 105-115. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref14">View at publisher</a></p>
<p id="bib15">You, Y., Sawikowska, A., Neumann, M., Pose, D., Capovilla, G., Langenecker, T., Neher, R.A., Krajewski, P., and Schmid, M. (2017). Temporal dynamics of gene expression and histone marks at the <i>Arabidopsis</i> shoot meristem during flowering. Nat. Commun. <i>8</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref15">View at publisher</a></p>
<p id="bib16">Yu, Y., Jia, T., and Chen, X. (2017). The ‘how’ and ‘where’ of plant microRNAs. New Phytol. <i>216</i>, 1002-1017. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref16">View at publisher</a></p>
<p id="bib17">Zhai, J., Arikit, S., Simon, S.A., Kingham, B.F., and Meyers, B.C. (2013). Rapid construction of parallel analysis of RNA end (PARE) libraries for Illumina sequencing. Methods <i>67</i>, 84-90. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref17">View at publisher</a></p>
<p id="bib18">Zhang, C., Barthelson, R.A., Lambert, G.M., and Galbraith, D.W. (2008). Global characterization of cell-specific gene expression through fluorescence-activated sorting of nuclei. Plant Physiol. <i>147</i>, 30-40. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00027-7/sref18">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Nicolás. G Bologna (<a href="mailto:nicolas.bologna@cragenomica.es">nicolas.bologna@cragenomica.es</a>).</p>
<h4>Materials Availability</h4>
<p>All material is available from the authors upon request.</p>
<h4>Data and Code Availability</h4>
<p>All data are available from the authors upon request.</p>
<h3>Acknowledgments</h3>
<p>We thank A. Schuetz at ETH Zurich Flow Cytomtery Core Facility for helping with the nuclei sorting; V.M. González at the CRAG Institute for the bioinformatic analysis of the RNA-seq data; M.J. Yanovsky Laboratory at Instituto de Investigaciones Bioquímicas de Buenos Aires for sharing their data; Researchers at CRAG Institute for providing different plant species material; cytometry and cell sorting facility-Hospital Clínic Barcelona for helping with the nuclear sorting of different plant species; and J. Moreno and M.F. Ercoli for comments on the manuscript. B.M. has received funding by a postdoctoral fellowship from the “Severo Ochoa Programme” (SEV-2015-0533) and a postdoctoral fellowship “Juan de la Cierva-Formación” (Fellowship reference: FJC2018-036091-I) from the Ministerio de Ciencia e Innovación/<a href="https://doi.org/10.13039/501100011033">Agencia Estatal de Investigación</a>. A.B. has a predoctoral fellowship from <a href="https://doi.org/10.13039/501100004837">Ministerio de Ciencia e Innovación</a> y el Fondo Social Europeo (grant reference: PRE2019-089555). N.G.B. is supported by FEDER/<a href="https://doi.org/10.13039/100014440">Ministerio de Ciencia, Innovación y Universidades</a> (MCIU) – Agencia Estatal de Investigación (AEI)-Project Reference (PGC2018-101075-A-I00) and from <a href="https://doi.org/10.13039/100010434">“la Caixa” Foundation</a> fellowship (ID 100010434), code number LCF/BQ/PI19/11690007. The N.G.B. laboratory also acknowledges the support from the CERCA Programme/Generalitat de Catalunya and from Ministerio De Ciencia e Innovación/Agencia Estatal  de Investigación. (AEI), through the “Severo Ochoa Programme for Centres of Excellence in R&amp;D” SEV-2015-0533 and CEX2019-000902-S.</p>
<h3>Author Contributions</h3>
<p>B.M. and V.B.B. performed experiments and statistical analysis in <i>Arabidopsis</i>. M.K. performed samples sorting. M.K. and N.G.B. optimized the sorting settings and sorting protocol. A.B. performed the nuclear sorting from rice, sorghum, tomato, tobacco, cucumber, and melon. C.A.B. generated <i>pSHR</i>:NTF reporter lines. C.A.B. and N.G.B. designed the cell-type-specific sorting approach. B.M. and N.G.B. analyzed results. B.M., M.K., and N.G.B. wrote the paper. N.B. supervised the project. All authors discussed the results and commented on the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>