<article>
<section>
<h1 id="header">SARS-CoV-2 proteome microarray for COVID-19 patient sera profiling</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Dan-yun Lai,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a></sup> He-wei Jiang,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Yang Li,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Hai-nan Zhang,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> and Sheng-ce Tao<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Shanghai Center for Systems Biomedicine, Key Laboratory of Systems Biomedicine (Ministry of Education), Shanghai Jiao Tong University, Shanghai, China</p>
<p id="aff2"><sup>2</sup>College of Life Science, Nankai University, Tianjin 300071, China</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:taosc@sjtu.edu.cn">taosc@sjtu.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101238">10.1016/j.xpro.2022.101238</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The immunogenicity of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) proteome is largely unknown. Here we describe a protocol for analyzing sera samples with SARS-CoV-2 proteome microarray. The proteins were expressed by either <i>E. coli</i> expression system or eukaryotic cell expression systems and obtained by affinity purification. The protocol includes microarray fabricating and sera profiling, which will be used to build an antibody response landscape for IgG and IgM. The protocol may help to facilitate a deeper understanding of immunity related to SARS-CoV-2.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00118-6/sref6">Li et al. (2021c)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Fabrication of the SARS-CoV-2 proteome microarray</li>
<li>Detailed procedure for sera profiling using SARS-CoV-2 proteome microarray</li>
<li>Description of the antibody responses elicited by nonstructural/accessory proteins</li>
<li>Construction of an antibody response landscape against SARS-CoV-2 proteome</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1499-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication (<a href="#bib3">Li et al., 2020</a>, <a href="#bib4">2021a</a>, <a href="#bib5">2021b</a>, <a href="#bib6">2021c</a>) in which we have profiled structural/non-structural proteins of SARS-CoV-2 using a plethora of sera from COVID-19 patients and control samples. COVID-19 patients and control samples used for this assay were collected from fully consented individuals under a protocol approved by the Institutional Ethics Review Committee of Foshan Fourth Hospital, Foshan, China (202005) and Tongji Hospital, Tongji Medical College, Huazhong University of Science and Technology, Wuhan, China (ITJ-C20200128). Use of this protocol may require specific institutional approval.</p>
<h3 id="sec1.1">Expression vector construction</h3>
<p>To get the protein library of SARS-CoV-2, we downloaded protein sequences of SARS-CoV-2 from GenBank.</p>
<ol>
<li>Determine the protein sequences based on a reference sequence from GenBank:MN908947.3.</li>
<li>Construction of expression vectors. Codon optimize the original sequences of the proteins according to the codon usage of <i>E. coli</i> and synthesized by Sangon Biotech (Shanghai, China), cloned into vector pET32a or pGEX-4T-1.</li>
<li>Overexpress and induce the proteins with 0.2 mM isopropyl-β-d-thiogalactoside (IPTG) .</li>
<li>Purify the supernatants with Ni<sup>2+</sup> Sepharose beads.</li>
<li>Verify the purified proteins by Western blotting with an anti-6×His antibody and Coomassie staining.</li>
</ol>
<h3 id="sec1.2">Microarray layout design and positive control preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>For microarray layout design, the layout is first designed, i.e., the proteins are individually assigned to designated positions on the microarray with three replications, all the proteins including positive and negative controls are printed on the microarrays. Positive and negative controls are applied to assure the results of the microarray experiment. The microarray printing program is set based on the microarray layout design.</p>
<ol start="6">
<li>Preparation of positive and negative controls. Positive control: Anti-Human IgG (0.1 mg/mL), Anti-Human IgM (0.1 mg/mL), and landmarkers (Fluorescent labeled protein for indicating array position): Cy3 labeled Anti-Human IgG antibodies (0.1 mg/mL), Cy5 labeled Anti-Human IgM antibodies (0.1 mg/mL). Negative/blank controls: Printing buffer.</li>
<li>Design the layout of microarray.
<ol type="a">
<li>A 75 mm × 25 mm slide can be fitted with 14 identical subarrays, each subarray contains 64 probes (SARS-CoV-2 proteins, positive and negative controls).</li>
<li>Design the array layout using the software of the Super Marathon microarray printer —— Command Center<sup>TM</sup> for Marathon v2.2.</li>
<li>There are 3 spots for each sample in a subarray.</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Positive sera sample preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>One positive control should be tested at the same time with other sera for each slide, and applied for normalization among slides. The purpose of including the positive control is to minimize the differences between different slides.</p>
<ol start="8">
<li>Serum samples were collected from convalescent COVID-19 patients from Foshan Fourth People’s Hospital. These COVID-19 patients were hospitalized and received treatment during the period from 2020-1-25 to 2020-2-27.</li>
<li>Centrifuge at 12,000 xg under 4°C for 20 min to remove the precipitants. Prepare positive control by mixing equal amount (10 μL) of serum from each of the sample. Thoroughly mix the positive control, aliquote to 10 μL and store at −80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Thoroughly mixing the individual sample can avoid bias.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th colspan="2">SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="4">Antibodies</td>
</tr>
<tr>
<td>Cy3-Goat Anti-Human IgG(@1:1000)</td>
<td colspan="2">Jackson ImmunoResearch</td>
<td>Cat# 109-165-008</td>
</tr>
<tr>
<td>Alexa Fluor 647 Donkey Anti-Human IgM(@1:1000)</td>
<td colspan="2">Jackson ImmunoResearch</td>
<td>Cat# 709-605-073</td>
</tr>
<tr>
<td>Anti-Human IgG(@1:1000)</td>
<td colspan="2">Sigma-Aldrich</td>
<td>Cat# I2136</td>
</tr>
<tr>
<td>Anti-Human IgM(@1:1000)</td>
<td colspan="2">Sigma-Aldrich</td>
<td>Cat# I2386</td>
</tr>
<tr>
<td>Cy3-Goat Anti-Mouse IgG(@1:1000)</td>
<td colspan="2">Jackson ImmunoResearch</td>
<td>Cat# 115-165-071</td>
</tr>
<tr>
<td>Cy5-Goat Anti-Mouse IgM(@1:1000)</td>
<td colspan="2">Jackson ImmunoResearch</td>
<td>Cat# 115-175-075</td>
</tr>
<tr>
<td>Anti-His antibody(@1:2000)</td>
<td colspan="2">Merck Millipore</td>
<td>Cat#05-949</td>
</tr>
<tr>
<td colspan="4">Chemicals, recombinant proteins</td>
</tr>
<tr>
<td>BSA (DNase, Protease, IgG free)</td>
<td colspan="2">Yeasen Biotech</td>
<td>Cat# 36103ES</td>
</tr>
<tr>
<td>SARS-CoV-2 Spike S1</td>
<td colspan="2">Hangzhou Bioeast biotech. Co.,Ltd.</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 Spike S1</td>
<td colspan="2">Sanyou biopharmaceuticals Co.,Ltd.</td>
<td>Cat# PNA002</td>
</tr>
<tr>
<td>SARS-CoV-2 E protein</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>Cat#PROTN_nCoVEHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 RBD</td>
<td colspan="2">Sanyou biopharmaceuticals Co.,Ltd.</td>
<td>Cat# PNA004</td>
</tr>
<tr>
<td>SARS-CoV-2 N protein</td>
<td colspan="2">Sanyou biopharmaceuticals Co.,Ltd.</td>
<td>Cat# PNA005</td>
</tr>
<tr>
<td>SARS-CoV-2 Protein N-Cter</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVN-CterHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 Protein N-Nter</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVN-NterHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 E protein</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVEHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP1</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP2</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVNSP2HG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP4</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP5</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoV3ClpHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP7</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP8</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP9</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP10</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 RdRp</td>
<td colspan="2">A gift from H. Eric Xu, ShanghaiInstitute of Materia Medica</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP14</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVNSP14HG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 NSP15</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVNdUHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2NSP16</td>
<td colspan="2">Healthcode Co., Ltd.</td>
<td>PROTN_nCoVOMTHG01000</td>
</tr>
<tr>
<td>SARS-CoV-2 ORF-3a</td>
<td colspan="2">In house (<a href="#bib6">Li et al., 2021c</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 ORF-3b</td>
<td colspan="2">In house (<a href="#bib6">Li et al., 2021c</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 ORF6</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 ORF-7b</td>
<td colspan="2">In house (<a href="#bib6">Li et al., 2021c</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SARS-CoV-2 ORF-9b</td>
<td colspan="2">In house (<a href="#bib2">Jiang et al., 2020</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Ni2+ Sepharose beads</td>
<td colspan="2">Senhui Microsphere Technology</td>
<td>Cat# 11-0010</td>
</tr>
<tr>
<td>GST Sepharose beads</td>
<td colspan="2">Senhui Microsphere Technology</td>
<td>Cat# 12-0010</td>
</tr>
<tr>
<td colspan="4">Software and algorithms</td>
</tr>
<tr>
<td>Marathon V 2.2</td>
<td colspan="2">Arrayjet</td>
<td><a href="https://www.arrayjet.co.uk/instruments/printers-in-detail">https://www.arrayjet.co.uk/instruments/printers-in-detail</a></td>
</tr>
<tr>
<td>R</td>
<td colspan="2"> </td>
<td>N/A</td>
</tr>
<tr>
<td colspan="4">Other</td>
</tr>
<tr>
<td>High-pressure cell cracker</td>
<td colspan="2">Union-Biotech (Shanghai) Co.,Ltd.</td>
<td>Cat# UH06</td>
</tr>
<tr>
<td>Super Marathon Microarrayer</td>
<td colspan="2">Arrayjet</td>
<td>Other microarray printers</td>
</tr>
<tr>
<td>LuxScan 10K-A Scanner</td>
<td colspan="2">CapitalBio</td>
<td rowspan="2">Other microarray scanners with 532 nm and 635 nm laser tube</td>
</tr>
<tr>
<td>GenePix 4200A Scanner</td>
<td colspan="2">Molecular Devices</td>
</tr>
<tr>
<td>SlideWasher™ 8</td>
<td colspan="2">CapitalBio</td>
<td>A normal staining rack with a compatible centrifuge</td>
</tr>
<tr>
<td>16-well chamber (with gasket)</td>
<td>Raybiotech</td>
<td>AA-FRAME-G16</td>
<td>Other similar products</td>
</tr>
<tr>
<td>Slide Container</td>
<td>In house</td>
<td>This paper</td>
<td>A lid of the rack for 10 μL tips</td>
</tr>
<tr>
<td>PATH substrate slide</td>
<td>Grace Bio-labs</td>
<td>805025</td>
<td>Others suitable for protein array</td>
</tr>
<tr>
<td>384 well plates</td>
<td>Arrayjet</td>
<td>AJC010</td>
<td>Other 384-well plates</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>50 mM</td>
<td>0.6 gms</td>
</tr>
<tr>
<td>NaCl</td>
<td>500 mM</td>
<td>2.922 gms</td>
</tr>
<tr>
<td>imidazole</td>
<td>20 mM</td>
<td>68 mg</td>
</tr>
<tr>
<td>PMSF</td>
<td>0.5 mM</td>
<td>8.7 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>NA</td>
<td>100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C for one months.</div>
<div class="note">
<span class="note-title">Note:</span> Initially, add all the regents to 90 mL of ddH<sub>2</sub>O, maintain the pH 8, and make the volume 100 mL, add PMSF before beginning the experiment.</div>
<table id="undtbl2">
<caption>Elution buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>50 mM</td>
<td>1.514 gms</td>
</tr>
<tr>
<td>NaCl</td>
<td>500 mM</td>
<td>7.305 gms</td>
</tr>
<tr>
<td>imidazole</td>
<td>300 mM</td>
<td>5.1 gms</td>
</tr>
<tr>
<td>Glycerol</td>
<td>10%</td>
<td>25</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>NA</td>
<td>225 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C for one months.</div>
<div class="note">
<span class="note-title">Note:</span> Initially, add all the regents to 200 mL of ddH<sub>2</sub>O, maintain the pH 8, and make the volume 225 mL, finally add the glycerol.</div>
<table id="undtbl3">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>0.3%</td>
<td>0.9 gms</td>
</tr>
<tr>
<td>PBS 10× pH 7.4</td>
<td>1×</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>NA</td>
<td>90 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution should be freshly prepared before use.</div>
<table id="undtbl4">
<caption>Incubation buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>0.1%</td>
<td>0.1 gms</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>PBS 10× pH 7.4</td>
<td>1×</td>
<td>10 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>NA</td>
<td>90 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution should be freshly prepared before use.</div>
<table id="undtbl5">
<caption>Wash buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS 10× pH 7.4</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>NA</td>
<td>899 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution can be stored at 4°C for at least 6 months.</div>
<table id="undtbl6">
<caption>Printing buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS 10× pH 7.4</td>
<td>1×</td>
<td>100 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>25%</td>
<td>250 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>NA</td>
<td>650 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This solution can be stored at 4°C for at least 6 months.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of proteins</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7 days</div>
<p>Besides the <i>E. coli</i> expression system, to cover the proteome of SARS-CoV-2 as complete as possible, and to obtain proteins with post-translational modifications (PTM), especially glycosylation, recombinant SARS-CoV-2 proteins were also produced using yeast cell-free systems or mammalian cell expression systems from a variety of commercial sources. Here we describe the affinity purification procedure (<a href="#key-resources-table">key resources table</a>).</p>
<ol>
<li>The genes synthesized by Sangon Biotech (Shanghai, China) were cloned into pET32a or pGEX-4T-1 vector and transformed into <i>E. coli</i> BL21 strain. Express the recombinant proteins in <i>E. coli</i> BL21 by growing cells in 200 mL LB medium to an A<sub>600</sub> of 0.6 at 37°C.</li>
<li>Induce protein expression. Add 0.2 mM isopropyl-β-d-thiogalactoside (IPTG) before incubating cells for 16 h at 16°C.</li>
<li>Prepare and re-suspend the cell pellets in lysis buffer, and lyse by a high-pressure cell cracker.</li>
<li>Centrifuge the Cell lysates at 12,000 × <i>g</i> for 20 min at 4°C.</li>
<li>Affinity purification. Purify the supernatants with Ni<sup>2+</sup> Sepharose beads, then wash with lysis buffer and elute with buffer containing 10% glycerol, 50 mM Tris-HCl pH8.0, 500 mM NaCl and 300 mM imidazole.</li>
<li>Verify the purified proteins by SDS-PAGE followed by western blotting using an anti-His antibody and Coomassie brilliant blue staining (<a href="#fig1">Figure 1</a>A).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1499-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Expected outcomes</div>
<p>(A) The SARS-CoV-2 proteins included in this proteome microarray. The upper panel is western blotting with an anti-6×His antibody. The lower panel is Coomassie brilliant blue staining. These proteins were prepared and collected from different sources (<a href="#bib2">Jiang et al., 2020</a>).</p>
<p>(B) The image of the SARS-CoV-2 proteome microarray, which was incubated with Anti-6×His antibody followed by Cy5 labeled secondary antibody.</p>
<p>(C) A image of serum sample profiling with the microarray same as in (B). Green, the 532 nm channel, IgG response; Red, the 635 nm channel, IgM response.</p>
<p>(D) The correlations of the overall IgG and IgM signal intensities between two repeats probed with the same serum.</p>
<p>(E) Construction of an antibody response landscape against SARS-CoV-2 proteome (<a href="#bib6">Li et al., 2021c</a>).</p>
<p>(F) Boxplots of IgG response for S1 and S2 proteins. The proteins labeled with bold and red were overexpressed in mammalian cell lines.</p>
<p>(G) Two examples of microarray scanning results. The right panel shows a good example which has a clear background and the signals are not over-exposed or too weak. In contrast, the left panel shows a bad example.</p>
<p>(H) The 16-well chamber.</p>
</figcaption>
</figure>
</li>
<li>Some of recombinant SARS-CoV-2 protein were also collected from commercial sources (please see the <a href="#key-resources-table">key resources table</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Please refer to <a href="#bib2">Jiang et al. (2020)</a> for more details.</div>
<h3 id="sec3.2">Protein microarray fabrication</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>High quality microarray is essential to ensure the final results. The detailed process of fabricating the microarray is as follows.</p>
<ol start="8">
<li>Transfer the proteins and controls to a 384-well plate.
<ol type="a">
<li>Dilute the proteins and controls with Printing buffer (1× PBS with 25% glycerol) to a final concentration of 0.3 mg/mL.</li>
<li>Add the proteins into the predetermined wells of a 384-well plate. For the SARS-CoV-2 proteome, the layout of the array was set as 32 × 6 features, each protein was repeated for 3 spots (<a href="#fig1">Figure 1</a>B). The appropriate volume range for each well is 10 μL–50 μL.
<div class="note">
<span class="note-title">Note:</span> the volume range varies among different types of 384-well plates.</div>
</li>
<li>After adding the protein samples, centrifuge the 384-well plate at 800 × <i>g</i> for 1 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> the plate with protein samples could be stored at −80°C for at least one year.</div>
</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Protein printing on microarray slide.
<ol type="a">
<li>Use the printer (Super Marathon) to print the proteins on to the PATH substrate slides according to the manufacturer’s instruction.
<div class="note">
<span class="note-title">Note:</span> The website of the printer: <a href="https://www.arrayjet.co.uk/instruments/printers-in-detail">https://www.arrayjet.co.uk/instruments/printers-in-detail</a>.</div>
</li>
<li>Incubate the microarrays at 4°C with humidity less than 50% overnight (over 12 h) for complete immobilization of proteins on the microarray.</li>
<li>Put the slides into the dedicated slide box (5 or 1 slide(s) per box), seal the box with Parafilm and store the box at −80°C.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>GAL (GenePix Array List) file generation.
<ol type="a">
<li>Create an Excel file and mark the position of samples on the 384-well plate.
<table id="undtbl7">
<thead>
<tr>
<th>Plate</th>
<th>Column</th>
<th>Row</th>
<th>Name</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>1</td>
<td>1</td>
<td>P1</td>
</tr>
<tr>
<td>1</td>
<td>1</td>
<td>2</td>
<td>P2</td>
</tr>
<tr>
<td>1</td>
<td>1</td>
<td>3</td>
<td>BSA</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
</li>
<li>Create GAL file using the printer software (Marathon V 2.2).</li>
</ol>
</li>
<li>Microarray quality evaluation. The purpose of this step is to detect whether all proteins are printed on the slides and whether the immobilized proteins are still active, so as to ensure that the subsequent sera profiling can be carried out.
<ol type="a">
<li>Blocking.
<ol type="i">
<li>Take out the microarrays from −80°C and leave them at 4°C for 30 min, then transfer them to room temperature for 15 min to enable temperature equilibrium.
<div class="critical">
<span class="critical-title">Critical:</span> for PATH substrates, pay attention to sealing in the process of rewarming, avoid condensation on the slide surface.</div>
</li>
<li>Prepare the blocking buffer (1× PBS with 3% BSA), add 30 mL blocking buffer to a container and kept at 25°C.</li>
<li>Face down the microarray and immerse the barcode side into blocking buffer, while keep the other side out, then slowly lay down the microarray until it is completely immersed into the buffer. Gently shake the microarray to make sure the bubbles on the microarray surface are driven out, turn over the microarray and immerse it into buffer.
<div class="note">
<span class="note-title">Note:</span> the whole process of this step should be slow and take at least 1.5 min to avoid trailing effect on the microarray.</div>
</li>
<li>Shake the microarrays at 20–30 rpm/min on a shaker for 3 h at 25°C.</li>
</ol>
</li>
<li>Wash and dry the microarrays.
<ol type="i">
<li>Discard the blocking buffer. Add 20–30 mL 1× PBS into the container, keep shaking (20–30 rpm) for 5 min at 25°C.</li>
<li>Discard the PBS buffer. Add 20–30 mL 0.2× PBS into the container, keep shaking (20–30 rpm) for 5 min at 25°C.</li>
<li>Discard the PBS buffer. Add 20–30 mL ddH<sub>2</sub>O into the container, keep shaking (20–30 rpm) for 5 min at 25°C.</li>
<li>Put the microarrays into the slide centrifuge SlideWash<sup>TM</sup> 8 with default parameters. Centrifuge the microarrays at 500 × <i>g</i> for 1–2 min to dry them.
<div class="note">
<span class="note-title">Note:</span> the dried microarrays can be stored at 4°C in a sealed box.</div>
</li>
</ol>
</li>
<li>Antibody incubation.
<ol type="i">
<li>Prepare incubation buffer. Dilute the Anti-His antibody at 1:5000 in PBST buffer (0.1% Tween 20 in 1× PBS buffer).</li>
<li>Add 3 mL incubation buffer into the microarray slide container. Shake the container at 4°C and 20–30 rpm for 12 h.</li>
<li>Discard the incubation buffer. Add 5 mL PBST buffer and shake at 25°C for 5 min. Repeat this step for 3 times.</li>
</ol>
</li>
<li>Secondary antibody incubation.
<ol type="i">
<li>Dilute Cy5 labeled Anti-Rabbit secondary antibody at a ratio of 1:1000 in PBST buffer.</li>
<li>Add the secondary antibody into the microarray slide container, shake at 25°C for 1 h.</li>
<li>Discard the incubation buffer. Add 5 mL PBST buffer and shake at 25°C for 5 min. Repeat this step for 3 times.</li>
<li>Dry the microarrays by centrifugation with SlideWash<sup>TM</sup> 8. Centrifuge the microarrays at 500 × <i>g</i> for 1–2 min.</li>
</ol>
</li>
<li>Microarray scanning
<ol type="i">
<li>Turn on the microarray scanner Genepix 4200A and warm up for 15 min.</li>
<li>Scan the microarray at both 532 nm (IgG) and 635 nm (IgM) channel. Recommended scanning parameters: 95% power, PMT 200-300. However, the optimal scanning parameters should be adjusted according to the overall signal intensity.</li>
<li>After scanning is completed, save the two images as a single TIF file <b>(</b><a href="#fig1">Figure 1</a>B<b>)</b>.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Serum profiling</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="12">
<li>Blocking. This step is same as the blocking step of quality evaluation.</li>
<li>Sera sample preparation. Centrifuge the tube with sera at 12,000 × g at 4°C for 20 min. Dilute the sera in incubation buffer which contains 1% BSA and 0.1% Tween 20 in 1× PBS (1:200).</li>
<li>Sample incubation.
<ol type="a">
<li>Install a 16-well chamber on the microarray to physically separate the subarrays.</li>
<li>Dilute the sera sample and the positive control at a ratio of 1:200 with incubation buffer (1× PBS with 0.1% tween-20 and 1% BSA). (To our experience, the dilution could range from 1:50 to 1:1000, however, to balance foreground and background, 1:200 is the best dilution for serum profiling on protein microarray. Thus, in the COVID-19 study, we empirically chose 1:200).</li>
<li>Add 200 μL diluted serum sample to each subarray. For each microarray, the last two subarrays are added with incubation buffer (1% BSA in PBST) and the positive sera sample as blank control and positive control, respectively. The blank control is to monitor unexpected contaminations.</li>
<li>Incubate the microarrays at room temperature (25°C) for 2 h.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Immediately record the sample number, including the block position and the slide number.</div>
<ol start="15">
<li>Wash the microarrays.
<ol type="a">
<li>Remove the sera from each subarray with a multichannel pipette, add 200 μL wash buffer (1× PBS with 0.1% tween-20), and pipette up and down for 10 times.</li>
<li>Repeat the above step twice.</li>
<li>Uninstall the 16-well chamber, place the microarrays in a container with ∼30 mL wash buffer. Keep the container on a shaker (90–100 rpm) at room temperature (25°C) for 10 min. Repeat this step twice.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not leave the surface of the slide dry.</div>
<ol start="16">
<li>Secondary antibody incubation
<ol type="a">
<li>Dilute the Cy3 labeled anti-Human IgG and Alexa Fluor 647 (635 nm excitation) labeled anti-Human IgM secondary antibody (1: 1000) with incubation buffer.</li>
<li>Discard the Wash buffer from the container and add 15 mL diluted secondary antibodies.</li>
<li>Keep the container on a shaker (20–30 rpm) for 1 h at room temperature (25°C). Keep the microarrays shielded from light.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 1) if only one microarray is used, a smaller container would be suggested with a smaller incubation volume. 2) Depending on the isotypes of antibody of interest, multiple isotypes could be detected simultaneously, only if the isotype specific fluorescence labeled secondary antibodies are available and compatible with the microarray scanner.</div>
<ol start="17">
<li>Wash and dry.
<ol type="a">
<li>Remove the solution from the container and add 30 mL Wash buffer. Keep the container at room temperature (25°C) and shake for 90–100 rpm on a shaker for 10 min. Repeat this step for three times.</li>
<li>Wash the slide with ddH<sub>2</sub>O for 10 min, and rinse with ddH<sub>2</sub>O for 10 s.</li>
<li>Dry the slide by centrifugation.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> From <b>step 15</b> on, the slides should be kept under dark.</div>
<ol start="18">
<li>Microarray scanning.
<ol type="a">
<li>Turn on the microarray scanner Genepix 4200A and warm up for 15 min.</li>
<li>Scan the microarray at 532 nm (IgG) and 635 nm (IgM) channel. Recommended scanning parameters: 95% power, PMT 200-300. However, the optimal scanning parameters should be adjusted according to the overall signal intensity.</li>
<li>After scanning is completed, save the result as one single TIF file <b>(</b><a href="#fig1">Figure 1</a>C<b>).</b>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The same scanning parameters should be applied to the microarrays in the same experiment.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid scratching on microarray surface in any way during the whole process.</div>
<h3 id="sec3.4">Microarray data processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Quantitative signal intensity of each protein indicates the antibody response of one serum sample, which can be acquired by transforming the fluorescence intensity of each spot in the microarray image to digital signal, <i>i. e</i>., the microarray data extraction. The data are normalized to calibrate the differences among different microarrays.</p>
<ol start="19">
<li>Extract microarray data.
<ol type="a">
<li>The software Genepix 6.0 enables automatic alignment. Using the GAL file <b>(step 10)</b> to align the features of the microarray image, before extract the data, it is necessary to double check the aligned features and adjust the incorrectly aligned features.</li>
<li>Click the “analysis” button to extract the data of all features and generate a GPR file (raw data) for each microarray.</li>
</ol>
</li>
<li>Calculate signal intensity.
<ol type="a">
<li>Calculate signal intensity for each feature: subtract Foreground Median (F median) with Background Median (B median). IgG (532 nm channel) and IgM (635 nm channel) are calculated separately.</li>
<li>Calculate the average of the triplicates of each protein in the same subarray.</li>
</ol>
</li>
<li>Normalization among microarrays. Use the data of block #14 in each slide (the positive sera sample) to calculate the normalization factor (Factor N) for each microarray. IgG (532 nm channel) and IgM (635 nm channel) are calculated separately.
<ol type="a">
<li>Exclude the features with low signal intensity (&lt; 300).</li>
<li>With the data of block #14 of all microarrays, calculate the overall average signal intensity for each protein (without the controls) to create a reference data set.</li>
<li>Calculate the Pearson correlation coefficient between the reference data set and each data set of block #14. Exclude the spots which away from the diagonal line when the coefficient is less than 0.98.</li>
<li>Calculate the Factor N for each using the following function in Excel software: LINEST (Y, X i<sup>,</sup> FALSE), where the Y is the reference data set, the X i is the data of block #14 from slide i , and FALSE means to make the constant term equal to 0.</li>
<li>For each slide, all the signal intensities are normalized by multiplying the corresponding Factor N.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The recombinant proteins generate obvious bands in gel electrophoresis (<a href="#fig1">Figure 1</a>A). The quality of SARS-CoV-2 protein microarray can be evaluated by anti-His antibody (<a href="#fig1">Figure 1</a>B). The fabricated protein microarray exhibits bright, uniform, organized and regular features. For sera from COVID-19 patients, the microarray image is expected to exhibit high differences among the proteins but normally all the spike proteins are positive. The background is low and uniform (<a href="#fig1">Figure 1</a>C). High correlations could be achieved between the repeats of the positive sera samples analyzed on different microarrays. The normalization factor (Factor N) could be easily calculated by linear fitting (<a href="#fig1">Figure 1</a>D). Construction of an antibody response landscape against SARS-CoV-2 proteome (<a href="#fig1">Figure 1</a>E). Except for S1 and N, which are known of highly antigenic, we found that several non-structural and accessory proteins elicited prevalent antibody responses, especially for IgG, including NSP1, NSP7, NSP8, RdRp, ORF3b and ORF9b, for which the positive rates are 38%, 48.4%, 27.9%, 30.3%, 52.1% and 28%, respectively.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The proteins were obtained by affinity purification, the activity of the protein is affected by the expression system, thus, it is not easy to keep the consistency of protein concentration and quality among different batches.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>High background of the microarray (step 18).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Strictly follow the step of microarray washing (step 17); Increase the blocking time (3–6 h) or concentration of the BSA in blocking buffer (step 12); Decrease the concentration of the serum samples (step 13); The power of the scanner may be too high (step 18).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>False positive signals of the proteins were observed for the blank control (# 13 Subarray) on some microarrays or some features which only contain printing buffer (step 16).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The False positive on the control microarray may be caused by contaminations when loading serum samples or contaminations when performing the pre-wash step. It could be the printing buffer, blocking buffer, incubation buffer or wash buffer is contaminated. Replace the current buffers with freshly prepared buffers, then re-analyze the samples and carefully operate during the incubation and wash steps.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Merged features on the microarray (step 18).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Try to increase the spot-to-spot distance in printing (step 9); lower the concentration of the proteins (step 8); extend the blocking time (step 12).</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The prepared protein is limited by expression and purification (step 5).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Acquire recombinant SARS-CoV-2 proteins produced using yeast cell-free systems or mammalian cell expression systems from a variety of commercial sources.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>The signal of proteins on the microarray may not be uniform (step 18).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Perform accurate protein quantification and test different protein dilutions (step 6) to determine the optimal concentration (step 8).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib2">Jiang, H.-W., Zhang, H.-N., Meng, Q.-F., Xie, J., Li, Y., Chen, H., Zheng, Y.-X., Wang, X.-N., Qi, H., Zhang, J., et al. (2020). SARS-CoV-2 Orf9b suppresses type I interferon responses by targeting TOM70. Cell. Mol. Immunol. <i>17</i>, 998-1000. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00118-6/sref2">View at publisher</a></p>
<p id="bib3">Li, Y., Lai, D.-Y., Zhang, H., Jiang, H., Tian, X., Ma, M., Qi, H., Meng, Q., Guo, S., Wu, Y., et al. (2020). Linear epitopes of SARS-CoV-2 spike protein elicit neutralizing antibodies in COVID-19 patients. Cell. Mol. Immunol. <i>17</i>, 1095-1097. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00118-6/sref3">View at publisher</a></p>
<p id="bib4">Li, Y., Ma, M.-L., Lei, Q., Wang, F., Hong, W., Lai, D.-Y., Hou, H., Xu, Z.-W., Zhang, B., Chen, H., et al. (2021a). Linear epitope landscape of the SARS-CoV-2 Spike protein constructed from 1,051 COVID-19 patients. Cell. Rep. <i>34</i>, 108915. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00118-6/sref4">View at publisher</a></p>
<p id="bib5">Li, Y., Lai, D., Lei, Q., Xu, Z.-W., Hou, H., Shan, H., Wang, F., Fan, X., and Tao, S. (2021b). Systematic evaluation of SARS-CoV-2 spike protein derived peptides for diagnosis of COVID-19 patients. Cell. Mol. Immunol. <i>18</i>, 621-631. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00118-6/sref5">View at publisher</a></p>
<p id="bib6">Li, Y., Xu, Z.-W., Lei, Q., Lai, D.-Y., Hou, H.-Y., Jiang, H.-W., Zheng, Y.-X., Wang, X.-N., Wu, J.-X., Ma, M.-L., et al. (2021c). Antibody landscape against SARS-CoV-2 reveals signifificant differences between non-structural/accessory and structural proteins. Cell. Rep. <i>36</i>, 109391. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00118-6/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Sheng-ce Tao (<a href="mailto:taosc@sjtu.edu.cn">taosc@sjtu.edu.cn</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze datasets/code.</p>
<h3>Acknowledgments</h3>
<p>This work was partially supported by the <a href="https://doi.org/10.13039/501100012166">National Key Research and Development Program of China</a> (grant no. 2016YFA0500600) and the <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (31970130, 31600672, 31670831, 31370813, 31900112, and 21907065).</p>
<h3>Author contributions</h3>
<p>S.-C.T. supervised the project. H.-W.J., H.-N.Z., Y.L., and D.-Y.L. conducted the experiments and data analysis. D.-Y.L., Y.L., H.-W.J., and S.-C.T. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>