<article>
<section>
<h1 id="header">Generation and validation of CRISPR-engineered human natural killer cell lines for research and therapeutic applications</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Anil Kumar,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Sung June Lee,<sup><a href="#aff1">1</a></sup> Qiao Liu,<sup><a href="#aff1">1</a></sup> Anthony K.N. Chan,<sup><a href="#aff1">1</a></sup> Sheela Pangeni Pokharel,<sup><a href="#aff1">1</a></sup> Jianhua Yu,<sup><a href="#aff2">2</a></sup> Chun-Wei Chen,<sup><a href="#aff1">1</a></sup> and Srividya Swaminathan<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Systems Biology, Beckman Research Institute of City of Hope, Monrovia, CA 91016, USA</p>
<p id="aff2"><sup>2</sup>Department of Hematology and Hematopoietic Stem Cell Transplantation, Beckman Research Institute of City of Hope, Duarte, CA 91010, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:anikumar@coh.org">anikumar@coh.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sswaminathan@coh.org">sswaminathan@coh.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100874">10.1016/j.xpro.2021.100874</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cytotoxic natural killer cells kill tumors and infected cells. We carried out CRISPR-based gene editing and transcriptional regulation in hard-to-manipulate NK-92 cells. NK-92-based therapies were found to be safe and efficacious in preclinical studies of cancers. Here, we have pioneered the generation and validation of NK-92 cells constitutively expressing Cas9 or dCas9 for knockout (CRISPRko), transcriptional activation (CRISPRa), or transcriptional repression (CRISPRi) of genes. Our CRISPR-engineered NK-92 cell platforms can be modified for research and off-the-shelf therapeutic applications.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Constitutively repressing or activating genes in human NK cells is difficult</li>
<li>CRISPR-engineered natural killer cells are attractive for therapeutic applications</li>
<li>Our approach generates stable CRISPRko, CRISPRi, and CRISPRa human NK cell lines</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1115-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 10–15 days for NK-92 cells; ∼ 3 days for K-562 cells; ∼ 3–6 days for HEK cells</div>
<ol>
<li>Thaw NK-92 natural killer (NK), chronic myeloid leukemia K-562, and Lenti-X293T human embryonic kidney (HEK) cell lines at 37°C in a metallic bead bath. Refer to <a href="#materials-and-equipment">materials and equipment</a> section for details of NK-92, K-562, and HEK cell culture growth media.</li>
<li>Transfer thawed NK-92 and K-562 cells into two separate 15 mL tubes containing 10 mL each of complete RPMI media. Transfer HEK cells into a single 15 mL tube containing 10 mL complete DMEM medium.</li>
<li>Pellet down the cells by centrifugation at 300 × <i>g</i> for 5 min at room temperature (RT; 18°C–22°C) and aspirate the supernatant.</li>
<li>Wash the NK-92, K-562, and HEK cells again with 10 mL of their respective media for completely removing DMSO (300 × <i>g</i>, 5 min at RT).</li>
<li>Resuspend NK-92 cells in 1 mL of complete RPMI media, count the cells, and seed 0.5–1 × 10<sup>5</sup> cells/200 μL/well in a 96-well plate (flat bottom) in NK cell media (complete RPMI + rhIL-2) at 37°C.
<ol type="a">
<li>When cells become confluent (media color changes to light yellow), spin down the cells (300 × <i>g</i> at RT for 5 min) and aspirate the media.</li>
<li>Resuspend the cells in 500 μL of fresh NK cell media and culture them in a 48-well plate at 37°C.</li>
<li>Wait until cells become confluent, then spin down, aspirate the media and transfer these cells to a T-25 flask containing 5 mL of fresh NK cell media.</li>
<li>After expansion in a T-25 flask for 3 days, cells will be ready to be used in transduction experiments.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For successful NK-92 cell transduction, use cells that have been cultured for at least 7 days after thawing. Once thawed, we do not recommend culturing NK-92 cells beyond 3 months.</div>
<ol start="6">
<li>Resuspend K-562 cells in 1 mL of complete RPMI media, count the cells, and seed ∼ 1 × 10<sup>6</sup> cells in 5 mL complete RPMI media in a T-25 culture flask. After 3 days of culture in T-25 flask, cells will be ready to be used as targets in NK cell cytotoxicity experiments.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For successful NK cell cytotoxicity analyses, use K-562 cells that have been in culture for at least 3 days after thawing. Once thawed, we do not recommend culturing K-562 cells beyond 3 months.</div>
<ol start="7">
<li>Resuspend HEK cells in 1 mL of complete DMEM media, count the cells, and seed ∼5 × 10<sup>5</sup> cells in a 150 mm culture dish in 15 mL complete DMEM.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> For successful lentiviral transduction, we recommend using HEK cells 1–2 passages (∼3–6 days) after thaw. Once thawed, we do not recommend culturing HEK cells beyond 5 passages.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>APC anti-human CD132 (IL2Rγ); working dilution 1:100</td>
<td>BioLegend</td>
<td>Cat# 338607, RRID: AB_2123585</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>NEB 5-alpha competent <i>E. coli</i>
</td>
<td>New England Biolabs</td>
<td>Cat# C2987U</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>GeneJET Gel Extraction Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# K0691</td>
</tr>
<tr>
<td>GeneJET Plasmid Miniprep Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# K0502</td>
</tr>
<tr>
<td>CellTrace Violet Cell Proliferation Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# C34557</td>
</tr>
<tr>
<td>NucleoSpin RNA Purification Kit</td>
<td>MACHEREY-NAGEL</td>
<td>Cat# 740955.250</td>
</tr>
<tr>
<td>NucleoBond Xtra Maxi Plus EF</td>
<td>MACHEREY-NAGEL</td>
<td>Cat# 740426.50</td>
</tr>
<tr>
<td>Lenti-X Concentrator</td>
<td>Takara Bio</td>
<td>Cat# 631231</td>
</tr>
<tr>
<td>SuperScript IV First Strand Synthesis System</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 18091050</td>
</tr>
<tr>
<td>Lipofectamine 2000</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11668019</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dulbecco’s phosphate-buffered saline (DPBS), pH 7.4 without Ca<sup>2+</sup> and Mg<sup>2+</sup>
</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 14190144</td>
</tr>
<tr>
<td>Recombinant human interleukin-2 (rhIL-2)</td>
<td>PeproTech</td>
<td>Cat# 200-02</td>
</tr>
<tr>
<td>Poly-L-lysine</td>
<td>R&amp;D Systems</td>
<td>Cat# 3438-100-01</td>
</tr>
<tr>
<td>Polybrene</td>
<td>Millipore Sigma</td>
<td>Cat# TR-1003-G</td>
</tr>
<tr>
<td>Sodium butyrate</td>
<td>Fisher Scientific</td>
<td>Cat# AAA1107922</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>Sigma-Aldrich</td>
<td>Cat# S2002-100G</td>
</tr>
<tr>
<td>Puromycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A1113803</td>
</tr>
<tr>
<td>Trypsin/EDTA Solution (TE)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R001100</td>
</tr>
<tr>
<td>LB Agar, Miller (Pre-Buffered Capsules)</td>
<td>Fisher Scientific</td>
<td>Cat# BP9734-500</td>
</tr>
<tr>
<td>Ethanol 70% mol. Grade</td>
<td>Fisher Scientific</td>
<td>Cat# BP8201500</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Fisher Scientific</td>
<td>Cat# BP2291</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Fisher Scientific</td>
<td>Cat# BP1760-5</td>
</tr>
<tr>
<td>Certified Agarose Powder</td>
<td>Bio-Rad</td>
<td>Cat# 1613100</td>
</tr>
<tr>
<td>4,6-Diamidino-2-phenylindole (DAPI); working dilution 1:5000</td>
<td>BioLegend</td>
<td>Cat# 422801</td>
</tr>
<tr>
<td>7-AAD Viability Staining Solution (200 tests)</td>
<td>BioLegend</td>
<td>Cat# 420403</td>
</tr>
<tr>
<td>Penicillin-Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15140122</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 21985023</td>
</tr>
<tr>
<td>MEM Non-essential Amino Acids (100X)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11140050</td>
</tr>
<tr>
<td>GlutaMAX (100X)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 35050061</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11360070</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 26140079</td>
</tr>
<tr>
<td>DMEM (Dulbecco's Modified Eagle Medium)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11965126</td>
</tr>
<tr>
<td>RPMI (Roswell Park Memorial Institute)-1640</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 11875119</td>
</tr>
<tr>
<td>SOS medium</td>
<td>New England Biolabs</td>
<td>Cat# B9020S</td>
</tr>
<tr>
<td>Terrific Broth (granulated)</td>
<td>Fisher Scientific</td>
<td>Cat# BP9728-500</td>
</tr>
<tr>
<td>PowerUp SYBR Green Master Mix</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A25742</td>
</tr>
<tr>
<td>BsmBI-v2</td>
<td>New England Biolabs</td>
<td>Cat# R0739S</td>
</tr>
<tr>
<td>CutSmart buffer (10X)</td>
<td>New England Biolabs</td>
<td>Cat# B7204S</td>
</tr>
<tr>
<td>T4 DNA ligase</td>
<td>New England Biolabs</td>
<td>Cat# M0202S</td>
</tr>
<tr>
<td>50X TAE Buffer</td>
<td>Bio-Rad</td>
<td>Cat# 1610743</td>
</tr>
<tr>
<td>1 kb Plus DNA Ladder</td>
<td>New England Biolabs</td>
<td>Cat# N3232S</td>
</tr>
<tr>
<td>Quick-Load Purple 1 kb DNA Ladder</td>
<td>New England Biolabs</td>
<td>Cat# N0552S</td>
</tr>
<tr>
<td>SYBR Green Nucleic Acid Gel Stain (10000X)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# S7563</td>
</tr>
<tr>
<td>Gel Loading Dye, Purple (6X), no SDS</td>
<td>New England Biolabs</td>
<td>Cat# B7025S</td>
</tr>
<tr>
<td>HyPure Molecular-Grade Water</td>
<td>Cytiva Lifesciences</td>
<td>Cat# SH30538.03</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>NK-92 cell line</td>
<td>ATCC</td>
<td>Cat# CRL-2407</td>
</tr>
<tr>
<td>Lenti-X 293T cell line</td>
<td>Takara Bio</td>
<td>Cat# 632180</td>
</tr>
<tr>
<td>K-562 cell line</td>
<td>ATCC</td>
<td>Cat# CCL-243</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Lenti Cas9-EGFP</td>
<td>Addgene</td>
<td>Cat# 63592, RRID: Addgene_63592</td>
</tr>
<tr>
<td>pCAG dCas9-KRAB-2A-EGFP</td>
<td>Addgene</td>
<td>Cat# 92396, RRID: Addgene_92396</td>
</tr>
<tr>
<td>dCAS9-VP64-GFP</td>
<td>Addgene</td>
<td>Cat# 61422, RRID: Addgene_61422</td>
</tr>
<tr>
<td>ipUSEPR</td>
<td>From Chun-Wei Chen</td>
<td>House-made plasmid (<a href="#bib35">Yang et al., 2021</a>)</td>
</tr>
<tr>
<td>pMD.2G</td>
<td>Addgene</td>
<td>Cat# 12259, RRID: Addgene_12259</td>
</tr>
<tr>
<td>psPAX2</td>
<td>Addgene</td>
<td>Cat# 12260, RRID: Addgene_12260</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides (5’→3′)</td>
</tr>
<tr>
<td>Control (Scramble) sgRNA (forward); CRISPRko/i/a</td>
<td>Derived from (<a href="#bib32">Wang et al., 2014</a>); ordered form Integrated DNA Technologies (IDT)</td>
<td>GTAGCGAACGTGTCCGGCGT</td>
</tr>
<tr>
<td>Control (Scramble) sgRNA (reverse)</td>
<td>Reverse complement of control sgRNA (forward); CRISPRko/i/a, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>ACGCCGGACACGTTCGCTAC</td>
</tr>
<tr>
<td>Red fluorescent protein (RFP) sgRNA (forward); CRISPRko</td>
<td>From Chun-Wei Chen Lab</td>
<td>GTCACCACATACGAAGACGG</td>
</tr>
<tr>
<td>RFP sgRNA (reverse)</td>
<td>Reverse complement of RFP sgRNA (forward); CRISPRko, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>CCGTCTTCGTATGTGGTGAC</td>
</tr>
<tr>
<td>IL2RG sgRNA (1) (forward); CRISPRko</td>
<td><a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a></td>
<td>GGTGCAGTACCGGACTGACT</td>
</tr>
<tr>
<td>IL2RG sgRNA (1) (reverse); CRISPRko</td>
<td>Reverse complement of IL2RG sgRNA (1) (forward); CRISPRko, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>AGTCAGTCCGGTACTGCACC</td>
</tr>
<tr>
<td>IL2RG sgRNA (2) (forward); CRISPRko</td>
<td><a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a></td>
<td>AGCCGCTTTAACCCACTCTG</td>
</tr>
<tr>
<td>IL2RG sgRNA (2) (reverse); CRISPRko</td>
<td>Reverse complement of IL2RG sgRNA (2) (forward); CRISPRko, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>CAGAGTGGGTTAAAGCGGCT</td>
</tr>
<tr>
<td>IL2RG sgRNA (1) (forward); CRISPRi</td>
<td><a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a></td>
<td>TGGTAATGATGGCTTCAACA</td>
</tr>
<tr>
<td>IL2RG sgRNA (1) (reverse); CRISPRi</td>
<td>Reverse complement of IL2RG sgRNA (1) (forward); CRISPRi, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>TGTTGAAGCCATCATTACCA</td>
</tr>
<tr>
<td>IL2RG sgRNA (2) (forward); CRISPRi</td>
<td><a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a></td>
<td>AGGGATGTGAATGGTAATGA</td>
</tr>
<tr>
<td>IL2RG sgRNA (2) (reverse); CRISPRi</td>
<td>Reverse complement of IL2RG sgRNA (2) (forward); CRISPRi, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>TCATTACCATTCACATCCCT</td>
</tr>
<tr>
<td>IL-15 sgRNA (forward); CRISPRa</td>
<td><a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a></td>
<td>ACACCTCCCGCGGAGACTGG</td>
</tr>
<tr>
<td>IL-15 sgRNA (reverse); CRISPRa</td>
<td>Reverse complement of IL-15 sgRNA (forward); CRISPRa, (<a href="https://www.bioinformatics.org/sms/rev_comp.html">https://www.bioinformatics.org/sms/rev_comp.html</a>)</td>
<td>CCAGTCTCCGCGGGAGGTGT</td>
</tr>
<tr>
<td>GAPDH (forward); qPCR</td>
<td>Derived from (<a href="#bib15">Kooreman et al., 2017</a>)</td>
<td>GTCTCCTCTGACTTCAACAGCG</td>
</tr>
<tr>
<td>GAPDH (reverse); qPCR</td>
<td>Derived from (<a href="#bib15">Kooreman et al., 2017</a>)</td>
<td>ACCACCCTGTTGCTGTAGCCAA</td>
</tr>
<tr>
<td>IL-15 (forward); qPCR</td>
<td>Derived from (<a href="#bib15">Kooreman et al., 2017</a>)</td>
<td>AACAGAAGCCAACTGGGTGAATG</td>
</tr>
<tr>
<td>IL-15 (reverse); qPCR</td>
<td>Derived from (<a href="#bib15">Kooreman et al., 2017</a>)</td>
<td>CTCCAAGAGAAAGCACTTCATTGC</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo 10.7.1 software</td>
<td>FlowJo (Becton Dickinson)</td>
<td><a href="https://www.flowjo.com/">https://www.flowjo.com/</a></td>
</tr>
<tr>
<td>sgRNA Designer, CRISPick</td>
<td>Broad Institute</td>
<td><a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>150 mm Surface-treated tissue culture dishes</td>
<td>Fisher Scientific</td>
<td>Cat# FB012925</td>
</tr>
<tr>
<td>Steriflip-HV Sterile Centrifuge Tube Top Filter Unit</td>
<td>Millipore Sigma</td>
<td>Cat# SE1M003M00</td>
</tr>
<tr>
<td>Fisherbrand Disposable PES filter Units (500 mL)</td>
<td>Fisher Scientific</td>
<td>Cat# FB12566504</td>
</tr>
<tr>
<td>Olympus 384-well PCR plate</td>
<td>Genesee Scientific</td>
<td>Cat# 24-305</td>
</tr>
<tr>
<td>96-Well Cell Culture Plates (flat bottom)</td>
<td>Genesee Scientific</td>
<td>Cat# 25-109</td>
</tr>
<tr>
<td>96-Well Cell Culture Plates (U bottom)</td>
<td>Genesee Scientific</td>
<td>Cat# 25-221</td>
</tr>
<tr>
<td>48-Well Cell Culture Plates</td>
<td>Genesee Scientific</td>
<td>Cat# 25-108</td>
</tr>
<tr>
<td>24-Well Cell Culture Plates</td>
<td>Genesee Scientific</td>
<td>Cat# 25-107</td>
</tr>
<tr>
<td>TC Treated Flasks, 25 mL, Vent (T-25)</td>
<td>Genesee Scientific</td>
<td>Cat# 25-205</td>
</tr>
<tr>
<td>0.2 mL 8-Strip PCR Tubes, Natural</td>
<td>Genesee Scientific</td>
<td>Cat# 27-125</td>
</tr>
<tr>
<td>50 mL Conical centrifuge tubes</td>
<td>Genesee Scientific</td>
<td>Cat# 28-106</td>
</tr>
<tr>
<td>15 mL Conical centrifuge tubes</td>
<td>Genesee Scientific</td>
<td>Cat# 28-103</td>
</tr>
<tr>
<td>FACS tubes (Falcon round-bottom polypropylene tubes)</td>
<td>Fisher Scientific</td>
<td>Cat# 14-959AA</td>
</tr>
<tr>
<td>14 mL Polypropylene round-bottom tubes with caps</td>
<td>Corning Life Sciences</td>
<td>Cat# 352059</td>
</tr>
<tr>
<td>1.7 mL Microtubes</td>
<td>Genesee Scientific</td>
<td>Cat# 22-281</td>
</tr>
<tr>
<td>0.6 mL Microtubes</td>
<td>Genesee Scientific</td>
<td>Cat# 24-272</td>
</tr>
<tr>
<td>1000 μL Pipette tips</td>
<td>Genesee Scientific</td>
<td>Cat# 24-830</td>
</tr>
<tr>
<td>200 μL Pipette tips</td>
<td>Genesee Scientific</td>
<td>Cat# 24-812</td>
</tr>
<tr>
<td>20 μL Pipette tips</td>
<td>Genesee Scientific</td>
<td>Cat# 24-804</td>
</tr>
<tr>
<td>10 μL Pipette tips</td>
<td>Genesee Scientific</td>
<td>Cat# 24-801</td>
</tr>
<tr>
<td>200 μL Round gel loading tips, 0.57 mm</td>
<td>Genesee Scientific</td>
<td>Cat# 24-113</td>
</tr>
<tr>
<td>FACSymphony<sup>TM</sup> A5 flow cytometer</td>
<td>Becton Dickinson (BD)</td>
<td>n/a</td>
</tr>
<tr>
<td>ProFlex PCR System</td>
<td>Thermo Fisher Scientific</td>
<td>n/a</td>
</tr>
<tr>
<td>NanoDrop<sup>TM</sup> One Spectrophotometer</td>
<td>Thermo Fisher Scientific</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Growth media for NK-92 cell line</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI-1640</td>
<td>N/A</td>
</tr>
<tr>
<td>FBS</td>
<td>10% (v/v)</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>Sodium pyruvate (100 mM)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>Minimum Essential Media (MEM) non-essential amino acid (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>GlutaMAX (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>2-Mercaptoethanol (50 mM)</td>
<td>0.01% (v/v)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Note: Mix all components and filter the media using sterile disposable vacuum filter with polyethersulfone (PES) membrane. This is complete RPMI media, to grow NK-92 cells, supplement complete media with recombinant human (rh) IL-2 (100 U/mL). Store the media at 4°C for up to 1 month.</p>
<table id="undtbl2">
<caption>Growth media for Lenti-X 293T cell line</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>N/A</td>
</tr>
<tr>
<td>FBS</td>
<td>10% (v/v)</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>Sodium pyruvate (100 mM)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>MEM non-essential amino acid (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>GlutaMAX (100X)</td>
<td>1% (v/v)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Note: Mix all these components and filter the media using sterile disposable vacuum filter with PES membrane. This is complete DMEM media. Store the media at 4°C for up to 1 month.</p>
<table id="undtbl3">
<caption>Growth media for K-562 cell line</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI-1640</td>
<td>N/A</td>
</tr>
<tr>
<td>FBS</td>
<td>10% (v/v)</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>Sodium pyruvate (100 mM)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>Minimum Essential Media (MEM) non-essential amino acid (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>GlutaMAX (100X)</td>
<td>1% (v/v)</td>
</tr>
<tr>
<td>2-Mercaptoethanol (50 mM)</td>
<td>0.01% (v/v)</td>
</tr>
</tbody>
</table>
<p class="table-legend">Note: K-562 cell line is grown in complete RPMI media described above. Mix all above components and filter the media using sterile disposable vacuum filter with PES membrane. Store the media at 4°C for up to 1 month.</p>
<table id="undtbl4">
<caption>Terrific Broth (TB) media (1 L)</caption>
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Terrific Broth powder</td>
<td>47.6 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>4 mL</td>
</tr>
<tr>
<td>Double distilled (dd) H<sub>2</sub>O</td>
<td>Up to 1000 mL</td>
</tr>
<tr>
<td>Ampicillin antibiotic</td>
<td>100 mg</td>
</tr>
</tbody>
</table>
<p class="table-legend">Note: Dissolve 47.6 g TB powder in 500 mL double distilled (dd) H<sub>2</sub>O and add 4 mL glycerol. Mix well and makeup the final media volume to 1 L with ddH<sub>2</sub>O and autoclave for 15 min at 121°C. Cool down to RT and add ampicillin antibiotic (working concentration is 100 μg/mL). Store the media at 4°C for up to 1 month.</p>
<p><b>Flow cytometry</b></p>
<p>Flow cytometry analysis was performed on BD FACSymphony<sup>TM</sup> A5 flow cytometer and data were analyzed using FlowJo 10.7.1 software.</p>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Anti-human IL2Rγ (APC)</td>
<td>1:100</td>
</tr>
<tr>
<td>4,6-diamidino-2-phenylindole (DAPI) (Stock 5 mg/mL)</td>
<td>1:5000</td>
</tr>
<tr>
<td>Fluorescence activated cell sorting (FACS) Buffer (Ca<sup>2+</sup> and Mg<sup>2+</sup> free DPBS + 2% FBS + 0.09% sodium azide)</td>
<td>N/A</td>
</tr>
</tbody>
</table>
<table id="undtbl6">
<caption>Fluorescence activated cell sorting (FACS) Buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ca<sup>2+</sup> and Mg<sup>2+</sup> free DPBS</td>
<td>N/A</td>
<td>490 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>2% (v/v)</td>
<td>10 mL</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>0.09% (w/v)</td>
<td>450 mg</td>
</tr>
</tbody>
</table>
<p class="table-legend"><b>Note:</b> Store the buffer at 4°C for up to 6 months.</p>
<table id="undtbl7">
<caption>Single guide RNA (sg) target sequences</caption>
<thead>
<tr>
<th>Name</th>
<th>Forward sequence (5’→3′)</th>
<th>Reverse sequence (5’→3′)</th>
<th>Application</th>
</tr>
</thead>
<tbody>
<tr>
<td>Control (Scramble) sgRNA</td>
<td>GTAGCGAACGTGTCCGGCGT</td>
<td>ACGCCGGACACGTTCGCTAC</td>
<td>CRISPRko/i/a</td>
</tr>
<tr>
<td>RFP sgRNA</td>
<td>GTCACCACATACGAAGACGG</td>
<td>CCGTCTTCGTATGTGGTGAC</td>
<td>CRISPRko</td>
</tr>
<tr>
<td>IL2RG sgRNA (1)</td>
<td>GGTGCAGTACCGGACTGACT</td>
<td>AGTCAGTCCGGTACTGCACC</td>
<td>CRISPRko</td>
</tr>
<tr>
<td>IL2RG sgRNA (2)</td>
<td>AGCCGCTTTAACCCACTCTG</td>
<td>CAGAGTGGGTTAAAGCGGCT</td>
<td>CRISPRko</td>
</tr>
<tr>
<td>IL2RG sgRNA (1)</td>
<td>TGGTAATGATGGCTTCAACA</td>
<td>TGTTGAAGCCATCATTACCA</td>
<td>CRISPRi</td>
</tr>
<tr>
<td>IL2RG sgRNA (2)</td>
<td>AGGGATGTGAATGGTAATGA</td>
<td>TCATTACCATTCACATCCCT</td>
<td>CRISPRi</td>
</tr>
<tr>
<td>IL-15 sgRNA</td>
<td>ACACCTCCCGCGGAGACTGG</td>
<td>CCAGTCTCCGCGGGAGGTGT</td>
<td>CRISPRa</td>
</tr>
</tbody>
</table>
<p class="table-legend">Forward sgRNAs each with a 5′ overhang “<b>CACCG”</b> and reverse sgRNAs each with 5’ “<b>AAAC</b>” and 3’ “<b>C</b>” overhangs were ordered form integrated DNA technologies (IDT). Refer to the section titled “oligo design for sgRNA production” for sgRNA design details.</p>
<table id="undtbl8">
<caption>Primers used in qPCR</caption>
<thead>
<tr>
<th>Name</th>
<th>Forward sequence</th>
<th>Reverse sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>GAPDH</td>
<td>GTCTCCTCTGACTTCAACAGCG</td>
<td>ACCACCCTGTTGCTGTAGCCAA</td>
</tr>
<tr>
<td>IL-15</td>
<td>AACAGAAGCCAACTGGGTGAATG</td>
<td>CTCCAAGAGAAAGCACTTCATTGC</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Lentivirus production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<ol>
<li>On day 1, coat a 150 mm culture dish with ∼3–5 mL of poly-L lysine solution (0.01%) by gently swirling to ensure even coating of the plate surface. Aspirate the solution and let the coated plate dry for 15 min at RT.</li>
<li>Aspirate the cell culture media from Lenti-X 293T HEK cells grown in a 150 mm culture dish, wash these cells with PBS, add 2 mL of 1X Trypsin solution to the plate. Gently swirl the plate to evenly cover the whole surface area and incubate the plate for 1 min at 37°C. Detach the cells by gently tapping the plate and inactivate the Trypsin enzyme by adding 10 mL of complete DMEM media. Spin them down at 350 × <i>g</i> for 5 min.</li>
<li>Resuspend the cells in 10 mL of complete DMEM media, count the cells, and seed 7–10 × 10<sup>6</sup> Lenti-X 293T HEK cells on the poly-L lysine-coated 150 mm plate. Make up the total cell culture media volume to 20 mL by adding complete DMEM media, gently swirl the plate so that cells are uniformly seeded and incubate them at 37°C.</li>
</ol>
<ol start="4">
<li>On day 2, take two 15 mL tubes, add 4 mL of plain DMEM media (media without FBS and PS and containing only 1X GlutaMAX (glutamine), non-essential amino acids and sodium pyruvate) in each tube. In the first tube, add 20 μg of the plasmid containing the <b>gene of interest</b> (here, Lenti-Cas9-EGFP or dCas9-VP64-GFP or dCas9-KRAB-GFP), 20 μg of packaging plasmid (psPAX2), and 5 μg of envelope plasmid (pMD.2G). In the second tube, add 80 μL of lipofectamine 2000. Incubate both tubes at RT for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Use plain DMEM i.e. serum-free DMEM, as serum inhibits transfection.</div>
<ol type="a">
<li>Mix lipofectamine-containing media with media containing the lentiviral vectors by gentle swirling or inverting and incubate the mix for 20 min at RT.</li>
<li>After incubation, aspirate the culture media from Lenti-X 293T HEK cells and add the lipofectamine and plasmid mix to these cells in a dropwise fashion over the entire plate. Make up the total media volume to 15 mL by including additional 7 mL of plain DMEM to Lenti-X293T HEK cells. Gently swirl the plate and incubate at 37°C for 3 h.</li>
<li>After incubation, add 15 mL complete FBS-containing DMEM media to the plate, making the total media volume to 30 mL. Incubate overnight (12–16 h) at 37°C.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>On day 3, aspirate the 30 mL of media from culture dish and add 20 mL of fresh complete DMEM media containing 400 μL of 500 mM sodium butyrate. Then, incubate for 5 h at 37°C to enhance the lentiviral production. After sodium butyrate induction, discard the media, replenish the cells with 16 mL fresh complete DMEM media, and incubate at 37°C for 24 h.</li>
</ol>
<ol start="6">
<li>On day 4, filter the media containing lentivirus using a sterile disposable vacuum filter unit (0.45 μm).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Clarification of media can also be done with centrifugation (400 × <i>g</i> for 5 min).</div>
<ol type="a">
<li>Concentrate lentivirus by adding 4 mL of lenti-X concentrator to 16 mL of viral supernatant, mixing well by pipetting, and incubating overnight (12–16 h) at 4°C.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>On day 5, centrifuge the mix at 1500 × <i>g</i> for 45 min, 4°C. After centrifugation, an off-white pellet will form at the bottom of centrifugation tube, without disturbing the pellet, aspirate the supernatant by suction using a glass Pasteur pipette. Finally, resuspend the off-white pellet in 1 mL of complete RPMI media.</li>
</ol>
<h3 id="sec3.2">Designing and cloning single guide RNAs (sgRNAs) for CRISPR-mediated gene editing and transcriptional modulation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 2 days</div>
<ol start="8">
<li>
<b>Oligo design for sgRNA production.</b> Go to <a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a> Broad institute web portal (<a href="#bib8">Doench et al., 2016</a>) .</li>
<li>Select the reference genome (here, Human GRCh37) and then select the mechanism of CRISPR editing (CRISPRko or CRISPRa or CRISPRi). Enzyme will be SpyoCas9.</li>
<li>Select quick look up in the target section and enter the target gene I.D. (e.g., IL2RG for IL2Rγ).</li>
<li>Enter the value in CRISPICK Quota box (this value represents how many sgRNA per gene will be designed) and submit.</li>
<li>Download the results by selecting picking result file (tab-limited text file).</li>
<li>Open the file in excel, this file contains the summary statistics for on- and off-target scores by pick order. Choose the guide RNA sequence-based on pick order rank (choose minimum 2–5 sgRNAs per gene). Also check the orientation of the sgRNA (sense or antisense).</li>
<li>Design the reverse complement of the sgRNA sequence.</li>
<li>Add the following overhang to the sgRNA sequence which will facilitate its ligation to linearized ipUSEPR plasmid
<ol type="a">
<li>Forward oligo: 5′ <b>CACCG</b>…guide RNA 3′</li>
<li>Reverse oligo: 5′ <b>AAAC</b>…guide RNA…<b>C</b> 3′</li>
</ol>
</li>
<li>Order the oligos from Integrated DNA Technologies (IDT) Inc.</li>
<li>
<b>Annealing.</b> Thaw and spin down the oligo vial at 300 × <i>g</i> for 2 min at RT and reconstitute the oligos in nuclease-free water to make a 100 μM stock (the amount of water added into the oligo vial to make the 100 μM stock = 10X number of nanomoles of oligo).</li>
</ol>
<ol start="18">
<li>Mix 10 μL of the forward oligo (100 μM) + 10 μL of the reverse oligo in a PCR tube and spin down at 300 × <i>g</i> for 2 min.<br> Use a thermocycler to anneal forward and reverse oligos:
<ol type="a">
<li>95°C 10 min × 1 cycle</li>
<li>Ramp down temperature from 95°C to 25°C, decrease rate = 0.5°C /1min × 140 cycles (total time = 140 min)</li>
<li>Hold at 4°C.</li>
</ol>
</li>
</ol>
<ol start="19">
<li>Take 1 μL of annealed oligo from step 18, add into 2 mL of ddH<sub>2</sub>O to reach a final dilution of 1/2000. These diluted oligos will be used as the insert for ligation.</li>
</ol>
<h3 id="sec3.3">Linearizing ipUSEPR plasmid for ligation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 1.5 days</div>
<ol start="20">
<li>
<b>Digestion of ipUSEPR plasmid.</b> Prepare the digestion mixture in 0.6 mL microtubes as follows:</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Volume/Conc</th>
</tr>
</thead>
<tbody>
<tr>
<td>ipUSEPR plasmid</td>
<td>40 μg</td>
</tr>
<tr>
<td>Cutsmart 10X (Cat# B7204S)</td>
<td>40 μL</td>
</tr>
<tr>
<td>BsmBI-v2 (Cat# R0739S)</td>
<td>16 μL</td>
</tr>
<tr>
<td>Rnase free water</td>
<td>Fill up to 400 μL</td>
</tr>
</tbody>
</table>
<ol start="21">
<li>Incubate the mixture overnight (12–16 h) at 55°C in a heat block.</li>
<li>Isolate the linearized plasmid by electrophoresis (0.8% agarose gel, 120 V for 2 h) followed by gel extraction.</li>
<li>Add 80 μL of 6X loading dye into 400 μL BsmbI digestion and load 50 μL in each well (Marker: NEB# N0552S, Gel loading dye: NEB# B7025S).</li>
<li>Gel extraction (GeneJET Gel Extraction kit # K0691):
<ol type="a">
<li>Weigh empty Eppendorf tubes and clip the expected band from gel into Eppendorf tubes.</li>
<li>Purify the linearized ipUSEPR plasmid as per manufacturer’s instructions and after elution, measure DNA concentration using a NanoDrop<sup>TM</sup> One spectrophotometer and store the linearized plasmid at −20°C.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Ligation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 14 h</div>
<ol start="25">
<li>Prepare ligation mixture in 0.2 mL PCR tubes as follows:</li>
</ol>
<table id="undtbl10">
<thead>
<tr>
<th>Reagents</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Linearized ipUSEPR plasmid (60–100 ng/μL)</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>10X T4 ligase buffer (supplied with T4 DNA ligase)</td>
<td>0.6 μL</td>
</tr>
<tr>
<td>T4 DNA ligase (NEB # M0202S)</td>
<td>0.3 μL</td>
</tr>
<tr>
<td>Master Mix (all above three products)</td>
<td>2.4 μL</td>
</tr>
<tr>
<td>1/2000 diluted oligo (remember adding H<sub>2</sub>O control)</td>
<td>3.6 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>6 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Do not forget to prepare a separate control tube which contains 3.6 μl of H<sub>2</sub>O instead of the diluted oligo.</div>
<ol start="26">
<li>Ligate using a thermocycler using following conditions:
<ol type="a">
<li>25°C 1 h × 1 cycle</li>
<li>16°C 12 h × 1 cycle</li>
<li>4°C hold</li>
</ol>
</li>
<li>Store ligation product at −20°C until transformation</li>
</ol>
<h3 id="sec3.5">Transformation (NEB-5 alpha competent <i>E. coli</i># C2987U)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 18 h</div>
<ol start="28">
<li>Thaw NEB 5-alpha competent <i>E. coli</i> cells on ice (50 μL/tube), and bring Ampicillin plates to RT.</li>
<li>Take 10 μL of competent cells and mix it with 6 μL of the ligated product in a 0.6 mL microtube and then, incubate on ice for 30 min.</li>
<li>Heat shock at 42°C for 30s in a heat block and put back on ice for 5 min.</li>
<li>Add 180 μL of SOC medium and grow the bacteria at 37°C in a shaker incubator for 1 h.</li>
<li>Spread all the medium onto the Ampicillin plate evenly and incubate the plates upside down at 37°C.</li>
<li>After 12–16 h, store the plates at 4°C until miniprep.</li>
</ol>
<h3 id="sec3.6">Miniprep (GeneJET Plasmid Miniprep Kit # K0502)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 18 h</div>
<ol start="34">
<li>Prepare 4 mL TB with Ampicillin in a 14 mL tube, pick a single colony with the help of a pipette tip from the bacterial plate into each tube, and shake overnight (12–16 h) at 37°C.</li>
<li>Take 1.8 mL bacteria into a 2 mL tube, and centrifuge at 11000 × <i>g</i> for 30s at RT to pellet down the bacteria.</li>
<li>Prepare the miniprep of plasmid DNA as per <a href="https://www.thermofisher.com/order/catalog/product/K0502">manufacturer’s instructions</a>.</li>
<li>In the final elution step, use 50 μL of elution buffer to elute the plasmid DNA from the column and measure concentration of the plasmid and store at −20°C.</li>
<li>Send 8 μL of 100–200 ng/μL DNA and 5 μL of 5 μM hU6 primer: “GAGGGCCTATTTCCCATGAT” per sample to <a href="https://www.etonbio.com/">Eton Biosciences</a> for sequencing to confirm the sgRNA sequence.</li>
</ol>
<h3 id="sec3.7">Maxiprep (NucleoBond Xtra Maxi Plus EF # 740426.50)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 1.5 days</div>
<ol start="39">
<li>Take 4 mL of TB medium in a 14 mL tube and inoculate with a single colony picked from ampicillin plate. Incubate at 37°C in a shaker incubator at 250 RPM for 5–6 h.</li>
<li>Add ∼1 mL of the inoculated media to 250 mL of TB media and grow the culture overnight (12–16 h) at 37°C in a shaker incubator at 250 RPM.</li>
<li>Harvest bacterial cells by centrifuging at 6000 × <i>g</i> for 15 min at 4°C. Then discard the supernatant.</li>
<li>Isolate the plasmid DNA as per manufacturer’s instructions. In the final elution step, use 400 μL of elution buffer to elute plasmid DNA from NucleoBond finalizer (provided in the kit).</li>
<li>Measure plasmid DNA concentration and store at −20°C.</li>
</ol>
<h3 id="sec3.8">Significance of genetically manipulating natural killer cells using CRISPR</h3>
<p>Natural killer (NK) cells are innate immune cells that protect against viral infections (<a href="#bib25">Strauss-Albee and Blish, 2016</a>) and development of cancers (<a href="#bib4">Cerwenka and Lanier, 2001</a>). NK cell-mediated immune surveillance is defective in infections and cancers for various reasons including NK cell exhaustion, defective homeostasis, and impaired NK cytotoxicity (<a href="#bib9">Duault et al., 2021</a>; <a href="#bib33">Wilk et al., 2021</a>). Our overall goal is to develop therapeutic NK cells which are functional and evade exhaustion through CRISPR-based genetic manipulation. To this end, we manipulated NK-92 cells , one of the platforms for NK therapies in clinical trials (<a href="#bib14">Klingemann et al., 2016</a>), using CRISPRko/Cas9 for knockout (<a href="#bib13">Jinek et al., 2012</a>), CRISPRi/dCas9-KRAB for transcriptional repression (<a href="#bib10">Gilbert et al., 2013</a>), and CRISPRa/dCas9-VP64 for transcriptional activation (<a href="#bib5">Chavez et al., 2015</a>) of genes that promote NK cytotoxicity and reduce NK exhaustion.</p>
<p>Below, we detail the protocols for generation of each of these multifunctional CRISPR-based NK cell platforms.</p>
<h3 id="sec3.9">Generating Cas9<sup>+</sup> NK-92 cells (CRISPRko NK-92 platform)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 7 weeks</div>
<ol start="44">
<li>Count NK-92 cells, resuspend ∼0.5–1.5 × 10<sup>6</sup> cells in 500 μL of complete RPMI media, and add 500 μL of concentrated Cas9-EGFP lentivirus (concentrated from 16 mL of lentiviral supernatant to 1 mL using lenti-X concentrator). Now the total volume will be 1 mL; to this, add polybrene (10 μg/mL) and rhIL-2 (100 U/mL). Seed these cells in a 24-well plate (1.5 × 10<sup>6</sup>/mL/well) and incubate at 37°C for 48 h.</li>
<li>On day 2, wash the cells with complete RPMI media, and replenish the NK-92 cells with NK cell media (RPMI media + rhIL-2). Pipette a small aliquot of the transduced cells (∼100 μL) into a 1.7 mL tube to measure percentages of EGFP<sup>+</sup> Cas9-transduced NK-92 cells by flow cytometry (<a href="#fig1">Figure 1</a>A upper panel).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1115-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Generation and validation of Cas9<sup>+</sup> NK-92 cell line</div>
<p>(A and B) Flow cytometry dot plots showing (A) frequencies of Cas9-EGFP<sup>+</sup> NK-92 cells on days 2 (upper panel) and 11 (lower panel) after transduction with CAS9-EGFP lentivirus, and (B) purity of Cas9-EGFP-transduced NK-92 cells immediately after sorting (day 0) and after expanding the sorted cells for 10 days.</p>
<p>(C) Histogram overlays comparing untransduced and Cas9-EGFP-transduced NK-92 cells for changes in CFSE-Violet fluorescence with respect to the day the cells were labeled with CFSE (day 0).</p>
<p>(D) Flow cytometry dot plots of Cas9-EGFP<sup>+</sup> NK-92 single cell clones expanded by limiting dilution method.</p>
<p>(E) Representative flow cytometry dot plots showing frequencies of sgRNA-RFP<sup>+</sup>Cas9-EGFP<sup>+</sup> NK-92 cells after transduction of clone-5 of Cas9-EGFP<sup>+</sup> NK-92 cells with either control sgRNA-RFP or RFP sgRNA-RFP to knockout RFP.</p>
<p>(F) Specific cytotoxicities of untransduced NK-92, polyclonal, and monoclonal (clone-5) dCas9-EGFP<sup>+</sup> NK-92 cells. Error bars represent mean ± SEM of three technical replicates.</p>
<p>(G and H) Graph showing fold change in frequencies of sgRNA-RFP<sup>+</sup>Cas9-EGFP<sup>+</sup> NK-92 cells with respect to day 1 after transduction with control sgRNA-RFP or IL-2RG sgRNA1-RFP or IL-2RG sgRNA2-RFP targeting IL2Rγ chain.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Because all NK cells including NK-92 are generally difficult to transduce, we obtained a transduction efficiency of only 1.32% after transducing NK-92 cells with Cas9 (<a href="#fig1">Figure 1</a>A upper panel).</div>
<ol start="46">
<li>Expand the Cas9-transduced NK-92 cells for the next 8–10 days by changing the NK cell media and re-plating the cells on every third day. Then, pipette a small aliquot of cells (∼0.5–1 × 10<sup>6</sup>) and stain them with DAPI and check the percentages of DAPI<sup>−</sup>Cas9-EGFP<sup>+</sup> viable cells (<a href="#fig1">Figure 1</a>A lower panel). This step is to ensure that transduction with Cas9 does not reduce the viability of NK-92 cells. We observed that NK cell viability was similar in untransduced and Cas9-transduced NK-92 cells (<a href="#fig1">Figure 1</a>A).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If the transduction efficiency is low, then it will be difficult to sort the transduced cells using FACS. In case of transduction efficiencies lower than 1%, expand cells for 1–2 weeks and transduce them again with an additional 500 μl of concentrated virus to increase the frequencies of transduced cells.</div>
<ol start="47">
<li>After expansion, stain the cells with DAPI and sort DAPI<sup>−</sup>EGFP<sup>+</sup> cells. Check the post-sort purity by flow cytometry (<a href="#fig1">Figure 1</a>B, <b>Day 0</b>). Expand the sorted cells as described below:
<ol type="a">
<li>Count the sorted cells and seed ∼ 0.5–1 × 10<sup>5</sup> cells/well in a 96-well flat bottom plate.</li>
<li>Expand these cells in NK media at 37°C. When cells become confluent (media color changes to light yellow), spin down the cells (300 × <i>g</i> at RT for 5 min) and aspirate the media.</li>
<li>Resuspend the cells in fresh 500 μL NK cell media and culture them in a 48 well plate at 37°C.</li>
<li>Wait until the cells become confluent, spin down, aspirate the media and transfer these cells to T-25 flask containing 5 mL of fresh NK cell media.</li>
<li>After expansion of NK-92 cells in a T-25 flask for 3 days, once again conduct flow cytometry to ensure that only EGFP<sup>+</sup> cells are expanding (<a href="#fig1">Figure 1</a>B, <b>Day 10</b>). Please note that cells from a single well of 96-well plate will be transferred to single well of 48-well plate and then to a T-25 flask.</li>
</ol>
</li>
<li>Compare the proliferation potential of transduced NK-92 cells with that of untransduced NK-92 cells to ensure that transduction with Cas9 does not reduce NK-92 cell proliferation:
<ol type="a">
<li>Reconstitute a vial of CFSE-violet dye (Thermo Scientific# C34557) in DMSO and add 20 μL of DMSO in the vial to make a 5 mM stock.</li>
<li>Spin down the untransduced and Cas9-transduced NK-92 cells (300 × <i>g</i> for 5 min at RT). Aspirate the culture media and resuspend the cells in PBS. Adjust the cell count to 1 × 10<sup>6</sup> NK-92 cells/mL in a 15 mL tube. Add 0.5 μL/mL of CFSE-violet dye and incubate these cells for 20 min at 37°C in the dark.</li>
<li>After incubation, add at least 5 times the original staining volume of complete RPMI media and incubate for 5 min at 37°C.</li>
<li>Centrifuge the cells (300 × <i>g</i> for 5 min), discard the media, and wash the cells again with complete RPMI media.</li>
<li>Adjust the cell count to 1 × 10<sup>6</sup> NK-92 cells/mL in NK cell media and seed 200 μL in 96-well flat-bottom plate (0.2 × 10<sup>6</sup> cells/well).</li>
<li>Incubate the cells for 96 h at 37°C and then analyze using flow cytometry for CFSE dilution (<a href="#fig1">Figure 1</a>C).</li>
<li>Analyze the data using FlowJo software and note down the median fluorescence intensity (MFI) values for CFSE and calculate the proliferation score (PS) using formula PS = Log<sub>2</sub> (MFI<sub>CFSE day 0</sub>/MFI<sub>CFSE day 4</sub>) (<a href="#bib1">Ahlen et al., 2009</a>). We found a less than 2-fold difference in the PS of untransduced NK-92 cells (PS=4.4) and Cas9<sup>+</sup> NK-92 cells (PS=3.5), suggesting that transduction with Cas9 did not abrogate or markedly impair NK-92 cell proliferation.</li>
</ol>
</li>
<li>Single cell isolation using limiting dilution technique:
<ol type="a">
<li>Count Cas9-EGFP<sup>+</sup> NK-92 cells using a hemocytometer and adjust the cell count to 5 × 10<sup>3</sup> cells/mL.</li>
<li>The final cell counts to seed a 96-well plate will be 5 cells/mL, and to achieve this count, mix 50 μL of 5 × 10<sup>3</sup> cells/mL solution with 49.95 mL of NK cell media.</li>
<li>Transfer 100 μL of 5 cells/mL solution to each well of 96-well round-bottom plates and incubate at 37°C. Carefully monitor these cells over a period of two weeks. When single-cell clumps have grown in size, remove these cells from 96-well plates using a 1 mL pipette, wash, and transfer to 48-well culture plate in NK cell media (500 μL/well).</li>
<li>After three days of culture in 48-well plate, analyze percentages of Cas9-EGFP<sup>+</sup> NK-92 cells in each clone using flow cytometry (<a href="#fig1">Figure 1</a>D) and culture the cells in a T-25 flask in NK cell media.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Accurately adjusting the cell count to 5 cells/mL and seeding 100 μl of this mix per well is most important step in the single NK cell expansion protocol. This minimizes the likelihood of seeding more than 1 cell/well. This step also prevents the expansion of untransduced EGFP<sup>−</sup> NK-92 cells in the culture.</div>
<p>Alternative method for single cell isolation: Flow cytometry-based sorting of single cell clones. However, sorting is not preferred because it is stressful for sensitive cells such as, human NK cells.</p>
<h3 id="sec3.10">Selecting cytotoxic Cas9<sup>+</sup> NK-92 single-cell clones for CRISPRko editing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 2 weeks</div>
<p>Cas9<sup>+</sup> NK-92 cells single cell clones were first evaluated for CRISPR editing efficiency by using a puromycin-tagged sgRNA that targets and knocks out the RFP present in the ipUSEPR sgRNA cloning plasmid, as described below:</p>
<ol start="50">
<li>To transduce single cell clones:
<ol type="a">
<li>Resuspend each single cell clone (here, clone-1 to -5, <a href="#fig1">Figure 1</a>D) in 1000 μL complete RPMI media (1 × 10<sup>6</sup> cells/clone) in 1.7 mL microtubes.</li>
<li>Divide single cell suspension from each clone into two parts and transfer into 1.7 mL microtubes.</li>
<li>To one portion, add 500 μL control sgRNA lentivirus and to other half add 500 μL lentivirus harboring sgRNA targeting RFP of ipUSEPR plasmid.</li>
<li>Now the total volume in each tube will be 1 mL, to this, add polybrene (10 μg/mL) and rhIL-2 (100 U/mL). Seed these cells in 24-well plate (0.5 × 10<sup>6</sup>/well) and incubate at 37°C for 48 h.</li>
</ol>
</li>
<li>On day 2, analyze frequencies of sgRNA-transduced RFP<sup>+</sup> cells by flow cytometry. Resuspend the remaining cells in 2 mL complete RPMI media containing rhIL-2 (100 U/mL) and puromycin (1 μg/mL) (<a href="#fig1">Figure 1</a>E, left panels).</li>
<li>After selecting transduced cells using puromycin for ∼10 days, we found that clone-5 showed the best CRISPR editing efficiency with percentages of RFP<sup>+</sup> cells reducing from 11.9% at day 2 to 0.46% at day 10 after puromycin selection (<a href="#fig1">Figure 1</a>E). Note that procedures for design and cloning of sgRNAs and production of sgRNA lentiviruses have been described previously (steps 1–43).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Before knocking out genes of interest in Cas9-transduced NK-92 single cell clones, one must confirm that constitutive expression of Cas9 does not impair the normal function (i.e. the cytotoxicity) of NK-92 cells.</div>
<p>We therefore compared the cytotoxic function of untransduced, bulk (polyclonal), and single cell Cas9-transduced NK-92 cells using a flow cytometry-based NK cytotoxicity assay. In this assay, we measured the ability of untransduced and Cas9<sup>+</sup> NK-92 cells to kill the standard NK-sensitive target human erythroleukemia cell line, K-562. Among the Cas9<sup>+</sup> NK-92 single cell clones, only clone-5 was utilized for the NK cytotoxicity/functionality assay because it was found to have the highest gene editing efficiency (<a href="#fig1">Figure 1</a>E). Below, we describe the procedure to conduct the NK cytotoxicity assay:</p>
<ol start="53">
<li>To distinguish NK-92 effector cells from target K-562 cells, label K-562 cells with CFSE using the procedure below:
<ol type="a">
<li>Reconstitute a vial of CFSE-violet dye (Thermo Scientific# C34557) by adding 20 μL DMSO in the vial to make a 5 mM stock.</li>
<li>Spin down K-562 cells (300 × <i>g</i> for 5 min at RT), resuspend the cell pellet in PBS in a 15 mL tube and adjust the cell count to 1 × 10<sup>6</sup> K-562 cells/mL. Add 0.5 μL/mL of CFSE-violet dye and incubate these cells for 20 min at 37°C in dark.</li>
<li>After incubation, add at least 5 times the original staining volume of complete RPMI-1640 media (RPMI-1640 + 10% FBS) and incubate for 5 min at 37°C.</li>
<li>Centrifuge the cells at 300 × <i>g</i> for 5 min at RT, discard the media, and wash the cells again with 5 mL of complete RPMI media before using them in cytotoxicity assays.</li>
</ol>
</li>
<li>Count NK-92 effector and K-562 target cells. Pipette mix these cells at different effector to target ratios in a U bottom 96-well plate in 200 μL complete RPMI media. Centrifuge the plate at 300 × <i>g</i> for 5 min at RT and incubate for 5 h at 37°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Don’t forget to include a control group that contains only CFSE labeled K-562 target cells to measure spontaneous cell death of the target in the absence of NK effector cells.</div>
<ol start="55">
<li>After incubation, add 3 μL of 7-AAD (50 μg/mL) in each well and mix by pipetting. Incubate the plate for 10 min at RT followed by transferring of cells into FACS tubes and analyze the cells by flow cytometry (<a href="#fig1">Figure 1</a>F). We calculate the specific cytotoxicity using the formula: [(7-AAD<sup>+</sup> target-cell frequency in coculture with effector cells − 7-AAD<sup>+</sup> target-cell frequency alone) / (100 − 7-AAD<sup>+</sup> target-cell frequency alone)] × 100.</li>
</ol>
<h3 id="sec3.11">Validating gene knockout in Cas9<sup>+</sup> NK-92 cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 10 days</div>
<p>As NK-92 cells are dependent on the cytokines IL-2 and IL-15 for their survival, knockout of the IL2R, to which these cytokines bind, should lead to the death of NK-92 cells (<a href="#bib3">Carson et al., 1997</a>; <a href="#bib29">Törnroos et al., 2019</a>). Therefore, we knocked out a required signaling component of the IL2R, the IL2Rγ chain (<a href="#bib31">Waickman et al., 2016</a>) to validate the successful generation of Cas9<sup>+</sup> NK-92 cells.</p>
<p>The procedures for designing and cloning IL2Rγ sgRNA, and that for producing IL2Rγ sgRNA lentiviruses have been described earlier.</p>
<ol start="56">
<li>To validate IL2Rγ gene knockout in Cas9<sup>+</sup> NK-92 cells:
<ol type="a">
<li>Count Cas9-EGFP<sup>+</sup> NK-92 cells, resuspend 1 × 10<sup>6</sup> cells in 1000 μL of complete RPMI media.</li>
<li>Divide the cell suspension into two equal parts and transfer to 1.7 mL microtubes.</li>
<li>Add 500 μL of concentrated lentivirus for control sgRNA or sgRNA targeting the IL2Rγ chain.</li>
<li>Now the total volume in each tube will be 1 mL. To these tubes, add polybrene (10 μg/mL) and rhIL-2 (100 U/mL).</li>
<li>Seed these cells in a 24-well plate (0.5 × 10<sup>6</sup> cells/well) and incubate at 37°C for 48 h. Refer to “<a href="#undtbl7">Single guide RNA (sg) target sequences table</a>” in <a href="#materials-and-equipment">materials and equipment</a> section for sequences of control or IL-2RG sgRNA used for CRISPRko.</li>
</ol>
</li>
<li>On day 2, wash the cells with complete RPMI media, resuspend the cells in 2 mL of media containing rhIL-2 (100 U/mL), and seed them in 24-well plate. Before seeding, take a small aliquot of cells to determine the frequencies of transduced cells by flow cytometry.</li>
<li>On day 2 after sgRNA transduction, monitor the levels of IL2Rγ chain on transduced NK-92 cells by flow cytometry (<a href="#fig1">Figures 1</a>G and 1H). Successful IL2R knockout is confirmed by the reduction in percentages of IL2Rγ sgRNA-RFP<sup>+</sup> Cas9-EGFP<sup>+</sup> cells over time as these cells fail to respond to IL-2 present in the culture medium. However, frequencies of control sgRNA-RFP<sup>+</sup> Cas9-EGFP<sup>+</sup> NK-92 cells do not reduce with time (<a href="#fig1">Figures 1</a>G and 1H), confirming that the transduction procedure itself doesn’t kill NK cells.</li>
</ol>
<h3 id="sec3.12">Generating dCas9-KRAB<sup>+</sup> NK-92 cells (CRISPRi NK-92 platform)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 7 weeks</div>
<p>The CRISPRi system employs a catalytically dead Cas9 (dCas9) without endonuclease activity fused to the KRAB transcriptional repressor (<a href="#bib18">Qi et al., 2013</a>). Specific sgRNAs allow the recruitment and binding of the dCas9-KRAB transcriptional repressor to target genes (<a href="#bib10">Gilbert et al., 2013</a>). The dCas9-KRAB-mediated transcriptional repression of genes is an alternative approach to the Cas9-mediated CRISPRko system. The dCas9-KRAB system mitigates the risk of off-target effects and exogenous DNA integration caused by the active Cas9 nuclease in the CRISPRko system (<a href="#bib30">Tycko et al., 2019</a>; <a href="#bib11">Hanlon et al., 2019</a>; <a href="#bib22">Saayman et al., 2016</a>).</p>
<p>Below, we detail the protocol for generating and validating NK-92 cells that constitutively express the dCas9-KRAB for CRISPRi-mediated transcriptional repression.</p>
<ol start="59">
<li>As with the generation of Cas9<sup>+</sup> NK-92 cells, resuspend ∼0.5–1.5 × 10<sup>6</sup> cells in 500 μL complete RPMI media + 500 μL concentrated dCas9-KRAB-GFP lentivirus + polybrene (10 μg/mL) + rhIL-2 (100 U/mL). Seed these cells in 24-well plate and incubate 37°C for 48 h.</li>
<li>After incubation, wash and resuspend in complete RPMI media supplemented with rhIL-2. Analyze percentages of dCas9-KRAB-GFP-transduced NK-92 cells using flow cytometry. Expand these cells to get enough cells for sorting with intermittent flow cytometry analysis of cells for GFP positivity (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1115-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Generation and validation of dCas9-KRAB<sup>+</sup> NK-92 cell line</div>
<p>(A and B) Flow cytometry dot plots showing (A) frequencies of dCas9-KRAB-GFP<sup>+</sup> NK-92 cells on days 2, 3, 8, and 22 after transduction with dCas9-KRAB-GFP lentivirus, and (B) purity of dCas9-KRAB-GFP-transduced NK-92 cells immediately after sorting (day 0) and after expanding the sorted cells for 10 days.</p>
<p>(C) Histogram overlays comparing untransduced and dCas9-KRAB-GFP-transduced NK-92 cells for changes in CFSE-Violet fluorescence with respect to the day the cells were labeled with CFSE (day 0).</p>
<p>(D) Flow cytometry dot plots of dCas9-KRAB-GFP<sup>+</sup> NK-92 single cell clones expanded by limiting dilution method.</p>
<p>(E) Specific cytotoxicities of untransduced NK-92, polyclonal, and monoclonal (clones 1 to 5) dCas9-KRAB-GFP<sup>+</sup> NK-92 cells. Error bars represent mean ± SEM of three technical replicates.</p>
<p>(F−G) Graphs showing fold change in frequencies of sgRNA-RFP<sup>+</sup>dCas9-KRAB-GFP<sup>+</sup> NK-92 cells with respect to day 1 after transduction with control sgRNA-RFP or IL-2RG sgRNA1-RFP or IL-2RG sgRNA2-RFP targeting IL2Rγ chain.</p>
</figcaption>
</figure>
</li>
<li>Stain the cells with DAPI and sort DAPI<sup>−</sup>GFP<sup>+</sup> cells, check post sort purity (<a href="#fig2">Figure 2</a>B).</li>
<li>Analyze proliferation potential (<a href="#fig2">Figure 2</a>C) and calculate the proliferation score (PS) as described previously (step 48) for Cas9<sup>+</sup> NK-92 cells. We found PS of dCas9-KRAB<sup>+</sup> NK-92 cells to be 5.4 which is less than 2-fold different from that of untransduced NK-92 cells (PS = 4.4). The comparable PS of untransduced and dCas9-KRAB<sup>+</sup> NK-92 cells suggest that constitutive expression of the dCas9-KRAB protein in NK-92 cells for CRISPRi does not abolish or markedly impair their natural proliferation.</li>
<li>Isolate single cell clones (<a href="#fig2">Figure 2</a>D) using the limiting dilution technique as described previously (step 49).</li>
</ol>
<h3 id="sec3.13">Selecting cytotoxic dCas9-KRAB<sup>+</sup> NK-92 single-cell clones for CRISPRi editing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 6 h</div>
<p>Because the dCas9-KRAB system uses catalytically dead Cas9 without endonuclease activity, we did not use the sgRNA targeting RFP that we previously used to measure Cas9 editing efficiency. Instead, we screened all dCas9-KRAB<sup>+</sup> NK-92 single cell clones that had expanded in culture for NK cytotoxicity by flow cytometry as described below.</p>
<ol start="64">
<li>Using a flow cytometry-based NK cytotoxicity assay (as described in steps 53–55), we measured the ability of untransduced and dCas9-KRAB<sup>+</sup> NK-92 cells to kill the standard NK-sensitive target human erythroleukemia cell line, K-562, as described earlier.</li>
<li>Among the Cas9-KRAB<sup>+</sup> NK-92 single cell clones, only clone-1 showed the highest cytotoxic activity against K-562 cells (<a href="#fig2">Figure 2</a>E). Therefore, only this clone was utilized for validation of the dCas9-KRAB system.</li>
</ol>
<h3 id="sec3.14">Validating transcriptional repression in dCas9-KRAB<sup>+</sup> NK-92 cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 days</div>
<p>We validated transcriptional repression in dCas9-KRAB in NK-92 cells using the same validation strategy as that described in Cas9<sup>+</sup> NK-92 cells, i.e., blocking the expression of the IL2Rγ chain required for NK-92 cell proliferation and survival, and examining the effects of this blockade on NK-92 cell turnover. The procedures for designing and cloning IL2Rγ sgRNAs, and that for producing IL2Rγ sgRNA lentiviruses have been described earlier (steps 1–43). Also, the procedure for sgRNA lentiviral transduction of dCas9-KRAB<sup>+</sup> NK-92 cells is identical to that outlined for Cas9<sup>+</sup> NK-92 cells (step 44). Starting on day 2 after transduction, we serially monitored the levels of IL2Rγ chain on transduced NK-92 cells by flow cytometry (<a href="#fig2">Figures 2</a>F and 2G). Successful IL2R knockout is confirmed by the reduction in percentages of IL2Rγ sgRNA-RFP<sup>+</sup> dCas9-KRAB-GFP<sup>+</sup> cells over time as these cells fail to respond to IL-2 present in the culture medium.</p>
<h3 id="sec3.15">Generating dCas9-VP64<sup>+</sup> NK-92 cells (CRISPRa NK-92 platform)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 7 weeks</div>
<p>The CRISPRa system employs a catalytically dead Cas9 (dCas9) without endonuclease activity fused to the VP64 transcriptional activator (<a href="#bib18">Qi et al., 2013</a>). Specific sgRNAs allow the recruitment and binding of the dCas9-VP64 transcriptional activator to target genes (<a href="#bib10">Gilbert et al., 2013</a>). Like the dCas9-KRAB system, the dCas9-VP64 system has reduced risk of off-target effects and exogenous DNA integration unlike the CRISPRko system (<a href="#bib30">Tycko et al., 2019</a>; <a href="#bib11">Hanlon et al., 2019</a>; <a href="#bib22">Saayman et al., 2016</a>). The CRISPRa approach to transcriptionally activate genes in human NK cells is superior to conventional lentiviral overexpression for two reasons: First, it is easier to introduce ∼20 nt single guide (sgRNAs) to simultaneously transcriptionally activate multiple genes by CRISPRa as opposed to sequentially transducing NK cells with large lentiviral overexpression vectors encoding the large open reading frame of each gene. Second, because transcript levels increase by ∼3-5-fold in CRISPRa as opposed to &gt;20-fold in conventional overexpression systems, cellular toxicity caused by excessive production of proteins can be avoided in CRISPRa.</p>
<p>Below, we detail the protocol for generating and validating NK-92 cells that constitutively express the dCas9-VP64 for CRISPRa-mediated transcriptional activation.</p>
<ol start="66">
<li>As with the generation of Cas9<sup>+</sup> and dCAS9-KRAB<sup>+</sup> NK-92 cells, count NK-92 cells, resuspend ∼0.5–1.5 × 10<sup>6</sup> cells in 500 μL of complete RPMI media in a 1.7 mL microtube, and add 500 μL of concentrated dCas9-VP64-GFP lentivirus. Now the total volume will be 1 mL. To this, add polybrene (10 μg/mL) and rhIL-2 (100 U/mL). Seed these cells in a 24-well plate and incubate at 37°C for 48 h.</li>
<li>On day 2, wash and then replenish the cells with complete RPMI media + rhIL2 (100 U/mL). Take a small aliquot of cells to measure the percentages of GFP<sup>+</sup> dCas9-VP64-transduced NK-92 cells using flow cytometry. We found that the percentages of dCas9-VP64-transduced NK-92 cells were 14.8% at day 2 and 14.5% at day 11 after expansion (<a href="#fig3">Figure 3</a>A).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1115-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Generation and validation of dCas9-VP64<sup>+</sup> NK-92 cell line</div>
<p>(A and B) Flow cytometry dot plots showing (A) frequencies of dCas9-VP64-GFP<sup>+</sup> NK-92 cells on days 2 (upper panel) and 11 (lower panel) after transduction with dCas9-VP64-GFP lentivirus, and (B) purity of dCas9-VP64-GFP-transduced NK-92 cells immediately after sorting (day 0) and after expanding the sorted cells for 10 days.</p>
<p>(C) Histogram overlays comparing untransduced and dCas9-VP64-GFP-transduced NK-92 cells for changes in CFSE-Violet fluorescence with respect to the day the cells were labeled with CFSE (day 0).</p>
<p>(D) Flow cytometry of dCas9-VP64-GFP<sup>+</sup> NK-92 single cell clones expanded by limiting dilution method.</p>
<p>(E) Specific cytotoxicities of untransduced NK-92, polyclonal, and monoclonal (clone-3) dCas9-VP64-GFP<sup>+</sup> NK-92 cells. Error bars represent mean ± SEM of three technical replicates.</p>
<p>(F) Flow cytometry dot plots showing the percentages of dCas9-VP64-GFP<sup>+</sup> NK-92 cells transduced with control sgRNA-RFP or IL-15 sgRNA-RFP on days 2 (upper panel) and 15 (lower panel) after transduction and puromycin selection.</p>
<p>(G) Fold change in IL-15 mRNA expression levels by qPCR in sgRNA-RFP<sup>+</sup> dCas9-VP64-GFP<sup>+</sup> NK-92 cells on day 15 after being transduced with control sgRNA-RFP or IL-15 sgRNA-RFP. Error bars represent mean ± SEM of three technical replicates, Exact p-value was calculated by Student's t-test.</p>
</figcaption>
</figure>
</li>
<li>Stain the cells with DAPI and sort DAPI<sup>−</sup>GFP<sup>+</sup> NK-92 cells by FACS. Check the post-sort purity by flow cytometry. We found that post-sort purities of dCas9-VP64-GFP<sup>+</sup> NK-92 cells were 99.8% at day 0 and 96.9% at day 10 after expansion (<a href="#fig3">Figure 3</a>B).</li>
<li>Compare the proliferation of dCas9-VP64<sup>+</sup> NK-92 cells and that of untransduced NK-92 cells (<a href="#fig3">Figure 3</a>C) by measuring the proliferation score (PS) using the formula previously described (step 48) for Cas9<sup>+</sup> and dCAS9-KRAB<sup>+</sup> NK-92 cells. We found PS of dCas9-VP64<sup>+</sup> NK-92 cells to be 3.4 which is less than 2-fold different from that of untransduced NK-92 cells (PS=4.4). The comparable PS of untransduced and dCas9-VP64<sup>+</sup> NK-92 cells suggest that constitutive expression of the dCas9-VP64 protein in NK-92 cells for CRISPRa does not abolish or markedly impair their natural proliferation.</li>
<li>Isolate the single cell clones of dCas9-VP64<sup>+</sup> NK-92 cells (<a href="#fig3">Figure 3</a>D) using the limiting dilution procedure previously described (step 49) for Cas9<sup>+</sup> and dCas9-KRAB<sup>+</sup> NK-92 cells in <a href="#fig1">Figures 1</a>D and <a href="#fig2">2</a>D.</li>
</ol>
<h3 id="sec3.16">Selecting cytotoxic dCas9-VP64<sup>+</sup> NK-92 single-cell clones for CRISPRa editing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 6 h</div>
<p>Because the dCas9-VP64 system uses catalytically dead Cas9 without endonuclease activity, we did not use the sgRNA targeting RFP that we previously used to measure Cas9 editing efficiency. Instead, we directly screened dCas9-VP64<sup>+</sup> NK-92 single cell clones that had expanded in culture for NK cytotoxicity by flow cytometry. Of note, all dCas9-VP64<sup>+</sup> NK-92 cells except clone-3 had contamination from GFP negative (dCas9-VP64<sup>−</sup>) NK-92 cells (<a href="#fig3">Figure 3</a>D). Therefore, the cytotoxicity assay was performed only using clone-3 of dCas9-VP64<sup>+</sup> NK-92 cells (<a href="#fig3">Figure 3</a>E). The procedure for performing cytotoxicity assays is similar to that described earlier (steps 53–55) for Cas9<sup>+</sup> and dCas9-KRAB<sup>+</sup> NK-92 cells.</p>
<h3 id="sec3.17">Validating transcriptional activation in dCas9-VP64<sup>+</sup> NK-92 cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 2–4 weeks</div>
<p>To validate dCas9-VP64<sup>+</sup> NK-92 cells, we transcriptionally activated the IL-15 gene which is required for not just survival and proliferation but also the cytotoxic function of NK cells (<a href="#bib7">Cooper et al., 2002</a>; <a href="#bib36">Zhang et al., 2018</a>). Alternately, one could transcriptionally activate IL-2.</p>
<p>The procedures for designing and cloning IL-15 sgRNA, and that for producing IL2Rγ sgRNA lentiviruses have been described earlier.</p>
<ol start="71">
<li>To validate transcriptional activation of IL-15 gene in dCas9-VP64<sup>+</sup> NK-92 cells:
<ol type="a">
<li>Resuspend 1 × 10<sup>6</sup> dCas9-VP64-GFP<sup>+</sup> NK-92 cells in 1000 μL of complete RPMI media in 1.7 mL microtube.</li>
<li>Divide the cell suspension into two parts and transfer to 1.7 mL microtubes.</li>
<li>Add 500 μL of concentrated lentivirus for control sgRNA-RFP or IL-15 sgRNA-RFP targeting the IL-15 gene.</li>
<li>Now the total volume in each tube will be 1 mL. To these tubes, add polybrene (10 μg/mL) and rhIL-2 (100 U/mL).</li>
<li>Seed these cells in a 24-well plate (0.5 × 10<sup>6</sup> cells/well) and incubate at 37°C for 48 h. See “<a href="#undtbl7">Single guide RNA (sg) target sequences</a>” table in <a href="#materials-and-equipment">materials and equipment</a> section for sequences of control sgRNA or IL-15 sgRNA used for CRISPRa.</li>
</ol>
</li>
<li>On day 2, before puromycin selection, take a small aliquot of transduced cells and analyze the frequencies of dCas9-VP64-GFP<sup>+</sup> sgRNA-RFP<sup>+</sup> transduced cells by flow cytometry (<a href="#fig3">Figure 3</a>F upper panel).</li>
<li>Resuspend the remaining cells in NK cell media containing puromycin (1 μg/mL) and select these cells in puromycin for ∼ 2–4 weeks until frequencies of dCas9-VP64-GFP<sup>+</sup> sgRNA-RFP<sup>+</sup> cells in the control sgRNA and IL-15 sgRNA groups reaches &gt;95% (<a href="#fig3">Figure 3</a>F lower panel).</li>
</ol>
<div>Alternative Method: The time period of puromycin selection may be prolonged and/or selection may be difficult if the gene modulated negatively impacts NK cell turnover. For such sgRNAs, we recommend flow-based sorting of dCas9-VP64-GFP <sup>+</sup> sgRNA-RFP<sup>+</sup> NK cells.</div>
<ol start="74">
<li>Proceed with RNA isolation (NucleoSpin RNA purification kit # 740955.250), cDNA synthesis (SuperScript IV First Strand Synthesis System # 18091050) and quantitate the mRNA expression of IL-15 in cells transduced with control sgRNA or IL-15 sgRNA by quantitative real-time PCR (<a href="#fig3">Figure 3</a>G).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol sequentially details the steps for genetically manipulating NK-92 cells using Cas9 mediated gene knockout as well as dCas9-mediated transcriptional repression and activation of genes. We also provide optimized and working sgRNA sequences which can be used by others to functionally validate the successful establishment of the Cas9<sup>+</sup> and dCas9<sup>+</sup> NK-92 cell lines. The effects of targeting specific genes in human NK cells on homeostasis, function, and therapeutic potential of NK cells can be studied using our Cas9<sup>+</sup> and dCas9<sup>+</sup> NK-92 platforms. We anticipate that our protocols can be extended to modify human NK cells derived from cord blood or peripheral blood of healthy donors using CRISPR for developing off-the-shelf NK cell-based therapies (<a href="#bib27">Suck et al., 2016</a>).</p>
<p>Our data suggest that the five major steps of the protocol, namely, transduction of NK cells with Cas9/dCas9, sorting of Cas9/dCas9<sup>+</sup> NK cells, single cell cloning of sorted Cas9/dCas9<sup>+</sup> NK cells, confirmation of cytotoxic function, and sgRNA-based validation of the CRISPR-engineered NK cells are feasible. However, at each of these steps, it is likely that the desired outcome may not be obtained due to technical difficulties. Therefore, in the troubleshooting section, we outline potential pitfalls and provide solutions to ensure that each of the steps leading to generation and validation of CRISPR-engineered NK cells is successful.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Before proceeding with CRIPSR-mediated target gene modification in Cas9<sup>+</sup> and dCas9<sup>+</sup> NK-92 cells, we ensured that transduction with Cas9 or dCas9 did not impair NK cell viability, proliferation, and cytotoxicity. Only single cell Cas9<sup>+</sup> and dCas9<sup>+</sup> NK-92 clones without disruptions in normal NK homeostasis were selected for functional validation and will be used by us in future experiments. For manipulating a given gene in Cas9<sup>+</sup> and dCas9<sup>+</sup> NK-92 cells during functional validation of the NK cell platforms, we selected two top-scoring sgRNAs with low off-target effects using the Broad Institute Genetic Perturbation Platform (GPP) web portal (<a href="#bib8">Doench et al., 2016</a>). We tested the abilities of each of these sgRNAs to knockout (Cas9), repress transcription (dCas9-KRAB), and activate transcription (dCas9-VP64) of genes in NK-92 cells. We found that for the Cas9 and dCas9-KRAB systems, both top scoring sgRNAs had identical effects. However, in the dCas9-VP64 system, only one sgRNA was successful in transcriptionally activating the target gene, and is shown. For experiments aimed at modulating specific genes of interest in these platforms, we recommend testing at least 2 sgRNAs per gene, as we demonstrate. Cytotoxicity assays and assays for functional validation of the Cas9 and dCas9 systems were conducted in three technical replicates per sample. P-values have been calculated using the Student’s t-test in GraphPad software. The level of statistical significance for all experiments has been set at P&lt;0.05.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>As innate immune cells, NK cells are generally refractory to viral transductions (<a href="#bib23">Schmidt et al., 2021</a>). For this reason, other groups have previously electroporated the Cas9 ribonucleoprotein complex into primary human NK cells (<a href="#bib17">Pomeroy et al., 2020</a>; <a href="#bib19">Rautela et al., 2020</a>; <a href="#bib2">Berrien-Elliott et al., 2020</a>). While electroporation is suitable for knocking out genes where transient expression of Cas9 protein is sufficient, dCas9-KRAB-mediated transcriptional repression and dCas9-VP64-mediated transcriptional activation require constitutive expression of dCas9-KRAB and dCas9-VP64, respectively. Therefore, NK-92 lymphoblasts, which are easier to manipulate using lentiviruses than normal NK cells, were used. Furthermore, because NK-92 cell-based therapies are currently in clinical trials for both hematological malignancies and solid tumors ((<a href="#bib16">Li et al., 2015</a>; <a href="#bib28">Tang et al., 2018</a>; <a href="#bib6">Chu et al., 2014</a>; <a href="#bib20">Rezvani and Rouce, 2015</a>; <a href="#bib14">Klingemann et al., 2016</a>), <a href="https://immunitybio.com/pipeline/">https://immunitybio.com/pipeline/</a>), we anticipate that our protocols for CRISPR-based editing of NK-92 cells will have clinical applications.</p>
<p>Although our protocol details the steps for engineering NK-92 cells using CRISPR, it can be extended to other malignant NK cell lines including NKL (<a href="#bib21">Robertson et al., 1996</a>), and NK101 (<a href="#bib34">Yang et al., 2019</a>) which are also being used in research and clinical applications. In the future, we intend to genetically manipulate primary human NK cells derived from cord blood or from peripheral blood of healthy donors using the above-described CRISPR-based methods. For modifying healthy human NK cells, we acknowledge that our current protocol may have to be modified because normal NK cells are more difficult to transduce than NK cell lines (<a href="#bib23">Schmidt et al., 2021</a>) . At present, it is difficult to predict the extent of modification necessary to constitutively edit healthy NK cells for CRISPR-based applications. However, we expect the sequence of steps to constitutively modify healthy NK cells for CRISPR-based applications to be identical to that described here for NK-92.</p>
<p>Because we used fresh lentiviral supernatant for transducing NK-92 without determining the lentiviral titer, frequencies of transduced cells vary in each experiment. Unfortunately, determining lentivirus titer using permissive cell lines including HEK cells does not directly translate to innate immune cells such as NK cells. Also, we do not recommend using MOI for estimating efficiencies of Cas9- and dCas9- transductions in NK-92 cells for two reasons:</p>
<p>First, increasing the MOI does not significantly improve transduction efficiencies of human NK cells (<a href="#bib26">Su et al., 2011</a>; <a href="#bib19">Rautela et al., 2020</a>), suggesting that MOI is not a reliable measure of estimating NK transduction efficiencies.</p>
<p>Second, a more recent study suggested that transduction efficiencies of cells in culture cannot be accurately estimated using MOI because cell cultures are heterogenous with some cells being more resistant than others to transduction and because virions may not contact each cell with the same efficiency. Furthermore, when a virus particle interacts with and infects the cell, it is a discrete event. Even at MOI=1, the same cell may become infected many times. Therefore, MOI can only be applied to a homogeneous cell population with each cell having identical transduction efficiency (<a href="#bib24">Shabram and Aguilar-Cordova, 2000</a>), which is not the case with NK cells cultures. In our experiments, this situation is further complicated by the general resistance of all human NK cells to viral transduction (<a href="#bib23">Schmidt et al., 2021</a>).</p>
<p>Instead of using MOI, we therefore recommend users of our protocol to apply 1% of transduced cells (on day 2 post infection), as the minimum cut off in Cas9 and dCas9 transduction experiments to facilitate expansion and sorting of these cells for next steps in CRISPR engineering.</p>
<p>The constitutive expression of the active Cas9 nuclease in NK-92 cells in the CRISPRko system may lead to off-target and deleterious genomic alterations in the NK cells. There are several methods to mitigate this potential problem. One method is to use electroporation of the Cas9 ribonucleoprotein (RNP) which will lower the time frame for which active Cas9 is present in the NK cell, thereby reducing the risk of off-target genetic alterations (<a href="#bib12">Hildreth et al., 2020</a>).</p>
<p>A better solution which avoids the use of Cas9 but achieves the same effect is the dCas9-KRAB system (<a href="#bib10">Gilbert et al., 2013</a>). In this system, as Cas9 is catalytically dead and the specificity of the gene targeted for transcriptional repression is decided by the sgRNA sequence (<a href="#bib18">Qi et al., 2013</a>), the off-target toxicity of Cas9 is drastically reduced. Furthermore, unlike the Cas9 system which knocks out the entire protein, the sgRNAs in the dCas9-KRAB system can be tailored to modulate the level of transcriptional repression of a target gene. Such downmodulation is especially important while engineering NK cells for therapeutic applications where complete knockout of a given gene using Cas9 may be detrimental to the NK cell.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>No transduction or low frequencies of Cas9- and dCas9- transduced NK-92 cells (step 44)</p>
<h3 id="sec7.2">Potential solutions</h3>
<p>Because NK cells are innate immune cells, they are difficult to transduce with lentiviruses (<a href="#bib23">Schmidt et al., 2021</a>). We found that low levels of transduction (&lt;1%) are mostly attributed to the lentivirus quality/titer. Potential solutions for this problem include:</p>
<ol type="a">
<li>making a fresh lot of virus, and on day 4 of the virus production, avoid filtering the lentiviral supernatant but rather centrifuge (300 × <i>g</i> for 5 min) it to remove debris before concentrating the virus.</li>
<li>at the final step of virus concentration on day 5, using 500 μL of fresh viral supernatant to transduce the NK-92 cells as freeze thawing of viral supernatant can affect the lentivirus quality and titer.</li>
<li>tracking the cells with low transduction efficiencies for at least a week. In some cases, although the efficiency of transduction is initially low, after a week the frequencies of GFP<sup>+</sup> transduced cells were found to be increased (<a href="#fig2">Figure 2</a>A).</li>
</ol>
<p>If none of the above potential solutions work, the only option is to restart the experiment from the first stage of virus production.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Transduction with Cas9 or dCas9 slows down or blocks the proliferation of NK cells (step 48)</p>
<h3 id="sec7.4">Potential solutions</h3>
<p>We suggest the following solutions to overcome the potential pitfall 2.</p>
<ol type="a">
<li>One must ensure that the NK-92 cell growth media is being supplemented with rhIL-2 (100 U/mL).</li>
<li>Because NK-92 cell proliferation is affected by the seeding density at the time of culture initiation, we suggest growing the post-sort Cas9<sup>+</sup> or dCas9<sup>+</sup> NK-92 cells in 96-well plates at high densities (∼0.5 × 10<sup>6</sup>/well) and transferring cells to 48-well plates and finally into T-25 flasks when these cells begin proliferating. If this method doesn’t work, one would infer that the reduced proliferation of transduced NK-92 cells may be caused by lentiviral integrations that disrupt genes involved in NK-92 cell proliferation. The only recourse in this case would be to restart the experiment.</li>
</ol>
<p>Depending on the type of application, we recommend the users to either proceed further after checking NK-92 proliferation or troubleshoot. For example, proliferation may be less critical for clinical applications where NK cytotoxicity is the most important and NK cells are irradiated before use (<a href="#bib14">Klingemann et al., 2016</a>; <a href="#bib28">Tang et al., 2018</a>).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Contamination of sorted and expanding Cas9<sup>+</sup> and dCas9<sup>+</sup> single cell clones with untransduced GFP<sup>-</sup> cells (step 49)</p>
<h3 id="sec7.6">Potential solutions</h3>
<p>FACS-based purification protocols do not lead to the isolation of the desired populations at 100% purity. For example, we observed an outgrowth of DAPI<sup>−</sup>GFP<sup>−</sup> cells after expanding some of the sorted dCas9-VP64<sup>+</sup> single cell clones (<a href="#fig3">Figure 3</a>E). We propose two potential solutions for this problem:</p>
<ol type="a">
<li>Sorting polyclonal dCas9-VP64<sup>+</sup> to remove DAPI<sup>-</sup>GFP<sup>-</sup> cells and then performing single cell cloning.</li>
<li>Seeding multiple 96 well plates with single cell clones to get at least ∼5–10 pure single cell colonies of dCas9-VP64<sup>+</sup> NK-92 cells.</li>
</ol>
<h3 id="sec7.7">Problem 4</h3>
<p>None of the selected clones show cytotoxicity at levels comparable to that of unmodified NK-92 cells (steps 53–55)</p>
<h3 id="sec7.8">Potential solutions</h3>
<p>We suggest seeding multiple (∼5–10) single cells clones to increase the chances of getting at least one Cas9/dCas9<sup>+</sup> NK cell clone with cytotoxicity comparable to that of untransduced NK-92 cells. We have observed that cytotoxicity varies at a clonal level, i.e., a Cas9/dCas9-transduced polyclonal NK population with cytotoxicity comparable to that of untransduced cells does not guarantee that single cells clones will be equally cytotoxic and vice-versa. For example, in <a href="#fig2">Figure 2</a>E we observe that polyclonal dCas9-KRAB<sup>+</sup> NK-92 cells have lower cytotoxicity whereas clone-1 of same population showed cytotoxicity at levels similar to untransduced NK-92 cells.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Transduction with the sgRNA does not yield the desired effect, i.e., knockout or transcriptional activation or transcriptional repression (steps 58,74)</p>
<h3 id="sec7.10">Potential solutions</h3>
<p>Selection of ∼2–5 sgRNAs per gene using the Broad Institute Genetic Perturbation Platform (<a href="#bib8">Doench et al., 2016</a>) increases the likelihood of finding the best working sgRNA which yields the desired gene regulation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ahlen, M.T., Husebekk, A., Killie, M.K., Skogen, B., and Stuge, T.B. (2009). T-cell responses associated with neonatal alloimmune thrombocytopenia: isolation of HPA-1a–specific, HLA-DRB3∗ 0101–restricted CD4+ T cells. J. Am. Soc. Hematol. <i>113</i>, 3838-3844. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref1">View at publisher</a></p>
<p id="bib2">Berrien-Elliott, M.M., Cashen, A.F., Cubitt, C.C., Neal, C.C., Wong, P., Wagner, J.A., Foster, M., Schappe, T., Desai, S., and McClain, E.J. (2020). Multidimensional analyses of donor memory-like NK cells reveal new associations with response after adoptive immunotherapy for leukemia. Cancer Discov. <i>10</i>, 1854-1871. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref2">View at publisher</a></p>
<p id="bib3">Carson, W.E., Fehniger, T.A., Haldar, S., Eckhert, K., Lindemann, M.J., Lai, C.-F., Croce, C.M., Baumann, H., and Caligiuri, M.A. (1997). A potential role for interleukin-15 in the regulation of human natural killer cell survival. J. Clin. Invest. <i>99</i>, 937-943. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref3">View at publisher</a></p>
<p id="bib4">Cerwenka, A. and Lanier, L.L. (2001). Natural killer cells, viruses and cancer. Nat. Rev. Immunol. <i>1</i>, 41-49. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref4">View at publisher</a></p>
<p id="bib5">Chavez, A., Scheiman, J., Vora, S., Pruitt, B.W., Tuttle, M., Iyer, E.P., Lin, S., Kiani, S., Guzman, C.D., Wiegand, D.J., et al. (2015). Highly efficient Cas9-mediated transcriptional programming. Nat. Methods <i>12</i>, 326-328. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref5">View at publisher</a></p>
<p id="bib6">Chu, J., Deng, Y., Benson, D.M., He, S., Hughes, T., Zhang, J., Peng, Y., Mao, H., Yi, L., Ghoshal, K., et al. (2014). CS1-specific chimeric antigen receptor (CAR)-engineered natural killer cells enhance in vitro and in vivo antitumor activity against human multiple myeloma. Leukemia <i>28</i>, 917-927. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref6">View at publisher</a></p>
<p id="bib7">Cooper, M.A., Bush, J.E., Fehniger, T.A., Vandeusen, J.B., Waite, R.E., Liu, Y., Aguila, H.L., and Caligiuri, M.A. (2002). In vivo evidence for a dependence on interleukin 15 for survival of natural killer cells. J. Am. Soc. Hematol. <i>100</i>, 3633-3638. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref7">View at publisher</a></p>
<p id="bib8">Doench, J.G., Fusi, N., Sullender, M., Hegde, M., Vaimberg, E.W., Donovan, K.F., Smith, I., Tothova, Z., Wilen, C., Orchard, R., et al. (2016). Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9. Nat. Biotechnol. <i>34</i>, 184-191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref8">View at publisher</a></p>
<p id="bib9">Duault, C., Kumar, A., Khani, A.T., Lee, S.J., Yang, L., Huang, M., Hurtz, C., Manning, B., Ghoda, L., McDonald, T.J., et al. (2021). Activated natural killer cells predict poor clinical prognosis in high-risk B-and T-cell acute lymphoblastic leukemia. Blood. . <a class="external-link" href="https://doi.org/10.1182/blood.2020009871">https://doi.org/10.1182/blood.2020009871</a></p>
<p id="bib10">Gilbert, L.A., Larson, M.H., Morsut, L., Liu, Z., Brar, G.A., Torres, S.E., Stern-Ginossar, N., Brandman, O., Whitehead, E.H., Doudna, J.A., et al. (2013). CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes. Cell <i>154</i>, 442-451. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref10">View at publisher</a></p>
<p id="bib11">Hanlon, K.S., Kleinstiver, B.P., Garcia, S.P., Zaborowski, M.P., Volak, A., Spirig, S.E., Muller, A., Sousa, A.A., Tsai, S.Q., Bengtsson, N.E., et al. (2019). High levels of AAV vector integration into CRISPR-induced DNA breaks. Nat. Commun. <i>10</i>, 1-11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref11">View at publisher</a></p>
<p id="bib12">Hildreth, A.D., Riggan, L., and O’Sullivan, T.E. (2020). CRISPR-Cas9 ribonucleoprotein-mediated genomic editing in primary innate immune cells. STAR Protoc. <i>1</i>, 100113. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref12">View at publisher</a></p>
<p id="bib13">Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J.A., and Charpentier, E. (2012). A programmable dual-RNA–guided DNA endonuclease in adaptive bacterial immunity. Science <i>337</i>, 816-821. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref13">View at publisher</a></p>
<p id="bib14">Klingemann, H., Boissel, L., and Toneguzzo, F. (2016). Natural killer cells for immunotherapy–advantages of the NK-92 cell line over blood NK cells. Front. Immunol. <i>7</i>, 91. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref14">View at publisher</a></p>
<p id="bib15">Kooreman, N.G., De Almeida, P.E., Stack, J.P., Nelakanti, R.V., Diecke, S., Shao, N.-Y., Swijnenburg, R.-J., Sanchez-Freire, V., Matsa, E., Liu, C., et al. (2017). Alloimmune responses of humanized mice to human pluripotent stem cell therapeutics. Cell Rep <i>20</i>, 1978-1990. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref15">View at publisher</a></p>
<p id="bib16">Li, Y., Yin, J., Li, T., Huang, S., Yan, H., Leavenworth, J., and Wang, X. (2015). NK cell-based cancer immunotherapy: from basic biology to clinical application. Sci. China Life Sci. <i>58</i>, 1233-1245. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref16">View at publisher</a></p>
<p id="bib17">Pomeroy, E.J., Hunzeker, J.T., Kluesner, M.G., Lahr, W.S., Smeester, B.A., Crosby, M.R., Lonetree, C.-L., Yamamoto, K., Bendzick, L., Miller, J.S., et al. (2020). A genetically engineered primary human natural killer cell platform for cancer immunotherapy. Mol. Ther. <i>28</i>, 52-63. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref17">View at publisher</a></p>
<p id="bib18">Qi, L.S., Larson, M.H., Gilbert, L.A., Doudna, J.A., Weissman, J.S., Arkin, A.P., and Lim, W.A. (2013). Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression. Cell <i>152</i>, 1173-1183. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref18">View at publisher</a></p>
<p id="bib19">Rautela, J., Surgenor, E., and Huntington, N.D. (2020). Drug target validation in primary human natural killer cells using CRISPR RNP. J. Leukoc. Biol. <i>108</i>, 1397-1408. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref19">View at publisher</a></p>
<p id="bib20">Rezvani, K. and Rouce, R.H. (2015). The application of natural killer cell immunotherapy for the treatment of cancer. Front. Immunol. <i>6</i>, 578. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref20">View at publisher</a></p>
<p id="bib21">Robertson, M., Cochran, K., Cameron, C., Le, J., Tantravahi, R., and Ritz, J. (1996). Characterization of a cell line, NKL, derived from an aggressive human natural killer cell leukemia. Exp. Hematol. <i>24</i>, 406-415. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref21">View at publisher</a></p>
<p id="bib22">Saayman, S.M., Lazar, D.C., Scott, T.A., Hart, J.R., Takahashi, M., Burnett, J.C., Planelles, V., Morris, K.V., and Weinberg, M.S. (2016). Potent and targeted activation of latent HIV-1 using the CRISPR/dCas9 activator complex. Mol. Ther. <i>24</i>, 488-498. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref22">View at publisher</a></p>
<p id="bib23">Schmidt, P., Raftery, M.J., and Pecher, G. (2021). Engineering NK cells for CAR therapy—recent advances in gene transfer methodology. Front. Immunol. <i>11</i>, 3404. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref23">View at publisher</a></p>
<p id="bib24">Shabram, P. and Aguilar-Cordova, E.J. (2000). Multiplicity of infection/multiplicity of confusion. Mol. Ther. <i>2</i>, 420-421. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref24">View at publisher</a></p>
<p id="bib25">Strauss-Albee, D.M. and Blish, C.A. (2016). Human NK cell diversity in viral infection: ramifications of ramification. Front. Immunol. <i>7</i>, 66. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref25">View at publisher</a></p>
<p id="bib26">Su, S., Betters, D.M., Ramanathan, M., Keyvanfar, K., Smith, A., Feng, X., Furutani, E., Carlsten, M., Lundqvist, A., and Childs, R. (2011). Optimizing lentiviral transduction of human natural killer cells. Am. Soc. Hematol. <i>118</i>, 4714. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref26">View at publisher</a></p>
<p id="bib27">Suck, G., Odendahl, M., Nowakowska, P., Seidl, C., Wels, W.S., Klingemann, H.G., and Tonn, T. (2016). NK-92: an ‘off-the-shelf therapeutic’for adoptive natural killer cell-based cancer immunotherapy. Cancer Immunol. Immunother. <i>65</i>, 485-492. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref27">View at publisher</a></p>
<p id="bib28">Tang, X., Yang, L., Li, Z., Nalin, A.P., Dai, H., Xu, T., Yin, J., You, F., Zhu, M., Shen, W., et al. (2018). First-in-man clinical trial of CAR NK-92 cells: safety test of CD33-CAR NK-92 cells in patients with relapsed and refractory acute myeloid leukemia. Am. J. Cancer Res. <i>8</i>, 1083. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref28">View at publisher</a></p>
<p id="bib29">Törnroos, H., Hägerstrand, H., and Lindqvist, C. (2019). Culturing the human natural killer cell line NK-92 in interleukin-2 and interleukin-15–Implications for clinical trials. Anticancer Res. <i>39</i>, 107-112. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref29">View at publisher</a></p>
<p id="bib30">Tycko, J., Wainberg, M., Marinov, G.K., Ursu, O., Hess, G.T., Ego, B.K., Li, A., Truong, A., Trevino, A.E., Spees, K., et al. (2019). Mitigation of off-target toxicity in CRISPR-Cas9 screens for essential non-coding elements. Nat. Commun. <i>10</i>, 1-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref30">View at publisher</a></p>
<p id="bib31">Waickman, A.T., Park, J.-Y., and Park, J.-H. (2016). The common γ-chain cytokine receptor: tricks-and-treats for T cells. Cell Mol Life Sci <i>73</i>, 253-269. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref31">View at publisher</a></p>
<p id="bib32">Wang, T., Wei, J.J., Sabatini, D.M., and Lander, E.S. (2014). Genetic screens in human cells using the CRISPR-Cas9 system. Science <i>343</i>, 80-84. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref32">View at publisher</a></p>
<p id="bib33">Wilk, A.J., Lee, M.J., Wei, B., Parks, B., Pi, R., Martínez-Colón, G.J., Ranganath, T., Zhao, N.Q., Taylor, S., Becker, W., et al. (2021). Multi-omic profiling reveals widespread dysregulation of innate immunity and hematopoiesis in COVID-19. J. Exp. Med. <i>218</i>, e20210582. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref33">View at publisher</a></p>
<p id="bib34">Yang, H.G., Kang, M.C., Kim, T.Y., Hwang, I., Jin, H.T., Sung, Y.C., Eom, K.-S., and Kim, S.W. (2019). Discovery of a novel natural killer cell line with distinct immunostimulatory and proliferative potential as an alternative platform for cancer immunotherapy. J. Immunother. Cancer <i>7</i>, 1-17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref34">View at publisher</a></p>
<p id="bib35">Yang, L., Chan, A., Miyashita, K., Delaney, C., Wang, X., Li, H., Pokharel, S., Li, S., Li, M., Xu, X., et al. (2021). High-resolution characterization of gene function using single-cell CRISPR tiling screen. Nat. Commun. <i>12</i>, 4063. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref35">View at publisher</a></p>
<p id="bib36">Zhang, M., Wen, B., Anton, O.M., Yao, Z., Dubois, S., Ju, W., Sato, N., Dilillo, D.J., Bamford, R.N., Ravetch, J.V., et al. (2018). IL-15 enhanced antibody-dependent cellular cytotoxicity mediated by NK cells and macrophages. Proc. Natl. Acad. Sci. U S A <i>115</i>, E10915-E10924. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00580-3/sref36">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Srividya Swaminathan (<a href="mailto:sswaminathan@coh.org">sswaminathan@coh.org</a>).</p>
<h4>Materials availability</h4>
<p>All plasmids described in this study have been deposited in Addgene. The NK-92, K-562 and Lenti-X 293T cell lines were obtained from American Type Culture Collection and Takara Bio respectively. The research described in this manuscript is covered by a pending US patent application.</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze any code or datasets.</p>
<h3>Acknowledgments</h3>
<p>This study was supported by the following awards to S.S.: Special Fellow Award from the Leukemia and Society (LLS 3366-17), <a href="https://doi.org/10.13039/100001422">American Society of Hematology</a> Scholar Award, <a href="https://doi.org/10.13039/100001797">PhRMA Foundation</a> Research Starter Grant in Drug Discovery (2021 RSGDS 730217), Toni Stephenson Lymphoma Center Pilot Award, P50 CA107399-12 <a href="https://doi.org/10.13039/100000054">National Cancer Institute</a> Lymphoma SPORE Career Enhancement Pilot Award (PIs: Stephen J. Forman, Larry W. Kwak), and Research Start-Up from City of Hope. C.-W.C is funded by <a href="https://doi.org/10.13039/100001422">American Society of Hematology</a> Scholar Award, Alex’s Lemonade Stand Foundation Innovation Award, Stand Up To Cancer &amp; <a href="https://doi.org/10.13039/501100000289">Cancer Research UK</a> Pediatric Cancer New Discoveries Challenge Award, and <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> Grants CA197498, CA233691, CA236626, CA233922, and CA243124.</p>
<h3>Author contributions</h3>
<p>A.K. performed most of the experiments, designed all experiments, and wrote the manuscript. S.J.L. assisted A.K. with experiments. Q.L. A.K.N.C., and S.P.P. provided reagents and technical expertise on CRISPR. J.Y. provided scientific expertise on NK cell biology. C.-W.C provided scientific expertise in CRISPR. S.S. conceived and supervised the study and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The research described in this manuscript is covered by a pending US patent application.</p>
</section>
</article>