<article>
<section>
<h1 id="header">Preparation of single cell suspensions enriched in mouse brain vascular cells for single-cell RNA sequencing</h1>
<p><time datetime="2021-08-06">Published: August 6, 2021</time></p>
<p>Akari Yamazaki,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Francis Shue,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a></sup> Yu Yamazaki,<sup><a href="#aff1">1</a></sup> Yuka A. Martens,<sup><a href="#aff1">1</a></sup> Guojun Bu,<sup><a href="#aff1">1</a>,<a href="#cor1">*</a></sup> and Chia-Chen Liu<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Neuroscience, Mayo Clinic, Jacksonville, FL 32224, USA</p>
<p id="aff2"><sup>2</sup>Neuroscience Graduate Program, Mayo Clinic, Jacksonville, FL 32224, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:Bu.guojun@mayo.edu">Bu.guojun@mayo.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:Liu.chiachen@mayo.edu">Liu.chiachen@mayo.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100715">10.1016/j.xpro.2021.100715</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cerebral blood vessels supply oxygen and nutrients, remove metabolic waste, and play a critical role in maintaining brain homeostasis. Cerebrovasculature is composed of heterogeneous populations of brain vascular cells (BVCs). A major challenge in effective cerebrovascular transcriptional profiling is high-quality BVC procurement, permitting high sequencing depth. Here, we establish cell isolation procedures for glio-vascular cell-enriched single-cell RNA sequencing enabling unbiased characterization of BVC transcriptional heterogeneity. Our approach can be used to address vascular-specific contribution to brain diseases.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib9">Yamazaki et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Isolation of brain vascular enriched cells suitable for single-cell RNA sequencing</li>
<li>Validation of brain vascular cell enrichment in cell isolates</li>
<li>Confirmation of highly viable single cells after isolation</li>
<li>Protocol is adapted for adult mouse brain tissues</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This vascular enriched single cell suspension protocol is adapted from a previous protocol with modifications (<a href="#bib7">Vanlandewijck et al., 2018</a>). This procedure has been established and optimized for isolation of glio-vascular cells (i.e., endothelial cells, smooth muscle cells, pericytes, and astrocytes) from cortical tissues of adult mice (6–12 months old). The enzymatic tissue digestion followed by the depletion of CD11b-positive cells, and myelin and dead cell removals allow greater representation of vascular cell populations in single-cell RNA sequencing (scRNA-Seq) with reduced number of other brain cell types including microglia, neurons, and oligodendrocytes. Adaptation of this protocol for aged-animals (more than 12 months old) has not been fully tested, and thus critical steps such as beads-mediated myelin removal may need to be modified.</p>
<h3 id="sec1.1">Preparation on the day before the experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>Reagent preparation
<ol type="a">
<li>
<b>MACS buffer</b>: Add BSA to (calcium/magnesium free) DPBS to the final concentration of 0.5%.
<table id="undtbl1">
<thead>
<tr>
<th colspan="5">MACS buffer</th>
</tr>
<tr>
<th>Reagent/Volume</th>
<th>For 100 mL</th>
<th>For 250 mL</th>
<th>For 500 mL</th>
<th>For 1000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>30% BSA (mL)</b></td>
<td>1.67</td>
<td>4.17</td>
<td>8.33</td>
<td>16.67</td>
</tr>
<tr>
<td><b>DPBS (mL)</b></td>
<td>98.33</td>
<td>245.83</td>
<td>491.67</td>
<td>983.33</td>
</tr>
<tr>
<td><b>Enough for (X Samples)</b></td>
<td>2.5</td>
<td>6.25</td>
<td>12.5</td>
<td>25</td>
</tr>
</tbody>
</table>
</li>
<li>
<b>Cell suspension solution</b>: Dilute MACS buffer (0.5% BSA) with (calcium/magnesium free) DPBS to the final concentration of 0.04% of BSA.
<table id="undtbl2">
<thead>
<tr>
<th colspan="5">Cell suspension solution</th>
</tr>
<tr>
<th>Reagent/Volume</th>
<th>For 100 mL</th>
<th>For 250 mL</th>
<th>For 500 mL</th>
<th>For 1000 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>MACS (mL)</b></td>
<td>8</td>
<td>20</td>
<td>40</td>
<td>80</td>
</tr>
<tr>
<td><b>DPBS (mL)</b></td>
<td>92</td>
<td>230</td>
<td>460</td>
<td>920</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> We recommend preparing MACS buffer and cell suspension solution at least one day before use. You can store premade MACS buffer and cell suspension solution at 4°C for short term. Buffers must be vacuum filtered through 0.22 μm filters in a biosafety cabinet to remove potential contamination. Work under sterile conditions. We recommend using buffers within 1 week to minimize the risk of contamination.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Aliquot of reagents (see table below)</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare and aliquot reagents under sterile conditions. We recommend using only sterile filter pipette tips when pipetting is involved.</div>
<div class="note">
<span class="note-title">Note:</span> Storage conditions and shelf life are as indicated per manufacturer instructions.</div>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Amount/Sample</th>
<th>Storage temperature (°C)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Buffer X (Neural Tissue Dissociation kit (P))</td>
<td>2,850 μL</td>
<td>4°C</td>
</tr>
<tr>
<td>Enzyme P (Neural Tissue Dissociation kit (P))</td>
<td>75 μL</td>
<td>−20°C</td>
</tr>
<tr>
<td>Buffer Y (Neural Tissue Dissociation kit (P))</td>
<td>30 μL</td>
<td>4°C</td>
</tr>
<tr>
<td>Enzyme A (Neural Tissue Dissociation kit (P))</td>
<td>15 μL</td>
<td>−20°C</td>
</tr>
<tr>
<td>Myelin Removal Beads II</td>
<td>200 μL</td>
<td>4°C</td>
</tr>
<tr>
<td>Red Blood Cell Lysis Solution</td>
<td>500 μL</td>
<td>4°C</td>
</tr>
<tr>
<td>CD11b MicroBeads mouse/human</td>
<td>10 μL</td>
<td>4°C</td>
</tr>
<tr>
<td>Binding Buffer Stock Solution (Dead Cell Removal Kit)</td>
<td>250 μL</td>
<td>4°C</td>
</tr>
<tr>
<td>Dead Cell Removal MicroBeads (Dead Cell Removal Kit)</td>
<td>100 μL</td>
<td>4°C</td>
</tr>
</tbody>
</table>
<h3 id="sec1.2">Preparation on the day of the experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60 min</div>
<ol start="3">
<li>Make sure that centrifuges and lab ovens have reached the required temperature before starting the procedure.
<ol type="a">
<li>Centrifuges to 4°C (Eppendorf; Cat# 5810R and 5425).</li>
<li>Laboratory oven to 37°C (Thomas Scientific; Cat# 1197F66).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Steps 1–3 can be done the day prior or the day of the experiment</div>
<ol start="4">
<li>Prepare Enzyme mix 1 and Enzyme mix 2 according to the table below: enzymes are provided in the Neural Tissue Dissociation kit (P) (Miltenyi Biotec; Cat# 130-092-628).</li>
</ol>
<table id="undtbl4">
<caption>
<b>Enzyme mix 1</b> (per 1 brain)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Buffer X</td>
<td>2,850 μL</td>
</tr>
<tr>
<td>Enzyme P</td>
<td>75 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>
<b>2,925</b> μL</td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>
<b>Enzyme mix 2</b> (per 1 brain)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Buffer Y</td>
<td>30 μL</td>
</tr>
<tr>
<td>Enzyme A</td>
<td>15 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>45 μL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Enzyme mix 1 and mix 2 should be made within 20 min of use.</div>
<div class="note">
<span class="note-title">Note:</span> The mouse cortical tissue weight is typically less than 600 mg. The allotted volume for the neural dissociation kit given above is optimized for 600 mg or less of starting tissue. However, if working with samples greater than 600 mg, linearly scale up reagent and total volume accordingly.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Bovine Serum Albumin solution</td>
<td>Sigma-Aldrich</td>
<td>A9576</td>
</tr>
<tr>
<td>Isoflurane</td>
<td>Piramal Critical Care</td>
<td>66794-017-25</td>
</tr>
<tr>
<td>DPBS (calcium and magnesium free)</td>
<td>Thermo Fisher Scientific</td>
<td>14190342</td>
</tr>
<tr>
<td>HBSS (with calcium and magnesium)</td>
<td>Thermo Fisher Scientific</td>
<td>14025092</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Neural Tissue Dissociation Kit (P)</td>
<td>Miltenyi Biotec</td>
<td>130-092-628</td>
</tr>
<tr>
<td>Myelin Removal Beads II</td>
<td>Miltenyi Biotec</td>
<td>130-096-733</td>
</tr>
<tr>
<td>CD11b MicroBeads, human and mouse</td>
<td>Miltenyi Biotec</td>
<td>130-093-634</td>
</tr>
<tr>
<td>Red Blood Cell Lysis Solution, 10×</td>
<td>Miltenyi Biotec</td>
<td>130-094-183</td>
</tr>
<tr>
<td>Dead Cell Removal Kit</td>
<td>Miltenyi Biotec</td>
<td>130-090-101</td>
</tr>
<tr>
<td>Trypan Blue</td>
<td>Bio-Rad</td>
<td>1450021</td>
</tr>
<tr>
<td>Chromium<sup>TM</sup> Single Cell 3′ Library &amp; Gel Beads Kit v2</td>
<td>10x Genomics</td>
<td>PN-120267</td>
</tr>
<tr>
<td>Chromium i7 Multiplex Kit</td>
<td>10x Genomics</td>
<td>PN-120262</td>
</tr>
<tr>
<td>Bioanalyzer 2100 High Sensitivity DNA Kit</td>
<td>Agilent Technologies</td>
<td>5067-4626</td>
</tr>
<tr>
<td>LIVE/DEAD<sup>TM</sup> Viability/Cytotoxicity Kit</td>
<td>Thermo Fisher Scientific</td>
<td>L3224</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>scRNA-Seq raw data</td>
<td><a href="#bib9">Yamazaki et al., 2021</a></td>
<td>AD Knowledge Portal: <a href="https://doi.org/10.7303/syn22313650">https://doi.org/10.7303/syn22313650</a>
</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse</td>
<td><a href="#bib9">Yamazaki et al., 2021</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Primers for real-time PCR : See <a href="#tbl1">Table 1</a>
</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>SsoAdvanced Universal SYBR® Green Supermix</td>
<td>Bio-Rad</td>
<td>1725274</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>10x Genomics Cell Ranger Single Cell Software Suite</td>
<td>10x Genomics</td>
<td>Version:3.0.0</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>70 μm Cell Strainer</td>
<td>BD Falcon</td>
<td>352350</td>
</tr>
<tr>
<td>Quadro MACS Separator</td>
<td>Miltenyi Biotec</td>
<td>130-090-976</td>
</tr>
<tr>
<td>Octo MACS Separator</td>
<td>Miltenyi Biotec</td>
<td>130-042-109</td>
</tr>
<tr>
<td>MACS MultiStand</td>
<td>Miltenyi Biotec</td>
<td>130-042-303</td>
</tr>
<tr>
<td>LS Columns</td>
<td>Miltenyi Biotec</td>
<td>130-042-401</td>
</tr>
<tr>
<td>MS Columns</td>
<td>Miltenyi Biotec</td>
<td>130-042-201</td>
</tr>
<tr>
<td>Petri dishes, polystyrene (100 mm)</td>
<td>Sigma-Aldrich</td>
<td>P5606-400EA</td>
</tr>
<tr>
<td>5 mL Snap cap Eppendorf tubes</td>
<td>Eppendorf</td>
<td>0030119401</td>
</tr>
<tr>
<td>15 mL Conical tube</td>
<td>Thermo Fisher Scientific</td>
<td>339650</td>
</tr>
<tr>
<td>50 mL Conical tube</td>
<td>Thermo Fisher Scientific</td>
<td>339652</td>
</tr>
<tr>
<td>1 mL Disposable syringe</td>
<td>BD</td>
<td>BD309659</td>
</tr>
<tr>
<td>20 G needle</td>
<td>BD</td>
<td>305175</td>
</tr>
<tr>
<td>23 G needle</td>
<td>BD</td>
<td>305145</td>
</tr>
<tr>
<td>Cotton-Plugged Borosilicate-Glass Pasteur Pipet</td>
<td>Fisher Scientific</td>
<td>13-678-8A</td>
</tr>
<tr>
<td>Refrigerated centrifuge</td>
<td>Eppendorf</td>
<td>5810R</td>
</tr>
<tr>
<td>Refrigerated centrifuge</td>
<td>Eppendorf</td>
<td>5425</td>
</tr>
<tr>
<td>Disposable Hemacytometers</td>
<td>Fisher Scientific</td>
<td>22-600-107</td>
</tr>
<tr>
<td>10× Chromium Controller</td>
<td>10x Genomics</td>
<td>120263</td>
</tr>
<tr>
<td>Dissection scissors</td>
<td>Sigma-Aldrich</td>
<td>S3146-1EA</td>
</tr>
<tr>
<td>Laboratory oven</td>
<td>Thomas Scientific</td>
<td>1197F66</td>
</tr>
<tr>
<td>End-over-end Multi-Functional Tube Rotator</td>
<td>Waken Btech</td>
<td>WKN-3063</td>
</tr>
<tr>
<td>Accuri C6 Flow Cytometer</td>
<td>BD Biosciences</td>
<td>N/A (discontinued)</td>
</tr>
<tr>
<td>BenchMixer Vortexer, Q-drive balancing, 200–3200 rpm, 115V</td>
<td>Benchmark Scientific</td>
<td>BV1000</td>
</tr>
</tbody>
</table>
<table id="tbl1">
<caption>Table 1. Primers for real-time PCR</caption>
<thead>
<tr>
<th>Cell type</th>
<th>Marker</th>
<th>Forward</th>
<th>Reverse</th>
</tr>
</thead>
<tbody>
<tr>
<td>Astrocytes</td>
<td>Gfap [1]</td>
<td>AGG GCG AAG AAA ACC GCA TC</td>
<td>TTC TTT GGT GCT TTT GCC CC</td>
</tr>
<tr>
<td>Endothelial Cells</td>
<td>Glut1 [2]</td>
<td>TTG CCC AGG TGT TTG GCT TA</td>
<td>GTA GCA GGG CTG GGA TGA AG</td>
</tr>
<tr>
<td>Endothelial Cells</td>
<td>Mcam [3]</td>
<td>CGT GCC GCG GGT GTG</td>
<td>ACT TGG CTG AAG TTG CCT GA</td>
</tr>
<tr>
<td>Oligo Precursor Cells</td>
<td>Olig1 [4]</td>
<td>ACT ATG CGA TTT CCC AGG CG</td>
<td>AAC CCA CCA GCT CAT ACA GC</td>
</tr>
<tr>
<td>Pericytes</td>
<td>Pdgfrb [5]</td>
<td>GGC AGA AGA AGC CAC GCT ATG</td>
<td>AAG GCA ACT GCA CAG GGT C</td>
</tr>
<tr>
<td>Neuron</td>
<td>Tubb3 [6]</td>
<td>TGA GGC CTC CTC TCA CAA GT</td>
<td>GTC GGG CCT GAA TAG GTG TC</td>
</tr>
<tr>
<td>Smooth Muscle Cells</td>
<td>Acta2 [7]</td>
<td>TTC ATT GGG ATG GAG TCA GCG</td>
<td>AAT GCC TGG GTA CAT GGT GG</td>
</tr>
<tr>
<td>House Keeping Gene</td>
<td>Actin [8]</td>
<td>AGT GTG ACGTTG ACATCC GTA</td>
<td>GCC AGA GCA GTA ATC TCC TTC</td>
</tr>
<tr>
<td>Microglia</td>
<td>Cd11b [9]</td>
<td>CTG GGA GAT GTG AAT GGA G</td>
<td>ACT GAT GCT GGC TAC TGA TG</td>
</tr>
</tbody>
</table>
<p class="table-legend">[1] <a href="#bib4">Itoh et al., 2016</a>; [2–5] In house; [6] <a href="#bib3">Huang et al., 2018</a>; [7] <a href="#bib5">Marchant et al., 2015</a>; [8] <a href="#bib3">Huang et al., 2018</a>; [9] <a href="#bib2">Holguin et al., 2007</a>.<br> Real-time PCR was conducted using a SYBR green mix with a QuantStudio Cycler. During the PCR stage, after the initial denaturation step at 95°C for 2 min, samples were held for 15 s at 95°C then decreased and held at 60°C for 1 min for 40 cycles. The final melt curve stage occurred immediately after the PCR stage where samples were held for 15 s at 95°C, decreased to 60°C for 1 min, and increased to 95°C for the final 15 s at a rate of 0.05°C/s. All intermediate temperature changes occurred at a rate of 1.6°C/s.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p><a href="#fig1">Figure 1</a> depicts the complete workflow for the procedure.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Workflow of single cell isolation from mouse brain tissue for glio-vascular cell enriched single-cell RNA sequencing</div>
<p>Graphical summary of the main steps to obtain a live single cell suspension of enriched mouse brain vascular cells.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> It is not recommended to handle more than 2 samples per person in each experiment to ensure high cell viability. This minimizes the processing time per sample in each step.</div>
<h3 id="sec2.1">Mouse brain preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>Anesthetize a mouse (6–12 months old, male or female) with isoflurane and conduct a trans-cardial perfusion using ice-cold DPBS.
<ol type="a">
<li>25 mL of PBS would be used during the perfusion at a flow of 10–15 mL/min.</li>
</ol>
</li>
<li>Collect and store brain tissue in ice-cold HBSS.</li>
</ol>
<h3 id="sec2.2">Cell dissociation and myelin removal</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 150 min</div>
<ol start="3">
<li>Prepare enzyme mix 1 (see <a href="#before-you-begin">before you begin</a>).</li>
<li>Warm enzyme mix 1 at 37°C for 10 min in a laboratory oven.</li>
<li>Dissect out cortical tissue on a 100 mm petri dish (<a href="#fig2">Figures 2</a>A–2D).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Brain tissue dissection, cell dissociation, and MACS separator setup</div>
<p>(A–D) Step-by-step dissection of cortical brain tissue. Scale bar, 1 cm. (A) Whole mouse brain. (B) Remove the cerebellum from the cerebrum. (C) Halve the cerebrum along the longitudinal fissure and dissect out the subcortex, hippocampus, and midbrain. (D) The final dissected cortex is used for downstream neural cell dissociation and isolation of vascular enriched cells.</p>
<p>(E–G) Brain tissue dissociation and generation of homogeneous cell suspension. (E) Consistency of the initial brain mince using surgical scissors in enzyme mix 1. Brain chunks should be small enough to pass through a Pasteur pipet. (F) Brain suspension with the addition of enzyme mix 2. The cortex is further dissociated after passing through a Pasteur pipet. At this stage the suspension should be turning cloudy. (G) Cell suspension after passing through a 20 G and 23 G needle. There are no visible chunks, and the suspension should be murky with a smooth consistency.</p>
<p>(H) Quadro MACS Separator set up.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend placing the dissected cortex on a Kimwipe soaked in PBS to prevent slippage during the dissection process. The process of dabbing the surfaces of brain cortical tissue with a Kimwipe helps in the removal of meninges that contain superficial vessels, meningeal, and fibroblast-like cells. This technique reduces potential sources of contamination during glio-vascular cell profiling.</div>
<ol start="6">
<li>Keep dissected brains fully submerged in cold HBSS as enzyme mix 1 is heating up.</li>
<li>Add 2925 μL of enzyme mix 1 into an empty 5 mL snap cap Eppendorf tube (per sample).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A 5 mL tube is recommended for this step as it is easier for dissection scissors to reach to the bottom to mince the samples.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Alternatively, the slicing process can be performed in petri dish with a blade or dissection scissor if the users find it easier to do so. The critical point is to mince the brain tissues efficiently to reduce the time between mincing and incubation.</div>
<ol start="8">
<li>Move brain cortices into the 5 mL tube with enzyme mix 1.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Remove excess HBSS when moving dissected cortices to 5 mL Eppendorf tubes.</div>
<ol start="9">
<li>Cut the brain tissue into small pieces with sharp dissection scissors, until small enough to pass through the opening of a Pasteur pipet (approximately 30 s), see <a href="#fig2">Figure 2</a>E.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not pipet to test if the chunks are small enough. Suction from pipetting may cause brain chunks to stick to the pipet tip filter leading to sample loss. Dissection scissors must be sterilized using 70% ethanol before use.</div>
<ol start="10">
<li>Incubate samples for 17 min at 37°C under continuous rotation at 20 rpm using a rotator.</li>
<li>During the incubation, prepare enzyme mix 2 at room temperature (RT; 20°C–26°C) (see <a href="#before-you-begin">before you begin</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not prewarm enzyme mix 2.</div>
<ol start="12">
<li>After incubation, remove the tube from the laboratory oven and add 45 μL of enzyme mix 2 per sample.</li>
<li>Mix the tissue/enzyme solution 10–15 times at a rate of 1 mL/s using a sterilized glass cotton-plugged Pasteur pipet topped with a rubber bulb (<a href="#fig2">Figure 2</a>F).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If a larger piece of tissue gets stuck to the pipet tip, tap the bottom of the tube until the tissue is dislodged. Make sure that the solution passes the pipet smoothly before moving on to the next step. Be careful that the solution does not touch the filter.</div>
<ol start="14">
<li>Incubate samples for 12 min at 37°C under continuous rotation at 20 rpm.</li>
<li>Pass the tissue/enzyme solution 5 times using a 1 mL syringe with a 20 G needle at a rate of 0.5 mL/s.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each pass is one cycle of (in/out)</div>
<ol start="16">
<li>Switch to a 1 mL syringe with 23 G needle and pass the tissue/enzyme solution 10 times at a rate of 0.25 mL/s until solid pieces are no longer visible (<a href="#fig2">Figure 2</a>G).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid introducing bubbles during the mixing process. Use of a syringe larger than 1 mL increases the probability of bubble introduction. We recommend using a 1 mL syringe.</div>
<div class="optional">
<span class="optional-title">Optional:</span> While they have not been tested in this protocol, commercialized tissue dissociating devices (e.g., gentleMACS™ Octo Dissociator with Heaters. Miltenyi Biotec; Cat# 130-096-427) might also be used for steps 9, 10, 13, 14, 15, 16, and 17. The users are recommended establishing a customized program for tissue dissociation based on our protocol.</div>
<ol start="17">
<li>Incubate samples for 10 min at 37°C under continuous rotation at 20 rpm.</li>
<li>During incubation, rinse a 70 μm cell strainer with 2 mL of ice-cold (calcium and magnesium free) DPBS.</li>
<li>Transfer the tissue/enzyme solution to a 50 mL conical tube and add 20 mL ice-cold DPBS.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> All steps from this time onward must be on ice unless noted otherwise.</div>
<ol start="20">
<li>Apply the tissue suspension to the 70 μm cell strainer placed on a separate 50 mL collection tube on ice.</li>
<li>Centrifuge tissue suspension at 300 × <i>g</i> for 5 min at 4°C.</li>
<li>Aspirate supernatant carefully and completely.</li>
<li>Resuspend the cell pellet in 1,800 μL of cold MACS buffer (4°C).</li>
<li>Add 200 μL of Myelin Removal Beads.</li>
<li>Mix well. Incubate for 15 min at 37°C without rotation.</li>
<li>After incubation, add 18 mL of cold MACS buffer.</li>
<li>Centrifuge at 300 × <i>g</i> for 5 min at 4°C.</li>
<li>During centrifugation, prepare three LS columns per sample (<a href="#fig2">Figure 2</a>H).</li>
<li>Place the columns on the Quadro MACS Separator with MACS MultiStand and rinse the columns with 3 mL of MACS buffer.</li>
<li>Aspirate the supernatant.</li>
<li>Resuspend the cell pellet in 3 mL MACS buffer.</li>
<li>Place 15 mL conical collection tubes below each column.</li>
<li>Apply 1 mL cell suspension per each LS column.</li>
<li>Wash column twice with 1 mL of MACS buffer.</li>
<li>Collect total effluent.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Wait until the buffer in the column goes through before adding the next 1 mL of buffer.</div>
<ol start="36">
<li>Combine eluents from the 3 separate columns of the same sample.</li>
<li>Centrifuge the eluent at 300 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<h3 id="sec2.3">Red blood cell removal</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="38">
<li>During centrifugation, prepare 1× Red Blood Cell Lysis Solution.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add 0.5 mL of 10× Red Blood Cell Lysis Solution to 4.5 mL ddH<sub>2</sub>O. (5 mL of 1× solution per sample). Keep the solution at RT.</div>
<ol start="39">
<li>Aspirate the supernatant.</li>
<li>Resuspend the cell pellet in 500 μL of ice-cold MACS buffer.</li>
<li>Add 5 mL of 1× Red Blood Cell Lysis Solution to the cell suspension.</li>
<li>Vortex gently for 5 s.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Please do not use vortex settings above 600 rpm.</div>
<ol start="43">
<li>Incubate at RT for 2 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Increased incubation time will damage the cells.</div>
<ol start="44">
<li>Centrifuge at 300 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<h3 id="sec2.4">Microglia removal</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="note">
<span class="note-title">Note:</span> In comparison to the entirety of the mouse cortex, our single cell suspension enriches vascular cell types (e.g., smooth muscle cells, pericytes, and endothelial cells) by targeted removal of microglia. The addition of the myelin removal beads also contributes to the removal of the oligodendrocyte population.</div>
<ol start="45">
<li>Aspirate the supernatant.</li>
<li>Resuspend the cell pellet in 90 μL of MACS buffer.</li>
<li>Add 10 μL of CD11b MicroBeads.</li>
<li>Incubate for 15 min in a 4°C refrigerator.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not incubate on ice.</div>
<ol start="49">
<li>After the 15 min incubation, add 2 mL of MACS buffer.</li>
<li>Centrifuge at 300 × <i>g</i> for 10 min at 4°C.</li>
<li>During centrifugation, set one MS column per sample on the Octo MACS Separator with MACS MultiStand and rinse columns with 500 μL of MACS buffer.</li>
<li>Place collection tubes on ice under each MS column to collect effluent.</li>
<li>Aspirate the supernatant.</li>
<li>Resuspend the cell pellet in 500 μL of MACS buffer.</li>
<li>Apply the cell suspension to the MS column.</li>
<li>Wash columns three times with 500 μL of MACS buffer.</li>
<li>Collect total effluent.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Wait until the buffer in the column goes through before adding the next 500 μL of MACS buffer.</div>
<ol start="58">
<li>Centrifuge the cell suspension at 300 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<h3 id="sec2.5">Dead cell removal</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="59">
<li>During centrifugation, prepare 1X Binding buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add 4.75 mL ddH<sub>2</sub>O to 0.25 mL of 20X Binding Buffer Stock Solution per sample. Keep on ice. This dilution yields 1.5 mL of excess buffer. 20X Binding Buffer Stock Solution is provided in the Dead Cell Removal Kit (Miltenyi Biotec; Cat# 130-090-101).</div>
<ol start="60">
<li>Aspirate the supernatant.</li>
<li>Resuspend the cell pellet in 100 μL of Dead Cell Removal MicroBeads (provided in the Dead Cell Removal Kit).</li>
<li>Incubate for 15 min at RT.</li>
<li>During incubation, rinse 1 MS column per sample with 500 μL of 1× Binding buffer.</li>
<li>After incubation, add 1 mL of 1× Binding buffer to the cell/bead suspension.</li>
<li>Place 15 mL collection tubes below each column.</li>
<li>Apply the cell suspension to the MS column.</li>
<li>Wash the column four times with 500 μL of 1× Binding buffer.</li>
<li>Collect everything.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Wait until the buffer in the column goes through before adding the next 500 μL of 1× Binding buffer.</div>
<ol start="69">
<li>Centrifuge at 300 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<h3 id="sec2.6">Cell viability assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="70">
<li>Aspirate supernatant and resuspend the cell pellet in 1 mL of cell suspension solution (0.04% BSA - see <a href="#before-you-begin">before you begin</a>).</li>
<li>Take 10 μL of the cell suspension and mix thoroughly with 10 μL of trypan blue.</li>
<li>Load 10 μL of the mixture onto a disposable hemacytometer and determine the cell viability using a bright-field microscope (<a href="#fig3">Figures 3</a>A and 3B).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Cell viability evaluation of isolated single cells from mouse cortical brain tissues</div>
<p>(A and B) Representative hemocytometer image of cells stained with trypan blue. Cell suspension and trypan blue (1:1 ratio) were mixed and loaded onto a disposable hemacytometer. Live cells with intact cell membranes exclude trypan blue (open arrow), whereas cells with membrane disruption (i.e., dead cells) are strained blue (closed arrow). Images were taken using an EVOS Microscope under 10× (A; scale bar, 400 μm) and 20× (B; Scale bar, 100 μm).</p>
<p>(C) Representative FSC-A/SSC-A plot and doublet discrimination plot with corresponding gating.</p>
<p>(D and E) Representative results of cell viability assay using flow cytometry. Sample viability was assessed by a live/dead stain using fluorogenic ester Calcein-AM (CAM) and the nucleic acid dye Ethidium homodimer-1 (EthD-1). A total of 5,000 cells were analyzed using BD Accuri Flow Cytometer. CAM positive cells are ‘viable/live,’ whereas EthD-1 positive cells are ‘dead’. The histogram plots show the fluorescence emission profiles of samples treated with CAM (green) or EthD-1 (red). Unstained sample fluorescence emission is shown in black. (C) The shift in FL1 fluorescence in CAM-treated cells indicates that over 97% of cells are viable. (D) The FL2 fluorescence shift in EthD-1 treated cells indicates that about 13% of cells have a loss of cell membrane integrity and are considered non-viable. X axis: fluorescence intensity; Y axis; relative cell count for each population (normalized to mode).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The cell viability can also be assessed by flow cytometry using the LIVE/DEAD Viability /Cytotoxicity Kit.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not proceed with scRNA-Seq if cell viability is below 90%.</div>
<ol start="73">
<li>Centrifuge at 300 × <i>g</i> for 5 min at 4°C.</li>
<li>Resuspend the cell pellet in the cell suspension solution. Adjust the cell suspension to ~ 1×10<sup>6</sup> cells/mL of live cells.</li>
</ol>
<h3 id="sec2.7">Single-cell RNA sequencing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 weeks</div>
<p>For complete details on scRNA-Seq and data analysis, please refer to <a href="#bib9">Yamazaki et al. (2021)</a>.</p>
<ol start="75">
<li>Proceed to the 10× Chromium protocol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Approximately 3,000 glio-vascular cells per animal were loaded into each sample well according to the 10× Chromium Controller protocol (10× Genomics).</div>
<div class="optional">
<span class="optional-title">Optional:</span> After loading into the 10× Chromium Controller, surplus cells can be frozen as cell pellet or in an appropriate buffer (for RNA extraction) at −80°C for real-time PCR experiments at a later time to examine if the isolated cells are vascular cell-enriched populations. Please also refer to “Validation of vascular cell-enriched populations” in the ‘<a href="#expected-outcomes">expected outcomes</a>’ section. To ensure the optimal cell viability for scRNA-seq, the isolated single cells should be subjected to 10× Chromium Controller immediately.</div>
<ol start="76">
<li>Prepare libraries for scRNA-Seq using Chromium™ Single Cell 3′ Library &amp; Gel Bead Kit v2. Single cell libraries were constructed according to manufacturer’s instructions and sequenced at the 3′ end by an Illumina HiSeq 4000 Sequencing System at an average depth of 60,000 reads per cell.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If you are testing differential gene expressions between condition A and B, consider using a sequencing design in which confounding caused by batch or lane effects are eliminated (<a href="#bib1">Auer and Doerge, 2010</a>). For example, if you have four scRNA-Seq libraries generated from four biological replicates (e.g., samples from animal 1 with condition A, animal 2 with condition A, animal 3 with condition B, and animal 4 with condition B), the four libraries are pooled, and the pool is divided into four equal amounts that are subjected for sequencing by multiple lanes. The number of lanes utilized for sequencing should be determined such that the targeted reading depth per cell is sufficiently recovered in each sample.</div>
<div class="note">
<span class="note-title">Note:</span> Our data in this manuscript was completed using a V2 kit. As newer kits become available, we recommend following manufacturer instructions for use.</div>
<ol start="77">
<li>Following scRNA-Seq, data can be analyzed by Seurat (<a href="#bib6">Sajita et al., 2015</a>) or other platforms.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol is intended for the recovery of glio-vascular cells suitable for scRNA-Seq. Approximately 100,000 - 300,000 glial and vascular cells can be obtained from one adult mouse brain (both cortices) using the above protocol. It should be noted that this can vary depending on the strain and age of mice. In our experiments, we typically obtain single cells with approximately 90% cell viability examined in a trypan blue exclusion assay (<a href="#fig3">Figures 3</a>A and 3B).</p>
<p>We further validate the isolated vascular cell-enriched compositions of our isolated cells through real-time PCR using a variety of cell type specific markers (<a href="#fig4">Figure 4</a>). We recommend performing real-time PCR analysis when familiarizing and establishing the protocol. Given that cells used for subsequent scRNA-Seq procedures (i.e., into 10× Chromium Controller) must be processed immediately, real-time PCR validation will not be performed the day of single cell isolation for scRNA-Seq analysis.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Enrichment of vascular cell populations in the single cell suspension evaluated by real-time PCR analysis</div>
<p>RNA was extracted from unprocessed cortical brain tissues and single cell suspensions, subject to reverse transcription, and subsequently real-time PCR analysis.</p>
<p>(A) The expression of vascular cell markers in total brain tissues (Brain) and vascular enriched single cells (Vascular), including <i>Acta2</i> (SMC; smooth muscle cell), <i>Glut1, Mcam</i> (EC; endothelial cell), and <i>Pdgfr</i> (PC; pericyte). Results were normalized to actin and graphed expressions are visualized as ratios to total brain tissues (N=3 biological repeats). Data represents mean ± S.E.M. ∗∗p &lt; 0.01. Two-tailed t-test.</p>
<p>(B) The expression of non-vascular cell markers in total brain tissues (Brain) and vascular enriched single cells (Vascular), including <i>Tubb3</i> (Neu; neuron), <i>Olig-1</i> (OPC; oligodendrocyte precursor cells), <i>Cd11b</i> (MG; microglia), and <i>GFAP</i> (AS; astrocyte). Results were normalized to actin and graphed expressions are visualized as ratios to total brain tissues (N=3–4 biological repeats). Data represents mean ± S.E.M. ∗p &lt; 0.05, N.S., not significant. Two-tailed t-test.</p>
</figcaption>
</figure>
<p>We performed quality control for library preparation where representative results during library preparation are shown in <a href="#fig5">Figure 5</a>. With this protocol, our scRNA-Seq dataset identified major cell types in the brain, including glio-vascular cells (i.e., astrocytes, smooth muscle cells, pericytes, and endothelial cells), and non-vascular cells, (i.e., neurons, microglia/macrophages, and fibroblasts) (<a href="#fig6">Figure 6</a>). The average number of cells sequenced on average was around 2800 with mean reads per cell and median genes per cell were 102500 and 1500, respectively. UMI counts per cell averaged around 3300, where our average cDNA concentration yield is approximately 18 nmol/L for 35 μL. Details of scRNA-Seq analyses can be found in our recent publication (<a href="#bib9">Yamazaki et al., 2021</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Validations for cDNA amplification and gene expression library construction</div>
<p>(A) Major steps for generating Single Cell 3ʹ Gene Expression libraries from single cell suspensions. Please refer to Chromium Single Cell 3ʹ Reagent Kits v2 user guide for details.</p>
<p>(B) cDNA quality control (QC) and quantification. Samples enriched in glial and vascular cells were processed using the Chromium Single Cell 3ʹ Reagent Kits. Specifically, after reverse transcription, cDNA amplification with 12 cycles of PCR, and SPRI-based sample clean-up, 1 μL of the undiluted sample was analyzed on an Agilent Bioanalyzer High Sensitivity DNA chip. Representative images of the Bioanalyzer microgel (left) and its electropherogram trace (right) are shown (n=2, independent; a1, a2).</p>
<p>(C) Post library construction QC. After passing quality control described in B, the samples were further processed with the Chromium Single Cell 3ʹ Library Kits. Specifically, after fragmentation, adapter ligation, sample index PCR amplification with cycles of 14, and SPRI-based sample clean-up, 1 μL of the sample at 1:5 dilution was analyzed on an Agilent Bioanalyzer High Sensitivity DNA chip. Representative images of the Bioanalyzer microgel (left) and its electropherogram trace (right) are shown (n=2, independent; b1, b2). Peaks at 35 and 10380 bp represent spiked-in controls. M, marker.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/905-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example of glio-vascular cell-enriched scRNA-Seq analysis</div>
<p>(A) A t-SNE plot showing a variety of cell types within the captured glio-vascular unit. SMC, smooth muscle cell; MΦ, macrophage.</p>
<p>(B) Feature plot of markers defining major cell types in scRNA-Seq datasets. Astrocyte (<i>Dclk1, Gfap</i>); pericyte (<i>Vtn</i>); Smooth muscle cells (<i>Acta2, Sncg</i>); Choroid plexus (<i>Enpp2</i>); Endothelial cell (<i>Cldn5</i>); Neuron (<i>Tubb3</i>).</p>
<p>(C) A representative split dot plot depicting marker genes for each cell population from <a href="#bib9">Yamazaki et al. 2021</a>.</p>
<p>(D) The scRNA-Seq datasets show differential transcriptomic profiles of glio-vascular cells. Metascape analysis of differentially expressed genes (DEGs) in the pericyte cluster showed an angiogenic upregulation in apoE3 mice. In contrast, glial activation, and cell-matrix adhesion increases were identified in the corresponding apoE4 astrocyte cluster. Figure reprinted with permission from <a href="#bib9">Yamazaki et al., 2021</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Our protocol is optimized for glio-vascular cell isolation, which requires strong enzymatic digestion. Such treatment conditions can potentially activate other cells, such as microglia which may influence results. To enrich glio-vascular cells, microglia are targeted for removal during the isolation process. Only CD11b is targeted due to market availability. If future beads targeting a different marker for microglia are available, additional steps may be added to the protocol to optimize microglial removal.</p>
<p>During troubleshooting and validation, we obtain approximately 70%–90% cell viability in a flow cytometry-based cell viability assay (<a href="#fig3">Figures 3</a>C–3E). Decreased cell viability in a flow cytometry live/dead stain may be due to debris contamination counting as dead cells. For the actual experiment, we recommend manual counting with a hemocytometer as it may improve live cell count accuracy and can be performed right away with a few μL of sample. Do not proceed with scRNA-Seq if cell viability is below 90%.</p>
<div class="note">
<span class="note-title">Note:</span> The general guideline is that cell viability must be assessed as quickly as possible, and can be examined by means of method of your choice that provides fast and accurate results.</div>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>The viability of the cells is below 90%. The major potential causes may include: 1) The incubation time in the red blood cell removal step (step 43) might be longer than necessary for well-perfused samples, contributing to decreased cell viability. 2) Incubation during the neural dissociation steps (steps 10, 14, and 17) is ideal for minced brain chunks. Increased incubation time to facilitate dissociation may decrease cell viability. Note that the duration of red blood cell removal step (step 43) is only 2 min. Please be cognizant of the time particularly during this step. 3) Strong vortexing or rough handling of cells may potentially induce cell death.</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Cell viability decreases with time following extraction. We recommend preparing all reagents prior to each step to avoid delay in any of the steps following cortex dissection. Incubation time for the red blood cell removal step (step 43) can be decreased to 1 min or as needed for optimal cell viability if the brain was well perfused.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>The number of cells ultimately sequenced in scRNA-Seq dataset is lower than expected.</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Re-evaluate cell preparation methods including cell viability, cell counting method and pipetting techniques (see also: TECHNICAL NOTE Guidelines for Accurate Target Cell Counts Using 10× Genomics Single Cell Solutions, 10× Genomics). Load 110%–120% of the targeted cell numbers to ensure that the numbers of cells ultimately sequenced in scRNA-Seq dataset are as close to the targeted cell numbers as possible.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Not enough vascular cells in the scRNA-Seq analysis after clustering and identifying cell types with cell-type specific markers (step 77).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>If there is an overabundance of microglia in the scRNA-Seq data, consider increasing the amount of CD11b beads in the microglia removal step (step 47). Beads against additional/alternative microglia markers can be considered when optimizing microglia removal.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>Substantial upregulation of immediate early genes, such as <i>Fos</i>, <i>Egr1</i>, <i>Fosb</i>, <i>Nr4a1</i>, and <i>Dusp1</i>, presumably associated with dissociation procedures, is detected in scRNA-Seq data.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Consider applying actinomycin D, a general transcription inhibitor, during dissociation procedures (<a href="#bib8">Wu et al., 2017</a>). However, we have not yet tested this out and have no data on the compatibility of actinomycin D with the 10× Single Cell 3′ workflows.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>There are not enough vascular cells for scRNA-Seq (step 72).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>Cell dissociation was too weak (step 14). Without proper digestion and dissociation, it will be difficult to see vascular cells in scRNA-Seq. Some training may be required to acquire to balance cell viability and BVC yield. If the digestion is too strong, there will be cell damage. If the digestion is too weak, then there will be a low yield of BVCs.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Auer, P.L. and Doerge, R.W. (2010). Statistical design and analysis of RNA sequencing data. Genetics <i>185</i>, 405-416. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref1">View at publisher</a></p>
<p id="bib2">Holguin, A., Frank, M.G., Biedenkapp, J.C., Nelson, K., Lippert, D., Watkins, L.R., Rudy, J.W., and Maier, S.F. (2007). Characterization of the temporo-spatial effects of chronic bilateral intrahippocampal cannulae on interleukin-1beta. J. Neurosci. Methods <i>161</i>, 265-272. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref2">View at publisher</a></p>
<p id="bib3">Huang, J., Li, J., Feng, C., Huang, X., Wong, L., Liu, X., Nie, Z., and Xi, G. (2018). Blood-brain barrier damage as the starting point of leukoaraiosis caused by cerebral chronic hypoperfusion and its involved mechanisms: effect of agrin and aquaporin-4. Biomed Res. Int. <i>2018</i>, 2321797. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref3">View at publisher</a></p>
<p id="bib4">Itoh, K., Ishihara, Y., Komori, R., Nochi, H., Taniguchi, R., Chiba, Y., Ueno, M., Takata-Tsuji, F., Dohgu, S., and Kataoka, Y. (2016). Levetiracetam treatment influences blood-brain barrier failure associated with angiogenesis and inflammatory responses in the acute phase of epileptogenesis in post-status epilepticus mice. Brain Res. <i>1652</i>, 1-13. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref4">View at publisher</a></p>
<p id="bib5">Marchant, V., Droguett, A., Valderrama, G., Burgos, M.E., Carpio, D., Kerr, B., Ruiz-Ortega, M., Egido, J., and Mezzano, S. (2015). Tubular overexpression of Gremlin in transgenic mice aggravates renal damage in diabetic nephropathy. Am. J. Physiol. Renal Physiol. <i>309</i>, F559-F568. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref5">View at publisher</a></p>
<p id="bib6">Sajita, R., Farrell, J.A., Gennert, D., Schier, A.F., and Regev, A. (2015). Spatial reconstruction of single-cell gene expression data. Nat. Biotechnol. <i>33</i>, 495-502. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref6">View at publisher</a></p>
<p id="bib7">Vanlandewijck, M., He, L., Mae, M.A., Andrae, J., Ando, K., Del Gaudio, F., Nahar, K., Lebouvuier, T., Lanvina, B., Gouveia, L., et al. (2018). A molecular atlas of cell types and zonation in the brain vascular. Nature <i>554</i>, 475-480. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref7">View at publisher</a></p>
<p id="bib8">Wu, Y.E., Pan, L., Zuo, Y., Li, X., and Hong, W. (2017). Detecting activated cell populations using single-cell RNA-seq. Neuron <i>96</i>, 313-329. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref8">View at publisher</a></p>
<p id="bib9">Yamazaki, Y., Liu, C.C., Yamazaki, A., Shue, F., Martens, Y.A., Chen, Y., Qiao, W., Kurti, A., Oue, H., Ren, Y., et al. (2021). Vascular ApoE4 impairs behavior by modulating gliovascular function. Neuron <i>109</i>, 1-10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00422-6/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information, requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Chia-Chen Liu (<a href="mailto:Liu.chiachen@mayo.edu">Liu.chiachen@mayo.edu</a>).</p>
<h4>Materials availability</h4>
<p>This protocol did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>No custom code is necessary to perform this protocol. All datasets mentioned in this protocol have been described in <a href="#bib9">Yamazaki et al., 2021</a>. The accession number for the single-cell RNA sequencing data reported in this paper is AD Knowledge Portal: <a href="https://doi.org/10.7303/syn22313650">https://doi.org/10.7303/syn22313650</a>. Graphical abstract was prepared using Biorender.com.</p>
<h3>Acknowledgments</h3>
<p>The authors thank Dr. Laura Lewis Tuffin for technical support and Tadafumi (Clark) Ikezu and Hongmei Li for careful reading of the manuscript. This research was supported by the <a href="https://doi.org/10.13039/100000002">National Institute of Health</a> (NIH) (grants R37AG027924, RF1AG051504, RF1AG057181, RF1AG046205, and P30AG062677 to G.B.; and grant R01AG062110 to C.-C.L.), a <a href="https://doi.org/10.13039/100007625">Cure Alzheimer’s Fund</a> (G.B.), the <a href="https://doi.org/10.13039/100000968">American Heart Association</a> (fellowship 17POST33410227 to Y.Y.), Florida Department of Health Ed and Ethel Moore Alzheimer’s Disease Research Program (8AZ07), and a grant from <a href="https://doi.org/10.13039/100006312">Brightfocus</a> (to C.-C.L.).</p>
<h3>Author contributions</h3>
<p>A.Y., F.S., and Y.Y. performed single-cell extraction; A.Y. and F.S. performed PCR validation experiments; A.Y. and Y.A.M. performed flow cytometry; F.S., Y.Y., Y.A.M., and C.-C.L. analyzed data; and A.Y., F.S., Y.Y., Y.A.M., G.B., and C.-C.L. wrote the manuscript, with input from all authors.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>