<article>
<section>
<h1 id="header">A protocol for the induction of experimental necrotizing enterocolitis in neonatal mice</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Lila S. Nolan,<sup><a href="#aff1">1</a></sup> Qingqing Gong,<sup><a href="#aff1">1</a></sup> Heather N. Hofmeister,<sup><a href="#aff1">1</a></sup> and Misty Good<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pediatrics, Division of Newborn Medicine, Washington University School of Medicine, St. Louis Children’s Hospital, 660 S. Euclid Avenue, Campus Box 8208, St. Louis, MO 63110, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:mistygood@wustl.edu">mistygood@wustl.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100951">10.1016/j.xpro.2021.100951</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Recapitulating human NEC using animal models has been insightful in dissecting the signaling pathways, immune-mediated mechanisms, genetic signatures, and the intestinal architecture of NEC. This protocol describes an <i>in vivo</i> murine NEC model, using hypoxia and formula containing lipopolysaccharide and enteric bacteria derived from an infant with NEC. With this mouse model, we aim to further dissect NEC pathogenesis and develop new therapeutic strategies.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref3">Mihi et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>This protocol describes the induction of murine necrotizing enterocolitis (NEC)</li>
<li>This mouse model is based on the use of hypoxia and formula supplemented with LPS</li>
<li>This mouse model can be used to study NEC pathogenesis and therapeutic targets</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1169-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Investigators are required to obtain appropriate ethical approval from their Institutional Animal Care and Use Committee (IACUC) or equivalent for the use of vertebrate animals. All animal experiments were performed in an American Association for the Accreditation of Laboratory Animal Care-accredited and specific pathogen-free animal facility at the Washington University School of Medicine in St. Louis. All mice were bred, maintained, and housed in accordance with the procedures outlined in the Guide for the Care and Use of Laboratory Animals under a study proposal approved by the IACUC (protocol no. 20-0050) of the Washington University School of Medicine in St. Louis.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>C57BL/6J, M. musculus, Neonatal mice of both sexes, postnatal day 4–7</td>
<td>The Jackson Laboratory</td>
<td>Stock No. 000664</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>GGCGACCTGGAAGTCCAACT</td>
<td>Integrated DNA Technologies</td>
<td>
<i>RPLO</i> (Mouse) forward primer for quantitative PCR</td>
</tr>
<tr>
<td>CCATCAGCACCACAGCCTTC</td>
<td>Integrated DNA Technologies</td>
<td>
<i>RPLO</i> (Mouse) reverse primer for quantitative PCR</td>
</tr>
<tr>
<td>AGTGTGGATCCCAAGCAATACCCA</td>
<td>Integrated DNA Technologies</td>
<td>
<i>Il1b</i> (Mouse) forward primer for quantitative PCR</td>
</tr>
<tr>
<td>TGTCCTGACCACTGTTGTTTCCCA</td>
<td>Integrated DNA Technologies</td>
<td>
<i>Il1b</i> (Mouse) reverse primer for quantitative PCR</td>
</tr>
<tr>
<td>CAGACAGAAGTCATAGCCACT</td>
<td>Integrated DNA Technologies</td>
<td>
<i>Cxcl2</i> (Mouse) forward primer for quantitative PCR</td>
</tr>
<tr>
<td>GGCACATCAGGTACGATCCA</td>
<td>Integrated DNA Technologies</td>
<td>
<i>Cxcl2</i> (Mouse) reverse primer for quantitative PCR</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td><a href="#bib9">Schneider et al., 2012</a></td>
<td><a href="https://imagej.nih.gov/ij">https://imagej.nih.gov/ij</a></td>
</tr>
<tr>
<td>GraphPad Prism Software, Version 9.0</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com">https://www.graphpad.com</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Infant Incubator, C100-200-2 Series, set at 37°C</td>
<td>Air Shields-Vickers</td>
<td>N/A</td>
</tr>
<tr>
<td>Hypoxia Chamber</td>
<td>Billups-Rothenberg, Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Handi+ Oxygen analyzer</td>
<td>Maxtec</td>
<td>N/A</td>
</tr>
<tr>
<td>Peripherally Inserted Central Catheter, 1.9 French, Single- Lumen</td>
<td>Utah Medical Products, Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Esbilac Puppy Milk replacer</td>
<td>Pet-Ag, Inc.</td>
<td>N/A</td>
</tr>
<tr>
<td>Similac Advance® OptiGRO<sup>TM</sup> with Iron infant formula</td>
<td>Abbott Nutrition</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Preparing the enteric bacterial challenge</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Preparation of the bacterial suspension for inoculation into the mouse formula is based on the intestinal microbiota obtained from the enteric contents of an infant with the most severe form of NEC, <i>NEC totalis</i> (<a href="#bib2">Good et al., 2014</a>; <a href="#bib3">Mihi et al., 2021</a>). The enteric bacterial stock was generated by overnight (16 h) culturing of the fecal slurry in the resected intestinal tissue from an infant with <i>NEC totalis</i>. The bacteria culture was then centrifuged, resuspended in 50% glycerol, aliquoted in cryovials and stored at −80°C.</p>
<ol>
<li>The day before initiating the model, use a sterile pipette tip to scrape approximately 10 μL of an enteric bacterial glycerol stock from a frozen cryovial.</li>
<li>Place the sterile pipette tip containing the bacterial aliquot into a 15-mL culture tube containing 2 mL of LB Broth. Culture the bacteria overnight (16 h) in an orbital shaker at 37°C with agitation speed at 150 rpm.
<ol type="a">
<li>A control culture tube with 2 mL of sterile LB broth should also be cultured simultaneously in the orbital shaker to ensure there is no concern for bacterial contamination of the LB broth.</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">Preparing the formula and allocating pups into experimental groups</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="3">
<li>The following morning, transfer 100 μL of the bacterial culture into four T75 culture flasks each containing 20 mL LB broth.</li>
<li>Culture this inoculum for a minimum of two hours in an orbital shaker with agitation speed at 150 rpm at 37°C.</li>
<li>At 7 AM on the first day of the model, in the animal facility, weigh the postnatal day 4 pups to be included in the model and randomly assign the animals into two experimental groups: dam-fed and formula-fed.
<ol type="a">
<li>For the dam-fed group, allow the mouse pups to remain with the mother for the full duration of the model. These pups will be served as dam-fed negative controls.</li>
<li>For the formula-fed group, separate the pups from the mother into a new cage and place the cage into a pre-warmed infant incubator at 37°C. This group will be housed in the incubator for the full duration of the model.</li>
</ol>
</li>
<li>Use a spectrophotometer to measure the density of the two-hour bacterial culture at 600 nm (OD<sub>600nm</sub>). Add 1 mL of sterile LB broth into a 1 cm cuvette and measure the OD<sub>600nm</sub> to serve as the blank. In a separate 1 cm cuvette, add 1 mL of the inoculum and measure the OD<sub>600nm</sub>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The OD<sub>600nm</sub> value should be 0.6 ± 0.02, corresponding with the exponential phase of bacterial growth.</div>
<div class="note">
<span class="note-title">Note:</span> If the OD<sub>600nm</sub> is greater than 0.6, use LB broth to dilute the culture until the diluted culture results in a goal OD<sub>600nm</sub>. If the OD<sub>600nm</sub> is less than 0.6, continue to culture the inoculum until the OD<sub>600nm</sub> reaches 0.6.</div>
<ol start="7">
<li>Once the inoculum has achieved the goal OD<sub>600nm</sub>, transfer 20 mL of the inoculum from each culture flask into separate 50-mL conical tubes and centrifuge at 3000 × <i>g</i> for 10 min. Discard the supernatant.</li>
<li>Resuspend the bacterial pellets each in 1 mL sterile phosphate-buffered saline (PBS) in a 50-mL conical tube.</li>
<li>Add 4 mL of the total resuspended bacteria into 20 mL of NEC formula (comprised of a 2:1 ratio of Similac Advance formula to Esbilac puppy milk) in a 50-mL conical tube.</li>
<li>Using the average weight of the pups from the morning, add 5 μg/g lipopolysaccharide (LPS) to the NEC formula. LPS serves as a ligand for innate immune receptor toll-like receptor 4 signaling, which has been shown to be required for the development of NEC via impairment of the integrity of the intestinal epithelial barrier (<a href="#bib4">Mihi and Good, 2019</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The addition of both enteric bacteria and LPS to the NEC formula is necessary for the induction of NEC-like pro-inflammatory response in this model.</div>
<div class="note">
<span class="note-title">Note:</span> This NEC formula with LPS contains the volume to be used for 6 feeds per 24-h period, with the first feeding timed for 10 AM on day 1 of the model and newly made formula at 10AM on days 2–3 of the model.</div>
<h3 id="sec2.3">Induction of experimental NEC</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="11">
<li>Perform scheduled feedings of the mouse pups beginning at 10 AM on Day 1 of the model. Fill a 1-mL syringe with NEC formula and attach it to a 1.9 French single-lumen PICC line.</li>
<li>Provide the mouse pups with oral gavage feedings of the NEC formula.
<ol type="a">
<li>Gently hold the pup by grasping the loose skin at the base of the neck and use forceps to grasp the distal end of the PICC line to gently introduce 2 cm of the PICC into the oropharynx and esophagus (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1169-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Illustration of insertion of the PICC line into the oropharynx and esophagus for delivery of NEC formula</div>
<p>Using forceps, the PICC line is gently inserted 2 cm for delivery of NEC formula into the stomach of the mouse pup. Figure created with <a href="http://biorender.com">biorender.com</a>.</p>
</figcaption>
</figure>
</li>
<li>There should be no significant resistance with the insertion of the PICC line.</li>
<li>Slowly dispense 80 μL of NEC formula into the stomach.</li>
<li>Then slowly withdraw the PICC line from the oral cavity.</li>
<li>Monitor the animal for increased respiratory effort or emesis associated with a malpositioned PICC line.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In our model, mouse pup feedings occur at 7 AM (starting on model day 2), 10 AM, 1 PM, 4 PM, 7 PM and 10 PM. Repeat step 12 at each of these time points.</div>
<ol start="13">
<li>Twice daily, immediately following the 1 PM and 7 PM feedings, subject the mouse pups to hypoxia.
<ol type="a">
<li>Place the pups in a hypoxia chamber connected to a gas tank containing 95% nitrogen and 5% oxygen for 10 min.</li>
<li>Carefully monitor the mouse pups for signs of apnea, cyanosis or pale discoloration, or abdominal distension in between feeds. Pups that present with these signs should be immediately euthanized per IACUC policy.</li>
</ol>
</li>
<li>At 3 PM on model days 1 and 2, prepare the bacterial culture for the next day, as described in steps 3 and 4, in order to prepare and provide NEC formula for the full duration of the model.</li>
<li>On model days 2 and 3, at 7 AM, obtain the daily weight of each pup to identify the daily average weight of the pups in each group and then provide oral gavage feeding using the formula prepared from the day prior.
<ol type="a">
<li>Be sure to first invert the 50-mL conical gently several times while avoiding the formation of air bubbles in the formula.</li>
<li>Prepare the new daily NEC formula as in steps 6–10. Perform scheduled oral gavage feedings and hypoxia exposure per steps 11–13 to complete the 72-h mouse model of NEC.</li>
</ol>
</li>
</ol>
<h3 id="sec2.4">Sacrificing animals and harvesting tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min per animal</div>
<ol start="16">
<li>On the morning of model day 4, obtain the daily weight of each pup. Euthanize the pups in accordance with IACUC and institutional policies. Make a vertical incision along the abdominal wall to expose the gastrointestinal tract.</li>
<li>Locate and resect the intestine from the duodenum to the distal rectum. Place the intestine in a petri dish containing ice-cold PBS. Isolate the desired segments of the intestine for tissue collection and processing.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This model focuses primarily on pathology within the ileum. The terminal part of the ileum, which is approximately 3 cm from the cecum, is resected. A small piece of the terminal ileum (1 cm) is placed in RNA-later<sup>TM</sup> for transcriptional analysis (<a href="#fig2">Figure 2</a>A). Another small piece of terminal ileum (1 cm) is snap-frozen in a 1.5 ml microcentrifuge tube and kept frozen at −80°C for protein analysis (<a href="#fig2">Figure 2</a>B).</div>
<div class="note">
<span class="note-title">Note:</span> For analysis of intestinal microbial communities, mouse fecal contents can be collected for 16s rRNA microbial analysis (<a href="#bib3">Mihi et al., 2021</a>). Locate and resect the intestine at the proximal colon, adjacent to the cecum and place the tissue on a clean surface. Using forceps, gently press on the exterior of the colon to allow fecal contents to pass along to the distal end of the intestinal lumen for collection. Fecal samples should be snap-frozen in a 1.5 ml microcentrifuge tube and stored at −80°C for later 16s rRNA microbial sequencing (<a href="#bib3">Mihi et al., 2021</a>). Mouse pups subjected to experimental NEC will exhibit an intestinal microbiome with an overabundance of Gammaproteobacteria species similar to human infants with NEC (<a href="#bib3">Mihi et al., 2021</a>; <a href="#bib6">Nolan et al., 2021</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1169-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Recommended segmental transection of small intestine during tissue collection</div>
<p>(A–C) During postmortem dissection, segment (A) of the terminal ileum is collected for transcriptional analysis, segment (B) is collected for protein analysis and segment (C) is resected for histological study. Figure created with <a href="http://Biorender.com">Biorender.com</a>.</p>
</figcaption>
</figure>
<h3 id="sec2.5">Preparation of intestines for cross-sectional histological analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min per animal</div>
<ol start="18">
<li>The remaining segment of the terminal ileum (<a href="#fig2">Figure 2</a>C) is flushed with PBS and then with 4% paraformaldehyde (PFA) using a 24-gauge gavage needle.</li>
<li>The tissue segment is transected longitudinally using fine scissors, doused in 4% PFA, and transferred to a container for immersion in 4% PFA for fixation. After 24 h at 4°C, discard the PFA and wash with PBS twice for 15 min.</li>
<li>Discard the PBS and then dehydrate with 50% ethanol for 15 min and then 70% ethanol for storage prior to embedding.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The fixation, wash and dehydration steps should be carried out at 4°C on a rocker.</div>
<ol start="21">
<li>The intestinal specimens are then carefully embedded in agar first and then paraffin. Tissue sections are cut on a microtome at 5-μm thickness and stained with hematoxylin and eosin for histological study.</li>
</ol>
<h3 id="sec2.6">Tissue processing for transcriptional analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<ol start="22">
<li>For RNA isolation from the transected terminal small intestine, add 1 mL TRIzol<sup>TM</sup> to the resected intestinal tissue stored in RNA Later (Invitrogen).</li>
<li>Add one steel bead per tube and homogenize the tissue using the Qiagen TissueLyser for 7 min at 50 oscillations per second.
<ol type="a">
<li>Once the tissue is completely homogenized, remove the steel bead, and add 200 μL of chloroform.</li>
<li>Vigorously invert the tube ten times and allow the samples to incubate at 68–77°F (20°C–25°C) for five minutes.</li>
</ol>
</li>
<li>Centrifuge the samples for 10 min at 18,000×<i>g</i> and 4°C.
<ol type="a">
<li>The solution will separate into three layers: a top aqueous layer, a white middle layer, and a bottom pink layer of Trizol.</li>
</ol>
</li>
<li>Transfer 500 μL of the aqueous layer into a 1.5 mL microcentrifuge tube. Add 500 μL of isopropanol and 3 μL glycogen, and gently invert the sample 10 times.</li>
<li>Incubate the samples at -20°C for 20 min.</li>
<li>Centrifuge the samples for 10 min at 18000×<i>g</i> and 4°C. Decant the supernatant and add 0.5 mL of 70% ethanol to each sample.</li>
<li>Centrifuge the samples for 5 min at 18000×<i>g</i> and carefully decant the supernatant.</li>
<li>Centrifuge the samples again for 1 min at 18000×<i>g.</i> Remove the remaining ethanol with a fine-tip pipette and allow the samples to air dry for at least 15 min.</li>
<li>When the pellet appears transparent, add 50 μL of RNase-free water and allow the pellets to hydrate for approximately 10 min.
<ol type="a">
<li>Vortex the samples and measure the RNA concentration using a spectrophotometer before storing the samples at −80°C.</li>
<li>Use the Qiagen QuantiTec Reverse Transcription Kit to synthesize the complementary DNA following the manufacturer’s protocol: <a href="https://www.qiagen.com/us/resources/resourcedetail?id=f0de5533-3dd1-4835-8820-1f5c088dd800&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=f0de5533-3dd1-4835-8820-1f5c088dd800&amp;lang=en</a>.</li>
</ol>
</li>
<li>Quantitative real-time PCR (qPCR) is performed using IQ SYBR Green Supermix and CFX Connect Real-Time PCR Detection System (Bio-Rad Laboratories).
<ol type="a">
<li>The CFX Connect Real-Time PCR Detection System utilizes a two-step amplification process by heating the samples to 95°C for five minutes, then alternating between 95°C for ten seconds and 55°C for 30 s, repeated for 40 cycles as described in the manufacturer’s instructions: <a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10000068707.pdf">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/10000068707.pdf</a>.</li>
<li>The primers are listed in <a href="#tbl1">Table 1</a>, and a melting curve was added to the amplification protocol to ensure the absence of primer dimer formation or multiple amplicons.
<table id="tbl1">
<caption>Table 1. Primers for qRT-PCR analysis of pro-inflammatory gene expression</caption>
<thead>
<tr>
<th>Gene</th>
<th>Species</th>
<th>Forward primer</th>
<th>Reverse primer</th>
<th>Amplicon size (base pairs)</th>
</tr>
</thead>
<tbody>
<tr>
<td><i>RPLO</i></td>
<td>Mouse</td>
<td>GGCGACCTGGAAGTCCAACT</td>
<td>CCATCAGCACCACAGCCTTC</td>
<td>143</td>
</tr>
<tr>
<td><i>Il1b</i></td>
<td>Mouse</td>
<td>AGTGTGGATCCCAAGCAATACCCA</td>
<td>TGTCCTGACCACTGTTGTTTCCCA</td>
<td>175</td>
</tr>
<tr>
<td><i>Cxcl2</i></td>
<td>Mouse</td>
<td>CCAGACAGAAGTCATAGCCACT</td>
<td>GGCACATCAGGTACGATCCA</td>
<td>217</td>
</tr>
</tbody>
</table>
</li>
<li>The expression of genes assessed by qRT-PCR was quantified relative to the housekeeping gene <i>RPLO.</i>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<h3 id="sec3.1">Assessment of NEC severity and intestinal injury</h3>
<p>As NEC continues to have a high morbidity and mortality rate in preterm infants, the importance of reliable modeling to investigate the pathogenesis and potential therapeutics remains a priority in NEC research. Experimental modeling of NEC using rodents is commonly used due to low cost of maintenance and ease of breeding (<a href="#bib8">Sulistyo et al., 2018</a>). Barlow and colleagues described an early model of NEC in 1974 using neonatal rats (<a href="#bib1">Barlow et al., 1974</a>). The Barlow model incorporated factors that are recognized as contributing factors in NEC development, including intestinal immaturity, hyperosmolar feeds, hypoxic stress and dysbiosis (<a href="#bib1">Barlow et al., 1974</a>). Subsequent experimental NEC models using rodents have been modified to include exposure to 100% nitrogen, cold stress or the addition of lipopolysaccharide to recapitulate the intestinal inflammation and injury observed human NEC (<a href="#bib8">Sulistyo et al., 2018</a>). Additionally, some investigators have used piglet or rabbit models of NEC in the attempt to create a model that more closely mimics the human disease; however, these models are limited by differences in pathologic intestinal changes when compared to human NEC, high cost of maintenance and challenges involved in creating transgenic animals (<a href="#bib8">Sulistyo et al., 2018</a>). Therefore, this protocol describes a method of inducing intestinal injury and an inflammatory response in neonatal mice to model the findings observed in human NEC (<a href="#bib5">Neu and Walker, 2011</a>; <a href="#bib7">Patel et al., 2020</a>).</p>
<p>During postmortem analysis, mouse pups with NEC demonstrate abdominal distension. In the assessment of gross organ disease severity, pups subjected to the NEC model exhibit significant and diffuse intestinal gaseous dilation, and they may occasionally develop intestinal perforation. When small intestinal tissue is collected for transcriptional analysis, formula-fed pups (FF) with experimental NEC have a significantly upregulated expression of pro-inflammatory markers, including <i>Il1b</i> and <i>Cxcl2</i> when compared with dam-fed (DF) control littermates (<a href="#fig3">Figure 3</a>A). The histological features of the small intestine from dam-fed pups compared with progressive severity of NEC injury are detailed in <a href="#fig3">Figure 3</a>B, in which pups with severe NEC experience disruption of the intestinal epithelial architecture and destruction of the intestinal villi. These findings recapitulate several of the features seen in human NEC.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1169-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Experimental murine NEC recapitulates inflammatory and histologic features of human NEC</div>
<p>(A) Formula-fed pups (FF) subjected to experimental NEC demonstrate significantly upregulated mRNA expression of pro-inflammatory markers <i>Il1b</i> and <i>Cxcl2</i> when compared with dam-fed (DF) control littermates.</p>
<p>(B) Representative histology from dam-fed pups compared with pups with progressive severity of NEC-like injury reveals intestinal villous destruction and disrupted overall intestinal architecture. ∗∗p &lt; 0.001 by Mann-Whitney U test.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is optimized to induce an NEC-like intestinal injury in 4-day-old neonatal mouse pups using a 72-h model. As this protocol requires young mice with intestinal immaturity, performing the model on older mice may not provide the same degree of intestinal injury and inflammation. Additionally, individual mouse or strain differences may impact the outcomes of this experimental NEC induction model.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Pups develop signs of apnea while in the hypoxia chamber (step 13b).</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Gently rock the chamber during the induction of hypoxia to stimulate spontaneous breathing reflexes.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>Pups develop respiratory distress or vomiting after the gavage of formula (step 12).</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Ensure that the PICC line is being inserted slowly into the oral cavity. There should be no significant resistance to the insertion of the PICC line. Resistance or bouncing of the PICC line when inserted into the oral cavity is concerning for curling of the PICC line in the oropharynx or accidental insertion into the trachea, and the PICC line should be immediately withdrawn and reinserted.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>Intestinal specimens are difficult to flush with PBS or 4% PFA during histology preparation (step 18).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>The induction of NEC results in very friable intestinal tissue. Flush the intestine gently to prevent damage to the intestinal architecture.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>The mortality rate of the model can be up to 50%.</p>
<h3 id="sec5.8">Potential solution</h3>
<p>Different mouse strains may differ in the degree of intestinal injury or mortality rate when subjected to experimental NEC. To reduce the risk of iatrogenic deaths, it is important to carefully dose the LPS per the daily weight of the pups, use fresh stocks of puppy milk and infant formula and ensure that the enteric bacterial stock is stored at −80°C when not in use. It is also important to train personnel on careful handling and gavage feeding of pups to avoid unintentional deaths. The mouse mortality should be assessed daily during the model to ensure an avoidance of preventative harm during the model. While inducing NEC-like disease in animals, the mice are euthanized if they exhibit signs of distress, pain, or discomfort and each animal is autopsied to determine their cause of death (whether iatrogenic or due to disease).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Barlow, B., Santulli, T.V., Heird, W.C., Pitt, J., Blanc, W.A., and Schullinger, J.N. (1974). An experimental study of acute neonatal enterocolitis–the importance of breast milk. J. Pediatr. Surg. <i>9</i>, 587-595. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref1">View at publisher</a></p>
<p id="bib2">Good, M., Sodhi, C.P., Ozolek, J.A., Buck, R.H., Goehring, K.C., Thomas, D.L., Vikram, A., Bibby, K., Morowitz, M.J., Firek, B., et al. (2014). Lactobacillus rhamnosus HN001 decreases the severity of necrotizing enterocolitis in neonatal mice and preterm piglets: evidence in mice for a role of TLR9. Am. J. Physiol. Gastrointest. Liver Physiol. <i>306</i>, G1021-G1032. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref2">View at publisher</a></p>
<p id="bib3">Mihi, B., Gong, Q., Nolan, L.S., Gale, S.E., Goree, M., Hu, E., Lanik, W.E., Rimer, J.M., Liu, V., Parks, O.B., et al. (2021). Interleukin-22 signaling attenuates necrotizing enterocolitis by promoting epithelial cell regeneration. Cell Rep. Med. <i>2</i>, 100320. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref3">View at publisher</a></p>
<p id="bib4">Mihi, B. and Good, M. (2019). Impact of toll-like receptor 4 signaling in necrotizing enterocolitis: the state of the science. Clin. Perinatol. <i>46</i>, 145-157. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref4">View at publisher</a></p>
<p id="bib5">Neu, J. and Walker, W.A. (2011). Necrotizing enterocolitis. New Engl. J. Med. <i>364</i>, 255-264. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref5">View at publisher</a></p>
<p id="bib6">Nolan, L.S., Mihi, B., Agrawal, P., Gong, Q., Rimer, J.M., Bidani, S.S., Gale, S.E., Goree, M., Hu, E., Lanik, W.E., et al. (2021). Indole-3-carbinol-dependent aryl hydrocarbon receptor signaling attenuates the inflammatory response in experimental necrotizing enterocolitis. ImmunoHorizons <i>5</i>, 193-209. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref6">View at publisher</a></p>
<p id="bib7">Patel, R.M., Ferguson, J., McElroy, S.J., Khashu, M., and Caplan, M.S. (2020). Defining necrotizing enterocolitis: current difficulties and future opportunities. Pediatr. Res. <i>88</i>, 10-15. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref7">View at publisher</a></p>
<p id="bib9">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nature Methods <i>9</i>, 671-675. <a class="external-link" href="https://doi.org/10.1515/iss-2017-0050">https://doi.org/10.1515/iss-2017-0050</a></p>
<p id="bib8">Sulistyo, A., Rahman, A., Biouss, G., Antounians, L., and Zani, A. (2018). Animal models of necrotizing enterocolitis: review of the literature and state of the art. Innovative Surg. Sci. <i>3</i>, 87-92. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00657-2/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Misty Good at <a href="mailto:mistygood@wustl.edu">mistygood@wustl.edu</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The published article includes all datasets generated or analyzed during this study. This study did not generate any code.</p>
<h3>Acknowledgments</h3>
<p>L.S.N. receives funding support from the National Institutes of Health (<a href="https://doi.org/10.13039/100000052">NIH</a>; F32DK130248) and the American Academy of Pediatrics Marshall Klaus Award. M.G. receives funding from the <a href="https://doi.org/10.13039/100000052">NIH</a> (R01DK118568, R01DK124614, and R01HD105301), the St. Louis Children’s Hospital Foundation, the Children's Discovery Institute of Washington University and St. Louis Children's Hospital, and the Department of Pediatrics at <a href="https://doi.org/10.13039/100011912">Washington University School of Medicine, St. Louis</a>. M.G. has also received sponsored research agreement funding from <a href="https://doi.org/10.13039/100007723">Takeda Pharmaceuticals</a> and Evive Biotech in the past year, but these funders did not play any role in this study.</p>
<h3>Author contributions</h3>
<p>Conceptualization, L.S.N. and M.G.; methodology, all authors; investigation, all authors.; writing – original draft, L.S.N. and M.G..; writing – review &amp; editing, all authors; funding acquisition, L.S.N. and M.G.; resources, M.G.; supervision, M.G.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>