<article>
<section>
<h1 id="header">Live and fixed imaging of translation sites at single mRNA resolution in the <i>Drosophila</i> embryo</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Daisy J. Vinter,<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Caroline Hoppe,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Hilary L. Ashe<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Faculty of Biology, Medicine and Health, University of Manchester, Manchester M13 9PT, UK</p>
<p id="fn1"><sup>2</sup>Present address: Department of Genetics, Yale University School of Medicine, New Haven, CT 06510, USA</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:daisy.vinter@postgrad.manchester.ac.uk">daisy.vinter@postgrad.manchester.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hilary.ashe@manchester.ac.uk">hilary.ashe@manchester.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100812">10.1016/j.xpro.2021.100812</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Significant regulation of gene expression is mediated at the translation level. Here, we describe protocols for imaging and analysis of translation at single mRNA resolution in both fixed and living <i>Drosophila</i> embryos. These protocols use the SunTag system, in which the protein of interest is visualized by inserting a peptide array that is recognized by a single chain antibody. Simultaneous detection of individual mRNAs using the MS2/MCP system or by smFISH allows translation sites to be identified and quantified.</p>
<p>For complete information on the generation and use of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref24">Vinter et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Visualization of translation of single mRNAs in <i>Drosophila</i> using SunTag labeling</li>
<li>Fixed embryo analysis reveals the mRNA translation efficiency in each expressing cell</li>
<li>Live SunTag confocal imaging uncovers mRNA translation dynamics</li>
<li>Image analysis pipelines allow quantitative and temporal information to be extracted</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Experimental design considerations</h3>
<p>Introduction of the SunTag array into the protein of interest (POI), and subsequent binding of a single chain antibody-fluorescent protein fusion (scFv-FP) to the nascent SunTag peptides (<a href="#fig1">Figure 1</a>A), allows translation to be visualized in the <i>Drosophila</i> embryo (<a href="#bib3">Dufourt et al., 2021</a>; <a href="#bib24">Vinter et al., 2021</a>). Insertion of SunTag at the start of the coding sequence enables rapid visualization of translation sites and maximizes their signal, facilitating early detection and quantitation.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of the SunTag imaging system</div>
<p>(A) Schematic showing the SunTag system used to detect translation sites in live embryos. scFv-mNeonGreen binds the SunTag peptides as they are translated by the ribosome.</p>
<p>(B) Schematic showing the SunTag detection used in fixed embryos. smFISH probes complementary to the SunTag sequences are used to identify individual mRNAs, and anti-mNeonGreen antibodies are used to detect scFv-mNeonGreen.</p>
<p>(C) Schematic showing the SunTag system used for imaging of single translation sites in live embryos. 128 MS2 loops are inserted into the 3′UTR. tdMCP-2xmRuby3-CAAX binds the MS2 loops and the CAAX sequence anchors the translating mRNA complex to the plasma membrane.</p>
</figcaption>
</figure>
<p>Single molecule fluorescent in situ hybridization (smFISH) is used with SunTag labeling (<a href="#fig1">Figure 1</a>B) to visualize translation of individual mRNAs in fixed embryos, whereas the MS2/MCP system for mRNA detection is exploited for live imaging of translation. Insertion of 128 MS2 copies gives the required sensitivity for single mRNA detection (<a href="#bib24">Vinter et al. 2021</a>; <a href="#bib3">Dufourt et al. 2021</a>). Introduction of the MS2 sequences into the 3′UTR is optimal, as stem loops in the 5′UTR can disrupt ribosome scanning and reduce translation efficiency (<a href="#bib19">Palam et al., 2011</a>; <a href="#bib23">Vattem and Wek, 2004</a>). If inserted into the coding sequence, the MS2 loops will be unwound by the translating ribosome, which will displace bound MCP and disrupt labeling of the mRNA (<a href="#bib8">Halstead et al., 2015</a>). As 3′UTR sequences are known to bind regulatory factors (<a href="#bib15">Mayr, 2017</a>), insertion of the MS2 loops at the beginning of the 3′UTR will leave it otherwise intact, reducing the potential of the MS2 loops to disrupt regulation. To track and image mRNAs for extended periods of time (minutes), an MCP-FP-CAAX fusion is used, which reduces mRNA movement via anchoring to the plasma membrane (<a href="#bib24">Vinter et al., 2021</a>; <a href="#bib26">Yan et al., 2016</a>) (<a href="#fig1">Figure 1</a>C).</p>
<p>We describe methods for imaging mRNA translation in fixed samples (<a href="#sec3.1">Section 1: Fixed imaging of individual mRNAs and translation sites</a>) and live imaging of both translation sites and anchored single mRNAs (<a href="#sec3.2">Section 2: Live imaging of translation sites</a>). Analysis pipelines for detecting mRNAs and determining their translation efficiency and ribosome density are also presented. The protocols described here have been optimized for the <i>scFv-mNeonGreen</i> (<i>scFv-mNG</i>), <i>SunTag-hunchback(hb)(+/-128xMS2)</i> and <i>tdMCP-mRuby3-CAAX</i> fly lines (<a href="#bib24">Vinter et al. 2021</a>), which we use as examples throughout. However, the protocols and analysis pipelines can be extended to any SunTag-POI or scFv-FP.</p>
<h3 id="sec1.2">Generation of the SunTag-POI fly line(s)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼8 or ∼12 weeks to generate a homozygous transgenic fly stock or CRISPR line, respectively</div>
<ol>
<li>Generate a fly line expressing the protein of interest with the SunTag sequences (SunTag-POI) either by genome editing or by introducing a transgene. MS2 sequences should also be inserted in the 3′UTR if live imaging of the mRNAs is required.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When using a SunTag-POI transgene it is important to include the coding sequence and both UTRs to recapitulate regulation of the endogenous mRNA more accurately.</div>
<div class="note">
<span class="note-title">Note:</span> Generation of SunTag-POI transgenics by phiC31-mediated integration (<a href="#bib1">Bischof et al., 2007</a>; <a href="#bib4">Fish et al., 2007</a>) allows an appropriate landing site to be used to avoid position effects. This is especially important when testing, for example, mutant transgenes in follow up experiments.</div>
<div class="note">
<span class="note-title">Note:</span> Detailed CRISPR protocols have been described (<a href="#bib9">Hoppe and Ashe, 2021a</a>; <a href="#bib27">Yu et al., 2021</a>), including methods for scarless editing (<a href="#bib12">Lamb et al., 2017</a>; <a href="#bib18">Nyberg et al., 2020</a>).</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse anti-mNeonGreen</td>
<td>Chromotek</td>
<td>RRID: AB_2827566</td>
</tr>
<tr>
<td>Donkey anti-Mouse IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, AlexaFluor 488</td>
<td>Thermo Fisher Scientific</td>
<td>RRID: AB_141607</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Methanol, Certified AR for Analysis</td>
<td>Fisher Scientific</td>
<td>M/4000/17</td>
</tr>
<tr>
<td>Bleach</td>
<td>Banner</td>
<td>Cat #: 1190012</td>
</tr>
<tr>
<td>Halocarbon Oil 700</td>
<td>Sigma</td>
<td>Cat #: H8898</td>
</tr>
<tr>
<td>Halocarbon Oil 27</td>
<td>Sigma</td>
<td>Cat #: H8773</td>
</tr>
<tr>
<td>DAPI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #: D1306</td>
</tr>
<tr>
<td>SSC Buffer 20<b>×</b> Concentrate</td>
<td>Sigma</td>
<td>Cat #: S6639</td>
</tr>
<tr>
<td>Formamide, Spectrophotometric Grade, ≥99%</td>
<td>Honeywell</td>
<td>Cat #: 295876</td>
</tr>
<tr>
<td>Formaldehyde Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #: 15627860</td>
</tr>
<tr>
<td>Heptane</td>
<td>Sigma</td>
<td>Cat #: H2198</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma</td>
<td>Cat #: P9416</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS) tablets</td>
<td>Sigma</td>
<td>Cat #: P4417</td>
</tr>
<tr>
<td>Dextran sulfate sodium salt from <i>Leuconostoc</i> spp.</td>
<td>Sigma</td>
<td>Cat #: D8906</td>
</tr>
<tr>
<td>ProLong Diamond Antifade Mountant</td>
<td>Thermo Fisher</td>
<td>Cat #: P36961</td>
</tr>
<tr>
<td>TE, pH 8.0, RNase-free</td>
<td>Thermo Fisher</td>
<td>Cat #: AM9849</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma</td>
<td>Cat #: X100</td>
</tr>
<tr>
<td>Ethylene glycol-bis(2-aminoethylether)-<i>N,N,N′,N′</i>-tetraacetic acid (EGTA)</td>
<td>Sigma</td>
<td>Cat #: E3889</td>
</tr>
<tr>
<td>Western Blocking Reagent, Solution</td>
<td>Sigma</td>
<td>Cat #: 11921673001</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>D.</i> <i>melanogaster</i> <i>w</i><sup><i>∗</i></sup> ; <i>nos-</i><i>scFv-mNeongreen</i>
</td>
<td>(<a href="#bib24">Vinter et al., 2021</a>)</td>
<td>Available on request</td>
</tr>
<tr>
<td>
<i>D.</i> <i>melanogaster</i> <i>w</i><sup><i>∗</i></sup> ;<i>; hbP2&gt;SunTag-hb</i>
</td>
<td>(<a href="#bib24">Vinter et al., 2021</a>)</td>
<td>Available on request</td>
</tr>
<tr>
<td>
<i>D.</i> <i>melanogaster</i> <i>w</i><sup><i>∗</i></sup> ;; <i>hbP2&gt;SunTag-hb-128xMS2</i>
</td>
<td>(<a href="#bib24">Vinter et al., 2021</a>)</td>
<td>Available on request</td>
</tr>
<tr>
<td>
<i>D.</i> <i>melanogaster</i> <i>w</i><sup><i>∗</i></sup> ;<i>;</i> <i>nos-</i><i>tdMCP-mRuby3-CAAX</i>
</td>
<td>(<a href="#bib24">Vinter et al., 2021</a>)</td>
<td>Available on request</td>
</tr>
<tr>
<td>
<i>D.</i> <i>melanogaster</i> <i>His-2Av-RFP</i>
</td>
<td>Bloomington Drosophila Stock Center</td>
<td>BL23650</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>SunTag Stellaris® smFISH probes (Quasar 570 tagged at the 3′ end)</td>
<td>Biosearch Technologies</td>
<td>See <a href="#tbl1">Table 1</a>
</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Example datasets</td>
<td>This study</td>
<td>Fixed data: Mendeley data <a href="https://doi.org/10.17632/3h73rbvbt9.1">https://doi.org/10.17632/3h73rbvbt9.1</a>Live data: Mendeley data <a href="https://doi.org/10.17632/5vs879zvz9.1">https://doi.org/10.17632/5vs879zvz9.1</a>
</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Imaris 9.2</td>
<td>Bitplane</td>
<td>RRID:SCR_007370</td>
</tr>
<tr>
<td>Huygens Professional Deconvolution software</td>
<td>SVI</td>
<td>RRID:SCR_014237-</td>
</tr>
<tr>
<td>Software for assignment to nucleus ‘sass’</td>
<td>(<a href="#bib11">Hoppe et al., 2020</a>; <a href="#bib24">Vinter et al., 2021</a>)</td>
<td>
<a href="https://github.com/TMinchington/sass">https://github.com/TMinchington/sass</a>, RRID:SCR_018797</td>
</tr>
<tr>
<td>Software for removal of nuclear spots</td>
<td>This study; (<a href="#bib24">Vinter et al., 2021</a>)</td>
<td><a href="https://github.com/DaisyVinter/StarProtocols_Vinter_2021">https://github.com/DaisyVinter/StarProtocols_Vinter_2021</a></td>
</tr>
<tr>
<td>Software for calculation of ribosome number</td>
<td>This study; (<a href="#bib24">Vinter et al., 2021</a>)</td>
<td><a href="https://github.com/DaisyVinter/StarProtocols_Vinter_2021">https://github.com/DaisyVinter/StarProtocols_Vinter_2021</a></td>
</tr>
<tr>
<td>Airlocalize</td>
<td><a href="#bib21">Trcek et al. 2017</a></td>
<td>Available on request from T. Lionnet</td>
</tr>
<tr>
<td>Python 3.7</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Fiji</td>
<td>(<a href="#bib28">Schindelin, 2012</a>)</td>
<td>RRID: SCR 002285</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Coverslips Nr. 1, 18 <b>×</b> 18 mm</td>
<td>Deltalab</td>
<td>Cat #: D101818</td>
</tr>
<tr>
<td>Coverslips Nr. 0, 18 <b>×</b> 18 mm</td>
<td>Scientific Laboratory Supplies</td>
<td>Cat #: MIC3100</td>
</tr>
<tr>
<td>Coverslips Nr. 1, 24 <b>×</b> 50 mm</td>
<td>Deltalab</td>
<td>Cat #: D102450</td>
</tr>
<tr>
<td>Slides, 26 <b>×</b> 76 mm</td>
<td>Deltalab</td>
<td>Cat #: D100004</td>
</tr>
<tr>
<td>Scintillation vial, glass, cap, 20 mL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #: 03-337-15</td>
</tr>
<tr>
<td>V-Vials with solid-top screw cap, capacity 0.3 mL</td>
<td>Sigma</td>
<td>Cat #: Z115053-12EA</td>
</tr>
<tr>
<td>Lumox dish 50, cell culture dish</td>
<td>Sarstedt AG &amp; Co</td>
<td>Cat #: 94.6077.305</td>
</tr>
<tr>
<td>Yellow double-sided plastic tape, 25 mm <b>×</b> 20 m</td>
<td>RS Components</td>
<td>Cat #: 555-033</td>
</tr>
<tr>
<td>Packaging tape</td>
<td>Tesa</td>
<td>Cat #: 4124</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>Embryo washing solution</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>4 g/L</td>
<td>4 g</td>
</tr>
<tr>
<td>10% Triton X-100</td>
<td>0.03%</td>
<td>3 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>997 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Embryo washing solution can be kept for long periods at 20<sup>o</sup>C.</p>
<table id="undtbl2">
<thead>
<tr>
<th>Fixing buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10 <b>×</b> PBS</td>
<td>1.33 <b>×</b>
</td>
<td>13.3 mL</td>
</tr>
<tr>
<td>0.333 M EGTA (pH 8.0)</td>
<td>0.067M</td>
<td>20 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>66.7 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Fixing buffer can be stored at 20<sup>o</sup>C for extended periods.</p>
<table id="undtbl3">
<thead>
<tr>
<th>Stellaris hybridization buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dextran Sulfate</td>
<td>100mg/mL</td>
<td>300 mg</td>
</tr>
<tr>
<td>Formamide</td>
<td>10%</td>
<td>300 μL</td>
</tr>
<tr>
<td>20<b>×</b> SSC</td>
<td>2<b>×</b>
</td>
<td>300 μL</td>
</tr>
<tr>
<td>Nuclease free H<sub>2</sub>O</td>
<td>n/a</td>
<td>2.4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>3 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">This will make enough hybridization buffer for one sample. Excess hybridization buffer can be stored at −20°C but should only be freeze-thawed a maximum of twice.</p>
<table id="undtbl4">
<thead>
<tr>
<th>Stellaris Wash buffer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Formamide</td>
<td>n/a</td>
<td>500 μL</td>
</tr>
<tr>
<td>20<b>×</b> SSC</td>
<td>2<b>×</b>
</td>
<td>500 μL</td>
</tr>
<tr>
<td>Nuclease free H<sub>2</sub>O</td>
<td>n/a</td>
<td>4 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>5 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">This will make enough wash buffer for one sample. Make up fresh for each experiment and store in the fridge between day 1 and 2 of smFISH protocol.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Section 1: Fixed imaging of individual mRNAs and translation sites</h3>
<p>This section describes how to collect and fix embryos for detection of <i>SunTag-hb</i> mRNAs and their translation sites using a combination of smFISH and immunofluorescence (<a href="#fig2">Figure 2</a>). Methods to calculate total mRNA number, the number of mRNAs that are translated, and ribosome number in the translation sites are also described.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Overview of the protocols for visualizing translation sites</div>
<p>The steps associated with the detection and analysis of translation sites in fixed and live embryos are shown, with the relevant timings.</p>
</figcaption>
</figure>
<h4 id="sec3.1.1">Set up embryo collection cages</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days</div>
<p>In this step, flies are crossed to allow the collection of embryos with maternally expressed <i>scFv-mNG</i> and zygotically expressed <i>SunTag-hb</i>.</p>
<ol>
<li>Collect virgin females homozygous for the <i>nos-scFv-mNG</i> transgene and males homozygous for the <i>SunTag-hb</i> transgene.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This cross will produce embryos that zygotically express one copy of <i>SunTag-hb</i>. If visualization of translation of the full complement of mRNAs from both alleles is required, balanced stocks can be generated to allow the collection of embryos that maternally express <i>scFv-mNG</i> and are homozygous for the <i>SunTag-POI</i> transgene/CRISPR modified locus.</div>
<div class="note">
<span class="note-title">Note:</span> Control embryos can also be collected from stocks carrying only the <i>SunTag-POI</i> or the <i>scFv-FP</i> transgene if required.</div>
<div class="note">
<span class="note-title">Note:</span> Fly lines maternally expressing <i>scFv-sfGFP</i> and <i>scFv-mScarlet</i> (<a href="#bib3">Dufourt et al., 2021</a>) are also available. There are also <i>UASp-scFv-sfGFP</i> (<a href="#bib29">Lu et al., 2016</a>) and <i>UASp-scFv-GFP-NLS</i> (<a href="#bib5">Formicola et al., 2021</a>) stocks that are compatible with the GAL4/UAS system, allowing translation to be imaged in a cell type specific manner at later developmental stages (<a href="#bib5">Formicola et al., 2021</a>).</div>
<ol start="2">
<li>Transfer at least 100 females and 50 males to a collection cage with an apple juice agar plate and yeast paste.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Increasing the number of flies in the collection cage will increase the number of embryos collected.</div>
<h4 id="sec3.1.2">Collect and fix embryos</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>In this step, embryos are collected and fixed for use in smFISH and immunofluorescence to allow <i>SunTag-hb</i> translation sites to be visualized.</p>
<ol start="3">
<li>Change the plate on the collection cage to a fresh apple juice agar plate with yeast and collect embryos at 25°C for the desired length of time. For collection of a range of nuclear cycle (nc) 12 – nc14 embryos, let the flies lay embryos for 3 h, then remove the plate and age the embryos for 1 h at 25°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We do not recommend shifting collection temperatures to change the speed of development, because translation is temperature sensitive.</div>
<ol start="4">
<li>Dechorionate the embryos by applying 50% domestic bleach to the plate and incubating for 2 min.</li>
<li>Transfer the embryos to a sieve by washing them off the plate with water or embryo washing solution.
<ol type="a">
<li>When in the sieve, wash thoroughly with water or embryo washing solution until no bleach remains.</li>
</ol>
</li>
<li>Transfer embryos into a glass scintillation vial containing 3 mL fixing buffer.</li>
<li>Add 4 mL heptane and 1 mL formaldehyde and shake at 300 rpm for 20 min.</li>
<li>Remove the liquid layer at the bottom of the vial with a plastic Pasteur pipette.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Take care not to lose any embryos while removing the liquid. It is better to retain embryos than to remove all of the liquid layer.</div>
<ol start="9">
<li>Devitellinize the embryos.
<ol type="a">
<li>Add 8 mL of methanol to the glass vial.</li>
<li>Shake the vial vigorously for 30 s and vortex for 30 s.</li>
<li>Remove the liquid and repeat at least twice more until the liquid is clear and has no floating membrane.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that the devitellinization step is complete so that the staining and imaging are optimal. Use more methanol washes and shaking if necessary.</div>
<ol start="10">
<li>Transfer the embryos in methanol to a 1.5 mL eppendorf tube. Remove as much methanol as possible and replace with fresh methanol. Embryos can then be used for smFISH and immunofluorescence or kept in methanol at −20°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Embryos can be kept at −20°C for at least three months without a reduction in the quality of the smFISH and immunofluorescence staining.</div>
<div class="note">
<span class="note-title">Note:</span> Depending on the number of flies in the collection cage and the efficiency of laying, embryo collections may need to be carried out on multiple days to collect enough embryos for smFISH and immunofluorescence.</div>
<h4 id="sec3.1.3">smFISH and immunofluorescence</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>In this step, embryos are stained with <i>SunTag</i> smFISH probes to recognize individual mRNAs and an anti-mNeonGreen antibody to identify the scFv-mNG signal (<a href="#fig1">Figure 1</a>B). Steps are performed at 20<sup>o</sup>C unless stated otherwise.</p>
<p>Day 1:</p>
<ol start="11">
<li>Prepare the SunTag smFISH probes (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Sequences of the SunTag stellaris probes</caption>
<thead>
<tr>
<th>Probe name</th>
<th>Probe sequence (5′-3′)</th>
<th>Probe name</th>
<th>Probe sequence (5′-3′)</th>
</tr>
</thead>
<tbody>
<tr>
<td>SunTag_1</td>
<td>Ccacttcgttctcaagatga</td>
<td>SunTag_25</td>
<td>cgcgacttcgttctctaaat</td>
</tr>
<tr>
<td>SunTag_2</td>
<td>gccagaacctttcttaagac</td>
<td>SunTag_26</td>
<td>ttcgataagagttcttcgcc</td>
</tr>
<tr>
<td>SunTag_3</td>
<td>ttgaaagcagttcttctcca</td>
<td>SunTag_27</td>
<td>ctcattttcgaggtggtagt</td>
</tr>
<tr>
<td>SunTag_4</td>
<td>ctcattttccaggtggtaat</td>
<td>SunTag_28</td>
<td>gtggtagttcttgctcaag</td>
</tr>
<tr>
<td>SunTag_5</td>
<td>aatttttgctcagcaactcc</td>
<td>SunTag_29</td>
<td>attcttgctgagcaattcct</td>
</tr>
<tr>
<td>SunTag_6</td>
<td>ttctttagtcgtgctacttc</td>
<td>SunTag_30</td>
<td>cgacttcgttctccaaatga</td>
</tr>
<tr>
<td>SunTag_7</td>
<td>tttcgagagtaactcctcac</td>
<td>SunTag_31</td>
<td>tttactcaacaattcctccc</td>
</tr>
<tr>
<td>SunTag_8</td>
<td>ccacttcgttttcgagatga</td>
<td>SunTag_32</td>
<td>cgacttcattttccaagtgg</td>
</tr>
<tr>
<td>SunTag_9</td>
<td>gataatagctcttctccaga</td>
<td>SunTag_33</td>
<td>ttgctcaataactcttcgcc</td>
</tr>
<tr>
<td>SunTag_10</td>
<td>tcattttcgaggtggtagtt</td>
<td>SunTag_34</td>
<td>ttcgttctccaagtggtaat</td>
</tr>
<tr>
<td>SunTag_11</td>
<td>acttcccttttttaagcgtg</td>
<td>SunTag_35</td>
<td>agttcttcgataagagctcc</td>
</tr>
<tr>
<td>SunTag_12</td>
<td>tcttggatagtagctcttca</td>
<td>SunTag_36</td>
<td>gcgacttcattctctaagtg</td>
</tr>
<tr>
<td>SunTag_13</td>
<td>ctacctcgttctcaagatga</td>
<td>SunTag_37</td>
<td>agtggtagttcttgctcaag</td>
</tr>
<tr>
<td>SunTag_14</td>
<td>tagttcttcgagagcagttc</td>
<td>SunTag_38</td>
<td>ttagatagtaactcttcccc</td>
</tr>
<tr>
<td>SunTag_15</td>
<td>gatcccttttttaatcgagc</td>
<td>SunTag_39</td>
<td>cctcgttctcgagatgataa</td>
</tr>
<tr>
<td>SunTag_16</td>
<td>tgaaagtagttcctcaccac</td>
<td>SunTag_40</td>
<td>gatagttcttcgacaggagt</td>
</tr>
<tr>
<td>SunTag_17</td>
<td>cttcgttttcgaggtggtaa</td>
<td>SunTag_41</td>
<td>cctttttaagtcttgcaacc</td>
</tr>
<tr>
<td>SunTag_18</td>
<td>ccctgaacctttctttaatc</td>
<td>SunTag_42</td>
<td>ttcttactgagtagttcctc</td>
</tr>
<tr>
<td>SunTag_19</td>
<td>tactcagtaattcttcaccc</td>
<td>SunTag_43</td>
<td>tcctgatcctttcttcaaac</td>
</tr>
<tr>
<td>SunTag_20</td>
<td>ctacctcattttccagatga</td>
<td>SunTag_44</td>
<td>cttttgagagcagttcttcg</td>
</tr>
<tr>
<td>SunTag_21</td>
<td>tttcgatagcaactcttcgc</td>
<td>SunTag_45</td>
<td>gcaacctcattttccaaatg</td>
</tr>
<tr>
<td>SunTag_22</td>
<td>tttttgagcctagcaacttc</td>
<td>SunTag_46</td>
<td>tgccacttcccttttttaaa</td>
</tr>
<tr>
<td>SunTag_23</td>
<td>agtggtagtttttcgagagc</td>
<td>SunTag_47</td>
<td>tttcgacagaagttcctcac</td>
</tr>
<tr>
<td>SunTag_24</td>
<td>tttgctcaataactcctcgc</td>
<td>SunTag_48</td>
<td>taagtcgggctacttcattc</td>
</tr>
</tbody>
</table>
<ol type="a">
<li>Reconstitute the lyophilized probes in 400μL RNase-free TE and mix by pipetting. This will produce a probe stock with a concentration of 12.5μM.</li>
<li>Make 20μL aliquots of the dissolved probes. Store at −20°C and protect from the light.</li>
</ol>
</li>
<li>Transfer the embryos to a glass V-Vial using a pipette with a cut tip. A volume of 50 μL of embryos per stain is optimal because some embryos are lost throughout the staining process.</li>
<li>Perform washes with 500 μL of the following solutions.
<ol type="a">
<li>Rock in 50:50 methanol:PBT (1 <b>×</b> PBS + 0.1% Tween-20) for 5 min.</li>
<li>Rock in PBT for 10 min. Repeat 3 more times.</li>
<li>Rock in 50% PBT, 50% Stellaris wash buffer for 10 min.</li>
<li>Rock twice in Stellaris wash buffer for 5 min.</li>
<li>Add 500 μL Stellaris wash buffer and incubate for 2 h in a 37°C water bath.</li>
</ol>
</li>
<li>Stain with SunTag smFISH probes.
<ol type="a">
<li>Dilute the SunTag probes to a final concentration of 50 nM in 300 μL Stellaris hybridization buffer.</li>
<li>Incubate the embryos in the probe mixture in a 37°C water bath for 12–18 h.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Incubate the embryos in the probe mixture for at least 12 h. Keep the Stellaris wash and hybridization buffers at 4°C for the duration of the experiment; fresh Stellaris wash buffer should be made up for each experiment.</div>
<div class="critical">
<span class="critical-title">Critical:</span> After probe hybridization, embryos should be kept in the dark as far as possible. Cover the V-Vials with foil to maintain darkness.</div>
<div class="note">
<span class="note-title">Note:</span> Using SunTag probes has the advantage that they are generic for different SunTag-POI mRNAs. Moreover, detection is limited to mRNAs carrying SunTag sequences, excluding mRNAs transcribed from the endogenous/unmodified locus, which simplifies the analysis.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> SunTag smiFISH (single molecule inexpensive FISH) probes, which have specific sequences against the SunTag and a common flap sequence for the hybridization of fluorescently labelled probes (<a href="#bib22">Tsanov et al., 2016</a>), may be used to reduce costs. In this case, incubate smiFISH probes with fluorescently tagged flap probes prior to dilution in Stellaris hybridization buffer (<a href="#bib2">Calvo et al., 2021</a>). We have not tested smiFISH probes against SunTag sequences, but have used them to detect other individual mRNAs (Hoppe et al, unpublished).</div>
<p>Day 2:</p>
<div class="critical">
<span class="critical-title">Critical:</span> Maintain the embryos in darkness as much as possible on Day 2.</div>
<ol start="15">
<li>Pre-warm Stellaris hybridization and wash buffers to 37°C in a water bath.</li>
<li>Perform washes to remove unbound probes with 500 μL of the following solutions.
<ol type="a">
<li>Wash embryos in Stellaris hybridization buffer for 30 min in a 37°C water bath.</li>
<li>Rinse with Stellaris wash buffer.</li>
<li>Wash embryos in Stellaris wash buffer for 15 min in a 37°C water bath. Repeat twice.</li>
<li>Rock in Stellaris wash buffer for 15 min.</li>
<li>Rock 3 times in PBT for 10 min.</li>
<li>Rock in PBT + 1 <b>×</b> Western Blocking Reagent (WBR) for 30 min.</li>
</ol>
</li>
<li>Incubate with the primary antibody.
<ol type="a">
<li>Dilute anti-mNeonGreen antibody (Chromotek, RRID: AB_2827566) 1:250 in PBT + 1 <b>×</b> WBR.</li>
<li>Rock in the primary antibody solution at 4°C for 12–18 h.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The primary antibody incubation can be shortened to 2 h at 20<sup>o</sup>C, allowing the protocol steps described for days 2 and 3 to be completed in a single day.</div>
<div class="note">
<span class="note-title">Note:</span> Not all antibodies are compatible with the smFISH protocol so new antibodies will have to be tested in combination with the smFISH protocol.</div>
<p>Day 3:</p>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the embryos in the dark where possible on Day 3.</div>
<ol start="18">
<li>Perform washes to remove any unbound antibody with 500 μL of the following solutions.
<ol type="a">
<li>Rinse the embryos with PBT twice.</li>
<li>Rock the embryos 4 times in PBT for 15 min.</li>
<li>Rock the embryos in PBT + 1 <b>×</b> WBR for 30 min.</li>
</ol>
</li>
<li>Incubate with the secondary antibody.
<ol type="a">
<li>Dilute anti-mouse AlexaFluor 488 (Thermo Fisher Scientific, RRID: AB_141607) at 1:250 in PBT + 1 <b>×</b> WBR.</li>
<li>Rock the embryos in the secondary antibody solution for 2 h at 20<sup>o</sup>C.</li>
</ol>
</li>
<li>Perform washes in 500 μL volumes.
<ol type="a">
<li>Rinse the embryos twice with PBT.</li>
<li>Rock twice in PBT for 15 min.</li>
<li>Dilute 500 μg/mL DAPI at 1:1000 in PBT. Rock embryos in DAPI solution for 15 min.</li>
<li>Rock in PBT for 15 min.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If using ProLong Diamond mountant with DAPI, omit the DAPI incubation step.</div>
<ol start="21">
<li>Mount embryos.
<ol type="a">
<li>Remove as much PBT as possible.</li>
<li>Add approximately 3 drops of ProLong Diamond to the embryos.</li>
<li>Use a glass pipette to draw up the embryos and pipette them onto a glass slide.</li>
<li>Carefully cover with a Nr 1, 24 mm <b>×</b> 50 mm coverslip by lowering onto the slide with tweezers.</li>
</ol>
</li>
<li>Dry the slides for 12–18 h at 20<sup>o</sup>C in the dark. The slides can then be used for imaging or can be stored at −20°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Slides can be stored at −20°C for at least 3 months without loss of staining, but we recommend imaging the embryos as soon as possible.</div>
<h4 id="sec3.1.4">Imaging</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h per image, dependent on image size</div>
<p>In this step, embryos are imaged using confocal microscopy. Images can be taken at the level of the expression domain, and the total numbers of mRNAs and translated mRNAs can be calculated. In addition, higher magnification images can be taken and used to calculate the number of ribosomes on each mRNA.</p>
<ol start="23">
<li>Image the stained embryos using confocal microscopy. 3 channels will be required, for SunTag smFISH, mNeonGreen staining, and DAPI (<a href="#sec6.1">Troubleshooting 1</a>). For analysis of translation along the anterior-posterior axis, align the axis diagonally to maximize the area of embryo imaged (see the example in <a href="#fig6">Figure 6</a>A) and for compatibility with axis calculation in the spots assignment script used in Step 34.
<ol type="a">
<li>For the example whole expression domain dataset, we used a Leica TCS SP8 AOBS inverted confocal microscope with 63<b>×</b>/1.40 HCX PL APO CS oil objective and 0.75<b>×</b> optical zoom, and white light laser set to 70%.
<ol type="i">
<li>Set the pinhole to 1 airy unit and scan speed to 400 Hz, with 2<b>×</b> line averaging and bit depth of 8. Acquire an image at 2048 x 2048 pixels, resulting in a pixel size of 0.12 μm x 0.12 μm, and z-step size of 300 nm, for 107 z-steps, giving an imaging depth of 31.8 μm. The z-depth should include the most apical to the most basal fluorescent signal in the SunTag smFISH and the scFv-mNG channels.</li>
<li>Collect the DAPI channel with 405 nm excitation (2%), using a PMT detector with collection set from 415 – 470 nm.</li>
<li>Collect the scFv-mNG (AlexaFluor 488) channel with 488 nm excitation (10%), using a hybrid detector with collection set from 500 – 540 nm.</li>
<li>Collect the SunTag smFISH (Quasar 570) channel with 548 nm excitation (15%) using a hybrid detector with collection set from 558 – 640 nm.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the sample slide reaches 20<sup>o</sup>C before imaging.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The expression domain can be aligned along the x or y axis, allowing cellular outputto be analyzed using the x or y position of nuclei.</div>
<div class="note">
<span class="note-title">Note:</span> For whole embryo/expression domain imaging, use an objective with magnification sufficient to capture the whole expression domain. We use a laser power for the SunTag smFISH channel which overexposes the transcription sites allowing individual mRNAs to be reliably imaged and detected by the analysis software. If the number of mRNAs within the transcription site is to be analyzed, a lower laser power must be used to prevent overexposure.</div>
<div class="note">
<span class="note-title">Note:</span> Imaging at least three biological replicates (embryos) is recommended as there will be cell-to-cell and embryo-to-embryo variation, especially when analyzing mutant genotypes. Images taken with the example settings will provide ∼250 (nc12) – 1000 (nc14) nuclei per embryo.</div>
<ol start="24">
<li>Image a high magnification region of a stained embryo for calculation of ribosome number.
<ol type="a">
<li>For the example ribosome number imaging dataset, we used a Leica TCS SP8 AOBS inverted confocal microscope with 100<b>×</b>/1.40 HCX PL APO oil objective and 3<b>×</b> optical zoom, and white light laser set to 70%.
<ol type="i">
<li>Set the pinhole to 0.65 airy unit and scan speed to 400 Hz, with 4<b>×</b> line averaging and bit depth of 8. Acquire an image at 4096 x 4096 pixels, resulting in a pixel size of 0.009 μm x 0.009 μm, and z-step size of 200 nm. The example dataset contains 64 slices, resulting in a total imaging depth of 12.6 μm in the z direction.</li>
<li>Collect the DAPI channel with 405 nm excitation (11%), using a PMT detector with collection set from 415 – 470 nm.</li>
<li>Collect the scFv-mNG (AlexaFluor 488) channel with 488 nm excitation (12%), using a hybrid detector with collection set from 500 – 540 nm, and gating of 1–6 μs.</li>
<li>Collect the SunTag smFISH (Quasar 570) channel with 548 nm excitation (20%) using a hybrid detector with collection set from 558 – 640 nm, and gating of 1–6 μs.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To calculate the number of ribosomes, translation sites must not be overexposed in order to capture the total amount of fluorescence. Ensure there are no overexposed pixels in the scFv-mNG channel.</div>
<ol start="25">
<li>
<b>(Optional)</b> Deconvolve images using Huygens Professional Deconvolution Software (SVI).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We deconvolve the images to improve the signal-to-noise ratio. However, this is not essential if deconvolution is not preferred, and the raw fluorescence will be sufficient to analyze the datasets.</div>
<h4 id="sec3.1.5">Analysis of the number of translated mRNAs</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼5 h per image, dependent on image size and the number of mRNAs, including the time for running the assignment script.</div>
<p>In these steps, the whole expression domain images are analyzed using Imaris 9.2 and a custom python script is used to quantify the total mRNA number and the number that are translated (<a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Overview of the analysis of translation sites from fixed embryo images</div>
<p>(A) The steps show how the mRNAs and scFv-mNG signals are detected, colocalized to identify translation sites, then assigned to the nearest nucleus.</p>
<p>(B) The steps show how to analyze the high magnification images to calculate the number of ribosomes per mRNA.</p>
</figcaption>
</figure>
<ol start="26">
<li>Load the image into Imaris 9.2. Most common image formats can be opened in Imaris 9.2. If using a later version of Imaris, run the image through the Imaris File Converter to convert the image to .ims format.</li>
<li>Using the Imaris ‘spots’ function, create spots to identify the mRNAs based on the Suntag smFISH signals (<a href="#fig3">Figure 3</a>A, Step 1). Follow the steps in the ‘spots’ creation wizard.
<ol type="a">
<li>Leave the first screen as default settings.</li>
<li>Select the channel used for SunTag smFISH.</li>
<li>Estimate the XY diameter by moving to ‘Slice’ view and measuring the diameter of individual mRNA foci. We find a diameter of 0.6 μm to be optimal but this will vary depending on the imaging settings.</li>
<li>Select ‘Model PSF-elongation along Z-axis’ and ‘Background Subtraction’.</li>
<li>To remove transcription sites from analysis, set the ‘Filter Type’ to ‘Intensity Sum Ch=X Img=1’ where Channel X is the channel used for SunTag smFISH that is currently being worked on. Select only spots which are low in fluorescence (the majority of the spots), removing the tail of spots with high fluorescence. This will remove any nascent transcription sites (brighter than individual mRNAs) from the analysis.</li>
<li>Add a filter with ‘Filter Type’ set to ‘Quality’. Drag the filter until all mRNA spots are selected.</li>
<li>Complete the spots wizard.</li>
<li>Name the spots ‘SunTag mRNA spots’.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It can be difficult to identify spots by eye when there is a large number of mRNAs in one area. We find it is simplest to change the quality filter while looking at a region with less mRNAs, for example the edge of an expression domain. Ensure that each mRNA molecule in this region is selected without any false positives (<a href="#fig3">Figure 3</a>A, Step 1). Check that no excess spots are selected by toggling on and off the ‘Spots’ view under ‘Scene’.</div>
<ol start="28">
<li>Use the Imaris ‘spots’ function to identify the potential translation sites based on the scFv-mNG signal (<a href="#fig3">Figure 3</a>A, Step 1). Follow the steps in the ‘spots’ creation wizard.
<ol type="a">
<li>Leave the first screen as the default settings.</li>
<li>Select the channel used for scFv-mNG.</li>
<li>Estimate the XY diameter by moving to ‘Slice’ view and measuring the diameter of individual scFv-mNG foci. We find a diameter of 0.5 μm to be optimal but this will vary depending on the imaging settings and the size of the scFv-mNG translation sites, which depends on translation efficiency. Note that the scFv-mNG foci will vary in size – use the diameter of the smaller spots to ensure all are captured.</li>
<li>Select ‘Model PSF-elongation along Z-axis’ and ‘Background Subtraction’.</li>
<li>Set the ‘Filter Type’ to ‘Quality’. Move the filter until all scFv-mNG spots are captured.</li>
<li>Complete the spots wizard.</li>
<li>Name the spots ‘scFv spots’.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The amount of background in the scFv-mNG channel will increase as translation occurs, due to increased numbers of SunTag-test proteins that have been translated and released, still bound by scFv-mNG. This can make distinguishing scFv-mNG translation sites from background more difficult. In this case, we recommend capturing all the translation sites (identifiable visually by colocalization with mRNAs) as spots. While most background signals will be avoided, this threshold may cause Imaris to detect some background spots (for example, in the nucleus) (<a href="#fig3">Figure 3</a>A, Step 1). However, as we call scFv-mNG signals as translation sites only if they have an overlapping smFISH signal, the colocalization step will eliminate scFv-mNG background spots that lack a colocalized mRNA (<a href="#sec6.3">Troubleshooting 2</a>).</div>
<ol start="29">
<li>Colocalize the SunTag mRNA and scFv spots to identify translation sites based on the presence of both an smFISH and scFv-mNG signal (<a href="#fig3">Figure 3</a>A, Step 2).
<ol type="a">
<li>In the ‘X-tensions’ tab, select the ‘Co-localize spots’ X-tension.</li>
<li>Select the two spots masks to co-localize –‘SunTag mRNA spots’ and ‘scFv spots’.</li>
<li>Input the threshold to call co-localization. We use a threshold of 0.3 (300 nm), which was chosen based on our data but is also consistent with the distance used in a previous SunTag study (<a href="#bib25">Wu et al., 2016</a>).</li>
</ol>
</li>
<li>Using the Imaris ‘Surfaces’ function, create surfaces to identify nuclei. Nuclei will be used to remove nuclear mRNAs from the analysis (optional) and to assign mRNAs to their closest nucleus. Follow the steps in the ‘surfaces’ creation wizard (<a href="#fig3">Figure 3</a>A, Step 3).
<ol type="a">
<li>Leave the first screen as the default settings.</li>
<li>Select the channel used for the nuclear stain.</li>
<li>Select ‘Smooth’ and set ‘Surfaces Detail’ to 0.5 μm.</li>
<li>Estimate the nuclear size by moving to ‘Slice’ view and measuring the diameter of individual nuclei.</li>
<li>Select ‘Background Subtraction (Local Contrast)’ and set ‘Diameter of largest Sphere which fits into the Object’ as the value measured in step d.</li>
<li>Change the Threshold (Background Subtraction) filter until all nuclei are covered.</li>
<li>Select ‘Enable’ to Split touching Objects. Set the ‘Seed Points Diameter’ to the value used for ‘Diameter of largest Sphere which fits into the Object’.</li>
<li>Set the ‘Filter Type’ to ‘Quality’ and drag until all nuclei have an object.</li>
<li>Complete the surfaces wizard.</li>
<li>Name the surfaces ‘nuclei’.</li>
</ol>
</li>
<li>
<b>(Optional)</b> Identify mRNAs inside the nucleus and check if any translation sites have been assigned inside the nucleus (<a href="#fig3">Figure 3</a>A, Step 4). This step is used to remove nuclear mRNAs so that the proportion of translated mRNAs is only calculated based on the total number of cytoplasmic mRNAs, which are accessible to the translation machinery. As there should be no translation sites identified in the nucleus, this step also functions as a control to check whether there are any false positive translation sites.
<ol type="a">
<li>Select ‘Suntag mRNA spots’ and run the Imaris X-tension ‘Split spots into surface objects’.</li>
<li>Select ‘scFv spots co-located’ and run the Imaris X-tension ‘Split spots into surface objects’.</li>
</ol>
</li>
<li>Export the Imaris data.
<ol type="a">
<li>Export the ‘SunTag mRNA spots’ data from step 27. This represents the total mRNA data. If removal of nuclear mRNAs is required, also export the ‘Suntag mRNA spots inside Surfaces’ folder from step 31a.</li>
<li>Export the ‘scFv spots co-located’ data, the translated mRNA data. If removal of any false positive translation sites is needed, also export the ‘scFv spots co-located inside Surfaces’ folder.</li>
<li>Export the ‘nuclei’ data.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure that ‘Position’ and all files starting with ‘Intensity’ are included in the data exported for ‘Suntag mRNA spots’ and ‘scFv spots co-located’, and that ‘Position’ and all files starting with ‘Ellipsoid Axis’ are included in the data exported for ‘nuclei’.</div>
<ol start="33">
<li>
<b>(Optional)</b> If removal of nuclear mRNAs is required for the analysis, run the data through the remove_nuclear.py script (<a href="https://github.com/DaisyVinter/StarProtocols_Vinter_2021">https://github.com/DaisyVinter/StarProtocols_Vinter_2021</a>) (<a href="#fig3">Figure 3</a>A, Step 5).
<ol type="a">
<li>Create a folder containing the ‘Suntag mRNA spots’ and ‘Suntag mRNA spots inside Surfaces’ data.</li>
<li>In Anaconda Prompt, type: python. Drag and drop the remove_nuclear.py script, and then drag and drop the folder created in step a.</li>
<li>The script will create two folders in the folder created in step a – ‘Suntag mRNA cytoplasmic spots’ and ‘Suntag mRNA nuclear spots’, containing the separate Imaris outputs for cytoplasmic and nuclear mRNAs. Use the ‘Suntag mRNA cytoplasmic spots’ folder in step 34a if only the cytoplasmic mRNA output is required.</li>
<li>Repeat steps a to c for the ‘scFv spots co-located’ and ‘scFv spots co-located inside Surfaces’ data.</li>
</ol>
</li>
<li>Run the data through the spotME_EmbryoMid.py script (<a href="https://github.com/TMinchington/sass">https://github.com/TMinchington/sass</a>) (<a href="#fig3">Figure 3</a>, Step 6). In this script, the total mRNAs and translated mRNAs are assigned to the nearest nuclei, to reveal the numbers in each of these virtual cell territories across the expression domain.
<ol type="a">
<li>Create a folder containing the ‘SunTag mRNA spots’ data (or ‘Suntag mRNA cytoplasmic spots’ if step 33 is used) and ‘nuclei’ data.</li>
<li>In Anaconda Prompt, type: python. Drag and drop the spotME_EmbryoMid.py script and then drag and drop the folder created in step a.</li>
<li>The output required is the ‘all_data_with_distance.txt’ in the ‘time_data’ folder. This is the data output for all mRNAs.</li>
<li>Repeat steps a to c with the ‘scFv spots co-located’ data instead of ‘SunTag mRNA spots’ data. This will give the data output for translated mRNAs only.</li>
<li>The ‘all_data_with_distance.txt’ file gives the number of mRNA or translation site spots assigned to each nearest nucleus, the x, y, and z position of each spot and nucleus, and the Imaris intensity data for each spot in each channel. In addition, the script calculates the anterior-posterior axis of embryos positioned as described in Step 23. The midline.png file shows the axis position (<a href="#fig6">Figure 6</a>B). The ‘all_data_with_distance.txt’ file also shows the position of each nucleus along the anterior posterior axis, and the perpendicular distance from the axis. If the embryo axis has instead been lined up along the x or y axis, the x or y positions can be used as a measurement along the axis.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other free to access programs are available to recognize foci in fluorescence images, including Airlocalize (<a href="#bib21">Trcek et al., 2017</a>) and FISH-quant (<a href="#bib17">Mueller et al., 2013</a>). However, we find these are less able to cope with large datasets than Imaris.</div>
<h4 id="sec3.1.6">Analysis of ribosome number</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼30 min per image</div>
<p>In this step, the number of ribosomes present in the translation sites are calculated, using the high magnification images collected. This quantification requires analysis of the sum fluorescence intensities of both single cytoplasmic proteins and active translation sites. Here, we use the freely available analysis software Airlocalize that uses a spot-detection algorithm (<a href="#bib14">Lionnet et al., 2011</a>; <a href="#bib21">Trcek et al., 2017</a>) (<a href="#fig3">Figure 3</a>B). This algorithm subtracts the background fluorescence that surrounds individual mRNA spots and therefore provides robust measurements even in images containing high background.</p>
<ol start="35">
<li>Prepare the high magnification images for analysis.
<ol type="a">
<li>Open the saved Leica image file in Fiji.</li>
<li>Save each channel collected separately as a .tif image.</li>
<li>Name the Suntag smFISH channel as suntag.tif, and the mNeonGreen channel as ng.tif, for compatibility with later analysis scripts.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To reduce the computational time in Airlocalize, a subset of the image can be used.</div>
<ol start="36">
<li>Identify and quantify the fluorescence of translation sites using Airlocalize.
<ol type="a">
<li>Open the Airlocalize software. When prompted select the type of data to be analyzed, here a 3D single image.</li>
<li>Navigate to and select the image that will be analyzed.
<ol type="i">
<li>Select the mNeonGreen image channel only, named ng.tif.</li>
</ol>
</li>
<li>Determine the general analysis parameters in the next window, ‘Quantification Parameters’.
<ol type="i">
<li>Use the default settings in the ‘PSF Width (pix)’ section. This will enable the user to manually select fluorescent spots that will be used to calculate the point spread function (PSF) (see Step d).</li>
<li>Select an output folder.</li>
<li>Enable the option ‘output image of spots’. This option will output a tif image file with all the spots that were recorded during the analysis. To verify the analysis accuracy this spot file can be overlaid with the microscopy image.</li>
</ol>
</li>
<li>Determine the PSF. Airlocalize visualizes the fluorescent profile along the x/y/z axis of a selected fluorescent spot and a Gaussian fit is applied to it (<a href="#fig3">Figure 3</a>B, Step 1).
<ol type="i">
<li>Select a translation site by hovering the cursor over the center of the fluorescent signal and left click verify selection. The x/y/z profiles of this spot will be displayed. Select ‘local gaussian fit’ to display the fit over the fluorescent profile. A good fit resembles a Gaussian distribution curve. Click ‘record fit result’ to provide training data for the algorithm. Repeat with ∼ 10 additional translation sites.</li>
<li>Advance to the next window by selecting ‘done’.</li>
</ol>
</li>
<li>Identify the translation sites using a detection threshold. Modify the threshold until all translation sites are recognized by a red signal overlay on the microscopy image but none of the single proteins are. Once the desired threshold is set, select ‘done’ (<a href="#fig3">Figure 3</a>B, Step 2).</li>
<li>Multiple files will be generated in the output folder.
<ol type="i">
<li>The spots.tif file contains all spots that were identified.</li>
<li>The .loc3 file contains the x/y/z position, fluorescence intensity and frame number (here 0) for each translation site.</li>
<li>The .par3 file contains all analysis parameters (including PSF in xy and z).</li>
<li>The .det file contains the results and parameters.</li>
</ol>
</li>
<li>Move the analysis files into a subfolder called ‘translation_sites’.</li>
</ol>
</li>
<li>Identify and quantify the fluorescence of single proteins using Airlocalize (<a href="#fig3">Figure 3</a>B, Step 3).
<ol type="a">
<li>To identify single proteins, repeat steps 36a and b.</li>
<li>To identify single proteins there are two options:
<ol type="i">
<li>Either follow steps 36c and d as outlined above</li>
<li>Or disable the option ‘set manually’ in the ‘PSF Width (pix)’ window. Instead modify the PSF values to the previously determined values in the translation site analysis. The PSF values are included in the .par3 text file. This option will skip step 36d and advance directly to step 36e.</li>
</ol>
</li>
<li>Follow steps 36e and f. Make sure that the threshold is set so that single proteins are recognized.</li>
<li>Move analysis files into a subfolder called ‘single_proteins’.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> By identifying single proteins we also by default record the translation sites again as their fluorescent intensity is greater than single proteins. However, the translation sites will later be removed from this dataset in step 39b.</div>
<ol start="38">
<li>Identify and quantify the fluorescence of mRNA molecules for colocalization analysis using Airlocalize (<a href="#fig3">Figure 3</a>B, Step 4).
<ol type="a">
<li>Follow steps 36a-f as outlined above using the SunTag mRNA microscopy channel. The input file should be called suntag.tif.</li>
<li>Move the output files into a subfolder called ‘mRNA’.</li>
</ol>
</li>
<li>Perform colocalization analysis and determine the number of ribosomes present in the translation sites.
<ol type="a">
<li>Open Anaconda prompt. Type: python. Drag and drop the folder containing the ribosome_number.py script (<a href="https://github.com/DaisyVinter/StarProtocols_Vinter_2021">https://github.com/DaisyVinter/StarProtocols_Vinter_2021</a>) and then drag and drop the folder containing the subfolders with the analysis (<a href="#fig3">Figure 3</a>B, Step 5).</li>
<li>The script will identify translation sites by colocalization with mRNA spots, discarding any ‘translation sites’ that do not colocalize. The script will then take the single proteins and discard any included translation sites. Following this, the median fluorescence of the remaining single proteins is calculated, and the fluorescence of translation sites is divided by this number.</li>
<li>The output will be a file called num_ribo.csv containing the number of ribosomes per translation site.</li>
<li>Apply a correction factor to account for ribosomes that have only translated part of the Suntag mRNA sequence and so do not have the full binding of fluorophores.
<ol type="i">
<li>The correction factor is calculated by (length<sub>SunTagCDS</sub> × 0.5 + length<sub>hbCDS</sub>)/length<sub>total</sub>. This assumes that ribosomes are uniformly distributed, so ribosomes that have only translated part of the Suntag sequence will on average have translated half of the sequence (<a href="#bib20">Pichon et al., 2016</a>).</li>
<li>Divide the number of ribosomes given in the output file in c by the correction factor calculated (<a href="#fig3">Figure 3</a>B, Step 6).</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The Airlocalize algorithm is MATLAB based but does not require a full MATLAB license and can be run solely with the MCRInstaller.</div>
<h3 id="sec3.2">Section 2: Live imaging of translation sites</h3>
<p>Here we describe how to image live embryos to visualize translation sites across the expression domain and gain spatiotemporal information about their distribution. Separate steps are also included in the protocol describing how to monitor the translation of single mRNAs in live embryos, in order to capture their translation dynamics (<a href="#fig2">Figure 2</a>).</p>
<h4 id="sec3.2.1">Prepare cover slips for live imaging</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>In this step, cover slips, coated thinly with heptane glue, are prepared for their use in live imaging experiments.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare cover slips the day before or on the day of the live imaging experiment.</div>
<ol start="40">
<li>Make a ball of unrolled sellotape and place it in a 50 mL bottle. Fill the bottle half full of heptane and soak the sellotape for at least 12 h so the glue will dissolve.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use yellow double-sided plastic tape, 25 mm x 20 m (RS Components, Cat# 555-033) or packaging tape (Tesa, Cat# 4124). Other brands of sticky tape will need to be tested to ensure they are not toxic to <i>Drosophila</i> embryos.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Heptane glue can be kept for many months at 20<sup>o</sup>C.</div>
<ol start="41">
<li>Take Nr. 1, 18 x 18 mm coverslips and pipette approximately 60 μL of heptane glue onto the coverslip. Leave the heptane glue to dry as a thin coating on the slide.</li>
</ol>
<h4 id="sec3.2.2">Set up the embryo collection cage</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 days</div>
<p>In this step, a collection cage is set up to produce embryos for live imaging (<a href="#fig4">Figure 4</a>, Step 1).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Overview of embryo mounting for live imaging of translation sites</div>
<p>The key steps required for the collection and mounting of embryos are shown.</p>
</figcaption>
</figure>
<ol start="42">
<li>Collect flies.
<ol type="a">
<li>
<i>For whole expression domain live imaging (Step 50):</i> Collect virgin females heterozygous for the <i>scFv-mNG</i> and <i>His-RFP</i> transgenes and males homozygous for the <i>SunTag-hb</i> transgene.</li>
<li>
<i>For single translation site live imaging (Step 51):</i> Collect virgin females heterozygous for the <i>scFv-mNG</i> and <i>tdMCP-2xmRuby3-CAAX</i> transgenes and males homozygous for the <i>SunTag-hb-128xMS2</i> transgene.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This cross will produce embryos with one copy, zygotically expressed, of the tagged gene of interest.</div>
<div class="note">
<span class="note-title">Note:</span> <i>nos-MCP-EGFP</i> (<a href="#bib7">Garcia et al., 2013</a>) and <i>nos-tdMCP-RFP</i> (<a href="#bib8">Halstead et al., 2015</a>) stocks are also available for detection of the mRNAs carrying MS2 sequences.</div>
<ol start="43">
<li>Transfer at least 50 females and 25 males to a collection cage with an apple juice agar plate and yeast.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Including more flies in the collection cage will give more embryos for live imaging.</div>
<h4 id="sec3.2.3">Collect and mount embryos</h4>
<div class="timing">
<span class="timing-title">Timing:</span> Embryo collection, ∼60 min; mounting, ∼10 min</div>
<p>In these steps, embryos are mounted for live imaging (<a href="#fig4">Figure 4</a>).</p>
<ol start="44">
<li>Change the collection cage plate to a new apple juice agar plate with yeast. Let the flies lay for the desired amount of time and remove the plate for embryo collection (<a href="#fig4">Figure 4</a>, Step 2).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For imaging <i>SunTag-hb</i> mRNAs, which are translated from nc11 – 14, we let flies lay for 1 h so that the embryos are sufficiently young to image the whole translation time.</div>
<ol start="45">
<li>Dechorionate the embryos by applying 50% domestic bleach to the plate and incubating for 2 min.</li>
<li>Transfer the embryos to a sieve by washing them off the plate with water or embryo washing solution. When in the sieve, wash thoroughly with water or embryo washing solution until no bleach remains.</li>
<li>Brush the embryos out of the sieve onto a fresh apple juice agar plate under a dissection microscope.</li>
<li>Arrange the embryos in one or more lines on the apple juice agar plate using a brush or tweezers (<a href="#fig4">Figure 4</a>, Step 3).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Arrange the embryos so that the embryo area to be imaged is facing upwards. For example, the <i>scFv-mNG</i>; <i>His-RFP</i>/<i>SunTag-hb</i> embryos were arranged with the lateral side facing upwards.</div>
<ol start="49">
<li>Mount the embryos on a Lumox dish (<a href="#fig4">Figure 4</a>, Step 4).
<ol type="a">
<li>Place two Nr. 0, 18 <b>×</b> 18 mm cover slips on the Lumox imaging dish, leaving a small gap of approximately 12 mm in between.</li>
<li>Fill the gap with 70 μL of halocarbon oil (7:1, halocarbon oil 700:halocarbon oil 27).</li>
<li>Use tweezers to pick up a heptane glue cover slip and carefully lower the glue side onto the embryos.</li>
<li>Transfer the coverslip with embryos onto a Lumox imaging dish, with embryos positioned in the imaging oil and the coverslip overlapping with the two coverslips placed on the imaging dish in step a.</li>
<li>Use nail polish to secure the embryo coverslip to the other coverslips.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A movie of this mounting technique can be found in (<a href="#bib10">Hoppe and Ashe, 2021b</a>).</div>
<h4 id="sec3.2.4">Live imaging of translation sites across the expression domain</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 2.5 h per video</div>
<p>In this step, <i>SunTag-hb</i> mRNA translation sites are imaged across the width of the expression domain in living embryos. For precise aging of embryos, the embryos are maternally expressing His-RFP, therefore we are visualizing translation sites without detecting the mRNAs (<a href="#fig1">Figure 1</a>A). However, this has the advantage that the mRNA is only modified by the addition of SunTag sequences and does not carry any MS2 loops, meaning there is less potential for endogenous regulation to be disrupted.</p>
<ol start="50">
<li>Collect a time-lapse movie of a developing embryo for ∼2 h. We use a Zeiss LSM 880 upright microscope with Fast Airyscan for imaging (<a href="#sec6.1">Troubleshooting 1</a> and <a href="#sec6.5">3</a>).
<ol type="a">
<li>Set up the microscope with two channels for scFv-mNG and His-RFP, both collecting with the Fast Airyscan.
<ol type="i">
<li>For the example dataset, we used a 40<b>×</b>/1.30 EC Plan-Neofluar DIC M27 oil objective and 1.1<b>×</b> optical zoom, with a bit depth of 16. The image was acquired at 2996 x 788 pixels, resulting in a pixel size of 0.06 μm, and 25 slices with a z-step size of 850 nm.</li>
<li>The His-RFP channel was excited with the 561 nm laser (1.5%) and the scFv-mNG channel with the 488 nm laser (5%).</li>
</ol>
</li>
<li>Locate the embryos using brightfield illumination.</li>
<li>Using the His-RFP channel, find an embryo of ∼15 min prior to the desired age of imaging. For imaging translation beginning at nc11, as for the <i>SunTag-hb</i> embryos, find an embryo where nuclei are just beginning to migrate to the surface.</li>
<li>Set up a z-stack including the whole nuclear depth and cytoplasm on either side of the nucleus.</li>
<li>Take a time series with the minimum possible gap between z-stacks until translation has ceased. An example of a <i>SunTag-hb</i> time-lapse experiment is shown in <a href="#mmc1">Methods video S1</a> (previously described in <a href="#bib24">Vinter et al. 2021</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Speed, resolution, and z-depth of imaging must be balanced to achieve the optimal settings for the experiment. A high frame rate is important to capture the dynamics of translation, but may impact on the image quality. If raw image quality is low, image analysis and identification of translation sites above background may be more difficult. Time resolution can be improved by imaging a smaller region of the embryo, or by reducing the z-stack. The settings used allow us to see a narrow region of the embryo (∼9 nuclear widths at nuclear cycle 14) extending along ∼30% of the anterior posterior axis, with a time resolution of ∼40s and imaging depth of 20.4 μm. The resolution allows for the recognition of individual translation sites in Imaris. If using a different microscope or imaging a different sized section of the embryo, these settings will need to be optimized.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Maximum intensity projection of an embryo showing <i>SunTag-hb</i> translation sites (gray) and Histone-RFP (magenta) through nc11-14, related to step 50</div>
<p>Embryos zygotically expressing <i>SunTag-hb</i> are from females carrying single copies of the<i>scFv-mNG</i>and<i>His-RFP</i>transgenes. Imaged with 46 sec time resolution between frames, time stamp is in min:sec.</p>
</figcaption>
</figure>
<h4 id="sec3.2.5">Live imaging of mRNAs and single translation sites</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼30 min per video</div>
<p>In this step, <i>SunTag-hb</i> mRNAs and translation sites are imaged in a small region of the expression domain in live embryos. As mRNAs move very rapidly, we are using tdMCP-2xmRuby3-CAAX to tether them to the membrane, thereby facilitating their tracking and analysis over time (<a href="#fig1">Figure 1</a>C).</p>
<ol start="51">
<li>Collect a high magnification time-lapse dataset of individual translation sites. We use a Zeiss LSM 880 upright microscope with Fast Airyscan for imaging (<a href="#sec6.5">Troubleshooting 3</a>).
<ol type="a">
<li>Set up the microscope with two channels for scFv-mNG and tdMCP-2xmRuby3-CAAX, both collecting with the Fast Airyscan.
<ol type="i">
<li>For the example dataset, we used a 63<b>×</b>/1.40 PL Apo oil objective and 8<b>×</b> optical zoom, with a bit depth of 16. The image was acquired at 184 <b>×</b> 184 pixels, resulting in a pixel size of 0.09 <b>×</b> 0.09 μm.</li>
<li>The MCP-mRuby3 channel was excited with the 561 nm laser (15%) and the scFv-mNG channel with the 488 nm laser (15%).</li>
</ol>
</li>
<li>Locate the embryos using brightfield illumination.</li>
<li>Locate the area where translation is occurring or expected to occur using the objective without zoom (<a href="#sec6.7">Troubleshooting 4</a>).</li>
<li>Zoom into the area so that only a few cells are visible.</li>
<li>Adjust the focus so that the membrane (labeled with tdMCP-2xmRuby3-CAAX) is in focus, with individual translation sites in mNG if the mRNAs are already being translated (<a href="#sec6.9">Troubleshooting 5</a>).</li>
<li>To achieve the best possible time resolution, a time lapse dataset was acquired with a single z plane. An example time-lapse experiment imaging individual translation sites is shown in <a href="#mmc2">Methods video S2</a>.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prior to nc14, membranes are only present apically in the embryo, then membrane invagination starts in nc14. Therefore, when imaging prior to nc14, it is necessary to focus on the top of the cells where the mRNAs and unbound tdMCP-mRuby3-CAAX proteins show the cell outline (<a href="#fig7">Figure 7</a>C). Once the membranes are located zoom in to image the translation sites.</div>
<div class="note">
<span class="note-title">Note:</span> To track single translation sites, speed, resolution, and z depth must be balanced. In the example data set, a single z plane and small region was used so that the image resolution was of the quality necessary to identify translation sites (pixel size 0.09 <b>×</b> 0.09μm), and the time resolution was ∼2s, which is fast enough to capture changes associated with translation initiation and elongation (<a href="#bib16">Morisaki and Stasevich, 2018</a>). When using a different microscope, settings will have to be adjusted to optimize for each individual experiment.</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. Single plane time-lapse experiment visualizing membrane tethered tdMCP-mRuby3-CAAX and bound <i>SunTag-hb-MS2</i> mRNAs (magenta) and translation sites bound by scFv-mNG (green), related to step 51</div>
<p>Embryos are from females carrying single copies of the<i>scFv-mNG</i>and<i>tdMCP-Ruby3-CAAX</i>transgenes crossed to<i>SunTag-hb-MS2</i>transgenic males. Merged and single channel views are shown. Time resolution is 1.98 sec per frame, the scale bar is 5 μm.</p>
</figcaption>
</figure>
<h4 id="sec3.2.6">Analysis of expression domain live imaging</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>These steps describe how to analyze the expression domain level live imaging data to reveal the number, intensity, and distribution of translation sites over time (<a href="#fig5">Figure 5</a>A).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Overview of the analysis of translation sites from the live imaging data</div>
<p>The analysis steps are shown for live imaging of (A) the expression domain and (B) anchored individual translation sites.</p>
</figcaption>
</figure>
<ol start="52">
<li>Using the Imaris ‘surface’ function, create surfaces to identify nuclei (<a href="#fig5">Figure 5</a>A, Step 1).
<ol type="a">
<li>In the first screen, enable ‘Track Surfaces (over Time)’.</li>
<li>Select the Channel used for His-RFP.</li>
<li>Select ‘Enable Smooth’ and set ‘Surfaces Detail’ to 1 μm, which will smooth the nuclear surface and aids object identification.</li>
<li>In the ‘Slice’ tab, measure the diameter of nuclei.</li>
<li>Select ‘Background Subtraction’ under Thresholding. Set ‘Diameter of largest Sphere which fits into the Object’ to the value measured in step d.</li>
<li>Adjust the threshold so that the nuclei are selected. Move between frames to check that nuclei are selected in all frames.</li>
<li>Select ‘Enable’ under ‘Split touching Objects (Region Growing)’. Set ‘Seed Points Diameter’ to the value measured in step d.</li>
<li>Filter the seed points by ‘Quality’ so that only nuclei are selected.</li>
<li>Use the ‘Number of Voxels Img=1’ filter to eliminate any small non-nuclear objects.</li>
<li>In the ‘Tracking’ screen, select the Algorithm as ‘Autoregressive Motion’, Max Distance as 5 μm, and Max Gap Size as 3. These settings work well to track nuclei (<a href="#fig5">Figure 5</a>A, Step 2) from nc11 to the onset of gastrulation in nc14, where nuclei do not move more than 5 μm between consecutive frames. If analyzing datasets with longer time resolution, determine the maximum nuclear movement between two consecutive frames using the ‘Slice’ tab.</li>
<li>Complete the surfaces wizard.</li>
</ol>
</li>
<li>Using the Imaris ‘spots’ function, create spots to identify the translation sites (<a href="#fig5">Figure 5</a>A, Step 3) (<a href="#sec6.3">Troubleshooting 2</a>).
<ol type="a">
<li>Deselect ‘Track Spots (over Time)’ in the first screen.</li>
<li>Select the Channel used for ‘scFv-mNG’.</li>
<li>In the ‘Slice’ tab, measure the size of the translation sites.</li>
<li>Enter the value into ‘Estimated XY diameter’ and select ‘Model PSF-elongation along Z-axis’ and ‘Background Subtraction’.</li>
<li>Use filters to select the scFv-mNG foci (translation sites). Useful filters include ‘Quality’, ‘Intensity Max’ and ‘Intensity Sum’. Move between frames to check foci are selected in all frames.</li>
<li>Complete the spots wizard.</li>
</ol>
</li>
<li>Export the Imaris data (<a href="#fig5">Figure 5</a>A, Step 4).
<ol type="a">
<li>Export the statistics in the ‘Statistics’ tab. Select the desired statistics in ‘Preferences’, which can include the scFv-mNG intensity of each spot and position of the spots in each time point.</li>
</ol>
</li>
</ol>
<h4 id="sec3.2.7">Analysis of single translation site live imaging</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>In this step, individual translation sites are tracked, and their fluorescence intensities quantified over time.</p>
<ol start="55">
<li>Use the Imaris ‘Spots’ function for tracking of translation sites over time (<a href="#fig5">Figure 5</a>B) (<a href="#sec6.11">Troubleshooting 6</a>).
<ol type="a">
<li>In the creation wizard select ‘Track spots over time’.</li>
<li>In the next window, select the scFv-mNG channel for analysis and estimate the xy diameter of the translation site. The full fluorescence of the translation site should be included within the spot object (<a href="#fig5">Figure 5</a>B, Step 1).</li>
<li>Using filters, make sure that the translation sites are selected through time. The most useful filters here are ‘Quality’, ‘Intensity Sum’ and ‘Intensity Max’.</li>
<li>For the tracking parameters select ‘Autoregressive Motion’, a ‘Maximum Distance’ of 5 μm and a ‘Max Gap Size’ of 2 (<a href="#fig5">Figure 5</a>B, Step 2).</li>
</ol>
</li>
<li>After successful tracking, export the analysis files.
<ol type="a">
<li>In the ‘Statistics’ tab, select ‘Export all Statistics to file’. The statistics that are exported can be defined by the user under settings and are dependent on the type of analysis required. These can include (but are not limited to) fluorescence intensity quantification, distance and speed of movement (<a href="#fig5">Figure 5</a>B, Step 3).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For fluorescence intensity quantitation it is important to perform a background correction. This can be achieved by manually placing spot objects (of identical volume to the analysis spots) into different background regions and measuring the fluorescence value. An average background value can then be subtracted from the translation site fluorescence. Depending on the movie length this can be time consuming. For a semi-automated way to remove background fluorescence see (<a href="#bib10">Hoppe and Ashe, 2021b</a>).</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The smFISH/immunofluorescence of fixed embryos will allow the total SunTag-POI mRNAs to be visualized; colocalization of the scFv-mNG signal will indicate that they are being translated (<a href="#fig6">Figure 6</a>A shows the data for <i>SunTag-hb</i> mRNAs). For each embryo, the total number of mRNAs per nuclear territory, the number of translated mRNAs per nuclear territory, and data relating this to position along the anterior-posterior axis can be outputted (<a href="#fig6">Figure 6</a>C). As part of the analysis, the number of total and translated mRNAs can also be visualized spatially for each cell in the expression domain using a heatmap color scale (<a href="#fig6">Figure 6</a>C). At nc13, the total number of <i>SunTag-hb</i> mRNAs declines towards the center of the embryo, but the proportion translated is constant (<a href="#bib24">Vinter et al., 2021</a>). However, we found altered translation dynamics at nc14, as translation of <i>SunTag-hb</i> mRNAs is restricted to only a stripe of cells at the posterior edge of the expression domain (<a href="#fig6">Figures 6</a>A and 6C) (<a href="#bib24">Vinter et al., 2021</a>). The data can also be analyzed to determine the ribosome number in each translation site (<a href="#fig6">Figures 6</a>D and 6E). Ribosome number can also be determined from live imaging data based on the relative fluorescent signals of translation sites versus single proteins (<a href="#bib20">Pichon et al., 2016</a>; <a href="#bib26">Yan et al., 2016</a>), if the latter can be reliably detected.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Expected outcomes from the fixed imaging of translation sites</div>
<p>(A) Image of the whole expression domain of a nuclear cycle 14 embryo expressing <i>scFv-mNG</i> and <i>SunTag-hb</i>, stained with smFISH probes for SunTag (magenta), anti-mNG antibody (green) and DAPI (blue). The lower panel shows a zoomed inset of the image, the position is shown by a box. The bright signals in the scFv-mNG channel, colocalizing with mRNAs, are translation sites (white arrowheads), the dimmer signals are single proteins and background signals. The weaker magenta signals (pink arrowheads) in the merged image represent untranslated mRNAs, the bright magenta signals are transcription sites. Upper panel, scale bar = 50 μm. Lower panel, scale bar = 5 μm.</p>
<p>(B) Output of the spots assignment script showing the calling of the AP axis (blue line). The nuclei highlighted at the anterior are used to find the axis.</p>
<p>(C) (i) The embryo is false colored by the number of mRNAs per nuclear territory as outputted by the spot assignment script. (ii) As in (i), but for the number of translated mRNAs. (iii) Graphs show the number of mRNAs or translated mRNAs per nuclear territory along the AP axis. Nuclear territories are binned along the AP axis (bin size = 20 μm) and the mean value per bin is determined. n = 3 embryos, mean ± SD is shown.</p>
<p>(D) Image of a nc13 embryo, expressing <i>scFv-mNG</i> and <i>SunTag-hb</i>, taken for quantification of the number of ribosomes in translation sites. The lower panel is a zoomed inset of the image, marked by a white box, showing bright translation sites (green) colocalized with mRNAs (magenta) and dimmer single proteins (green). White, green and pink arrowheads mark translation sites, single proteins and untranslated mRNAs, respectively. The bright signals in the <i>SunTag-hb</i> channel in the top image are transcription sites. Upper panel, scale bar = 5 μm. Lower panel, scale bar = 1 μm.</p>
<p>(E) Ribosome number quantification for the region of interest from (D). n = 146 translation sites from 1 nc13 embryo, mean ± SD shown.</p>
</figcaption>
</figure>
<p>The data on the single cell efficiency of translation determined by the fixed embryo analysis is complemented by the live imaging data that provide temporal information about translation (<a href="#fig7">Figures 7</a>A and 7B, <a href="#mmc1">Methods video S1</a>). For example, live imaging of the <i>SunTag-hb</i> translation sites across the expression domain reveals that the domain of translated <i>SunTag-hb</i> mRNAs gradually contracts from the anterior of the embryo starting at late nc13, so that only a stripe of translation remains in early nc14 embryos (<a href="#bib24">Vinter et al., 2021</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/995-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Expected outcomes from the live imaging of translation sites</div>
<p>(A) Still from Movie 1 of the expression domain at mid nc13 in an embryo expressing <i>Histone-RFP</i>, <i>scFv-mNG</i> and <i>SunTag-hb</i>, showing nuclei (magenta) and translation sites (green). The embryo is oriented with anterior to the left. Scale bar = 20 μm.</p>
<p>(B) Example graph of cumulative translation over time (in fluorescence arbitrary units) in the embryo shown in <a href="#mmc1">Movie 1</a>. The vertical bars mark the start of nc12, 13 and 14.</p>
<p>(C) Still from a movie showing that the tethered tdMCP-2xRuby3-CAAX proteins label membranes. Scale bar = 20 μm.</p>
<p>(D) Still from <a href="#mmc2">Movie 2</a> of individual translation sites in an embryo expressing <i>scFv-mNG</i>, <i>tdMCP-mRuby3-CAAX</i> and <i>SunTag-hb-MS2</i>, showing mRNAs and membranes (magenta) and translation sites (green). The bright magenta foci are likely individual mRNAs. Arrowheads point to the translation sites used for fluorescence quantification in (E). Scale bar = 5 μm.</p>
<p>(E) An example graph of fluorescent traces of individual translation sites from the movie in (D).</p>
</figcaption>
</figure>
<p>Live imaging of single translation sites allows the translation of individual mRNAs to be followed over time (<a href="#fig7">Figures 7</a>D and 7E, <a href="#mmc2">Methods video S2</a>). Imaging at this resolution allowed us to capture different events in the translation cycle, including release of a newly synthesized protein and translation termination (<a href="#bib24">Vinter et al., 2021</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>When live imaging the whole expression domain using the method described in this protocol, it is not possible to resolve single mRNAs as well as translation sites, as MS2 loops are not included. Use of MS2 loops, an MCP-FP and His-FP in a third channel (for example, His2Av-eBFP2 (<a href="#bib6">Fukaya et al., 2017</a>)) may allow this, but we have not tested this. In addition, weaker translation sites with less ribosomes and therefore less fluorescence may be missed. While live imaging can give temporal information about translation, it is unable to show the proportion of mRNAs translating, as untranslated mRNAs cannot be visualized on a large scale. Fixed imaging with smFISH to visualize single mRNAs gives information about the number of translated and untranslated mRNAs, however it cannot give information about changes over time. Therefore, the fixed and live approaches are complementary and allow different information about translation to be collected.</p>
<p>Live imaging and tracking of single translation sites are limited by the time resolution that can be achieved by the microscope used, and the length of time single translation sites can be followed due to movement out of the frame of imaging or bleaching. These limitations will be highly dependent on the microscope system and settings used. In addition, we use a CAAX tag in the MCP protein to anchor translating mRNAs to the plasma membrane. Although this method allows for longer tracking times, it is possible that mislocalizing the translating mRNA in this way may affect the regulation of translation. Many mRNAs in <i>Drosophila</i> embryos are subcellularly localized (<a href="#bib13">Lecuyer et al., 2007</a>), and recent data has shown that translation efficiency of some mRNAs is dependent on their localization (<a href="#bib3">Dufourt et al., 2021</a>). We recommend that fixed imaging without the MCP protein is always carried out to check that translation is not dependent on the subcellular localization of the specific mRNA that is being investigated. Finally, insertion of the MS2 sequences into the 3′UTR may disrupt translational regulation. However, this can be assessed by comparing the translation efficiency of the test mRNAs with and without any MS2 loops in fixed embryos.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Translation sites cannot be visualized when imaging fixed (Step 23) or live (Step 50) embryos.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>We find that the microscope settings described work robustly for the <i>SunTag-hb</i> and <i>scFv-mNG</i> fly lines. If a different tagged mRNA of interest is investigated, it may be weakly translated, which would decrease the brightness of the translation site fluorescent signal. Try increasing the laser power used for the scFv channel.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Non-translation sites are detected as spots in the Imaris analysis of the fixed (Step 28) and live (Step 53) embryo data.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Imaris may incorrectly identify background scFv signal (either free scFv-FP or SunTag proteins that have been bound by scFv-FP and then diffused away) as spots, in both fixed and live imaging. In fixed imaging, these non-translation site spots will be removed by the process of colocalizing scFv spots with SunTag mRNAs, as only translation sites will be colocalized with mRNAs. In this case, use the Imaris thresholds that capture all visible translation sites, ignoring any non-translation site spots that are identified. It is possible to check that known non-translation site spots are removed by comparing the original scFv spots mask with the scFv spots co-located mask.</p>
<p>If Imaris recognizes background noise rather than bright fluorescent foci in live imaging, adjust the ‘Quality’ and ‘Intensity Sum’ filter thresholds so that these spots are eliminated. It may be that, for different time points in the movie, different thresholds are necessary to recognize all translation sites and eliminate background noise spots. In this case, divide the time series and set the thresholds separately, then combine using the Imaris ‘Merge’ function. If single translation sites are tracked over time, it is possible to perform the spot identification using different thresholds and merge them into one spot object file prior to tracking.</p>
<p>A smoothing wavelet filter Imaris X-tension (available at <a href="https://github.com/zindy/Imaris">https://github.com/zindy/Imaris</a>) can also be used on the scFv-mNG channel to remove background noise. Spots can then be detected using this channel, and statistics exported for the fluorescence of the original scFv-mNG channel so the original fluorescence is reported.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Developmental defects occur when live imaging (Steps 50 and 51).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Developmental defects may occur due to stress on the embryo. Check that the temperature in the room is not above 27°C, and do not attach the coverslip with too much pressure. In addition, developmental defects occur at a higher frequency if the flies used for embryo collection are older. If the cage used is over 10 days old, collect fresh virgin females and males and set up a new collection cage.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The developmental stage or position in the embryo cannot be identified when imaging single translation sites (Step 51c).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>As His-RFP is not used for crosses where the aim is to image single translation sites, it can be difficult to identify the position in the embryo or embryonic stage, especially at higher zoom. Try viewing the embryo in brightfield illumination, which will show if embryos have cellularized or gastrulated. Membranes should be visible in the tdMCP-2xmRuby3-CAAX channel at the top of the cell, as well as some nuclear signal in the scFv-mNG channel (particularly before translation has begun), as unbound scFv-mNG is localized to the nucleus. Use these channels at low zoom to identify when the embryo has reached an appropriate age and zoom in to the region of interest of the expression domain. In addition, His2Av-eBFP2 fly lines are available (<a href="#bib6">Fukaya et al., 2017</a>), which could be introduced to allow nuclei to be imaged.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Cannot find mRNAs or translation sites when live imaging a high magnification region of the embryo (Step 51e).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Ensure the experiment is set up so that the membrane is in focus, as this is where translating mRNAs will be tethered. We find it easier to focus on tethered mRNAs to set up the imaging experiment. In our experience it is harder to keep single mRNAs in focus than their bulkier translation sites. In addition, the cell membranes become very thin shortly before and during mitosis, making it easier to follow individual mRNAs. Note that there is high tdMCP-mRuby3-CAAX signal at the membrane based on concentration of the tdMCP-mRuby3-CAAX proteins, even in the absence of bound mRNAs.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Tracking is unsuccessful for individual translation sites (Step 55).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>If the spots function tracking does not work well, the translation site can be tracked manually. In the first creation wizard window select ‘Skip automatic creation, edit manually’. In the ‘Edit’ tab, select ‘Auto-connect to selected Spot’ and ‘Enable delay before auto-advancing’ in the ’Manual Tracking’ section. In the first frame to be used for tracking, manually add a spot (pointer in the ‘select’ mode, press shift and left click). After placing the first spot, the software will advance automatically to the next time frame. Place the next spot until you reach the last frame that should be used for tracking.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bischof, J., Maeda, R.K., Hediger, M., Karch, F., and Basler, K. (2007). An optimized transgenesis system for Drosophila using germ-line-specific phiC31 integrases. Proc. Natl. Acad. Sci. U S A <i>104</i>, 3312-3317. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref1">View at publisher</a></p>
<p id="bib2">Calvo, L., Ronshaugen, M., and Pettini, T. (2021). smiFISH and embryo segmentation for single-cell multi-gene RNA quantification in arthropods. Commun. Biol. <i>4</i>, 352. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref2">View at publisher</a></p>
<p id="bib3">Dufourt, J., Bellec, M., Trullo, A., Dejean, M., De Rossi, S., Favard, C., and Lagha, M. (2021). Imaging translation dynamics in live embryos reveals spatial heterogeneities. Science <i>372</i>, 840-844. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref3">View at publisher</a></p>
<p id="bib4">Fish, M.P., Groth, A.C., Calos, M.P., and Nusse, R. (2007). Creating transgenic Drosophila by microinjecting the site-specific phiC31 integrase mRNA and a transgene-containing donor plasmid. Nat. Protoc. <i>2</i>, 2325-2331. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref4">View at publisher</a></p>
<p id="bib5">Formicola, N., Heim, M., Dufourt, J., Lancelot, A.S., Nakamura, A., Lagha, M., and Besse, F. (2021). Tyramine induces dynamic RNP granule remodeling and translation activation in the Drosophila brain. eLife <i>10</i>, e65742. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref5">View at publisher</a></p>
<p id="bib6">Fukaya, T., Lim, B., and Levine, M. (2017). Rapid rates of Pol II elongation in the Drosophila embryo. Curr. Biol. <i>27</i>, 1387-1391. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref6">View at publisher</a></p>
<p id="bib7">Garcia, H.G., Tikhonov, M., Lin, A., and Gregor, T. (2013). Quantitative imaging of transcription in living Drosophila embryos links polymerase activity to patterning. Curr. Biol. <i>23</i>, 2140-2145. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref7">View at publisher</a></p>
<p id="bib8">Halstead, J.M., Lionnet, T., Wilbertz, J.H., Wippich, F., Ephrussi, A., Singer, R.H., and Chao, J.A. (2015). Translation. An RNA biosensor for imaging the first round of translation from single cells to living animals. Science <i>347</i>, 1367-1671. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref8">View at publisher</a></p>
<p id="bib9">Hoppe, C. and Ashe, H.L. (2021a). CRISPR-Cas9 strategies to insert MS2 stem-loops into endogenous loci in Drosophila embryos. STAR Protoc. <i>2</i>, 100380. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref9">View at publisher</a></p>
<p id="bib10">Hoppe, C. and Ashe, H.L. (2021b). Live imaging and quantitation of nascent transcription using the MS2/MCP system in the Drosophila embryo. STAR Protoc. <i>2</i>, 100379. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref10">View at publisher</a></p>
<p id="bib11">Hoppe, C., Bowles, J.R., Minchington, T.G., Sutcliffe, C., Upadhyai, P., Rattray, M., and Ashe, H.L. (2020). Modulation of the promoter activation rate dictates the transcriptional response to graded BMP signaling levels in the Drosophila embryo. Dev. Cell <i>54</i>, 727-741 e727. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref11">View at publisher</a></p>
<p id="bib12">Lamb, A.M., Walker, E.A., and Wittkopp, P.J. (2017). Tools and strategies for scarless allele replacement in Drosophila using CRISPR/Cas9. Fly (Austin) <i>11</i>, 53-64. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref12">View at publisher</a></p>
<p id="bib13">Lecuyer, E., Yoshida, H., Parthasarathy, N., Alm, C., Babak, T., Cerovina, T., Hughes, T.R., Tomancak, P., and Krause, H.M. (2007). Global analysis of mRNA localization reveals a prominent role in organizing cellular architecture and function. Cell <i>131</i>, 174-187. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref13">View at publisher</a></p>
<p id="bib14">Lionnet, T., Czaplinski, K., Darzacq, X., Shav-Tal, Y., Wells, A.L., Chao, J.A., Park, H.Y., de Turris, V., Lopez-Jones, M., and Singer, R.H. (2011). A transgenic mouse for in vivo detection of endogenous labeled mRNA. Nat. Methods <i>8</i>, 165-170. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref14">View at publisher</a></p>
<p id="bib29">Lu, W., Winding, M., Lakonishok, M., Wildonger, J., and Gelfand, V.I. (2016). Microtubule–microtubule sliding by kinesin-1 is essential for normal cytoplasmic streaming in Drosophila oocytes. Proc. Natl. Acad. Sci. USA <i>113</i>, E4995-E5004. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/opt87RfsfioaP">View at publisher</a></p>
<p id="bib15">Mayr, C. (2017). Regulation by 3'-untranslated regions. Annu. Rev. Genet. <i>51</i>, 171-194. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref15">View at publisher</a></p>
<p id="bib16">Morisaki, T. and Stasevich, T.J. (2018). Quantifying single mRNA translation kinetics in living cells. Cold Spring Harb. Perspect. Biol. <i>10</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref16">View at publisher</a></p>
<p id="bib17">Mueller, F., Senecal, A., Tantale, K., Marie-Nelly, H., Ly, N., Collin, O., Basyuk, E., Bertrand, E., Darzacq, X., and Zimmer, C. (2013). FISH-quant: automatic counting of transcripts in 3D FISH images. Nat. Methods <i>10</i>, 277-278. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref17">View at publisher</a></p>
<p id="bib18">Nyberg, K.G., Nguyen, J.Q., Kwon, Y.J., Blythe, S., Beitel, G.J., and Carthew, R. (2020). A pipeline for precise and efficient genome editing by sgRNA-Cas9 RNPs in Drosophila. Fly (Austin) <i>14</i>, 34-48. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref18">View at publisher</a></p>
<p id="bib19">Palam, L.R., Baird, T.D., and Wek, R.C. (2011). Phosphorylation of eIF2 facilitates ribosomal bypass of an inhibitory upstream ORF to enhance CHOP translation. J. Biol. Chem. <i>286</i>, 10939-10949. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref19">View at publisher</a></p>
<p id="bib20">Pichon, X., Bastide, A., Safieddine, A., Chouaib, R., Samacoits, A., Basyuk, E., Peter, M., Mueller, F., and Bertrand, E. (2016). Visualization of single endogenous polysomes reveals the dynamics of translation in live human cells. J. Cell Biol. <i>214</i>, 769-781. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref20">View at publisher</a></p>
<p id="bib28">Schindelin, J., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nature Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/optHMp1YZNrtQ">View at publisher</a></p>
<p id="bib21">Trcek, T., Lionnet, T., Shroff, H., and Lehmann, R. (2017). mRNA quantification using single-molecule FISH in Drosophila embryos. Nat. Protoc. <i>12</i>, 1326-1348. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref21">View at publisher</a></p>
<p id="bib22">Tsanov, N., Samacoits, A., Chouaib, R., Traboulsi, A.M., Gostan, T., Weber, C., Zimmer, C., Zibara, K., Walter, T., Peter, M., et al. (2016). smiFISH and FISH-quant - a flexible single RNA detection approach with super-resolution capability. Nucleic Acids Res. <i>44</i>, e165. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref22">View at publisher</a></p>
<p id="bib23">Vattem, K.M. and Wek, R.C. (2004). Reinitiation involving upstream ORFs regulates ATF4 mRNA translation in mammalian cells. Proc. Natl. Acad. Sci. U S A <i>101</i>, 11269-11274. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref23">View at publisher</a></p>
<p id="bib24">Vinter, D.J., Hoppe, C., Minchington, T.G., Sutcliffe, C., and Ashe, H.L. (2021). Dynamics of hunchback translation in real time and at single mRNA resolution in the Drosophila embryo. Development <i>148</i>, dev196121. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref24">View at publisher</a></p>
<p id="bib25">Wu, B., Eliscovich, C., Yoon, Y.J., and Singer, R.H. (2016). Translation dynamics of single mRNAs in live cells and neurons. Science <i>352</i>, 1430-1435. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref25">View at publisher</a></p>
<p id="bib26">Yan, X., Hoek, T.A., Vale, R.D., and Tanenbaum, M.E. (2016). Dynamics of translation of single mRNA molecules in vivo. Cell <i>165</i>, 976-989. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref26">View at publisher</a></p>
<p id="bib27">Yu, J.J.S., Vincent, J.P., and McGough, I.J. (2021). One-step CRISPR-Cas9 protocol for the generation of plug &amp; play conditional knockouts in Drosophila melanogaster. STAR Protoc. <i>2</i>, 100560. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00518-9/sref27">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Hilary Ashe (<a href="mailto:hilary.ashe@manchester.ac.uk">hilary.ashe@manchester.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>The fly lines used in this protocol are available upon request from the lead contact.</p>
<h4>Data and code availability</h4>
<p>Example data sets have been deposited to Mendeley Data: <a href="https://doi.org/10.17632/3h73rbvbt9.1">https://doi.org/10.17632/3h73rbvbt9.1</a> and <a href="https://doi.org/10.17632/5vs879zvz9.1">https://doi.org/10.17632/5vs879zvz9.1</a>.</p>
<p>The analysis scripts are available at GitHub: <a href="https://github.com/TMinchington/sass">https://github.com/TMinchington/sass</a> and <a href="https://github.com/DaisyVinter/StarProtocols_Vinter_2021">https://github.com/DaisyVinter/StarProtocols_Vinter_2021</a>.</p>
<p>Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</p>
<h3>Acknowledgments</h3>
<p>We thank Tom Minchington for modifying the analysis script, Egor Zindy for his Imaris X-tension, and Catherine Sutcliffe for helpful discussions. Some of the figures were created using <a href="http://BioRender.com">BioRender.com</a>. We also thank the Manchester Bioimaging Facility for their help with microscopy. The Bioimaging Facility microscopes used in this study were purchased with grants from the <a href="https://doi.org/10.13039/501100000268">BBSRC</a>, <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> and the <a href="https://doi.org/10.13039/100007545">University of Manchester</a> Strategic Fund. This work was supported by a <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> Investigator award to H.L.A. (204832/Z/16/Z) and by a <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> PhD studentship to D.J.V. (215206/Z/19/Z).</p>
<h3>Author contributions</h3>
<p>Methodology and writing, D.J.V., C.H., and H.L.A.; investigation, D.J.V. and C.H.; funding acquisition and supervision, H.L.A.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>