<article>
<section>
<h1 id="header">CRISPitope: A generic platform to model target antigens for adoptive T cell transfer therapy in mouse tumor models</h1>
<p><time datetime="2021-12-24">Published: December 24, 2021</time></p>
<p>Maike Effern,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Nicole Glodde,<sup><a href="#aff1">1</a></sup> Emma Bawden,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Jana Liebing,<sup><a href="#aff1">1</a></sup> Daniel Hinze,<sup><a href="#aff1">1</a></sup> Thomas Tüting,<sup><a href="#aff3">3</a></sup> Thomas Gebhardt,<sup><a href="#aff2">2</a></sup> and Michael Hölzel<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Experimental Oncology (IEO), Medical Faculty, University Hospital Bonn, University of Bonn, 53105 Bonn, Germany</p>
<p id="aff2"><sup>2</sup>Department of Microbiology &amp; Immunology, The University of Melbourne at the Peter Doherty Institute for Infection and Immunity, Melbourne, VIC, Australia</p>
<p id="aff3"><sup>3</sup>Laboratory of Experimental Dermatology, Department of Dermatology, University of Magdeburg, Magdeburg, Germany</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:maike.effern@ukbonn.de">maike.effern@ukbonn.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:michael.hoelzel@ukbonn.de">michael.hoelzel@ukbonn.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101038">10.1016/j.xpro.2021.101038</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol details the procedure for CRISPR-assisted insertion of epitopes (CRISPitope), a flexible approach for generating tumor cells expressing model CD8<sup>+</sup> T cell epitopes fused to endogenously encoded gene products of choice. CRISPitope-engineered tumor cells can be recognized by T cell receptor-transgenic (TCRtg) CD8<sup>+</sup> T cells that are widely used in immunology research. Using mice inoculated with CRISPitope-engineered tumor cells, researchers can investigate how the choice of the target antigen for T cell immunotherapies influences treatment efficacy and resistance mechanisms.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00744-9/sref2">Effern et al. (2020)</a>.</p>




<div class="highlights">


</div>
<div class="graphical-abstract">


</div>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Genome editing of tumor cells for experimental standardized T cell recognition</li>
<li>Tagging endogenous gene products with model CD8+ T cell epitopes in tumor cells</li>
<li>Assessing T cell responses directed against differentially regulated gene products</li>
<li>Approach allows studying the impact of the target antigen choice on T cell therapy</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Adoptive T cell transfer therapy (ACT) is a rapidly advancing treatment modality that can improve cancer patient outcome. Selecting optimal target antigen(s) for ACT remains a central challenge in the field. Specificity of the T cell receptor to tumor antigen and the peptide-MHC binding affinity are key considerations for selection of target antigens. More recently, the nature of the gene product encoding the antigen, such as its expression level, expression pattern and epigenetic regulation, has received increasing attention. However, it is difficult to investigate T cell responses to differentially regulated gene products and resulting epitopes, as different peptide-MHC binding affinities compromise comparability. As a solution, we developed a CRISPR-Cas9-based approach termed CRISPR-assisted insertion of epitopes (CRISPitope) to fuse model CD8<sup>+</sup> T cell epitopes to the C-terminus of selected endogenous gene products. We inoculated mice with CRISPitope-engineered melanoma cells that are targetable by the same TCRtg CD8<sup>+</sup> T cells to investigate how target antigens that are derived from different endogenously-encoded gene products affect responsiveness and resistance to ACT.</p>
<p>CRISPitope is a modified version of CRISPR-assisted insertion tagging (CRISPaint) (<a href="#bib3">Schmid-Burgk et al., 2016</a>) which is a CRISPR-Cas9-based approach that facilitates the insertion of large heterologous DNA fragments into the mammalian genome using a ligase-4-dependent mechanism. The CRISPaint approach is a three-plasmid-based system. The first plasmid (target selector) encodes a gene-specific single guide RNA (sgRNA) and Cas9 endonuclease. The Cas9-sgRNA complex introduces a double-strand break (DSB) upstream of the gene of interest’s stop codon. The second plasmid (frame selector) also encodes a sgRNA and Cas9 endonuclease and mediates the cleavage of a third plasmid (universal donor) while taking the reading frames into account. Depending on the target selector one of three frame selectors (+0, +1 or +2) has to be selected. The cleaved, linearized universal donor plasmid integrates into the mammalian genome at the site of the DSB. Integration of the universal donor DNA occurs via the non-homologous end joining (NHEJ) pathway. This leads to C-terminally tagged fusion proteins under the control of the respective endogenous gene promoter. For our purpose, we designed universal donor plasmids that encode tags with the following basic structure: fluorescent protein–FLAG-tag–T cell epitope–T2A self-cleaving peptide–antibiotic selection marker (<a href="#fig1">Figure 1</a>A). This universal donor design allows for visualization by microscopy, detection by flow cytometry and fluorescent cell sorting by the fluorescent protein, detection by Western blot via the FLAG-tag, recognition by epitope-specific T cells by detection of the model CD8<sup>+</sup> T cell epitope, and selection by antibiotics via the resistance cassettes (<a href="#fig1">Figure 1</a>B). We generated several universal donor plasmids that offer flexibility in terms of fluorescent proteins, resistance cassettes, peptide affinities, MHC class I restriction, and TCRtg T cells.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. CRISPR-assisted insertion of epitopes (CRISPitope) to fuse model CD8<sup>+</sup> T cell epitopes to endogenous gene products of interest</div>
<p>(A) Graphical depiction of the CRISPitope method using a three-plasmid system (target selector, frame selector and universal donor).</p>
<p>(B) Modularity of the universal donor plasmids depicting different fluorescent proteins, CD8<sup>+</sup> T cell epitopes, their MHC class I restriction and corresponding T cell receptor (TCR)-transgenic (TCRtg) T cell and resistance cassettes. Images were taken from <a href="#bib2">Effern et al. (2020)</a>.</p>
</figcaption>
</figure>
<p>All experiments presented in this protocol were performed using the murine melanoma cell line HC.PmelKO which is derived from HCmel12 cells. HCmel12 mouse melanoma cells were derived from a serial transplant of a primary melanoma in <i>Hgf-Cdk4</i><sup><i>R24C</i></sup> mice (<a href="#bib1">Bald et al., 2014</a>). In order to generate HC.PmelKO cells, HCmel12 cells, homozygous for mutant <i>Cdk4</i><sup><i>R24C</i></sup><i>,</i> were modified by CRISPR-Cas9 to ablate the <i>Pmel</i> gene which encodes the protein PMEL (also known as SILV or gp100). PMEL, a protein expressed by melanoma cells and melanocytes, harbors the H2-D<sup>b</sup> restricted low affinity epitope EGSRNQDWL (mgp100, murine gp100) recognizable by the gp100-specific Pmel-1 TCRtg CD8<sup>+</sup> T cells. Gene deletion of <i>Pmel</i> enabled the high affinity human gp100 epitope KVPRNQDWL (hgp100) to be introduced for gene product tagging by CRISPitope, as model epitope for Pmel-1 TCRtg CD8<sup>+</sup> T cells.</p>
<p>Whilst this protocol describes CRISPitope-engineering using the model epitope hgp100 for a particular murine melanoma cell line, this technique can be used to modify other cell lines with any epitope sequence of choice. A prerequisite for successful CRISPitope-engineering is that the cells must be transfectable. We have also successfully modified the more commonly used murine melanoma cell line B16, the human melanoma cell line MZ-7 and cell lines from other cancer entities such as MC38 (murine colon carcinoma) or ID8 (murine ovarian carcinoma). Moreover, we have successfully tested the model epitopes from Ovalbumin (Ova; AA<sub>257–264</sub>: SIINFEKL) and Herpes simplex virus 1 glycoprotein B (HSV-1 gB; AA<sub>498–505</sub>: SSIEFARL) that can be recognized by OT-I or gBT-I TCRtg CD8<sup>+</sup> T cells, respectively. More detailed information on using different cell lines can be found throughout the protocol.</p>
<p>This STAR protocol describes the generation of CRISPitope-engineered tumor cells and an <i>in vitro</i> T cell recognition assay. This protocol does not describe injection of the CRISPitope-engineered tumor cells in mice followed by ACT immunotherapy <i>in vivo</i>, as a detailed description would be beyond the scope of this protocol.</p>
<h3 id="sec1.1">Obtain Pmel-1 TCRtg T cell mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–8 weeks</div>
<ol>
<li>Obtain Pmel-1 TCRtg T cell mice (B6.Cg-Thy1a/Cy Tg(TcraTcrb)8Rest/J) from Jackson Laboratory (Bar Harbor, ME, USA).
<ol type="a">
<li>All experiments involving animals presented in this protocol were approved by the local government authorities (LANUV, NRW, Germany) and performed in adherence to the national and institutional guidelines for the care and use of laboratory animals.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol describes T cell activation using the hgp100 CD8+ T cell model epitope and pmel-1 TCRtg T cells. However, there are different universal donor plasmids available that encode for different model CD8+ T cell epitopes. If you do not have the Pmel-1 TCRtg mouse line available, this protocol is also suitable for the OT-I TCRtg mouse line (<a href="https://www.jax.org/strain/003831">https://www.jax.org/strain/003831</a>) or gBT-I TCRtg mouse line (<a href="http://www.informatics.jax.org/allele/MGI:3722258">http://www.informatics.jax.org/allele/MGI:3722258</a>).</div>
<h3 id="sec1.2">Obtain and amplify target selector, frame selector and universal donor plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 weeks</div>
<ol start="2">
<li>Obtain empty target selector pX330 plasmid, frame selectors and universal donor plasmids from Addgene (Watertown, MA, USA; <a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Overview of the plasmids used for the CRISPitope technique including the plasmid name, Addgene plasmid ID and corresponding Addgene URL</caption>
<thead>
<tr>
<th>Plasmid name</th>
<th>Addgene plasmid ID</th>
<th>URL</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Target selector</td>
</tr>
<tr>
<td>pX330-U6-Chimeric_BB-CBh-hSpCas9 (short: pX330)</td>
<td>42230</td>
<td><a href="https://www.addgene.org/42230/">https://www.addgene.org/42230/</a></td>
</tr>
<tr>
<td colspan="3">Frame selectors</td>
</tr>
<tr>
<td>pCAS9-mCherry-Frame +0</td>
<td>66939</td>
<td><a href="https://www.addgene.org/66939/">https://www.addgene.org/66939/</a></td>
</tr>
<tr>
<td>pCAS9-mCherry-Frame +1</td>
<td>66940</td>
<td><a href="https://www.addgene.org/66940/">https://www.addgene.org/66940/</a></td>
</tr>
<tr>
<td>pCAS9-mCherry-Frame +2</td>
<td>66941</td>
<td><a href="https://www.addgene.org/66941/">https://www.addgene.org/66941/</a></td>
</tr>
<tr>
<td colspan="3">Universal donors</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-PuroR [M1G]</td>
<td>171800</td>
<td><a href="https://www.addgene.org/171800/">https://www.addgene.org/171800/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
<td>171801</td>
<td><a href="https://www.addgene.org/171801/">https://www.addgene.org/171801/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-mgp100-PuroR [M1G]</td>
<td>171802</td>
<td><a href="https://www.addgene.org/171802/">https://www.addgene.org/171802/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-gB-PuroR [M1G]</td>
<td>171803</td>
<td><a href="https://www.addgene.org/171803/">https://www.addgene.org/171803/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-Ova-PuroR [M1G]</td>
<td>171804</td>
<td><a href="https://www.addgene.org/171804/">https://www.addgene.org/171804/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-BlastR [M1G]</td>
<td>171805</td>
<td><a href="https://www.addgene.org/171805/">https://www.addgene.org/171805/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-hgp100-BlastR [M1G]</td>
<td>171806</td>
<td><a href="https://www.addgene.org/171806/">https://www.addgene.org/171806/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-gB-BlastR [M1G]</td>
<td>171807</td>
<td><a href="https://www.addgene.org/171807/">https://www.addgene.org/171807/</a></td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-Ova-BlastR [M1G]</td>
<td>171808</td>
<td><a href="https://www.addgene.org/171808/">https://www.addgene.org/171808/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-PuroR [M1G]</td>
<td>171809</td>
<td><a href="https://www.addgene.org/171809/">https://www.addgene.org/171809/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-hgp100-PuroR [M1G]</td>
<td>171810</td>
<td><a href="https://www.addgene.org/171810/">https://www.addgene.org/171810/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-mgp100-PuroR [M1G]</td>
<td>171811</td>
<td><a href="https://www.addgene.org/171811/">https://www.addgene.org/171811/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-gB-PuroR [M1G]</td>
<td>171812</td>
<td><a href="https://www.addgene.org/171812/">https://www.addgene.org/171812/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-Ova-PuroR [M1G]</td>
<td>171813</td>
<td><a href="https://www.addgene.org/171813/">https://www.addgene.org/171813/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-BlastR [M1G]</td>
<td>171814</td>
<td><a href="https://www.addgene.org/171814/">https://www.addgene.org/171814/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-hgp100-BlastR [M1G]</td>
<td>171815</td>
<td><a href="https://www.addgene.org/171815/">https://www.addgene.org/171815/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-gB-BlastR [M1G]</td>
<td>171816</td>
<td><a href="https://www.addgene.org/171816/">https://www.addgene.org/171816/</a></td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-Ova-BlastR [M1G]</td>
<td>171817</td>
<td><a href="https://www.addgene.org/171817/">https://www.addgene.org/171817/</a></td>
</tr>
</tbody>
</table>
</li>
<li>Day 1 after receiving the agar stabs from Addgene: streak the bacteria on LB agar plates with ampicillin according to the instructions sent with the plasmids</li>
<li>Day 2–3 after receiving the agar stabs from Addgene: inoculate a single colony from the cultured agar plate in 100 mL LB broth with ampicillin, and incubate the LB broth overnight (∼ 16 h) at 37°C in a shaking incubator (180 rpm). The following day make midipreparation of the plasmid DNA using Thermo Fisher Scientific PureLink<sup>TM</sup> HiPure Plasmid Midiprep kit according to the manufacturer’s instructions (<a href="https://assets.fishersci.com/TFS-Assets/LSG/manuals/purelink_hipure_plasmid_dna_purification_man.pdf">https://assets.fishersci.com/TFS-Assets/LSG/manuals/purelink_hipure_plasmid_dna_purification_man.pdf</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Resuspend DNA in ddH<sub>2</sub>O instead of TE buffer.</div>
<ol start="5">
<li>After midipreparation of plasmid DNA, measure the concentration using a NanoDrop spectrophotometer. Store the plasmid at −20°C (&gt; 5 years) or at 4°C for short-term storage. Plasmid integrity can be assessed by performing agarose gel electrophoresis.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> You can use a kit from a different manufacturer for midipreparations e.g. Qiagen.</div>
<h3 id="sec1.3">Target gene selection, sgRNA design and synthesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="6">
<li>Choose your target gene product of interest (GPOI) that you want to modify using the CRISPitope approach.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This protocol describes how to modify murine CDK4 using frame selector +2 with four different universal donor plasmids in detail. However, the same protocol applies when choosing a different target gene (either mouse or human), different frame selectors and different universal donors. In the publication by <a href="#bib2">Effern et al. (2020)</a>, the following other GPOI were modified using the CRISPitope approach (<a href="#tbl2">Table 2</a>):</div>
<div class="note">
<span class="note-title">Note:</span> When choosing your target gene of interest consider the following aspects: In contrast to sgRNA selection strategies for generating knockout cell line models, the selection of sgRNAs for gene tagging is restricted by the availability of a suitable protospacer adjacent motif (PAM) for SpCas9 at the particular genomic region of interest (in our case the encoded stop codon of the GPOI). In addition, one should consider subcellular localization (nuclear, cytoplasmic, membrane, etc.), expression levels (high, intermediate, low), post-transcriptional/post-translational processing and expression pattern (housekeeping, inducible, etc.). The CRISPitope approach generates a C-terminal fusion protein of your GPOI with a universal donor plasmid. C-terminal tagging of your GPOI might interfere with its function e.g., by subcellular mislocalization, which may preclude the generation of stable cell lines expressing the endogenously encoded GPOIs. Furthermore, some amino acids at the C-terminus can be lost due to the tagging approach (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Mmc1.xlsx">Table S1</a>) which could interfere with the function of your GPOI. It is important to check the structure and important domains of your GPOI in a database such as Uniprot (<a href="https://www.uniprot.org/">https://www.uniprot.org/</a>). Strategies to mitigate the potential issue of protein dysfunctionality will be discussed in the below.</div>
<table id="tbl2">
<caption>Table 2. Overview of the additional GPOIs modified using the CRISPitope technique in the publication <a href="#bib2">Effern et al. (2020)</a> including the corresponding frame selector and universal donor plasmid</caption>
<thead>
<tr>
<th>GPOI</th>
<th>Frame selector</th>
<th>Universal donor</th>
</tr>
</thead>
<tbody>
<tr>
<td>CDK4 (Mus musculus)</td>
<td>+2</td>
<td>pCRISPaint-mNeon-PuroR [M1G]pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
</tr>
<tr>
<td>RAB38 (Mus musculus)</td>
<td>+2</td>
<td>pCRISPaint-mNeon-PuroR [M1G]pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
</tr>
<tr>
<td>TYRP1 (Mus musculus)</td>
<td>+2</td>
<td>pCRISPaint-mNeon-PuroR [M1G]pCRISPaint-mNeon-F-hgp100-PuroR [M1G]pCRISPaint-mScarlet-PuroR [M1G]pCRISPaint-mScarlet-F-mgp100-PuroR [M1G]pCRISPaint-mScarlet-F-hgp100-PuroR [M1G]</td>
</tr>
<tr>
<td>DCT (Mus musculus)</td>
<td>+2</td>
<td>pCRISPaint-mNeon-PuroR [M1G]pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
</tr>
<tr>
<td>GPNMB (Mus musculus)</td>
<td>+0</td>
<td>pCRISPaint-mNeon-PuroR [M1G]pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
</tr>
<tr>
<td>SOX10 (Mus musculus)</td>
<td>+1</td>
<td>pCRISPaint-mNeon-PuroR [M1G]pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
</tr>
<tr>
<td>ACTB (Mus musculus)</td>
<td>+2</td>
<td>pCRISPaint-mNeon-BlastR [M1G]pCRISPaint-mNeon-F-gB-BlastR [M1G]pCRISPaint-mScarlet-BlastR [M1G]pCRISPaint-mScarlet-F-gB-BlastR [M1G]</td>
</tr>
<tr>
<td>ACTG1 (Mus musculus)</td>
<td>+2</td>
<td>pCRISPaint-mNeon-BlastR [M1G]pCRISPaint-mNeon-F-gB-BlastR [M1G]</td>
</tr>
</tbody>
</table>
<ol start="7">
<li>Once you have chosen your GPOI, refer to <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Mmc1.xlsx">Table S1</a> (mouse or human) which provides all the information needed to design and obtain a sgRNA for your gene of interest. The table was adapted from Schmid-Burgk and colleagues who provided a list of pre-selected, sequence-optimized target sites close to the translation termination codons of nearly all mouse and human protein coding genes to simplify the application of this method (<a href="#bib3">Schmid-Burgk et al., 2016</a>).
<ol type="a">
<li>
<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Mmc1.xlsx">Table S1</a> shows the gene name, target site, fully designed top and bottom strand oligonucleotide including DNA overhangs for cloning into BbsI-restricted pX330, the corresponding frame selector, the cut site relative to the stop codon and the number of amino acids lost at the C-terminus.</li>
</ol>
</li>
<li>Order the fully designed top and bottom strand oligonucleotides from an oligonucleotide manufacturer. The oligonucleotides used in this protocol were purchased from Microsynth AG (Balgach, Switzerland) but other companies provide the same service e.g., Integrated DNA Technologies IDT.</li>
</ol>
<h3 id="sec1.4">Culture cell line to be modified by CRISPitope-approach</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<ol start="9">
<li>The target cell line HC.PmelKO to be used in this protocol should be thawed and cultured for &gt; 5 days prior to transfection. A vial of frozen cells was retrieved from the −80°C freezer and the cells were rapidly thawed using warm complete RPMI. After thawing, the cells were washed once with warm RPMI and then cultured in complete RPMI. One day after thawing the cells, the media was changed. Two days after thawing, the cells were split (1/5 to 1/10; dependent on the confluency). Afterwards, cells were split twice per week at a 1/10 ratio. Keep the cells in culture for a maximum of 8–10 passages.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This experiment can also be performed with the more commonly used B16 murine melanoma cell line, MZ7 human melanoma cell line, ID8 ovarian cancer cell line or MC38 colon carcinoma cell line or with other cell lines that can be transfected including suspension cells.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>AF700 Mouse anti-rat CD90/mouse CD90.1 antibody (Clone OX-7) (dilution 1:100)</td>
<td>BD Biosciences</td>
<td>Cat-No: 557266, RRID: AB_396611</td>
</tr>
<tr>
<td>APC anti-mouse Vβ 13 TCR antibody (Clone MR12-3) (dilution 1:50)</td>
<td>BD Biosciences</td>
<td>Cat-No: 561542, RRID: AB_10894393</td>
</tr>
<tr>
<td>APC Streptavidin (dilution 1:100)</td>
<td>BD Biosciences</td>
<td>Cat-No: 554067, RRID:AB_10050396</td>
</tr>
<tr>
<td>APC/Cyanine7 anti-mouse CD45 antibody (clone 30-F11) (dilution 1:200)</td>
<td>BioLegend</td>
<td>Cat-No: 103116, RRID: AB_312981</td>
</tr>
<tr>
<td>Biotin anti-mouse H-2D[b] antibody (clone KH95) (dilution 1:50)</td>
<td>BD Biosciences</td>
<td>Cat-No: 553572, RRID:AB_394930</td>
</tr>
<tr>
<td>Brilliant Violet 605™ anti-mouse CD69 antibody (clone H1.2F3) (dilution 1:100)</td>
<td>BioLegend</td>
<td>Cat-No: 104529, RRID: AB_11203710</td>
</tr>
<tr>
<td>Brilliant Violet 711™ anti-mouse CD8α antibody (clone 53-6.7) (dilution 1:200)</td>
<td>BioLegend</td>
<td>Cat-No: 100748, RRID: AB_2562100</td>
</tr>
<tr>
<td>Mouse monoclonal β-Actin (clone C4) (dilution 1:1000)</td>
<td>Santa Cruz</td>
<td>Cat-No: sc-47778, RRID: AB_2714189</td>
</tr>
<tr>
<td>Mouse monoclonal CDK4 (Clone DCS-35) (dilution 1:100)</td>
<td>Santa Cruz</td>
<td>Cat-No: sc-23896, RRID:AB_627239</td>
</tr>
<tr>
<td>Purified anti-mouse CD16/32 antibody (dilution 1:100)</td>
<td>BioLegend</td>
<td>Cat-No: 101302, RRID: AB_312800</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> DH10β</td>
<td>Originally obtained from laboratory of Veit Hornung, University Hospital Bonn, Germany</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>Carl Roth</td>
<td>Cat-No: 3738.5</td>
</tr>
<tr>
<td>Agar-Agar, Kobe I</td>
<td>Carl Roth</td>
<td>Cat-No: 5210.2</td>
</tr>
<tr>
<td>Agarose Standard</td>
<td>Carl Roth</td>
<td>Cat-No: 3810.4</td>
</tr>
<tr>
<td>Ampicillin sodium salt</td>
<td>Carl Roth</td>
<td>Cat-No: K029.2</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 21985-023</td>
</tr>
<tr>
<td>BbsI and corresponding NEBuffer<sup>TM</sup> 2.1</td>
<td>New England Biolabs</td>
<td>Cat-No: R0539L</td>
</tr>
<tr>
<td>Blasticidin S hydrochloride</td>
<td>Sigma-Aldrich</td>
<td>Cat-No: 15205-25MG</td>
</tr>
<tr>
<td>DAPI (4′,6-diamidino-2-phenylindole) (reconstituted in ddH<sub>2</sub>O at 3mM)</td>
<td>BioLegend</td>
<td>Cat-No: 422801</td>
</tr>
<tr>
<td>DMSO (Dimethyl sulfoxide)</td>
<td>Carl Roth</td>
<td>Cat-No: 4720.1</td>
</tr>
<tr>
<td>DPBS (1x) (Dulbecco’s Phosphate Buffered Saline)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 4190-094</td>
</tr>
<tr>
<td>EDTA solution (0.5 M, pH 8.0)</td>
<td>Invitrogen</td>
<td>Cat-No: AM9261</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Sigma-Aldrich</td>
<td>Cat-No: F7524</td>
</tr>
<tr>
<td>GeneRuler 1 kb DNA Ladder</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: SM0311</td>
</tr>
<tr>
<td>H2-D<sup>b</sup> binding peptide KVPRQDWL (hgp100<sub>25-33</sub>) (reconstituted in ddH<sub>2</sub>O at 1 mg/mL)</td>
<td>Anaspec Inc.</td>
<td>Cat-No: AS-62589</td>
</tr>
<tr>
<td>HEPES (4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid)</td>
<td>Carl Roth</td>
<td>Cat-No: 9105.4</td>
</tr>
<tr>
<td>HEPES (4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 15630-106</td>
</tr>
<tr>
<td>Hydrochloric acid 32% (HCl)</td>
<td>Carl Roth</td>
<td>Cat-No: P074.1</td>
</tr>
<tr>
<td>LB-Medium (Lennox)</td>
<td>Carl Roth</td>
<td>Cat-No: X964.2</td>
</tr>
<tr>
<td>L-Glutamine 200 mM (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 25030-024</td>
</tr>
<tr>
<td>MEM Non-Essential Amino Acids</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 11140-050</td>
</tr>
<tr>
<td>Opti-MEM<sup>TM</sup> I Reduced Serum Medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 31985070</td>
</tr>
<tr>
<td>Penicillin Streptomycin (100×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 15140-122</td>
</tr>
<tr>
<td>Puromycin (hydrochloride)</td>
<td>Cayman Chemical</td>
<td>Cat-No: 13884</td>
</tr>
<tr>
<td>RBC (Red Blood Cell) Lysis buffer (10×)</td>
<td>BioLegend</td>
<td>Cat-No: 420301</td>
</tr>
<tr>
<td>Recombinant human IL-2 (Aldesleukin)</td>
<td>Novartis Pharma</td>
<td>PZN: 01138131</td>
</tr>
<tr>
<td>Recombinant murine IFN-γ</td>
<td>PeproTech</td>
<td>Cat-No: 315-05</td>
</tr>
<tr>
<td>RPMI Medium 1640 (1×)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 21875-034</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>Carl Roth</td>
<td>Cat-No: 9265.2</td>
</tr>
<tr>
<td>Sodium hydroxide (NaOH)</td>
<td>Carl Roth</td>
<td>Cat-No: 6771.3</td>
</tr>
<tr>
<td>Sodium Pyruvate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: 11360-070</td>
</tr>
<tr>
<td>SYBR® Safe DNA gel stain</td>
<td>Invitrogen</td>
<td>Cat-No: S33102</td>
</tr>
<tr>
<td>T4 DNA Ligase and corresponding buffer</td>
<td>New England Biolabs</td>
<td>Cat-No: M202S</td>
</tr>
<tr>
<td>TRIS (tris(hydroxymethyl)aminomethane)</td>
<td>Carl Roth</td>
<td>Cat-No: AE15.3</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>FuGENE® HD Transfection Reagent</td>
<td>Promega</td>
<td>Cat-No: E2311</td>
</tr>
<tr>
<td>MACS SmartStrainer</td>
<td>Miltenyi Biotec</td>
<td>Cat-No: 130-098-462</td>
</tr>
<tr>
<td>MEGAquick-spin<sup>TM</sup> plus (Fragment DNA purification kit)</td>
<td>iNtRON Biotechnology</td>
<td>Cat-No: 17290</td>
</tr>
<tr>
<td>PureLink<sup>TM</sup> HiPure Plasmid Midiprep Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat-No: K210005</td>
</tr>
<tr>
<td>QIAprep Spin Miniprep Kit</td>
<td>Qiagen</td>
<td>Cat-No: 12123</td>
</tr>
<tr>
<td>Spin columns for DNA extraction</td>
<td>Canvax</td>
<td>Cat-No: PL0251</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Mouse: HC.PmelKO</td>
<td><a href="#bib2">Effern et al. (2020)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: Pmel-1 TCRtg T cell mouse (B6.Cg-Thy1a/Cy Tg(TcraTcrb)8Rest/J) (sex: male or female; age range: 8–16 weeks)</td>
<td>The Jackson Laboratory</td>
<td>Cat-No: 005023</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Cdk4_TS5′-CACCGCACTCCTACCTGCACAAGG-3′</td>
<td>Microsynth</td>
<td>N/A</td>
</tr>
<tr>
<td>Cdk4_BS5′-AAACCCTTGTGCAGGTAGGAGTGC-3′</td>
<td>Microsynth</td>
<td>N/A</td>
</tr>
<tr>
<td>pX330-U6-seq-f (sequencing primer for pX330)5′-AGGGCCTATTTCCCATGATTC-3′</td>
<td>Microsynth</td>
<td>N/A</td>
</tr>
<tr>
<td>Cdk4 forward primer (for PCR)5′-GAATCTCTGCCTTCCGAGCC-3′</td>
<td>Microsynth</td>
<td>N/A</td>
</tr>
<tr>
<td>mNeon reverse primer (for PCR)5′-CGCTTGCCATTTCCAGTGG-3′</td>
<td>Microsynth</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pX330 (pX330-U6-Chimeric_BB-CBh-hSpCas9)</td>
<td>Addgene</td>
<td>Cat-No: 42230; RRID:Addgene_42230</td>
</tr>
<tr>
<td>Frame selector +0 (pCAS9-mCherry-Frame +0)</td>
<td>Addgene</td>
<td>Cat-No: 66939; RRID:Addgene_66939</td>
</tr>
<tr>
<td>Frame selector +1 (pCAS9-mCherry-Frame +1)</td>
<td>Addgene</td>
<td>Cat-No: 66940; RRID:Addgene_66940</td>
</tr>
<tr>
<td>Frame selector +2 (pCAS9-mCherry-Frame +2)</td>
<td>Addgene</td>
<td>Cat-No: 66941; RRID:Addgene_66941</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171800; RRID: Addgene_171800</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-hgp100-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171801; RRID: Addgene_171801</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-mgp100-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171802; RRID: Addgene_171802</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-gB-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171803; RRID: Addgene_171803</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-Ova-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171804; RRID: Addgene_171804</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171805; RRID: Addgene_171805</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-hgp100-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171806; RRID: Addgene_171806</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-gB-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-Nr:171807; RRID: Addgene_171807</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-Ova-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171808; RRID: Addgene_171808</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171809; RRID: Addgene_171809</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-hgp100-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171810; RRID: Addgene_171810</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-mgp100-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171811; RRID: Addgene_171811</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-gB-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171812; RRID: Addgene_171812</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-Ova-PuroR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171813; RRID: Addgene_171813</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171814; RRID: Addgene_171814</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-hgp100-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171815; RRID: Addgene_171815</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-gB-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171816; RRID: Addgene_171816</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-Ova-BlastR [M1G]</td>
<td>Addgene</td>
<td>Cat-No: 171817; RRID: Addgene_171817</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>FlowJo v10</td>
<td>Tree Star. Inc.</td>
<td><a href="https://flowjo.com/">https://flowjo.com/</a></td>
</tr>
<tr>
<td>SnapGene</td>
<td>Insightful Science</td>
<td><a href="https://www.snapgene.com">https://www.snapgene.com</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>NanoDrop spectrophotometer</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Thermocycler</td>
<td>N/A</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>LB broth with ampicillin (timing: 1 day)</p>
<ul>
<li>Prepare ampicillin stock solution (100 mg/mL in ddH<sub>2</sub>O) (timing: 30 min). Store at −20°C (maximum storage time: 4–6 months).</li>
<li>Weigh out and mix all the components for the LB broth (timing: 30 min).</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LB-Medium (Lennox)</td>
<td>20 g</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>5 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>1 L</td>
</tr>
</tbody>
</table>
<ul>
<li>Autoclave LB broth and store at 4°C (timing: 3–5 h) (maximum storage time: 1–2 months).</li>
<li>Immediately before use, add ampicillin stock solution to LB broth (1:1000 dilution). Final concentration of ampicillin in LB broth: 100 μg/mL.</li>
</ul>
<p>LB agar plates with ampicillin (timing: 1 day)</p>
<ul>
<li>Prepare ampicillin stock solution (100 mg/mL in ddH<sub>2</sub>O) (timing: 30 min). Store at −20°C (maximum storage time: 4–6 months).</li>
<li>Weigh out and mix all the components for the LB agar (timing: 30 min).</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>LB-Medium (Lennox)</td>
<td>20 g</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>5 g</td>
</tr>
<tr>
<td>Agar-Agar, Kobe I</td>
<td>20 g</td>
</tr>
<tr>
<td>dH<sub>2</sub>O</td>
<td>1 l</td>
</tr>
</tbody>
</table>
<ul>
<li>Autoclave LB agar (timing: 3–5 h).</li>
<li>Cool LB agar down to 57°C (timing: 30 min–1 h).</li>
<li>Once the agar has cooled to 57°C, add 1000 μL ampicillin stock solution to 1 l LB agar (1:1000 dilution). The final concentration of ampicillin in LB agar plates is 100 μg/mL.</li>
<li>Swirl the agar bottle to ensure even distribution of the antibiotic throughout the agar.</li>
<li>Pour 20 mL of the LB agar with ampicillin into petri dishes (timing: 30 min).</li>
<li>Let the agar solidify at room temperature (timing: 30 min). Then store at 4°C (maximum storage time: 1–2 months).</li>
</ul>
<p>Annealing buffer (timing: 1 h)</p>
<table id="undtbl3">
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>100 mM</td>
<td>292.2 mg</td>
</tr>
<tr>
<td>HEPES</td>
<td>5 mM</td>
<td>595.078 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Weigh out and add NaCl and HEPES to 40 mL of ddH<sub>2</sub>O.</li>
<li>Adjust pH to 7.4 using hydrochloric acid (HCl).</li>
<li>Once the pH is adjusted, fill up annealing buffer to 50 mL with ddH<sub>2</sub>O.</li>
<li>Store at room temperature (18°C–22°C) (maximum storage time: 1 year).</li>
</ul>
<p>TAE buffer for agarose gels (50×) (timing: 30 min)</p>
<table id="undtbl4">
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>EDTA solution, 0.5 M, pH 8.0</td>
<td>50 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>Tris</td>
<td>2 M</td>
<td>242 g</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>1M</td>
<td>57.1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>-</td>
<td>1 l</td>
</tr>
</tbody>
</table>
<ul>
<li>Weigh out 242 g of Tris and dissolve in 700 mL of dH<sub>2</sub>O.</li>
<li>Carefully add 57.1 mL of acetic acid and 100 mL of 0.5 M EDTA (pH 8.0) solution.</li>
<li>Adjust the solution to a final volume of 1 l (timing: 30 min).</li>
<li>Store stock solution at room temperature (18°C–22°C) (maximum storage time: 1 year).</li>
<li>Dilute stock solution at 1:50 with ddH<sub>2</sub>O for agarose gel electrophoresis.</li>
</ul>
<p>Complete RPMI (cRPMI) and Cryo medium (timing: 15 min)</p>
<table id="undtbl5">
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fetal bovine serum</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>2 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>100 U/mL + 100 μg/mL</td>
<td>5 mL</td>
</tr>
<tr>
<td>MEM Non-Essential Amino Acids</td>
<td>-</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sodium Pyruvate</td>
<td>1 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>22 μM</td>
<td>200 μL</td>
</tr>
<tr>
<td>HEPES (from Thermo Fisher Scientific)</td>
<td>0.1 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>RPMI Medium 1640 (1x)</td>
<td>-</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Add supplements to RPMI medium 1640.</li>
<li>Store cRPMI at 4°C (maximum storage time: 1–2 months).</li>
<li>For cryo medium, supplement cRPMI with 10% DMSO.</li>
</ul>
<p>Flow cytometry buffer (timing: 10 min)</p>
<table id="undtbl6">
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Fetal bovine serum</td>
<td>2%</td>
<td>10 mL</td>
</tr>
<tr>
<td>EDTA solution (0.5 M, pH 8.0)</td>
<td>2 mM</td>
<td>2 mL</td>
</tr>
<tr>
<td>PBS</td>
<td> </td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<ul>
<li>Add FBS and EDTA to PBS in a sterile biosafety cabinet.</li>
<li>Keep the buffer sterile to avoid contamination.</li>
<li>Store at 4°C (maximum storage time: 6 months).</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Generation of target selector</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>These steps describe the generation of the target selector by cloning an sgRNA oligonucleotide insert into a plasmid that encodes Cas9.</p>
<p>Day 1</p>
<ol>
<li>Digest plasmid pX330 with BbsI restriction enzyme.
<ol type="a">
<li>Digest 10 μg of pX330 plasmid with 7 μL of BbsI restriction enzyme at 37°C for 8 h.
<table id="undtbl7">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>pX330 plasmid</td>
<td>10 μg</td>
</tr>
<tr>
<td>BbsI restriction enzyme</td>
<td>7 μL</td>
</tr>
<tr>
<td>NEBuffer<sup>TM</sup> 2.1</td>
<td>3 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>Top up to 30 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Run sample on a 0.7% (w/v) agarose gel in 1×TAE buffer with SYBR® Safe DNA gel stain at 140 V for 30 min. Successful digestion should yield an 8,462 base pairs (bp)and a 22 bp fragment (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Representative picture of agarose gel electrophoresis of pX330 plasmid digested with BbsI</div>
<p>The picture shows undigested pX330 and BbsI-digested pX330 plasmid. Successful digestion yields a 8,462 bp and a 22 bp fragment; the 22 bp fragment is usually not detectable. Digested, linearized plasmid shows a different running pattern compared to undigested, circular plasmid. Ladder: GeneRuler 1 kb DNA ladder (Thermo Scientific<sup>TM</sup>).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Use a 1 kb DNA ladder for size comparison and run undigested pX330 plasmid as a control. You will not be able to detect the 22 bp excised fragment but undigested pX330 will serve as an indicator for linearization of the plasmid. Digested, linearized plasmid shows a different running pattern compared to undigested, circular plasmid (<a href="#fig2">Figure 2</a>).</div>
</li>
<li>Excise the 8,642 bp DNA fragment from the gel and perform a gel extraction using the iNtRON Biotechnology MEGAquick-spinTM plus Fragment DNA purification kit according to manufacturer’s instructions (<a href="https://intronbio.com:6001/intronbioen/product/product_view.php?PRDT_ID=7#none">https://intronbio.com:6001/intronbioen/product/product_view.php?PRDT_ID=7#none</a>). Elute the digested plasmid in ddH<sub>2</sub>O.
<div class="note">
<span class="note-title">Note:</span> You can use a gel extraction kit from a different company as well e.g. Qiagen.</div>
</li>
<li>Measure the concentration of the purified plasmid using a NanoDrop spectrophotometer.</li>
<li>Store digested plasmid at −20°C (&gt; 5 years). Alternatively, the digested plasmid can be stored at 4°C (up to 1 month). Plasmid integrity can be assessed by performing agarose gel electrophoresis. Please also check the Addgene homepage regarding options for long-term storage of DNA (<a href="https://blog.addgene.org/ways-to-elute-and-store-plasmid-dna">https://blog.addgene.org/ways-to-elute-and-store-plasmid-dna</a>).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Protocol can be paused and DNA can be stored as indicated in step 1e before proceeding with step 2.</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Annealing of sgRNA oligonucleotide inserts.
<div class="note">
<span class="note-title">Note:</span> This step can be performed before and/or during step 1.</div>
<ol type="a">
<li>Dissolve oligonucleotides in ddH<sub>2</sub>O to a final concentration of 100 μM.</li>
<li>Set up the following reaction for the <i>Cdk4</i> targeting sgRNA oligonucleotide insert in a PCR tube:
<table id="undtbl8">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<i>Cdk4</i> top strand oligonucleotide</td>
<td>1 μL</td>
</tr>
<tr>
<td>
<i>Cdk4</i> bottom strand oligonucleotide</td>
<td>1 μL</td>
</tr>
<tr>
<td>Annealing buffer</td>
<td>48 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> If you want to clone more than one sgRNA at the same time, set up the reaction above for each individual pair of sgRNA oligonucleotide insert.</div>
</li>
<li>Anneal the sgRNA oligonucleotides using the following program in a thermocycler:
<table id="undtbl9">
<thead>
<tr>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>90°C</td>
<td>4 min</td>
</tr>
<tr>
<td>70°C</td>
<td>10 min</td>
</tr>
<tr>
<td>69°C → 21°C</td>
<td>Decrease temperature every minute by 1°C until you reach 21°C (room temperature)</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> If you do not have a thermocycler, you can also use a heating block. Heat block to 95°C. Incubate your sgRNA oligonucleotides for 5 min. Switch off the heat block and let it cool to room temperature with your samples still in the heating block. In our laboratory, both methods have been used in the past. We did not observe major differences in efficiency between the two different methods.</div>
</li>
<li>Store annealed sgRNA oligonucleotides at 4°C (up to a month) or at −20°C (&gt;1 year).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Protocol can be paused and annealed oligonucleotides can be stored as indicated in step 2d before proceeding with step 3.</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Cloning of the annealed <i>Cdk4</i> sgRNA oligonucleotide into the digested purified pX330 plasmid.
<ol type="a">
<li>Set up the ligation reaction for the <i>Cdk4</i> sgRNA as follows:
<table id="undtbl10">
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Annealed <i>Cdk4</i> sgRNA oligonucleotide (from step 2)</td>
<td>1 μL</td>
</tr>
<tr>
<td>BbsI-digested purified pX330 (from step 1)</td>
<td>100 ng</td>
</tr>
<tr>
<td>T4 DNA Ligase Buffer (10×)</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>Top up to 10 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> If you want to clone more than one sgRNA at the same time, set up the reaction above for each individual annealed sgRNA oligonucleotide insert.</div>
</li>
<li>Incubate ligation overnight (∼16 h) at 16°C.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Protocol can be paused and ligated plasmids can be stored at -20°C (up to a month) before proceeding with step 3c.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare a negative control reaction (i.e. ligation of BbsI-digested purified pX330 plasmid alone without annealed sgRNA oligonucleotide as insert).</div>
Day 2</li>
<li>Transform the ligation reaction into <i>Escherichia coli</i> DH10β chemically competent bacteria.
<ol type="i">
<li>Thaw bacteria on ice.</li>
<li>Add 5 μL of the ligation mix to 50 μL of bacteria.</li>
<li>Incubate for 10 min on ice.</li>
<li>Perform a heat shock. If the bacteria are in a microcentrifuge tube, incubate for 1 min at 42°C. If the bacteria are in PCR tubes, incubate for 45 s at 42°C.</li>
<li>Incubate for 2 min on ice.</li>
<li>Plate the bacteria on pre-warmed (37°C) LB agar plates containing ampicillin.</li>
<li>Incubate plates upside down at 37°C overnight (16–18 h).</li>
<li>Assess the plate for colony growth. Multiple colonies should have grown on the pX330-sgRNA plates whilst the negative control plate should not have any colony growth (see <a href="#troubleshooting">troubleshooting</a> section if observed otherwise).</li>
<li>Store plates in the fridge (for up to 1 week).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Protocol can be paused and LB agar plates can be stored at 4°C (up to 1 week) before proceeding with step 4.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<p>Day 3 and 4</p>
<ol start="4">
<li>Amplification and isolation of pX330 containing the sgRNA oligonucleotide insert.
<ol type="a">
<li>Inoculation of LB broth for minipreparation of plasmid DNA.
<ol type="i">
<li>Pick 5 colonies from each plate.</li>
<li>For each individual bacteria colony, prepare a microcentrifuge tube (2 mL volume) with 1.5 mL LB broth with ampicillin. <b><u>Carefully</u></b> poke holes into the lid with a needle for better aeration.</li>
<li>Use a sterile pipette tip to pick a single colony and inoculate the LB broth containing ampicillin with the single colony.</li>
<li>Incubate the LB broth overnight (16–18 h) in a shaking heat block at 37°C and 750 rpm.</li>
</ol>
</li>
<li>Isolate plasmid DNA from bacteria using the Qiagen QIAprep Spin Miniprep Kit according to manufacturer’s instructions (<a href="https://www.qiagen.com/us/resources/resourcedetail?id=0bd0c5fb-c271-43e7-af43-32d539374fa9&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=0bd0c5fb-c271-43e7-af43-32d539374fa9&amp;lang=en</a>). Elute DNA in ddH<sub>2</sub>O.
<div class="note">
<span class="note-title">Note:</span> You can also use a miniprep kit from a different manufacturer e.g. Thermo Fisher Scientific.</div>
</li>
<li>Measure the concentration of the plasmid using a NanoDrop spectrophotometer.<br> Day 5 and following</li>
<li>Verify integration of the sgRNA into pX330 by Sanger sequencing. Select two or three isolated plasmids and sequence from the U6 promoter using the primer pX330-U6-seq-f (keep the remaining plasmids as backup). It depends on your Sanger sequencing service of choice, how long it takes until you receive sequencing results. The sgRNA sequence should be inserted between the U6 promoter and the sgRNA scaffold, i.e., between the BbsI restriction sites 5′-CACC-3′ and 5′-GTTT-3’ (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Plasmid map of pX330 with integrated sgRNA</div>
<p>Plasmid map of pX330 including the following features: the sequencing primer pX330-U6-seq-f (purple), the U6 promoter (white), BbsI restriction sites (black), the sgRNA integration site (red), the gRNA scaffold (light blue), the CMV enhancer (white) and 3xFLAG-Cas9 with a nuclear localization sequence (purple), origin of replication (yellow) and an Ampicillin resistance (light green). The plasmid map was generated using SnapGene software (from Insightful Science; available at <a href="http://snapgene.com">snapgene.com</a>).</p>
</figcaption>
</figure>
</li>
<li>Store the plasmid at −20°C (&gt; 5 years) or short-term at +4°C.</li>
<li>Once you have verified your pX330-sgRNA plasmid, you can proceed to perform a midipreparation of your plasmid using the Thermo Fisher Scientific PureLink<sup>TM</sup> HiPure Plasmid Midiprep kit according to manufacturer’s instructions (<a href="https://assets.fishersci.com/TFS-Assets/LSG/manuals/purelink_hipure_plasmid_dna_purification_man.pdf">https://assets.fishersci.com/TFS-Assets/LSG/manuals/purelink_hipure_plasmid_dna_purification_man.pdf</a>).</li>
<li>After midipreparation of your plasmid DNA, measure the concentration using a NanoDrop spectrophotometer.</li>
<li>The target selector plasmid generated in this step is named pX330-Cdk4_Ct.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Transfection of murine melanoma cell lines with CRISPitope plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 days</div>
<p>The following steps describe the transfection of the murine melanoma cell line HC.PmelKO with three CRISPitope plasmids (target selector, frame selector and universal donor) using the Promega Fugene® HD transfection reagent. During these steps, four different cell lines will be generated that differ by the integration of different universal donor plasmid DNA into the 3′-end of the coding sequence of the <i>Cdk4</i> gene. The following universal donor plasmids are used during this step: pCRISPaint-mNeon-PuroR [M1G], pCRISPaint-mNeon-F-hgp100-PuroR [M1G], pCRISPaint-mScarlet-BlastR [M1G], and pCRISPaint-mScarlet-F-hgp100-BlastR [M1G].</p>
<div class="note">
<span class="note-title">Note:</span> From this point onwards, all steps should be performed in a sterile biosafety cabinet (class II).</div>
<p>Day 1 in the morning</p>
<ol start="5">
<li>For each transfection, plate 50,000 HC.PmelKO melanoma cells in 100 μL cRPMI, into a flat-bottom 96-well, in the morning. Seed one additional well, which will not be transfected and serve as a negative control for antibiotic selection (see below).</li>
<li>Dilute plasmids as follows:</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th>Plasmid</th>
<th>Concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td>Target selector (pX330-Cdk4_Ct)</td>
<td>50 ng/μL</td>
</tr>
<tr>
<td>Frame selector (+2)</td>
<td>50 ng/μL</td>
</tr>
<tr>
<td>Universal donors: pCRISPaint-mNeon-PuroR [M1G] <b><u>or</u></b> pCRISPaint-mNeon-F-hgp100-PuroR [M1G] <b><u>or</u></b> pCRISPaint-mScarlet-BlastR [M1G] <b><u>or</u></b> pCRISPaint-mScarlet-F-hgp100-BlastR [M1G].</td>
<td>100 ng/μL</td>
</tr>
</tbody>
</table>
<p>Day 1 in the afternoon</p>
<ol start="7">
<li>Six hours after plating the melanoma cells, prepare the transfection mixture. Firstly, mix Opti-MEM<sup>TM</sup> I Reduced serum medium with the three CRISPitope plasmids. Vortex the mixture and perform a quick spin. Add the Fugene® HD transfection reagent, gently resuspend, and then perform another quick spin. Incubate the mixture for 15 min at room temperature (18°C–22°C).</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Opti-MEM<sup>TM</sup> I Reduced serum medium</td>
<td>7 μL</td>
</tr>
<tr>
<td>Target selector (pX330-Cdk4_Ct) [50 ng/μL]</td>
<td>1 μL</td>
</tr>
<tr>
<td>Frame selector (+2) [50 ng/μL]</td>
<td>1 μL</td>
</tr>
<tr>
<td>Universal donor [100 ng/μL] pCRISPaint-mNeon-PuroR [M1G] <b><u>or</u></b> pCRISPaint-mNeon-F-hgp100-PuroR [M1G] <b><u>or</u></b> pCRISPaint-mScarlet-BlastR [M1G] <b><u>or</u></b> pCRISPaint-mScarlet-F-hgp100-BlastR [M1G].</td>
<td>1 μL</td>
</tr>
<tr>
<td>Fugene® HD transfection reagent</td>
<td>0.6 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Set up an individual transfection mixture for each transfection with a different universal donor plasmid.</div>
<ol start="8">
<li>Add the transfection mixture to the adherent melanoma cells and resuspend the media gently without disturbing the cells.</li>
<li>Incubate cells for 48 h in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</li>
</ol>
<ol start="10">
<li>The newly generated cell lines are called:
<ol type="a">
<li>HC.PmelKO.CDK4<sup>mut</sup>-mNeon (universal donor: pCRISPaint-mNeon-PuroR [M1G])</li>
<li>HC.PmelKO.CDK4<sup>mut</sup> -NFhgp100 (universal donor: pCRISPaint-mNeon-F-hgp100-PuroR [M1G])</li>
<li>HC.PmelKO.CDK4<sup>mut</sup> -mScarlet (universal donor: pCRISPaint-mScarlet-BlastR [M1G])</li>
<li>HC.PmelKO.CDK4<sup>mut</sup> -SFhgp100 (universal donor: pCRISPaint-mScarlet-F-hgp100-BlastR [M1G])
<div class="note">
<span class="note-title">Note:</span> After 24–48 h you should be able to see mCherry positive cells. The frame selector encodes the mCherry fluorescent protein as an internal transfection control.</div>
<div class="note">
<span class="note-title">Note:</span> Transfection efficiency varies between cell lines. For HC.PmelKO, we have observed transfection efficiencies between 20 and 50%. In the more widely used B16 mouse melanoma cell line, we have observed transfection efficiencies between 60 and 80% whereas we have only observed a transfection efficiency between 10 and 20% for the murine ovarian cancer cell line ID8.</div>
<div class="note">
<span class="note-title">Note:</span> This experimental setup can be scaled up to bigger well sizes e.g. using 12-well plates. The surface area of a well of a 12-well plate is approximately ten times the surface as of a well of a 96-well plate. In order to scale up the experiment, plate 500,000 HC.PmelKO melanoma cells in 1 ml of cRPMI instead of 50,000 cells in 100 μL. Mix 70 μL of Opti-MEM<sup>TM</sup> I reduced serum medium, 10 μL target selector [50 ng/μL], 10 frame selector [50 ng/μL], universal donor [100 ng/μL] and 6 μL Fugene® HD transfection reagent as described above (step 7). Add the transfection mixture to the adherent cells.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This experiment can be performed with any cell line (including suspension cells) that can be transfected. Examples are the more commonly used B16 murine melanoma cell line, MZ7 human melanoma cell line, ID8 ovarian cancer cell line and MC38 colon carcinoma cell line. Cell size differs significantly between cell lines. Aim for a cell density that represents 60–70% confluency upon transfection as this works best for most cell lines.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Some cell lines require more time to adhere to the tissue culture surface than others. In this case, plate the cells in the afternoon/evening and perform transfection the following morning.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Antibiotic selection and sorting of CRISPitope-engineered cell lines</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 weeks</div>
<p>This section outlines steps to enrich CRISPitope-engineered cells by antibiotic selection and fluorescent cell sorting.</p>
<p>2 Days after transfection</p>
<ol start="11">
<li>Antibiotic selection of CRISPitope-engineered cell lines
<ol type="a">
<li>Start the antibiotic selection, 48 h after transfection.</li>
<li>Remove media from the tissue culture plate and replace with cRPMI containing either puromycin (final concentration: 2 μg/mL) or blasticidin (final concentration: 5 μg/mL). Puromycin selection usually takes 3–4 days whilst blasticidin selection typically takes 5–7 days. Replace the medium supplemented with antibiotics every 48 h.
<div class="critical">
<span class="critical-title">Critical:</span> Add antibiotic-containing media to control cells that have not been transfected. These cells will serve as control to determine when to stop the selection.</div>
</li>
<li>When the control cells are dead, remove the antibiotic-containing media and replace with fresh cRPMI.</li>
<li>Leave the cells in cRPMI for 1–2 weeks to recover.</li>
<li>When the wells are confluent, expand the cells to a bigger well.</li>
<li>Perform flow cytometry analysis to determine the frequency of either mNeon or mScarlet positive cells. The data shown in this protocol was acquired using an LSR Fortessa flow cytometer (BD Biosciences) and analysis performed on FlowJo.
<ol type="i">
<li>Information on the fluorescent proteins: mNeon (<a href="https://www.fpbase.org/protein/mneongreen/">https://www.fpbase.org/protein/mneongreen/</a>) and mScarlet (<a href="https://www.fpbase.org/protein/mscarlet/">https://www.fpbase.org/protein/mscarlet/</a>).
<div class="note">
<span class="note-title">Note:</span> Efficiency is higher when the protein of interest is expressed at higher levels. If your GPOI is expressed at low levels, you can start your antibiotic selection later that 48 h after transfection.</div>
<div class="note">
<span class="note-title">Note:</span> If using a different cell line, titrate the antibiotic selection media before you start your experiment. Antibiotic working concentrations can differ significantly between cell lines.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">After recovery of the cells from antibiotic selection</h3>
<ol start="12">
<li>Cell sorting of antibiotic-selected CRISPitope-engineered cell lines.
<div class="note">
<span class="note-title">Note:</span> After antibiotic selection, not all cells will be positive for the CRISPitope modification. Therefore, successfully transfected cells can be enriched by cell sorting for mNeon or mScarlet positive cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> mCherry (fluorescent protein in frame selector) and mScarlet (fluorescent protein in universal donor) have similar emission spectra. The mCherry signal will only be transiently expressed in cells (2–3 weeks) whereas the mScarlet signal will be stably expressed. Consider this before planning a cell sort.</div>
<ol type="a">
<li>Harvest cells in cRPMI.</li>
<li>Sort either mNeon or mScarlet positive cells into cRPMI. In this protocol, cells were sorted using the FACS Aria III (BD Biosciences).</li>
<li>Collect at least 20,000 mNeon or mScarlet positive cells for a polyclonal culture.
<div class="note">
<span class="note-title">Note:</span> Monoclonal cultures may be obtained by sorting individual cells into each well of a 96-well plate containing 100 μL cRPMI. However, note that not all cell lines grow well as monoclonal cultures.</div>
</li>
<li>Wash the cells in fresh cRPMI after sorting.</li>
<li>Plate the cells in an appropriately sized well.</li>
<li>Expand the cells after cell sorting and make cryo stocks of your new cell line.
<ol type="i">
<li>For cryo stocks, harvest cells and spin them at 400 <i>g</i> for 5 min.</li>
<li>Discard the supernatant and resuspend the cell pellet in cryo media (cRPMI supplemented with 10% DMSO). Transfer cell suspension to a cryo vial and place the cryo vial in a Mr. Frosty<sup>TM</sup> freezing container or similar. Freeze at −80°C.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">After recovery of the cells from cell sorting</h3>
<ol start="13">
<li>Validation of antibiotic-selected and sorted cells.
<ol type="a">
<li>Harvest the antibiotic-selected and sorted cells.</li>
<li>Perform flow cytometry analysis to validate that all your cells are either mNeon or mScarlet positive.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> You can perform other assays in addition/as an alternative to flow cytometry for validation of your newly generated CRISPitope-engineered cell line. Additional assays that can be performed include PCR or Western blotting. PCR followed by Sanger sequencing for the modified 3′ end of your gene of interest can be used to detect successful introduction of the universal donor DNA at the locus of interest. For this you have to design a forward primer that binds at 3′ end of the gene of interest (in the last exon) and a reverse primer that binds in the integrated universal donor (<a href="#fig4">Figures 4</a>A and 4B).</div>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. PCR and Western blot analysis results of CRISPitope-engineered melanoma cells</div>
<p>(A) Depiction of engineered endogenous CDK4<sup>mut</sup> fusion products with mNeon (top) and NF-hgp100 tag (bottom).</p>
<p>(B) Detection of mNeon and NF-hgp100 tag by PCR analysis of genomic DNA (gDNA) and complementary DNA (cDNA) from non-engineered HC.PmelKO and CRISPitope-engineered HC.PmelKO.CDK4<sup>mut</sup>-mNeon and HC.PmelKO.CDK4<sup>mut</sup>-NF-hgp100 melanoma cells. Amplified region (CRISPitope amplicon) is depicted. β-Actin was used as loading control.</p>
<p>(C) Western blot analysis of CDK4 expression in CRISPitope-engineered melanoma cells. β-Actin was used as loading control. Images from (A) and (C) were taken from <a href="#bib2">Effern et al. (2020)</a>.</p>
</figcaption>
</figure>
<p>For Western blotting, you will need an antibody for your protein of interest. It is most likely that only one of the alleles will be modified using the CRISPitope approach, whereas the other allele will remain unmodified. If CRISPitope-engineering was successful, you should observe two bands of your protein of interest at different molecular weights. The band with the higher molecular weight (approximately 30 kDa heavier than wildtype protein) represents the modified allele (<a href="#fig4">Figure 4</a>C). Preferably, a Western blot antibody specific for your endogenous GPOI is used as this allows visualization of tagged and untagged GPOIs. In case that no suitable antibody is available you either use antibodies against FLAG-tag or mNeon for the detection of the tagged GPOI. <a href="#fig4">Figure 4</a> depicts the results from PCR and Western blot analysis. However, as these two methods are not the main focus of this STAR protocol, there is no detailed description of these methods. However, we listed the primer sequences and the Western blot antibody clones in the key resource table. Description of the methods can be found in the publication where this protocol is related to <a href="#bib2">Effern et al. (2020)</a>.</p>
<p>As almost every gene product can be targeted by this approach, there is no single universal assay to validate the functionality of the tagged gene product. In our lab, depending on the study question, we assess the functionality of the tagged gene products using different methods: immunofluorescence microscopy to ensure correct subcellular localization, Western blotting, qRT-PCR and flow cytometry for endogenous expression level and regulation. Depending on the focus of your project, several additional analyses may be needed to confirm full functionality of the tagged GPOI such as incorporation in endogenous protein complexes (e.g., by co-immunoprecipitation or mass spectrometry) or confirmation that the tagged GPOI can compensate for the wildtype GPOI in isolated single cell clones that harbor only homozygously tagged GPOI. Nevertheless, most assays to confirm functionality will largely depend on your GPOI and may differ from our general recommendations.</p>
<h3 id="sec3.6">T cell activation assay using TCRtg CD8<sup>+</sup> T cells and CRISPitope-engineered melanoma cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>This section describes the preparation and plating of CRISPitope-engineered melanoma cell lines and Pmel-1 TCRtg T cells and the subsequent assessment of T cell activation. T cell activation is assessed via detection of the surface marker CD69 by flow cytometry, after co-culture.</p>
<ol start="14">
<li>Preparation and stimulation of CRISPitope-engineered melanoma cells.
<ol type="a">
<li>IFN-γ stimulation
<ol type="i">
<li>Plate 1.5 × 10<sup>6</sup> newly generated CRISPitope-engineered HC.PmelKO melanoma cells or untransfected HC.PmelKO control cells in a 6-well in 1 mL cRPMI media (enough for approx. 15 activation conditions).</li>
<li>Wait until cells have adhered (approximately 6 h), then add 1 mL cRPMI with 2000 U mL<sup>−1</sup> recombinant murine IFN-γ to the HC.PmelKO melanoma cells (final concentration of IFN-γ: 1000 U mL<sup>−1</sup>).</li>
<li>Culture the cells for 3 days in IFN-γ-containing cRPMI in a humidified incubator at 37°C with 5% CO<sub>2</sub> to induce upregulation of antigen processing and presentation.
<div class="critical">
<span class="critical-title">Critical:</span> Plate additional 1.5 × 10<sup>6</sup> CRISPitope-engineered HC.PmelKO or untransfected HC.PmelKO control melanoma cells in 2 mL cRPMI media per well of a 6-well plate and incubate for 3 days. These cells will not be stimulated with IFN-γ and thus do not upregulate antigen processing and presentation. The non-stimulated cells do not induce T cell activation and serve as negative control.</div>
</li>
</ol>
</li>
<li>After 3 days, re-plate IFN-γ stimulated and unstimulated cells for a T cell co-culture assay (Please do not discard leftover cells as these are still needed for step 15).
<ol type="i">
<li>In the morning of the T cell activation assay, harvest melanoma cells using a cell scraper.
<div class="critical">
<span class="critical-title">Critical:</span> Do not use Trypsin to harvest the cells as this removes the hgp100 epitope bound to MHC class I (H2-D<sup>b</sup>) on the surface of the melanoma cells.</div>
</li>
<li>Count the cells using a hemocytometer and Trypan blue for dead cell exclusion.</li>
<li>Adjust the melanoma cell concentration to 1 × 10<sup>5</sup> cells in 100 μL cRPMI supplemented with 1000 U mL<sup>−1</sup> IFN-γ for cells that were previously stimulated with IFN-γ; do not add IFN-γ to the unstimulated negative control cells.</li>
<li>Plate 100 μL of the cell suspension per well of a 96-well plate (an example layout is shown in <a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Example plate layout for melanoma cell - T cell co-culture assay</div>
<p>Newly generated CRISPitope-engineered melanoma cell lines plus a control cell line (HC.PmelKO) were either left unstimulated or were stimulated with 1000 U mL<sup>−1</sup> IFN-γ for 3 days. Melanoma cells were then re-plated to ensure the same cell number per well (100,000 melanoma cells per well in a 96-well plate). Untransfected HC.PmelKO cells serve as negative control, whereas peptide-pulsed HC.PmelKO cells serve as positive control. Upon adherence of the melanoma cells to the cell culture surface, 150,000 naïve pmel-1 TCRtg CD8<sup>+</sup> T cells were added per well. The co-cultured melanoma and T cells were incubated for 16 h. The assay was performed in technical duplicates.</p>
</figcaption>
</figure>
</li>
<li>Let the cells adhere (takes around 6 h).</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="15">
<li>Assess MHC class I expression on unstimulated and IFN-γ-stimulated melanoma cells using flow cytometry.
<ol type="a">
<li>Use leftover cells from 14.b for flow cytometry analysis of MHC class I surface expression on the melanoma cells.</li>
<li>Transfer melanoma cells to a round-bottom 96-well plate.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min.</li>
<li>Discard the supernatant.</li>
<li>Wash the cells once with 200 μL flow cytometry buffer.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min.</li>
<li>Discard the supernatant.</li>
<li>Stain unstimulated and IFN-γ- stimulated melanoma cells with antibody for H2-D<sup>b</sup>. Mix the following components and add the mixture to the melanoma cells:
<table id="undtbl13">
<thead>
<tr>
<th>Component</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>Flow cytometry buffer</td>
<td>50 μL</td>
</tr>
<tr>
<td>Biotin Anti-mouse H-2D[b] antibody</td>
<td>1 μL</td>
</tr>
<tr>
<td>APC Streptavidin</td>
<td>0.5 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> You can also use a directly-conjugated antibody or choose a different fluorophore to match the laser and filter setup of your flow cytometer. Avoid using FITC or PE as these fluorophores overlap in spectra with mNeon or mScarlet, respectively.</div>
</li>
<li>Incubate cells in the fridge for 20 min.</li>
<li>Add 200 μL flow cytometry buffer.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min.</li>
<li>Discard the supernatant.</li>
<li>Resuspend the cells in 100 μL flow cytometry buffer.</li>
<li>Prepare DAPI solution. Add 1 μL DAPI stock solution (3 mM) to 500 μL flow cytometry buffer (1:500 dilution). You will need 100 μL of diluted DAPI solution per flow cytometry sample.</li>
<li>Add 100 μL DAPI solution for dead cell exclusion 5 min before you analyze the sample by flow cytometry (final concentration of DAPI: 3 μM).</li>
<li>Perform flow cytometry and assess the upregulation of H2-D<sup>b</sup> on IFN-γ stimulated melanoma cells compared to unstimulated cells. The data in this protocol was acquired with an LSR Fortessa flow cytometer (BD Biosciences). Analysis was performed using FlowJo.</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Isolation of splenocytes from naïve Pmel-1 TCRtg CD8<sup>+</sup> T cell mice.
<div class="note">
<span class="note-title">Note:</span> This part of the protocol should be performed while cells from 14.b establish adherence.</div>
<ol type="a">
<li>Harvest a spleen(s) from a Pmel-1 TCRtg mouse.</li>
<li>Place the spleen(s) in DPBS on ice.</li>
<li>Place spleen(s) onto a 70 μm cell strainer set on top of a 50 mL canonical tube.</li>
<li>Using a 2 mL syringe plunger, mechanically dissociate the spleen(s) through the strainer.</li>
<li>Rinse the strainer and the plunger with sterile DPBS (up to 50 mL) to ensure that all the splenocytes are collected in the tube.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min at 4°C.</li>
<li>Discard the supernatant.</li>
<li>Resuspend the cell pellet in 1 mL 1×RBC lysis buffer (1 mL per spleen) and incubate at room temperature (18°C–22°C) for 2 min.</li>
<li>Top up the tube to 50 mL with DPBS.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min at 4°C.</li>
<li>Discard the supernatant.</li>
<li>Resuspend the cells in 5 mL cRPMI and count the cells using a hemocytometer.</li>
<li>Take a small aliquot of the cells (20 μL) for flow cytometry analysis to determine the frequency of Pmel-1 TCRtg CD8<sup>+</sup> T cells.
<div class="note">
<span class="note-title">Note:</span> Greater than 95% of the CD8<sup>+</sup> T cells in the Pmel-1 TCRtg mice express the transgenic TCR amounting to about 20% of all splenocytes (<a href="https://www.jax.org/strain/005023">https://www.jax.org/strain/005023</a>).</div>
</li>
<li>Wash the splenocytes with 200 μL flow cytometry buffer.</li>
<li>Block unspecific binding of flow cytometry antibodies with purified CD16/32 antibody (FC block). Add 50 μL of flow cytometry buffer supplemented with 0.25 μg (0.5 μL) purified anti-mouse CD16/32 antibody. Incubate on ice or in the fridge for 10 min.</li>
<li>Stain the cells with the following antibodies in flow cytometry buffer:
<table id="undtbl14">
<thead>
<tr>
<th>Component</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>Flow cytometry buffer</td>
<td>50 μL</td>
</tr>
<tr>
<td>APC/Cyanine7 anti-mouse CD45 antibody</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Brilliant Violet<sup>TM</sup> 711 anti-mouse CD8α antibody</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>AF700 anti-rat CD90/anti-mouse CD90.1 antibody</td>
<td>1 μL</td>
</tr>
<tr>
<td>APC anti-mouse Vβ13 TCR antibody</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> You can also use different fluorophores to match the laser and filter setup of your flow cytometer. However, avoid the use of FITC or PE as these fluorophores overlap with mNeon or mScarlet, respectively.</div>
</li>
<li>Incubate the cells in the fridge for 20 min.</li>
<li>Add 200 μL of flow cytometry buffer to cells.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min at 4°C.</li>
<li>Discard the supernatant.</li>
<li>Resuspend the cells in 100 μL of flow cytometry buffer.</li>
<li>Prepare DAPI working solution. Add 1 μL DAPI stock solution (3 mM) to 500 μL flow cytometry buffer (1:500 dilution). You will need 100 μL of diluted DAPI solution per flow cytometry sample.</li>
<li>Add 100 μL DAPI working solution for dead cell exclusion 5 min before you analyze the sample by flow cytometry (final concentration of DAPI: 3 μM).</li>
<li>Perform flow cytometry analysis and assess the frequency of live CD45<sup>+</sup> CD8<sup>+</sup> CD90.1<sup>+</sup> TCR Vβ13<sup>+</sup> T cells (i.e., Pmel-1 TCRtg T cells). Data in this protocol was acquired on an LSR Fortessa flow cytometer (BD Biosciences). Analysis was performed using FlowJo.</li>
</ol>
</li>
</ol>
<ol start="17">
<li>Preparation of positive control for T cell activation assay.
<ol type="a">
<li>Approximately 5 h after you re-plated the melanoma cells, start preparing the positive control for T cell activation. HC.PmelKO melanoma cells (unstimulated and IFN-γ stimulated) pulsed with hgp100 peptide will serve as positive controls.</li>
<li>Dilute 1 μL peptide stock solution (1 mg mL<sup>−1</sup>) in 500 μL cRPMI.</li>
<li>Add 100 μL this peptide solution to the wells that will serve as positive control which already contain 100 μL cRPMI (<a href="#fig5">Figure 5</a>). Final peptide concentration is 1 μg mL<sup>−1</sup>.</li>
<li>Incubate for 45–60 min in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</li>
</ol>
</li>
<li>Co-culture of naïve Pmel-1 TCRtg CD8<sup>+</sup> T cells and (IFN-γ stimulated) CRISPitope-engineered HCmel12 melanoma cells.
<ol type="a">
<li>Adjust cell concentration to 1.5x10<sup>5</sup> Pmel-1 TCRtg CD8<sup>+</sup> T cells in 200 μL cRPMI supplemented with 30 U mL<sup>−1</sup> human recombinant IL-2.</li>
<li>Remove cRPMI from the melanoma cells.</li>
<li>Wash the melanoma cells gently with DPBS.</li>
<li>Add 200 μL Pmel-1 TCRtg CD8<sup>+</sup> T cell suspension to the melanoma cells.</li>
<li>Incubate the co-cultured cells for 16 h in a humidified incubator at 37°C with 5% CO<sub>2</sub>.</li>
</ol>
</li>
</ol>
<ol start="19">
<li>Assessment of T cell activation by flow cytometry.
<ol type="a">
<li>Harvest the Pmel-1 TCRtg CD8<sup>+</sup> T cells using a multichannel pipette.</li>
<li>Transfer the T cells to a round-bottom 96-well plate.</li>
<li>Spin the plate at 400 <i>g</i> for 5 min at 4°C.</li>
<li>Discard the supernatant.</li>
<li>Add 200 μL of flow cytometry buffer to the wells that contained melanoma cells and the T cells and wash the wells with flow cytometry buffer by pipetting. This ensures that you collect all the T cells.</li>
<li>Consolidate the cells from the washing step with the cells that you have already harvested.</li>
<li>Spin the plate at 400 <i>g</i> for 5 min at 4°C.</li>
<li>Discard the supernatant.</li>
<li>Wash the splenocytes with 200 μL flow cytometry buffer.</li>
<li>Block unspecific binding of flow cytometry antibodies with purified CD16/32 antibody (FC block). Add 50 μL of flow cytometry buffer supplemented with 0.25 μg (0.5 μL) purified anti-mouse CD16/32 antibody. Incubate on ice or in the fridge for 10 min.</li>
<li>Stain the cells with the following antibodies in flow cytometry buffer:
<table id="undtbl15">
<thead>
<tr>
<th>Component</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>Flow cytometry buffer</td>
<td>50 μL</td>
</tr>
<tr>
<td>APC/Cyanine7 anti-mouse CD45 antibody</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Brilliant Violet<sup>TM</sup> 711 anti-mouse CD8α antibody</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>AF700 anti-rat CD90/anti-mouse CD90.1 antibody</td>
<td>1 μL</td>
</tr>
<tr>
<td>APC anti-mouse Vβ13 TCR antibody</td>
<td>2 μL</td>
</tr>
<tr>
<td>Brilliant Violet<sup>TM</sup> 605 anti-mouse CD69 antibody</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> You can also use different fluorophores to match the laser and filter setup of your flow cytometer. However, avoid the use of FITC or PE as these fluorophores overlap with mNeon or mScarlet, respectively.</div>
</li>
<li>Incubate in the fridge for 20 min.</li>
<li>Add 200 μL flow cytometry buffer.</li>
<li>Spin the cells at 400 <i>g</i> for 5 min.</li>
<li>Discard the supernatant.</li>
<li>Resuspend the cells in 100 μL flow cytometry buffer.</li>
<li>Prepare DAPI working solution. Add 1 μL DAPI stock solution (3 mM) to 500 μL flow cytometry buffer (1:500 dilution). You will need 100 μL of the diluted DAPI solution per flow cytometry sample.</li>
<li>Add 10 μL DAPI working solution for dead cell exclusion 5 min before flow cytometry analysis (final concentration of DAPI: 3 μM).</li>
<li>Perform flow cytometry analysis and assess the frequency of live CD45<sup>+</sup> CD8<sup>+</sup> CD90.1<sup>+</sup> TCR Vβ13<sup>+</sup> CD69<sup>+</sup> T cells (i.e., activated pmel-1 TCRtg T cells). The data in this protocol was acquired on an LSR Fortessa flow cytometer (BD Biosciences). Analysis was performed using FlowJo.
<div class="note">
<span class="note-title">Note:</span> There are different universal donor plasmids available that encode different model CD8<sup>+</sup> T cell epitopes. If you do not have the Pmel-1 TCRtg mouse line available, this protocol is also suitable for the OT-I TCRtg mouse line (<a href="https://www.jax.org/strain/003831">https://www.jax.org/strain/003831</a>) or gBT-I TCRtg mouse line (<a href="http://www.informatics.jax.org/allele/MGI:3722258">http://www.informatics.jax.org/allele/MGI:3722258</a>).</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Stimulation of the melanoma cells for 72 h with IFN-γ leads to upregulation of MHC class I (H2-D<sup>b</sup>) (<a href="#fig6">Figures 6</a>A and 6B). After co-culturing the CRISPitope-engineered melanoma cells and the Pmel-1 TCRtg CD8<sup>+</sup> T cells, we expect to see upregulation of the early activation marker CD69 on the T cells that have been co-cultured with IFN-γ stimulated HC.PmelKO.CDK4<sup>mut</sup>-NF-hgp100 (<a href="#fig6">Figures 6</a>C and 6D) and HC.PmelKO.CDK4<sup>mut</sup>-SF-hgp100 melanoma cells. Upregulation of the early activation marker CD69 on the T cells that have been co-cultured with unstimulated melanoma cells or IFN-γ stimulated HC.PmelKO.CDK4<sup>mut</sup>-mNeon and HC.PmelKO.CDK4<sup>mut</sup>-mScarlet melanoma cells is not expected. Unstimulated melanoma cells express a very low level of surface MHC class I that is essential for presentation of the epitope to the T cells. The melanoma cell lines HC.PmelKO.CDK4<sup>mut</sup>-mNeon and HC.PmelKO.CDK4<sup>mut</sup>-mScarlet only express mNeon or mScarlet fluorescent protein but do not express the hgp100 epitope and can therefore not induce T cell activation.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Recognition of CRISPitope-engineered melanoma cells by Pmel-1 TCRtg CD8<sup>+</sup> T cells</div>
<p>(A and B) Quantification of frequency (A) and mean fluorescence intensity (MFI) (B) of H2-D<sup>b</sup> surface expression on indicated CRISPitope-engineered melanoma cells treated with IFN-γ for 72 h (n ≥ 3; mean ± SD).</p>
<p>(C and D) Quantification showing CD69 surface expression on CD8<sup>+</sup> CD90.1<sup>+</sup> Pmel-1 TCRtg T cells 16 h after co-culture with indicated CRISPitope-engineered melanoma cells (C) (n ≥ 3; mean ± SD) and representative flow cytometric plots (D). Statistics: ∗∗p &lt; 0.01, n.s. not significant; Mann-Whitney test and unpaired two-sided t test. The data shown in this figure was adapted from <a href="#bib2">Effern et al. (2020)</a>.</p>
</figcaption>
</figure>
<p>Once CRISPitope-engineered melanoma cells have been successfully validated <i>in vitro</i>, they can be used to inoculate mice. CRISPitope-engineered tumors can then be treated with ACT immunotherapy using CD8<sup>+</sup> TCRtg T cells. The description of these experiments is however beyond the scope of this protocol. In brief, as described in the publication by <a href="#bib2">Effern et al., 2020</a>, wild type C57BL/6 mice were inoculated subcutaneously with HC.PmelKO.CDK4<sup>mut</sup>-mNeon and HC.PmelKO.CDK4<sup>mut</sup>-NF-hgp100 melanoma cells. We started with our experimental ACT immunotherapy treatment protocol once the tumors reached a diameter of 3–5 mm. In brief, mice were injected with a single dose of the lymphodepleting chemotherapeutic drug cyclophosphamide one day prior to the transfer of the Pmel-1 TCRtg CD8<sup>+</sup> T cells. The following day, the mice were intravenously injected with naïve Pmel-1 TCRtg CD8<sup>+</sup> T cells and additionally with an adenovirus coding for hgp100 to activate the T cells <i>in vivo.</i> The mice were treated with CpG/PolyI:C on day 3, 6 and 9 to stimulate the innate immune system. We observed a response to ACT immunotherapy in the majority of the treated mice. In approximately half of the mice, we observed complete regression of the tumors. In the remaining mice, tumors escaped or recurred from ACT in a period over several months. The key finding was that the resistance mechanism of the recurrent melanomas was highly dependent on the GPOI targeted by ACT. We observed antigen loss, epigenetic silencing, melanoma dedifferentiation and increased expression of T cell inhibitory receptors and ligands. The strength of the CRISPitope technique is that it allows researchers to model resistance mechanisms to different ACT target antigens in a highly standardized and comparable manner as the same TCR-transgenic T cells be used to target different GPOIs. This allows us to analyze resistance mechanisms independent of antigen affinity and TCR specificity and rather focus on the nature and regulation of the targeted GPOI. antigen Please refer to the publication by Effern and colleagues for further details (<a href="#bib2">Effern et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol and the corresponding publication (<a href="#bib2">Effern et al., 2020</a>) describe the generation of melanoma cells that express model CD8<sup>+</sup> T cell epitopes under the control of promoters that produce relevant melanoma antigens. This technique provides a platform to mimic cancer immunotherapy directed against different targets by using the same TCRtg T cell species.</p>
<p>In the publication by Effern et al., we show a variety of different cellular targets that have been successfully modified using the CRISPitope approach. We targeted various melanosomal (<i>Tyrp1</i>, <i>Rab38</i>, <i>Dct</i>, <i>Gpnmb</i>), nuclear (<i>Cdk4</i>, <i>Sox10</i>) and housekeeping (<i>Actb</i>, <i>Actg1</i>) GPOIs with different universal donor plasmids. However, some key aspects should be considered when choosing your GPOI. The CRISPitope approach generates a C-terminal fusion protein of your GPOI with a universal donor plasmid. Tagging of the C-terminus of your GPOI might interfere with its function. Furthermore, some amino acids at the C-terminus can be lost due to the tagging approach which could interfere with the function of your GPOI. For some GPOIs, a suitable Protospacer Adjacent Motif (PAM) might not be available in the C-terminal region. In this case, one could consider using an alternative Cas endonuclease with a different PAM specificity. The PAM sequence serves as a binding signal for the Cas endonuclease, but the exact sequence depends on the Cas protein used. The described CRISPitope approach uses the popular <i>Streptococcus pyogenes</i> Cas9 (SpCas) that recognizes the PAM 3′NGG. However, there are variants of Cas9 that were modified or that originate from different bacteria species, recognizing different PAMs. These might therefore allow targeting of GPOI that do not have classical PAM towards the 3′ end of the protein coding region.</p>
<p>When choosing your GPOI, you should also consider its subcellular localization (nuclear, cytoplasmic, membrane, etc.), expression levels (high, intermediate, low), post-transcriptional/post-translational processing and expression pattern (housekeeping, inducible, etc.). The nature of the targeted proteins and its intracellular localization is important and can affect cell mediated tumor immunity as demonstrated by <a href="#bib4">Zeelenberg et al. (2011)</a>. In this study, T cell responses to antigens located in different subcellular localizations were compared. Tumor-secreted, vesicle-bound antigens were observed to elicit a stronger CD8<sup>+</sup> T cell response compared to secreted soluble proteins or non-secreted cell-associated proteins. Indeed, CRISPitope provides a standardized experimental platform to compare the impact of antigens with different subcellular localization. Expression levels and expression patterns can also impact the suitability of a GPOI. GPOIs that are expressed at low levels are likely to result in low expression of the fluorescent protein tag, the CD8<sup>+</sup> T cell model epitope and the antibiotic resistance cassette from the integrated universal donor plasmid. Consequently, further enrichment of successfully engineered cells via antibiotic selection or cell sorting might not be possible. Additionally, low expression of the GPOI also results in low expression of the model epitope thus compromising recognition by T cells, leading to a failure to mount an immune response <i>in vivo</i> without additional activation of T cells. Furthermore, GPOIs that are only expressed upon stimulation are relatively difficult to target. If your protein of interest is not expressed in your cell line under homeostasis, you can treat your cells with a low dose of a drug, cytokine, etc. that induces the expression of your protein. In this case, the treatment of cells to induce protein expression prior to transfection is ideal. At the latest, this should be performed before you start the antibiotic selection. The same holds true for the enrichment of engineered cells by cell sorting and for the T cell activation assay. CRISPitope-engineered cell lines with a defect in antigen-processing or -presentation will not be recognized.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>During cloning of your sgRNA into the BbsI-digested pX330 plasmid, you do not observe any colony growth or you observe a lot of colony growth on the negative control plate (step 3c).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This indicates that the pX330 plasmid was not fully digested. Repeat the digestion and incubate at 37°C for &gt; 8 h.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>You do not see mCherry positive cells under the microscope 24–48 h after transfection (step 9).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In the event that mCherry positive cells cannot be observed using a microscope, flow cytometry analysis can be performed to detect weak mCherry signal from the frame selector. Few mCherry positive cells may also be observed if transfection efficiency is not high in your cell line of choice.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Transfection efficiency is low (step 5–10).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>In case of poor transfection efficiency, consider adjusting the transfection protocol. For the majority of cell lines tested in our laboratory a ratio 3:1 FuGENE®HD transfection reagent:DNA (0.3 μL reagent:100 ng DNA in a 96-well) works best. Nevertheless, it can differ for other cell lines. If adjusting the transfection protocol does not work, consider using a different transfection reagent. Although Fugene® HD transfection reagent works well even for hard to transfect cell lines, it might not be the reagent of choice when using a different cell line. Alternative transfection reagents that can be used are Lipofectamine TM 2000 or 3000 (ThermoFisher Scientific). Depending on the cell line, transfection efficiency can be enhanced by adjusting the culturing conditions of the cell line of choice. For some cell lines 60–70% confluency might not be ideal.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>CRISPitope-engineered cell lines that do not survive antibiotic selection (step 11).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>There are two possible solutions for this problem.</p>
<p>Solution 1: An antibiotic concentration that is too high will result in the death of cells including those that express the antibiotic resistance cassette. Titration of the antibiotic used for selection is essential. Untransfected cells treated with puromycin should die within 3–4 days and untransfected cells treated with blasticidin should die within 5–7 days. If cell death is observed earlier than these time frames, the concentration of antibiotics may be too high.</p>
<p>Solution 2: If your tagged GPOI is not expressed at the protein level, there will be no protein expression of the antibiotic resistance cassette. Make sure that your protein is expressed. If your protein of interest is not expressed in your cell line under homeostasis, you can treat your cells with a low dose of a drug, cytokine, etc. that induces expression of your protein. It might be helpful to start inducing the expression of your protein of interest prior to transfection, but at the latest before you start the antibiotic selection. The same holds true for cell sorting experiments. If your protein of interest is not expressed, you will not observe expression of the fluorescent protein. It is possible to start antibiotic selection later than 48 h after transfection. This way protein encoding the antibiotic resistance cassette can accumulate in the cells.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>CRISPitope-engineered cell lines cannot be sorted as they do not express the fluorescent protein (step 12).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If your tagged GPOI is not expressed at the protein level, there will be no protein expression of the fluorescent protein. Make sure that your GPOI is expressed. If your protein of interest is not expressed in your cell line under homeostasis, you can treat your cells with a low dose of a drug, cytokine, etc. that induces expression of your protein. It might be helpful to start inducing the expression of your protein of interest prior to transfection, but at the latest before you start the antibiotic selection. The same holds true for cell sorting experiments. If your protein of interest is not expressed, you will not observe expression of the fluorescent protein.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>CRISPitope-engineered cell lines do not induce T cell activation (step 19).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>Firstly, ensure that IFN-γ treatment has induced MHC-Class I expression on the surface of the melanoma cells. If MHC Class I is not expressed, the epitope cannot be presented to the T cells and therefore cannot activate the T cells. Secondly, ensure that your tagged gene is expressed at the protein level. If your protein is not sufficiently expressed under homeostasis, T cells likely cannot be activated. In case that stimuli (e.g., cytokines, growth factors, drug or cellular stress) are known to induce expression of your GPOI, you may consider to stimulate your cells also with the respective stimulus. As a specific example for melanoma cells, there are certain GPOIs expressed in dedifferentiated melanoma cells that are known to be induced by pro-inflammatory cytokines such as TNF-α. Thus, co-stimulation with TNF-α to induce GPOI expression and IFN-γ to induce antigen processing and presentation may be necessary to achieve robust T cell activation.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Bald, T., Quast, T., Landsberg, J., Rogava, M., Glodde, N., Lopez-Ramos, D., Kohlmeyer, J., Riesenberg, S., van den Boorn-Konijnenberg, D., Hömig-Hölzel, C., et al. (2014). Ultraviolet-radiation-induced inflammation promotes angiotropism and metastasis in melanoma. Nature <i>507</i>, 109-113. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00744-9/sref1">View at publisher</a></p>
<p id="bib2">Effern, M., Glodde, N., Braun, M., Liebing, J., Boll, H.N., Yong, M., Bawden, E., Hinze, D., van den Boorn-Konijnenberg, D., Daoud, M., et al. (2020). Adoptive T cell therapy targeting different gene products reveals diverse and context-dependent immune evasion in melanoma. Immunity <i>53</i>, 564-580.e9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00744-9/sref2">View at publisher</a></p>
<p id="bib3">Schmid-Burgk, J.L., Höning, K., Ebert, T.S., and Hornung, V. (2016). CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism. Nat. Commun. <i>7</i>, 12338. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00744-9/sref3">View at publisher</a></p>
<p id="bib4">Zeelenberg, I.S., van Maren, W.W.C., Boissonnas, A., Van Hout-Kuijer, M.A., Den Brok, M.H.M.G.M., Wagenaars, J.A.L., van der Schaaf, A., Jansen, E.J.R., Amigorena, S., Théry, C., et al. (2011). Antigen localization controls T cell-mediated tumor immunity. J. Immunol. <i>187</i>, 1281-1288. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00744-9/sref4">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact Michael Hölzel (<a href="mailto:michael.hoelzel@ukbonn.de">michael.hoelzel@ukbonn.de</a>).</p>
<h4>Materials availability</h4>
<p>The universal donor plasmids generated during this study (<a href="#bib2">Effern et al., 2020</a>) have been deposited to Addgene (<a href="#tbl3">Table 3</a>).</p>
<table id="tbl3" class="dense-table">
<caption>Table 3. Overview of the universal donor plasmids that have been generated during the this study (<a href="#bib2">Effern et al., 2020</a>) and have been deposited to Addgene</caption>
<thead>
<tr>
<th>Plasmid name</th>
<th>Fluorescent protein</th>
<th>Synthetic tag</th>
<th>Epitope</th>
<th>Antibiotic resistance</th>
<th>Addgene Cat# / RRID</th>
</tr>
</thead>
<tbody>
<tr>
<td>pCRISPaint-mNeon-PuroR</td>
<td>mNeon</td>
<td>-</td>
<td>-</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171800; RRID: Addgene_171800</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-hgp100-PuroR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>hgp100<sub>25–33</sub> (KVPRNQWL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171801; RRID: Addgene_171801</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-mgp100-PuroR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>mgp100<sub>25–33</sub> (EGSRNQWL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171802; RRID: Addgene_171802</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-gB-PuroR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>HSV-1 gB<sub>498–505</sub> (SSIEFARL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171803; RRID: Addgene_171803</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-Ova-PuroR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>Ova<sub>257–264</sub> (SIINFEKL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171804; RRID: Addgene_171804</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-BlastR</td>
<td>mNeon</td>
<td>-</td>
<td>-</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171805; RRID: Addgene_171805</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-hgp100-BlastR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>hgp100<sub>25–33</sub> (KVPRNQWL)</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171806; RRID: Addgene_171806</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-gB-BlastR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>HSV-1 gB<sub>498–505</sub> (SSIEFARL)</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171807; RRID: Addgene_171807</td>
</tr>
<tr>
<td>pCRISPaint-mNeon-F-Ova-BlastR</td>
<td>mNeon</td>
<td>FLAG</td>
<td>Ova<sub>257–264</sub> (SIINFEKL)</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171808; RRID: Addgene_171808</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-PuroR</td>
<td>mScarlet</td>
<td>-</td>
<td>-</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171809; RRID: Addgene_171809</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-hgp100-PuroR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>hgp100<sub>25–33</sub> (KVPRNQWL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171810; RRID: Addgene_171810</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-mgp100-PuroR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>mgp100<sub>25–33</sub> (EGSRNQWL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171811; RRID: Addgene_171811</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-gB-PuroR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>HSV-1 gB<sub>498–505</sub> (SSIEFARL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171812; RRID: Addgene_171812</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-Ova-PuroR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>Ova<sub>257–264</sub> (SIINFEKL)</td>
<td>Puromycin [M1G]</td>
<td>Cat-Nr: 171813; RRID: Addgene_171813</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-BlastR</td>
<td>mScarlet</td>
<td>-</td>
<td>-</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171814; RRID: Addgene_171814</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-hgp100-BlastR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>hgp100 (KVPRNQWL; 25–33)</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171815; RRID: Addgene_171815</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-Ova-BlastR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>Ova (SIINFEKL; 257–264)</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171816; RRID: Addgene_171816</td>
</tr>
<tr>
<td>pCRISPaint-mScarlet-F-gB-BlastR</td>
<td>mScarlet</td>
<td>FLAG</td>
<td>HSV-1 gB (SSIEFARL; 498–505)</td>
<td>Blasticidin [M1G]</td>
<td>Cat-Nr: 171817; RRID: Addgene_171817</td>
</tr>
</tbody>
</table>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze datasets or code.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1288-Mmc1.xlsx">Table S1. Overview of fully designed sgRNAs for human and murine GPOIs, related to Before you begin – step 6</a></p>
<h3>Acknowledgments</h3>
<p>We thank P. Wurst and An. Dolf for help with flow cytometry. We thank the UKB core facility flow cytometry. This work was supported by the <a href="https://doi.org/10.13039/501100005972">Deutsche Krebshilfe</a> (German Cancer Aid) through a junior research group grant within the Mildred Scheel School of Oncology (MSSO) Cologne-Bonn (70113307) to N.G., the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> (DFG, German Research Foundation) within GRK 2168 to M.H., scholarships by the <a href="https://doi.org/10.13039/501100001782">University of Melbourne</a> to M.E. and E.B., a scholarship to M.E. within GRK 2168, and the DFG under Germany’s Excellence Strategy–EXC2151–390873048. T.G. and M.H. are supervisors in the GRK 2168 international graduate school, Bonn-Melbourne. M.H. is a member of the DFG Excellence Cluster ImmunoSensation<sup>2</sup> (EXC 2151). We thank D. Corvino, S. Ng, and M. Yong from the Institute of Experimental Oncology for critical reading. The graphical abstract, parts of <a href="#fig2">Figures 2</a> and <a href="#fig5">5</a> were generated using <a href="http://biorender.com">biorender.com</a>. <a href="#fig3">Figure 3</a> was generated using SnapGene software (from insightful Science; available at <a href="http://snapgene.com">snapgene.com</a>). <a href="#fig1">Figures 1</a>, <a href="#fig4">4</a>A, 4C, and <a href="#fig6">6</a> are reprinted from (<a href="#bib2">Effern et al., 2020</a>), with permission from Elsevier.</p>
<h3>Author contributions</h3>
<p>Conceptualization, M.E., N.G., T.T., T.G., and M.H.; Methodology, M.E., N.G., and E.B.; Validation, M.E., N.G., E.B.; Formal Analysis, M.E. and N.G.; Investigation, M.E., N.G., and E.B.; Resources, J.L., D.H., T.G., and M.H.; Data Curation, M.E., N.G., T.G., and M.H.; Writing – Original Draft, M.E. and M.H.; Writing – Review &amp; Editing, M.E., N.G., E.B., D.H., T.G., and M.H.; Supervision, N.G., T.G., and M.H.; Project Administration, T.G. and M.H.; Funding Acquisition, N.G., T.G., and M.H.</p>
<h3>Declaration of interests</h3>
<p>N.G. declares receipt of honoraria for the InCa advisory board of Novartis unrelated to the submitted work. M.H. declares receipt of honoraria for the scientific advisory board of Novartis and honoraria from Bristol-Myers Squibb unrelated to the submitted work. M.H. declares research support by Noxxon Pharma AG unrelated to this work.</p>
</section>
</article>