<article>
<section>
<h1 id="header">Orthotopic T-cell receptor replacement in primary human T cells using CRISPR-Cas9-mediated homology-directed repair</h1>
<p><time datetime="2021-12-24">Published: December 24, 2021</time></p>
<p>Carolin Moosmann,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Thomas R. Müller,<sup><a href="#aff2">2</a></sup> Dirk H. Busch,<sup><a href="#aff2">2</a></sup> and Kilian Schober<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Mikrobiologisches Institut–Klinische Mikrobiologie, Immunologie und Hygiene, Universitätsklinikum Erlangen, Friedrich-Alexander-Universität Erlangen-Nürnberg, Erlangen, Germany</p>
<p id="aff2"><sup>2</sup>Institute for Medical Microbiology, Immunology and Hygiene, Technische Universität München, Munich, Germany</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:carolin.moosmann@uk-erlangen.de">carolin.moosmann@uk-erlangen.de</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:kilian.schober@uk-erlangen.de">kilian.schober@uk-erlangen.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101031">10.1016/j.xpro.2021.101031</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Adoptive T cell therapy using T-cell receptor (TCR)-engineered T cells allows to redirect T cell specificity and to target any antigen of interest. Here, we apply advanced genetic engineering using clustered regularly interspaced short palindromic repeats (CRISPR)-associated protein 9 (Cas9) for simultaneous editing of TCR α- and β-chains in primary human T cells. Together with non-virally delivered template DNA, this CRISPR-Cas9-system allows for elimination of the endogenous TCR and orthotopic placement of TCR α- and β-chains.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref8">Schober et al. (2019)</a> and <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref4">Müller et al. (2021)</a>.</p>








<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>OTR protocol generates advanced TCR-transgenic T cells within a week</li>
<li>Targeted CRISPR-Cas9-mediated large gene knock-in is completely non-viral</li>
<li>OTR T cells are closer to physiological T cells than conventionally engineered T cells</li>
<li>Protocol can also be adapted for other applications, e.g., gene tagging</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the application of the CRISPR-Cas9-system for orthotopic TCR replacement (OTR) in primary human T cells using ribonucleoproteins (RNPs) to mediate homology-directed repair (HDR). Linearized double-stranded DNA (dsDNA) serves as HDR template for the knock-in (KI) and is designed to encode the full αβ TCR. T cells are activated by surface-bound anti-CD3 and anti-CD28 antibodies for two days. RNPs create double-strand breaks in the endogenous human TCR α-chain constant (<i>hTRAC)</i> and human TCR β-chain constant (<i>hTRBC</i>) loci. Both RNPs comprise the respective guide RNA (gRNA) and the Cas9 protein. To produce these <i>hTRAC</i> and <i>hTRBC</i> RNPs, trans-activating crRNA (tracrRNA) is incubated with <i>hTRAC</i> or <i>hTRBC</i> CRISPR RNA (crRNA), respectively, to form gRNAs before electroporation enhancer and Cas9 protein are added. Along with the DNA template, both RNPs are electroporated into the activated T cells. The resulting OTR leads to changes at the TCR DNA, RNA and protein level (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic of orthotopic TCR replacement through CRISPR-Cas9-mediated gene editing</div>
<p>On the TCR gene level, <i>hTRAC</i> and <i>hTRBC</i> RNPs create double-strand breaks (illustrated by gray double line) leading to homology-directed repair (HDR)-mediated integration of the DNA template into to the <i>TRAC</i> exon and non-homologous end-joining (NHEJ) in the <i>TRBC</i> locus<i>.</i> On the transcript level, the knock-in (KI) of the HDR template results in the knock-out (KO) of endogenous <i>TRAC</i> transcription. For <i>TRBC</i>, NHEJ leads to a frameshift which introduces a stop codon and results in the KO of endogenous TCR β-chain. On the protein level, only truncated endogenous but full transgenic TCR α-chain and β-chains are expressed. On the T cell surface, only the transgenic αβ TCR is expressed.</p>
</figcaption>
</figure>
<p>For this ‘single KI’ strategy, the HDR template is inserted into the endogenous <i>TRAC</i> locus, with concomitant knock-out (KO) of endogenous <i>TRBC</i>. A ‘dual KI’ strategy with individual insertion of transgenic α-chain into endogenous <i>TRAC</i> and transgenic β-chain into endogenous <i>TRBC</i> is also possible, albeit with lower overall αβ TCR re-expression rate. In both cases, OTR generates engineered T cells expressing only the transgenic TCR on their surface.</p>
<h3 id="sec1.1">DNA template design</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 week</div>
<p>Directed by a specific <i>hTRAC</i> gRNA, Cas9 nuclease creates a double-strand break in the first <i>TRAC</i> exon. For recombinant <i>Streptococcus pyogenes</i> Cas9 nuclease (used in this protocol) the necessary protospacer adjacent motif (PAM) sequence is 5′-NGG-3′ (here the PAM sequence is 3′-GGC-5′ on the antisense strand, and the <i>hTRAC</i> gRNA is complementary to the sense strand). The nuclease cuts 3 bp downstream of the PAM sequence. After the double-strand break, HDR enables insertion of the linearized dsDNA template into the genomic DNA.</p>
<p>The structure of the HDR template is illustrated in <a href="#fig2">Figure 2</a>. The construct is flanked by left and right homology arms (LHA and RHA) required for HDR and comprises the full α- and β-chains of the TCR to be introduced, self-cleaving peptides (P2A and T2A) to ensure separation and polycistronic expression of both TCR-chains, and a poly-A tail (bGHpA). The left homology arm (LHA) consists of 396 bp including a 5′-region with a primer site and is located upstream of the <i>hTRAC</i> gRNA cut site. The LHA ends 2 bp upstream of the <i>hTRAC</i> gRNA cut site (at base 26 of <i>TRAC</i>) to be in-frame with the TCR J-segment after splicing. The LHA is followed by the self-cleaving peptide P2A which allows the separation of the following β-chain segments from the LHA. The β-chain consists of the human variable parts and the murine constant region. The variable part comprises the VDJ-region starting from the respective leader sequence of the V-gene until the end of the respective J-gene of the chosen TCR β-chain. Transgenic VDJ β-transcription is therefore initiated with endogenous VJ-transcription without the use of extrinsic promoters. Here, the <i>hTRBC</i> is exchanged for its murine counterpart which includes an additional cysteine bridge (<i>mTRBC</i>-Cys) (<a href="#bib2">Cohen et al., 2007</a>). This modification allows for improved surface expression of the TCR and facilitates identification of re-expressed transgenic TCR and clear distinction from endogenous human TCR (<a href="#bib2">Cohen et al., 2007</a>). In short-term <i>in vitro</i> assays, transgenic TCRs with a fully human TCR constant region did not show any functional differences to TCRs with a murine constant region after OTR (<a href="#bib8">Schober et al., 2019</a>). The β-chain is followed by another self-cleaving peptide (T2A) to ensure polycistronic expression of β- and α-chain. The subsequent α-chain is designed according to the same principles as the β-chain: The variable part (VJα) is followed by the murine <i>TRAC</i> (<i>mTRAC</i>-Cys). The stop codon (TGA) concludes the transgenic TCR (α-chain) sequence. The following bovine growth hormone polyA signal (bGHpA) is not strictly necessary but included in this construct to stabilize the mRNA transcript. When human TCR constant regions are included in the construct design, the transgenic <i>hTRAC</i> region only encompasses the part that lies upstream of the editing site (in the first <i>TRAC</i> exon) and then seamlessly continues into the RHA. This approach additionally requires the introduction of a silent mutation in the PAM sequence to avoid editing once the transgenic TCR is integrated into the endogenous <i>TRAC</i> locus. The RHA concludes the HDR template and consists of 330 bp including a 3′ region with a primer site and is located downstream of the cut site.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. DNA template design for TCR α- and β-chain integration into human <i>TRAC</i> locus via homology-directed repair</div>
<p>LHA, left homology arm; RHA, right homology arm; VDJβ, variable parts of β-chain; <i>mTRBC-Cys</i>, murine TCRβ constant region with additional cysteine bridge; VJα, variable parts of α-chain; <i>mTRAC-Cys</i>, murine TCRα constant region with additional cysteine bridge; bGHpA, bovine growth hormone poly-A tail; T2A and P2A, self-cleaving peptide inserts, <i>TRAC</i> exon 1 is illustrated in orange and includes the PAM sequence (yellow). For sequences of these segments see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Mmc1.pdf">Table S1</a>.</p>
</figcaption>
</figure>
<ol>
<li>The HDR template is assembled <i>in silico</i> by combining the segments as described above. All sequences of these building blocks (updated from <a href="#bib8">Schober et al., 2019</a>) can be found in the supplemental information (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Mmc1.pdf">Table S1</a>).</li>
<li>The DNA construct can then be ordered as a sequence-verified plasmid gene via commercial providers.</li>
<li>After delivery, lyophilized plasmids are reconstituted to 60 ng/μL for HDR template amplification by PCR (and potentially vector amplification by Midiprep to generate back-up stocks).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The addition of Cas9 Target Sequences (CTS) – 20 bp gRNA recognition sequences – at the 5′-end of the DNA template has been described to improve TCR re-expression rates as these facilitate the delivery of the HDR template to the correct position in the genome (<a href="#bib5">Nguyen et al., 2020</a>; <a href="#bib9">Shy et al., 2021</a>). A single CTS at the 5′-end of the DNA template was shown to be sufficient for this effect. Constructs with 4–8 mismatched base pairs at the 5′-end of this CTS (allowing the Cas9 nuclease to bind, but not to cut) showed the greatest improvement in re-expression rates. Additionally, increased re-expression rates and cell viability have been demonstrated to result from the use of hybrid DNA templates. These hybrid DNA templates comprise single-stranded DNA (ssDNA) template including a CTS at the 5′-end with an annealing oligonucleotide covering the gRNA recognition site, the PAM sequence, and approximately 20 bp of the downstream homology arm of the HDR template) (<a href="#bib9">Shy et al., 2021</a>).</div>
<h3 id="sec1.2">Reconstitute reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<p>In preparation for the experiment, some reagents require reconstitution at their respective concentrations. Prior to electroporation, tracrRNA, crRNAs and electroporation enhancer are reconstituted. Additionally, interleukins (IL-2, IL-7 and IL-15) used for T-cell activation are also reconstituted and aliquoted.</p>
<ol start="4">
<li>Reconstitute tracrRNA, <i>hTRAC</i> crRNA and <i>hTRBC</i> crRNA to 80 μM in Nuclease-Free Duplex Buffer (IDT). Aliquot and store at −20°C.</li>
<li>Reconstitute electroporation enhancer to 400 μM in Nuclease-Free Duplex Buffer (IDT). Aliquot and store at −20°C.</li>
<li>Reconstitute IL-2 to 10 μg/mL (100,000 U/mL) and IL-7 and IL-15 to 25 μg/mL by following manufacturer's instructions. Aliquot and store at −20°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For the IL-2 used here, the International Unit value was 2.18 IU/ng. 10 μg/μL is therefore equivalent to 21,800 IU/mL.</div>
<h3 id="sec1.3">Double-stranded DNA production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="7">
<li>Perform PCR for amplification of linearized double-stranded HDR template.
<ol type="a">
<li>Prepare 8 × 100 μL PCR master mix (see <a href="#materials-and-equipment">materials and equipment</a>) for each HDR construct (without plasmid DNA).</li>
<li>Distribute 99 μL master mix into PCR-tubes.</li>
<li>Add 1 μL of plasmid DNA (pre-diluted to 60 ng/μL) per tube.
<div class="optional">
<span class="optional-title">Optional:</span> Include negative control by adding nuclease-free PCR grade water instead of template DNA to the sample.</div>
</li>
<li>Mix by vortexing.</li>
<li>Spin down tubes to ensure the entire reaction mixture is at the bottom of the tube.</li>
<li>Ensure all the lids are properly sealed and place PCR tubes into thermocycler.</li>
<li>Run PCR with the following conditions:
<table id="undtbl1">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95 °C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td rowspan="3">34 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>62 °C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72 °C</td>
<td>3 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72 °C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4 °C</td>
<td colspan="2">Unlimited</td>
</tr>
</tbody>
</table>
<div class="optional">
<span class="optional-title">Optional:</span> Depending on the thermocycler, PCR cycling conditions (e.g., temperatures) may have to be adjusted.</div>
</li>
<li>Keep samples at 4°C until further processing (running agarose gel or purification).</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Run agarose gel to ensure successful amplification of the HDR template (see <a href="#sec6.1">problem 1</a>).
<ol type="a">
<li>Prepare 1% agarose gel including peqGREEN (1:20,000) as DNA/RNA dye.
<div class="optional">
<span class="optional-title">Optional:</span> Other DNA/RNA dyes can be also used.</div>
</li>
<li>Prepare marker and samples by mixing 6 × TriTrack DNA loading Dye with GeneRuler 1 kb DNA Ladder or PCR product, respectively.</li>
<li>Mix samples by pipetting and load them onto the agarose gel.</li>
<li>Run gel at 120 V, 400 mA, 100 W for 20–45 min.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> If the gel cannot be imaged immediately after electrophoresis is completed, reduce voltage (to 10 V) to avoid diffusion of the bands.</div>
</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Image gel under UV light to check if the PCR was successful (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Exemplary 1% agarose gel showing successful amplification of HDR template DNA</div>
<p>M: GeneRuler 1 kb DNA Ladder; Full αβ TCR construct (2.8 kb); TCR α-chain (1.8 kb); TCR β-chain (1.9 kb) and W: water control (negative control).</p>
</figcaption>
</figure>
</li>
<li>Replicates showing only one clear band of the expected size can be pooled for further processing (purification).</li>
<li>After amplification, the pooled remaining double-stranded HDR template of the PCR reaction is isolated using the MinElute PCR Purification Kit.</li>
<li>Follow manufacturer's protocol: <a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/dna-clean-up/minelute-pcr-purification-kit/">https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/dna-clean-up/minelute-pcr-purification-kit/</a> (as of Sep. 22, 2021).</li>
<li>Elute in 10 μL per column.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> It is possible to load two PCR samples (of the same construct, approx. 180 μL) onto the same column: Mix buffer PB and PCR reaction mix according to manufacturer's instruction, load mixture onto column and spin down. After discarding flow-through, load the remaining mixture before continuing with the next step of the protocol. Since still only 10 μL are used for elution, this increases the concentration of purified DNA per eluate.</div>
<ol start="14">
<li>Measure concentration of purified PCR product using a NanoDrop 1000 spectrophotometer blanked with elution buffer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Other machines for determining nucleic acid concentration can also be used.</div>
<div class="note">
<span class="note-title">Note:</span> Concentration should range from 0.5–1.0 μg/μL. Lower concentrations will dilute the DNA/RNP-mix as higher volumes of DNA template are required per electroporation. This might reduce re-expression rates. If possible, it is recommended to adjust DNA concentration to 1.0 μg/μL.</div>
<ol start="15">
<li>Store purified HDR template DNA at −20°C.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Armenian hamster anti-mouse TCR-β chain, APC/Fire750-conjugated</td>
<td>BioLegend</td>
<td>H57-597; Cat#109246; RRID: AB_2629697</td>
</tr>
<tr>
<td>Mouse anti-human CD45, ECD-conjugated</td>
<td>Beckman Coulter</td>
<td>J.33; Cat#A07784;</td>
</tr>
<tr>
<td>Mouse monoclonal anti-human CD8a, APC-eFluor® 780-conjugated</td>
<td>Thermo Fisher Scientific</td>
<td>OKT8; Cat#47-0086-42; RRID: AB_2573945</td>
</tr>
<tr>
<td>Mouse monoclonal anti-human CD8a, PE-conjugated</td>
<td>Thermo Fisher Scientific</td>
<td>OKT8; Cat#12-0086-42; RRID: AB_10732344</td>
</tr>
<tr>
<td>Mouse monoclonal anti-human TCR α/β, FITC-conjugated</td>
<td>BioLegend</td>
<td>IP26; Cat#306706; RRID: AB_314644</td>
</tr>
<tr>
<td>Purified monoclonal mouse anti-human CD28</td>
<td>BioLegend</td>
<td>CD28.2; Cat#302902; RRID: AB_314304</td>
</tr>
<tr>
<td>Purified monoclonal mouse anti-human CD3</td>
<td>BioLegend</td>
<td>OKT3; Cat#317302; RRID: AB_571927</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human blood/PMBCs</td>
<td>Healthy volunteers</td>
<td>Transfusionsmedizin, UK Erlangen</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Albumin from bovine serum (BSA)</td>
<td>VWR Life Science</td>
<td>Cat#A3294 CAS: 9048-46-8</td>
</tr>
<tr>
<td>Alt-R® Cas9 Electroporation Enhancer</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1075916</td>
</tr>
<tr>
<td>Alt-R® S.p. Cas9 Nuclease V3</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1081059</td>
</tr>
<tr>
<td>ß-Mercaptoethanol (50 mM)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#31350010</td>
</tr>
<tr>
<td>DNA/RNA dye, peqGREEN</td>
<td>VWR Peqlab</td>
<td>Cat#732-3196</td>
</tr>
<tr>
<td>dNTP Mix (10 mM each)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#R01922</td>
</tr>
<tr>
<td>Dulbecco’s Phosphate Buffered Saline (PBS)</td>
<td>Sigma-Aldrich</td>
<td>Cat#D8537</td>
</tr>
<tr>
<td>GeneRuler 1 kb DNA Ladder</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#SM0311</td>
</tr>
<tr>
<td>Gentamicin (50 mg/mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15750060</td>
</tr>
<tr>
<td>Heat-Inactivated Fetal Bovine Serum (South America)</td>
<td>Anprotec</td>
<td>Cat#AC-SM-0027</td>
</tr>
<tr>
<td>HEPES</td>
<td>Carl Roth</td>
<td>Cat#HN77.3; CAS: 7365-45-9</td>
</tr>
<tr>
<td>Herculase II Fusion DNA Polymer</td>
<td>Agilent Technologies</td>
<td>Cat#600677</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>Sigma-Aldrich</td>
<td>Cat#G8540; CAS: 56-85-9</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122;</td>
</tr>
<tr>
<td>Poly-L-glutamic acid sodium salt, mol wt 15,000–50,000 (PGA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#P4761-100mg; CAS: 26247-79-0</td>
</tr>
<tr>
<td>Propidium Iodide</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#P1304MP; CAS: 25535-16-4</td>
</tr>
<tr>
<td>Recombinant Human IL-15</td>
<td>PeproTech</td>
<td>Cat#200-15; Accession Number: P40933</td>
</tr>
<tr>
<td>Recombinant Human IL-2</td>
<td>PeproTech</td>
<td>Cat#200-02; Accession Number: P60568</td>
</tr>
<tr>
<td>Recombinant Human IL-7</td>
<td>PeproTech</td>
<td>Cat#200-07; Accession Number: P13232</td>
</tr>
<tr>
<td>RPMI 1640 Medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#21875034</td>
</tr>
<tr>
<td>RT-PCR Grade Water</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM9935</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>MinElute PCR Purification Kit</td>
<td>Qiagen</td>
<td>Cat#28004</td>
</tr>
<tr>
<td>P3 Primary Cell 4D-Nucleofector<sup>TM</sup> X Kit S</td>
<td>Lonza</td>
<td>Cat#V4XP-3032</td>
</tr>
<tr>
<td>P3 Primary Cell 96-well Nucleofector<sup>TM</sup> Kit (960 RCT)</td>
<td>Lonza</td>
<td>Cat#V4SP-3960</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Alt-R® CRISPR-Cas9 tracrRNA</td>
<td>Integrated DNA Technologies</td>
<td>Cat#1072534</td>
</tr>
<tr>
<td>
<i>hTRAC</i> crRNA sequence (on antisense strand): 5′-AGAGTCTCTCAGCTGGTACA-3′</td>
<td><a href="#bib6">Ren et al. (2017)</a></td>
<td>n/a</td>
</tr>
<tr>
<td>
<i>hTRAC</i> HDR genomic forward primer:5′-CTGCCTTTACTCTGCCAGAG-3′</td>
<td><a href="#bib4">Müller et al. (2021)</a></td>
<td>n/a</td>
</tr>
<tr>
<td>
<i>hTRAC</i> HDR genomic reverse primer:5′- CATCATTGACCAGAGCTCTG-3′</td>
<td><a href="#bib4">Müller et al. (2021)</a></td>
<td>n/a</td>
</tr>
<tr>
<td>
<i>hTRBC</i> crRNA sequence:5′-GGAGAATGACGAGTGGACCC-3′</td>
<td><a href="#bib8">Schober et al. (2019)</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>HDR DNA template sequence</td>
<td><a href="#bib8">Schober et al. (2019)</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo, Version 10.7.2</td>
<td>Becton Dickinson &amp; Company (BD)</td>
<td>
<a href="https://www.flowjo.com/solutions/flowjo">https://www.flowjo.com/solutions/flowjo</a>;RRID: SCR_008520</td>
</tr>
<tr>
<td>GraphPad Prism 8.3.0 (538)</td>
<td>GraphPad Software</td>
<td>
<a href="https://www.graphpad.com/">https://www.graphpad.com/</a>; RRID: SCR_002798</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>4D-Nucleofector<sup>TM</sup> Core Unit</td>
<td>Lonza</td>
<td>Cat#AAF-1002B</td>
</tr>
<tr>
<td>4D-Nucleofector<sup>TM</sup> X Unit</td>
<td>Lonza</td>
<td>Cat#AAF-1002X</td>
</tr>
<tr>
<td>BD LSRFortessa™ Cell Analyzer</td>
<td>BD Biosciences</td>
<td>n/a</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl2">
<caption>PCR master mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (for 100 μL reaction)</th>
</tr>
</thead>
<tbody>
<tr>
<td>PCR grade water</td>
<td>n/a</td>
<td>65.0 μL</td>
</tr>
<tr>
<td>5× Herculase II Reaction Buffer</td>
<td>1×</td>
<td>20 μL</td>
</tr>
<tr>
<td>
<i>hTRAC</i> HDR genomic forward primer (10 mM)</td>
<td>0.4 mM</td>
<td>4.0 μL</td>
</tr>
<tr>
<td>
<i>hTRAC</i> HDR genomic reverse primer (10 mM)</td>
<td>0.4 mM</td>
<td>4.0 μL</td>
</tr>
<tr>
<td>dNTPs (10 mM)</td>
<td>0.5 mM</td>
<td>5.0 μL</td>
</tr>
<tr>
<td>Herculase II Fusion DNA Polymerase</td>
<td>1.0%</td>
<td>1.0 μL</td>
</tr>
<tr>
<td>DNA (prediluted to 60 ng/μL)</td>
<td>600 ng/mL</td>
<td>1.0 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>100 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">PCR master mix is freshly prepared on ice for direct use.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In addition to Herculase II Fusion DNA Polymerase, other polymerases are commercially available (e.g. Ampli Taq 360). Adjustments to the PCR conditions might be necessary.</div>
<table id="undtbl3">
<caption>SC<sup>+</sup> supplement</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>ß-Mercaptoethanol (50 mM)</td>
<td>1 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>Gentamicin (50 mg/mL)</td>
<td>1 mg/mL</td>
<td>10 mL</td>
</tr>
<tr>
<td>HEPES</td>
<td>23.83 g/L</td>
<td>11.9 g</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>4.0 g/L</td>
<td>2.0 g</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>2,000 U/mL</td>
<td>100 mL</td>
</tr>
<tr>
<td>RPMI 1640 Medium</td>
<td>n/a</td>
<td>380 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20 °C for several months.</p>
<table id="undtbl4">
<caption>SC<sup>-</sup> supplement</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>ß-Mercaptoethanol (50 mM)</td>
<td>1 mM</td>
<td>2.4 mL</td>
</tr>
<tr>
<td>HEPES</td>
<td>23.83 g/L</td>
<td>2.86 mg</td>
</tr>
<tr>
<td>L-Glutamine</td>
<td>4.0 g/L</td>
<td>0.48 g</td>
</tr>
<tr>
<td>RPMI 1640 Medium</td>
<td>n/a</td>
<td>116.8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>120 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20 °C for several months.</p>
<table id="undtbl5">
<caption>Antibiotic mix (as supplement for SC<sup>-</sup> medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gentamicin (50 mg/mL)</td>
<td>1 mg/mL</td>
<td>0.4 mL</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>2,000 U/mL</td>
<td>4.0 mL</td>
</tr>
<tr>
<td>RPMI 1640 Medium</td>
<td>n/a</td>
<td>15.6 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20 °C for several months.</p>
<table id="undtbl6">
<caption>Complete RPMI medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI 1640 Medium</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>Heat-Inactivated Fetal Bovine Serum (FBS)</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>SC<sup>+</sup> supplement</td>
<td>5%</td>
<td>25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>575 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4 °C for several weeks.</p>
<table id="undtbl7">
<caption>FACS buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS buffer, pH 7.4</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>0.5%</td>
<td>2.5 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4 °C for 2–4 weeks.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Thawing and resting PBMCs overnight</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>When working with frozen PBMCs, these steps prepare the cells for subsequent stimulation. The overnight rest is not necessary when working with freshly isolated PBMCs.</p>
<ol>
<li>Warm complete RPMI medium in water bath set at 37°C.</li>
<li>Thaw PBMCs as quickly as possible to reduce the time that the cells spend in freezing medium (containing DMSO).
<ol type="a">
<li>Hold cryovials in water bath until cells start thawing. Then pipette thawing cell suspension up and down to speed up the thawing process.</li>
<li>Transfer thawed cells into 50 mL tube with pre-warmed complete RPMI medium.</li>
</ol>
</li>
<li>Wash cells twice by centrifuging (480 × <i>g</i>, 7 min, 20°C–22°C), discarding supernatant and resuspending in pre-warmed complete RPMI medium.</li>
<li>Count cells and seed at 2 × 10<sup>6</sup> cells/mL in complete RPMI medium supplemented with 50 U/mL IL-2.</li>
<li>Incubate cells 18–24 h at 37°C, 5% CO<sub>2</sub>.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Instead of working with frozen PBMCs, PBMCs also can be freshly isolated (by density gradient centrifugation) from a healthy donor and directly activated (without overnight rest).</div>
<h3 id="sec3.2">Coating of plates with anti-CD3/CD28 antibodies</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h - 1 day</div>
<p>This step prepares the tissue-culture plates and/or flasks for activation of PBMCs by coating them with anti-CD3 and anti-CD28 antibodies. The required volume of coating solution depends on the cell numbers to be activated and therefore on the number of wells (more precisely, the area) to be coated. Here, we describe the process for the 24-well plate format. To adapt activating antibody-coating and T cell activation to different formats (e.g., 48-well plates of T25-flasks), calculate the area in relation to the surface of a 24-well plate well and adjust required volume of coating solution and cell suspension accordingly.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Dynabeads® Human T-Activator CD3/CD28 which are covalently coupled to anti-CD3 and anti-CD28 antibodies can be used for polyclonal T cell activation (<a href="#bib7">Roth et al., 2018</a>).</div>
<ol start="6">
<li>Dilute anti-CD3 and anti-CD28 antibodies to a final concentration of 1 μg/mL in sterile PBS buffer.</li>
<li>Coat 24-well plate with 500 μL/well.</li>
<li>Seal plates with Parafilm® to avoid evaporation and ensure sterile conditions.</li>
<li>Keep plates at 4°C 18–24 h.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Antibody coating can also be performed for 2 h at 37°C.</div>
<h3 id="sec3.3">T cell activation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>For activation of T cells, PBMCs are incubated on tissue-culture plates with surface-bound anti-CD3 and anti-CD28-antibodies. Interleukins (IL-2, IL-7 and IL-15) are added to the culture medium.</p>
<p>Per CRISPR sample, 1 × 10<sup>6</sup> cells are electroporated. In addition to the OTR samples, control samples (TCR KO and mock) are included in the experiment. TCR KO (KO of endogenous α-chain and β-chain) samples are required to evaluate the efficiency endogenous TCR KO and while mock samples (unedited cells) serve as negative control.</p>
<ol start="10">
<li>Count PBMCs.</li>
<li>Resuspend PBMCs at 1 × 10<sup>6</sup> cells/mL in complete RPMI medium supplemented with 300 U/mL IL-2, 5 ng/mL IL-7 and 5 ng/mL IL-15.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During the activation, usually up to half of the PBMCs do not survive and it is therefore important to account for this cell death at this activation step. When working with frozen PBMCs, activate at least 2.5 × 10<sup>6</sup> cells per CRISPR sample (2.5 × the amount of cells required for electroporation).</div>
<ol start="12">
<li>Wash 24-well plates twice with PBS without letting the wells dry out.</li>
<li>Add 1.5 mL of the PBMC suspension from step 11 per well to the coated 24-well plates.</li>
<li>Incubate at 37°C, 5% CO<sub>2</sub> for two days (48 h).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> To improve T cell viability after electroporation and TCR re-expression rates, the addition of DNA-sensing inhibitors has been reported to be beneficial (<a href="#bib3">Kath et al., 2021</a>). DNA-sensing inhibitors like RU.521 (small-molecule inhibitor of cyclic GMP-AMP synthase (cGAS)) and the synthetic oligonucleotide ODN A151 (TLR9 antagonist and inhibitor of other cytosolic DNA-sensors) can be added 6 h before the end of incubation to increase survival and TCR re-expression rates (<a href="#bib3">Kath et al., 2021</a>).</div>
<h3 id="sec3.4">Preparation of electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>T cell activation is stopped and cells are prepared for electroporation by transferring them into a 96-well plate (‘cells plate’).</p>
<p>Additionally, antibiotic-free SC<sup>-</sup> medium is prepared. The SC<sup>-</sup> medium comprises the same ingredients and supplements as complete RPMI medium except for antibiotics (penicillin-streptomycin, gentamicin). As each CRISPR sample is seeded at 1 × 10<sup>6</sup> cells/mL after electroporation, 24-well plates with 1 mL SC<sup>-</sup> medium per well are prepared and pre-warmed in the incubator.</p>
<div class="note">
<span class="note-title">Note:</span> The viability of KI cells is higher when the cells are cultivated in antibiotic-free SC<sup>-</sup> medium for 12–16 h after electroporation. After 12–16 h, cell culture is continued in complete (antibiotics-containing) RPMI medium (SC<sup>+</sup>).</div>
<ol start="15">
<li>Check for cell cluster formation under an inverse microscope.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cluster formation is crucial as this indicates the intended T cell activation, which is a prerequisite for successful genetic editing.</div>
<ol start="16">
<li>Recover the activated cells from the wells and pool them into a 50 mL tube.</li>
<li>Wash cells by centrifuging (480 × <i>g</i>, 7 min, 20°C–22°C), discarding the supernatant and resuspending in complete RPMI medium.</li>
<li>Count PBMCs.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cell numbers are usually lower than before activation as only T cells among PBMCs are being activated and some T cells may also undergo activation-induced cell death. Apoptotic bodies were removed during the washing steps and cell number calculations for the subsequent steps were based on the cell count of live cells. No further steps (e.g. FACS sorting) were performed to separate dead from live cells.</div>
<ol start="19">
<li>Centrifuge (480 × <i>g</i>, 7 min, 20°C–22°C), discard supernatant and resuspend at a density of 5 × 10<sup>6</sup> cells/mL in complete RPMI medium.</li>
<li>Seed 1 × 10<sup>6</sup> cells (200 μL of cell suspension) per CRISPR sample into a 96-well V-bottom ‘cells plate' well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this step, the plate layout is important. Make sure to choose a plate layout that is appropriate for the use of a multi-channel pipette, which allows a fast transfer to Nucleocuvettes<sup>TM</sup> (e.g., when working with 16-well Nucleocuvette<sup>TM</sup> Strips, prepare the cells in rows of 8).</div>
<ol start="21">
<li>Keep the 96-well plate at 37°C, 5% CO<sub>2</sub> until further use.</li>
<li>Cells of the mock samples can be resuspended to a density of 1 × 10<sup>6</sup> cells/mL in complete RMPI medium supplemented with 180 U/mL IL-2 and directly transferred into the 24-well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Mock samples are not being electroporated in this protocol as previous data have shown that electroporation itself (without the addition of DNA or RNPs) has no considerable effect on the endogenous TCR expression.</div>
<ol start="23">
<li>Per CRISPR sample, prepare 1 mL of SC<sup>-</sup> medium for cell cultivation after electroporation (at 1 × 10<sup>6</sup> cells/mL) by supplementing RPMI 1640 medium with 10% heat-inactivated FBS, 5% SC<sup>-</sup> (see <a href="#materials-and-equipment">materials and equipment</a>) and 180 U/mL IL-2.</li>
<li>Distribute 1 mL SC<sup>-</sup> medium per well into 24-well plates and keep the plate at 37°C, 5% CO<sub>2</sub> until further use to pre-warm the medium.</li>
</ol>
<h3 id="sec3.5">RNP production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60–90 min</div>
<p>CRISPR-Cas9-mediated OTR requires <i>hTRAC</i> and <i>hTRBC</i> RNPs for simultaneous editing of TCR α- and β-chains. Both RNPs are complexes of Cas9 protein and gRNA. Respective gRNAs consist of target-specific <i>hTRAC</i> or <i>hTRBC</i> crRNA sequences, directing Cas9 nuclease to the respective genetic loci, as well as of tracrRNA, serving as scaffold for the Cas9 protein.</p>
<p>For final electroporation, 3 μL of each RNP (final concentration 20 μM) are needed per sample. Here, we first produce 40 μM gRNAs by mixing equimolar amounts of tracrRNA with <i>hTRAC</i> or <i>hTRBC</i> crRNA. After the addition of electroporation enhancer (to enhance editing efficiency), the RNP production is concluded by mixing and then incubating equal volumes of gRNA and Cas9 protein (see <a href="#fig4">Figure 4</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Workflow of <i>hTRAC</i> and <i>hTRBC</i> ribonucleoproteins (RNPs) production</div>
<p>tracrRNA (black) is mixed with respective crRNA (<i>hTRAC</i> crRNA, light orange or <i>hTRBC</i> crRNA, dark orange) to produce gRNA. Electroporation enhancer (green asterisk) and then Cas9 protein (grey) are added to conclude RNP production.</p>
</figcaption>
</figure>
<div class="optional">
<span class="optional-title">Optional:</span> The addition of poly-L-glutamic acid (PGA) to RNPs has been described to improve editing efficiency and cell survival rates (<a href="#bib3">Kath et al., 2021</a>; <a href="#bib5">Nguyen et al., 2020</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Due to working with small volumes, pipetting errors and inaccuracies can have a considerable effect. We therefore advise to prepare additional 10% of RNPs to account for pipetting errors.</div>
<ol start="25">
<li>Mix equal volumes of tracrRNA (80 μM stock) with <i>hTRAC</i> or <i>hTRBC</i> crRNA (80 μM stock), respectively, to prepare 40 μM <i>hTRAC</i> and 40 μM <i>hTRBC</i> gRNA. Mix well by pipetting.</li>
<li>If necessary, shortly spin down to ensure the complete reaction mixture is at the bottom of the tube.</li>
<li>Heat at 95°C for 5 min in PCR cycler and then allow RNPs to cool to 20°C–22°C on bench top.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Run gRNA production protocol by 95°C for 5 min, followed by hold at 21°C. Using a 1.3 °C/s ramp (maximal ramp for this specific PCR cycler) we were able to produce gRNA for the RNPs which yielded very high TCR KO efficiencies.</div>
<ol start="28">
<li>Store gRNA on ice if you do not continue directly.</li>
</ol>
<ol start="29">
<li>Dilute Cas9 nuclease to 6 μM with commercial PBS buffer.
<div class="critical">
<span class="critical-title">Critical:</span> Minimize the time that Cas9 nuclease is at room temperature (20°C–22°C) and return to -20°C as quickly as possible. Take from stock only what you need for the experiment. Do not dilute the stock.</div>
<ol type="a">
<li>Calculate required volumes of Cas9 protein (62 μM stock) and commercial PBS buffer.</li>
<li>Prepare tube with required volume of PBS buffer.</li>
<li>Set up pipette for the required volume of Cas9 nuclease.</li>
<li>Quickly take Cas9 stock from −20°C. To avoid foaming, do not mix the Cas9 stock before taking out the required volume to prepare the dilution.</li>
<li>Slowly add Cas9 with prepared pipette and swirl slowly while pipetting.
<div class="note">
<span class="note-title">Note:</span> When adding the nuclease, you will see streaks of the Cas9 stock in the PBS buffer. After having added the required volume of Cas9, make sure to thoroughly mix by pipetting until all streaks have disappeared.</div>
</li>
</ol>
</li>
</ol>
<ol start="30">
<li>Keep diluted Cas9 on ice if you do not continue directly, otherwise bring it to 20°C–22°C for RNP assembly.</li>
<li>Add electroporation enhancer (400 μM stock) to gRNAs for a final concentration of 20 μM.</li>
<li>To mix equal volumes of diluted Cas9 nuclease (6 μM) and gRNA (40 μM), very slowly add Cas9 to the gRNA while moving pipette tip in cycles.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Always add diluted Cas9 protein to gRNA solution, not the other way around.</div>
<ol start="33">
<li>Mix by pipetting.</li>
<li>Check if RNP solution is clear and does not contain any precipitate.</li>
<li>Incubate RNPs for 15 min at 20°C–22°C.</li>
<li>Keep RNPs on ice until further use if they are processed the same day.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Store RNPs at −80°C if they are not processed the same day.</div>
<h3 id="sec3.6">Prepare electroporation buffer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Prepare 20 μL P3 electroporation buffer (Lonza) per condition. As this buffer is toxic for cells, the time cells spend in this buffer should be kept to a minimum.</p>
<ol start="37">
<li>Per sample, 20 μL P3 buffer are prepared by mixing 16.4 μL P3 with 3.6 μL supplement.</li>
<li>Keep at 20°C–22°C or on ice, if needed later.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> To reduce costs, Lonza Nucleocuvettes<sup>TM</sup> can be washed with ethanol and reused. When working with reused Nucleocuvettes<sup>TM</sup>, the self-made buffers such as ‘1M’ (<a href="#bib1">Chicaybam et al., 2013</a>) can be used instead of commercial P3 electroporation buffer. However, reuse of Nucleocuvettes<sup>TM</sup> increases the risk of error codes during electroporation.</div>
<h3 id="sec3.7">Electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min (hands-on) + 12–16 h (additional cell culture)</div>
<p>For electroporation, activated T cells are resuspended in electroporation buffer and then mixed with HDR template and RNPs (<a href="#fig5">Figure 5</a>). After transfer into the 16-well Nucleocuvette<sup>TM</sup> Strip, cells are electroporated in the Lonza 4D-Nucleofector<sup>TM</sup>. By electroporation, RNPs and template DNA are delivered into the nucleus of T cells where genetic editing occurs.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Genetic engineering of T cells via CRISPR-Cas9-mediated TCR KI</div>
<p>Activated T cells with endogenous TCRs (orange) are required for successful editing. In preparation for electroporation, the linearized double-stranded DNA template is mixed with <i>hTRAC</i> and <i>hTRBC</i> RNPs. T cells resuspended in electroporation buffer are mixed with the DNA-RNPs mixture and electroporated. Respective crRNAs direct the Cas9 nuclease either to the <i>TRAC</i> or <i>TRBC</i> locus and allow for CRISPR-Cas9-mediated DNA double-strand breaks (right). At the <i>TRAC</i> locus, homology-directed repair (HDR) results in the targeted KI of the HDR template DNA encoding the full transgenic αβ TCR which leads to a simultaneous KO of the endogenous <i>TRAC</i> (black triangle). At the <i>TRBC</i> locus, the double-strand break is resolved by non-homologous end-joining leading to the KO of the endogenous <i>TRBC</i> (black cross). On the surface of edited T cells only the transgenic TCR (blue) is expressed.</p>
</figcaption>
</figure>
<p>When working with 16-well Nucleocuvette<sup>TM</sup> Strips, cells are prepared in the ‘cells plate’ (<a href="#fig6">Figure 6</a>) in rows of 8 (see step 20). To minimize the time that cells spend in the electroporation buffer, only (up to) 8 samples are electroporated at a time. In preparation for electroporation, respective cells are transferred to the ‘centrifugation plate’ (<a href="#fig6">Figure 6</a>) and spun down. After centrifugation, cells are resuspended in P3 electroporation buffer, mixed with DNA/RNP-mix in the ‘pipetting plate’ (<a href="#fig6">Figure 6</a>) and transferred to Nucleocuvette<sup>TM</sup> Strip.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Workflow of preparing samples for electroporation</div>
<p><b>Step 1</b>: The targeting construct mix is prepared in the <i>‘</i>pipetting plate’ (blue plate) by mixing DNA template and RNPs (light blue wells). Additionally, 20 μL P3 buffer is added to the plate (light green wells). Cells to be electroporated are transferred from the ‘cells plate’ (orange plate) to the ‘centrifugation plate’ (grey plate). <b>Step 2</b>: After centrifugation, the supernatant in the ‘centrifugation plate’ is discarded. <b>Step 3</b>: Cell pellets are resuspended in 20 μL of P3 electroporation buffer. <b>Step 4</b>: 25 μL of resuspended cells (dark green) are transferred to the DNA/RNP-mix wells in the ‘pipetting plate’ and mixed. <b>Step 5</b>: 32 μL of cells/DNA/RNPs-mix (dark blue) is transferred to the 16-well Nucleocuvette<sup>TM</sup> Strip. <b>Step 6</b>: Nucleocuvette<sup>TM</sup> Strip is placed in the Nucleofector<sup>TM</sup> and cells are electroporated. Afterwards, cells are transferred to the 24-well plate with pre-warmed SC<sup>-</sup> medium (not shown). <b>Step 7</b>: Process is repeated for remaining samples.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> For electroporation, all reagents and cuvettes need to be at room temperature.</div>
<ol start="39">
<li>Switch on and set up Nucleofector<sup>TM</sup>.
<ol type="a">
<li>Select wells to be electroporated.</li>
<li>Select electroporation program using the pulse sequence EH100.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Different pulse sequences can be used (e.g., the use of EH115 has been successfully applied (<a href="#bib7">Roth et al., 2018</a>). After testing various pulse settings, we found that EH100 yields the best combination of editing efficiency and cell viability in our setting.</div>
<ol start="40">
<li>Prepare electroporation buffer for samples of the first electroporation (up to 8 samples). Distribute 20 μL of P3 electroporation buffer per well into a V-bottom 96-well plate (‘pipetting plate’) following the layout of the ‘cells plate’ (<a href="#fig6">Figure 6</a>).</li>
<li>Prepare targeting construct mix for the first OTR samples in the ‘pipetting plate’:
<ol type="a">
<li>First, add 0.5 μg template DNA to the wells of the ‘pipetting plate’. Volume of DNA depends on the concentration after purification. Values should range between 0.5 and 1.5 μL.</li>
<li>Add 3.0 μL <i>hTRAC</i> RNP + 3.0 μL <i>hTRBC</i> RNP to the template DNA (see <a href="#sec6.3">problem 2</a>).</li>
</ol>
</li>
<li>For the TCR KO sample, mix only 3.0 μL <i>hTRAC</i> RNP + 3.0 μL <i>hTRBC</i> RNP in the ‘pipetting plate’ to prepare targeting construct mix. Afterwards, treat TCR KO sample exactly as the OTR samples.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> When working with ssDNA templates, higher amounts of template can be used since ssDNA demonstrates less toxicity (<a href="#bib7">Roth et al., 2018</a>). However, the use of ssDNA templates increases the complexity of the protocol as templates need to be singularized.</div>
<ol start="43">
<li>Incubate at 20°C–22°C for at least 30 s.</li>
<li>Transfer cells to be electroporated to the 96-well V-bottom ‘centrifugation plate’ and spin down.
<ol type="a">
<li>Centrifuge for 3 min at 480 × <i>g</i>, 20°C–22°C.</li>
<li>Swiftly discard remaining supernatant.</li>
</ol>
</li>
<li>Set multichannel to 20 μL and take up 20 μL P3 buffer ‘pipetting plate’ (see <a href="#sec6.5">problem 3</a>).</li>
<li>Resuspend cell pellet in 20 μL P3 buffer. Avoid air bubbles and be cautious not to create any foam.</li>
<li>Increase pipetting volume to 25 μL. This 5 μL-increase in volume accounts for the left-over supernatant in the well.</li>
<li>Take 25 μL of resuspended cells of the ‘centrifugation plate’ and transfer to DNA/RNP-wells of the ‘pipetting plate’.</li>
<li>Mix by carefully pipetting up and down. Avoid air bubbles or foam.</li>
<li>Increase pipetting volume to 32 μL. This 7 μL-increase in volume accounts for the 2 × 3 μL of RNPs and the approx. 1 μL of DNA.</li>
<li>Quickly transfer cells/DNA/RNP-mix to fresh 16-well Nucleocuvette<sup>TM</sup> Strip.</li>
<li>Place lid on top and tap Nucleocuvette<sup>TM</sup> Strip on the bench to remove any air bubbles (see <a href="#sec6.5">problem 3</a>).</li>
<li>Place Nucleocuvette<sup>TM</sup> Strip into Nucleofector<sup>TM</sup>.</li>
<li>Start electroporation program: EH100 (see <a href="#sec6.7">problem 4</a>)</li>
<li>Transfer electroporated cells to SC<sup>-</sup> medium.
<ol type="a">
<li>Add 80 μL of pre-warmed SC<sup>-</sup> medium (from the 24-well plate (step 24) to the Nucleocuvette<sup>TM</sup> well and mix (to dilute P3 buffer).</li>
<li>Transfer cells to 24-well plate with 1 mL of pre-warmed SC<sup>-</sup> medium per well to seed the cells at a density of 1 × 10<sup>6</sup> cells/mL.</li>
<li>Take some SC<sup>-</sup> medium from the same well to flush Nucleocuvette<sup>TM</sup> well to collect any left-over cells.</li>
</ol>
</li>
<li>Incubate cells at 37°C, 5% CO<sub>2</sub> for 12–16 h.</li>
</ol>
<h3 id="sec3.8">Addition of antibiotics</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min (hands-on) + 3 days (additional cell culture)</div>
<p>Viability of KI cells is higher when they are cultivated in antibiotic-free SC<sup>-</sup> medium for 12–16 h after electroporation. Afterwards, SC<sup>-</sup> medium can be replaced by complete RPMI medium (including antibiotics) or the SC<sup>-</sup> medium can simply be supplemented with an antibiotic mix to produce complete RPMI medium.</p>
<ol start="57">
<li>Here, the SC<sup>-</sup> medium is supplemented with 5% antibiotics mix (see <a href="#materials-and-equipment">materials and equipment</a>) 12–16 h after electroporation.
<ol type="a">
<li>Per well, add 50 μL of antibiotics mix to the 1 mL SC<sup>-</sup> medium.</li>
<li>As the mock sample was already cultured in complete RPMI medium, no antibiotics need to be added here.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since the mock sample is not electroporated, mock cells tend to proliferate faster than those of electroporated samples. Therefore, it might be necessary to split mock sample cells (or at least add some fresh medium) if the medium starts to turn yellow.</div>
<ol start="58">
<li>Incubate the cells at 37°C, 5% CO<sub>2</sub> for another 3 days until flow cytometry analysis.</li>
</ol>
<h3 id="sec3.9">Validation of TCR KI by flow cytometry analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h - 1 day</div>
<p>Since RNPs are stable for around 72 h, flow cytometry analysis can be performed from day 3 after electroporation onwards. Transgene expression levels can be slightly higher at later time points, but are not expected to change substantially. To evaluate the efficiency of transgenic TCR re-expression and endogenous TCR KO, cells are stained for transgenic and endogenous TCR and analyzed by flow cytometry. As the transgenic TCR is equipped with a murine TCR β constant region (mTRBC), staining of mTRBC allows identification of transgenic TCRs and differentiation from endogenous TCRs (hTCR). Additionally, CD8 and/or CD4 expression (here: CD8) is analyzed. Our target population is the mTRBC<sup>+</sup>hTCR<sup>-</sup> population of (in this case) CD8<sup>+</sup> cells. This population is made up of cells expressing the transgenic TCR (mTRBC<sup>+</sup>) with simultaneous KO of the endogenous TCR (hTCR<sup>-</sup>).</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> When working with an HDR template containing fully human TCR sequences, transgenic TCRs can be identified and differentiated by antigen-specific peptide-MHC (pMHC)-multimer staining. Another approach, albeit less precise, would be antibody staining for the specific Vβ-chain of the transgenic TCR.</div>
<div class="note">
<span class="note-title">Note:</span> Staining is performed on ice.</div>
<ol start="59">
<li>Resuspend cells in the cell culture well before taking an aliquot for staining.</li>
<li>Recover 25–50 μL of cells per sample (including TCR KO and mock samples) and transfer sample to a 96-well V-bottom plate.</li>
<li>Return the plate with the remaining cells to the incubator (at 37°C, 5% CO<sub>2</sub>) until further use.</li>
<li>Additionally, recover mock cells for single color (S.C.) stainings and add them to the plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 100,000 cells per S.C. are sufficient to set up the instrument and create compensation controls.</div>
<ol start="63">
<li>Wash 2.5 × with FACS buffer:
<ol type="a">
<li>Fill wells to 200 μL with cold FACS wash buffer, centrifuge (3 min, 480 × <i>g</i>, 4°C), discard supernatant (= 0.5 × wash step).</li>
<li>Resuspend pellet in 200 μL cold FACS buffer, centrifuge, discard supernatant (= 1 × wash step). Repeat this washing step once.</li>
</ol>
</li>
<li>Prepare antibody master mix for 50 μL staining volume per sample by adding antibodies (anti-CD8-PE, anti-hTCR-FITC and anti-mTRBC-APC/Fire750) in their respective dilution to the FACS buffer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> CD3-staining can be included to facilitate the identification of cells expressing the transgenic TCR as these cells should be CD3<sup>+</sup>mTRBC<sup>+</sup> double positive.</div>
<ol start="65">
<li>Resuspend cell pellets of samples in 50 μL antibody master mix.</li>
<li>Resuspend cell pellets of S.C. in 50 μL FACS buffer and add one of the S.C. antibodies (anti-CD8-PE, anti-hTCR-FITC, anti-CD45-ECD and anti-mTRBC-APC-eFluor780) per well in the respective dilution.</li>
<li>Mix samples by pipetting and incubate 20 min on ice in the dark.</li>
<li>Fill wells to 200 μL with FACS buffer, centrifuge (3 min, 480 × g, 4°C) and discard supernatant.</li>
<li>Resuspend CRISPR-samples (excluding S.C.) in 50 μL of Propidium Iodide (PI) solution (2 μg/mL) and incubate for 3 min on ice in the dark.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> PI can be added to the CRISPR-samples 3 min before the end of the incubation. After 3 min of incubation with PI, samples can be washed 2.5 ×.</div>
<ol start="70">
<li>Wash 1.5 × with FACS buffer (see step 63).</li>
<li>Resuspend the sample in 200 μL FACS buffer and filter through a nylon mesh (mesh width: 100 μm).</li>
<li>Acquire samples at flow cytometer.</li>
<li>Quantify percentages and counts of CD8<sup>+</sup> mTRBC<sup>+</sup>hTCR<sup>-</sup> cells (see <a href="#sec6.9">problems 5</a> and <a href="#sec6.11">6</a>).</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The described DNA template generation and purification is expected to yield 0.5–1.0 μg/μL. Exemplary flow cytometry results after OTR are shown in <a href="#fig7">Figure 7</a>. TCR KO efficiencies are expected to range from 95-99% if only the KO is performed and 90%–95% if TCR KO and KI are performed simultaneously. TCR re-expression after KI of the full transgenic TCR into <i>TRAC</i> is expected to range from 5-15%. This value becomes considerably lower (i.e., 1%–2%) for dual TCR α- and β-chain editing, when the α-chain is targeted to the <i>TRAC</i> and the β-chain is simultaneously targeted to the <i>TRBC</i>. For both engineering approaches, a distinct mTRBC<sup>+</sup>hTCR<sup>-</sup> population is expected. Additionally, the absence of a double positive mTRBC<sup>+</sup>hTCR<sup>+</sup> population indicates that no mispairing between transgenic and endogenous TCR-chains occurs (<a href="#sec6.11">problem 6</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Exemplary flow cytometry plots after OTR</div>
<p>Mock sample (not electroporated), TCR KO (KO of endogenous α-chain and β-chain), Single KI (Full TCR targeted into <i>TRAC</i> with concomitant β-chain KO) and Dual KI (α-chain targeted into <i>TRAC</i> and the β-chain into <i>TRBC</i>). All samples are pre-gated on living, single CD8<sup>+</sup> lymphocytes.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol was optimized for TCR replacement in primary human T cells. When working with other cell types and different receptors, experimental conditions and settings most likely need adaption. This protocol is limited to the electroporation of 1 × 10<sup>6</sup> cells per Nucleocuvette<sup>TM</sup> well. Upscaling of the electroporation process can be achieved by adapting this protocol to the use of 100 μL Nucleocuvette<sup>TM</sup> Vessels. This allows for the electroporation of up to 30 × 10<sup>6</sup> cells per cartridge.</p>
<p>TCR re-expression rates of 5–15% are expected. DNA-sensing inhibitors (see <a href="#sec3.3">T cell activation</a>), PGA (see <a href="#sec3.5">RNP production</a>), HDR-enhancers (<a href="#bib3">Kath et al., 2021</a>; <a href="#bib5">Nguyen et al., 2020</a>) have been described to enhance re-expression rates. The combination of DNA-dependent protein kinase (DNA-PK) inhibitor M3814 and HDAC class I/II Inhibitor Trichostatin A has been shown to increase re-expression rates especially in combination the hybrid DNA templates described above (see <a href="#sec1.1">DNA template design</a>) (<a href="#bib9">Shy et al., 2021</a>).</p>
<p>Typically, the TCR re-expression rates (that is the transgenic protein surface expression rates) are lower for TCRs than they are for chimeric antigen receptors (CARs). However, if there are still B cells present in the culture system (often in the first 1–2 weeks after electroporation), CAR-expression levels can be very low to non-existent as B cells induce downregulation of CARs on edited cells.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>PCR was not successful as there is more or less than one band showing on agarose gel.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>If no bands are visible, make sure the primer sequence is correct and DNA is added (in the correct concentration).</p>
<p>If more than one band appears one the agarose gel, the primers may have bound at more than the intended sites on the plasmid. To fine-tune the PCR, adjust the annealing temperature. Depending on the polymerase, extension time and number of cycles might also need adjustment.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Prepared RNPs are not sufficient to prepare all samples.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Prepare new RNPs to continue the current experiment. To avoid this problem in future experiments, increase pipetting error margins considered in preparation of RNPs.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Air bubble is trapped in Nucleocuvette<sup>TM</sup>.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>If there is a big air bubble underneath the liquid surface, firmly tab (avoid spilling) the Nucleocuvette<sup>TM</sup> on the bench. Tapping the cuvette angled to the side works best for this purpose. To avoid trapping air, carefully pipette the liquid to the bottom of the Nucleocuvette<sup>TM</sup>.</p>
<p>Smaller air bubbles on the surface can be burst (or pushed up until they burst) with a fresh pipette tip. To avoid small bubbles in the Nucleocuvette<sup>TM</sup>, it is crucial to prevent air bubbles or even foaming during the resuspension of the cells in P3 electroporation buffer and the subsequent mixing with RNPs.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Nucleofector<sup>TM</sup> displays warning (yellow) or error (red) code during electroporation.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Warning and error codes can be due to several reasons. By clicking on the respective well, the error code can be displayed. Explanation, possible causes and suggested procedures are listed in the manufacturer's manual (Chapter 4: Error Codes of the 4D-Nucleofector™ System, page 35–36) and on their website .</p>
<p>Generally, we would advise to continue with the experiment as TCR replacement can still be successful even after error codes, albeit often with lower TCR re-expression rates. If enough stimulated cells are left over, repeating sample preparation and electroporation for the respective well(s) in a fresh Nucleocuvette<sup>TM</sup> is the best solution.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Cell viability after electroporation is lower than expected.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Reduced cell viability might result from prolonged time which cells had to spend in P3 electroporation buffer. Although 16 samples can be electroporated at once, we would advise to only work with up to 8 samples at a time. This reduces the time before electroporated cells are transferred to the SC<sup>-</sup> medium. Additionally, introducing DNA-sensing inhibitors into the protocol has been described to increase cell viability (<a href="#bib3">Kath et al., 2021</a>).</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Flow cytometry analysis does not show the expected results.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>A larger, unedited (hTCR<sup>+</sup>mTRBC<sup>-</sup>) population indicates that too much DNA template was used. In contrast, using too little DNA template achieves high KO rates but correspondingly fewer KI cells.</p>
<p>Adjusting the amount of DNA template accordingly should solve these problems. TCR KO rates &lt;90%–95% in the KO only control indicate problems with CRISPR-Cas9 editing (independent from HDR) and/or the protocol.</p>
</section>
<section>
<h2 id="institutional_permissions">Institutional permissions</h2>
<p>Written informed consent was obtained from the donors, and usage of the blood samples was approved according to national law by the local Institutional Review Board (Ethikkommission der Medizinischen Fakultät der Technischen Universität München). Information about age and gender of donors is not available. The study conforms to the standards of the Declaration of Helsinki.</p>
<h2 id="references">References</h2>
<p id="bib1">Chicaybam, L., Sodre, A.L., Curzio, B.A., and Bonamino, M.H. (2013). An efficient low cost method for gene transfer to T lymphocytes. PLoS One <i>8</i>, e60298. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref1">View at publisher</a></p>
<p id="bib2">Cohen, C.J., Li, Y.F., El-Gamil, M., Robbins, P.F., Rosenberg, S.A., and Morgan, R.A. (2007). Enhanced antitumor activity of T cells engineered to express T-cell receptors with a second disulfide bond. Cancer Res. <i>67</i>, 3898-3903. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref2">View at publisher</a></p>
<p id="bib3">Kath, J., Du, W., Thommandru, B., Turk, R., Amini, L., Stein, M., Zittel, T., Martini, S., Ostendorf, L., Wilhelm, A., et al. (2021). Fast, efficient and virus-free generation of TRAC-replaced CAR T cells. BioRxiv. <a class="external-link" href="https://doi.org/10.1101/2021.02.14.431017">https://doi.org/10.1101/2021.02.14.431017</a></p>
<p id="bib4">Müller, T.R., Jarosch, S., Hammel, M., Leube, J., Grassmann, S., Bernard, B., Effenberger, M., Andrä, I., Chaudhry, M.Z., Käuferle, T., et al. (2021). Targeted T cell receptor gene editing provides predictable T cell product function for immunotherapy. Cell Rep. Med. <i>2</i>, 100374. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref4">View at publisher</a></p>
<p id="bib5">Nguyen, D.N., Roth, T.L., Li, P.J., Chen, P.A., Apathy, R., Mamedov, M.R., Vo, L.T., Tobin, V.R., Goodman, D., Shifrut, E., et al. (2020). Polymer-stabilized Cas9 nanoparticles and modified repair templates increase genome editing efficiency. Nat. Biotechnol. <i>38</i>, 44-49. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref5">View at publisher</a></p>
<p id="bib6">Ren, J., Liu, X., Fang, C., Jiang, S., June, C.H., and Zhao, Y. (2017). Multiplex genome editing to generate universal CAR T cells resistant to PD1 inhibition. Clin. Cancer Res. <i>23</i>, 2255-2266. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref6">View at publisher</a></p>
<p id="bib7">Roth, T.L., Puig-Saus, C., Yu, R., Shifrut, E., Carnevale, J., Li, P.J., Hiatt, J., Saco, J., Krystofinski, P., Li, H., et al. (2018). Reprogramming human T cell function and specificity with non-viral genome targeting. Nature <i>559</i>, 405-409. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref7">View at publisher</a></p>
<p id="bib8">Schober, K., Müller, T.R., Gökmen, F., Grassmann, S., Effenberger, M., Poltorak, M., Stemberger, C., Schumann, K., Roth, T.L., Marson, A., and Busch, D.H. (2019). Orthotopic replacement of T-cell receptor α- and β-chains with preservation of near-physiological T-cell function. Nat. Biomed. Eng. <i>3</i>, 974-984. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00737-1/sref8">View at publisher</a></p>
<p id="bib9">Shy, B.R., Vykunta, V., Ha, A., Roth, T.L., Talbot, A., Nguyen, D.N., Chen, Y.Y., Blaeschke, F., Vedova, S., Mamedov, M.R., et al. (2021). Hybrid ssDNA repair templates enable high yield genome engineering in primary cells for disease modeling and cell therapy manufacturing. BioRxiv. <a class="external-link" href="https://doi.org/10.1101/2021.09.02.458799">https://doi.org/10.1101/2021.09.02.458799</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to the lead contact, Kilian Schober (<a href="mailto:kilian.schober@uk-erlangen.de">kilian.schober@uk-erlangen.de</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any new data sets or code. Data sets supporting the protocol and further examples of OTR are described by <a href="#bib8">Schober et al. (2019)</a> and <a href="#bib4">Müller et al. (2021)</a>.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1282-Mmc1.pdf">Table S1. HDR DNA template sequence (TCR 6-2), related to before you begin step 1</a></p>
<h3>Acknowledgments</h3>
<p>The work of C.M. and K.S. was supported by the German Federal Ministry of Education and Research (BMBF; project 01KI2013). The work of T.R.M and D.H.B. was supported by <a href="https://doi.org/10.13039/100009139">German Center for Infection Research</a> (DZIF) as well as by the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> (DFG, German Research Foundation) SFB 1321/TP17, SFB 1054/B09, and SFB 1371/TP04. We thank Ev-Marie Schuster for critical reading of the manuscript.</p>
<h3>Author contributions</h3>
<p>Conceptualization, K.S., T.R.M., and D.H.B.; Methodology, K.S., T.R.M., and D.H.B.; Validation, K.S., C.M., T.R.M., and D.H.B; Investigation, K.S., C.M., and T.R.M.; Writing – Original Draft, C.M. and K.S.; Writing – Review &amp; Editing, K.S., C.M., T.R.M., and D.H.B.</p>
<h3>Declaration of interests</h3>
<p>D.H.B. is co-founder of STAGE Cell Therapeutics GmbH (now Juno Therapeutics, a Bristol-Myers Squibb Company) and T Cell Factory B.V. (now Kite, a Gilead Company). D.H.B. has a consulting contract with and receives sponsored research support from Juno Therapeutics. D.H.B. is member of the Scientific Advisory Board of Immatics. All other authors declare no competing interests.</p>
</section>
</article>