<article>
<section>
<h1 id="header">Protocol for analyzing the biosynthesis and degradation of N-glycan precursors in mammalian cells</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>Yoichiro Harada,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup> Kazuki Nakajima,<sup><a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Shengtao Li,<sup><a href="#aff3">3</a>,<a href="#fn1">4</a></sup> Tadashi Suzuki,<sup><a href="#aff3">3</a></sup> and Naoyuki Taniguchi<sup><a href="#aff1">1</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Glyco-Oncology and Medical Biochemistry, Osaka International Cancer Institute, 3-1-69 Otemae, Chuo-ku, Osaka 541-8567, Japan</p>
<p id="aff2"><sup>2</sup>Center for Joint Research Facilities Support, Research Promotion and Support Headquarters, Fujita Health University, Toyoake, Aichi 470-1192, Japan</p>
<p id="aff3"><sup>3</sup>Glycometabolic Biochemistry Laboratory, RIKEN Cluster for Pioneering Research, Saitama 351-0198, Japan</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yoharada3@mc.pref.osaka.jp">yoharada3@mc.pref.osaka.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100316">10.1016/j.xpro.2021.100316</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>N-glycosylation is a fundamental post-translational protein modification in the endoplasmic reticulum of eukaryotic cells. The biosynthetic and catabolic flux of N-glycans in eukaryotic cells has long been analyzed by metabolic labeling using radiolabeled sugars. Here, we introduce a non-radiolabeling protocol for the isolation, structural determination, and quantification of N-glycan precursors, dolichol-linked oligosaccharides, and the related metabolites, including phosphorylated oligosaccharides and nucleotide sugars. Our protocol allows for capturing of the biosynthesis and degradation of N-glycan precursors at steady state.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib3">Harada et al. (2013)</a>, <a href="#bib4">Harada et al. (2020)</a>, and <a href="#bib8">Nakajima et al. (2013)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Purification of DLOs, POSs, and nucleotide sugars from adherent mammalian cells</li>
<li>Fluorescent labeling of glycans liberated from DLOs and POSs</li>
<li>Liquid chromatography analysis of the fluorescently labeled glycans</li>
<li>Liquid chromatography-mass spectrometry analysis of nucleotide sugars</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol described here is optimized for the analysis of steady-state levels of DLOs, POSs, and nucleotide sugars in adherent mammalian cells. Nucleotide sugars, which are typically synthesized <i>de novo</i> from glucose, serve as building blocks for the assembly of DLOs in the endoplasmic reticulum (ER) (<a href="#fig1">Figure 1</a>). The fully assembled, mature form of a DLO contains a tetradecaoligosaccharide comprised of three glucose (Glc), nine mannose (Man) and two <i>N</i>-acetylglucosamine (GlcNAc) residues. When cells are deprived of glucose, the efficiency of nucleotide sugar biosynthesis becomes low, and reductions in guanosine diphosphate (GDP)-Man levels cause the biosynthetic arrest of DLOs, which, in turn, induce the premature degradation of the biosynthetic intermediates of DLOs (<a href="#bib3">Harada et al., 2013</a>). This type of DLO degradation results in the release of POSs with a single phosphate group into the cytosol (<a href="#bib2">Harada et al., 2016</a>). POS release can also be induced by treating cells with 2-deoxyglucose (2-DG), which is metabolically incorporated into DLOs through nucleotide sugar metabolism, resulting in the degradation of the abnormal 2-DG-containing DLO intermediates into POSs (<a href="#bib4">Harada et al., 2020</a>). Mammalian cells also contain neutral free glycans generated by the degradation of DLOs in the ER or that of N-glycosylated proteins either in the cytosol or in lysosomes (<a href="#bib1">Harada et al., 2015</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The biosynthesis and degradation of dolichol-linked oligosaccharides in the endoplasmic reticulum.</div>
<p>Glucose is metabolized to nucleotide sugars, e.g., UDP-<i>N</i>-acetylglucosamine (UDP-GlcNAc), GDP-mannose (GDP-Man) and UDP-glucose (UDP-Glc), in the cytosol. These nucleotide sugars are directly used as sugar donor substrates for the assembly of dolichol-linked oligosaccharides (DLOs) on the cytosolic side of the endoplasmic reticulum (ER). When Man<sub>5</sub>GlcNAc<sub>2</sub>-pyrophosphate (PP)-dolichol is formed, the biosynthetic intermediate is transferred to the luminal side of the ER. In the ER lumen, Glc<sub>3</sub>Man<sub>9</sub>GlcNAc<sub>2</sub>-PP-dolichol is formed by using dolichol phosphate-linked mannose (Dol-P-Man) and dolichol phosphate-linked glucose (Dol-P-Glc) as the sugar donor substrates. Dol-P-Man and Dol-P-Glc are formed on the cytosolic side of the ER membrane from GDP-Man and UDP-Glc, respectively, and they are then transferred to the ER lumen. When glucose availability is low, the biosynthesis of DLOs is arrested at an early stage (Man<sub>0-3</sub>GlcNAc<sub>2</sub>-PP-dolichol) due to the shortage of GDP-Man, resulting in the degradation of the DLO intermediates into phosphorylated oligosaccharides (POSs). 2-Deoxyglucose (2-DG) is an analog of glucose and it can be metabolized to unnatural forms of nucleotide sugars, i.e., UDP-2-DG and GDP-2-DG (<a href="#bib4">Harada et al., 2020</a>). GDP-2-DG can serve as a sugar donor substrate for DLO biosynthesis and incorporated into the positions where mannose is supposed to be located. This metabolic hijacking induces the degradation of the aberrantly formed DLO intermediates into POSs (<a href="#bib4">Harada et al., 2020</a>).</p>
</figcaption>
</figure>
<p>Mammalian cells are cultured and separated into ethanol-soluble (POSs, neutral free glycans and nucleotide sugars) and ethanol-insoluble (DLOs) fractions. POSs can be further separated from neutral free glycans by anion-exchange chromatography. DLOs are prepared from the insoluble fraction by extraction with a mixture of solvents composed of chloroform, methanol, and water. The glycan moiety of DLOs and POSs is liberated from the aglycon moiety (i.e., dolichol-pyrophosphate and a phosphate group, respectively), and the exposed reducing end is fluorescently labeled by reaction with 2-aminopyridine (PA). The PA-labeled glycans are then separated and quantitated by size-fractionation HPLC. Nucleotide sugars are separately purified from the ethanol-soluble fraction through solid-phase extraction and are then analyzed by liquid chromatography (LC)-electrospray ionization (ESI)-tandem mass spectrometry (MS/MS) without any derivatization.</p>
<h3 id="sec1.1">Cell culture</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h for cell plating and 2 days for cell culture</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: cell culture must be performed in an appropriate biosafety cabinet, and appropriate gloves and goggles need to be worn.</div>
<ol>
<li>Prepare a complete medium with or without 2-deoxyglucose (2-DG) and a glucose-deprived medium. Warm the media at 37°C before use.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The complete medium should contain 10% fetal bovine serum (FBS), as well as 25 mM glucose to ensure cell viability and the robust biosynthesis of DLOs. We previously demonstrated that in immortalized mouse embryonic fibroblasts (MEFs), the amounts of DLOs decrease by approximately 50% with no induction of POS release at 24 h after switching the glucose concentration of DMEM from 25 mM to 5 mM (<a href="#bib3">Harada et al., 2013</a>). This finding indicates that the biosynthesis of DLOs normally occurs under physiological glucose conditions (i.e., 5 mM) but that it does not reach a plateau. However, investigators need to take into account that high concentrations of glucose may cause undesirable non-enzymatic reactions, such as glycation (<a href="#bib7">Nagai et al., 2005</a>). The glucose-deprived medium should contain 0.5 mM glucose and 10% FBS. The concentration of 2-DG in the complete medium should be 2.5 mM to induce POS release.</div>
<ol start="2">
<li>Trypsinize and plate 1 × 10<sup>6</sup> of immortalized mouse embryonic fibroblasts (MEFs) or mouse melanoma B16-F10 cells on a ϕ100 mm culture dish containing 10 mL of the complete medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare two identical dishes for each experimental condition. One dish is used for the preparation of DLOs, POSs, or nucleotide sugars, while the other dish is used for counting cell numbers and to normalize the amounts of DLOs, POSs and nucleotide sugars.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use at least a one ϕ100 mm culture dish to obtain sufficient numbers of cells for the preparation of DLOs, POSs, and nucleotide sugars.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Optimize the number of cells for plating so as to obtain approximately an 80%–90% confluency after 48 h incubation in the complete medium. Do not let the cells grow to a confluent stage.</div>
<ol start="3">
<li>Incubate the cells for 24 h under optimized culture conditions.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> MEFs are cultured at 37°C in a 5% CO<sub>2</sub> atmosphere. B16-F10 are cultured at 37°C in a 8% CO<sub>2</sub> atmosphere. The culture condition described here for B16-F10 cells is very important, since B16-F10 cells start to secrete large amounts of pigments and stop dividing when cultured in the complete medium in a 5% CO<sub>2</sub> atmosphere. The complete medium used in this protocol contains 3.7 g/L of sodium bicarbonate, which results in the pH of the medium being 7.8 in a 5% CO<sub>2</sub> atmosphere and 7.4 in a 8% CO<sub>2</sub> atmosphere.</div>
<ol start="4">
<li>At 24-h post-seeding, the cells are washed two times with 5 mL each of 1× PBS.</li>
<li>The cells are further incubated for up to 24 h in 10 mL of the complete medium with or without 2-DG, or in 10 mL of a glucose-deprived medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Incubating the cells in glucose-deprived medium for longer periods of time often causes the cells being detached from the culture dish. This protocol uses uncoated dishes for cell cultures, but culture dishes that are coated with substrates (e.g., poly-L-lysine or collagen) may improve the attachment of cells to the dish.</div>
<h3 id="sec1.2">HPLC setting for analysis of PA-labeled glycans</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Separation, detection, and quantification of fluorescently labeled glycans require an HPLC system equipped with gradient pumps, a degasser (optional), an auto-sampler or a manual injector, a column holder, and a fluorescence detector.</p>
<ol start="6">
<li>The elution is performed by using two solvent systems as follows: eluent A, 97% (v/v) acetonitrile in 0.3% (v/v) acetate; eluent B, 20% (v/v) acetonitrile in 0.3% (v/v) acetate (pH 7.0).</li>
<li>Connect the Shodex NH2P-50 4E column to the HPLC unit. The gradient program is as follows (expressed as the percentage of eluent B): 0–5 min, isocratic 3%; 5–8 min, 3%–33%; 8–40 min, 33%–71%; 40–60 min, 3%. The flow rate is set to 0.8 mL/min. The fluorescence detector is set to the excitation wavelength at 310 nm and emission wavelength at 380 nm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To obtain consistent results between experiments, run the HPLC program at least once by injecting water (10 μL) before analyzing external standards and your samples.</div>
<h3 id="sec1.3">PA-labeled glucose oligomers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>PA-labeled glucose oligomers are used as external standards for standardization of the elution positions of PA-labeled glycans and their quantification. Since PA-labeled glucose oligomers are no longer commercially available, they will need to be prepared from an unlabeled glucose homopolymer as described in this section. Adjustment to the concentrations of “in-house” PA-labeled glucose oligomers can be carried out upon request.</p>
<ol start="8">
<li>Dissolve 10 μg (1 vial) of non-labeled AdvanceBio glucose homopolymer standard in 100 μL of Milli-Q water.</li>
<li>Transfer 50 μL of the solution to a fresh 1.5-mL centrifuge tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The rest of solution (50 μL) can be stored at −20°C.</div>
<ol start="10">
<li>Take the solution to dryness using a Speed-Vac.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
<ol start="11">
<li>Fluorescently label the glucose homopolymer with PA (see the <a href="#step-by-step-method-details">Step-by-step method details</a> section “<a href="#sec3.5">Fluorescent Labeling of Glycans with PA</a>”).</li>
<li>Purify the PA-glucose oligomers (see the <a href="#step-by-step-method-details">Step-by-step method details</a> section “<a href="#sec3.5">Fluorescent Labeling of Glycans with PA</a>”).</li>
<li>Dilute the PA-labeled glucose oligomers with water at a concentration of 2 pmol/μL as PA-glucose hexamer and store at −20°C.</li>
</ol>
<h3 id="sec1.4">LC-MS setting for analysis of nucleotide sugars</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<div class="note">
<span class="note-title">Note:</span> Use an Agilent 1100 series HPLC system coupled to an Esquire HCT ion trap mass spectrometer (Bruker Daltonics) or equivalent.</div>
<ol start="14">
<li>The elution is performed by using two solvent systems as follows: eluent C, 20 mM triethylammonium acetate, pH 6.0; eluent D (eluent C/acetonitrile, 8:2 (v/v)).</li>
<li>Connect the Inertsil ODS-3 column to the LC system. The gradient program is as follows (expressed as the percentage of eluent D): 0–13 min, isocratic 0%; 13–35 min, 0%–77%; 35–36 min, 77%–100%; 36–50 min, 100%; 50–70 min, isocratic 0%. The flow rate is set to 0.2 mL/min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The use of Inertsil ODS-3 column, or alternatively, Inertsil ODS-4 column is crucial for the successful separation of nucleotide sugars. These columns are end-capped reversed-phase columns with a higher hydrophobicity (surface area: 450 m<sup>2</sup>/g) than a conventional column.</div>
<h3 id="sec1.5">Alternative LC-MS setting</h3>
<p>Use a Nexera HPLC system coupled to an LCMS-8060 triple quadrupole mass spectrometer (Shimadzu).</p>
<p>Equilibrate the zwitterionic (ZIC) column with the phosphocholine phase (ZIC-cHILIC, 2.1 mm i.d. × 150 mm, 3 μm; Merck SeQuant) with 20 mM acetate buffer, pH 4.7, containing 90% acetonitrile. The gradients are as follows: 90%(v/v) buffer A for 13 min, 0%–77% (v/v) linear gradient of buffer B for 22 min, 77%–100% (v/v) buffer B (A plus 20% acetonitrile) for 1 min, and 100% buffer B for 14 min (<a href="#bib6">Ito et al., 2014</a>).</p>
<h3 id="sec1.6">Mass spectrometer settings</h3>
<p>Analysis of nucleotide sugars is conducted in the multiple reaction monitoring (MRM) mode using specific precursor-product ion pairs. Thus, the compound parameters for standard nucleotide sugars are individually determined by directly infusing the nucleotide sugar samples into the flowing eluent in the LC-MS system that is disconnected from the separation column. The employed precursor ion, product ions, and collision energies (amplitude in case of Esquire HCT) are as follows.</p>
<div class="note">
<span class="note-title">Note:</span> The parameters for the Esquire HCT ion trap mass spectrometer should be set as follows: Nitrogen gas used as nebulized- and drying gas are set at 30 psi and 10 L/min, respectively. Argon gas (purity, &gt; 99.9995%) is used for collision-induced dissociation (CID). Drying gas temperatures are set at 350°C. Detail expert parameters are automatically set by smart parameter setting as follows: Target Mass, <i>m/z</i> 600, compound stability, 100%; and optimize, normal.</div>
<div class="note">
<span class="note-title">Note:</span> The parameters for the LCMS8060 triple quadrupole mass spectrometer should be set as follows: Nitrogen gas used as nebulized- and drying gas should be set at 3 L/min and 10 L/min, respectively. Argon gas (purity, &gt; 99.9995%) is also used. Heat block and desolvation line temperatures should be set at 400°C and 250°C, respectively. The MS instrument is programmed to perform MRM with 3 ms pause time (dead time between MRM transitions) and 20 ms dwell time (ion monitoring time).</div>
<table id="undtbl1">
<thead>
<tr>
<th rowspan="2">MRM settings</th>
<th rowspan="2">Specific precursor → product ion pairs</th>
<th>Esquire HCT</th>
<th>LCMS8060</th>
</tr>
<tr>
<th>Amplitude (V)</th>
<th>Collision energy (V)</th>
</tr>
</thead>
<tbody>
<tr>
<td>UDP-Glc and UDP-Gal</td>
<td>
<i>m/z 565.1</i> → <i>323.1</i> [UMP-H]<sup>−</sup>
</td>
<td>0.65</td>
<td>26</td>
</tr>
<tr>
<td>UDP-GlcA</td>
<td>
<i>m/z 579.1</i> → <i>323.1</i> [UMP-H]<sup>−</sup>
</td>
<td>0.65</td>
<td>26</td>
</tr>
<tr>
<td>UDP-GlcNAc and UDP-GalNAc</td>
<td>
<i>m/z 606.1</i> → <i>384.1</i> [UDP-H-H<sub>2</sub>O]<sup>−</sup>
</td>
<td>0.75</td>
<td>26</td>
</tr>
<tr>
<td>UDP-2-DG</td>
<td>
<i>m/z 549.1</i> → <i>323.1</i> [UMP-H]<sup>−</sup>
</td>
<td>0.65</td>
<td>25</td>
</tr>
<tr>
<td>GDP-Man</td>
<td>
<i>m/z 604.1</i> → <i>442.1</i> [GDP-H]<sup>−</sup>
</td>
<td>0.70</td>
<td>24</td>
</tr>
<tr>
<td>GDP-Fuc and GDP-2-DG</td>
<td>
<i>m/z 588.1</i> → <i>442.1</i> [GDP-H]<sup>−</sup>
</td>
<td>0.70</td>
<td>25</td>
</tr>
<tr>
<td>GDP-Glc</td>
<td>
<i>m/z 604.1</i> → <i>362.1</i> [GMP-H]<sup>−</sup>
</td>
<td>0.65</td>
<td>24</td>
</tr>
<tr>
<td>CMP-NeuAc</td>
<td>
<i>m/z 613.1</i> → <i>322.1</i> [CMP-H]<sup>−</sup>
</td>
<td>0.45</td>
<td>20</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM, high glucose</td>
<td>FUJIFILM Wako</td>
<td>Cat# 044-29765</td>
</tr>
<tr>
<td>DMEM, no glucose</td>
<td>Gibco</td>
<td>Cat# 11966025</td>
</tr>
<tr>
<td>D-PBS (-) for cell culture</td>
<td>FUJIFILM Wako</td>
<td>Cat# 045-29795</td>
</tr>
<tr>
<td>0.05% (w/v) trypsin, 0.53 mM ethylenediaminetetraacetic acid (EDTA)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 202-16931</td>
</tr>
<tr>
<td>0.4% (w/v) Trypan blue</td>
<td>FUJIFILM Wako</td>
<td>Cat# 207-17081</td>
</tr>
<tr>
<td>rAPid alkaline phosphatase</td>
<td>Sigma-Aldrich</td>
<td>Cat# 4898133001</td>
</tr>
<tr>
<td>Magnesium chloride (MgCl<sub>2</sub>·6H<sub>2</sub>O)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 135-00165CAS: 7791-18-6</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 191-01665CAS: 7647-14-5</td>
</tr>
<tr>
<td>Potassium chloride (KCl)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 163-03545CAS: 7447-40-7</td>
</tr>
<tr>
<td>Disodium hydrogen phosphate (Na<sub>2</sub>HPO<sub>4</sub>)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 196-02835CAS: 10039-32-4</td>
</tr>
<tr>
<td>Potassium dihydrogen phosphate (KH<sub>2</sub>PO<sub>4</sub>)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 169-04245CAS: 7778-77-0</td>
</tr>
<tr>
<td>2-Amino-2-hydroxymethyl-1,3-propanediol (Tris)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 207-06275CAS: 77-86-1</td>
</tr>
<tr>
<td>2-Aminopyridine (fluorescent labeling grade)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 011-14181CAS: 504-29-0</td>
</tr>
<tr>
<td>Dimethylamine-borane</td>
<td>FUJIFILM Wako</td>
<td>Cat#026-08402CAS: 74-94-2</td>
</tr>
<tr>
<td>Ammonium bicarbonate (LC-MS grade)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 40867-50G-FCAS: 1066-33-7</td>
</tr>
<tr>
<td>Hydrochloric acid</td>
<td>FUJIFILM Wako</td>
<td>Cat# 080-01066CAS: 7647-01-0</td>
</tr>
<tr>
<td>2-Propanol</td>
<td>FUJIFILM Wako</td>
<td>Cat# 166-04836CAS: 67-63-0</td>
</tr>
<tr>
<td>Methanol (used for preparation of DLOs and POSs)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 131-01826CAS: 67-56-1</td>
</tr>
<tr>
<td>Chloroform</td>
<td>FUJIFILM Wako</td>
<td>Cat# 038-02606CAS: 67-66-3</td>
</tr>
<tr>
<td>Ethanol (used for preparation of DLOs and POSs)</td>
<td>FUJIFIML Wako</td>
<td>Cat# 057-00456CAS: 64-17-5</td>
</tr>
<tr>
<td>Ethanol (HPLC grade) (used for preparation of nucleotide sugars)</td>
<td>Kanto Chemical</td>
<td>Cat# 14033-1BCAS: 64-17-5</td>
</tr>
<tr>
<td>Methanol (LC-MS grade)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 134-14523CAS: 67-56-1</td>
</tr>
<tr>
<td>Acetonitrile (LC-MS grade)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 018-19583CAS: 75-05-8</td>
</tr>
<tr>
<td>Distilled water (LC-MS grade)</td>
<td>Kanto Chemical</td>
<td>Cat# 11307-76CAS: 7732-18-5</td>
</tr>
<tr>
<td>Triethylamine (LC-MS grade)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 65897-50MLCAS: 121-44-8</td>
</tr>
<tr>
<td>Trifluoroacetic acid (HPLC grade)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 206-10731CAS: 76-05-1</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>FUJIFILM Wako</td>
<td>Cat# 017-00256CAS: 64-19-7</td>
</tr>
<tr>
<td>AdvanceBio glucose homopolymer standard</td>
<td>Agilent</td>
<td>Cat#GKI-4503</td>
</tr>
<tr>
<td>Glucose</td>
<td>FUJIFILM Wako</td>
<td>Cat# 049-31165CAS: 50-99-7</td>
</tr>
<tr>
<td>2-Deoxyglucose (2-DG)</td>
<td>FUJIFILM Wako</td>
<td>Cat# 040-06481CAS: 154-17-6</td>
</tr>
<tr>
<td>GDP-glucose (GDP-Glc)</td>
<td>Sigma-Aldrich</td>
<td>Cat# G7502CAS: 103301-72-0</td>
</tr>
<tr>
<td>GDP-mannose (GDP-Man)</td>
<td>Calbiochem</td>
<td>Cat# 067-04531CAS: 103301-73-1</td>
</tr>
<tr>
<td>GDP-fucose (GDP-Fuc)</td>
<td>Calbiochem</td>
<td>Cat# A117-50CAS: 15839-70-0</td>
</tr>
<tr>
<td>Uridine diphosphate-glucose (UDP-Glc)</td>
<td>Toyobo</td>
<td>YM7094CAS: 28053-08-9</td>
</tr>
<tr>
<td>UDP-galactose (UDP-Gal)</td>
<td>Toyobo</td>
<td>YM7213CAS: 137868-52-1</td>
</tr>
<tr>
<td>UDP-<i>N</i>-acetylglucosamine (UDP-GlcNAc)</td>
<td>Toyobo</td>
<td>YM7191CAS: 91183-98-1</td>
</tr>
<tr>
<td>UDP-<i>N</i>-acetylgalactosamine (UDP-GalNAc)</td>
<td>Toyobo</td>
<td>YM7958CAS: 108320-87-2</td>
</tr>
<tr>
<td>UDP-glucuronic acid (UDP-GlcA)</td>
<td>Toyobo</td>
<td>YM7095CAS: 63700-19-6</td>
</tr>
<tr>
<td>Cytidine monophosphate-neuraminic acid (CMP-NeuAc)</td>
<td>Toyobo</td>
<td>YM7215CAS: 1007117-62-5</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Mouse melanoma B16-F10</td>
<td>American Type Culture Collection</td>
<td>Cat# CRL-6475</td>
</tr>
<tr>
<td>Immortalized mouse embryonic fibroblasts (MEFs)</td>
<td>(<a href="#bib3">Harada et al., 2013</a>)</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Data Analysis version 3.1</td>
<td>Bruker Daltonics</td>
<td>n/a</td>
</tr>
<tr>
<td>Excel 2019</td>
<td>Microsoft</td>
<td>n/a</td>
</tr>
<tr>
<td>LabSolutions</td>
<td>Shimadzu</td>
<td>n/a</td>
</tr>
<tr>
<td>LabSolutions Insight LC-MS</td>
<td>Shimadzu</td>
<td>n/a</td>
</tr>
<tr>
<td>Plot2</td>
<td><a href="https://apps.micw.org/apps/plot2/index.php">https://apps.micw.org/apps/plot2/index.php</a></td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Millex-GV syringe filter unit, 0.22 μm, PVDF, 33 mm</td>
<td>Merk Millipore</td>
<td>Cat# SLGVR33RS</td>
</tr>
<tr>
<td>Envi-Carb column (250 mg) (used for nucleotide sugars)</td>
<td>Supelco Inc</td>
<td>Cat# NTCC-360/75-3-155</td>
</tr>
<tr>
<td>InertSep GC (150 mg/3 mL) (used for DLOs and POSs)</td>
<td>GL Science</td>
<td>Cat# 5010-68000</td>
</tr>
<tr>
<td>Sep-Pak Accell Plus QMA cartridge for POSs</td>
<td>Waters</td>
<td>Cat# WAT020545</td>
</tr>
<tr>
<td>MonoFas DNA purification kit I (used for PA-labeled glycans)</td>
<td>ANIMOS</td>
<td>Cat# A01-0002</td>
</tr>
<tr>
<td>Shodex NH2P-50 4E column (4.6 × 250 mm) for fluorescence HPLC</td>
<td>Shodex</td>
<td>Cat# F7630001</td>
</tr>
<tr>
<td>Inertsil ODS-3 column (3 μm, 150 × 2.1 mm) for LC-ESI-MS</td>
<td>GL Science</td>
<td>Cat# 5020-04415</td>
</tr>
<tr>
<td>0.3-mL PP Snap vial</td>
<td>GL Science</td>
<td>Cat# 1030-51053</td>
</tr>
<tr>
<td>Snap cap</td>
<td>GL Science</td>
<td>Cat# 1030-46250</td>
</tr>
<tr>
<td>Speed-Vac vacuum concentrators</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# DNA120-115</td>
</tr>
<tr>
<td>Bioruptor</td>
<td>Cosmo Bio</td>
<td>Cat# UCD-250</td>
</tr>
<tr>
<td>Handy sonic disruptor</td>
<td>Tomy Seiko</td>
<td>Cat# THU80</td>
</tr>
<tr>
<td>Pressure+</td>
<td>Biotage</td>
<td>Cat# PPM-96</td>
</tr>
<tr>
<td>Syringe without needle, luer lock tip, centered, 5 mL</td>
<td>Terumo</td>
<td>Cat# SS-05SZ</td>
</tr>
<tr>
<td>Glass test tube with screw cap, 9 mL</td>
<td>Corning</td>
<td>Cat# 9825-13</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Protocol for the preparation of reagents and equipment set up.</p>
<table id="undtbl2">
<caption>Complete DMEM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM, high glucose</td>
<td>n/a</td>
<td>450 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Add 2-DG at appropriate concentrations if necessary. Store at 4°C until use.</div>
<table id="undtbl3">
<caption>10× Phosphate-buffered saline for use in the biochemical experiments</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>1,370 mM</td>
<td>80 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>100 mM</td>
<td>14.2 g</td>
</tr>
<tr>
<td>KCl</td>
<td>27 mM</td>
<td>2.0 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>18 mM</td>
<td>2.4 g</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>Approx. 800 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Bring the volume up to 1 L with Milli-Q water. Store at 25°C.</div>
<div class="note">
<span class="note-title">Note:</span> Dilute 10× PBS with Milli-Q water to prepare 1× PBS as working solution for biochemical experiments. Store at 25°C.</div>
<table id="undtbl4">
<caption>500 mM glucose</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glucose</td>
<td>500 mM</td>
<td>900 mg</td>
</tr>
<tr>
<td>D-PBS (-)</td>
<td>n/a</td>
<td>Approx. 9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Completely dissolve the glucose and then bring the volume to 10 mL with D-PBS (-).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Filter the glucose solution through a Millex-GV syringe filter unit (0.22 μm, PVDF, 33 mm).</div>
<div class="note">
<span class="note-title">Note:</span> Store 1.5-mL aliquots of the stock solution in centrifuge tubes at −20°C.</div>
<table id="undtbl5">
<caption>Glucose-deprived DMEM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM without glucose</td>
<td>n/a</td>
<td>450 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>500 mM glucose in D-PBS (-)</td>
<td>0.5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at 4°C until use.</div>
<table id="undtbl6">
<caption>1 M Tris-HCl buffer (pH 7.4)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>1 M</td>
<td>121 g</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>Approx. 800 mL</td>
</tr>
<tr>
<td>HCl</td>
<td>n/a</td>
<td>Adjust pH to 7.4</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> While adjusting the pH to 7.4 with HCl, immerse the container in a water bath that is maintained at 30°C, since the addition of HCl generates heat, which also decreases the pH of the buffer. The lowest temperature that most water baths can maintain at is room temperature + 5°C (in most cases 30°C). Thus, investigators eventually will need to equilibrate the buffer at 25°C and check if the pH is 7.4 before bringing the volume to 1 L with Milli-Q water. If the pH increases, the pH should be adjusted to 7.4 with HCl. then bring the volume to 1 L with Milli-Q water. Store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: HCl is a volatile and corrosive irritant and should be handled in a fume hood and appropriate gloves and goggles should be worn.</div>
<table id="undtbl7">
<caption>5 M NaCl</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>5 M</td>
<td>29.22 g</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>Approx. 70 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Heat at 60°C while dissolving NaCl in water. Then bring the volume up to 100 mL with Milli-Q water. Store at 25°C.</div>
<table id="undtbl8">
<caption>10 mM Tris-HCl buffer (pH 7.4) containing 70 mM NaCl</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl (pH7.4)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>70 mM</td>
<td>1.4 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>97.6 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl9">
<caption>Cell lysis solution (70% (v/v) ethanol)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ethanol</td>
<td>70% (v/v)</td>
<td>70 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>30 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store at a temperature ranging from −20°C to −80°C before use.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Immerse the container in a cold bath that can be prepared by adding dry ice to ethanol.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Use ethanol (HPLC grade) for preparing the nucleotide sugars.</div>
<table id="undtbl10">
<caption>Chloroform/methanol (C/M; 2:1, v/v)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Chloroform</td>
<td>66.7% (v/v)</td>
<td>60 mL</td>
</tr>
<tr>
<td>Methanol</td>
<td>33.3% (v/v)</td>
<td>30 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>90 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare in a glass bottle just before use and store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: chloroform and methanol are volatile irritants. They should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl11">
<caption>8 mM MgCl<sub>2</sub>
</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>MgCl<sub>2</sub> hexahydrate</td>
<td>8 mM</td>
<td>81.3 mg</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>Approx. 40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Completely dissolve the powder and then bring the volume up to 50 mL with Milli-Q water. Prepare just before use and store at 25°C.</div>
<table id="undtbl12">
<caption>4 mM MgCl<sub>2</sub> in methanol/water (M/W; 1:1, v/v)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Methanol</td>
<td>50% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>8 mM MgCl<sub>2</sub> in Milli-Q water</td>
<td>4 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare in a glass bottle just before use and store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: methanol is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl13">
<caption>Chloroform/methanol/water (C/M/W; 10:10:3, v/v/v)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Chloroform</td>
<td>43.5% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Methanol</td>
<td>43.5% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>13% (v/v)</td>
<td>15 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>115 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare in a glass bottle just before use and store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: chloroform and methanol are volatile irritants. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl14">
<caption>40 mM HCl</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HCl</td>
<td>40 mM</td>
<td>0.033 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>9.967 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: HCl is a volatile irritant and corrosive. Aliquot in a fume hood and wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Store at 25°C and store at 25°C.</div>
<table id="undtbl15">
<caption>20 mM HCl in 50% (v/v) 2-propanol</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2-Propanol</td>
<td>50% (v/v)</td>
<td>5.0 mL</td>
</tr>
<tr>
<td>40 mM HCl</td>
<td>20 mM</td>
<td>5.0 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: HCl is a volatile irritant. Aliquot in a fume hood and wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl16">
<caption>95% (v/v) acetonitrile</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetonitrile</td>
<td>95% (v/v)</td>
<td>95 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetonitrile is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl17">
<caption>25% (v/v) acetonitrile</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetonitrile</td>
<td>25% (v/v)</td>
<td>25 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetonitrile is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl18">
<caption>200 mM Triethylamine acetate buffer (pH 7.0)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triethylamine</td>
<td>200 mM</td>
<td>14.4 mL</td>
</tr>
<tr>
<td>Acetate</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>Approximately 480 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Adjust pH to 7.0 with acetate and then bring the volume up to 500 mL with Milli-Q water. Prepare just before use and store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: triethylamine and acetate are volatile irritants. They should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl19">
<caption>50 mM Triethylamine acetate buffer (pH 7.0)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>200 mM Triethylamine acetate buffer (pH 7.0)</td>
<td>50 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl20">
<caption>80% (v/v) acetonitrile with 0.1% trifluoroacetate (TFA)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetonitrile</td>
<td>80% (v/v)</td>
<td>80 mL</td>
</tr>
<tr>
<td>TFA</td>
<td>0.1% (v/v)</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>19.9 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetonitrile and TFA are volatile irritants. TFA is also corrosive. They should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl21">
<caption>25% (v/v) acetonitrile in 50 mM Triethylamine acetate buffer (pH 7.0)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetonitrile</td>
<td>25% (v/v)</td>
<td>25 mL</td>
</tr>
<tr>
<td>200 mM triethylamine acetate buffer (pH 7.0)</td>
<td>50 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetonitrile is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl22">
<caption>2-Aminopyridine (PA) solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2-Aminopyridine</td>
<td>n/a</td>
<td>552 mg</td>
</tr>
<tr>
<td>Acetate</td>
<td>n/a</td>
<td>200 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Dissolve PA in acetate by heating at 80°C. The working solution should be stored at −20°C in the dark.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetate is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl23">
<caption>Dimethylamine-borane complex solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Dimethylamine-borane complex</td>
<td>n/a</td>
<td>100 mg</td>
</tr>
<tr>
<td>Acetate</td>
<td>n/a</td>
<td>200 μL</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: the working solution must be prepared just before use. Do not leave the tubes closed and discard the solution immediately after use, as hydrogen generates.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetate is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl24">
<caption>Eluent A for conventional HPLC</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid</td>
<td>0.3% (v/v)</td>
<td>3.0 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>66.28 mL</td>
</tr>
<tr>
<td>14% Ammonia</td>
<td>0.01% (v/v)</td>
<td>0.72 mL</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>93% (v/v)</td>
<td>930 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: ammonia and acetate are volatile irritants. They should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<table id="undtbl25">
<caption>Eluent B for conventional HPLC</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid</td>
<td>0.3% (v/v)</td>
<td>3.0 mL</td>
</tr>
<tr>
<td>Milli-Q water</td>
<td>n/a</td>
<td>Approx. 700 mL</td>
</tr>
<tr>
<td>14% Ammonia</td>
<td>n/a</td>
<td>Adjust pH to 7.0</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>20% (v/v)</td>
<td>200 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Adjust pH to 7.0 with ammonia. Carefully monitor the pH, as it steeply increases from around pH 6. Then bring the volume to 800 mL with Milli-Q water and mix with 200 mL of acetonitrile.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: ammonia and acetate are volatile irritants. They should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl26">
<caption>GDP-glucose stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>GDP-glucose</td>
<td>4 mM</td>
<td>2.4 mg</td>
</tr>
<tr>
<td>Distilled water</td>
<td>n/a</td>
<td>1.0 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1.0 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store 0.01-mL aliquots of the stock solution in centrifuge tubes at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> Dilute the stock solution with distilled water to 0.5 mM before use.</div>
<table id="undtbl27">
<caption>Nucleotide sugar stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>UDP-GlcNAc/GalNAcUDP-Glc/GalGDP-ManGDP-FucUDP-GlcACMP-NeuAc</td>
<td>4 mM4 mM4 mM4 mM4 mM4 mM</td>
<td>2.60 mg2.44 mg2.60 mg2.53 mg2.58 mg2.55 mg</td>
</tr>
<tr>
<td>Distilled water</td>
<td>n/a</td>
<td>1.0 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1.0 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Store 0.01-mL aliquots of the stock solution in centrifuge tubes at −20°C.</div>
<h3 id="sec2.1">Bioruptor setting</h3>
<p>In order to increase temperature in disruption unit, program the digital timer for the following conditions; ON/OFF pulse time: 30 s; and total time: 5 min.</p>
<table id="undtbl28">
<caption>Eluent C for LC-MS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triethylamine</td>
<td>200 mM</td>
<td>14.4 mL</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>0.5%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>n/a</td>
<td>Approximately 480 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<div class="note">
<span class="note-title">Note:</span> Carefully adjust pH of the solution to 6.0 by adding 1 M acetic acid, and bring the volume up to 500 mL with distilled water.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: triethylamine and acetate are volatile irritants. They should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
<table id="undtbl29">
<caption>Eluent D for LC-MS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>HPLC eluent C</td>
<td>80% (v/v)</td>
<td>160 mL</td>
</tr>
<tr>
<td>Acetonitrile</td>
<td>20% (v/v)</td>
<td>40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Prepare just before use and store at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: acetonitrile is a volatile irritant. It should be handled in a fume hood and investigators should wear appropriate gloves and goggles.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Cell counting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Protocol for cell counting to normalize the amounts of DLOs, POSs, and nucleotide sugars.</p>
<div class="note">
<span class="note-title">Note:</span> Cell counting should be performed at which cells are harvested for preparation of DLOs, POSs, and nucleotide sugars.</div>
<ol>
<li>Aspirate culture medium from a ϕ100 mm culture dish.</li>
<li>Wash the cells once with 5 mL of D-PBS (-).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> D-PBS (-) should not contain calcium chloride or magnesium chloride, since these inorganic salts inhibit the action of trypsin.</div>
<ol start="3">
<li>Add 1 mL of 0.05% (w/v) trypsin-0.53 mM EDTA to dish.</li>
<li>Incubate cells for 1–2 min at 37°C in a CO<sub>2</sub> incubator.</li>
<li>Tap the lateral side of the dish to detach cells.</li>
<li>Add 9 mL of the complete medium.</li>
<li>Pipette up and down to dissociate cells.</li>
<li>Transfer the cell suspension to a fresh 15-mL centrifuge tube.</li>
<li>Mix 10 μL of the cell suspension and 10 μL of 0.4% (w/v) trypan blue dye in a 1.5-mL centrifuge tube.</li>
<li>Count the live and dead cells using a hemocytometer.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Since necrotic cells are no longer metabolically active, these cells should be minimized in the cell preparation in order to obtain reliable results. For reference, we use cell cultures for the extraction of DLOs, POSs, and nucleotide sugars only when the number of trypan blue-positive necrotic cells is less than 10% of the total cell numbers.</div>
<h3 id="sec3.2">Preparation of cell homogenates for extraction of DLOs and POSs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>This protocol allows for the separation of adherent mammalian cells into ethanol-insoluble fraction, which contains DLOs, and ethanol-soluble fraction, which contains POSs (<a href="#fig2">Figure 2</a>). Handling time should be minimized (as short as possible), because of the relatively labile nature of DLOs and POSs.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. A schematic diagram showing the preparation of cell homogenates for extraction of DLOs and POSs</div>
</figcaption>
</figure>
<ol start="11">
<li>Aspirate the culture medium from a ϕ100 mm culture dish.</li>
<li>Wash the cells two times with 5 mL each of 1× PBS that was warmed up at 25°C.</li>
<li>Aspirate the residual PBS completely.</li>
<li>Place the dish on ice.</li>
<li>Remove the cell lysis solution (70% [v/v] ethanol) from a freezer or a dry ice-ethanol bath, and immediately add 2 mL of the lysis solution to the dish.</li>
<li>Scrape cells with a scraper on ice and transfer the homogenate into a fresh 15-mL centrifuge tube by pipetting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The end of the tips may be cut with clean scissors, as cells form large aggregates upon scraping in 70% ethanol.</div>
<ol start="17">
<li>Repeat steps 15 and 16 one more time to maximize yields.</li>
<li>Vortex the homogenate (∼ 4 mL) for 10 s and incubate for 15 min at 0°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This incubation time should be accurate.</div>
<ol start="19">
<li>Centrifuge the homogenate at 10,000 × <i>g</i> for 10 min at 4°C.</li>
<li>Transfer the supernatant (∼ 4 mL) to a fresh 15-mL centrifuge tube by pipetting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The supernatant contains POSs.</div>
<ol start="21">
<li>Keep the remaining cell pellet for the extraction of DLOs (see next section).</li>
<li>Evaporate the supernatant to dryness using a Speed-Vac or under a nitrogen stream.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Pellets and dried samples can be stored at −20°C for at least 1 week.</div>
<h3 id="sec3.3">Preparation of DLOs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 h</div>
<p>Protocol for the extraction and mild acid hydrolysis of DLOs, followed by desalting of the released free glycans (<a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Preparation of DLOs</div>
<p>(A) A schematic diagram showing the extraction of DLOs and liberation of the glycan moiety (free glycans).</p>
<p>(B) A simplified protocol for DLO extraction (steps 1–4), mild acid hydrolysis of DLOs and desalting of the resulting free glycans (step 5).</p>
</figcaption>
</figure>
<ol start="23">
<li>Extraction of DLOs (<a href="#fig3">Figure 3</a>B, steps 1–4).
<div class="critical">
<span class="critical-title">Critical:</span> All centrifugation steps must be performed at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> A swing rotor must be used for all centrifugation steps.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not chill the organic solvents for DLO extraction. Keep them at 25°C.</div>
<ol type="a">
<li>Add 2 mL of methanol to the cell pellets that were prepared at step 21.</li>
<li>Vortex vigorously until the cell pellets become uniformly dispersed.
<div class="note">
<span class="note-title">Note:</span> A probe sonicator can be used to disperse cell pellets in methanol if the pellet forms hard aggregates and are difficult to disperse.</div>
</li>
<li>Centrifuge at 1,800 × <i>g</i> for 5 min at 25°C.</li>
<li>Discard the supernatant by decantation.</li>
<li>Repeat steps a–d one more time.</li>
<li>Add 2 mL of chloroform/methanol (C/M; 2:1, v/v) to the pellets.</li>
<li>Vortex vigorously until the cell pellets become uniformly dispersed.</li>
<li>Centrifuge at 1,800 × <i>g</i> for 5 min at 25°C.</li>
<li>Discard the supernatant by decantation.</li>
<li>Repeat steps f–i one more time.</li>
<li>Dry the pellets under a nitrogen stream.
<div class="critical">
<span class="critical-title">Critical:</span> It is critical to remove as much of the chloroform as possible, because chloroform, if much remains in the samples, it interferes with the extraction of DLOs in the next step. However, do not overdry the samples, as the pellets will become fluffy and can blow away.</div>
</li>
<li>Add 2 mL of 4 mM MgCl<sub>2</sub> in methanol/water (M/W; 1:1, v/v) to the pellet.
<div class="critical">
<span class="critical-title">Critical:</span> MgCl<sub>2</sub> in this solution is essential to extract DLOs from the pellet.</div>
</li>
<li>Vortex vigorously until the cell pellet becomes uniformly dispersed.</li>
<li>Centrifuge at 1,800 × <i>g</i> for 5 min at 25°C.</li>
<li>Discard the supernatant completely by decantation.
<div class="critical">
<span class="critical-title">Critical:</span> It is critical to remove the solution as much as possible. If much remains, it results in the separation of an aqueous-organic solvent phase in the next step.</div>
</li>
<li>Repeat steps l–o one more time.</li>
<li>Add 2 mL of chloroform/methanol/water (C/M/W; 10:10:3, v/v/v) to the pellet.</li>
<li>Vortex vigorously until the cell pellets become uniformly dispersed.</li>
<li>Close the cap of tubes and incubate them for 10 min at 37°C.</li>
<li>Centrifuge at 1,800 × <i>g</i> for 5 min at 25°C.</li>
<li>Transfer the supernatant to a fresh 9-mL glass test tube with a screw cap by using a glass Pasteur pipette with a nipple.
<div class="critical">
<span class="critical-title">Critical:</span> Caution: the supernatant contains the extracted DLOs.</div>
</li>
<li>Repeat steps q–t for the remaining pellets one more time.</li>
<li>Transfer the second supernatant to the glass tube that contains the first supernatant.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The combined supernatant can be stored at −20°C for at least 1 week.</div>
</li>
<li>Evaporate the combined supernatant to dryness using a Speed-Vac or under a nitrogen stream.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The dried DLO pellets can be stored at −20°C at least for 1 week.</div>
</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Mild acid hydrolysis of DLOs (<a href="#fig3">Figure 3</a>B, step 5).
<ol type="a">
<li>Add 1 mL of 20 mM HCl in 50% (v/v) 2-propanol to the dried DLO pellets.</li>
<li>Close the cap, briefly vortex, and allow the tubes to stand on a metal rack.</li>
<li>Heat the tubes for 30 min at 100°C in an oven.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A heat block can also be used.</div>
</li>
<li>Chill the tubes in a rack by immersing them into ice-water bath for 3 min.
<div class="critical">
<span class="critical-title">Critical:</span> Caution: tubes and racks are extremely hot. Wear appropriate gloves.</div>
</li>
<li>Evaporate the hydrolysates to dryness using a Speed-Vac or under a nitrogen stream.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: after evaporating HCl, the Speed-Vac should be washed by evaporating 100 mL of water.</div>
</li>
<li>Add 1 mL of Milli-Q water to the dried pellets.</li>
<li>Resuspend the released oligosaccharides by vigorous vortexing for 10 s.</li>
<li>Centrifuge the tubes at 1,800 × <i>g</i> for 5 min at 25°C.</li>
<li>Transfer the supernatant to a fresh 15-mL centrifuge tube.
<div class="note">
<span class="note-title">Note:</span> The supernatant contains the released oligosaccharides.</div>
</li>
<li>Repeat steps f–h one more time.</li>
<li>Transfer the second supernatant to the tube that contains the first supernatant.</li>
</ol>
</li>
</ol>
<ol start="25">
<li>Desalting of the released free glycans (<a href="#fig3">Figure 3</a>B, step 5).
<ol type="a">
<li>Set fresh 15-mL centrifuge tubes on a tube rack (1 tube/each sample).</li>
<li>Place an InertSep GC column (1 column/each sample) in the 15-mL centrifuge tube.</li>
<li>Add 2.5 mL of 100% acetonitrile to the top of the column.
<div class="note">
<span class="note-title">Note:</span> Activation step.</div>
</li>
<li>Let the solution pass through the column and discard the eluate.</li>
<li>Add 2.5 mL of Milli-Q water to the top of the column.
<div class="note">
<span class="note-title">Note:</span> Wash step.</div>
</li>
<li>Let the solution pass through the column and discard the eluate.</li>
<li>Repeat steps e and f one more time.</li>
<li>Add the combined supernatant (approximately 2 mL) obtained in step 24k to the top of the activated InertSep GC column.
<div class="note">
<span class="note-title">Note:</span> Binding step.</div>
</li>
<li>Let the sample completely enter the column and discard the eluate.</li>
<li>Add 2.5 mL of Milli-Q water to the top of the column.
<div class="note">
<span class="note-title">Note:</span> Wash step.</div>
</li>
<li>Let water pass through the column and discard the eluate.</li>
<li>Repeat steps j and k one more time.</li>
<li>Place fresh 15-mL centrifuge tubes to a tube rack.</li>
<li>Wipe the outside of the InertSep GC columns with clean KimWipes and place the columns in fresh 15-mL centrifuge tubes.</li>
<li>Add 2.5 mL of 25% (v/v) acetonitrile to the top of the column.
<div class="note">
<span class="note-title">Note:</span> Elution step.</div>
</li>
<li>Collect the eluate (2.5 mL).
<div class="note">
<span class="note-title">Note:</span> The eluate contains free glycans released from DLOs.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The samples can be stored at −20°C for at least 1 week.</div>
</li>
<li>Concentrate the eluate obtained in step p to ∼0.5 mL using a Speed-Vac or under nitrogen stream.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.” If the eluate is completely dried, add 500 μL of water, vortex, spin down, and proceed to step s.</div>
</li>
<li>Vortex for 10 s and spin down.</li>
<li>Transfer the solution to a fresh 1.5-mL centrifuge tube.</li>
<li>Dry the solution completely using a Speed-Vac or under a nitrogen stream.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The dried pellets can be stored at −20°C for at least 1 week.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Residual water increases the epimerization of the innermost GlcNAc to ManNAc of free glycans during the fluorescent labeling with PA.</div>
<h3 id="sec3.4">Preparation of POSs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Protocol for separation of POSs from neutral free glycans, dephosphorylation of POSs, and desalting of the dephosphorylated free glycans (<a href="#fig4">Figure 4</a>A).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Preparation of POSs</div>
<p>(A) A schematic diagram showing the purification of POSs and liberation of the glycan moiety (free glycans).</p>
<p>(B) A simplified protocol for size-exclusion chromatography (step 1), anion-exchange chromatography (step 2), dephosphorylation of POSs, and desalting of the resulting free glycans (step 3). SEC, size-exclusion chromatography; AP, alkaline phosphatase.</p>
</figcaption>
</figure>
<ol start="26">
<li>Equilibration of size-exclusion columns (SEC, 1 column for each sample).
<ol type="a">
<li>Open the top lid of the PD MidiTrap G-25 column, discard the storage buffer, and remove the bottom cap.</li>
<li>Stand the columns on the attached rack and place a waste container below the columns.</li>
<li>Fill the top space of the column with Milli-Q water.</li>
<li>Let the water pass through the column.</li>
<li>Repeat steps c and d two more times.</li>
<li>Cap the bottom end of the column and close the top lid until use.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Equilibrate the column just before use.</div>
<ol start="27">
<li>Equilibration of Sep-Pak Accell Plus QMA cartridge (1 cartridge for each sample).
<ol type="a">
<li>Remove the piston from a 5-mL syringe.</li>
<li>Connect the outlet of the syringe to the inlet of a QMA cartridge.</li>
<li>Add 5 mL of 10 mM Tris-HCl (pH 7.4) to the syringe.</li>
<li>Place a piston and push it to elute buffer at a rate of approximately 2 drops/s.</li>
<li>Dry the cartridge by blowing out air from a syringe.</li>
<li>Disconnect the syringe and remove the piston.</li>
<li>Connect the syringe outlet to the cartridge and store at room temperature until use.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Equilibrate the cartridge just before use.</div>
<ol start="28">
<li>Add 1.0 mL of Milli-Q water to the dried cell extract obtained in step 22 (<a href="#fig4">Figure 4</a>B, step 1).</li>
<li>Vortex vigorously until the pellet is completely dispersed.</li>
<li>Transfer the suspension to a fresh 1.5-mL centrifugation tube.</li>
<li>Centrifuge at 15,000 × <i>g</i> for 3 min at 25°C.</li>
</ol>
<ol start="32">
<li>Desalting by size-exclusion chromatography (<a href="#fig4">Figure 4</a>B, step 1).
<ol type="a">
<li>Add the supernatant (exactly 1 mL) to the top of PD MidiTrap G-25 columns that have been equilibrated with water (see step 26).
<div class="note">
<span class="note-title">Note:</span> Make sure that water is completely removed from top space of the column before adding the sample.</div>
</li>
<li>Let the sample enter into the column and discard the eluate.</li>
<li>Place a 2.0-mL centrifuge tube just beneath the column outlet.</li>
<li>Add 1.5 mL of water to the top of the column.</li>
<li>Collect the eluate (1.5 mL).
<div class="note">
<span class="note-title">Note:</span> This eluate contains POSs.</div>
</li>
</ol>
</li>
</ol>
<ol start="33">
<li>Add 15 μL of 1 M Tris-HCl (pH 7.4) to the eluate and mix thoroughly (<a href="#fig4">Figure 4</a>B, step 2).</li>
</ol>
<ol start="34">
<li>Anion-exchange chromatography (<a href="#fig4">Figure 4</a>B, step 2).
<div class="note">
<span class="note-title">Note:</span> The objective of this step is to separate the POSs from other neutral materials (e.g., neutral free glycans).</div>
<ol type="a">
<li>Add the eluate (1.5 mL) to the syringe that was connected to Sep-Pak Accell Plus QMA cartridge (see step 27).
<div class="note">
<span class="note-title">Note:</span> Binding step of POSs.</div>
</li>
<li>Place a piston and push at a rate of approximately 2 drops/s.</li>
<li>Discard the flow-through.</li>
<li>Dry the cartridge by blowing out air from a syringe.</li>
<li>Disconnect the syringe from QMA cartridge and remove the piston.</li>
<li>Connect the same syringe to the QMA cartridge.</li>
<li>Add 5 mL of 10 mM Tris-HCl (pH 7.4) to the syringe.
<div class="note">
<span class="note-title">Note:</span> Wash step.</div>
</li>
<li>Place a piston and push at a rate of approximately 2 drops/s.</li>
<li>Discard the flow-through.</li>
<li>Dry the cartridge by blowing out air from a syringe.</li>
<li>Disconnect the syringe from QMA cartridge and remove the piston.</li>
<li>Repeat steps from g to k one more time.</li>
<li>Connect a fresh 5-mL syringe without a piston to the QMA cartridge.</li>
<li>Stand fresh 15-mL centrifuge tubes in a rack.</li>
<li>Add 5 mL of 10 mM Tris-HCl (pH 7.4) containing 70 mM NaCl to the syringe.
<div class="note">
<span class="note-title">Note:</span> Elution step.</div>
</li>
<li>Place a piston and push at a rate of approximately 2 drops/s.</li>
<li>Collect the eluate in a 15-mL centrifuge tube.</li>
<li>Mix the eluate by vortexing and spin down.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The samples can be stored at −20°C for at least 2–3 days.</div>
<ol start="35">
<li>Dephosphorylation of POSs (<a href="#fig4">Figure 4</a>B, step 3).
<ol type="a">
<li>Divide the eluate (5 mL) obtained in step 34r equally to two portions (2.5 mL each).</li>
<li>Add 5 μL of rAPid alkaline phosphatase to one half and mix thoroughly. The other half is left untreated with phosphatase. Spin both down.</li>
<li>Incubate the samples with or without phosphatase for 16 h at 37°C.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The reaction mixtures can be stored at −20°C for at least 2–3 days.</div>
<ol start="36">
<li>Desalting of dephosphorylated free glycans (<a href="#fig4">Figure 4</a>B, step 3).
<ol type="a">
<li>Set fresh 15-mL centrifuge tubes on a tube rack (1 tube/each sample).</li>
<li>Place an InertSep GC column (1 column/each sample) in the 15-mL centrifuge tube.</li>
<li>Add 2.5 mL of 100% acetonitrile to the top space of the column.
<div class="note">
<span class="note-title">Note:</span> Activation step.</div>
</li>
<li>Let the solution pass through the column and discard the eluate.</li>
<li>Add 2.5 mL of Milli-Q water to the top space of the column.
<div class="note">
<span class="note-title">Note:</span> Wash step.</div>
</li>
<li>Let the solution pass through the column and discard the eluate.</li>
<li>Repeat steps e and f one more time.</li>
<li>Add the reaction mixtures (2.5 mL) obtained in step 35c to the activated InertSep GC column.
<div class="note">
<span class="note-title">Note:</span> Binding step.</div>
</li>
<li>Let the samples enter the column and discard the eluate.</li>
<li>Add 2.5 mL of Milli-Q water to the top space of the column.
<div class="note">
<span class="note-title">Note:</span> Wash step.</div>
</li>
<li>Let the solution pass through the column and discard the eluate.</li>
<li>Repeat steps j and k one more time.</li>
<li>Place fresh 15-mL centrifuge tubes in a tube rack.</li>
<li>Wipe the outside of InertSep GC columns with clean KimWipes and place the columns into fresh 15-mL centrifuge tubes.</li>
<li>Add 2.5 mL of 25% (v/v) acetonitrile to the top space of the column.
<div class="note">
<span class="note-title">Note:</span> Elution step.</div>
</li>
<li>Collect the eluate (2.5 mL).
<div class="note">
<span class="note-title">Note:</span> Only dephosphorylated POSs (free glycans) are eluted from the column at this step. POSs are retained in the column.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The samples can be stored at −20°C for at least 1 week.</div>
</li>
<li>Concentrate the eluate obtained at step 36p to ∼0.5 mL by using Speed-Vac or under nitrogen stream.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.” If the eluate is completely dried, add 500 μL of water, vortex, spin down, and proceed to step s.</div>
</li>
<li>Vortex well and spin down.</li>
<li>Transfer the solution to a fresh 1.5-mL centrifuge tube.</li>
<li>Dry the solution completely by using Speed-Vac or under a nitrogen stream.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The dried samples can be stored at −20°C for at least 1 week.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Residual water increases epimerization of innermost GlcNAc to ManNAc of free glycans during the fluorescent labeling with PA.</div>
<h3 id="sec3.5">Fluorescent labeling of glycans with PA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h for labeling and 30 min for purification</div>
<p>The protocol for the fluorescent labeling of free glycans with PA and purification of the PA-labeled glycans (<a href="#fig5">Figure 5</a>A).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Fluorescent labeling of free glycans and the HPLC analysis</div>
<p>(A) A simplified protocol for the fluorescent labeling of free glycans with 2-aminopyridine (PA) (step 1) and purification of the PA-labeled glycans (step 2).</p>
<p>(B) HPLC setting for analysis of PA-labeled glycans. DAB, dimethylamine-borane complex.</p>
</figcaption>
</figure>
<ol start="37">
<li>PA labeling (<a href="#fig5">Figure 5</a>A, step 1).
<ol type="a">
<li>Dissolve the PA in the solution by heating at 80°C.</li>
<li>Vortex and spin down.</li>
<li>Immediately pipette out 20 μL of PA solution and add it to the dried free glycan preparations obtained in steps 25t (DLOs) or 36t (POSs).
<div class="critical">
<span class="critical-title">Critical:</span> Caution: PA solution is easily solidified.</div>
</li>
<li>Vortex for 10 s and spin down.</li>
<li>Incubate for 1 h at 80°C in the dark.
<div class="note">
<span class="note-title">Note:</span> Use a heat block covered with aluminum foil.</div>
</li>
<li>Add 20 μL of the dimethylamine-borane complex solution to the reaction mixture.
<div class="note">
<span class="note-title">Note:</span> Prepare the dimethylamine-borane complex solution just before use.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Caution: discard the solution immediately after use.</div>
</li>
<li>Mix the reaction mixture thoroughly by vortexing and spin down.</li>
<li>Incubate for 1 h at 80°C in the dark.
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The reaction mixture can be stored at −20°C for 2–3 days.</div>
</li>
<li>Add 960 μL of acetonitrile to the reaction mixture.</li>
<li>Mix the reaction mixture thoroughly by vortexing and spin down.</li>
</ol>
</li>
</ol>
<ol start="38">
<li>Activation of MonoFas spin columns.
<ol type="a">
<li>Add 200 μL of Milli-Q water to the MonoFas spin column.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s and discard the eluate.</li>
<li>Add 700 μL of 100% acetonitrile to the spin column.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s and discard the eluate.</li>
<li>Repeat c and d one more time.</li>
</ol>
</li>
<li>Purification of PA-labeled glycans (<a href="#fig5">Figure 5</a>A, step 2).
<ol type="a">
<li>Add 700 μL of a PA labeling reaction mixture to the activated MonoFas spin column.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s and discard the eluate.</li>
<li>Add the remaining PA labeling reaction mixture to the same spin column.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s and discard the eluate.</li>
<li>Add 700 μL of 95% (v/v) acetonitrile to the spin column.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s and discard the eluate.</li>
<li>Repeat steps e and f one more time.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s to remove the remaining acetonitrile solution.</li>
<li>Remove the spin column from tube and place onto a fresh 1.5-mL centrifuge tube.</li>
<li>Add 100 μL of Milli-Q water to the spin column.</li>
<li>Centrifuge at 10,000 × <i>g</i> for 30 s.</li>
<li>Repeat steps j and k one more time in the same tube.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The combined eluates (∼200 μL) contain PA-labeled glycans. The eluate can be concentrated using a Speed-Vac.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The combined eluates can be stored at −20°C for several month to several years.</div>
<h3 id="sec3.6">Size-fractionation HPLC analysis of PA-labeled glycans</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h for equilibration and 1 additional h per sample</div>
<p>Protocol for separation and quantification of PA-labeled glycans by size-fractionation HPLC (<a href="#fig5">Figure 5</a>B).</p>
<ol start="40">
<li>Start the HPLC system, connect Shodex NH2P-50 4E column, and set the gradient program (see the <a href="#materials-and-equipment">Materials and equipment</a> section “HPLC settings for the detection of PA-labeled glycans”).</li>
<li>Equilibrate the column for 20 min by running a mixed eluent (97% HPLC eluent A and 3% HPLC eluent B; see the <a href="#materials-and-equipment">Materials and equipment</a> section for recipes of these HPLC eluents).</li>
<li>Add 10 μL of samples to the bottom of snap vials.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Take great care not to leave any air between sample solution and the bottom of vials.</div>
<ol start="43">
<li>Seal the vial with a snap cap.</li>
<li>Set the vial to the auto-sampler.</li>
<li>Analyze the samples in the sequence in the following order: Milli-Q water (10 μL), standard PA-labeled glycans with known concentrations (e.g., PA-glucose oligomers; 2 pmol as PA-labeled glucose hexamer), and your PA-labeled glycan samples.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Standard PA-labeled glycans are used as an external standard to standardize the elution positions and to quantitate the PA-labeled glycans in the samples.</div>
<div class="note">
<span class="note-title">Note:</span> Collect peaks from the waste line of the HPLC if PA-labeled glycans are further analyzed by matrix-assisted laser desorption/ionization (MALDI)-time-of-flight (TOF)-mass (MS) analysis.</div>
<ol start="46">
<li>To shut down HPLC, replace the HPLC lines with 20% (v/v) acetonitrile.</li>
<li>Wash the HPLC column with the storage solvent (20% (v/v) acetonitrile) for 20 min at a flow rate of 0.8 mL/min.</li>
<li>Disconnect the HPLC column, cap both ends, and store at room temperature.</li>
<li>Connect the outlet of the injector and the inlet of the detector with a plug to prevent the HPLC lines from drying out.</li>
</ol>
<h3 id="sec3.7">Preparation of cellular nucleotide sugars</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h for the preparation of cell homogenates and 6 h for the purification of nucleotide sugars</div>
<p>Protocol for the preparation of cell homogenates and purification of nucleotide sugars (<a href="#fig6">Figure 6</a>A).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Preparation and LS-ESI-MS analysis of cellular nucleotide sugars</div>
<p>(A) A simplified scheme for cell lysis (step 1) and the purification of nucleotide sugars (step 2).</p>
<p>(B) HPLC and ESI-MS setting for nucleotide sugar analysis.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Due to the relatively labile nature of nucleotide sugars, it is important to perform the extraction rapidly. In particular, the centrifugation step should be avoided when collecting cell pellets.</div>
<ol start="50">
<li>Preparation of cell homogenates (<a href="#fig6">Figure 6</a>A, step 1).
<ol type="a">
<li>Culture the cells in ϕ100 mm dishes (see the “<a href="#sec1.1">Cell culture</a>” of the “<a href="#before-you-begin">Before you begin</a>”).</li>
<li>Wash the cells once with 5 mL of ice-cold PBS.</li>
<li>Aspirate the residual PBS completely.</li>
<li>Add 500 μL of ice-cold 70% (v/v) ethanol to the dish.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The dish may be placed on dry ice.</div>
</li>
<li>Scrape the cells and pipette in and out three times to disperse cell aggregates.</li>
<li>Transfer the cell suspension (∼500 μL) into a fresh 1.5-mL centrifuge tube.</li>
<li>Homogenize the cell suspension using a Bioruptor for 5 min at 0°C.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The cell suspension may be homogenized by using the Handy Sonic Disruptor for 1 min on ice.</div>
</li>
<li>Add 5 μL of unnatural 0.5 mM GDP-Glc (2.5 nmol) to the cell suspension.
<div class="note">
<span class="note-title">Note:</span> GDP-Glc is added to the samples as an internal standard to calculate yields of the preparation.</div>
</li>
<li>Centrifuge the cell suspension at 16,000 × <i>g</i> for 15 min at 4°C.</li>
<li>Transfer the supernatant to a 1.5-mL centrifuge tube.</li>
<li>Freeze the supernatant at −20°C and evaporate by using a Speed-Vac concentrator.
<div class="note">
<span class="note-title">Note:</span> The temperature of the centrifugal part of the Speed-Vac should be set to “Ambient.”</div>
</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried cellular extracts can be stored at −20°C for several months.</div>
<ol start="51">
<li>Purification of nucleotide sugars (<a href="#fig6">Figure 6</a>A, step 2).
<ol type="a">
<li>Activate ENVI-Carb columns by adding 2 mL of 80% (v/v) acetonitrile in 0.1% (v/v) TFA.
<div class="critical">
<span class="critical-title">Critical:</span> Caution: do not allow the ENVI-Carb columns to dry out after this activation step.</div>
</li>
<li>Wash the column with 1 mL of distilled water.</li>
<li>Dissolve the freeze-dried samples in 1 mL of 10 mM NH<sub>4</sub>HCO<sub>3</sub>.</li>
<li>Apply the sample (1 mL) to the activated ENVI-Carb column.</li>
<li>Sequentially wash the column once with 1 mL of water, 1 mL of 25% (v/v) acetonitrile, 200 μL of water, and 1 mL of 50 mM triethylamine acetate buffer (pH 7.0).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Use the Pressure+ vacuum to speed up these elution processes.</div>
</li>
<li>Elute the nucleotide sugars from the column with 1 mL of 25% (v/v) acetonitrile in 50 mM triethylamine acetate buffer (pH 7.0).</li>
<li>Evaporate the eluates to dryness using a Speed-Vac concentrator.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Dried nucleotide sugars can be stored at −20°C for several months.</div>
<h3 id="sec3.8">Ion-pair reversed-phase LC ESI-MS for simultaneous quantification of nucleotide sugars</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Protocol for the separation, identification and quantification of nucleotide sugars by ion-pair reversed-phase LC ESI-MS/MS (<a href="#fig6">Figure 6</a>B).</p>
<ol start="52">
<li>Set the MS conditions as described in the “<a href="#before-you-begin">Before you begin</a>” section under “<a href="#sec1.6">Mass spectrometer settings</a>.”</li>
<li>Program the <i>m/z</i> values for precursor ions and product ions.</li>
<li>For the quantification of each nucleotide sugar, first run authentic standard mixtures containing 10 pmol/each, and confirm that they produce a robust peak on the LC-MS system.</li>
<li>Prepare the calibration curve on the same run on the day as an LC/MS run, to quantify nucleotide sugars. Analyze the diluted authentic standards at the following concentrations: 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2.5, 5, 10, 25, and 50 μM.</li>
<li>Dissolve the extracted nucleotide sugars in distilled water (50 μL), and apply aliquots (10 μL) for LC-MS analysis.</li>
<li>Extract the peak areas of nucleotide sugars in the extraction ion chromatograms.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Alternative Ion-pair reversed-phase conventional HPLC</div>
<p>UDP-GalNAc and UDP-GlcNAc were co-eluted in the LC-MS analysis. To quantify of those, it is recommended to use a conventional HPLC system without MS as follows.</p>
<p>The method for simultaneous determination of nucleotide sugars using conventional HPLC coupled with UV detection has been previously reported (<a href="#bib9">Nakajima et al., 2010</a>). The use of a non-volatile tetrabutylammonium hydrogen sulfate is crucial for complete separation, however the ion-pair reagent is incompatible with MS detection.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Adherent mammalian cells are separated into soluble (POSs, nucleotide sugars and neutral free glycans) and insoluble (DLOs) fractions by aqueous ethanol extraction, followed by a centrifugation step. DLOs are comprised of a glycan chain linked to a dolichol lipid through a pyrophosphate group. The glycan chain can be released from the pyrophosphate dolichol by mild acid hydrolysis (<a href="#fig3">Figure 3</a>), which exposes the reducing end of the glycan for PA labeling. As shown in size-fractionation HPLC analysis (<a href="#fig7">Figure 7</a>A), the immortalized mouse embryonic fibroblasts (MEFs) that were cultured in a glucose-rich medium exclusively accumulate the fully assembled DLO, which contains three Glc, nine Man, and two GlcNAc residues. PA-labeled glucose oligomers (degree of polymerization [DP]: 3–15) are used to standardize the elution position (<a href="#fig7">Figure 7</a>B). The DP of glucose oligomers is expressed as glucose units.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Size-fractionation HPLC profiles of PA-labeled glycans derived from DLOs and PA-labeled glucose oligomers</div>
<p>(A) A representative HPLC chart of PA-labeled glycans derived from DLOs in mouse embryonic fibroblasts (MEFs), which were cultured in the presence of 5 mM glucose.</p>
<p>(B) A representative HPLC chart of PA-labeled glucose oligomers (degree of polymelization [DP] = 3-15; Takara, discontinued). The DP numbers are expressed as glucose units. Asterisk indicates a peak derived from labeling reagents.</p>
</figcaption>
</figure>
<p>Separating POSs from neutral materials (e.g., neutral free glycans from the cytosol and lysosomes) by anion-exchange chromatography is necessary to distinguish glycans derived from POSs and the preexisting neutral free glycans. POSs contain a single phosphate group, capping the reducing end of the glycan. Thus, the phosphate groups of POSs need to be removed before PA labeling (<a href="#fig4">Figure 4</a>A). POS preparations are also processed without dephosphorylation, allowing for the specific detection of POSs and ruling out contamination of neutral free glycans. As shown in the size-fractionation HPLC analysis (<a href="#fig8">Figure 8</a>A), in the case of MEFs cultured in a glucose-rich medium, only very small amounts of POSs (i.e., Man<sub>5</sub>GlcNAc<sub>2</sub>-P) accumulate. In contrast, in the case of glucose-deprived MEFs, large amounts of POSs containing short glycan chains (e.g., Man<sub>0–7</sub>GlcNAc<sub>2</sub>-P) accumulate (<a href="#fig8">Figure 8</a>B). These glycan structures are identical to those of the biosynthetic intermediates of DLOs (<a href="#bib2">Harada et al., 2016</a>) (<a href="#fig1">Figure 1</a>). The glycan structure of POSs depends on the cell line, but they are generally Man<sub>0–8</sub>GlcNAc<sub>2</sub> (<a href="#bib3">Harada et al., 2013</a>; <a href="#bib4">Harada et al., 2020</a>). In the case of Mouse B16-F10 cells cultured in a medium containing 2-DG, significant amounts of POSs containing Man, 2-DG and GlcNAc residues accumulate (<a href="#fig8">Figure 8</a>C) (<a href="#bib4">Harada et al., 2020</a>). Almost no peaks, except for those from labeling reagents, were detected when POSs are processed and fluorescently labeled without dephosphorylation, confirming that the neutral free glycans are removed from the POS preparations.</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Size-fractionation analysis of PA-labeled glycans derived from POSs</div>
<p>(A and B) Representative HPLC charts of PA-labeled glycans derived from POSs in MEFs, which were cultured in the presence of 5 mM glucose (A) or 0.5 mM glucose (B).</p>
<p>(C) A representative HPLC chart of PA-labeled glycans derived from POSs in B16-F10 cells, which were cultured in the presence of 2.5 mM 2-deoxyglucose (2-DG). The HPLC chart of (C) is the uncropped version of the original <a href="#fig3">Figure 3</a>G in (<a href="#bib4">Harada et al., 2020</a>). Figure reprinted with permission from <a href="#bib4">Harada et al., 2020</a>. POSs were dephosphorylated (+) or left untreated (-) with alkaline phosphatase prior to desalting. Glucose units are indicated on the charts. The deduced structures of PA-labeled glycans are shown in the charts. Asterisks indicate peaks derived from labeling reagents.</p>
</figcaption>
</figure>
<p>Nucleotide sugars are hydrophilic metabolites comprised of monosaccharide linked to the nucleoside base through a (pyro)phosphate group. Solid-phase extraction in the ion-pair reversed-phase mode is required for their purification. In addition, the LC-MS/MS method with triethylamine facilitates the separation of each nucleotide sugar, but not nucleotides. The nucleotides are observed as broad peaks due to their absorption to the separation column, resulting in the specific detection of nucleotide sugars. Therefore, chromatographic conditions with triethylamine are suitable, not only to quantify the nucleotide sugar levels but also to search for unknown nucleotide sugars. As shown in <a href="#fig9">Figure 9</a>A, abundant UDP-sugars and less abundant levels of nucleotide sugars such as GDP-Fuc and CMP-NeuAc, are clearly detected. In addition, by a specific head group survey called a precursor ion scan, UDP-2-DG and GDP-2-DG are detected as unknown peaks in mouse B16-F10 cells that are cultured in a 2-DG-containing medium (<a href="#bib4">Harada et al., 2020</a>) (<a href="#fig9">Figure 9</a>B).</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. LC-ESI-MS analysis of cellular nucleotide sugars</div>
<p>(A) Representative extraction ion chromatograms (EIC) of nucleotide sugars derived from MEFs, which were cultured in the presence of 5 mM glucose. CMP-<i>N</i>-acetylneuraminic acid (CMP-NeuAc); UDP-glucose (UDP-Glc); UDP-<i>N</i>-acetylhexosamine (UDP-<i>N</i>-acetylglucosamine [UDP-GlcNAc] plus UDP-<i>N</i>-acetylgalactosamine [UDP-GalNAc]); GDP-mannose (GDP-Man); UDP-glucuronic acid (UDP-GlcA); GDP-fucose (GDP-Fuc). Signal intensity of the individual nucleotide sugars is indicated in parentheses. The amounts of nucleotide sugars are expressed as pmol per 10<sup>4</sup> cells.</p>
<p>(B) A representative total ion chromatogram (TIC, top panel) of nucleotide sugars derived from B16-F10 cells, which were cultured in the presence of 2.5 mM 2-DG. EIC (middle panel) of UDP-2-deoxyglucose (UDP-2-DG), GDP-2-DG, and GDP-Fuc. Bottom panel is the magnified view of the dashed box. The panels in (B) are from the original Figures S2A-C in <a href="#bib4">Harada et al. (2020)</a>. Figure reprinted with permission from <a href="#bib4">Harada et al. (2020)</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">DLOs and POSs</h3>
<h4 id="sec5.1.1">Data processing</h4>
<p>HPLC data need to be converted to ASCII format. Open Plot2, which is a conversion software program. Go to [file]→ [Import ASCII]. Select the formatted files and click [open]. HPLC charts are now shown in a window on your screen (<a href="#fig10">Figure 10</a>A). Select [Inspector] window. Click [Axis Inspector] and adjust x axis (fluorescent intensity) and y axis (retention time, min) if necessary (<a href="#fig10">Figure 10</a>B). You can choose to show or not show ticks on the axis by clicking [Enable] boxes in [Axis Inspector]. The tick style on x axis and y axis can also be changed in [Axis Format Inspector] if necessary. For changing the color of HPLC charts, go to [Inspector] screen and check the left cell of the sample number by double clicking (<a href="#fig10">Figure 10</a>C). Click [Data Style Inspector] and select colors in [Data] (<a href="#fig10">Figure 10</a>D). Save the charts. To export the file as an image, go to [File] → [Export image] and select the file format to export.</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Data processing for HPLC analysis</div>
<p>(A) An over view of Plot2.</p>
<p>(B) Adjustment of the x axis.</p>
<p>(C and D) Selection of a HPLC chart for color change.</p>
</figcaption>
</figure>
<h4 id="sec5.1.2">Quantification</h4>
<p>Open HPLC charts in an appropriate software for data analysis (e.g., LabSolutions from Shimazu). Reanalyze the data by drawing appropriate baselines on individual chromatograms. Calculate the area for a PA-labeled glucose hexamer (corresponding to 2 pmol) in PA-labeled glucose oligomers. Reanalyze peaks in your samples as well. The amounts of PA-labeled glycans in your samples are calculated by the following equations:</p>
<p><i>Q</i>(<i>pmol</i>) = <i>A<sub>Sample</sub></i> × 2 (<i>pmol</i>)/<i>A<sub>Std</sub></i><br> <i>Q<sub>Total</sub></i>(<i>pmol</i>) = <i>Q</i>(<i>pmol</i>) × <i>X</i>(<i>μL of total sample volume</i>)/<i>Y</i>(<i>μL of analyzed sample volume</i>)<br> <i>Q<sub>Normalized</sub></i>(<i>pmol</i>/1 × 10<sup>6</sup> <i>cells</i>) = <i>Q<sub>Total</sub></i>(<i>pmol</i>)/<i>Z</i>( × 10<sup>6</sup> <i>cells</i>)</p>
<p>Where <i>Q</i>, <i>Q</i><sub>Total</sub>, <i>Q</i><sub>Normalized</sub> are the amounts of the analyzed, total, and normalized PA-labeled glycans in your samples, respectively; <i>A</i><sub>Sample</sub> and <i>A</i><sub>Std</sub> are the areas of the samples and PA-glucose hexamer, respectively; <i>X</i> and <i>Y</i> are the volumes of total and analyzed volumes of PA-labeled glycans, respectively; and <i>Z</i> is the number of cells (expressed as × 10<sup>6</sup> cells) counted at which the cells are harvested for preparation of DLOs and POSs.</p>
<h4 id="sec5.1.3">Yields</h4>
<p>The assembly of DLOs is extremely efficient under normal culture conditions, resulting in the accumulation of the fully assembled DLO (∼20 pmol/1 × 10<sup>6</sup> cells). In contrast, biosynthetic intermediates of DLOs are rarely detected under normal culture conditions. POSs accumulate only when the biosynthesis of DLOs is inefficient, and therefore POSs are rarely detected under normal culture conditions. However, glucose deprivation or the 2-DG treatment of cells evokes the release of POSs from DLO intermediates, causing the accumulation of POSs with short glycan chains (∼20 pmol total/1 × 10<sup>6</sup> cells). Under such conditions, the amounts of the fully assembled DLO dramatically decrease and DLO intermediates are also nearly undetectable, because the intermediates are degraded into POSs.</p>
<h3 id="sec5.2">Nucleotide sugars</h3>
<h4 id="sec5.2.1">Data processing</h4>
<p>LC-MS/MS data must to be converted to a text format by Lab Solution Insight (Shimadzu). First, open a batch file using the LabSolutions real time analysis program. Several common graphs can then be displayed (<a href="#fig11">Figure 11</a>). Select [Survey] to display the chromatograms of all registered nucleotide sugars. When it is not possible to detect small peaks (e.g., GDP-2-DG) by automated peak integration, the data can be reanalyzed by manual peak integration. Click on the manual peak integration procedures, and click on the link point to draw appropriate baselines on individual chromatograms. After editing all of the results, save them as a processed file. Finally, go to [Export], and select [summary view] format to export.</p>
<figure id="fig11"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/451-Fig11.jpg" alt="Fig11.jpg">
<figcaption>
<div class="figcaption-title">Figure 11. Data processing for LC-MS analysis</div>
<p>(A) A result window showing the peak areas of the nucleotide sugars calculated by automated peak integration. The numbers in the “Area” columns indicate peak areas of nucleotide sugars in B16-F10 cells cultured in the presence or absence of 2.5 mM 2-DG.</p>
<p>(B) Survey windows for GDP-deoxy hexose (deoxy Hex), i.e., GDP-Fuc and GDP-2-DG, in samples in A. The results obtained from two biological replicates are shown. Manual peak integration should be performed if peaks cannot be detected due to their small peak intensities, e.g., GDP-2-DG.</p>
<p>(C) A result window showing the peak areas of GDP-2-DG calculated by manual peak integration.</p>
</figcaption>
</figure>
<h4 id="sec5.2.2">Quantification</h4>
<p>Nucleotide sugars are identified based on their retention times, and the levels are calculated from the peak areas using a standard curve. GDP-Glc can be used as an internal standard, as it does not occur naturally in mammalian cells. Thus, these results are normalized against the recovery of GDP-Glc and are expressed as pmol/mg proteins or pmol/1 × 10<sup>7</sup> cells.</p>
<h4 id="sec5.2.3">Yields</h4>
<p>Using the protocol described above, nucleotide sugar levels can be quantified from 1 × 10<sup>4</sup>–1 × 10<sup>7</sup> cells. The recovery of GDP-Glc is greater than 85%, resulting in the accumulation of abundant levels of UDP-hexose in MEFs (approximately 5,000 pmol/mg protein). Thus, UDP-hexose (i.e., UDP-Glc and UDP-Gal) and UDP-<i>N</i>-acetylhexosamine (i.e., UDP-GlcNAc and UDP-GalNAc) can be quantified from 1 × 10<sup>4</sup> cells (<a href="#fig9">Figure 9</a>A: approximately 50 fmol). Treating cells with 2-DG (e.g., B16-F10 cells) results in the accumulation of unnatural UDP-2-DG (∼2,000 pmol/1 × 10<sup>7</sup> cells) and GDP-2-DG (5 pmol/1 × 10<sup>7</sup> cells).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol described here allows for the detection of the steady-state levels of DLOs, POSs, and nucleotide sugars. If the metabolic flux of these compounds needs to be evaluated, metabolic labeling of cells with radiolabeled sugars (e.g., [2-<sup>3</sup>H]mannose and [6-<sup>3</sup>H]glucosamine) (<a href="#bib10">Rosner et al., 1982</a>) or stable isotope-labeled sugars (e.g., [<sup>13</sup>C<sub>6</sub>]glucose) (<a href="#bib8">Nakajima et al., 2013</a>), followed by chase experiments, should be performed.</p>
<p>Structural analysis of PA-labeled glycans requires additional experimental approaches. Your PA-labeled samples are digested with α1-2/3/6-mannosidase (Cat# P0768, New England BioLabs), followed by size-fractionation HPLC, to identify peaks that contain PA-labeled glycans (<a href="#bib5">Hirayama et al., 2010</a>). Peaks that have disappeared or shifted by the mannosidase digestion in size-fractionation HPLC are manually collected during the run. These fractions are then subjected to MALDI-TOF-MS analysis to determine the number of hexose, <i>N</i>-acetylhexosamine, and deoxy hexose molecules in the PA-labeled glycans. Further stringent determination of glycan isoforms requires reversed-phase HPLC analysis of your PA-labeled samples, in parallel with standard PA-labeled glycans with known structures (<a href="#bib11">Suzuki et al., 2008</a>).</p>
<p>Fluorescent labeling of POSs requires dephosphorylation, a process that hinders the determination of the number of phosphate groups of POSs. This limitation can be overcome by applying an intact POS preparation to ion-paired, reversed-phase LC-ESI-MS/MS analysis (<a href="#bib2">Harada et al., 2016</a>). This analysis has demonstrated that POSs that have accumulated in glucose-deprived MEFs contain a single phosphate group.</p>
<p>Searching for unknown nucleotides requires alternate approaches by a specific head group survey using triple quadrupole mass spectrometry. Precursor ion scans are carried out based on the comprehensive detection of nucleotide motifs in complex mixtures using structure-specific fragmentation. For example, UDP-2-DG can be detected by a precursor ion scan of <i>m/z</i> 323.1 for UDP residues, resulting in the detection of <i>m/z</i> values of 549.1.</p>
<p>The protocol is optimized to maintain a high recovery of nucleotide sugars. If common nucleotides are to be analyzed, the sample extraction and purification steps need to be adjusted for either of them. Extraction with ice-cold 5% perchloric acid are often used for the analysis of cellular nucleotides.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>B16-F10 cells are not dividing and are releasing large amounts of pigments.</p>
<div class="note">
<span class="note-title">Note:</span> This problem occurs, at least in our hands, when B16-F10 cells are cultured in a 5% CO<sub>2</sub> atmosphere in DMEM (high glucose), which contains 3.7 g/L sodium bicarbonate.</div>
<h3 id="sec7.2">Potential solution</h3>
<ul>
<li>Increase the CO<sub>2</sub> concentration to 8%.</li>
<li>A culture medium that is adjusted to pH 7.4 with 4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES) may be used.</li>
</ul>
<h3 id="sec7.3">Problem 2</h3>
<p>Cells are 100% confluent.</p>
<h3 id="sec7.4">Potential solution</h3>
<ul>
<li>The over growth of cells may cause the degradation of DLOs, due to the shortage of glucose in the culture medium. Reduce the number of cells at seeding or shorten the culture periods.</li>
</ul>
<h3 id="sec7.5">Problem 3</h3>
<p>Yield of DLOs is low.</p>
<h3 id="sec7.6">Potential solution</h3>
<ul>
<li>Do not overgrow cells.</li>
<li>Cells are deprived of glucose.</li>
<li>Maintain the cell lysis time at precisely 15 min at step 18.</li>
<li>Evaporate chloroform as much as possible at step 23k.</li>
<li>Check if 4 mM MgCl<sub>2</sub> is added to methanol/water (M/W; 1:1, v/v) used at step 23l.</li>
</ul>
<h3 id="sec7.7">Problem 4</h3>
<p>Yield of POSs is low.</p>
<h3 id="sec7.8">Potential solution</h3>
<ul>
<li>Cells were not deprived of glucose or were not treated with 2-DG. For a stringent glucose deprivation, use a culture medium containing no glucose and dialyzed FBS (26400044, Thermo Fischer Scientific) at a concentration of 10% (v/v).</li>
<li>Maintain the cell lysis time a precisely 15 min at step 18.</li>
<li>Check if pH of Tris-HCl buffer for anion-exchange chromatography at steps 27, 33, and 34 is accurately adjusted to 7.4.</li>
</ul>
<h3 id="sec7.9">Problem 5</h3>
<p>Cells are detached from dishes under glucose deprivation conditions.</p>
<h3 id="sec7.10">Potential solution</h3>
<ul>
<li>Optimize the culture periods, cell numbers, and glucose concentrations, which allow for cells to attach to the dish and for detection of POSs as well.</li>
<li>Wash the glucose-deprived cells very carefully with 1 × PBS before harvesting.</li>
<li>Use surface-coated culture dishes (e.g., poly-L-lysine or collagen).</li>
</ul>
<h3 id="sec7.11">Problem 6</h3>
<p>Aggregates are formed upon the addition of acetonitrile to PA-labeled reaction mixtures at step 37i. As a consequence, the background of the HPLC charts is disturbed.</p>
<div class="note">
<span class="note-title">Note:</span> This problem frequently occurs when large amounts of proteins and/or peptides remain in the samples before PA labeling.</div>
<h3 id="sec7.12">Potential solution</h3>
<ul>
<li>Once formed, do not remove the aggregates. The removal lowers the yield of PA-labeled glycans. Apply the entire solution to MonoFas spin columns for the purification of PA-labeled glycans.</li>
<li>Before desalting the samples with InertSep GC columns at steps 25 and 36, the amounts of proteins and/or peptides in the samples can be reduced by passing them through a column containing layers of Dowex AG1-X2 (lower part; resin volume, 500 μL; 200–400 mesh; acetate form) and AG50-X8 (upper part; resin volume, 500 μL; 200–400 mesh; H<sup>+</sup> form) (<a href="#bib5">Hirayama et al., 2010</a>). This process will lose negatively- or positively-charged free glycans, if any, from the samples.</li>
<li>Freshly prepare 70% (v/v) ethanol for cell lysis.</li>
</ul>
<h3 id="sec7.13">Problem 7</h3>
<p>Retention time of PA-labeled glucose oligomers in size-fractionation HPLC greatly changes between experiments.</p>
<h3 id="sec7.14">Potential solution</h3>
<ul>
<li>Equilibrate the HPLC column for a longer time (&gt; 20 min or 10 column volumes) before running HPLC program.</li>
<li>Always run the HPLC program two times with water (10 μL) before analyzing PA-labeled glucose oligomers.</li>
<li>Accurately adjust pH of the HPLC eluent B to 7.0.</li>
<li>Use freshly prepared HPLC eluents A and B in every experiment.</li>
<li>Replace the HPLC column with a new one.</li>
</ul>
<h3 id="sec7.15">Problem 8</h3>
<p>Pressure of the HPLC pump is too high.</p>
<h3 id="sec7.16">Potential solution</h3>
<ul>
<li>Check the flow-rates of the HPLC pump.</li>
<li>Check if the pressure is low when the HPLC column is disconnected. If so, replace the column with a new one.</li>
</ul>
<h3 id="sec7.17">Problem 9</h3>
<p>Pressure of HPLC pump is too low or fluctuating.</p>
<h3 id="sec7.18">Potential solution</h3>
<ul>
<li>Check if the eluent bottles are placed at a higher position than HPLC pumps.</li>
<li>Check if any air bubbles are stuck in HPLC line or column.</li>
<li>Check if any leakage occurs in HPLC system.</li>
</ul>
<h3 id="sec7.19">Problem 10</h3>
<p>PA-labeled glucose oligomers are not detected or the signal is very weak.</p>
<h3 id="sec7.20">Potential solution</h3>
<ul>
<li>Check if the excitation and emission wave lengths are correct.</li>
<li>Check if any leakage occurs in the HPLC system.</li>
<li>Check if HPLC injector is properly connected.</li>
<li>Check if the volume of a sample loop attached to HPLC injector is large enough to inject 10 μL of samples.</li>
</ul>
<h3 id="sec7.21">Problem 11</h3>
<p>Yield of nucleotide sugars is low.</p>
<h3 id="sec7.22">Potential solution</h3>
<ul>
<li>Use different lots of ENVI-Carb columns for purification of nucleotide sugars.</li>
<li>Use freshly prepared 50 mM triethylamine buffer (pH 7.0) for the purification of nucleotide sugars.</li>
</ul>
<h3 id="sec7.23">Problem 12</h3>
<p>Loss of peak resolution of nucleotide sugars during LC-ESI-MS.</p>
<h3 id="sec7.24">Potential solution</h3>
<ul>
<li>Use a new Inertsil ODS-3 separation column.</li>
<li>Wash the separation column with 20% acetonitrile without triethylamine for 20 min to prevent column deterioration.</li>
</ul>
<h3 id="sec7.25">Problem 13</h3>
<p>Quantitative results of nucleotide sugars are fluctuating.</p>
<h3 id="sec7.26">Potential solution</h3>
<ul>
<li>Repeat the preparation of the cell homogenates. The cell samples should be collected quickly and stored in liquid nitrogen to prevent the degradation of nucleotide sugars.</li>
<li>Optimize the culture periods and glucose concentrations in the culture medium.</li>
</ul>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Harada, Y., Hirayama, H., and Suzuki, T. (2015). Generation and degradation of free asparagine-linked glycans. Cell Mol. Life Sci. <i>72</i>, 2509-2533. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref1">View at publisher</a></p>
<p id="bib2">Harada, Y., Huang, C., Yamaki, S., Dohmae, N., and Suzuki, T. (2016). Non-lysosomal degradation of singly phosphorylated oligosaccharides initiated by the action of a cytosolic endo-beta-N-acetylglucosaminidase. J. Biol. Chem. <i>291</i>, 8048-8058. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref2">View at publisher</a></p>
<p id="bib3">Harada, Y., Nakajima, K., Masahara-Negishi, Y., Freeze, H.H., Angata, T., Taniguchi, N., and Suzuki, T. (2013). Metabolically programmed quality control system for dolichol-linked oligosaccharides. Proc. Natl. Acad. Sci. U S A <i>110</i>, 19366-19371. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref3">View at publisher</a></p>
<p id="bib4">Harada, Y., Nakajima, K., Suzuki, T., Fukushige, T., Kondo, K., Seino, J., Ohkawa, Y., Suzuki, T., Inoue, H., Kanekura, T., et al. (2020). Glycometabolic regulation of the biogenesis of small extracellular vesicles. Cell Rep. <i>33</i>, 108261. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref4">View at publisher</a></p>
<p id="bib5">Hirayama, H., Seino, J., Kitajima, T., Jigami, Y., and Suzuki, T. (2010). Free oligosaccharides to monitor glycoprotein endoplasmic reticulum-associated degradation in Saccharomyces cerevisiae. J. Biol. Chem. <i>285</i>, 12390-12404. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref5">View at publisher</a></p>
<p id="bib6">Ito, J., Herter, T., Baidoo, E.E., Lao, J., Vega-Sanchez, M.E., Michelle Smith-Moritz, A., Adams, P.D., Keasling, J.D., Usadel, B., Petzold, C.J., et al. (2014). Analysis of plant nucleotide sugars by hydrophilic interaction liquid chromatography and tandem mass spectrometry. Anal. Biochem. <i>448</i>, 14-22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref6">View at publisher</a></p>
<p id="bib7">Nagai, R., Deemer, E.K., Brock, J.W., Thorpe, S.R., and Baynes, J.W. (2005). Effect of glucose concentration on formation of AGEs in erythrocytes in vitro. Ann. N. Y. Acad. Sci. <i>1043</i>, 146-150. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref7">View at publisher</a></p>
<p id="bib8">Nakajima, K., Ito, E., Ohtsubo, K., Shirato, K., Takamiya, R., Kitazume, S., Angata, T., and Taniguchi, N. (2013). Mass isotopomer analysis of metabolically labeled nucleotide sugars and N- and O-glycans for tracing nucleotide sugar metabolisms. Mol. Cell Proteomics <i>12</i>, 2468-2480. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref8">View at publisher</a></p>
<p id="bib9">Nakajima, K., Kitazume, S., Angata, T., Fujinawa, R., Ohtsubo, K., Miyoshi, E., and Taniguchi, N. (2010). Simultaneous determination of nucleotide sugars with ion-pair reversed-phase HPLC. Glycobiology <i>20</i>, 865-871. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref9">View at publisher</a></p>
<p id="bib10">Rosner, M.R., Hubbard, S.C., Ivatt, R.J., and Robbins, P.W. (1982). N-Asparagine-linked oligosaccharides: biosynthesis of the lipid-linked oligosaccharides. Methods Enzymol. <i>83</i>, 399-408. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref10">View at publisher</a></p>
<p id="bib11">Suzuki, T., Matsuo, I., Totani, K., Funayama, S., Seino, J., Taniguchi, N., Ito, Y., and Hase, S. (2008). Dual-gradient high-performance liquid chromatography for identification of cytosolic high-mannose-type free glycans. Anal. Biochem. <i>381</i>, 224-232. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00023-X/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yoichiro Harada (<a href="mailto:yoharada3@mc.pref.osaka.jp">yoharada3@mc.pref.osaka.jp</a>).</p>
<h4>Materials Availability</h4>
<p>This study generated PA-glucose oligomers. Materials generated in this study are available upon request from the lead contact.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate new datasets that can be deposited to the currently available public databases. However, MS/MS data are available from the lead contact upon reasonable request.</p>
<h3>Acknowledgments</h3>
<p>This work was supported, in part, by <a href="https://doi.org/10.13039/501100001691">JSPS</a> KAKENHI grant number (19K06546 to Y.H., 18K08257 to K.N., and 18H03990 to T.S.), the <a href="https://doi.org/10.13039/100007449">Takeda Science Foundation</a> (Y.H.), <a href="https://doi.org/10.13039/501100006264">RIKEN</a> Pioneering Project (“Glycolipidologue Initiative” to T.S.), and <a href="https://doi.org/10.13039/100009619">Japan Agency for Medical Research and Development</a> (AMED, 20ae0101070h0005 to N.T.). We wish to thank Dr. Milton Feather for his English editing.</p>
<h3>Author Contributions</h3>
<p>Y.H., K.N., and S.L. wrote the draft of the manuscript. All authors reviewed and revised the manuscript, and approved the final version.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>