<article>
<section>
<h1 id="header">Wide-field calcium imaging of cortex-wide activity in awake, head-fixed mice</h1>
<p><time datetime="2021-11-26">Published: November 26, 2021</time></p>
<p>Chi Ren<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Takaki Komiyama<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Neurobiology Section, Center for Neural Circuits and Behavior, Department of Neurosciences, and Halıcıoğlu Data Science Institute, University of California San Diego, La Jolla, CA 92093, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:c3ren@ucsd.edu">c3ren@ucsd.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:tkomiyama@ucsd.edu">tkomiyama@ucsd.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100973">10.1016/j.xpro.2021.100973</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Characterizing cortex-wide neural activity is essential for understanding large-scale interactions among distributed cortical regions. Here, we describe a protocol using wide-field calcium imaging to monitor the cortex-wide activity in awake, head-fixed mice. This approach provides sufficient signal-to-noise ratio and spatiotemporal resolution to capture large-scale neural activity in behaving mice on a moment-by-moment basis. The use of genetically encoded calcium indicators allows longitudinal imaging over months and can achieve cell-type specificity. We also describe a pipeline to process the imaging data.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref15">Makino et al. (2017)</a> and <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref12">Liu et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Skull-intact surgical preparation to allow optical access to the mouse dorsal cortex</li>
<li>Imaging cortex-wide calcium activity in awake, head-fixed mice</li>
<li>An analysis pipeline to extract neural activity from wide-field image frames</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Wide-field calcium imaging with genetically encoded calcium indicators, especially the GCaMP family, has become an attractive approach to visualize the cortex-wide neural activity in living mice under various sensory and cognitive processes (<a href="#bib4">Cardin et al., 2020</a>; <a href="#bib21">Ren and Komiyama, 2021</a>). It provides a good spatiotemporal resolution by simultaneous illumination of large cortical areas and a high signal-to-noise ratio from bright GCaMP fluorescence. Genetic encoding of GCaMP also enables stable expression for longitudinal recordings and probing specific neuronal populations through genetically restricted expression. The protocol below describes the specific steps for using wide-field calcium imaging to monitor the cortex-wide activity in transgenic mice expressing GCaMP6s widely in cortical excitatory neurons. However, this protocol can also be used to image the cortex-wide activity in specific neuronal populations, such as neurons in specific cortical layers (<a href="#bib9">Gilad et al., 2018</a>; <a href="#bib25">Valley et al., 2020</a>) and different subtypes of cortical inhibitory neurons (<a href="#bib1">Allen et al., 2017</a>). Furthermore, this protocol can be adapted to image the cortex-wide fluorescence signals from other genetically encoded indicators, such as indicators for neurotransmitters (<a href="#bib13">Lohani et al., 2020</a>; <a href="#bib29">Xie et al., 2016</a>). In addition, this protocol can also be incorporated with other recording modalities, such as electrophysiological recordings, to simultaneously record from the cortex and deep brain regions (<a href="#bib12">Liu et al., 2021</a>; <a href="#bib20">Peters et al., 2021</a>). Therefore, this protocol has a broad application to study different macroscopic properties of brain functions. The readers are encouraged to reference a recent protocol paper of similar experiments (<a href="#bib8">Couto et al., 2021</a>). It should be noted that this protocol, together with any modifications, should be approved by the Institutional Animal Care and Use Committee (IACUC) before implementation.</p>
<h3 id="sec1.1">Set up mouse crosses for broad expression of GCaMP6s in cortical excitatory neurons</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 months</div>
<ol>
<li>Cross tetO-GCaMP6s transgenic mice (<a href="#bib27">Wekselblatt et al., 2016</a>) to CaMKIIα-tTA transgenic mice (<a href="#bib16">Mayford et al., 1996</a>) to generate offspring with GCaMP6s widely expressed in cortical excitatory neurons. Both homozygotes and heterozygotes can be used as parents for crossing. When heterozygous parents are used, offspring carrying both transgenic alleles should be identified by PCR genotyping. GCaMP6s expression can also be confirmed by fluorescence imaging through the intact skull in an early step of the surgical procedure.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The use of transgenic mice with endogenous expression of the indicator allows broad and stable expression in specific neuronal populations throughout the cortex. The exact transgenic mouse lines depend on the purpose of the study and the neurons of interest. If particular transgenic lines are not available, expression of the indicator throughout the cortex can also be achieved using AAV-PHP.eB. This AAV variant crosses the blood-brain barrier and delivers genes systemically in the brain (<a href="#bib6">Chan et al., 2017</a>). Although the authors do not have direct experience using this approach for the protocol described here, see reference (<a href="#bib17">Michelson et al., 2019</a>) for a detailed comparison between transgenic and AAV-PHP.eB-mediated expression of GCaMP6s in wide-field calcium imaging.</div>
<div class="note">
<span class="note-title">Note:</span> The exact time varies depending on the age of mice required for the experiments. For example, we usually start the surgical preparation on ∼2–3-month old mice.</div>
<h3 id="sec1.2">Preparation for skull-intact surgery</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>The surgery setup (<a href="#fig1">Figure 1</a>) consists of a stereomicroscope (Leica M125, Leica Microsystems), a fiber optic light source with gooseneck (KL 1500 LED, Leica Microsystems), a servo-controlled heating pad (Harvard Apparatus), a gas anesthesia head holder (nose cone and bite bar, KOPF) and an induction chamber (Vetequip) connected to the isoflurane vaporizer and anesthesia system (Vetequip), a hot bead sterilizer (Fine Science Tools), and surgical tools (Fine Science Tools). In this protocol, we use a custom-built stainless steel headbar attached to the skull for head fixation on the stage under the wide-field microscope (<a href="#fig2">Figure 2</a>). The headbar is custom manufactured (e.g., <a href="http://emachineshop.com">emachineshop.com</a>). Other designs could also be used if the head-fixation can be achieved without blocking the field of view over the dorsal cortex.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Surgery setup and key reagents for the skull-intact surgery</div>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. The custom-built stainless steel headbar (left) attached to the skull (middle) for head fixation on the stage (right) under the wide-field microscope</div>
</figcaption>
</figure>
<ol start="2">
<li>Sterilize the surgery table, surgical tools, and headbar.
<ol type="a">
<li>Sterilize the surface of the surgery table with chlorhexidine solution.</li>
<li>Sterilize surgical tools, including scalpel, scissors, forceps, double-end carver, and the headbar using a hot bead sterilizer.</li>
</ol>
</li>
<li>Cut sterile gel foam into ∼ 1 × 1 mm small pieces.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The gel foam (1975, Ethicon) listed here can be used dry or saturated with saline. However, some gel foams must be moisturized with saline before applying to the wound. Check the direction of your gel foam and soak small pieces of gel foam in saline if necessary.</div>
<ol start="4">
<li>Turn on the servo-controlled heating pad which maintains the mouse body temperature at 35°C–37°C during surgery.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Isoflurane</td>
<td>VetOne</td>
<td>Cat#502017</td>
</tr>
<tr>
<td>Chlorhexidine solution (2%)</td>
<td>AgriLabs</td>
<td>NDC 57561-142-03</td>
</tr>
<tr>
<td>Saline (0.9% sodium chloride)</td>
<td>ICU Medical</td>
<td>NDC 0990-7983-09</td>
</tr>
<tr>
<td>Vaseline Healing Jelly Original</td>
<td>Vaseline</td>
<td><a href="https://www.vaseline.com/us/en/products/healing-jelly/vaseline-healing-jelly-original.html">https://www.vaseline.com/us/en/products/healing-jelly/vaseline-healing-jelly-original.html</a></td>
</tr>
<tr>
<td>Betadine (Solimo First Aid Antiseptic, 10%)</td>
<td>Solimo</td>
<td><a href="https://www.amazon.com/Amazon-Brand-Antiseptic-Povidone-Solution/dp/B07CJXW5GY">https://www.amazon.com/Amazon-Brand-Antiseptic-Povidone-Solution/dp/B07CJXW5GY</a></td>
</tr>
<tr>
<td>Cyanoacrylate glue (All Purpose, Precision Tip)</td>
<td>Krazy Glue</td>
<td><a href="https://www.amazon.com/Krazy-Glue-Tube-Original-07/dp/B000Q3GW3A/ref=sr_1_18?crid=3CVVKM5JKHFB8&amp;dchild=1&amp;keywords=krazy+glue&amp;qid=1626673121&amp;sprefix=krazy%2Caps%2C147&amp;sr=8-18">https://www.amazon.com/Krazy-Glue-Tube-Original-07/dp/B000Q3GW3A/ref=sr_1_18?crid=3CVVKM5JKHFB8&amp;dchild=1&amp;keywords=krazy+glue&amp;qid=1626673121&amp;sprefix=krazy%2Caps%2C147&amp;sr=8-18</a></td>
</tr>
<tr>
<td>Dental acrylic cement (Contemporary Ortho-Jet™ Powder and Jet Liquid)</td>
<td>Lang Dental</td>
<td>
<a href="https://www.langdental.com/products-Contemporary-Ortho-Jettm-Powder-50">https://www.langdental.com/products-Contemporary-Ortho-Jettm-Powder-50</a><a href="https://www.langdental.com/products-Jet-Liquid-46">https://www.langdental.com/products-Jet-Liquid-46</a>
</td>
</tr>
<tr>
<td>Hydrogen peroxide (3%)</td>
<td>Swan</td>
<td>NDC 0869-0871-43</td>
</tr>
<tr>
<td>Buprenorphine Hydrochloride (0.3mg/mL)</td>
<td>Par Pharmaceutical</td>
<td>NDC 42023-179-05</td>
</tr>
<tr>
<td>Baytril (100 mg/mL)</td>
<td>Bayer</td>
<td>NADA 141-068</td>
</tr>
<tr>
<td>Dexamethasone (2 mg/mL)</td>
<td>VetOne</td>
<td>Cat#501012</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: CaMKIIa-tTA, B6;CBA-Tg(Camk2a-tTA)1Mmay/J</td>
<td>The Jackson Laboratory</td>
<td>RRID: IMSR_JAX: 003010</td>
</tr>
<tr>
<td>Mouse: tetO-GCaMP6s, B6;DBA-Tg(tetO-GCaMP6s)2Niell/J</td>
<td>The Jackson Laboratory</td>
<td>RRID: IMSR_JAX: 024742</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>HCImage Live</td>
<td>Hamamatsu Photonics</td>
<td>RRID: SCR_015041</td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks</td>
<td>RRID: SCR_001622</td>
</tr>
<tr>
<td>JADER</td>
<td><a href="#bib5">Cardoso (1999)</a></td>
<td><a href="https://www.mathworks.com/matlabcentral/mlc-downloads/downloads/submissions/67527/versions/3/previews/jadeR.m/index.html">https://www.mathworks.com/matlabcentral/mlc-downloads/downloads/submissions/67527/versions/3/previews/jadeR.m/index.html</a></td>
</tr>
<tr>
<td>Custom MATLAB code for data processing</td>
<td><a href="#bib15">Makino et al. (2017)</a></td>
<td>GitHub: <a href="https://github.com/CRen2333/Wide-field-calcium-imaging">https://github.com/CRen2333/Wide-field-calcium-imaging</a>
</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Leica M125 stereomicroscope</td>
<td>Leica Microsystems</td>
<td><a href="https://www.leica-microsystems.com/products/stereo-microscopes-macroscopes/p/leica-m125-c/">https://www.leica-microsystems.com/products/stereo-microscopes-macroscopes/p/leica-m125-c/</a></td>
</tr>
<tr>
<td>Fiber optic light source with gooseneck (KL 1500 LED)</td>
<td>Leica Microsystems</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse gas anesthesia head holder (nose cone and bite bar)</td>
<td>KOPF</td>
<td>Cat#1923-B</td>
</tr>
<tr>
<td>Stainless steel headbar</td>
<td>Custom-made from eMachineShop</td>
<td>N/A</td>
</tr>
<tr>
<td>Dental drill handpiece</td>
<td>Midwest Tradition</td>
<td>N/A</td>
</tr>
<tr>
<td>Drill bit (FG 1/4)</td>
<td>Henry Schein</td>
<td>Cat#1007205</td>
</tr>
<tr>
<td>Scalpel (#3)</td>
<td>Fine Science Tools</td>
<td>Cat#10003-12</td>
</tr>
<tr>
<td>Fine scissors</td>
<td>Fine Science Tools</td>
<td>Cat#14060-11</td>
</tr>
<tr>
<td>Forceps</td>
<td>Fine Science Tools</td>
<td>Cat#11223-20</td>
</tr>
<tr>
<td>Double-end carver</td>
<td>Henry Schein</td>
<td>Cat#101-0333</td>
</tr>
<tr>
<td>Compressed air duster</td>
<td>Office Depot</td>
<td>Cat#337994</td>
</tr>
<tr>
<td>Gel foam</td>
<td>Ethicon</td>
<td>Cat#1975</td>
</tr>
<tr>
<td>Heat shrink tube</td>
<td>Amazon</td>
<td>N/A</td>
</tr>
<tr>
<td>Servo-controlled heating pad</td>
<td>Harvard Apparatus</td>
<td>Cat#50-7220F</td>
</tr>
<tr>
<td>Hot bead sterilizer</td>
<td>Fine Science Tools</td>
<td>Cat#18000-45</td>
</tr>
<tr>
<td>Isoflurane vaporizer and anesthesia system</td>
<td>Vetequip</td>
<td>Cat#901806</td>
</tr>
<tr>
<td>Induction chamber for gas anesthesia</td>
<td>Vetequip</td>
<td>Cat#941444</td>
</tr>
<tr>
<td>Axio Zoom.V16, objective lens (1 × , 0.25 NA, FWD 56 mm)</td>
<td>ZEISS</td>
<td>RRID: SCR_016980</td>
</tr>
<tr>
<td>Illuminator HXP 200C</td>
<td>ZEISS</td>
<td>Cat#423612-9901-000</td>
</tr>
<tr>
<td>ORCA-Flash4.0</td>
<td>Hamamatsu Photonics</td>
<td><a href="https://www.hamamatsu.com/jp/en/product/type/C13440-20CU/index.html">https://www.hamamatsu.com/jp/en/product/type/C13440-20CU/index.html</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>This protocol uses a commercial fluorescence stereoscope (Axio Zoom.V16, Zeiss) equipped with a scientific camera (ORCA-Flash4.0, Hamamatsu) for wide-field calcium imaging.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> There are a number of vendors that produce wide-field microscopes. Wide-field microscopes can also be custom-built (<a href="#bib8">Couto et al., 2021</a>). Scientific cameras with a large field of view, high quantum efficiency, and fast frame rates are suitable for this protocol.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Skull-intact surgical preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>This section describes the steps of a skull-intact surgical preparation that allows longitudinal optical access through the intact skull over months.</p>
<ol>
<li>Anesthetize the adult mouse (between 6 weeks and 4 months old, either gender) with isoflurane (3% in 100% O<sub>2</sub> for induction) using inhalation in an induction box.</li>
<li>Transfer the anesthetized mouse to an isoflurane ventilator with a nose cone. The percent isoflurane is regulated to achieve steady breathing rates (∼1 breath/s) throughout the surgery (usually 1–1.5% in 100% O<sub>2</sub>). The mouse is kept on a servo-controlled heating pad to maintain body temperature, and the breathing rate is monitored during the entire surgery.</li>
<li>Cover the eyes with copious amounts of Vaseline (<a href="#fig3">Figure 3</a>A, indicated by white arrowheads) to protect against drying and potential exposure to irritant chemicals (e.g., cyanoacrylate glue and dental acrylic cement) in the following steps.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Surgical procedures of the skull-intact preparation for wide-field calcium imaging</div>
<p>(A) Protect the eyes with copious amounts of Vaseline (indicated by white arrowheads). Clean and antisepticise the surgical region (marked by the white dashed line). Make a small incision (∼2 mm) on the scalp using fine scissors (marked by the white scissor symbol).</p>
<p>(B) Starting from the small incision, cut off a circular piece of the scalp to expose the skull covering the dorsal cortex and cerebellum.</p>
<p>(C) Gently remove the periosteum on the skull surface using a scalpel. Note that the skull is less glossy after removing the periosteum compared to (B). Next, detach the muscles from the occipital bone (within the black dashed line) with a scalpel and push the muscles away.</p>
<p>(D) Insert small pieces of gel foam into the gaps between muscles and the occipital bone to stop bleeding. Wipe the skull with cotton swabs soaked in 3% hydrogen peroxide to remove any remaining soft tissues.</p>
<p>(E) Comparison between moisturized and dry skulls. Note that the dry skull is whiter and less transparent. It is critical to dry the skull completely before applying cyanoacrylate glue, as the glue becomes opaque when encountering a wet surface.</p>
<p>(F) Remove the gel foam and apply small drops of cyanoacrylate glue to cover the muscles and the occipital bone (marked by the black dashed line). Let the cyanoacrylate glue cure.</p>
<p>(G) Add drops of dental acrylic cement with a double-end carver to build a thick layer on top of the glue. Let the dental acrylic cement cure. Apply cyanoacrylate glue with a double-end carver to cover the skin along the cuts (indicated by yellow arrowheads).</p>
<p>(H) Implant the headbar. Apply a small amount of cyanoacrylate glue to the interparietal bone (marked by the black dashed ellipse in G) and place the headbar ∼0.5–1 mm posterior to the lambda. Adjust the angle of the headbar by eyeballing until it is parallel to the transverse plane of the skull (the top side of the headbar and the transverse plane of the skull are marked by yellow dashed lines in the bottom panel). Hold the headbar in place until it stays steady.</p>
<p>(I) Secure the headbar. Left, add drops of cyanoacrylate glue to build a glue layer between the headbar and cement (indicated by the yellow shading area). Let the glue cure. Middle and right, apply dental acrylic cement around the headbar to build up the attachment between bones and the headbar. Make sure the bones posterior to the lambda and the middle part of the headbar are entirely covered. Let the dental acrylic cement cure.</p>
<p>(J) Build the cyanoacrylate glue layer on the skull. Apply a small amount of cyanoacrylate glue and evenly distribute it to form a thin layer covering the entire skull. Wait for the cyanoacrylate glue to cure to form a solid, smooth, and transparent layer. Repeat 3–4 times to thicken the glue layer. The skull should gradually become more transparent within 20 min, and the blood vessels should become clearly visible compared to the dry skull in (E).</p>
<p>(K) Protect the eyes from the excitation light of imaging. Cut a chunk (∼5 mm) of black heat shrink tube (∼2 cm diameter) and glue it to the circumference of the skull using cyanoacrylate glue. Then fill the gap between the heat shrink tube and mouse head with black dental acrylic cement from the outside.</p>
<p>Scale bars in (D), 1 mm, in (E and J), 0.5 mm.</p>
</figcaption>
</figure>
</li>
<li>Clean and antisepticise the scalp and the surrounding region (<a href="#fig3">Figure 3</a>A, marked by the white dashed line).
<ol type="a">
<li>Wipe the surgical region with cotton swabs soaked in 70% ethanol 2–3 times.</li>
<li>Wipe the surgical region with cotton swabs soaked in betadine until the entire surgical region has been infiltrated.</li>
</ol>
</li>
<li>Make a small incision (∼2 mm) on the scalp between two ears using fine scissors (<a href="#fig3">Figure 3</a>A, marked by the white scissor symbol). Starting from the small incision, cut off a circular piece of the scalp to expose the skull covering the dorsal cortex and cerebellum. Retract the remaining scalp around the cuts with forceps to fully expose the dorsal skull (<a href="#fig3">Figure 3</a>B).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> We have found that shaving hair or not before cutting off the scalp does not impact the surgical success rate. However, if there is interfering hair left on the skull after cutting off the scalp, remove the hair with clean cotton swabs or forceps to prevent hindrance to a clear field of view for wide-field calcium imaging.</div>
<div class="critical">
<span class="critical-title">Critical:</span> When cutting the scalp close to the eyes, use forceps to lift the scalp and confirm the location of the retro-orbital sinus first. Do not damage the retro-orbital sinus.</div>
<ol start="6">
<li>Gently remove the periosteum on the skull surface using a scalpel. Note that the skull is less glossy after removing the periosteum (<a href="#fig3">Figure 3</a>C, compare with <a href="#fig3">Figure 3</a>B). If bone bleeding happens, use a dry cotton swab to press the bleeding spot until bleeding stops (usually within 5 min). If the bleeding continues after pressing for more than 10 min, terminate the surgery and exclude the animal from the experiment. Rinse the skull with saline several times to remove any bloodstains. <a href="#sec7.1">Troubleshooting 1</a>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Be particularly gentle when cleaning the periosteum along the bone sutures to avoid bleeding under the skull.</div>
<ol start="7">
<li>Detach the muscles from the occipital bone (<a href="#fig3">Figure 3</a>C, muscles within the black dashed line) with a scalpel and push the muscles away. This is to extend the bone interface for cyanoacrylate glue and dental acrylic cement. This is critical for the steady implantation of the headbar. Next, insert small pieces of gel foam into the gaps between muscles and the occipital bone to stop bleeding (<a href="#fig3">Figure 3</a>D).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Some gel foams must be moisturized with saline before applying to the wound. Check the direction of your gel foam and soak small pieces of gel foam in saline if necessary. Though the gel foam (1975, Ethicon) listed in this protocol can be used both dry and saturated with saline, we note that dry gel form is more efficient in stopping muscle bleedings.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Keeping the skull clean and smooth is critical to achieving even optical access through the dorsal cortex. When detaching the muscles, be gentle to avoid making deep scratches or notches on the occipital bone and skull, which can cause bone bleeding and roughen the skull surface. If bleeding is too heavy to stop with gel foam, press on the bleeding site with clean and dry cotton swabs, and the bleeding usually stops within 5 min. If any blood spreads to the skull and generates stains, rinse the skull with saline and wipe with dry cotton swabs after bleeding stops to remove bloodstains. If bleeding continues for more than 20 min and cannot be slowed down, terminate the surgery and exclude the animal from the experiment.</div>
<ol start="8">
<li>Wipe the skull to remove any remaining soft tissues with cotton swabs soaked in 3% hydrogen peroxide.</li>
<li>Wait ∼5 min to dry the dorsal skull. The skull should become whiter and less transparent as it dries (see <a href="#fig3">Figure 3</a>E for the comparison between dry and moisturized skulls). <a href="#sec7.3">Troubleshooting 2</a>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cyanoacrylate glue becomes opaque when encountering a wet surface. Make sure the skull is dry before continuing. If the drying is too slow, gently blow the skull with a compressed air duster to facilitate drying and ensure dryness. Do not blow too hard, as a strong airflow may detach the skin around the cuts and cause bleeding near the retro-orbital sinus.</div>
<ol start="10">
<li>Fill the gap between muscles and the occipital bone.
<ol type="a">
<li>Gently remove the gel foam between muscles and the occipital bone with forceps.</li>
<li>Apply small drops of cyanoacrylate glue to cover the muscles and the occipital bone (<a href="#fig3">Figure 3</a>F, the area encompassed by the black dashed line. Note that the area shows high gloss when covered with cyanoacrylate glue).</li>
<li>After the cyanoacrylate glue cures, add drops of dental acrylic cement with a double-end carver to build a thick layer on top of the glue (<a href="#fig3">Figure 3</a>G).</li>
<li>Apply cyanoacrylate glue with a double-end carver to cover the skin along the cuts (<a href="#fig3">Figure 3</a>G, indicated by yellow arrowheads). <a href="#sec7.5">Troubleshooting 3</a>
</li>
</ol>
</li>
<li>Implant the headbar.
<ol type="a">
<li>Apply a small amount of cyanoacrylate glue to the interparietal bone (<a href="#fig3">Figure 3</a>G, marked by the black dashed ellipse).</li>
<li>Place the headbar ∼0.5–1 mm posterior to the lambda.</li>
<li>Adjust the angle of the headbar by eyeballing until it is parallel to the transverse plane of the skull (<a href="#fig3">Figure 3</a>H, the top side of the headbar and the transverse plane of the skull are marked by yellow dashed lines).</li>
<li>Hold the headbar in place until it stays steady.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Placing the headbar parallel to the transverse plane of the skull is important to reduce distortion in imaging and help alignment across mice. It also keeps the mice in a more comfortable posture during imaging.</div>
<ol start="12">
<li>Secure the headbar.
<ol type="a">
<li>Add drops of cyanoacrylate glue through the holes on the headbar to build a glue layer between the headbar and cement (<a href="#fig3">Figure 3</a>I, left, indicated by the yellow shading area).</li>
<li>After the glue cures, apply dental acrylic cement with a double-end carver around the headbar to build up the attachment between bones and the headbar. Make sure the bones posterior to the lambda are entirely covered. Embed the middle part of the headbar in cement to secure the headbar further (<a href="#fig3">Figure 3</a>I, middle and right). <a href="#sec7.7">Troubleshooting 4</a>
</li>
<li>Let the dental acrylic cement cure.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Applying enough cyanoacrylate glue and dental acrylic cement is important to firmly fix the headbar onto the skull in living mice due to the small interface between the headbar and the skull. Otherwise, the headbar may detach from the skull during imaging.</div>
<ol start="13">
<li>Build a thin cyanoacrylate glue layer on the skull.
<ol type="a">
<li>Apply a small amount of cyanoacrylate glue to the skull over the dorsal cortex and evenly distribute it using a double-end carver to form a thin layer covering the entire skull.</li>
<li>Wait for the cyanoacrylate glue to cure to form a solid, smooth, and transparent layer. Curing usually takes ∼5 min.</li>
</ol>
</li>
<li>Repeat step 13 3–4 times. The skull should gradually become more transparent within 20 min, and the brain blood vessels should become clearly visible (<a href="#fig3">Figure 3</a>J, compare with the dry skull in <a href="#fig3">Figure 3</a>E). <a href="#sec7.9">Troubleshooting 5</a>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Applying small amounts of glue multiple times is more efficient than applying a large amount at once as the curing requires moisture in the air.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If bleeding starts after applying cyanoacrylate glue and spreads, carefully drill off the glue with a dental drill (Midwest Tradition) after the glue layer cures. To avoid overheating during drilling, stop every 30 s to let the surface cool down and blow with a compressed air duster to facilitate cooling. Next, blow off the debris with a compressed air duster and reapply cyanoacrylate glue to form a smooth and transparent layer.</div>
<ol start="15">
<li>Cut a chunk (∼5 mm) of black heat shrink tube (∼2 cm diameter) and glue it to the circumference of the skull using cyanoacrylate glue. Then fill the gap between the heat shrink tube and mouse head with black dental acrylic cement from the outside (<a href="#fig3">Figure 3</a>K). This is to minimize the entry of the excitation light (blue light, ∼485 nm) to the eyes during imaging.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> There are different colors of the dental acrylic cement powder, but the black one is most effective in blocking the light. If you do not have black dental acrylic cement powder in hand, use the least transparent powder you have. After the cement cures, cover it with a layer of black electrical tape or paint it with black marker pens to help block the light.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Do not let black dental acrylic cement flow onto the transparent glue layer on the dorsal skull to avoid blocking the imaging field of view.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Exposure to the excitation blue light for wide-field calcium imaging can cause retinal damage, especially after frequent imaging or a single long imaging session (e.g., 3 h) (<a href="#bib19">Nakamura et al., 2018</a>). In addition, when imaging is performed in alternating blocks (e.g., 5-min on and 5-min off), the light onset may activate the visual cortex and complicate the interpretation of activity. Therefore, it is critical to minimize the entry of the excitation blue light to the eyes.</div>
<ol start="16">
<li>Inject Buprenorphine for pain relief (0.1 mg/kg of body weight) and Baytril to prevent infection (10 mg/kg of body weight) subcutaneously.</li>
<li>After the dental acrylic cement in step 15 is cured, take the mouse off from the nose cone and monitor it as it recovers from anesthesia.</li>
<li>Check the optical clarity of the skull-intact preparation frequently after the surgery. The window should stay clear for months when no infection occurs. In our experience, the preparation of most animals can stay clear for at least 2 months. The longest experiment we have performed lasted 10 weeks and the preparation stayed clear throughout the experiment.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Although rare, infection can still happen within the first week after surgery. If the preparation appears cloudy and yellow due to infection in the skull, inject a mixture of Baytril (10 mg/kg of body weight) and Dexamethasone (1 mg/kg of body weight) subcutaneously for consecutive 3–5 days to stop infection and reduce inflammation. If the infection continues or the preparation does not become clear for imaging, terminate the experiment of this animal and exclude it from the study.</div>
<h3 id="sec3.2">Wide-field calcium imaging in living mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min to 3 h</div>
<p>This section describes the steps of wide-field calcium imaging through the skull-intact preparation in living mice.</p>
<ol start="19">
<li>Prepare the imaging setup.
<ol type="a">
<li>Turn on the imaging light source (HXP 200 C, Zeiss), the microscope (Axio Zoom.V16 equipped with an objective lens (1 ×, 0.25 NA, FWD 56 mm), Zeiss), and the camera (ORCA-Flash4.0, Hamamatsu). The filter set for imaging GCaMP signals is commercially installed in the microscope. It consists of a bandpass filter for the excitation light (485 ± 17 nm), a beamsplitter (500 nm), and a tunable bandpass filter centered at 520 nm for the emission light.</li>
<li>Open the HCImage Live software (Hamamatsu). Set frame size to 512 × 512 pixels and frame rate to 30 Hz.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The pixel number and frame rate vary depending on the experimental design and the kinetics of the indicator. For imaging GCaMP signals, frame size is often between 512 × 512 to 1024 × 1024 pixels, and frame rate is in a range of 15–60 Hz.</div>
<ol start="20">
<li>Prepare the mouse for imaging.
<ol type="a">
<li>Fix the mouse to the stage under the microscope through the headbar.</li>
<li>Clean the window surface with cotton swabs soaked in 70% ethanol.</li>
</ol>
</li>
<li>Choose the field of view (FOV).
<ol type="a">
<li>Adjust the position of the stage to place the cortex in the center of the FOV (∼11 mm × 11 mm).</li>
<li>Adjust the height of the objective until the blood vessels are in focus. Due to the curvature of the cortex, the lateral blood vessels around the edges will be out of focus and appear blurry (<a href="#fig4">Figure 4</a>A, middle).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Wide-field calcium imaging in the transgenic mouse with a broad expression of GCaMP6s in cortical excitatory neurons</div>
<p>(A) Left, imaging setup. Middle, an example field of view showing the fluorescence signal from most of the dorsal cortex. Note that the lateral blood vessels around the edges are out of focus and appear blurry due to the curvature of the cortex. Right, cortical regions (based on the mouse brain atlas from the Allen Institute (<a href="#bib26">Wang et al., 2020</a>)) simultaneously recorded by wide-field calcium imaging. Orange dashed box represents the area within the FOV. M2, Secondary motor cortex; M1, primary motor cortex; S1, primary somatosensory cortex; PPC, posterior parietal cortex; S2, secondary somatosensory cortex; Aud., auditory cortex; RSC, retrosplenial cortex; Vis., visual cortex.</p>
<p>(B) Example image frames of cortex-wide activity (Δf/f) showing different patterns and activity traces of individual pixels in a behaving mouse. Not that the activation in small cortical areas can be resolved by wide-field calcium imaging. Gray dashed lines indicate the time of image frames in activity traces of example pixels.</p>
<p>Adapted from <a href="#bib21">Ren and Komiyama (2021)</a>. Scale bars in (A), 5 mm, in (B), for image frames, 2 mm, for activity traces, veritical, 0.1 Δf/f, horizontal, 10 s.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Minimize the power of excitation light during this process to avoid photobleaching of GCaMP signals.</div>
<ol start="22">
<li>Adjust the power of the excitation light (blue light, ∼485 nm for imaging GCaMP signals). To avoid saturation, the brightest pixels (usually located in the retrosplenial cortex) are usually around 50–60% of the full dynamic range. For imaging GCaMP6s in cortical excitatory neurons, the light power is ∼10–20 mW to achieve a good signal quality without inducing significant photobleaching.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> During repeated imaging, we use the average frame from the first imaging session as a reference and align the FOV in the following imaging sessions to this reference image based on landmarks (e.g., blood vessels, small bright spots on dental acrylic cement) through eyeballing. The x–y location can be aligned by adjusting the position of the stage as in step 21. No precise x–y alignment is required during image acquisition, as small misalignments will be corrected by the offline image registration (see ‘<a href="#sec5.2">image registration across sessions within the same mouse</a>’ section for details). The z depth can be aligned by adjusting the height of the objective until the blood vessels show similar clarity as the reference image.</div>
<div class="note">
<span class="note-title">Note:</span> With a power of 10–20 mW, the imaging depth is down to ∼200 μm, covering cortical layer 1 and shallow layer 2/3. As the light intensity decays quickly during the propagation in brain tissue (decreasing to 50% at ∼100 μm and 10% at ∼200 μm) (<a href="#bib30">Yizhar et al., 2011</a>), the majority of signals likely come from layer 1.</div>
<div class="note">
<span class="note-title">Note:</span> To minimize potential photobleaching due to repeated imaging in longitudinal studies, imaging can be performed with a certain duty cycle and increase intervals between imaging sessions. Here we provide our imaging schedule as a reference for repeated imaging: imaging is performed with a 50% duty cycle (5-min on and 5-min off) for ∼30 min every other day for a 22-day behavior task (11 imaging days in total).</div>
<ol start="23">
<li>Set the acquisition to internal or external trigger mode in HCImage Live software based on the experiments and start imaging acquisition.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Under the internal mode, the image acquisition is controlled by HCImage Live software and independent of external triggers. The internal mode is used when no alignment or synchronization is required between imaging and other recordings, such as imaging spontaneous cortical activity.</div>
<div class="note">
<span class="note-title">Note:</span> Under the external trigger mode, the image acquisition is controlled by external devices through TTL. It is suitable when imaging needs to be synchronized to behavior recordings or other recording modalities, such as imaging cortex-wide activity during task performance or simultaneous recording with electrophysiology.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using this protocol, we can perform a skull-intact surgical preparation that allows optical access to most of the dorsal cortex for imaging cortex-wide neural activity in living mice (<a href="#fig4">Figure 4</a>A). The surgery is less invasive than craniotomy so the mice can recover from the surgery quickly and have a lower infection probability. According to our experience, the skull-intact preparation can stay clear for at least 2 months, and no obvious photobleaching occurs with appropriate intensity of the excitation light, allowing repeated imaging over time. This is particularly beneficial for studies requiring longitudinal recordings, such as monitoring the cortex-wide activity throughout long-term learning. Wide-field calcium imaging also provides a good spatiotemporal resolution and a high signal-to-noise ratio to resolve diverse cortex-wide activity patterns on a moment-by-moment basis (<a href="#fig4">Figure 4</a>B). Taken together, these advantages have made wide-field calcium imaging an attractive tool to uncover cortex-wide dynamics in various cognitive processes, such as sensory perception, sensorimotor integration, decision-making, and learning (<a href="#bib4">Cardin et al., 2020</a>; <a href="#bib21">Ren and Komiyama, 2021</a>). Here, we use two of our studies as example cases to demonstrate the capacity of wide-field calcium imaging in resolving cortex-wide dynamics in living mice.</p>
<h3 id="sec4.1">Case 1: Characterizing learning-related dynamics of cortex-wide activity during motor learning (<a href="#bib15">Makino et al., 2017</a>)</h3>
<p>This study investigated how motor learning modulates the cortex-wide neural dynamics during the acquisition of new motor skills. We performed longitudinal wide-field calcium imaging in transgenic mice expressing GCaMP6s in cortical excitatory neurons while they learned a lever-press motor task (<a href="#fig5">Figure 5</a>A). By analyzing the spatiotemporal properties of the cortex-wide activity during lever-pressing movements (<a href="#fig5">Figure 5</a>B), we found that a more compressed and reliable cortex-wide activity gradually emerged with learning. Furthermore, learning also rerouted the cortical activity flow, inducing a novel activity stream flowing from the secondary motor cortex (M2) to the rest of the cortex (<a href="#fig5">Figure 5</a>C). Meanwhile, M2 acquired an indispensable role in coordinating cortex-wide dynamics for newly acquired motor skills. These results suggest that motor learning reconfigures the cortex-wide neural network, forming a more efficient and reliable representation of the learned behavior.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Two example cases using wide-field calcium imaging to investigate cortex-wide dynamics</div>
<p>(A) Imaging cortex-wide activity with wide-field calcium imaging during a motor learning task. In this task, water-restricted mice are trained to press a lever during an auditory cue to receive a water reward.</p>
<p>(B) Cortex-wide activity aligned to the lever-pressing movement onset averaged across movements in a single session from an example animal. Note that the lever-pressing movement evokes distributed activation of most of the cortex.</p>
<p>(C) Learning changes the activity flow throughout the cortex, inducing a secondary activity stream flowing from the secondary motor cortex to the rest of the cortex. Black arrows indicate the directions of the activity flow. Adapted from <a href="#bib15">Makino et al. (2017)</a>.</p>
<p>(D) Simultaneous recordings of cortex-wide activity using wide-field calcium imaging and hippocampal activity using a newly developed transparent and flexible electrode array (Neuro-FITM). Yellow dashed lines mark the edge of the electrode array for visualization. Left, wide-field surgical preparation with the electrode array implanted. Right, example field of view under the wide-field microscope.</p>
<p>(E) Diverse cortical activity patterns during hippocampal SWRs. Adapted from <a href="#bib12">Liu et al. (2021)</a>.</p>
<p>(F) Different cortex-wide activity patterns are associated with distinct hippocampal neural population activity during SWRs.</p>
<p>Scale bars in (D), 5 mm, in (E), vertical, 200 μV, horizontal, 50 ms.</p>
</figcaption>
</figure>
<h3 id="sec4.2">Case 2: Characterizing cortex-wide activity associated with hippocampal sharp-wave ripples (SWRs) (<a href="#bib12">Liu et al., 2021</a>)</h3>
<p>This study investigated the large-scale interactions between cortex and hippocampus during hippocampal SWRs during spontaneous activity in awake mice. We performed simultaneous recordings of cortex-wide activity using wide-field calcium imaging and hippocampal activity using a newly developed transparent and flexible electrode array (Neuro-FITM, <a href="#fig5">Figure 5</a>D). We found diverse cortical activity patterns accompanied hippocampal SWRs (<a href="#fig5">Figure 5</a>E). Furthermore, using dimensionality reduction analysis, we extracted several major types of cortex-wide activity patterns during SWRs, each associated with distinct hippocampal neural population activity (<a href="#fig5">Figure 5</a>F). These results suggest selective and diverse interactions between the large-scale cortical network and hippocampus during SWRs.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<div class="timing">
<span class="timing-title">Timing:</span> ∼24 h/1-h continuous imaging at 30 Hz</div>
<h3 id="sec5.1">Obtain Δf/f time series for each pixel in wide-field images</h3>
<p>This section describes the steps to obtain Δf/f from image frames. Data processing is performed using custom codes in MATLAB (GitHub: <a href="https://github.com/CRen2333/Wide-field-calcium-imaging">https://github.com/CRen2333/Wide-field-calcium-imaging</a>).</p>
<ol>
<li>To reduce data size and noise, images can first be binned from 512 × 512 pixels (∼21.5 × 21.5 μm<sup>2</sup>/pixel) to 128 × 128 pixels (∼86 × 86 μm<sup>2</sup>/pixel).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Although the binning sacrifices the spatial resolution, it also reduces the noise and compresses the data size for processing. After binning, the spatial resolution (∼86 × 86 μm<sup>2</sup>/pixel) is still sufficient to resolve activity within individual cortical regions (<a href="#fig4">Figure 4</a>B).</div>
<ol start="2">
<li>To account for different fluorescence intensities across pixels, Δf/f time series for each pixel is calculated to measure the relative changes in fluorescence intensity caused by activity (<a href="#fig6">Figure 6</a>).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Obtain Δf/f time series for each pixel in wide-field images</div>
<p>Raw fluorescence signal, estimated baseline (f), and Δf/f traces of two example pixels with different brightness are shown here. Note that the estimated baseline (f) captures the difference in fluorescence intensity between two pixels and the slow drifts in raw fluorescence traces (potentially due to the slow decay time of GCaMP6s). After normalization, the Δf/f time series shows a more stable baseline level while fast transients are retained.</p>
</figcaption>
</figure>
<ol type="a">
<li>The time-varying baseline fluorescence (f) of each pixel is estimated for a given time point as the 10th percentile value over 30 s around it.</li>
<li>For the beginning and end of each continuous image stack, the following and preceding 15 s window is used to determine the baseline, respectively.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The estimated baseline generated by this parameter set is close to the fluorescence intensity where no apparent activation occurs. It filters out fast signal transients but captures slow drifts in the fluorescence trace (e.g., the gradual decay of signal intensity during long-time imaging or accumulated intensity drifts due to the slow decay time of GCaMP6s). After normalization, the Δf/f time series shows a more stable baseline level while fast transients are retained.</div>
<h3 id="sec5.2">Image registration across sessions within the same mouse</h3>
<p>This section describes the steps to register image frames across multiple imaging sessions within each animal.</p>
<div class="note">
<span class="note-title">Note:</span> We recommend doing image registration before running principal component analysis (PCA) and independent component analysis (ICA) on imaging data combined across sessions as described below. Registration can minimize the effects of variability in head position across sessions so that the top components from PCA and ICA will be focused on variances from brain activity.</div>
<ol start="3">
<li>Calculate the average frame of each imaging session for registration, and the average frame of the first imaging session is taken as the reference image.</li>
<li>For each imaging session, we use ‘imregconfig’ and ‘imregtform’ MATLAB functions to generate a geometric transformation matrix of the averaged frame to align with the reference image.</li>
<li>Use ‘imwarp’ MATLAB function to apply the geometric transformation matrix to the entire imaging session.</li>
</ol>
<h3 id="sec5.3">Removing hemodynamic artifacts using principal component analysis and independent component analysis</h3>
<p>The excitation and emission wavelengths (∼485 and ∼520 nm, respectively) of GCaMP are in the absorption spectrum of oxyhemoglobin and deoxyhemoglobin, so the raw fluorescence signal of wide-field calcium imaging is contaminated by hemodynamic changes. We use a PCA-and-ICA-based method to remove hemodynamic signals from wide-field signals (<a href="#fig7">Figure 7</a>). In brief, we first perform PCA to reduce the data dimensionality while still preserving most of the variance. Next, different signal sources (independent components, ICs) are extracted from principal components (PCs) based on their statistical independence through ICA. Identified ICs correspond to functionally distinct cortical regions and artifacts.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1195-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Procedures of using PCA-and-ICA analysis to remove hemodynamic artifacts from wide-field signals</div>
<p>(A) Transform the 3D Δf/f image stack (row pixel × column pixel × frame) to a 2D matrix (pixel × frame) for PCA where pixels are treated as variables and frames are treated as observations.</p>
<p>(B) Extract spatial components by PCA-and-ICA analysis. Most spatial ICs correspond to known cortical regions or hemodynamic artifacts. ICs corresponding to artifacts are visually identified and excluded.</p>
<p>(C) Reconstruct Δf/f images by multiplying the remaining ICs corresponding to cortical regions with their time series, then adding the mean value from the original Δf/f image stack for each pixel. The resulting reconstructed images retain the activity of cortical regions while effectively reducing artifacts. White arrowheads indicate the elimination of hemodynamic signals in reconstructed images compared to the raw images.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> This strategy of identifying independent signal sources in recorded mixtures has been widely used for analyzing data of other recording modalities, such as functional magnetic resonance imaging (fMRI) and electroencephalogram (EEG) (<a href="#bib3">Brown et al., 2001</a>). It is also simple to implement and effective for data from wide-field calcium imaging.</div>
<div class="note">
<span class="note-title">Note:</span> Extracting 40 ICs from 37.8 GB data (128 × 128 pixels × 324,000 frames of a 3-h imaging session) takes ∼15–20 h using JADER algorithm for ICA (including ∼1 h for PCA) on a server with dual CPU (Intel Xeon E5-2630 v4, 6 cores and 12 threads each). Different ICA algorithms can also be applied based on the available computing power, see reference (<a href="#bib22">Sahonero-Alvarez and Calderon, 2017</a>) for a detailed comparison between different algorithms. However, we also note that hemodynamics signals can be removed by using a secondary wavelength of light at the isosbestic point of the sensor to estimate the reflectance changes caused by hemodynamics (<a href="#bib14">Ma et al., 2016</a>).</div>
<ol start="6">
<li>First, transform the 3D Δf/f image stack (row pixel × column pixel × frame) to a 2D matrix (pixel × frame) for PCA (<a href="#fig7">Figure 7</a>A).</li>
<li>Extract spatial PCs for ICA. In PCA, we treat pixels as variables and frames as observations. The first 40 PCs that explain ∼95% of the total variance are retained for subsequent ICA. For GCaMP signals from cortex-wide excitatory neurons, 30–50 PCs are often sufficient to capture &gt; 90% of the total variance.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In studies using different indicators or imaging different neuronal populations (<a href="#bib18">Musall et al., 2019</a>; <a href="#bib23">Shimaoka et al., 2019</a>), the number of retained PCs may vary based on the signal quality and the fraction of variance explained.</div>
<ol start="8">
<li>Feed retained PCs into ICA to extract functional modules. The ICA algorithm adopted in our protocol is JADER, which decomposes mixed signals into independent components by minimizing the mutual information (<a href="#bib5">Cardoso, 1999</a>). JADER extracts 40 ICs based on the first 40 PCs. Here, ICs are spatial modes forced to be as independent as possible. Most ICs correspond to known cortical regions or hemodynamic artifacts (<a href="#fig7">Figure 7</a>B).</li>
<li>ICs corresponding to artifacts are visually identified and excluded.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The number of components excluded can vary across animals due to individual variations in blood vessel pattern and hemodynamic signal strength.</div>
<ol start="10">
<li>Reconstruct images using the remaining ICs corresponding to cortical regions by calculating the products of these ICs with their time series (<a href="#fig7">Figure 7</a>C).</li>
<li>For each pixel, its mean value from the original Δf/f image stack is added, because the initial PCA subtracts the mean of each pixel. The resulting reconstructed image stacks specifically retain the activity of cortical regions while effectively reducing artifacts (<a href="#fig7">Figure 7</a>C).</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>First, the skull-intact surgical preparation requires relatively strong fluorescence signals in imaging samples due to limited skull transmission (<a href="#bib24">Soleimanzad et al., 2017</a>). To image dim samples (e.g., GCaMP fluorescence signals from sparse neuronal populations), replacing the skull with a transparent glass window (<a href="#bib10">Kim et al., 2016</a>) to increase light transmission might be necessary to improve the signal-to-noise ratio. Second, due to the light scattering in brain tissue, the wide-field signals mainly come from superficial cortical layers. Therefore, it may be impracticable to image deep layers with wide-field calcium imaging directly. Third, as wide-field calcium imaging does not possess single-cell resolution, the signal in each pixel is an integration of activity from both local neurons and long-range axonal projections. Therefore, the wide-field calcium signal is not a clean representation of local neural activity, which calls for caution in interpreting the results.</p>
<p>Recently, wide-field calcium imaging has been combined with other recording modalities, such as fMRI, two-photon calcium imaging, and electrophysiological recordings (<a href="#bib2">Barson et al., 2020</a>; <a href="#bib7">Clancy et al., 2019</a>; <a href="#bib11">Lake et al., 2020</a>; <a href="#bib12">Liu et al., 2021</a>; <a href="#bib20">Peters et al., 2021</a>; <a href="#bib28">Xiao et al., 2017</a>). These multimodal recording techniques complement the limitations of wide-field calcium imaging in recording depth and spatial resolution, extending the applications in at least two aspects. First, they offer new opportunities to investigate the large-scale interactions between the cortex and subcortical regions during various cognitive processes. Second, simultaneous recoding at single-cell and cortex-wide level bridges the gap between neural activities at different spatial scales and helps to reveal how local circuits are embedded in larger neural networks.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Bloodstains on the skull surface and in glue layers (related to step 6).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Before applying cyanoacrylate glue, use a dry cotton swab to press the bleeding spot until bleeding stops and rinse the skull with saline several times to clean any bloodstains. If the bleeding does not stop after pressing for 10 min, terminate the surgery and exclude the animal from the experiment. If bleeding starts after applying cyanoacrylate glue and spreads, carefully drill off the glue with a dental drill after the glue layer cures. To avoid overheating during drilling, stop every 30 s to let the surface cool down and blow with a compressed air duster to facilitate cooling. Next, blow off the debris with a compressed air duster and reapply cyanoacrylate glue to form a smooth and transparent layer. If bleeding does not stop or the bloodstains contaminate a large area (e.g., covering more than half of a cortical region), terminate the surgery and exclude the animal from the experiment.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Glue layers are opaque (related to steps 8–9).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Make sure the soft tissues are cleared, and the skull is dry before applying cyanoacrylate glue. The skull should become whiter and less transparent as it dries (see <a href="#fig3">Figure 3</a>E for the comparison between dry and moisturized skulls). To ensure dryness, gently blow the skull with a compressed air duster to facilitate drying until the skull stops becoming further whiter. However, do not blow too hard as a strong airflow may detach the skin around the cuts and cause bleeding near the retro-orbital sinus.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Infection along the scalp cuts (related to step 10).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>The infection will loosen the attachment between glue layers and the skull. Add cyanoacrylate glue to seal the cuts further. Inject Baytril (10 mg/kg of body weight) subcutaneously for consecutive 3–5 days to stop infection. If the infection continues, terminate the experiment and exclude the animal from the study.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Headbar detaches during head-fixation (related to step 12).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>This happens when there is not enough cyanoacrylate glue and dental acrylic cement between bones and the headbar. If only a small gap appears between the skull and glue layer, fill the gap and seal the edge with cyanoacrylate glue. Next, add cyanoacrylate glue and dental acrylic cement between bones and the headbar to form a firm fixation. Inject Baytril (10 mg/kg of body weight) subcutaneously to prevent infection. If the detaching cannot be fixed or it causes excessive bleeding on the skull, terminate the experiment and exclude the animal from the study.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>The surface of glue layers is wrinkled and frosted (related to steps 13–14).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>This happens when a large amount of cyanoacrylate glue is applied at a time, and the curing speed differs between the center and the edge. Only apply a small amount of glue each time and let the glue cure before adding another layer. If this issue still occurs, carefully shave the glue to an even layer with a dental drill. To avoid overheating during drilling, stop every 30 s to let the surface cool down and blow with a compressed air duster to facilitate cooling. After shaving, blow off debris with a compressed air duster and reapply glue to form a smooth and transparent surface.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Allen, W.E., Kauvar, I.V., Chen, M.Z., Richman, E.B., Yang, S.J., Chan, K., Gradinaru, V., Deverman, B.E., Luo, L., and Deisseroth, K. (2017). Global representations of goal-directed behavior in distinct cell types of mouse neocortex. Neuron <i>94</i>, 891-907.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref1">View at publisher</a></p>
<p id="bib2">Barson, D., Hamodi, A.S., Shen, X., Lur, G., Constable, R.T., Cardin, J.A., Crair, M.C., and Higley, M.J. (2020). Simultaneous mesoscopic and two-photon imaging of neuronal activity in cortical circuits. Nat. Methods <i>17</i>, 107-113. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref2">View at publisher</a></p>
<p id="bib3">Brown, G.D., Yamada, S., and Sejnowski, T.J. (2001). Independent component analysis at the neural cocktail party. Trends Neurosci. <i>24</i>, 54-63. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref3">View at publisher</a></p>
<p id="bib4">Cardin, J.A., Crair, M.C., and Higley, M.J. (2020). Mesoscopic imaging: shining a wide light on large-scale neural dynamics. Neuron <i>108</i>, 33-43. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref4">View at publisher</a></p>
<p id="bib5">Cardoso, J.F. (1999). High-order contrasts for independent component analysis. Neural Comput. <i>11</i>, 157-192. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref5">View at publisher</a></p>
<p id="bib6">Chan, K.Y., Jang, M.J., Yoo, B.B., Greenbaum, A., Ravi, N., Wu, W., Sánchez-guardado, L., Lois, C., Mazmanian, S.K., Deverman, B.E., et al. (2017). Engineered adeno-associated viruses for efficient and noninvasive gene delivery throughout the central and peripheral nervous systems. Nat. Neurosci. <i>20</i>, 1172-1179. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref6">View at publisher</a></p>
<p id="bib7">Clancy, K.B., Orsolic, I., and Mrsic-Flogel, T.D. (2019). Locomotion-dependent remapping of distributed cortical networks. Nat. Neurosci. <i>22</i>, 778-786. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref7">View at publisher</a></p>
<p id="bib8">Couto, J., Musall, S., Sun, X.R., Khanal, A., Gluf, S., Saxena, S., Kinsella, I., Abe, T., Cunningham, J.P., Paninski, L., et al. (2021). Chronic, cortex-wide imaging of specific cell populations during behavior. Nat. Protoc. <i>16</i>, 3241-3263. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref8">View at publisher</a></p>
<p id="bib9">Gilad, A., Gallero-Salas, Y., Groos, D., and Helmchen, F. (2018). Behavioral strategy determines frontal or posterior location of short-term memory in neocortex. Neuron <i>99</i>, 814-828.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref9">View at publisher</a></p>
<p id="bib10">Kim, T.H., Zhang, Y., Lecoq, J., Jung, J.C., Li, J., Zeng, H., Niell, C.M., and Schnitzer, M.J. (2016). Long-term optical access to an estimated one million neurons in the live mouse cortex. Cell Rep. <i>17</i>, 3385-3394. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref10">View at publisher</a></p>
<p id="bib11">Lake, E.M.R., Ge, X., Shen, X., Herman, P., Hyder, F., Cardin, J.A., Higley, M.J., Scheinost, D., Papademetris, X., Crair, M.C., et al. (2020). Simultaneous cortex-wide fluorescence Ca<sup>2+</sup> imaging and whole-brain fMRI. Nat. Methods <i>17</i>, 1262-1271. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref11">View at publisher</a></p>
<p id="bib12">Liu, X., Ren, C., Lu, Y., Liu, Y., Kim, J.H., Leutgeb, S., Komiyama, T., and Kuzum, D. (2021). Multimodal neural recordings with Neuro-FITM uncover diverse patterns of cortical–hippocampal interactions. Nat. Neurosci. <i>24</i>, 886-896. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref12">View at publisher</a></p>
<p id="bib13">Lohani, S., Moberly, A.H., Benisty, H., Landa, B., Jing, M., Li, Y., Higley, M.J., and Cardin, J.A. (2020). Dual color mesoscopic imaging reveals spatiotemporally heterogeneous coordination of cholinergic and neocortical activity. BioRxiv. 418632. <a class="external-link" href="https://doi.org/10.1101/2020.12.09.418632">https://doi.org/10.1101/2020.12.09.418632</a></p>
<p id="bib14">Ma, Y., Shaik, M., Kim, S., Kozberg, M., Zhao, H., Yu, H., and Hillman, E. (2016). Wide-field optical mapping of neural activity and brain haemodynamics: considerations and novel approaches. Philos. Trans. R. Soc. Lond. B. Biol. Sci. <i>371</i>, 20150360. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref14">View at publisher</a></p>
<p id="bib15">Makino, H., Ren, C., Liu, H., Kim, A.N., Kondapaneni, N., Liu, X., Kuzum, D., and Komiyama, T. (2017). Transformation of cortex-wide emergent properties during motor learning. Neuron <i>94</i>, 880-890.e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref15">View at publisher</a></p>
<p id="bib16">Mayford, M., Bach, M.E., Huang, Y.Y., Wang, L., Hawkins, R.D., and Kandel, E.R. (1996). Control of memory formation through regulated expression of a CaMKII transgene. Science <i>274</i>, 1678-1683. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref16">View at publisher</a></p>
<p id="bib17">Michelson, N.J., Vanni, M.P., and Murphy, T.H. (2019). Comparison between transgenic and AAV-PHP.eB-mediated expression of GCaMP6s using in vivo wide-field functional imaging of brain activity. Neurophotonics <i>6</i>, 025014. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref17">View at publisher</a></p>
<p id="bib18">Musall, S., Kaufman, M.T., Juavinett, A.L., Gluf, S., and Churchland, A.K. (2019). Single-trial neural dynamics are dominated by richly varied movements. Nat. Neurosci. <i>22</i>, 1677-1686. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref18">View at publisher</a></p>
<p id="bib19">Nakamura, M., Yako, T., Kuse, Y., Inoue, Y., Nishinaka, A., Nakamura, S., Shimazawa, M., and Hara, H. (2018). Exposure to excessive blue LED light damages retinal pigment epithelium and photoreceptors of pigmented mice. Exp. Eye Res. <i>177</i>, 1-11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref19">View at publisher</a></p>
<p id="bib20">Peters, A.J., Fabre, J.M.J., Steinmetz, N.A., Harris, K.D., and Carandini, M. (2021). Striatal activity topographically reflects cortical activity. Nature <i>591</i>, 420-425. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref20">View at publisher</a></p>
<p id="bib21">Ren, C. and Komiyama, T. (2021). Characterizing cortex-wide dynamics with wide-field calcium imaging. J. Neurosci. <i>41</i>, 4160-4168. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref21">View at publisher</a></p>
<p id="bib22">Sahonero-Alvarez, G. and Calderon, H. (2017). A comparison of SOBI, FastICA, JADE and infomax algorithms. In IMCIC 2017 - 8th Int. Multi-conference Complexity, Informatics Cybern. Proc. 2017-march, pp. 17-22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref22">View at publisher</a></p>
<p id="bib23">Shimaoka, D., Steinmetz, N.A., Harris, K.D., and Carandini, M. (2019). The impact of bilateral ongoing activity on evoked responses in mouse cortex. Elife <i>8</i>, e43533. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref23">View at publisher</a></p>
<p id="bib24">Soleimanzad, H., Gurden, H., and Pain, F. (2017). Optical properties of mice skull bone in the 455- to 705-nm range. J. Biomed. Opt. <i>22</i>, 010503. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref24">View at publisher</a></p>
<p id="bib25">Valley, M.T., Moore, M.G., Zhuang, J., Mesa, N., Castelli, D., Sullivan, D., Reimers, M., and Waters, J. (2020). Separation of hemodynamic signals from GCaMP fluorescence measured with wide-field imaging. J. Neurophysiol. <i>123</i>, 356-366. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref25">View at publisher</a></p>
<p id="bib26">Wang, Q., Ding, S.L., Li, Y., Royall, J., Feng, D., Lesnar, P., Graddis, N., Naeemi, M., Facer, B., Ho, A., et al. (2020). The allen mouse brain common coordinate framework: a 3D reference atlas. Cell <i>181</i>, 936-953.e20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref26">View at publisher</a></p>
<p id="bib27">Wekselblatt, J.B., Flister, E.D., Piscopo, D.M., and Niell, C.M. (2016). Large-scale imaging of cortical dynamics during sensory perception and behavior. J. Neurophysiol. <i>115</i>, 2852-2866. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref27">View at publisher</a></p>
<p id="bib28">Xiao, D., Vanni, M.P., Mitelut, C.C., Chan, A.W., LeDue, J.M., Xie, Y., Chen, A.C., Swindale, N.V., and Murphy, T.H. (2017). Mapping cortical mesoscopic networks of single spiking cortical or sub-cortical neurons. Elife <i>6</i>, e19976. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref28">View at publisher</a></p>
<p id="bib29">Xie, Y., Chan, A.W., McGirr, A., Xue, S., Xiao, D., Zeng, H., and Murphy, T.H. (2016). Resolution of high-frequency mesoscale intracortical maps using the genetically encoded glutamate sensor iGluSnFR. J. Neurosci. <i>36</i>, 1261-1272. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref29">View at publisher</a></p>
<p id="bib30">Yizhar, O., Fenno, L.E., Davidson, T.J., Mogri, M., and Deisseroth, K. (2011). Optogenetics in neural systems. Neuron <i>71</i>, 9-34. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00679-1/sref30">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Takaki Komiyama (<a href="mailto:tkomiyama@ucsd.edu">tkomiyama@ucsd.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new materials.</p>
<h4>Data and code availability</h4>
<p>Data are available from the corresponding author upon reasonable request. The custom code for data processing is available at GitHub (<a href="https://github.com/CRen2333/Wide-field-calcium-imaging">https://github.com/CRen2333/Wide-field-calcium-imaging</a>).</p>
<h3>Acknowledgments</h3>
<p>This work was supported by grants from <a href="https://doi.org/10.13039/100000052">NIH</a> (R01 NS091010, R01 EY025349, R01 DC014690, and P30 EY022589), <a href="https://doi.org/10.13039/100000001">NSF</a> (1940202), and <a href="https://doi.org/10.13039/100000008">David &amp; Lucile Packard Foundation</a> to T.K.</p>
<h3>Author contributions</h3>
<p>C.R. performed the surgery, imaging experiments, and reprocessing of imaging data. C.R. wrote the manuscript with inputs from T.K.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>