<article>
<section>
<h1 id="header">Single-cell lactate production rate as a measure of glycolysis in endothelial cells</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Devin Harrison,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">4</a></sup> David Wu,<sup><a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a></sup> Jun Huang,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#cor1">*</a></sup> and Yun Fang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Graduate Program in Biophysical Sciences, The University of Chicago, Chicago, IL 60637, USA</p>
<p id="aff2"><sup>2</sup>Department of Medicine, Biological Sciences Division, The University of Chicago, Chicago, IL 60637, USA</p>
<p id="aff3"><sup>3</sup>Pritzker School of Molecular Engineering, The University of Chicago, Chicago, IL 60637, USA</p>
<p id="fn1"><sup>4</sup>These authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:huangjun@uchicago.edu">huangjun@uchicago.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:yfang1@medicine.bsd.uchicago.edu">yfang1@medicine.bsd.uchicago.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100807">10.1016/j.xpro.2021.100807</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Heterogeneous metabolism supports critical single-cell functions. Here, we describe deep-learning-enabled image analyses of a genetically encoded lactate-sensing probe which can accurately quantify metabolite levels and glycolytic rates at the single-cell level. Multiple strategies and test data have been included to obviate possible obstacles including successful sensor expression and accurate segmentation. This protocol reliably discriminates between metabolically diverse subpopulations which can be used to directly link metabolism to functional phenotypes by integrating spatiotemporal information, genetic or pharmacological perturbations, and real-time metabolic states.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1038/s42255-021-00390-y">Wu et al. (2021a)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for reliable discrimination of heterogenous single-cell glycolytic states</li>
<li>Multiple strategies for metabolite-sensing probe expression in primary cells</li>
<li>Deep-learning-enabled image segmentation test data and pipeline</li>
<li>Accurate quantification and error estimation of lactate levels and glycolytic rates</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol involves imaging of endothelial cells expressing a genetically encoded sensor for lactate, Laconic (<a href="#bib6">San Martín et al., 2013</a>), within commercially available Ibidi imaging chambers capable of fluid exchange. Laconic is composed of a bacterially-derived lactate binding region flanked by two fluorophores, mTFP and Venus, capable of undergoing fluorescence resonance energy transfer (FRET). In the presence of lactate, the fluorophores will increase in molecular distance, thus reducing FRET. Prior to evaluating the lactate production rate, the sensor, imaging chamber, microscope, and computer for analysis must be set up.</p>
<h3 id="sec1.1">Prepare laconic for genetic introduction to cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–4 weeks</div>
<div class="note">
<span class="note-title">Note:</span> Laconic can be introduced to cells using a variety of transfection methods. We found adenoviral transduction into human aortic endothelial cells (HAEC) resulted in reliably homogenous expression and minimal cell death but have also used mRNA transfection in both primary cells and cell lines. See <a href="#fig1">Figure 1</a> for comparison of expression via these methods. For generating FRET control proteins, please see Additional Controls.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Comparison of different methods of Laconic expression in endothelial cells</div>
<p>Comparison of mRNA versus adenoviral introduction of Laconic into human aortic endothelial cells after 48 h mRNA transfections can sometimes lead to cell elongation whereas transduced cells retain their original cobblestone-like morphology. Scale bars = 200 μm.</p>
<p>(A) 160 ng/cm<sup>2</sup> Laconic mRNA transfection.</p>
<p>(B) 50 MOI ad-Lac transduction.</p>
</figcaption>
</figure>
<ol>
<li>For adenoviral transduction:
<ol type="a">
<li>Acquire Laconic plasmid from Addgene (#44238)</li>
<li>Entrust Vector Biolabs to create Ad5 adenoviral Laconic from the Addgene plasmid (<a href="https://www.vectorbiolabs.com/adenovirus-construction-service/">Link to manufacturer’s services</a>)</li>
</ol>
</li>
</ol>
<ol start="2">
<li>For mRNA production:
<ol type="a">
<li>Generate template for <i>in vitro</i> transcription via PCR using the T7-F and BGH-R primers (see <a href="#key-resources-table">key resources table</a>) and protocol listed in <a href="#tbl1">Table 1</a> and NEB Q5 Polymerase
<table id="tbl1">
<caption>Table 1. Generating Laconic template for <i>in vitro</i> transcription</caption>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>63°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1.5 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>Forever</td>
<td> </td>
</tr>
</tbody>
</table>
<p class="table-legend">Forward (T7-F) and reverse (BGH-R) primers are used in the following PCR reaction in order to generate the template for <i>in vitro</i> transcription. PCR reaction steps are based on NEB Q5 Polymerase usage guidelines expecting a 2403 bp amplicon.</p>
</li>
<li>Purify PCR product using QIAquick PCR Purification Kit (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/dna-clean-up/qiaquick-pcr-purification-kit/">Link to manufacturer's protocol</a>)
<div class="note">
<span class="note-title">Note:</span> PCRs can be checked by running on a 1% agarose gel for the correct amplicon size (2403 bp).</div>
</li>
<li>Using the PCR-generated template from 2b, create mRNA using mMESSAGE mMACHINE™ T7 ULTRA Transcription Kit (<a href="https://www.thermofisher.com/order/catalog/product/AM1344">Link to manufacturer's protocol</a>)</li>
<li>After cleaning and eluting mRNA, aliquot (100 ng/μL) and store at −80°C up to 6 months for minimal degradation. (<a href="https://www.thermofisher.com/order/catalog/product/AM1908">Link to manufacturer's protocol</a>)
<ol type="i">
<li>When eluting mRNA, preheat the elution solution to 95°C and elute three times from one column using MEGAclear™ Transcription Cleanup Kit.</li>
<li>Measure the concentration of the mRNA with a spectrophotometer, such as a BioTek Cytation 3 instrument or NanoDrop, using 1–2 μL of sample.</li>
<li>Concentration of the mRNA can be adjusted with the supplied elution buffer.</li>
<li>The lithium chloride precipitation method in the mMESSAGE mMACHINE™ T7 ULTRA Transcription Kit is also sufficient.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> In addition to Vector Biolabs, adenoviruses can be generated in house (<a href="https://www.addgene.org/guides/adenovirus/">Link to useful protocol</a>) and is reviewed in (<a href="#bib3">Danthinne and Imperiale, 2000</a>). The University of Iowa has a viral vector core facility (<a href="https://medicine.uiowa.edu/vectorcore/about-our-products/adenovirus-products/custom-adenovirus-constructs">Link to website</a>). Other commercial sources are also available such as Vigene Biosciences (<a href="https://www.vigenebio.com/adenovirus-packaging/">Link to manufacturer’s service</a>) and abm (<a href="https://www.abmgood.com/Custom-Recombinant-Adenovirus-Service.html">Link to manufacturer’s service</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Create mRNA transcripts for mTFP and Venus separately (the FRET pair in Laconic) for testing fluorescence bleed through in the filter sets in the microscope (see Additional Controls section).</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Culturing human aortic endothelial cells (HAECs)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 h</div>
<ol start="3">
<li>Culture HAECs in a 37°C, 5% CO<sub>2</sub> with humidity incubator. HAECs are suitable to use after they have been passaged at least once.
<ol type="a">
<li>Pipette 10 mL of EGM2 complete media into a T75 flask and let warm in incubator.</li>
<li>Rapidly defrost HAECs in a 37°C water bath, shaking.</li>
<li>Immediately after defrosting, spray the cryo-tube with 70% ethanol and wipe off. In a sterile hood with laminar flow, pipette cells into the pre-warmed T75 flask. Wash the inside of the cryo-tube with media, aspirate, and pipette into T75 flask.</li>
<li>After 4 h and after cells have adhered to flask, aspirate media and add new, 37°C warmed complete media (to remove DMSO).</li>
<li>Change media on HAECs every 2–3 days. Split when 80% confluent (see section on plating cells below for further instructions).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> HAECs are recommended for use within passages 4–8.</div>
<h3 id="sec1.3">Setup of ibidi imaging chambers</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>Ibidi imaging chambers are capable of fluid exchange which will be critical for establishing the calibration curve so that you are able to convert fluorescent values to lactate concentrations. See <a href="#fig2">Figure 2</a> for assembly.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Ibidi imaging chamber assembly</div>
<p>Components of the Ibidi imaging chamber are shown separated in (A) and assembled in (B).</p>
<p>(A) Male connector (1) is attached to tubing (2) to form the output or waste end of the apparatus which will connect to the Ibidi plate (3). Male connector (4) is attached to tubing (5) then to a female connector (6). The in-line injection port (7) is attached to the female end (6) to form the input or injection end of the apparatus. Following rinsing, the cap (8) must be added to close the input port. Once the input and output lines are assembled, they may be reused for multiple Ibidi plates.</p>
<p>(B) When performing imaging experiments, connect the input line before the output line into the bottom and top ports of one well in the Ibidi plate (3).</p>
</figcaption>
</figure>
<ol start="4">
<li>Measure tubing length so that it can be used in your microscope set-up
<ol type="a">
<li>Ideally, your microscope will have a chamber for controlling heat, humidity, and CO<sub>2</sub>. In order to have fluid exchange without disturbing the chamber, the tubing should be long enough to reach outside of the chamber. The plastic versions of the Ibidi chambers are permeable to gas.</li>
<li>Test the volume of tubing you have by injecting a known amount of liquid through the line, then measuring how much comes out the other end. This will be important for determining the amount of solution you’ll need to inject for thorough washing.</li>
</ol>
</li>
<li>Assemble waste/output tubing
<ol type="a">
<li>Attach one set of tubing to one elbow male connectors</li>
</ol>
</li>
<li>Assemble input/injection tubing
<ol type="a">
<li>Attach other set of tubing to one elbow male connector on left side</li>
<li>Attach female connector on opposite side</li>
<li>In-line injection port can be attached to female connector</li>
<li>Rinse thoroughly with 70% ethanol and PBS using syringes</li>
</ol>
</li>
<li>Tubing can be saved for multiple experiments as long as the assembled apparatus is rinsed thoroughly with 70% ethanol and PBS between samples</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Human VE-Cadherin/CDH5 (D87F2) XP® Rabbit mAb (1:100)</td>
<td>Cell Signaling Technology</td>
<td>Cat#2500S; RRID# AB_10839118</td>
</tr>
<tr>
<td>Alexa Fluor 488 Goat Anti-Rabbit IgG (1:1000)</td>
<td>Life Technologies</td>
<td>Cat#A11034; RRID# AB_2576217</td>
</tr>
<tr>
<td>Alexa Fluor 594 Goat Anti-Mouse IgG (1:1000)</td>
<td>Life Technologies</td>
<td>Cat#A11037; RRID# AB_2534095</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Adenovirus-Laconic</td>
<td>This paper, generated by Vector Biolabs</td>
<td><a href="https://www.vectorbiolabs.com/adenovirus-construction-service/">https://www.vectorbiolabs.com/adenovirus-construction-service/</a></td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Dimethyloxalylglycine (DMOG)</td>
<td>Sigma-Aldrich</td>
<td>Cat#D3695</td>
</tr>
<tr>
<td>4-Chloromercuribenzoic acid (pCMBA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#C5913</td>
</tr>
<tr>
<td>Phloretin</td>
<td>Sigma-Aldrich</td>
<td>Cat#P7912</td>
</tr>
<tr>
<td>Nigericin</td>
<td>Tocris</td>
<td>Cat#4312</td>
</tr>
<tr>
<td>Glucose</td>
<td>Sigma-Aldrich</td>
<td>Cat#G8270</td>
</tr>
<tr>
<td>Sodium lactate</td>
<td>Sigma-Aldrich</td>
<td>Cat#L7022-5G</td>
</tr>
<tr>
<td>Rotenone</td>
<td>Sigma-Aldrich</td>
<td>Cat#R8875</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>Sigma-Aldrich</td>
<td>Cat#S3014-25KG</td>
</tr>
<tr>
<td>Potassium chloride (KCl)</td>
<td>Sigma-Aldrich</td>
<td>Cat#P9333-500G</td>
</tr>
<tr>
<td>HEPES (solid)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#BP310-500</td>
</tr>
<tr>
<td>HEPES (solution)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15630080</td>
</tr>
<tr>
<td>Calcium chloride (CaCl<sub>2</sub>)</td>
<td>Sigma-Aldrich</td>
<td>Cat#C8106</td>
</tr>
<tr>
<td>Magnesium sulfate (MgSO4)</td>
<td>Sigma-Aldrich</td>
<td>Cat# M7506</td>
</tr>
<tr>
<td>Sodium bicarbonate (NaHCO<sub>3</sub>)</td>
<td>Sigma-Aldrich</td>
<td>Cat#S5761-500G</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS), pH 7.4, without Ca<sup>++</sup> or Mg<sup>++</sup>
</td>
<td>Invitrogen</td>
<td>Cat#10010</td>
</tr>
<tr>
<td>Dulbecco’s phosphate buffer saline (DPBS) with Ca<sup>++</sup> or Mg<sup>++</sup>
</td>
<td>Invitrogen</td>
<td>Cat#14040</td>
</tr>
<tr>
<td>TrypLE express</td>
<td>Gibco</td>
<td>Cat#12605010</td>
</tr>
<tr>
<td>Fluorobrite DMEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A1896601</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Life Technologies</td>
<td>Cat#31985-070</td>
</tr>
<tr>
<td>DMEM high glucose</td>
<td>Invitrogen</td>
<td>Cat#NP0366</td>
</tr>
<tr>
<td>GeneJammer Transfection Reagent</td>
<td>Agilent</td>
<td>Cat#204130</td>
</tr>
<tr>
<td>MessengerMAX</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#LMRNA003</td>
</tr>
<tr>
<td>EGM2 complete media</td>
<td>Lonza</td>
<td>Cat#CC4176 and CC3156</td>
</tr>
<tr>
<td>Paraformaldehyde, 32%</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#15710</td>
</tr>
<tr>
<td>Triton-X 100</td>
<td>Sigma-Aldrich</td>
<td>Cat#X100-100ML</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A2153</td>
</tr>
<tr>
<td>Tris-buffered saline (TBS), 10<b>×</b>
</td>
<td>Sigma-Aldrich</td>
<td>Cat#T5912</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma-Aldrich</td>
<td>Cat#P9416</td>
</tr>
<tr>
<td>Mounting medium</td>
<td>Thermo Fisher</td>
<td>Cat#P36965</td>
</tr>
<tr>
<td>Hoechst33342</td>
<td>Invitrogen</td>
<td>Cat#H3570</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>mMESSAGE mMACHINE™ T7 ULTRA Transcription Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM1344</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>QIAGEN</td>
<td>Cat#28104</td>
</tr>
<tr>
<td>MEGAclear™ Transcription Clean-Up Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM1908</td>
</tr>
<tr>
<td>NEB Q5® High-Fidelity 2X Master Mix</td>
<td>New England Biolabs</td>
<td>Cat# M0492S</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Sample data (<a href="#bib8">Wu et al., 2021</a>b)</td>
<td><a href="http://Zenodo.org">Zenodo.org</a></td>
<td>https://doi.org/10.5281/zenodo.4638059</td>
</tr>
<tr>
<td>Ground truth sample data set and sample network (<a href="#bib4">Harrison, Devin et al., 2021</a>)</td>
<td><a href="http://Zenodo.org">Zenodo.org</a></td>
<td>https://doi.org/10.5281/zenodo.4898134</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Human aortic endothelial cells (HAECs)</td>
<td>Lonza</td>
<td>Cat#CC-2535</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>T7-F, TAATACGACTCACTATAGGG</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>BGH-R, TAGAAGGCACAGTCGAGG</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>mTFPr, TTAGGTACCAGATCTCTTGTA</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td>T7-Venus, TTAATATAATACGACTCACTATAGGGACGATTCGATGAAGATCAG</td>
<td>IDT</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Laconic (<a href="#bib6">San Martín et al., 2013</a>)</td>
<td>Addgene</td>
<td>Cat#44238</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>CellProfiler 3.1.8 (<a href="#bib1">Carpenter et al., 2006</a>)</td>
<td>Broad Institute</td>
<td><a href="https://cellprofiler.org/">https://cellprofiler.org/</a></td>
</tr>
<tr>
<td>MATLAB R2018b, including Deep Learning Toolbox</td>
<td>MathWorks</td>
<td><a href="https://www.mathworks.com/products/deep-learning.html">https://www.mathworks.com/products/deep-learning.html</a></td>
</tr>
<tr>
<td>Prism 8</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td>MicroManager</td>
<td>Vale Lab, UCSF</td>
<td><a href="https://micro-manager.org/">https://micro-manager.org/</a></td>
</tr>
<tr>
<td>MATLAB code and scripts</td>
<td>This paper</td>
<td><a href="https://github.com/wulab-code/STAR_methods">https://github.com/wulab-code/STAR_methods</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Spectrophotometer</td>
<td>BioTek Cytation 3</td>
<td>Cat#CYT3MF</td>
</tr>
<tr>
<td>Olympus IX-71 epifluorescence microscope</td>
<td>Olympus</td>
<td>N/A</td>
</tr>
<tr>
<td>Objective (10<b>×</b> Plan C Achromat, NA 0.25)</td>
<td>Olympus</td>
<td>N/A</td>
</tr>
<tr>
<td>Excitation filter 438/24</td>
<td>Semrock</td>
<td>Cat#FF02-438/24-25</td>
</tr>
<tr>
<td>mTFP emission 483/32</td>
<td>Semrock</td>
<td>Cat#FF01-483/32-25</td>
</tr>
<tr>
<td>Venus emission 542/27</td>
<td>Semrock</td>
<td>Cat#FF01-542/27-25</td>
</tr>
<tr>
<td>FRET dichroic FF458-Di02</td>
<td>Semrock</td>
<td>Cat#FF458-Di02-25<b>×</b>36</td>
</tr>
<tr>
<td>Hoechst excitation 387/50</td>
<td>Semrock</td>
<td>Cat#FF01-387/11-25</td>
</tr>
<tr>
<td>Hoechst emission, 447/60</td>
<td>Semrock</td>
<td>Cat#FF02-447/60-25</td>
</tr>
<tr>
<td>Hoechst dichroic FF409-Di03</td>
<td>Semrock</td>
<td>Cat#FF409-Di03-25<b>×</b>36</td>
</tr>
<tr>
<td>TRITC excitation 534/22</td>
<td>Semrock</td>
<td>Cat#FF01-543/22-25</td>
</tr>
<tr>
<td>TRITC emission 593/40</td>
<td>Semrock</td>
<td>Cat#FF01-593/40-25</td>
</tr>
<tr>
<td>TRITC dichroic FF562-Di03</td>
<td>Semrock</td>
<td>Cat#FF562-Di03-25<b>×</b>36</td>
</tr>
<tr>
<td>Prime 95B</td>
<td>Photometrics</td>
<td>N/A</td>
</tr>
<tr>
<td>EXFO X-Cite 120Q</td>
<td>Excelitas</td>
<td>Cat#010–00157</td>
</tr>
<tr>
<td>Controller for shutter and filter wheel</td>
<td>Prior</td>
<td>Cat#ProscanIII</td>
</tr>
<tr>
<td>Automated excitation shutter</td>
<td>Prior</td>
<td>Cat#HF202HT</td>
</tr>
<tr>
<td>Automated emission filter wheel</td>
<td>Prior</td>
<td>Cat#HF108IX3</td>
</tr>
<tr>
<td>μ-Slide VI Luer, ibidi imaging chambers</td>
<td>ibidi</td>
<td>Cat#80606</td>
</tr>
<tr>
<td>5 m Silicone tubing 0.8 mm ID</td>
<td>ibidi</td>
<td>Cat#10841</td>
</tr>
<tr>
<td>50 Elbow Luer Connector Male</td>
<td>ibidi</td>
<td>Cat#10802</td>
</tr>
<tr>
<td>25 Luer Lock Connector Female</td>
<td>ibidi</td>
<td>Cat#10825</td>
</tr>
<tr>
<td>In-line Luer Injection Port</td>
<td>ibidi</td>
<td>Cat#10820</td>
</tr>
<tr>
<td>BD needle 25 g 0.62 in</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10777-019</td>
</tr>
<tr>
<td>1 mL Syringes</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#14–829-45</td>
</tr>
<tr>
<td>10 mL Syringe</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#14–955-459</td>
</tr>
<tr>
<td>75cm<sup>2</sup> Tissue Culture Flask</td>
<td>DOT Scientific</td>
<td>Cat# 557341</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Stock solutions (all filter sterilized)</h3>
<p>pCMBA (MCT inhibitor), 500 mM in DMSO, store at −20°C for 1 year</p>
<p>Phloretin (alternative MCT inhibitor for pCMBA), 100 mM in DMSO, store at −20°C for 1 year</p>
<p>Glucose, 1 M in ddH<sub>2</sub>O, store at 4°C long term (1 year) or room temperature (20°C–22°C) short term (1 month)</p>
<p>Nigericin, 10 mM in absolute ethanol, store at −20°C for 1 year</p>
<p>Rotenone, 2.5 mM in DMSO, store at −20°C for 1 year</p>
<p>Sodium Lactate, 1 M in water, store at room temperature for 1 year</p>
<h3 id="sec2.2">Additional solutions</h3>
<p>TBST. Dilute 10<b>×</b> TBS into ddH<sub>2</sub>O, and add 0.1% Tween-20, mix thoroughly. Store at room temperature for 1 year</p>
<p>Paraformaldehyde, 4%. Dilute stock (32%) into PBS, pH 7.4 to make 4% solution before use. Can be frozen at −20°C for up to 12 months.</p>
<p>Blocking buffer: TBST with 3% BSA. Store at 4°C for 6 months.</p>
<p>Viral Media: DMEM high glucose with 10mM HEPES. Store at 4°C for 6 months.</p>
<table id="undtbl1">
<caption>Intracellular Buffer (ICB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>10 mM</td>
</tr>
<tr>
<td>KCl</td>
<td>130 mM</td>
</tr>
<tr>
<td>HEPES</td>
<td>1.25 mM</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
</tr>
</tbody>
</table>
<p class="table-legend">Filter sterilize and store at room temperature (20°C–22°C) for 2 years.</p>
<table id="undtbl2">
<caption>Extracellular Buffer (ECB)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration (mM)</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>112 mM</td>
</tr>
<tr>
<td>KCl</td>
<td>5 mM</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>1.25 mM</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>1.25 mM</td>
</tr>
<tr>
<td>HEPES</td>
<td>10 mM</td>
</tr>
<tr>
<td>NaHCO<sub>3</sub>
</td>
<td>24 mM</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
</tr>
</tbody>
</table>
<p class="table-legend">pH to 7.4, filter sterilize, and store at room temperature (20°C–22°C) for 2 years.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Phloretin can be used in place of pCMBA for MCT blockade (see Limitations). Trypsin can be used instead of TrypLE.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Any software can be used to generate figures from analyzed data but we use Prism 8.</div>
<div class="note">
<span class="note-title">Note:</span> ICB and ECB are shelf stable for up to two years but should be checked for formation of precipitate.</div>
<h3 id="sec2.3">Ensure optical setup works with laconic</h3>
<p>Laconic is comprised of a bacterial lactate binding domain (LldR) flanked by two fluorophores capable of undergoing Fluorescence resonance energy transfer (FRET). As lactate concentration increases, the FRET ratio between the two fluorophores (mTFP and Venus) is reduced (<a href="#bib6">San Martín et al., 2013</a>). Acquire filter sets that allow for excitation of mTFP and allow for emission in both the donor (mTFP) and acceptor (Venus) channels. Additional sets should be acquired for nuclei and membrane labeling (Hoechst and VE-CADH, respectively) but could also be used for multiplexed imaging.</p>
<p>This experiment requires a standard inverted microscope with either a stage top incubator on top or enclosure. Our heated, humidified, and CO2 controlled incubator was home built, but commercial ones are available – we recommend Okolab (<a href="http://www.oko-lab.com/live-cell-imaging/cage-incubator">Link to manufacturer’s website</a>) or Ibidi (<a href="https://ibidi.com/stage-top-incubators/230-ibidi-stage-top-incubation-system-universal-fit-for-1-chamber-co2.html">Link to manufacturer’s website</a>). No special modifications to the microscope are necessary except for the incubator conditions and automated filter wheel; thus, any inverted microscope could in theory work. We suggest also having a computer-controlled shutter for automatically shuttering the emission light in between frame acquisitions to reduce photobleaching. For acquiring the emission in two channels, an automated emission filter wheel must be used (see <a href="#key-resources-table">key resources table</a>). We used an epi-illumination metal halide light source. However, an LED light source should also be sufficient. Micro-manager was used to control the automated shutter and filter wheel through a controller, as well as camera.</p>
<ul>
<li>Olympus IX-71 epifluorescence microscope with a 10× 0.25 numerical aperture (NA) air objective and Prior automated emission filter wheel and excitation shutter. We chose a 10× objective for maximal data acquisition while still being able to segment the cells. Furthermore, the high depth of field allowed us to capture the fluorescence of the entire cell without resorting to z-stacks.</li>
<li>FRET filter sets were from Semrock: excitation 438/24, mTFP emission 483/32, Venus emission 542/27, and dichroic FF458-Di02.</li>
<li>Blue (Hoechst) and red (TRITC) filter sets for visualizing Hoechst stain and VE-CADH were from Semrock: Hoechst excitation 387/50, Hoechst emission, 447/60, Hoechst dichroic FF409-Di03, TRITC excitation 543/22, TRITC emission 593/40, TRITC dichroic FF562-Di03.</li>
<li>Illumination: EXFO X-Cite metal halide light source</li>
<li>Camera: Photometrics Prime95B. CMOS camera with high sensitivity, dynamic range, as well as speed. The protocol will likely work with other cameras – integration times may vary.</li>
</ul>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Instead of using an automated emission filter wheel, split emission systems such as the Optisplit II by Cairn (<a href="https://www.cairn-research.co.uk/product/optosplit-ii/">Link to Manufacturer’s website</a>) can be employed for simultaneous acquisition of both emission channels, although this will change the downstream analysis considerably.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Filter sets with very little bleed through are necessary to avoid misinterpretation of FRET. For instance, the excitation in the mTFP channel should minimally excite Venus. The filter sets specified in this protocol minimize bleed through using the excitation light source specified; inclusion of bleed through did not change the calculation of FRET. Changing the light source may change the bleed through characteristics (see Additional Controls).</div>
<div class="optional">
<span class="optional-title">Optional:</span> Additional filter sets such as those used for TRITC, are not necessary but increase the potential for multiplexed imaging.</div>
<h3 id="sec2.4">Deep learning computer</h3>
<p>For training of the segmentation network, a computer outfitted with GPUs will drastically improve training time and segmentation. The computer should also have enough RAM for holding batches of images in memory when performing training. A minimum of 64 GB of memory is recommended. A CPU that has multiple cores is also critical for downstream analysis. For a quick comparison of CPUs check <a href="http://Geekbench.com">Geekbench.com</a>. Pcpartpicker.com also has useful guides and compatibility charts for parts for building one’s own computer, as the pricing and availability of PC parts rapidly evolves. Computers sufficient to perform relatively quick computations are also available from retailers such as Lamda Labs, Bizon Tech, or Nvidia.</p>
<ul>
<li>We used 2 NVIDIA GeForce 1080 Ti and 1 NVIDIA Titan V GPUs.</li>
<li>Our CPU is an AMD Ryzen Threadripper 12-core processor running at 3.50 GHz built in 2018.</li>
</ul>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> It is possible to perform deep learning on cloud computing servers readily available in large Universities or private providers such as Amazon Web Services. The code in this document assumes that computation is occurring locally on a Windows PC.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Introduce laconic genetically: Day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.4 h with overnight (12–24 h) incubation</div>
<p>In order to have cells expressing Laconic at highest levels, introduce laconic adenovirus vector (ad-Lac) 2 days prior to imaging.</p>
<ol>
<li>Plating cells
<ol type="a">
<li>Aspirate media from T75 containing HAEC that is ∼80–100% confluent and wash flask with warmed, sterile 5 mL PBS and aspirate.</li>
<li>Add 1mL TrypLE and incubate in 37°C, 5% CO2 with humidity incubator for 1–2 min.</li>
<li>Check cells under microscope. If cells have become rounded, tap flask gently 10–20 times so that they come off. If cells have not become rounded, wait 1 min. Do not exceed 3 min total. Aim for ∼80% of cells to detach.</li>
<li>Add 5 mL EGM2 complete media to flask, washing the bottom of the flask to remove any adherent cells.</li>
<li>Aspirate cells and spin at 100 g, 5 min at room temperature.</li>
<li>Aspirate supernatant and resuspend cells in 1mL of EGM2 complete media.</li>
<li>Count cells.</li>
<li>Seed cells into wells so that there are ∼200,000 cells/well (in 12 well plate) with enough time before transduction for the cells to adhere (1 day for HAECs).</li>
</ol>
</li>
<li>Prepare reagents
<ol type="a">
<li>Warm viral media and complete media in 37°C water bath</li>
<li>Defrost adenovirus laconic stock from −80°C on ice</li>
</ol>
</li>
<li>Prepare transduction master mix for half culture volume (500 μL for 12 well)
<ol type="a">
<li>Multiplicity of infection (MOI) = Plaque forming units (PFU) of virus used for infection / number of cells. We recommend starting with a MOI of 50:1.
<ol type="i">
<li>E.g.: for a virus of 3.40E+10 pfu/mL to be used at 50:1 MOI used in 200,000 cells/12 well, use 0.29 μL virus and 1.5 μL GeneJammer in 500 μL viral media per well</li>
</ol>
</li>
<li>Make master mix if using multiple wells</li>
<li>Mix by gentle swirling/pipetting. Do not vortex.</li>
</ol>
</li>
<li>Aspirate viral media and apply transduction mixture to each well
<ol type="a">
<li>Incubate for 1 h in 37°C at 5% CO<sub>2</sub>
</li>
</ol>
</li>
<li>Remove viral media gently and replace with complete media</li>
<li>Incubate overnight (12–24 h) at 37°C, 5% CO2, humidified.</li>
<li>Visualize transfection efficiency on a fluorescence microscope (<a href="#fig1">Figure 1</a>). Transfection efficiency can be roughly estimated by using a microscope and looking for signs of altered cell morphology which signals unhealthy cells. Every cell should be fluorescent with enough signals to obtain an accurate intracellular lactate calibration curve using a reasonable camera integration time (<a href="#fig3">Figure 3</a>). Too much adenovirus (in our hands, 200:1 MOI) results in altered cell morphology and cell death.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Laconic calibration curve</div>
<p>mTFP/Venus (or 1/FRET) change is linearly correlated over 6 orders of magnitude of log lactate (n = 287 cells; black line is semilog fit with R<sup>2</sup> = 0.96, gray dotted lines are 95% confidence interval); error bars are SEM. Standard deviation of regression coefficients for R<sup>2</sup> for is 0.049.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Virus must be kept on ice to ensure transduction efficiency. Virus should be aliquoted appropriately so that each tube isn’t thawed more than twice.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Anything containing virus must be used with caution according to BSL-2 guidelines. Disposables and solutions containing virus must be disposed of separately from other reagents in 10% bleach.</div>
<div class="note">
<span class="note-title">Note:</span> GeneJammer is not necessary but will improve expression.</div>
<div class="note">
<span class="note-title">Note:</span> Solution changes must be done gently following transduction as the cells are especially sensitive to lifting off the plate.</div>
<h3 id="sec3.2">mRNA transfection of laconic (alternative to adenovirus)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h with overnight (12–24 h) incubation</div>
<p>For mRNA transfection using Lipofectamine<sup>TM</sup> MessengerMax <sup>TM</sup> reagent:</p>
<ol start="8">
<li>Prepare cells
<ol type="a">
<li>Seed adherent cells into wells so that there are ∼200,000 cells/well (in 12 well plate) with enough time before transduction for the cells to adhere (1 day for human aortic endothelial cells). See above on plating cells for directions.</li>
</ol>
</li>
<li>Prepare reagents
<ol type="a">
<li>Warm Opti-MEM and EGM2 complete media in 37°C water bath</li>
<li>Defrost mRNA stock from −80°C on ice</li>
</ol>
</li>
<li>Make master mix of lipofectamine MessengerMax reagent (Link to manufacturer's protocol)
<ol type="a">
<li>for 12 well plate: 50 μL total solution = 3 μL MessengerMax into 47 μL Opti-MEM.</li>
<li>Create master mix for multiple wells</li>
<li>Vortex master mix and incubate at room temperature for 10 min while you prepare mRNA mixes</li>
</ol>
</li>
<li>Prepare mRNA solutions so that final concentration is 160 ng/cm<sup>2</sup>
<ol type="a">
<li>For 4 cm<sup>2</sup> well (12 well), 50 μL solution = 7.1 μL mRNA (100 ng/cm<sup>2</sup>) into 42.8 μL Opti-MEM</li>
<li>Mix by triturating</li>
</ol>
</li>
<li>Add Master mix of lipofectamine reagent to mRNA solutions in 1:1 volume ratio (50 μL:50 μL for 12 well)
<ol type="a">
<li>Mix by triturating</li>
<li>Incubate at room temperature for 5 min</li>
</ol>
</li>
<li>During incubation in step 12b, replace media in wells containing cells to 900 μL complete media total</li>
<li>Add 100 μL transfection solution to each well</li>
<li>Incubate overnight for a minimum of 12 h up to 24 h.</li>
<li>Visualize transfection efficiency on a fluorescence microscope (<a href="#fig1">Figure 1</a>). In HAECs, mRNA transfection is noticeably less efficient than using adenovirus (<a href="#fig1">Figure 1</a>) but still useable from a data acquisition standpoint. Too little mRNA and fluorescence is not enough, but too much mRNA causes cell death.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> mRNA must be kept on ice to minimize mRNA degradation. mRNA should be aliquoted so that each aliquot isn’t thawed more than twice.</div>
<div class="note">
<span class="note-title">Note:</span> mRNA transfections can be further optimized for your cells by a dose titration. We typically try a range of 25–200 ng/cm<sup>2</sup>.</div>
<h3 id="sec3.3">Plate cells into imaging chambers: Day 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<p>For ease of solution switching, cells must be plated into Ibidi flow chambers. However, this environment is slightly hypoxic with a smaller volume than traditional cell culture plates so cells must have media changes every 24 h (<a href="#fig2">Figure 2</a>).</p>
<ol start="17">
<li>Warm PBS, TrypLE, and cell media in 37°C water bath</li>
<li>For adherent cells, wash cells twice with PBS and remove by aspiration.</li>
<li>Add TrypLE to virally transduced cells (50 μL per 12-well), incubate for 2 min at 37°C and 5% CO<sub>2</sub>
<ol type="a">
<li>Check for adequate detachment after 1 min, can tap dish to mechanically dislodge cells</li>
<li>When 80% cells have detached, halt trypsinization by adding at least 3× volume complete media</li>
</ol>
</li>
<li>Spin cells down at 900 g at room temperature and resuspend to 75,000 cells/50 μL (for Ibidi 6-well plate) complete media
<ol type="a">
<li>Tip Ibidi plate and add 50 μL into the elevated port, pipetting as close to the plate as possible, to ensure the cells spread between ports, incubate for 20 min at room temperature in the sterile hood.</li>
<li>Add 60 μL complete media to each port to prevent drying</li>
</ol>
</li>
<li>Incubate overnight (12–24 h) to ensure attachment</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When plating cells, make sure the solution of suspended cells is adequately mixed. This will ensure the sample number of cells makes it into each well and therefore the cells will be at the same density. We note changes in cell density may have an effect on their metabolism.</div>
<h3 id="sec3.4">Prepare injection solutions: Day 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>Pharmacological perturbation requires two solutions which must be made fresh before each experiment. Since Laconic measures glycolysis, it is important to block any mitochondrial consumption of pyruvate which is accomplished with rotenone. Nigericin is an ionophore used to clamp the intracellular pH.</p>
<ol start="22">
<li>Warm Fluorobrite media and ICB in 37°C water bath.</li>
<li>Thaw stock solution of pCMBA and nigericin on ice.</li>
<li>Mix calibration curve solutions via dilution series (<a href="#tbl2">Table 2</a>; Example calibration curve, <a href="#fig3">Figure 3</a>).
<table id="tbl2">
<caption>Table 2. Solutions for calibration curve</caption>
<thead>
<tr>
<th>Solution #</th>
<th>Component</th>
<th>Stock solution</th>
<th>Final concentration</th>
<th>Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="4">1</td>
<td>ICB</td>
<td>1×</td>
<td>1×</td>
<td>988.6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>2.5 mM</td>
<td>1 μM</td>
<td>0.4</td>
</tr>
<tr>
<td>Nigericin</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1</td>
</tr>
<tr>
<td>Lactate</td>
<td>1 M</td>
<td>10 mM</td>
<td>10</td>
</tr>
<tr>
<td rowspan="4">2</td>
<td>ICB</td>
<td>1×</td>
<td>1×</td>
<td>988.6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>2.5 mM</td>
<td>1 μM</td>
<td>0.4</td>
</tr>
<tr>
<td>Nigericin</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1</td>
</tr>
<tr>
<td>Lactate</td>
<td>100 mM</td>
<td>1 mM</td>
<td>10</td>
</tr>
<tr>
<td rowspan="4">3</td>
<td>ICB</td>
<td>1×</td>
<td>1×</td>
<td>988.6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>2.5 mM</td>
<td>1 μM</td>
<td>0.4</td>
</tr>
<tr>
<td>Nigericin</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1</td>
</tr>
<tr>
<td>Lactate</td>
<td>10 mM</td>
<td>100 μM</td>
<td>10</td>
</tr>
<tr>
<td rowspan="4">4</td>
<td>ICB</td>
<td>1×</td>
<td>1×</td>
<td>988.6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>2.5 mM</td>
<td>1 μM</td>
<td>0.4</td>
</tr>
<tr>
<td>Nigericin</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1</td>
</tr>
<tr>
<td>Lactate</td>
<td>1 mM</td>
<td>10 μM</td>
<td>10</td>
</tr>
<tr>
<td rowspan="4">5</td>
<td>ICB</td>
<td>1×</td>
<td>1×</td>
<td>988.6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>2.5 mM</td>
<td>1 μM</td>
<td>0.4</td>
</tr>
<tr>
<td>Nigericin</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1</td>
</tr>
<tr>
<td>Lactate</td>
<td>100 μm</td>
<td>1 mM</td>
<td>10</td>
</tr>
<tr>
<td rowspan="4">6</td>
<td>ICB</td>
<td>1×</td>
<td>1×</td>
<td>988.6</td>
</tr>
<tr>
<td>Rotenone</td>
<td>2.5 mM</td>
<td>1 μM</td>
<td>0.4</td>
</tr>
<tr>
<td>Nigericin</td>
<td>10 mM</td>
<td>10 μM</td>
<td>1</td>
</tr>
<tr>
<td>Lactate</td>
<td>10 μm</td>
<td>0.1 mM</td>
<td>10</td>
</tr>
</tbody>
</table>
<p class="table-legend">Solutions for the calibration curve should be made the same day as imaging. Intracellular buffer (ICB) will be used as the solvent with varying concentrations of lactate in the presence of nigericin and rotenone for permeabilization. Lactate should first be serial diluted 1:10 5× then used to make to the following solutions.</p>
<ol type="a">
<li>Make 10 μM nigericin and 2 μM of rotenone in 10 mL of ICB</li>
<li>Use stock solution of lactate to generate 2 mL of 10 mM lactate</li>
<li>10-fold Dilution series from 10 mM to 0.0001 mM (dilute 6<b>×</b>) with each solution at 1 mL final volume in ICB with rotenone and nigericin</li>
</ol>
</li>
<li>Mix MCT blockade solution for LPR (lactate production rate) assay (<a href="#tbl3">Table 3</a>)
<table id="tbl3">
<caption>Table 3. Example solution calculator for LPR injection solutions</caption>
<thead>
<tr>
<th>Solution #</th>
<th>Component</th>
<th>Stock solution</th>
<th>Final concentration</th>
<th>Amount (μL)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2">1</td>
<td>ECB</td>
<td>1×</td>
<td>1×</td>
<td>990</td>
</tr>
<tr>
<td>Glucose</td>
<td>1 M</td>
<td>10 mM</td>
<td>10</td>
</tr>
<tr>
<td rowspan="3">2</td>
<td>ECB</td>
<td>1×</td>
<td>1×</td>
<td>989</td>
</tr>
<tr>
<td>Glucose</td>
<td>1 M</td>
<td>10 mM</td>
<td>10</td>
</tr>
<tr>
<td>pCMBA</td>
<td>0.5 M</td>
<td>0.5 mM</td>
<td>1</td>
</tr>
</tbody>
</table>
<p class="table-legend">Solutions should be prepared fresh on the same day as imaging. Extracellular buffer (ECB) or glucose-free Fluorobrite can be used as the solvent which should be warmed prior to solution preparation. If using glucose-containing Fluorobrite, glucose does not need to be added. The example calculations are dependent on 1 mL solution injections which should be adjusted depending on the length (and therefore volume) of the tubing.</p>
<ol type="a">
<li>500 μM pCMBA in Fluorobrite with glucose or ECB with glucose, 1 mL per well of cells</li>
<li>Bring 1 mL of Fluorobrite or ECB per well to wash cells prior to imaging</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Calibration curve only needs to be performed once. When performing LPR experiments, calibration curve can be omitted (step 24).</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Solutions can stay at room temperature for up to an hour.</div>
<h3 id="sec3.5">Perform imaging of intracellular lactate or lactate production rate (LPR): Day 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<p>Injection of solutions and data acquisition at the microscope. This section can be used with any data acquisition, including calibration curve of intracellular lactate and measurement of intracellular glycolysis (lactate production rate, LPR). Calibration curve only needs to be performed once unless the optical set-up is changed (filter sets, for example).</p>
<ol start="26">
<li>Turn on light source, microscope, computer, imaging software (such as Micromanager), and additional controls such as stage, CO<sub>2</sub>, and temperature. Wait at least 1 h for the microscope to thermally stabilize.</li>
<li>Serum starve the cells in the Ibidi plate prior to imaging
<ol type="a">
<li>Warm imaging solution (Fluorobrite or ECB) to 37°C</li>
<li>Replace complete media in Ibidi plate containing cells with imaging solution
<ol type="i">
<li>On the left port of the Ibidi plate, remove 90 μL of media. To the right port, add 90 μL of imaging solution. Perform 3 exchanges in total to completely remove the complete media.</li>
</ol>
</li>
<li>Incubate for at least one hour prior to imaging at 37°C</li>
</ol>
</li>
<li>Wash the tubing with at least 5<b>×</b> volume with a 10 mL syringe filled with PBS then assemble imaging chamber.
<ol type="a">
<li>Connect input port before output port.
<ol type="i">
<li>After the final rinse, inject imaging solution (<a href="#tbl2">Tables 2</a> and <a href="#tbl3">3</a>) into the input line using a needle and 1 mL syringe through the in-line injection port, holding the male elbow connector end at the same height as the injection port in order to ensure the entire tubing contains imaging solution liquid before adding the cap (<a href="#fig2">Figure 2</a>). Capping the input line to close the input port will prevent fluid from leaking out of the input line into the imaging chamber. However, some diffusion is inevitable. Therefore, perform imaging away from the input port.</li>
<li>Fill the ports on the Ibidi plate with the imaging solution using a 200 μL pipette so that there is no air within the port.</li>
<li>Attach the input line first, press down firmly to ensure attachment. Then repeat attaching the output line.</li>
</ol>
</li>
<li>Ensure waste line is securely placed into waste container and that the input line is accessible without disturbing the imaging focus.</li>
</ol>
</li>
<li>Set imaging parameters
<ol type="a">
<li>Exposure should be set to 100–200 milliseconds at a frame rate of 1 frame/2 s in both donor and FRET channels to minimize photobleaching.</li>
<li>Choose a representative field of view away from the injection port and edges of well or multiple fields of view if you have stage control. <a href="#troubleshooting">Troubleshooting 1</a>, <a href="#troubleshooting">Troubleshooting 2</a>.</li>
</ol>
</li>
<li>Start imaging with injections
<ol type="a">
<li>Attach a 25g needle to a new 1 mL syringe, used for injecting drugs
<ol type="i">
<li>Pull liquid up into the syringe slowly to avoid bubble formation</li>
<li>Flick syringe as needed to eliminate bubbles</li>
<li>Use a different needle/syringe for each subsequent injection</li>
</ol>
</li>
<li>Start imaging. Pause image acquisition if necessary to inject each subsequent solution. We perform this action slowly (1 mL over 5 s) to avoid cavitation and bubble formation. If doing the intracellular lactate calibration curve, perform those injections in increasing lactate concentration. Otherwise, follow the injection protocol for LPR. See <a href="#tbl2">Tables 2</a> and <a href="#tbl3">3</a> for defined solutions. <a href="#troubleshooting">Troubleshooting 3</a>, <a href="#troubleshooting">Troubleshooting 4</a>
</li>
<li>Acquire at least 2 min of data after each injection.</li>
</ol>
</li>
<li>At the end of the assay, dispose of the cells unless using them for further analysis.
<ol type="a">
<li>Wash the tubing 3× with 10<b>×</b> volume using 70% ethanol</li>
</ol>
</li>
<li>Transfer data and turn off imaging equipment</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The complete media contains growth factors that can activate the cells, thus changing their metabolism. Therefore, cells must be starved for at least one and up to two hours prior to imaging.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Assembly requires attention to eliminating air bubbles. Additionally, if you connect the waste line first, you may pull liquid from the plate out, leaving the cells dry and the plate susceptible to bubble formation. If a bubble is injected into the imaging chamber, cells could be ripped off the plate.</div>
<div class="note">
<span class="note-title">Note:</span> To prevent bubble formation, place Ibidi chamber and tubing in incubator for 10 min prior to starting injections to bring everything to the same temperature. Make sure all solutions are at 37°C.</div>
<div class="note">
<span class="note-title">Note:</span> Exposure time will be dependent on the expression level, light source strength, objective, and camera sensitivity. We suggest that the histogram of intensities fills at least ¼ the dynamic range of the camera. <a href="#troubleshooting">Troubleshooting 1</a>
</div>
<div class="note">
<span class="note-title">Note:</span> After each injection, check focus to ensure that there has not been any focus drift. <a href="#troubleshooting">Troubleshooting 4</a>
</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Data can be analyzed at any time.</div>
<h3 id="sec3.6">Construct deep learning network for semantic segmentation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–7 days, depending on size of data set, computer memory and speed</div>
<p>For successful discrimination of metabolic subpopulations, accurate cell segmentation is critical. We therefore employed semantic segmentation for endothelial cells which have complex morphology. One increasingly powerful method for rapid segmentation is the use of artificial intelligence which has achieved human-level performance (<a href="#bib5">Ronneberger et al., 2015</a>). A subset of AI is deep learning, in which convolutional neural networks are used to perform recognition tasks, where many of the rules governing object assignment are hidden, as opposed to rule-based segmentation where all rules must be a priori explicitly laid out. For accurate Deep Learning to perform semantic segmentation, having a large ground truth dataset for network training is essential. A sample ground truth dataset can be downloaded at (<a href="#bib4">Harrison, Devin et al., 2021</a>), ground truth_v2.zip. Code for this section can be downloaded and followed along at: <a href="https://github.com/wulab-code/STAR_methods">https://github.com/wulab-code/STAR_methods</a>. Please open “code_testing_new.m” for sample code. Be sure all scripts are in the same directory to eliminate path issues.</p>
<ol start="33">
<li>Generate ground truth dataset: Try to make conditions of ground truth dataset as close to how data will be taken so that the AI can do the least amount of guessing. Transfect with laconic as above on a glass substrate such as Ibidi 8-well or Labtek 8 well to facilitate high resolution microscopy. The goal here is to generate microscopy datasets that will be easily labeled as cell body (using transfected Laconic), cell boundaries (using VE-CADH antibody), nuclei (with Hoechst dye), and background.
<ol type="a">
<li>Fix cells: After cells have grown to confluency and are appropriately expressing Laconic, wash cells with PBS at 37°C once and immediately fix using 4% PFA for 10 min at room temperature.</li>
<li>After fixation, wash with PBS 3×, 5 min each, on a rocking platform at room temperature.</li>
<li>Wash 3× with PBS, 5 min each, shaking, room temperature.</li>
<li>Block with blocking buffer for 30 min at room temperature, shaking</li>
<li>Primary antibody overnight (12–24 h) 1:100 VE-CADH in blocking buffer on a rocking platform at 4°C.</li>
<li>Wash with PBS 3×, 5 min each, shaking, room temperature.</li>
<li>Secondary antibody: 1:1000 goat anti-rabbit 549 for 1 h in blocking buffer at room temperature, shaking</li>
<li>Wash with PBS 3×, 5 min each, shaking, room temperature.</li>
<li>Stain nucleus: treat samples with 1:1000 Hoechst dye in PBS for 10 min at room temperature, then rinse with PBS once.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> After cells have been fixed, they can be stored at 4°C in PBS for imaging later. If the chambers are properly sealed to prevent drying out (plastic wrap or parafilm) then cells can be imaged months later. Mounting media can be added as a preservative and cells can be stored at 4°C for years.</div>
</li>
<li>Take images at same magnification / settings as experiment, as many as possible – green channel which will see laconic and label cytoplasm, red channel which will image VE-CADH and the cell boundary and blue channel which will image Hoechst and hence the nucleus of the cell. <a href="#troubleshooting">Troubleshooting 5</a>
</li>
</ol>
</li>
</ol>
<ol start="34">
<li>Label ground truth images: After generation of ground truth data set (<a href="#bib4">Harrison, Devin et al., 2021</a>) now we have to label all the pixels in the ground truth images. There are many ways to do this, including by manual inspection, or with custom image segmentation software. One highly accessible approach is to use free, open-source software such as CellProfiler (<a href="#bib1">Carpenter et al., 2006</a>). CellProfiler can use the nuclei as seeds for the cells then use the gradient of Laconic towards the cell edge as a marker for the boundary of the cell.
<ol type="a">
<li>Download CellProfiler (<a href="#key-resources-table">key resources table</a>) and load images according to your folder and file structure</li>
<li>Calculate the illumination function for your microscope. Use “CorrectIlluminationCalculate” function. Select the input image and name the output image. Calculate the Background image and do not dilate the objects in the final averaged image. Select a block size which is larger than the object (a cell); in this case, 150 is sufficient. Do not rescale the illumination function. Calculate the function based for each image. Smooth the background image with a Gaussian function.</li>
<li>Perform background correction on all images. Use the “CorrectIlluminationApply” function to perform background correction. Select the input images (these are your original images) and name the output image. Use the illumination function calculated in step 34b. The illumination function should be subtracted from the input image.</li>
<li>Identify primary objects (nuclei). Use the “IdentifyPrimaryObjects” function. Select input image for Hoechst based images. Enter the typical diameter of the object in pixel units. Discard objects outside the diameter range; discard objects touching the border of the image. Use an adaptive thresholding strategy with Otsu thresholding method, with three-class thresholding. Assign pixels in the middle intensity range to the foreground. Threshold smoothing scale and correction factor can be modified but default settings work. Choose adaptive window of 50 and distinguish clumped objects by shape. Use propagation method to draw dividing lines between clumped objects. Automatically calculate the size of the smoothing filter for declumping, automatically calculate the minimum allowed distance between local maxima, and speed up by using a lower-resolution image to find local maxima. Fill holes in identified objects after both thresholding and declumping. Typically, this will result in detection of nuclei that are bigger than what is perceived by the user.</li>
<li>Shrink the nuclei. Use the “ExpandOrShrinkObjects” function. Select the input object for nuclei and shrink the objects by a specified number of pixels. This has to be eyeballed, but 3 is sufficient.</li>
<li>Now identify the cell bodies. Use “IdentifySecondaryObjects.” Select input image for Laconic based images, and select the shrunken nuclei as the input objects. Select propagation as the method to be used to identify secondary objects. Use an adaptive thresholding strategy with Otsu thresholding method, with three-class thresholding. Assign pixels in the middle intensity range to the foreground. Threshold smoothing scale and correction factor can be modified but default settings work. Choose adaptive window of 50 and regulation factor of 0.05. Fill holes in identified objects and do not discard objects touching the border of the image.</li>
<li>Now identify cytoplasm. Use “IdentifyTertiaryObjects.” Select the cell body objects from step 34f. Select the smaller identified objects (shrunken nuclei). And do not shrink smaller objects prior to subtraction.</li>
<li>Fill objects. Now that the cytoplasm has been identified, use “FillObjects”. This function will fill in any unmarked cytoplasm, using a minimum hole size of 54 and planewise fill.</li>
<li>To create cell boundaries, shrink the cytoplasm by 1 using the “ExpandOrShrinkObjects” function.</li>
<li>Shrink the nuclei by 1 pixel using the “ExpandOrShrinkObjects” function.</li>
<li>Now using the shrunken cytoplasm, identify the cytoplasm object by using “IdentifyTertiaryObjects.”</li>
<li>Now using the shrunken nuclei, identify the nuclei object by using “IdentifyTertiaryObjects.”</li>
<li>Now using the cytoplasm with nuclei, identify the cytoplasm object by using “IdentifyTertiaryObjects” and smaller object is new shrunken nuclei.</li>
<li>Convert the following objects to image using “ConvertObjectsToImage” function: Filled cytoplasm, cytoplasm with nuclei, cytoplasm outlier, shrunken nuclei, shrunken nuclei_1 and shrunken cytoplasm</li>
<li>Save images to disk as Portable Network Graphics (png) using “SaveImages” function. The rest of the protocol assumes that images are saved in png format.</li>
</ol>
</li>
</ol>
<ol start="35">
<li>Construct deep learning algorithm using the Deep Learning toolbox in MATLAB: MATLAB is advantageous in that minimal computer programming is required. Smallest feature size will determine how many convolutions you need to perform. The idea is to use deconvolutions to increase the receptive field for each neuron and then localize where the neurons fire using upconvolutions. Finally, an inverse weight function will be used to account for unequal distributions of the pixel classes (background pixels are usually overrepresented, compared to boundary, cytoplasm, and nuclei, and could therefore bias the network). The first part will involve creating a datastore and using augmentation to increase the effective size of the dataset. The second part involves creating the segmentation network and assessing its performance.
<ol type="a">
<li>Download and install latest version of MATLAB with Deep Learning toolbox. Download the scripts from GitHub (<a href="https://github.com/wulab-code/STAR_methods">Link to code</a>). Open sample code “code_testing_new.m” in MATLAB.
<ol type="i">
<li>Create a new script. All code referenced below are in “code_testing_new.m”. The code is either directly shown below or referenced by line number. For each new line in the script, there will be a double arrow “&gt;&gt;” and can be typed directly into the script</li>
</ol>
</li>
<li>This script assumes you are on a Windows PC. In this example, the “basedir” is the location of the groundtruth images with are .png format files. This script also assumes that you are using MATLAB 2018b although the script should be compatible with newer versions.
<div class="textbox">
<p><code>&gt;&gt; clear</code></p>
<p><code>&gt;&gt; addpath('C:∖Program</code></p>
<p>  <code>Files∖MATLAB∖R2018b∖examples∖images∖main')</code></p>
<p><code>&gt;&gt; basedir = 'D:∖images∖';</code></p>
<p><code>&gt;&gt; ground_truth = 'D:∖images∖groundtruth';</code></p>
</div>
</li>
<li>Define classes.
<div class="textbox">
<p><code>&gt;&gt; classNames = ["background","border","cell","nuclei"];</code></p>
</div>
</li>
<li>Randomize into training and validation datasets. Pick the fraction of images you wish to use for training (frac_training) from the total number of groundtruth images (num_images), which the computer is blinded to when performing validation and testing. The script automatically divides the remaining non-training images into validation and testing evenly. A good starting point is 70% of images devoted to training. Num_images defines the number of groundtruth images.
<div class="textbox">
<p><code>&gt;&gt; num_images = 1000;</code></p>
<p><code>&gt;&gt; frac_training = 0.7;</code></p>
<p><code>&gt;&gt; [training, validation, testing] = ...</code></p>
<p>  <code>randomize_images(num_images,frac_training);</code></p>
</div>
</li>
<li>Make and store training, validation, and testing datasets into different directories: one for the labeled images and one for the images themselves
<div class="textbox">
<p><code>&gt;&gt; divide_images(...</code></p>
<p><code>basedir,ground_truth,training,validation,testing);</code></p>
</div>
</li>
<li>Create image and pixel datastore for training and validation.
<div class="textbox">
<p><code>&gt;&gt; imds = imageDatastore(fullfile(basedir,'training'),</code></p>
<p>  <code>... 'FileExtensions',{'.png'});</code></p>
<p><code>&gt;&gt; classNames =</code></p>
<p>  <code>["background","border","cell","nuclei"];</code></p>
<p><code>&gt;&gt; pixelLabelIds = [0 1 2 3];</code></p>
<p><code>&gt;&gt; pxds = pixelLabelDatastore(fullfile(basedir, ...</code></p>
<p>  <code>'training_label'),classNames,pixelLabelIds);</code></p>
<p><code>&gt;&gt; imds_validation = imageDatastore(fullfile(basedir, ...</code></p>
<p>  <code>'validation'),'FileExtensions',{'.png'});</code></p>
<p><code>&gt;&gt; pxds_validation = pixelLabelDatastore(fullfile(...</code></p>
<p>  <code>basedir,'validation_label'),classNames,pixelLabelId</code></p>
<p>  <code>s);</code></p>
<p><code>&gt;&gt; pximds_validation = pixelLabelImageDatastore(...</code></p>
<p>  <code>imds_validation,pxds_validation);</code></p>
</div>
</li>
<li>Augmentation can be used to increase the amount of data available to the network. Define augmentation function.
<div class="textbox">
<p><code>&gt;&gt; augmenter = imageDataAugmenter('RandRotation',[-180</code></p>
<p>      180],...</p>
<p>    <code>'RandXReflection',true,...</code></p>
<p>    <code>'RandYReflection',true,...</code></p>
<p>    <code>'RandXShear',[0 10],...</code></p>
<p>    <code>'RandYShear',[0 10]);</code></p>
</div>
</li>
<li>Define patch extraction datastore
<div class="textbox">
<p><code>&gt;&gt; patchds = randomPatchExtractionDatastore(...</code></p>
<p>  <code>imds,pxds,128,'PatchesPerImage',256, ...</code></p>
<p>  <code>'DataAugmentation',augmenter);</code></p>
<p><code>&gt;&gt; patchds_validation =</code></p>
<p>  <code>randomPatchExtractionDatastore(...</code></p>
<p>  <code>imds_validation,pxds_validation,128,'PatchesPerImag</code></p>
<p>  <code>e',256);</code></p>
</div>
</li>
<li>Preview the augmented patches
<div class="textbox">
<p><code>&gt;&gt; minibatch = preview(patchds);</code></p>
<p><code>&gt;&gt; montage(minibatch.InputImage,'DisplayRange',[0 255])</code></p>
</div>
</li>
<li>Create inverse weighting function.
<div class="textbox">
<p><code>&gt;&gt; tbl = countEachLabel(pxds);</code></p>
<p><code>&gt;&gt; totalNumberOfPixels = sum(tbl.PixelCount);</code></p>
<p><code>&gt;&gt; frequency = tbl.PixelCount / totalNumberOfPixels;</code></p>
<p><code>&gt;&gt; classWeights = 1./frequency;</code></p>
</div>
</li>
<li>Generate graph of the network (<a href="#fig4">Figure 4</a>). “numFilters” determines the number of channels in the output of the convolutional layer. “filterSize” is the size of the local regions that are convoluted and should therefore be matched to the smallest feature which in our case is a few pixels. 3 is the smallest filter size.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Deep learning architecture for semantic segmentation</div>
<p>Deep learning architecture for semantic segmentation, which consists of a contracting path (encoder) and an expansive path (decoder), forming a U-Net.</p>
</figcaption>
</figure>
Lines 43–174</li>
<li>Set options for the training. These are trial and error, but below are some starting points.
<div class="textbox">
<p><code>&gt;&gt; initialLearningRate = 0.05;</code></p>
<p><code>&gt;&gt; maxEpochs = 50;</code></p>
<p><code>&gt;&gt; minibatchSize = 16;</code></p>
<p><code>&gt;&gt; l2reg = 0.0001;</code></p>
<p><code>&gt;&gt; options = trainingOptions('sgdm',...</code></p>
<p>    <code>'InitialLearnRate', initialLearningRate, ...</code></p>
<p>    <code>'Momentum',0.99,...</code></p>
<p>    <code>'L2Regularization',l2reg,...</code></p>
<p>    <code>'MaxEpochs',maxEpochs,...</code></p>
<p>    <code>'MiniBatchSize',minibatchSize,...</code></p>
<p>    <code>'VerboseFrequency',20,...</code></p>
<p>    <code>'LearnRateSchedule','piecewise',...</code></p>
<p>    <code>'LearnRateDropFactor',0.1, ...</code></p>
<p>    <code>'LearnRateDropPeriod',20, ...</code></p>
<p>    <code>'Shuffle','every-epoch',...</code></p>
<p>    <code>'Plots','training-progress',...</code></p>
<p>    <code>'GradientThresholdMethod','l2norm′,..</code></p>
<p>    <code>'CheckpointPath',fullfile(basedir,'checkpoint'),...</code></p>
<p>    <code>'GradientThreshold',0.05, ...</code></p>
<p>    <code>'Verbose',true, ...</code></p>
<p>    <code>'ExecutionEnvironment','multi-gpu');</code></p>
</div>
</li>
<li>Train algorithm and save the network (<a href="#fig5">Figure 5</a>). This can take anywhere from overnight (12 h) to 1 week depending on the speed of your computer. You should see the accuracy increase and the loss decrease as the training attempts to converge. Accuracy should be above 80% and loss should be below 1 to have acceptable results. <a href="#troubleshooting">Troubleshooting 6</a>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Training and validation of the convolutional neural network</div>
<p>Accuracy (A) and loss (B) as a function of training epoch. Training was stopped when there was no further change to loss or accuracy.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; [net,info] = trainNetwork(patchds,lgraph2,options);</code></p>
<p><code>&gt;&gt; save network.mat net</code></p>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> We suggest imaging at least 20,000 cells for deep learning but any additional images will help. Only 70% of images will be used for training. The remaining 30% will be split evenly for validation and testing of the deep learning network.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Although we did not specify in the CellProfiler section (27), it is possible to use the VE-CADH signal as the boundary and the Laconic and Hoechst signal as markers for the cell body and nucleus when labeling cells for ground truth.</div>
<div class="note">
<span class="note-title">Note:</span> If the computer runs out of memory, try reducing the minibatch size. If the learning rate is not fast enough, adjust the momentum, LearnRateDropFactor and LearnRatePeriod. While these are good starting points, this is entirely trial and error. Typically, the accuracy should increase almost immediately in an upward trajectory from 0 to 60% within one 50 iterations with slow improvement afterwards. While the speed of calculation depends ultimately on the amount, quality of data, and computer specifications, a typical run on a fast computer with 3 GPUs (circa 2018) takes 24 h.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> It may be possible to use transfer learning using a published network to avoid creating one’s own deep learning network.</div>
<p><a href="#troubleshooting">Troubleshooting 7</a>: MATLAB code has difficulty executing</p>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> As this all takes place in a computer, resuming can happen at any time.</div>
<h3 id="sec3.7">Semantic segmentation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [4 h - days], depending on computer memory and speed, and size of dataset</div>
<ol start="36">
<li>Pre-processing images after imaging Laconic. Pre-processing images is important to remove imaging artifacts such as vignetting or uneven illumination, in order to have the best performance (assuming that the ground truth dataset is ideal). The network will likely perform well on unoptimized images as well. One can in theory test this under quantification and statistical analysis below. It is possible to take experiment images and process them first in CellProfiler, but here we opt to use MATLAB so that everything is “one-click”. The first step is to eliminate vignetting and uneven illumination, using a tophat filter. However, to specify filter properties, one needs to know the imaging properties of the microscope. Start a new script. Specify the camera and magnification parameters including pixel size (in real space). Specify the approximate size of a cell. A good rule of thumb is to make the tophat filter about 3× the size of a cell (<a href="#bib2">Crocker and Grier, 1996</a>). Specify the expected input image size for the network and bit depth. Make sure that the image size and bit depth are the same as that which was used in creating the deep learning network above (here, the image size was 1200<b>×</b>1200 and the bit depth was 8 bit).
<div class="textbox">
<p><code>&gt;&gt; pixel_size = 11e-6;</code></p>
<p><code>&gt;&gt; magnification = 10;</code></p>
<p><code>&gt;&gt; cellsize = 11e-6;</code></p>
<p><code>&gt;&gt; tophatw = 3∗round(cellsize/(pixel_size/magnification));</code></p>
<p><code>&gt;&gt; h = fspecial('disk',tophatw);</code></p>
<p><code>&gt;&gt; thresh = 0;</code></p>
<p><code>&gt;&gt; targetimsize = [1200 1200];</code></p>
<p><code>&gt;&gt; targetbitsize = 8;</code></p>
</div>
<ol type="a">
<li>Resize image and use tophat to get rid of long wavelength intensity fluctuations where venusimage and tfpimage are the files of the Venus image and the mTFP image, respectfully, although in principle it doesn’t matter.</li>
</ol>
</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; venusimage = 'venus.tif';</code></p>
<p><code>&gt;&gt; tfpimage = 'tfp.tif';</code></p>
<p><code>&gt;&gt; [venus_orig tfp_orig thresh_venus thresh_tfp] = ...</code></p>
<p>    <code>resizetophatim(venusimage,tfpimage,...</code></p>
<p>    <code>targetimsize,h,thresh,targetbitsize);</code></p>
</div>
<ol start="37">
<li>Perform semantic segmentation of images: It doesn’t matter if you do it on the background corrected Venus image or mTFP image. Here it is done on the Venus image
<ol type="a">
<li>Load the network:
<div class="textbox">
<p><code>&gt;&gt; segnet = load('network.mat');</code></p>
</div>
</li>
<li>Segment the image. It is much faster with a GPU than using a CPU.
<div class="textbox">
<p><code>&gt;&gt; output = semanticseg(venus_orig,segnet.net,... ’ExecutionEnvironment’,’gpu’);</code></p>
</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.8">Post-processing of images</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [2–16 h], depending on computer memory and speed, and size of dataset</div>
<ol start="38">
<li>Post-processing after semantic segmentation: The output is a categorical array (1–4). The segmentation needs to be cleaned up as artifacts can be introduced. This script filters out small cells and small nuclei and saves the output. Convert the output image to a labeled image (see <a href="#fig6">Figure 6</a> for example of original image, after semantic segmentation, and after additional post processing). See how well the segmented image compares to the original. In this example, the post processed image (‘Additional filtering’) has interpreted some noise as cells. This can be improved by increasing the size and/or accuracy of the ground truth data set.
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Single field-of-view segmentation example</div>
<p>The contrast adjusted original Venus image (left) undergoes semantic segmentation (center) which classifies cell boundaries but must be filtered additionally to remove non-cell objects (right). Scale bar is 264 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; [filtered_image filtered_nuclei] =</code></p>
<p>  <code>filter_network_output(output);</code></p>
<p><code>&gt;&gt; bw_filtered_image = bwlabel(filtered_image);</code></p>
<p><code>&gt;&gt; bw_filtered_nuclei = bwlabel(filtered_nuclei);</code></p>
<p><code>&gt;&gt; figure</code></p>
<p><code>&gt;&gt; output_im = double(output);</code></p>
<p><code>&gt;&gt; bw_im = double(bw_filtered_image);</code></p>
<p><code>&gt;&gt; venus_im = imread(venusimage);</code></p>
<p><code>&gt;&gt; J = imadjust(venus_im,stretchlim(venus_im),[]);</code></p>
<p><code>&gt;&gt; [X2]= output_im/max(output_im(:));</code></p>
<p><code>&gt;&gt; [X3]= bw_im/max(bw_im(:));</code></p>
<p><code>&gt;&gt; map = jet;</code></p>
<p><code>&gt;&gt; subplot(1,3,1)</code></p>
<p><code>&gt;&gt; imshow(J)</code></p>
<p><code>&gt;&gt; title('Contrast adjusted Venus image')</code></p>
<p><code>&gt;&gt; subplot(1,3,2)</code></p>
<p><code>&gt;&gt; imshow(X2,'Colormap',map)</code></p>
<p><code>&gt;&gt; title('Semantic segmentation')</code></p>
<p><code>&gt;&gt; subplot(1,3,3)</code></p>
<p><code>&gt;&gt; imshow(X3,'Colormap',map)</code></p>
<p><code>&gt;&gt; title('Additional filtering')</code></p>
</div>
<ol start="39">
<li>Decompose the segmented image into individual cells: The images have been segmented, but each pixel in each image is just now a number – 1–4 indicates background, cell boundary, cytoplasm, and nuclei. The cells have to be individually identified, which is the purpose of this part. Using parallel processing significantly speeds up the time required.</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; expand = 3;</code></p>
<p><code>&gt;&gt; venusresize = imresize(imread(venusimage),targetimsize);</code></p>
<p><code>&gt;&gt; tfpresize = imresize(imread(tfpimage),targetimsize);</code></p>
<p><code>&gt;&gt; imdata = cell_decomposition(bw_filtered_image,...</code></p>
<p>  <code>bw_filtered_nuclei,venusresize,tfpresize,expand);</code></p>
</div>
<ol start="40">
<li>Above, everything was done on one single image. Now we will put it into a loop using parallel processing. Place all variables that don't change outside the loop. We recommended that you use GPUs to do semantic segmentation first, then use regular CPUs (in parallel) for the rest. Use the demo_images.zip located here (<a href="#bib4">Harrison, Devin et al., 2021</a>). Unzip them into a directory titled “demo_images”.
<ol type="a">
<li>First loop the part that requires GPU<br> Lines 227–248.</li>
<li>Next loop the part that requires CPU</li>
</ol>
</li>
</ol>
<p>Lines 250–278.</p>
<div class="note">
<span class="note-title">Note:</span> A sample data set (in addition to the one referenced explicitly in the code above) is located at (<a href="#bib8">Wu et al., 2021b</a>) corresponding with figures in (<a href="#bib7">Wu et al., 2021a</a>). We recommend using the dataset corresponding to Figure 1e. Sample 1 is control and sample 2 is with DMOG. Injections for pCMBA occur at frame 120.</div>
<h3 id="sec3.9">Link cell trajectories</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ol start="41">
<li>Linking together cell trajectories: Now that all images have been segmented, the next task is to link the intensities of each cell into a trajectory which is accomplished below by creating a matrix of cell positions and finding the nearest cell in an adjacent image. Save the output.</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; input_directory = 'singlecell_output';</code></p>
<p><code>&gt;&gt; [venus tfp venus_bg tfp_bg linkage] = ...</code></p>
<p>    <code>cell_linking(input_directory,targetimsize);</code></p>
<p><code>&gt;&gt; output_directory = 'singlecell_links';</code></p>
<p><code>&gt;&gt; mkdir(output_directory);</code></p>
<p><code>&gt;&gt; save(fullfile(output_directory,'linkvenus.mat'),'venus');</code></p>
<p><code>&gt;&gt; save(fullfile(output_directory,'linktfp.mat'),'tfp');</code></p>
<p><code>&gt;&gt; save(fullfile(output_directory,'linkvenusbg.mat'),'venus_bg');</code></p>
<p><code>&gt;&gt; save(fullfile(output_directory,'linktfpbg.mat'),'tfp_bg');</code></p>
<p><code>&gt;&gt; save(fullfile(output_directory,'linkage.mat'),'linkage');</code></p>
</div>
<h3 id="sec3.10">Analysis of data</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Now that the experiment is complete, we need to analyze the data and calculate the rate of intracellular glycolysis or the lactate production rate (LPR). To obtain the LPR, calculate the slope of the mTFP/Venus signal (the lactate signal) over a 40 s trajectory and fit it against a linear model during the portion of the time the cells were exposed to MCT1 inhibitor according to the following equation (Equation 1):</p>
<p class="math-ml"><math> <mrow> <mi>L</mi> <mi>P</mi> <mi>R</mi> <mo>=</mo> <mfrac> <mrow> <mo>Δ</mo> <mi>L</mi> <mi>o</mi> <msub> <mi>g</mi> <mn>10</mn> </msub> <mi>L</mi> <mi>a</mi> <mi>c</mi> <mi>t</mi> <mi>a</mi> <mi>t</mi> <mi>e</mi> </mrow> <mrow> <mo>Δ</mo> <mi>t</mi> </mrow> </mfrac> <mo>.</mo> </mrow> </math></p>
<ol start="42">
<li>Load the linked trajectories: At this point the segmented images in each channel (mTFP, Venus) have been linked through time in “linktfp.mat”, “linkvenus.mat” which contains the mean intensity of the segmented cell in each respective channel. Background of the channels has been saved in “linktfpbg.mat” and “linkvenusbg.mat”. These are loaded by selecting the appropriate directory and subdirectory containing the linked files:</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; load(fullfile(output_directory,'linktfp.mat'));</code></p>
<p><code>&gt;&gt; load(fullfile(output_directory,'linkvenus.mat'));</code></p>
<p><code>&gt;&gt; load(fullfile(output_directory,'linktfpbg.mat'));</code></p>
<p><code>&gt;&gt; load(fullfile(output_directory,'linkvenusbg.mat'));</code></p>
</div>
<ol start="43">
<li>Calculate the inverse FRET ratio. Inverse FRET ratio is calculated by dividing the mTFP by Venus, which were determined from the total cytoplasmic intensity. One can also check the background doesn’t contribute significantly to the (inverse) FRET signal by calculating the FRET of the background. Each row of the “fret” or “fret_bg” matrix is the single cell inverse FRET ratio (<a href="#fig7">Figure 7</a>).
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. LPR single-cell trace</div>
<p>Example of the 1/FRET trace of an individual cell during the LPR assay. Arrows indicate solution changes.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; fret = tfp./venus;</code></p>
<p><code>&gt;&gt; fret_bg = (tfp-tfp_bg)./(venus - venus_bg);</code></p>
</div>
<ol start="44">
<li>Load the image data and set the LPR calculation over 40 s: Next, we calculate the LPR by fitting the rate of increase in mTFP/Venus signal over 40 s specified by dist (here, dist = 20 since we took 1 frame every 2 s).</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; imdata = load(fullfile('singlecell_output', ...</code></p>
<p>    <code>[zerostr(5,0) '_data.mat']));</code></p>
<p><code>&gt;&gt; dist = 20;</code></p>
</div>
<ol start="45">
<li>Calculate the LPR for each solution change. Iterate through the fret variable, which is a matrix containing in each row the average fret value of a cell, and in each column the fret value of that cell as a function of time. The signal is filtered with a median filter of width 5. We look for changes in solution that tells us when to start calculating the slope. Keep on looking for changes in solution until you have run out (“solution_changes”, specified a priori). Fit the relevant data with a line and calculate the slope. Save this variable. In this example (from the demo images dataset), solutions were injected in frame 150 (glucose) and frame 300 (glucose and pCMBA).</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; solution_changes = [150 300];</code></p>
<p><code>&gt;&gt; clear slope gof_r2 cellsize cells</code></p>
<p><code>&gt;&gt; for i = 1:size(fret,1)</code></p>
<p>  <code>mfret = medfilt1(fret(i,:),5);</code></p>
<p>  <code>for h = 1:length(solution_changes)</code></p>
<p>        <code>if h ∼= length(solution_changes)</code></p>
<p>                <code>vec =</code></p>
<p>    <code>mfret(solution_changes(h):solution_changes(h+1));</code></p>
<p>      <code>else</code></p>
<p>          <code>vec = mfret(solution_changes(h):length(mfret));</code></p>
<p>      <code>end</code></p>
<p>      <code>idx = find(vec == min(vec));</code></p>
<p>      <code>if h == length(solution_changes)</code></p>
<p>            <code>if length(mfret)-</code></p>
<p>    <code>(solution_changes(h)+min(idx)) &lt; dist</code></p>
<p>              <code>idx = 1;</code></p>
<p>        <code>end</code></p>
<p>      <code>end</code></p>
<p>      <code>x = solution_changes(h)+min(idx)-1;</code></p>
<p>        <code>[fitobj gof] =</code></p>
<p>    <code>fit((x:x+dist)',mfret(x:x+dist)','poly1′);</code></p>
<p>      <code>slope(i,h) = fitobj.p1;</code></p>
<p>      <code>gof_r2(i,h) = gof.rsquare;</code></p>
<p>        <code>end</code></p>
<p>    <code>end</code></p>
</div>
<ol start="46">
<li>Now go through the fitted slopes and filter it for fit slopes that have an R<sup>2</sup> value that is high enough to ensure that the fit parameters correspond will with the data (specified by “filter” parameter). In our hands, R<sup>2</sup> values &gt; 0.8 are sufficient. Here, the slope has been fit into another variable. In the code below, we use R<sup>2</sup> value of 0.9 (‘filter’ variable).</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; filter = 0.9;</code></p>
<p><code>&gt;&gt; slope_n(1).slope = [];</code></p>
<p><code>&gt;&gt; for m = 1:length(slope (1,:))</code></p>
<p>  <code>c = 1;</code></p>
<p>  <code>for i = 1:size(slope,1)</code></p>
<p>    <code>if gof_r2(i,m) &gt; filter</code></p>
<p>      <code>slope_n(m).slope(c) = slope(i,m);</code></p>
<p>      <code>slope_n(m).r2(c) = gof_r2(i,m);</code></p>
<p>      <code>slope_n(m).index(c) = i;</code></p>
<p>      <code>c = c+1;</code></p>
<p>    <code>end</code></p>
<p>  <code>end</code></p>
</div>
<ol start="47">
<li>At this point, save the results in a file.</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; save results.mat slope_n</code></p>
</div>
<ol start="48">
<li>The LPR can be plotted either in MATLAB or another plotting program. The factor 0.01221 is the slope of the semilog change in lactate, as generated from the calibration curve (<a href="#fig3">Figure 3</a>), divided by the inverse frame rate (Equation 1). Sample data for Laconic calibration can be downloaded from (<a href="#bib8">Wu et al., 2021b</a>), Figure Extended Data 1a (<a href="#bib7">Wu et al., 2021a</a>). Injection times for different concentrations of lactate during calibration process are available here (<a href="#bib4">Harrison, Devin et al., 2021</a>).</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; glucose = slope_n(1).slope /0.01221);</code></p>
<p><code>&gt;&gt; lactate = slope_n(2).slope /0.01221);</code></p>
</div>
<ol start="49">
<li>Inspect a good trace (<a href="#fig7">Figure 7</a> demonstrates a single cell trace of 1/FRET):</li>
</ol>
<div class="textbox">
<p><code>&gt;&gt; plot(fret(slope_n(2).index(50),1:450),'k.')</code></p>
<p><code>&gt;&gt; text(150,0.6075,'∖leftarrow Glucose injection')</code></p>
<p><code>&gt;&gt; text(300,0.6075,'∖leftarrow pCMBA injection')</code></p>
<p><code>&gt;&gt; xlabel('Frame')</code></p>
<p><code>&gt;&gt; ylabel('1/FRET′)</code></p>
</div>
<div class="optional">
<span class="optional-title">Optional:</span> Instead of calculating LPR (step 44), one can also directly compute the intracellular lactate concentration if there is a permeabilization step used as a reference (as for determining the inverse FRET/lactate calibration).</div>
<p><a href="#troubleshooting">Troubleshooting 8</a>: Low number of useable FRET traces</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol allows the user to use deep learning to perform semantic segmentation of cells, dividing images of cells into nucleus, cytoplasm, cell boundary, and background. This data is all saved in .mat files on the hard drive and be easily used in other downstream applications or analysis. This deep learning strategy for segmentation can easily be applied to difference cell types. This method of semantic segmentation is agnostic to Laconic and can be used in different cell labeling situations.</p>
<p>Generally, users should expect to generate FRET traces for hundreds of cells per field of view. Each individual trace is a time-series record of intracellular FRET. After the microscope system is calibrated, it is possible to map the FRET values to obtain absolute values of intracellular lactate. With knowledge of when the different compounds were injected, users can also calculate the lactate production rate, or glycolytic rate of individual cells. This protocol can be easily adapted to inspect the cell morphology, expression level of a given fluorescently labeled protein, or cell position and correlate it to LPR. This methodology can also be extended to use in different cell types. All the code is provided in GitHub, is open source, and free to be modified.</p>
<p>Here, we specifically demonstrate the LPR of a sample of HAECs treated with 500 μm DMOG and with vehicle (DMSO) overnight (<a href="#bib7">Wu et al., 2021a</a>). DMOG stimulates HIF-1a which increases glycolysis. LPR was calculated as above and exported to Prism (<a href="#fig8">Figure 8</a>).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. LPR bulk data for distinguishable populations in HAECs treated with control vs DMOG</div>
<p>LPR assay was performed on vehicle DMSO-treated (–) or DMOG-treated (+) endothelial cells (n = 126 –, 71 +; 164 –, 226 +; 181 −, 221 +; 120 −, 262 +; 191 −, 234 +; for 5 biological replicates error bars are SEM). p = 0.0044 by Student's t test.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">Network performance estimation</h3>
<p>There are many ways to estimate the error in machine learning or deep learning networks. Errors can be calculated for each class and a composite score can be calculated. Each estimation of error has its own positives and negative attributes, as error estimation is ultimately a little bit subjective, as one needs to choose which statistics to focus on. Common estimates of errors include the Matthew’s Correlation Coefficient, Global accuracy, Class specific accuracy, Boundary F1 score, precision, recall and Dice score. Examples on how to calculate each below.</p>
<ol>
<li>Save the network and run a test to see how accurate it is. It is important to only perform analysis on non-training data otherwise one will get artificially inflated scores. Calculate the global, class specific accuracy, and boundary F1 score.</li>
</ol>
<p>Lines 358–370.</p>
<ol start="2">
<li>Calculate standard deviation.</li>
</ol>
<p>Lines 380–414.</p>
<ol start="3">
<li>Calculate Matthews Correlation Coefficient where TP = true positive, TN = true negative, FP = false positive, FN = false negative.</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>M</mi> <mi>C</mi> <mi>C</mi> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>T</mi> <mi>P</mi> <mo>×</mo> <mi>T</mi> <mi>N</mi> <mo>−</mo> <mi>F</mi> <mi>P</mi> <mo>×</mo> <mi>F</mi> <mi>N</mi> </mrow> <msqrt> <mrow> <mrow> <mo>(</mo> <mrow> <mi>T</mi> <mi>P</mi> <mo>+</mo> <mi>F</mi> <mi>P</mi> </mrow> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mrow> <mi>T</mi> <mi>P</mi> <mo>+</mo> <mi>F</mi> <mi>N</mi> </mrow> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mrow> <mi>T</mi> <mi>N</mi> <mo>+</mo> <mi>F</mi> <mi>P</mi> </mrow> <mo>)</mo> </mrow> <mrow> <mo>(</mo> <mrow> <mi>T</mi> <mi>N</mi> <mo>+</mo> <mi>F</mi> <mi>N</mi> </mrow> <mo>)</mo> </mrow> </mrow> </msqrt> </mfrac> </mrow> </math></p>
<p>Lines 372–377.</p>
<ol start="4">
<li>Calculate precision where TP = true positive, FP = false positive</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>P</mi> <mi>r</mi> <mi>e</mi> <mi>c</mi> <mi>i</mi> <mi>s</mi> <mi>i</mi> <mi>o</mi> <mi>n</mi> <mo>=</mo> <mfrac> <mrow> <mi>T</mi> <mi>P</mi> </mrow> <mrow> <mi>T</mi> <mi>P</mi> <mo>+</mo> <mi>F</mi> <mi>P</mi> </mrow> </mfrac> </mrow> </math></p>
<p>Lines 419–422.</p>
<ol start="5">
<li>Calculate recall where TP = true positive, FN = false negative.</li>
</ol>
<p class="math-ml"><math> <mrow> <mi>R</mi> <mi>e</mi> <mi>c</mi> <mi>a</mi> <mi>l</mi> <mi>l</mi> <mo>=</mo> <mspace></mspace> <mfrac> <mrow> <mi>T</mi> <mi>P</mi> </mrow> <mrow> <mi>T</mi> <mi>P</mi> <mo>+</mo> <mi>F</mi> <mi>N</mi> </mrow> </mfrac> </mrow> </math></p>
<p>Lines 426–429.</p>
<ol start="6">
<li>Calculate the Dice score<br> Lines 434–449.
<ol type="a">
<li>Calculate the mean Dice score for each class<br> Lines 451–454.</li>
<li>Calculate the standard deviation of the Dice score for each class</li>
</ol>
</li>
</ol>
<p>Lines 456–459.</p>
<h3 id="sec5.2">Image segmentation error estimation</h3>
<p>To estimate the error in deep learning assisted semantic segmentation, we first compute the error between ground truth fluorescence and segmented fluorescence. We use a test data set of cells expressing laconic for which we have ground truth and segmented these images with the deep learning algorithm. Then using the same image analysis pipeline, we plot the cell fluorescence as estimated from deep learning networks against the ground truth cell fluorescence (n ∼ 20,000 cells) (<a href="#fig9">Figure 9</a>). In this example, there is a consistent underestimation of the actual fluorescence after segmentation/deep learning. In part, this is because in the post-processing of segmented images cells are artificially shrunken then expanded during morphological operations. This parameter can easily be modified as needed (by modifying function ‘filter_network_output.m’). This is also dependent on how “good” the deep learning network is – increase the size and quality of ground truth images such that the accuracy if &gt; 90%.</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Ground truth fluorescence vs. predicted fluorescence</div>
<p>To evaluate the performance of segmentation, computed fluorescence is compared to ground truth fluorescence, defined as the signal inside the cell boundary. In this example, there is a consistent underestimation of the total cell fluorescence. The color code indicates number of cells. Pure black indicates 1 cell. White indicates 0 cells. The red line is the fit between ground truth and predicted fluorescence. Fit equation and goodness of fit is inset.</p>
</figcaption>
</figure>
<div class="textbox">
<p>&gt;&gt; output_directory = 'network_test';</p>
<p>&gt;&gt; imds = imageDatastore(fullfile(basedir, 'testing'));</p>
<p>&gt;&gt; pxdsTruth = pixelLabelDatastore(fullfile(...</p>
<p>      basedir, 'testing_label'),classNames,pixelLabelIds);</p>
<p>&gt;&gt; pxdsResults = pixelLabelDatastore(output_directory, ...</p>
<p>      classNames,pixelLabelIds);</p>
<p>&gt;&gt; n_images = 65;</p>
<p>&gt;&gt; [cell_intensity_truth_total,cell_intensity_DL_total] =...</p>
<p>  evalTotalError(imds,pxdsTruth,pxdsResults,n_images,targetimsize);</p>
<p>&gt;&gt; save('error_estimation.mat', ...</p>
<p>      'cell_intensity_truth_total','cell_intensity_DL_total');</p>
<p>&gt;&gt; x = cell_intensity_truth_total';</p>
<p>&gt;&gt; y = cell_intensity_DL_total';</p>
<p>&gt;&gt; [fitobj gof] = fit(x,y,'poly1′);</p>
<p>&gt;&gt; plot(cell_intensity_truth_total,cell_intensity_DL_total,'kx')</p>
<p>&gt;&gt; hold all</p>
<p>&gt;&gt; xax = linspace(min(cell_intensity_truth_total), ...</p>
<p>    max(cell_intensity_truth_total),100);</p>
<p>&gt;&gt; plot(xax,fitobj.p1∗xax +fitobj.p2 ,'r-','LineWidth',2)</p>
<p>&gt;&gt; xlabel('Ground truth fluorescence')</p>
<p>&gt;&gt; ylabel('Computed fluorescence')</p>
<p>&gt;&gt; text(60,120,['y = ' num2str(fitobj.p1) ' + 'num2str(fitobj.p2)])</p>
<p>&gt;&gt; hold off</p>
</div>
<h3 id="sec5.3">Total LPR error estimation</h3>
<p>To estimate the total estimated error in LPR, from semantic segmentation to LPR slope calculation, we turn to Monte Carlo simulations of LPR (bootstrapping). Using a dataset which provides a real measure between ground truth and fluorescence, we generate a line by randomly sampling ground truth fluorescence and the corresponding deep-learning generated fluorescence, 100 times. The slope of this line (ground truth vs. deep-learning) is then calculated by performing linear regression. This random sampling of fluorescence values, followed by slope calculation, simulates LPR calculation in real samples. The distribution of these bootstrapped slopes is plotted after repeating this 10,000 times, resulting in a Gaussian distribution of slopes (<a href="#fig10">Figure 10</a>).</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/986-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. Total quantification of LPR</div>
<p>Data from <a href="#fig9">Figure 9</a> were randomly sampled 100 times and fitted to a line to get the slope, which represents a simulated LPR calculation. The black dots represent the frequency distribution of the simulated LPR after repeating this process 10,000 times. The red line is a Gaussian fit with mean 0.89 and standard deviation of 0.08. In this example, the deviation of the mean from 1.0 indicates that there is a consistent under-estimation of the “true” LPR, and a roughly 8% error. The deviation from 1.0 is likely due to the presence of consistent underestimation of total cell fluorescence, as in <a href="#fig9">Figure 9</a>.</p>
</figcaption>
</figure>
<div class="textbox">
<p>&gt;&gt; bootstrap_slope = [];</p>
<p>&gt;&gt; for i = 1:10000</p>
<p>      idx = randi([1 length(cell_intensity_truth_total)],1,100);</p>
<p>      x = cell_intensity_truth_total(idx);</p>
<p>      y = cell_intensity_DL_total(idx);</p>
<p>      fitobj = fit(x',y','poly1′);</p>
<p>      bootstrap_slope(i) = fitobj.p1;</p>
<p>  end</p>
<p>&gt;&gt; h = histogram(slope);</p>
<p>&gt;&gt; x = ((h.BinEdges(2:end) - h.BinEdges(1:end-1))/2) + ...</p>
<p>      h.BinEdges(1:end-1);</p>
<p>&gt;&gt; y = h.Values;</p>
<p>&gt;&gt; [curve fitobj gof] = fit(x',y','gauss1′);</p>
<p>&gt;&gt; histogram(slope)</p>
<p>&gt;&gt; hold all</p>
<p>&gt;&gt; plot(curve)</p>
<p>&gt;&gt; text(0.2,750,['y = ' num2str(curve.a1) ...</p>
<p>        '∗exp(-((x-' num2str(curve.b1) ')/' num2str(curve.c1)')ˆ2)'])</p>
</div>
<h3 id="sec5.4">Additional controls: Estimation of bleed through in microscope filter sets</h3>
<p>Bleed through in FRET occurs when excitation of the acceptor occurs at the wavelengths used to excite the donor, leading to acceptor emission that does not occur due to FRET (acceptor bleed through). Similarly, donor fluorescence can be detected in the acceptor channel (donor bleed through), artificially inflating FRET. A simple way to estimate degree of bleed through is to transfect cells with either mTFP or Venus without its pair and perform imaging as if imaging FRET. Then transfect cells with Laconic and image using the same parameters as those with mTFP and Venus alone. Compare the average signal level in minimal levels of lactate (maximal FRET) of each condition (mTFP alone, Venus alone, and Laconic).</p>
<p>To estimate the bleed through in the filter sets, create mRNA transcripts for mTFP and Venus separately (the FRET pair in Laconic), transfect the individual fluorophores as well as Laconic independently into HAECs and image.</p>
<ol start="7">
<li>Generate PCR templates for mTFP and Venus using laconic plasmid.
<ol type="a">
<li>Generate template for <i>in vitro</i> transcription of mTFP via PCR using the T7-F and mTFPr primers (see <a href="#key-resources-table">key resources table</a>) and protocol listed in <a href="#tbl1">Table 1</a> and NEB Q5 Polymerase. The annealing temperature should be 60°C and the extension time is 25 s.</li>
<li>Generate template for <i>in vitro</i> transcription of Venus via PCR using the T7-Venus and BGH-R primers (see <a href="#key-resources-table">key resources table</a>) and protocol listed in <a href="#tbl1">Table 1</a> and NEB Q5 Polymerase. The annealing temperature should be 60°C and the extension time is 25 s.</li>
<li>Purify PCR products using QIAquick PCR Purification Kit (<a href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/dna-purification/dna-clean-up/qiaquick-pcr-purification-kit/">Link to manufacturer's protocol</a>)</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PCRs can be checked by running on a 1% agarose gel for the correct amplicon size (853 bp for mTFP and 890 bp for Venus).</div>
<ol start="8">
<li>Create mRNA of mTFP and Venus
<ol type="a">
<li>Using the PCR-generated templates from 2c, create mRNA using mMESSAGE mMACHINE™ T7 ULTRA Transcription Kit (<a href="https://www.thermofisher.com/order/catalog/product/AM1344">Link to manufacturer's protocol</a>)</li>
<li>After cleaning and eluting mRNA, aliquot (100 ng/μL) and store at -80°C for minimal degradation. (<a href="https://www.thermofisher.com/order/catalog/product/AM1908">Link to manufacturer's protocol</a>)</li>
</ol>
</li>
<li>Transfect cells with mTFP, Venus, and Laconic as in mRNA transfection of Laconic above.</li>
<li>Image each fluorophore using the FRET filter sets after starving cells in glucose-free ECB for at least 1 h.</li>
<li>Compare donor channel and acceptor channel for all three transfections. The amount of donor bleed through (imaging mTFP transfection alone in the acceptor channel) and acceptor bleed through (imaging Venus transfection alone in the acceptor channel) can be subtracted from the donor and acceptor channels imaged using Laconic. We found, using the filter sets and illumination light source in this protocol, that bleed through was not greater than 5% of the laconic signal. Thus, it did not significantly affect the calculated FRET.</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Use of pCMBA will be toxic to the cells so if cells are needed following the assay, Phloretin can be used instead but should be washed out immediately following imaging. Injections of bubbles can ruin experiments. Segmentation can only be optimized by acquiring the right annotated training data. Environmentally controlled imaging chamber is ideal but not completely necessary—this would require careful attention to pH of buffers.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low expression level of Laconic</p>
<p>When selecting a field of view for performing imaging, you may notice low expression or heterogeneous expression such that each field of view minimizes the number of usable cells or data points. Additionally, the exposure time may have to be increased in order to boost your signal (step 29).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Dose titration of mRNA or virus</p>
<p>If expression levels are consistently low, we recommend dose titrating the mRNA or virus. If the expression levels are low for one or two experiments, consider changing aliquots. Both the laconic mRNA and adenovirus must be kept at -80°C for storage and freeze-thawed no more than twice. Adequate signal must be present to achieve accurate measures of intracellular lactate level. This can be tested by dose titration of mRNA or virus and performing intracellular lactate calibration (<a href="#fig3">Figure 3</a>). Some heterogeneity in cellular fluorescence is expected but, in our experience, does not affect intracellular lactate measurement as long as there is enough signal to noise.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Elongated cells</p>
<p>When selecting a field of view for performing imaging, you may notice altered morphology of the endothelial cells. We note this occurs more often when using mRNA transfection versus adenoviral transduction (see <a href="#fig1">Figure 1</a>) (step 29).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Adenoviral transduction</p>
<p>HAECs must not be used past passage 8. As the cells age, the morphology is more likely to change. We recommend using adenoviral transduction in lieu of mRNA if many experiments are to be performed for consistency of expression. Try avoiding using the same aliquot of defrosted cells for more than one month.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Injection causes cells to detach</p>
<p>Bubbles have been retained or formed in the imaging chamber or tubing (step 30).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Eliminate bubbles and make sure tubing and injection fluid is at the right temperature.</p>
<p>Bubbles may appear when the tubing is inserted into the Ibidi chambers. Flush the injection tubing slowly and thoroughly. Air may have been caught somewhere in the injection system. When attaching tubing to the chamber, make sure that no bubbles are entrained. When injecting, it must be done at a slow enough pace to not cause cavitation. Try reducing injection speed as well as to not dislodge any trapped air. Make sure all fluids are at the same temperature to prevent degassing.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Injection causes focus changes</p>
<p>After injecting solutions, focus is lost, likely due to rapid injection or accidental touching the microscope stage (step 30).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Equilibrate the microscope at the right temperature</p>
<p>First, fix the sample tightly on the stage to prevent accidental movement. Make sure the microscope has thermally equilibrated – turn on the scope and stage top incubator for at least an hour prior to imaging. Finally, re-focus the sample quickly after injection prior to acquiring data.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Too much intracellular labeling of VE-CADH</p>
<p>Occasionally, VE-CADH also stains the cell interior which will adversely affect cell boundary classification (step 33).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Try alternative microscopy approach or modify CellProfiler algorithm</p>
<p>If this is the case, either optimize staining with a different membrane marker, do not permeabilize, or try confocal microscopy and image different focal planes. In CellProfiler, one can also create a logical exclusion where overlap of VE-CADH and either Laconic or Hoechst causes VE-CADH to be deleted.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Deep learning accuracy does not reach above 80%</p>
<p>Deep learning is unable to accurately semantically segment cells and properly classify pixels in an image (step 35).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Ground truth is not accurate or there is not enough ground truth data.</p>
<p>Acquire more ground truth data. Carefully inspect the fixed samples and the output of CellProfiler and check to make sure that classification is accurate.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>MATLAB code has difficulty executing</p>
<p>Errors are returned at the command prompt (step 35) .</p>
<h3 id="sec7.14">Potential solution</h3>
<p>Make sure that the working directory is correct.</p>
<p>Make sure that the working directory is the same directory as with all the scripts. Check through the code to ensure that the directory referencing the location of images is correct.</p>
<h3 id="sec7.15">Problem 8</h3>
<p>Low number of useable FRET traces</p>
<p>After processing imaging data with what appears to be hundreds of cells and adequate signal, only a few traces (out of hundreds) have the requisite R<sup>2</sup> for statistical testing (step 46).</p>
<h3 id="sec7.16">Potential solution</h3>
<p>Ensure the ECB is at the correct pH</p>
<p>The pH of the ECB could change after sitting in the 37°C, 5% CO<sub>2</sub> incubator for too long, since it is a bicarbonate-based buffering system. Store ECB in an open container in a 37°C, 5% CO<sub>2</sub> incubator for an hour in a small volume then measure pH. If the pH has changed substantially, make sure that the experiment is performed before this occurred. Alternatively, use Fluorobrite DMEM.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Carpenter, A.E., Jones, T.R., Lamprecht, M.R., Clarke, C., Kang, I., Friman, O., Guertin, D.A., Chang, J., Lindquist, R.A., Moffat, J., et al. (2006). CellProfiler: image analysis software for identifying and quantifying cell phenotypes. Genome Biol. <i>7</i>, R100. <a class="external-link" href="https://doi.org/10.1186/gb-2006-7-10-r100">https://doi.org/10.1186/gb-2006-7-10-r100</a></p>
<p id="bib2">Crocker, J.C. and Grier, D.G. (1996). Methods of digital video microscopy for colloidal Studies. J. Colloid Interface Sci. <i>179</i>, 298-310. <a class="external-link" href="https://doi.org/10.1006/jcis.1996.0217">https://doi.org/10.1006/jcis.1996.0217</a></p>
<p id="bib3">Danthinne, X. and Imperiale, M.J. (2000). Production of first generation adenovirus vectors: a review. Gene Ther. <i>7</i>, 1707-1714. <a class="external-link" href="https://doi.org/10.1038/sj.gt.3301301">https://doi.org/10.1038/sj.gt.3301301</a></p>
<p id="bib4">Harrison, D., Wu, D., Fang, Y., and Huang, J. (2021). Human aortic endothelial cell ground truth dataset version 3 (Zenodo). <a class="external-link" href="https://doi.org/10.5281/zenodo.4898134">https://doi.org/10.5281/zenodo.4898134</a></p>
<p id="bib5">Ronneberger, O., Fischer, P., and Brox, T. (2015). U-Net: Convolutional Networks for Biomedical Image Segmentation. In Medical Image Computing and Computer-Assisted Intervention – MICCAI 2015, N. Navab, J. Hornegger, W.M. Wells, and A.F. Frangi, eds. (Springer International Publishing), pp. 234-241. <a class="external-link" href="https://doi.org/10.1007/978-3-319-24574-4_28">https://doi.org/10.1007/978-3-319-24574-4_28</a></p>
<p id="bib6">San Martín, A., Ceballo, S., Ruminot, I., Lerchundi, R., Frommer, W.B., and Barros, L.F. (2013). A genetically encoded FRET lactate sensor and its use to detect the warburg effect in single cancer cells. PLoS One <i>8</i>, e57712. <a class="external-link" href="https://doi.org/10.1371/journal.pone.0057712">https://doi.org/10.1371/journal.pone.0057712</a></p>
<p id="bib7">Wu, D., Harrison, D.L., Szasz, T., Yeh, C.-F., Shentu, T.-P., Meliton, A., Huang, R.-T., Zhou, Z., Mutlu, G.M., Huang, J., et al. (2021a). Single-cell metabolic imaging reveals a SLC2A3-dependent glycolytic burst in motile endothelial cells. Nat. Metab. <i>3</i>, 714-727. <a class="external-link" href="https://doi.org/10.1038/s42255-021-00390-y">https://doi.org/10.1038/s42255-021-00390-y</a></p>
<p id="bib8">Wu, D., Harrison, D., Szasz, T., Yeh, C.F., Shentu, T.P., Meliton, A., Huang, R.T., Zhou, Z., Mutlu, G.M., Huang, J., et al. (2021b). Single-cell metabolic imaging reveals a SLC2A3-dependent glycolytic burst in motile endothelial cells (Zenodo). <a class="external-link" href="https://doi.org/10.5281/zenodo.4638059">https://doi.org/10.5281/zenodo.4638059</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yun Fang, <a href="mailto:yfang1@medicine.bsd.uchicago.edu">yfang1@medicine.bsd.uchicago.edu</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The protocol includes all datasets generated or analyzed for this study. Data and code are available without restriction. Data is available on Zenodo (https://doi.org/10.5281/zenodo.4638059 and https://doi.org/10.5281/zenodo.4898134). All the code is provided in GitHub (<a href="https://github.com/wulab-code/STAR_methods">https://github.com/wulab-code/STAR_methods</a>).</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grants R00AI106941 (J.H.), R21AI120010 (J.H.), R01HL138223 (Y.F.), and R01HL136765 (Y.F.), <a href="https://doi.org/10.13039/100000002">NIH</a> New Innovator Award DP2AI144245 (J.H.), T32EB009412 (D.H.), T32HL007381 (D.H.), F32HL134288 (D.W.), <a href="https://doi.org/10.13039/100000002">NIH</a> Pathway to Independence Award K99HL145113536 (D.W.), and R00HL145113 (D.W.). <a href="https://doi.org/10.13039/100007065">NVIDIA</a> GPU Grant (D.W.), <a href="https://doi.org/10.13039/100008597">CSCTR</a> Early Career Development Award (D.W.), and <a href="https://doi.org/10.13039/100000001">NSF</a> Career Award 1653782 (J.H.) also supported this work.</p>
<h3>Author contributions</h3>
<p>D.H and D.W. designed and evaluated assay and wrote manuscript. Y.F. and J.H. wrote manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>