<article>
<section>
<h1 id="header">An Optimized Protocol for Human Direct Cardiac Reprogramming</h1>
<p><time datetime="2020-02-07">Published: February 7, 2020</time></p>
<p>Tiffany A. Garbutt,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a></sup> Yang Zhou,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Benjamin Keepers,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Jiandong Liu,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> and Li Qian<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn4">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pathology and Laboratory Medicine, University of North Carolina, Chapel Hill, NC 27599, USA</p>
<p id="aff2"><sup>2</sup>McAllister Heart Institute, University of North Carolina, Chapel Hill, NC 27599, USA</p>
<p id="fn1"><sup>3</sup>Current Address: Department of Biomedical Engineering, University of Alabama at Birmingham, Birmingham, AL 35294, USA</p>
<p id="fn2"><sup>4</sup>These authors contributed equally</p>
<p id="fn3"><sup>5</sup>Technical Contact</p>
<p id="fn4"><sup>6</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yangzhou@uab.edu">yangzhou@uab.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:li_qian@med.unc.edu">li_qian@med.unc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2019.100010">10.1016/j.xpro.2019.100010</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Direct cardiac reprogramming, the conversion of fibroblasts into cardiomyocyte-like cells (iCMs), is an attractive approach to heal the injured heart. Here we present a new approach to human cardiac reprogramming that utilizes a polycistronic three-factor reprogramming cocktail and one microRNA. Our protocol produces cardiac Troponin T positive human iCMs (hiCMs) at an efficiency of 40%–60%, approximately double that of previous protocols, within just 2 weeks. The resulting hiCMs display cardiomyocyte-like sarcomere structure, gene expression, and calcium oscillation.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib11">Zhou et al. (2019)</a>.</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>An optimized and updated protocol for human direct cardiac reprogramming</li>
<li>Human iCMs can be generated at an efficiency of 40-60%</li>
<li>The most minimalistic approach using one polycistronic construct and a microRNA</li>
<li>Detailed subsections of fibroblast isolation, viral prep, and cardiac reprogramming</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/25-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<p>Prepare the below media and prewarm at 37°C for at least 30 min prior to beginning each respective section of this protocol. Refer to <a href="#krt">Key Resources Table</a> for a complete list of materials and equipment.</p>
<ol>
<li>Human Cardiac Fibroblast Medium (HCF): Iscove’s Modified Dulbecco’s Medium (IMDM), supplemented with 20% fetal bovine serum (FBS), and 1% penicillin/streptomycin (P/S)</li>
<li>H9 Derived Fibroblast Medium (H9F): Dulbecco’s Modification of Eagle Medium (DMEM) supplemented with 20% FBS</li>
<li>293T Medium: DMEM supplemented with 10% FBS, 1x Non-essential amino acids (NEAA), and 1x P/S</li>
<li>293T Transfection Medium: DMEM supplemented with 10%FBS, and 1x NEAA</li>
<li>Induced Cardiomyocyte Medium (iCM): DMEM supplemented with10%FBS and 20% M199</li>
<li>Cardiomyocyte Medium (CM): RPMI-1640 medium supplemented with 2% B27, 2% FBS, 0.05% BSA, 50 μg/mL ascorbic acid, 0.2mM Glutamax, and 1x NEAA.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal to cardiac troponin T</td>
<td>ICC: abcam clone 1F11<br> Flow: Thermo Fisher clone: 3-11</td>
<td>ICC: Cat# ab10214; RRID: AB_2206574<br> Flow: Cat# MS-295-P; RRID: AB_61806</td>
</tr>
<tr>
<td>Mouse monoclonal to a-Actinin (clone EA-53)</td>
<td>sigma</td>
<td>Cat# A7811; RRID: AB_476766</td>
</tr>
<tr>
<td>Donkey anti-mouse IgG(H+L), AlexaFluor647 conjugated</td>
<td>Jackson Lab</td>
<td>Cat# 715-605-151; RRID: AB_2340863</td>
</tr>
<tr>
<td>Donkey anti-rabbit IgG(H+L), AlexaFluor488 conjugated</td>
<td>Jackson Lab</td>
<td>Cat# 711-545-152; RRID: AB_2313584</td>
</tr>
<tr>
<td colspan="3">Bacterial and Virus Strains</td>
</tr>
<tr>
<td>E.coli: HB101</td>
<td>Promega</td>
<td>L2015</td>
</tr>
<tr>
<td>E.coli: XL10-Gold Ultracompetent cells</td>
<td>Agilent</td>
<td>200315</td>
</tr>
<tr>
<td colspan="3">Biological Samples</td>
</tr>
<tr>
<td>Human cardiac tissue from a male patient</td>
<td>Duke Human Heart Repository</td>
<td><a href="https://sites.duke.edu/dhhr/">https://sites.duke.edu/dhhr/</a></td>
</tr>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Phosphate-buffered saline (PBS)</td>
<td>Fisher BioReagents</td>
<td>BP399-20</td>
</tr>
<tr>
<td>Poly-l-lysine</td>
<td>Electron microscopy science (EMS)</td>
<td>19321-B</td>
</tr>
<tr>
<td>DMEM</td>
<td>Corning</td>
<td>10-013-CV</td>
</tr>
<tr>
<td>IMDM</td>
<td>GIBCO</td>
<td>21980</td>
</tr>
<tr>
<td>DMEM/F12</td>
<td>GIBCO</td>
<td>11320-033</td>
</tr>
<tr>
<td>mTeSR1 medium</td>
<td>STEMCELL Technologies</td>
<td>85850</td>
</tr>
<tr>
<td>FBS</td>
<td>Millipore</td>
<td>TMS-013-B</td>
</tr>
<tr>
<td>Non-essential amino acids</td>
<td>Corning</td>
<td>25-025-CI</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>Corning</td>
<td>30-002-CI</td>
</tr>
<tr>
<td>PEG6000</td>
<td>Sigma</td>
<td>81255-2.5g</td>
</tr>
<tr>
<td>0.05% Trypsin</td>
<td>Corning</td>
<td>25-052-CI</td>
</tr>
<tr>
<td>M199</td>
<td>Corning</td>
<td>10-060-CV</td>
</tr>
<tr>
<td>Polybrene</td>
<td>Millipore</td>
<td>TR-1003-G</td>
</tr>
<tr>
<td>SurCoat</td>
<td>Celltron</td>
<td>SC-9035</td>
</tr>
<tr>
<td>RPMI1640</td>
<td>gibco</td>
<td>11875-093</td>
</tr>
<tr>
<td>B27 supplement</td>
<td>gibco</td>
<td>17504-044</td>
</tr>
<tr>
<td>BSA</td>
<td>Hyclone</td>
<td>SH3057402</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>gibco</td>
<td>35050-061</td>
</tr>
<tr>
<td>Gelatin</td>
<td>sigma</td>
<td>G1393-100ml</td>
</tr>
<tr>
<td>Dispase</td>
<td>STEMCELL Technologies</td>
<td>07923</td>
</tr>
<tr>
<td>Ascorbic acid</td>
<td>Sigma-Aldrich</td>
<td>A-4544</td>
</tr>
<tr>
<td>Puromycin</td>
<td>Thermo Fisher</td>
<td>A1113803</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>NanoFect transfection Reagent</td>
<td>Alstem</td>
<td>NF101</td>
</tr>
<tr>
<td colspan="3">Experimental Models: Cell Lines</td>
</tr>
<tr>
<td>Human: 293LTV Cell Line</td>
<td>Cell Biolabs</td>
<td>LTV-100; RRID: CVCL_JZ09</td>
</tr>
<tr>
<td>Human: ESC H9 derived fibroblasts: H9F</td>
<td>(<a href="#bib2">Fu et al., 2013</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Human: primary cardiac fibroblasts: HCF1</td>
<td>(<a href="#bib11">Zhou et al., 2019 </a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pMXs-puro-hMGT</td>
<td>(<a href="#bib11">Zhou et al., 2019 </a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pBabe-miR-133</td>
<td>(<a href="#bib6">Nam et al., 2013</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pLL3.7-UBCp-Ef1a-pur-Kan</td>
<td>Gift - Lab of Qing Zhang</td>
<td>N/A</td>
</tr>
<tr>
<td>pLL3.7-UBCp-hMGT-Ef1a-puro</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pLL3.7-CAG-hMGT-Ef1a-puro</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pLenti-CMV-GFP-Puro (658-5)</td>
<td>Addgene</td>
<td>17448; RRID:Addgene_17448</td>
</tr>
<tr>
<td>pLenti-Ef1a-hMGT-PGK-puro</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pMXs-puro-dsRed</td>
<td>(<a href="#bib9">Wang et al., 2015a</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pMXs-GFP</td>
<td>(<a href="#bib9">Wang et al., 2015a</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pMXs-tdTomato</td>
<td>(<a href="#bib4">Liu et al., 2017a</a>), (<a href="#bib5">Liu et al., 2017b</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>pCMV-VSV-G</td>
<td>Addgene</td>
<td>8454; RRID:Addgene_8454</td>
</tr>
<tr>
<td>Gag-Pol Retroviral Vector</td>
<td>Cell Biolabs, Inc.</td>
<td>RV-11</td>
</tr>
<tr>
<td>psPAX2</td>
<td>Addgene</td>
<td>12260; RRID:Addgene_12260</td>
</tr>
<tr>
<td>pMD2.G</td>
<td>Addgene</td>
<td>12259; RRID:Addgene_12259</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>EVOS microscope system</td>
<td>Invitrogen</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Retro-X qRT-PCR titration kit</td>
<td>Clontech</td>
<td>631453</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<ul>
<li>SureCoat: Melt the coating medium in a 37°C water bath until the solution is completely liquid (∼45 min).</li>
<li>1x Poly-L-lysine: Add 5 mL of 0.1% Poly-L-lysine into 500 mL PBS to make working solution.</li>
<li>0.1% Gelatin: Dilute 25 mL 2% gelatin into 475 mL PBS to make 0.1% gelation solution.</li>
<li>40% PEG/PBS: Dissolve 200 g of PEG6000 into 500 mL PBS and filter the solution with 0.22 μm filter.</li>
<li>Eppendorf centrifuge 5810R: Cool down the centrifuge to 4°C before use for precipitation of viral particles.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Generation of Human Fibroblasts</h3>
<p>Human fibroblasts can be isolated from human heart tissue or differentiated from a human embryonic stem cell line. The choice of which avenue to collect human cardiac fibroblasts depends largely on availability and research purpose. Human specimens are rare, and their collection requires either the use of available repositories or a medical-school collaboration. In this protocol we acquired human cardiac tissue from the Duke Human Repository and isolated human cardiac fibroblasts using the below described protocol. Of note, primary cardiac fibroblasts are the target cells of our interest as direct cardiac reprogramming aims to regenerate cardiomyocytes <i>in situ</i>.</p>
<p>Alternatively, human fibroblasts can be easily and largely generated from a human embryonic stem cell line. Therefore, it would be a better choice to use them if large quantities of cells are required for experiments like screening. In this protocol, we acquired previously differentiated H9F fibroblasts from the lab of Jidong Fu. We briefly summarize this protocol below. Please see <a href="#bib2">Fu et. al. (2013)</a> for further details on H9F derivation from the H9 human embryonic stem cell line.</p>
<h4>Isolation of Human Cardiac Fibroblasts from Heart Tissue</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 8 Days</div>
<ol>
<li>Prepare a 10-cm gelatin coated plate: Coat a 10-cm tissue culture plate with 0.1% gelatin. Incubate the plate for 15-30 min in a 37°C, 5% CO<sub>2</sub> tissue culture incubator.</li>
<li>After incubation, carefully aspirate the gelatin without touching the bottom of the plate and replace with 2 mL of pre-warmed HCF medium.</li>
<li>Wash the human heart tissue with PBS for several times to remove blood.</li>
<li>Mince the adult human heart tissue into small aggregates (∼1mm<sup>3</sup>) using a sterile razor blade.</li>
<li>Transfer the tissue aggregates to the prepared 10-cm gelatin coated plate containing 2 mL of HCF medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> After 4 hours of incubation, add 8 ml of warm HCF medium slowly and carefully along the wall of the plate to not disturb the attached aggregates (<a href="#mmc1">Video S1 </a>).</div>
<figure id="mmc1"><video controls="controls">
                <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/25-Mmc1.mp4" type="video/mp4"></source>
                Your browser does not support HTML5 video.
            </video>
<figcaption>
<div class="figcaption-title">Video S1. Careful Application of Media to Culture Plate</div>
<p>Carefully aspirate the media without touching the bottom of the plate. Media can then be added by one of two methods (1) by leaning the pipette against the side of the tissue culture plate wall or (2) by touching the pipette tip against the wall of the tissue culture plate and allowing the media to gently cover the cells. For both techniques slowly and gently expel the media from the pipette. Leave a little bit of media in the pipette tip to prevent expelling the last bit of media too forcefully.</p>
</figcaption>
</figure>
<ol start="6">
<li>Incubate for several days (∼7 days) to allow for explant culture and cell migration. (see Troubleshooting 1)</li>
<li>When the cells reach 100% confluency, dissociate the explanted cardiac fibroblasts into single cells by adding 2 – 3 mL of 0.05% trypsin to the 10-cm plate, and incubate for 5 min at 37°C (see Troubleshooting 2).</li>
<li>Add 5 mL of HCF medium to inactivate the trypsin and mix well by gently pipetting up and down using a 5 mL pipette.</li>
<li>Strain the cell solution through a 40 μm cell strainer.</li>
<li>Collect the flow through and pellet the cells by centrifuge at 200 x g for 5 min.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> 1 x 10<sup>6</sup> of cells can then be plated on a gelatin precoated plate (see Troubleshooting 3 &amp; 4).</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Alternatively, cells may be stored at −80 °C for short-term storage or in liquid nitrogen for long-term storage until later use. Freeze cells in 10% DMSO/FBS.</div>
<h4>Alternative Method: Differentiation of Fibroblasts from H9 Human Embryonic Stem Cells</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3 Weeks</div>
<p>In this protocol, we used H9 cells that were cultured with mTeSR1 on one plate of a 6-well plate at 80%–90% confluency for further differentiation.</p>
<ol start="11">
<li>Prewarm 6 mL of Dispase in a 37°C water bath for 10 min.</li>
<li>Remove mTeSR1 from the well and rinse with 1 mL/well of DMEM/F12.</li>
<li>Add 1 mL per well of warm Dispase and incubate at 37°C for 7 min.</li>
<li>Discard Dispase and rinse with 1 mL/well of DMEM/F12 twice.</li>
<li>Add 1 mL per well of mTeSR1 and gently scratch the well with a 1 mL pipette.</li>
<li>Repeat step 15 with another 1 mL/well of mTeSR1 to avoid over pipetting.</li>
<li>Pellet cell aggregates by centrifuge at 50 x g for 3 min</li>
<li>Carefully resuspend aggregates with H9F medium and transfer into low attachment 10-cm Petri dish.</li>
<li>Allow embryoid bodies to form during culture for seven days. Change the medium every other day.</li>
<li>After seven days, plate the embryoid bodies onto a gelatin-coated tissue culture plate containing H9F medium to facilitate the differentiation of cardiac fibroblasts. (See Troubleshooting 5).</li>
<li>Split the cells every four – five days to maintain 80% confluence.</li>
<li>After 2 weeks the cells are ready for use.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Alternatively, the cells may be frozen down in 1:10 mixture of DMSO/FBS at a density of 1 x 10<sup>6</sup>. The cells may be stored at −80 °C for 2-3 weeks or transferred to liquid nitrogen for long-term storage until use.</div>
<h4>Preparation of the Reprogramming Viruses</h4>
<p>The hMGT133 protocol has been tested with both retroviral and lentiviral packaging (<a href="#fig1">Figure 1</a>). Retrovirus hMGT will infect only mitotically active cells. Whereas lentiviral hMGT will infect both non-dividing and actively dividing cell types, enabling the infection of a greater variety of cell types than retroviruses. The user should decide which type of viral packaging would be best to suit their desired cell type and outcomes. The protocol for both types of viral packaging is described below (see Troubleshooting 6).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/25-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Four hMGT Plasmid Variations to Choose from Depending on Experimental Needs</div>
<p>(A) A retrovirus polycistronic hMGT construct.</p>
<p>(B) A polycistronic hMGT lentiviral construct based off of the pll3.7 backbone (gifted from the lab of Qing Zhang), where hMGT is driven by a UBC promoter and puromycin is driven by an Ef1α promoter.</p>
<p>(C) A polycistronic hMGT lentiviral construct based off of the pll3.7 backbone, in which hMGT is driven by CAG and puromycin selection is driven by an EF1α promoter.</p>
<p>(D) A polycistronic hMGT lentiviral construct based off of the pLenti-GFP backbone (Addgene, cat. no. 17448), in which hMGT is driven by an Ef1α promoter and puromycin selection is driven from a PGK promoter.</p>
</figcaption>
</figure>
<div class="timing">
<span class="timing-title">Timing:</span> 5-6 Days</div>
<ol start="23">
<li>On day 0, coat a 10-cm tissue culture plate with 1x poly-L-lysine and incubate for 5-10 min at 20 to 25°C (see Troubleshooting 7).</li>
<li>Prewarm premade 293T Medium and 0.05% Trypsin in 37°C water bath for 30 min.</li>
<li>Seed 5 × 10<sup>6</sup> cells of 293T cells onto the plate and culture for 17 – 18 h in 293T medium (see Troubleshooting 8 &amp; 9 and <a href="#fig2">Figure 2</a>A).</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/25-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Preparation of Retroviral Particles</div>
<p>(A) An example of 80%–90% confluency of 293T cells. Cells should appear similarly immediately prior to transduction. Scale bar for both images is 100 μm.</p>
<p>(B) Viral precipitation gathering at the bottom of 50 mL tube after a 17 - 18 h incubation with PEG/PBS.</p>
<p>(C) After centrifugation, the viral pellet should appear as a white or beige pellet at the bottom of the 50 mL tube. All traces of liquid should be removed after second spin. The beige / white pellet should appear as shown here, without any excess liquid.</p>
</figcaption>
</figure>
<ol start="26">
<li>On day 1, pre-warm DMEM medium with no additives to 20 to 25°C</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> see Troubleshooting 10.</div>
<ol start="27">
<li>Replace the spent 293T medium with 9 mL of 293T transfection medium at least one hour before transfection.</li>
<li>Prepare the below listed two solutions in 1.5 mL eppendorf tubes for each intended retrovirus. If also packaging miR-133 prepare an additional DNA solution and NanoFect solution for that retrovirus as indicated by the (<b>OR</b>) below:</li>
</ol>
<table>
<thead>
<tr>
<th>Retrovirus</th>
<th>Lentivirus</th>
</tr>
</thead>
<tbody>
<tr>
<td><b>(1) DNA / Plasmid Mix</b></td>
<td><b>(1) DNA / Plasmid Mix</b></td>
</tr>
<tr>
<td>5 μg gag/pol(<a href="#bib3">Ieda et al., 2010</a>)</td>
<td>7 μg psPAX2</td>
</tr>
<tr>
<td>2.5 μg VSVG</td>
<td>3 μg pMD2.G</td>
</tr>
<tr>
<td>15 μg hMGT <b>OR</b> miR-133</td>
<td>10 μg hMGT</td>
</tr>
<tr>
<td>In 500 μl DMEM</td>
<td>In 500 μl DMEM</td>
</tr>
<tr>
<td><b>(2) NanoFect Mix</b></td>
<td><b>(2) NanoFect Mix</b></td>
</tr>
<tr>
<td>45 μl NanoFect</td>
<td>45 μl NanoFect</td>
</tr>
<tr>
<td>In 500 μl DMEM</td>
<td>In 500 μl DMEM</td>
</tr>
</tbody>
</table>
<ol start="29">
<li>Vortex each solution at medium speed thoroughly and incubate for 5 min at 20 to 25°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Transfer Mix (2) into Mix (1) and vortex again for 10 seconds (see Troubleshooting 11).</div>
<ol start="30">
<li>Quickly spin down the solution from tube wall and cap.</li>
<li>Incubate the final mixture for 15-20 min at 20 to 25°C.</li>
<li>Add the final 1 mL of transfection mixture in a circular dropwise motion to the 293T cells, starting at the outer rim of the dish and ending in the center of the 10-cm plate.</li>
<li>Gently swirl the plate in back and forth and side-to-side motion to ensure even distribution of the transfection mixture. Incubate the plate in a 37°C, 5% CO<sub>2</sub> tissue culture incubator for 17- 18 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The following day (day 2) early in the morning (<b>do not exceed 20 hours post transfection</b>) aspirate the medium with careful attention to the appropriate procedure for discarding BSL2 virus containing medium.</div>
<ol start="34">
<li>Replace the spent medium with 8 mL of fresh 293T medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> <i>Virus Collection Point 1: </i> Approximately 48 hours after transfection (on day 3), collect the supernatant using a disposable syringe and filter it using 0.45 μm syringe filter into a 50 ml conical tube. Store this tube for 17 – 18 hours at 4°C.</div>
<ol start="35">
<li>Gently and slowly add 8 mL of fresh 293T medium along the side of the culture plate and incubate in a 37°C, 5% CO<sub>2</sub> tissue culture incubator for 17 – 18 h. Be careful not to dislodge the cells when removing or adding the medium.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> <i>Virus Collection Point 2 </i>: On day 4, repeat step one from day 3, by filtering the additional 8 ml of medium into the same 50 ml conical tube that was stored at 4 °C for 17 – 18 hours.</div>
<ol start="36">
<li>To precipitate the virus, add 2 mL of 40% of PEG/PBS solution to every 8 mL of viral supernatant. The final concentration of PEG is 8%.</li>
<li>Mix well by inverting the solution five to six times. Refrigerate the solution at 4°C for 17 – 18 h (see Troubleshooting 12).</li>
<li>On day 5, centrifuge the PEG-viral mixture at 3000 x g for 30 min at 4°C. After centrifugation, the viral particle may appear as a beige or white pellet at the bottom of the tube (<a href="#fig2">Figure 2 </a>B).</li>
<li>Carefully discard the supernatant without disturbing the pellet.</li>
<li>Centrifuge the tube again at 3000 x g for five minutes to get rid of any residual liquid.</li>
<li>After this second spin, remove all traces of liquid by aspiration. Be careful NOT to disturb the pellet (<a href="#fig2">Figure 2 </a>C).</li>
<li>Resuspend each viral pellet per 10-cm dish in 100 μl of cold, sterile DMEM at 4°C (see Troubleshooting 13).</li>
<li>The virus is now ready for use.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> It can be aliquoted in cryogenic vials and stored at −80 °C for future long-term use or at 4 °C for a few days.</div>
<h4>Cardiac Reprogramming</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 Weeks</div>
<p>The hMGT133 method of human cardiac reprogramming described below is a fast and efficient way to directly reprogram human cardiac fibroblasts using a minimal number of transcription factors and additives. Reprogrammed hiCMs can be collected for characterization within just two weeks of transduction and demonstrate cardiomyocyte-like properties (<a href="#bib11">Zhou et al., 2019</a>).</p>
<p>Human cardiac fibroblasts are cultured and split every four days after revival to ensure their increased growth (see Troubleshooting 14).</p>
<ol start="44">
<li>The day before reprogramming, precoat the desired number of wells on a 24 well plate with pre-dissolved SureCoat for 1 h at 37°C. (see Troubleshooting 15).</li>
<li>After 1 h aspirate the excess SureCoat and seed human cardiac fibroblasts at a density of 2 × 10<sup>4</sup> cells per 24-well (see Troubleshooting 16 &amp; 17).</li>
<li>On day 0, replace the fibroblast medium in each well with 500 μl of iCM medium (<a href="#fig3">Figure 3 </a>A).</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/25-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Brightfield Image of Reprogrammed hiCMs</div>
<p>(A) Standard human cardiac fibroblasts at reprogramming day zero.</p>
<p>(B) 11 days after transduction, hMGT133 infected cells start to undergo some morphology changes.</p>
<p>(C) 14 days after reprogramming, cells that were not infected with hMGT133 appear like standard fibroblasts.</p>
<p>(D and E) At reprogramming day 14, fibroblasts that were infected with hMGT133 are morphologically distinct from the fibroblast appearance at day zero and have a cardiomyocyte-like appearance. Note, that depending on the density of cells in culture, the individual cell shape of hiCMs may be clearly seen in a less dense culture (D), or less clearly seen in a denser culture (E). hiCM appearance as seen in either (D) or (E) is fine and is due the final density of cells. Scale bar for all images is 100 μm.</p>
</figcaption>
</figure>
<ol start="47">
<li>Add 10 μl of hMGT retro or lentivirus, 10 μl of miR-133 retrovirus, and 8 μg/mL polybrene per 24-well (see Troubleshooting 18).</li>
<li>Incubate for 48 h in a 37°C, 5% CO <sub>2</sub> tissue culture incubator.</li>
<li>On day 2, remove the spent iCM medium and replace with 1 mL of fresh iCM medium supplemented with 1 μg/mL puromycin.</li>
<li>Incubate the cells in iCM/puromycin medium in a 37°C, 5% CO<sub>2</sub> tissue culture incubator for four days to select for cells that have successfully been infected with the retrovirus or the lentivirus (see Troubleshooting 19 &amp; 20).</li>
<li>On day 6, Replace the iCM/puromycin medium with fresh iCM media lacking puromycin. The only cells remaining in culture should be those that have successfully taken up the hMGT construct.</li>
<li>Incubate for four days in a 37°C, 5% CO<sub>2</sub> tissue culture incubator.</li>
<li>On day 10, aspirate the spent iCM medium and replace with 1 mL of CM medium (see Troubleshooting 21). The cell morphology begins to change at this time point (<a href="#fig3">Figure 3</a>B).</li>
<li>Incubate for four days in a 37°C, 5% CO<sub>2</sub> tissue culture incubator.</li>
<li>On day 14, the reprogrammed cells should now transform from fibroblast morphology to a round shaped morphology (<a href="#fig3">Figure 3 </a>C-E) and can be collected for characterization of hiCM phenotype and reprogramming efficiency.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>Induced cardiomyocytes can be generated and collected for downstream characterization within just two weeks of transduction with hMGT and miR-133. We recommend the below listed characterization assays: Immunocytochemistry for the visual detection of cardiomyocyte markers, FACS for a quantitative estimate of reprogramming efficiency, qRT-PCR for mRNA cardiomyocyte profile detection, and calcium oscillation to estimate the functional capacity of reprogrammed iCMs (<a href="#bib11">Zhou et al., 2019 </a>). A list of recommend antibodies for immunocytochemistry and FACS is provided in the <a href="#krt">Key Resources Table</a>. We recommend dedicating at least two-three wells on a reprogrammed 24-well plate to each desired characterization assay to allow for adequate quantification based on technical repeats. We also recommend the co-characterization of control cells that were treated with control immunofluorescent plasmids, such as GGT, as an objective measure of viral packaging and reprogramming efficiency (see quality control notes below).</p>
<h3>Quality Control: Measure of Viral Packaging</h3>
<p>When performing the viral packaging of hMGT and miR-133, we recommend separately packaging a control plasmid with a fluorescent tag such as pMXs-puro-dsRed, pMXs-GFP or pMXs-tdTomato (<a href="#bib4">Liu et al., 2017a </a>), (<a href="#bib9">Wang et al., 2015a</a>). These control plasmids can be used to estimate transfection efficiency, viral infection efficiency, reprogramming efficiency and troubleshoot any problems with viral packaging.</p>
<h3>Quality Control: Measure of Reprogramming Efficiency</h3>
<p>Viral titer can be determined using the Retro-X qRT-PCR titration kit from Clontech as previously described (<a href="#bib8">Qian et al., 2013 </a>). This would entail transducing cells with five and tenfold serial dilutions and freezing the virus until a qRT-PCR could be performed and analyzed. Retroviruses are very sensitive to multiple freeze thaws. As such, the viral titer once thawed prior to reprogramming might be lower than calculated via qRT-PCR. In contrast, we have found that freshly harvested viruses yield higher reprogramming efficiency than frozen viruses. Furthermore, our previous studies have shown that the growth conditions of packaging cells directly relate to viral quality. Thus, controlling for packaging cell density (293T density) and transfection plasmid amount allows for high and relatively consistent viral titers (<a href="#bib10">Wang et al., 2015b </a>).</p>
<p>To empirically determine viral titer, we highly recommend seeding extra wells for the infection of a GGT control. The pMXs-GGT control plasmid includes polycistronic GFP, Gata4 and Tbx5, which are unable to initiate cardiac reprogramming, but allows for the visual identification of infected cells. Considering the similar size of the pMXs-GGT construct to the hMGT construct, the two DNA plasmids should be packaged at comparable efficiencies. As such, we usually use the same batch of pMXs-GGT as a titer indicator of hMGT. GFP can be visualized in human fibroblasts 48 h post-infection if viruses work properly. At reprogramming day 14, more than 50% of GFP expression in GGT-infected cells indicate that the successful packaging of virus and an increased chance to obtain good hiCM reprogramming efficiency.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although the efficiency of hiCM reprogramming has been significantly improved by using the simple approach we described above, there are still concerns and hurdles that must be overcome before this new technology can be widely used for further basic research and clinical translation. The following specific limitations should be mentioned. First, this approach highly depends on the quality and quantity of viruses. The preparation of viruses is not only time, cost and labor consuming, but may also be somewhat variable between different users and/or labs. In addition, to minimize safety concerns for future clinical application, it is better to figure out a non-integrative method to achieve hiCM reprogramming.</p>
<p>Second, based on our experience and our recently reported experimental findings, fibroblasts that have taken up the reprogramming cocktails quickly exit the cell cycle (<a href="#bib11">Zhou et al., 2019</a>). Furthermore, directly converted hiCMs, more like adult cardiomyocytes, are vulnerable to enzymatic dissociation or passaging and long-term culture. As a result, it is challenging to generate a large-scale quantity of hiCMs because of the lack of proliferative ability.</p>
<p>Lastly, in the scope of our experience, this protocol is feasible to generate hiCMs specifically from human cardiac fibroblasts or H9 fibroblasts. The use of a different starting cell type such as dermal fibroblasts, or other types of somatic cells would require cell-type specific user optimization, such as (1) modification of viral constructs using a lentiviral backbone (<a href="#fig1">Figure 1 </a>) with a tissue-specific promoter, (2) optimization of cell culture conditions and timeline according to cell growth rate and response to reprogramming factors or selection antibiotics, (3) adjustment of hMGT, miR-133, and/or packaging plasmid ratios, and (4) identification and supplementation of additional reprogramming factors or small molecules.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Generation of Human Fibroblasts</h3>
<h4>Problem 1</h4>
<p>How to ensure the tissue aggregates attach well and grow fast?</p>
<h4>Potential Solution</h4>
<p>During the propagation period, avoid disturbing the culture dish. Check the cell growth under microscopy every three days and exchange HCF medium.</p>
<h4>Problem 2</h4>
<p>What if explant cells cannot reach 100% confluency after long-term culture?</p>
<h4>Potential Solution</h4>
<p>We recommend replating the cells into new dishes for no longer than 14 days of culture. Long-term culture of cells might lead to cell senescence. This circumstance might result from low seed density and poor tissue attachment.</p>
<h4>Problem 3</h4>
<p>How to maintain healthy and fast proliferating cardiac fibroblasts?</p>
<h4>Potential Solution</h4>
<p>The seeding density of human cardiac fibroblasts is important for cell maintenance. A seeding density of 1 × 10<sup>6</sup> of cells allows the cells to propagate appropriately and reach to 100% confluency after 4 days of culture. Lower density results in impaired ability of cell proliferation.</p>
<h4>Problem 4</h4>
<p>How to ensure a pure cardiac fibroblast population from explanted tissue?</p>
<h4>Potential Solution</h4>
<p>The user may choose to perform cell sorting to ensure a pure population of human cardiac fibroblasts. However, it is not necessary. In our hands following this isolation protocol, more than 90% of isolated human cardiac fibroblasts were positive for the fibroblast marker vimentin. Furthermore, we found little difference in the reprogramming efficiency of sorted and unsorted human cardiac fibroblasts. The adult heart has limited proliferation ability. As such, it is unlikely that other cardiac cell types such as cardiomyocytes or endothelial cells will expand as cardiac fibroblasts in the given cell culture conditions (gelatin-coated dish and HCF medium).</p>
<h4>Problem 5</h4>
<p>How to ensure a pure cardiac fibroblast population from H9 differentiated cell lines?</p>
<h4>Potential Solution</h4>
<p>H9F medium is preferable for fibroblast growth. It is still recommended that the user perform THY1 cell sorting to ensure the collection of a pure fibroblast population. For this protocol, we acquired previously sorted H9F fibroblasts from the lab of Jidong Fu. Please see Fu’s paper for further details on H9F derivation (<a href="#bib2">Fu et al., 2013 </a>).</p>
<h3>Preparation of the Reprogramming Viruses</h3>
<h4>Problem 6</h4>
<p>Are there available lentiviral constructs if the user does not wish to use a retroviral platform or would like to reprogram a different cell type?</p>
<h4>Potential Solution</h4>
<p>The Qian lab has created three lentiviral plasmids in which polycistronic hMGT expression is driven by different promoters (<a href="#fig1">Figure 1</a>). Promoter choice may affect lentiviral reprogramming efficiency, with some promoters being particularly adept at reprogramming certain cell types. We have previously determined that the UBC promoter successfully drives high levels of exogenous hMGT in human cardiac fibroblasts to efficiently facilitate human cardiac reprogramming (<a href="#bib11">Zhou et al., 2019</a>). However, if the user is starting with a different cell type, we suggest choosing the lentiviral promoter that has been empirically shown to work best in the desired starting population. For example, Ef1α is uniquely efficient at driving expression in embryonic stem cells (<a href="#bib1">Chung et al., 2002 </a>), (<a href="#bib7">Norrman et al., 2010</a>).</p>
<h4>Problem 7</h4>
<p>How to prevent 293T cells detaching from the dish during media changes?</p>
<h4>Potential Solution</h4>
<p>The purpose of poly-L-lysine coating is to prevent the cells from detaching from the dish when changing medium. The user may also follow the techniques demonstrated in <a href="#mmc1">video 1</a> of this protocol.</p>
<h4>Problem 8</h4>
<p>How to ensure the ideal 293T seeding for viral packaging?</p>
<h4>Potential Solution</h4>
<p>Uneven distribution of cells may result in low titer of viruses. Swirl the plate in all directions when seeding the cells. The user may check cell distribution under the microscope prior to incubation. Also, the user should carefully swirl the plate after adding each respective plasmid.</p>
<h4>Problem 9</h4>
<p>What is the ideal cell density of 293T for transfection?</p>
<h4>Potential Solution</h4>
<p>The cell density of 293T cells may vary in different hands. The goal is to reach 80%–90% confluency the next day when transfection will be conducted (<a href="#fig2">Figure 2</a>A).</p>
<h4>Problem 10</h4>
<p>How to ensure efficient transfection?</p>
<h4>Potential Solution</h4>
<p>It is also highly recommended to use fresh opened DMEM for transfection.</p>
<h4>Problem 11</h4>
<p>How to ensure efficient transfection?</p>
<h4>Potential Solution</h4>
<p>The order of mixing matters. We found that the package efficiency reduces if Mix (1) is added into Mix (2).</p>
<h4>Problem 12</h4>
<p>How long does it take for viral particles to sufficiently precipitate in PEG/PBS solution?</p>
<h4>Potential Solution</h4>
<p>The PEG/PBS solution should be incubated at least 8 h at 4°C for proper precipitation. A longer incubation time not exceeding 3 days also works.</p>
<h4>Problem 13</h4>
<p>How to easily reconstitute virus pellet after centrifuge?</p>
<h4>Potential Solution</h4>
<p>We recommend immersing the pellet in DMEM at 4°C for 5 – 17 h. The reconstituted solution should look cloudy.</p>
<h3>Cardiac Reprogramming</h3>
<h4>Problem 14</h4>
<p>Do fibroblast passages affect reprogramming efficiency and outcome?</p>
<h4>Potential Solution</h4>
<p>We recommend that freshly thawed cells are passaged at least one time prior to beginning the reprogramming protocol, to ensure a healthier and more robust starting population. For the primary cardiac fibroblast, passages greater than 18 are not recommended to use for reprogramming.</p>
<h4>Problem 15</h4>
<p>Can the user use other types of coating media, such as gelatin?</p>
<h4>Potential Solution</h4>
<p>Besides SureCoat, gelatin and fibronectin can also be used for coating, although the reprogramming efficiency is compromised a little bit when using the other two coating materials (<a href="#bib11">Zhou et al., 2019</a>). Also, the morphology of iCMs might vary when different coating materials are used.</p>
<h4>Problem 16</h4>
<p>What might cause excessive death of cells after viral infection?</p>
<h4>Potential Solution</h4>
<p>Uneven distribution of cells may result in cell death after viral infection.</p>
<h4>Problem 17</h4>
<p>Does starting cell density need to be adjusted in a different experiment setting?</p>
<h4>Potential Solution</h4>
<p>The starting cell density varies for different experiments, such as a knockdown experiment. Coinfection of shRNA lentiviruses with retroviruses of reprogramming factors leads to more cell death and reduced survival after viral infection. In such case, we seed 4 × 10<sup>4</sup> cells per well for reprogramming.</p>
<h4>Problem 18</h4>
<p>Can this protocol be multiplexed for multiple wells and reprogramming experiments?</p>
<h4>Potential Solution</h4>
<p>If multiple wells are used, we recommend creating a master mix using these ratios that can be aliquoted to each well to increase reproducibility and overall protocol efficiency.</p>
<h4>Problem 19</h4>
<p>When to expect uninfected cells to die after adding puromycin?</p>
<h4>Potential Solution</h4>
<p>The uninfected cells without expression of puromycin usually die between day four and day six.</p>
<h4>Problem 20</h4>
<p>How to identify puromycin concentration for selection if a different starting cell type is used?</p>
<h4>Potential Solution</h4>
<p>We identified the concentration and time period of puromycin selection according to a death curve tested in fibroblasts isolated from human cardiac tissue and fibroblasts differentiated from H9 human embryonic stem cells at the abovementioned cell density. If different types of human fibroblasts will be used, we recommend performing a pilot experiment to identify the proper concentration and treatment of puromycin.</p>
<h4>Problem 21</h4>
<p>Can hiCMs be stably maintained for long-term culture?</p>
<h4>Potential Solution</h4>
<p>Reprogrammed hiCMs can be stably maintained in CM media for a long time depending on the purpose of the experiments. Of note, the cells will no longer proliferate with long-term culture.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chung, S., Andersson, T., Sonntag, K.-C., Björklund, L., Isacson, O., and Kim, K.-S. (2002). Analysis of different promoter systems for efficient transgene expression in mouse embryonic stem cell lines. Stem Cells <i>20</i>, 139-145.</p>
<p id="bib2">Fu, J.D., Stone, N.R., Liu, L., Spencer, C.I., Qian, L., Hayashi, Y., Delgado-Olguin, P., Ding, S., Bruneau, B.G., and Srivastava, D. (2013). Direct reprogramming of human fibroblasts toward a cardiomyocyte-like state. Stem Cell Reports <i>1</i>, 235-247.</p>
<p id="bib3">Ieda, M., Fu, J.D., Delgado-Olguin, P., Vedantham, V., Hayashi, Y., Bruneau, B.G., and Srivastava, D. (2010). Direct reprogramming of fibroblasts into functional cardiomyocytes by defined factors. Cell <i>142</i>, 375-386.</p>
<p id="bib4">Liu, Z., Chen, O., Wall, J.B.J., Zheng, M., Zhou, Y., Wang, L., Ruth Vaseghi, H., Qian, L., and Liu, J. (2017a). Systematic comparison of 2A peptides for cloning multi-genes in a polycistronic vector. Sci. Rep. <i>7</i>, 2193.</p>
<p id="bib5">Liu, Z., Wang, L., Welch, J.D., Ma, H., Zhou, Y., Vaseghi, H.R., Yu, S., Wall, J.B., Alimohamadi, S., Zheng, M., et al. (2017b). Single-cell transcriptomics reconstructs fate conversion from fibroblast to cardiomyocyte. Nature <i>551</i>, 100-104.</p>
<p id="bib6">Nam, Y.J., Song, K., Luo, X., Daniel, E., Lambeth, K., West, K., Hill, J.A., DiMaio, J.M., Baker, L.A., Bassel-Duby, R., and Olson, E.N. (2013). Reprogramming of human fibroblasts toward a cardiac fate. Proc. Natl. Acad. Sci. USA <i>110</i>, 5588-5593.</p>
<p id="bib7">Norrman, K., Fischer, Y., Bonnamy, B., Wolfhagen Sand, F., Ravassard, P., and Semb, H. (2010). Quantitative comparison of constitutive promoters in human ES cells. PLoS ONE <i>5</i>, e12413-e12413.</p>
<p id="bib8">Qian, L., Berry, E.C., Fu, J.D., Ieda, M., and Srivastava, D. (2013). Reprogramming of mouse fibroblasts into cardiomyocyte-like cells in vitro. Nat. Protoc. <i>8</i>, 1204-1215.</p>
<p id="bib9">Wang, L., Liu, Z., Yin, C., Asfour, H., Chen, O., Li, Y., Bursac, N., Liu, J., and Qian, L. (2015a). Stoichiometry of Gata4, Mef2c, and Tbx5 influences the efficiency and quality of induced cardiac myocyte reprogramming. Circ. Res. <i>116</i>, 237-244.</p>
<p id="bib10">Wang, L., Liu, Z., Yin, C., Zhou, Y., Liu, J., and Qian, L. (2015b). Improved Generation of Induced Cardiomyocytes Using a Polycistronic Construct Expressing Optimal Ratio of Gata4, Mef2c and Tbx5. J Vis Exp. <a href="https://doi.org/10.3791/53426">10.3791/53426</a>.</p>
<p id="bib11">Zhou, Y., Liu, Z., Welch, J.D., Gao, X., Wang, L., Garbutt, T., Keepers, B., Ma, H., Prins, J.F., Shen, W., et al. (2019). Single-Cell Transcriptomic Analyses of Cell Fate Transitions during Human Cardiac Reprogramming. Cell Stem Cell <i>25</i>, 149-164.e9.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Supplemental Information</h3>
<p>Supplemental Information can be found online at <a href="https://doi.org/10.1016/j.xpro.2019.100010">https://doi.org/10.1016/j.xpro.2019.100010 </a>.</p>
<h3>Acknowledgments</h3>
<p>This study was supported by UNC Integrative Vascular Biology Training Grant <a href="https://doi.org/10.13039/100000002">NIH</a> T32 HL069768 (PI: C. Mack) to B.K.; NIH / <a href="https://doi.org/10.13039/100000050">NHLBI</a> R01HL139880, R01HL139976 to J.L.; AHA 18TPA34180058, NIH / NHLBI R01HL128331, and R01HL144551 to L.Q.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, T.A.G., Y.Z., and L.Q.; Methodology, T.A.G., Y.Z., B.K., and L.Q.; Investigation, T.A.G., Y.Z., B.K., and L.Q.; Writing – T.A.G., Y.Z., and L.Q.; Supervision, J.L. and L.Q.; Funding Acquisition, J.L. and L.Q.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>