<article>
<section>
<h1 id="header">Integrated protocol for exitron and exitron-derived neoantigen identification using human RNA-seq data with ScanExitron and ScanNeo</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Ting-You Wang<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and Rendong Yang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>The Hormel Institute, University of Minnesota, Austin, MN 55912, USA</p>
<p id="aff2"><sup>2</sup>Masonic Cancer Center, University of Minnesota, Minneapolis, MN 55455, USA</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:tywang@umn.edu">tywang@umn.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:yang4414@umn.edu">yang4414@umn.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100788">10.1016/j.xpro.2021.100788</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Exitron splicing (EIS) events in cancers can disrupt functional protein domains to cause cancer driver effects. EIS has been recognized as a new source of tumor neoantigens. Here, we describe an integrated protocol for EIS and EIS-derived neoantigen identification using RNA-seq data. The protocol constitutes a step-by-step guide from data collection to neoantigen prediction.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref10">Wang et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for identifying exitron and exitron-derived neoantigens</li>
<li>Special focus on data preparation, and troubleshooting</li>
<li>Optional steps for applying this protocol to analyze TCGA PRAD cancer cohort</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/977-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Data collection</h3>
<p>This integrated protocol to analyze RNA sequencing (RNA-seq) data includes two components: ScanExitron (<a href="#bib10">Wang et al., 2021</a>) and ScanNeo (<a href="#bib11">Wang et al., 2019</a>) (<a href="#fig1">Figure 1</a>). ScanExitron was designed to detect exitron splicing events from short-read RNA-seq data, such as those produced by the Illumina sequencing platform from The Cancer Genome Atlas (TCGA) study (<a href="#bib10">Wang et al., 2021</a>). ScanNeo was originally developed for insertion and deletion (indel) derived neoantigen detection. Because of the similarity between deletions and EIS events in their effects changing protein sequences, ScanNeo is capable of detecting exitron-derived neoantigen directly. By definition, exitrons are cryptic introns with both their splice sites inside an annotated protein-coding exon. Therefore, human reference gene annotation is needed to identify <i>bona fide</i> exitrons. We recommend using the GRCh38 gene annotation GTF file from the GENCODE project (<a href="#bib2">Frankish et al., 2019</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/977-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Flow chart showing exitron and exitron-derived neoantigens detection with ScanExitron and ScanNeo</div>
</figcaption>
</figure>
<p>The protocol below describes ScanExitron applications analyzing a toy example data set and a real data set from the TCGA prostate cancer (PRAD) cohort, respectively.</p>
<div class="note">
<span class="note-title">Note:</span> Example data can be found at <a href="https://github.com/ylab-hi/ScanExitron/tree/master/example_data">https://github.com/ylab-hi/ScanExitron/tree/master/example_data</a>.</div>
<p>The RNA-seq alignment files in BAM format for TCGA PRAD cohort can be downloaded from NCI Genomic Data Commons (<a href="https://portal.gdc.cancer.gov/">https://portal.gdc.cancer.gov/</a>). A single representative aliquot was selected per participant for cases where more than one aliquot was available. Thus, 496 PRAD primary tumor samples and 52 normal samples were kept.</p>
<p>HLA class I four-digit types of 495 out of 496 TCGA PRAD samples were obtained from (<a href="#bib9">Thorsson et al., 2018</a>) (<a href="https://gdc.cancer.gov/about-data/publications/panimmune">https://gdc.cancer.gov/about-data/publications/panimmune</a>). For the remaining one sample used in this study, ScanNeo was employed for HLA class I typing.</p>
<h3 id="sec1.2">Optional reads alignment</h3>
<p>If the users are dealing with their in-house RNA-seq data in raw FASTQ format, the alignment step will be needed before running the protocol. ScanExitron requires the input to be a BAM file, which is provided by a splice-aware aligner, such as HISAT2 (<a href="#bib3">Kim et al., 2019</a>). We recommend aligning the raw read FASTQ file using HISAT2 with Hierarchical Graph Ferragina-Manzini (HGFM) index built with known transcripts annotations. Users can build the HGFM index on their own (<a href="http://daehwankimlab.github.io/hisat2/howto/#build-hgfm-index-with-transcripts">http://daehwankimlab.github.io/hisat2/howto/#build-hgfm-index-with-transcripts</a>) or download the HGFM index (genome_tran) directly (<a href="http://daehwankimlab.github.io/hisat2/download/">http://daehwankimlab.github.io/hisat2/download/</a>).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Example data (example.bam file)</td>
<td>This paper</td>
<td><a href="https://github.com/ylab-hi/ScanExitron/blob/master/example_data/">https://github.com/ylab-hi/ScanExitron/blob/master/example_data/</a></td>
</tr>
<tr>
<td>RNA-Seq data from TCGA PRAD cohort</td>
<td>NCI Genomic Data Commons</td>
<td><a href="https://portal.gdc.cancer.gov">https://portal.gdc.cancer.gov</a></td>
</tr>
<tr>
<td>HLA types for TCGA cohort</td>
<td><a href="#bib9">Thorsson et al., 2018</a></td>
<td><a href="https://gdc.cancer.gov/about-data/publications/panimmune">https://gdc.cancer.gov/about-data/publications/panimmune</a></td>
</tr>
<tr>
<td>GENCODE human gene annotations</td>
<td><a href="#bib2">Frankish et al., 2019</a></td>
<td><a href="https://www.gencodegenes.org/human/">https://www.gencodegenes.org/human/</a></td>
</tr>
<tr>
<td>Human reference genome NCBI build 38, GRCh38</td>
<td>Genome Reference Consortium</td>
<td><a href="http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/">http://www.ncbi.nlm.nih.gov/projects/genome/assembly/grc/human/</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>HISAT2</td>
<td><a href="#bib3">Kim et al., 2019</a></td>
<td>RRID:SCR_015530;<a href="http://daehwankimlab.github.io/hisat2/">http://daehwankimlab.github.io/hisat2/</a>
</td>
</tr>
<tr>
<td>ScanExitron</td>
<td><a href="#bib10">Wang et al., 2021</a></td>
<td><a href="https://github.com/ylab-hi/ScanExitron">https://github.com/ylab-hi/ScanExitron</a></td>
</tr>
<tr>
<td>Pyfaidx v0.5.9.2</td>
<td><a href="#bib13">Shirley et al., 2015</a></td>
<td><a href="https://github.com/mdshw5/pyfaidx">https://github.com/mdshw5/pyfaidx</a></td>
</tr>
<tr>
<td>SamTools v1.12</td>
<td><a href="#bib14">Li et al., 2009</a></td>
<td>RRID:SCR_00210;<a href="http://www.htslib.org/">http://www.htslib.org/</a>
</td>
</tr>
<tr>
<td>BEDTools v2.26.0</td>
<td><a href="#bib15">Quinlan, 2014</a></td>
<td>RRID:SCR_006646; <a href="https://github.com/arq5x/bedtools2">https://github.com/arq5x/bedtools2</a>
</td>
</tr>
<tr>
<td>RegTools v0.4.2</td>
<td><a href="#bib16">Feng et al., 2018</a></td>
<td><a href="https://github.com/griffithlab/regtools">https://github.com/griffithlab/regtools</a></td>
</tr>
<tr>
<td>ScanNeo</td>
<td><a href="#bib11">Wang et al., 2019</a></td>
<td>RRID:SCR_019253;<a href="https://github.com/ylab-hi/ScanNeo">https://github.com/ylab-hi/ScanNeo</a>
</td>
</tr>
<tr>
<td>transIndel v2.0</td>
<td><a href="#bib17">Yang et al., 2018</a></td>
<td><a href="https://github.com/cauyrd/transIndel">https://github.com/cauyrd/transIndel</a></td>
</tr>
<tr>
<td>OptiType v1.2</td>
<td><a href="#bib18">Szolek et al., 2014</a></td>
<td><a href="https://github.com/FRED-2/OptiType">https://github.com/FRED-2/OptiType</a></td>
</tr>
<tr>
<td>Yara aligner v1.0.2</td>
<td><a href="#bib19">Siragusa et al., 2013</a></td>
<td><a href="https://github.com/seqan/seqan/tree/master/apps/yara">https://github.com/seqan/seqan/tree/master/apps/yara</a></td>
</tr>
<tr>
<td>Variant Effect Predictor v102.0</td>
<td><a href="#bib5">McLaren et al., 2016</a></td>
<td>RRID:SCR_007931; <a href="https://useast.ensembl.org/info/docs/tools/vep/script/index.html">https://useast.ensembl.org/info/docs/tools/vep/script/index.html</a>
</td>
</tr>
<tr>
<td>Sambamba v0.8.0</td>
<td><a href="#bib20">Tarasov et al., 2015</a></td>
<td><a href="https://lomereiter.github.io/sambamba/">https://lomereiter.github.io/sambamba/</a></td>
</tr>
<tr>
<td>IEDB MHC class I peptide binding prediction tools v3.1</td>
<td><a href="#bib21">Vita et al., 2019</a></td>
<td><a href="https://downloads.iedb.org/tools/mhci/3.1/">https://downloads.iedb.org/tools/mhci/3.1/</a></td>
</tr>
<tr>
<td>BWA v0.7.17</td>
<td><a href="#bib22">Li and Durbin, 2009</a></td>
<td>RRID:SCR_010910;<a href="http://bio-bwa.sourceforge.net/">http://bio-bwa.sourceforge.net/</a>
</td>
</tr>
<tr>
<td>PyVCF v0.6.8</td>
<td>N/A</td>
<td><a href="https://github.com/jamescasbon/PyVCF/">https://github.com/jamescasbon/PyVCF/</a></td>
</tr>
<tr>
<td>Picard v2.24.0</td>
<td>Broad Institute</td>
<td><a href="https://broadinstitute.github.io/picard/">https://broadinstitute.github.io/picard/</a></td>
</tr>
<tr>
<td>HDF5 v1.10.4</td>
<td>The HDF Group</td>
<td><a href="http://www.hdfgroup.org/HDF5/">http://www.hdfgroup.org/HDF5/</a></td>
</tr>
<tr>
<td>Tabix v1.12</td>
<td><a href="#bib14">Li et al., 2009</a></td>
<td>RRID:SCR_00210;<a href="http://www.htslib.org/">http://www.htslib.org/</a>
</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>PC with 4 CPU cores and 16GB RAM</td>
<td>AMD</td>
<td>N/A</td>
</tr>
<tr>
<td>HPC system with 16 CPU cores and 64GB RAM</td>
<td>AMD</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>Data (RNA-seq alignment files in BAM format – see <a href="#sec1.1">data collection</a> in <a href="#before-you-begin">before you begin</a>)</p>
<h3 id="sec2.1">Software</h3>
<p>ScanExitron and its dependencies. ScanExitron is implemented in Python 3. While different versions of the Python software and associated packages may work correctly with ScanExitron, the authors use Python 3.7 and the following packages at the indicated versions when writing this protocol:</p>
<ul>
<li>pyfaidx (v0.5.9.2)</li>
<li>SamTools (v1.12)</li>
<li>BEDTools (v2.26.0)</li>
<li>RegTools (v0.4.2)</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> ScanExitron is not compatible with RegTools (v0.5 or above) in its current design.</div>
<p>ScanNeo and its dependencies. ScanNeo is also implemented in Python 3. When writing this protocol, the authors use Python 3.7 and the following packages at the indicated versions:</p>
<ul>
<li>transIndel (v2.0)</li>
<li>IEDB MHC class I peptide binding prediction tools (v3.1)</li>
<li>optitype (v1.3.5)</li>
<li>BWA (v0.7.17)</li>
<li>Sambamba (v0.8.0)</li>
<li>BEDTools (v2.26.0)</li>
<li>Variant Effect Predictor (v102.0)</li>
<li>coincbc (v2.10.5)</li>
<li>razers3 (v3.5.8)</li>
<li>Picard (v2.24.0)</li>
<li>Yara (v1.0.2)</li>
<li>pyomo (v5.7.3)</li>
<li>PyVCF (v0.6.8)</li>
<li>HDF5 (v1.10.4)</li>
<li>tabix (v1.12)</li>
<li>pyfaidx (v0.5.9.2)</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Step 1: Installing ScanExitron and ScanNeo</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60 min</div>
<p>Full installation of ScanExitron and ScanNeo includes downloading the ScanExitron and ScanNeo packages from GitHub. An example of how to perform all steps of this protocol using example data is available on the project GitHub at <a href="https://github.com/ylab-hi/ScanExitron/wiki/Exitron-and-exitron-derived-neoantigen-identification-with-ScanExitron-and-ScanNeo">https://github.com/ylab-hi/ScanExitron/wiki/Exitron-and-exitron-derived-neoantigen-identification-with-ScanExitron-and-ScanNeo</a></p>
<ol>
<li>Installing ScanExitron
<ol type="a">
<li>Install ScanExitron dependencies
<ol type="i">
<li>Install RegTools v0.4.2
<div class="textbox">
<p><code>$ git clone --depth 1 --branch 0.4.2</code> <a href="https://github.com/griffithlab/regtools.git">https://github.com/griffithlab/regtools.git</a></p>
</div>
</li>
<li>Install other dependent packages via conda.
<div class="textbox">
<p><code>$ conda install -c bioconda samtools bedtools pyfaidx</code></p>
</div>
</li>
</ol>
</li>
<li>Install ScanExitron by running the following code:
<div class="textbox">
<p><code>$ git clone</code> <a href="https://github.com/ylab-hi/ScanExitron.git">https://github.com/ylab-hi/ScanExitron.git</a></p>
</div>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Check if all required dependencies are downloaded and installed correctly. Originally, installing packages via conda will automatically check for and install the required dependencies. However, errors during installation could occur when installing on computational environments (<a href="#sec7.1">Troubleshooting 1</a> and <a href="#sec7.3">Troubleshooting 2</a>).</div>
<ol start="2">
<li>Installing ScanNeo
<ol type="a">
<li>Install ScanNeo dependencies
<ol type="i">
<li>Install transIndel v2.0
<div class="textbox">
<p><code>$ git clone</code> <a href="https://github.com/cauyrd/transIndel">https://github.com/cauyrd/transIndel</a></p>
</div>
Add the directory of transIndel_build_RNA.py and transIndel.py to the $PATH environment variable.</li>
<li>Install IEDB HLA class I binding prediction tools (<a href="https://downloads.iedb.org/tools/mhci/3.1/IEDB_MHC_I-3.1.tar.gz">https://downloads.iedb.org/tools/mhci/3.1/IEDB_MHC_I-3.1.tar.gz</a>)</li>
<li>Install other dependent packages via conda.
<div class="textbox">
<p><code>$ conda install -c bioconda optitype ensembl-vep sambamba bedtools picard bwa yara razers3 pyfaidx pyvcf</code></p>
<p><code>$ conda install -c conda-forge coincbc</code></p>
<p><code>$ conda install -c anaconda hdf5</code></p>
</div>
</li>
<li>Install VEP annotations and plugins Install VEP annotations using the following command.
<div class="textbox">
<p><code>$ vep_install -a cf -s homo_sapiens -y GRCh38 –CONVERT</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> Before install VEP annotations, make sure the directory of executable file vep_install is in the $PATH environment variable (<a href="#sec7.5">Troubleshooting 3</a>).</div>
Install two VEP plugins for ScanNeo.
<div class="textbox">
<p><code>$ git clone</code> <a href="https://github.com/ylab-hi/ScanNeo.git">https://github.com/ylab-hi/ScanNeo.git</a></p>
<p><code>$ cd VEP_plugins</code></p>
<p><code>$ cp Downstream.pm ∼/.vep/Plugins</code></p>
<p><code>$ cp Wildtype.pm ∼/.vep/Plugins</code></p>
</div>
</li>
<li>Configure optitype and yara index according to the ScanNeo manual (<a href="https://github.com/ylab-hi/ScanNeo">https://github.com/ylab-hi/ScanNeo</a>).</li>
</ol>
</li>
<li>Install ScanNeo by running the following code:
<div class="textbox">
<p><code>$ git clone</code> <a href="https://github.com/ylab-hi/ScanNeo.git">https://github.com/ylab-hi/ScanNeo.git</a></p>
</div>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure the directories of all the executable files are in the $PATH environment variable.</div>
<h3 id="sec3.2">Step 2: Preparing the reference genome sequences and gene annotation files</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>ScanExitron utilized annotated coding sequence (CDS) regions to probe the exitrons, and it also extracted splice sites using the reference genome sequences. The human reference genome sequences and gene annotation will be used.</p>
<ol start="3">
<li>Preparing human reference genome sequences in FASTA format.</li>
</ol>
<p>Download hg38 FASTA human reference genomes from UCSC genome browser (<a href="https://hgdownload.cse.ucsc.edu/goldenpath/hg38/bigZips/hg38.fa.gz">https://hgdownload.cse.ucsc.edu/goldenpath/hg38/bigZips/hg38.fa.gz</a>) and unzip it.</p>
<ol start="4">
<li>Preparing reference gene annotation in GTF format.
<ol type="a">
<li>Download hg38 annotation file from GENCODE project (ftp://ftp.ebi.ac.uk/pub/databases/gencode/Gencode_human/release_37/gencode.v37.annotation.gtf.gz)</li>
<li>Extract the protein-coding CDS regions</li>
</ol>
</li>
</ol>
<p>In Unix/Linux system, the protein-coding exons regions can be extracted using “cat”, “awk” and “tr” commands, as followed.</p>
<div class="textbox">
<p><code>$ cat gencode.v37.annotation.gtf | awk 'OFS="\t" {if ($3=="CDS") {print $1,$4-1,$5,$10,$16,$7}}' | tr -d '";' &gt; gencode.hg38.CDS.bed</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> Make sure the input RNA-seq BAM files used the same coordinate system as the reference genome and the reference annotations files. Otherwise, you have to remap the RNA-seq reads with the corresponding reference genome.</div>
<h3 id="sec3.3">Step 3: Running ScanExitron</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 <b>min</b>
</div>
<p>After installing all of the dependencies and preparing the reference genome sequences and annotation files, it is time to run ScanExitron. ScanExitron can be used only in UNIX/Linux systems currently. Additional details for running ScanExitron and updates to the parameters can be found at the project GitHub repository (<a href="https://github.com/ylab-hi/ScanExitron">https://github.com/ylab-hi/ScanExitron</a>).</p>
<div class="note">
<span class="note-title">Note:</span> Here we only provided the running time for the toy example dataset, which contains three exitrons. The actual running time for the real sample is dependent on the number of junction reads and the number of exitrons in it.</div>
<ol start="5">
<li>Make necessary modifications to the configuration file of ScanExitron.</li>
</ol>
<p>Replace the items in config.ini with the reference genome sequences and annotation files prepared in <a href="#sec3.2">step 2: preparing the reference genome sequences and gene annotation files</a>. The example config.ini file can be found at <a href="https://github.com/ylab-hi/ScanExitron/blob/master/config.ini.example">https://github.com/ylab-hi/ScanExitron/blob/master/config.ini.example</a> (<a href="#sec7.7">Troubleshooting 4</a>).</p>
<ol start="6">
<li>Run ScanExitron with the following command:
<div class="textbox">
<p><code>$ ScanExitron.py -i example.bam --ao 3 --pso 0.05 -m 50 -r hg38</code></p>
</div>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the input RNA-seq BAM files used the same coordinate system as the reference genome and the reference annotations files (<a href="#sec7.9">Troubleshooting 5</a>).</div>
<div class="note">
<span class="note-title">Note:</span> In practice, the different parameter settings will result in the different number of exitrons identified. For example, if you set a higher alternate allele observation (AO) and percent spliced out (PSO) (<a href="#bib10">Wang et al., 2021</a>), you will get a smaller number of exitrons. The details for these two metrics are described in <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>. Additional details for running ScanExitron and updates to the parameters can be found at the project GitHub repository (<a href="https://github.com/ylab-hi/ScanExitron">https://github.com/ylab-hi/ScanExitron</a>).</div>
<p>Multiple files will be generated in this step, including “example.hq.bam”, “example.hq.bam.bai”, “example.hq.janno” and “example.exitron”.</p>
<p>The identified exitrons are stored in the example.exitron file (<a href="#tbl1">Table 1</a>). <a href="#fig2">Figure 2</a> illustrates these detected EIS events using Integrative Genomics Viewer (IGV) (<a href="#bib7">Robinson et al., 2011</a>).</p>
<table id="tbl1">
<caption>Table 1. The identified exitrons in the example data set</caption>
<thead>
<tr>
<th>chrm:start-end</th>
<th>ao</th>
<th>strand</th>
<th>gene_symbol</th>
<th>Length</th>
<th>splice_site</th>
<th>pso</th>
<th>psi</th>
<th>dp</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr22:29489329–29489390</td>
<td>169</td>
<td>+</td>
<td>NEFH</td>
<td>60</td>
<td>GC-AG</td>
<td>0.261</td>
<td>0.739</td>
<td>648</td>
</tr>
<tr>
<td>chr22:29489371–29489432</td>
<td>80</td>
<td>+</td>
<td>NEFH</td>
<td>60</td>
<td>GT-AG</td>
<td>0.115</td>
<td>0.885</td>
<td>696</td>
</tr>
<tr>
<td>chr22:29489593–29489618</td>
<td>36</td>
<td>+</td>
<td>NEFH</td>
<td>24</td>
<td>GC-AG</td>
<td>0.0848</td>
<td>0.915</td>
<td>424</td>
</tr>
</tbody>
</table>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/977-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Three exitron splicing (EIS) events identified in <i>NEFH</i> gene loci by ScanExitron from the example RNA-seq data</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Differential analysis will be available if researchers have groups of samples of interest (<a href="#sec7.11">Troubleshooting 6</a>).</div>
<p>In order to feed the ScanExitron results to ScanNeo, output files of ScanExitron are required to be converted to VCF format using the utility script named exitron2vcf.py contained in the ScanExitron utils folder with the following command:</p>
<div class="textbox">
<p><code>$ exitron2vcf.py -i example.exitron -o example.vcf</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The directory of exitron2vcf.py should be in the $PATH environment variable.</div>
<h3 id="sec3.4">Step 4: Running ScanNeo</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<p>After running ScanExitron for the sample dataset, we get a list of exitron splicing events in the example.vcf file. In practice, you have to also run ScanExitron for the corresponding normal samples aiming to obtain exitrons that are tumor specific. Here, we assume all the exitrons identified in the sample dataset are tumor-specific exitrons (TSEs).</p>
<p>It is time to run ScanNeo to generate exitron-derived neoantigens. ScanNeo can be used only in UNIX/Linux systems currently. Additional details for running ScanNeo and updates to the parameters can be found at the project GitHub repository (<a href="https://github.com/ylab-hi/ScanNeo">https://github.com/ylab-hi/ScanNeo</a>).</p>
<ol start="7">
<li>Make necessary modifications to the configuration file of ScanNeo.</li>
</ol>
<p>Replace the items in config.ini with the reference genome sequences and gene annotation files prepared in <a href="#sec3.2">step 2: preparing the reference genome sequences and gene annotation files</a>. The example config.ini file can be found at <a href="https://github.com/ylab-hi/ScanNeo/blob/master/config.ini.example">https://github.com/ylab-hi/ScanNeo/blob/master/config.ini.example</a>. (<a href="#sec7.7">Troubleshooting 4</a>)</p>
<div class="note">
<span class="note-title">Note:</span> The reference genome sequences field is mandatory for this protocol. The gene annotation field is necessary when calling indels using ScanNeo. Yara HLA index field is necessary when HLA typing using ScanNeo.</div>
<ol start="8">
<li>Run ScanNeo
<ol type="a">
<li>ScanNeo first added corresponding reference and alternate allele sequences to each EIS event. Next, these events were annotated with variant effect predictor (VEP) (<a href="#bib5">McLaren et al., 2016</a>). Run this annotation step of ScanNeo using the following command.
<div class="textbox">
<p><code>$ ScanNeo.py anno -i example.vcf -o example.vep.vcf</code></p>
</div>
</li>
<li>Neoantigen prediction step of ScanNeo used VEP annotated VCF file as input to predict neoantigens using the following command.
<div class="textbox">
<p><code>$ ScanNeo.py hla -i example.vep.vcf --alleles HLA-A∗68:02,HLA-A∗23:01,HLA-B∗07:02,HLA-B∗53:01,HLA-C∗07:02,HLA-C∗04:01 -t 16 --af PSO -e 9 -p /path/to/iedb/ -o example.tsv</code></p>
</div>
</li>
</ol>
</li>
</ol>
<p>The putative exitron-derived neoantigens are stored in the example.tsv file (<a href="#tbl2">Table 2</a>).</p>
<table id="tbl2">
<caption>Table 2. The predicted exitron-derived neoantigens in the example data set</caption>
<thead>
<tr>
<th>Chrom</th>
<th>Start</th>
<th>Stop</th>
<th>Gene name</th>
<th>HLA allele</th>
<th>Peptide length</th>
<th>MT epitope seq</th>
<th>WT epitope seq</th>
<th>Best MT score method</th>
<th>Best MT score</th>
<th>Corresponding WT score</th>
</tr>
</thead>
<tbody>
<tr>
<td>chr22</td>
<td>29489329</td>
<td>29489389</td>
<td>NEFH</td>
<td>HLA-B∗07:02</td>
<td>9</td>
<td>SPPEAKSPA</td>
<td>SPPEAKSPE</td>
<td>NetMHCpan</td>
<td>399.52</td>
<td>7247.45</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> This is a good time to compare your output results files to example files provided in the ScanExitron GitHub repository (<a href="https://github.com/ylab-hi/ScanExitron/tree/master/example_data">https://github.com/ylab-hi/ScanExitron/tree/master/example_data</a>) to ensure that you have run the protocol correctly.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once you know the parameters you wish to use and have successfully run ScanExitron and ScanNeo, you may find this to be a good place to pause and evaluate the results before proceeding with the optional steps.</div>
<h3 id="sec3.5">Optional step 5: Running this protocol for TCGA PRAD cohort</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 h</div>
<p>As a matter of fact, we have to use exitrons that are tumor-specific to predict neoantigens. We used TCGA PRAD cohort that includes 496 tumor and 52 tumor-adjacent normal samples to demonstrate how to use this protocol.</p>
<ol start="9">
<li>For every sample in TCGA PRAD cohort, we identified EIS events of PRAD tumor and normal samples following the instructions in <a href="#sec3.3">step 3: running ScanExitron</a><b>.</b> Then we generated a list of tumor-specific exitrons (TSEs) by excluding the EIS events in tumor samples that were also found in more than three normal samples. We achieved this filtering process using in-house Python scripts, which are available at <a href="https://github.com/ylab-hi/ScanExitron/wiki/Exitron-and-exitron-derived-neoantigen-identification-with-ScanExitron-and-ScanNeo">https://github.com/ylab-hi/ScanExitron/wiki/Exitron-and-exitron-derived-neoantigen-identification-with-ScanExitron-and-ScanNeo</a>. A summary of identified exitrons and TSEs in PRAD is described in <a href="#fig3">Figure 3</a>.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/977-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Tumor-specific exitron (TSE) splicing events detection in PRAD cohort</div>
<p>(A) The proportion of frameshift and inframe TSEs in PRAD tumors.</p>
<p>(B) The proportion of genes with and without exitrons in PRAD tumors.</p>
<p>(C) Exitron size distribution of TSEs identified in PRAD tumors.</p>
<p>(D) PSO distribution of TSEs identified in PRAD tumors.</p>
</figcaption>
</figure>
</li>
<li>Run step 8 using the same parameters for TSEs of every sample in VCF format, we identified exitron-derived neoantigens for PRAD cohort (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/977-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The loads of TSEs, frameshift TSEs, inframe TSEs, neoantigen-yielding TSEs, neoantigen-yielding frameshift TSEs, neoantigen-yielding inframe TSEs, and putative TSE neoantigens in PRAD tumors</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The timing didn’t include downloading PRAD BAM files. In step 9, we submitted 16 jobs in the Slurm queue system. Every job only required one CPU core. In step 10, we used 20 jobs, every job required 16 CPU cores. Because ScanNeo implemented a parallel computing architecture, we highly recommend users set more CPU cores for it.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>At the end of the process of the example dataset, you will have two main text files; (1) the exitron splicing events identified (Data showed in <a href="#tbl1">Table 1</a> and <a href="#fig2">Figure 2</a>) and (2) the predicted exitron-derived neoantigens (Data showed in <a href="#tbl2">Table 2</a>). At the end of the process of the PRAD RNA-seq dataset, you will have TSE events for 496 PRAD patients and the corresponding predicted neoantigens (Data plotted in <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For every exitron splicing event identified, we used two measurements to quantify the exitron splicing event, that is, AO and PSO (<a href="#bib10">Wang et al., 2021</a>) . AO is the number of splice junction reads supporting exitron splicing. PSO metric was used to measure the percentage of transcripts in which a given exitron is spliced. Generally speaking, higher AO and PSO metrics indicated exitron splicing events with high confidence. Besides AO and PSO, we also reported percent spliced-in (PSI) (<a href="#bib8">Schafer et al., 2015</a>) as the counterpart of PSO and the average depth of the identified exitron splicing event in the ScanExitron output. Additional details for ScanExitron results can be found at the project GitHub repository (<a href="https://github.com/ylab-hi/ScanExitron">https://github.com/ylab-hi/ScanExitron</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The accuracy of exitron identification with ScanExitron is dependent on the accuracy of splice junctions from the RNA-seq BAM file and the completeness of CDS annotations. Firstly, due to the complexity of alternative splicing within a gene and the short-reads length, splice-aware aligners could produce large numbers of false-positive junctions (<a href="#bib1">Engstrom et al., 2013</a>). There is no optimal solution so far. But we can still mitigate it in two ways. One way is to make aligners prefer to use known splice sites by using the index built with known transcripts annotations, as we suggested in the <a href="#sec1.2">optional reads alignment</a> section. The other obvious way is to increase the read length when possible. Secondly, even for model organisms such as human, the reference annotations are incomplete, thus genuine exitrons with supporting junctions may be missed owing to the lack of overlapped annotated CDS annotations. Thus, in practice, we highly suggested using the latest gene annotations when possible.</p>
<p>Currently, the neoantigen prediction workhorse of this protocol, ScanNeo, only supports two well-established and popular MHC class I prediction algorithms, aka, NetMHC (<a href="#bib4">Lundegaard et al., 2008</a>) and NetMHCpan (<a href="#bib6">Nielsen and Andreatta, 2016</a>). Alternative versatile prediction algorithms should be used for neoantigen prediction. Thus, we plan to update ScanNeo to incorporate more MHC class I prediction approaches.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Install the software-dependent packages (Steps 1 and 2).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>When possible, use Anaconda (<a href="https://www.anaconda.com/">https://www.anaconda.com/</a>) to install Python 3 and its dependent packages. To order to avoid potential conflicts with installed Python packages, you can create a new conda environment to install all the necessary packages using the “conda create” command.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Software versions specific requirements (Steps 1 and 2).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Make sure that Python and other dependencies versions are appropriate.</p>
<p>You can use Anaconda to specify the version of the installed package, using the following commands:</p>
<div class="textbox">
<p><code>$ conda install &lt;package&gt;=&lt;version&gt;</code></p>
</div>
<p>Or use GitHub tag to specify the package version.</p>
<h3 id="sec7.5">Problem 3</h3>
<div class="textbox">
<p><code>$ git clone --depth 1 --branch &lt;version&gt;</code> <a href="https://github.com/%3cpackage%3e.git">https://github.com/&lt;package&gt;.git</a></p>
</div>
<p>You are receiving a “command not found” error message (Step 2), when you are trying to install VEP annotations using vep_install or run other conda installed executable files such as bedtools and sambamba. This indicated that the executable files are not in the $PATH environment variable.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Add Anaconda bin directory to the $PATH environment variable in the file ∼/.bashrc.</p>
<div class="textbox">
<p><code>export PATH="/path/to/Anaconda3/Python3/bin:$PATH"</code></p>
</div>
<h3 id="sec7.7">Problem 4</h3>
<p>You are receiving a “configparser.NoSectionError” error message (Step 5 and 7).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Place config.ini file to the location of ScanExitron or ScanNeo.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>You are receiving an “Errors in BED line” error message (Step 6). This indicated the input RNA-seq BAM file used GRCh37/GRCh38 contig names, such as ‘1’, ‘2’, instead of hg37/hg38 contig names, such as ‘chr1’, ‘chr2’.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>If you have the raw RNA-seq reads in FASTQ format, you can realign the reads using hg38/hg19 reference genome sequences. Otherwise, you can extract the reads from the RNA-seq BAM file using Picard SamToFastq (<a href="https://broadinstitute.github.io/picard/command-line-overview.html#SamToFastq">https://broadinstitute.github.io/picard/command-line-overview.html#SamToFastq</a>), then realign the reads.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>How to perform a differential analysis of exitrons between two groups of samples (Step 6 and <a href="#tbl1">Table 1</a>).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>First, following steps 1–6, you can detect a list of exitrons for every sample. Second, organize the exitron results of all samples to form a table of PSO values. In this table, you should put PSO values in the cell for the corresponding row (exitron splicing event) and column (sample). Because you have two groups of samples, you can use a linear model or statistical tests (e.g., T-test) to calculate the statistical significance (p-value) for each exitron. If there are multiple exitrons in the table, multiple testing correction is needed to adjust the p-values.</p>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="questions"><b>Q:</b> I have installed RegTools v0.4.2, but I cannot use ScanExitron and I received "Checking for 'regtools': ERROR - could not find 'regtools' Exiting" error message.</p>
<p class="answer"><b>A:</b> Add the directory of regtools executable to the $PATH environment variable in the file ∼/.bashrc.<br><code>export PATH="/path/to/regtools/bin/:$PATH"</code></p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Engstrom, P.G., Steijger, T., Sipos, B., Grant, G.R., Kahles, A., Ratsch, G., Goldman, N., Hubbard, T.J., Harrow, J., Guigo, R., et al. (2013). Systematic evaluation of spliced alignment programs for RNA-seq data. Nat. Methods <i>10</i>, 1185-1191. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref1">View at publisher</a></p>
<p id="bib16">Feng, Y.-Y., Ramu, A., Cotto, K.C., Skidmore, Z.L., Kunisaki, J., Conrad, D.F., Lin, Y., Chapman, W.C., Uppaluri, R., Govindan, R., Griffith, O.L., and Griffith, M. (2018). RegTools: Integrated analysis of genomic and transcriptomic data for discovery of splicing variants in cancer. bioRxiv. <a class="external-link" href="https://doi.org/10.1101/436634">https://doi.org/10.1101/436634</a></p>
<p id="bib2">Frankish, A., Diekhans, M., Ferreira, A.M., Johnson, R., Jungreis, I., Loveland, J., Mudge, J.M., Sisu, C., Wright, J., Armstrong, J., et al. (2019). GENCODE reference annotation for the human and mouse genomes. Nucleic Acids Res. <i>47</i>, D766-D773. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref2">View at publisher</a></p>
<p id="bib3">Kim, D., Paggi, J.M., Park, C., Bennett, C., and Salzberg, S.L. (2019). Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat. Biotechnol. <i>37</i>, 907-915. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref3">View at publisher</a></p>
<p id="bib22">Li, H. and Durbin, R. (2009). Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics <i>25</i>, 1754-1760. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/optNgBtpHBLFf">View at publisher</a></p>
<p id="bib14">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and Durbin, R.; 1000 Genome Project Data Processing Subgroup (2009). The Sequence Alignment/Map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/optI9uGryBtZv">View at publisher</a></p>
<p id="bib4">Lundegaard, C., Lamberth, K., Harndahl, M., Buus, S., Lund, O., and Nielsen, M. (2008). NetMHC-3.0: accurate web accessible predictions of human, mouse and monkey MHC class I affinities for peptides of length 8-11. Nucleic Acids Res. <i>36</i>, W509-W512. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref4">View at publisher</a></p>
<p id="bib5">McLaren, W., Gil, L., Hunt, S.E., Riat, H.S., Ritchie, G.R., Thormann, A., Flicek, P., and Cunningham, F. (2016). The ensembl variant effect predictor. Genome Biol. <i>17</i>, 122. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref5">View at publisher</a></p>
<p id="bib6">Nielsen, M. and Andreatta, M. (2016). NetMHCpan-3.0; improved prediction of binding to MHC class I molecules integrating information from multiple receptor and peptide length datasets. Genome Med. <i>8</i>, 33. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref6">View at publisher</a></p>
<p id="bib15">Quinlan, A.R. (2014). BEDTools: The Swiss-Army Tool for Genome Feature Analysis. Curr. Protoc. Bioinformatics <i>47</i>, 11.12.1-34. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/optwZHvlwRRXx">View at publisher</a></p>
<p id="bib7">Robinson, J.T., Thorvaldsdottir, H., Winckler, W., Guttman, M., Lander, E.S., Getz, G., and Mesirov, J.P. (2011). Integrative genomics viewer. Nat. Biotechnol. <i>29</i>, 24-26. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref7">View at publisher</a></p>
<p id="bib8">Schafer, S., Miao, K., Benson, C.C., Heinig, M., Cook, S.A., and Hubner, N. (2015). Alternative splicing signatures in RNA-seq data: percent spliced in (PSI). Curr. Protoc. Hum. Genet. <i>87</i>, 11 16 1-11 16 14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref8">View at publisher</a></p>
<p id="bib13">Shirley, M.D., Ma, Z., Pedersen, B.S., and Wheelan, S.J. (2015). Efficient “pythonic” access to FASTA files using pyfaidx. PeerJ PrePrints. <a class="external-link" href="https://doi.org/10.7287/peerj.preprints.970v1">https://doi.org/10.7287/peerj.preprints.970v1</a></p>
<p id="bib19">Siragusa, E., Weese, D., and Reinert, K. (2013). Fast and accurate read mapping with approximate seeds and multiple backtracking. Nucleic Acids Res. <i>41</i>, e78. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/optJuA1MjIwfn">View at publisher</a></p>
<p id="bib18">Szolek, A., Schubert, B., Mohr, C., Sturm, M., Feldhahn, M., and Kohlbacher, O. (2014). OptiType: precision HLA typing from next-generation sequencing data. Bioinformatics <i>30</i>, 3310-3316. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/opth6negYpOJN">View at publisher</a></p>
<p id="bib20">Tarasov, A., Vilella, A.J., Cuppen, E., Nijman, I.J., and Prins, P. (2015). Sambamba: fast processing of NGS alignment formats. Bioinformatics <i>31</i>, 2032-2034. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/optOk72uzv9Ae">View at publisher</a></p>
<p id="bib9">Thorsson, V., Gibbs, D.L., Brown, S.D., Wolf, D., Bortone, D.S., Ou Yang, T.H., Porta-Pardo, E., Gao, G.F., Plaisier, C.L., Eddy, J.A., et al. (2018). The Immune Landscape of Cancer. Immunity <i>48</i>, 812-830 e14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref9">View at publisher</a></p>
<p id="bib21">Vita, R., Mahajan, S., Overton, J.A., Dhanda, S.K., Martini, S., Cantrell, J.R., Wheeler, D.K., Sette, A., and Peters, B. (2019). The Immune Epitope Database (IEDB): 2018 update. Nucleic Acids Res. <i>47</i>, D339-D343. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/optv74WB7rbEB">View at publisher</a></p>
<p id="bib10">Wang, T.Y., Liu, Q., Ren, Y., Alam, S.K., Wang, L., Zhu, Z., Hoeppner, L.H., Dehm, S.M., Cao, Q., and Yang, R. (2021). A pan-cancer transcriptome analysis of exitron splicing identifies novel cancer driver genes and neoepitopes. Mol. Cell <i>81</i>, 2246-2260 e12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref10">View at publisher</a></p>
<p id="bib11">Wang, T.Y., Wang, L., Alam, S.K., Hoeppner, L.H., and Yang, R. (2019). ScanNeo: identifying indel-derived neoantigens using RNA-Seq data. Bioinformatics <i>35</i>, 4159-4161. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/sref11">View at publisher</a></p>
<p id="bib17">Yang, R., Van Etten, J.L., and Dehm, S.M. (2018). Indel detection from DNA and RNA sequencing data with transIndel. BMC Genomics <i>19</i>, 270. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00494-9/opt0IpLb2X37b">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Rendong Yang (<a href="mailto:yang4414@umn.edu">yang4414@umn.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The example data set for this study is available at <a href="https://github.com/ylab-hi/ScanExitron">https://github.com/ylab-hi/ScanExitron</a>.</p>
<h3>Acknowledgments</h3>
<p>We acknowledge the following sources of funding: <a href="https://doi.org/10.13039/100000005">DoD</a> (W81XWH-19-1-0161) to R.Y. and Eagles Telethon Postdoctoral Fellowship to T.-Y.W. We thank Dr. Jeffrey McDonald at The Hormel Institute for his technical support for computing facilities. Support from the Minnesota Supercomputer Institute (MSI) is also gratefully acknowledged.</p>
<h3>Author contributions</h3>
<p>Writing, T.-Y.W. and R.Y.; development and processing, T.-Y.W.; funding acquisition, R.Y.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>