<article>
<section>
<h1 id="header">Evaluation of Jumonji C lysine demethylase substrate preference to guide identification of <i>in vitro</i> substrates</h1>
<p><time datetime="2022-04-01">Published: April 1, 2022</time></p>
<p>Matthew Hoekstra,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Anand Chopra,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> William G. Willmore,<sup><a href="#aff1">1</a></sup> and Kyle K. Biggar<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#fn3">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Biochemistry and Department of Biology, Carleton University, 1125 Colonel By Drive, Ottawa, ON K1S 5B6, Canada</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kyle_biggar@carleton.ca">kyle_biggar@carleton.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101271">10.1016/j.xpro.2022.101271</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Within the realm of lysine methylation, the discovery of lysine methyltransferase (KMTs) substrates has been burgeoning because of established systematic substrate screening protocols. Here, we describe a protocol enabling the systematic identification of JmjC KDM substrate preference and <i>in vitro</i> substrates. Systematically designed peptide libraries containing methylated lysine residues are used to characterize enzyme-substrate preference and identify new candidate substrates <i>in vitro</i>.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/J.AB.2021.114429">Hoekstra and Biggar (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Use of a permutated substrate library to define JmjC KDM recognition motifs</li>
<li>JmjC KDM activity is measured via luminescent detection of succinate</li>
<li>Recognition motifs enable prediction of novel <i>in vitro</i> substrates of JmjC KDMs</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the specific steps for determining a substrate recognition motif for the KDM5A demethylase, as well as how to leverage this information for <i>in vitro</i> substrate discovery. Results for KDM3A are shown alongside KMD5A to highlight (1) how KDM substrate preference deviates between enzymes and (2) that the method is not solely applicable to a single enzyme.</p>
<h3 id="sec1.1">Theoretical considerations</h3>
<p>The procedure for determining JmjC KDM substrate recognition motifs involves monitoring enzyme activity towards a library of peptides representing systematic mutations from the canonical substrate sequence (i.e., peptide permutation library). Analogously, this approach has been established for KMTs (<a href="#bib2">Dhayalan et al., 2011</a>; <a href="#bib10">Kudithipudi et al., 2014</a>; <a href="#bib11">Lanouette et al., 2015</a>; <a href="#bib17">Rathert et al., 2008a</a>, <a href="#bib18">2008b</a>). Thus, this methodology is limited to those JmjC KDMs with a known peptide substrate. Many methyl-modifying enzymes have established substrates within the histone code (<a href="#bib6">Hyun et al., 2017</a>). For example, the canonical histone substrate for the KDM5 family of enzymes is histone H3 trimethylated at lysine-4 (i.e., H3-K4me3).</p>
<h3 id="sec1.2">Practical considerations</h3>
<p>To note, the methodology begins assuming a pure source of recombinant enzyme is available. Recombinant KDM5A and KDM3A enzymes were expressed and purified as described (<a href="#bib9">Krishnan and Trievel, 2016</a>; <a href="#bib19">Rose et al., 2012</a>). Furthermore, although our example monitors JmjC KDM activity via detection of succinate (i.e., turnover of the 2-oxoglutarate cofactor), there are other methods for detecting <i>in vitro</i> JmjC KDM activity. This includes other techniques monitoring cofactor turnover and formaldehyde formation that would also be viable alternatives for detecting enzyme activity ((<a href="#bib4">Hirsilä et al., 2003</a>; <a href="#bib7">Kivirikko and Myllylä, 1982</a>; <a href="#bib8">Krishnan et al., 2012</a>; <a href="#bib13">Luo et al., 2006</a>).</p>
<h3 id="sec1.3">Design permutated peptide substrate (PPS) library sequences</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>Permutation libraries consist of peptide sequences whereby a given residue position is mutated to all other naturally occurring amino acids while leaving the remainder of the sequence unaltered. Systematically performing this for multiple residue positions enables the assessment of amino acid specificity of peptide-protein interactions over a defined window. The specificity of numerous KMTs has been mapped via this approach and the epitope is generally defined by the residues occurring directly proximal to the modification site. Structural analysis of JMJD14, a plant KDM5 enzyme, found that residues directly proximal to the H3-K4me3 target site are important for substrate recognition (<a href="#bib23">Yang et al., 2018</a>). Thus, the permutated window consisted of residues -3 to +5 amino acids relative to the H3-K4me3 methylation site (<u>ART</u><b>K</b><u>QTARK</u>STGGKA; K4 position bold, permutated window underlined).</p>
<ol>
<li>Retrieve peptide sequence known to permit demethylase activity (e.g., the H3-K4 sequence [ART(Kme3)QTARKSTGGKA] was used for KDM5A).</li>
<li>At each residue position within the desired window (e.g., −3 to +5 relative to the methylation site), generate 19 other sequences whereby the wild-type residue is exchanged to another naturally occurring amino acid (<a href="#tbl1">Table 1</a> shows an example for the -3 position only).
<table id="tbl1">
<caption>Table 1. Example of permutation of the H3-K4me3 peptide sequence at the -3 position (underlined)</caption>
<thead>
<tr>
<th>Mutation</th>
<th>Peptide sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>A(WT)</td>
<td>
<u>A</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>R</td>
<td>
<u>R</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>N</td>
<td>
<u>N</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>D</td>
<td>
<u>D</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>C</td>
<td>
<u>C</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>Q</td>
<td>
<u>Q</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>E</td>
<td>
<u>E</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>G</td>
<td>
<u>G</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>H</td>
<td>
<u>H</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>I</td>
<td>
<u>I</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>L</td>
<td>
<u>L</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>K</td>
<td>
<u>K</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>M</td>
<td>
<u>M</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>F</td>
<td>
<u>F</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>P</td>
<td>
<u>P</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>S</td>
<td>
<u>S</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>T</td>
<td>
<u>T</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>W</td>
<td>
<u>W</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>Y</td>
<td>
<u>Y</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
<tr>
<td>V</td>
<td>
<u>V</u>RT(Kme3)QTARKSTGGKA</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Add a tryptophan residue to the C-terminal end of the peptide sequence, separated from the peptide sequence by a flexible linker (e.g., 6-aminohexanoic acid (ahx)), to enable quantification of peptide concentration through tryptophan fluorescence.</div>
<h3 id="sec1.4">Preparation of peptides, cofactors, and buffer stocks</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Due to the focus of the protocol described herein, the nonessential need to obtain peptides via the same methodology used, and the broad availability of peptide synthesis protocols, a detailed synthesis protocol is not described. To note, the peptides were synthesized following standard Fmoc (N-(9-fluorenyl)methoxycarbonyl) chemistry, on an automated ResPep SL peptide synthesizer (Intavis), at a scale of 2 μmol following procedures previously described (<a href="#bib14">McKenna et al., 2021</a>; <a href="#bib22">Wei et al., 2018</a>).</p>
<ol start="3">
<li>Dissolve dry peptides in 1× phosphate-buffered saline (PBS; pH 7), or another activity-compatible buffer), to a final concentration of 10 mM.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Given that peptides were synthesized at a scale of 2 μmol, dissolve the peptides in 200 μL of 1× PBS to obtain a concentration of 10 mM.</div>
<div class="note">
<span class="note-title">Note:</span> Peptides synthesized in-house may be highly acidic due to residue trifluoroacetic acid and may require pH to be adjusted to 7 by gradual addition of sodium hydroxide (a 5 M NaOH solution was used to add low volumes of 5–10 μL to peptides).</div>
<div class="critical">
<span class="critical-title">Critical:</span> all buffers and reagents should be prepared in ATP-free water. This includes cofactors and buffers described below.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If peptides are synthesized in-house, it is highly recommended to quantify the peptides to ensure proper concentration. If an ahx-Trp is added to the C-terminal ends of peptide sequences, concentration may be determined by absorbance at 280 nm and the extinction coefficient of the peptide (available using the Expasy ProtParam tool; <a href="https://web.expasy.org/protparam/">https://web.expasy.org/protparam/</a>).</div>
<ol start="4">
<li>Prepare 4× peptide stocks (40 μM) by diluting peptides in 1× PBS. Next, prepare a 1 mM peptide stock solution for the wild-type substrate peptide in 1× PBS.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Peptide stocks should be aliquoted and stored at −20°C until use.</div>
<ol start="5">
<li>Make 10 mM solutions of ascorbic acid, α-ketoglutarate, and Fe(II)SO<sub>4</sub> in a volume of 1 mL using ATP-free water, in separate 1.5 mL Eppendorf tubes.
<ol type="a">
<li>Dilute 10 mM cofactor stocks to 1 mM with ATP-free water.</li>
</ol>
</li>
<li>Make a 40 mL solution of 0.5 M HEPES in a 50 mL conical tube. Dissolve HEPES in 30 mL of ATP-free water and adjust pH to 7.5 using 5 M NaOH. Top up volume to 40 mL to yield a final concentration of 0.5 M HEPES.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Stores HEPES solution at 4°C protected from light.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>KDM5A<sub>1-801</sub>-His</td>
<td>SGC Oxford</td>
<td>n/a</td>
</tr>
<tr>
<td>KDM3A<sub>515-1317</sub>-His-Flag</td>
<td>(<a href="#bib19">Rose et al., 2012</a>)</td>
<td>n/a</td>
</tr>
<tr>
<td>Fmoc-Ala-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41004</td>
</tr>
<tr>
<td>Fmoc-Arg(Pbf)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41002</td>
</tr>
<tr>
<td>Fmoc-Asn(Trt)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41007</td>
</tr>
<tr>
<td>Fmoc-Asp(OtBu)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41019</td>
</tr>
<tr>
<td>Fmoc-Cys(Trt)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41008</td>
</tr>
<tr>
<td>Fmoc-Glu(OtBu)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41005</td>
</tr>
<tr>
<td>Fmoc-Gln(Trt)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41011</td>
</tr>
<tr>
<td>Fmoc-Gly-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41010</td>
</tr>
<tr>
<td>Fmoc-His(Trt)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41017</td>
</tr>
<tr>
<td>Fmoc-Ile-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41018</td>
</tr>
<tr>
<td>Fmoc-Leu-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41003</td>
</tr>
<tr>
<td>Fmoc-Lys(Boc)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41001</td>
</tr>
<tr>
<td>Fmoc-Met-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41020</td>
</tr>
<tr>
<td>Fmoc-Phe-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41013</td>
</tr>
<tr>
<td>Fmoc-Pro-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41009</td>
</tr>
<tr>
<td>Fmoc-Ser(tBu)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41006</td>
</tr>
<tr>
<td>Fmoc-Thr(tBu)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41016</td>
</tr>
<tr>
<td>Fmoc-Trp(Boc)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41012</td>
</tr>
<tr>
<td>Fmoc-Tyr(tBu)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41014</td>
</tr>
<tr>
<td>Fmoc-Val-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 41015</td>
</tr>
<tr>
<td>Fmoc-Lys(Boc,Me)-OH</td>
<td>P3 Biosystems</td>
<td>Cat# 47238</td>
</tr>
<tr>
<td>Fmoc-Lys(Me)2-OH.HCl</td>
<td>P3 Biosystems</td>
<td>Cat# 47237</td>
</tr>
<tr>
<td>Fmoc-Lys(Me)3-OH chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat# F5062</td>
</tr>
<tr>
<td>HBTU</td>
<td>P3 Biosystems</td>
<td>Cat# 31001</td>
</tr>
<tr>
<td>NMM</td>
<td>Alfa Aesar</td>
<td>Cat# A12158</td>
</tr>
<tr>
<td>Rink Amide MBHA Resin</td>
<td>P3 Biosystems</td>
<td>Cat# 52002</td>
</tr>
<tr>
<td>Acetic anhydride</td>
<td>Fisher Scientific</td>
<td>Cat# 108-24-7</td>
</tr>
<tr>
<td>Piperidine</td>
<td>Sigma-Aldrich</td>
<td>Cat# 104094</td>
</tr>
<tr>
<td>DMF</td>
<td>Fisher Scientific</td>
<td>Cat# D119-20</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Commercial Alcohols</td>
<td>Cat# P016EAAN</td>
</tr>
<tr>
<td>DCM</td>
<td>Acros Organics</td>
<td>Cat# 354800025</td>
</tr>
<tr>
<td>Trifluoroacetic acid</td>
<td>Fisher Scientific</td>
<td>Cat# L06374</td>
</tr>
<tr>
<td>TIPS</td>
<td>Acros Organics</td>
<td>Cat# 214922500</td>
</tr>
<tr>
<td>Ethyl Ether</td>
<td>Fisher Scientific</td>
<td>Cat# E138-4</td>
</tr>
<tr>
<td>HEPES</td>
<td>BioShop</td>
<td>Cat# HEP005.1</td>
</tr>
<tr>
<td>Fe(II)SO<sub>4</sub>
</td>
<td>BDH Chemicals</td>
<td>Cat# B28400</td>
</tr>
<tr>
<td>α-ketoglutarate</td>
<td>Sigma-Aldrich</td>
<td>Cat# K-3752</td>
</tr>
<tr>
<td>Ascorbic acid</td>
<td>J.T. Baker Chemical Co.</td>
<td>Cat# B581.5</td>
</tr>
<tr>
<td>Tris(2-carboxyethyl)phosphine (TCEP) HCl salt</td>
<td>Sigma-Aldrich</td>
<td>Cat# 646547</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>BioShop</td>
<td>Cat# ALB001.100</td>
</tr>
<tr>
<td>Dimethyl sulfoxide</td>
<td>Bio Basic Canada</td>
<td>Cat# D0231</td>
</tr>
<tr>
<td>NaCl</td>
<td>BioShop</td>
<td>Cat# SOD002.1</td>
</tr>
<tr>
<td>KCl</td>
<td>BioShop</td>
<td>Cat# POC308.500</td>
</tr>
<tr>
<td>Sodium phosphate monobasic monohydrate</td>
<td>Sigma-Aldrich</td>
<td>Cat# S9638</td>
</tr>
<tr>
<td>Sodium phosphate dibasic anhydrous</td>
<td>BioShop</td>
<td>Cat# SPD307.500</td>
</tr>
<tr>
<td>Potassium phosphate monobasic</td>
<td>BioShop</td>
<td>Cat# PPM666.1</td>
</tr>
<tr>
<td>Acetic acid</td>
<td>Anachemia</td>
<td>Cat# 00598-463</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Succinate-Glo JmjC Demethylase/Hydroxylase Assay</td>
<td>Promega Corporation (<a href="#bib1">Alves et al., 2018</a>)</td>
<td>Cat# V7990</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Peptide Specificity Analyst (PeSA)</td>
<td>(<a href="#bib21">Topcu and Biggar, 2019</a>)</td>
<td>https://doi.org/10.5281/zenodo.6323540</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BioTek Cytation 5 microplate reader</td>
<td>BioTek</td>
<td>Cat# BTCYT5M</td>
</tr>
<tr>
<td>Greiner 384-well plate, white</td>
<td>Greiner Bio-One</td>
<td>Cat# 781075</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>2× Reaction Buffer/Peptide Mix</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.5 M HEPES (pH 7.5)</td>
<td>50 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>1 mM ascorbic acid</td>
<td>200 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>1 mM Fe(II)SO<sub>4</sub>
</td>
<td>20 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>1 mM 2-oxoglutarate</td>
<td>20 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>DMSO</td>
<td>2%</td>
<td>10 μL</td>
</tr>
<tr>
<td>ATP-free water</td>
<td>n/a</td>
<td>310 μL</td>
</tr>
<tr>
<td>1 mM peptide</td>
<td>20 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>2× Reaction Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>0.5 M HEPES (pH 7.5)</td>
<td>50 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>1 mM ascorbic acid</td>
<td>200 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>1 mM Fe(II)SO<sub>4</sub>
</td>
<td>20 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>1 mM 2-oxoglutarate</td>
<td>20 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>DMSO</td>
<td>2%</td>
<td>10 μL</td>
</tr>
<tr>
<td>ATP-free water</td>
<td>n/a</td>
<td>320 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Succinate Detection Reagent I</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Succinate-Glo<sup>TM</sup> Solution</td>
<td>n/a</td>
<td>5 μL</td>
</tr>
<tr>
<td>Acetoacetyl-CoA 100×</td>
<td>1×</td>
<td>5 μL</td>
</tr>
<tr>
<td>Succinate-Glo<sup>TM</sup> Buffer</td>
<td>n/a</td>
<td>500 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>510 μL</b></td>
</tr>
</tbody>
</table>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Here a peptide list comprising the known methylproteome was scored using JmjC KDM recognition motifs. This peptide list is provided (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Mmc1.xlsx">Table S1</a>, accessed from PhosphoSitePlus on 12-03-2020), however any peptide list can be used (<a href="#bib16">Hornbeck et al., 2012</a>).</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Determine non-saturating enzyme concentration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Given a fixed set of reaction parameters (e.g., cofactor concentration, pH, temperature, etc.), a user can determine the optimal concentration range of JmjC KDM yielding a non-saturated signal. Establishing a non-saturating enzyme concentration is key for downstream steps when comparing the effect of different peptides on JmjC KDM activity. This is achieved by dilution series of the given KDM from the highest concentration possible to a low or sub nanomolar concentration (e.g., &lt;1–10 nM). In our prototypic example of KDM5A and KDM3A, by dilution series we determined enzyme concentrations yielding non-saturating signal (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Determining non-saturating JmjC KDM concentration</div>
<p>(A and B) Two-fold dilution series of recombinant (A) KDM5A and (B) KDM3A enzymes, observing activity towards H3-K4me3 and H3-K9me2 peptides, respectively. The data represents the average of three luminescent readings and error bars represent the SEM (n=3). Data for KDM5A was previously published (<a href="#bib5">Hoekstra and Biggar, 2021</a>).</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> To maximize the signal-to-background ratio, we recommend users of this methodology establish optimal conditions (e.g., pH, temperature, etc.) prior to beginning this workflow. This may be done experimentally or by literature search.</div>
<div class="note">
<span class="note-title">Note:</span> Commercially available <i>Promega Succinate-Glo</i>™ assay reagents are stored at −80°C. Begin thawing reagent components on ice ∼1 h before use. For Acetoacetyl-CoA 100×, specifically, thaw at 22°C–24°C for 5 min before addition to Succinate-Glo<sup>TM</sup> Buffer.</div>
<ol>
<li>Place a white 384-well microplate on a Peltier device and set temperature to 4°C.</li>
<li>Prepare 2× Reaction Buffer/Peptide Mix.</li>
<li>Add 2.5 μL of 2× Reaction Buffer/Peptide Mix to all experimental wells in the 384-well microplate.</li>
<li>Thaw JmjC KDM stock protein on ice.</li>
<li>Perform a 20 μL dilution series (e.g., 4× or 2×) of the JmjC KDM protein in 1.5 mL Eppendorf tubes on ice in the same buffer used initially for storage (see <a href="#tbl2">Table 2</a> for an example dilution range).
<table id="tbl2" class="dense-table">
<caption>Table 2. Example of plate layout for JmjC 2× dilution series</caption>
<thead>
<tr>
<th>Column</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>9</th>
<th>10</th>
<th>11</th>
<th>12</th>
<th>13</th>
</tr>
</thead>
<tbody>
<tr>
<td>Row A-C (n=3)</td>
<td>NEC</td>
<td>10</td>
<td>5.0</td>
<td>2.5</td>
<td>1.3</td>
<td>0.63</td>
<td>0.31</td>
<td>0.16</td>
<td>0.078</td>
<td>0.039</td>
<td>0.020</td>
<td>0.01</td>
<td>0.005</td>
</tr>
</tbody>
</table>
<p class="table-legend">Initial concentration of purified KDM5A was 20 μM, yielding a final reaction concentration of 10 μM. Concentrations listed are in μM.</p>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Save at least a 20 μL aliquot of storage buffer for the “no enzyme” control.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Dilution series may be performed in higher or lower volumes depending on the number of replicate reactions performed.</div>
<ol start="6">
<li>Set Peltier device to 23°C.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other temperatures for the JmjC KDM reaction may be used (e.g., optimal temperature may be determined prior to this protocol or published in literature).</div>
<ol start="7">
<li>To initiate the reaction, add 2.5 μL of each JmjC KDM dilution to a specific microplate well containing the 2× Reaction Buffer/Peptide Mix (<a href="#tbl2">Table 2</a> shows an example plate layout).
<ol type="a">
<li>Set the 384-well microplate on a plate shaker for 2 min at 22°C–25°C.</li>
<li>Centrifuge the plate at 240 × <i>g</i> for 1 min at 23°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For the no enzyme control (NEC), add 2.5 μL of storage buffer containing an absence of JmjC KDM protein.</div>
<ol start="8">
<li>Allow the demethylation reaction to occur for 60 min at 23°C.</li>
<li>Add 5 μL of Succinate Detection Reagent I to each assay well.
<ol type="a">
<li>Shake the 384-well microplate for 2 min at 22°C–25°C.</li>
<li>Centrifuge the plate at 240 × <i>g</i> for 1 min at 23°C and incubate for 60 min at 23°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The time of JmjC reaction may be adjusted, however, the reaction time used in this step must be kept consistent when performing all downstream JmjC KDM reactions. For example, reactions with KDM3A occurred for 3 h.</div>
<ol start="10">
<li>Add 10 μL of Succinate Detection Reagent II.
<ol type="a">
<li>Shake the 384-well microplate for 2 min at 22°C–25°C.</li>
<li>Centrifuge the plate at 240 xg for 1 min at 23°C and incubate for 10 min at 23°C.</li>
</ol>
</li>
<li>Read the luminescence from each well using a microplate reader.</li>
<li>Calculate the average luminescence signal across technical replicates.</li>
<li>For each condition, subtract the average signal from that of the average of the NEC to account for background luminescence.</li>
<li>Using data analysis/graphing software, such as GraphPad Prism, visualize the data to determine non-saturating and linear dose responsive JmjC KDM concentrations (<a href="#fig1">Figure 1</a>). Choose a concentration of protein within the mid-linear range of relative activity.
<ol type="a">
<li>See <a href="#sec6.1">Troubleshooting 1</a> if JmjC KDM activity is difficult to observe.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Validate KDM methyl-state preference</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>This step aims to validate the methyl-state preference (i.e., mono-, di-, tri-methylation) of the recombinant JmjC KDM for an established substrate. This is important as many JmjC KDMs have been shown to be capable of discriminating between different methyl-states of the same substrate. As the peptide libraries used for this method are synthesized as one methyl-state, this validation should be performed prior to commercially ordering or synthesizing the full peptide permutation library. Given the optimal concentration of the JmjC KDM determined in the previous step, the JmjC KDM activity can be monitored by performed reactions with peptide substrates of different methylation states. For assay validation, it is important to include an unmethylated substrate and a ‘no peptide control’ (NPC) alongside the standard NEC.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Some JmjC KDMs can convert 2-oxoglutarate to succinate in an appreciable extent in the absence of methylated peptide substrate. As a result, a ‘no peptide control (NPC) is critical to use going forward to assess the true level of relative demethylation activity (<a href="#tbl3">Table 3</a>).</div>
<ol start="15">
<li>Thaw 40 μM wild-type substrate (including null-, mono-, di-, and tri-methyl state peptides) on ice.
<table id="tbl3">
<caption>Table 3. Example of plate layout for testing JmjC KDM activity towards differentially methylated peptides (monomethylation [Kme1], demethylation [Kme2], and trimethylation [Kme3])</caption>
<thead>
<tr>
<th>Column</th>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
</tr>
</thead>
<tbody>
<tr>
<td>Row A-C (n=3)</td>
<td>NEC</td>
<td>NPC</td>
<td>Null</td>
<td>Kme1</td>
<td>Kme2</td>
<td>Kme3</td>
</tr>
</tbody>
</table>
</li>
<li>Repeat steps 1–3, except using 2× Reaction Buffer (similar to 2× Reaction Buffer/Peptide Mix minus peptide substrate).
<ol type="a">
<li>Keep 2× Reaction Buffer on ice for same day use.</li>
</ol>
</li>
<li>Add 1.25 μL of 40 μM wild-type peptide substrates, individually, to each experimental well containing 2× Reaction Buffer.
<ol type="a">
<li>For the NPC, use 1.25 μL of 1× PBS.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If peptides were dissolved in a buffer other than 1× PBS, use the peptide buffer for the NPC.</div>
<ol start="18">
<li>Thaw JmjC KDM stock protein on ice and dilute to 4× the optimal concentration determined in step 14.
<ol type="a">
<li>Considering the number of experimental conditions requiring enzyme (5 in total), the volume of JmjC KDM needed per reaction (1.25 μL), and the number of replicates (e.g., n=3), make at least 20.63 μL of 2× JmjC KDM (this is 10% excess of the minimum volume needed (18.75 μL)).</li>
<li>Perform the protein dilution with the JmjC storage buffer.</li>
</ol>
</li>
<li>Set Peltier device to 23°C (or appropriate constant temperature).</li>
<li>Add 1.25 μL of 4× JmjC KDM dilution to all experimental wells, except for the NEC. Add 1.25 μL JmjC KDM storage buffer to the NEC.</li>
<li>Repeat steps 8–13.</li>
<li>For each condition, subtract the signal from that of the NPC to account for demethylation-uncoupled 2-oxoglutarate turnover.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Represent methyl-state preference as relative JmjC KDM activity; normalize subtracted luminescent signals to that of the peptide displaying the highest signal. Again, using data analysis/ graphing software visualize the data to represent methyl-state preference (<a href="#fig2">Figure 2</a>).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Validation of JmjC KDM methyl-state preference</div>
<p>(A and B) Methyl-state specificity of (A) KDM5A and (B) KDM3A towards differentially methylated H3-K4 and H3-K9 peptides, respectively. KDM activity towards non-methylated, mono-, di-, and tri-methylated (i.e., me0/1/2/3) peptides at the indicated residues was assessed. Data represents the mean luminescence and standard deviation (n=3) normalized to the max signal. Data for KDM5A was previously published (<a href="#bib5">Hoekstra and Biggar, 2021</a>).</p>
</figcaption>
</figure>
<h3 id="sec3.3">Determine JmjC KDM substrate preference by permutated peptide substrate (PPS) library</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h (per mutation position)</div>
<p>These steps aim to map JmjC KDM substrate preference by monitoring activity towards all peptides in the PPS library. At this point, all peptides within the library should be methylated according to the preferred methyl-state (e.g., H3-K4me3 and H3-K9me2 for KDM5A and KDM3A, respectively). The experimental set-up is performed in the same manner as the previous section (methyl-state validation), except using permutated peptides alongside the wild-type peptide and reaction controls (<a href="#tbl4">Table 4</a>). In analyzing the data relative to the positive control (wild-type peptide) and negative controls (no peptide and no enzyme controls), one can determine the relative effect of individual amino acid mutations on JmjC KDM activity.</p>
<table id="tbl4" class="dense-table">
<caption>Table 4. Example of plate layout for PPS library experiments</caption>
<thead>
<tr>
<th rowspan="2">Column</th>
<th colspan="20">Experimental permutated peptides</th>
<th colspan="2">Controls</th>
</tr>
<tr>
<th>1</th>
<th>2</th>
<th>3</th>
<th>4</th>
<th>5</th>
<th>6</th>
<th>7</th>
<th>8</th>
<th>9</th>
<th>10</th>
<th>11</th>
<th>12</th>
<th>13</th>
<th>14</th>
<th>15</th>
<th>16</th>
<th>17</th>
<th>18</th>
<th>19</th>
<th>20</th>
<th>21</th>
<th>22</th>
</tr>
</thead>
<tbody>
<tr>
<td>Row A-D (n=4)</td>
<td>R</td>
<td>H</td>
<td>D</td>
<td>E</td>
<td>N</td>
<td>Q</td>
<td>A</td>
<td>G</td>
<td>P</td>
<td>I</td>
<td>L</td>
<td>V</td>
<td>M</td>
<td>F</td>
<td>Y</td>
<td>W</td>
<td>S</td>
<td>T</td>
<td>C</td>
<td>K</td>
<td>NPC</td>
<td>NEC</td>
</tr>
</tbody>
</table>
<p class="table-legend">The following is set up for mutations in one residue position. Each single letter amino acid designation is representative of the amino acid substitution occurring in the mutated peptide substrate.</p>
<div class="note">
<span class="note-title">Note:</span> To limit variability in the start of reaction time between experimental conditions; it is recommended to assess JmjC KDM activity towards peptides representing one residue position of the PPS library at a time.</div>
<ol start="23">
<li>Thaw 40 μM peptide stocks for the PPS library position being tested on ice.</li>
<li>Make fresh 2× Reaction Buffer (e.g., for each residue position, 242 μL of 2× Reaction Buffer is required (22 conditions × 4 replicates × 2.5 μL = 220 μL + 10% extra).</li>
<li>Repeat steps 1–3, using the freshly prepared 2× Reaction Buffer.</li>
<li>Add 1.25 μL of 4× (40 μM) peptide, individually, to experimental wells containing 2× Reaction Buffer. For the NPC, use 1.25 μL of 1× PBS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Again, if peptides were dissolved in a buffer other than 1× PBS, use that buffer for the NPC.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Ensure each run of the experiment contains a reaction condition with a wild-type peptide. All mutation positions have a wild-type peptide. However, if mutation positions are not run in full, a wild-type peptide must be included.</div>
<ol start="27">
<li>Thaw JmjC KDM stock on ice and dilute to 4× the optimal concentration determined in the previous step in JmjC storage buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Consider the number of experimental conditions requiring enzyme (e.g., the volume of JmjC KDM needed per reaction well (1.25 μL), and the number of replicates (e.g., n=4), make 115.5 μL of 2× JmjC KDM (this is 10% excess of what is needed; 21 conditions × 4 replicates × 1.25 μL = 105 μL)).</div>
<ol start="28">
<li>Set Peltier device to 23°C (or appropriate constant temperature).</li>
<li>Add 1.25 μL of 4× JmjC KDM solution to all experimental peptide conditions (wild-type peptide and mutant peptides) and the NPC well. Add 1.25 μL JmjC KDM storage buffer to the NEC well.</li>
<li>Repeat steps 8–13.</li>
<li>For each condition, subtract the average signal of the NPC to account for demethylation-uncoupled 2-oxoglutarate turnover.</li>
<li>To assess the relative effect of mutations on JmjC KDM activity; normalize substrate luminescent signals to that of the wild-type peptide.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Normalization is achieved from dividing the luminescent signals of reactions with experimental peptides by that of the WT peptide. This is critical to gauge the relative effect of individual mutations on enzyme activity.</div>
<ol start="33">
<li>Repeat steps 23–32 for each mutation position.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Represent relative activity data as a heat-map to visualize JmjC KDM substrate preference (<a href="#fig3">Figure 3</a>) using basic data analysis software (e.g., Microsoft Excel).</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representation of KDM substrate preference as a heat-map</div>
<p>(A and B) Relative activity of recombinant (A) KDM5A and (B) KDM3A were assessed towards permutation libraries of their corresponding canonical substrates of preferred methyl state (i.e., H3-K4me3 and H3-K9me2, respectively). X-axis and y-axis represent the wild-type peptide sequence and amino acid substitutions, respectively. Location of wild-type peptide (i.e., relative activity 1.0) spots are defined by the pink borders.</p>
</figcaption>
</figure>
<h3 id="sec3.4">JmjC KDM recognition motif generation, substrate prediction, and <i>in vitro</i> validation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<p>The major aim of this step is to use JmjC KDM substrate preference to prioritize peptides to be tested for <i>in vitro</i> KDM activity. Peptide Specificity Analyst (PeSA) software can be used to easily produce a candidate KDM recognition motif (<a href="#bib21">Topcu and Biggar, 2019</a>). In turn, these candidate recognition motifs can then be used to score and prioritize the methyllysine proteome for candidate substrates.</p>
<ol start="34">
<li>Download and open PeSA software (<a href="#bib21">Topcu and Biggar, 2019</a>; also available at <a href="https://github.com/EmineTopcu/PeSA">https://github.com/EmineTopcu/PeSA</a>).</li>
<li>Select “Permutation Array” tab at the menu.</li>
<li>Click “Load from File” and upload your JmjC KDM quantification matrix.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The quantification matrix can be uploaded with either the relative activity values or the background subtracted luminescent values. If the latter, normalization to the wild-type luminescent values in each position can be performed within the software to obtain relative activity values. Under “Normalized”, select “Per Row/Column” (see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Mmc1.xlsx">Table S2</a> for an example of the KDM5A quantification matrix).</div>
<div class="note">
<span class="note-title">Note:</span> Experimentally, only the residue positions flanking the methylation site are assessed. However, to maintain the central target lysine within the visualization of the recognition motif; in the quantification matrix one can assign the central methylated K the relative value of “1” and all other mutations in this central position as “0”.</div>
<ol start="37">
<li>Set the “Threshold” to the desired the value.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In this context the “Threshold” defines the minimum level of relative JmjC KDM activity for an amino acid substitution to be included within the candidate recognition motif. Keep in mind that stringency increases with higher threshold values.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Another threshold value may be used and should be considered when <a href="#sec6.3">Troubleshooting problem 2</a>. <a href="#fig4">Figure 4</a> represents KDM5A and KDM3A recognition motifs set at various threshold values (e.g., 0.25, 0.5, 0.75, and 1.0).</div>
<ol start="38">
<li>Obtain the recognition motif under the “motif” tab on the left-hand side of the window.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representation of the screening data as PeSA generated sequence motif</div>
<p>(A and B) The recognition motifs for (A) KDM5A and (B) KDM3A depict amino acid substitutions maintaining a minimum level relative activity defined by the threshold values on the y-axis.</p>
</figcaption>
</figure>
</li>
<li>Score peptides that possess windows which reflect known methylation sites in the methylproteome, using the PeSA generated candidate recognition motif.
<ol type="a">
<li>Each position in the queried sequence which shares an amino acid with the recognition motif contributes a value of 1 to the total score. An example of scoring is provided in <a href="#tbl5">Table 5</a>.
<table id="tbl5" class="dense-table">
<caption>Table 5. Example of motif-based scoring</caption>
<thead>
<tr>
<th rowspan="2">Protein-site</th>
<th rowspan="2">Queried sequence</th>
<th colspan="9">Position and tolerable substitutions within motif</th>
<th rowspan="2">PeSA score</th>
</tr>
<tr>
<th>-4 [CRGQ]</th>
<th>-3 [RIQCVAKT]</th>
<th>-2 [A]</th>
<th>-1 [R]</th>
<th>0 [K]</th>
<th>+1 [RTS]</th>
<th>+2 [T]</th>
<th>+3 [MG]</th>
<th>+4 [G]</th>
</tr>
</thead>
<tbody>
<tr>
<td>H3-K9 (known)</td>
<td>QTARKSTGG</td>
<td><b>Q</b></td>
<td><b>T</b></td>
<td><b>A</b></td>
<td><b>R</b></td>
<td><b>K</b></td>
<td><b>S</b></td>
<td><b>T</b></td>
<td><b>G</b></td>
<td><b>G</b></td>
<td>9</td>
</tr>
<tr>
<td>RPA2-K693 (unknown)</td>
<td>CQMGKQTMG</td>
<td><b>C</b></td>
<td><b>Q</b></td>
<td><b>M</b></td>
<td><b>G</b></td>
<td><b>K</b></td>
<td><b>Q</b></td>
<td><b>T</b></td>
<td><b>M</b></td>
<td><b>G</b></td>
<td>6</td>
</tr>
</tbody>
</table>
<p class="table-legend">Example of motif-based scoring. Example of scoring 9-mer sequences with KDM3A motif (defined by amino acids maintaining at least 100% relative activity). Tolerable amino acids at each position are shown in brackets. Matching and mismatching residues, compared to the motif, are shown in green and red, respectively. PeSA score is defined by the number of matching residues.</p>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Given the recognition motif produced and Kme position, ensure that the queried windows are of the same length and Kme residue in question is in the correct position. The PeSA score reflects the number of residues in the queried peptide matching those in the recognition motif at their corresponding positions. As a result, the predictions were made based on a 9-residue recognition motif, and therefore, the highest possible score obtainable is a total PeSA score of 9 (i.e., for windows exactly matching the recognition motif) (<a href="#tbl5">Table 5</a>).</div>
<ol start="40">
<li>Determine mean (μ) and standard deviation (σ) of PeSA scores across all queried peptide sequences.</li>
<li>To define high-ranking peptide sequences, determine the PeSA score corresponding to 2 standard deviations above the mean.
<ol type="a">
<li>For KDM5A, using the 0.5 relative activity recognition motif, a PeSA score of 9 reflects high-ranking peptide sequences (μ = 6.6, σ = 1.2) (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Distribution of peptide scores and testing <i>in vitro</i> KDM5A activity</div>
<p>(A) Gaussian distribution of PeSA scores of methylproteome peptides scored with the 0.5 relative activity KDM5A recognition motif. High- and low-ranking substrates are defined by PeSA scores occurring 2 standard deviations (σ = 1.2) above (or equal to) or below (or equal to) the population mean (μ = 6.6), respectively.</p>
<p>(B) Relative KDM5A activity towards differentially classified peptides sequences. Substrates were randomly chosen within their respective classification and KDM5A activity is shown as relative to the H3-K4me3 substrate.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
<li>Repeat steps 23–32 in the previous section, except using high-ranking peptides instead of PPS library peptides to validate the <i>in vitro</i> activity of substrate predictions/rankings (<a href="#fig5">Figure 5</a>B).
<ol type="a">
<li>See <a href="#sec6.3">Troubleshooting 2</a> if JmjC KDM shows no activity to none, or few, of the predicted JmjC KDM substrates</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the focus is purely on JmjC KDM substrate discovery, we recommend testing as many of the high-ranking peptides as possible, if not all. However, if the user is also interested in assessing whether the method accurately predicts or enriches in highly active <i>in vitro</i> substrates for the select JmjC KDM, one can assess a select number of random substrates with high-, medium-, and low-ranking PeSA scores (<a href="#fig6">Figure 6</a> demonstrates this analysis for KDM5A).</div>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Luminescent signal from KDM3A reaction, detected with the original (black) and modified (blue) assay protocols</div>
<p>Results for the modified assay protocol were plotted over the results of the original assay protocol displayed in <a href="#fig1">Figure 1</a>B.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>In our experience, PeSA scores of large lists of peptide sequences (e.g., methylproteome; n=2,155 sequences) follow a Gaussian distribution. For example, <a href="#fig5">Figure 5</a>A shows the distribution of PeSA scores of methylproteome sequences, scored with the 0.5 relative activity recognition motif of KDM5A. Thus, applying appropriate thresholds, based on standard deviation, to classify substrates as ‘high-ranking’ will result in a small number of candidate substrates to test <i>in vitro</i> JmjC KDM activity.</p>
<p>If testing relative JmjC KDM activity towards predicted substrates, we expect PeSA scores to correlate with <i>in vitro</i> JmjC KDM activity. For example, we sought to test KDM5A activity towards a handful of high, medium- and low-ranking substrates (<a href="#fig5">Figure 5</a>B). We were able to observe that none of the medium and low-ranking substrates displayed any significant KDM5A activity (activity defined as greater than 50% of H3-K4me3 activity). Furthermore, we observed KDM5A activity towards 90% of high-ranked peptides. This verifies that the 0.5 relative activity KDM5A recognition motif was accurate in predicting peptide sequences that would be amenable to KDM5A activity <i>in vitro</i>. Ultimately, this added step of validation does prioritize several potential substrates to refine and focus further validation efforts on.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<h3 id="sec5.1">Limitation 1</h3>
<p>A limitation to consider is that the relative importance of the individual amino acids at each position in the recognition motif is determined in the context of a fixed sequence and thus sequence bias may influence the results. This is a consequence of using permutation-based exploration of enzyme specificity. It is certainly possible that the relative importance of a given amino acid is only observable when occurring in the presence of specific amino acids at other positions. Supporting this, and in the context of methyl-binding domains, JMJD2A-double Tudor domain binding to permutations of H3-K23me3, H3-K4me3, and H4-K20me3 show distinct specificities depending on wild-type sequence used (<a href="#bib12">Liu et al., 2013</a>).</p>
<h3 id="sec5.2">Limitation 2</h3>
<p>The protocol defined here assumes recognition of substrates by a given enzyme is specified by the residues directly proximal to the modification site (i.e., −4 to +5 positions). Although this has been shown to be the case for many KMT enzymes, and for Suv39H2 the +5 position also determines specificity (<a href="#bib20">Schuhmacher et al., 2015</a>), recognition of substrates may also be determined by more distant interactions. Additionally, on the H3 tail, more distal sequence elements were recently shown to be important for KDM5A-dependent demethylation of the H3-K4 site (<a href="#bib15">Petronikolou et al., 2020</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low level of observable enzyme activity (i.e., low signal-to-background ratio). This problem is referring to corresponding protocol step 14.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>A low observable enzyme activity could be due to several factors such as reaction buffer composition, pH, temperature, time, as well as substrate and cofactor concentration. Each factor mentioned may be optimized individually. Additionally, additives may affect enzyme activity. For example, KDM3A activity may be improved nearly two-fold by addition of TCEP at certain concentrations, whereas higher concentrations of sodium chloride hinder KDM3A activity (<a href="#bib24">Yu et al., 2014</a>). Furthermore, peptide length may influence the level of enzyme activity. For example, the affinity of KDM3A for the H3-K9 substrate has been observed to be nearly 200-fold greater when using 21-mer peptides compared to 15-mer peptides (<a href="#bib3">Goda et al., 2013</a>; <a href="#bib24">Yu et al., 2014</a>).</p>
<p>Furthermore, if the level of uncoupled 2-oxoglutarate conversion (i.e., succinate production in absence of peptide) is relatively high compared to succinate formation in the presence of peptide; true demethylation activity may be difficult to observe using this assay or other assays detecting 2-oxoglutarate turnover. If this is the case, we suggest using assays directly assessing demethylation (e.g., formaldehyde turnover). However, if a significant difference is still able to be observed between the two conditions (i.e., JmjC KDM activity in the presence of peptide versus no peptide), succinate detection may still be a feasible technique. In this case, the uncoupled 2-oxoglutarate conversion must be considered as a baseline.</p>
<p>Finally, as JmjC KDMs belong to the Fe(II)/2-oxoglutarate-dependent family of dioxygenases they are susceptible to product inhibition conferred by the formation of succinate. The potency of inhibition varies depending on the specific enzyme of interest. To attenuate this effect, the assay can be performed in a format such that any succinate formed is immediately converted during the demethylation reaction, rather than converting succinate at the end of the JmjC KDM reaction (e.g., incorporating coupling enzymes in the KDM reaction). To achieve this, Succinate-Glo Solution and Acetoacetyl-CoA, provided in the assay kit, may be incorporated in the demethylation reaction by ensuring each of these components are diluted 100-fold in the final assay volume. Although the materials in the kit utilized is proprietary, succinate may be directly converted to succinyl-CoA by 3-oxoacid-CoA-transferase in the presence of acetoacetyl-CoA. For KDM3A, specifically, we found that the modified protocol increases the luminescent signal (<a href="#fig6">Figure 6</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The JmjC KDM assessed shows no activity to none, or few, of the predicted JmjC KDM substrates. This problem is referring to corresponding protocol step 42.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>No activity towards any of the substrate predictions could be due to several factors, such as limitations inherent to permutation-based exploration of specificity. The latter cannot be addressed by adjusting variables within this protocol and would need to be resolved by assessing specificity in another manner or by using multiple substrate permutations. However, the adjustable variables include reducing the stringency of substrate predictions to be more permissive for substrate discovery (i.e., use recognition motif defined by a lower activity threshold). It may also be beneficial to test activity of the given JmjC KDM at multiple substrate concentrations. Finally, and related to Limitation 2, the critical positions defining specificity may exist outside of the permutated positions and thus it may be beneficial to expand this window and re-predicting substrates based on this expanded recognition motif.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alves, J., Vidugiris, G., Goueli, S.A., and Zegzouti, H. (2018). Bioluminescent high-throughput succinate detection method for monitoring the activity of JMJC histone demethylases and Fe(II)/2-oxoglutarate-dependent dioxygenases. SLAS Discov. <i>23</i>, 242-254. <a class="external-link" href="https://doi.org/10.1177/2472555217745657">https://doi.org/10.1177/2472555217745657</a></p>
<p id="bib2">Dhayalan, A., Kudithipudi, S., Rathert, P., and Jeltsch, A. (2011). Specificity analysis-based identification of new methylation targets of the SET7/9 protein lysine methyltransferase. Chem. Biol. <i>18</i>, 111-120. <a class="external-link" href="https://doi.org/10.1016/J.CHEMBIOL.2010.11.014">https://doi.org/10.1016/J.CHEMBIOL.2010.11.014</a></p>
<p id="bib3">Goda, S., Isagawa, T., Chikaoka, Y., Kawamura, T., and Aburatani, H. (2013). Control of histone H3 lysine 9 (H3K9) methylation state via cooperative two-step demethylation by Jumonji domain containing 1A (JMJD1A) homodimer ∗. J. Biol. Chem. <i>288</i>, 36948-36956. <a class="external-link" href="https://doi.org/10.1074/JBC.M113.492595">https://doi.org/10.1074/JBC.M113.492595</a></p>
<p id="bib4">Hirsilä, M., Koivunen, P., Günzler, V., Kivirikko, K.I., and Myllyharju, J. (2003). Characterization of the human prolyl 4-hydroxylases that modify the hypoxia-inducible factor. J. Biol. Chem. <i>278</i>, 30772-30780. <a class="external-link" href="https://doi.org/10.1074/JBC.M304982200">https://doi.org/10.1074/JBC.M304982200</a></p>
<p id="bib5">Hoekstra, M. and Biggar, K.K. (2021). Identification of in vitro JMJD lysine demethylase candidate substrates via systematic determination of substrate preference. Anal. Biochem. <i>633</i>, 114429. <a class="external-link" href="https://doi.org/10.1016/J.AB.2021.114429">https://doi.org/10.1016/J.AB.2021.114429</a></p>
<p id="bib6">Hyun, K., Jeon, J., Park, K., and Kim, J. (2017). Writing, erasing and reading histone lysine methylations. Exp. Mol. Med. <i>49</i>, e324. <a class="external-link" href="https://doi.org/10.1038/emm.2017.11">https://doi.org/10.1038/emm.2017.11</a></p>
<p id="bib7">Kivirikko, K.I. and Myllylä, R. (1982). Posttranslational enzymes in the biosynthesis of collagen: intracellular enzymes. Methods Enzymol. <i>82</i>, 245-304. <a class="external-link" href="https://doi.org/10.1016/0076-6879(82)82067-3">https://doi.org/10.1016/0076-6879(82)82067-3</a></p>
<p id="bib8">Krishnan, S., Collazo, E., Ortiz-Tello, P.A., and Trievel, R.C. (2012). Purification and assay protocols for obtaining highly active Jumonji C demethylases. Anal. Biochem. <i>420</i>, 48-53. <a class="external-link" href="https://doi.org/10.1016/J.AB.2011.08.034">https://doi.org/10.1016/J.AB.2011.08.034</a></p>
<p id="bib9">Krishnan, S. and Trievel, R.C. (2016). Purification, biochemical analysis, and structure determination of JmjC lysine demethylases. In Methods in Enzymology (Academic Press Inc.), pp. 279-301. <a class="external-link" href="https://doi.org/10.1016/bs.mie.2016.01.023">https://doi.org/10.1016/bs.mie.2016.01.023</a></p>
<p id="bib10">Kudithipudi, S., Lungu, C., Rathert, P., Happel, N., and Jeltsch, A. (2014). Substrate specificity analysis and novel substrates of the protein lysine methyltransferase NSD1. Chem. Biol. <i>21</i>, 226-237. <a class="external-link" href="https://doi.org/10.1016/j.chembiol.2013.10.016">https://doi.org/10.1016/j.chembiol.2013.10.016</a></p>
<p id="bib11">Lanouette, S., Davey, J.A., Elisma, F., Ning, Z., Figeys, D., Chica, R.A., and Couture, J.F. (2015). Discovery of substrates for a SET domain lysine methyltransferase predicted by multistate computational protein design. Structure <i>23</i>, 206-215. <a class="external-link" href="https://doi.org/10.1016/J.STR.2014.11.004">https://doi.org/10.1016/J.STR.2014.11.004</a></p>
<p id="bib12">Liu, H., Galka, M., Mori, E., Liu, X., Lin, Y.f., Wei, R., Pittock, P., Voss, C., Dhami, G., Li, X., et al. (2013). A method for systematic mapping of protein lysine methylation identifies new functions for HP1β in DNA damage repair. Mol. Cell <i>50</i>, 723. <a class="external-link" href="https://doi.org/10.1016/J.MOLCEL.2013.04.025">https://doi.org/10.1016/J.MOLCEL.2013.04.025</a></p>
<p id="bib13">Luo, L., Pappalardi, M.B., Tummino, P.J., Copeland, R.A., Fraser, M.E., Grzyska, P.K., and Hausinger, R.P. (2006). An assay for Fe(II)/2-oxoglutarate-dependent dioxygenases by enzyme-coupled detection of succinate formation. Anal. Biochem. <i>353</i>, 69-74. <a class="external-link" href="https://doi.org/10.1016/J.AB.2006.03.033">https://doi.org/10.1016/J.AB.2006.03.033</a></p>
<p id="bib14">McKenna, M., Balasuriya, N., Zhong, S., Li, S.S.C., and O’Donoghue, P. (2021). Phospho-form specific substrates of protein kinase B (AKT1). Front. Bioeng. Biotechnol. <i>8</i>, 1580. <a class="external-link" href="https://doi.org/10.3389/FBIOE.2020.619252/BIBTEX">https://doi.org/10.3389/FBIOE.2020.619252/BIBTEX</a></p>
<p id="bib15">Petronikolou, N., Longbotham, J.E., and Fujimori, D.G. (2020). Extended recognition of the histone H3 tail by histone demethylase KDM5A. Biochemistry <i>59</i>, 647-651. <a class="external-link" href="https://doi.org/10.1021/acs.biochem.9b01036">https://doi.org/10.1021/acs.biochem.9b01036</a></p>
<p id="bib16">Hornbeck, P.V., Kornhauser, J.M., Tkachev, S., Zhang, B., Skrzypek, E., Murray, B., Latham, V., and Sullivan, M. (2012). PhosphoSitePlus: a comprehensive resource for investigating the structure and function of experimentally determined post-translational modifications in man and mouse. Nucleic Acids Res. <i>40</i>, D261-D270. <a class="external-link" href="https://doi.org/10.1093/NAR/GKR1122">https://doi.org/10.1093/NAR/GKR1122</a></p>
<p id="bib17">Rathert, P., Dhayalan, A., Murakami, M., Zhang, X., Tamas, R., Jurkowska, R., Komatsu, Y., Shinkai, Y., Cheng, X., and Jeltsch, A. (2008a). Protein lysine methyltransferase G9a acts on non-histone targets. Nat. Chem. Biol. <i>4</i>, 344-346. <a class="external-link" href="https://doi.org/10.1038/nchembio.88">https://doi.org/10.1038/nchembio.88</a></p>
<p id="bib18">Rathert, P., Zhang, X., Freund, C., Cheng, X., and Jeltsch, A. (2008b). Analysis of the substrate specificity of the Dim-5 histone lysine methyltransferase using peptide arrays. Chem. Biol. <i>15</i>, 5-11. <a class="external-link" href="https://doi.org/10.1016/J.CHEMBIOL.2007.11.013">https://doi.org/10.1016/J.CHEMBIOL.2007.11.013</a></p>
<p id="bib19">Rose, N.R., Woon, E.C.Y., Tumber, A., Walport, L.J., Chowdhury, R., Li, X.S., King, O.N.F., Lejeune, C., Ng, S.S., Krojer, T., et al. (2012). The plant growth regulator daminozide is a selective inhibitor of the human KDM2/7 histone demethylases. J. Med. Chem. <i>55</i>, 6639. <a class="external-link" href="https://doi.org/10.1021/JM300677J">https://doi.org/10.1021/JM300677J</a></p>
<p id="bib20">Schuhmacher, M.K., Kudithipudi, S., Kusevic, D., Weirich, S., and Jeltsch, A. (2015). Activity and specificity of the human SUV39H2 protein lysine methyltransferase. Biochim. Biophys. Acta <i>1849</i>, 55-63. <a class="external-link" href="https://doi.org/10.1016/J.BBAGRM.2014.11.005">https://doi.org/10.1016/J.BBAGRM.2014.11.005</a></p>
<p id="bib21">Topcu, E. and Biggar, K.K. (2019). PeSA: a software tool for peptide specificity analysis. Comput. Biol. Chem. <i>83</i>, 107145. <a class="external-link" href="https://doi.org/10.1016/j.compbiolchem.2019.107145">https://doi.org/10.1016/j.compbiolchem.2019.107145</a></p>
<p id="bib22">Wei, R., Kaneko, T., Liu, X., Liu, H., Li, L., Voss, C., Liu, E., He, N., and Li, S.S.C. (2018). Interactome mapping uncovers a general role for numb in protein kinase regulation. Mol. Cell. Proteomics <i>17</i>, 2216-2228. <a class="external-link" href="https://doi.org/10.1074/MCP.RA117.000114">https://doi.org/10.1074/MCP.RA117.000114</a></p>
<p id="bib23">Yang, Z., Qiu, Q., Chen, W., Jia, B., Chen, X., Hu, H., He, K., Deng, X., Li, S., Tao, W.A., et al. (2018). Structure of the arabidopsis JMJ14-H3K4me3 complex provides insight into the substrate specificity of KDM5 subfamily histone demethylases. Plant Cell <i>30</i>, 167-177. <a class="external-link" href="https://doi.org/10.1105/tpc.17.00666">https://doi.org/10.1105/tpc.17.00666</a></p>
<p id="bib24">Yu, W., Eram, M.S., Hajian, T., Szykowska, A., Burgess-Brown, N., Vedadi, M., and Brown, P.J. (2014). A scintillation proximity assay for histone demethylases. Anal. Biochem. <i>463</i>, 54-60. <a class="external-link" href="https://doi.org/10.1016/J.AB.2014.06.023">https://doi.org/10.1016/J.AB.2014.06.023</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests should be directed to and will be fulfilled by the lead contact Dr. Kyle K. Biggar (<a href="mailto:kyle_biggar@carleton.ca">kyle_biggar@carleton.ca</a>).</p>
<h4>Materials availability</h4>
<p>This protocol is not associated with any newly generated materials.</p>
<h4>Data and code availability</h4>
<p>The protocol includes the methylproteome peptide list (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Mmc1.xlsx">Table S1</a>) accessed from PhosphoSitePlus on 12-03-2020 (<a href="#bib16">Hornbeck et al., 2012</a>).</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Mmc1.xlsx">Table S1. Peptide list comprising the known methylproteome to be scored using JmjC KDM recognition motifs, related to materials and equipment</a></p>
<p><a id="mmc2" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1526-Mmc2.xlsx">Table S2. An example of the KDM5A quantification matrix used for scoring potential substrates, related to step 36</a></p>
<h3>Acknowledgments</h3>
<p>This work was supported by a <a href="https://doi.org/10.13039/501100000038">National Science and Engineering Research Council</a> (NSERC) Canada Discovery grant to K.K.B. (grant no. RGPIN-2016-06151) and W.G.W. (grant no. RGPIN-2017-06414). A.C. held a Canada Graduate Scholarship – Doctoral (CGS-D) from the NSERC of Canada. The KDM5A Sf9 construct was a generous gift from the Structural Genomics Consortium Oxford group. The KDM3A construct was a generous gift from Nicola Burgess-Brown.</p>
<h3>Author contributions</h3>
<p>Methodology, M.W. and A.C.; data curation, M.H. and A.C.; validation, M.H.; writing – original draft, M.H. and A.C.; writing – review &amp; editing, A.C., K.K.B., and W.G.W.; conceptualization, K.K.B.; resources, K.K.B.; funding acquisition, K.K.B.; supervision, K.K.B. and W.G.W.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>