<article>
<section>
<h1 id="header">Polysome profiling followed by quantitative PCR for identifying potential micropeptide encoding long non-coding RNAs in suspension cell lines</h1>
<p><time datetime="2021-12-24">Published: December 24, 2021</time></p>
<p>Cai Han,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup> Linyu Sun,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Qi Pan,<sup><a href="#aff1">1</a></sup> Yumeng Sun,<sup><a href="#aff1">1</a></sup> Wentao Wang,<sup><a href="#aff1">1</a></sup> and Yueqin Chen<sup><a href="#aff1">1</a>,<a href="#fn3">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>MOE Key Laboratory of Gene Function and Regulation, State Key Laboratory for Biocontrol, School of Life Sciences, Sun Yat-sen University, Guangzhou, Guangdong 510275, China</p>
<p id="fn1"><sup>2</sup>These authors contributed equally</p>
<p id="fn2"><sup>3</sup>Technical contact</p>
<p id="fn3"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:hancai@mail2.sysu.edu.cn">hancai@mail2.sysu.edu.cn</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:lsscyq@mail.sysu.edu.cn">lsscyq@mail.sysu.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101037">10.1016/j.xpro.2021.101037</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Micropeptides are emerging as important regulators of various cellular processes. Long non-coding RNAs (lncRNAs) serve as a source of micropeptide-encoding small reading frames. The techniques to detect micropeptides or translating lncRNAs, such as mass spectrometry and ribosome profiling, are sophisticated and expensive. Here, we present an easy and cost-effective protocol to screen for potential micropeptide-encoding lncRNAs by polysome profiling in suspension cell lines. When combined with quantitative PCR, this protocol facilitates the identification of a number of translating lncRNAs simultaneously.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref7">Sun et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for rapid detection of the potential micropeptide-encoding lncRNAs</li>
<li>Detail a complete schedule for polysome profiling</li>
<li>Provide a protocol to distinguish RNAs with low or high translation activity</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Make sure that all the required reagents and items (refer to “<a href="#key-resources-table">key resources table</a>” and “<a href="#materials-and-equipment">materials and equipment</a>” section) are ready to use. This protocol describes steps for MV4-11 cells. However, the adaptability of this method makes its application extended to other suspension cells or adherent cells with necessary modifications.</p>
<p>RNase-free environment is required for successfully executing this protocol. RNase ZAP<sup>TM</sup> could be applied for the work area, disposable gloves and other contacts to eliminate ubiquitous RNase contamination. Additional precautions, including the use of RNase-free consumables, baking of glass wares at 180°C for at least 4 h, operations and samples kept at 4°C, and the wearing of headgear, should also be taken.</p>
<h3 id="sec1.1">MV4-11 cell culture and expansion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<ol>
<li>The minimum number of cells we used on day 1 of the protocol is 2 ∗ 10<sup>7</sup> MV4-11 cells per sample. The fewer number of cells than that may bring difficulties to recover sufficient RNAs for subsequent analysis. Expand the cells to the desired number ahead of time.</li>
</ol>
<h3 id="sec1.2">Preparation of buffer stock</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="2">
<li>The reagents required for this protocol are easy to get. The stock buffers are prepared as listed below. It is essential to prepare buffer stock and all the solutions mentioned below with DEPC-treated water.</li>
</ol>
<h3 id="sec1.3">Just in time preparation of sucrose buffer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="3">
<li>The day before starting the experiment (day 0), prepare the sucrose buffer as described below. Vortex to facilitate the dissolve of sucrose in the buffer and let it stand for overnight (14–16 h) to eliminate bubbles (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Preparation of sucrose buffer</div>
<p>(A) Sucrose buffer with bubbles caused by vortexing to dissolve sucrose.</p>
<p>(B) Sucrose buffer without bubbles after standing overnight (14–16 h).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The sucrose buffers could be filtered through 0.22 μm filters.</div>
<div class="note">
<span class="note-title">Note:</span> For 10% and 45% sucrose buffer, preparation of about 7 mL buffer each is sufficient for one sample when using SW41 ultracentrifuge tubes. At least two samples (one experimental sample and one blank sample) are needed in the subsequent ultracentrifugation for balancing.</div>
<h3 id="sec1.4">Just in time preparation of lysis buffer</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="4">
<li>On day 1, prepare fresh lysis buffers as described below. The RNase inhibitor, proteinase inhibitor or phenylmethylsulfonyl fluoride (PMSF), dithiothreitol (DTT), and cycloheximide are added before use.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Cycloheximide (CHX)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 239764</td>
</tr>
<tr>
<td>RNasin® Ribonuclease Inhibitor</td>
<td>Promega</td>
<td>Cat# N2115</td>
</tr>
<tr>
<td>1×complete ULTRA protease inhibitor</td>
<td>Roche</td>
<td>Cat# 5892970001</td>
</tr>
<tr>
<td>Phenylmethylsulfonyl fluoride (PMSF)</td>
<td>Beyotime</td>
<td>Cat# ST506</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>Sigma-Aldrich</td>
<td>Cat# D0632</td>
</tr>
<tr>
<td>RNase ZAP<sup>TM</sup>
</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9780</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>Sigma-Aldrich</td>
<td>Cat# D6750</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat# V900116</td>
</tr>
<tr>
<td>Magnesium chloride</td>
<td>Sigma-Aldrich</td>
<td>Cat# M8266</td>
</tr>
<tr>
<td>IGEPAL</td>
<td>Sigma-Aldrich</td>
<td>Cat# 18896</td>
</tr>
<tr>
<td>Tris</td>
<td>Asegene</td>
<td>Cat# AS430789</td>
</tr>
<tr>
<td>IMDM modified medium with L-glutamine, HEPES</td>
<td>HyClone</td>
<td>Cat# SH30228.01</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Gibco</td>
<td>Cat# 10270-106</td>
</tr>
<tr>
<td>RNAiso Plus</td>
<td>Takara</td>
<td>Cat# 9109</td>
</tr>
<tr>
<td>GlycoBlue™ Coprecipitant</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM9516</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat# 459844</td>
</tr>
<tr>
<td>Chloroform</td>
<td>Sigma-Aldrich</td>
<td>Cat# C2432</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>Sigma-Aldrich</td>
<td>Cat# 563935</td>
</tr>
<tr>
<td>DEPC Treated Water</td>
<td>Sangon Biotech</td>
<td>Cat# B5001005-0500</td>
</tr>
<tr>
<td>Phosphate buffered saline (PBS)</td>
<td>Cellcook</td>
<td>Cat# CM2018</td>
</tr>
<tr>
<td>Hydrochloric Acid</td>
<td>Sigma-Aldrich</td>
<td>Cat# 258148</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PrimeScript RT reagent Kit</td>
<td>Takara</td>
<td>Cat# RR047A</td>
</tr>
<tr>
<td>TB Green Premix Ex Taq</td>
<td>Takara</td>
<td>Cat# RR420B</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>MV4-11</td>
<td>ATCC</td>
<td>Cat# CRL-9591; RRID: CVCL_0064</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>AL035446.1</i> Primer:Forward - GAAAAATGCGTGCCCTCTTGTReverse - TCTTGGGCTCCATCATTCATCC</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LINC01433</i> Primer:Forward - TTTGGAGCTTCCCTGCAGACReverse - CCGCTAGTGCATGACAGTGA</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>YTHDF3-AS1</i> Primer:Forward - CCAAACCCAGTTCAGCCGTAReverse - TCAGAGCGGAGTCAAAAGCA</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>AC090515.2</i> Primer:Forward - TGGTGAGACATTCGCCCTACReverse - CCACTCCATGCAAAGTGAAGC</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>CASC2</i> Primer:Forward - AGCCATCTACGGTAACCCAGTAGReverse - AAACCCACATCTGTCCCAAATT</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>PROSER2-AS1 Primer:Forward - GCCACAGCCATTGTCTTCATCReverse - ACCAGTGATGCCTCCCGTT</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>APPLE</i> Primer:Forward - CGGTTGACCTGAGCCTACTTCReverse - CTGACTAGGCGACCGGCA</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>AL713998.1</i> Primer:Forward - TCTTCTTGGGGAGACAGCCTReverse - GGTTGTGTGGCTGGCAAATG</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>AL356356.1</i> Primer:Forward - CCAGCTCTTGCCAGCTATGTReverse - GCAGATGAGCAAACTGCACC</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LINC00649</i> Primer:Forward - TCTCTGGCACGAAAGTGACCReverse - AATAGATGCAAGCCGGGACT</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>SATB2-AS2</i> Primer:Forward - GACCGGTAACACCAAGAGCCReverse - GTCTCTCTCTGCGGCTTGTC</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>AC091057.1</i> Primer:Forward - AGACTTTGTTCGGTCCACGGReverse - TCTGCTCCCGGTAAAGATGC</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LINC00106</i> Primer:Forward - AGGACACCGTCTGTCTTACGReverse - CCAGTGGTCACCTGAGATGG</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>LINC01215</i> Primer:Forward - CCCCATGTAGTCAGCTCGTTReverse - GGAAATGCAACTGGACCCAC</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>hY1</i> Primer:Forward - GGCTGGTCCGAAGGTAGTGAReverse - GCAGTAGTGAGAAGGGGGGA</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>GAPDH</i> Primer:Forward - AGGTGAAGGTCGGAGTCAACReverse - AGTTGAGGTCAATGAAGGGG</td>
<td>Ruibiotech</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Triax™ FlowCell software</td>
<td>Biocomp Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>GraphPad Prism 8</td>
<td>GraphPad Software</td>
<td><a href="https://www.graphpad.com/scientific-%20software/prism/">https://www.graphpad.com/scientific- software/prism/</a></td>
</tr>
<tr>
<td>QuantStudio Software v1.3</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Excel</td>
<td>Microsoft</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Minisart® Syringe Filters</td>
<td>Sartorius</td>
<td>Cat# 16533</td>
</tr>
<tr>
<td>Syringe</td>
<td>Jumin Bio-technologies</td>
<td>Cat# 1.2x35TWLB</td>
</tr>
<tr>
<td>Optima L-100 XP Ultracentrifuge</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>SW 41 Ti Swinging-Bucket Rotor</td>
<td>Beckman Coulter</td>
<td>Cat# 331362</td>
</tr>
<tr>
<td>Open-Top Thinwall Ultra-Clear Tube</td>
<td>Beckman Coulter</td>
<td>Cat# 344059</td>
</tr>
<tr>
<td>Gradient Master<sup>TM</sup>
</td>
<td>Biocomp Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Piston Gradient Fractionator™</td>
<td>Biocomp Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Triax<sup>TM</sup> Flow Cell</td>
<td>Biocomp Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Fraction Collector</td>
<td>Biocomp Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>QuantStudio 6 Flex Real-Time PCR System</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>MicroAmp™ Optical Adhesive Film</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 4311971</td>
</tr>
<tr>
<td>MicroAmp™ Fast Optical 96-Well Reaction Plate with Barcode, 0.1 mL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 4346906</td>
</tr>
<tr>
<td>NanoDrop 2000/2000c Spectrophotometer</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Centrifuge 5424 R</td>
<td>Eppendorf</td>
<td>N/A</td>
</tr>
<tr>
<td>Centrifuge 5810 R</td>
<td>Eppendorf</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL Centrifuge Tubes</td>
<td>Corning</td>
<td>Cat# 430829</td>
</tr>
<tr>
<td>Cell culture flask, 75 cm<sup>2</sup>
</td>
<td>Corning</td>
<td>Cat# 430641</td>
</tr>
<tr>
<td>Axygen® 1.5 mL MaxyClear Snaplock Microcentrifuge Tube</td>
<td>Corning</td>
<td>Cat# MCT-150-C</td>
</tr>
<tr>
<td>Micro Centrifuge Tubes, 2 mL</td>
<td>Jet Bio-Filtration</td>
<td>Cat# CFT001020</td>
</tr>
<tr>
<td>0.2 mL Flat PCR Tube 8-Cap Strips, optical, ultraclear</td>
<td>Bio-Rad Laboratories</td>
<td>Cat# TCS0803</td>
</tr>
<tr>
<td>Cellometer Auto T4 Bright Field Cell Counter</td>
<td>Nexcelom Bioscience</td>
<td>N/A</td>
</tr>
<tr>
<td>SD100 Slides</td>
<td>Nexcelom Bioscience</td>
<td>Cat# CHT4-SD100-002</td>
</tr>
<tr>
<td>CO<sub>2</sub> Incubator</td>
<td>Panasonic</td>
<td>Cat# MCO-170AICUVL-PC</td>
</tr>
<tr>
<td>Esco Biological Safety Cabinets</td>
<td>ESCO Technologies</td>
<td>N/A</td>
</tr>
<tr>
<td>Vortex Mixer V6</td>
<td>ESSEN Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>SevenCompact pH meter S220</td>
<td>METT.LER TOLEDO</td>
<td>Cat# 30019028</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>1 M</td>
<td>6.057 g / 50 mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>4 M</td>
<td>11.688 g / 50 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>1 M</td>
<td>4.761 g / 50 mL</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>1 M</td>
<td>1.55 g / 10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">All the stock solutions can be stored at 4°C up to one year.</p>
<table id="undtbl2">
<caption>Lysis buffer (pH 7.4)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (1 M)</td>
<td>25 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>NaCl (4 M)</td>
<td>100 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>IGEPAL</td>
<td>1 %</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>1 %</td>
<td>0.5 g</td>
</tr>
<tr>
<td>DEPC H<sub>2</sub>O</td>
<td>n/a</td>
<td>up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">This buffer can be stored at 4°C up to one year.<br> The 1 mM DTT, 100 μg/mL cycloheximide, 40 U/mL RNase inhibitor and 1 × protease inhibitor should be added before use. (<a href="#sec7.1">Troubleshooting 1</a>)</p>
<div class="critical">
<span class="critical-title">Critical:</span> HCl is highly volatile and is a high-risk inhalation hazard. It is toxic and causes severe skin burns and serious eye damage. Opening a concentrated HCl (36%) should be done in a fume hood. Wear personal protective equipment.</div>
<table id="undtbl3">
<caption>Sucrose buffer (pH 7.4)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (1 M)</td>
<td>25 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>NaCl (4 M)</td>
<td>100 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>DEPC H<sub>2</sub>O</td>
<td>n/a</td>
<td>up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">This buffer can be stored at 4°C up to one year.<br> The 100 μg/mL cycloheximide, 40 U/mL RNase inhibitor and 1 mM PMSF should be added before use.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Whole cell lysate preparation – day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–1.5 h</div>
<p>Cells are harvested and lysed for subsequent ultracentrifugation on sucrose cushion.</p>
<ol>
<li>Grow MV4-11 cells with IMDM modified medium (10% fetal bovine serum) in the culture flask at 37°C and 5% CO<sub>2</sub> to the desired number. Start culture at 1∗10<sup>5</sup>–2∗10<sup>5</sup>cells/mL and maintain between 1∗10<sup>5</sup> and 1∗10<sup>6</sup> cells/mL.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If necessary, diluting the cells or passage the cells to a new flask to keep the cells on log phase at the day before starting the experiment. The polysome peak of over confluent cells that are less proliferative will not appear as robust because proliferation and translation rate are closely correlated (<a href="#bib4">Johnson et al., 1976</a>).</div>
<ol start="2">
<li>Treat the cells with 100 μg/mL cycloheximide. After gentle shaking to mix thoroughly, incubate the cells in the culture flask at 37°C and 5% CO<sub>2</sub> for 15 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For suspension cells, it is recommended to count the desired number of cells before cycloheximide treatment. Too fewer cells may produce small peaks and weak signals below the detection threshold.</div>
<div class="note">
<span class="note-title">Note:</span> Cycloheximide is a translation elongation inhibitor and used to prevent translating RNA from running off (<a href="#bib6">Schneider-Poetsch et al., 2010</a>). The cycloheximide treatment duration should be optimized for different cell lines. Preliminary experiments are needed to screen for the optimal cycloheximide concentration to generate robust polysome peaks.</div>
<ol start="3">
<li>Harvest and wash cells with ice-cold PBS supplemented with 100 μg/mL cycloheximide.
<ol type="a">
<li>Collect cells in the culture with a 50 mL falcon tube and centrifuge the sample at 500 × <i>g</i> for 3 min at 4°C.</li>
<li>Discard the supernatant and wash the cells with ice-cold PBS supplemented with 100 μg/mL cycloheximide.</li>
<li>Centrifuge the cells at 500 × <i>g</i> for 3 min at 4°C. Repeat the wash step one more time.</li>
<li>Centrifuge the cells at 500 × <i>g</i> for 3 min at 4°C. Remove the supernatant as much as possible with precaution. Aspiration with a vacuum pump is preferred.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep the samples and solution on ice all the time. Make sure that all the items and reagents are ready to use so that the experiment can proceed quickly without haste. Leaving cells on ice for too long will result in suboptimal quality of polysome preparation.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Cell pellets may be flash frozen in the liquid nitrogen and stored at −80°C until use. However, it is always recommended to use fresh cells for the subsequent analysis.</div>
<ol start="4">
<li>Suspend the cell pellets by gently pipetting in 200–300 μL lysis buffer supplemented with 40 U/mL RNase inhibitor, 1 mM DTT, 1 mM PMSF, and 100 μg/mL cycloheximide. Immediately place the sample on ice for 30 min with occasional inverting to facilitate cell lysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of lysis buffer used is adjusted according to the cell number. Prepare excessive amount of fresh lysis buffer in advance and settle the buffer in the refrigerator at 4°C for a period of time to reduce bubbles caused by detergent.</div>
<ol start="5">
<li>Centrifuge the cell lysate at 13,000 × <i>g</i> for 10 min at 4°C. Transfer the supernatant to a new pre-chilled 1.5 mL tube. Place the tube on ice until use.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Alternatively, the cell lysate may be flash frozen in liquid nitrogen and stored at −80°C for a month. However, fresh cell lysates are always recommended.</div>
<div class="note">
<span class="note-title">Note:</span> Avoid the contamination by precipitations at the bottom when transferring supernatant. It is recommended to immediately load the lysis on sucrose cushion as this will improve the quality of the preparation.</div>
<h3 id="sec3.2">Sucrose cushion preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–20 min</div>
<p>Prepare sucrose gradient for density gradient centrifugation.</p>
<div class="note">
<span class="note-title">Note:</span> Prepare the sucrose gradient while the cell sample is lysed on ice as described above.</div>
<ol start="6">
<li>Mark the half-full position with the marker casing tube provided by the gradient maker (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. 10%–45% sucrose gradient loading</div>
<p>(A) Use the tube holder to mark the half-full point as shown by the black marker.</p>
<p>(B) Add 10% sucrose buffer by a 1 mL pipette.</p>
<p>(C) Add 45% sucrose buffer a long blunt needle.</p>
</figcaption>
</figure>
</li>
<li>Add 10% sucrose buffer first with 1 mL pipette until it reaches the half-full position (<a href="#fig2">Figure 2</a>B).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Touch the tip to the upper side wall of the tube and dispensing the buffer along the wall at a slow constant rate. Be careful to not generate bubbles that may interfere with following gradient making and polysome sedimentation.</div>
<ol start="8">
<li>Use long blunt needle to reach the bottom of the ultracentrifuge tube and layer 45% sucrose buffer until the upper 10% sucrose buffer reaches the top of the tube (<a href="#fig2">Figure 2</a>C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This process should be slow and gentle at a constant rate to avoid disturbing the 10% sucrose buffer layer.</div>
<ol start="9">
<li>Lid the tube with the rate zonal cap provided by the gradient maker.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When lidding the tube, contact the sucrose buffer with one side of the cap at first, and then slowly drop the other side that has the air hole into the tube to avoid making bubbles. The air hole is used to vent the bubbles (<a href="#fig3">Figure 3</a>).</div>
<ol start="10">
<li>Put the tube into the magnetic tube holder on the gradient maker.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure the plate that the magnetic tube holder is attached on is level (<a href="#fig4">Figures 4</a>A and 4B).</div>
<ol start="11">
<li>Run the sucrose gradient maker by selecting “10%–45%”. It takes about 1 min and 30 s to finish (<a href="#fig4">Figure 4</a>C). The sucrose gradient is ready to use immediately.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Alternatively, the prepared sucrose cushion could be stored at −80°C for up to 6 months until use.</div>
<div class="note">
<span class="note-title">Note:</span> Slide the magnetic tube holder out slowly and gently upon the ending of gradient making. Take extra precautions when moving the tube to avoid disturbing the prepared sucrose gradient.</div>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Secure the tube with rate zonal cap</div>
<p>(A) The opposite side of air hole contacts the sucrose cushion first.</p>
<p>(B) Drop the cap slowly and gently.</p>
<p>(C) Lid the tube without making bubbles.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. 10%–45% sucrose gradient preparation by gradient maker</div>
<p>(A) Check for the levelness of the magnetic plate.</p>
<p>(B) Adjust levelness using the button indicated by red arrow if the plate is unlevel.</p>
<p>(C) Select a gradient program and run it. In this example, 10%–45% gradient is used.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Polysome sedimentation by ultracentrifugation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Ultracentrifugation to isolate 40S subunit, 60S subunit, 80S monosome, and polysome-bound RNA with different densities.</p>
<ol start="12">
<li>Lift up the cap of the tube and layer the cell lysate onto the sucrose cushion. The A260 and A280 of MV4-11 cell lysate is approximately 65 and 35 respectively as measured by the NanoDrop 2000. If applicable, load lysis buffer onto the sucrose cushion of blank sample group.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a 200 μL pipette to layer the cell sample onto the sucrose cushion. Touch the pipette tip to the upper side of the tube and dispense the cell lysate slowly and gently to avoid disturbing the sucrose gradient (<a href="#fig5">Figure 5</a>).</div>
<ol start="13">
<li>Place the sucrose cushion tube loaded with samples into the SW41 rotor bucket.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Loading cell lysate onto sucrose cushion</div>
<p>(A) Use a 200 μL to layer the cell lysate without disturbing the sucrose gradient.</p>
<p>(B) The cell lysate loaded onto the sucrose gradient as indicated by the red arrow.</p>
</figcaption>
</figure>
</li>
<li>Use an analytical scale to balance the opposing rotor buckets.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Remember to put the cap together with the bucket on the analytical scale. If unbalanced, load lysis buffer onto the cushion to balance the buckets. It is acceptable that the weight differences between opposing buckets are under 0.003 <i>g</i>.</div>
<ol start="15">
<li>Once the buckets are balanced, screw the cap into the bucket and hook the buckets onto the rotor hanger. The buckets must be arranged symmetrically and all buckets, whether loaded or empty, must be hooked to rotor.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Screw the cap anticlockwise at first until you hear the click sound that indicates the cap fitted into the thread. Then screw the cap in the clockwise direction until there is metal-to metal contact.</div>
<ol start="16">
<li>Carefully lift the rotor with steady hands and install it on the drive hub of the ultracentrifuge perpendicularly.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> You can check for the proper installation by manually turning the rotor clockwise and observe the speed reading on the screen.</div>
<ol start="17">
<li>Centrifuge at 36,000 rpm (222,227 × <i>g</i>) for 3 h at 4°C with SW41 Ti rotor.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not choose max deceleration for ultracentrifugation. Choose speed “5” instead for optimal polysome sedimentation.</div>
<div class="note">
<span class="note-title">Note:</span> Check for the proper functioning of the ultracentrifuge occasionally to avoid the occurrence of errors.</div>
<h3 id="sec3.4">Polysome fractionation</h3>
<p>Collect each ribosome-bound RNA fractions for subsequent analysis.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="18">
<li>Turn on the fractionation system 30 min ahead of the ending of ultracentrifugation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Switch on pump, UV detector, fractionation collector, and computer. Press “SCAN” on the control panel first, otherwise the computer may not be successfully connected (<a href="#fig6">Figures 6</a>A and 6B).</div>
<div class="note">
<span class="note-title">Note:</span> Check whether the fractionation collector is at the initial position; if not, press “END” on the control panel of the fractionation station (<a href="#fig6">Figure 6</a>C).</div>
<ol start="19">
<li>Open the software Triax™ FlowCell on the computer and set the parameters for fractionation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The parameters set for the fractionation station is shown in <a href="#fig7">Figure 7</a>.</div>
<ol start="20">
<li>Fill the syringe attached to the fractionation station with DEPC-treated water to extensively rinse the system.</li>
<li>Press “air” on the control panel to remove any residual water. Repeat the rinse and air operations for two more times.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> A text run with DEPC-treated water is recommended to check whether the system works properly.</div>
<ol start="22">
<li>Prepare 2 mL tube for collecting. Place them into tube rack on the fractionation collector.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The tubes are placed into the rack to match the order of the sequentially collected fractions (<a href="#fig8">Figure 8</a>).</div>
<ol start="23">
<li>Once the ultracentrifugation for polysome sedimentation ends, take out the rotor and place it onto the adapter.</li>
<li>Use a screwdriver to remove the cap of the bucket, and take out the centrifuge tube with tweezers.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Take extra care to ensure not to disturb the gradient at this stage. Be careful to not make the bucket sway on the hanger when taking it out from the ultracentrifuge.</div>
<ol start="25">
<li>Put the cap of the SW41 tube holder onto the centrifuge tube of cell sample (<a href="#fig9">Figures 9</a>A–9C).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. The fractionation station</div>
<p>(A) Overview of the fractionation station. It includes the piston tip and UV detector to collect and detect sucrose gradients (upper), the control panel (lower), and the fractionation collector to dispense each fraction into the tubes on the rack (left upper).</p>
<p>(B) The close view of control panel. Red arrows indicate the controls for Air and Rinse.</p>
<p>(C) The panel for fractionation collector. The red arrow indicates the “END” button. Press “END” could reset the fractionation collector to the initial position.</p>
</figcaption>
</figure>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. The parameters set in the software Triax™ FlowCell for the fractionation station</div>
<p>(A) The parameters for UV detector and sucrose gradients.</p>
<p>(B) The parameters for fractionation collector.</p>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. The order for the arrangement of collecting tubes</div>
</figcaption>
</figure>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Load the sample tube with the tube holder for collection</div>
<p>(A–C) Fit the cap of the tube holder onto the sample tube.</p>
<p>(D–F) Install the loaded tube holder onto the fractionation station.</p>
</figcaption>
</figure>
</li>
<li>Place the capped centrifuge tube into the SW41 tube holder provided by the fractionation station. Gently screw the cap clockwise to secure the centrifuge tube (<a href="#fig9">Figure 9</a>D).</li>
<li>Transfer the loaded tube holder to let it slide into the fractionation station and turn it clockwise to be ready for collection (<a href="#fig9">Figures 9</a>E and 9F).</li>
<li>Start the run by clicking the “START SCAN” in the software program. This will start the collection of each fraction by a piston tip from top to bottom, and at the same time the UV absorbance at 260 nm is recorded to plot the tracing. Make sure that every drop of each fraction is collected into its corresponding 2 mL tube (<a href="#sec7.3">Troubleshooting 2</a>, <a href="#sec7.5">3</a>, and <a href="#sec7.7">4</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not turn on the light from the piston tip for a prolonged period of time. The heat generated from the light may cause RNA degradation in the sucrose gradient.</div>
<div class="note">
<span class="note-title">Note:</span> Press “Air” when the run ends to blow the residual drops out into the last tube for collection.</div>
<ol start="29">
<li>Transfer the 2 mL tube on the ice and add equal volume RNAiso Plus solution immediately. Vigorously vortex each tube to mix thoroughly.</li>
<li>Flash freeze each fraction with RNAiso Plus solution in liquid nitrogen and store them at −80°C until use.</li>
</ol>
<p>Caution: RNAiso Plus solution is toxic and harmful for the researcher and surrounding environment. Precautionary measures should be taken, including working in a fume hood, wearing disposable latex gloves, coat, and goggles.</p>
<div class="note">
<span class="note-title">Note:</span> Clearly mark the fraction number on the tube. Be careful not to spill RNAiso Plus solution which may erase the maker.</div>
<ol start="31">
<li>Rinse and air the fractionation system three times before another run. After all the runs finish, launch the last run with DEPC-treated water and extensively rinse the system to remove any residual sucrose.</li>
</ol>
<h3 id="sec3.5">RNA extraction-day 2</h3>
<p>This step can isolate RNA from sucrose buffer by RNAiso Plus solution.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h</div>
<ol start="32">
<li>Take samples from −80°C and quickly thaw them at room temperature (20°C–25°C).</li>
<li>Add 0.2 mL of chloroform per 1 mL of RNAiso Plus solution.</li>
<li>Vortex vigorously for 30 s to mix the chloroform thoroughly.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> RNAiso Plus solution is caustic so that precautionary measures should be taken, including the wearing of eye protection. The tube cap should be tightly closed, and if necessary, use cap locks.</div>
<ol start="35">
<li>Keep the samples at room temperature (20°C–25°C) for 5 min.</li>
<li>Centrifuge the samples at 12,000 × <i>g</i> for 15 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The centrifuge should be pre-cooled to 4°C before use.</div>
<div class="note">
<span class="note-title">Note:</span> The samples are separated into three layers: the bottom layer is a red organic phase containing protein; the middle layer is a white semi-solid containing DNA; the top layer is a colorless aqueous phase containing RNA.</div>
<ol start="37">
<li>Transfer the top layer into new RNase-free 1.5 mL tubes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Be careful not to touch the middle layer as it could lead to DNA contamination.</div>
<div class="note">
<span class="note-title">Note:</span> Two or more tubes should be used for each fraction to make sure that the volume of the aqueous phase is not over than 0.75 mL to allow the addition of isopropanol.</div>
<ol start="38">
<li>Add equal volume of ice-cold isopropanol and 1 μL GlycoBlue™ Coprecipitant. Mix thoroughly.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is recommended to add carrier because the RNA recovered from polysome fractions may not be easily visible. GlycoBlue™ Coprecipitant is glycogen linked with a blue dye. It facilitates nucleic acid precipitation and increases the size and visibility of the pellet without interfering subsequent analysis.</div>
<ol start="39">
<li>Keep the mixture at room temperature (20°C–25°C) for 10 min to precipitate.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Keep the mixture at −20°C for at least 1 hour to increase RNA yield.</div>
<ol start="40">
<li>Centrifuge at 12,000 × <i>g</i> for 10 min at 4°C.</li>
<li>Discard the supernatant carefully without touching the pellet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Centrifuge shortly and remove the remaining isopropanol completely.</div>
<ol start="42">
<li>Add 1 mL ice-cold 75% ethanol to wash the RNA pellet.</li>
<li>Vortex briefly and centrifuge at 7,500 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not disperse the pellet during vortexing.</div>
<ol start="44">
<li>Discard the supernatant carefully and add 1 mL ice-cold 100% ethanol to wash the RNA pellet.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The RNA can be stored in 100% ethanol at −20°C for several months.</div>
<ol start="45">
<li>Vortex briefly and centrifuge at 7,500 × <i>g</i> for 5 min at 4°C.</li>
<li>Centrifuge shortly and remove the remaining ethanol completely.</li>
<li>Air dry the pellet for 5 min.</li>
<li>Dissolve the pellet with 10–30 μL RNase-free water.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The final volume of the RNA solution should be identical for each fraction of polysome profiling.</div>
<div class="note">
<span class="note-title">Note:</span> The RNA solution should be kept on ice for immediate subsequent analyses to prevent degradation.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Alternative, the RNA solution could be stored at −80°C. RNA is generally stable at −80°C for up to a year. Use aliquots to avoid freeze-thaw cycles.</div>
<ol start="49">
<li>Measure the RNA concentrations by NanoDrop™ 2000 spectrophotometer (<a href="#sec7.7">Troubleshooting 5</a>).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The DNase I treatment should be applied if a genomic DNA elimination reaction is not included in the subsequent reverse-transcription step.</div>
<h3 id="sec3.6">RNA detection by quantitative reverse transcription PCR (RT-qPCR)</h3>
<p>RNA quantifications for each fraction to map the distribution of potentially translating lncRNAs in polysome profiles.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h (depending on number of samples)</div>
<div class="note">
<span class="note-title">Note:</span> RNAs are reversely transcribed to cDNA. The synthesized cDNA could serve as template for quantification of lncRNAs of interest by real-time PCR. The following reverse-transcription steps are performed using PrimeScript™ RT Reagent Kit with gDNA Eraser.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other kits or methods could also be used for reverse transcription.</div>
<ol start="50">
<li>Prepare a genomic DNA elimination reaction mix as described in the following table.</li>
</ol>
<p>Dispense equal volume of RNA samples from different fractions into each reaction.</p>
<table id="undtbl4">
<thead>
<tr>
<th colspan="2">gDNA elimination reaction</th>
</tr>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>5<b>×</b> gDNA Eraser Buffer</td>
<td>2 μL</td>
</tr>
<tr>
<td>gDNA Eraser</td>
<td>1 μL</td>
</tr>
<tr>
<td>RNA</td>
<td>up to 1 μg</td>
</tr>
<tr>
<td>RNase-Free dH<sub>2</sub>O</td>
<td>to 10 μL</td>
</tr>
<tr>
<td>Total</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> All reaction components should be added on ice. A mix except for RNA samples is recommended to save time and should be prepared with at least one extra reaction.</div>
<div class="note">
<span class="note-title">Note:</span> To calculate the percent RNA of each fraction, the equal volume of RNA solution should be used for each fraction of polysome profiling.</div>
<ol start="51">
<li>Gently mix the reagent and quickly spin the samples to collect all liquid to the bottom of tubes.</li>
<li>Incubate the samples at 42°C for 2 min and place them at 4°C until use.</li>
<li>Prepare a reverse-transcription reaction mix shown below. Add 10 μL of mix to each reaction solution from last step.</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th colspan="2">Reverse-transcription reaction</th>
</tr>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Reaction solution from step 52</td>
<td>10 μL</td>
</tr>
<tr>
<td>5<b>×</b> PrimeScript Buffer 2</td>
<td>4 μL</td>
</tr>
<tr>
<td>PrimeScript RT EnzymeMix I</td>
<td>1 μL</td>
</tr>
<tr>
<td>RT Primer Mix</td>
<td>1 μL</td>
</tr>
<tr>
<td>RNase-Free dH<sub>2</sub>O</td>
<td>40 μL</td>
</tr>
<tr>
<td>Total</td>
<td>20 μL</td>
</tr>
</tbody>
</table>
<ol start="54">
<li>Gently mix the reagent and quickly spin the samples to collect all liquid to the bottom of tubes.</li>
<li>Incubate the samples at 37°C for 15 min, 85°C for 5 s and place them at 4°C until use.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The cDNAs could be stored at −20°C for long term.</div>
<ol start="56">
<li>The cDNAs should be diluted at least two times with sterilized double-distilled water before use.</li>
<li>Prepare the real-time PCR reaction mix except for cDNAs as described in the following table.</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th colspan="2">Real-time PCR</th>
</tr>
<tr>
<th>Reagent</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>TB Green <i>Premix Ex</i><i>Taq</i> II (Tli RNaseH Plus) (2<b>×</b>)</td>
<td>5 μL</td>
</tr>
<tr>
<td>PCR Forward Primer (10 μM)</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>PCR Reverse Primer (10 μM)</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>ROX Reference Dye or Dye II (50<b>×</b>)</td>
<td>0.2 μL</td>
</tr>
<tr>
<td>RT reaction solution (cDNA) from step 56</td>
<td>2 μL</td>
</tr>
<tr>
<td>Sterile purified water</td>
<td>2.4 μL</td>
</tr>
<tr>
<td>Total</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> All reaction components should be added on ice.</div>
<ol start="58">
<li>Mix the reaction solution and quickly spin the samples.</li>
<li>Transfer 8 μL of the mix to 96/384-well plate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The 384-well plate is recommended to detect several lncRNAs for the same batch of fractions on the same plate.</div>
<ol start="60">
<li>Add 2 μL of the diluted cDNA to the corresponding wells.</li>
<li>Seal the 96/384-well plate by an optical adhesive film.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure that the wells were completely sealed to avoid sample evaporation during PCR reaction.</div>
<ol start="62">
<li>Centrifuge the plate at 1,500 × <i>g</i> for 2 min at 4°C to collect all liquid to the bottom of the plate and to eliminate bubbles.</li>
<li>Run the real-time PCR reaction with the appropriate program. An example is shown in the following table.</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>95°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td rowspan="2">PCR reaction</td>
<td>95°C</td>
<td>5 s</td>
<td rowspan="2">40 cycles</td>
</tr>
<tr>
<td>60°C</td>
<td>30 s</td>
</tr>
</tbody>
</table>
<ol start="64">
<li>The percent lncRNA of interest in each fraction is calculated by RT-qPCR data as described in <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a> section.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>A successful polysome profiling should show clear 40S subunit, 60S subunit, 80S monosome, and several polysome peaks. The polysome peaks represent the RNAs bound to increasing number of ribosomes (&gt;=2). The robustness of polysome peaks is dependent on the proliferation state of the cell samples and varies between different cell lines. Generally, at least three polysome peaks are observed for a good polysome profile.</p>
<p>It is important to include both positive and negative controls when screening for potentially translating lncRNAs by RT-qPCR following RNA extractions of each fraction. The mRNA of housekeeping genes, such as GAPDH, could serve as the positive control; the known cytoplasmic non-coding RNAs, such as hY1, could serve as the negative control. The peak of positive control, GAPDH, should be in the high polysome fraction, while the negative control, hY1, should be enriched in the submonosome fractions. The candidate lncRNAs that have prominent peaks in the polysome fractions may indicate activate translation in cells. An example is shown for polysome profiling in MV4-11 cells and RT-qPCR detection of the candidate lncRNA distributions in each fraction (<a href="#fig10">Figure 10</a>), as we previously described (<a href="#bib7">Sun et al., 2021</a>).</p>
<figure id="fig10"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1284-Fig10.jpg" alt="Fig10.jpg">
<figcaption>
<div class="figcaption-title">Figure 10. An example of polysome profiling followed by lncRNA distribution calculations</div>
<p>(A) Polysome profile of MV4-11 cells.</p>
<p>(B) Distributions of lncRNAs of interest. Protein-coding mRNA GAPDH is used as a positive control. Non-coding RNA hY1 is used as a negative control.</p>
<p>Figure reprinted with permission from <a href="#bib7">Sun et al. (2021)</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The calculations of the percent RNA in each fraction are as follows:</p>
<p class="math-ml"><math> <mtext>Δ</mtext> <msub> <mtext>C</mtext> <mtext>X</mtext> </msub> <mo>=</mo> <msub> <mtext>CT</mtext> <mtext>X</mtext> </msub> <mo>-</mo> <msub> <mtext>CT</mtext> <mn>1</mn> </msub> </math></p>
<p class="math-ml"><math> <mrow> <mtext>% RNA</mtext> <mo>=</mo> <mfrac> <msup> <mn>2</mn> <mrow> <mo>-</mo> <mtext>Δ</mtext> <msub> <mtext>CT</mtext> <mtext>X</mtext> </msub> </mrow> </msup> <mrow> <msup> <mn>2</mn> <mrow> <mo>-</mo> <mtext>Δ</mtext> <msub> <mtext>CT</mtext> <mn>1</mn> </msub> </mrow> </msup> <mo>+</mo> <msup> <mn>2</mn> <mrow> <mo>-</mo> <mtext>Δ</mtext> <msub> <mtext>CT</mtext> <mn>2</mn> </msub> </mrow> </msup> <mo>+</mo> <mtext>...</mtext> <mo>+</mo> <msup> <mn>2</mn> <mrow> <mo>-</mo> <mtext>Δ</mtext> <msub> <mtext>CT</mtext> <mtext>Y</mtext> </msub> </mrow> </msup> </mrow> </mfrac> <mo>∗</mo> <mn>100</mn> </mrow> </math></p>
<p>X = the number of the fraction that is calculated; Y= the total number of fractions.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol requires a large number of cells to perform. Too few cells may result in low RNA yield and unreliable results. Therefore, this protocol may not apply to rare and precious cell samples.</p>
<p>The part of polysome profiling is tricky due to a couple of reasons as follows: 1. RNA prone to rapid degradation; 2. the requirement for preservation of both RNA and protein to detect the translation complex; 3. the need to keep the sucrose gradient undisturbed despite that the researcher has to move it, layer the lysate onto it, and collect fractions from it. Although manual collection of each fraction is feasible, it is uneasy to do it without disturbing the gradient. Therefore, it is recommended to use a fractionation station to collect each fraction, which may not be handy in some cases. It is also a prerequisite to carry on the experiment in an isolated and clean environment. The RNA and protein are prone to degradation in a crowded and contaminated workplace.</p>
<p>As no control could be used for controlling the potential differences in RNA extraction, it requires high demand for handling the samples of each fraction with identical operations. The misoperation for one sample may lead to incorrect results. The inclusion of positive control, such as coding-gene GAPDH, and negative control, such as non-coding hY1, is important to evaluate the robustness of the results.</p>
<p>The polysome profiling analysis described here is rapid and cost-effective. It allows the evaluation of overall translation status and the status of individual RNAs, which comes handy in screening for potential micropeptide-encoding lncRNAs. The major limit of this method is the lack of information regarding ribosome localization on lncRNA, therefore it is impossible to predict which potential small open reading frame in the lncRNA encodes micropeptide. In contrast, ribosome profiling followed by RNA sequencing (Ribo-seq) could determine the ribosome position at a single-nucleotide resolution. However, Ribo-seq is unable to determine the translation status of an lncRNA like polysome profiling analysis does. For example, in the scenario where two ribosomes show occupancy on two lncRNA As (A1 and A2) at position a and b; the Ribo-seq analysis cannot distinguish between that the two ribosomes occupy A1 or A2 at these two positions and that the two ribosomes occupy A1 and A2 at position a and b respectively. Furthermore, the information of regions that are unprotected by ribosomes is lost in Ribo-seq, which makes it impossible to distinguish the transcript variants that have different translation efficiencies (<a href="#bib3">Ingolia, 2014</a>). Therefore, the protocol described here and Ribo-seq are complementary methods to look for micropeptide-encoding lncRNAs. Moreover, polysome profiling could also combine with microarray and RNA-sequencing for high-throughput screening. In addition, ribosome affinity purification followed by RNA analysis is another method that can be used for characterizing ribosome-bound RNAs (<a href="#bib1">Halbeisen et al., 2009</a>; <a href="#bib2">Heiman et al., 2014</a>). This method could remove contaminations that might exist in sucrose gradients, such as lipid rafts, P-body components or pseudo-polysome (<a href="#bib1">Halbeisen et al., 2009</a>; <a href="#bib8">Thermann and Hentze, 2007</a>). Despite this advantage, it cannot determine the translation efficiency of lncRNAs except for their bindings to ribosome. In sum, these methods should be combined under different scenarios to screen for the potential translated lncRNAs.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The sodium deoxycholate could not be dissolved in the lysis buffer.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Reprepare the lysis buffer and adjust the addition order of sodium deoxycholate. Add all the reagents except sodium deoxycholate and adjust the pH under ∼7.8. Then add the sodium deoxycholate followed by vortexing for dissolution. Adjust the pH to the final 7.4.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>The fractionation system is clogged.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Sucrose solutions should be filtered through 0.22 μm filters to prevent clogging. Once clogged, immediately use a large amount of DEPC-treated water to extensively rinse and air the system until the water flow smoothly.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>No peaks or small peaks are detected and recorded by the fractionation system.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Increase the number of cells used. The RNA amount of each fraction yielded by too few cells may be under the threshhold of UV detector.</p>
<p>The RNA might be degraded due to RNase activity in reagents and environment. The dissolved cycloheximide solutions and buffers can be sterilized by filtering through 0.22 μm filters before adding RNase and protease inhibitors. Change the disposable latex gloves frequently, and apply RNase ZAP<sup>TM</sup> on it and other contacts. The material and equipment that make close contact with the sucrose gradient, such as the rate zonal cap for sucrose gradient making and the piston tip for collection, could also be cleaned by RNase ZAP<sup>TM</sup> followed by extensive washes with DEPC-treated water.</p>
<p>The sucrose may be disturbed. Remake the sucrose buffer and be careful not to make bubbles when layering the gradient. Keep steady hand when handling it and make moves with no rushes.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Non-robust or flat polysome peaks are observed.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Use fresh cycloheximide solution. It is better to use fresh-made or stored cycloheximide solution under one month in the freezer.</p>
<p>Try to adjust the MgCl<sub>2</sub> concentration in the lysis and sucrose buffer. Mg<sup>2+</sup> is important for complex formation and prevents polysomes from dissociating into subunits (<a href="#bib5">Klein et al., 2004</a>). Optimal Mg<sup>2+</sup> concentration may be beneficial for obtaining robust polysome peaks.</p>
<p>Adjust the cell numbers cultured in the flask or dishes to avoid over confluency so that cells are in a proliferative state.</p>
<p>Harvest cells as quickly as possible and cool them immediately on ice. Shorten the time interval between the completion of cell lysis and loading onto the sucrose cushion.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Low RNA yield from the extraction of each fraction.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Increase the number of cells used. The amount of RNA retrieved from some fractions might be relatively low, so a large number of cells are needed to obtain sufficient RNA.</p>
<p>Use ethanol precipitation instead of isopropanol to increase the amount of RNA recovered. Additionally, precipitate the RNA in -80°C overnight (14–16 h) to improve RNA yield.</p>
<p>Pool the corresponding fractions from parallel samples together to increase the final RNA amount. If doing so, it is important to make the gradients of parallel samples as identical as possible.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Halbeisen, R.E., Scherrer, T., and Gerber, A.P. (2009). Affinity purification of ribosomes to access the translatome. Methods <i>48</i>, 306-310. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref1">View at publisher</a></p>
<p id="bib2">Heiman, M., Kulicke, R., Fenster, R.J., Greengard, P., and Heintz, N. (2014). Cell type-specific mRNA purification by translating ribosome affinity purification (TRAP). Nat. Protoc. <i>9</i>, 1282-1291. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref2">View at publisher</a></p>
<p id="bib3">Ingolia, N.T. (2014). Ribosome profiling: new views of translation, from single codons to genome scale. Nat. Rev. Genet. <i>15</i>, 205-213. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref3">View at publisher</a></p>
<p id="bib4">Johnson, L.F., Levis, R., Abelson, H.T., Green, H., and Penman, S. (1976). Changes in RNA in relation to growth of the fibroblast. IV. Alterations in theproduction and processing of mRNA and rRNA in resting and growing cells. J. Cell Biol. <i>71</i>, 933-938. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref4">View at publisher</a></p>
<p id="bib5">Klein, D.J., Moore, P.B., and Steitz, T.A. (2004). The contribution of metal ions to the structural stability of the large ribosomal subunit. RNA <i>10</i>, 1366-1379. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref5">View at publisher</a></p>
<p id="bib6">Schneider-Poetsch, T., Ju, J., Eyler, D.E., Dang, Y., Bhat, S., Merrick, W.C., Green, R., Shen, B., and Liu, J.O. (2010). Inhibition of eukaryotic translation elongation by cycloheximide and lactimidomycin. Nat. Chem. Biol. <i>6</i>, 209-217. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref6">View at publisher</a></p>
<p id="bib7">Sun, L., Wang, W., Han, C., Huang, W., Sun, Y., Fang, K., Zeng, Z., Yang, Q., Pan, Q., Chen, T., et al. (2021). The oncomicropeptide APPLE promotes hematopoietic malignancy by enhancing translation initiation. Mol. Cell. <i>81</i>, P4493-P4508.E9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref7">View at publisher</a></p>
<p id="bib8">Thermann, R. and Hentze, M.W. (2007). Drosophila miR2 induces pseudo-polysomes and inhibits translation initiation. Nature <i>447</i>, 875-878. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00743-7/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yueqin Chen (<a href="mailto:lsscyq@mail.sysu.edu.cn">lsscyq@mail.sysu.edu.cn</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique materials.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any data or code</p>
<h3>Acknowledgments</h3>
<p>This research was supported by the <a href="https://doi.org/10.13039/501100003453">Natural Science Foundation of Guangdong Province</a> (2021B1515020002 and 2018A030310073), the <a href="https://doi.org/10.13039/501100002858">China Postdoctoral Science Foundation</a> (2021 T140771), and the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (81770174 and 31700719).</p>
<h3>Author contributions</h3>
<p>Conceptualization and Protocol Optimization, C.H. and L.S.; Investigation and Writing, C.H. and L.S.; Review and Editing, Q.P., Y.S., W.W., and Y.C.; Funding Acquisition, Y.S., W.W., and Y.C.; Supervision, Y.C.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>