<article>
<section>
<h1 id="header">Infecting human hematopoietic stem and progenitor cells with SARS-CoV-2</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Hector Huerga Encabo,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Rachel Ulferts,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor2">**</a></sup> Aneesh Sharma,<sup><a href="#aff1">1</a></sup> Rupert Beale,<sup><a href="#aff2">2</a></sup> and Dominique Bonnet<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>Hematopoietic Stem Cell Laboratory, The Francis Crick Institute, London, NW1 1AT, UK</p>
<p id="aff2"><sup>2</sup>Cell Biology of Infection Laboratory, The Francis Crick Institute, London NW1 1AT, UK</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:hector.encabo@crick.ac.uk">hector.encabo@crick.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:rachel.ulferts@crick.ac.uk">rachel.ulferts@crick.ac.uk</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:dominique.bonnet@crick.ac.uk">dominique.bonnet@crick.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100903">10.1016/j.xpro.2021.100903</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Determining how hematopoietic stem and progenitor cells (HSPCs) can be infected by viruses is necessary to understand and predict how the immune system will drive the host response. We present here a protocol to analyze the capacity of SARS-CoV-2 to infect different subsets of human HSPCs, inlcuding procedures for SARS-CoV-2 production and titration, isolation of human HSPCs from different sources (bone marrow, umbilical cord, or peripheral blood), and quantification of SARS-Cov-2 infection capacity by RT-qPCR and colony forming unit assay.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.02.001">Huerga Encabo et al. (2021)</a></p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Production and titration of SARS-CoV-2</li>
<li>Isolation of human hematopoietic stem and progenitor cells from different sources</li>
<li>Analysis of the infection capacity of SARS-CoV-2 in HSPCs</li>
<li>Despite their intrinsic antiviral protection HSPCs can be infected by SARS-CoV-2</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1168-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<div class="critical">
<span class="critical-title">Critical:</span> SARS-CoV-2 and any material that has been in contact with the virus should be handled and disposed of in accordance with your local regulations.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Institutional permission to manipulate human samples should be approved before starting the protocol.</div>
<p>Ensure you have availability of a suitable source of HSPCs (i.e., cord blood, bone marrow or peripheral blood). In the case of processing fresh samples you should be processing a sample that has been acquired within the last 24 h. In the case of processing cryopreserved mononuclear cell suspension you should proceed from step 53 for the isolation of hematopoietic stem and progenitor cells (HSPCs) populations.</p>
<p>The isolation of primary human stem and progenitor hematopoietic cells under sterile conditions has been optimized in this protocol using FACSAria III in a CL2 cabinet.A</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>CD34-PECy7 (dilution factor for staining 1/100)</td>
<td>eBioscience</td>
<td>clone 4H11</td>
</tr>
<tr>
<td>CD34- PerCP-Cy5.5 (dilution factor for staining 1/50)</td>
<td>BD Pharmingen</td>
<td>clone 8G12</td>
</tr>
<tr>
<td>CD38-PE (dilution factor for staining 1/50)</td>
<td>BD Pharmingen</td>
<td>clone HIT2</td>
</tr>
<tr>
<td>CD117-BB700 (dilution factor for staining 1/100)</td>
<td>BD Bioscience</td>
<td>clone YB5.B8</td>
</tr>
<tr>
<td>CD117-PECy7 (dilution factor for staining 1/100)</td>
<td>eBioscience</td>
<td>clone 104D2</td>
</tr>
<tr>
<td>CD71-APC (dilution factor for staining 1/100)</td>
<td>eBioscience</td>
<td>clone OKT9</td>
</tr>
<tr>
<td>CD235a-FITC (dilution factor for staining 1/100)</td>
<td>BD Pharmingen</td>
<td>clone HIR2</td>
</tr>
<tr>
<td>CD14-APCCy7 (part of Lineage exclusion gate) (1/100)</td>
<td>BioLegend</td>
<td>clone M5E2</td>
</tr>
<tr>
<td>CD16-APCCy7 (part of Lineage exclusion gate) (1/100)</td>
<td>BioLegend</td>
<td>clone B73.1</td>
</tr>
<tr>
<td>CD19-APCCy7 (part of Lineage exclusion gate) (1/100)</td>
<td>BioLegend</td>
<td>clone HIB19</td>
</tr>
<tr>
<td>CD3-APCCy7 (part of Lineage exclusion gate) (1/100)</td>
<td>BioLegend</td>
<td>clone SK7</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>SARS-CoV-2 isolate</td>
<td>(<a href="#bib1">Huerga Encabo et al., 2021</a>)(<a href="#bib1">Huerga Encabo et al., 2021</a>)</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human umbilical cord blood</td>
<td>Royal London Hospital (London, U.K.)</td>
<td>NA</td>
</tr>
<tr>
<td>Human peripheral blood from males and females between 20 and 50 years old</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>Human bone marrow aspirates from males and females between 30 and 40 years old</td>
<td>STEMCELL Technologies</td>
<td>Cat#70001</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Ficoll</td>
<td>GE Healthcare/Cytiva</td>
<td>Cat#17-1440-03</td>
</tr>
<tr>
<td>Ammonium Chloride (Red Cell Lysis Buffer)</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>Gibco</td>
<td>Cat#10270-106</td>
</tr>
<tr>
<td>Dimethyl Sulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>Cat#D4540</td>
</tr>
<tr>
<td>MethoCult H4435 (100 mL)</td>
<td>STEMCELL Technologies</td>
<td>Cat#04435</td>
</tr>
<tr>
<td>Stemspan SFEM II</td>
<td>STEMCELL Technologies</td>
<td>Cat#9605</td>
</tr>
<tr>
<td>Human TPO</td>
<td>PeproTech</td>
<td>Cat#300-18</td>
</tr>
<tr>
<td>Human SCF</td>
<td>PeproTech</td>
<td>Cat#300-07</td>
</tr>
<tr>
<td>Human Flt3-Ligand</td>
<td>PeproTech</td>
<td>Cat#300-19</td>
</tr>
<tr>
<td>Avicell, Pharma grade RC581</td>
<td>IMCD</td>
<td>N/A</td>
</tr>
<tr>
<td>DMEM, high glucose, with glutamine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11995065</td>
</tr>
<tr>
<td>FCS</td>
<td>Sigma</td>
<td>Cat#F7524</td>
</tr>
<tr>
<td>Formaldehyde solution for molecular biology, 36.5%–38% in H<sub>2</sub>O</td>
<td>Sigma</td>
<td>Cat#F8775</td>
</tr>
<tr>
<td>Triton-X100</td>
<td>Sigma</td>
<td>Cat#T9284</td>
</tr>
<tr>
<td>NaCl</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>KCl</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>Na2 EDTA</td>
<td>Francis Crick Institute</td>
<td>NA</td>
</tr>
<tr>
<td>HEPES, 1M Buffer Solution</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15630049</td>
</tr>
<tr>
<td>Trypsin-EDTA (0.5%)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15400054</td>
</tr>
<tr>
<td>100<b>×</b> Pen/strep (optional)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>Toluidine Blue O</td>
<td>Sigma</td>
<td>Cat#T3260</td>
</tr>
<tr>
<td>DAPI</td>
<td>BD Biosciences</td>
<td>Cat#564907</td>
</tr>
<tr>
<td>PowerUP SYBR Green</td>
<td>Applied Biosystems</td>
<td>Cat#15310939</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>EasySep Human Progenitor Cell Enrichment Kit II</td>
<td>STEMCELL Technologies</td>
<td>Cat#17936</td>
</tr>
<tr>
<td>RNeasy Microkit</td>
<td>QIAGEN</td>
<td>Cat#74004</td>
</tr>
<tr>
<td>SuperScript III First-Strand Synthesis</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#18080051</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Vero E6 cells [Vero C1008, Vero 76, clone E6, Vero E6]</td>
<td>ECACC ATCC</td>
<td>85020206 CRL-1586</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>ACE2_forward:5′-GCTGCACAACCTTTTCTGCT-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>ACE2_reverse:5′-AAATGCTTAGGTGTGGCTGC-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>TMPRSS2_forward: 5′-GTACCTGCATCAACCCCTCT-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>TMPRSS2_reverse: 5′-TATAGCCCATGTCCCTGCAG-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>SARS-CoV2_N_forward: 5′-CACATTGGCACCCGCAATC-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>SARS-CoV2_N_reverse:5′-GAGGAACGAGAAGAGGCTTG-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>SARS-CoV2_E_forward:5′-ACAGGTACGTTAATAGTTAATAGCGT-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>SARS-CoV2_E_reverse:5′-ATATTGCAGCAGTACGCACACA-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>SARS-CoV2_RdP_forward:5′-GTGARATGGTCATGTGTGGCGG-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>SARS-CoV2_RdP_reverse:5′-CARATGTTAAASACACTATTAGCATA-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>GAPDH_forward: 5′-GGAGCGAGATCCCTCCAAAAT-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td>GAPDH_reverse: 5′-GGCTGTTGTCATACTTCTCATGG-3′</td>
<td>Invitrogen</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>QuantStudio Real-Time PCR Software v1.7.1</td>
<td>Applied Biosystems</td>
<td>NA</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Falcon® 50 mL Tubes</td>
<td>Corning</td>
<td>Cat#352070</td>
</tr>
<tr>
<td>T175 Flask</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#159910</td>
</tr>
<tr>
<td>2 mL Syringe</td>
<td>BD (Emerald)</td>
<td>Cat#307727</td>
</tr>
<tr>
<td>Sterican MIX Blunt Fill Needle</td>
<td>B. Braun</td>
<td>Cat#4038088-01</td>
</tr>
<tr>
<td>TC-Treated 6, 12 or 48 well plate</td>
<td>Corning</td>
<td>Cat#CLS3516- Cat#CLS3513- Cat#CLS3548</td>
</tr>
<tr>
<td>Falcon® 5 mL Polypropylene Test Tubes</td>
<td>Corning</td>
<td>Cat#352063</td>
</tr>
<tr>
<td>0.2 mL PCR Tubes</td>
<td>Bio-Rad</td>
<td>Cat#TFI0201</td>
</tr>
<tr>
<td>Eppendorf Safe-Lock Tubes, 1.5 mL</td>
<td>Eppendorf</td>
<td>Cat#0030120086</td>
</tr>
<tr>
<td>EasySep Magnet</td>
<td>STEMCELL Technologies</td>
<td>Cat#18000</td>
</tr>
<tr>
<td>MicroAmp Optical 384-Well Reaction Plate</td>
<td>Applied Biosystems</td>
<td>Cat#4309849</td>
</tr>
<tr>
<td>FACSAria III</td>
<td>Becton Dickinson</td>
<td>NA</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Fixative solution/ 10% neutral buffered formalin</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Formaldehyde solution for molecular biology, 36.5–38% in H<sub>2</sub>O</td>
<td>∼3.6%</td>
<td>10 mL</td>
</tr>
<tr>
<td>PBS</td>
<td>n/a</td>
<td>90 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Make fresh on day of use and keep at room temperature (RT; 20°C–25°C)</p>
<table id="undtbl2">
<caption>Complete growth medium for Vero E6 cells/ D10</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>450 mL</td>
</tr>
<tr>
<td>FCS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, use within 4 weeks, bring to 37°C prior to use</p>
<table id="undtbl3">
<caption>Virus growth medium / D2</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>48 mL</td>
</tr>
<tr>
<td>FCS</td>
<td>2%</td>
<td>1 mL</td>
</tr>
<tr>
<td>1 M Hepes</td>
<td>20 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, use within 4 weeks, bring to 37°C prior to use</p>
<table id="undtbl4">
<caption>Plaque assay overlay, volume per 6-well plate</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Virus growth medium</td>
<td>0.5<b>×</b>
</td>
<td>6.5 mL</td>
</tr>
<tr>
<td>2.4% Avicel in water</td>
<td>1.2%</td>
<td>6.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>13 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C, use within 4 weeks, bring to 37°C prior to use</p>
<table id="undtbl5">
<caption>2.4% avicel in water</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Avicel</td>
<td>2.4<b>×</b>
</td>
<td>1.2 g</td>
</tr>
<tr>
<td>Water (MQ)</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Stir the water vigorously while slowly adding the avicel to avoid clumps from forming, sterilize by autoclaving on a liquid cycle, store at room temperature (RT; 20°C–25°C), use within 3 month, redistribute by swirling vigorously prior to use</p>
<table id="undtbl6">
<caption>1<b>×</b> PBS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td> </td>
<td>8 g</td>
</tr>
<tr>
<td>KCl</td>
<td> </td>
<td>0.25 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td> </td>
<td>1.437 g</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td> </td>
<td>0.25 g</td>
</tr>
<tr>
<td>Dist. Water (make up to)</td>
<td>n/a</td>
<td>Up to 1000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize by autoclaving on a liquid cycle, store at RT, use within 1 year</p>
<table id="undtbl7">
<caption>1<b>×</b> staining solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Toluidine blue</td>
<td>0.1% (w/v)</td>
<td>0.5 g</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>0.1% (v/v)</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Dist. Water (make up to)</td>
<td>n/a</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at RT for up to 1 year</p>
<table id="undtbl8">
<caption>Red Cell Lysis Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NH<sub>4</sub>Cl</td>
<td>0.15M</td>
<td>80.24 g</td>
</tr>
<tr>
<td>KHCO<sub>3</sub>
</td>
<td>10 mM</td>
<td>10.012 g</td>
</tr>
<tr>
<td>Na<sub>2</sub> EDTA</td>
<td>0.1 mM</td>
<td>0.372 g</td>
</tr>
<tr>
<td>Dist. Water (make up to)</td>
<td>n/a</td>
<td>Up to 10000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>10000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 year</p>
<table id="undtbl9">
<caption>PBS+2%FBS</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>FBS</td>
<td>2%</td>
<td>10 mL</td>
</tr>
<tr>
<td>1<b>×</b> PBS (make up to)</td>
<td>n/a</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C and use within 4 weeks</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of virus stock</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [4 days]</div>
<p>This step enables the propagation of a defined reference virus strain in a standard cell line, Vero E6. Infectivity of the resultant virus stock will then be determined in the subsequent section (beginning at step 19).</p>
<ol>
<li>For each 15 mL of virus stock to be produced prepare one 90% confluent 75 cm<sup>2</sup> flask of Vero E6 cells. Prepare one additional flask per experiment for the mock control.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Properties of cell lines can change during prolonged passaging. Prepare stocks of early passages of your cell line and store in liquid nitrogen. Regularly (∼20 passages, or if changes to morphology of the cell line or infectability are observed) restart your cell culture with an early passage vial.</div>
<ol start="2">
<li>Calculate the required volume of virus stock (inoculum) for a multiplicity of infection (MOI) of 0.01 PFU per cell.
<ol type="a">
<li>A 75 cm<sup>2</sup> flask of Vero E6 cells will contain approximately 10<sup>7</sup> cells.</li>
<li>The aim is to inoculate at a low multiplicity of infection (MOI, i.e., &lt;0.01 PFU per cell) to avoid the build-up of defective-interfering particles during sequential subculture. A lower MOI than 0.01 can be used as determined by the user of this protocol.</li>
</ol>
</li>
<li>Calculate the required volume of D2 medium (3 mL minus inoculum per 75 cm<sup>2</sup> flask) and add this to a 5 mL reagent tube.</li>
<li>Add the inoculum to the reagent tube from step 3.</li>
<li>Remove the supernatant from the flasks.</li>
<li>Add the diluted inoculum to all flasks to be inoculated.</li>
<li>Add 3 mL of D3 medium to the mock control flask.</li>
<li>Incubate 30 min inside the microbiolocial safety cabinet to allow the virus to attach. Redistribute the supernatant repeatedly to prevent the cells from drying out.</li>
<li>Remove the supernatant and dispose as infectious waste according to local regulations.</li>
<li>Rinse all flasks twice with 10 mL D2.</li>
<li>Add 15 mL D2 to each flask.
<ol type="a">
<li>Incubate at 37°C and 5% CO<sub>2</sub>.</li>
</ol>
</li>
<li>Starting from 2 days post infection (p.i.) observe the cell monolayers for signs of cytopathic effect (CPE) twice a day (morning and afternoon) using the inverted phase contrast microscope. The mock control flask serves as a comparison for an uninfected cell layer (see <a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1168-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Phase contrast images showing a representative example of cytophatic effect (full CPE) in Vero E6 cells compared to control (mock)</div>
<p>Scale bar 50 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> No CPE should be visible in the mock control flask. CPE in the control flask indicates poor sterile technique. This is particularly important if different strains or isolates are grown at the same time.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Harvesting before obvious cytopathic effects risks a very low titer. Harvesting long after peak production risks losing infectivity due to thermal instability of the virus.</div>
<ol start="13">
<li>Once fulminant CPE is observed, transfer the supernatant into a centrifuge tube.
<ol type="a">
<li>Dispose of the flasks and remaining cells in accordance with local safety regulations.</li>
</ol>
</li>
<li>Transfer the supernatant of the mock control flask into another centrifuge tube.
<ol type="a">
<li>Dispose of the flasks and remaining cells in accordance with local safety regulations.</li>
</ol>
</li>
<li>Pellet the cell debris by centrifugation at 500 <i>g</i> for 10 min at 4°C.</li>
<li>Transfer 200–1000 μL aliquots of the supernatant into screw cap tubes.</li>
<li>Transfer a 500 μL aliquot of the mock control supernatant into a screw cap tube.
<ol type="a">
<li>The rest of the mock control supernatant can be discarded.</li>
</ol>
</li>
<li>Store the aliquots at −80°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The infectivity of the stock decreases with freeze thawing. For maximum accuracy, store the aliquots overnight (16 h) or longer at −80°C before continuing with the next step. Adjust the volume of the aliquots to avoid subsequent freeze thaw cycles. 200 and 500 μl aliquots are good starting points.</div>
<h3 id="sec3.2">Determining the infectious particles by plaque assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> [4 days]</div>
<p>This step is used to quantify infectious SARS-CoV-2 particles per mL of virus stock. A 10-fold serial dilution of the original stock is added to confluent cell monolayers and overlaid with a viscous medium to restrict diffusion of progeny virions. This prevents convective spread of the virus, so each infectious particle generates a single plaque. Counting of the plaques at different dilutions can be used to estimate the number of infectious particles in the original sample.</p>
<ol start="19">
<li>Plate 5<b>×</b>10<sup>5</sup> Vero E6 cells in 2 mL complete growth medium per well of a six-well plate. Incubate overnight (16 h) at 37°C with 5% CO<sub>2</sub>. Prepare 1 plate per virus stock to be quantified and one plate for the mock control supernatant.
<ol type="a">
<li>Alternatively, plate 1.5<b>×</b>10<sup>5</sup> Vero E6 cells in 2 mL growth medium per well of a six-well plate. Incubate 3 d at 37°C with 5% CO<sub>2</sub>. This allows setting up the plates on Friday and continuing with the inoculation step on Monday.</li>
</ol>
</li>
<li>Confirm that the cells have formed a confluent monolayer by observing them under the microscope.</li>
<li>Virus dilutions can be made in 1.5 mL reaction tubes or alternatively, in 96-well deep block plates (2.2 mls).</li>
<li>For 1.5 mL reaction tubes:
<ol type="a">
<li>Take the 6 1.5 mL reaction tubes for the dilution series required and label with dilution (this may be between 10<sup>−1</sup> and 10<sup>−6</sup> for each isolate) and isolate name.</li>
<li>Add 900 μL D2 to each tube/well. Prepare six tubes per virus stock to be titrated.</li>
<li>Transfer 100 μL virus to the first tube/well. Discard tip.</li>
<li>Mix the first virus dilution by pipetting up and down several times.</li>
<li>Transfer 100 μL of this diluted virus into the second tube/well. Discard tip.</li>
<li>Continue tenfold dilution series to 10<sup>6</sup> dilution.</li>
</ol>
</li>
<li>Repeat step 22 for every virus stock and for the mock control supernatant.</li>
<li>Aspirate the medium from each well of one 6-well plate.</li>
<li>Quickly transfer 0.5 mL of each dilution of each virus into a well. Start with the highest dilution (10<sup>−6</sup>) of virus working up to the lowest dilution. It is not necessary to change tips between dilutions.</li>
<li>Incubate at room temperature for 40 min, gently tilting the plates every 5 min to prevent the cell layer from drying out.</li>
<li>Add 2 mL overlay medium to each well. There is no need to remove the supernatant.
<ol type="a">
<li>Mix overlay and inoculum by gently swirling the plate.</li>
<li>Incubate undisturbed at 37°C with 5% CO<sub>2</sub> for 3 days. Troubleshooting 1</li>
</ol>
</li>
<li>Visible plaques are usually observed after 3 days but this can be reduced to 2 days or increased to 4 days depending on the virus strain.</li>
<li>Remove the supernatant by aspiration and dispose as contaminated waste.</li>
<li>Add 5 mL fixative per well and incubate for a minimum of 1 h at room temperature inside the microbiological safety cabinet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Adjust this step to conform with local rules for inactivation of the virus and decontamination of the plate.</div>
<ol start="31">
<li>Remove the fixative and add 2 mL of PBS per well.</li>
<li>Remove the PBS and add a minimum of 1 mL staining solution to each well. Incubate for ∼4 h or until sufficient staining of the remaining cell monolayer is observed.</li>
<li>Remove the staining solution.</li>
<li>Gently wash away the remaining staining solution with water.</li>
<li>Take care not to apply water with force as this can damage the cell monolayer.</li>
<li>Invert the plates and allow the cell monolayer to airdry.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Image the plate wells.</div>
<ol start="37">
<li>Count wells that contain approximately 10–50 plaques. Check that the 10-fold dilutions agree to approximately 2-fold.</li>
<li>Calculate the plaque forming units (PFU) per mL of virus stock as follows: Multiply the obtained number of plaques by the dilution and correct for volume of virus added (e.g., if you obtained 43 plaques in the well to which you added 500 μL of the 10<sup>−5</sup> dilution, the titer of the original stock is 43 <b>×</b> 10<sup>5</sup> <b>×</b> 2 = 8.6 <b>×</b> 10<sup>6</sup> PFU ml<sup>−1</sup>). <a href="#sec6.3">Troubleshooting 2</a>
</li>
</ol>
<h3 id="sec3.3">Isolation of human hematopoietic stem and progenitor cells from bone marrow, umbilical cord or peripheral blood</h3>
<div class="timing">
<span class="timing-title">Timing:</span> approximately 8 h</div>
<ol start="39">
<li>Make sure to have a separate, new disposal bag for anything that comes into contact with the blood so it can be discarded immediately after finishing.</li>
<li>Turn on the hood and spray down with 70% ethanol, along with pipet-aides, tissue culture flasks (T175s), pipettes, etc.</li>
<li>Estimate the total volume of fresh blood or bone marrow (BM) aspirate that you have. When using human bone marrow purchased from StemCell Technologies (Cat#70001), you will begin the protocol at this point by defrosting the sample in pure FBS and centrifuge at 300 <i>g</i> for 5 min at 20°C and continue with step 52.</li>
<li>You will need to dilute the blood or BM 2:1 with PBS (e.g., if you have 150 mL of blood in total, you will need to add 300 mL of PBS).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Diluting the blood 3<b>×</b> (e.g., 150 ml of blood with 450 ml of PBS) can give a cleaner layer of mononuclear cells after centrifugation.</div>
<ol start="43">
<li>Pour the blood or BM into a large tissue culture flask and mix with an appropriate volume of PBS.</li>
<li>Prepare 15 mL of Ficoll into 50 mL falcon tubes (prepare enough tubes considering you are going to add 30 mL of PBS-diluted blood in each tube).</li>
<li>Making sure the blood and PBS are mixed, layer 30 mL of blood very gently so as to not mix it with the Ficoll.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If there is an option, select the lowest flow setting on the pipette and angle the tube as horizontal as possible without spilling the Ficoll and blood within. Both of these will minimise the risk of the blood mixing in with the Ficoll (<a href="#fig2">Figure 2</a>A).</div>
<ol start="46">
<li>Spin down at 450 <i>g</i> for 30 min and at 20°C–25°C, making sure there is no brake on.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Having any type of brake on the centrifuge will disturb the layer of mononuclear cells.</div>
<div class="critical">
<span class="critical-title">Critical:</span> It is imperative that the centrifuge be at room temperature (20°C–25°C) for the Ficoll to retain its correct desity.</div>
<div class="note">
<span class="note-title">Note:</span> After the spin, there should be 4 distinct layers: a bottom layer of dark red blood cells with the Ficoll above, a top layer of serum, and a thin layer in the middle (between the serum and Ficoll) of mononuclear cells. You want to collect the mononuclear cells (<a href="#fig2">Figure 2</a>B).</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1168-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Mononuclear cell isolation using Ficoll gradient</div>
<p>Try to pipette 30 mL of blood very gently and angle the tube as horizontal as possible to not mix it with the Ficoll (A). After the centrifugation you should see 4 layers (B).</p>
</figcaption>
</figure>
<ol start="47">
<li>Take the mononuclear cell layer using a pipette and transfer it to a new 50 mL falcon tube. For this, you can pool the mononuclear cell layers from several falcons from the first spin until you have 20–25 mL in total in the new falcon tubes.</li>
<li>Dilute 1:1 with PBS+2% FBS (i.e., add 20–25 mL of PBS + 2% FBS to 20–25 mL of mononuclear cell solution) and centrifuge at 300 <i>g</i> for 5 min at 20°C.</li>
<li>Remove the supernatant from all of the falcon tubes.</li>
<li>To lyse the red blood cells resuspend one cell pellet with 20 mL ammonium chloride. Transfer this resuspended cell pellet to the next falcon and use it to resuspend the next pellet. Repeat the process in order to resuspend up to 5 cell pellets in 20 mL ammonium chloride. Leave at 20°C–25°C for 5 min. <a href="#sec6.5">Troubleshooting 3</a>
</li>
<li>Add 2 mL of pure FBS (heat inactivated) to buffer the lysis and centrifuge at 300 <i>g</i> for 5 min at 20°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When using human bone marrow purchased from StemCell Technologies (Cat#70001), you will begin the protocol at this point by defrosting the sample in pure FBS, then centrifuge at 300 <i>g</i> for 5 min at 20°C and continue with step 52.</div>
<ol start="52">
<li>Remove the supernatant and resuspend the cell pellet in 5 mL 2% FBS/PBS.</li>
<li>Count the cells and continue with the HSPC purification.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, you can freeze the cells in FBS with 10% DMSO (<a href="#bib2">Mitrus et al., 2018</a>) and continue another day. Just be aware that a cycle of freeze/thaw will reduce your yield of isolated cells considerably. <a href="#sec6.7">Troubleshooting 4</a>
</div>
<ol start="54">
<li>Use the EasySep Human Progenitor Cell Enrichment kit (StemCell Technologies, Cat#19356)
<ol type="a">
<li>Prepare your mononuclear cell suspension at 5 <b>×</b> 10<sup>7</sup> cells per mL in 5 mL polystyrene tubes in a maximum volume of 2 mL.</li>
<li>Add 50 ul/mL of Enrichment Cocktail to the sample and incubate at 20°C–25°C for 15 min.</li>
<li>Add 100 ul of Magnetic particles per mL of sample and incubate at 20°C–25°C for 5 min.</li>
<li>Top up to 2.5 mL with 2% FBS/PBS and place the tube into the magnet and incubate at 20°C–25°C for 10 min. The magnet draws out the magnetic beads which are attached to the lineage cells to be discarded.</li>
<li>Carefully transfer the non-magnetic cell suspension with the pipette into a new 5 mL polystyrene tube and centrifuge at 300 <i>g</i> for 5 min at 20°C.</li>
</ol>
</li>
</ol>
<ol start="55">
<li>Stain the enriched cell suspension with the antibody cocktail listed in <a href="#key-resources-table">key resources table</a> to define and sort for hematopoietic stem cells and erythroid progenitors. This should be carried out using the following steps:
<ol type="a">
<li>Prepare the antibody cocktail in the following way (antibody dilutions together with the identifier codes can also be found in the <a href="#key-resources-table">key resources table</a>), mixing the cocktail thoroughly via pipetting:</li>
<li>As indicated above, each sample should be stained in a solution made up of the flow buffer (2% FBS in PBS) and the antibodies. We recommend making a master mix for all of your samples. In order to do this, multiply the volumes of each reagent in the table above by N+1, where N is the number of samples you have to ensure you have enough staining volume for the last sample. E.g. if you have 10 samples, multiply the staining volume by 11 to produce the master mix for the antibody cocktail.
<div class="note">
<span class="note-title">Note:</span> We recommend titrating the staining volumes of each antibody prior the execuation of the experiment as the optimal titers may vary according to the number of cells that are to be stained and the level of expression of each marker/antigen within those cells. As with the master mix, the reagent volumes should be multiplied up according to the total volume used per sample (e.g. multiply all of the reagent volumes in the table above by a factor of 3 if it is determined that 300 uL of staining volume is needed per sample).</div>
<div class="critical">
<span class="critical-title">Critical:</span> You must ensure that the antibody vials are opened and handled without direct exposure to light (e.g. switch off the light of the fume hood you are working in), to avoid photobleaching of the conjugated antibodies.</div>
</li>
<li>Centrifuge the polystyrene tubes with the non-magnetic cell suspension (from step 54e) for 5 min at 300 <i>g</i> and 20°C.</li>
<li>Remove the supernatant from the sample tubes and add the necessary volume of the antibody cocktail per sample.</li>
<li>Resuspend the pellet of cells in each tube in the antibody cocktail via pipetting.</li>
<li>Incubate at 4°C for 30–60 min, covering with aluminum foil if necessary to protect from light.</li>
<li>After the incubation period finishes, wash the samples by adding the flow buffer (see <a href="#tbl1">Table 1</a> for details) at a volume which is at least 10-fold the staining volume e.g., at least 1 mL flow buffer for 100 uL staining volume.
<table id="tbl1">
<caption>Table 1. Reagents and relative volumes to use in order to stain samples for fluorescence assisted cell sorting</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Volume (per 10<sup>6</sup> cells) in μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Lineage cocktail-APCCy7</td>
<td>1 (of each Ab)</td>
</tr>
<tr>
<td>CD34-PECy7</td>
<td>1</td>
</tr>
<tr>
<td>CD38-PE</td>
<td>2</td>
</tr>
<tr>
<td>CD117-BB700</td>
<td>1</td>
</tr>
<tr>
<td>CD71-APC</td>
<td>1</td>
</tr>
<tr>
<td>CD235a-FITC</td>
<td>1</td>
</tr>
<tr>
<td>Flow buffer: 2% FBS in PBS</td>
<td>94</td>
</tr>
<tr>
<td>TOTAL</td>
<td>100</td>
</tr>
</tbody>
</table>
</li>
<li>Centrifuge the samples for 5 min at 300 <i>g</i> and 20°C.</li>
<li>Prepare the solution to be used for staining of the viability marker. For this, dilute DAPI (BD Biosciences, Cat#56407) 1:1000 in the flow buffer.
<div class="note">
<span class="note-title">Note:</span> As with the staining volume, the volume of the DAPI solution to be used per sample should be titrated according to the number of cells present and to ensure an appropriate concentration of cells in the solution for the purposes of an efficient sorting process.</div>
</li>
<li>Remove the supernatant from the sample tubes and resuspend the pellets in the DAPI solution via pipetting.</li>
<li>Filter the samples in order to remove any clumps in the cell suspension that may block the sorter.</li>
</ol>
</li>
</ol>
<ol start="56">
<li>Prepare 4 × 1.5 mL Eppendorf tubes for Fluorescence-Activated Cell Sorting (FACS) collection of each sample by adding 200 μL of pure FBS into each tube. Up to 4 populations can be sorted simultaneously, as available instrumentation allows (e.g., HSCs, ERP-S1, ERP-S2 and ERP-S3 as in <a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1168-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative gating strategy to define hematopoietic stem cells and erythroid progenitors</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PBS+2% FBS can be used as collection media but you achieve better recovery of cells with pure FBS.</div>
<ol start="57">
<li>Centrifuge the sorted cell populations at 300 <i>g</i> for 5 min at 20°C and resuspend in StemSpan SFEM (Stem Cell Technologies) supplemented with 100 ng/mL rhFLT-3L, 100 ng/mL rhSCF and 100 ng/mL rhTPO (e.g., 5,000 cells resuspended in 400 μL in a 48-well plate). <a href="#sec6.9">Troubleshooting 5</a>
</li>
</ol>
<h3 id="sec3.4">Infection of hematopoietic stem and progenitor cells (HSPCs) with SARS-CoV-2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 24–30 h</div>
<ol start="58">
<li>Infect Hematopoietic cells with SARS-CoV-2 (add the virus directly into the well) at a MOI of 0.5 or 5 PFU per cell for 30 min (to analyze binding capacity of the virus to cells) or 24 h (to analyze active viral infection) in the incubator (37°C, 5% CO<sub>2</sub>).</li>
<li>Transfer cells from the 48-well plate to 1.5 mL Eppendorf tubes and wash three times with PBS (4 min at 300 <i>g</i>) to remove unbound virus prior to lysis for RNA isolation (step 60) or to culture in methylcellulose for a colony forming unit (CFU) assay (see step 64).</li>
</ol>
<h3 id="sec3.5">SARS-Cov-2 quantification by reverse transcription quantitative real-time PCR (RT-qPCR)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 h</div>
<ol start="60">
<li>After removing the last PBS wash, lyse the pellet with RLT buffer from the RNeasy Microkit (Qiagen, Cat# 74004) and vortex thoroughly.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Samples can be stored in RLT buffer at −15°C to −25°C for 3–4 weeks.</div>
<ol start="61">
<li>Perform RNA isolation according to the manufacturer’s instructions.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To minimize loss of material, elute by adding 13 μl elution buffer. As the dead volume of the column is 2 μl, this will result in 11 μl of eluted RNA, which is the maximum volume recommended in the reverse transcription reaction.</div>
<ol start="62">
<li>Use all 11 μL of RNA to perform cDNA synthesis using SuperScript III First-Strand Synthesis system (Thermo Fisher Scientific, Cat# 18080051).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Transcription First Strand cDNA synthesis kit (Roche, Cat#04896866001).</div>
<ol start="63">
<li>Analyze SARS-CoV-2 detection by quantitative real-time PCR using the primers described in the <a href="#key-resources-table">key resources table</a> (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1168-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. SARS-CoV-2 quantification by RT-qPCR</div>
<p>Representative Amplification plots (A and C) and Melting Curve plots (B and D) from viral gene N. Samples that are positively infected show Ct values of 20–28 in the amplification plot (A) and clear and uniform peak in the melting curve plot (B). Notice that samples that are not infected could give a late signal, above 35 Ct (C) due to background noise (D).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We recommend using 384-well reaction plates and performing the reaction in 10 μl volume to maximize the number of reactions and target genes you can run.</div>
<h3 id="sec3.6">Colony forming unit assay and viral quantification in colonies</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10–14 days</div>
<ol start="64">
<li>(From step 58). After 24 h of infection, 5,000 cells will be seeded for 10–14 days in a methylcellulose-based medium with recombinant cytokines for human cells (StemCell Technologies, Cat#04435) following these steps:
<ol type="a">
<li>Incubate the different hematopoietic stem and progenitor cells during the 24 h infection period in 48 well-plates, using 400 μL media (same conditions as in step 58).</li>
<li>After 24 h of infection, add 1 mL of methylcellulose to the media using blunt end needles.</li>
<li>Thoroughly mix the cells in the media with the methylcellulose using a 2 mL syringe.</li>
<li>Take all the methylcellulose mix with the syringe and divide into two wells of 12 well-plates, adding 700 μL into each well.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Add 2 mL PBS into three empty wells of the 12-well plate, to prevent the methylcellulose from drying out during the incubation period. If this occurs, then it will make picking the colonies very difficult.</div>
<ol start="65">
<li>Incubate for 10–14 days in the incubator (37°C, 5% CO<sub>2</sub>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Colony density must be appropriate for picking individual colonies. Start screening the colony plates under the microscope by day 7 to ensure that you can pick individual colonies (<a href="#fig5">Figure 5</a>). <a href="#sec6.11">Troubleshooting 6</a>
</div>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1168-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Colony forming unit assay after SARS-CoV-2 infection</div>
<p>Representative examples of appropriate colony density to pick individual colonies (left) and overconfluent colony plate (right) where it is difficult to distinguish and pick indiviudal colonies. Scale bar 100 μm.</p>
</figcaption>
</figure>
<ol start="66">
<li>Check the colonies under the microscope to count, score and pick them. For details about the type of colonies to expect, please refer to (<a href="#bib1">Huerga Encabo et al., 2021</a>).
<ol type="a">
<li>Using the microscope make sure you choose the appropriate slide holder e.g., the adjustable version for the individual plates and the larger rectangular version for the multi-well plates used for counting.
<div class="note">
<span class="note-title">Note:</span> If individual colonies are spread and visible without using a microscope you can directly pick the colonies.</div>
</li>
<li>Mark the plate with a pen at the bottom (i.e., at 6 o’clock) as this will be useful when rotating the plate at the end to check for colonies on the borders- you will know when you have finished as the marks will be in the same position as at the start.</li>
<li>Start counting from the top left of the plate and work across. Then when reaching the other side, go down (making sure there is a little overlap from the 'row' above) and work across from right to left. Repeat this process to cover the entire area of the plate.</li>
<li>Acording to the number of colonies counted, prepare 1.5 mL Eppendorf tubes with 350 μL of RLT buffer from the RNeasy Microkit (Qiagen, Cat# 74004) in each. One tube will be used for every individual colony picked, so that they can be placed directly into the RLT buffer.</li>
<li>Pick and score each colony, using a p10 pipette set at 10 μL.</li>
<li>Transfer the colony/methylcellulose mixture into the RLT buffer by pipetting up and down repeatedly. Then press all the way down to fully dispense from the pipette tip, before discarding the tip and changing to the next one.</li>
<li>Work your way around the plate doing this until you reach the bottom.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Once done, you need to check the borders: start in the top left corner again and then rotate the plate, focusing on the same part of the plate the whole time. The idea is to find more colonies on the borders which may have been previously obscured by the margins of the plate holder. Using the mark that was made earlier, you will know once you have fully rotated the plate.</div>
<ol start="67">
<li>Proceed to the RNA extraction, retrotranscription and RT-qPCR (as done in steps 61 and 62) for the viral quantification from the different colonies</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Using SARS-CoV-2 England/02 we routinely obtain stocks with titers in the high 10<sup>6</sup> to low 10<sup>7</sup> PFU ml<sup>−1</sup> range. This might differ depending on the variant.</p>
<p>We tend to obtain 100–150 <b>×</b> 10<sup>6</sup> mononuclear cells per 50 mL of freshly acquired peripheral or cord blood (processed within 24 h of sampling the cord blood from the donor).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>There may be variability in the exact yield of mononuclear cells and therefore haematopoietic stem and progenitor cells extracted from the blood or bone marrow source selected. This will in part be influenced by the quality of the source (especially how long after its acquisition it is processed or the thawing of frozen samples) and the efficiency of FACS.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Irregular shaped plaques in the plaque assay (step 27 of <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>This generally indicates that the plates have been moved during the incubation time. If the plaques appear to have comets this is usually caused by small movements during the incubation period. Check that the incubator is placed on a solid surface and not exposed to vibrations. Do not slam the doors of the incubator shut.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low titer of virus stocks (step 38 of <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>(1) Ensure that you harvest the virus containing supernatant at the optimal time as explained in step 12. Prepare several infected flasks and harvest at different time points to obtain the best titer. (2) Different virus variants might have different requirements for expression levels of hACE2 and hTMPRSS2. Obtaining Vero cells that constitutively overexpress one or both of these proteins can improve stock titers of some variants. (3) Accumulation of defective interfering particles during consecutive passages. Prepare an early passage inoculation stock and use this stock for the preparation of your working stock. Make sure that you infect cells at on MOI of 0.01 PFU per cell or lower.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Incomplete lysis of red blood cells during the protocol to obtain HSPCs (step 50 of <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>There are four steps to take: (1) Try increasing the ratio of ammonium chloride to cells being lysed, for example by resuspending fewer cell pellets (three pellets instead of five) per each 20 mL of ammonium chloride. (2) Leave the pellets to resuspend for a longer time period in the ammonium chloride, for example ten instead of five minutes. (3) If the previous two measures do not work, change the batch of ammonium chloride if possible as the issue may be batch specific. (4) Alternatively, you can also incubate the cells with the lysis buffer in a water bath at 37°C.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Poor yield of mononuclear cells from blood or bone marrow source (step 53 of <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>A poor yield would be defined as being below the expected 100–150 <b>×</b> 10<sup>6</sup> mononuclear cells per 50 mL blood. As mentioned under ‘limitations’, it is important to process a source that is recently acquired (within the past 24 h) and avoid freeze-thaw the sample.</p>
<p>Within the protocol, there are a couple of solutions: (1) Further dilute the blood with PBS (step 4 of <a href="#step-by-step-method-details">step-by-step method details</a>) beyond the 2:1 ratio, for example trying a 3:1 or 4:1 ratio to increase separation during the Ficoll spin; (2) Ensure that all of the layer containing mononuclear cells is removed (step 9 of <a href="#step-by-step-method-details">step-by-step method details</a>), returning several times with the pipette to do so if necessary.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Poor yield of HSPCs (step 57 of <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Ensure that an accurate count of the mononuclear cells has been performed prior to using the Human Progenitor Cell Enrichment Kit. This will mean that a sufficient volume of reagents (e.g., the selection cocktail and magnetic particles) from the kit are used for the number of cells you have. Also be sure to filter the mononuclear cells before carrying out the enrichment as they may form clumps which can interfere with the magnetic selection process.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Colonies are either too sparse or too densely populated to recover a significant yield (step 65 of <a href="#step-by-step-method-details">step-by-step method details</a>).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>In either case, refer back to the number of cells seeded per well as this may need to be adjusted accordingly if the colony density becomes a consistent issue.</p>
<p>Consider seeding a range of cells either below or above the recommended 5,000 cell number and incubating for 10–14 days, to determine which dose is most appropriate for you.</p>
<p>If there are concerns about colonies being too densely populated, you can also monitor the wells of methylcellulose from day 7 onwards and decide to pick them earlier within the 10–14 days incubation period if they reach a suitable level of density.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Huerga Encabo, H., Grey, W., Garcia-Albornoz, M., Wood, H., Ulferts, R., Aramburu, I.V., Kulasekararaj, A.G., Mufti, G., Papayannopoulos, V., Beale, R., et al. (2021). Human erythroid progenitors are directly infected by SARS-CoV-2: implications for emerging erythropoiesis in severe COVID-19 patients. Stem Cell Rep. <i>16</i>, 428-436. <a class="external-link" href="https://doi.org/10.1016/j.stemcr.2021.02.001">https://doi.org/10.1016/j.stemcr.2021.02.001</a></p>
<p id="bib2">Mitrus, I., Smagur, A., Fidyk, W., Czech, M., Prokop, M., Chwieduk, A., Glowala-Kosinska, M., Czerw, T., Sobczyk-Kruszelnicka, M., Mendrek, W., et al. (2018). Reduction of DMSO concentration in cryopreservation mixture from 10% to 7.5% and 5% has no impact on engraftment after autologous peripheral blood stem cell transplantation: Results of a prospective, randomized study. Bone Marrow Transplant. <i>53</i>, 274-280. <a class="external-link" href="https://doi.org/10.1038/s41409-017-0056-6">https://doi.org/10.1038/s41409-017-0056-6</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dominique Bonnet (<a href="mailto:dominique.bonnet@crick.ac.uk">dominique.bonnet@crick.ac.uk</a>).</p>
<h4>Materials availability</h4>
<p>SARS-CoV-2 can be supplied following MTA agreement.</p>
<h4>Data and code availability</h4>
<p>This study did not generate datasets or codes</p>
<h3>Acknowledgments</h3>
<p>We want to thank the staff at the Flow Cytometry core facility of the Francis Crick Institute for their valuable help. This work was supported by the Francis Crick Institute, which receives its core funding from <a href="https://doi.org/10.13039/501100000289">Cancer Research UK</a> (FC001045), the <a href="https://doi.org/10.13039/501100000265">UK Medical Research Council</a> (FC001045), and the <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> (FC001045) to D.B. H.H.E. is a recipient of a Kay Kendall Leukaemia Fund Fellowship.</p>
<h3>Author contributions</h3>
<p>Conceptualization, H.H.E. and D.B.; methodology, H.H.E. and R.U.; protocol validation and writing paper, H.H.E., R.U., and A.S.; review &amp; editing paper, H.H.E., R.U., A.S., R.B., and D.B.; funding acquisition, D.B.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>