<article>
<section>
<h1 id="header">Analysis of reactive astrogliosis in mouse brain using <i>in situ</i> hybridization combined with immunohistochemistry</h1>
<p><time datetime="2021-03-05">Published: March 5, 2021</time></p>
<p> Ranjithmenon Muraleedharan,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Diana Nardini,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Ronald Raymond Waclaw,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup> and Biplab Dasgupta<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#fn3">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Oncology, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</p>
<p id="aff2"><sup>2</sup>Division of Experimental Hematology and Cancer Biology, Cincinnati Children’s Hospital Medical Center, Cincinnati, OH 45229, USA</p>
<p id="aff3"><sup>3</sup>Department of Pediatrics, University of Cincinnati College of Medicine, Cincinnati, OH 45229, USA</p>
<p id="fn1"><sup>4</sup>Present address: Therapeutic Radiology, Yale School of Medicine, Hunter Building, 15 York Street, New Haven, CT 06520, USA</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ronald.waclaw@cchmc.org">ronald.waclaw@cchmc.org</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:biplab.dasgupta@cchmc.org">biplab.dasgupta@cchmc.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100375">10.1016/j.xpro.2021.100375</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Reactive astrogliosis is characterized by a profound change in astrocyte phenotype in response to all CNS injuries. Here, we present a revised <i>in situ</i> hybridization and immunohistochemistry (IHC) protocol to label the reactive astrocytes in the mouse brain. Several approaches for quantifying astrocyte reactivity lacked sensitivity to discriminate across the spectrum. We optimized <i>in situ</i> hybridization followed by IHC. We provide a staining protocol for quantitative measures of astrocyte reactivity as an independent confirmation of the magnitude of reactive gliosis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib8">Muraleedharan et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Quantitative measurement of reactive gliosis in mouse brain</li>
<li>Dual staining identifies reactive gliosis occurs in response to chronic CNS pathology</li>
<li>Hybridization phase includes LCN2 and C3 mRNA hybridization and signal development</li>
<li>Immunostaining phase includes antibody detection of GFAP protein</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/501-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This dual staining protocol was created to detect reactive astrocytes in mouse brain sections, allowing the distinction between normal and activated astrocytes. The LCN2 &amp; C3 genes were chosen because they are widely validated genes used as an indirect marker of astrocytes activation in the brain. Here we employ newly designed probes to simultaneously detect LCN2 or C3 mRNA followed by immunostaining with GFAP antibody of high specificity for astrocytes.</p>
<p>The protocol detailed below describes the procedure for brain dissection and sectioning, <i>in situ</i> hybridization to detect LCN2 or C3, and immunostaining to detect GFAP protein. The comments outlining critical steps are provided.</p>
<div class="note">
<span class="note-title">Note:</span> The <i>in situ</i> hybridization protocol is a modification from the protocol published by <a href="#bib13">Toresson et al., 1999</a> (<a href="#bib13">Toresson et al., 1999</a>) and <a href="#bib2">Kohli et al., 2018</a> (<a href="#bib2">Kohli et al., 2018</a>).</div>
<h3 id="sec1.1">Designing the RNA probes</h3>
<p>Optimal probe lengths are usually considered to be 300–500 bases, however, longer probes may result in slightly higher background labeling or may not penetrate tissue. RNA probes are PCR amplified from cDNA and generated with standard methods from Promega and Roche.</p>
<div class="critical">
<span class="critical-title">Critical:</span> All solutions, materials, and equipment should be RNase-free.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure you optimized the antibodies before you start the combined <i>in situ</i> hybridization and IHC protocol.</div>
<h3 id="sec1.2">RNA probe synthesis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<p>The following protocol describes the production of DIG labeled RNA probes.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Throughout the protocol, work on a clean bench, use RNase/DNase-free tips, reagents, and tubes to avoid probe degradation.</div>
<ol>
<li>PCR amplification of probe region from cDNA obtained from postnatal mouse brain RNA-PCR reaction with primers (Note: 3′ primer should contain the T3 sequence at the 5′ end)- Example for LCN2:5′ Primer – GCTGTCGCTACTGGATCAGA3′ Primer (T3 sequence in italics/bold) – <b>ATTAACCCTCACTAAAGG</b> TGGTGGTGTTAAGACAGGTGG</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> GoTaq from Promega is used for PCR amplification of region.</div>
<p>PCR reaction mix per sample. Run eight samples for one probe.</p>
<table id="undtbl1">
<thead>
<tr>
<th>GoTaq master mix</th>
<th>12.5 μL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Primer 1 (25 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Primer 2 (25 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>cDNA (approx. 2,000 ng/μL)</td>
<td>0.75 μL</td>
</tr>
<tr>
<td>RNase-free water</td>
<td>9.75 μL</td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>20 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ol start="2">
<li>Isolate and extract PCR product from 2% agarose gel using Qiagen Gel Extraction kit methods. <a href="https://www.qiagen.com/us/shop/pcr/qiaquick-gel-extraction-kit/">https://www.qiagen.com/us/shop/pcr/qiaquick-gel-extraction-kit/</a>
</li>
<li>Prepare <i>in vitro</i> transcription reaction.</li>
</ol>
<table id="undtbl3">
<thead></thead>
<tbody>
<tr>
<td>PCR product of target region</td>
<td>1 μg template DNA</td>
</tr>
<tr>
<td>5× reaction buffer</td>
<td>8 μL</td>
</tr>
<tr>
<td>DIG labeling Mix dNTPs</td>
<td>4 μL</td>
</tr>
<tr>
<td>RNase inhibitor</td>
<td>1 μL</td>
</tr>
<tr>
<td>T3 RNA polymerase</td>
<td>3 μL</td>
</tr>
<tr>
<td>RNase-free water</td>
<td>fill up to 40 μL total volume</td>
</tr>
</tbody>
</table>
<p>Incubate at 37°C for 4 h</p>
<ol start="4">
<li>Add 10× DNase Buffer (4 μL) and DNase I (1.5 μL) and incubate for 1 h at 37°C.</li>
<li>Stop probe reactions using Ammonium acetate/EDTA stop solution (15 μL; see the recipe for Stock Stop Solution in “<a href="#materials-and-equipment">Materials and equipment</a>”), RNase-free water (95 μL), and cold Isopropanol (151 μL).</li>
<li>Place probe tubes at −20°C for 12–18 h.</li>
<li>Spin probe tubes at 21,130 × <i>g</i> for 20 min at 4°C, carefully remove supernatant and add 300 μL of 100% ethanol and spin again at 21,130 × <i>g</i> for 20 min at 4°C.</li>
<li>Carefully remove supernatant and dry pellet at 20°C–22°C for 5–7 min.</li>
<li>Resuspend pellet with 17 μL of RNase-free water.</li>
<li>Run 2 μL on a 1% agarose gel to check probe.</li>
<li>Store probe tubes at −80°C for long term storage (up to 6 months) or −20°C for short term storage (up to 72 h).</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-GFAP antibody</td>
<td>DAKO</td>
<td>DAKO: Z0334</td>
</tr>
<tr>
<td>Anti-rabbit biotinylated secondary antibody</td>
<td>DAKO</td>
<td>DAKO- E0353</td>
</tr>
<tr>
<td>Anti-DIG antibody</td>
<td>Roche</td>
<td>11093274910</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>16% PFA</td>
<td>Electron Microscopy Sciences</td>
<td>15710</td>
</tr>
<tr>
<td>10× dPBS</td>
<td>Life Technologies</td>
<td>14200166</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Life Technologies</td>
<td>25530049</td>
</tr>
<tr>
<td>RNaseA</td>
<td>Invitrogen</td>
<td>12091021</td>
</tr>
<tr>
<td>Torula yeast RNA (tRNA)</td>
<td>Sigma-Aldrich</td>
<td>R6625-25G</td>
</tr>
<tr>
<td>Levamisol</td>
<td>Sigma-Aldrich</td>
<td>L9756-5G</td>
</tr>
<tr>
<td>BM-Purple</td>
<td>Roche</td>
<td>11442074001</td>
</tr>
<tr>
<td>Normal goat serum (NGS)</td>
<td>Jackson ImmunoResearch</td>
<td>005-000-121</td>
</tr>
<tr>
<td>20× SSC</td>
<td>Invitrogen</td>
<td>15557044</td>
</tr>
<tr>
<td>Dextran sulfate</td>
<td>Fisher Bioreagents</td>
<td>BP1585-100</td>
</tr>
<tr>
<td>50× Denhardt’s solution</td>
<td>Sigma-Aldrich</td>
<td>D2532-5mL</td>
</tr>
<tr>
<td>Formamide (high stringency)</td>
<td>Sigma-Aldrich</td>
<td>F7503-1L</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Fisher Bioreagents</td>
<td>BP337-100</td>
</tr>
<tr>
<td>Formamide (Hyb buffer)</td>
<td>Sigma-Aldrich</td>
<td>F9037-100mL</td>
</tr>
<tr>
<td>NaCl</td>
<td>Fisher Bioreagents</td>
<td>BP358212</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Fisher Bioreagents</td>
<td>BP1531</td>
</tr>
<tr>
<td>Tris base</td>
<td>Fisher Chemical</td>
<td>T3951</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub> anhydrous</td>
<td>Sigma-Aldrich</td>
<td>S3139</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub> anhydrous</td>
<td>Sigma-Aldrich</td>
<td>S3264</td>
</tr>
<tr>
<td>1 M Tris-HCl pH 7.5</td>
<td>Invitrogen</td>
<td>15567027</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>
</td>
<td>Sigma-Aldrich</td>
<td>M8266-1kG</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>Fisher Chemical</td>
<td>A416500</td>
</tr>
<tr>
<td>Ammonium acetate</td>
<td>Sigma-Aldrich</td>
<td>A1542</td>
</tr>
<tr>
<td>Ultra Pure 0.5 M EDTA</td>
<td>Life Technologies</td>
<td>15575020</td>
</tr>
<tr>
<td>1 M Tris pH 9.5</td>
<td>Teknova</td>
<td>T1095</td>
</tr>
<tr>
<td>Sucrose solution</td>
<td>Sigma-Aldrich</td>
<td>S0389</td>
</tr>
<tr>
<td>Acetone</td>
<td>Fisher Chemical</td>
<td>A181</td>
</tr>
<tr>
<td>Hydrogen peroxide solution</td>
<td>Sigma</td>
<td>H1009-5ML</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma</td>
<td>T8787</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>VECTASTAIN ABC Reagent</td>
<td>Vectorlabs</td>
<td>PK-6100</td>
</tr>
<tr>
<td>VECTASHIELD Vibrance Antifade Mounting Medium</td>
<td>Vectorlabs</td>
<td>H-1700-2</td>
</tr>
<tr>
<td>DAB substrate</td>
<td>Sigma-Aldrich</td>
<td>D5637</td>
</tr>
<tr>
<td>T3 RNA polymerase</td>
<td>Promega</td>
<td>P2083</td>
</tr>
<tr>
<td>5× reaction buffer</td>
<td>Promega</td>
<td>P1181</td>
</tr>
<tr>
<td>10× DNase buffer</td>
<td>Thermo Fisher formerly Ambion</td>
<td>AM8160G</td>
</tr>
<tr>
<td>DNase I</td>
<td>Thermo Fisher formerly Ambion</td>
<td>AM2235</td>
</tr>
<tr>
<td>RNase inhibitor</td>
<td>NEB</td>
<td>M0307S</td>
</tr>
<tr>
<td>GoTaq Master Mixes</td>
<td>Promega</td>
<td>M7122</td>
</tr>
<tr>
<td>Qiaquick Kit</td>
<td>Qiagen</td>
<td>28706</td>
</tr>
<tr>
<td>DIG DNA Labeling Mix (DIG labeling Mix dNTPs)</td>
<td>Millipore Sigma</td>
<td>11277065910</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: male 4–24 months AMPKb1lox/lox AMPKb2-/- Nestin Cre</td>
<td>DasGupta Lab</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>LCN2-5′</td>
<td>IDT</td>
<td>GCTGTCGCTACTGGATCAGA</td>
</tr>
<tr>
<td>LCN2-3- with T3</td>
<td>IDT</td>
<td>ATTAACCCTCACTAAAGG TGGTGGTGTTAAGACAGGTGG</td>
</tr>
<tr>
<td>CompC3-5′</td>
<td>IDT</td>
<td>TGGGCAAGACAGTCGTCATC</td>
</tr>
<tr>
<td>CompC3-3′ with T3</td>
<td>IDT</td>
<td>ATTAACCCTCACTAAAGG TGGATCTGGTACGGGGAAGT</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>GraphPad Prism 8</td>
<td>GraphPad</td>
<td>RRID: SCR_002798 <a href="https://www.graphpad.com/">https://www.graphpad.com/</a>
</td>
</tr>
<tr>
<td>ImageJ</td>
<td><a href="https://imagej.nih.gov/ij/download.html">https://imagej.nih.gov/ij/download.html</a></td>
<td>ImageJ; RRID: SCR_003070</td>
</tr>
<tr>
<td>Leica Acquisition Software (LASX) suite</td>
<td>Leica</td>
<td>n/a</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Richard-Allen Scientific Neg-50 Frozen Section Medium</td>
<td>Thermo Scientific</td>
<td>6502</td>
</tr>
<tr>
<td>Dry ice</td>
<td>n/a</td>
<td>n/a</td>
</tr>
<tr>
<td>Leica light microscope DM2500</td>
<td>Leica</td>
<td>n/a</td>
</tr>
<tr>
<td>Lecia DMC6200 camera</td>
<td>Leica</td>
<td>n/a</td>
</tr>
<tr>
<td>Leica cryostat</td>
<td>Leica</td>
<td>n/a</td>
</tr>
<tr>
<td>Superfrost slides</td>
<td>VWR</td>
<td>48311-703</td>
</tr>
<tr>
<td>Plastic molds disposable deep base molds, 37 × 24 × 10 mm, 500 PCS/pack</td>
<td>Ihcworld</td>
<td>M475-10</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">RNase-free pre-hybridization and hybridization solution</h3>
<p>Store at −20°C for 6 months</p>
<div class="note">
<span class="note-title">Note:</span> 50% dextran sulfate solution is viscous and should be pipetted with care to ensure precision in volume measurements.</div>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Deionized formamide</td>
<td>50% (v/v)</td>
<td>25 mL</td>
</tr>
<tr>
<td>50 mg/mL torula yeast tRNA</td>
<td>200 μg/mL</td>
<td>200 μL</td>
</tr>
<tr>
<td>50× Denhardt's solution</td>
<td>1×</td>
<td>1 mL</td>
</tr>
<tr>
<td>50% Dextran sulfate</td>
<td>10%</td>
<td>10 mL</td>
</tr>
<tr>
<td>Salt solution</td>
<td>10% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td>RNase-free ultrapure water</td>
<td>n/a</td>
<td>9 mL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.2">Salt solution</h3>
<table id="undtbl5">
<caption>Make 5 mL aliquots and store at −20°C for 6 months</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>3 M</td>
<td>17.5 g</td>
</tr>
<tr>
<td>Tris-Cl pH 8.0</td>
<td>0.089 M</td>
<td>1.4 g</td>
</tr>
<tr>
<td>Tris Base</td>
<td>0.011 M</td>
<td>0.14 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub> anhydrous</td>
<td>0.05 M</td>
<td>0.6 g</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>0.05 M</td>
<td>0.71 g</td>
</tr>
<tr>
<td>EDTA pH 8.0 (0.5 M)</td>
<td>0.05 M</td>
<td>10 mL</td>
</tr>
<tr>
<td>RNase-free ultrapure water</td>
<td>n/a</td>
<td>Fill up to 100 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.3">Stock stop solution</h3>
<table id="undtbl6">
<caption>Stop solution can be aliquoted and stored at −20°C for 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Ammonium acetate</td>
<td>5 M</td>
<td>1.927 g</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.1 M</td>
<td>1 mL</td>
</tr>
<tr>
<td>RNase-free ultrapure water</td>
<td>n/a</td>
<td>Fill up to 5 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.4">High-stringency wash buffer</h3>
<table id="undtbl7">
<caption>Stop solution can be aliquoted and stored at 22°C for 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Formamide</td>
<td>50% (v/v)</td>
<td>125 mL</td>
</tr>
<tr>
<td>20× SSC</td>
<td>2×</td>
<td>25 mL</td>
</tr>
<tr>
<td>RNase-free ultrapure water</td>
<td>n/a</td>
<td>Fill up to 250 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.5">RNase buffer</h3>
<table id="undtbl8">
<caption>Stop solution can be aliquoted and stored at 22°C for 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5 M)</td>
<td>0.5 M</td>
<td>50 mL</td>
</tr>
<tr>
<td>Tris-HCl pH 7.5 (1 M)</td>
<td>0.01 M</td>
<td>5 mL</td>
</tr>
<tr>
<td>EDTA pH8 (0.5 M)</td>
<td>0.005 M</td>
<td>5 mL</td>
</tr>
<tr>
<td>RNase-free ultrapure water</td>
<td>n/a</td>
<td>Fill up to 500 mL</td>
</tr>
</tbody>
</table>
<h3 id="sec2.6">NTMT buffer</h3>
<table id="undtbl9">
<caption>Stop solution can be aliquoted and stored at −20°C for 6 months.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5 M)</td>
<td>0.1 M</td>
<td>3 mL</td>
</tr>
<tr>
<td>Tris-HCl pH 9.5 (1 M)</td>
<td>0.1 M</td>
<td>15 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>0.05 M</td>
<td>7.5 mL</td>
</tr>
<tr>
<td>Tween 20</td>
<td>0.1%</td>
<td>150 μL</td>
</tr>
<tr>
<td>RNase-free ultrapure water</td>
<td>n/a</td>
<td>Fill up to 150 mL</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Collection and preparation of mouse brain for ISH-tissue dissection, fixing, and cryoprotection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes the protocol from dissecting mouse brain until it is fixed and ready for embedding. Keep the time between perfusion or sacrificing of the animal and fixation as short as possible, for best preservation of morphology.</p>
<ol>
<li>Day 1: Animal procedure. Place mouse in isoflurane chamber to anesthetize the animal.</li>
<li>Prior to surgery, a ketamine/xylazine mixture dissolved in saline (up to 80 mg/kg body weight ketamine and 10 mg/kg body weight xylazine) is administered via intraperitoneal injection (27-gauge needle and 1 mL syringe).</li>
<li>Use toe pinch response method to determine depth of anesthesia. Animal must be unresponsive before continuing.</li>
<li>Once the animal has reached a surgical plane of anesthesia, place mouse on its back on the shallow tray.</li>
<li>Make a 5–6-cm lateral incision through the integument and abdominal wall just beneath the rib cage.</li>
<li>Continue the diaphragm incision along the entire length of the rib cage to expose the pleural cavity.</li>
<li>Insert the 30-gauge needle into the posterior end of the left ventricle, being careful to keep the tip of the needle in the lumen of the ventricle.</li>
<li>Finally, make an incision to the animal's right atrium using iris scissors to create as large an outlet as possible without damaging the descending aorta. At this point the animal is ready to be perfused.</li>
<li>Perfusion. Open and attach outlet port to needle base and pump up the manometer bulb to a pressure of 80 mm Hg.</li>
<li>Perfuse with saline solution for 10 min.</li>
<li>After 10 min, switch the stopcock to allow for flow of formalin solution, at same flow rate as saline.</li>
<li>Perfuse with 4% PFA solution for 10–15 min.</li>
<li>The mouse should be stiff at this stage.</li>
<li>Separate the mouse head with surgical scissors by making a cut posterior from the ears. Make a caudal midline incision in the skin and work rostrally to remove the skin from skull.</li>
<li>Starting from the caudal part, cut through the top of the skull along midline and between the eyes with Iris scissors. Remove the parietal and frontal bone plates by tilting one side of a bone plate each time and snapping it off with tweezers.</li>
<li>Gently tilt the brain upward from the anterior part with tweezers and cut the optic nerves and other cranial nerves. Gently lift the brain out of the skull.</li>
<li>Remove the brain and place it in a vial of fixative containing fluid at least 10× the volume of the brain itself. Swirl the vial occasionally.</li>
<li>Fix biopsies (3×3 mm) immediately in freshly prepared 4% (w/v) PFA/PBS for 12–18 h.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> When handling paraformaldehyde (PFA), both solid and aqueous, wear PPE and use a safety cabinet. Prepare formaldehyde solution by dissolving 4% (w/v) PFA into 1× phosphate buffered saline (PBS) solution using heat (55°C). Filter the formaldehyde solution with filter paper.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Proper fixation protocol followed by an appropriate tissue preparation form the basis for good histology (<a href="#bib6">Montero, 2003</a>). The ideal fixative should preserve the antigens in the tissue in a form that is accessible and recognizable to the antibodies applied for the labeling technique.</div>
<ol start="19">
<li>Day 2: Cryoprotection. Following fixation, keep the brain in 15% sucrose for 2–3 h and 30% sucrose for 12–18 h.</li>
<li>Brain can be embedded for cryosectioning.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step is important for keeping the intact structure of the brain anatomy, including astrocytes morphology.</div>
<h3 id="sec3.2">Preparation of frozen tissue blocks/embedding and sectioning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This part describes the preparation of frozen tissue blocks for the mouse brain.</p>
<ol start="21">
<li>Day 3: Label plastic molds for frozen block preparation with a permanent marker. It is necessary to make marks on the mold to indicate the orientation of the brain. Place a piece of clear tape over the labeling to prevent smearing. Place the molds on a flat surface, cover the bottom of the mold with the NEG50 or OCT freezing compound.</li>
<li>Prechill acetone in a Petri dish placed on dry ice pellets. Keep this Petri dish on top of dry ice inside a Styrofoam box.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Tissues should be frozen as rapidly as possible to avoid ice crystals. The method we prefer uses dry ice in pellet form. Work in a fume hood. Put a minimal amount of acetone into a petri dish so the bottom is covered. Carefully add a couple small dry ice pellets to the dish with acetone, leaving enough room for the mold. When the pellets stop bubbling vigorously, the "slurry" is ready. To maintain the temperature, you might have to add more dry ice pellets. Once you have filled the mold and oriented the tissue, place it in the cold acetone to freeze it.</div>
<div class="critical">
<span class="critical-title">Critical:</span> This step is important for keeping the intact structure of the brain anatomy, including astrocytes morphology.</div>
<ol start="23">
<li>Remove the brain from the 30% sucrose solution. Use forceps to keep the brain in upright position. Hold a corner of the brain on a Kimwipe to release excess liquid and then place the brain on the empty dish on ice. Cut the bottom part of the brain to keep it level, if required.</li>
<li>Place the brain into molds filled with NEG50. Mix the brain gently with NEG50. Orient the brain as required for the most efficient sectioning. Cover the tissue with NEG50 and fill up the mold.</li>
<li>Freeze quickly by placing the mold on petri dish with acetone on dry ice. Make sure the acetone only goes to the bottom quarter of the mold. Allow it to stand on dry ice until the block is fully white and solid. Dry the bottom of the mold with a paper towel. Wrap the mold with aluminum foil or parafilm and label it. Store at −20°C for short storage (5 days or less) or −80°C for long term storage.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> Frozen blocks can be stored at −80°C for years. However, after freezing, the block should never be allowed to thaw. For transport, keep the blocks on dry ice.</div>
<h3 id="sec3.3">Cryosectioning, fixation, and permeabilization of the sections</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days, 1–2 h each day; 2 h incubation</div>
<p>This step describes how to section the frozen tissue and then fix and permeabilize the sections</p>
<ol start="26">
<li>Cryosectioning. Label the slides in order.</li>
<li>Set up the cryostat (Equipment Setup) and clean the stage. Use a new blade for sectioning.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep frozen blocks inside the cryostat. Do not let the blocks thaw or even soften. However, blocks stored at −80°C must be allowed to warm to −20°C prior to sectioning.</div>
<ol start="28">
<li>Cut 10–15 μm thick frozen sections. Note: The suggested cryostat temperature is between −15°C and −23°C. The section will curl if the specimen is too cold. If it is too warm, it will stick to the knife.</li>
<li>Thaw-mount the sections onto Superfrost-coated positive charged histological slides.</li>
<li>Dry the slides for approximately 10 min on a slide warmer at 37°C. Slides containing cryostat sections can be stored at −20°C to −-80°C for up to 12 months.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The specimen preparation technique greatly contributes to the histochemical properties of the tissue and hence the excellence of the subsequent histology. In this study, cryosectioning performed well in respect to the detection of intracellular peptide antigen like GFAP. The penetration of the IHC staining into the cryostat sections is key to good staining. A number of requirements regarding section thickness, staining intensity, and penetration must be carefully tested (<a href="#bib14">Wirenfeldt et al., 2003</a>). In the present study, GFAP-positive cells visualized better at 12 μm section.</div>
<div class="note">
<span class="note-title">Note:</span> The drying time is important to properly mount the tissue sample onto the slide. Insufficient drying may cause section detachment. To avoid RNA degradation, the drying time should not exceed 10 min.</div>
<ol start="31">
<li>Fixation and Permeabilization of the Sections: day 1. When staining cryostat sections stored in a freezer, thaw the slides at 20°C–22°C for 10–20 min. Procedures from here can be done in a coplin jar or slide holder.</li>
<li>Post-fix sections for 5 min with sterile 4% PFA in PBS.</li>
<li>Wash with sterile PBS, 2× 5 min each time.</li>
<li>Permeabilize sections for 2 min with 1 μg/mL RNase-free Proteinase K in PBS.</li>
<li>Wash with PBS, 1 × 5 min.</li>
<li>Re-fix sections for 5 min with 4% PFA in PBS.</li>
<li>Wash with PBS, 3× 5 min each time.</li>
<li>Incubate sections with 70% EtOH for 5 min.</li>
<li>Followed with 95% EtOH for 2 min.</li>
<li>Air dry in open humid chambers 5 min or until dry</li>
<li>Prepare hybridization probe in Hybridization Solution in dry dock at 80°C for 3 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> RNA probe is generated based on standard protocol from Promega.</div>
<p>The majority of probes work between dilutions of 1:400–1:1,000 in hybridization buffer. Using two dilutions is helpful if probe produces high background. <a href="https://www.promega.com/products/cloning-and-dna-markers/molecular-biology-enzymes-and-reagents/t3-rna-polymerase/?catNum=P2083">https://www.promega.com/products/cloning-and-dna-markers/molecular-biology-enzymes-and-reagents/t3-rna-polymerase/?catNum=P2083</a></p>
<ol start="42">
<li>Pipette 250–300 μL/slide of probe and place a parafilm coverslip over the sections. Place slides in humid chamber and incubate for 12–18 h at 65°C.</li>
<li>Day 2: Turn on water baths 37°C and 65°C</li>
<li>Remove coverslips from slides by immersing in 5× SSC. Use fine forceps if needed. Slides are now back in coplin jar.</li>
<li>In water bath at 65°C, wash sections with high-stringency wash for 30 min.</li>
<li>In water bath at 37°C, wash sections with RNase buffer for 3 × 10 min each time</li>
<li>To digest any single-stranded (unbound) RNA probe, incubate sections for 30 min at 37°C in RNase Buffer containing 20 μg/mL RNase A.</li>
<li>In water bath at 37°C, wash sections with RNase buffer for 15 min.</li>
<li>In water bath at 65°C, high-stringency wash sections for 2 × 20 min.</li>
<li>In water bath at 37°C, wash sections in 2× SSC for 15 min.</li>
<li>In water bath at 37°C, wash sections with 0.1× SSC for 15 min.</li>
<li>Wash the sections with PBT (PBS + 0.1% Tween 20) at 20°C–22°C for 15 min.</li>
<li>Cover slides for 120 min with 250–300 μL/slide blocking solution (10% NGS in PBT with 0.5 mg/mL Levamisol) in humid chamber.</li>
<li>Wash the slides with PBT at 20°C–22°C 4 × 5 min each time.</li>
<li>Cover the slides with 250–300 μL/slide anti-DIG antibody 1:2,500 with10% NGS in PBT in a humid chamber at 4°C for 12–18 h.</li>
<li>Day 3: Decant anti-DIG antibody and wash the slides in coplin jar with PBT 6 × over 5 h.</li>
<li>Wash sections with NTMT buffer 2 × 10 min each time.</li>
<li>Cover each sample slide with approximately 300 μL of BM-Purple exposure solution for 30 min to 16 h in a dark humid chamber until desired color has developed.</li>
<li>When color development is optimal (<a href="#fig1">Figure 1</a>), stop the color reaction by incubating the slides in PBS 2 × 5 min each time.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/501-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Example of combined IHC-<i>in situ</i> hybridization staining</div>
<p>Combined IHC-<i>in situ</i> hybridization protocol shows expression of (A) LCN2 (lipocalin 2) and (B) C3 (complement C3) in astrocytes</p>
<p>AMPK KO mouse astrocytes labeled with GFAP antibodies and hybridized with the RNA probes set for LCN2 and C3 shown in blue. Insets show magnifications of co-localization of LCN2 or C3 with GFAP. Scale bars, 50 μm.</p>
</figcaption>
</figure>
</li>
<li>Slides can be moved to IHC steps now</li>
</ol>
<h3 id="sec3.4">Immunohistochemistry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="61">
<li>Wash sections in PBS for 5 min.</li>
<li>Wash with 0.3% H<sub>2</sub>O<sub>2</sub> in PBS to inhibit endogenous peroxidase activity for 10 min at 20°C–22°C</li>
<li>Wash sections in PBS for 3 × 5 min each time.</li>
<li>Dry back of slide with tissue paper and add ∼325 μL anti-GFAP antibody (1:1,000) in blocking buffer and incubate for 12–18 h in humid chamber at 20°C–22°C.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Antibody concentration needs to be optimized, as its critical for the success in GFAP staining. We systematically tested different GFAP antibodies and concentrations. In our hands, the staining result and the morphology of GFAP-positive cells visualized in this study are consistent with the original description of astrocytes.</div>
<ol start="65">
<li>Day 4: Wash sections in PBS for 3 × 5 min each time.</li>
<li>Add ∼325 μL biotinylated secondary antibody (1:500) in blocking buffer and incubate 2 h in humid chamber.</li>
<li>Wash sections in PBS for 3 × 5 min each time.</li>
<li>Incubated the sections in avidin and biotinylated horseradish peroxidase macromolecular complex (ABC reagent), 10 μL/mL each of reagents A and B and 10% Triton X-100 in 1 mL PBS for 1 h.</li>
<li>Wash sections in PBS for 3 × 5 min each time.</li>
<li>Add ∼300 μL DAB per slide. Carefully observe the color development through a microscope and stop the reaction when the desired color obtained. DAB prepared according to the manufacturer’s directions. <a href="https://vectorlabs.com/dab-peroxidase-hrp-substrate.html">https://vectorlabs.com/dab-peroxidase-hrp-substrate.html</a>
</li>
<li>Wash sections in PBS for 3 × 5 min each time.</li>
<li>Mount slides with appropriately sized coverslip and Vectashield antifade mounting medium</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Any standardized protocol for imaging can be used. We image with a Leica light microscope DM2500 with a Lecia DMC6200 camera and perform image acquisition and processing in the Leica Acquisition Software (LASX) suite. All imaging data are analyzed by Image J.</div>
<h3 id="sec3.5">Quantification of the GFAP-positive astrocytes</h3>
<p>Assessing astrocyte structure and morphology requires complementary approaches to assess varying levels of detail. Currently, there are multiple tools available to analyze and quantify the extremely complex astrocyte morphology (<a href="#bib10">Parekh and Ascoli, 2013</a>) (<a href="#bib3">Kulkarni et al., 2015</a>). The choice between options is normally based upon both the level of detail needed to answer the scientific question and the resources that are available. Reproducible visualization of astrocytes in brain is essential for quantitative studies of the cellular changes in CNS diseases. Lyck et al describe a detailed procedure to identify antibodies and protocols that could preserve and visualize astrocytes and neurons for quantitative stereological studies (<a href="#bib5">Lyck et al., 2008</a>). Another reliable protocol is to perform intracellular iontophoresis of astrocytes using fluorescent Lucifer yellow dye in lightly fixed brain tissue from adult mice (<a href="#bib7">Moye et al., 2019</a>). More recently, a method utilizes a genetically encoded membrane associated fluorescent marker of astrocytes, called lymphocyte-specific protein tyrosine kinase (Lck), which is tagged with green fluorescent protein (GFP)(<a href="#bib12">Testen et al., 2020</a>).</p>
<p>For the quantitative evaluation in this study, we identified astrocytes as typical gray matter bushy cells with numerous short and ramified processes; fibrous astrocytes were typical white matter star-shaped cells with long, thin, and relatively non-ramified processes. We only considered reactive astrocytes as LCN2 or C3 positive cells that co-stained with GFAP and exhibited a clear soma with minimum overlapping branches in the fields.</p>
<p>To calculate astrocyte density, we selected three consecutive visual fields under 20× objective. We counted the number of astrocytes using Image J Cell Counter plugin.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Reactive gliosis consists of a rapid induction of gene expression after insult and earlier studies confirmed induced Lcn2 and C3 as markers of reactive astrocytes (<a href="#bib15">Zamanian et al., 2012</a>). Immunohistochemistry method for GFAP, and the use of RNA probes for LCN2 and C3 is attractive. However, for mouse brain this is problematic due to the highly nonspecific interaction and signal interference from the nonspecific binding, which generates a non-measurable signal.</p>
<p>A recent study published from our lab used a genetic loss-of-function approach producing a brain specific AMPKβ1/β2 deletion model to demonstrate that AMPK activity is required for cortical neurogenesis (<a href="#bib8">Muraleedharan et al., 2020</a>). Our study showed at 2 years of age, neuronal count, cortical thickness, and brain volume of Ampk KO mice were substantially reduced. At 24 months, the number of GFAP-positive astrocytes was greatly increased in the AMPK Knockout brain suggestive of reactive gliosis that occurs in response to chronic CNS pathology. The reactive nature of these astrocytes in AMPK-deficient brains was confirmed by combined IHC-<i>in situ</i> hybridization that showed increased expression of LCN2 (lipocalin 2) and C3 (complement C3) in GFAP-positive Astrocytes (<a href="#fig1">Figure 1</a>).</p>
<p>This labeling technique provides good signal upon hybridization under a standard microscope. The major benefit of combining these two techniques is that changes in gene expression in pathological situations or following physiological or pharmacological manipulations can be semi-quantified. Our method of identifying LCN2; GFAP double positive or C3; GFAP double positive reactive astrocytes can be integrated with immunohistochemical detection of neurons, oligodendrocytes, or microglia to investigate cell-cell interaction in the normal or pathological brain tissue sections.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<ol>
<li>Draw boxes of same area over each region and count the selected region for GFAP-positive cells using the cell counter plugin in ImageJ. Every 12<sup>th</sup> section is stained for GFAP and counted and quantified as the number of GFAP -positive cells.</li>
<li>For cell counts in cortical layers, count all cells in a 250-μm-wide columnar area from the white matter to the pial surface. Any cell where the morphology is not clearly visible should be excluded from analysis.</li>
<li>At least 5 sections in each group are analyzed. The same approach can be applied to GFAP/<i>Lcn2</i> or GFAP/<i>Comp.C3</i> combined <i>in situ</i>/IHC stained cells.</li>
</ol>
<p>Statistics was performed using Prism software (version 4) (<a href="#bib8">Muraleedharan et al., 2020</a>). For all experiments, we used one-way ANOVAs to determine statistical significance, which was defined as p ≤ 0.05.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p><i>In situ</i> hybridization provides transcriptomic information in the spatial cellular context. However, a few key limitations of the conventional ISH procedure become apparent. A disadvantage of applying <i>in situ</i> hybridization techniques is the difficulty in identifying targets that have RNA copies. One of the challenges in achieving high signal sensitivity for ISH is that the DAB reaction proceeds rapidly to completion and cannot be monitored and stopped when an optimal signal-to-noise ratio has been reached.</p>
<p>Another important concern of <i>in situ</i> hybridization is to establish whether RNA levels of a given gene can be used as proxies for the corresponding protein levels. Considering, several studies have showed that proteome and transcriptome abundances are not satisfactorily correlated to act as proxies for each other (<a href="#bib9">Nagaraj et al., 2011</a>; <a href="#bib11">Payne, 2015</a>). We have not checked the consistency between RNA level and protein level in our models, but there are several publications showed LCN2 and C3 mRNA levels correlate with protein levels in brain(<a href="#bib1">Ip et al., 2011</a>; <a href="#bib4">Lian et al., 2016</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Poor morphology of the tissue, such as cracks or folds (step 22 in <a href="#step-by-step-method-details">Step-by-step method details</a>).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>This could be due to sample freezing too slowly or maintaining the wrong temperature during cryosectioning. Collect new samples and snap-freeze them.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>RNA signal indistinguishable from background (steps 43–59).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Change reagent(s) that might be contaminated with RNase. Use filter tips when pipetting. It could be due to low hybridization temperature and/or low washing temperature. If your RNA length permits, try adding more probe. Verify and maintain correct hybridization temperature (65°C) and washing temperatures. For long probes or those with a high GC content, increase the temperature of the hybridization and wash steps to 68°C.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ip, J.P.K., Noçon, A.L., Hofer, M.J., Lim, S.L., Müller, M., and Campbell, I.L. (2011). Lipocalin 2 in the central nervous system host response to systemic lipopolysaccharide administration. J. Neuroinflammation <i>8</i>, 124. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref1">View at publisher</a></p>
<p id="bib2">Kohli, V., Nardini, D., Ehrman, L.A., and Waclaw, R.R. (2018). Characterization of Glcci1 expression in a subpopulation of lateral ganglionic eminence progenitors in the mouse telencephalon. Dev. Dyn. <i>247</i>, 222-228. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref2">View at publisher</a></p>
<p id="bib3">Kulkarni, P.M., Barton, E., Savelonas, M., Padmanabhan, R., Lu, Y., Trett, K., Shain, W., Leasure, J.L., and Roysam, B. (2015). Quantitative 3-D analysis of GFAP labeled astrocytes from fluorescence confocal images. J. Neurosci. Methods <i>246</i>, 38-51. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref3">View at publisher</a></p>
<p id="bib4">Lian, H., Litvinchuk, A., Chiang, A.C.-A., Aithmitti, N., Jankowsky, J.L., and Zheng, H. (2016). Astrocyte-microglia cross talk through complement activation modulates amyloid pathology in mouse models of Alzheimer’s disease. J. Neurosci. <i>36</i>, 577-589. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref4">View at publisher</a></p>
<p id="bib5">Lyck, L., Dalmau, I., Chemnitz, J., Finsen, B., and Schrøder, H.D. (2008). Immunohistochemical markers for quantitative studies of neurons and glia in human neocortex. J. Histochem. Cytochem. <i>56</i>, 201-221. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref5">View at publisher</a></p>
<p id="bib6">Montero, C. (2003). The antigen-antibody reaction in immunohistochemistry. J. Histochem. Cytochem. <i>51</i>, 1-4. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref6">View at publisher</a></p>
<p id="bib7">Moye, S.L., Diaz-Castro, B., Gangwani, M.R., and Khakh, B.S. (2019). Visualizing astrocyte morphology using lucifer yellow iontophoresis. J. Vis. Exp. 151. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref7">View at publisher</a></p>
<p id="bib8">Muraleedharan, R., Gawali, M.V., Tiwari, D., Sukumaran, A., Oatman, N., Anderson, J., Nardini, D., Bhuiyan, M.A.N., Tkáč, I., Ward, A.L., et al. (2020). AMPK-regulated astrocytic lactate shuttle plays a non-cell-autonomous role in neuronal survival. Cell Rep. <i>32</i>, 108092. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref8">View at publisher</a></p>
<p id="bib9">Nagaraj, N., Wisniewski, J.R., Geiger, T., Cox, J., Kircher, M., Kelso, J., Pääbo, S., and Mann, M. (2011). Deep proteome and transcriptome mapping of a human cancer cell line. Mol. Syst. Biol. <i>7</i>, 548. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref9">View at publisher</a></p>
<p id="bib10">Parekh, R. and Ascoli, G.A. (2013). Neuronal morphology goes digital: a research hub for cellular and system neuroscience. Neuron <i>77</i>, 1017-1038. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref10">View at publisher</a></p>
<p id="bib11">Payne, S.H. (2015). The utility of protein and mRNA correlation. Trends Biochem. Sci. <i>40</i>, 1-3. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref11">View at publisher</a></p>
<p id="bib12">Testen, A., Kim, R., and Reissner, K.J. (2020). High-resolution three-dimensional imaging of individual astrocytes using confocal microscopy. Curr. Protoc. Neurosci. <i>91</i>, e92. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref12">View at publisher</a></p>
<p id="bib13">Toresson, H., Mata de Urquiza, A., Fagerström, C., Perlmann, T., and Campbell, K. (1999). Retinoids are produced by glia in the lateral ganglionic eminence and regulate striatal neuron differentiation. Development <i>126</i>, 1317-1326. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref13">View at publisher</a></p>
<p id="bib14">Wirenfeldt, M., Dalmau, I., and Finsen, B. (2003). Estimation of absolute microglial cell numbers in mouse fascia dentata using unbiased and efficient stereological cell counting principles. Glia <i>44</i>, 129-139. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref14">View at publisher</a></p>
<p id="bib15">Zamanian, J.L., Xu, L., Foo, L.C., Nouri, N., Zhou, L., Giffard, R.G., and Barres, B.A. (2012). Genomic analysis of reactive astrogliosis. J. Neurosci. <i>32</i>, 6391-6410. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00082-4/sref15">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Biplab DasGupta (<a href="mailto:Biplab.Dasgupta@cchmc.org">biplab.dasgupta@cchmc.org</a> ).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate/analyze [datasets/code].</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (NIH) grants R01 NS075291 and R01 NS099162 (to B.D.) and NIH grant R01 NS088529 (to R.W.).</p>
<h3>Author Contributions</h3>
<p>D.N. performed the experiments; R.M., R.W., and B.D. wrote the method; R.W. and B.D. supervised the work.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>