<article>
<section>
<h1 id="header">Preparation and culture of bone marrow-derived macrophages from mice for functional analysis</h1>
<p><time datetime="2021-01-08">Published: January 8, 2021</time></p>
<p>Gotaro Toda,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> Toshimasa Yamauchi,<sup><a href="#aff1">1</a></sup> Takashi Kadowaki,<sup><a href="#aff3">3</a></sup> and Kohjiro Ueki<sup><a href="#aff2">2</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Diabetes and Metabolic Diseases, Graduate School of Medicine, the University of Tokyo, Tokyo, Japan</p>
<p id="aff2"><sup>2</sup>Department of Molecular Diabetic Medicine, Diabetes Research Center, Research Institute, National Center for Global Health and Medicine, Tokyo, Japan</p>
<p id="aff3"><sup>3</sup>Toranomon Hospital, Tokyo, Japan</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:gtoda-tky@umin.ac.jp">gtoda-tky@umin.ac.jp</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:uekik@ri.ncgm.go.jp">uekik@ri.ncgm.go.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100246">10.1016/j.xpro.2020.100246</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The assessment of macrophage function has been a topic of intense discussion due to multiple subtypes. This protocol describes the collection of bone marrow cells from the femur and tibia of mice, differentiation into bone marrow-derived macrophages (BMDM cells), and sampling from cultures. This protocol focuses on the efficient preparation of BMDM cells, providing a way to assess the function of macrophages.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib5">Toda et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Fast and efficient collection of bone marrow from mice</li>
<li>Differentiation into BMDM that are &gt;90% positive for CD11b and F4/80</li>
<li>Culture of BMDM for <i>in vitro</i> functional analysis of RNA and protein</li>
<li>Cells can be assessed without assuming polarization to M1/M2 subtype</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Prepare mice</h3>
<div class="note">
<span class="note-title">Note:</span> Mice should be available sufficiently before the <i>in vitro</i> experiment. Although we have not experienced marked differences in the viability of harvested cells among differently aged mice between 6–12 weeks of age or between male/female mice, the mice used should be of the same age and sex in the same experiment.</div>
<p>Mice used in this protocol can be fasted or fed ad libitum at time of collection of bone marrow cells. Cell viability should be evaluated in relation to wild type mice in mice with defects in hematopoietic cells.</p>
<h3 id="sec1.2">Prepare reagents and equipment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="note">
<span class="note-title">Note:</span> 1 week before <i>in vitro</i> experiment. 1 h before collection of bone marrow cells.</div>
<ol>
<li>Add 10% FBS (heat inactivated) and 1% penicillin/streptomycin to DMEM high glucose (DMEM 10% FBS) and store at 4°C until preparation of the bone marrow culture medium.</li>
<li>Keep DMEM (10% FBS) at 37°C. Add CSF-1 to obtain bone marrow culture medium after collection of bone marrow cells. Freshly prepare before harvesting bone marrow (<a href="#bib8">Zhang et al., 2008</a>).</li>
<li>Prepare ice cold PBS (10 mL/ mouse) in 10 mL syringe with 23G needle, culture plate (one 12 well plate/ mouse), scissors, and forceps.</li>
<li>Bring mice on site after preparation.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>PE Rat Anti-CD11b</td>
<td>BD</td>
<td>553311</td>
</tr>
<tr>
<td>APC anti-mouse F4/80 antibody</td>
<td>BioLegend</td>
<td>123116</td>
</tr>
<tr>
<td>7-AAD</td>
<td>BD</td>
<td>559925</td>
</tr>
<tr>
<td>Phospho-Akt (Ser473) Rabbit mAb</td>
<td>Cell Signaling</td>
<td>9271</td>
</tr>
<tr>
<td>Phospho mTOR antibody</td>
<td>Cell Signaling</td>
<td>2971</td>
</tr>
<tr>
<td>ECL anti-rabbit IgG horseradish peroxidase lined whole antibody</td>
<td>GE</td>
<td>NA934-1ML</td>
</tr>
<tr>
<td>GAPDH rabbit monoclonal antibody</td>
<td>Cell Signaling</td>
<td>2118</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Recombinant mouse IL-13</td>
<td>R and D</td>
<td>413-ML-005</td>
</tr>
<tr>
<td>Recombinant mouse CSF-1</td>
<td>Gibco</td>
<td>PMC2044</td>
</tr>
<tr>
<td>DMEM (high glucose)</td>
<td>Gibco</td>
<td>5796</td>
</tr>
<tr>
<td>Stain buffer</td>
<td>BD</td>
<td>554656</td>
</tr>
<tr>
<td>FBS</td>
<td>Biowest</td>
<td>S1650</td>
</tr>
<tr>
<td>Penicillin streptomycin</td>
<td>Gibco</td>
<td>10378016</td>
</tr>
<tr>
<td>PBS tablets</td>
<td>Takara</td>
<td>T900</td>
</tr>
<tr>
<td>0.05% Trypsin-EDTA</td>
<td>Gibco</td>
<td>25300-054</td>
</tr>
<tr>
<td>Cell lysis buffer</td>
<td>CST</td>
<td>9803</td>
</tr>
<tr>
<td>Complete protease inhibitor cocktail</td>
<td>Roche</td>
<td>11697498001</td>
</tr>
<tr>
<td>4× Laemmli sample buffer</td>
<td>Bio-Rad</td>
<td>1610747</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>C57BL6/J</td>
<td>Jackson Laboratory</td>
<td>664</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>IL-10 Fw GCTCTTACTGACTGGCATGAG</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>IL-10 Rv CGCAGCTCTAGGAGCATGTG</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>TNF-α Fw CTGAACTTCGGGGTGATCGG</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>TNF-α Rv GGCTTGTCACTCGAATTTTGAGA</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Arg1 Fw CTCCAAGCCAAAGTCCTTAGAG</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Arg1 Rv GGAGCTGTCATTAGGGACATCA</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Prism 7</td>
<td>GraphPad</td>
<td>N/A</td>
</tr>
<tr>
<td>MS Excel</td>
<td>Microsoft</td>
<td>N/A</td>
</tr>
<tr>
<td>Photoshop CC</td>
<td>Adobe</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cyclophillin probe</td>
<td>Applied Biosystems</td>
<td>4316034</td>
</tr>
<tr>
<td>Cell strainer (100 μm)</td>
<td>Falcon</td>
<td>352360</td>
</tr>
<tr>
<td>Lysing buffer</td>
<td>BD</td>
<td>555899</td>
</tr>
<tr>
<td>Cell scraper</td>
<td>IWAKI</td>
<td>9000-220-CS</td>
</tr>
<tr>
<td>RNeasy mini kit</td>
<td>QIAGEN</td>
<td>74106</td>
</tr>
<tr>
<td>Round bottom polystyrene tubes</td>
<td>Falcon</td>
<td>352058</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<thead>
<tr>
<th>Bone marrow culture medium</th>
<th>Final concentration (mM or μM)</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>N/A</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
</tr>
<tr>
<td>Penicillin streptomycin</td>
<td>1%</td>
</tr>
<tr>
<td>M-CSF1</td>
<td>1 ng/mL (test multiple concentrations)</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Collection of bone marrow cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<div class="note">
<span class="note-title">Note:</span> Day 0</div>
<p>This step prepares the bone marrow cells including progenitors which will differentiate into bone marrow-derived macrophages after culture. The time it takes to start of culture in the bone marrow culture medium should be minimal to maximize cell yield and to avoid contamination.</p>
<ol>
<li>Euthanize mouse by an appropriate method approved by the supervising institution. Appropriately remove excess blood in the following steps.</li>
<li>Soak with 70% ethanol. (Also see <a href="#sec6.1">Troubleshooting problem 1</a>)</li>
<li>Remove skin and expose lower limb.</li>
<li>Cut major muscles near the base of the lower limb to expose the hip joint.</li>
<li>Cut femur near the base (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Illustration describing steps 1–5.</div>
</figcaption>
</figure>
</li>
<li>Cut Achilles tendon and remove major muscles around tibia.</li>
<li>Remove major muscles around femur (<a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Illustration describing steps 6 and 7.</div>
</figcaption>
</figure>
</li>
<li>Cut at knee joint.</li>
<li>Remove major muscles around tibia, and remove fibula (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Illustration describing steps 8 and 9.</div>
</figcaption>
</figure>
</li>
<li>Remove the epiphyses of femur and tibia so that the bone marrow can be accessed from the ends with a 23G needle. This can be confirmed by poking the ends with a needle (Also see <a href="#sec6.3">Troubleshooting problem 2</a>).</li>
<li>Wash the bone marrow into a 50 mL tube by slowly injecting approximately 2–3 mL PBS per bone. The bone marrow will come out from the other end. Make sure the PBS does not backflow, to avoid contamination. Keep the bone marrow on ice.</li>
<li>Repeat with other lower limb (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Illustration describing steps 10–12.</div>
</figcaption>
</figure>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For collection of bone marrow from 5 mice or more, or if collection takes more than 1 h, use DMEM without FBS for washing bone marrow.</div>
<ol start="13">
<li>Centrifuge bone marrow suspension at 200 × <i>g</i>, 5 min at 4°C.</li>
<li>Decant or aspirate PBS.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> To remove erythrocytes resuspend in 1× lyse buffer on ice, and let sit for 3 min. Add 10 mL DMEM (10% FBS 1% penicillin streptomycin) and centrifuge suspension at 200 × <i>g</i>, 5 min at 4°C, and repeat if necessary.</div>
<h3 id="sec3.2">Differentiation into bone marrow-derived macrophages</h3>
<ol start="15">
<li>Resuspend bone marrow (approx. 1 × 10<sup>7</sup> live cells per mouse) in 1 mL bone marrow culture medium, and filter through a 100 μm cell strainer.</li>
<li>Pipette suspension (approx. 1 × 10<sup>7</sup> live cells) into total volume of bone marrow culture medium (approx. 12 mL for 12 well plate). 6 well plates and 24 well plates can also be used.</li>
<li>Add bone marrow culture medium to wells (1 mL/well for 12 well plate). Culture at 37°C 5% CO<sub>2</sub>.</li>
<li>Add half this volume of bone marrow culture medium on day 4 (e.g., 500 μL/well for 12 well plate). Adherent cells should show 1–2 protrusions at low magnification.</li>
</ol>
<h3 id="sec3.3">Preparation for <i>in vitro</i> experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<div class="note">
<span class="note-title">Note:</span> Day 6</div>
<p>This step describes the preparation for <i>in vitro</i> stimulation using BMDMs. FBS contains signaling peptides etc. that may obscure, for example, detection of phosphorylation in response to stimulation of interest. Therefore, a period of serum-free culture before stimulation is recommended in such experiments. Overnight culture in serum-free medium does not have a marked impact on cell viability, but baseline Akt phosphorylation is suppressed which makes detection of phosphorylation in response to <i>in vitro</i> stimulation clearer.</p>
<ol start="19">
<li>Wash cells with sterile PBS, and change to DMEM 10% FBS, and continue culture at 37°C 5% CO<sub>2</sub>.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> For detection of phosphorylation signals etc., change to DMEM without FBS at this point.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<div class="note">
<span class="note-title">Note:</span> Day 7 (2 h before <i>in vitro</i> experiment)</div>
<ol start="20">
<li>Change medium to DMEM without FBS if not already changed.</li>
</ol>
<h3 id="sec3.4">Sampling from bone marrow-derived macrophage cultures</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min to several hours</div>
<p>This step describes sampling from BMDM cultures. To minimize altered signaling events, or gene expression, sampling directly from the culture plate is recommended.</p>
<ol start="21">
<li>Change medium to medium containing stimulation of interest.</li>
<li>After stimulation, wash cells twice with PBS, and aspirate.</li>
<li>For creating a cell suspension, add 0.05% trypsin-EDTA (approx. 100 μL/well for 12 well plate) and incubate at 37°C for 3 min. Add DMEM (10% FBS, 1 mL/well for 12 well plate), dislodge the cells using a cell scraper, and transfer to a 5 mL round bottom tube.</li>
<li>Centrifuge at 200 × <i>g</i>, 4°C for 3 min.</li>
<li>Suspend cells in 1 mL FACS buffer for counting, and subsequent analysis.</li>
<li>If sampling of protein or RNA takes place later, freeze the washed culture plate with liquid nitrogen, and keep in −80°C freezer. In the following steps, add extraction buffer directly to the wells, and pipette repeatedly until no adherent cells remain. Optionally, a cell scraper may be used to dislodge the cells.</li>
<li>For extracting protein samples for western blot, add cell lysis buffer supplemented with 1× protease cocktail to the wells (approx. 100 μL/well for a 12 well plate), and transfer to a 1.5 mL sampling tube.</li>
<li>Sonicate for 5 min at 4°C, and vortex.</li>
<li>Centrifuge for 10 min at 10,000 × <i>g</i> and transfer supernatant to a 1.5 mL sampling tube. Avoid aspirating or contacting the sediment (Trouble shooting problem 3). 100–200 μg of protein is typically extracted.</li>
<li>Add 1/3 volume of 4× Laemmli buffer, and boil for 10 min.</li>
<li>Keep at −20°C until SDS-PAGE.</li>
<li>For extracting RNA using RNeasy mini kit, add 350 μL buffer RLT to the wells. Final elution should use 50–100 μL TE. 5–10 μg RNA is typically extracted. Keep at −80°C until gene expression analysis.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>From three male C57BL6/J mice at 8 weeks of age weighing 24.7 ± 0.8 g, (1.6 ± 0.2) × 10<sup>7</sup> live bone marrow cells were isolated. During culture, (1.2 ± 0.1) × 10<sup>5</sup> cells were adherent at day 4, and (1.7 ± 0.1) × 10<sup>5</sup> cells were adherent at day 7. Examples of gene expresion analyses using FACS, RT-PCR, and western blot are shown (<a href="#fig5">Figures 5</a>, <a href="#fig6">6</a>, and <a href="#fig7">7</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. FACS analysis of BMDM cell suspension</div>
<p>Over 90% of the cells were positive for CD11b and F4/80, which are markers for murine macrophages (<a href="#bib7">Wynn et al., 2013</a>).</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Gene expression assessed by RT-PCR using RNA extracted from BMDM cultures stimulated overnight with 10 ng/mL LPS and 50 ng/mL IL-13</div>
<p>Genes known to be expressed by M1 and M2 macrophages could be detected (<a href="#bib3">Martinez and Gordon, 2014</a>; <a href="#bib7">Wynn et al., 2013</a>).</p>
</figcaption>
</figure>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/339-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Akt phosphorylation assessed by western blot of protein extracted from BMDM cultures stimulated with 50 ng/mL IL-13</div>
<p>Signaling in response to IL-13 has been described by others (<a href="#bib1">Brown Lobbins et al., 2018</a>; <a href="#bib2">Jiang et al., 2000</a>).</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The cultures are viable for at least 3–4 days after differentiation, but have little or no replicating potential after differentiation, and therefore the <i>in vitro</i> experiment should conclude without passage.</p>
<p>Although the BMDM cultures generated using this protocol are useful for analyzing genes known to be expressed by multiple macrophage subtypes, they cannot be used to assess different macrophage functions between tissues (<a href="#bib5">Toda et al., 2020</a>; <a href="#bib6">Weischenfeldt and Porse, 2008</a>). Additionally, this protocol only covers the assessment of BMDM cells before polarization into different macrophage subsets as described by others (<a href="#bib7">Wynn et al., 2013</a>). The cells may be further cultured with IFN-γ, IL-4 etc. to assess the function of polarized macrophage subsets although one should be aware that functions of such strongly polarized macrophages only limitedly reflect the <i>in vivo</i> variety of macrophages (<a href="#bib4">Murray et al., 2014</a>). As assessment of macrophage function using unpolarized BMDM cultures generated as described in this protocol provides a view of cellular function without narrowing the scope of the analysis to a certain polarization, it provides candidates of cellular functions some of which in combination with <i>in vivo</i> analyses may subsequently prove to be of importance in physiological or pathological conditions. The BMDM cultures described in this protocol, as in all immune cell cultures do not completely reflect the variety of immune cells <i>in vivo</i>, and future development and utilization of single-cell techniques may be key to further analysis of immune cell function in high spatiotemporal resolution.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Generated cells are sparse.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>BMDM cultures generated should be approximately 75% confluent. If cells are sparser, the amount of collected bone marrow may be inadequate perhaps due to cutting the edges of the femur and tibia too much. The edges of the bone should be cut ∼0.5 mm at a time to minimize loss. Additionally, some mouse models may have smaller amounts of bone marrow cells than others, and may require isolation of more bone marrow to acquire the same amount of BMDM cells as wild type mice.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Generated cells are not viable.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>BMDM cultures will become nonadherent when contaminated. Mice should be thoroughly soaked with 70% ethanol before exposing the bone marrow. Feces and fur are inevitably close to the site of dissection, and exposed bone must not touch them. If possible, the isolation of bone marrow may be done in sterile conditions.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Western blot resulted in bands with too much background.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>As the volume of the lysate is typically &lt;100 μL, this may occur when the supernatant of the lysate after centrifugation is inadequately aspirated, particularly when the sediment is aspirated. The steps following sonication should be done with care.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Brown Lobbins, M.L., Shivakumar, B.R., Postlethwaite, A.E., and Hasty, K.A. (2018). Chronic exposure of interleukin-13 suppress the induction of matrix metalloproteinase-1 by tumour necrosis factor α in normal and scleroderma dermal fibroblasts through protein kinase B/Akt. Clin. Exp. Immunol. <i>191</i>, 84-95.</p>
<p id="bib2">Jiang, H., Harris, M.B., and Rothman, P. (2000). IL-4/IL-13 signaling beyond JAK/STAT. J. Allergy Clin. Immunol. <i>105</i>, 1063-1070.</p>
<p id="bib3">Martinez, F.O. and Gordon, S. (2014). The M1 and M2 paradigm of macrophage activation: time for reassessment. F1000Prime Rep. <i>6</i>, 13.</p>
<p id="bib4">Murray, P.J., Allen, J.E., Biswas, S.K., Fisher, E.A., Gilroy, D.W., Goerdt, S., Gordon, S., Hamilton, J.A., Ivashkiv, L.B., Lawrence, T., et al. (2014). Macrophage activation and polarization: nomenclature and experimental guidelines. Immunity <i>41</i>, 14-20.</p>
<p id="bib5">Toda, G., Soeda, K., Okazaki, Y., Kobayashi, N., Masuda, Y., Arakawa, N., Suwanai, H., Masamoto, Y., Izumida, Y., Kamei, N., et al. (2020). Insulin- and lipopolysaccharide-mediated signaling in adipose tissue macrophages regulates postprandial glycemia through Akt-mTOR activation. Mol. Cell <i>79</i>, 43-53.e4.</p>
<p id="bib6">Weischenfeldt, J. and Porse, B. (2008). Bone marrow-derived macrophages (BMM): isolation and applications. CSH Protoc. <i>2008</i>.</p>
<p id="bib7">Wynn, T.A., Chawla, A., and Pollard, J.W. (2013). Macrophage biology in development, homeostasis and disease. Nature <i>496</i>, 445-455.</p>
<p id="bib8">Zhang, X., Goncalves, R., and Mosser, D.M. (2008). The isolation and characterization of murine macrophages. Curr. Protoc. Immunol. <i>Chapter 14</i>. Unit 14 11.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Kohjiro Ueki (<a href="mailto:uekik@ri.ncgm.go.jp">uekik@ri.ncgm.go.jp</a>).</p>
<h4>Materials availability</h4>
<p>This protocol does not include unique materials.</p>
<h4>Data and code availability</h4>
<p>This protocol does not include data sets.</p>
<h3>Acknowledgments</h3>
<p>This work has been funded by <a href="https://doi.org/10.13039/501100001691">Japan Society for the Promotion of Science</a> grants 16K15488 (to K.U.) and 17K16141 and 19K17954 (to G.T.); <a href="https://doi.org/10.13039/100009619">Japan Agency for Medical Research and Development</a> grant 16fk0210204h0002 (to K.U.); and grants from the <a href="https://doi.org/10.13039/501100012627">Japan Diabetes Foundation</a>, the <a href="https://doi.org/10.13039/501100013236">MSD Life Science Foundation</a>, and the <a href="https://doi.org/10.13039/501100008880">Kanae Foundation for the Promotion of Medical Science</a> (to G.T.). Hiroko Uchida holds the copyright of the graphical abstract and schemes. Please contact "me@uchidahiroko.com" for secondary use of these graphics.</p>
<h3>Author Contributions</h3>
<p>G.T. performed the experiments and wrote the protocol. T.Y., T.K., and K.U. helped in planning the experiments.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>