<article>
<section>
<h1 id="header">Protocol for studying microbiome impact on host energy and reproduction in <i>Drosophila</i>
</h1>
<p><time datetime="2022-03-25">Published: March 25, 2022</time></p>
<p>Yulia Gnainsky,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup> Maxim Itkin,<sup><a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup> Tevie Mehlman,<sup><a href="#aff2">2</a></sup> Alexander Brandis,<sup><a href="#aff2">2</a></sup> Sergey Malitsky,<sup><a href="#aff2">2</a></sup> and Yoav Soen<sup><a href="#aff1">1</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biomolecular Sciences, Weizmann Institute of Science, Rehovot 7670001, Israel</p>
<p id="aff2"><sup>2</sup>Life Sciences Core Facilities, Weizmann Institute of Science, Rehovot 7670001, Israel</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yulia.gnainsky@weizmann.ac.il">yulia.gnainsky@weizmann.ac.il</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:maxim.itkin@weizmann.ac.il">maxim.itkin@weizmann.ac.il</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101253">10.1016/j.xpro.2022.101253</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p><i>Drosophila</i> gut microbiome in flies has been shown to have a systemic influence on energy production by the host and the energetic investment in growth and reproduction. Here we describe a protocol for studying the mechanisms responsible for this remote regulation by gut bacteria. This protocol enables whole-body and ovary-specific quantification of energy-storing molecules as well as identification of host metabolites and pathways that are regulated by gut microbiome-derived factors. Similar procedures are applicable to additional treatments and genetic manipulations.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref4">Gnainsky et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for studying <i>Drosophila</i> gut bacteria impact on host metabolism and reproduction</li>
<li>Preparation of germ-free flies and evaluation of oocyte development</li>
<li>An assay for sensitive detection and quantification of energy-storing molecules</li>
<li>Metabolomic analysis and identification of altered metabolic pathways</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1520-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Raising flies for egg collection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–18 days</div>
<ol>
<li>Transfer ∼120 flies wild type (∼1:1 females:males) into bottles with standard food (see the recipe in <a href="#tbl1">Table 1</a>) and let them lay eggs for 3–4 days at 25°C in a 12 h light:12 h dark cycle controlled incubator.
<table id="tbl1">
<caption>Table 1. Fly food</caption>
<thead>
<tr>
<th>Ingredient</th>
<th>Final concentration</th>
<th>Amount for 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agar</td>
<td>0.5% (w/v)</td>
<td>5 g</td>
</tr>
<tr>
<td>Yeast</td>
<td>1.2% (w/v)</td>
<td>12 g</td>
</tr>
<tr>
<td>Yellow cornmeal</td>
<td>5% (w/v)</td>
<td>50 g</td>
</tr>
<tr>
<td>Molasses</td>
<td>6% (v/v)</td>
<td>60 mL</td>
</tr>
<tr>
<td>methyl paraben (10% in 96%ETOH w/v)</td>
<td>1.2% (v/v)</td>
<td>12 mL</td>
</tr>
<tr>
<td>Propionic acid</td>
<td>0.375% (v/v)</td>
<td>3.75 mL</td>
</tr>
</tbody>
</table>
</li>
<li>Remove the flies and keep the bottles at 25°C in a 12 h light: 12 h dark cycle controlled incubator for 15–18 days to obtain mature adult flies at age 5–8 days after eclosion.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For studying the effect of gut microbiota on reproduction functions, flies, free of Wolbachia (endosymbiont), would be recommended.</div>
<div class="note">
<span class="note-title">Note:</span> Adjust the starting number of flies when using mutant/transgenic strains.</div>
<h3 id="sec1.2">Preparation of sterile fly food media</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<div class="note">
<span class="note-title">Note:</span> For the following experiments we used 1.2% yeast diet (<a href="#tbl1">Table 1</a>), in order to emphasize the contribution of metabolites derived from the bacteria and not from the food (“<a href="#materials-and-equipment">materials and equipment</a>”, <a href="#troubleshooting">troubleshooting 1</a>). Some of the microbiome removal effects are diminished in the rich diet (<a href="#bib2">Elgart et al., 2016</a>).</div>
<ol start="3">
<li>Sterilize all the required equipment (vials, plugs, paint brush, etc.) by autoclave or UV light.</li>
<li>For 1 L of fly food add 5 g agar into 500 mL tap water. Heat and stir until water is boiling and agar is dissolved.</li>
<li>Add 12 g yeast and 400 mL water. Stir and boil for 5–10 min.</li>
<li>Add 50 g yellow cornmeal and 60 mL molasses and water to a total volume of 1 L. Bring the media to a boil and cook for not less than 10 min, stirring constantly.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Next steps of food preparation are done in sterile environment in a biological safety cabinet.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Autoclave the food.</div>
<ol start="7">
<li>Transfer the food to the biological safety cabinet, allow it to cool to 60°C and add 12 mL of methyl paraben solution (10% w/v in 96% ethanol, ETOH) and 3.75 mL propionic acid.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this point, sterile compounds, such as inhibitors, vitamins or other treatments can be added to the food if needed.</div>
<ol start="8">
<li>Pour 9–10 mL of fly food into sterilized vials and cool the food to ∼25°C before plugging the vials with sterilized plugs.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The vials can be kept in the biological safety cabinet overnight (∼18 h) for immediate use or at 4°C for one week.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Agar-Agarite</td>
<td>Hispanagar</td>
<td>Cat#A700</td>
</tr>
<tr>
<td>Yeast</td>
<td>Bravo</td>
<td>N/A</td>
</tr>
<tr>
<td>Yellow cornmeal (Polenta Bergamasca)</td>
<td>Nicoli</td>
<td> </td>
</tr>
<tr>
<td>Molasses</td>
<td>Grandma's Original Unsulphured Molasses</td>
<td>Cat#71942</td>
</tr>
<tr>
<td>96% Ethanol (ETOH)</td>
<td>Gadot group</td>
<td>N/A</td>
</tr>
<tr>
<td>Methylparaben (10% in 96% ETOH w/v)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Propionic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat#P5561</td>
</tr>
<tr>
<td>Formic acid 98%–100% for LC-MS LiChropur ®</td>
<td>Merck</td>
<td>Cat#5.33002</td>
</tr>
<tr>
<td>Acetic acid 100% for LC-MS LiChropur™</td>
<td>Merck</td>
<td>Cat#5.33001</td>
</tr>
<tr>
<td>Acetonitrile hypergrade for LC-MS LiChrosolv®</td>
<td>Supelco/Merck</td>
<td>Cat#1.00029</td>
</tr>
<tr>
<td>2-Propanol hypergrade for LC-MS LiChrosolv®</td>
<td>Merck</td>
<td>Cat#1.02781</td>
</tr>
<tr>
<td>Ammonium acetate</td>
<td>Honeywell/Fluka/Merck</td>
<td>Cat#14267</td>
</tr>
<tr>
<td>Ammonium carbonate for HPLC</td>
<td>Honeywell/Fluka/Merck</td>
<td>Cat#74415</td>
</tr>
<tr>
<td>Ammonium bicarbonate for LC-MS</td>
<td>Honeywell/Fluka/Merck</td>
<td>Cat#40867</td>
</tr>
<tr>
<td>Ammonia solution 25% for analysis</td>
<td>Merck</td>
<td>Cat#1.05432</td>
</tr>
<tr>
<td>Methanol hypergrade for LC-MS LiChrosolv® (MeOH)</td>
<td>Supelco/Merck</td>
<td>Cat#106035</td>
</tr>
<tr>
<td>17:0 Phosphatidylcholine</td>
<td>Avanti/Merck</td>
<td>Cat#850360P</td>
</tr>
<tr>
<td>Ceramide/Sphingoid Internal Standard Mixture I</td>
<td>Avanti/Merck</td>
<td>Cat#LM6002</td>
</tr>
<tr>
<td>d5-TG Internal Standard Mixture I</td>
<td>Avanti/Merck</td>
<td>Cat#LM6000</td>
</tr>
<tr>
<td>17:0 Phosphatidylethanolamine</td>
<td>Avanti/Merck</td>
<td>Cat#830756P</td>
</tr>
<tr>
<td>Palmitic acid-13C</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#605573</td>
</tr>
<tr>
<td>Tert-Butyl methyl ether for liquid chromatography LiChrosolv® (TMBE)</td>
<td>Supelco/Merck</td>
<td>Cat#1.01845</td>
</tr>
<tr>
<td>Formaldehyde solution</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#F8775</td>
</tr>
<tr>
<td>Cell Free Amino Acid Mixture - <sup>13</sup>C,<sup>15</sup>N</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#767964</td>
</tr>
<tr>
<td>Adenosine-<sup>13</sup>C<sub>10</sub> 5-triphosphate disodium salt</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#710695</td>
</tr>
<tr>
<td>Adenosine 5′-triphosphate disodium salt hydrate</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#A6419</td>
</tr>
<tr>
<td>Adenosine 5′-diphosphate sodium salt</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#A2754</td>
</tr>
<tr>
<td>Formaldehyde solution for molecular biology, 36.5–38%</td>
<td>Sigma-Aldrich/Merck</td>
<td>Cat#F8775</td>
</tr>
<tr>
<td>Sodium Hypochlorite 6% (Bleach)</td>
<td>Bio-Lab Ltd.</td>
<td>Cat#193603</td>
</tr>
<tr>
<td>Dulbecco’s phosphate buffered saline 1× (PBS)</td>
<td>Biological Industries</td>
<td>Cat#02-020-1A</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Vectashield with DAPI</td>
<td>Vector labs</td>
<td>Cat#H-1200</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>D. melanogaster: OrR</td>
<td>Gift from Prof. Adi Salzberg</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>16S rDNA 8FAGAGTTTGATCCTGGCTCAG</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>16S rDNA E939R CTTGTGCGGGCCCCCGTCAATTC</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Graphpad Prism</td>
<td>GraphPad software</td>
<td><a href="http://www.graphpad.com/">http://www.graphpad.com/</a></td>
</tr>
<tr>
<td>UNIFI Version 1.9.2</td>
<td>Waters Corp.</td>
<td>N/A</td>
</tr>
<tr>
<td>TraceFinder</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>MetaboAnalyst 4.0</td>
<td><a href="#bib1">Chong et al. (2018)</a></td>
<td><a href="https://www.metaboanalyst.ca/">https://www.metaboanalyst.ca/</a></td>
</tr>
<tr>
<td>MassLynx 4.1 with TargetLynx</td>
<td>Waters Corp.</td>
<td>N/A</td>
</tr>
<tr>
<td>MATLAB</td>
<td>MathWorks, Inc</td>
<td><a href="https://www.mathworks.com/">https://www.mathworks.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Embryo collection cage</td>
<td>Genesee Scientific</td>
<td>Cat#59-101</td>
</tr>
<tr>
<td>FlyStuff Flypad (CO<sub>2</sub> anesthetizing apparatus 1 flowbed)</td>
<td>Genesee Scientific</td>
<td>Cat#59-119</td>
</tr>
<tr>
<td>Narrow Drosophila vials 25 × 95 mm</td>
<td>RUNLAB</td>
<td>Cat#8730RS</td>
</tr>
<tr>
<td>Cell strainer (100 μm)</td>
<td>SPL Life Sciences</td>
<td>Cat#93100</td>
</tr>
<tr>
<td>Yeast Extract–Peptone–Dextrose (YPD) plates</td>
<td>Hy laboratories</td>
<td>Cat#PD220</td>
</tr>
<tr>
<td>wwPTFE 0.2 μM 47 MM Disc 50/PK</td>
<td>Pall</td>
<td>Cat#60539</td>
</tr>
<tr>
<td>Stainless Steel Beads 3.2 mm</td>
<td>Ornat</td>
<td>Cat#ADV-SSB32</td>
</tr>
<tr>
<td>Eppendorf® Safe-Lock microtubes, PCR clean, 2 mL</td>
<td>Eppendorf/Merck</td>
<td>Cat#Z606324</td>
</tr>
<tr>
<td>Eppendorf® Safe-Lock microtubes, PCR clean, 1.5 mL</td>
<td>Eppendorf/Merck</td>
<td>Cat#Z606316</td>
</tr>
<tr>
<td>HPLC glass vials Chromacol FISV(A), 300 μL, Fixed insert</td>
<td>Thermo Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>Pestle motor mixer</td>
<td>Argos Technologies</td>
<td>Cat#A0001</td>
</tr>
<tr>
<td>Thermo Mixer C</td>
<td>Eppendorf</td>
<td>Cat#01004114</td>
</tr>
<tr>
<td>Strata-XL-AW 100 mm, 30 mg/mLs</td>
<td>Phenomenex</td>
<td>Cat#8B-S051-TAK</td>
</tr>
<tr>
<td>SPE vacuum manifold 24-port</td>
<td>Grace Alltech</td>
<td>Cat#210224</td>
</tr>
<tr>
<td>Xevo TQ-S triple quadrupole mass spectrometer</td>
<td>Waters Corp</td>
<td>N/A</td>
</tr>
<tr>
<td>Acquity I-class UPLC system</td>
<td>Waters Corp</td>
<td>N/A</td>
</tr>
<tr>
<td>UPLC HSS T3 column 100 × 2.1 mm 1.8-μm</td>
<td>Waters Corp</td>
<td>N/A</td>
</tr>
<tr>
<td>UPLC HSS T3 pre-column 5 × 2.1 mm 1.8-μm</td>
<td>Waters Corp</td>
<td>N/A</td>
</tr>
<tr>
<td>SuperFrost Microscope slides, Cut edges, White</td>
<td>Menzel Glaser Gmbh</td>
<td>Cat#AA00008032E</td>
</tr>
<tr>
<td>Cover Slips 24 × 24 mm #1</td>
<td>Menzel Glaser Gmbh</td>
<td>N/A</td>
</tr>
<tr>
<td>Dumont #5 Dumostar Biology Tip Forceps</td>
<td>Fine Science tools (FST)</td>
<td>Cat#11295-10</td>
</tr>
<tr>
<td>Dumont #5 Dumoxel Standard Tip Forceps</td>
<td>Fine Science tools (FST)</td>
<td>Cat#11251-30</td>
</tr>
<tr>
<td>Moria MC1 Pin Holder</td>
<td>Fine Science tools (FST)</td>
<td>Cat#26016-12</td>
</tr>
<tr>
<td>Stainless Steel Minutien Pins - 0.1 mm Diameter</td>
<td>Fine Science tools (FST)</td>
<td>Cat#26002-10</td>
</tr>
<tr>
<td>Homogenizer (grinder)</td>
<td>RETSCH</td>
<td>Cat#MM400</td>
</tr>
<tr>
<td>Kimwipes (disposable wipes)</td>
<td>Kimtech Science</td>
<td>N/A</td>
</tr>
<tr>
<td>Milli-Q® Water Purification System supplemented with LC-Pak® Polisher</td>
<td>MERCK</td>
<td>N/A</td>
</tr>
<tr>
<td>Rocking shaker DRS-12</td>
<td>Elmi</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>System setup for quantification of ATP and ADP</p>
<ul>
<li>The LC-MS/MS instrument (see <a href="#key-resources-table">key resources table</a>) equipped with an electrospray ion source and operated in positive ion mode was used to analyze nucleotides.</li>
<li>Chromatographic separation is done in UPLC HSS T3 column equipped with corresponding pre-column (see <a href="#key-resources-table">key resources table</a>) using a gradient as described in <a href="#tbl2">Table 2</a>.
<table id="tbl2">
<caption>Table 2. Gradient of mobile phases (step 25), column temperature 25°C</caption>
<thead>
<tr>
<th> </th>
<th>Time (min)</th>
<th>Mobile phaseA (%)</th>
<th>Mobile phaseB (%)</th>
<th>Flow rate (mL/min)</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>0</td>
<td>99.8%</td>
<td>0.2%</td>
<td>0.3</td>
</tr>
<tr>
<td>2</td>
<td>3</td>
<td>Linear increase 99.8%–0%</td>
<td>0.2%–100%</td>
<td>0.3</td>
</tr>
<tr>
<td>3</td>
<td>3.5</td>
<td>0%</td>
<td>100%</td>
<td>0.3</td>
</tr>
<tr>
<td>4</td>
<td>4</td>
<td>0%–99.8%</td>
<td>100%–0.2%</td>
<td>0.3</td>
</tr>
<tr>
<td>5</td>
<td>4.5–7</td>
<td>99.8%</td>
<td>0.2%</td>
<td>0.3</td>
</tr>
</tbody>
</table>
</li>
<li>Samples are kept at 8°C and automatically injected in a volume of 3 mL.</li>
<li>For mass spectrometry Argon is used as collision gas with a flow rate of 0.1 mL/min.</li>
<li>The capillary voltage is set to 2.50 kV, source temperature 150°C, desolvation temperature 400°C, cone gas flow 150 L/h, desolvation gas flow 800 L/h.</li>
<li>MassLynx and TargetLynx software (version 4.1, Waters) were applied for the acquisition and analysis of data.</li>
</ul>
<p>System setup for semi-quantitative metabolomics</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Rearing flies with and without native gut bacteria</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15–18 days</div>
<p>The following procedure is used to prepare flies with and without native gut microbiota.</p>
<ol>
<li>Collect the eggs.
<ol type="a">
<li>Place embryo collection cage on a CO<sub>2</sub> anesthetizing flypad at the minimal flow, necessary to anaesthetize flies. Transfer mature adult females (5–8 days old) from 6-8 bottles (see above) into the embryo collection cage (one cage per treatment). Use Yeast Extract–Peptone–Dextrose (YPD) plates to close the cage.
<div class="note">
<span class="note-title">Note:</span> We use YPD plates to promote egg laying without supplementation of yeasts.</div>
</li>
<li>Keep the flies on CO<sub>2</sub> anesthetizing flypad for 1.5 min.</li>
<li>Remove the cage from the anesthetizing apparatus and let the flies lay eggs on the YPD plate for 45 min - 1 h at 25°C.
<div class="note">
<span class="note-title">Note:</span> This step (synchronization) allows flies to release the old eggs and avoid larval hatching prior to removal procedure of gut bacteria (step 3). Lack of synchronization can lead to bacterial contamination of the germ-free vials.</div>
</li>
<li>Repeat steps 1 b and c (2<sup>nd</sup> synchronization).
<div class="optional">
<span class="optional-title">Optional:</span> Change the laying plate to a new YPD plate for a 2<sup>nd</sup> round of laying.</div>
</li>
<li>Anesthetize the flies in the cage, replace the synchronization plate to a new YPD plate and incubate for ∼3 h at RT to get the desired number of eggs.
<div class="note">
<span class="note-title">Note:</span> The time of egg laying depends on the fly’s stock.</div>
</li>
<li>Discard the flies or keep them for recolonization treatment (see below).</li>
<li>Remove flies that are stuck to the agar.</li>
</ol>
</li>
<li>Prepare control flies (flies with native bacteria intact).
<ol type="a">
<li>Add double distilled water (DDW) to the plate with eggs (from step 1) and gently detach the eggs from the surface using a paint brush.</li>
<li>Rinse the eggs, transfer them to the cell strainer (100 μm) and keep the strainer in the plate with DDW during the transfer procedure (step 2 c).
<div class="note">
<span class="note-title">Note:</span> To maximize the collection of eggs from the plate, repeat steps 2 a, b.</div>
</li>
<li>Pick up (∼20–30) eggs on the tip of the paint brush and gently transfer them to the surface of sterile food in vials.
<div class="note">
<span class="note-title">Note:</span> To prevent egg drying during the transfer, the strainer should be kept wet inside the 35 mm tissue culture plate.</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Remove gut bacteria by dechorionation to generate germ-free (GF) flies.
<ol type="a">
<li>Add 3% sodium hypochlorite solution to the plate with eggs (from step 1), gently detach the eggs from the surface using a paint brush and incubate the eggs for 2 min.</li>
<li>Collect the cells by transferring them through the cell strainer (100 μm).</li>
<li>Collect the remaining eggs from the plate by washing the plate again with 70% ETOH and place the strainer into a 35 mm plate with 70% ETOH.</li>
<li>In the biological safety cabinet prepare one culture plate (35 mm) with sterile 70% ETOH and 3 culture plates (35 mm) with sterile DDW.</li>
<li>Transfer the strainer with the eggs into a biological safety cabinet.</li>
<li>Use forceps to place the strainer into a new culture plate with 70% ETOH.</li>
<li>Wash the eggs three times with sterile DDW. Each time transfer the cell strainer to a new plate with DDW.</li>
<li>Use the paint brush to gently transfer dechorionated (∼20–30) eggs onto the surface of sterile food.</li>
</ol>
</li>
<li>Recolonize the eggs with native gut bacteria.
<ol type="a">
<li>Prepare vials with GF eggs, as described in step 3.</li>
<li>Transfer five males (from step 1 d) to the vial with GF eggs.</li>
<li>Let males “contaminate” the food with native gut bacteria for 1–2 days at 25°C, 60% humidity, 12 h light:12 h dark cycle controlled incubator and then discard them.</li>
</ol>
</li>
<li>Incubate all the vials (control and GF with and without recolonization) at 25°C, 60% humidity, 12 h light:12 h dark cycle controlled incubator and follow fly development (larval development, pupariation, and eclosion).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> In GF flies, we expect a developmental delay of 1–2 days.</div>
<ol start="6">
<li>Two days after eclosion transfer newly emerged flies (day 0–2) into new vials and incubate for 5 more days at 25°C, 60% humidity in a 12 h light:12 h dark cycle controlled incubator.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> GF flies should be transferred in the sterile conditions.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Combine flies from 2 vials in 1 new vial.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to keep the original vials for later test of germ-free conditions (step 8).</div>
<ol start="7">
<li>Collect flies and tissue for subsequent analyses.
<ol type="a">
<li>Collect separately males and females into 2 mL Eppendorf tube, weigh and freeze immediately in a dry ice/liquid nitrogen.
<ol type="i">
<li>For metabolic profiling: 40–50 flies per sample;</li>
<li>For targeted metabolomics: 10 flies per sample.</li>
</ol>
</li>
<li>Transfer to −80°C for storage.</li>
<li>Dissect ovaries:
<ol type="i">
<li>For metabolic profiling: 40–50 ovaries per sample. Freeze immediately on dry ice.</li>
<li>For targeted metabolomics: 10 ovaries per sample. Freeze immediately on dry ice.</li>
<li>For staging: collect 15–20 ovaries to ice-cold phosphate buffered saline (PBS) and proceed to DAPI staining (steps 9–21).</li>
</ol>
</li>
</ol>
</li>
<li>Check germ-free conditions of the food by plating food samples from the used vials on YPD plates.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Confirm germ-free conditions using PCR with universal 16S ribosomal DNA primers (for example, 8F and 939R oligonucleotides, see <a href="#key-resources-table">key resources table</a>). The PCR is performed on fly homogenate. Product size ∼940, annealing step at 60°C.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Confirm the expected delay in larval development under germ-free conditions. Note that the removal of gut microbiota by dechorionation typically results in delayed pupation which depends on the choice of diet (<a href="#bib3">Fridmann-Sirkis et al., 2014</a>).</div>
<h3 id="sec3.2">Analysis of ovarian development</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step enables evaluation of ovarian development by staging ovarian egg chambers. Number of mature oocytes in the ovary indicates the status of the oogenesis process.</p>
<ol start="9">
<li>Dissect 15–20 pairs of ovaries from each treatment and transfer them to the Eppendorf tube with 0.5 mL of PBSx1.</li>
<li>Let the ovaries settle down and aspirate PBS.</li>
<li>Add 3.7% formaldehyde (FA) to fixate the ovaries and incubate for 20 min at 25°C with shaking (∼25–27 rpm).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Fixation solution must be prepared freshly before each use.</div>
<ol start="12">
<li>Let the ovaries settle down, discard FA and wash twice with 1.5 mL PBS.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The procedure can be paused at this point and the sample can be kept in 4°C for a few days.</div>
<ol start="13">
<li>Discard the PBS and incubate the ovaries with 0.5 mL 1% Triton (in PBS) for 30 min.</li>
<li>Wash twice with PBS. Discard the last wash and add 1 mL PBS.</li>
<li>Collect the ovaries with Pasteur pipette and place them on microscope slide.</li>
<li>Place the ovaries in the center and carefully dry the liquid around them using disposable wipers.</li>
<li>Add one drop of Vectashield mounting medium with DAPI.</li>
<li>Arrange the ovaries on the slide for imaging using forceps.</li>
<li>Use pin holders to gently separate the ovarioles from each other, keeping the ovary intact and cover the slide with coverslip.</li>
<li>Take images of the DAPI-stained ovaries using fluorescent inverted microscope (4× magnification).</li>
<li>Evaluate ovarian development by scoring the number of egg chambers at different stages in each ovary (<a href="#fig1">Figure 1</a>) (<a href="#bib2">Elgart et al., 2016</a>; <a href="#bib4">Gnainsky et al., 2021</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1520-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Staging of DAPI-stained ovaries</div>
<p>(A) Representative images of DAPI-stained ovaries of 6-day OrR females, control (Ctrl) and germ-free with and without gut bacteria (GF+bac and GF, respectively). Scale bar 100 μm.</p>
<p>(B) Numbers of stage 11–14 egg chambers in ovary for cases in (A). Data are represented as mean ± SEM.</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec3.3">Quantitative analysis of energy-carrying molecules</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days: 21 h for step 22, 21 h for step 23, 40 min for step 24, and 7.5 min per sample for step 25</div>
<p>This step aims at analyzing the microbiome impact on energy production by measuring the levels of energy-carrying molecules using quantitative LC-MS/MS. The protocol below describes the specific steps for ATP and ADP measurements (we also used this protocol for GTP and GDP).</p>
<div class="note">
<span class="note-title">Note:</span> Prepare fresh solutions on the day of use. Calculate the required volumes, taking into consideration the number of samples, dilutions, washes, <i>etc</i>.</div>
<ol start="22">
<li>Preparation of sample extracts.
<ol type="a">
<li>Add 1 mL of 70%-aqueous methanol (MeOH) to frozen flies/tissue (from step 7).
<div class="note">
<span class="note-title">Note:</span> 70%-MeOH should be pre-chilled to ∼4°C before use (on ice or in a refrigerator).</div>
</li>
<li>Homogenize tissue using handheld pestle motor mixer on ice for 1–2 min.</li>
<li>Prepare 1 mM aqueous solution of internal standard <sup>13</sup>C<sub>10</sub>-ATP.</li>
<li>Add 5 μL 1 mM aqueous solution of internal standard <sup>13</sup>C<sub>10</sub>-ATP to each extract.</li>
<li>Agitate the mixture in Thermo Mixer C at 9°C and 1,000 rpm for 10 min.</li>
<li>Centrifuge at 21,000 × <i>g</i> at 4°C for 5 min to pellet proteins and cell debris, and transfer the supernatant to a clean 1.5 mL Eppendorf tube, without disturbing the precipitate.</li>
<li>Evaporate the supernatant using a speedvac.
<div class="note">
<span class="note-title">Note:</span> To remove MeOH usually takes ∼2 h.</div>
</li>
<li>Freeze the residual aqueous solution by immersing Eppendorf’s bottom in liquid nitrogen for 1 min.</li>
<li>Place frozen sample in the freeze dryer for complete evaporation overnight (∼18 h).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Evaporated extract can be stored at −80°C for at least two weeks.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> MeOH evaporation can be performed by a stream of nitrogen gas at 25°C.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Organic solvents, such as MeOH are highly toxic. The use of these agents should be performed in a fume hood. During speedvac evaporation of these organic solvents, the exhaust of the vacuum pump should be drawn to the fume hood or to exhaust of a proper scavenging line.</div>
</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Purification of extracts by Solid-phase extraction (SPE).
<div class="note">
<span class="note-title">Note:</span> Purification of nucleotides on polymeric weak anion column is required to remove all other components of the sample that significantly reduce signal intensity in mass spectrometer (matrix effect).</div>
<ol type="a">
<li>Prepare following solutions:
<ol type="i">
<li>1 mM aqueous solution of internal standard <sup>13</sup>C<sub>10</sub>-ATP.</li>
<li>DDW.</li>
<li>Methanol (MeOH).</li>
<li>50%-aqueous MeOH.</li>
<li>Formic acid/MeOH/DDW (2:25:73 v/v/v).</li>
<li>Ammonia/MeOH/DDW (8:25:67 v/v/v).
<div class="note">
<span class="note-title">Note:</span> Ammonia is 25%-aqueous ammonia solution.</div>
</li>
<li>Ammonia/MeOH/DDW (8:50:42 v/v/v).</li>
</ol>
</li>
<li>Condition SPE column.
<ol type="i">
<li>Place the column on a vacuum manifold.</li>
<li>Add 1 mL MeOH and let it flow through the column at a rate of about 1 mL/min.</li>
<li>Add 1 mL formic acid/MeOH/DDW (2:25:73, v/v/v), and let it flow at the same rate.</li>
<li>Wash the column with 1 mL DDW at the same rate flow.</li>
</ol>
</li>
<li>Purify the sample.
<ol type="i">
<li>Add 200 μL water to re-suspend the sample pellet, vortex shortly. Load the suspension onto the SPE column, let it flow through at about 0.5 mL/min.</li>
<li>Wash the column with 1 mL water, then with 1 mL 50% MeOH at about 1 mL/min.</li>
<li>Elute the sample with 1 mL ammonia/MeOH/water (8:25:67, v/v/v), and then 1 mL ammonia/MeOH/water (8:50:42, v/v/v) (see preparation in 23 a), collecting the eluates.</li>
</ol>
</li>
<li>Evaporate the collected elutes as described in steps 22 g–i.
<div class="critical">
<span class="critical-title">Critical:</span> Use flows as drops and not as stream to avoid reduction in metabolite recovery and purification quality.</div>
<div class="note">
<span class="note-title">Note:</span> If there is no liquid flow in steps b and c, use a slight vacuum suction to initiate the process.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Evaporated extract can be stored at −80°C for at least 2 weeks.</div>
</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Generating a standard curve.
<ol type="a">
<li>Prepare Mobile phase A: 10 mM ammonium acetate and 5 mM ammonium bicarbonate buffer, pH 7.65 (adjusted with 10% acetic acid).</li>
<li>Prepare Mobile phase B: acetonitrile.</li>
<li>Prepare ATP and ADP standard solutions for the standard curve:
<ol type="i">
<li>Dissolve pre-weighted amounts of ATP and ADP in DDW to get <b>1 mg/mL</b> solutions of each.</li>
<li>50 μg/mL (Stock solution): Mix 5 μL of each 1 mg/mL solution with 95 μL DDW.</li>
<li>25 μg/mL (Solution I): Dilute 50 μL of each 50 μg/mL stock solution with 50 μL DDW.</li>
<li>2.5 μg/mL (Solution II): Dilute 10 μL of each 25 μg/mL solution with 90 μL of DDW.</li>
<li>0.25 μg/mL (Solution III): Dilute 10 μL of 2.5 μg/mL solution with 90 μL of DDW.</li>
<li>
<b>10 μg/mL standard solution</b>: Add 40 μL of Solution I to 50 μL of mobile phase A (step 24), add 10 μL of 1 mM internal standard <sup>13</sup>C<sub>10</sub>-ATP (step 23 a i).</li>
<li>
<b>5 μg/mL standard solution</b>: Add 20 μL of Solution I to 70 μL of mobile phase A (step 24), add 10 μL of 1 mM internal standard <sup>13</sup>C<sub>10</sub>-ATP (step 23 a i).</li>
<li>
<b>1 μg/mL standard solution:</b> Add 40 μL of Solution II to 50 μL of mobile phase A (step 24), add 10 μL of 1 mM internal standard <sup>13</sup>C<sub>10</sub>-ATP (step 23 a i).</li>
<li>
<b>0.5 μg/mL standard solution</b>: Add 20 μL of Solution II to 50 μL of mobile phase A (step 24), add 10 μL of 1 mM internal standard <sup>13</sup>C<sub>10</sub>-ATP (step 23 a i).</li>
<li>
<b>0.1 μg/mL standard solution</b>: Add 40 μL of solution III to 50 μL of mobile phase A (step 24), add 10 μL of 1 mM internal standard <sup>13</sup>C<sub>10</sub>-ATP (step 23 a i).</li>
<li>
<b>0.01 μg/mL standard solution</b>: Add 4 μL of solution III to 50 μL of mobile phase A (step 24), add 10 μL of 1 mM internal standard <sup>13</sup>C<sub>10</sub>-ATP (step 23 a i).
<div class="note">
<span class="note-title">Note:</span> Final concentration of the internal standard – 100 μM.</div>
</li>
</ol>
</li>
<li>Place the prepared solution after short vortex into 250-μL inserts of LC-MS vials.</li>
</ol>
</li>
</ol>
<ol start="25">
<li>Measuring the levels of ATP and ADP using UPLC-MSMS.
<ol type="a">
<li>Preparation of samples.
<ol type="i">
<li>Re-suspend dried samples in 50 μL of mobile phase A, vortex shortly.</li>
<li>Centrifuge at 21,000 × <i>g</i> at 4°C for 5 min to rid of possible insoluble material, and transfer the supernatant to a clean 250-μL insert of LC-MS vials without disturbing the precipitate.</li>
</ol>
</li>
<li>Run LC-MS/MS analysis using system setup as described in <a href="#materials-and-equipment">materials and equipment</a> section.</li>
<li>Acquisition and analysis of data is done using MassLynx and TargetLynx softwares (version 4.1), respectively.</li>
<li>Detection of the compounds is done using multiple-reaction monitoring (MRM) parameters (<a href="#tbl3">Table 3</a>).</li>
</ol>
</li>
</ol>
<table id="tbl3">
<caption>Table 3. Multiple-reaction monitoring (MRM) parameters for mass spectrometer (step 25)</caption>
<thead>
<tr>
<th>Name of the compound</th>
<th>Retention time (min)</th>
<th>Transition (m/z)</th>
<th>Cone voltage (V)</th>
<th>Collision energy (eV)</th>
</tr>
</thead>
<tbody>
<tr>
<td>ADP</td>
<td>1.33</td>
<td>428.0 &gt; 136.1428.0 &gt; 348.1</td>
<td>2525</td>
<td>2514</td>
</tr>
<tr>
<td>ATP</td>
<td>1.54</td>
<td>507.9 &gt; 136.0507.9 &gt; 410.0</td>
<td>1414</td>
<td>3518</td>
</tr>
</tbody>
</table>
<h3 id="sec3.4">Preparing solutions for biphasic extraction (for metabolomics and lipidomics)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<ol start="26">
<li>Solutions cooled to −20°C:
<ol type="a">
<li>TMBE (methyl-tertbutyl-ether): MeOH (3:1, v/v) containing following internal standards: 0.1 μg/mL of phosphatidylcholine (17:0/17:0), 0.4 μg/mL of phosphatidylethanolamine (17:0/17:0), 0.15 nmol/mL of ceramide/sphingoid internal standard mixture I, 0.0267 μg/mL d5-TG internal standard mixture I and 0.1 μg/mL palmitic acid-<sup>13</sup>C.</li>
<li>H<sub>2</sub>O: MeOH (3:1, v/v), containing following internal standards: cell-free amino acid mixture - <sup>13</sup>C,<sup>15</sup>N.</li>
</ol>
</li>
<li>Solutions at 25°C:
<ol type="a">
<li>MeOH: H<sub>2</sub>O (1:1, v/v) (for dissolving dry polar sample).</li>
<li>TMBE.</li>
</ol>
</li>
<li>Running buffers for the UPLS-MS machines.
<ol type="a">
<li>Polar running phase.
<ol type="i">
<li>Mobile phase A: 20 mM ammonium carbonate with 0.1% ammonia hydroxide in H<sub>2</sub>O: acetonitrile (80:20, v/v).</li>
<li>Mobile phase B: Acetonitrile.
<div class="note">
<span class="note-title">Note:</span> Adjust pH in mobile phase A, using ammonia solution 25% to pH=9.4. <a href="#troubleshooting">Troubleshooting 2</a>.</div>
</li>
</ol>
</li>
<li>Semipolar running phase:
<ol type="i">
<li>Mobile phase A: Acetonitrile: H<sub>2</sub>O (95:5, v/v). with 0.1% formic acid.</li>
<li>Mobile phase B: Acetonitrile with 0.1% formic acid.</li>
</ol>
</li>
<li>Lipid running phase:
<ol type="i">
<li>Mobile phase A: H<sub>2</sub>O: acetonitrile: 2-propanol 46:38:16 (v/v/v) with 1% 1 M NH<sub>4</sub>Ac, 0.1% acetic acid.</li>
<li>Mobile phase B: H<sub>2</sub>O: Acetonitrile: 2-propanol 1:69:30 (v/v/v) with 1% 1 M NH<sub>4</sub>Ac, 0.1% acetic acid. (also used to dissolve dried lipid sample for injection).
<div class="note">
<span class="note-title">Note:</span> For solution preparations, use DDW from Milli-Q® Water Purification System supplemented with LC-Pak® Polisher.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.5">Preparation of extracts for metabolomics and lipidomics</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The following procedure extracts a wide range of metabolites from fly ovary liophilaed powder prior to the LC-MS analysis. Each extraction contains analytes from a particular biological sample and can be applied to dissected tissues, organs or whole flies.</p>
<div class="note">
<span class="note-title">Note:</span> If the samples to be collected during several days, it is critical to snap-freeze all the samples at the same hours during the collection day to avoid metabolic effects of the circadian clock in the study subjects.</div>
<ol start="29">
<li>Lyophilize snap-frozen samples of flies and ovaries overnight (∼18 h) in 2 mL safe-lock microcentrifuge tubes.</li>
<li>Add two metal balls into each tube and grind the samples to a fine powder by shaking in a homogenizer mixer mill at 30 Hz for two minutes.</li>
<li>Pre-cool (−20°C) extraction mixture (TMBE: MeOH, (3:1, v/v) containing following internal standards: 0.1 μg/mL of phosphatidylcholine (17:0/17:0), 0.4 μg/mL of phosphatidylethanolamine (17:0/17:0), 0.15 nmol/mL of ceramide/sphingoid internal standard mixture I, 0.0267 μg/mL d5-TG internal standard mixture I and 0.1 μg/mL palmitic acid-<sup>13</sup>C.</li>
<li>Add 1 mL of the pre-cooled extraction mixture to each sample tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step should be done as quickly as possible due to the low viscosity of TMBE. To avoid extraction mixture dripping out from the pipette tip, we suggest pre-wetting the tip before use with extraction solution by pipetting it in and out.</div>
<ol start="33">
<li>Mix immediately using a vortex mixer until the tissue is homogenized in the extraction mixture.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step is crucial since it is used to precipitate the proteins and inactivate their enzymatic activities.</div>
<ol start="34">
<li>Incubate samples in a sonicator bath for 30 min at 4°C. <a href="#troubleshooting">Troubleshooting 3</a>
</li>
<li>Vortex samples every 10 min.</li>
<li>Add 500 μL of H<sub>2</sub>O: MeOH (3:1, v/v), containing the following internal standards: Cell-Free amino acid mixture - <sup>13</sup>C,<sup>15</sup>N to each sample tube.</li>
<li>Vortex for 0.5 min.</li>
<li>Centrifuge samples at 20,000 × <i>g</i> for 5 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> At this point, there are two immiscible liquid phases with a solid pellet in the bottom of the tubes.</div>
<div class="note">
<span class="note-title">Note:</span> For high-quality extraction, the starting material should be ground into fine homogenous powder. If there is no suitable available homogenizer, it is possible to use mortar and pestle.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Carefully handle the tubes to prevent mixing the two liquid phases and disrupting the precipitated pellet.</div>
<ol start="39">
<li>Transfer 500 μL from the upper, lipid-containing phase, into a labeled 2 mL microcentrifuge tube.</li>
<li>Start evaporation using speedvac without heating.</li>
<li>Add to remaining fraction 0.5 mL of TMBE and repeat extraction process as described in steps 29–35.</li>
<li>Transfer 500 μL of upper phase (lipid-enriched) from 2<sup>nd</sup> extraction into the tube with the first portion and continue evaporation using speedvac without heating.</li>
<li>Transfer 400 μL from the lower phase (polar and semi-polar metabolites) into a labeled 1.5 mL microcentrifuge tube and evaporate using speedvac without heating.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The evaporation of the organic samples can be done by a gentle stream of nitrogen.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Dried polar extracts can be immediately used for UPLC-MS analysis (steps 48–50) or stored for several weeks at −80°C.</div>
<ol start="44">
<li>Remove the remaining aqueous/lipid phase from the protein pellet by pipetting off the excess volume.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Protein pellets can be used for normalization and other analyses.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Protein pellets can be stored at −80°C for further analysis.</div>
<h3 id="sec3.6">Analysis of lipids using UPLC-qToF-IMS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Running time 25 min per sample</div>
<p>Steps 45–47 enable chromatographic separation of metabolites, determining their mass spectra and identifying specific lipid species.</p>
<ol start="45">
<li>Dissolve lipid extracts (step 42) before injection into lipid gradient analysis.
<ol type="a">
<li>Add 250 μL of Mobile phase B, described in step 28 c ii, to dry pellet from step 42.</li>
<li>Vortex.</li>
<li>Sonicate in sonicator bath for 5 min at 4°C and vortex again.</li>
<li>Centrifuge at 20,000 × <i>g</i> for 10 min.</li>
<li>Transfer 150 μL into HPLC glass vial for injection.</li>
</ol>
</li>
<li>Analyze the lipid extract by the ACQUITY UPLC system coupled to a Vion IMS qToF mass spectrometer.
<ol type="a">
<li>Insert the glass HPLC vials into a cooled autosampler (6°C).</li>
<li>Using a UPLC system running at a flow rate of 400 μL/min, inject 1 μL of sample on a ACQUITY UPLC BEH C8 column (2.1 × 100 mm, i.d., 1.7 μm) (Waters Corp., MA, USA) at 40°C.</li>
<li>Use the mobile phases gradient (<a href="#tbl4">Table 4</a>), for the chromatographic separation.
<table id="tbl4">
<caption>Table 4. Conditions for lipid gradient (mobile phases, steps 45–46)</caption>
<thead>
<tr>
<th> </th>
<th>Time (min)</th>
<th>Flow rate (mL/min)</th>
<th>Mobile phaseA (%)</th>
<th>Mobile phaseB (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>0.00</td>
<td>0.4</td>
<td>100.0</td>
<td>0.0</td>
</tr>
<tr>
<td>2</td>
<td>1.00</td>
<td>0.4</td>
<td>100.0</td>
<td>0.0</td>
</tr>
<tr>
<td>3</td>
<td>12.00</td>
<td>0.4</td>
<td>25.0</td>
<td>75.0</td>
</tr>
<tr>
<td>4</td>
<td>16.00</td>
<td>0.4</td>
<td>0.0</td>
<td>100.0</td>
</tr>
<tr>
<td>5</td>
<td>21.50</td>
<td>0.4</td>
<td>0.0</td>
<td>100.0</td>
</tr>
<tr>
<td>6</td>
<td>22.00</td>
<td>0.4</td>
<td>100.0</td>
<td>0.0</td>
</tr>
<tr>
<td>7</td>
<td>25.00</td>
<td>0.4</td>
<td>100.0</td>
<td>0.0</td>
</tr>
</tbody>
</table>
</li>
<li>Acquire the mass spectra in the positive and negative ionization modes using an MS instrument covering the mass range between 50 and 1,800 m/z.</li>
</ol>
</li>
<li>Analysis of the output.
<ol type="a">
<li>Identify lipids by comparison of their retention time, accurate mass, fragmentation pattern and ion mobility values to existing in-house lipid library, as well as to standards when available using UNIFI 1.9.2 software.</li>
<li>Export the result table.</li>
</ol>
</li>
</ol>
<h3 id="sec3.7">Analysis of polar using UPLC-Q-Exactive Orbitrap</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Running time 23 min per sample</div>
<p>The following procedure enables separation and identification of polar metabolites.</p>
<ol start="48">
<li>Dissolve polar extract (step 43) before injection into polar gradient analysis.
<ol type="a">
<li>Add 150 μL of MeOH: H<sub>2</sub>O, 1:1 (v/v) to dry pellet.</li>
<li>Mix by vortex.</li>
<li>Sonicate in sonicator bath for 5 min at 4°C.</li>
<li>Centrifuge at 20,000 × <i>g</i> for 10 min.</li>
<li>Transfer supernatant to a new 1.5 mL tube and centrifuge again.</li>
<li>Transfer remaining 100 μL into HPLC glass vial for injection.</li>
</ol>
</li>
<li>Analyze the polar extract by the ACQUITY UPLC system coupled to Q-Exactive Orbitrap.
<ol type="a">
<li>Insert the glass vials into a cooled autosampler (6°C).</li>
<li>Use a UPLC system running at a flow rate of 200 μL/min inject 1 μL per sample on a SeQuant Zic-pHilic (150 mm × 2.1 mm) column with the SeQuant guard column (20 mm × 2.1 mm) at 35°C. <a href="#troubleshooting">Troubleshooting 4</a>
</li>
<li>Use the Mobile phases gradient described in <a href="#tbl5">Table 5</a> for the chromatographic separation.
<table id="tbl5">
<caption>Table 5. Conditions for polar gradient (mobile phases, step 49)</caption>
<thead>
<tr>
<th> </th>
<th>Time (min)</th>
<th>Flow rate (mL/min)</th>
<th>Mobile phaseA (%)</th>
<th>Mobile phaseB (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>0.00</td>
<td>0.2</td>
<td>25.0</td>
<td>75.0</td>
</tr>
<tr>
<td>2</td>
<td>2.00</td>
<td>0.2</td>
<td>25.0</td>
<td>75.0</td>
</tr>
<tr>
<td>3</td>
<td>14.00</td>
<td>0.2</td>
<td>75.0</td>
<td>25.0</td>
</tr>
<tr>
<td>4</td>
<td>18.00</td>
<td>0.2</td>
<td>75.0</td>
<td>25.0</td>
</tr>
<tr>
<td>5</td>
<td>19.0</td>
<td>0.2</td>
<td>25.0</td>
<td>75.0</td>
</tr>
<tr>
<td>6</td>
<td>23.0</td>
<td>0.2</td>
<td>25.0</td>
<td>75.0</td>
</tr>
</tbody>
</table>
</li>
<li>Acquire the mass spectra in the negative ionization mode using an MS instrument covering the mass range between 50 and 750 m/z.</li>
</ol>
</li>
<li>Analysis of the output.
<ol type="a">
<li>Process the data using TraceFinder software, identifying the detected compounds by retention time and fragments.</li>
<li>Verify using an in-house-generated polar mass spectra library.</li>
</ol>
</li>
</ol>
<h3 id="sec3.8">Analysis of semi-polar co using UPLC-qToF-IMS</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Running time 28 min per sample</div>
<p>Steps 51 and 52 extend the UPLC-qToF-IMS analysis (above) to semi-polar metabolites.</p>
<ol start="51">
<li>Analyze the polar extract on the ACQUITY UPLC system coupled to a Vion IMS qToF mass spectrometer.
<ol type="a">
<li>Insert the glass vials into a cooled autosampler (6°C).</li>
<li>Use a UPLC system running at a flow rate of 300 μL/min Inject 1 μL of sample on a ACQUITY UPLC BEH C18 column (2.1 × 100 mm, i.d., 1.7 μm) at 35°C.</li>
<li>Use the Mobile phases gradient (<a href="#tbl6">Table 6</a>) for the chromatographic separation.
<table id="tbl6">
<caption>Table 6. Conditions for semi-polar gradient (mobile phases, step 51)</caption>
<thead>
<tr>
<th> </th>
<th>Time (min)</th>
<th>Flow rate (mL/min)</th>
<th>Mobile phaseA (%)</th>
<th>Mobile phaseB (%)</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>0.00</td>
<td>0.3</td>
<td>100.0</td>
<td>0.0</td>
</tr>
<tr>
<td>2</td>
<td>22.00</td>
<td>0.3</td>
<td>72.0</td>
<td>28.0</td>
</tr>
<tr>
<td>3</td>
<td>36.00</td>
<td>0.3</td>
<td>0.0</td>
<td>100.0</td>
</tr>
<tr>
<td>4</td>
<td>38.00</td>
<td>0.3</td>
<td>0.0</td>
<td>100.0</td>
</tr>
<tr>
<td>5</td>
<td>38.50</td>
<td>0.3</td>
<td>100.0</td>
<td>0.0</td>
</tr>
<tr>
<td>6</td>
<td>40.00</td>
<td>0.3</td>
<td>100.0</td>
<td>0.0</td>
</tr>
</tbody>
</table>
</li>
<li>Acquire the mass spectra in the positive and negative ionization modes using an MS instrument covering the mass range between 50 and 2,000 m/z.</li>
</ol>
</li>
<li>Analysis of the output.
<ol type="a">
<li>Identify metabolites by comparison of Retention time, accurate mass and ion mobility of the peaks to the in-house semi-polar library (UNIFI 1.9.2).</li>
<li>Export the result table.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The protocol of semi-quantitative metabolomics enables analysis of a wide range of polar, semi-polar and lipid metabolites in a given sample and provides an opportunity to detect treatment- or condition-specific changes in the profile of metabolites. We used it to identify 197 polar metabolites and 151 lipids in dissected ovaries and whole-bodies of GF vs. control females on day 6 (Figures S4 and S5 in <a href="#bib4">Gnainsky et al., 2021</a>). Global analysis of metabolic changes in GF vs. control females, revealed a tendency for increase in lipid, but not in polar metabolites (Figure S5 in <a href="#bib4">Gnainsky et al., 2021</a>). Higher resolution analysis assisted in identifying specific pathways that were affected by removal of extracellular gut microbiota. These include modifications in lipid metabolism, citric acid cycle (TCA), glycolysis, pentose phosphate and electron transport chain pathways, as well as changes in the levels of metabolites involved in energy transfer (Figures 4 and 5 in <a href="#bib4">Gnainsky et al., 2021</a>).</p>
<p>The above analysis also assists in selecting pathways/metabolites of interest for further mechanistic investigation. This was specifically demonstrated by the strong association between lower FAD levels in GF females and the increased levels of almost all the metabolites that are degraded by FAD-dependent enzymes (Figure 4C in <a href="#bib4">Gnainsky et al., 2021</a>). This finding can provide a very simple mechanism for the global changes in a wide range of diverse metabolites (e.g., essential amino acids, lipids, etc) that were also noted in other GF hosts, including mammals (<a href="#bib5">Mardinoglu et al., 2015</a>; <a href="#bib7">Velagapudi et al., 2010</a>), but remained largely unexplained.</p>
<h3 id="sec4.1">Analysis of energy-carrying molecules</h3>
<p>Our quantitative LC-MS/MS protocol was designed for the sensitive detection, verification and quantification of metabolites, such as ATP, GTP, etc. It provides quantitative results while requiring smaller sample sizes compared to the semi-quantitative method of metabolomic profiling. We previously used this protocol on 10 flies and 10 tissues such as ovaries, guts, heads as well as 40–50 mature oocytes (Figures 2, 3, and S2 in <a href="#bib4">Gnainsky et al., 2021</a>), but it has been also successfully tested on a single fly. The changes in the relative levels of ATP and ADP were confirmed by both the quantitative and semi-quantitative LC-MS methods (<a href="#fig2">Figure 2</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Relative abundance of each metabolite is obtained by two normalizations of its peak area. We first divide the peak area of each metabolite by the peak area of internal standard from the same chemical group. In the absence of a suitable standard, we divide the peak area by the average over peak areas of all the internal standards that are included in the analysis. In the second normalization, we further divide the normalized peak area by the number of flies (or ovaries) in each biological sample, or alternatively, by the sample’s weight (mg) (<a href="#bib4">Gnainsky et al., 2021</a>). <a href="#troubleshooting">Troubleshooting 5</a>. The normalized metabolomics data is processed using a software (e.g., MetaboAnalyst, <a href="https://www.metaboanalyst.ca">https://www.metaboanalyst.ca</a>; (<a href="#bib1">Chong et al., 2018</a>; <a href="#bib8">Wishart, 2020</a>)) and/or custom analysis tools that enable comparison between samples, detection of significant changes in metabolite levels and identification of altered metabolic pathways.</p>
<p>Statistical analysis of changes in the numbers stage-specific egg chambers and fold-changes in the absolute levels of ATP and ADP, is done using Prism Graphpad software. Comparison between samples is preceded by testing the assumption of normal distribution (e.g., Shapiro-Wilk and D'Agostino-Pearson). Normally distributed data is analyzed using unpaired t-test or ANOVA with Tukey’s multiple comparison test, whereas non-normal distributions is analyzed by Mann Whitney test or non-parametric ANOVA (Kruskal-Wallis) with Dunn’s multiple comparison test (<a href="#fig1">Figures 1</a> and <a href="#fig2">2</a>); (<a href="#bib4">Gnainsky et al., 2021</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1520-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Effect of gut bacteria removal on ATP and ADP levels in the ovaries, analyzed by quantitative and semi-quantitative LC-MS</div>
<p>(A and B) (A) Measurements of ATP and (B) ADP by quantitative LC-MS (qLC-MS, left panel) and by semi-quantitative LC-MS (Metabolomics), normalized by the number of ovaries and by weight (middle and right panels, respectively). Data are represented as mean ± SEM.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> Similar processing can be done by alternative software applications.</div>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The decreased levels of some of the metabolites of GF flies may reflect intrinsic changes in the host and/or loss of bacterial-derived metabolites. Distinguishing these options requires application of the protocol to additional types of samples. The ability to detect low abundant metabolites may require increased sample size. In the process of metabolite extraction and analysis by LC-MS, NADH and FADH<sub>2</sub> can be oxidized, respectively, into NAD<sup>+</sup> into FAD<sup>+</sup>. Special consideration and testing are therefore required prior to using this method to distinguish reduced and non-reduced forms of these metabolites (i.e., FADH<sub>2</sub>-FAD and NADH-NAD<sup>+</sup>). The number of identified compounds in metabolomics analysis depends on multiple parameters, such as: the size of reference library to which the peaks are compared, the gradient of chromatographic separation, the choice of column and LC machine, the sensitivity of the Mass spectrometer sensitivity. Co-eluted metabolites that have the same mass and fragmentation pattern may be indistinguishable by the current semi-quantitative LC-MS-based metabolomics technique. Accordingly, the identifications are considered putative, even if the pattern of the specific metabolite precisely matches that of its standard.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Rearing GF flies on rich diets (e.g., with ≥ 5% yeast) complements the loss of many bacterial-derived metabolites and can substantially mask out impacts of deficiencies in metabolites that are cannot be synthesized by the host (<a href="#bib2">Elgart et al., 2016</a>; <a href="#bib6">Storelli et al., 2011</a>) (in ‘<a href="#sec1.2">preparation of sterile fly food media</a>’).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>If the stock line does not require rich diet for maintenance, use standard diet with 0.8–1.2% yeast (steps in ‘<a href="#sec1.2">preparation of sterile fly food media</a>’, <a href="#tbl1">Table 1</a>). The goal is to increase sensitivity to the loss of bacterial-derived metabolites while avoiding starvation that can generate more serious deficiencies that are not necessarily related to the loos of bacteria.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>During the polar metabolite profiling using SeQuant Zic-pHilic column (step 49), pH deviations of the mobile phases could cause a shift in retention time followed by misidentification of metabolites during the library-based validation.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>It is essential to prepare mobile phases at pH=9.4 as recommended in step 28 and use calibrated pH-meter.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Long sonication during metabolite extraction could overheat the extraction solution and compromise the accuracy of profiling due to possible destruction of the heat-sensitive lipids and polar metabolites (step 34).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Keep the extracted samples below zero by: (i) using pre-cooled (−20°C) extraction solution and (ii) sonicating the samples with ice (in the sonication bath).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>SeQuant Zic-pHilic column (150 mm × 2.1 mm) is prone to overpressure, especially when proteins in the polar phase are not completely filtered out. Adding the SeQuant guard column (20 mm × 2.1 mm) keeps the column operational for longer time but might not resolve the problem for an extended usage period (step 49).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Two centrifugation steps are crucial for long column life and resolve the polar phase contamination by protein. The 2<sup>nd</sup> centrifugation at 20,000 × <i>g</i> for 10 min while dissolving polar metabolites (step 48 e) is critical to remove proteins present in the polar phase that could potentially clog the column. Do not skip this step.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Some treatments can modify the actual tissue composition of the sample (either tissue, organ or whole body), resulting in changes that are not limited to size and/or weight. For example, the removal of gut microbiota in flies that are reared on standard diet leads to a complicated outcome that combines a reduction in ovary size (and weight) with altered fraction of mature oocytes. Such a sample- or treatment-specific change in organ composition cannot be properly taken into account by normalization based on sample weight or total protein content (which can therefore lead to misinterpretations) (in ‘<a href="#expected-outcomes">quantification and statistical analysis</a>’).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The confounding effect of weight/protein/biomass-based normalization can be avoided by normalizing with respect to the number of individuals (or organs) from which the sample was prepared. This alternative, however, does not compensate for differential extraction and/or processing of the sample. To maximize the generation of biological insights, we therefore recommend the use of both methods, as long as the above considerations are kept in mind to avoid misinterpretations.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Chong, J., Soufan, O., Li, C., Caraus, I., Li, S., Bourque, G., Wishart, D.S., and Xia, J. (2018). MetaboAnalyst 4.0: towards more transparent and integrative metabolomics analysis. Nucleic Acids Res. <i>46</i>, W486-W494. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref1">View at publisher</a></p>
<p id="bib2">Elgart, M., Stern, S., Salton, O., Gnainsky, Y., Heifetz, Y., and Soen, Y. (2016). Impact of gut microbiota on the fly’s germ line. Nat. Commun. <i>7</i>, 11280. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref2">View at publisher</a></p>
<p id="bib3">Fridmann-Sirkis, Y., Stern, S., Elgart, M., Galili, M., Zeisel, A., Shental, N., and Soen, Y. (2014). Delayed development induced by toxicity to the host can be inherited by a bacterial-dependent, transgenerational effect. Front. Genet. <i>5</i>, 27. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref3">View at publisher</a></p>
<p id="bib4">Gnainsky, Y., Zfanya, N., Elgart, M., Omri, E., Brandis, A., Mehlman, T., Itkin, M., Malitsky, S., Adamski, J., and Soen, Y. (2021). Systemic regulation of host energy and oogenesis by microbiome-derived mitochondrial coenzymes. Cell Rep. <i>34</i>, 108583. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref4">View at publisher</a></p>
<p id="bib5">Mardinoglu, A., Shoaie, S., Bergentall, M., Ghaffari, P., Zhang, C., Larsson, E., Backhed, F., and Nielsen, J. (2015). The gut microbiota modulates host amino acid and glutathione metabolism in mice. Mol. Syst. Biol. <i>11</i>, 834. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref5">View at publisher</a></p>
<p id="bib6">Storelli, G., Defaye, A., Erkosar, B., Hols, P., Royet, J., and Leulier, F. (2011). Lactobacillus plantarum promotes Drosophila systemic growth by modulating hormonal signals through TOR-dependent nutrient sensing. Cell Metab. <i>14</i>, 403-414. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref6">View at publisher</a></p>
<p id="bib7">Velagapudi, V.R., Hezaveh, R., Reigstad, C.S., Gopalacharyulu, P., Yetukuri, L., Islam, S., Felin, J., Perkins, R., Boren, J., Oresic, M., and Backhed, F. (2010). The gut microbiota modulates host energy and lipid metabolism in mice. J. Lipid Res. <i>51</i>, 1101-1112. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref7">View at publisher</a></p>
<p id="bib8">Wishart, D.S. (2020). Statistical evaluation and integration of multi-omics data with MetaboAnalyst. In Processing Metabolomics and Proteomics Data with Open Software: A Practical Guide, Robert Winkler, ed. (The  Royal  Society  of  Chemistry), pp. 281-301. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00133-2/sref8">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yulia Gnainsky (<a href="mailto:Yulia.gnainsky@weizmann.ac.il">Yulia.gnainsky@weizmann.ac.il</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>Metabolomics dataset supporting the current study have not been deposited in a public repository because it is still being used in a related ongoing study, but it is available from the corresponding author on request.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/100000925">Sir John Templeton Foundation</a> (grants no. 40663 and 61122) and by a research grant from Madame Olga Klein – Astrachan. Metabolomics profiling was supported by Vera and John Schwartz Family Center for Metabolic Biology. We thank the Weizmann Fly’s core facility, metabolic profiling unit, and targeted metabolomics unit of Life Sciences Core Facilities for providing technical support.</p>
<h3>Author contributions</h3>
<p>Y.G. and Y.S. designed and supervised the study. Y.G. conducted experiments and data analysis. M.I. and S.M. constructed and implemented metabolomics and lipidomics analysis. A.B. and T.M. performed measurements of ATP and ADP. Y.G., M.I., S.M., A.B., and Y.S. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>