<article>
<section>
<h1 id="header">Protocols for endothelial cell isolation from mouse tissues: brain, choroid, lung, and muscle</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Nadine V. Conchinha,<sup><a href="#aff1">1</a>,<a href="#fn4">8</a></sup> Liliana Sokol,<sup><a href="#aff1">1</a>,<a href="#fn4">8</a></sup> Laure-Anne Teuwen,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn4">8</a></sup> Koen Veys,<sup><a href="#aff1">1</a>,<a href="#fn4">8</a></sup> Sébastien J. Dumas,<sup><a href="#aff1">1</a></sup> Elda Meta,<sup><a href="#aff1">1</a></sup> Melissa García-Caballero,<sup><a href="#aff1">1</a></sup> Vincent Geldhof,<sup><a href="#aff1">1</a></sup> Rongyuan Chen,<sup><a href="#aff3">3</a></sup> Lucas Treps,<sup><a href="#aff1">1</a></sup> Mila Borri,<sup><a href="#aff1">1</a></sup> Pauline de Zeeuw,<sup><a href="#aff1">1</a></sup> Kim D. Falkenberg,<sup><a href="#aff1">1</a></sup> Charlotte Dubois,<sup><a href="#aff1">1</a>,<a href="#fn1">5</a></sup> Magdalena Parys,<sup><a href="#aff1">1</a>,<a href="#fn2">6</a></sup> Laura P.M.H. de Rooij,<sup><a href="#aff1">1</a></sup> Katerina Rohlenova,<sup><a href="#aff1">1</a></sup> Jermaine Goveia,<sup><a href="#aff1">1</a></sup> Luc Schoonjans,<sup><a href="#aff1">1</a>,<a href="#aff3">3</a></sup> Mieke Dewerchin,<sup><a href="#aff1">1</a></sup> Guy Eelen,<sup><a href="#aff1">1</a></sup> Xuri Li,<sup><a href="#aff3">3</a></sup> Joanna Kalucka,<sup><a href="#aff1">1</a>,<a href="#aff4">4</a>,<a href="#fn3">7</a>,<a href="#fn5">9</a>,<a href="#cor1">*</a></sup> and Peter Carmeliet<sup><a href="#aff1">1</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#fn6">10</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Laboratory of Angiogenesis and Vascular Metabolism, Center for Cancer Biology, VIB, &amp; Department of Oncology and Leuven Cancer Institute (LKI), KU Leuven, Leuven, Belgium</p>
<p id="aff2"><sup>2</sup>Translational Cancer Research Unit, GZA Hospitals Sint-Augustinus, Antwerp 2610, Belgium and Center for Oncological Research, University of Antwerp, Antwerp 2000, Belgium</p>
<p id="aff3"><sup>3</sup>State Key Laboratory of Ophthalmology, Zhongshan Ophthalmic Center, Sun Yat-Sen University, Guangzhou 510060, Guangdong, P.R. China</p>
<p id="aff4"><sup>4</sup>Department of Biomedicine, Aarhus University, Aarhus 8000, Denmark</p>
<p id="fn1"><sup>5</sup>Present address: Laboratory of Cell Physiology, Inserm U 1003, Bat. SN 3, UFR de Biologie, Université de Lille 1, 59655 Villeneuve d'Ascq, France</p>
<p id="fn2"><sup>6</sup>Present address: Flow Cytometry Core Facility, Sloan Kettering Institute, Memorial Sloan Kettering, Cancer Center, New York, NY, USA</p>
<p id="fn3"><sup>7</sup>Present address: Aarhus Institute of Advanced Studies (AIAS) &amp; Department of Biomedicine, Aarhus University, Aarhus 8000, Denmark</p>
<p id="fn4"><sup>8</sup>These authors contributed equally</p>
<p id="fn5"><sup>9</sup>Technical contact</p>
<p id="fn6"><sup>10</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:joanna.kalucka@aias.au.dk">joanna.kalucka@aias.au.dk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:peter.carmeliet@kuleuven.be">peter.carmeliet@kuleuven.be</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100508">10.1016/j.xpro.2021.100508</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Endothelial cells (ECs) harbor distinct phenotypical and functional characteristics depending on their tissue localization and contribute to brain, eye, lung, and muscle diseases such as dementia, macular degeneration, pulmonary hypertension, and sarcopenia. To study their function, isolation of pure ECs in high quantities is crucial. Here, we describe protocols for rapid and reproducible blood vessel EC purification established for scRNA sequencing from murine tissues using mechanical and enzymatic digestion followed by magnetic and fluorescence-activated cell sorting.</p>
<p>For complete details on the use and execution of these protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref11">Kalucka et al. (2020)</a>, <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref15">Rohlenova et al. (2020)</a>, and <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref9">Goveia et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocols for isolation of murine endothelial cells designed for scRNA-seq</li>
<li>Rapid and efficient isolation of ECs from brain, choroid, lung, and muscle</li>
<li>Combination of magnetic and fluorescent activated cell sorting</li>
<li>High purity and quality of isolated murine endothelial cells evident from scRNA-seq</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocols were established using 8-week-old male C57BL6/J mice purchased from Charles River (strain code: 632). All experimental procedures for the establishment and application of the protocols were done under approval by the Institutional Animal Ethics Committee of the KU Leuven (Belgium); protocol number P012/2018.</p>
<p>The protocols described below provide the details on EC isolation from one tissue at a time. However, it is possible to isolate cells from multiple organs of the same mouse. In that case, we recommend to perform transcardial perfusion with ice-cold PBS at a perfusion rate of 2 mL/min for 5 min, followed by additional perfusion with digestion buffer: Supplemented KnockOut<sup>TM</sup> DMEM-medium with 0.1% (w/v) collagenase I (Thermo Fisher Scientific, Cat#17018029), 0.1% (w/v) collagenase II (Thermo Fisher Scientific, Cat#17101015) and 7.5 μg/mL DNase I (Sigma-Aldrich, Cat#D4527-10KU) at a perfusion rate of 2 mL/min for 5 min. The purpose of this step is to remove blood from the blood vessels and replace it with the digestion buffer to ensure efficient digestion. Additionally, if cells from multiple tissues are isolated simultaneously from the same mouse, we suggest to assign one person per single organ isolation.</p>
<p>The cells have been sorted using the BD FACSAria™ III sorter. Considering EC fragility, the following settings have been used: nozzle size - 100 μm, pressure - 20 psi. To maximize sorted EC purity and efficiency, we have been using the 4-way purity and sorting ECs on flow rate 1, respectively.</p>
<p>The optical paths used per fluorochrome:</p>
<ol>
<li>FITC - Laser: Blue 488; Detector 502 LP; Filter set up: 530/30 BP</li>
<li>PE-Cy7 - Laser: Yellow-Green 561; Detector 735 LP; Filter set up: 780/60 BP</li>
<li>eFluor450 - Laser: Violet 407; Filter set up: 450/40 BP</li>
</ol>
<p>The settings are designed for EC sorting with the BD FACSAria™ III sorter and have to be optimized specifically to the sorter of choice.</p>
<p>Of note, the protocols are based on magnetic bead depletion of immune cells (CD45)/epithelial cells (EpCAM) and/or enrichment of ECs by magnetic beads (CD31) and fluorescent activated cell sorting. The listed dilution of antibodies might vary between lots and manufacturers/vendors. Therefore, we highly recommend optimizing these parameters for each antibody before use.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rat anti-mouse CD31 FITC (clone 390)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11-0311-82 RRID: AB_465012</td>
</tr>
<tr>
<td>Rat anti-mouse CD45 PE-Cy7 (clone 30-F11)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#25-0451-82RRID: AB_2734986</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Antibiotic-antimycotic</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15240062</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA Fraction V)</td>
<td>Sigma-Aldrich</td>
<td>Cat#10735086001</td>
</tr>
<tr>
<td>Collagenase type I</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#17018029</td>
</tr>
<tr>
<td>Collagenase type II</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#17101015</td>
</tr>
<tr>
<td>Collagenase type IV</td>
<td>Worthington Biochemical</td>
<td>Cat#LS004188</td>
</tr>
<tr>
<td>Dispase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#17105-041</td>
</tr>
<tr>
<td>DNase I</td>
<td>Sigma-Aldrich</td>
<td>Cat#D4527-10KU</td>
</tr>
<tr>
<td>DMEM, high glucose</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11965092</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma-Aldrich</td>
<td>Cat#ED2P-500G</td>
</tr>
<tr>
<td>Endothelial cell growth factor supplements (ECGS/Heparin)</td>
<td>PromoCell</td>
<td>Cat#C-30120</td>
</tr>
<tr>
<td>Fetal bovine serum (FBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#16000044</td>
</tr>
<tr>
<td>Fixable Viability Dye eFluor™ 450</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#65-0863-18</td>
</tr>
<tr>
<td>Hank's balanced salt solution (HBSS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#14025092</td>
</tr>
<tr>
<td>KnockOut<sup>TM</sup> DMEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10829018</td>
</tr>
<tr>
<td>MEM NEAA</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11140035</td>
</tr>
<tr>
<td>Penicillin/streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>Phosphate-buffered saline (DPBS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#14190094</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11360070</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>CD31 MicroBeads, mouse</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-097-418</td>
</tr>
<tr>
<td>CD45 MicroBeads, mouse</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-052-301</td>
</tr>
<tr>
<td>EpCAM MicroBeads, mouse</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-105-958</td>
</tr>
<tr>
<td>Neural Tissue Dissociation Kit (P)</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-092-628</td>
</tr>
<tr>
<td>Myelin Removal Beads II</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-096-733</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>RNA-sequencing raw and analyzed data mouse EC (brain, lung, muscle)</td>
<td>(<a href="#bib11">Kalucka et al., 2020</a>)</td>
<td>ArrayExpress:E-MTAB-8077</td>
</tr>
<tr>
<td>RNA-sequencing raw and analyzed data mouse EC (choroid)</td>
<td>(<a href="#bib15">Rohlenova et al., 2020</a>)</td>
<td>ArrayExpress:E-MTAB-8119</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>C57BL6/J male mice</td>
<td>Charles River</td>
<td>Strain code: 632</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BIOMEX</td>
<td>(<a href="#bib18">Taverna et al., 2020</a>)</td>
<td><a href="https://www.vibcancer.be/software-tools/BIOMEX">https://www.vibcancer.be/software-tools/BIOMEX</a></td>
</tr>
<tr>
<td>Cell Ranger; version 2.2.0</td>
<td>10x Genomics</td>
<td>tenx, RRID:SCR_01695</td>
</tr>
<tr>
<td>FindClusters (Seurat package; version 2.3.4)</td>
<td>(<a href="#bib17">Satija et al., 2015</a>)</td>
<td>(Seurat; RRID: SCR_016341)</td>
</tr>
<tr>
<td>flashpcaR<i>;</i> version <i>2.0</i>
</td>
<td>(<a href="#bib1">Abraham et al., 2017</a>)</td>
<td><a href="https://github.com/gabraham/flashpca/releases">https://github.com/gabraham/flashpca/releases</a></td>
</tr>
<tr>
<td>FlowJo (version 8.8.6)</td>
<td><a href="https://www.flowjo.com">https://www.flowjo.com</a></td>
<td>(FlowJo, RRID: SCR_008520)</td>
</tr>
<tr>
<td>NormalizeData (Seurat package; version 2.3.4)</td>
<td>(<a href="#bib17">Satija et al., 2015</a>)</td>
<td>(Seurat; RRID: SCR_016341)</td>
</tr>
<tr>
<td>Rtsne package; version 0.15</td>
<td>(<a href="#bib19">Van Der Maaten and Hinton, 2008</a>)</td>
<td><a href="https://cran.r-project.org/web/packages/Rtsne/index.html">https://cran.r-project.org/web/packages/Rtsne/index.html</a></td>
</tr>
<tr>
<td>Seurat FindVariableGenes</td>
<td>(<a href="#bib17">Satija et al., 2015</a>)</td>
<td>(Seurat; RRID: SCR_016341)</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>40 μm Cell strainer</td>
<td>Sigma-Aldrich</td>
<td>Cat#CLS431750-50EA</td>
</tr>
<tr>
<td>70 μm Cell strainer</td>
<td>Sigma-Aldrich</td>
<td>Cat#CLS431751-50EA</td>
</tr>
<tr>
<td>100 μm Cell strainer</td>
<td>Sigma-Aldrich</td>
<td>Cat#CLS431752-50EA</td>
</tr>
<tr>
<td>BD FACSAria™ III sorter</td>
<td>BD Biosciences</td>
<td>N/A</td>
</tr>
<tr>
<td>Centrifuge tube, conical, HDPE CentriStar™, PP, 15 mL</td>
<td>VWR</td>
<td>Cat#734-1867</td>
</tr>
<tr>
<td>Centrifuge tube, conical, HDPE CentriStar™, PP, 50 mL</td>
<td>VWR</td>
<td>Cat#734-1869</td>
</tr>
<tr>
<td>gentleMACS™ C Tubes</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-093-237</td>
</tr>
<tr>
<td>gentleMACS™ Octo Dissociator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-095-937</td>
</tr>
<tr>
<td>gentleMACS™ Dissociator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-093-235</td>
</tr>
<tr>
<td>Glass Pasteur pipette with narrow tip</td>
<td>VWR</td>
<td>Cat#612-3813</td>
</tr>
<tr>
<td>HulaMixer™ Sample Mixer</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15920D</td>
</tr>
<tr>
<td>LS columns</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-401</td>
</tr>
<tr>
<td>MS columns</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-201</td>
</tr>
<tr>
<td>MACS MultiStand</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-303</td>
</tr>
<tr>
<td>MiniMACS™ Separator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-090-312</td>
</tr>
<tr>
<td>Multipurpose centrifuge and microcentrifuge</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Perfusion pump: Perfusor® fm (MFC)</td>
<td>B. Braun Malaysia</td>
<td>N/A</td>
</tr>
<tr>
<td>Needle 20G 1.5" × 100</td>
<td>Terumo Agani</td>
<td>Cat#AN-2038R</td>
</tr>
<tr>
<td>Surgical Scalpel Blade No 10</td>
<td>Swann-Morton</td>
<td>Cat#0201</td>
</tr>
<tr>
<td>Syringe 1 mL (without needle)</td>
<td>HSW HENKE-JECT®</td>
<td>Cat#8300014579</td>
</tr>
<tr>
<td>Syringe Pump Harvard Apparatus</td>
<td>Harvard Apparatus</td>
<td>Cat#PHD 22/2000</td>
</tr>
<tr>
<td>QuadroMACS™ Separator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-091-051</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Media and buffers</h3>
<p>The following media/buffers are required.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Protocols are tissue-specific, please read the instructions carefully before proceeding with the isolations.</div>
<ul>
<li>PBS-based <b>Wash Buffer</b> (necessary for all tissues) containing:
<ul>
<li>0.5% (w/v) BSA (BSA Fraction V, Sigma-Aldrich, Cat#10735086001)</li>
<li>2 mM EDTA (Sigma-Aldrich, Cat#ED2P-500G) in PBS (Thermo Fisher Scientific, Cat#14190-094)</li>
</ul>
</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>BSA</td>
<td>0.5%</td>
<td>2.5 g</td>
</tr>
<tr>
<td>EDTA</td>
<td>2 mM</td>
<td>0.2 mL (from 5 M pre-prepared stock solution, according to manufacturer's instructions)</td>
</tr>
<tr>
<td>PBS</td>
<td>-</td>
<td>499.8 mL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Wash Buffer can be prepared 1 day in advance and stored for up to 3 days at 4°C.</div>
<ul>
<li>For digestion of choroid and lung tissues prepare <b>Supplemented KnockOut</b><sup><b>TM</b></sup> <b>DMEM-medium</b> containing:
<ul>
<li>KnockOut<sup>TM</sup> DMEM-medium (Thermo Fisher Scientific, Cat#10829018)</li>
<li>1% (v/v) Penicillin/Streptomycin (Thermo Fisher Scientific, Cat#15140122)</li>
<li>2× Antibiotic-Antimycotic (Thermo Fisher Scientific, Cat#15240062)</li>
<li>1 mM Sodium Pyruvate (Thermo Fisher Scientific, Cat#11360070)</li>
<li>1× MEM Non-Essential Amino Acids Solution (MEM-NEAA) (Thermo Fisher Scientific, Cat#11140035)</li>
<li>1× Endothelial Cell Growth Factor supplements (ECGS/ Heparin) (PromoCell, Cat#C-30120)</li>
</ul>
</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KnockOut<sup>TM</sup> DMEM-medium</td>
<td>-</td>
<td>473 mL</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>1% (v/v)</td>
<td>5 mL</td>
</tr>
<tr>
<td>Antibiotic-Antimycotic</td>
<td>2×</td>
<td>10 mL</td>
</tr>
<tr>
<td>Sodium Pyruvate</td>
<td>1 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>MEM Non-Essential Amino Acids Solution</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>Endothelial Cell Growth Factor supplements (ECGS/ Heparin)</td>
<td>1×</td>
<td>2 mL (one vial)</td>
</tr>
</tbody>
</table>
<p>Please check the specific step-by-step protocols below for additional details. Supplemented KnockOut<sup>TM</sup> DMEM-medium can be stored at 4°C, up to 3 months in a sterile manner. The digestion enzymes (Collagenases, Dispase and DNase) have to be added freshly to the digestion medium before the start of the isolation procedures.</p>
<p>ECGS/Heparin is added to the media to sustain EC viability. If there is any indication that the growth factors included in ECGS might affect isolated cells and alter the results according to the designed experiments, these can be left out.</p>
<h3 id="sec2.2">Equipment</h3>
<p>The following equipment was used for the described protocols. Please check the protocols for specific instructions. For details about the equipment and alternatives please, check the section “equipment and reagent alternatives”.</p>
<ul>
<li>gentleMACS™ Octo Dissociator (Miltenyi Biotec, Cat#130-095-937)</li>
<li>gentleMACS™ Dissociator (Miltenyi Biotec, Cat#130-093-235)</li>
<li>HulaMixer™ Sample Mixer (Thermo Fisher Scientific, Cat#15920D)</li>
<li>MACS MultiStand (Miltenyi Biotec, Cat#130-042-303)</li>
<li>MiniMACS™ Separator (Miltenyi Biotec, Cat#130-090-312)</li>
<li>QuadroMACS™ Separator (Miltenyi Biotec, Cat#130-091-051)</li>
<li>Water bath or incubator adjusted to 37°C (see specific protocols for details)</li>
<li>Multipurpose- and Micro-centrifuge (see specific protocols for details)</li>
<li>BD FACSAria™ III sorter</li>
<li>Perfusion pump: Perfusor® fm (MFC) - B. Braun Malaysia or Harvard Apparatus PHD 22/2000.</li>
</ul>
<h3 id="sec2.3">Equipment and reagent alternatives:</h3>
<ul>
<li>GentleMACS™ Dissociator (Miltenyi Biotec, Cat#130-093-235) used in the digestion protocols of lung tissue (step 12d) can be replaced by gentleMACS™ Octo Dissociator (Miltenyi Biotec, Cat#130- 095-937). Use of the gentleMACS™ (Octo) Dissociator has been optimized for this protocol to reach the highest percentage of isolated ECs from lungs. Using another mechanical tissue dissociation method (i.e., scalpel blades or razor blades) can be optional, however any deviations from the described method need to be further optimized and we cannot guarantee the expected outcome.</li>
<li>We recommend the use of a HulaMixer™ Sample Mixer (Thermo Fisher Scientific, Cat#15920D) for the digestion of brain tissue (steps 2f, 2j and 2l) and the digestion of muscle tissue (step 17e), however other orbital tube rotators may be used after adjustment of the rotation speed.</li>
<li>MACS® Columns contain a matrix composed of superparamagnetic spheres, which are covered with a cell-friendly coating. When the column is placed in a MACS Separator, the spheres amplify the magnetic field by 10.000-fold. MS columns (Miltenyi Biotec, Cat#130-042-201) and LS columns (Miltenyi Biotec, Cat#130-042-401) are designed for positive selection and depletion of strongly magnetically labeled cells. The difference between both is the loading capacity of total and labeled cells (for MS column: labeled cells: up to 1×10<sup>7</sup> and total cells: up to 2×10<sup>8</sup>. For LS column: labeled cells: up to 1×10<sup>8</sup> and total cells: up to 2×10<sup>9</sup>). MS columns (Miltenyi Biotec, Cat#130-042-201) can be used with the MiniMACS™ Separator (Miltenyi Biotec, Cat#130-090-312) or the OctoMACS™ Separator (Miltenyi Biotec, Cat#130-042-108). LS columns (Miltenyi Biotec, Cat#130-042-401) can be used with the QuadroMACS™ Separator (Miltenyi Biotec, Cat#130-091-051) or MidiMACS™ Separator (Miltenyi Biotec, Cat#130-042-302). Miltenyi Biotec also offers alternative columns, such as LD columns (Miltenyi Biotec, Cat#130-042-901; designed for depletion of even weakly labeled cells) or AutoMACS columns (Miltenyi Biotec, Cat#130-021-101; designed both for positive and negative selection). However, using other columns than the ones mentioned in the protocols may require further optimization. Additionally, non-column based magnetic isolations methods are available (e.g.: from STEMCELL Technologies) that could be optimized and used as alternatives for depletion and enrichment purposes. Of note, the myelin, immune and epithelial cell depletion with MicroBeads in the above protocols is performed with LS columns (Miltenyi Biotec, Cat#130-042-401) or MS columns (Miltenyi Biotec, Cat#130-042-201). This means that some weakly labeled, yet positive cells may not be depleted from the cell suspension. These non-endothelial and contaminating cells will be depleted during FACS sorting.</li>
<li>In the above protocols we suggest to use the following reagents:
<ul>
<li>antibodies: CD31 (PECAM-1) Monoclonal Antibody (390), e.g.: CD31-FITC (Thermo Fisher Scientific, Cat#11-0311-82; 1:300), CD45 Monoclonal Antibody (30-F11), e.g.: CD45-PE-Cyanine7 (Thermo Fisher Scientific, Cat#25-0451-82; 1:700);</li>
<li>viability dye: e.g.: eFluor 450 (Thermo Fisher Scientific, Cat#65-0863-18; 1:1000).</li>
</ul>
</li>
</ul>
<p>Different antibodies with similar properties (antigen and clone of the antibody), or with different clonality, conjugated with fluorochrome of choice can be used. Please note that staining efficiency depends on the clonality, fluorochrome, vendor and/or the LOT number of the antibody as well as on the sorter used for FACS. We recommend performing optimization of the staining panel and antibody titration.</p>
<ul>
<li>The BD FACSAria™ III sorter was used to perform fluorescent activated sorting. However, any other sorter that allows to sort the cells at the required optical paths can be used. Please, note that the settings of sorting are specific for the brand and model of the sorter and optimization of the procedure always needs to be performed.</li>
<li>A perfusion pump Perfusor® fm (MFC) - B. Braun Malaysia or Harvard Apparatus PHD 22/2000 was used to perform the transcardial perfusion, however any other perfusion pump that allows the setting described in the protocols can be used.</li>
<li>Digestion efficiency may depend on the vendor and/or the LOT number of the enzymes used. We recommend to test all newly purchased reagents before performing final experiments.</li>
<li>Please note that, depending on the intended further use of the isolated ECs, working in a laminar flow hood and applying sterile lab-practice may be necessary. For example, if the cells will be used for cell culture purposes, all the reagents for isolation should be kept and used in a sterile manner and we would recommend to perform the EC isolation under a laminar flow cabinet to avoid any external contamination. However, if sterility of the purified ECs is not necessary after the isolation (e.g., nucleic acid isolation, for sequencing, Western blotting, cell staining) the protocols can be performed outside the laminar flow cabinet.</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Brain endothelial cell isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 h 30 min – 6 h. For preparation of brain digestion buffer, 30 min. For digestion of brain tissue, 1 h 30 min. For myelin depletion using murine Myelin Removal MicroBeads, 75 min. For endothelial cell enrichment using CD31 murine MicroBeads, 45 min. For FACS, 1–2 h.</div>
<p><a href="#fig1">Figure 1</a>A shows a detailed scheme of brain ECs isolation.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. EC isolation from mouse brain</div>
<p>(A) Detailed scheme illustrating isolation of ECs from brain.</p>
<p>(B) Representative FACS plots for the gating strategy to sort ECs from brain based on sorting live/CD45<sup>-</sup>/CD31<sup>+</sup> cells.</p>
</figcaption>
</figure>
<ol>
<li>Preparation of brain digestion buffer
<ul>
<li>The following protocol, describing the isolation of brain ECs, includes the use of the Neural Tissue Dissociation kit (P) (Miltenyi Biotec, Cat#130-092-628), which allows mechanical and enzymatic dissociation of brain tissue before removal of myelin (as described previously (<a href="#bib20">Vanlandewijck et al., 2018</a>)). Reagents of this kit need to be prepared in advance according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/neural-tissue-dissociation-kits.html#gref">https://www.miltenyibiotec.com/US-en/products/neural-tissue-dissociation-kits.html#gref</a>). The protocol is designed to process the entire brain (without olfactory bulb). Described volumes are optimized for the protocol and suffice for 2 (pooled) adult murine brains.</li>
</ul>
</li>
</ol>
<ol start="2">
<li>Digestion of brain tissue
<ol type="a">
<li>Prepare enzyme mix 1 (mix 5.7 mL of buffer X with 150 μL of enzyme P from the Neural Tissue Dissociation kit (P) (Miltenyi Biotec, Cat#130-092-628)) and equilibrate to 37°C.</li>
<li>Before dissecting the brain, perform transcardial perfusion via the left ventricle with ice-cold PBS at a perfusion rate of 2 mL/min for 5 min.</li>
<li>Remove the brain surgically and place it in ice-cold Dulbecco’s modified Eagle’s medium (DMEM, high glucose, Thermo Fisher Scientific, Cat#11965092) supplemented with 1× penicillin/streptomycin (Thermo Fisher Scientific, Cat#15140122).</li>
<li>Transfer the dissected brain to a Petri dish chilled on ice and mince the tissue with a scalpel blade until no more large pieces remain. The pieces should be small enough to pass through a cut P1000 pipette tip (approx. 0.8 mm diameter).</li>
<li>Transfer the brain pieces to a 15 mL conical tube with a cut P1000 pipette tip and mix them with enzyme mix 1.</li>
<li>Incubate for 17 min at 37°C on a tube rotator (e.g.: HulaMixer™ Sample Mixer (Thermo Fisher Scientific, Cat#15920D)) (max. 20 rpm).</li>
<li>During incubation prepare enzyme mix 2 (mix 60 μL of buffer Y with 30 μL of enzyme A from the Neural Tissue Dissociation kit (P)).</li>
<li>Take the cell suspension from the tube rotator and add enzyme mix 2.</li>
<li>Mix the suspension with a Pasteur pipette by pipetting up and down (10 times).
<div class="note">
<span class="note-title">Note:</span> When clogging occurs, remove the clog by tapping the tip of the Pasteur pipette against the bottom of 15 mL conical tube until the clog resolves.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The suspension should pass through the Pasteur pipette without clogging before proceeding to the next step.</div>
</li>
<li>Incubate for 12 min at 37°C with slow rotation (10 rpm) on the tube rotator (e.g.: HulaMixer™ Sample Mixer (Thermo Fisher Scientific, Cat#15920D)).</li>
<li>Pass the cell suspension firmly, but without creating bubbles or foam, through a 20G needle (use a 1 mL syringe). Repeat this step approximately 10 times.
<div class="critical">
<span class="critical-title">Critical:</span> The cell suspension should pass through the needle without clogging, and no major tissue pieces should be visible after this step.</div>
</li>
<li>Incubate the cell suspension for an additional 10 min at 37°C with slow rotation (10 rpm) on the tube rotator (e.g.: HulaMixer™ Sample Mixer (Thermo Fisher Scientific, Cat#15920D)).</li>
<li>Transfer the cell suspension to a 50 mL conical tube, add 40 mL of ice-cold HBSS (containing Mg<sup>2+</sup>, Ca<sup>2+</sup>) and filter through a 70 μm cell strainer.
<div class="critical">
<span class="critical-title">Critical:</span> From this moment on, the cell suspension should always be kept at 4°C.</div>
</li>
<li>Centrifuge the cell suspension at 300 <i>g</i> for 5 min at 4°C.</li>
<li>Transfer the supernatant to a new 50 mL conical tube, keep the pellet at 4°C.</li>
<li>Repeat steps 2n and 2o two more times for a total of three cycles of centrifugation.</li>
</ol>
</li>
</ol>
<ol start="3">
<li>Myelin depletion using murine Myelin Removal MicroBeads
<ol type="a">
<li>Pool the pellets from steps 2n to 2p resuspend in 3600 μL Wash Buffer and add 400 μL Myelin Removal Beads II (Miltenyi Biotec, Cat#130-096-733).</li>
<li>Mix and incubate at 4°C for 15 min.
<div class="note">
<span class="note-title">Note:</span> Process fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Add Wash Buffer to a total volume of 40 mL.</li>
<li>Centrifuge at 300 <i>g</i> for 5 min at 4°C, remove the supernatant.
<ol type="i">
<li>During the centrifugation prepare 3 collection tubes and LS columns (Miltenyi Biotec, Cat#130-042-401) according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ls-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ls-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 3 mL of Wash Buffer.</li>
</ol>
</li>
<li>Resuspend the pellet in 6 mL of Wash Buffer.</li>
<li>Add 1 mL of the cell suspension on each column (keep the rest of the cell suspension, ± 3 mL).
<div class="critical">
<span class="critical-title">Critical:</span> Collect the eluent containing the cells.</div>
</li>
<li>After the suspension has run through the column, wash the column twice by adding 1 mL of Wash Buffer once the column reservoir is empty.
<div class="critical">
<span class="critical-title">Critical:</span> Collect eluent to the conical tube from step (3f). This fraction contains ECs.</div>
</li>
<li>Prepare another 3 LS columns and repeat the procedure for the remaining sample (± 3 mL left).</li>
<li>Centrifuge the total eluent at 300 <i>g</i> for 5 min, remove the supernatant.</li>
<li>Pool the pellets in 2 mL of Wash Buffer and centrifuge at 300 <i>g</i> for 5 min, remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Endothelial cell enrichment using CD31 murine MicroBeads
<ol type="a">
<li>Resuspend the pellet in an appropriate amount of Wash Buffer and add the appropriate volume of CD31 MicroBeads (Miltenyi Biotec, Cat#130-097-418) to the suspension according to the manufacturer's instructions.
<div class="note">
<span class="note-title">Note:</span> We suggest to resuspend cells in 90 μL of Wash Buffer, determine the cell number and follow the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref">https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref</a>) i.e., for up to 1 × 10<sup>7</sup> total cells, resuspend the cells in 90 μL of Wash Buffer and add 10 μL of CD31 MicroBeads. If fewer than 1 × 10<sup>7</sup> cells are available use the same volumes as indicated above. When working with higher cell numbers than 1 × 10<sup>7</sup> cells, scale up all reagent volumes and total volumes accordingly.</div>
<div class="note">
<span class="note-title">Note:</span> In case the sample is too viscous or contains clumps, filter through a 40 μm cell strainer to prevent clogging of columns in the next steps (do not filter if not necessary).</div>
</li>
<li>Mix well and incubate for 20 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Process samples fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Wash the cells by adding 3 mL of Wash Buffer and centrifuge at 300 <i>g</i> for 5 min at 4°C.
<ol type="i">
<li>During the centrifugation prepare collection tubes and MS columns (Miltenyi Biotec, Cat#130-042-201) according to the manufacturer’s instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 0.5 mL of Wash Buffer.</li>
</ol>
</li>
<li>Remove the supernatant and resuspend the pellet in 0.5 mL of Wash Buffer.</li>
<li>Apply the cell suspension onto the prepared MS column through a 40 μm cell strainer.</li>
<li>Wash the MS column 3 times with 0.5 mL Wash Buffer, adding buffer each time once the column reservoir is empty.
<div class="note">
<span class="note-title">Note:</span> The eluent (CD31-negative fraction) from step 4f can be used to prepare controls for FACS analysis or can be discarded if not further needed.</div>
</li>
<li>Remove the MS column from the separator and place it onto a new 15 mL conical collection tube.</li>
<li>Pipet 1 mL Wash Buffer onto the MS column. Immediately flush out the fraction containing the magnetically labeled cells (CD31-positive fraction) by firmly applying the plunger supplied with the column.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>FACS
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all required controls for flow cytometry analysis beforehand i.e., unstained cell (US) control, viability control (VB), Fluorescence Minus One (FMO) control, isotype controls (optional), as well as the compensation controls (one for each fluorophore) for proper set up of the cytometer.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend to use the cells from the final step of the isolation as controls (US, VB, FMO, isotype controls). However, if the cell number is limited, one can use the cells from, e.g.: CD31-negative fraction, to prepare the controls. For compensation controls we would recommend to use compensation beads (e.g., OneComp eBeads™ Compensation Beads, Thermo Fisher Scientific, Cat#01-1111-41) to ensure clear fluorescence signal and sufficient compensation.</div>
<ol type="a">
<li>Centrifuge the cell suspension at 300 <i>g</i> for 5 min, remove the supernatant.</li>
<li>Resuspend the pellet in 500 μL Wash Buffer-based staining solution containing:
<ol type="i">
<li>CD31 (PECAM-1) Monoclonal Antibody (390), e.g.: CD31-FITC (Thermo Fisher Scientific, Cat#11-0311-82; 1:300).</li>
<li>CD45 Monoclonal Antibody (30-F11), e.g.: CD45-PE-Cyanine7 (Thermo Fisher Scientific, Cat#25-0451-82; 1:700).</li>
<li>Viability Dye, e.g.: eFluor 450 (Thermo Fisher Scientific, Cat#65-0863-18; 1:1000).</li>
</ol>
</li>
<li>Stain the cells for 30 min at 4°C in the dark.</li>
<li>Add 3 mL of Wash Buffer and centrifuge the stained cells at 300 <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend the pellet in Wash Buffer and proceed with FACS.</li>
<li>Sort viable, CD45<sup>-</sup>/CD31<sup>+</sup> cells to collection tubes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To ensure high viability of ECs, sort the ECs to an Eppendorf tube containing collection medium (10% (v/v) FBS in PBS, Thermo Fisher Scientific, Cat#A38401).</div>
<p><a href="#fig1">Figure 1</a>B shows representative FACS plots and gating strategy of viable, CD45<sup>-</sup>, CD31<sup>+</sup> brain ECs.</p>
<h3 id="sec3.2">Choroid endothelial cell isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼5 h 30 min. For preparation of choroid digestion buffer, 30 min. For digestion of choroid tissue, 1 h 30 min. For immune cell depletion using CD45 murine MicroBeads, 45 min. For cell enrichment using CD31 murine MicroBeads, 45 min. For FACS, 1–2 h.</div>
<p><a href="#fig2">Figure 2</a>A shows a detailed scheme of choroid ECs isolation.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. EC isolation from mouse choroid</div>
<p>(A) Detailed scheme illustrating isolation of ECs from choroid.</p>
<p>(B) Representative FACS plots for the gating strategy to sort ECs from choroid based on sorting live/CD45<sup>-</sup>/CD31<sup>+</sup> cells.</p>
</figcaption>
</figure>
<ol start="6">
<li>Preparation of choroid digestion buffer
<ol type="a">
<li>Right before isolation prepare choroid digestion buffer containing:
<ol type="i">
<li>Supplemented KnockOut<sup>TM</sup> DMEM-medium (see details in section ‘<a href="#materials-and-equipment">materials and equipment</a>’)</li>
<li>0.3% (w/v) Collagenase I</li>
<li>0.25 U/mL Dispase</li>
<li>7.5 μg/mL DNAse I
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Supplemented KnockOut<sup>TM</sup> DMEM-medium</td>
<td>-</td>
<td>8.925 mL</td>
</tr>
<tr>
<td>Collagenase I</td>
<td>0.3% (w/v)</td>
<td>30 mg</td>
</tr>
<tr>
<td>Dispase</td>
<td>0.25 U/mL</td>
<td>1 mL (from 2.5 U/mL stock solution prepared according to manufacturer's instructions)</td>
</tr>
<tr>
<td>DNAse I</td>
<td>7.5 μg/mL</td>
<td>75 μL (from 1 mg/mL stock solution prepared according to manufacturer's instructions)</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Store the freshly prepared choroid digestion buffer at 4°C until needed.</li>
<li style="list-style-type: none;">5 mL of choroid digestion buffer suffices for choroids from 6 adult mice.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Digestion of choroid tissue
<div class="note">
<span class="note-title">Note:</span> prior transcardial perfusion is not required for isolation of choroid ECs.</div>
<div class="note">
<span class="note-title">Note:</span> We suggest to perform dissection of choroids as follows: Sacrifice mice by cervical dislocation and collect eyes by inserting scissors along the eye into the orbital cavity. Cut the four optical muscles and the optic nerve, which appear as a white cord behind the eye. Then place the eyes in PBS and remove periocular tissue under a microscope. Dissect the retinal pigment epithelium (RPE)-choroid-sclera complex from the enucleated eyes by peeling off the vitreous body and retina.</div>
<ol type="a">
<li>Transfer dissected choroids to a 50 mL conical tube containing 5 mL of choroid digestion buffer.</li>
<li>Pipet the choroids vigorously using a P1000 micropipette.</li>
<li>Incubate in a 37°C water bath for 30–40 min.
<ol type="i">
<li>During the incubation, pipet the choroids vigorously using a P1000 micropipette every 5–10 min.</li>
</ol>
</li>
<li>Stop the enzymatic reaction by adding 5 mL of Wash Buffer.</li>
<li>Filter the cells through a 100 μm cell strainer into a new 50 mL conical tube.</li>
<li>Rinse the used conical tube with an additional 3 mL of Wash Buffer and filter the suspension through the 100 μm cell strainer.</li>
<li>Transfer the cell suspension to a 15 mL conical tube and centrifuge at 300 <i>g</i> for 7 min.</li>
<li>Remove the supernatant and wash the cells with 5 mL of Wash Buffer.</li>
<li>Centrifuge the cell suspension at 300 <i>g</i> for 5 min, remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Immune cell depletion using CD45 murine MicroBeads
<ol type="a">
<li>Resuspend the pellet in an appropriate amount of Wash Buffer and add the appropriate volume of CD45 MicroBeads (Miltenyi Biotec, Cat#130-052-301) to the suspension according to the manufacturer's instructions.
<div class="note">
<span class="note-title">Note:</span> We suggest to resuspend cells in 90 μL of Wash Buffer, determine the cell number and follow the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/cd45-microbeads-mouse.html?countryRedirected=1#gref">https://www.miltenyibiotec.com/US-en/products/cd45-microbeads-mouse.html?countryRedirected=1#gref</a>) i.e., for up to 1 × 10<sup>7</sup> total cells, resuspend the cells in 90 μL of Wash Buffer and add 10 μL of CD45 MicroBeads. If fewer than 1 × 10<sup>7</sup> cells are available use the same volumes as indicated above. When working with higher cell numbers than 1 × 10<sup>7</sup> cells, scale up all reagent volumes and total volumes accordingly.</div>
</li>
<li>Mix well and incubate for 15 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Process samples fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Wash cells by adding 3 mL of Wash Buffer and centrifuge at 300 <i>g</i> for 5 min at 4°C.
<ol type="i">
<li>During the centrifugation prepare collection tubes and MS columns (Miltenyi Biotec, Cat#130-042-201) according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 0.5 mL of Wash Buffer.</li>
</ol>
</li>
<li>Rinse MS columns with 0.5 mL of Wash Buffer.</li>
<li>Remove the supernatant and resuspend the pellet in 0.5 mL of Wash Buffer.</li>
<li>Apply the cell suspension onto the prepared MS column through a 40 μm cell strainer.</li>
<li>Wash the MS column 3 times with 0.5 mL Wash Buffer adding buffer each time once the column reservoir is empty.</li>
<li>Collect the eluent from steps 8f and 8g to the conical tube. This fraction contains ECs.</li>
<li>Centrifuge the total eluent (CD45-negative fraction) at 300 <i>g</i> for 5 min, remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="9">
<li>Cell enrichment using CD31 murine MicroBeads
<ol type="a">
<li>Resuspend the pellet in an appropriate amount of Wash Buffer and add the appropriate volume of CD31 MicroBeads (Miltenyi Biotec, Cat#130-097-418) to the suspension according to the manufacturer's instructions.
<div class="note">
<span class="note-title">Note:</span> We suggest to resuspend the cells in 90 μL of Wash Buffer, determine the cell number and follow the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref">https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref</a>) i.e., for up to 1 × 10<sup>7</sup> total cells resuspend cells in 90 μL of Wash Buffer and add 10 μL of CD31 MicroBeads. If fewer than 1 × 10<sup>7</sup> cells are available use the same volumes as indicated above. When working with higher cell numbers than 1 × 10<sup>7</sup> cells, scale up all reagent volumes and total volumes accordingly.</div>
</li>
<li>Mix and incubate for 15 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Process fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Wash cells by adding 3 mL of Wash Buffer and centrifuge at 300 <i>g</i> for 5 min at 4°C.
<ol type="i">
<li>During the centrifugation prepare collection tubes and MS columns (Miltenyi Biotec, Cat#130-042-201) according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 0.5 mL of Wash Buffer.</li>
</ol>
</li>
<li>Remove the supernatant and resuspend the pellet in 0.5 mL of Wash Buffer.</li>
<li>Apply the cell suspension onto the prepared MS columns through a 40 μm cell strainer to remove potential cell aggregates in order to prevent clogging of the column.</li>
<li>Wash the MS column 3 times with 0.5 mL Wash Buffer adding buffer each time once the column reservoir is empty.
<div class="note">
<span class="note-title">Note:</span> The eluent (CD31-negative fraction) from step 9f can be used to prepare controls for FACS analysis or can be discarded if not further needed.</div>
</li>
<li>Remove the MS column from the separator and place it onto a new 15 mL conical collection tube.</li>
<li>Pipet 1 mL Wash Buffer onto the MS column. Immediately flush out the fraction containing the magnetically labeled cells (CD31-positive fraction) by firmly applying the plunger supplied with the column.</li>
</ol>
</li>
</ol>
<ol start="10">
<li>FACS
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all required controls for flow cytometry analysis beforehand i.e., unstained cell (US) control, viability control (VB), Fluorescence Minus One (FMO) control, isotype controls (optional), as well as the compensation controls (one for each fluorophore) for proper set up of the cytometer.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend to use the cells from the final step of the isolation as controls (US, VB, FMO, isotype controls). However, if the cell number is limited, one can use the cells from, e.g.: CD31-negative fraction, to prepare the controls. For compensation controls we would recommend to use compensation beads (e.g., OneComp eBeads™ Compensation Beads, Thermo Fisher Scientific, Cat#01-1111-41) to ensure clear fluorescence signal and sufficient compensation.</div>
<ol type="a">
<li>Centrifuge the cell suspension at 300 <i>g</i> for 5 min, remove the supernatant.</li>
<li>Resuspend the pellet in 500 μL Wash Buffer-based staining solution containing:
<ol type="i">
<li>CD31 (PECAM-1) Monoclonal Antibody (390), e.g.: CD31-FITC (Thermo Fisher Scientific, Cat#11-0311-82; 1:300).</li>
<li>CD45 Monoclonal Antibody (30-F11), e.g.: CD45-PE-Cyanine7 (Thermo Fisher Scientific, Cat#25-0451-82; 1:700).</li>
<li>Viability Dye, e.g.: eFluor 450 (Thermo Fisher Scientific, Cat#65-0863-18; 1:1000).</li>
</ol>
</li>
<li>Stain the cells for 30 min at 4°C in the dark.</li>
<li>Add 3 mL of Wash Buffer and centrifuge the stained cells at 300 <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend the pellet in Wash Buffer and proceed with FACS.</li>
<li>Sort viable, CD45<sup>-</sup>/CD31<sup>+</sup> cells to collection tubes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To ensure high viability of ECs, sort the ECs to an Eppendorf tube containing collection medium (10% (v/v) FBS in PBS, Thermo Fisher Scientific, Cat#A38401).</div>
<p><a href="#fig2">Figure 2</a>B shows representative FACS plots and gating strategy of viable, CD45<sup>-</sup>, CD31<sup>+</sup> choroid ECs.</p>
<h3 id="sec3.3">Lung endothelial cell isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼5 h For preparation of lung digestion buffer, 30 min. For digestion of lung tissue, 1 h. For immune and epithelial cell depletion using CD45 and EpCAM murine MicroBeads, 45 min. For endothelial cell enrichment using CD31 murine MicroBeads, 45 min. For FACS, 1–2 h.</div>
<p><a href="#fig3">Figure 3</a>A shows a detailed scheme of lung ECs isolation.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. EC isolation from mouse lung</div>
<p>(A) Detailed scheme illustrating isolation of ECs from lung.</p>
<p>(B) Representative FACS plots for the gating strategy to sort ECs from lung based on sorting live/CD45<sup>-</sup>/CD31<sup>+</sup> cells.</p>
</figcaption>
</figure>
<ol start="11">
<li>Preparation of lung digestion buffer
<ol type="a">
<li>Right before isolation prepare lung digestion buffer containing:
<ol type="i">
<li>Supplemented KnockOut<sup>TM</sup> DMEM-medium (see details in section ‘<a href="#materials-and-equipment">materials and equipment</a>’)</li>
<li>0.1% (w/v) Collagenase II</li>
<li>0.25% (w/v) Collagenase IV</li>
<li>15 μg/mL DNase I
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Supplemented KnockOut<sup>TM</sup> DMEM-medium</td>
<td>-</td>
<td>8.850 mL</td>
</tr>
<tr>
<td>Collagenase II</td>
<td>0.1% (w/v)</td>
<td>10 mg</td>
</tr>
<tr>
<td>Collagenase IV</td>
<td>0.25% (w/v)</td>
<td>25 mg</td>
</tr>
<tr>
<td>DNAse I</td>
<td>15 μg/mL</td>
<td>150 μL (from 1 mg/mL stock solution prepared according to manufacturer's instructions)</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Store freshly prepared lung digestion buffer at 4°C until needed.</li>
<li style="list-style-type: none;">5 mL of lung digestion buffer suffices for lungs from 1 adult mouse.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Digestion of lung tissue
<ol type="a">
<li>Before dissecting the lungs, perform transcardial perfusion via the right ventricle with ice-cold PBS at a perfusion rate of 2 mL/min for 5 min.</li>
<li>Dissect the lungs, cut the lungs (with small scissors or scalpel blade) into single lobes and rinse on a Petri dish with ice-cold PBS.</li>
<li>Place the lung lobes of 1 mouse in a gentleMACS C tube (Miltenyi Biotec, Cat#130-093-237) with 5 mL of lung digestion buffer. Keep the tube and the digestion buffer on ice.
<div class="critical">
<span class="critical-title">Critical:</span> Make sure the tube is closed tightly.</div>
</li>
<li>Put tubes in a MACS™ Dissociator (Miltenyi Biotec, Cat#130-093-235):
<ol type="i">
<li>Run m_lung_01 protocol (pre-programmed by manufacturer).</li>
<li>Incubate sample in digestion buffer in a 37°C water bath for 30 min.
<div class="note">
<span class="note-title">Note:</span> Shake the tube vigorously by hand every 5 min for faster tissue dissociation.</div>
</li>
<li>Run m_lung_02 protocol (pre-programmed by manufacturer).</li>
</ol>
</li>
<li>Filter the sample through a 40 μm cell strainer into a 50 mL conical tube, rinse the strainer with 5 mL Wash buffer (use a P1000 micropipette for both steps) and transfer the filtered suspension to a 15 mL conical tube.</li>
<li>Centrifuge the cell suspension at 250 <i>g</i> for 5 min and remove the supernatant.</li>
<li>Resuspend the pellet in 5 mL Wash Buffer, pipet up and down using a P1000 micropipette.</li>
<li>Centrifuge the cell suspension at 250 <i>g</i> for 5 min and remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Immune and epithelial cell depletion using CD45 and EpCAM murine MicroBeads
<ol type="a">
<li>Resuspend the pellet in an appropriate amount of Wash Buffer and add the appropriate volume of CD45 MicroBeads (Miltenyi Biotec, Cat#130-052-301) and EpCAM MicroBeads (Miltenyi Biotec, Cat#130-105-958) to the suspension according to the manufacturer's instructions.
<div class="note">
<span class="note-title">Note:</span> We suggest to resuspend cells in 80 μL of Wash Buffer, determine the cell number and follow the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/cd45-microbeads-mouse.html?countryRedirected=1#gref">https://www.miltenyibiotec.com/US-en/products/cd45-microbeads-mouse.html?countryRedirected=1#gref</a>; <a href="https://www.miltenyibiotec.com/US-en/products/cd326-epcam-microbeads-mouse.html#gref">https://www.miltenyibiotec.com/US-en/products/cd326-epcam-microbeads-mouse.html#gref</a>) i.e., for up to 1 × 10<sup>7</sup> total cells, resuspend cells in 80 μL of Wash Buffer and add 10 μL of CD45 MicroBeads and 10 μL of EpCAM MicroBeads. If fewer than 1 × 10<sup>7</sup> cells are available use the same volumes as indicated above. When working with higher cell numbers than 1 × 10<sup>7</sup> cells, scale up all reagent volumes and total volumes accordingly.</div>
</li>
<li>Mix well and incubate for 15 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Process samples fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Wash the cells by adding 3 mL of Wash Buffer and centrifuge at 250 <i>g</i> for 5 min at 4°C.
<ol type="i">
<li>During the centrifugation step, prepare collection tubes and LS columns (Miltenyi Biotec, Cat#130-042-401) according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ls-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ls-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 3 mL of Wash Buffer.</li>
</ol>
</li>
<li>Remove the supernatant and resuspend the pellet in 0.5 mL of Wash Buffer.</li>
<li>Apply the cell suspension onto onto a prepared LS column through a 40 μm cell strainer (to remove potential cell aggregates in order to prevent clogging of the column).
<div class="critical">
<span class="critical-title">Critical:</span> Collect the unlabeled cells (the eluent) in a 15 mL conical tube (this fraction contains the ECs).</div>
</li>
<li>Wash LS column 3 times with 3 mL Wash Buffer, adding buffer each time once the column reservoir is empty.</li>
<li>Collect eluent from steps 13e and 13f into the conical tube. This fraction contains ECs.</li>
<li>Centrifuge the total eluent at 250 <i>g</i> (CD45, EpCAM-negative fraction) for 5 min, remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Endothelial cell enrichment using CD31 murine MicroBeads
<ol type="a">
<li>Resuspend the pellet in an appropriate amount of Wash Buffer and add the appropriate volume of CD31 MicroBeads (Miltenyi Biotec, Cat#130-097-418) to the suspension according to the manufacturer's instructions.
<div class="note">
<span class="note-title">Note:</span> We suggest to resuspend cells in 90 μL of Wash Buffer, determine the cell number and follow the the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref">https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref</a>) i.e., for up to 1 × 10<sup>7</sup> total cells resuspend cells in 90 μL of Wash Buffer and add 10 μL of CD31 MicroBeads. If fewer than 1 × 10<sup>7</sup> cells are available use the same volumes as indicated above. When working with higher cell numbers than 1 × 10<sup>7</sup> cells, scale up all reagent volumes and total volumes accordingly.</div>
</li>
<li>Mix and incubate for 15 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Process the samples fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Wash the cells by adding 3 mL of Wash Buffer and centrifuge at 250 <i>g</i> for 5 min at 4°C.
<ol type="i">
<li>During the centrifugation prepare collection tubes and LS columns (Miltenyi Biotec, Cat#130-042-401) according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ls-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ls-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 3 mL of Wash Buffer.</li>
</ol>
</li>
<li>Remove the supernatant and resuspend the pellet in 0.5 mL of Wash Buffer.</li>
<li>Apply the cell suspension onto a prepared LS column through a 40 μm cell strainer and collect the unlabeled cells (the eluent) in a 15 mL conical tube (CD31-negative fraction).</li>
<li>Wash the LS column 3 times with 3 mL Wash Buffer, adding buffer each time once the column reservoir is empty.</li>
<li style="list-style-type: none;">
<b><i>Note:</i></b> The eluent (CD31-negative fraction) from step 14f can be used to prepare controls for FACS analysis or discarded if not further needed.</li>
<li>Remove the LS column from the separator and place it onto a new 15 mL conical collection tube.</li>
<li>Pipet 5 mL Wash Buffer onto the LS column. Immediately flush out the fraction containing the magnetically labeled cells (CD31-positive fraction) by firmly applying the plunger supplied with the column.</li>
</ol>
</li>
</ol>
<ol start="15">
<li>FACS
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all required controls for flow cytometry analysis beforehand i.e., unstained cell (US) control, viability control (VB), Fluorescence Minus One (FMO) control, isotype controls (optional), as well as the compensation controls (one for each fluorophore) for proper set up of the cytometer.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend to use the cells from the final step of the isolation as controls (US, VB, FMO, isotype controls). However, if the cell number is limited, one can use the cells from, e.g.: CD31-negative fraction, to prepare the controls. For compensation controls we would recommend to use compensation beads (e.g., OneComp eBeads™ Compensation Beads, Thermo Fisher Scientific, Cat#01-1111-41) to ensure clear fluorescence signal and sufficient compensation.</div>
<ol type="a">
<li>Centrifuge the cell suspension at 250 <i>g</i> for 5 min, remove the supernatant.</li>
<li>Resuspend the pellet in 0.5 mL Wash Buffer-based staining solution containing:
<ol type="i">
<li>CD31 (PECAM-1) Monoclonal Antibody (390), e.g.: CD31-FITC (Thermo Fisher Scientific, Cat#11-0311-82; 1:300).</li>
<li>CD45 Monoclonal Antibody (30-F11), e.g.: CD45-PE-Cyanine7 (Thermo Fisher Scientific, Cat#25-0451-82; 1:700).</li>
<li>Viability Dye, e.g.: eFluor 450 (Thermo Fisher Scientific, Cat#65-0863-18; 1:1000).</li>
</ol>
</li>
<li>Stain cells for 30 min at 4°C in the dark.</li>
<li>Add 3 mL of Wash Buffer and centrifuge the stained cells at 250 <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend the pellet in Wash Buffer and proceed with FACS.</li>
<li>Sort viable, CD45<sup>-</sup>/CD31<sup>+</sup> cells to collection tubes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To ensure high viability of ECs, sort the ECs to an Eppendorf tube containing collection medium (10% (v/v) FBS in PBS, Thermo Fisher Scientific, Cat#A38401).</div>
<div class="note">
<span class="note-title">Note:</span> To determine the efficiency of epithelial cell depletion by EpCAM MicroBeads (Miltenyi Biotec, Cat#130-105-958) an antibody against CD326 (EpCAM) can be added to the staining panel (e.g.: CD326 (EpCAM)-PE Monoclonal Antibody (G8.8), eBioscience™ Cat#:12-5791-82; 1:500). However, we would recommend optimization of the staining. We would like to acknowledge that the EpCAM staining is optional and might influence the FACS sorting of ECs.</div>
<p><a href="#fig3">Figure 3</a>B shows representative FACS plots and gating strategy of viable, CD45<sup>-</sup>, CD31<sup>+</sup> lung ECs.</p>
<h3 id="sec3.4">Muscle endothelial cell isolation (m. soleus and m. extensor digitorum longus [EDL])</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼4 h 45 min. For preparation of muscle digestion buffer, 30 min. For digestion of muscle tissue, 1 h 30 min. For endothelial cell enrichment using CD31 murine MicroBeads, 45 min. For FACS, 1–2 h.</div>
<p><a href="#fig4">Figure 4</a>A shows a detailed scheme of muscle ECs isolation.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. EC isolation from mouse muscle</div>
<p>(A) Detailed scheme illustrating isolation of ECs from muscle.</p>
<p>(B and C) Representative FACS plots for the gating strategy to sort ECs from EDL (B) and soleus (C) muscle based on sorting live/CD45<sup>-</sup>/CD31<sup>+</sup> cells.</p>
</figcaption>
</figure>
<ol start="16">
<li>Preparation of muscle digestion buffer
<ol type="a">
<li>Right before isolation prepare muscle digestion buffer containing:
<ol type="i">
<li>PBS</li>
<li>0.2% (w/v) Collagenase IV</li>
<li>1.25 U/mL Dispase</li>
<li>2 mM CaCl<sub>2</sub>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>-</td>
<td>4.980 mL</td>
</tr>
<tr>
<td>Collagenase IV</td>
<td>0.2% (w/v)</td>
<td>25 mg</td>
</tr>
<tr>
<td>Dispase</td>
<td>1.25 U/mL</td>
<td>5 mL (from 2.5 U/mL stock solution prepared according to manufacturer's instructions)</td>
</tr>
<tr>
<td>CaCl<sub>2</sub>
</td>
<td>2 mM</td>
<td>20 μL (from 1 M stock solution CaCl<sub>2</sub> prepared according to manufacturer's instructions)</td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
<li>Store the freshly prepared muscle digestion buffer at 4°C until needed.</li>
<li style="list-style-type: none;">10 mL of muscle digestion buffer suffices for soleus from 8 adult mice or EDL from 8 adult mice.</li>
</ol>
</li>
</ol>
<ol start="17">
<li>Digestion of muscle tissue
<div class="note">
<span class="note-title">Note:</span> prior transcardial perfusion is not required for isolating muscle ECs.</div>
<ol type="a">
<li>Dissect soleus and EDL muscle from tendon to tendon as is described in (<a href="#bib10">Hakim et al., 2013</a>) (<a href="https://www.jove.com/video/50183/evaluation-muscle-function-extensor-digitorum-longus-muscle-ex-vivo">https://www.jove.com/video/50183/evaluation-muscle-function-extensor-digitorum-longus-muscle-ex-vivo</a> (from min 1:57)).
<ol type="i">
<li>Keep the soleus muscles separate from the EDL muscles for the entire procedure.</li>
</ol>
</li>
<li>Put the muscle tissue on an ice-cold Petri dish containing 1 mL of muscle digestion buffer.</li>
<li>Cut into pieces with surgical blades until a homogenous paste-like slurry is formed.
<div class="note">
<span class="note-title">Note:</span> Process samples fast, keep the cutting time under 10 min in order to keep good viability of the cells.</div>
</li>
<li>Put ice-cold muscle digestion buffer (9 mL of digestion buffer per 16 muscles) onto the sample and transfer into a 15 mL conical tube.</li>
<li>Rotate the conical tube on the tube rotator (e.g.: HulaMixer™ Sample Mixer (Thermo Fisher Scientific, Cat#15920D)) at 12 rpm for 45 min at 37°C.
<ol type="i">
<li>Pipet up and down every 15 min with a Pasteur pipette.</li>
</ol>
</li>
<li>Add 10 mL of Wash Buffer to stop the digestion.</li>
<li>Filter through a 70 μm cell strainer and then through a 40 μm cell strainer.</li>
<li>Centrifuge the sample at 300 <i>g</i> for 20 min, remove the supernatant.</li>
</ol>
</li>
</ol>
<ol start="18">
<li>Endothelial cell enrichment using CD31 murine MicroBeads
<ol type="a">
<li>Resuspend the pellet in an appropriate amount of Wash Buffer and add the appropriate volume of CD31 MicroBeads (Miltenyi Biotec, Cat#130-097-418) to the suspension according to the manufacturer's instructions.
<div class="note">
<span class="note-title">Note:</span> We suggest to resuspend the cells in 90 μL of Wash Buffer, determine the cell number and follow the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref">https://www.miltenyibiotec.com/US-en/products/cd31-microbeads-mouse.html?countryRedirected=1#gref</a>) i.e. for up to 1 × 10<sup>7</sup> total cells, resuspend the cells in 90 μL of Wash Buffer and add 10 μL of CD31 MicroBeads. If fewer than 1 × 10<sup>7</sup> cells are available use the same volumes as indicated above. When working with higher cell numbers than 1 × 10<sup>7</sup> cells, scale up all reagent volumes and total volumes accordingly.</div>
</li>
<li>Mix well and incubate for 15 min at 4°C.
<div class="note">
<span class="note-title">Note:</span> Process the samples fast in order to avoid non-specific binding of beads at 20°C–22°C.</div>
</li>
<li>Wash the cells by adding 3 mL of Wash Buffer and centrifuge at 300 <i>g</i> for 5 min at 4°C.
<ol type="i">
<li>During the centrifugation prepare collection tubes and MS columns (Miltenyi Biotec, Cat# 130-042-201) according to the manufacturer's instructions (<a href="https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref">https://www.miltenyibiotec.com/US-en/products/ms-columns.html#gref</a>) i.e., place the column in the suitable magnetic separator and prepare the column by rinsing with 0.5 mL of Wash Buffer.</li>
</ol>
</li>
<li>Remove the supernatant and resuspend the pellet in 0.5 mL of Wash Buffer.</li>
<li>Apply the cell suspension to the prepared MS column and collect the unlabeled cells (the eluent) in a 15 mL conical tube (CD31-negative fraction).</li>
<li>Wash the MS column 3 times with 0.5 mL Wash Buffer adding buffer each time once the column reservoir is empty.
<div class="note">
<span class="note-title">Note:</span> The eluent (CD31-negative fraction) from step 18f can be used to prepare controls for FACS analysis or discarded if not further needed.</div>
</li>
<li>Remove the MS column from the separator and place onto a new 15 mL conical collection tube.</li>
<li>Pipet 1 mL Wash Buffer onto the MS column. Immediately flush out the fraction containing the magnetically labeled cells (CD31-positive fraction) by firmly applying the plunger supplied with the column.</li>
</ol>
</li>
</ol>
<ol start="19">
<li>FACS
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all required controls for flow cytometry analysis beforehand i.e., unstained cell (US) control, viability control (VB), Fluorescence Minus One (FMO) control, isotype controls (optional), as well as the compensation controls (one for each fluorophore) for proper set up of the cytometer.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend to use the cells from the final step of the isolation as controls (US, VB, FMO, isotype controls). However, if the cell number is limited, one can use the cells from, e.g.: CD31-negative fraction, to prepare the controls. For compensation controls we would recommend to use compensation beads (e.g., OneComp eBeads™ Compensation Beads, Thermo Fisher Scientific, Cat#01-1111-41) to ensure clear fluorescence signal and sufficient compensation.</div>
<ol type="a">
<li>Centrifuge the cell suspension at 300 <i>g</i> for 5 min, remove the supernatant.</li>
<li>Resuspend the pellet in 500 μL Wash Buffer-based staining solution containing:
<ol type="i">
<li>CD31 (PECAM-1) Monoclonal Antibody (390), e.g.: CD31-FITC (Thermo Fisher Scientific, Cat#11-0311-82; 1:300).</li>
<li>CD45 Monoclonal Antibody (30-F11), e.g.: CD45-PE-Cyanine7 (Thermo Fisher Scientific, Cat#25-0451-82; 1:700).</li>
<li>Viability Dye, e.g.: eFluor 450 (Thermo Fisher Scientific, Cat#65-0863-18; 1:1000).</li>
</ol>
</li>
<li>Stain the cells for 30 min at 4°C in the dark.</li>
<li>Add 3 mL of Wash Buffer and centrifuge the stained cells at 300 <i>g</i> for 5 min at 4°C, remove the supernatant.</li>
<li>Resuspend the pellet in Wash Buffer and proceed with FACS.</li>
<li>Sort viable, CD45<sup>-</sup>/CD31<sup>+</sup> cells to collection tubes.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To ensure high viability of ECs, sort the ECs to an Eppendorf tube containing collection medium (10% (v/v) FBS in PBS, Thermo Fisher Scientific, Cat#A38401).</div>
<p><a href="#fig4">Figures 4</a>B and 4C shows representative FACS plots and gating strategy of viable, CD45<sup>-</sup>, CD31<sup>+</sup> EDL (B) and soleus (C) muscle ECs.</p>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The above protocols describe consistent methods for isolating high purity blood vessel ECs from distinct tissues: brain, choroid, lung and skeletal muscle. The protocols involve mechanical and enzymatic digestion, CD31 MicroBeads enrichment and fluorescence-activated cell sorting. The final content of ECs in the cell suspension after digestion and depletion/enrichment with MicroBeads and before FACS sorting (in percentage of the number of events recorded during FACS sorting) varies from around 7.5% for EDL muscle, 35% for choroid and soleus to 50% for brain and lung. The viability of the sorted ECs measured immediately after FACS varies from 60% (for soleus muscle) to 97.5% (for brain). For details about the flow cytometry analysis and viability see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Mmc1.xls">Table S1</a>. <a href="#fig1">Figures 1</a>, <a href="#fig2">2</a>, <a href="#fig3">3</a>, and <a href="#fig4">4</a> show representative FACS plots and gating strategy for ECs sorting.</p>
<p>To confirm the quality and purity of isolated ECs we have used the scRNA-seq data generated from ECs isolated during construction of the “Single-Cell Transcriptome Atlas of Murine Endothelial Cells” (<a href="#bib11">Kalucka et al., 2020</a>) and clustered the cells per organ. The EC clusters and non-EC clusters were annotated based on the expression of known EC and non-EC marker genes, including <i>Pecam1</i> and <i>Cdh5</i> (vascular ECs), <i>Prox1</i> and <i>Lyve-1</i> (lymphatic ECs), <i>Col1a1</i> (fibroblasts), <i>Hba-a1</i>, <i>Hba-a2</i>, <i>Hbb-bs</i> (red blood cells), <i>Pdgfrb</i> (pericytes) and <i>Acta2</i> (smooth muscle cells). The isolation of brain ECs resulted in 92.8% pure, high-quality ECs expressing the canonical EC markers CD31 (<i>Pecam1</i>), VE-Cadherin (<i>Cdh5</i>), <i>Icam2</i>, Endoglin/CD105 (<i>Eng</i>) and VEGFR2 (<i>Kdr</i>). The 7.2% non-ECs cell lacked expression of EC markers and were contaminated with red blood cells (<a href="#fig5">Figures 5</a>A and 5B; <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Mmc1.xls">Table S1</a>). ECs from EDL muscle had 86.8% purity. The cluster of contaminating cells had low expression of EC markers, low quality due to the low number of genes expressed per cell and contained red blood cells (<a href="#fig6">Figures 6</a>A and 6B; <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Mmc1.xls">Table S1</a>). The cell isolation from lung and soleus muscle resulted in 86.4% and 88.7% high-quality, pure ECs respectively. Both organs contained clusters of contaminating cells with low quality (low number of genes expressed per cell), lacking canonical EC markers expression. Additionally, the lung had a small non-EC cluster (5.2%) expressing fibroblast markers (e.g. <i>Cola1a</i>) and soleus had a non-EC cluster (1.9%) expressing smooth muscle cell markers (<i>Acta2</i>) and pericyte markers (<i>Pdgfrb</i>) (<a href="#fig5">Figures 5</a>C, 5D, <a href="#fig6">6</a>C, and 6D; <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Mmc1.xls">Table S1</a>).</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Characterization of the brain and lung EC sample purity</div>
<p>(A) t-SNE visualization of scRNA-seq analyses on ECs isolated from mouse brain showing representative EC and non-EC gene markers expression. Red arrowheads are pointing at cells highly expressing the marker gene. Color scale: red, high expression; blue, low expression.</p>
<p>(B) Top: t-SNE visualization of brain (non-) ECs color coded per cell type. Red arrowheads are pointing at non-ECs. Bottom: bar plot illustrating the quantification of EC and non-ECs.</p>
<p>(C) t-SNE visualization of scRNA-seq analyses on ECs isolated from mouse lung showing representative EC and non-EC gene markers expression and the number of genes expressed per cells. Red arrowheads are pointing at cells highly expressing the marker gene (or cells with low gene number expression). Color scale: red, high expression; blue, low expression.</p>
<p>(D) Top: t-SNE visualization of lung (non-) ECs and contaminating cells color coded per cell type. Red arrowheads are pointing at non-ECs. Bottom: bar plot illustrating the quantification of ECs, non-ECs and contaminating cells.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Characterization of the EDL and soleus muscle EC sample purity</div>
<p>(A) t-SNE visualization of scRNA-seq analyses on ECs isolated from EDL muscle: representative EC and non-EC gene markers expression and the number of genes expressed per cells. Red arrowheads are pointing at cells highly expressing the marker gene (or cells with low gene number expression). Color scale: red, high expression; blue, low expression.</p>
<p>(B) Top: t-SNE visualization of EDL ECs and contaminating cells color coded per cell type. Bottom: bar plot illustrating the quantification of EC and contaminating cells.</p>
<p>(C) t-SNE visualization of scRNA-seq analyses on ECs isolated from soleus muscle: representative EC markers expression and the number of genes expressed per cells. Red arrowheads are pointing at cells highly expressing the marker gene (or cells with low gene number expression). Color scale: red, high expression; blue, low expression.</p>
<p>(D) Top: t-SNE visualization of soleus (non-) ECs and contaminating cells color coded per cell type. Red arrowheads are pointing at non-ECs. Bottom: bar plot illustrating the quantification of ECs, non-ECs and contaminating cells.</p>
</figcaption>
</figure>
<p>Of note, for choroid ECs isolation, the cells were subjected to immune cell depletion and EC enrichment, using CD45 and CD31 MACS MicroBeads respectively, prior to the library preparation and scRNA-seq. ECs were selected <i>in silico</i> (<a href="#bib15">Rohlenova et al., 2020</a>). Fluorescent activated cell sorting was performed for ECs isolation, optimization and scRNA-seq validation (for details see (<a href="#bib15">Rohlenova et al., 2020</a>)), thus no purity and viability data were collected after the FACS.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<h3 id="sec5.1">A brief overview of data processing and <i>in silico</i> EC selection (related to <a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>)</h3>
<ol>
<li>Generate gene expression matrices using the CellRanger software (10x Genomics).</li>
<li>Aggregate sample data using CellRanger software, and process raw data further in R (version 3.4.4).</li>
<li>Perform the following quality control steps on the pooled tissue datasets:
<ol type="a">
<li>genes expressed by fewer than 10 cells or with a row average of &lt; 0.002 should not be considered and therefore removed;</li>
<li>cells that expressed fewer than 300 genes (low quality), and cells that expressed over 4,000 genes (potential doublets) should be excluded from further analysis;</li>
<li>cells in which over 10% of unique molecular identifiers (UMIs) were derived from the mitochondrial genome should be removed.</li>
</ol>
</li>
<li>Normalize the data using the <i>NormalizeData</i> function as implemented in the <i>Seurat</i> package (<a href="#bib17">Satija et al., 2015</a>).</li>
<li>Cluster the cells per organ prior to <i>in silico</i> EC selection for each organ separately.</li>
<li>First, for EC selection, identify highly variable genes using the <i>Seurat FindVariableGenes</i> function (mean lower threshold = 0.0125, mean higher threshold = 8, dispersion threshold = 0.5.</li>
<li>Auto-scale the data (using highly variable genes only) and summarize by principal component analysis (PCA) using the <i>flashPCA</i> package (<a href="#bib1">Abraham et al., 2017</a>)</li>
<li>Visualize the data using t-Distributed Stochastic Neighbor Embedding (t-SNE, <i>Rtsne</i> package; top 8 principal components (PCs)) (<a href="#bib19">Van Der Maaten and Hinton, 2008</a>).</li>
<li>Perform graph-based clustering to cluster cells according to their gene expression profile using the <i>FindClusters</i> function in <i>Seurat</i> (clustering resolution = 1, k-nearest neighbors = 10).</li>
<li>Annotate EC clusters based on the expression of known EC and non-EC marker genes, including <i>Pecam1</i> and <i>Cdh5</i> (vascular ECs), <i>Prox1</i> and <i>Lyve-1</i> (lymphatic ECs), <i>Col1a1</i> (fibroblasts), <i>Hba-a1, Hba-a2, Hbb-bs</i> (red blood cells), <i>Pdgfrb</i> (pericytes) and <i>Acta2</i> (smooth muscle cells).</li>
</ol>
<p>Of note, the data processing and <i>in silico</i> EC selection (steps 4–9) was performed using algorithms implemented in the BIOMEX software (<a href="#bib18">Taverna et al., 2020</a>).</p>
<p>All raw sequencing data referred to in this study are available at ArrayExpress (ArrayExpress: E-MTAB-8077; brain, muscle, lung) (<a href="#bib11">Kalucka et al., 2020</a>) and ArrayExpress: E-MTAB-8119: choroid (<a href="#bib15">Rohlenova et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>We acknowledge a number of limitations related to the described above protocols. First, the protocols were established using 8-week-old male C57BL6/J mice, and therefore further adjustments might be needed to isolate ECs from different mouse strains (BALB/c, CD-1 or SCID), gender, or from mice at different developmental stages. Second, the protocols are optimized for blood vessel ECs and not for lymphatic ECs. Thus, further adjustments have to be made to specifically isolate lymphatic ECs. For additional information about isolation of lymphatic ECs from e.g.: murine lymph nodes and murine embryos we refer to (<a href="#bib8">Fujimoto et al., 2020</a>; <a href="#bib5">Crosswhite, 2018</a>). Third, ECs isolated using the protocols described above were used for multiple transcriptomics approaches (e.g.: bulk or single cell RNA sequencing). Additional optimizations should be performed in order to use isolated cells for other applications, e.g.: <i>in vitro</i> cell culture (<a href="#bib12">Marelli-Berg et al., 2000</a>; <a href="#bib4">Choi et al. 2015</a>). We do acknowledge several other published protocols regarding (endothelial) cell isolation from murine organs (e.g. brain: (<a href="#bib2">Assmann et al., 2017</a>; <a href="#bib6">Czupalla et al., 2018</a>; <a href="#bib16">Ruck et al., 2014</a>); muscle: (<a href="#bib13">Müntefering et al., 2019</a>); lung: (<a href="#bib3">Cheung and Marelli-Berg, 2018</a>; <a href="#bib14">Nakano et al., 2018</a>)). We would like to point out that our protocols' advantages and novelty mostly rely on the combination of a rapid method for fresh cell isolation, simplicity of the protocols, specific design per organ to get highest quantity of ECs and high purity and quality of cells directly after the isolation (see <a href="#fig5">Figures 5</a> and <a href="#fig6">6</a>). Our protocols can also be combined and allow for isolation of ECs from multiple tissues from the same animal. Additionally, estimated time in the protocols was established for isolation of one tissue type from up to 3 mice by one person. When isolating from higher number of animals the estimated time (or number of required people) for each step may increase.</p>
<p>Of note, some protocols described above may slightly differ from the isolation protocols published in (<a href="#bib11">Kalucka et al., 2020</a>; <a href="#bib15">Rohlenova et al., 2020</a>; <a href="#bib9">Goveia et al., 2020</a>) due to further optimization and adjustments to improve EC isolation efficiency. Additionally, if multiple organs will be isolated from the same mouse, we suggest to have one person assigned per single organ isolation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p><b>Low number of isolated ECs</b>. A low number of isolated ECs may occur due to i.) under- or over- digestion of the dissected tissue (steps 2, 7b-c, 12c-e, 17c-e) ii.) poor enrichment with CD31 MicroBeads (Miltenyi Biotec, Cat130-097-418) (steps 4, 9, 14, 18) or iii.) loss of ECs during cell sorting (e.g,. due to decreased cell viability or poor labeling with antibodies) (steps 5, 10, 15, 19).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>To avoid extensive loss of ECs, we recommend to optimize the mechanical steps of digestion (vigorous shaking by hand, pipetting, cutting with scalpel blade, rotation speed) in order to obtain the maximum possible number of ECs from each tissue. Moreover, we recommend to optimize the volume of CD31 MicroBeads (Miltenyi Biotec, Cat#130-097-418) as well as the antibody concentrations according to the manufacturer's instructions, to avoid loss of ECs during magnetic- and cell sorting, respectively. If the problem persists, we advise to use and pool additional mice for the isolation of ECs of the particular organ. Additionally, other positive and negative selection technologies could be optimized and used, for example non-column based magnetic isolations (STEMCELL Technologies).</p>
<h3 id="sec7.3">Problem 2</h3>
<p><b>Low viability of isolated ECs.</b> Low viability of ECs could be caused by the extended time of the isolation procedure, extended digestion time and the number of manipulations on the cells (steps 2, 3-4, 7c, 8-9, 12c-e, 13-14, 17e, 18).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>We recommend to process samples fast to reduce the time of isolation and to store cell solutions at 4°C or on ice during the isolation procedure. Optimal enzymatic digestion time is crucial for cell viability. If low EC viability persists during sorting, we advise to adjust the time of enzymatic digestion. Additionally, the concentration of the cell suspension should be adjusted when starting FACS for optimal flow rate to reduce sorting time.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Abraham, G., Qiu, Y., and Inouye, M. (2017). FlashPCA2: principal component analysis of Biobank-scale genotype datasets. Bioinformatics <i>33</i>, 2776-2778. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref1">View at publisher</a></p>
<p id="bib2">Assmann, J.C., Müller, K., Wenzel, J., Walther, T., Brands, J., Thornton, P., Allan, S.M., and Schwaninger, M. (2017). Isolation and cultivation of primary brain endothelial cells from adult mice. Bio Protoc. <i>7</i>, e2294. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref2">View at publisher</a></p>
<p id="bib3">Cheung, K.C.P. and Marelli-Berg, F.M. (2018). Isolation of microvascular endothelial cells. Bio Protoc. <i>8</i>, e2886. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref3">View at publisher</a></p>
<p id="bib4">Choi, S., Kim, J.A., Kim, K.C., and Suh, S.H. (2015). Isolation and in vitro culture of vascular endothelial cells from mice. Korean J. Physiol. Pharmacol. <i>19</i>, 35-42. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref4">View at publisher</a></p>
<p id="bib5">Crosswhite, P. (2018). Isolation of LYVE-1+ endothelial cells from mouse embryos. Bio Protoc. <i>8</i>, e2962. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref5">View at publisher</a></p>
<p id="bib6">Czupalla, C.J., Yousef, H., Wyss-Coray, T., and Butcher, E.C. (2018). Collagenase-based single cell isolation of primary murine brain endothelial cells using flow cytometry. Bio Protoc. <i>8</i>, e3092. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref6">View at publisher</a></p>
<p id="bib8">Fujimoto, N., He, Y., D’Addio, M., Tacconi, C., Detmar, M., and Dieterich, L.C. (2020). Single-cell mapping reveals new markers and functions of lymphatic endothelial cells in lymph nodes. bioRxiv. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref8">View at publisher</a></p>
<p id="bib9">Goveia, J., Rohlenova, K., Taverna, F., Treps, L., Conradi, L.-C., Pircher, A., Geldhof, V., De Rooij, L.P.M.H., Kalucka, J., Sokol, L., et al. (2020). An Integrated gene expression landscape profiling approach to identify lung tumor endothelial cell heterogeneity and angiogenic candidates. Cancer Cell <i>37</i>, 21-36.e13. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref9">View at publisher</a></p>
<p id="bib10">Hakim, C.H., Wasala, N.B., and Duan, D. (2013). Evaluation of muscle function of the extensor digitorum longus muscle ex vivo and tibialis anterior muscle in situ in mice. J. Vis. Exp. <a class="external-link" href="https://doi.org/10.3791/50183">https://doi.org/10.3791/50183</a></p>
<p id="bib11">Kalucka, J., De Rooij, L.P.M.H., Goveia, J., Rohlenova, K., Dumas, S.J., Meta, E., Conchinha, N.V., Taverna, F., Teuwen, L.-A., Veys, K., et al. (2020). Single-cell transcriptome atlas of murine endothelial cells. Cell <i>180</i>, 764-779.e20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref11">View at publisher</a></p>
<p id="bib12">Marelli-Berg, F.M., Peek, E., Lidington, E.A., Stauss, H.J., and Lechler, R.I. (2000). Isolation of endothelial cells from murine tissue. J. Immunol. Methods <i>244</i>, 205-215. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref12">View at publisher</a></p>
<p id="bib13">Müntefering, T., Michels, A.P.E., Pfeuffer, S., Meuth, S.G., and Ruck, T. (2019). Isolation of primary murine skeletal muscle microvascular endothelial cells. J. Vis. Exp. <a class="external-link" href="https://doi.org/10.3791/58901">https://doi.org/10.3791/58901</a></p>
<p id="bib14">Nakano, H., Nakano, K., and Cook, D.N. (2018). Isolation and purification of epithelial and endothelial cells from mouse lung. Methods Mol. Biol. (Clifton, N.J.) <i>1799</i>, 59-69. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref14">View at publisher</a></p>
<p id="bib15">Rohlenova, K., Goveia, J., García-Caballero, M., Subramanian, A., Kalucka, J., Treps, L., Falkenberg, K.D., De Rooij, L.P.M.H., Zheng, Y., Lin, L., et al. (2020). Single-cell RNA sequencing maps endothelial metabolic plasticity in pathological angiogenesis. Cell Metab. <i>31</i>, 862-877.e14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref15">View at publisher</a></p>
<p id="bib16">Ruck, T., Bittner, S., Epping, L., Herrmann, A.M., and Meuth, S.G. (2014). Isolation of primary murine brain microvascular endothelial cells. J. Vis. Exp. <i>93</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref16">View at publisher</a></p>
<p id="bib17">Satija, R., Farrell, J.A., Gennert, D., Schier, A.F., and Regev, A. (2015). Spatial reconstruction of single-cell gene expression data. Nat. Biotechnol. <i>33</i>, 495-502. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref17">View at publisher</a></p>
<p id="bib18">Taverna, F., Goveia, J., Karakach, T.K., Khan, S., Rohlenova, K., Treps, L., Subramanian, A., Schoonjans, L., Dewerchin, M., Eelen, G., et al. (2020). BIOMEX: an interactive workflow for (single cell) omics data interpretation and visualization. Nucleic Acids Res. <i>48</i>, W385-W394. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref18">View at publisher</a></p>
<p id="bib19">Van Der Maaten, L.J.P. and Hinton, G.E. (2008). Visualizing high-dimensional data using t-SNE. J. Machine Learn. Res. <i>9</i>, 27. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref19">View at publisher</a></p>
<p id="bib20">Vanlandewijck, M., He, L., Mae, M.A., Andrae, J., Ando, K., Del Gaudio, F., Nahar, K., Lebouvier, T., Lavina, B., Gouveia, L., et al. (2018). A molecular atlas of cell types and zonation in the brain vasculature. Nature <i>554</i>, 475-480. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00215-X/sref20">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to the lead contact, Peter Carmeliet (<a href="mailto:peter.carmeliet@kuleuven.be">peter.carmeliet@kuleuven.be</a>), or technical contact, Joanna Kalucka (<a href="mailto:joanna.kalucka@aias.au.dk">joanna.kalucka@aias.au.dk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>Original/source data for datatype in the paper is available in ArrayExpress: E-MTAB-8077 and ArrayExpress: E-MTAB-8119. The published articles by Kalucka et al. and Rohlenova et al. include all datasets generated or analyzed during this study.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/996-Mmc1.xls">Table S1. Fraction of cells associated to the gating settings with final yield of FACS sorted ECs, related to steps 5f, 10f, 15f, 19f</a></p>
<h3>Acknowledgments</h3>
<p>We acknowledge the help of A. Bouché and P. Vanwesemael for technical assistance. J.K., N.V.C., K.V., L.T., M.B., P.d.Z., and K.R. are supported by the Fonds voor Wetenschappelijk Onderzoek (<a href="https://doi.org/10.13039/501100003130">FWO</a>); J.K. by AIAS-CO-FUND II: GA: MSCA: 754513 and The Aarhus University Research Foundation; <a href="https://doi.org/10.13039/501100003554">Lundbeckfonden</a>: R307-2018-3667, <a href="https://doi.org/10.13039/501100002808">Carlsberg Fonden</a>: CF19-0687, Riisfort Fonden, A.P. Møller Fonden and Steno Diabetes Center Aarhus (SDCA). L.-A.T. by <a href="https://doi.org/10.13039/501100007660">University of Antwerp</a>; V.G. by Strategisch Basisonderzoek Fonds voor Wetenschappelijk Onderzoek-Vlaanderen (SB-FWO); S.J.D. and K.D.F. by a Marie Curie-IEF Fellowship; P.C. is supported by a long-term structural Methusalem funding by the Flemish Government, Fonds Wetenschappelijk onderzoek (FWO), <a href="https://doi.org/10.13039/501100013845">Foundation Against Cancer</a> (2016-078), <a href="https://doi.org/10.13039/100010663">European Research Council</a> (ERC) Advanced Research Grant (EU-ERC743074), ERC Proof of Concept (ERC-713758), VIB TechWatch and by the Novo Nordisk Foundation (NNF19OC0055802). All the figures were created with BioRender.</p>
<h3>Author contributions</h3>
<p>N.V.C., L-A.T., K.V., L. Sokol, J.K., and L.d.R. wrote the manuscript. The protocol for EC isolation from choroid was optimized by J.K.; lung was optimized by L.-A.T.; brain was optimized by K.V. and N.V.C.; and muscle was optimized by L.-A.T. and K.V.; J.K., N.V.C., L. Sokol, E.M., L.-A.T., K.V., R.C., L.T., M.B., C.D., S.J.D., V.G, M.G.-C., P.d.Z., and K.D.F. participated in EC isolations. J.K., L.P.M.H.d.R., K.R., J.G., and L. Sokol performed scRNA-seq and bioinformatic analysis and result visualization. J.K. and M.P. performed flow cytometry. L.P.M.H.d.R., L. Schoonjans, M.D., G.E., J.K., K.R., J.G., X.L., and P.C. provided advice and discussed results. J.K. coordinated optimization of protocols. P.C. conceptualized the study.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>