<article>
<section>
<h1 id="header">Cell-specific mitochondria affinity purification (CS-MAP) from <i>Caenorhabditis elegans</i>
</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Arnaud Ahier,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Tessa Onraet,<sup><a href="#aff1">1</a></sup> and Steven Zuryn<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Clem Jones Centre for Ageing Dementia Research, Queensland Brain Institute, The University of Queensland, Brisbane, QLD 4072, Australia</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:a.ahier@uq.edu.au">a.ahier@uq.edu.au</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:s.zuryn@uq.edu.au">s.zuryn@uq.edu.au</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100952">10.1016/j.xpro.2021.100952</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Cell-Specific Mitochondria Affinity Purification (CS-MAP) enables isolation and purification of intact mitochondria from individual cell types of <i>Caenorhabditis elegans</i>. The approach is based on the cell-specific expression of a recombinant hemagglutinin (HA)-tag fused to the TOMM-20 protein that decorates the surface of mitochondria, thereby allowing their immunomagnetic purification. This protocol describes the CS-MAP procedure performed on large populations of animals. The purified mitochondria are suitable for subsequent nucleic acid, protein, and functional analyses.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref2">Ahier et al. (2018</a>, <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref1">2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A quick protocol to isolate cell-specific mitochondria from animals</li>
<li>Purified mitochondria are functional and can be isolated from multiple cell types</li>
<li>Mitochondrial DNA and proteins can be analysed on purified mitochondria</li>
<li>A simple and cost-effective protocol requiring common lab materials</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol requires the use of previously generated <i>C. elegans</i> transgenic lines (see <a href="#key-resources-table">key resources table</a>) expressing the recombinant fusion protein TOMM-20::mKate2::HA under the control of a range of cell-type-specific promoters, which are available through the <i>Caenorhabditis</i> Genetics Center. TOMM-20 (WormBase : WBGene00009092) , the translocase of the outer mitochondrial membrane complex subunit 20, is anchored to the surface of mitochondria and exposes mKate2 and the HA-tag to the cytoplasmic matrix outside the mitochondrion. The mKate2 red fluorophore allows tracking of labelled mitochondria throughout the procedure. The HA-tag (YPYDVPDYA) is used for the immunomagnetic purification of the organelle. This protocol can be used to purify mitochondria from various cell-types in large starting populations of animals. To some extent, mitochondria from a single cell-type can be isolated and enriched when a large population of animals, each with a single labeled cell, are pooled together. However, this limits subsequent analyses to nucleic acids (<a href="#bib2">Ahier et al., 2018</a>). The size of the starting population of animals can vary based on the cell-type used and the desired application. We recommend growing a synchronized population of 5,000 to 10,000 animals on one 100 mm Petri dish containing Nematode Growth Media (NGM) seeded with the <i>Escherichia coli</i> strain OP50 and multiplying the number of dishes to reach the required quantity of animals for the planned application (i.e., nucleic acid, protein, or functional analysis).</p>
<div class="note">
<span class="note-title">Note:</span> OP50 is an <i>Escherichia coli</i> strain conventionally used as the bacterial food source of <i>C. elegans</i> in laboratory contexts. Liquid OP50 culture is spread onto the surface of NGM plates to establish a bacterial lawn.</div>
<p>An overview and flowchart of the CS-MAP protocol is presented in <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview and flowchart of the CS-MAP protocol</div>
</figcaption>
</figure>
<h3 id="sec1.1">Building the homemade CS-MAP purification device</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<p>The CS-MAP homemade purification setup is readily constructed using inexpensive materials common to most molecular biology laboratories (please see <a href="#materials-and-equipment">materials and equipment</a> and <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a> for additional information regarding the device setup). Once built, the device can be reused multiple times.</p>
<ol>
<li>Use a 4 L sharps waste container (or similar) as a receptacle to collect liquid waste during the CS-MAP procedure.</li>
<li>Tape a 96-well PCR tube rack to the top opening of the container.</li>
<li>Tape a neodymium cylinder magnet (diameter 15 mm, height 25 mm, weight 60 g, axially magnetized) to the PCR tube rack.</li>
</ol>
<p>In subsequent steps, a 1 mL micropipette tip, a 25 mL pipette, and an electronic pipettor (pipettor) will be required.</p>
<h3 id="sec1.2">Preparing the buffer solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<ol start="4">
<li>Prepare a 100 mL stock of Mitochondrial Isolation Buffer (MIB). Add one tablet of protease inhibitor cocktail (S8830, Sigma-Aldrich) to the solution and aliquot into 10 mL volumes before storing at −20°C (see <a href="#materials-and-equipment">materials and equipment</a> for additional information regarding the recipe). Thaw and keep on ice before use.</li>
<li>Prepare a 100 mL stock of Phosphate EDTA BSA (PEB) Buffer.
<ol type="a">
<li>Add one tablet of protease inhibitor cocktail (S8830, Sigma-Aldrich) to the solution and aliquot into 10 mL volumes before storing at −20°C (see <a href="#materials-and-equipment">materials and equipment</a> for additional information regarding the recipe).</li>
<li>Thaw and keep on ice before use.</li>
</ol>
</li>
</ol>
<h3 id="sec1.3">Preparing a large population of staged <i>C. elegans</i> animals</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7–9 days</div>
<ol start="6">
<li>Transfer 30 to 50 animals of the fourth larval (L4) stage to a 100 mm diameter NGM plate seeded with OP50 bacteria (<a href="#bib11">Stiernagle, 2006</a>).
<ol type="a">
<li>Allow the animals to grow in a 20°C incubator for 4–5 days to obtain a mixed-staged animal population. The population’s development must be monitored to ensure that the animals are harvested before they exhaust their food supply of OP50.</li>
</ol>
</li>
<li>Collect animals by washing the plate with 14 mL of M9 buffer using a pipettor into a 15 mL Falcon conical tube.</li>
<li>To release eggs from the animals, centrifuge the tube at 200 <b>×</b> <i>g</i> for 2 min, remove the supernatant and add 1–2 mL of worm bleach solution (see <a href="#materials-and-equipment">materials and equipment</a> for recipe) to the pelleted animals. Mix gently by inverting the tube 3<b>×</b>.</li>
<li>Carefully monitor the bleaching under a dissecting microscope and neutralize the solution when the animals begin to break open (usually after 1–5 min) by adding 12 mL of M9 buffer. Practically, neutralize the solution when at least one of the following are observed: at least 1 to 2 animals adopt a perpendicular shape, several individual animals are seen in fragments, or tissues such as the gonad or intestine are seen extruding from the animal.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Sodium hypochlorite is a skin and eye irritant that can cause severe skin burns and eye damage following contact or severe respiratory and digestive issues when inhaled or ingested. Wear personal protective equipment including protective gloves, eye shields, and clothing.</div>
<ol start="10">
<li>Rinse the released eggs by pelleting them at 200 × <i>g</i> for 2 min, removing the supernatant, and adding 14 mL of fresh M9 buffer. Repeat this step at least twice to eliminate as many remnants of the worm bleach solution as possible.</li>
<li>Resuspend the pellet of eggs in 100 μL of M9, gently swirl the eggs with the pipette tip, and dispatch 3 individual drops of 5 μL onto a glass slide.</li>
<li>Count the number of eggs in each drop and estimate the number of eggs per μL by averaging the results of the 3 drops.</li>
<li>Pipette a volume of the resuspended egg pellet that contains ∼10,000 eggs onto a 100 mm diameter NGM plate seeded with OP50 bacteria. Repeat as necessary to reach the required quantity of animals for the desired application (see <a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Size of starting population based on desired application</caption>
<thead>
<tr>
<th>Application</th>
<th>Starting population</th>
<th>Quantity of 100 mm NGM plate</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nucleic acid analyses (Quantitative PCR [qPCR], PCR) as shown previously in <a href="#bib2">Ahier et al. (2018)</a> and <a href="#bib1">(2021)</a>.</td>
<td>10,000 to 20,000 animals</td>
<td>1 to 4 (according to <i>Escherichia coli</i> OP50 quantity on dishes).</td>
</tr>
<tr>
<td>Protein analyses (e.g., Western Blot) as shown previously in <a href="#bib2">Ahier et al. (2018)</a>.</td>
<td>50,000 to 100,000 animals</td>
<td>5 to 20 (according to <i>E. coli</i> OP50 quantity on dishes)</td>
</tr>
<tr>
<td>Functional analyses (e.g., Seahorse metabolic assay) as shown previously in <a href="#bib2">Ahier et al. (2018)</a>.</td>
<td>50,000 to 100,000 animals</td>
<td>5 to 20 (according to <i>E. coli</i> OP50 quantity on dishes)</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not dispatch too many eggs onto the NGM plate to ensure that the animals do not run out of OP50 food before reaching adulthood. Dispatching 5,000 eggs onto two 100 mm diameter NGM plates may be a safer option if unsure. Starved animals are not suitable for the CS-MAP procedure as their cuticle may be resistant to douncing.</div>
<ol start="14">
<li>Grow animals until the population reaches the L4 larval stage. The L4 larval stage usually occurs around 44–56 h after the bleached eggs have been seeded onto NGM plates (20°C) (<a href="#fig2">Figure 2</a>). At this point the population is ready to be collected for the CS-MAP procedure. We recommend using animals at a mature L4 stage (56 h) to avoid any L3 larval stage contamination. The procedure also works well on young adult animals (56–65 h; <a href="#fig2">Figure 2</a>).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Timeline of <i>C. elegans</i> development</div>
<p>The diagram shows the <i>C. elegans</i> developmental timeline at 20°C with approximative hour for each stage transition. Adapted from <a href="#bib3">Byerly et al., 1976</a>.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This method of synchronizing the animals does not guarantee a perfectly homogenous population of same-staged animals. Therefore, the growth of the overall population must be monitored. Also, this timeline can vary according to the genetic background of the animals, as certain mutants may be delayed in growth.</div>
<div class="note">
<span class="note-title">Note:</span> The developmental rate of the animals can be increased or decreased by incubation at 25°C or 15°C, respectively.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal Anti-HA magnetic beads (100 μg)</td>
<td>Pierce</td>
<td>Cat# 88837; RRID:AB_2861399</td>
</tr>
<tr>
<td>Rabbit Anti-HA-Tag Monoclonal Antibody (C29F4; 1:5000)</td>
<td>Cell Signaling Technology</td>
<td>Cat# 3724, RRID:AB_1549585</td>
</tr>
<tr>
<td>Mouse monoclonal anti-MTCO1 [1D6E1A8] (1:1000)</td>
<td>Abcam</td>
<td>Cat# ab14705; RRID:AB_2084810</td>
</tr>
<tr>
<td>Mouse monoclonal anti-ATPB [3D5] (1:1000)</td>
<td>Abcam</td>
<td>Cat# ab14730; RRID:AB_301438</td>
</tr>
<tr>
<td>Mouse monoclonal anti-HSP-60 [HSP60] (1:250)</td>
<td>DSHB (<a href="#bib7">Hadwiger et al., 2010</a>)</td>
<td>Cat# HSP60; RRID:AB_10572252</td>
</tr>
<tr>
<td>Mouse monoclonal anti-ELT-2 [455-2A4] (1:200)</td>
<td>DSHB (<a href="#bib7">Hadwiger et al., 2010</a>)</td>
<td>DSHB Cat# 455-2A4, RRID:AB_2618114</td>
</tr>
<tr>
<td>Mouse monoclonal anti-DLG-1 [DLG1] (1:100)</td>
<td>DSHB (<a href="#bib7">Hadwiger et al., 2010</a>)</td>
<td>DSHB Cat# DLG1, RRID:AB_2314321</td>
</tr>
<tr>
<td>IRDye 800CW goat anti-rabbit (1:20000)</td>
<td>LI-COR Biosciences</td>
<td>Cat# 925-32211; RRID:AB_2651127</td>
</tr>
<tr>
<td>IRDye 680RD goat anti-mouse (1:20000)</td>
<td>LI-COR Biosciences</td>
<td>Cat# 925-68070; RRID:AB_2651128</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> OP50</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>OP50</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Sigma-Aldrich</td>
<td>V3021</td>
</tr>
<tr>
<td>SIGMAFAST Protease Inhibitor Cocktail Tablets</td>
<td>Sigma-Aldrich</td>
<td>S8830</td>
</tr>
<tr>
<td>Mannitol</td>
<td>Sigma-Aldrich</td>
<td>M4125</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>S7903</td>
</tr>
<tr>
<td>BSA</td>
<td>Bovogen Biologicals</td>
<td>BSAS 0.1</td>
</tr>
<tr>
<td>PBS 10<b>×</b>
</td>
<td>Lonza</td>
<td>17-517Q</td>
</tr>
<tr>
<td>KCl</td>
<td>Sigma-Aldrich</td>
<td>P9541</td>
</tr>
<tr>
<td>NP40</td>
<td>Sigma-Aldrich</td>
<td>I8896</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma-Aldrich</td>
<td>P9416</td>
</tr>
<tr>
<td>Gelatin from bovine skin</td>
<td>Sigma-Aldrich</td>
<td>G9391</td>
</tr>
<tr>
<td>HA peptide</td>
<td>Sigma-Aldrich</td>
<td>I2149</td>
</tr>
<tr>
<td>Glycogen</td>
<td>Thermo Fisher Scientific</td>
<td>AM9510</td>
</tr>
<tr>
<td>Phenol-chloroform-isoamyl alcohol</td>
<td>Sigma-Aldrich</td>
<td>77617</td>
</tr>
<tr>
<td>4<b>×</b> Laemmli Sample Buffer</td>
<td>Bio-Rad</td>
<td>1610747</td>
</tr>
<tr>
<td>GeneRuler DNA Ladder Mix</td>
<td>Thermo Fisher Scientific</td>
<td>SM0331</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain N2 (wild-type) L4 stage, hermaphrodite</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>N2 (strain) WormBase: WBStrain00000001</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain SJZ47 (<i>foxSi16 [myo-3p::tomm-20::mKate2::HA::tbb-2 3′ UTR] I</i>) L4 stage, hermaphrodites</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>SJZ47 (<a href="#bib2">Ahier et al., 2018</a>)</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain SJZ216 (<i>foxSi44 [rgef-1p::tomm-20::mKate2::HA::tbb-2 3′ UTR] I</i>) L4 stage, hermaphrodites</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>SJZ216 (<a href="#bib2">Ahier et al., 2018</a>)</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain SJZ213 (<i>foxSi41 [dpy-7p::tomm-20::mKate2::HA::tbb-2 3′ UTR] I</i>) L4 stage, hermaphrodites</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>SJZ213 (<a href="#bib2">Ahier et al., 2018</a>)</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain SJZ204 (<i>foxSi37 [ges-1p::tomm-20::mKate2::HA::tbb-2 3′ UTR] I</i>) L4 stage, hermaphrodites</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>SJZ204 (<a href="#bib2">Ahier et al., 2018</a>)</td>
</tr>
<tr>
<td>
<i>C. elegans</i>: strain SJZ328 (<i>foxSi75 [eft-3p::tomm-20::mKate2::HA::tbb-2 3′ UTR] I</i>) L4 stage, hermaphrodites</td>
<td>
<i>Caenorhabditis</i> Genetics Center (CGC)</td>
<td>SJZ328 (<a href="#bib2">Ahier et al., 2018</a>)</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>CD15 5′-agcgtcatttattgggaagaagac-3′</td>
<td>(<a href="#bib2">Ahier et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>AA113 5′-ccttccaaatactccgtctgc-3′</td>
<td>(<a href="#bib2">Ahier et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>ges-1F1 5′-gctctccgcatgctaatgaatacc-3′</td>
<td>(<a href="#bib2">Ahier et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>ges-1R1 5′-ctaactggtctccattctacatcttc-3′</td>
<td>(<a href="#bib2">Ahier et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>AA178 5′-cataccgaataaacatcagggtaatct-3′</td>
<td>(<a href="#bib1">Ahier et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>AA179 5′-acgggtgttacactatgatgaaga-3′</td>
<td>(<a href="#bib1">Ahier et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SZ35 5′-aggctaagccggggtaagtt-3′</td>
<td>(<a href="#bib1">Ahier et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SZ36 5′-gccaaaagcttaaactgcgg-3′</td>
<td>(<a href="#bib1">Ahier et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Zen2</td>
<td>Zeiss</td>
<td>version 2.0.0.0</td>
</tr>
<tr>
<td>FIJI</td>
<td>(<a href="#bib10">Schindelin et al., 2012</a>)</td>
<td>1.52p</td>
</tr>
<tr>
<td>GraphPad Prism v9.1.0</td>
<td>GraphPad Software</td>
<td><a href="https://www.graphpad.com/">https://www.graphpad.com/</a></td>
</tr>
<tr>
<td>Rotor-Gene Q Pure Detection software v2.3.1</td>
<td>QIAGEN</td>
<td><a href="https://www.qiagen.com/">https://www.qiagen.com/</a></td>
</tr>
<tr>
<td>Microsoft Office 365</td>
<td>Microsoft Corporation</td>
<td><a href="https://www.microsoft.com/en-au/microsoft-365">https://www.microsoft.com/en-au/microsoft-365</a></td>
</tr>
<tr>
<td>Adobe Illustrator CC 2021</td>
<td>Adobe</td>
<td><a href="https://www.adobe.com/">https://www.adobe.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Z2 imager microscope with an Axiocam 506 mono camera</td>
<td>Zeiss</td>
<td>Zeiss</td>
</tr>
<tr>
<td>Thermo Scientific S1 Pipet Filler</td>
<td>Thermo Fisher Scientific</td>
<td>S1</td>
</tr>
<tr>
<td>Serological pipet 25 mL</td>
<td>Falcon</td>
<td>357525</td>
</tr>
<tr>
<td>Sigma Dounce tissue grinder pestle - small clearance (B), working volume 7 mL</td>
<td>Sigma-Aldrich</td>
<td>P1235</td>
</tr>
<tr>
<td>Dounce tissue grinder tube working volume 7 mL</td>
<td>Sigma-Aldrich</td>
<td>T0566</td>
</tr>
<tr>
<td>NanoPhotometer</td>
<td>Implen</td>
<td>N60</td>
</tr>
<tr>
<td>16-Tube SureBeads™ Magnetic Rack</td>
<td>Bio-Rad</td>
<td>1614916</td>
</tr>
<tr>
<td>Rotor-Gene machine</td>
<td>QIAGEN</td>
<td>NA</td>
</tr>
<tr>
<td>Odyssey Fc Imager</td>
<td>LI-COR</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>A list of materials required to build the homemade CS-MAP setup and a description of it are presented in <a href="#fig3">Figures 3</a> and <a href="#fig4">4</a>.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. List of materials required to build the homemade CS-MAP setup</div>
<p>(A–F) (A) A 4 L waste container or similar, (B) a 96-well PCR tube rack holder, (C) a neodymium cylinder magnet (diameter 15 mm, height 25 mm, weight 60 g, axially magnetized), (D) a 1 mL plastic tip, (E) a 25 mL pipette, (F) an electronic pipette (pipettor) or equivalent.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The homemade CS-MAP setup</div>
<p>(A) Photograph of the assembled device. The magnet and the PCR rack are taped to the top of the container. A 1 mL tip is placed next to the flat surface of the magnet.</p>
<p>(B) Photograph of the device during immunomagnetic immobilization of mitochondria to the tip wall closest to the magnet. Note that the micropipette must remain attached to the tip to prevent the loss of the sample prior to forming a magnetically trapped pellet.</p>
<p>(C) Photograph of the device during the washing step. The 25 mL pipette is placed into the opening of the 1 mL tip and PEB is released by gravity flow by carefully unplugging the pipettor.</p>
</figcaption>
</figure>
<table id="undtbl1">
<caption>Worm bleach solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Bleach (sodium hypochlorite 8%–14%)</td>
<td>n/a</td>
<td>4 mL</td>
</tr>
<tr>
<td>10 N NaOH</td>
<td>n/a</td>
<td>5 mL</td>
</tr>
<tr>
<td>H<sub>2</sub>O</td>
<td>n/a</td>
<td>41 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 22°C–25°C , protected from light, for a maximum of 2 weeks. The potency of worm bleach solution decreases over time, partly because of its photosensitive nature.</p>
<table id="undtbl2">
<caption>PEB: Phosphate EDTA BSA Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>20 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>1%</td>
<td>2 g</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>2 mM</td>
<td>0.8 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 200 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Add 1 tablet of protease inhibitor cocktail per 100 mL of buffer prior to use. PEB with protease inhibitor cocktail can be stored at −20°C for a maximum of 6 months. PEB used for washing in the immunomagnetic purification step should be prepared without protease inhibitor cocktail, kept on ice for 1 h, and used within the same day.</p>
<div class="critical">
<span class="critical-title">Critical:</span> As the high content of BSA in PEB makes it highly prone to contamination, store at −20°C for long-term storage or at 4°C for no more than 1 day.</div>
<table id="undtbl3">
<caption>MIB: Mitochondrial Isolation Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCl</td>
<td>50 mM</td>
<td>0.744 g</td>
</tr>
<tr>
<td>Mannitol</td>
<td>110 mM</td>
<td>4 g</td>
</tr>
<tr>
<td>Sucrose</td>
<td>70 mM</td>
<td>4.792 g</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>0.1 mM</td>
<td>0.04 mL</td>
</tr>
<tr>
<td>1 M Tris HCl pH 7.4</td>
<td>5 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 200 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Add 1 tablet of protease inhibitor cocktail per 100 mL of solution prior to use. MIB with protease inhibitor cocktail can be stored at −20°C for a maximum of 6 months.</p>
<table id="undtbl4">
<caption>M9 buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>3 g</td>
<td>22 mM</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>6 g</td>
<td>42 mM</td>
</tr>
<tr>
<td>NaCl</td>
<td>5 g</td>
<td>86 mM</td>
</tr>
<tr>
<td>MgSO<sub>4</sub>
</td>
<td>0.25 g</td>
<td>1 mM</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 22°C–25°C for a maximum of 6 months.</p>
<table id="undtbl5">
<caption>MLB: Mitochondrial Lysis Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M KCl</td>
<td>50 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>1 M Tris HCl pH 8</td>
<td>10 mM</td>
<td>5 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>
</td>
<td>2.5 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>NP-40</td>
<td>0.45%</td>
<td>2.25 mL</td>
</tr>
<tr>
<td>Tween-20</td>
<td>0.45%</td>
<td>2.25 mL</td>
</tr>
<tr>
<td>Gelatin</td>
<td>0.01%</td>
<td>50 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for a maximum of 6 months. Add 0.1 mg·mL<sup>−1</sup> proteinase K immediately before use.</p>
<table id="undtbl6">
<caption>RIPA Buffer: Radioimmunoprecipitation Assay Buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris HCl pH 8</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>0.5 M EDTA</td>
<td>1 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Sodium deoxycholate</td>
<td>0.1%</td>
<td>0.1 g</td>
</tr>
<tr>
<td>SDS</td>
<td>0.1%</td>
<td>0.1 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>140 mM</td>
<td>0.8 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for a maximum of 6 months. Add 1 tablet of protease inhibitor cocktail per 100 mL of buffer immediately before use.</p>
<table id="undtbl7">
<caption>TBST: Tris-buffered saline with 0.1% Tween-20 detergent</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS 10<b>×</b> pH 7.5</td>
<td>1<b>×</b>
</td>
<td>100 mL</td>
</tr>
<tr>
<td>Tween-20</td>
<td>0.1%</td>
<td>1 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>Up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 22°C–25°C for a maximum of 6 months.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Animal harvesting and cleaning</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 min</div>
<p>Synchronized adult animals are harvested and cleared of <i>E. coli</i> OP50 before being homogenized.</p>
<ol>
<li>Harvest animals (see Experimental models: Organisms/strains of the <a href="#key-resources-table">key resources table</a> for strains to use: <i>myo-3/eft-3/ges-1/rgef-1/dpy-7p::tomm-20::mKate2::HA</i> or N2 [as a control]) by washing them off the plate with two successive volumes of 7.5 mL of M9 buffer into a 15 mL Falcon conical tube using a pipettor. Washing animals off the plate with two successive volumes of M9 buffer ensures that the maximum numbers of animals are collected before beginning the procedure.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If multiple plates are being used per sample, wash the animals off one plate and onto the next with 7.5 mL of M9 buffer to maximise the collection of animals. Continue this process on each successive plate and then collect the animals in a 15 mL Falcon tube at the end. Repeat this sequential collection technique with another 7.5 mL of fresh M9 buffer and add to the Falcon tube.</div>
<ol start="2">
<li>Centrifuge the tube at 200 × <i>g</i> for 2 min and replace the M9 supernatant with 14 mL of fresh M9 buffer.</li>
<li>Repeat the previous washing step once more to remove as much OP50 from the sample as possible. See troubleshooting <a href="#sec7.1">problem 1</a>.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Rotate the animals in 10 mL of M9 buffer in a 15 mL Falcon tube for 30 min at 30 RPM at 22°C–25°C to clear bacteria from the animals’ intestines. After rotating, pellet the animals by centrifugation at 200 × <i>g</i> for 2 min and wash the pellet once more with 14 mL of M9 buffer.</div>
<ol start="4">
<li>Pellet the animals by centrifugation at 200 × <i>g</i> for 2 min and replace the M9 supernatant with 14 mL of ddH<sub>2</sub>O to remove traces of the M9 buffer and thus reduce the salinity of the solution.</li>
<li>Centrifuge again at 200 × <i>g</i> and remove the ddH<sub>2</sub>O supernatant. Keep the sample on ice for less than 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Allow more than 20 min for the processing of multiple samples or multiple plates per sample.</div>
<h3 id="sec3.2">Generating an enriched mitochondrial fraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 40 min</div>
<p>The sample is homogenized using a glass Dounce tissue grinder set with a low clearance pestle (usually called B clearance [P1235, Sigma-Aldrich]). A crude animal homogenate is then created by differential centrifugation.</p>
<div class="critical">
<span class="critical-title">Critical:</span> From this point onwards, all procedures must be performed on ice or at 4°C and all tubes must be cooled prior to use.</div>
<ol start="6">
<li>Suspend the sample in 1.5 mL of cold Mitochondrial Isolation Buffer (MIB) with protease inhibitor cocktail.</li>
<li>Transfer the sample with a 1 mL tip and a micropipette into a cold 7 mL Dounce tissue grinder tube (T0566, Sigma-Aldrich).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The animals tend to adhere to the plastic walls of the pipette tip. To minimize this, animals can be sequentially transferred into the Dounce tissue grinder tube in successive volumes of 300 μL. This reduces the surface area available for animals to adhere to and therefore reduces the animal loss during the transfer.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prewashing the pipette tip with a mild detergent might not be suitable, as subsequent traces of detergent can irremediably damage the released mitochondria (<a href="#bib6">Gurtubay et al., 1980</a>).</div>
<ol start="8">
<li>Perform 20 to 30 Dounce tissue grinder strokes.
<ol type="a">
<li>Dounce the animals by pushing the pestle towards the bottom of the Dounce reservoir in a motion parallel to the tube. Before reaching the bottom of the tube, twist the pestle, keeping the shaft of the pestle parallel to the tube walls, and then pull the pestle out of the buffer.</li>
<li>Repeat for a total of 20–30 strokes. Strokes must be regularly spaced, relatively fast, and go in and out of the buffer for each individual stroke. A medium to strong resistance against the sample must be experienced during the douncing for the animals to be homogenized.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> This step is crucial and can determine the success of the entire procedure. Twenty to 30 strokes usually suffice to extract mitochondria from the major tissues tested (body wall muscle [BWM], intestine, hypodermis, neurons, and germline). Importantly, the glass Dounce tissue grinder set can erode over time and produce inconsistent homogenization results. To prevent both erosion of the Dounce set and cross-contamination between samples, the Dounce should be meticulously washed with demineralized water between each individual sample. After each experiment, the Dounce must be meticulously washed with demineralized water and air-dried before storage. If results become inconsistent due to the wearing of the Dounce, consider using a new Dounce set.</div>
<div class="note">
<span class="note-title">Note:</span> During homogenization, a fraction of animals is broken into multiple fragments and mitochondria are released from cells into the MIB. A fraction of animals in the sample may still be alive but suffering from damage that is severe enough to release some of their mitochondria into the MIB. Over-douncing the sample may potentially fragment mitochondria and generate a large amount of cellular debris that is very difficult to clear during the purification steps described below.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Homogenization is more efficient on animals synchronized at L4 larval stage to adulthood as early larval stages tend to be more resistant to douncing. For this reason, a synchronised population is preferred to achieve the best results.</div>
<ol start="9">
<li>Transfer the homogenate into a 1.5 mL microcentrifuge tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At this step, the homogenate should be gently mixed by pipetting a few times up and down with a volume of 300 μL using a 1 mL tip. Then, 300 μL (containing intact animals, animal fragments, mitochondria, and various debris) should be saved as Input 1 (also called whole animal homogenate). Input 1 should be centrifuged at 12,000 × <i>g</i> for 10 min and the supernatant discarded before being frozen (not necessarily flash frozen) at -80°C for later processing (for PCR, qPCR, or Western blot applications - see <a href="#expected-outcomes">expected outcomes</a> for processing information). For metabolic assays, Input 1 should be kept on ice and processed directly at the end of the CS-MAP procedure. In principle, frozen tissues shouldn’t be used for physiological assays.</div>
<ol start="10">
<li>After separating 300 μL of the homogenate for use as Input 1, remove intact animals, animal fragments, and large pieces of cellular debris from the remaining 1.2 mL of the animal homogenate from step 9 by a low-speed centrifugation at 200 × <i>g</i> for 5 min.
<ol type="a">
<li>Transfer the supernatant to a new 1.5 mL tube.</li>
</ol>
</li>
<li>Refine the sample further with a second centrifugation at 800 × <i>g</i> for 10 min and transfer the supernatant to a new 1.5 mL tube. The supernatant is now a crude animal homogenate.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A small aliquot of the crude animal homogenate can potentially be saved as Input 2.</div>
<ol start="12">
<li>Pellet the enriched mitochondrial fraction within the crude animal homogenate by centrifugation at 12,000 × <i>g</i> for 10 min.</li>
<li>Discard the supernatant and quickly add 20 μL of MIB to the enriched mitochondrial fraction. See troubleshooting <a href="#sec7.3">problem 2</a>.</li>
<li>Resuspend the enriched mitochondrial fraction in MIB by vigorously pipetting the solution up and down for 1 min using a P20 micropipette. As mitochondria can agglutinate, this will help ensure uniform distribution of mitochondria throughout the MIB.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> At this point, the quality of the enriched mitochondrial fraction should be assessed using a fluorescence microscope. This is performed by observing the presence of mitochondria labelled by the red fluorophore mKate2. Pipette 1 μL of the sample from step 14 onto a glass slide and cover with a coverslip. Observe the sample under a fluorescent compound microscope by using a 40<b>×</b> oil objective (EC Plan-Neofluar 40<b>×</b>/1.30 Oil DIC M27) and selecting the red channel (excitation wavelength 587 nm, emission wavelength 610 nm, light source intensity 50%, exposure time 3 s, binning mode 2.2). The first time the protocol is performed for a given cell-type, the 587 nm red channel of the microscope should be calibrated to background fluorescence levels using wild-type (N2) animals that have undergone the same CS-MAP procedure. If the appropriate microscope settings are not used (e.g., excessively long exposure time), irrelevant background dots may be misidentified as mKate2-positive mitochondria. A typical observation of an enriched mitochondrial fraction obtained from N2 animals and animals harbouring mKate2-labeled mitochondria is shown in <a href="#fig6">Figures 6</a>A and 6B (see <a href="#expected-outcomes">expected outcomes</a>). The quantity of mKate2-labelled mitochondria can vary from one strain to another, depending on the cell-type labeled by the fluorophore. If no fluorescent mitochondria are observed, the procedure has not worked properly and should be restarted.</div>
<div class="note">
<span class="note-title">Note:</span> Wild-type (N2) animals generally don’t produce particles that exhibit autofluorescence at a brightness comparable to mKate2-labeled mitochondria. Only some rare red particles can be observed but can easily be discriminated from mKate2-labelled mitochondria by their aberrant shape or brightness.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Mitochondria, various organelles, vesicular fragments, and other debris of the enriched mitochondrial fraction tend to clump together. Pipetting the sample up and down with a P20 micropipette for 1 min helps to disrupt these clumps. The enriched mitochondrial fractions shown in <a href="#fig6">Figure 6</a>B (see <a href="#expected-outcomes">expected outcomes</a>) are relatively free from major clumps with red-labeled mitochondria uniformly distributed throughout the image and are thus suitable for subsequent steps. If clumps are observable, pipette the sample up and down with a P20 micropipette an extra 20 times and observe 0.5 μL of the sample under the microscope. Repeat again if required.</div>
<h3 id="sec3.3">Binding of anti-HA magnetic beads to <sup>HA-tagged</sup>mitochondria</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h 15 min</div>
<p>The enriched mitochondrial fraction is incubated with anti-HA magnetic beads, which selectively bind tissue-specific <sup>HA-tagged</sup>mitochondria.</p>
<div class="critical">
<span class="critical-title">Critical:</span> To be performed on ice or at 4°C.</div>
<ol start="15">
<li>Add 400 μL of cold Phosphate EDTA BSA buffer (PEB) with protease inhibitor cocktail to the 20 μL of MIB-suspended enriched mitochondrial fraction and homogenize the sample by vigorously pipetting up and down 10 times using a P200 micropipette set to 200 μL. Transfer the contents into a 0.6 mL round-bottom microtube.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Consider pipetting up and down more times if clumps are still present in the sample. This step is empirical, as it can be difficult to spot clumps of mitochondria under the microscope once they are diluted in 400 μL of PEB.</div>
<div class="note">
<span class="note-title">Note:</span> Performing the next incubation step in a smaller 0.6 mL round-bottom microtube minimizes the contact of mitochondria with the plastic walls of the tube, thereby reducing their loss.</div>
<ol start="16">
<li>Carefully wash 100 μg of anti-HA magnetic beads (88837, Pierce) in a 1.5 mL microcentrifuge tube by adding 1 mL of cold PEB with protease inhibitor cocktail and flicking gently to mix.
<ol type="a">
<li>Place the microcentrifuge tube into a standard magnetic rack (SureBeads, Bio-Rad or equivalent) and leave for 1–2 min until the magnetic beads have assembled against the magnet.</li>
<li>Carefully remove clear PEB before removing the tube from the magnetic rack and repeating the washing process at least 6 times.</li>
</ol>
</li>
<li>Resuspend the clean anti-HA magnetic beads in 100 μL of PEB with protease inhibitor cocktail.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Commercial anti-HA magnetic beads are usually stored in 0.05% Tween-20 detergent and 0.05% NaN<sub>3</sub>. Trace amounts of these reagents may affect mitochondrial physiology or lyse the organelles. Therefore, careful washing of the anti-HA magnetic beads prior to use is critical.</div>
<div class="note">
<span class="note-title">Note:</span> 100 μg of anti-HA magnetic beads are used per sample. If the experiment involves multiple samples, calculate the μg of beads required in total and wash together. To obtain the greater quantity of mitochondria required for protein extraction, up to 200 μg of anti-HA magnetic beads have been used per sample.</div>
<ol start="18">
<li>Add the washed 100 μg of anti-HA magnetic beads (in 100 μL of PEB with protease inhibitor cocktail) to the 400 μL sample from step 15.</li>
<li>Wrap Parafilm around the closed lid of the tube to prevent it from opening and incubate at 4°C under gentle rotation, using an Intelli-Mixer RM-2 (ELMI) on program F1 at 8 RPM for 1 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Incubation can be increased to 2 h with similar results.</div>
<h3 id="sec3.4">Immunomagnetic purification of mitochondria</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Mitochondria bound by immunomagnetic beads are column-purified on the homemade CS-MAP setup (<a href="#fig4">Figure 4</a>). The procedure is shown step-by-step in <a href="#mmc1">Methods video S1</a>: Purification of mitochondria.</p>
<ol start="20">
<li>Carefully pipette up the sample with a 1 mL tip and a P1000 micropipette.</li>
<li>Position the tip – still connected to the micropipette – into the magnetic field of the magnet (adjacent to one of the flat surfaces of the magnet) on the PCR rack of the homemade CS-MAP device (see <a href="#fig4">Figure 4</a>B and <a href="#fig5">5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Pellet of immunomagnetic mitochondria</div>
<p>Mitochondria attached to magnetic beads aggregate on the wall of the tip closest to the magnet after 1 min. White arrow: magnetic bead pellet.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not release the micropipette from the tip until the magnetic beads have formed a visible pellet on the wall of the tip closest to the magnet. It usually takes 1 min (<a href="#fig5">Figure 5</a>).</div>
<ol start="22">
<li>Once a visible pellet of magnetic beads is visible on the wall of the tip (after 1 min), carefully remove the micropipette from the tip, without disturbing the tip or magnet. When the micropipette is released, the supernatant will drain into the waste bucket.</li>
<li>Using a pipettor and a 25 mL plastic serological pipette, aspirate 25 mL of cold PEB.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> PEB used for washing does not require the protease inhibitor cocktail.</div>
<ol start="24">
<li>Plug the tip of the 25 mL pipette into the top of the 1 mL tip and carefully unplug the pipette from the pipettor without disrupting the 1 mL tip or magnet. Allow the PEB to flow through the 1 mL tip and into the waste bucket. This step washes the magnetic beads that remain held in place by the magnet.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If an air bubble forms in the 1 mL tip preventing uniform washing, reattach the pipettor to the 25 mL pipette and very briefly re-aspire less than 1 ml of liquid. This causes a slight backflow in liquid that refills the 1 ml tip and can clear the bubble. Once the bubble is removed, unplug the pipette from the pipettor once again.</div>
<ol start="25">
<li>Once all the PEB has passed through the column, remove the 25 mL pipette and discard.
<ol type="a">
<li>Next, remove the 1 mL tip from the magnetic field, place it into a 1.5 mL microcentrifuge tube, and elute the bead pellet by adding 1 mL of PEB into the tip.</li>
<li>Reconnect the P1000 micropipette to the tip containing the beads and pipette up and down a few times to resuspend the beads (see <a href="#mmc1">Methods video S1</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For a low stringency wash, only 1 wash with 25 mL of PEB is sufficient, so directly proceed to step 28. A low stringency wash improves the yield of immunomagnetic-purified mitochondria at the expense of purity. For a higher stringency wash, proceed to step 24. A high stringency wash involves a second wash with 25 mL of PEB and improves the purity of immunomagnetic-purified mitochondria at the expense of yield. A low stringency wash is preferred when protein and metabolic analyses will be performed but a high stringency wash is required for analysis of nucleic acids.</div>
<ol start="26">
<li>Using a standard magnetic rack (SureBeads, Bio-Rad, or equivalent), pellet the magnetic beads and replace the PEB used for the elution with 1 mL of fresh PEB.</li>
<li>Vortex the sample 3 times for 2 s each.</li>
<li>Place the tube under gentle rotation for 10 min at 4°C.</li>
<li>Perform a second wash with 25 mL of cold PEB on the CS-MAP homemade setup (steps 18 to 23).</li>
<li>Remove PEB using a standard magnetic rack and resuspend the immunomagnetic-purified (IP<sup>ed</sup>) mitochondria in 20 μL of cold MIB with protease inhibitor cocktail. The sample of immunomagnetic-purified (IP<sup>ed</sup>) mitochondria is ready to use and must be kept on ice. See troubleshooting <a href="#sec7.5">problem 3</a> and <a href="#sec7.7">problem 4</a> for potential issues.</li>
</ol>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Methods video S1. Purification of mitochondria, related to step 20</div>
<p>Step 1) A 1 mL micropipette tip containing the immunomagnetic mitochondria is positioned next to the flat surface of the magnet. Step 2) the immunomagnetic mitochondria are trapped by the magnet to the wall of the 1 mL micropipette tip closest to the magnet (1 min waiting). Step 3) A 25 mL pipette containing PEB is plugged into the top of the 1 mL micropipette tip to create a transitory washing column and the sample is washed using gravity flow. Step 4) Purified immunomagnetic mitochondria are collected with PEB and concentrated into a reduced volume of MIB.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Immunomagnetic-purified (IP<sup>ed</sup>) mitochondria are resuspended in 20 μL of cold MIB and kept on ice. The sample is ready to use for subsequent applications.</p>
<p>Enriched mitochondrial fraction and IP<sup>ed</sup> mitochondria outcomes from the CS-MAP method are presented in <a href="#fig6">Figure 6</a>.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Enriched mitochondrial fraction and IP<sup>ed</sup> mitochondria outcomes from the CS-MAP method</div>
<p>The outcomes of wild-type (N2) animals and a strain carrying the transgene <i>myo-3p::TOMM-20::mKate2::HA</i> as a single copy insertion into the universal <i>MosSCI</i> insertion site <i>oxti185 I</i> (SJZ47), which allows the purification of mitochondria specifically from body wall muscle cells (BWM), are presented. One μL of the enriched mitochondrial fraction (upper part of the figure) and 1 μL of the purified mitochondria sample (bottom part of the figure) are shown. The enriched mitochondrial fraction shows uniformly-spread fine debris and some aggregates: (A and B). Note that some large debris is visible (e.g., fragments of pharynx and tips of tails) but is rare and usually removed during the washing step. The enriched mitochondrial fraction of SJZ47 shows numerous mKate2-labelled mitochondria compared to the N2 enriched mitochondrial fraction: (A and B). The IP<sup>ed</sup> mitochondrial fraction shows unbound anti-HA magnetic beads for the N2 control (C) and anti-HA magnetic beads bound to mKate2-labelled mitochondria for SJZ47 (E). Aggregates of mitochondria-magnetic beads are shown in (F). In D), unbound anti-HA magnetic beads are visible. Observations were performed using a fluorescent compound microscope (40<b>×</b> magnification, oil objective, DIC and red channel [excitation wavelength 587 nm, emission wavelength 610 nm]). Scale bar for (A–C). and (E) 100 μm. Scale bar for (D and F) 10 μm.</p>
</figcaption>
</figure>
<p>The quality of the sample should be evaluated using a fluorescence compound microscope. To do so, 1 μL of the IP<sup>ed</sup> mitochondria sample is placed on a glass slide and covered with a coverslip. Samples are observed using a 40<b>×</b> oil objective (EC Plan-Neofluar 40<b>×</b>/1.30 Oil DIC M27) with DIC or Normarski and the red fluorescence channel (excitation wavelength 587 nm, emission wavelength 610 nm, light source intensity 50%, exposure time 3 s, binning mode 2.2). Differences between control experiments performed using N2 animals and strains of interest are indicative of purified mitochondria from the cell-type of interest. It is normal for multiple fluorescently labelled mitochondria to be bound to a single anti-HA magnetic bead or form aggregates with multiple anti-HA magnetic beads (<a href="#fig6">Figures 6</a>E and 6F). The number of fluorescently labelled mitochondria can vary according to the cell-type from which they were purified. DIC observation allows confirmation of the absence of fine (and possibly coarse) debris which was not visible in the enriched mitochondrial fraction (<a href="#fig6">Figures 6</a>A vs 6C).</p>
<p>Ideally, the IP<sup>ed</sup> mitochondria should be used or processed immediately for any subsequent application even though mitochondria are still visually normal for up to 4 h after purification when kept on ice or at 4°C. We have previously used IP<sup>ed</sup> mitochondria for different applications, including PCR, Western blot, and functional Seahorse assays (<a href="#bib2">Ahier et al., 2018</a>). IP<sup>ed</sup> mitochondria must be prepared accordingly to the target application. If required, the number of purified mitochondria can also be visually quantified.</p>
<h3 id="sec4.1">Nucleic acids analyses (qPCR and PCR)</h3>
<p>The nucleotide composition of the mitochondrial genome (mutation types and levels) can vary between tissues types (<a href="#bib8">Hahn and Zuryn, 2019</a>). As such, it is often informative to analyze mtDNA after CS-MAP. For such analyses (e.g., PCR or quantitative PCR), a standard magnetic rack (SureBeads, Bio-Rad or equivalent) is used to pellet the magnetic beads and replace the MIB with 100 μL of mitochondrial lysis buffer (MLB).</p>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Sample can be frozen in 100 μL of MLB at −80°C for 1 month for later processing at the same time as Input 1.</div>
<p>Samples are lysed for 1 h at 65°C before inactivating proteinase K at 95°C for 15 min. A standard phenol-chloroform DNA extraction is then performed, followed by an ethanol precipitation of DNA using 10 μg of glycogen as a carrier. The DNA pellet is resuspended in 20 μL of 10 mM Tris pH 8.0, 1 mM EDTA buffer. It can be difficult to accurately quantify mtDNA using a NanoPhotometer (N60, IMPLEN); however, 0.01–1 μL is generally used for PCR applications.</p>
<div class="note">
<span class="note-title">Note:</span> Input 1 can be thawed, resuspended in 200 μL of MLB, and processed in parallel to the IP<sup>ed</sup> mitochondria sample. The sample should be processed similarly to the IP<sup>ed</sup> mitochondria sample except for the lysing step which should be performed for 2–3 h at 65°C, as the sample contains fragments of animals and entire animals. Because of the presence of nuclear DNA (nDNA), mtDNA cannot be quantified in Input 1. Using a NanoPhotometer (N60, IMPLEN), total DNA can be quantified and diluted to 10 ng·μL<sup>−1</sup>. For PCR applications, 0.01–1 μL is used. If required, Input 2 can be processed as the IP<sup>ed</sup> mitochondria sample.</div>
<p>The enrichment of mtDNA after CS-MAP may reflect the enrichment of the mitochondrial fraction and can be evaluated, as seen in <a href="#fig7">Figure 7</a>. We usually use a semi-quantitative PCR approach where we amplify a sequence of the mtDNA that we normalize to an amplified target of the nDNA of the animal (<a href="#fig7">Figure 7</a>A). This approach provides an overview of the purity of the sample and helps verify the use of this sample for further analyses. Alternatively, if a more accurate quantification is required, a qPCR measuring the ratio of mtDNA to nDNA in Input 1 compared to IP<sup>ed</sup> mitochondria can be performed (<a href="#fig7">Figures 7</a>B and 7C).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Quantification of mtDNA enrichment after CS-MAP</div>
<p>Two different types of quantification (semi-quantitative and quantitative PCR (qPCR)) were performed on the same samples for comparison.</p>
<p>(A) Semi-quantitative PCR showing the enrichment of mitochondrial DNA (mtDNA) over nuclear DNA (nDNA) after CS-MAP. To evaluate mtDNA enrichment, the primers AA113 and CD15 (<a href="#bib2">Ahier et al., 2018</a>) were used with Phusion DNA Polymerase (Biolabs, M0530) under the following cycle conditions: 98°C for 30 s, followed by 34 cycles of 98°C for 10 s, 65°C for 15 s, and 72°C for 1 min. To evaluate nDNA levels, the primers ges-1F1 and ges-1R1 (<a href="#bib2">Ahier et al., 2018</a>) were used with the aforementioned polymerase under the following cycle conditions: 98°C for 30 s, followed by 34 cycles of 98°C for 10 s, 55°C for 15 s, and 72°C for 1 min. 1 μL of the input diluted at 10 ng·μL<sup>-1</sup> and 1 μL of the CS-MAP sample were used per reaction. nDNA has occasionally been detected in the CS-MAP sample. GeneRuler DNA Ladder Mix (SM0331, Thermo Scientific) was used for this gel.</p>
<p>(B and C) The samples used in A) were analyzed by qPCR using a Rotor-Gene Q real-time PCR (QIAGEN) machine. To quantify mtDNA levels, the primers AA178 and AA179 were used, while the primers SZ35 and SZ36 (<a href="#bib1">Ahier et al., 2021</a>) were used to quantify nDNA. qPCR was performed using the following cycle conditions: 95°C for 10 min, followed by 40 cycles of 95°C for 15 s and 58°C for 40 s using the Sensifast SYBR No-ROX Kit (BIO-98005, Bioline). 1 μL of the input diluted at 10 ng·μL<sup>-1</sup> and 1 μL of the undiluted CS-MAP sample were used per technical triplicate. The efficiency of enrichment can vary due to experimental variability and/or cell-type used. We usually observe a 20- to 50-fold enrichment with some rare cases where nuclear DNA was not detected within 40 qPCR cycles. (B) Table showing the obtained Ct values and the calculation steps of the applied comparative Ct method. ΔCt corresponds to nDNA average Ct value - mtDNA average Ct value and relative enrichment (RE) corresponds to 2<sup>ΔCt</sup>. (C) Quantification of mtDNA enrichment relative to nDNA. An 18.4-fold enrichment was observed in this experiment. IP: ImmunoPurified mitochondria; Input: Corresponding input 1.</p>
</figcaption>
</figure>
<h3 id="sec4.2">Protein analyses (western blot)</h3>
<p>For standard Western blot assays, the total or ½ of the total sample of immunopurified (IP<sup>ed</sup>) mitochondria are loaded into a 10-well gel (1.5 mm, 1653365, Bio Rad) Mini-PROTEAN (1658030, Bio-Rad) electrophoresis system. As we have never been able to efficiently elute <sup>HA- tagged</sup>mitochondria from the anti-HA magnetic beads using a HA peptide (I2149, Sigma-Aldrich), total protein concentration cannot be quantified using a standard BCA Protein Assay. See troubleshooting <a href="#sec7.9">problem 5</a> for further information. Protein is extracted from IP<sup>ed</sup> mitochondria by lysing the sample in 15 μL of Radioimmunoprecipitation Assay (RIPA) buffer for 20 min on ice. Then, using a standard magnetic rack (SureBeads, Bio-Rad or equivalent), the magnetic beads (still coated with their anti-HA antibodies) are pelleted and the lysate, with mitochondrial proteins, is collected. This step greatly reduces the presence of mouse HA-antibodies in the protein sample. 5 μL of 4<b>×</b> Laemmli Sample Buffer (1610747, Bio-Rad) is added to the sample which is then denatured at 100°C for 5 min, centrifuged at 12,000 × <i>g</i> for 10 min, and loaded into one well of an electrophoresis gel. Alternatively, the IP<sup>ed</sup> mitochondria can be resuspended in 20 μL of standard 1<b>×</b> Laemmli Sample Buffer and denatured by boiling at 100°C for 5 min, centrifuged at 12,000 × <i>g</i> for 10 min, and loaded in one well of an electrophoresis gel.</p>
<div class="note">
<span class="note-title">Note:</span> Input 1 can be thawed, resuspended in 200 μL of RIPA buffer with protease inhibitor cocktail, and lysed for 30 min on ice. Animal tissues can be dounced (50 strokes) with a 2 mL Dounce tissue grinder (D8938, Sigma-Aldrich) and potentially sonicated with a probe sonication system on ice (5 × 30 s, amplitude: 10%–40%; Sonics VibraCell VCX130), as shown previously (<a href="#bib5">Cummins et al., 2019</a>). The sample should be centrifuged for 30 min at 18,710 × <i>g</i> at 4°C, with the supernatant kept as the protein lysate. 5 μL of 4<b>×</b> Laemmli Sample Buffer (1610747, Bio-Rad) should be added to 15 μL of the protein lysate and the sample denatured at 100°C for 5 min, centrifuged at 12,000 × <i>g</i> for 10 min, and entirely loaded in one well of an electrophoresis gel. Alternatively, Input 1 can be denatured by boiling at 100°C for 5 min in 100 μL of standard 1<b>×</b> Laemmli Sample Buffer with 5–20 μL loaded into the well of an electrophoresis gel system.</div>
<p>Protein extracts obtained from Input 1 and IP<sup>ed</sup> mitochondria can be used to assess the enrichment of mitochondrial proteins over proteins from different cellular compartments by Western blot, thereby reflecting the enrichment of the mitochondrial fraction. Protein samples are loaded on a 10% polyacrylamide gel and blotted on an Immobilon-FL PVDF membrane (IPFL00010) previously activated with ethanol. The membrane is blocked with 5% bovine serum albumin (BSA) in TBST buffer before incubation with primary and secondary antibodies (the latter coupled to either IRDye 680RD or 800RD) and imaged using an Odyssey Fc Imager (Li-COR). We usually verify the enrichment of mitochondrial proteins compared to the nuclear Erythroid-Like-Transcription (ELT-2) factor monoclonal antibody [455-2A4] from the Developmental Studies Hybridoma Bank (DSHB, Iowa). Alternatively, the cell adhesion protein Drosophila Discs LarGe homolog-1 (DLG-1) monoclonal antibody ([DLG1], DHSB, Iowa). ELT-2 and DLG-1 are abundantly detected by Western blot in Input 1 but are absent from the IP<sup>ed</sup> mitochondrial fraction. To observe an example of ELT-2, see <a href="#fig3">Figure 3</a>D of previously published work (<a href="#bib2">Ahier et al., 2018</a>). For DLG-1, data is not shown. To confirm the presence of mitochondrial proteins in Input 1 and the IP<sup>ed</sup> mitochondrial fraction, a non-exhaustive list of antibodies can be successfully used (<a href="#tbl2">Table 2</a>). Antibodies against MTCO1 (1D6E1A8), ATPB (3D5), HSP-60 (HSP60), and HA-tag (C29F4, for TOMM-20::mKate2::HA bait detection) work efficiently for Western blot applications, as seen in <a href="#fig3">Figures 3</a>B and 3D in previous publications (<a href="#bib2">Ahier et al., 2018</a>).</p>
<table id="tbl2">
<caption>Table 2. Experimental details about the antibodies used for Western blot</caption>
<thead>
<tr>
<th>Target</th>
<th>Antibody</th>
<th>Host</th>
<th>Subcellular localisation</th>
<th>Working dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>ELT-2</td>
<td>[455-2A4]</td>
<td>Mouse</td>
<td>Nucleus</td>
<td>1:200</td>
</tr>
<tr>
<td>DLG-1</td>
<td>[DLG1]</td>
<td>Mouse</td>
<td>Cellular membrane</td>
<td>1:100</td>
</tr>
<tr>
<td>MTCOI/CTC-1</td>
<td>1D6E1A8<sup><a href="#tblfn1">a</a></sup>
</td>
<td>Mouse</td>
<td>Mitochondrion</td>
<td>1:1000</td>
</tr>
<tr>
<td>ATPB/ATP-2</td>
<td>3D5</td>
<td>Mouse</td>
<td>Mitochondrion</td>
<td>1:1000</td>
</tr>
<tr>
<td>HSP60/HSP-60</td>
<td>HSP60</td>
<td>Mouse</td>
<td>Mitochondrion</td>
<td>1:250</td>
</tr>
<tr>
<td>TOMM-20::mKate2::HA</td>
<td>C29F4</td>
<td>Rabbit</td>
<td>Mitochondrion</td>
<td>1:5000</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Do not boil the sample before loading on gel.</p>
<div class="note">
<span class="note-title">Note:</span> PVDF membrane is incubated with primary antibodies for 12–16 h at 4°C under gentle rocking in 5% bovine serum albumin (BSA) in Tris-buffered saline with 0.1% Tween 20 detergent (TBST).</div>
<div class="note">
<span class="note-title">Note:</span> PVDF membrane is incubated with secondary antibodies (Goat anti-mouse - IRDye 680RD or Goat anti-rabbit- IRDye 800CW, both diluted at 1:20000) in 5% BSA in TBST for 1 h at 22°C–25°C under gentle rocking. The membrane is washed 3 × 10 min in TBST between primary and secondary incubations and before image acquisition.</div>
<h3 id="sec4.3">Functional assays</h3>
<p>For functional assays, cell-specific mitochondria must be purified in the morning and assayed the same day (within 4 h of purification). As their elution from the anti-HA magnetic beads using a HA peptide is inefficient, mitochondria are used attached to anti-HA magnetic beads. The whole sample, or potentially ½ of it, can be used to measure oxygen consumption rate as in the Seahorse metabolic assay using the Seahorse XFe96 FluxPak kit (ref. SEA102601100, Agilent) and an Agilent Seahorse XFe Analyzer (<a href="#bib2">Ahier et al., 2018</a>).</p>
<div class="note">
<span class="note-title">Note:</span> Input 1, which was kept on ice during the CS-MAP procedure, can also be used in the assay.</div>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The number of purified mitochondria can be quantified by imaging a glass slide without a coverslip of 2 μL of the sample from step 29 (<a href="#sec3.4">immunomagnetic purification of mitochondria</a>), as shown in <a href="#fig8">Figure 8</a>. As the mitochondria are still bound to the anti-HA magnetic beads, a magnet can be used to carefully attract IP<sup>ed</sup> mitochondria toward the border of the liquid drop to concentrate them into a small crescent shape (<a href="#fig8">Figures 8</a>A and 8B). The concentrated IP<sup>ed</sup> mitochondria are then imaged at 20<b>×</b> magnification (objective: EC plan-Neofluar 20<b>×</b>/0.50 M27, without immersion) under a fluorescence compound microscope (light source intensity 53%, exposure time 11 s, excitation wavelength 587 nm, emission wavelength 610 nm, binning mode 2.2) to create a montage of the whole crescent. Imaging must be performed quickly before the sample dries out. The total number of mKate2-labelled mitochondria can be quantified with a tally counter using the Grid plugin (Wayne Rasband) in FIJI (<a href="#bib10">Schindelin et al., 2012</a>).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1178-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Quantification of mKate2-labelled IP<sup>ed</sup> mitochondria obtained from SJZ47 <i>myo-3p::TOMM-20::mKate2::HA</i>
</div>
<p>IP<sup>ed</sup> mitochondria were attracted to the edge of the sample using a magnet.</p>
<p>(A and B) An approximative and manual assembling of a crescent shape formed by the beads and mKate2-labelled <sup>HA-tagged</sup>mitochondria is shown. The white dotted line indicates the front of the crescent.</p>
<p>(C–F) A portion of the crescent shape structure formed by the beads and mitochondria is shown. (C and D) mKate2-labelled <sup>HA-tagged</sup>mitochondria bound to anti-HA magnetic beads. (E and F) Anti-HA magnetic beads saturated with HA-peptide block the binding of the <sup>HA-tagged</sup>mitochondria (negative control). Mitochondria were quantified using FIJI and the Grid plugin (shown as a yellow grid). Observations were performed with a fluorescence compound microscope using DIC and red channel (excitation wavelength 587 nm, emission wavelength 610 nm), at 20<b>×</b> magnification, non-immersed. Scale bar: 100 μm.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol is not efficient on young larval stages (i.e., L1 to L3) due their small size and physical resistance to Dounce-based homogenization. Performing this protocol on a mixed staged population can lead to the presence of intact young larval stage animals in the final purified mitochondria sample, which may compromise downstream applications.</p>
<p>The expression pattern of the <i>TOMM-20::mKate2::HA</i> transgene can influence the efficiency of the protocol. Even though we have shown that this protocol can be applied to purify mitochondria from a single cell in a large population of animals (<a href="#bib2">Ahier et al., 2018</a>), final yields of purified mitochondria can be low. Furthermore, some cells can be refractory to physical homogenization (e.g., pharyngeal muscle cells).</p>
<p>We have never been able to efficiently elute HA-tagged mitochondria from anti-HA magnetic beads using a HA peptide. Similar issues have been reported for 3<b>×</b>HA-tagged mitochondria (<a href="#bib9">Kuhnert et al., 2020</a>). We believe that this is due to the strong interaction between the HA-tag and the HA antibody. The presence of HA antibodies in the sample can potentially preclude further mass spectrometric analyses.</p>
<p>We have established the CS-MAP method exclusively using a HA tag as our efforts to develop the method using a FLAG (DYKDDDDK) tag were unsuccessful. Even though the reason for this failure remains unclear, similar results have been observed previously (<a href="#bib4">Chen et al., 2016</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Washing the animals in M9 buffer does not appear to clear them of bacteria as the sample still appears cloudy after several washes (<a href="#sec3.1">animal harvesting and cleaning</a> step 2).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Although <i>E. coli</i> OP50 bacteria are relatively easy to clear with M9 solution, extra washes can be performed if the sample still appears cloudy. Other strains of bacteria (e.g., <i>E. coli</i> HB101) can be more difficult to clear from the animal sample. Unidentified bacterial contamination can elude removal by washing as it can remain stuck to the animal. This contamination can later adhere to the anti-HA magnetic beads. Therefore, contamination can preclude the normal procedure and should be removed by additional washes where necessary.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>The enriched mitochondrial fraction contains low amounts of mKate2-labeled mitochondria (<a href="#sec3.2">generating an enriched mitochondrial fraction</a> step 13).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>1) The expression of the <i>TOMM-20::mKate2::HA</i> transgene in a limited number of cells can explain the subsequent low yield of extracted mKate2-labelled mitochondria. In this case, starting from a larger population of animals may be beneficial. However, the douncing step is the most common source of variability in the quality of the enriched mitochondrial fraction. Indeed, the Dounce tissue grinder set erodes after multiple uses and becomes less efficient. When douncing efficiency is lost, less mitochondria are released. Attempting to counteract this by over-douncing can create more debris and generate fragmented mitochondria. Instead, a new Dounce should be used. Additionally, we observed inconsistent outcomes when douncing samples suspended in volumes less than 1.5 mL. Optimizing the number of douncing strokes has been tried with variable outcomes. 2) Perform a proper calibration of the red channel of the microscope (with an N2 control condition). Even though settings are dependent on the microscope, several seconds of exposure time may be required to observe mKate2-labelled mitochondria. Saved microscope settings can be reused in the future.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Low yields of mKate2-labeled IP<sup>ed</sup> mitochondria are obtained at the end of the CS-MAP procedure (<a href="#sec3.4">immunomagnetic purification of mitochondria</a> step 30).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>1) First of all, assess the quality of the enriched mitochondrial fraction. If too few mitochondria are present, do not proceed to the next step. Note that the enriched mitochondrial fraction shown in <a href="#fig6">Figure 6</a>B is especially good, but approximately half as many mKate2-labelled mitochondria as the quantity shown here is still suitable for proceeding to the next step. 2) Ensure that the magnetic beads have been carefully washed before use. Any traces of detergent or NaN<sub>3</sub>, in which beads are stored, will affect or destroy mitochondria. 3) Avoid bacterial contamination when growing animals. Some bacteria can stick to the magnetic beads and decrease their efficiency. 4) Excessive large debris and aggregates in the enriched mitochondrial fraction can affect the subsequent immunopurification step. Try to disrupt aggregates by pipetting the sample up and down (as described in steps 13 and 14 of <a href="#sec3.2">generating an enriched mitochondrial fraction</a>).</p>
<h3 id="sec7.7">Problem 4</h3>
<p>High levels of debris, nuclear proteins, or nuclear DNA are contaminating the final IP<sup>ed</sup> mitochondria sample (<a href="#sec3.4">immunomagnetic purification of mitochondria</a> step 30).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>1) Normally, the stringent washing procedure presented in this protocol prevents this issue. Nevertheless, experimental variation can occur and the presence of obvious debris among mKate2-labelled mitochondria can be assessed using a fluorescent compound microscope. If required, an extra wash can be performed. Also, the washing efficiency can be monitored using anti-HA magnetic beads blocked with HA peptide (I2149, Sigma-Aldrich). Ten μg·mL<sup>−1</sup> of HA- peptide can be used to block 100–200 μg of anti-HA magnetic beads. As anti-HA magnetic beads blocked with HA peptide cannot bind HA-labeled mitochondria, a properly washed sample should be free of mKate2-labelled mitochondria. Ultimately, the presence of nuclear protein contamination can be assessed by Western Blot (see <a href="#expected-outcomes">expected outcomes</a> – proteins analyses) while potential nuclear DNA contamination can be assessed by semi-quantitative PCR or qPCR, as described in <a href="#expected-outcomes">expected outcomes</a> – nucleic acid analyses, <a href="#fig7">Figure 7</a>, and previous publications (<a href="#bib2">Ahier et al., 2018</a>, <a href="#bib1">2021</a>). 2) Monitor the quality of PEB used for washing. The level of BSA in this buffer makes it highly prone to contamination; therefore, it is advised to prepare it immediately prior to use or keep it frozen at −20°C.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Western blot analyses of the IP<sup>ed</sup> mitochondria sample are compromised because of the detection of the mouse anti-HA antibody (used for CS-MAP) due to cross reaction with an anti-mouse secondary antibody to detect other proteins of interest (<a href="#expected-outcomes">expected outcomes</a> – protein analyses).</p>
<h3 id="sec7.10">Potential solution</h3>
<p><sup>HA-tagged</sup>mitochondria cannot be eluted efficiently using a HA peptide (I2149, Sigma-Aldrich). Therefore, when <sup>HA-tagged</sup>mitochondria bound to HA-beads are processed for a Western blot, as described in <a href="#expected-outcomes">expected outcomes</a> – protein analyses, mitochondrial proteins and mouse anti-HA antibodies are detected on the same membrane. This precludes detection of most proteins between 50 and 25 kDa using a primary antibody raised in mice, as the relevant secondary antibody will strongly cross-detect both the light and heavy chains of the anti-HA IgG antibody, masking any other signal. To avoid this, we lyse the <sup>HA-tagged</sup>mitochondria bound to HA-beads with RIPA buffer on ice before using a suitable magnet or magnetic rack to transfer only the supernatant to a new tube. As the inner and outer mitochondrial membranes are disrupted through this process, this method facilitates the extraction of most mitochondrial proteins with the exception of the TOMM-20::mKate2::HA fusion protein, which will partially remain attached to the HA-beads.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ahier, A., Dai, C.Y., Kirmes, I., Cummins, N., Hung, G.C.C., Gotz, J., and Zuryn, S. (2021). PINK1 and parkin shape the organism-wide distribution of a deleterious mitochondrial genome. Cell Rep. <i>35</i>, 109203. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref1">View at publisher</a></p>
<p id="bib2">Ahier, A., Dai, C.Y., Tweedie, A., Bezawork-Geleta, A., Kirmes, I., and Zuryn, S. (2018). Affinity purification of cell-specific mitochondria from whole animals resolves patterns of genetic mosaicism. Nat. Cell Biol. <i>20</i>, 352-360. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref2">View at publisher</a></p>
<p id="bib3">Byerly, L., Cassada, R.C., and Russell, R.L. (1976). The life cycle of the nematode Caenorhabditis elegans. I. Wild-type growth and reproduction. Dev. Biol. <i>51</i>, 23-33. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref3">View at publisher</a></p>
<p id="bib4">Chen, W.W., Freinkman, E., Wang, T., Birsoy, K., and Sabatini, D.M. (2016). Absolute quantification of matrix metabolites reveals the dynamics of mitochondrial metabolism. Cell <i>166</i>, 1324-1337 e11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref4">View at publisher</a></p>
<p id="bib5">Cummins, N., Tweedie, A., Zuryn, S., Bertran-Gonzalez, J., and Gotz, J. (2019). Disease-associated tau impairs mitophagy by inhibiting Parkin translocation to mitochondria. EMBO J. <i>38</i>, e99360. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref5">View at publisher</a></p>
<p id="bib6">Gurtubay, J.I., Goni, F.M., Gomez-Fernandez, J.C., Otamendi, J.J., and Macarulla, J.M. (1980). Triton X-100 solubilization of mitochondrial inner and outer membranes. J. Bioenerg. Biomembr. <i>12</i>, 47-70. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref6">View at publisher</a></p>
<p id="bib7">Hadwiger, G., Dour, S., Arur, S., Fox, P., and Nonet, M.L. (2010). A monoclonal antibody toolkit for C. elegans. PLoS One <i>5</i>, e10161. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref7">View at publisher</a></p>
<p id="bib8">Hahn, A. and Zuryn, S. (2019). The cellular mitochondrial genome landscape in disease. Trends Cell. Biol. <i>29</i>, 227-240. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref8">View at publisher</a></p>
<p id="bib9">Kuhnert, F., Stefanski, A., Overbeck, N., Drews, L., Reichert, A.S., Stuhler, K., and Weber, A.P.M. (2020). Rapid single-step affinity purification of HA-tagged plant mitochondria. Plant Physiol. <i>182</i>, 692-706. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref9">View at publisher</a></p>
<p id="bib10">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref10">View at publisher</a></p>
<p id="bib11">Stiernagle, T. (2006). Maintenance of C. elegans (WormBook), pp. 1-11. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00658-4/sref11">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Steven Zuryn, E-mail: <a href="mailto:s.zuryn@uq.edu.au">s.zuryn@uq.edu.au</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyse datasets and codes.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/501100000925">NHMRC</a> project grants GNT1128381 (to S.Z.) and GNT1162553 (to S.Z.), <a href="https://doi.org/10.13039/100007615">ARC</a> discovery grants DP200101630 (to S.Z.), a <a href="https://doi.org/10.13039/100008763">Clem Jones Centre for Ageing</a> Dementia Research flagship grant (to S.Z.), and a Stafford Fox senior research fellowship (to S.Z.). Some strains were provided by the CGC, which is funded by the <a href="https://doi.org/10.13039/100000002">NIH</a> Office of Research Infrastructure Programs (P40 OD010440). Some antibodies were obtained from the Developmental Studies Hybridoma Bank (DSHB), created by the NICHD of the NIH and maintained at The University of Iowa, Department of Biology, Iowa City, IA 52242.</p>
<h3>Author contributions</h3>
<p>A.A. carried out experiments. S.Z. contributed to the interpretation of experiments. A.A. and S.Z. designed and interpreted experiments. A.A., T.O., and S.Z. wrote the paper. All authors approved the final manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>