<article>
<section>
<h1 id="header">Live imaging and quantitation of nascent transcription using the MS2/MCP system in the <i>Drosophila</i> embryo</h1>
<p><time datetime="2021-03-19">Published: March 19, 2021</time></p>
<p>Caroline Hoppe<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Hilary L. Ashe<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Faculty of Biology, Medicine and Health, University of Manchester, Manchester M13 9PT, UK</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:caroline.hoppe@manchester.ac.uk">caroline.hoppe@manchester.ac.uk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:hilary.ashe@manchester.ac.uk">hilary.ashe@manchester.ac.uk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100379">10.1016/j.xpro.2021.100379</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Visualizing transcription live in <i>Drosophila</i> is providing important new insights into the spatiotemporal regulation of transcription. Here, we describe a protocol to visualize and quantitate transcription from gene loci that are tagged with MS2 stem-loop sequences in the <i>Drosophila</i> embryo. MS2 stem-loop sequences are recognized by a coat protein fused to a fluorescent protein and visualized with microscopy. We also describe an analysis pipeline to extract and subsequently quantify transcription dynamics.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib23">Hoppe et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Temporal transcription dynamics can be determined using MS2 live imaging</li>
<li>
<i>Drosophila</i> embryos are collected and mounted for live imaging</li>
<li>Nascent transcription in the early embryo is detected by live confocal imaging</li>
<li>An image analysis pipeline extracts the transcriptional profile for each nucleus</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Experimental design considerations</h3>
<p>The MS2/MS2 coat protein (MCP) imaging system (<a href="#fig1">Figure 1</a>) has been used to extract transcription dynamics at the single-cell level with high spatial and temporal resolution in tissue culture cells and <i>in vivo</i> (<a href="#bib3">Bertrand et al., 1998</a>; <a href="#bib9">Dufourt et al., 2018</a>; <a href="#bib10">Falo-Sanjuan et al., 2019</a>; <a href="#bib16">Fukaya et al., 2017</a>; <a href="#bib17">Fusco et al., 2003</a>; <a href="#bib19">Garcia et al., 2013</a>; <a href="#bib23">Hoppe et al., 2020</a>; <a href="#bib24">Koromila and Stathopoulos, 2019</a>; <a href="#bib27">Lee et al., 2019</a>; <a href="#bib30">Lionnet et al., 2011</a>; <a href="#bib31">Lucas et al., 2013</a>; <a href="#bib36">Scholes et al., 2019</a>; <a href="#bib40">Yamada et al., 2019</a>). In <i>Drosophila</i> this technique has been used to investigate how target genes respond to different signals and transcription factor inputs (<a href="#bib2">Berrocal et al., 2020</a>; <a href="#bib4">Bothma et al., 2014</a>; <a href="#bib10">Falo-Sanjuan et al., 2019</a>; <a href="#bib19">Garcia et al., 2013</a>; <a href="#bib23">Hoppe et al., 2020</a>; <a href="#bib25">Lammers et al., 2020</a>; <a href="#bib31">Lucas et al., 2013</a>, <a href="#bib32">2018</a>; <a href="#bib40">Yamada et al., 2019</a>), the function of enhancers and their contribution to transcriptional output (<a href="#bib5">Bothma et al., 2015</a>; <a href="#bib15">Fukaya et al., 2016</a>; <a href="#bib24">Koromila and Stathopoulos, 2019</a>; <a href="#bib36">Scholes et al., 2019</a>), transcriptional memory (<a href="#bib11">Ferraro et al., 2016</a>) and transvection (<a href="#bib28">Lim et al., 2018a</a>). The MS2 imaging system uses arrays of the MS2 bacteriophage derived stem-loop sequences that are inserted into non-coding regions of either endogenous genes (<a href="#bib14">Fukaya, 2020</a>; <a href="#bib23">Hoppe et al., 2020</a>; <a href="#bib29">Lim et al., 2018b</a>, <a href="#bib28">2018a</a>) or transgenes (<a href="#bib10">Falo-Sanjuan et al., 2019</a>; <a href="#bib16">Fukaya et al., 2017</a>; <a href="#bib19">Garcia et al., 2013</a>; <a href="#bib31">Lucas et al., 2013</a>). Once transcribed by RNA Polymerase II (Pol II), the RNA MS2 stem-loop is recognized by its cognate coat protein MCP, which binds the loop structure as a dimer (<a href="#bib39">Wu et al., 2012</a>). Here, we describe how to mount embryos and image nascent transcription in living <i>Drosophila</i> embryos using confocal microscopy and how to quantify nascent transcription using an image analysis pipeline (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Visualization of transcription with the MS2/MCP imaging system</div>
<p>The MS2 live imaging system uses RNA stem-loop repeats, derived from the MS2 bacteriophage that are bound by the cognate MS2-coat protein (MCP) fused to a fluorescent protein. Transcription factor = TF.</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Overview showing image acquisition and image analysis covered in this protocol</div>
<p>The steps are outlined with associated timings.</p>
</figcaption>
</figure>
<p>Visualization of nascent transcription is achieved by fusing the MCP protein to a fluorescent protein. The fluorescent signal intensity is influenced by the position of the MS2 stem-loops within the gene sequence. The signal will be brighter when imaging transcription if the MS2 stem-loops are inserted in the 5′ UTR (as shown in <a href="#fig1">Figure 1</a>) as opposed to their placement in the 3′ UTR. For the latter, MCP fluorescence will only become visible after Pol II has transcribed the 3′ UTR region, which is followed quickly by transcription termination and dissociation of the transcript from the nascent site of transcription, resulting in a weaker fluorescent signal (<a href="#bib19">Garcia et al., 2013</a>).</p>
<p>In addition, the fluorescence intensity is dependent on the number of MS2 stem-loop repeats inserted into the gene of interest. Most <i>Drosophila</i> studies use an MS2 stem-loop cassette containing 24 repeats, which has been shown to be sufficient to detect mRNA transcription dynamics (<a href="#bib10">Falo-Sanjuan et al., 2019</a>; <a href="#bib19">Garcia et al., 2013</a>; <a href="#bib23">Hoppe et al., 2020</a>; <a href="#bib25">Lammers et al., 2020</a>; <a href="#bib31">Lucas et al., 2013</a>). However, a stronger fluorescent signal is expected when imaging a gene with 128× MS2 stem-loops inserted (<a href="#bib37">Tantale et al., 2016</a>), which will facilitate the study of weakly transcribed genes.</p>
<p><i>Drosophila</i> stocks are available that express MCP fused to different fluorescent proteins including GFP, RFP and mCherry. For example the following fly lines are available: P{nos-MCP-EGFP} [RRID:BDSC_60340], P{UAS-MCP-RFP} [RRID:BDSC_27419], P{nos-NLS-mCherry-MCP} (<a href="#bib42">Bothma et al., 2018</a>), P{nos-MCP-mCherry} (<a href="#bib43">Liu et al., 2020</a>) and P{nos-MCP-mCherry-FLAG} (<a href="#bib21">Hayashi et al., 2014</a>). During image analysis nuclear markers are used to identify and track nuclei through time, such as Histone2Av or Nucleoporin(107) fluorescent protein fusions (<a href="#bib19">Garcia et al., 2013</a>; <a href="#bib31">Lucas et al., 2013</a>).</p>
<div class="note">
<span class="note-title">Note:</span> A protocol outlining MS2 stem-loop insertion into transgenes and live imaging was published by <a href="#bib18">Garcia and Gregor, 2018</a>. A protocol describing how to insert MS2 stem-loop sequences into endogenous gene loci is described in the accompanying paper (<a href="#bib22">Hoppe and Ashe, 2021</a>).</div>
<div class="note">
<span class="note-title">Note:</span> In addition to the MS2/MCP system, an orthogonal mRNA tagging system, based on the <i>Pseudomonas</i> phage derived PP7 stem-loops and respective PCP coat protein, can be used to visualize nascent transcription (<a href="#bib7">Chao et al., 2008</a>; <a href="#bib35">Pichon et al., 2018</a>).</div>
<h3 id="sec1.2">Prepare heptane glue coated coverslips for the live imaging setup</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min to coat coverslips; previously, soak the tape overnight (approximately 14 h) to remove glue from it</div>
<p>The heptane glue coated coverslips will be used to position embryos for live-cell imaging.</p>
<ol>
<li>Unroll sticky tape and transfer several lengths into a 50 mL reagent bottle.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Make sure that the sticky tape is not poisonous to <i>Drosophila</i> embryos. We use yellow double-sided plastic tape, 25 mm × 20 m (RS Components, Cat# 555-033) or packaging tape (Tesa, Cat# 4124).</div>
<ol start="2">
<li>Fill half of the bottle with heptane to dissolve the glue from the tape.</li>
<li>Keep the sticky tape in heptane for at least 14 h.</li>
<li>Pipette 60 μL viscous heptane glue onto the center of the coverslip (No. 1.5, 18 × 18 mm; Scientific Laboratory Supplies, Cat# MIC3120).</li>
<li>Let heptane evaporate in a fume hood until a thin film of glue is visible on the coverslip.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the heptane glue is too viscous, dilute with more heptane. If you see debris on the coverslip during imaging, remove heptane glue from the sticky tape filled bottle, spin down in a microcentrifuge and transfer the supernatant to a new tube to use on new coverslips.</div>
<div class="note">
<span class="note-title">Note:</span> Make sure to protect the coverslips from dust and other dirt. For example, keep prepared coverslips in a clean Petri dish or another closed container.</div>
<div class="note">
<span class="note-title">Note:</span> Ideally prepare coverslips a few hours before the live imaging experiment. If stored in a closed container, coverslips can be kept for up to two weeks.</div>
<h3 id="sec1.3">Set up fly crosses for the live imaging experiment</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 days</div>
<p>The purpose of this step is to set up fly crosses that are then used for embryo collection. Female flies of the genotype His-RFP; MCP-GFP (RRID:BDSC_60340) maternally deposit mature His-RFP and MCP-GFP fusion proteins into the egg. His-RFP is used to locate and track nuclei during imaging to aid analysis.</p>
<ol start="6">
<li>In order to obtain sufficient embryo numbers, a fly cage containing approximately 100 virgin females of the genotype His-RFP; MCP-GFP and at least 50 males carrying the MS2 loop locus is set up, ideally three days before the live imaging experiment and kept at 25°C (<a href="#fig3">Figure 3</a>, step 1). Males homozygous for the MS2 insertion should be used if possible.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Overview of the live imaging experiment</div>
<p>Female virgin flies (His-RFP; MCP-GFP) are crossed to homozygous males (24xMS2-gene) and kept in a collection bottle (1). Embryos are aged for 90 min and collected on apple juice agar plates (2) before embryos are dechorionated using 50% bleach (3). Dried embryos are lined up on a raised agar plate and attached to a heptane glue coated cover slip, which is added on top of a cover slip/halocarbon oil bridge (4). Embryo development is imaged on a confocal microscope (5), and single-cell fluorescent traces are extracted during subsequent analysis (6).</p>
</figcaption>
</figure>
<ol type="a">
<li>Cages are set up with apple juice agar plates containing yeast paste.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The protocol described here visualizes active transcription from one allele per nucleus.</div>
<div class="note">
<span class="note-title">Note:</span> Do not keep the fly cage for longer than one week as, in our experience, the incidence of developmental defects increases with age.</div>
<div class="optional">
<span class="optional-title">Optional:</span> In order to reduce the inherent background of unbound MCP-GFP, the overall concentration can be reduced by using females that are heterozygous for His-RFP; MCP-GFP. The amount of maternally deposited His-RFP and MCP-GFP protein will be reduced but still sufficient for imaging of transcription (<a href="#bib14">Fukaya, 2020</a>; <a href="#bib23">Hoppe et al., 2020</a>). This approach was used to acquire the example dataset supplied with this protocol.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> To image both gene alleles at the same time, His-RFP; MCP-GFP flies can be crossed first to homozygous MS2 flies and the resulting female offspring can be crossed again to males carrying the MS2 locus. Half of the resulting embryos will be homozygous for the imaging locus if the tagged locus is located on an autosome or a quarter for a gene on the X-chromosome.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Halocarbon oil 27</td>
<td>Sigma</td>
<td>Cat# H8773; CAS: 9002-83-9</td>
</tr>
<tr>
<td>Halocarbon oil 700</td>
<td>Sigma</td>
<td>Cat# H8898; CAS: 9002-83-9</td>
</tr>
<tr>
<td>2Work thin bleach</td>
<td>Banner</td>
<td>Cat# 1190012</td>
</tr>
<tr>
<td>TritonX-100</td>
<td>Sigma</td>
<td>Cat# X100; CAS: 9002-93-1</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Example datasets and analysis files</td>
<td>This study</td>
<td>Mendeley Data DOI: 10.17632/jv9fjpn9cg.1</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>; <i>y</i><sup><i>1</i></sup><i>w∗</i>; P{w+mC=His2Av-mRFP1}; P{w+mC=nos-MCP.EGFP}</td>
<td>Bloomington Drosophila Stock Center</td>
<td>RRID:BDSC_60340</td>
</tr>
<tr>
<td>
<i>D. melanogaster</i>; <i>y</i><sup><i>1</i></sup><i>w</i><sup><i>67c23</i></sup><i>; 24xMS2-ush</i>
</td>
<td><a href="#bib23">Hoppe et al., 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Imaris 9.2</td>
<td>Bitplane</td>
<td>RRID:SCR_007370</td>
</tr>
<tr>
<td>LAS X</td>
<td>Leica Microsystems Inc</td>
<td><a href="https://www.cellularimaging.nl/leica-las-x/">https://www.cellularimaging.nl/leica-las-x/</a></td>
</tr>
<tr>
<td>Code combining single-cell traces and spot detection from live imaging movies and using Imaris output</td>
<td><a href="#bib23">Hoppe et al., 2020</a></td>
<td>
<a href="https://github.com/TMinchington/sass">https://github.com/TMinchington/sass</a>, RRID:SCR_018797</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Leica TCS SP8 AOBS inverted microscope</td>
<td>Leica</td>
<td>N/A</td>
</tr>
<tr>
<td>Lumox dish 50, cell culture dish</td>
<td>Sarstedt AG &amp; Co</td>
<td>Cat# 94.6077.305</td>
</tr>
<tr>
<td>Coverslips no. 1.5, 18 × 18 mm</td>
<td>Scientific Laboratory Supplies</td>
<td>Cat# MIC3120</td>
</tr>
<tr>
<td>Coverslips no. 0, 18 × 18 mm</td>
<td>Scientific Laboratory Supplies</td>
<td>Cat# MIC3100</td>
</tr>
<tr>
<td>Yellow double-sided plastic tape</td>
<td>RS Components</td>
<td>Cat# 555-033</td>
</tr>
<tr>
<td>Packaging tape</td>
<td>Tesa</td>
<td>Cat# 4124</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Embryo collection and mounting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> Embryo collection, ∼90 min; mounting, ∼10 min</div>
<p>This section describes how embryos are collected and mounted for live-cell microscopy.</p>
<ol>
<li>On the morning of the experiment, change the fly cage plate to a fresh apple juice agar plate containing yeast paste and wait 1 h, to give the flies time to lay retained embryos. After 1 h change the plate again. If the gene of interest is expressed during nuclear cycle (nc) 14, collect embryos for 90 mins (<a href="#fig3">Figure 3</a>, step 2).</li>
<li>Embryos are washed off the plate with NaCl (0.7%)/TritonX-100 (0.05%; Sigma, Cat# X100) wash solution and dechorionated by adding freshly prepared 50% bleach (Banner, Cat# 1190012) to the plate for 2 min. Next, decant embryos into a sieve and wash them with water until they are free of bleach (<a href="#fig3">Figure 3</a>, step 3). Dry the sieve with paper towels and remove as much water as possible from the embryos.</li>
<li>Use a blade or tweezers to cut a square from a fresh apple juice plate, turn the plate over, and place the square on top of the plate so it is elevated. Move the plate under a dissection microscope. Transfer embryos onto the agar square using a brush (<a href="#fig3">Figure 3</a>, step 4).</li>
<li>Line up the embryos and orient them so that the area in which the gene of interest will be expressed is facing upwards (<a href="#mmc1">Methods video S1</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Position the embryos in one or two lines to make navigation under the microscope easier.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Mmc1.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Mounting <i>Drosophila</i> embryos for live imaging experiments, related to step 4</div>
</figcaption>
</figure>
<p>Mounting approximately 20 embryos will ensure that multiple embryos are at the correct developmental stage for imaging.</p>
<div class="note">
<span class="note-title">Note:</span> Try to exclude embryos that have already entered late nuclear cycles or are gastrulating. These embryos appear to have a transparent periphery and a more dense and structured center (<a href="#mmc1">Methods video S1</a>).</div>
<ol start="5">
<li>Next, prepare a cover slip/halocarbon oil bridge on a Lumox imaging dish (Sarstedt AG &amp; Co, Cat# 94.6077.305) (<a href="#fig3">Figure 3</a>, step 4).
<ol type="a">
<li>Use a cut pipette tip to add 70 μL of halocarbon oil (7:1, halocarbon oil 700:halocarbon oil 27; Sigma Cat# H8898 and Cat# H8773) onto the middle of the imaging dish.</li>
<li>On each side of the drop of oil place a coverslip (no. 0, 18 × 18 mm; Scientific Laboratory Supplies, Cat# MIC3100) and create a bridge (see <a href="#mmc1">Methods video S1</a> and <a href="#fig3">Figure 3</a>, step 4). The two coverslips should be approximately 0.6 cm apart.
<ol type="i">
<li>No. 0 cover slips used in this setup have a thickness of approximately 100 μm (Scientific Laboratory Supplies), whereas a <i>Drosophila</i> embryo has an approximate height of 180 μm (<a href="#bib33">Markow et al., 2009</a>). Therefore, the embryo becomes flattened against the coverslip.</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare one Lumox imaging dish per experiment. The dishes have a gas-permeable membrane and can be reused a few times before the membrane becomes too flexible (see <a href="#sec6.1">troubleshooting 1</a>).</div>
<ol start="6">
<li>Fix embryos in place with a heptane glue coated coverslip (<a href="#fig3">Figure 3</a>, step 4).
<ol type="a">
<li>Carefully pick up the heptane glue coated coverslip and gently drop it onto the embryos with the heptane glue side facing down toward the embryos.</li>
<li>Use tweezers to (very carefully) press the coverslip onto the embryos. You will notice that the embryos flatten slightly and press against the coverslip.</li>
<li>Pick up the coverslip with the embryos attached to it and place it onto the coverslip-bridge, sandwiching the embryos between the dish membrane and the coverslip in the drop of oil. Secure the top coverslip in place with a dot of nail polish.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> If the embryos are not attached properly to the heptane glue coated coverslip, they might start sliding during the live imaging time lapse. If embryos are flattened too much by the heptane glue coated coverslip, they may arrest during a mitotic cleavage cycle or display mitotic defects (see <a href="#sec6.1">troubleshooting 1</a>).</div>
<h3 id="sec2.2">Live-cell microscopy of nascent transcription</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>In this section, the microscopy setup and settings are outlined to visualize nascent transcription. The prepared sample can be imaged on an upright or inverted microscope setup. Here, embryos were imaged on an inverted Leica SP8 confocal microscope (<a href="#fig3">Figure 3</a>, step 5).</p>
<ol start="7">
<li>For imaging experiments, we use a Leica TCS SP8 AOBS inverted confocal microscope with a resonant scan head and a 40×/ 1.3 HC PL apochromatic oil objective with the pinhole set to 1.3 airy units.
<ol type="a">
<li>The microscope settings used to image the sample dataset were as follows:
<ol type="i">
<li>The scanning frequency of the resonance head was 8,000 Hz (with a minimum zoom of 1.25).</li>
<li>Scanning was set to bidirectional.</li>
<li>Images were acquired at 8 bit and with 1,024 pixels per line and 700 lines. These settings result in a pixel size of 378.79 nm × 378.79 nm, an image size of 387.5 μm × 264.77 μm and a pixel dwell time of 1.04 μs. This setup allows us to capture most of the <i>Drosophila</i> embryo.</li>
</ol>
</li>
<li>Images were collected using hybrid detectors and fluorophores were excited with a white laser. The example dataset was acquired using the following settings:
<ol type="i">
<li>White laser set to 70% with 488 nm (8%), 561 nm (2%) and 8× line averaging. These settings result in the following background corrected laser powers: 6.34 μW (488 nm) and 2.88 μW (561 nm). Ensure minimal photobleaching (see <a href="#sec6.3">troubleshooting 2</a>) and that the signal is not saturated.</li>
</ol>
</li>
<li>Z-stacks were acquired continuously with 1 μm steps. A depth of 55 μm per z-stack can be achieved with a final temporal resolution of 20 s per time frame.
<ol type="i">
<li>If the gene of interest is not highly expressed, the z-step size can be reduced to ensure that the transcription site is imaged in multiple z slices.</li>
</ol>
</li>
</ol>
</li>
<li>Find embryos on the imaging dish using brightfield illumination and rotate to align the embryos horizontally with their AP axis.</li>
<li>Check the age of embryos by looking at the nuclei using fluorescence illumination in the 561 nm excitation line. Pick an embryo where the nuclei have migrated to the surface and are currently undergoing nc13 (<a href="#fig3">Figure 3</a>, step 5).</li>
</ol>
<ol start="10">
<li>Set up a time lapse and include the full depth of nuclei in the z-direction. A maximum projected movie of the example time-lapse experiment can be found in <a href="#mmc2">Methods video S2</a> (previously described in <a href="#bib23">Hoppe et al., 2020</a>).
<ol type="a">
<li>It is advisable to set up the z-stack to include a number of slices below the nuclear volume at the start of imaging. This extra depth ensures that the nascent transcription sites are fully included within the imaging volume while nuclei elongate during nc14 (see example datasets).
<div class="optional">
<span class="optional-title">Optional:</span> Deconvolve the time-lapse data using the inbuilt Leica lightning deconvolution software. As an alternative, the Huygens Professional Deconvolution software can be used to deconvolve images but struggles with large file sizes.</div>
<div class="critical">
<span class="critical-title">Critical:</span> When imaging make sure that the full fluorescence of the transcription sites is included in the z-stack (not cut off on the top or bottom). If the expression domain of the gene of interest is very wide, imaging is potentially complicated by the degree of embryo flattening. In our experience, an active transcription site is visible in approximately 6–8 z slices at its maximum transcriptional activity.</div>
<div class="note">
<span class="note-title">Note:</span> Depending on the expression level, the fluorescence might be very weak and the imaging/genomic setup will need to be optimized (see <a href="#sec6.5">troubleshooting 3</a>).</div>
<div class="note">
<span class="note-title">Note:</span> If a different microscope is used the most important setting to consider is the imaging speed. As transcription is a very dynamic progress, care should be taken to accommodate a high frame rate. The time resolution of 20 s, as described in this protocol, can be improved further by imaging a smaller region of the embryo or smaller z-stacks. Achieving a high time resolution is often associated with a tradeoff in the image quality. Low raw image quality can complicate image analysis and potentially mask real signal under the background haze. Therefore, we recommend to use a microscope with a resonant scanner, a Piezo stage fixture, a Piezo objective collar, or alternative options to ensure high speed imaging.</div>
</li>
</ol>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. Maximum intensity projection of time-lapse experiment showing transcription of the endogenous <i>ush</i> gene using the MS2/MCP system, related to step 10</div>
<p>Time-lapse video was taken on an inverted SP8 confocal microscope (40× objective, depth of 55 μm and a final temporal resolution of 20 s per time frame).</p>
</figcaption>
</figure>
</li>
</ol>
<h3 id="sec2.3">Tracking of nuclei and identification of transcription sites</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼ 4.5 h/time-lapse dataset</div>
<p>This next section describes the initial semi-automated data analysis pipeline to extract the fluorescent signal from each nucleus and track the signal over time (<a href="#fig3">Figure 3</a>, step 6). This pipeline combines data analysis performed using Imaris software 9.2 (Bitplane, Oxford Instruments, Concord MA, RRID:SCR_007370) with open-access, custom python scripts. A sample time-lapse dataset (Embryo1_full_analysis.ims) with full nuclear traces and the corresponding analysis files is provided with this protocol. Additionally, a beginners dataset containing a small number of nuclei, time points and the corresponding analysis files is provided.</p>
<p>Files are deposited at Mendeley Data: https://doi.org/10.17632/jv9fjpn9cg.1</p>
<div class="note">
<span class="note-title">Note:</span> This analysis pipeline was conceptualized to be easily applicable to time-lapse datasets acquired using different microscopes, magnifications and of different embryo regions. In addition, this pipeline does not require coding knowledge or familiarity with MATLAB/python or other programming languages. A similar Imaris based pipeline has been developed independently by <a href="#bib40">Yamada et al (2019)</a>.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The time it takes to analyze an embryo dataset varies significantly with the quality of raw data, the size of the field of view and the number of time frames. The example dataset includes a cropped field of view containing approximately. 116 nuclei. The beginners dataset provides approximately 20 nuclei for quicker analysis using this pipeline.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other custom software is available to track nuclei and quantitate nascent transcription. Custom MATLAB scripts have been generated by <a href="#bib19">Garcia et al., 2013</a> and further optimized by <a href="#bib25">Lammers et al., 2020</a>. MitoTrack is an open-access software that can track nuclei and extract MCP fluorescence from maximum projected images, while keeping a lineage tree history through cell divisions (<a href="#bib38">Trullo et al., 2020</a>).</div>
<div class="note">
<span class="note-title">Note:</span> We outline the rationale of each analysis step to enable users of other analysis platforms to incorporate some aspects of this pipeline.</div>
<ol start="11">
<li>Load time-lapse microscopy data into Imaris 9.2 to render images in 3D (<a href="#fig4">Figure 4</a>, steps 1–3).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Semi-automated analysis pipeline</div>
<p>The developing <i>Drosophila</i> embryo is imaged over time using z-slices to cover the full depth of the nucleus and capture the full fluorescence (1). Images can be analyzed in the Imaris software (2), which allows the images to be rendered in 3D (3). The time-lapse dataset is cropped to a region of interest for further analysis. Nuclei are segmented using the “surfaces” module, the His-RFP channel and either “absolute intensity” or “background subtraction” thresholding (4), before they are tracked through time using an autoregressive motion algorithm (5). Transcription sites are segmented using the “spots” function and the MCP-GFP channel (6).</p>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Imaris supports all common microscopy file formats.</div>
<p><b>Nuclear segmentation:</b> As a first step, nuclei are segmented by creating a surface over objects detected in the His-RFP channel. In order to ease segmentation, the detectable objects are smoothed using a Gaussian filter (“smooth surface detail” setting).</p>
<ol start="12">
<li>Identify and segment nuclei based on the His-RFP fluorescence channel and using the inbuilt Imaris “surfaces” function. Follow the algorithm steps as they are displayed in the software.
<ol type="a">
<li>Select “track surfaces over time” in algorithm settings and either analyze the entire image or select a region of interest.</li>
<li>In the next step, select the His-RFP channel as the source channel and set “smooth surface detail” to 1 μm.</li>
<li>Different thresholding options can work well to segment nuclei.
<ol type="i">
<li>Absolute intensity thresholding option: When choosing “absolute intensity” select “split touching objects” to prevent the fusion of multiple nuclei. Defining the seed point diameter as 3 μm is a good starting point (<a href="#fig4">Figure 4</a>, step 4).</li>
<li>Background subtraction thresholding option: When choosing the “background subtraction” thresholding method, choose 1.3 μm as a start value for “diameter of largest sphere which fits into object.” In the next step, move the slider until nuclei are sufficiently identified and shaded (<a href="#fig4">Figure 4</a>, step 4). In our experience, it is easiest to make sure that the shaded spheres do not touch each other, instead of using the seed point option, to prevent any fusion to form one nucleus in subsequent steps.
<div class="note">
<span class="note-title">Note:</span> Nuclei in the example dataset were segmented using the “background subtraction” thresholding. Examples of both thresholding methods for one time point are included in the example dataset and were used to generate the images in <a href="#fig4">Figure 4</a>, step 4.</div>
</li>
</ol>
</li>
<li>In the next window, choose classifiers to separate nuclei by selecting as many filters as needed to obtain the best segmentation result. Filters can help to remove false-positive results from the segmentation. We find the following filters especially helpful: Number of voxels, Position Z, Distance to image border XY, Quality and Area.
<div class="critical">
<span class="critical-title">Critical:</span> Achieving a good segmentation result is important to prevent false-positive/-negative results during later analysis steps.</div>
</li>
</ol>
</li>
</ol>
<p><b>Nuclear tracking:</b> Imaris offers a number of different tracking algorithms that predict the surface (nuclear) object positions in the next frame. Based on the predicted positions the best matches are selected using a linear assignment algorithm to form track connections. The autoregressive motion tracking option considers the previous time point and predicts the same amount of object movement in the next time frame. The user can determine the distance an object is allowed to deviate from the predicted path (“maximum travel distance” input). More detailed information can be found in the Imaris handbook.</p>
<ol>
<li style="list-style-type: none;">
<ol start="5" type="a">
<li>Nuclei are tracked through time in 3D using the inbuilt autoregressive motion algorithm. The maximum frame gap size is set to 5 and the maximum travel distance is set to 5 μm (<a href="#fig4">Figure 4</a>, step 5).</li>
<li>After tracking is complete, tracks and nuclei can be corrected manually in the “edit” and “edit tracks” tabs.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Segmentation accuracy may vary based on raw data quality. Sometimes it can be difficult to find segmentation settings that will work on the whole time-lapse dataset. This is especially apparent if nuclei change shape or photobleaching has occurred. In this case, it can be easier to disable “track surfaces over time” in the surface wizard and segment nuclei in smaller time frame groups. Segment 10–50 time frames at a time by selecting them in the “region of interest” window. Follow the segmentation wizard to the end. Then, merge nuclei groups by selecting “merge” in the surfaces editing tab. After merging all nuclei, select “tracking” in the “rebuild” tab to track nuclei through time. Using this approach, nuclei segmentation can be manually adjusted and corrected before tracking. Nuclei in the example dataset were segmented in sets of 40 time frames, then merged and tracked.</div>
</li>
</ol>
</li>
</ol>
<p><b>Transcription site identification:</b> To determine the transcription activity, we quantify the full (sum) fluorescence of nascent transcription sites in 3D. This is achieved by placing spheres around the transcription sites that encapsulate the full fluorescence, which can then be measured.</p>
<ol start="13">
<li>Active transcription sites are identified in the MCP-GFP channel, using the inbuilt Imaris "spots" function. Follow the steps of the creation wizard.
<ol type="a">
<li>Analyze the full image or choose a region of interest. Select the MCP-GFP channel as “source channel” to identify nascent transcription foci.</li>
<li>In the next window, specify the spot/sphere size.
<ol type="i">
<li>Transcription foci can be estimated to be 1.8 μm in XY diameter.</li>
<li>The “model PSF-elongation along z-axis” will differ based on the microscope that was used and needs to be determined based on the dataset to be analyzed.</li>
</ol>
</li>
<li>In the next step, three-dimensional spheres, of identical size, are placed around the transcription signal by the software (<a href="#fig4">Figure 4</a>, step 6). It is important that the full fluorescence is contained within the sphere.
<ol type="i">
<li>When analyzing a new genotype, check that the full fluorescence is contained within the sphere at time points where the fluorescent signal is brightest.</li>
<li>If this is not the case, return to the previous step in the creation wizard and adjust the sphere size.</li>
</ol>
</li>
<li>Next, filters are used to classify spots. Select as many filters as needed to remove false-positive spots and identify as many “real” fluorescence spots as possible. We find the following filters especially helpful: Quality, Sum Intensity of MCP-GFP channel, Position Z, and Distance to image border XY.
<ol type="i">
<li>False-positive spots are most often detected outside the z-plane of transcription sites with a lower sum fluorescence than the average transcription site.</li>
</ol>
</li>
<li>Before continuing, check the spots that were created by the software by navigating through the time frames. When the fluorescent signal is very dim, transcription sites are sometimes not detected (see <a href="#sec6.7">troubleshooting 4</a>) or false-positive spots are added (see <a href="#sec6.9">troubleshooting 5</a>).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To change the size of all spot objects at the same time, a python based Imaris X-Tension “XTSpotResizer” is available. This X-Tension, written by Egor Zindy, can be downloaded at <a href="https://github.com/zindy/Imaris">https://github.com/zindy/Imaris</a> and integrated into Imaris under Edit -&gt; Preferences -&gt; Custom Tools. Locate the folder containing the downloaded “SpotResizeDialog.py,” “XTSpotReziser.py,” “TkDialog.py,” and “BridgeLib.py” in the X-Tension folder field and add it to the custom tools. The X-tension will appear in the Image Processing menu.</div>
<div class="note">
<span class="note-title">Note:</span> In the example dataset, transcription sites were excluded from nuclei sitting on the edge of the field of view as the onset of transcription could have been missed or the nascent transcription sites are not fully present in the field of view.</div>
<p><b>Background correction:</b> The inherent background fluorescence, originating from unbound MCP-GFP, is removed using the sass.py script. A linear regression line is fitted based on background fluorescence values determined in this step. The line equation will then be used to determine the average background fluorescence present at each time point and subtracted from the transcription site fluorescence measurements.</p>
<ol start="14">
<li>Manually add a second set of spots, of the same size as spots used to identify transcription foci, to generate background readings.
<ol type="a">
<li>Identify the first time frame that contains an MS2 signal. Add “new spots” using the Imaris wizard. On the first wizard tab, select “skip automatic creation, edit manually.” Manually add 4 spots (pointer in the “select” mode, press shift and left click) in the nuclear layer but outside of the expression domain or away from nuclei with active transcription. Make sure the background spots have the same size as spots used to identify transcription foci.</li>
<li>Add 4 background spots in every third-fifth time frame until the end of the time-lapse movie. We find these time intervals to be sufficient to create an accurate linear regression line.</li>
</ol>
</li>
<li>Export data files:
<ol type="a">
<li>Under File -&gt; Preferences -&gt; Statistics -&gt; Surfaces make sure that at a minimum “Position X/Y/Z,” “Ellipsoid Axis CX/CY/CZ” and “Ellipsoid Axis length C” are selected.</li>
<li>The “Position” output file will record all nuclear centroid X/Y/Z positions, nuclear ID and TrackID for every time frame. The Ellipsoid Axis C X/Y/Z positions and length determines the orientation of the ellipsoid nuclear axis in the C direction (<a href="#fig5">Figure 5</a>A).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Spot assignment and mitotic wave correction</div>
<p>(A) Each nascent transcription site is assigned to the closest nucleus in three dimensions using the proximity to the nuclear ellipsoid axis C (blue). Use of the ellipsoid axis C instead of a perpendicular axis (white line) results in a more accurate spot assignment in curved regions of the embryo.</p>
<p>(B) Time differences arising from the mitotic wave are corrected during the analysis. The expression domain is separated into mitotic zones that contain nuclei that undergo telophase in the same frame. The x-axis positions of the zone borders are recorded together with the number of time frames that need to be adjusted in order to reset t = 0.</p>
</figcaption>
</figure>
</li>
<li>Under File -&gt; Preferences -&gt; Statistics -&gt; Spots make sure that at least “Intensity Sum” and “Position X/Y/Z” are selected. Add other desired fluorescent measurements.</li>
<li>Select the surface icon that contains the tracked nuclei. Select the “statistics” tab and “detailed.” Export “all statistics to file.” Make sure the filename contains the word “cells” (<a href="#fig6">Figure 6</a>, step 1). This generates a folder containing an excel sheet for each statistic.
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Combining the Imaris output files of nuclei and spots using a custom python script</div>
<p>The statistic files of spots and surfaces are exported from the Imaris software (1) and saved in a subfolder. Using the custom python script “sass,” each detected transcription site is assigned to the closest nucleus (2). “Sass” outputs multiple images to investigate the analysis accuracy including plots of the expression domain (3) and the percent of nuclei that were wrongly assigned 2 or 3 spots (4). All the data are outputted in a large data file (5).</p>
</figcaption>
</figure>
</li>
<li>Select the spots icon that contains the identified transcription sites. Select the “statistics” tab and select “detailed.” Export “all statistics to file.” Make sure the filename contains the word “spots.”</li>
<li>Select the spots icon that contains generated background spots. In the “statistics” tab select “detailed.” Export “all statistics to file.” Make sure the filename contains the word “background.”</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> A specific folder structure is needed to run the custom python script: parent folder -&gt; subfolder containing Imaris statistics. The example dataset contains the following structure: Analysis -&gt; Embryo 1 (<a href="#fig6">Figure 6</a>, step 2).</div>
<div class="optional">
<span class="optional-title">Optional:</span> <i>Drosophila</i> embryos undergo rapid mitotic divisions during early development. After nuclei have migrated to the embryo periphery, mitotic divisions occur in a wave (<a href="#bib12">Foe, 1989</a>; <a href="#bib13">Foe and Alberts, 1983</a>). The time differences in transcriptional onset that result from the mitotic wave can be corrected using the following pipeline. The temporal profile of cells is synchronized using the file “key_time.txt” contained within the sample analysis file. In this file, manually note the x-axis positions (Imaris slice view, increasing numbers from left to right) of boundaries that contain a group of similarly timed nuclei (columns 1 and 2) and in column 3 note down the time frame in which nuclei undergo telophase. During subsequent data analysis, time frames are adjusted by the calculated time correction using the telophase frame number within mitotic zones (<a href="#fig5">Figure 5</a>B).</div>
<h3 id="sec2.4">Assigning transcription sites to nuclei</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼10 min/time-lapse dataset</div>
<p><b>Assign transcription sites to nuclei:</b> The custom python script uses the Ellipsoid axis to assign spots to nuclei instead of the centroid position. This helps to assign each transcription site to the correct nucleus while accounting for the embryo curvature (see <a href="#fig5">Figure 5</a>A for illustration).</p>
<ol start="16">
<li>Download the custom python package “sass” (<b>S</b>imple <b>A</b>ssignment of <b>S</b>pots to <b>S</b>urfaces), which links the identified transcription foci to their respective nuclei. The package containing all scripts can be downloaded here: <a href="https://github.com/TMinchington/sass">https://github.com/TMinchington/sass</a>
<ol type="a">
<li>Make sure that python 3 (or Anaconda3) is installed on the computer.</li>
<li>Using the Anaconda Prompt, navigate to the folder that contains the sass.py file (alternatively, drag and drop the sass.py file into the command prompt).</li>
<li>Type: python sass.py [drag and drop the parent folder containing the subfolder with the exported Imaris statistics] and run the analysis (<a href="#fig6">Figure 6</a>, step 2).
<ol type="i">
<li>If the optional step of mitosis correction was skipped, sass.py will show this message before finishing “No key time file detected. Would you like to stop and create this file now? (y/n).” Answer with “n” in the command line.</li>
</ol>
</li>
<li>The output folder is generated in the subfolder containing the Imaris statistics. The folder “time data” contains the finished analysis files.
<ol type="i">
<li>The image mid_line_expression.png shows a summary image of nuclei with active transcription (green), nuclei without transcription (blue) and the midline of the expression domain (<a href="#fig6">Figure 6</a>, step 3).
<div class="note">
<span class="note-title">Note:</span> Nuclei in <a href="#fig6">Figure 6</a>, step 3 are identified as inactive (blue) because either no transcription sites were identified in step 13 or because the nucleus was excluded as it is not present in the field of view during the full transcription period (see the Note below).</div>
</li>
<li>The image assignments.png file shows the percentage of nuclei that were assigned 1 spot (blue line), 2 spots (orange line) or 3 spots (green line) for each time frame (X-axis) (<a href="#fig6">Figure 6</a>, step 4). This graph can be used as a visual aid to assess the segmentation success. Optimally, nuclei should only have been assigned 1 spot (see <a href="#sec6.11">troubleshooting 6</a>).</li>
<li>The file “mid_line.txt” contains the coordinates for the expression domain midline.</li>
<li>The file “backed_down-hc.txt” contains the tracked nuclei and transcription spots (identify by nuclear ID in the column “nuc” and spot ID in the column “spot”) and their 3D positions (columns “nucx/y/z” and “spotsx/y/z”). The nuclear X/Y/Z positions can be used to determine the nuclear position along the embryo’s AP and DV axis. The fluorescence value, background corrected value and the fluorescence intensity can be found in the columns “value,” “corrected_val” and “variable” respectively. Make sure to filter for the MCP-GFP channel in the column “channel” for further analysis. The distance to the midline can be found in the column “dis_mid” and is computed in μm. Imaging time can be found in the column “time” and is calculated in min (<a href="#fig6">Figure 6</a>, step 5). If the optional step of mitosis adjustment was included, the adjusted time can be found in column “time_shift” (<a href="#fig6">Figure 6</a>, step 5).</li>
</ol>
</li>
<li>Using the nuclear ID, full traces of transcriptional activity can be plotted or used for further analysis. If multiple spots are assigned to a single nucleus see <a href="#sec6.11">troubleshooting 6</a>.
<div class="note">
<span class="note-title">Note:</span> The default microscopy time interval setting is 20 s in the python package. This can be overwritten by running the scripts with the following modification: python sass.py <b>--i [time interval in sec]</b> [folder path to analyse]<i>.</i> A time interval of 60 s will result in full integers, as the time interval is displayed in min. For example: python sass.py --i 60 “D:\Carol\Methods paper\Analysis.”</div>
<div class="note">
<span class="note-title">Note:</span> For downstream analysis, we recommend using “Intensity Sum” fluorescence values as these contain the full fluorescence detected in 3D.</div>
<div class="note">
<span class="note-title">Note:</span> The file “backed_down-hc.txt” only contains a subset of nuclei. Contained in this list are nuclei that were present in the first and last time frame during gene transcription. Nuclei that entered or exited the field of view are excluded. Full data can be found in the “backed_down.txt” file.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The quality of the analyzed data should be inspected before continuing with downstream analysis. The assignments.png file in the analysis folder can be used as an indicator for successful nuclear tracking and spot identification (less than 5% of nuclear traces have more than 1 spot assigned). If this value is found to be above 5% this could be due to false-positive spots or gaps in nuclear tracks (see <a href="#sec6.11">troubleshooting 6</a>)<b>.</b> We also recommend plotting the mean fluorescence intensity and transcription onset time of all analyzed nuclei as well as a few individual nuclear traces to visualize obvious problems with the dataset (in comparison with other biological replicates) before progressing with downstream analysis.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Tagging genomic regions of interest (either endogenous loci or transgenes) with MS2 stem-loops results in a bright fluorescent signal representing nascent transcription when imaged using a confocal microscope (<a href="#fig7">Figure 7</a>A). Visualization of the single-cell transcription profiles of the Bone Morphogenetic Protein (BMP) signaling pathway target genes <i>u-shaped</i> (<i>ush</i>) and <i>hindsight</i> identified significant differences in transcription kinetics based on the level of signaling that cells are exposed to (<a href="#bib23">Hoppe et al., 2020</a>). These genes are expressed during nc14 and are important for early embryonic development (<a href="#bib1">Ashe et al., 2000</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/518-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Analysis of 24xMS2-<i>ush</i> transcription profiles in the <i>Drosophila</i> embryo</div>
<p>(A) Still from a cropped time lapse movie maximum projected showing active <i>ush</i> transcription. Transcription is shown by MCP-GFP fluorescence (gray) and nuclei are shown by His-RFP (magenta). The enlarged region (lower panel) shows one active transcription site per nucleus.</p>
<p>(B) Representative transcription traces of <i>ush</i> over developmental time. Nuclear IDs are shown for identification of the traces in the example dataset.</p>
<p>(C) Transcription onset times of <i>ush</i>.</p>
<p>(D) Mean <i>ush</i> fluorescence over time combines the fluorescence values for all nuclei.</p>
<p>(E) Heatmap of single-cell fluorescence- traces, sorted according to transcription onset (scale as indicated).</p>
<p>Scale bar, 15 μm (A, top) and 10 μm (A, bottom). Median (C) and mean ± 95% confidence intervals (D). (C)–(E) contain data from 116 nuclei that are tracked and present in the example dataset.</p>
</figcaption>
</figure>
<p>The example dataset included for this protocol shows <i>ush</i> transcription during nc14 (<a href="#fig7">Figure 7</a>A). Using the semi-automated image analysis pipeline outlined here, the nascent mRNA signal is extracted and single-cell fluorescent traces can be displayed, highlighting differences in transcription between cells (<a href="#fig7">Figure 7</a>B). Additionally, the transcription onset time of individual cells can be determined placing the median onset time of <i>ush</i> transcription at 31 min into nc14 in the example dataset (<a href="#fig7">Figure 7</a>C). As well as analyzing single-cell transcription dynamics, nuclei can be grouped and the mean transcriptional activity of <i>ush</i> determined (<a href="#fig7">Figure 7</a>D). The differences in single-cell transcription profiles and the transcriptional onset time can be visualized using heatmaps to investigate spatial and temporal transcription dynamics (<a href="#fig7">Figure 7</a>E).</p>
<p>The fluorescence emitted by a nascent transcription site is assumed to be directly proportional to the number of mRNA molecules that are produced. Due to this relationship, the fluorescent signal can be converted into the number of elongating Pol II molecules using a conversion based on single molecule fluorescent <i>in situ</i> hybridization (smFISH) data. Descriptions of how the MCP-GFP fluorescence can be converted into Pol II number can be found in these studies (<a href="#bib19">Garcia et al., 2013</a>; <a href="#bib23">Hoppe et al., 2020</a>; <a href="#bib25">Lammers et al., 2020</a>).</p>
<div class="note">
<span class="note-title">Note:</span> We would recommend analyzing at least three biological replicates per genotype and a minimum of 100 nuclear traces per replicate (embryo). To achieve robust modeling results using the memory adjusted Hidden Markov Model (<a href="#bib6">Bowles et al., 2020</a>; <a href="#bib25">Lammers et al., 2020</a>) (see <a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>) we recommend using a minimum of 1,000 data points (for example 50 nuclear traces with a transcription signal in 20 time frames).</div>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>To analyze transcription dynamics in greater detail, a memory adjusted Hidden Markov Model can be used to extract transcriptional burst kinetics (<a href="#bib6">Bowles et al., 2020</a>; <a href="#bib23">Hoppe et al., 2020</a>; <a href="#bib25">Lammers et al., 2020</a>). Using this computational approach, the underlying promoter states corresponding to each time frame in the time-lapse data can be inferred and transcription parameters can be calculated. This approach assumes a two-state promoter model, where the promoter transitions between ON and OFF states with specific rates <i>k</i><sub><i>on</i></sub> and <i>k</i><sub><i>off</i></sub> (<a href="#bib25">Lammers et al., 2020</a>; <a href="#bib41">Zoller et al., 2018</a>).</p>
<p>By calculating transcription parameters, the transcriptional response to, for example, signaling levels can be quantified and compared between genotypes or at different positions within the embryo. A full analysis of how single cells decode the BMP signal through modulation of target gene transcription can be found in <a href="#bib23">Hoppe et al., 2020</a>. Detailed information to implement downstream analysis using the memory adjusted Hidden Markov Model can be found in <a href="#bib25">Lammers et al., 2020</a> and <a href="#bib6">Bowles et al., 2020</a>, which can also be used to infer transcription parameters at single-cell resolution (<a href="#bib6">Bowles et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although the MS2/MCP live imaging system offers a robust tool to image gene transcription, it has some limitations. Firstly, transcription from weak promoters will be hard to detect and quantify. Due to the inherent background of unbound MCP-GFP protein in the nucleus, a weak transcription signal will result in a low signal-to-noise ratio and therefore be hard to detect.</p>
<p>Secondly, most published <i>Drosophila</i> lines expressing the MCP fusion protein use a monomer MCP molecule. However, <i>in vitro</i> experiments showed varying degrees of saturation of the MS2 stem-loops with MCP-GFP, particularly at lower MCP-GFP concentrations (<a href="#bib17">Fusco et al., 2003</a>; <a href="#bib39">Wu et al., 2012</a>). Hence, the amount of nascent fluorescence may not be exactly proportional to the mRNA number produced, as is assumed for analysis. The low occupancy was increased <i>in vitro</i> by expressing MCP as a single chain tandem dimer (<a href="#bib39">Wu et al. 2012</a>). Fly lines using an MCP tandem dimer are predicted to promote uniform binding to MS2 loops even at lower concentrations of the MCP-fluorophore fusion protein. Fly lines are available with NLS-2xMCP-TagRFP-T (<a href="#bib20">Halstead et al., 2015</a>), 2xMCP-2xRFP and 2xMCP-2xmRuby3 (Ashe lab unpublished, available on request from the lead contact).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Embryos slide during the live imaging experiment or show mitotic division arrest.</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Both observations can be caused by problems with the mounting of the embryos onto the heptane glue coated coverslip or if the Lumox imaging dish membrane is too flexible.</p>
<p>If embryos were not attached properly to the heptane glue covered coverslip, they might slide during the live-cell imaging experiment. This could be caused by the heptane glue being too dilute. Add new double-sided tape to the heptane glue bottle. Once the glue has been dissolved, the heptane/glue solution should become more viscous. Alternatively, the embryos were not pressed against the coverslip sufficiently. After dropping the coverslip onto the embryos, make sure that the embryo is flattened against the coverslip by carefully pressing onto the coverslip with tweezers. Sliding or other movement can also be caused if the membrane of the Lumox imaging dish is too flexible. When reusing the imaging dishes, the membrane can become damaged from removing coverslip bridges from the previous experiments. Make sure to not reuse the dishes more than 2–3 times and if in doubt use a fresh dish.</p>
<p>On occasion, we have observed that nuclei can arrest during mitotic cleavage cycles in parts or the full embryo. We have identified one reason causing this arrest to be too much pressure of the coverslip against the embryo. Be careful not to attach the embryo too forcefully to the coverslip.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Photobleaching is observed during the time-lapse experiment.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>If photobleaching is observed, decrease the laser power and/or reduce the sampling frequency.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Unable to distinguish MCP-GFP background fluorescence from real MS2/MCP fluorescent signal or very low MS2/MCP fluorescent signal.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Try to increase the laser power and use very sensitive detectors. Increase the signal-to-noise ratio by using flies that are heterozygous for the MCP-GFP transgene. This reduces the amount of unbound MCP-GFP that is maternally deposited. Inserting the MS2 stem-loops into the 5′ UTR of a gene will maximize the fluorescence as the stem-loops will be bound by MCP-GFP for longer.</p>
<p>Another approach to increase the signal-to-noise ratio is to increase the number of MS2 stem-loop repeats that are inserted into the endogenous gene locus, for example to 128 copies (<a href="#bib37">Tantale et al., 2016</a>). More MS2 stem-loops will bind more fluorophores, which will increase the overall fluorescent signal aiding detection. However, the insertion of more MS2 stem-loops will increase the size of the DNA sequence inserted, potentially leading to artifacts in gene expression regulation.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Dim transcription sites are not detected by Imaris.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>If transcription foci are very dim they might not be identified and need to be added manually. This can be done in the editing tab with the pointer in the “select” mode. Hover over the transcription spot, press shift, and left click to add a sphere. Adjust the sphere size to match the other added spots. Depending on the number of transcription sites that need to be added manually, correction can be very time consuming. If the number of spots that need to be corrected causes a significant increase in analysis time, consider analyzing a smaller region of interest or changing the microscopy settings to enhance the quality of raw data (higher resolution/ higher magnification/ increased laser power).</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Many false-positive spots are detected by Imaris.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If many false-positive spots are detected by the software, try to break up spot detection into smaller time intervals. Assign spots to 5–10 time frames at a time and merge spot files afterwards. Remove remaining false-positive spots manually by highlighting the spot and deleting it in the editing tab. If the time it takes to manually correct the dataset increases the analysis time by hours, consider analyzing a smaller subset of nuclei or changing the microscopy settings (see Problem 4).</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Multiple transcription spots have been assigned to one nucleus.</p>
<h3 id="sec6.12">Potential solution</h3>
<p>There are different reasons for wrongly assigned spots. Firstly, if false-positive spots (that are not transcription spots) were identified during spot creation, they will have been assigned to the closest nucleus, which might show active transcription. False-positive spots can often be identified by a very low fluorescent value (close to background values) recorded for this spot.</p>
<p>Secondly, a mistake could have been made during the segmentation of nuclei. If two nuclei are very close to each other, they can be fused into one during nuclei detection. In the subsequent tracking one nuclear trace would then be missing a time point (where the nucleus was fused to its neighbor). If a transcription site was detected in the time frame with the missing nucleus, it will be wrongly assigned to a different nucleus. Alternatively, if two nuclei come into too close proximity it is possible that a transcription site was wrongly assigned to a neighbor based on a closer proximity in three dimensions.</p>
<p>Finally, wrongly assigned transcription foci can occur at the edge of the time-lapse video. If nuclei are entering or exiting the field of view they might not have been detected during nuclear segmentation as they can appear smaller with less fluorescent intensity when they are partially cut off. If the transcription sites of these nuclei were detected, they will have been wrongly assigned to a neighboring nucleus. In our experience it works best to exclude nuclei and their respective transcription sites that start off on the edge of the field of view.</p>
<p>To check the percentage of nuclei that have 2 or more transcription foci assigned to them, check the assignments.png file in the analysis folder (<a href="#fig6">Figure 6</a>, step 4).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ashe, H.L., Mannervik, M., and Levine, M. (2000). Dpp signaling thresholds in the dorsal ectoderm of the <i>Drosophila</i> embryo. Development <i>127</i>, 3305-3312. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref1">View at publisher</a></p>
<p id="bib2">Berrocal, A., Lammers, N., Garcia, H.G., and Eisen, M.B. (2020). Kinetic sculpting of the seven stripes of the <i>Drosophila</i> even-skipped gene. Elife <i>9</i>, e61635. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref2">View at publisher</a></p>
<p id="bib3">Bertrand, E., Chartrand, P., Schaefer, M., Shenoy, S.M., Singer, R.H., and Long, R.M. (1998). Localization of ASH1 mRNA particles in living yeast. Mol. Cell <i>2</i>, 437-445. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref3">View at publisher</a></p>
<p id="bib4">Bothma, J.P., Garcia, H.G., Esposito, E., Schlissel, G., Gregor, T., and Levine, M. (2014). Dynamic regulation of eve stripe 2 expression reveals transcriptional bursts in living <i>Drosophila</i> embryos. Proc. Natl. Acad. Sci. U S A <i>111</i>, 10598-10603. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref4">View at publisher</a></p>
<p id="bib5">Bothma, J.P., Garcia, H.G., Ng, S., Perry, M.W., Gregor, T., and Levine, M. (2015). Enhancer additivity and non-additivity are determined by enhancer strength in the <i>Drosophila</i> embryo. eLife <i>4</i>. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref5">View at publisher</a></p>
<p id="bib42">Bothma, J.P., Norstad, M.R., Alamos, S., and Garcia, H.G. (2018). LlamaTags: A Versatile Tool to Image Transcription Factor Dynamics in Live Embryos. Cell <i>173</i>, 1810-1822.e16. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/optiudpQxHYcN">View at publisher</a></p>
<p id="bib6">Bowles, J.R., Hoppe, C., Ashe, H.L., and Rattray, M. (2020). Scalable inference of transcription kinetic parameters from MS2 time series data. bioRxiv, 2020.12.04.412049 <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref6">View at publisher</a></p>
<p id="bib7">Chao, J.A., Patskovsky, Y., Almo, S.C., and Singer, R.H. (2008). Structural basis for the coevolution of a viral RNA-protein complex. Nat. Struct. Mol. Biol. <i>15</i>, 103-105. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref7">View at publisher</a></p>
<p id="bib9">Dufourt, J., Trullo, A., Hunter, J., Fernandez, C., Lazaro, J., Dejean, M., Morales, L., Nait-Amer, S., Schulz, K.N., Harrison, M.M., et al. (2018). Temporal control of gene expression by the pioneer factor Zelda through transient interactions in hubs. Nat. Commun. <i>9</i>, 5194. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref9">View at publisher</a></p>
<p id="bib10">Falo-Sanjuan, J., Lammers, N.C., Garcia, H.G., and Bray, S.J. (2019). Enhancer priming enables fast and sustained transcriptional responses to notch signaling. Dev. Cell <i>50</i>, 411-425.e8. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref10">View at publisher</a></p>
<p id="bib11">Ferraro, T., Esposito, E., Mancini, L., Ng, S., Lucas, T., Coppey, M., Dostatni, N., Walczak, A.M., Levine, M., and Lagha, M. (2016). Transcriptional memory in the <i>Drosophila</i> embryo. Curr. Biol. <i>26</i>, 212-218. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref11">View at publisher</a></p>
<p id="bib12">Foe, V.E. (1989). Mitotic domains reveal early commitment of cells in <i>Drosophila</i> embryos. Development <i>107</i>, 1-22. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref12">View at publisher</a></p>
<p id="bib13">Foe, V.E. and Alberts, B.M. (1983). Studies of nuclear and cytoplasmic behaviour during the five mitotic cycles that precede gastrulation in <i>Drosophila</i> embryogenesis. J. Cell Sci. <i>61</i>, 31-70. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref13">View at publisher</a></p>
<p id="bib14">Fukaya, T. (2020). Dynamic regulation of anterior-posterior patterning genes in living <i>Drosophila</i> embryos. bioRxiv. <a class="external-link" href="https://doi.org/10.1101/2020.11.25.395277">https://doi.org/10.1101/2020.11.25.395277</a></p>
<p id="bib15">Fukaya, T., Lim, B., and Levine, M. (2016). Enhancer control of transcriptional bursting. Cell <i>166</i>, 358-368. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref15">View at publisher</a></p>
<p id="bib16">Fukaya, T., Lim, B., and Levine, M. (2017). Rapid rates of Pol II elongation in the <i>Drosophila</i> embryo. Curr. Biol. <i>27</i>, 1387-1391. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref16">View at publisher</a></p>
<p id="bib17">Fusco, D., Accornero, N., Lavoie, B., Shenoy, S.M., Blanchard, J.M., Singer, R.H., and Bertrand, E. (2003). Single mRNA molecules demonstrate probabilistic movement in living mammalian cells. Curr. Biol. <i>13</i>, 161-167. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref17">View at publisher</a></p>
<p id="bib18">Garcia, H.G. and Gregor, T. (2018). Live imaging of mRNA synthesis in <i>Drosophila</i>. Methods Mol. Biol. <i>1649</i>, 349-357. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref18">View at publisher</a></p>
<p id="bib19">Garcia, H.G., Tikhonov, M., Lin, A., and Gregor, T. (2013). Quantitative imaging of transcription in living <i>Drosophila</i> embryos links polymerase activity to patterning. Curr. Biol. <i>23</i>, 2140-2145. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref19">View at publisher</a></p>
<p id="bib20">Halstead, J.M., Lionnet, T., Wilbertz, J.H., Wippich, F., Ephrussi, A., Singer, R.H., and Chao, J.A. (2015). An RNA biosensor for imaging the first round of translation from single cells to living animals. Science <i>347</i>, 1367-1671. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref20">View at publisher</a></p>
<p id="bib21">Hayashi, R., Wainwright, S.M., Liddell, S.J., Pinchin, S.M., Horswell, S., and Ish-Horowicz, D. (2014). A genetic screen based on in vivo RNA imaging reveals centrosome-independent mechanisms for localizing gurken transcripts in <i>Drosophila</i>. G3 (Bethesda) <i>4</i>, 749-760. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref21">View at publisher</a></p>
<p id="bib22">Hoppe, C. and Ashe, H.L. (2021). CRISPR-Cas9 strategies to insert MS2 stem-loops into endogenous loci in Drosophila embryos. STAR Protocols. <a class="external-link" href="https://doi.org/10.1016/j.xpro.2021.100380">https://doi.org/10.1016/j.xpro.2021.100380</a></p>
<p id="bib23">Hoppe, C., Bowles, J.R., Minchington, T.G., Sutcliffe, C., Upadhyai, P., Rattray, M., and Ashe, H.L. (2020). Modulation of the promoter activation rate dictates the transcriptional response to graded BMP signaling levels in the <i>Drosophila</i> embryo. Dev. Cell <i>54</i>, 727-741. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref23">View at publisher</a></p>
<p id="bib24">Koromila, T. and Stathopoulos, A. (2019). Distinct roles of broadly expressed repressors support dynamic enhancer action and change in time. Cell Rep. <i>28</i>, 855-863.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref24">View at publisher</a></p>
<p id="bib25">Lammers, N.C., Galstyan, V., Reimer, A., Medin, S.A., Wiggins, C.H., and Garcia, H.G. (2020). Multimodal transcriptional control of pattern formation in embryonic development. Proc. Natl. Acad. Sci. U S A <i>117</i>, 836-847. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref25">View at publisher</a></p>
<p id="bib27">Lee, C.H., Shin, H., and Kimble, J. (2019). Dynamics of notch-dependent transcriptional bursting in its native context. Dev. Cell <i>50</i>, 435. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref27">View at publisher</a></p>
<p id="bib28">Lim, B., Heist, T., Levine, M., and Fukaya, T. (2018a). Visualization of transvection in living <i>Drosophila</i> embryos. Mol. Cell <i>70</i>, 287-296.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref28">View at publisher</a></p>
<p id="bib29">Lim, B., Fukaya, T., Heist, T., and Levine, M. (2018b). Temporal dynamics of pair-rule stripes in living <i>Drosophila</i> embryos. Proc. Natl. Acad. Sci. U S A <i>115</i>, 8376-8381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref29">View at publisher</a></p>
<p id="bib30">Lionnet, T., Czaplinski, K., Darzacq, X., Shav-Tal, Y., Wells, A.L., Chao, J.A., Park, H.Y., de Turris, V., Lopez-Jones, M., and Singer, R.H. (2011). A transgenic mouse for in vivo detection of endogenous labeled mRNA. Nat. Methods <i>8</i>, 165-170. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref30">View at publisher</a></p>
<p id="bib43">Liu, J., Hansen, D., Eck, E., Kim, Y.J., Turner, M., Alamos, S., and Garcia, H.G. (2020). Single-cell characterization of the eukaryotic transcription cycle using live imaging and statistical inference. BioRxiv, 08.29.273474 <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref43">View at publisher</a></p>
<p id="bib31">Lucas, T., Ferraro, T., Roelens, B., De Las Heras Chanes, J., Walczak, A.M., Coppey, M., and Dostatni, N. (2013). Live imaging of bicoid-dependent transcription in <i>Drosophila</i> embryos. Curr. Biol. <i>23</i>, 2135-2139. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref31">View at publisher</a></p>
<p id="bib32">Lucas, T., Tran, H., Perez Romero, C.A., Guillou, A., Fradin, C., Coppey, M., Walczak, A.M., and Dostatni, N. (2018). 3 minutes to precisely measure morphogen concentration. PLoS Genet. <i>14</i>, e1007676. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref32">View at publisher</a></p>
<p id="bib33">Markow, T.A., Beall, S., and Matzkin, L.M. (2009). Egg size, embryonic development time and ovoviviparity in <i>Drosophila</i> species. J. Evol. Biol. <i>22</i>, 430-434. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref33">View at publisher</a></p>
<p id="bib35">Pichon, X., Lagha, M., Mueller, F., and Bertrand, E. (2018). A growing toolbox to image gene expression in single cells: sensitive approaches for demanding challenges. Mol. Cell <i>71</i>, 468-480. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref35">View at publisher</a></p>
<p id="bib36">Scholes, C., Biette, K.M., Harden, T.T., and DePace, A.H. (2019). Signal integration by shadow enhancers and enhancer duplications varies across the <i>Drosophila</i> embryo. Cell Rep. <i>26</i>, 2407-2418.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref36">View at publisher</a></p>
<p id="bib37">Tantale, K., Mueller, F., Kozulic-Pirher, A., Lesne, A., Victor, J.-M., Robert, M.-C., Capozi, S., Chouaib, R., Bäcker, V., Mateos-Langerak, J., et al. (2016). A single-molecule view of transcription reveals convoys of RNA polymerases and multi-scale bursting. Nat. Commun. <i>7</i>, 12248. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref37">View at publisher</a></p>
<p id="bib38">Trullo, A., Dufourt, J., and Lagha, M. (2020). MitoTrack, a user-friendly semi-automatic software for lineage tracking in living embryos. Bioinformatics <i>36</i>, 1300-1302. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref38">View at publisher</a></p>
<p id="bib39">Wu, B., Chao, J.A., and Singer, R.H. (2012). Fluorescence fluctuation spectroscopy enables quantitative imaging of single mRNAs in living cells. Biophys. J. <i>102</i>, 2936-2944. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref39">View at publisher</a></p>
<p id="bib40">Yamada, S., Whitney, P.H., Huang, S.-K., Eck, E.C., Garcia, H.G., and Rushlow, C.A. (2019). The <i>Drosophila</i> pioneer factor Zelda modulates the nuclear microenvironment of a dorsal target enhancer to potentiate transcriptional output. Curr. Biol. <i>29</i>, 1387-1393.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref40">View at publisher</a></p>
<p id="bib41">Zoller, B., Little, S.C., and Gregor, T. (2018). Diverse spatial expression patterns emerge from unified kinetics of transcriptional bursting. Cell <i>175</i>, 835-847.e5. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00086-1/sref41">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Hilary L. Ashe (<a href="mailto:Hilary.Ashe@manchester.ac.uk">hilary.ashe@manchester.ac.uk</a>).</p>
<h4>Materials Availability</h4>
<p>The His-RFP; MCP-GFP fly line used in this protocol is available from the Bloomington <i>Drosophila</i> Stock Center and the <i>24xMS2-ush</i> line is available upon request from the lead contact.</p>
<h4>Data and Code Availability</h4>
<p>Two example datasets, a beginner and full dataset, containing Imaris files and image analysis files are deposited at Mendeley Data https://doi.org/10.17632/jv9fjpn9cg.1. The custom analysis code “sass” that links Imaris output files is available at GitHub (<a href="https://github.com/TMinchington/sass">https://github.com/TMinchington/sass</a>, RRID:SCR_018797). The modeling software mentioned in the “<a href="#quantification-and-statistical-analysis">quantification and statistical analysis</a>” section is described in <a href="#bib6">Bowles et al. (2020)</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank Catherine Sutcliffe and Lauren Forbes-Beadle for helpful discussions, Sophie Frampton and Sanjai Patel for their help in making the methods movie, Egor Zindy for his Imaris X-tension, and Steven Marsden for measuring the laser power. Figures were partially created using <a href="http://BioRender.com">BioRender.com</a>. This work was supported by a <a href="https://doi.org/10.13039/100004440">Wellcome Trust</a> Investigator award to H.L.A. (204832/Z/16/Z).</p>
<h3>Author Contributions</h3>
<p>Methodology, C.H. and H.L.A.; Investigation, C.H.; Writing, C.H. and H.L.A.; Funding Acquisition, H.L.A.; Supervision, H.L.A.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>