<article>
<section>
<h1 id="header">A low-cost and open-source protocol to produce key enzymes for molecular detection assays</h1>
<p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
<p>Gabriel Mendoza-Rojas,<sup><a href="#aff1">1</a></sup> Vanessa Sarabia-Vega,<sup><a href="#aff1">1</a></sup> Ana Sanchez-Castro,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Lesia Tello,<sup><a href="#aff1">1</a></sup> Luis Cabrera-Sosa,<sup><a href="#aff1">1</a></sup> Jose A. Nakamoto,<sup><a href="#aff1">1</a></sup> Katherin Peñaranda,<sup><a href="#aff1">1</a></sup> Vanessa Adaui,<sup><a href="#aff1">1</a></sup> Roberto Alcántara,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> and Pohl Milón<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Centre for Research and Innovation, Faculty of Health Sciences, Universidad Peruana de Ciencias Aplicadas (UPC), Lima 15023, Peru</p>
<p id="aff2"><sup>2</sup>Postgraduate Unit, Medicine Faculty, Universidad Nacional Mayor de San Marcos, Lima 15001, Peru</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:pcmeralc@upc.edu.pe">pcmeralc@upc.edu.pe</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:pmilon@upc.edu.pe">pmilon@upc.edu.pe</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100899">10.1016/j.xpro.2021.100899</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe a detailed step-by-step protocol for the expression, purification, quantification, and activity determination of key enzymes for molecular detection of pathogens. Based on previous reports, we optimized the protocol for LbCas12a, Taq DNA polymerase, M-MLV reverse transcriptase, and TEV protease to make it compatible with minimal laboratory equipment, broadly available in low- and middle-income countries. The enzymes produced with this protocol have been successfully used for molecular detection applications.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.crmeth.2021.100093">Alcántara et al. (2021a</a>, <a class="external-link" href="https://doi.org/10.1016/j.xpro.2021.100878">2021b)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Open-source and low-cost production schemes of Taq, M- MLV RT, and Cas12a enzymes</li>
<li>Compatible with basic equipment to produce enzymes for molecular detection</li>
<li>Enzyme preparations validated for SARS-CoV-2 detection</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This segment of the protocol describes the methodology to obtain enzyme-expressing <i>E. coli</i> for key recombinant proteins for use in molecular detection of pathogens. To obtain competent cells use your technique of choice. Here, the Mix &amp; Go <i>E.coli</i> Transformation kit (Zymo Research, Cat. No.: T3002) was used. The <i>E. coli</i> BL21(DE3) strain was used for protein expression.</p>
<div class="note">
<span class="note-title">Note:</span> For the composition and recipes of buffers and media, see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>.</div>
<h3 id="sec1.1">Obtaining enzyme-expressing <i>E. coli</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> three days</div>
<div class="note">
<span class="note-title">Note:</span> pRK793 plasmid encoding TEV protease in <i>E.coli</i> BL21(DE3) pRIL cells is commercially available (Addgene, Cat. No.: 8827) (<a href="#bib9">Kapust et al., 2001</a>); thus, the transformation step can be omitted.</div>
<ol>
<li>Media preparation<br> The following media volumes are required to obtain enzyme-expressing <i>E. coli</i>, including the experimental assay (i.e., transformation with the plasmid of interest), positive and negative controls. Prepare Super Optimal Broth (SOB) and Super Optimal broth with Catabolite repression (SOC) media for bacterial transformation, and Luria-Bertani (LB) medium for bacterial growth under antibiotic selection (See <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a> for media recipes).
<ol type="a">
<li>SOB medium: Prepare a volume of 50 mL. Dispense and sterilize 10 mL aliquots. Store at 4°C for no more than one month.</li>
<li>SOC medium: Prepare it fresh for the transformation process. Do not store. Add 400 μL of 10% glucose to 9.6 mL of SOB medium, to get a final concentration of 0.4% glucose.</li>
<li>LB medium:
<ol type="i">
<li>Taq and RT: Prepare 250 mL LB agar with 30 μg/mL Kanamycin (Kan) + 17 μg/mL Chloramphenicol (Cam) for 10 plates.</li>
<li>Cas12a: Prepare 150 mL LB agar with 100 μg/mL Ampicillin (Amp) for 6 plates.</li>
<li>Tev: Prepare 150 mL LB agar with 100 μg/mL Amp + 30 μg/mL Cam for 6 plates.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Bacterial transformation Here we used the Mix &amp; Go <i>E.coli</i> Transformation kit (Zymo Research, Cat. No. T3002) which allows the preparation of chemically competent cells and highly efficient DNA transformation with a brief heat shock at 37°C (i.e., at the SOC addition step, as detailed below). However, other heat shock transformation alternatives as well as electroporation can be performed instead.
<div class="note">
<span class="note-title">Note:</span> Pre-warm SOC medium for 1 hour at 37°C before use.</div>
<ol type="a">
<li>Transform 100 μL of competent <i>E. coli</i> BL21(DE3) cells. Use with 10 ng of the corresponding plasmid DNA (try to keep volume of DNA less than 5% of the total volume).</li>
<li>Mix gently and incubate on ice for 5–10 min.</li>
<li>Add four volumes of pre-warmed (at 37°C for about 1 h) SOC medium (400 μL of SOC for 100 μL of competent cells).</li>
<li>Incubate at 37°C for 1 h with gentle mixing (200–300 RPM) to let transformed cells recover from the treatment and begin to express the plasmid-encoded antibiotic resistance gene (<a href="#fig1">Figure 1</a>).
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> During the one-hour incubation in step (d), pre-warm the LB agar plates with the corresponding antibiotics for 1 hour at 37°C.</div>
</li>
<li>Plate 50–100 μL of each transformation tube in the pre-warmed LB agar plates (at 37°C for 1 h) with the corresponding antibiotics. Use an L-shaped loop to spread evenly (<a href="#fig1">Figure 1</a>).</li>
<li>Incubate at 37°C for 14–16 h. If just a few or none transformed colonies did grow on the LB plate, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.1">problems 1</a> and <a href="#sec6.3">2</a>.
<div class="note">
<span class="note-title">Note:</span> Maintain sterility throughout the plating process. The working surface and the L-shaped loop can be both sterilized with 70% ethanol or using a Bunsen burner. Disposable sterile L-shaped loops can also be used.</div>
<div class="note">
<span class="note-title">Note:</span> Include the following negative controls: empty plate (plate control for contamination), untransformed cells (transformation control), and untransformed cells in plates without the selection antibiotic (cell viability control). If available, use a positive control for transformation with any available plasmid carrying the corresponding resistance marker gene.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Many cells are provided with plasmids which could be used as positive controls for transformation.</div>
<div class="note">
<span class="note-title">Note:</span> Isolated colonies must be observed, otherwise see the <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.1">problems 1</a> and <a href="#sec6.3">2</a>.</div>
</li>
<li>Select three to five 2–4 mm well-isolated bacterial colonies to test and mark them on the plate using a permanent marker.</li>
<li>To confirm transformation, perform colony PCR with specific primers for the plasmid or gene of interest. Pick one-tenth of each selected colony with a sterile pipette tip and dip the tip into the PCR reaction mixture (<a href="#fig1">Figure 1</a>). Include a positive control using a plasmid dilution (5 ng plasmid DNA per PCR reaction).
<div class="note">
<span class="note-title">Note:</span> Taking more cells can result in inhibition of the PCR reaction.</div>
<div class="note">
<span class="note-title">Note:</span> In this protocol, universal primers for the T7 promoter and terminator (p<i>olA</i> gene, <i>pol</i> gene and <i>cas12a</i> gene), as well as 5TpET28 and M13F primer (p<i>1</i> gene) are used for amplification.</div>
Set of primers used for colony PCR:
<table id="undtbl1">
<thead>
<tr>
<th>Primer</th>
<th>Sequence (5’ – 3′)</th>
<th>Annealing temperature (<sup>o</sup>C)</th>
</tr>
</thead>
<tbody>
<tr>
<td>T7 forward</td>
<td>TAATACGACTCACTATAGG</td>
<td rowspan="2">55</td>
</tr>
<tr>
<td>T7 terminator</td>
<td>GCTAGTTATTGCTCAGCGG</td>
</tr>
<tr>
<td>5TpET28</td>
<td>GGAATTGTGAGCGGATAAC</td>
<td rowspan="2">55</td>
</tr>
<tr>
<td>M13F</td>
<td>GTAAAACGACGGCCAG</td>
</tr>
</tbody>
</table>
</li>
<li>Run the following PCR thermocycling program:
<table id="undtbl2">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature (°C)</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>95</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95</td>
<td>30 s</td>
<td rowspan="3">20</td>
</tr>
<tr>
<td>Annealing</td>
<td>55</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>-- min∗</td>
</tr>
<tr>
<td>Final extension</td>
<td>72</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>10</td>
<td colspan="2">∞</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗Extension time varies depending on the indicated gene:<br> <i>polA</i> gene (Taq DNA polymerase): 3 min<br> <i>pol</i> gene (M-MLV Reverse transcriptase): 3 min<br> <i>cas12a</i> gene (LbCas12a): 5 min 30 s<br> <i>p1</i> gene (TEV protease): 2 min 30 s</p>
</li>
<li>Analyze the PCR products by running 5 μL of each PCR sample on a 1% (w/v) agarose gel. Here we used 1 μL of a 6X stock of SafeGreen for visualization of DNA bands. The expected band sizes are <i>polA</i> 2775 bp, <i>pol</i> 2255 bp, <i>cas12a</i> 5200 bp, and <i>p1</i> 2079 bp. A GeneRuler™ 1 kb DNA ladder was used (ThermoFisher) (<a href="#fig1">Figure 1</a>).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Any other low-mutagenic equivalent dyes, such as SYBR®Safe, EvaGreen® or non-hazardous non-cytotoxic like GelRed® or GelGreen® can also be used. Alternatively, stain with Ethidium Bromide (EtBr) (0.5 μg/ml final concentration on the agarose gel).</div>
</li>
<li>Spread one verified colony in the respective agar LB agar plate with antibiotics. Incubate at 37°C for 14–16 h. Store the plates at 4°C for back-up until the end of the protein expression workflow.
<div class="note">
<span class="note-title">Note:</span> Pick an isolated colony from (k) and start an overnight culture in 50 mL LB with the corresponding antibiotic and incubate at 37°C for 14–16 hours at 150 RPM. From this overnight culture prepare 1500 μL 50% glycerol stocks in 2 mL cryovials (750 μL culture/ 750 μL sterile 100% glycerol). Store at −80°C (<a href="#fig1">Figure 1</a>).</div>
</li>
</ol>
</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Transformation and screening of bacterial colonies for protein expression in <i>E. coli</i>
</div>
<p>The scheme summarizes all steps to obtain and verify by colony PCR the transformed BL21 <i>E. coli</i> cells that express the key molecular biology enzymes Taq DNA Polymerase, M-MLV Reverse Transcriptase, and LbCas12a. Additionally, BL21 cells obtained from Addgene and carrying the plasmid encoding the TEV protease were checked by colony PCR. The TEV protease is required to cleave LbCas12a from the expressing fusion protein MBP-LbCas12a. Ab: antibiotic.</p>
</figcaption>
</figure>
<h3 id="sec1.2">General experimental considerations</h3>
<ol start="3">
<li>Consider using fresh or non-long term stored competent <i>E. coli</i> cells. The BL21(DE3)pLysS strain was used in the described protocol. However, other <i>E. coli</i> strains could be evaluated if they are compatible with the expression plasmids.</li>
<li>It is important to maintain the cold chain over the course of recombinant enzyme production. If a cold room is available, the chromatography and dialysis steps should be performed in this facility. Alternatively, protein fractions must be kept as much as possible on ice.</li>
<li>It is highly recommended to perform the cell lysis, chromatography, and dialysis steps, on the same day. Avoid freezing unpurified protein fractions.</li>
<li>Protein expression could be performed with your lab standardized conditions (i.e., temperature, induction time, or IPTG inducer concentration). We recommend performing an analytical/pilot assay before a preparative one to evaluate protein expression performance.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>E.coli</i> BL21(DE3)pLysS</td>
<td>Promega</td>
<td>Cat # L1195</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Calbiochem</td>
<td>Cat # 171254-5GM</td>
</tr>
<tr>
<td>Chloramphenicol</td>
<td>Sigma-Aldrich</td>
<td>Cat # C0378-25G</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Sigma-Aldrich</td>
<td>Cat # K1876-25G</td>
</tr>
<tr>
<td>Isopropyl ß-D-1 thiogalactopyranoside</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # R0392</td>
</tr>
<tr>
<td>HEPES</td>
<td>Merck Millipore</td>
<td>Cat # 391338-1KG</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Liofilchem</td>
<td>Cat # 611004</td>
</tr>
<tr>
<td>Yeast extract</td>
<td>Liofilchem</td>
<td>Cat # 611005</td>
</tr>
<tr>
<td>Glucose</td>
<td>Merck Millipore</td>
<td>Cat # 1083371000</td>
</tr>
<tr>
<td>Ammonium acetate</td>
<td>Merck Millipore</td>
<td>Cat # 101116</td>
</tr>
<tr>
<td>Potassium chloride</td>
<td>Merck Millipore</td>
<td>Cat # 104936</td>
</tr>
<tr>
<td>Magnesium chloride hexahydrate</td>
<td>Merck Millipore</td>
<td>Cat # 105833</td>
</tr>
<tr>
<td>Magnesium sulfate</td>
<td>Fisher Scientific</td>
<td>Cat # M63-500</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>Sigma-Aldrich</td>
<td>Cat # 2830-OP</td>
</tr>
<tr>
<td>Coomassie Brilliant blue R250</td>
<td>Sigma-Aldrich</td>
<td>Cat # 1125530025</td>
</tr>
<tr>
<td>Absolute ethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat # 1009832500</td>
</tr>
<tr>
<td>Glacial acetic acid</td>
<td>Sigma-Aldrich</td>
<td>Cat # 1000631000</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma-Aldrich</td>
<td>Cat # P9416-50ML</td>
</tr>
<tr>
<td>NP-40</td>
<td>Merck Millipore</td>
<td>Cat # 492016-100ML</td>
</tr>
<tr>
<td>TEMED</td>
<td>Merck Millipore</td>
<td>Cat # 8930-100ML</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat # T8787-50ML</td>
</tr>
<tr>
<td>Maltose monohydrate</td>
<td>LOBA Chemie</td>
<td>Cat # 04495</td>
</tr>
<tr>
<td>ß-mercaptoethanol</td>
<td>Merck Millipore</td>
<td>Cat # 444203</td>
</tr>
<tr>
<td>Lysozyme</td>
<td>PanReac Applichem</td>
<td>Cat # A3711</td>
</tr>
<tr>
<td>Trehalose</td>
<td>Sigma-Aldrich</td>
<td>Cat # T9449-25G</td>
</tr>
<tr>
<td>cOmplete® Protease Inhibitor Cocktail</td>
<td>Roche</td>
<td>Cat # 11697498001</td>
</tr>
<tr>
<td>Sodium chloride</td>
<td>Merck Millipore</td>
<td>Cat # 106404</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Sigma-Aldrich</td>
<td>Cat # I0125</td>
</tr>
<tr>
<td>Glycerol 85%</td>
<td>Sigma-Aldrich</td>
<td>Cat # 104094</td>
</tr>
<tr>
<td>EDTA</td>
<td>Merck Millipore</td>
<td>Cat # 324503</td>
</tr>
<tr>
<td>SureCast™ Ammonium Persulfate (APS)</td>
<td>Invitrogen</td>
<td>Cat # HC2005</td>
</tr>
<tr>
<td>Dodecyl sulfate sodium salt</td>
<td>Merck Millipore</td>
<td>Cat # 1137601000</td>
</tr>
<tr>
<td>Tris, Hydrochloride, Molecular Biology Grade</td>
<td>Merck Millipore</td>
<td>Cat # US1648317</td>
</tr>
<tr>
<td>Tris base ULTROL grade</td>
<td>Calbiochem</td>
<td>Cat # 643811</td>
</tr>
<tr>
<td>Dithiothreitol (DTT)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # R0682</td>
</tr>
<tr>
<td>dNTP Mix (10 mM each)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # R0194</td>
</tr>
<tr>
<td>SafeGreen</td>
<td>abm</td>
<td>Cat # G108-G</td>
</tr>
<tr>
<td>TriTrack DNA loading dye 6X</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # R1161</td>
</tr>
<tr>
<td>Agarose powder</td>
<td>Cleaver Scientific</td>
<td>Cat # CSL-AG500</td>
</tr>
<tr>
<td>Polyacrylamide (40% acrylamide and bis-acrylamide)</td>
<td>Merck Millipore</td>
<td>Cat # 1006381000</td>
</tr>
<tr>
<td>OmniPur Water DEPC treated, sterile, nuclease-free</td>
<td>Sigma-Aldrich</td>
<td>Cat # 9610-1L</td>
</tr>
<tr>
<td>DNaseI</td>
<td>PanReac Applichem</td>
<td>Cat # A3778</td>
</tr>
<tr>
<td>Unstained Protein Standard, Broad Range (10–200 kDa) ladder</td>
<td>New England Biolabs</td>
<td>Cat # P7717S</td>
</tr>
<tr>
<td>AmpliSize Molecular Ruler</td>
<td>Biorad</td>
<td>Cat # 1708200</td>
</tr>
<tr>
<td>GeneRuler 1 kb DNA Ladder</td>
<td>Thermo Scientific</td>
<td>Cat # SM0311</td>
</tr>
<tr>
<td>GeneRuler 100 bp DNA Ladder</td>
<td>Thermo Scientific</td>
<td>Cat # SM0242</td>
</tr>
<tr>
<td>Unstained Protein Molecular Weight Marker (14.4–116 kDa) ladder</td>
<td>Thermo Scientific</td>
<td>Cat # 26610</td>
</tr>
<tr>
<td>Bovine Serum Albumin, lyophilized powder</td>
<td>Sigma-Aldrich</td>
<td>Cat # A9418-100G</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>BSA protein standard</td>
<td>Sigma-Aldrich</td>
<td>Cat # P0834</td>
</tr>
<tr>
<td>RevertAid First Strand cDNA Synthesis kit</td>
<td>Thermo Scientific</td>
<td>Cat # K1622</td>
</tr>
<tr>
<td>Standard Taq Buffer</td>
<td>New England Biolabs</td>
<td>Cat # M0273S</td>
</tr>
<tr>
<td>NEBuffer™ r2.1 (10X)</td>
<td>New England Biolabs</td>
<td>Cat # B6002S</td>
</tr>
<tr>
<td>Mix&amp;Go! <i>E. coli</i> transformation kit</td>
<td>Zymo Research</td>
<td>Cat # T3002</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>ssDNA oligo: T7 promoter forward primer TAATACGACTCACTATAGG</td>
<td>Commercial</td>
<td>N/A</td>
</tr>
<tr>
<td>ssDNA oligo: T7 terminator reverse primer GCTAGTTATTGCTCAGCGG</td>
<td>Commercial</td>
<td>N/A</td>
</tr>
<tr>
<td>ssDNA oligo: M13F forward primer GTAAAACGACGGCCAG</td>
<td>Commercial</td>
<td>N/A</td>
</tr>
<tr>
<td>ssDNA oligo: 5TpET28 reverse primer GGAATTGTGAGCGGATAAC</td>
<td>Commercial</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pRK793 plasmid</td>
<td><a href="#bib9">Kapust et al., 2001</a></td>
<td>Addgene plasmid # 8827</td>
</tr>
<tr>
<td>pMBP-LbCas12a Plasmid</td>
<td><a href="#bib5">Chen et al., 2018</a></td>
<td>Addgene plasmid # 113431</td>
</tr>
<tr>
<td>pET-28a_6H-MMLV_RT_D524N_-6H</td>
<td><a href="#bib7">Graham et al., 2021</a></td>
<td><a href="https://gitlab.com/tjian-darzacq-lab/bearmix">https://gitlab.com/tjian-darzacq-lab/bearmix</a></td>
</tr>
<tr>
<td>pET-28a_6H-TAQ_E602D</td>
<td><a href="#bib7">Graham et al., 2021</a></td>
<td><a href="https://gitlab.com/tjian-darzacq-lab/bearmix">https://gitlab.com/tjian-darzacq-lab/bearmix</a></td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ImageJ</td>
<td>(<a href="#bib19">Schneider et al., 2012</a>)</td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>GraphPad Prism version 9</td>
<td>GraphPad Software, La Jolla, CA</td>
<td><a href="http://www.graphpad.com">www.graphpad.com</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Membrane filter 0.45 μM pore size</td>
<td>Merck</td>
<td>Cat # 10418024</td>
</tr>
<tr>
<td>HisTrap FF Crude</td>
<td>Cytiva</td>
<td>Cat # 11000458</td>
</tr>
<tr>
<td>HiTrap SP Sepharose FF</td>
<td>Cytiva</td>
<td>Cat # 17505401</td>
</tr>
<tr>
<td>HiTrap Capto SP ImpRes</td>
<td>Cytiva</td>
<td>Cat # 17546851</td>
</tr>
<tr>
<td>MBPTrap HP</td>
<td>Cytiva</td>
<td>Cat # 28918778</td>
</tr>
<tr>
<td>D-Tube<sup>TM</sup> Dialyzer Maxi, MWCO 12–14 kDa</td>
<td>Merck</td>
<td>Cat # 71510</td>
</tr>
<tr>
<td>96-well flat clear bottom black polystyrene TC-treated microplates</td>
<td>Corning®</td>
<td>Cat # 3603</td>
</tr>
<tr>
<td>NanoDrop One microvolume UV-Vis spectrophotometer</td>
<td>Thermo Fisher</td>
<td>Cat # 840-329700</td>
</tr>
<tr>
<td>safeVIEW: LED/Blue Light Transilluminator</td>
<td>Cleaver Scientific</td>
<td>Cat # SAFEVIEW</td>
</tr>
<tr>
<td>Thermal Cycler T100</td>
<td>BioRad</td>
<td>Cat # 1861096</td>
</tr>
<tr>
<td>AkTA<sup>TM</sup> Start</td>
<td>Cytiva</td>
<td>N/A</td>
</tr>
<tr>
<td>Cytation<sup>TM</sup> 5 Cell Imaging Multi-Mode Reader</td>
<td>BioTek Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>SynergyTM H1 Hybrid Multi-Mode Reader</td>
<td>BioTek Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Supplemental data</td>
<td>Mendeley Data</td>
<td><a href="https://doi.org/10.17632/wynyf35xf3.1">https://doi.org/10.17632/wynyf35xf3.1</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Obtaining enzyme-expressing <i>E. coli</i>
</h3>
<p>In this protocol expendable materials including microcentrifuge tubes, culture plates, glassware for media preparation, and pipette tips are not included in the <a href="#key-resources-table">key resources table</a>, as these materials must be available in the laboratory. Furthermore, basic equipment of a microbiology and biochemistry laboratory is needed, including an autoclave, bacteria culture incubator and water bath, conventional benchtop microcentrifuge and minicentrifuges, heating block, power source and agarose and SDS-PAGE electrophoresis equipment.</p>
<p>For colony PCR, DNA Taq polymerase, and M-MLV reverse transcriptase enzymatic activity assessment, we used a PCR thermocycler from Bio-Rad.. Alternatively, any PCR thermocycler can be used to corroborate enzymatic activity. For visualization of the PCR products we used SafeGreen dye and a blue light transilluminator (Cleaver Scientific). Other equivalent dyes, such as SYBR®Safe, EvaGreen®, GelRed®, GelGreen® or staining by Ethidium Bromide can also be used. Images from DNA gels, as from protein gels, were obtained using a gel documentation system (Cleaver Scientific), coupled to a digital camera or smartphone, and analyzed with ImageJ.</p>
<h3 id="sec2.2">Expression in <i>E. coli</i> and protein purification</h3>
<p>For collecting cells after protein expression, we used a preparative refrigerated centrifuge (Awel) using 50 mL conical tubes. Alternatively, tubes or bottles for larger volumes can be used depending on rotor availability (i.e., Sorvall or Beckman systems). Protein purification was performed using a Fast Protein Liquid Chromatography (FPLC) system (Amersham). Additionally, a manual procedure using disposable syringes of 10 mL is described.</p>
<p>For protein quantification, we used a Nanodrop One spectrophotometer (Thermofisher) and a SynergyTM H1 Hybrid Multi-Mode plate reader (Biotek) for Bradford analysis. The latter was also used for corroborating Cas12a enzymatic activity in the fluorescence detection assay optimized and reported on (<a href="#bib1">Alcántara et al., 2021a</a>, <a href="#bib2">2021b</a>). Details on bacteria strains used are described within the adequate step and in the <a href="#key-resources-table">key resources table</a>. All reagents and their corresponding identifiers used in this protocol, as the alternatives, have been included in the <a href="#key-resources-table">key resources table</a>. All media preparation instructions for culture and the required buffer recipes for protein purification are described in the <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>The present work describes adapted protocols for the production of LbCas12a (henceforth Cas12a) (<a href="#bib5">Chen et al., 2018</a>; <a href="#bib18">Zetsche et al., 2015</a>), Taq DNA polymerase (Taq) (<a href="#bib6">Chien et al., 1976</a>; <a href="#bib7">Graham et al., 2021</a>; <a href="#bib12">Lawyer et al., 1989</a>), M-MLV reverse transcriptase (RT) (<a href="#bib7">Graham et al., 2021</a>; <a href="#bib10">Kotewicz et al., 1985</a>, <a href="#bib11">1988</a>), and TEV protease (Tev)(<a href="#bib9">Kapust et al., 2001</a>; <a href="#bib16">Reischl et al., 1997</a>). These enzymes were intended to be used for molecular detection of SARS-CoV-2 viral RNA by an RT-PCR coupled to CRISPR/Cas detection (<a href="#bib1">Alcántara et al., 2021a</a>, <a href="#bib2">2021b</a>). However, they can be used in a variety of molecular biology applications, such as nucleic acid amplification, DNA target detection, and <i>in vitro</i> DNA digestion. Protocols were adapted to use minimal laboratory equipment, broadly available in low- and middle-income countries.</p>
<h3 id="sec3.1">Expression in <i>E. coli</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> two days</div>
<p>The following culture and reagent volumes are calculated for one liter (1 L) of protein expression (<a href="#fig2">Figure 2</a>). Scale proportionally if required. Consider additional 200 mL of LB medium to be used as controls (scaling is not required). Sterilize the media by autoclaving at 15 psi and 121°C for 15 min. Wait for the media to cool down to approximately 45°C (approximately 20 min) and proceed to add the corresponding antibiotic.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Protein expression workflow in <i>E. coli</i>
</div>
</figcaption>
</figure>
<ol>
<li>Media preparation
<ol type="a">
<li>Total medium conditions for Taq and RT enzymes: Prepare 2.2 L of LB broth with 30 μg/mL Kan + 17 μg/mL Cam.</li>
<li>Total medium conditions for the Cas12a enzyme: Prepare 1.2L of LB broth with 100 μg/mL Amp.</li>
<li>Total medium condition for the Tev enzyme: Prepare 1.2 L LB broth with 100 μg/mL Amp + 30 μg/mL Cam.</li>
</ol>
</li>
<li>Starter culture
<ol type="a">
<li>Transfer a freshly transformed colony using a pipette tip to a 250 mL flask containing 50 mL of LB with appropriate selection antibiotic(s) (see above section: <a href="#sec2.1">obtaining enzyme-expressing E. coli</a>) (<a href="#fig2">Figure 2</a>).</li>
<li>Incubate at 37°C for 14–16 h with shaking at 150 RPM.</li>
</ol>
</li>
<li>Induction of protein expression
<ol type="a">
<li>Inoculate 1 L of fresh media (LB with the corresponding antibiotics) with the starter culture to reach a starting optical density at 600 nm (OD<sub>600</sub>) of 0.05.</li>
<li>Incubate at 37°C with shaking at 150 RPM until OD<sub>600</sub> = 0.5 (1.5 h) for Tev and LbCas12a or OD<sub>600</sub> = 0.8 for Taq and RT (2 h).</li>
<li>Take a 200 μL sample for pre-induction monitoring.</li>
<li>IPTG induction:
<ol type="i">
<li>For Taq, Tev, and RT, add IPTG to a final concentration of 1 mM and incubate at 37°C for 3 h at 150 RPM.</li>
<li>For Cas12a, incubate on ice or at 4°C for 10–15 min. Add IPTG to a final concentration of 0.2 mM and incubate at 22°C–25°C for 14–16 h at 150 RPM.</li>
</ol>
</li>
<li>For monitoring the induction of protein expression, measure the OD<sub>600</sub> and take 200 μL of culture at 2–3-time intervals (each hour for Taq, Tev, and RT). For Cas12a, take a sample before cell harvest (store on ice) (<a href="#fig3">Figure 3</a>A). Keep culture samples on ice.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Analysis of protein expression by SDS-PAGE</div>
<p>(A) The workflow shows the sample collection from pre-induction to up to 3 intervals of post-induction. Cells collected from 200 μL aliquots were lysed and the cell lysates were analyzed by SDS-PAGE gel electrophoresis. Proteins were visualized by Coomassie blue staining. Stained acrylamide gels show M-MLV RT (B), Taq DNA polymerase (C), MBP-LbCas12a fusion protein (D), and the TEV protease (E). The bands corresponding to each protein molecular weight are highlighted by a dotted-stroke box, showing increased band intensity from pre- to post-induction samples. Due to the presence of the auto-cleavage site, two different molecular weight bands are observed on (E), corresponding to the MBP tag protein and TEV protease. The SDS-PAGE gel percentage used is shown in the bottom right corner of the gel picture.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Cell harvest
<ol type="a">
<li>Keep flasks with induced cultures on ice. Use 50 mL conical tubes for collecting cells. Avoid loading more than 45 mL of cell culture in each tube. Centrifuge the culture at 3000 × <i>g</i> for 5 min at 4°C. Discard the supernatant.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Tubes or bottles for larger volumes can be used depending on rotor availability (i.e. Sorvall or Beckman systems).</div>
</li>
<li>Weigh the cell pellet using an empty tube to tare the balance.</li>
<li>Wash cells with 10 mL per gram of cell pellet weight (1:10 volume buffer/ gram cells) using TAKM or HAKM buffer (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>). Gently resuspend until the pellet is homogenized.</li>
<li>Centrifuge at 3000 × <i>g</i> for 15 min at 4°C. Discard the supernatant and weigh the cell pellet.</li>
<li>Resuspend the cells in 5 mL per gram of cell pellet weight with the indicated buffers prepared with cOmplete™ protease inhibitor cocktail (Use 1 tablet for 50 mL of each buffer) (pre-cooled at 4°C, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>):
<ol type="i">
<li>RT: MB1 buffer</li>
<li>Taq: TaB1 buffer</li>
<li>Cas12: HAKM</li>
<li>Tev: TeB1 buffer</li>
</ol>
</li>
<li>Add lysozyme at 100 μg/mL final concentration to the resuspended cells and mix.</li>
<li>Add DNase I at 40 μg/mL final concentration to the resuspended cells and mix.</li>
<li>Add MgCl<sub>2</sub> at 10 mM final concentration to Taq, RT, and Tev resuspended cells and mix.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Resuspended bacterial cells can be stored at −80°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Induction confirmation
<ol type="a">
<li>Centrifuge the 200 μL samples obtained before induction and during induction at 3000 × <i>g</i> for 5 min at 4°C. Discard the supernatant (<a href="#fig3">Figure 3</a>A).</li>
<li>Resuspend the cell pellets with SDS sample buffer (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>) as follows:
<p class="math-ml"><math> <mrow> <mtext>SDS sample buffer </mtext> <mrow> <mo>(</mo> <mtext>μL</mtext> <mo>)</mo> </mrow> <mo>=</mo> <mfrac> <mrow> <mtext>OD </mtext> <mn>600</mn> <mtext> ∗ Cell Vol μL</mtext> </mrow> <mn>2</mn> </mfrac> </mrow> </math></p>
</li>
<li>Heat the SDS-treated samples at 95°C for 10 min, spin briefly.</li>
<li>Run a polyacrylamide gel electrophoresis (SDS-PAGE) using 15 μL of each sample as follows:
<ol type="i">
<li>Taq and RT: 10% SDS-PAGE (<a href="#fig3">Figures 3</a>B and 3C)</li>
<li>Cas12a: 7.5% SDS-PAGE (<a href="#fig3">Figure 3</a>D)</li>
<li>Tev: 12.5% SDS-PAGE (<a href="#fig3">Figure 3</a>E)</li>
</ol>
</li>
<li>Run the gel for 5 min at 100 V followed by 55 min at 200 V.</li>
<li>Stain the gel using Coomassie blue dye or other alternatives available (<a href="#bib20">Sasse and Gallagher, 2009</a>).</li>
<li>Destain the gel by incubating it in 80 mL of destaining solution (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>) at 22°C–25°C with agitation for 3 h. Change the destaining solution several times until most of the Coomassie blue dye is removed from the gel matrix. Proteins will appear as blue bands. If protein bands are not visible, see the <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.5">problem 3</a>.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Protein purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> three days</div>
<ol start="6">
<li>Cell lysis
<ol type="a">
<li>Use two or three freeze-thaw cycles for cell lysis (<a href="#fig4">Figure 4</a>A). Thaw the frozen cell resuspension on ice and manually mix every 3 min. Once thawed, re-freeze it at −80°C for 15 min. Thaw it again at 4°C on ice, mixing every 3 min.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Take advantage of this time to cool down (4°C) the centrifuge with an appropriate rotor for cell debris removal.</div>
</li>
<li>Proceed as indicated for each protein (<a href="#fig4">Figure 4</a>A): M-MLV reverse transcriptase (RT)
<ol type="i">
<li>Adjust NaCl concentration to 370 mM using a 4 M stock. Mix gently.</li>
<li>Adjust to 20 mL per gram of cell pellet weight with MB1 buffer containing 370 mM NaCl.</li>
<li>Centrifuge at 11000 × <i>g</i> for 45 min at 4°C.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other tubes can be used depending on rotor availability (i.e. Sorvall or Beckman systems). If using 50 mL conical tubes for collecting cells, avoid loading more than 45 mL of cell lysates in each tube. Make sure that the selected tubes resist the applied G forces.</div>
</li>
<li>Transfer the supernatant to a new 50 mL tube. Maintain the clarified lysate at 4°C. Keep the cell pellet for SDS-PAGE analysis in case the protein is undetected in the supernatant.
<div class="note">
<span class="note-title">Note:</span> Avoid freezing or storing crude lysates.</div>
Taq DNA polymerase</li>
<li>Heat the lysis suspension at 80°C for one hour.</li>
<li>Adjust to 20 mL per gram of cell pellet weight with TaB1 buffer.</li>
<li>Adjust glycerol to 10% final concentration.</li>
<li>Add imidazole to a 10 mM final concentration (use a 2 M imidazole stock solution, pH 8.0).</li>
<li>Add β-mercaptoethanol to a 6 mM final concentration.</li>
<li>Centrifuge at 11000 × <i>g</i> for 45 min at 4°C.</li>
<li>Transfer the supernatant to a new 50 mL tube. Maintain the clarified lysate at 4°C.
<div class="note">
<span class="note-title">Note:</span> Avoid freezing or storing crude lysates.</div>
LbCas12a and TEV protease</li>
<li>Centrifuge at 11000 × <i>g</i> for 45 min at 4°C.</li>
<li>Transfer the supernatant to a new 50 mL tube. Maintain the clarified lysate at 4°C.
<div class="note">
<span class="note-title">Note:</span> Avoid freezing or storing crude lysates.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> During centrifugation, take advantage of this time to set up the first step of affinity chromatography (See <a href="#mmc2">Methods video S1</a>). Optimize your steps and timing to reduce as much as possible the time between cell lysis and loading the lysate onto the first chromatographic column.</div>
Break Point: Final purification steps differ from this point onwards as they depend on the desired protein, and whether the purification is manual or automated. Filter the supernatant with a 0.45 μm syringe filter before purification. This will avoid clogging of the column in both manual and automated systems.</li>
</ol>
</li>
</ol>
</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Cell lysis and protein purification</div>
<p>(A) The workflow summarizes the most important steps of the cell lysis on the first part of day 1 (Day 1a). Each protein and the corresponding buffers have been allocated a specific color: M-MLV-RT (black), Taq DNA polymerase (blue), MBP-LbCas12a (purple) and TEV protease (green).</p>
<p>(B) The top-to-bottom scheme is divided into affinity (Day 1b) and ion exchange chromatography (Day 2b). Each protein and the corresponding buffers have been colored as in (A). A right-sided, purple-colored scheme has been included for MBP-LbCas12a representing the extra steps required for its purification. Extra steps for TEV protease purification have been highlighted in green. The days and the processes including day transitions are shown in light blue.</p>
</figcaption>
</figure>
<ol start="7">
<li>Chromatography (<a href="#fig4">Figure 4</a>B) All the steps described below can be performed using a Fast Protein Liquid Chromatography (FPLC) system (<a href="#bib13">Madadlou et al., 2016</a>). A manual procedure using disposable syringes of 10 mL (with a flow rate of 1 mL/min, 30–40 drops/min) can also be performed (See <a href="#mmc2">Methods video S1</a>). All the required buffer recipes are described in <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>.
<div class="note">
<span class="note-title">Note:</span> For manual elution consider preparing gradient changes in buffer salt concentration at 50 mM increasing steps, according to the required concentration range for each protein.</div>
<ol type="a">
<li>M-MLV reverse transcriptase (RT) Day 1: Affinity Chromatography (<a href="#fig4">Figure 4</a>B)
<ol type="i">
<li>Remove the storage solution (20% ethanol) from the HisTrap FF Crude column (1 mL column volume (CV)) with 10 CV of ultra-pure water. For the automated process we used a flow rate of 1 mL/min, and for the manual process ∼1 mL/min or 1 drop every 3 s) (See <a href="#mmc2">Methods video S1</a>)</li>
<li>Equilibrate the column with 10 CV of MB1 buffer.</li>
<li>Filter the clarified supernatant with a 0.45 μm filter to avoid column clogging and load the sample onto the column. Collect the flow-through (FT) in 15 mL conical tubes (See Tip 1 on <a href="#mmc6">Methods video S5</a>).
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Wash with 10 CV of MB2 buffer followed by 10 CV of MB1 buffer. Collect washes in 15 mL conical tubes (<a href="#mmc3">Methods video S2</a>).
<div class="note">
<span class="note-title">Note:</span> Steps v. through vii, are shown in <a href="#mmc4">Methods video S3</a>.</div>
</li>
<li>Elute the protein with 8 mL of MB3 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.</li>
<li>Clean the column with 20 CV of MB1 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 10% SDS-PAGE (100 V for 5 min followed by 200 V for 50 min). Coomassie blue staining can be used. <a href="#fig5">Figure 5</a>A shows the expected results. If protein band is not observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.7">problem 4</a>.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Each fraction can be analyzed by absorbance at 280 nm or by the Bradford assay to identify the presence of protein.</div>
<div class="note">
<span class="note-title">Note:</span> M-MLV reverse transcriptase shows an expected size of 71 kDa. For this protocol the Unstained Protein Standard, Broad Range (10–200 kDa) (NEB) ladder was used (see <a href="#key-resources-table">key resources table</a>).</div>
</li>
<li>Select and pool fractions considering quantity and purity as seen in <a href="#fig5">Figure 5</a>A (eluted fractions 2, 3 and 4). Dialyze using 1 L of MB4 buffer on ice (or cold room set at 4°C if available) for 16 ± 2 h using a D-Tube™ dialyzer Maxi, molecular weight cutoff (MWCO) from 12 to 14 kDa (Merck).
<div class="note">
<span class="note-title">Note:</span> Each dialyzer tube (D-Tube™ dialyzer Maxi) can contain up to 3 mL, equivalent to three 1 mL-fractions. Any dialysis technique or other dialyzer tubes with higher capacity can be used in this step.</div>
</li>
<li>To measure the post-chromatography protein yield, calculate the ratio between recovered protein over total input protein by absorbance at 280 nm, Bradford assay or alternatively by pixel densitometry of the SDS-PAGE gel.<br> Day 2: Ion Exchange Chromatography (<a href="#fig4">Figure 4</a>B; <a href="#mmc5">Methods video S4</a>)</li>
<li>Collect the protein from the dialyzer tube in 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section - <a href="#sec6.9">problem 5</a>.</li>
<li>Remove the storage solution (20% ethanol) from the HiTrap SP column (1 mL CV) with 10 CV of ultra-pure water.</li>
<li>Equilibrate the column with 10 CV of MB4 buffer.</li>
<li>Load the column with the RT protein sample purified by affinity chromatography (step 7a., Day 1., viii.).
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Wash the loaded column with 10 CV of MB4 buffer. Collect it in a 15 mL conical tube.</li>
<li>Elution step using the FPLC system: Set a gradient from 0% to 100% (i.e., in 20 CV) of MB5 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If using the manual mode, prepare 50 mL of buffer stocks from 100 mM to 1 M NaCl (MB5 buffer) by varying the NaCl concentration in 50 mM increments. Use 2 CV of each stock during elution.</div>
</li>
<li>Clean the column with 10 CV of MB5 buffer, 10 CV of MB4 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 10% SDS-PAGE (100 V for 5 min followed by 200 V for 50 min). <a href="#fig5">Figure 5</a>B shows the expected results. If protein band is not observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.7">problem 4</a>.
<div class="note">
<span class="note-title">Note:</span> M-MLV reverse transcriptase shows an expected size of 71 kDa (<a href="#fig5">Figure 5</a>A). For this protocol the Unstained Protein Standard, Broad Range (10–200 kDa) (NEB) ladder was used (see <a href="#key-resources-table">key resources table</a>).</div>
</li>
<li>Pool fractions considering quantity and purity for dialysis with 1 L of MB6 buffer on ice (or 4°C cold room if available) for 16 ± 2 h placing the protein in a D-Tube™ dialyzer Maxi.</li>
<li>(Day 3) Collect the protein from the dialyzer tube into 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.9">problem 5</a>.</li>
<li>Quantify the protein by densitometry analysis using BSA as a standard or by the Bradford assay (See <a href="#sec3.3">enzyme quantification</a> section). Store 20 μL aliquots at −80°C. Flash freeze in liquid nitrogen or dry ice ethanol/methanol/isopropanol bath if available.</li>
</ol>
</li>
<li>Taq DNA polymerase Day 1: Affinity Chromatography (<a href="#fig4">Figure 4</a>B)
<ol type="i">
<li>Remove the storage solution (20% ethanol) from the HisTrap FF Crude column (1 mL column volume) with 10 CV of ultra-pure water (See <a href="#mmc2">Methods video S1</a>).</li>
<li>Equilibrate the column with 10 CV of TaB1 buffer.</li>
<li>Filter the clarified supernatant with a 0.45 μm filter and load the sample on the column (See Tip 1 on <a href="#mmc6">Methods video S5</a>). Collect the flow-through (FT) in conical 15 mL tubes.
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Wash with 10 CV of TaB2 buffer followed by 10 CV of TaB3 buffer. Collect them in 15 mL conical tubes (<a href="#mmc3">Methods video S2</a>).
<div class="note">
<span class="note-title">Note:</span> Steps v. through vii., are shown in <a href="#mmc4">Methods video S3</a>.</div>
</li>
<li>Elute the protein with 8 mL of TaB4 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.</li>
<li>Clean the column with 5 CV of TaB1 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 10% SDS-PAGE (at 100 V for 5 min followed by 200 V for 50 min). Coomassie blue staining can be used. <a href="#fig6">Figure 6</a>A shows the expected results. If protein band is not observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.7">problem 4</a>.
<div class="note">
<span class="note-title">Note:</span> Taq DNA polymerase shows an expected size of 94 kDa. For this protocol the Unstained Protein Standard, Broad Range (10–200 kDa) (NEB) ladder was used.</div>
</li>
<li>Pool fractions considering quantity and purity for dialysis against 1 L of TaB5 buffer on ice (or 4°C cold room if available) for 16 ± 2 h placing the protein in a D-Tube™ dialyzer Maxi, MWCO from 12 to 14 kDa. Any available dialysis technique or dialyzer tube can also be used.<br> Day 2: Ion Exchange Chromatography (<a href="#fig4">Figure 4</a>B; <a href="#mmc5">Methods video S4</a>)</li>
<li>Collect the protein from the dialyzer tube into 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.9">problem 5</a>.</li>
<li>Remove the storage solution (20% ethanol) from the HiTrap Heparin HP column with 10 CV of ultra-pure water.</li>
<li>Equilibrate the column with 10 CV of TaB5 buffer.</li>
<li>Load the column with the Taq protein sample purified by affinity chromatography (step 7b., Day 1., viii.).
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Wash twice with 10 CV of TaB5 buffer. Collect them in 15 mL conical tubes.</li>
<li>Elution step using the FPLC system: Set a gradient from 0% to 100% (i.e., 20 CV) of TaB6 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If using the manual mode, prepare 50 mL buffer stocks from 100 mM to 1 M NaCl (TaB6 buffer) by varying the NaCl concentration in 50 mM increments. Use 2 CV of each stock during elution.</div>
</li>
<li>Clean the column with 10 CV of TaB5 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 10% SDS-PAGE (100 V for 5 min followed by 200 V for 50 min). Coomassie blue staining can be used. <a href="#fig6">Figure 6</a>B shows the expected results. If protein band is not observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.7">problem 4</a>.
<div class="note">
<span class="note-title">Note:</span> Taq DNA polymerase shows an expected size of 94 kDa (<a href="#fig6">Figure 6</a>). For this protocol the Unstained Protein Standard, Broad Range (10–200 kDa) (NEB) ladder was used.</div>
</li>
<li>Pool fractions considering quantity and purity for dialysis against 1 L of TaB7 buffer (at 4°C or cold room if available) for 16 ± 2 h placing the protein in a D-Tube™ dialyzer Maxi.</li>
<li>(Day 3) Collect the protein from the dialyzer tube into 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.9">problem 5</a>.</li>
<li>Quantify the protein by densitometry analysis using BSA as a standard or by the Bradford assay (See <a href="#sec3.3">enzyme quantification</a> section). Store 20 μL aliquots at −80°C. Flash freeze in liquid nitrogen or dry ice ethanol/methanol/isopropanol bath if available.</li>
</ol>
</li>
<li>Cas12a Day 1: Affinity Chromatography (<a href="#fig4">Figure 4</a>B)
<ol type="i">
<li>Remove the storage solution (20% ethanol) from the HisTrap FF crude column (1 mL column volume) with 10 CV of ultra-pure water (See <a href="#mmc2">Methods video S1</a>).</li>
<li>Equilibrate the column with 10 CV of CB1 buffer.</li>
<li>Filter the clarified supernatant with a 0.45 μm filter and load the sample on the column (see Tip 1 on <a href="#mmc6">Methods video S5</a>). Collect the flow-through (FT) in 15 mL tubes.
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Wash with 10 CV of CB2 buffer followed by 10 CV of CB3 buffer. Collect them in 15 mL conical tube (<a href="#mmc3">Methods video S2</a>).
<div class="note">
<span class="note-title">Note:</span> Steps v. through vii., are shown in <a href="#mmc4">Methods video S3</a>.</div>
</li>
<li>Elute the protein with 8 mL of CB4 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.</li>
<li>Clean the column with 5 CV of CB2 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 7.5% SDS-PAGE (100 V for 5 min followed by 200 V for 50 min). Coomassie blue staining can be used. <a href="#fig7">Figure 7</a>A shows the expected results. If protein band is not observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.7">problem 4</a>.
<div class="note">
<span class="note-title">Note:</span> MBP-LbCas12a shows an expected size of 172 kDa. For this protocol the Unstained Protein Standard, Broad Range (10–200 kDa) (NEB) ladder was used.</div>
</li>
<li>Pool the protein-containing fractions considering quantity and purity, for TEV protease digestion and dialysis.</li>
<li>Add TEV protease to a 2.21 μM final concentration.</li>
<li>Dialyze Tev-treated fractions of LbCas12a against 1 L of CB5 buffer on ice (or cold room set at 4°C if available) for 16 ± 2 h placing the protein in a D-Tube<sup>TM</sup> dialyzer Maxi.<br> Day 2a: MBP Affinity Chromatography (<a href="#fig4">Figure 4</a>B)</li>
<li>Collect the protein from the dialyzer tube into 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.9">problem 5</a>.</li>
<li>Assess TEV protease cleavage performance and protein purity by 12.5% SDS-PAGE (100 V for 5 min followed by 200 V for 50 min). Use Coomassie blue for staining. <a href="#fig7">Figure 7</a>B shows the expected results.</li>
<li>Remove the storage solution (20% ethanol) from the MBPTrap HP column (1 mL column volume) with 10 CV of ultra-pure water.</li>
<li>Equilibrate the column with 10 CV of CB6 buffer.</li>
<li>Load the column with the Tev-treated protein and collect the FT. Keep it on ice.</li>
<li>Assess purification performance by 12.5% SDS-PAGE (100 V for 5 min followed by 200 V for 50 min). Use Coomassie blue for staining.</li>
<li>Wash the column with 10 CV of CB7 buffer to remove the MBP trapped in the column.</li>
<li>Clean the column with 10 CV of CB6 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.<br> Day 2b: Ion Exchange Chromatography (<a href="#fig4">Figure 4</a>B)</li>
<li>Remove the storage solution (20% ethanol) from the HiTrap Capto SP ImpRes Sepharose column (1 mL column volume) with 10 CV of ultra-pure water.</li>
<li>Equilibrate the column with 10 CV of CB6 buffer.</li>
<li>Load the FT collected from the MBPTrap HP to the column HiTrap Capto SP ImpRes Sepharose column (7c, Day2a, step xv.), and wash with 10 CV of CB6 buffer. Collect it in a 15 mL conical tube.
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Elution step using the FPLC system: Set a gradient from 0% to 100% (i.e., 20 CV) of CB8 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If using the manual mode, prepare 50 mL buffer stocks from 125 mM to 1 M KCl (CB8 buffer) by varying the salt concentration in 50 mM increments. Use 2 CV of each stock during elution.</div>
</li>
<li>Clean the column with 10 CV of CB8 buffer, 10 CV of CB6 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 7.5% SDS-PAGE (100 V for 5 min and 200 V for 50 min). <a href="#fig7">Figure 7</a>C shows the expected results.
<div class="note">
<span class="note-title">Note:</span> LbCas12a shows an expected size of 130 KDa (<a href="#fig7">Figure 7</a>B). For this protocol the Unstained Protein Standard, Broad Range (10–200 KDa) (NEB) ladder was used.</div>
</li>
<li>Pool fractions considering quantity and purity for dialysis against 1 L of CB9 buffer (or 4°C cold room if available) for 16 ± 2 h placing the protein in a D-Tube™ dialyzer Maxi.</li>
<li>(Day 3) Collect the protein from the dialyzer tube into 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.9">problem 5</a>.</li>
<li>Quantify the protein by densitometry analysis using BSA as a standard or by the Bradford assay (See <a href="#sec3.3">enzyme quantification</a> section). Store 20 μL aliquots at −80°C. Flash freeze in liquid nitrogen or dry ice ethanol/methanol/isopropanol bath if available.</li>
</ol>
</li>
<li>TEV protease Day 1: Affinity Chromatography (<a href="#fig4">Figure 4</a>B)
<ol type="i">
<li>Remove the storage solution (20% ethanol) from the HisTrap FF Crude column (1 mL column volume) with 10 CV of ultra-pure water (See <a href="#mmc2">Methods video S1</a>).</li>
<li>Equilibrate the column with 10 CV of TeB2 buffer.</li>
<li>Filter the clarified supernatant with a 0.45 μm filter and load the sample on the column (See Tip 1 on <a href="#mmc6">Methods video S5</a>). Collect the FT in 15 mL tubes.
<div class="note">
<span class="note-title">Note:</span> Keep all purification intermediates until SDS-PAGE analysis. A 20 μL sample of each step can be prepared with SDS sample buffer during pauses.</div>
</li>
<li>Wash with 10 CV of TeB3 buffer. Collect it in 15 mL conical tubes.</li>
<li>Elute the protein with 20 CV of TeB4 buffer. Collect 0.5 mL fractions in 1.5 mL microcentrifuge tubes.</li>
<li>Clean the column with 5 CV of TeB2 buffer, 10 CV of ultra-pure water, and store it with 5 CV of 20% ethanol.</li>
<li>Assess purification performance by 12.5% SDS-PAGE (100 V for 5 min and 200 V for 50 min). <a href="#fig8">Figure 8</a> shows the expected results. If protein band is not observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.7">problem 4</a>.
<div class="note">
<span class="note-title">Note:</span> TEV protease has an expected size of 28 kDa (<a href="#fig8">Figure 8</a>). For this protocol, the Unstained Protein Molecular Weight Marker (14.4–116 kDa) (Thermo Scientific) ladder was used.</div>
</li>
<li>Pool fractions considering quantity and purity for dialysis against 1 L of TeB5 buffer for 16 ± 2 h placing the protein in D-Tube™ dialyzer Maxi on ice (or 4°C cold room if available).</li>
<li>(Day 2) Collect the protein from the dialyzer tube into 1.5 mL tubes, centrifuge at 15000 × <i>g</i> for 15 min at 4°C and transfer the supernatant to new 1.5 mL tubes. If a large protein pellet is observed, see <a href="#troubleshooting">troubleshooting</a> section – <a href="#sec6.9">problem 5</a>.</li>
<li>Quantify the protein by densitometry analysis using BSA as a standard or by the Bradford assay (See <a href="#sec3.3">enzyme quantification</a> section). Store 20 μL aliquots at −80°C. Flash freeze in liquid nitrogen or dry ice ethanol/methanol/isopropanol bath if available.
<div class="note">
<span class="note-title">Note:</span> <a href="#mmc6">Methods video S5</a> shows tips to improve manual chromatography performance.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc2.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S1. Procedure to equilibrate a column before use, related to step 7</div>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Purification of M-MLV reverse transcriptase (RT)</div>
<p>HisTrap FF and HiTrap SP columns were used for affinity and ion-exchange chromatography, respectively.</p>
<p>(A) The samples collected for cell lysis monitoring, the column post-loading flow-through (FT) and the washes, together with the main fractions obtained from affinity chromatography (AC) were analyzed by 10% SDS-PAGE gel electrophoresis and visualized by Coomassie blue staining.</p>
<p>(B) The post-dialysis samples from AC, the FT and the washes, together with the obtained fractions from the buffer salt concentration gradient obtained from ion-exchange chromatography (IEC) were analyzed as in (A). The salt concentrations at which the protein eluted are shown. The white dotted-stroke box shows the fractions selected and pooled for dialysis. The molecular weight of the protein is shown in the right and the SDS-PAGE gel percentage used is shown in the bottom left corners of the gel pictures.</p>
</figcaption>
</figure>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Purification of Taq DNA polymerase</div>
<p>HisTrap FF and HiTrap Heparin HP columns were used for affinity and ion-exchange chromatography, respectively.</p>
<p>(A) The samples collected for cell lysis monitoring, the column post-loading FT and the washes, together with the main fractions obtained from AC were analyzed by 10% SDS-PAGE gel electrophoresis and visualized by Coomassie blue staining.</p>
<p>(B) The post-dialysis samples from AC, the FT and the washes, together with the obtained fractions from the buffer salt concentration gradient obtained from IEC were analyzed as in (A). The salt concentrations at which the protein eluted are shown. The white dotted-stroke box shows the fractions selected and pooled for dialysis. Protein precipitation was observed on AC post-dialysis pooled fractions (P1 and P2). The molecular weight of the protein is shown in the right and the SDS-PAGE gel percentage used is shown in the bottom corners of the gel pictures.</p>
</figcaption>
</figure>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Affinity chromatography of MBP-LbCas12a, TEV digestion and purification of LbCas12a</div>
<p>HisTrap FF and MBPTrap HP columns (A and B) were used for affinity chromatography and a HiTrap Capto SP ImpRes Sepharose column (C) for ion-exchange chromatography.</p>
<p>(A) The samples collected for cell lysis monitoring, the column post-loading flow-through (FT) and the washes, together with the main fractions obtained from AC were analyzed by SDS-PAGE gel electrophoresis and visualized by Coomassie blue staining.</p>
<p>(B) The post-dialysis pooled samples were digested with TEV protease and loaded into the MBPTrap HP column. The white top dotted-stroke boxes show the bands resulting from the (MBP)LbCas12a before and after TEV protease treatment; and the bottom ones show the 28 kDa and 42 kDa bands corresponding to the TEV protease and the MBP tag proteins, respectively. The pre- and post-cleavage molecular weights are shown in the right.</p>
<p>(C) The post-dialysis samples from (B), the collected FT from (C) and the washes, together with the obtained fractions from the buffer salt concentration gradient obtained from IEC were analyzed as in (A). The salt concentrations at which the protein eluted are shown. The white dotted-stroke box shows the fractions selected and pooled for dialysis. The band corresponding to LbCas12a is marked with an arrow, while the MBP band is marked with a star symbol. The resulting molecular weights are on the right (A-B) and on the left corner (C) of the picture. The SDS-PAGE gel percentage used is shown in the bottom corners of the gel pictures.</p>
</figcaption>
</figure>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Purification of TEV protease</div>
<p>The HisTrap FF column was used for affinity chromatography. The samples collected for cell lysis monitoring, the column post-loading FT and the washes, together with the main fractions obtained from AC were analyzed by SDS-PAGE gel electrophoresis and visualized by Coomassie blue staining. The molecular weight of the TEV protease is shown in the left and the SDS-PAGE gel percentage used is shown in the bottom corner of the gel pictures.</p>
</figcaption>
</figure>
<figure id="mmc3"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc3.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S2. Details of the wash step for manual chromatography, related to step 7</div>
</figcaption>
</figure>
<figure id="mmc4"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc4.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S3. Protein elution for affinity chromatography and purification performance assessment, related to step 7</div>
</figcaption>
</figure>
<figure id="mmc5"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc5.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S4. Protein elution for ion exchange chromatography, related to step 7</div>
</figcaption>
</figure>
<figure id="mmc6"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc6.mp4" type="video/mp4"></source>
    Your browser does not support HTML5 video.
  </video>
<figcaption>
<div class="figcaption-title">Methods video S5. Tips for a better manual chromatography performance, related to step 7</div>
</figcaption>
</figure>
<h3 id="sec3.3">Enzyme quantification</h3>
<ol start="8">
<li>Quantify the purified aliquots according to the method of your preference.
<div class="note">
<span class="note-title">Note:</span> We recommend quantifying all four enzymes using the same quantification method. Here we describe three methods for protein quantification that can be used depending on equipment availability.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A quality control assay can be performed (<a href="#bib15">Raynal et al., 2014</a>). As some buffers might interfere with concentration measurement, consider the buffer used to choose a quantification assay.</div>
<ol type="a">
<li>Absorbance at 280 nm
<div class="note">
<span class="note-title">Note:</span> Here we use the NanoDrop spectrophotometer. Alternatively, any spectrophotometer that reads in the UV range can be used with appropriate quartz cuvettes.</div>
<ol type="i">
<li>Clean the pedestal with dH<sub>2</sub>O. Apply 2 μL in the bottom pedestal arm, then lower the upper pedestal to form a liquid column. Wait for a couple of minutes. Wipe away the water from both pedestals using a clean wipe (i.e., Kimtech Science<sup>TM</sup> Kimwipe<sup>TM</sup>).</li>
<li>Select the Protein A280 option and perform the blank reading using 1–2 μL of the respective storage buffer.
<div class="note">
<span class="note-title">Note:</span> Make sure that your buffer does not contain high concentrations of any reagent that reports a considerable absorbance at 280 nm (i.e., Triton X-100 that is used to store the TEV protease).</div>
</li>
<li>When the measurement is complete, raise the upper pedestal arm and clean the buffer from both upper and bottom measurement surfaces using a clean dry paper wipe.</li>
<li>Enter the sample name and select “E and MW” as sample type option to use the extinction coefficient and molecular weight of the protein to calculate the sample concentration.
<div class="note">
<span class="note-title">Note:</span> For LbCas12a use 167780 M<sup>−1</sup> cm<sup>−1</sup> extinction coefficient, with a MW of 130 kDa; for Taq DNA polymerase use 112760 M<sup>−1</sup> cm<sup>−1</sup> extinction coefficient, with a MW of 94 kDa for and for M-MLV reverse transcriptase use 79780 M<sup>−1</sup> cm<sup>−1</sup> extinction coefficient, with a MW of 71 kDa.</div>
</li>
<li>Place 2 μL of your sample and measure in triplicates. Calculate the average and the standard deviation. It is recommended to measure at least two different dilutions of the protein stock.</li>
<li>Clean both pedestal arms with a dry wipe. Then, continue with the next sample.</li>
</ol>
</li>
<li>Gel quantification
<div class="note">
<span class="note-title">Note:</span> For gel quantification, a protein standard is required. BSA could be used as standard (i.e., BSA protein standard, Cat. No.: P0834, Sigma-Aldrich).</div>
<ol type="i">
<li>Prepare at least five dilutions of the standard protein. We recommend using a range between 1 – 10 μg/sample (20 μL). Then mix each dilution with the SDS sample buffer.</li>
<li>Prepare serial dilutions of your protein, at least three sample dilutions. Then mix the diluted samples with the SDS sample buffer.</li>
<li>Heat the diluted standard and protein samples at 95°C for 5 min. Spin briefly.</li>
<li>Prepare the SDS-PAGE gel according to the percentage required for each protein.</li>
<li>Load the standard and samples (100 V for 15 min followed by 200 V for 40 min).</li>
<li>Stain the gel with Coomassie blue dye or another method available. Destain the gel matrix as described above and visualize the protein bands.</li>
<li>Gel images can be analyzed with ImageJ software. A step-by-step protocol for using ImageJ is available in the following link: <a href="https://imagejdocu.tudor.lu/gui/analyze/gels">https://imagejdocu.tudor.lu/gui/analyze/gels</a>
</li>
<li>Plot the band density (Y-axis) of the standard curve versus BSA concentration (X-axis) (<a href="#bib17">Syrový and Hodný, 1991</a>).</li>
<li>Determine the unknown sample concentration of the protein using linear regression of your standard curve using the function:
<p class="math-ml"><math> <mrow> <mtext>Y </mtext> <mrow> <mo>(</mo> <mtext>density</mtext> <mo>)</mo> </mrow> <mo>=</mo> <mtext>m </mtext> <mrow> <mo>(</mo> <mrow> <mrow> <mo>[</mo> <mtext>X</mtext> <mo>]</mo> </mrow> <mfrac> <mtext>μg</mtext> <mtext>μL</mtext> </mfrac> </mrow> <mo>)</mo> </mrow> <mo>+</mo> <mtext>b</mtext> </mrow> </math></p>
<br> With (m) being the slope and (b) the background. [X] = (Y – b)/m</li>
<li>Calculate the sample concentration by interpolating the corresponding density with the linear regression. Multiply the resulting concentrations by the dilution factor applied at the (b.ii) step. Average and calculate your standard deviation.</li>
</ol>
</li>
<li>Bradford quantification
<div class="note">
<span class="note-title">Note:</span> For Bradford quantification, a protein standard is required. BSA could be used as standard (i.e., BSA protein standard, Cat. No.: P0834, Sigma-Aldrich).</div>
<div class="note">
<span class="note-title">Note:</span> The procedure could be performed in a 96-well plate if a plate reader is available (e.g., Synergy<sup>TM</sup> H1 Hybrid Multi-Mode plate reader) or in microcentrifuge tubes if a spectrophotometer is available.</div>
<ol type="i">
<li>Prepare five dilutions (use storage buffer as diluent) of the protein standard. We recommend using a range between 15 – 250 μg/mL in 100 μL. Prepare a blank (storage buffer only) as well. The volume can be scaled depending on the spectrophotometer that is available.</li>
<li>Prepare three 10 μL serial dilutions of the proteins (1/10- 1 is recommended).</li>
<li>Add 90 μL of Bradford (Bio-Rad) reagent to each dilution in a well or microcentrifuge tube.</li>
<li>Mix gently 3 to 5 times.</li>
<li>Incubate at 22°C–25°C for 5 min in the dark.</li>
<li>Read absorbance at 595 nm.</li>
<li>Normalize the raw data by subtracting the blank absorbance (storage buffer only).</li>
<li>Plot the normalized absorbance (Y-axis) of the standard curve <i>versus</i> BSA concentration (X-axis) (<a href="#bib3">Bradford, 1976</a>).</li>
<li>Determine the unknown concentration of the protein, using linear regression of your standard curve using the formula:
<p class="math-ml"><math> <mrow> <mtext>Y</mtext> <mrow> <mo>(</mo> <mrow> <mtext>Abs</mtext> <mn>595</mn> </mrow> <mo>)</mo> </mrow> <mo>=</mo> <mtext>m</mtext> <mrow> <mo>(</mo> <mrow> <mrow> <mo>[</mo> <mtext>X</mtext> <mo>]</mo> </mrow> <mfrac> <mtext>μg</mtext> <mtext>mL</mtext> </mfrac> </mrow> <mo>)</mo> </mrow> <mo>+</mo> <mtext>b </mtext> </mrow> </math></p>
<br> With (m) being the slope and (b) the background. [X] = (Y – b)/m</li>
<li>Calculate the sample concentration by interpolating the corresponding Abs595 (blank subtracted) with the linear regression. Multiply the resulting concentrations by the dilution factor applied at the (c.ii) step. Average and calculate your standard deviation.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Enzyme activity evaluation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> One day</div>
<ol start="9">
<li>Enzyme activity tests An RT-PCR assay can be performed to confirm that the Taq DNA polymerase and the M-MLV reverse transcriptase are in an active state (<a href="#fig9">Figure 9</a>A). Here, we describe a one-step RT-PCR assay. For protocol details, please refer to (<a href="#bib2">Alcántara et al., 2021b</a>). For buffer composition and a recipe for enzyme dilution and RT-PCR reaction see the <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>.
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> A two-step RT-PCR reaction can also be performed. Additionally, a commercial buffer can be used to evaluate enzymatic activity. We tested the buffers and components of the RevertAid First Strand cDNA Synthesis Kit (Cat. No. K1622, ThermoFisher) for M-MLV reverse transcriptase; and Standard Taq Buffer (Cat. No. M0273S, NEB) for Taq DNA polymerase.</div>
<ol type="a">
<li>Prepare a 20X working stock of each enzyme as follows:
<ol type="i">
<li>20X working stock of M-MLV reverse transcriptase: Dilute the enzyme stock in MB6 buffer to a final concentration of 34 ng/μL.</li>
<li>20X working stock of Taq DNA polymerase: Dilute the enzyme stock in MB6 buffer to a final concentration of 32 ng/μL.</li>
</ol>
</li>
<li>Prepare the master mix according to the following recipe, scale the required volume of each reagent according to the number of samples to test. It is recommended to prepare the master mix for 1-2 additional reactions to account for volume loss during pipetting.
<div class="note">
<span class="note-title">Note:</span> We recommend performing the RT-PCR using positive and negative standardized controls.</div>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Initial concentration</th>
<th>Final concentration</th>
<th>Volume (μL) per reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPB4X</td>
<td>4X</td>
<td>1X</td>
<td>5</td>
</tr>
<tr>
<td>M-MLV RT</td>
<td>20X</td>
<td>1X</td>
<td>1</td>
</tr>
<tr>
<td>Taq polymerase</td>
<td>20X</td>
<td>1X</td>
<td>1</td>
</tr>
<tr>
<td>Reverse primer</td>
<td>5 μM</td>
<td>0.2 μM</td>
<td>0.8</td>
</tr>
<tr>
<td>Forward primer</td>
<td>5 μM</td>
<td>0.2 μM</td>
<td>0.8</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>-</td>
<td>-</td>
<td>9.4</td>
</tr>
<tr>
<td>Sample/controls</td>
<td>-</td>
<td>-</td>
<td>2</td>
</tr>
<tr>
<td><b>Final volume</b></td>
<td> </td>
<td> </td>
<td><b>20</b></td>
</tr>
</tbody>
</table>
</li>
<li>To monitor potential reaction contaminations, a non-template negative control (NTC) should be included in each assay. For the NTC, nuclease-free water must be added to the respective PCR tube</li>
<li>Run the following RT-PCR program:
<table id="undtbl4">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Step</th>
<th>Temperature (°C)</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Reverse transcription</td>
<td>50</td>
<td>20 min</td>
<td>1</td>
</tr>
<tr>
<td>Initial denaturation</td>
<td>95</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95</td>
<td>5 s</td>
<td rowspan="2">45</td>
</tr>
<tr>
<td>Annealing/Extension</td>
<td>55</td>
<td>30 s</td>
</tr>
<tr>
<td>Hold</td>
<td>10</td>
<td colspan="2">∞</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Total run time 1:40 hours</div>
</li>
<li>The RT-PCR products can be visualized by agarose gel electrophoresis (<a href="#fig9">Figure 9</a>B). Use a percentage of agarose according to the expected RT-PCR product size (3–5%).</li>
</ol>
</li>
</ol>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. Enzyme activity evaluation and quantitative and qualitative analysis</div>
<p>A RT-PCR assay was performed to confirm that the Taq DNA polymerase and the M-MLV reverse transcriptase were in an active state.</p>
<p>(A) Purified enzyme stocks were analyzed by SDS-PAGE gel electrophoresis and visualized by Coomassie blue staining. Average purity ranged between 90% and 99% for each enzyme.</p>
<p>(B) RT-PCR reaction for functional enzyme confirmation. Results with commercial and in-house produced enzymes using a positive control (+) and a non-template negative control (-, NTC) were analyzed by 1.5% agarose gel electrophoresis and visualized with a blue light transilluminator.</p>
<p>(C) CRISPR/Cas reaction for functional enzymes confirmation. The fluorescent target detection was based on the indicated RT-PCR products on (B). The dark and light blue lines represent the normalized fluorescence intensity (RFU) over time (min) of commercial and in-house LbCas12a proteins, respectively. The full lines represent the positive control reactions, while the dotted lines the negative NTC controls. The fluorescence was measured on a SynergyTM H1 Hybrid Multi-Mode plate reader (Biotek).</p>
<p>(D) The bar graph shows the mean of the normalized fluorescence intensity ratio of the positive control over the NTC controls. The bars represent the mean of five measurements, and the error bars the standard deviation of the mean. Bar colors as on (C).</p>
<p>(E) The CRISPR/Cas reaction was also assessed qualitatively by mixing the reaction mixture in a tube, without exposure to light for 30 min and visualization with a blue light transilluminator.</p>
<p>(F) The same reaction and visualization method as in (E) was used for positive samples with different levels of SARS-CoV-2 RNA viral load. This panel has been elaborated from (<a href="#bib1">Alcántara et al., 2021a</a>)(<a href="#fig5">Figure 5</a>).</p>
</figcaption>
</figure>
<ol start="10">
<li>CRISPR/Cas reaction for functional enzymes confirmation A CRISPR/Cas assay can be performed to confirm that the LbCas12a is in an active state. Here, we describe a fluorescent detection assay using the target-specific RT-PCR products as input DNA (<a href="#fig9">Figures 9</a>C and 9D). For methodology details for SARS-CoV-2 detection, please refer to (<a href="#bib1">Alcántara et al., 2021a</a>). For buffer composition and a recipe for the CRISPR/Cas reaction see the <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>.
<div class="note">
<span class="note-title">Note:</span> A crRNA sequence and synthetic dsDNA target or the corresponding primers to amplify the target region are necessary before performing the CRISPR/Cas assay. For a quick guide for crRNA design, please see <a href="https://international.neb.com/faqs/2018/05/03/how-do-i-design-a-guide-rna-for-use-with-engen-lba-cas12a">https://international.neb.com/faqs/2018/05/03/how-do-i-design-a-guide-rna-for-use-with-engen-lba-cas12a</a>.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The 1X NEBuffer<sup>TM</sup> r2.1 (Cat. No. B6002S, NEB) can be used as an alternative for reaction buffer. Local production of the crRNA used here can be found in (<a href="#bib2">Alcántara et al., 2021b</a>).</div>
<ol type="a">
<li>Perform the crRNA re-folding by incubation at 65°C for 10 min followed by cooling at 22°C–25°C for 10 min.</li>
<li>Prepare a 10X CRISPR-Cas12a complex according to the following recipe:
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration per reaction</th>
</tr>
</thead>
<tbody>
<tr>
<td>LbCas12a</td>
<td>0.1μM</td>
</tr>
<tr>
<td>Re-folded crRNA</td>
<td>0.15 μM</td>
</tr>
<tr>
<td>Dual-labeled single-stranded DNA (ssDNA) reporter probe</td>
<td>3 μM</td>
</tr>
<tr>
<td>1X CrB1<sup>∗</sup>
</td>
<td>-</td>
</tr>
<tr>
<td>Final volume</td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">∗1X CRISPR/Cas buffer 1, see <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1</a>.</p>
</li>
<li>Incubate the 10X CRISPR-Cas12a complex at 22°C–25°C for 10 min. Protect it from light.</li>
<li>Add MgCl<sub>2</sub> to the 1X CrB1 to get a final concentration of 17.65 mM. This is labeled 1X CrB2.</li>
<li>The CRISPR/Cas assay is prepared according to the following scheme:
<table id="undtbl6">
<thead>
<tr>
<th>PipettingOrder</th>
<th>Reagent</th>
<th>Initial concentration</th>
<th>Final concentration</th>
<th>Final volume (μL) per assay</th>
<th>Notes</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>1X CrB2 (17.65 mM MgCl<sub>2</sub>)</td>
<td>1X</td>
<td>1X</td>
<td>85</td>
<td rowspan="2">Mix separately</td>
</tr>
<tr>
<td>2</td>
<td>RT-PCR product or 1 μM synthetic template</td>
<td>-</td>
<td>-</td>
<td>5</td>
</tr>
<tr>
<td>3</td>
<td>CRISPR-Cas12a complex</td>
<td>10X</td>
<td>1X</td>
<td>10</td>
<td> </td>
</tr>
<tr>
<td> </td>
<td>Final volume</td>
<td>-</td>
<td>-</td>
<td>100</td>
<td> </td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> You can prepare the CRISPR/Cas assay in separate tubes, or PCR strips.</div>
</li>
<li>Once the CRISPR/Cas reaction is prepared, incubate it without light exposure at 22°C–25°C for 30 min.</li>
<li>Transfer the tubes to a blue transilluminator and close the filter tap (<a href="#fig9">Figures 9</a>E and 9F).</li>
<li>Register the results of direct fluorescence visualization using a photo camera or a smartphone.
<div class="note">
<span class="note-title">Note:</span> We recommend using a black cabinet, box, or darkroom to improve direct visualization of the results.</div>
</li>
<li>Alternatively, a fluorescence 96-well plate reader was used. Fluorescence was excited at 491 nm and measured at 525 nm on a SynergyTM H1 Hybrid Multi-Mode plate reader (Biotek) (<a href="#fig9">Figures 9</a>C and 9D).</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Each required enzyme is intended to be produced to high purity following a four-day scheme. Typically, 3–5 g of dry cells are obtained from 1 L of cell culture. After chromatography steps, 6–12 mL of protein are obtained and stored. The standardized protocols show high purity and yields of the enzymes. Average purity ranged between 90% and 99%. Regarding protein production, Cas12a shows the highest yield with 2 mg/L of cell culture, followed by the Taq DNA polymerase with 1.2 mg/L of cell culture. M-MLV reverse transcriptase shows the lowest yield with 0.6 mg/L of cell culture. Concerning the total number of reactions, 1 L of cell culture can produce enzymes for 20 to 40 thousand one-step RT-PCR reactions, and around 20 thousand CRISPR/Cas detection reactions.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The described protocol is not intended to be used for commercial purposes. Batch-to-batch variation in expression and purification of recombinant enzymes could result in subtle changes of activity. Therefore, each new batch of recombinant enzyme must be evaluated to guarantee the quality of each preparation. <i>In vitro</i> assays such as RT-PCR and CRISPR/Cas reactions have been described here to evaluate the functionality of the recombinant enzymes. However, long-term enzyme stability has not been assessed. The enzyme activity has been validated in a molecular diagnostic pipeline (<a href="#bib1">Alcántara et al., 2021a</a>). However, we recommend local validation with the pertinent samples of interest. Check your national and/or local regulatory guidelines. A series of technical guides for <i>in vitro</i> diagnostic validation is available at the WHO website (<a href="https://extranet.who.int/pqweb/vitro-diagnostics/technical-guidance-series">https://extranet.who.int/pqweb/vitro-diagnostics/technical-guidance-series</a>). The enzymes described here are the wild-type version. Modifications on their amino acid sequences have been reported to increase activity and genome-wide optimization of <i>E. coli</i> could improve production yields (<a href="#bib14">Mahalik et al., 2014</a>). The yield of protein expression can vary depending on the <i>E. coli</i> strain used, type of medium, and culture conditions for expression.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Transformed bacteria did not grow on LB plate + antibiotics (<a href="#sec2.1">obtaining enzyme-expressing E. coli</a> section, step 2b). Competent cells transformed with the correct plasmid must grow on an LB agar plate or LB broth supplemented with the appropriate antibiotic (i.e., for the plasmids used in this protocol: kanamycin, ampicillin, or chloramphenicol).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>It is recommended to use fresh competent cells. Usually, incubation at 37°C for 16–18 h is sufficient to see colonies on LB plate or turbidity in LB broth. The efficiency of transformation may decrease when using long-stored competent cells. Check the procedure used for competent cells preparation. It is important to maintain the cold chain along the total process to reduce the loss of cell viability. Check that culture media is supplemented with the adequate antibiotic concentration. If other transformation techniques or a modified version was performed, we recommend assessing the methodology described in literature (<a href="#bib4">Chan et al., 2013</a>; <a href="#bib8">Inoue et al., 1990</a>)</p>
<h3 id="sec6.3">Problem 2</h3>
<p>No or few isolated bacterial colonies are observed on plates (<a href="#sec2.1">obtaining enzyme-expressing E. coli</a> section, step 2b). Isolated colonies are required to identify recombinant bacteria harboring the right plasmid by colony PCR. Although the antibiotic restrains the growth of bacteria without the plasmid, not all observed colonies may contain the plasmid.</p>
<h3 id="sec6.4">Potential solution</h3>
<p>If there are few isolated colonies, you can re-plate one colony using a streak technique in a new plate. One important tip before the plate streak is to check that the culture plate is already dry. Always incubate the culture plates at 37°C for a few minutes before the plate streak (i.e., particularly if using stored culture plates). Finally, check that the antibiotic concentration is adequate for the bacterial selection. If it is necessary, prepare new plates and/or antibiotic stocks.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Lack of protein induction as monitored by SDS-PAGE gel (<a href="#sec3.1">expression in E. coli</a> section, step 5f). Induction monitoring is an essential step to verify that proteins are being expressed correctly. SDS-PAGE gel evaluation can give insights if it is necessary to optimize expression conditions.</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Verify the time and temperature used for induction and the concentration of IPTG. Review colony PCR confirmation. Verify and change IPTG stocks. If the protein band is not seen, the expression protocol must be optimized.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Lack of proteins in the elution fractions as monitored by SDS-PAGE gel (<a href="#sec3.2">protein purification</a> section, steps 7a–d). Protein purification monitoring is crucial to determine which aliquots will be pooled for downstream steps (i.e., concentration, quantification, or secondary purification). Depending on the volume of expression culture, the number of aliquots that contain the protein of interest may vary. Usually, after all the purification steps, 6–12 mL of protein are obtained from 1 L of expression culture. It is recommended to keep all the purification fractions until the SDS-PAGE is done.</p>
<h3 id="sec6.8">Potential solution</h3>
<p>First, identify in which fraction (flow-through or washing) is the protein of interest by SDS-PAGE electrophoresis. If protein bands are seen in other fractions than the elution, we recommend to (1) check the pH of the used buffers, since low or high pH could affect the performance of the purification; especially when an ion exchange column is used. (2) If the elution is done with a unique concentration elution buffer, consider using a gradient to improve/optimize the elution step. (3) In the case of using an affinity column, check that the column is re-equilibrated and freshly reconstituted before the chromatography. If protein bands are not seen in any purification fraction, check the conditions of the previous cell lysis step. Optimization could be necessary, or the expressed protein could be in the insoluble cell fraction.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Protein is precipitating after the dialysis step (<a href="#sec3.2">protein purification</a> section, steps 7a–d). A protein pellet after centrifugation may indicate that the protein is precipitating. Small pellets are commonly found in many protein preparations.</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If the protein pellet is large, evaluate and revise the dialysis buffer composition. If the equipment is available (e.g., Tycho or Prometheus NT48, Nanotemper), evaluate the stability of the protein in the selected buffer.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Alcántara, R., Peñaranda, K., Mendoza, G., Nakamoto, J.A., Martins-Luna, J., Valle, J. del, Adaui, V., and Pohl, M. (2021a). Unlocking SARS-CoV-2 detection in low- and middle-income. Cell Rep. Methods. . <a class="external-link" href="https://doi.org/10.1016/j.crmeth.2021.100093">https://doi.org/10.1016/j.crmeth.2021.100093</a></p>
<p id="bib2">Alcántara, R., Peñaranda, K., Mendoza-Rojas, G., Nakamoto, J.A., Dueñas, E., Alvarez, D., Adaui, V., and Milón, P. (2021b). UnCovid: a versatile, low-cost, and open-source protocol for SARS-CoV-2 RNA detection. STAR Protoc. . <a class="external-link" href="https://doi.org/10.1016/j.xpro.2021.100878">https://doi.org/10.1016/j.xpro.2021.100878</a></p>
<p id="bib3">Bradford, M. (1976). A rapid and sensitive method for the quantitation of microgram quantities of protein utilizing the principle of protein-dye binding. Anal Biochem. <i>72</i>, 248-254. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref3">View at publisher</a></p>
<p id="bib4">Chan, W.-T., Verma, C.S., Lane, D.P., and Gan, S.K.-E. (2013). A comparison and optimization of methods and factors affecting the transformation of Escherichia coli. Biosci. Rep. <i>33</i>, e00086. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref4">View at publisher</a></p>
<p id="bib5">Chen, J.S., Ma, E., Harrington, L.B., Costa, M.D., Tian, X., Palefsky, J.M., and Doudna, J.A. (2018). CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science <i>360</i>, eaar6245. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref5">View at publisher</a></p>
<p id="bib6">Chien, A., Edgar, D.B., and Trela, J.M. (1976). Deoxyribonucleic acid polymerase from the extreme thermophile Thermus aquaticus. J. Bacteriol. <i>127</i>, 1550-1557. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref6">View at publisher</a></p>
<p id="bib7">Graham, T.G.W., Dugast-Darzacq, C., Dailey, G.M., Nguyenla, X.H., Dis, E.V., Esbin, M.N., Abidi, A., Stanley, S.A., Darzacq, X., and Tjian, R. (2021). Open-source RNA extraction and RT-qPCR methods for SARS-CoV-2 detection. PLoS One <i>16</i>, e0246647. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref7">View at publisher</a></p>
<p id="bib8">Inoue, H., Nojima, H., and Okayama, H. (1990). High efficiency transformation of Escherichia coli with plasmids. Gene <i>96</i>, 23-28. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref8">View at publisher</a></p>
<p id="bib9">Kapust, R.B., Tözsér, J., Fox, J.D., Anderson, D.E., Cherry, S., Copeland, T.D., and Waugh, D.S. (2001). Tobacco etch virus protease: mechanism of autolysis and rational design of stable mutants with wild-type catalytic proficiency. Protein Eng. Des. Sel <i>14</i>, 993-1000. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref9">View at publisher</a></p>
<p id="bib10">Kotewicz, M.L., D’Alessio, J.M., Driftmier, K.M., Blodgett, K.P., and Gerard, G.F. (1985). Cloning and overexpression of Moloney murine leukemia virus reverse transcriptase in Escherichia coli. Gene <i>35</i>, 249-258. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref10">View at publisher</a></p>
<p id="bib11">Kotewicz, M.L., Sampson, C.M., D’Alessio, J.M., and Gerard, G.F. (1988). Isolation of cloned Moloney murine leukemia virus reverse transcriptase lacking ribonuclease H activity. Nucleic Acids Res. <i>16</i>, 265-277. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref11">View at publisher</a></p>
<p id="bib12">Lawyer, F.C., Stoffel, S., Saiki, R.K., Myambo, K., Drummond, R., and Gelfand, D.H. (1989). Isolation, characterization, and expression in Escherichia coli of the DNA polymerase gene from Thermus aquaticus. J. Biol. Chem <i>264</i>, 6427-6437. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref12">View at publisher</a></p>
<p id="bib13">Madadlou, A., O’Sullivan, S., and Sheehan, D. (2016). Protein chromatography, methods and protocols. Methods Mol. Biol. <i>1485</i>, 365-373. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref13">View at publisher</a></p>
<p id="bib14">Mahalik, S., Sharma, A.K., and Mukherjee, K.J. (2014). Genome engineering for improved recombinant protein expression in Escherichia coli. Microb. Cell Fact <i>13</i>, 177. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref14">View at publisher</a></p>
<p id="bib15">Raynal, B., Lenormand, P., Baron, B., Hoos, S., and England, P. (2014). Quality assessment and optimization of purified protein samples: why and how?. Microb. Cell Fact <i>13</i>, 180. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref15">View at publisher</a></p>
<p id="bib16">Reischl, U., Polayes, D.A., Parks, T.D., Johnston, S.A., and Dougherty, W.G. (1997). Molecular diagnosis of infectious diseases. Methods Mol. Med. <i>13</i>, 169-183. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref16">View at publisher</a></p>
<p id="bib20">Sasse, J. and Gallagher, S. (2009). Staining Proteins in Gels. Current Protocols in Molecular Biology <i>85</i> (<i>1</i>). . <a class="external-link" href="https://doi.org/10.1002/0471142727.mb1006s85">https://doi.org/10.1002/0471142727.mb1006s85</a></p>
<p id="bib19">Schneider, C., Rasband, W., and Eliceiri, K. (2012). NIH Image to ImageJ: 25 years of image analysis. Nature Methods <i>9</i> (<i>7</i>), 671-675. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/optSg95QCPCTZ">View at publisher</a></p>
<p id="bib17">Syrový, I. and Hodný, Z. (1991). Staining and quantification of proteins separated by polyacrylamide gel electrophoresis. J. Chromatogr. B Biomed. Sci. Appl. <i>569</i>, 175-196. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref17">View at publisher</a></p>
<p id="bib18">Zetsche, B., Gootenberg, J.S., Abudayyeh, O.O., Slaymaker, I.M., Makarova, K.S., Essletzbichler, P., Volz, S.E., Joung, J., van der Oost, J., Regev, A., et al. (2015). Cpf1 is a single RNA-guided endonuclease of a class 2 CRISPR-cas system. Cell <i>163</i>, 759-771. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00605-5/sref18">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Pohl Milón (<a href="mailto:pmilon@upc.pe">pmilon@upc.pe</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents. All the plasmids used in this study are available in the Addgene catalog (See <a href="#key-resources-table">key resources table</a>).</p>
<h4>Data and code availability</h4>
<p>Original data have been deposited to Mendeley Data: .1 <a href="https://doi.org/10.17632/wynyf35xf3.2">https://doi.org/10.17632/wynyf35xf3.2</a>.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1144-Mmc1.xlsx">Data S1. Growth media and buffers used in this protocol, related to steps 1–10</a></p>
<h3>Acknowledgments</h3>
<p>We are very thankful to Dr. Marcos Milla for donating equipment that was used in this study and others. We would also like to thank all lab members of the Adaui and Milón groups for their help, support, and great working atmosphere. This work was supported by grants from the Peruvian <a href="https://doi.org/10.13039/501100010751">Fondo Nacional de Desarrollo Científico, Tecnológico y de Innovación Tecnológica</a> [070-2020-FONDECYT] to VA and [036-2019-FONDECYT-BM-INC.INV] and [154-2017-FONDECYT] to PM.</p>
<h3>Author contributions</h3>
<p>Conceptualization and methodology, R.A., K.P., J.A.N., L.C.-S., and P.M.; investigation, G.M.-R., A.S.-C., V.S.-V., L.T., K.P., and R.A.; formal analysis, R.A., K.P., and P.M., writing – original draft, V.S.-V, A.S.-C., K.P., R.A., and P.M., writing – review &amp; editing, V.S.-V., A.S.-C., G.M.-R., R.A., L.T., K.P., J.A.N., L.C.-S., V.A., and P.M.; visualization, V.S.-V. and A.S.-C.; supervision, R.A. and P.M.; project administration, P.M. and K.P.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>