<article>
<section>
<h1 id="header">Protocol for using NoBadWordsCombiner to merge and minimize “bad words” from BLAST hits against multiple eukaryotic gene annotation databases</h1>
<p><time datetime="2021-10-22">Published: October 22, 2021</time></p>
<p>Xi Zhang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">5</a>,<a href="#cor1">*</a></sup> Yining Hu,<sup><a href="#aff3">3</a></sup> and David Roy Smith<sup><a href="#aff4">4</a>,<a href="#fn2">6</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biochemistry and Molecular Biology, Dalhousie University, Halifax, NS B3H 4R2, Canada</p>
<p id="aff2"><sup>2</sup>Institute for Comparative Genomics, Dalhousie University, Halifax, NS B3H 4R2, Canada</p>
<p id="aff3"><sup>3</sup>Department of Computer Science, Western University, London, ON N6A 5B7, Canada</p>
<p id="aff4"><sup>4</sup>Department of Biology, Western University, London, ON N6A 5B7, Canada</p>
<p id="fn1"><sup>5</sup>Technical contact</p>
<p id="fn2"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:xzha25@uwo.ca">xzha25@uwo.ca</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:dsmit242@uwo.ca">dsmit242@uwo.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100888">10.1016/j.xpro.2021.100888</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Annotating protein-coding genes can be challenging, especially when searching for the best hits against multiple functional databases. This is partly because of "bad words" appearing as top hits, such as hypothetical or uncharacterized proteins. To help alleviate some of these issues, we designed a bioinformatics tool called NoBadWordsCombiner, which efficiently merges the hits from various databases, strengthening gene definitions by minimizing functional descriptions containing "bad words." Unlike other available tools, NoBadWordsCombiner is user friendly, but it does require users to have some general bioinformatics skills, including a basic understanding of the BLAST package and dash shell in Linux/Unix environments.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref12">Zhang et al. (2021a)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>NoBadWordsCombiner tool can merge and minimize “bad words” during gene annotation</li>
<li>Bad words include hypothetical and uncharacterized protein descriptions</li>
<li>Gene definitions are strengthened by Pfam domains and KEGG pathways</li>
<li>An overview of the gene annotations is summarized in a 23-column table</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1104-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Next-generation sequencing (NGS) technologies can generate huge amounts of molecular sequence data (<a href="#bib11">Yandell and Ence, 2012</a>). Functional annotations of protein-coding genes from NGS data can be easily acquired via database searches, including NCBI-NR (<a href="#bib9">Pruitt et al., 2005</a>), UniProtKB/Swiss-Prot (<a href="#bib3">Boutet et al., 2007</a>), and TrEMBL (<a href="#bib2">Boeckmann et al., 2003</a>). But the results of these searches often include ‘bad words’, such as best hits to hypothetical proteins or uncharacterized proteins, which can confuse the interpretation of gene annotation results. Indeed, it was reported that 20–30% of the annotations from assembled chlamydomonadalean nuclear genomes are represented by hypothetical proteins, including those from the <i>Chlamydomonas reinhardtii</i> genome (<a href="#bib12">Zhang et al., 2021a</a>). For various other recently sequenced genomes, the percentage of hypothetical proteins can be even higher (<a href="#bib6">Galperin, 2001</a>). It can be time-consuming to manually curate the functional hits from Basic Local Alignment Search Tool (BLAST) searches. This can be especially true if trying to minimize hits containing ‘bad words’ (e.g., hypothetical proteins) when the redundant hits have meaningful functional annotations (i.e., without ‘bad words’).</p>
<p>Currently, there are very few user-friendly bioinformatics tools for merging and minimizing ‘bad words’ during functional gene annotation, and those that are available typically involve custom programing scripts with a steep learning curve (<a href="#bib5">De Wit et al., 2012</a>). Here, we present NoBadWordsCombiner, an open-source, user-friendly bioinformatics web tool for efficiently merging and minimizing ‘bad words’ scanned from various functional annotation databases. This tool can plugin to external databases, such as the Kyoto Encyclopedia of Genes and Genomes (KEGG) (<a href="#bib7">Kanehisa and Goto, 2000</a>) and InterProScan (<a href="#bib10">Quevillon et al., 2005</a>), to strengthen the definition of gene annotations. NoBadWordsCombiner does require users to have some basic familiarity with bioinformatics. They must be comfortable with the BLAST package (<a href="#bib1">Altschul et al., 1997</a>), the dash shell in Linux/Unix environments, and inputting files from third-party tools, such as InterProScan (<a href="#bib10">Quevillon et al., 2005</a>) and KEGG (BlastKOALA and GhostKOALA) (<a href="#bib8">Kanehisa et al., 2016</a>).</p>
<p>Recently, we sequenced, assembled, and annotated the nuclear genome of the Antarctic green alga <i>Chlamydomonas</i> sp. UWO241 (<a href="#bib12">Zhang et al., 2021a</a>), hereafter referred to as UWO241. During our analysis of this genome, we designed and applied the NoBadWordsCombiner tool during the functional annotation stage, which greatly minimized descriptions containing ‘bad words’. The protocol presented here describes how to use NoBadWordsCombiner for merging and minimizing ‘bad words’ from eukaryotic gene annotation databases. The model psychrophilic green alga UWO241 is used as a case-study for this goal.</p>
<h3 id="sec1.1">Overview</h3>
<p>NoBadWordsCombiner merges the functional gene annotation BLAST hits from NCBI nr, UniProtKB/Swiss-Prot, and TrEMBL database searches. Specifically, it removes redundancy from descriptions containing hits to hypothetical or uncharacterized proteins (not including instances when all hits are hypothetical/uncharacterized). Then, the definition of the combined hits is strengthened via protein functional domains and pathway information based on data from the InterPro and KEGG databases. Finally, the overview of the gene annotations with the minimized ‘bad words’ is summarized in a mega table.</p>
<h3 id="sec1.2">Downloading the software and prerequisites</h3>
<p>NoBadWordsCombiner can be operated on the web (<a href="http://hsdfinder.com/combiner/">http://hsdfinder.com/combiner/</a>) or the local environment (Linux and Python 3) after downloading the software package from GitHub (<a href="https://github.com/zx0223winner/%20NoBadWordsCombiner">https://github.com/zx0223winner/ NoBadWordsCombiner</a>). To run locally, pre-installed Python (preferably Python 3) and Linux (e.g., Ubuntu 20.04 LTS) environments are required. The BLAST and InterProScan software packages as well as the online KEGG pathways tools BlastKOALA and GhostKOALA (<a href="#bib8">Kanehisa et al., 2016</a>) can be accessed via the links in the <a href="#key-resources-table">key resources table</a>.</p>
<div class="note">
<span class="note-title">Note:</span> A minimum specification requirement is a computer with 2 cores, 4 GB of RAM, and 256 GB storage, which should allow the ‘bad words’ to be merged and minimized within a few minutes.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>
<i>Chlamydomonas</i> sp. UWO241 (renamed <i>Chlamydomonas priscuii</i>)</td>
<td>(<a href="#bib12">Zhang et al., 2021a</a>)</td>
<td>Genbank: GCA_016618255</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BLAST v2.2.26</td>
<td>(<a href="#bib1">Altschul et al., 1997</a>)</td>
<td><a href="http://ftp//ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/">ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/</a></td>
</tr>
<tr>
<td>UniProtKB/Swiss-Prot</td>
<td>(<a href="#bib3">Boutet et al., 2007</a>)</td>
<td><a href="https://www.uniprot.org/uniprot/?query=reviewed:yes">https://www.uniprot.org/uniprot/?query=reviewed:yes</a></td>
</tr>
<tr>
<td>UniProtKB/TrEMBL</td>
<td>(<a href="#bib2">Boeckmann et al., 2003</a>)</td>
<td><a href="https://www.uniprot.org/uniprot/?query=reviewed:no">https://www.uniprot.org/uniprot/?query=reviewed:no</a></td>
</tr>
<tr>
<td>NCBI-NR</td>
<td>(<a href="#bib9">Pruitt et al., 2005</a>)</td>
<td><a href="https://www.ncbi.nlm.nih.gov/refseq/">https://www.ncbi.nlm.nih.gov/refseq/</a></td>
</tr>
<tr>
<td>InterProScan v4.7</td>
<td>(<a href="#bib10">Quevillon et al., 2005</a>)</td>
<td><a href="http://www.ebi.ac.uk/interpro/download/">http://www.ebi.ac.uk/interpro/download/</a></td>
</tr>
<tr>
<td>BlastKOALA or GhostKOALA</td>
<td>(<a href="#bib7">Kanehisa and Goto, 2000</a>; <a href="#bib8">Kanehisa et al., 2016</a>)</td>
<td><a href="https://www.kegg.jp">https://www.kegg.jp</a></td>
</tr>
<tr>
<td>NoBadWordsCombiner</td>
<td>This article</td>
<td><a href="http://hsdfinder.com/combiner">http://hsdfinder.com/combiner</a></td>
</tr>
<tr>
<td>Python 3</td>
<td>N/A</td>
<td><a href="https://www.python.org/downloads/">https://www.python.org/downloads/</a></td>
</tr>
<tr>
<td>Django v3.1.5</td>
<td>N/A</td>
<td><a href="https://www.djangoproject.com/download/">https://www.djangoproject.com/download/</a></td>
</tr>
<tr>
<td>pandas v1.2.2</td>
<td>N/A</td>
<td><a href="https://pandas.pydata.org">https://pandas.pydata.org</a></td>
</tr>
<tr>
<td>blastxml_to_tabular.py</td>
<td>(<a href="#bib4">Cock et al., 2015</a>)</td>
<td><a href="https://github.com/peterjc/galaxy_blast/blob/master/tools/ncbi_blast_plus/blastxml_to_tabular.py">https://github.com/peterjc/galaxy_blast/blob/master/tools/ncbi_blast_plus/blastxml_to_tabular.py</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>The software implementation was written in Python 3 using the following custom scripts and platforms: <i>NoBadWordsCombiner.py</i>, which enables the ‘bad words’ to be merged and minimized from BLAST hits against multiple eukaryotic gene annotation databases and protein signature databases (e.g., Pfam); Django (3.1.5), a Python-based web platform, which maintains the web server; and pandas (1.2.2), the software library used for manipulating the data. <i>Blastxml_to_tabular.py</i> (<a href="#bib4">Cock et al., 2015</a>) is a custom Python script that can convert a BLAST XML file to the desired tabular output. The NCBI-NR and UniProtKB/TrEMBL databases, including the gene annotations, are computationally analyzed, whereas the UniProtKB/Swiss-Prot database is manually curated and, thus, contains fewer annotations of hypothetical proteins. A full list of the utilized packages and database, including links, can be found in the <a href="#key-resources-table">key resources table</a>. The full NoBadWordsCombiner source code can be found in the GitHub repository. A useful hands-on tutorial (Online_NoBadWordsCombiner Tutorial.pdf) can also be accessed under the tutorial directory of GitHub.</p>
<p>The test input data consist of BLAST and protein signature results from InterProScan (<a href="#bib10">Quevillon et al., 2005</a>).</p>
<div class="note">
<span class="note-title">Note:</span> Five mandatory tab-delimited tables are needed to run the tool. The first and second input documents of the NCBI-NR and SwissProt database BLAST results have 14 columns (<a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>). These two tables are parsed from the local BLAST results via a custom Python script (<i>Blastxml_to_tabular.py</i>). The third and fourth input files were designed as a 1-column gene name list file and a 2-column KEGG annotation file, respectively (<a href="#tbl3">Tables 3</a> and <a href="#tbl4">4</a>). The fifth input document of the InterProScan results has 13 columns (<a href="#tbl5">Table 5</a>). The KO accession with each gene model identifier was retrieved from the KEGG database (<a href="#bib7">Kanehisa and Goto, 2000</a>). In the following step-by-step protocol, we use the deduced protein sequences from the UWO241 genome annotation (<a href="#bib12">Zhang et al., 2021a</a>) to show how to generate these tables.</div>
<table id="tbl1" class="dense-table">
<caption>Table 1. Input file example of NCBI nr database BLAST result</caption>
<thead>
<tr>
<th>QueryAcc</th>
<th>Query_Length</th>
<th>HitDescription</th>
<th>HitName</th>
<th>HitLength</th>
<th>HitBits</th>
<th>HSP_rank</th>
<th>%ID</th>
<th>eValue</th>
<th>Query_Start</th>
<th>Query_End</th>
<th>Hit_start</th>
<th>Hit_end</th>
<th>HSP_length</th>
</tr>
</thead>
<tbody>
<tr>
<td>g1.t1</td>
<td>817</td>
<td>hypothetical protein CEUSTIGMA_g3421.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995578|dbj|GAX75978.1|</td>
<td>1443</td>
<td>260.766</td>
<td>1</td>
<td>54.2635659</td>
<td>1.41E-75</td>
<td>10</td>
<td>774</td>
<td>11</td>
<td>268</td>
<td>258</td>
</tr>
<tr>
<td>g2.t1</td>
<td>399</td>
<td>ankyrin, partial [Anaeromyces robustus]</td>
<td>gi|1183350135|gb|ORX78377.1|</td>
<td>235</td>
<td>65.4698</td>
<td>1</td>
<td>40.2298851</td>
<td>3.61E-10</td>
<td>19</td>
<td>279</td>
<td>18</td>
<td>96</td>
<td>87</td>
</tr>
<tr>
<td>g3.t1</td>
<td>3567</td>
<td>hypothetical protein CEUSTIGMA_g3419.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995576|dbj|GAX75976.1|</td>
<td>1103</td>
<td>172.17</td>
<td>1</td>
<td>38.4615385</td>
<td>1.15E-39</td>
<td>805</td>
<td>1674</td>
<td>330</td>
<td>597</td>
<td>299</td>
</tr>
<tr>
<td>g4.t1</td>
<td>963</td>
<td>hypothetical protein CEUSTIGMA_g3418.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995575|dbj|GAX75975.1|</td>
<td>623</td>
<td>310.457</td>
<td>1</td>
<td>89.5061728</td>
<td>1.17E-97</td>
<td>469</td>
<td>954</td>
<td>172</td>
<td>333</td>
<td>162</td>
</tr>
<tr>
<td>g6.t1</td>
<td>291</td>
<td>hypothetical protein CHLRE_10g421079v5 [Chlamydomonas reinhardtii]</td>
<td>gi|1335042461|gb|PNW77074.1|</td>
<td>103</td>
<td>82.8037</td>
<td>1</td>
<td>58.3333333</td>
<td>1.66E-18</td>
<td>103</td>
<td>282</td>
<td>34</td>
<td>93</td>
<td>60</td>
</tr>
<tr>
<td>g7.t1</td>
<td>7908</td>
<td>hypothetical protein CEUSTIGMA_g3945.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238994727|dbj|GAX76500.1|</td>
<td>2934</td>
<td>156.377</td>
<td>1</td>
<td>32.6785714</td>
<td>7.48E-34</td>
<td>6334</td>
<td>7761</td>
<td>2313</td>
<td>2860</td>
<td>560</td>
</tr>
<tr>
<td>g9.t1</td>
<td>471</td>
<td>hypothetical protein CEUSTIGMA_g3416.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995573|dbj|GAX75973.1|</td>
<td>139</td>
<td>164.466</td>
<td>1</td>
<td>62.1212121</td>
<td>3.00E-49</td>
<td>76</td>
<td>468</td>
<td>11</td>
<td>139</td>
<td>132</td>
</tr>
<tr>
<td>g10.t1</td>
<td>1827</td>
<td>hypothetical protein GPECTOR_108g190 [Gonium pectorale]</td>
<td>gi|1004134917|gb|KXZ42995.1|</td>
<td>463</td>
<td>331.257</td>
<td>1</td>
<td>78.8288288</td>
<td>1.18E-103</td>
<td>580</td>
<td>1245</td>
<td>48</td>
<td>269</td>
<td>222</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
<table id="tbl2" class="dense-table">
<caption>Table 2. Input file example of SwissProt database BLAST result</caption>
<thead>
<tr>
<th>QueryAcc</th>
<th>Query_Length</th>
<th>HitDescription</th>
<th>HitName</th>
<th>HitLength</th>
<th>HitBits</th>
<th>HSP_rank</th>
<th>%ID</th>
<th>eValue</th>
<th>Query_Start</th>
<th>Query_End</th>
<th>Hit_start</th>
<th>Hit_end</th>
<th>HSP_length</th>
</tr>
</thead>
<tbody>
<tr>
<td>g2.t1</td>
<td>399</td>
<td>2-5A-dependent ribonuclease OS=Mus musculus OX=10090 GN=Rnasel PE=1 SV=2</td>
<td>sp|Q05921|RN5A_MOUSE</td>
<td>735</td>
<td>48.1358</td>
<td>1</td>
<td>34.8837209</td>
<td>4.14E-06</td>
<td>25</td>
<td>267</td>
<td>125</td>
<td>206</td>
<td>86</td>
</tr>
<tr>
<td>g3.t1</td>
<td>3567</td>
<td>DNA mismatch repair protein MSH6 OS=Arabidopsis thaliana OX=3702 GN=MSH6 PE=1 SV=2</td>
<td>sp|O04716|MSH6_ARATH</td>
<td>1324</td>
<td>53.5286</td>
<td>1</td>
<td>41.8181818</td>
<td>1.61E-05</td>
<td>379</td>
<td>543</td>
<td>121</td>
<td>175</td>
<td>55</td>
</tr>
<tr>
<td>g4.t1</td>
<td>963</td>
<td>Eukaryotic peptide chain release factor GTP-binding subunit ERF3A OS=Homo sapiens OX=9606 GN=GSPT1 PE=1 SV=1</td>
<td>sp|P15170|ERF3A_HUMAN</td>
<td>499</td>
<td>234.958</td>
<td>1</td>
<td>72.2972973</td>
<td>2.94E-72</td>
<td>511</td>
<td>954</td>
<td>69</td>
<td>216</td>
<td>148</td>
</tr>
<tr>
<td>g9.t1</td>
<td>471</td>
<td>Thylakoid-associated protein slr0729 OS=Synechocystis sp. (strain PCC 6803 / Kazusa) OX=1111708 GN=slr0729 PE=4 SV=1</td>
<td>sp|P72673|Y729_SYNY3</td>
<td>101</td>
<td>47.7506</td>
<td>1</td>
<td>29.4736842</td>
<td>1.49E-06</td>
<td>187</td>
<td>468</td>
<td>11</td>
<td>99</td>
<td>95</td>
</tr>
<tr>
<td>g10.t1</td>
<td>1827</td>
<td>Threonylcarbamoyl-AMP synthase OS=Schizosaccharomyces pombe (strain 972 / ATCC 24843) OX=284812 GN=sua5 PE=3 SV=1</td>
<td>sp|O94530|SUA5_SCHPO</td>
<td>408</td>
<td>217.238</td>
<td>1</td>
<td>50.8264463</td>
<td>3.67E-63</td>
<td>580</td>
<td>1242</td>
<td>58</td>
<td>299</td>
<td>242</td>
</tr>
<tr>
<td>g15.t1</td>
<td>270</td>
<td>Protein transport protein Sec61 subunit beta OS=Chlamydomonas reinhardtii OX=3055 GN=SEC61B PE=1 SV=1</td>
<td>sp|A8I6P9|SC61B_CHLRE</td>
<td>89</td>
<td>65.4698</td>
<td>1</td>
<td>53.7037037</td>
<td>2.15E-14</td>
<td>106</td>
<td>261</td>
<td>36</td>
<td>89</td>
<td>54</td>
</tr>
<tr>
<td>g16.t1</td>
<td>897</td>
<td>Probable prolyl 4-hydroxylase 4 OS=Arabidopsis thaliana OX=3702 GN=P4H4 PE=2 SV=1</td>
<td>sp|Q8LAN3|P4H4_ARATH</td>
<td>298</td>
<td>157.147</td>
<td>1</td>
<td>41.0788382</td>
<td>9.49E-45</td>
<td>1</td>
<td>708</td>
<td>69</td>
<td>289</td>
<td>241</td>
</tr>
<tr>
<td>g17.t1</td>
<td>1104</td>
<td>GATA transcription factor 3 OS=Arabidopsis thaliana OX=3702 GN=GATA3 PE=2 SV=2</td>
<td>sp|Q8L4M6|GATA3_ARATH</td>
<td>269</td>
<td>62.003</td>
<td>1</td>
<td>56.097561</td>
<td>1.17E-09</td>
<td>79</td>
<td>201</td>
<td>171</td>
<td>211</td>
<td>41</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
<table id="tbl3">
<caption>Table 3. Input file example of gene name list</caption>
<thead>
<tr>
<th>Gene name</th>
</tr>
</thead>
<tbody>
<tr>
<td>g1.t1</td>
</tr>
<tr>
<td>g2.t1</td>
</tr>
<tr>
<td>g3.t1</td>
</tr>
<tr>
<td>g4.t1</td>
</tr>
<tr>
<td>g5.t1</td>
</tr>
<tr>
<td>g6.t1</td>
</tr>
<tr>
<td>g7.t1</td>
</tr>
<tr>
<td>…</td>
</tr>
</tbody>
</table>
<table id="tbl4">
<caption>Table 4. Input file example of KO accession with each gene model identifier retrieved from the KEGG database</caption>
<thead>
<tr>
<th>Gene identifier</th>
<th>KO accession</th>
</tr>
</thead>
<tbody>
<tr>
<td>g59.t1</td>
<td>K10849</td>
</tr>
<tr>
<td>g60.t2</td>
<td>K17087</td>
</tr>
<tr>
<td>g61.t2</td>
<td>N/A</td>
</tr>
<tr>
<td>g62.t1</td>
<td>N/A</td>
</tr>
<tr>
<td>g63.t2</td>
<td>N/A</td>
</tr>
<tr>
<td>g64.t1</td>
<td>N/A</td>
</tr>
<tr>
<td>g65.t1</td>
<td>K15172</td>
</tr>
<tr>
<td>g66.t1</td>
<td>K02519</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
<table id="tbl5" class="dense-table">
<caption>Table 5. Input file example of InterProScan database result</caption>
<thead>
<tr>
<th>Protein accession</th>
<th>Unique code</th>
<th>Sequence length</th>
<th>Protein signature</th>
<th>Signature accession</th>
<th>Signature description</th>
<th>Start location</th>
<th>Stop location</th>
<th>E-value</th>
<th>Status</th>
<th>Date</th>
<th>InterPro accession</th>
<th>InterPro description</th>
</tr>
</thead>
<tbody>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>SUPERFAMILY</td>
<td>SSF82153</td>
<td>N/A</td>
<td>129</td>
<td>260</td>
<td>9.42E-10</td>
<td>T</td>
<td>31-03-2019</td>
<td>IPR036378</td>
<td>FAS1 domain superfamily</td>
</tr>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>ProSiteProfiles</td>
<td>PS50213</td>
<td>FAS1/BIgH3 domain profile.</td>
<td>111</td>
<td>257</td>
<td>9.579</td>
<td>T</td>
<td>31-03-2019</td>
<td>IPR000782</td>
<td>FAS1 domain</td>
</tr>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>Pfam</td>
<td>PF02469</td>
<td>Fasciclin domain</td>
<td>123</td>
<td>259</td>
<td>5.80E-09</td>
<td>T</td>
<td>31-03-2019</td>
<td>IPR000782</td>
<td>FAS1 domain</td>
</tr>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>ProSitePatterns</td>
<td>PS00183</td>
<td>Ubiquitin-conjugating enzymes active site.</td>
<td>69</td>
<td>84</td>
<td>-</td>
<td>T</td>
<td>31-03-2019</td>
<td>IPR023313</td>
<td>Ubiquitin-conjugating enzyme, active site</td>
</tr>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>SMART</td>
<td>SM00212</td>
<td>N/A</td>
<td>2</td>
<td>148</td>
<td>1.10E-36</td>
<td>T</td>
<td>31-03-2019</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>PANTHER</td>
<td>PTHR44511</td>
<td>N/A</td>
<td>2</td>
<td>119</td>
<td>1.50E-59</td>
<td>T</td>
<td>31-03-2019</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>g5250.t1</td>
<td>f246997202ceeb0ebfd5ea2f454be9a2</td>
<td>262</td>
<td>SMART</td>
<td>SM00554</td>
<td>N/A</td>
<td>145</td>
<td>260</td>
<td>7.20E-07</td>
<td>T</td>
<td>31-03-2019</td>
<td>IPR000782</td>
<td>FAS1 domain</td>
</tr>
<tr>
<td>g15700.t1</td>
<td>e9641f2405b85bc4c48a85029514acf0</td>
<td>799</td>
<td>MobiDBLite</td>
<td>mobidb-lite</td>
<td>consensus disorder prediction</td>
<td>347</td>
<td>361</td>
<td>-</td>
<td>T</td>
<td>31-03-2019</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparing the NCBI-NR and UniProtKB/Swiss-Prot protein BLAST-search result files</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼2 days (depending on the amount of the data, computing power, and Internet speed.)</div>
<p>Upload protein BLAST-search result files from your genome of interest in tab-separated values (tsv) format as the input files (<a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>) of NoBadWordsCombiner. This protocol will go over how to acquire local BLAST-search results via an example FASTA file. The example file as well as the hands-on tutorial (Online_NoBadWordsCombiner Tutorial.pdf) can be acquired from GitHub under the tutorial directory (<a href="#fig1">Figure 1</a>A).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1104-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The NoBadWordsCombiner home page</div>
<p>(A) The GitHub web interface of this tool.</p>
<p>(B) Uploading the necessary input files.</p>
<p>(C) The interface of running the tool.</p>
<p>(D) The output example of the tool.</p>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> You can ignore this step and proceed with your own protein data set if you know how to acquire the appropriate BLASTP search results.</div>
<ol>
<li>Download the BLAST package and FASTA file. A BLAST-search result example file is found in the ZIP file in the GitHub ‘tutorial’ directory under the name NoBadWordsCombiner_file_examples.zip (<a href="#fig1">Figure 1</a>A). Also, the ‘blastdbv5-user-guide.pdf’ document in the GitHub ‘tutorial’ directory contains complementary vignettes to help guide users.
<ol type="a">
<li>Download the BLAST Package via <a href="https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/">https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/</a>. Please select the appropriate version based on your computer operating system (Windows, MacOS, or Linux)</li>
<li>Unzip the ‘NoBadWordsCombiner_file_examples.zip’ file; the file named ‘Chlamydomonas_UWO241_protein.fasta’ is the example FASTA file.</li>
</ol>
</li>
</ol>
<div class="textbox">
<p><code># Display the first ten rows of the FASTA file.</code></p>
<p><code>&gt; head Chlamydomonas_UWO241_protein.fasta</code></p>
<p><code>&gt;g1.t1</code></p>
<p><code>MAATVENVVERVKSFSSVVRGVKSGKPDGATTQLVQETIEILATYCDFEEVVPV</code></p>
<p><code>CLKFLDEVLTAAPQTSTLIRLEGGAK</code></p>
<p><code>IFPSIIRNFMGVDASILALCAKVMCKCASGSPAMQHHLVKEKGLPTLLLSCCSA</code></p>
<p><code>HAGEPAVVGPLLEVLVALARYSKGAT</code></p>
<p><code>ALSNANLVHACKELLVGLMGHWHAFGMVLKLIKSVMKHEGPCLAALKAGEVVRL</code></p>
<p><code>LLGVARLVSRMPDQRKLLKRASRTLW</code></p>
<p><code>VLSQRSLHPLPEMELNWPHTHTHTHTHTHTHT</code></p>
<p><code>&gt;g2.t1</code></p>
<p><code>MMMLAYRFGFTTLMYATVKGHADAMRLLLKHPSADTAAMMMLTDIRGCTALMFA</code></p>
<p><code>AQDGHVNAIRMLLDHPSADVAARIAV</code></p>
<p><code>RSTVGISALTSAAGFAAGQPTLSRRASPARSCTPLLFLLRRVAVEPQLCDTQ</code></p>
<p><code>&gt;g3.t1</code></p>
<p><code>MVPTDGARHGWTATSLPAILGAASHAKITVQQLVVGGPPPSCPYGPEIVGRSLS</code></p>
<p><code>LFSKSAKTWDRAPGGVVSAFCAATGE</code></p>
</div>
<ol start="2">
<li>Set up the manually curated UniProtKB/Swiss-Prot database and computationally calculated NCBI-NR database.
<ol type="a">
<li>The ‘uniprot_sprot.fasta.gz’ file can be downloaded directly from <a href="https://www.uniprot.org/downloads">https://www.uniprot.org/downloads</a>. When downloading, choose Reviewed (Swiss-Prot) in FASTA format under the parent directory. The NCBI-NR BLAST v5 databases can be accessed via <a href="https://ftp.ncbi.nlm.nih.gov/blast/db">https://ftp.ncbi.nlm.nih.gov/blast/db</a>. Some necessary files (e.g., nr.00.tar.gz, nr.00.tar.gz.md5, and nr.01.tar.gz) can be automatically downloaded via a custom Perl script at step 2c.</li>
<li>The ‘makeblastdb’ command will construct a protein database by taking in the FASTA file with the parameter (-in), setting up the database type (e.g., protein) with the parameter (-dbtype protein), and titling the name of database (e.g., uniport_prot_database) with parameters (-title database_name). The ‘-out’ option will yield the database output name (e.g., uniport_db).
<div class="textbox">
<p><code># Note: if your FASTA data are</code> <code>represented by</code> <code>nucleotide</code><code>s</code><code>, you can</code></p>
<p><code>change the database type with the parameter (-dbtype</code></p>
<p><code>nucl)</code></p>
<p><code>&gt; ./makeblastdb -in uniprot_sprot.fasta -dbtype prot -</code></p>
<p><code>title uniprot_prot_database -out uniprot_db</code></p>
</div>
</li>
<li>To download the NCBI-NR v5 databases, use the Perl script ‘update_blastdb.pl’, which is in the downloaded BLAST+ package (<a href="https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/">https://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/</a>). # This command will download the NCBI-NR database (<a href="https://ftp.ncbi.nlm.nih.gov/blast/db/v5">https://ftp.ncbi.nlm.nih.gov/blast/db/v5</a>) with the name ‘nr’ without using the makeblastdb command to redo it. It could take minutes to hours for processing, depending on the speed of the Internet.
<ol type="i">
<li>Users can first check all available databases via the command below.
<div class="textbox">
<p><code>&gt; perl update_blastdb.pl --blastdb_version 5 –</code></p>
<p><code>showall</code></p>
<p><code># This will give the results like this:</code></p>
<p><code># Connected to NCBI; downloading BLASTDBv5</code></p>
<p><code># human_genome</code></p>
<p><code># landmark</code></p>
<p><code># …</code></p>
</div>
</li>
<li>Users can then run the command below to download the nr database, which includes 55 volumes of data (&gt;100 Gb). Alternatively, users can manually download these 110 files (i.e., nr.00.tar.gz, nr.00.tar.gz.md5, etc.) from the link: <a href="https://ftp.ncbi.nlm.nih.gov/blast/db/v5">https://ftp.ncbi.nlm.nih.gov/blast/db/v5</a>.
<div class="textbox">
<p><code>&gt; perl update_blastdb.pl --blastdb_version 5</code></p>
<p><code>nr --decompress</code></p>
<p><code># This will bring the results like this:</code></p>
<p><code># Connected to NCBI; downloading BLASTDBv5</code></p>
<p><code># Downloading nr (55 volumes) ...</code></p>
<p><code># Downloading nr.00.tar.gz...</code></p>
<p><code># Downloading nr.00.tar.gz.md5</code></p>
<p><code>#...</code></p>
</div>
</li>
</ol>
</li>
<li>Use the BLASTP search option to blast the amino acid sequences against the uniport_db and nr_v5 databases. The BLASTP command can do the protein similarity search by searching the query file (Chlamydomonas_UWO241_protein.fasta) against the protein database created from the former step with default parameters, such as ‘-evalue’ (indicating the significance of the BLAST hits), ‘-outfmt’ (meaning the format of the BLAST result), and ‘-out’ (telling the file name of the output file; e.g., BLASTP_UWO241_uniprot.xml).
<div class="textbox">
<p><code>&gt;./blastp -query Chlamydomonas_UWO241_protein.fasta -db</code></p>
<p><code>uniprot_db -out BLASTP_UWO241_uniprot.xml -evalue 1e-5 -</code></p>
<p><code>outfmt 5</code></p>
</div>
</li>
<li>The BLAST XML file (-outfmt 5) can include useful information compared to the BLAST Tabular file (-outfmt 6), such as the aligned sequence, the sequence of the hit, and the description of hits in the database. However, the XML format is not human-readable. Users will need to employ a commonly used parser (<i>Blastxml_to_tabular.py</i>) (<a href="#bib4">Cock et al., 2015</a>), which is a custom python script, to convert a BLAST XML file to a desired tabular output (tab-delimited file).
<ol type="i">
<li>Users can first download the script <i>Blastxml_to_tabular.py</i> via the link from the <a href="#key-resources-table">key resources table</a>. Then run the command below.
<div class="textbox">
<p><code>&gt; python blastxml_to_tabular.py -c</code></p>
<p><code>qseqid,qlen,salltitles,sseqid,slen,bitscore,qframe,</code></p>
<p><code>pident,evalue,qstart,qend,sstart,send,length</code></p>
<p><code>BLASTP_UWO241_uniprot.xml &gt;</code></p>
<p><code>BLASTP_UWO241_uniprot.tsv</code></p>
</div>
</li>
<li>The parameters behind the option ‘-c’ in the <i>Blastxml_to_tabular.py</i> script will yield desired columns in the output tab-delimited file. For example, ‘qseqid’ refers to query sequence ID and ‘qlen’ refers to query length. These desired parameters will create a 14-column table (e.g., <a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>).</li>
</ol>
</li>
<li>To speed up the NCBI BLAST search, users can specify one or more (comma-delimited) taxids, or a file containing multiple taxids on the command-line. For example, to search against the <i>Chlamydomonas reinhardtii</i> (Taxonomy ID: 147) and <i>Chlamydomonas eustigma</i> (Taxonomy ID: 57939) nuclear genomes, use the command shown below. Also, we recommend that users browse the ‘blastdbv5-user-guide.pdf’ document in the GitHub ‘tutorial’ directory to familiarize themselves with the creation of either taxids or taxidlist.
<div class="textbox">
<p><code>&gt;./blastp -query Chlamydomonas_UWO241_protein.fasta -db</code></p>
<p><code>nr –taxids 147,57939 -out BLASTP_UWO241_NCBI-NR.xml -</code></p>
<p><code>evalue 1e-5 -outfmt 5</code></p>
<p><code># Multiple taxonomy IDs are delimited by ','.</code></p>
<p><code>#Similar to Step 2e, the BLASTP_UWO241_NCBI-NR.xml file</code></p>
<p><code>will be converted to BLASTP_UWO241_NCBI-NR.tsv via the</code></p>
<p><code>command below.</code></p>
</div>
<div class="textbox">
<p><code>&gt; python blastxml_to_tabular.py -c</code></p>
<p><code>qseqid,qlen,salltitles,sseqid,slen,bitscore,qframe,pide</code></p>
<p><code>nt,evalue,qstart,qend,sstart,send,length</code></p>
<p><code>BLASTP_UWO241_NCBI-NR.xml &gt; BLASTP_UWO241_NCBI-NR.tsv</code></p>
<p><code># The option ‘-c’ refers to the desired output columns</code></p>
<p><code>which can be set in comma-delimited format (e.g.,</code></p>
<p><code>qseqid,qlen,salltitles)</code></p>
</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to use the BLASTP option, which allows for greater sensitivity. The BLAST output parameter must be in format 5. Users can adjust the parameter of the E-value, but we recommend that it be no greater than 1e-5 (to ensure accurate predictions). <a href="#sec6.1">Troubleshooting 1</a>.</div>
</li>
</ol>
</li>
</ol>
<ol start="3">
<li>This will give two BLAST result files formed by 14-column spreadsheets, including key information, such as query name and percentage identity.</li>
<li>The 14-column explanation of parsed BLAST search result files (<a href="#tbl1">Tables 1</a> and <a href="#tbl2">2</a>).
<ol type="a">
<li>QueryAcc (e.g., g2.t1)</li>
<li>Query_Length (e.g., 399)</li>
<li>HitDescription (e.g., ankyrin, partial [Anaeromyces robustus])</li>
<li>HitName (e.g., gi|1183350135|gb|ORX78377.1|)</li>
<li>HitLength (e.g., 235)</li>
<li>HitBits (e.g., 65.4698)</li>
<li>HSP_rank (e.g., 1)</li>
<li>%ID (e.g., 40.2298851)</li>
<li>eValue (e.g., 3.61E-10)</li>
<li>Query_Start (e.g., 19)</li>
<li>Query_end (e.g., 279)</li>
<li>Hit_start (e.g., 18)</li>
<li>Hit_end (e.g., 96)</li>
<li>HSP_length (e.g., 87)</li>
</ol>
</li>
<li>If users want to upload different BLAST files or mistakenly submitted an incorrect file, they can reload the browser page or simply overlap with another file. <a href="#sec6.3">Troubleshooting 2</a>
</li>
</ol>
<h3 id="sec3.2">Preparing the gene name list and a gene list with KO annotations from the KEGG database</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h (depending on the queuing time of GhostKOALA)</div>
<p>Users can retrieve the third and fourth files from the genome FASTA file and the KEGG database, which include the correlation of the KO accession with each gene model identifier (<a href="#fig1">Figure 1</a>B). The gene name file is the baseline to merge all the different functional annotations.</p>
<ol start="6">
<li>Users can acquire the gene name list file by the following command lines. We use the FASTA file from the UWO241 genome as an example (<a href="#tbl3">Table 3</a>). # ‘grep’ is the command used in the dash shell to grasp each line containing the word pattern of ‘&gt;’. ‘sed’ is used to substitute all the ‘&gt;’ into none, which generates a clean name list file.
<ol type="a">
<li>Users can first test the function of grep via the command below:
<div class="textbox">
<p><code>&gt; grep “&gt;” Chlamydomonas_UWO241_protein.fasta| wc</code></p>
<p><code># This should turn out the results as follows:</code></p>
<p><code># 16325 16325 168617</code></p>
</div>
</li>
<li>Users can then carry out the following step to acquire a gene name file.
<div class="textbox">
<p><code>&gt; grep “&gt;” Chlamydomonas_UWO241_protein.fasta | sed</code></p>
<p><code>‘s/&gt;//g’ &gt; UWO241-gene_name_list.txt</code></p>
</div>
As for a gene list with KO annotation, users have the option to use the GhostKOALA (genome size ≥ 300MB) or BlastKOALA analysis tool of KEGG to acquire the KO annotation file of the genome (<a href="https://www.kegg.jp/ghostkoala/">https://www.kegg.jp/ghostkoala/</a>). Below, we provide the necessary steps for using the tools:</li>
</ol>
</li>
</ol>
<ol start="7">
<li>BlastKOALA accepts a smaller dataset and is suitable for annotating high-quality genomes.
<ol type="a">
<li>Upload the query amino acid sequences in FASTA format.</li>
<li>Enter the taxonomy group of your genome.</li>
<li>Enter the KEGG GENES database file to be searched.</li>
<li>Enter your email address. An email will be sent to you for confirmation of your input data. Click the link in the email to initiate your job; you will receive another email once it is finished.</li>
</ol>
</li>
<li>GhostKOALA accepts larger datasets (e.g., 300 Mb) and is suitable for annotating metagenomes.
<ol type="a">
<li>Upload the query amino acid sequences in FASTA format.</li>
<li>Enter the KEGG GENES database file to be searched.</li>
<li>Enter your email address. Same as above (7d).</li>
</ol>
</li>
<li>From the email link of KEGG, users can download the gene list with the associated KO annotations. The format of the output file is referred to in <a href="#tbl4">Table 4</a>. Explanation of the 2-column input file for KO accession (<a href="#tbl4">Table 4</a>):
<ol type="a">
<li>Gene identifier (e.g., g59.t1)</li>
<li>KO accession (e.g., K10849)</li>
</ol>
</li>
<li>Use the GhostKOALA or BlastKOALA analysis tool of KEGG to acquire the KO annotation file of your genome (<a href="https://www.kegg.jp/ghostkoala/">https://www.kegg.jp/ghostkoala/</a>). We provide an example of a KO annotation file under the GitHub directory of the tutorial with the name NoBadWordsCombiner_file_examples.zip. <a href="#sec6.5">Troubleshooting 3</a>
</li>
</ol>
<h3 id="sec3.3">Preparing the InterProScan search result file</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 h</div>
<p>Upload an InterProScan search result file of your genome in tab-delimited format as the fifth input file (<a href="#tbl5">Table 5</a>). Users must individually download and install InterProScan to acquire the input file for the NoBadWordsCombiner tool. The latest InterProScan documentation can be found via the link <a href="https://interproscan-docs.readthedocs.io/en/latest/index.html">https://interproscan-docs.readthedocs.io/en/latest/index.html</a>. Here, we provide the necessary steps for using InterProScan:</p>
<ol start="11">
<li>Installation requirements:
<ol type="a">
<li>InterProScan is developed to run on Linux and no versions are planned for Windows or Apple (MAC OS X) operating systems.</li>
<li>Software requirements: 64-bit Linux; Perl 5; Python 3; Java JDK/JRE version 11.</li>
<li>Obtaining the core InterProScan software (Direct link: ftp://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.51-85.0/interproscan-5.51-85.0-64-bit.tar.gz).</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> this is a large file (around 8 Gb).</div>
<ol start="12">
<li>Running InterProScan:
<ol type="a">
<li>Once the InterProScan package is uncompressed, it can be run directly from the command line.
<div class="textbox">
<p><code>#If run this script with no arguments, the usage</code></p>
<p><code>instructions will be presented.</code></p>
<p><code>&gt;./interproscan.sh</code></p>
</div>
</li>
<li>Run the shell script below:
<div class="textbox">
<p><code># interproscan.sh is the command taking in the input</code></p>
<p><code>file with parameter (-i) and setting up the format of</code></p>
<p><code>output file (e.g., tsv format). ‘-dp’ is to ensure all</code></p>
<p><code>the database matches proceeded in local environment.</code></p>
<p><code>&gt;./interproscan.sh -i</code></p>
<p><code>Chlamydomonas_UWO241_protein.fasta -f tsv -dp</code></p>
</div>
</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Output files:
<ol type="a">
<li>InterProScan should run without any warning, and it will create a tsv output file (i.e., Chlamydomonas_UWO241_protein.fasta.tsv) containing several member database matches, including Pfam. For your convenience, an example of an InterProScan search result is found in the ZIP file under the GitHub directory of tutorial with the name NoBadWordsCombiner_file_examples.zip. <a href="#sec6.7">Troubleshooting 4</a>
</li>
</ol>
</li>
</ol>
<ol start="14">
<li>The 13-column explanation of InterProScan search result file (<a href="#tbl5">Table 5</a>):
<ol type="a">
<li>Protein accession (e.g., g5250.t1)</li>
<li>Sequence unique code (e.g., f246997202ceeb0ebfd5ea2f454be9a2)</li>
<li>Sequence length (e.g., 262)</li>
<li>Protein signature (e.g., Pfam)</li>
<li>Signature accession (e.g., PF02469)</li>
<li>Signature description (e.g., Fasciclin domain)</li>
<li>Start location (e.g., 123)</li>
<li>Stop location (e.g., 259)</li>
<li>E-value (or score) (e.g., 5.80E-09)</li>
<li>Status - is the status of the match (T: true)</li>
<li>Date - is the date of the run (e.g., 31-03-2019)</li>
<li>InterPro annotations - accession (e.g., IPR000782)</li>
<li>InterPro annotations - description (e.g., FAS1 domain)
<div class="note">
<span class="note-title">Note:</span> Before clicking the submission button, users can select one of nine protein signatures (i.e., Pfam, CDD, Hamap, PRINTS, ProDom, ProSitePattern, ProSiteProfiles, SFLD, or TIGRFAM). We set the Pfam domain parameter as the default in order to collect larger database entries and because it has been widely used in many sequence analysis and genome annotation projects. Users can select other protein signatures, such as CDD, which can utilize 3D structures to decipher sequence structure and functional relationships. The descriptions of the various protein signatures are shown below:</div>
<ol type="i">
<li>Pfam: A large collection of protein families, each represented by multiple sequence alignments and hidden Markov models (HMMs).</li>
<li>CDD: Prediction of Conserved Domains Database.</li>
<li>Hamap is a system for the classification and annotation of protein sequences.</li>
<li>PRINTS: A fingerprint is a group of conserved motifs used to characterize a protein family.</li>
<li>ProDom is a comprehensive set of protein domain families automatically generated from the UniProt Knowledge Database.</li>
<li>ProSitePatterns and ProSiteProfiles: PROSITE consists of documentation entries describing protein domains, families, and functional sites as well as associated patterns and profiles to identify them.</li>
<li>SFLDs are protein families based on Hidden Markov Models or HMMs.</li>
<li>TIGRFAMs are protein families based on Hidden Markov Models or HMMs.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Output file of the NoBadWordsCombiner tool</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼3 min</div>
<ol start="15">
<li>Tap the submit button and a pending image will jump out (<a href="#fig1">Figure 1</a>C). It usually takes less than three minutes to run with a 200 Mb genome-sized file (<a href="#fig1">Figure 1</a>D). <a href="#sec6.9">Troubleshooting 5</a>
</li>
<li>The output of 23-column tab-delimited mega table (<a href="#tbl6">Table 6</a>)
<table id="tbl6" class="dense-table">
<caption>Table 6. Output file example of 23-column mega table via NoBadWordsCombiner</caption>
<thead>
<tr>
<th>ID</th>
<th>Gene</th>
<th>Length</th>
<th>NoBadName_Hit_Des</th>
<th>NoBadName_Hit_Name</th>
<th>NoBadName_%ID</th>
<th>NoBadName_eValue</th>
<th>NCBI_Hit_Des</th>
<th>NCBI_Hit_Name</th>
<th>NCBI_%ID</th>
<th>NCBI_eValue</th>
<th>Swiss_Hit_Des</th>
<th>Swiss_Hit_Name</th>
<th>Swiss_%ID</th>
<th>Swiss_eValue</th>
<th>KEGG_KO</th>
<th>KEGG_Des</th>
<th>Pfam</th>
<th>Pfam_No</th>
<th>Pfam_Des</th>
<th>Pfam_evalue</th>
<th>Interpro_No</th>
<th>Interpro_domain</th>
</tr>
</thead>
<tbody>
<tr>
<td>0</td>
<td>QueryAcc</td>
<td>Query_Length</td>
<td>HitDescription</td>
<td>HitName</td>
<td>%ID</td>
<td>eValue</td>
<td>HitDescription</td>
<td>HitName</td>
<td>%ID</td>
<td>eValue</td>
<td>HitDescription</td>
<td>HitName</td>
<td>%ID</td>
<td>eValue</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>1</td>
<td>g1.t1</td>
<td>817</td>
<td>hypothetical protein CEUSTIGMA_g3421.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995578|dbj|GAX75978.1|</td>
<td>54.2635659</td>
<td>1.41E-75</td>
<td>hypothetical protein CEUSTIGMA_g3421.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995578|dbj|GAX75978.1|</td>
<td>54.2635659</td>
<td>1.41E-75</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>2</td>
<td>g2.t1</td>
<td>399</td>
<td>2-5A-dependent ribonuclease OS=Mus musculus OX=10090 GN=Rnasel PE=1 SV=2</td>
<td>sp|Q05921|RN5A_MOUSE</td>
<td>34.8837209</td>
<td>4.14E-06</td>
<td>ankyrin, partial [Anaeromyces robustus]</td>
<td>gi|1183350135|gb|ORX78377.1|</td>
<td>40.2298851</td>
<td>3.61E-10</td>
<td>2-5A-dependent ribonuclease OS=Mus musculus OX=10090 GN=Rnasel PE=1 SV=2</td>
<td>sp|Q05921|RN5A_MOUSE</td>
<td>34.8837209</td>
<td>4.14E-06</td>
<td>N/A</td>
<td>N/A</td>
<td>Pfam</td>
<td>PF12796</td>
<td>Ankyrin repeats (3 copies)</td>
<td>1.80E-11</td>
<td>IPR020683</td>
<td>Ankyrin repeat-containing domain</td>
</tr>
<tr>
<td>3</td>
<td>g3.t1</td>
<td>3567</td>
<td>DNA mismatch repair protein MSH6 OS=Arabidopsis thaliana OX=3702 GN=MSH6 PE=1 SV=2</td>
<td>sp|O04716|MSH6_ARATH</td>
<td>41.8181818</td>
<td>1.61E-05</td>
<td>hypothetical protein CEUSTIGMA_g3419.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995576|dbj|GAX75976.1|</td>
<td>38.4615385</td>
<td>1.15E-39</td>
<td>DNA mismatch repair protein MSH6 OS=Arabidopsis thaliana OX=3702 GN=MSH6 PE=1 SV=2</td>
<td>sp|O04716|MSH6_ARATH</td>
<td>41.8181818</td>
<td>1.61E-05</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>4</td>
<td>g4.t1</td>
<td>963</td>
<td>Eukaryotic peptide chain release factor GTP-binding subunit ERF3A OS=Homo sapiens OX=9606 GN=GSPT1 PE=1 SV=1</td>
<td>sp|P15170|ERF3A_HUMAN</td>
<td>72.2972973</td>
<td>2.94E-72</td>
<td>hypothetical protein CEUSTIGMA_g3418.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995575|dbj|GAX75975.1|</td>
<td>89.5061728</td>
<td>1.17E-97</td>
<td>Eukaryotic peptide chain release factor GTP-binding subunit ERF3A OS=Homo sapiens OX=9606 GN=GSPT1 PE=1 SV=1</td>
<td>sp|P15170|ERF3A_HUMAN</td>
<td>72.2972973</td>
<td>2.94E-72</td>
<td>K03267</td>
<td>ERF3, GSPT; peptide chain release factor subunit 3</td>
<td>Pfam</td>
<td>PF00009</td>
<td>Elongation factor Tu GTP binding domain</td>
<td>1.70E-34</td>
<td>IPR000795</td>
<td>Transcription factor, GTP-binding domain</td>
</tr>
<tr>
<td>5</td>
<td>g6.t1</td>
<td>291</td>
<td>hypothetical protein CHLRE_10g421079v5 [Chlamydomonas reinhardtii]</td>
<td>gi|1335042461|gb|PNW77074.1|</td>
<td>58.3333333</td>
<td>1.66E-18</td>
<td>hypothetical protein CHLRE_10g421079v5 [Chlamydomonas reinhardtii]</td>
<td>gi|1335042461|gb|PNW77074.1|</td>
<td>58.3333333</td>
<td>1.66E-18</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>6</td>
<td>g7.t1</td>
<td>7908</td>
<td>hypothetical protein CEUSTIGMA_g3945.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238994727|dbj|GAX76500.1|</td>
<td>32.6785714</td>
<td>7.48E-34</td>
<td>hypothetical protein CEUSTIGMA_g3945.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238994727|dbj|GAX76500.1|</td>
<td>32.6785714</td>
<td>7.48E-34</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>7</td>
<td>g9.t1</td>
<td>471</td>
<td>Thylakoid-associated protein slr0729 OS=Synechocystis sp. (strain PCC 6803 / Kazusa) OX=1111708 GN=slr0729 PE=4 SV=1</td>
<td>sp|P72673|Y729_SYNY3</td>
<td>29.4736842</td>
<td>1.49E-06</td>
<td>hypothetical protein CEUSTIGMA_g3416.t1 [Chlamydomonas eustigma]</td>
<td>gi|1238995573|dbj|GAX75973.1|</td>
<td>62.1212121</td>
<td>3.00E-49</td>
<td>Thylakoid-associated protein slr0729 OS=Synechocystis sp. (strain PCC 6803 / Kazusa) OX=1111708 GN=slr0729 PE=4 SV=1</td>
<td>sp|P72673|Y729_SYNY3</td>
<td>29.4736842</td>
<td>1.49E-06</td>
<td>N/A</td>
<td>N/A</td>
<td>Pfam</td>
<td>PF11378</td>
<td>Protein of unknown function (DUF3181)</td>
<td>6.90E-26</td>
<td>IPR021518</td>
<td>Protein of unknown function DUF3181</td>
</tr>
<tr>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
<td>…</td>
</tr>
</tbody>
</table>
<ol type="a">
<li>ID (e.g., 2)</li>
<li>Gene or QueryAcc (e.g., g2.t1)</li>
<li>Length or Query_Length (e.g., 817)</li>
<li>NoBadName_Hit_Des or HitDescription (e.g., 2-5A-dependent ribonuclease OS=Mus musculus OX=10090 GN=Rnasel PE=1 SV=2)</li>
<li>NoBadName_Hit_Name or HitName (e.g., sp|Q05921|RN5A_MOUSE)</li>
<li>NoBadName_%ID or %ID (e.g., 34.8837209)</li>
<li>NoBadName_eValue or eValue (e.g., 4.14E-06)</li>
<li>NCBI_Hit_Des or HitDescription (e.g., ankyrin, partial [Anaeromyces robustus])</li>
<li>NCBI_Hit_Name or HitName (e.g., gi|1183350135|gb|ORX78377.1|)</li>
<li>NCBI_%ID or %ID (e.g., 40.2298851)</li>
<li>NCBI_eValue or eValue (e.g., 3.61E-10)</li>
<li>Swiss_Hit_Des or HitDescription (e.g., 2-5A-dependent ribonuclease OS=Mus musculus OX=10090 GN=Rnasel PE=1 SV=2)</li>
<li>Swiss_Hit_Name or HitName (e.g., sp|Q05921|RN5A_MOUSE)</li>
<li>Swiss_%ID or %ID (e.g., 34.8837209)</li>
<li>Swiss_eValue or eValue (e.g., 4.14E-06)</li>
<li>KEGG_KO (e.g., K03267)</li>
<li>KEGG_Des (e.g., ERF3, GSPT; peptide chain release factor subunit 3)</li>
<li>Protein signatures (e.g., Pfam)</li>
<li>Pfam_No (e.g., PF12796)</li>
<li>Pfam_Des (e.g., Ankyrin repeats (3 copies))</li>
<li>Pfam_evalue (e.g., 1.80E-11)</li>
<li>Interpro_No (e.g., IPR020683)</li>
<li>Interpro_domain (e.g., Ankyrin repeat-containing domain)</li>
</ol>
</li>
<li>The reason we created two columns for the header (e.g., NoBadName_Hit_Des or HitDescription) is to reduce ambiguity. A uniform header is needed when merging different databases. We also included the percentage identity and E-value for each type of BLAST search result, which can be easily compared. If two BLAST database hit descriptions are both without ‘bad words’, the one with lower E-value will be chosen.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>NoBadWordsCombiner is a free and straightforward online bioinformatics software tool for merging and minimizing hypothetical or uncharacterized proteins from various eukaryotic functional annotation databases. It provides a mega table combined with protein annotation database entries, such as InterPro, Pfam, and KEGG KO. To compare the data in the mega table, users can have an overview of the gene’s annotation patterns, including information on pathways, gene family domain, and gene family.</p>
<p>The aim of this tool is to assist with high-quality gene model annotations of eukaryotic nuclear genomes. We provided a real example of this tool in action: the file contains the gene models and their functional descriptions for the UWO241 genome (<a href="#bib12">Zhang et al., 2021a</a>), which greatly aided downstream analyses of this genome, such as detecting highly similar duplicated genes (HSDs) as well as horizontally transferred genes and gene family expansions (<a href="#bib13">Zhang et al., 2021b</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>NoBadWordsCombiner is limited to presenting gene annotations in a mega table without any plots or statistical interpretations, such as the total number of ‘bad words’, the frequency of genes containing ‘bad words’, or what types of genes have ‘bad words’. E-values are only used to measure the better BLAST results when both hits contain no ‘bad words’. If the genome is misassembled, using E-values alone might infer false positives. In the future, we hope to incorporate the threshold of aligned length, percentage of pairwise identity, and number of domains into the algorithm. For example, only when a certain criterion is satisfied (e.g., pairwise aligned length ≥ 50 amino acids, percentage identity ≥ 30%, and at least one domain), will the E-value be used to judge a better BLAST functional hit.</p>
<p>The web tool is reliant on third-party tools to generate the input files, such as InterProScan (<a href="#bib10">Quevillon et al., 2005</a>) and the KEGG tools Ghost KOALA or BlastKOALA (<a href="#bib8">Kanehisa et al., 2016</a>). Users need to be familiar with the basic BLAST package and dash shell in Linux/Unix environments. Notably, there is a steep learning curve for users without any bioinformatics or programming experience. It is our hope to further develop the tool, removing some of the middle steps. For now, we have provided the build-in reference files for each input file as well as example data to facilitate the usability of the tool.</p>
<p>In the future, NoBadWordsCombiner will be further improved, including continuous updating by considering more functional eukaryotic databases. It will also be expanded so it can work on other types of genomic data, such as prokaryotic and organelle genomes.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Why does BLASTP need to be chosen as an option? What E-value shall I choose? (Step 2)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Make sure to use the BLASTP option because amino acid sequences are generally more highly conserved than their corresponding nucleotide sequences. We recommend the E-value to be no larger than 1e-5 to ensure accurate prediction.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Can I resubmit the input files? (Step 5)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Yes. Simply re-fresh (reload) the browser page.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Why is the KEGG KO annotation file needed and what does it look like? (Step 10)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The example file has been provided with the name ‘Input_4_NoBadWords_ko’ from GitHub. The file documents the correlation of KO accession with each gene model identifier, which can be used to strengthen the gene functional category.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Is it difficult to run InterProScan? (Step 13)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>No. It is straightforward to run the tool. A real example of a InterProScan result has been provided at GitHub in the NoBadWordsCombiner_file_examples.zip file named ‘Input_5_NoBadWords_Pfam’. It is a tab-delimited file including the protein signatures, such as Pfam domain and InterPro annotations.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>How does the tool proceed if the BLAST hits inferring hypothetical or uncharacterized proteins come from multiple databases? (step 17)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>If BLAST database hit descriptions from multiple databases all contain ‘bad words’, the one with the lowest e-value will be chosen.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Altschul, S.F., Madden, T.L., Schäffer, A.A., Zhang, J., Zhang, Z., Miller, W., and Lipman, D.J. (1997). Gapped BLAST and PSI-BLAST: a new generation of protein database search programs. Nucleic Acids Res. <i>25</i>, 3389-3402. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref1">View at publisher</a></p>
<p id="bib2">Boeckmann, B., Bairoch, A., Apweiler, R., Blatter, M.-C., Estreicher, A., Gasteiger, E., Martin, M.J., Michoud, K., O'Donovan, C., and Phan, I. (2003). The Swiss-Prot protein knowledgebase and its supplement TrEMBL in 2003. Nucleic Acids Res. <i>31</i>, 365-370. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref2">View at publisher</a></p>
<p id="bib3">Boutet, E., Lieberherr, D., Tognolli, M., Schneider, M., and Bairoch, A. (2007). UniProtKB/Swiss-Prot: the manually annotated section of the UniProt KnowledgeBase. In Plant Bioinformatics. Methods in Molecular Biology, D. Edwards, ed. (Humana Press). <a class="external-link" href="https://doi.org/10.1007/978-1-59745-535-0_4">https://doi.org/10.1007/978-1-59745-535-0_4</a>.</p>
<p id="bib4">Cock, P.J., Chilton, J.M., Grüning, B., Johnson, J.E., and Soranzo, N. (2015). NCBI BLAST+ integrated into Galaxy. Gigascience <i>4</i>, 39. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref4">View at publisher</a></p>
<p id="bib5">De Wit, P., Pespeni, M.H., Ladner, J.T., Barshis, D.J., Seneca, F., Jaris, H., Therkildsen, N.O., Morikawa, M., and Palumbi, S.R. (2012). The simple fool's guide to population genomics via RNA-Seq: an introduction to high-throughput sequencing data analysis. Mol. Ecol. Resour. <i>12</i>, 1058-1067. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref5">View at publisher</a></p>
<p id="bib6">Galperin, M.Y. (2001). Conserved ‘hypothetical’proteins: new hints and new puzzles. Comp. Funct. Genomics <i>2</i>, 14-18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref6">View at publisher</a></p>
<p id="bib7">Kanehisa, M. and Goto, S. (2000). KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. <i>28</i>, 27-30. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref7">View at publisher</a></p>
<p id="bib8">Kanehisa, M., Sato, Y., and Morishima, K. (2016). BlastKOALA and GhostKOALA: KEGG tools for functional characterization of genome and metagenome sequences. J. Mol. Biol. <i>428</i>, 726-731. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref8">View at publisher</a></p>
<p id="bib9">Pruitt, K.D., Tatusova, T., and Maglott, D.R. (2005). NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins. Nucleic Acids Res. <i>33</i>, D501-D504. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref9">View at publisher</a></p>
<p id="bib10">Quevillon, E., Silventoinen, V., Pillai, S., Harte, N., Mulder, N., Apweiler, R., and Lopez, R. (2005). InterProScan: protein domains identifier. Nucleic Acids Res. <i>33</i>, 116-120. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref10">View at publisher</a></p>
<p id="bib11">Yandell, M. and Ence, D. (2012). A beginner's guide to eukaryotic genome annotation. Nat. Rev. Genet. <i>13</i>, 329-342. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref11">View at publisher</a></p>
<p id="bib12">Zhang, X., Cvetkovska, M., Morgan-Kiss, R., Hüner, N.P., and Smith, D.R. (2021a). Draft genome sequence of the Antarctic green alga Chlamydomonas sp. UWO241. iScience <i>24</i>, 102084. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref12">View at publisher</a></p>
<p id="bib13">Zhang, X., Hu, Y., and Smith, D.R. (2021b). Protocol for HSDFinder: Identifying, annotating, categorizing, and visualizing duplicated genes in eukaryotic genomes. STAR Protoc. <i>2</i>, 100619. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00594-3/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact David Roy Smith (<a href="mailto:dsmit242@uwo.ca">dsmit242@uwo.ca</a>) and technical contact Xi Zhang (<a href="mailto:xzha25@uwo.ca">xzha25@uwo.ca</a>)</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The eukaryotic genomic datasets supporting the conclusions of this article are available from NCBI (<a href="https://www.ncbi.nlm.nih.gov/">https://www.ncbi.nlm.nih.gov</a>). The NoBadWordsCombiner source code has been deposited at <a href="https://github.com/zx0223winner/NoBadWordsCombiner">https://github.com/zx0223winner/NoBadWordsCombiner</a>. The web server of NoBadWordsCombiner is freely available under <a href="http://hsdfinder.com/combiner">http://hsdfinder.com</a>/combiner.</p>
<h3>Acknowledgments</h3>
<p>The authors gratefully acknowledge funding Discovery Grants from the <a href="https://doi.org/10.13039/501100000038">Natural Sciences and Engineering Research Council of Canada</a> (NSERC). We appreciate the constructive suggestions from all the anonymous reviewers.</p>
<h3>Author contributions</h3>
<p>The study was conceptualized by X.Z. and D.R.S. The data were analyzed by X.Z., and Y.H. implemented the NoBadWordsCombiner website. X.Z. and D.R.S. drafted the manuscript, and all authors commented to produce the manuscript for peer review.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>