<article>
<section>
<h1 id="header">Laser microscopy acquisition and analysis of premotor synapses in the murine spinal cord</h1>
<p><time datetime="2022-03-11">Published: March 11, 2022</time></p>
<p>Jannik M. Buettner,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Toni Kirmann,<sup><a href="#aff1">1</a></sup> George Z. Mentis,<sup><a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Stefan Hallermann,<sup><a href="#aff1">1</a></sup> and Christian M. Simon<sup><a href="#aff1">1</a>,<a href="#fn2">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Carl-Ludwig-Institute for Physiology, Leipzig University, 04103 Leipzig, Germany</p>
<p id="aff2"><sup>2</sup>Center for Motor Neuron Biology and Disease, Columbia University, New York, NY 10032, USA</p>
<p id="aff3"><sup>3</sup>Depts. of Pathology and Cell Biology and Neurology, Columbia University, New York, NY 10032, USA</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:christian.simon@medizin.uni-leipzig.de">christian.simon@medizin.uni-leipzig.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101236">10.1016/j.xpro.2022.101236</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Loss of synapses on spinal motor neurons is a major feature of several neurodegenerative diseases; however, analyzing these premotor synapses is challenging because of their small size and high density. This protocol describes confocal and Stimulated Emission Depletion (STED) imaging of murine spinal premotor synapses and their segment-specific quantification by confocal microscopy. We detail the preparation of spinal cord segments, followed by image acquisition and analysis. This protocol enables in-depth analysis of pathological changes in spinal premotor synapses during neurodegeneration.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref3">Buettner et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Dissection and sectioning of identified murine spinal cord segments</li>
<li>Fluorescent labeling of spinal premotor synapses and motor neurons</li>
<li>Confocal and super-resolution acquisition of individual premotor synapses</li>
<li>Quantification of premotor synapses onto motor neurons</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol describes the specific steps to analyze the number and density of synapses contacting motor neuron cell bodies and dendrites. Before you can begin with the immunofluorescence staining, imaging and analysis of premotor synapses, murine spinal cords must be dissected, fixed and subsequently cut in serial transverse sections. The protocol below describes specific steps using spinal cords from mice transcardially perfused with paraformaldehyde (PFA) working at any age ranging from neonatal to adult. However, these protocols also work for spinal cords of juvenile (&lt;postnatal day (P)12) mice when natively dissected with oxygenated artificial cerebrospinal fluid and PFA “immersion” postfixed (for details see <a href="#bib10">Mentis et al., 2011</a>; <a href="#bib15">Simon et al., 2021</a>; <a href="#bib16">Simon et al., 2017</a>; <a href="#bib17">Simon et al., 2019</a>). While a wide-range of established antibodies as synaptic markers work reliably with this protocol (<a href="#bib3">Buettner et al., 2021</a>), we focus here on the vesicular glutamate transporter 1 (VGluT1) positive proprioceptive synapses which are severely affected in animal models for motor neuron diseases (<a href="#bib1">Baczyk et al., 2020</a>; <a href="#bib3">Buettner et al., 2021</a>; <a href="#bib4">Fletcher et al., 2017</a>).</p>
<p>Breeding and experiments were performed in the animal facilities of the Faculty of Medicine, University of Leipzig according to European (Council Directive 86/609/EEC) and German (Tierschutzgesetz) guidelines for the welfare of experimental animals and the regional directorate (Landesdirektion) of Leipzig (animal protocol: T01/21). We used male and females of <i>C57BL/6J</i> wild-type (JAX strain #000664) and <i>ChAT-Cre</i>; <i>tdTomato</i> (JAX strain #006410 and #007908) mice in the age range from P10 to 110 in our experiments.</p>
<p>Before the experiment can begin, mice need to be bred to the desired age and solutions have to be prepared as described below (refer to “<a href="#materials-and-equipment">materials and equipment</a>” for detailed recipe).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Guinea-pig anti-VGluT1 (1:5000)</td>
<td>Synaptic Systems</td>
<td>135 304; RRID:AB_887878</td>
</tr>
<tr>
<td>Goat anti-ChAT (1:500)</td>
<td>Millipore</td>
<td>AB144P; RRID:AB_2079751</td>
</tr>
<tr>
<td>Mouse anti-NeuN (1:2000)</td>
<td>Millipore</td>
<td>MAB377RRID:AB_2298772</td>
</tr>
<tr>
<td>Rabbit anti-Munc 13-1 (1:500)</td>
<td>Synaptic Systems</td>
<td>126 103RRID:AB_887733</td>
</tr>
<tr>
<td>Donkey anti-goat Alexa Fluor 488 (1:1000)</td>
<td>Jackson ImmunoResearch</td>
<td>705-545-147; RRID:AB_2336933</td>
</tr>
<tr>
<td>Donkey anti-mouse Alexa Fluor 488 (1:1000)</td>
<td>Jackson ImmunoResearch</td>
<td>715-165-150; RRID:AB_2340813</td>
</tr>
<tr>
<td>Donkey anti-goat Cy3 (1:1000)</td>
<td>Jackson ImmunoResearch</td>
<td>705-165-147; RRID:AB_2307351</td>
</tr>
<tr>
<td>Donkey anti-guinea pig Cy3 (1:1000)</td>
<td>Jackson ImmunoResearch</td>
<td>706-165-148RRID:AB_2340460</td>
</tr>
<tr>
<td>Donkey anti-guinea pig Alexa Fluor 647 (1:1000)</td>
<td>Jackson ImmunoResearch</td>
<td>706-606-148;RRID: AB_2340477</td>
</tr>
<tr>
<td>Donkey anti-rabbit Alexa Fluor 647 (1:1000)</td>
<td>Jackson ImmunoResearch</td>
<td>711-605-152RRID: AB_2492288</td>
</tr>
<tr>
<td>Goat anti-rabbit IgG, STAR Red Abberior (1:500)</td>
<td>Abberior Instruments</td>
<td>STRED-1002-500UGRRID:AB_2833015</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>PBS</td>
<td>Sigma-Aldrich</td>
<td>Cat# P3813</td>
</tr>
<tr>
<td>Triton-X 100</td>
<td>Sigma-Aldrich</td>
<td>CAS 9002-93-1</td>
</tr>
<tr>
<td>Normal Donkey Serum</td>
<td>Jackson ImmunoResearch</td>
<td>Cat# 071-000-121</td>
</tr>
<tr>
<td>16% Formaldehyde Solution Methanol-free</td>
<td>Thermo Scientific</td>
<td>Cat# 28906</td>
</tr>
<tr>
<td>Agar</td>
<td>Sigma-Aldrich</td>
<td>CAS 9002-18-0</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>CAS 56-81-5</td>
</tr>
<tr>
<td>Sylgard™ 184 Silicone Elastomer</td>
<td>Dow</td>
<td>BIESTERFELD SPEZIALCHEMIE GMBH</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>Sigma-Aldrich</td>
<td>CAS 26628-22-8</td>
</tr>
<tr>
<td>Isoflurane</td>
<td>Baxter</td>
<td>HDG9623</td>
</tr>
<tr>
<td>Metamizole</td>
<td>Sigma-Aldrich</td>
<td>CAS 5907-38-0</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: <i>C57BL/6J</i> wild-type P10, female</td>
<td>JAX</td>
<td>#000664</td>
</tr>
<tr>
<td>Mouse: <i>ChAT-Cre</i>; <i>tdTomato</i>, (heterozygous for both transgenes) P14-P110, female and male</td>
<td>JAX</td>
<td>#006410 and #007908</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>LAS X</td>
<td>Leica Microsystems</td>
<td>RRID:SCR_013673;<a href="https://www.leica-microsystems.com/de/produkte/mikroskop-software/p/leica-las-x-ls/">https://www.leica-microsystems.com/de/produkte/mikroskop-software/p/leica-las-x-ls/</a>
</td>
</tr>
<tr>
<td>Prism 9</td>
<td>GraphPad</td>
<td>RRID:SCR_002798;<a href="https://www.graphstats.net/prism-free-trial">https://www.graphstats.net/prism-free-trial</a>
</td>
</tr>
<tr>
<td>Fiji-ImageJ</td>
<td>National Institutes of Health</td>
<td>RRID:SCR_003070;<a href="https://imagej.net/software/fiji/">https://imagej.net/software/fiji/</a>
</td>
</tr>
<tr>
<td>Imspector Software</td>
<td>Abberior Instruments</td>
<td>RRID:SCR_015249</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Leica TCS SP8 confocal microscope</td>
<td>Leica Microsystems</td>
<td>RRID:SCR_018169</td>
</tr>
<tr>
<td>Abberior Instruments STED Expert Line Göttingen, Germany</td>
<td>Abberior Instruments</td>
<td><a href="https://abberior-instruments.com/">https://abberior-instruments.com/</a></td>
</tr>
<tr>
<td>Netwell insert</td>
<td>Science service</td>
<td><a href="https://www.scienceservices.de/netwelltm-set-sterile.html">https://www.scienceservices.de/netwelltm-set-sterile.html</a></td>
</tr>
<tr>
<td>Rotilabo disposable weighing pans</td>
<td>Carl Roth</td>
<td>Cat# 1878.2</td>
</tr>
<tr>
<td>12-well plate</td>
<td>Thermo Scientific</td>
<td>Cat#150200</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>4% PFA fixing solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>16% Formaldehyde</td>
<td>4%</td>
<td>10 mL</td>
</tr>
<tr>
<td>1% PBS</td>
<td>n/a</td>
<td>30 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">4% PFA solution needs to be prepared freshly.</p>
<table id="undtbl2">
<caption>PBS with 0.02% sodium azide</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sodium azide</td>
<td>0.02%</td>
<td>0.1 g</td>
</tr>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>to 500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">PBS with 0.02% sodium azide can be stored several months at 4°C.</p>
<table id="undtbl3">
<caption>PBS with 0.3% Triton X-100 (PBS-T)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>498.5 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>0.3%</td>
<td>1.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">PBS-T can be stored several months at 4°.</p>
<table id="undtbl4">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Normal donkey serum</td>
<td>5%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>PBS-T</td>
<td>n/a</td>
<td>9.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Blocking buffer can be stored up to 2 days at 4°C.</p>
<table id="undtbl5">
<caption>Agar embedding buffer for vibratome</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Agar</td>
<td>5%</td>
<td>5 g</td>
</tr>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Agar embedding has to be made fresh right before embedding the spinal cord into it. Requires heat and stirring to dissolve Agar in 1× PBS (see below in “<a href="#step-by-step-method-details">step-by-step method details</a> – Part 2” and <a href="#mmc2">Video S2</a>).</p>
<table id="undtbl6">
<caption>Mounting medium</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycerol</td>
<td>70%</td>
<td>7 mL</td>
</tr>
<tr>
<td>1× PBS</td>
<td>30%</td>
<td>3 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for several months.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Paraformaldehyde, sodium azide and Triton X-100 are hazardous: always wear proper protective equipment and use only in a fume hood. Paraformaldehyde waste needs to be collected and disposed of according to institutional regulations.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Mouse perfusion and post fixation of the spinal cord (part 1)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This first part describes the perfusion of the mouse and the post fixation of spinal cord tissue which can be then stored for several months.</p>
<ol>
<li>Inject 200 mg/kg Metamizole (stock concentration: 100 mg/mL, dissolved in saline) intraperitoneally 30 min prior to perfusion followed by deep isoflurane anesthesia. Toe pinch reflection is applied to monitor the depth of anesthesia. It usually takes 5–10 min to achieve deep anesthesia.
<ol type="a">
<li>The type of drugs/anesthesia might vary depending on the approved animal protocol. Another frequently drug alternative prior to perfusion is Ketamine/Xylazine (Ketamine: 50–80 mg/kg; Xylazine: 5–10 mg/kg) via intraperitoneal injection.</li>
</ol>
</li>
<li>After the mouse is in deep anesthesia, lay it on its back and pin it down on its four limbs.</li>
<li>Use forceps and dissecting scissors to open the skin and expose the chest cavity.</li>
<li>Cut open the diaphragm using standard scissors and be careful to not pierce the heart.</li>
<li>Grab at the base of the sternum, cut through the ribcage and lift it to expose the heart.</li>
<li>Insert the perfusion needle in the apex of the left ventricle and cut open the right atrium with a small cut (by use of fine scissors) to perfuse the mice with 1× PBS at room temperature (20°C–23°C) for 5 min with a flow rate of 3 mL/min flush following with 4% PFA in PBS for 15 min at the same flowrate.</li>
<li>Evaluate success of perfusion by checking whether liver and muscles became pale and hardened (if blood was not completely removed from the liver, the perfusion is considered unsuccessful).</li>
<li>After successful perfusion, remove the mouse’s head, tail, the fur of the back and the extremities as well as internal intestinal organs to allow better liquid exposure.</li>
<li>Place the remaining body into a 50 mL Falcon tube with 4% PFA in PBS for immersion postfixation overnight (12–14 h). Alternatively, to reduce chemical and storage usage, dissect out the spinal cord immediately as described below and store it in a 10 mL Falcon tube with 4% PFA in PBS for immersion postfixation overnight (12–14 h) at 4°C.</li>
<li>Transfer body/spinal cord into PBS with 0.02% sodium azide for storage. Spinal cord can be stored in this way for several months, however we usually process the spinal cord the following days.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Evaluate success of perfusion by checking whether liver and muscles became pale and hard. Also check after laminectomy if the <i>arteria spinalis anterior</i> is without any residue of blood. Inadequate perfusion will result in poor immunohistochemistry stainings (see <a href="#sec7.1">troubleshooting 1</a>, <a href="#fig6">Figure 6</a>).</div>
<h3 id="sec3.2">Dissection and transverse sectioning of a spinal segment (part 2)</h3>
<p>This section explains the dissection of identified spinal segments and subsequent vibratome sectioning.</p>
<h4 id="sec3.2.1">Dissection of a spinal segment</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>Here, we describe the dissection steps to isolate individual spinal segments (see also <a href="#mmc1">Video S1</a>).</p>
<ol start="11">
<li>Dispose PBS 0.02% sodium azide and wash the mouse carcass two times with PBS.</li>
<li>Pin the carcass with its paws ventral side up on a Sylgard layer of a dissecting chamber filled with PBS and remove all organs, muscles and connective tissue using “Dumont 2SP Forceps” and “Spring Scissors – angled to side” by Fine scientific tools GmbH to clear your view on the spinal column (<a href="#fig1">Figure 1</a>, <b>first panel</b>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Dissection of identified spinal segments</div>
<p>Image series shows step-by-step dissection of a P14 murine spinal cord. From left to right: ventral view of the perfused mouse carcass with intact vertebral column, after laminectomy, isolated spinal cord and zoomed-in lumbar region with identified ventral roots (dotted white lines) and spinal segments (gray dotted lines). Note that the L6 (white arrow) ventral root is significantly smaller than the L5 (yellow arrow head), making it a reliable landmark for identification of spinal segments. Scale bar = 5 mm.</p>
</figcaption>
</figure>
</li>
<li>Gain access to spinal cord by a ventral rostral-to-caudal laminectomy using “Dumont #5 Forceps” and “Spring Scissors – angled to side” by Fine scientific tools GmbH together with a binocular microscope with 5× magnification (<a href="#fig1">Figure 1</a>, <b>second panel,</b> <a href="#mmc1">Video S1</a>).</li>
<li>Carefully dissect out the entire spinal cord in rostral-to-caudal direction and pin it into the Sylgard chamber using the “Vannas Spring Scissors - 3 mm Cutting Edge” and the “Dumont #5 Forceps” by FST GmbH (<a href="#fig1">Figure 1</a>, <b>third panel,</b> <a href="#mmc1">Video S1</a>).</li>
<li>Carefully free the spinal cord from the <i>Dura mater</i> and pull all ventral roots towards the rostral direction to reveal their point of origin (<a href="#fig1">Figure 1</a>, <b>fourth panel</b>).</li>
<li>The lumbar ventral roots gradually increase by size from L1 to L5. In contrast, L6 ventral root decreases again in size, making L5 ventral root a reliable landmark for the identification of spinal cord segments. Count from L5 ventral root to the desired ventral root of interest and cut out right underneath this root and the next rostral root with a scalpel to isolate your spinal segment of choice (<a href="#fig1">Figure 1</a>, <b>fourth panel</b>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Be very careful with the dissection. Ripping off a single ventral root can accidentally result in false identification of the spinal segment.</div>
<figure id="mmc1"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Mmc1.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Video S1. Dissection of the spinal cord, related to steps 13 and 14</div>
</figcaption>
</figure>
<h4 id="sec3.2.2">Transverse sectioning of a spinal segment</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step describes the transverse sectioning of a spinal segment with the vibratome prior to immunohistochemistry (see also <a href="#mmc2">Videos S2</a> and <a href="#mmc3">S3</a>). In our experience, vibratome sections have three major advantages compared to cryosections: 1. no cryo-preservation is required, 2. thicker sections are possible, which allows better visualization of entire motor neuron cell bodies and enlarged dendritic trees, 3. better preservation of tissue and cell structure. These 75-μm-sections are suitable for confocal and Stimulated Emission Depletion (STED) analysis.</p>
<ol start="17">
<li>Embedding of spinal cord segments into Agar buffer (<a href="#mmc2">Video S2</a>).
<ol type="a">
<li>Mix 5 g of Agar and with a total volume of 100 mL 1× PBS in a 250 mL beaker. Heat up the solution in the beaker at 340°C and stirred at 400 rpm for 10 min to dissolve Agar completely in 1× PBS.</li>
<li>Pour 8 mL hot Agar (190°C) into disposable weighing pans and wait until it stops fuming.</li>
<li>Carefully place the spinal cord segment into the warm Agar (80°C) using forceps.</li>
<li>When the Agar block is solidified, trim it (with the aid of safety blades) accordingly to the size of the spinal segment. Tip: leave a ∼2–3 mm extra Agar on the bottom of the spinal segment to glue this side to the platform of the vibratome.</li>
<li>Proceed directly to vibratome sectioning and make sure the Agar block is covered in 1× PBS to avoid drying out the tissue (in case you cannot proceed immediately, Agar block can be stored at least one day in 1× PBS).</li>
</ol>
</li>
<li>Transverse sectioning of the spinal segment (<a href="#mmc3">Video S3</a>)
<ol type="a">
<li>Glue the Agar block containing the spinal segment in a rostral-vertical orientation onto the platform of the vibratome with Super Glue and wait at least one minute.</li>
<li>Transfer the platform into the cutting chamber of the vibratome filled with 1× PBS.</li>
<li>Cut the segment into 75 μm thick slices with a speed of 7 and frequency 7 (Leica vibratome settings). Collect each free floating spinal section directly with a brush and transfer it into 1× PBS in a modified 12-well plate filled with 5 mm Sylgard blackened with graphite powder (one spatula of powder per 24 mL) and Netwell insert (<a href="#fig2">Figure 2</a>) to facilitate handling of the sections during immunohistochemistry. Avoid adding more than six spinal cord sections into one Netwell insert to ensure proper antibody penetration.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Modified 12-well plate for immunohistochemistry (IHC)</div>
<p>Left panel: top view from a modified 12-well plate filled with graphite powder blackened Sylgard. Right panel: side view from one well with a Netwell insert for easy transfer of the spinal cord section during IHC. Scale bar = 1 cm.</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Check intact gross morphology (without any cuts or damage) of the spinal cord sections by eye or magnification glass of the vibratome before proceeding to the next step.</div>
<figure id="mmc2"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Mmc2.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Video S2. Embedding of spinal segment, related to step 17</div>
</figcaption>
</figure>
<figure id="mmc3"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Mmc3.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Video S3. Transverse sectioning of spinal segment, related to step 18</div>
</figcaption>
</figure>
<h3 id="sec3.3">Immunohistochemistry (part 3)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This part includes the staining procedure of sequential spinal cord sections. These mounted sections can be subsequently scanned with confocal or super-resolution STED microscopy. Important: the sections stay in the same Netwell insert during the entire staining protocol. Only the 12-well plates filled with the indicated solutions will be exchanged for every step as described below. Therefore, a minimum of two modified 12-well plates are required for performing immunohistochemistry (one 12-well plate with the sections in the Netwell inserts which will be transferred to second 12-well plate containing already the next step solutions). Thereby, the spinal cord sections do not have to be touched and remain intact (see also the end of <a href="#mmc3">Video S3</a>).</p>
<ol start="19">
<li>Incubate sections in blocking buffer (5% normal donkey serum and 0.3% Triton X-100 in 1× PBS) for 90 min at room temperature. Important: each well requires a minimum volume of 700 μL solution per well for each step.</li>
<li>Prepare primary antibody combinations according to the antibody list (<a href="#tbl1">Table 1</a>) with blocking buffer.
<table id="tbl1">
<caption>Table 1. Antibody combinations to visualize premotor synapses</caption>
<thead>
<tr>
<th>Primary antibodies</th>
<th>Secondary antibody</th>
<th>Purpose</th>
</tr>
</thead>
<tbody>
<tr>
<td>ChAT (goat)VGluT1 (guinea pig)</td>
<td>Donkey α-goat Cy3Donkey α-guinea pig Cy5</td>
<td>Visualize all (ChAT+) motor neurons and (VGluT1+) proprioceptive synapses (confocal microscopy)</td>
</tr>
<tr>
<td>NeuN (mouse)ChAT (goat)VGluT1 (guinea pig)</td>
<td>Donkey α-mouse Alexa488Donkey α-goat Cy3Donkey α-guinea pig Alexa 647</td>
<td>Visualize α- (ChAT+/NeuN+) and γ- (ChAT+/NeuN-) motor neurons and (VGluT1+) proprioceptive synapses (confocal microscopy)</td>
</tr>
<tr>
<td>ChAT (goat)VGluT1 (guinea pig)Munc13-1 (rabbit)</td>
<td>Donkey α-goat Alexa488Donkey α-guinea pig Cy3Goat α-rabbit STAR Red</td>
<td>Visualize all (ChAT+) motor neurons and (VGluT1+) proprioceptive synapses (confocal microscopy) and (Munc 13-1+) intrasynaptic proteins (STED)</td>
</tr>
</tbody>
</table>
</li>
<li>Incubate spinal cord sections in primary antibodies overnight (12–14 h) on an orbital shaker at a speed of 50 rpm at room temperature.</li>
<li>Next day, wash sections with 1× PBS for 6 × 10 min.</li>
<li>Prepare secondary antibody solutions for the corresponding primary antibodies (<a href="#tbl1">Table 1</a>).</li>
</ol>
<ol start="24">
<li>Incubate sections with secondary antibodies for 3 h at room temperature to visualize primary antibodies.
<div class="critical">
<span class="critical-title">Critical:</span> <u>(Important for super-resolution STED)</u> the required photostable secondary antibodies for STED from Abberior anti-rabbit are only available made in goat (<a href="#tbl1">Table 1</a>). The ChAT antibody is also made in goat and requires a donkey <u>anti-goat</u> secondary antibody labeled with Alexa Fluor 488. Therefore, the donkey anti-goat secondary antibody labeled with Alexa Fluor 488 will label the secondary antibodies from Abberior when applied together. For this combination, apply secondary antibodies <b>sequentially</b>, but not simultaneously.</div>
<ol type="a">
<li>Apply first the donkey anti-goat Alexa-488,</li>
<li>Wash the sections 3 × 10 min with PBS</li>
<li>Apply the Abberior antibody made in goat.
<div class="note">
<span class="note-title">Note:</span> This will resolve the cross-binding of secondary antibodies.</div>
</li>
</ol>
</li>
</ol>
<ol start="25">
<li>Wash sections with PBS for 3 × 10 min.</li>
<li>Create a hydrophobic barrier on the glass slide with a PAP pen liquid blocker around the slide perimeter.</li>
<li>Mount spinal sections on glass slides with Glycerol/PBS (7:3) mounting medium using a fine brush.
<ol type="a">
<li>First, carefully take out a single section from the PBS solution within the Netwell insert by lifting it up with the brush. The section will be wrapped around the brush.</li>
<li>Then put the section within the hydrophobic barrier by slowly rolling the brush against the glass slide. The section will attach completely intact to the glass slide.</li>
<li>Use a small piece of tissue to remove remaining PBS on the glass slide to avoid sliding of the sections.</li>
<li>Repeat this with all sections from the same well and place them next to each other. Recommendation: Do not mount more than six sections per slide. Three sections in the upper row and three sections in the lower row.</li>
<li>Put a drop of Glycerol/PBS (∼60 μL) carefully in the center of the glass slide.</li>
</ol>
</li>
<li>Put cover glass on the slides, push off bubbles and remove extra mounting medium.</li>
<li>Seal the cover glass and slide with nail polish and store them at 4°C. Sections can be stored for several months. Do not start imaging before nail polish is completely hardened (&gt;1 h).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Use highly photostable secondary antibodies for detailed scanning of small structures with STED microscopy. Secondary antibodies labeled with a fluorochrome (Alexa Fluor and Cy series) usually used for immunohistochemistry and confocal imaging will bleach fast (<a href="#sec7.3">troubleshooting 2</a>, <a href="#fig7">Figure 7</a>) and cannot be used for STED analysis.</div>
<h3 id="sec3.4">Confocal and STED imaging of premotor synapses (part 4a)</h3>
<p>This section describes in detail imaging acquisition of premotor synapses by confocal microscopy and distribution of presynaptic proteins by STED microscopy.</p>
<h4 id="sec3.4.1">Confocal imaging acquisition</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼1 h per section, total of 3 h per animal</div>
<p>Confocal scanning is applied to count premotor synapses. The goal is to scan at least five complete motor neuron cell bodies and ten motor neuron dendrites up to a distance of 100 μm from the soma per section. Three sections per spinal segment are scanned to get a total of at least 15 motor neurons and 30 dendrites per animal. We used here a SP8 Leica confocal microscope.</p>
<ol start="30">
<li>First scan the spinal cord section with a 10× objective to double check the gross morphology and get a sense of the relative position within the spinal cord. An intact transverse spinal cord section (no holes or damage) and a crisp choline acetyltransferase (ChAT) and vesicular glutamate transporter 1 (VGluT1) staining are desirable for optimal results (<a href="#fig3">Figure 3</a>A, <b>left panel</b>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Confocal imaging of proprioceptive synapses</div>
<p>(A) Confocal images of an overview scan of a P14 spinal hemicord (left panel) with a 10× objective and zoomed-in scan of the motor neuron pool (right panel) with a 63× objective using ChAT (green) as a motor neuron marker in the ventral horn and VGluT1 (magenta) to visualize proprioceptive synapses. Scale bar, hemicord = 200 μm, motor neuron pool = 20 μm.</p>
<p>(B) Six example confocal planes of the same ChAT+ motor neuron (note that there will be more single optical planes) in a z-stack to quantify every VGluT1+ proprioceptive synapse on the motor neuron cell body. Scale bar = 20 μm.</p>
<p>(C) Left panel: confocal image of a ChAT+ motor neuron with its innervating VGluT1+ proprioceptive synapses. Dotted white line marks the motor neuron outline. Right panel: same scan as left, but without the ChAT signal for better overview. White arrow marks proprioceptive synapses innervating the motor neuron. Yellow arrowheads mark VGluT1+ synapses that are not in contact with the motor neuron and therefore cannot be included in the analysis. Scale bar = 10 μm.</p>
</figcaption>
</figure>
</li>
<li>For imaging premotor synapses, magnify on the ventral ChAT+ motor neuron pool with a 63× glycerol objective with a zoom factor of 1 (<a href="#fig3">Figure 3</a>A, <b>right panel</b>). Alternatively, a 40× objective with a zoom factor of 1.5 can be used.</li>
<li>Use an image stack to ∼50 μm of the 75 μm thick transverse spinal section avoiding the edges of the section (top and bottom), which usually contain higher background due to unspecific binding of secondary antibodies on the section’s surface.</li>
<li>Adjust laser intensity appropriately to improve signal-to-noise ratio for each channel.</li>
<li>To include every (VGluT1+) synapse as well as the entire cell bodies of motor neurons, sequentially scan the 50 μm stack at 0.4 μm z-steps with a resolution of 1024 × 1024 pixels, frequency of 100 Hz, and line average of 4. Tip: Use bidirectional scan for faster imaging acquisition, on the assumption that bidirectional scanning does not result in “doubling” of structures, indicative of improper laser alignment.</li>
<li>Once the scan is finished, go through the stack to ensure proper intensity of the signal throughout the stack and overall image quality (<a href="#fig3">Figure 3</a>B). In certain cases, laser strength compensation can be used (available through a software module).</li>
<li>Repeat procedure for two more sections from the same segment for a total of three sections per animal.</li>
<li>Save data file and proceed to image analysis.</li>
</ol>
<h4 id="sec3.4.2">STED imaging acquisition</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min per synapse, total of 6 h per animal</div>
<p>STED imaging is applied to visualize the distribution of presynaptic proteins. As a proof-of-principle, we scanned the VGluT1 and ChAT channel in confocal mode and used the improved resolution of STED microscopy to distinguish individual spots of the presynaptic protein Munc13-1, which is currently discussed as a possible candidate for vesicular release sites (<a href="#bib2">Böhme et al., 2016</a>; <a href="#bib8">Karlocai et al., 2021</a>; <a href="#bib13">Sakamoto et al., 2018</a>). We used here an Expert Line Abberior microscope.</p>
<ol start="38">
<li>To visualize the distribution of presynaptic proteins, scan the vibratome sections with STED microscopy stained with appropriate photostable secondary antibodies suitable for STED (see <a href="#tbl1">Table 1</a>, <a href="#sec7.3">troubleshooting 2</a>, <a href="#fig7">Figure 7</a>).</li>
<li>First locate ChAT+ motor neurons in the ventral horn with a 60× oil objective using the confocal mode.</li>
<li>Zoom into a selected proprioceptive synapse of interest with the 60× oil objective and visualize it in confocal mode (<a href="#fig4">Figure 4</a>, <b>first and second panel</b>)
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. STED imaging of presynaptic protein Munc13-1</div>
<p>First panel: confocal single plane image of a VGluT1+ proprioceptive synapse (magenta) onto ChAT+ motor neuron (green) of a P14 WT animal. Dotted white line marks the proprioceptive synapses. Second panel: confocal single plane image of Munc13-1 distribution (cyan) within a proprioceptive synapse. Third panel: STED image of the presynaptic Munc-13 distribution (cyan). Fourth panel: merge of confocal single pane images of ChAT, VGluT1 and STED image of Munc13-1. Scale bar = 1 μm.</p>
</figcaption>
</figure>
</li>
<li>Set a z-Stack for the synapse of interest with the confocal laser.</li>
<li>Adjust laser intensity appropriately to improve signal-to-noise ratio for each channel.</li>
<li>Sequentially scan the synapse in confocal mode for the ChAT+ motor neuron and VGluT1+ proprioceptive synapse, but scan the intrasynaptic protein Munc13-1 in STED mode. For STED, use a pixel size of 20 nm, a dwell time of 6 μs, and a line accumulation of 3 (<a href="#fig4">Figure 4</a>).</li>
<li>Repeat for ten synapses per animals.</li>
</ol>
<h3 id="sec3.5">Analysis of premotor synapses (part 4b)</h3>
<h4 id="sec3.5.1">Quantification of synaptic number</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This part of the analysis describes the quantification of confocal images of premotor synapses onto motor neurons. We usually conduct our analysis with LAS X free software from Leica, but any other software (e.g., Fiji-ImageJ, Zeiss ZEN) is suitable (see also <a href="#mmc4">Video S4</a>). The image analysis of STED images critically depends on the scientific question (for recent examples of analysis of STED images of Munc13, see e.g., <a href="#bib2">Böhme et al., 2016</a>; <a href="#bib6">Fukaya et al., 2021</a>; <a href="#bib7">Holderith et al., 2020</a>; <a href="#bib8">Karlocai et al., 2021</a>; <a href="#bib9">Liu et al., 2018</a>).</p>
<div class="critical">
<span class="critical-title">Critical:</span> Count <u>all</u> proprioceptive synapses on the cell body of the motor neuron to avoid biased analysis with appropriate software (see point 46, <a href="#mmc4">Video S4</a>). To do so, include only motor neurons of which the entire cell body has been scanned with the z-stack. Importantly, only ∼5% of the proprioceptive synapse are located on the cell body and the majority (95%) are on motor neuron proximal dendrites (<a href="#bib11">Mentis et al., 2006</a>; <a href="#bib12">Rotterman et al., 2014</a>). Therefore, the analysis of synaptic density on proximal dendrites is critical to make a statement of the quantity of premotor synapses on motor neurons.</div>
<ol start="45">
<li>Go through the z-stack and mark motor neurons of which their entire cell body has been included in this scan.</li>
<li>Digitally zoom onto one motor neuron cell body and slowly go through the z-stack marking every single VGluT1+ proprioceptive synapse over the entire surface of the motor neuron soma by applying manually a multiple point counter tool such as LAS X or Fiji-ImageJ. Go back and forward through the z-stack of the entire cell body to ensure no omission or double counting of synapses (see <a href="#mmc4">Video S4</a>).</li>
<li>Repeat this procedure for at least 5 motor neurons of one z-Stack and a total of 15 motor neurons per animal.</li>
<li>Proceed with the quantification of the synaptic density of the dendrites. Count the number of VGluT1+ synapses over the entire surface of the primary dendrite up to a distance of 100 μm from the soma as previously established (<a href="#bib10">Mentis et al., 2011</a>).</li>
<li>To do so, follow the ChAT+ dendrites originating from the motor neuron soma throughout the z-Stack and divide the length of the dendrite into 50 μm long areas. The first dendritic bin was from 0-50 μm (bin 0–50 μm), the second one from 50-100 μm (bin 50–100 μm) measured from the motor neuron cell body (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Analysis of dendritic synaptic density</div>
<p>Upper panel: example of a single confocal plane of a ChAT+ motor neuron (green) and VGluT1+ (magenta) proprioceptive synapse from a z-stack of a P14 WT mouse. White dotted lines mark motor neuron soma and two primary dendrites. Middle panel: same image as above but ChAT channel has been deactivated for better overview. The length of the proximal primary dendrites is divided into 0–50 μm (green line) and 50–100 μm (gray line) bins away from the soma. Scale bar = 25 μm Bottom panel: magnified image of a proximal dendrite (ChAT+, green, dotted white line) with innervating proprioceptive synapses (VGluT1, magenta). Scale bar = 10 μm. Note: dendrites cannot always be followed up to 100 μm length in a single confocal plane. To follow the dendrite in detail, go through the z-stack and quantify the synapses in every panel.</p>
</figcaption>
</figure>
</li>
<li>Count every VGluT1+ proprioceptive synapse in the restricted bin area by going through the z-stack and calculate the density in the selected bin as number of synapses/dendritic length (i.e., 50 μm)).</li>
<li>Repeat this for at least ten dendrites per z-Stack and a total of a minimum of 30 dendrites per animal, for at least three animals per genotype.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Count only synapses which are in direct contact with a ChAT signal. Exclude synapses which are close to motor neurons, but are not in apposition with dendrites (<a href="#fig3">Figure 3</a>C). ChAT antibodies label α- and γ-motor neurons. Importantly, γ-motor neurons receive less than one proprioceptive synapse on average compared to ∼30 proprioceptive synapses innervating α-motor neurons (<a href="#bib5">Friese et al., 2009</a>). To avoid contamination of γ-motor neurons apply NeuN as an additional marker, see <a href="#sec7.5">troubleshooting 3</a>, <a href="#fig8">Figure 8</a>).</div>
<figure id="mmc4"><video controls="controls">
    <source src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Mmc4.mp4" type="video/mp4">
    Your browser does not support HTML5 video.
  </source></video>
<figcaption>
<div class="figcaption-title">Video S4. Quantification of proprioceptive synapses, related to step 46</div>
</figcaption>
</figure>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The outcome of the protocol should be a consistent quantification of premotor synapses onto motor neuron cell bodies and proximal dendrites in control animals compared to mouse mutants with neurodegeneration. Wild-type mice older than two weeks have approximately 30 proprioceptive synapses per motor neuron soma and a synaptic density on proximal dendrites of ∼0.3 synapses/μm for the bin of 0–50 μm and ∼0.2 synapses/μm for the bin 50–100 μm (<a href="#bib3">Buettner et al., 2021</a>; <a href="#bib10">Mentis et al., 2011</a>). New born mice have significant lower numbers of proprioceptive synapses (<a href="#bib3">Buettner et al., 2021</a>; <a href="#bib10">Mentis et al., 2011</a>), since proprioceptive synapses increase the coverage of motor neurons until the 2<sup>nd</sup>-3<sup>rd</sup> postnatal week (<a href="#bib11">Mentis et al., 2006</a>). Mouse models with motor neuron diseases exhibit a strong reduction of proprioceptive synapses (<a href="#bib3">Buettner et al., 2021</a>; <a href="#bib10">Mentis et al., 2011</a>).</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>The analysis should be performed in a blinded manner to avoid any biases from the investigator. For each mouse spinal cord sample, six serial sections should be stained for each experiment, and at least three spinal cord sections should be scanned as a confocal z-stack for synaptic quantification. For each mouse, 15 motor neuron cell bodies and 30 proximal dendrites with bins of 0–50 μm and 50–100 μm length should be quantified. The mean of synaptic numbers onto the motor neuron soma and the dendritic density should be formed for each mouse. At least three different mice have to be analyzed. For intrasynaptic proportion within the synapses, at least 30 synapses per mouse should be analyzed. Comparison between two groups should be analyzed by unpaired two-tailed Student’s t-test if data is normally distributed and comparison among three groups should be analyzed by one-ANOVA test or a Kruskal–Wallis test.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol reliably reveals the number of premotor synapses. However, confocal microscopy cannot always resolve whether a VGluT1 positive putative synapse contains a single large synapse or two small synapses in very close proximity which would require STED imaging or electron microscopy resolution. These synapses in very close proximity represent only a small amount of all proprioceptive synapses and therefore are of minor concern for the quantification, especially when control and mutant mice are quantified blinded in the same way.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Insufficient perfusion during <b>step 7</b> leads to degenerating cells and denatured epitopes. This will result in compromised staining and analysis (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Examples and cellular consequences of insufficient perfusion</div>
<p>(A) Images of a P110 mouse rib cage, after laminectomy (overview and magnification) following a good (upper panel) and poor or inadequate (lower panel) perfusion. Note: blood vessels are still clearly visible following a poor perfusion. Gray dotted line marks <i>arteria spinalis anterior</i>. Scale bar = 5 mm.</p>
<p>(B) Upper panel: Confocal images of a poorly perfused P35 mouse spinal hemicord with ChAT+ (green) interneurons (white dotted square) and motor neurons (MN, white dotted circle). Genetically expressed Tdtomato (magenta) in ChAT+ neurons in a <i>ChAT-cre</i>; <i>tdTomato</i> mouse. Scale bar = 100 μm. Lower panel: Magnified motor neuron pool of this hemicord. Scale bar = 20 μm. Note: Although motor neurons are still present in a poorly perfused mouse (Tdtomato signal), ChAT antibodies do not label them adequately, while ChAT+ interneurons are still labeled.</p>
</figcaption>
</figure>
<h3 id="sec7.2">Potential solution</h3>
<p>First signs of a proper insertion of the perfusion needle is the whitening and stiffness of the intestinal organs. While performing the dissection of the spinal cord, check if blood vessels are still visible. A good perfused mouse should show no signs of blood vessels within the rib cage or the <i>arteria spinalis anterior</i> on the ventral side of the spinal cord (<a href="#fig6">Figure 6</a>A). If blood is still visible in this structure, do not proceed further to part 2. Even a short postfixation after a poor perfusion is not sufficient to obtain reliable results. Interestingly, the motor neurons are most vulnerable due to poor perfusion within the spinal cord. Insufficient perfusion results in almost complete lack motor neurons detected by anti-ChAT antibody, while cholinergic synapses (e.g., C-boutons) and cholinergic interneurons near the central canal can still be identified (<a href="#fig6">Figure 6</a>B, <b>upper row</b>). An endogenously expressed fluorescence marker (floxed <i>tdTomato</i>) within the motor neuron pool (<i>ChAT-Cre</i>) shows that motor neurons are not lost due to poor perfusion, but the epitopes within the motor neurons for recognizing antibodies (e.g., ChAT, NeuN and others) are lost and therefore cannot be visualized by IHC anymore (<a href="#fig6">Figure 6</a>B, <b>lower row</b>).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Secondary antibodies bleaches during first scan of STED imaging (<b>step 24</b>).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Use photostable secondary antibodies designed for STED microscopy (e.g., Abberior Star Red/Orange) in contrast to commonly used secondary antibodies for confocal microscopy (e.g., Alexa series) (<a href="#fig7">Figure 7</a>).</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Photostable Abberior secondary antibodies are necessary for STED imaging</div>
<p>(A) Confocal images of proprioceptive synapses (magenta) and Munc13-1 (cyan) of a P14 WT mouse visualized either with secondary antibodies from Alexa (left panel) or Abberior (right panel). Scale bar = 2 μm. Confocal images of Munc13-1 before (middle panel) and after (lower panel) five STED scans.</p>
<p>(B) Quantification of normalized photon counts acquired during five confocal scans with interleaved STED scans. n = 3 synapses per group, error bars represent mean with SD. p &lt; 0.01 for each time point with multiple unpaired t-test.</p>
</figcaption>
</figure>
<h3 id="sec7.5">Problem 3</h3>
<p>Contamination of your premotor synaptic quantification due to inclusion of γ-motor neurons in the analysis. ChAT antibodies label α- and γ-motor neurons. Usually, γ-motor neurons in adult mice are significantly smaller than α-motor neurons and contain almost no proprioceptive synaptic inputs (less than one proprioceptive synapse; (<a href="#bib5">Friese et al., 2009</a>). However, some analysis might contain developing motor neuron pools in which the size is not a reliable determinator of α- and γ-motor neurons (i.e., during the first postnatal week; (<a href="#bib14">Shneider et al., 2009</a>). Furthermore, neurodegeneration can result in shrunk α-motor neurons and strong reduction of proprioceptive synapses which allow contamination of γ-motor neurons in the analysis (<b>step 49</b>).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>It has been shown that NeuN antibodies marks α-, but not γ-motor neurons (<a href="#bib14">Shneider et al., 2009</a>). NeuN and ChAT costaining distinguish reliably between α- (ChAT+, NeuN+) and γ-motor neurons (ChAT+, NeuN-) (<a href="#fig8">Figure 8</a>).</p>
<figure id="fig8"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig8.jpg" alt="Fig8.jpg">
<figcaption>
<div class="figcaption-title">Figure 8. Proprioceptive inputs synapse onto α- but not γ-motor neurons</div>
<p>Confocal image of an α-motor neuron (left, CHAT+ (green), NeuN+ (blue)) and γ-motor neuron (right, CHAT+, NeuN-) of a P110 mouse. Proprioceptive synapses (VGluT1+, magenta) innervate only the α-motor neuron which is larger. Scale bar = 15 μm.</p>
</figcaption>
</figure>
<h3 id="sec7.7">Problem 4</h3>
<p>Synapses are counted for analysis which are in close proximity to the motor neurons, but not in apposition to the motor neuron membrane (<b>step 49</b>).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Zoom into the motor neuron soma or dendrite and ensure that VGluT1+ synapses are in direct contact with the ChAT signal. If a gap can be detected between the ChAT and VGlutT1 signal, exclude this synapse from analysis (<a href="#fig3">Figure 3</a>C).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Synaptic numbers vary due to false identification of the spinal segment. Synaptic inputs in wild-type mice and especially during neurodegeneration vary greatly between spinal segments (<a href="#bib3">Buettner et al., 2021</a>; <a href="#bib10">Mentis et al., 2011</a>). A mouse model for motor neuron disease exhibit for example a strong loss of proprioceptive synapses in an upper lumbar segment, but little vulnerability in lower lumbar segments (<b>step 16</b>).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Always compare premotor synaptic inputs of the <u>same</u> spinal segment across different genotypes. Use the described L5/L6 landmark for correct identification of the spinal segment (<a href="#fig1">Figure 1</a>).</p>
<h3 id="sec7.11">Problem 6</h3>
<p>ChAT staining does not sufficiently label dendrites for synaptic density analysis. On rare occasion, ChAT staining is weak or gives a strong background making dendritic analysis difficult (<b>step 7</b>).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>A weak ChAT staining is very likely due to insufficient perfusion and weak penetration of the antibody. Normally, the ChAT antibody staining labels the motor neuron soma and proximal dendrites up to 100 μm very reliably, as seen in a ChAT counterstaining with a mouse genetically expressing the Cre-dependent fluorochrome tdTomato in motor neurons under a <i>ChAT-Cre</i> promoter (<a href="#fig9">Figure 9</a>). Also, avoid using the primary ChAT antibody several times in different experiments. Depending on the quality (type, lot) of the antibody, a higher concentration might resolve the issue with visualizing longer proximal dendrites.</p>
<figure id="fig9"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1493-Fig9.jpg" alt="Fig9.jpg">
<figcaption>
<div class="figcaption-title">Figure 9. ChAT antibodies reliably label large dendritic trees</div>
<p>Confocal image of a motor neuron pool visualized with antibodies against ChAT (green, left panel) and endogenously expressed <i>tdTomato</i> (magenta, middle panel) within motor neurons by a <i>ChAT-Cre</i> promoter in a P10 mouse. Right panel: merged overall (white) of ChAT IHC and genetically expressed <i>tdTomato</i>. Scale bar = 40 μm. Note: ChAT IHC reliably labels dendrites up to 100 μm.</p>
</figcaption>
</figure>
<h3 id="sec7.13">Problem 7</h3>
<p>Most types of premotor synapses onto motor neurons are not too dense (e.g., C-boutons, VGluT1) and can be easily identified as individual proprioceptive synapses. However, some types of synapses (e.g., VGAT) are very dense and total synaptic numbers cannot be quantified (<b>step 49</b>).</p>
<h3 id="sec7.14">Potential solution</h3>
<p>When the synaptic density is so high on soma or dendrites that individual synapses cannot be distinguish, even in a single optical plane, the synaptic density per perimeter needs to be quantified instead of total synaptic numbers. VGAT synapses were quantified as synapses per perimeter of the motor neuron soma. Three different optical planes should be scanned of each motor neuron to ensure consistency. Then quantify the number of VGAT+ synapses onto ChAT+ motor neurons and the perimeter of the soma on each plane, for example, by Leica LAS X software. Division of synaptic number by the perimeter gives the synaptic density of each panel. Average the synaptic density of all three panels for each individual motor neuron, see also (<a href="#bib3">Buettner et al., 2021</a>).</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Baczyk, M., Alami, N.O., Delestree, N., Martinot, C., Tang, L., Commisso, B., Bayer, D., Doisne, N., Frankel, W., Manuel, M., et al. (2020). Synaptic restoration by cAMP/PKA drives activity-dependent neuroprotection to motoneurons in ALS. J. Exp. Med. <i>217</i> (<i>8</i>), 1-20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref1">View at publisher</a></p>
<p id="bib2">Böhme, M.A., Beis, C., Reddy-Alla, S., Reynolds, E., Mampell, M.M., Grasskamp, A.T., Lutzkendorf, J., Bergeron, D.D., Driller, J.H., Babikir, H., et al. (2016). Active zone scaffolds differentially accumulate Unc13 isoforms to tune Ca<sup>2+</sup> channel-vesicle coupling. Nat. Neurosci. <i>19</i>, 1311-1320. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref2">View at publisher</a></p>
<p id="bib3">Buettner, J.M., Sime Longang, J.K., Gerstner, F., Apel, K.S., Blanco-Redondo, B., Sowoidnich, L., Janzen, E., Langenhan, T., Wirth, B., and Simon, C.M. (2021). Central synaptopathy is the most conserved feature of motor circuit pathology across spinal muscular atrophy mouse models. iScience <i>24</i>, 103376. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref3">View at publisher</a></p>
<p id="bib4">Fletcher, E.V., Simon, C.M., Pagiazitis, J.G., Chalif, J.I., Vukojicic, A., Drobac, E., Wang, X., and Mentis, G.Z. (2017). Reduced sensory synaptic excitation impairs motor neuron function via Kv2.1 in spinal muscular atrophy. Nat. Neurosci. <i>20</i>, 905-916. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref4">View at publisher</a></p>
<p id="bib5">Friese, A., Kaltschmidt, J.A., Ladle, D.R., Sigrist, M., Jessell, T.M., and Arber, S. (2009). Gamma and alpha motor neurons distinguished by expression of transcription factor Err3. Proc. Natl. Acad. Sci. U S A <i>106</i>, 13588-13593. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref5">View at publisher</a></p>
<p id="bib6">Fukaya, R., Maglione, M., Sigrist, S.J., and Sakaba, T. (2021). Rapid Ca<sup>2+</sup> channel accumulation contributes to cAMP-mediated increase in transmission at hippocampal mossy fiber synapses. Proc. Natl. Acad. Sci. U S A <i>118</i>. e2016754118. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref6">View at publisher</a></p>
<p id="bib7">Holderith, N., Heredi, J., Kis, V., and Nusser, Z. (2020). A high-resolution method for quantitative molecular analysis of functionally characterized individual synapses. Cell Rep. <i>32</i>, 107968. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref7">View at publisher</a></p>
<p id="bib8">Karlocai, M.R., Heredi, J., Benedek, T., Holderith, N., Lorincz, A., and Nusser, Z. (2021). Variability in the Munc13-1 content of excitatory release sites. Elife <i>10</i>, e67468. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref8">View at publisher</a></p>
<p id="bib9">Liu, C., Kershberg, L., Wang, J., Schneeberger, S., and Kaeser, P.S. (2018). Dopamine secretion is mediated by sparse active zone-like release sites. Cell <i>172</i>, 706-718 e715. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref9">View at publisher</a></p>
<p id="bib10">Mentis, G.Z., Blivis, D., Liu, W., Drobac, E., Crowder, M.E., Kong, L., Alvarez, F.J., Sumner, C.J., and O'Donovan, M.J. (2011). Early functional impairment of sensory-motor connectivity in a mouse model of spinal muscular atrophy. Neuron <i>69</i>, 453-467. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref10">View at publisher</a></p>
<p id="bib11">Mentis, G.Z., Siembab, V.C., Zerda, R., O'Donovan, M.J., and Alvarez, F.J. (2006). Primary afferent synapses on developing and adult Renshaw cells. J. Neurosci. <i>26</i>, 13297-13310. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref11">View at publisher</a></p>
<p id="bib12">Rotterman, T.M., Nardelli, P., Cope, T.C., and Alvarez, F.J. (2014). Normal distribution of VGLUT1 synapses on spinal motoneuron dendrites and their reorganization after nerve injury. J. Neurosci. <i>34</i>, 3475-3492. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref12">View at publisher</a></p>
<p id="bib13">Sakamoto, H., Ariyoshi, T., Kimpara, N., Sugao, K., Taiko, I., Takikawa, K., Asanuma, D., Namiki, S., and Hirose, K. (2018). Synaptic weight set by Munc13-1 supramolecular assemblies. Nat. Neurosci. <i>21</i>, 41-49. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref13">View at publisher</a></p>
<p id="bib14">Shneider, N.A., Brown, M.N., Smith, C.A., Pickel, J., and Alvarez, F.J. (2009). Gamma motor neurons express distinct genetic markers at birth and require muscle spindle-derived GDNF for postnatal survival. Neural Dev. <i>4</i>, 42. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref14">View at publisher</a></p>
<p id="bib15">Simon, C.M., Blanco-Redondo, B., Buettner, J.M., Pagiazitis, J.G., Fletcher, E.V., Sime Longang, J.K., and Mentis, G.Z. (2021). Chronic pharmacological increase of neuronal activity improves sensory-motor dysfunction in spinal muscular atrophy mice. J. Neurosci. <i>41</i>, 376-389. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref15">View at publisher</a></p>
<p id="bib16">Simon, C.M., Dai, Y., Van Alstyne, M., Koutsioumpa, C., Pagiazitis, J.G., Chalif, J.I., Wang, X., Rabinowitz, J.E., Henderson, C.E., Pellizzoni, L., et al. (2017). Converging mechanisms of p53 activation drive motor neuron degeneration in spinal muscular atrophy. Cell Rep. <i>21</i>, 3767-3780. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref16">View at publisher</a></p>
<p id="bib17">Simon, C.M., Van Alstyne, M., Lotti, F., Bianchetti, E., Tisdale, S., Watterson, D.M., Mentis, G.Z., and Pellizzoni, L. (2019). Stasimon contributes to the loss of sensory synapses and motor neuron death in a mouse model of spinal muscular atrophy. Cell Rep. <i>29</i>, 3885-3901 e3885. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00116-2/sref17">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Christian M. Simon, email: <a href="mailto:Christian.Simon@medizin.uni-leipzig.de">Christian.Simon@medizin.uni-leipzig.de</a></p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique data sets or code.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank Florian Gerstner for acquiring images of the equipment and spinal cord dissection. We would also like to thank Dr. Johannes Hirrlinger and Dr. Tobias Langenhan for providing reagents and facility access. This work was supported by German Research Foundation grants SI-1969/2-1, SI-1969/3-1, and SMA-Europe to C.M.S.; R01-NS078375 (<a href="https://doi.org/10.13039/100000065">NINDS</a>) and R01-AA027079 (<a href="https://doi.org/10.13039/100000002">NIH</a> Blueprint for Neuroscience Research) to G.Z.M., a <a href="https://doi.org/10.13039/501100000781">European Research Council</a> Consolidator Grant (ERC CoG 865634) and German Research Foundation grant HA6386/10-1 to S.H.</p>
<h3>Author contributions</h3>
<p>C.M.S. designed and supervised the study. J.M.B. performed the experiments and analysis. G.Z.M. established the protocol for the confocal analysis of premotor synapses. T.K., S.H., and J.M.B. established the protocol for super-resolution imaging. C.M.S. and J.M.B. wrote the manuscript with input from all authors.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>