<article>
<section>
<h1 id="header">A protocol for efficient CRISPR-Cas9-mediated knock-in in colorectal cancer patient-derived organoids</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Takuya Okamoto,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a></sup> Yasuko Natsume,<sup><a href="#aff1">1</a></sup> Hitomi Yamanaka,<sup><a href="#aff1">1</a></sup> Mayuko Fukuda,<sup><a href="#aff1">1</a></sup> and Ryoji Yao<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Cell Biology, Cancer Institute, Japanese Foundation for Cancer Research, Tokyo, 135-8550, Japan</p>
<p id="aff2"><sup>2</sup>Department of Surgery, Graduate School of Medicine, Kyoto University, Kyoto 606-8507, Japan</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:ryao@jfcr.or.jp">ryao@jfcr.or.jp</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100780">10.1016/j.xpro.2021.100780</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Patient-derived organoids (PDOs) recapitulate the cellular heterogeneity of the original colorectal tumor tissue. Here, we describe a protocol to generate genetically modified PDOs to investigate cancer stem cells. This protocol uses the CRISPR-Cas9 system to knock-in the IRES-EGFP-P2A-iCaspase9 cassette into the 3′ UTR of the potential cancer stem cell marker gene, which allows us to investigate their potential for self-replication and pluripotency. We describe the procedure for generating mutant PDOs and their application for stem cell research.</p>
<p>For complete details on the generation and use of this protocol, please refer to Okamoto et al. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref2">Okamoto et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A detailed protocol for efficient CRISPR-Cas9 mediated knock-in in PDOs</li>
<li>Generation of IRES-EGFP-P2A-iCaspase9 cassette for simple preparation of donor vector</li>
<li>Visualization of cancer stem cells in PDOs using time-lapse imaging microscopy</li>
<li>Specific elimination of cancer stem cells by adding AP20187</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1020-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes the general procedure to culture PDOs. PDOs were established according to procedures, which were approved by the Research Ethics Board at the JFCR Cancer Institute (Tokyo, Japan). The procedure can be used to culture PDOs derived from colorectal cancer (CRC) patients of various ages and gender.</p>
<h3 id="sec1.1">Preparation of Noggin conditioned medium</h3>
<div class="note">
<span class="note-title">Note:</span> We generated Noggin-expressing L1 cells to prepare conditioned medium. For details on the generation of Noggin-expressing L1 cells, please refer to Sakahara et al. (<a href="#bib3">Sakahara et al., 2019</a>). The Noggin conditioned medium can be replaced by commercially available recombinant protein (Preprotec, Cat# 120-10C).</div>
<ol>
<li>Cryopreserved Noggin-expressing L1 cells are thawed and cultured in a 100-mm dish in 10% fetal bovine serum (FBS)/DMEM. G418 is added to the medium starting from three days after thawing at a concentration of 800 μg/mL.</li>
<li>The cells are trypsinized withTrypLE enzyme and are split at a ratio of 1:10 into 100-mm dishes in 10% FBS/DMEM. The medium at this and following steps does not contain G418. Typically, we use 100 plates to prepare the conditioned medium.</li>
<li>Cells are cultured until reaching confluence.</li>
<li>The medium is removed, and 10 mL of Advanced DMEM/F12 Supplemented with 2 mM GlutaMAX, 10 mM HEPES, 100 μg/mL each of Pen/Strep (basal medium) is added to each 100-mm dish.</li>
<li>The conditioned medium is collected after incubation for 4 days, then spun at R.T. at 500 × <i>g</i> for 5 min. The supernatant is saved as the 1st conditioned medium at −20°C.</li>
<li>Ten milliliters of basal medium are added to each 100-mm dish.</li>
<li>The conditioned medium is collected as described in step 5 and stored as the 2nd conditioned medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The 1st and 2nd conditioned media are reserved separately until their Noggin activities are determined. To avoid repeated freeze/thaw cycles, 1ml of each media is set aside and is used for the luciferase assay as described below. The remainder is dispensed into 50ml centrifuge tubes. All conditioned media are stored at −20°C.</div>
<ol start="8">
<li>The activity of Noggin is checked by a dual-luciferase reporter assay using the BMP promoter.</li>
<li>293T cells are plated at 1×10ˆ4 cells/well in collagen-coated 96-well plates and incubated at 37°C overnight.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use collagen-coated dish to prevent 293T cells from detaching the plate during the transfection of reporter plasmids。</div>
<ol start="10">
<li>DNA diluent is prepared by mixing the pGL3-BRE plasmid, pRL-tk plasmid and Opti-MEM. Lipofectamine diluent is prepared by mixing Lipofectamine 2000 in Opti-MEM and incubated at R.T. for 5 min. DNA- and lipofectamine-diluent are mixed and incubated at R.T. for 15 min (<a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. Preparation of transfection reagents for dual-luciferase assay</caption>
<thead>
<tr>
<th> </th>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount (per 100 wells)</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="3">DNA-diluent</td>
<td>pGL3-BRE</td>
<td>10 μM</td>
<td>5 μg</td>
</tr>
<tr>
<td>pRL-tk</td>
<td>10 μM</td>
<td>0.5 μg</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>N/A</td>
<td>2500 μL</td>
</tr>
<tr>
<td rowspan="2">lipofectamine-diluent</td>
<td>Lipofectamine 2000</td>
<td>N/A</td>
<td>50 μL</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>N/A</td>
<td>2500 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Next, 10 mL of 10% FBS/DMEM is added, and 150 μL is dispensed to each well of 293T-plated 96-well dishes, and incubated at 37°C overnight.</li>
<li>Test conditioned medium is prepared by serial dilutions ranging from 2- to 16-fold containing 2 and 5 ng of BMP4 solution. Two different concentrations of BMP4 is used to precisely determine the Noggin activity of the conditioned media.</li>
<li>The medium is changed to test the conditioned medium, and the cells are incubated at 37°C for 24 h.</li>
<li>The luciferase activities are measured using the Dual-Luciferase Reporter Assay System according to the manufacturer’s instructions. Typically, x8 and x4 dilutions of conditioned medium release the inhibition of BRE promoter activity of BMP4 at 2 and 5 ng/mL, respectively (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1020-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Evaluation of Noggin-conditioned medium</div>
<p>The value of pGL3-BRE (Firefly) was normalized to that of pRL-tk (Renilla). The activity of Noggin-conditioned medium was determined from the inhibition of BMP4-induced transcriptional activity as measured by pGL3-BRE. Cells were treated with 2 or 5 ng/mL of recombinant BMP4 and indicated dilution of Noggin conditioned medium, and data are shown as relative values compared to those obtained by twofold dilution of Noggin-conditioned medium. Data are represented as mean ± SEM.</p>
</figcaption>
</figure>
</li>
<li>The Noggin conditioned medium can be stored at −20°C for 6 months. Repeated freezing and thawing should be avoided.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The activity of 1st and 2nd conditioned media are measured separately. When both medium show sufficient activities, they are combined and the mixture is used for further study. If either medium does not have sufficient activity, only the active medium is used.</div>
<h3 id="sec1.2">PDOs culture</h3>
<div class="note">
<span class="note-title">Note:</span> This protocol is set up for a set of PDOs established from stage IV CRC, which can be cultured in ENR medium. The medium most suitable for each PDOs should be used for efficient genome editing and for the evaluation of stemness.</div>
<ol start="16">
<li>The cryopreserved PDOs are placed in 10 mL of basal medium in a 15 mL conical tube.</li>
<li>PDOs are spun at R.T. for 500 ×<i>g</i> for 5 min. The supernatant is removed and PDOs pellets are put on ice.</li>
<li>PDOs are resuspended in Matrigel, and 25 μL is dispensed into each well of a prewarmed 48-well plate. Typically, 10–20 PDOs are resuspended in 25 μL of Matrigel.</li>
<li>The plate is placed in a 37°C incubator for 10 min to solidify the Matrigel, and then 250 μL of ENR medium is added.</li>
<li>The medium is changed every 3 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Generally, PDOs recover their original morphology at 3–5 days after thawing. Healthy PDOs subsequently increase their sizes.</div>
<ol start="21">
<li>To expand the culture, the PDOs are collected in 10 mL of basal medium in a 15 mL conical tube. Typically, PDOs are expanded every 7–10 days.</li>
<li>PDOs are collected, plated and incubated as described in steps 17–19.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Stable3</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# C737303</td>
</tr>
<tr>
<td>DH5alpha</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 18265017</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Matrigel</td>
<td>Corning</td>
<td>Cat# 356231</td>
</tr>
<tr>
<td>Advanced DMEM/F-12</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12634-028</td>
</tr>
<tr>
<td>GlutaMAX</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 35050-061</td>
</tr>
<tr>
<td>HEPES</td>
<td>Sigma-Aldrich</td>
<td>Cat# H3537</td>
</tr>
<tr>
<td>Penicillin/Streptomycin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 15140-122</td>
</tr>
<tr>
<td>Primocin</td>
<td>InvivoGen</td>
<td>Cat# ant-pm-2</td>
</tr>
<tr>
<td>B27 Supplement</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# A1895601</td>
</tr>
<tr>
<td>N-Acetyl-L-cysteine</td>
<td>Sigma-Aldrich</td>
<td>Cat# A9165</td>
</tr>
<tr>
<td>Gastrin</td>
<td>Sigma-Aldrich</td>
<td>Cat# G9145</td>
</tr>
<tr>
<td>Recombinant Mouse EGF</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# PMG8043</td>
</tr>
<tr>
<td>Recombinant Mouse R-Spondin1</td>
<td>R&amp;D systems</td>
<td>Cat# 3474-RS</td>
</tr>
<tr>
<td>Noggin CM</td>
<td>In house production</td>
<td>N/A</td>
</tr>
<tr>
<td>TrypLE Express Enzyme</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12604-013</td>
</tr>
<tr>
<td>Y-27632</td>
<td>Sigma-Aldrich</td>
<td>Cat# Y0503</td>
</tr>
<tr>
<td>Opti-MEM</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 31985-062</td>
</tr>
<tr>
<td>Puromycin</td>
<td>Sigma-Aldrich</td>
<td>Cat# P9620</td>
</tr>
<tr>
<td>Neomycin (G418)</td>
<td>Sigma-Aldrich</td>
<td>Cat# N1876</td>
</tr>
<tr>
<td>Cell Recovery solution</td>
<td>Corning</td>
<td>Cat# 354243</td>
</tr>
<tr>
<td>Recovery Cell Culture Freezing Medium</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 12648010</td>
</tr>
<tr>
<td>PBS</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 14190-144</td>
</tr>
<tr>
<td>AP20187</td>
<td>Clontech</td>
<td>Cat# 635058</td>
</tr>
<tr>
<td>Lipofectamine 2000</td>
<td>Thermo Fisher Scientific</td>
<td>Cat #11668027</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAamp DNA Mini Kit</td>
<td>QIAGEN</td>
<td>Cat# 51304</td>
</tr>
<tr>
<td>PCR DIG Probe Synthesis Kit</td>
<td>Roche</td>
<td>Cat# 11636090910</td>
</tr>
<tr>
<td>DIG Luminescent Detection Kit</td>
<td>Roche</td>
<td>Cat# 11363514910</td>
</tr>
<tr>
<td>Dual-Luciferase Reporter Assay System</td>
<td>Promega</td>
<td>Cat# E1980</td>
</tr>
<tr>
<td>DNA Ligation Kit Ver.1</td>
<td>Takara</td>
<td>Cat# 6021</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>Noggin1-expressing L1 cells</td>
<td>(<a href="#bib3">Sakahara et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Patient-derived organoids</td>
<td>(<a href="#bib2">Okamoto et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Southern blot probe: hOLFM4-F6:CATGGTGGTGTGGTGAACATCAGC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Southern blot probe: hOLFM4-R7:GCCCTCTTTCCCTGTGTTTGTGTC</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Southern blot probe: RFP-F1:CTCCGTGAACGGCCACGAGTTCGAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Southern blot probe: RFP-R1:GGTGGTCTTGACCTCGGCGTCGTAG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Southern blot probe: EGFP-F1:GGCGATGCCACCTACGGCAAGCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Southern blot probe: EGFP-R1:CTGCTGGTAGTGGTCGGCGAGCTG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>OLFM4-Donor-Top oligo: hOLFM4-DONOR11-5SCCGG CCTGCAGATATCTAAGTAAGTGG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>OLFM4-Donor-Bottom oligo: hOLFM4-DONOR11-5ASGGCC CCACTTACTTAGATATCTGCAGG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>OLFM4-Donor-Top oligo: hOLFM4-DONOR11-3SCCTGCAGATATCTAAGTAAGTGG</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>OLFM4-Donor- Bottom oligo: hOLFM4-DONOR11-3ASCTAG CCACTTACTTAGATATCTGCAGG AT</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pRL-tk</td>
<td>Promega</td>
<td>Cat# E2241</td>
</tr>
<tr>
<td>pGL3-BRE</td>
<td>Addgene</td>
<td>Cat# 45126</td>
</tr>
<tr>
<td>pCMMP-MCS-IRES-eGFP</td>
<td>Addgene</td>
<td>Cat# #36953</td>
</tr>
<tr>
<td>pCMMP-MCS-IRES-mRFP</td>
<td>Addgene</td>
<td>Cat# #36972</td>
</tr>
<tr>
<td>pCMMP-MCS-IRES-eGFP-P2A-iCas9</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pCMMP-MCS-IRES-eGFP-P2A-iCas9-OLFM4</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pX330</td>
<td>Addgene</td>
<td>Cat# 42230</td>
</tr>
<tr>
<td>pX330-hOLFM4</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>pCR-hOLFM4</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>PB-CMV-MCS-EF1a-Puro</td>
<td>SBI</td>
<td>Cat# PB510B-1</td>
</tr>
<tr>
<td>PB-EF1a-MCS-IRES-Neo</td>
<td>SBI</td>
<td>Cat# PB533A-2</td>
</tr>
<tr>
<td>Super piggyBac transposase expression vector</td>
<td>SBI</td>
<td>Cat# PB210PA-1</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Super electroporator NEPA21 type II</td>
<td>Nepa Gene</td>
<td><a href="http://www.nepagene.jp/products_nepagene_0001.htmL">http://www.nepagene.jp/products_nepagene_0001.htmL</a></td>
</tr>
<tr>
<td>ChemiDoc XRS+</td>
<td>Bio-Rad</td>
<td>Cat# 1708265J1PC</td>
</tr>
<tr>
<td>CellVoyager CV1000</td>
<td>Yokogawa</td>
<td><a href="https://www.innervision.co.jp/041products/2010/p1001_14etc.htmL">https://www.innervision.co.jp/041products/2010/p1001_14etc.htmL</a></td>
</tr>
<tr>
<td>40 μm cell strainer</td>
<td>Falcon</td>
<td>Cat# 352340</td>
</tr>
<tr>
<td>15 mL Centrifuge tube</td>
<td>IWAKI</td>
<td>Cat# 2425-015</td>
</tr>
<tr>
<td>50 mL Centrifuge tube</td>
<td>IWAKI</td>
<td>Cat# 2445-050</td>
</tr>
<tr>
<td>48-Well plate</td>
<td>IWAKI</td>
<td>Cat# 3830-048</td>
</tr>
<tr>
<td>100 mm Dish</td>
<td>IWAKI</td>
<td>Cat# 3020-100</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment-(optional)">Materials and equipment (optional)</h2>
<table id="undtbl1">
<caption>organoid culture medium (ENR medium)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Advanced DMEM/F12Supplemented with 2 mM GlutaMAX, 10 mM HEPES, 100 U/mL Pen, 100 μg/mL Strep(basal medium)(store at 4°C for three months)</td>
<td>N/A</td>
<td>8,764 μL</td>
</tr>
<tr>
<td>B27 minus vitamin A (50<b>×</b>)(store at −20°C for 6 months)</td>
<td>1<b>×</b>
</td>
<td>200 μL</td>
</tr>
<tr>
<td>N-acetylcysteine (500 mM)(store at −20°C for 6 months)</td>
<td>1.25 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>Gastrin (100 μM)(store at −20°C for 6 months)</td>
<td>0.01 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>EGF (100 μg/mL)(store at −20°C for 6 months)</td>
<td>50 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>R-Spondin 1 (100 μg/mL)(store at −20°C for 6 months)</td>
<td>50 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Noggin CM(store at −20°C for 6 months)</td>
<td>10%</td>
<td>1,000 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>N/A</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This protocol describes the knock-in of the marker cassette into the OLFM4 locus, which we identified as stem cell markers of PDOs established from CRC (<a href="#bib2">Okamoto et al., 2021</a>). However, we have also successfully used this protocol to visualize and evaluate their progeny. Thus, the main steps of the protocol can be widely adapted for analyzing various types of cells.</p>
<p>CRISPR/Cas9 (clustered regularly interspaced short palindromic repeats (CRISPR)-associated protein 9) technology has been widely used to manipulate the genome of mammalian cells. To visualize stem cells, in-frame insertion of a marker gene into a gene of interest has been widely used. However, this modification inevitably disrupts the expression of genes of interest. Additionally, we have experienced a lack of expression of inserts, most likely due to epigenomic modification (<a href="#bib4">Schuijers et al., 2014</a>). Based on these observations, we inserted IRES marker gene cassettes into the 3′ UTR of the target gene.</p>
<p>The transfection efficiency varies among PDOs, and we adopted the method of transposon-based integration with electroporation developed by Fujii et al., which allows drug selection of transfected cells using the PiggyBac system(<a href="#bib1">Fujii et al., 2015</a>). We used the homology-independent targeted integration method (HITI) (<a href="#bib6">Suzuki et al., 2016</a>) because it allowed highly efficient knock-in of donor DNA at the target sites. Generally, 5%–10% of clones harbored the expected genome editing.</p>
<h3 id="sec3.1">sgRNA design and preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 week</b>
</div>
<ol>
<li>sgRNA are designed using online tools, including CHOPCHOP (<a href="http://chopchop.cbu.uib.no">http://chopchop.cbu.uib.no</a>), CRISPOR (<a href="http://crispor.tefor.net">http://crispor.tefor.net</a>) and CRISPick (<a href="https://portals.broadinstitute.org/gppx/crispick/public">https://portals.broadinstitute.org/gppx/crispick/public</a>).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is important to confirm whether the crRNA recognition sequences of PDOs contain mutations or SNVs. We recommend exome sequencing because it also provides information about chromosome aneuploidy. Sanger sequencing of the PCR amplicon is an alternative choice for verification.</div>
<ol start="2">
<li>To prepare the sgRNA oligonucleotides for vector insertion, the sense and antisense oligonucleotides are mixed in a 0.2 mL tube following volumes.</li>
</ol>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Top oligo (100 μM)</td>
<td>10 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Bottom oligo (100 μM)</td>
<td>10 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>8 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<ol start="3">
<li>The tube is placed in a thermocycler, incubated at 95°C for 5 min, and left at RT for 1 h.</li>
<li>The annealed oligos are diluted into 250 μL ddH<sub>2</sub>O.</li>
<li>The plasmid, pX330, is digested with Age I by mixing the reaction following reagents and volumes. Incubated at 37°C for 2 h.</li>
</ol>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Plasmid</td>
<td>0.01 μg/μL</td>
<td>10 μL</td>
</tr>
<tr>
<td>Buffer, 10<b>×</b>
</td>
<td>1<b>×</b>
</td>
<td>10 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>78 μL</td>
</tr>
<tr>
<td>Age I (5000 U/mL)</td>
<td>0.1 U/mL</td>
<td>2 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>100 μL</td>
</tr>
</tbody>
</table>
<ol start="6">
<li>Digested plasmid is purified by phenol/chloroform extraction, followed by ethanol precipitation.</li>
<li>The annealed sgRNA oligonucleotide and digested plasmid are ligated and incubated at 16°C for 30 min according to the manufacturer’s instructions (DNA Ligation Kit Ver. 1).</li>
</ol>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Plasmid</td>
<td>0.003 pmol/μL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Diluted sgRNA oligo (0.4 μM)</td>
<td>6.7 nM</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>4 μL</td>
</tr>
<tr>
<td>Reaction buffer (A)</td>
<td>N/A</td>
<td>40 μL</td>
</tr>
<tr>
<td>Enzyme solution (B)</td>
<td>N/A</td>
<td>10 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>60 μL</td>
</tr>
</tbody>
</table>
<ol start="8">
<li>10 μL of ligation product is transferred into 100 μL of a competent Stbl3 strain, incubated for 30 min, heat-shocked at 42°C for 60 s and incubated on ice for 3 min. Next, 500 μL of SOC medium is added and incubated at 37°C for 30 min, and 100 μL of suspension is spread onto an LB plate containing 100 μg/mL ampicillin. The plate is incubated at 37°C for 16 h.</li>
<li>Colonies are picked and cultured in 3 mL LB medium, and the plasmid is purified using a QIA prep Spin Miniprep Kit.</li>
<li>Plasmids are verified by using the T7sgRNA-R primer for sequencing and further prepared using a QIA prep MAXI column.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Endotoxin-free plasmids are necessary for transfer into organoids.</div>
<h3 id="sec3.2">Preparing donor plasmids</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>1 week</b>
</div>
<p>The plasmids pCMMP-MCS-IRES-EGFP (Addgene #36953) and pCMMP-MCS-IRES-mRFP (Addgene #36972) are two useful resources to visualize stem cells with EGFP and RFP, respectively. We generated an EGFP-inducible Caspase9 fusion gene cassette, pCMMP-MCS-IRES-EGFP-P2A-inducible Caspase9, which allows the ablation of target cells by exposing genome-edited PDOs to AP20187 , a chemical compound that homodimerizes Caspase9 and induces apoptosis (<a href="#fig2">Figure 2</a>). All these constructs have AgeI and NotI sites at the 5′ region and PacI and NheI sites at the 3′ region, which allows the directional cloning of target sequences of sgRNA.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1020-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Donor plasmid construction</div>
<p>(A) Schematic representation of a donor fragment using the pCMMP-MCS-IRES-eGFP-P2A-iCas9 plasmid. Two annealed oligonucleotides were inserted into the AgeI/NotI site at the 5′ end and the PcaI/NheI site at the 3′ end.</p>
<p>(B) Nucleotide sequences of 5′- and 3′-insertion sites in pCMMP-MCS-IRES-eGFP-P2A-iCas9.</p>
</figcaption>
</figure>
<ol start="11">
<li>Top and bottom oligonucleotides are prepared as shown in <a href="#fig2">Figure 2</a>B. Note that the top and bottom oligonucleotides are to be inserted into the 5′ cloning sites of the donor vector contain AgeI and NotI cloning sites, and those to be inserted into the 3′ cloning sites contain PacI and NheI sites.</li>
<li>Anneal oligonucleotides used for insertion at the 5′ cloning site as described in steps 2–4.</li>
<li>The pCMM-MCS-IRES-eGFP-iCas9 plasmid is digested with AgeI, and the fragment was purified as described in steps 5–6.</li>
<li>Dissolve the digested plasmid in 90 μL of ddH<sub>2</sub>O.</li>
<li>The purified AgeI-digested plasmid is digested with NotI as described in step 5.</li>
<li>The AgeI/NotI double-digested plasmid is purified by nucleospin gel and PCR clean-up kits. The plasmid can be eluted twice with 20 μL of ddH<sub>2</sub>O, and the final volume of the digested plasmid should be 40 μL.</li>
<li>DNA concentration is determined using spectrophotometer.</li>
<li>The annealed sgRNA oligonucleotide and digested plasmid are mixed according to the manufacturer’s instructions (DNA ligation kit Ver. 1, Takara) and incubated at 16°C for 30 min. We recommend to use 25 pmol of sgRNA oligonucleotide and 100 fmol of digested plasmid for efficient ligation.</li>
<li>Competent DH5alpha is transformed with the ligated product, and the plasmid is prepared as described in steps 8–10.</li>
<li>The annealed oligonucleotide used for insertion at the 3′ cloning site has appealed and is as described in step 12.</li>
<li>Digest pCMM-MCS-IRES-eGFP-iCas9 in which double-stranded oligonucleotide cloned in the 5′ cloning site with PacI and purify as described in step 13–15.</li>
<li>The fragment is digested from step 22 with NheI and purified as described in steps 16 and 17.</li>
<li>The annealed oligonucleotide from step 21 and PacI/NheI double-digested plasmid are described in step 18.</li>
<li>The competent Stbl3 strain is transformed, and the plasmid is prepared as described in step 20.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Stbl3 is recommended because the donor plasmid used for HITI contains insertions of identical sequences at 5′ and 3′ cloning sites.</div>
<h3 id="sec3.3">PDOs electroporation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>2 hours</b>
</div>
<p>We used a NEPA21 electroporator (Nepa Gene) to transfer plasmid DNA to PDOs. The electroporation protocol was originally developed for NHEJ-mediated gene disruption and for HDR-mediated gene modification (<a href="#bib1">Fujii et al., 2015</a>). We have modified this protocol to optimize HITI-mediated knock-in of the marker cassette. For complete details of the organoid electroporation, please refer to the original protocol (<a href="#bib1">Fujii et al., 2015</a>).</p>
<ol start="25">
<li>Passage PDOs 3 days before electroporation.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The population of OLFM4-expressing cells increases during the 3–4 days after passage and then decreases thereafter.</div>
<div class="note">
<span class="note-title">Note:</span> Typically, 1 to 2<b>×</b>10ˆ5 cells were collected from 1-well of PDOs, and 6 wells of PDOs were sufficient for each electroporation.</div>
<ol start="26">
<li>The medium is removed, and 500 μL of TrypLE solution supplemented with 10 μM Y-27632 is added to each well. PDO Matrigel is collected into a 15 mL centrifuge tube using a 1000 μL pipette.</li>
<li>Cells are incubated in 37°C water bath for 15–30 min until Matrigel is properly dissolved in TrypLE solution.</li>
<li>Basal medium is added up to 10 mL, and the mixture is pipetted and filtered through 40 μm cell strainer.</li>
<li>The PDOs at R.T. are spun at 500 ×g for 5 min, remove the supernatant.</li>
<li>One milliliter of opti-MEM is added. Pipette with 1000 μL pipette well and count the cell number.</li>
<li>PDOs are washed at R.T. at 500 ×g for 5 min, and the supernatant is removed.</li>
<li>Cells are resuspended at 2<b>×</b>10ˆ6/mL in Opti-MEM.</li>
<li>Transfection components are prepared, following reagents and volumes.</li>
</ol>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>piggyBac Vector (1 μg/mL)(PB-CMV-MCS-EF1a-Puro or PB-EF1a-MCS-IRES-Neo)</td>
<td>50 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Super piggyBac transposase expression vector (1 μg/mL)</td>
<td>10 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>sgRNA expressing vector (5 μg/mL)</td>
<td>100 μg/mL</td>
<td>2μL</td>
</tr>
<tr>
<td>Donor vector (5 μg/mL)</td>
<td>100 μg/mL</td>
<td>2μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>10 μL</td>
</tr>
</tbody>
</table>
<ol start="34">
<li>Ninety microliters of cell suspension from step 33 are added to the electroporation cuvette.</li>
<li>Cells are electroporated with the following settings.</li>
</ol>
<table id="undtbl6">
<thead>
<tr>
<th> </th>
<th>Poring pulse</th>
<th>Transfer pulse</th>
</tr>
</thead>
<tbody>
<tr>
<td>Voltage</td>
<td>175 V</td>
<td>20 V</td>
</tr>
<tr>
<td>Pulse Length</td>
<td>5 ms</td>
<td>50 ms</td>
</tr>
<tr>
<td>Pulse interval</td>
<td>50 ms</td>
<td>50 ms</td>
</tr>
<tr>
<td>Number of pulses</td>
<td>2</td>
<td>5</td>
</tr>
<tr>
<td>Decay rate</td>
<td>10%</td>
<td>40%</td>
</tr>
<tr>
<td>Polarity</td>
<td>+</td>
<td>+/−</td>
</tr>
</tbody>
</table>
<ol start="36">
<li>Next, 400 μL of opti-MEM supplemented with 10 μM Y-27632 is added to the electroporation cuvette, and the cells are transferred into a 15 mL centrifuge tube using the plastic pipette provided with the cuvette.</li>
<li>The cells are left at R.T. for 30 min.</li>
<li>A centrifuge tube at R.T. at 500 ×g for 5 min.</li>
<li>Remove the supernatant and suspend the pellet in 200 μL of Matrigel.</li>
<li>Plate cells in 25 μL of Matrigel onto each well of a prewarmed 48-well plate.</li>
<li>The plate is incubated at 37°C for 10 min to solidify the Matrigel.</li>
<li>ENR medium supplemented with 10 μM of Y-27632 is added.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The stock solution of Y-27632 (10mM) is stored at −20°C for 6 months and is added to the medium just prior to use.</div>
<ol start="43">
<li>The cells are incubated at 37°C for 24 h, and the medium is replaced with 250 μL of ENR medium.</li>
<li>The medium is changed every 3 days.</li>
</ol>
<h3 id="sec3.4">Selection of drug resistant clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>3 weeks</b>
</div>
<p>To select the transfected cells, we used puromycin at a concentration of 2 μg/mL when using PB-CMV-MCS-EF1a-Puro. Alternatively, neomycin can be used at a concentration of 800 μg/mL when using PB-EF1a-MCS-IRES-NEO.</p>
<ol start="45">
<li>The selection is started by adding a selective agent to the medium starting from day 5 after electroporation.</li>
<li>The medium is supplemented with a selective agent every 3–4 days.</li>
<li>Typically, 2–3 resistant clones appear in each well. The expression of fluorescent proteins can be detected under a stereomicroscope when the expression level of the target gene is high. (<a href="#fig2">Figure 2</a>A)</li>
<li>Isolate the organoids by gentle pipetting using 200 μL of pipette and transfer onto a 100-mm dish. Single organoids were picked up and transferred to each well of a 96-well U-bottom plate (<a href="#fig3">Figure 3</a>).
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1020-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Picking up the PDOs</div>
<p>(A) The organoids were pipetted into 48-well plates, placed on 100-mm dishes, and observed under a stereomicroscope equipped with fluorescent light.</p>
<p>(B) Using a 200 μL pipette, the EGFP-positive clone was picked up and transferred to a 96-well U-bottom plate.</p>
</figcaption>
</figure>
</li>
<li>Add 10 μL of Matrigel and incubate the plate in a 37°C incubator for 10 min to solidify the Matrigel.</li>
<li>Add 100 μL of ENR medium supplemented with a selective agent.</li>
<li>Each organoid is cultured in ENR medium supplemented with a selective agent.</li>
<li>For passaging, modified PDOs are transferred into 15 mL centrifuge tubes, centrifuged at R.T. for 5 min, and carefully removed the supernatant with a 1 mL pipette.</li>
<li>Then, 50 μL of Matrigel is added to each well, and a 25 μL droplet is placed in each well of a 48-well plate after pipetting well.</li>
<li>Next, 250 μL of ENR medium is added, and the medium is changed every 3 days.</li>
<li>To cryopreserve the PDOs, they are collected using a 1 mL pipette into 15 mL centrifuge cells.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> PDOs are collected three to four days after passage, during vigorous growth. Overgrowth of PDOs significantly reduces the recovery efficiency.</div>
<ol start="56">
<li>The centrifuge tube is spun R.T. at 500 ×g for 5 min, and the supernatant is carefully removed.</li>
<li>One milliliter of Recovery Cell Culture Freezing Medium is added and transferred to 2 mL cryovials.</li>
<li>The tubes are stored at −80°C for up to 1 week.</li>
<li>The frozen cryovials are transferred to a liquid N<sub>2</sub> tank.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Longer storage at −80°C significantly reduces the recovery efficiency.</div>
<h3 id="sec3.5">Isolation of genomic DNA to screen PDO clones</h3>
<div class="timing">
<span class="timing-title">Timing:</span> <b>6 hours</b>
</div>
<p>The clonal populations of PDOs are used for identification of genome editing. Each clone is expanded from single well in 96 well plate to multiple wells of 48 well plate. Typically, six to eight wells of a 48-well plate yield 20–40 μg of genomic DNA, which is sufficient to identify genome-edited clones.</p>
<ol start="60">
<li>To isolate genomic DNA from organoids, the medium is removed from each well.</li>
<li>Add 500 μL of cell recovery solution to each well, collect the organoids into a 15 mL centrifuge tube and leave on ice for 30 min with occasional pipetting.</li>
<li>The PDOs are spun at R.T. at 500 ×g for 5 min to remove the supernatant.</li>
<li>One milliliter of PBS is added, and the PDOs are transferred to a 1.5 mL centrifuge tube.</li>
<li>The PDOs are spun at 4°C for 5 min, and the supernatant is carefully removed using a 1 mL pipette.</li>
<li>The PDO pellet is frozen in liquid N<sub>2</sub>. Frozen samples can be preserved at −80°C until use.</li>
<li>Genomic DNA is extracted using a QIAamp DNA mini kit following the manufacturer’s instructions.</li>
</ol>
<h3 id="sec3.6">Southern blotting</h3>
<p>Genetic modification can be identified by PCR-based screening. However, HITI-mediated knock-in often causes unintended modification, such as tandem duplicated integration. Southern blotting validates the structure of the targeted alleles and integrations of the donor vectors.</p>
<p>For complete details on Southern blotting, please refer to Southern (<a href="#bib5">Southern, 2006</a>).</p>
<ol start="67">
<li>Next, 30 μL of 10<b>×</b> restriction enzyme buffer is added to 14 μg of DNA in 270 μL of TE.</li>
<li>Genomic DNA was digested by mixing the reaction following reagents and volumes. Incubated at 37°C overnight.</li>
</ol>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Genomic DNA</td>
<td>0.01 μg/μL</td>
<td>14 μg</td>
</tr>
<tr>
<td>Buffer, 10<b>×</b>
</td>
<td>1<b>×</b>
</td>
<td>30 μL</td>
</tr>
<tr>
<td>Enzyme (100,000units/mL)</td>
<td>1 unit/μL</td>
<td>3 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>N/A</td>
<td>up to 300 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>300 μL</td>
</tr>
</tbody>
</table>
<ol start="69">
<li>Next, 1 μL of restriction enzyme is added and incubated for an additional 2 h.</li>
<li>30 μL of 3 M sodium acetate (pH 7.0) and 750 μL of ethanol are added.</li>
<li>DNA is spun down at 4°C at 15,000 ×g for 30 min, and the supernatant is discarded.</li>
<li>Add 700 μL of 70% ethanol, spin DNA down at 4°C at 15,000 × g for 30 min, and carefully discard the supernatant.</li>
<li>Air dry the pellet by leaving the tube open on the bench for 15 min.</li>
<li>Dissolve the pellet in 10 μL of TE and add 2 μL of 6<b>×</b> loading dye.</li>
<li>For a 15 cm <b>×</b> 15 cm gel, 150 mL of 0.7% agarose is prepared in boiling 1<b>×</b> TBE and cooled.</li>
<li>The gel is placed in the tank and covered with 1<b>×</b> TBE, and the comb is removed to load the sample.</li>
<li>Run overnight at 20 V.</li>
<li>The gel is imaged on a UV trans illuminator after staining the gel by adding 6 μL of 10 mg/mL in a 300 mL TBE for 30 min.</li>
<li>The gel is transferred to a tray for 15 min with 500 mL of 0.5 N NaOH/1.5 M NaCl, and a second 30 min wash is performed.</li>
<li>The gel is rinsed three times with distilled water.</li>
<li>The gel is transferred to a tray for 30 min with 500 mL of 1.5 M NaCl/0.5 M Tris-HCl (pH 7.5), and repeat a second 15 min wash.</li>
<li>The transfer apparatus is assembled by placing a piece of blotting paper on a glass plate and putting the support inside a large baking dish.</li>
<li>The dish is filled with 10<b>×</b> SSC.</li>
<li>The Hybond-N+ membrane is soaked in 10<b>×</b> SSC for 5 min.</li>
<li>The gel from step 82 is removed and inverted so that it underside is uppermost.</li>
<li>The membrane is placed on top of the gel and any bubbles removed.</li>
<li>Two thick blotting papers are placed on top of the membrane.</li>
<li>Paper towels are placed on the blotting papers.</li>
<li>A glass plate is placed on the top of the stack and weighed to 500 g.</li>
<li>Allow the transfer of DNA overnight.</li>
<li>The paper towels and the blotting paper are removed, the position of the gel slots on the membrane is marked, and the gel is peeled from the membrane.</li>
<li>the membrane is soaked in 6<b>×</b> SSC for 5 min and air dried, and DNA is fixed to membrane by UV irradiation (0.12 J/cm<sup>2</sup>).</li>
</ol>
<h3 id="sec3.7">Detection of target allele and inserted fragment</h3>
<p>Digoxigenin (DIG)-labeled probes are sensitive enough to detect wild-type and mutant alleles of PDOs. A segment corresponding to the genomic sequence of OLFM4 (hOLFM4 probe) is employed to validate the insertion into the expected site, and that corresponding to the EGFP sequence (EGFP probe) is used to validate the copy number of the donor vector (<a href="#fig4">Figure 4</a>). We generally validate the insertion of the donor vector into the OLFM4 locus first and then evaluate the copy number of EGFP after reprobing the first probe.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1020-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Southern blot analysis of HITI-mediated genome editing</div>
<p>(A) Schematic representation of knock-in of the IRES-EGFP-P2A-iCas9 cassette into the 3′ UTR of the OLFM4 gene. The hOLFM4 probe detected 11.8 kb and 5.1 kb fragments of wild-type and KI alleles, respectively. The EGFP probe identified a 9.4 kb fragment of KI-allele.</p>
<p>(B) Analysis of 16 puromycin-resistant clones. Five clones contained monoallelic mutation (shown by yellow numbers), and five clones harbored biallelic mutations (shown by red numbers).</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1020-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. PDOs harboring the IRES-EGFP-P2A-iCas9 fragment in the 3′ UTR of the OLFM4 gene</div>
<p>(A) Organoids released from Matrigel were incubated with Hoechst 33342 and examined under confocal laser microscopy.</p>
<p>(B and C) Time-lapse imaging of PDO. PDOs were embedded into Matrigel on glass-bottom plates and cultured for three days with (B) or without AP20147 (C). Note that EGFP expression of PDOs was decreased (B), because OLFM4-postive stem cells reduced their self-replication potential late after passage (<a href="#bib2">Okamoto et al., 2021</a>).</p>
</figcaption>
</figure>
<ol start="93">
<li>The DIG-labeled probe is prepared using a PCR DIG Probe Synthesis kit (Roche) following conditions. pCR2.1-hOLFM4 and pCMMP-MCS-IRES-EGFP are used as templates for OLFM4 and EGFP probes, respectively. Probes can be stored at −20°C for at least two months.</li>
</ol>
<table id="undtbl8">
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>95°C</td>
<td>2 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="3">10 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>58°C (hOLFM4)67°C (EGFP)</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>7 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<ol start="94">
<li>The hybridization buffer (6<b>×</b> SSC, 5<b>×</b> Denhardt’s solution, 100 μg/mL salmon testis DNA, 0.5% SDS, 50% formamide) is prepared and preheated at 42°C.</li>
<li>Membrane is placed in a plastic bag, into which 250 mL of preheated-hybridization buffer is poured. The bag is sealed and incubated for 1 h.</li>
<li>Twenty microliters of DIG-labeled probe is placed in a 1.5 mL tube, and 30 μL of ddH<sub>2</sub>O is added.</li>
<li>The probe is denatured by placing it in a heat block heated at 100°C for 5 min and immediately placing on ice for 3 min.</li>
<li>Probe solution is added into 10 mL of hybridization buffer preheated at 42°C.</li>
<li>The prehybridization buffer is replaced with hybridization buffer containing a DIG-labeled probe, the bag is sealed, and hybridization is conducted at 42°C overnight.</li>
<li>The membrane is removed and put into a clean plastic tray with a lid.</li>
<li>The membrane is washed for 5 min with 0.1% SDS, 2<b>×</b> SSC at R.T. twice.</li>
<li>The membrane is washed for 30 min with 0.1% SDS and 2<b>×</b> SSC at 65°C once.</li>
<li>The membrane is washed for 5 min with 0.1% SDS, 0.2 <b>×</b> SSC at R.T. once.</li>
<li>The probe is detected using a DIG Luminescent Detection Kit (Roche) according to the manufacturer’s instructions.</li>
<li>To visualize the signal, the membrane is placed with the DNA facing up on Saran wrap. We use a ChemiDoc XRC plus system (Bio-Rad) for visualization, setting the first image to 30, the last image to 120 and the number of images to 4.</li>
<li>To strip the membrane, it is washed with alkaline buffer (0.2 M NaOH, 0.1% SDS) twice for 5 min at R.T., followed by washing with 2<b>×</b> SSC twice.</li>
<li>The process from step 94 is repeated to hybridize with the EGFP probe and detect the signal.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>An example of screening of the HITI-mediated IRES-EGFP-P2A-iCas9 cassette into the 3′UTR of the OLFM4 gene is shown in <a href="#fig4">Figure 4</a>. In this experiment, 62.5% of clones harbored the expected integration, including five biallelic mutations.</p>
<p>The signal intensity of EGFP in PDOs mirrors the expression level of endogenous OLFM4 in the original clone (<a href="#bib2">Okamoto et al., 2021</a>). OLFM4-positive cells were detected in a 3-dimensional image under confocal laser scanning microscopy (<a href="#fig5">Figure 5</a>A). They are also suited for time-lapse imaging analysis (<a href="#fig5">Figure 5</a>B). Note that inducible Caspase 9 effectively induced apoptosis in response to AP20178 exposure.</p>
<p>EGFP-positive clones can be isolated using fluorescence-activated cell sorting (FACS), and the ability of self-replication and multiple differentiation to evaluate stemness can be tested (<a href="#bib2">Okamoto et al., 2021</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol suffers from limitations that are inherent to the expression of knock-in genes. The major limitation is that the expression of marker genes is affected by the integration site. It depends on several factors, such as the stability, splicing variants and ternary structure of mRNA transcribed from the target gene. Despite the large number of knock-in studies, it is still challenging to predict an integration site that faithfully mirrors the expression of target genes. Thus, we strongly recommend testing whether the expression of fluorescent proteins recapitulates that of endogenous before proceeding to in-depth analyses.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low genome editing activity of sgRNA (step 1)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Low efficiency of genome editing occurs due to several reasons. Most likely, the poor potential of sgRNA to induce DSB is the cause of low efficiency. The efficiency of genome editing also depends on PDOs. We recommend designing and synthesizing additional sgRNAs. The efficiency of sgRNA can be tested by the induction of in-dels after introducing sgRNA plasmids into the PDOs of interest.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Considerable cell death after electroporation (step 35).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Up to 2 <b>×</b> 10ˆ6 cells can be used per electroporation. If electroporation kills all cells, reduce the concentration of plasmids, while the ratio of each vectors remains constant. If still all cells are killed, the poring voltage and poring pulse length can be reduced, but this often reduces the efficiency of electroporation.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>PDOs do not grow after electroporation (step 47).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Organoid cells require high activities of niche factors for clonal outgrowth. Check the activities of Noggin conditioned medium and use those having high activity. It should be noted that the R-Spondin activity of recombinant protein might vary among the lots. Several recombinant proteins as well as R-Spondin-expressing cells are commercially available (Merk, Cat#SCC111), and these are the alternatives resources of R-Spondin.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Too many colonies are appeared after drug selection (step 47).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The optimal concentration of puromycin sometimes varies among the PDOs. In our hands, 2 μg/mL is the optimal concentration, but it can be increased up to 10 μg/mL. Optimal concentration on PDOs of interest can be determined by titrating different concentrations of drug and selecting the lowest concentration that kills all cells after 5–7 days.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Loss of organoids while transferring to 96 well plate (step 48).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Try to remove the Matrigel as much as possible by repeated gentle pipetting. The use of low retention tip (Cat# 2069-05, Thermo Fisher Scientific) prevents organoids from sticking them to the tip.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Fujii, M., Matano, M., Nanki, K., and Sato, T. (2015). Efficient genetic engineering of human intestinal organoids using electroporation. Nat. Protoc. <i>10</i>, 1474-1485. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref1">View at publisher</a></p>
<p id="bib2">Okamoto, T., duVerle, D., Yaginuma, K., Natsume, Y., Yamanaka, H., Kusama, D., Fukuda, M., Yamamoto, M., Perraudeau, F., Srivastava, U., et al. (2021). Comparative analysis of patient-matched PDOs revealed a reduction in OLFM4-associated clusters in metastatic lesions in colorectal Cancer. Stem Cell Rep. <i>16</i>, 1-14. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref2">View at publisher</a></p>
<p id="bib3">Sakahara, M., Okamoto, T., Oyanagi, J., Takano, H., Natsume, Y., Yamanaka, H., Kusama, D., Fusejima, M., Tanaka, N., Mori, S., et al. (2019). IFN/STAT signaling controls tumorigenesis and the drug response in colorectal cancer. Cancer Sci. <i>110</i>, 1293-1305. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref3">View at publisher</a></p>
<p id="bib4">Schuijers, J., van der Flier, L.G., van Es, J., and Clevers, H. (2014). Robust cre-mediated recombination in small intestinal stem cells utilizing the olfm4 locus. Stem Cell Rep. <i>3</i>, 234-241. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref4">View at publisher</a></p>
<p id="bib5">Southern, E. (2006). Southern blotting. Nat. Protoc. <i>1</i>, 518-525. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref5">View at publisher</a></p>
<p id="bib6">Suzuki, K., Tsunekawa, Y., Hernandez-Benitez, R., Wu, J., Zhu, J., Kim, E.J., Hatanaka, F., Yamamoto, M., Araoka, T., Li, Z., et al. (2016). In vivo genome editing via CRISPR/Cas9 mediated homology-independent targeted integration. Nature <i>540</i>, 144-149. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00486-X/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Ryoji Yao (<a href="mailto:ryao@jfcr.or.jp">ryao@jfcr.or.jp</a>).</p>
<h4>Materials availability</h4>
<p>Reagents generated within this study are available upon request.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or codes.</p>
<h3>Acknowledgments</h3>
<p>We thank all members of the Yao lab for valuable discussions and practical input, in particular Mayuko Yamamoto, for her help in culturing PDOs. This work was supported in part by <a href="https://doi.org/10.13039/501100001700">MEXT</a>/JSPS KAKENHI grant numbers 16H06276 (R.Y.), 16K14620 (R.Y.), 17H063333 (R.Y.), 18H02684 (R.Y), and 16H06279 (PAGS).</p>
<h3>Author contributions</h3>
<p>T.O. and R.Y. conceived and designed experiments; Y.N., H.Y., and M.F. performed experiments; H. Y. and R.Y. analyzed the data; R.Y. wrote the manuscript; all authors reviewed and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>