<article>
<section>
<h1 id="header">Genetic mosaic dissection of candidate genes in mice using mosaic analysis with double markers</h1>
<p><time datetime="2021-11-19">Published: November 19, 2021</time></p>
<p>Nicole Amberg<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> and Simon Hippenmeyer<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Institute of Science and Technology Austria, Am Campus 1, 3400 Klosterneuburg, Austria</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:simon.hippenmeyer@ist.ac.at">simon.hippenmeyer@ist.ac.at</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100939">10.1016/j.xpro.2021.100939</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Mosaic analysis with double markers (MADM) technology enables the generation of genetic mosaic tissue in mice. MADM enables concomitant fluorescent cell labeling and introduction of a mutation of a gene of interest with single-cell resolution. This protocol highlights major steps for the generation of genetic mosaic tissue and the isolation and processing of respective tissues for downstream histological analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref4">Contreras et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Generation of genetic mosaic mice using mosaic analysis with double markers (MADM)</li>
<li>Tissue harvesting from experimental MADM mice</li>
<li>Processing and imaging of MADM-labeled tissue</li>
<li>Distinct analyses to assess cell-autonomous gene function in MADM mice</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Background</h3>
<p>This protocol describes the generation of genetic mosaic mice using mosaic analysis with double markers (MADM) (<a href="#bib22">Zong et al., 2005</a>; <a href="#bib4">Contreras et al., 2021</a>). A genetic mosaic individual consists of cells of distinct genotypes, for example harboring homozygous mutant cells next to wild-type cells. This phenomenon occurs naturally and is widespread across multicellular organisms. The advantages of using genetic mosaic animals for experimental studies are manifold: The controlled induction of genetic mosaicism in experimental animals allows to alter gene function at high spatiotemporal resolution and has led to many fundamental discoveries of gene function and cellular mechanisms in diverse biological contexts. Genetic mosaicism can be induced sparsely, so that individual mutant cells will be surrounded by unperturbed cells. Thus, mosaicism enables the study of cell-autonomous gene function at single cell resolution. Individual mutant cells can be studied in many ways, for example in lineage tracing, cell competition assays, morphological analysis and disease modelling (<a href="#bib7">Hippenmeyer et al., 2010</a>; <a href="#bib5">Gao et al., 2014</a>; <a href="#bib9">Joo et al., 2014</a>; <a href="#bib1">Beattie et al., 2017</a>; <a href="#bib21">Tian et al., 2020</a>; <a href="#bib4">Contreras et al., 2021</a>).</p>
<p>In this protocol, we present an experimental pipeline for the generation of MADM-labeled mosaic tissues and highlight a variety of downstream analysis applications. Our protocol is based on the use of distinct Cre drivers in combination with MADM cassettes targeted to all 19 mouse autosomes (<a href="#bib4">Contreras et al., 2021</a>).</p>
<p>Mosaic analysis with double markers (MADM) is a genetic technology that allows concomitant fluorescent labeling and introduction of a homozygous mutation of choice with unprecedented single cell resolution in a defined genetic lineage (<a href="#bib22">Zong et al., 2005</a>; <a href="#bib4">Contreras et al., 2021</a>). MADM is based on the use of split marker genes that consist of two reciprocal cassettes of partial coding sequences for green fluorescent protein (GFP) and tandem dimer Tomato (TdT), interspersed by <i>loxP</i> sites (<a href="#fig1">Figure 1</a>A). The two reciprocal MADM cassettes need to be maintained in two different mouse lines, the GT (N-GFP/C-TdT) line and the TG (N-TdT/C-GFP) line (<a href="#bib22">Zong et al., 2005</a>; <a href="#bib7">Hippenmeyer et al., 2010</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The MADM principle</div>
<p>(A) For MADM, two reciprocally chimeric marker genes are targeted to identical loci on homologous chromosomes. The chimeric marker genes (<i>GT</i> and <i>TG</i> alleles) consist of partial coding sequences for green (eGFP[G]) and red (tdT[T]) fluorescent proteins separated by an intron containing the <i>loxP</i> site. Following Cre recombinase-mediated interchromosomal recombination during mitosis, functional green and red fluorescent proteins are reconstituted.</p>
<p>(B) G2-X segregation of the recombinant chromosomes results in two daughter cells each expressing one of the two fluorescent proteins (recombination in G2 of the cell cycle followed by X segregation). Introduction of a mutation distal to one MADM cassette (here the TG cassette) allows the generation of genetic mosaics with single cell resolution, with wild-type daughter cells labeled in red and homozygous mutant sibling cells labelled in green in an unlabeled heterozygous environment.</p>
<p>(C) G2-Z segregation results in two daughter cells with no genotype alteration. One cell shows double labeling (= yellow), while the other daughter cell is unlabeled.</p>
<p>(D) G1 and G<sub>0</sub> events also result in yellow cell labeling without change in genotype.</p>
<p>Reproduced and adapted with permission from (<a href="#bib4">Contreras et al., 2021</a>).</p>
</figcaption>
</figure>
<p>In order to generate genetic mosaic tissue, a modified gene of interest needs to be meiotically recombined with one of the MADM chromosomes (here shown as coupled to the TG line, resulting in an animal being heterozygous for the gene modification of interest). Additionally, one of the lines needs to carry a Cre driver of interest for the generation of experimental animals. Cells with MADM-induced mosaicism are generated by Cre/<i>loxP</i>-dependent mitotic interchromosomal recombination during the G2 phase in dividing progenitor/stem cells followed by X-segregation (that is the transmission of the two recombinant chromosomes to distinct daughter cells, resulting in individual green (GFP<sup>+</sup>) mutant and red (tdT<sup>+</sup>) wild-type cells) (<a href="#fig1">Figures 1</a>A and 1B). Recombination in the G2 phase of the cell cycle with segregation of both recombinant chromosomes to the same daughter cell (Z-segregation) does not alter the genotype and results in one yellow GFP<sup>+</sup>/tdT<sup>+</sup> heterozygous cell which serves as control. At the same time, Z-Segregation results in transmission of the non-recombined chromosomes to the other sibling daughter cell and thus gives rise to an unlabeled cell (unchanged genotype, heterozygous) (<a href="#fig1">Figure 1</a>C). G1 and/or postmitotic G<sub>0</sub> events also result in yellow cells (<a href="#fig1">Figure 1</a>D).</p>
<p>Importantly, the existence of sparse green mutant, red wild-type and yellow heterozygous cells in an otherwise unlabeled heterozygous environment not only allows to study gene function, but also to perform gene dosage analysis at the individual cell level within the same tissue, in a cell-type-specific manner.</p>
<p>Here, we demonstrate the coupling of the mutant allele of interest with the TG line, which upon successful Cre-mediated recombination will result in genetic mosaic tissue consisting of sparse genetically modified cells labeled in green, while corresponding wild-type cells will be labeled in red and all other cells (yellow and unlabeled) will be heterozygous (<a href="#fig1">Figure 1</a>).</p>
<p>For more details on the MADM technology, please refer to <a href="#bib22">Zong et al. (2005)</a>, <a href="#bib2">Beattie et al. (2020)</a>, <a href="#bib14">Laukoter et al. (2020a)</a>, and <a href="#bib4">Contreras et al. (2021)</a>.</p>
<p>For the successful generation of experimental MADM mice, a number of considerations have to be taken into account before starting with breeding (also see <a href="#sec7.1">troubleshooting problem 1</a>):</p>
<ol>
<li>Institutional and governmental permission and oversight information for the animal study should be obtained. In this study, experimental procedures were discussed and approved by the institutional ethics and animal welfare committees at IST Austria in accordance with good scientific practice guidelines and national legislation (license number: BMWF-66.018/0007-II/3b/2012 and BMWFW-66.018/0006-WF/V/3b/2017).</li>
<li>For MADM to work, the presence of a Cre recombinase, active in stem or progenitor cells of your tissue of interest, is required. In other words, any Cre-driver can be used, provided Cre is expressed in dividing stem and progenitor cells, to target cell types and lineages of interest.</li>
<li>MADM can be employed for both population and clonal analysis. Depending on the aim of your experiments, you are required to choose a Cre driver enabling either population analysis (constitutively active Cre) or clonal analysis (e.g., tamoxifen (TM)-inducible CreER driver).</li>
</ol>
<p>Population analysis allows the analysis of a high number of single, sparsely labeled cells in one organ to obtain a highly quantitative read-out. However, constitutively active Cre will result in MADM events at any time point during the entire time window of Cre activity (e.g., <i>Emx1</i>-Cre activity starts at ∼E9.5 and is detectable up to early postnatal age). Given this relatively long activity window, the detection of labeled cells does not provide information about the spatiotemporal origin of these cells or the stem cell division mode.</p>
<p>In order to get to spatiotemporally controlled induction of MADM events clonal analysis, employing a TM-inducible CreER driver, can be used. Injection of one single low dose of TM allows the induction of G2-X MADM events only at the injection time point, thus generating high spatiotemporal resolution and conserving information about stem cell division modes. For more detailed information about the usage of the MADM system for clonal analysis, please refer to (<a href="#bib2">Beattie et al., 2020</a>).</p>
<ol start="4">
<li>Make use of meiotic recombination in the germline to genetically link a mutant allele of your gene of interest to the corresponding MADM chromosome (<a href="#bib7">Hippenmeyer et al., 2010</a>; <a href="#bib15">Laukoter et al., 2020b</a>) (<a href="#fig1">Figure 1</a>). The probability for meiotic recombination that results in the linkage of the mutant allele with the MADM cassette can be estimated: By determining the genetic distance (centi-Morgan [cM]) between the gene locus of interest and the MADM cassette on a particular chromosome by using for example Mouse Genome Informatics (MGI) database (<a href="http://www.informatics.jax.org">www.informatics.jax.org</a>) (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Calculation of meiotic recombination frequency and breeding schemes for MADM mice</div>
<p>(A) Calculation of meiotic recombination frequency to genetically link the gene of interest (here <i>Ndel1</i>) to the corresponding MADM chromosome (here MADM-11).</p>
<p>(B) Breeding strategy 1 for maximal output efficiency in the generation of experimental MADM mice. (i) In the first breeding step, the mouse line carrying the mutation of interest is bred to the TG cassette of the corresponding MADM chromosome. (ii) In a second step, MADM cassettes are homozygosed by crossing the trans-heterozygote offspring from (i) with animals homozygous for the TG cassette. Thereby, the mutation of <i>gene-X</i> is genetically linked (through meiotic recombination) to the MADM TG cassette on the corresponding MADM chromosome. (iii) In parallel to steps (i) and (ii), the first breeding step for the Cre driver is performed by crossing the Cre line of choice with the GT line. (iv) In a second step, MADM cassettes are homozygosed by performing an intercross of the double heterozygous offspring from (iii). (v) Mating setup resulting in experimental MADM mice with mutant cells labelled in green and wild-type cells labelled in red. (vi) Summary of the required breeding cages to obtain 5 experimental MADM mice. The breeding strategy allows to obtain 50% of genetic mosaic MADM animals from one experimental mating.</p>
<p>(C) Breeding strategy 2 for maximal temporal efficiency in the generation of experimental MADM mice. (i) In the first breeding step, the mutation of <i>gene-X</i> is genetically linked to the MADM TG cassette on the corresponding MADM chromosome. (ii) In parallel, the Cre driver is bred to the GT line. All mice of the F1 generation will be heterozygous for MADM cassettes, <i>gene-X</i> and <i>Cre</i>, respectively. (iii) The breeding step rendering the MADM cassettes homozygous is skipped and F1 mice are used directly to set up the mating resulting in experimental MADM animals with mutant cells labelled in green and wild-type cells labelled in red. (iv) Summary of the required breeding cages to obtain 5 experimental MADM mice. The strategy allows to obtain mosaic animals at a frequency of ∼5% from ∼10 experimental matings.</p>
</figcaption>
</figure>
</li>
<li>Choose MADM chromosomes according to the chromosomal location of your gene of interest. For example, if your gene of interest is located on chromosome 7, you need to recombine it with a MADM cassette localized on chr.7 (MADM-7). If your gene of interest is located on chromosome 11 (as shown in our example in <a href="#fig2">Figure 2</a>), you need to recombine it with a MADM cassette localized on chr.11 (MADM-11). We have reported distinct MADM recombination frequencies for different MADM chromosomes. Consider the frequency when evaluating your results, particularly when frequency is low and results appear highly variable (also see <a href="#sec7.5">troubleshooting problem 3</a>). In the original manuscript corresponding to this protocol, we provide a detailed description of MADM GT and TG cassettes for all 19 mouse autosomes including primer sequences for genotyping (<a href="#bib4">Contreras et al., 2021</a>).</li>
</ol>
<p>We provide an example in <a href="#fig2">Figure 2</a> using the candidate gene <i>Ndel1</i>, which is located on chromosome 11, and thus needs to be recombined to MADM chromosome 11. MADM-11 cassettes are ∼3 cM distal to the centromere and <i>Ndel1</i> is located ∼42 cM distal to the centromere, or ∼39 cM away from the MADM cassettes. Thus, there is ∼39% probability that meiotic recombination will occur in the recombinant generating mating. However, since only half of the progeny will be of the desired genotype (TG,mutant), while the other half will be wild-type (+,+), animals with the appropriate genotype will be generated at a rate of ∼20% (39% × ½) (<a href="#fig2">Figure 2</a>A).</p>
<ol start="6">
<li>The larger the genetic distance between the MADM cassette and the genomic locus of your gene of interest, the higher the recombination probability. The scale of the mating for generating recombinants using meiotic recombination needs to be adjusted according to the recombination probability. In other words, the probability of meiotic recombination determines the amount of matings you require to set up in order to successfully obtain mice where MADM cassettes and genes of interest are genetically linked. The higher the recombination probability, the less breeding cages will be needed to obtain recombinants.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All information provided in this section is specific to the example gene <i>Ndel1</i> in order to provide numerical calculation support to the reader. The numbers and probabilities for your specific experiments require adjustments based on your genes of interest and the required MADM chromosomes.</div>
<ol start="7">
<li>The MADM targeting constructs were introduced into C57BL/6N embryonic stem cells and injected into blastocysts isolated from BALB/cRj females. The offspring were bred with C57BL/6NRj females. Further backcrossing of offspring positive for MADM cassettes was performed with CD1 mice to obtain MADM stock lines of mixed CD1-C57BL/6J genetic background. Consider variances in litter size or phenotype based on mouse background, particularly when you are usually employing pure inbred mice for your experiments.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>PFA</td>
<td>Sigma-Aldrich</td>
<td>Cat# 441244</td>
</tr>
<tr>
<td>10<b>×</b> PBS</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 70011</td>
</tr>
<tr>
<td>Triton X 100</td>
<td>Sigma-Aldrich</td>
<td>Cat# T878</td>
</tr>
<tr>
<td>Tween 20</td>
<td>Sigma-Aldrich</td>
<td>Cat# P9416</td>
</tr>
<tr>
<td>0.2M Tris HCL pH 8.5</td>
<td>CARL ROTH</td>
<td>Cat# 9090</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>·2H<sub>2</sub>O</td>
<td>Sigma-Aldrich</td>
<td>Cat# 567549</td>
</tr>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Sigma-Aldrich</td>
<td>Cat# S3264</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>Cat# G5516</td>
</tr>
<tr>
<td>Citric Acid</td>
<td>CARL ROTH</td>
<td>Cat# X863.1</td>
</tr>
<tr>
<td>NaOH</td>
<td>CARL ROTH</td>
<td>Cat# T198.1</td>
</tr>
<tr>
<td>Heat-inactivated Horse Serum (HS)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#26050088</td>
</tr>
<tr>
<td>Avertin (2,2,2 triboromoethanol)</td>
<td>Sigma-Aldrich</td>
<td>Cat# T48402</td>
</tr>
<tr>
<td>t-amylalcohol (2-methyl-2-butanol)</td>
<td>Sigma-Aldrich</td>
<td>Cat# 240486</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Honeywell</td>
<td>Cat# 24194</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat# S8501</td>
</tr>
<tr>
<td>O.C.T. Tissue Tek</td>
<td>Sakura</td>
<td>Cat# 4583</td>
</tr>
<tr>
<td>Mowiol 4-88</td>
<td>CARL ROTH</td>
<td>Cat# 0713</td>
</tr>
<tr>
<td>DABCO</td>
<td>CARL ROTH</td>
<td>Cat# 0718</td>
</tr>
<tr>
<td>DAPI</td>
<td>Molecular Probes</td>
<td>Cat# D1306</td>
</tr>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Chicken anti-GFP antibody, 1:400</td>
<td>Aves</td>
<td>RRID:AB_10000240</td>
</tr>
<tr>
<td>Goat anti-tdT antibody, 1:400</td>
<td>SICGEN ANTIBODIES</td>
<td>RRID:AB_2333092</td>
</tr>
<tr>
<td>rabbit anti-PhosphoH3 Ser10, 1:800</td>
<td>CellSignaling Technology</td>
<td>RRID:AB_1549592</td>
</tr>
<tr>
<td>Donkey anti-chicken-FITC secondary antibody, 1:500</td>
<td>Invitrogen</td>
<td>RRID:AB_923386</td>
</tr>
<tr>
<td>Donkey anti-goat Alexa568 secondary antibody, 1:1000</td>
<td>Molecular Probes</td>
<td>RRID:AB_2534104</td>
</tr>
<tr>
<td>Donkey anti-rabbit Alexa647 secondary antibody, 1:1000</td>
<td>Molecular Probes</td>
<td>RRID:AB_2762835</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: MADM-7-GT</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:021457</td>
</tr>
<tr>
<td>Mouse: MADM-7-TG</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:021458</td>
</tr>
<tr>
<td>Mouse: MADM-11-GT</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:013749</td>
</tr>
<tr>
<td>Mouse: MADM-11-TG</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:013751</td>
</tr>
<tr>
<td>Mouse: MADM-18-GT</td>
<td>(<a href="#bib4">Contreras et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: MADM-18-TG</td>
<td>(<a href="#bib4">Contreras et al., 2021</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: <i>Emx1</i>-Cre</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:005628</td>
</tr>
<tr>
<td>Mouse: <i>Emx1</i>-CreER</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:027784</td>
</tr>
<tr>
<td>Mouse: <i>Nestin</i>-Cre</td>
<td>(<a href="#bib16">Petersen et al., 2002</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: <i>Nestin</i>-CreER</td>
<td>(<a href="#bib8">Imayoshi et al., 2006</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: <i>Hprt</i>-Cre</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:004302</td>
</tr>
<tr>
<td>Mouse: <i>Apc</i>-flox</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:009045</td>
</tr>
<tr>
<td>Mouse: <i>TrkC</i>
</td>
<td>(<a href="#bib20">Tessarollo et al., 1997</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Mouse: <i>Ndel1</i>
</td>
<td>The Jackson Laboratory</td>
<td>RRID:IMSR_JAX:026958</td>
</tr>
<tr>
<td>Mouse: <i>Lgl1-</i>flox</td>
<td>(<a href="#bib12">Klezovitch et al., 2004</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>ZEN blue</td>
<td>ZEISS</td>
<td><a href="http://www.zeiss.com/microscopy/en_us/products/microscope-software/zen.html#introduction">http://www.zeiss.com/microscopy/en_us/products/microscope-software/zen.html#introduction</a></td>
</tr>
<tr>
<td>Photoshop</td>
<td>Adobe</td>
<td><a href="http://adobe.com/products/photoshop">adobe.com/products/photoshop</a></td>
</tr>
<tr>
<td>GraphPad Prism</td>
<td>GraphPad</td>
<td><a href="https://www.graphpad.com/scientific-software/prism/">https://www.graphpad.com/scientific-software/prism/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Histosette (embedding cassette for tissue fixation)</td>
<td>Sakura</td>
<td>Cat# 4170-01</td>
</tr>
<tr>
<td>Embedding molds for peripheral organs</td>
<td>Sakura</td>
<td>Cat# 4557</td>
</tr>
<tr>
<td>Embedding molds for coronal brain sections</td>
<td>Polysciences, Inc..</td>
<td>Cat# 18986-1</td>
</tr>
<tr>
<td>Embedding molds for sagittal brain sections</td>
<td>Polysciences, Inc.</td>
<td>Cat# 18646A-1</td>
</tr>
<tr>
<td>6 well plates</td>
<td>TPP</td>
<td>Cat# 92406</td>
</tr>
<tr>
<td>24 well plates</td>
<td>TPP</td>
<td>Cat# 92424</td>
</tr>
<tr>
<td>Flask Filters 500mL</td>
<td>TPP</td>
<td>Cat# 99505</td>
</tr>
<tr>
<td>Microfuge Tubes 1.5 mL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# AM12450</td>
</tr>
<tr>
<td>50 mL Centrifuge Tubes</td>
<td>Sarstedt, Inc.</td>
<td>Cat# 62.547.254</td>
</tr>
<tr>
<td>15 mL Centrifuge Tubes</td>
<td>Sarstedt, Inc.</td>
<td>Cat# 62.554.502</td>
</tr>
<tr>
<td>Syringe 60mL</td>
<td>Kendall</td>
<td>Cat# 560125</td>
</tr>
<tr>
<td>Syringe 10 mL Omnifix</td>
<td>B.Braun</td>
<td>Cat# 4617100V</td>
</tr>
<tr>
<td>Needle 20G Sterican</td>
<td>B.Braun</td>
<td>Cat# 4657705</td>
</tr>
<tr>
<td>0.2micron filter</td>
<td>Nalgene</td>
<td>Cat# 194-2520</td>
</tr>
<tr>
<td>Peristaltic Pump</td>
<td>Watson-Marlow</td>
<td>Cat# 323 S/D</td>
</tr>
<tr>
<td>Hydrophobic Pen</td>
<td>DAKO</td>
<td>Cat# S2002</td>
</tr>
<tr>
<td>Petri dish</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# NC9565080</td>
</tr>
<tr>
<td>Slide Moisture Chamber black</td>
<td>Newcomer Supply</td>
<td>Cat# 68432A</td>
</tr>
<tr>
<td>Superfrost Glass Slides</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# J1800AMNZ</td>
</tr>
<tr>
<td>Cover Slips 24 × 50 mm</td>
<td>VWR</td>
<td>631–0147</td>
</tr>
<tr>
<td>Dissection Tools</td>
<td>Freudenberg FST</td>
<td>Various, depending on the specific needs of the experiment</td>
</tr>
<tr>
<td>Fine Brush Size 1</td>
<td>Ted Pella, Inc.</td>
<td>Cat# 11859</td>
</tr>
<tr>
<td>LSM 800 Confocal</td>
<td>ZEISS</td>
<td>N/A</td>
</tr>
<tr>
<td>Cryostat Cryostar NX70</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Essential buffers and anesthesia for tissue harvesting (prepare 1 day prior to the experiment)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 60 min</div>
<table id="undtbl1">
<caption>1M PB buffer (pH 7.4)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2M NaH<sub>2</sub>PO<sub>4</sub> <sup>.</sup> 2H<sub>2</sub>O (monobasic sodium phosphate)</td>
<td>1M</td>
<td>47.5 mL</td>
</tr>
<tr>
<td>2M Na<sub>2</sub>HPO<sub>4</sub> (dibasic sodium phosphate)</td>
<td>1M</td>
<td>202.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>250 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1M</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Keep at 22°C–25°C.</p>
<table id="undtbl2">
<caption>2N NaOH</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaOH</td>
<td>2N</td>
<td>40 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>2N</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Keep at 22°C–25°C.</p>
<table id="undtbl3">
<caption>4% PFA (in 0.1M PB buffer)—always prepare fresh PFA</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Paraformaldehyde (PFA)</td>
<td>4%</td>
<td>40 g</td>
</tr>
<tr>
<td>2N NaOH</td>
<td>0.01N</td>
<td>4 mL</td>
</tr>
<tr>
<td>1M PB buffer</td>
<td>0.1M</td>
<td>100 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>850 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>4%</b></td>
<td>
<b>1</b><b>,</b><b>000 mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Details for PFA preparation: (1) Dissolve 40 g of PFA in 750 mL distilled water. (2) Add 4 mL 2N NaOH. (3) Place on a stirring heating plate at 55°C until all PFA has dissolved. (4) Add 100 mL 1M PB buffer. (5) Fill up the volume to 1000 mL with distilled water. (6) Filter through a flask filter to avoid residual undissolved PFA particles. (7) Store at 4°C for up to 1–2 days.</p>
<table id="undtbl4">
<caption>Anesthesia stock solution (100%)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Avertin</td>
<td>100%</td>
<td>7 g</td>
</tr>
<tr>
<td>t-amylalcohol</td>
<td>n/a</td>
<td>7 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>100%</b></td>
<td><b>7 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Details for anesthesia preparation: (1) Vortex until the solution appears homogeneous. (2) Wrap in tin foil to protect from light and store at 22°C–25°C for up to 6 months.</p>
<table id="undtbl5">
<caption>Anesthesia working solution (2.5%)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>100% anesthesia stock solution</td>
<td>2N</td>
<td>0.875 mL</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>34.125 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>2.5%</b></td>
<td><b>35 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Details for anesthesia preparation: (1) Vortex until the solution appears homogeneous. (2) Filter the solution into a fresh tube using a 60 mL syringe and a 0.2micron filter. (3) Store at 4°C for up to one month.</p>
<table id="undtbl6">
<caption>1<b>×</b> PBS:</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>100 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>900 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1×</b></td>
<td>
<b>1</b><b>,</b><b>000 mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Keep at 22°C–25°C.</p>
<table id="undtbl7">
<caption>Sucrose</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>30%</td>
<td>300 g</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>Add up to 1,000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>30%</b></td>
<td>
<b>1</b><b>,</b><b>000 mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">We recommend to filter the sucrose solution through a 500 mL flask filter to avoid contamination with microorganisms. Store at 4°C.</p>
<h3 id="sec2.2">Prepare perfusion equipment (on the day of experiment)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min</div>
<ul>
<li>Clean all dissection tools (2 curved forceps, large and small scissors) with 70% ethanol.</li>
<li>Prepare a perfusion pad and whipping tissues.</li>
<li>Prepare a peristaltic pump or two 10 mL syringes and needles for perfusion.</li>
<li>Prepare 6-well plates filled with 4% PFA to collect brain tissues.</li>
<li>Prepare histosettes and a beaker filled with 4% PFA to collect peripheral organs.</li>
</ul>
<h3 id="sec2.3">Prepare essential buffers and reagents for histological stainings (on the day of the experiment)</h3>
<table id="undtbl8">
<caption>Cryoprotectant</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycerol</td>
<td>60%</td>
<td>300 mL</td>
</tr>
<tr>
<td>1M PB buffer</td>
<td>0.1M</td>
<td>50 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>150 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>60%</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C.</p>
<table id="undtbl9">
<caption>PBS with 0.5% Tween (PBS-T)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tween20</td>
<td>0.5%</td>
<td>5 mL</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>995 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>0.5%</b></td>
<td><b>1000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Add 5 mL Tween20 to 1,000 mL 1<b>×</b> PBS and mix well. Store at 22°C–25°C.</p>
<table id="undtbl10">
<caption>Citrate buffer pH 6.0</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Citric acid anhydrous (192M)</td>
<td>1M</td>
<td>1.92 g</td>
</tr>
<tr>
<td>Tween20</td>
<td>0.05%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>1M</b></td>
<td>
<b>1</b><b>,</b><b>000 mL</b>
</td>
</tr>
</tbody>
</table>
<p class="table-legend">Details for citrate buffer preparation: (1) Dissolve 1.92 g citric acid anhydrous (192M) in 1000 mL distilled water. (2) Adjust to pH 6.0 using NaOH. (3) Add 0.5 mL of Tween20. (4) Store at 4°C.</p>
<table id="undtbl11">
<caption>Triton-X-100 working solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton-X-100</td>
<td>20%</td>
<td>2 mL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>8 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20%</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store it in a 15 mL Falcon tube at 22°C–25°C.</p>
<table id="undtbl12">
<caption>Blocking buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Horse Serum</td>
<td>10%</td>
<td>1 mL</td>
</tr>
<tr>
<td>Triton-X-100 working solution</td>
<td>0.5%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>8.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store it in a 15 mL Falcon tube at 4°C.</p>
<table id="undtbl13">
<caption>Staining buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Horse Serum</td>
<td>5%</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Triton-X-100 working solution</td>
<td>0.5%</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>9.25 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>10 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store it in a 15 mL Falcon tube at 4°C.</p>
<div class="note">
<span class="note-title">Note:</span> You can prepare 1,000 ml of Blocking and Staining Buffer and aliquot in 50 mL or 15 mL tubes, which can be stored at −20°C until further usage. Once thawed, keep at 4°C and use up within 1 week.</div>
<table id="undtbl14">
<caption>DAPI stock solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 vial DAPI</td>
<td>5 mg/mL</td>
<td>10 mg</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>2 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>5 mg/mL</b></td>
<td><b>2 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Aliquot in 1.5 mL tubes and store at −20°C.</p>
<table id="undtbl15">
<caption>DAPI working solution</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DAPI stock solution</td>
<td>300nM</td>
<td>1 μL</td>
</tr>
<tr>
<td>1<b>×</b> PBS</td>
<td>1<b>×</b>
</td>
<td>20 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>300nM</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 2°C–6°C up to one week, protected from light.</p>
<table id="undtbl16">
<caption>Embedding medium Mowiol-DABCO</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycerol</td>
<td>n/a</td>
<td>6 g</td>
</tr>
<tr>
<td>Mowiol</td>
<td>n/a</td>
<td>4 g</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>n/a</td>
<td>6 mL</td>
</tr>
<tr>
<td>0.2M Tris-HCl (pH 8.5)</td>
<td>0.1</td>
<td>12 mL</td>
</tr>
<tr>
<td>DABCO</td>
<td>n/a</td>
<td>25 mg</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>25 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Details for Mowiol-DABCO preparation: (1) Use a balance to weight 6.0 g Glycerol and 2.4 g Mowiol. Given the high viscosity of Glycerol, it is more accurate to measure its weight than its volume. (2) Mix both reagents and dissolve for 1 h on a stirring plate. (3) Add 6 mL distilled water and mix for another hour. (4) Add 12 mL 0.2M Tris-HCl (pH 8.5) and incubate for 2 h at 50°C on a stirring heating plate. (5) It is likely that not all Mowiol is going to dissolve. Thus, you can add an additional centrifugation step to remove undissolved Mowiol from the preparation by centrifuging for 15 min at 5,000 g and keeping the supernatant. (6) Add 25 mg DABCO and dissolve so that you have a clear solution. (7) Aliquot in 1.5 mL tubes and store at −20°C until further usage.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Breeding of experimental MADM mice for the generation of genetic mosaic tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–7 months</div>
<p>In order to generate genetic mosaic mice, you can either chose a breeding strategy maximizing the efficiency of experimental MADM animal output (here called “strategy 1”) or a breeding strategy maximizing the speed of obtaining experimental MADM animals (here called “strategy 2”) (<a href="#fig2">Figures 2</a>B and 2C). We provide an estimated amount of breeding mice required to obtain a realistic number of experimental MADM animals (<a href="#fig2">Figures 2</a>B and 2C). We provide step-by-step breeding guidance in the following section of this protocol (also see <a href="#sec7.1">troubleshooting problem 1</a>):</p>
<ol>
<li>For both breeding strategies, perform Breeding step 1 for <i>gene-X</i> of interest: In order to link the mutation of <i>gene-X</i> to the MADM TG cassette, cross <i>MADM</i><sup><i>TG/TG</i></sup> animals with a mouse line mutant for <i>gene-X</i> (<a href="#fig2">Figures 2</a>B(i) and 2C(i)). The resulting F1 generation will be trans-heterozygous for the MADM TG cassette and the mutation of interest (genotype will be <i>MADM</i><sup><i>TG/+</i></sup><i>; gene-X</i><sup><i>+/-</i></sup>).</li>
<li>For both breeding strategies, perform Breeding step 1 for the Cre driver (<a href="#fig2">Figures 2</a>B(iii) and 2C(ii)): cross <i>MADM</i><sup><i>GT/GT</i></sup> animals with a Cre driver of choice. The resulting F1 generation will be double heterozygous for the MADM GT cassette and the Cre driver specific for the cell type of interest (<i>MADM</i><sup><i>GT/+</i></sup><i>; Cre</i><sup><i>+/-</i></sup>). Perform this step in parallel to the above step.</li>
</ol>
<p>Allow F1 generation mice to grow up to 8 weeks of age. For breeding strategy 2, proceed to step 6. For breeding strategy 1, continue with steps 3, 4 and 5.</p>
<ol start="3">
<li>For strategy 1, perform Breeding step 2 for the mutant by crossing <i>MADM</i><sup><i>TG/+</i></sup><i>;gene-X</i><sup><i>+/-</i></sup> mice with <i>MADM</i><sup><i>TG/TG</i></sup> animals in order to generate F2 offspring homozygous for the MADM TG cassette and meiotically linked <i>gene-X</i> (<i>MADM</i><sup><i>TG/TG,gene-X</i></sup>) (<a href="#fig2">Figure 2</a>B(ii)).</li>
<li>For strategy 1, perform Breeding step 2 for the Cre driver by intercrossing <i>MADM</i><sup><i>GT/+</i></sup><i>; Cre</i><sup><i>+/-</i></sup> mice with <i>MADM</i><sup><i>GT/+</i></sup><i>; Cre</i><sup><i>+/-</i></sup> animals in order to generate F2 offspring homozygous for the MADM GT cassette and the Cre driver. The resulting genotype will be <i>MADM</i><sup><i>GT/GT</i></sup><i>; Cre</i><sup><i>+/+</i></sup> (<a href="#fig2">Figure 2</a>B(iv)). Perform this step parallel to step 3.</li>
</ol>
<p>Allow F2 generation mice to grow up to 8 weeks of age.</p>
<ol start="5">
<li>For the generation of experimental MADM animals according to strategy 1, cross <i>MADM</i><sup><i>TG/TG,gene-X</i></sup> mice with <i>MADM</i><sup><i>GT/GT</i></sup><i>; Cre</i><sup>+/+</sup> animals in order to generate experimental offspring with genetic mosaic tissue with green mutant, red wild-type and yellow heterozygous cells (<a href="#fig2">Figure 2</a>B(v)).<br> The genotype of the experimental MADM animals is <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Cre</i><sup><i>+/-</i></sup> (<a href="#fig2">Figure 2</a>B(v)).<br> The probability to obtain experimental MADM animals from this breeding is 50%. The probability for control-MADM animals is also 50%, i.e., animals with MADM-labelling but without mutation. With an estimated litter size of 10 pups per litter, this means that you require 1 breeding cage to obtain a total of 5 experimental MADM mice (<a href="#fig2">Figure 2</a>B(vi)).
<ol type="a">
<li>Check for the vaginal plug to monitor successful mating and to expedite the experimental planning.</li>
<li>Male and female offspring can be used as experimental animals.</li>
<li>Perform experiment at developmental stage(s) of choice.</li>
</ol>
</li>
</ol>
<ol start="6">
<li>For the generation of experimental MADM animals according to strategy 2, cross trans-heterozygous <i>MADM</i><sup><i>TG/+</i></sup><i>;gene-X</i><sup><i>+/-</i></sup>mice directly with double heterozygous <i>MADM</i><sup><i>GT/+</i></sup><i>; Cre</i><sup><i>+/-</i></sup> animals in order to generate experimental MADM offspring with genetic mosaic tissue with green mutant, red wild-type and yellow heterozygous cells (<a href="#fig2">Figure 2</a>C(iii)).<br> The genotype of the experimental MADM animals is <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Cre</i><sup><i>+/-</i></sup> (<a href="#fig2">Figure 2</a>C(iii)). The probability to obtain experimental MADM animals from this breeding is ∼5%. With an estimated litter size of 10 pups per litter, you require ∼10 breeding cages and ∼100 pups to obtain a total of 5 experimental MADM mice (<a href="#fig2">Figure 2</a>C(iv)).
<ol type="a">
<li>Check for the vaginal plug to monitor successful mating and to expedite the experimental planning.</li>
<li>Male and female offspring can be used as experimental animals.</li>
<li>Perform experiment at developmental stage(s) of choice.
<div class="note">
<span class="note-title">Note:</span> In the following examples described in this protocol, all tissue was analyzed at specific postnatal stages. However, experimental animals can also be used at embryonic stages, P0, and any postnatal age depending on your experimental needs.</div>
In order to provide a broad overview of applications suitable for histological analysis using MADM in this protocol, we made use of a number of different Cre drivers: (1) constitutively active <i>Emx1</i>-Cre and TM-inducible <i>Emx1-</i>CreER that primarily targets radial glia progenitors in the dorsal forebrain, which give rise to excitatory projection neurons and glial cells in the cerebral cortex (<a href="#bib6">Gorski et al., 2002</a>; <a href="#bib10">Kessaris et al., 2006</a>). (2) We include an example using constitutively active <i>Nestin</i>-Cre and inducible <i>Nestin-</i>CreER (<a href="#bib16">Petersen et al., 2002</a>; <a href="#bib8">Imayoshi et al., 2006</a>), which is a marker of stem cells across the nervous system and therefore results in MADM labeling in all neural cell types including Purkinje cells; (3) and make use of constitutively active <i>Hprt</i>-Cre (<a href="#bib19">Tang et al., 2002</a>) which shows ubiquitous spatiotemporal expression. The <i>Hprt</i>-Cre driver was utilized to investigate peripheral organs, in particular the intestine.<br> MADM can also be applied for experiments involving flow cytometry. For details and a step-by-step protocol on the experimental workflow specific for flow cytometry experiments, we refer to (<a href="#bib14">Laukoter et al., 2020a</a>).<br> Specific considerations when breeding experimental MADM animals are provided in the next two sections of the protocol:</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Generation of experimental MADM mice for postnatal histological analysis in brain</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks–12 months</div>
<ol start="7">
<li>For population analysis, cross <i>MADM</i><sup><i>GT/GT</i></sup><i>; Emx1</i><sup><i>Cre/Cre</i></sup> or <i>MADM</i><sup><i>GT/GT</i></sup><i>; Nestin-Cre</i><sup><i>+</i>/<i>+</i></sup> females with <i>MADM</i><sup><i>TG/TG;gene-X</i></sup> males in order to generate offspring with genetic mosaic brain tissue (<a href="#fig2">Figure 2</a>B, bottom panel) (also see <a href="#sec7.7">troubleshooting problem 4</a>).<br> For clonal analysis, cross <i>MADM</i><sup><i>GT/GT</i></sup><i>; Emx1</i><sup><i>CreER/CreER</i></sup> or <i>MADM</i><sup><i>GT/GT</i></sup><i>; Nestin-CreER</i><sup><i>+/+</i></sup> females with <i>MADM</i><sup><i>TG/TG;gene-X</i></sup> males in order to generate offspring with genetic mosaic brain tissue following TM injection at a time point of choice (<a href="#fig2">Figure 2</a>B, bottom panel) (also see <a href="#sec7.9">troubleshooting problem 5</a>). For simplicity, the above mentioned distinct Cre/ER drivers will be abbreviated as “Cre” throughout step 7.
<ol type="a">
<li>Check for the vaginal plug to monitor successful mating and to facilitate the experimental planning.</li>
<li>Males and females of the resulting F1 generation can be used as experimental animals (genotype will be <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Cre</i><sup><i>+/-</i></sup><i>).</i>
<div class="note">
<span class="note-title">Note:</span> Upon crossing <i>MADM</i><sup><i>GT/GT</i></sup><i>; Cre</i><sup>+/+</sup> females with <i>MADM</i><sup><i>TG/TG,gene-X</i></sup> males to generate <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Cre</i><sup><i>+/-</i></sup> mosaic animals, green cells harboring a homozygous mutation in <i>gene-X</i> will show uniparental paternal chromosome disomy (patUPD, PP), while red wild-type cells will show uniparental maternal chromosome disomy (matUPD, MM) (<a href="#fig3">Figure 3</a>A). We refer to this breeding scheme as ‘default breeding scheme.’</div>
<div class="note">
<span class="note-title">Note:</span> In order to reverse the fluorescent colors to control for a potential bias of the fluorescent reporter (color swap), couple the mutation to the MADM GT cassette and use <i>MADM</i><sup><i>TG/TG</i></sup> females and <i>MADM</i><sup><i>GT/GT,gene-X</i></sup><i>; Cre</i><sup><i>+/+</i></sup> males (<a href="#bib14">Laukoter et al., 2020a</a>). Here, red cells harboring a homozygous mutation in <i>gene-X</i> will show patUPD (PP), while green wild-type cells will show matUPD (MM) (<a href="#fig3">Figure 3</a>B). For efficient conduction of this control experiment, we recommend to perform the color swap in the context of population analysis using a constitutively active Cre driver and not a TM-inducible CreER.</div>
<div class="note">
<span class="note-title">Note:</span> In order to reverse the parents to control for a potential genomic imprinting bias of the parental origin of the mutation (parent swap), use <i>MADM</i><sup><i>TG/TG,gene-X</i></sup> females and <i>MADM</i><sup><i>GT/GT</i></sup><i>; Cre</i><sup><i>+/+</i></sup> males (<a href="#bib14">Laukoter et al., 2020a</a>) to generate <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Cre</i><sup><i>+/-</i></sup> mosaic animals. Here, green cells harboring a homozygous mutation in <i>gene-X</i> will show matUPD (MM), while red wild-type cells will show patUPD (PP) (<a href="#fig3">Figure 3</a>C). Given that the generation of stock lines for your experiments will yield <i>MADM</i><sup><i>TG/TG,gene-X</i></sup> females and <i>MADM</i><sup><i>GT/GT</i></sup><i>; Cre</i><sup><i>+/+</i></sup> males in any case, use the opportunity of having these mice available and perform the parent swap in parallel to your ‘default breeding’ setup. For efficient conduction of this control experiment, we recommend to perform it in the context of population analysis using a constitutively active Cre driver and not a TM-inducible CreER.</div>
</li>
<li>Perform experiment at your desired postnatal time point, for example when generation of cell types is completed and the cells have reached a fully developed, mature state (in our examples shown in this protocol we chose time points between P1 and P21).</li>
</ol>
</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Breeding schemes for default setup, color swap, and parent swap</div>
<p>(A) Schematic showing the default breeding scheme using males where <i>gene-X</i> is meiotically linked to the MADM TG cassette, while females transmit the GT cassette and the Cre driver. Resulting offspring will contain red wild-type cells with uniparental maternal chromosome disomy (matUPD, MM), while green mutant cells will harbor uniparental paternal chromosome disomy (patUDP, PP).</p>
<p>(B) Schematic of the breeding scheme for a color swap experiment using females homozygous for the MADM TG cassette, while males transmit the Cre driver plus <i>gene-X</i> meiotically linked to the GT cassette. Resulting offspring will have green wild-type cells with matUPD (MM), while red mutant cells will harbor patUPD (PP).</p>
<p>(C) Schematic of the breeding scheme for a parent swap experiment using females where <i>gene-X</i> is meiotically linked to the MADM TG cassette, while males transmit the GT cassette and the Cre driver. Resulting offspring will contain red wild-type cells with patUPD (PP), while green mutant cells will harbor matUPD (MM).</p>
<p>Parts of the figure were reproduced and adapted with permission from (<a href="#bib1">Beattie et al., 2017</a>).</p>
</figcaption>
</figure>
<h3 id="sec3.3">Generation of experimental MADM mice for postnatal histological analysis of genetic mosaic tissue in peripheral organs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 weeks–12 months</div>
<ol start="8">
<li>Cross <i>MADM</i><sup><i>GT/GT</i></sup><i>; Hprt</i><sup><i>Cre</i>/+</sup> females with <i>MADM</i><sup><i>TG/TG,gene-X</i></sup> males in order to generate offspring with genetic mosaic tissues.
<ol type="a">
<li>Check for the vaginal plug to monitor successful mating and to facilitate the experimental planning.</li>
<li>Males and females of the resulting F1 generation can be used as experimental animals (genotype will be <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Hprt</i><sup><i>Cre</i>/+</sup>).
<div class="critical">
<span class="critical-title">Critical:</span> The <i>Hprt</i>-Cre transgene is located on the X chromosome, with one of the two X chromosomes undergoing random inactivation in females. Thus, Cre activity and MADM labelling are highly variable in <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Hprt</i><sup><i>Cre/</i>+</sup> females. We therefore recommend to primarily use males for your analysis in order to obtain reproducible results.</div>
<div class="note">
<span class="note-title">Note:</span> Upon crossing <i>MADM</i><sup><i>GT/GT</i></sup><i>; Hprt</i><sup><i>Cre</i>/+</sup> females with <i>MADM</i><sup><i>TG/TG,gene-X</i></sup> males to generate <i>MADM</i><sup><i>GT/TG,gene-X</i></sup><i>; Hprt</i><sup><i>Cre</i>/+</sup> mosaic animals, green cells harboring a homozygous mutation in <i>gene-X</i> will show patUPD, while red wild-type cells will show matUPD.</div>
<div class="note">
<span class="note-title">Note:</span> If desired, perform color swap and parent swap (<a href="#bib14">Laukoter et al., 2020a</a>) as described in the notes for step 7 of the protocol.</div>
</li>
<li>Perform the experiment at the desired postnatal time point, to e.g., analyze cellular phenotypes arising during development in mature tissue and/or trace disease phenotypes when a disease phenotype is evident (up to several months of age).</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Mouse perfusion and dissection for collection of MADM-labelled tissue for histological analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12–16 h</div>
<p>This section of the protocol documents the perfusion and dissection of postnatal mice for organ removal and further downstream histological analysis.</p>
<div class="critical">
<span class="critical-title">Critical:</span> 4% PFA should be ice-cold and protected from light.</div>
<ol start="9">
<li>Arrange all solutions and tools required for the tissue collection at the dissection area
<ol type="a">
<li>Anesthesia</li>
<li>1<b>×</b> PBS</li>
<li>4% PFA</li>
<li>Peristaltic pump or perfusion syringes</li>
<li>Dissection tools</li>
<li>6-well plates</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure no air bubbles are trapped in the tubes of the peristaltic pump or within the syringe containing the perfusion buffers. Bubbles may destroy the tissue during the perfusion process. Run the tubes of the pump with PBS prior to starting the perfusion procedure to make sure all air is removed.</div>
<ol start="10">
<li>Anesthetize the animal by intraperitoneal injection of anesthesia working solution at a volume of 0.16–0.24 mL/10 g mouse (equivalent to 400–600 mg/kg).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Mice require deep anesthesia before any further procedure can be applied. Test the efficiency of anesthesia by pinching a paw with forceps. Only when the mouse does not display any movement after pinching, you can proceed with the dissection and perfusion.</div>
<ol start="11">
<li>Place the anesthetized animal in supine position on a perfusion tray and disinfect fur with 70% ethanol.</li>
<li>Below the rib cage, make a medial incision with scissors and surgical forceps through the skin and the underlying muscle layer (<a href="#fig4">Figure 4</a>A). Continue to cut laterally.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Tissue harvesting for histological analysis of experimental MADM mice</div>
<p>(A–C) Mouse dissection and initiation of perfusion.</p>
<p>(D–Q) Mouse dissection to isolate and embed brain.</p>
<p>(R–V) Isolation and embedding of colon.</p>
</figcaption>
</figure>
</li>
<li>Lift the tip of the sternum and snip the diaphragm. Then trim the rib cage to expose the heart.</li>
<li>Using small iris scissors make an incision to the posterior end of the right atrium for the blood to drain during the following perfusion procedure (<a href="#fig4">Figure 4</a>B).</li>
<li>Insert a needle attached to a peristaltic pump or a syringe filled with 1<b>×</b> PBS into the lower left ventricle (<a href="#fig4">Figure 4</a>C).</li>
<li>Perfuse with PBS until the liver turns pale (requires approximately 5–10 mL of PBS).</li>
<li>Once complete, switch the perfusion solution to 4% PFA.
<div class="note">
<span class="note-title">Note:</span> Perfusion requirements need to be adjusted according to the age of the animals. (a) For the perfusion of pups: perfuse manually using 10 mL syringes. Prepare 2 distinct 10 mL syringes: one filled with 1<b>×</b> PBS and another filled with 4% PFA. Once perfusion with PBS is completed, replace PBS-containing syringe plus needle with fresh needle and syringe containing PFA through the same puncture hole in the heart. (b) For the perfusion of juvenile and adult mice, use a peristaltic pump. Place one end of the tube of the peristaltic pump into a flask or Falcon tube filled with PBS. After perfusion with PBS (speed: 5 mL at 2–4 mL/min in juvenile animals; 10 mL at 4–6 mL/min for adult mice), switch to 4% PFA.</div>
<div class="critical">
<span class="critical-title">Critical:</span> briefly stop the peristaltic pump and exchange the PBS solution with ice-cold 4% PFA. This is critical to avoid bubbles in the pump tubing. Resume perfusion with PFA at the same speed.</div>
</li>
</ol>
<ol start="18">
<li>Remove mouse from perfusion pad and dissect the organ of interest. If organ of interest is brain, continue to follow the protocol. If organ of interest is a peripheral organ, proceed to step 27.</li>
<li>Dissection of brain:
<ol type="a">
<li>Decapitate mouse.</li>
<li>Remove the skin over the skull (<a href="#fig4">Figure 4</a>D).</li>
<li>Cut the skull at the naso-frontal suture (in front of the olfactory bulbs) (<a href="#fig4">Figure 4</a>E).</li>
<li>Then cut along the midline (<a href="#fig4">Figure 4</a>F), followed by a cut along the parietal-interparietal sutures.</li>
<li>Lift the skull to the side to expose the brain (<a href="#fig4">Figure 4</a>G).</li>
<li>Place curved forceps or a thin spatula underneath the brain and remove it from the skull (<a href="#fig4">Figure 4</a>H).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid mechanical damage of brain tissue during the dissection to ensure you obtain intact tissue for high quality analysis.</div>
<ol start="20">
<li>Transfer the brain to a well filled with ice-cold 4%PFA in a 6-well plate (<a href="#fig4">Figure 4</a>I).</li>
<li>Incubate at 4°C for minimum 10 h.</li>
<li>Next day, wash 3<b>×</b> with 1<b>×</b> PBS for 5 min each.</li>
<li>Transfer brain to a 15 mL Falcon tube filled with 10 mL of 30% Sucrose.</li>
<li>Incubate at 4°C until the brain has sunk to the bottom of the Falcon tube.</li>
<li>Remove brain from sucrose and gently remove residual sugar from the surface of the brain using lint-free tissues.</li>
<li>Depending on the cell type of interest and the needs of the experiment, embed brain in coronal or sagittal orientation:
<ol type="a">
<li>For coronal orientation (<a href="#fig4">Figure 4</a>J), place brain ventral-side down on a piece of lint-free tissue and remove the cerebellum using a razor blade (<a href="#fig4">Figures 4</a>K and 4L). Place brain with the caudal side down in an embedding mold (<a href="#fig4">Figure 4</a>M), fill mold and cover tissue with O.C.T. Tissue tek and freeze. Once frozen, store it at −80°C.</li>
<li>For sagittal orientation (<a href="#fig4">Figure 4</a>N), place brain ventral-side down on a piece of lint-free tissue and cut it in two halves along the midline using a razor blade (<a href="#fig4">Figures 4</a>O and 4P). Place brain with the medial side down in an embedding mold (<a href="#fig4">Figure 4</a>Q), fill mold and cover tissue with O.C.T. Tissue tek and freeze. Once frozen, store it at −80°C.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> You can keep the frozen tissue at −80°C until further usage.</div>
</li>
</ol>
</li>
</ol>
<ol start="27">
<li>Dissection of peripheral organ (this protocol is using the colon as an example):
<ol type="a">
<li>Remove colon from the mouse and place it on lint-free tissue.</li>
<li>Cut open longitudinally (<a href="#fig4">Figure 4</a>R).</li>
<li>Remove faeces.</li>
<li>Make colon into a Swiss roll by using a pipet tip to roll it up (<a href="#fig4">Figure 4</a>S).</li>
<li>Transfer colon into a histosette (<a href="#fig4">Figure 4</a>T) for further handling.</li>
</ol>
</li>
<li>Incubate in a beaker filled with 4% PFA for 4 h at 22°C–25°C (<a href="#fig4">Figure 4</a>U).</li>
<li>Wash 3<b>×</b> with 1<b>×</b> PBS for 5 min each.</li>
<li>Incubate in 30% Sucrose at 4°C for minimum 10 h.</li>
<li>Remove histosette with colon from sucrose, take out colon and gently remove residual sugar from the surface of the organ by using lint-free tissues.</li>
<li>Transfer colon to an embedding mold, fill mold and cover tissue with O.C.T. Tissue Tek and freeze (<a href="#fig4">Figure 4</a>V). Once frozen, store it at −80°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> You can keep the frozen tissue at −80°C until further usage.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Optimal tissue freezing during embedding is achieved when embedding molds are placed on dry ice. Control the tissue during the freezing process from time to time and avoid loss of position within the mold.</div>
<h3 id="sec3.5">Tissue sectioning for histological analysis</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min<b>–</b><b>3 h</b>
</div>
<p>This section of the protocol lists details on sectioning MADM-labelled tissue for confocal microscopy.</p>
<ol start="33">
<li>Remove tissue from −80°C and transfer to a cryostat cooled to −20°C.</li>
<li>Let tissue adapt to temperature change for 20 min.</li>
<li>Remove tissue block from the embedding mold.</li>
<li>Attach the tissue block to the specimen disk in the cryostat by applying a drop of OCT to the disk and placing the tissue block directly into the OCT. Allow to freeze for 1–2 min.</li>
<li>Insert disk with attached tissue block in the sectioning holder by placing the tissue in a correct orientation.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For brain sections, insert sectioning disc with cortex facing towards the top. For peripheral organ sections, insert sectioning disc with organ of choice in an orientation that guarantees good sectioning quality for your required analysis.</div>
<ol start="38">
<li>Start cutting by trimming the tissue block in 45 μm thick sections until the region of interest is reached.</li>
<li>Slice the tissue in 30 μm thick sections. Either collect the slices in individual wells of a 24 well plate (= floating sections) or mount them directly onto frosted glass slides.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Floating sections can be kept in PBS at 4°C for up to 48 h. When longer storage is required, exchange the PBS with cryoprotectant and store at −20°C.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Directly mounted sections can be stored in a slide box at −20°C until further usage.</div>
<h3 id="sec3.6">Tissue mounting</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 15 min–1 h</div>
<p>This part of the protocol can be skipped if the sliced tissue is directly mounted onto frosted glass slides in the course of the sectioning process. In that case, immediately proceed to step 45 of the protocol. If you prepared floating sections, continue with step 40.</p>
<ol start="40">
<li>Fill a Petri dish with PBS-T.</li>
<li>Place a frosted glass slide at the edge of the petri dish so that is covered up nearly to the label with PBS-T.</li>
<li>Transfer all of your desired sections into the dish filled with PBS-T.</li>
<li>Use a small paint brush to maneuver the sections onto the slide and arrange them side by side.</li>
<li>Once all sections are placed onto the slide (for brain usually ∼12−16 sections/slide), horizontally transfer the slide to a dark slide chamber and let the sections dry completely (∼15 min) to make them adhere to the glass. Either proceed to the next steps of the staining procedure or pause the protocol at this point.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store the mounted slides at 4°C for 10–16 h and continue with the protocol on the next day.</div>
<h3 id="sec3.7">Staining</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–16 h</div>
<p>This section of the protocol explains the staining procedure for MADM-labelled cells together with an additional, cell-type specific marker staining. All steps are performed at 22°C–25°C unless otherwise stated. Please refer to <a href="#sec7.3">troubleshooting problem 2</a> when observing variable signals.</p>
<ol start="45">
<li>If your sections were floating sections and have been freshly mounted and dried, immediately proceed to step 46 of the protocol. If your sections were already mounted during sectioning and stored at −20°C, take the slides from the freezer and let them dry for 15 min.</li>
<li>Encircle staining area with a hydrophobic pen.</li>
<li>Rehydrate and wash sections 3<b>×</b> 5 min with 1<b>×</b> PBS to remove residual PBS-T.</li>
<li>Meanwhile, heat up an oven to 85°C. Take the opportunity to put the antigen retrieval buffer into the oven to have the solution pre-heated. Our most commonly used antigen retrieval solution is Citrate Buffer pH 6.0.</li>
<li>Incubate your sections with 1 mL pre-heated citrate buffer pH 6.0 for 25 min at 85°C in a dark and closed staining chamber (make sure it is closed. If it is open, buffer will evaporate and samples will dry out).</li>
<li>Remove the staining chamber and let chill on your bench for 15 min. Open it on one corner a tiny bit so the samples can cool down more efficiently (don’t open too much, otherwise the buffer will evaporate and samples will dry out).</li>
<li>Wash 3<b>×</b> 5 min with 1<b>×</b> PBS.</li>
<li>Incubate with 500 μL Blocking Solution for 1 h.</li>
<li>Place humidified tissues into the staining chamber to prevent your samples from dehydration during the next steps of the protocol.</li>
<li>Dilute primary antibodies in Staining Solution, add 250 μL antibody mix for one sample and incubate 8–12 h at 4°C.</li>
<li>Wash 3<b>×</b> 5 min with 1<b>×</b> PBS.</li>
<li>Dilute secondary antibodies in Staining Solution, add 250 μL antibody mix for one sample and incubate for 2 h.</li>
<li>Wash 3<b>×</b> 5 min with 1<b>×</b> PBS.</li>
<li>Add 1 mL DAPI working solution for 15 min.</li>
<li>Wash 1<b>×</b> 10 min in PBS</li>
<li>Embed the slides with Mowiol and let dry at 22°C–25°C for 30 min.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Keep the slides at 4°C until imaging. Image quality is highest upon timely acquisition of images following the staining procedure. We recommend to image the sections within 14 days after completion of step 60.</div>
<div class="note">
<span class="note-title">Note:</span> Instead of mounting the floating sections onto glass slides prior to the staining procedure, the whole staining protocol can be performed on floating sections in a 24well plate. Buffers are removed manually using a pasteur pipet or a P1000. We recommend the following volumes when performing stainings in 24well plates:</div>
<table id="undtbl17">
<thead>
<tr>
<th colspan="2">Buffer volumes per well for staining of floating cryosections</th>
</tr>
<tr>
<th>Reagent</th>
<th>Volume (μl)</th>
</tr>
</thead>
<tbody>
<tr>
<td>PBS</td>
<td>1,000</td>
</tr>
<tr>
<td>Antigen retrieval buffer</td>
<td>500</td>
</tr>
<tr>
<td>Blocking Buffer</td>
<td>500</td>
</tr>
<tr>
<td>Staining Buffer (including antibody mix)</td>
<td>250</td>
</tr>
<tr>
<td>DAPI working solution</td>
<td>500</td>
</tr>
</tbody>
</table>
<p>Here, sections should be mounted after the final washing step following the secondary antibody staining. Mounting process is equivalent to steps 40 to 44 of the protocol. Once mounting and section drying is completed, you can proceed with DAPI staining and Mowiol embedding (steps 58 to 60 of the protocol).</p>
<table id="undtbl18">
<caption>Antibodies and their dilutions for MADM staining</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Species</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>Anti-GFP</td>
<td>Chicken</td>
<td>1:400</td>
</tr>
<tr>
<td>Anti-mCherry</td>
<td>Goat</td>
<td>1:400</td>
</tr>
<tr>
<td>Anti-chicken FITC</td>
<td>Donkey</td>
<td>1:500</td>
</tr>
<tr>
<td>Anti-goat Alexa568</td>
<td>Donkey</td>
<td>1:1000</td>
</tr>
</tbody>
</table>
<table id="undtbl19">
<caption>Antibodies and their dilutions for additional marker staining, here an example for staining performed in the colon (<a href="#fig6">Figure 6</a>)</caption>
<thead>
<tr>
<th>Antibody</th>
<th>Species</th>
<th>Dilution</th>
</tr>
</thead>
<tbody>
<tr>
<td>Anti-PhosphoH3</td>
<td>Rabbit</td>
<td>1:800</td>
</tr>
<tr>
<td>Anti-rabbit Alexa647</td>
<td>Donkey</td>
<td>1:1000</td>
</tr>
</tbody>
</table>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Examples and analysis of genetic mosaic MADM animals with mosaicism in the brain</div>
<p>(A–C) Schematic of G2-X MADM events resulting in (A) genetic mosaic tissue using (B) constitutively active Cre driver for population analysis and (C) tamoxifen-inducible CreER driver for clonal analysis.</p>
<p>(D–H) Analysis of central nervous system (CNS) stem cell output in <i>Lgl1</i>-MADM mice using <i>Lgl1</i> mutant allele (<a href="#bib12">Klezovitch et al., 2004</a>). (D) schematic of <i>Lgl1</i>-MADM cortical tissue; (E and F) population analysis of <i>Lgl1</i> using <i>Emx1</i>-Cre driver (<a href="#bib6">Gorski et al., 2002</a>) with (E) overview of a representative brain section at P21, (E′) insert showing the cortex, and (F) quantification of the G/R ratio of astrocytes from wild-type mice and <i>Lgl1</i>-MADM mice; (G and H) clonal analysis of <i>Lgl1</i> using <i>Emx1</i>-CreER driver (<a href="#bib10">Kessaris et al., 2006</a>) with (G) reconstruction of a representative clone at P21 and (H) quantification of the number of green astrocytes in green subclones for wild-type and <i>Lgl1</i> mutant clones. Scale bars: (E) 500 μm, (E′) 60 μm, (G) 70 μm). Values represent mean ± SEM. ns, nonsignificant, ∗∗∗p &lt; 0.001. Parts of the figure and its legend were adapted and reused from (<a href="#bib1">Beattie et al., 2017</a>) with permission.</p>
<p>(I–N) Analysis of cortical projection neuron migration in <i>Ndel1</i>-MADM mice using <i>Ndel1</i> knock-out allele (<a href="#bib18">Sasaki et al., 2005</a>). (I) schematic of <i>Ndel1</i>-MADM cortical tissue; (J–L) population analysis of <i>Ndel1</i> using <i>Emx1</i>-Cre driver (<a href="#bib6">Gorski et al., 2002</a>) with (J) overview of a representative image of the cortex section at P1, (K) schematic indicating partitioning of the cortex into layers and layer-specific quantification of red and green neurons, and (L) quantification of the relative distribution of red wild-type and green <i>Ndel1</i> mutant cells in the respective layers; (M and N) clonal analysis of <i>Ndel1</i> using <i>Nestin</i>-CreER driver [line 1 in (<a href="#bib8">Imayoshi et al., 2006</a>)] with (M) reconstruction of a representative clone at E18.5 and (N) quantification of the relative distribution of red wild-type and green <i>Ndel1</i> mutant cells in the respective layers. Scale bars: (J) 150 μm, (M) 100 μm. Values represent mean ± SEM. ns, nonsignificant; ∗p &lt; 0.05 and ∗∗∗p &lt; 0.001. Parts of the figure and its legend were adapted and reused from (<a href="#bib7">Hippenmeyer et al., 2010</a>) with permission.</p>
<p>(O–R) Analysis of Purkinje cell morphology in <i>TrkC</i>-MADM mice using <i>TrkC</i> knock-out allele (<a href="#bib20">Tessarollo et al., 1997</a>) and <i>Nestin</i>-Cre driver (<a href="#bib16">Petersen et al., 2002</a>). (O) schematic of <i>TrkC</i>-MADM neuronal tissue; (P and Q) representative high magnification images of (P) red wild-type and (Q) green <i>TrkC</i> mutant Purkinje cells in the cerebellum of P21 <i>TrkC</i>-MADM mice, (R) quantification of the height deficiency index of red wild-type and green <i>TrkC</i> mutant cells within the Molecular layer of the cerebellum. Scale bars: (P and Q) 50 μm. Presented morphology values are means ± SEM. ∗∗∗ p &lt; 0.001, one-way analysis of variance (ANOVA) with Tukey’s multiple comparisons test. Parts of the figure and its legend were adapted and reused from (<a href="#bib9">Joo et al., 2014</a>) with permission.</p>
</figcaption>
</figure>
<h3 id="sec3.8">Microscopy and image processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–12 h</div>
<p>In this section of the protocol, we describe the microscopy settings and subsequently required confocal image processing for further downstream analysis.</p>
<ol start="61">
<li>For the analysis of MADM-labeled tissue we recommend to use an inverted confocal microscope, in our case Zeiss LSM800 with corresponding image acquisition software ZEN blue.</li>
<li>Select the correct laser lines and filters. For MADM tissue with additional marker staining, select excitation: 405 nm (for DAPI), 488 nm (for FITC / GFP), 561 nm (for Alexa568 / tdT) and 640 nm (for Alexa647).</li>
<li>Set to pinhole to 1 airy unit for optimal imaging quality.</li>
<li>Select the functions “Z-stack” and “Tiles” from the acquisition dashboard.</li>
<li>Select objective, scanning speed, resolution and averaging appropriate to the needs of your experiments.</li>
<li>Adjust laser intensity and gain for each channel.</li>
<li>Identify your area of interest and set position and imaging tiles to cover the area of interest. Adjust the z-stack so that all MADM-labeled cells in the tissue are captured.</li>
<li>Assign the center position of your Z-stack to the tiles and the positions and click “Start” to initiate image acquisition.</li>
<li>Save the image in .czi file format.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For tissue overview images and cell number quantification, select a 10<b>×</b> objective and use a scanning speed pixel dwell value of 1.52–2.06 μs (values 7–8 in the ZEN blue image acquisition software) with resolution 1024 <b>×</b> 1024 and no averaging. For high-quality images and morphological analysis, use a high magnification objective (e.g. 63<b>×</b>) and use a scanning speed pixel dwell value of maximum 2.06 μs (up to value 7 in the ZEN blue image acquisition software) with resolution 1024 <b>×</b> 1024 and 2<b>×</b> or 4<b>×</b> averaging.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The protocol can be stopped at this point. Image processing and analysis can be performed at any time after image acquisition.</div>
<ol start="70">
<li>Once image acquisition is completed, use the image processing dashboard in ZEN blue and perform maximum intensity projection using the “Orthogonal Projection” function.</li>
<li>Following projection, perform “Stitching” function by fusing tiles.</li>
<li>Save the projected and stitched image as a new .czi file.</li>
<li>Export the projected and stitched .czi file to TIFF format. Export DAPI / GFP / tdT / Alexa647 channels individually for subsequent image analysis.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Compare your readout of green and red cells (such as cell number, cell position, morphological parameters, presence of tumors) in genetic mosaic MADM animals. Since the genotypes of red cells (wild-type) and green cells (mutant) are different, any deviation from a comparable green-to-red (G/R) read-out indicates a cell-autonomous function of <i>gene-X</i> in your cell type of interest.</p>
<p>In order to study stem cell output by counting cell numbers, determine the G/R cell ratio of your cell type of interest (<a href="#fig5">Figures 5</a>F and 5H): If G/R ratio were above 1, it indicates a proliferation increase of green mutant stem cells, whereas a G/R ratio below 1 indicates cell loss either through diminished stem cell proliferation or apoptosis of either stem cells or their descendants. Thus, <i>gene-X</i> has a cell-autonomous function in governing overall stem cell output.</p>
<p>To study cell migration, measure cell position and the number of red wild-type and green mutant cells in certain positions (<a href="#fig5">Figures 5</a>L and 5N): A deviation of numbers of green cells vs. red cells in particular bins is indicative of a cell migration alteration. A large absence of green mutant neurons from cortical layers and their accumulation in the white-matter demonstrates a cell-autonomous function of <i>gene-X</i> in neuronal migration. Based on the sparse endogenous labeling with GFP and tdT, MADM is also well suited to study cell migration in live imaging assays and 4D tracing with high resolution (<a href="#bib23">Riccio et al., 2016</a>; <a href="#bib7">Hippenmeyer et al., 2010</a>).</p>
<p>Determine morphological parameters, such as the height index (<a href="#fig5">Figure 5</a>R): calculate the height index by the formula a/b (a = the distance from the pial surface to the top of the arbor of a Purkinje cell; b= the molecular layer span) and compare the values for red wild-type and green mutant cells side-by-side. If green mutant cells display higher values for height parameters than red wild-type cells, this result indicates a decreased height of the analyzed cells. As a conclusion, <i>gene-X</i> exerts a cell-autonomous function in cellular morphology.</p>
<p>In order to study a disease model such as tumorigenesis (<a href="#fig6">Figure 6</a>C), compare the numbers and area of red and green tumors in your tissue of choice. An increased number of green tumors and tumor area indicates a cell-autonomous function of <i>gene-X</i> in stem cell proliferation and tumor initiation, whereas a decreased number indicates a cell-autonomous disadvantage in tumor growth.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1163-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Example and analysis of genetic mosaic MADM mice with mosaicism in peripheral organs</div>
<p>(A–C) MADM analysis of <i>Apc</i> tumor suppressor gene in the colon using <i>Apc</i>-flox allele (<a href="#bib3">Cheung et al., 2010</a>) and <i>Hprt</i>-Cre driver (<a href="#bib19">Tang et al., 2002</a>). (A) schematic of <i>Apc</i>-MADM tissue; (B) representative image of an adenoma arising from green <i>Apc</i> mutant stem cells in the colon of a 3month old <i>Apc</i>-MADM mouse. Inserts (i) and (ii) show unlabeled healthy colon epithelium, where proliferating cells (as indicated by M-Phase marker staining with P-H3 antibody) are only located in the crypt bottom, but not above the crypt region or along the differentiated epithelium. In contrast insert (iii) highlights a subepithelial region within the adenoma displaying proliferating cells. (C) Quantification of the tumor area originating from red wild-type and green <i>Apc</i> mutant cells. Scale bars: (B) 200 μm, (i, ii, iii) 40 μm. Values represent means ± standard deviation.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<div class="timing">
<span class="timing-title">Timing:</span> several hours or days</div>
<p>Once a stem cell undergoes a G2-X MADM event, one green labelled mutant and one red labelled wild-type daughter cell is generated in an unlabeled environment (<a href="#fig5">Figure 5</a>A). Such sparsely labelled cells can be subjected to a variety of downstream analyses, such as lineage tracing, cell number quantification, positional analysis or morphological analysis (<a href="#fig5">Figure 5</a>).</p>
<p>These distinct types of downstream analysis can be performed for both, population and clonal analysis. While population analysis results in sparse MADM-labeling with no spatiotemporal information (<a href="#fig5">Figure 5</a>B) clonal analysis employing a TM-inducible CreER driver can provide high spatiotemporal resolution (<a href="#fig5">Figure 5</a>C).</p>
<p>The following steps of the protocol provide a short description of the distinct types of histological analysis enabled by the MADM-induced genetic mosaics (also see <a href="#sec7.5">troubleshooting problem 3</a>):</p>
<ol>
<li>For the determination of stem cell output, quantify the number of red and green cells in your acquired images using the counting tool in Photoshop. To do this, load all individual channel images into one image stack to keep channels separate. Use the counting tool to manually quantify the number of distinct cell types derived from red wild-type and green mutant stem cells. Afterwards, calculate and plot the green-to-red (G/R) cell ratio. (<a href="#fig5">Figures 5</a>D–5H).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> if photoshop is not accessible, alternative freely accessible software such as Fiji can be used for the quantification of green and red cells. In Fiji, use the tool ‘multi-points’ for quantification.</div>
<ol start="2">
<li>For the assessment of cellular migration, partition the tissue into defined layers or bins. Next, quantify the number of individual red wild-type and green mutant cells within these layers or bins by using the ‘counting’ tool in photoshop. Plot the cell number per bin for red and green cells side-by-side and compare their distribution to determine if mutant cells are found in a different position than wild-type cells (<a href="#fig5">Figures 5</a>I–5N).</li>
<li>In order to determine cellular morphology, import your .czi image into a software appropriate for morphological analysis, such as Imaris and use the filament tracer function. Determine parameters such as cell volume or complexity using Sholl analysis. Compare the values for red wild-type and green mutant cells to assess if the mutation causes a morphology phenotype (<a href="#fig5">Figures 5</a>O–5R). For assessment of cell height, define specific parameters, such as the distance from the pial surface to the top of the arbor of a Purkinje cell (parameter a) and the molecular layer span (parameter b) (<a href="#fig5">Figure 5</a>Q). Such parameters can be measured with the ‘ruler’ tool using ZEN software on the .czi file of your image or in photoshop or Fiji using other image formats such as .tif.</li>
<li>For disease modeling such as the investigation of tumor initiation and growth, quantify the number of individual tumors using the counting tool in photoshop or determine the size of the tumor area by delineation of the tumor margin in ZEN blue or in photoshop (<a href="#fig6">Figure 6</a>).</li>
</ol>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The genome-wide library of MADM mice allows to employ MADM to study the vast majority, nearly genome-wide (&gt;96%), of autosomal genes in the mouse genome. Mouse autosomes have a telocentric conformation and we thus inserted the MADM cassettes as close as possible to the centromere to maximize the number of genes located distally to the MADM cassette insertion site. In order to be meiotically linked to the corresponding MADM chromosome, the gene of interest needs to be located distally to the MADM cassette. The MADM principle relies on meiotic recombination to genetically link a mutation to the MADM cassette. However, too close proximity of the gene of interest to the MADM cassette might make a recombination event difficult or impossible. In such cases, other techniques offering sparse induction of a mutation of choice could be used, such as MADR (<a href="#bib11">Kim et al., 2019</a>), Dual ifgMosaic (<a href="#bib17">Pontes-Quero et al., 2017</a>) or BATTLE (<a href="#bib13">Kohara et al., 2020</a>).</p>
<p>The library of MADM mice currently does not include strains with MADM cassettes inserted on the sex chromosomes, thus while the vast majority (&gt;96%) of genes can be studied, the genes located on the X or Y chromosome are not yet covered by the MADM technique.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Low number of experimental mice (breeding of experimental MADM mice – steps 1–6).</p>
<h3 id="sec7.2">Potential solution 1</h3>
<p>Make sure the genetic background of your mice does not exhibit negative effects on their mating performance. Increase the number of breeding cages to gain a good number of F1 or F2 mice to start with.</p>
<p>Furthermore, control whether you have correctly followed the provided breeding strategies. Wrongly genotyped or selected parental mice can result in strongly decreased efficiency in the generation of experimental animals.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Weak or variable MADM signal without staining for GFP and tdT (Staining – steps 45–60).</p>
<h3 id="sec7.4">Potential solution 2</h3>
<p>Endogenous fluorescence intensity of GFP and tdT are pH sensitive. Make sure all of your buffers have the correct pH.</p>
<p>Additionally, perform staining for GFP and tdT using the antibodies indicated above.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Highly variable results in MADM lines with low recombination frequency (Quantification and Data Visualization – steps 74–77).</p>
<h3 id="sec7.6">Potential solution 3</h3>
<p>Analysis of all 19 MADM lines showed that different MADM chromosomes harbor distinct recombination frequencies [sparse, intermediate or dense, (<a href="#bib4">Contreras et al., 2021</a>)]. The required number of experimental animals and the number of tissue sections used for analysis need to be scaled up when working with a MADM chromosome that gives sparse labeling. We recommend using the entire amount of tissue sections available for one animal to avoid introduction of an analysis bias when only using a few sections from a low recombination line.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low breeding performance (Generation of experimental MADM mice for histological analysis, steps 7–8).</p>
<h3 id="sec7.8">Potential solution 4</h3>
<p>Discuss with your veterinarian how to optimize the mating conditions (e.g., by providing special food or housing conditions).</p>
<h3 id="sec7.9">Problem 5</h3>
<p>High abortion rate upon Tamoxifen injection for clonal analysis (Generation of experimental MADM mice for histological analysis, step 7).</p>
<h3 id="sec7.10">Potential solution 5</h3>
<p>Titrate the concentration of Tamoxifen according to your experimental needs to find an optimal balance between survival and clone numbers. Additionally, recover pups by caesarian sections and perform fostering to increase the survival rate of experimental animals.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Beattie, R., Postiglione, M.P., Burnett, L.E., Laukoter, S., Streicher, C., Pauler, F.M., Xiao, G., Klezovitch, O., Vasioukhin, V., Ghashghaei, T.H., and Hippenmeyer, S. (2017). Mosaic analysis with double markers Reveals distinct sequential functions of Lgl1 in neural stem cells. Neuron <i>94</i>, 517-533.e3. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref1">View at publisher</a></p>
<p id="bib2">Beattie, R., Streicher, C., Amberg, N., Cheung, G., Contreras, X., Hansen, A.H., and Hippenmeyer, S. (2020). Lineage tracing and clonal analysis in developing cerebral cortex using mosaic analysis with double markers (MADM). J. Vis. Exp. <a class="external-link" href="https://doi.org/10.3791/61147">https://doi.org/10.3791/61147</a></p>
<p id="bib3">Cheung, A.F., Carter, A.M., Kostova, K.K., Woodruff, J.F., Crowley, D., Bronson, R.T., Haigis, K.M., and Jacks, T. (2010). Complete deletion of Apc results in severe polyposis in mice. Oncogene <i>29</i>, 1857-1864. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref3">View at publisher</a></p>
<p id="bib4">Contreras, X., Amberg, N., Davaatseren, A., Hansen, A.H., Sonntag, J., Andersen, L., Bernthaler, T., Streicher, C., Heger, A., Johnson, R.L., et al. (2021). A genome-wide library of MADM mice for single-cell genetic mosaic analysis. Cell Rep. <i>35</i>, 109274. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref4">View at publisher</a></p>
<p id="bib5">Gao, P., Postiglione, M.P., Krieger, T.G., Hernandez, L., Wang, C., Han, Z., Streicher, C., Papusheva, E., Insolera, R., Chugh, K., et al. (2014). Deterministic progenitor behavior and unitary production of neurons in the neocortex. Cell <i>159</i>, 775-788. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref5">View at publisher</a></p>
<p id="bib6">Gorski, J.A., Talley, T., Qiu, M., Puelles, L., Rubenstein, J.L., and Jones, K.R. (2002). Cortical excitatory neurons and glia, but not GABAergic neurons, are produced in the Emx1-expressing lineage. J. Neurosci. <i>22</i>, 6309-6314. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref6">View at publisher</a></p>
<p id="bib7">Hippenmeyer, S., Youn, Y.H., Moon, H.M., Miyamichi, K., Zong, H., Wynshaw-Boris, A., and Luo, L. (2010). Genetic mosaic dissection of Lis1 and Ndel1 in neuronal migration. Neuron <i>68</i>, 695-709. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref7">View at publisher</a></p>
<p id="bib8">Imayoshi, I., Ohtsuka, T., Metzger, D., Chambon, P., and Kageyama, R. (2006). Temporal regulation of Cre recombinase activity in neural stem cells. Genesis <i>44</i>, 233-238. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref8">View at publisher</a></p>
<p id="bib9">Joo, W., Hippenmeyer, S., and Luo, L. (2014). Dendrite morphogenesis depends on relative levels of NT-3/TrkC signaling. Science <i>346</i>, 626-629. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref9">View at publisher</a></p>
<p id="bib10">Kessaris, N., Fogarty, M., Iannarelli, P., Grist, M., Wegner, M., and Richardson, W.D. (2006). Competing waves of oligodendrocytes in the forebrain and postnatal elimination of an embryonic lineage. Nat. Neurosci. <i>9</i>, 173-179. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref10">View at publisher</a></p>
<p id="bib11">Kim, G.B., Rincon Fernandez Pacheco, D., Saxon, D., Yang, A., Sabet, S., Dutra-Clarke, M., Levy, R., Watkins, A., Park, H., Abbasi Akhtar, A., et al. (2019). Rapid generation of somatic mouse mosaics with locus-specific, stably Integrated transgenic elements. Cell <i>179</i>, 251-267 e24. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref11">View at publisher</a></p>
<p id="bib12">Klezovitch, O., Fernandez, T.E., Tapscott, S.J., and Vasioukhin, V. (2004). Loss of cell polarity causes severe brain dysplasia in Lgl1 knockout mice. Genes Dev. <i>18</i>, 559-571. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref12">View at publisher</a></p>
<p id="bib13">Kohara, K., Inoue, A., Nakano, Y., Hirai, H., Kobayashi, T., Maruyama, M., Baba, R., and Kawashima, C. (2020). BATTLE: genetically engineered strategies for split-tunable allocation of multiple transgenes in the nervous system. iScience <i>23</i>, 101248. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref13">View at publisher</a></p>
<p id="bib14">Laukoter, S., Amberg, N., Pauler, F.M., and Hippenmeyer, S. (2020a). Generation and isolation of single cells from mouse brain with mosaic analysis with double markers-induced uniparental chromosome disomy. STAR Protoc. <i>1</i>, 100215. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref14">View at publisher</a></p>
<p id="bib15">Laukoter, S., Beattie, R., Pauler, F.M., Amberg, N., Nakayama, K.I., and Hippenmeyer, S. (2020b). Imprinted Cdkn1c genomic locus cell-autonomously promotes cell survival in cerebral cortex development. Nat. Commun. <i>11</i>, 195. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref15">View at publisher</a></p>
<p id="bib16">Petersen, P.H., Zou, K., Hwang, J.K., Jan, Y.N., and Zhong, W. (2002). Progenitor cell maintenance requires numb and numblike during mouse neurogenesis. Nature <i>419</i>, 929-934. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref16">View at publisher</a></p>
<p id="bib17">Pontes-Quero, S., Heredia, L., Casquero-Garcia, V., Fernandez-Chacon, M., Luo, W., Hermoso, A., Bansal, M., Garcia-Gonzalez, I., Sanchez-Munoz, M.S., Perea, J.R., et al. (2017). Dual ifgMosaic: a versatile method for multispectral and combinatorial mosaic gene-function analysis. Cell <i>170</i>, 800-814 e18. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref17">View at publisher</a></p>
<p id="bib23">Riccio, P., Cebrian, C., Zong, H., Hippenmeyer, S., and Constantini, F. (2016). Ret and Etv4 Promote Directed Movements of Progenitor Cells during Renal Branching Morphogenesis. PloS Biol <i>14</i>, e1002488. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/opt5G7Q35RQF8">View at publisher</a></p>
<p id="bib18">Sasaki, S., Mori, D., Toyo-Oka, K., Chen, A., Garrett-Beal, L., Muramatsu, M., Miyagawa, S., Hiraiwa, N., Yoshiki, A., and Wynshaw-Boris, A. (2005). Complete loss of Ndel1 results in neuronal migration defects and early embryonic lethality. Mol. Cell. Biol. <i>25</i>, 7812-7827. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref18">View at publisher</a></p>
<p id="bib19">Tang, S.H.E., Silva, F.J., Tsark, W.M., and Mann, J.R. (2002). A Cre/loxP-deleter transgenic line in mouse strain 129S1/SvImJ. Genesis <i>32</i>, 199-202. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref19">View at publisher</a></p>
<p id="bib20">Tessarollo, L., Tsoulfas, P., Donovan, M.J., Palko, M.E., Blair-Flynn, J., Hempstead, B.L., and Parada, L.F. (1997). Targeted deletion of all isoforms of the trkC gene suggests the use of alternate receptors by its ligand neurotrophin-3 in neuronal development and implicates trkC in normal cardiogenesis. Proc. Natl. Acad. Sci. U S A <i>94</i>, 14776-14781. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref20">View at publisher</a></p>
<p id="bib21">Tian, A., Kang, B., Li, B., Qiu, B., Jiang, W., Shao, F., Gao, Q., Liu, R., Cai, C., Jing, R., et al. (2020). Oncogenic state and cell identity combinatorially dictate the susceptibility of cells within glioma development hierarchy to IGF1R targeting. Adv. Sci. <i>7</i>, 2001724. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref21">View at publisher</a></p>
<p id="bib22">Zong, H., Espinosa, J.S., Su, H.H., Muzumdar, M.D., and Luo, L. (2005). Mosaic analysis with double markers in mice. Cell <i>121</i>, 479-492. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00645-6/sref22">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact Simon Hippenmeyer (<a href="mailto:simon.hippenmeyer@ist.ac.at">simon.hippenmeyer@ist.ac.at</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>No custom code is necessary to perform this protocol.</p>
<h3>Acknowledgments</h3>
<p>This research was supported by the Scientific Service Units (SSU) at <a href="https://doi.org/10.13039/100014988">IST Austria</a> through resources provided by the Bioimaging (BIF) and Preclinical Facilities (PCF). We particularly thank Mohammad Goudarzi for assistance with photography of mouse perfusion and dissection. N.A. received support from <a href="https://doi.org/10.13039/501100002428">FWF Firnberg-Programm</a> (T 1031). This work was also supported by <a href="https://doi.org/10.13039/100014988">IST Austria</a> institutional funds; FWF SFB F78 to S.H.; and the <a href="https://doi.org/10.13039/100010663">European Research Council</a> (ERC) under the European Union’s Horizon 2020 research and innovation programme (grant agreement no. 725780 LinPro) to S.H.</p>
<h3>Author contributions</h3>
<p>All authors conceived the project and edited and proof-read the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>