<article>
<section>
<h1 id="header">Detection, sorting, and immortalization of dual expresser lymphocytes from human peripheral blood samples</h1>
<p><time datetime="2021-11-05">Published: November 5, 2021</time></p>
<p>Rizwan Ahmed,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> Kusuma Ananth,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Zahra Omidian,<sup><a href="#aff1">1</a></sup> Neha Majety,<sup><a href="#aff1">1</a></sup> Hao Zhang,<sup><a href="#aff2">2</a></sup> and Abdel Rahim A. Hamad<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Pathology, Johns Hopkins University School of Medicine, Baltimore, MD 21205, USA</p>
<p id="aff2"><sup>2</sup>Department of Molecular Microbiology and Immunology, Johns Hopkins Bloomberg School of Public Health, Baltimore, MD 21205, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:rahmed14@jhu.edu">rahmed14@jhu.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:ahamad@jhu.edu">ahamad@jhu.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100925">10.1016/j.xpro.2021.100925</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>This protocol describes how to identify Dual Expressers (DEs), a rare type of lymphocyte that co-expresses B-cell receptors and T-cell receptors, by flow cytometry using a cocktail of four antibodies. It also shows the subsequent gating strategy for detecting and sorting DEs and the generation of EBV-immortalized DE lymphoblastoid cell lines and clones for antibody production and cloning antigen receptors. Use of this protocol maximizes detection of DEs and minimizes inclusion of doublets.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.cell.2019.05.007">Ahmed et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol to detect DEs and distinguish from T and B cells by flow cytometry</li>
<li>Sequential gating strategy to maximize and optimize detection of DEs</li>
<li>Strategy for efficient FACS-sorting and checking purity of sorted DEs</li>
<li>Protocol for generating EBV-immortalized DE lines and clones</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1140-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Isolation and aliquoting of PBMCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<p>This procedure includes the following: (a) Collecting peripheral blood samples from donors (patients with an autoimmune disease or healthy control subjects) using an approved IRB protocol, (b) Isolating peripheral blood mononuclear cells (PBMCs) by performing density gradient centrifugation using Ficoll-paque density gradient method according to manufacturer’s instructions: <a href="https://www.sigmaaldrich.com/US/en/technical-documents/protocol/clinical-testing-and-diagnostics-manufacturing/hematology/recommended-standard-method">https://www.sigmaaldrich.com/US/en/technical-documents/protocol/clinical-testing-and-diagnostics-manufacturing/hematology/recommended-standard-method</a>. (c) Aliquoting PBMCs (5–10 <b>×</b>10<sup>6</sup> cells in 1 mL freezing medium) per cryovial and store at −80°C for short terms (up to 6 months) and in liquid nitrogen for long term storage (up to I year).</p>
<h3 id="sec1.2">Thawing cryopreserved PBMCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<ol>
<li>Thaw cryopreserved PBMCs by placing cryovials (one at a time) in a 37°C water bath and shake the vial gently until about 95% of the freezing medium has melted and only a small chunk of ice remains. This should take about 1 min. Immediately place the cryovial on ice and quickly move to the next step to minimize the amount of time thawed cells stay in freezing medium.</li>
</ol>
<ol start="2">
<li>Wash PBMCs with wash medium (see <a href="#materials-and-equipment">materials and equipment</a>).
<ol type="a">
<li>Invert the cryovial twice to resuspend PBMCs.</li>
<li>Open the vial and slowly transfer cells from each cryovial to a 15 mL polyethylene tube containing 10 mL of prewarmed (37°C) wash medium.
<div class="note">
<span class="note-title">Note:</span> We find that the use of wash medium ensures maximum viability, stability and maintains integrity of thawed-PBMCs. The wash buffer also reduces cell loss through aggregation.</div>
</li>
<li>Centrifuge cells at 400 <b>×</b> <i>g</i> for 5 min at room temperature (RT; 22°C–25°C).</li>
<li>Remove the supernatant carefully with a pipette and perform an additional wash with complete tissue culture medium (CTM) (see <a href="#materials-and-equipment">materials and equipment</a>).</li>
<li>Resuspend the pellet and filter the cell suspension using a 70 μm cell strainer, and place in a new 15 mL polyethylene tube.</li>
<li>Count viable cells using the Trypan Blue exclusion method. We routinely obtain ∼ 90%–95% viability.</li>
<li>If needed, use CTM to adjust cell density to the desired concentration (e.g., 1 <b>×</b> 10<sup>6</sup> cells / mL).</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>APC anti-human CD5 (L17F12)</td>
<td>BioLegend</td>
<td>Cat#364016; RRID: AB_2565726</td>
</tr>
<tr>
<td>BV421 anti-human CD19 (HIB19)</td>
<td>BD Biosciences</td>
<td>Cat#562440; RRID: AB_11153299</td>
</tr>
<tr>
<td>AF-488 anti-human TCRαβ (IP26)</td>
<td>BioLegend</td>
<td>Cat#306712; RRID: AB_528967</td>
</tr>
<tr>
<td>PE anti-human IgD (IA6-2)</td>
<td>BioLegend</td>
<td>Cat#348204; RRID: AB_10553900</td>
</tr>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>Epstein-Barr Virus B95-8</td>
<td>American Type Cell Culture (ATCC)</td>
<td>Cat#ATCC VR-1492</td>
</tr>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>Human peripheral blood samples (8- to 80-year-old males and females)</td>
<td>Patients with autoimmune disease or healthy controls</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>2-mercaptoethanol</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#21985023</td>
</tr>
<tr>
<td>Antibiotic Antimycotic Solution (AAS)</td>
<td>Sigma Aldrich</td>
<td>Cat#A5955-100ML</td>
</tr>
<tr>
<td>Benzonase nuclease</td>
<td>Sigma Aldrich</td>
<td>Cat#E8263</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Sigma Aldrich</td>
<td>Cat##A1933-100G</td>
</tr>
<tr>
<td>CpG ODN 2006 (ODN 7909)</td>
<td>InvivoGen</td>
<td>Cat#tlrl-2006-1</td>
</tr>
<tr>
<td>Dimethyl Sulfoxide (DMSO)</td>
<td>Fisher Scientific</td>
<td>Cat#BP231-100</td>
</tr>
<tr>
<td>Fetal Bovine Serum (FBS)</td>
<td>Sigma Aldrich</td>
<td>Cat#F2442-500ML</td>
</tr>
<tr>
<td>Ficoll Paque Plus</td>
<td>MilliporeSigma</td>
<td>Cat#GE17-1440-02</td>
</tr>
<tr>
<td>Hanks’ Balanced Salt Solution (HBSS)</td>
<td>MilliporeSigma</td>
<td>Cat#55037C</td>
</tr>
<tr>
<td>HEPES</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15630-080</td>
</tr>
<tr>
<td>Live/Dead Fixable Aqua</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#L34957</td>
</tr>
<tr>
<td>Non-essential amino acids (NEAA)</td>
<td>Quality Biological</td>
<td>Cat#116-078-721</td>
</tr>
<tr>
<td>Phosphate Buffered Saline (PBS)</td>
<td>Corning</td>
<td>Cat#46-013-CM</td>
</tr>
<tr>
<td>RPMI with L-glutamine</td>
<td>Corning</td>
<td>Cat#10-040-CV</td>
</tr>
<tr>
<td>Sodium Azide</td>
<td>Sigma Aldrich</td>
<td>Cat# S2002</td>
</tr>
<tr>
<td>Sodium Pyruvate</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#11360070</td>
</tr>
<tr>
<td>Propidium Iodide</td>
<td>BD Biosciences</td>
<td>Cat#556463</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>IRR-MRC-5 irradiated Fibroblast</td>
<td>American Type Cell Culture (ATCC)</td>
<td>Cat#ATCC 55-X</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>FlowJo</td>
<td>TreeStar</td>
<td><a href="https://www.flowjo.com/solutions/flowjo">https://www.flowjo.com/solutions/flowjo</a></td>
</tr>
<tr>
<td colspan="3">Others</td>
</tr>
<tr>
<td>0.22 μm vacuum filtration flask</td>
<td>Corning</td>
<td>Cat#431097</td>
</tr>
<tr>
<td>70 μm cell strainers</td>
<td>Falcon</td>
<td>Cat#352350</td>
</tr>
<tr>
<td>FACS tubes</td>
<td>MTC Bio</td>
<td>Cat#T9014</td>
</tr>
<tr>
<td>FACS tubes with 35 μm cell strainer caps</td>
<td>Falcon</td>
<td>Cat#352235</td>
</tr>
<tr>
<td>Multi-color Flow cytometer</td>
<td>Of your choice</td>
<td>N/A</td>
</tr>
<tr>
<td>MoFlo Legacy High Speed Cell Sorter</td>
<td>Beckman Coulter (or a sorter of your choice)</td>
<td>beckmancoulter.com</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>Freezing medium</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI with L-glutamine</td>
<td>n/a</td>
<td>200 mL</td>
</tr>
<tr>
<td>DMSO</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>50%</td>
<td>250 mL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare freezing medium under sterile conditions using a biosafety cabinet. Combine all the reagents, then filter the freezing media using a 0.2 μm filter (vacuum filtration flask). Aliquot filtered medium into 50 mL falcon tubes and store at −20°C for up to three weeks.</p>
<table id="undtbl2">
<caption>Wash medium</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Complete tissue culture medium (CTM)</td>
<td>n/a</td>
<td>10 mL</td>
</tr>
<tr>
<td>Benzonase (250,000 U/mL)</td>
<td>0.02%</td>
<td>2 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare wash medium under sterile conditions using a biosafety cabinet and store at 4°C for up to three weeks.</p>
<table id="undtbl3">
<caption>Complete tissue culture medium (CTM)</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI with L-glutamine</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>Heat-inactivated FBS</td>
<td>8.7%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Anti-mycotic and antibiotic solution (AAS)</td>
<td>1%</td>
<td>6 mL</td>
</tr>
<tr>
<td>HEPES buffer</td>
<td>1%</td>
<td>6 mL</td>
</tr>
<tr>
<td>Non-essential amino acids (NEAA)</td>
<td>1%</td>
<td>6 mL</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>1%</td>
<td>6 mL</td>
</tr>
<tr>
<td>Beta mercaptoethanol</td>
<td>0.08%</td>
<td>450 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>574.45 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare CTM under sterile conditions in a biosafety cabinet. After mixing all reagents, filter the CTM using a 0.22 μm filter (vacuum filtration flask) and store at 4°C until use for about three weeks.</p>
<table id="undtbl4">
<caption>FACS buffer</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1<b>×</b> Phosphate buffered saline (PBS)</td>
<td>n/a</td>
<td>500 mL</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>1%</td>
<td>5 g</td>
</tr>
<tr>
<td>Sodium azide</td>
<td>0.05%</td>
<td>250 mg</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C and use within three weeks.</p>
<table id="undtbl5">
<caption>Lymphocyte collection medium</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI with L-glutamine</td>
<td>n/a</td>
<td>229.6 mL</td>
</tr>
<tr>
<td>Heat-inactivated FBS</td>
<td>50%</td>
<td>250 mL</td>
</tr>
<tr>
<td>Anti-mycotic and antibiotic solution (AAS)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>HEPES buffer</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Non-essential amino acids (NEAA)</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Beta mercaptoethanol</td>
<td>0.08%</td>
<td>400 μL</td>
</tr>
<tr>
<td>Total</td>
<td> </td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare the collecting medium under sterile conditions using a biosafety cabinet. After combining all reagents, filter the medium using a 0.22 μm filter (vacuum filtration flask) and store at 4°C until use within three weeks.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by step method details</h2>
<h3 id="sec3.1">Part I. Detection of DEs by flow cytometry</h3>
<p>This section details the methods for: (A) Staining PBMCs with our standard cocktail of 4-fluorophore-conjugated antibodies specific for CD19, CD5, IgD, and TCRαβ. (B) Sequential gating strategy to simultaneously detect DE cells, conventional T cells (Tcon) and conventional B cells (Bcon) cells in the same PBMC sample and compare their phenotypes – See below for details.</p>
<h4 id="sec3.1.1">Identifying DEs by flow cytometry</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<div class="note">
<span class="note-title">Note:</span> Use fresh PBMCs for flow cytometry whenever possible. However, when using cryopreserved PBMCs, additional attention should be paid to exclude debri and doublets.</div>
<div class="note">
<span class="note-title">Note:</span> To properly detect DEs, you will need to stain sample(s) with the cocktail of 4 antibodies (See <a href="#tbl1">Table 1</a>). In addition, you will need 5 compensation control tubes (4 tubes for single fluorophore staining, and 1 tube for unstained sample).</div>
<div class="optional">
<span class="optional-title">Optional:</span> You may stain your sample(s) with Live/Dead dye, such as PI or Live/Dead Aqua. It is highly recommended to use Live/Dead in your initial experiments until you become familiar with analyzing DEs or when you are expecting high numbers of dead cells in your samples.</div>
<table id="tbl1">
<caption>Table 1. Antibody cocktail used for detecting DEs</caption>
<thead>
<tr>
<th>Surface marker</th>
<th>Fluorophore</th>
<th>
<sup><a href="#tblfn1">a</a></sup>Suggested dilution factor</th>
</tr>
</thead>
<tbody>
<tr>
<td>CD5</td>
<td>APC</td>
<td>1:20</td>
</tr>
<tr>
<td>CD19</td>
<td>Brilliant Violent 421</td>
<td>1:20</td>
</tr>
<tr>
<td>TCRαβ</td>
<td>Alexa Fluor 488</td>
<td>1:20</td>
</tr>
<tr>
<td>IgD</td>
<td>PE</td>
<td>1:20</td>
</tr>
</tbody>
</table>
<p id="tblfn1" class="table-footnote"><sup>a</sup>We suggest you determine optimal concentrations of your antibodies by doing your own titrations.</p>
<ol>
<li>Count live cells using the Trypan blue dye exclusion method.</li>
<li>Label your FACS tubes with permanent marker.</li>
<li>Add 1 to 2 million cells in 100 μL to each FACS tube. If you do not have enough cells, you may use fewer cells for compensation (200,000 cells per control tube).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> you may use compensation beads instead of cells. However, we prefer to use cells from the same sample because beads are usually more homogeneous and emit fluorescent intensity that does not necessarily match the fluorescent intensity of the molecule on the surface of cells of your interest.</div>
<ol start="4">
<li>Stain your sample(s) and controls:
<ol type="a">
<li>If you would like, stain your sample(s) first with the Live/Dead dye of your choice. We generally use PI (1 μg/mL) or Live/Dead Aqua (1 μL of dye as prepared in kit in 1 mL PBS). If you choose to stain your samples with Live/Dead, make sure you make a single-color compensation control for it and follow manufacturer’s instructions.</li>
<li>Stain your sample(s) with the cocktail of fluorophore-conjugated antibodies specific for TCRαβ, IgD, CD5, and CD19. We use 5 μL of each antibody and stain in FACS buffer, at a final volume of 100 μL. See <a href="#tbl1">Table 1</a> for the fluorophores we use and suggested dilution factors.
<div class="note">
<span class="note-title">Note:</span> if you are staining multiple samples, you may prepare a cocktail sufficient for all of them.</div>
</li>
<li>Stain your single-color compensation controls.</li>
<li>Vortex tubes and incubate on ice for 20–30 min, protected from light.</li>
</ol>
</li>
</ol>
<ol start="5">
<li>Wash your sample(s) and controls with 2–3 mL of FACS buffer. Centrifuge at 525 xg (1500 rpm) and discard supernatants.</li>
<li>Resuspend samples in 600 μL of FACS buffer.</li>
<li>Vortex tubes. They are now ready for flow cytometric analysis.</li>
<li>Move to your FACS instrument and perform compensation setup using your compensation controls.</li>
<li>Run your samples and acquire 500,000–1,000,000 events per sample when possible.</li>
<li>Export the FSC files.</li>
</ol>
<h4 id="sec3.1.2">Sequential gating strategy to identify DEs and analyze their phenotypes</h4>
<p>This section details the sequential gating strategy we employ for identifying and analyzing DEs in acquired samples using the FlowJo software. The protocol is excellent for analyzing samples for the presence and determining the frequency of DEs.</p>
<div class="note">
<span class="note-title">Note:</span> When desired, stain samples for appropriate markers for additional phenotypic and functional characterization of DEs and compare their properties to those of autologous B<sub>con</sub> and T<sub>con</sub>.</div>
<div class="timing">
<span class="timing-title">Timing:</span> 30–60 min</div>
<ol start="11">
<li>Open FlowJo and drag your FCS files into the application</li>
<li>Open your sample files.</li>
<li>Gate total lymphocytes using the FSC-Area vs SSC-Area plot.</li>
<li>Gate on single cells and exclude doublets using the FSC-Area vs FSC-Height followed by SSC-Area vs SSC-Height plots. Alternatively, doublets can be excluded by using FSC-Height vs FSC-Width and SSC-Height vs SSC-Width plots.</li>
<li>If you stained your samples with Live/Dead, gate on live cells using a histogram of the Live/Dead staining as recommended by the manufacturer.</li>
<li>Draw a 2-parameter dot plot using CD5 vs CD19. Gate on B<sub>con</sub> (CD5<sup>−</sup> CD19<sup>+</sup>), T<sub>con</sub> (CD5<sup>+</sup> CD19<sup>−</sup>) and the CD5<sup>+</sup> CD19<sup>+</sup> subpopulations (we often use rectangle gates).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When gating for DEs, avoid CD5<sup>high</sup> CD19<sup>high</sup> cells because they are almost always highly enriched for doublets. Thus, it is recommended that when gating on the CD5<sup>+</sup> CD19<sup>+</sup> population limit gating to CD19<sup>+</sup>CD5<sup>low</sup> cells (see <a href="#fig1">Figures 1</a> and <a href="#fig2">2</a>).</div>
<ol start="17">
<li>Analyze each subpopulation for the presence of TCRαβ<sup>+</sup>IgD<sup>+</sup> DEs. Use the B<sub>con</sub> population as a reference to place the gates that distinguish between TCRαβ<sup>+</sup> and TCRαβ<sup>−</sup> and IgD<sup>+</sup> and IgD<sup>−</sup> subpopulations.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> DEs are mainly found within the CD5<sup>+</sup> CD19<sup>+</sup> population and are defined as lymphocytes that coexpress TCRαβ and surface immunoglobulin (BCR). According to the surface immunoglobulin expressed, DEs are divided into TCRαβ<sup>+</sup> IgD<sup>+</sup> or TCR<sup>+</sup> IgD<sup>−</sup> subpopulation. The latter are often class-switched to either IgG or IgA. The phenotype of TCR<sup>+</sup>IgD<sup>−</sup> cells should be confirmed by staining for the expression of IgG or IgA. For more details about the gating strategy, please refer to <a href="#fig1">Figures 1</a> and <a href="#fig2">2</a> and supplementary data in the original manuscript (<a href="#bib2">Ahmed et al., 2021</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> While gating DEs, an important step is to exclude doublets. We do so in 2 ways: 1) as mentioned in step 14, and 2) by ensuring that DEs that are gated are CD5<sup>low</sup> and TCR<sup>low</sup> (See <a href="#fig3">Figure 3</a>). We believe that the CD5<sup>high</sup> and TCRαβ<sup>high</sup> cells are mostly doublets and not DEs.</div>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1140-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Consequential gating strategy for acquiring samples and detecting DEs, Bcon, and Tcon by using FACS and FlowJo</div>
<p>(A) Dot plots show how to sequentially gate live single lymphocytes and exclude doublets and dead cells in your PBMC sample.</p>
<p>(B) Left dot plot shows how to use a 2-parameter plot of CD19 versus CD5 to divide gated lymphocytes into B<sub>con</sub> (CD5<sup>−</sup>CD19<sup>+</sup>); T<sub>con</sub> (CD5<sup>+</sup>CD19<sup>−</sup>), and CD5<sup>+</sup>CD19<sup>+</sup> populations. Right dot plots show how to analyze each gated subpopulation for DEs as identified by the coexpression of TCRαβ and BCR. Note: most DEs are located within the CD5<sup>+</sup>CD19<sup>+</sup> compartments, rare among B<sub>con</sub>, and virtually absent among T<sub>con</sub>. Consequently, analysis and sorting of DEs, is limited to DEs localized within the CD5<sup>+</sup> CD19<sup>+</sup> population. Numbers indicate percentages in each quadrant. This figure is adopted from the manuscript by Ahmed and colleagues (<a href="#bib1">Ahmed et al., 2019</a>).</p>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1140-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Steps used to sort and verify identity of DEs, Bcon, and Tcon</div>
<p>(A) In this series of dot plots, consecutive gating was used to identify lymphocytes and exclude doublets and dead cells, respectively, among PBMCs stained with the four-antibody cocktail.</p>
<p>(B) Left dot plot (CD5 vs CD19) shows how gated live lymphocyte singlets are divided into CD5<sup>−</sup>CD19<sup>+</sup> Bcon, CD5<sup>+</sup>CD19<sup>−</sup> Tcon, and CD5<sup>+</sup>CD19<sup>+</sup> subpopulations – The identities of CD5<sup>−</sup>CD19<sup>+</sup> cells and CD5<sup>+</sup>CD19<sup>−</sup> cells as Bcon and Tcon cells are confirmed by analysis each subpopulation for the lack or expression of the TCRαβ, respectively (not shown). Right dot plot (IgD vs TCRαβ) shows how gated CD5<sup>+</sup>CD19<sup>+</sup> cells are used to identify DE subsets. DEs are divided into a major CD5<sup>+</sup>CD19<sup>+</sup>TCRαβ<sup>+</sup>IgD<sup>+</sup> subset and a minor CD5<sup>+</sup>CD19<sup>+</sup>TCRαβ<sup>+</sup>IgD<sup>−</sup> cells (contains class-switched DEs, see Ahmed et al. <a href="#bib1">Ahmed et al. (2019)</a> for details).</p>
<p>(C) This series of plots shows the steps used to verify identity of sorted DEs by re-running an aliquot of sorted DEs. Bcon, and Tcon cells. Numbers in quadrants indicate percentages.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1140-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Exclusion of CD5<sup>hi</sup> and TCRαβ<sup>hi</sup> cells from analysis enhances purity of sorted DEs</div>
<p>In this experiment, a PBMC sample from T1D patients was stained using our four-antibody cocktail, acquired and two gating strategies used to identify and sort DEs as shown below. Purity was determined by rerunning sorted DEs. <b>Top panel.</b> The plots show how inclusion of all CD5<sup>+</sup> cells (left plot) and TCRαβ<sup>+</sup> cells (middle plot) significantly lowers purity of sorted DEs (right plot). The number in the right plot shows percentage of bona fide DEs (19%) among sorted cells. <b>Bottom panel</b>. The plots show how restricting gating to CD5<sup>low</sup>CD19+ (left plot) and TCRαβ<sup>low</sup> cells (middle plot) cells significantly increases purity of sorted DEs (right plot). The number in the right plot shows percentage of bona fide DEs (82%) among sorted cells.</p>
</figcaption>
</figure>
<h3 id="sec3.2">Part II. Strategy for sorting DE cells</h3>
<h4 id="sec3.2.1">Preparation of single-cell suspensions and antibody staining</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="18">
<li>We sort DEs using about 50 <b>×</b>10<sup>6</sup> fresh PBMCs initially suspended in 500 μL FACS buffer.
<ol type="a">
<li>About 50 μL (∼ 5 <b>×</b>10<sup>6</sup> cells) will be used for preparing the single color compensation and unstained controls.</li>
<li>The remaining 45 <b>×</b>10<sup>6</sup> cells in about 450 μL will be used for sorting.</li>
</ol>
</li>
</ol>
<ol start="19">
<li>We use the same staining strategy and the antibody cocktail described in Part I: <a href="#sec3.1.1">Identifying DEs by flow cytometry</a>. See <a href="#tbl1">Table 1</a> for the antibody panel and dilution factors.
<ol type="a">
<li>Prepare and label single-color compensation control tubes and unstained control tube by pipetting 10 μL of PBMC suspension (5 <b>×</b>10<sup>5</sup> cells) from the “master tube” into each control tube. Then add 90 μL FACS buffer for a total volume of 100 μL in the compensation control tubes.</li>
<li>Stain the sorting sample using 20 μL of each fluorophore-conjugated antibody and stain the compensation control tubes using 5 μL of antibody.
<div class="note">
<span class="note-title">Note:</span> It is always recommended that you identify and use appropriate antibody concentrations.</div>
<div class="note">
<span class="note-title">Note:</span> Importantly, we use propodeum iodide (PI) – however, you can use your preferred viability dye – for detection and exclusion of dead cells. We perform this step after staining and immediately before running samples and data acquisition (see step 21 in Part II: <a href="#sec3.2.2">Sorting of DEs and autologous Tcon and Bcon cells</a>).</div>
</li>
<li>Gently vortex each FACS tube for mixing cells and homogeneous staining.</li>
<li>Incubate tubes for 20 min on ice or at 4<sup>o</sup>C, protected from light.</li>
<li>Wash samples at least three times using 2–3 mL of FACS buffer each time.</li>
<li>Finally, resuspend the sorting sample into a total volume of 3 mL and each of the compensation control samples into a total volume of 500 μL of FACS buffer. Now, the samples are ready for sorting.</li>
</ol>
</li>
</ol>
<h4 id="sec3.2.2">Sorting of DEs and autologous Tcon and Bcon cells</h4>
<p>We use High Speed MoFlo Legacy Cell Sorter - Beckman Coulter – However, you can use a sorter of your choice.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 h</div>
<ol start="20">
<li>Immediately before sorting, filter the “sort sample” using a FACS tube with 35 μm cell strainer cap to eliminate residual cell clumps.</li>
<li>Add PI to identify and exclude dead cells using stock solution of 1 mg/mL and final dilution of 1–2 μg / mL.</li>
<li>Setting up the compensation and gating strategy
<ol type="a">
<li>Acquire approximately 10,000 events from each compensation tube to perform compensation setup.</li>
<li>Finalize the compensation using the fully-stained sample and make final adjustments as necessary (see <a href="#fig2">Figure 2</a>).</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Collection of sorted subpopulations
<ol type="a">
<li>Set up your sorter for sorting. We use a 70 μm nozzle, sheath pressure of 60 psi, drop drive frequency of 95–98 kHz. The flow rate for sorting is approximately 1000 events/second.</li>
<li>Sort your sample into three or four subpopulations, depending on whether DEs are sorted as separate (IgD<sup>+</sup> and IgD<sup>−</sup>) subsets or as a one mixed population. The other two subpopulations are B<sub>con</sub> and T<sub>con</sub> cells.</li>
<li>Sort DEs (IgD<sup>+</sup> and IgD<sup>−</sup>) into 1.5 mL microcentrifuge tubes each containing about 700–800 μL of lymphocyte collection medium (See <a href="#materials-and-equipment">materials and equipment</a>).</li>
<li>Sorted B<sub>con</sub> and T<sub>con</sub> cells into 5 mL polypropylene tubes each containing 1 mL of lymphocyte collection medium.
<div class="note">
<span class="note-title">Note:</span> You can sort cells into your own preferred medium that is appropriate with the goal of your experiment.</div>
</li>
<li>
<b>Purity Check.</b> Checking the purity of sorted cells is an extremely crucial step in validating identity of sorted DEs and whether they are contaminated by doublets (which usually fall apart during a sample re-run).
<ol type="i">
<li>To check purity of sorted DE cells, re-run an aliquot of sorted DE cells (100–150 μL) on the sorter using the original gating strategy.
<div class="note">
<span class="note-title">Note:</span> Because of the low number of collected events please do not wash, re-stain or centrifuge samples to avoid cell loss; also note that the flow rate rarely exceeds 1000 event/sec during a re-run.</div>
<div class="note">
<span class="note-title">Note:</span> If necessary, you can dedicate a full sorting session for checking the validity of your gating strategy and use the information gained to inform future sorting strategies.</div>
</li>
<li>Determine the percentage of DE cells in sorted sample by using the same gating strategy.
<div class="note">
<span class="note-title">Note:</span> We generally achieve 70%–90% purity for sorted DE cells and 99% for B and T cell populations.</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="24">
<li>Sorted cells are ready to be used for specific purposes.</li>
</ol>
<h3 id="sec3.3">Part III. Generation of Epstein-Barr Virus (EBV)-immortalized DE lymphoblastoid cell lines (LCLs) and single DE clones</h3>
<p>This section details our method for immortalizing DEs using EBV to generate DE lymphoblastoid cell lines and clones using an established method (<a href="#bib4">Traggiai et al., 2004</a>). All steps should be performed under sterile conditions in a biosafety cabinet.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 weeks</div>
<p>The procedure requires culturing sorted DEs onto fibroblast feeder layers in the presence of infectious EBV as outlined below:</p>
<ol start="25">
<li>Day 1: Preparation of fibroblast feeder layers
<ol type="a">
<li>Thaw 1 vial of irradiated-fibroblasts (ATCC 55-X) using the same method described in the “<a href="#sec1.2">thawing cryopreserved PBMCs</a>” section. Each vial usually contains 1 <b>×</b> 10<sup>6</sup> cells in 1 mL.</li>
<li>Resuspend fibroblasts to a cell density of 2.5 <b>×</b>10<sup>5</sup> / mL using CTM.</li>
<li>Gently pipette your sample up and down with a serological pipette to thoroughly mix the cells.</li>
<li>Plate 200 μL of irradiated-fibroblasts onto wells of a 96-well flat bottom tissue-culture plate (50,000 cells / well).</li>
<li>Incubate the tissue culture plate overnight (18 h) at 37°C in a humidified incubator with 5% CO<sub>2</sub>. It is recommended that you subculture at least once on fibroblast-coated wells to ensure robust growth.</li>
</ol>
</li>
<li>Day 2: Sorting and culturing DEs on fibroblast layers in the presence of EBV-infectious medium
<ol type="a">
<li>Sort DEs, using the method described in <a href="#sec3.2">Part II</a>, using freshly isolated PBMCs (See <a href="#fig2">Figure 2</a>).</li>
<li>Prepare infectious medium by diluting 1 mL EBV solution (B95-8 cells - ATCC VR-1492) to 30% in CTM containing 2.5 μg/mL CpG ODN 2006 (ODN7909)</li>
<li>Suspend sorted DEs in the EBV-infectious medium</li>
<li>Seed DEs onto the fibroblast feeder layers using 50 or 100 DE cells / well.</li>
</ol>
</li>
</ol>
<ol start="27">
<li>Day 3–7: Check cultures for DE growth
<ol type="a">
<li>Check cultures on a daily basis under light microscope for cell growth and any sign of contamination.</li>
<li>After about one week, clusters of dividing cells become visible in culture wells under light microscopy. As time progresses, dividing cells will make clumps that are visible macroscopically and culture medium begins to turn yellow.</li>
<li>Maintain the culture medium by replacing half of the spent medium with fresh infectious medium (CTM containing 2.5 μg/mL CpG ODN, 2006) usually when cell density reaches to 80%–90%.</li>
<li>Typically, culture medium will need replenishment every 5–7 days.</li>
<li>EBV-transformed DEs gradually expand and cultures need to be scaled up by transferring contents into wells of 24-well tissue culture plate and then onto wells of 6-well plates.
<div class="optional">
<span class="optional-title">Optional:</span> you can subculture on fibroblast feeding layers to enhance viability and growth.</div>
</li>
<li>Expand cultures to 75–100 mL using flasks over the next few weeks. These lines are referred to as DE lymphoblastoid cell lines (LCLs). Store generated LCLs at −80°C or liquid nitrogen.
<div class="optional">
<span class="optional-title">Optional:</span> you can subculture on fibroblast feeding layers to enhance viability and growth.</div>
</li>
</ol>
</li>
</ol>
<ol start="28">
<li>Generation of single clones from immortalized DE LCLs using limiting dilution: Use LCL to generate single DE clones, using standard limiting dilution methods (<a href="#bib3">Hamad et al., 1994</a>). Serially dilute and seed 10, 3.3, 1.1, and 0.33 DEs onto fibroblast coated 96-well microplates using 100 μL of CpG 2006 containing CTM as described above. Using clones from 0.33 cell / well ensures a maximum of one cell per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Potential uses of clones include <b>1)</b> Antibody production, and <b>2)</b> Cloning, expression and generation of recombinant BCR and TCR from the same DE cell.</div>
<div class="note">
<span class="note-title">Note:</span> Cryopreservation of DE LCLs and clones: When freezing down cells, centrifuge cells at 500 <b>×</b> <i>g</i> for 5–6 minutes at room temperature. Remove supernatants and re-suspend cell pellets in cold freezing media at 0.5–1<b>×</b> 10<sup>7</sup> cells / mL. Place tubes in liquid nitrogen for long term or −80<sup>°</sup>C for short term storage.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol allows you to detect DEs in PBMC samples using flow cytometry and to simultaneously analyze DE, Tcon, and Bcon in the same sample. The protocol also outlines the steps for sorting DEs along with autologous Tcon and Bcon cells. Furthermore, the protocol describes how to generate immortalized DE lymphoblastoid cell lines and single clones that can be used for production of recombinant BCRs and TCRs and analysis of their TCR and BCR repertoires. Some key steps are shown in <a href="#fig1">Figures 1</a>, <a href="#fig2">2</a>, and <a href="#fig3">3</a>.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>DEs are a rare population of lymphocyte so any flow cytometric analysis should include acquisition of sufficient numbers of PBMCs (500,000–1,000,000 events) in order to ensure detection of sufficient numbers of DEs. The frequency of DEs varies between patients and healthy controls, and they usually comprise between 1%–3% of CD5<sup>+</sup>CD19<sup>+</sup> cells.</p>
<p>When gating DEs, it is crucial to exclude doublets. To do this, gate on single cells using the FSC-Area vs FSC-Height and SSC-Area and SSC-Height plots. In addition, when gating the CD5+ CD19+ population, gate on CD5<sup>low</sup> but not CD5<sup>high</sup> cells and avoid TCRαβ<sup>hi</sup> cells (mostly or all doublets).</p>
<p>When sorted DEs are to be used for repertoire analysis, samples should be examined for both TCRαβ and BCR repertoires if cell number permits (detection of rearranged genes of both receptors confirms identity of sorted cells). Also, it is recommended to simultaneously analyze autologous T<sub>con</sub> and B<sub>con</sub> cells as controls.</p>
<p>When working with DEs from human PBMCs, it is important to remember that there are many variables outside of your control that can affect your analysis. For example, if your patients are taking an immunosuppressive medication, it may affect the way that the DEs respond to stimulation in your experiments and may dampen their effector functions. Our unpublished data show that DEs are highly sensitive to steroids that significantly reduce their frequencies.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Unable to observe distinct B<sub>con</sub>, T<sub>con</sub>, and CD5+ CD19+ populations (step 16 in Part I: <a href="#sec3.1.2">Sequential gating strategy to identify DEs and analyze their phenotypes</a>).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>To troubleshoot this problem, you may try using fresh PBMCs, as opposed to cryopreserved PBMCs, because we find that DEs are prone to undergoing apoptosis if they are cryopreserved. In addition, you may titrate your antibodies to determine the optimal concentration of each antibody. Remember that with more colors used, detecting rare populations becomes more challenging.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Since DE cells are rare events, it is difficult to sort sufficient numbers of cells to work with for subsequent experiments (like antigen receptor repertoire analysis or cell culture related experiments) (step 23c in <a href="#sec3.2">Part II</a>).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>We find that the frequency of DEs is rare and varies with different donors. To maximize the recovery of sorted DE cells, we suggest performing preliminary screening of DEs in samples from different donors using regular FACS analysis (as described above) to identify and select samples with higher frequencies of DEs for sorting experiments.</p>
<div class="note">
<span class="note-title">Note:</span> Generally, if we start with 5 <b>×</b>10<sup>7</sup> PBMCs, we often sort 800 to 1000 DE cells.</div>
<h3 id="sec6.5">Problem 3</h3>
<p>Poor EBV transformation rate (<a href="#sec3.3">Part III</a>).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>To improve the transformation efficiency, we suggest considering the following steps:</p>
<p>Try different numbers (10, 50, 100, 500 cells per well) for immortalization to generate LCLs.</p>
<p>Check the viability of fibroblast cells before preparing feeder layers (especially if cells are frozen). Viability should be more than 95%–98%.</p>
<p>Use fresh PBMCs as the recovery of live sorted DE cells is often better compared to using cryopreserved PBMCs.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Slow or poor growth of immortalized DE clones (<a href="#sec3.3">Part III</a>).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>We observed that adding human plasma to CTM significantly improves growth of EBV immortalized single DE cells after performing the limiting dilution. We would also suggest testing mycoplasma contamination if the cultures have slow or poor growth for a long period of time.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ahmed, R., Omidian, Z., Giwa, A., Cornwell, B., Majety, N., Bell, D.R., Lee, S., Zhang, H., Michels, A., Desiderio, S., et al. (2019). A public BCR present in a unique dual-receptor-expressing lymphocyte from type 1 diabetes patients encodes a potent T cell autoantigen. Cell <i>177</i>, 1583-1599.e16. <a class="external-link" href="https://doi.org/10.1016/j.cell.2019.05.007">https://doi.org/10.1016/j.cell.2019.05.007</a></p>
<p id="bib2">Ahmed, R., Omidian, Z., Giwa, A., Donner, T., Jie, C., and Hamad, A.R.A. (2021). A reply to “TCR+/BCR+ dual-expressing cells and their associated public BCR clonotype are not enriched in type 1 diabetes. Cell <i>184</i>, 840-843. <a class="external-link" href="https://doi.org/10.1016/j.cell.2020.11.036">https://doi.org/10.1016/j.cell.2020.11.036</a></p>
<p id="bib3">Hamad, A.R., Herman, A., Marrack, P., and Kappler, J.W. (1994). Monoclonal antibodies defining functional sites on the toxin superantigen staphylococcal enterotoxin B. J. Exp. Med. <i>180</i>, 615-621. <a class="external-link" href="https://doi.org/10.1084/jem.180.2.615">https://doi.org/10.1084/jem.180.2.615</a></p>
<p id="bib4">Traggiai, E., Becker, S., Subbarao, K., Kolesnikova, L., Uematsu, Y., Gismondo, M.R., Murphy, B.R., Rappuoli, R., and Lanzavecchia, A. (2004). An efficient method to make human monoclonal antibodies from memory B cells: potent neutralization of SARS coronavirus. Nat. Med. <i>10</i>, 871-875. <a class="external-link" href="https://doi.org/10.1038/nm1080">https://doi.org/10.1038/nm1080</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Abdel Rahim Hamad (<a href="mailto:ahamad@jhmi.edu">ahamad@jhmi.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate new unique reagents than published in the original manuscript <a href="#bib1">Ahmed et al. (2019)</a>.</p>
<h3>Acknowledgments</h3>
<p>We thank participants enrolled in this study and their families for blood donation. This work was supported by <a href="https://doi.org/10.13039/100000002">NIH</a>, United States grant R0 AI099027 (A.R.A.H.); the Norman Raab Foundation; <a href="https://doi.org/10.13039/100000888">W.M. Keck foundation</a>. R.A. is partly supported by <a href="https://doi.org/10.13039/100000901">Diabetes Research Connection</a> grant.</p>
<h3>Author contributions</h3>
<p>R.A., K.A., and Z.O. conducted the experiments, interpreted data, and wrote the paper; H.Z. conducted sorting of DEs; N.M. critically read and offered constructive edits to the manuscript; A.R.A.H. supervised the study and edited the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>