<article>
<section>
<h1 id="header">Generation of RNA sequencing libraries for transcriptome analysis of globin-rich tissues of the domestic dog</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Sini Ezer,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">9</a>,<a href="#fn2">10</a>,<a href="#cor1">*</a></sup> Masahito Yoshihara,<sup><a href="#aff3">3</a></sup> Shintaro Katayama,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Carsten Daub,<sup><a href="#aff3">3</a>,<a href="#aff4">4</a></sup> Hannes Lohi,<sup><a href="#aff1">1</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a></sup> Kaarel Krjutskov,<sup><a href="#aff7">7</a>,<a href="#aff8">8</a></sup> and Juha Kere<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Folkhälsan Research Center, 00290 Helsinki, Finland</p>
<p id="aff2"><sup>2</sup>Stem Cells and Metabolism Research Program, University of Helsinki, 00014 Helsinki, Finland</p>
<p id="aff3"><sup>3</sup>Department of Biosciences and Nutrition, Karolinska Institutet, 14183 Huddinge, Sweden</p>
<p id="aff4"><sup>4</sup>Science for Life Laboratory, Stockholm, Sweden</p>
<p id="aff5"><sup>5</sup>Department of Medical and Clinical Genetics, University of Helsinki, 00014 Helsinki, Finland</p>
<p id="aff6"><sup>6</sup>Department of Veterinary Biosciences, University of Helsinki, 00014 Helsinki, Finland</p>
<p id="aff7"><sup>7</sup>Competence Centre of Health Technologies, 50411 Tartu, Estonia</p>
<p id="aff8"><sup>8</sup>Department of Obstetrics and Gynecology, Institute of Clinical Medicine, University of Tartu, Tartu, Estonia</p>
<p id="fn1"><sup>9</sup>Technical contact</p>
<p id="fn2"><sup>10</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:sini.ezer@helsinki.fi">sini.ezer@helsinki.fi</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:juha.kere@ki.se">juha.kere@ki.se</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100995">10.1016/j.xpro.2021.100995</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>We have developed a protocol for barcoded cDNA libraries of 48 samples to study gene expression across tissues in the domestic dog, <i>Canis familiaris,</i> by modifying the Single-Cell Tagged Reverse Transcription (STRT) protocol (<a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref2">Islam et al., 2012</a>, <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref3">2014</a>). The cDNA reads represent mRNA 5′ ends, enabling the study of transcription start sites (TSS). Our modifications include longer UMIs for molecular counting and Globin-Lock® to deplete globin mRNAs that are abundant in blood and blood-rich tissues dominating all reads.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>transcriptome analysis across tissues of domestic dog, <i>Canis familiaris</i>
</li>
<li>RNA-seq library preparation for 48 tissue samples in parallel</li>
<li>depletion of abundant globin mRNAs from blood and blood-rich tissues</li>
<li>study of transcription start sites with cDNA reads from 5′end</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1238-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Background and motivation</h3>
<p>In the Single-Cell Tagged Reverse Transcription method (STRT-Seq), reads are obtained from 5′ end, which enables study of the transcription start sites. We have modified the method to analyze the RNA extracted from blood, cells, and tissues. Only a small amount, 20–40 ng of template RNA, is required, and thus the method is suitable for studies with limited availability of starting material. Our modifications included longer UMIs, 8 bp instead of 6 bp, which means a sixteen-fold increase in the distinct labels. We have redesigned the primers and streamlined the protocol so that it is possible to complete the library preparation in one day. Second day is needed for quality control. There are several pause points for flexibility.</p>
<p>High content of globin mRNA (gmRNA), which can constitute 50% or even more of the mRNA content in red blood cells, can complicate transcriptome analysis of blood and blood-rich tissues. We attempted to study transcript profile across tissues of domesticated dog <i>Canis familiaris</i>. However, after autopsy high content of blood remains within many tissues, such as in bone marrow, liver and spleen; for example, 8%–60% of RNA-seq reads in dog spleen in our preliminary trial were from hemoglobin alpha or beta genes (<i>data not shown</i>). We have previously described a fast and effective method, GlobinLock (GL), where gmRNA-specific oligonucleotides are added into initial RNA denaturation step (<a href="#bib6">Krjutškov et al., 2016</a>). Here we describe improved efficiency of the method by including Locked Nucleic Acids (LNA) in the species-specific oligo design for the dog. The protocol is universal to other species by switching species-specific Globin-Lock® blocking oligonucleotides.</p>
<p>One limiting parameter for use of RNA-Seq has been its high cost, which can be significantly reduced by multiplexing the samples. The size of the library multiplex level in the protocol is 48. Typically, two or more libraries can be processed simultaneously. To prepare multiple libraries per study, the sample layout should be considered carefully as guided by <a href="#bib4">Katayama et al. (2019)</a>.</p>
<h3 id="sec1.2">Animal samples</h3>
<p>The dog samples used in the protocol were from pet dogs euthanized by the owner’s decision. The Ethical committee of the Faculty of Veterinary Sciences evaluated and accepted the animal proceedings under the ethical permissions ESAVI/7482/04.10.07/2015 and ESAVI/343/04.10.07/2016.</p>
<h3 id="sec1.3">Workspace setup</h3>
<p>To avoid contamination, you should have two separate areas for pre- and post-PCR work with designated consumables and small equipment (mini-centrifuge, vortex and pipettors) in each area. We recommend using two thermocyclers, one in the pre-PCR area for reverse transcription reaction only, and another one for amplifications in post-PCR area.</p>
<p>Refer to “<a href="#materials-and-equipment">materials and equipment</a>” for the list of buffers and solutions to be prepared before starting the protocol.</p>
<h3 id="sec1.4">Sample RNA preparations</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol>
<li>Analyze the purity and the integrity of the template RNA with TapeStation, Bioanalyzer or Qubit IQ assay. Only samples with RIN&gt;7.5 should be used for optimal data.</li>
<li>Dilute the RNA sample to 20 ng/μL and use 2 μL per well (total 40 ng).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Keep your RNA sample tubes on ice at all times while making the dilutions.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>RiboLock Rnase inhibitor, 40 U/μL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EO0382</td>
</tr>
<tr>
<td>Phusion DNA polymerase, 2 U/μL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#F-530S</td>
</tr>
<tr>
<td>NEBNext End Repair Enzyme</td>
<td>New England Biolabs</td>
<td>Cat#E6050S</td>
</tr>
<tr>
<td>E.coli DNA ligase</td>
<td>New England Biolabs</td>
<td>Cat#M0205S</td>
</tr>
<tr>
<td>NEBNext dA-Tailing Reaction Buffer, 10×</td>
<td>New England Biolabs</td>
<td>Cat#B6059S</td>
</tr>
<tr>
<td>Klenow exo, 5 U/μL</td>
<td>New England Biolabs</td>
<td>Cat#M0212S</td>
</tr>
<tr>
<td>T4 DNA ligase, 5 U/μL</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15224-041</td>
</tr>
<tr>
<td>SuperScript II Reverse Transcriptase, 200U/μL</td>
<td>Invitrogen</td>
<td>Cat#18064-014</td>
</tr>
<tr>
<td>Betaine 5M</td>
<td>Sigma-Aldrich</td>
<td>Cat#B0300-1VL</td>
</tr>
<tr>
<td>DTT No-Weigh Format</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#A39255</td>
</tr>
<tr>
<td>Tris 1 M, pH 8</td>
<td>Ambion</td>
<td>Cat#AM9855G</td>
</tr>
<tr>
<td>Ultra Pure Tris 1 M, pH 7.5</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15567-027</td>
</tr>
<tr>
<td>2M KCL, RNAase free</td>
<td>Ambion</td>
<td>Cat#AM9640G</td>
</tr>
<tr>
<td>1M MgCl<sub>2,</sub> RNAase free</td>
<td>Ambion</td>
<td>Cat#AM9530G</td>
</tr>
<tr>
<td>5M NaCl, RNAase free</td>
<td>Ambion</td>
<td>Cat#AM9760G</td>
</tr>
<tr>
<td>EDTA 0.5 M, RNAase free</td>
<td>Ambion</td>
<td>Cat#AM9260G</td>
</tr>
<tr>
<td>NucleoSpin Gel and PCR Clean-up</td>
<td>MACHEREY-NAGEL</td>
<td>Cat#740609.250</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma-Aldrich</td>
<td>Cat#G5516-100ML</td>
</tr>
<tr>
<td>Triton X-100 10%</td>
<td>Sigma-Aldrich</td>
<td>Cat#93443-100ML</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Fisher Scientific</td>
<td>Cat#BP337-100</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td>Ambion</td>
<td>Cat#AM9937</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>ERCC RNA Spike-in Mix</td>
<td>Ambion</td>
<td>Cat#4456740</td>
</tr>
<tr>
<td>KAPA Library Quantification Kit</td>
<td>Roche</td>
<td>Cat#KK4824</td>
</tr>
<tr>
<td>NucleoSpin Gel and PCR Clean-up</td>
<td>Macherey-Nagel</td>
<td>Cat#740609.250</td>
</tr>
<tr>
<td>Dynabeads MyOne C1 Streptavidin</td>
<td>Invitrogen</td>
<td>Cat#65001</td>
</tr>
<tr>
<td>Agencourt AMPure XP</td>
<td>Beckman Coulter</td>
<td>Cat#A63881</td>
</tr>
<tr>
<td>NextSeq 500/550 High Output v2 kit</td>
<td>Illumina</td>
<td>Cat#FC-404-2005</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Primers for library preparation, see <a href="#tbl1">Table 1</a>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>Sequences for barcode primers, see <a href="#tbl2">Table 2</a>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>STRT2 pipeline b3e589c</td>
<td>This paper</td>
<td><a href="https://github.com/my0916/STRT2">https://github.com/my0916/STRT2</a></td>
</tr>
<tr>
<td>Picard v2.23.4</td>
<td><a href="https://github.com/broadinstitute/picard">https://github.com/broadinstitute/picard</a></td>
<td><a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a></td>
</tr>
<tr>
<td>HISAT2 v2.2.1</td>
<td><a href="#bib5">Kim et al. (2019)</a></td>
<td><a href="https://daehwankimlab.github.io/hisat2/">https://daehwankimlab.github.io/hisat2/</a></td>
</tr>
<tr>
<td>SAMtools v1.10</td>
<td><a href="#bib7">Li et al. (2009)</a></td>
<td><a href="http://www.htslib.org/">http://www.htslib.org/</a></td>
</tr>
<tr>
<td>BEDtools v2.29.2</td>
<td><a href="#bib10">Quinlan and Hall (2010)</a></td>
<td><a href="http://bedtools.readthedocs.io/">http://bedtools.readthedocs.io/</a></td>
</tr>
<tr>
<td>R v4.0.0</td>
<td><a href="#bib11">R core Team (2020)</a></td>
<td><a href="https://www.r-project.org/">https://www.r-project.org/</a></td>
</tr>
<tr>
<td>R Package: ggplot2 v 3.3.3</td>
<td><a href="#bib13">Wickham and Grolemund (2016)</a></td>
<td><a href="https://ggplot2.tidyverse.org/">https://ggplot2.tidyverse.org/</a></td>
</tr>
<tr>
<td>featureCounts v2.0.0</td>
<td><a href="#bib8">Liao et al. (2014)</a></td>
<td><a href="http://subread.sourceforge.net/">http://subread.sourceforge.net/</a></td>
</tr>
<tr>
<td>StringTie v2.1.4</td>
<td><a href="#bib9">Pertea et al. (2015)</a></td>
<td><a href="https://ccb.jhu.edu/software/stringtie/">https://ccb.jhu.edu/software/stringtie/</a></td>
</tr>
<tr>
<td>FastQC v0.11.8</td>
<td><a href="https://github.com/s-andrews/FastQC">https://github.com/s-andrews/FastQC</a></td>
<td><a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a></td>
</tr>
<tr>
<td>MultiQC v1.9</td>
<td><a href="#bib1">Ewels et al. (2016)</a></td>
<td><a href="https://multiqc.info/">https://multiqc.info/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Thermocycler with a heated lid</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Focused ultrasonicator for DNA fragmentation</td>
<td>Covaris or compatible</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic stand for 1.5mLtubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Thermal Mixer (Thermal Shaker) for 1.5 mL tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Bench Top Centrifuge for 1.5/2 mL tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>VersiPlates, 96-well Low Profile PCR Strip Tube Plate</td>
<td>Thermo Scientific</td>
<td>Cat#AB-1800</td>
</tr>
<tr>
<td>VersiCap Mat, Domed Cap Stripes</td>
<td>Thermo Scientific</td>
<td>Cat#AB-1810</td>
</tr>
<tr>
<td>DNA LoBind Tubes 1.5 mL</td>
<td>Eppendorf</td>
<td>Cat#30108051</td>
</tr>
<tr>
<td>8-channel Multipipette</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>TapeStation (for quality control)</td>
<td>Agilent</td>
<td>N/A</td>
</tr>
<tr>
<td>D1000 ScreenTape</td>
<td>Agilent</td>
<td>Cat#5067- 5582</td>
</tr>
<tr>
<td>D1000 Reagents</td>
<td>Agilent</td>
<td>Cat#5067- 5583</td>
</tr>
</tbody>
</table>
<table id="tbl1">
<caption>Table 1. Primers for library preparation</caption>
<thead>
<tr>
<th>Primer</th>
<th>Seguence</th>
<th>NOTE</th>
</tr>
</thead>
<tbody>
<tr>
<td>GLα1 (canine globin alpha 1)</td>
<td>T+TTTTG+CAGCCCA+CTCAGACTTT+ATTCAAA+CATC-3Phos</td>
<td>+N = LNA nucleotide</td>
</tr>
<tr>
<td>GLα2 (canine globin alpha 2)</td>
<td>T+TTTTGGGC+GCAGGAGGC+GGCCCA+G-3Phos</td>
<td>+N = LNA nucleotide</td>
</tr>
<tr>
<td>GLβ (canine globin beta)</td>
<td>T+TTTT+GAGCTGA+AGGTTCT+TTATTAGG+CAGAAG-3Phos</td>
<td>+N = LNA nucleotide</td>
</tr>
<tr>
<td>TSO 8 UMI Biotin</td>
<td>Biotin-ACGACGCTCTTCCGATCTNNNNNNNNrGrGrG</td>
<td> </td>
</tr>
<tr>
<td>T30-VN GL</td>
<td>TTAAGCAGTGGTATCAACGCAGAGTCGACTTTTTTTTTTTTTTTTTTTTTTTTTTTTTVN</td>
<td> </td>
</tr>
<tr>
<td>universal primer PCR1c</td>
<td>AAGCAGTGGTATCAACGCAGAGT</td>
<td> </td>
</tr>
<tr>
<td>adapter1</td>
<td>CAAGCAGAAGACGGCATACGAGATT</td>
<td> </td>
</tr>
<tr>
<td>adapter2</td>
<td>Phos-ATCTCGTATGCCGTCTTCTGCTTG</td>
<td> </td>
</tr>
<tr>
<td>PCR-2cF</td>
<td>CAAGCAGAAGACGGCATACGAGATT</td>
<td> </td>
</tr>
<tr>
<td>PCR-2cR</td>
<td>AATGATACGGCGACCACCGAGATCTACACCACGTCTGAACTCCAGTCAC</td>
<td> </td>
</tr>
<tr>
<td>STRT2seq</td>
<td>CGAGATCTACACCACGTCTGAACTCCAGTCAC</td>
<td> </td>
</tr>
</tbody>
</table>
<table id="tbl2">
<caption>Table 2. Barcode oligos</caption>
<thead>
<tr>
<th>Oligo name</th>
<th>Barcode</th>
<th>5′ modification</th>
<th>Sequence</th>
</tr>
</thead>
<tbody>
<tr>
<td>Barcode 1</td>
<td>AGTTCA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>AGTTCA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 2</td>
<td>CAGATC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CAGATC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 3</td>
<td>CGATGT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CGATGT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 4</td>
<td>TTAGGC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TTAGGC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 5</td>
<td>ATCACG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ATCACG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 6</td>
<td>TGACCA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TGACCA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 7</td>
<td>ACATGT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ACATGT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 8</td>
<td>GCCAAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GCCAAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 9</td>
<td>CATGAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CATGAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 10</td>
<td>TATTGT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TATTGT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 11</td>
<td>AAAGTT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>AAAGTT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 12</td>
<td>TCTACC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TCTACC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 13</td>
<td>TTGGAC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TTGGAC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 14</td>
<td>CAAAGT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CAAAGT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 15</td>
<td>ATGCTT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ATGCTT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 16</td>
<td>GTGGTA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GTGGTA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 17</td>
<td>GCAGGA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GCAGGA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 18</td>
<td>GGACAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GGACAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 19</td>
<td>ATTCCA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ATTCCA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 20</td>
<td>AGTTAC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>AGTTAC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 21</td>
<td>TGAAGC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TGAAGC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 22</td>
<td>TAGCAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TAGCAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 23</td>
<td>GTTGCC</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GTTGCC</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 24</td>
<td>ACGTTG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ACGTTG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 25</td>
<td>TAAGGG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TAAGGG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 26</td>
<td>GCCTAG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GCCTAG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 27</td>
<td>CTCGCA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CTCGCA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 28</td>
<td>CCGAAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CCGAAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 29</td>
<td>GGGTTT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GGGTTT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 30</td>
<td>GTAATG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GTAATG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 31</td>
<td>TAGAGA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TAGAGA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 32</td>
<td>AGATGG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>AGATGG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 33</td>
<td>ATCTCT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ATCTCT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 34</td>
<td>CGTATT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CGTATT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 35</td>
<td>ACTTAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ACTTAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 36</td>
<td>GATCTT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GATCTT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 37</td>
<td>GCTGTG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GCTGTG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 38</td>
<td>ACAATA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ACAATA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 39</td>
<td>TTCATA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TTCATA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 40</td>
<td>TTAACT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TTAACT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 41</td>
<td>ATACAG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>ATACAG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 42</td>
<td>AGTCGT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>AGTCGT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 43</td>
<td>CTGTGT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CTGTGT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 44</td>
<td>CCTAGA</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CCTAGA</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 45</td>
<td>CCATCT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CCATCT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 46</td>
<td>GACACT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>GACACT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 47</td>
<td>TGGATG</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>TGGATG</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
<tr>
<td>Barcode 48</td>
<td>CTCCAT</td>
<td>Biotin</td>
<td>CACGTCTGAACTCCAGTCAC<b>CTCCAT</b>ACACTCTTTCCCTACACGACGCTCTTCCGATCT</td>
</tr>
</tbody>
</table>
<p class="table-legend">The barcode sequence within each primer is shown in bold.</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="critical">
<span class="critical-title">Critical:</span> Prepare all the buffers and solutions in the pre-PCR area, preferably in a PCR workstation, and use nuclease-free water.</div>
<h3 id="sec2.1">ERCC spike-in control dilution (for step 6)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<p>ERCC RNA Spike-in Mix requires dilution. To avoid repeated thaw and freeze cycles, make aliquots as follows.</p>
<ul>
<li>Create ten 1:10 dilutions and then 1:100 dilutions as working stocks.</li>
<li>Make aliquots of 1.3 μL from the 1:100 solution and store at −70°C (can be stored up to 12 months).</li>
<li>Thaw an aliquot on ice and take 1 μL for each library master mix.</li>
</ul>
<h3 id="sec2.2">Barcode primers (for step 10)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ul>
<li>Dissolve each Barcode primer (<a href="#tbl2">Table 2</a>) in nuclease-free water to make 100 μM stocks (store at −20°C. Stable up to 12 months).</li>
<li>Prepare 48 well plate with 10 μM barcoded primer solutions in numerical order. Store at −20°C (stable up to 12 months).</li>
<li>Thaw for 2–3 h at +4°C before use.</li>
</ul>
<h3 id="sec2.3">Preparation of double-stranded adapter cassette (for step 44)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<table id="undtbl1">
<caption>Adapter cassette 20 μM</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Adapter-1 ssDNA (100 μM)</td>
<td>20 μM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Adapter-2 ssDNA (100 μM)</td>
<td>20 μM</td>
<td>20 μL</td>
</tr>
<tr>
<td>KCl (2M)</td>
<td>50 mM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td>n/a</td>
<td>57.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td>
<b>100</b> μL</td>
</tr>
</tbody>
</table>
<ul>
<li>Mix the reagents in a 1.5 mL tube.</li>
<li>Heat 5 min at 72°C in a heat block.</li>
<li>Turn the heat block off and let the heat block cool slowly to 20°C–22°C.</li>
<li>Store the solution at −20°C (stable up to 12 months).</li>
<li>Thaw on ice before each use.</li>
</ul>
<div class="note">
<span class="note-title">Note:</span> We recommend using a heat block rather than a water-bath, to allow gradual cooling and annealing to take around 1 h.</div>
<table id="undtbl2">
<caption>Buffer EB</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH 8), (1 M)</td>
<td>5 mM</td>
<td>0.25 mL</td>
</tr>
<tr>
<td>Tween-20 (10%)</td>
<td>0.02%</td>
<td>0.1 mL</td>
</tr>
<tr>
<td>Nuclease-Free water</td>
<td>n/a</td>
<td>49.65 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–22°C up to 12 months.</p>
<table id="undtbl3">
<caption>Buffer 2×BP</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl (pH 7.5), (1 M)</td>
<td>10 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>10 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>2 M</td>
<td>20 mL</td>
</tr>
<tr>
<td>Tween-20 (10%)</td>
<td>0.01%</td>
<td>0.05 mL</td>
</tr>
<tr>
<td>Nuclease-Free water</td>
<td>n/a</td>
<td>28.45 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 20°C–22°C up to 12 months.</p>
<div class="note">
<span class="note-title">Note:</span> All the reagents used for buffers A-GL, B-GL, C and buffer C concentrate should be nuclease free.</div>
<table id="undtbl4">
<caption>Buffer A-GL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration in RT reaction (6 μL)</th>
<th>Amount for 1 rxn</th>
<th>Master mix for 1 library</th>
</tr>
<tr>
<th>Reagent</th>
<th>Final concentration in GL (3 μL)</th>
<th>Amount for 1 rxn</th>
<th>Master mix for 1 library</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>1.03 μL</td>
<td>56.65 μL</td>
</tr>
<tr>
<td>dNTP mix (10 mM each)</td>
<td>1 mM</td>
<td>0.6 μL</td>
<td>33 μL</td>
</tr>
<tr>
<td>KCl (2 M)</td>
<td>75 mM</td>
<td>0.22 μL</td>
<td>12.1 μL</td>
</tr>
<tr>
<td>Triton X-100 (10%)</td>
<td>0.05%</td>
<td>0.03 μL</td>
<td>1.65 μL</td>
</tr>
<tr>
<td>Tris-HCl (pH 7.5), (1 M)</td>
<td>5 mM</td>
<td>0.03 μL</td>
<td>1.65 μL</td>
</tr>
<tr>
<td>primer GLα1 (1 mM)</td>
<td>10 µM</td>
<td>0.03 μL</td>
<td>1.65 μL</td>
</tr>
<tr>
<td>primer GLα2 (1 mM)</td>
<td>10 µM</td>
<td>0.03 μL</td>
<td>1.65 μL</td>
</tr>
<tr>
<td>primer GLβ (1 mM)</td>
<td>10 µM</td>
<td>0.03 μL</td>
<td>1.65 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>2.0 μL</b></td>
<td><b>110.0 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C up to 6 months.</p>
<table id="undtbl5">
<caption>Buffer B-GL</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration in RT (6 μL)</th>
<th>Amount for 1 rxn</th>
<th>Master mix for 1 library</th>
</tr>
</thead>
<tbody>
<tr>
<td>Betaine (5 M)</td>
<td>1 M</td>
<td>1.113 μL</td>
<td>66.78 μL</td>
</tr>
<tr>
<td>Tris-HCl (pH 8.5), (1 M)</td>
<td>20 mM</td>
<td>0.12 μL</td>
<td>7.2 μL</td>
</tr>
<tr>
<td>primer T30-VN GL (100 μM)</td>
<td>0.2 μM</td>
<td>0.012 μL</td>
<td>0.72 μL</td>
</tr>
<tr>
<td>DTT (500 mM)</td>
<td>10 mM</td>
<td>0.12 μL</td>
<td>7.2 μL</td>
</tr>
<tr>
<td>primer TSO-8UMI (100 μM)</td>
<td>1.5 μM</td>
<td>0.09 μL</td>
<td>5.4 μL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>7.5 mM</td>
<td>0.045 μL</td>
<td>2.7 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1.5 μL</b></td>
<td><b>90.0 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C up to 6 months.</p>
<table id="undtbl6">
<caption>Buffer C</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration in RT (6 μL)</th>
<th>Amount for 1 rxn</th>
<th>Master mix for 1 library</th>
</tr>
</thead>
<tbody>
<tr>
<td>Glycerol</td>
<td> </td>
<td>0.2724 μL</td>
<td>13.1 μL</td>
</tr>
<tr>
<td>SuperScript II (200 U/μL)</td>
<td>3.3 U/μL</td>
<td>0.1 μL</td>
<td>5.9 μL</td>
</tr>
<tr>
<td>RiboLOCK (40 U/μL)</td>
<td>0.7 U/μL</td>
<td>0.1 μL</td>
<td>6.3 μL</td>
</tr>
<tr>
<td>DTT (500 mM)</td>
<td>5 mM</td>
<td>0.06 μL</td>
<td>3.6 μL</td>
</tr>
<tr>
<td>C buffer concentrate (see below)</td>
<td>n/a</td>
<td>0.0176 μL</td>
<td>1.1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>0.5 μL</b></td>
<td><b>30.0 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C up to 6 months.</p>
<div class="note">
<span class="note-title">Note:</span> Buffer C is very viscous, and thus larger excess volume is calculated for the master mixes of Buffer B and C. To pipette viscous solutions accurately pause longer during aspiration, as well as dispensing.</div>
<table id="undtbl7">
<caption>C Buffer concentrate stock</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl (5 M)</td>
<td>100 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Tris-HCl (pH 7.5), (1 M)</td>
<td>5 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>Triton X-100 (10%)</td>
<td>0.10%</td>
<td>50 μL</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>0.1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>176 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C up to 12 months.</p>
<div class="note">
<span class="note-title">Note:</span> Final concentration of C buffer concentrate is calculated for the RT-reaction.</div>
<table id="undtbl8">
<caption>PCR-2 primer mix (for step 50)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>primer PCR-2cF (100 μM)</td>
<td>10 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>primer PCR-2cR (100 μM)</td>
<td>10 μM</td>
<td>10 μL</td>
</tr>
<tr>
<td>Nuclease free water</td>
<td>n/a</td>
<td>80 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C up to 12 months.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p><a href="#fig1">Figure 1</a> shows the schematic overview of the method.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1238-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Overview of the library preparation with the required primers</div>
<p>Template RNA is mixed with ERCC Spike-in RNA, which is added equally to each sample for normalization, and with globin-LNA primers, which hybridize to the sequence adjacent to poly-A<sup>+</sup> tail of globin mRNA thus blocking its’ transcription. First-strand cDNA synthesis occurs using an oligo(dT) primer with the addition of 3-6 cytosines. Oligo TSO-8UMI promotes template switching, which introduces UMIs into the cDNA. The cDNA is amplified, and the well specific barcodes are introduced at the 5′end. The samples are then pooled and ligated to an adapter cassette. The 5′end is further amplified for sequencing. QC1 to QC3 show the steps where a sample is taken for quality control.</p>
</figcaption>
</figure>
<h3 id="sec3.1">Globin-Lock® and reverse transcription with template switching</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol>
<li>Before starting, get your cycler ready for the STRT-GL reaction at 80°C.</li>
<li>Mix <b>2 μL RNA</b> 20 ng/μL (47 samples) and <b>2 μL Buffer A-GL Master mix</b> on each sample well on VersiPlate.</li>
<li>For negative control, mix <b>2 μL H</b><sub><b>2</b></sub><b>O</b> with <b>2 μL buffer A-GL Master mix.</b>
</li>
<li>Seal the plate and spin drops down (1 min at 1500 × <i>g</i>).</li>
<li>Proceed to the RNA denaturation step and leave the plate at 40°C on hold, until you are ready with step 6.</li>
<li>Prepare RT mixture of <b>90 μL Buffer B-GL Master mix</b> and <b>30 μL Buffer C Master mix</b> per one library (48 wells) and add <b>1 μL Spike-In</b> (1:100 dilution) per one library.</li>
<li>Add <b>2 μL</b> RT mixture to each well on 40°C. Total volume in each well is now 6 μL. Mix by pipetting up and down several times.</li>
<li>Continue with <b>STRT-GL</b> reaction.</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th colspan="3">STRT-GL reaction conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>Warm-up/hold</td>
<td>80°C</td>
<td>hold</td>
</tr>
<tr>
<td>RNA Denaturation</td>
<td>80°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Globin-Lock Hybridization</td>
<td>65°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Loading</td>
<td>40°C</td>
<td>hold</td>
</tr>
<tr>
<td>RT Reaction</td>
<td>40°C</td>
<td>60 min</td>
</tr>
<tr>
<td>Inactivation</td>
<td>85°C</td>
<td>5 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>forever</td>
</tr>
</tbody>
</table>
<ol start="9">
<li>Spin down (1 min at 1500 × <i>g</i>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To avoid evaporation of the small reaction volume, we recommend using low-profile VersiPlates. However, you need to check the compatibility of the plates with your thermocycler. We are using Bio-Rad T100 Thermal Cycler with PCR plate Pressure Pad to ensure tight sealing of the lid.</div>
<h3 id="sec3.2">Introduction of barcodes and full transcript amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="10">
<li>Keep the reaction plate on ice and add <b>2 μL</b> well-specific barcoded primer (10 μM) to each sample.</li>
<li>Make the following PCR master mix.</li>
</ol>
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Amount for 1 rxn</th>
<th>Master mix for one library</th>
</tr>
</thead>
<tbody>
<tr>
<td>Nuclease-Free Water</td>
<td>7.4 μL</td>
<td>407 μL</td>
</tr>
<tr>
<td>Phusion GC buffer, 5×</td>
<td>4 μL</td>
<td>220 μL</td>
</tr>
<tr>
<td>DMSO (100%)</td>
<td>0.6 μL</td>
<td>33 μL</td>
</tr>
<tr>
<td>dNTP mix (each 10 mM)</td>
<td>0.4 μL</td>
<td>22 μL</td>
</tr>
<tr>
<td>Phusion DNA polymerase (2 U/μL)</td>
<td>0.4 μL</td>
<td>22 μL</td>
</tr>
<tr>
<td>Universal primer PCR1c (100 μM)</td>
<td>0.2 μL</td>
<td>11 μL</td>
</tr>
<tr>
<td><b>Total volume</b></td>
<td><b>13 μL</b></td>
<td><b>715 μL</b></td>
</tr>
</tbody>
</table>
<ol start="12">
<li>Pipette <b>13 μL</b> PCR Master mix to each well in the reaction plate and mix by pipetting up and down several times.</li>
<li>Seal the the plate and move to post-PCR area.</li>
<li>Start the PCR program <b>STRT-P1.</b>
</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th colspan="4">STRT-P1 PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">10 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>62°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>5 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>forever</td>
<td> </td>
</tr>
</tbody>
</table>
<h3 id="sec3.3">Sample pooling and purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="15">
<li>Spin the plate (1 min at 1500 × <i>g</i>). Pool the samples from all the wells into one 1.5 mL LoBind tube.</li>
<li>Purify the content with PCR Clean-up column (NucleoSpin Gel and PCR Clean-up/Macherey-Nagel) according to manufacturer’s instructions. <a href="https://www.mn-net.com/media/pdf/02/1a/74/Instruction-NucleoSpin-Gel-and-PCR-Clean-up.pdf">https://www.mn-net.com/media/pdf/02/1a/74/Instruction-NucleoSpin-Gel-and-PCR-Clean-up.pdf</a>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> You can also use equivalent kit for clean-up from other suppliers; however, we have not tested the impact of alternatives on protocol performance.</div>
<ol start="17">
<li>Elute with <b>32 μL</b> elution buffer from the kit into a clean 1.5 mL LoBind tube.</li>
<li>Take <b>2 μL</b> aside for TapeStation (HS D1000) for first quality control <b>(sample QC-1).</b>
<ol type="a">
<li>
<b>30 μL</b> is left for the next step.</li>
<li>Keep the <b>QC-1</b> sample on ice or at −20°C until you are ready with your <b>QC-2</b> sample (step 26) and analyze them simultaneously.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The sample can be stored at −20°C up to 1 month.</div>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Fragmentation of full-length cDNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="19">
<li>Mix AMPure XP beads well. (You should have homogenous brown solution and no bead precipitates. It is recommended to take an aliquot of 200–500 μL into a 2 mL tube and mix by vortexing before each use.)</li>
<li>Add <b>24 μL</b> AMPure XP beads to the <b>30 μL</b> cDNA tube (from step 18). Mix well by vortexing and keep 10 min at 20°C–22°C.</li>
<li>Place the tube on the magnetic stand for at least 3 min or until the supernatant is clear and discard the supernatant.</li>
<li>Remove the tube from the magnetic rack and re-suspend the beads in <b>100 μL</b> EB. Wait for at least 1 min.</li>
<li>Capture the beads on a magnet and transfer the supernatant (<b>100 μL</b>, containing the cDNA) into a clean 1.5 mL LoBind tube.</li>
<li>Shear the amplified cDNA with Covaris Focused ultrasonicator (or compatible), using 130 μL microtubes. Set the parameters to produce 300 bp fragments according to the manufacturer’s protocol. <a href="https://www.covaris.com/protocols/?sku=500295">https://www.covaris.com/protocols/?sku=500295</a>
</li>
<li>Purify and concentrate the fragmented product with PCR Clean-up column (NucleoSpin Gel and PCR Clean-up). Elute with <b>32 μL</b> elution buffer from the kit into a clean 1.5 mL LoBind tube.</li>
<li>Take <b>2 μL</b> aside for TapeStation for the second quality control step (<b>sample QC-2</b>)<b>. 30 μL</b> is left for the next step.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The sample can be stored at −20°C for up to 1 month.</div>
<ol start="27">
<li>Analyze the <b>QC-1</b> (from step 18) and <b>QC-2</b> samples (from step 26) with Tape Station before proceeding with the protocol. See <a href="#fig2">Figure 2</a> for expected analysis results. Typical concentrations are:
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1238-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Library quality control with TapeStation</div>
<p>In QC1, after amplification of the full-sized cDNA, you should see variation of transcript sizes (from about 200 bp to &gt;2000 bp). In QC2, the cDNA has been fragmented to size around 300 bp. In QC3 after further amplification, size selection and purification, sharper peak around 300 bp should be visible.</p>
</figcaption>
</figure>
</li>
</ol>
<p><b>QC-1</b>: 5–50 ng/μL</p>
<p><b>QC-2</b>: 2–20 ng/μL</p>
<p>If the analysis results are as expected, proceed to the next step. <a href="#sec7.1">Troubleshooting 1</a> and <a href="#sec7.3">2</a>.</p>
<h3 id="sec3.5">End-repair, adapter ligation and 5′end amplification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<div class="critical">
<span class="critical-title">Critical:</span> In the following enzymatic steps, incubations are performed with cDNA bound to beads; gentle mixing of the reaction tube is required to avoid sinking of the beads to the bottom of the tube. We have used Eppendorf ThermoMixer C, but equivalent thermal mixer from other suppliers can also be used.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you do not have a thermal mixer, the mixing can be done manually by pipetting at 10 min intervals to disperse the beads.</div>
<ol start="28">
<li>Wash Dynabeads MyOne C1 Streptavidin beads as follows:
<ol type="a">
<li>Take <b>12 μL</b> MyOne C1 Streptavidin beads in a 1.5 mL LoBind tube.</li>
<li>Capture the beads using a magnet, and withdraw the supernatant from the tube.</li>
<li>Remove the tube from the magnetic rack and re-suspend the beads in <b>200 μL</b> 2×BP buffer. Capture the beads and remove the supernatant.</li>
<li>Repeat step 28c. once more.</li>
<li>Re-suspend the beads in <b>30 μL</b> 2×BP buffer.</li>
</ol>
</li>
<li>Mix the washed beads (<b>30 μL</b>) and the fragmentized cDNA (<b>30 μL</b>, from step 26) in a 1.5 mL LoBind tube. Final concentration of the binding buffer will be now 1xBP.</li>
<li>Incubate 20 min on 500 rpm in thermal mixer at 20°C–22°C<b>.</b>
</li>
<li>Place the tube on a magnetic rack until the beads clearly separate from the liquid (1–3 min). Discard the supernatant and remove the tube from the magnetic rack.</li>
<li>Add <b>200 μL</b> of EB buffer. Re-suspend the beads thoroughly by pipetting, and then place the tube again on a magnetic stand (1–3 min).</li>
<li>Discard the supernatant and re-suspend the beads in <b>16 μL</b> buffer EB.</li>
<li>Add the following reagents:</li>
</ol>
<table id="undtbl12">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Beads containing cDNA in EB (from step 33)</td>
<td>16 μL</td>
</tr>
<tr>
<td>NEBNext End Repair Buffer, 10×</td>
<td>2 μL</td>
</tr>
<tr>
<td>NEBNext End Repair Enzyme Mix</td>
<td>1 μL</td>
</tr>
<tr>
<td>
<i>E.coli</i> ligase</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<ol start="35">
<li>Incubate 20 min in a thermal shaker on 500 rpm at 20°C–22°C<b>.</b>
</li>
<li>Place the tube on a magnetic rack for 1–3 min and withdraw the supernatant from the tube. Remove the tube from the magnetic rack and re-suspend the beads in <b>200 μL</b> EB.</li>
<li>Repeat step 36 once more.</li>
<li>Place the tube on a magnetic rack for 1–3 min and withdraw the supernatant.</li>
<li>Re-suspend in <b>17 μL</b> EB. Add the following reagents:</li>
</ol>
<table id="undtbl13">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Beads containing cDNA in EB (from step 39)</td>
<td>17 μL</td>
</tr>
<tr>
<td>NEBNext dA tailing buffer, 10×</td>
<td>2 μL</td>
</tr>
<tr>
<td>Klenow exo- (5 U/μL)</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<ol start="40">
<li>Incubate 20 min in a thermal shaker on 500 rpm at 37°C.</li>
<li>Place the tube on a magnetic rack for 1–3 min and withdraw the supernatant from the tube. Remove the tube from the magnetic rack and re-suspend the beads in <b>200 μL</b> EB.</li>
<li>Repeat step 40 once more.</li>
<li>Place the tube on a magnetic rack for 1–3 min and withdraw the supernatant.</li>
<li>Re-suspend in <b>18 μL</b> EB. Add the following reagents:</li>
</ol>
<table id="undtbl14">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Beads containing cDNA in EB (from step 44)</td>
<td>18 μL</td>
</tr>
<tr>
<td>T4 DNA Ligase buffer (10×)</td>
<td>3 μL</td>
</tr>
<tr>
<td>Adapter cassette (20 μM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>PEG-4000 (50%)</td>
<td>6 μL</td>
</tr>
<tr>
<td>T4 DNA ligase (5 U/μL)</td>
<td>2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>30 μL</b></td>
</tr>
</tbody>
</table>
<ol start="45">
<li>Mix gently by pipetting. Do not vortex. Incubate 60 min at 20°C–22°C and mix every 15 min during the incubation by pipetting gently up and down.</li>
<li>Place the tube on a magnetic rack for 1–3 min and withdraw the supernatant from the tube. Remove the tube from the magnetic rack and re-suspend the beads in <b>200 μL</b> EB.</li>
<li>Repeat step 46 twice more.</li>
<li>Place the tube on a magnetic rack for 1–3 min and withdraw the supernatant.</li>
<li>Re-suspend the beads in <b>20 μL</b> EB.</li>
<li>Transfer the solution with the beads into a 0.2 mL PCR tube. Add the following reagents and start the <b>KIN-P2</b> program<b>.</b>
</li>
</ol>
<table id="undtbl15">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Beads containing cDNA in EB (from step 49)</td>
<td>20 μL</td>
</tr>
<tr>
<td>Phusion Hot master mix, 2×</td>
<td>25 μL</td>
</tr>
<tr>
<td>PCR-2 primer mix (10 μM each)</td>
<td>5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl16">
<thead>
<tr>
<th colspan="4">KIN-P2 PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">10 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>62°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>forever</td>
<td> </td>
</tr>
</tbody>
</table>
<ol start="51">
<li>Transfer the reaction into a 1.5 mL LoBind tube and capture the streptavidin beads. Collect the supernatant, <b>50 μL</b> in volume, which now contains the PCR product, and transfer it into a clean tube.</li>
<li>Add <b>45 μL</b> AMPureXP beads. Mix well by vortex.</li>
<li>Incubate 10 min at 20°C–22°C and place the tube on the magnetic stand for at least 3 min or until the supernatant is clear.</li>
<li>Discard the supernatant. Spin 10 s on a mini-centrifuge (or 11,000 × <i>g</i> on a table top centrifuge) to collect drops.</li>
<li>Place back on magnet and remove all the traces of supernatant.</li>
<li>Resuspend the beads in <b>20 μL</b> EB.</li>
<li>Wait for 1–3 min and place on magnet.</li>
<li>Transfer the clear supernatant (ready library) to a clean 1.5 mL LoBind tube.</li>
<li>Take <b>2 μL</b> for TapeStation for final quality control (<b>sample QC-3</b>).</li>
<li>Store the ready library at −20°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The ready library can be stored at −20°C for up to 6 month.</div>
<h3 id="sec3.6">Library quality control and concentration</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="61">
<li>Analyze the <b>QC-3</b> sample (from step 59) with TapeStation D1000 ScreenTape system. See <a href="#fig2">Figure 2</a> for expected analysis results.</li>
<li>Determine the concentration of the library with the KAPA Library Quantification Kit. (<a href="https://pim-eservices.roche.com/eLD/api/downloads/ca670ceb-fb38-eb11-0291-005056a71a5d?countryIsoCode=pi">https://pim-eservices.roche.com/eLD/api/downloads/ca670ceb-fb38-eb11-0291-005056a71a5d?countryIsoCode=pi</a>).</li>
</ol>
<p>The typical concentration for the ready library is 2–30 nM. <a href="#sec7.5">Troubleshooting 3</a>.</p>
<div class="critical">
<span class="critical-title">Critical:</span> Accurate molar concentration of the library is required to determine how much to load for sequencing. You can determine the concentration from the QC3 sample with TapeStation, however, we have measured it also using KAPA Library Quantification Kit. If there is a difference between the two measurements, we have relied on the result from the KAPA Kit.</div>
<h3 id="sec3.7">Sequencing</h3>
<p>We have sequenced the libraries with NextSeq 500 using High Output v2 kit, 75 cycles. Sequencing was done with custom primer STRT2seq (<a href="#tbl1">Table 1</a>).</p>
<p>Library loading concentration was 2.0 pM and we have added PhiX to 20%.</p>
<h3 id="sec3.8">Sequence data processing</h3>
<p><b>Timing</b><b>:</b> typically up to 1 day, depending on the sequencing depth and size of the library</p>
<p>This section describes the bioinformatics pipeline processing the STRT sequencing data from Illumina Base Call Format (.bcl), including demultiplexing, mapping, quality check, and quantification. For a detailed analysis process and settings, the pipeline can be accessed at: <a href="https://github.com/my0916/STRT2">https://github.com/my0916/STRT2</a>.</p>
<ol start="63">
<li>Install the STRT2 pipeline from the GitHub repository. Type the following command in the terminal window and hit the enter key.</li>
</ol>
<div class="textbox">
<p><code>git clone</code> <a href="https://github.com/my0916/STRT2.git">https://github.com/my0916/STRT2.git</a></p>
</div>
<ol start="64">
<li>Install the dependencies.
<ol type="a">
<li>Mandatory software are listed below. The following versions were used by the authors in preparing this protocol, while different versions may work correctly.
<ol type="i">
<li>Picard (v2.23.4)</li>
<li>HISAT2 (v2.2.1)</li>
<li>SAMtools (v1.10)</li>
<li>Bedtools (v2.29.2)</li>
<li>Subread (v2.0.0)</li>
<li>R (v4.0.0)</li>
<li>Ruby (v2.6.2)</li>
</ol>
</li>
<li>Software required for the optional tools are listed below. The following versions were used by the authors in preparing this protocol, while different versions may work correctly.
<ol type="i">
<li>StringTie (v2.1.4)</li>
<li>FastQC (v0.11.8)</li>
<li>MultiQC (v1.9)</li>
</ol>
</li>
<li>Alternatively, conda environment can be created from <code>condaEnv.yml</code> with the following commands. Please make sure that Anaconda or Miniconda is installed.
<div class="textbox">
<p><code>conda env create -f condaEnv.yml</code></p>
<p><code>conda activate STRT2_env</code></p>
</div>
</li>
<li>For Swedish UPPMAX (<a href="https://www.uppmax.uu.se/">https://www.uppmax.uu.se/</a>) users, these software are available through the <code>module</code> command in the scripts (<code>STRT2-UPPMAX.sh</code>, <code>STRT2-TFE-UPPMAX.sh</code>, and <code>fastq-fastqc-uppmax.sh</code>).</li>
</ol>
</li>
</ol>
<ol start="65">
<li>Prepare the index and dictionary before running the pipeline. Details are described in <a href="https://github.com/my0916/STRT2">https://github.com/my0916/STRT2#How-to-build-HISAT2-index</a>. Here we describe the case for the dog genome canFam3.
<ol type="a">
<li>Obtain the genome sequences of reference and ERCC spike-ins.
<div class="textbox">
<p><code>wget</code> <a href="http://hgdownload.cse.ucsc.edu/goldenPath/canFam3/bigZips/canFam3.fa.gz">http://hgdownload.cse.ucsc.edu/goldenPath/canFam3/bigZips/canFam3.fa.gz</a></p>
<p><code>unpigz -c canFam3.fa.gz | ruby -ne ‘$ok = $_ !∼ /ˆ&gt;chrUn_/ if $_ =∼ /ˆ&gt;/; puts $_ if $ok’&gt; canFam3_reference.fasta</code></p>
<p><code>wget</code> <a href="https://www-s.nist.gov/srmors/certificates/documents/SRM2374_putative_T7_products_NoPolyA_v2.FASTA">https://www-s.nist.gov/srmors/certificates/documents/SRM2374_putative_T7_products_NoPolyA_v2.FASTA</a></p>
<p><code>cat SRM2374_putative_T7_products_NoPolyA_v2.FASTA &gt;&gt; canFam3_reference.fasta</code></p>
</div>
</li>
<li>Extract splice sites and exons from a GTF file using the Python scripts implemented in HISAT2 (v2.1.0) (<a href="#bib5">Kim et al., 2019</a>). Here Ensembl transcript map (canFam3.transMapEnsemblV4.gtf.gz) was downloaded from the UCSC Table Browser.
<div class="textbox">
<p><code>unpigz -c canFam3.transMapEnsemblV4.gtf.gz | hisat2_extract_splice_sites.py - | grep -v ˆchrUn&gt; canFam3.ss</code></p>
<p><code>unpigz -c canFam3.transMapEnsemblV4.gtf.gz | hisat2_extract_exons.py - | grep -v ˆchrUn&gt; canFam3.exon</code></p>
</div>
</li>
<li>Build the HISAT2 index of the reference genome and ERCC Spike-in RNAs with the <code>hisat2-build</code> function, which outputs a set of files with the suffix ‘<code>.ht2</code>’.
<div class="textbox">
<p><code>hisat2-build canFam3_reference.fasta --ss canFam3.ss --exon canFam3.exon canFam3_reference</code></p>
</div>
</li>
<li>Create the sequence dictionary for the reference and Spike-in sequences with Picard (v2.23.4) (<a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a>) <code>CreateSequenceDictionary</code>.
<div class="textbox">
<p><code>java -jar picard.jar CreateSequenceDictionary R=canFam3_reference.fasta O=canFam3_reference.dict</code></p>
</div>
</li>
</ol>
</li>
</ol>
<ol start="66">
<li>Run the STRT2 pipeline with the following command. Please make sure that barcode sequence with barcode name (<a href="#tbl2">Table 2</a>) is prepared as <code>barcode.txt</code>.
<div class="textbox">
<p><code>./STRT2.sh -o {output} -g {reference_genome} -a {annotation}\</code></p>
<p><code>-b {path/to/BaseCalls} -i {path/to/index} -c {Sequencing_center}\</code></p>
<p><code>-r {Run_barcode}</code></p>
</div>
Wherein: <code>{output}</code> defines the prefix of output files.<br> <code>{reference_genome}</code> is the reference genome such as canFam3.<br> <code>{annotation}</code> is the gene annotation used for quality check and counting. For canFam3, <code>ref</code> (RefSeq) or <code>ens</code> (Ensembl) can be chosen. See also the GitHub repository’s README.md file for details.<br> <code>{path/to/BaseCalls}</code> is the path to the Illumina basecalls directory.<br> <code>{path/to/index}</code> is the directory and basename of the HISAT2 index for the reference genome.<br> <code>{Sequencing_center}</code> is the name of the sequencing center that produced the reads. <code>{Run_barcode}</code> is the barcode of the run. Prefixed to read names.
<div class="note">
<span class="note-title">Note:</span> In this case, set the parameters ‘<code>--genome canFam3</code>’ and ‘<code>--annotation ens</code>’. In the default settings, the annotation for quality check and quantification is set to RefSeq.</div>
Details of the pipeline are as follows.
<ol type="a">
<li>The raw base call (BCL) files are demultiplexed with Picard <code>ExtractIlluminaBarcodes</code> and <code>IlluminaBasecallsToSam</code> to generate unaligned BAM files based on the well-specific barcodes (<a href="#tbl2">Table 2</a>).</li>
<li>The unaligned BAM files are converted to FASTQ files with Picard <code>SamToFastq</code> and aligned using HISAT2 to the dog reference genome (canFam3) and ERCC Spike-in RNAs (SRM 2374) [NIST SRM 2374 Certificate of Analysis, <a href="https://www-s.nist.gov/srmors/certificates/2374.pdf">https://www-s.nist.gov/srmors/certificates/2374.pdf</a>, 2013.] with the Ensembl transcript map provided by UCSC (transMapEnsemblV4) as a guide of exon junctions.</li>
<li>The aligned BAM files are merged with the original unaligned BAM files to create a unique molecular identifier (UMI)-annotated BAM files by Picard <code>MergeBamAlignment</code>.</li>
<li>The UMI-annotated BAM files corresponding to each sample derived from 4 lanes are merged using Picard <code>MergeSamFiles</code>, and potential PCR duplicates are marked with Picard <code>Markduplicates</code>.</li>
<li>For quality check, the resulting BAM files per sample are indexed, and then all the qualified reads, reads without potential duplicates, and total mapped reads are counted based on SAM flags using SAMtools (v1.9) (<a href="#bib7">Li et al., 2009</a>). In addition, reads mapped to protein-coding genes and Spike-in RNAs, as well as 5′-ends of them, are counted using BEDtools (v2.29.2) (<a href="#bib10">Quinlan and Hall. 2010</a>). Values of i) log<sub>10</sub> of total mapped read counts, ii) mapping rates, iii) log<sub>10</sub> of Spike-in RNA read counts, iv) ratios of total mapped read counts versus Spike-in RNA read counts, v) 5′-end capture rates of Spike-in RNAs, and vi) 5′-end capture rates of protein-coding genes are plotted with the R (v4.0.0) (<a href="#bib11">R core Team, 2020</a>) package ggplot2 (v3.3.3) (<a href="#bib13">Wickham and Grolemund 2016</a>), marking the outlier samples with the barcode numbers. In <a href="#fig2">Figure 2</a>, globin mRNA rates were calculated by dividing the reads mapped to hemoglobin alpha and beta (HBA and HBB) genes by all the qualified reads per sample.</li>
<li>For quantification, Subread featureCounts (v2.0.0) (<a href="#bib8">Liao et al., 2014</a>) is used to assign the reads to 5′-end of genes with parameters ‘<code>-s 1 --largestOverlap --ignoreDup --primary</code>’. Here, uniquely mapped reads within the 5′- UTR or 500 bp upstream of the protein-coding genes and the first 50 bp of Spike-in sequences are counted.
<div class="optional">
<span class="optional-title">Optional:</span> In addition, one can perform TFE (transcript far 5′-end)-based analysis originally developed in <a href="#bib12">Töhönen et al. (2015)</a>, by running <code>STRT-TFE.sh</code> or <code>STRT2-TFE-UPPMAX.sh</code>. Here, TFEs are defined as the first exon (5′-end region) of assembled STRT reads mapped to the genome using StringTie (v2.1.4) and assigned with unique IDs. TFE-level reads are also quantified as described above. Details are described in the GitHub repository’s TFE-README.md.</div>
<div class="note">
<span class="note-title">Note:</span> If TFE-based analysis is planned, add the option ‘<code>--dta</code>’ (downstream-transcriptome-assembly) for <code>STRT2.sh</code> or <code>STRT2-UPPMAX.sh</code>, which is required in the HISAT2 mapping process, that helps the transcript assembly by StringTie.</div>
<div class="optional">
<span class="optional-title">Optional:</span> After running the pipeline above, one can generate fastq files for each sample from the output BAM files in the <code>fastq</code> directory by running <code>fastq-fastQC.sh</code> or <code>fastq-fastQC-UPPMAX.sh</code>. These fastq files (without duplicated reads) can be submitted to public sequence databases. FastQC files are also generated for each fastq file in the <code>fastqc</code> directory. Based on the FastQC results, MultiQC report (<code>MultiQC_report.html</code>) is generated.</div>
</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The expected results of library quality control stages with TapeStation D1000 ScreenTape system are shown in <a href="#fig2">Figure 2</a>. The typical concentration for the ready library is 2–30 nM.</p>
<p>To study the effect of Globin-Lock®, we designed two libraries with identical layouts, one with the addition of Globin-Lock® step and the other without. We selected several tissues, including those with a high blood content, such as liver and spleen, and in total 40 ng of purified RNA was used for 48-plex libraries. <a href="#fig3">Figure 3</a>. Shows the comparison of the <b>a</b>) relative Globin mRNA amount and <b>b</b>) mapped reads in each library. The amount of globin mRNAs was highest in spleen and in bone marrow. However, the depth of sequencing improved with Globin-Lock® also in other tissues, and especially in tissues with a high amount of globin mRNAs.</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1238-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Effect of Globin-Lock on mapped reads</div>
<p>Two libraries with identical layouts, one with the addition of Globin-Lock® step and the other without were compared.</p>
<p>(A) Relative amount of globin mRNA in the sample RNA.</p>
<p>(B) Mapped reads in the sample RNA. For each tissue, several replicates are shown from different dogs; the number of replicates were two for colon, three for masticatory muscle, four for bone marrow, five for liver, lymph node, lung and thyroid gland, and six for frontal cortex, pancreas, and spleen. The depth of sequencing improved with Globin-Lock® in many tissues, especially in tissues with a high amount of globin mRNAs, such as spleen and bone marrow.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="quantitative-and-statistical-analysis">Quantitative and statistical analysis</h2>
<p>Statistics of redundancy, mapped reads, and mapped rate for an example library is shown in <a href="#fig4">Figure 4</a>. After sequencing, expected redundancy for a successful library is ∼1.5, mapped reads should be <u>&gt;</u> 1M, and mapped rate 60%–80% in the latest dog genome.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1238-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Output from the sequence analysis</div>
<p>Redundancy, mapped reads, and mapped rate from a successful library containing 46 RNA samples of different dog tissues and two non-template controls (NTC).</p>
</figcaption>
</figure>
<p>The library contained 46 RNA samples from different dog tissues and two non-template controls, and the sequences were mapped to Canine genome. Please note that spike-in reads are included as mapped reads in the pipeline.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>High quality of RNA is crucial for optimal results, and in addition to RNA integrity, impurities such as salt residues can affect the efficiency of reverse transcription. From some tissues, such as gut and adipocytes, it can be extremely difficult to extract RNA in good quality. The quality and amount of the starting material also affects the final yield of the cDNA library. The number of PCR cycles given in the protocol usually gives enough material for sequencing, however, the number of cycles needs to be calibrated for each application to find the least number of cycles required.</p>
<p>One of the advantages of the protocol is that it allows identification of transcription start sites. However, since the reads are obtained only from the 5′ end of each cDNA, it is not suitable for analysis of splice isoforms.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The amount of cDNA products in QC1 is low (step 27).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Check the quality of your RNA samples. It should not contain genomic DNA, and absorbance ratios 260/280 nm and 260/230 nm should both be close to 2.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Dominant peak at around 100 bp in the QC1 (step 27).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>During PCR, the primers can form dimers to produce a byproduct around 100 bp. As long as the band is not dominant, and the amount of cDNA is good, this will not affect the end-result. However, the band may become dominant, if your mRNA quality is poor, or the concentration is too low. Make sure the quality of you RNA is sufficient and the concentration is 20 ng/μL.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>The concentration of the ready library is low (below 2 nM) (steps 61 and 62).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Usually 10 cycles in KIN-P2 (step 50) is enough to give sufficient yield for sequencing. However, if needed, one or two additional cycles can be added to the amplification.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Low yield of the sequencing result (sequence data).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Illumina’s Sequencing Analysis Viewer (<a href="https://support.illumina.com/sequencing/sequencing_software/sequencing_analysis_viewer_sav.html">https://support.illumina.com/sequencing/sequencing_software/sequencing_analysis_viewer_sav.html</a> ) helps us dissect the problem; also for the troubleshooting, we start the data processing from the BCL files instead of the FASTQ files. For example, the sequencing quality dropped around the polyG part of the TSO, PhiX concentration should be increased.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Biases in the expression data between the libraries (sequence data).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>The sample layout must be considered carefully when one study uses two or more libraries. For example, to reduce the biases, NBGLM-LBC (<a href="#bib4">Katayama et al., 2019</a>) can be applied. Also, as described in steps 61 and 62, the concentrations of the ready libraries are an important checkpoint before the sequencing.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Reads in the negative control (sequence data).</p>
<h3 id="sec7.12">Potential solution</h3>
<p>Reads in the negative control point to a contamination in the PCR reaction. Follow general recommendations for PCR setup. Dedicate a separate work area for pre-PCR work and decontaminate the area after each use. We also recommend use of <a href="https://www.thermofisher.com/fi/en/home/life-science/lab-plasticware-supplies/pipettes-pipette-tips/handheld-pipette-tips.html">pipette tips with aerosol barriers</a>.</p>
</section>
<section>
<h2 id="consortia">Consortia</h2>
<p>We thank resources and members of the Dog Genome Annotation (DoGA) Consortium (Hannes Lohi, Juha Kere, Carsten Daub, Marjo Hytönen, César L. Araujo, Ileana B. Quintero, Kaisa Kyöstilä, Maria Kaukonen, Meharji Arumilli, Milla Salonen, Riika Sarviaho, Julia Niskanen, Sruthi Hundi, Jenni Puurunen, Sini Sulkama, Sini Karjalainen, Antti Sukura, Pernilla Syrjä, Niina Airas, Henna Pekkarinen, Ilona Kareinen, Anna Knuuttila, Heli Nordgren, Karoliina Hagner, Tarja Pääkkönen, Kaarel Krjutskov, Sini Ezer, Shintaro Katayama, Masahito Yoshihara, Auli Saarinen, Abdul Kadir Mukarram, Matthias Hörtenhuber, Amitha Raman, Irene Stevens)</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ewels, P., Magnusson, M., Lundin, S., and Käller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics <i>32</i>, 3047-3048. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref1">View at publisher</a></p>
<p id="bib2">Islam, S., Kjällquist, U., Moliner, A., Zajac, P., Fan, J.B., Lönnerberg, P., and Linnarsson, S. (2012). Highly multiplexed and strand-specific single-cell RNA 5' end sequencing. Nat. Protoc. <i>7</i>, 813-828. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref2">View at publisher</a></p>
<p id="bib3">Islam, S., Zeisel, A., Joost, S., La Manno, G., Zajac, P., Kasper, M., Lönnerberg, P., and Linnarsson, S. (2014). Quantitative single-cell RNA-seq with unique molecular identifiers. Nat. Methods <i>11</i>, 163-166. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref3">View at publisher</a></p>
<p id="bib4">Katayama, S., Skoog, T., Söderhäll, C., Einarsdottir, E., Krjutškov, K., and Kere, J. (2019). Guide for library design and bias correction for large-scale transcriptome studies using highly multiplexed RNAseq methods. BMC Bioinformatics <i>20</i>, 418. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref4">View at publisher</a></p>
<p id="bib5">Kim, D., Paggi, J.M., Park, C., Bennett, C., and Salzberg, S.L. (2019). Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. Nat. Biotechnol. <i>37</i>, 907-915. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref5">View at publisher</a></p>
<p id="bib6">Krjutškov, K., Koel, M., Roost, A.M., Katayama, S., Einarsdottir, E., Jouhilahti, E.M., Söderhäll, C., Jaakma, Ü., Plaas, M., Vesterlund, L., et al. (2016). Globin mRNA reduction for whole-blood transcriptome sequencing. Sci. Rep. <i>6</i>, 31584. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref6">View at publisher</a></p>
<p id="bib7">Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., and Durbin, R.; 1000 Genome Project Data Processing Subgroup (2009). The sequence alignment/map format and SAMtools. Bioinformatics <i>25</i>, 2078-2079. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref7">View at publisher</a></p>
<p id="bib8">Liao, Y., Smyth, G.K., and Shi, W. (2014). featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics <i>30</i>, 923-930. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref8">View at publisher</a></p>
<p id="bib9">Pertea, M., Pertea, G.M., Antonescu, C.M., Chang, T.C., Mendell, J.T., and Salzberg, S.L. (2015). StringTie enables improved reconstruction of a transcriptome from RNA-seq reads. Nat. Biotechnol. <i>33</i>, 290-295. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref9">View at publisher</a></p>
<p id="bib10">Quinlan, A.R. and Hall, I.M. (2010). BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics <i>26</i>, 841-842. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref10">View at publisher</a></p>
<p id="bib11">R Core Team (2020). R: A Language and Environment for Statistical Computing (R Foundation for Statistical Computing). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref11">View at publisher</a></p>
<p id="bib12">Töhönen, V., Katayama, S., Vesterlund, L., Jouhilahti, E.M., Sheikhi, M., Madissoon, E., Filippini-Cattaneo, G., Jaconi, M., Johnsson, A., Bürglin, T.R., et al. (2015). Novel PRD-like homeodomain transcription factors and retrotransposon elements in early human development. Nat. Commun. <i>6</i>, 8207. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref12">View at publisher</a></p>
<p id="bib13">Wickham, H. and Grolemund, G. (2016). R for Data Science : Import, Tidy, Transform, Visualize, and Model Data (O'Reilly). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00701-2/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Sini Ezer (<a href="mailto:sini.ezer@helsinki.fi">sini.ezer@helsinki.fi</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The data generated during this study will be available upon request.</p>
<p>The code generated during this study is available in <a href="https://doi.org/10.5281/zenodo.5636103">https://doi.org/10.5281/zenodo.5636103</a></p>
<h3>Acknowledgments</h3>
<p>All funding sources; <a href="https://doi.org/10.13039/501100002341">The Academy of Finland</a>, HiLife, <a href="https://doi.org/10.13039/501100004012">Jane and Aatos Erkko Foundation</a>, <a href="https://doi.org/10.13039/501100006306">Sigrid Jusélius Foundation</a>, Scandinavia–Japan Sasakawa Foundation, Japan Eye Bank Association, <a href="https://doi.org/10.13039/501100007263">Astellas Foundation for Research on Metabolic Disorders</a>, <a href="https://doi.org/10.13039/501100001691">Japan Society for the Promotion of Science</a> (JSPS) Overseas Research Fellowships.</p>
<p>We thank Auli Saarinen and Mari Tervaniemi for technical assistance and Biomedicum Functional Genomics Unit (FuGU), University of Helsinki, for library sequencing services.</p>
<p>Part of the computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC 2017/7-317.</p>
<h3>Author contributions</h3>
<p>S.E. prepared the libraries and wrote the manuscript. K.K. developed the protocol. M.Y. and S.K. developed and performed the data analysis. J.K., C.D., and H.L. conceived and designed the project and directed the work. DOGA consortium collected and prepared the animal tissues.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>