<article>
<section>
<h1 id="header">Protocol to alter a protein’s phase separation capacity to control cell fate transitions</h1>
<p><time datetime="2021-10-15">Published: October 15, 2021</time></p>
<p>Qian Ma,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">7</a>,<a href="#fn2">8</a>,<a href="#cor1">*</a></sup> Jia Wang,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">7</a>,<a href="#fn2">8</a>,<a href="#cor2">**</a></sup> Haopeng Yu,<sup><a href="#aff4">4</a>,<a href="#aff5">5</a></sup> Junyi Sun,<sup><a href="#aff3">3</a></sup> Jun Sun,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Jiahao Chen,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a></sup> Yangyinhui Yu,<sup><a href="#aff3">3</a></sup> Peihang Fang,<sup><a href="#aff3">3</a></sup> Qi Tian,<sup><a href="#aff3">3</a></sup> and Junjun Ding<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff6">6</a>,<a href="#fn3">9</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>RNA Biomedical Institute, Sun Yat-Sen Memorial Hospital, Zhongshan School of Medicine, Sun Yat-Sen University, Guangzhou 510080, China</p>
<p id="aff2"><sup>2</sup>Center for Stem Cell Biology and Tissue Engineering, Key Laboratory for Stem Cells and Tissue Engineering, Ministry of Education, Sun Yat-Sen University, Guangzhou 510080, China</p>
<p id="aff3"><sup>3</sup>Department of Cell Biology, Zhongshan School of Medicine, Sun Yat-Sen University, Guangzhou 510080, China</p>
<p id="aff4"><sup>4</sup>West China Biomedical Big Data Center, West China Hospital/West China School of Medicine, Sichuan University, Chengdu 610041, China</p>
<p id="aff5"><sup>5</sup>Med-X Center for Informatics, Sichuan University, Chengdu 610041, China</p>
<p id="aff6"><sup>6</sup>Department of Histology and Embryology, School of Basic Medical Sciences, Guangzhou Medical University, Guangzhou 511436, China</p>
<p id="fn1"><sup>7</sup>These authors contributed equally</p>
<p id="fn2"><sup>8</sup>Technical contact</p>
<p id="fn3"><sup>9</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:maqian57@163.com">maqian57@163.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:15931071629@163.com">15931071629@163.com</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:dingjunj@mail.sysu.edu.cn">dingjunj@mail.sysu.edu.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100887">10.1016/j.xpro.2021.100887</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Phase separation of proteins regulates transcription. Here, we present a protocol to manipulate phase separation capacity of a protein. We use this protocol to disrupt phase separation by mutating residues at intrinsically disordered regions (IDRs). Further, we rescue the disabled phase separation by fusing an IDR known to drive phase separation. Phase separation promotes cell fate transitions, whereas disruption of phase attenuates the transitions. The major challenge is how to effectively predict mutation residues.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="https://doi.org/10.1016/j.stem.2021.04.023">Wang et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Phase separation with liquid-like behavior can be verified <i>in vitro</i> and <i>in vivo</i>
</li>
<li>Residue mutation at IDR can disrupt phase separation capacity of a protein</li>
<li>The disabled phase separation of a protein can be rescued by fusing an IDR</li>
<li>Phase separation regulates cell fate transitions</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1101-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This method is introduced using OCT4 as an example. Actually, this protocol can be widely applied to other proteins with potential phase-separated capacity. The protocol employs somatic cell reprogramming from mouse embryonic fibroblasts (MEFs) to induced pluripotent stem cells (iPSCs) as the cell fate transition system, and hence most of the key steps is performed in MEFs. However, this protocol is not restricted to MEFs and can be widely applied in other cell types such as cancer cells, stem cells or differentiated cells et al.</p>
<h3 id="sec1.1">Selection of perturbation sites</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol>
<li>Acidic mutation of IDR
<ol type="a">
<li>Analyze the protein sequence and identify IDR of human OCT4 protein.</li>
<li>label all glutamic acid (E) at IDR in red and label all aspartic acid (D) at IDR in blue, and replace them with alanine (A) (<a href="#fig1">Figure 1</a>A) (<a href="#bib2">Boija et al., 2018</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1101-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Protein sequence used in this study</div>
<p>(A) The protein sequences of the acidic mutant OCT4. Shadows stand for IDRs.</p>
<p>(B) amino acid 324–326 deleted OCT4. Red arrows stand for the deletion site.</p>
<p>(C and D) The amino acid sequence of IDRs of FUS (C) and hnRNPA (D) respectively.</p>
</figcaption>
</figure>
</li>
<li>Synthesize the whole DNA sequence of the acidic mutant OCT4 and homologous arms for vector construction. DNA synthesis was completed by BGI Genomics, Shenzhen, China. The purification method is PAGE.</li>
</ol>
</li>
<li>Key residue deleted mutation of IDR
<ol type="a">
<li>Based on the features of the known phase separated proteins from the current phase separation databases including LLPSDB, PhaSepDB, PhaSePro and DrLLPS, we predict key amino acid residues of proteins for phase separation. We use a large number of characteristic signals at the protein residue level to establish machine learning prediction model related to phase-separated capacity of a protein. We found the variance of amino acid sequence, mutation or deletion, leads to the alteration of the characteristic signals. We can amplify the alteration of characteristic signals by the model to evaluate the influence of variance of amino acid sequence on phase-separated capacity. A link to predict phase-related residues was provided as: <a href="https://github.com/jsun9003/PSPHunter">https://github.com/jsun9003/PSPHunter</a>
</li>
<li>The prediction strategy is that each three consecutive amino acids are considered as a unit. The phase-separated probability of each unit-deleted OCT4 is calculated and compared with WT OCT4 to generate the probability score. The lower the score, the higher influence of the unit on phase separation.</li>
<li>We selected the key residues by considering three aspects: 1) lower probability score, 2) lower DNA binding, and 3) at the disordered regions. In the light of this standard, deletion of amino acids 324–326 was considered as a potential perturbation to disrupt OCT4 phase separation (<a href="#fig1">Figure 1</a>B).</li>
</ol>
</li>
</ol>
<h3 id="sec1.2">Rescue of phase separation by IDR fusion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ol start="3">
<li>Fusing the defective proteins with protein domains known to drive phase separation can restore the phase-separated capacity of the defective proteins (<a href="#bib4">Liu et al., 2020</a>). Therefore, the protein and DNA sequence of the IDRs of FUS and hnRNPA were collected for use (<a href="#fig1">Figures 1</a>C and 1D).</li>
<li>Add a homologous arm at each side of the IDR fragment gotten from the last step, one of which is repeating the last 20 base pairs (bp) of mutant, another is the same as the first 20 bp of the vector next to the mutant correspondingly.</li>
<li>Synthesize the fragments for vector construction. DNA synthesis was completed by BGI Genomics, Shenzhen, China. The purification method is PAGE.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The aim of this step is to get the fragment of mutant followed by an IDR, which can be reconstructed into an existing OCT4-mutant plasmid. However, researchers can also de novo synthesis a complete fragment as soon as the sequences of mutant, IDR and homologous arms matched to the vectors are put orderly.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>OCT4 (dilution 1:1000)</td>
<td>Santa Cruz</td>
<td>Cat# sc-5279; RRID: AB_628051</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>DMEM Medium</td>
<td>HyClone</td>
<td>Cat# SH30022.01</td>
</tr>
<tr>
<td>DMEM/F12 1:1 Medium</td>
<td>Gibco</td>
<td>Cat# C11330500BT</td>
</tr>
<tr>
<td>Fetal Bovine Serum</td>
<td>VISTECH</td>
<td>Cat# SE100-B</td>
</tr>
<tr>
<td>Trypsin/EDTA</td>
<td>Corning</td>
<td>Cat# 25–051</td>
</tr>
<tr>
<td>KnockOut<sup>TM</sup> Serum Replacement</td>
<td>Gibco</td>
<td>Cat# 10828028</td>
</tr>
<tr>
<td>PD0325901</td>
<td>Selleck</td>
<td>Cat# S1036</td>
</tr>
<tr>
<td>CHIR99021</td>
<td>Selleck</td>
<td>Cat# S1263</td>
</tr>
<tr>
<td>N2 Supplement</td>
<td>Thermo Fisher</td>
<td>Cat# 17502-048</td>
</tr>
<tr>
<td>Puromycin</td>
<td>Sigma-Aldrich</td>
<td>Cat# 540222</td>
</tr>
<tr>
<td>B27 Supplement</td>
<td>Gibco</td>
<td>Cat# 17504044</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>Sigma</td>
<td>Cat# M6250</td>
</tr>
<tr>
<td>Neurobasal</td>
<td>Thermo Fisher</td>
<td>Cat# 21103-049</td>
</tr>
<tr>
<td>NEAA</td>
<td>Thermo Fisher</td>
<td>Cat# 11140050</td>
</tr>
<tr>
<td>GlutaMax</td>
<td>Thermo Fisher</td>
<td>Cat# 35050061</td>
</tr>
<tr>
<td>EcoRI-hf</td>
<td>NEB</td>
<td>Cat# R3101S</td>
</tr>
<tr>
<td>NotI-hf</td>
<td>NEB</td>
<td>Cat# R3189S</td>
</tr>
<tr>
<td>BamHI-hf</td>
<td>NEB</td>
<td>Cat# R3136S</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>NEB</td>
<td>Cat# M0202</td>
</tr>
<tr>
<td>1M Tris-HCl pH=7.5</td>
<td>Solarbio</td>
<td>Cat# T1140</td>
</tr>
<tr>
<td>Glycerol</td>
<td>Sigma</td>
<td>Cat# G5516-1L</td>
</tr>
<tr>
<td>Proteinase inhibitor</td>
<td>Roche</td>
<td>Cat# 1169749800</td>
</tr>
<tr>
<td>Bovine Serum Albumin</td>
<td>Sigma</td>
<td>Cat# A7906</td>
</tr>
<tr>
<td>RNaseA</td>
<td>Thermo</td>
<td>Cat# EN0531</td>
</tr>
<tr>
<td>Benzonase</td>
<td>Sigma</td>
<td>Cat# E1014</td>
</tr>
<tr>
<td>GelCode<sup>TM</sup> Blue Safe Protein Stain Buffer</td>
<td>Thermo Scientific</td>
<td>Cat# 24594</td>
</tr>
<tr>
<td>5M NaCl solution</td>
<td>Sigma</td>
<td>Cat# S5150</td>
</tr>
<tr>
<td>BES buffered saline</td>
<td>Sigma</td>
<td>Cat# 14280-100ML</td>
</tr>
<tr>
<td>Propidium iodide</td>
<td>eBioscience</td>
<td>Cat# 88–8007-72</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma</td>
<td>Cat# D9542</td>
</tr>
<tr>
<td>DTT</td>
<td>Solarbio</td>
<td>Cat# D8220</td>
</tr>
<tr>
<td>IPTG</td>
<td>Solarbio</td>
<td>Cat# I1020-5</td>
</tr>
<tr>
<td>PEG-8000</td>
<td>Sigma</td>
<td>Cat# 89510-250G-F</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNAzol</td>
<td>MRC</td>
<td>Cat# RN190-500</td>
</tr>
<tr>
<td>EndoFree Plasmid Midi Kit</td>
<td>Cwbio</td>
<td>Cat# CW2105S</td>
</tr>
<tr>
<td>E.Z.N.A. Gel Extraction Kit</td>
<td>Omega</td>
<td>Cat# D2500-02</td>
</tr>
<tr>
<td>TIANquick Midi Purification Kit</td>
<td>TIANGEN</td>
<td>Cat# DP204-03</td>
</tr>
<tr>
<td>TIANamp Genomic DNA Kit</td>
<td>TIANGEN</td>
<td>Cat# DP304-02</td>
</tr>
<tr>
<td>His-tag Protein Purification Kit</td>
<td>Beyotime</td>
<td>Cat# P2226</td>
</tr>
<tr>
<td>Trelief™ SoSoo Cloning Kit</td>
<td>Tsingke</td>
<td>Cat# TSV-S2</td>
</tr>
<tr>
<td>SYBR qPCR Master Mix</td>
<td>Vazyme</td>
<td>Cat# Q711-00</td>
</tr>
<tr>
<td>Leukocyte Alkaline Phosphatase Kit</td>
<td>Sigma</td>
<td>Cat# 86R</td>
</tr>
<tr>
<td>Alkaline Phosphatase Stain Kit</td>
<td>Yeasen</td>
<td>Cat# 40749ES60</td>
</tr>
<tr>
<td>Qubit<sup>TM</sup> 1<b>×</b> dsDNA HS Assay Kit</td>
<td>Invitrogen</td>
<td>Cat# Q33230</td>
</tr>
<tr>
<td>Enhanced BCA Protein Assay Kit</td>
<td>Beyotime</td>
<td>Cat# P0010</td>
</tr>
<tr>
<td>SDS-PAGE Gel Kit</td>
<td>Beyotime</td>
<td>Cat# P0012A</td>
</tr>
<tr>
<td>NuPAGE<sup>TM</sup> Sample Reducing Agent</td>
<td>Thermo Scientific</td>
<td>Cat# NP0009</td>
</tr>
<tr>
<td>Clarity<sup>TM</sup> Western ECL Substrate</td>
<td>Bio-Rad</td>
<td>Cat# 1705061</td>
</tr>
<tr>
<td>Centrifugal Filter</td>
<td>Millipore</td>
<td>Cat# UFC803096</td>
</tr>
<tr>
<td>ZymoTaq<sup>TM</sup> Premix</td>
<td>Zymo Research</td>
<td>Cat# E2003</td>
</tr>
<tr>
<td>Q5 High-Fidelity 2<b>×</b> Master Mix</td>
<td>NEB</td>
<td>Cat# M0492L</td>
</tr>
<tr>
<td>DreamTaq Green PCR Master Mix</td>
<td>Thermo Fisher</td>
<td>Cat# K1081</td>
</tr>
<tr>
<td>Primesoript RT Master Mix</td>
<td>Takara</td>
<td>Cat# RR036A</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>rtTA-OG2-OSKM transgenic MEFs</td>
<td>Laboratory of Shaorong Gao</td>
<td>N/A</td>
</tr>
<tr>
<td>OG2 MEFs</td>
<td>Laboratory of Jiekai Chen</td>
<td>N/A</td>
</tr>
<tr>
<td>OD14 MEFs</td>
<td>Laboratory of Jiekai Chen</td>
<td>N/A</td>
</tr>
<tr>
<td>ZHBTc4 PSCs</td>
<td>Laboratory of Shaorong Gao</td>
<td>N/A</td>
</tr>
<tr>
<td>Plat-E</td>
<td>Laboratory of Jiekai Chen</td>
<td>N/A</td>
</tr>
<tr>
<td>293T cells</td>
<td>ATCC</td>
<td>Cat# CRL-3216; RRID: CVCL_0063</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>BL21 <i>E.coli</i>
</td>
<td>TransGen Biotech</td>
<td>Cat# CD201-02</td>
</tr>
<tr>
<td>Nikon Eclipse Ts2R-FL</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
<tr>
<td>Covaris sonicator E220</td>
<td>Covaris</td>
<td>N/A</td>
</tr>
<tr>
<td>LightCycler 480 II</td>
<td>Roche</td>
<td>N/A</td>
</tr>
<tr>
<td>T100<sup>TM</sup> Thermal Cycler</td>
<td>Bio-Rad</td>
<td>Cat# T100</td>
</tr>
<tr>
<td>Qubit<sup>TM</sup> 4 Fluorometer</td>
<td>Invitrogen</td>
<td>Cat# Q33226</td>
</tr>
<tr>
<td>Nikon A1 laser confocal microscope</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Vector construction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This step describes how to construct plasmids with aimed genes for prokaryotic and eukaryotic cells respectively.</p>
<ol>
<li>Pet28a vector constriction for in vitro protein purification (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1101-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Vector construction</div>
<p>(A) pet28a vectors containing each type of OCT4 mutants were constructed by homologous recombination for in vitro droplet formation assay.</p>
<p>(B) pMXs vectors containing each type of OCT4 mutants were constructed for in vivo functional studies.</p>
</figcaption>
</figure>
<ol type="a">
<li>Pet28a prokaryotic expression vector is used for in vitro protein purification and in vitro droplet formation assay. This vector contains His-tag fragment for in vitro protein purification, and EGFP fragment for detecting green fluorescence. The DNA fragments of WT <i>OCT4</i>, acidic mutant <i>OCT4 (amOCT4)</i>, del324-326 <i>OCT4 (delOCT4)</i> and the IDRs of FUS or hnRNPA were cloned into pet28a respectively to generate pet28a-<i>wtOCT4</i>, pet28a-<i>amOCT4</i>, pet28a-<i>delOCT4</i> pet28a-<i>FUS</i> and pet28a-<i>hnRNPA</i> vectors.</li>
<li>The IDRs of FUS or hnRNPA proteins were fused into pet28a-<i>amOCT4</i> and pet28a-<i>delOCT4</i> vectors by homologous recombination respectively to obtain phase-rescued vectors pet28a-<i>amOCT4-FUS</i>, pet28a-<i>amOCT4-hnRNPA</i>, pet28a-<i>delOCT4-FUS</i> and pet28a-<i>delOCT4-hnRNPA.</i>
<ol type="i">
<li>EcoRI is used to digest Pet28a vector at 37°C for 12 h. The digested vectors were purified by gel electrophoresis in 1.5% agarose gel and then collected by Gel Extraction Kit. The reaction system is as follows:
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Pet28a vector</td>
<td>80ng</td>
<td>10 μL</td>
</tr>
<tr>
<td>EcoRI-hf (20 units/μL)</td>
<td>100 units</td>
<td>5 μL</td>
</tr>
<tr>
<td>10 X Cutsmart Buffer</td>
<td>1 X</td>
<td>5 μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>30 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The Time for digestion is better be controlled within 8–12 hours, as too short will cut not enough and too long will digest overly.</div>
</li>
<li>To inactivate restriction enzymes, the whole system is incubated at 65°C for 20 min when the cutting process has been completed. Put the digestion mixture into DNA gel and perform electrophoresis. Cut the target fragments with lancet and purify by Omega DNA purification Kit.</li>
<li>To get the inserted fragments, amplify the IDRs of FUS or hnRNPA by PCR using specific primers containing homologous arms. The PCR products were purified by columns.
<table id="undtbl2">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2<b>×</b> Q5</td>
<td>1<b>×</b>
</td>
<td>25 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Pet28a-<i>FUS/hnRNPA</i> vector</td>
<td>50ng</td>
<td>5 μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>15 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>PCR reactions with following conditions
<table id="undtbl3">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial denaturation</td>
<td>98°C</td>
<td>30s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10s</td>
<td rowspan="3">40</td>
</tr>
<tr>
<td>Annealing</td>
<td>72°C</td>
<td>30s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>12°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
</li>
<li>Insert the IDR fragments into the digested vectors by homologous recombination. The incubation condition is at 50°C for 50 min.
<table id="undtbl4">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2<b>×</b> Soso</td>
<td>1<b>×</b>
</td>
<td>10 μL</td>
</tr>
<tr>
<td>Digested vectors</td>
<td>50 ng</td>
<td>X μL</td>
</tr>
<tr>
<td>Inserted fragments</td>
<td>6 times of Mol<sub>(digested vector)</sub>
</td>
<td>X μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>Add H<sub>2</sub>O to 20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The best mole ratio between digested vector and inserted fragments for high efficiency of homologous recombination is 1:6. The account of digested vector is 50ng.</div>
</li>
</ol>
</li>
<li>Transformation.
<ol type="i">
<li>Take the competent bacterial BL21 from the refrigerator at −80°C, and thaw on ice for 5 min.</li>
<li>Add 10 μL ligation product into 50 μL competent, mix gently and incubate on ice for 30 min.</li>
<li>Incubate the tubes in 42°C for 45 s, followed by cooled on ice for 2 min.</li>
<li>Add 500 μL of antibiotic-free LB liquid medium and shake at 37°C for 1 h.</li>
<li>Centrifuge at 700 <b>×</b> <i>g</i> for 5 min. Discard 480 μL supernatant, resuspend the pellet, drop all onto the plate, and coat evenly. Invert the plate and incubate at 37°C for 12 h.</li>
<li>Pick bacteria mono colony and rotate at 37°C with 200 μL kanamycin-positive LB media for 2 h. Test the sequence of clones.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>pMXs vector construction for in vivo cell transfection (<a href="#fig2">Figure 2</a>B).
<ol type="a">
<li>pMXs eukaryotic expression vector is used for in vivo cell transfection and functional studies. The corresponding DNA fragments were cloned into pMXs vectors from pet28a vectors by enzyme cut and ligation, respectively.
<ol type="i">
<li>NotI and BamHI are used to digest pMXs vectors at 37°C for 12 h.
<table id="undtbl5">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>pMXs vector</td>
<td>5 μg</td>
<td>5.8 μL</td>
</tr>
<tr>
<td>NotI-hf (20 units/μL)</td>
<td>50 units</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>BamHI-hf (20 units/μL)</td>
<td>50 units</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 X Cutsmart Buffer</td>
<td>1<b>×</b>
</td>
<td>5 μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>34.2 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The time for digestion should be controlled within 8–12h, as too short will not cut enough and too long will digest overly.</div>
</li>
<li>To inactivate restriction enzymes, the whole system is incubated at 65°C for 20 min when the cutting process has been complete.</li>
<li>The digested vectors were purified by gel electrophoresis in 1.5% agarose gel. Cut and collect the target fragment (the size is about 4529 bp) by Gel Extraction Kit. The reaction system is as follows:</li>
<li>Inserted fragments are amplified from pet28a vectors by PCR using specific primers containing NotI and BamHI restriction sites. The PCR products were purified by columns.
<table id="undtbl6">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2<b>×</b> Q5</td>
<td>1<b>×</b>
</td>
<td>25 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Pet28a vectors</td>
<td>50ng</td>
<td>5 μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>15 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>NotI and BamHI are used to digest PCR products at 37°C for 12 h. The digested PCR products were purified by gel electrophoresis in 1.5% agarose gel and collected by Gel Extraction Kit. The reaction system is as follows:
<table id="undtbl7">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PCR product</td>
<td>2 μg</td>
<td>X μL</td>
</tr>
<tr>
<td>NotI-hf (20 units/μL)</td>
<td>50 units</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>BamHI-hf (20 units/μL)</td>
<td>50 units</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>10 <b>×</b> Cutsmart Buffer</td>
<td>1<b>×</b>
</td>
<td>5 μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>Add to 50 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Ligation of digested pMXs vectors and inserted fragments at 16°C for 12 h.
<table id="undtbl8">
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Digested pMXs vectors</td>
<td>100 ng</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>Digested inserted fragment</td>
<td>5 times of Mol<sub>(digested vector)</sub>
</td>
<td>X μL</td>
</tr>
<tr>
<td>T4 ligase (400 units/μL)</td>
<td>400 units</td>
<td>1 μL</td>
</tr>
<tr>
<td>10 <b>×</b> T4 ligation buffer</td>
<td>1<b>×</b>
</td>
<td>2 μL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>Add to 20 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The best mole ratio between digested vector and inserted fragments for high efficiency of ligation is 1:5. The account of digested vector is 100ng.</div>
</li>
</ol>
</li>
<li>Transformation.
<ol type="i">
<li>Take the competent bacterial DH5-α from the refrigerator at −80°C, and thaw on ice for 5 min.</li>
<li>Add 10 μL ligation product into 50 μL competent, mix gently and incubate on ice for 30 min.</li>
<li>Incubate the tubes in 42°C for 45 s, followed by cooled on ice for 2 min.</li>
<li>Add 500 μL of antibiotic-free LB liquid medium and shake at 37°C for 1 h.</li>
<li>Centrifuge at 700 <b>×</b> <i>g</i> for 5 min. Discard 480 μL supernatant, resuspend the pellet, drop all onto the plate, and coat evenly. Turn the plate over and incubate at 37°C for 12 h.</li>
<li>Pick bacteria colony and rotate at 37°C with 200 μL kanamycin-positive LB media for 2 h. Test the sequence of clones.</li>
</ol>
</li>
</ol>
</li>
</ol>
<h3 id="sec2.2">
<i>In vitro</i> droplet formation assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>This step aims to test the potential phase-separated ability of WT OCT4, amOCT4, delOCT4, amOCT4-FUS, amOCT4-hnRNPA, delOCT4-FUS and delOCT4-hnRNPA in vitro. A flow chart was provided to describe the process of this assay (<a href="#fig3">Figure 3</a>A).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1101-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Protein purification for <i>in vitro</i> droplet formation assay</div>
<p>(A) Schemes for protein purification and in vitro droplet formation assay.</p>
<p>(B) Bacterial pellets show fluorescence when expressing GFP (green) or mCherry (red).</p>
<p>(C) Proteins are purified by His-tag column.</p>
<p>(D) The purified OCT4-GFP proteins which have green fluorescence are collected.</p>
<p>(E) Proteins are separated by western blotting gel. The interested proteins with correct size are identified.</p>
<p>(F) Proteins are concentrated to a small volume (about 200uL) for next use.</p>
</figcaption>
</figure>
<ol start="3">
<li>A fresh bacterial colony was cultured into 30 mL LB media containing kanamycin for 12 h at 37°C.</li>
<li>Then the cells were diluted 1:30 in 300 mL LB with freshly added kanamycin and grown 2 h at 37°C to make sure OD600 up to 0.6–0.8.</li>
<li>Then temperature was decreased to 16°C and 0.3 mM IPTG (Solarbio) was added. Incubate at 16°C for 16 h.</li>
<li>Collect the bacterial pellets by centrifuging at 3000 <b>×</b> <i>g</i> at 18°C for 15 min. The bacterial pellets which express GFP or mCherry proteins exhibit green or red fluorescence respectively (<a href="#fig3">Figure 3</a>B).</li>
<li>Bacterial pellets were resuspended in 15 mL of non-denaturing lysis buffer (Beyotime) containing protease inhibitors and sonicated (30 cycles of 30 s on, 15 s off).</li>
<li>The lysates were cleared by centrifugation at 3000 <b>×</b> <i>g</i> for 30 min and added to 1 mL of BeyoGold<sup>TM</sup> His-tag Purification Resin (Beyotime) that had been pre-equilibrated with 5 volumes of the same buffer.</li>
<li>Tubes containing this agarose slurry were rotated for 1.5 h at 4°C.</li>
<li>The slurry was poured into a column (<a href="#fig3">Figure 3</a>C), washed with 10 volumes of the washing buffer and eluted six times with elution buffer containing 250 mM imidazole. Each time using about 500 uL elution buffer. Collect the flow-through solution with green fluorescence (<a href="#fig3">Figure 3</a>D).</li>
<li>20 uL of the collected protein solution was run on a 12% western blotting gel to check the protein bands with correct size (<a href="#fig3">Figure 3</a>E).</li>
<li>Transfer the protein solution into the dialysis bag and dialyze at 4°C for 12 h in 1 L dialysis buffer with rotation.</li>
</ol>
<table id="undtbl9">
<thead>
<tr>
<th colspan="3">Dialysis buffer</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1M Tris-HCl, pH=7.5</td>
<td>50 mM</td>
<td>50 mL</td>
</tr>
<tr>
<td>5M NaCl</td>
<td>125 mM</td>
<td>25 mL</td>
</tr>
<tr>
<td>DTT</td>
<td>n/a</td>
<td>1 mL</td>
</tr>
<tr>
<td>Glycerol</td>
<td>10%</td>
<td>100 mL</td>
</tr>
<tr>
<td>ddH2O</td>
<td>n/a</td>
<td>824 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<ol start="13">
<li>The recombinant GFP or mCherry fusion proteins were concentrated in centrifugal filters (Millipore) to get about 200 uL protein for use (<a href="#fig3">Figure 3</a>F).</li>
<li>The concentration of protein solutions was detected by Enhanced BCA Protein Assay Kit (Beyotime). 20 nM of proteins were mixed with 10% PEG-8000 (Sigma), and solve in 125 mM NaCl (Sigma) to get protein mixture.</li>
<li>Load one droplet of the protein mixture onto a glass slide, cover with a cover glass, and immediately imaged with a fluorescence microscopy (Nikon Eclipse Ts2R-FL). Microscopy condition is to adjust objective at 40<b>×</b> and eyepiece at 10<b>×</b>. The exposure time is 500 ms.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> the assay would be paused at Step 10 and the collected protein solution should be stored at −80°C no more than half a year.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Sonication generates heat which may destroy proteins. To ensure protein quality in the sonication step 7, the protein solution must be kept on ice during sonication.</div>
<h3 id="sec2.3">
<i>In vitro</i> fast recovery after photobleaching (FRAP)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step describes a method to evaluate whether OCT4 droplets have liquid-like behavior.</p>
<ol start="16">
<li>The proteins with an appropriate concentration mixed with 10% PEG-8000 (Sigma) and solve in 125 mM NaCl (Sigma), and was immediately loaded onto a homemade chamber comprising a glass slide, and imaged with a Nikon A1 laser confocal microscope using a 100<b>×</b> oil-immersion objective lens (HP Apo TIRF 100xH, 1.49 NA, Nikon).</li>
<li>5 pulses of laser (488 nm) at a 50 us dwell time was applied to the droplet, and recovery was imaged on Nikon A1 laser confocal microscope every 1 s for the indicated time periods.</li>
<li>The fluorescence intensity of bleached cell at each time point was normalized by fluorescence intensity at background region and fluorescence intensity of adjacent unbleached cell.</li>
<li>The images were analyzed using NIS-Elements software.</li>
</ol>
<h3 id="sec2.4">
<i>In vitro</i> droplet fusion</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step describes a method to evaluate whether OCT4 droplets have liquid-like behavior.</p>
<ol start="20">
<li>The proteins with an appropriate concentration mixed with 10% PEG-8000 (Sigma) and solve in 125 mM NaCl (Sigma), and was immediately loaded onto a homemade chamber comprising a glass slide, and imaged with a fluorescence microscopy (Nikon Eclipse Ts2R-FL).</li>
<li>Image two proximate droplets fused into one large droplet in a time-dependent manner. The fusion between droplets may be completed within 1 min.</li>
</ol>
<h3 id="sec2.5">Virus packaging</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>This step describes how to package virus and vectors for cell infection.</p>
<ol start="22">
<li>18–24 h before transfection, seed 12.5<b>×</b>10<sup>6</sup> 293T cells in a 10 cm plate by using DMEM containing 10% fetal bovine serum (FBS) without adding P/S, and ensure cells reach near 80–90% confluence when transfection.</li>
<li>Add 16.7 μg target plasmid, 11.17 μg psPAX2, 7.4 μg VSV-G and 450 μL water to an Eppendorf tube, then mix them up.</li>
</ol>
<table id="undtbl10">
<thead>
<tr>
<th colspan="2">Viral solution</th>
</tr>
<tr>
<th>Reagents</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>VSV-G</td>
<td>7.4 ug</td>
</tr>
<tr>
<td>psPAX2</td>
<td>11.17 ug</td>
</tr>
<tr>
<td>Target plasmid</td>
<td>16.7 ug</td>
</tr>
<tr>
<td>H2O</td>
<td>450 uL</td>
</tr>
</tbody>
</table>
<ol start="24">
<li>Add 62.5 μL 2M CaCl<sub>2</sub> to water-DNA mixture.</li>
<li>Add 500uL 2<b>×</b> BES buffered saline in a 15 mL conical tube, followed by adding the DNA mixture dropwise to the 2<b>×</b> BES buffered saline and blow evenly.</li>
<li>Incubate at 20°C for 15 min, until the solution become slightly cloudy.</li>
<li>In the meantime, aspirate media off the 293T cells seeded the day before and add 10 mL fresh media (without P/S) with 25 nM chloroquine.</li>
<li>Add precipitate dropwise to 293T cells, distributing evenly.</li>
<li>Incubate plate at 37°C for 12 h.</li>
<li>18–24 h post incubation, change media and check for reporter gene expression (if applicable).</li>
<li>Collect supernatant at 48–72 h post transfection. Virus is good for about 2 weeks at 4°C. Store concentrated virus at −80°C.</li>
</ol>
<h3 id="sec2.6">Virus infection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<p>This major step describes how to infect MEFs with virus.</p>
<ol start="32">
<li>Recover MEFs and plant into 24-well plate.</li>
<li>Infect MEFs with lentivirus containing pMXs-wtOCT4, pMXs-amOCT4, pMXs-delOCT4, pMXs-amOCT4-FUS, pMXs-amOCT4-hnRNPA, pMXs-delOCT4-FUS and pMXs-delOCT4-hnRNPA plasmids respectively. The ratio of medium to virus solution is 3:1.</li>
<li>For the next day, the infected MEFs were selected by 2 μg/mL puromycin and 300 μg/mL hygromycin for 5 days.</li>
<li>After selection, the second passage MEFs were used in the next step.</li>
</ol>
<h3 id="sec2.7">Immunofluorescence (IF)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="36">
<li>Plate cells on the 14 mm slides, and wash twice with 2 mL warmed PBS.</li>
<li>Use 4% paraformaldehyde to fix the cells for 15 min at 25°C. Then paraformaldehyde was removed and the cells were permeabilized by 0.5% Triton X-100 for 5 min.</li>
<li>Block cells with 200 μL blocking buffer (consists of 5% BSA, 0.5% Triton X-100, 94.5% PBS; Triton X-100 was added freshly just before use) for 30 min at 25°C.</li>
<li>After aspirating blocking buffer, primary antibody was added at a certain concentration (1:500) in blocking buffer at 4°C for 12 h or 1 h at 25°C.</li>
<li>Wash cells with PBS five times followed by incubation with labeled secondary antibody at a certain concentration of 1:2000 in blocking buffer for 1 h at 25°C without light exposure.</li>
<li>After two washes of PBS, cells were stained with 3 μM DAPI (Sigma) in PBS for 10 min at 25°C.</li>
<li>Finally, images were acquired on Nikon A1 laser confocal microscope using the module of ECLIPSE Ti-E with N-STORM. Cells labeled with OCT4-GFP were stimulated by a 488-nm laser. Nuclei stained with DAPI were stimulated by a 405-nm laser.</li>
</ol>
<h3 id="sec2.8">
<i>In vivo</i> FRAP</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>This step describes a method to evaluate whether OCT4 in vivo condensates have liquid-like behavior.</p>
<ol start="43">
<li>The plenty-OCT4-GFP vectors were introduced into mouse embryonic stem cells (mESCs) by viral infection.</li>
<li>The mESCs then were cultured in dish whose bottom was gelatin-coated, and doxycycline (DOX) was added to stimulate stable OCT4-GFP expression (final concentration: 2ug/mL).</li>
<li>Change medium every day, and keep culture by DOX medium for 72 h.</li>
<li>Select the mono colonies with GFP fluorescence, and plant the colonies onto glass bottom for imaging.</li>
<li>Fluorescence images of GFP were acquired on a Nikon A1 laser confocal microscope using a 100<b>×</b> oil-immersion objective lens (HP Apo TIRF 100xH, 1.49 NA, Nikon). 5 pulses of laser (488 nm) at a 50 us dwell time was applied to the droplet, and recovery was imaged on the Nikon A1 laser confocal microscope every 1 s for the indicated time periods.</li>
<li>The fluorescence intensity of bleached cell at each time point was normalized by fluorescence intensity at background region and fluorescence intensity of adjacent unbleached cells.</li>
<li>The images were analyzed using NIS-Elements software.</li>
</ol>
<h3 id="sec2.9">MEF reprogramming</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 days</div>
<p>This step aims to clarify the influence of OCT4 mutations and rescued OCT4 mutants on cell fate transitions.</p>
<ol start="50">
<li>Experimental design.
<ol type="a">
<li>MEFs are recovered into 6 cm plate and expanded.</li>
<li>The MEFs are sub-cultured and planted into four of 12-well plates as follows and each well has 1.5<b>×</b>10<sup>4</sup> cells.</li>
<li>Plate A contains 16 wells which are used to AP staining and count OCT4-GFP positive clones after reprogramming.</li>
<li>Plate B contains 16 wells for RNA extraction and qPCR. The time points for cell collection are day 0, 2, 4, 6, 8, 10 and 12.</li>
<li>Plate C contains 16 wells for collection of iPSC clones and further culture.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 16 wells are calculated as duplicate for empty vector, wtOCT4, amOCT4, delOCT4, amOCT4-FUS, amOCT4-hnRNPA, delOCT4-FUS and delOCT4-hnRNPA, respectively.</div>
<ol start="51">
<li>Reprogramming.
<ol type="a">
<li>MEFs are cultured in iPSC medium (DMEM supplemented with 10% FBS, 10% Knockout serum replacement, 0.1mMb-mercaptoethanol, NEAA, 2 mM Glutamax, Nucleoside MIX, leukemia inhibitory factor (LIF)). Medium is changed every day for 12 days to get the iPSC clones.</li>
</ol>
</li>
</ol>
<table id="undtbl11">
<thead>
<tr>
<th colspan="3">iPSC medium</th>
</tr>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM</td>
<td>n/a</td>
<td>375 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Knockout serum replacement</td>
<td>10%</td>
<td>50 mL</td>
</tr>
<tr>
<td>Nucleoside MIX</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>Glutamax</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>NEAA</td>
<td>1%</td>
<td>5 mL</td>
</tr>
<tr>
<td>b-mercaptoethanol (0.1 M)</td>
<td>0.1 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>LIF</td>
<td>2%</td>
<td>9.5 mL</td>
</tr>
<tr>
<td><b>Total volume</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<h3 id="sec2.10">Colony formation assay (CFA)</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>CFA is performed to evaluate reprogramming efficiency in response to each perturbation of OCT4.</p>
<ol start="52">
<li>After reprogramming, the iPSC colonies are stained with alkaline phosphatase (AP) for 1 h, followed by washing with PBS for three times.</li>
<li>The positively stained clones are counted and compared between the experimental and control groups.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The positive colonies may represent a color from pink to red, while the negative may appear transparent without any color staining.</div>
<h3 id="sec2.11">Flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Flow cytometry is performed to evaluate reprogramming efficiency by counting OCT4-GFP positive cells in response to each perturbation of OCT4.</p>
<ol start="54">
<li>The cells were harvested at 12th day during reprogramming, washed twice with PBS, and the OCT4-GFP positive cell number was considered for reprogramming efficiency by flow cytometry.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All colony formation assays and OCT4-GFP flow cytometry assays were performed for three biological replicates.</div>
<h3 id="sec2.12">qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>This step is performed to check the expressions of pluripotency genes in response to each perturbation of OCT4.</p>
<ol start="55">
<li>RNA extraction and reverse transcription
<ol type="a">
<li>Total RNA was extracted from cell pellets using RNAzol reagent (MRC) and cDNA was synthesized using Primesoript RT Master Mix (Takara).</li>
</ol>
</li>
<li>qPCR
<ol type="a">
<li>qPCR was performed using SYBR qPCR Master Mix on LightCycler 480 II system.</li>
</ol>
</li>
<li>Data analysis
<ol type="a">
<li>The fold change (FC) of experimental group versus control group was calculated. Ct was calculated with Ct = Ct (test gene) - Ct (Ref. gene). Ct was calculated with Ct = Ct (mutant or rescue group) – Ct (wide type group).</li>
<li>The FC of a test gene in experimental group versus control group was calculated with FC = 2ˆ(-Ct).</li>
<li>Each gene tested in triplicates in every independent experiment, and all experiments were triplicated.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>OCT4 alone can form irregular aggregates (<a href="#fig4">Figure 4</a>A), which is not associated with phase separation. The IDR of MED1 is required for OCT4 to form circle-like droplets (<a href="#fig4">Figures 4</a>B and 4C). These results demonstrate that OCT4 phase separation needs the participation of coactivator such as MED1. OCT4 droplets can fuse, and can fast recover after photobleaching, indicating OCT4 droplets have liquid-like behavior. These claims are also validated by in vivo study showing that OCT4 can form phase-separated condensates in vivo, which can fast recover after photobleaching (<a href="#bib10">Wang et al., 2021</a>). Both acidic mutation and residue deletion can disrupt OCT4 condensates in vivo and OCT4 droplets in vitro, demonstrating IDR perturbations can destroy the phase separation capacity of proteins. Furthermore, the disabled phase separation ability of OCT4 mutants can be rescued by fusing an IDR of FUS or hnRNPA (<a href="#bib10">Wang et al., 2021</a>). These results support that our approach can effectively control phase-separated capacity of proteins.</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1101-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Phase separation regulates cell fate transitions</div>
<p>(A–C) Representative images of droplet formation showing that OCT4 can form droplets in vitro with MED1-IDR. 20uM proteins were added into droplet formation buffer with 125 mM NaCl and 10% PEG8000.</p>
<p>(D) CFA is performed to compare the influence of WT, mutant and IDR-fused OCT4 on reprogramming efficiency. Mutations of OCT4 reduce the reprogramming efficiency, which is restored by IDR fusion. Data are presented as mean±SD. ∗∗p&lt;0.01 compared with mutant OCT4; ##p&lt;0.01, ###p&lt;0.001compared with hnRNPA- or FUS-IDR. iPSC colony numbers are from three biological replicates.</p>
<p>(E) OCT4-GFP positive cell flow cytometry was performed to compare the influence of WT, mutant and IDR-fused OCT4 on reprogramming efficiency. Mutations of OCT4 reduce the reprogramming efficiency, which is restored by IDR fusion. Data are presented as mean±SD. ∗∗p&lt;0.01 compared with mutant OCT4; ##p&lt;0.01, ###p&lt;0.001compared with hnRNPA- or FUS-IDR. OCT4-GFP positive cell numbers are from three biological replicates.</p>
<p>(F and G) Comparison of reprogramming efficiency by OSN expressions among WT, mutant and IDR fused OCT4 cells. ∗p&lt;0.05, ∗∗p&lt;0.01 compared with mutant OCT4 cells. ###p&lt;0.001 compared with hnRNPA- or FUS-IDR cells. The expressions are from three biological replicates.</p>
</figcaption>
</figure>
<p>Co-overexpression of OCT4, SOX2, KLF4 and MYC (OSKM) in MEFs can initiate somatic cell reprogramming (<a href="#bib9">Takahashi and Yamanaka, 2006</a>). However, overexpression of OCT4 mutants with SKM reduces reprogramming efficiency by reducing iPSC clones (<a href="#fig4">Figure 4</a>D), decreasing GFP-OCT4 positive cells (<a href="#fig4">Figure 4</a>E) and down-regulating Oct4, Sox2 and Nanog (OSN) expressions (<a href="#fig4">Figures 4</a>F and 4G), indicating phase disruption can attenuate reprogramming. Importantly, IDR fusion which rescues phase-separated capacity of OCT4 restores reprogramming efficiency (<a href="#fig4">Figures 4</a>D–4G). These results demonstrate that disruption of phase separation attenuates reprogramming, which can be restored by phase rescue via IDR fusion.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>IDR is necessary for protein to form phase-separated condensates. Therefore, changing the component of amino acids in IDR may influence the phase-separated capacity. Two ways, acidic mutation and alkaline mutation, have been reported to effectively disrupt phase separation (<a href="#bib2">Boija et al., 2018</a>; <a href="#bib8">Sabari et al., 2018</a>). However, considerable amino acids are replaced by these ways and we cannot make sure whether other functions of the protein are changed. To address this problem, we predict new amino acid residues of proteins for phase separation based on existent database. Although a key residue has been found to largely contribute to phase separation in this study, the accuracy of the prediction has not been fully validated by enough functional studies. Thus, more experimental validation should be performed to prove its reliability. Moreover, effective validation strategy to make sure that the other functions of protein have not been changed in response to these perturbations including mutation and rescue is also lacking. Together, the potential application of an improved protocol for controlling phase-separated capacity of proteins is expected and promising.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>The reliability of the new algorithm to precisely predict key amino acid residues of proteins for phase separation (<a href="#before-you-begin">before you begin</a>).</p>
<h3 id="sec5.2">Potential solution</h3>
<p>Select more potential phase-separated proteins and predict more potential residues for phase separation capacity. Then considerable functional studies should be performed to validate the preciseness and reliability of the algorithm.</p>
<h3 id="sec5.3">Problem 2</h3>
<p>How to exclude other potential influence of these perturbations on protein functions except phase separation (<a href="#before-you-begin">before you begin</a>)?</p>
<h3 id="sec5.4">Potential solution</h3>
<p>Several biomolecular experiments should be performed to evaluate protein functions upon perturbations, such as ChIP for evaluating DNA binding change, Co-immunoprecipitation for evaluating protein interaction ability, and immunofluorescence for evaluating nuclear localization.</p>
<h3 id="sec5.5">Problem 3</h3>
<p>It is hard to control sonication conditions. Sometimes the bacterial pellet cannot be fully lysed; sometimes the proteins released from cell lysis are denatured by high temperature generated from sonication (Step 7).</p>
<h3 id="sec5.6">Potential solution</h3>
<p>First, to make sure full lysis, the amount of bacterial should not be too much. The OD value of bacterial solution should be kept to 0.6–0.8. Second, to avoid protein denature by sonication, the protein should be kept on ice during whole sonication process. Ice should be supplemented in the break during sonication.</p>
<h3 id="sec5.7">Problem 4</h3>
<p>Sometimes the western blotting result shows that the size of proteins purified from bacterial is inconsistent with what we expect (Step 11).</p>
<h3 id="sec5.8">Potential solution</h3>
<p>It may be caused by that the eukaryotic protein codon is not familiar by prokaryotic bacteria, and protein expression may be stop by bacterial. Therefore, codon optimization should be performed to meet the preference of prokaryotic bacteria.</p>
<h3 id="sec5.9">Problem 5</h3>
<p>It is difficult to capture the image of droplet fusion (Step 21).</p>
<h3 id="sec5.10">Potential solution</h3>
<p>To increase the rate of droplet fusion, the droplets should be kept moving. Therefore, the imaging capture should be completed early after putting the cover glass on the slide, since droplet moving slow down with time.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib2">Boija, A., Klein, I.A., Sabari, B.R., Dall'Agnese, A., Coffey, E.L., Zamudio, A.V., Li, C.H., Shrinivas, K., Manteiga, J.C., Hannett, N.M., et al. (2018). Transcription factors activate genes through the phase-separation capacity of their activation domains. Cell <i>175</i>, 1842-1855 e1816. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00593-1/sref2">View at publisher</a></p>
<p id="bib4">Liu, X., Shen, J., Xie, L., Wei, Z., Wong, C., Li, Y., Zheng, X., Li, P., and Song, Y. (2020). Mitotic implantation of the transcription factor prospero via phase separation drives terminal neuronal differentiation. Dev. Cell <i>52</i>, 277-293 e278. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00593-1/sref4">View at publisher</a></p>
<p id="bib8">Sabari, B.R., Dall'Agnese, A., Boija, A., Klein, I.A., Coffey, E.L., Shrinivas, K., Abraham, B.J., Hannett, N.M., Zamudio, A.V., Manteiga, J.C., et al. (2018). Coactivator condensation at super-enhancers links phase separation and gene control. Science <i>361</i>, eaar3958. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00593-1/sref8">View at publisher</a></p>
<p id="bib9">Takahashi, K. and Yamanaka, S. (2006). Induction of pluripotent stem cells from mouse embryonic and adult fibroblast cultures by defined factors. Cell <i>126</i>, 663-676. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00593-1/sref9">View at publisher</a></p>
<p id="bib10">Wang, J., Yu, H., Ma, Q., Zeng, P., Wu, D., Hou, Y., Liu, X., Jia, L., Sun, J., Chen, Y., et al. (2021). Phase separation of OCT4 controls TAD reorganization to promote cell fate transitions. Cell Stem Cell. . <a class="external-link" href="https://doi.org/10.1016/j.stem.2021.04.023">https://doi.org/10.1016/j.stem.2021.04.023</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Junjun Ding (<a href="mailto:dingjunj@mail.sysu.edu.cn">dingjunj@mail.sysu.edu.cn</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate any unique reagents.</p>
<h4>Data and code availability</h4>
<p>The accession number for HiC data reported in this paper is Sequence Read Archive (SRA): PRJNA650173 and PRJNA715050.</p>
<h3>Acknowledgments</h3>
<p>This research was funded by grants from the <a href="https://doi.org/10.13039/501100013290">National Key Research and Development Program</a> (2017YFA0102800), the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (grant nos. 31970811, 31771639, and 32170798), the Frontier Research Program of Bioland Laboratory (Guangzhou Regenerative Medicine and Health Guangdong Laboratory) (2018GZR110105007), the <a href="https://doi.org/10.13039/100012541">Guangdong Innovative and Entrepreneurial Research Team Program</a> (2016ZT06S029) to J.D., the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (grant no. 81703086) to J.W., and the <a href="https://doi.org/10.13039/100014717">National Natural Science Foundation of China</a> (grant no. 32100927) and <a href="https://doi.org/10.13039/501100004829">Science and Technology Department of Sichuan Province</a> (no. 2021YFS0158) to H.Y.</p>
<h3>Author contributions</h3>
<p>J.D. conceptualized and supervised the project. Q.M., J.W., H.Y., J.S., and Y.Y. developed, wrote, and edited the protocol. Q.M., J.W., J.C., Q.T., and P.F. performed the experiments. J.S. predicted key residues for phase separation.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>