<article>
<section>
<h1 id="header">Ultra-long DNA molecule isolation from plant nuclei for ultra-long read genome sequencing</h1>
<p><time datetime="2021-02-19">Published: February 19, 2021</time></p>
<p>Dessireé Zerpa-Catanho,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Xiaodan Zhang,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Jinjin Song,<sup><a href="#aff1">1</a></sup> Alvaro G. Hernandez,<sup><a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor1">*</a></sup> and Ray Ming<sup><a href="#aff1">1</a>,<a href="#fn3">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Plant Biology, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</p>
<p id="aff2"><sup>2</sup>Roy J. Carver Biotechnology Center, University of Illinois at Urbana-Champaign, Urbana, IL 61801, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:aghernan@illinois.edu">aghernan@illinois.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:rayming@illinois.edu">rayming@illinois.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100343">10.1016/j.xpro.2021.100343</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Long and ultra-long read DNA sequencing technologies require high molecular weight DNA with high quality and sufficient quantity, which could be challenging to obtain from recalcitrant plant tissues. We describe a protocol to isolate ultra-long DNA from 12 species for ultra-long read genome sequencing. A suitable nuclei lysis buffer is critical for DNA quality and yield. This protocol will enable individual labs to isolate high molecular weight DNA at a rapid pace with low cost from a variety of plant species.</p>
<p>For complete information on the use and execution of this protocol, please refer to: <a href="#bib5">Zhang et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A protocol for ultra-long DNA isolation from plant tissues for genome sequencing</li>
<li>Optimized isolation and lysis of nuclei from plant leaf tissues</li>
<li>Optimized purification of DNA samples to achieve high quality</li>
<li>Production of high quality ultra-long DNA samples at low cost</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Our protocol combines initial steps for preparing megabase (Mb) genomic DNA from plants for bacterial artificial chromosome library preparation (<a href="#bib2">Luo and Wing, 2003</a>) and next-generation sequencing genomic DNA extraction (<a href="#bib1">Healey et al., 2014</a>) from plant nuclei, similar to <a href="#bib3">Peterson et al. (1997)</a> and <a href="#bib4">Rana et al. (2019)</a>, with DNA purification steps using QIAGEN Genomic Tip (according to the manufacturer instructions). A detailed overview of the entire protocol is shown in <a href="#fig1">Figures 1</a>, <a href="#fig2">2</a>, and <a href="#fig3">3</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic overview of the plant nuclei isolation step</div>
</figcaption>
</figure>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Schematic overview of the DNA extraction step</div>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Schematic overview of DNA purification and the quality control step</div>
</figcaption>
</figure>
<p>This protocol allows high molecular weight DNA extraction (fragments between 50 and 200 kb) from nuclei isolated from 5 grams of plant leaves. High molecular weight DNA extracted using this protocol has been used to produce high quality sequencing libraries for Oxford Nanopore Technologies with yields ranging from 1–17 Gb per flow cell and producing reads with lengths ranging between 1,018 and 327,925 bp. However, high molecular weight DNA extracted using this protocol could be used for other purposes.</p>
<p>Before beginning, researchers need to prepare stock and working solutions.</p>
<h3 id="sec1.1">Preparation of stock solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–2 h</div>
<ol>
<li>1 M Tris-HCl. To prepare 1 L:
<ol type="a">
<li>Add 121.1 g Tris-Base to 800 mL water.</li>
<li>Dissolve and adjust the pH to 8.0 with HCl.</li>
<li>Adjust the volume to 1 L with water.</li>
<li>Autoclave the solution at 121°C for 30 min.</li>
<li>Store the solution at 20°C–25°C until use.</li>
</ol>
</li>
<li>0.5 M EDTA. To prepare 1 L:
<ol type="a">
<li>Add 186.2 g EDTA to 800 mL water.</li>
<li>Stir and adjust the pH with NaOH. EDTA will be completely dissolved when the pH reaches 8.0.</li>
<li>Adjust the volume to 1 L with water.</li>
<li>Autoclave the solution at 121°C for 30 min.</li>
<li>Store the solution at 20°C–25°C until use.</li>
</ol>
</li>
<li>5 M NaCl. To prepare 150 mL:
<ol type="a">
<li>Add 43.83 g NaCl to 80 mL water.</li>
<li>Stir and dissolve NaCl completely.</li>
<li>Adjust the volume to 150 mL with water.</li>
<li>Autoclave the solution at 121°C for 30 min.</li>
<li>Store the solution at 20°C–25°C until use.</li>
</ol>
</li>
<li>1 M spermidine. To prepare 10 mL:
<ol type="a">
<li>Add 2.54 g spermidine to 10 mL autoclaved water.</li>
<li>Store the solution at -20°C for up to 2 months.</li>
</ol>
</li>
<li>1 M spermine. To prepare 10 mL:
<ol type="a">
<li>Add 3.48 g to 10 mL autoclaved water.</li>
<li>Store the solution at -20°C for up to 2 months.</li>
</ol>
</li>
<li>2% SDS. To prepare 1 L:
<ol type="a">
<li>Add 20 g SDS to 800 mL water.</li>
<li>Stir and dissolve SDS completely.</li>
<li>Adjust the volume to 1 L with water.</li>
<li>Autoclave the solution at 121°C for 30 min.</li>
<li>Store the solution at 20°C–25°C until use.</li>
</ol>
</li>
<li>100 mM Tris-HCl saturated phenol. To prepare 50 mL:
<ol type="a">
<li>Add 50 mL of 0.5 M Tris-HCl (pH 8.0) to 50 mL liquified phenol.</li>
<li>Stir for 15 min and turn off the stirrer to allow the two phases to separate.</li>
<li>Aspirate the upper aqueous phase.</li>
<li>Add 50 mL of 0.1 M Tris-HCl (pH 8.0) to the organic phase.</li>
<li>Stir for 15 min and turn off the stirrer to allow the two phases to separate again.</li>
<li>Aspirate the upper aqueous phase.</li>
<li>Measure the pH of the phenolic phase with pH paper.
<ol type="i">
<li>If pH &gt; 7.8, the phenol is equilibrated.</li>
<li>If pH &lt; 7.8, repeat steps d, e, f, and g.</li>
</ol>
</li>
<li>Add 5 mL of 0.1 M Tris-HCl (pH 8.0) containing 0.2% β-mercaptoethanol to the phenolic phase.</li>
<li>Store the saturated phenol at 4°C for up to a month.</li>
</ol>
</li>
</ol>
<div class="caution">
<span class="caution-title">Caution:</span> Wear appropriate personal protective equipment (nitrile gloves, safety googles, lab coat, closed-toe shoes and long pants) when handling phenol and β-mercaptoethanol. All solutions containing phenol and/or β-mercaptoethanol should be handled inside a chemical hood.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Distilled ultra-pure water must be used when water is mentioned.</div>
<h3 id="sec1.2">Preparation of working solutions</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1.5–2 h</div>
<ol start="8">
<li>Proteinase K solution (1 mg/mL). To prepare 10 mL:
<ol type="a">
<li>Dissolve 10 mg of proteinase K in 10 mL autoclaved water.</li>
<li>Aliquot the solution into 1.5 mL Eppendorf tubes.</li>
<li>Store at -20°C until use. 10 mL is enough for 33 samples.</li>
</ol>
</li>
<li>RNase A solution (10 mg/mL). To prepare 1 mL:
<ol type="a">
<li>Dissolve 10 mg of RNase A in 1 mL autoclaved water.</li>
<li>Aliquot the solution into 1.5 mL Eppendorf tubes.</li>
<li>Store at -20°C until use. 1 mL is enough for 200 samples.</li>
</ol>
</li>
<li>Chloroform: isoamyl alcohol (24:1). To prepare 25 mL:
<ol type="a">
<li>Mix 24 mL of chloroform with 1 mL isoamyl alcohol.</li>
<li>Store at 20°C–25°C inside a chemical fume hood until use.</li>
</ol>
</li>
<li>Phenol: chloroform: isoamyl alcohol (25:24:1). To prepare 50 mL:
<ol type="a">
<li>Mix 25 mL 100 mM Tris-HCl saturated phenol, 24 mL of chloroform and 1 mL of isoamyl alcohol (24:1).</li>
<li>Store at 4°C for up to a month.</li>
</ol>
</li>
<li>Cold isopropyl alcohol (isopropanol): Aliquot isopropanol and store it at -20°C before use.</li>
<li>Cold 70% ethanol. To prepare 500 mL:
<ol type="a">
<li>Add 350 mL of ethanol to 150 mL autoclaved water.</li>
<li>Aliquot 70% ethanol and store it at -20°C before use.</li>
</ol>
</li>
<li>Prepare NIB, NIBT, NIBM and DNA Extraction Buffer according to their recipe (see <a href="#materials-and-equipment">Materials and equipment</a> section).</li>
</ol>
<div class="caution">
<span class="caution-title">Caution:</span> Wear appropriate personal protective equipment (nitrile gloves, safety googles, lab coat, closed-toe shoes, and long pants) when handling phenol, β-mercaptoethanol, chloroform, or isoamyl alcohol. All solutions containing phenol, β-mercaptoethanol, chloroform, or isoamyl alcohol should be handled inside a chemical hood.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Distilled ultra-pure water must be used when water is mentioned.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Biological samples</td>
</tr>
<tr>
<td>
<i>Carica papaya</i> (var. “SunUp”)</td>
<td>HARC, Kunia, Hawaii</td>
<td>SunUp</td>
</tr>
<tr>
<td><i>Jacaratia spinosa</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>N92-20</td>
</tr>
<tr>
<td><i>Vasconcellea microcarpa</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>UH ACC #100</td>
</tr>
<tr>
<td><i>Vasconcellea stipulata</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>HCAR177</td>
</tr>
<tr>
<td><i>Vasconcellea cauliflora</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>N12-17</td>
</tr>
<tr>
<td><i>Vasconcellea monoica</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>HCAR171</td>
</tr>
<tr>
<td><i>Vasconcellea goudotiana</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>HCAR167</td>
</tr>
<tr>
<td><i>Vasconcellea quercifolia</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>HCAR175</td>
</tr>
<tr>
<td><i>Jarilla caudata</i></td>
<td>USDA, ARS, Hilo, Hawaii</td>
<td>N12-21</td>
</tr>
<tr>
<td>
<i>Saccharum spontaneum</i> L.</td>
<td>HARC, Kunia, Hawaii</td>
<td>SES-208</td>
</tr>
<tr>
<td><i>Saccharum officinarum</i></td>
<td>HARC, Kunia, Hawaii</td>
<td>LA Purple</td>
</tr>
<tr>
<td>
<i>Ananas comosus</i> (var. “MD-2”)</td>
<td>UIUC, Urbana, Illinois</td>
<td>MD-2</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tris-base</td>
<td>Fisher Scientific</td>
<td>Cat#BP152-1</td>
</tr>
<tr>
<td>HCl</td>
<td>Fisher Scientific</td>
<td>Cat#A144-212</td>
</tr>
<tr>
<td>EDTA</td>
<td>VWR Chemicals BDH</td>
<td>Cat#BDH4616-500G</td>
</tr>
<tr>
<td>KCl</td>
<td>Fisher Scientific</td>
<td>Cat#P217-500</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Sigma-Aldrich</td>
<td>Cat#S0389-5KG</td>
</tr>
<tr>
<td>Spermidine</td>
<td>Sigma-Aldrich</td>
<td>Cat#S2626-5G</td>
</tr>
<tr>
<td>Spermine</td>
<td>Sigma-Aldrich</td>
<td>Cat#S4264-5G</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>Sigma-Aldrich</td>
<td>Cat#T8787-250ML</td>
</tr>
<tr>
<td>β-Mercaptoethanol</td>
<td>Calbiochem, Millipore-Sigma</td>
<td>Cat#444203</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Sigma-Aldrich</td>
<td>Cat#P2308-500MG</td>
</tr>
<tr>
<td>RNase A</td>
<td>Roche, Millipore-Sigma</td>
<td>Cat#10-109-169-001</td>
</tr>
<tr>
<td>NaCl</td>
<td>Fisher Scientific</td>
<td>Cat#S271-500</td>
</tr>
<tr>
<td>SDS</td>
<td>Fisher Scientific</td>
<td>Cat#BP166-500</td>
</tr>
<tr>
<td>PVP-40</td>
<td>Fisher Scientific</td>
<td>Cat#BP431-100</td>
</tr>
<tr>
<td>NaHSO<sub>3</sub>
</td>
<td>EMD Millipore</td>
<td>Cat#SX0344-1</td>
</tr>
<tr>
<td>Phenol</td>
<td>Fisher Scientific</td>
<td>Cat#A931l-1</td>
</tr>
<tr>
<td>Chloroform</td>
<td>Macron Chemicals</td>
<td>Cat#4440-08</td>
</tr>
<tr>
<td>Isoamyl alcohol</td>
<td>Fisher Scientific</td>
<td>Cat#A393-500</td>
</tr>
<tr>
<td>Isopropanol</td>
<td>Fisher Scientific</td>
<td>Cat#A416-4</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Decon Laboratories</td>
<td>Cat#2701</td>
</tr>
<tr>
<td>Buffer G2</td>
<td>QIAGEN</td>
<td>Cat#1014636</td>
</tr>
<tr>
<td>Buffer QBT</td>
<td>QIAGEN</td>
<td>Cat#1905</td>
</tr>
<tr>
<td>Buffer QC</td>
<td>QIAGEN</td>
<td>Cat#19055</td>
</tr>
<tr>
<td>Buffer QF</td>
<td>QIAGEN</td>
<td>Cat#19056</td>
</tr>
<tr>
<td>Buffer AE</td>
<td>QIAGEN</td>
<td>Cat#19077</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>QIAGEN Genomic Tip 20/G</td>
<td>QIAGEN</td>
<td>Cat#10223</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Scissors</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Balance</td>
<td>Mettler Toledo</td>
<td>N/A</td>
</tr>
<tr>
<td>Liquid nitrogen</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Mortar and pestles</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Ice buckets with ice</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>250 mL flasks</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>1 L beakers</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Magnetic stirrer</td>
<td>VWR</td>
<td>Cat#12620-974</td>
</tr>
<tr>
<td>Stir bars</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Plastic funnels</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Chemical fume hood</td>
<td>Kewaunnee Scientific Corporation</td>
<td>N/A</td>
</tr>
<tr>
<td>Cheesecloth</td>
<td>Fisher Scientific</td>
<td>Cat#06665-28</td>
</tr>
<tr>
<td>Miracloth</td>
<td>EMD Millipore</td>
<td>Cat#475855-1R</td>
</tr>
<tr>
<td>50 mL Falcon tubes</td>
<td>Corning</td>
<td>Cat#430829</td>
</tr>
<tr>
<td>15 mL tubes</td>
<td>Corning</td>
<td>Cat#430790</td>
</tr>
<tr>
<td>Refrigerated centrifuge for 50 mL tubes</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>Refrigerated centrifuge for 15 mL tubes</td>
<td>Beckman Coulter</td>
<td>N/A</td>
</tr>
<tr>
<td>Incubator with shaker</td>
<td>New Brunswick Scientific</td>
<td>N/A</td>
</tr>
<tr>
<td>1.5 mL Eppendorf tubes</td>
<td>Fisher Brand, Fisher Scientific</td>
<td>Cat#05-408-129</td>
</tr>
<tr>
<td>Bemis PM999 Parafilm</td>
<td>Fisher Scientific</td>
<td>Cat#13-374-12</td>
</tr>
<tr>
<td>Standard aluminum foil</td>
<td>Reynolds Wrap</td>
<td>N/A</td>
</tr>
<tr>
<td>1 mL micropipette tips</td>
<td>Fisher Brand, Fisher Scientific</td>
<td>Cat#02-707-400</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Biological materials</h3>
<p>To test this protocol, we used <i>Carica papaya</i> leaves from the commercial variety “SunUp”. Besides <i>C. papaya</i>, we have successfully used this approach with different plant species of the Caricaceae family for Oxford Nanopore Technologies (ONT) DNA sequencing: <i>Jacaratia spinosa</i>, <i>Vasconcellea microcarpa</i>, <i>Vasconcellea stipulata</i>, <i>Vasconcellea cauliflora</i>, <i>Vasconcellea monoica</i>, and <i>Jarilla caudata</i>. We have tried the protocol with other plant species without sequencing the DNA output: <i>Saccharum spontaneum L.</i> (SES-208 accession, obtained from Dr. Ming-Li Wang, Hawaii Agriculture Research Center), <i>Saccharum officinarum</i> (LA purple accession, obtained from Dr. Erik Sacks, University of Illinois at Urbana-Champaign), <i>Ananas comosus</i> (MD-2 variety, obtained from commercial fruits), <i>Vasconcellea goudotiana</i>, and <i>Vasconcellea quercifolia</i>. Caricaceae plant samples used for this study were obtained from Dr. Ming-Li Wang and Tyler Jones at Hawaii Agriculture Research Center and from Tracie Matsumoto at the Agricultural Research Service of the U.S. Department of Agriculture.</p>
<h3 id="sec2.2">General supplies</h3>
<p>Liquid nitrogen. Clean mortar and pestles (1 pair per sample). Autoclaved 250 mL flasks (3 flasks per sample). Autoclaved 1 mL micropipette tip boxes. Autoclaved pieces of cheesecloth. Autoclaved pieces of Miracloth. 1 L beakers (1 per sample). Clean funnels (2 per sample). Ice bucket filled with ice. 1 mL micropipettes. Scissors.</p>
<div class="caution">
<span class="caution-title">Caution:</span> Wear appropriate personal protective equipment (cryo-gloves, safety googles, lab coat, closed-toe shoes, and long pants) when handling liquid nitrogen.</div>
<h3 id="sec2.3">Equipment</h3>
<p>Balance (PB602-S, Mettler Toledo). Magnetic stirrer (Dylastir Stirrer, VWR) and clean stir bars (2 per sample). Centrifuge for nuclei isolation in 50 mL tubes (Allegra 25-R centrifuge, Beckman Coulter). Centrifuge for DNA extraction in 15 mL tubes (Allegra X-15R centrifuge, Beckman Coulter). Incubator with shaker (Innova 4200, New Brunswick Scientific). Chemical fume hood (PartF 3337000, Kewaunnee Scientific Corporation).</p>
<h3 id="sec2.4">Buffer recipes</h3>
<table id="undtbl1">
<caption>
<b>Nuclei Isolation Buffer (NIB)</b>: Store the solution at 4°C for up to a month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH 8.0</td>
<td>10 mM</td>
<td>10 mL</td>
</tr>
<tr>
<td>0.5 M EDTA, pH 8.0</td>
<td>10 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>KCl</td>
<td>100 mM</td>
<td>7.46 g</td>
</tr>
<tr>
<td>Sucrose</td>
<td>0.5 M</td>
<td>171.20 g</td>
</tr>
<tr>
<td>1 M Spermidine</td>
<td>4 mM</td>
<td>4 mL</td>
</tr>
<tr>
<td>1 M Spermine</td>
<td>1 mM</td>
<td>1 mL</td>
</tr>
<tr>
<td>Autoclaved water</td>
<td>n/a</td>
<td>965 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>
<b>NIBT (NIB supplemented with Triton X-100)</b>: Store the solution at 4°C for up to a month.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NIB</td>
<td>n/a</td>
<td>45 mL</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>10%</td>
<td>5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>
<b>NIBM (NIB supplemented with β-mercaptoethanol</b>): Store the solution at 4°C for up to a month.</caption>
<thead>
<tr>
<th>NIBM</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NIB</td>
<td>n/a</td>
<td>399.6 mL</td>
</tr>
<tr>
<td>β-mercaptoethanol</td>
<td>0.1%</td>
<td>400 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>400 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>
<b>DNA Extraction Buffer</b>: Store the solution at 20°C–25°C until use.</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1 M Tris-HCl, pH 8.0</td>
<td>100 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>0.5 M EDTA, pH 8.0</td>
<td>50 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>5 M NaCl</td>
<td>500 mM</td>
<td>100 mL</td>
</tr>
<tr>
<td>2% SDS</td>
<td>1.25%</td>
<td>625 mL</td>
</tr>
<tr>
<td>Water</td>
<td>n/a</td>
<td>75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1,000 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave the solution at 121°C for 30 min. Just before use, add 20 g PVP-40 and 3.8 g NaHSO<sub>3</sub> to the DNA Extraction Buffer.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> QIAGEN Genomic Tip 20/G could be substitute for QIAGEN Genomic Tip 100/G or QIAGEN Genomic Tip 500/G.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Nuclei isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step accomplishes the isolation of nuclei from ground plant leaves. Fresh or frozen (stored at -80°C) plant leaf tissue can be used as starting material. <a href="#fig1">Figure 1</a> shows a schematic summary of this step.</p>
<ol>
<li>Collect enough young leaves of the selected plant (at least 5 grams) and process it immediately or flash freeze it in liquid nitrogen and store it at -80°C until use.</li>
<li>In the chemical fume hood, set up a magnetic stirrer with an empty 1 L beaker on top of the stirrer (<a href="#fig4">Figure 4</a>).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Correct settings for nuclei isolation using the protocol</div>
<p>(A) Equipment that needs to be placed inside the chemical fume hood.</p>
<p>(B) Ice used during nuclei isolation to keep the sample at 4°C.</p>
<p>(C) Example of the expected size of a nuclei pellet.</p>
</figcaption>
</figure>
</li>
<li>Weight 5 grams of leaf tissue and quickly transfer to a pre-cooled mortar.</li>
<li>Grind 5 grams of frozen leaf tissue in liquid nitrogen with a mortar and pestle until the fine power can be seen.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Always keep enough liquid nitrogen in the mortar to prevent tissue from thawing during grinding.</div>
<ol start="5">
<li>Transfer the grinded tissue to a 250 mL flask containing 100 mL of cold NIBM, add a stir bar, and seal the flask with aluminum foil. Transfer the flask to the 1 L beaker on top of the magnetic stirrer inside the chemical fume hood and add ice to the beaker (<a href="#fig4">Figure 4</a>A and 4B).</li>
</ol>
<div class="caution">
<span class="caution-title">Caution:</span> This step as well as all the following steps should be performed inside a chemical fume hood.</div>
<ol start="6">
<li>Keep the flask on ice for 15 min with gentle stirring (set the stirrer speed to less than 85 rpm).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid using high stirrer speeds because it could break the nuclei and you will lose nuclear DNA. If the ice melts, replace the ice bath and continue stirring. Low temperature and gentle stirring preserve the integrity of nuclei and guarantee higher DNA concentration.</div>
<ol start="7">
<li>Filter the homogenate through four layers of cheesecloth and one layer of Miracloth and collect the filtered solution in a new 250 mL flask. Squeeze the pellet to allow maximum recovery of nuclei and rinse the flask with 5 mL of NIBM.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Change your gloves between samples and before moving on to the next steps.</div>
<ol start="8">
<li>Filter the solution again through one layer of Miracloth and collect the nuclei solution in a flask containing a stirrer bar.</li>
<li>Add 5 mL of cold NIBT to the filtered nuclei solution, seal the flask with aluminum foil and keep the flask on ice for 15 min with gentle stirring .(set the stirrer speed to less than 85 rpm).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid using high stirrer speeds because it could break the nuclei and you will lose nuclear DNA. If the ice melts, replace the ice bath and continue stirring. Low temperature and gentle stirring preserve the integrity of nuclei and guarantee higher DNA concentration.</div>
<ol start="10">
<li>Transfer the mixture into 50 mL Falcon tubes (2 per sample; <a href="#fig4">Figure 4</a>C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To prevent the contents from spilling during centrifugation steps, parafilm can be used to secure the cap of the Falcon tubes.</div>
<ol start="11">
<li>Centrifuge the Falcon tubes at 2,400 × <i>g</i> at 4°C for 15 min. If you do not see a white pellet in the bottom, see <a href="#troubleshooting">Troubleshooting 1</a>.</li>
<li>Carefully discard most of the supernatant without disturbing the pellet (∼ 35 to 40 mL) and resuspend the pellet in the remaining buffer (∼15 mL) by gently tapping the tubes.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid vortexing or vigorously shaking the tubes because it could break the nuclei and you will lose nuclear DNA.</div>
<ol start="13">
<li>Adjust the volumes of both tubes to 45 mL with fresh NIBM and gently mix by inversion.</li>
<li>Centrifuge the Falcon tubes at 2,400 × <i>g</i> at 4°C for 15 min. <a href="#troubleshooting">Troubleshooting 1</a>.</li>
<li>Repeat step 12.</li>
<li>Combine the contents of both Falcon tubes in a single Falcon tube and adjust the volumes of the tubes to 45 mL with fresh NIBM.</li>
<li>Centrifuge the Falcon tube at 2,400 × <i>g</i> at 4°C for 15 min. <a href="#troubleshooting">Troubleshooting 1</a>.</li>
<li>Repeat the washing steps (steps 12, 13, and 14) one more time for a total of three washes. Gently discard all the supernatant by pouring without disturbing the pellet (∼ 50 mL) and resuspend the pellet in 1.7 mL warm (65°C) DNA Extraction Buffer. <a href="#troubleshooting">Troubleshooting 2</a>.</li>
<li>Add 300 μL of proteinase K (1 mg/mL) and mix gently by inversion.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Avoid vortexing or vigorously shaking the tubes because it could shear the DNA.</div>
<h3 id="sec3.2">DNA extraction</h3>
<div class="timing">
<span class="timing-title">Timing:</span> hands-on time 4 h</div>
<p>This step accomplishes the extraction of DNA from lysed nuclei using phenol:chloroform:isoamyl alcohol to remove proteins, RNase A to remove RNA, and isopropanol or NaCl and ethanol to precipitate DNA. <a href="#fig2">Figure 2</a> shows a schematic summary of this step.</p>
<ol start="20">
<li>Incubate the nuclei solution at 60°C in an incubator with gentle shaking (250 rpm) for at least 18 h. Longer incubation time can be used to increase DNA yield (18 h to 20 h). <a href="#troubleshooting">Troubleshooting 2</a>.</li>
<li>Centrifuge the Falcon tube at 5,000 × <i>g</i> at 4°C for 5 min to collect all the liquid on the walls of the tube at the bottom of the tube.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Cut the pipette tip or use wide bore tip to avoid shearing the DNA during transfers. This should be applied to steps 22, 24, 25, and 27.</div>
<ol start="22">
<li>Transfer supernatant into one 15 mL tube (expect 2 mL).</li>
<li>Add 2 mL of phenol: chloroform: isoamyl alcohol (25:24:1), gently mix the content by inversion (at least 5 min) and centrifuge at 5,000 × <i>g</i> at 4°C for 10 min.</li>
</ol>
<div class="caution">
<span class="caution-title">Caution:</span> Wear adequate personal protective equipment and work inside a chemical fume hood.</div>
<ol start="24">
<li>Transfer the top transparent aqueous phase (2 mL) to a new 15 mL tube and repeat step 22 using an equal volume (2 mL) of chloroform: isoamyl alcohol (24:1).</li>
<li>Transfer the aqueous phase to a new 15 mL tube and add 5 μL of RNase A (at 10 mg/mL) to the solution and incubate at 37°C for 15 min.</li>
<li>Add an equal volume of chloroform: isoamyl alcohol (24:1), gently mix the contents by inversion (at least 5 min) and centrifuge at 5,000 × <i>g</i> at 4°C for 10 min.</li>
<li>Transfer the aqueous phase to a new 15 mL tube and precipitate the DNA as follows.</li>
<li>For DNA precipitation you can follow one of two different protocols:
<ol type="a">
<li>Add an equal volume of cold (-20°C) isopropanol and mix by inversion. Incubate the tubes on ice for at least 1 h and centrifuge at 5,000 × <i>g</i> at 4°C for 10 min to pellet the DNA. <a href="#troubleshooting">Troubleshooting 3</a>.</li>
<li>Add ½ volume of 5 M NaCl to the sample and mix by inversion. Then, add 3 volumes of cold (-20°C) 95% ethanol and mix by inversion. Incubate the tubes at -20°C for 1 h (do not use more than 1 h because NaCl could precipitate and contaminate the sample) and centrifuge at 5,000 × <i>g</i> at 4°C for 10 min to pellet the DNA. <a href="#troubleshooting">Troubleshooting 3</a>.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After this step, handle the tubes containing DNA on ice, unless indicated otherwise.</div>
<ol start="29">
<li>Carefully discard the supernatant without disturbing the DNA pellet.</li>
<li>Wash the DNA pellet with 4 mL of cold (-20°C) 70% ethanol and centrifuge at 5,000 × <i>g</i> at 4°C for 10 min.</li>
<li>Discard the ethanol and air dry the pellet for 30 min (up to 1 h) at 20°C–25°C. If extracting DNA from nuclei of sugarcane samples, proceed to step 44 (No need for further purification for sugarcane DNA samples).</li>
<li>To the DNA pellet, add 2 mL of G2 Buffer (QIAGEN Cat#1014636). Incubate the mixture at 50°C for 15 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Do not pipette or vortex to solubilize the pellet faster, it will solubilize by itself.</div>
<h3 id="sec3.3">DNA purification and quality control</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 h</div>
<p>This step allows to improve the A260/A230 ratio using a QIAGEN Genomic Tip 20/G column. Other kits for genomic DNA purification can be used (GenFind and Zymo columns) as long as you validate that they yield similar results. This step allows to check the concentration and quality of DNA before sequencing. <a href="#fig3">Figure 3</a> shows a schematic overview of this step.</p>
<ol start="33">
<li>Equilibrate the QIAGEN Genomic Tip 20/G column using 1 mL Buffer QBT (QIAGEN Cat#1905).</li>
<li>Pour the extracted DNA sample through the equilibrated column.</li>
<li>Once the liquid has passed through the column, add 1 mL of Buffer QC (QIAGEN Cat#19055) to wash the column.</li>
<li>Repeat step 35 two more times for a total of 3 wash steps.</li>
<li>After washing, place the column over a clean 15 mL tube and elute the genomic DNA with 1 mL of pre-warmed (55°C) Buffer QF (QIAGEN Cat#19056). After the liquid has passed through the column, add an additional 1 mL of pre-warmed (55°C) Buffer QF.</li>
<li>Allow the eluate to cool down at 20°C–25°C.</li>
<li>Add 1.4 mL of cold (-20°C) isopropanol to the eluted DNA and mix gently by inversion (no more than 10 times).</li>
<li>Incubate the mixture at -20°C for 1 h (up to 12 h).</li>
<li>Centrifuge the tube at 3,500 × <i>g</i> at 4°C for 30 min and discard the supernatant without disturbing the DNA pellet.</li>
<li>Add 4 mL of cold (-20°C) 70% ethanol and mix by inversion (no more than 10 times).</li>
<li>Centrifuge the tube at 3,500 × <i>g</i> at 4°C for 10 min and discard the supernatant without disturbing the DNA pellet.</li>
<li>Air dry the pellet for 30 min (up to 1 h) at 20°C–25°C.</li>
<li>Resuspend the DNA pellet in 50 μL Elution Buffer (10 mM Tris-HCl; pH 8.5), AE buffer (10 mM Tris-HCl; 0.5 mM EDTA; pH 9.0) or PCR grade water at 20°C–25°C, and store it at 4°C until sequencing. <a href="#troubleshooting">Troubleshooting 3, 4, or 5</a>.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not pipette or vortex to solubilize the pellet, because it could shear the DNA. Avoid freezing and unfreezing the samples before sequencing to prevent DNA shearing.</div>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> DNA can be stored at 4°C for 5 days before quality control.</div>
<ol start="46">
<li>Check the DNA quality by running 1 μL of DNA in a 1% agarose gel.</li>
<li>Check the DNA concentration with Nanodrop or using Qubit.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol allows the extraction of high molecular weight DNA at a reasonable estimated cost ($46.60 per sample). Details of plant species in which the protocol have been used are shown <a href="#tbl1">(Tables 1</a> and <a href="#tbl3">3</a>). Between 2 to 32 μg of high molecular weight DNA (with DNA fragments ranging between 50 and 200 kb) were obtained when using 5 grams of adult plant leaves as starting material (<a href="#fig5">Figure 5</a>). After ultra-long read DNA sequencing with Oxford Nanopore Technologies, sequencing yields ranged from 1 to 17 Gb per flow cell, with read lengths between 1,018 and 327,925 bp (<a href="#tbl2">Table 2</a>, <a href="#fig6">Figure 6</a>). As shown in <a href="#tbl2">Table 2</a> and <a href="#fig6">figure 6</a>, we successfully obtained long and ultra-long reads from high molecular weight DNA extracted from different plant species using this protocol. The obtained data will help with the assembly of these plant genomes.</p>
<table id="tbl1">
<caption>Table 1. Quality and quantity of DNA samples from different plant species extracted with this protocol and estimated by Nanodrop</caption>
<thead>
<tr>
<th>#</th>
<th>Species</th>
<th>Accession</th>
<th>DNA (ng/μL)</th>
<th>A<sub>260</sub>/A<sub>280</sub> value</th>
<th>A<sub>260</sub>/A<sub>230</sub> value</th>
<th>Purified with QIAGEN Genomic Tip</th>
<th>Sequenced</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td rowspan="3"><i>C. papaya</i></td>
<td>pML941</td>
<td>226.91</td>
<td>2.17</td>
<td>0.54</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>2</td>
<td>pML943</td>
<td>111.00</td>
<td>2.03</td>
<td>0.91</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>3</td>
<td>pML944</td>
<td>116.00</td>
<td>2.21</td>
<td>0.63</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>4</td>
<td><i>V. monoica</i></td>
<td>Monoica</td>
<td>150.00</td>
<td>2.11</td>
<td>0.59</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>5</td>
<td rowspan="2"><i>V. microcarpa</i></td>
<td>female</td>
<td>103.00</td>
<td>2.18</td>
<td>0.48</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>6</td>
<td>male</td>
<td>127.00</td>
<td>2.27</td>
<td>0.48</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>7</td>
<td rowspan="2"><i>J. spinosa</i></td>
<td>female</td>
<td>781.30</td>
<td>1.72</td>
<td>0.51</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>8</td>
<td>male</td>
<td>2,327.40</td>
<td>1.92</td>
<td>0.89</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>9</td>
<td rowspan="2"><i>V. stipulata</i></td>
<td>female</td>
<td>201.70</td>
<td>1.84</td>
<td>2.19</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>10</td>
<td>male</td>
<td>458.00</td>
<td>1.87</td>
<td>2.09</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>11</td>
<td rowspan="2"><i>V. cauliflora</i></td>
<td>female</td>
<td>674.00</td>
<td>1.82</td>
<td>2.11</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>12</td>
<td>male</td>
<td>413.00</td>
<td>1.86</td>
<td>2.15</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>13</td>
<td rowspan="2"><i>J. caudata</i></td>
<td>female</td>
<td>76.40</td>
<td>1.89</td>
<td>2.46</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>14</td>
<td>male</td>
<td>146.70</td>
<td>1.73</td>
<td>2.00</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>15</td>
<td rowspan="2"><i>V. stipulata</i></td>
<td>female</td>
<td>155.50</td>
<td>2.10</td>
<td>0.37</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>16</td>
<td>male</td>
<td>180.40</td>
<td>1.79</td>
<td>0.35</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>17</td>
<td rowspan="2"><i>V. cauliflora</i></td>
<td>female</td>
<td>90.00</td>
<td>2.13</td>
<td>0.49</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>18</td>
<td>male</td>
<td>237.20</td>
<td>2.09</td>
<td>0.58</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>19</td>
<td rowspan="2"><i>V. goudotiana</i></td>
<td>female</td>
<td>84.20</td>
<td>2.05</td>
<td>0.56</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>20</td>
<td>male</td>
<td>137.50</td>
<td>2.05</td>
<td>0.67</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>21</td>
<td rowspan="2"><i>V. quercifolia</i></td>
<td>female</td>
<td>66.50</td>
<td>2.16</td>
<td>0.50</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>22</td>
<td>male</td>
<td>99.00</td>
<td>2.29</td>
<td>0.39</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>23</td>
<td rowspan="2"><i>V. parviflora</i></td>
<td>female</td>
<td>74.00</td>
<td>2.57</td>
<td>0.23</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>24</td>
<td>male</td>
<td>63.90</td>
<td>2.38</td>
<td>0.30</td>
<td>no</td>
<td>no</td>
</tr>
<tr>
<td>25</td>
<td><i>A. comosus</i></td>
<td>MD2</td>
<td>99.70</td>
<td>1.64</td>
<td>0.69</td>
<td>no</td>
<td>no</td>
</tr>
</tbody>
</table>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Fragment size of DNA samples extracted with the protocol showing the presence of high molecular weight DNA when analyzed with a Fragment Analyzer (Agilent, CA)</div>
<p>Sample number of sequenced samples specified as in <a href="#tbl2">Table 2</a>.</p>
</figcaption>
</figure>
<table id="tbl2">
<caption>Table 2. Sequencing output of <i>Caricaceae</i> DNA samples extracted with the protocol</caption>
<thead>
<tr>
<th rowspan="2">#</th>
<th rowspan="2">Species</th>
<th rowspan="2">Accession</th>
<th rowspan="2">DNA total (μg)</th>
<th rowspan="2">Purification method after DNA extraction</th>
<th rowspan="2">Total bases (Mb)</th>
<th rowspan="2">Total reads (#)</th>
<th colspan="3">Read length (kb)</th>
</tr>
<tr>
<th>Max</th>
<th>Min</th>
<th>Mean</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td rowspan="4"><i>C. papaya</i></td>
<td rowspan="2">pML941</td>
<td rowspan="2">7.00</td>
<td>None</td>
<td>1,488</td>
<td>99,488</td>
<td>171.83</td>
<td>3.00</td>
<td>14.96</td>
</tr>
<tr>
<td>2</td>
<td>GenFind</td>
<td>6,048</td>
<td>512,295</td>
<td>192.35</td>
<td>3.00</td>
<td>11.81</td>
</tr>
<tr>
<td>3</td>
<td>pML943</td>
<td>10.43</td>
<td>GenFind</td>
<td>5,250</td>
<td>508,245</td>
<td>139.64</td>
<td>2.82</td>
<td>10.33</td>
</tr>
<tr>
<td>4</td>
<td>pML944</td>
<td>10.90</td>
<td>GenFind</td>
<td>6,808</td>
<td>670,521</td>
<td>147.82</td>
<td>2.84</td>
<td>10.15</td>
</tr>
<tr>
<td>5</td>
<td><i>V. monoica</i></td>
<td>Monoica</td>
<td>4.16</td>
<td>Zymo Large Construct</td>
<td>10,361</td>
<td>778,228</td>
<td>123.53</td>
<td>2.88</td>
<td>13.31</td>
</tr>
<tr>
<td>6</td>
<td rowspan="2"><i>V. microcarpa</i></td>
<td>female</td>
<td>4.46</td>
<td>Zymo Large Construct</td>
<td>7,364</td>
<td>569,062</td>
<td>107.62</td>
<td>2.92</td>
<td>12.94</td>
</tr>
<tr>
<td>7</td>
<td>male</td>
<td>5.29</td>
<td>Zymo Large Construct</td>
<td>14,016</td>
<td>1,042,128</td>
<td>130.81</td>
<td>2.93</td>
<td>13.45</td>
</tr>
<tr>
<td>8</td>
<td rowspan="2"><i>J. spinosa</i></td>
<td>female</td>
<td>6.36</td>
<td>Zymo Large Construct</td>
<td>16,979</td>
<td>1,700,484</td>
<td>83.65</td>
<td>2.85</td>
<td>9.99</td>
</tr>
<tr>
<td>9</td>
<td>male</td>
<td>21.24</td>
<td>Zymo Large Construct</td>
<td>12,271</td>
<td>1,247,404</td>
<td>80.37</td>
<td>2.86</td>
<td>9.84</td>
</tr>
<tr>
<td>10</td>
<td rowspan="2"><i>V. cauliflora</i></td>
<td>female</td>
<td>9.68</td>
<td>QIAGEN Genomic Tip</td>
<td>4,669</td>
<td>539,088</td>
<td>221.91</td>
<td>1.06</td>
<td>8.66</td>
</tr>
<tr>
<td>11</td>
<td>male</td>
<td>21.07</td>
<td>QIAGEN Genomic Tip</td>
<td>4,748</td>
<td>1,008,991</td>
<td>199.88</td>
<td>1.02</td>
<td>4.71</td>
</tr>
<tr>
<td>12</td>
<td rowspan="2"><i>J. caudata</i></td>
<td>female</td>
<td>32.35</td>
<td>QIAGEN Genomic Tip</td>
<td>3,005</td>
<td>644,245</td>
<td>92.10</td>
<td>1.05</td>
<td>4.66</td>
</tr>
<tr>
<td>13</td>
<td>male</td>
<td>19.00</td>
<td>QIAGEN Genomic Tip</td>
<td>1,023</td>
<td>440,277</td>
<td>327.93</td>
<td>1.03</td>
<td>2.32</td>
</tr>
<tr>
<td>14</td>
<td rowspan="2"><i>V. stipulata</i></td>
<td>female</td>
<td>2.06</td>
<td>QIAGEN Genomic Tip</td>
<td>3,789</td>
<td>543,323</td>
<td>122.54</td>
<td>1.06</td>
<td>6.97</td>
</tr>
<tr>
<td>15</td>
<td>male</td>
<td>3.96</td>
<td>QIAGEN Genomic Tip</td>
<td>4,389</td>
<td>672,481</td>
<td>133.51</td>
<td>1.04</td>
<td>6.53</td>
</tr>
</tbody>
</table>
<p class="table-legend">Non-purified sample had lower sequencing output (Total reads), while purified samples generated higher sequencing output. Different protocols for DNA purification were used before sequencing</p>
<table id="tbl3">
<caption>Table 3. Quantity and quality of papaya and sugarcane DNA samples extracted with the protocol and estimated by Nanodrop</caption>
<thead>
<tr>
<th>#</th>
<th>Species</th>
<th>Precipitation</th>
<th>Purified with QIAGEN Genomic Tip</th>
<th>DNA (ng/μL)<sup><a href="#tblfn1">a</a></sup>
</th>
<th>A<sub>260</sub>/A<sub>280</sub> value<sup><a href="#tblfn1">a</a></sup>
</th>
<th>A<sub>260</sub>/A<sub>230</sub> value<sup><a href="#tblfn1">a</a></sup>
</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td rowspan="4"><i>C. papaya</i></td>
<td>isopropanol</td>
<td>no</td>
<td>371.58 ± 7.50</td>
<td>1.84 ± 0.04</td>
<td>0.86 ± 0.16</td>
</tr>
<tr>
<td>2</td>
<td>NaCl + ethanol</td>
<td>no</td>
<td>322.02 ± 32.59</td>
<td>1.84 ± 0.04</td>
<td>0.82 ± 0.15</td>
</tr>
<tr>
<td>3</td>
<td>isopropanol</td>
<td>yes</td>
<td>187.79 ± 25.1</td>
<td>1.81 ± 0.02</td>
<td>2.10 ± 0.15</td>
</tr>
<tr>
<td>4</td>
<td>NaCl + ethanol</td>
<td>yes</td>
<td>140.04 ± 17.3</td>
<td>1.73 ± 0.01</td>
<td>1.91 ± 0.06</td>
</tr>
<tr>
<td>5</td>
<td><i>S. spontaneum L.</i></td>
<td>NaCl + ethanol</td>
<td>no</td>
<td>559.42 ± 254.76</td>
<td>1.83 ± 0.05</td>
<td>1.60 ±0.18</td>
</tr>
<tr>
<td>6</td>
<td><i>S. officinarum</i></td>
<td>NaCl + ethanol</td>
<td>no</td>
<td>492.03 ± 284.09</td>
<td>1.84 ± 0.03</td>
<td>1.61 ± 0.11</td>
</tr>
</tbody>
</table>
<p class="table-legend">The quality of DNA samples (A<sub>260</sub>/A<sub>230</sub>) depends on the plant species. Purification improved the quality of DNA samples.</p>
<p id="tblfn1" class="table-footnote"><sup>a</sup>Average of three replicates ± standard error.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/477-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Read length distribution of sequenced DNA samples extracted with this protocol for each plant species</div>
<p>The legend shows the corresponding sample number of sequenced samples shown in <a href="#tbl2">Table 2</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Some DNA samples (<i>J. spinosa</i> female and male) extracted with this protocol contained a significant amount of traces of DNA (fragments &lt; 50 kb) as shown in <a href="#fig3">Figure 3</a>. This indicates that DNA fragmentation occurred during extraction. The starting material for nuclei isolation of these samples was grounded leaf tissue that had been stored at -80°C for 4 or 5 years. No immediate access to fresh leaves from these plant species was possible. We suspect that storing conditions or potential melting of the tissue while 5 grams were weighted before nuclei isolation, could have contributed to fragmentation of the DNA samples. Therefore, we recommend the immediate use of fresh leaf tissue for nuclei isolation, over the use of ground frozen tissue stored at -80°C. For short-term storage of fresh tissue (1 week or less), we recommend storing fresh leaf tissue on ice (4°C) until extraction. For long-term storage of fresh tissue (more than a week), we recommend storing intact fresh leaf tissue (not ground) at -20°C or -80°C. If long-term storage is used, thawing should be avoided before grinding.</p>
<p>We noticed that DNA purity depends on the plant species, as shown when extracting sugarcane (<a href="#tbl3">Table 3</a>). When trying this protocol with plant species other than sugarcane and without adding a DNA purification step, the A<sub>260</sub>/A<sub>230</sub> ratios were low (less than 1.0) (<a href="#tbl1">Table 1</a> and <a href="#tbl3">Table 3</a>). When comparing A<sub>260</sub>/A<sub>230</sub> values among species (<a href="#tbl3">Table 3</a>), <i>S. spontaneum</i> samples precipitated with ethanol and without purification had significantly higher A<sub>260</sub>/A<sub>230</sub> values than papaya samples that were precipitated with ethanol (p-value = 0.0295, t-test). Furthermore, <i>S. officinarum</i> samples precipitated with ethanol and without purification had significantly higher A<sub>260</sub>/A<sub>230</sub> values than papaya samples that were precipitated with ethanol (p-value = 0.0129, t-test).</p>
<p>Without any purification step, the sequencing output of DNA extracted with this protocol was limited (<a href="#tbl2">Table 2</a>). For samples from some of the plant species (<i>Ananas comosus</i>, <i>Vasconcellea stipulata</i>, <i>Vasconcellea goudotiana</i>, <i>Vasconcellea quercifolia</i>, and <i>Vasconcellea cauliflora</i>), without a DNA purification step, A<sub>260</sub>/A<sub>230</sub> ratios were too low to use for sequencing (<a href="#tbl1">Table 1</a>), and other kits for genomic DNA purification tested by the sequencing center (GenFind and Zymo columns) did not work properly with these samples. Columns from the Zymo kit consistently got clogged. As a result, the samples were not sequenced. Therefore, a purification step with QIAGEN Genomic Tip is recommended over purification with other commercial kits and it is part of the protocol. As shown in <a href="#tbl1">Table 1</a>, when samples from <i>Vasconcellea stipulata</i> and <i>Vasconcellea cauliflora</i> were purified with QIAGEN Genomic Tip their A<sub>260</sub>/A<sub>230</sub> values improved (<a href="#tbl1">Table 1</a>) and reached acceptable values for library preparation and genome sequencing (<a href="#tbl2">Table 2</a>). The capacity of the QIAGEN Genomic Tip column could be changed (100/G or 500/G), but for the purposes of this paper obtaining between 2 to 5 μg of DNA was enough as starting material for sequencing. If the capacity of the QIAGEN Genomic Tip column is changed, buffer volumes should be adjusted accordingly (following the instructions of the manufacturer).</p>
<p>The purification step significantly reduced the amount of DNA obtained from <i>C. papaya</i> (<a href="#tbl3">Table 3</a>), but it was a necessary step to improve the A<sub>260</sub>/A<sub>230</sub> ratios of <i>Caricaceae</i> samples before sequencing. As shown in <a href="#tbl3">Table 3</a>, DNA extracted from papaya samples and precipitated with isopropanol had significantly higher DNA concentration (p-value = 0.0022, t-test), but significantly lower A<sub>260</sub>/A<sub>230</sub> values (p-value = 0.0052, t-test) than samples that were purified. DNA extracted from papaya samples and precipitated with ethanol also had significantly higher DNA concentration (p-value = 0.0079, t-test), but significantly lower A<sub>260</sub>/A<sub>230</sub> values (p-value = 0.0027, t-test) than samples that were purified.</p>
<p>DNA precipitation with isopropanol or NaCl and ethanol produced similar outputs (p-value &gt; 0.05, t-test) (<a href="#tbl3">Table 3</a>). Therefore, users can choose between both protocols for DNA precipitation.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Not enough nuclei isolated is a problem. If there is not nuclei pellet, or the pellet is not white, it means there is not enough nuclei. Possible causes for this problem are:</p>
<ol>
<li>An inadequate composition of the Nuclei Isolation Buffer for the plant species. Some plant species might contain enzymes (proteases) that digest proteins in the nuclei membrane causing lysis of nuclei membrane during nuclei isolation.</li>
<li>Not enough starting materials or tissues. Depending on the plant species, 5 grams of tissue might not produce enough nuclei.</li>
<li>Breaking nuclei by vigorously shaking the solution during the extraction or during nuclei resuspension in NIBM.</li>
</ol>
<h3 id="sec6.2">Potential solution</h3>
<ol>
<li>Changing the Nuclei isolation Buffer composition could help increase the nuclei output of certain plant species. For example, including protease inhibitors could prevent nuclei lysis occurring during the isolation process.</li>
<li>Increasing the amounts of tissues could increase the nuclei output. Add 10 mL of NIBM per 1 g of plant tissue.</li>
<li>Decreasing the stirrer speed during extraction of nuclei.</li>
<li>Using a clean paintbrush to resuspend the nuclei in the NIBM instead of tapping the tubes during the washing steps.</li>
</ol>
<h3 id="sec6.3">Problem 2</h3>
<p>Low DNA concentration is a problem. The expected total DNA amount should be more than 2 μg, if the total amount is less than that, the concentration is considered low for sequencing with Oxford Nanopore Technologies. Possible causes of this problem are:</p>
<ol>
<li>Nuclei lysis is not effective, or DNA Extraction Buffer is inadequate for the plant species.</li>
<li>Incubation time for nuclei lysis is not enough.</li>
<li>Precipitation with NaCl and ethanol is not efficient. If there is not nuclei pellet, or the pellet is not white, it means there is not enough nuclei. Possible causes for this problem are:</li>
<li>An inadequate composition of the Nuclei Isolation Buffer for the plant species. Some plant species might contain enzymes (proteases) that digest proteins in the nuclei membrane causing lysis of nuclei membrane during nuclei isolation.</li>
<li>Not enough starting materials or tissues. Depending on the plant species, 5 grams of tissue might not produce enough nuclei.</li>
<li>Breaking nuclei by vigorously shaking the solution during the extraction or during nuclei resuspension in NIBM.</li>
</ol>
<h3 id="sec6.4">Potential solution</h3>
<ol>
<li>Changing the DNA Extraction Buffer composition could help increase the DNA output.</li>
<li>Increasing incubation time for nuclei lysis can help increase the DNA output. We do not recommend incubating for less than 18 h.</li>
<li>Always prepare fresh 70% ethanol for precipitation or use isopropanol for DNA precipitation instead. Isopropanol precipitation was as efficient as ethanol precipitation (<a href="#tbl3">Table 3</a>).</li>
</ol>
<h3 id="sec6.5">Problem 3</h3>
<p>Low A<sub>260</sub>/A<sub>230</sub> values is a problem. The A<sub>260</sub>/A<sub>230</sub> ratio is expected to be ∼1.8. Possible causes of this problem are:</p>
<ol>
<li>Contamination with phenol, chloroform, plant phenolic compounds, starch, or salts.</li>
<li>The quality of the DNA depends on chemical or molecular composition in leaves of the plant species. For some plant species an additional purification step could be required.</li>
</ol>
<h3 id="sec6.6">Potential solution</h3>
<ol>
<li>Pipette slowly when transferring the aqueous phase during DNA extraction, to avoid contamination with phenol or with chloroform. When transferring the aqueous phase, the pipette tip should not get too close to the interphase.</li>
<li>Precipitation time in NaCl and ethanol or in isopropanol should not be longer than 1 h. If longer than 1 h, other compounds could co-precipitate with the DNA.</li>
<li>Proceed with an additional DNA purification step if required. For pineapple, the kit QIAGEN® Genomic-Tip 20/G should be used for purification.</li>
</ol>
<h3 id="sec6.7">Problem 4</h3>
<p>Low A<sub>260</sub>/A<sub>280</sub> values is a problem. The A<sub>260</sub>/A<sub>280</sub> ratio is expected to be ∼1.8. This problem is caused by protein contamination.</p>
<h3 id="sec6.8">Potential solution</h3>
<ol>
<li>Consider increasing the concentration of proteinase K during nuclei lysis and extract an additional time with chloroform: isoamyl alcohol. No white interphase should be observed when extracting with chloroform: isoamyl alcohol for the last time.</li>
</ol>
<h3 id="sec6.9">Problem 5</h3>
<p>Degraded or sheared DNA is a problem. DNA is sensitive to enzyme degradation (DNases), but also to pipetting and mixing by vortex.</p>
<h3 id="sec6.10">Potential solution</h3>
<ol>
<li>Clean all materials with either bleach or DNase or treat it with UV.</li>
<li>Keep DNA samples at 4°C during the extraction process.</li>
<li>Do not vortex your DNA sample.</li>
<li>Cut the pipette tips or use wide bore tips to transfer aqueous solutions containing DNA before autoclaving them.</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Healey, A., Furtado, A., Cooper, T., and Henry, R.J. (2014). Protocol: a simple method for extracting next-generation sequencing quality genomic DNA from recalcitrant plant species. Plant Methods <i>10</i>, 21. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00050-2/sref1">View at publisher</a></p>
<p id="bib2">Luo, M. and Wing, R.A. (2003). An improved method for plant BAC library construction. Methods Mol. Biol. <i>236</i>, 3-20. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00050-2/sref2">View at publisher</a></p>
<p id="bib3">Peterson, D.G., Boehm, K.S., and Stack, S.M. (1997). Isolation of milligram quantities of DNA from tomato (<i>Lycopersicon esculentum</i>), plant containing high levels of polyphenolic compounds. Plant Mol. Biol. Rep. <i>15</i>, 148-153. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00050-2/sref3">View at publisher</a></p>
<p id="bib4">Rana, M.M., Aycan, M., Takamatsu, T., Kaneko, K., Mitsui, T., and Itoh, K. (2019). Optimized nuclear pellet method for extracting next-generation sequencing quality genomic DNA from fresh leaf tissue. Methods Protoc. <i>2</i>, 54. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00050-2/sref4">View at publisher</a></p>
<p id="bib5">Zhang, et al. (2020). Genomes of the Banyan Tree and pollinator wasp provide insights into fig-wasp coevolution. Cell <i>183</i>, 875-889.e17. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00050-2/sref5">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and request for resources and reagents should be directed to and will be fulfilled by the lead contact, Ray R. Ming (<a href="mailto:rayming@illinois.edu">rayming@illinois.edu</a>).</p>
<h4>Materials Availability</h4>
<p>This study did not generate new unique materials or reagents.</p>
<h4>Data and Code Availability</h4>
<p>The published article includes all datasets generated or analyzed during this study.</p>
<h3>Acknowledgments</h3>
<p>We thank Tracie Matsumoto for providing plant materials of Caricaceae species and Tyler Jones and Ming-Li Wang for providing sugarcane and papaya samples. This protocol was developed as part of the Plant Genome Research Program DBI-1546890 awarded to R.M. by the National Science Foundation (NSF). Sequencing was done by the Roy J. Carver Biotechnology Center at the University of Illinois.</p>
<h3>Author Contributions</h3>
<p>R.M. conceived the protocol; D.Z.C. and X.Z. optimized the protocol; J.S. ordered all reagents and materials; D.Z.C., X.Z., and J.S. carried out the experiments with different plant species; A.H. provided sequencing results and useful recommendations for DNA purification; D.Z.C., R.M., and A.H. wrote and corrected the manuscript.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>