<article>
<section>
<h1 id="header">Purification of GFP-tagged nuclei from frozen livers of INTACT mice for RNA- and ATAC-sequencing</h1>
<p><time datetime="2021-09-10">Published: September 10, 2021</time></p>
<p>Anne Loft,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#fn1">7</a>,<a href="#cor1">*</a></sup> Stephan Herzig,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a></sup> and Søren Fisker Schmidt<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#aff6">6</a>,<a href="#fn2">8</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Institute for Diabetes and Cancer (IDC), Helmholtz Zentrum München, German Research Center for Environmental Health, Neuherberg 85764, Germany</p>
<p id="aff2"><sup>2</sup>Joint Heidelberg-IDC Translational Diabetes Program, Heidelberg University Hospital, Heidelberg 69120, Germany</p>
<p id="aff3"><sup>3</sup>Molecular Metabolic Control, Medical Faculty, Technical University Munich, München 80333, Germany</p>
<p id="aff4"><sup>4</sup>German Center for Diabetes Research (DZD), Neuherberg 85764, Germany</p>
<p id="aff5"><sup>5</sup>Department of Biochemistry and Molecular Biology, University of Southern Denmark, Odense 5230, Denmark</p>
<p id="aff6"><sup>6</sup>Center for Functional Genomics and Tissue Plasticity (ATLAS), University of Southern Denmark, Odense 5230, Denmark</p>
<p id="fn1"><sup>7</sup>Technical contact</p>
<p id="fn2"><sup>8</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:anlo@bmb.sdu.dk">anlo@bmb.sdu.dk</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:sfs@bmb.sdu.dk">sfs@bmb.sdu.dk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100805">10.1016/j.xpro.2021.100805</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Isolation of nuclei tagged in specific cell types (INTACT) allows for stress-free and high-throughput analyses of cellular subpopulations. Here, we present an improved protocol for isolation of pure and high-quality GFP-labeled nuclei from frozen livers of INTACT mice, as well as protocols for downstream sequencing analyses. The adaptation to frozen tissue provides a pause point that allows sampling at multiple time points and/or phenotypic characterization of livers prior to nuclei isolation and downstream analyses.</p>
<p>For complete details on the use of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref7">Loft et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Optimized protocol for isolation of nuclei from frozen livers</li>
<li>Protocol for immunopurification of nuclei tagged in specific cell types (INTACT)</li>
<li>Preparation of nuclear RNA- and ATAC-seq libraries</li>
<li>Nuclei from frozen and fresh livers perform equally in downstream assays</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol presents an optimized strategy for preparing pure, high-quality nuclei of a certain cellular subpopulation from frozen mouse livers. The protocol relies on the INTACT methodology and generation of so-called INTACT mice that apply a Cre-Lox system to express a tagged version of a nuclear membrane protein allowing for pull-down based affinity purification of genetically labeled nuclei from defined cell populations (<a href="#bib5">Deal and Henikoff, 2010</a>; <a href="#bib8">Mo et al., 2015</a>). INTACT has the advantage over e.g., fluorescence-activated cell sorting-based approaches in that it poses less stress to the cells/nuclei during the isolation, and further that a high number of samples can be handled within a relatively short timeframe.</p>
<p>However, not all experimental setups allow working on fresh tissue preparations; thus, we have optimized the protocol to obtain high-quality nuclei from frozen liver tissue. In the first sections, we describe how to purify hepatocyte nuclei from frozen livers of healthy HEP-INTACT male and female mice of 3–6 months of age, but we have also applied this protocol to obtain hepatocyte nuclei from older mice (6–12 months) with fatty and fibrotic livers as well as myeloid nuclei from livers of MAC-INTACT mice. In the later sections, we outline how these high-quality nuclei easily can be prepared for nuclear RNA-seq, ATAC-seq and proteomics approaches. We further provide examples that the nuclei preparations from frozen tissues perform equally well compared to preparations from fresh tissues in terms of quality, purity and in downstream sequencing applications.</p>
<p>This optimized protocol is therefore well-suited for experiments that do not allow for the use of fresh tissue preparations such as time course experiments and experiments that involve preceding phenotypic characterization such as histology.</p>
<h3 id="sec1.1">Generate HEP INTACT mice</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–6 month</div>
<p>To allow for selective pulldown of the cell type of interest CAG-Sun1/sfGFP reporter mice (B6;129-Gt(ROSA)26Sor<sup>tm5(CAG-Sun1/sfGFP)Nat</sup>/J) (<a href="#bib8">Mo et al., 2015</a>) should be crossed with specific Cre-driver mice. Specific labeling of hepatocytes can be achieved by crossing the CAG-Sun1/sfGFP reporter mice with Albumin-Cre mice (B6N.Cg-Speer6-ps1<sup>Tg(Alb-cre)21Mgn</sup>/J) (<a href="#bib9">Postic et al., 1999</a>)) to generate so-called HEP-INTACT mice.</p>
<div class="note">
<span class="note-title">Note:</span> We have also generated MAC-INTACT mice with myeloid-specific labeling by crossing the CAG-Sun1/sfGFP reporter mice with LysM-Cre mice (B6N.129P2(B6)-Lyz2<sup>tm1(cre)Ifo</sup>/J) (<a href="#bib2">Clausen et al., 1999</a>)). The following protocol also works with MAC-INTACT mice, although a few steps need to be adapted as outlined in the text.</div>
<div class="note">
<span class="note-title">Note:</span> If possible, breed both parental lines as homozygous prior to intercross, to allow for generation of INTACT mice heterozygous for both Cre and for the CAG-Sun1/sfGFP reporter at a 100% rate. This is possible for CAG-Sun1/sfGFP reporter mice as well as for both the abovementioned Albumin-Cre mice and LysM-Cre mice but needs to be assessed for each individual Cre-driver line. Depending on the research question, backcrossing to a defined background may need to be considered. The genetic composition of CAG-Sun1/sfGFP reporter mice available at Jax is approximately 75% B6J and 25% 129SV. Speed congenics can be considered for accelerating backcrossing to a specific desired background.</div>
<div class="note">
<span class="note-title">Note:</span> Specificity and efficacy of the Cre-driver can be assessed using e.g., immunofluorescence prior to proceeding to nuclear isolation and purification. Of note, we observed no labeling of nuclei in the absence of Cre-expression in the liver. Livers of Cre- animals can be used as negative controls in the isolation protocol, particularly for establishing the protocol. However, for larger experiments, nuclei isolated from the whole liver suspension are more cost-efficient controls, and it is further an advantage that these are obtained from the same animals as the purified nuclei of interest.</div>
<h3 id="sec1.2">Prepare buffers and reagents for nuclei purification</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol>
<li>Clean all instruments, including grinding jars and dounce homogenizers with cleaning agents for removing RNases (e.g., RNaseZap; Sigma; R2020) and wash with DEPC-treated water before start.</li>
<li>Prepare low sucrose buffer (LSB), high sucrose buffer (HSB), wash buffer (WB) and WB, incl. RNasin.
<ol type="a">
<li>Filter buffers using a 0.2 μm syringe filter and pre-cool buffers on ice.</li>
</ol>
</li>
<li>Prepare nuclear extraction buffer (NUN) if performing protein analyses.</li>
<li>Reconstitute lyophilized anti-GFP antibody in 0.5 mL PBS, pH 7.4 to prepare a 0.2 mg/mL stock solution</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Reconstituted antibody can be stored for up to 3 months at 4°C with the addition of 2 mM sodium azide.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rabbit monoclonal anti-GFP antibody<i>Reconstitute lyophilized anti-GFP antibody in 0.5 mL PBS, pH 7.4.</i>
</td>
<td>Thermo Fisher Scientific</td>
<td>Cat: G10362;RRID:AB_2536526</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Invitrogen™ TRIzol™ Reagent</td>
<td>Thermo Fisher Scientific</td>
<td>15596018</td>
</tr>
<tr>
<td>cOmplete™, EDTA-free Protease Inhibitor</td>
<td>Roche/Sigma</td>
<td>11873580001</td>
</tr>
<tr>
<td>cOmplete™ Mini, EDTA-free Protease Inhibitor</td>
<td>Roche/Sigma</td>
<td>11836170001</td>
</tr>
<tr>
<td>RNasin® Plus Rnase Inhibitor</td>
<td>Promega</td>
<td>N2615</td>
</tr>
<tr>
<td>Invitrogen™ Dynabeads™ Protein G</td>
<td>Thermo Fisher Scientific</td>
<td>10004D</td>
</tr>
<tr>
<td>Gibco™ Trypan Blue Solution, 0.4%</td>
<td>Thermo Fisher Scientific</td>
<td>15250061</td>
</tr>
<tr>
<td>DAPI</td>
<td>Sigma</td>
<td>D9542</td>
</tr>
<tr>
<td>Sucrose BioReagent</td>
<td>Sigma</td>
<td>S1888</td>
</tr>
<tr>
<td>IGEPAL® CA-630</td>
<td>Sigma</td>
<td>56741</td>
</tr>
<tr>
<td>DL-Dithiothreitol (DTT)</td>
<td>Sigma</td>
<td>D0632</td>
</tr>
<tr>
<td>Spermine tetrahydrochloride, BioReagent</td>
<td>Sigma</td>
<td>S1141</td>
</tr>
<tr>
<td>Spermidine, BioReagent</td>
<td>Sigma</td>
<td>S0266</td>
</tr>
<tr>
<td>Invitrogen™ 5M NaCl, RNase-free</td>
<td>Thermo Fisher Scientific</td>
<td>AM9760G</td>
</tr>
<tr>
<td>Invitrogen™ 2M KCl, RNase-free</td>
<td>Thermo Fisher Scientific</td>
<td>AM9640G</td>
</tr>
<tr>
<td>Invitrogen™ 1M MgCl<sub>2</sub>, RNase-free</td>
<td>Thermo Fisher Scientific</td>
<td>AM9530G</td>
</tr>
<tr>
<td>Invitrogen™ Ultrapure™ 1M Tris-HCl, pH 7.5</td>
<td>Thermo Fisher Scientific</td>
<td>15567027</td>
</tr>
<tr>
<td>Invitrogen™ UltraPure™ 1 M Tris-HCl Buffer, pH 8.0</td>
<td>Thermo Fisher Scientific</td>
<td>15568025</td>
</tr>
<tr>
<td>Invitrogen™ Ultrapure™ Urea</td>
<td>Thermo Fisher Scientific</td>
<td>15505027</td>
</tr>
<tr>
<td>Gibco™ 1<b>×</b> PBS, pH 7.4</td>
<td>Thermo Fisher Scientific</td>
<td>10010023</td>
</tr>
<tr>
<td>Invitrogen™ Nuclease-Free Water</td>
<td>Thermo Fisher Scientific</td>
<td>4387936</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>RNase-Free DNase Set (DNase I and Buffer RDD)</td>
<td>QIAGEN</td>
<td>79254</td>
</tr>
<tr>
<td>MinElute Reaction Cleanup Kit</td>
<td>QIAGEN</td>
<td>28204</td>
</tr>
<tr>
<td>MinElute PCR Purification Kit</td>
<td>QIAGEN</td>
<td>28004</td>
</tr>
<tr>
<td>NEBNext® High-Fidelity 2<b>×</b> PCR Master Mix</td>
<td>New England Biolabs Inc.</td>
<td>M0541S</td>
</tr>
<tr>
<td>High Sensitivity DNA Kit</td>
<td>Agilent</td>
<td>5067-4626</td>
</tr>
<tr>
<td>VAHTS Stranded mRNA-seq Library Kit for Illumina</td>
<td>Vazyme</td>
<td>NR602</td>
</tr>
<tr>
<td>TruePrep™ DNA Library Prep Kit V2 for Illumina® (contains 5<b>×</b> TTBL and TTE Mix V50)</td>
<td>Vazyme</td>
<td>TD501</td>
</tr>
<tr>
<td>TruePrep Index Kit V2 for Illumina®</td>
<td>Vazyme</td>
<td>TD202</td>
</tr>
<tr>
<td>Pierce™ BCA Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>23225</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>INTACT RNA-seq data</td>
<td>This study</td>
<td>GEO:GSE181940</td>
</tr>
<tr>
<td>INTACT ATAC-seq data</td>
<td>This study</td>
<td>GEO:GSE181937</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>B6;129-Gt(ROSA)26Sor</i><sup><i>tm5(CAG-Sun1/sfGFP)Nat</i></sup> <i>/ J mice</i> Female and male mice aged 8–26 wks were used for breeding.</td>
<td>JAX</td>
<td>Stock: 021039</td>
</tr>
<tr>
<td>
<i>B6N.Cg-Speer6-ps1</i><sup><i>Tg(Alb-cre)21Mgn</i></sup> <i>/ J</i> Female and male mice aged 8–26 wks were used for breeding.</td>
<td>JAX</td>
<td>Stock: 018961</td>
</tr>
<tr>
<td>
<i>B6N.129P2(B6)-Lyz2</i><sup><i>tm1(cre)Ifo</i></sup> <i>/ J</i> Female and male mice aged 8–26 wks were used for breeding.</td>
<td>JAX</td>
<td>Stock: 018956</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>TissueLyser II</td>
<td>QIAGEN</td>
<td>85300</td>
</tr>
<tr>
<td>Grinding Jar Set, Teflon</td>
<td>QIAGEN</td>
<td>69986</td>
</tr>
<tr>
<td>DWK Life Sciences Kimble™ Kontes™ Dounce Tissue Grinders</td>
<td>DWK Life Sciences</td>
<td>8853000015</td>
</tr>
<tr>
<td>100 μm CellTrics filter unit</td>
<td>Sysmex Deutschland GmbH</td>
<td>04-0042-2318</td>
</tr>
<tr>
<td>Invitrogen™ DynaMag™-2 Magnet</td>
<td>Thermo Fisher Scientific</td>
<td>12321D</td>
</tr>
<tr>
<td>Eppendorf™ DNA LoBind Tube 1.5 mL / 2 mL</td>
<td>Sigma</td>
<td>EP0030108051 /EP0030108078</td>
</tr>
<tr>
<td>EconoSpin Micro Columns</td>
<td>Epoch</td>
<td>3010-250</td>
</tr>
<tr>
<td>RPE buffer concentrate</td>
<td>QIAGEN</td>
<td>1018013</td>
</tr>
<tr>
<td>AMPure XP Beads</td>
<td>Beckman Coulter</td>
<td>A63881</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">TissueLyser II (QIAGEN) including grinding jar set</h3>
<p>This equipment is needed to pulverize frozen liver tissue before nuclei purification, which allows for an easier downstream douncing procedure, especially for fatty or fibrotic liver tissues, ultimately resulting in a higher nuclei recovery. We used Teflon grinding jar sets for the tissue pulverization and they worked impeccably. However, over time the Teflon grinding jars tended to break due to handling in liquid nitrogen and therefore steel grinding jars could be considered instead, although not tested in the current protocol. Furthermore, there are several alternative ways to pulverize frozen liver tissues (<a href="#bib6">Liang et al., 2019</a>), which can be used instead of the suggested method. E.g., for small amounts of tissue, a liquid nitrogen-cooled mortar and pestle may be preferable.</p>
<h3 id="sec2.2">Douncers for tissue homogenization</h3>
<p>For tissue homogenization, we used 15 mL glass dounce tissue grinders supplied with a loose pestle for initial sample reduction and a tight pestle to form the final homogenate. The protocol is optimized for initial tissue homogenization using 4 mL low sucrose buffer per 0.5 g of tissue, and we tested that these douncers effectively could homogenize up to at least 1 g of liver tissue. Alternative (smaller) douncers could be considered for douncing smaller amounts of pulverized liver tissue, but we recommend to always test the impact of alternative douncers on the protocol outcome.</p>
<h3 id="sec2.3">Antibodies</h3>
<p>Various antibodies may be used for the immunoprecipitation (IP) of the INTACT-labeled nuclei. In this protocol, we applied a monoclonal anti-GFP antibody (Thermo Fisher Scientific; G10362) targeting the two tandem copies of the superfolder GFP tagged to the SUN1 nuclear membrane protein. However, it is also possible to use anti-MYC antibodies targeting the six copies of the MYC epitope on SUN1. During the protocol optimization, we tested various antibodies and they performed differently in terms of yield and purity. Notably, one antibody (anti-MYC tag antibody (Thermo Fisher Scientific; MA1-21316)) led to extensive nuclei aggregation following the pulldown procedure and another (anti-MYC tag antibody (Abcam; ab9106)) appeared to unexpectedly interact with nuclei originating from hepatic stellate cells. Therefore, these antibodies should be avoided, and alternative antibodies should be thoroughly tested before applied in this protocol.</p>
<h3 id="sec2.4">Nanodrop and bioanalyzer for assessment of sample quality</h3>
<p>In this protocol, we used the NanoDrop™ 2000 (Thermo Fisher Scientific) to evaluate the purity and concentration of the nuclear RNA; however, alternative similar instruments can easily be used for this purpose. Furthermore, we used the Bioanalyzer 2100 (Agilent) to assess the integrity of the RNA and to quantify libraries during the sequencing preparations. Alternative instruments may be used here as well.</p>
<h3 id="sec2.5">Buffers for nuclei isolation</h3>
<p>The following recipes are sufficient for nuclei preparation from 12 frozen mouse livers.</p>
<table id="undtbl1">
<caption>Low sucrose buffer (LSB)</caption>
<thead>
<tr>
<th>Reagent (stock conc)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>250 mM</td>
<td>8.56 g</td>
</tr>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>15 mM</td>
<td>1.5 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>KCl (2 M)</td>
<td>25 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>Spermidine (1 M)</td>
<td>0.5 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Spermine (1 M)</td>
<td>0.15 mM</td>
<td>15 μL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td>RNasin (only for RNA analyses)</td>
<td>15 U/mL</td>
<td>150 μL</td>
</tr>
<tr>
<td>EDTA-free protease inhibitor cocktail (PIC)</td>
<td> </td>
<td>2 tablets</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td> </td>
<td>Up to 100 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>100 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl2">
<caption>High sucrose buffer (HSB)</caption>
<thead>
<tr>
<th>Reagent (stock conc)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>2 M</td>
<td>27.36 g</td>
</tr>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>15 mM</td>
<td>0.6 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>0.2 mL</td>
</tr>
<tr>
<td>KCl (2 M)</td>
<td>25 mM</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Spermidine (1 M)</td>
<td>0.5 mM</td>
<td>20 μL</td>
</tr>
<tr>
<td>Spermine (1 M)</td>
<td>0.15 mM</td>
<td>6 μL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>1 mM</td>
<td>40 μL</td>
</tr>
<tr>
<td>RNasin (only for RNA analyses)</td>
<td>15 U/mL</td>
<td>60 μL</td>
</tr>
<tr>
<td>EDTA-free PIC</td>
<td> </td>
<td>1 tablet</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td> </td>
<td>Up to 40 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>40 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl3">
<caption>Wash buffer (WB)</caption>
<thead>
<tr>
<th>Reagent (stock conc)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>IGEPAL CA-630 (5%)</td>
<td>0.35%</td>
<td>17.5 mL</td>
</tr>
<tr>
<td>Sucrose</td>
<td>250 mM</td>
<td>21.4 g</td>
</tr>
<tr>
<td>Tris-HCl, pH 7.5 (1 M)</td>
<td>15 mM</td>
<td>3.75 mL</td>
</tr>
<tr>
<td>MgCl<sub>2</sub> (1 M)</td>
<td>5 mM</td>
<td>1.25 mL</td>
</tr>
<tr>
<td>KCl (2 M)</td>
<td>25 mM</td>
<td>3.125 mL</td>
</tr>
<tr>
<td>Spermidine (1 M)</td>
<td>0.5 mM</td>
<td>125 μL</td>
</tr>
<tr>
<td>Spermine (1 M)</td>
<td>0.15 mM</td>
<td>37.5 μL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>1 mM</td>
<td>250 μL</td>
</tr>
<tr>
<td>EDTA-free PIC</td>
<td> </td>
<td>2 tablets</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td> </td>
<td>Up to 250 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl4">
<caption>WB, incl. RNasin</caption>
<thead>
<tr>
<th>Reagent (stock conc)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>WB (prepared from above)</td>
<td> </td>
<td>30 mL</td>
</tr>
<tr>
<td>RNasin</td>
<td>15 U/mL</td>
<td>45 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>30 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> DTT, spermine, spermidine, RNasin, and protease inhibitor should always be added right before the start of the experiment, the remaining buffer components can be prepared in advanced and kept for a maximum of 2 weeks at −20°C.</div>
<div class="note">
<span class="note-title">Note:</span> The sucrose (especially in HSB) might need some time to solubilize. It is possible to use e.g., a roller mixer to facilitate the process. Further, the resuspension of the PIC tablet in HSB can also be facilitated using a roller mixer.</div>
<div class="note">
<span class="note-title">Note:</span> 1M DTT, spermine, and spermidine stock aliquots can be prepared in advance and stored at −20°C. Do not re-freeze a thawed aliquot.</div>
<div class="note">
<span class="note-title">Note:</span> A 5% IGEPAL solution can be prepared beforehand and stored for several months at 20°C–25°C.</div>
<div class="note">
<span class="note-title">Note:</span> RNasin is only needed for experiments intended for RNA analyses.</div>
<table id="undtbl5">
<caption>NUN extraction buffer</caption>
<thead>
<tr>
<th>Reagent (stock conc)</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl, pH 8 (1 M)</td>
<td>50 mM</td>
<td>50 μL</td>
</tr>
<tr>
<td>Urea</td>
<td>2 M</td>
<td>120 mg</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>600 mM</td>
<td>120 μL</td>
</tr>
<tr>
<td>DTT (1 M)</td>
<td>1 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>mini EDTA-free PIC</td>
<td> </td>
<td>1 tablet</td>
</tr>
<tr>
<td>Nuclease-Free Water</td>
<td> </td>
<td>Up to 1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>NA</b></td>
<td><b>1 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Add water as the last component, since urea will not dissolve easily in water.</div>
<div class="note">
<span class="note-title">Note:</span> NUN extraction buffer should be used within one day</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> It should be possible to replace all reagents listed in the <a href="#key-resources-table">key resources table</a> with equivalent items from other suppliers, but this has not been tested.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Pulverize frozen liver extracts</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h for 12 tissue preparations</div>
<p>In our experience, the best preparations of nuclei from snap-frozen liver samples are obtained by pulverizing the tissue into fine powder before starting the purification of the nuclei.</p>
<ol>
<li>Isolation of mouse liver tissues.
<ol type="a">
<li>Sacrifice the mouse humanely following an approved ethical protocol.</li>
<li>Pin down the mouse and open the abdominal cavity.</li>
<li>Isolate all liver lobes using scissors and forceps.</li>
<li>Cut the liver lobes into smaller pieces (approx. 5<b>×</b>5 mm) and wash liver pieces in 1<b>×</b> PBS.</li>
<li>Snap-freeze liver pieces in liquid nitrogen.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, liver tissue pieces can be stored at −80°C for later downstream processing allowing for e.g., initial histological assessment of the liver.</div>
<ol start="2">
<li>Grinding of liver tissue.
<ol type="a">
<li>Unscrew the lid of the grinding jar and pre-chill the jar, lid, and the grinding ball in liquid nitrogen.</li>
<li>Put the grinding ball back into the jar, add the liver pieces into the jar and close the lid firmly.</li>
<li>Grind the liver tissue at 30 Hz for 1 min at the Tissuelyser until the tissue is completely pulverized.</li>
<li>Carefully transfer the pulverized tissue into e.g., a 15 mL conical tube and be sure to scrape of remaining pulverized tissue remnants from the side of the jar and from the grinding ball.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> It is extremely important that all items are kept very cold during tissue handling. It might be necessary to wear extra protection of your hands due to the handling of these very cold items.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, the pulverized tissues can be stored at −80°C for later downstream processing. In order to have sufficient time for performing the following steps (i.e., isolation and purification of GFP-labeled nuclei as well as the first steps in the preparation of the nuclei for sequencing) within a working day, we normally pause the protocol at this point.</div>
<h3 id="sec3.2">Isolation of nuclei from pulverized liver tissue</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h for 12 nuclei preparations</div>
<p>The following steps describe how to purify high-quality nuclei from pulverized liver tissue using a glass dounce homogenizer.</p>
<ol start="3">
<li>Homogenize the liver (see <a href="#sec6.1">troubleshooting 1</a>).
<ol type="a">
<li>Precool centrifuges to 4°C.</li>
<li>Resuspend ∼0.5 g pulverized tissue in 4 mL ice-cold 1<b>×</b> PBS by tapping e.g., in a 5 mL conical tube.
<div class="note">
<span class="note-title">Note:</span> If processing livers with high fat content, the nuclei yield per gram of liver will be lower and a greater amount of pulverized liver tissue can be used as starting material. We have tested that the 15 mL douncer can hold up to approx. 1 g of tissue using 8 mL LSB and still perform equally well. However, it is important not to overload the douncer with more than 1 g of pulverized liver (see <a href="#sec6.1">troubleshooting 1</a>). Then rather perform multiple rounds of douncing and combine before proceeding with the IP.</div>
</li>
<li>Centrifuge at 2,000<b>×</b><i>g</i> for 3 min at 4°C.</li>
<li>Remove supernatant and resuspend pellet in 4 mL LSB.</li>
<li>Transfer to douncer and dounce 10 times with loose pestle.</li>
<li>Add 300 μL of a 5% IGEPAL solution to make the solution ∼0.35% IGEPAL, mix tubes and let stand at 4°C for 3–5 min.</li>
<li>Dounce 5 times more with tight pestle.</li>
<li>Filter the homogenate through a 100 μm CellTrics filter unit on a 15 mL canonical tube.</li>
<li>Wash the douncer with 1 mL LSB and add through the filter to homogenate in step h.</li>
<li>Spin filtered homogenate at 600<b>×</b><i>g</i> for 10 min at 4°C.</li>
<li>Resuspend pellet in 9<b>×</b> volume of HSB. Mix by pipetting.</li>
<li>Aliquot up to 1.8 mL in 2 mL tubes.
<div class="note">
<span class="note-title">Note:</span> Normally 9<b>×</b> volume equals to 0.9–1.8 mL HSB per sample. If more than 1.8 mL is needed per sample divide the volume into two 2 mL tubes.</div>
<div class="note">
<span class="note-title">Note:</span> It is recommended to double check that the nuclei pellets have gone completely into suspension. If not, mix samples again by pipetting right before spinning.</div>
</li>
<li>Spin 15 min at 15,000<b>×</b><i>g</i> at 4°C.</li>
<li>Carefully remove as much supernatant as possible.</li>
<li>Resuspend pellets in total 1 mL WB, incl. RNasin per sample.
<div class="note">
<span class="note-title">Note:</span> After pelleting the nuclei in HSB, these might be hard to resuspend and need to be pipetted extensively, but still carefully, for complete resuspension.</div>
<div class="note">
<span class="note-title">Note:</span> The dense sucrose cushion is recommended to remove debris and provide pure and intact nuclei. To avoid transferring the debris on the tube sides, we recommend resuspending first in 500 μL WB, incl. RNasin, transfer this volume to a new tube and then add additional 500 μL WB, incl. RNasin.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Carry out above steps on ice or in a cold room at 4°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="4">
<li>Inspection and counting the number of whole liver nuclei (see <a href="#fig1">Figures 1</a>A and 1B, <a href="#expected-outcomes">expected outcomes</a>, and <a href="#sec6.3">troubleshooting 2</a> and <a href="#sec6.5">3</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Nuclei purification from pulverized fresh and frozen liver tissue</div>
<p>(A and B) Nuclei were isolated from (A) fresh and (B) frozen livers, stained with Trypan blue and investigated using a microscope. Scale bars represent 200 μm.</p>
<p>(C and D) Nuclei were immunoprecipitated using anti-GFP antibody and magnetic Dynabeads, stained with Trypan blue and investigated using a microscope. Scale bars represent 100 μm.</p>
</figcaption>
</figure>
<ol type="a">
<li>Take out 10 μL aliquot, dilute it 1:20 in WB and mix 1:1 with Trypan blue.</li>
<li>Inspect and count the nuclei under a brightfield microscope using a 10–20<b>×</b> objective.</li>
<li>Dilute the nuclei suspension to 10 mill nuclei per mL in WB, incl. RNasin.</li>
<li>Collect 15 mill nuclei (1.5 mL) for IP of hepatocyte nuclei.</li>
</ol>
</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Collect 1 mill nuclei for whole liver nuclei RNA-seq and 25,000 for whole liver nuclei ATAC-seq and store on ice until further processing. These whole liver nuclei can in the end be used for control purposes, e.g., for determining enrichment/depletion of marker gene expression compared to the bead-bound nuclei by qPCR or RNA-seq.</div>
<div class="critical">
<span class="critical-title">Critical:</span> An appropriate counting of the nuclei is important for the downstream processes. We recommend counting nuclei manually using a standard hemocytometer. Alternatively, the nuclei can be counted using an automated cell counter. Since the absolute nuclei numbers often vary between the different methods, it is important to use the same strategy for each experiment.</div>
<div class="note">
<span class="note-title">Note:</span> Nuclei of high quality will appear round and intact with well-defined edges under the brightfield microscope at 40–60<b>×</b> magnification, whereas nuclei of poor quality lose their intactness and begin to show evidence of membrane blebbing (for examples of nuclei with different quality refer to <a href="https://kb.10xgenomics.com/hc/en-us/articles/360020348651-How-can-I-assess-the-quality-of-my-nuclei-for-Single-Cell-ATAC-or-Single-Cell-Multiome-ATAC-GEX-Sequencing-">https://kb.10xgenomics.com/hc/en-us/articles/360020348651-How-can-I-assess-the-quality-of-my-nuclei-for-Single-Cell-ATAC-or-Single-Cell-Multiome-ATAC-GEX-Sequencing-</a>).</div>
<div class="note">
<span class="note-title">Note:</span> For other conditions or types of labeled nuclei, the number of input nuclei can be scaled proportionally to the fraction of GFP+ nuclei. For example, when purifying GFP-labeled myeloid nuclei originating from frozen livers of healthy MAC INTACT mice (consisting of 7%–10% GFP+ nuclei), we used approximately 80 mill whole liver nuclei as input for the IP but kept the reaction volume and amount of antibody and Dynabeads as described below.</div>
<h3 id="sec3.3">Immunoprecipitation of GFP-labeled nuclei</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h for 12 nuclear preparations</div>
<p>The following steps describe how to pull down GFP-labeled nuclei from purified whole liver nuclei.</p>
<ol start="5">
<li>Washing of Dynabeads.
<ol type="a">
<li>Aliquot and wash 100 μL of Protein G Dynabeads (per sample) 2<b>×</b> with 800 μL Wash Buffer in a 2 mL tube on a DynaMag™-2 Magnet.
<div class="optional">
<span class="optional-title">Optional:</span> Protein G Dynabeads can also be batch-washed in 15 mL tubes using e.g., a DynaMag™-15 magnet (ThermoFisher Scientific; 12301D). To have enough beads for 12 nuclear preparations, use 1350 μL beads and 12 mL WB.</div>
</li>
<li>Resuspend beads in 100 μL per sample of WB, incl. RNasin.</li>
<li>Pipette 20 μL of beads into new 2 mL tube (labeled “Preclear”).</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Pre-clearing of nuclei.
<ol type="a">
<li>Transfer the nuclei suspension into the preclear tube and mix.</li>
<li>End-to-end rotate at 4°C for 10–15 min.</li>
</ol>
</li>
</ol>
<ol start="7">
<li>IP of GFP-labeled nuclei.
<ol type="a">
<li>While nuclei sample is preclearing, prepare 2 mL tubes with 3 μg of anti-GFP antibody (i.e., 15 μL) and 200 μL WB, incl. RNasin in each tube.</li>
<li>Place precleared sample on magnet for 4–5 min.</li>
<li>While standing on the magnet, pipet the supernatant from the precleared sample into the tube with antibody.</li>
<li>End-to-end rotate at 4°C for 30 min.</li>
<li>Resuspend the remaining 80 μL of washed beads and add to tube with antibody and nuclei.</li>
<li>End-to-end rotate at 4°C for 20 min.</li>
<li>Place sample on magnet for 4–5 min.</li>
<li>While standing on the magnet, completely remove the supernatant.
<div class="optional">
<span class="optional-title">Optional:</span> the supernatant contains the nuclei that was not captured in the IP reaction (primarily non GFP-labeled nuclei) and can be saved as controls for later analyses. However, from frozen liver tissues this fraction also contains a considerable number of unbound GFP+ nuclei.</div>
</li>
<li>Remove samples from magnet; add 1.8 mL of WB and let sit for 30 s.
<div class="note">
<span class="note-title">Note:</span> RNasin is omitted from the WB during the washes to reduce costs.</div>
</li>
<li>Pipette extensively (but carefully) up and down to resuspend. After initial resuspension, pipette up and down an additional 5 times.</li>
<li>Repeat step g-j twice.</li>
<li>Finally take the tube of the magnet, add 1 mL WB, incl. RNasin and let sit over the nuclei for 2 min before resuspending.
<div class="critical">
<span class="critical-title">Critical:</span> carry out all steps on ice or in the cold room at 4°C, including the steps involving a magnet or an end-to-end rotator.</div>
</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Inspection and counting of bead-bound nuclei (see also <a href="#fig1">Figures 1</a>C and 1D, <a href="#expected-outcomes">expected outcomes</a> and <a href="#sec6.7">troubleshooting 4</a>).
<ol type="a">
<li>Take out 10 μL aliquot of bead-bound nuclei, dilute it 1:5 in WB and mix 1:1 with Trypan blue.</li>
<li>Count the number of nuclei using a brightfield microscope using a 10–20<b>×</b> objective.</li>
<li>Calculate the fraction of recovered nuclei by comparing the number of bead-bound nuclei with the number of whole liver nuclei initially added to the IP reaction (i.e., 15 mill nuclei).
<div class="optional">
<span class="optional-title">Optional:</span> Determine purity and yield of GFP+ nuclei (see <a href="#fig2">Figure 2</a>, <a href="#expected-outcomes">expected outcomes</a> and <a href="#sec6.9">troubleshooting 5</a>)</div>
</li>
<li>Add 1 μL DAPI to a 10 μL aliquot of bead-bound and whole liver nuclei (diluted to approx. 7500 nuclei / μL)</li>
<li>Count 100–200 nuclei with a fluorescence microscope equipped with appropriate filters for detecting GFP and DAPI using a 10–20<b>×</b> objective.</li>
<li>Quantify the number of bead-bound GFP+ nuclei vs. DAPI-stained nuclei in both samples.</li>
<li>Determine the purity by considering the fraction of GFP+ nuclei vs total DAPI-stained nuclei in the bead-bound sample.</li>
<li>To determine the yield, compare the actual number of recovered GFP+ nuclei with the expected number of recovered GFP+ nuclei, which can be calculated as follows: Expected # of GFP+ nuclei = Total # of nuclei added to IP x % GFP+ nuclei in the original whole liver sample.
<div class="note">
<span class="note-title">Note:</span> The unbound Dynabeads will result in some green auto fluorescence; however, since the beads are much smaller than the complexes of nuclei and beads, these should be easily distinguishable when counting.</div>
<div class="note">
<span class="note-title">Note:</span> If handling a larger number of nuclei preparations, the easier option for determining purity and yield will be to take some images using the fluorescence microscope coupled to a computer and do the quantification with e.g., ImageJ (<a href="https://imagej.nih.gov/ij/download.html">https://imagej.nih.gov/ij/download.html</a>) at a later time point.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> At this point, it is possible to save (extra) nuclei for later processing or alternative applications by using a slow-freezing protocol. For further details, please refer to <a href="#bib3">Corces et al., 2019</a>; <a href="#bib4">Corces et al., 2017</a>, which describe how purified nuclei can be frozen and subsequently used for ATAC-seq and other downstream applications without adversely affecting data quality. However, for the current protocol, we recommend continuing with the procedure until nuclei are stored in TRIzol (for RNA analyses), in NUN buffer (for protein analyses) or until the transposition reaction followed by the DNA purification has been performed (for ATAC-seq analyses).</div>
</li>
</ol>
</li>
</ol>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. GFP+ nuclei detection in whole liver and bead-bound nuclei preparations</div>
<p>(A and B) Whole liver and (C and D) bead-bound nuclei were isolated from frozen livers, stained with DAPI and subsequently (A and C) DAPI and (B and D) GFP signals were investigated using a fluorescence microscope. Scale bars represent 100 μm.</p>
</figcaption>
</figure>
<h3 id="sec3.4">Downstream applications</h3>
<p>In the following sections we describe three different downstream applications for the purified nuclei. The different procedures can be performed independently, and in any order. However, since the procedure for preparation of nuclei for RNA-seq has a safe pause point at an early step in the procedure, we normally begin with this procedure.</p>
<h3 id="sec3.5">Prepare nuclei for RNA-seq</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼20 min until nuclei are stored in TRIzol, 2–3h for 12 complete RNA purifications, and 1 additional day if preparing RNA for sequencing</div>
<p>The following steps describe how to isolate high-quality RNA from bead-bound and whole liver nuclei.</p>
<ol start="9">
<li>Harvest nuclei for RNA isolation.
<ol type="a">
<li>Whole liver nuclei.
<ol type="i">
<li>Spin down 1 mill nuclei at 600<b>×</b><i>g</i> for 5 min at 4°C.</li>
<li>Remove supernatant and leave 20–30 μL WB, incl. RNasin behind to resuspend pellet again.</li>
<li>Vortex nuclei to resuspend, add 500 μL of TRIzol and vortex until nuclei are completely dissolved.</li>
</ol>
</li>
<li>For bead-bound nuclei.
<ol type="i">
<li>Take a volume out corresponding to 1 mill nuclei.</li>
<li>Use a magnet to remove supernatant leaving 20–30 μL WB, incl. RNasin behind to resuspend pellet again.
<div class="critical">
<span class="critical-title">Critical:</span> Failure to properly resuspend the nuclei pellet will cause nuclei to remain aggregated when adding TRIzol.</div>
</li>
<li>Briefly spin down to collect nuclei and WB in the bottom of the tube.</li>
<li>For one tube at a time vortex, immediately add 500 μL of TRIzol and vortex until completely dissolved</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>RNA column purification procedure
<ol type="a">
<li>Add 0.2 volume (100 μL) of chloroform and vortex vigorously for 30 s to mix phases.</li>
<li>Incubate 15 min at 20°C–25°C and spin at 10,000<b>×</b><i>g</i> for 10 min at 4°C.</li>
<li>Transfer the upper phase (approx. 200 μL) to a 1.5 mL tube and place on ice.
<div class="critical">
<span class="critical-title">Critical:</span> Do not touch the interphase!</div>
</li>
<li>Add 0.6 volume (120 μL) 96% ethanol, briefly vortex, and briefly spin down.</li>
<li>Load the indicated volume onto an EconoSpin microcolumn, spin 13.000<b>×</b><i>g</i> for 30 s at 20°C–25°C.
<div class="note">
<span class="note-title">Note:</span> Close the lid on the column gently.</div>
</li>
<li>Wash the column once with 500 μL diluted RPE buffer at 20°C–25°C. Spin at 13,000<b>×</b><i>g</i> for 30 s at 20°C–25°C and discard the flow through.
<div class="critical">
<span class="critical-title">Critical:</span> Confirm that ethanol has been added to the RPE buffer before use.</div>
<div class="note">
<span class="note-title">Note:</span> A multistep pipette can be used if handling multiple samples.</div>
</li>
<li>Make a DNase I mix, by combining 10 μL DNase I stock solution with 70 μL Buffer RDD. Mix by gently inverting the tube and shortly spin down the tube before use.
<div class="note">
<span class="note-title">Note:</span> It is possible to prepare a DNase mastermix if multiple samples are processed.</div>
</li>
<li>Add the DNase I mix (80 μL) directly to the microcolumn membrane, and place on the benchtop (20°C–30°C) for 15 min.</li>
<li>Add 500 μL diluted RPE buffer to the microcolumn with the DNase I incubation mix. Close the lid gently, and centrifuge at 13,000<b>×</b><i>g</i> for 30 s at 20°C–25°C to wash the spin column membrane free of DNaseI.</li>
<li>Discard the flow-through and collection tube.</li>
<li>Place the microcolumn in a new 2 mL collection tube and wash the microcolumn one additional time with 500 μL diluted RPE buffer.</li>
<li>Spin at 13,000<b>×</b><i>g</i> for 30 s at 20°C–25°C and discard the flow through.</li>
<li>Place the microcolumn back in the collection tube and spin dry at 13,000<b>×</b><i>g</i> for 2 min at 20°C–25°C.</li>
<li>Transfer the column to a new 1.5 mL tube and incubate the column with lid open for 5–10 min at 20°C–25°C to completely evaporate residual ethanol.
<div class="critical">
<span class="critical-title">Critical:</span> Be careful not to touch the sides of the microcolumn while transferring the column between tubes!</div>
</li>
<li>Elute in 16 μL RNAse-free water by adding to the center of the column and incubate for 2 min at 20°C–25°C.</li>
<li>Spin at 9,000<b>×</b><i>g</i> for 2 min at 20°C–25°C and collect eluate.
<div class="note">
<span class="note-title">Note:</span> If lid breaks, transfer the eluate to a new 1.5 mL tube.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Proceed for downstream applications or store at −80°C.</div>
</li>
</ol>
</li>
</ol>
<ol start="11">
<li>Measure RNA concentration
<ol type="a">
<li>Measure the RNA concentration and quality on 1 μL of sample on the Nanodrop.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The ratios A260/A280 and A260/A230 should be ≥ 1.8 and ideally ≥2.</div>
<div class="note">
<span class="note-title">Note:</span> If 1 mill nuclei are used for the RNA-analyses, the expected RNA concentrations are &gt;50 ng/μL. A lower number of nuclei can be used for RNA analyses depending on the downstream purposes.</div>
<div class="optional">
<span class="optional-title">Optional:</span> Determine RNA integrity.</div>
<p>It is normally recommended to measure the quality of the RNA e.g., using the RNA integrity number (RIN) computed by the Bioanalyzer. However, since the RIN score is based on the ratio of the 18S to 28S ribosomal RNA molecules this is not a suitable measure for nuclear RNA. In fact, we usually obtained RIN scores around 2–3 for the nuclear RNA, which still performed impeccably in the RNA sequencing preparations. Instead, we used the Bioanalyzer profiles to manual inspect the RNA integrity, since we observed a clear difference in the amount of degraded RNA with the presence of RNasin in the buffers (<a href="#fig3">Figures 3</a>A and 3B).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. RNasin in INTACT buffers prevents degradation of nuclear RNA</div>
<p>Nuclei were isolated and immunoprecipitated without or with the presence of RNasin in the buffers, as indicated in the protocol. RNA was subsequently column-purified, and RNA integrity was assayed using a Bioanalyzer 2100.</p>
<p>(A) When RNasin was not added to the buffers, the bioanalyzer run showed traces of highly fragmented RNA.</p>
<p>(B) When RNasin was added to the buffers, the RNA became less degraded. Note that peaks corresponding to 18S and 28S rRNA fragments are absent from the nuclear RNA trace and therefore the RNA integrity score is not a reliable measure for assessing quality of nuclear RNA.</p>
</figcaption>
</figure>
<ol start="12">
<li>Prepare RNA for sequencing
<ol type="a">
<li>100–200 ng of RNA can be prepared for sequencing using the VAHTS Stranded mRNA-seq Library Kit for Illumina according to the manufacture’s recommendations (<a href="https://v4.cecdn.yun300.cn/100001_2005225118%2FNR602.pdf">https://v4.cecdn.yun300.cn/100001_2005225118%2FNR602.pdf</a>), except that no enrichment of mRNA was performed. Instead, 5 μL of (diluted) RNA was added directly to 13.5 μL of Frag/Prime Buffer (1.13 in the original manual) to fragment the RNA before proceeding with the cDNA synthesis.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A lot of manufacturers offer kits for preparation of RNA for sequencing and can be used as alternatives here.</div>
<h3 id="sec3.6">Prepare nuclei for ATAC-seq</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h for 12 transpositions reactions, 1 additional day if preparing genomic DNA for sequencing.</div>
<p>The following steps describe how to perform a Tn5 transposase reaction (<a href="#bib1">Buenrostro et al., 2015</a>) on purified bead-bound and whole liver nuclei and subsequently prepare the DNA for sequencing. In the following, we enrich specifically for DNA fragments in nucleosome-free regions (NFR) since we are particularly focused on the accessible regions and transcription factor binding events in these regions. If also interested in mono-, di- and multi-nucleosome fragments the protocol should be adjusted accordingly.</p>
<ol start="13">
<li>Perform transposition reaction.
<ol type="a">
<li>Thaw 5<b>×</b> TTBL from the TruePrep DNA Library Prep Kit V2 at 20°C–25°C. Mix by inverting.</li>
<li>For whole liver nuclei.
<ol type="i">
<li>Spin down 25,000 nuclei at 600<b>×</b><i>g</i> for 5 min at 4°C.</li>
<li>Remove supernatant completely.</li>
</ol>
</li>
<li>For bead-bound nuclei.
<ol type="i">
<li>Take a volume out corresponding to 25,000 nuclei.</li>
<li>Use a magnet to remove supernatant completely.</li>
</ol>
</li>
<li>Prepare a transposition reaction mastermix (50 μL per sample):
<table id="undtbl6">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5 X TTBL buffer</td>
<td>10 μL</td>
</tr>
<tr>
<td>TTE Mix V50</td>
<td>3.5 μL</td>
</tr>
<tr>
<td>Nuclease-free water</td>
<td>36.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Resuspend the nuclear pellet directly in the transposition reaction mixture by gently pipetting for 20 times.</li>
<li>Put the tube in the thermal block at 37°C for 30 min at 1000<b>×</b>rpm with heat lid on at 105°C.</li>
</ol>
</li>
</ol>
<ol start="14">
<li>Purify DNA fragment
<ol type="a">
<li>Add 5 volumes (250 μL) of Buffer ERC and 5 μL Na-acetate (3 M, pH = 5.2) from the MinElute Reaction Cleanup Kit to the transposition reaction mix.
<div class="critical">
<span class="critical-title">Critical:</span> Check that the color of the mixture is yellow similar to the Buffer ERC.</div>
</li>
<li>Mix by vortexing and incubate for 5 min at 20°C–25°C.</li>
<li>Transfer the cleaned-up reaction mix by:
<ol type="i">
<li>For bead-bound nuclei, place mixture on the magnet and transfer supernatant to a MinElute column.</li>
<li>For whole liver nuclei, transfer the whole reaction mix to a MinElute column.</li>
</ol>
</li>
<li>Proceed with Qiagen MinElute Reaction Cleanup according to manufactures recommendations (<a href="https://www.qiagen.com/us/resources/download.aspx?id=8f6b09b2-6dcd-4b55-bb4a-255ede40ca3b&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=8f6b09b2-6dcd-4b55-bb4a-255ede40ca3b&amp;lang=en</a>) and elute DNA in 21 μL elution buffer.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The samples can be stored at −20°C until ready for amplification.</div>
<div class="note">
<span class="note-title">Note:</span> The transposition reaction reagents from Vazyme are not compatible with the OMNI ATAC reagents described in (<a href="#bib4">Corces et al., 2017</a>).</div>
</li>
</ol>
</li>
</ol>
<ol start="15">
<li>PCR Enrichment and purification
<ol type="a">
<li>Place the sterile PCR tube on ice and add each component in order:
<table id="undtbl7">
<thead>
<tr>
<th>Component</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tranposed DNA</td>
<td>10 μL</td>
</tr>
<tr>
<td>2<b>×</b> NEBNext PCR Master Mix</td>
<td>12 μL</td>
</tr>
<tr>
<td>N5XX</td>
<td>1 μL</td>
</tr>
<tr>
<td>N7XX</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>24 μL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> The TruePrep Index Kit from Vazyme provides 8 kinds of N5XX and 12 kinds of N7XX. Choose according to the number of samples and strategy of Index selection. Due to adaptor hopping, unique combinations of N5XX and N7XX should be selected for each sample, if possible.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend running the sample index PCR step using only half of the transposed DNA sample volume. The remaining sample volume can be stored at −20°C up to a month and may be used for another round of sample index PCR if necessary.</div>
</li>
<li>Place the PCR tube in the PCR instrument.</li>
<li>Set heat lid on (105°C) and run the PCR as follows:
<table id="undtbl8">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial chain displacement reaction</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>15 s</td>
<td rowspan="3">11–13 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>60°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>1 min</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Infinite</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> For most preparations, 11 cycles of PCR are sufficient. If there is not sufficient DNA for sequencing, use the other half of the transposed DNA volume to perform another round of sample index PCR with more cycles.</div>
</li>
<li>Perform cleanup of libraries after the PCR using MinElute PCR Purification Kit according to the manufacturer’s recommendations (<a href="https://www.qiagen.com/us/resources/download.aspx?id=8f6b09b2-6dcd-4b55-bb4a-255ede40ca3b&amp;lang=en">https://www.qiagen.com/us/resources/download.aspx?id=8f6b09b2-6dcd-4b55-bb4a-255ede40ca3b&amp;lang=en</a>) and elute in 21 μL elution buffer.
<div class="critical">
<span class="critical-title">Critical:</span> Use different reagents for pre- and post-amplification samples and consider working in designated pre- and post-amplification areas to avoid contamination with amplified PCR-products.</div>
</li>
</ol>
</li>
</ol>
<ol start="16">
<li>Examine PCR libraries and perform size fractionation of pooled libraries.
<ol type="a">
<li>Run 1 μL of each library (diluted 1:3 in water) using the High Sensitivity DNA kit according to the manufacturer’s recommendations (<a href="https://www.agilent.com/cs/library/usermanuals/Public/G2938-90321_SensitivityDNA_KG_EN.pdf">https://www.agilent.com/cs/library/usermanuals/Public/G2938-90321_SensitivityDNA_KG_EN.pdf</a>) (<a href="#fig4">Figures 4</a>A and 4B and <a href="#sec6.11">troubleshooting 6</a>).</li>
<li>Estimate the molarity of each library in the desired size range and pool samples for sequencing.</li>
<li>Fill up the volume to a total 50 μL with nuclease-free water before the next step.</li>
<li>Mix AMPure® XP beads by vortexing and add 32.5 μL (0.65<b>×</b>) of beads to the purified PCR products (50 μL). Mix thoroughly by gently pipetting for 10 times and incubate at 20°C–25°C for 5 min.</li>
<li>Place the reaction tube on the magnetic stand after a brief centrifugation. Let the tube rest until the solution completely clarifies (around 5 min)</li>
<li>
<u>Transfer the supernatant carefully to a new tube</u> and discard the beads.</li>
<li>Mix AMPure® XP beads by vortexing and add 55 μL (1.1<b>×</b>) of beads to the supernatant. Mix thoroughly by gently pipetting for 10 times and incubate at 20°C–25°C for 5 min.</li>
<li>Place the reaction tube on the magnetic stand after a brief centrifugation. Let the tube rest until the solution completely clarifies</li>
<li>Remove the supernatant carefully and <u>store the beads</u>.</li>
<li>Add 200 μL of fresh prepared 80% ethanol to wash the beads. Incubate at 20°C–25°C for 30 s and discard the supernatant. Repeat for a total of two washes.</li>
<li>Open the tube lid and air-dry the beads at 20°C–25°C for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> Do not overdry the beads.</div>
<div class="note">
<span class="note-title">Note:</span> Keep the tube on the magnetic stand while washing and air-drying</div>
</li>
<li>Remove the tube from magnetic stand and add 20 μL of nuclease-free water to elute the DNA. Mix by vortexing or gently pipetting.</li>
<li>Place the reaction tube in the magnetic frame after a brief centrifugation. Let the tube rest until the solution completely clarifies.</li>
<li>Carefully remove 18 μL of supernatant to a new tube
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The size selected libraries can be stored at −20°C until ready for library quality control.</div>
</li>
<li>Run 1 μL of the pooled size selected library (diluted 1:3 in H2O) using the High Sensitivity DNA kit according to the manufacturer’s recommendations (<a href="https://www.agilent.com/cs/library/usermanuals/Public/G2938-90321_SensitivityDNA_KG_EN.pdf">https://www.agilent.com/cs/library/usermanuals/Public/G2938-90321_SensitivityDNA_KG_EN.pdf</a>) and estimate the molarity (<a href="#fig4">Figures 4</a>C and 4D).</li>
</ol>
</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Optimal and suboptimal ATAC-seq libraries</div>
<p>ATAC-seq library quality and quantity were assayed using a Bioanalyzer 2100 (A and B) on the individual libraries after the PCR and initial clean up and (C and D) on the pooled libraries after size fractionation. The peaks of the DNA trace display the periodicity of the chromatin structure representing nucleosome-free regions (NFR), as well as mono-, di-, and multi-nucleosome fragments. (A and B) Examples of DNA traces where (A) a high number of DNA fragments are present in the peak corresponding to the NFR and (B) most DNA represents multi-nucleosome fragments. (C and D) Following pooling of individual libraries and subsequent size fractionation, a further enrichment of DNA fragments in NFR can be achieved, even for suboptimal libraries as observed in B (see also <a href="#sec6.11">troubleshooting 6</a>).</p>
</figcaption>
</figure>
<h3 id="sec3.7">Prepare nuclei for protein analyses</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h for 12 preparations</div>
<ol start="17">
<li>Harvest of protein.
<ol type="a">
<li>Spin down 2 mill nuclei at 600<b>×</b><i>g</i> for 5 min (whole liver nuclei) or use a magnet (bead-bound nuclei).</li>
<li>Completely remove all the supernatant.</li>
<li>Resuspend pellet in 20 μL WB, then add an equal volume of 20 μL NUN extraction buffer and pipet extensively up and down.
<div class="note">
<span class="note-title">Note:</span> After 3–4 rounds of pipetting the solution starts to clump.</div>
</li>
<li>Vortex, spin down shortly and incubate on ice for 20 min.</li>
<li>Spin 10 min at 21,000<b>×</b><i>g</i> at 4°C.</li>
<li>Transfer supernatant to a new tube.</li>
<li>Take a small aliquot out for determination of the protein concentration.
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The nuclear extracts can be stored at −80°C until ready for downstream applications.</div>
</li>
</ol>
</li>
</ol>
<ol start="18">
<li>Determine nuclear protein concentration using e.g., the BCA Protein Assay Kit according to the manufacturer’s recommendations (<a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0011430_Pierce_BCA_Protein_Asy_UG.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0011430_Pierce_BCA_Protein_Asy_UG.pdf</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> 2 mill hepatocyte nuclei yield around 40 μg protein using this procedure and therefore the expected protein concentration should be around 1 μg/μL. If more protein is needed for the downstream applications, the number of input nuclei should be adjusted accordingly.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>From 0.5 mg of pulverized frozen liver derived from healthy male and female mice aged 3–6 months, we recovered in the range of 60–80 mill nuclei following the initial nuclei purification. The number of nuclei obtained per mg decreased significantly when handling livers from obese mice with fatty livers.</p>
<p>Using healthy livers obtained from HEP-INTACT mice, we found that around 70% of the total number of isolated nuclei was GFP+ (<a href="#fig5">Figure 5</a>). When purifying labeled nuclei using the indicated anti-GFP antibody, we typically recovered 50%–70% of the GFP+ nuclei, presumably due to some loss of nuclei in the washing steps. Importantly, we observed a comparable high purity (&gt; 95%) of nuclei recovered from fresh and frozen livers (<a href="#fig5">Figure 5</a>). Of note, we found some GFP+ nuclei in the supernatant following the IP when purifying nuclei from frozen liver tissue (<a href="#fig5">Figure 5</a>). We believe this is due to incomplete exposure of the epitope recognized by the antibody because of incomplete removal of the outer nuclear membrane when isolating nuclei from frozen livers.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Comparison of INTACT preparations from fresh and frozen liver tissues</div>
<p>Percentage of GFP+ nuclei in the whole liver, as well as in the bead-bound and supernatant fractions from fresh or frozen liver samples (n=6). Colored bars with percentage number represent mean and vertical lines indicate ±SEM. 2-way ANOVA with post-hoc Tukey’s multiple comparison test as indicated; Statistical significance is indicated by ∗∗∗∗p&lt;0.0001, ns = not significant.</p>
</figcaption>
</figure>
<p>Using frozen livers from healthy MAC-INTACT mice, which consist of 7%–10% GFP+ nuclei, we typically recovered around 70% of the GFP+ nuclei and achieved a purity of 90%–95%.</p>
<p>We further tested that downstream sequencing assays (i.e., RNA and ATAC-seq) worked equally well for INTACT preparations from fresh and frozen liver isolated from HEP-INTACT mice subjected to a 16 h overnight fast (<a href="#fig6">Figure 6</a>). Here, principal component analysis of the nuclear RNA-seq data demonstrated that GFP+ nuclei isolated from fresh and frozen livers colocalize and separate strongly from GFP- nuclei obtained from fresh livers (<a href="#fig6">Figure 6</a>A). Furthermore, clustering analyses of ATAC-seq data from the GFP+ nuclei fractions showed that nuclei from fresh and frozen livers clustered together both in the fed and in the fasting conditions (<a href="#fig6">Figure 6</a>B). Importantly, for both differentially expressed genes and differentially accessible genomic sites (fresh versus frozen) we found a strong correlation for GFP+ nuclei from fresh and frozen livers (<a href="#fig6">Figures 6</a>C and 6D). See <a href="#bib7">Loft et al., 2021</a> for details on data analyses.</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Performance of bead-bound nuclei isolated from fresh and frozen livers in RNA- and ATAC-seq assays</div>
<p>(A) Principal component (PC) analyses of the RNA-seq data from GFP+ (bead-bound) nuclei obtained from fresh or frozen livers and GFP- (supernatant) obtained from fresh livers of HEP INTACT mice subjected to ad libitum feeding (Ad lib) or an overnight 16 h fast (fasted) (n=3).</p>
<p>(B) Clustering of the sample-to-sample distance of the 1000 ATAC-seq sites with highest variance identified in GFP+ nuclei isolated from fresh and frozen livers of HEP INTACT mice subjected to ad libitum feeding (Ad lib) or an overnight 16 h fast (fasted) (n=3).</p>
<p>(C and D) Correlation of log2 fold changes (log2 FC) for all (C) differentially expressed genes or (D) differentially accessible sites identified in GFP+ nuclei isolated from fresh and frozen livers (Ad libitum fed vs fasted, padj&lt;0.01, log2 FC &lt; -1 or &gt; 1). Red, genes/sites only differentially expressed/accessible in the fresh condition; orange, genes/sites only differentially expressed/accessible in the frozen condition; gray, genes/sites differentially expressed/accessible in both conditions. <i>r</i> indicates Pearson’s correlation.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The nuclei isolation protocol presented here has not been tested for compatibility with single cell approaches; however, it is likely that the Dynabead-based enrichment of nuclei is not compatible with e.g., 10x Chromium Controller system due to the interference of the magnetic Dynabeads with single cell partitioning. Alternative magnetic beads, e.g., Miltenyi Biotec Microbeads, should however be fully compatible with the 10x single cell systems and may be implemented in the current protocol as a replacement of Dynabeads, if single cell analyses are required.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Liver tissue is not homogenized after the suggested number of strokes (step 3)</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Since the protocol starts with pulverized tissue, 5–10 stokes with each pestle will normally be sufficient. However, the amounts of strokes may vary depending on the douncers used and might need to be further optimized. In our hands, pulverized fibrotic livers did not require additional strokes compared to healthy livers. This protocol uses a 15 mL douncer and up to 1 g of pulverized liver tissue can be effectively dounced using the appropriate amount of LSB (i.e., 4 mL LSB per 0.5 g tissue). It is very important not to overload the douncer with tissue and therefore advisable to dounce multiple rounds if a whole mouse liver (&gt;1 g) is needed and combine before proceeding with the IP.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Nuclei are of poor quality after the initial isolation (steps 3 and 4)</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Excessive homogenization might negatively impact the quality of the nuclei. If the nuclei are of a poor quality after the initial isolation, it might be necessary to reduce the number of strokes during douncing. Alternatively, one can try to reduce the incubation time in LSB incl. 0.35% IGEPAL (step 3f) or reduce the concentration of IGEPAL to less than 0.35%.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Nuclei are aggregating after the initial isolation (step 3 and 4)</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Using the suggested protocol, we did not experience nuclei aggregations after the initial douncing procedure. However, it is important to completely resuspend the nuclei after the HSB spin, which require that the suggested spin time of 15 min is not exceeded. During the optimization of this protocol, we found that some alternative nuclei isolation protocols using different buffers (e.g., Nuclei EZ Prep (NUC101-1KT, Sigma)) tended to yield a significant number of aggregating nuclei after the initial isolation (<a href="#fig7">Figure 7</a>A). Therefore, we cannot recommend the use of alternative nuclei isolation buffers unless tested thoroughly.</p>
<figure id="fig7"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/988-Fig7.jpg" alt="Fig7.jpg">
<figcaption>
<div class="figcaption-title">Figure 7. Aggregation of nuclei</div>
<p>Trypan blue-stained nuclei isolated from frozen livers showing nuclei aggregation either due to (A) the use of Nuclei EZ Prep in the initial nuclei isolation or due to (B) the use of an anti-MYC antibody in the IP or (C) vigorously shaking following the IP. Scale bars represent 200 μm.</p>
</figcaption>
</figure>
<h3 id="sec6.7">Problem 4</h3>
<p>Nuclei are aggregating after the IP (steps 7 and 8)</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Using the suggested protocol, we generally did not find that nuclei aggregation was a major problem after the IP. However, during the optimization of the protocol, we found that some preparations yielded nuclei that clumped heavily together (<a href="#fig7">Figure 7</a>B), which seemed to be highly dependent on the antibody used and the stress posed to the nuclei during the IP. If single nuclei preparations are a strict requirement, it is possible to increase the fraction of these by using a 20 μM filter before or during the IP, but this will result in significant loss of material. It is also possible to add 0.05%–1% BSA to the buffers but note that this should not be done for preparations intended for protein analyses.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Low yield of bead-bound nuclei (steps 7 and 8)</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The yield varies greatly depending on the cell type being recovered and the antibody used for the IP and alternative antibodies can be tested to potentially increase the yield. In the original INTACT protocol (<a href="#bib8">Mo et al., 2015</a>), it was suggested that the yield can be increased by performing repeated cycles (5–7 times) of placing the bead-bound nuclei on the magnet stand and then completely resuspending these by inversion (in step 7g). However, at least for hepatocyte nuclei isolated from the liver, we experienced that this led to dramatic aggregation of the hepatocyte nuclei (<a href="#fig7">Figure 7</a>C), making it difficult to use these nuclei for downstream approaches.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Suboptimal ATAC-seq trace after PCR amplification of transposed DNA (steps 13–16)</p>
<h3 id="sec6.12">Potential solution</h3>
<p>If only a small fraction of DNA is NFR fragments after PCR amplification (see <a href="#fig4">Figure 4</a>B), more material will have to be used for the size fractionation to obtain an adequate amount of NFR-associated DNA fragments for sequencing. Adjusting the conditions of the transposition reaction (e.g., the nuclei-to-enzyme ratio or the reaction time) and/or the number of PCR cycles might be able to remedy this problem.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Buenrostro, J.D., Wu, B., Chang, H.Y., and Greenleaf, W.J. (2015). ATAC-seq: A method for assaying chromatin accessibility genome-wide. Curr. Protoc. Mol. Biol. <i>109</i>, 21 29 21-29. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref1">View at publisher</a></p>
<p id="bib2">Clausen, B.E., Burkhardt, C., Reith, W., Renkawitz, R., and Forster, I. (1999). Conditional gene targeting in macrophages and granulocytes using LysMcre mice. Transgenic Res. <i>8</i>, 265-277. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref2">View at publisher</a></p>
<p id="bib3">Corces, M.R., Greenleaf, W.J., and Chang, H.Y. (2019). Isolation of Nuclei from Frozen Tissue for ATAC-Seq and Other Epigenomic Assays (Protocolsio). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref3">View at publisher</a></p>
<p id="bib4">Corces, M.R., Trevino, A.E., Hamilton, E.G., Greenside, P.G., Sinnott-Armstrong, N.A., Vesuna, S., Satpathy, A.T., Rubin, A.J., Montine, K.S., Wu, B., et al. (2017). An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues. Nat. Methods <i>14</i>, 959-962. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref4">View at publisher</a></p>
<p id="bib5">Deal, R.B. and Henikoff, S. (2010). A simple method for gene expression and chromatin profiling of individual cell types within a tissue. Dev. Cell <i>18</i>, 1030-1040. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref5">View at publisher</a></p>
<p id="bib6">Liang, N., Fan, R., Goni, S., and Treuter, E. (2019). preparation of frozen liver tissues for integrated omics analysis. Methods Mol. Biol. <i>1951</i>, 167-178. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref6">View at publisher</a></p>
<p id="bib7">Loft, A., Alfaro, A.J., Schmidt, S.F., Pedersen, F.B., Terkelsen, M.K., Puglia, M., Chow, K.K., Feuchtinger, A., Troullinaki, M., Maida, A., et al. (2021). Liver-fibrosis-activated transcriptional networks govern hepatocyte reprogramming and intra-hepatic communication. Cell Metab. <i>33</i>, 1685-1700.e9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref7">View at publisher</a></p>
<p id="bib8">Mo, A., Mukamel, E.A., Davis, F.P., Luo, C., Henry, G.L., Picard, S., Urich, M.A., Nery, J.R., Sejnowski, T.J., Lister, R., et al. (2015). Epigenomic signatures of neuronal diversity in the mammalian brain. Neuron <i>86</i>, 1369-1384. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref8">View at publisher</a></p>
<p id="bib9">Postic, C., Shiota, M., Niswender, K.D., Jetton, T.L., Chen, Y., Moates, J.M., Shelton, K.D., Lindner, J., Cherrington, A.D., and Magnuson, M.A. (1999). Dual roles for glucokinase in glucose homeostasis as determined by liver and pancreatic beta cell-specific gene knock-outs using Cre recombinase. J. Biol. Chem. <i>274</i>, 305-315. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00511-6/sref9">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Søren Fisker Schmidt, <a href="mailto:sfs@bmb.sdu.dk">sfs@bmb.sdu.dk</a>.</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>The accession codes for the INTACT ATAC-seq and RNA-seq data reported in this paper are GEO: GSE181937 and GEO: GSE181940.</p>
<h3>Acknowledgments</h3>
<p>We thank colleagues for input to the protocol and acknowledge the team of technicians at IDC for assistance with mouse work. The graphical abstract was created with <a href="http://BioRender.com">BioRender.com</a>. This work was financially supported by grants from the Danish National Research Foundation (DNRF grant No. 141 to the Center for Functional Genomics and Tissue Plasticity [ATLAS]), the Novo Nordisk Foundation (NNFOC150019050) to S.F.S., and the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> (DFG, German Research Foundation) through the Collaborative Research Center (CRC) 1118, project number 236360313 to S.H.. A.L. is supported by fellowships from the <a href="https://doi.org/10.13039/501100011747">Novo Nordisk Foundation</a> (NNF16OC0020742), the <a href="https://doi.org/10.13039/100008392">Danish Independent Research Council</a> | Medical Sciences (DFF-6110-00265), an <a href="https://doi.org/10.13039/100004410">EMBO</a> Long-Term Fellowship (ALTF 897–2015), and a DZD grant NEXT.</p>
<h3>Author contributions</h3>
<p>Conceptualization, A.L. and S.F.S.; methodology, A.L. and S.F.S.; formal analyses, A.L. and S.F.S.; investigation, A.L. and S.F.S.; resources, A.L., S.H., and S.F.S.; writing – original draft, A.L. and S.F.S.; writing – review &amp; editing, A.L., S.H., and S.F.S.; visualization, A.L.; project administration, A.L., S.H., and S.F.S.; funding acquisition, A.L., S.H., and S.F.S.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>