<article>
<section>
<h1 id="header">Sucrose gradient chromatin enrichment for quantitative proteomics analysis in budding yeast</h1>
<p><time datetime="2021-09-17">Published: September 17, 2021</time></p>
<p>Kiran Challa,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn3">4</a>,<a href="#cor1">*</a></sup> Jan Seebacher,<sup><a href="#aff1">1</a></sup> and Susan M. Gasser<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#fn4">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Friedrich Miescher Institute for Biomedical Research, Maulbeerstrasse 66, 4058 Basel, Switzerland</p>
<p id="fn1"><sup>2</sup>Present address: Laboratory of Mechano Genomics, Paul Scherrer Institute, 5232 Villigen, Switzerland</p>
<p id="fn2"><sup>3</sup>Present address: ISREC Foundation, AGORA Cancer Center, Chemin du Bugnon 25A, 1005 Lausanne, Switzerland</p>
<p id="fn3"><sup>4</sup>Technical contact</p>
<p id="fn4"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:kiran.challa@psi.ch">kiran.challa@psi.ch</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:susan.gasser@fmi.ch">susan.gasser@fmi.ch</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100825">10.1016/j.xpro.2021.100825</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Here, we describe a fractionation protocol optimized to quantify changes in relative abundance of the chromatin-bound proteome (chromatome) by tandem mass tag multiplexing-based tandem mass spectrometry. It has been applied to yeast cells before and after exposure to DNA-damaging drugs to characterize changes in chromatin composition induced by the DNA damage response. We detail steps for stringent chromatin fractionation, sample preparation for mass spectrometry, and its evaluation.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref1">Challa et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for yeast chromatin-associated proteome isolation and analysis</li>
<li>TMT multiplex mass tag labeling of chromatin-bound proteome</li>
<li>Sample preparation for mass spectrometry and proteomics data analysis</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1004-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">BEFORE YOU BEGIN</h2>
<p>This protocol is designed to increase the sensitivity and detection of quantitative changes in the abundance of nuclear or chromatin-bound proteins, under various conditions using haploid budding yeast (<i>Saccharomyces cerevisiae</i>). In our case strains in the W303-1A background were used (<a href="#bib1">Challa et al., 2021</a>), but the protocol is applicable to any haploid <i>MAT</i><b><i>a</i></b> strain. Chromatin-bound proteins make up a small fraction (in amount, not in complexity) of the yeast proteome. Thus, enriching for chromatin prior to mass spectroscopy greatly enhances the sensitivity with which one can monitor differences. If one wants to compare two conditions (as here, with and without an exogenous DNA damaging agent), it is essential that the strain is first synchronized in the cell cycle so that the changes due to the damaging agent are compared in the same cell cycle phase. This also applies to the comparison of two mutants: the two strains should be at the same stage of the cell division cycle, before and after treatment. We synchronize all cultures by pheromone arrest (G1/S), prior to chemical treatment, but one can also arrest at the G1/S boundary, and then release for progression through S (e.g., in the presence of hydroxyurea (HU) or nocodazole to arrest at G2/M phase). Cell cycle differences should be expected and may mask or bias changes arising from experimental conditions.</p>
<p>It is recommended to characterize the strains used in this protocol for cell division characteristics, arrest efficiency, and response to the conditions imposed, prior to undertaking TMT multiplexing-based tandem mass spectrometry. Later, controlling for enrichment of factors that one expects to be enriched or depleted (e.g., replication or repair enzymes, transcription machineries) can be used to validate more subtle changes. As described here, the protocol is not designed to optimize the detection of post-translational modifications (PTMs), although we were able to detect states of acetylation, methylation and ubiquitination on histones and other abundant chromatin factors (for Hho1, see <a href="#bib1">Challa et al., 2021</a>). Minor changes to the protocol would be needed to improve recovery and detection of PTMs.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Rad53</td>
<td><a href="#bib4">Hauer et al., 2017</a></td>
<td>Gasser Lab</td>
</tr>
<tr>
<td>Mouse monoclonal anti-MYC</td>
<td>FMI Antibody service</td>
<td>#9E10</td>
</tr>
<tr>
<td>Rat monoclonal anti-Tubulin</td>
<td>Bio-Rad</td>
<td>MCA78G</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>PIPES</td>
<td>Calbiochem</td>
<td>Cat# 528132</td>
</tr>
<tr>
<td>EDTA</td>
<td>Sigma</td>
<td>Cat# ED2SS</td>
</tr>
<tr>
<td>Na-azide</td>
<td>Merck</td>
<td>Cat# 6688.01</td>
</tr>
<tr>
<td>DTT</td>
<td>Sigma</td>
<td>Cat# D-6052</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>Merck</td>
<td>Cat# 1.05099</td>
</tr>
<tr>
<td>KH<sub>2</sub>PO<sub>4</sub>
</td>
<td>Merck</td>
<td>Cat# 1.05108</td>
</tr>
<tr>
<td>Sorbitol</td>
<td>Sigma</td>
<td>Cat# S-1876</td>
</tr>
<tr>
<td>β-Mercaptoethonol</td>
<td>Sigma</td>
<td>Cat# M-6250</td>
</tr>
<tr>
<td>Zymolase 100T</td>
<td>US Biological</td>
<td>Cat# Z1004</td>
</tr>
<tr>
<td>PMSF</td>
<td>Sigma</td>
<td>Cat# P-7626</td>
</tr>
<tr>
<td>cOmplete EDTA-free Protease Inhibitor</td>
<td>Roche (Sigma)</td>
<td>Cat# 5056489001</td>
</tr>
<tr>
<td>HEPES</td>
<td>Merck</td>
<td>Cat# 15230</td>
</tr>
<tr>
<td>Spermidine</td>
<td>Sigma</td>
<td>Cat# S-2501</td>
</tr>
<tr>
<td>TritonX100</td>
<td>Sigma</td>
<td>Cat# X-100</td>
</tr>
<tr>
<td>Sucrose</td>
<td>Serva</td>
<td>Cat# 35579</td>
</tr>
<tr>
<td>Formaldehyde</td>
<td>Thermo Scientific</td>
<td>Cat# 28908</td>
</tr>
<tr>
<td>Ultra Clear Beckman Coulter-14mL</td>
<td>Beckman Coulter</td>
<td>Cat# 344060</td>
</tr>
<tr>
<td>Glycine</td>
<td>Merck</td>
<td>Cat# 1.04201</td>
</tr>
<tr>
<td>EGTA</td>
<td>Sigma</td>
<td>Cat# E-4378</td>
</tr>
<tr>
<td>Na<sub>3</sub>VO<sub>4</sub>
</td>
<td>Sigma</td>
<td>Cat# S6508</td>
</tr>
<tr>
<td>SDS</td>
<td>Sigma</td>
<td>Cat# L-4509</td>
</tr>
<tr>
<td>Urea</td>
<td>Sigma</td>
<td>Cat# U-0631</td>
</tr>
<tr>
<td>RNaseA</td>
<td>Sigma</td>
<td>Cat# R-5125</td>
</tr>
<tr>
<td>Benzonase</td>
<td>Sigma</td>
<td>Cat# E1010-5KU</td>
</tr>
<tr>
<td>Hydroxylamine</td>
<td>Thermo Scientific</td>
<td>Cat# 90115</td>
</tr>
<tr>
<td>Na-citrate</td>
<td>Merck</td>
<td>Cat# 12005</td>
</tr>
<tr>
<td>Propidium iodide</td>
<td>Sigma</td>
<td>Cat# P-4170</td>
</tr>
<tr>
<td>NuPAGE</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# NP0322BOX</td>
</tr>
<tr>
<td>Zeocin</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 25005</td>
</tr>
<tr>
<td>Anhydrous acetonitrile</td>
<td>Sigma</td>
<td>Cat# SIAL271004</td>
</tr>
<tr>
<td>Ammonium formate</td>
<td>Sigma</td>
<td>Cat# 17843</td>
</tr>
<tr>
<td>YMC-Triart C18 0.5 × 250 mm column</td>
<td>YMC Europe GmbH</td>
<td>Cat# TA12S03-25J0AU</td>
</tr>
<tr>
<td>Ammonium hydroxide solution for HPLC</td>
<td>Sigma</td>
<td>Cat# 17837</td>
</tr>
<tr>
<td>PepMap 100 C18 2 cm trap</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 164946</td>
</tr>
<tr>
<td>15 cm EASY-Spray C18 column</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# ES801</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>iST NHS 12× sample kit</td>
<td>PreOmics</td>
<td>Cat# PO00026</td>
</tr>
<tr>
<td>Qubit<sup>TM</sup> Protein Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# Q33211</td>
</tr>
<tr>
<td>Qubit<sup>TM</sup> RNA HS Assay Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# Q32852</td>
</tr>
<tr>
<td>TMT 6plex™ Isobaric Label Reagent Set, 1 × 0.8 mg</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 90061</td>
</tr>
<tr>
<td>TMT10plex™ Isobaric Label Reagent Set, 8 × 0.2 mg</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 90309</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Proteomics raw data PRIDE database accession code</td>
<td>This method; <a href="#bib1">Challa et al., 2021</a>
</td>
<td>PXD022369</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Genotype</td>
<td>Background</td>
<td>Strain number</td>
</tr>
<tr>
<td>
<i>MAT</i><b><i>a</i></b><i>, ade2-1 , trp1-1 , his3-11 , -15, ura3-1, leu2-3, -112, can1-100, RAD5+, RAD52-YFP; NUP49-GFP; ADE2::TetR-mCherry; lys5::LacI-CFP::TRP; leu2::LoxP; ZWF1: (Lmn::lys5::IsceIcs::LEU2::LacO array::Lmn); met10::lmn adaptamers::HIS3::TetOps-LexA</i>
</td>
<td>W303-1A; <a href="#bib4">Hauer et al., 2017</a>
</td>
<td>GA-6879</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>RStudio</td>
<td>N/A</td>
<td><a href="https://rstudio.com/products/rstudio/download/%20Version%201.2.5033">https://rstudio.com/products/rstudio/download/ Version 1.2.5033</a></td>
</tr>
<tr>
<td>R</td>
<td>N/A</td>
<td><a href="https://www.r-project.org/version%204.0.0">https://www.r-project.org/version 4.0.0</a></td>
</tr>
<tr>
<td>Proteome Discoverer PD2. 2.2.0.388</td>
<td>N/A</td>
<td><a href="https://www.thermofisher.com/order/catalog/product/OPTON-30945#/OPTON-30945">https://www.thermofisher.com/order/catalog/product/OPTON-30945#/OPTON-30945</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Orbitrap Fusion Lumos Tribrid Mass Spectrometer</td>
<td>Thermo Fisher Scientific</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Reagent preparation</h3>
<ul>
<li>
<b>1 M DTT:</b> Dissolve 1.54 g of 1,4-Dithiothreitol in 8 mL of dH<sub>2</sub>O and adjust the volume to 10 mL. Store aliquots at −20°C for 6 months.</li>
<li>
<b>0.5 M EDTA-KOH (pH 8.0):</b> Dissolve 186.1 g Ethylenediaminetetraacetic disodium salt dihydrate with 900 mL dH<sub>2</sub>O. Adjust pH with KOH and bring total volume to 1 liter (L). Store at 22°C–25°C (room temperature) for 3–6 months.</li>
<li>
<b>0.5 M EGTA-NaOH (pH 8.0):</b> Dissolve 19.0 g of Ethylene glycol-bis(β-aminoethyl ether)-N,N,N’N’-tetraacetic acid in 60 mL of dH<sub>2</sub>O and adjust the pH with NaOH. Bring total volume to 100 mL using dH<sub>2</sub>O. Store at 22°C–25°C for 6 months.</li>
<li>
<b>1 M Glycerol-2-phosphate:</b> Dilute 30.6 g glycerol phosphate disodium salt pentahydrate in 80 mL of dH<sub>2</sub>O and adjust volume to 100 mL. Store at 4°C for 6 months.</li>
<li>
<b>1.25 M glycine:</b> Dissolve 9.38 g of glycine in 100 mL (final vol) 1× PBS (pH 7.4). Store the solution at 4°C.</li>
<li>
<b>1 M HEPES-NaOH (pH 7.5):</b> Dilute 238.3 g HEPES in 900 mL dH<sub>2</sub>O. Adjust the pH with NaOH and bring volume to 1 l with dH<sub>2</sub>O. Store at 4°C.</li>
<li>
<b>2 M KCl:</b> Dilute 149.1 g of KCl in 900 mL of dH<sub>2</sub>O and bring the volume to 1 L after dissolving. Store at 22°C–25°C for 6 months.</li>
<li>
<b>1 M MgCl</b><sub><b>2</b></sub><b>:</b> Dissolve 203.3 g MgCl<sub>2</sub>•6H<sub>2</sub>O in 900 mL of dH<sub>2</sub>O and adjust the volume to 1 L after dissolving. Store at 22°C–25°C for 6 months.</li>
<li>
<b>200 mM NaF:</b> Dissolve 0.839 g of sodium fluoride in 90 mL of dH<sub>2</sub>O and adjust the volume to 100 mL using dH<sub>2</sub>O. Store the solution at 4°C for 6 months</li>
<li>
<b>10% Na-Azide:</b> Dissolve 10 g of NaN<sub>3</sub> in 90 mL of dH<sub>2</sub>O and bring the volume to 100 mL with dH<sub>2</sub>O. Store the solution at 4°C for 2–3 weeks.</li>
<li>
<b>10× PBS:</b> In 800 mL of dH<sub>2</sub>O dissolve 80 g of NaCl, 2 g of KCl, 17.4 g of Na<sub>2</sub>HPO<sub>4</sub> •2H<sub>2</sub>0 and 2.4 g of KH<sub>2</sub>PO<sub>4</sub>. Adjust the pH to 7.4 with HCl and bring the volume to 1 L with dH<sub>2</sub>O.</li>
<li>
<b>0.5 M PIPES-KOH (pH 7.4):</b> Dilute 75.6 g of PIPES in 400 mL of dH<sub>2</sub>O and adjust the pH with KOH. Bring the volume to 500 mL using dH<sub>2</sub>O. Store at 22°C–25°C for 3–6 months.</li>
<li>
<b>200 mM PMSF:</b> Dissolve 3.48 g phenylmethylsulfonyl fluoride in 80 mL of ethanol (100%). After dissolving completely, bring up to 100 mL with ethanol. Store at −20°C or make fresh for use.</li>
<li>
<b>1 M Potassium Phosphate Buffer:</b> Dissolve 9.3 g KH<sub>2</sub>PO<sub>4</sub> and 6.3 g K<sub>2</sub>HPO<sub>4</sub> in 80 mL of dH<sub>2</sub>O. Adjust the pH 7 with KOH. Adjust the volume to 100 mL with dH<sub>2</sub>O. Filter sterilize and store at 22°C–25°C for 3–6 months.</li>
<li>
<b>2 M sorbitol:</b> Dissolve 182 g of sorbitol in 400 mL of dH<sub>2</sub>O and adjust the volume to 500 mL. Store at 4°C for 6 months.</li>
<li>
<b>50 mM Sodium orthovanadate (Na</b><sub><b>3</b></sub><b>VO</b><sub><b>4</b></sub><b>):</b> Dissolve 0.92 g Na<sub>3</sub>VO<sub>4</sub> in 95 mL dH<sub>2</sub>O. Adjust the solution to pH 10.0, at which point the solution turns yellow. Boil the solution for 10 min to clarify. If necessary, adjust the pH one more time and adjust the final volume to 100 mL. Store aliquots at −20°C.</li>
<li>
<b>0.5 M Spermidine:</b> Dissolve 1.09 g of spermidine free base in 13.77 mL of dH<sub>2</sub>O for a final volume of 15 mL. Store aliquots at −20°C.</li>
<li>
<b>0.5 M Spermine:</b> Dissolve 5.08 g of spermine in 49.43 mL of dH<sub>2</sub>O for a final volume of 50 mL. Store aliquots at −20°C.</li>
<li>
<b>1 M Tris-HCl (pH 8.0):</b> Dilute 121 g Tris base (Tris(hydroxymethyl)aminomethane) in 900 mL of dH<sub>2</sub>O. Adjust pH to 8.0 with HCl, and bring total volume to 1 L using dH<sub>2</sub>O. Store at 22°C–25°C for 3–6 months.</li>
<li>
<b>10% Triton X-100:</b> Add 5 mL of Triton X-100 to 45 mL of dH<sub>2</sub>O. Stir gently for at least 1 h for complete dispersion. Store at 4°C for up to 3 months.</li>
<li>
<b>0.1 M ZnSO</b><sub><b>4</b></sub><b>:</b> Dilute 2.88 g of zinc sulfate heptahydrate (ZnSO<sub>4</sub>·7H<sub>2</sub>0) in 70 mL of dH<sub>2</sub>O and adjust the volume to 100 mL using dH<sub>2</sub>O. Store at 4°C for 6 months.</li>
<li>
<b>Formaldehyde solution</b>: Formaldehyde can be used from a high-grade commercially available 37% stock (however, the commercial stock may have formed long polymers already, thus it is preferable to dissolve para-formaldehyde freshly to ensure maximal reactivity with short formaldehyde chains). To dissolve para-formaldehyde, see the immunofluorescence protocol in (<a href="#bib10">Meister et al., 2010</a>). For best results do not store freshly dissolved para-formaldehyde and use immediately.</li>
<li>
<b>200 mM ammonium formate:</b> Dissolve 12.6 g of ammonium formate in 950 mL of dH<sub>2</sub>O. Adjusted to pH 10 with 10% ammonium hydroxide and adjust final volume to 1 L using dH<sub>2</sub>O.</li>
</ul>
<p><b>Buffer A (10X):</b> 200 mM HEPES-KOH (pH7.5), 800 mM KCl, 80 mM EDTA-KOH, 5 mM spermidine and 2 mM spermine (check pH after addition of all reagents and adjust to pH=7.5 as necessary. Prepare 100 mL of stock and store at 4°C for a month).</p>
<p><b>Buffer B:</b> 100 mM PIPES-KOH (pH 7.4), 0.1M EDTA-KOH, 0.1% Na-Azide, 10 mM DTT. Always prepare fresh buffer and 100 mL of buffer is sufficient for two 1 L cultures.</p>
<p><b>Buffer C:</b> 5 0mM K-Phosphate buffer (pH 7.0), 1.1 M sorbitol, 1 mM 2-Mercaptoethanol, 0.5 mM MgCl<sub>2</sub>. Always prepare fresh buffer and 40 mL of buffer is sufficient for two 1 L cultures.</p>
<p><b>Spheroplast Wash Buffer:</b> 0.25x Buffer A (10X), 1 M sorbitol, 0.5 mM PMSF, Protease inhibitor 1x (Roche). Always prepare fresh buffer and 40 mL of buffer is sufficient for two 1 L cultures.</p>
<p><b>E Buffer:</b> 30 mM KCl, 50 mM HEPES-KOH (pH7.5), 2.5 mM MgCl<sub>2</sub>, 0.1 mM ZnSO<sub>4</sub>, 2 mM NaF, 0.5 mM spermidine, 10 mM glycerol-2-phosphate, 0.1 mM Na<sub>3</sub>VO<sub>4</sub>, 0.2 mM PMSF, Roche Protease inhibitors 1x, Roche phosphatase inhibitors 1x. Always prepare fresh buffer and 50 mL of buffer is sufficient for two 1 L cultures.</p>
<p><b>30% Sucrose cushion:</b> Dissolve 9 g of sucrose in 28.8 mL of E buffer and adjust total volume to 30 mL using E buffer. Always prepare fresh solution.</p>
<p><b>TEE Buffer:</b> 1 0mM Tris-HCl (pH 8.0), 1 mM EDTA, 0.5 mM EGTA, 1 mM PMSF, 10 mM glycerol-2-phosphate, 0.1 mM Na<sub>3</sub>VO<sub>4</sub>, Roche protease inhibitors 1x. Always prepare fresh buffer and 20 mL of buffer is sufficient for two 1 L cultures.</p>
<p><b>Urea Buffer:</b> Dissolve 9.6 g urea in 20 mL of TEE buffer. Always prepare fresh urea buffer.</p>
<p><b>Zymolyase 100T:</b> Dissolve 20 mg/mL of Zymolyase in 50 mM Tris-HCl (pH 8). Store aliquots at −20°C for 6 months, although freshly prepared Zymolyase solutions give more consistent cell lysis in each experiment. Alternatively, purified lyticase (β-glucanase of <i>Oerskovia xanthineolytica</i>) can be used (<a href="#bib14">Verdier et al., 1990</a>).</p>
<p>For mass spectrometry we used an Orbitrap Fusion or Orbitrap Fusion Lumos Tribrid mass spectrometer equipped with an EASY spray source (both Thermo Fisher Scientific).</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step by step method details</h2>
<h3 id="sec3.1">Part 1: Budding yeast cell culture, treatment, and chromatin isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<h4 id="sec3.1.1">Budding yeast cell culture, synchronization, and DNA damage treatment</h4>
<ol>
<li>Grow 2 L of the desired <i>MAT</i><b><u>a</u></b> budding yeast strain for 14–16 h (h) in YPAD media at 30°C until logarithmic growth phase (OD600= 0.4–0.6, equivalent to 5 × 10<sup>6</sup> cells/mL). See <a href="#fig1">Figure 1</a> for workflow of the protocol. We used 1 l per treatment per strain analyzed, thus from a 2 L synchronized culture one derives one treated sample and one control (<a href="#bib1">Challa et al., 2021</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1004-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Workflow for sucrose gradient chromatin enrichment for mass spectrometry protocol</div>
<p>See text for details. Part of this figure is reprinted with permission from <a href="#bib1">Challa et al. (2021)</a>.</p>
</figcaption>
</figure>
</li>
<li>Spin the cells, 2000 × <i>g</i>, 5 min, wash 1x with 50 mL H<sub>2</sub>O and replace with 2 L fresh YPAD containing alpha factor (1 mL, 1:2000, stock 5 mg/mL). The appropriate alpha factor dose varies among strains, thus the optimal amount for G1 arrest by 60–90 min, such that the culture has &lt;10% budded cells (bud emergence correlates with entrance into S phase), should be determined ahead of time.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> if comparing two strains, it is important that they synchronize similarly, as pheromone induces a physiological mating program. If one strain forms shmoos and the other does not (<a href="#fig2">Figure 2</a>A), this will affect gene expression and the proteome. The goal of synchronization is to achieve 90% arrest at G1 with no small buds, without inducing shmoo formation.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1004-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Reproducibility and chromatin enrichment measurements</div>
<p>(A) Sketch of successful alpha factor arrest and inappropriate shmoo formation, which is indicative of too much alpha factor. Shown below are representative light microscope images of alpha factor arrested cells before and after incubation with nonionic detergent, which lyses spheroplasts (wild-type background).</p>
<p>(B) Flow cytometry (FACS) analysis of cell cycle stages for a wild-type background.</p>
<p>(C) Coomassie blue stained SDS-PAGE of WT chromatin fractionated sample showing whole cell extract (WCE), supernatant (SUP), 0 and 8 0min formaldehyde cross-linked, Wash fraction, Chromatin fraction and recombinant histone H3+H4 mix (1μg). Fractions and volumes loaded are shown on top of the gel.</p>
<p>(D) Reproducibility among normalized chromatome (C) samples of the indicated yeast strains (for full genotype see <a href="#bib1">Challa et al., 2021</a>) in Undamaged (UD) and Damaged (D) TMT 9plex samples. The samples a, b, c, for each condition (UD or D) are separate biological replicates of one strain.</p>
<p>(E) Correlation plot for chromatome components vs total proteome components. All DNA binding proteins (highlighted in green; 96%) are enriched in the chromatome, as are histones (highlighted in purple). This validates our purification protocol. Panels C, D and E of <a href="#fig2">Figure 2</a> are reprinted with permission from <a href="#bib1">Challa et al. (2021)</a>.</p>
</figcaption>
</figure>
<ol start="3">
<li>Incubate the cultures with shaking at 25°C for 60 min and add another full dose of alpha factor (1 mL). Incubate a further 40 min (check by light microscopy for arrest without new buds) and collect the samples for FACS analysis before and after alpha factor arrest to confirm cell cycle stage (<a href="#fig2">Figure 2</a>B).</li>
<li>Split the synchronized cells into two 1 L cultures and start treatment by adding Zeocin® to final concentration 300 μg/mL to one culture only and incubate a further 60 min at 25°C. The control should receive an equal volume of the solvent minus Zeocin®. Collect 10 mL samples for Western blot analysis before and after Zeocin® treatment to check the DNA damage status. This can be done with antibodies that identify DNA damage-induced markers such as phosphorylated Rad53 or γH2A (see Western blot sample collection and processing below). Other DNA damaging agents can be used. For instance, cells can be released from alpha factor-containing media into 0.2M hydroxyurea, to monitor replication fork arrest induced changes in chromatin (<a href="#bib5">Hurst et al., 2021</a>, in revision). Cell cycle progression should be comparable. Check by FACS and microscopy.</li>
<li>Spin down cells at 25°C; 2000 × <i>g</i>, 5 min and wash 1× with 50 mL PBS (2000 × <i>g</i>, 3 min spin).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Avoid storing pellets in −80°C and continue the next steps on same day for better yield of proteins.</div>
<h4 id="sec3.1.2">Spheroplasting and cell lysis</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="6">
<li>Resuspend cells in 45 mL Buffer B. Incubate at 22°C–25°C (room temperature) slowly shaking/rotating for 10 min in 50 mL Falcon® tube or equivalent. The sodium azide (NaN<sub>3</sub>) in the buffer will kill the cells.</li>
<li>Spin down 2000 × <i>g</i> for 2 min and resuspend thoroughly in 20 mL Buffer C in a 50 mL Falcon tube.</li>
<li>Add 750 μL Zymolyase® T100 stock (20 mg/mL in H<sub>2</sub>O – final concentration about 0.75 mg/mL) and incubate at 30°C shaking very slowly for 15–20 min. Invert the tubes every 5 min, and check for efficient spheroplasting by mixing a 1:1 ratio of sample and E Buffer +0.5% TritonX-100. Observe under a light microscope for the percentage of lysed vs unlysed cells (see <a href="#fig2">Figure 2</a>A) to determine when spheroplasting is complete (&gt;80%–90% of cells should lyse).</li>
<li>Spin down spheroplasts at 1500 × <i>g</i>, 2 min at 4°C, and wash in 10 mL of 4°C spheroplast wash buffer (SW buffer) carefully dislodging the pellet with a cut pipet tip or a rubber tipped baton. Spin down at 1500 × <i>g</i>, 2 min, and wash one more time with 10 mL chilled SW buffer.</li>
<li>Resuspend spheroplasts without very extensive pipetting in 3 mL of E buffer that lacks TritonX-100.</li>
<li>Add 5–8 mL of E Buffer +0.5% TritonX-100 so that final volume is 10 mL and mix on ice by inversion. Lyse for approximately 5 min. Check the cell lysis under the microscope (see <a href="#fig2">Figure 2</a>A). Keep 1 × 100 μL and 1 × 500 μL of total cell extract for Western blot and mass spec analysis, respectively.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Thorough resuspension of cells prior to spheroplasting and a thorough but gentle resuspension of spheroplasts in wash buffer using cut pipet tip / rubber capped pasture pipet to avoid unequal suspension or premature lysis, are important. Both may lead to chromatin-bound protein loss.</div>
<h4 id="sec3.1.3">Chromatin fractionation and cross-linking</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="12">
<li>Overlay the total lysis mixture on 5 mL of 30% sucrose in E buffer +0.25% TritonX-100 (with 1.5% formaldehyde final concentration) in an Ultra Clear Beckman Coulter 14 mL tube.</li>
<li>Spin at 11100 rpm (21875 × <i>g</i>) for 15 min at 4°C in a Beckman Coulter SW40 Ti swinging bucket rotor.</li>
<li>Keep 100 μL and 1 mL samples of supernatant (above the sucrose cushion) for Western blotting and mass spectrometry analysis (if necessary).</li>
<li>Remove the supernatant and resuspend pellet gently in 5 mL of E buffer +0.25% TritonX-100 with 1% final formaldehyde concentration (diluted from 37% stock or use freshly dissolved para-formaldehyde, see <b>Reagent preparation</b>).</li>
<li>Transfer to a 5 mL Eppendorf tube and incubate for 60 min at 4°C rotating slowly.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> take samples for different crosslinking time points here: 50 μL at 0 min, 30 min, and 60 min, and again after a further 25 min at 22°C–25°C (85 min), see next step (see <a href="#fig2">Figure 2</a>C)</div>
<ol start="17">
<li>Place the tube at 22°C–25°C and incubate for another 25 min on a rolling device (= 85 min sample)</li>
<li>Add 250 μL of 2.5 M glycine (125 mM final concentration) and incubate for 10 min to quench the crosslinking reaction</li>
<li>Spin 14000 × <i>g</i> for 15 min at 4°C (<i>if desired, keep 500 μL of wash fraction to determine chromatin loss by Western blot for histones or calculation of genomic DNA lost</i>) and discard supernatant.</li>
<li>For subsequent steps, resuspend chromatin pellet carefully, but completely, in a final volume of 4 mL TEE buffer, in a 5 mL Eppendorf tube</li>
</ol>
<h4 id="sec3.1.4">RNase digestion and urea washes</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="21">
<li>To each sample add RNaseA (100x stock) to a final concentration of 0.2 mg/mL and incubate for 20 min at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> keep 100 μL before and after addition of RNaseA to check for proper RNA digestion by Qubit® RNA detection kit</div>
<ol start="22">
<li>Spin 14000 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend pellet carefully (using cut pipette tip) but fully in 0.5 mL TEE buffer + 2% SDS.</li>
<li>Add 1.5 mL urea buffer (freshly prepared 8M urea stock in TEE) and mix thoroughly by inverting the tube several times.</li>
<li>Spin 14000 × <i>g</i> for 5 min at 22°C–25°C (keep 500 μL of supernatant for analysis “Urea wash” (see <a href="#fig2">Figure 2</a>C) and discard supernatant.</li>
<li>Repeat washes twice (first in 0.5 mL TEE buffer + 2% SDS then 1.5 mL TEE + urea)</li>
<li>To wash out the urea, resuspend in 2 mL TEE buffer + 2% SDS.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> try to be fast with the urea washes and do not allow the sample to go above 25°C as urea causes protein carbamylation.</div>
<ol start="28">
<li>Spin 14000 × <i>g</i> for 5 min at 22°C–25°C , discard supernatant.</li>
<li>Add 1.5 mL TEE buffer and only flick tube, do not pipet up and down. Spin 14000 × <i>g</i> for 5 min at 22°C–25°C, discard supernatant.</li>
</ol>
<p>Dissolve the chromatin pellet in 200–300 μL of TEE. Take aliquot of 20 μL, and at this point flash freeze the main chromatin samples in liquid nitrogen and store at −80°C for use in TMT labeling steps described below.</p>
<ol start="30">
<li>Heat the 20 μL aliquot to 95°C for 15 min to reverse crosslinking for Western blot analysis and protein concentration determination.</li>
<li>TCA precipitate (<a href="#bib7">Koontz, 2014</a>) the heated aliquot and resuspend in 20 μL (20 mM Tris-Cl pH 7.5 + 4 M urea). Then measure whole protein content using the Qubit® Protein assay kit using BSA as standards (expected concentrations range from 2.5–5 μg/ μL).</li>
</ol>
<h3 id="sec3.2">Part 2: Isobaric peptide labeling using the PreOmics iST-NHS kit and tandem mass tag (TMT) reagents and analysis</h3>
<h4 id="sec3.2.1">Starting material</h4>
<p>This labeling protocol assumes that the desired amount of peptides after pooling all TMT-labeled samples (see below) is roughly 500–750 μg, or 50–75 μg per TMT channel, sufficient for single shot LC-MS/MS analysis and offline basic pH reversed-phase fractionation. The PreOmics iST-NHS kit is compatible with N-hydroxysuccinimide (NHS) ester-based amine-reactive isobaric reagents due to the lack of primary amine-containing components and by using 2-chloro-N-N′-diethylacetamide for cysteine alkylation. iST kit protocols and components can be found on the PreOmics website <a href="https://www.preomics.com/resources">https://www.preomics.com/resources</a>. The original publication regarding iST kits is (<a href="#bib8">Kulak et al., 2014</a>).</p>
<h4 id="sec3.2.2">Lysis and benzonase treatment</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 h</div>
<ol start="32">
<li>Take an aliquot of 100–150 μg chromatin sample, sediment by centrifugation and resuspend and dissolve in 100 μL of PreOmics LYSE NHS component</li>
<li>Heat the chromatin samples at 95°C (to reverse crosslinking) for 15 min.</li>
<li>Add 2 μL Benzonase (stock: 250 U/μL) and digest while shaking slowly for 20 min at 37°C.</li>
<li>Sonicate the chromatin samples in Bioruptor® at 4°C for 10 min, 30 s ON, 30 s OFF, with maximal power settings.</li>
<li>Spin 300 × <i>g</i> for 10 s at 22°C–25°C to recover vaporized droplets.</li>
<li>Place half (50 μL) of the sample into a 1.5 mL Eppendorf tube. The sample should be around 50–75 μg of protein. Keep the rest of the sample in −80°C as reserve.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> if desired, measure the protein content again using the Qubit Protein assay.</div>
<h4 id="sec3.2.3">Digestion and labeling (using PreOmics iST-NHS kit and buffers therein)</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 6–12 h</div>
<p>Any sample-to-sample variability in protein concentrations, buffer composition, digestion, labeling, quenching, mixing, desalting or storage of each sample can result in artifacts of the proteomic analysis. We assume equal amounts of protein input, of similar composition, all in the same sample buffer, for digestion, to ensure comparable peptide amounts and reagent-to-analyte ratios. By using commercial reagent kits and performing steps of the protocol for each sample in parallel, using original reagent and solvent vials, many issues can be avoided. It is advised to build thorough quality checks into the protocol. Digestion and labeling steps are performed for all samples in separate plastic tubes using PreOmics iST-NHS kit components. Tandem-Mass-Tag (TMT) reagents are sensitive to hydrolysis. To avoid reduced labeling efficiency, store TMT reagents in a closed zip-bag with desiccant, equilibrate bag to 22°C–25°C before opening, and use TMT solutions immediately for labeling. We use a TMT reagent to peptide ratio of 4:1, as recommended by PreOmics, with a ratio of acetonitrile:aqueous buffer volume of 41:100 (&gt; 40% anhydrous organic solvent). For example, one vial of 0.2 mg TMT10plex reagent is sufficient to label 50 ug of peptides.</p>
<p>As a recommended quality check for digestion, an aliquot of each sample should be quenched, desalted and analyzed using the Pierce peptide quantification assay (Thermo Fisher), and through a single shot LC-MS/MS analysis to assess sample complexity and total amounts, as evident from the MS chromatograms and base peak signal intensities. Digestion and labeling efficiency can be assessed in a database search by allowing up to 3 enzyme missed cleavages and variable TMT modifications. We usually observe around 90% of peptide-to-spectrum matches in a whole cell extract proteomic experiment with no missed cleavages. In budding yeast chromatome samples, which are dominated by histone peptide identifications which have a significantly higher arginine and lysine content than whole cell extract samples, we commonly observe more missed cleavages; this number was frequently found to be between 60% and 80%. When searching for fixed TMT lysine modifications (all lysines are assumed to be TMT labeled), but variable TMT modification on peptide N-termini (either TMT labeled or unlabeled), the fraction of unmodified peptide N-termini of all confidently identified peptide-to spectrum matches in the experiment in percentage provides an estimate of the TMT labeling efficiency. We generally aim for TMT labeling efficiencies &gt;99%. It is also common to mix equal amounts of aliquots of all TMT labeled samples intended to be used for a multiplex experiment prior to a quality control analysis by LC-MS, and check the same QC parameters as above, including the quantification of TMT reporter ion intensities. By generating boxplots for all TMT abundance channels and/or summing all non-normalized TMT abundances per channel, issues of sample loading or mixing errors for individual samples can be spotted easily as outliers and corrected by adjusting sample amounts in the pooling step. After pooling of TMT-labeled samples we use PreOmics cartridges for peptide purification. Carefully document the desired experimental design by labeling tubes with sample names and TMT channel names (126, 127N, 127C, etc.). One should keep track of TMT reagent lot numbers.</p>
<ol start="38">
<li>Add 210 μL of PreOmics RESUSPEND to PreOmics DIGEST and shake (500 rpm, 10 min, 22°C–25°C).</li>
<li>Add 50 μL of this mixture to each protein sample tube and place it in a pre-heated heating block (37°C, 500 rpm; 8 h).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> keep the lysis buffer and digestion buffer volume ratio to 1:1.</div>
<div class="note">
<span class="note-title">Note:</span> we performed 8 h digestion for convenience (end of the day), but according to the PreOmics protocol 3 h digestion is sufficient.</div>
<ol start="40">
<li>Add 41 μL of anhydrous acetonitrile to each TMT label until dissolved.</li>
<li>To each of the digested protein samples, add one of the dissolved TMT reagents of your choice and, incubate while shaking (22°C–25°C ; 1 h).</li>
<li>Add 2 μL of 50% hydroxylamine to each sample tube and mix to quench the labeling reaction.</li>
<li>Add 100 μL of PreOmics STOP to each sample and shake (500 rpm, 1 min, 22°C–25°C).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> in this sample acidification step, a precipitate may be observed, turning the sample slightly cloudy.</div>
<ol start="44">
<li>Pool all the labeled samples for TMT-multiplexing into a single tube, vortex and centrifuge briefly to remove any precipitate.</li>
</ol>
<h4 id="sec3.2.4">Peptide sample desalting</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–3 h</div>
<p>Place PreOmics CARTRIDGE into ADAPTER and WASTE tube, use multiple cartridges in parallel to purify the entire mixture.</p>
<ol start="45">
<li>Transfer only the supernatant of the pooled TMT sample to a PreOmics cartridge, if there is a precipitate. The maximum peptide binding capacity per cartridge is 100 μg; adjust the loading volume per cartridge accordingly.</li>
<li>Spin the cartridge in a centrifuge 2800 × <i>g</i> for 1–3 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> adjust the centrifugation speed and time (maximum speed 6000 × <i>g</i>, 5 min) to ensure complete flow-through.</div>
<ol start="47">
<li>Add 200 μL of PreOmics WASH1 to cartridge and repeat step 46, discard flow-through.</li>
<li>Add 200 μL of PreOmics WASH2 to cartridge and repeat step 46, discard flow-through.</li>
<li>Place the cartridge in a fresh collection tube using PreOmics ADAPTORs.</li>
<li>Add 100μL of PreOmics ELUTE to each cartridge and repeat step 46.</li>
<li>Remove cartridge and place collection tube in a speed-vacuum concentrator (45°C, until completely dry, about 45 min).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Store the dry peptides in −80°C for the next steps.</div>
<h4 id="sec3.2.5">High pH fractionation and mass spectrometry analysis</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 days</div>
<ol start="52">
<li>Fractionate aliquots of 50–100 μg TMT-labeled peptides dissolved in 40 μL of 10 mM ammonium formate (pH 10, Buffer A) by high-pH reversed phase chromatography using an Agilent 1100 HPLC system (Agilent Technologies) equipped with an auto sampler, a YMC Triart C18 0.5 × 250 mm column (YMC Europe GmbH) and a fraction collector. Separate the peptides using the following chromatography method: binary buffer system: high pH Buffer A: 20mM ammonium formate in water, pH10, high pH Buffer B: 20mM ammonium formate pH 10 in 90% acetonitrile, Flow rate: 12 μL/min, Gradient: 115 min in total (mobile phase compositions in %B): 0–5 min 2%–15%, 5–15 min 15%–25%, 15–80 min 25%–45%, 80–85 min 45%–65%, 85–95 min 65%–100%, 95–100 min 100%, 100–101 min 100-2%, 101–115 min 2%.</li>
<li>Collect fractions from 3–112 min, then combine (concatenate) fractions (<a href="#bib15">Yang et al., 2012</a>) to minimize the total number of samples for LC-MS analysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> choose the kind of fractionation, concatenation and MS run time that achieves an acceptable ratio of instrument time vs. proteome coverage with your instrument configuration. We routinely collect 13.8 μL/fraction (96 fractions) into a microtiter plate and pairwise combine fractions from the earlier and the latter half of the gradient (i.e., fr1+fr49, fr2+fr50, etc.) to achieve a total of 48 fractions for each experiment, which we then analyzed with a 60 min MS method. If this concatenation seems excessive for an analysis in budding yeast, alternative schemes can be employed.</div>
<ol start="54">
<li>For each LC–MS run, approximately 1 μg of peptides were loaded onto a PepMap 100 C18 2 cm trap (Thermo Fisher) using an EASY nLC-1000 system (Thermo Fisher).</li>
<li>On-line peptide separation was performed on a 15 cm EASY-Spray C18 column (ES801, Thermo Fisher) by applying the following chromatography method: buffer A: 0.1% formic acid, buffer B: 0.1% formic acid in acetonitrile, flow rate 250 nl/min, gradient 60 min in total, (mobile phase compositions in % B): 0-1:20 min 3%–6%, 1:20-49:30 min 6%–22%, 49:30–53 min 22%–40%, 53–54 min 40%–80%, 54–60 min 80–80% at a flow rate of 300 nL/min.</li>
<li>An Orbitrap Fusion or Orbitrap Fusion Lumos Tribrid mass spectrometer equipped with an EASY spray source (both Thermo Fisher Scientific) was operated in a data-dependent mode and TMT reporter ions were quantified from MS2 fragment ion spectra. Every 3 s, the most intense precursor ions from the Orbitrap survey scan (MS1) were selected for higher-energy collisional dissociation (HCD) fragmentation with 37% normalized collision energy. The Orbitrap analyzer was used to record MS2 spectra at either 15k (TMT6plex) or 50k (TMT9plex) resolution. To improve quantification accuracy and to avoid peptide fragment co-fragmentation, use of the instrument’s TMT Synchronous Precursor Selection (SPS) MS3 feature and real-time database search capabilities may be a good alternative.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We chose to use MS2 spectra detected in the Orbitrap analyzer for TMT peptide quantification and identification to achieve deep-proteome coverage and to allow potential localization of low abundant post-translational modifications where high mass accuracy and resolution can be beneficial.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>The protocol described here enables users to isolate, identify and quantify chromatin-bound proteins from budding yeast in a robust and simple manner. The chromatin-bound fraction of the total proteome is called the chromatome. The expected yield of this preparation from 1 liter culture of exponentially growing budding yeast (OD600 = 5 × 10<sup>6</sup> cells/mL) is in range of 2–2.5 mg of protein. Expected outcomes for individual steps are summarized in <a href="#tbl1">Table 1</a>. The correlation among biological replicates for the relative abundance and presence of chromatin-bound proteins purified and assessed as described above, is extremely high (<a href="#fig2">Figure 2</a>D). The success of the protocol should be tested by monitoring the enrichment of histones and DNA binding proteins in the chromatin fraction as shown in (<a href="#fig2">Figure 2</a>E). In our hands, the chromatome is highly enriched for histones and almost all DNA binding factors but contains non-nuclear contaminants. In our case we identified reproducibly about 4500 proteins (two-thirds of the total proteome), although the relative abundance of individual proteins in the proteome, as compared to the chromatome, were extremely different (<a href="#bib1">Challa et al., 2021</a>). Whereas 700 proteins showed ≥ 2-fold increased or decreased abundance in the chromatome of Zeocin-treated wild-type (GA-6879) cells, only 5 proteins showed a similar change in the proteome sample from the same strain. We attribute this to reduced sensitivity for chromatin proteins to the fact that they are of low abundance in comparison to ribosomes, actin, and other abundant cytoplasmic proteins.</p>
<table id="tbl1">
<caption>Table 1. Expected outcomes at various steps of protocol</caption>
<thead>
<tr>
<th>Steps</th>
<th>Reasons for sample collection</th>
<th>Storage conditions</th>
<th>Methods for the analysis</th>
<th>Expected outcome</th>
</tr>
</thead>
<tbody>
<tr>
<td>Step 3</td>
<td>Alpha factor arrest</td>
<td>4°C</td>
<td>FACS analysis</td>
<td>Cell cycle arrest at G1 phase (<a href="#fig2">Figure 2</a>B)</td>
</tr>
<tr>
<td>Step 4</td>
<td>DNA damage or Zeocin treatment</td>
<td>−20°C</td>
<td>Western blotting</td>
<td>Increase in DNA damage markers like γH2A (<a href="#bib1">Challa et al., 2021</a>)</td>
</tr>
<tr>
<td>Step 11</td>
<td>Total protein analysis for mass spectrometry</td>
<td>−80°C</td>
<td>SDS-PAGE and Mass spectrometry</td>
<td>Compare total protein vs chromatin bound proteins for chromatin enrichment (<a href="#fig2">Figure 2</a>E)</td>
</tr>
<tr>
<td>Step 14</td>
<td>Supernatant analysis</td>
<td>−20°C</td>
<td>Western blotting</td>
<td>Chromatin bound proteins such as Histones should not be enriched in sup fraction (<a href="#bib1">Challa et al., 2021</a>)</td>
</tr>
<tr>
<td>Step 16</td>
<td>DNA crosslinking efficiency</td>
<td>−20°C</td>
<td>SDS-PAGE</td>
<td>Higher molecular weight proteins should be detected on SDS-PAGE for efficient crosslinking (<a href="#fig2">Figure 2</a>C)</td>
</tr>
<tr>
<td>Step 19</td>
<td>Protein loss after crosslinking</td>
<td>−20°C</td>
<td>SDS-PAGE</td>
<td>Very low protein should be detected in wash fraction</td>
</tr>
<tr>
<td>Step 21</td>
<td>RNA digestion</td>
<td>−20°C</td>
<td>Qubit® RNA detection kit</td>
<td>Very low RNA levels should be detected after RNA digestion</td>
</tr>
<tr>
<td>Step 25</td>
<td>Protein loss during urea washes</td>
<td>−20°C</td>
<td>SDS-PAGE</td>
<td>Very low protein should be detected in urea wash samples (<a href="#fig2">Figure 2</a>C)</td>
</tr>
<tr>
<td>Step 30</td>
<td>Reverse crosslinking efficiency</td>
<td>−20°C</td>
<td>SDS-PAGE</td>
<td>Should detect multiple bands from low to high molecular weight to confirm reverse crosslinking</td>
</tr>
</tbody>
</table>
<p>The number of proteins identified in the chromatin samples depends on the efficiency of the TMT labeling and resolution of proteomic analysis. In budding yeast several studies have used the TMT10plex method to quantify the phosphoproteome (<a href="#bib13">Touati et al., 2019</a>; <a href="#bib16">Zhang et al., 2019</a>) with a protocol described in (<a href="#bib6">Jones et al., 2020</a>). Recently, TMTpro-18plex series was developed and this method allows one to design more complex experiments (<a href="#bib9">Li et al., 2021</a>). In our supporting publication, we used the TMT 6plex and TMT10plex (in our experimental setup we used 9plex labels from 10plex kit) method to analyze the chromatome differences before and after DNA damage in three different strain backgrounds. It is important to characterize the conditions you want to compare in the same experiment. We successfully characterized differences in the occupancy of histones, DNA repair proteins and ubiquitin ligases on chromatin following DNA damage, and specific predictions were borne out by protein tagging and Chromatin Immunoprecipitation.</p>
<p>Due to its high reproducibility and efficient enrichment for chromatin-bound proteins, this method of fractionation for quantitative mass spectrometry can be applied efficiently to other yeast mutants and other stress conditions (e.g., exposure to hydroxyurea or other damage agents). The proteome and chromatome of HU-arrested S-phase cells is reported in <a href="#bib5">Hurst et al., 2021</a>.</p>
<div class="note">
<span class="note-title">Note:</span> detailed experimental evidence for the above steps is found in (<a href="#bib1">Challa et al., 2021</a>)</div>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Perform a data base search, i.e., Sequest HT against the Uniprot <i>S. cerevisiae</i> FASTA protein sequence database in Proteome Discoverer PD2.2.2.0.388 (Thermo Fisher Scientific), and quantify reporter ion intensities from FTMS HCD MS2 spectra. MSraw data were loaded as fractions, re-calibrated and searched against the Uniprot Yeast database (downloaded from Uniport on April4 2017 with the query: organism ’’Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker’s yeast) [559292]’’ AND proteo-me:up000002311) and commonly observed contaminants. For protein quantification, make sure to include static modifications of TMT (+229 Da) at lysine and peptide N-termini, PreOmics NHS-iST Cys alkyation (+113 Da). It is important to account for various protein N-terminal processing events, as protein N-termini may be dynamically modified with TMT (229 Da) or acetyl (42 Da), with or without initiator methionines removed (+98 Da for Met-loss+TMT or -89 Da for Met-loss+Acetyl, respectively). In addition, the variable modifications can be included in the analysis, such as phosphorylation of serines or threonines (Phospho, +79.966 Da). The false discovery rate for peptide and protein identification was set to 0.01 based on the target-decoy (concatenated) search strategy (<a href="#bib3">Elias and Gygi, 2010</a>). Protein abundances were calculated based on the summed signal-to-noise ratio of all unique and razor peptide reporter ion signals above 10. Peptide abundances with more than 50% isolation interference (Co-isolation threshold) were not considered. Protein abundances were normalized based on the total peptide amount, resulting in equal sums of abundances for all samples. Filtered tables were exported for further analysis in RStudio (version 1.2.5033) and R (version 4.0.0). To determine differential abundances, abundance values were log2 transformed after addition of pseudocounts, and then subjected to differential analysis using linear models as implemented in the R package limma (3.44.3). The eBayes function in limma was applied to compute the t-statistics for differential abundance (adj.P.Val) of specific sample groups.</p>
<p>Look for potential changes in histone peptide abundance by searching unmapped or ‘leftover’ MS2 spectra, and applying the same search settings against only the histones in the UniProt Yeast database, with fixed modifications of TMT (+229 Da) at peptide N-termini, but allowing dynamic modifications at lysines with either TMT (+229 Da or monomethyl (Methyl_TMT, +241 Da), dimethyl (+28 Da), trimethyl (+42 Da), or the ubiquityl remnant (GG_TMT, +343 Da). Peptide-to-spectrum-matches need to be stringently filtered for peptides containing unmodified lysine residues lacking TMT-labels, or C-terminal modifications incompatible with trypsin digestion. Furthermore, apply conservative score thresholds, i.e., for a minimal Sequest "XCorr" value.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This method was developed to determine global changes in a population of chromatin-bound proteins. Naturally, changes that occur at a single gene locus are unlikely to be detected by this method. A further limitation of the method may be that formaldehyde cross-linking is unsuitable for certain chromatin factors. However, we recovered ∼3200 proteins (out of a total of 6000 in yeast) in our analyses, which is a high fraction compared to other methods used for the isolation of chromatin proteomics in yeast. Using this procedure we do not achieve a complete separation of chromatin-bound and non-bound proteins (see <a href="#fig2">Figure 2</a>E), even though, as far as we are aware, no other chromatin fractionation method achieves better separation with this sensitivity and reproducibility. Given the overlap, however, finding a protein in the chromatome fraction is not sufficient to conclude that is associated with chromatin <i>in vivo</i>. ChIP (Chromatin Immuno-precipitation) / iPOND (isolation of proteins on nascent DNA) (<a href="#bib2">Dungrawala and Cortez, 2015</a>; <a href="#bib11">Nelson et al., 2006</a>) or immunofluorescence / GFP fusion tracking may be required to validate the chromatin localization of components recovered in the chromatome (<a href="#bib1">Challa et al., 2021</a>).</p>
<p>In proteomics, PTM peptides are preferentially analyzed following specific PTM enrichment protocols to increase the chances of confident PTM identification and localization. For example, phosphopeptides can be enriched for by metal oxide affinity chromatography, and antibody-based methods can be used to enrich for proteins bearing the ubiquitin remnant motif (K-ε-GG). Histone modifications, in particular located in isobaric modified histone peptides with multiple potential modification sites, are best quantified using heavy stable isotope-labeled reference peptides, chemical derivatization of lysines, and targeted MS methods. In our experiment we achieved a very high proteome coverage with an apparent enrichment for histones. However, PTM analysis from non-enriched samples should be performed with caution to avoid false positives.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>Inefficient cell synchronization or shmoo formation.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Cell synchronization is very important to avoid cell cycle-related differences among samples. Standardize alpha factor concentration in small volume cultures before the actual experiment for each <i>MAT</i><b><u>a</u></b> strain and check cell cycle arrest to achieve 90% absence of small buds (bud size &lt;50% the mother cell) by light microscope. Idealized arrest is shown in <a href="#fig2">Figure 2</a>A. Excess alpha factor can lead to shmoo formation which means the cells have commenced a mating program (<a href="#fig2">Figure 2</a>A). YPAD adjusted to pH=5.0 can enhance alpha factor synchronization, as can use of a <i>bar1</i> mutant. Upon appropriate arrest, FACS analysis should show exclusively 1N peak (<a href="#fig2">Figure 2</a>B).</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Zeocin® treatment does not work efficiently.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>There are variations in the efficacy among Zeocin® batches, and this may also be true for other DNA damaging reagents. Therefore, always use the same batch of Zeocin® for triplicate experiments and standardize initially with small volume cultures before treating with large cultures. Always check markers for DNA damage by Western blot to confirm that DNA damage induction is consistently above the control.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Incomplete cell lysis.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Use freshly prepared Zymolyase® or lyticase (<a href="#bib14">Verdier et al., 1990</a>) (<a href="#bib12">Scott and Schekman, 1980</a>) and adjust the concentration based on the cell number (see <a href="#fig2">Figure 2</a>A). Ensure that the cells are well resuspended before adding the glycolytic reagents, and check cell lysis under a light microscope. Adjust time of incubation based on percentage of lysis.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Variable cross-linking efficiency.</p>
<h3 id="sec7.8">Potential solution</h3>
<p>Check the cross-linking efficiency of step 16 samples by SDS-PAGE followed by Coomassie blue staining. To avoid ineffective cross-linking use a freshly prepared formaldehyde solution. Avoid using chilled E Buffer to dilute the formaldehyde stock.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Loss of sample during urea washes.</p>
<h3 id="sec7.10">Potential solution</h3>
<p>Do quick urea washes in steps 23–27. If necessary, collect the urea washes and check for protein loss by SDS-PAGE. If there is excessive protein loss during the urea washes adjust the time and SDS concentration accordingly. Always prepare fresh urea buffer and to avoid carbamylation of proteins at high temperatures, see step 27.</p>
<h3 id="sec7.11">Problem 6</h3>
<p>Low chromatin protein yield.</p>
<h3 id="sec7.12">Potential solution</h3>
<p>This is due to multiple reasons in sample preparations.</p>
<p>Avoid starting the experiment with frozen cell pellets. Cells can become fragile and may lyse upon thawing. It is recommended to proceed with chromatin fractionation on day that cells are collected.</p>
<p>Take sufficient starting material to get a good amount of protein. Follow the step 1 for amount of starting material used in the experiment.</p>
<p>Maintain a high chromatin cross-link efficiency, as mentioned in steps 15–18.</p>
<h3 id="sec7.13">Problem 7</h3>
<p>Incomplete digestion and/or incomplete cysteine reduction/alkylation</p>
<h3 id="sec7.14">Potential solution</h3>
<p>pH should be above 7, ideally 8.0–8.5. Wash TCA precipitate thoroughly and check pH after addition of PreOmics LYSE NHS to the sample before proceeding, adjust pH with non-primary amine reagents or buffers (i.e., TEAB, HEPES).</p>
<h3 id="sec7.15">Problem 8</h3>
<p>Poor TMT labeling</p>
<h3 id="sec7.16">Potential solution</h3>
<p>Always use freshly opened labeling reagents for the TMT labeling, store reagents in closed zip-bag with desiccant, use freshly opened anhydrous solvents. If poor labeling is observed in a single sample, one specific label may have hydrolyzed. Then, repeat the labeling reaction ensuring that organic-aqueous buffer ratio and the pH are correct. If it is observed in all samples, double check the steps prior to digestion to avoid amine-containing buffers in the sample (e.g., Tris) and use non-amine-containing buffers like HEPES. Always keep the frozen backup sample to repeat the labeling experiment and use fresh batch of reagents in case of low TMT labeling.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Challa, K., Schmid, C.D., Kitagawa, S., Cheblal, A., Iesmantavicius, V., Seeber, A., Amitai, A., Seebacher, J., Hauer, M.H., Shimada, K., et al. (2021). Damage-induced chromatome dynamics link Ubiquitin ligase and proteasome recruitment to histone loss and efficient DNA repair. Mol. Cell <i>81</i>, 811-829.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref1">View at publisher</a></p>
<p id="bib2">Dungrawala, H. and Cortez, D. (2015). Purification of proteins on newly synthesized DNA using iPOND. Methods Mol. Biol. <i>1228</i>, 123-131. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref2">View at publisher</a></p>
<p id="bib3">Elias, J.E. and Gygi, S.P. (2010). Target-decoy search strategy for mass spectrometry-based proteomics. Methods Mol. Biol. <i>604</i>, 55-71. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref3">View at publisher</a></p>
<p id="bib4">Hauer, M.H., Seeber, A., Singh, V., Thierry, R., Sack, R., Amitai, A., Kryzhanovska, M., Eglinger, J., Holcman, D., Owen-Hughes, T., et al. (2017). Histone degradation in response to DNA damage enhances chromatin dynamics and recombination rates. Nat. Struct. Mol. Biol. <i>24</i>, 99-107. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref4">View at publisher</a></p>
<p id="bib5">Hurst, V., Challa, K., Jonas, F., Forey, R., Sack, R., Seebacher, J., Schmid, C.D., Barkai, N., Shimada, K., Gasser, S.M., and Poli, J. (2021). A regulatory phosphosite on Mec1 controls RNAPII and RNAPIII occupancy during replication stress. EMBO J. <i>40</i>. <a class="external-link" href="https://doi.org/10.15252/embj.2021108439">https://doi.org/10.15252/embj.2021108439</a></p>
<p id="bib6">Jones, A.W., Flynn, H.R., Uhlmann, F., Snijders, A.P., and Touati, S.A. (2020). Assessing budding yeast phosphoproteome dynamics in a time-resolved manner using TMT10plex mass tag labeling. STAR Protoc. <i>1</i>, 100022. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref6">View at publisher</a></p>
<p id="bib7">Koontz, L. (2014). TCA precipitation. Methods Enzymol. <i>541</i>, 3-10. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref7">View at publisher</a></p>
<p id="bib8">Kulak, N.A., Pichler, G., Paron, I., Nagaraj, N., and Mann, M. (2014). Minimal, encapsulated proteomic-sample processing applied to copy-number estimation in eukaryotic cells. Nat. Methods <i>11</i>, 319-324. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref8">View at publisher</a></p>
<p id="bib9">Li, J., Cai, Z., Bomgarden, R.D., Pike, I., Kuhn, K., Rogers, J.C., Roberts, T.M., Gygi, S.P., and Paulo, J.A. (2021). TMTpro-18plex: the expanded and complete set of TMTpro reagents for sample multiplexing. J. Proteome Res. <i>20</i>, 2964-2972. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref9">View at publisher</a></p>
<p id="bib10">Meister, P., Gehlen, L.R., Varela, E., Kalck, V., and Gasser, S.M. (2010). Visualizing yeast chromosomes and nuclear architecture. Methods Enzymol. <i>470</i>, 535-567. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref10">View at publisher</a></p>
<p id="bib11">Nelson, J.D., Denisenko, O., and Bomsztyk, K. (2006). Protocol for the fast chromatin immunoprecipitation (ChIP) method. Nat. Protoc. <i>1</i>, 179-185. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref11">View at publisher</a></p>
<p id="bib12">Scott, J.H. and Schekman, R. (1980). Lyticase: endoglucanase and protease activities that act together in yeast cell lysis. J. Bacteriol. <i>142</i>, 414-423. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref12">View at publisher</a></p>
<p id="bib13">Touati, S.A., Hofbauer, L., Jones, A.W., Snijders, A.P., Kelly, G., and Uhlmann, F. (2019). Cdc14 and PP2A phosphatases cooperate to shape phosphoproteome dynamics during mitotic exit. Cell Rep. <i>29</i>, 2105-2119.e4. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref13">View at publisher</a></p>
<p id="bib14">Verdier, J.M., Stalder, R., Roberge, M., Amati, B., Sentenac, A., and Gasser, S.M. (1990). Preparation and characterization of yeast nuclear extracts for efficient RNA polymerase B (II)- dependent transcription in vitro. Nucleic Acids Res. <i>18</i>, 7033-7039. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref14">View at publisher</a></p>
<p id="bib15">Yang, F., Shen, Y., Camp, D.G., and Smith, R.D. (2012). High pH reversed-phase chromatography with fraction concatenation as an alternative to strong-cation exchange chromatography for two-dimensional proteomic analysis. Expert Rev. Proteomics <i>9</i>, 129-134. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref15">View at publisher</a></p>
<p id="bib16">Zhang, L., Winkler, S., Schlottmann, F.P., Kohlbacher, O., Elias, J.E., Skotheim, J.M., and Ewald, J.C. (2019). Multiple layers of phospho-regulation coordinate metabolism and the cell cycle in budding yeast. Front. Cell Dev. Biol. <i>7</i>, 338. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00531-1/sref16">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and request for resources and reagents should be directed to and will be fulfilled by the Lead Contact: Prof. Susan M. Gasser (<a href="mailto:susan.gasser@fmi.ch">susan.gasser@fmi.ch</a>)</p>
<h4>Materials availability</h4>
<p>All noncommercially available materials used in this study are available from the lead contact (SMG) upon request. For ordering details contact the technical contact (KC).</p>
<h4>Data and code availability</h4>
<p>Proteomics data were deposited in the PRIDE database with the accession code Pride PXD022369. All other data and code supporting the findings of this study are available from the lead contact without any further restrictions.</p>
<h3>Acknowledgments</h3>
<p>We thank Christoph Schmid (FMI) for analysis of proteomics data and Vytautas Iesmantavicius for help in FMI proteomics facility. We are grateful for initial R scripts provided by Paul Ginno and Sebastien Kaluscha of the Schübeler Laboratory. We thank all member of the Gasser Laboratory for the valuable discussions and support. This work was supported by the <a href="https://doi.org/10.13039/501100001711">Swiss National Science Foundation</a> grant 31003A_176286 and the Novartis Research Foundation.</p>
<h3>Author contributions</h3>
<p>K.C., J.S., and S.M.G. planned this project and wrote the manuscript. K.C. and J.S. conducted experiments, analyzed, and interpreted data.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>