<article>
<section>
<h1 id="header">Isolation and functional characterization of primary endothelial cells from mouse cerebral cortex</h1>
<p><time datetime="2021-12-17">Published: December 17, 2021</time></p>
<p>Julie Ouellette<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#cor1">*</a></sup> and Baptiste Lacoste<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn2">5</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Ottawa Hospital Research Institute, Neuroscience Program, Ottawa, ON K1H8M5, Canada</p>
<p id="aff2"><sup>2</sup>University of Ottawa, Faculty of Medicine, Department of Cellular and Molecular Medicine, Ottawa, ON K1H8M5, Canada</p>
<p id="aff3"><sup>3</sup>University of Ottawa Brain and Mind Research Institute, Ottawa, ON K1H8M5, Canada</p>
<p id="fn1"><sup>4</sup>Technical contact</p>
<p id="fn2"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:jouel100@uottawa.ca">jouel100@uottawa.ca</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:blacoste@uottawa.ca">blacoste@uottawa.ca</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.101019">10.1016/j.xpro.2021.101019</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Endothelial cells (ECs) lining blood vessels are implicated in organ development, function, and maintenance. We present a detailed protocol enabling isolation and characterization of primary mouse brain ECs, including quality controls and functional assays. These procedures promote survival of primary brain ECs for the assessment of endothelial health. Since alterations in brain ECs are involved in the onset and progression of neurological disorders, this protocol represents a valuable tool to better understand the roles of ECs in brain health.</p>
<p>For complete details on the use and execution of this profile, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref4">Ouellette et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Streamlined approach to determine the health of cultured mouse brain endothelial cells</li>
<li>Protocol focused on primary endothelial cell from mouse cerebral cortex</li>
<li>Complete procedures from molecular analysis to cell behavior including quality controls</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1236-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The following protocol describes the specific steps to isolate, purify, maintain and characterize primary mouse brain (cortical) endothelial cells (ECs). These procedures have been optimized for isolation of ECs from young mice at postnatal day (P) 14 of either sex. While the following protocols may be used for isolation of adult primary mouse brain ECs, adult ECs will grow significantly slower (please read the <a href="#limitations">limitations</a> section). Refer to the <a href="#key-resources-table">key resources table</a> for a complete list of reagents and tools. All studies were approved by the University of Ottawa Animal Care Committee and conducted in accordance to guidelines of the Canadian Council on Animal Care.</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Rat anti-CD31 (use at 1:200)</td>
<td>BD Pharmingen</td>
<td>Cat#553370</td>
</tr>
<tr>
<td>Rabbit anti-VE-Cadherin (use at 1:500)</td>
<td>Abcam</td>
<td>Cat#ab33168</td>
</tr>
<tr>
<td>Goat anti-Iba1 (use at 1:1000)</td>
<td>Abcam</td>
<td>Cat#ab107159</td>
</tr>
<tr>
<td>Goat anti-PDGRβ (use at 1:200)</td>
<td>R&amp;D systems</td>
<td>Cat#AF1042</td>
</tr>
<tr>
<td>Guinea pig anti-NeuN (use at 1:1500)</td>
<td>Millipore Sigma</td>
<td>Cat#ABN90</td>
</tr>
<tr>
<td>Guinea pig anti-GFAP (use at 1:1000)</td>
<td>Synaptic Systems</td>
<td>Cat#173004</td>
</tr>
<tr>
<td>Mouse anti-αSMA (use at 1:200)</td>
<td>Sigma-Aldrich</td>
<td>Cat#A2547</td>
</tr>
<tr>
<td>Donkey anti-Rat IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 488 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A-21208</td>
</tr>
<tr>
<td>Donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 568 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A-10042</td>
</tr>
<tr>
<td>Donkey anti-Rabbit IgG (H+L) Highly Cross-Adsorbed Secondary Antibody, Alexa Fluor 647 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A-31573</td>
</tr>
<tr>
<td>Donkey anti-goat IgG (H+L) cross-absorbed secondary antibody, Alexa Fluor 568 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A-11057</td>
</tr>
<tr>
<td>Goat anti-guinea pig IgG (H+L) Highly cross-absorbed secondary antibody, Alexa Fluor 488 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A-11073</td>
</tr>
<tr>
<td>Goat anti-guinea pig IgG (H+L) Highly cross-absorbed secondary antibody, Alexa Fluor 647 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A-21450</td>
</tr>
<tr>
<td>Donkey anti-mouse IgG (H+L) cross-absorbed secondary antibody, Alexa Fluor 568 (use at 1:300)</td>
<td>Invitrogen</td>
<td>Cat#A10037</td>
</tr>
<tr>
<td>Alexa Fluor™ 488 Phalloidin (use at 1:40)</td>
<td>Invitrogen</td>
<td>Cat#A12379</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>EGM<sup>TM</sup>-2 MV Microvascular Endothelial Cell Growth Medium-2 BulletKit<sup>TM</sup>
</td>
<td>Lonza</td>
<td>Cat#CC-3202</td>
</tr>
<tr>
<td>Penicillin-Streptomycin (10,000 U/mL)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#15140122</td>
</tr>
<tr>
<td>HBSS without calcium/magnesium</td>
<td>Gibco</td>
<td>Cat#14170-120</td>
</tr>
<tr>
<td>HBSS with calcium/magnesium</td>
<td>Gibco</td>
<td>Cat#14025-092</td>
</tr>
<tr>
<td>Attachment factor 1×</td>
<td>Gibco</td>
<td>Cat#S-006-100</td>
</tr>
<tr>
<td>Albumin, Bovine (BSA)</td>
<td>VWR</td>
<td>Cat#0332-100G</td>
</tr>
<tr>
<td>CD31 MicroBeads, Mouse</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-097-418</td>
</tr>
<tr>
<td>Myelin Removal Beads II, human, mouse, rat</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-096-733</td>
</tr>
<tr>
<td>CD45 MicroBeads, Mouse</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-052-301</td>
</tr>
<tr>
<td>Sterile 1× PBS</td>
<td>Wisent BioProducts</td>
<td>Cat#311-011-CL</td>
</tr>
<tr>
<td>TrypLE™ Express Enzyme (1×), no phenol red</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12604013</td>
</tr>
<tr>
<td>Alconox detergent</td>
<td>VWR</td>
<td>Cat#21835-032</td>
</tr>
<tr>
<td>Virkon® S Disinfectant and Virucide</td>
<td>Fisher Scientific</td>
<td>Cat#NC9549979</td>
</tr>
<tr>
<td>CellEvent™ Caspase-3/7 Green Detection</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#C10423</td>
</tr>
<tr>
<td>Staurosporine</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# BP2541100</td>
</tr>
<tr>
<td>Hoechst 33342, Trihydrochloride, Trihydrate-10 mg/mL Solution in Water</td>
<td>Invitrogen</td>
<td>Cat#H3570</td>
</tr>
<tr>
<td>Paraformaldehyde 20% Solution, EM grade</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#15713-S</td>
</tr>
<tr>
<td>Fluoromount-G™</td>
<td>Electron Microscopy Sciences</td>
<td>Cat#17984-25</td>
</tr>
<tr>
<td>Triton-X-100</td>
<td>VWR</td>
<td>Cat#0694-1L</td>
</tr>
<tr>
<td>Fish gelatin blocking buffer, 10%</td>
<td>VWR</td>
<td>Cat#M319-500mL</td>
</tr>
<tr>
<td>Donkey serum, sterile filtered</td>
<td>Wisent Bioproducts</td>
<td>Cat#035-150</td>
</tr>
<tr>
<td>Dimethyl sulfoxide,DMSO</td>
<td>sigma-Aldrich</td>
<td>Cat#D8418-100mL</td>
</tr>
<tr>
<td>DAPI</td>
<td>Invitrogen</td>
<td>Cat#D1306</td>
</tr>
<tr>
<td>Matrigel® Matrix Basement Membrane, Growth Factor Reduced</td>
<td>Fisher Scientific</td>
<td>Cat#354230</td>
</tr>
<tr>
<td>Sodium chloride (NaCl)</td>
<td>VWR</td>
<td>Cat#0241-5KG</td>
</tr>
<tr>
<td>Sodium phosphate dibasic anhydrous (Na<sub>2</sub>HPO<sub>4</sub>)</td>
<td>VWR</td>
<td>Cat#0404-1KG</td>
</tr>
<tr>
<td>Sodium phosphate monobasic (NaH<sub>2</sub>PO<sub>4</sub>H<sub>2</sub>O)</td>
<td>VWR</td>
<td>Cat#BDH9298.2.5</td>
</tr>
<tr>
<td>β-mercaptoethanol, Molecular biology grade</td>
<td>Merck Millipore</td>
<td>Cat#444203</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Neural Tissue Dissociation Kit (P)</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-092-628</td>
</tr>
<tr>
<td>Click-iT™EdU Alexa Fluor™ 594 Imaging Kit</td>
<td>Invitrogen</td>
<td>Cat#C10339</td>
</tr>
<tr>
<td>E.Z.N.A.® HP Total RNA Kit</td>
<td>OMEGA bio-tek</td>
<td>Cat#R6812-01</td>
</tr>
<tr>
<td>RNase-free DNase I Set</td>
<td>OMEGA bio-tek</td>
<td>Cat#E1091</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>Mouse: B6129SF1/J (14 days old, male or female)</td>
<td>The Jackson Laboratory</td>
<td>Cat#101043</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>IncuCyte® ZOOM Software</td>
<td>Essen Bioscience</td>
<td>N/A</td>
</tr>
<tr>
<td>Fiji Image J</td>
<td><a href="#bib5">Schindelin et al., 2012</a></td>
<td><a href="https://imagej.net/software/fiji/">https://imagej.net/software/fiji/</a></td>
</tr>
<tr>
<td>Angiogenesis Analyzer for Image J</td>
<td><a href="#bib1">Carpentier, 2012</a></td>
<td><a href="https://imagej.nih.gov/ij/macros/toolsets/Angiogenesis%20Analyzer.txt">https://imagej.nih.gov/ij/macros/toolsets/Angiogenesis%20Analyzer.txt</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>70 μM pore size strainer</td>
<td>VWR</td>
<td>Cat#76327-100</td>
</tr>
<tr>
<td>LS columns</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-401</td>
</tr>
<tr>
<td>LD columns</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-901</td>
</tr>
<tr>
<td>MACS® MultiStand</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-303</td>
</tr>
<tr>
<td>QuadroMACS™ separator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-090-976</td>
</tr>
<tr>
<td>IncuCyte® ImageLock Plates</td>
<td>Essen Bioscience</td>
<td>Cat#4379</td>
</tr>
<tr>
<td>IncuCyte® 96-well WoundMaker Tool</td>
<td>Essen Bioscience</td>
<td>Cat#4563</td>
</tr>
<tr>
<td>German glass coverslips, 12 mm round, #1 thickness</td>
<td>Mandel</td>
<td>Cat#NEU-GG-12-1.5-OZ</td>
</tr>
<tr>
<td>Corning® cryogenic vials, internal thread</td>
<td>Millipore sigma</td>
<td>Cat#CLS431386</td>
</tr>
<tr>
<td>Mr. Frosty™ Freezing Container</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#5100-0001</td>
</tr>
<tr>
<td>Superfrost Plus Microscope Slides</td>
<td>Fisher Scientific</td>
<td>Cat#12-550-15</td>
</tr>
<tr>
<td>96-Well Tissue Culture Plate, Non-treated, Sterilized, Non-Pyrogenic</td>
<td>VWR</td>
<td>Cat#10861-562</td>
</tr>
<tr>
<td>IncuCyte™ Zoom® Apparatus</td>
<td>Essen Bioscience</td>
<td>N/A</td>
</tr>
<tr>
<td>Nikon Eclipse TE2000-E inverted microscope</td>
<td>Nikon</td>
<td>N/A</td>
</tr>
<tr>
<td>Zeiss Axio Imager M2 microscope</td>
<td>Zeiss</td>
<td>N/A</td>
</tr>
<tr>
<td>Axiocam 506 mono</td>
<td>Zeiss</td>
<td>N/A</td>
</tr>
<tr>
<td>ApoTome.2 module</td>
<td>Zeiss</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Section 1: Primary mouse cerebral cortex endothelial cell isolation and culture</h3>
<h4 id="sec3.1.1">Preparation of reagents and medium for primary mouse cerebral cortex endothelial cell isolation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min (on the day prior to cell isolation)</div>
<ol>
<li>Reagent preparation
<ol type="a">
<li>Enzyme P [included in the “Neural Dissociation Kit (P)”] is ready to use. Prepare aliquots of 200 μL to avoid repeated freeze-thaw cycles. Store aliquots at −20°C. This kit contains buffers and enzymes for 50 isolations.</li>
<li>Resuspend lyophilized Enzyme A with 1 mL of Buffer A in provided vial labeled Enzyme A (included in the same kit). Do not vortex. The final Enzyme A solution is clear when fully resuspended. Prepare aliquots of 40 μL and store at −20°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To resuspend Enzyme A, slowly pipette up and down (∼5–10 times) using a 1 mL pipette until powder is no longer visible.</div>
<ol start="2">
<li>Endothelial cell medium preparation
<ol type="a">
<li>Thaw SingleQuots™ Kit (included in EGM™-2 MV Microvascular Endothelial Cell Growth Medium-2 BulletKit™). Add SingleQuots™ Kit to basal medium. Do not refreeze.</li>
<li>Add 1% penicillin/streptomycin to prepared BulletKit™ medium.</li>
</ol>
</li>
<li>Autoclave all dissection tools (curved forceps, straight forceps, large scissors, curved small scissors).</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Once thawed, SingleQuots™ Kit must be mixed with basal medium within 72 h. After BulletKit™ medium is prepared, use within 1 month.</div>
<h4 id="sec3.1.2">Mouse cerebral cortex dissection</h4>
<p>Two cortical hemispheres (i.e., one mouse brain) are ideal for plating endothelial cells in 2 wells of a 6-well plate to promote cell survival and growth following isolation.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 90 min (<b>the day of cell isolation)</b>
</div>
<ol start="4">
<li>Clean dissection bench using 70% or 100% ethanol.</li>
<li>Submerge dissection tools and razor blade in 100% ethanol for a minimum of 30 min prior to dissection.</li>
<li>Cortical dissection (<a href="#fig1">Figures 1</a>A–1H)
<ol type="a">
<li>Prepare a 12-well plate with 1 mL of cold HBSS (without calcium/magnesium).</li>
<li>Sacrifice the mouse by cervical dislocation.
<div class="note">
<span class="note-title">Note:</span> No anesthesia or carbon dioxide overdose is recommended before the cervical dislocation. This might disrupt endothelial cell health. A guillotine can be used for quick euthanasia.</div>
</li>
<li>Decapitate mouse and extract brain rapidly from the skull (<a href="#bib2">Collins et al., 2018</a>; <a href="#bib3">Meyerhoff et al., 2021</a>).</li>
<li>Transfer the brain into a petri dish containing cold HBSS without calcium/magnesium. Remove the cerebellum and brain stem, leaving cerebral hemispheres intact.</li>
<li>Bisect cerebral hemispheres along midline using a sterile razor blade. Repeat with the other hemisphere.</li>
<li>Place each hemisphere with the medial side facing up. Using curved forceps remove striatum, hippocampus and thalamus, leaving the cortex intact. Repeat with the other hemisphere.</li>
<li>Using a razor blade, mince cortex in small pieces and delicately transfer fragments using a sterile 3 mL plastic transfer pipette to a 12-well plate containing 1 mL of cold HBSS without calcium/magnesium.</li>
<li>Repeat with next mouse.</li>
<li>Once dissections are complete, delicately transfer each pair of minced cortex submerged in cold HBSS (without calcium/magnesium) from the 12-well plate into a 15 mL conical tube placed on ice, using a 3 mL transfer pipette.</li>
</ol>
</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1236-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Mouse cerebral cortex dissection from freshly extracted brain</div>
<p>(A) The brain is removed fresh from skull, and the cerebellum is discarded (dotted line depicts cut). Scale bar: 2 mm.</p>
<p>(B and C) Cerebral hemispheres are separated using a razor blade or scalpel (dotted midline shows cut axis). Dorsal view is shown in C (i.e., cortex surface up).</p>
<p>(D) Hemispheres are placed medial side facing up. Dotted line depicts the interior of the cortical ‘cup’ (i.e., striatum, hippocampus and thalamus) that will be scooped out. Scale bar: 2 mm.</p>
<p>(E) Using curved forceps, remove striatum, hippocampus and thalamus from cortical cup (see D). Straight forceps may be used to stabilize the brain during dissection.</p>
<p>(F and G) Remove leftover olfactory bulb including pigmented vessels using razor blade or scalpel (dotted line in F depicts cut). Scale bar: 2 mm.</p>
<p>(H) Using a razor blade, mince cortical pair in medium-to-small pieces to facilitate ensuing tissue dissociation (avoid mincing too small and for too long to limit damage to endothelial cells). Scale bar: 2 mm.</p>
<p>(I–K) Tissue dissociation steps for cell isolation. (I) Enzyme mix 1 and 2 have been added to minced tissue. (J) Cortical pair is dissociated after being pipetted up and down ~10 times with a 3mL transfer pipette. (K) Cortical pair is fully dissociated after being pipetted up and down ~10 times with a glass Pasteur pipette.</p>
</figcaption>
</figure>
<h4 id="sec3.1.3">Primary endothelial cell isolation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>All following steps are performed in sterile conditions in a biosafety cabinet. These steps have been adapted from the Miltenyi biotec “Neural Dissociation Kit (P)” and CD31 MicroBeads protocols found on Miltenyi biotec website. The volumes described below are for the isolation of one mouse brain (i.e., two cerebral cortices). The end result consists of a semi-pure endothelial cell culture. A pure endothelial cell culture will be reached after 48 h in culture (see steps 33–34).</p>
<ol start="7">
<li>Prepare a 6-well plate for endothelial cell culture.
<ol type="a">
<li>Add 2 mL of 100% ethanol per well of a 6-well plate. Let plate incubate at room temperature in the biosafety cabinet for 1 h. Aspirate 100% ethanol and let plate dry.
<div class="note">
<span class="note-title">Note:</span> Prepare 2 wells of a 6-well plate for each pair of cortical hemispheres.</div>
</li>
<li>Add 1 mL of 1× attachment factor (Gibco, Cat#S-006-100) to each well and let sit at room temperature for 30 min. Aspirate 1× attachment factor and let plate dry.
<div class="note">
<span class="note-title">Note:</span> The attachment factor contains gelatin which promotes the growth of microvascular endothelial cells.</div>
</li>
</ol>
</li>
</ol>
<ol start="8">
<li>Prepare PEB buffer (see <a href="#tbl1">Table 1</a>).
<table id="tbl1">
<caption>Table 1. PEB buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Albumin, bovine (BSA)</td>
<td>0.5%</td>
<td>0.25 g</td>
</tr>
<tr>
<td>HBSS without calcium/magnesium</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for up to 4 weeks.</p>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Steps 7 and 8 may be completed at the same time as the following steps of the cell isolation.</div>
<ol start="9">
<li>Prepare enzyme mix 1.
<ol type="a">
<li>Mix 1900 μL of buffer X with 50 μL of enzyme P [included in “Neural dissociation kit (P)”] for a final volume of 1950 μL. Do not vortex.</li>
<li>Pre-heat mixture at 37°C for 15 min.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare separate tubes of enzyme mix 1 for each mouse.</div>
<ol start="10">
<li>Centrifuge minced cortex at 300×<i>g</i> for 2 min at room temperature and aspirate supernatant.</li>
<li>Add 1950 μL of pre-heated enzyme mix 1 to each tube containing minced cortex.</li>
<li>Incubate minced cortex for 15 min at 37°C. Tap and gently swirl the tube (∼5 times) to dislodge tissue every 5 min.</li>
<li>Prepare enzyme mix 2.
<ol type="a">
<li>Mix 20 μL of buffer Y with 10 μL of enzyme A [included in “Neural dissociation kit (P)”] for a final volume of 30 μL. Do not vortex.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare separate tubes of enzyme mix 2 for each mouse.</div>
<ol start="14">
<li>Add 30 μL of enzyme mix 2 to each tube containing minced cortex following 15 min incubation (<a href="#fig1">Figure 1</a>I). Invert gently.</li>
<li>Dissociate cortical pair mechanically using a sterile 3 mL plastic transfer pipette, pipetting up and down slowly (∼10 times; <a href="#fig1">Figure 1</a>J). Avoid forming air bubbles.</li>
<li>Incubate tissue at 37°C for 10 min. Tap and gently swirl the tube every 5 min.</li>
<li>Dissociate tissue mechanically using a glass Pasteur pipette (<a href="#fig1">Figure 1</a>K). Avoid forming air bubbles.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Continue with dissociation until tissue can be pipetted up and down without difficulty (∼10 times). Pieces of tissue may still be visible following second dissociation.</div>
<ol start="18">
<li>Incubate dissociated tissue at 37°C for 10 min. Tap and gently swirl the tube every 5 min.</li>
<li>Place a 70 μM strainer on a 50 mL tube and wash strainer with 1 mL of HBSS with calcium/magnesium.</li>
<li>Apply cell suspension to prepared 70 μM strainer and add 10 mL of HBSS with calcium/magnesium to strainer.</li>
<li>Discard 70 μM strainer and centrifuge cell suspension at 300×<i>g</i> for 10 min at room temperature. Aspirate supernatant.</li>
<li>Resuspend cells with 1 mL of PEB buffer. Transfer to a 1.5 mL tube.</li>
<li>Centrifuge cell suspension at 300×<i>g</i> for 5 min at 4°C and aspirate supernatant.</li>
<li>Resuspend cell pellet in 270 μL of PEB buffer.</li>
<li>Prepare CD31 MicroBeads
<ol type="a">
<li>Vortex CD31 MicroBeads bottle.</li>
<li>Add 30 μL of CD31 MicroBeads to cell suspension.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> CD31 MicroBeads have been designed for the positive selection of endothelial cells from mouse tissues.</div>
<ol start="26">
<li>Incubate cell suspension containing CD31 MicroBeads for 20 min at 4°C while being agitated on a rotator.</li>
<li>Add 1 mL of PEB buffer to cell suspension and centrifuge at 300×<i>g</i> for 5 min at 4°C. Aspirate supernatant.</li>
<li>Resuspend pellet with 1 mL of PEB buffer.</li>
<li>Preparation for positive selection
<ol type="a">
<li>Place LS columns on magnetic separator (MACS® MultiStand and QuadroMACS™ Separator) with a 15 mL collection tube. One LS column per mouse is required.</li>
<li>Add 2 mL of PEB buffer to each LS column.</li>
<li>Warm 50 mL of endothelial cell culture medium at room temperature.</li>
</ol>
</li>
<li>Apply cell suspension to LS column and wash columns 3 times with 0.5 mL of PEB buffer. Discard total effluent.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Following positive selection, endothelial cells are trapped within the LS column.</div>
<ol start="31">
<li>Endothelial cell collection
<ol type="a">
<li>Label 15 mL conical tubes.</li>
<li>Remove LS column from magnetic separator and place on labelled 15 mL conical tube.</li>
<li>Add 2 mL of endothelial cell culture medium to each LS column. Immediately flush out endothelial cells by firmly pushing the plunger into the LS column.</li>
</ol>
</li>
<li>Add 1 mL of endothelial cells to the 1× attachment factor coated 6-well plate and 1 mL of endothelial cell medium to each well. Place plate in a 37°C incubator at 5% CO<sub>2</sub>.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Each mouse cortex will yield 2 wells of a 6-well plate using this protocol. Elution volume may be increased to plate endothelial cells in additional wells. However, caution must be taken when doing this approach as endothelial cells in culture require to be at a high density to assure attachment when first plated following isolation. This cell isolation protocol results in a semi-pure endothelial cell culture. A pure endothelial cell population will be achieved within 48 h of culture (see steps 33–34).</div>
<h4 id="sec3.1.4">Maintenance of primary mouse cerebral cortex ECs</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>Endothelial cells may require up to 7 days to reach confluency. However, growth rate will depend on initial endothelial cell plating density. Culture medium must be changed 48 h after isolation to remove non-adherent cells and replenish nutrients. After 48 h of culture a pure endothelial cell population is achieved.</p>
<ol start="33">
<li>Aliquot endothelial cell medium. A total of 2 mL is required per well. Let medium warm up at room temperature for 15 min. Slowly aspirate endothelial medium from wells containing cells and replace with fresh medium.</li>
<li>Endothelial cell medium must be replaced every second day up until 80% confluence is reached (∼7 days). <a href="#sec6.1">Troubleshooting 1</a>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Once endothelial cells are 70%–80% confluent, they may be used for subsequent quality controls and functional assessment. Conversely, cells may be frozen for later use using 10% DMSO.</div>
<h4 id="sec3.1.5">Freezing stocks of primary mouse cerebral cortex ECs</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="35">
<li>Harvest primary endothelial cells
<ol type="a">
<li>Aliquot TrypLE™ Express Enzyme (1×) in a 15 mL conical tube (1 mL per well of a 6-well plate). Incubate at 37°C for 30 min. Aliquot endothelial cell medium [same volume as TrypLE™ Express Enzyme (1×)].</li>
<li>Aspirate old culture medium from endothelial cells and wash twice with sterile 1× PBS. Aspirate 1× PBS.</li>
<li>Add 1 mL of TrypLE™ Express Enzyme (1×) per well containing cells and place in the 37°C, 5% CO<sub>2</sub> incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> After 5 minute incubation tap plate. Most cells should be detached. If several cells are still attached to the plate, place plate back in the 37°C, 5% CO<sub>2</sub> incubator for 2 min.</div>
</li>
<li>After incubation, add 1 mL of endothelial cell medium to each well and tap plate to detach cells.</li>
</ol>
</li>
</ol>
<ol start="36">
<li>Transfer cells to a 15 mL conical tube and centrifuge for 5 min at 500×<i>g.</i> Aspirate supernatant. Resuspend cell pellet using endothelial cell medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cells from one well should be frozen in one cryovial.</div>
<ol start="37">
<li>Add 900 μL of resuspended cells to each cryovial.</li>
<li>Slowly add 100 μL of sterile DMSO to each cryovial. Slowly pipette cells up and down to mix DMSO.</li>
<li>Place cryovials containing cells in Mr. Frosty™ freezing container.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This container ensures optimal cooling rate for cell preservation.</div>
<ol start="40">
<li>Place container at −80°C for 24 h.</li>
<li>Transfer cryovials from Mr. Frosty™ freezing container to liquid nitrogen for long term storage.</li>
</ol>
<h4 id="sec3.1.6">Thawing stocks of primary mouse cerebral cortex ECs</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="42">
<li>Prepare 6-well plate coated with 1× attachment factor as described for cell isolation. Warm 10 mL of endothelial cell medium at room temperature.</li>
<li>Aliquot 2 mL of pre-heated endothelial cell medium in a 15 mL conical tube.</li>
<li>Remove cryovial from liquid nitrogen and place in 37°C water bath for 1–2 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A small piece of frozen cells should still be visible after cryovial is removed from 37°C water bath.</div>
<ol start="45">
<li>Transfer cells to prepared 15 mL conical tube containing 2 mL of endothelial cell medium.</li>
<li>Centrifuge cells for 5 min at 500×<i>g.</i> Aspirate supernatant.</li>
<li>Resuspend pellet in endothelial cell medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> One cryovial should be separated into two wells of a 6-well plate with a final volume of 2 mL per well.</div>
<ol start="48">
<li>Place cells in a 37°C, 5% CO<sub>2</sub> incubator.</li>
<li>Endothelial cell medium must be replaced every second day up until 80% confluence is reached. Cells may be used for subsequent quality controls and experiments.</li>
</ol>
<h3 id="sec3.2">Section 1': Purification of endothelial RNA from mouse cerebral cortex</h3>
<p>The described cell isolation protocol in steps 1–32 yields a semi-pure population of ECs (purified passively through changes of culture medium). Indeed, after 48 h of culture, a pure EC population is reached (<a href="#fig2">Figure 2</a>). However, in order to perform the <u>acute</u> purification of endothelial RNA following EC isolation from mouse cerebral cortex, additional selection steps must be completed to ensure optimal purity of isolated ECs. Note that with the preparation below, a lower cell density is obtained compared to the isolation protocol described in steps 4–32.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1236-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Quality controls with primary mouse cortical ECs isolated from P14 male animals</div>
<p>(A and B) Control of endothelial culture purity using double staining for phalloidin and VE-Cadherin, at two time points. At 24 h, VE-Cadherin-negative contaminating cells (∗) can be observed, scale bar: 150μM (A). At 48 h, endothelial culture purity is achieved, as all cells display VE-Cadherin immunoreactivity, scale bar: 50μM (B).</p>
<p>(C) <i>Left,</i> Assessment of endothelial gene enrichment using RNAseq data (<a href="#bib4">Ouellette et al., 2020</a>) normalized to a publicly available database (<a href="#bib6">Zhang et al., 2014</a>; <a href="http://www.brainrnaseq.org/">http://www.brainrnaseq.org/</a>). <i>Right,</i> Assessment of neuronal contamination (same as <i>Left</i>). These data show that a very low level of contamination was achieved.</p>
<p>(D) Representative images (<i>left</i>) and quantification (<i>right</i>) of immunocytochemical staining for endothelial markers CD31 or VE-Cadherin. Distribution of VE-Cadherin fluorescence intensity is shown as example (10 μM × 20 μM white boxes represent areas of quantified staining). Scale bar: 40μM.</p>
<p>(E) Representative images (<i>left</i>) and quantification (<i>right</i>) for the assessment of apoptosis in EC cultures. The Caspase-3/7 green assay reveals apoptotic rates. Scale bar: 300μM.</p>
<p>In (A, B, D, and E), stained endothelial cells were imaged with a Zeiss Axio Imager M2 microscope equipped with a digital camera (Axiocam 506 mono) and the ApoTome.2 module.</p>
</figcaption>
</figure>
<h4 id="sec3.2.1">Acute isolation of primary cerebral cortex endothelial cells</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h [Start from step 21 of the primary EC isolation]</div>
<ol start="50">
<li>Prepare 50 mL of sterile 1× PBS and 0.5% BSA buffer (<a href="#tbl2">Table 2</a>).
<table id="tbl2">
<caption>Table 2. Sterile 1× PBS (50 mM, pH 7.4) and 0.5% BSA buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Albumin, bovine (BSA)</td>
<td>0.5%</td>
<td>0.25 g</td>
</tr>
<tr>
<td>Sterile 1× PBS</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at −20°C for up to 4 weeks.</p>
</li>
<li>Resuspend cells with 180 μL of prepared sterile 1× PBS and 0.5% BSA buffer and transfer to a 1.5 mL tube.</li>
<li>Add 20 μL of myelin removal beads to each tube.</li>
<li>Incubate at 4<sup>ο</sup>C for 15 min while being agitated on a rotator.</li>
<li>Preparation for negative selection
<ol type="a">
<li>Place LS columns on magnetic separator (MACS® MultiStand and QuadroMACS™ Separator) with a 15 mL collection tube. One LS column per mouse is required.</li>
<li>Add 3 mL of sterile 1× PBS and 0.5% BSA buffer to each LS column.</li>
</ol>
</li>
<li>Apply cell suspension to LS column and wash columns 2 times with 1 mL of sterile 1× PBS and 0.5% BSA buffer. Collect unlabeled cells that pass through the column.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not throw out effluent, this is the unlabeled cell fraction.</div>
<ol start="56">
<li>Centrifuge cell suspension in 15 mL collection tube at 300×<i>g</i> for 10 min. Aspirate supernatant completely.</li>
<li>Resuspend cell pellet in 270 μL of PEB buffer (<a href="#tbl1">Table 1</a>) and transfer to a new 1.5 mL tube.</li>
<li>Prepare CD45 MicroBeads for negative selection
<ol type="a">
<li>Vortex CD45 MicroBeads bottle.</li>
<li>Add 30 μL of CD45 MicroBeads to cell suspension.</li>
</ol>
</li>
<li>Incubate cell suspension containing CD45 MicroBeads for 20 min at 4°C while being agitated on a rotator.</li>
<li>Add 1 mL of PEB buffer to cell suspension and centrifuge at 300×<i>g</i> for 10 min at 4°C. Aspirate supernatant.</li>
<li>Resuspend pellet with 500 μL of PEB buffer.</li>
<li>Preparation for negative selection
<ol type="a">
<li>Place LD columns on magnetic separator (MACS® MultiStand and QuadroMACS™ Separator) with a 15 mL collection tube. One LD column per mouse is required.</li>
<li>Add 2 mL of PEB buffer to each LD column.</li>
</ol>
</li>
<li>Apply cell suspension to LD column and wash columns 2 times with 1 mL of PEB buffer. Collect unlabeled cells that pass through the column.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Do not throw out effluent, this is the unlabeled cell fraction.</div>
<ol start="64">
<li>Continue with step 23–31 in primary endothelial cell isolation protocol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Elution may be performed using PEB buffer instead of endothelial cell culture medium only for acute RNA extraction.</div>
<h4 id="sec3.2.2">RNA purification</h4>
<p>All following steps have been adapted from the OMEGA bio-tek “E.Z.N.A.® HP Total RNA kit” protocol.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h</div>
<ol start="65">
<li>Add 20 μL β-mercaptoethanol in 1 mL of GTC Lysis buffer found in OMEGA bio-tek E.Z.N.A.® HP Total RNA kit. Prepare 1 mL 70% ethanol.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Volume of prepared β-mercaptoethanol mixed with GTC Lysis buffer and 70% ethanol depends on number of samples to undergo the RNA purification (350 μL of buffer and 70% ethanol/sample is required).</div>
<ol start="66">
<li>Centrifuge 15 mL tubes containing ECs at 500×<i>g</i> for 5 min and aspirate supernatant.</li>
<li>Resuspend cells in 350 μL of GTC Lysis buffer mixed with β-mercaptoethanol. Vortex to mix thoroughly.</li>
<li>Insert a RNA homogenizer mini columns into a 2 mL collection tube.</li>
<li>Transfer lysate (from step 67) to the RNA homogenizer mini column. Centrifuge at 13,000×<i>g</i> for 1 min. Save filtrate and discard RNA homogenizer mini column.</li>
<li>Add 350 μL of 70% ethanol to filtrate. Vortex to mix thoroughly.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Using scissors cut the cap from a 2 mL tube in order to close the 2 mL collection tube. The kit does not provide caps for vortexing.</div>
<ol start="71">
<li>Insert a HiBlind® RNA mini column into a new 2 mL collection tube.</li>
<li>Transfer 700 μL sample to the HiBlind® RNA mini column. Centrifuge at 10,000×<i>g</i> for 1 min. Discard filtrate and reuse collection tube.</li>
<li>Prepare DNase I digestion mixture
<ol type="a">
<li>Add 73.5 μL of E.Z.N.A.® DNase I digestion buffer and 1.5 μL of RNase-free DNase I (20Kunitz/ μL) to a 1.5 mL tube.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Volume describe in step 73 is for one sample. Adjust volumes accordingly for total number of samples.</div>
<ol start="74">
<li>Add 250 μL of RNA wash buffer I to HiBlind® RNA mini column of each sample. Centrifuge at 10,000×<i>g</i> for 1 min. Discard the filtrate and reuse collection tube.</li>
<li>Add 75 μL of DNase I digestion mixture directly to the surface of the membrane of the HiBlind® RNA mini column.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Mixture must be added on the membrane to ensure DNA digestion is complete.</div>
<ol start="76">
<li>Let sit at room temperature for 15 min.</li>
<li>Add 250 μL RNA wash buffer I. Let sit at room temperature for 2 min. Centrifuge at 10,000×<i>g</i> for 1 min. Discard filtrate and reuse collection tube.</li>
<li>Add 500 μL of RNA wash buffer II prepared with 100% ethanol (preparation according to manufacturer’s protocol). Centrifuge at 10,000×<i>g</i> for 1 min. Discard filtrate and reuse collection tube.</li>
<li>Repeat step 78.</li>
<li>Centrifuge the empty HiBlind® RNA mini column at maximum speed for 2 min to dry the column.</li>
<li>Transfer HiBlind® RNA mini column to a clean 1.5 mL tube.</li>
<li>Add 60 μL of DNase-free water to HiBlind® RNA mini column. Centrifuge at maximum speed for 2 min.</li>
<li>Pass eluted sample (60 μL) through HiBlind® RNA mini column a second time. Centrifuge at maximum speed for 2 min.</li>
<li>Measure RNA concentration using Nanodrop. <a href="#sec6.3">Troubleshooting 2</a>
</li>
<li>Store eluted RNA at −80°C.</li>
<li>Extracted RNA may be used for subsequent reverse transcription- quantitative polymerase chain reaction (RT-qPCR), RNA deep-sequencing or microarray technologies.</li>
</ol>
<h3 id="sec3.7">Section 2: Validation of healthy primary mouse cerebral cortex endothelial cells</h3>
<p>The following steps are essentially completed before starting any assessment on endothelial cells. The quality controls are required when experiments involve cells isolated from mice with different genotypes or receiving treatments. This will identify any basic alterations between groups. These additional steps consist at a minimum confirmation of a pure endothelial cell population using immunocytochemistry, transcriptomics technologies, number of apoptotic cells, as well as endothelial cell proliferation.</p>
<h4 id="sec3.7.1">Quality control #1: Confirmation of the endothelial identity of primary ECs in culture</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼6 days</div>
<p>This quality control step consists of confirming the identity of the isolated cells. Here, immunocytochemistry is performed using endothelial cell markers (<a href="#fig2">Figures 2</a>A, 2B, and 2D).</p>
<ol start="87">
<li>Preparation of 12-well plate.
<ol type="a">
<li>Add a 12 mm glass coverslip to each well of a 12-well plate.</li>
<li>Add 1 mL of 100% ethanol to each well containing a 12 mm glass coverslip and let incubate for 1 h in sterile conditions. This is to sterilize glass coverslips.</li>
<li>After incubation, aspirate 100% ethanol and let dry.</li>
<li>Coat 12-well plate containing 12 mm glass coverslip with 500 μL of 1× attachment factor and let incubate for 30 min at room temperature in sterile conditions.</li>
<li>After incubation, aspirate 1× attachment factor and let dry.</li>
</ol>
</li>
<li>Harvest primary endothelial cells
<ol type="a">
<li>Aliquot TrypLE™ Express Enzyme (1×) in a 15 mL conical tube (1 mL per well of 6-well plate). Incubate at 37°C for 30 min. Aliquot endothelial cell medium [same volume as TrypLE™ Express Enzyme (1×)].
<div class="note">
<span class="note-title">Note:</span> One confluent well of a 6-well plate contains approximately 8 × 10<sup>4</sup> cells to 1 × 10<sup>5</sup> cells. A total of 2–3 confluent wells of a 6-well plate is required to seed 12 wells of a 12-well plate with 1 × 10<sup>4</sup> cells per well.</div>
</li>
<li>Aspirate old culture medium from endothelial cells and wash twice with sterile 1× PBS. Aspirate 1× PBS.</li>
<li>Add 1 mL of TrypLE™ Express Enzyme (1×) per well containing cells and place in a 37°C, 5% CO<sub>2</sub> incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> After 5 minute incubation tap plate. Most cells should be detached. If several cells are still attached to the plate, place the plate back in the 37°C, 5% CO<sub>2</sub> incubator for 2 min.</div>
</li>
<li>After incubation, add 1 mL of endothelial cell medium to each well and tap plate to detach cells.</li>
<li>Transfer cells to a 15 mL conical tube and centrifuge for 5 min at 500×<i>g.</i> Aspirate supernatant. Resuspend cells in 1 mL of endothelial cell medium.</li>
</ol>
</li>
<li>Count cells using a hemocytometer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Cells may be counted using an automated cell counter.</div>
<ol start="90">
<li>Seed cells at a density of 1 × 10<sup>4</sup> cells per well in a coated 12-well plate with a final volume of 1 mL per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cells are added directly to the coverslip to encourage attachment of cells. Once all cells have been seeded, medium is added.</div>
<ol start="91">
<li>Place 12-well plate containing cells in a 37°C, 5% CO<sub>2</sub> incubator for 3–4 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Endothelial cell media requires to be replaced every second day until 40%–60% confluence is reached.</div>
<ol start="92">
<li>Once cells have reached required confluency on glass coverslips, aspirate endothelial culture medium from 12-well plate containing cells. Wash cells twice with 1 mL sterile 1× PBS.</li>
<li>Fixation of endothelial cells
<ol type="a">
<li>Prepare 4% PFA in 1× PBS.</li>
<li>Add 500 μL of 4% PFA to 12-well plate containing 12 mm glass coverslips and cells. Let incubate for 10 min at room temperature.</li>
<li>Aspirate 4% PFA from cells and wash twice with 1 mL sterile 1× PBS. After second wash, add 1 mL of sterile 1× PBS to each well.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Once cells are fixed, the next steps do not need to be performed under a biosafety cabinet but can be continued at the bench.</div>
<ol start="94">
<li>Immunocytochemistry of endothelial cells.
<ol type="a">
<li>Prepare blocking solution [10% donkey serum, 0.5% fish gelatin mixed in 0.1% PBST (see <a href="#tbl3">Tables 3</a> and <a href="#tbl4">4</a>)]. A total of 1.5 mL per well is required for entire experiment.
<div class="note">
<span class="note-title">Note:</span> We use donkey serum because secondary antibody used here were raised in donkey. In general, the serum contained in the blocking solution should match species of the secondary antibodies.</div>
</li>
<li>Add 500 μL of blocking solution to each well containing cells and let incubate at room temperature for 60 min. Aspirate blocking solution.</li>
<li>Add 500 μL primary antibodies diluted in blocking solution to each well. We used endothelial markers, rabbit anti-VE-Cadherin at a concentration of 1:500 and rat anti-CD31 at a concentration of 1:200.
<div class="note">
<span class="note-title">Note:</span> A combination of additional non-endothelial markers is suggested to confirm the purity of isolated cells when initially establishing this protocol. These markers include primary antibodies for neurons (guinea-pig anti-NeuN, 1:1500), pericytes (goat anti-PDGFRß, 1:200), astrocytes (guinea-pig anti-GFAP, 1:1000), microglia (goat anti-Iba1, 1:1000) and smooth muscle cells (mouse anti-αSMA, 1:200).</div>
</li>
<li>Incubate at room temperature for 2 h while being slowly agitated on an orbital shaker.</li>
<li>Wash cells 3 × 5 min with 2 mL of 0.2% PBST (see <a href="#tbl3">Tables 3</a> and <a href="#tbl5">5</a>). Aspirate 0.2% PBST.</li>
<li>Add 500 μL of secondary antibodies diluted in blocking solution to each well. We used donkey anti-rabbit IgG (H+L) highly cross-absorbed secondary antibody, Alexa Fluor 568 at a concentration of 1:300 and donkey anti-rat IgG (H+L) highly cross-absorbed secondary antibody, Alexa Fluor 488 at a concentration of 1:300.
<div class="note">
<span class="note-title">Note:</span> For non-endothelial markers, suggested secondary antibodies consist of donkey anti-goat IgG (H+L) cross-absorbed secondary antibody Alexa Fluor 568 at a concentration of 1:300 for pericytes or microglia, goat anti-guinea pig IgG (H+L) highly cross-absorbed secondary antibody Alexa Fluor 488 at a concentration of 1:300 for astrocytes, goat anti-guinea pig IgG (H+L) highly cross-absorbed secondary antibody Alexa Fluor 647 at a concentration of 1:300 for neurons and donkey anti-mouse IgG (H+L) cross-absorbed secondary antibody Alexa Fluor 568 at a concentration of 1:300 for smooth muscle cells. Another suggested antibody to confirm pure EC population includes Alexa Fluor 488-Phalloidin for staining F-actin. This stain may be added to the wells following the 2 washes of 0.2% PBST (step 94h). Dilute Phalloidin at a concentration of 1:40 in the blocking solution and add to cells for 1 h. After incubation, proceed with step 94i.</div>
</li>
<li>Incubate at room temperature for 60 min while protected from light and slowly agitated on an orbital shaker. <a href="#sec6.5">Troubleshooting 3</a>
</li>
<li>Wash cells twice for 5 min with 2 mL of 0.2% PBST. Aspirate 0.2% PBST.</li>
<li>Add 1 mL of DAPI in 0.1M PB solution (see <a href="#tbl6">Table 6</a>) at a concentration of 1:20,000 to each well. Incubate for 5 min while protected from light. Aspirate DAPI solution.</li>
<li>Wash cells twice for 5 min with 1 mL of 0.1M PB.</li>
<li>Mounting slides
<ol type="i">
<li>Add ∼10 μL of Fluoromount-G™ media on a slide.</li>
<li>Using forceps remove glass coverslip with cells from 12-well plate.
<div class="critical">
<span class="critical-title">Critical:</span> Glass coverslips are extremely fragile. Do not apply too much pressure with forceps as they will break. Practice removal of glass coverslip from 12-well plate before attempting with cells. If glass coverslip breaks and falls on bench, it can still be imaged.</div>
</li>
<li>Dip coverslip in water and place face down on slide with Fluoromount-G™ drop.</li>
<li>Let slide dry at room temperature for 1 h while being protected from light. Place slides at 4°C overnight to dry.</li>
<li>Image cells.</li>
</ol>
</li>
</ol>
</li>
</ol>
<table id="tbl3">
<caption>Table 3. 1× PBS (50 mM, pH 7.4) buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>41.34 mM</td>
<td>5.87 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>H<sub>2</sub>O</td>
<td>8.69 mM</td>
<td>1.20 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>154 mM</td>
<td>9.00 g</td>
</tr>
<tr>
<td>Sterile water</td>
<td>n/a</td>
<td>Fill up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (20°C–22°C) for up to 1 year.</p>
<table id="tbl4">
<caption>Table 4. 0.1% PBST buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton® X-100</td>
<td>0.1%</td>
<td>250 μL</td>
</tr>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>249.75 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (20°C–22°C) for up to 1 year.</p>
<table id="tbl5">
<caption>Table 5. 0.2% PBST buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Triton® X-100</td>
<td>0.2%</td>
<td>500 μL</td>
</tr>
<tr>
<td>1× PBS</td>
<td>n/a</td>
<td>249.5 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>250 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (20°C–22°C) for up to 1 year.</p>
<table id="tbl6">
<caption>Table 6. PB (0.1M, pH 7.4) buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>HPO<sub>4</sub>
</td>
<td>82.7 mM</td>
<td>11.74 g</td>
</tr>
<tr>
<td>NaH<sub>2</sub>PO<sub>4</sub>H<sub>2</sub>O</td>
<td>17.39 mM</td>
<td>2.40 g</td>
</tr>
<tr>
<td>Sterile water</td>
<td>n/a</td>
<td>Fill up to 1 L</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at room temperature (20°C–22°C) for up to 1 year.</p>
<h4 id="sec3.7.2">Quality control #2: Measure the expression of signature endothelial genes and non-endothelial genes from purified RNA</h4>
<p>Additional gene expression assays such as RT-qPCR, RNA deep sequencing (RNAseq) or microarray technologies (<a href="#bib4">Ouellette et al., 2020</a>) can be used to confirm EC population purity following the acute purification of RNA described in <a href="#sec3.2">purification of endothelial RNA from mouse cerebral cortex</a> (<a href="#fig2">Figure 2</a>C).</p>
<h4 id="sec3.7.3">Quality control #3: Evaluation of apoptotic rate in cultured primary mouse cerebral cortex endothelial cells</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>The following step consist of a third quality control. This assay allows for the detection and quantification of programmed cell death following caspase activation (<a href="#fig2">Figure 2</a>E).</p>
<ol start="95">
<li>Preparation of 96-well plate.
<ol type="a">
<li>Coat a 96-well plate with 100 μL of 1× attachment factor.</li>
<li>Let plate incubate for 30 min at room temperature in sterile conditions.</li>
<li>Aspirate 1× attachment factor and let dry.</li>
</ol>
</li>
<li>Harvest primary endothelial cells.
<ol type="a">
<li>Aliquot TrypLE™ Express Enzyme (1×) in a 15 mL conical tube (1 mL per well of 6-well plate). Incubate at 37°C for 30 min. Aliquot endothelial cell medium [same volume as TrypLE™ Express Enzyme (1×)].
<div class="note">
<span class="note-title">Note:</span> One confluent well of a 6-well plate contains approximately 8 × 10<sup>4</sup> cells to 1 × 10<sup>5</sup> cells. One confluent well can seed up to one 96-well plate.</div>
</li>
<li>Aspirate old culture medium from endothelial cells and wash twice with sterile 1× PBS. Aspirate 1× PBS.</li>
<li>Add 1 mL of TrypLE™ Express Enzyme (1×) per well containing cells and place in a 37°C, 5% CO<sub>2</sub> incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> After 5 minute incubation tap plate. Most cells should be detached. If several cells are still attached to the plate, place the plate back in the 37°C, 5% CO<sub>2</sub> incubator for 2 min.</div>
</li>
<li>After incubation, add 1 mL of endothelial cell medium to each well and tap plate to detach cells.</li>
<li>Transfer cells to a 15 mL conical tube and centrifuge for 5 min at 500×<i>g.</i> Aspirate supernatant. Resuspend cells in 1 mL of endothelial cell medium.</li>
</ol>
</li>
<li>Count cells using a hemocytometer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Cells may be counted using an automated cell counter.</div>
<ol start="98">
<li>Seed cells at a density of 5 × 10<sup>3</sup> cells per well in coated 96-well plate with a final volume of 200 μL per well.</li>
<li>Place 96-well plate containing cells in a 37°C, 5% CO<sub>2</sub> incubator for 48 h.</li>
<li>Visually confirm that cells have attached to plate with 70%–80% confluence.</li>
<li>Treat 3–4 wells with 1 μM staurosporine for 6 h to act as the positive control.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Number of wells for positive control depends on available cells.</div>
<ol start="102">
<li>Remove media and replace with 5 μM of CellEvent™ Caspase-3/7 Green Detection Reagent diluted in endothelial cell medium to each well and incubate for 30 min in a 37°C, 5% CO<sub>2</sub> incubator.</li>
<li>Wash cells twice using 100 μL sterile 1× PBS.</li>
<li>Fix endothelial cells using 100 μL of 2% paraformaldehyde for 30 min and wash cells twice using 100 μL sterile 1× PBS.</li>
<li>Stain nuclei using Hoechst 33342 following manufacturers protocol.</li>
<li>Image cells.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Cellomics ArrayScan VTI high-content imaging platform running on HCS Studio software was used to image caspase-3 positive cells.</div>
<ol start="107">
<li>Following imaging, count caspase-3 positive cells (green) using Fiji-Image J (<a href="#bib5">Schindelin et al., 2012</a>).</li>
</ol>
<h4 id="sec3.7.4">Quality control #4: Quantification of endothelial cell proliferation in culture using a thymidine analogue</h4>
<div class="timing">
<span class="timing-title">Timing:</span> ∼6 days</div>
<p>This fourth quality control step consist of the assessment of proliferation in cell culture using an 5-Ethynyl-2′-deoxyuridine (EdU) incorporation assay.</p>
<ol start="108">
<li>Preparation of 12-well plate.
<ol type="a">
<li>Add a 12 mm glass coverslip to each well of a 12-well plate.</li>
<li>Add 1 mL of 100% ethanol to each well containing a 12 mm glass coverslip and let incubate for 1 h in sterile conditions. This is to sterilize glass coverslips.</li>
<li>After incubation, aspirate 100% ethanol and let dry.</li>
<li>Coat 12-well plate containing 12 mm glass coverslip with 500 μL of 1× attachment factor and let incubate for 30 min at room temperature in sterile conditions.</li>
<li>After incubation, aspirate 1× attachment factor and let dry.</li>
</ol>
</li>
<li>Harvest primary endothelial cells.
<ol type="a">
<li>Aliquot TrypLE™ Express Enzyme (1×) in a 15 mL conical tube (1 mL per well of 6-well plate). Incubate at 37°C for 30 min. Aliquot endothelial cell medium [same volume as TrypLE™ Express Enzyme (1×)].</li>
<li>Aspirate old culture medium from endothelial cells and wash twice with sterile 1× PBS. Aspirate 1× PBS.</li>
<li>Add 1 mL of TrypLE™ Express Enzyme (1×) per well containing cells and place in a 37°C, 5% CO<sub>2</sub> incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> After 5 minute incubation tap plate. Most cells should be detached. If several cells are still attached to the plate, place the plate back in the 37°C, 5% CO<sub>2</sub> incubator for 2 min.</div>
</li>
<li>After incubation, add 1 mL of endothelial cell medium to each well and tap plate to detach cells.</li>
<li>Transfer cells to a 15 mL conical tube and centrifuge for 5 min at 500×<i>g.</i> Aspirate supernatant. Resuspend cells in 1 mL of endothelial cell medium.</li>
</ol>
</li>
<li>Count cells using a hemocytometer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Cells may be counted using an automated cell counter.</div>
<ol start="111">
<li>Seed cells at a density of 5 × 10<sup>3</sup> cells per well in coated 12-well plate with a final volume of 1 mL per well.</li>
<li>Place 12-well plate containing cells in a 37°C, 5% CO<sub>2</sub> incubator for 3–4 days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Endothelial cell media requires to be replaced every second day until ∼60% confluence is reached.</div>
<ol start="113">
<li>Rates of endothelial cell proliferation was determined using a Click-iT 5-ethynyl-2′-deoxyuridine (EdU) assay. Protocol was adapted from the manufacturer’s instructions.
<ol type="a">
<li>Prepare 20 mM stock solution of EdU (5 mg of EdU in 1 mL of sterile 1× PBS).</li>
<li>Dilute 20 mM EdU in 1/10 endothelial cell medium (5 μL of 20 mM EdU in 45 μL of endothelial cell medium).</li>
<li>Add 10 μL of diluted EdU in 490 μL of endothelial cell medium (final concentration 20 μM in 500 μL).
<div class="note">
<span class="note-title">Note:</span> This volume is for 1 well of a 12-well plate. Volume will need to be modified accordingly for more wells containing cells.</div>
</li>
<li>Remove 500 μL of endothelial cell medium of each well of 12-well plate containing cells.</li>
<li>Add 500 μL of EdU at 20 μM to each well containing cells.
<div class="note">
<span class="note-title">Note:</span> A final volume of 1 mL is in each well containing cells. Therefore, final concentration of EdU in well is 10 μM.</div>
</li>
<li>Incubate cells in a 37°C, 5% CO<sub>2</sub> incubator for 48 h without changing endothelial cell medium.</li>
<li>Fixation of endothelial cells
<ol type="i">
<li>Prepare 4% PFA in 1× PBS.</li>
<li>Add 500 μL of 4% PFA to 12-well plate containing 12 mm glass coverslips and cells. Let incubate for 10 min at room temperature.</li>
<li>Aspirate 4% PFA from cells and wash twice with 1 mL of 3% BSA in 1× PBS.
<div class="note">
<span class="note-title">Note:</span> Once cells are fixed, the next steps do not need to be performed under a biosafety cabinet but may be continued at the bench.</div>
</li>
<li>Aspirate washing solution. Add 1 mL of 0.5% Triton® X-100 in 1× PBS to each well. Incubate at room temperature for 20 min.</li>
</ol>
</li>
<li>EdU detection
<ol type="i">
<li>Prepare 1× Click-iT® reaction cocktail according to manufacturer’s protocol (found in Click-iT™ EdU Alexa Fluor™ 594 Imaging Kit).</li>
<li>Remove 0.5% Triton® X-100 in 1× PBS solution from each well. Wash cells twice with 1 mL of 3% BSA in 1× PBS.</li>
<li>Add 0.5 mL of 1× Click-iT® reaction cocktail to each well containing cells.</li>
<li>Incubate plate for 30 min at room temperature, while protected from light.</li>
<li>Remove reaction cocktail and wash cells once with 1 mL of 3% BSA in 1× PBS.</li>
<li>Proceed to immunocytochemistry. Follow protocol as of step 94c.
<div class="note">
<span class="note-title">Note:</span> Primary antibody and secondary antibody used following EdU incorporation assay are rabbit anti-VE-Cadherin at a concentration of 1:500 and donkey anti-rabbit IgG (H+L) highly cross-absorbed secondary antibody, Alexa Fluor 647 at a concentration of 1:300. <a href="#sec6.7">Troubleshooting 4</a>
</div>
</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="114">
<li>Image cells.
<ol type="a">
<li>Take ten images using 20× objective.</li>
<li>Count EdU positive cells using the Cell Counter plugin in Fiji-Image J (<a href="#bib5">Schindelin et al., 2012</a>).</li>
</ol>
</li>
</ol>
<h3 id="sec3.8">Section 3: Functional assessment using cultured primary mouse cerebral cortex endothelial cells</h3>
<h4 id="sec3.8.1">Assessment of endothelial cell migration</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>Cells must have reached 70%–80% confluence before completing scratch wound assay. This assay is a convenient method for analysis of cell migration (<a href="#fig3">Figure 3</a>A). This protocol is adapted from the Incucyte ZOOM® 96-Well Scratch Wound Cell Migration &amp; Invasion Assays User Manual. <a href="#sec6.9">Troubleshooting 5</a></p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1236-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Expected outcomes with primary mouse cerebral cortex ECs isolated from P14 male animals</div>
<p>(A) Representative images (<i>left</i>) and quantification (<i>right</i>) for a scratch wound-healing assays used to measure migration of primary mouse brain ECs. The orange lines indicate cellular migration fronts. Scale bar: 300μM.</p>
<p>(B) <i>In vitro</i> network formation assay using primary mouse brain ECs from P14 brains to assess vascular network formation and remodeling over 48 h in a growth factor-reduced Matrigel®. <i>Left</i>, representative images at the 8hr time point, scale bar: 500μM. <i>Right</i>, Quantifications of network density (i.e., total endothelial tube length) and network nodes (i.e., total number of branching hubs). Image in (B) was captured using a Nikon Eclipse TE2000- E inverted microscope. All data displayed are mean ± S.E.M.</p>
</figcaption>
</figure>
<ol start="115">
<li>Prepare 96-well ImageLock plate.
<ol type="a">
<li>Coat a 96-well ImageLock plate with 100 μL of 1× attachment factor.</li>
<li>Let plate incubate for 30 min at room temperature in sterile conditions.</li>
<li>Aspirate 1× attachment factor and let dry.</li>
</ol>
</li>
<li>Harvest primary endothelial cells.
<ol type="a">
<li>Aliquot TrypLE™ Express Enzyme (1×) in a 15 mL conical tube (1 mL per well of 6-well plate). Incubate at 37°C for 30 min. Aliquot endothelial cell medium [same volume as TrypLE™ Express Enzyme (1×)].</li>
<li>Aspirate old culture medium from endothelial cells and wash twice with sterile 1× PBS. Aspirate 1× PBS.</li>
<li>Add 1 mL of TrypLE™ Express Enzyme (1×) per well containing cells and place in a 37°C, 5% CO<sub>2</sub> incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> After 5 minute incubation tap plate. Most cells should be detached. If several cells are still attached to the plate, place the plate back in the 37°C, 5% CO<sub>2</sub> incubator for 2 min.</div>
</li>
<li>After incubation, add 1 mL of endothelial cell medium to each well and tap plate to detach cells.</li>
<li>Transfer cells to a 15 mL conical tube and centrifuge for 5 min at 500×<i>g.</i> Aspirate supernatant. Resuspend cells in 1 mL of endothelial cell medium.</li>
</ol>
</li>
</ol>
<ol start="117">
<li>Count cells using a hemocytometer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Cells may be counted using an automated cell counter.</div>
<ol start="118">
<li>Seed cells at a density of 2 × 10<sup>4</sup> cells per well in coated 96-well ImageLock plate with a final volume of 100 μL per well.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All wells of 96-well ImageLock plate must have 100 μL, even if there are no cells in wells. Do not write on top of the plate.</div>
<ol start="119">
<li>Place 96-well ImageLock plate containing cells in a 37°C, 5% CO<sub>2</sub> incubator for 48 h.</li>
<li>Preparing and creating wounds using WoundMaker™
<ol type="a">
<li>Fill wash boat 1 with 45 mL of sterile water and place WoundMaker™ pin block in water for 5 min.</li>
<li>Fill wash boat 2 with 45 mL of 70% ethanol and place WoundMaker™ pin block in the solution for 5 min.</li>
<li>Place WoundMaker™ pin block on an empty wash boat.</li>
<li>Insert 96-well ImageLock plate containing cells and media into the base plate holder. Remove plate cover.</li>
<li>Place WoundMaker™ pin block onto the base plate holding the cells. Push and hold lever.
<div class="critical">
<span class="critical-title">Critical:</span> If cells are too confluent, large sheets of cells will be removed and wound will not be uniform between wells. <a href="#sec6.11">Troubleshooting 6</a>
</div>
</li>
<li>Lift WoundMaker™ pin block and replace with 96-well ImageLock plate cover.</li>
<li>Aspirate and replace endothelial cell culture medium to remove floating cells.</li>
</ol>
</li>
<li>Place 96-well ImageLock plate containing cells and 100 μL fresh media into the IncuCyte™ Zoom® Apparatus to record cell migration.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> ZOOM software is scheduled for repeat scanning with images taken every 2 h for 24 h. In the software make sure ‘scan type’ is set to Scratch Wound and Wide Mode.</div>
<ol start="122">
<li>Clean WoundMaker™ after use.
<ol type="a">
<li>Fill wash boat 4 with 45 mL of 0.5% Alconox and place WoundMaker™ pin block in solution for 5 min.</li>
<li>Fill wash boat 5 with 45 mL of 1% Virkon S. and place WoundMaker™ pin block in the solution for 5 min.</li>
<li>Fill wash boat 1 with 45 mL of sterile water and place WoundMaker™ pin block in water for 5 min.</li>
<li>Fill wash boat 2 with 45 mL of 70% ethanol and place WoundMaker™ pin block in the solution for 5 min.</li>
<li>Place WoundMaker™ on its base and store at room temperature.</li>
</ol>
</li>
<li>Quantification of migration assay is performed using IncuCyte® ZOOM Software. More detailed information on data analysis can be found at the Essen BioScience website.</li>
</ol>
<h4 id="sec3.8.2">Assessment of endothelial cell angiogenic activity</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<p>The following steps are to be completed after cell isolation and 70%–80% confluence has been reached. This endothelial cell tube formation assay measures the ability of endothelial cells to form capillary-like structures in a extracellular matrix (Matrigel®; <a href="#fig3">Figure 3</a>B).</p>
<p><b>Day prior to tube formation assay</b></p>
<ol start="124">
<li>Place a 96-well plate and 200 μL pipette tips at −20°C overnight.</li>
<li>Reduced growth factor Matrigel® should be at 4°C overnight to ensure slow thawing.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Reduced growth factor Matrigel® must be handled with care. Rapid thawing will lead to polymerization of matrix. Matrix must always be on ice or at 4°C.</div>
<p><b>Day of tube formation assay</b></p>
<ol start="126">
<li>Harvest primary endothelial cells and preparation of growth factor reduced Matrigel® plate
<ol type="a">
<li>Aliquot TrypLE™ Express Enzyme (1×) in a 15 mL conical tube (1 mL per well of 6-well plate). Incubate at 37°C for 30 min. Aliquot endothelial cell medium [same volume as TrypLE™ Express Enzyme (1×)].</li>
<li>Place 96-well plate and 200 μL pipette tips (from the −20°C freezer) as well as growth factor reduced Matrigel on ice in the biosafety cabinet.</li>
<li>Aspirate old culture medium from endothelial cells and wash twice with sterile 1× PBS. Aspirate 1× PBS.</li>
<li>Add 1 mL of TrypLE™ Express Enzyme (1×) per well containing cells and place in a 37°C, 5% CO<sub>2</sub> incubator for 5 min.
<div class="critical">
<span class="critical-title">Critical:</span> After 5 minute incubation, tap plate. Most cells should be detached. If several cells are still attached to the plate, return plate to the 37°C, 5% CO<sub>2</sub> incubator for 2 min.</div>
</li>
<li>During 5 min incubation, add 50 μL of growth factor reduced Matrigel® per well of cold 96-well plate. Quickly remove any visible bubbles in wells containing growth factor reduced Matrigel®. After removing bubbles, a minimum of 40 μL growth factor reduced Matrigel® must be left in the well for assay. Place 96-well plate containing growth factor reduced Matrigel in a 37°C, 5% CO<sub>2</sub> incubator for 30 min.
<div class="note">
<span class="note-title">Note:</span> Hold the neck of the Matrigel® bottle to reduce heat transfer. Prepare 3 wells of growth factor reduced Matrigel per mouse for accurate assessment of angiogenic capacity. Always leave the 96-well plate on ice to prevent Matrigel® polymerization until all wells are ready to be transferred to the incubator.</div>
</li>
<li>After incubation, add 1 mL of endothelial cell medium to each well of 6-well plate and tap plate to detach cells.</li>
<li>Transfer cells to a 15 mL conical tube and centrifuge for 5 min at 500×<i>g.</i> Aspirate supernatant. Resuspend cells in 1 mL of endothelial cell medium.</li>
</ol>
</li>
</ol>
<ol start="127">
<li>Count cells using a hemocytometer.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Cells may be counted using an automated cell counter.</div>
<ol start="128">
<li>A total of 6 × 10<sup>4</sup> cells is required per mouse in a final volume of 450 μL of endothelial cell medium.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For each mouse, 3 wells of a 96-well plate will be prepared. A total of 6 × 10<sup>4</sup> cells per mouse is needed. Each well of the 96-well plate will contain 2 × 10<sup>4</sup> cells. One confluent well of a 6-well plate is needed to prepare 3 wells of a 96-well plate with matrigel (one confluent well contains approximately 8 × 10<sup>4</sup> cells to 1 × 10<sup>5</sup> cells).</div>
<ol start="129">
<li>After the 30 min incubation of the 96-well plate containing growth factor reduced Matrigel®, seed 2 × 10<sup>4</sup> cells (150 μL) per well containing matrix.</li>
<li>Place 96-well plate containing matrix and endothelial cells in a 37°C, 5% CO<sub>2</sub> incubator until imaging time points (4 h, 8 h, 24 h, 48 h). <a href="#sec6.13">Troubleshooting 7</a>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Imaging was completed with a Nikon Eclipse TE2000-E inverted microscope.</div>
<ol start="131">
<li>Quantification of tube formation assay using Fiji-image J (<a href="#bib5">Schindelin et al., 2012</a>).
<ol type="a">
<li>Download macro: Angiogenesis Analyzer for ImageJ (<a href="#bib1">Carpentier, 2012</a>)</li>
<li>Install and open macro in Fiji-image J.</li>
<li>Choose file containing all tube formation images and select “Analyze HUVECS Phase Contrast” and let macro run.</li>
<li>Once analysis is complete, export file in Excel. Branching length is found under “Total branching length” column and number of nodes is found under “Nb master junction” column. Using this data angiogenic activity differences between may be determined.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Our protocol yields pure primary mouse brain EC cultures, whereby ECs can be utilized for subsequent characterization and functional assays. In culture, ECs will reach a pure population within 48 h of culture, and confluence within 7 days. Using a double staining of phalloidin and VE-Cadherin revealed a pure EC population after 48 h in culture as all cells are VE-Cadherin positive (<a href="#fig2">Figures 2</a>A and 2B). As of this stage, population purity and health can be confirmed, and assessment of functional characteristics can be completed. Of note, endothelial behavior will start to change after two passages. Therefore, cell isolation from multiple mice may be required to complete the entirety of this protocol, in order to eliminate the need of additional passaging of cells. We emphasize the importance of validating the nature of ECs and of confirming their health in culture prior to any functional assessment. Additional transcriptomics assays can confirm EC population purity following acute cell isolation. Using deep RNA sequencing, we confirmed endothelial gene enrichment from purified endothelial RNA as well as a very low neuronal contamination (<a href="#fig2">Figure 2</a>C). Following immunocytochemical staining for endothelial markers CD31 and VE-cadherin (other markers can be used), intercellular junctions can be delineated and their distribution quantified. For instance, normal distribution of staining can be quantified across cell-cell junctions using the profile plot tool in ImageJ. A peak in staining intensity should be observed at cell-cell junctions (<a href="#fig2">Figure 2</a>D). Using the caspase-3/7 green assay to assess apoptotic rates of cortical ECs allows direct count of the proportion of apoptotic cells. Number of apoptotic cells (green) are counted and expressed as % of total cells in each image. In our hands, healthy WT mouse brain ECs isolated at P14 yield ∼4% of apoptotic ECs (<a href="#fig2">Figure 2</a>E). Our protocol also describes a fourth quality control with the measure of EC proliferation. The number of EdU positive cells is counted and expressed as % of total cells. For more information, please consult our recent publication <a href="#bib4">Ouellette et al. (2020)</a> in which this experiment was performed using human-derived endothelial cells. Once EC identity and health are confirmed, these cells can undergo various functional assays. This protocol describes two assays: a cell migration assay and an angiogenic activity assay. For cell migration assessment, a scratch wound is imaged and measured during 24 h. Imaging at different time points during this period allows to measure the percent of healed area throughout time (<a href="#fig3">Figure 3</a>A). To measure angiogenic activity of ECs, a Matrigel®-based network formation assay is used. Imaging at different time points allows to quantify the formation of new endothelial networks. At 8 h post-seeding, branching and nodes can be readily identified in this 3D matrix. An extensive network will be formed until ∼12 h, after which the network will collapse due to nutrient depletion within the well (<a href="#fig3">Figure 3</a>B).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol has been optimized for the isolation and characterization of primary mouse cerebral cortex endothelial cells (ECs) isolated from P14 pups. While these steps can be applied to younger (e.g., embryos, &lt;P14) or adult (≥P50) mice, note that cell isolations from adult mice are more challenging and yield a lower number of ECs. Thus, adult mice cortices may need to be pooled to get a higher cell density. Moreover, adult primary brain ECs are more sensitive to manipulations and require an extended amount of time for growth before functional assays.</p>
<p>The cell isolation procedure described in <a href="#sec3.1">primary mouse cerebral cortex endothelial cell isolation and culture</a> yields a semi-pure EC population, in part contaminated by myelin debris; therefore, the precise number of ECs following this isolation cannot be determined. However, once EC colonies grow and expand, a total of 80,000–100,000 confluent cells can be counted in one well from a 6-well plate. With acute isolation of ECs for gene expression studies (<a href="#sec3.2">purification of endothelial RNA from mouse cerebral cortex</a>), the number of cells is not counted to preserve cell population integrity by reducing manipulations. As noted in <a href="#bib4">Ouellette et al. (2020)</a>, ECs from two P14 brains (four cerebral cortices) are pooled into one biological replicate to obtain a sufficient amount of total RNA for transcriptomic studies.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Low number cells have attached to culture plate (step 34).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Following cell isolation decrease elution volume in order to have a higher density of endothelial cells for initial plating. Plate all cells within one well of a coated 6-well plate.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Low RNA concentration (step 84).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>To increase RNA concentration, pool ECs from 4 cortices (2 mice) following ECs elution from LS column. This will increase the number of cells prior to the RNA extraction. Therefore, yield a higher concentration of RNA.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Endothelial cells have detached during immunocytochemistry (step 94g).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>During all incubations, plates containing cells may be slowly agitated. However, if cells are being washed away during incubation or washing steps, leave plate on the bench without being agitated during incubations. During washing steps, a plastic transfer pipette with a 200 μL tip can be used to aspirate buffers to decrease the chance of disturbing the cells.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Emission bleed between A594 and A647 (step 113h).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>An alternate fluorophore may be used for immunocytochemistry. Secondary antibody donkey anti-rabbit IgG (H+L) highly cross-absorbed secondary antibody, Alexa Fluor 488 at a concentration of 1: 300 is recommended instead of Alexa Fluor 647.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Incucyte ZOOM® equipment is not available (steps 115–123).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>A traditional scratch wound assay can be used in this quality control protocol. In this case, the plate is prepared as described in step 95. Endothelial cells are seeded at a density 2 × 10<sup>4</sup> cells per well of a coated 96-well plate. The plate containing the cells is placed in a 37°C, 5% CO<sub>2</sub> incubator for 48 h. A sterile 200 μL pipette tip is used to create a straight wound withing each well. Cell migration imaging at time points of interest can be completed using an inverted phase contrast microscope attached to a camera until the wound is closed.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>Large cell sheets have been removed following wound (step 120e).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>If cells are too confluent, large cell sheets will be removed during wound creating: 1) decrease seeding density of endothelial cells, or 2) create wound after 24 h of plating instead of 48 h. By decreasing cell seeding density and time before assay, a single layer of cells will be formed. This single layer of cells will lead to a defined wound.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>Cells within matrix appear round and no branching has formed after 24 h (step 130).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>If cells are dying within the matrix, it is possible that the amount of time allotted for cell counting is too long. Thus, it would be beneficial to harvest cells in batches that way they are out of an optimal growing environment for a shorter amount of time increasing their chance of survival.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Carpentier, G. (2012). Contribution: Angiogenesis Analyzer (ImageJ News). <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref1">View at publisher</a></p>
<p id="bib2">Collins, S., Wagner, C., Gagliardi, L., Kretz, P., Fischer, M.-C., Kessler, P., Kannan, M., and Yalcin, B. (2018). A method for parasagittal sectioning for neuroanatomical quantification of brain structures in the adult mouse. Curr. Protoc. Mouse Biol. <i>8</i>, e48. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref2">View at publisher</a></p>
<p id="bib3">Meyerhoff, J., Muhie, S., Chakraborty, N., Naidu, L., Sowe, B., Hammamieh, R., Jett, M., and Gautam, A. (2021). Microdissection of mouse brain into functionally and anatomically different regions. J. Vis. Exp. <i>168</i>, e61941. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref3">View at publisher</a></p>
<p id="bib4">Ouellette, J., Toussay, X., Comin, C.H., Costa, L.D.F., Ho, M., Lacalle-Aurioles, M., Freitas-Andrade, M., Liu, Q.Y., Leclerc, S., Pan, Y., et al. (2020). Vascular contributions to 16p11.2 deletion autism syndrome modeled in mice. Nat. Neurosci. <i>23</i>, 1090-1101. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref4">View at publisher</a></p>
<p id="bib5">Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., et al. (2012). Fiji: an open-source platform for biological-image analysis. Nat. Methods <i>9</i>, 676-682. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref5">View at publisher</a></p>
<p id="bib6">Zhang, Y., Chen, K., Sloan, S.A., Bennett, M.L., Scholze, A.R., O'Keeffe, S., Phatnani, H.P., Guarnieri, P., Caneda, C., Ruderisch, N., et al. (2014). An RNA-sequencing transcriptome and splicing database of glia, neurons, and vascular cells of the cerebral cortex. J. Neurosci. <i>34</i>, 11929-11947. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00725-5/sref6">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Baptiste Lacoste: <a href="mailto:blacoste@uottawa.ca">blacoste@uottawa.ca</a></p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>Data and codes are available from the lead contact upon request as well as from supplementary material from <a href="#bib4">Ouellette et al. (2020)</a> <i>Nat. Neurosci</i>.</p>
<h3>Acknowledgments</h3>
<p>For this work, J.O. was supported by Ph.D. scholarship awards from the Canadian Institutes of Health Research (CIHR) and the Canadian Vascular Network (CVN), and B.L. by a research grant from the <a href="https://doi.org/10.13039/501100000024">CIHR</a> (grant #388805). We thank the Cell Biology and Image Acquisition (CBIA) Core Facility at Faculty of Medicine at University of Ottawa.</p>
<h3>Author contributions</h3>
<p>J.O. optimized and carried out experiments, wrote the manuscript draft and participated in figure generation. B.L. proofread the manuscript, provided guidance in writing, and participated in figure assembly.</p>
<h3>Declaration of interests</h3>
<p>Authors declare no competing interest.</p>
</section>
</article>