<article>
<section>
<h1 id="header">Fluorescent tagging of endogenous proteins with CRISPR/Cas9 in primary mouse neural stem cells</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Christopher S. Morrow,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a></sup> Tiaira J. Porter,<sup><a href="#aff1">1</a></sup> and Darcie L. Moore<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Neuroscience, University of Wisconsin-Madison, Madison, WI 53705, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:darcie.moore@wisc.edu">darcie.moore@wisc.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100744">10.1016/j.xpro.2021.100744</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Although exogenous overexpression of a protein fused to a fluorescent tag can provide insight for the protein’s function, it also can produce artifacts attributed to its upregulation and may not fully report the endogenous regulation of the protein of interest. To circumvent these issues, we adapted a protocol to label endogenous proteins with fluorescent tags in primary adult mouse neural stem cells <i>in vitro</i>. Here, we describe reagent construction, reagent delivery, and a screening strategy to isolate edited cells.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref15">Morrow et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Fluorescent tagging of endogenous proteins in primary mouse neural stem cells</li>
<li>CRISPR-Cas9-based knock-in reagent design, construction, and delivery</li>
<li>Strategies for isolating successfully edited primary mouse neural stem cells</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>To label endogenous proteins with fluorescent tags with CRISPR/Cas9 through homology directed repair (HDR), neural stem cells (NSCs) are transfected with a plasmid driving expression of SpCas9 protein and a single guide RNA (sgRNA; including the variable crRNA which targets a specific location in the genome fused to a consistent tracrRNA which scaffolds the sgRNA with Cas9), as well as a separate targeting vector plasmid encoding the fluorescent tag (<a href="#bib5">Chu et al., 2015</a>, <a href="#bib7">Doudna and Charpentier, 2014</a>, <a href="#bib13">Mali et al., 2013</a>, <a href="#bib11">Kwart et al., 2017</a>, <a href="#bib16">Paquet et al., 2016</a>, <a href="#bib18">Ran et al., 2013</a>). SpCas9 protein induces a double-stranded break in the genome as directed by the sgRNA, and the targeting vector plasmid provides a DNA template for the cell to repair the cut through HDR, thereby allowing the knock-in of the desired fluorescent tag coding sequence into the genome (<a href="#bib3">Bressan et al., 2017</a>, <a href="#bib6">Dewari et al., 2018</a>). As an example, this protocol will describe how to label the intermediate filament vimentin in NSCs with an mNeon fluorescent tag as previously described (<a href="#bib15">Morrow et al., 2020</a>). Although this protocol is specifically tailored to primary mouse NSCs, this protocol can be adapted for performing fluorescent tagging in other mammalian cell types, and components of this protocol should be modified to meet specific experimental objectives. Isolation and culturing of primary mouse NSCs has been previously described (<a href="#bib1">Ahmed et al., 2021</a>, <a href="#bib17">Peltier et al., 2010</a>, <a href="#bib15">Morrow et al., 2020</a>).</p>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>DH5α-T1</td>
<td>Invitrogen</td>
<td>Cat#:12297016</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>NEB</td>
<td>Cat#:M0202S</td>
</tr>
<tr>
<td>T4 Polynucleotide Kinase (PNK)</td>
<td>NEB</td>
<td>Cat#:M0201S</td>
</tr>
<tr>
<td>Phusion High-Fidelity DNA Polymerase</td>
<td>Invitrogen</td>
<td>Cat#:F530L</td>
</tr>
<tr>
<td>Poly-L-ornithine</td>
<td>Sigma</td>
<td>Cat#:P3655</td>
</tr>
<tr>
<td>Laminin</td>
<td>Sigma</td>
<td>Cat#:L2020</td>
</tr>
<tr>
<td>Shrimp Alkaline Phosphatase</td>
<td>NEB</td>
<td>Cat#:M0371S</td>
</tr>
<tr>
<td>DNAse</td>
<td>Sigma-Aldrich</td>
<td>Cat#:D5025-15KU</td>
</tr>
<tr>
<td>Trypsin</td>
<td>Invitrogen</td>
<td>25300-054</td>
</tr>
<tr>
<td>Versene</td>
<td>Thermo Fisher Scientific</td>
<td>15040066</td>
</tr>
<tr>
<td>Trypsin Inhibitor</td>
<td>Sigma</td>
<td>T6522</td>
</tr>
<tr>
<td>DMEM/F12 GlutaMax</td>
<td>Invitrogen</td>
<td>10565018</td>
</tr>
<tr>
<td>Penicillin-streptomycin-fungizone</td>
<td>Invitrogen</td>
<td>15140122</td>
</tr>
<tr>
<td>B27</td>
<td>Invitrogen</td>
<td>17504044</td>
</tr>
<tr>
<td>FGF-2</td>
<td>PeproTech</td>
<td>100-18B</td>
</tr>
<tr>
<td>EGF</td>
<td>PeproTech</td>
<td>AF-100-15</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Gibson Assembly Cloning Kit</td>
<td>NEB</td>
<td>Cat#:E5510S</td>
</tr>
<tr>
<td>Mouse Neural Stem Cell Nucleofector Kit</td>
<td>Lonza</td>
<td>Cat#:VPG-1004</td>
</tr>
<tr>
<td>EndoFree Plasmid Maxi Kit</td>
<td>QIAGEN</td>
<td>Cat#:12362</td>
</tr>
<tr>
<td>QIAquick Gel Extraction Kit</td>
<td>QIAGEN</td>
<td>Cat#:28704</td>
</tr>
<tr>
<td>GeneJET Plasmid Miniprep Kit</td>
<td>Invitrogen</td>
<td>Cat#:R1213</td>
</tr>
<tr>
<td>LIVE/DEAD Viability/Cytotoxicity Kit</td>
<td>Invitrogen</td>
<td>L3224</td>
</tr>
<tr>
<td>Click-iT EdU Cell Proliferation Kit</td>
<td>Invitrogen</td>
<td>C10337</td>
</tr>
<tr>
<td>PureLink™ Genomic DNA Mini Kit</td>
<td>Thermo Fisher Scientific</td>
<td>K182001</td>
</tr>
<tr>
<td colspan="3">Experimental models: Cell lines</td>
</tr>
<tr>
<td>Mouse Adult Hippocampal Neural Stem Cells</td>
<td><a href="#bib15">Morrow et al 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pSpCas9(BB)-2A-Puro (PX459) V2.0</td>
<td>Addgene</td>
<td>Cat#:62988</td>
</tr>
<tr>
<td>pSpCas9(vimentin)-2A-Puro (PX459) V2.0</td>
<td><a href="#bib15">Morrow et al 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td>pENTR-vimentin-mNeon (targeting vector backbone)</td>
<td><a href="#bib15">Morrow et al 2020</a></td>
<td>N/A</td>
</tr>
<tr>
<td>pENTR4-HaloTag w876-1</td>
<td>Addgene</td>
<td>Cat#:29644</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>See <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Mmc1.xlsx">Table S1</a>
</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>Benchling</td>
<td>Benchling.com</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec2.1">Part A-Design gene editing reagents</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Start by conceptually designing the fluorescent tag. The first decision regards where the fluorescent tag will be placed in the genome. Depending on the gene of interest, there may be many variants expressed, and a unique 5′ and 3′ terminus on each variant. Create a fluorescent tag that will minimally disrupt folding and function of the protein of interest as well as other variants encoded by the gene of interest. Then identify a suitable 1) crRNA to target the site of fluorescent tag knock-in, 2) fluorescent tag to label the protein of interest, 3) linker peptide to fuse the protein of interest to the selected fluorescent tag, and 4) design a targeting vector encoding a fluorescent tag which will tag the protein of interest.</p>
<h3 id="sec2.2">sgRNA design</h3>
<p>An sgRNA is made up of a crRNA, which is a 20 nucleotide sequence which corresponds to the region of the genome you wish to target, and a tracrRNA which scaffolds the sgRNA to SpCas9. The tracrRNA is constant, while the crRNA is changed based on the region of the genome being targeted. Importantly, immediately adjacent to the 3′ end of the crRNA should be an “NGG” (protospacer adjacent motif (PAM)) sequence which allows for cleavage by SpCas9 (<a href="#fig1">Figure 1</a>). Online tools examine the region of the genome you wish to target and look for 20 nucleotide sequences adjacent to NGG sites and then probe for relative off-target homology across the rest of the genome. Ideal crRNAs for tagging purposes will have minimal homology with the rest of the genome to decrease the chances of off-target genome cleavage by SpCas9 and will cut within 20–30 nucleotides of either the 5′ or 3′ end (depending on which end of the protein is being targeted for fluorescent tagging) of the gene encoding your protein of interest (<a href="#bib16">Paquet et al., 2016</a>). Here we will discuss how to identify optimal crRNA sequences using the online tool Benchling (<a href="http://benchling.com">benchling.com</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. sgRNA and targeting vector design</div>
<p>The region of the genome encoding vimentin’s carboxyl terminus was probed for PAM sites (brown) to identify optimal crRNAs around our site in which we aimed to place the fluorescent tag (green). Here we show an example crRNA (blue) which we used to place a coding sequence for mNeon on vimentin’s 3′ terminus in the mouse genome. The stop codon is listed in pink. The site of double stranded cleavage by SpCas9 is denoted by a purple dashed line. The site of the fluorescent tag knock-in is denoted by a gray dashed line. Yellow colored boxes denote <i>vimentin</i> exons, white boxes denote <i>vimentin</i> introns and dashed lines denote untranslated regions (UTRs). The targeting vector cartoon on the bottom depicts how to construct a targeting vector for knock-in of a linker peptide (orange) coding sequence and mNeon fluorescent tag (green) coding sequence on to vimentin’s 3′ terminus. Genomic DNA is abbreviated as gDNA.</p>
</figcaption>
</figure>
<ol>
<li>Using a web browser, go to <a href="http://benchling.com">benchling.com</a>, create an account and then start a new project by clicking on the plus sign on the left-side bar.</li>
<li>After creating a project, click on the plus sign on the left-side bar, move over “CRISPR” in the menu and then select “CRISPR Guides.”</li>
<li>Input the sequence you wish to target by either providing coordinates, gene name, raw sequence, or a sequence file. Then hit “Next.”</li>
<li>Select “Single guide,” guide length: 20, the correct genome species for your cell type and PAM: NGG and then click “Finish.”</li>
<li>Below the middle of the screen where your sequence of interest is displayed, select the specific region of DNA you wish to target by either typing in the start and end position in the panel on the right or clicking and dragging your mouse over the DNA in the middle panel of Benchling. Then click the green “+” sign in the panel on the right. Now you should see a list of potential crRNA sequences.</li>
<li>Select crRNA(s) that are the closest to either the 5′ or 3′ end of your gene of interest (depending on which end of the protein you are tagging with a fluorescent tag) with the highest “On-Target Score” (on-target scores above 60 are considered good cut sites). Choosing a crRNA that is closest to your site of fluorescent tag insertion will increase chances of success. In general, avoid using a sgRNA that cuts further than 30 nucleotides from your site of fluorescent tag knock-in (<a href="#bib16">Paquet et al., 2016</a>). For example, to target the 3′ terminus of vimentin we use the following crRNA: GAATAAAAATTGCACACACT.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Using a sgRNA cut site which will be disrupted once a fluorescent tag has been successfully achieved (knock-in of the fluorescent tag occurs within the crRNA target site) simplifies creation of the targeting vector by eliminating a need to mutate the PAM site. Mutating the PAM site prevents cutting of your targeting vector and/or recutting by SpCas9 following successful incorporation of your fluorescent tag sequence in to the genome.</div>
<h3 id="sec2.3">Fluorescent tag selection</h3>
<ol start="7">
<li>There are many suitable fluorescent tags encoded by nucleotides with a combination of strengths and weaknesses (<a href="#bib20">Shaner et al., 2005</a>). When selecting a fluorescent tag to label your protein of interest, consider the following:
<ol type="a">
<li>Excitation/emission of your fluorescent tag – Choose a fluorescent tag that you can easily image with equipment accessible to your lab and that is compatible with other assays you plan to perform once you have generated cells with your desired fluorescent tag.</li>
<li>Stability and intensity of the fluorescent tag – Choose a fluorescent tag that will not quickly photobleach and that is relatively bright.</li>
<li>Toxicity of the fluorescent tag – Some fluorescent tags, such as mCherry and DsRed are known to aggregate, and thus using these fluorescent tags as your tag may be undesirable (<a href="#bib2">Bell et al., 2019</a>). Choose a fluorescent tag with minimal toxicity.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For a general overview of candidate fluorescent tags encoded by nucleotides, visit <a href="https://www.addgene.org/fluorescent-proteins/">https://www.addgene.org/fluorescent-proteins/</a>.</div>
<div class="note">
<span class="note-title">Note:</span> We recommend using either mNeon as a “green” fluorescent tag or mScarlet as a “red” fluorescent tag to label your protein of interest as these are relatively brighter fluorescent tags with increased photostability and decreased aggregation propensity compared with other potential fluorescent tags.</div>
<h3 id="sec2.4">Linker design</h3>
<ol start="8">
<li>It is often desirable to have a linker peptide that will connect your fluorescent tag to your protein of interest, as opposed to directly attaching the fluorescent tag to your protein of interest. This increases the chances that the fluorescent tag and endogenous protein of interest will properly fold and minimally interfere with the function and localization of your endogenous protein of interest. Linker peptide sequences are normally between 2–20 amino acids long and typically consist of amino acids such as glycine and serine to provide flexibility (<a href="#bib21">Snapp, 2005</a>). For example, in our tagging of the 3′ terminus of mouse vimentin with mNeon, we used a linker peptide with the following composition: Amino Acids – GGGGSGGGGSGGGGS; Nucleotides – GGTGGTGGCGGTTCAGGCGGAGGTGGCTCTGGCGGTGGCGGATCG</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The following are alternative options for flexible linker peptides (<a href="#bib19">Sabourin et al., 2007</a>, <a href="#bib10">Huston et al., 1988</a>, <a href="#bib4">Chen et al., 2013</a>):</div>
<ul>
<li>Amino Acids – GGGGGGGG; Nucleotides – GGTGGGGTGGTGGTGGTGGTGGT</li>
<li>Amino Acids – GGGGSGGGGSGGGGS; Nucleotides – GGTGGTGGTGGTAGTGGTGGTGGTGGTAGTGGTGGTGGTGGTAGT</li>
</ul>
<h3 id="sec2.5">Targeting vector design</h3>
<ol start="9">
<li>The targeting vector is a DNA sequence containing the sequence you wish to place into the genome flanked by DNA sequences homologous to the loci in the genome your fluorescent tag will be placed within (<a href="#fig1">Figures 1</a> and <a href="#fig3">3</a>). To design the targeting vector, arrange the linker peptide and fluorescent tag DNA sequences according to whether they will be placed on the amino or carboxyl terminus of your protein of interest and then add homologous sequences of DNA on either side of the DNA sequence you wish to place in the genome (<a href="#fig1">Figure 1</a>). We recommend using homology arms that are at least ~500 nucleotides long.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Although larger homology arms generally yield higher efficiency reactions, homology arms as short as 50–100 nucleotides have been shown to be sufficient to successfully tag epitopes in the genome with fluorescent tags (<a href="#bib12">Li et al., 2014</a>, <a href="#bib6">Dewari et al., 2018</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> If your fluorescent tag is placed on the amino terminus of your protein of interest, be sure to remove the stop codon from the fluorescent tag so that translation does not stop after your fluorescent tag is produced. Conversely, if your fluorescent tag is placed on the carboxyl terminus of your protein of interest, place your fluorescent tag upstream of the stop codon of the endogenous gene interest so that translation does not stop prior to translation of your fluorescent tag. In this situation, be sure to place the stop codon in the repair template to make sure translation ends correctly/in frame.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Prioritizing sgRNA cut sites that are outside the coding sequence (untranslated regions) can reduce the chances of creating mutations through non-homologous end joining (NHEJ) in the non-tagged allele of your protein of interest (<a href="#bib3">Bressan et al., 2017</a>, <a href="#bib14">Merkle et al., 2015</a>).</div>
<h3 id="sec2.6">Part B-Synthesize the SpCas9:sgRNA dual plasmid and the targeting vector plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 week</div>
<p>Here we describe how to generate the reagents required to create successful fluorescent tags in genomic DNA of primary adult mouse NSCs using CRISPR/Cas9. First we will discuss how to clone a crRNA sequence into a SpCas9:sgRNA dual plasmid, which already includes the constant tracrRNA. Second, we will discuss how to assemble the targeting vector in a separate plasmid.</p>
<h3 id="sec2.7">Clone the crRNA into the SpCas9:sgRNA dual plasmid</h3>
<p>This section of the protocol describes how to generate a SpCas9:sgRNA dual plasmid which includes the crRNA chosen in Part A through restriction digestion/ligation cloning (adapted from (<a href="#bib18">Ran et al., 2013</a>)). SpCas9 and the tracrRNA are already present in the SpCas9:sgRNA dual plasmid backbone, and thus, only the crRNA needs to be inserted. As an example, in this protocol we will describe how to make a plasmid encoding a sgRNA targeting the DNA sequence encoding the 3′ terminus of vimentin in the mouse genome (Addgene #62988); however, this protocol should work for any sgRNA. In brief, a template plasmid harboring an empty crRNA cloning site and the SpCas9 sequence will be cut and a crRNA insert will be prepared and ligated into the empty crRNA cloning site.</p>
<ol start="10">
<li>Order DNA oligonucleotides according to <a href="#fig3">Figure 3</a> by adding the indicated nucleotides in red to the forward or reverse complement of the crRNA DNA sequence respectively (<a href="#fig2">Figure 2</a>, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Mmc1.xlsx">Table S1</a>). These oligonucleotides will be referred to as “crRNA F oligonucleotide” or “crRNA R oligonucleotide” in this protocol.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. crRNA oligonucleotide (oligo) design for cloning the crRNA (blue) into the SpCas9:sgRNA dual plasmid</div>
<p>Order oligonucleotides with your crRNA sequence and the attached nucleotides (red) to create sticky ends which will ligate into the linearized SpCas9:sgRNA dual plasmid (<a href="#bib18">Ran et al., 2013</a>). Note that the PAM site (brown) is removed. Genomic DNA is abbreviated as gDNA.</p>
</figcaption>
</figure>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Primer design for cloning the targeting vector using a Gibson Assembly</div>
<p>(A) Primers are used to amplify each fragment with additional DNA sequences homologous to neighboring fragments of the targeting vector.</p>
<p>(B) Amplified fragments with homology to neighboring fragments are ligated by Gibson Assembly to create the targeting vector plasmid.</p>
</figcaption>
</figure>
<ol type="a">
<li>The nucleotide residues in red will serve as sticky ends which will allow for ligation of the crRNA insert into the SpCas9:sgRNA dual plasmid at the cut BbsI site.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Addition of a “G” at the 5′ of the crRNA is only necessary if the crRNA does not already start with a “G” residue.</div>
<ol start="11">
<li>Create crRNA insert
<ol type="a">
<li>Prepare a reaction with the following components to anneal and phosphorylate the oligos for insertion into the SpCas9:sgRNA dual plasmid.
<table id="undtbl1">
<caption>crRNA Insert Synthesis</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>crRNA F oligonucleotide (100 <b>μ</b>M)</td>
<td>1 μL</td>
</tr>
<tr>
<td>crRNA R oligonucleotide (100 <b>μ</b>M)</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 Ligase Buffer (10X)</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 PNK</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>6 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Run the reaction in a thermal cycler with the following conditions: 37°C for 30 min, 95°C for 5 min, decrease temperature from 95°C to 25°C at a declining rate of 5°C per minute.</li>
</ol>
</li>
</ol>
<ol start="12">
<li>Restriction digest of the SpCas9:sgRNA dual plasmid (pSpCas9(BB)-2A-Puro (PX459) V2.0)
<ol type="a">
<li>Prepare a restriction digest reaction with the following components to linearize the plasmid and dephosphorylate the sticky ends to prevent self-ligation.
<table id="undtbl2">
<caption>Restriction digest to linearize the SpCas9:sgRNA dual plasmid backbone</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>pSpCas9(BB)-2A-Puro (PX459) V2.0</td>
<td>1 μg</td>
</tr>
<tr>
<td>NEBuffer 2.1 (10X)</td>
<td>5 μL</td>
</tr>
<tr>
<td>BbsI</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>x μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Digest for 4 h at 37°C.</li>
<li>After 4 h, add 1 μL shrimp alkaline phosphatase and incubate at 37°C for 1 additional hour.</li>
<li>Gel purify the linearized plasmid (Qiagen 28704; <a href="https://www.qiagen.com/us/products/catalog/sample-technologies/dna-sample-technologies/dna-cleanup/qiaquick-gel-extraction-kit/?cmpid=QVenSPP1404quartzySC">https://www.qiagen.com/us/products/catalog/sample-technologies/dna-sample-technologies/dna-cleanup/qiaquick-gel-extraction-kit/?cmpid=QVenSPP1404quartzySC</a>). For pSpCas9(BB)-2A-Puro (PX459) V2.0 the linearized plasmid should be ~9.1 kB.
<div class="note">
<span class="note-title">Note:</span> Running a “no restriction enzyme” control, where enzymes are not present in the reaction, can be helpful to ensure the enzyme is cutting properly. When linearized, DNA will run true to size, whereas non-linearized plasmid DNA can run faster or slower based on its supercoiled state.</div>
</li>
<li>Ligation of crRNA insert and linearized SpCas9:sgRNA dual plasmid</li>
<li>Prepare a ligation reaction with the following components.
<table id="undtbl3">
<caption>Ligation reaction for inserting the cRNA insert into the linearized <b>SpCas9</b>:sgRNA dual plasmid</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Gel purified linearized plasmid</td>
<td>10 ng</td>
</tr>
<tr>
<td>gRNA insert (Diluted 1:200 in ddH<sub>2</sub>0)</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 DNA ligase buffer (10X)</td>
<td>1 μL</td>
</tr>
<tr>
<td>T4 DNA ligase</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O to 10 μL</td>
<td>x μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Incubate for 30 min at room temperature (25°C).
<div class="note">
<span class="note-title">Note:</span> Running a “no insert” control where all components are added to the ligation reaction aside from the crRNA insert will inform on the efficiency of your cloning after the transformation is complete by allowing you to visualize the fraction of colonies in your reaction that are religated or empty backbones.</div>
</li>
</ol>
</li>
</ol>
<ol start="13">
<li>Transformation of ligated SpCas9:sgRNA dual plasmid
<ol type="a">
<li>Thaw 50 μL DH5α on ice for 20 min.</li>
<li>Add 3 μL of the ligation reaction to the DH5α on ice and then let the samples sit on ice for 20 min.</li>
<li>Heat shock samples for 30 s at 42°C.</li>
<li>Immediately following heat shock, place samples back on ice for 2 min and add 250 μL SOC recovery media.</li>
<li>Shake at 200 rotations per minute (rpm) at 37°C for 1 h in a bacterial shaker.</li>
<li>Plate 250 μL transformed bacteria in SOC on to a LB agar plate with 50 μg/mL ampicillin and grow at 37°C for 18 h.</li>
</ol>
</li>
<li>Screen clones by performing minipreps on clones (Invitrogen R1213) and then performing restriction digests with BbsI. The BbsI restriction digest site should be lost upon insertion of the DNA sequence of the crRNA.</li>
<li>Analyze clones that are not cut with BbsI using Sanger Sequencing to validate insertion of the DNA sequence of the crRNA using the following sequencing primer: GAGGGCCTATTTCCCATGATT.</li>
<li>Prepare plasmid DNA for delivery to NSCs by performing an endotoxin free maxi plasmid purification.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Performing a DNA preparation with an endotoxin removal step (e.g., Qiagen 12362) is essential for viability of electroporated cells.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Many plasmids have been generated for delivery of SpCas9 and an sgRNA to cells, all with subtly different utilities. For example, some plasmids are viral backbones allowing for delivery of SpCas9 and the sgRNA through generation of viral particles (Addgene 98290). Other plasmids harbor different reporters such as fluorescent reporters (GFP; Addgene 48138) or antibiotic resistance reporters (Puromycin; Addgene 62988).</div>
<h3 id="sec2.8">Clone the targeting vector plasmid</h3>
<p>This section of the protocol describes how to generate a plasmid encoding the desired fluorescent tag and a linker peptide flanked by homologous regions of the genome upstream or downstream of the fluorescent tag knock-in site through a Gibson Assembly reaction (<a href="#bib8">Gibson, 2011</a>, <a href="#bib9">Gibson et al., 2009</a>). For example, to create a targeting vector plasmid for labeling vimentin with mNeon on its 3′ terminus, a Gibson Assembly reaction will be performed to create a plasmid which includes the following sequence: 5′ homology region + linker + mNeon + 3′ homology region (<a href="#fig1">Figures 1</a> and <a href="#fig3">3</a>).</p>
<p>Gibson Assembly allows for the simultaneous ligation of multiple fragments of DNA that are over 200 nucleotides long which have at least 15–30 nucleotides of homology with each other on the ends of each adjacent fragment (<a href="#fig3">Figure 3</a>). First, PCRs are performed to create fragments with overlapping homology to adjacent fragments (<a href="#fig3">Figure 3</a>A). Second, fragments with overlapping homology to adjacent fragments are ligated into a linearized plasmid to assembly the targeting vector plasmid (<a href="#fig3">Figure 3</a>B).</p>
<p>As the linker peptide coding sequence is often a short fragment less than 200 nucleotides, it is optimal to attach the linker peptide coding sequence to an adjacent fragment, such as the fluorescent tag, (<a href="#fig3">Figure 3</a>A PCR 1-2) prior to Gibson Assembly to increase the efficiency of the Gibson Assembly.</p>
<ol start="17">
<li>Obtain coding sequences for all components of the designed targeting vector plasmid.
<ol type="a">
<li>Coding sequences for fluorescent tags and many linker peptides can be obtained commercially from vendors such as Addgene or generated through commercial DNA synthesis services.</li>
<li>Homology arms can be generated through PCR using genomic DNA as a template or through commercial DNA synthesis services.</li>
</ol>
</li>
<li>Gibson Assembly Overlapping Primer Design
<ol type="a">
<li>Design primers with a melting temperature greater than 55°C and less than 70°C that will amplify each DNA fragment and add a 15–30 nucleotide overhang sequence that is homologous to each adjacent DNA fragments in the theoretically designed targeting vector (<a href="#fig3">Figure 3</a>).
<ol type="i">
<li>For example, the forward primer used to amplify the 3′ homology arm should contain a 15–30 nucleotide overhang homologous to mNeon and the reverse primer should contain a 15–30 nucleotide overhang homologous to pENTR (<a href="#fig3">Figure 3</a>).</li>
</ol>
</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For general recommendations on primer design visit the NEBuilder Gibson Assembly Tool (<a href="http://www.NEBGibson.com">www.NEBGibson.com</a>). In general, higher overlap between fragments will yield a higher efficiency during the Gibson Assembly reaction.</div>
<ol start="19">
<li>Create fragment with linker peptide attached to the fluorescent tag through assembly PCR.
<ol type="a">
<li>Perform a PCR to create an amplicon of the fluorescent tag coding sequence with 15–30 nucleotides of homology to the linker peptide coding sequence (<a href="#fig3">Figure 3</a>A PCR 1).
<table id="undtbl4">
<caption>PCR Protocol for Amplification of DNA Fragments to be used in the Gibson Assembly</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Phusion HF Buffer (5X)</td>
<td>10 μL</td>
</tr>
<tr>
<td>dNTPs (10 mM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Template DNA</td>
<td>250 ng genomic DNA;50 ng plasmid DNA</td>
</tr>
<tr>
<td>Phusion DNA Polymerase</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>0 to 50 μL</td>
<td>X μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl5">
<caption>PCR cycling conditions</caption>
<thead>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>Tm+3°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>15 s per kB</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"><math> <mrow> <mi>∞</mi> </mrow> </math></td>
</tr>
</tbody>
</table>
</li>
<li>Perform an assembly PCR using reaction recipe and conditions listed in step 19 to ligate the fluorescent tag coding sequence with the linker peptide coding sequence and attach a 15–30 nucleotide overhang homologous to adjacent fragments in the targeting vector plasmid (<a href="#fig3">Figure 3</a>A PCR 2).</li>
</ol>
</li>
</ol>
<ol start="20">
<li>PCR amplification of DNA fragments
<ol type="a">
<li>Perform a PCR using the reaction recipe and conditions listed in step 19 to generate remaining fragments (e.g., 3′ and 5′ homology arms) for Gibson Assembly with a 15–30 nucleotide overhang homologous to adjacent fragments in the targeting vector plasmid (<a href="#fig3">Figure 3</a>A PCR 3-4).
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Fragments can also be purchased using DNA synthesis services commercially available.</div>
<div class="note">
<span class="note-title">Note:</span> If amplifying fragments from genomic DNA is challenging for your region of interest, adding 0.5 mM MgCl<sub>2</sub> and/or 3% dimethyl sulfoxide (DMSO) can increase the chances of successfully amplifying each fragment. Separately, you can also perform sequential PCRs by first amplifying your fragment without the 15–30 nucleotide DNA sequences homologous to adjacent fragments. Then, upon isolating your fragment from genomic DNA, a second PCR can be performed to attach adapter sequences for the Gibson Assembly reaction.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Other high-fidelity polymerases can be used instead of Phusion, such as Q5 High-Fidelity DNA Polymerase (NEB E0555S).</div>
</li>
<li>Following completion of the reaction, gel purify (Qiagen 28704) each DNA fragment.</li>
</ol>
</li>
</ol>
<ol start="21">
<li>Restriction digest of targeting vector
<ol type="a">
<li>Prepare a restriction digest with the following recipe to linearize the targeting vector backbone plasmid.
<table id="undtbl6">
<caption>Representative restriction digest for targeting vector plasmid</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>pENTR4-HaloTag w876-1</td>
<td>1 μg</td>
</tr>
<tr>
<td>NEBuffer 1.1 (10X)</td>
<td>5 μL</td>
</tr>
<tr>
<td>KpnI</td>
<td>1 μL</td>
</tr>
<tr>
<td>NcoI</td>
<td>1 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O</td>
<td>x μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Incubate the reaction at 37°C for 1 h.</li>
<li>Gel purify the linearized plasmid. For pENTR4-HaloTag w876-1 the linearized plasmid should be ~2.3 kB long with a ~1 kB fragment cut out of the original plasmid.
<div class="note">
<span class="note-title">Note:</span> Here we use two restriction enzymes with non-complementary cut sites. This decreases the chances of the linearized targeting vector plasmid backbone self-ligating, which reduce efficiency of the Gibson Assembly.</div>
<div class="note">
<span class="note-title">Note:</span> The targeting vector backbone plasmid as we describe is used only to amplify the targeting vector, and thus many plasmids could be suitable targeting vector plasmids. In general, smaller vectors will yield higher HDR efficiencies and targeting vector plasmids should not encode other transcripts. Other suitable targeting vector backbones include but are not limited to: (Addgene 12257 and 10791).</div>
</li>
</ol>
</li>
</ol>
<ol start="22">
<li>Once your DNA inserts (e.g., vimentin 5′ homology region + peptide linker + mNeon + 3′ homology region) and digested targeting vector plasmid are prepared using the previous steps, set up the following Gibson Assembly reaction on ice:
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DNA Fragments</td>
<td>For a 2-3 fragment assembly use 0.5 picomoles of each fragment/linearized targeting vector; for 4–6 fragment assembly use 1 picomole of each fragment/linearized targeting vector</td>
</tr>
<tr>
<td>2X NEB Gibson Assembly Master Mix</td>
<td>10 μL</td>
</tr>
<tr>
<td>Nuclease-free water to 20 μL</td>
<td>X μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>20 μL</b></td>
</tr>
</tbody>
</table>
<ol type="a">
<li>Incubate the samples at 50°C for 60 min. Following incubation, store samples on ice for same day transformation, or at −20 for subsequent transformation.</li>
</ol>
</li>
</ol>
<ol start="23">
<li>Transform Gibson Assembly reaction into competent cells
<ol type="a">
<li>Thaw 50 μL DH5α on ice for 20 min.</li>
<li>Add 2 μL of the ligation reaction to the DH5α on ice and then let the samples sit on ice for 20 min.</li>
<li>Heat shock samples for 30 s at 42°C.</li>
<li>Immediately following heat shock, place samples back on ice and add 250 μL SOC recovery media.</li>
<li>Shake at 200 rotations per minute at 37°C for 1 h in a bacterial shaker.</li>
<li>Plate 100 μL sample on to a LB agar plate with ampicillin and grow at 37°C for 18 h.</li>
</ol>
</li>
<li>Screen clones for assembly of your targeting vector either by PCR colony test, restriction digest or by Sanger sequencing using a sequencing primer ~100 nucleotides upstream of the targeting vector.</li>
<li>Isolate plasmid DNA for delivery to NSCs by performing an endotoxin free maxi plasmid purification.</li>
</ol>
<h3 id="sec2.9">Part C- transfect NSCs with the targeting vector plasmid and the SpCas9:sgRNA dual plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>NSCs will be electroporated with both the SpCas9:sgRNA dual plasmid and the assembled targeting vector plasmid and then cultured for 3–10 days to achieve genetic knock-in of the fluorescent tag coding sequence into the genome and expression of the desired endogenously tagged gene. To electroporate NSCs, we recommend using the Amaxa Nucleofector system.</p>
<ol start="26">
<li>Trypsinize proliferating NSCs in culture (<a href="#bib15">Morrow et al., 2020</a>, <a href="#bib1">Ahmed et al., 2021</a>, <a href="#bib17">Peltier et al., 2010</a>) to generate a single-cell suspension:
<ol type="a">
<li>Use a pipette to detach any adherent cells off of the dish and then create a pellet by centrifugation at 120 × <i>g</i> for 4 min.</li>
<li>Resuspend pellet in 100 μL 0.05% trypsin (Invitrogen 25300-054) in Versene (Thermo Fisher 15040066) and incubate at 37°C for 5 min.</li>
<li>Quench the trypsinization with 200 μL volume of trypsin inhibitor (Sigma T6522) and then mechanically dissociate the sample with a pipette by pipetting 15–20 times vigorously.</li>
<li>Further quench the trypsinization by adding 3 mL of NSC media DMEM/F12 GlutaMax (Invitrogen 10565018) with penicillin-streptomycin-fungizone (1:100, Invitrogen 15140122), B27 (1:50, Invitrogen 17504044), and 20 ng/mL FGF-2 and EGF (PeproTech 100-18B and AF-100-15).</li>
<li>Remove the trypsin by pelleting the sample through centrifugation at 120 × <i>g</i> for 4 min and aspirating the supernatant.</li>
</ol>
</li>
<li>Count cells using a hemocytometer or cell counter and create a pellet of 4 million NSCs through centrifugation at 120 × <i>g</i> for 4 min.</li>
<li>Resuspend pellet by adding 100 μL Mouse Neural Stem Cell Nucleofector Solution and 3.5 μg SpCas9:sgRNA dual plasmid and 3.5 μg targeting vector plasmid, by pipetting gently up and down 3–5 times.</li>
<li>Immediately transfer resuspension to an electroporation cuvette, close the cuvette with the cap and then electroporate using program A-033 on a Nucleofector device.</li>
<li>Immediately following electroporation, quench the sample by adding 1 mL prewarmed NSC media to the cuvette.</li>
<li>Use a plastic disposable pipette to transfer the electroporated cell solutions, avoiding the “foam-like” cellular debris, to a 150 mm diameter uncoated culture dish (150 cm<sup>2</sup> area) with 10 mL of NSC media and as quickly as possible return cells to the incubator.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Moving as fast as possible post-electroporation will ensure the highest NSC viability.</div>
<div class="note">
<span class="note-title">Note:</span> It is expected to see significant cell death (~50%+ of cells) immediately after electroporation due to the electroporation itself. Using less NSCs will increase the amount of cell death during electroporation. Finally, avoidance of collection of the foam-like cellular debris following electroporation will improve longer-term survival.</div>
<div class="optional">
<span class="optional-title">Optional:</span> 24 hours following electroporation, trypsinize NSCs and replate them into a 10 cm dish to remove dead cells and debris. This step additionally can increase survival of the culture.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If you are unable to achieve viability after electroporation sufficient to isolate successfully edited cells, you can further increase survival by plating NSCs onto Poly-L-Ornithine (PLO)- and laminin-coated culture dishes, such as Fisher Scientific 12-565-163 (coating as previously described: incubate with PLO for 1 hour at 37°C (PLO; 10 μg/mL in ddH<sub>2</sub>O) and with laminin for 3 hours at 37°C (5 μg/mL in PBS)) (<a href="#bib15">Morrow et al., 2020</a>). Once electroporated NSCs are plated onto coated wells, they can be washed with warm media to remove cellular debris and dead cells ~4 hours following plating, limiting the requirement for trypsinization.</div>
<h3 id="sec2.10">Part D- isolation of fluorescently tagged NSCs</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 month</div>
<p>Electroporated NSCs from the previous section of the protocol will be single cell-sorted and expanded clonally over the course of roughly 2–4 weeks and then screened for zygosity of the fluorescent tag knock-in by PCR on genomic DNA isolated from clones.</p>
<ol start="32">
<li>24–72 h following electroporation, use a fluorescent microscope to visualize electroporated samples for knock-in of the desired fluorescent tag (<a href="#fig4">Figure 4</a>A).
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative data for tagging the carboxyl terminus of vimentin with mNeon</div>
<p>(A and B) NSCs were electroporated either with or without plasmids for a CRISPR/Cas9 mediated tagging (green) of vimentin’s carboxyl terminus with mNeon and A) imaged with a fluorescent microscope and B) analyzed with a flow cytometer. Panel A shows a fixed NSC with the nucleus labeled with Hoechst (blue). Panel B shows mNeon fluorescence and forward scatter-area (FSC-A) of single cells.</p>
<p>(C) Following expansion of clones expressing mNeon, genomic DNA was extracted from two positive clones and analyzed for either a homozygous (homo) or heterozygous (het) fluorescent tag by PCR. Genomic DNA is abbreviated as gDNA. Scale bar, 10 μm.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If attempting to place a peptide that is not a fluorescent tag into the genome, you can alternatively screen clones by performing a PCR and/or Sanger Sequencing.</div>
<div class="note">
<span class="note-title">Note:</span> Depending on the synthesis and degradation rate of your protein of interest, the time it takes for the fluorescent tag to be placed into the genome and the fluorescent reporter to become detectable in live cells with a fluorescent microscope can be variable. As a general rule of thumb, you should be able to detect fluorescence within 72 hours following electroporation for most proteins that are expressed in your cell type.</div>
<div class="note">
<span class="note-title">Note:</span> If your fluorescently tagged protein is highly expressed, you may be able to detect fluorescence in live cells on plastic culture dishes. However, if your protein of interest is not as highly expressed, you may not be able to detect successful fluorescent tags using a fluorescent microscope on plastic culture dishes in live cells due to a suboptimal refractive index of plastic. To increase the likelihood of detecting correctly tagged cells, you can alternatively plate NSCs onto coated glassware or plastic with a refractive index optimized for your fluorescent microscope’s objective lenses. Further, fixing cells and removing autofluorescent compounds in the media can additionally increase clarity of signal. Lastly, you can also run trypsinized samples through a flow cytometer to probe for dimmer cells harboring fluorescent tags.</div>
<ol start="33">
<li>After visualizing expression of your fluorescent tag, use flow cytometry to sort single cells with the fluorescent tag.
<ol type="a">
<li>On the day of cell sorting, precoat a 96-well plate with PLO for 1 h at 37°C (PLO; 10 μg/mL in ddH<sub>2</sub>O) followed by laminin for 3 h at 37°C (5 μg/mL in PBS).</li>
<li>Trypsinize cells and then sort single cells using a fluorescence activated cell sorter directly into a 96 well plate preloaded with media (<a href="#fig4">Figure 4</a>B). We recommend resuspending samples in prechilled dPBS with 1 mM ethylenediaminetetraacetic acid (EDTA) and 1 μg/mL DNAse (Sigma-Aldrich D5025-15KU) and keeping samples on ice until completion of cell sorting. We recommend using a larger nozzle for cell sorting (~130 μm).</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Immediately following cell sorting, spin down the plate in a centrifuge at 120 × <i>g</i> for 4 minutes to increase the chances cells will stick to the bottom of the dish and survive.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> If you do not need to isolate a pure population of heterozygous or homozygous knock-ins, you can bulk sort all positive fluorescent cells into one well. This substantially expedites the timeline.</div>
<div class="note">
<span class="note-title">Note:</span> Following single cell sorting, it takes ~25–35 days for a single young adult mouse hippocampal NSC clone to expand to fill one well of a 96 well plate. Typically only ~30% of single-cell sorted clones are able to successfully expand. During expansion, change media at least once every 72 hours (<a href="#bib15">Morrow et al., 2020</a>).</div>
<ol start="34">
<li>After expanding clones with fluorescent tags, clones should be screened to identify heterozygous and homozygous fluorescent tag knock-in. We recommend generating pellets of at least 10,000 cells to extract genomic DNA for a PCR. To do this, during the first split, take half of the cells to create a pellet for sequencing and use the other half of the cells to continue expanding the clone.</li>
<li>After creating cell pellets from each clone, extract genomic DNA using a commercially available kit (e.g., Thermo Fisher Scientific K182001) and set up a PCR with the following parameters using primers that flank the site in which your fluorescent tag is placed, but which begin amplification outside of the homology arms (<a href="#fig4">Figure 4</a>C, <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Mmc1.xlsx">Table S1</a>).</li>
</ol>
<table id="undtbl8">
<caption>PCR to screen for heterozygous and homozygous fluorescent tag knock-ins</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Genomic DNA</td>
<td>250 ng</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>dNTPs (10 mM)</td>
<td>1 μL</td>
</tr>
<tr>
<td>GC Buffer (5X)</td>
<td>10 μL</td>
</tr>
<tr>
<td>DMSO</td>
<td>1.5 μL</td>
</tr>
<tr>
<td>Phusion DNA Polymerase</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>ddH<sub>2</sub>O to 50 μL</td>
<td>x μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>Tm+3°C</td>
<td>20 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>15 s per kB</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>10 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2"><math> <mrow> <mi>∞</mi> </mrow> </math></td>
</tr>
</tbody>
</table>
<ol start="36">
<li>Run the PCR reaction on an agarose gel to visualize PCR products and determine which clones are heterozygotes or homozygotes by probing for the presence of bands indicating alleles that are either harboring (larger PCR product) or lacking (smaller PCR product) your desired fluorescent tag (<a href="#fig4">Figure 4</a>C).</li>
<li>Perform Sanger Sequencing to validate that the fluorescent tag has been successfully placed in the genome without any mutations to the fluorescent tag, linker peptide or endogenous protein of interest.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> PCR reaction conditions may need to be optimized depending on the targeted gene.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>This protocol describes how to tag endogenous proteins in primary adult mouse NSCs with fluorescent tags to visualize their subcellular localization and regulation. Though efficiency of the reaction can vary based on many components of the reaction design, in our experiment where we placed coding sequences for a linker peptide and the fluorescent tag mNeon on the 3′ terminus of mouse vimentin (~750 nucleotide insert size), the efficiency of knock-in was ~1 in 200–500 cells prior to FACS enrichment (with ~100% being mNeon+ after FACS enrichment and ~20% of mNeon+ cells after FACS enrichment being homozygously tagged cells).</p>
<p>Further, upon generation of fluorescent tags, each clone should be validated to ensure tagging of the protein of interest has not significantly altered cellular behavior. We recommend performing the following control experiments: 1) Measure protein expression with a western blot to assess the effects of the fluorescent tag on expression of the protein of interest; 2) Immunostain for your protein of interest to confirm whether the fluorescent tag has changed the localization of your protein of interest; 3) Measure general cell health through analyzing proliferation rate (Thermo C10337) and viability (Thermo L3224).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>There are a variety of factors that will contribute to the chances of successfully labeling an endogenous protein of interest with a fluorescent tag. sgRNAs require a PAM site and thus where you can cut the genome with SpCas9 is limited by the PAM sites present at your region of interest. Labeling proteins endogenously with fluorescent tags can also be toxic if the fluorescent tag causes the protein to misfold or dysfunction. In the event that your protein of interest is lowly expressed, it is also possible that you could create successful fluorescent tags but fail to see fluorescence due to low expression of your protein of interest. Creating fluorescent tag knock-in reagents and applying them to cells is also time consuming and thus limits throughput.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec5.1">Problem 1</h3>
<p>Failing to clone the SpCas9:sgRNA dual plasmid (Part B, step 15)</p>
<h3 id="sec5.2">Potential solution 1</h3>
<p>Confirm linearization of the SpCas9:sgRNA dual plasmid</p>
<p>Running a no restriction enzyme control can ensure that the plasmid is linearizing to receive the crRNA insert.</p>
<h3 id="sec5.3">Potential solution 2</h3>
<p>Use a commercial manufacturer to generate the SpCas9:sgRNA dual plasmid</p>
<p>Vectors for expressing sgRNAs and SpCas9 can be commercially generated. (e.g., <a href="https://en.vectorbuilder.com/resources/vector-system/pRP_gRNA_parent.html">https://en.vectorbuilder.com/resources/vector-system/pRP_gRNA_parent.html</a>)</p>
<h3 id="sec5.4">Problem 2</h3>
<p>Failing to clone the targeting vector plasmid (Part B, step 24)</p>
<h3 id="sec5.5">Potential solution 1</h3>
<p>Confirm linearization of the targeting vector plasmid backbone.</p>
<p>Running a no restriction enzyme control can ensure that the plasmid is linearizing to receive the targeting vector when performing the Gibson Assembly.</p>
<h3 id="sec5.6">Potential solution 2</h3>
<p>Perform additional assembly PCRs prior to the Gibson Assembly to lower the number of fragments in the Gibson Assembly.</p>
<p>Reducing the number of fragments that need to be ligated in the Gibson Assembly will increase the efficiency of the Gibson Assembly.</p>
<h3 id="sec5.7">Potential solution 3</h3>
<p>Use a DNA synthesis service to generate the targeting vector as a linear double stranded oligonucleotide</p>
<p>Targeting vectors can also be generated through commercial DNA synthesis services. (e.g., <a href="https://www.genewiz.com/en/Public/Services/Gene-Synthesis">https://www.genewiz.com/en/Public/Services/Gene-Synthesis</a>)</p>
<h3 id="sec5.8">Problem 3</h3>
<p>Low tagging efficiency (Part C, step 33)</p>
<h3 id="sec5.9">Potential solution 1</h3>
<p>Using a different crRNA</p>
<p>Some crRNAs work better than others for a myriad of reasons. If your efficiency of fluorescent tag knock-in into the genome is poor, you may benefit from trying alternative crRNAs around your desired site of fluorescent tag knock-in.</p>
<h3 id="sec5.10">Potential solution 2</h3>
<p>Increasing length of the homology arms</p>
<p>Increasing the length of the homology arms in the targeting vector can increase the efficiency of recombination following double stranded DNA cleavage by SpCas9.</p>
<h3 id="sec5.11">Potential solution 3</h3>
<p>Selecting with puromycin</p>
<p>Performing a puromycin selection can eliminate cells that did not uptake the SpCas9:sgRNA dual plasmid and thereby increase efficiency of the reaction.</p>
<h3 id="sec5.12">Potential solution 4</h3>
<p>Decreasing toxicity associated with your inserting a fluorescent tag on your protein of interest</p>
<p>Depending on the design of your fluorescent tag and the function of your protein of interest, some fluorescent tags may have toxicity associated with loss- or gain-of-function phenotypes resulting from your designed fluorescent tag. If your fluorescent tag is toxic, you could potentially reduce toxicity through altering the design of the fluorescent tag by changing parameters, such as which end of the polypeptide the fluorescent tag is added and the flexibility and length of the linker peptide connecting the fluorescent tag to your protein of interest.</p>
<h3 id="sec5.13">Potential solution 5</h3>
<p>Validate the fluorescent tag sequence in genomic DNA</p>
<p>It is possible that either due to an error in template design, creation, or a stochastic in the specific clone you are analyzing, the fluorescent tag could be incorrectly placed into the genome in such a way that it is no longer in frame. Validating that the targeting vector is the correct sequence and/or validating the specific sequence placed into the genome can provide insight into whether low efficiency of the reaction is tied to an error caused by the fluorescent tag being out of the correct open reading frame.</p>
<h3 id="sec5.14">Problem 4</h3>
<p>Only generating heterozygous fluorescent tags (Part D, step 36)</p>
<h3 id="sec5.15">Potential solution</h3>
<p>Sequential treatment</p>
<p>If you are only able to generate heterozygous knock-ins and are interested in examining homozygous knock-ins, you could try first generating a pure population of heterozygous knock-ins and then repeating the reaction a second time on the heterozygous knock-ins to generate homozygous knock-ins.</p>
<h3 id="sec5.16">Problem 5</h3>
<p>Fluorescent tag knock-in is toxic (Part C, step 33)</p>
<h3 id="sec5.17">Potential solution</h3>
<p>Alter the design of the fluorescent tag</p>
<p>If the fluorescent tag knock-in is not viable, you could try changing elements of the reaction design, such as changing which fluorescent tag is used to label the protein of interest, or changing which terminus of the protein the fluorescent tag is fused to.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Ahmed, A., Isaksen, T.J., and Yamashita, T. (2021). Protocol for mouse adult neural stem cell isolation and culture. STAR Protoc. <i>2</i>, 100522. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref1">View at publisher</a></p>
<p id="bib2">Bell, C.G., Lowe, R., Adams, P.D., Baccarelli, A.A., Beck, S., Bell, J.T., Christensen, B.C., Gladyshev, V.N., Heijmans, B.T., Horvath, S., et al. (2019). DNA methylation aging clocks: challenges and recommendations. Genome Biol. <i>20</i>, 249. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref2">View at publisher</a></p>
<p id="bib3">Bressan, R.B., Dewari, P.S., Kalantzaki, M., Gangoso, E., Matjusaitis, M., Garcia-Diaz, C., Blin, C., Grant, V., Bulstrode, H., Gogolok, S., et al. (2017). Efficient CRISPR/Cas9-assisted gene targeting enables rapid and precise genetic manipulation of mammalian neural stem cells. Development <i>144</i>, 635-648. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref3">View at publisher</a></p>
<p id="bib4">Chen, X., Zaro, J.L., and Shen, W.C. (2013). Fusion protein linkers: property, design and functionality. Adv. Drug Deliv. Rev. <i>65</i>, 1357-1369. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref4">View at publisher</a></p>
<p id="bib5">Chu, V.T., Weber, T., Wefers, B., Wurst, W., Sander, S., Rajewsky, K., and Kuhn, R. (2015). Increasing the efficiency of homology-directed repair for CRISPR-Cas9-induced precise gene editing in mammalian cells. Nat. Biotechnol. <i>33</i>, 543-548. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref5">View at publisher</a></p>
<p id="bib6">Dewari, P.S., Southgate, B., Mccarten, K., Monogarov, G., O'Duibhir, E., Quinn, N., Tyrer, A., Leitner, M.C., Plumb, C., Kalantzaki, M., et al. (2018). An efficient and scalable pipeline for epitope tagging in mammalian stem cells using Cas9 ribonucleoprotein. Elife <i>7</i>, e35069. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref6">View at publisher</a></p>
<p id="bib7">Doudna, J.A. and Charpentier, E. (2014). Genome editing. The new frontier of genome engineering with CRISPR-Cas9. Science <i>346</i>, 1258096. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref7">View at publisher</a></p>
<p id="bib8">Gibson, D.G. (2011). Enzymatic assembly of overlapping DNA fragments. Methods Enzymol. <i>498</i>, 349-361. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref8">View at publisher</a></p>
<p id="bib9">Gibson, D.G., Young, L., Chuang, R.Y., Venter, J.C., Hutchison, C.A., and Smith, H.O. (2009). Enzymatic assembly of DNA molecules up to several hundred kilobases. Nat. Methods <i>6</i>, 343-345. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref9">View at publisher</a></p>
<p id="bib10">Huston, J.S., Levinson, D., Mudgett-Hunter, M., Tai, M.S., Novotny, J., Margolies, M.N., Ridge, R.J., Bruccoleri, R.E., Haber, E., Crea, R., et al. (1988). Protein engineering of antibody binding sites: recovery of specific activity in an anti-digoxin single-chain Fv analogue produced in Escherichia coli. Proc. Natl. Acad. Sci. U S A <i>85</i>, 5879-5883. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref10">View at publisher</a></p>
<p id="bib11">Kwart, D., Paquet, D., Teo, S., and Tessier-Lavigne, M. (2017). Precise and efficient scarless genome editing in stem cells using CORRECT. Nat. Protoc. <i>12</i>, 329-354. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref11">View at publisher</a></p>
<p id="bib12">Li, K., Wang, G., Andersen, T., Zhou, P., and Pu, W.T. (2014). Optimization of genome engineering approaches with the CRISPR/Cas9 system. PLoS One <i>9</i>, e105779. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref12">View at publisher</a></p>
<p id="bib13">Mali, P., Yang, L., Esvelt, K.M., Aach, J., Guell, M., Dicarlo, J.E., Norville, J.E., and Church, G.M. (2013). RNA-guided human genome engineering via Cas9. Science <i>339</i>, 823-826. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref13">View at publisher</a></p>
<p id="bib14">Merkle, F.T., Neuhausser, W.M., Santos, D., Valen, E., Gagnon, J.A., Maas, K., Sandoe, J., Schier, A.F., and Eggan, K. (2015). Efficient CRISPR-Cas9-mediated generation of knockin human pluripotent stem cells lacking undesired mutations at the targeted locus. Cell Rep. <i>11</i>, 875-883. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref14">View at publisher</a></p>
<p id="bib15">Morrow, C.S., Porter, T.J., Xu, N., Arndt, Z.P., Ako-Asare, K., Heo, H.J., Thompson, E.A.N., and Moore, D.L. (2020). Vimentin Coordinates Protein Turnover at the Aggresome during Neural Stem Cell Quiescence Exit. Cell Stem Cell <i>26</i>, 558-568.e9. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref15">View at publisher</a></p>
<p id="bib16">Paquet, D., Kwart, D., Chen, A., Sproul, A., Jacob, S., Teo, S., Olsen, K.M., Gregg, A., Noggle, S., and Tessier-Lavigne, M. (2016). Efficient introduction of specific homozygous and heterozygous mutations using CRISPR/Cas9. Nature <i>533</i>, 125-129. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref16">View at publisher</a></p>
<p id="bib17">Peltier, J., Agrawal, S., Robertson, M.J., and Schaffer, D.V. (2010). In vitro culture and analysis of adult hippocampal neural progenitors. Methods Mol. Biol. <i>621</i>, 65-87. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref17">View at publisher</a></p>
<p id="bib18">Ran, F.A., Hsu, P.D., Wright, J., Agarwala, V., Scott, D.A., and Zhang, F. (2013). Genome engineering using the CRISPR-Cas9 system. Nat. Protoc. <i>8</i>, 2281-2308. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref18">View at publisher</a></p>
<p id="bib19">Sabourin, M., Tuzon, C.T., Fisher, T.S., and Zakian, V.A. (2007). A flexible protein linker improves the function of epitope-tagged proteins in Saccharomyces cerevisiae. Yeast <i>24</i>, 39-45. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref19">View at publisher</a></p>
<p id="bib20">Shaner, N.C., Steinbach, P.A., and Tsien, R.Y. (2005). A guide to choosing fluorescent proteins. Nat. Methods <i>2</i>, 905-909. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref20">View at publisher</a></p>
<p id="bib21">Snapp, E. (2005). Design and use of fluorescent fusion proteins in cell biology. Curr. Protoc. Cell Biol. <i>Chapter 21</i>, 21 4 1-21 4 13. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00451-2/sref21">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Darcie L. Moore (<a href="http://darcie.moore@wisc.edu">darcie.moore@wisc.edu</a>).</p>
<h4>Materials availability</h4>
<p>Starting plasmids for cloning of crRNAs and targeting vector are available on Addgene as indicated in the text. All other reagents are commercially available with the vendors indicated in the text. The specific plasmids generated to fluorescently label vimentin with mNeon are available from the lead contact upon request (Darcie L. Moore; <a href="mailto:darcie.moore@wisc.edu">darcie.moore@wisc.edu</a>).</p>
<h4>Data and code availability</h4>
<p>This study did not generate/analyze any data sets.</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/934-Mmc1.xlsx">Table S1. Example primers and DNA sequences used for creating Vimentin-mNeon NSCs, related to steps 15, 18, 24, and 35</a></p>
<h3>Acknowledgments</h3>
<p>We thank members of the Moore Lab for their input and the <a href="https://doi.org/10.13039/100007015">UW-Madison</a> flow cytometry core (P30 CA014520 and 1S10RR025483-01). We thank our funding sources: <a href="https://doi.org/10.13039/100000002">NIH</a> T32 T32GM008688 (to C.S.M.), Diana Jacobs Kalman Fellowship from <a href="https://doi.org/10.13039/100000965">AFAR</a> (to C.S.M.), Wisconsin Graduate Fellowship (to C.S.M.), a SciMed Graduate Research Fellowship (to T.J.P.), <a href="https://doi.org/10.13039/100000002">NIH</a> T32 T32AG000213 (to T.J.P.), an <a href="https://doi.org/10.13039/100000965">AFAR</a> Young Investigator Award (to D.L.M.), the Sloan Foundation Fellowship (to D.L.M.), a Vallee Scholar Award (to D.L.M.), and a <a href="https://doi.org/10.13039/100000002">NIH</a> New Innovator Award 1DP2OD025783 (to D.L.M.).</p>
<h3>Author contributions</h3>
<p>C.S.M. and D.L.M. conceived the idea for the protocol. T.J.P. wrote the section of the protocol outlining how to clone the targeting vector. C.S.M. wrote all other parts of the protocol with input from D.L.M. and T.J.P.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>