<article>
<section>
<h1 id="header">Purification and cryo-EM structure determination of <i>Arabidopsis thaliana</i> GLR3.4</h1>
<p><time datetime="2021-10-01">Published: October 1, 2021</time></p>
<p>Shanti Pal Gangwar,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a></sup> Marriah N. Green,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a></sup> Maria V. Yelshanskaya,<sup><a href="#aff1">1</a></sup> and Alexander I. Sobolevsky<sup><a href="#aff1">1</a>,<a href="#fn2">4</a>,<a href="#fn3">5</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biochemistry and Molecular Biophysics, Columbia University, 650 West 168th Street, New York, NY 10032, USA</p>
<p id="aff2"><sup>2</sup>Institute of Human Nutrition, Columbia University, 630 West 168th Street, New York, NY 10032, USA</p>
<p id="fn1"><sup>3</sup>These authors contributed equally</p>
<p id="fn2"><sup>4</sup>Technical contact</p>
<p id="fn3"><sup>5</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:as4005@cumc.columbia.edu">as4005@cumc.columbia.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100855">10.1016/j.xpro.2021.100855</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Ionotropic glutamate receptors (iGluRs) are ligand-gated ion channels that play crucial roles in the central nervous system. iGluR homologs, termed glutamate receptor-like channels (GLRs), have been found in plants. Investigating the structural and functional relationship between iGluRs and GLRs was limited by GLR protein expression, purification, and structural characterization. Here, we provide a detailed protocol for <i>Arabidopsis thaliana</i> GLR3.4 (<i>At</i>GLR3.4) expression in a mammalian cell line and purification for structure determination by cryogenic electron microscopy (cryo-EM).</p>
<p>For the complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref7">Green et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Protocol for expression of plant glutamate receptor-like channel (GLR) in HEK cells</li>
<li>Purification protocol that yields pure monodisperse tetrameric GLR protein</li>
<li>Purified protein subjected to cryo-EM analysis produces high-resolution GLR structure</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1057-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>This protocol was used in a recent publication (<a href="#bib7">Green et al., 2021</a>) to purify <i>Arabidopsis thaliana</i> GLR3.4 (<i>At</i>GLR3.4) recombinant protein expressed in a mammalian cell line for high-resolution full-length structure determination by single-particle cryo-EM. Before the experiment, prepare all the buffers to be used in subsequent steps one day in advance unless otherwise mentioned.</p>
<h3 id="sec1.1">Buffer preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>Prepare 1L of T buffer, 100 mL of Cell Lysis buffer, 25 mL of Solubilization buffer, 300 mL of Size-Exclusion Chromatography (SEC) buffer and 20 mL of Strep Elution buffer.</p>
<h3 id="sec1.2">Construct preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<ol>
<li>Subclone DNA for full-length <i>At</i>GLR3.4 (Met1-Thr959) into a pEG BacMam vector (<a href="#bib6">Goehring et al., 2014</a>) using gene-specific primers. Introduce the thrombin cleavage site (LVPRGS), followed by an eGFP and a streptavidin (strep) affinity tag (WSHPQFEK) at the C-terminus (<a href="#fig1">Figure 1</a>).
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1057-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. <i>At</i>GLR3.4 construct schematic</div>
</figcaption>
</figure>
</li>
<li>Design the gene-specific forward primer with the SalI and reverse primer with the NotI site to amplify the <i>At</i>GLR3.4 DNA flanked by Kozak sequence (KZK) and thrombin site to clone in pEG BacMam. Conduct polymerase chain reaction (PCR) using the information described in the tables below.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> PCR reaction can be stored at 4°C for 24 h or at −20°C for long-term storage.</div>
<ol start="3">
<li>Run a 1% agarose gel to verify the size and purity the PCR products.</li>
<li>Add 1 μL of DpnI enzyme and incubate the reaction at 37°C for 1 h.</li>
<li>Perform DNA cleanup using any commercially available kit. We use Monarch New England Bio Labs’ PCR and DNA Cleanup Kit following manufactures protocol.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> DNA can be stored at 4°C for 24 h or at −20°C for long-term storage.</div>
<ol start="6">
<li>Digest PCR DNA and pEG BacMam plasmid in two separate 1.5-mL tubes following the instruction in the table below. Incubate the digest reaction at 37°C for 1 h.</li>
<li>Perform a DNA cleanup using any commercially available kit as mentioned above.</li>
<li>Ligate both the PCR and linearized plasmid following the instruction in the table below. Incubate the ligation reaction at ∼20°C–25°C for 10–15 min. Use the insert and vector DNA at 3:1 molar ratio.</li>
<li>Transform DH5α competent cells and select the transformants on LB agar plates containing 100 μg/mL ampicillin.
<ol type="a">
<li>Incubate the plates at 37°C for 12–15 h.</li>
<li>Inoculate a single colony in LB media supplemented with 100 μg/mL ampicillin and perform a miniprep to extract and purify the plasmid DNA.</li>
<li>Sequence the DNA to confirm that the construct sequence is correct before proceeding.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The circular plasmid that we used was engineered to have a Kozak sequence, thrombin recognition site, eGFP, and Strep-tag as described in <a href="#fig1">Figure 1</a>.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Tris-HCl</td>
<td>Fisher Scientific</td>
<td>Cat# BP152-1</td>
</tr>
<tr>
<td>NaCl</td>
<td>Fisher Scientific</td>
<td>Cat# BP358-212</td>
</tr>
<tr>
<td>L-Glutamate</td>
<td>Sigma</td>
<td>Cat# 49621</td>
</tr>
<tr>
<td>PMSF</td>
<td>Acros Organics</td>
<td>Cat# 215740500</td>
</tr>
<tr>
<td>2-Mercaptoethanol (βME)</td>
<td>Acros Organics</td>
<td>Cat# 125470100</td>
</tr>
<tr>
<td>Thrombin</td>
<td>Haematologic Technologies</td>
<td>Cat# HCT-0020</td>
</tr>
<tr>
<td>D-Desthiobiotin</td>
<td>Sigma-Aldrich</td>
<td>Cat#D1411</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>Gibco</td>
<td>Cat# 16140071</td>
</tr>
<tr>
<td>Sf-900 III SFM</td>
<td>Gibco</td>
<td>Cat# 12658027</td>
</tr>
<tr>
<td>Freestyle 293 expression medium</td>
<td>Gibco</td>
<td>Cat# 12338018</td>
</tr>
<tr>
<td>Sodium butyrate</td>
<td>Acros Organics</td>
<td>Cat# 263191000</td>
</tr>
<tr>
<td>Digitonin</td>
<td>Cayman Chemical Company</td>
<td>Cat#14952</td>
</tr>
<tr>
<td>Aprotinin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A1153</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>Sigma-Aldrich</td>
<td>Cat#L0649</td>
</tr>
<tr>
<td>Pepstatin A</td>
<td>Sigma-Aldrich</td>
<td>Cat#P4265</td>
</tr>
<tr>
<td>Kanamycin</td>
<td>Fisher Scientific</td>
<td>Cat# BP906-5</td>
</tr>
<tr>
<td>Gentamicin sulfate</td>
<td>Sigma-Aldrich</td>
<td>Cat#G1914</td>
</tr>
<tr>
<td>Tetracycline</td>
<td>Fisher Scientific</td>
<td>Cat# BP912</td>
</tr>
<tr>
<td>Bluo-Gal</td>
<td>Life Technologies</td>
<td>Cat# 15519028</td>
</tr>
<tr>
<td>IPTG</td>
<td>Zymo Research</td>
<td>Cat# I1001-5</td>
</tr>
<tr>
<td>Sall-HF</td>
<td>New England Biolabs</td>
<td>Cat# R3138S</td>
</tr>
<tr>
<td>Notl-HF</td>
<td>New England Biolabs</td>
<td>Cat#3189S</td>
</tr>
<tr>
<td>Quick ligase</td>
<td>New England Biolabs</td>
<td>Cat#M2200</td>
</tr>
<tr>
<td>Monarch Plasmid Miniprep Kit</td>
<td>New England Biolabs</td>
<td>Cat #T1010L</td>
</tr>
<tr>
<td>Monarch DNA and PCR Cleanup Kit</td>
<td>New England Biolabs</td>
<td>Cat #T1030L</td>
</tr>
<tr>
<td>Nitrocellulose membrane filter (0.22 μm)</td>
<td>Merck Millipore</td>
<td>Cat#SA1J789H5</td>
</tr>
<tr>
<td>Wizard Plus Minipreps DNA Purification System</td>
<td>Promega</td>
<td>Cat#A7510</td>
</tr>
<tr>
<td>Phenol:chloroform:isoamyalcohol</td>
<td>Life Technologies</td>
<td>Cat# 15593031</td>
</tr>
<tr>
<td>Cellfectin II Reagent</td>
<td>Gibco</td>
<td>Cat#58760</td>
</tr>
<tr>
<td>Ethanol 200 Proof</td>
<td>Decon Labs, Inc.</td>
<td>Cat #2701</td>
</tr>
<tr>
<td>Centrifugal Filter Unit (Amicon Ultra-15)</td>
<td>Sigma-Aldrich</td>
<td>Cat#UFC910024</td>
</tr>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Coordinates of full-length <i>At</i>GLR3.4</td>
<td>(<a href="#bib7">Green et al., 2021</a>)</td>
<td>PDB: 7LZH</td>
</tr>
<tr>
<td>Cryo-EM map of full-length <i>At</i>GLR3.4</td>
<td>(<a href="#bib7">Green et al., 2021</a>)</td>
<td>EMDB: EMD-23606</td>
</tr>
<tr>
<td>Coordinates of <i>At</i>GLR3.4-S1S2<sub>Glu</sub>
</td>
<td>(<a href="#bib7">Green et al., 2021</a>)</td>
<td>PDB: 7LZ0</td>
</tr>
<tr>
<td>Crystal structure of GluA2</td>
<td>(<a href="#bib15">Sobolevsky et al., 2009</a>)</td>
<td>PDB: 3KG2</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>HEK 293S GnTI<sup>−</sup>
</td>
<td>ATCC</td>
<td>Cat#CRL-3022</td>
</tr>
<tr>
<td>Sf9</td>
<td>Gibco</td>
<td>Cat#12659017</td>
</tr>
<tr>
<td>DH10Bac</td>
<td>Life Technologies</td>
<td>Cat#10361012</td>
</tr>
<tr>
<td>DH5α</td>
<td>Zymo Research</td>
<td>Cat#T3007</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>
<i>At</i>GLR3.4 amplification primer: 5′- gtcgactccgccaccatgggatttttggtgatgataagag -3′</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>At</i>GLR3.4 amplification primer: 5′- cggcaccagagtaatttcgccatgttgtgattgtga -3′</td>
<td>This paper</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>gCTF</td>
<td>(<a href="#bib20">Zhang, 2016</a>)</td>
<td><a href="http://www.mrc-lmb.cam.ac.uk/kzhang/">http://www.mrc-lmb.cam.ac.uk/kzhang/</a></td>
</tr>
<tr>
<td>Motioncor2</td>
<td>(<a href="#bib21">Zheng et al., 2017</a>)</td>
<td><a href="https://msg.ucsf.edu/software">https://msg.ucsf.edu/software</a></td>
</tr>
<tr>
<td>RELION 3.0</td>
<td>(<a href="#bib22">Zivanov et al., 2018</a>)</td>
<td><a href="http://www2.mrc-lmb.cam.ac.uk/relion/">http://www2.mrc-lmb.cam.ac.uk/relion/</a></td>
</tr>
<tr>
<td>UCSF Chimera</td>
<td>(<a href="#bib12">Pettersen et al., 2004</a>)</td>
<td><a href="https://www.cgl.ucsf.edu/chimera/">https://www.cgl.ucsf.edu/chimera/</a></td>
</tr>
<tr>
<td>cryoSPARC</td>
<td>(<a href="#bib13">Punjani et al., 2017</a>)</td>
<td><a href="https://cryosparc.com/">https://cryosparc.com/</a></td>
</tr>
<tr>
<td>PHENIX</td>
<td>(<a href="#bib1">Adams et al., 2010</a>)</td>
<td><a href="https://www.phenix-online.org/">https://www.phenix-online.org/</a></td>
</tr>
<tr>
<td>Coot</td>
<td>(<a href="#bib5">Emsley and Cowtan, 2004</a>)</td>
<td><a href="http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot">http://www2.mrc-lmb.cam.ac.uk/Personal/pemsley/coot</a></td>
</tr>
<tr>
<td>SWISS-MODEL</td>
<td>(<a href="#bib18">Waterhouse et al., 2018</a>)</td>
<td><a href="https://swissmodel.expasy.org/">https://swissmodel.expasy.org/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>CF-1.2/1.3-2Au 200 mesh holey carbon grids</td>
<td>Protochips</td>
<td>Cat#CF-1.2/1.3-2Au</td>
</tr>
<tr>
<td>Gold wire</td>
<td>Ted Pella, Inc.</td>
<td>Cat#21-10</td>
</tr>
<tr>
<td>Superose6 10/300 column</td>
<td>GE Healthcare</td>
<td>Cat# 17–5172-01</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<table id="undtbl1">
<caption>T buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris-HCl pH 8.0 (1 M)</td>
<td>20 mM</td>
<td>20 mL</td>
</tr>
<tr>
<td>NaCl (5 M)</td>
<td>150 mM</td>
<td>30 mL</td>
</tr>
<tr>
<td>Milli-Q H<sub>2</sub>O</td>
<td>n/a</td>
<td>950 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>1 L</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Run T buffer through a 0.22 μm filter and store at 4°C for up to one month.</p>
<table id="undtbl2">
<caption>Cell Lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>T buffer</td>
<td>n/a</td>
<td>100 mL</td>
</tr>
<tr>
<td>β-mercaptoethanol (βME) (14.3 M)</td>
<td>1 mM</td>
<td>7.1 μL</td>
</tr>
<tr>
<td>Aprotinin (0.8 mM)</td>
<td>0.8 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Leupeptin (4.3 mM)</td>
<td>4.3 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Pepstatin A (2 mM)</td>
<td>2 μM</td>
<td>100 μL</td>
</tr>
<tr>
<td>Phenylmethylsulphonyl fluoride (1 M)</td>
<td>1 mM</td>
<td>100 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>∼100 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Protease inhibitors and βME should be added right before buffer use. Keep at 4°C, if needed can be stored at 4°C for up to 2 hours.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Protein degradation during the purification procedure can have detrimental effects on the resulting protein quality. It is, therefore, important that fresh reducing agents and protease inhibitors are added at the steps indicated in the protocol.</div>
<table id="undtbl3">
<caption>Solubilization buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>T buffer</td>
<td>n/a</td>
<td>25 mL</td>
</tr>
<tr>
<td>β-mercaptoethanol (βME) (14.3 M)</td>
<td>1 mM</td>
<td>1.78 μL</td>
</tr>
<tr>
<td>Aprotinin (0.8 mM)</td>
<td>0.8 μM</td>
<td>25 μL</td>
</tr>
<tr>
<td>Leupeptin (4.3 mM)</td>
<td>4.3 μM</td>
<td>25 μL</td>
</tr>
<tr>
<td>Pepstatin A (2 mM)</td>
<td>2 μM</td>
<td>25 μL</td>
</tr>
<tr>
<td>Phenylmethylsulphonyl fluoride (1 M)</td>
<td>1 mM</td>
<td>25 μL</td>
</tr>
<tr>
<td>Digitonin</td>
<td>2%</td>
<td>0.5 g</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>∼25 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Digitonin is not readily soluble at room temperature. Therefore, add digitonin to the T buffer, heat the mixture up to 70°C–75°C and stir at medium-low speed until completely dissolved. Keep at 4°C, if needed can be stored at 4°C for up to 2 hours.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Protease inhibitors and βME should be added to the chilled digitonin plus T buffer mixture right before membrane solubilization.</div>
<table id="undtbl4">
<caption>SEC buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>T buffer</td>
<td>n/a</td>
<td>300 mL</td>
</tr>
<tr>
<td>Digitonin</td>
<td>0.05%</td>
<td>150 mg</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>300 mL</b></td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> Digitonin is not readily soluble at room temperature. Therefore, add digitonin to the T buffer, heat the mixture up to 70°C–75°C and stir at medium-low speed until completely dissolved. After digitonin is completely dissolved, cool to ∼4°C–20°C. Once cooled, run SEC buffer through 0.22 μM filter. Keep at 4°C, if needed can be stored at 4°C for up to 24 hours.</div>
<table id="undtbl5">
<caption>Strep Elution buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>SEC buffer</td>
<td>n/a</td>
<td>20 mL</td>
</tr>
<tr>
<td>D-desthiobiotin</td>
<td>2.5 mM</td>
<td>10.8 mg</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>20 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">If needed, adjust the pH of Strep Elution buffer to 8.0, run it through 0.22 μM filter and store at 4°C for up to 12 h.</p>
<table id="undtbl6">
<caption>PCR reaction</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Final concentration</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>Autoclaved Milli-Q H<sub>2</sub>O</td>
<td>n/a</td>
<td>32 μL</td>
</tr>
<tr>
<td>5<b>×</b> Q5 Reaction Buffer</td>
<td>1<b>×</b>
</td>
<td>10 μL</td>
</tr>
<tr>
<td>Forward Primer (10 μM)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Reverse Primer (10 μM)</td>
<td>0.5 μM</td>
<td>2.5 μL</td>
</tr>
<tr>
<td>Template DNA</td>
<td>1–10 ng</td>
<td>1 μL</td>
</tr>
<tr>
<td>dNTPs (10 mM)</td>
<td>200 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Q5 High-Fidelity DNA Polymerase</td>
<td>0.04 U/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl7">
<caption>Thermocycler settings included lid temperature at 105°C and volume of 50 μL.</caption>
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Number of cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">25–35 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>55°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Final extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td colspan="2">Forever</td>
</tr>
</tbody>
</table>
<table id="undtbl8">
<caption>Digest reaction</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>PCR DNA/Plasmid</td>
<td>80 μL</td>
</tr>
<tr>
<td>CutSmart Buffer 10<b>×</b>
</td>
<td>10 μL</td>
</tr>
<tr>
<td>SalI-HF</td>
<td>2 μL</td>
</tr>
<tr>
<td>NotI-HF</td>
<td>2 μL</td>
</tr>
<tr>
<td>Milli-Q H2O</td>
<td>6 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
<table id="undtbl9">
<caption>Ligation reaction</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>PCR DNA</td>
<td>40–50 ng</td>
</tr>
<tr>
<td>Plasmid DNA</td>
<td>40–60 ng</td>
</tr>
<tr>
<td>Quick ligase buffer (2<b>×</b>)</td>
<td>5 μL</td>
</tr>
<tr>
<td>Quick ligase</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Milli-Q H2O</td>
<td>---</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>10 μL</b></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>Below we provide a detailed step-by-step protocol about the <i>At</i>GLR3.4 protein purification, sample preparation, and cryo-EM structure determination (<a href="#fig2">Figure 2</a>).</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1057-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Overview of the protocol of <i>At</i>GLR3.4 protein expression, purification, sample preparation, and cryo-EM structure determination</div>
<p>The image has been created using BioRender (<a href="https://biorender.com/">https://biorender.com/</a>).</p>
</figcaption>
</figure>
<h3 id="sec3.1">Bacmid preparation and isolation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>According to the manufacturer protocol, bacmid preparation and isolation were carried out using DH10Bac competent cells (Bac-to-Bac, Invitrogen). The steps are described here.</p>
<ol>
<li>For transposition, prepare LB agar plates containing 50 μg/mL kanamycin, 7 μg/mL gentamicin, 10 μg/mL tetracycline, 200 μg/mL Bluo-gal and 40 μg/mL IPTG.</li>
<li>Transformation of bacmid
<ol type="a">
<li>Thaw DH10Bac (Life Technologies Cat#10361012) competent cells on ice and dispense 25–30 μL to a prechilled 1.5-mL polypropylene tube.</li>
<li>Add approximately 1–10 ng (1–2 μL) of plasmid DNA (pEG BacMam_<i>At</i>GLR3.4) and mix gently by flicking/tapping the tube several times.</li>
<li>Incubate on ice for 15–30 min.</li>
<li>Transfer the tube to a water bath at 42°C for 45 s.</li>
<li>After heat shock, immediately transfer the tube back to ice and let it cool on ice for 2–3 min.</li>
<li>Add 200 μL SOC media to the tube and transfer to an incubator shaker set at 220 rpm and incubate for 4 h at 37°C.</li>
<li>Pipette 20–100 μL of transformation reaction on the plates and spread evenly.</li>
</ol>
</li>
<li>Grow transformed cells
<ol type="a">
<li>Incubate the plates for at least 24 h at 37°C. Blue and white colonies will appear; for better results, incubate the plates for 48 h to distinguish the blue from the white colonies.</li>
<li>Carefully inoculate a single white colony in 5–6 mL LB broth supplemented with 50 μg/mL kanamycin, 7 μg/mL gentamicin, and 10 μg/mL tetracycline.</li>
<li>Incubate for 12–15 h at 37°C with continuous rotation at 220–250 rpm.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Overnight (12–16 h)</div>
<ol start="4">
<li>Isolate bacmid DNA
<ol type="a">
<li>Centrifuge the culture at ∼3,100 g for 10 min. Then, decant to discard the supernatant and save the cell pellet.
<div class="note">
<span class="note-title">Note:</span> If needed, the pellet can be stored at −20°C for up to a couple of weeks prior to bacmid DNA isolation. For bacmid isolation, we use reagents from the Promega mini prep kit. Reagents from other mini prep kits can also be used.</div>
</li>
<li>Resuspend the cell pellet in 200 μL resuspension buffer until homogeneous, via pipetting up and down and/or by vortexing. Transfer the suspension to a 1.5-mL Eppendorf tube.</li>
<li>Add 200 μL lysis buffer and mix by gently inverting the tube several (∼15–20) times. The suspension in the tube turns slightly viscous.</li>
<li>Add 200 μL neutralization buffer and mix by gentle inversion several (∼20) times. A white precipitate will be observed.</li>
<li>Centrifuge the tube at ∼21,100 <i>g</i> for 10 min at ∼20°C–25°C.</li>
<li>Avoiding the pellet, carefully collect and transfer the supernatant to a fresh 1.5-mL Eppendorf tube. In the fume hood, add an equal volume (typically around 500–600 μL) of Tris equilibrated Phenol:Chloroform: Isoamyl-alcohol (25:24:1) and mix by inverting gently several times for 1 min.</li>
<li>Spin at ∼21,100 <i>g</i> for 5 min at ∼20°C–25°C. In the fume hood, gently collect the top layer supernatant (∼500–600 μL) and transfer it to a new 1.5-mL Eppendorf tube. Add 2<b>×</b> volume (∼1–1.2 mL) of ice-cold 100% ethanol, mix well by gently inverting for 30 s, and incubate at −20°C for a minimum of 30 min.</li>
<li>Centrifuge at ∼21,100 <i>g</i> for 20–30 min at 4°C. A white pellet is observed at the bottom of the tube. Carefully decant to discard the supernatant and add 1 mL ice-cooled 70% ethanol.</li>
<li>Invert the tube gently several times to wash the pellet. Centrifuge for 5 min at ∼21,100 <i>g</i> at 4°C.</li>
<li>Discard the supernatant by gentle aspiration (make sure to not discard the pellet as it may dislodge from the bottom).</li>
<li>To avoid traces of remaining ethanol, air dry the pellet for 15–20 min at ∼20°C–25°C in a fume hood. Add 40–50 μL of autoclaved milli-Q water and dissolve the pellet by gentle tapping/flicking. Once the pellet is dissolved, place the tube with bacmid DNA on ice. Now the bacmid DNA is ready to be used for baculovirus production and can be stored at −80°C for an extended period.</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Transfection of Sf9 cells with recombinant bacmid DNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–6 days</div>
<p>Production of the first-generation baculovirus (P1 virus).</p>
<div class="note">
<span class="note-title">Note:</span> The following steps should be performed under sterile conditions in a laminar hood.</div>
<ol start="5">
<li>In a 6-well plate, add 2 mL of Sf9 cells per well, with the Sf9 cell density of 0.5–0.75 <b>×</b>10<sup>6</sup>/mL (1–1.5 <b>×</b>10<sup>6</sup> cells per P1 virus). Allow the cells to adhere for 30 min to 1 h in a dark incubator at 27°C.</li>
<li>In the meantime, take a 1.5-mL Eppendorf tube and add:
<ol type="a">
<li>180 μL of SF900 media</li>
<li>10 μL of Cellfectin II</li>
<li>10 μL of thawed bacmid DNA</li>
</ol>
</li>
<li>Close and tap the tube to mix. Let mixture sit for 20–30 min in the hood.</li>
<li>Add the mixture of SF900 media, bacmid DNA, and Cellfectin II dropwise to the cells in the well.</li>
<li>Incubate the plate at 27°C for 4–6 days. Observe the plate under the microscope to monitor cell behavior and fluorescence if the gene of interest is fused with a fluorescent marker gene.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> It is always better to have a fluorescent marker fused to the protein of interest to monitor baculovirus preparation and protein expression. The <i>At</i>GLR3.4 P1 virus is typically harvested on the 5<sup>th</sup> day post-transfection because most of the cells already express the protein (monitored by eGFP fluorescence), while very little cell death is observed.</div>
<ol start="10">
<li>To harvest the P1 virus, filter the 2 mL medium using a 0.2 μm sterile syringe filter in a laminar hood and store the filtered virus at 4°C protected from light. The P1 virus can be stored for 4–6 months.</li>
</ol>
<h3 id="sec3.3">P2 virus production</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 days</div>
<div class="note">
<span class="note-title">Note:</span> The following steps (except for centrifugation and resuspension) should be performed under sterile conditions in a laminar hood.</div>
<ol start="11">
<li>In a laminar hood and sterile conditions, infect 500 mL of Sf9 cells at a cell density of 1.5<b>×</b>10<sup>6</sup> cells/mL in a 1-L non-baffled Erlenmeyer flask by adding 500 μL of P1 virus. Incubate the Sf9 cell culture in a shaker at 27°C and 115 rpm for 72–120 h. Keep the cell culture protected from light.</li>
<li>Centrifuge the culture at ∼5,000 <i>g</i> for 15 min at 4°C in a Sorvall centrifuge. Discard the cell pellet and save the supernatant containing the P2 virus.</li>
<li>Ultracentrifuge the supernatant at 4°C for 1 h at ∼61,000 <i>g</i> in a preparative ultracentrifuge with a Type 45Ti Beckman Coulter rotor.</li>
<li>A small translucent pellet is observed at the bottom of the tube. Discard the supernatant and resuspend the pellet in 50 or 25 mL Gibco FreeStyle 293 expression media supplemented with 2% of γ-irradiated fetal bovine serum (FBS) to get 10<b>×</b> or 20<b>×</b> concentrated P2 virus, respectively.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Always run the FBS through a 0.22 μm filter before adding to the media (perform this procedure under sterile conditions in a laminar hood).</div>
<ol start="15">
<li>Pass the resuspension through a 0.22 μm filter into a sterile 50-mL tube and store at 4°C well protected from light. The P2 virus stock can be stored for 1–2 months at 4°C while protected from light.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> To fully dissolve the pellet, allow it to stay in the resuspension media in the dark at 4°C for no longer than 24 hours prior to filtration.</div>
<h3 id="sec3.4">Transduction of HEK 293S GnTI<sup>−</sup> cells with P2 baculovirus</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 days</div>
<ol start="16">
<li>In a laminar hood and sterile conditions, prepare 800 mL of HEK 293S GnTI<sup>−</sup> cell culture at the density of 2.5–3.5 <b>×</b> 10<sup>6</sup> cells/mL maintained in Freestyle 293 media supplemented with 2% FBS in 2-L baffled Erlenmeyer flask and add 8 mL of 20<b>×</b> concentrated P2 virus stock. Incubate the cell culture at 37°C in an orbital shaker at 110 rpm and 5% CO<sub>2</sub>.</li>
<li>To enhance the protein expression, add 10 mM of sodium butyrate to the culture 12–20 h post-transduction and decrease the incubator temperature to 30°C.</li>
<li>After 72–74 h post-transduction, harvest the cells by centrifugation at ∼5,000 <i>g</i> (in a Sorvall RC 5B Plus centrifuge) for 15 min at 4°C. Discard the supernatant, wash the cell pellet by resuspension in phosphate-buffered saline (PBS, pH 8.0), and transfer the suspension to a 50-mL Falcon tube. Centrifuge at ∼3,100 <i>g</i> for 10 min at 4°C. Discard the supernatant and store the cell pellet at −80°C until further use.</li>
</ol>
<h3 id="sec3.5">Purification of <i>At</i>GLR3.4</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<div class="note">
<span class="note-title">Note:</span> The buffers should be filtered using 0.22 μm filter (Merck-Millipore) and chilled to 4°C prior to utilization. When required, add β-mercaptoethanol (βME) and protease inhibitors right before buffer usage. All protein purification steps are to be carried out at 4°C or on ice, unless otherwise noted.</div>
<ol start="19">
<li>Resuspend the cell pellet in ice-cold Cell Lysis buffer. Add a stir bar and ∼40 mL of the Cell Lysis buffer to the 50-mL falcon tube with the frozen cell pellet till it reaches 45-mL volume and resuspend the pellet by rocking on a platform or by vortexing until the cell pellet becomes detached and dissolved.</li>
<li>Disrupt the cells by sonication using Misonix Sonicator with a preset program with six cycles (3 min total process time) at the amplitude of 8, 15 s pulse on time, and 15 s pulse off time. The sonication steps must be carried out on a stir plate (at medium speed stirring) on ice or at 4°C to avoid heating and denaturing of the protein. Repeat this process 2 more times or until optimal cell lysis.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For optimal cell lysis, check a small droplet of the lysate under a light microscope to confirm that cells are lysed post-sonication, as the appearance will be distinctly different from cells prior to sonication.</div>
<ol start="21">
<li>Centrifuge the cell lysate suspension at ∼3,100 <i>g</i> (using an Eppendorf Centrifuge 5810) for 10 min at 4°C to remove cell debris and unbroken cells.</li>
<li>Collect the clarified supernatant by decanting the supernatant into a prechilled polycarbonate bottle assembly for ultracentrifugation. Ultracentrifuge in a Type 45Ti Beckman Coulter fixed-angle rotor at ∼186,000 <i>g</i> for 1 h at 4°C to obtain the membrane fraction.</li>
<li>Discard the supernatant and mechanically homogenize the obtained membrane fraction pellet in Cell Lysis buffer (∼25 mL) using a homogenizer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We use a prechilled dounce homogenizer and move a plunger forth and back at least 6 times to fully homogenize the pellet in the Cell Lysis buffer.</div>
<ol start="24">
<li>Solubilize the membrane protein from the homogenate by adding an equal volume of solubilization buffer to the homogenate (totaling ∼50 mL with a final ∼1% concentration of the detergent, digitonin in our case) and stir/rotate (low-medium speed to prevent bubble formation) at 4°C for ∼2 h (1 h minimum).</li>
<li>Ultracentrifuge the solubilizate at ∼186,000 <i>g</i> in a Type 45Ti Beckman Coulter fixed-angle rotor at 4°C for 1 h to remove insoluble material.</li>
<li>Collect the supernatant and add 2 mL of strep resin prewashed and equilibrated in T buffer. Rotate the mix for 12–14 h at 4°C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> overnight (12–15 h)</div>
<ol start="27">
<li>Purify the protein using affinity chromatography.
<ol type="a">
<li>Equilibrate 30-mL chromatography gravity column by rinsing column with milli-Q water and allowing SEC buffer to flow through.</li>
<li>Transfer the <i>At</i>GLR3.4 protein-bound strep resin to the empty pre-equilibrated gravity chromatography column and collect the flow-through.</li>
<li>Wash the resin by pouring 25 mL of SEC buffer into the column.</li>
<li>Elute the <i>At</i>GLR3.4 protein with ∼10–15 mL of freshly made Strep Elution buffer. Observe the eGFP-tagged protein eluting from the column by the color of the collected protein appearing green and the resin becoming white. No more Strep Elution buffer is needed once the strep resin is white, stripped of its eGFP green color, indicating that the protein has been eluted from the column.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When adding buffer to the column, wait until the buffer from the previous step is almost finished flowing through the column but do not let the column run dry.</div>
<ol start="28">
<li>Measure the concentration of the protein, add 1/300 (w/w) thrombin and incubate at 22°C for 90 min to cleave off eGFP and the strep tag. Protein concentration can be measured using a spectrophotometer set to A<sub>280</sub> (absorbance at 280 nm) and blanked with the elution buffer.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Successful thrombin cleavage can be monitored by fluorescence-detection size-exclusion chromatography (FSEC) (<a href="#bib8">Kawate and Gouaux, 2006</a>) and SDS-PAGE (<a href="#fig3">Figure 3</a>).</div>
<ol start="29">
<li>Concentrate the thrombin digest reaction to ∼500 μL using 100-kDa NMWL centrifugal filter.</li>
<li>Transfer the concentrated protein to a new 1.5-mL tube and centrifuge the concentrated protein at ∼86,500 <i>g</i> for 30 min at 4°C using a Sorvall MTX150 Micro-Ultracentrifuge (Thermo Fisher Scientific) and a S100AT4 rotor.</li>
<li>Avoiding the pellet, inject the supernatant into a 500-μL loop connected to a Superose™ 6 10/300 GL SEC column attached to an AKTA FPLC (GE Healthcare) and pre-equilibrated in SEC Buffer.</li>
<li>At the end of the SEC column run at 0.5 mL/min, pool the peak fractions corresponding to <i>At</i>GLR3.4 tetramer (<a href="#fig4">Figure 4</a>) and concentrate to 3–4 mg/mL using 100-kDa NMWL centrifugal filter. Typically, ∼800 mL of HEK 293S GnTI<sup>−</sup> cell culture yields ∼1 mg of purified protein. The concentrated, purified protein can be stored at 4°C, structurally stable, and functionally active for one week.
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1057-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. On the left, FSEC traces for <i>At</i>GLR3.4 before (blue) and after (red) thrombin cleavage</div>
<p>The chromatograms were recorded using Superose 6 column at the flow rate of 0.5 mL/min using eGFP fluorescence (excitation, 488 nm; emission, 507 nm). On the right, SDS-PAGE demonstrating <i>At</i>GLR3.4 bands before and after thrombin digest at two (1/1,000 and 1/300) thrombin/<i>At</i>GLR3.4-eGFP ratios.</p>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1057-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. On the left, SEC profile for purified <i>At</i>GLR3.4</div>
<p>The chromatogram was recorded using Superose 6 column at the flow rate of 0.5 mL/min using A<sub>280</sub> absorbance. In the middle, SDS-PAGE for the protein peak fractions outlined by the red dashed lines on the SEC plot. On the right, normalized FSEC traces for purified <i>At</i>GLR3.4, rat GluA2-γ2 fusion (<a href="#bib16">Twomey et al., 2016</a>, <a href="#bib17">2017</a>), mouse TRPV3 (<a href="#bib11">Nadezhdin et al., 2021</a>; <a href="#bib14">Singh et al., 2018</a>) and human TRPV6 (<a href="#bib3">Bhardwaj et al., 2020</a>; <a href="#bib10">McGoldrick et al., 2018</a>). The latter three membrane proteins represent molecular weight markers and confirm the tetrameric assembly of <i>At</i>GLR3.4. The chromatograms were recorded using Superose 6 column at the flow rate of 0.5 mL/min using tryptophan fluorescence (excitation, 280 nm; emission, 334 nm).</p>
</figcaption>
</figure>
</li>
<li>Inspect the purity of the protein by sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE).</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Protein purity can also be assessed by FSEC.</div>
<h3 id="sec3.6">Grid sample preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–3 h</div>
<ol start="34">
<li>For cryo-EM grid sample preparation, use UltrAuFoil CF-1.2/1.3-2Au 200 mesh holey carbon grids covered with a thin layer of gold according to the published method (<a href="#bib23">Russo and Passmore, 2014</a>).
<div class="optional">
<span class="optional-title">Optional:</span> The user can optimize various grid types and vitrification parameters however they see fit.</div>
<ol type="a">
<li>Deposit ∼50 nm of gold onto the grids using gold wire and the Edwards Auto 306 evaporator.
<div class="note">
<span class="note-title">Note:</span> This grid gold-coating step is only necessary for carbon-coated grids.</div>
</li>
<li>Then remove the carbon with the Gatan Solarus (model 950) Advanced Plasma Cleaning System Ar/O<sub>2</sub> treatment (4 min, 50 watts, 35.0 sccm Ar, 11.5 sccm O<sub>2</sub>).
<div class="note">
<span class="note-title">Note:</span> This carbon removal step is only necessary after the previous grid gold-coating step.</div>
</li>
<li>Before applying the purified protein sample to the gold-coated side of the grid, subject the grids to a H<sub>2</sub>/O<sub>2</sub> plasma treatment using the Gatan Solarus (model 950) Advanced Plasma Cleaning System (20 s, 10 watts, 6.4 sccm H<sub>2</sub>, 27.5 sccm O<sub>2</sub>) with the gold-coated side facing up.
<div class="note">
<span class="note-title">Note:</span> Glow discharging step could be used as an alternative to plasma cleaning.</div>
</li>
</ol>
</li>
</ol>
<ol start="35">
<li>Prior to applying the purified protein to the grid, ultracentrifuge the protein at ∼86,500 <i>g</i> for 30 min at 4°C using a SorvallMTX150 Micro-Ultracentrifuge (Thermo Fisher Scientific) and a S100AT4 rotor. Transfer the supernatant to a prechilled tube, avoiding any possible protein precipitation.</li>
<li>For vitrification, set the vitrobot to 4°C with 100% humidity, a 15 s wait time, and a 4–5 s blot time with a 4–5 blot force. Apply 3 μL of purified <i>At</i>GLR3.4 protein at 3–3.5 mg/mL to the gold-coated side of the grid and plunge freeze the grid into liquid ethane cooled to ∼−190°C in liquid nitrogen using a Mark IV vitrobot (Thermo fisher scientific).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> After grids are plunge frozen, they must remain at cryogenic temperatures and avoid contamination as well as exposure to humidity or condensation.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The prepared grids can stay in liquid nitrogen for long-term storage.</div>
<h3 id="sec3.7">Grid screening, cryo-EM data collection, and processing</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2–4 weeks</div>
<ol start="37">
<li>Test the grids on screening transmission electron microscopes (TEMs), such as Glacios or F20 (FEI Thermo Scientific), and assess particle distribution, orientation, and ice quality.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The grids can be loaded onto the F20 microscope straight from the grid storage box. However, the grids need to be clipped before being screened on the Glacios microscope.</div>
<ol start="38">
<li>Collect cryo-EM data from the prescreened grids on a FEI Titan Krios TEM (Thermo Fisher Scientific) operating at 300 kV and equipped with a direct electron detection (DED) camera in counting mode with ∼50 frames per movie and a total dose of ∼58 electrons per Å<sup>2</sup>, a physical pixel size of ∼0.83 Å, and a defocus value between −0.5 to −2.5 μm. We used Krios with a post-column GIF Quantum energy filter and a Gatan K3 Summit DED camera, Gatan, Pleasanton, CA, USA.</li>
<li>Process the data in RELION 3.1 (<a href="#bib22">Zivanov et al., 2018</a>) or CryoSparc (<a href="#bib13">Punjani et al., 2017</a>). Perform beam-induced motion correction using MotionCor2 (Dose per frame of 1.16, EER fractionation -32, B Factor -150, number of patches 5<b>×</b>5) (<a href="#bib21">Zheng et al., 2017</a>) and contrast transfer function (CTF) estimation using CTFFIND4.1 (<a href="#bib20">Zhang, 2016</a>) in RELION 3.1.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All the processing steps can also be performed in cryoSPARC.</div>
<ol start="40">
<li>Manually pick ∼3,000 particles by selecting any particle shape that commonly appears in the micrographs, recognizing reoccurring shapes as well as any shape that potentially resembles an iGluR.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> We used the particle/mask diameter of 230 Å.</div>
<ol start="41">
<li>2D-classify these manually selected particles to generate templates for further template-based auto picking in RELION 3.1.</li>
<li>4<b>×</b>4 bin the picked particle images to a pixel size of ∼3.32 Å/pixel and subject them to 3D classification into 10 classes (C1 symmetry). Select the best classes, which corresponded to the Y shape of iGluR (<a href="#fig5">Figure 5</a>).
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1057-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Cryo-EM processing workflow</div>
</figcaption>
</figure>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A model-generated iGluR map (for example, using the GluA2 model, PDB: 3KG2 (<a href="#bib15">Sobolevsky et al., 2009</a>)), low-pass filtered to 40 Å, can be used as an initial reference.</div>
<ol start="43">
<li>Extract particles for the best 3D classes without binning to an original pixel size (∼0.83 Å/pixel).</li>
</ol>
<p>Perform 3D refinement using C2 symmetry in RELION 3.1. At this step, the postprocessed map from RELION had the resolution of ∼3.98 Å (FSC = 0.143) from 277,615 particles (<a href="#fig5">Figure 5</a>).</p>
<ol start="44">
<li>Carry out micelle subtraction. For micelle subtraction, first create a mask using 3D refined map as input from the last 3D refinement step in Relion. Then run Particle subtraction using optimizer.star file from the last 3D refinement step and the mask as an input. Carry out multiple rounds of 3D classification (C1 symmetry) without angular sampling to reduce particle heterogeneity followed by 3D refinement. Also, perform Bayesian polishing and CTF refinement.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Vary the regularization parameter (T) and iterations for 3D classification. We used T= 4 and iterations = 40 in 3D classification without angular sampling. Multiple rounds of 3D classification into 8 classes followed by 3D refinement on the particles from the best class using C2 symmetry and postprocessing yielded a map with resolution of 3.73Å from 118,592 particles (FSC=0.143) in RELION. Every time we used the 3D refined map from the previous job as an input reference in the subsequent 3D classification job.</div>
<ol start="45">
<li>To improve the map quality, import the particles from the last 3D refinement job from RELION to cryoSPARC and clean them up by 2D classification.</li>
<li>Select the best 2D classes and subject the corresponding particles to homogeneous and non-uniform refinement in cryoSPARC applying C2 symmetry. The map after non-uniform refinement in cryoSPARC had a resolution of 3.57 Å (FSC = 0.143) (<a href="#bib4">Chen et al., 2013</a>) from 110,630 particles.</li>
<li>Estimate the local resolution using the unfiltered half-maps and Resmap (<a href="#bib9">Kucukelbir et al., 2014</a>) (<a href="#fig5">Figure 5</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Local resolution can also be estimated in cryoSPARC.</div>
<ol start="48">
<li>Use UCSF Chimera (<a href="#bib12">Pettersen et al., 2004</a>) to visualize the EM density maps.</li>
</ol>
<h3 id="sec3.8">Model building and refinement</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<p>Once the high-resolution cryo-EM map is obtained, build atomic models of <i>At</i>GLR3.4.</p>
<ol start="49">
<li>As guides, use the crystal structure of <i>At</i>GLR3.4-S1S2 (PDB: 7LZ0) (<a href="#bib7">Green et al., 2021</a>) to build the ligand-binding domain and homology modeling in SWISS-MODEL (<a href="#bib18">Waterhouse et al., 2018</a>) using the GluA2 crystal structure as a guide (PDB: 3KG2, (<a href="#bib15">Sobolevsky et al., 2009</a>)) to build the amino terminal and transmembrane domains. Finalize the model building manually using COOT (<a href="#bib5">Emsley and Cowtan, 2004</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Since <i>At</i>GLR3.4 is a tetramer, it is advisable to first build two subunits, A and B, and then duplicate the AB dimer following the C2 symmetry of the map to make the CD dimer and assemble the ABCD tetramer.</div>
<ol start="50">
<li>Refine the obtained atomic model of <i>At</i>GLR3.4 in real-space using PHENIX (<a href="#bib1">Adams et al., 2010</a>).</li>
<li>Validate the model quality. For this purpose, we used the validation programs MolProbity (score = 1.69, clash score = 4.49, poor rotamers = 0.44%) (<a href="#bib19">Williams et al., 2018</a>) and EMRinger (EMRinger score = 2.41, optimal Threshold = 0.67, Rotamer-ratio = 0.77) (<a href="#bib2">Barad et al., 2015</a>) in Phenix.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Approximately ∼ 1 mg of purified <i>At</i>GLR3.4 can be obtained from 800 mL HEK cells.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The multidomain architecture and flexibility of <i>At</i>GLR3.4 could limit the resolution.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>No PCR fragment amplified or incorrect size of the PCR product (step 3 of the Construct preparation).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Optimize the primer annealing temperature by running a gradient PCR and optimize the primer extension time.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Few or no transformants (step 9 of the Construct preparation and step 3 of the main protocol).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>Check the efficiency of the competent cells by transforming a control circular plasmid. Also check the temperature of the water bath for heat shock.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>For different membrane proteins, one may need to use different detergents for solubilization (step 24 of the main protocol).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Check which detergent best extracts the protein of interest by running a detergent screen via FSEC on crude cell samples solubilized in different detergents.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>Inefficient thrombin cleavage and eGFP removal (step 28 of the main protocol).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Optimize the concentration of the protein and thrombin amount in the reaction mixture. Sometimes, extremely high concentration of the protein results in incomplete thrombin digestion.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Too high or too low particle density on the grid (step 37 of the main protocol).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>Adjust the protein concentration accordingly or vary the blot time and blot force when using the Vitrobot.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Adams, P.D., Afonine, P.V., Bunkoczi, G., Chen, V.B., Davis, I.W., Echols, N., Headd, J.J., Hung, L.W., Kapral, G.J., Grosse-Kunstleve, R.W., et al. (2010). PHENIX: a comprehensive Python-based system for macromolecular structure solution. Acta Crystallogr. D Biol. Crystallogr. <i>66</i>, 213-221. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref1">View at publisher</a></p>
<p id="bib2">Barad, B.A., Echols, N., Wang, R.Y., Cheng, Y., DiMaio, F., Adams, P.D., and Fraser, J.S. (2015). EMRinger: side chain-directed model and map validation for 3D cryo-electron microscopy. Nat. Methods <i>12</i>, 943-946. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref2">View at publisher</a></p>
<p id="bib3">Bhardwaj, R., Lindinger, S., Neuberger, A., Nadezhdin, K.D., Singh, A.K., Cunha, M.R., Derler, I., Gyimesi, G., Reymond, J.L., Hediger, M.A., et al. (2020). Inactivation-mimicking block of the epithelial calcium channel TRPV6. Sci. Adv. <i>6</i>, eabe1508. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref3">View at publisher</a></p>
<p id="bib4">Chen, S., McMullan, G., Faruqi, A.R., Murshudov, G.N., Short, J.M., Scheres, S.H., and Henderson, R. (2013). High-resolution noise substitution to measure overfitting and validate resolution in 3D structure determination by single particle electron cryomicroscopy. Ultramicroscopy <i>135</i>, 24-35. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref4">View at publisher</a></p>
<p id="bib5">Emsley, P. and Cowtan, K. (2004). Coot: model-building tools for molecular graphics. Acta Crystallogr. D Biol. Crystallogr. <i>60</i>, 2126-2132. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref5">View at publisher</a></p>
<p id="bib6">Goehring, A., Lee, C.H., Wang, K.H., Michel, J.C., Claxton, D.P., Baconguis, I., Althoff, T., Fischer, S., Garcia, K.C., and Gouaux, E. (2014). Screening and large-scale expression of membrane proteins in mammalian cells for structural studies. Nat. Protoc. <i>9</i>, 2574-2585. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref6">View at publisher</a></p>
<p id="bib7">Green, M.N., Gangwar, S.P., Michard, E., Simon, A.A., Portes, M.T., Barbosa-Caro, J., Wudick, M.M., Lizzio, M.A., Klykov, O., Yelshanskaya, M.V., et al. (2021). Structure of the Arabidopsis thaliana glutamate receptor-like channel GLR3.4. Mol. Cell <i>81</i>, 3216-3226. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref7">View at publisher</a></p>
<p id="bib8">Kawate, T. and Gouaux, E. (2006). Fluorescence-detection size-exclusion chromatography for precrystallization screening of integral membrane proteins. Structure <i>14</i>, 673-681. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref8">View at publisher</a></p>
<p id="bib9">Kucukelbir, A., Sigworth, F.J., and Tagare, H.D. (2014). Quantifying the local resolution of cryo-EM density maps. Nat. Methods <i>11</i>, 63-65. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref9">View at publisher</a></p>
<p id="bib10">McGoldrick, L.L., Singh, A.K., Saotome, K., Yelshanskaya, M.V., Twomey, E.C., Grassucci, R.A., and Sobolevsky, A.I. (2018). Opening of the human epithelial calcium channel TRPV6. Nature <i>553</i>, 233-237. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref10">View at publisher</a></p>
<p id="bib11">Nadezhdin, K.D., Neuberger, A., Trofimov, Y.A., Krylov, N.A., Sinica, V., Kupko, N., Vlachova, V., Zakharian, E., Efremov, R.G., and Sobolevsky, A.I. (2021). Structural mechanism of heat-induced opening of a temperature-sensitive TRP channel. Nat. Struct. Mol. Biol. <i>28</i>, 564-572. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref11">View at publisher</a></p>
<p id="bib12">Pettersen, E.F., Goddard, T.D., Huang, C.C., Couch, G.S., Greenblatt, D.M., Meng, E.C., and Ferrin, T.E. (2004). UCSF Chimera--a visualization system for exploratory research and analysis. J. Comput. Chem. <i>25</i>, 1605-1612. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref12">View at publisher</a></p>
<p id="bib13">Punjani, A., Rubinstein, J.L., Fleet, D.J., and Brubaker, M.A. (2017). cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. Nat. Methods <i>14</i>, 290-296. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref13">View at publisher</a></p>
<p id="bib23">Russo, C.J. and Passmore, L.A. (2014). Electron microscopy: Ultrastable gold substrates for electron cryomicroscopy. Science <i>346</i>, 1377-1380. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/optztn34pfV87">View at publisher</a></p>
<p id="bib14">Singh, A.K., McGoldrick, L.L., and Sobolevsky, A.I. (2018). Structure and gating mechanism of the transient receptor potential channel TRPV3. Nat. Struct. Mol. Biol. <i>25</i>, 805-813. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref14">View at publisher</a></p>
<p id="bib15">Sobolevsky, A.I., Rosconi, M.P., and Gouaux, E. (2009). X-ray structure, symmetry and mechanism of an AMPA-subtype glutamate receptor. Nature <i>462</i>, 745-756. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref15">View at publisher</a></p>
<p id="bib16">Twomey, E.C., Yelshanskaya, M.V., Grassucci, R.A., Frank, J., and Sobolevsky, A.I. (2016). Elucidation of AMPA receptor-stargazin complexes by cryo-electron microscopy. Science <i>353</i>, 83-86. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref16">View at publisher</a></p>
<p id="bib17">Twomey, E.C., Yelshanskaya, M.V., Grassucci, R.A., Frank, J., and Sobolevsky, A.I. (2017). Channel opening and gating mechanism in AMPA-subtype glutamate receptors. Nature <i>549</i>, 60-65. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref17">View at publisher</a></p>
<p id="bib18">Waterhouse, A., Bertoni, M., Bienert, S., Studer, G., Tauriello, G., Gumienny, R., Heer, F.T., de Beer, T.A.P., Rempfer, C., Bordoli, L., et al. (2018). SWISS-MODEL: homology modelling of protein structures and complexes. Nucleic Acids Res. <i>46</i>, W296-W303. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref18">View at publisher</a></p>
<p id="bib19">Williams, C.J., Headd, J.J., Moriarty, N.W., Prisant, M.G., Videau, L.L., Deis, L.N., Verma, V., Keedy, D.A., Hintze, B.J., Chen, V.B., et al. (2018). MolProbity: more and better reference data for improved all-atom structure validation. Protein Sci. <i>27</i>, 293-315. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref19">View at publisher</a></p>
<p id="bib20">Zhang, K. (2016). Gctf: real-time CTF determination and correction. J. Struct. Biol. <i>193</i>, 1-12. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref20">View at publisher</a></p>
<p id="bib21">Zheng, S.Q., Palovcak, E., Armache, J.P., Verba, K.A., Cheng, Y., and Agard, D.A. (2017). MotionCor2: anisotropic correction of beam-induced motion for improved cryo-electron microscopy. Nat. Methods <i>14</i>, 331-332. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref21">View at publisher</a></p>
<p id="bib22">Zivanov, J., Nakane, T., Forsberg, B.O., Kimanius, D., Hagen, W.J., Lindahl, E., and Scheres, S.H. (2018). New tools for automated high-resolution cryo-EM structure determination in RELION-3. Elife <i>7</i>, e42166. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00561-X/sref22">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Dr. Alexander I. Sobolevsky (<a href="mailto:as4005@cumc.columbia.edu">as4005@cumc.columbia.edu</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate any new unique reagents. Further information requests about materials and reagents should be directed to the lead contact, Dr. Alexander I. Sobolevsky (<a href="mailto:as4005@cumc.columbia.edu">as4005@cumc.columbia.edu</a>).</p>
<h4>Data and code availability</h4>
<p>No new code has been generated during this study. The EMDB and PDB codes generated by this data have already been deposited and reported in our previous paper (<a href="#bib7">Green et al., 2021</a>).</p>
<h3>Acknowledgments</h3>
<p>We are thankful to Kirill Nadezhdin and Arthur Neuberger for providing MW membrane protein standards. A.I.S. was supported by the <a href="https://doi.org/10.13039/100000002">National Institutes of Health</a> (CA206573, NS083660, and NS107253) and the <a href="https://doi.org/10.13039/100000001">National Science Foundation</a> (1818086).</p>
<h3>Author contributions</h3>
<p>S.P.G., M.N.G., and M.V.Y. performed the experiments. M.N.G., S.P.G., and A.I.S. wrote the manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>