<article>
<section>
<h1 id="header">An Optimized Chromatin Immunoprecipitation Protocol for Quantification of Protein-DNA Interactions</h1>
<p><time datetime="2020-03-27">Published: March 27, 2020</time></p>
<p>Wim J. de Jonge,<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> Mariël Brok,<sup><a href="#aff1">1</a></sup> Patrick Kemmeren,<sup><a href="#aff1">1</a></sup> and Frank C.P. Holstege<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Princess Máxima Center for Pediatric Oncology, Heidelberglaan 25, 3584 CS Utrecht, the Netherlands</p>
<p id="fn1"><sup>2</sup>Technical Contact</p>
<p id="fn2"><sup>3</sup>Lead Contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:w.dejonge@prinsesmaximacentrum.nl">w.dejonge@prinsesmaximacentrum.nl</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:f.c.p.holstege@prinsesmaximacentrum.nl">f.c.p.holstege@prinsesmaximacentrum.nl</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2020.100020">10.1016/j.xpro.2020.100020</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Transcription factors are important regulators of cell fate and function. Knowledge about where transcription factors are bound in the genome is crucial for understanding their function. A common method to study protein-DNA interactions is chromatin immunoprecipitation (ChIP). Here, we present a revised ChIP protocol to determine protein-DNA interactions for the yeast <i>Saccharomyces cerevisiae</i>. We optimized several aspects of the procedure, including cross-linking and quenching, cell lysis, and immunoprecipitation steps. This protocol facilitates sensitive and reproducible quantitation of protein-DNA interactions.</p>
<p>For complete details on the use and execution of this protocol, please refer to (<a href="#bib5">de Jonge et al., 2019</a>).</p>
<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Chromatin immunoprecipitation protocol to quantify protein-DNA interactions</li>
<li>Optimized for sensitivity and robustness</li>
<li>Optimized for quantitative comparisons between experiments, e.g., in time series</li>
<li>Highlights common ChIP pitfalls, variable steps, and how to increase reproducibility</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>


<section>
<h2 id="before-you-begin">Before You Begin</h2>
<div class="timing">
<span class="timing-title">Timing:</span> 0.5–4 h</div>
<ol>
<li>If needed, prepare buffers. Make sure that there is enough of all solutions that are needed before cell growth, especially the 4.5 M Tris pH 8.0. Per sample ∼55 ml 4.5M Tris pH 8.0 is needed to properly quench the formaldehyde.</li>
<li>Make sure to use fresh formaldehyde that is &lt; 3 months old, preferably &lt; 1 month.</li>
<li>4 days before the start of the experiment (day −3), streak the strains of interest on appropriate selection plates and incubate for 3 days @ 30<sup>o</sup>C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If possible, use a strain that has an epitope tagged protein of interest (e.g. HA, FLAG, V5, etc). High quality antibodies are available for these tags, which makes the immunoprecipitation (IP) more efficient. This protocol was optimized for the use of a V5 tag and an anti-V5 antibody, nevertheless, it should be applicable for other antibodies as well. Optimal incubation conditions may differ between antibodies.</div>
<ol start="4">
<li>The day before the start of the experiment (day 0), in the morning, pick at least three colonies for each strain/condition from a fresh plate and inoculate each colony in 1.5 ml medium in a 24-well plate. Grow the cultures while shaking (230 rpm) at 30<sup>o</sup>C. At the end of the day, combine each well (1.5 ml) with 13.5 ml 30<sup>o</sup>C for a culture of 15 ml in a 100 ml flask and grow for 12–16 hours.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> All incubations are performed at 30<sup>o</sup>C and 230 rpm.</div>
<div class="note">
<span class="note-title">Note:</span> We use SC medium for all experiments, but any other appropriate medium can be used as well.</div>
<div class="note">
<span class="note-title">Note:</span> If a larger volume of starting culture is required on day 1, it is possible to combine three 1.5 ml cultures to make a culture of 20 ml by adding 15.5 ml of medium.</div>
</section>
<section>
<h2 id="key-resources-table">Key Resources Table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>V5 Tag monoclonal Antibody IP grade (mouse)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# R960-25, RRID:AB_2556564</td>
</tr>
<tr>
<td>Goat Anti-Mouse IgG (H L)-HRP Conjugate antibody</td>
<td>Bio-Rad</td>
<td>Cat# 170-6516, RRID:AB_11125547</td>
</tr>
<tr>
<td colspan="3">Chemicals, Peptides, and Recombinant Proteins</td>
</tr>
<tr>
<td>Yeast Nitrogen Base w/o AA, Carbohydrate &amp; w/AS (YNB) (Powder)</td>
<td>US Biological</td>
<td>Cat#Y2025</td>
</tr>
<tr>
<td>37% formaldehyde</td>
<td>Sigma-Aldrich</td>
<td>Cat#252549</td>
</tr>
<tr>
<td>UltraPure™ Tris Buffer (powder format)</td>
<td>Invitrogen</td>
<td>Cat#15504020</td>
</tr>
<tr>
<td>Zymolyase 20T</td>
<td>MP biomedical</td>
<td>Cat#08320921</td>
</tr>
<tr>
<td>Aprotinin</td>
<td>Sigma-Aldrich</td>
<td>Cat#A6279</td>
</tr>
<tr>
<td>Pepstatin A</td>
<td>Sigma-Aldrich</td>
<td>Cat#P4265-1MG</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>Sigma-Aldrich</td>
<td>Cat#L2884-1MG</td>
</tr>
<tr>
<td>PMSF</td>
<td>Sigma-Aldrich</td>
<td>Cat#P7626</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>Roche</td>
<td>Cat#03115852001</td>
</tr>
<tr>
<td>Broad Range Protein Marker</td>
<td>New England Biolabs</td>
<td>Cat#P7708</td>
</tr>
<tr>
<td>Dynabeads Protein G</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#10003D</td>
</tr>
<tr>
<td>RNAse A/T1</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#EN0551</td>
</tr>
<tr>
<td colspan="3">Critical Commercial Assays</td>
</tr>
<tr>
<td>QIAquick PCR Purification Kit</td>
<td>Qiagen</td>
<td>Cat#28106</td>
</tr>
<tr>
<td>TGX Stain-Free™ FastCast™ Acrylamide Kit (10%)</td>
<td>Bio-Rad</td>
<td>Cat#1610182</td>
</tr>
<tr>
<td>Western Lightning Plus-ECL, Enhanced Chemiluminescence Substrate</td>
<td>Perkin Elmer</td>
<td>Cat#NEL105001EA</td>
</tr>
<tr>
<td>High Sensitivity DNA Kit</td>
<td>Agilent</td>
<td>Cat#5067-4626</td>
</tr>
<tr>
<td>IQ SYBR Green super mix</td>
<td>Bio-Rad</td>
<td>Cat#1708886</td>
</tr>
<tr>
<td>Precision Blue™ Real-Time PCR Dye</td>
<td>Bio-Rad</td>
<td>Cat#1725555</td>
</tr>
<tr>
<td colspan="3">Software and Algorithms</td>
</tr>
<tr>
<td>ImageJ software</td>
<td><a href="#bib13">Schneider et al, 2012</a></td>
<td><a href="https://imagej.nih.gov/ij/">https://imagej.nih.gov/ij/</a></td>
</tr>
<tr>
<td>CFX Maestro software</td>
<td>Bio-Rad</td>
<td>Cat#12004110</td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>Cell Density Meter</td>
<td>VWR</td>
<td>Cat#634-0882</td>
</tr>
<tr>
<td>Heated magnetic stir plate C-MAG HS 7</td>
<td>IKA</td>
<td>Cat#0003581200</td>
</tr>
<tr>
<td>Zirconium/silica beads 0.5 mm</td>
<td>BioSpec Products</td>
<td>Cat#11079105z</td>
</tr>
<tr>
<td>Analog Disruptor Genie</td>
<td>Scientific Industries/VWR</td>
<td>Cat#444-0943</td>
</tr>
<tr>
<td>BD Microlance Hypod. naald - 23G 1 - 0,6 x 25 m</td>
<td>Becton Dickinson</td>
<td>Cat#300800</td>
</tr>
<tr>
<td>Cooled micro centrifuge for 1.5-2.0 ml tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Cooled tabletop centrifuge for 50 ml tubes</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Vortex</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Bioruptor pico microtubes</td>
<td>Diagenode</td>
<td>Cat#C30010016</td>
</tr>
<tr>
<td>Bioruptor pico sonicator device</td>
<td>Diagenode</td>
<td>Cat#B01060010</td>
</tr>
<tr>
<td>Thermoshaker (preferred) or heat block</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Rotating wheel (one at 4<sup>o</sup>C and one at 19-22<sup>o</sup>C)</td>
<td>N/A</td>
<td>N/A</td>
</tr>
<tr>
<td>Bioanalyzer</td>
<td>Agilent</td>
<td>Cat#G2939B</td>
</tr>
<tr>
<td>DynaMag-2 magnetic stand</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#12321D</td>
</tr>
<tr>
<td>1.5 ml DNA LoBind Tube</td>
<td>Eppendorf</td>
<td>Cat#0030108051</td>
</tr>
<tr>
<td>CoolRack XT PCR384 for 384-wel</td>
<td>Corning</td>
<td>Cat#432055</td>
</tr>
<tr>
<td>Hard-Shell® Thin-Wall 384-Well Skirted PCR Plates, white</td>
<td>Bio-Rad</td>
<td>Cat#HSP3805</td>
</tr>
<tr>
<td>Microseal® 'B' PCR Plate Sealing Film</td>
<td>Bio-Rad</td>
<td>Cat#msb1001</td>
</tr>
<tr>
<td>CFX384 Touch Real-Time PCR<br>Detection system</td>
<td>Bio-Rad</td>
<td>Cat#698-2901</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and Equipment</h2>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol uses a Bioanalyzer to assess the extent of DNA fragmentation. Alternatives are for example the TapeStation (Agilent) or LabChip (PerkinElmer). The TapeStation offers more flexibility in the number of samples that can be analyzed at once and the LabChip can process more samples at the same time. The use of a Bioanalyzer (or equivalent) is preferred over an agarose gel since the Bioanalyzer has a better resolution in assessing the exact fragmentation pattern. An agarose gel can distinguish large differences in shearing (500–1,000 bp fragments vs 200–300 bp fragments), but with an agarose gel it is hard to distinguish between more subtle differences (200–300 bp fragments vs 300–400 bp fragments). In addition, the Bioanalyzer (or equivalent) is also more sensitive and only requires a small fraction of material compared to what is needed to visualize fragmentation using an agarose gel. Nevertheless, if analysis by Bioanalyzer (or equivalent) is not possible, the fragmentation can also be assessed using agarose gel electrophoresis.</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> This protocol describes the qPCR procedure using a CFX384 Touch Real-Time PCR Detection system and corresponding reagents, in 10 μl reaction volumes. Any other qPCR machine can be used as well. We recommend using a 384-well format, since this allows examination of multiple IP-input pairs with several primer pairs in the same plate.</div>
<table>
<caption>FA lysis buffer</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Stock Concentration</th>
<th>Add to 500 ml</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES-KOH pH 7.5</td>
<td>50 mM</td>
<td>1 M</td>
<td>25 ml</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>5 M</td>
<td>15 ml</td>
</tr>
<tr>
<td>EDTA pH 8.0</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>1 ml</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1 % (v/v)</td>
<td>10% (v/v)</td>
<td>50 ml</td>
</tr>
<tr>
<td>Na-deoxycholate</td>
<td>0.1% (w/v)</td>
<td>10% (w/v)</td>
<td>5 ml</td>
</tr>
<tr>
<td>SDS</td>
<td>0.1% (w/v)</td>
<td>10% (w/v)</td>
<td>5 ml</td>
</tr>
<tr>
<td>MilliQ</td>
<td> </td>
<td> </td>
<td>399 ml</td>
</tr>
</tbody>
</table>
<p>We filter sterilize the FA lysis buffer. Store at 4<sup>o</sup>C. Right before use, add 30 μl Aprotinin, 1 μl Pepstatin, 1 μl Leupeptin, and 10 μl PMSF per 1 ml of buffer (see Other Solutions below).</p>
<table>
<caption>4.5M Tris pH 8.0</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Stock Concentration</th>
<th>Add to 1 L</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td> </td>
<td>545.13 g</td>
</tr>
<tr>
<td>HCl</td>
<td>12.1 M</td>
<td>∼211.5 ml</td>
</tr>
<tr>
<td>MilliQ</td>
<td> </td>
<td>Up to 1 L</td>
</tr>
</tbody>
</table>
<p>We filter sterilize the 4.5 M Tris buffer, although autoclaving should also be possible. Store at 19<b>–</b>22°C.</p>
<div class="note">
<span class="note-title">Note:</span> Making 4.5M Tris pH 8.0 is challenging, because this concentration nearly reaches the solubility limit of Tris. We recommend making 2 liters at once. For each 100 ml culture, cross-linked with 2% formaldehyde, almost 55 ml of 4.5M Tris pH 8.0 is used to quench (final concentration: 1.5M). This means that to quench 18 samples, nearly 1 liter of 4.5M Tris pH 8.0 is needed. When dissolving Tris, a substantial amount of HCl must be added to fully dissolve Tris (∼211.5 ml per liter), which should be taken into account when adding MQ.</div>
<table>
<caption>ChIP wash buffer 1 (optional)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Stock Concentration</th>
<th>Add to 50 ml</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES-KOH pH 7.5</td>
<td>50 mM</td>
<td>1 M</td>
<td>2.5 ml</td>
</tr>
<tr>
<td>NaCl</td>
<td>500 mM</td>
<td>5 M</td>
<td>5 ml</td>
</tr>
<tr>
<td>EDTA pH 8.0</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>0.1 ml</td>
</tr>
<tr>
<td>Triton X-100</td>
<td>1 % (v/v)</td>
<td>10% (v/v)</td>
<td>5 ml</td>
</tr>
<tr>
<td>Na-deoxycholate</td>
<td>0.1% (w/v)</td>
<td>10% (w/v)</td>
<td>0.5 ml</td>
</tr>
<tr>
<td>SDS</td>
<td>0.1% (w/v)</td>
<td>10% (w/v)</td>
<td>0.5 ml</td>
</tr>
<tr>
<td>MilliQ</td>
<td> </td>
<td> </td>
<td>36.4 ml</td>
</tr>
</tbody>
</table>
<p>Store at 19–22°C. Preferably add right before use 30 μl Aprotinin, 1 μl Pepstatin, 1 μl Leupeptin, and 10 μl PMSF per 1 ml of buffer.</p>
<table>
<caption>ChIP wash buffer 2 (optional)</caption>
<thead>
<tr>
<th>Reagent</th>
<th>Final Concentration</th>
<th>Stock Concentration</th>
<th>Add to 50 ml</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris pH 8.0</td>
<td>10 mM</td>
<td>1 M</td>
<td>0.5 ml</td>
</tr>
<tr>
<td>LiCl</td>
<td>250 mM</td>
<td>5 M</td>
<td>2.5 ml</td>
</tr>
<tr>
<td>Nonidet P-40∗</td>
<td>0.5% (v/v)</td>
<td>10% (v/v)</td>
<td>2.5 ml</td>
</tr>
<tr>
<td>Na-deoxycholate</td>
<td>0.5% (w/v)</td>
<td>10% (w/v)</td>
<td>2.5 ml</td>
</tr>
<tr>
<td>EDTA pH 8.0</td>
<td>1 mM</td>
<td>0.5 M</td>
<td>0.1 ml</td>
</tr>
<tr>
<td>MilliQ</td>
<td> </td>
<td> </td>
<td>41.9 ml</td>
</tr>
</tbody>
</table>
<p>Store at 19<b>–</b>22°C. Preferably add 30 μl Aprotinin, 1 μl Pepstatin, 1 μl Leupeptin, and 10 μl PMSF per 1 ml of buffer, right before use.<br> <sup>*</sup>Nonidet P-40 is discontinued. Substitutes are available, but we did not test these.</p>
<table>
<caption>Other solutions</caption>
<thead>
<tr>
<th>Name</th>
<th>Reagents</th>
</tr>
</thead>
<tbody>
<tr>
<td>TBS</td>
<td>150 mM NaCl, 10 mM Tris pH 7.5</td>
</tr>
<tr>
<td>Aprotinin</td>
<td>3–21 TIU/mL as purchased from supplier. Store at 4<sup>o</sup>C.</td>
</tr>
<tr>
<td>Pepstatin A</td>
<td>1 mg/ml pepstatin A in 100% methanol (1.51 mM). Store at −20<sup>o</sup>C.</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>1 mg/ml leupeptin in MQ (2.10 mM). Store at −20<sup>o</sup>C.</td>
</tr>
<tr>
<td>PMSF</td>
<td>200 mM Phenylmethanesulfonyl fluoride (PMSF) in isopropanol. Heat to 37<sup>o</sup>C to dissolve and store at −20<sup>o</sup>C.</td>
</tr>
<tr>
<td>Proteinase K</td>
<td>10 μg/μl in TE (10 mM Tris pH 8.0, 1 mM EDTA pH 8.0)</td>
</tr>
<tr>
<td>PBS</td>
<td>137 mM NaCl, 2.7 mM KCl, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, 1.47 mM KH<sub>2</sub>PO<sub>4</sub> <br>1 mM CaCl, 0.5 mM MgCl, pH adjusted to 7.3 using HCl</td>
</tr>
<tr>
<td>PBS-T</td>
<td>PBS + 0.02% Tween-20 (v/v)</td>
</tr>
<tr>
<td>TE/SDS</td>
<td>10 mM Tris pH 8.0, 1 mM EDTA pH 8.0, 1% SDS (w/v)</td>
</tr>
<tr>
<td>BSA in TBS-T</td>
<td>Dissolved to 10 mg/ml in TBS-T (150 mM NaCl, 10 mM Tris pH 7.5, 0.05% Tween-20)</td>
</tr>
<tr>
<td>5X sample buffer (optional)</td>
<td>5% SDS (w/v), 200 mM Tris pH 6.8, 25% glycerol (v/v), 1.43 M β-mercaptoethanol, 0.032% bromophenol blue (w/v)</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-Step Method Details</h2>
<h3>Growth and <i>In Vivo</i> Cross-Linking – Day 1</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6–8 h</div>
<p>Cells are grown and cross-linked using formaldehyde to trap protein-DNA interactions.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol is an optimization of a previous protocol (<a href="#bib1">Aparicio et al., 2004</a>).</div>
<div class="note">
<span class="note-title">Note:</span> Use an appropriate number of replicates. Since there are many steps in the protocol, the chances of variation arising among samples are substantial. We recommend to always use at least three biological replicates, if possible. In case of a tagged protein, the ideal biological replicates are independently transformed strains. These should be independently streaked and grown. If not available, different colonies should be grown independently as biological replicates.</div>
<div class="note">
<span class="note-title">Note:</span> To control for artefactual binding of the protein of interest, take along a non-cross-linking sample. If there is no binding observed in this control, this indicates that there is no artefactual binding of the protein of interest during the procedure. Treat this culture exactly the same as the cross-linked samples but omit the formaldehyde addition step. This control should be taken along every first time a new protein of interest is tested. When using tagged strains, it is also good practice to take along an untagged (wild-type) control.</div>
<ol>
<li>
<i>Day 1.</i> Measure the optical density (OD) of the culture (1:50) and dilute the cultures in 110 ml pre warmed medium to an OD of 0.20 (∼ 0.5∗10<sup>7</sup> cells/ml) in a 500 ml Erlenmeyer, mix well and measure the starting OD of the culture.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> There is an extra 10 ml for sampling OD. Before the addition of formaldehyde, remove what is left of this, to make sure that there is exactly 100 ml culture left.</div>
<div class="note">
<span class="note-title">Note:</span> Since timing of formaldehyde cross-linking is crucial, and this is performed using magnetic stir plates, it is best to start the cultures in such a way that only a few samples are ready at the same time. If, for example, a time course experiment is performed with replicates, having 1 hour in between each replicate time course will provide enough time to finish harvesting the first, before the second has to be processed. We strongly recommend harvesting the cultures with two people at the same time to speed up the process.</div>
<ol start="2">
<li>Grow yeast until OD 0.8 (2 doublings), this is equivalent to approximately 2∗10<sup>7</sup> cells/ml.</li>
<li>Before addition of formaldehyde, put the cultures on a heated (30<sup>o</sup>C) stir plate and add a (clean!) magnetic stir bar. Add 5.7 ml of 37% formaldehyde to a final concentration of 2% to the cultures. Incubate outside the incubator on the heated (30<sup>o</sup>C) stir plate under agitation/stirring for 5 min.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Use a setting for the stir plate that allows for quick and proper mixing. Be careful not to mix too vigorously, however, especially avoiding foaming of the sample. For example, with the stir plate IKA C-MAG HS 7 and a stir bar of approximately 6 cm, set the motility to 1.</div>
<div class="note">
<span class="note-title">Note:</span> It is useful to make aliquots (in 50 ml tubes) of formaldehyde and 4.5M Tris pH 8.0 in advance. The correct volume of formaldehyde and 4.5M Tris pH 8.0 can be easily poured into the culture. Be careful to pour straight into the culture without touching the walls of the Erlenmeyer. If possible, it is best to perform this step in a fume hood.</div>
<div class="note">
<span class="note-title">Note:</span> To accurately control timing of formaldehyde, the mixing is done using a heated magnetic stir plate and stir bar. Before the first use, check, using a thermometer, which temperature setting of the stir plate will keep the culture at 30<sup>o</sup>C during the mixing.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Never use formaldehyde that is &gt; 3 months old, preferably use formaldehyde &lt; 1 month old. We recommend purchasing small bottles of formaldehyde and finishing them within a few (2<b>–</b>3) weeks after the first use, with minimal opening/closing of the bottle. Formaldehyde polymerizes when exposed to oxygen, which will compromise the cross-linking efficiency.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The timing and final concentration of formaldehyde used for cross-linking is crucial, since the extent of cross-linking strongly affects ChIP signals and downstream processing steps, see also (<a href="#bib5">de Jonge et al., 2019</a>). The optimal cross-linking parameters (concentration and timing) depend on the protein of interest. We recommend keeping the cross-linking time as short as possible, but cross-linking up to 30 minutes may be used, depending on the formaldehyde concentration.</div>
<ol start="4">
<li>Add Tris 4.5 M pH 8.0 to a final concentration of 1.5 M to the cultures by adding 52.9 ml to stop the cross-linking reaction. Incubate the culture and Tris for 1 min using the same agitation as the cross-linking.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For efficient quenching, use Tris pH 8.0 in a ∼2.25 fold molar excess. For 2% formaldehyde, use a final concentration of 1.5M Tris, and for 1% formaldehyde, use a final concentration of 750 mM Tris.</div>
<div class="critical">
<span class="critical-title">Critical:</span> To properly control the timing of cross-linking, the formaldehyde must be quenched efficiently. Many protocols use sub-stoichiometric levels (e.g. 125 mM / 250 mM) of glycine to quench. This is very inefficient, and cross-linking will continue in the presence of glycine, see also (<a href="#bib5">de Jonge et al., 2019</a>; <a href="#bib18">Zaidi et al., 2017</a>). Failure to efficiently quench formaldehyde can be a major source of variation in ChIP signals! Tris is a much more efficient quencher than the commonly used glycine (<a href="#bib4">Hoffman et al., 2015</a>; <a href="#bib5">de Jonge et al., 2019</a>; <a href="#bib15">Sutherland et al., 2008</a>). Although concerns have been raised that Tris might reverse the formaldehyde cross-links, we have found no evidence of this using short incubation times (<a href="#bib5">de Jonge et al., 2019</a>).</div>
<ol start="5">
<li>Split each culture over 3x 50 ml tubes and spin at 3,220g (4,000 rpm) for 3 min at 4<sup>o</sup>C (Eppendorf 5810 R).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> keep the samples cold in between the centrifugation steps, by putting them on ice. Try to be as quick as possible. Even though quenching with Tris is efficient and does not reverse cross-links, it is best to keep the time that samples spend on ice to a minimum.</div>
<ol start="6">
<li>Wash pellets by resuspending the first pellet in 10 ml ice-cold TBS pH 7.5. Add the resuspended pellet to the second pellet and resuspend. Repeat this with the third pellet to combine the three pellets (final volume 10 ml) and spin down for 3 min at 3,220g (4,000 rpm) at 4<sup>o</sup>C (Eppendorf 5810 R).</li>
<li>Remove supernatant, resuspend pellet in 1 ml ice-cold MQ, and transfer to a 2.0 ml Eppendorf tube.</li>
<li>Spin down at 3,381g (6,000 rpm) for 20 sec in a centrifuge (Eppendorf 5424).</li>
<li>Remove supernatant, freeze pellet in liquid nitrogen and store at –80<sup>o</sup>C.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The pellets can be stored at −80<sup>o</sup>C for at least several months.</div>
<h3>Bead Beating Procedure – Day 2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3–4 h</div>
<p>Cells are lysed and chromatin is fragmented. Fragmentation is needed to solubilize chromatin and to increase the resolution of ChIP.</p>
<div class="note">
<span class="note-title">Note:</span> This protocol was optimized for 100 ml of mid-log cells, OD = 0.8 (2∗10<sup>7</sup> cells/ml). Preferably process 6 samples at once to increase speed and prevent proteolytic degradation of the samples. If need be, up to 12 samples can be processed at the same time. Be aware that with increasing number of samples (and thus processing time) the risk of proteolytic degradation of the protein of interest also increases.</div>
<div class="note">
<span class="note-title">Note:</span> The FA lysis buffer is kept on ice and four different protease inhibitors are added. Approximately 1.9 ml FA lysis buffer per sample is needed. Add the following protease inhibitors as listed in <a href="#tbl1">Table 1</a>.</div>
<table id="tbl1">
<caption>Table 1. Protease Inhibitors to Add to 1 ml of FA Lysis Buffer</caption>
<thead>
<tr>
<th>Name</th>
<th>Amount to Add per ml</th>
<th>Note</th>
</tr>
</thead>
<tbody>
<tr>
<td>Aprotinin</td>
<td>30 μl</td>
<td>Stock at 4<sup>o</sup>C 33.3x</td>
</tr>
<tr>
<td>Leupeptin</td>
<td>1 μl</td>
<td>Stock in MQ −20<sup>o</sup>C, 1,000x</td>
</tr>
<tr>
<td>Pepstatin A</td>
<td>1 μl</td>
<td>Stock in Methanol −20<sup>o</sup>C, 1,000x</td>
</tr>
<tr>
<td>PMSF</td>
<td>10 μl</td>
<td>Stock in isopropanol −20<sup>o</sup>C, 100x</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> Keep the samples on ice all the time and pre-chill all tubes on ice! Work as fast as possible to prevent degradation.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Protein degradation during the procedure can have detrimental effects on the ChIP signal. It is therefore important that fresh protease inhibitors are added at the steps indicated in the protocol.</div>
<ol start="10">
<li>Add 500 μl of zirconium beads (0.5 μm) to 2.0 ml screw-cap tubes. Measure the right amount of beads using a 0.5 ml Eppendorf tube. The zirconium beads will disrupt the cell wall upon bead beating in the genie disruptor.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Pre-cool centrifuges for both 2.0 ml and 50 ml tubes (Eppendorf 5810 R and 5424 R).</div>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> the cells may be lysed by making spheroplasts using zymolyase, which may be less cumbersome. However, zymolyase preparations are known to have proteases present, which may cause extensive proteolytic degradation of the protein of interest (<a href="#bib5">de Jonge et al., 2019</a>). If lysis with zymolyase is preferred, it would be best to add protease inhibitors to all wash and digestion buffers, and to wash extensively using 1M Sorbitol to remove the majority of proteases. However, caution is required to ensure that spheroplasts are not lysed during the washing steps.</div>
<ol start="11">
<li>Add the protease inhibitors to the FA lysis buffer right before you add the buffer to the cells. For 6 cell pellets, add to 14.5 ml of FA lysis buffer: 450 μl aprotinin, 15 μl pepstatin, 15 μl leupeptin first and then 150 μl PMSF.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> PMSF is a highly toxic neurotoxin. Be very careful when preparing the stock, and always add this in a fume hood. To add, take the stock out of the −20<sup>o</sup>C freezer and warm the tube by rubbing it between your hands (wearing gloves!). PMSF crystallizes when cold, it will need to be re-solubilized. Alternate between a gentle vortex and rubbing until re-solubilized. When you add the PMSF, mix immediately to prevent precipitation. Add PMSF last as it will lose activity rapidly in an aqueous environment.</div>
<ol start="12">
<li>Resuspend the frozen pellet carefully in 900 μl of FA lysis buffer with protease inhibitors and transfer to a 2.0 ml screw-cap tube with beads very slowly, to prevent any air getting trapped in the beads.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> take a 5 μl sample for western here (pre-bead beat sample). Add 5 μl FA lysis buffer, 2.5 μl of 5x sample buffer and boil for 30 min @ 95<sup>o</sup>C.</div>
<div class="note">
<span class="note-title">Note:</span> Try to prevent trapping air between the beads, as this will lead to foaming. If there is any air trapped in the beads: close tube and turn it upside down to release trapped air.</div>
<ol start="13">
<li>Add more FA lysis buffer containing protease inhibitors to fill the tube completely (∼ 1ml), keeping as little air as possible and close the tube carefully.</li>
<li>Disrupt cells with a genie disrupter at 4°C by bead beating 7 times 3 min (2,850 rpm). Put samples on ice for 1 min between each run.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During this time pre-chill the sonicator and prepare/label tubes needed for the next step!</div>
<ol start="15">
<li>Prepare 15 ml tubes containing a 1 ml pipette tip (cut the end of the 1 ml tips to avoid blockage, <a href="#fig1">Figure 1</a>A).</li>
</ol>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. How to Prepare 15-ml Tubes for Recovering Chromatin</div>
<p>(A) Cut the end of a 1 ml pipette tip and place in a labelled 15 ml tube.</p>
<p>(B) Burn a hole in the bottom of the 2-ml screw cap tube containing the chromatin using a hot 23G needle and quickly place in the 15 ml tube, on top of the 1 ml pipette tip.</p>
<p>(C) Burn a second hole in the top of the screw-cap tube, to facilitate the extraction. Although it is not shown in the figure, always keep all the tubes on ice.</p>
</figcaption>
</figure>
<ol start="16">
<li>Recover each extract by burning a hole in the bottom of the screw-cap tube with a hot 23G needle and quickly placing it in the 15 ml tube on top of the 1 ml pipette tip (<a href="#fig1">Figure 1</a>B). Burn a second hole in the top of the tube to facilitate release (<a href="#fig1">Figure 1</a>C). Centrifuge this combination at 201g (1,000 rpm) for 1 minute at 4<sup>o</sup>C (Eppendorf 5810 R).</li>
<li>Transfer the complete lysates to a 2.0 ml Eppendorf tube and spin down at 1,503g (4,000 rpm) for 2 min at 4°C (Eppendorf 5424 R).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> This step will remove the majority of the cell debris that could interfere in the subsequent sonication. The supernatant should contain the chromatin.</div>
<ol start="18">
<li>Transfer all supernatant to a new 2.0 ml Eppendorf tube (take as much as possible (∼1,400 μl))</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> With longer cross-linking times, there is an increased chance of chromatin co-precipitating with the pellet. If this happens, it should be visible as a vague white layer on top of the pellet. Make sure to transfer this chromatin pellet together with the supernatant, otherwise the majority of the (cross-linked) DNA will be lost. What counts as “long cross-linking times” depends on the concentration of formaldehyde used. With 1% formaldehyde we noticed the co-precipitation of the chromatin when cross-linking for 20 or 30 minutes, but not with cross-linking for 10 minutes. We have not checked this using 2% formaldehyde, but extrapolating we expect that this could happen when cross-linking for 10 minutes or longer.</div>
<div class="optional">
<span class="optional-title">Optional:</span> a second centrifugation step (21,130g, 15,000 rpm, 5424 R Eppendorf) may be added here to separate the chromatin from unbound / cytosolic proteins. This chromatin pellet should be resuspended in the same volume of FA lysis buffer (1,400 μl). The chromatin pellet is usually difficult to resuspend. A 23G-needle may be used to facilitate resuspending. We usually omit this step, but when the chromatin and soluble proteins are not separated, this means that unbound TF is mixed with the chromatin during all subsequent steps, which could potentially lead to <i>in vitro</i> binding to the DNA. When adding this step, bear in mind that with lower cross-linking times (5 min or less) the chromatin may not fully precipitate, and a part of the chromatin may be lost. Keeping a part of the supernatant from this step for Western / DNA quantitation will show the efficiency of chromatin recovery.</div>
<ol start="19">
<li>Transfer 2x 300 μl to 1.5 ml Bioruptor Pico sonication tubes (300 μl per tube, this is about half of the total volume). Only half of the sample is used to make sure that the sonication and/or IP can be repeated if needed. Snap freeze the remainder of the cross-linked chromatin and store at −80<sup>o</sup>C.</li>
<li>Add fresh protease inhibitors. Make a mix (for 12 sonication tubes) of 63 μl aprotinin, 2.1 μl leupeptin and 2.1 μl pepstatin. First add 4.8 μl to all 300 μl samples in sonication tubes and then add 3 μl of PMSF (in fume hood), mix immediately by very gentle vortexing, making sure the extract does not touch the lid!</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> take a 10 μl sample for western here (post lysis, pre sonication). Add 2.5 μl of 5x sample buffer and boil for 30 min @ 95<sup>o</sup>C.</div>
<ol start="21">
<li>Shear for 10 or 8 cycles 15’’ on, 30’’ off (<a href="#tbl2">Table 2</a>) using a Bioruptor Pico that is connected to a water cooler, which is set to 4<sup>o</sup>C.</li>
</ol>
<table id="tbl2">
<caption>Table 2. Suggested Number of Cycles (15’’ on, 30’’ off) to Use for Shearing the Samples with a Bioruptor Pico, Depending on the Cross-Linking Time</caption>
<thead>
<tr>
<th>Time of Cross-Linking</th>
<th>Number of Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>0 min</td>
<td>10</td>
</tr>
<tr>
<td>5 min</td>
<td>10</td>
</tr>
<tr>
<td>10 min</td>
<td>8</td>
</tr>
<tr>
<td>20 min</td>
<td>8</td>
</tr>
</tbody>
</table>
<p>These are the suggested values for cross-linking with 2% formaldehyde.</p>
<div class="note">
<span class="note-title">Note:</span> The parameters of sonication are highly dependent on the device used and need to be optimized for each device. The size range that should be aimed for is the same for all shearing devices (see note at step 32).</div>
<div class="note">
<span class="note-title">Note:</span> During sonication the sample heats up due to the physical force that is exerted on the chromatin. Using short “on” times will keep the extent of heating to a minimum, which helps to prevent protein degradation. Large proteins (&gt; 80<b>–</b>100 kDa) are particularly sensitive to this degradation (<a href="#bib5">de Jonge et al., 2019</a>; <a href="#bib8">Pchelintsev et al., 2016</a>). If the degradation is very severe, enzymatic fragmentation with for example micrococcal nuclease (MNase) may be used instead of sonication.</div>
<ol start="22">
<li>Spin down sample for 20 min at max speed (21,130g, 15,000 rpm, 5424 R Eppendorf) at 4<sup>o</sup>C.</li>
<li>Combine the two supernatants per sample in a new 2.0 ml Eppendorf tube.</li>
<li>Take 20 μl for QC and take 1 μl for Bradford analysis (dilute 1:41 by adding it to 40 μl of MQ). Expect values around 2<b>–</b>5 μg/μl.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> also take 10 μl for a Western (post sonication), add 2.5 μl of 5x sample buffer and boil for 30 min @ 95<sup>o</sup>C. (For the Western load 10 μl on the gel). When running a Western, stain with an antibody that is suitable for Westerns.</div>
<ol start="25">
<li>Snap freeze the sample in liquid nitrogen and store at −80<sup>o</sup>C.</li>
<li>Continue with reverse cross-linking for QC.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The chromatin can be stored at −80<sup>o</sup>C for several months.</div>
<h3>Reversing the Cross-Link for QC – Day 2 &amp; Day 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h (day 3)</div>
<p>A small portion of the sheared DNA (from bead beating procedure – step 26) is de-cross-linked, cleaned and analyzed using a Bioanalyzer to check the efficiency of the shearing.</p>
<ol start="27">
<li>Put together:</li>
</ol>
<ul>
<li>20 μl chromatin extract.</li>
<li>TE/SDS to 95 μl.</li>
</ul>
<ol start="28">
<li>Reverse cross-link by incubating 12<b>–</b>16 hours at 65°C in a thermoshaker (800 rpm).</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> continue with cleaning of the DNA the next day</div>
<ol start="29">
<li>The next morning (<i>Day 3</i>): Add 5 μl RNase A/T1 Mix and incubate for 30 min at 37°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A mix of RNAse A + RNAse T1 is used, because RNAse A treatment alone was often insufficient to digest all RNA, and a peak of small degraded RNA species was still visible in the Bioanalyzer electropherogram.</div>
<ol start="30">
<li>Add 40 μl of proteinase K (10 μg/μl) and incubate for 2 hours at 37°C.</li>
<li>Clean up with the Qiagen PCR purification kit (all steps performed at 19–22°C):</li>
</ol>
<ol type="a">
<li>Add 5 volumes of PB buffer (700 μl for 140 μl sample) and mix well.</li>
<li>Add 500 μl sample to spin column.</li>
<li>Spin for 1 min at max speed (21,130g, 15,000 rpm, 5424 R Eppendorf).</li>
<li>Discard flow-through and add the remaining sample (340 μl).</li>
<li>Spin for 1 min at max speed, discard flow-through.</li>
<li>Wash 3 times with 500 μl of PE and spin down 1 min at max speed, discard flow-through between washes.</li>
<li>Transfer column to new 2.0 ml tube without lid and spin down at max speed for 1 min (to remove last bit of PE buffer).</li>
<li>Elute by adding 40 μl of EB, incubate for 1 min, and spin down at max speed in 1.5 ml Eppendorf tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> A Qiagen PCR cleanup kit is used to clean up QCs, as well as the IP and input samples. The recovery of this kit is about 70%. We tried several other cleanups (Zymoclear ChIP kit, Ampure beads, phenol chloroform isolation and Qiagen MinElute DNA kit), but the Qiagen PCR cleanup kit was equal or better in both recovery and reproducibility. If preferred, any alternative cleanup strategy may be used as well, as long as it provides consistent recoveries.</div>
<ol start="32">
<li>Check DNA fragment size by loading 1 μl of purified DNA on Bioanalyzer with a High-Sensitivity DNA Chip (or equivalent).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The electropherogram of the Bioanalyzer should show a peak between 200<b>–</b> 300 bp, as this fragment size is optimal for sequencing (<a href="#fig2">Figure 2</a>). Depending on the application, longer fragment sizes may also be acceptable. If there is still a large amount of longer fragments detected, shearing of the chromatin can be repeated for a few additional cycles, but care should be taken that this can also increase proteolytic degradation of these samples. When shearing is repeated, fresh protease inhibitors should be added.</div>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Electropherogram Showing the Distribution of DNA Fragments of a Properly Sheared Sample</div>
<p> </p>
</figcaption>
</figure>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> The chromatin can be stored at −80<sup>o</sup>C for several months.</div>
<h3>Immunoprecipitation Using Magnetic Beads – Day 3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–5 h</div>
<p>The protein of interest is recognized by an antibody that binds to magnetic beads. Non-specific binding is largely washed away with different wash buffers. First bind the antibody to chromatin (steps 33<b>–</b>37). While incubating, prepare the magnetic beads (steps 38<b>–</b>47). Next, bind chromatin bound antibody to the beads (step 48). Finally, wash the beads (steps 49<b>–</b>52) before proceeding with reversal of cross-linking and cleanup (steps 53<b>–</b>62).</p>
<ol start="33">
<li>Thaw the chromatin on ice and take a 20 μl aliquot for input control. Keep on ice while processing the IP samples. Continue with the input samples at step 58.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> During this step make a mix of (per 6 samples): 210 μl Aprotinin, 7 μl Leupeptin and 7 μl Pepstatin.</div>
<ol start="34">
<li>Transfer 450 μl of the chromatin to a (cold) 1.5 ml Eppendorf tube.</li>
<li>Add 16 μl of the protease inhibitor mix and subsequently add 5 μl PMSF (in fumehood). Mix immediately by vortexing gently.</li>
<li>Add 1.2 μg (1 μl) of antibody to the chromatin extract.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The amount of antibody that is needed is dependent on the antibody and the abundance of the target. The optimal amount of antibody can best be determined empirically. 1.2 μg antibody worked well for all V5-tagged proteins that we tested.</div>
<ol start="37">
<li>Incubate with rotation (20 rpm) for 2h at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The optimal incubation time depends on the antibody used; this can best be determined empirically.</div>
<ol start="38">
<li>During the incubation of the chromatin with the antibody, the magnetic beads can be prepared: Resuspend the magnetic beads (Dynabeads) in the vial (vortex &gt; 30 sec).</li>
<li>Transfer 25 μl (0.75 mg) beads per ChIP to a 1.5 ml Eppendorf tube.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> when doing many ChIPs, close the bottle and vortex again for a few seconds after filling a few samples to keep the beads well mixed.</div>
<ol start="40">
<li>Place the tube on the magnet (DynaMag-2) to separate the beads from the solution and remove the supernatant.</li>
<li>Wash once with 500 μl PBS-T (PBS/0.02% Tween-20) by adding the PBS-T followed by gentle vortexing.</li>
<li>Place the tube on the magnet to separate the beads from the solution and remove the supernatant.</li>
<li>Add 400 μl PBS + 25 μl BSA (10mg/ml in TBS-T).</li>
<li>Incubate with rotation (20 rpm) for ∼2h at 4°C.</li>
<li>About 5-10 minutes before the incubation of the chromatin + antibody is ready. Place the tube on the magnet to separate the beads from the solution and remove the supernatant</li>
<li>Wash once with 500 μl PBS-T.</li>
<li>Place the tube on the magnet to separate the beads from the solution and remove the supernatant. Close the tube to prevent the beads from drying up.</li>
<li>After 2 hours of incubation at 4<sup>o</sup>C add 5 μl PMSF to the sample containing the antigen (CE 450 μl + V5 antibody), add to the beads and gently vortex to mix the beads with the chromatin. Incubate with rotation (20 rpm) for 20 min at 19<b>–</b>22°C to allow the antibody to bind to the beads.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Depending on the antibody this incubation may need to be longer.</div>
<ol start="49">
<li>Spin down briefly and place the tube on the magnet.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> Transfer the supernatant to a clean tube for further analysis, if desired. This can be used, for example, to examine the fraction of protein that was not immunoprecipitated.</div>
<ol start="50">
<li>Wash the bead complex 2 times using 200 μl PBS for each wash. Separate on the magnet between each wash, remove supernatant and resuspend by gentle vortexing.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> if the protein of interest is particularly sensitive to proteolytic degradation add the protease inhibitors also to all the ChIP wash buffers. Per 1 ml of wash buffer add: 30 μl aprotinin, 1 μl pepstatin, 1 μl leupeptin and 10 μl PMSF.</div>
<div class="optional">
<span class="optional-title">Optional:</span> If the protein of interest is a particularly strong binder, more stringent washes may help to remove non-cross-linked proteins. Use wash buffer 1 and/or wash buffer 2 (2<b>–</b>3 washes of 200 μl each). Taking along a non-cross-linking control should show whether a protein can bind to DNA without cross-linking and whether the more stringent washes are sufficient. More stringent washes will significantly reduce the background signal, but depending on the affinity of the antibody may also reduce the specific ChIP signal.</div>
<ol start="51">
<li>Wash the beads 1 time using 200 μl PBS-T. Resuspend by gentle vortexing, separate on the magnet after washing and remove supernatant.</li>
<li>Resuspend the beads in 100 μl PBS-T and transfer the bead suspension to a clean LoBind Eppendorf tube. This is recommended to avoid co-elution of proteins bound to the tube wall.</li>
<li>Proceed to Elution and reverse cross-linking.</li>
</ol>
<h3>Elution and Reverse Cross-Linking – Day 3 &amp; Day 4</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>The protein-DNA cross-links are reversed by heating and the DNA is eluted from the beads.</p>
<ol start="54">
<li>Place the tube (from step 52 in "Immunoprecipitation using magnetic beads") on the magnet and remove the supernatant.</li>
<li>Add 98 μl TE/SDS.</li>
<li>Resuspend the bead-antibody-antigen complex by gentle vortexing.</li>
<li>Reverse the cross-links for 12<b>–</b>16 hours using shaking (800 rpm) at 65<sup>o</sup>C.</li>
<li>For input samples, add 75 μl TE/SDS to the 20 μl extract (from Immunoprecipitation using magnetic beads – step 33), mix and incubate using shaking (800 rpm) at 65<sup>o</sup>C as well.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> continue with cleaning of the DNA the next day</div>
<ol start="59">
<li>The next morning (<i>Day 4</i>): add 2 μl RNAse A/T1 mix to the IP samples and 5 μl to the input samples and incubate 30 min @ 37°C.</li>
<li>Add 40 μl of proteinase K (10 mg/ml in TE), and incubate for 2 hours at 37°C.</li>
<li>Place the tube on the magnet and transfer the supernatant containing eluted DNA (IPs) to a clean LoBind-tube.</li>
<li>Clean up with the Qiagen PCR purification up kit, similar to the cleanup for QC.</li>
</ol>
<ol type="a">
<li>Add 5 volumes of PB buffer (700 μl for 140 μl sample) and mix well</li>
<li>Add 500 μl sample to spin column.</li>
<li>Spin for 1 min at max speed (21,130g, 15,000 rpm, 5424 R Eppendorf).</li>
<li>Discard flow-through and add the remaining sample (340 μl).</li>
<li>Spin for 1 min at max speed, discard flow-through.</li>
<li>Wash 3 times with 500 μl of PE and spin down at max speed, discard flow-through between washes.</li>
<li>Transfer column to an empty 2.0 ml tube without lid and spin down at max speed for 1 min to remove last bit of PE buffer.</li>
<li>Elute by adding 40 μl of EB, incubate for &gt;1 min, and spin down at max speed in a new 1.5 ml Lobind tube.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> The DNA can also be eluted in 30 μl of EB for more concentrated DNA.</div>
<ol start="63">
<li>Quantify the binding using qPCRs and/or proceed to make sequencing libraries.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause Point:</span> the DNA can be stored at 4<sup>o</sup>C for several days. For long term storage we recommend storing the cleaned DNA at −20<sup>o</sup>C.</div>
<h3>Quantification of Binding Using qPCR - Day 5</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 h</div>
<p>Binding levels of the protein of interest are quantified using qPCR. The relative binding frequency is estimated by calculating the percentage of input recovered in the IP.</p>
<div class="note">
<span class="note-title">Note:</span> If it is already known which loci should be bound by the protein of interest, binding to a few of these regions can be measured using qPCR. Even if the aim is to quantify the samples using sequencing, an initial check by qPCR is recommended to make sure that the procedure worked.</div>
<div class="note">
<span class="note-title">Note:</span> This qPCR protocol was designed for 10 μl reactions in 384-well format using IQ SYBR Green super mix (Bio-Rad) and a CFX384 Touch Real-Time PCR Detection system (Bio-Rad). The exact procedure differs depending on the reagents and machine used, but the general design principles are the same for all qPCRs. An overview of the reagents needed for 1 qPCR reaction are listed in <a href="#tbl3">Table 3</a>.</div>
<table id="tbl3">
<caption>Table 3. Reagents of a Single 10 μl qPCR Reaction</caption>
<thead>
<tr>
<th>Reagents</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>IQ SYBR Green super mix</td>
<td>5 μl</td>
</tr>
<tr>
<td>MQ</td>
<td>2.8 μl</td>
</tr>
<tr>
<td>Forward + Reverse primer mix (10 μM each)</td>
<td>0.2 μl</td>
</tr>
<tr>
<td>Template</td>
<td>2 μl</td>
</tr>
<tr>
<td>Optional: Precision blue</td>
<td>0.014 μl (1/70 μl)</td>
</tr>
</tbody>
</table>
<div class="critical">
<span class="critical-title">Critical:</span> When designing primers for qPCR, make sure the amplicons are not too big (preferably &lt; 140 bp, optimally ∼100 bp). If the average size of the sheared DNA is close to the size of the amplicon, this will lead to inefficient amplification.</div>
<ol start="64">
<li>Make a working stock of each primer pair combination by adding both the forward and the reverse primer to a final concentration of 10 μM. (For example, add to 160 μl MQ, 20 μl of 100 μM forward primer and 20 μl of 100 μM reverse primer for a 200 μl stock).</li>
<li>Prepare a 96-well plate with the DNA as is shown in <a href="#fig3">Figure 3</a>. First fill the plate with MQ needed in each well, then add the DNA.</li>
</ol>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Example of the Layout for the 96-Well Plate Containing the (un)diluted Standard Curve, IP and Input Samples</div>
<p>This example is for 6 IPs.</p>
</figcaption>
</figure>
<ol type="a">
<li>Make a calibration curve of sheared reference DNA in the first column of the plate. Make a 5-step 10-fold serial dilution of the reference DNA. Also fill 2 wells with MQ as a no template control. Measure at least in duplicate, in which case 4 μl per primer pair is needed.</li>
<li>In the next column(s) add the IPs. Depending on the efficiency of the ChIP, and the abundance of the protein of interest, the IPs can be diluted 2x / 4x. Always measure the IPs in technical quadruplicate, 8 μl per IP per primer pair is needed.</li>
<li>Add the diluted inputs (50x diluted) in the column(s) after the IPs. If possible, always measure the inputs in technical quadruplicate as well. If more space is needed on the 384-well plate, the inputs can be measured in triplicate as well.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> When only a few samples are run at the same time, PCR strip tubes may also be used.</div>
<div class="note">
<span class="note-title">Note:</span> When doing qPCR, always take a standard curve along for each primer pair on each plate. It is best if this standard is created from fragmented, cleaned, genomic DNA and that the same material is used for all standard curves. This way different plates can be compared with each other. Ideally, this reference DNA should be isolated from the same strain or background as the samples of interest. Our undiluted reference DNA has a concentration of <b>∼</b> 0.25 ng<b>/</b>μl.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure to properly mix after each dilution of the standard curve and the samples. Failure to mix properly will give unreproducible qPCR results.</div>
<ol start="66">
<li>Make a pre-mix of 2.8 μl MQ and 5 μl IQ SYBR Green super mix for the number of desired reactions + 10%. Mix well and pipette 7.8 μl ∗ number of desired reactions per primer pair + 5% to labelled 2.0 ml tubes.</li>
<li>Add 0.2 μl forward + reverse primers mix ∗ the number of reaction per primer pair + 5% to the tubes to make the primer-pair specific master mixes. Mix well.</li>
</ol>
<div class="optional">
<span class="optional-title">Optional:</span> When using BR white plates (Bio-Rad) it can be tricky to see which wells are filled and which are not. We usually add precision blue in a 1:700 dilution to each master mix to more easily visualize the filled wells.</div>
<ol start="68">
<li>Add 8 μl of the primer specific master mixes to wells of the 384-well plate. It is recommended to use an electronic pipette to increase speed and accuracy.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Keep both plates cold during preparation, to keep evaporation to a minimum. If possible, use a plate cooler for the 384-well plate or an inverted heat block in a bucket filled with ice, to provide a flat surface.</div>
<ol start="69">
<li>Using an 8-channel multichannel pipette transfer column-by-column 2 μl of template from the 96-well plate to the 384-well plate (<a href="#fig4">Figure 4</a>).</li>
</ol>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Example of a 384-Well Plate Layout for 6 IPs and 4 Primer Pairs</div>
</figcaption>
</figure>
<div class="note">
<span class="note-title">Note:</span> To keep track of which wells are filled and which are not, we recommend printing out the plate layout and mark where you are after each template addition.</div>
<ol start="70">
<li>When all the DNA is transferred seal the plate properly with a seal compatible with qPCR.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Check the seal for damage or dirty spots. The qPCR machine will measure the fluorescence through the seal, anything on the seal that changes the path of the light can affect the measured values.</div>
<div class="note">
<span class="note-title">Note:</span> To mix, briefly vortex the plate at half speed using a standard vortex followed by a quick spin (200g).</div>
<ol start="71">
<li>Run the qPCR using an appropriate program. An example of a quick 2-step protocol can be found in <a href="#tbl4">Table 4</a>.</li>
</ol>
<table id="tbl4">
<caption>Table 4. Example of a 2-step qPCR Program with Melting Curve</caption>
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
</tr>
</thead>
<tbody>
<tr>
<td>1</td>
<td>95<sup>o</sup>C</td>
<td>3:00</td>
</tr>
<tr>
<td>2</td>
<td>95<sup>o</sup>C</td>
<td>0:15</td>
</tr>
<tr>
<td>3</td>
<td>60<sup>o</sup>C</td>
<td>0:30 + measure fluorescence</td>
</tr>
<tr>
<td>4</td>
<td colspan="2">Go to step 2, 39x more times</td>
</tr>
<tr>
<td>5</td>
<td>95<sup>o</sup>C</td>
<td>1:00</td>
</tr>
<tr>
<td>6</td>
<td colspan="2">Melting curve: 70<sup>o</sup>C to 95<sup>o</sup>C, increment 0.5<sup>o</sup>C + measure fluorescence</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> A 2-step protocol can only be used with primers that have a melting temperature ≥ 61<sup>o</sup>C in the conditions of the qPCR. If possible, take this into account when designing primers. The conditions of the qPCR using IQ SYBR Green super mix are: Na<sup>+</sup> 50 mM, Oligos 0.2 μM, Mg<sup>++</sup> 6 mM and dNTP 0.4 mM</div>
<div class="note">
<span class="note-title">Note:</span> A melting curve at the end of the program is important to confirm that only a single specific product was formed.</div>
<ol start="72">
<li>Calculate the efficiency of the primers (the Bio-Rad CFX Meastro software will do this automatically). The efficiency should be between 90%<b>–</b>110% (1.9<b>–</b>2.1).</li>
<li>After exporting the data to the format of choice (for example excel file), calculate the starting quantities (SQs) of all reactions for each primer pair using the corresponding standard curve (the Bio-Rad CFX Meastro software will do this automatically).</li>
<li>First calculate the average SQ of each sample by averaging the four replicates.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> sometimes a reaction of a single replicate can fail. If there is a clear single outlier, this outlier can be removed to get a more accurate estimate of the SQ. As a rule of thumb, the replicates should have a coefficient of variation (standard deviation / average) &lt; 20%.</div>
<ol start="75">
<li>Correct each reaction for the dilution by multiplying the SQs by the dilution factor.</li>
<li>Correct all samples for the starting volume: 20 μl for the inputs and 450 μl for the IPs.</li>
<li>Calculate the % of input value by dividing the value of each IP by its corresponding input. See <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Mmc1.xlsx">Table S1</a> for an example of these calculations.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> For accurate quantitation, the Cq (Ct) values of both the IP and the input samples should be within the values of the standard curve.</div>
<div class="note">
<span class="note-title">Note:</span> Always include a background control with each qPCR: a promoter that does not have binding of the TF of interest. When a background control is included, the ratio over background can be calculated. However, if the background is very low, which is usually the case when using magnetic beads, the background can be hard to quantify using qPCR. If there is a lot of variation in the background signal, the enrichment over background will also have a lot of variation.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected Outcomes</h2>
<p>A successful ChIP should show a clear enrichment of its expected binding sites over background. The actual values of the ChIP signal (% of input) varies between proteins. This is dependent on the frequency of binding, or, in other words, the percentage of cells in the population where the protein of interest is bound at a given time, as well as the efficiency of the IP itself. For strongly bound histone proteins, which are relatively easy to ChIP, enrichment values of ∼20%<b>–</b>50% can be reached. Abundant transcription factors often have ChIP signals between 0.5%<b>–</b>5% of input, while for less abundant proteins values of 0.1% of input or lower can be expected.</p>
<p>An example of binding levels of an abundant transcription factor, Abf1, to two of its targets (FCF1 and NHX1) and a background region (TFC1) are shown in <a href="#fig5">Figure 5</a>. A wildtype (WT) strain, without a V5 tagged protein, is shown as a control. Indeed, the WT IP shows no binding or enrichment of any of the targets over background (<a href="#fig5">Figures 5</a>A and 5B). The target FCF1 has stronger binding compared to the target NHX1. This difference in binding levels is also observed when quantifying the binding using ChIP-seq (<a href="#fig5">Figure 5</a>C). An example of how these values were calculated can be found in the <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Mmc1.xlsx">Table S1</a>.</p>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Binding Levels of the Yeast Protein Abf1 Quantified Using ChIP</div>
<p>(A and B) Barplots showing the results from qPCR represented as (A) % of input or (B) fold enrichment over a background region (TFC1). A wild-type (WT) strain without the V5 tag shows no binding or enrichment of the Abf1 targets. Error bars represent the standard deviation of 3 independent biological replicates.</p>
<p>(C) Genome browser snapshot showing the binding, or lack thereof, of Abf1 (top lane) to the same targets as the qPCR. The wild-type strain shows no binding. All regions are scaled to the same height. The black bars represent the location of the primers used for the respective qPCRs. All tracks show the average signal of three biological replicates.</p>
<p>The data that was used to generate (A) and (B) is provided in the <a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/71-Mmc1.xlsx">Table S1</a>.</p>
</figcaption>
</figure>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>Although ChIP is a very common technique to assess protein-DNA binding, it has clear limitations. Formaldehyde can only form cross-links between DNA and proteins when they are ∼2 Å apart (<a href="#bib4">Hoffman et al., 2015</a>). This means that proteins that do not directly contact DNA can only be cross-linked to DNA through other proteins, which is inefficient. Use of a protein-protein cross-linker such as for example dimethyl adipimidate or disuccinimidyl glutarate together with formaldehyde leads to more efficient ChIP for these proteins (<a href="#bib6">Kurdistani and Grunstein, 2003; Tian et al.</a>, <a href="#bib17">2012</a>). For a complete list of protein cross-linkers and their properties see (<a href="#bib6"> Kurdistani and Grunstein, 2003</a>; <a href="#bib17">Tian et al., 2012</a>). In addition, formaldehyde mainly forms cross-links between G/C bases and specific residues in the protein of interest. If there are limited positions where these are in close proximity, proteins may be less efficiently cross-linked to some sites compared to others (<a href="#bib10">Rossi et al., 2018a</a>).</p>
<p>There is evidence that, at least for one protein, interactions with DNA that are very short lived (&lt; 5 s) are not efficiently detected by ChIP (<a href="#bib12">Schmiedeberg et al., 2009</a>). This suggests that ChIP might not be suitable for proteins that bind DNA only very transiently.</p>
<p>Most ChIP protocols use sonication to fragment DNA. If such a protocol is used for ChIP sequencing, the resolution of the ChIP peaks is limited by the average fragment size. If a higher resolution is required, the DNA may be further fragmented following sonication, using λ-exonuclease (<a href="#bib9">Rhee and Pugh, 2011</a>; <a href="#bib11">Rossi et al., 2018b</a>) or the DNA may be fragmented enzymatically using MNase instead of sonication (<a href="#bib3">Gutin et al., 2018</a>; <a href="#bib14">Skene and Henikoff, 2015</a>). The enzymatic fragmentation will create smaller fragments that increase the resolution of the sequencing but will make it more difficult to quantify binding by qPCR.</p>
<p>ChIP is known to have artefacts. Genomic loci with high expression (such as tRNAs or highly expressed genes) often show up as peaks in ChIP-seq or ChIP-chip experiments (<a href="#bib7">Park et al., 2013</a>; <a href="#bib16">Teytelman et al., 2013</a>). Long cross-linking times increase the severity of these artefacts (<a href="#bib2">Baranello et al., 2015</a>). If there is interest to quantify protein binding to such loci, proper controls should be taken along, such as ChIP for a protein that does not bind DNA (e.g. GFP).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3>Problem</h3>
<p>Low DNA yield (visible in the Bioanalyzer electropherogram, step 32)</p>
<h3>Potential Solution</h3>
<p>Low DNA yields can be caused by inefficient cell lysis. The extent of lysis can be assessed by taking a small aliquot of cell lysate and examine this using a hemocytometer under a standard microscope (after step 14). Longer cross-linking times may make the cell wall tougher and this can reduce lysis efficiency.</p>
<p>With long cross-linking times, the chromatin may co-precipitate with the unbroken cells and cell debris (Bead beating procedure, step 17). If this happens, the chromatin is visible as a vague white layer on top of the pellet. Make sure to take as much of this chromatin layer as possible together with the rest of the supernatant. Although we did not test this, centrifugation at a lower speed could prevent the chromatin from co-precipitating with the cell debris.</p>
<h3>Problem</h3>
<p>Inconsistent shearing of DNA (visible in the Bioanalyzer electropherogram, step 32)</p>
<h3>Potential Solution</h3>
<p>When shearing the chromatin (step 21), it is important to closely follow the recommendations of the manufacturer of the sonicator. For example, the Bioruptor Pico has a maximum limit of buffer used for each of its tubes. When the volume in the tubes exceeds this limit, the shearing can become unreproducible. We also noticed that using lower concentrations of chromatin (by diluting the samples) can help achieve more consistent shearing. In this case, also bigger volumes should be used in the IP step.</p>
<p>Potentially, a higher concentration of SDS could help to increase the reproducibility of the shearing. Higher SDS concentrations can, however, interfere with the subsequent IP step. The samples may need to be diluted with FA lysis buffer without SDS to lower the concentration again.</p>
<p>As an alternative for sonication, enzymatic fragmentation using for example MNase could also be used. When using MNase, the lysis buffer may need to be adapted to allow efficient MNase digestion. We have not tested the lysis efficiency and compatibility with MNase digestion of different lysis buffers. Alternatively, after separating the chromatin from the cell debris, the chromatin may be precipitated using a high-speed spin (optional centrifugation after step 18). The pelleted chromatin can then be resuspended using an MNase digestion buffer. The latter strategy is preferred as this circumvents the need to adapt the lysis buffer. Be aware that extensive MNase digestion will create smaller fragments compared to sonication. This will increase the resolution of ChIP-seq, but will make quantification with qPCR more difficult.</p>
<h3>Problem</h3>
<p>Low ChIP signal</p>
<h3>Potential Solution</h3>
<p>Formaldehyde polymerizes in the presence of oxygen. When the formaldehyde is old, or the bottle is opened very frequently, formaldehyde may polymerize extensively. This will lower the cross-linking efficiency. Replace the bottle of formaldehyde when this happens.</p>
<p>With very stringent washes (steps 50<b>–</b>52), the binding of the antibody to the protein of interest may be disrupted, depending on the affinity. Less stringent washes may be used, but care has to be taken that the protein of interest does not bind to DNA <i>in vitro</i> during the procedure. Bear in mind that washing less stringently will also increase the background levels.</p>
<p>The protein of interest may degrade extensively during the procedure, which happens for large proteins, especially. Take aliquots at several steps of the protocol (indicated as optional steps at step 12, 20 and 24) and determine the extent of degradation at each step using Western blotting. If extensive degradation is a problem, consider adding more protease inhibitors, sonicating more gently or enzymatic fragmentation of the DNA (see problem: inconsistent shearing of DNA).</p>
<h3>Problem</h3>
<p>High variability in ChIP signal</p>
<h3>Potential Solution</h3>
<p>ChIP protocols involve many steps, and small differences at any step can lead to changes in ChIP signal. It is therefore important to be consistent and accurate at all steps. The main sources of variation are often differences in the extent of cross-linking, protein degradation or DNA shearing. It is therefore important to accurately control the time of cross-linking, to make sure that the samples remain cool throughout the procedure, to pre-cool all tubes and buffers and to add protease inhibitors at the steps indicated.</p>
<p>When the background signal is very low, there may only be a few molecules present in the qPCR reaction. At this lower limit, a larger number of replicates is needed to accurately estimate the background levels. Although the limit may depend on the experimental setup, as a rule of thumb Cq values above 30 will be more variable. Values close to 35 likely represent only a few molecules present in the reaction, and therefore will be hard to accurately quantify.</p>
</section>
<section>
<h2 id="q-and-a">Q&amp;A</h2>
<p class="question"><b>Q:</b> In Step 21:Did you also try to use MNase in this protocol?</p>
<p class="answer"><b>A:</b> We did not try to use MNase with our ChIP protocol. We comment about the lysis buffer in the troubleshooting paragraph.</p>
<p class="question"><b>Q:</b> In the Troubleshooting section, could you please explain better how the lysis buffer should be adapted for the MNase digestion?</p>
<p class="answer"><b>A:</b> We have not tested how the lysis buffer can be adapted.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aparicio, O., Geisberg, J.V., and Struhl, K. (2004). Chromatin immunoprecipitation for determining the association of proteins with specific genomic sequences in vivo. Curr. Protoc. Cell Biol. Chapter 17, Unit 17.7.</p>
<p id="bib2">Baranello, L., Kouzine, F., Sanford, S., and Levens, D. (2015). ChIP bias as a function of cross-linking time. Chromosome Res. <i>24</i>, 175-181.</p>
<p id="bib3">Gutin, J., Sadeh, R., Bodenheimer, N., Joseph-Strauss, D., Klein-Brill, A., Alajem, A., Ram, O., and Friedman, N. (2018). Fine-resolution mapping of TF binding and chromatin interactions. Cell Rep. <i>22</i>, 2797-2807.</p>
<p id="bib4">Hoffman, E.A., Frey, B.L., Smith, L.M., and Auble, D.T. (2015). Formaldehyde crosslinking: a tool for the study of chromatin complexes. J. Biol. Chem. <i>290</i>, 26404-26411.</p>
<p id="bib5">de Jonge, W.J., Brok, M., Kemmeren, P., and Holstege, F.C.P. (2019). An extensively optimized chromatin immunoprecipitation protocol for quantitatively comparable and robust results. BioRxiv 835926.</p>
<p id="bib6">Kurdistani, S.K., and Grunstein, M. (2003). In vivo protein-protein and protein-DNA crosslinking for genomewide binding microarray. Methods <i>31</i>, 90-95.</p>
<p id="bib7">Park, D., Lee, Y., Bhupindersingh, G., and Iyer, V.R. (2013). Widespread misinterpretable ChIP-seq bias in yeast. PloS One <i>8</i>, e83506.</p>
<p id="bib8">Pchelintsev, N.A., Adams, P.D., and Nelson, D.M. (2016). Critical parameters for efficient sonication and improved chromatin immunoprecipitation of high molecular weight proteins. PloS One <i>11</i>, e0148023.</p>
<p id="bib9">Rhee, H.S., and Pugh, B.F. (2011). Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution. Cell <i>147</i>, 1408-1419.</p>
<p id="bib10">Rossi, M.J., Lai, W.K.M., and Pugh, B.F. (2018a). Genome-wide determinants of sequence-specific DNA binding of general regulatory factors. Genome Res. <i>28</i>, 497-508.</p>
<p id="bib11">Rossi, M.J., Lai, W.K.M., and Pugh, B.F. (2018b). Simplified ChIP-exo assays. Nat. Commun. <i>9</i>, 1-13.</p>
<p id="bib12">Schmiedeberg, L., Skene, P., Deaton, A., and Bird, A. (2009). A temporal threshold for formaldehyde crosslinking and fixation. PLOS ONE <i>4</i>, e4636.</p>
<p id="bib13">Schneider, C.A., Rasband, W.S., and Eliceiri, K.W. (2012). NIH Image to ImageJ: 25 years of image analysis. Nat. Methods <i>9</i>, 671-675.</p>
<p id="bib14">Skene, P.J., and Henikoff, S. (2015). A simple method for generating high-resolution maps of genome-wide protein binding. ELife <i>4</i>, e09225.</p>
<p id="bib15">Sutherland, B.W., Toews, J., and Kast, J. (2008). Utility of formaldehyde cross-linking and mass spectrometry in the study of protein-protein interactions. J. Mass Spectrom. JMS <i>43</i>, 699-715.</p>
<p id="bib16">Teytelman, L., Thurtle, D.M., Rine, J., and van Oudenaarden, A. (2013). Highly expressed loci are vulnerable to misleading ChIP localization of multiple unrelated proteins. Proc. Natl. Acad. Sci. U. S. A. <i>110</i>, 18602-18607.</p>
<p id="bib17">Tian, B., Yang, J., and Brasier, A.R. (2012). Two-step crosslinking for analysis of protein-chromatin interactions. Methods Mol. Biol. Clifton NJ <i>809</i>, 105-120.</p>
<p id="bib18">Zaidi, H., Hoffman, E.A., Shetty, S.J., Bekiranov, S., and Auble, D.T. (2017). Second-generation method for analysis of chromatin binding with formaldehyde-cross-linking kinetics. J. Biol. Chem. <i>292</i>, 19338-19355.</p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Supplemental Information</h3>
<p>Supplemental Information can be found online at <a href="https://doi.org/10.1016/j.xpro.2020.100020">https://doi.org/10.1016/j.xpro.2020.100020</a>.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank Marit de Kort for testing and critical reading of the protocol. This work was supported by the <a href="https://doi.org/10.13039/501100003246">Netherlands Organisation for Scientific Research</a> (NWO) grant and by the <a href="https://doi.org/10.13039/100010663">European Research Council</a> (ERC) grant 671174 DynaMech.</p>
<h3>Author Contributions</h3>
<p>Conceptualization, W.J.d.J., P.K., and F.C.P.H.; Investigation, W.J.d.J. and M.B.; Writing – Original Draft, W.J.d.J. and F.C.P.H.; Writing – Review &amp; Editing, W.J.d.J., M.B., P.K., and F.C.P.H.; Funding Acquisition, P.K. and F.C.P.H.; Supervision, P.K. and F.C.P.H.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>