<article>
<section>
<h1 id="header">TARSII and CARSII: Two approaches for SNP-independent identification of germline differentially methylated regions in mammals</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Wenhao Zhang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#fn1">6</a></sup> and Yi Zhang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#aff3">3</a>,<a href="#aff4">4</a>,<a href="#aff5">5</a>,<a href="#fn2">7</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Howard Hughes Medical Institute, Boston Children’s Hospital, Boston, MA 02115, USA</p>
<p id="aff2"><sup>2</sup>Program in Cellular and Molecular Medicine, Boston Children’s Hospital, Boston, MA 02115, USA</p>
<p id="aff3"><sup>3</sup>Division of Hematology/Oncology, Department of Pediatrics, Boston Children’s Hospital, Boston, MA 02115, USA</p>
<p id="aff4"><sup>4</sup>Department of Genetics, Harvard Medical School, Boston, MA 02115, USA</p>
<p id="aff5"><sup>5</sup>Harvard Stem Cell Institute, WAB-149G, 200 Longwood Avenue, Boston, MA 02115, USA</p>
<p id="fn1"><sup>6</sup>Technical contact</p>
<p id="fn2"><sup>7</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yzhang@genetics.med.harvard.edu">yzhang@genetics.med.harvard.edu</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101240">10.1016/j.xpro.2022.101240</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Identifying germline differentially methylated regions (DMRs) in outbred mammals remains a challenge because of difficulty in obtaining single-nucleotide polymorphisms (SNPs). To overcome this difficulty, we developed two computational approaches, TARSII and CARSII, which allow accurate prediction of germline DMRs from DNA methylomes independent of SNPs. Furthermore, we introduce an easy and quick way to validate the predicted germline DMRs with allelic DNA methylation using CGmapTools. Collectively, our strategy can greatly facilitate <i>de novo</i> identification of germline DMRs in outbred mammals.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref4">Chu et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>TARSII and CARSII predict germline DMRs from DNA methylomes independent of SNPs</li>
<li>Detailed protocol on how to run TARSII/CARSII and interpret the results</li>
<li>An easy and quick way to validate novel germline DMRs predicted by TARSII/CARSII</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Genomic imprinting is the preferential expression of one of the parental alleles, with silencing of the other allele driven by epigenetics (<a href="#bib2">Barlow and Bartolomei, 2014</a>). Although both allelic DNA methylation and H3K27me3 mediate genomic imprinting, allelic DNA methylation plays a major role for imprinting regulation in somatic tissues (<a href="#bib2">Barlow and Bartolomei, 2014</a>; <a href="#bib3">Chen and Zhang, 2020</a>; <a href="#bib9">Inoue et al., 2017</a>). Although allelic DNA methylation from parental alleles can be calculated using strain-specific single-nucleotide polymorphisms (SNPs) in inbred species, such strategy is difficult to be applied to outbred mammals such as human and monkey. Currently, <i>de novo</i> identification of genome-wide allelic differentially methylated regions (DMRs) in human requires integrating hundreds of methylomes from different individuals based on SNP analysis (<a href="#bib14">Zink et al., 2018</a>) or studying special samples such as those with uniparental disomy (<a href="#bib10">Joshi et al., 2016</a>). These methods are both resource and time consuming and thus cannot be widely used in outbred mammals.</p>
<p>To overcome this difficulty, we have developed two computational approaches TARSII (tissue-associated, reads-based, SNP-free approach for identifying imprint-DMRs) and CARSII (CpG-island-associated, reads-based, SNP-free approach for identifying imprint-DMRs), which allow prediction of germline DMRs from whole DNA methylomes generated by next-generation sequencing without SNPs. TARSII is capable of accurately identifying genome-wide germline DMRs using as few as six DNA methylomes from different somatic tissues, which can be derived from either the same individual or different individuals. CARSII is designed to predict germline DMRs from a single DNA methylome. DNA methylomes of sperm and oocyte, or that of uniparental early embryos are also required for categorizing the parental origin of the predicted imprinted DMRs in SNP-independent manner.</p>
<p>Although our approaches can predict germline DMRs with a relative high accuracy, validation by allelic methylation analyses is recommended, especially for predicting novel germline DMRs. To this end, we introduce an easy and quick method based on the CGmapTools (<a href="#bib7">Guo et al., 2018</a>). CGmapTools identifies SNPs directly from the DNA methylome and can calculate allelic DNA methylation associated with individual SNP (<a href="#bib7">Guo et al., 2018</a>). By first predicting germline DMRs in genome-wide level using TARSII/CARSII and then validating the predicted germline DMRs using a few SNPs inside, novel germline DMRs can be easily and efficiently identified in outbred mammals with limited samples.</p>
<h3 id="sec1.1">Prepare your DNA methylome datasets</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 weeks</div>
<div class="note">
<span class="note-title">Note:</span> Our instruction to process the DNA methylome datasets for TARSII and CARSII is based on Linux system. The timing is calculated based on a computer with 64 Gb memory and 4 cores, which is also the minimal requirement.</div>
<p>To properly execute TARSII and CARSII, the users will need to first prepare DNA methylomes and generate the required files as instructed below. We recommend using DNA methylomes from at least six different somatic tissues derived from all three germ layers. In addition, DNA methylomes of sperm/oocyte or uniparental early embryos are also required to categorize the parental origin of those predicted imprinted DMRs. Currently, only reads aligned with Bismark and deduplicated with Picard tools are compatible to TARSII/CARSII<u>.</u></p>
<p>On the other hand, some software (TARSII/CARSII, Bismark, Picard tools, CGmapTools, Trim galore, see also <a href="#key-resources-table">key resources table</a>) are required to be installed in computer before processing. To install the software, please refer to the tutorials by clicking the related links under the IDENTIFIER column in <a href="#key-resources-table">key resources table</a>.</p>
<p>The users can process the raw reads of DNA methylomes for TARSII/CARSII following the steps below:</p>
<ol>
<li>Trim sequencing adaptors of the FASTQ file using Trim Galore.
<ol type="a">
<li>For data generated with traditional whole genome bisulfite sequencing (WGBS) method:
<div class="textbox">
<p><code># Take 5mC.fastq as an example of the input fastq file</code></p>
<p><code># if standard sequencing adaptor of Illumina is used:</code></p>
<p><code>Trim_galore 5mC.fastq --illumina</code></p>
<p><code># if custom adaptor is used:</code></p>
<p><code>Trim_galore 5mC.fastq --adaptor custom_adaptor</code></p>
</div>
</li>
<li>For data generated with post-bisulfite adaptor tagging (PBAT):
<div class="note">
<span class="note-title">Note:</span> Removal of the first and last 9 base pairs in PBAT with “--clip_R1 9” and “--three_prime_clip_R1 9” parameters is recommended to reduce the influence of the random sequences added during library preparation.</div>
<div class="textbox">
<p><code># Take 5mC.fastq as an example of the input fastq file</code></p>
<p><code>Trim_galore 5mC.fastq --dont_gzip --clip_R1 9 --three_prime_clip_R1 9</code></p>
</div>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Aligning reads with Bismark following the command:</li>
</ol>
<div class="textbox">
<p><code># Input fastq file should be trimmed before processing</code></p>
<p><code>bismark --fastq -L 30 -N 1 --non_directional genome_build_folder</code></p>
<p><code>5mC_trimmed.fastq</code></p>
</div>
<ol start="3">
<li>Remove the PCR duplicates with Picard tools and generate sam file following the commands:</li>
</ol>
<div class="textbox">
<p><code>java -Xmx32g -jar picard.jar SortSam INPUT=5mC_bismark_bt2.bam</code></p>
<p><code>OUTPUT=picard_sorted.bam SORT_ORDER=coordinate CREATE_INDEX=true</code></p>
<p><code># Then</code></p>
<p><code>java -Xmx32g -jar picard.jar MarkDuplicates INPUT=picard_sorted.bam</code></p>
<p><code>METRICS_FILE=metrics.txt REMOVE_DUPLICATES=true ASSUME_SORTED=true</code></p>
<p><code>TMP_DIR=tmp_dir VALIDATION_STRINGENCY=SILENT CREATE_INDEX=true</code></p>
<p><code># The deduplicated reads generated above will be stored in the file</code></p>
<p><code>named picard_deduplicated.bam, which can be converted to sam file with</code></p>
<p><code>the following command:</code></p>
<p><code>samtools view picard_deduplicated.bam &gt; picard_deduplicated.sam</code></p>
</div>
<ol start="4">
<li>Generate wig file of DNA methylation at base resolution with Bismark following the commands:</li>
</ol>
<div class="textbox">
<p><code># Extract methylated cytosines from input bam file</code></p>
<p><code>bismark_methylation_extractor --no_overlap --report --bedGraph --cutoff</code></p>
<p><code>3 picard_deduplicated.bam</code></p>
<p><code># Calculate DNA methylation level</code></p>
<p><code>coverage2cytosine --merge_CpG --genome_folder genome_build_folder -o</code></p>
<p><code>output picard_deduplicated.bismark.cov.gz</code></p>
<p><code># Generate wig file with DNA methylation level at base resolution</code></p>
<p><code>awk ‘{printf “%s\t%d\t%.2f\n”, $1, $2, $4/100}’</code></p>
<p><code>output.CpG_report.merge_CpG_evidence.cov &gt; 5mC.wig</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> For each CpG site, we require its methylated state detected by at least three times, which can be fulfilled by adding parameter “--cutoff 3” in “bismark_methylation_extractor” command. We also require CpGs of both forward and reverse strands to be merged, which can be fulfilled by adding parameter “--merge_CpG” in “coverage2cytosine” command.</div>
<p>The users can refer to more detailed instruction in Bismark by clicking on this link (<a href="https://github.com/FelixKrueger/Bismark/tree/master/Docs">https://github.com/FelixKrueger/Bismark/tree/master/Docs</a>) or generate the wig file using customized pipelines. The file for DNA methylation level is tab separated with columns of chromatin, CG base position (merge both forward and reverse strands) and DNA methylation levels (ranges from 0 to 1) (<a href="#fig1">Figure 1</a>).</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Format example of the input files required by TARSII and CARSII</div>
<p>(A) Format example of the input sam file prossed from Bismark and MarkDuplicates.</p>
<p>(B) Format example of the input wig file containing DNA methylation levels at base resolution.</p>
</figcaption>
</figure>
<div class="critical">
<span class="critical-title">Critical:</span> Notably, traditional WGBS method requires to first break genomic DNA into fragments, ligate adaptors to the fragments, and then perform bisulfite treatment to the ligated fragments. This strategy may cause loss of DNA methylation in part of the reads of the read 2 file in pair-end sequencing. Thus, if the WGBS library is constructed using the way described above, we suggest to either only use the read 1 file or remove the reads in the read 2 file with sam Flag marked as 177 or 129 (two reads mapped to the same DNA strand). If the genomic DNA is first processed by bisulfite treatment and then ligated to adaptors as used in PBAT, both read 1 and read 2 files in pair-end sequencing can be included. In that case, the two fastq files in pair-end sequencing can be merged and processed as a single file.</div>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Deposited data</td>
</tr>
<tr>
<td>Public DNA methylomes of mouse somatic tissues</td>
<td>(<a href="#bib8">Hon et al., 2013</a>)</td>
<td>GEO: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE42836">GSE42836</a>
</td>
</tr>
<tr>
<td>Public DNA methylomes of mouse gemmates</td>
<td>(<a href="#bib13">Wang et al., 2014</a>)</td>
<td>GEO: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56697">GSE56697</a>
</td>
</tr>
<tr>
<td>Public DNA methylomes of human somatic tissues</td>
<td>(<a href="#bib5">Court et al., 2014</a>)</td>
<td>GEO: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52578">GSE52578</a>
</td>
</tr>
<tr>
<td>Public DNA methylomes of human somatic tissues</td>
<td>NCBI’s Gene Expression Omnibus</td>
<td>GEO: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE16256">GSE16256</a>
</td>
</tr>
<tr>
<td>Public DNA methylomes of human uniparental early embryos</td>
<td>(<a href="#bib12">Leng et al., 2019</a>)</td>
<td>GEO: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE133856">GSE133856</a>
</td>
</tr>
<tr>
<td>Public DNA methylomes of monkey somatic tissues and uniparental early embryos</td>
<td>(<a href="#bib4">Chu et al., 2021</a>)</td>
<td>GEO: <a href="https://www-ncbi-nlm-nih-gov.ezproxy.u-pec.fr/geo/query/acc.cgi?acc=GSE159347">GSE159347</a>
</td>
</tr>
<tr>
<td>Script in this protocol</td>
<td>Mendeley Data</td>
<td>Mendeley Data: <a href="https://doi.org/10.17632/747r4k4mnz.1">https://doi.org/10.17632/747r4k4mnz.1</a>
</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>TARSII/CARSII</td>
<td>(<a href="#bib4">Chu et al., 2021</a>)</td>
<td><a href="https://doi.org/10.5281/zenodo.5484230">https://doi.org/10.5281/zenodo.5484230</a></td>
</tr>
<tr>
<td>Bismark</td>
<td>(<a href="#bib11">Krueger and Andrews, 2011</a>)</td>
<td><a href="https://github.com/FelixKrueger/Bismark">https://github.com/FelixKrueger/Bismark</a></td>
</tr>
<tr>
<td>Picard Tools</td>
<td>Broad Institute</td>
<td><a href="http://broadinstitute.github.io/picard/">http://broadinstitute.github.io/picard/</a></td>
</tr>
<tr>
<td>CGmapTools</td>
<td>(<a href="#bib7">Guo et al., 2018</a>)</td>
<td><a href="https://cgmaptools.github.io/">https://cgmaptools.github.io/</a></td>
</tr>
<tr>
<td>Trim Galore</td>
<td>Felix Krueger in Babraham Institute</td>
<td><a href="https://doi.org/10.5281/zenodo.5127899">https://doi.org/10.5281/zenodo.5127899</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Input datasets</h3>
<p>To get started, the users can use the public methylomes of somatic tissues and gemmates/uniparental embryos from mouse, human or monkey. These datasets were used and can be found in our previous study (<a href="#bib4">Chu et al., 2021</a>). For convenience, the accession numbers of these datasets are also listed in the <a href="#key-resources-table">key resources table</a>.</p>
<p>Here, we use the human DNA methylomes (brain, muscle, aorta, lung, liver, intestine, androgenetic 8-cell embryo, parthenogenetic 8-cell embryo) as a working example.</p>
<p>Input files:</p>
<ul>
<li>Sam file: DNA methylomes of somatic tissues should be processed into sam file by alignment with Bismark and removing duplicates with MarkDuplicate in Picard Tools as instructed above (<a href="#fig1">Figure 1</a>A).</li>
<li>Wig file: DNA methylomes of somatic tissues, sperm/oocyte or uniparental early embryos should be processed into wig file showing DNA methylation levels at base resolution as instructed above (<a href="#fig1">Figure 1</a>B)</li>
</ul>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Install TARSII and CARSII</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 min</div>
<div class="note">
<span class="note-title">Note:</span> The timing calculated for each step of TARSII and CARSII is based on a computer with 8 Gb memory and 1 core. A minimal of 8 Gb memory and 1 core is required. TARSII and CARSII are functional on Linux and Mac OSX systems.</div>
<ol>
<li>The TARSII/CARSII packages are available through the following public repository:</li>
</ol>
<p><a href="https://doi.org/10.5281/zenodo.5484230">https://doi.org/10.5281/zenodo.5484230</a></p>
<p>To activate TARSII and CARSII, you may follow the command lines below:</p>
<div class="textbox">
<p><code>cd TARSII/CARSII_folder</code></p>
<p><code>chmod u+x ∗</code></p>
<p><code>cd ./bin</code></p>
<p><code>chmod u+x ∗</code></p>
</div>
<h3 id="sec3.2">Predict germline DMRs using TARSII</h3>
<p>Considering germline DMRs are composed of both hypomethylated and hypermethylated alleles, those regions should have partial DNA methylation and should enrich for both hypomethylated and hypermethylated reads (<a href="#fig2">Figure 2</a>A). Because DMRs are believed to be generally conserved across different tissues (<a href="#bib1">Babak et al., 2015</a>), identifying common candidate DMRs across different tissues can help reduce FDR. These are the basis for imprinting prediction by TARSII.</p>
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Model for prediction of putative germline DMRs by TARSII</div>
<p>(A) Schematic model showing allelic DNA methylation for a typical maternal imprinted DMR.</p>
<p>(B) Schematic models showing the strategies for identifying PMDs from the genome and for identifying candidate DMRs from the PMDs in step 2.</p>
<p>(C) Schematic model showing the strategy for integrated analysis of candidate DMRs in all tissues in step 4.</p>
<p>(D) Schematic model showing the strategy for categorizing the parental origin of the predicted imprinted DMRs in step 7.</p>
</figcaption>
</figure>
<h3 id="sec3.3">Identify candidate DMRs from individual DNA methylome by TARSII</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>In this step, candidate DMRs in each tissue are selected out by two analyses. First, genomic regions containing consecutive CpG sites (&gt;= 10) with 5mC level ranging from 0.3 to 0.7 are selected out as partially methylated domains (PMDs) (<a href="#fig2">Figure 2</a>B). Then, the PMDs enriched for both hypomethylated reads (5mC &lt;= 0.2, reads percentage &gt;= 30%) and hypermethylated reads (5mC &gt;= 0.8, reads percentage &gt;= 30%) are identified as candidate DMRs (<a href="#fig2">Figure 2</a>B).</p>
<ol start="2">
<li>Candidate DMRs of individual somatic tissue methylome are identified following the command:</li>
</ol>
<div class="textbox">
<p><code># Take cortex methylome data as an example</code></p>
<p><code>TARSII_step1_DMR_identify.sh -x cortex_5mC.wig -s</code></p>
<p><code>cortex_picard_deduplicated.sam -o cortex</code></p>
<p><code># Run this script for DNA methylome of each somatic tissue</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The input files and output file name are mandatory; other parameters have been optimized but can be adjusted according to user-specific requests.</div>
<p>Parameters should be provided:</p>
<p>-x    wig file presenting DNA methylation levels at base resolution</p>
<p>-s    sorted sam file with duplicates removed</p>
<p>-o    output file name</p>
<p>Parameters available to be adjusted through [options]:</p>
<p>-n    minimal CpG number required to be included in a PMD. Default: 10 (&gt;= 1)</p>
<p>-m    minimal methylation level for the CpG sites in a PMD. Default: 0.3 (ranges from 0 to 1)</p>
<p>-M    maximal methylation level for the CpG sites in a PMD. Default: 0.7 (ranges from 0 to 1)</p>
<p>-r    minimal CpG number required in a single read. Default: 3 (&gt;= 1)</p>
<p>-l    minimal number of reads required to be aligned to a PMD. Default: 30 (&gt;=1)</p>
<p>-b    bin number to categorize methylation levels of the reads in a PMD. Default: 5 (&gt;= 2)</p>
<div class="note">
<span class="note-title">Note:</span> -b 5 means to categorize the reads into 5 groups with methylation levels range from 0.0–0.2, 0.2–0.4, 0.4–0.6, 0.6–0.8, and 0.8–1.0. Option -c/-C is applied to the first/last bin</div>
<p>-c    minimal percentages of hypomethylated reads versus total reads for a candidate DMR. Default: 0.3 (30%) (ranges from 0 to 1)</p>
<p>-C    minimal percentages of hypermethylated reads versus total reads for a candidate DMR. Default: 0.3 (30%) (ranges from 0 to 1)</p>
<ol start="3">
<li>Following completion of step 2, users will get 3 output files in working directory that include (troubleshooting <a href="#sec6.1">problem 1</a>):
<ol type="a">
<li>A bed file containing all PMDs identified from the input DNA methylome.</li>
<li>A tab separated file containing all PMDs with additional information as percentage of hypomethylated reads, percentage of hypermethylated reads and total reads number in each bin.</li>
<li>A bed file containing all candidate DMRs identified from the input DNA methylome for processing in the next step.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Identify common candidate DMRs from different somatic tissues</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<p>In this step, the overlapping candidate DMRs from all tissues are merged as one (<a href="#fig2">Figure 2</a>C). Then, if the candidate DMRs from at least 5 different tissues show overlap with the merged candidate DMRs, those candidate DMRs are defined as putative imprinted DMRs identified by TARSII (<a href="#fig2">Figure 2</a>C).</p>
<ol start="4">
<li>The putative imprinted DMRs are identified through integrated analysis of candidate DMRs in different somatic tissues following the command:</li>
</ol>
<div class="textbox">
<p><code>TARSII_step2_DMR_integration.sh -f ‘cerebellum_DMR_candidate.bed</code></p>
<p><code>cortex_DMR_candidate.bed heart_DMR_candidate.bed</code></p>
<p><code>intestine_DMR_candidate.bed kidney_DMR_candidate.bed</code></p>
<p><code>liver_DMR_candidate.bed’ -o human</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> Users should include all the bed files of candidate DMRs and an output file name to run this script. The minimal number of tissues required for a putative imprinted DMR to be identified could be adjusted with the parameter -n. However, users should be cautious that reducing the cutoff will increase false discovery rate, while increasing the cutoff may increase the false negative rate.</div>
<p>Parameters should be provided:</p>
<p>-f    all bed files containing candidate DMRs identified from step 1</p>
<div class="note">
<span class="note-title">Note:</span> File names should be located within ‘ ’ symbol</div>
<p>-o    output file name</p>
<p>Parameters available to be adjusted through [options]:</p>
<p>-n    minimal number of tissues for a putative imprinted DMR to be commonly identified in (&gt;= 1)</p>
<ol start="5">
<li>Following completion of step 4, users will get 1 bed file presenting the putative imprinted DMRs in working directory.</li>
</ol>
<h3 id="sec3.5">Categorize parental origin of the putative imprinted DMRs predicted by TARSII</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Since the prediction of TARSII is independent of SNPs, the parental origin of these imprinted DMRs cannot be directly inferred from somatic tissue methylomes by TARSII. Nevertheless, considering the allelic DNA methylation of germline DMRs is inherited from gametes, the differentially methylated regions between sperm and oocyte, as well as between androgenetic and parthenogenetic early embryos, can help infer the parental origin of the predicted imprinted DMRs without SNP information.</p>
<p>In this step, paternal and maternal DMRs in sperm/oocyte or uniparental early embryos are identified. Based on those parental DMRs, TARSII predicted putative imprinted DMRs can be categorized into maternal germline DMRs, paternal germline DMRs and somatic DMRs (<a href="#fig2">Figure 2</a>D).</p>
<ol start="6">
<li>Wig files showing DNA methylation levels at base resolution from sperm/oocyte or uniparental early embryos are required, as shown in <a href="#fig1">Figure 1</a>B. To prepare for these files, please refer to “<a href="#before-you-begin">before you begin</a>” and our previous study (<a href="#bib4">Chu et al., 2021</a>).</li>
<li>Parental origin of the putative imprinted DMRs predicted by TARSII can be categorized following the command:</li>
</ol>
<div class="textbox">
<p><code>TARSII_step3_germline_DMR.sh -p androgenetic_5mC.wig -m</code></p>
<p><code>parthenogenetic_5mC.wig -b human_putative_imprinted_DMR.bed -o human</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The input files and output file name are mandatory; other parameters have been optimized but can be adjusted according to user-specific requests.</div>
<p>Parameters should be provided:</p>
<p>-p    wig file presenting DNA methylation levels at base resolution of sperm/androgenetic embryos</p>
<p>-m    wig file presenting DNA methylation levels at base resolution of oocyte/parthenogenetic embryos</p>
<p>-b    bed file of the imprinted DMRs predicted by TARSII (generated from step 4)</p>
<p>-o    output file name</p>
<p>Parameters available to be adjusted through [options]:</p>
<p>-d    minimal cutoff to define a differentially methylated CpG site for a DMR in gemmates. Default: 0.5 (ranges from 0 to 1)</p>
<p>-n    minimal CpG number required to be included in a DMR in gemmates. Default: 10. (&gt;= 1)</p>
<p>-c    maximal paternal methylation level in a maternal DMR in gemmates. Default: 0.15 (ranges from 0 to 1)</p>
<p>-C    maximal maternal methylation level in a paternal DMR in gemmates. Default: 0.30 (ranges from 0 to 1)</p>
<ol start="8">
<li>Following completion of step 7, users will get 5 files in working directory. File 3, 4, 5 (below) are the final results generated by TARSII (<a href="#troubleshooting">troubleshooting</a> <a href="#sec6.3">problems 2</a> and <a href="#sec6.5">3</a>).
<ol type="a">
<li>A bed file contains paternal DMRs in gemmates with average methylation levels of maternal and paternal alleles.</li>
<li>A bed file contains maternal DMRs in gemmates with average methylation levels of paternal and maternal alleles</li>
<li>A bed file contains maternal germline DMRs predicted by TARSII.</li>
<li>A bed file contains paternal germline DMRs predicted by TARSII.</li>
<li>A bed file contains somatic DMRs predicted by TARSII.</li>
</ol>
</li>
</ol>
<h3 id="sec3.6">Predict germline DMRs using CARSII</h3>
<p>TARSII predicts the germline DMRs through integrated analyses of DNA methylomes from different somatic tissues. However, in some case, identification of tissue-specific imprinting is required. To help predict germline DMRs in a single tissue independent of SNPs, we developed another computational tool, CARSII.</p>
<p>Different from the genome-wide identification of germline DMRs in TARSII, only CpG islands (genomic regions with high CpG density) are included in the analysis of CARSII. This is mainly because: 1) DNA methylation in CpG islands is generally under rigid regulation of multiple transcription factors and epigenetic regulators (<a href="#bib6">Deaton and Bird, 2011</a>). Thus, the DNA methylation in CpG islands is more stable across different cells compared to that of a random region in genome; 2) the majority of germline DMRs in mammals, such as mouse and human, overlap with GpG islands. In contrast, CpG islands only occupy a small portion of the genome (<a href="#bib4">Chu et al., 2021</a>). Thus, the chances for a germline DMR to be identified from a CpG island are much higher than that from a random region in the genome.</p>
<p>Notably, the definition of CpG islands may vary according to different standards used. Nevertheless, we recommend using the CpG islands defined in the UCSC Genome Browser database, which could be found in the following link:</p>
<p><a href="https://hgdownload.soe.ucsc.edu/downloads.html">https://hgdownload.soe.ucsc.edu/downloads.html</a></p>
<h3 id="sec3.7">Identify candidate differentially methylated CpG-islands (DMCs) from a single DNA methylome by CARSII</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 h</div>
<p>The biological basis of CARSII is similar to TARSII, which assumes germline DMRs are enriched for both hypomethylated and hypermethylated alleles (<a href="#fig2">Figure 2</a>A). However, due to heterogeneity of the cells in a tissue, methylation inconsistency in part of the CpG islands and experimental variations caused by batch effects, the false discovery rate (FDR) of CARSII is relatively higher than that of TARSII. Thus, to help reduce the FDR in CARSII, we designed additional steps to remove germline DMRs that may be resulted from random effect or methylation inconsistency.</p>
<p>In this step, CpG islands that enriched for both hypomethylated reads (5mC &lt;= 0.2, reads percentage &gt;= 30%) and hypermethylated reads (5mC &gt;= 0.8, reads percentage &gt;= 30%) are first selected as candidate DMCs (<a href="#fig3">Figure 3</a>A). Then, random test is performed to calculate the FDR for each candidate DMC and those with FDR &lt; 0.05 were removed (<a href="#fig3">Figure 3</a>B). Finally, the methylation consistency in each candidate DMC is determined and candidate DMCs with inconsistent DNA methylation along the CpG island are removed (<a href="#fig3">Figure 3</a>C).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Model for prediction of putative germline DMCs by CARSII</div>
<p>(A–C) Schematic models showing the strategies for quantifying methylated reads (A), random testing reads distribution (B) and analyzing methylation consistency (C) in candidate DMCs.</p>
</figcaption>
</figure>
<ol start="9">
<li>Candidate DMCs from DNA methylome of a certain somatic tissue are identified by CARSII following the command:</li>
</ol>
<div class="textbox">
<p><code># Take cortex methylome data as an example</code></p>
<p><code>CARSII_step1_DMC_identify.sh -g human_CpG_island.bed -x cortex_5mC.wig</code></p>
<p><code>-s cortex_picard_deduplicated.sam -o cortex</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The input files and output file names are mandatory; other parameters have been optimized but can be adjusted according to user-specific requests:</div>
<p>Parameters should be provided:</p>
<p>-g    bed file of all the CpG islands</p>
<p>-x    wig file presenting DNA methylation levels at base resolution</p>
<p>-s    sorted sam file with duplicates removed</p>
<p>-o    output file name</p>
<p>Parameters available to be adjusted through [options]:</p>
<p>-r    minimal CpG number required in a single read. Default: 3 (&gt;= 1)</p>
<p>-l    minimal number of reads required to be aligned to a CpG island. Default: 20 (&gt;= 1)</p>
<p>-b    bin number to categorize methylation levels of the reads in a CpG island. Default: 5 (&gt;= 2)</p>
<div class="note">
<span class="note-title">Note:</span> -b 5 means to categorize the reads into 5 groups with methylation levels range from 0.0–0.2, 0.2–0.4, 0.4–0.6, 0.6–0.8, and 0.8–1.0. Option -c/-C is applied to the first/last bin</div>
<p>-c    minimal percentages of hypomethylated reads versus total reads for a candidate DMC. Default: 0.3 (30%) (ranges from 0 to 1)</p>
<p>-C    minimal percentages of hypermethylated reads versus total reads for a candidate DMC. Default: 0.3 (30%) (ranges from 0 to 1)</p>
<p>-p    maximal false discovery rate for a candidate DMC. Default: 0.05 (&lt;= 0.05)</p>
<p>-t    test times for calculation of false discovery rate. Default: 10000 (&gt;=1)</p>
<p>-d    maximal methylation differences allowed within a candidate DMC. Default: 0.2 (ranges from 0 to 1)</p>
<div class="critical">
<span class="critical-title">Critical:</span> We do not recommend using CARSII to directly predict non-germline/somatic DMRs because no allelic DNA methylation in gametes could be used to confirm the imprinting status for a somatic DMR.</div>
<ol start="10">
<li>Following completion of step 9, the user will get 2 files in working directory (<a href="#troubleshooting">troubleshooting</a> <a href="#sec6.1">problem 1</a>):
<ol type="a">
<li>A tab separated file containing all CpG islands with the percentage of hypomethylated reads, percentage of hypermethylated reads, total reads number in a CpG island and reads number in each bin.</li>
<li>A bed file containing predicted candidate DMCs by CARSII. The candidate DMCs in this file are viewed as the putative imprinted DMCs predicated by CARSII.</li>
</ol>
</li>
</ol>
<h3 id="sec3.8">Categorize parental origin of the putative imprinted DMCs predicted by CARSII</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<p>Similar to TARSII, in this step, paternal and maternal-specific methylated CpG-islands in gemmates are identified by analyzing DNA methylomes in sperm/oocyte or uniparental early embryos (<a href="#fig2">Figure 2</a>D). Then, based on those parental-specific methylated CpG-islands, putative imprinted DMCs are categorized into maternal germline DMCs, paternal germline DMCs and somatic DMCs (<a href="#fig2">Figure 2</a>D).</p>
<ol start="11">
<li>Parental origin of imprinted DMCs predicted by CARSII can be identified following the command:</li>
</ol>
<div class="textbox">
<p><code># Take cortex methylome data as an example</code></p>
<p><code>CARSII_step2_germline_DMR.sh -p androgenetic_5mC.wig -m</code></p>
<p><code>parthenogenetic_5mC.wig -b cortex_putative_imprinted_DMC.bed -o cortex</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The input files and output file names are mandatory; other parameters have been optimized but can be adjusted according to user-specific requests.</div>
<p>Parameters should be provided:</p>
<p>-p    wig file presenting DNA methylation levels at base resolution of sperm/androgenetic embryos</p>
<p>-m    wig file presenting DNA methylation levels at base resolution of oocyte/parthenogenetic embryos</p>
<p>-b    bed file of putative imprinted DMCs generated from step 1 script</p>
<p>-o    output file name</p>
<p>Parameters available to be adjusted through [options]:</p>
<p>-d    minimal cutoff to define a differentially methylated CpG site for a DMC in gemmates. Default: 0.5 (ranges from 0 to 1)</p>
<p>-c    maximal paternal methylation in a maternal DMC in gemmates. Default: 0.15 (ranges from 0 to 1)</p>
<p>-C    maximal maternal methylation in a paternal DMC in gemmates. Default: 0.30 (ranges from 0 to 1)</p>
<ol start="12">
<li>Following completion of step 11, the user will get 3 files in working directory (<a href="#troubleshooting">troubleshooting</a> <a href="#sec6.3">problems 2</a> and <a href="#sec6.7">4</a>):
<ol type="a">
<li>A bed file containing maternal germline DMCs predicted by CARSII</li>
<li>A bed file containing paternal germline DMCs predicted by CARSII.</li>
<li>A bed file containing somatic DMCs predicted by CARSII.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> As noted above, we do not recommend applying CARSII to predict somatic DMCs. Nevertheless, if the user does use this approach to predict somatic DMCs, validation by allelic methylation analysis is suggested before moving forward (see the following part).</div>
<h3 id="sec3.9">Analysis of allelic DNA methylation for the putative imprinted DMRs with CGmapTools</h3>
<p>Although the putative germline DMRs predicted by TARSII and CARSII is relatively accurate (<a href="#bib4">Chu et al., 2021</a>), certain level of false discovery rate is unavoidable without the information of the allelic DNA methylation. Therefore, we introduce CGmapTools (<a href="#bib7">Guo et al., 2018</a>) to help easily and quickly validate the imprinted DMRs predicted by TARSII/CARSII. CGmapTools is capable of identifying SNPs directly from DNA methylomes and calculating allelic methylation levels associated with those SNPs.</p>
<p>By combining TARSII/CARSII and CGmapTools, putative imprinted DMRs in the whole genome can be identified first using TARSII/CARSII independent of SNPs. Then, allelic methylation of the putative imprinted DMRs can be calculated by CGmapTools with only a few SNPs located within those DMRs. In this way, instead of collecting large number of DNA methylomes from many different individuals and SNPs from their parents’ genomes, only a few DNA methylomes are sufficient for accurate <i>de novo</i> identification of imprinted regions in outbred mammals.</p>
<p>Here, we only provide an integrated analysis of CGmaptools and TARSII/CARSII. For a complete and detailed instructions on CGmapTools, please refer to the published study (<a href="#bib7">Guo et al., 2018</a>) and the link below:</p>
<p><a href="https://cgmaptools.github.io/">https://cgmaptools.github.io/</a></p>
<h3 id="sec3.10">Identify SNPs from DNA methylome using CGmapTools</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<ul>
<li>In this step, SNPs in certain DNA methylome are extracted using CGmapTools (<a href="#fig4">Figure 4</a>A). In TARSII, we can extract SNPs from every DNA methylome that used for analysis.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Validation of predicted germline DMRs by CGmapTools</div>
<p>(A) schematic models showing allelic methylation analysis in predicted germline DMRs by CGmapTools. Red arrows indicate SNPs.</p>
<p>(B) A list of human putative germline DMRs predicted by TARSII. Red color indicates the predicated germline DMRs are associated with SNPs identified from somatic tissue methylomes by CGmapTools.</p>
<p>(C) Bar plots showing allelic DNA methylation of TARSII predicted human germline DMRs. The allelic methylation surrounding each SNP located inside the predicted germline DMRs is calculated by CGmapTools. Stars indicate significance of the allelic methylation differences. ∗: p-value &lt; 1.0E-3, ∗∗: p-value &lt; 1.0E-4, ∗∗∗: p-value &lt; 1.0E-5. N.S. not statistically significant.</p>
</figcaption>
</figure>
</li>
</ul>
<ol start="13">
<li>SNPs from DNA methylome can be extracted following the commands:</li>
</ol>
<div class="textbox">
<p><code># Take cortex methylome data as an example</code></p>
<p><code>cgmaptools convert bam2cpmap -b cortex_picard_deduplicated.bam -g</code></p>
<p><code>human_genome.fa -o cortex</code></p>
<p><code># Then:</code></p>
<p><code>cgmaptools snv -i cortex.ATCGmap.gz -m bayes -v cortex_SNP.vcf -o</code></p>
<p><code>cortex_SNP.snv --bayes-dynamicP</code></p>
</div>
<ol start="14">
<li>Extracting SNPs located within the putative imprinted DMRs predicted by TARSII/CARSII using the perl script (<a href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-Mmc1.zip">Data S1</a>) and command:</li>
</ol>
<div class="textbox">
<p><code># Take cortex methylome data as an example</code></p>
<p><code>Extract_SNPs_from_DMRs.pl cortex_SNP.vcf cortex_SNP.snv</code></p>
<p><code>human_putative_imprinted_DMR.bed</code></p>
</div>
<p>This script generates a selected vcf file containing SNPs located within the putative imprinted DMRs as listed in the input bed file.</p>
<div class="note">
<span class="note-title">Note:</span> CGmapTools perform allelic analysis but cannot distinguish parental origin. To infer the parental origin of non-germline/somatic DMRs, SNPs identified from parental genomes are still needed.</div>
<h3 id="sec3.11">Calculate allelic DNA methylation using CGmapTools</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 5 min</div>
<ol start="15">
<li>Allelic DNA methylation is calculated by CGmapTools following the command:</li>
</ol>
<div class="textbox">
<p><code># Take cortex methylome data as an example</code></p>
<p><code>cgmaptools asm -r human_genome.fa -b cortex_picard_deduplicated.bam -l</code></p>
<p><code>cortex_SNP_DMRs.vcf &gt; cortex_SNP_DMRs.asm</code></p>
</div>
<div class="note">
<span class="note-title">Note:</span> The asm file contains information on allelic DNA methylation value, p-value and false discovery rate for each SNP in the analyzed DNA methylome. A TRUE or FALSE judgement is provided as an inference to users about whether the allelic methylation associated with a particular SNP is significantly different or not (<a href="#troubleshooting">troubleshooting</a> <a href="#sec6.9">problem 5</a>).</div>
<div class="critical">
<span class="critical-title">Critical:</span> Since the length and range of imprinted DMRs predicted by TARSII/CARSII can vary when compared to the real imprinted DMRs, some SNPs located within the predicted imprinted DMRs may actually located outside of the real imprinted DMRs. Thus, to check the accurate location of individual SNP using genome browser tools on UCSC genome browser or integrative genomic viewer (IGV) is highly recommended. In general, informative SNPs should locate within a clear partially methylated domain. On the other hand, it would be more solid if several SNPs within certain DMR are all exhibit significant allelic DNA methylation differences.</div>
<ol start="16">
<li>As an example, we predicted the germline DMRs by TARSII using human somatic tissue methylomes as indicated in “<a href="#before-you-begin">before you begin</a>”. In total, 29 of germline DMRs were identified, most of which are maternal germline DMRs except H19 (<a href="#fig4">Figure 4</a>B). 22 of the 29 germline DMRs are associated with at least 1 SNP identified by CGmapTools based on the human somatic tissue methylomes (<a href="#fig4">Figure 4</a>B). All of those SNP-associated germline DMRs exhibit clear allele-specific DNA methylation (<a href="#fig4">Figure 4</a>C), confirming the high accuracy of our approaches.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>TARSII and CARSII are designed for efficient identification of germline DMRs in animal without the pre-knowledge of SNPs. These approaches can greatly reduce the time and resources needed for imprinting analysis in outbred mammals. TARSII predicts the germline DMRs that are imprinted in different somatic tissues from a genome-wide level. CARSII only focuses on CpG islands and identifies germline DMRs that contain or overlap with CpG islands. Since a single DNA methylome is the minimal requirement for CARSII, tissue-specific germline DMRs can be identified by CARSII. Both TARSII and CARSII allow <i>de novo</i> discovery of germline DMRs. For validation of novel germline DMRs predicted by TARSII/CARII, we introduced CGmapTools, which identifies SNPs directly from DNA methylome and calculates allelic DNA methylation associated with those SNPs (<a href="#fig4">Figure 4</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>TARSII and CARSII are computational tools for predicting germline DMRs. Due to its SNP-independent nature, TARSII and CARSII can be conveniently applied to germline DMR identification in outbred mammals with minimal requirement of DNA methylomes. However, false discovery rate is unavoidable due to factors such as cell heterogeneity, sequencing depth, and experimental variations. The FDR is higher in CARSII considering only a single DNA methylome is used. To reduce the FDR, one strategy is to perform allelic DNA methylation analysis using CGmapTools, to validate the predicted DMRs with SNPs inside (<a href="#fig4">Figures 4</a>A–4C). This validation is especially important if the users attempt to investigate the somatic DMRs predicted by TARSII/CARSII, as the lack of support by inherited allelic DNA methylation from gametes could increase the FDR.</p>
<p>Notably, TARSII and CARSII are not efficient to predict short DMRs (CpG number &lt; 10) because short DMRs are likely to have fewer CpG sites and lower reads coverage. By simply decreasing cutoffs of CpG number and reads coverage will result in significant increase in FDR (<a href="#bib4">Chu et al., 2021</a>). To reduce the FDR in TARSII and CARSII, we included algorithms with intention to remove the regions with low CpG density or low coverage of sequencing reads. Nevertheless, short germline DMRs are not common and the majority of germline DMRs can be efficiently identified using our approaches in at least mouse, human and monkey (<a href="#bib4">Chu et al., 2021</a>).</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>TARSII/CARSII does not generate expected file contents after finishing the first step (see steps 2–3 and 9–10).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>Please make sure your input files are correctly formatted. Especially, only sam file generated from Bismark and deduplicated with Picard tools (MarkDuplicates) is compatible with TARSII/CARSII. For input file format details, please refer to “<a href="#materials-and-equipment">materials and equipment</a>” and <a href="#fig1">Figure 1</a>.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>The number of germline DMRs predicted by TARSII/CARSII is much fewer than expected (see steps 6–8 and 11–12).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>In our protocol we do not specify the requirement of sequencing depth and reads length for TARSII/CARSII prediction, as those can be flexible. However, since TARSII and CARSII are reads-based approaches, deeper in sequencing depth and longer in reads length for the DNA methylomes used will improve the prediction outcome. For a general recommendation, over 100 bp in reads length and over 100 million monoclonal reads for each DNA methylome will ensure a reasonable outcome for TARSII/CARSII.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Some predicted germline DMRs by TARSII from one group of datasets are not identified as germline DMRs in another group of datasets (see steps 6–8).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>In outbred mammals, some imprinted regions can be influenced by SNPs or other variations in DNA sequences from different individuals. Thus, it is possible that the predicted germline DMRs are individual-specific. To validate those germline DMRs, analysis of allelic DNA methylation is required. To better reduce the influences of individual genetic background on germline DMR prediction, we highly recommend including somatic tissues from different individuals in TARSII.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The germline DMRs predicted by CARSII do not meet the expected accuracy (see steps 11–12).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>Since only a single DNA methylome is applied in CARSII, the predicted results can vary among different DNA methylomes. To increase the performance of CARSII, we recommend users to apply a few DNA methylomes from different sources if available. By comparing results from different DNA methylomes of the same tissue, users can select out a list of common predicted germline DMRs, which will be more accurate.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>Significant differences of allelic DNA methylation can be observed on certain SNPs from the calculated results of CGmapTools, but CGmapTools provides a “FALSE” judgement (see step 15).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>CGmapTools sets a relatively high standard to exclude the SNPs with potential bi-allelic methylation states, which sometimes will also exclude the SNPs within the known imprinted regions. To help make an accurate judgement, a few more SNPs in the same predicted germline DMRs should be included in analysis. If available, bisulfite PCR including the targeted SNP can be performed to provide extra information, considering the genomic regions associated with a certain SNP by bisulfite PCR can be much longer than that by the sequencing reads.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Babak, T., DeVeale, B., Tsang, E.K., Zhou, Y., Li, X., Smith, K.S., Kukurba, K.R., Zhang, R., Li, J.B., van der Kooy, D., et al. (2015). Genetic conflict reflected in tissue-specific maps of genomic imprinting in human and mouse. Nat. Genet. <i>47</i>, 544-549. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref1">View at publisher</a></p>
<p id="bib2">Barlow, D.P. and Bartolomei, M.S. (2014). Genomic imprinting in mammals. Cold Spring Harb Perspect. Biol. <i>6</i>, a018382. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref2">View at publisher</a></p>
<p id="bib3">Chen, Z. and Zhang, Y. (2020). Maternal H3K27me3-dependent autosomal and X chromosome imprinting. Nat. Rev. Genet. <i>21</i>, 555-571. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref3">View at publisher</a></p>
<p id="bib4">Chu, C., Zhang, W., Kang, Y., Si, C., Ji, W., Niu, Y., and Zhang, Y. (2021). Analysis of developmental imprinting dynamics in primates using SNP-free methods to identify imprinting defects in cloned placenta. Dev. Cell <i>56</i>, 2826-2840.e7. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref4">View at publisher</a></p>
<p id="bib5">Court, F., Tayama, C., Romanelli, V., Martin-Trujillo, A., Iglesias-Platas, I., Okamura, K., Sugahara, N., Simon, C., Moore, H., Harness, J.V., et al. (2014). Genome-wide parent-of-origin DNA methylation analysis reveals the intricacies of human imprinting and suggests a germline methylation-independent mechanism of establishment. Genome Res. <i>24</i>, 554-569. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref5">View at publisher</a></p>
<p id="bib6">Deaton, A.M. and Bird, A. (2011). CpG islands and the regulation of transcription. Genes Dev. <i>25</i>, 1010-1022. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref6">View at publisher</a></p>
<p id="bib7">Guo, W., Zhu, P., Pellegrini, M., Zhang, M.Q., Wang, X., and Ni, Z. (2018). CGmapTools improves the precision of heterozygous SNV calls and supports allele-specific methylation detection and visualization in bisulfite-sequencing data. Bioinformatics <i>34</i>, 381-387. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref7">View at publisher</a></p>
<p id="bib8">Hon, G.C., Rajagopal, N., Shen, Y., McCleary, D.F., Yue, F., Dang, M.D., and Ren, B. (2013). Epigenetic memory at embryonic enhancers identified in DNA methylation maps from adult mouse tissues. Nat. Genet. <i>45</i>, 1198-1206. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref8">View at publisher</a></p>
<p id="bib9">Inoue, A., Jiang, L., Lu, F., Suzuki, T., and Zhang, Y. (2017). Maternal H3K27me3 controls DNA methylation-independent imprinting. Nature <i>547</i>, 419-424. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref9">View at publisher</a></p>
<p id="bib10">Joshi, R.S., Garg, P., Zaitlen, N., Lappalainen, T., Watson, C.T., Azam, N., Ho, D., Li, X., Antonarakis, S.E., Brunner, H.G., et al. (2016). DNA methylation profiling of uniparental disomy subjects provides a map of parental epigenetic bias in the human genome. Am. J. Hum. Genet. <i>99</i>, 555-566. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref10">View at publisher</a></p>
<p id="bib11">Krueger, F. and Andrews, S.R. (2011). Bismark: a flexible aligner and methylation caller for bisulfite-seq applications. Bioinformatics <i>27</i>, 1571-1572. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref11">View at publisher</a></p>
<p id="bib12">Leng, L., Sun, J., Huang, J., Gong, F., Yang, L., Zhang, S., Yuan, X., Fang, F., Xu, X., Luo, Y., et al. (2019). Single-cell transcriptome analysis of uniparental embryos reveals parent-of-origin effects on human preimplantation development. Cell Stem Cell <i>25</i>, 697-712.e6. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref12">View at publisher</a></p>
<p id="bib13">Wang, L., Zhang, J., Duan, J., Gao, X., Zhu, W., Lu, X., Yang, L., Zhang, J., Li, G., Ci, W., et al. (2014). Programming and inheritance of parental DNA methylomes in mammals. Cell <i>157</i>, 979-991. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref13">View at publisher</a></p>
<p id="bib14">Zink, F., Magnusdottir, D.N., Magnusson, O.T., Walker, N.J., Morris, T.J., Sigurdsson, A., Halldorsson, G.H., Gudjonsson, S.A., Melsted, P., Ingimundardottir, H., et al. (2018). Insights into imprinting from parent-of-origin phased methylomes and transcriptomes. Nat. Genet. <i>50</i>, 1542-1552. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00120-4/sref14">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yi Zhang (<a href="mailto:yzhang@genetics.med.harvard.edu">yzhang@genetics.med.harvard.edu</a>)</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets.</p>
<p>To help the users get started, we listed several public datasets of DNA methylomes in human, monkey and mouse with accession numbers provided in <a href="#key-resources-table">key resources table</a>.</p>
<p>The TARSII and CARSII packages are available for download at <a href="https://doi.org/10.5281/zenodo.5484230">https://doi.org/10.5281/zenodo.5484230</a>. Additional script for integration analysis of CGmapTools and TARSII/CARSII in this protocol is available through <a href="#appsec2">supplemental information</a> or Mendeley Data (<a href="https://doi.org/10.17632/747r4k4mnz.1">https://doi.org/10.17632/747r4k4mnz.1</a>)</p>
<h3>Supplemental information</h3>
<p><a id="mmc1" href="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1502-Mmc1.zip">Data S1. Extract SNPs from DMRs, related to step 14</a></p>
<h3>Acknowledgments</h3>
<p>This work is supported by <a href="https://doi.org/10.13039/100000002">NIH</a> (R01HD092465). Y.Z. is an investigator of the Howard Hughes Medical Institution.</p>
<h3>Author contributions</h3>
<p>Conceptualization, Y.Z.; methodology, W.Z.; data analysis, W.Z.; writing, W.Z. and Y.Z.; supervision, Y.Z.; funding acquisition, Y.Z.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>