<article>
<section>
<h1 id="header">Primary culture of immature, naïve mouse CD4<sup>+</sup> T cells</h1>
<p><time datetime="2021-08-20">Published: August 20, 2021</time></p>
<p>Wei Wang<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#cor1">*</a></sup> and Xingbin Ai<sup><a href="#aff1">1</a>,<a href="#fn2">3</a>,<a href="#cor2">**</a></sup></p>
<p id="aff1"><sup>1</sup>Division of Neonatology, Department of Pediatrics, Massachusetts General Hospital, Boston, MA 02115, USA</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:wwang@mgh.harvard.edu">wwang@mgh.harvard.edu</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:xai@partners.org">xai@partners.org</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100756">10.1016/j.xpro.2021.100756</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Primary T-cell culture is an invaluable model for investigating mechanisms underlying T-cell differentiation and function in health and disease. However, different culture conditions are required for immature versus mature CD4<sup>+</sup> T cells. Here, we provide an improved culture protocol for immature naïve mouse CD4<sup>+</sup> T cells, including details for splenocyte isolation, naïve CD4<sup>+</sup> T-cell purification and differentiation, and functional evaluation via flow cytometry. This protocol can also be applied for immature human CD4<sup>+</sup> T cells.</p>
<p>For complete details on the execution of this protocol, please refer to <a href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref12">Wang et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>We provide a protocol of immature naïve CD4<sup>+</sup> T cell culture</li>
<li>This protocol can be used for mouse and human immature CD4<sup>+</sup> T cells</li>
<li>We give details of cytokine and transcriptional factor detection by flow cytometry</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol below describes specific steps for the isolation and culture of naïve CD4<sup>+</sup> T cells from neonatal mouse (both male and female) spleen at postnatal day 21 (P21) and provides details of suboptimal differentiation conditions for effector Th cells such that neurotransmitter modulators of differentiation can be identified. Because of a limited number of splenocytes from each spleen at this age, we pool cells from three P21 mice in one batch of isolation. Investigators should be proficient with the use of a flow cytometer with large multi-parameter capabilities (e.g., LSRFortessa) and data analysis by FlowJo. Slight modification of this protocol permits primary culture of CD4<sup>+</sup> T cells isolated from lymph nodes associated with lung samples from human donors under 13 years of age.</p>
<h3 id="sec1.1">Culture medium and buffer preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol>
<li>See <a href="#materials-and-equipment">materials and equipment</a> for preparation of the required culture medium and buffers.</li>
<li>Store the medium and buffers at 4°C for up to 1 month.</li>
</ol>
<h3 id="sec1.2">Recombinant protein stock preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="3">
<li>Dissolve all the recombinant proteins in sterile 1×PBS (pH 7.4) at 100 × working concentration and make aliquots. The working concentration for each recombinant protein is listed in <a href="#tbl5">Tables 5</a>, <a href="#tbl6">6</a>, <a href="#tbl7">7</a>, and <a href="#tbl8">8</a>.</li>
<li>Keep one aliquot of each recombinant protein at 4°C for up to 1 month and store the rest at −80°C for up to 1 year.</li>
</ol>
<h3 id="sec1.3">Chemical stock solution preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="5">
<li>Dissolve the Concanavalin A (ConA, in water), Phorbol 12-myristate 13-acetate (PMA, in dimethyl sulfoxide) and Ionomycin calcium salt (in dimethyl sulfoxide) at 100 × working concentration and make aliquots. The working concentration for each recombinant protein is listed in <a href="#tbl5">Tables 5</a>, <a href="#tbl6">6</a>, <a href="#tbl7">7</a>, and <a href="#tbl8">8</a>.</li>
<li>Store them at −20°C for up to 1 year.</li>
</ol>
<h3 id="sec1.4">Pre-coat the 48-well tissue culture plate with αCD3</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol start="7">
<li>Dilute αCD3e in sterile 1×PBS (pH 7.4) to a final concentration of 1 μg/mL.</li>
<li>Add 300 μL to one well of a 48-well tissue culture plate and seal the plate with parafilm.</li>
<li>Incubate in the fridge at 4°C overnight (15–18 h).</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> The coating can also be done by incubating at room temperature (20°C–22°C) for 1 h or in a cell incubator for 2–3 hrs.</div>
<h3 id="sec1.5">Disinfection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min</div>
<ol start="10">
<li>Autoclave scissors and forceps and keep them sterile.</li>
<li>Mix 350 mL ethanol with water to make a 500 mL of 70% ethanol solution and store at room temperature.</li>
<li>Turn on the UV light in the tissue culture cabinet for at least 15 min.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Mouse monoclonal anti-Stat5 (Y694), FITC, cloneSRBCZX, 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#11-9010-41; RRID: AB_2572520</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD4, APC, clone RM4-5, 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#17-0042-82; RRID: AB_469323</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD62L, PE, clone MEL-14, 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#12-0621-81; RRID: AB_465720</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD25, Alexa Fluor® 488, clone eBio3C7 (3C7), 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#53-0253-82; RRID: AB_763471</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD44, APC, clone IM7, 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#17-0441-81; RRID: AB_469389</td>
</tr>
<tr>
<td>Rat monoclonal anti-IL-13, PE, clone eBio13A, 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#12-7133-41; RRID: AB_10852712</td>
</tr>
<tr>
<td>Rat monoclonal anti-IL-17, PerCP-Cy5.5, clone eBio17B7, 1:200 dilution</td>
<td>eBioscience</td>
<td>Cat#45-7177-80; RRID: AB_925754</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD4, PE-Cy7, clone GK1.5, 1:200 dilution</td>
<td>BioLegend</td>
<td>Cat#100421; RRID: AB_312706</td>
</tr>
<tr>
<td>Rat monoclonal anti-CD45, PE-Cy7, clone 30-F11, 1:200 dilution</td>
<td>BioLegend</td>
<td>Cat#103114; RRID: AB_312979</td>
</tr>
<tr>
<td>Rat monoclonal anti-IFN-γ, FITC, clone XMG1.2, 1:200 dilution</td>
<td>BD Biosciences</td>
<td>Cat#562019; RRID: AB_10893998</td>
</tr>
<tr>
<td>Armenian hamster monoclonal anti-CD3e, clone 145-2C11</td>
<td>BD Biosciences</td>
<td>Cat#553058; RRID: AB_394591</td>
</tr>
<tr>
<td>Armenian hamster monoclonal anti-CD28, clone 37.51</td>
<td>BD Biosciences</td>
<td>Cat#553295; RRID: AB_394764</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Concanavalin A (ConA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#L7647-100MG</td>
</tr>
<tr>
<td>Phorbol 12-myristate 13-acetate (PMA)</td>
<td>Sigma-Aldrich</td>
<td>Cat#P8139-1MG</td>
</tr>
<tr>
<td>Ionomycin calcium salt</td>
<td>Sigma-Aldrich</td>
<td>Cat#I0634-1MG</td>
</tr>
<tr>
<td>2-Mercaptoethanol</td>
<td>Sigma-Aldrich</td>
<td>Cat#M6250-100ML</td>
</tr>
<tr>
<td>Dimethyl sulfoxide</td>
<td>Sigma-Aldrich</td>
<td>Cat# D8418-500ML</td>
</tr>
<tr>
<td>Methanol</td>
<td>Fisher Scientific</td>
<td>Cat#AC326950010</td>
</tr>
<tr>
<td>Ethanol</td>
<td>Fisher Scientific</td>
<td>Cat#BP2818-4</td>
</tr>
<tr>
<td>Ethylenediamine tetraacetate (EDTA)</td>
<td>Fisher Scientific</td>
<td>Cat#BP2483500</td>
</tr>
<tr>
<td>Bovine serum albumin (BSA)</td>
<td>Fisher Scientific</td>
<td>Cat#BP1605100</td>
</tr>
<tr>
<td>Mouse rIL-4</td>
<td>BioLegend</td>
<td>Cat#574302</td>
</tr>
<tr>
<td>Mouse rIL-12</td>
<td>R&amp;D Systems</td>
<td>Cat#419-ML-010</td>
</tr>
<tr>
<td>Mouse rIL-6</td>
<td>R&amp;D Systems</td>
<td>Cat#406-ML-005</td>
</tr>
<tr>
<td>Mouse rIL-23</td>
<td>R&amp;D Systems</td>
<td>Cat#1887-ML-010</td>
</tr>
<tr>
<td>Human rTGF-β</td>
<td>R&amp;D Systems</td>
<td>Cat#240-B-002</td>
</tr>
<tr>
<td>Human rIL-2</td>
<td>R&amp;D Systems</td>
<td>Cat#202-IL-010</td>
</tr>
<tr>
<td>RPMI 1640 with L-glutamine</td>
<td>Gibco</td>
<td>Cat#11875-119</td>
</tr>
<tr>
<td>Fetal bovine serum</td>
<td>Gibco</td>
<td>Cat#10437-028</td>
</tr>
<tr>
<td>pH7.4 1×PBS w/o Ca and Mg</td>
<td>Gibco</td>
<td>Cat#10010-072</td>
</tr>
<tr>
<td>Penicillin-streptomycin solution</td>
<td>VWR</td>
<td>Cat#16777-164</td>
</tr>
<tr>
<td>RNaseOUT™ Recombinant RNase Inhibitor</td>
<td>Thermo Fisher</td>
<td>Cat#10777-019</td>
</tr>
<tr>
<td>DTT (0.1 M, in Superscript III Reverse Transcriptase Kit)</td>
<td>Thermo Fisher</td>
<td>Cat#18080-044</td>
</tr>
<tr>
<td>dNTPs</td>
<td>NEB</td>
<td>Cat# N0446S</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Red blood cell lysis buffer</td>
<td>Sigma-Aldrich</td>
<td>Cat#R7757</td>
</tr>
<tr>
<td>Paraformaldehyde 4% in PBS</td>
<td>VWR</td>
<td>Cat#AAJ61899-AP</td>
</tr>
<tr>
<td>Cell Stimulation Cocktail (plus protein transport inhibitors)</td>
<td>eBioscience</td>
<td>Cat#00-4975-03</td>
</tr>
<tr>
<td>Fixable Viability Dye eFluor 780</td>
<td>eBioscience</td>
<td>Cat#65-0865-14</td>
</tr>
<tr>
<td>Fixation/permeabilization solution</td>
<td>BD Biosciences</td>
<td>Cat#554722</td>
</tr>
<tr>
<td>Perm/Wash solution</td>
<td>BD Biosciences</td>
<td>Cat#554723</td>
</tr>
<tr>
<td>Naive CD4<sup>+</sup> T cells isolation kit, mouse</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-104-453</td>
</tr>
<tr>
<td>LS columns</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-401</td>
</tr>
<tr>
<td>MidiMACS Separator</td>
<td>Miltenyi Biotec</td>
<td>Cat#130-042-302</td>
</tr>
<tr>
<td>RNeasy Kit</td>
<td>QIAGEN</td>
<td>Cat#74106</td>
</tr>
<tr>
<td>QIAshredder</td>
<td>QIAGEN</td>
<td>Cat#79656</td>
</tr>
<tr>
<td>Superscript III Reverse Transcriptase Kit</td>
<td>Thermo Fisher</td>
<td>Cat#18080-044</td>
</tr>
<tr>
<td>TaqMan® Fast Universal PCR Master Mix (2<b>×</b>)</td>
<td>Thermo Fisher</td>
<td>Cat#4366073</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>Mouse: C57BL/6</td>
<td>The Jackson Laboratory</td>
<td>Cat#000664</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Mouse <i>Il13</i> TaqMan primers with probe</td>
<td>Thermo Fisher</td>
<td>Cat#Mm00434204_m1</td>
</tr>
<tr>
<td>Mouse <i>Ifng</i> TaqMan primers with probe</td>
<td>Thermo Fisher</td>
<td>Cat#Mm01168134_m1</td>
</tr>
<tr>
<td>Mouse <i>Il17</i> TaqMan primers with probe</td>
<td>Thermo Fisher</td>
<td>Cat#Mm00439618_m1</td>
</tr>
<tr>
<td>Mouse <i>18s</i> TaqMan primers with probe</td>
<td>Thermo Fisher</td>
<td>Cat#Mm03928990_g1</td>
</tr>
<tr>
<td>Oligo dT</td>
<td>Invitrogen</td>
<td>Cat#18418-020</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>BD FACSDiva Software v8.0</td>
<td>BD</td>
<td><a href="https://www.bdbiosciences.com/cn/instruments/research/software/flow-cytometry-acquisition/bdfacsdiva-software/m/111112/resourcestools">https://www.bdbiosciences.com/cn/instruments/research/software/flow-cytometry-acquisition/bdfacsdiva-software/m/111112/resourcestools</a></td>
</tr>
<tr>
<td>FlowJo v10.7.0</td>
<td>Tree Star</td>
<td><a href="https://www.flowjo.com">https://www.flowjo.com</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>15 mL Centrifuge tubes</td>
<td>Corning</td>
<td>Cat#352196</td>
</tr>
<tr>
<td>50 mL Centrifuge tubes</td>
<td>Corning</td>
<td>Cat#430291</td>
</tr>
<tr>
<td>48-Well tissue culture plate (flatten bottom)</td>
<td>Corning</td>
<td>Cat#353230</td>
</tr>
<tr>
<td>96-Well round bottom plate</td>
<td>Corning</td>
<td>Cat#3788</td>
</tr>
<tr>
<td>5 mL Round-bottom tubes</td>
<td>Corning</td>
<td>Cat#352235</td>
</tr>
<tr>
<td>100 mm Petri dish</td>
<td>Fisher Scientific</td>
<td>Cat#FB0875712</td>
</tr>
<tr>
<td>96-well Unskirted qPCR Plates, Low-Profile</td>
<td>Bio-Rad</td>
<td>Cat#MLL-9601</td>
</tr>
<tr>
<td>Optical Flat 8-Cap Strips for qPCR plate</td>
<td>Bio-Rad</td>
<td>Cat#TCS-0803</td>
</tr>
<tr>
<td>0.22 μm Vacuum filter</td>
<td>EMD Millipore</td>
<td>Cat#SCGPU01RE</td>
</tr>
<tr>
<td>70 μm Cell strainer</td>
<td>CELLTREAT</td>
<td>Cat#229483</td>
</tr>
<tr>
<td>Hemocytometer</td>
<td>Bulldog Bio</td>
<td>Cat#DHC-N51</td>
</tr>
<tr>
<td>1.5 mL Centrifuge tubes</td>
<td>USA Scientific</td>
<td>Cat#1615-5500</td>
</tr>
<tr>
<td>Centrifuge</td>
<td>Eppendorf</td>
<td>5810R</td>
</tr>
<tr>
<td>Cell culture CO<sub>2</sub> incubator</td>
<td>Thermo Fisher</td>
<td>HEPA Class100</td>
</tr>
<tr>
<td>Thermal cycler</td>
<td>Bio-Rad</td>
<td>C1000 Touch</td>
</tr>
<tr>
<td>NanoDrop™ 2000 Spectrophotometer</td>
<td>Thermo Fisher</td>
<td>ND-2000</td>
</tr>
<tr>
<td>Real-time PCR System</td>
<td>Bio-Rad</td>
<td>CFX96</td>
</tr>
<tr>
<td>BD LSRFortessa flow cytometer</td>
<td>BD</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<div class="critical">
<span class="critical-title">Critical:</span> 2-mercaptoethanol is toxic. It can cause irritation to the nasal passageway and respiratory tract upon inhalation, irritation to the skin upon contact, vomiting and stomach pain through ingestion, and potential death if severe exposure occurs. 2-mercaptoethanol should be handled under the fume hood.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Neonatal mouse spleen isolation and single splenocyte suspension preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 45–55 min</div>
<p>This step contains information on how to isolate the mouse spleen and how to prepare single splenocyte suspension.</p>
<ol>
<li>Euthanize P21 mice by CO<sub>2</sub> inhalation.</li>
<li>Lay a mouse on its back on a dissection mat, pin the paws on the mat, and spray with 70% ethanol to wet the fur.</li>
<li>Make an incision in the peritoneum and cut the inferior vena cava to bleed the mouse to reduce the red blood cells left in the spleen.</li>
<li>Gently lift the spleen by a pair of forceps and remove the attached pancreatic tissue with scissors. Transfer the spleen to a 15 mL collection tube containing 5 mL of T cell culture medium (<a href="#tbl3">Table 3</a>) with forceps.</li>
<li>Pour the spleen with T cell culture medium into a 100 mm Petri dish.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To maintain the aseptic condition, work in the tissue culture hood.</div>
<ol start="6">
<li>Put a 70 μm cell strainer on top of a 50 mL conical tube and rinse the cell strainer with 1 mL ice cold 1×PBS (pH 7.4).</li>
<li>Smash the spleen on the cell strainer using a sterile plunger of a 1 mL syringe and flush the cell strainer with 10 mL ice cold 1×PBS (pH7.4).</li>
<li>Discard the cell strainer and centrifuge the sample at 400 × <i>g</i> for 15 min at 4°C.</li>
<li>Discard the supernatant by gently flipping the 50 mL tube. Add 2 mL red blood cell lysis buffer to the tube and resuspend the splenocytes by gentle pipetting. Incubate for 3 min on ice.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Optimal lysis of erythrocytes is usually achieved by incubating at room temperature. However, for subsequent naive CD4<sup>+</sup> T cell purification, the manufacturer suggests that cells always be kept cold. So, we keep the cells on ice.</div>
<ol start="10">
<li>Add 5 mL of T cell culture medium to stop the lysis of red blood cells and centrifuge the sample at 400 × <i>g</i> for 5 min at 4°C.</li>
<li>Discard the supernatant. Resuspend the splenocytes with 10 mL of ice cold pH 7.4 1×PBS and keep the cells on ice until finishing the cell counting.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Repeat step 9–10 if the cell pellet is still red.</div>
<ol start="12">
<li>Take out 20 μL of the cell suspension, dilute 10 times in 0.1% trypan blue, and determine the splenocyte number with hemocytometer.</li>
<li>Take 7 × 10<sup>6</sup> splenocytes prior to naïve CD4<sup>+</sup> T cell purification for staining of the compensation control and for the evaluation of naive CD4<sup>+</sup> T cells percentage before purification by flow cytometry.</li>
</ol>
<h3 id="sec3.2">Neonatal naïve CD4<sup>+</sup> T-cell purification via magnetic bead-based negative selection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 25–30 min</div>
<p>This step contains information on how to purify naïve CD4<sup>+</sup> T cells from the splenocyte suspension using a magnetic bead-based isolation kit. More information about this procedure can be found on manufacturer’s website: <a href="https://www.miltenyibiotec.com/US-en/products/naive-cd4-t-cell-isolation-kit-mouse.html%23gref">https://www.miltenyibiotec.com/US-en/products/naive-cd4-t-cell-isolation-kit-mouse.html#gref</a></p>
<ol start="14">
<li>Centrifuge the sample at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant completely.</li>
<li>Resuspend the cell pellet in 400 μL of purification buffer (<a href="#tbl1">Table 1</a>) per 10⁸ total cells and then add 100 μL of Biotin-Antibody Cocktail per 10⁸ total cells. Mix well and incubate for 5 min in the refrigerator (2–8°C).
<table id="tbl1">
<caption>Table 1. Purification buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1×PBS (pH 7.4)</td>
<td>n/a</td>
<td>49.8 mL</td>
</tr>
<tr>
<td>BSA</td>
<td>0.5% (w/w)</td>
<td>0.25 g</td>
</tr>
<tr>
<td>EDTA (0.5 M)</td>
<td>2 mM</td>
<td>200 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Sterilize with 0.22 μm filter and store at 4°C for up to 1 month.</p>
</li>
<li>Add 200 μL of purification buffer, 200 μL of Anti-Biotin MicroBeads and 100 μL of CD44 MicroBeads per 10⁸ total cells in sequence. Mix them well and incubate for 10 min in the refrigerator (2–8°C).</li>
<li>Place a LS column in the magnetic field of MidiMACS Separator and prepare the LS column by rinsing with 3 mL of purification buffer.</li>
<li>Apply the cell suspension onto the column and collect the flow-through that contain unlabeled cells enriched in naïve CD4<sup>+</sup> T cells.</li>
<li>Wash the column with 3 mL of purification buffer. Collect unlabeled cells in the flow-through and combine with the flow-through from step 18.</li>
<li>Determine the cell number with hemocytometer. Take 1 × 10<sup>6</sup> cells for staining to evaluate the purity of naïve CD4<sup>+</sup> T cells by flow cytometry.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Here, we used the magnetic bead-based negative selection isolation kit to purify naïve CD4<sup>+</sup> T cells because it is effective and time saving. Magnetic bead-based positive selection isolation kit can also be used for purification. Another alternative approach is to FACS-sort naïve CD4<sup>+</sup> T cells that yields slightly higher purity; however, it takes longer time and requires a cell sorter. As few as 0.2 million cells can be used for staining to evaluate the purity.</div>
<h3 id="sec3.3">Evaluate naïve CD4<sup>+</sup> T-cell purity by flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<p>This step contains information on antibody staining to evaluate the purity of the isolated naïve CD4<sup>+</sup> T cells by flow cytometry. In total, 7 × 10<sup>6</sup> splenocytes (before purification) and 1 × 10<sup>6</sup> purified naïve CD4<sup>+</sup> T cells will be used. <a href="#fig1">Figure 1</a> shows the allocation of these cells for different staining in a 96-well plate.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Layout of a 96-well round bottom plate setup for the staining of compensation controls, splenocytes and naïve CD4<sup>+</sup> T cells after enrichment</div>
</figcaption>
</figure>
<ol start="21">
<li>Take 1 × 10<sup>6</sup> splenocytes and fix them with 4% PFA for 10 min on ice to generate a positive control for dead cell staining.</li>
<li>Wash twice of the fixed cells by adding 300 μL of FACS buffer (<a href="#tbl2">Table 2</a>). Centrifuge at 400 × <i>g</i> for 5 min at 4°C to collect the cell pellet.
<table id="tbl2">
<caption>Table 2. FACS buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>1×PBS (pH 7.4)</td>
<td>n/a</td>
<td>49 mL</td>
</tr>
<tr>
<td>FBS</td>
<td>2% (v/v)</td>
<td>1 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>50 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 month.</p>
</li>
<li>Mix fixed cells with 1 × 10<sup>6</sup> unfixed splenocytes and use this sample as a single-stained cell compensation control for Fixable Viability Dye.</li>
<li>Transfer 1 × 10<sup>6</sup> unfixed cells to each allocated well of a 96-well round bottom plate and centrifuge the plate at 400 × <i>g</i> for 5 min at 4°C. Discard the supernatant completely.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During the following steps, cells have to be protected from light with aluminum foil.</div>
<ol start="25">
<li>During the spin, prepare Live/Dead Fixable Viability Dye eFluor 780 staining solution by diluting 1 μL of a concentrated stock with 2 mL of 1×PBS (pH 7.4).</li>
<li>Resuspend the cell pellet in one well as the compensation control for Fixable Viability Dye staining and the two wells for multiple staining with 100 μL of diluted Live/Dead Fixable Viability Dye eFluor 780. Resuspend the cell pellet in other wells with 200 μL 1×PBS (pH 7.4) and incubate for 30 min at 4°C.</li>
<li>During the incubation, prepare multiple staining antibody solution as shown in <a href="#tbl4">Table 4</a> and single staining solutions.
<table id="tbl3">
<caption>Table 3. T cell culture medium</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>RPMI 1640 with L-glutamine</td>
<td>n/a</td>
<td>445 mL</td>
</tr>
<tr>
<td>Penicillin-streptomycin solution (10×)</td>
<td>1×</td>
<td>5 mL</td>
</tr>
<tr>
<td>2-mercaptoethanol (1 M)</td>
<td>50 μM</td>
<td>25 μL</td>
</tr>
<tr>
<td>FBS</td>
<td>10% (v/v)</td>
<td>50 mL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>500 mL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Store at 4°C for up to 1 month.</p>
<table id="tbl4">
<caption>Table 4. Antibody mix for cell surface marker staining of splenocytes</caption>
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Final dilution</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PE-Cy7</td>
<td>CD4</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>PE</td>
<td>CD62L</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>FITC</td>
<td>CD25</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>APC</td>
<td>CD44</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>FACS buffer</td>
<td>n/a</td>
<td>n/a</td>
<td>98 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
<p class="table-legend">Prepare immediately before use.</p>
</li>
<li>Centrifuge the plate at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant. Wash the cell pellet twice using 300 μL of FACS buffer.</li>
<li>Resuspend the cell pellet in the two wells for multiple staining with 100 μL of antibody mix 1 solution and cell pellets in other wells for single staining with 100 μL of respective single staining solution. Resuspend the cell pellet in the well for Fixable Viability Dye staining with 100 μL of FACS buffer. Incubate for 30 min at 4°C.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> According to the manufacturer’s protocol, it is best to stain with Live/Dead Fixable Viability Dye in azide and protein-free PBS for the brightest staining.</div>
<ol start="30">
<li>Wash twice by adding 300 μL of FACS buffer, centrifuge the plate at 400 × <i>g</i> for 5 min at 4°C, and discard the supernatant.</li>
<li>Resuspend the cell pellet in 300 μL of FACS buffer. Passage the cell suspension through the 40 μm cell strainer on top of the 5 mL Falcon® tube before analysis with Fortessa 405/488/561/640 (BD Biosciences) flow cytometer.</li>
<li>See <a href="#fig2">Figure 2</a> for the gating strategy of naïve CD4<sup>+</sup> T cell analysis with Flowjo software. Purity over 95% is considered acceptable. If the purity is less than 95%, another round of enrichment is suggested.
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Gating strategy used to evaluate the purity of naïve CD4<sup>+</sup> T cells before and after enrichment</div>
<p>(A) Representative result of naïve CD4<sup>+</sup> T cells purity within splenocytes from C57BL/6J mouse at P21.</p>
<p>(B) Representative result of naïve CD4<sup>+</sup> T cells purity after enrichment.</p>
</figcaption>
</figure>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> OneComp eBeads™ (01-1111-42, Invitrogen) can also be used for compensation purposes. A mouse, rat or hamster origin antibody with APC-Cy7 labeling should be used for the compensation of Fixable Viability Dye eFluor 780.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Since some surface markers, like CD62L, are temperature sensitive, always keep the plate on ice during the process of staining and use a centrifuge setting at 4°C.</div>
<h3 id="sec3.4">Neonatal naïve CD4<sup>+</sup> T-cell differentiation <i>in vitro</i>
</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<p>This step contains information on differentiation conditions of neonatal naïve CD4<sup>+</sup> T cells.</p>
<ol start="33">
<li>Centrifuge the purified naïve CD4<sup>+</sup> T cells at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant completely.</li>
<li>During the spin, wash the αCD3 pre-coated, 48-well tissue culture plate with 1mL 1×PBS (pH 7.4).</li>
<li>Resuspend the cell pellet with T cell culture medium (<a href="#tbl3">Table 3</a>) and adjust the cell density to 1 × 10<sup>6</sup> cells/mL.</li>
<li>Seed 5 × 10<sup>5</sup> cells in 0.5 mL cell suspension in each well of the 48-well tissue culture plate.</li>
<li>For Th0, Th1, Th2 and Th17 cultures, add respective antibodies, chemicals and recombinant proteins following the instructions in <a href="#tbl5">Tables 5</a>, <a href="#tbl6">6</a>, <a href="#tbl7">7</a>, and <a href="#tbl8">8</a>.
<table id="tbl5">
<caption>Table 5. Th0 culture recipe (suboptimal condition)</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PMA</td>
<td>100 nM</td>
<td>1 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>ConA</td>
<td>50 μg/mL</td>
<td>0.5 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Ionomycin</td>
<td>25 μM</td>
<td>250 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>αCD28</td>
<td>500 μg/mL</td>
<td>1 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>Human rIL-2</td>
<td>500 ng/mL</td>
<td>5 ng/mL</td>
<td>5 μL</td>
</tr>
</tbody>
</table>
<table id="tbl6">
<caption>Table 6. Th1 culture recipe (suboptimal condition)</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PMA</td>
<td>100 nM</td>
<td>1 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>ConA</td>
<td>50 μg/mL</td>
<td>0.5 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Ionomycin</td>
<td>25 μM</td>
<td>250 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>αCD28</td>
<td>500 μg/mL</td>
<td>1 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>Human rIL-2</td>
<td>500 ng/mL</td>
<td>5 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Mouse rIL-12</td>
<td>1 μg/mL</td>
<td>10 ng/mL</td>
<td>5 μL</td>
</tr>
</tbody>
</table>
<table id="tbl7">
<caption>Table 7. Th2 culture recipe (suboptimal condition)</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PMA</td>
<td>100 nM</td>
<td>1 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>ConA</td>
<td>50 μg/mL</td>
<td>0.5 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Ionomycin</td>
<td>25 μM</td>
<td>250 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>αCD28</td>
<td>500 μg/mL</td>
<td>1 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>Human rIL-2</td>
<td>500 ng/mL</td>
<td>5 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Mouse rIL-4</td>
<td>1 μg/mL</td>
<td>10 ng/mL</td>
<td>5 μL</td>
</tr>
</tbody>
</table>
<table id="tbl8">
<caption>Table 8. Th17 culture recipe (suboptimal condition)</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Final concentration</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PMA</td>
<td>100 nM</td>
<td>1 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>ConA</td>
<td>50 μg/mL</td>
<td>0.5 μg/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Ionomycin</td>
<td>25 μM</td>
<td>250 nM</td>
<td>5 μL</td>
</tr>
<tr>
<td>αCD28</td>
<td>500 μg/mL</td>
<td>1 μg/mL</td>
<td>1 μL</td>
</tr>
<tr>
<td>Human rIL-2</td>
<td>500 ng/mL</td>
<td>5 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Human rTGF-β</td>
<td>200 ng/mL</td>
<td>2 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Mouse rIL-6</td>
<td>2 μg/mL</td>
<td>20 ng/mL</td>
<td>5 μL</td>
</tr>
<tr>
<td>Mouse rIL-23</td>
<td>1 μg/mL</td>
<td>10 ng/mL</td>
<td>5 μL</td>
</tr>
</tbody>
</table>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Prepare immediately before use and add the reagents individually to the culture medium. As human IL-2 stimulates proliferation of mouse T cells at similar concentration, we used human IL-2 in our protocol. Alternatively, mouse IL-2 can also be used at the same concentration.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare immediately before use and add reagents individually to the culture medium. As human IL-2 stimulates proliferation of mouse T cells at similar concentration, we used human IL-2 in our protocol. Alternatively, mouse IL-2 can also be used at the same concentration.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare immediately before use and add reagents individually to the culture medium. As human IL-2 stimulates proliferation of mouse T cells at similar concentration, we used human IL-2 in our protocol. Alternatively, mouse IL-2 can also be used at the same concentration.</div>
<div class="note">
<span class="note-title">Note:</span> Prepare immediately before use and add reagents individually to the culture medium. As human IL-2 stimulates proliferation of mouse T cells at similar concentration, we used human IL-2 in our protocol. Alternatively, mouse IL-2 can also be used at the same concentration. The optimal working concentration of TGF-β should be titrated by the operator, since it can vary among different batches. As naive CD4<sup>+</sup> T cells do not express IL-23R and IL-23 promotes maintenance but not commitment to the Th17 lineage (<a href="#bib11">Stritesky et al., 2008</a>), add IL-23 after 2 days. Only adding the 5μL of IL-23 to each well with the already existing culture medium containing the other reconbimants/reagents.</div>
<div class="note">
<span class="note-title">Note:</span> Add IL-23 after 2 days.</div>
<ol start="38">
<li>Put the plate into a CO<sub>2</sub> incubator and culture for 4 days at 37°C with 5% CO<sub>2</sub>.</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> we have employed a suboptimal differentiation condition in our screen for neurotransmitters that can modulate CD4<sup>+</sup> T cell differentiation. Standard T cell differentiation can be achieved by adding anti-mIL-4 antibody (10 μg/mL) to Th1 culture, anti-mIFN-γ antibody (10 μg/mL) to Th2 culture, anti-mIL-4 antibody (10 μg/mL) and anti-mIFN-γ antibody (10 μg/mL) to Th17 culture at the beginning of each culture.</div>
<h3 id="sec3.5">Evaluate T-cell differentiation by cytokine gene expression assays via qPCR</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3.5–4 h</div>
<p>This step contains information on RNA extraction and quantification of cytokine gene expression by qPCR to evaluate T cell differentiation.</p>
<ol start="39">
<li>RNA extraction. More information about RNA extraction can be found on manufacturer’s website: <a href="https://www.qiagen.com/us/resources/resourcedetail?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en">https://www.qiagen.com/us/resources/resourcedetail?id=14e7cf6e-521a-4cf7-8cbc-bf9f6fa33e24&amp;lang=en</a>
<ol type="a">
<li>Determine the cell number at the end of T cell culture. Collect cells (no more than 5 × 10<sup>6</sup> cells) in a 1.5 mL centrifuge tube by spinning for 5 min at 400 × <i>g</i>. Carefully discard all the supernatant.
<div class="note">
<span class="note-title">Note:</span> Incomplete removal of the supernatant will inhibit lysis and dilute the lysate. These factors may reduce the RNA yield.</div>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Proceed with RNA extraction following manufacturer’s protocol or snap freeze the cell pellet in liquid nitrogen before storage at −80°C.</div>
</li>
<li>Flick the tube to thoroughly loosen the cell pellet and add 350 μL Buffer RLT. Mix well by pipetting.</li>
<li>Transfer the lysate into a QIAshredder spin column placed in a 2 mL collection tube and centrifuge for 2 min at 8000 × <i>g</i>.</li>
<li>Add 1 volume of 70% ethanol to the homogenized lysate, and pipet up and down to mix well.
<div class="critical">
<span class="critical-title">Critical:</span> Do not centrifuge.</div>
</li>
<li>Transfer up to 700 μL of the sample to an RNeasy spin column placed in a 2 mL collection tube. Centrifuge for 15 s at 8000 × <i>g</i>. Discard the flow-through.
<div class="optional">
<span class="optional-title">Optional:</span> If performing optional on-column DNase digestion to eliminating genomic DNA contamination, follow Appendix D in manufacturer’s protocol after performing this step.</div>
</li>
<li>Add 700 μL Buffer RW1 to the column and centrifuge for 15 s at 8000 × <i>g</i> to wash the column. Discard the flow-through.</li>
<li>Add 500 μL Buffer RPE to the column and centrifuge for 15 s at 8000 × <i>g</i> to wash the column. Discard the flow-through.</li>
<li>Add 500 μL Buffer RPE to the column and centrifuge for 2 min at 8000 × <i>g</i> to wash the column.
<div class="optional">
<span class="optional-title">Optional:</span> Place the column into a new 2 mL collection tube and centrifuge at 8000 <b>×</b> <i>g</i> for 1 min to completely remove Buffer RPE.</div>
</li>
<li>Place the RNeasy spin column into a new 1.5 mL collection tube. Add 30 μL RNase-free water to the spin column membrane and centrifuge for 1 min at 8000 × <i>g</i> to elute the RNA.</li>
</ol>
</li>
</ol>
<ol start="40">
<li>First-strand cDNA synthesis. More information about the first-strand cDNA synthesis procedure can be found on manufacturer’s website: <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/superscriptIII_man.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/superscriptIII_man.pdf</a>
<ol type="a">
<li>Determine the RNA concentration of each sample by NanoDrop™ 2000 spectrophotometer.</li>
<li>Follow <a href="#tbl9">Table 9</a> to make the mix for step 1.
<table id="tbl9">
<caption>Table 9. cDNA mix 1</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>Total RNA</td>
<td>n/a</td>
<td>1 μg (Volume may vary among samples)</td>
</tr>
<tr>
<td>oligo(dT)<sub>20</sub>
</td>
<td>50 μM</td>
<td>1 μL</td>
</tr>
<tr>
<td>dNTP Mix</td>
<td>10 mM</td>
<td>1 μL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>n/a</td>
<td>To 12 μL</td>
</tr>
</tbody>
</table>
<div class="note">
<span class="note-title">Note:</span> According to the manufactory’s protocol, the amount of starting material can vary from 1 pg–5 μg of total RNA. In our protocol, we used 1 μg of total RNA.</div>
</li>
<li>Add the above components to a nuclease-free microcentrifuge tube. Heat mixture to 65°C for 5 min and incubate on ice for at least 1 min.</li>
<li>During the incubation, follow <a href="#tbl10">Table 10</a> to prepare the mix for step 2.
<table id="tbl10">
<caption>Table 10. cDNA mix 2</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>First-Strand Buffer</td>
<td>5 X</td>
<td>4 μL</td>
</tr>
<tr>
<td>DTT</td>
<td>0.1 M</td>
<td>2 μL</td>
</tr>
<tr>
<td>RNaseOUT™ Inhibitor</td>
<td>40 units/μL</td>
<td>1 μL</td>
</tr>
<tr>
<td>SuperScript™ III RT</td>
<td>200 units/μL</td>
<td>1 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Collect the contents of the tubes by brief centrifugation and add cDNA mix 2 to the tubes.</li>
<li>Mix by gently pipetting up and down and incubate at 50°C for 50 min.</li>
<li>Inactivate the reaction by heating at 85°C for 5 min and hold at 4°C.</li>
</ol>
</li>
<li>PCR reaction. More information about PCR reaction procedure can be found on manufacturer’s website: <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/cms_042252.pdf">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/cms_042252.pdf</a>
<ol type="a">
<li>Dilute the cDNA template to 10 ng/μL with distilled water.</li>
<li>Follow <a href="#tbl11">Table 11</a> to make the mix for PCR reaction. Mix TaqMan® PCR Master Mix with different TaqMan® primers first and transfer them to a 96 well Unskirted qPCR Plate. Add diluted cDNA templates to respective wells.
<table id="tbl11">
<caption>Table 11. PCR reaction mix</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Amount per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>TaqMan® Fast Universal PCR Master Mix (2✕)</td>
<td>2×</td>
<td>10 μL</td>
</tr>
<tr>
<td>TaqMan® primers with probe</td>
<td>20 ×</td>
<td>1 μL</td>
</tr>
<tr>
<td>cDNA template + RNase-free water</td>
<td>10 ng/μL</td>
<td>9 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td>n/a</td>
<td>20 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Cover the qPCR plate with optical flat 8-cap strips and centrifuge at 600 × <i>g</i> for 1 min.</li>
<li>Follow <a href="#tbl12">Table 12</a> to set up the PCR program in a real-time PCR machine.
<table id="tbl12">
<caption>Table 12. PCR program</caption>
<thead>
<tr>
<th>Step</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Hot start</td>
<td>95°C</td>
<td>3 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>95°C</td>
<td>10 s</td>
<td rowspan="2">40</td>
</tr>
<tr>
<td>Annealing + Extension</td>
<td>60°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>∞</td>
<td>1</td>
</tr>
</tbody>
</table>
</li>
<li>See <a href="#fig4">Figure 4</a> for representative relative expression levels of <i>Il13</i>, <i>Ifng,</i> and <i>Il17</i> in CD4<sup>+</sup> T cells after differentiation for 4 days.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Here we used two-step RT-PCR. Alternatively, a one-step RT-PCR assay may be used to save time and reagent cost.</div>
<h3 id="sec3.6">Evaluate T-cell differentiation by cytokine expression assays via flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 7–7.5 h</div>
<p>This step contains information on fixation, permeabilization, antibody staining for different cytokines in differentiated CD4<sup>+</sup> T cells by flow cytometry.</p>
<ol start="42">
<li>Cell stimulation
<ol type="a">
<li>Dilute the 500× Cell Stimulation Cocktail (plus protein transport inhibitors) in the cultures to 1×.</li>
<li>Put the cultures back to CO<sub>2</sub> incubator and incubate for 4 h.</li>
<li>Collect the cells by centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend the cell pellet in 300 μL of FACS buffer and transfer to each allocated well of a 96-well round bottom plate and centrifuge the plate at 400 × <i>g</i> for 5 min at 4°C. Discard the supernatant completely.</li>
</ol>
</li>
</ol>
<ol start="43">
<li>Live/Dead cell staining and surface marker staining
<ol type="a">
<li>During the spin, prepare Live/Dead Fixable Viability Dye eFluor 780 staining solution by diluting 1 μL of original solution with 2 mL of 1×PBS (pH 7.4).
<div class="note">
<span class="note-title">Note:</span> During the following steps, cells have to be protected from light.</div>
</li>
<li>Resuspend the cell pellet with 100 μL of diluted Live/Dead Fixable Viability Dye eFluor 780 and incubate 30 min at 4°C. Keep extra cells for other compensation single staining.
<div class="critical">
<span class="critical-title">Critical:</span> Surface marker staining should be performed here with Live/Dead cell staining before fixation as fixatives can cause antigen epitope structures to be altered, which might render the antibodies unable to bind to their targets. This effect depends on the clone of antibody you're using. The antibodies we used are compatible with fixation and the surface markers staining can be performed with intracellular cytokines staining after fixation. Check this website (<a href="https://www.biolegend.com/en-us/fixation">https://www.biolegend.com/en-us/fixation</a>) or consult the literature to confirm whether the antibodies you used are compatible with fixation.</div>
</li>
<li>Wash twice by adding 300 μL of FACS buffer, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
</ol>
</li>
</ol>
<ol start="44">
<li>Fixation and permeabilization
<ol type="a">
<li>Resuspend cells in 300 μL Fixation/permeabilization solution per 1 million cells. Mix well to dissociate the cell pellet and prevent cross-linking of individual cells.</li>
<li>Fix for 30 min at 4°C.</li>
</ol>
</li>
<li>Immunostaining
<ol type="a">
<li>Wash twice by adding 300 μL of 1× Perm/Wash solution, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>During the spin, prepare multiple staining antibody solution as shown in <a href="#tbl13">Table 13</a> and single staining solution.
<table id="tbl13">
<caption>Table 13. Antibody mix for the cytokines staining of CD4<sup>+</sup> T cells</caption>
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Final dilution</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PE-Cy7</td>
<td>CD45</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>APC</td>
<td>CD4</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>FITC</td>
<td>IFN-γ</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>PE</td>
<td>IL-13</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>PerCP-Cy5.5</td>
<td>IL-17</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>1× Perm/Wash solution</td>
<td>n/a</td>
<td>n/a</td>
<td>97.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Resuspend the cell pellet in the well for multiple staining with 100 μL of antibody mix solution shown in <a href="#tbl13">Table 13</a>. Resuspend the cell pellets in the wells for single staining with 100 μL of respective single staining solution. Resuspend the cell pellet in the well for no staining and Fixable Viability Dye staining with 100 μL of 1× Perm/Wash solution. Incubate for 30 min at 4°C.</li>
<li>Wash twice by adding 300 μL of 1× Perm/Wash solution, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend the cell pellet in 300 μL of FACS buffer. Passage the cell suspension through the 40 μm cell strainer on top of the 5 mL Falcon® tube before analysis with Fortessa 405/488/561/640 (BD Biosciences) flow cytometer.</li>
<li>See <a href="#fig5">Figure 5</a> for the gating strategy of cytokines analysis in CD4<sup>+</sup> T cell with Flowjo software.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Splenocytes or OneComp eBeads™ (01-1111-42, Invitrogen) can be used for compensation. If OneComp eBeads™ are used, a mouse, rat or hamster origin antibody with APC-Cy7 labeling should be used for compensation of Fixable Viability Dye eFluor 780.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Always keep the plate on ice during the staining procedure and use a centrifuge set at 4°C.</div>
<h3 id="sec3.7">Evaluate the level of p-STAT5 in differentiated CD4<sup>+</sup> T cells by flow cytometry</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 4–4.5 h</div>
<p>This step contains information on fixation, permeabilization, antibody staining for phosphorylated STAT5 in differentiated CD4<sup>+</sup> T cells by flow cytometry.</p>
<ol start="46">
<li>Live/Dead cell staining
<ol type="a">
<li>Take 1 × 10<sup>6</sup> cells and fix them with 4% PFA for 10 min on ice to generate a positive cell control for dead cells staining.</li>
<li>Wash twice by adding 300 μL of FACS buffer, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Mix fixed cells with 1 × 10<sup>6</sup> unfixed cells and use this sample as a single-stained cell compensation control for the Fixable Viability Dye.</li>
<li>Transfer 1 × 10<sup>6</sup> cells to each labeled 1.5 mL centrifuge tube and centrifuge the sample at 400 × <i>g</i> for 5 min at 4°C. Discard the supernatant completely.
<div class="note">
<span class="note-title">Note:</span> To avoid excessive cell loss, we used 1.5 mL centrifuge tube instead of 96-well round bottom plate in this section.</div>
</li>
<li>During the spin, prepare Live/Dead Fixable Viability Dye eFluor 780 staining solution by diluting 1 μL of original solution with 2 mL of 1×PBS (pH 7.4).
<div class="note">
<span class="note-title">Note:</span> During the following steps, cells have to be protected from light.</div>
</li>
<li>Resuspend the cell pellet in the tube for multiple staining and single staining compensation control for the Fixable Viability Dye with 100 μL of diluted Live/Dead Fixable Viability Dye eFluor 780. Resuspend the cell pellets in other tubes with 1×PBS (pH 7.4) and incubate 30 min at 4°C.</li>
<li>Wash twice by adding 300 μL of FACS buffer, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
</ol>
</li>
</ol>
<ol start="47">
<li>Fixation
<ol type="a">
<li>Resuspend cells in 100 μL 4% formaldehyde per 1 million cells. Mix well to dissociate the cell pellet and prevent cross-linking of individual cells.</li>
<li>Fix for 30 min at 4°C.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Due to the nature of fixatives, they can cause antigen epitope structures to be altered, which might render the antibodies unable to bind to their targets. Surface marker staining before fixation is suggested. This effect depends on the clone of antibody you're using. The antibodies we used are compatible with fixation and the surface markers staining can be performed with intracellular cytokines staining after fixation. Check this website (<a href="https://www.biolegend.com/en-us/fixation">https://www.biolegend.com/en-us/fixation</a>) or consult the literature to confirm whether the antibodies you used are compatible with fixation.</div>
<ol start="48">
<li>Permeabilization
<ol type="a">
<li>Permeabilize cells by slowly adding ice-cold 100% methanol to pre-chilled cells while gently vortexing to a final concentration of 90% methanol.</li>
<li>Permeabilize for 45 min on ice.</li>
</ol>
</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> Proceed with immunostaining or store cells at −20°C in 90% methanol.</div>
<ol start="49">
<li>Immunostaining
<ol type="a">
<li>Wash twice by adding 300 μL of FACS buffer, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>During the spin, prepare multiple staining antibody solution as shown in <a href="#tbl14">Table 14</a> and single staining solution.
<table id="tbl14">
<caption>Table 14. Antibody mix for the p-STAT5 staining of CD4<sup>+</sup> T cells</caption>
<thead>
<tr>
<th>Fluorophore</th>
<th>Marker</th>
<th>Final dilution</th>
<th>Volume per sample</th>
</tr>
</thead>
<tbody>
<tr>
<td>PE-Cy7</td>
<td>CD45</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>FITC</td>
<td>p-STAT5</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>APC</td>
<td>CD4</td>
<td>1/200</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>FACS buffer</td>
<td>n/a</td>
<td>n/a</td>
<td>98.5 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td><b>n/a</b></td>
<td><b>n/a</b></td>
<td><b>100 μL</b></td>
</tr>
</tbody>
</table>
</li>
<li>Resuspend the cell pellet in the well for multiple staining with 100 μL of antibody mix solution. Resuspend the cell pellets in the wells for single staining with 100 μL of respective single staining solution. Resuspend the cell pellet in the well for no staining and Fixable Viability Dye staining with 100 μL of FACS buffer. Incubate for 30 min at 4°C.</li>
<li>Wash twice by adding 300 μL of FACS buffer, centrifuge at 400 × <i>g</i> for 5 min at 4°C and discard the supernatant.</li>
<li>Resuspend the cell pellet in 300 μL of FACS buffer. Passage the cell suspension through the 40 μm cell strainer on top of the 5 mL Falcon® tube before analysis with Fortessa 405/488/561/640 (BD Biosciences) flow cytometer.</li>
<li>See <a href="#fig6">Figure 6</a> for the gating strategy of p-STAT5 analysis in CD4<sup>+</sup> T cell with Flowjo software.</li>
</ol>
</li>
</ol>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> Splenocytes or OneComp eBeads™ (01-1111-42, Invitrogen) can be used for compensation. If OneComp eBeads™ are used, a mouse, rat or hamster origin antibody with APC-Cy7 labeling should be used for compensation of Fixable Viability Dye eFluor 780.</div>
<div class="critical">
<span class="critical-title">Critical:</span> Always keep the plate on ice during the staining procedure and use a centrifuge set at 4°C.</div>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>If successful, using the gating schema displayed in <a href="#fig2">Figure 2</a>B, over 95% of live cells after purification should be naïve CD4<sup>+</sup> T cells. Complete activation of T cells results in CD25 (Interleukin-2 receptor alpha chain) expression (<a href="#bib10">Schlegel, 1997</a>). CD44 localizes in lipid rafts with the T cell receptor (TCR) complex and is also up-regulated early after TCR engagement (<a href="#bib3">DeGrendele et al., 1997</a>). Therefore, a large majority of T cells (∼98%) after isolation should be CD25<sup>-</sup> and CD44<sup>-</sup> compared to a significant number of CD25<sup>+</sup> and CD44<sup>+</sup> cells before purification. L-selectin, also known as CD62L, is a cell adhesion molecule found on naïve T cells (<a href="#bib4">Goldstein et al., 1989</a>). Over 95% of cells after purification should be CD62L<sup>+</sup>.</p>
<p>After 4 days culture under Th0 condition, immature CD4<sup>+</sup> T cells from P21 mice with Con A, PMA and ionomycin stimulation should proliferate very well and form big clusters (<a href="#fig3">Figure 3</a>) like mature CD4<sup>+</sup> T cells from adult mice without Con A, PMA and ionomycin (<a href="#fig3">Figure 3</a>). Immature CD4<sup>+</sup> T cells from P21 mouse without Con A, PMA and ionomycin stimulation couldn’t proliferate and failed to form big clusters (<a href="#fig3">Figure 3</a>).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Representative images of P21 and adult mouse CD4<sup>+</sup> T cells cultured under Th0 condition with/without Con A, PMA and ionomycin for 4 days</div>
<p>Scale bar, 100 μm.</p>
</figcaption>
</figure>
<p>During TCR activation and in the presence of a particular cytokine milieu, naïve CD4<sup>+</sup> T cells differentiate into one of several lineages of T helper (Th) cells, including Th1, Th2, Th17, and iTreg, that can be defined by signature cytokine production and their unique function. Th1 cells make IFN-γ as their signature cytokine. Th2 cells predominately produce IL-4, IL-5, and IL-13 (<a href="#bib5">Killar et al., 1987</a>; <a href="#bib9">Mosmann et al., 1986</a>). Th17 cells are characterized by the production of IL-17A, IL-17F, and IL-22 as signature cytokines (<a href="#bib1">Aggarwal et al., 2003</a>). After 4 days in culture, neonatal mouse CD4<sup>+</sup> T cells should proliferate well and form clusters after adding low dosages of PMA, ConA and Ionomycin. Under the listed culture conditions in <a href="#tbl5">Tables 5</a>, <a href="#tbl6">6</a>, <a href="#tbl7">7</a>, and <a href="#tbl8">8</a>, suboptimal Th1, Th2 and Th17 differentiation should be achieved. As shown in <a href="#fig4">Figure 4</a>, robust <i>Ifng</i>, <i>Il13</i> and <i>Il17</i> mRNA elevation should be detected by q-PCR in Th1, Th2 and Th17 culture conditions, respectively. As shown in <a href="#fig5">Figure 5</a>, a substantial IFN-γ, IL-13 and IL-17 positive population should also be detected by flow cytometry in Th1, Th2 and Th17 culture conditions, respectively. However, since the differentiation condition in our assays is suboptimal, spontaneous differentiation of naïve CD4<sup>+</sup> T cells may occur at a low level. As such, signature genes of other types of Th cells may be found in a specific differentiation condition. For example, <i>Il13</i> expression may be detectable in Th1 culture, since the anti-IL-4 antibody is not added to the Th1 culture to completely block Th2 differentiation (<a href="#fig4">Figure 4</a>). A small portion of cells from Th17 condition are also IFN-γpositive as detected by flow cytometry as INF-γantibody was not added to Th17 condition (<a href="#fig5">Figure 5</a>).</p>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Representative results of relative expression level of signature cytokines in differentiated CD4<sup>+</sup> T cells of neonatal C57BL/6J mouse</div>
<p>(A) Representative result of <i>Ifng</i> mRNA level in differentiated CD4<sup>+</sup> T cells of neonatal C57BL/6J mouse.</p>
<p>(B) Representative result of <i>Il13</i> mRNA level in differentiated CD4<sup>+</sup> T cells of neonatal C57BL/6J mouse.</p>
<p>(C) Representative result of <i>Il17</i> mRNA level in differentiated CD4<sup>+</sup> T cells of neonatal C57BL/6J mouse. Data are represented as mean ± SEM.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Gating strategy for analysis of signature cytokines in neonatal mouse T helper cells cultured for 4 days</div>
</figcaption>
</figure>
<p>IL-2 promotes T cell proliferation and Th2 differentiation by transcriptionally activating the expression of the <i>Il2ra</i>, <i>Il4</i>, and <i>Il4ra</i> genes via phosphorylating STAT5 (<a href="#bib2">Cote-Sierra et al., 2004</a>; <a href="#bib13">Zhu et al., 2003</a>). The differentiated Th2 cells can produce more IL-2 to maintain the proliferation and enhance the differentiation (<a href="#bib14">Zhu et al., 2010</a>). After 4 day’s differentiation under Th2 condition, a robust signal of phosphorylated STAT5 should be detected in live CD4<sup>+</sup> T cell singlets (<a href="#fig6">Figure 6</a>). IL-2 signaling via STAT5 activation can also potentiate the Th1 fate by inducing IL12Rβ2 and T-bet expression, thereby allowing the cell to respond to IL-12 and polarize toward the Th1 fate (<a href="#bib7">Liao et al., 2011</a>). IL-2/STAT5 signaling can also affect Th17 development by downregulating expression of the IL6R, which is required to activate STAT3 (<a href="#bib6">Laurence et al., 2007</a>). Therefore, some levels of phosphorylated STAT5 can also be detected in Th1 cells, but very few phosphorylated STAT5 positive cells can be detected in Th17 cells (<a href="#fig6">Figure 6</a>).</p>
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/936-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. Gating strategy for analysis of phosphorylated STAT5 positive population in neonatal mouse in neonatal mouse T helper cells cultured for 4 days</div>
</figcaption>
</figure>
<p>By slight modification, this protocol is adaptable to the establishment of primary cultures of human naïve CD4<sup>+</sup> T cells isolated from lymph nodes associated with lung samples from donors under 13 years of age. These modifications include the method of enzymatic dissociation of the dissected lymph nodes and the antibodies against specific human epitopes. For dissociation, lymph nodes are digested with 400 U/mL Collagenase Type 4 (LS004188, Worthington-Biochem), 0.1% Dispase II (4942078001, Sigma-Aldrich) and 20 mg/mL DNase I (439807, NEB) for 30 min at 37°C. After enzymatic digestion, the sample is subjected to mechanical dissociation by pushing through a syringe with 18-gauze needle 10 times before passage through 70 μm cell strainer. Naïve CD4<sup>+</sup> T cells are enriched using a human naïve CD4<sup>+</sup> T cell isolation kit (Miltenyi Biotec, 130-094-131). The 48-well plate was pre-coated with 2 μg/mL αhCD3 (eBioscience, 16-0037-85). To induce Th2 differentiation, naïve CD4<sup>+</sup> T cells are cultured in the presence of 2 μg/mL αhCD28, 5 ng/mL rhIL-2 (R&amp;D Systems, 202-IL-010) and 10 ng/mL rhIL-4 (BioLegend, 204-IL-010). All the other reagents, including ConA, PMA, and ionomycin, should be added similarly as in the primary culture of immature mouse CD4<sup>+</sup> T cells. The Th2 differentiation is similar to mice samples.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>This protocol provides the basics to neonatal mouse CD4<sup>+</sup> T cell culture and differentiation <i>in vitro</i>. We only cultured naïve CD4<sup>+</sup> T cells from mice as early as at P21 (<a href="#bib12">Wang et al., 2019</a>). Younger CD4<sup>+</sup> T cells culture were never tested in our system and adjustments may be needed to achieve good culture for younger CD4<sup>+</sup> T cells.</p>
<p>Here, we only tested signature cytokines production and phosphorylated STAT5 in the nuclei by flow cytometry. Other function assays, including cell proliferation and transcription factor activation, can also be performed using this protocol with slight adjustments.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>The purity of naïve CD4<sup>+</sup> T cells after enrichment is low (step 31).</p>
<h3 id="sec6.2">Potential solution</h3>
<p>A potential reason for this problem might be that the number of splenocytes is not accurately determined such that there are more cells than the capacity of the added antibody cocktail and magnetic beads. To improve the efficacy of naïve CD4<sup>+</sup> T cell purification, please ensure that cell counting is accurate. In addition, 10% more of the antibody cocktail and magnetic beads than the recommended amount in this protocol may be used.</p>
<h3 id="sec6.3">Problem 2</h3>
<p>There aren’t many T cell clusters formed in culture and the majority of the cells are dead during the process of T cell differentiation (step 38).</p>
<h3 id="sec6.4">Potential solution</h3>
<p>The potential reason for this problem might be that cell number determination for naïve CD4<sup>+</sup> T cells before seeding is not correct and the cell density is too low. Activated T cells expand rapidly at a high cell density but undergo apoptosis at a low cell density (<a href="#bib8">Ma et al., 2010</a>). A better counting method should be adapted to accomplish an accurate cell density at seeding. The final cell density should be around 1 × 10<sup>6</sup> cells/mL. If the total cell number is too low, a flat 96-well plate can be used instead of the 48-well plate. Another potential reason for this problem might be technical issues with CO<sub>2</sub> incubator. Make sure the temperature and the CO<sub>2</sub> concentration is correct. Unproper storage (e.g., repeating freeze/thaw) can also lead to culture failure.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>The yield and the quality of the extracted RNA are low (step 40a).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>The concentration of the extracted RNA tested by Nanodrop should be over 100 ng/μL. The potential reason for low concentration might be that cultured CD4<sup>+</sup> T cells are not healthy and majority of them are dead or dying. Check the potential solution for problem 2 to make sure the culture is in good condition before RNA isolation. Another potential reason for this problem might be that the final cell number for RNA extraction is too low. In case of a low cell number (such as in 96-well culture condition), use PicoPure RNA isolation kit (KIT0204, Thermo Fisher) to extract RNA. The PicoPure® RNA Isolation Kit is designed to recover high-quality total RNA consistently from fewer than ten cells, even from a single cell. The OD260/OD280 ratio tested in Nanodrop should be around 1.9–2.1. If it is too high, there may be protein contamination. Increasing Buffer RW1 washing may help. If it is too low, there may be RNA degradation. RNase decontamination, for example, using RNase free water and 1.5 mL tubes, is suggested.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>The relative expression level of signature cytokine genes is low (step 41e).</p>
<h3 id="sec6.8">Potential solution</h3>
<p>The potential reason for this problem might be that the purity of naïve CD4<sup>+</sup> T cells is low and the sample is contaminated with already differentiated T cells that may affect <i>in vitro</i> differentiation. Check the potential solution for problem 1 to make sure the purity of naïve CD4<sup>+</sup> T cells is over 95%. Respective antibodies can also be added to the cultures to achieve better differentiation. Another potential reason for this problem might be that the quality of the extracted RNA is poor. Check the potential solution for problem 3 to make sure high quality of the extracted RNA. Unproper storage (e.g., repeating freeze/thaw) can also lead to culture failure.</p>
<h3 id="sec6.9">Problem 5</h3>
<p>There are few cells left after wash and spin before the detection of p-STAT5 by flow cytometry (step 49d).</p>
<h3 id="sec6.10">Potential solution</h3>
<p>The potential reason for this problem might be that too many cells get lost during wash and spin. The solution is to increase the initial cell number for staining and/or increase the spinning speed to ensure that all the cells are pelleted down.</p>
<h3 id="sec6.11">Problem 6</h3>
<p>The signal of phosphorylated STAT5 is too weak (step 49e).</p>
<h3 id="sec6.12">Potential solution</h3>
<p>The potential reason for this problem might be that fixation and cell permeabilization are not ideal. The solution to this problem may be to try different fixation and permeabilization methods. These include the intracellular staining kit with saponin (554722 and 554723, BD Biosciences) and transcription factor staining buffer set (00–5523, Invitrogen). In our hands, fixation with 4% PFA followed by methanol permeabilization generates the best staining result.</p>
<h3 id="sec6.13">Problem 7</h3>
<p>The separation between p-STAT5 positive and negative population is not clear (step 49e).</p>
<h3 id="sec6.14">Potential solution</h3>
<p>The potential reason for this problem might be that dead cells are not excluded. Dead cells have high autofluorescence and will lead to false positivity, which may make the identification of weakly positive samples and rare populations difficult. Adding the Fixable Viability Dye to the staining panel, as we have described, will help reduce the background and make the separation clearer. Another potential reason is the antibody concentration is not optimal. Antibody titration will help to solve this problem.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Aggarwal, S., Ghilardi, N., Xie, M.H., de Sauvage, F.J., and Gurney, A.L. (2003). Interleukin-23 promotes a distinct CD4 T cell activation state characterized by the production of interleukin-17. J. Biol. Chem. <i>278</i>, 1910-1914. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref1">View at publisher</a></p>
<p id="bib2">Cote-Sierra, J., Foucras, G., Guo, L., Chiodetti, L., Young, H.A., Hu-Li, J., Zhu, J., and Paul, W.E. (2004). Interleukin 2 plays a central role in Th2 differentiation. Proc. Natl. Acad. Sci. U S A <i>101</i>, 3880-3885. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref2">View at publisher</a></p>
<p id="bib3">DeGrendele, H.C., Kosfiszer, M., Estess, P., and Siegelman, M.H. (1997). CD44 activation and associated primary adhesion is inducible via T cell receptor stimulation. J. Immunol. <i>159</i>, 2549-2553. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref3">View at publisher</a></p>
<p id="bib4">Goldstein, L.A., Zhou, D.F., Picker, L.J., Minty, C.N., Bargatze, R.F., Ding, J.F., and Butcher, E.C. (1989). A human lymphocyte homing receptor, the hermes antigen, is related to cartilage proteoglycan core and link proteins. Cell <i>56</i>, 1063-1072. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref4">View at publisher</a></p>
<p id="bib5">Killar, L., MacDonald, G., West, J., Woods, A., and Bottomly, K. (1987). Cloned, Ia-restricted T cells that do not produce interleukin 4(IL 4)/B cell stimulatory factor 1(BSF-1) fail to help antigen-specific B cells. J. Immunol. <i>138</i>, 1674-1679. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref5">View at publisher</a></p>
<p id="bib6">Laurence, A., Tato, C.M., Davidson, T.S., Kanno, Y., Chen, Z., Yao, Z., Blank, R.B., Meylan, F., Siegel, R., Hennighausen, L., et al. (2007). Interleukin-2 signaling via STAT5 constrains T helper 17 cell generation. Immunity <i>26</i>, 371-381. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref6">View at publisher</a></p>
<p id="bib7">Liao, W., Lin, J.X., Wang, L., Li, P., and Leonard, W.J. (2011). Modulation of cytokine receptors by IL-2 broadly regulates differentiation into helper T cell lineages. Nat. Immunol. <i>12</i>, 551-559. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref7">View at publisher</a></p>
<p id="bib8">Ma, Q., Wang, Y., Lo, A.S., Gomes, E.M., and Junghans, R.P. (2010). Cell density plays a critical role in ex vivo expansion of T cells for adoptive immunotherapy. J. Biomed. Biotechnol. <i>2010</i>, 386545. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref8">View at publisher</a></p>
<p id="bib9">Mosmann, T.R., Cherwinski, H., Bond, M.W., Giedlin, M.A., and Coffman, R.L. (1986). Two types of murine helper T cell clone. I. Definition according to profiles of lymphokine activities and secreted proteins. J. Immunol. <i>136</i>, 2348-2357. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref9">View at publisher</a></p>
<p id="bib10">Schlegel, P.G. (1997). The role of adhesion and costimulation molecules in graft-versus-host disease. Acta Haematol. <i>97</i>, 105-117. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref10">View at publisher</a></p>
<p id="bib11">Stritesky, G.L., Yeh, N., and Kaplan, M.H. (2008). IL-23 promotes maintenance but not commitment to the Th17 lineage. J. Immunol. <i>181</i>, 5948-5955. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref11">View at publisher</a></p>
<p id="bib12">Wang, W., Cohen, J.A., Wallrapp, A., Trieu, K.G., Barrios, J., Shao, F., Krishnamoorthy, N., Kuchroo, V.K., Jones, M.R., Fine, A., et al. (2019). Age-related dopaminergic innervation augments T helper 2-type allergic inflammation in the postnatal lung. Immunity <i>51</i>, 1102-1118 e1107. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref12">View at publisher</a></p>
<p id="bib13">Zhu, J., Cote-Sierra, J., Guo, L., and Paul, W.E. (2003). Stat5 activation plays a critical role in Th2 differentiation. Immunity <i>19</i>, 739-748. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref13">View at publisher</a></p>
<p id="bib14">Zhu, J., Yamane, H., and Paul, W.E. (2010). Differentiation of effector CD4 T cell populations (∗). Annu. Rev. Immunol. <i>28</i>, 445-489. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00462-7/sref14">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Xingbin Ai (<a href="mailto:xai@partners.org">xai@partners.org</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents</p>
<h4>Data and code availability</h4>
<p>This study did not generate or analyze any datasets.</p>
<h3>Acknowledgments</h3>
<p>We would like to thank the MGH Pathology, Flow, Image and Mass Cytometry Cores for providing training, support, and access to the instruments. Research in the authors’ laboratory is supported by <a href="https://doi.org/10.13039/100000002">NIH</a> grant (1R01HL132991) to X.A.</p>
<h3>Author contributions</h3>
<p>Conceptualization, W.W. and X.A.; methodology, W.W.; investigation, W.W.; writing –original draft, W.W.; writing – review &amp; editing, X.A.; funding acquisition, X.A.; supervision, X.A.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>