<article>
<section>
<h1 id="header">Protocol to identify ligands of odorant receptors using two-electrode voltage clamp combined with the <i>Xenopus</i> oocytes heterologous expression system</h1>
<p><time datetime="2022-03-18">Published: March 18, 2022</time></p>
<p>Song Cao,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">3</a>,<a href="#cor1">*</a></sup> Yang Liu,<sup><a href="#aff1">1</a>,<a href="#fn1">3</a>,<a href="#cor2">**</a></sup> and Guirong Wang<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn2">4</a>,<a href="#cor3">***</a></sup></p>
<p id="aff1"><sup>1</sup>State Key Laboratory for Biology of Plant Diseases and Insect Pests, Institute of Plant Protection, Chinese Academy of Agricultural Sciences, Beijing 100193, China</p>
<p id="aff2"><sup>2</sup>Shenzhen Branch, Guangdong Laboratory for Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen 518120, China</p>
<p id="fn1"><sup>3</sup>Technical contact</p>
<p id="fn2"><sup>4</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:csong_onnz@163.com">csong_onnz@163.com</a></p>
<p id="cor2"><sup>**</sup>Correspondence: <a href="mailto:yangliu@ippcaas.cn">yangliu@ippcaas.cn</a></p>
<p id="cor3"><sup>***</sup>Correspondence: <a href="mailto:wangguirong@caas.cn">wangguirong@caas.cn</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2022.101249">10.1016/j.xpro.2022.101249</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Two-electrode voltage clamp (TEVC) combined with the <i>Xenopus laevis</i> oocytes heterologous expression system is a powerful electrophysiological tool widely used to study the properties of many transmembrane proteins. Here, we describe a protocol using this combined approach to identify the ligands of odorant receptors that form ligand-gated ion channels. We detail the procedures for site-directed mutagenesis, oocyte microinjection, and TEVC recording. This protocol can also be used to identify the key residues and illustrate the structure-function relationships of these proteins.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref2">Cao et al. (2021)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>A simple approach to identify the ligands of odorant receptors in <b><i>Xenopus</i> oocytes</b>
</li>
<li>Detailed procedures to perform TEVC recoding combined with site-directed mutagenesis</li>
<li>Identifying the key amino acid residues for ligand binding of odorant receptors</li>
<li>Applicable for studying other proteins encoding ligand-gated ion channels</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1500-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<h3 id="sec1.1">Primer design</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 20 min</div>
<ol>
<li>Design primers for preparing the mutants of target plasmids. The point mutation replacing the leucine with a valine at 321 position of HvirOR6 is introduced by site-directed mutagenesis using the following primers:
<ol type="a">
<li>Forward: 5′-CAAGTGGCTGTCTC<u>GTA</u>CTTCTCGAATG-3′</li>
<li>Reverse: 5′-CTTGGTGAAACGCGTAATATATGAAC-3′</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The mutation codon is introduced with forward primer and marked with underline. The primers used to mutate target plasmid must be phosphorylated at the 5′ end to eliminate the need for a separate phosphorylation step before direct ligation. One or both primers can be designed with desired mutation(s), and each primer can have more than one mismatch, either separated by correctly matched nucleotides or present in consecutive nucleotides. Generally, the length of the correctly matched sequence in the mutagenic primers should be in average 24–30 nucleotides. The desired mutation should be in the middle of the primer with 10–15 perfectly matched nucleotides on each side. The primers are recommended to be purified with polyacrylamide gel electrophoresis (PAGE).</div>
<h3 id="sec1.2">Reagent preparation</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 h</div>
<ol start="2">
<li>Tetracycline stock solution (50 mg/mL)
<ol type="a">
<li>Dissolve the 0.5 g tetracycline hydrochloride into 10 mL distilled water. Mix thoroughly and filter through 0.22 μm PES syringe filters to sterilize the solution and filter insoluble impurities. Store in 1 mL aliquots at −20°C for 1 year.</li>
</ol>
</li>
<li>Streptomycin stock solution (100 mg/mL)
<ol type="a">
<li>Dissolve the 1 g streptomycin sulfate salt into 10 mL distilled water. Mix thoroughly and filter through 0.22 μm PES syringe filters. Store in 1 mL aliquots at −20°C for 1 year.</li>
</ol>
</li>
<li>Sodium pyruvate stock solution (275 mg/mL)
<ol type="a">
<li>Dissolve 5.5 g sodium pyruvate into 20 mL distilled water. Mix thoroughly and filter through 0.22 μm PES syringe filters. Store in 1 mL aliquots at −20°C for 1 year.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Bacterial and virus strains</td>
</tr>
<tr>
<td>
<i>Trans</i>1-T1 Competent Cells</td>
<td>TransGEN</td>
<td>Cat#CD501-03</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Phenol: Chloroform: Isoamyl Alcohol 25:24:1</td>
<td>Sigma-Aldrich</td>
<td>Cat#P3803</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>TransGEN</td>
<td>Cat#GG101-01</td>
</tr>
<tr>
<td>Tryptone</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#LP0042B</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#LP0021B</td>
</tr>
<tr>
<td>Agar</td>
<td>Coolaber</td>
<td>Cat#CA1331</td>
</tr>
<tr>
<td>Tetracycline hydrochloride</td>
<td>Amresco</td>
<td>Cat#0422</td>
</tr>
<tr>
<td>Streptomycin sulfate</td>
<td>Amresco</td>
<td>Cat#0382</td>
</tr>
<tr>
<td>Sodium pyruvate</td>
<td>Sigma-Aldrich</td>
<td>Cat#P2256</td>
</tr>
<tr>
<td>Horse serum</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#26050-070</td>
</tr>
<tr>
<td>Dimethyl sulfoxide (DMSO)</td>
<td>Sigma-Aldrich</td>
<td>Cat#D4540; CAS: 67-68-5</td>
</tr>
<tr>
<td>(<i>Z</i>)-9-tetradecenal (Z9-14:Ald)</td>
<td>Changzhou Nimrod Inc.</td>
<td>P5126; CAS: 53939-27-8</td>
</tr>
<tr>
<td>(<i>Z</i>)-9-hexadecenal (Z9-16:Ald)</td>
<td>Changzhou Nimrod Inc.</td>
<td>P5111; CAS: 56219-04-6</td>
</tr>
<tr>
<td>Mineral oil</td>
<td>Sigma-Aldrich</td>
<td>Cat#M8410</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>PhusionTM Site-Directed Mutagenesis Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#F541</td>
</tr>
<tr>
<td>TIANprep Mini Plasmid Kit</td>
<td>TIANGEN</td>
<td>Cat#DP103-03</td>
</tr>
<tr>
<td>mMESSAGE mMACHINE SP6 Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#AM1340</td>
</tr>
<tr>
<td colspan="3">Experimental models: Organisms/strains</td>
</tr>
<tr>
<td>
<i>Xenopus laevis</i> oocytes</td>
<td>Xenopus Resource Center (Hangzhou, China)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>pSP64DV</td>
<td><a href="#bib5">Lu et al. (2007)</a></td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Oligonucleotides</td>
</tr>
<tr>
<td>Forward: 5′-CAAGTGGCTGTCTCGTACTTCTCGAATG-3′</td>
<td>Sangon Biotech</td>
<td>N/A</td>
</tr>
<tr>
<td>Reverse: 5′-CTTGGTGAAACGCGTAATATATGAAC-3′</td>
<td>Sangon Biotech</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Software and algorithms</td>
</tr>
<tr>
<td>pCLAMP 10.2</td>
<td>Molecular Device</td>
<td>N/A</td>
</tr>
<tr>
<td>GraphPad Prism 8.0</td>
<td>GraphPad Software</td>
<td><a href="https://www.graphpad.com">https://www.graphpad.com</a></td>
</tr>
<tr>
<td>SAS v8</td>
<td>SAS</td>
<td><a href="https://www.sas.com/">https://www.sas.com/</a></td>
</tr>
<tr>
<td colspan="3">Other</td>
</tr>
<tr>
<td>0.22 μm PES Syringe Filters</td>
<td>Any supplier</td>
<td>N/A</td>
</tr>
<tr>
<td>50 mL centrifuge tubes</td>
<td>Sangon Biotech</td>
<td>N/A</td>
</tr>
<tr>
<td>Plastic Pasteur pipette</td>
<td>Any supplier</td>
<td>N/A</td>
</tr>
<tr>
<td>90-mm Petri dish</td>
<td>Any supplier</td>
<td>N/A</td>
</tr>
<tr>
<td>60-mm Petri dish</td>
<td>Corning Incorporated</td>
<td>Cat#430166</td>
</tr>
<tr>
<td>24-well plates</td>
<td>Corning Incorporated</td>
<td>Cat#3524</td>
</tr>
<tr>
<td>Incubator</td>
<td>SANYO</td>
<td>MIR-154</td>
</tr>
<tr>
<td>Stereo Microscopes</td>
<td>Olympus</td>
<td>SZX7</td>
</tr>
<tr>
<td>Micropipette puller</td>
<td>Sutter Instrument</td>
<td>P-1000</td>
</tr>
<tr>
<td>Glass capillaries</td>
<td>WPI</td>
<td>Cat#1B120F-3</td>
</tr>
<tr>
<td>Microinjector</td>
<td>WPI</td>
<td>Nanoliter injector 2010</td>
</tr>
<tr>
<td>Silver wires</td>
<td>Warner Instruments</td>
<td>Cat#AG25-10</td>
</tr>
<tr>
<td>OC-725C Oocyte Clamp Amplifier</td>
<td>Warner Instruments</td>
<td>N/A</td>
</tr>
<tr>
<td>Digidata 1440A Data Acquisition System</td>
<td>Molecular Devices</td>
<td>N/A</td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">Preparation of LB media for bacterial culture</h3>
<ul>
<li>LB sterile liquid medium</li>
</ul>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>10 g/L</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>0.5 g/L</td>
<td>5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>10 g/L</td>
<td>10 g</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>To a final volume of 1,000 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix thoroughly with a magnetic stirrer and sterilize the medium in autoclave sterilizer appliance that generate temperatures around 130°C and store at 25°C for one month.</p>
<ul>
<li>LB agar medium</li>
</ul>
<table id="undtbl2">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tryptone</td>
<td>10 g/L</td>
<td>10 g</td>
</tr>
<tr>
<td>Yeast Extract</td>
<td>0.5 g/L</td>
<td>5 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>10 g/L</td>
<td>10 g</td>
</tr>
<tr>
<td>Agar</td>
<td>15 g/L</td>
<td>15 g</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>To a final volume of 1,000 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix thoroughly with a magnetic stirrer and sterilize the medium in autoclave sterilizer appliance that generate temperatures around 130°C. After cooling around 40°C, add 500 μL Ampicillin, mix thoroughly and aliquot 15 mL into each 90-mm Petri dishes and store at 4°C for one month.</p>
<h3 id="sec2.2">Preparation of materials/equipment for oocytes microinjection and culture</h3>
<p>For preparing oocytes collection you will need:</p>
<p>90-mm Petri dish</p>
<p>60-mm Petri dish</p>
<p>Plastic Pasteur pipette to sort and transfer oocytes</p>
<p>24-well plates</p>
<p>Incubator to keep oocytes at 18°C</p>
<h3 id="sec2.3">Solutions for oocytes microinjection and culture</h3>
<ul>
<li>10× Ringer’s buffer</li>
</ul>
<table id="undtbl3">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>NaCl</td>
<td>960 mM</td>
<td>56.1 g</td>
</tr>
<tr>
<td>KCl</td>
<td>20 mM</td>
<td>1.5 g</td>
</tr>
<tr>
<td>MgCl<sub>2</sub>⋅6H<sub>2</sub>O</td>
<td>50 mM</td>
<td>10.2 g</td>
</tr>
<tr>
<td>HEPES</td>
<td>50 mM</td>
<td>11.9 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td>To a final volume of 1,000 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Adjust pH to 7.6 with NaOH. Filter-sterilize the solution and store at 4°C for several months.</p>
<div class="alternatives">
<span class="alternatives-title">Alternatives:</span> 1× Ringer’s buffer can also be prepared by adding 10× stock solutions of each ingredient. Before that, prepare 10× stock solutions of each component by dissolving each chemical compound into 1,000 mL distilled water, adjust pH of HEPES stock, filter sterilize and keep at 4°C. Prepare 1× Ringer's buffer fresh prior to use.</div>
<ul>
<li>100× CaCl<sub>2</sub> solution</li>
</ul>
<table id="undtbl4">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>CaCl<sub>2</sub>⋅2H<sub>2</sub>O</td>
<td>60 mM</td>
<td>4.41 g</td>
</tr>
<tr>
<td>Distilled water</td>
<td>To a final volume of 500 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>60 mM</td>
<td>500 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Filter-sterilize the solution and store at 4°C for several months.</p>
<ul>
<li>Incubation medium</li>
</ul>
<table id="undtbl5">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× Ringer’s buffer</td>
<td>n/a</td>
<td>100 mL</td>
</tr>
<tr>
<td>Sodium pyruvate stock solution</td>
<td>550 μg/mL</td>
<td>2 mL</td>
</tr>
<tr>
<td>Tetracycline stock solution</td>
<td>50 μg/mL</td>
<td>1 mL</td>
</tr>
<tr>
<td>Streptomycin stock solution</td>
<td>100 μg/mL</td>
<td>1 mL</td>
</tr>
<tr>
<td>Dialyzed horse serum</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
<tr>
<td>Distilled water</td>
<td>To a final volume of 1,000 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>1,000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Filter-sterilize the solution. Prepare on the day of frog dissection.</p>
<h3 id="sec2.4">Reagents and solutions for electrophysiological recordings</h3>
<ul>
<li>KCl solution</li>
</ul>
<table id="undtbl6">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>KCl</td>
<td>3 M</td>
<td>11.175 g</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>To a final volume of 1,000 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>50 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Filter-sterilize the solution and store at room temperature (15°C–25°C) for several months.</p>
<ul>
<li>Working solution</li>
</ul>
<table id="undtbl7">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>10× Ringer’s buffer</td>
<td>n/a</td>
<td>200 mL</td>
</tr>
<tr>
<td>100× CaCl<sub>2</sub> stock solution</td>
<td>600 μM</td>
<td>20 mL</td>
</tr>
<tr>
<td>Ultra-pure water</td>
<td>To a final volume of 2,000 mL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>n/a</td>
<td>2,000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix thoroughly and store at room temperature for several months.</p>
<ul>
<li>1 M candidate ligand Z9-14:Ald stock solution</li>
</ul>
<table id="undtbl8">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Z9-14:Ald</td>
<td>1 M</td>
<td>125.36 μL</td>
</tr>
<tr>
<td>DMSO</td>
<td>To a final volume of 500 μL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>1 M</td>
<td>500 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix thoroughly and store at −20°C for several months.</p>
<ul>
<li>1 M candidate ligand Z9-16:Ald stock solution</li>
</ul>
<table id="undtbl9">
<thead>
<tr>
<th>Reagent</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>Z9-16:Ald</td>
<td>1 M</td>
<td>141.91 μL</td>
</tr>
<tr>
<td>DMSO</td>
<td>To a final volume of 500 μL</td>
<td>n/a</td>
</tr>
<tr>
<td>Total</td>
<td>1 M</td>
<td>500 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Mix thoroughly and store at −20°C for several months.</p>
<h3 id="sec2.5">Experimental model and subject details</h3>
<p>The female sexually mature <i>Xenopus</i> frogs used in this study are purchased from the Xenopus Resource Center (Hangzhou, China). Frogs are maintained in a recirculating tank system in a room at a temperature of 18°C–20°C. All experiments involving frogs are approved by the Institutional Animal Care and Use Committee of Chinese Academy of Agricultural Sciences, Beijing, China.</p>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Site-directed mutagenesis of target plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day for plasmid mutation and 20 min for sequence analysis</div>
<p>The protocol is performed using a Phusion<sup>TM</sup> Site-Directed Mutagenesis Kit (<a href="https://www.thermofisher.cn/order/catalog/product/F541">https://www.thermofisher.cn/order/catalog/product/F541</a>) based on the manufacturer’s instructions. The protocol can also be used to introduce insertions and deletions in any type of plasmid DNA (<a href="#bib9">Römer et al., 2009</a>; <a href="#bib1">Briant et al., 2017</a>). The main procedures are shown in <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1500-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Schematic workflow of site-directed mutagenesis of target plasmid</div>
</figcaption>
</figure>
<ol>
<li>PCR to amplify the target plasmid using the above-mentioned primers (Primer design).
<ol type="a">
<li>Prepare the mix below:
<table id="undtbl10">
<thead>
<tr>
<th>Reagent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>5× Phusion HF Buffer</td>
<td>10 μL</td>
</tr>
<tr>
<td>10 mM dNTPs</td>
<td>1 μL</td>
</tr>
<tr>
<td>Forward primer (10 μM)</td>
<td>2 μL</td>
</tr>
<tr>
<td>Reverse primer (10 μM)</td>
<td>2 μL</td>
</tr>
<tr>
<td>HvirOR6-pSP64DV plasmid</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>Phusion Hot Start DNA Polymerase (2 U/μL)</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>
<i>dd</i> H<sub>2</sub>O</td>
<td>Fill to 50 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Use the following thermocycler parameters for the PCR.
<table id="undtbl11">
<thead>
<tr>
<th colspan="4">PCR cycling conditions</th>
</tr>
<tr>
<th>Steps</th>
<th>Temperature</th>
<th>Time</th>
<th>Cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98°C</td>
<td>1 min</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98°C</td>
<td>10 s</td>
<td rowspan="3">25 cycles</td>
</tr>
<tr>
<td>Annealing</td>
<td>67°C</td>
<td>30 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72°C</td>
<td>2 min</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72°C</td>
<td>5 min</td>
<td>1</td>
</tr>
<tr>
<td>Hold</td>
<td>4°C</td>
<td>Forever</td>
<td> </td>
</tr>
</tbody>
</table>
</li>
</ol>
</li>
</ol>
<ol start="2">
<li>Digest the PCR product with the enzyme DpnI to digest the Dam-methylated plasmid DNA template.
<ol type="a">
<li>Add 1 μL of FastDigest DpnI enzyme (included in kit) to the PCR product. Mix thoroughly by pipetting.</li>
<li>Incubate at 37°C for 30 min.</li>
</ol>
</li>
<li>Prepare 10 μL ligation reaction mix.
<ol type="a">
<li>Prepare the mix below:
<table id="undtbl12">
<thead>
<tr>
<th>Regent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>PCR product</td>
<td>4 μL</td>
</tr>
<tr>
<td>5× Rapid Ligation Buffer</td>
<td>2 μL</td>
</tr>
<tr>
<td>T4 DNA Ligase</td>
<td>0.5 μL</td>
</tr>
<tr>
<td>
<i>dd</i> H<sub>2</sub>O</td>
<td>Fill to 10 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Centrifuge briefly and incubate at 25°C for 30 min, then chill on ice.</li>
</ol>
</li>
<li>Perform transformation reaction.
<ol type="a">
<li>5 μL of the product from the ligation reaction is used to transform <i>Trans</i>1-T1 <i>Escherichia coli</i> competent cells.</li>
<li>Cultivate the transformants on Ampicillin (working concentration 50 μg/mL) LB agar plates at 37°C for 12–14 h.</li>
</ol>
</li>
<li>In the next day, 3–5 individual colonies are cultured in 500 μL of liquid Ampicillin (working concentration 50 μg/mL) LB medium at 37°C for 4–6 h with shaking at 220 rpm. The cultured medium containing each colony is sent to sequence to check for the right mutations.</li>
</ol>
<h3 id="sec3.2">Preparation of cRNA</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 1 day</div>
<p>The protocol is performed using a mMESSAGE mMACHINE SP6 Kit according to the manufacturer’s instructions (<a href="https://www.thermofisher.cn/order/catalog/product/AM1340">https://www.thermofisher.cn/order/catalog/product/AM1340</a>).</p>
<ol start="6">
<li>Culture the colonies of wild type HvirOR6 and its mutant, HvirOR6-L321V, in liquid Ampicillin LB at 37°C for 14 h in an incubator with shaking at 220 rpm.</li>
<li>Extract the plasmid DNA using a Plasmid DNA extraction kit (TIANprep Mini Plasmid Kit).</li>
<li>Plasmid DNA is linearized with restriction enzyme SmaI and then purified with Phenol: Chloroform: Isoamyl alcohol (25:24:1), and resuspended in 7 μL nuclease-free water.</li>
<li>The quantity and integrity of linearized plasmid DNA are determined using a Nanodrop ND-1000 spectrophotometer (Nano-Drop products, Wilmington, USA) and gel electrophoresis analysis.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The plasmid DNA must be completely digested before the transcription reaction, because even a small amount of circular template can adversely affect the transcription output.</div>
<ol start="10">
<li>Prepare 20 μL <i>in vitro</i> transcription reaction mix.
<ol type="a">
<li>Prepare the mix below:
<table id="undtbl13">
<thead>
<tr>
<th>Regent</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>2× CAP/NTP</td>
<td>10 μL</td>
</tr>
<tr>
<td>10× Reaction Buffer</td>
<td>2 μL</td>
</tr>
<tr>
<td>Linearized plasmid template</td>
<td>6 μL</td>
</tr>
<tr>
<td>Enzyme Mix</td>
<td>2 μL</td>
</tr>
</tbody>
</table>
</li>
<li>Pipette the mixture gently, and briefly centrifuge. Incubate at 37°C for 2 h.</li>
</ol>
</li>
<li>Stop the reaction and precipitate the RNA by adding 30 μL nuclease-free water and 30 μL LiCl Precipitation Solution (included in kit). Chill for 1 h at −20°C.</li>
<li>Centrifuge at 13,523 g at 4°C for 30 min to pellet the RNA.</li>
<li>Wash the RNA twice with 1 mL 70% ethanol. The synthesized cRNA is diluted into nuclease-free water to final concentration of 2 μg/μL and stored in 1 μL aliquots at −70°C until use.</li>
</ol>
<h3 id="sec3.3">Oocytes microinjection</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol start="14">
<li>Choose mature and healthy oocytes (stage V-VII) under a stereomicroscope and transfer them into a new 60 mm Petri dishes (<a href="#fig2">Figure 2</a>A).
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1500-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. The main steps of oocytes microinjection and TEVC recording</div>
<p>(A) Select mature and healthy oocytes.</p>
<p>(B) Microinject oocytes under a stereomicroscope.</p>
<p>(C) Culture injected oocytes into 24-well plates in an incubator at 18°C.</p>
<p>(D) Perform TEVC recording experiment.</p>
</figcaption>
</figure>
</li>
<li>Mix 1 μL cRNA of <i>HvirOR6</i> or 1 μL cRNA of its mutant, <i>HvirOR6-L321V</i> with 1 μL cRNA of Orco (<i>HvirOrco</i>), respectively.</li>
<li>Prepare injection needles by pulling glass capillaries with a micropipette puller. The tip of a needle is cut with a blade to a 20 μm opening in diameter.</li>
<li>Fill the glass needle with mineral oil and install it onto the dispenser of a microinjector.</li>
<li>Transfer the 2 μL cRNA of each mixture onto a piece of parafilm and draw it into the glass needle under the control of the microinjector manipulator.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Make sure no air has been drawn into the glass needle, which will greatly affect the injection of oocytes.</div>
<ol start="19">
<li>Inject 27.6 nL cRNA mixture into each oocyte under a stereomicroscope (<a href="#fig2">Figure 2</a>B). Oocytes microinjected with 27.6 nL RNase-free water are used as the negative control. After injection, transfer the injected oocytes into 24-well plates (2–3 oocytes in each well) filled with incubation medium. Keep the oocytes in the incubator for 3–5 days for heterologous expression at 18°C (<a href="#fig2">Figure 2</a>C).</li>
</ol>
<h3 id="sec3.4">Electrophysiological recording with TEVC</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<p>The TEVC experiment is performed with the equipment shown in <a href="#fig2">Figure 2</a>D. The main operation steps are as follows:</p>
<ol start="20">
<li>Prepare working solutions of Z9-14:Ald and Z9-16:Ald.
<ol type="a">
<li>Dilute 5 μL of 1 M Z9-14:Ald or Z9-16:Ald stock solution into 50 mL working solution in a 50 mL tube to prepare working solutions of 10<sup>-4</sup> M for each chemical. And 3 × 10<sup>-4</sup> M working solution of each chemical can be prepared by adding 15 μL of 1 M chemical working solution into 50 mL working solution.</li>
<li>10<sup>-5</sup> M and 3 × 10<sup>-5</sup> M working solutions of each chemical are prepared by adding 5 mL of 10<sup>-4</sup> M and 3 × 10<sup>-4</sup> M chemical working solution into 45 mL working solution, respectively. Similarly, chemical solutions of other concentrations including 10<sup>-6</sup> M, 10<sup>-7</sup> M and 3 × 10<sup>-6</sup> M are prepared.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The chemical working solutions should be freshly prepared.</div>
<ol start="21">
<li>Prepare silver wires for recording. The silver wires used in voltage electrode, current electrode and bath probe electrodes must be chlorinated prior to assembly and use. Briefly polish silver wires with sandpaper, and then immerse them in NaClO solution to half length, until the immersed part becomes uniformly black.</li>
<li>Load the working solution and chemical working solutions in each perfusion tubes, and adjust the height of tubes to keep the flow rate at about 5 mL/min.</li>
<li>Prepare glass electrodes. Pull glass capillaries to a fine taper using a micropipette puller with proper parameters.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> The parameters are set according to the RAMP value for each batch of glass capillaries. Test the RAMP value of one glass capillary each time a new batch of glass capillaries is to be used.</div>
<ol start="24">
<li>Fill 3 M KCl into two glass electrodes (about 2/3 volume of the glass electrode). Insert an oxidized silver wire and a glass electrode into each holder, and then install the holder on micromanipulators.</li>
<li>Measure the resistance of the electrodes.
<ol type="a">
<li>Turn on OC-725C Oocyte Clamp Amplifier and Digidata 1440A Data Acquisition System, and open the Clampex 10.2 software.</li>
<li>Move the electrodes into bath solution with the micromanipulators and turn “Vm OFFSET” and “Ve OFFSET” to zero.</li>
<li>Press buttons “Vm Electrode Test” and “Ve Electrode Test” respectively to display the resistance of each electrode. The electrode resistance should be in the range of 0.1–1 MΩ.</li>
</ol>
</li>
<li>Transfer an oocyte expressing the target gene into the hole at the middle of bath chamber with a pipette.</li>
<li>Move the voltage electrode to gently impale the oocyte by operating the micromanipulator, and then the current electrode. Generally, the resting membrane potential of the oocyte is displayed in the range of −10 to −50 mV.</li>
<li>Turn the buttons “CLAMP” to “SLOW” and “GAIN” to “MAX” on the Digidata 1440A Data Acquisition System, and click the button “Record” on the interface of Clampex 10.2 software to start recording.</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Chemical-induced currents are recorded with the TEVC at holding potential of -80 mV. Signals are amplified using an OC-725C amplifier with lowpass filtered at 50 Hz and digitized at 1 kHz. Data acquisition is carried out with Digidata 1440A Data Acquisition System and pCLAMP 10.2 software.</div>
<ol start="29">
<li>Stimulate oocytes with different concentrations of chemical solutions for desired purposes.
<ol type="a">
<li>For the purpose to identify ligands of HvirOR6 and its mutant. In this experiment, the oocytes expressing HvirOR6 or its mutant are challenged with 10<sup>-4</sup> M of Z9-14:Ald and Z9-16:Ald.</li>
<li>For the purpose to study the response sensitivity of HvirOR6 or its mutant to chemicals. In this stage, the oocytes expressing HvirOR6 or its mutant are treated with increasing concentrations (3 × 10<sup>-4</sup> M to 10<sup>-7</sup> M) of chemical solutions.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> During the recording, chemical solutions are supplied for about 15 s at a flow rate of 5 mL/min. After each stimulus, the oocytes are washed with working solution until the current returns to the baseline.</div>
<ol start="30">
<li>Stop the recording. After challenged with all stimuli, click the button “Stop” to end the recording, and then turn the “GAIN” to “0”, and “CLAMP” to “OFF” in turn.</li>
<li>Withdraw the two electrodes carefully and remove the tested oocyte. Put another oocyte in the chamber and start a new recording as described above.</li>
<li>Finish the recording. Close the Clampex 10.2 software, and turn off the machines. Empty the remaining salt solutions in the perfusion system by washing with clean water.</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>Examples of the expected outcomes in this protocol are given in <a href="#fig3">Figures 3</a>, <a href="#fig4">4</a>, and <a href="#fig5">5</a>. By using the site-directed mutagenesis protocol, we can easily obtain a mutated plasmid with desired mutation site(s). <a href="#fig3">Figure 3</a> shows an example of a mutation at 321 amino acid site of HvirOR6 in vector pSP64DV, where the nucleotide thymine mutated into guanine and resulting in a mutation of the amino acid leucine (L) into valine (V).</p>
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1500-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. The sequencing chromatograms of HvirOR6 and its mutant, HvirOR6-L321V</div>
</figcaption>
</figure>
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1500-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. The electrophysiological responses of HvirOR6 and HvirOR6-L321V to 10<sup>-4</sup> M chemical solutions</div>
<p>(A) Inward current responses of oocytes expressing HvirOR6 to Z9-14:Ald and Z9-16:Ald.</p>
<p>(B) Inward current responses of oocytes expressing HvirOR6-L321V to Z9-14:Ald and Z9-16:Ald.</p>
<p>(C) Response profiles of oocytes expressing HvirOR6 and its mutant to Z9-14:Ald and Z9-16:Ald. Error bars indicate SEM (n = 16–22).</p>
<p>Figure adapted from <a href="#bib2">Cao et al. (2021)</a>.</p>
</figcaption>
</figure>
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1500-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. The response sensitivity of HvirOR6 and HvirOR6-L321V to Z9-14:Ald and Z9-16:Ald</div>
<p>(A) HvirOR6 expressing oocytes stimulated with a range of Z9-14:Ald concentrations.</p>
<p>(B) Dose-response curve of HvirOR6 expressing oocytes treated with Z9-14:Ald. EC50 = 6.196 × 10<sup>-6</sup> M (n = 10).</p>
<p>(C) HvirOR6 and HvirOR6-L321V expressing oocytes stimulated with a range of Z9-16:Ald concentrations.</p>
<p>(D) Dose-response curve of HvirOR6 and HvirOR6-L321V expressing oocytes treated with Z9-16:Ald. EC50 = 2.476 × 10<sup>-5</sup> M and 4.083 × 10<sup>-5</sup> M, respectively (n = 7).</p>
<p>Figure adapted from <a href="#bib2">Cao et al. (2021)</a>.</p>
</figcaption>
</figure>
<p>The example shown in <a href="#fig4">Figure 4</a> reveals the binding profiles of HvirOR6 and its mutant to candidate ligands, Z9-14:Ald and Z9-16:Ald. <a href="#fig4">Figures 4</a>A and 4B show the electrophysiological responses of HvirOR6 and HvirOR6-L321V to 10<sup>-4</sup> M tested chemical solutions, which indicates that HvirOR6 and its mutant both can be activated by Z9-14:Ald and Z9-16:Ald but with different binding affinities. <a href="#fig4">Figure 4</a>C shows that when mutating the amino acid L at 321 position of HvirOR6 to V, the mutant almost loses its response to Z9-14:Ald (t = 13.3, P &lt; 0.001), but the response to Z9-16:Ald does not change significantly (t = -0.06, P = 0.9532) at 10<sup>-4</sup> M concentration. The results of <a href="#fig4">Figure 4</a> imply that L321 is a critical amino acid for the ligands binding of HvirOR6.</p>
<p>To further clarify the effect of L321 to the function of HvirOR6, we perform dose-response experiments to compare the binding ability of HvirOR6 and its mutant to their ligands. <a href="#fig5">Figures 5</a>A and 5B show the dose-response curve of HvirOR6 to Z9-14:Ald and the EC50 value is 6.196 × 10<sup>-6</sup> M. <a href="#fig5">Figures 5</a>C and 5D show the dose-response curves of HvirOR6 and its mutant, HvirOR6-L321V, to Z9-16:Ald, and their EC50 values are 2.476 × 10<sup>-5</sup> M and 4.629 × 10<sup>-5</sup> M, respectively. These results indicate that the mutation of one site, 321, causes a functional impact of HvirOR6, demonstrating the critical and essential role of this amino acid site.</p>
<div class="critical">
<span class="critical-title">Critical:</span> To fit data to produce a dose-response curve, we can perform the following steps: a. Create an XY table, and enter the data. Fill the concentrations into “X” column, ligand names into “Group A, B, C” et al. and replicate values into corresponding tables. b. Analyze the data. Click “Analyze” to transform the concentrations using function X = Log (X). c. Fit a dose-response curve. Click “Fit a curve with nonlinear regression” in “Analyze”, choose “Nonlinear regression (curve fit)” in “XY analyses”, then click “log(agonist) vs. response – Variable slope (four parameters)” in “Dose-response-Stimulation”. d. Beautify and export the figure. To fit a dose-response curve and calculate ligand's EC50 values, usually 5 or more groups of data are needed.</div>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>For statistical analysis of the response intensities of HvirOR6 and its mutant to ligands, the mean response value of HvirOR6 to Z9-14:Ald was normalized as 1, and all response values were normalized by dividing the mean response value. The normalized response values were used to compare with the response value of HvirOR6 to the two chemical compounds individually using Student’s <i>t-test</i> with SAS v8 for Windows. Dose-response curves and EC50 value were fitted and calculated with GraphPad Prism 8.0 software.</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The quality of oocytes can affect the electrophysiological responses of oocytes expressing target genes to ligands, both on the response intensities and sensitivities. Thus, only mature and healthy oocytes should be selected and used in the experiment. Besides, it is hard to perform the dose-response experiment for a target protein with small response to ligand. And the EC50 value is not accurate, either. Because during recording, the baseline may slightly fluctuate due to some background noise or/and changes in liquid height. The response values are obtained by calculating the difference value between the baseline and the trough, but their locations are artificially determined. If the response of a receptor to a ligand is very small, the response value is easily influenced by multiple factors, and it is easier to cause some errors.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The quality of oocytes affects the results of TEVC experiments (steps 24 and 29).</p>
<h3 id="sec7.2">Potential solution</h3>
<p>Considering the possible impacts caused by the state of oocytes on identifying ligands and response sensitivities of target proteins, we recommend using good quality oocytes at least from 3 different frogs in the experiments. In addition, at least 3 oocytes from each frog should be used to verify the results. For the purpose of identifying key amino acid residues, the response intensities and sensitivities of target proteins and their mutants to candidate ligands should be compared with oocytes from a same frog. Furthermore, it is better to perform the electrophysiological recordings at around the same time after RNA injection to eliminate possible differences on the expression time or level of proteins in different times.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Hard to find out key amino acid residues involved in ligand binding (steps 1–5).</p>
<h3 id="sec7.4">Potential solution</h3>
<p>First, find out if there are any orthologous genes that have different response profiles or show divergent sensitivity to a same ligand. After sequences alignment, their diverse amino acid residues are potential candidates (<a href="#bib7">Pellegrino et al., 2011</a>; <a href="#bib4">Leary et al., 2012</a>; <a href="#bib2">Cao et al., 2021</a>). Second, divide the amino acid sequence into several regions and construct chimeric genes. After functional comparison of original sequence and chimeric genes, find out the key regions (<a href="#bib10">Yang et al., 2017</a>; <a href="#bib2">Cao et al., 2021</a>). Third, previous studies imply that the amino acids at the transmembrane domains and extracellular loops (ECL) affect the function of proteins (<a href="#bib6">Nichols and Luetje, 2010</a>; <a href="#bib7">Pellegrino et al., 2011</a>; <a href="#bib4">Leary et al., 2012</a>; <a href="#bib3">Hughes et al., 2014</a>; <a href="#bib8">Rahman and Luetje, 2017</a>; <a href="#bib10">Yang et al., 2017</a>; <a href="#bib2">Cao et al., 2021</a>).</p>
<h3 id="sec7.5">Problem 3</h3>
<p>How to identify and select mature and healthy oocytes (step 14).</p>
<h3 id="sec7.6">Potential solution</h3>
<p>Stage V-VII oocytes are selected and used in the next microinjection and electrophysiological recording experiments. These oocytes are characterized by their size (1.0–1.3 mm in diameter) and appearance (good contrast between the dark pigmented animal hemisphere and the yellowish vegetal hemisphere). The healthy oocytes are usually round with uniform color in each side.</p>
<h3 id="sec7.7">Problem 4</h3>
<p>Hard to cut pipet tips to a suitable size for oocyte microinjection (step 16).</p>
<h3 id="sec7.8">Potential solution</h3>
<p>The appropriate size of pipets is important for oocyte microinjection, because a pipet with smaller size will be easily blocked and is hard for injection, while a pipet with bigger size will cause any damage to oocytes. Thus, it is necessary to produce some pipets with suitable size. We usually cut the pipet tips under a stereomicroscope with the maximum power, and then measure the opening diameter using a stereomicroscope with a graduated scale. We can adjust the cutting site according to the opening size. When the size is suitable for microinjection, other pipets can also be cut under a stereomicroscope with the maximum power by comparing with the well prepared one. It is easy to cut pipet tips with good opening size for someone experienced.</p>
<h3 id="sec7.9">Problem 5</h3>
<p>Hard to regulate the external interference to electrophysiological signal during recording (steps 21–29).</p>
<h3 id="sec7.10">Potential solution</h3>
<p>First, maintain the machine well grounded. Second, chlorinate the silver wires uniformly used in electrodes. Third, keep a steady flow rate of chemical solutions and working solution. Fourth, prepare a pair of glass electrodes in good shapes. Adjust the parameters to produce electrodes with short but sharp tips, and test the resistances in a proper range (step 25). Test the RAMP value of one glass capillary each time a new batch of glass capillaries is to be used. Besides, it is recommended to purchase and use capillaries of a same brand.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Briant, K., Johnson, N., and Swanton, E. (2017). Transmembrane domain quality control systems operate at the endoplasmic reticulum and Golgi apparatus. PLoS One <i>12</i>, e0173924. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref1">View at publisher</a></p>
<p id="bib2">Cao, S., Liu, Y., Wang, B., and Wang, G. (2021). A single point mutation causes one-way alteration of pheromone receptor function in two <i>Heliothis</i> species. iScience <i>24</i>, 102981. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref2">View at publisher</a></p>
<p id="bib3">Hughes, D.T., Wang, G., Zwiebel, L.J., and Luetje, C.W. (2014). A determinant of odorant specificity is located at the extracellular loop 2-transmembrane domain 4 interface of an <i>Anopheles gambiae</i> odorant receptor subunit. Chem. Senses <i>39</i>, 761-769. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref3">View at publisher</a></p>
<p id="bib4">Leary, G.P., Allen, J.E., Bunger, P.L., Luginbill, J.B., Linn, C.E., Macallister, I.E., Kavanaugh, M.P., and Wanner, K.W. (2012). Single mutation to a sex pheromone receptor provides adaptive specificity between closely related moth species. Proc. Natl. Acad. Sci. U.S.A. <i>109</i>, 14081-14086. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref4">View at publisher</a></p>
<p id="bib5">Lu, T., Qiu, Y., Wang, G., Kwon, J.Y., Rutzler, M., Kwon, H., Pitts, R.J., van Loon, J.J.A., Takken, W., Carlson, J.R., and Zwiebel, L.J. (2007). Odor coding in the maxillary palp of the malaria vector mosquito <i>Anopheles gambiae</i>. Curr. Biol. <i>17</i>, 1533-1544. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref5">View at publisher</a></p>
<p id="bib6">Nichols, A.S. and Luetje, C.W. (2010). Transmembrane segment 3 of <i>Drosophila melanogaster</i> odorant receptor subunit 85b contributes to ligand-receptor interactions. J. Biol. Chem. <i>285</i>, 11854-11862. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref6">View at publisher</a></p>
<p id="bib7">Pellegrino, M., Steinbach, N., Stensmyr, M.C., Hansson, B.S., and Vosshall, L.B. (2011). A natural polymorphism alters odour and DEET sensitivity in an insect odorant receptor. Nature <i>478</i>, 511-514. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref7">View at publisher</a></p>
<p id="bib8">Rahman, S. and Luetje, C.W. (2017). Mutant cycle analysis identifies a ligand interaction site in an odorant receptor of the malaria vector <i>Anopheles gambiae</i>. J. Biol. Chem. <i>292</i>, 18916-18923. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref8">View at publisher</a></p>
<p id="bib9">Römer, P., Recht, S., and Lahaye, T. (2009). A single plant resistance gene promoter engineered to recognize multiple TAL effectors from disparate pathogens. Proc. Natl. Acad. Sci. U.S.A. <i>106</i>, 20526-20531. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref9">View at publisher</a></p>
<p id="bib10">Yang, K., Huang, L., Ning, C., and Wang, C. (2017). Two single-point mutations shift the ligand selectivity of a pheromone receptor between two closely related moth species. eLife <i>6</i>, e29100. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(22)00129-0/sref10">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Guirong Wang (<a href="mailto:wangguirong@caas.cn">wangguirong@caas.cn</a>).</p>
<h4>Materials availability</h4>
<p>No materials were generated in this study.</p>
<h4>Data and code availability</h4>
<p>No data or code was generated in this study.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by <a href="https://doi.org/10.13039/501100001809">National Natural Science Foundation of China</a> (31725023 to G.W. and 32072509 to Y.L.) and Shenzhen Science and Technology Program (KQTD20180411143628272 to G.W.).</p>
<h3>Author contributions</h3>
<p>Conceptualization, S.C., Y.L., and G.W.; methodology, S.C. and Y.L.; investigation, S.C. and Y.L.; writing – original draft, S.C. and Y.L.; writing – review &amp; editing, S.C., Y.L., and G.W.; funding acquisition, Y.L. and G.R.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>