<article>
<section>
<h1 id="header">Isolation and immunofluorescence staining of <i>Aspergillus fumigatus</i> conidia-containing phagolysosomes</h1>
<p><time datetime="2021-02-05">Published: February 5, 2021</time></p>
<p>Marie Goldmann,<sup><a href="#aff1">1</a>,<a href="#fn1">4</a></sup> Franziska Schmidt,<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a></sup> Irene Kyrmizi,<sup><a href="#aff3">3</a>,<a href="#fn1">4</a></sup> Georgios Chamilos,<sup><a href="#aff3">3</a>,<a href="#fn1">4</a></sup> and Axel A. Brakhage<sup><a href="#aff1">1</a>,<a href="#aff2">2</a>,<a href="#fn1">4</a>,<a href="#fn2">5</a>,<a href="#fn3">6</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Molecular and Applied Microbiology, Leibniz Institute for Natural Product Research and Infection Biology – Hans Knöll Institute (HKI), 07745 Jena, Germany</p>
<p id="aff2"><sup>2</sup>Department of Microbiology and Molecular Biology, Institute of Microbiology, Friedrich Schiller University Jena, 07745 Jena, Germany</p>
<p id="aff3"><sup>3</sup>Department of Medicine, University of Crete, and Institute of Molecular Biology and Biotechnology, Foundation for Research and Technology, Heraklion, Crete 71300, Greece</p>
<p id="fn1"><sup>4</sup>The authors contributed equally</p>
<p id="fn2"><sup>5</sup>Technical contact</p>
<p id="fn3"><sup>6</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:axel.brakhage@leibniz-hki.de">axel.brakhage@leibniz-hki.de</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100328">10.1016/j.xpro.2021.100328</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>The analysis of phagolysosomes within professional phagocytic cells is facilitated by their isolation. Here, we optimized a protocol for the isolation of intact phagolysosomes from macrophages infected with the spores of <i>Aspergillus fumigatus</i>. Purified phagolysosomes allow improved immunostaining, e.g., of phagolysosomal membrane proteins, or proteome analysis.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a href="#bib4">Schmidt et al. (2020)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>Improved protocol to isolate intact phagolysosomes from macrophages</li>
<li>Optimized cell separation allows phagolysosome analyses without cell background</li>
<li>Isolated phagolysosomes are usable for immunofluorescence and protein analyses</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical Abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/460-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>The protocol is optimized from the published work of <a href="#bib1">Kyrmizi et al. (2018)</a>.</p>
<ol>
<li>Phagocytic cells should be maintained according to the accepted protocols. Their viability should be higher than 90%, and confluency in flasks or wells should be around 80%.</li>
<li>For efficient uptake by phagocytic cells, fungal spores should be freshly harvested from mycelia.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Anti-vATPase V<sub>1</sub> antibody</td>
<td>Abcam</td>
<td>#ab73404</td>
</tr>
<tr>
<td>DyLight633</td>
<td>Invitrogen</td>
<td>#35562</td>
</tr>
<tr>
<td colspan="3">Chemicals</td>
</tr>
<tr>
<td>ATP</td>
<td>Sigma-Aldrich</td>
<td>A7699-1G</td>
</tr>
<tr>
<td>Biocoll</td>
<td>Merck</td>
<td>L6113</td>
</tr>
<tr>
<td>Bovine serum albumin</td>
<td>Sigma-Aldrich</td>
<td>A3294-50G</td>
</tr>
<tr>
<td>Mounting medium</td>
<td>Vector Laboratories</td>
<td>H-1000-10</td>
</tr>
<tr>
<td>Protease inhibitor</td>
<td>cOmplete, Roche</td>
<td>5892791001</td>
</tr>
<tr>
<td colspan="3">Experimental models: cell lines</td>
</tr>
<tr>
<td>RAW264.7</td>
<td>ATCC</td>
<td>Cat#TIB-71</td>
</tr>
<tr>
<td colspan="3">Experimental models: organisms/strains</td>
</tr>
<tr>
<td>
<i>Aspergillus fumigatus</i> strain ATCC 46645 <i>pksP</i>
</td>
<td>Jena Microbial Resource Collection (JMRC), HKI Jena</td>
<td>(<a href="#bib2">Langfelder et al., 1998</a>)</td>
</tr>
<tr>
<td colspan="3">Software</td>
</tr>
<tr>
<td>BioRender</td>
<td>BioRender 2021</td>
<td><a href="https://biorender.com/">https://biorender.com/</a></td>
</tr>
<tr>
<td>Zeiss ZEN software 2.3</td>
<td>Zeiss</td>
<td><a href="https://www.zeiss.de/">https://www.zeiss.de/</a></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<p>For cultivation of RAW264.7 macrophages, the listed DMEM medium is recommended.</p>
<table id="undtbl1">
<thead>
<tr>
<th>Reagent</th>
<th>Vendor/ manufacturer</th>
<th>Final concentration</th>
<th>Amount</th>
</tr>
</thead>
<tbody>
<tr>
<td>DMEM medium (#11960044, high glucose)</td>
<td>Gibco</td>
<td>89%</td>
<td>178 mL</td>
</tr>
<tr>
<td>Fetal calf serum</td>
<td>GE Healthcare</td>
<td>10%</td>
<td>20 mL</td>
</tr>
<tr>
<td>Ultraglutamine</td>
<td>Gibco</td>
<td>1%</td>
<td>2 mL</td>
</tr>
<tr>
<td>Gentamicin sulfate 50 mg/mL</td>
<td>Gibco</td>
<td>27.5 μg/mL</td>
<td>110 μL</td>
</tr>
<tr>
<td><b>Total</b></td>
<td> </td>
<td> </td>
<td><b>200 mL</b></td>
</tr>
</tbody>
</table>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<h3 id="sec3.1">Preparation of <i>A. fumigatus</i> conidia and macrophage cell line (RAW264.7)</h3>
<p>For the isolation of phagolysosomes from macrophages seeding of a defined amount of macrophages in well plates and cultivation of <i>A. fumigatus</i> conidia until sporulation is necessary and described below.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 10 min</div>
<ol>
<li>Revival of <i>A. fumigatus</i> conidia from cryopreservation
<ol type="a">
<li>Use an inoculation loop to spread conidia on a petri dish with <i>Aspergillus Minimal Medium</i> agar (<a href="#bib3">Pontecorvo et al., 1953</a>)</li>
<li>Incubate conidia-plated agar plate for 5 days at 37°C</li>
</ol>
</li>
</ol>
<div class="timing">
<span class="timing-title">Timing:</span> 30 min plus 3 days</div>
<ol start="2">
<li>Seed macrophage cell line
<ol type="a">
<li>Preheat cell culture medium (see “<a href="#materials-and-equipment">Materials and equipment</a>”), which should be stored at 4°C, to 37°C in a water bath</li>
<li>Prepare 2 × 25 cm<sup>2</sup> cell culture flasks with 3 × 10<sup>5</sup> macrophages (revived from cryopreserved cell line) and 7 mL culture medium each</li>
<li>Incubate flasks for 3 days at 37°C and 5% (v/v) CO<sub>2</sub>
</li>
</ol>
</li>
</ol>
<div class="timing">
<span class="timing-title">Timing:</span> 1 h plus 1 day</div>
<ol start="3">
<li>Harvest conidia 5 days after mycelial growth and sporulation on agar plates
<ol type="a">
<li>Slowly add 10 mL 0.9% (w/v) NaCl, 0.01% (v/v) Tween 20 solution to the petri dish</li>
<li>Use a T-formed spatula to scrap the conidia</li>
<li>Measure concentration of conidia by using a 1:100 dilution and counting with a Thoma chamber or another counting method</li>
</ol>
</li>
<li>Harvest adherent macrophages with a confluency of 80%–90%
<ol type="a">
<li>Pre-warm culture medium to 37°C in water bath</li>
<li>Remove the old medium from macrophages’ culture flask by aspiration</li>
<li>Add 3 mL of fresh pre-warmed medium</li>
<li>Use a cell scraper to scrape the macrophages from the base of the culture flask</li>
<li>Collect cell suspension in a 15 mL Falcon tube</li>
<li>Measure cell concentration by Thoma chamber or another counting system</li>
</ol>
</li>
<li>Seed cells for infection with <i>A. fumigatus</i> conidia
<ol type="a">
<li>Add 5 mL medium + 4 × 10<sup>6</sup> macrophages to each well of a 4-well plate (21,8 cm<sup>2</sup>/well)</li>
<li>Incubate cells 24 h at 37°C and 5% (v/v) CO<sub>2</sub>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.2">Infection of macrophages with <i>A. fumigatus</i> conidia</h3>
<p>Macrophages are incubated with a defined amount of conidia to allow phagocytosis and formation of a conidia-containing phagolysosome. The implementation of infection is explained in the following.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol start="6">
<li>Replace culture medium of the 4-well plate containing macrophages
<ol type="a">
<li>Remove the old medium</li>
<li>Add 3 mL pre-warmed fresh medium to the cells</li>
</ol>
</li>
<li>Phagocytosis of <i>A. fumigatus</i> conidia by macrophages
<ol type="a">
<li>Infect macrophages using an MOI of 5; incubate for 30 min at 4°C to synchronize phagocytosis</li>
<li>Allow phagocytosis for 2 h at 37°C and 5% (v/v) CO<sub>2</sub>
</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Isolation of conidia-containing phagolysosomes</h3>
<p>Formed phagolysosomes are isolated by the described cell lysis and Biocoll-separating process.</p>
<div class="timing">
<span class="timing-title">Timing:</span> 3 h</div>
<ol start="8">
<li>Isolation of infected cells from plate
<ol type="a">
<li>Pre-cool a tabletop centrifuge to 4°C</li>
<li>Remove old medium of the 4 wells of the infected macrophages by aspiration</li>
<li>Wash cells by adding 3 mL PBS per well and remove PBS</li>
<li>Harvest cells with cell scraper in 1.5 mL PBS per well</li>
<li>From each well, transfer cell suspension to a separate 1.5 mL Eppendorf tube</li>
<li>Centrifuge at 200 × <i>g</i> for 5 min at 4°C</li>
</ol>
</li>
<li>Homogenization of cells
<ol type="a">
<li>Discard supernatant</li>
<li>Resuspend cell pellet in 1 mL homogenization buffer (250 mM sucrose, 3 mM imidazole, pH 7.4) containing protease inhibitors</li>
<li>Lyse cells on ice by passing the cell suspensions 20 times per pellet through a 26G needle on a 1 mL syringe</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> Slowly pass the cell suspension through a needle to avoid air bubbles in the tube. Passing times should not fall below 20 times to improve cell lysis.</div>
<ol start="10">
<li>Check cell breakage
<ol type="a">
<li>Add 5 μL trypan blue to 5 μL cell suspension</li>
<li>Microscopic observation shows blue staining, which indicates disrupted cell membranes</li>
</ol>
</li>
<li>Release of actin/myosin interaction
<ol type="a">
<li>Incubate each cell homogenate with 300 μL of 10 mM ATP for 15 min with rotation at 4°C</li>
</ol>
</li>
<li>Separation of phagolysosomes from cell debris
<ol type="a">
<li>Split the 1 mL homogenate into portions of 500 μL; carefully overlay each 500 μL on top of 500 μL Biocoll (1.077 g/mL) in a reaction tube by using a 200 μL tip</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> A small pipette tip avoids too fast layering of the homogenate onto Biocoll and improves separating process.</div>
<ol start="2" type="a">
<li>Centrifuge at 600 × <i>g</i> for 20 min at 4°C</li>
<li>Discard supernatant</li>
<li>Wash pelleted conidia-containing phagolysosomes twice with ice-cold 500 μL PBS with centrifugation in-between for 5 min, 600 × <i>g</i>, at 4°C</li>
<li>Resuspend pellet in 150 μL ice-cold PBS and collect all samples in one tube</li>
<li>Seed 20 μL suspension + 1 mL PBS or count phagolysosomes and place a certain amount on a coverslip in a 24-well plate and store at 4°C (possible for max. 1 day) to allow the phagolysosomes to attach onto the coverslip. Alternatively, isolated phagolysosomes can be lysed for Western Blot analysis.</li>
</ol>
</li>
</ol>
<h3 id="sec3.4">Immunofluorescence staining of conidia-containing phagolysosomes</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 6 h</div>
<p>Membrane-associated targets of isolated phagolysosomes can be stained with specific antibodies using immunofluorescence technology as described below.</p>
<ol start="13">
<li>Fixation of phagolysosomes
<ol type="a">
<li>Slowly remove PBS from each well</li>
<li>Add 350 μL methanol/acetone (80%/20% (v/v)) or 3.5% (v/v) formaldehyde/PBS or 4% PFA for 15 min RT (depending on the target)</li>
<li>Wash three times with 350 μL PBS</li>
</ol>
</li>
<li>Blocking and staining
<ol type="a">
<li>Remove PBS</li>
<li>Add 350 μL 2% (w/v) BSA/PBS for 1 h</li>
<li>Remove blocking solution</li>
<li>Add 300 μL primary antibody diluted in blocking solutions as recommended for 1.5 h</li>
<li>Wash three times with PBS for 30 s</li>
<li>Remove PBS</li>
<li>Add 300 μL secondary antibody diluted in blocking solutions as recommended for 1 h</li>
<li>Wash three times with PBS for 30 s</li>
<li>Mount coverslips on a glass slide with mounting medium</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>For each condition, you should prepare 2 wells of a 4-well plate to isolate a sufficient number of phagolysosomes for immunofluorescence. If ∼20 μL of phagolysosome suspension are applied to a glass slide of a 24-well plate, it is possible to analyze 7–8 wells per condition in total. An example of an immunofluorescence staining of the vATPase V<sub>1</sub> subunit on isolated phagolysosomes of RAW264.7 macrophages infected with <i>A. fumigatus pksP</i> conidia is shown in <a href="#fig1">Figure 1</a>.</p>
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/460-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. Immunofluorescence staining of vATPase V<sub>1</sub> subunit on isolated phagolysosomes of RAW264.7 macrophages infected with <i>A. fumigatus pksP</i> conidia</div>
<p>Anti-vATPase V<sub>1</sub> antibody (Abcam, #ab73404) was used as primary antibody in a 1:500 dilution. Goat-anti-rabbit DyLight633 (Invitrogen, #35562) was used as secondary antibody in a 1:200 dilution. Arrows indicate isolated phagolysosomes. ∗ indicates intact macrophage.</p>
</figcaption>
</figure>
<p>The protocol can also be used for isolation of phagolysosomes from other cell lines and primary cells. Its suitability was confirmed for J774A.1 macrophage cell line and primary bone marrow-derived monocytes/macrophages from C57BL/6 mice.</p>
</section>
<section>
<h2 id="quantification-and-statistical-analysis">Quantification and statistical analysis</h2>
<p>Samples were visualized using a Zeiss LSM 780 confocal microscope and images were processed with the Zeiss ZEN software. Phagolysosomes with a positive signal for a specific target were counted and related to non-stained phagolysosomes. One hundred conidia-containing phagolysosomes per biological replicate were evaluated. The values represent the mean ± SD of three biological replicates. Additionally, immunofluorescence intensities could also be evaluated bioinformatically (<a href="#bib4">Schmidt et al., 2020</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The protocol leads to highly purified phagolysosomes; however, some cell debris or intact macrophages may remain in the suspension. A distinction of phagolysosomes from cell debris or intact cells is feasible by microscopy. A quantification of exclusively phagolysosomal proteins by Western blot proved to be more difficult.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec7.1">Problem 1</h3>
<p>The phagolysosomal suspension remains intact cells.</p>
<h3 id="sec7.2">Potential solution</h3>
<p>If effective isolation is uncertain, verification of damaged cells <i>via</i> trypan blue staining is necessary. Additional passing steps through a needle are recommended to further reduce the number of intact cells in the sample. Additionally, a careful overlaying improves phase separation between Biocoll and lysate. The use of a yellow tip for a pipetman facilitates the overlaying and improves the separation of phagolysosomes in the pellet and intact cells in the Biocoll-supernatant.</p>
<h3 id="sec7.3">Problem 2</h3>
<p>Phagolysosomes are not efficiently stained.</p>
<h3 id="sec7.4">Potential solution</h3>
<p>Successful staining of phagolysosomes with antibodies often depends on the fixation reagent, e.g., formaldehyde or methanol/acetone. If the immunofluorescence signal is weak or non-specific, other fixation methods or antibody concentrations can be assessed. For formaldehyde fixation, an addition of 0.3 M glycine to the blocking solution blocks free aldehyde groups and prevents non-specific binding.</p>
<h3 id="sec7.5">Problem 3</h3>
<p>Isolation of phagolysosomes only allows immunofluorescence of target structures on the outer membrane of phagolysosomes.</p>
<h3 id="sec7.6">Potential solution</h3>
<p>If your target of interest is expected on the inner phagolysosomal membrane, a permeabilization step with 1% (v/v) Triton X-100/PBS is recommended.</p>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Kyrmizi, I., Ferreira, H., Carvalho, A., Figueroa, J.A.L., Zarmpas, P., Cunha, C., Akoumianaki, T., Stylianou, K., Deepe, G.S., Samonis, G., et al. (2018). Calcium sequestration by fungal melanin inhibits calcium-calmodulin signalling to prevent LC3-associated phagocytosis. Nat. Microbiol. <i>3</i>, 791-803. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00035-6/sref1">View at publisher</a></p>
<p id="bib2">Langfelder, K., Jahn, B., Gehringer, H., Schmidt, A., Wanner, G., and Brakhage, A.A. (1998). Identification of a polyketide synthase gene (pksP) of <i>Aspergillus fumigatus</i> involved in conidial pigment biosynthesis and virulence. Med. Microbiol. Immunol. <i>187</i>, 79-89. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00035-6/sref2">View at publisher</a></p>
<p id="bib3">Pontecorvo, G., Roper, J.A., Hemmons, L.M., MacDonald, K.D., and Bufton, A.W. (1953). The genetics of <i>Aspergillus nidulans</i>. Adv. Genet. <i>5</i>, 141-238. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00035-6/sref3">View at publisher</a></p>
<p id="bib4">Schmidt, F., Thywißen, A., Goldmann, M., Cunha, C., Cseresnyés, Z., Schmidt, H., Rafiq, M., Galiani, S., Gräler, M.H., Chamilos, G., et al. (2020). Flotillin-dependent membrane microdomains are required for functional phagolysosomes against fungal infections. Cell Rep. <i>32</i>, 108017. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00035-6/sref4">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article Info</h2>
<h3>Resource Availability</h3>
<h4>Lead Contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the Lead Contact, Axel A. Brakhage (<a href="mailto:axel.brakhage@leibniz-hki.de">axel.brakhage@leibniz-hki.de</a>).</p>
<h4>Materials Availability</h4>
<p>Any material generated in this study is available from the Lead Contact with a completed Materials Transfer Agreement.</p>
<h4>Data and Code Availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the International Leibniz Research School (ILRS) as part of the excellence graduate school Jena School for Microbial Communication (JSMC) funded by the <a href="https://doi.org/10.13039/501100001659">Deutsche Forschungsgemeinschaft</a> (DFG) and the Carl-Zeiss Foundation, and the <a href="https://doi.org/10.13039/501100001659">DFG</a>-funded CRC 1278 "PolyTarget" (project B02 to A.A.B.). G.C. was supported by grants from the Greek State Scholarship Foundation (I.K.Y.), the Hellenic General Secretariat for Research and Technology-Excellence program (ARISTEIA), an Advanced Research Grant from Institut Mérieux (#4719), and H2020-SC1-BHC-2018-2020 (HDM-FUN 847507). The graphical abstract was created with <a href="http://biorender.com">biorender.com</a>.</p>
<h3>Author Contributions</h3>
<p>M.G., F.S., and I.K. conducted experiments and analyzed data; A.A.B. and G.C. designed the research and analyzed data; M.G., F.S., I.K., G.C., and A.A.B. wrote the paper.</p>
<h3>Declaration of Interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>