<article>
<section>
<h1 id="header">Genome editing in archaeal viruses and endogenous viral protein purification</h1>
<p><time datetime="2021-09-24">Published: September 24, 2021</time></p>
<p>Lauge Alfastsen,<sup><a href="#aff1">1</a></sup> Xu Peng,<sup><a href="#aff1">1</a></sup> and Yuvaraj Bhoobalan-Chitty<sup><a href="#aff1">1</a>,<a href="#fn1">2</a>,<a href="#fn2">3</a>,<a href="#cor1">*</a></sup></p>
<p id="aff1"><sup>1</sup>Department of Biology, University of Copenhagen, Ole Maaløes Vej 5, 2200 Copenhagen N, Denmark</p>
<p id="fn1"><sup>2</sup>Technical contact</p>
<p id="fn2"><sup>3</sup>Lead contact</p>
<p id="cor1"><sup>*</sup>Correspondence: <a href="mailto:yuvarajb@bio.ku.dk">yuvarajb@bio.ku.dk</a></p>
<p><span class="open-access">Open Access</span> • DOI: <a href="https://doi.org/10.1016/j.xpro.2021.100791">10.1016/j.xpro.2021.100791</a></p>
</section>
<section>
<h2 id="summary">Summary</h2>
<p>Archaea-infecting viruses are morphologically and genomically among the most diverse entities. Unfortunately, they are also fairly understudied due to a lack of efficient genetic tools. Here, we present a detailed protocol for the CRISPR/Cas-based genome editing of the virus SIRV2 infecting the genus <i>Sulfolobus</i>, which could easily be adapted to other archaeal viruses. This protocol also includes the procedure for endogenous viral protein purification and identification, allowing for assessing the molecular mechanisms behind virus life cycle and virus-host interactions.</p>
<p>For complete details on the use and execution of this protocol, please refer to <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref9">Mayo-Muñoz et al. (2018)</a> and <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref2">Bhoobalan-Chitty et al. (2019)</a>.</p>




<div class="highlights">
<h3>Highlights</h3>
<ul>
<li>CRISPR-based genome editing of lytic archaeal viruses</li>
<li>Electroporation procedure for hyperthermophilic archaea</li>
<li>Large-scale cultivation and protein purification from thermophilic archaeon <i>Sulfolobus</i>
</li>
<li>Endogenous viral protein purification and detection of protein-protein interactions</li>
</ul>
</div>
<div class="graphical-abstract">
<h3>Graphical abstract</h3>
<figure><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-GA.jpg" alt="GraphicalAbstract.jpg"></figure>
</div></section>
<section>
<h2 id="before-you-begin">Before you begin</h2>
<p>Lytic viruses of archaea have become a surprising source of novel anti-CRISPRs (Acrs), best exemplified by <i>Sulfolobus islandicus</i> rod-shaped virus 2 (SIRV2) which encodes homologs of all known archaeal Acrs, AcrID1 (<a href="#bib5">He et al., 2018</a>), AcrIIIB1 (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>), AcrIII-1 (<a href="#bib1">Athukoralage et al., 2020</a>). AcrID1 and AcrIIIB1 were identified, and their mechanisms deduced, using a bottom-up genetics approach, involving virus genome editing (<a href="#bib9">Mayo-Munoz et al., 2018</a>). The protocols described below include details on genomic deletions and insertions in SIRV2, along with a description of the steps involved in the purification of genomically tagged viral proteins expressed during infection.</p>
<h3 id="sec1.1">Prepare electrocompetent <i>Sulfolobus islandicus</i> cells</h3>
<div class="timing">
<span class="timing-title">Timing:</span> ∼10 days</div>
<ol>
<li>Inoculate <i>S. islandicus</i> LAL14/1 Δarrays cells from the −80°C glycerol stock into a glass tube with 5 mL fresh SCVU medium and incubate the suspension at 78°C and 150 rpm (rotations per minute) for optimal growth.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Unless stated otherwise, suspensions of <i>S. islandicus</i> LAL14/1 and its derivatives should always be grown at the conditions described above.</div>
<ol start="2">
<li>Once the optical density at 600 nm (OD<sub>600</sub>) has reached 0.6–0.8 (after approx. 36 h), transfer a 3-mL aliquot of the culture into an Erlenmeyer flask with fresh SCVU medium to a final volume of 30 mL. Allow this culture to grow for ∼36 h.</li>
<li>When the 30-mL culture reaches an OD<sub>600</sub> of 0.6–0.8, transfer a specific volume of the culture into fresh SCVU medium to a final volume of 30 mL and allow the culture to grow for approximately 24 h. The transferred culture volume <i>V</i><sub><i>1</i></sub> should be calculated according to <a href="#fd1">Equation (1)</a> such that the initial OD<sub>600</sub> of the new culture will be 0.05:</li>
</ol>
<p id="fd1" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 1)</mtext></mtd> <mtd> <mrow> <msub> <mi>V</mi> <mn>1</mn> </msub> <mo>=</mo> <mfrac> <mn>0.05</mn> <mrow> <mi>O</mi> <msub> <mi>D</mi> <mn>600</mn> </msub> </mrow> </mfrac> <mo>⋅</mo> <mn>30</mn> <mspace></mspace> <mi>m</mi> <mi>l</mi> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<ol start="4">
<li>Once the overnight culture has reached an OD<sub>600</sub> of 0.3–0.4, transfer a specific volume of the culture into fresh SCVU medium to a final volume of 30 mL with an initial OD<sub>600</sub> of 0.05. Allow this culture to grow for approximately 24 h.</li>
<li>Repeat Step 4 two more times. After the final growth, transfer the culture to a 300-mL Erlenmeyer flask with fresh SCVU medium to a final volume of 100 mL with an initial OD<sub>600</sub> of 0.05. Allow this suspension to grow overnight (for approx. 14 h).</li>
<li>At OD<sub>600</sub> ≅ 0.2, remove the 100-mL culture from the incubator and let it cool down to room temperature (RT; 18°C–30°C) before continuing. If the measured OD<sub>600</sub> is higher than 0.25, then prepare a new 100-mL culture with an initial OD<sub>600</sub> of 0.1 and allow it to grow for 5–6 h until the OD<sub>600</sub> reaches 0.2.</li>
<li>Once cool, split the 100-mL culture into two 50-mL Falcon tubes and collect the cells by spinning down the culture at 6,300 × <i>g</i> for 5 min.</li>
<li>Discard the supernatant and resuspend the cell pellets in 40 mL 20 mM sucrose (RT) with a narrow-tipped (5 mL or 10 mL) pipette. Do not vortex. Spin down the resuspended cells at 6,300 × <i>g</i> for 5 min.</li>
<li>Discard the supernatant and resuspend the cell pellets in 25 mL 20 mM sucrose (RT) with a pipette. Do not vortex. Spin down the resuspended cells at 6,300 × <i>g</i> for 5 min.</li>
<li>Discard the supernatant and resuspend the cell pellet in 10 mL 20 mM sucrose (RT) with a pipette. Do not vortex. Spin down the resuspended cells at 6,300 × <i>g</i> for 5 min.</li>
<li>Discard the supernatant and resuspend the cell pellet in 1 mL 20 mM sucrose (RT) with a pipette. Do not vortex. Transfer the resuspended cells to a microcentrifuge tube and spin it down at 6,300 × <i>g</i> for 5 min.</li>
<li>Discard the supernatant and resuspend the cell pellet in 0.5 mL 20 mM sucrose (RT) with a pipette. Do not vortex.</li>
<li>The density of the 0.5-mL cell resuspension should correspond to an OD<sub>600</sub> of 10. To verify this, take out a 25-μL aliquot from the resuspension, dilute with 975-μL 20 mM sucrose (40<b>×</b> dilution), and measure the OD<sub>600</sub> of this dilution. Based on the measured OD<sub>600</sub>, adjust the density of the final cell resuspension by adding a volume <i>V</i><sub><i>2</i></sub> of 20 mM sucrose to the cells, as given from <a href="#fd2">Equation (2)</a>:</li>
</ol>
<p id="fd2" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 2)</mtext></mtd> <mtd> <mrow> <msub> <mi>V</mi> <mn>2</mn> </msub> <mo>=</mo> <mfrac> <mrow> <mi>O</mi> <msub> <mi>D</mi> <mn>600</mn> </msub> <mo>⋅</mo> <mn>40</mn> <mo>⋅</mo> <mi>X</mi> </mrow> <mn>10</mn> </mfrac> <mo>−</mo> <mi>X</mi> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>Here, <i>OD</i><sub><i>600</i></sub> is the optical density of the 40<b>×</b> dilution and <i>X</i> is the volume of the resuspended cells that is left in the microcentrifuge tube (from step 12) after the 25-μL aliquot has been removed. This volume can be estimated with a pipette.</p>
<ol start="14">
<li>Transfer 50-μL aliquots of the adjusted cell resuspension into microcentrifuge tubes and store at −80°C until needed.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The strain <i>S. islandicus</i> LAL14/1 and its derivatives used throughout this protocol are uracil-auxotrophic mutants, to allow for stable maintenance of plasmids such as pGE1 or pGE2, which harbour the wild-type <i>pyrEF</i> operon. Thus, plasmid-free cells will only grow in SCVU medium.</div>
<div class="note">
<span class="note-title">Note:</span> Due to the high incubation temperature (78°C), the opening of the Erlenmeyer flasks should be covered with two layers of aluminium foil to prevent evaporation of culture media.</div>
<div class="critical">
<span class="critical-title">Critical:</span> If the growth rate of the culture is slower than expected based on approximate times given, repeat the protocol from step 3.</div>
<h3 id="sec1.2">Prepare a sample of SIRV2 and determine the virus titer by plaque assay</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12–14 days</div>
<ol start="15">
<li>Inoculate the host strain <i>S. islandicus</i> LAL14/1 Δarrays from the −80°C stock into a glass tube with 5 mL fresh SCVU medium and incubate the suspension at 78°C and 150 rpm.</li>
<li>Once the OD<sub>600</sub> has reached 0.6–0.8 (after approx. 36 h), transfer a 3-mL aliquot of the culture into an Erlenmeyer flask with fresh SCVU medium to a final volume of 30 mL. Allow this culture to grow for ∼36 h.</li>
<li>When the 30-mL culture of <i>S. islandicus</i> LAL14/1 Δarrays reaches an OD<sub>600</sub> of 0.6–0.8, transfer appropriate amounts of the culture into two Erlenmeyer flasks with SCVU medium to a final volume of 30 mL each such that the initial OD<sub>600</sub> of both the cultures will be around 0.15 (≅ 10<sup>8</sup> cells/mL). Infect one of the cultures with pre-titrated SIRV2 or its mutants at a multiplicity of infection (MOI) of around 10<sup>−3</sup>, while keeping the other culture uninfected. Start incubation.</li>
<li>Monitor the OD<sub>600</sub> of the infected culture and the non-infected control culture regularly for two days. If the OD<sub>600</sub>-ratio (infected/non-infected) is above 0.2 after two days, repeat the procedure from step 17 with higher MOI.</li>
<li>Isolate the virus particles by centrifuging the infected culture at 10,000 × <i>g</i> for 6 min and transfer the supernatant, containing the viruses, into a new microcentrifuge tube.</li>
<li>Plaque assay:
<ol type="a">
<li>Cast 0.7% Gelzan™ CM/SCVU plates for the plaque assay (at least four plates per sample of virus). Allow the medium to solidify at RT for 40 min.</li>
<li>Make 10-fold serial dilutions of the latest SIRV2 sample using 1<b>×</b> medium salts solution. Typically, virus dilutions in the range of 10<sup>−5</sup> to 10<sup>−8</sup> will give the most distinguishable plaques.</li>
<li>In glass tubes, mix 100 μL of each virus dilution with 2 mL of <i>S. islandicus</i> LAL14/1 Δarrays host cell suspension in early exponential growth phase (OD<sub>600</sub> ≅ 0.2). Allow these host-virus suspensions to incubate for 30 min.</li>
<li>Approximately 10 min before incubation is finished, place the following materials in an incubator set to 78°C:
<ol type="i">
<li>The 0.7% Gelzan™ CM/SCVU plates with their lids on</li>
<li>0.4% Gelzan™ CM solution</li>
</ol>
</li>
<li>Add 2 mL of pre-warmed 0.4% Gelzan™ CM to the host-virus suspension, swirl the glass tube gently and thoroughly to ensure proper mixing, and immediately pour the mixture onto the pre-warmed 0.7% Gelzan™/SCVU plate. Gently distribute the mixture evenly across the bottom layer and put the lid back on the plate. Repeat this procedure for every host-virus suspension.</li>
<li>Once the top layer has solidified (after approx. 30 min), stack the plates in a sealed plastic bag and place them in an air-tight container. Incubate at 78°C.</li>
<li>After 2–3 days, circular zones of clearing known as <i>plaques</i> should have emerged on the plates, each representing a successful infection of the <i>S. islandicus</i> LAL14/1 Δarrays host by the virus. The titer, <i>T</i> in the original (undiluted) virus sample can be calculated according to <a href="#fd3">Equation (3)</a> below:</li>
</ol>
</li>
</ol>
<p id="fd3" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 3)</mtext></mtd> <mtd> <mrow> <mi>T</mi> <mo>=</mo> <mfrac> <mrow> <mfrac> <mi>x</mi> <mrow> <mn>100</mn> <mspace></mspace> <mi>μ</mi> <mi>l</mi> </mrow> </mfrac> <mo>⋅</mo> <mn>1000</mn> <mspace></mspace> <mfrac> <mrow> <mi>μ</mi> <mi>l</mi> </mrow> <mrow> <mi>m</mi> <mi>l</mi> </mrow> </mfrac> <mspace></mspace> </mrow> <msup> <mn>10</mn> <mrow> <mo>−</mo> <mi>z</mi> </mrow> </msup> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
<p>Here, <i>x</i> is the amount of plaque forming units (PFUs) counted on a given plate and <i>z</i> is the order of magnitude of the virus dilution that the 100-μL aliquot was taken from to make that particular plate. For example, if 100 μL of the 10<sup>−7</sup> virus dilution results in 244 PFUs, we estimate that the virus titer of the original sample is 2.44·10<sup>10</sup> PFU/mL.</p>
<div class="note">
<span class="note-title">Note:</span> Only plaques that are well-defined and distinguishable should be considered as PFUs. Plates where there are too many plaques should therefore be excluded from the assay.</div>
<ol start="21">
<li>The virus sample should be stored at RT until needed to ensure that the calculated virus titer remains accurate.</li>
</ol>
</section>
<section>
<h2 id="key-resources-table">Key resources table</h2>
<table id="krt">
<thead>
<tr>
<th>REAGENT or RESOURCE</th>
<th>SOURCE</th>
<th>IDENTIFIER</th>
</tr>
</thead>
<tbody>
<tr>
<td colspan="3">Antibodies</td>
</tr>
<tr>
<td>Goat anti-Mouse IgG (H+L) SecondaryAntibody, HRP (used at 1:3000 dilution)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# 32430,RRID AB_1185566</td>
</tr>
<tr>
<td>HA Tag Monoclonal Antibody (2–2.2.14)(used at 1:10000 dilution)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat # 26183RRID AB_10978021</td>
</tr>
<tr>
<td colspan="3">Bacterial, archaeal and virus strains</td>
</tr>
<tr>
<td>
<i>Escherichia coli</i> DH5α</td>
<td>Novagen</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Sulfolobus islandicus</i> LAL14/1</td>
<td>(<a href="#bib7">Jaubert et al., 2013</a>)</td>
<td>GenBank: CP003928.1</td>
</tr>
<tr>
<td>
<i>Sulfolobus islandicus</i> LAL14/1 Δ<i>cas6(I-D)</i>
</td>
<td>(<a href="#bib5">He et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Sulfolobus islandicus</i> LAL14/1Δ<i>cas6(I-D)</i>/p<i>Cmr6</i>α<i>-HA</i>
</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Sulfolobus islandicus</i> LAL14/1 Δarrays</td>
<td>(<a href="#bib6">He et al., 2017</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Sulfolobus islandicus</i> LAL14/1 Δarrays/pSIRV2M<sub><i>gp48his</i></sub>
</td>
<td>(<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>Sulfolobus islandicus</i> rod-shaped virus 2 mutant (SIRV2M)</td>
<td>(<a href="#bib5">He et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SIRV2MΔ<i>gp48</i> (SIRV2M<sub>II</sub>)</td>
<td>(<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>SIRV2M<sub><i>gp48his</i></sub>
</td>
<td>(<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Chemicals, peptides, and recombinant proteins</td>
</tr>
<tr>
<td>Ampicillin</td>
<td>Merck</td>
<td>CAS#69-53-4</td>
</tr>
<tr>
<td>Gelzan<sup>TM</sup> CM</td>
<td>Merck</td>
<td>CAS#71010-52-1</td>
</tr>
<tr>
<td>Gelrite®</td>
<td>Carl Roth</td>
<td>CAS#71010-52-1</td>
</tr>
<tr>
<td>4-Aminobenzoic acid</td>
<td>Merck</td>
<td>CAS#150-13-0</td>
</tr>
<tr>
<td>Lipoic acid</td>
<td>Merck</td>
<td>CAS#1077-28-7</td>
</tr>
<tr>
<td>Thiamine-HCl</td>
<td>Merck</td>
<td>CAS#67-03-8</td>
</tr>
<tr>
<td>Riboflavin</td>
<td>Merck</td>
<td>CAS#83-88-5</td>
</tr>
<tr>
<td>Nicotinic acid</td>
<td>Merck</td>
<td>CAS#59-67-6</td>
</tr>
<tr>
<td>DL-Calcium pantothenic acid</td>
<td>Merck</td>
<td>CAS#137-08-6</td>
</tr>
<tr>
<td>Pyridoxine-HCl</td>
<td>Merck</td>
<td>CAS#58-56-0</td>
</tr>
<tr>
<td>Biotin</td>
<td>Merck</td>
<td>CAS#58-85-5</td>
</tr>
<tr>
<td>Folic acid</td>
<td>Merck</td>
<td>CAS#59-30-3</td>
</tr>
<tr>
<td>Cyanocobalamin</td>
<td>Merck</td>
<td>CAS#68-19-9</td>
</tr>
<tr>
<td>Difco™ Casamino Acids, vitamin assay</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#228830</td>
</tr>
<tr>
<td>Coomassie Brilliant Blue R-250 Dye</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#20278</td>
</tr>
<tr>
<td>dNTP set (100 mM solutions) diluted to 2.5 mM each (to obtain 10<b>×</b> dNTP)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#R0181</td>
</tr>
<tr>
<td>Activated charcoal</td>
<td>Sigma-Aldrich</td>
<td>CAS#7440-44-0</td>
</tr>
<tr>
<td>Acetic acid glacial</td>
<td>VWR Chemicals</td>
<td>CAS#64-19-7</td>
</tr>
<tr>
<td>Sucrose</td>
<td>VWR Chemicals</td>
<td>CAS#57-50-1</td>
</tr>
<tr>
<td>Glycerol (87% stock solution)</td>
<td>VWR Chemicals</td>
<td>CAS#56-81-5</td>
</tr>
<tr>
<td>DTT</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#R0862</td>
</tr>
<tr>
<td>SDS</td>
<td>PanReac AppliChem</td>
<td>CAS#151-21-3</td>
</tr>
<tr>
<td>Tris</td>
<td>PanReac AppliChem</td>
<td>CAS#77-86-1</td>
</tr>
<tr>
<td>HEPES</td>
<td>VWR Chemicals</td>
<td>CAS#7365-45-9</td>
</tr>
<tr>
<td>Imidazole</td>
<td>Sigma-Aldrich</td>
<td>CAS#288-32-4</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>MP Biomedicals, LLC</td>
<td>CAS#62625-28-9</td>
</tr>
<tr>
<td>Tween-20</td>
<td>Sigma-Aldrich</td>
<td>CAS#9005-64-5</td>
</tr>
<tr>
<td>Skim milk</td>
<td>VWR Chemicals</td>
<td>Cat#84615.0500</td>
</tr>
<tr>
<td>Uracil</td>
<td>PanReac Chemicals</td>
<td>CAS#66-22-8</td>
</tr>
<tr>
<td colspan="3">Critical commercial assays</td>
</tr>
<tr>
<td>Phusion™ High-Fidelity DNA Polymerase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# F530L</td>
</tr>
<tr>
<td>GeneJET™ PCR Purification Kit</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# K0702</td>
</tr>
<tr>
<td>FastDigest™ LguI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# FD1934</td>
</tr>
<tr>
<td>FastDigest™ PaeI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# FD0604</td>
</tr>
<tr>
<td>FastDigest™ XhoI</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# FD0694</td>
</tr>
<tr>
<td>FastAP™ Thermosensitive Alkaline Phosphatase</td>
<td>Thermo Fisher Scientific</td>
<td>Cat# EF0651</td>
</tr>
<tr>
<td colspan="3">
<sup><a href="#dtbl1fnlowast">∗</a></sup>Oligonucleotides</td>
</tr>
<tr>
<td>Seq-F:5′-GTACATAGTGGTACATTAAAGTAC-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>Seq-R:5′-GCCCTAACAGATAAGTATAGTAAA-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>gp48</i> in chk F:5′-GATTCTTGAACTCTAATACTGGTT-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td>
<i>gp48</i> in chk R:5′-TGAACTGTGCATATTATAGTAGCA-3′</td>
<td>This study</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">Recombinant DNA</td>
</tr>
<tr>
<td>Plasmid: pGE1</td>
<td>(<a href="#bib10">Peng et al., 2015</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pGE2</td>
<td>(<a href="#bib5">He et al., 2018</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: pΔ<i>SIRV2gp48</i>
</td>
<td>(<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td>Plasmid: p<i>SIRV2Mgp48his</i>
</td>
<td>(<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>)</td>
<td>N/A</td>
</tr>
<tr>
<td colspan="3">
<sup><a href="#dtbl1fnhash">#</a></sup>Other</td>
</tr>
<tr>
<td>HisTrap<sup>TM</sup> High Performance column (1 mL)</td>
<td>Cytiva</td>
<td>Cat#17524701</td>
</tr>
<tr>
<td>Amersham<sup>TM</sup> Protran<sup>TM</sup> 0.45μm NC</td>
<td>Cytiva</td>
<td>Cat#10600002</td>
</tr>
<tr>
<td>Gene Pulser/MicroPulser electroporation cuvettes, 0.1 cm gap (brown cap)</td>
<td>Bio-Rad</td>
<td>Cat#1652089</td>
</tr>
<tr>
<td>Pierce™ Protein Concentrator, PES (10K MWCO)</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#88513</td>
</tr>
<tr>
<td>Non-pyrogenic syringe filters (pore size 0.45-μm)</td>
<td>Frisenette</td>
<td>Cat#CALS2504100S</td>
</tr>
<tr>
<td>Non-pyrogenic syringe filters (pore size 0.22-μm)</td>
<td>Frisenette</td>
<td>Cat#CALS2502100S</td>
</tr>
<tr>
<td>Disposable PES Bottle Top Filters</td>
<td>Fisher Scientific</td>
<td>Cat#FB12566510</td>
</tr>
<tr>
<td>Thermal Cycler</td>
<td>Applied Biosystems</td>
<td>Cat#43-757-86</td>
</tr>
<tr>
<td>ThermoMixer® C</td>
<td>Eppendorf</td>
<td>Cat#5382000015</td>
</tr>
<tr>
<td>Biophotometer® D30</td>
<td>Eppendorf</td>
<td>N/A</td>
</tr>
<tr>
<td>Gene Pulser Xcell™ Electroporation System</td>
<td>Bio-Rad</td>
<td>Cat#1652666/1652668</td>
</tr>
<tr>
<td>Gene Pulser Electroporation Cuvettes, 0.1 cm gap</td>
<td>Bio-Rad</td>
<td>Cat#1652089</td>
</tr>
<tr>
<td>New Brunswick<sup>TM</sup> Innova® 42/42R</td>
<td>Eppendorf</td>
<td>Cat#M1335-0012</td>
</tr>
<tr>
<td>New Brunswick<sup>TM</sup> Innova® 44/44R</td>
<td>Eppendorf</td>
<td>Cat#M1282-0002</td>
</tr>
<tr>
<td>Sorvall LYNX 6000 Superspeed Centrifuge</td>
<td>Thermo Fisher Scientific</td>
<td>Cat#75006590</td>
</tr>
<tr>
<td>FPG12800 French Press</td>
<td>Homogenising Systems Ltd</td>
<td>N/A</td>
</tr>
<tr>
<td>Soniprep 150</td>
<td>MSE LTD</td>
<td>N/A</td>
</tr>
<tr>
<td>UVP Doc-It</td>
<td>Analytik Jena</td>
<td>Cat#98-0068-01</td>
</tr>
<tr>
<td>Mini-PROTEAN® Electrophoresis System</td>
<td>Bio-Rad</td>
<td>Cat#1658000EDU</td>
</tr>
<tr>
<td>Trans-Blot® SD Semi-Dry Transfer Cell</td>
<td>Bio-Rad</td>
<td>Cat#1703940</td>
</tr>
</tbody>
</table>
<p id="dtbl1fnlowast" class="table-footnote"><sup>∗</sup>For additional oligonucleotides mentioned in this protocol please refer to (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>).</p>
<p id="dtbl1fnhash" class="table-footnote"><sup>#</sup><i>Alternatives</i>: All equipment can be replaced by others with the same function.</p>
</section>
<section>
<h2 id="materials-and-equipment">Materials and equipment</h2>
<h3 id="sec2.1">
<i>Sulfolobus</i> growth medium</h3>
<p>The SCV medium used throughout this protocol consists of medium salts and trace elements solution supplemented with a Ca/Mg solution, 0.2% sucrose, 0.2% Casamino acids (CAA), as well as a vitamin mixture (<a href="#bib13">Zillig et al., 1993</a>). For cultivation of plasmid-free strains, the medium is furthermore supplemented with uracil to a final concentration of 20 μg/mL, hence referred to as SCV<u>U</u> medium.</p>
<p>The medium salts solution is a glycine-buffered composition of three crucial salts. We recommend preparing it as a 10<b>×</b> medium salts solution according to the table below:</p>
<table id="undtbl1">
<caption>Medium salts solution (10<b>×</b>)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Stock concentration</th>
<th>Volume per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>
</td>
<td>227 mM</td>
<td>1.89 M (25% w/v)</td>
<td>120 mL</td>
</tr>
<tr>
<td>Glycine</td>
<td>93 mM</td>
<td>1.33 M (10% w/v)</td>
<td>70 mL</td>
</tr>
<tr>
<td>K<sub>2</sub>HPO<sub>4</sub>
</td>
<td>29 mM</td>
<td>0.57 M (10% w/v)</td>
<td>51 mL</td>
</tr>
<tr>
<td>KCl</td>
<td>13 mM</td>
<td>0.67 M (5% w/v)</td>
<td>20 mL</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>N/A</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To make the stock solutions, weigh each component in a beaker and add ~800 mL distilled H<sub>2</sub>O. Once the components have dissolved, fill up with distilled H<sub>2</sub>O to 1000 mL and filter-sterilize each stock solution through a 0.22-μm filter.<br> Then, add the specified volumes of the stock solutions to a 1000-mL beaker and mix the solution with a magnetic stir bar while filling up with distilled H<sub>2</sub>O to 1000 mL. Autoclave the solution at 115°C for 15 min. The 10<b>×</b> medium salts solution can be stored at RT for more than a year.</p>
<p>A Calcium/Magnesium (Ca/Mg) stock solution should be prepared and stored individually:</p>
<table id="undtbl2">
<caption>Ca/Mg solution</caption>
<thead>
<tr>
<th>Component</th>
<th>Stock concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>MgCl<sub>2</sub>•6H<sub>2</sub>O</td>
<td>1.0 M</td>
<td>203.3 g</td>
</tr>
<tr>
<td>Ca(NO<sub>3</sub>)<sub>2</sub>•4H<sub>2</sub>O</td>
<td>0.3 M</td>
<td>70.8 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Fill a beaker with 400-mL distilled H<sub>2</sub>O and add 101.65 g MgCl<sub>2</sub>•6H<sub>2</sub>O and 35.40 g Ca(NO<sub>3</sub>)<sub>2</sub>•4H<sub>2</sub>O while the solution is thoroughly mixed by a magnetic stir bar. Fill up with distilled H<sub>2</sub>O to 500 mL and autoclave the solution at 115°C for 15 min. The Ca/Mg solution can be stored at RT for more than a year.</p>
<p>Apart from the medium salts solution and the Ca/Mg solution, nine additional salts are required but only in trace amounts. We recommend 1% or 0.2% stock solutions of each salt be prepared first and mixed together according to the volumes given in the table below:</p>
<table id="undtbl3">
<caption>Trace elements solution (10<b>×</b>)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Stock concentration (w/v)</th>
<th>Volume per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>•10H<sub>2</sub>O</td>
<td>64 μM</td>
<td>1%</td>
<td>2.4 mL</td>
</tr>
<tr>
<td>FeSO<sub>4</sub>•7H<sub>2</sub>O</td>
<td>72 μM</td>
<td>1%</td>
<td>2.05 mL</td>
</tr>
<tr>
<td>MnCl<sub>2</sub>•4H<sub>2</sub>O</td>
<td>40 μM</td>
<td>1%</td>
<td>800 μL</td>
</tr>
<tr>
<td>ZnSO<sub>4</sub>•7H<sub>2</sub>O</td>
<td>3.7 μM</td>
<td>1%</td>
<td>110 μL</td>
</tr>
<tr>
<td>CuSO<sub>4</sub>•5H<sub>2</sub>O</td>
<td>1.0 μM</td>
<td>0.2%</td>
<td>125 μL</td>
</tr>
<tr>
<td>Na<sub>2</sub>MoO<sub>4</sub>•2H<sub>2</sub>O</td>
<td>0.61 μM</td>
<td>0.2%</td>
<td>75 μL</td>
</tr>
<tr>
<td>VOSO<sub>4</sub>•5H<sub>2</sub>O</td>
<td>0.59 μM</td>
<td>0.2%</td>
<td>75 μL</td>
</tr>
<tr>
<td>NiSO<sub>4</sub>•6H<sub>2</sub>O</td>
<td>0.19 μM</td>
<td>0.2%</td>
<td>25 μL</td>
</tr>
<tr>
<td>CoSO<sub>4</sub>•7H<sub>2</sub>O</td>
<td>0.18 μM</td>
<td>0.2%</td>
<td>25 μL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To make the stock solutions, weigh each trace element in a beaker and add ~80 mL distilled H<sub>2</sub>O. Once the components have dissolved, fill up with distilled H<sub>2</sub>O to 100 mL and filter-sterilize each stock solution through a 0.22-μm filter. To prepare the stock solution of FeSO<sub>4</sub> add some 0.5 M HCl before adding the distilled H<sub>2</sub>O to allow for proper dissolution. Next, add the specified volumes of the stock solutions to a 1000-mL beaker and mix the solution with a magnetic stir bar while filling up with distilled H<sub>2</sub>O. Adjust the pH to 3.0 using a 50% (v/v) sulfuric acid solution and fill up with distilled H<sub>2</sub>O to 1000 mL. Autoclave the solution at 115°C for 15 min. The 10<b>×</b> trace elements solution can be stored at RT for more than a year.</p>
<p>The stock solutions of sucrose, CAA, uracil, and vitamin should be made according to the tables below:</p>
<table id="undtbl4">
<caption>Vitamin mixture (200<b>×</b>)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Stock concentration (w/v)</th>
<th>Stock storage temperature</th>
<th>Volume per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>4-Aminobenzoic acid</td>
<td>140 μM</td>
<td>0.2%</td>
<td>4°C</td>
<td>10 mL</td>
</tr>
<tr>
<td>Lipoic acid</td>
<td>96 μM</td>
<td>1%</td>
<td>RT</td>
<td>2 mL</td>
</tr>
<tr>
<td>Thiamine-HCl (B1)</td>
<td>59 μM</td>
<td>1%</td>
<td>RT</td>
<td>2 mL</td>
</tr>
<tr>
<td>Riboflavin (B2)</td>
<td>53 μM</td>
<td>1%</td>
<td>RT</td>
<td>2 mL</td>
</tr>
<tr>
<td>Nicotinic acid (B3)</td>
<td>160 μM</td>
<td>1%</td>
<td>RT</td>
<td>2 mL</td>
</tr>
<tr>
<td>DL-Calcium pantothenic acid (B5)</td>
<td>42 μM</td>
<td>1%</td>
<td>4°C</td>
<td>2 mL</td>
</tr>
<tr>
<td>Pyridoxine-HCl (B6)</td>
<td>97 μM</td>
<td>1%</td>
<td>RT</td>
<td>2 mL</td>
</tr>
<tr>
<td>Biotin (B7)</td>
<td>32 μM</td>
<td>0.4%</td>
<td>4°C</td>
<td>2 mL</td>
</tr>
<tr>
<td>Folic acid (B9)</td>
<td>18 μM</td>
<td>1%</td>
<td>RT</td>
<td>0.8 mL</td>
</tr>
<tr>
<td>Cyanocobalamin (B12)</td>
<td>14 μM</td>
<td>0.5%</td>
<td>4°C</td>
<td>4 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">To make the stock solutions, weigh each vitamin in a beaker and add ∼80 mL distilled H<sub>2</sub>O. Once dissolved completely, fill up with distilled H<sub>2</sub>O to 100 mL and filter-sterilize each stock solution through a 0.22-μm filter. Store the stock solutions at the assigned temperatures concealed from any light.<br> Then, add the specified volumes of the stock solutions to a 1000-mL beaker and mix the solution with a magnetic stir bar while filling up with distilled H<sub>2</sub>O to 1000 mL. Filter-sterilize the solution through a 0.22-μm filter and distribute the 200<b>×</b> vitamin mixture into glass tubes as 5-mL aliquots. We recommend heating the tubes at 78°C for 1 h afterward. The tubes can be stored at 4°C in a darkroom for ∼ 30 days.</p>
<table id="undtbl5">
<caption>Sucrose solution (20%)</caption>
<thead>
<tr>
<th>Component</th>
<th>Amount per liter (g)</th>
</tr>
</thead>
<tbody>
<tr>
<td>Sucrose</td>
<td>200 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Depending on how many liters of SCV medium is required, fill a beaker with ∼300 mL distilled, autoclaved H<sub>2</sub>O and add 100 g sucrose while the solution is gently mixed by a magnetic stir bar. Once dissolved, fill up with distilled, autoclaved H<sub>2</sub>O to 500 mL. Autoclave the solution at 115°C for 15 min and distribute the solution into glass tubes as 10-mL aliquots. We recommend heating the tubes at 78°C for 1 h afterward. The tubes can be stored at 4°C for ∼ 30 days.</p>
<table id="undtbl6">
<caption>CAA solution (15%)</caption>
<thead>
<tr>
<th>Component</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>CAA</td>
<td>150 g</td>
</tr>
<tr>
<td>Activated coal</td>
<td>0.4 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Fill a beaker with ∼300 mL distilled, autoclaved H<sub>2</sub>O and add 75 g CAA while the solution is gently mixed by a magnetic stir bar. Furthermore, add 0.2 g activated coal to remove impurities. For proper dissolution it is necessary to heat the solution to near boiling point. Once dissolved, filter the solution twice to remove coal and impurities. The solution is then autoclaved at 115°C for 15 min and followingly distributed into glass tubes as 14-mL aliquots. We recommend heating the tubes at 78°C for 1 h afterward. The tubes can be stored at 4°C for ∼ 30 days.</p>
<table id="undtbl7">
<caption>Uracil solution (100<b>×</b>)</caption>
<thead>
<tr>
<th>Component</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Uracil</td>
<td>2 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Depending on the volume of SCV medium required, fill a beaker with ∼80 mL distilled, autoclaved H<sub>2</sub>O and add 0.2 g Uracil while the solution is gently mixed with a magnetic stir bar. Once dissolved, fill up with distilled, autoclaved H<sub>2</sub>O to 100 mL. Filter-sterilize the solution through a 0.22-μm filter. The uracil stock can be stored at RT for 6 months.</p>
<p>Once the stock solutions have been made and sterilized, mix them together in the volumes specified in the table below in order to make the 2<b>×</b> SCVU medium:</p>
<table id="undtbl8">
<caption>2<b>×</b>SCVU</caption>
<thead>
<tr>
<th>Stock solutions</th>
<th>Final concentrations</th>
<th>Volume per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Medium salts solution (10<b>×</b>)</td>
<td>2<b>×</b>
</td>
<td>200 mL</td>
</tr>
<tr>
<td>Trace elements solution (10<b>×</b>)</td>
<td>2<b>×</b>
</td>
<td>200 mL</td>
</tr>
<tr>
<td>Ca/Mg solution</td>
<td>2.0 mM Mg<sup>2+</sup>, 0.6 mM Ca<sup>2+</sup>
</td>
<td>2 mL</td>
</tr>
<tr>
<td>Sucrose solution (20%)</td>
<td>4.0 g/l</td>
<td>20 mL</td>
</tr>
<tr>
<td>CAA solution (15%)</td>
<td>4.2 g/l</td>
<td>28 mL</td>
</tr>
<tr>
<td>Vitamin mixture (200<b>×</b>)</td>
<td>2<b>×</b>
</td>
<td>10 mL</td>
</tr>
<tr>
<td>Uracil solution (100<b>×</b>) <sup><a href="#dtbl9fna">a</a></sup>
</td>
<td>0.04 g/l</td>
<td>20 mL</td>
</tr>
<tr>
<td>Distilled, autoclaved H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">The 2<b>×</b>SCVU medium can be stored at RT for ∼ 10 days.</p>
<p id="dtbl9fna" class="table-footnote"><sup>a</sup>Uracil should be omitted from the medium when culturing plasmid-carrying strains.</p>
<div class="optional">
<span class="optional-title">Optional:</span> The stock solutions of sucrose and CAA can alternatively be sterilized by filtering through a 0.22-μm filter. The only solution that strictly cannot tolerate autoclaving at 115°C for 15 minutes is the vitamin mixture.</div>
<p>When growth on solid medium is required, the 2<b>×</b>SCVU medium is mixed with a gelling agent such as Gelzan™ CM and some additional Ca/Mg solution as described in the table below:</p>
<table id="undtbl9">
<caption>0.7% Gelzan™ CM/SCVU solid media plates (enough for ∼12 plates)</caption>
<thead>
<tr>
<th>Component</th>
<th>Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td>2<b>×</b>SCVU</td>
<td>200 mL</td>
</tr>
<tr>
<td>1.4% (w/v) Gelzan™ CM</td>
<td>200 mL</td>
</tr>
<tr>
<td>1 M MgCl<sub>2</sub>/0.3 M Ca(NO<sub>3</sub>)<sub>2</sub>
</td>
<td>3.6 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Pour the 2<b>×</b>SCVU medium and the 1.4% Gelzan™ CM into separate 500-mL Duran® bottles. Add the Ca/Mg solution to the bottle with the 2<b>×</b>SCVU medium. Microwave both bottles (with caps slightly loosened) until the solutions start boiling. Pour the content of either of the bottles into the other, close the cap, and swirl the bottle thoroughly to ensure proper mixing. Pour ∼30 mL of the mix into polystyrene petri dishes and let the medium solidify for ∼30 min at RT.</p>
<div class="note">
<span class="note-title">Note:</span> Based on our experience, plates for the plaque assays should be made with Gelzan™ CM (Merck) as the gelling agent, whereas plates used for plating of electroporants (described later in this protocol) should be made with Gelrite® (Carl Roth) as the gelling agent.</div>
<div class="critical">
<span class="critical-title">Critical:</span> The 50% (v/v) sulfuric acid solution causes severe skin burns and eye damage upon contact and is corrosive to the respiratory tract. It should always be handled under ventilation while wearing a protective face-shield and acid-resistant gloves.</div>
<h3 id="sec2.2">Miscellaneous solutions</h3>
<table id="undtbl10">
<caption>Lysis buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES (pH 7.5)</td>
<td>50 mM</td>
<td>11.92 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>8.76 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>5% (v/v)</td>
<td>57 mL of an 87% (v/v) glycerol stock solution</td>
</tr>
<tr>
<td>Imidazole</td>
<td>30 mM</td>
<td>2.04 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">Fill a 1000-mL beaker with ∼700 mL milli-Q H<sub>2</sub>O and add the specific amounts of components while the solution is stirred vigorously by a magnetic stir bar. Adjust the pH to 7.5 with concentrated HCl or KOH and fill up to 1000-mL with milli-Q H<sub>2</sub>O. Filter-sterilize (Disposable PES bottle Top Filters) the buffer. The buffer can be stored at RT for 1–2 month.</p>
<table id="undtbl11">
<caption>Elution buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>HEPES (pH 7.5)</td>
<td>50 mM</td>
<td>11.92 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>150 mM</td>
<td>8.76 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>5% (v/v)</td>
<td>57 mL of an 87% (v/v) glycerol stock solution</td>
</tr>
<tr>
<td>Imidazole</td>
<td>500 mM</td>
<td>34.03 g</td>
</tr>
</tbody>
</table>
<p class="table-legend">The instructions and storage conditions of the elution buffer are the same as those of the lysis buffer.</p>
<table id="undtbl12">
<caption>4<b>×</b> SDS-PAGE sample loading dye</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per 10 mL</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 6.8)</td>
<td>200 mM</td>
<td>0.24 g</td>
</tr>
<tr>
<td>DTT</td>
<td>400 mM</td>
<td>0.62 g</td>
</tr>
<tr>
<td>Glycerol</td>
<td>24% (v/v)</td>
<td>2.75 mL of an 87% (v/v) glycerol stock solution</td>
</tr>
<tr>
<td>SDS</td>
<td>8% (w/v)</td>
<td>0.80 g</td>
</tr>
<tr>
<td>Bromophenol blue</td>
<td>0.04% (w/v)</td>
<td>4 mg</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 10 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">We recommend preparing the 4<b>×</b> SDS-PAGE sample loading dye as a 10-mL batch. Mix the components in a 50-mL beaker under agitation by a magnetic stir bar and fill up to 8 mL with distilled H<sub>2</sub>O. Disperse the loading dye into microcentrifuge tubes as 0.5-mL aliquots. These tubes can be stored at −20°C for a year.</p>
<table id="undtbl13">
<caption>1<b>×</b> SDS-PAGE running buffer</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris</td>
<td>25 mM</td>
<td>3.03 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>192 mM</td>
<td>14.41 g</td>
</tr>
<tr>
<td>SDS</td>
<td>0.1% (w/v)</td>
<td>1 g</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Pour ∼900 mL of distilled H<sub>2</sub>O in a 1000-mL Duran® bottle and add the specified amounts of the components while the solution is agitated by a magnetic stir bar. Once dissolved, fill up to 1000 mL with distilled H<sub>2</sub>O. There is no need for pH adjustment for the running buffer (the pH should set around 8.3). The running buffer can be stored at RT for a year.</p>
<table id="undtbl14">
<caption>SDS-PAGE gel staining solution</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid glacial</td>
<td>10% (v/v)</td>
<td>100 mL</td>
</tr>
<tr>
<td>Ethanol</td>
<td>45% (v/v)</td>
<td>450 mL</td>
</tr>
<tr>
<td>Coomassie Brilliant Blue R-250</td>
<td>0.25% (w/v)</td>
<td>2.5 g</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Pour the components into a 1000-mL beaker under vigorous agitation by a magnetic stir bar. Stir for at least 2 h. Transfer to a Duran® bottle after filtering. The solution is stable at RT concealed from any light for a year.</p>
<table id="undtbl15">
<caption>SDS-PAGE gel destaining solution</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Acetic acid glacial</td>
<td>7.5% (v/v)</td>
<td>75 mL</td>
</tr>
<tr>
<td>Ethanol</td>
<td>20% (v/v)</td>
<td>200 mL</td>
</tr>
<tr>
<td>Distilled H<sub>2</sub>O</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Add the specified volumes of the solutions together in a 1000-mL Duran® bottle. The destaining solution can be stored at RT for a year</p>
<table id="undtbl16">
<caption>Semi-dry transfer buffer (for Western blot)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 7.5)</td>
<td>25 mM</td>
<td>3.03 g</td>
</tr>
<tr>
<td>Glycine</td>
<td>192 mM</td>
<td>14.41 g</td>
</tr>
<tr>
<td>Ethanol</td>
<td>20% (v/v)</td>
<td>200 mL</td>
</tr>
<tr>
<td>Milli-Q</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">Fill a 1000-mL beaker with ∼700 mL milli-Q H<sub>2</sub>O and add the specified amounts of each component while the solution is stired vigorously by a magnetic stir bar. Adjust the pH with concentrated HCl and fill up to 1000 mL with milli-Q H<sub>2</sub>O. The transfer buffer can be stored at 4°C for ∼ 2 months.</p>
<table id="undtbl17">
<caption>Wash buffer (for Western blot)</caption>
<thead>
<tr>
<th>Component</th>
<th>Final concentration</th>
<th>Amount per liter</th>
</tr>
</thead>
<tbody>
<tr>
<td>Tris (pH 7.5)</td>
<td>25 mM</td>
<td>3.03 g</td>
</tr>
<tr>
<td>NaCl</td>
<td>100 mM</td>
<td>5.84 g</td>
</tr>
<tr>
<td>Tween-20</td>
<td>0.5% (v/v)</td>
<td>0.5 mL</td>
</tr>
<tr>
<td>Milli-Q</td>
<td>N/A</td>
<td>Up to 1000 mL</td>
</tr>
</tbody>
</table>
<p class="table-legend">The instructions and storage conditions of the wash buffer are the same as those of the semi-dry transfer buffer, except the wash buffer should be can at RT for ∼ 3 months.</p>
<table id="undtbl18">
<caption>Other solutions</caption>
<thead>
<tr>
<th>Name</th>
<th>Recipe</th>
</tr>
</thead>
<tbody>
<tr>
<td>1.4% (w/v) Gelrite®</td>
<td>2.8 g Gelrite® in 200 mL distilled H<sub>2</sub>O</td>
</tr>
<tr>
<td>0.4% (w/v) Gelrite®</td>
<td>0.4 g Gelrite® in 100 mL distilled H<sub>2</sub>O</td>
</tr>
<tr>
<td>1.4% (w/v) Gelzan™ CM</td>
<td>2.8 g Gelzan™ CM in 200 mL distilled H<sub>2</sub>O</td>
</tr>
<tr>
<td>0.4% (w/v) Gelzan™ CM</td>
<td>0.4 g Gelzan™ CM in 100 mL distilled H<sub>2</sub>O</td>
</tr>
</tbody>
</table>
<p class="table-legend">Autoclave the solutions at 121°C for 15 min. The Gelrite® and Gelzan™ solutions can be stored at RT for ∼ 30 days.</p>
<div class="critical">
<span class="critical-title">Critical:</span> The concentrated HCl and KOH used for pH adjustments cause severe skin burns and eye damage upon contact and are corrosive to the respiratory tract. These reagents should always be handled under ventilation while wearing a protective face-shield and acid-resistant gloves.</div>
</section>
<section>
<h2 id="step-by-step-method-details">Step-by-step method details</h2>
<p>This protocol provides detailed guidelines for performing genome editing of archaeal lytic viruses, exemplified here with the virus SIRV2. Essentially, the entire procedure relies on the construction of a genome editing plasmid, based on the shuttle vectors pGE1/pGE2. <i>S. islandicus</i> LAL14/1 carries one subtype I-A, one subtype I-D and two subtype III-B (Cmr-α and Cmr-γ) systems along with five CRISPR arrays. Arrays neighboring subtype I-A encompass repeat sequences (referred to as type I-A repeats) that are utilized by subtype I-A and subtype III-B (Cmr-α), the other three arrays encompass repeat sequences (referred to as type I-D repeats) utilized by subtype I-D and subtype III-B (Cmr-γ) (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>). Once transformed into <i>S. islandicus</i> LAL14/1 Δarrays, a fully assembled pGE1/pGE2 allows the endogenous type I and type III CRISPR-Cas systems of the host cell to be repurposed to perform genome editing of SIRV2 during infections.</p>
<p>Two variable DNA elements in pGE1/pGE2 make this genome editing possible:</p>
<ul>
<li>A spacer DNA element that promotes targeting of the original virus but not the mutant virus. This is made possible by the mini-CRISPR array harbored on the genome editing plasmid. Briefly, the spacer is inserted between two type I-A repeats (for pGE1) or two type I-D repeats (for pGE2) located downstream of an arabinose-inducible promoter (<a href="#fig1">Figure 1</a>), causing CRISPR RNA (crRNA) species to be produced. The crRNAs will in turn guide the endogenous CRISPR-Cas effector proteins towards the cognate protospacer in the original virus, leading to mostly fatal breaks in its DNA genome.
<figure id="fig1"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-Fig1.jpg" alt="Fig1.jpg">
<figcaption>
<div class="figcaption-title">Figure 1. The concepts of the CRISPR-Cas based genome editing of lytic archaeal viruses</div>
<p>Genome editing of lytic archaeal viruses takes place during infection of a host cell that carries the appropriate genome editing plasmid (pGE1/pGE2). The genome editing plasmid harbors a mini-CRISPR array constituted by a single repeat-spacer-repeat unit located downstream from an arabinose-inducible promoter (P<sub>araS</sub>). The resulting CRISPR RNA (crRNA) will help the endogenous CRISPR-Cas systems to recognize and cleave the protospacer of the infecting virus, as determined by the spacer sequence defined by the researcher. Some viruses will followingly repair their genomes through a double-crossover event with the donor DNA element present on pGE1/pGE2 (highlighted in gray), thus giving rise to a mutant virus that will not be targeted. This method has been used previously to delete the gene <i>gp48</i> from SIRV2M (A) and to insert an affinity tag onto the C-terminus of <i>gp48</i> (B). Primers are indicated above/below the regions of the virus genome to which they anneal.</p>
</figcaption>
</figure>
</li>
<li>A donor DNA element, which represents the desired virus mutation. This mutant allele is flanked by regions perfectly homologous to the regions flanking the corresponding wild-type allele in the virus. This shared homology allows the donor DNA to be inserted into the viral genome via a double-crossover event (<a href="#fig1">Figure 1</a>), thus generating the mutant virus.</li>
</ul>
<p>The result is that the original virus is eliminated while the mutant virus remains untargeted and is allowed to propagate.</p>
<p>Throughout this protocol, we will occasionally help explain selected steps by giving examples from two previous works: the targeted deletion of <i>gp48</i> from the genome of SIRV2M (<a href="#fig1">Figure 1</a>A), and the insertion of an octahistidine (8xHis)-tag onto the C-terminus of <i>gp48</i> in the genome of SIRV2M (<a href="#fig1">Figure 1</a>B) (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>).</p>
<h3 id="sec3.1">Construction of the genome editing plasmid</h3>
<h4 id="sec3.1.1">Design, assembly, and insertion of the spacer DNA element</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 4 days</div>
<ol>
<li>Within the region of interest in the virus genome, select a 39- to 45-bp protospacer (<a href="#fig2">Figure 2</a>A) that fulfills the following criteria:
<figure id="fig2"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-Fig2.jpg" alt="Fig2.jpg">
<figcaption>
<div class="figcaption-title">Figure 2. Design of the spacer DNA element and sequence features of pGE2</div>
<p>The region of interest in the virus genome (A, top) is screened for a non-redundant protospacer with a 5′-end flanked by a 5′-GTN-3′ protospacer adjacent motif (PAM) compatible with the type I-D CRISPR-Cas systems of the host. Here, ‘N’ can assume any nucleotide. Based on the selected protospacer, the spacer DNA element is assembled from two ssDNA oligonucleotides with sequences complementary to each strand of the protospacer (A, bottom). Furthermore, each ssDNA oligonucleotide includes 3-nt 5′-overhanging sequences, allowing the final spacer DNA element to be cloned in between the type subtype I-D repeats found in pGE2. The backbone of pGE2 (A, top) contains an ampicillin resistance marker (AmpR) and the <i>pyrEF</i> operon which allows for stable maintenance of pGE2 in <i>E. coli</i> DH5α and <i>S. islandicus</i> LAL14/1, respectively. The multiple cloning site of pGE2 (A, bottom) consists of an insertion site for the donor DNA (between the PaeI and XhoI restriction sites) as well as a type I-D mini-CRISPR-array where the insertion site for the spacer is located (between the two oppositely oriented LguI sites). An arabinose promoter (ara-box), B recognition element (BRE), and TATA-box ensure that the mini-CRISPR-array is transcribed. Please note, the multiple cloning site shown here neither contains a spacer nor a donor DNA. TSS = Transcriptional Start Site.</p>
</figcaption>
</figure>
<ol type="a">
<li>The protospacer should be absent or disrupted in the desired mutant virus.</li>
<li>Ideally, the protospacer should be chosen from the template strand, as type III CRISPR-Cas systems can only recognize and bind transcript-RNA rather than DNA.</li>
<li>The 5′-end of the protospacer should be flanked by a protospacer adjacent motif (PAM) compatible with the subtype I-A or subtype I-D CRISPR-Cas systems of <i>S. islandicus</i> LAL14/1, i.e., 5′-CCN-3′ or 5′-GTN-3′, respectively (<a href="#bib4">Gudbergsdottir et al., 2011</a>; <a href="#bib8">Lin et al., 2020</a>). Here, ‘N’ can assume any nucleotide.</li>
</ol>
</li>
<li>To create the spacer element, design and order a pair of single-stranded (ss)DNA oligos that are complementary to the two strands of the selected protospacer. The two ssDNA oligos should have 3-nucleotide (nt) 5′-overhangs with sequences AAG/AGC or AAC/TAC to match the oppositely oriented LguI sites of pGE1 or pGE2, respectively (<a href="#fig2">Figure 2</a>B).
<ol type="a">
<li>Add equal volumes of the two ssDNA oligos (10 μM stocks) to a microcentrifuge tube, mix by vortexing, and incubate at 95°C for 10 min in a heat block.</li>
<li>Once incubation is done, turn off the heat block and allow the mixture to cool down to RT gradually to promote the hybridization of the two ssDNA oligos.</li>
<li>The resulting double-stranded (ds)DNA spacer element will be 39 bp in length with 3-nt 5′-overhangs that allows it to be inserted in pGE1 or pGE2. Store at −20°C until needed.</li>
</ol>
</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The backbones of pGE1 and pGE2 contain mini-CRISPR arrays made with type I-A repeats and type I-D repeats, respectively, but apart from this the two plasmids have identical sequences. For clarity, the map and sequence of the multiple cloning site (MCS) of pGE2 is shown in <a href="#fig2">Figure 2</a>B. For the rest of the protocol, we will keep using pGE2 as an example.</div>
<ol start="3">
<li>Clone the dsDNA spacer element (from step 2) into a linearized (FD LguI) sample of pGE2 to construct a plasmid encoding a mini-CRISPR array expressing a spacer targeting the original SIRV2 genome (pGE2-Spc<i>gp48</i>).</li>
<li>The purified pGE2-Spc<i>gp48</i> sample should be stored at −20°C until needed.</li>
</ol>
<h4 id="sec3.1.2">Design, assembly, and insertion of the donor DNA element</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<ol start="5">
<li>The donor DNA element is assembled via fusion PCR by overlap extension (<a href="#fig3">Figure 3</a>). To do this, design and order two pairs of primers which will anneal to and amplify the regions of the virus genome located immediately up and downstream, respectively, from the target gene (deletions) and target site (modifications). The resulting amplicons are referred to as the left and right arm of the donor DNA. Each arm should be 200–500 bp in length. Importantly, every primer used for donor DNA assembly should include a 5′-overhang of a particular sequence:
<figure id="fig3"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-Fig3.jpg" alt="Fig3.jpg">
<figcaption>
<div class="figcaption-title">Figure 3. Assembly of donor DNA elements via fusion PCR by overlap extension</div>
<p>The examples shown illustrate how the donor DNA elements were generated that promote (A) the deletion of <i>gp48</i> and (B) the insertion of a histidine tag onto the C-terminus of gp48, as originally performed by (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>). Here, the SIRV2 genome serves as template in the first round of PCRs [1], by which two DNA fragments are formed, denoted as the left and right arm of the donor DNA. Due to the specific sequences included in the 5′-overhangs of the inner (fusion) primers, the left and right arms will contain overlapping sequences that allow the arms to hybridize when mixed [2]. In these hybrids, the strand from either arm can use the strand from the opposite arm as a template for extension in the second round of PCR [3], thus yielding a complete, double-stranded donor DNA. This is further amplified by the outer primers, which are also responsible for integrating restriction sites PaeI and XhoI in the ends of the donor DNA. PS = protospacer.</p>
</figcaption>
</figure>
<ol type="a">
<li>The two <i>outer</i> primers (e.g., <i>gp48</i> Kn PaeI F and <i>gp48</i> Kn XhoI R, as shown in <a href="#fig3">Figure 3</a>A) should include sequences for the restriction sites PaeI and XhoI in their 5′-overhangs such that the donor DNA will have restriction sites matching those found in the MCS of pGE2 (<a href="#fig2">Figure 2</a>).</li>
<li>The two <i>inner</i> (fusion) primers should contain a shared sequence in their 5′-overhangs, thus allowing the left and right arms of the donor DNA to overlap and fuse. The specific sequence of the 5′-ends of these inner primers also determine the kind of mutation made:
<ol type="i">
<li>For deletions (<a href="#fig3">Figure 3</a>A), the 5′-overhanging sequence of each inner primer should be identical to the sequence on the opposite side of the region intended for deletion.</li>
<li>For insertion of affinity tags (<a href="#fig3">Figure 3</a>B), the genetic sequence encoding the tag should be included within the 5′-overhanging sequences of both inner primers.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="6">
<li>Set up the first round of PCR reactions according to the tables below. To exemplify, we here describe how the arms for the donor DNA promoting the insertion of the 8xHis-tag on the C-terminus of <i>gp48</i> (<a href="#fig3">Figure 3</a>B) were made:
<ol type="a">
<li>Two individual PCR mixtures, one for each arm of the donor DNA:
<table id="undtbl19">
<thead>
<tr>
<th colspan="2">Left arm of donor DNA</th>
<th colspan="2">Right arm of donor DNA</th>
</tr>
</thead>
<tbody>
<tr>
<td>Component</td>
<td>Volume (μl)</td>
<td>Component</td>
<td>Volume (μl)</td>
</tr>
<tr>
<td>5<b>×</b> HF buffer</td>
<td>10</td>
<td>5<b>×</b> HF buffer</td>
<td>10</td>
</tr>
<tr>
<td>10<b>×</b> dNTPs</td>
<td>5</td>
<td>10<b>×</b> dNTPs</td>
<td>5</td>
</tr>
<tr>
<td>Forward outer primer (SIRV2<i>gp48</i> Chis PaeI F, 10 μM)</td>
<td>2.5</td>
<td>Forward inner primer (SIRV2<i>gp48</i> Chis Fus F, 10 μM)</td>
<td>2.5</td>
</tr>
<tr>
<td>Reverse inner primer (SIRV2<i>gp48</i> Chis Fus R, 10 μM)</td>
<td>2.5</td>
<td>Reverse outer primer (SIRV2<i>gp48</i> Chis XhoI R, 10 μM)</td>
<td>2.5</td>
</tr>
<tr>
<td>Phusion™ High-Fidelity DNA Polymerase</td>
<td>0.2</td>
<td>Phusion™ High-Fidelity DNA Polymerase</td>
<td>0.2</td>
</tr>
<tr>
<td>SIRV2 genomic DNA (1 ng/ul)</td>
<td>1</td>
<td>SIRV2 genomic DNA (1 ng/ul)</td>
<td>1</td>
</tr>
<tr>
<td>Milli-Q H<sub>2</sub>O</td>
<td>up to 50</td>
<td>Milli-Q H<sub>2</sub>O</td>
<td>up to 50</td>
</tr>
<tr>
<td>Total volume</td>
<td>50</td>
<td>Total volume</td>
<td>50</td>
</tr>
</tbody>
</table>
</li>
<li>The PCR program is the same for either PCR:
<table id="undtbl20">
<thead>
<tr>
<th>Stage</th>
<th>Temperature (°C)</th>
<th>Duration</th>
<th>Number of cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98</td>
<td>5 s</td>
<td rowspan="3">×31</td>
</tr>
<tr>
<td>Annealing</td>
<td>59.5</td>
<td>22 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>8 s</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72</td>
<td>7 min</td>
<td>1</td>
</tr>
</tbody>
</table>
<p class="table-legend">The annealing temperature is equal to the salt-adjusted melting temperature of the primers, as calculated by OligoCalc (Link: <u>OligoCalc: Oligonucleotide Properties Calculator</u> (<a href="http://northwestern.edu">northwestern.edu</a>)).</p>
</li>
</ol>
</li>
</ol>
<ol start="7">
<li>Verify the length/purity of the left and right arm PCR fragments by agarose gel electrophoresis.</li>
<li>Purify the PCR fragments of the left and right arm with the GeneJET™ PCR Purification Kit.</li>
</ol>
<ol start="9">
<li>Set up the second round of PCR according to the tables below:
<ol type="a">
<li>The PCR mixture should contain both arms of the donor DNA at equal concentrations.
<table id="undtbl21">
<thead>
<tr>
<th colspan="2">Left arm of donor DNA</th>
</tr>
<tr>
<th>Component</th>
<th>Volume (μl)</th>
</tr>
</thead>
<tbody>
<tr>
<td>5<b>×</b> HF buffer</td>
<td>10</td>
</tr>
<tr>
<td>10<b>×</b> dNTPs</td>
<td>5</td>
</tr>
<tr>
<td>Forward outer primer (SIRV2<i>gp48</i> Chis PaeI F, 10 μM)</td>
<td>2.5 <sup><a href="#dtbl22fna">a</a></sup>
</td>
</tr>
<tr>
<td>Reverse outer primer (SIRV2<i>gp48</i> Chis Fus R, 10 μM)</td>
<td>2.5 <sup><a href="#dtbl22fna">a</a></sup>
</td>
</tr>
<tr>
<td>Left arm of donor DNA (1 ng/μL)</td>
<td>1</td>
</tr>
<tr>
<td>Right arm of donor DNA (1 ng/μL)</td>
<td>1</td>
</tr>
<tr>
<td>Phusion™ High-Fidelity DNA Polymerase</td>
<td>0.2</td>
</tr>
<tr>
<td>Milli-Q H<sub>2</sub>O</td>
<td>up to 50</td>
</tr>
<tr>
<td>Total volume</td>
<td>50</td>
</tr>
</tbody>
</table>
<p id="dtbl22fna" class="table-footnote"><sup>a</sup>The two outer primers should not be added to the reaction mixture until after the hold at 4°C.</p>
</li>
<li>The PCR program is split into two stages, the two outer primers are not added until the completion of the first stage. This is because the left and right arms of the donor DNA should first denature, mix, and reanneal, thus allowing for DNA polymerase extension from the exposed 3′-ends of the overlapping sequences (as shown in <a href="#fig3">Figure 3</a>).
<table id="undtbl22">
<thead>
<tr>
<th>Stage</th>
<th>Temperature (°C)</th>
<th>Duration</th>
<th>Number of cycles</th>
</tr>
</thead>
<tbody>
<tr>
<td>Initial Denaturation</td>
<td>98</td>
<td>30 s</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98</td>
<td>5 s</td>
<td rowspan="3">×3</td>
</tr>
<tr>
<td>Annealing</td>
<td>65.8</td>
<td>22 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>8 s</td>
</tr>
<tr>
<td>Hold</td>
<td>4</td>
<td>∞</td>
<td>1</td>
</tr>
<tr>
<td>Denaturation</td>
<td>98</td>
<td>5 s</td>
<td rowspan="3">×28</td>
</tr>
<tr>
<td>Annealing</td>
<td>67.6</td>
<td>22 s</td>
</tr>
<tr>
<td>Extension</td>
<td>72</td>
<td>15 s</td>
</tr>
<tr>
<td>Final Extension</td>
<td>72</td>
<td>7 min</td>
<td>1</td>
</tr>
</tbody>
</table>
<p class="table-legend">The annealing temperature is equal to the salt-adjusted melting temperature of the primers/overlapping sequences, as calculated by OligoCalc (Link: <u>OligoCalc: Oligonucleotide Properties Calculator (</u><a href="http://northwestern.edu">northwestern.edu</a><u>)</u>).</p>
</li>
</ol>
</li>
</ol>
<ol start="10">
<li>Analyze the fusion PCR fragment by agarose gel electrophoresis to verify that the left and right arms have fused together and made a full-length donor DNA element.</li>
<li>Purify the donor DNA element with the GeneJET™ PCR Purification Kit and digest the donor DNA element (FD PaeI/XhoI).</li>
<li>Clone the digested donor DNA element into the linearized (FD PaeI/XhoI) and dephosphorylated (Fast AP) sample of pGE2-Spc<i>gp48</i>, resulting in a plasmid carrying both the donor DNA and a spacer-harboring mini-CRISPR array (p<i>SIRV2Mgp48his</i>). We routinely used <i>Escherichia coli</i> DH5α for cloning work and for purifying plasmids at high yields. Plasmid preparations of high yield and purity are crucial for successful electroporation into <i>S. islandicus</i> LAL14/1.</li>
<li>As a final validation to mark the end of the cloning efforts, send a sample of the purified p<i>SIRV2Mgp48his</i> for sequencing at an external partner (e.g., Eurofins Genomics, Germany).</li>
<li>The purified p<i>SIRV2Mgp48his</i> should be stored at −20°C until needed.</li>
</ol>
<h3 id="sec3.2">Electroporation of the genome editing plasmid</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 13–18 days</div>
<ol start="15">
<li>The electroporations should be performed with the Gene Pulser Xcell™ electroporation system with parameters set to 1.5 kV, 600 Ω and 25 μF (<a href="#bib3">Deng et al., 2009</a>; <a href="#bib11">Schleper et al., 1992</a>).</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Transforming plasmid DNA into <i>S. islandicus</i> LAL14/1 cells by electroporation is a difficult task that depends on the purity of the plasmid sample, the density of the competent cells, and the timing of each step in general.</div>
<ol start="16">
<li>For each constructed plasmid to be electroporated, prepare the following materials:
<ol type="a">
<li>Cast a 0.7% gelrite/SCV plate.</li>
<li>Transfer 5 mL 2<b>×</b>SCV medium and 5 mL 0.4% gelrite into separate glass tubes and preheat them in an incubator set to 78°C.</li>
<li>Transfer 950 μL of 1<b>×</b> medium salts solution to a microcentrifuge tube and preheat it to 78°C on a heat block placed next to the Gene Pulser Xcell™ electroporation system.</li>
</ol>
</li>
<li>Unpack the Gene Pulser/MicroPulser electroporation cuvettes (0.1 cm gap) and place them within reach on the same workbench as the Gene Pulser Xcell™ electroporation system.</li>
<li>From the −80°C freezer, take out the microcentrifuge tubes containing the 50-μL aliquots of electrocompetent <i>S. islandicus</i> LAL14/1 Δarrays cells (described in previous sections) and thaw the cells on ice for ∼5 min.</li>
<li>Once thawed, transfer 1 μL of a purified plasmid sample (corresponding to 300–500 ng of plasmid DNA) to the 50 μL aliquot of cells. Gently flick the microcentrifuge tube a few times to ensure proper mixing. As a negative control, make a mixture with milliQ-H<sub>2</sub>O instead of the plasmid.</li>
<li>Transfer the mixture to a Gene Pulser/MicroPulser electroporation cuvette (0.1 cm gap), place the brown cap back on, and tap the cuvette on the desk a few times to thoroughly collect the mixture in the bottom of the gap. Place the loaded cuvette in the shockpod of the GenePulser Xcell™ electroporation system.</li>
<li>Pulse the mixture and note the time of conductance.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Successful electroporations of plasmid DNA into <i>S. islandicus</i> LAL14/1 cells usually have a time of conductance around 9.6–9.8 milliseconds.</div>
<ol start="22">
<li>Immediately after electroporation, rescue the cells by transferring ≅ 600 μL of the preheated medium salts solution to the cuvette, mix briefly by pipetting up and down, and then transfer the cells and the medium salts back into the microcentrifuge tube on the heat block, set to 78°C. Allow the cells to recover and incubate for 30 min.
<ol type="a">
<li>10 min before incubation is done, place the 0.7% gelrite/SCV plates (with their lids still on) in the 78°C incubator.</li>
</ol>
</li>
<li>Add 100 μL of the electroporated cell suspension into the 5 mL of preheated 2<b>×</b>SCV medium, then add the 5 mL of preheated 0.4% gelrite, and swirl the tube thoroughly to ensure proper mixing. Gently distribute this mixture onto the preheated 0.7% gelrite/SCV plate.</li>
<li>Once the top layer has solidified (after approx. 30 min), stack the plates in sealed plastic bags and place them in an air-tight container. Incubate the plates for 7–10 days at 78°C.</li>
<li>Single colonies will have emerged on the plates (<a href="#fig4">Figure 4</a>A). They appear as sparks of an irregular shape. Use a pipette to inoculate individual colonies into 15 μL of 1<b>×</b> medium salts solution and resuspend the cells by vigorous pipetting.
<figure id="fig4"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-Fig4.jpg" alt="Fig4.jpg">
<figcaption>
<div class="figcaption-title">Figure 4. Electroporation of genome editing plasmid into <i>S. islandicus</i> LAL14/1 Δarrays and verification of transformants</div>
<p>Colonies of <i>Sulfolobus</i> transformed with plasmid (A, left panel) become visible after 10 days of incubation at 78°C. Transformations performed with milliQ-H<sub>2</sub>O instead of plasmid (A, right panel) is a useful negative control to identify possible false positives. To obtain a decent amount of cells, the transformants are inoculated, resuspended, and spotted onto a new Gelrite/SCV plate followed by incubation for another 3–5 days at 78°C (B). Any spots showing growth (e.g., Col. 1) are tested for plasmid presence by PCR (C), using Seq-F and Seq-R primers that anneal to the backbone of pGE2. As negative control (–), the PCR was performed with milliQ-H<sub>2</sub>O instead of cells.</p>
</figcaption>
</figure>
</li>
<li>Once resuspended, spot the cell suspension onto a fresh 0.7% gelrite/SCV plate (<a href="#fig4">Figure 4</a>B). Pack these plates as described previously and incubate for 3–5 days at 78°C.</li>
<li>In case of growth, scrape a small amount of the <i>Sulfolobus</i> cells from the spot with a pipette tip or inoculation loop and resuspended in 20 μL milliQ-H<sub>2</sub>O. Dilute the resuspension 50-fold with milliQ-H<sub>2</sub>O.</li>
<li>Perform a PCR to confirm the presence of the plasmid p<i>SIRV2Mgp48his</i> within the inoculated colonies using the primers Seq-F and Seq-R (<a href="#fig4">Figure 4</a>C).</li>
<li>Transfer positive spots into 20 mL of 1<b>×</b> SCV medium and incubate at 78°C for 1–2 days.</li>
<li>For long-term storage of the constructed strains, we recommend preparing glycerol stocks:
<ol type="a">
<li>Subculture transformants at a starting OD<sub>600</sub> of 0.05.</li>
<li>After incubation for 36 h or when OD<sub>600</sub> ≅ 0.6–0.7, centrifuge 10 mL of the culture at 6,300 × g for 5 min.</li>
<li>Resuspend the pellet in 670 μL of 1<b>×</b> medium salts solution and add 200 μL of sterile 87% (v/v) glycerol in a microcentrifuge tube.</li>
<li>The glycerol stocks of the cells can be stored at −80°C.</li>
</ol>
</li>
</ol>
<h3 id="sec3.3">Genome editing of SIRV2</h3>
<div class="timing">
<span class="timing-title">Timing:</span> 12 days</div>
<ol start="31">
<li>Inoculate the <i>S. islandicus</i> LAL14/1 Δarrays strain carrying the genome editing plasmid (hereafter, referred to as Δarrays/pSIRV2M<i>gp48his</i>) into 30 mL fresh SCV medium in an Erlenmeyer flask such that the initial OD<sub>600</sub> will be around 0.05. Start incubation.</li>
<li>Once the culture of Δarrays/pSIRV2M<i>gp48his</i> has reached OD<sub>600</sub> = 0.6–0.8 (after approx. 36 h), transfer a specific volume of the suspension into fresh SCV medium to a final volume of 30 mL, such that the OD<sub>600</sub> of the new culture starts at 0.05. This volume is calculated by Equation <a href="#fd1">(1)</a> as used in previous sections. Allow this culture to incubate for 1 h at 78°C.</li>
</ol>
<ol start="33">
<li>Add a specific volume of the titrated virus sample to the 30-mL culture of Δarrays/pSIRV2M<i>gp48his</i> such that the MOI of the culture will be around 0.1.
<ol type="a">
<li>In general, the volume <i>V</i><sub><i>vir</i></sub> of virus sample required to reach any given MOI is calculated according to <a href="#fd4">Equation (4)</a>:
<p id="fd4" class="math-ml"><math> <mtable> <mlabeledtr> <mtd><mtext>(Equation 4)</mtext></mtd> <mtd> <mrow> <msub> <mi>V</mi> <mrow> <mi>v</mi> <mi>i</mi> <mi>r</mi> </mrow> </msub> <mo>=</mo> <mfrac> <mrow> <mi>M</mi> <mi>O</mi> <mi>I</mi> <mo>⋅</mo> <mi>O</mi> <msub> <mi>D</mi> <mn>600</mn> </msub> <mo>⋅</mo> <mfrac> <mrow> <msup> <mn>10</mn> <mn>8</mn> </msup> <mspace></mspace> <mfrac> <mrow> <mi>C</mi> <mi>F</mi> <mi>U</mi> </mrow> <mrow> <mi>m</mi> <mi>l</mi> </mrow> </mfrac> </mrow> <mn>0.2</mn> </mfrac> <mo>⋅</mo> <mrow> <mo>(</mo> <mn>30</mn> <mspace></mspace> <mi>m</mi> <mi>l</mi> <mspace></mspace> <mo>−</mo> <mspace></mspace> <mi>Y</mi> <mo>)</mo> </mrow> </mrow> <mi>X</mi> </mfrac> </mrow> </mtd> </mlabeledtr> </mtable> </math></p>
Here, <i>OD</i><sub><i>600</i></sub> is the measured optical density of the 30-mL culture after 1 h, <i>Y</i> is the volume (in ml) that was removed from the 30-mL culture to measure the OD<sub>600</sub>, and <i>X</i> is the titer (given in PFU/mL) of the virus sample, as determined by plaque assay in a previous section.</li>
<li>In addition, prepare the following two cultures:
<ol type="i">
<li>A culture of plasmid-free Δarrays infected with the virus sample at an MOI around 0.1</li>
<li>An uninfected culture of Δarrays/pSIRV2M<i>gp48his</i> where the virus sample is replaced by a similar volume of sterile 1<b>×</b> medium salts solution.</li>
</ol>
</li>
</ol>
</li>
</ol>
<ol start="34">
<li>Incubate the suspensions for two days while monitoring the OD<sub>600</sub> regularly (<a href="#fig5">Figure 5</a>A). Around 24 and 48 h post infection (hpi), take out 1-mL samples from the culture and store them at RT.
<figure id="fig5"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-Fig5.jpg" alt="Fig5.jpg">
<figcaption>
<div class="figcaption-title">Figure 5. Genome editing of SIRV2</div>
<p>Growth curves (A) of a genome editing host (Δarrays/pΔ<i>SIRV2gp48</i>) in the presence and absence of an infecting virus. Results from three biological replicates are shown and error bars indicate the corresponding standard deviations. The double-headed arrows indicate the crucial difference in OD<sub>600</sub> that should be observed between infection of the genome editing host and infection of a CRISPR deficient (Δarrays) host. A plaque assay (B) is performed to allow for the isolation of viruses with a single genotype. The genotype of each inoculated plaque is verified by PCR (C). For illustration, we here show the PCR fragments amplified from the original virus (SIRV2M) as well as the two mutant viruses (SIRV2MΔ48 and SIRV2M<sub><i>gp48his</i></sub>) mentioned throughout this protocol. The PCRs check for the presence of the genomic region of interest containing <i>gp48</i> (C, left panel), the absence of <i>gp48</i> in the virus genome (C, middle panel), and the presence of an octahistidine (8xHis)-tag in the virus genome (C, right panel). The primer pairs are shown below each panel and are further illustrated in <a href="#fig1">Figure 1</a>.</p>
</figcaption>
</figure>
</li>
<li>Isolate the virus particles by centrifuging the collected samples at 10,000 × <i>g</i> for 6 min and transfer the supernatant into new microcentrifuge tubes.</li>
<li>Analyze the virus samples by PCR to point out the positive virus sample. The PCR mixture should contain a primer that anneals specifically to the donor DNA that has been inserted (e.g., virus his chkF) and another primer that anneals to the flanking region in the virus genome (e.g., <i>gp48</i> out chkF). This setup is illustrated in <a href="#fig1">Figure 1</a>B. Importantly, the latter primer should not anneal to p<i>SIRV2Mgp48his</i>, as this would give rise to false positives.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the band corresponding to the mutant virus is either absent or faint, repeat the genome editing protocol from step 33 by propagating an aliquot of the latest virus sample for a further two days in a fresh culture of Δarrays/pSIRV2M<i>gp48his</i>.</div>
<ol start="37">
<li>Upon confirmation, the supernatants containing the positive virus mutants should be employed in a plaque assay, following the protocol described in step 20 of the “<a href="#before-you-begin">before you begin</a>”-section.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> While the purpose of the plaque assay in the “<a href="#before-you-begin">before you begin</a>”-section was to determine the virus titre, the purpose of the present plaque assay is to isolate individual plaques with the mutant genotype.</div>
<ol start="38">
<li>Using the 1000-μL pipette, pick 10–20 individual plaques that have emerged from the 0.7% Gelzan CM/SCVU plates (<a href="#fig5">Figure 5</a>B) and inoculate them individually into 20 μL of 1<b>×</b> medium salts solution. Resuspend the plaques thoroughly by pipetting up and down a few times.</li>
<li>Repeat the PCR from step 36, this time using individual, resuspended plaques as template. Analyze the amplicons by agarose gel electrophoresis in order to identify the plaques with the desired mutant genotype (<a href="#fig5">Figure 5</a>C).</li>
<li>As a final validation of the selected virus, send the amplified genomic fragments for sequencing at an external service (e.g., Eurofins Genomics, Germany). Align the mutant virus sequence with the reference genome using pairwise sequence alignment tools (BLAST) to confirm the accuracy of the desired virus modification.</li>
</ol>
<h3 id="sec3.4">Pull-down of a genomically tagged virus protein expressed during infection</h3>
<p>As a proof-of-concept of the methods presented above, we will end this protocol by giving an example of how the constructed mutant viruses can improve research within the field of archaeal host-virus interactions. Specifically, we will describe how a virus protein such as gp48/AcrIIIB1, which has been genomically tagged with an 8xHis-tag, can be purified from an infected host culture. Compared to having the virus protein expressed from an expression vector inside the host, expressing and purifying the protein from the virus itself during infection ensures accuracy in terms of expression levels and native interaction partners.</p>
<h4 id="sec3.4.1">Virus preparation</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 5–7 days</div>
<ol start="41">
<li>Inoculate the Δarrays/pSIRV2M<i>gp48his</i> strain from the −80°C glycerol stock (described in step 30) into 30 mL fresh SCV medium in an Erlenmeyer flask. Allow the culture to incubate for two days at 78°C and 150 rpm.</li>
<li>Once the 30-mL culture has reached OD<sub>600</sub> ≅ 0.8–1.0, dilute the culture with fresh SCV medium to a final volume of 1 liter. Measure the initial OD<sub>600</sub> (should be around 0.025) and allow the 1-liter culture to incubate until OD<sub>600</sub> ≅ 0.1.</li>
<li>Prepare a 10<sup>−3</sup> dilution of the latest SIRV2M<i>gp48his</i> sample in 1<b>×</b> medium salts solution and transfer 300 μL into the 1-liter Δarrays/pSIRV2M<i>gp48his</i> culture to start the virus propagation. Incubate the host-virus suspension for 36 h while monitoring the OD<sub>600</sub> regularly every 12 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> If the OD<sub>600</sub> of the host-virus suspension is above 0.4 at 36 hpi, repeat the procedure from step 43 with a fresh 1-litre host suspension and a less diluted virus sample.</div>
<ol start="44">
<li>Isolate the SIRV2M<sub><i>gp48his</i></sub> virus particles by centrifuging the 1-liter culture at 10,000 × <i>g</i> for 6 min and transfer the supernatant into a new flask.</li>
<li>Perform a plaque assay to determine the virus titer in the infected culture supernatant, following the instructions described previously in step 20 of the “<a href="#before-you-begin">before you begin</a>”-section.</li>
</ol>
<h4 id="sec3.4.2">Infection and harvesting of cells</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 6 days</div>
<ol start="46">
<li>Inoculate the <i>S. islandicus</i> LAL14/1 Δ<i>Cas6(I-D)</i> and <i>S. islandicus</i> LAL14/1 Δ<i>Cas6(I-D)</i>/p<i>Cmr6α-HA</i> host strains from their respective −80°C glycerol stocks into individual Erlenmeyer flasks each filled with 20 mL fresh SCV(U) medium. Allow the two cultures to incubate for two days.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> Be aware that <i>S. islandicus</i> LAL14/1 Δ<i>Cas6(I-D)</i> should be grown in SCVU medium whereas <i>S. islandicus</i> LAL14/1 Δ<i>Cas6(I-D)</i>/p<i>Cmr6α-HA</i> should be grown in SCV medium due to its plasmid.</div>
<ol start="47">
<li>Once the 20-mL cultures have reached OD<sub>600</sub> ≅ 0.8–1.0, transfer each culture into fresh, pre-heated medium to final volumes of 120 mL. Incubate the cultures for approximately 24 h.</li>
<li>Split each 120-mL culture in two and transfer each half into fresh, pre-heated medium to a final volume of 1 liter (i.e., two liters of culture is started per strain in this step). Allow these cultures to incubate for ∼20 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> When incubating large batches of an <i>S. islandicus</i> culture, we recommend dispersing the culture volume into 1-litre aliquots and growing each aliquot in parallel in 2-litre Duran® bottles which can fit in an incubator large enough to contain and heat all flasks to 78°C simultaneously. In our case, we have used two New Brunswick<sup>TM</sup> INNOVA 44 incubator shakers.</div>
<ol start="49">
<li>When the four 1-liter cultures have reached OD<sub>600</sub> ≅ 0.4, transfer 150-mL of each culture into new 2-liter Duran® bottles containing 850 mL fresh, pre-heated medium. In total, 12 liters will be made for each host strain. The initial OD<sub>600</sub> of these cultures should be around 0.06. Allow all parallel cultures to incubate in the same incubator overnight (approx. 12–14 h).</li>
<li>Once the cultures have reached early exponential growth phase (OD<sub>600</sub> = 0.2–0.3), infect each culture with a specific volume of the SIRV2M<sub><i>gp48his</i></sub> sample (from step 44) to arrive at a MOI = 1. This volume is calculated by <a href="#fd4">Equation (4)</a>, remembering to replace “30 mL” with “1000 mL” in the formula. Allow the infected cultures to continue their incubation for 3 h.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The incubation time depends on the expression phase of the target protein. Purification of target proteins corresponding to early expressed genes should be harvested 2–3 hpi, those corresponding to middle/late expressed genes should be harvested at a later time point, but prior to cell lysis (8 hpi).</div>
<ol start="51">
<li>Remove the bottles from the incubator and allow the cultures to cool down to RT.</li>
<li>Once cooled, harvest the cells from the 2<b>×</b>12 liters of culture by centrifugation at 6,300 × <i>g</i> and 16°C for 10 min. We recommend using the Sorvall LYNX 6000 superspeed centrifuge.
<ol type="a">
<li>Discard the supernatants and weigh the cell pellets.</li>
<li>Resuspend the cell pellets in cool (4°C) lysis buffer, using 5 mL of buffer per gram of pellet. This step should be performed on ice. Resuspend by pipetting, not by vortexing.</li>
</ol>
</li>
<li>Transfer the resuspended cell pellets into Falcon tubes and store at −80°C until needed.</li>
</ol>
<h4 id="sec3.4.3">Protein purification/protein pull-down</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 2 days</div>
<ol start="54">
<li>Thaw the resuspended cell pellets by placing the Falcon tubes in a water bath for ∼ 30 min.</li>
<li>Lyse the archaeal cells by the following steps:
<ol type="a">
<li>First, a round of sonication (30 cycles of 3-s pulses intervened by 3-s pauses).</li>
<li>Then, two rounds of homogenization with a French press equilibrated with the lysis buffer.</li>
<li>Finally, end with another round of sonication, using the same program as above.</li>
</ol>
</li>
</ol>
<div class="critical">
<span class="critical-title">Critical:</span> To avoid contamination between the two host strain suspensions, rinse the sonicator rod with 70% ethanol and run some 20% ethanol through the French press several times between individual samples. This clean-up should also be done before and after the instruments are used.</div>
<ol start="56">
<li>Remove all cell debris from the lysed cell samples by centrifugation at 16,000 × <i>g</i> and 4°C for 45 min. The precipitate will be dark brown/gray, while the supernatant appears beige. Move both supernatants to new Falcon tubes and repeat the centrifugation until the precipitates become smaller than a fingernail and the supernatants become slightly transparent.</li>
<li>Meanwhile, equilibrate 2<b>×</b>2 mL of nickel nitrilotriacetic acid (Ni-NTA) agarose resin (QIAGEN):
<ol type="a">
<li>Spin down the agarose resins at 800 × <i>g</i> for 3 min in a swinging-bucket centrifuge.</li>
<li>Discard the storage buffer and resuspend each resin bed in 10 mL of lysis buffer.</li>
<li>Repeat substeps 57.a and 57.b at least five times.</li>
</ol>
</li>
<li>Filter the supernatants (from step 56) through sterile, non-pyrogenic, hydrophilic filters (0.45 μm) and mix with the equilibrated Ni-NTA agarose resins (from step 57) in 50-mL Falcon tubes. Allow both mixtures to incubate overnight (approx. 14 h) at 4°C on a rotary wheel set to 10 rpm.</li>
</ol>
<div class="note">
<span class="note-title">Note:</span> The slow rotation speed is applied to minimise the generation of foam, which is a potential source of protein loss in this protocol. Furthermore, use the two Falcon tubes as counterweights for each other to stabilise the rotor speed.</div>
<ol start="59">
<li>Spin the protein-bound Ni-NTA resins at 800 × <i>g</i> for 3 min in a swinging-bucket centrifuge.</li>
<li>Discard the supernatants and wash the protein-bound Ni-NTA resins:
<ol type="a">
<li>Add 10 mL of lysis buffer to each resin bed and incubate for 5 min on the rotary wheel set to 10 rpm to ensure proper mixing.</li>
<li>Spin down the agarose resins at 800 × <i>g</i> for 3 min in a swinging-bucket centrifuge and remove the supernatant.</li>
<li>Repeat substeps 60.a and 60.b at least five times to remove all unbound proteins.</li>
<li>After the last centrifugation, resuspend the resin beds in 0.8–1.0 mL lysis buffer each and transfer the mixtures from the Falcon tubes to microcentrifuge tubes.</li>
<li>Spin down the microcentrifuge tubes at 800 × <i>g</i> for 3 min and discard the supernatant.</li>
</ol>
</li>
<li>Elute the proteins that are bound to the Ni-NTA resins:
<ol type="a">
<li>Add an equal volume of elution buffer to the protein-bound Ni-NTA agarose resin beds and incubate the tubes for 5 min on a rotary wheel set to 10 rpm.</li>
<li>Spin down the microcentrifuge tube with the elution mixture at 800 × <i>g</i> for 3 min and transfer the supernatant to a new microcentrifuge tube. Refill the microcentrifuge with a fresh bed volume of elution buffer.</li>
<li>Repeat substeps 61.a to 61.b at least five times to extract all proteins from the resin.</li>
</ol>
</li>
<li>Combine the collected eluates into one volume and concentrate the proteins via spin filtering, e.g., by using the Pierce™ Protein Concentrator PES (10K MWCO). Briefly, the protein solutions are added to separate spin columns and centrifuged at 10,000 × <i>g</i> in rounds of 4 min until only 100 μL is left in each spin column. Transfer the retentates into new microcentrifuge tubes and store them at 4°C until needed.</li>
<li>Resolve the concentrated protein samples by SDS-PAGE using the mini-PROTEAN® Electrophoresis system (Bio-Rad, USA).
<ol type="a">
<li>Prior to gel loading, mix 15 μL of each concentrated protein sample with 5 μL of 4<b>×</b> SDS loading buffer, incubate for 5 min on a heat block set to 95°C, and collect the mixtures at the bottom of the microcentrifuge tube by a brief spin down.</li>
<li>The samples were separated on a 12% SDS-PAGE gel at 16 mA for up to 2 h with the voltage set to a maximum of 120 V.</li>
<li>After the run, remove the gel from the chamber and stain it for 1 h with a Coomassie Brilliant Blue (R-250)-based staining solution.</li>
<li>Destain the gels repeatedly in destaining solution until the protein bands can be distinguished clearly from the background (as shown best in <a href="#fig6">Figure 6</a>A).
<figure id="fig6"><img src="https://prod-shared-star-protocols.s3.amazonaws.com/protocols/1031-Fig6.jpg" alt="Fig6.jpg">
<figcaption>
<div class="figcaption-title">Figure 6. SDS-PAGE and Western Blots showing virus protein expressed and purified during infection</div>
<p>The His-tagged copy of gp48/AcrIIIB1 was expressed from the virus SIRV2M<sub><i>gp48his</i></sub> during infection and subsequently used as bait in a protein pull-down. The proteins in the pull-down samples were separated on a 12% SDS-polyacrylamide gel (left panel). The migrations and relative intensities of the bands that appear in the gel correspond well to the size and stoichiometry of the six subunits of the Cmr-α complex (indicated to the right of the gel). The concentrated pull-down samples were separated on a 12% SDS-polyacrylamide gel, the presence of human influenza hemagglutinin (HA)-tag on the target protein (Cmr6α) was detected using the primary and secondary antibodies, HA Tag Monoclonal Antibody (2–2.2.14) and Goat anti-Mouse IgG (H+L) Secondary Antibody, HRP respectively (right panel).</p>
</figcaption>
</figure>
</li>
</ol>
</li>
</ol>
<h4 id="sec3.4.4">Western blot analysis</h4>
<div class="timing">
<span class="timing-title">Timing:</span> 1–2 days</div>
<ol start="64">
<li>Resolve the concentrated protein samples on another 12% SDS-PAGE gel, this time without staining and destaining the gel after the run. Instead, wash the separation gel with distilled H<sub>2</sub>O.</li>
<li>Transfer the proteins from the SDS-PAGE gel to a Nitrocellulose membrane by Semi-Dry transfer:
<ol type="a">
<li>Prepare the blotting papers and the Nitrocellulose membrane (Amersham<sup>TM</sup> Protran<sup>TM</sup> 0.45 μm NC) by soaking them individually in the transfer buffer.</li>
<li>The transfer assembly should be arranged top-to-bottom in the following order: cathode, blotting paper, SDS-polyacrylamide gel, nitrocellulose membrane, blotting paper, anode.
<ol type="i">
<li>Remember to remove any air bubbles during this assembly by gently rolling a cylindrical object over the setup before placing it between the cathode and anode.</li>
</ol>
</li>
<li>Connect the transfer assembly to the power source and perform the protein transfer at 70 mA for 75 min. The voltage should be set to any value &gt;20 V to prevent the voltage from being the limiting factor during the transfer.</li>
</ol>
</li>
<li>Once the transfer is complete, carefully separate the nitrocellulose membrane from the gel and the blotting papers, and incubate the membrane in wash buffer (WB) for 2 min.</li>
<li>Prepare blocking buffer by dissolving 1.5 grams of skim milk in the WB.</li>
<li>Replace the WB with 30 mL of blocking buffer (the exact volume depends on the size of the membrane incubation tray) and incubate for 30 min at RT on a shaking platform.</li>
</ol>
<div class="pause-point">
<span class="pause-point-title">Pause point:</span> The membrane soaked in the blocking buffer can be placed overnight (approx. 14 hours) at 4°C (no shaking).</div>
<ol start="69">
<li>Wash the nitrocellulose membrane three times with the WB for 3 min each. Incubate at RT on a shaking platform.
<ol type="a">
<li>If the membrane was incubated at 4°C overnight, it should first be allowed to incubate at RT for 30 min on a shaking platform before this washing step.</li>
</ol>
</li>
<li>Prepare the primary antibody solution by mixing “HA Tag Monoclonal Antibody (2–2.2.14)” and WB at 1:10000 (v/v) final ratio.</li>
<li>Discard the WB and incubate the membrane in the primary antibody solution for 1–4 h at RT on a shaking platform.</li>
<li>Repeat step 69.</li>
<li>Prepare the secondary antibody solution by mixing “Goat anti-Mouse IgG (H+L) Secondary Antibody, HRP”, and WB at 1:3000 (v/v) final ratio.</li>
<li>Discard the WB and incubate the membrane in the secondary antibody solution for 1 h at RT on a shaking platform.</li>
<li>Repeat step 69. After the final wash, mix equal volumes of Stable Peroxide Solution and the Luminol/Enhancer Solution from the SuperSignal<sup>TM</sup> West Pico PLUS Chemiluminescent Substrate Kit (Substrate mixture).</li>
<li>After the final wash, remove the WB as much as possible without drying the membrane and transfer the Nitrocellulose membrane into a dry container. Spread the substrate mixture uniformly across the membrane and incubate for 2–3 min.</li>
<li>Place the membrane between two transparent plastic sheets, remove any air bubbles, and bind it within a Kodak BioMax Cassette.</li>
<li>Signal development:
<ol type="a">
<li>In a darkroom, place an X-ray film (AGFA CP-BU, Kruuse) directly on top of the plastic-wrapped membrane.</li>
<li>After 30–60 s, develop the film using the developer and fixer solutions. The resulting image is supposed to look like that shown in <a href="#fig6">Figure 6</a>B.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="expected-outcomes">Expected outcomes</h2>
<p>During the initial hours post infection, the virus-infected <i>Sulfolobus</i> host (carrying the genome editing plasmid) should show similar growth as the uninfected culture. A steep growth retardation should be seen 24 h post infection in the infected culture, indicating the propagation of a virus with the ability to overcome CRISPR-Cas immunity (<a href="#fig5">Figure 5</a>A). In comparison, the <i>Sulfolobus</i> host without genome editing plasmid should show growth retardation immediately. Mutant viruses with the desired modifications (deletions or insertions) should be observed in the liquid culture and a pure mutant post plaque assay (<a href="#fig5">Figures 5</a>B and 5C).</p>
<p>Upon Immobilized Metal Affinity Chromatography purification, virus protein should be purified along with potential host or viral interacting partners. The SDS-polyacrylamide gel (<a href="#fig6">Figure 6</a>, left panel) shows the proteins present in the pull-down sample with AcrIIIB1Chis as bait. This result suggests that our target protein interacts with the host CRISPR-Cas effector complex, as we see protein bands from the pull-down sample with sizes corresponding to the six Cmr subunits of Cmr complex. The amount of protein purified is also an indication of its endogenous levels during viral infection.</p>
<p>The western blot analysis shows a positive signal around 25 kDa corresponding to Cmr6<i>α</i>-HA in the purification from the SIRV2M<sub><i>gp48his</i></sub>-infected Δ<i>Cas6(I-D)</i>/p<i>Cmr6α-HA</i> culture (<a href="#fig6">Figure 6</a>, right panel). Although the concentrated elution mixture from SIRV2M<sub><i>gp48his</i></sub>-infected Δ<i>Cas6(I-D)</i> purification shows purified gp48/AcrIIIB1 and bands corresponding to the Cmr-α complex, the Cmr6α from this purification does not carry the HA tag and hence is not recognized by the anti-HA antibody (<a href="#fig6">Figure 6</a>, right panel). The remaining proteins from the gp48 pulldown were previously identified, by mass spectrometry, to be components of the CRISPR-Cas type III-B system (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>).</p>
</section>
<section>
<h2 id="limitations">Limitations</h2>
<p>The potential toxicity of the donor DNA in <i>Sulfolobus</i> is an important limitation, making some genes inaccessible to genome editing. Viral proteins, even in truncated form, could induce cell death by inhibiting the functions of crucial host proteins. Furthermore, donor DNA encoding truncated proteins along with strong native promoters could result in overexpression, utilizing machinery necessary for proper functioning of the host.</p>
<p>Viral genes with very low endogenous expression are not ideal for purification under native conditions. In such cases, plasmid-based expression is a better choice.</p>
<p>In some cases, due to the absence of an appropriate PAM sequence around the termini, both protein termini could be inaccessible to tag insertions, making affinity purification of the viral protein impossible.</p>
<p>Currently, this genome editing method has been demonstrated in the lytic virus SIRV2 and should be applicable to all members of the order Ligamenvirales. CRISPR-Cas based targeting of temperate archaeal viruses has not been well studied, hence the technique described here could be ineffective among temperate phages of archaea.</p>
<p>Viral genome editing requires viruses susceptible to targeting by at least one host encoded CRISPR-Cas system. Hence, homologous of known Acrs are to be deleted prior to further genome editing. Early transcribed genes are sometimes susceptible to type III targeting despite Acrs being encoded on the virus (<a href="#bib2">Bhoobalan-Chitty et al., 2019</a>). In case of viruses which encode unidentified Acrs, genome editing could be achieved by targeting early expressed genes with type III CRISPR-Cas systems.</p>
</section>
<section>
<h2 id="troubleshooting">Troubleshooting</h2>
<h3 id="sec6.1">Problem 1</h3>
<p>Lack of colonies after electroporation of <i>S. islandicus</i> (step 25).</p>
<h3 id="sec6.2">Potential solutions</h3>
<ol>
<li>Check the efficiency of the competent cells with an empty vector. If the electroporation efficiency is low it will be necessary to prepare the electrocompetent cells again, possibly increasing the number of wash steps with 20 mM Sucrose.</li>
<li>The region neighboring the target site carries other genetic elements with potential toxicity to host when encoded on a plasmid (e.g., promoter regions). To solve this, smaller homologous arms/donor DNA regions could be selected to reduce toxicity. The size of the homologous arms could be as short as 200 bps.</li>
<li>Make sure to use uracil-deprived/-less medium after the electroporation, as the selection of positive transformants relies on the uracil-synthesizing genes <i>pyrEF</i> to be absent from the chromosome and present in the backbone of the genome editing plasmids.</li>
</ol>
<p>For additional troubleshooting please refer to (<a href="#bib12">Zhao et al., 2021</a>).</p>
<h3 id="sec6.3">Problem 2</h3>
<p>Desired viral mutation is not observed (step 36):</p>
<ol>
<li>Deletion not observed.</li>
<li>Nonsensical tag inserted.</li>
</ol>
<h3 id="sec6.4">Potential solutions</h3>
<p>Use alternative or additional spacers to increase targeting efficiency. Increase the length of the homologous arms/donor DNA to increase efficiency of recombination. In case of deletion of essential genes, we recommend using a mixture of deletion strain along with a strain complementing the target gene (Zhang et al., Unpublished).</p>
<p>Nonsensical tags are observed when the insertion sequence interferes with the normal expression or protein function. In such case, either use the other protein terminus for tag insertion or use an alternative protein purification tag.</p>
<h3 id="sec6.5">Problem 3</h3>
<p>Unable to separate original and mutant virus with plaque assay (step 39).</p>
<h3 id="sec6.6">Potential solution</h3>
<p>Perform several rounds of transfer of mutant and original virus mixture in liquid culture prior to the final purification with plaque assay. Multiple spacers on the plasmid or alternative protospacer target site will increase the effectiveness of targeting the original virus.</p>
<h3 id="sec6.7">Problem 4</h3>
<p>No protein visible in the SDS-polyacrylamide gel after purification (step 63).</p>
<h3 id="sec6.8">Potential solution</h3>
<ol>
<li>The terminal at which the tag was added into the protein could affect the expression or the stability of the protein. Furthermore, the tag could be inaccessible to downstream protein purification methods. Compare infectivity of original and mutant virus. If there is a difference in infectivity, use alternative tags or include linkers between protein termini and tag to decrease its influence on protein structure and stability. Alternatively, change the location of the tag to the opposite terminal of the protein.</li>
<li>Low protein expression from the virus genome:
<ol type="a">
<li>Increase MOI and culture volume.</li>
<li>Replace native virus promoter with alternative viral promoter with a proven higher expression level.</li>
</ol>
</li>
</ol>
</section>
<section>
<h2 id="references">References</h2>
<p id="bib1">Athukoralage, J.S., McMahon, S.A., Zhang, C., Gruschow, S., Graham, S., Krupovic, M., Whitaker, R.J., Gloster, T.M., and White, M.F. (2020). An anti-CRISPR viral ring nuclease subverts type III CRISPR immunity. Nature <i>577</i>, 572-575. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref1">View at publisher</a></p>
<p id="bib2">Bhoobalan-Chitty, Y., Johansen, T.B., Di Cianni, N., and Peng, X. (2019). Inhibition of type III CRISPR-cas immunity by an archaeal virus-encoded anti-CRISPR protein. Cell <i>179</i>, 448-458 e411. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref2">View at publisher</a></p>
<p id="bib3">Deng, L., Zhu, H., Chen, Z., Liang, Y.X., and She, Q. (2009). Unmarked gene deletion and host-vector system for the hyperthermophilic crenarchaeon Sulfolobus islandicus. Extremophiles <i>13</i>, 735-746. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref3">View at publisher</a></p>
<p id="bib4">Gudbergsdottir, S., Deng, L., Chen, Z., Jensen, J.V., Jensen, L.R., She, Q., and Garrett, R.A. (2011). Dynamic properties of the Sulfolobus CRISPR/Cas and CRISPR/Cmr systems when challenged with vector-borne viral and plasmid genes and protospacers. Mol. Microbiol. <i>79</i>, 35-49. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref4">View at publisher</a></p>
<p id="bib5">He, F., Bhoobalan-Chitty, Y., Van, L.B., Kjeldsen, A.L., Dedola, M., Makarova, K.S., Koonin, E.V., Brodersen, D.E., and Peng, X. (2018). Anti-CRISPR proteins encoded by archaeal lytic viruses inhibit subtype I-D immunity. Nat. Microbiol. <i>3</i>, 461-469. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref5">View at publisher</a></p>
<p id="bib6">He, F., Vestergaard, G., Peng, W., She, Q., and Peng, X. (2017). CRISPR-Cas type I-A Cascade complex couples viral infection surveillance to host transcriptional regulation in the dependence of Csa3b. Nucleic Acids Res. <i>45</i>, 1902-1913. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref6">View at publisher</a></p>
<p id="bib7">Jaubert, C., Danioux, C., Oberto, J., Cortez, D., Bize, A., Krupovic, M., She, Q., Forterre, P., Prangishvili, D., and Sezonov, G. (2013). Genomics and genetics of Sulfolobus islandicus LAL14/1, a model hyperthermophilic archaeon. Open Biol. <i>3</i>, 130010. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref7">View at publisher</a></p>
<p id="bib8">Lin, J., Fuglsang, A., Kjeldsen, A.L., Sun, K., Bhoobalan-Chitty, Y., and Peng, X. (2020). DNA targeting by subtype I-D CRISPR-Cas shows type I and type III features. Nucleic Acids Res. <i>48</i>, 10470-10478. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref8">View at publisher</a></p>
<p id="bib9">Mayo-Munoz, D., He, F., Jorgensen, J.B., Madsen, P.K., Bhoobalan-Chitty, Y., and Peng, X. (2018). Anti-CRISPR-based and CRISPR-based genome editing of sulfolobus islandicus rod-shaped Virus 2. Viruses <i>10</i>, 695. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref9">View at publisher</a></p>
<p id="bib10">Peng, W., Feng, M., Feng, X., Liang, Y.X., and She, Q. (2015). An archaeal CRISPR type III-B system exhibiting distinctive RNA targeting features and mediating dual RNA and DNA interference. Nucleic Acids Res. <i>43</i>, 406-417. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref10">View at publisher</a></p>
<p id="bib11">Schleper, C., Kubo, K., and Zillig, W. (1992). The particle SSV1 from the extremely thermophilic archaeon Sulfolobus is a virus: demonstration of infectivity and of transfection with viral DNA. Proc. Natl. Acad. Sci. U S A <i>89</i>, 7645-7649. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref11">View at publisher</a></p>
<p id="bib12">Zhao, R., Yang, Y., Yang, K., and Han, W. (2021). Expression, purification, and characterization of a membrane-associated cyclic oligo-adenylate degrader from Sulfolobus islandicus. STAR Protoc. <i>2</i>, 100299. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref12">View at publisher</a></p>
<p id="bib13">Zillig, W., Kletzin, A., Schleper, C., Holz, I., Janekovic, D., Hain, J., Lanzendörfer, M., and Kristjansson, J.K. (1993). Screening for sulfolobales, their plasmids and their viruses in icelandic solfataras. Syst. Appl. Microbiol. <i>16</i>, 609-628. <a class="external-link" href="http://refhub.elsevier.com/S2666-1667(21)00497-4/sref13">View at publisher</a></p>
</section>
<section>
<h2 id="article-info">Article info</h2>
<h3>Resource availability</h3>
<h4>Lead contact</h4>
<p>Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Yuvaraj Bhoobalan-Chitty (<a href="mailto:yuvarajb@bio.ku.dk">yuvarajb@bio.ku.dk</a>).</p>
<h4>Materials availability</h4>
<p>This study did not generate new unique reagents.</p>
<h4>Data and code availability</h4>
<p>This study did not generate any unique datasets or code.</p>
<h3>Acknowledgments</h3>
<p>This work was supported by the <a href="https://doi.org/10.13039/100008392">Danish Council for Independent Research/Technology and Production</a> DFF-7017-00060 (to X.P.), the <a href="https://doi.org/10.13039/501100009708">Novo Nordisk Fonden</a>/Hallas Møller Ascending Investigator Grant NNF17OC0031154 (to X.P.), and the <a href="https://doi.org/10.13039/501100009708">Novo Nordisk Fonden</a> Postdoctoral Fellowship in Bioscience and Basic Biomedicine Grant NNF21OC0067491 (to Y.B.-C.). We thank all the past and present members of the Danish Archaea Centre for their input and contributions and especially Safir Hien Phan for her guidance with preparation of protocols relating to the <i>Sulfolobus</i> medium.</p>
<h3>Author contributions</h3>
<p>Y.B.-C. and X.P. conceived the project and designed the experiments, Y.B.-C. and L.A. performed the experiments; L.A. and Y.B.-C. wrote the manuscript together with X.P. All authors read and approved the final manuscript.</p>
<h3>Declaration of interests</h3>
<p>The authors declare no competing interests.</p>
</section>
</article>