<body>
            <h1>Rapid Multiplexed Reduced Representation Bisulfite Sequencing Library Prep (rRRBS)</h1>
            <p><strong>Authors:</strong> Lisa-Marie  Legault, Donovan  Chan, Serge  McGraw</p>
            <h2>Abstract</h2>
            <p>DNA methylation is a common mechanism of epigenetic regulation involved in transcriptional modulation and genome stability. With the evolution of next-generation sequencing technologies, establishing quantitative genome-wide DNA methylation profiles is becoming routine in many laboratories. However, many of these approaches take several days to accomplish and use subjective PCR methods to amplify sequencing libraries, which can induce amplification bias. Here we propose a rapid Reduced Representation Bisulfite Sequencing (rRRBS) protocol to minimize PCR amplification bias and reduce total time of multiplexed library construction. In this modified approach, the precise quantification of the final library amplification step is accomplished and monitored by qPCR, instead of using standard PCR and gel electrophoresis, to determine the appropriate number of cycles to perform. The main advantages of this rRRBS method are: i) Reduced amount of amplification enzyme used for library prep, ii) Reduced number of PCR cycles resulting in less PCR amplification bias, and iii) Preparation of quality multiplexed rRRBS libraries in only ~2 days.</p>
            <p><strong>Keywords:</strong> DNA methylation, Reduced representation bisulfite sequencing, Epigenetics, Next-generation sequencing, Multiplex libraries</p>
            <h2>Background</h2><p>With the development of high-throughput sequencing technologies, evaluation of genome-wide DNA methylation profiles is more accessible than before. For the human genome, various commercial targeted methyl capture sequencing panels and methylation arrays are available and allow high-throughput, quantitative interrogation of methylation sites at single-nucleotide resolution. However, most of these commercial tools are not offered for other model species. Thus, cost-effective sequencing-based methods for genome-wide methylation analysis, not targeting species-specific genomic sequences, are still highly relevant and valuable especially when profiling large cohorts of samples. One of these techniques is Reduced Representation Bisulfite Sequencing (RRBS). This method enables you to quantitatively investigate the DNA methylation profile of ~10% of the overall CpGs, clustered in fragments composing approximately 1% of the genome, with a preference for CpG rich regions (<em>e.g.</em>, CpG islands, promoters) (Meissner <em>et al.</em>, 2005 and 2008; Smith <em>et al.</em>, 2009; Gu <em>et al.</em>, 2011). <br/><br/>Here, we present a rapid RRBS (rRRBS) protocol (PichÃ© <em>et al.</em>, 2018) that significantly decreases hands-on time, allowing to perform the entire protocol in ~2 days compared to the 7 days usually needed (Gu <em>et al.</em>, 2011; Boyle <em>et al.</em>, 2012). Importantly, by using a qPCR amplification step, we now minimize amplification bias during multiplexed library construction (Figure 1). Also, compared to the original and other RRBS protocols (Gu <em>et al.</em>, 2011; Boyle <em>et al.</em>, 2012; McGraw <em>et al.</em>, 2015) our modified protocol reduces the quantity of reagent and enzyme used as we no longer test for the optimal number of cycles needed before the final library amplification step. This method is suitable for DNA methylation studies from cells or tissues from any common research species (<em>e.g.</em>, human, mouse, rat, zebrafish).<br/><br/><img alt="" class="layerphoto" height="135" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217180944_9613.jpg" width="180"/><br/><strong>Figure 1. Graphical summary of the principal step of the rRRBS procedure</strong></p>
        </body>

<p class="pptt" id="biaoti37789">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips</li><li>1.5 ml DNase/RNase-free microtubes</li><li>0.2 ml DNase/RNase-free PCR strip tubes with individual caps</li><li>QIAamp DNA micro or mini kit (QIAGEN, catalog numbers: 56304 or 51306)</li><li>QuBit dsDNA BR assay kit (Thermo Fisher Scientific, catalog number: Q32853)</li><li>Unmethylated Î» phage DNA (Promega, catalog number: D1521)</li><li>MspI 100 U/Î¼l (New England Biolabs, catalog number: R0106M)</li><li>Klenow fragment 5,000 U/ml (New England Biolabs, catalog number: M0212L)<br/></li><li>dNTP set (4 x 100 mM) (Thermo Fisher Scientific, catalog number: 10297018)</li><li>AMPure XP magnetic beads in PEG solution (Beckman Coulter, catalog number: A63881)</li><li>NEBNext<sup>Â®</sup> Multiplex Oligos for Illumina<sup>Â®</sup> (Methylated Adapter, Index Primers Set 1) (New England Biolabs, catalog number: E7535L)</li><li>T4 DNA ligase 2,000,000 U/ml (New England Biolabs, catalog number: M0202M)</li><li>EpiTect Fast bisulfite kit (QIAGEN, catalog number: 59826)</li><li>SYBR Green, 10,000x (Thermo Fisher Scientific, Invitrogen, catalog number: S7563)</li><li>KAPA HiFi Uracil+ 2x mastermix (Roche, catalog number: KK2802)</li><li>Bioanalyzer High Sensitivity DNA Assay (Agilent, catalog number: 5067-4626)</li><li>DNase/RNase free water</li><li>EB buffer (QIAGEN, catalog number: 19086)</li><li>NEB Buffer 2 (New England Biolabs, catalog number: B7002S)</li><li>Ethanol 100%<br/></li></ol><p class="pptt" id="biaoti37790">### Equipment</p><ol class="npwtwok"><li>P10, P20 pipets</li><li>QuBit fluorometric device (Thermo Fisher Scientific, catalog number: Q33226)</li><li>Vortex</li><li>Mini-centrifuge</li><li>Thermocycler</li><li>Heating block</li><li>DynaMag<sup>TM</sup>-96 Side Magnet (Thermo Fisher Scientific, catalog number: 12331D)</li><li>DynaMag<sup>TM</sup>-2 Magnet (Thermo Fisher Scientific, catalog number: 12321D)</li><li>qPCR machine</li><li>qPCR plates2100 </li><li>Bioanalyzer instrument (Agilent, catalog number: G2939BA)</li><li>Illumina sequencing apparatus</li><li>-20 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti37791">### Procedure</p><ol class="npwone"><li>DNA extraction and quantification (estimated time: overnight lysis + 2 h)<br/>DNA is extracted from tissues or cultured cells using QIAGEN QIAamp DNA micro or mini kit, depending on the size of the original sample, following manufacturerâs recommendations. The Qubit fluorometric device with dsDNA BR assay kit is then used to quantify DNA samples. To ensure efficiency of bisulfite conversion (Procedure F), 0.5% (2.5 ng) of unmethylated Î» phage DNA can be added as a spike-in with 500 ng of sample DNA to provide an internal control. <br/><br/></li><li>Enzymatic digestion (estimated time: overnight)<br/><ol class="npwtwo"><li>For each sample, in a 0.2 ml PCR tube from a strip tube with individual caps, dilute 500 ng of DNA in DNase/RNase-free water for a final volume of 26.8 Î¼l. <br/><em>Note: Use 0.2 ml PCR strip tubes with individual caps at every step to avoid contamination of your samples.</em> </li><li>Prepare a master mix of MspI enzyme by mixing 3 Î¼l of 10x NEB Buffer 2 and 0.2 Î¼l of 100 U/Î¼l MspI enzyme per reaction (Table 1). <br/><br/><strong>Table 1. Preparation of MspI digestion mix</strong><br/><img alt="" class="layerphoto" height="93" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217183830_2149.jpg" width="500"/><br/><br/></li><li>Add 3.2 Î¼l of the master mix to each sample tube. Mix by pulse vortexing and briefly centrifuge (<em>i.e.</em>, spin down) reaction. </li><li>Incubate the digestion reaction in a thermocycler overnight at 37 Â°C, with lid set at 42 Â°C.</li></ol><br/></li><li>Gap filling and A-tailing (estimated time: 1 h)<br/>Gap filling and A-tailing are achieved using Klenow fragment and a dNTPs mixture (10 mM dATP, 1 mM dCTP, 1 mM dGTP).<br/><ol class="npwtwo"><li>Prepare the dNTPs mixture by mixing 176 Î¼l of DNase/RNase-free water, 20 Î¼l of dATP (100 mM), 2 Î¼l dCTP (100 mM) and 2 Î¼l dGTP (100 mM). Aliquots of dNTPs can be stored at -20 Â°C for future rRRBS experiments.</li><li>For each sample, prepare a master mix by combining 1 Î¼l of the dNTP mixture and 1 Î¼l of Klenow fragment enzyme (Table 2). <br/><br/><strong>Table 2. Preparation of gap filling and A-tailing mix</strong><br/><img alt="" class="layerphoto" height="93" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217183849_9958.jpg" width="500"/><br/><br/></li><li>Without deactivating MspI digestion, add 2 Î¼l of the master mix in each sample tube. Mix by pulse vortexing and briefly centrifuge. </li><li>Place tubes back in the thermocycler for a cycle of 20 min at 30 Â°C and 20 min at 37 Â°C. The lid should not be heated. </li></ol><br/></li><li>Bead clean-up (estimated time: 1 h 30 min)<br/>After the gap filling and A-tailing step, AMPure XP magnetic beads in PEG solution are used to clean up the reactions from previous reagents. To improve resuspension of the beads later on, warm up EB buffer in a heating block at 55 Â°C. <br/><em>Note: Warming up the EB buffer at 55 Â°C will facilitate the resuspension of the beads, especially if they are over dried.</em><br/><ol class="npwtwo"><li>Add 64 Î¼l of beads (2x ratio of beads [Note 1]) to each sample. Mix by slowly pipetting up and down at least 10 times. </li><li>Incubate mixture for 30 min at room temperature.</li><li>Add 0.2 ml tubes on a DynaMag<sup>TM</sup>-96 Side magnet for 5 min. At this stage, DNA is bound to the AMPure XP magnetic beads. </li><li>Without disturbing the beads, carefully remove and discard the supernatant.</li><li>Wash the beads by adding 100 Î¼l of 80% freshly prepared ethanol. Incubate for 5 min and remove EtOH without disturbing the beads.</li><li>Repeat Step D5 for a second wash.</li><li>Remove the residual EtOH with a P10 pipet.</li><li>Dry the beads for about 5 min. Do not over dry (<em>i.e.</em>, bead ring appears cracked if over dried) as this will significantly decrease elution efficiency (Note 2). </li><li>Remove the tubes from the magnetic rack and resuspend the beads in 26 Î¼l warmed EB buffer by repetitive pipetting, and incubate for 2 min at room temperature (not on the rack). </li><li>Place the tubes back in the magnetic rack and incubate for at least 5 min. Make sure all the beads are bound to the wall of the tube.</li><li>Carefully remove the supernatant without taking any beads and transfer into a new PCR tube.</li></ol><br/></li><li>Adapter ligation and bead clean-up (estimated time: 2 h)<br/>The next stage is to ligate the adapters for the future qPCR amplification and index fixation steps. Here, the NEBNext<sup>Â®</sup> Multiplex Oligos for Illumina<sup>Â®</sup> (Methylated Adapter, Index Primers Set 1) kit is used in combination with the T4 DNA ligase.<br/><ol class="npwtwo"><li>For each reaction, prepare a master mix of 8.5 Î¼l of DNase/RNase-free water, 5 Î¼l of T4 DNA ligase enzyme 10x buffer, 10 Î¼l of NEB methylated adapters and 1.5 Î¼l of T4 DNA ligase. It is crucial to use methylated adapters to avoid any incompatibility at the amplification step with the index primers. If non-methylated adapters are used, the bisulfite treatment will convert non-methylated cytosine into thymidine, and this will interfere with the recognition of the adapter sequence in future reactions (Table 3).<br/><br/><strong>Table 3. Preparation of adapter ligation mix</strong><br/><img alt="" class="layerphoto" height="144" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217183924_3298.jpg" width="500"/><br/><br/></li><li>On ice, add 25 Î¼l of the master mix to each 26 Î¼l DNA sample. Mix by pulse-vortexing and briefly centrifuge.</li><li>Incubate for 20 min at 20 Â°C in a thermocycler with a non-heated lid. </li><li>Remove the tubes from the thermocycler and add 3 Î¼l of USER enzyme (included in NEBNext<sup>Â®</sup> Multiplex Oligos kit) to every reaction. Keep the reaction tubes at room temperature while adding the enzyme. Mix by pulse-vortexing, briefly centrifuge and place the tubes back in the thermocycler for 15 min at 37 Â°C with a non-heated lid. </li></ol><br/><em>Note: A second AMPure XP magnetic bead clean-up step is performed.</em><br/><ol class="npwtwo" start="5"><li>Add 55 Î¼l of beads to each sample. Mix by slowly pipetting up and down at least 10 times. </li><li>Incubate for 5 min at room temperature.</li><li>Place the tubes on a DynaMag<sup>TM</sup>-96 Side magnet for 5 min. At this stage, DNA is captured by the beads. </li><li>Without disturbing the beads, carefully remove and discard the supernatant.</li><li>Wash the beads by adding 100 Î¼l of 80% freshly prepared EtOH. Incubate for 5 min and remove EtOH.</li><li>Repeat Step E9 for a second wash.</li><li>Remove the residual EtOH with a P10 pipet.</li><li>Dry the beads for about 5 min. Do not over dry (Note 2).</li><li>Remove tubes from the magnetic rack, resuspend the beads in 25 Î¼l warmed EB buffer, and incubate for 2 min at room temperature (not on the magnetic rack). </li><li>Place the tubes back in the magnetic rack and incubate for at least 5 min to make sure all the beads stick to the magnet.</li><li>Carefully remove the supernatant without taking any beads and put into a new PCR tube.<br/><em>Note: The samples can be frozen (-20 Â°C) at this time to continue later (Note 3).</em> </li></ol><br/></li><li>Bisulfite conversion (estimated time: 3 h)<br/>Sodium bisulfite DNA treatment allows for discrimination between methylated and unmethylated cytosines. Compared to previously described RRBS methods, our bisulfite conversion procedure reduces significantly time associated with this step. <br/>Since sodium bisulfite-treated DNA is more sensitive to freezing and thawing, it is better to perform the bisulfite conversion and the amplification of rRRBS libraries without stopping to avoid any degradation (Note 3). (Table 4)<br/><br/><strong>Table 4. Preparation of sodium bisulfite conversion reaction</strong> (from QIAGEN EpiTect Fast bisulfite kit)<br/><img alt="" class="layerphoto" height="90" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217183945_6589.jpg" width="500"/><br/><br/><ol class="npwtwo"><li>Add 85 Î¼l of bisulfite mix to each 25 Î¼l DNA sample. Pipet up and down for at least 10 times.</li><li>Add 30 Î¼l of DNA protect buffer. Mix by pulse-vortexing, briefly centrifuge and place tubes in the thermocycler.</li><li>Run 2 cycles: 95 Â°C for 5 min and 60 Â°C for 20 min as recommended by the manufacturerâs protocol.</li></ol>During the last minutes of incubation, a preparation of carrier RNA (QIAGEN EpiTect Fast bisulfite kit) is made by mixing the appropriate quantity of carrier RNA with BL buffer following the indication from the kit handbook. For the second part of the bisulfite reaction procedure (<em>i.e.</em>, cleanup steps), follow the manufacturerâs protocol. To elute the DNA, carefully add 15 Î¼l of EB buffer in the middle of the column and incubate at room temperature for 5 min, then centrifuge for 1 min at full speed. Elution step is repeated with another 15 Î¼l of EB buffer.<br/><br/></li><li>Amplification of rRRBS libraries<br/>This step, one of the main improvements of our approach, uses a quantitative-PCR (qPCR) method to directly monitor the final amplification of rRRBS libraries to minimize the over cycling effect. In previous protocols, samples were tested with varying PCR cycle numbers to determine the minimal cycle number to be used in the final library preparation. For each sample, the appropriate number of cycles was determined by visualizing the PCR reactions on gel electrophoresis, and then a final library amplification step was performed on the rest of the samples. This protocol is therefore faster and accurately defines the appropriate number of PCR cycles required for library amplification. <br/><ol class="npwtwo"><li>Prepare a large-scale PCR mix reaction for each sample following the indication provided in Table 5. NEB Universal primer and Indexing primers are provided with the methylated adapterâs kit. SYBR Green is diluted to 10x in water from 10,000x stock.<br/><br/><strong>Table 5. Preparation of qPCR mix for library amplification</strong><br/><img alt="" class="layerphoto" height="194" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217184002_4563.jpg" width="500"/><br/><br/></li><li>Divide the 200 Î¼l reaction mix of each sample by distributing 25 Î¼l in individual wells of an 8-strip PCR tube (0.2 ml). Insert PCR strips into the Roche LightCycler96 qPCR instrument and run under conditions indicated in Table 6.<br/><br/><strong>Table 6. qPCR amplification protocol</strong><br/><img alt="" class="layerphoto" height="193" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217184019_0840.jpg" width="500"/><br/><br/></li><li>Stop the qPCR reaction at the end of the linear phase of amplification, after about 12 to 15 cycles, before the plateau phase is reached (Figure 2). <br/><em>Note: When various libraries are amplified simultaneously, they might not all reach the plateau phase at the same time. Samples with adequate amplification can be quickly removed at the end of an amplification cycle, and transferred to another PCR machine for the final 7 min of extension step. </em> </li><li>Perform a final extension step of 7 min at 72 Â°C. <br/><br/><img alt="" class="layerphoto" height="168" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217183657_6435.jpg" width="180"/><br/><strong>Figure 2. Schematic of qPCR library amplification plots.</strong> A. The red line represents the total amplified product accumulation at each qPCR cycle. The blue rectangle represents the period when the qPCR library amplification should be stopped; following the exponential phase and just before the plateau phase. B-C. Examples of library amplifications for 2 sample (each separated in 8 wells) stopped at (B) cycle 14 and (C) cycle 16.</li></ol><br/></li><li>Clean-up and size selection of rRRBS libraries (estimated time: 1 h 30 min)<br/>After amplification of rRRBS libraries by qPCR, the last clean-up step is performed using AMPure XP magnetic beads to remove reagent and primer-dimers. <br/><ol class="npwtwo"><li>For each sample, pool all 8 PCR reactions into a 1.5 ml low-binding DNase/RNase-free tube. </li><li>Calculate the precise volume of each reaction and add the same volume of beads to the PCR reaction to have a 1x ratio (Note 1) (<em>i.e.</em>, add 200 Î¼l of beads if 200 Î¼l of reaction remains after the PCR). Mix by pipetting up and down. The DNA/bead 1x ratio will remove most fragments below 200 bp and all smaller fragments below 100 bp.</li><li>Incubate for 15 min at room temperature and gently tap the tube every 5 min to mix beads and DNA. </li><li>Place tubes in a magnetic rack and let stand 10 min. </li><li>Carefully remove and discard the supernatant without disturbing the beads.</li><li>By keeping the tubes on the magnetic rack, wash the beads by adding 1 ml of 80% freshly prepared EtOH. Incubate for 5 min and remove EtOH.</li><li>Repeat Step H6 for a second wash.</li><li>Remove the residual EtOH with a P20 pipet.</li><li>Dry the beads for about 10 min. Do not over dry (Note 2). </li><li>Remove the tube from the magnetic rack and resuspend the beads in 40 Î¼l of pre-warmed EB buffer. Mix by pipetting, and incubate for 5 min at room temperature (not on the rack). </li><li>Put the tubes back in the magnetic rack and incubate for at least 5 min to ensure all the beads stick to the magnet.</li><li>Carefully remove the supernatant without taking any beads and transfer the solution into a new 1.5 ml tube.</li></ol><br/></li><li>Quality control (estimated time: 1 h 30 min)<br/>Assess the quality and concentration of the sample by performing a High Sensitivity DNA Assay on the 2100 Bioanalyzer instrument. Concentration varies depending on the original amount of input DNA and the efficiency of the different steps. The peak should be between 200 bp and 300 bp, and concentration at approximatively 5 ng/Î¼l or higher. If a peak of primer-dimer is visible on the Bioanalyzer, at a length of about 75 bp or 150 bp (see Figure 3), perform a second DNA/beads cleanup and repeat the Bioanalyzer quality assessment before sequencing.<br/><em>Note: If you observe a peak at around 75 bp or 150 bp, perform a second DNA/beads clean-up with a 1x ratio of beads to get rid of all the primer-dimer as those smaller fragments can affect the quality of the sequencing.</em><br/><br/><img alt="" class="layerphoto" height="112" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217183714_9023.jpg" width="300"/><br/><strong>Figure 3. Quality controls of the final libraries by Bioanalyzer.</strong> A. Example of a good library with the major peak between 200 bp and 300 bp, no significant peak is observed at a lower size. B. Example of a library containing primer dimers (significant peak at ~75 bp and/or 150 bp). For this sample, a second bead clean-up (with 1x ratio of beads) would have to be performed to remove the primer dimers before sequencing the library.<br/><br/></li><li>Sequencing<br/>Paired-end sequencing (&gt; 100 bp read length) of libraries is performed on Illumina apparatus. The number of multiplex libraries per sequencing lane depends on the sequencing technology used and the species investigated (<em>i.e.</em>, the total number of CpG of species) (Doherty and Couldrey, 2014). One should aim to obtain at least 20-30 million reads per sample for downstream bioinformatics analysis. <br/></li></ol><p class="pptt" id="biaoti37792">### Data analysis</p><p class="ppzz">Various bioinformatics tools are widely available to analyze genome-wide DNA methylation sequencing data, and are usually customized depending on how data will be interpreted and visualized (<em>e.g.</em>, single CpG methylation, tilling-window approach). Details of basic analysis are detailed in previous papers (Magnus <em>et al.</em>, 2014; McGraw <em>et al.</em>, 2015; PichÃ© <em>et al.</em>, 2018). Briefly, Trim Galore (Krueger, 2015) is used for sequence trimming and quality control, then reads are aligned to the reference genome using BSMAP (Xi and Li, 2009), which is specific for the alignment of bisulfite-treated sequenced DNA. If Î» DNA was spike-in, you can calculate the bisulfite conversion rate by mapping this genome and determining the percentage of unmethylated cytosine (sequenced as thymine) at the position of a cytosine in the Î» phage reference genome (Lister <em>et al.</em>, 2011). Methylation calls are obtained using BSMAP and differentially methylated regions across two conditions are obtained using the R package methylKit (Akalin <em>et al.</em>, 2012). Differentially methylated sequences are annotated using HOMER v4.10 (Heinz <em>et al.</em>, 2010). Final output processed data can be used in various other downstream analyses, such as functional annotation tools to highlight enrichment of specific biological [<em>e.g.</em>, DAVID (Dennis <em>et al.</em>, 2003), Metascape (Tripathi <em>et al.</em>, 2015)], or converted into specific formats, such as a bedGraph, in order to view methylation levels in various genome browsers [<em>e.g.</em>, USCS genome browser, IGV (Thorvaldsdottir <em>et al.</em>, 2013)].</p><p class="pptt" id="biaoti37793">### Notes</p><ol class="npwtwok"><li>The ratio of beads is crucial to ensure a good clean-up. To make sure to add the right volume of beads, you can measure the exact volume of sample using a pipet at every clean-up step.</li><li>Bead ring appears cracked if you over dry it. The beads will be harder to resuspend and this will decrease nucleic acid elution efficiency.</li><li>Stopping point: It is recommended to stop after the adapter ligation step (Procedure E) or to go through all the protocol in the same day.<br/></li></ol><p class="pptt" id="biaoti37794">### Acknowledgments</p><p class="ppzz">Research was funded by an NSERC discovery grant to SM. LML received scholarship from Fondation du CHU Sainte-Justine and RÃ©seau QuÃ©bÃ©cois en Reproduction. SM is supported by Chercheur-boursier Junior 1 from the Fonds de recherche du QuÃ©bec â SantÃ© (FRQS). We thank Elizabeth Maurice-Elder for editing.</p><p class="pptt" id="biaoti37795">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti37796">### References</p><ol class="npwtwok"><li>Akalin, A., Kormaksson, M., Li, S., Garrett-Bakelman, F. E., Figueroa, M. E., Melnick, A. and Mason, C. E. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23034086" target="_blank">methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles.</a> <em>Genome Biol </em>13(10): R87.</li><li>Boyle, P., Clement, K., Gu, H., Smith, Z. D., Ziller, M., Fostel, J. L., Holmes, L., Meldrim, J., Kelley, F., Gnirke, A. and Meissner, A. (2012). <a href="https://genomebiology.biomedcentral.com/track/pdf/10.1186/gb-2012-13-10-r92" target="_blank">Gel-free multiplexed reduced representation bisulfite sequencing for large-scale DNA methylation profiling.</a> <em>Genome Biol</em> 13(10): R92.</li><li>Dennis, G., Jr., Sherman, B. T., Hosack, D. A., Yang, J., Gao, W., Lane, H. C. and Lempicki, R. A. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12734009" target="_blank">DAVID: database for annotation, visualization, and integrated discovery.</a> <em>Genome Biol</em> 4(5): P3.</li><li>Doherty, R. and Couldrey, C. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24860595" target="_blank">Exploring genome wide bisulfite sequencing for DNA methylation analysis in livestock: a technical assessment.</a> <em>Front Genet </em>5: 126. </li><li>Gu, H., Smith, Z. D., Bock, C., Boyle, P., Gnirke, A. and Meissner, A. (2011). <a href="http://www.nature.com/articles/nprot.2010.190" target="_blank">Preparation of reduced representation bisulfite sequencing libraries for genome-scale DNA methylation profiling.</a> <em>Nat Protoc</em> 6(4): 468-481. </li><li>Heinz, S., Benner, C., Spann, N., Bertolino, E., Lin, Y. C., Laslo, P., Cheng, J. X., Murre, C., Singh, H. and Glass, C. K. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20513432" target="_blank">Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities.</a> <em>Mol Cell </em>38(4): 576-589.</li><li>Krueger, F. (2015). Trim Galore!: A wrapper tool around Cutadapt and FastQC to consistently apply quality and adapter trimming to FastQ files: 0.4.</li><li>Lister, R., Pelizzola, M., Kida, Y. S., Hawkins, R. D., Nery, J. R., Hon, G., Antosiewicz-Bourget, J., O'Malley, R., Castanon, R., Klugman, S., Downes, M., Yu, R., Stewart, R., Ren, B., Thomson, J. A., Evans, R. M. and Ecker, J. R. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21289626" target="_blank">Hotspots of aberrant epigenomic reprogramming in human induced pluripotent stem cells.</a> <em>Nature </em>471(7336): 68-73.</li><li>Magnus, N., Garnier, D., Meehan, B., McGraw, S., Lee, T. H., Caron, M., Bourque, G., Milsom, C., Jabado, N., Trasler, J., Pawlinski, R., Mackman, N. and Rak, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24520174" target="_blank">Tissue factor expression provokes escape from tumor dormancy and leads to genomic alterations.</a> <em>Proc Natl Acad Sci U S A </em>111(9): 3544-3549.</li><li>McGraw, S., Zhang, J. X., Farag, M., Chan, D., Caron, M., Konermann, C., Oakes, C. C., Mohan, K. N., Plass, C., Pastinen, T., Bourque, G., Chaillet, J. R. and Trasler, J. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25578964" target="_blank">Transient DNMT1 suppression reveals hidden heritable marks in the genome.</a> <em>Nucleic Acids Res </em>43(3): 1485-1497.</li><li>Meissner, A., Gnirke, A., Bell, G. W., Ramsahoye, B., Lander, E. S. and Jaenisch, R. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16224102" target="_blank">Reduced representation bisulfite sequencing for comparative high-resolution DNA methylation analysis.</a> <em>Nucleic Acids Res</em> 33(18): 5868-5877. </li><li>Meissner, A., Mikkelsen, T. S., Gu, H., Wernig, M., Hanna, J., Sivachenko, A., Zhang, X., Bernstein, B. E., Nusbaum, C., Jaffe, D. B., Gnirke, A., Jaenisch, R. and Lander, E. S. (2008). <a href="http://www.nature.com/articles/nature07107" target="_blank">Genome-scale DNA methylation maps of pluripotent and differentiated cells.</a> <em>Nature </em>454(7205): 766-770. </li><li>PichÃ©, J., Gosset, N., Legault, L.-M., Pacis, A., Oneglia, A., Caron, M., Chetaille, P., Barreiro, L., Liu, D., Qi, X., Nattel, S., Leclerc, S., Breton-LarrivÃ©e, M., Andelfinger, G., Bakkers, J., Loeys, B., PucÃ©at, M., McGraw, S. and Andelfinger, G. (2018). <a href="http://www.sciencedirect.com/science/article/pii/S2352345X18301577" target="_blank">Molecular signature of CAID syndrome: noncanonical roles of SGO1 in regulation of TGF-Î² signaling and epigenomics.</a> CMGH Doi:10.1016/j.jcmgh.2018.10.011</li><li>Smith, Z. D., Gu, H., Bock, C., Gnirke, A. and Meissner, A. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19442738" target="_blank">High-throughput bisulfite sequencing in mammalian genomes.</a> <em>Methods</em> 48(3): 226-232. </li><li>Thorvaldsdottir, H., Robinson, J. T. and Mesirov, J. P. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22517427" target="_blank">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</a> <em>Brief Bioinform </em>14(2): 178-192.</li><li>Tripathi, S., Pohl, M. O., Zhou, Y., Rodriguez-Frandsen, A., Wang, G., Stein, D. A., Moulton, H. M., DeJesus, P., Che, J., Mulder, L. C., Yanguez, E., Andenmatten, D., Pache, L., Manicassamy, B., Albrecht, R. A., Gonzalez, M. G., Nguyen, Q., Brass, A., Elledge, S., White, M., Shapira, S., Hacohen, N., Karlas, A., Meyer, T. F., Shales, M., Gatorano, A., Johnson, J. R., Jang, G., Johnson, T., Verschueren, E., Sanders, D., Krogan, N., Shaw, M., Konig, R., Stertz, S., Garcia-Sastre, A. and Chanda, S. K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26651948" target="_blank">Meta- and orthogonal integration of influenza "OMICs" data defines a role for UBR4 in virus budding.</a> <em>Cell Host Microbe </em>18(6): 723-735.</li><li>Xi, Y. and Li, W. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19635165" target="_blank">BSMAP: whole genome bisulfite sequence MAPping program.</a> <em>BMC Bioinformatic</em>s 10: 232.<br/></li></ol>