<body>
            <h1>On-demand Labeling of SNAP-tagged Viral Protein for Pulse-Chase Imaging, Quench-Pulse-Chase Imaging, and Nanoscopy-based Inspection of Cell Lysates</h1>
            <p><strong>Authors:</strong> Roland  Remenyi, Raymond  Li, Mark  Harris</p>
            <h2>Abstract</h2>
            <p>Advanced labeling technologies allow researchers to study protein turnover inside intact cells and to track the labeled protein in downstream applications. In the context of a viral infection, the combination of imaging and fluorescent labeling of viral proteins sheds light on their biological activity and interaction with the host cell. Initial approaches have fused fluorescent proteins such as green fluorescent protein (GFP) to the viral protein-of-interest. In contrast, self-labeling enzyme tags such as the commercial SNAP-tag, a modified version of human O<sup>6</sup>-alkylguanine-DNA-alkyltransferase, covalently link synthetic ligands, which users can add on demand. The first two protocols presented here build on previously published protocols for fluorescent labeling in pulse-chase and quench-pulse-chase experiments; the combination of fluorescent labeling with advanced light microscopy visualizes the dynamic turnover of the SNAP-tagged viral protein in intact mammalian cells. A third protocol also outlines how to inspect cellular lysates microscopically for detergent-resistant assemblies of the labeled viral protein. These protocols showcase the flexibility of the SNAP-based labeling system for tracking a viral protein-of-interest in live cells, intact fixed cells, and cell lysates. Moreover, the protocols employ recently developed commercial microscopes (<em>e.g.</em>, Airyscan microscopy) that balance resolution, speed, phototoxicity, photobleaching, and ease-of-use. </p>
            <p><strong>Keywords:</strong> Fusion proteins, Stress granules, RNPs, Site-specific protein labeling, Chemical labeling, Cell biology, Bioimaging, Live cell imaging, Chikungunya, CHIKV, Alphavirus, Arbovirus, nsP3</p>
            <h2>Background</h2><p>To better understand the role of a viral protein during the infectious life cycle, we have adapted existing strategies that facilitate the determination of intracellular protein location, the assessment of protein dynamics in intact cells, and the continued tracking of the viral protein after lysis of the host cell. The first two protocols build on previously published protocols for on-demand SNAP-labeling and the analysis of protein turnover (Bodor <em>et al.</em>, 2012). In previous work, our laboratory has fused a viral protein-of-interest, nonstructural protein 3 (nsP3) of Chikungunya virus (CHIKV), to the SNAP-tag, a modified form of a 20-kDa monomeric DNA repair enzyme (Remenyi <em>et al.</em>, 2017 and 2018). During SNAP-labeling, the addition of a synthetic O<sup>6</sup>-benzylguanine (BG) derivative results in a covalent bond between a reactive cysteine residue in the SNAP-tag and the BG-probe (Keppler <em>et al.</em>, 2004a and 2004b).<br/><br/>We have also combined SNAP-labeling with a protocol for inspecting cell lysates via light microscopy, which enables visualization of detergent-resistant protein assemblies. Similar approaches have allowed researchers to detect stable granular assemblies of GFP-tagged stress-granule proteins in cell lysates (Jain <em>et al.</em>, 2016; Wheeler <em>et al.</em>, 2017). Stress granules are assemblies of RNA and protein (RNPs), which form under conditions of cellular stress (Kedersha <em>et al.</em>, 2005). It is now possible to isolate the more stable stress granule core from both yeast and mammalian cells (Jain <em>et al.</em>, 2016). SNAP-labeling offers an alternative way of tracking tagged viral proteins that may be present in similar subcellular assemblies. Hence, Protocol 3 may not only be useful to study the biochemical nature of viral proteins but also to track any cellular protein that resides in non-membranous organelles such as RNPs and stress granules. For example, integration of the SNAP-tag into the development of cell lines that produce fluorescently tagged stress granules (Kedersha <em>et al.</em>, 2008) could increase experimental flexibility during dynamic and quantitative imaging of these cellular sub-compartments.<br/><br/>Our three protocols also take advantage of recently developed commercial imaging systems for multi-color fluorescence microscopy. We analyze labeled samples in Protocol 1 with live-cell imaging (Figure 1) and thus recommend following general procedures for controlling temperature, reducing phototoxicity, limiting photobleaching, and maintaining cell viability (Frigault <em>et al.</em>, 2009). The chosen 2018 Nikon Ti-E2 system allowed us to image a large field-of-view, lessen focus drift with a proprietary Perfect Focus System (PFS), and record multiple positions with the motorized stage. Moreover, illumination with an LED and light exposures not exceeding 1 s allowed for gentler imaging compared to the typical imaging setup of a laser scanning confocal microscope.<br/><br/>For the analysis of labeled protein in Protocols 2 and 3, we chose a confocal imaging setup with proprietary ZEISS Airyscan technology (Figure 1), which is a commercial version of the âimage scanning microscopyâ approach (Muller and Enderlein, 2010; Sheppard <em>et al.</em>, 2013). Airyscan microscopy represents one of the recent innovations in fluorescence super-resolution microscopy, also referred to as nanoscopy (Li <em>et al.</em>, 2018). The Airyscan technology improves system resolution with an improved detector design, which features a 32-channel detector array (Huff, 2015). With a new 2D super-resolution mode for Airyscan, this detection approach can now enhance resolution 2-fold while lowering the required fluorescence intensity to obtain high-quality images (Huff <em>et al.</em>, 2017). The Airyscan system allowed us to stain our samples with fluorescent dyes from the SNAP product range (<em>e.g.</em>, SNAP-Cell<sup>Â®</sup> 647-SiR and SNAP-Cell<sup>Â®</sup> TMR-Star) and detect them even at low levels during the recovery phase of the quench-pulse-chase protocol. Increased sensitivity also helped in the nanoscopy-based inspection of cellular lysate in Protocol 3 and allowed the visualization of granular structures made up of the SNAP-tagged viral protein. The combination of SNAP-based labeling and innovative detection via Airyscan has the potential to further bioimaging with higher resolution, sensitivity, and user-friendliness.<br/><br/><img alt="" class="layerphoto" height="154" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190217/20190217232228_3201.jpg" width="180"/><br/><strong>Figure 1. Overview of experimental protocols.</strong> This flowchart outlines the essential steps in Protocols 1, 2, and 3. For the definition of âstable CHIKV cellsâ, see Protocol 1, Procedure section. </p>
        </body>

<p class="pptt" id="biaoti_no">### Protocol 1: Pulse-chase experiments for long-term imaging with Nikon Ti2-EMaterials and Reagents</p><ol class="npwtwok"><li>10-cm Petri dish (Corning, catalog number: 430167)</li><li>Micropipette tips, serological pipettes, pipette aids, and microtubes for liquid handling<br/><ol class="npwthree"><li>TipOne<sup>Â®</sup> 1,000 Âµl XL (catalog number: S1122-1830)</li><li>TipOne<sup>Â®</sup> 200 Âµl (catalog number: S1120-8810)</li><li>TipOne<sup>Â®</sup> 20 Âµl filter tips (catalog number: S1120-1810) or SARSTEDT pipette tip 10 Âµl (catalog number: 70.1130.600)</li><li>Fisherbrand<sup>TM</sup> 5 ml serological pipets (catalog number: 13-676-10H)</li><li>Fisherbrand<sup>TM</sup> 10 ml serological pipets (catalog number: 13-676-10J)</li><li>Fisherbrand<sup>TM</sup> 25 ml serological pipets (catalog number: 13-676-10K)</li><li>Drummond Pipet-Aid XL (catalog number: 4-000-205)</li><li>Microtube 1.5 ml (SARSTEDT, catalog number: 72.690.001)</li></ol></li><li>Nunc 35-mm glass bottom (#1.5 or 0.16-0.19 mm thickness) dishes with 27 mm viewing area (Thermo Fisher Scientific, catalog number: 150682)<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Only open packages inside a biosafety cabinet and reseal any remaining dishes to avoid contamination.</em> </li><li><em>Use any dish or slide format that is compatible with an inverted microscope. Match the thickness of the glass bottom with the suggested thickness found on the microscopeâs objective (typically #1.5 or 0.16-0.19 mm thickness). We prefer the dishes listed above because of their large viewing area. The cell lines used in this protocol can grow on glass substrates, but surface treatment (</em><em>e.g.</em><em>, Poly-Lysine coating) may be necessary for other cell lines.</em> </li></ol></li><li>HuH-7 cell line (<a href="http://cellbank.nibiohn.go.jp/~cellbank/en/search_res_det.cgi?ID=385" target="_blank">Japanese Collection of Research Bioresources [JCBR]</a>, catalog number: JCBR 0403)<br/><em>Note: HuH-7 is a well-differentiated hepatocyte-derived cellular carcinoma cell line. It was originally taken from a liver tumor in a Japanese male in 1982 (Nakabayashi </em><em>et al.</em><em>, 1982). The cells used in the creation of this protocol were obtained from John McLauchlan (Centre for Virus Research, Glasgow).</em> </li><li>A plasmid that encodes SNAP<sub>f</sub><sup>Â®</sup>, a SNAP-tag protein (NEB, catalog number: N9183S; or as part of the SNAP-Cell<sup>Â®</sup> Starter Kit, NEB, catalog number: E9100S)</li><li>For initial validation: plasmid that encodes pSNAP<sub>f</sub>-Cox8A control plasmid (also part of the SNAP-Cell<sup>Â®</sup> Starter Kit, NEB, catalog number: E9100S)</li><li>Trypsin EDTA solution (Sigma, catalog number: T3924-500ML)</li><li>Specific reagents for our cell line (for basic tissue culture), which we derived from the hepatoma cell line HuH-7<br/><ol class="npwthree"><li>Dulbecco's modified Eagle's medium (Sigma, catalog number: D6429-500ML)</li><li>100% Fetal Calf Serum (Gibco, catalog number: 10500-064)</li><li>Gibco MEM Nonessential amino acids solution (100x), store at 4 Â°C up to 24 months from the date of manufacture (Thermo Fisher Scientific, catalog number: 11140050) </li><li>Gibco HEPES (N-2-hydroxyethylpiperazine-N-2-ethane sulfonic acid) buffer (100x), store at 4 Â°C up to 24 months from the date of manufacture (Thermo Fisher Scientific, catalog number: 15630130) </li></ol></li><li>FluoroBrite DMEM, store at 4 Â°C up to 24 months from the date of manufacture (Thermo Fisher Scientific, catalog number: A1896701)</li><li>Cell-permeable SNAP-Cell<sup>Â®</sup> 647-SiR (New England Biolabs, catalog number: S9102S)<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>Package contains 30 nmol of the substrate. Resuspend with 50 Âµl of sterile dimethyl sulfoxide (DMSO, Fisher Bioreagents<sup>TM</sup>, catalog number: BP231-100) to make up a stock solution, which can be stored at -20 Â°C. We have used stock solutions that have been stored for up to 3 years. However, the manufacturerâs recommended shelf-life is three months dissolved in DMSO and two years dry.</em> </li><li><em>NEB also offers cell-impermeable SNAP probes (âCell Surfaceâ Probes); use these probes for studies of viral proteins that accumulate at the surface of host cells. </em> </li></ol></li><li>Optional: Red fluorescent or orange fluorescent cellular dye (<em>e.g.</em>, MitoTracker<sup>TM</sup> Orange CMTMRos, Thermo Fisher Scientific, catalog number: M7510)<br/><em>Note: To prepare a stock solution, dissolve lyophilized MitoTracker</em><sup><em>TM</em></sup><em> probe in DMSO to a final concentration of 1 mM and store frozen and protected from light.</em> </li><li>Molecular Probes Invitrogen ProLong Live Antifade Reagent, for live cell imaging (Thermo Fisher Scientific, catalog number: P36975)<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Store at 2-8 Â°C for short-term storage and â¤ -20 Â°C for long-term storage.</em> </li><li><em>According to the manufacturerâs instructions, use the product within 30 days when stored at 2-8 Â°C. When stored at â¤ -20 Â°C, the product is stable for at least six months with up to four freeze-thaw cycles.</em> </li></ol></li><li>Glutamax-I (Thermo Fisher Scientific, catalog number: 35050061)</li><li>Live cell imaging solution (see Recipes)</li><li>Tissue culture medium containing serum (see Recipes)<br/></li></ol><p class="pptt" id="biaoti38071">### Equipment</p><ol class="npwtwok"><li>Air-displacement micropipettes<br/><ol class="npwthree"><li>Starlab ErgoOne<sup>Â®</sup> 100-1,000 Âµl Single-Channel Pipette (catalog number: S7110-1000) or Gilson F123602 PIPETMAN Classic Pipet P1000 (Fisher Scientific, catalog number: 10387322)</li><li>Starlab ErgoOne<sup>Â®</sup> 20-200 Âµl Single-Channel Pipette (catalog number: S7100-2200) or Gilson F123615 PIPETMAN Classic Pipet P100 (Fisher Scientific, catalog number: 10442412)</li><li>Starlab ErgoOne<sup>Â®</sup> 2-20 Âµl Single-Channel Pipette (catalog number: S7100-0220) or Starlab ErgoOne<sup>Â®</sup> 0.1-2.5 Âµl Single-Channel Pipette (catalog number: S7100-0125) or Gilson F144801 PIPETMAN Classic Pipet P2 (Fisher Scientific, catalog number: 10635313) or Gilson F123600 PIPETMAN Classic Pipet P20 (Fisher Scientific, catalog number: 10082012)</li></ol></li><li>Equipment for basic cell culture techniques and aseptic procedures, <em>i.e.,</em> <ol class="npwthree"><li>Biosafety cabinet (<em>e.g.</em>, Thermo Scientific Holten Safe 2010 Model 1.2, catalog number: 8207071100)</li><li>Humidified incubator set to 37 Â°C (Panasonic Incusafe, catalog number: MCO-20AIC)</li></ol><em>Note: Any manufacturer and model of a biosafety cabinet will do, but the chosen biosafety cabinet needs to be appropriate for the containment of cells and viruses. When handling infectious viruses, use the proper facilities, practices, and procedures. Refer to local and international guidelines for laboratory biosafety. </em> </li><li>-20 Â°C freezer (Labcold, catalog number: RLCF1520)</li><li>Water bath (<em>e.g.</em>, Clifton, catalog number: NE2-8D; however, any water bath set to 37 Â°C may be used)</li><li>Table-top microcentrifuge, 4 Â°C to room temperature (RT), max speed â¥ 12,000 <em>x g</em> (Eppendorf, catalog number: 5424R)<br/><em>Note: For spinning down precipitate that may form during extended storage of SNAP probes.</em> </li><li>Live-cell imaging microscope for long-term imaging. For example, a Nikon Ti2-E inverted widefield microscope (for a similar setup, contact your local Nikon representative to create a customized order for the microscope system)<br/><ol class="npwthree"><li>Nikon Ti2-E inverted microscope stand</li><li>Motorized stage (standard Ti2 encoded motorized XY stage)</li><li>Lumencor Spectra X LED light source</li><li>CFI Plan Apo Lambda 60x oil/1.4 NA objective</li><li>Photometric Prime 95B sCMOS monochrome camera</li><li>Semrock 32 mm filter (Green) for Nikon Ti2-E: GFP-4050B Filter cube with Ex 466/40 single-band bandpass filter (product code: FF01-466/40-25), DM495 dichroic beamsplitter (product code: FF495-Di03-25x36), BA525/50 single-band bandpass filter (product code: FF03-525/50-25)</li><li>Semrock 32 mm filter (Red) for Nikon Ti2-E: Cy3-4040C Filter cube with Ex 531/40 single-band bandpass filter (product code: FF01-531/40-25), Dichroic Mirror DM 562 (product code: FF562-Di03-25x36, Barrier Filter: BA 593/40 (product code: FF01-593/40-25)</li><li>Semrock 32 mm filter (Far-Red) for Nikon Ti2-E: Cy5-4040C Filter cube with Ex 628/40 single-band bandpass filter (product ID: FF02-628/40-25), Single Band Emitter (product ID: FF01-692/40-25), Single Band Dichroic (product ID: FF660-Di02-25x36) </li><li>Okolab stage top incubator, catalog number: H301-NIKON-NZ100/200/500-N. Set to 37 Â°C with 5% CO<sub>2</sub> (any manufacturer and model that can be mounted on a Nikon Ti2 motorized stage will do)</li><li>Perfect Focus System (PFS)</li></ol><em>Notes: </em><br/><ol class="npwfour"><li><em>The above list is not exhaustive but rather lists the essential components that have been useful in the creation of Protocol 1. Consult with your local Nikon representative for the remaining components of a customized microscope system.</em> </li><li><em>We prefer to image cells in open tissue-culture dishes with a coverslip-like glass bottom, which necessitates an inverted microscope stand. </em> </li><li><em>As Protocol 1 is a powerful approach for long-term imaging of the labeled protein, it is important to keep cells in focus and correct for focus drift caused by thermal and mechanical conditions. This live-cell imaging system included Nikonâs fourth-generation Perfect Focus System (PFS), which monitors the partial reflection of a low-power infrared laser beam on the interface between the dishâs glass bottom and liquid media above. This feature provides continuous and real-time focus drift correction. </em> </li><li><em>A large (25 mm x 25 mm) field of view (FOV) enables increased data throughput and capture of additional cells that would be outside the normal FOV. </em> </li><li><em>We preferred an LED light source over laser-based illumination. LED illumination limited photobleaching and phototoxicity; âgentleâ imaging was essential during acquisition of five-dimensional datasets (5-D: multi-color 3-D Z-stacks over time). </em> </li><li><em>A motorized microscope stage and the ability to record multiple positions during one experiment further increased the throughput of the imaging system.</em> </li></ol></li></ol><p class="pptt" id="biaoti38072">### Software</p><ol class="npwtwok"><li>Nikon NIS Elements AR imaging software<br/>Nikon NIS Elements AR imaging software controls all components of the microscope. Use NIS Elements to set-up parameters for acquisition of 5-D datasets. Users can also create rendered videos within NIS Elements and save individual frames of 5-D data. For our widefield images, we also used a Richardson-Lucy algorithm (set to 10 iterations) to deconvolve datasets within the Elements software. <br/></li></ol><p class="pptt" id="biaoti38073">### Procedure</p><p class="ppzz">A published labeling procedure forms the basis of the protocol described here (Bodor <em>et al.</em>, 2012). We build on this protocol by also describing a live-cell imaging setup that is suitable for long-term examination of protein turnover in five dimensions (<em>i.e.</em>, 3-D multi-color fluorescence microscopy over time). <br/><br/><em>Note: Carry out all liquid handling steps that involve live cells inside the biosafety cabinet. Only use sterile pipettors, serological pipets, micropipettors, microtubes, and tips.</em> </p><ol class="npwtwok"><li>Production of stable CHIKV cells<br/>We have previously used the HuH-7 hepatoma cell line to derive cells that stably harbor a modified CHIKV replicon; we designed this replicon to encode a fusion protein of CHIKV nsP3 and the SNAP-tag (Remenyi <em>et al.</em>, 2018). We will refer to the cells from this secondary cell line as âstable CHIKV cellsâ in the rest of the protocol. These cells also constitutively express the green fluorescent protein ZsGreen. <br/><ol class="npwthree"><li>Use standard molecular cloning methods to generate N-, C-terminal, or internal SNAP-tag fusions. In our experimental system, we inserted the SNAP-tag within the C-terminal region of CHIKV nsP3 and added a flexible linker of Glycine amino acids at the N-terminal and C-terminal junctions (Remenyi <em>et al.</em>, 2017). A plasmid that encodes SNAP<sub>f</sub><sup>Â®</sup>, a SNAP-tag protein, is available from NEB (N9183S; or as part of the SNAP-Cell<sup>Â®</sup> Starter Kit, E9100S).</li><li>After the SNAP-sequence has been inserted into a viral genome, evaluate whether modified viruses or replicons remain viable and whether the tagged protein can carry out the same biological function as the untagged protein. The methods for verification will vary depending on the virus and the protein-of-interest.</li><li>Maintain cells in preferred tissue-culture format: we routinely passage our stable CHIKV cells in 10-cm Petri dishes. </li><li>We have developed this protocol with the stable CHIKV cell line, which we derived from HuH-7 cells. Other cell lines that support prolonged replication of non-cytotoxic CHIKV replicons include the C2C12 (mouse myoblast) cell line (Remenyi <em>et al.</em>, 2018) and BHK-21 (baby hamster kidney) cell line (Utt <em>et al.</em>, 2015). However, we have not yet validated Protocols 1-3 in these cell lines. </li></ol></li><li>Detach stable CHIKV cells from the growth surface by adding enough trypsin to cover the cells that are attached to the surface of the respective culturing vessel (<em>e.g.</em>, 1-2 ml of trypsin for 10-cm dishes) and seed into a 35-mm dish with coverslip bottom. Prepare a second and third dish containing positive and negative control.<br/><ol class="npwthree"><li>As a negative control, we seed naÃ¯ve cells (known not to express SNAP-tag protein) in a second dish. </li><li>NEBâs SNAP-Cell Starter Kit also contains a positive control plasmid (pSNAP<sub>f</sub>-Cox8A Control Plasmid), which can be transfected into cells to produce SNAP-tagged cytochrome c oxidase with a well-characterized mitochondrial localization. Thus, the third dish may contain cells transfected with pSNAP<sub>f</sub>-Cox8A control plasmid or any plasmid encoding a SNAP-tagged protein with well-characterized subcellular localization. </li></ol><em>Notes:</em><br/><ol class="npwfour"><li><em>For beginners, we recommend handling only three dishes at a time (one dish for the sample, one dish for the negative control, and one dish for the positive control) during the labeling phase of this protocol. Advanced users requiring higher throughput may consider 35-mm imaging dishes with four compartments (</em><em>e.g.</em><em>, ibidi Âµ-Dish 35 mm Quad, catalog number 80416) instead of using individual dishes. Carry out simultaneous experiments (</em><em>e.g.</em><em>, two sample conditions, one negative control and one positive control) in the subdivisions of the dish. </em> </li><li><em>The exact amount of trypsin needed to dissociate adherent cells is dependent on the cell type and age of cells.</em> </li><li><em>We adjust the cell seeding density depending on the length of our desired chase period and the duplication time of the cell line we use for the experiment. </em> </li><li><em>We typically aim for 60% to 80% confluency. For example, assuming a doubling time of 24 h and a desired chase period of 24 h, we would seed our SNAP-tagged cell line to 20% confluency, stain with SNAP-reagents the following day (= pulse, at 40% confluency) and image during a 24-h chase (allowing cells to reach 80% confluency during live-cell imaging).</em> </li></ol></li><li>Incubate cells under standard growth conditions (<em>i.e.</em>, 37 Â°C at 5% CO<sub>2</sub>) overnight. </li><li>On the next day, take the frozen stock solution of SNAP-Cell<sup>Â®</sup> 647 SiR from the -20 Â°C freezer and thaw at room temperature. </li><li>In a 1.5-ml microtube, dilute thawed SNAP-Cell<sup>Â®</sup> 647 SiR; perform a 1:1,000 (final concentration of 0.6 ÂµM) dilution in complete cell culture media. Final volume should be at least 0.6 ml to cover the dish area above the coverslip. Users can increase the volume to 1 ml to reduce the risk of drying out the cells. Vortex briefly (for â¥ 5 s) or pipet the diluted labeling solution up and down (ten times). <br/><ol class="npwthree"><li>SNAP-Cell<sup>Â®</sup> probes are cell-permeable. Our protocols label a fusion protein of CHIKV nsP3 and the SNAP-tag. This fusion protein localizes to intracellular compartments, and hence we only stain with cell-permeable probes.</li><li>Other probes from NEB include SNAP-Cell<sup>Â®</sup> 505 (green fluorescent), SNAP-Cell<sup>Â®</sup> Oregon Green (green fluorescent), and SNAP-Cell<sup>Â®</sup> TMR-Star (red fluorescent). Optimize final concentrations for different SNAP-tagged proteins and cell lines; factors like protein abundance and non-specific binding may vary depending on the tagged protein or cell line. </li><li>According to the manufacturer, optimal substrate concentrations range from 1 to 20 ÂµM, with best results usually obtained at concentrations between 1 and 5 ÂµM. We have found that even 0.6 ÂµM of SNAP-Cell<sup>Â®</sup> SiR provided sufficient staining. We do not prepare more media than we expect to consume within one hour. Always include a negative control (naÃ¯ve cells known not to express SNAP-tag protein) when optimizing labeling conditions.<br/><em>Note: Recipe for three dishes (using 0.6 ml in Step 7, scale accordingly for more dishes): </em><em>1998 Âµl Complete Cell Culture Media; 2 Âµl SNAP-Cell</em><sup><em>Â®</em></sup><em> 647 SiR.</em> </li></ol></li><li>Spin the diluted labeling solution for 5 min at maximum speed (â¥ 12,000 <em>x g</em>) to remove possible insoluble fluorescent debris. Take care not to disturb the pellet when removing supernatant (which may be invisible).</li><li>Replace the medium on stable CHIKV cells with 0.6-1 ml of SNAP-tag labeling medium (pre-heated to 37 Â°C in a water bath). Incubate for 15 min at 37 Â°C, 5% CO<sub>2</sub>.<br/><em>Note: At this step, the benzyl group on the SNAP-Cell</em><sup><em>Â®</em></sup><em> 647 SiR substrate will covalently link to the SNAP-tag and release guanine. We found that a 15-min incubation gave us optimal labeling. According to the manufacturer, optimal reaction times range from 5 to 30 min, respectively, depending on experimental conditions and expression levels of the SNAP-tagged protein. </em> </li><li>During this 15-min incubation period, prepare a cold water bath containing ice and water. Remove a frozen aliquot of ProLong Live reagent from the -20 Â°C freezer and thaw the aliquot in the cold water bath. Do not exceed 37 Â°C while thawing or using the reagent. We keep the reagent in the ice bath until Step 10. </li><li>Wash the cells three times each with 2 ml of tissue culture medium containing serum (pre-heated to 37 Â°C). </li><li>Replace the regular cell-culture media with 2 ml of fresh live cell imaging solution, consisting of FluoroBrite DMEM, supplemented with fetal bovine serum (at a final concentration of 10%), HEPES, Glutamax-I, Nonessential Amino Acids, and ProLong Live Antifade Reagent (see Recipes).</li><li>Place cells back into the incubator after the final wash. Incubate for another 30 min, 37 Â°C at 5% CO<sub>2</sub>.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>The primary purpose of this step is to reduce the background staining of the SNAP reagent. The background staining of some SNAP probes, such as SNAP Cell</em><sup><em>Â®</em></sup><em> TMR-Star can be problematic in some cell lines whereas less background staining is observed in others (Cole, 2014). If the background staining is an issue, we recommend reducing the labeling time, concentration of probe, or increasing the number of washes. Step 10 also starts the incubation with ProLong Live reagent. </em> </li><li><em>The manufacturerâs instructions of the ProLong Live reagent suggest incubating cells in the dark for 15 min to 2 h. </em> </li><li><em>In our experience, by the time the sample dish reaches the microscope, final imaging settings have been set up, and the actual image acquisition starts, the total incubation period of cells with media containing ProLong Live will be at least 1.5 h (~40 min for processing three dishes in Steps 10-12, ~20 min for transferring dishes from tissue culture facility to microscope, and ~30 min for setting up imaging conditions). </em> </li></ol></li><li>Wash cells as described in Step 9 with regular cell-culture media. Replace media with 2 ml of live cell imaging media (made up in Step 10). <br/><em>Note: We supplement the live cell imaging media with ProLong Live solution at this step if live-cell imaging does not exceed 24 h. The manufacturer does not recommend leaving ProLong Live solution on live cells for more than 24 h. We also calculate the chase period from the completion of this step, since it marks the last time point at which the fluorescent substrate can label SNAP-tagged proteins.</em> </li><li>Optional: Stain cells with red fluorescent or orange fluorescent cellular dye (<em>e.g.</em>, MitoTracker<sup>TM</sup> Orange CMTMRos). <br/><ol class="npwthree"><li>Dilute 1 mM MitoTracker<sup>TM</sup> stock solution to the final working concentration (25-500 nM) in âlive cell imaging bufferâ. </li><li>Remove media from dishes and add pre-warmed (37 Â°C) staining solution containing MitoTracker<sup>TM</sup> probe.</li><li>Return dishes to the humidified incubator and incubate for 15-45 min, at 37 Â°C with 5% CO<sub>2</sub>.</li><li>After incubation period is complete, replace staining solution with fresh pre-warmed âlive cell imaging bufferâ. </li></ol></li><li>Transfer the three dishes (sample, positive control, and negative control) to microscope area for live-cell imaging with Nikon Ti2-E system. The live-cell imaging setup for SNAP-tagged cells is similar to standard configurations for live-cell fluorescence imaging. <br/><ol class="npwthree"><li>We recommend <a href="https://www.microscopyu.com/applications/live-cell-imaging" target="_blank">Nikonâs resource on âlive cell imagingâ</a> for an introduction on the appropriate microscope setup for timelapse imaging.</li><li>For additional resources, contact your local Nikon representative for NIS Elements Training handouts on âAdvanced Acquisitionâ modes (<em>i.e.</em>, Multi-channel, Multi-point, Timelapse, and Z-stack)</li><li>For alternative live-cell imaging setups, refer to Bodor <em>et al.</em> (2012). </li><li>We used a widefield imaging setup for extended imaging of the same field-of-view. We obtained high-quality results with a Nikon Ti2-E system. </li><li>Several factors determined our preference for this system, namely (i) the Ti2-E is equipped with a unique perfect focus system (PFS) that automatically corrects focus drift in real time during a prolonged period of imaging (ii) imaging with an LED light source allows for gentler imaging compared to laser-based confocal systems (iii) multipoint Z-stacks can be acquired quickly as a result of faster device movement and image acquisition (iv) quick acquisition reduces overall light exposure and subsequent phototoxicity (v) the Ti2-E provides a large field of view (FOV), which captures a large amount of cells within one FOV, and (vi) multi-point acquisitions further increase the throughput of the system.</li></ol></li><li>Image cells with the preferred imaging system<br/><ol class="npwthree"><li>SNAP Cell<sup>Â®</sup> 647-SiR should have an excitation maximum at 645 nm and an emission maximum at 661 nm. </li><li>With the Nikon Ti2-E inverted microscope, we use standard filter settings for the Cy5 dye. Stable CHIKV cells also endogenously express the green fluorescent ZsGreen reporter protein, which has an excitation maximum of 493 nm and an emission peak at 505 nm (image with a standard GFP filter set). </li><li>The advantage of using the far-red SNAP Cell<sup>Â®</sup> 647-SiR is that additional labeling with a red fluorescent cellular dye (<em>e.g.</em>, MitoTracker<sup>TM</sup> Orange) and imaging with filter settings for Cy3 dye is possible. Figure 2 shows representative images from a timelapse series, in which we set the microscope to take Z-stacks every 15 min for a total of 24 h. <br/><br/><img alt="" class="layerphoto" height="223" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190218/20190218002037_1925.jpg" width="150"/><br/><strong>Figure 2. Combination of 5-D imaging and pulse-chase experiments.</strong> A. We only show selected frames from a multi-position timelapse series, in which the microscope acquired Z-stacks every 15 min at eight positions. In this setup, each Z-stack (composed of 41 slices) was completed within 45 s, whereas it took 6.5 min to obtain eight positions. The images on the left display all channels with pseudo-colors (green: ZsGreen, yellow: MitoTracker<sup>TM</sup> Orange, magenta: SNAP-nsP3). Note that granular structures labeled at 0 h were still present at 3.75 h. This continued presence indicated that these structures remained stable for hours. B. Selected frames from the same timelapse series, this time presented in volume view. The images on the left display all channels with pseudo-colors (green: ZsGreen, yellow: MitoTracker<sup>TM</sup> Orange, magenta: SNAP-nsP3). </li></ol></li></ol><p class="pptt" id="biaoti38074">### Recipes</p><ol class="npwtwok"><li>Live cell imaging solution (enough to add 2 ml of solution to three 35-mm dishes in Steps 10, 12, and optional Step 13)<br/>2.7 ml Fetal Calf Serum<br/>270 Âµl HEPES<br/>270 Âµl Glutamax-I<br/>270 Âµl MEM Nonessential Amino Acids<br/>270-540 Âµl ProLong Live Antifade Reagent<br/>22.96-23.22 ml FluoroBrite DMEM<br/>The total volume of live cell imaging solution: 27 ml</li><li>Tissue culture medium containing serum<br/>500 ml Dulbecco's modified Eagle's medium<br/>56.5 ml Fetal Calf Serum <br/>5.6 ml MEM Nonessential amino acids solution <br/>5.6 ml HEPES <br/></li></ol><p class="pptt" id="biaoti_no">### Protocol 2: Quench-pulse-chase experiments paired with ZEISS LSM 880 Airyscan MicroscopyMaterials and Reagents</p><ol class="npwtwok"><li>Micropipette tips, serological pipettes, pipette aids, and microtubes for liquid handling<br/><ol class="npwthree"><li>TipOne<sup>Â®</sup> 1,000 Âµl XL filter tips (catalog number: S1122-1830)</li><li>TipOne<sup>Â®</sup> 200 Âµl filter tips (catalog number: S1120-8810)</li><li>TipOne<sup>Â®</sup> 20 Âµl filter tips (catalog number: S1120-1810) or SARSTEDT pipette tip 10 Âµl (catalog number: 70.1130.600)</li><li>Fisherbrand<sup>TM</sup> 5 ml serological pipets (catalog number: 13-676-10H)</li><li>Fisherbrand<sup>TM</sup> 10 ml serological pipets (catalog number: 13-676-10J)</li><li>Fisherbrand<sup>TM</sup> 25 ml serological pipets (catalog number: 13-676-10K)</li><li>Drummond Pipet-Aid XL (catalog number: 4-000-205)</li><li>Microtube 1.5 ml (SARSTEDT, catalog number: 72.690.001)</li><li>15 ml Centrifuge Tubes, Conical, Sterile (Starlab, catalog number: E1415-0200)</li></ol></li><li>Glass slide (Academy, catalog number: N/A142)</li><li>Filter paper (Whatman<sup>Â®</sup>, catalog number: 1001025)</li><li>24-well plates (CytoOne<sup>Â®</sup>, catalog number: CC7682-7524)</li><li>Clean, sterile 13 mm coverslips (# 1.5) (Academy, catalog number: NPS16/1818)</li><li>Parafilm<sup>Â®</sup> M (Bemis, catalog number: PM-996)</li><li>Aluminum foil (Caterwrap, catalog number: AKL-300-030M)</li><li>10-cm dishes</li><li>Stable CHIKV cells (see Protocol 1)</li><li>Specific reagents for our cell line (for basic tissue culture), which we derived from the hepatoma cell line HuH-7:<br/><ol class="npwthree"><li>Dulbecco's modified Eagle's medium (Sigma, catalog number: D6429-500ML)</li><li>100% Fetal Calf Serum (Gibco, catalog number: 10500-064)</li><li>Gibco MEM Non-essential amino acids solution (100x), store at 4 Â°C up to 24 months from the date of manufacture (Thermo Fisher Scientific, catalog number: 11140050)</li><li>Gibco HEPES (N-2-hydroxyethylpiperazine-N-2-ethane sulfonic acid) buffer (100x), store at 4 Â°C up to 24 months from the date of manufacture (Thermo Fisher Scientific, catalog number: 15630130)</li></ol></li><li>Trypsin EDTA (Sigma, catalog number: T3924-500ML)</li><li>PBS, Phosphate Buffered Saline (VWR Lifescience, catalog number: E404-100TABS)</li><li>Antibody for immunolabeling of the SNAP-tagged protein<br/><em>Note: We use a rabbit antibody produced in-house that detects CHIKV nsP3 (Remenyi </em><em>et al.</em><em>, 2017 and 2018). </em> </li><li>Goat anti-rabbit IgG (H+L) Cross-Absorbed Secondary Antibody, DyLight<sup>Â®</sup> 405 (Thermo Fisher Scientific, Invitrogen, catalog number: 35551)<br/><em>Note: The combination of Dylight</em><sup><em>Â®</em></sup><em> 405, ZsGreen, and TMR-Star provides effective color separation and sensitivity for three-color imaging.</em> </li><li>SNAP-Cell<sup>Â®</sup> TMR-Star (New England Biolabs, catalog number: S9105S; or as part of the SNAP-Cell<sup>Â®</sup> Starter Kit, catalog number: E9100S)<br/><em>Note: Prepare stock solution as described for SNAP-Cell</em><sup><em>Â®</em></sup><em> 647-SiR.</em> </li><li>SNAP-Cell<sup>Â®</sup> Block (bromothenylpteridine, BTP, New England Biolabs, catalog number: S9106S; or as part of the SNAP-Cell<sup>Â®</sup> Starter Kit, catalog number: E9100S). Storage: -20 Â°C for at least three years dry or three months as a stock solution dissolved in DMSO</li><li>Fixative, 4% Formaldehyde in phosphate buffered saline (PBS), pH 6.9<br/><em>Note: For the detailed protocol how to prepare the fixative, see: </em><a href="https://www.rndsystems.com/resources/protocols/protocol-making-4-formaldehyde-solution-pbs" target="_blank"><em>https://www.rndsystems.com/resources/protocols/protocol-making-4-formaldehyde-solution-pbs</em></a><em>. Briefly, prepare the fixative from Paraformaldehyde powder (Fisher Chemical, catalog number: T353-500). Dissolve paraformaldehyde powder in 1x sterile PBS (made from tablets) (VWR Lifescience, catalog number: E404-100TABS). Adjust the pH to 6.9 with diluted Hydrochloric Acid (HCl) (Fisher Chemical, catalog number: H/1100/PB17). For long-term storage, keep 15-ml aliquots at -20 Â°C. When needed, thaw aliquots (vortex thoroughly to remove any precipitation) and use for each experiment. Store any remaining solution for up to 1 month at 4 Â°C.</em> </li><li>Mounting Medium (ProLong Diamond Antifade Mountant, catalog number: P36965)<br/><em>Note: We use ProLong Diamond because it protects both fluorescent dyes (in our setup: SiR + DyLight</em><sup><em>Â®</em></sup><em> 405) and fluorescent proteins (in our setup: ZsGreen) from fading.</em> </li><li>Complete Cell Culture Media (see Recipes)<br/></li></ol><p class="pptt" id="biaoti38075">### Equipment</p><ol class="npwtwok"><li>Micropipettes<br/><ol class="npwthree"><li>Starlab ErgoOne<sup>Â®</sup> 100-1,000 Âµl Single-Channel Pipette (catalog number: S7110-1000) or Gilson F123602 PIPETMAN Classic Pipet P1000 (Fisher Scientific, catalog number: 10387322)</li><li>Starlab ErgoOne<sup>Â®</sup> 20-200 Âµl Single-Channel Pipette (catalog number: S7100-2200) or Gilson F123615 PIPETMAN Classic Pipet P100 (Fisher Scientific, catalog number: 10442412)</li><li>Starlab ErgoOne<sup>Â®</sup> 2-20 Âµl Single-Channel Pipette (catalog number: S7100-0220) or Starlab ErgoOne<sup>Â®</sup> 0.1-2.5 Âµl Single-Channel Pipette (catalog number: S7100-0125) or Gilson F144801 PIPETMAN Classic Pipet P2 (Fisher Scientific, catalog number: 10635313) or Gilson F123600 PIPETMAN Classic Pipet P20 (Fisher Scientific, catalog number: 10082012)</li></ol></li><li>Metal tweezers with fine tips for lifting and handling glass coverslips (EMS, catalog number: 78316-1)</li><li>Equipment for basic cell culture techniques and aseptic procedures, <em>i.e.</em>,<br/><ol class="npwthree"><li>Biosafety cabinet (<em>e.g.</em>, Thermo Scientific Holten Safe 2010 Model 1.2, catalog number: 8207071100)</li><li>Humidified incubator set to 37 Â°C (Panasonic Incusafe, catalog number: MCO-20AIC)</li></ol><em>Note: Any manufacturer and model of a biosafety cabinet will do, but the chosen biosafety cabinet needs to be appropriate for the containment of cells and viruses. When handling infectious viruses, use the proper facilities, practices, and procedures. Refer to local and international guidelines for laboratory biosafety.</em> </li><li>-20 Â°C freezer (Labcold, catalog number: RLCF1520)</li><li>Water bath (<em>e.g.</em>, Clifton, catalog number: NE2-8D; however, any water bath set to 37 Â°C may be used)</li><li>Table-top microcentrifuge, 4 Â°C to room temperature (RT), max speed â¥ 12,000 <em>x g</em> (Eppendorf, model: 5424R)<br/><em>Note: For spinning down precipitate that may form during extended storage of SNAP probes.</em> </li><li>Safety glasses (any manufacturer or model that protects users from formaldehyde splashes will do)</li><li>Confocal laser scanning microscope. For example, a customized ZEISS LSM 880 system that includes the following essential components (for a similar setup, contact your local ZEISS representative for exact ordering information as product codes may differ from customer to customer and country to country):<br/><ol class="npwthree"><li>Axio Imager Z2 stand, motorized (upright system), product ID: 430000-9902-000</li><li>Motorized Stage, Scanning Stage 130x85 STEP, product ID: 432033-9902-00</li><li>Mounting Frame 160x116 f/ Slides 76x26, product ID: 432315-0000-000</li><li>Scan module LSM 880, product ID: 000000-1994-956</li><li>Support f/scan module LSM (Imager Tube), product ID: 000000-1265-660</li><li>Stepper motor control f, 2 Axes SMC2009, product ID: 432929-9011-000</li><li>Real-time controller standard, product ID: 000000-2031-918</li><li>Objective C PApo 63x/1.4 Oil DIC UV-IR, product ID: 421782-9900-799</li><li>488 Vis Laser, Laser Argon Multiline 25 mW, product ID: 000000-2086-081</li><li>561 Vis Laser, Laser 561 nm for LSM 710, product ID: 00000-1410-117</li><li>633 Vis Laser, Laser Rack LSM 880 incl. 633 Laser, product ID: 000000-2085-478</li><li>Airyscan SR module GaAsP for LSM, product ID: 000000-2058-580</li><li>Emission filter BP495-550 + LP570 for Airyscan, product ID: 000000-2070-488 </li><li>Emission filter BP570-620 + LP645 for Airyscan, product ID: 000000-2070-489</li><li>Double BP 420-480 + BP495-620 for Airyscan, product ID: 000000-2095-049</li><li>Double BP 465-505 + LP525 for Airyscan, product ID: 000000-2095-051</li><li>Double BP 420-480 + LP605 for Airyscan, product ID: 000000-2095-052 </li><li>User PC advanced Z55A highend, product ID: 000000-2142-968</li><li>âAiryscan Fastâ illumination module upgrade for 1x LSM 880 system</li></ol><em>Notes: </em><br/><ol class="npwfour"><li><em>The above list is not exhaustive but only lists essential components that we have found useful for our protocols. Consult with ZEISS about additional components to complete the microscope system (</em><em>e.g.</em><em>, joystick for stage, hardware license keys, beam splitter, switching mirror, and nosepiece). </em> </li><li><em>Follow the manufacturerâs recommendations for applying all necessary Airyscan settings within the ZEN microscope software. Our LSM 880 Airyscan microscope could provide a maximum lateral resolution of 140 nm and axial resolution of 400 nm for a fluorophore emitting at 480 nm (images processed with ZEN Black software). The resolution can increase even further to 120 nm XY and 350 nm Z resolution with the use of ZEN Blue software. </em> </li><li><em>Our original system (LSM 880 with Airyscan) was retroactively upgraded to an âAiryscan Fastâ system, whereas todayâs systems can be customized with an âAiryscan Fastâ module from the start. The âAiryscan Fastâ illumination system allows simultaneous illumination of 4 pixels simultaneously to allow very fast and gentle imaging of samples, including the option to scan at super-resolution with 1.5x resolution improvement in XY &amp; Z, imaging at up to 19 images/second with 512 x 512 pixels, 27 images per second at 480 x 480, and 6 images per second at 1024 x 1024.</em> </li></ol></li></ol><p class="pptt" id="biaoti38076">### Software</p><ol class="npwtwok"><li>ZEISS ZEN software (ZEN 2.3 system HWL for FAST Airyscan)<br/><em>Note: ZEISS ZEN software drives all components of the LSM 880 system, including setup of Airyscan imaging. ZEN software can also acquire and process raw super-resolution image datasets; save processed image files as â.cziâ files, which can be exported to bioimaging analysis software (</em><em>e.g.</em><em>, Icy software). Saving in this file format ensures the preservation of all the metadata associated with each imaging experiment. </em> </li><li>Icy bioimaging software<br/><em>Note: Bioimaging analysis software. We recommend the free software </em><a href="http://icy.bioimageanalysis.org/" target="_blank"><em>Icy</em></a><em> to visualize, annotate and quantify bioimaging datasets, which can be imported from ZEISS ZEN software packages (de Chaumont </em><em>et al.</em><em>, 2012). We found that Icy has an intuitive user interface.</em><br/></li></ol><p class="pptt" id="biaoti38077">### Procedure</p><p class="ppzz">We use a published labeling protocol (Bodor <em>et al.</em>, 2012) to analyze ânewâ (<em>i.e.</em>, freshly translated) pools of the SNAP-tagged viral protein (SNAP-nsP3). By combining the labeling approach with a sensitive detection method, ZEISS LSM 880 Airyscan microscopy, we can visualize the re-emergence of ânewâ protein and the intracellular sites where ânewâ proteins accumulate. In the first step (called âquenchâ), a nonfluorescent SNAP-substrate covalently binds to the pool of SNAP-nsP3 present at the onset of an experiment (Figure 3, diagram). After a given amount of time (chase), a second, fluorescent substrate (pulse) labels the cells as described in Protocol 1 (Figure 3, diagram). The pulse only stains the protein pool synthesized during the chase period. Thus, ZEISS LSM 880 Airyscan microscopy will only make this ânewâ (<em>i.e.</em>, freshly translated) pool visible. The total pool of SNAP-nsP3 (pulsed pool + quench pool) can be stained with a standard immunofluorescence assay approach, which reveals the quenched pool that would otherwise remain undetected (Figure 3). <br/><br/><em>Note: Carry out all liquid handling steps that involve live cells or formaldehyde-containing wells inside the biosafety cabinet. Only use sterile pipettors, pipets, micropipettors, microtubes, and tips when working with live cells. Wear safety glasses when handling formaldehyde solutions.</em> </p><ol class="npwtwo"><li>Place sterile coverslips in separate wells of a 24-well plate.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Adjust the total number of wells according to the number of âpost-chaseâ timepoints. Also include a positive control (</em><em>e.g.</em><em>, a well without any SNAP Cell</em><sup><em>Â®</em></sup><em> Block added) and negative control (</em><em>e.g.</em><em>, a well without any chase period). </em> </li><li><em>In this protocol: two post-chase timepoints (3 h, 6 h) and two controls (four wells in total).</em> </li></ol></li><li>Detach stable CHIKV cells from growth surface by adding enough trypsin solution (pre-heated to 37 Â°C in a water bath) to cover the cells that are attached to the surface of the respective culturing vessel (<em>e.g.</em>, 1-2 ml of trypsin solution for 10-cm dishes) and seed stable CHIKV cells in wells from Step 1.<br/>Seeding density will again depend on the total duration of the experiment and the doubling time of cells. We aim for about ~80% confluency at the time of fixation of the 0-h, 3-h, and 6-h samples. Thus, we also seed fewer cells in the 3-h and 6-h wells to account for the longer incubation times compared to the 0-h well (10-15% less in 3-h well and 20-30% less in 6-h well).</li><li>Incubate at 37 Â°C, 5% CO<sub>2</sub> overnight.</li><li>On the next day, take the frozen stock solution of SNAP Cell<sup>Â®</sup> Block from the -20 Â°C freezer and thaw at room temperature. In a 1.5-ml microtube, dilute SNAP Cell<sup>Â®</sup> Block to a final concentration of 2 ÂµM in complete media. Vortex briefly (for â¥ 5 s). Prepare &gt; 200 Âµl per coverslip. Use diluted reagent within the hour.</li><li>Replace media on cells with 200 Âµl of the SNAP Cell<sup>Â®</sup> Block diluted solution (pre-heated to 37 Â°C in a water bath) per well (three wells in total). Do not change media on the fourth well (this will serve as the âpositive controlâ). Incubate for 30 min. </li><li>Wash cells three times with 1 ml of complete media (pre-heated to 37 Â°C in a water bath; this washes away any free substrate, which would interfere with downstream applications). Remove media after the last wash step.</li><li>Add 1 ml of complete media (pre-heated to 37 Â°C in a water bath) to each well and place cells into a tissue-culture incubator for 30 min.</li><li>Wash cells as in Step 6. To prevent the cells from drying out, do not remove the wash media after the last wash step. </li><li>Prepare a new 24-well plate and add 1 ml of 4% formaldehyde fixative to two of the wells. </li><li>Use forceps to transfer one of the âquenchedâ coverslips and the âunblockedâ coverslip to the new plate and submerge in formaldehyde (make sure the side with the layer of cells remains up). The fixed cells on the âquenchedâ coverslip will serve as the âno chaseâ control. The cells on the âunblockedâ coverslip will serve as a positive control for SNAP-reagent staining.</li><li>Place the original plate, which contains the remaining coverslips, back into the tissue-culture incubator. Incubate cells for the desired chase period of 3 h. </li><li>Place the new plate containing fixed coverslips at 4 Â°C for storage until all coverslips for all chase periods have been collected. Seal edges of the plate during storage to avoid excessive evaporation (strips of Parafilm M work well).</li></ol><p class="ppzz"><em>Note: If the quench was indeed complete, no labeling should occur during the âpulseâ period. If Airyscan microscopy can still detect unquenched SNAP-tagged protein, this indicates that the available SNAP Cell</em><sup><em>Â®</em></sup><em> Block reagent did not fully quench the pre-existing pool. Repeat experiments with an increased concentration of SNAP Cell</em><sup><em>Â®</em></sup><em> Block or with a prolonged incubation time until experimental conditions lead to complete quenching of SNAP-tagged protein.</em> </p><ol class="npwtwo" start="13"><li>After three hours, transfer one coverslip to formaldehyde-containing well as described in Step 10 (remove the plate from 4 Â°C storage and add 1 ml of 4% formaldehyde to a third well). The fixed cells on this coverslip will serve as the â3-h chaseâ sample. </li><li>Place the original plate, which contains the remaining coverslip, back into the tissue culture incubator. Incubate cells for another three hours. Place plate that holds fixed âno chaseâ and â3-h chaseâ coverslips back to 4 Â°C for storage.</li><li>Repeat Steps 13 and 14 (transfer coverslips into a fourth, formaldehyde-containing well). The fixed cells on this last coverslip will serve as the â6-h chaseâ sample. Incubate for 30 min at room temperature or for 1-2 h at 4 Â°C to complete the fixation process of â6-h chaseâ sample.</li><li>In the biosafety cabinet, wash âNo chaseâ, âNo quenchâ, â3-h chaseâ, and â6-h chaseâ controls three times with PBS to remove the formaldehyde fixative.<br/><em>Note: At this point, users also have the option to handle 24-well plates outside a biosafety cabinet as the cells have already undergone chemical fixation. However, we still prefer to carry out Steps 17, 19, and 20 inside a biosafety cabinet to avoid the contamination of stock solutions and cell culture media.</em> </li><li>Take the frozen stock solution of SNAP Cell<sup>Â®</sup> TMR-Star from -20 Â°C freezer and thaw at room temperature. In a 1.5-ml microtube, dilute SNAP-Cell<sup>Â®</sup> TMR-Star; perform a 1:600 (final concentration of 1 ÂµM) dilution in complete cell culture media. Prepare at least 200 Âµl per well to cover the entire coverslip. Vortex briefly (for â¥ 5 s) or pipet the diluted labeling solution up and down (ten times).<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>This protocol should also be compatible with SNAP-Cell</em><sup><em>Â®</em></sup><em> 647 SiR. Use the same labeling conditions as described in Protocol 1 (</em><em>i.e.</em><em>, 1:1,000 dilution, 15 min incubation).</em> </li><li><em>Recipe for four coverslips in 24-well plate, scale according to the number of wells:</em><br/><em> 998.4 Âµl Complete Cell Culture Media</em><br/><em> 1.6 Âµl SNAP-Cell</em><sup><em>Â®</em></sup><em> TMR-Star</em> </li></ol></li><li>Spin diluted labeling solution for 5 min at maximum speed (â¥ 12,000 <em>x g</em>) to remove possible insoluble fluorescent debris. Take care not to disturb the pellet when removing supernatant (which may be invisible).</li><li>Replace the medium on stable CHIKV cells with 200 Âµl of SNAP-tag labeling medium. Incubate for 15 min at 37 Â°C, 5% CO<sub>2</sub>. Protect samples from excessive light exposure during Steps 20-23 (<em>e.g.</em>, wrap 24-well plate in aluminum foil).<br/><em>Note: We found that a 15-min incubation gave us optimal labeling. According to the manufacturer, optimal reaction times range from 5 to 30 min, respectively, depending on experimental conditions and expression levels of the SNAP-tagged protein.</em> </li><li>Wash the cells three times each with 1 ml of tissue culture medium containing serum (pre-heated to 37 Â°C).</li><li>Use a preferred immunofluorescence assay protocol to stain the pool of total tagged protein. At this point, users can handle 24-well plates outside the biosafety cabinetâcritical reagents in our case: primary antibodies from rabbit antiserum against nsP3 and dye-conjugated secondary antibodies (anti-rabbit DyLight<sup>Â®</sup> 405). </li><li>Mount the immunolabeled coverslips in preferred mounting medium (<em>e.g.</em>, ProLong Diamond Antifade Mountant):<br/><ol class="npwthree"><li>Place a droplet of mountant on a glass slide, remove immunostained coverslips from 24-well plate using forceps, and slowly lower coverslip (cell side down) onto the droplet. </li><li>Use tissue or filter paper to wipe off any excess mountant. We usually fit two to three coverslips on a standard glass slide.</li></ol></li><li>Allow mountant to cure at room temperature and in the dark (overnight or longer). We store cured slides at 4 Â°C. </li><li>Image with ZEISS LSM 880 Airyscan microscopy<br/><ol class="npwthree"><li><em>SNAP Cell</em><sup><em>Â®</em></sup><em> TMR-Star should have an excitation maximum at 554 nm and an emission maximum at 580 nm. Set up three-color imaging with a blue, green, and red channel. Refer to ZEN software manuals for set-up of ZEISS LSM 880 Airyscan microscopy.</em> </li><li><em>Upon completion of image acquisition and processing, we use Icy bioimaging software to open saved and processed Airyscan data, which we saved in the .czi file format. Figure 3 shows representative images from a three-color imaging experiment, including screenshots of the Icy histogram viewer for each channel. </em><br/><div style="white-space:nowrap;"><br/></div></li></ol><img alt="" class="layerphoto" height="206" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190218/20190218002100_8821.jpg" width="150"/><br/><strong>Figure 3. Airyscan microscopy after chase period reveals pools of unblocked viral protein.</strong> Representative images from a quench-pulse-chase experiment. Note that the co-distribution of the total nsP3 pool (stained with anti-nsP3 antibody) and the new pool of nsP3 (stained with TMR-Star) suggested a lack of spatial separation of old and new pools of SNAP-nsP3. We set up the microscope for three-color imaging in the blue, green, and red channels. We adjusted image contrast within the Icy platform by dragging the adjustable bounds of the histogram viewer (marked by arrows), which enhanced the contrast in the selected channel without altering the data. We chose a viewing range that provided the best contrast for SNAP-nsP3 channel at the 3-h and 6-h timepoints. We also used the same viewing range to display the data from the positive and negative controls (âHistogram rangeâ window, marked by asterisk *, view range minimum of 0 and maximum of 8469.0 pixel intensity values). Note that by applying the colormap âFireâ (within Icy software) we could better display low-intensity granular structures. </li></ol><p class="pptt" id="biaoti38078">### Recipes</p><ol class="npwtwok"><li>Complete Cell Culture Media<br/>500 ml Dulbecco's modified Eagle's medium<br/>56.5 ml Fetal Calf Serum <br/>5.6 ml MEM Nonessential amino acids solution <br/>5.6 ml HEPES <br/></li></ol><p class="pptt" id="biaoti_no">### Protocol 3: Tracking SNAP-tagged viral protein assemblies in cell lysates with ZEISS LSM 880 Airyscan microscopyMaterials and Reagents</p><ol class="npwtwok"><li>Micropipette tips, serological pipettes, pipette aids, and microtubes for liquid handling:<br/><ol class="npwthree"><li>TipOne<sup>Â®</sup> 1,000 Âµl XL filter tips (catalog number: S1122-1830)</li><li>TipOne<sup>Â®</sup> 200 Âµl filter tips (catalog number: S1120-8810)</li><li>TipOne<sup>Â®</sup> 20 Âµl filter tips (catalog number: S1120-1810) or SARSTEDT pipette tip 10 Âµl (catalog number: 70.1130.600)</li><li>Fisherbrand<sup>TM</sup> 5 ml serological pipets (catalog number: 13-676-10H)</li><li>Fisherbrand<sup>TM</sup> 10 ml serological pipets (catalog number: 13-676-10J)</li><li>Fisherbrand<sup>TM</sup> 25 ml serological pipets (catalog number: 13-676-10K)</li><li>Drummond Pipet-Aid XL (catalog number: 4-000-205)</li><li>1.5-ml Microtube (SARSTEDT, catalog number: 72.690.001)</li><li>15 ml Centrifuge Tubes, Conical, Sterile (Starlab, catalog number: E1415-0200)</li></ol></li><li>Parafilm<sup>Â®</sup> M (Bemis, catalog number: PM996)</li><li>Aluminum foil (Caterwrap, catalog number: AKL-300-030M)</li><li>Moistened paper (<em>e.g.</em>, cut sheets of blot absorbent filter paper, Biorad, catalog number: 1703965)</li><li>10-cm Petri dish (Corning, catalog number: 430167)</li><li>Ibidi 2-well Âµ-slide, an all-in-one chamber slide with polymer coverslip and ibiTreat surface for optimal cell adhesion (Ibidi, catalog number: 80286)<br/><em>Note: The IbiTreat surface modification makes the polymer coverslip surface hydrophilic. Alternative approaches to increase the adhesiveness of coverslip surfaces (</em><em>i.e.</em><em>, Poly-Lysine treatment) and coated glass coverslips may be used (Wheeler </em><em>et al.</em><em>, 2017). </em> </li><li>Greiner CELLSTAR multiwall culture plates, 6-well (Merck/Sigma Aldrich, catalog number: Greiner 657160)</li><li>Stable CHIKV cells (see Protocol 1)</li><li>Trypsin EDTA (Sigma, catalog number: T3924-500ML)</li><li>Materials and reagents for basic tissue culture (see Protocol 1 and 2)</li><li>SNAP-Cell<sup>Â®</sup> TMR-Star (New England Biolabs, catalog number: S9105S; or as part of the SNAP-Cell<sup>Â®</sup> Starter Kit, catalog number: E9100S), Prepare stock solution as described for SNAP-Cell<sup>Â®</sup> 647-SiR</li><li>PBS, Phosphate Buffered Saline (VWR Lifescience, catalog number: E404-100TABS)<br/></li><li>4% Formaldehyde (see Protocol 2 Step 15)</li><li>Triton X-100 (Sigma, catalog number: T924-500ml)</li><li>KCl (Fisher Chemical, catalog number: P/4240/53)</li><li>NaCl (Fisher Chemical, catalog number: S/3160/60)</li><li>Magnesium chloride (MgCl<sub>2</sub>) hexahydrate 99.0-101.0%, VWR Chemicals, catalog number: 25108.260)</li><li>Glycerol (Fisher Chemical, catalog number: G/0650/17)</li><li>Piperazine-N,Nâ²-bis(2-ethanesulfonic acid) (PIPES) (Sigma, catalog number: P1851-100G)</li><li>Leupeptin, hemisulfate salt â¥ 85% by HPLC (Sigma, catalog number: L8511-5MG)</li><li>Pepstatin A (Sigma, catalog number: P5318-5MG)</li><li>Aprotinin, from bovine lung (Sigma, catalog number: A6279-5ML)</li><li>AEBSF (Pefabloc, Sigma, catalog number: 76307-100MG)</li><li>NaOH (Fisher Chemical, catalog number: S/4920/60)</li><li>Home-made Glasgow Lysis Buffer (GLB) containing protease inhibitors (see Recipes)<br/><ol class="npwthree"><li>1x GLB, combine the stock solutions, Glycerol, Triton-X, and protease inhibitors (see Recipes)</li><li>This protocol uses our laboratoryâs preferred lysis buffer. We have not tried other lysis buffers that use NP-40 or SDS as detergents. However, note that 0.5% NP-40 is a component of the stress granule lysis buffer used to prepare cell lysates for stress granule core isolation (Wheeler <em>et al.</em>, 2017). </li></ol></li><li>Stock solutions for GLB (see Recipes)</li><li>Stock solutions of protease inhibitors (see Recipes)</li><li>1x GLB (see Recipes)</li><li>Tissue culture medium containing serum (see Recipes)<br/></li></ol><p class="ppzz"><em>Note: For #14-24, alternatives of equal purity are also suitable.</em> </p><p class="pptt" id="biaoti38079">### Equipment</p><ol class="npwtwok"><li>Micropipettes<br/><ol class="npwthree"><li>Starlab ErgoOne<sup>Â®</sup> 100-1,000 Âµl Single-Channel Pipette (catalog number: S7110-1000) or Gilson F123602 PIPETMAN Classic Pipet P1000 (Fisher Scientific, catalog number: 10387322)</li><li>Starlab ErgoOne<sup>Â®</sup> 20-200 Âµl Single-Channel Pipette (catalog number: S7100-2200) or Gilson F123615 PIPETMAN Classic Pipet P100 (Fisher Scientific, catalog number: 10442412)</li><li>Starlab ErgoOne<sup>Â®</sup> 2-20 Âµl Single-Channel Pipette (catalog number: S7100-0220) or Starlab ErgoOne<sup>Â®</sup> 0.1-2.5 Âµl Single-Channel Pipette (catalog number: S7100-0125) or Gilson F144801 PIPETMAN Classic Pipet P2 (Fisher Scientific, catalog number: 10635313) or Gilson F123600 PIPETMAN Classic Pipet P20 (Fisher Scientific, catalog number: 10082012)</li></ol></li><li>Cell scrapers (Fisher Scientific, catalog number: 08-100-241)</li><li>Equipment for basic cell culture techniques and aseptic procedures, <em>i.e</em>.,<br/><ol class="npwthree"><li>Biosafety cabinet (<em>e.g.</em>, Thermo Scientific Holten Safe 2010 Model 1.2, catalog number: 8207071100) </li><li>Humidified incubator set to 37 Â°C (Panasonic Incusafe, catalog number: MCO-20AIC)</li></ol><em>Note: Any manufacturer and model of a biosafety cabinet will do, but the chosen biosafety cabinet needs to be appropriate for the containment of cells and viruses. When handling infectious viruses, use proper facilities, practices, and procedures. Refer to local and international guidelines for laboratory biosafety.</em> </li><li>-20 Â°C freezer (Labcold, catalog number: RLCF1520)</li><li>Water bath (<em>e.g.</em>, Clifton, catalog number: NE2-8D; however, any type of water bath set to 37 Â°C is appropriate)</li><li>Table-top microcentrifuge, 4 Â°C to room temperature (RT), max speed â¥ 12,000 <em>x g</em> (Eppendorf, 5424R)<br/><em>Note: For spinning down precipitate that may form during extended storage of SNAP probes.</em> </li><li>Fume hood (any manufacturer or model that will protect users from formaldehyde fumes will do)</li><li>Confocal laser scanning microscope. For example, a customized ZEISS LSM 880 system that includes the following essential components (for a similar setup, contact your local ZEISS representative for exact ordering information as product codes may differ from customer to customer and country to country):<br/><ol class="npwthree"><li>Axio Imager Z2 stand, motorized (upright system), product ID: 430000-9902-000 </li><li>Motorized Stage, Scanning Stage 130x85 STEP, product ID: 432033-9902-00</li><li>Mounting Frame 160x116 f/ Slides 76x26, product ID: 432315-0000-000</li><li>Scan module LSM 880, product ID: 000000-1994-956</li><li>Support f/scan module LSM (Imager Tube), product ID: 000000-1265-660</li><li>Stepper motor control f, 2 Axes SMC2009, product ID: 432929-9011-000</li><li>Real-time controller standard, product ID: 000000-2031-918</li><li>Objective C PApo 63x/1.4 Oil DIC UV-IR, product ID: 421782-9900-799</li><li>488 Vis Laser, Laser Argon Multiline 25 mW, product ID: 000000-2086-081</li><li>561 Vis Laser, Laser 561nm for LSM 710, product ID: 00000-1410-117</li><li>633 Vis Laser, Laser Rack LSM 880 incl. 633 Laser, product ID: 000000-2085-478</li><li>Airyscan SR module GaAsP for LSM, product ID: 000000-2058-580</li><li>Emission filter BP495-550 + LP570 for Airyscan, product ID: 000000-2070-488 </li><li>Emission filter BP570-620 + LP645 for Airyscan, product ID: 000000-2070-489</li><li>Double BP 420-480 + BP495-620 for Airyscan, product ID: 000000-2095-049</li><li>Double BP 465-505 + LP525 for Airyscan, product ID: 000000-2095-051</li><li>Double BP 420-480 + LP605 for Airyscan, product ID: 000000-2095-052 </li><li>User PC advanced Z55A highend, product ID: 000000-2142-968</li><li>Transmitted light detector T-PMT, product ID: 000000-2014-999</li><li>âAiryscan Fastâ illumination module upgrade for 1x LSM 880 system </li></ol><em>Notes:</em><br/><ol class="npwfour"><li><em>The above list is not exhaustive but only lists essential components that we have found useful for our protocols. Consult with ZEISS about additional components to complete the microscope system (</em><em>e.g.</em><em>, joystick for stage, hardware license keys, beam splitter, switching mirror and nosepiece).</em> </li><li><em>Follow the manufacturerâs recommendations for applying all necessary Airyscan settings within the ZEN microscope software. Our Airyscan microscope could provide a maximum lateral resolution of 140 nm and axial resolution of 400 nm for a fluorophore emitting at 480 nm (images processed with ZEN Black software). Airyscan resolution can increase even further to 120 nm XY and 350 nm Z resolution with the use of ZEN Blue software. </em> </li><li><em>Our original system (LSM 880 with Airyscan) was retroactively upgraded to an âAiryscan Fastâ system, whereas todayâs systems can be customized with an âAiryscan Fastâ module from the start. The âAiryscan Fastâ illumination system allows simultaneous illumination of 4 pixels simultaneously to allow very fast and gentle imaging of samples, including the option to scan at super resolution with 1.5x resolution improvement in XY &amp; Z, imaging at up to 19 images/second with 512 x 512 pixels, 27 images per second at 480 x 480, and 6 images per second at 1024 x 1024.</em> </li></ol></li></ol><p class="pptt" id="biaoti38080">### Software</p><ol class="npwtwok"><li>ZEISS ZEN software (ZEN 2.3 system HWL for FAST Airyscan)<br/><em>Note: ZEISS ZEN software drives all components of the LSM 880 system, including setup of Airyscan imaging. ZEN software can also acquire and process raw super-resolution image datasets; save processed image files as â.cziâ files, which can be exported to bioimaging analysis software (</em><em>e.g.</em><em>, Icy software). Saving in this file format ensures the preservation of all the metadata associated with each imaging experiment. </em> </li><li>Icy bioimaging software<br/><em>Note: Bioimaging analysis software. We recommend the free software </em><a href="http://icy.bioimageanalysis.org/" target="_blank"><em>Icy</em></a><em> to visualize, annotate and quantify bioimaging datasets, which can be imported from ZEISS ZEN software packages (de Chaumont </em><em>et al.</em><em>, 2012). We found that Icy has an intuitive user interface.</em><br/></li></ol><p class="pptt" id="biaoti38081">### Procedure</p><p class="ppzz">This protocol uses fluorescence light microscopy to reveal stable assemblies of SNAP-nsP3 protein that persist in cell lysates. It is partly based on an isolation protocol (Wheeler <em>et al.</em>, 2017) that has been used to determine the proteome and substructure of stress granules (Jain <em>et al.</em>, 2016). Our protocol adds additional flexibility in labeling (by using the SNAP labeling system) and detection (by using the sensitive ZEISS LSM 880 Airyscan confocal imaging system). <br/><br/><em>Note: Carry out all liquid handling steps that involve live cells inside the biosafety cabinet. Only use sterile pipettors, pipets, micropipettors, microtubes, and tips. After cell lysis, users can handle samples outside the biosafety cabinet. Wear safety glasses when handling formaldehyde solutions.</em> </p><ol class="npwtwok"><li>Detach stable CHIKV cells with trypsin and seed in at least one well of a 6-well microtiter plate. Also, detach and seed naÃ¯ve cells, which do not express SNAP-tagged proteins, in at least one well; these cells will serve as a negative control.</li><li>Incubate cells under standard growth conditions (<em>i.e.</em>, 37 Â°C at 5% CO<sub>2</sub>) overnight.</li><li>On the next day, take frozen SNAP Cell<sup>Â®</sup> TMR-Star from -20 Â°C freezer and thaw at room temperature. In a 1.5-ml microtube, dilute SNAP-Cell<sup>Â®</sup> TMR-Star 1:600 (final concentration of 1 ÂµM) dilution in complete cell culture media. Prepare at least 1 ml per well to cover the entire area of the well. Vortex briefly (for â¥ 5 s) or pipet the diluted labeling solution up and down (ten times). We have only validated this protocol with the SNAP-Cell<sup>Â®</sup> TMR-Star reagent. However, the protocol should also be compatible with SNAP-Cell<sup>Â®</sup> 647 SiR. Use the same labeling conditions as described in Protocol 1 (<em>i.e.</em>, 1:1,000 dilution, 15 min incubation).<br/><em>Note: Recipe for two dishes, scale accordingly for additional dishes:</em><br/><em>           1996.8 Âµl Complete Cell Culture Media</em><br/><em>           3.2 Âµl SNAP-Cell</em><sup><em>Â®</em></sup><em> TMR-Star</em> </li><li>Spin diluted labeling solution for 5 min at maximum speed (â¥ 12,000 <em>x g</em>) to remove possible insoluble fluorescent debris. Take care not to disturb the pellet when removing supernatant (which may be invisible).</li><li>Replace the medium on stable CHIKV cells with 1 ml of SNAP-tag labeling medium (pre-heated to 37 Â°C in a water bath). Also, replace medium in the ânegative controlâ well. Incubate for 15 min at 37 Â°C, 5% CO<sub>2</sub>.<br/><em>Note: We found that a 15-min incubation gave us optimal labeling. According to the manufacturer, optimal reaction times range from 5 to 30 min, respectively, depending on experimental conditions and expression levels of the SNAP-tagged protein.</em> </li><li>Wash the cells three times each with 2 ml of tissue culture medium containing serum (pre-heated to 37 Â°C in a water bath). Do not remove media after the final wash.</li><li>Place cells back into the humidified incubator for another 45-60 min.<br/>We use an extended wash-out period (compared to Protocol 1) to reduce the non-specific background-binding further. Moreover, we do not add ProLong Gold Live in this protocol, as live-cell imaging is only limited to an optional quality-control-step in Step 8.</li><li>Optional: Confirm fluorescent staining of SNAP-nsP3 with microscopy<br/><ol class="npwthree"><li>A basic widefield fluorescent microscope equipped with a 10x or 20x objective is sufficient to evaluate the quality of SNAP-labeling through the microscope eyepiece. If documentation of staining quality is needed, take pictures with a connected camera. Also, confirm the absence of staining in the well that contains negative control. </li><li>We recommend this step when testing labeling conditions for the first time. Perform imaging quickly and proceed to Step 9 as soon as possible to limit the deterioration of cell health. If this is not logistically possible, we recommend proceeding to Step 9 directly.</li></ol></li><li>Replace complete cell media with 1 ml PBS and use separate cell scrapers to detach cells from growth area in each well. Transfer the resulting cell suspension to a 1.5 ml microtube. Pellet cells at 1,500 <em>x g</em>, 3 min at room temperature. Remove supernatant. <br/><em>Optional pause point: We freeze pellets at â¤ -20 Â°C if we want to carry out lysis at a later time.</em> </li><li>Lyse pellets by adding 300 Âµl of ice-cold Glasgow Lysis buffer containing protease inhibitors (see Recipes). Ensure complete re-suspension of pellet through repeated pipetting, flicking the tube, or vortexing. If pellet came from the freezer, thaw on ice for 5 min before adding lysis buffer. </li><li>Vortex Lysates for 30 s. Place on ice for 30 s. We return samples to ice in between vortexing cycles to prevent an extended incubation period at room temperature. Alternatively, lysates can be vortexed in a cold room.</li><li>Repeat Step 11 three times.</li><li>Spin at 850 <em>x g</em> for 5 min at 4 Â°C to remove remaining cellular debris.<br/><em>Note: We do not further purify assemblies of SNAP-nsP3 after this step but instead use the crude supernatant in the microscopic analysis. It would be interesting to test whether subsequent centrifugation at 18,000 </em><em>x g</em><em> can pellet assemblies of SNAP-nsP3; this step is essential in the isolation of a pure population of stress granule cores (Wheeler </em><em>et al.</em><em>, 2017). After this step, users can handle cell lysates outside a biosafety cabinet.</em> </li><li>Transfer the entire volume of supernatant to a two-well Ibidi chambered plastic slide. Discard pelleted cellular debris. <br/><em>Note: We have noticed that the supernatant may appear turbid at this step. Proteins that are components of RNPs are known to undergo liquid-liquid phase transitions, and turbidity is due to the formation of small protein-rich droplets (Molliex </em><em>et al.</em><em>, 2015). However, we have not yet tested whether purified SNAP-nsP3 can form similar droplets in solution.</em> </li><li>Incubate overnight at 4 Â°C in a humidified environment in the dark.<br/><ol class="npwthree"><li>To obtain humidified conditions, we cover the Ibidi slide with the supplied plastic cover and place the slide in a 10-cm Petri dish that also contains moistened paper (<em>e.g.</em>, cut sheets of Western blotting filter). </li><li>We further seal dishes with Parafilm M and wrap them in aluminum foil to prevent light exposure. We incubate overnight to give assemblies of SNAP-nsP3 enough time to settle to the bottom of the chamber. </li><li>Future experiments may determine the minimum duration for deposition of SNAP-nsP3 to the bottom of the chambered slide; this information would be useful for reducing the overall time required to complete Protocol 3. </li></ol></li><li>In a fume hood, add 1 ml of 4% formaldehyde to wells containing cell lysates. Incubate for one hour at room temperature. <br/><em>Note: Slowly dispense formaldehyde with a micropipette. We add this step to fix assemblies of SNAP-nsP3; fixation reduces the likelihood that the assemblies detach from the well surface during subsequent washing steps. </em> </li><li>Wash wells three times with PBS. Ensure that wells do not dry out in between washes. </li><li>Transfer slide to microscopy system of choice:<br/>As described in Protocol 2, we use the ZEISS LSM 880 imaging system operated in Airyscan mode. Use the appropriate filter settings for TMR-Star, which should have an excitation maximum at 554 nm and an emission maximum at 580 nm. Also, acquire an image in brightfield mode. This image can provide an additional record of the examined biostructure.</li><li>We recommend taking at least three images for each experiment: <br/>We acquire images with the 63x objective. Export processed Airyscan files to Icy bioimaging software. Adjust the image contrast within the Icy software by dragging the boundaries of the viewing range in the softwareâs histogram viewer (see Figure 4, arrows). We prefer viewing the SNAP-nsP3 channel with the âFireâ colormap. Use Icyâs controls to zoom into regions-of-interest. See Figure 4 for representative images. <br/><br/><img alt="" class="layerphoto" height="231" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190218/20190218002118_2685.jpg" width="150"/><br/><strong>Figure 4. Analysis of cell lysates with Airyscan microscopy. </strong>We processed samples according to Protocol 3 and detected granular structures containing TMR-Star-labelled SNAP-nsP3. We chose the âFireâ colormap in Icy to display signals from the TMR-Star staining. We also adjusted image contrast by dragging the adjustable bounds of the histogram viewer (marked by arrows). We acquired images at a zoom factor of 1 with a 63x objective. During image processing, we also magnified regions-of-interest (ROI) by increasing the digital zoom within the Icy software (<em>i.e.</em>, by a factor of 3.3 for ROI1 and 20 for ROI2). Lastly, we also acquired a brightfield channel to provide a reference and reveal all contrast-producing structures present in the lysate. <br/></li></ol><p class="pptt" id="biaoti38082">### Recipes</p><ol class="npwtwok"><li>Home-made Glasgow Lysis Buffer (GLB) with protease inhibitors, 5 ml volume<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190218/20190218002156_0631.jpg" width="600"/><br/><br/></li><li>Stock solutions for GLB<br/>Add the appropriate amount of solids to distilled H<sub>2</sub>O to make stock solutions of PIPES (adjust pH to 7.2 with 3 M NaOH), KCl, NaCl, MgCl<sub>2</sub> at concentration listed in the table. Store these stock solutions at 4 Â°C.</li><li>Stock solutions of protease inhibitors, store in the following manner: <br/>Leupeptin 1 mg/ml at -20 Â°C<br/>Pepstatin A 1 mg/ml at -20 Â°C<br/>Aprotinin 2 mg/ml at 4 Â°C<br/>AEBSF (Pefabloc) 100 mM at 4 Â°C</li><li>1x GLB<br/>Combine the stock solutions, Glycerol, Triton-X, and protease inhibitors according to the table. Adjust volume to 5 ml with distilled H<sub>2</sub>O and pH to 7.2 with 3 M NaOH<br/><em>Note: Use fresh 1x GLB for each experiment.</em> </li><li>Tissue culture medium containing serum<br/>500 ml Dulbecco's modified Eagle's medium<br/>56.5 ml Fetal Calf Serum <br/>5.6 ml MEM Nonessential amino acids solution <br/>5.6 ml HEPES <br/></li></ol><p class="pptt" id="biaoti38083">### Data analysis</p><p class="ppzz">We consider the unprocessed image files that microscope users save in either the Nikon NIS Elements AR software (.nd2 file format) or ZEISS ZEN software (.czi file format) to be ârawâ data. Thus, data analysis in these protocols was for qualitative purposes and made up exclusively of image visualization and digital processing of acquired images (<em>e.g.</em>, adjusting brightness, contrast, and pseudo-colors) in either Nikon NIS Elements AR software or Icy bioimaging software. Although it is also possible to extract quantitative data from these images, our initial method development, which focused on the application of new labeling and imaging methods, did not include these types of bioimage-informatics approaches. Future studies may benefit greatly from incorporating rigorous bioimage-informatics techniques to the protocol described here.</p><p class="pptt" id="biaoti38084">### Acknowledgments</p><p class="ppzz">A Wellcome Trust Investigator Award funded this work (WT 096670, awarded to Mark Harris). Purchase of shared equipment was made possible by a Wellcome Trust Multi-user equipment award (Zeiss LSM 880 instrument, WT104918MA, âMultifunctional imaging of living cells for biomedical sciencesâ). The funders had no role in study design, data collection, and interpretation, or the decision to submit the work for publication. We thank Dr. Sally Boxall and the Bio-imaging Facility within the Faculty of Biological Sciences of the University of Leeds for access and help with Airyscan microscopes. The authors acknowledge Gina Gamble and Dr. Kate Lewis for their help using the Nikon Ti2-E Inverted microscope. We recognize the previous study by Bodor <em>et al.</em> (2012) which described the pulse-chase and quench-pulse-chase approaches. We are grateful to Mark B. Carascal and Dr. Samuel Ko for comments during the revision of the manuscript.</p><p class="pptt" id="biaoti38085">### Competing interests</p><p class="ppzz">The authors declare that no conflicts of interest or competing interests exist.</p><p class="pptt" id="biaoti38086">### References</p><ol class="npwtwok"><li>Bodor, D. L., Rodriguez, M. G., Moreno, N. and Jansen, L. E. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23129118" target="_blank">Analysis of protein turnover by quantitative SNAP-based pulse-chase imaging.</a> <em>Curr Protoc Cell Biol</em> Chapter 8: Unit8.8. </li><li>Cole, N. B. (2014). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=24510614" target="_blank">Site-specific protein labeling with SNAP-tags.</a> <em>Curr Protoc Protein Sci</em> 73: Unit 30.1. </li><li>de Chaumont, F., Dallongeville, S., Chenouard, N., Herve, N., Pop, S., Provoost, T., Meas-Yedid, V., Pankajakshan, P., Lecomte, T., Le Montagner, Y., Lagache, T., Dufour, A. and Olivo-Marin, J. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743774" target="_blank">Icy: an open bioimage informatics platform for extended reproducible research.</a> <em>Nat Methods</em> 9(7): 690-696. </li><li>Frigault, M. M., Lacoste, J., Swift, J. L. and Brown, C. M. (2009). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=19261845" target="_blank">Live-cell microscopyâtips and tools.</a> <em>J Cell Sci </em>122(Pt 6): 753-767. </li><li>Huff, J., Bergter, A., Birkenbeil, J., Kleppe, I., Engelmann, R. and Krzic, U. (2017). <a href="https://www.nature.com/articles/nmeth.f.404" target="_blank">The new 2D Superresolution mode for ZEISS Airyscan.</a> <em>Nat Methods </em>14: 1223.</li><li>Huff, J. (2015). <a href="https://www.nature.com/articles/nmeth.f.388" target="_blank">The Airyscan detector from ZEISS: confocal imaging with improved signal-to-noise ratio and super-resolution.</a> <em>Nat Methods </em>12: 1205.</li><li>Jain, S., Wheeler, J. R., Walters, R. W., Agrawal, A., Barsic, A. and Parker, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26777405" target="_blank">ATPase-modulated stress granules contain a diverse proteome and substructure.</a> <em>Cell </em>164(3): 487-498. </li><li>Kedersha, N., Stoecklin, G., Ayodele, M., Yacono, P., Lykke-Andersen, J., Fritzler, M. J., Scheuner, D., Kaufman, R. J., Golan, D. E. and Anderson, P. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15967811" target="_blank">Stress granules and processing bodies are dynamically linked sites of mRNP remodeling.</a> <em>J Cell Biol </em>169(6): 871-884. </li><li>Kedersha, N., Tisdale, S., Hickman, T. and Anderson, P. (2008). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=19111193" target="_blank">Real-time and quantitative imaging of mammalian stress granules and processing bodies.</a> <em>Methods Enzymol </em>448: 521-552.</li><li>Keppler, A., Kindermann, M., Gendreizig, S., Pick, H., Vogel, H. and Johnsson, K. (2004a). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=15003606" target="_blank">Labeling of fusion proteins of O<sup>6</sup>-alkylguanine-DNA alkyltransferase with small molecules <em>in vivo</em> and <em>in vitro</em>.</a> <em>Methods</em> 32(4): 437-444. </li><li>Keppler, A., Pick, H., Arrivoli, C., Vogel, H. and Johnsson, K. (2004b). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=15226507" target="_blank">Labeling of fusion proteins with synthetic fluorophores in live cells.</a> <em>Proc Natl Acad Sci U S A </em>101(27): 9955-9959.</li><li>Li, C., Kuang, C. and Liu, X. (2018). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=29708726" target="_blank">Prospects for fluorescence nanoscopy.</a> <em>ACS Nano</em> 12(5): 4081-4085. </li><li>Molliex, A., Temirov, J., Lee, J., Coughlin, M., Kanagaraj, A. P., Kim, H. J., Mittag, T. and Taylor, J. P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26406374" target="_blank">Phase separation by low complexity domains promotes stress granule assembly and drives pathological fibrillization.</a> <em>Cell </em>163(1): 123-133. </li><li>Muller, C. B. and Enderlein, J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20867000" target="_blank">Image scanning microscopy.</a> <em>Phys Rev Lett </em>104(19): 198101. </li><li>Nakabayashi, H., Taketa, K., Miyano, K., Yamane, T., Sato, J. (1982). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6286115" target="_blank">Growth of human hepatoma cells lines with differentiated functions in chemically defined medium.</a> <em>Cancer Res </em>42(9): 3858-63.</li><li>Remenyi, R., Gao, Y., Hughes, R. E., Curd, A., Zothner, C., Peckham, M., Merits, A. and Harris, M. (2018). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=29875241" target="_blank">Persistent replication of a chikungunya virus replicon in human cells is associated with presence of stable cytoplasmic granules containing nonstructural protein 3.</a> <em>J Virol</em> 92(16). </li><li>Remenyi, R., Roberts, G. C., Zothner, C., Merits, A. and Harris, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28720784" target="_blank">SNAP-tagged chikungunya virus replicons improve visualisation of non-structural protein 3 by fluorescence microscopy.</a><em> Sci Rep</em> 7(1): 5682. </li><li>Sheppard, C. J., Mehta, S. B. and Heintzmann, R. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23903171" target="_blank">Superresolution by image scanning microscopy using pixel reassignment.</a> <em>Opt Lett </em>38(15): 2889-2892. </li><li>Utt, A., Das, P. K., Varjak, M., Lulla, V., Lulla, A., Merits, A. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25552719" target="_blank">Mutations conferring a noncytotoxic phenotype on chikungunya virus replicons compromise enzymatic properties of nonstructural protein 2.</a> <em>J Virol </em>89(6): 3145-3162. </li><li>Wheeler, J. R., Jain, S., Khong, A. and Parker, R. (2017). <a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Citation&amp;list_uids=28457979" target="_blank">Isolation of yeast and mammalian stress granule cores.</a> <em>Methods</em> 126: 12-17. <br/></li></ol>