<body>
            <h1>QsRNA-seq: A protocol for Generating Libraries for High-throughput Sequencing of Small RNAs</h1>
            <p><strong>Authors:</strong> Alla Fishman, Ayelet T. Lamm</p>
            <h2>Abstract</h2>
            <p>Small RNAs (sRNAs) are 20-30 nt long non-coding RNA molecules that regulate essentially all cellular processes. Besides being an intensively studied topic in academic research, sRNAs also hold a promise as clinical biomarkers. While the need for expressional profiling of sRNAs is growing, preparation of sRNA libraries for high-throughput sequencing (HTS) remains technically challenging, due to their small size. The common PAGE-based protocol is time-consuming and inefficient due to material loss, while gel-free protocols generate libraries of insufficient quality. To overcome these shortcomings, we modified the conditions of size-selection by Solid Phase Reversible Immobilization (SPRI) in a way that allows separation of nucleic acids shorter than 100 nt and differing in length by only 20 nt. Implementing the method for preparation of small RNA libraries for HTS resulted in QsRNA-seq, a gel-free, fast and easy-to-perform protocol, amenable to automation, generating very clean libraries that result in high-depth expression data. The protocol also utilizes Unique Molecular Identifiers (UMI) for reduction of library preparation biases and to quantify expression levels. QsRNA-seq provides an excellent solution to the growing needs for small RNA expression profiling for research clinical use. </p>
            <p><strong>Keywords:</strong> Small RNA, miRNA, siRNA, piRNA, High-throughput sequencing, Gene expression, Size-selection, UMI</p>
            <h2>Background</h2><p>Although small RNAs, which are 20-30 nt long regulatory non-coding RNA molecules, constitute a tiny percentage of the total cellular RNA, they have a huge impact on many cellular processes (including, gene regulation, organism development, adaptation to changing environment and so on) (Jost <em>et al.</em>, 2011). There are several classes of small RNA (Youngman and Claycomb, 2014) with microRNAs (miRNAs) being the most extensively studied (Bartel, 2009). <br/><br/>Besides being of a great interest for basic research, miRNAs hold promise as diagnostic markers. Due to their exceptional stability, intact miRNAs are present in preserved tumor samples (formalin-fixed paraffin-embedded [FFPE]), which exhibit a near total mRNA degradation (Nagy <em>et al.</em>, 2016). Moreover, stable miRNAs circulate in body fluids such as plasma, urine and sputum of healthy and diseased patients (Mitchell <em>et al.</em>, 2008). Liquid biopsy of circulating miRNAs can serve as potential markers for a broad range of human diseases especially for diseases for which ordinary biopsies cannot be obtained, such as cardiovascular diseases, and psychiatric diseases (Kim, 2015; Backes <em>et al.</em>, 2016). <br/><br/>While assessing the repertoire and abundance of cellular miRNAs became an important aspect in many studies, the tools to do so lag behind. High-throughput sequencing (HTS) is a gold standard method for mRNA profiling, extensively used in both basic research and clinical settings. In striking contrast, HTS profiling of miRNAs is not widespread, as preparation of small RNA libraries remains technically challenging. <br/><br/>To become compatible with Illumina or other HTS platforms, small RNAs are ligated from both sides (5â and 3â) to oligonucleotide adapters and then reverse transcribed. The resulting small RNA library is then amplified by PCR with primers complementary to the 5â and 3â adapters that include sequences recognized by the sequencing machinery (Lu <em>et al.</em>, 2007). If small RNAs are not separated from other RNA species, in particular from tRNA, which are highly abundant and close in size (~70 bp), the resulting libraries will be highly contaminated by irrelevant sequences. In addition, 3â and 5â-adapters ligate to each other, forming adapter-dimers, which contaminate libraries with null sequences. Thus, for generation of a high-quality sRNA library, a preparation protocol must enable separation of small RNA from total RNA and separation of the ligation products from free adapters. Size-selection on Solid Phase Reversible Immobilization (SPRI) paramagnetic beads (Lundin <em>et al.</em>, 2010), used in preparation of mRNA and DNA libraries, was not suitable for preparation of small RNA libraries, as the lower limit of size discrimination is 100 nt (Paithankar <em>et al.</em>, 1991). The only remaining option, electrophoresis on PAGE gel, is cumbersome, time-consuming, requires technical skills, results in a huge loss of the product and eliminates the possibility of automatization. Preventing adapter-dimer contamination by molecular biology strategies (Kawano <em>et al.</em>, 2010; Vigneault <em>et al.</em>, 2012) is not a good alternative to size-selection as it does not solve the problem of contamination by irrelevant sequences. <br/><br/>To allow easy size selection of short nucleic acid molecules, we modified the SPRI method by using a combination of two crowding agents, Polyethylene Glycol (PEG) and Isopropanol. This modification enables separation of nucleic acid fragments shorter than 100 nt, differing in length by only 20 nt (Fishman <em>et al.</em>, 2018). Exploiting this new separation method, we designed a protocol for preparation of small RNA libraries for high-throughput sequencing based on Lau <em>et al.</em> (2001) and we named the protocol QsRNA-seq. The input material for the protocol is either total RNA or low molecular weight RNA fraction (LMW RNA, &lt; 200 nt). During the protocol three SPRI-based size selection steps are performed, each using different conditions, to allow separation of sRNA from total RNA and of ligation products from free adapters after each ligation (for the protocol outline see the Procedure section below). In addition, the protocol supports the possibility of multiplexing samples by adding a 4-nt barcode to the 5â-adapter. The protocol also supports using unique molecular identifiers (UMI) for reduction of ligation bias (Fuchs <em>et al.</em>, 2015) and correction of amplification bias (Kivioja <em>et al.</em>, 2011) by adding an 8-nt random sequence to the 5â-adapter as well. To correct the amplification bias, identical small RNA sequences with the same UMI are considered to be amplification products and merged to one sequence (<em>i.e.</em>, collapsed) during data processing. By comparing libraries generated from the same RNA using adapters with the same barcode either with UMI or without UMI, we found that adding UMI significantly reduced both biases (Fishman <em>et al.</em>, 2018). <br/><br/>QsRNA-seq has several significant advantages over the existing protocols. The entire protocol can be performed in one day and does not require high manual skills. Because it does not require any gel use there is less material lost, which enables small quantities of the input RNA to be analyzed. Moreover, it allows simultaneous processing of many samples as well as automation, turning the preparation of small RNA libraries into a routine procedure. In addition, usage of UMI allows reduction of library preparation biases, resulting in more reliable quantitation. To summarize, QsRNA-seq is an easy-to-use protocol that generates reliable and quantifiable high-throughput small RNA libraries.</p>
        </body>

<p class="pptt" id="biaoti27064">## Materials and Reagents</p><ol class="npwtwok"><li>RNase-free plastic pipette tips with filter (10 Î¼l, 20 Î¼l, 200 Î¼l, 1,000 Î¼l)</li><li>1.7 ml RNase-free microtubes</li><li>0.2 ml RNase-free PCR tubes</li><li>miRVana miRNA Isolation Kit (Thermo Fisher Scientific, catalog number: AM1560) or TRIzol reagent (Ambion, catalog number: 15596026) with Direct-Zol RNA MiniPrep Plus (Zymo Research, catalog number: R2072)</li><li>RiboLock RNase Inhibitor (Thermo Fisher Scientific, catalog number: EO0381)</li><li>T4 RNA ligase 1 (ssRNA Ligase) (NEB, catalog number: M0204S)</li><li>T4 RNA ligase 2, truncated (NEB, catalog number: M0242S)</li><li>RppH enzyme (NEB, catalog number M0356S)</li><li>QScript Flex cDNA synthesis kit (Quanta, catalog number: 95049-025)</li><li>Phusion High-Fidelity DNA Polymerase (NEB, catalog number: M0530S)</li><li>Agencourt Ampure XP (Beckman Coulter, catalog number: A63881) or SPRI select reagent Kit (Beckman Coulter, catalog number: B23319)</li><li>RNA BR assay kit (Quibit, catalog number: Q10210)</li><li>dsDNA HS assay kit (Quibit, catalog number: Q10210)</li><li>PEG 8000 (NEB, catalog number: B1004)</li><li>Nuclease-free water (Sigma-Aldrich, catalog number: W4502)</li><li>Ethyl Alcohol absolute </li><li>Isopropyl-alcohol (Isopropanol) chemical grade </li><li>Ethidium Bromide (hylabs, catalog number: BP451)</li><li>Gel loading dye purple (6x) (NEB, catalog number: B7024)</li><li>[Optional] Low-range ultra agarose (Bio-Rad, catalog number: 161-3107)</li><li>ATP (NEB, catalog number: P0756S)</li><li>DMSO (NEB, catalog number B0515)</li><li>Tris (Spectrum, catalog number: s1519)</li><li>Boric Acid (Bio-Lab, catalog number: 000201059100)</li><li>EDTA (J.T. Baker, catalog number: 8993-01)</li><li>50 bp DNA ladder (NEB, catalog number: N3236s)</li><li>Oligonucleotides: all the oligonucleotides were ordered from IDT (<a href="http://www.idtdna.com/" target="_blank">www.idtdna.com</a>)<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221193740_1092.jpg" width="700"/><br/></li><br/><li>[Optional] NucleoSpin gel and PCR Clean-up kit (Macherey-Nagel, catalog number: 740609.50) or Gel/PCR DNA fragments Extraction kit (Geneaid, catalog number: DF004)</li><li>5x TBE (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27065">### Equipment</p><ol class="npwtwok"><li>Micro-pipettors (Gilson, Pipetman)</li><li>DynaMag-2 magnetic stand (Thermo Fisher Scientific, catalog number: 12321D)</li><li>Benchtop microcentrifuge, non-refrigerated (5424 with Rotor FA-45-24-11, Eppendorf, catalog number: 5424000010)</li><li>Vortex mixer, flat head, Vortex-Genie 2 (Scientific Industries, model: SI-0266)</li><li>Cooling block for 1.7 ml tubes (MS, catalog number: MC-0203)</li><li>Agarose gel electrophoresis apparatus</li><li>Electrophoresis power supply </li><li>Gel imager (Bio-Rad, Gel Doc XR+)</li><li>Fluorometer, Quibit 2.0 (Invitrogen, model: Q32866)</li><li>Thermocycler (Biometra, TProfessional)<br/></li><li>Tapestation/Bioanalyzer<br/></li></ol><p class="pptt" id="biaoti27066">### Procedure</p><p class="ppzz">Protocol outline:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221013927_7266.jpg" width="700"/> </p><br/><ol class="npwone"><li>Preparation of RNA samples<br/><ol class="npwtwo"><li>Prepare your RNA samples using column based RNA extraction kits<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>RNA origin: using this protocol we successfully prepared libraries from </em><em>C. elegans</em><em>s, Drosophila, human, Arabidopsis and fungi RNA. Thus we assume it will work for other species as well.</em> </li><li><em>Column based kits enable non-biased recovery of small RNA. Do not use protocols based on precipitation of RNA by isopropanol, as they result in a poor recovery of small RNA. In our hands, using either </em><a href="http://tools.thermofisher.com/content/sfs/manuals/fm_1560.pdf" target="_blank"><em>MiRVana miRNA isolation kit</em></a><em> (</em><em>Ambion</em><em>) or </em><a href="https://www.zymoresearch.eu/media/amasty/amfile/attach/_R2070_R2071_R2072_R2073_Direct-zol_RNA_MiniPrep_Plus_ver._1.0.0.pdf" target="_blank"><em>Direct-Zol RNA miniprep</em></a><em> (</em><em>Zymo</em><em>) resulted in excellent libraries. </em> </li><li><em>Some kits, for example MiRVana kit (Ambion) allow extraction of either total RNA or of separated High molecular weight (HMW) and Low molecular weight (LMW) fractions. Using LMW as starting material results in cleaner libraries.</em> </li></ol></li><li>Measure sample concentration <br/><em>Note: We use Qubit Fluorometer, but NanoDrop can be used as well.</em> </li><li>Estimate RNA quality by electrophoresis or better by Bioanalyzer/TapeStation<br/><em>Note: To avoid library contamination by degradation products of longer RNA types, use only high-quality RNA (as indicated by sharp non-degraded rRNA bands if evaluating by gel electrophoresis or RNA integrity number [RIN] &gt; 8.5 if using Bioanalyzer/TapeStation).</em> </li></ol><br/></li><li>Ligation of 3â-adapter<br/><em>Note: Ligation reaction between sRNA and 5â-Adenylated 3â-adapter is catalyzed by T4 RNA ligase 2 truncated in the absence of free ATP.</em> <br/><ol class="npwtwo"><li>Combine in an RNase-free 1.7 ml microtube<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014018_8517.jpg" width="350"/><br/><br/></li><li>Mix well by pipetting up and down 6-8 times or by gentle vortex. </li><li>Incubate at 72 Â°C for 3 min.</li><li>Transfer directly to ice for 2 min. </li><li>Spin down briefly.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>If you are not limited by RNA quantity, start with 5 Î¼g Total RNA or 0.5 Î¼g of LMW RNA, but it is also possible to start with 1/10 and even 1/100 lesser quantities; we successfully prepared a library from 2 ng of purified pool of synthetic 22 nt oligo. For </em><em>C. elegans</em><em> 100 Âµl worm/egg pellet generates ~200 Âµg total RNA. </em> </li><li><em>Excess of adapter promotes generation of adapter dimers. 0.5 Î¼l of 100 Î¼M 3â-adapter can be used for 5 Î¼g Total RNA or 0.5 Î¼g LMW RNA. For lower starting quantities dilute both the 3â-adapter and the 5â-adapter accordingly. </em> </li><li><em>It is important to transfer the samples into ice immediately after the denaturation steps to prevent reformation of secondary structures.</em> </li></ol></li><li>Add to the sample<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014204_9908.jpg" width="350"/><br/><br/></li><li>Mix well by pipetting up and down 6-8 times or by gentle vortex. </li><li>Incubate at 25 Â°C for 1 h. <br/><em>Notes: </em><br/><ol class="npwthree"><li><em>T4 Ligase 2 truncated arrives with 50% PEG in a separate tube. </em> </li><li><em>Add RNase inhibitor first to prevent RNA degradation.</em> </li></ol></li></ol><br/></li><li>Dephosphorylation (optional)<br/>This step is required only if small RNAs with triphosphate structures at the 5â-end are needed (instead of monophosphate), such as secondary siRNA in <em>C. elegans</em> and retroviral infection-derived miRNA in mammalians. RNA ligases are not able to ligate triphosphorylated 5â ends. RppH enzyme removes two terminal phosphates, generating monophosphate 5â-termini, which can then be ligated to the 5â-adapter.<br/><ol class="npwtwo"><li>To the ligation reaction add 1 Î¼l of RppH enzyme (NEB, 5 U/Î¼l).</li><li>Incubate for 30 min at 37 Â°C.</li></ol><br/></li><li>1<sup>st</sup> SPRI-based size selection<br/><em>Note: During this step molecules &gt; 60 nt, comprising longer RNA species, including tRNA, whether adapter-ligated or not, are bound to beads, while adapter-ligated sRNA (37-45 bp) and free 3â-adapter (18 bp) stay in the supernatant. Following magnetic separation, the supernatant is transferred into a new tube while the beads are discarded. </em> <ol class="npwtwo"><li> Add 40 Î¼l of RNase-free H<sub>2</sub>O to the 20 Î¼l of ligation reaction for a final volume of 60 Î¼l. </li><li>Add 67 Î¼l of Agencourt Ampure XP beads and 78 Î¼l of 100% isopropanol, to obtain a final concentrations of 7.5% PEG and 38% isopropanol (for explanations regarding the concentrations of the crowding agents PEG and Isopropanol see Table 2 below). </li><li> Mix well by pipetting up and down 6-8 times or by gentle vortex. </li><li>Incubate at room temperature for 5 min. Place the reaction tube on a magnet stand and allow the beads to settle for 7 min. </li><li> Carefully transfer the supernatant to a new 1.7 ml tube (discard the beads). <br/><em>Notes:</em> <ol class="npwthree"><li><em>Agencourt Ampure XP are paramagnetic SPRI beads in a solution containing 20% PEG. </em> </li><li><em>Optimal conditions for size-selection depend on the length of the fragments being separated. The conditions above are optimized for separation of adapter-ligated sRNA (37-45 bp) from fragments &gt; 60 bp. If preparing a library from longer RNA or/and using custom adapters with a different length choose the optimum conditions for the 1</em><sup><em>st</em></sup><em> size-selection step using Table 2 from the appendix protocol below. </em> </li><li><em> Beckman currently manufactures two types of SPRI beads: Ampure XP and SPRI-select. Both beads are in 20% PEG. SPRI-select is RNase free certified and pass more stringent Quality Assurance (QA), to ensure reproducibility, however, they are much more expensive. In our hands, both bead types produced identical results.</em> </li><li><em>Since precision is important, avoid carryover.</em> </li><li><em>Aliquot SPRI beads into 1.7 ml tubes to prevent contamination (1 ml beads suffices for 3 libraries).</em> </li></ol><br/></li></ol></li><li>2<sup>nd</sup> SPRI-based size selection<br/>During this step adapter-ligated-sRNA (37-45 nt) are bound to beads, while free sRNA (19-27 nt) and 3â-adapter (18 nt) stay in the supernatant. Following magnetic separation, the supernatant is removed and discarded while adapter-ligated sRNAs are eluted from beads.<ol class="npwtwo"><li>To the supernatant from the previous step add 86 Âµl of Agencourt Ampure XP beads and 144 Âµl 100% isopropanol, to obtain the final concentrations of 7.5% PEG and 51% isopropanol. </li><li>Mix well by pipetting up and down 6-8 times or by gentle vortex. </li><li>Incubate at room temperature for 5 min.</li><li>Place the tube on a magnet stand and allow the beads to settle for 7 min. </li><li>Carefully remove and discard the clear supernatant. </li><li>Proceed according to wash and elution protocol below. </li><li>Elute in 19 Î¼l RNase-free H<sub>2</sub>O.</li></ol><em>Note: Optimal conditions for size-selection depend on the length of the fragments being separated. The conditions above are optimized for separation of the ligation products from non-ligated sRNAs that are 19-27 nt long and 18 nt long 3â-adapter. If preparing a library from longer RNA or/and using custom adapters of a different length choose the optimum conditions for the 2</em><sup><em>nd</em></sup><em> SPRI-based size-selection step by using Table 2 from the appendix protocol below. </em><br/><br/></li><li>Wash and elution protocol<ol class="npwtwo"><li>With the tube still on the magnet stand, add 500 Î¼l of 85% non-denatured ethanol (freshly made from 100% ethanol), without disturbing the beads.</li><li>Incubate at room temperature for 30 s. </li><li>Carefully remove the ethanol without disturbing the beads and discard it.</li><li>Repeat the ethanol wash.</li><li>Let the beads air dry for 5 min. Aspirate residual ethanol and let it dry for additional 2 min. Make sure no residual ethanol is present before proceeding to elution.</li><li>Remove the tube from the magnet stand and add the desired volume of molecular biology grade RNase-free water directly on the beads. </li><li>Gently mix by pipetting the entire volume 10 times. </li><li>Incubate at room temperature for 2 min.</li><li>Spin down briefly.</li><li>Place the microtube on the magnetic stand and allow the beads to settle for 2 min or until the supernatant turns completely clear. </li><li>Transfer the desired volume of the clear supernatant containing small RNA into a new 1.7 ml microtube.</li></ol><em>Note: Always use freshly prepared 85% ethanol and keep the tube close during the protocol. Reduced ethanol concentration due to evaporation may result in detaching of short molecules from the beads.</em><br/><br/></li><li>Ligation of the 5â-adapter <br/><em>Note: Ligation reaction between 3â-ligated-sRNA and the 5â-adapter is catalyzed by T4 RNA ligase 1 in the presence of ATP.</em> <ol class="npwtwo"><li>Combine<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014356_8147.jpg" width="350"/><br/><br/></li><li>Mix well by pipetting up and down 6-8 times or by gentle vortex.</li><li>Incubate at 72 Â°C for 3 min.</li><li>Transfer directly to ice for 2 min. </li><li>Spin down briefly.</li><li>Prepare the ligation mix:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014445_7125.jpg" width="350"/><br/><br/></li><li>Vortex and spin down the ligation mix.</li><li>Add 11 Î¼l of ligation mix to the tube with the small RNA sample. </li><li>Incubate at 37 Â°C for 1 h.</li><li>Incubate at 95 Â°C for 30 s.</li><li>Transfer directly to ice for 2 min.</li><li>Spin down briefly.<br/></li></ol></li><em>Notes:</em> <ol class="npwthree"><li><em>Blocking oligo hybridizes with the free 3â-adapter, turning it into dsDNA, which is not a substrate for RNA ligase, thus preventing generation of adapter-dimer. </em> </li><li><em>Samples are heated to 95 Â°C to release the blocking oligo that might have annealed to the 3â-adapter bound to the small RNA. Free oligos will be removed by SPRI separation. If not released and removed, the oligo may compete with the reverse primer during the RT step and PCR reaction (the oligo is blocked, so no product will be generated, but the yield might be compromised).</em> </li><li><em>Aliquot ATP to prevent its degradation. Thaw the ATP solution at the bench and not in your palm.</em> </li></ol><br/><li>3<sup>rd</sup> SPRI-based size-selection<br/><em>Note: During this step 3â-5â ligated small RNA (64-72 nt) are bound to the beads, while adapter-dimer (45 nt), free 5â-adapter (27 nt) and blocking oligo (18 nt) stay in the supernatant. Following magnetic separation, the supernatant is removed and discarded while the 3â-5â ligated sRNAs are eluted from the beads.</em> <ol class="npwtwo"><li>Add 20 Î¼l of RNase-free H<sub>2</sub>O to 30 Î¼l of the ligation reaction for a final volume of 50 Î¼l. </li><li>Add 68 Î¼l of Agencourt Ampure XP beads and 64 Î¼l of 100% isopropanol, to obtain final concentrations of 7.5% PEG and 35 % isopropanol. </li><li>Mix well by gently pipetting up and down 10 times.</li><li>Incubate at room temperature for 5 min. </li><li>Place the tube on the magnet stand and allow the beads to settle for 10 min.</li><li>Carefully remove and discard the clear supernatant. </li><li>Proceed according to the above wash and elution protocol (Procedure F). </li><li>Elute in 15 Î¼l H<sub>2</sub>O. </li><li>Transfer 12.5 Î¼l of 3â-5â linked-small RNA to a PCR tube.<br/><br/></li></ol></li><li>Reverse transcription <br/><em>Note: The RT protocol below is compatible with qScript Flex cDNA synthesis kit (Quanta). If using another kit for RT follow the kitâs instructions. In our hands, Invitrogen Superscript III and Superscript II resulted in an inferior yield.</em> <ol class="npwtwo"><li>Add to a tube with 12.5 Î¼l of 3â-5â linked-small RNA:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014541_6050.jpg" width="270"/><br/><br/></li><li>Gently pipette the entire volume up and down 6-8 times. </li><li>Spin down briefly.</li><li>Transfer to a PCR machine.</li><li>Incubate at 65 Â°C for 5 min. </li><li>Cool to 42 Â°C.</li><li>Add to the sample at 42 Â°C:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014627_7524.jpg" width="350"/><br/><br/></li><li>Gently pipette the entire volume up and down 6-8 times. </li><li>Incubate at 42 Â°C for 60 min.</li><li>Incubate at 85 Â°C for 5 min. </li><li>Cool to 10 Â°C.</li><li>Proceed to PCR amplification.</li></ol><em>Notes:</em> <ol class="npwthree"><li><em>GSP Enhancer is a component of qScript Flex cDNA synthesis kit used for RT performed with specific primers. </em> </li><li><em>Samples can be kept at -20 Â°C for up to a week.</em> </li></ol><br/></li><li>Pilot cDNA amplification<br/><em>Note: The aim of this step is to determine the optimal number of amplification cycles. After the system has been calibrated, pilot PCR step can be omitted. In this case, perform the main PCR directly with the known optimal cycle number. </em> <ol class="npwtwo"><li>Combine in a PCR tube:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014716_9749.jpg" width="350"/><br/><br/></li><li>Perform PCR for 23 cycles. </li><li>Pause the thermocycler after cycles: 11, 14, 17, 20, and 23; open the tubes without removing them from the cycler and withdraw 7 Î¼l for analysis.</li><li>PCR conditions:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:200px;"><tbody><tr><td>98 Â°C<br/></td><td>30 s<br/></td><td><br/></td></tr><tr><td>Cycles 1-4<br/></td><td>98 Â°C<br/></td><td>10 s</td></tr><tr><td><br/></td><td>50 Â°C<br/></td><td>30 s</td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>15 s</td></tr><tr><td>Cycles 5-20<br/></td><td>98 Â°C<br/></td><td>10 s</td></tr><tr><td><br/></td><td>64 Â°C<br/></td><td>30 s</td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>15 s</td></tr><tr><td>72 Â°C<br/></td><td>2 min<br/></td><td><br/></td></tr><tr><td>10 Â°C<br/></td><td>hold</td><td><span style="color:#002000;font-family:Arial,Helvetica,sans-serif;"></span><br/></td></tr></tbody></table><em>Notes:</em> </li><ol class="npwthree"><li><em>The number of cycles can be raised if using small starting RNA quantities.</em> </li><li><em>Aliquot dNTPs to prevent freeze-thaw cycle degradation. </em> </li><li><em>RT product can constitute no more than 1/10 volume of a PCR reaction.</em><br/></li></ol></ol><br/></li><li>Pilot gel electrophoresis<br/><em>Note: Samples can be checked by TapeStation/Bioanalyzer instead.</em> <ol class="npwtwo"><li>Cast 3.0% agarose/TBE gel. </li><li>Load 5 Î¼l of each one of the withdrawn samples.</li><li>Run samples at 80 V for 1.5 h.</li><li>Take a picture.</li></ol><em>Notes:</em> <ol class="npwthree"><li><em>Use Purple or Orange loading dyes (NEB) as they produce no UV visible bands masking the library.</em> </li><li><em>Use TBE buffer instead of TAE, as it allows a better resolution of small fragments.</em><br/><br/></li></ol></li><li>Evaluation of the PCR product<br/><em>Note: Evaluate the PCR products to determine the optimal PCR cycle number for the main PCR. The optimal number of cycles results in a clearly visible but not too thick/smeary product band and the best library/artifact bands ratio (for typical PCR results see Figure 1).<br/></em><br/>Possible PCR products:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014814_8493.jpg" width="350"/><br/><em>Note: Truncated primer-dimer originates from annealing of a reverse primer to one of random 8N sequences during RT or PCR. It lacks barcode and a part of the 8N sequence.</em><br/><br/><img alt="" class="layerphoto" height="104" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221012313_9366.jpg" width="220"/><br/><strong>Figure 1. Example of a Pilot PCR electrophoresis.</strong> A. Library prepared from 1 Âµg of LMW RNA fraction extracted from <em>C. elegans</em> worms synchronized to L4 developmental stage was amplified for 11, 14, 17 and 20 cycles. B. Library prepared from 1 Âµg of total RNA extracted from <em>C. elegans</em> worms synchronized to L4 developmental stage was amplified for 14, 17, 20 and 23 cycles. Five Âµl of each amplification product was electrophoresed on 3% agarose-TBE gel. The sRNA Library product is indicated by a red arrow. The ~130 bp library product is visible starting at cycle 11 in (A) and cycle 14 in (B). The upper ~270 bp band originates from snRNA and the lower 93 bp band originates from truncated adapter-dimer. Based on these results the chosen number of cycles for the main PCR was 14 cycles for library (A) and 17 cycles for library (B).<br/><br/></li><li>Main cDNA amplification<br/>Prepare PCR reaction as above and amplify with the predetermined optimal number of cycles. <br/><em>Note: If skipping the pilot PCR, take an aliquot after the amplification for quality check before proceeding to library purification.</em><br/><br/></li><li>Cleaning PCR product <br/><em>Notes:</em> <ol class="npwthree"><li><em>Most contaminating bands, except for a primer-dimer, can be efficiently removed by a conventional double-side size-selection using SPRI beads in 20% PEG (for typical cleaning output see Figure 2). </em> </li><li><em>A 105 bp long primer-dimer is inefficiently removed by SPRI beads. Low level of contamination by primer-dimer is acceptable, but in a case of a prominent primer-dimer band, we suggest cleaning the PCR product by gel-extraction.</em><br/><br/></li></ol><u>Cleaning on SPRI beads</u> <ol class="npwtwo"><li>Removal of bands &gt; 200 bp<ol class="npwthree"><li>Transfer 50 Î¼l of the PCR product to a 1.7 ml tube.</li><li>Add beads as follows:<ol class="npwfour"><li>For libraries predominantly contaminated with higher bands (x1.4V, x2.0V), add 70 Î¼l. </li><li>For libraries predominantly contaminated with lower bands (x1.2V, x1.8V), add 60 Î¼l.</li></ol></li><li>Mix the total reaction volume by pipetting 10 times.</li><li>Incubate at room temperature for 5 min. </li><li>Place the tube on the magnetic stand and allow the beads to settle for 7 min. </li><li>Transfer the entire clear supernatant to a new tube (discard the beads).</li></ol></li><li>Removal of bands &lt; 100 bp<ol class="npwthree"><li>Add 30 Î¼l beads to the supernatant from the previous step.</li><li>Mix the total reaction volume by pipetting 10 times.</li><li>Incubate at room temperature for 5 min. </li><li>Place the reaction vessels on the magnet stand and allow the beads to settle for 7 min. </li><li>Carefully remove and discard the clear supernatant. </li><li>Proceed according to wash and elution protocol above (Procedure F). </li><li>Elute with 20 Î¼l H<sub>2</sub>O.<br/></li></ol></li><li>Examples of sRNA libraries before and after cleaning are presented in Figure 2.</li></ol><em>Note: Cleaning a PCR product is performed by conventional double-side size-selection (using SPRI beads only) according to </em><a href="https://research.fhcrc.org/content/dam/stripe/hahn/methods/mol_biol/SPRIselect User Guide.pdf" target="_blank"><em>the SPRI based size-selection guide of Beckman-Coulter</em></a><em>. At the first step bands longer than the library are bound to the beads, while the library remains in the supernatant and is transferred to a new tube. On the second step a new batch of beads is added to increase PEG concentration, the library is bound to the beads, while the shorter contaminants remain in the supernatant and are discarded. The volume of the added beads is calculated in relation to the volume of the nucleic acid solution. In order to clean the libraries contaminated predominantly by longer bands use 70 Î¼l of the beads at the 1</em><sup><em>st</em></sup><em> step, which correspond to 1.4x volume of the 50 Î¼l of nucleic acid solution (1.4x V), and 100 Î¼l (70 + 30) beads at the 2</em><sup><em>nd</em></sup><em> step correspond to 2.0x V. In order to clean a library contaminated predominantly by shorter bands use 60 Î¼l of the beads at the 1</em><sup><em>st</em></sup><em> step, which correspond to 1.2x V, while 90 Î¼l (60 + 30) beads at the 2</em><sup><em>nd</em></sup><em> step correspond to 1.8x V.</em><br/><br/></li><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" height="110" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221012516_7650.jpg" width="150"/><br/><strong>Figure 2. Removal of library contaminants by size-selection on SPRI beads.</strong> TapeStation traces of PCR products before (left panels: A, C) and after (right panels: B, D) cleaning using SPRI beads. Libraries were prepared from 5 Âµg total RNA extracted from <em>C. elegans</em> synchronized to L4 developmental stage. Upper row (A-B): The major contaminant is 270 bp long constituting ~65% of the total PCR product. Cleaning by double side size selection using 1.4x V beads followed by 2.0x V beads reduced the contaminant to below 10%. Lower row (C-D): The major contaminant is 93 bp long constituting ~30% of the total PCR product. Cleaning by double side size selection using 1.2x V beads followed by 1.8x V beads reduced the contaminant to below 10%. sRNA library corresponding peaks are marked by red arrows. Peak sizes are indicated. The outer peaks titled âLowerâ and âUpperâ are 25 nt and 1,500 nt molecular size markers, respectively. </p><br/><u>Cleaning by gel-extraction (optional)</u><br/><em>Note: Clean by gel-extraction instead of SPRI beads in a case of a heavy contamination with primer-dimer.</em> <ol class="npwtwo"><li>Cast 3.5% agarose or 3.0% low-range ultra agarose (Bio-Rad) gel in TBE. </li><li>Electrophorese sample at 80 V for 2.5 h. </li><li>Excise the library band and weight the slice. </li><li>Chop the slice finely (use separate razor for each library). </li><li>Extract using gel extraction kit.<br/><em>Note: In our hands, NucleoSpin gel and PCR Clean-up kit (Macherey-Nagel) and Gel/PCR DNA fragments Extraction kit (Geneaid) resulted in a satisfactory yield.</em> </li><li>Elute with 15 Î¼l H<sub>2</sub>O/elution buffer.<br/><br/></li></ol><li>Final product validation<ol class="npwtwo"><li>Measure product concentration.</li><li>Validate the library purity by TapeStation/Bioanalyzer.<br/><br/></li></ol></li><li>Library storage<br/>Once the library is ready for high-throughput sequencing (HTS), store library at -20 Â°C until use. <br/></li></ol><br/><p class="ppzz"><strong>Appendix protocol: Fine-tuning of binding conditions for different sizes of ligated molecules</strong><br/><br/>Size selection conditions of the main protocol above are suited for ligation of 19-27 nt long small RNA to 18 nt long 3â-adapter and 27 nt long 5â-adapter. As binding efficiency of a molecule is dependent on its size, if preparing a library from longer RNA molecules or if using custom adapters of a different length, adjustment of the conditions may be required for one or all size-selection steps.</p><ol class="npwtwo"><li>Calculate the sizes of the fragments that are needed to be separated (see Table 1 for example)<br/><br/><strong>Table 1. Sizes of ingredients, intermediates and products of small RNA library preparation process described in the main protocol above.</strong> Numbers in brackets correspond to predominantly miRNA libraries, such as in human.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221014930_3669.jpg" width="300"/><br/><br/></li><li>Determine the optimal separation conditions for the calculated sizes using Table 2.<br/><br/><strong>Table 2. Conditions for SPRI-based separation of short RNA molecules</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221015027_9076.jpg" width="700"/><br/><br/>The table presents approximate sizes of fragments that a given combination of PEG and isopropanol allows to separate, <em>i.e.</em>, a minimal length of a longer fragment (that is bound to the beads) and the maximal length of a shorter fragment (that stays in the supernatant). Poor binding is defined as &lt; 5%, efficient binding is defined as &gt; 50%. Note that the PEG concentration is always 7.5% and that the volume of RNA in H<sub>2</sub>O is 50 Î¼l. <br/><br/><strong><em>Examples of Table 2 usage</em></strong><br/><u>Example 1:</u> At the 3<span style="font-size:11.1111px;"><sup>rd</sup></span> SPRI selection step of the protocol above we need to separate the 3â-5â-adapter-ligated sRNAs (64-72 bp) from adapter dimers (45 bp). According to Table 2 the suitable conditions, <em>i.e.</em>, conditions that allow efficient binding of 64 nt long fragments and poor binding of 45 nt long fragments, are 7.5% PEG and 35% isopropanol. To obtain these conditions we need to bring the sample to a volume of 50 Î¼l by adding H<sub>2</sub>O and then to add 68 Î¼l 20% PEG (beads solution) and 64 Î¼l isopropanol. <br/>  If the RNA solution already contains PEG, the quantity of PEG needed to be added would need to be recalculated. <br/><br/><u>Example 2:</u> At the 1<sup>st</sup> SPRI selection of the protocol above, according to Table 2 we need to add 77 Î¼l of 20% PEG to 50 Î¼l RNA in water to obtain a final concentration of 7.5% PEG. However, the ligation reaction already contains 4 Î¼l of 50% PEG, which equals to 10 Î¼l of 20% PEG. Therefore, the amount of added PEG has to be reduced by 10 Î¼l (67 Î¼l instead of 77 Î¼l), in parallel the volume of H<sub>2</sub>O was increased by 10 Î¼l (40 Î¼l instead of 30 Î¼l). <br/>  Similar adjustments have to be done for both PEG and Isopropanol when cleaning the fragment of interest first from higher and then from lower bands (double-side selection). <br/><br/><u>Example 3:</u> At the 2<sup>nd</sup> SPRI selection, for separation of the ligated sRNA from free adapter, 163 Î¼l of 20% PEG and 222 Î¼l isopropanol are needed to obtain a final concentration of 7.5% PEG and 51% isopropanol. However, the sample already contains 77 Î¼l of 20% PEG and 78 Î¼l isopropanol added at the 1<sup>st</sup> size-selection, thus only 86 Î¼l of beads solution and 144 Î¼l 100% isopropanol are needed to be added.<br/>  For each combination of PEG and isopropanol, binding efficiency of a fragment longer than the minimal length listed in the table will be higher, while that of a fragment shorter than the maximal length will be lower than the value appearing in the table. Better separation between the fragments can be achieved on the expense of recovery. Pure longer fragment can be obtained if using slightly lower isopropanol concentration, while pure shorter fragment can be obtained by double-side size-selection if removing longer fragment at higher isopropanol concentration.<br/><br/><u>Example 4:</u> When separating 37 nt and 58 nt long fragments, to obtain pure 58 nt fragment perform binding with 38% isopropanol. At this concentration, some of the 58 nt fragments will remain unbound, but not the 37 nt fragments. In order to obtain pure 37 nt, first perform binding using 44% isopropanol. All 58 nt fragments and some of the 37 nt fragments will be removed from the solution. The remaining pure 37 nt long fragment is next recovered from the solution using 51% isopropanol. If wishing to obtain both fragments, perform the first binding using 41% isopropanol and the second binding using 51%. Upon these conditions, each of the fragments will be slightly contaminated by the other. For additional information see Fishman <em>et al.</em> (2018).<br/><br/><strong><em>Testing binding conditions other than the ones listed in Table 2</em></strong> <ol class="npwtwo"><li>Calculate quantities of Isopropanol and beads (in 20% PEG) to be added.<br/>According to the equation:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221020145_0167.jpg" width="200"/><br/><br/>where,<br/>V is total volume<br/>X is volume of nucleic acid solution<br/>P is desired concentration of PEG (in %)<br/>Q is desired concentration of (%) isopropanol<br/><br/>A total volume of binding solution is equal to:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190221/20190221020222_3650.jpg" width="200"/><br/><br/><ol class="npwthree"><li>Substitute P and Q for the desired concentrations of PEG and Isopropanol, respectively and X for the volume of nucleic acid solution (we recommend 50 Î¼l) and calculate a total volume of binding solution (V).</li><li>Next, calculate the volumes of beads and Isopropanol that are equal to 5PV/100 and QV/100, respectively.</li></ol></li><li>Measure binding efficiency<ol class="npwthree"><li>Prepare solution of 100 ng of synthetic single-stranded DNA oligonucleotide in 50 Î¼l H<sub>2</sub>O in a 1.7 ml microtube. </li><li>Add beads and Isopropanol at volumes determined using the calculation method above.</li><li>Mix thoroughly. </li><li>Incubate at room temperature for 5 min. </li><li>Separate on a magnet stand for 7 min.</li><li>Carefully remove the supernatant and wash the beads twice with 85% ethanol according to wash and elution protocol. Elute in 20 Î¼l H<sub>2</sub>O. </li><li>Measure oligonucleotide concentrations of the input oligonucleotide solution and the eluted sample by Qubit<sup>Â®</sup> Fluorometer using Qubit<sup>Â®</sup> ssDNA Assay Kit (Molecular probes). Calculate the binding efficiency as the percentage of the output oligonucleotide from the input quantity.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti27067">### Recipes</p><ol class="npwtwok"><li>5x TBE <br/>Dissolve 54 g Tris base (FW = 121.14) and 27.5 g boric acid (FW = 61.83) in 900 ml deionized water<br/>Add 20 ml of 0.5 M EDTA (pH 8.0) <br/>Adjust the solution to a final volume of 1 L with deionized water<br/><em>Notes:</em> <ol class="npwthree"><li><em>The pH must be ~8.3 (do not adjust pH!)</em> </li><li><em>Discard if precipitate was formed.</em> </li><li><em>Prepare and run agarose gel in 1x TBE.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti27068">### Acknowledgments</p><p class="ppzz">We thank Prof. emeritus Vladimir Lin from the Technion Faculty of Mathematics for his help in development of the equation for calculating volumes of PEG and Isopropanol. <br/>  This work was funded by The Israeli Centers of Research Excellence (I-CORE) program, (Center No. 1796/12 to ATL), The Israel Science Foundation (grant Nos. 644/13 and 927/18 to ATL), and Israel Cancer Research Fund (ICRF). This protocol was adapted from Fishman <em>et al.</em> (2018).</p><p class="pptt" id="biaoti27069">### Competing interests</p><p class="ppzz">PCT Patent Application No. PCT/IL2017/050761, entitled âMethods for isolation and quantification of short nucleic acid moleculesâ, authored by A. F. and A. T. L. was filed by <em>Technion Research and Developmental Foundation</em> on 6 July 2017. The application claims priority from US Provisional Application No. 62/359,245.</p><p class="pptt" id="biaoti27070">### References</p><ol class="npwtwok"><li>Backes, C., Meese, E. and Keller, A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27378479" target="_blank">Specific miRNA disease biomarkers in blood, serum and plasma: challenges and prospects.</a> <em>Mol Diagn Ther</em> 20(6): 509-518.</li><li>Bartel, D. P. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19167326" target="_blank">MicroRNAs: target recognition and regulatory functions.</a> <em>Cell</em> 136(2): 215-233.</li><li>Fishman, A., Light, D. and Lamm, A. T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30107842" target="_blank">QsRNA-seq: a method for high-throughput profiling and quantifying small RNAs.</a> <em>Genome Biol</em> 19(1): 113.</li><li>Fuchs, R. T., Sun, Z., Zhuang, F. and Robb, G. B. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25942392" target="_blank">Bias in ligation-based small RNA sequencing library construction is determined by adaptor and RNA structure.</a> <em>PLoS One </em>10(5): e0126049.</li><li>Jost, D., Nowojewski, A. and Levine, E. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21266101" target="_blank">Small RNA biology is systems biology.</a> <em>BMB Rep</em> 44(1): 11-21.</li><li>Kawano, M., Kawazu, C., Lizio, M., Kawaji, H., Carninci, P., Suzuki, H. and Hayashizaki, Y. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20964636" target="_blank">Reduction of non-insert sequence reads by dimer eliminator LNA oligonucleotide for small RNA deep sequencing.</a> <em>Biotechniques</em> 49(4): 751-755.</li><li>Kim, Y. K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26306299" target="_blank">Extracellular microRNAs as biomarkers in human disease.</a> <em>Chonnam Med J</em> 51(2): 51-57.</li><li>Kivioja, T., Vaharautio, A., Karlsson, K., Bonke, M., Enge, M., Linnarsson, S. and Taipale, J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22101854" target="_blank">Counting absolute numbers of molecules using unique molecular identifiers.</a> <em>Nat Methods</em> 9(1): 72-74.</li><li>Lau, N. C., Lim, L. P., Weinstein, E. G. and Bartel, D. P. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11679671" target="_blank">An abundant class of tiny RNAs with probable regulatory roles in <em>Caenorhabditis elegans</em>.</a> <em>Science</em> 294(5543): 858-862.</li><li>Lu, C., Meyers, B. C. and Green, P. J. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17889797" target="_blank">Construction of small RNA cDNA libraries for deep sequencing.</a> <em>Methods</em> 43(2): 110-117.</li><li>Lundin, S., Stranneheim, H., Pettersson, E., Klevebring, D. and Lundeberg, J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20386591" target="_blank">Increased throughput by parallelization of library preparation for massive sequencing.</a> <em>PLoS One</em> 5(4): e10029.</li><li>Mitchell, P. S., Parkin, R. K., Kroh, E. M., Fritz, B. R., Wyman, S. K., Pogosova-Agadjanyan, E. L., Peterson, A., Noteboom, J., O'Briant, K. C., Allen, A., Lin, D. W., Urban, N., Drescher, C. W., Knudsen, B. S., Stirewalt, D. L., Gentleman, R., Vessella, R. L., Nelson, P. S., Martin, D. B. and Tewari, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18663219" target="_blank">Circulating microRNAs as stable blood-based markers for cancer detection.</a> <em>Proc Natl Acad Sci U S A</em> 105(30): 10513-10518.</li><li>Nagy, Z. B., Wichmann, B., Kalmar, A., Bartak, B. K., Tulassay, Z. and Molnar, B. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26678076" target="_blank">miRNA Isolation from FFPET Specimen: A technical comparison of miRNA and total RNA isolation methods.</a> <em>Pathol Oncol Res</em> 22(3): 505-513.</li><li>Paithankar, K. R. and Prasad, K. S. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2030954" target="_blank">Precipitation of DNA by polyethylene glycol and ethanol.</a> <em>Nucleic Acids Res</em> 19(6): 1346.</li><li>Vigneault, F., Ter-Ovanesyan, D., Alon, S., Eminaga, S., D, C. C., Seidman, J. G., Eisenberg, E. and G, M. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22470142" target="_blank">High-throughput multiplex sequencing of miRNA.</a> <em>Curr Protoc Hum Genet </em>Chapter 11: Unit 11.12. 1-10.</li><li>Youngman, E. M. and Claycomb, J. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25505902" target="_blank">From early lessons to new frontiers: the worm as a treasure trove of small RNA biology.</a> <em>Front Genet</em> 5: 416.<br/></li></ol>