<body>
            <h1>A Radioisotope-free Oligosaccharyltransferase Assay Method</h1>
            <p><strong>Authors:</strong> Takahiro  Yamasaki, Daisuke Kohda</p>
            <h2>Abstract</h2>
            <p>Glycosylation of asparagine residues is widespread in Eukarya, and occurs in virtually all Archaea and some eubacterial species. A membrane-bound enzyme, oligosaccharyltransferase, catalyzes the transfer of an oligosaccharide chain from a sugar donor (lipid-linked oligosaccharide, LLO) to an asparagine residue in the consensus sequence, Asn-X-Ser/Thr (X â  Pro), in proteins. The <em>in vitro</em> oligosaccharyl transfer assay reaction mixture contains a detergent-solubilized oligosaccharyltransferase (OST), a sugar donor LLO, and a sugar acceptor peptide. Previous assay methods are problematic, in terms of the use of radioactive compounds and the cumbersome separation procedures using lectin binding or two-phase partitioning. Here, we describe a new oligosaccharyl transfer assay method, which is radioisotope-free and relies on a different separation mechanism. The glycopeptide products are separated from unreacted peptides by SDS-PAGE. A fluorescent dye is attached to the peptide substrate during custom peptide synthesis. The fluorescent imaging of the SDS-PAGE gels ensures high sensitivity and quantitative performance. The user-friendly PAGE format is particularly suitable for presentation in scientific papers. For illustrative applications, time-course and peptide library experiments are shown.</p>
            <p><strong>Keywords:</strong> Oligosaccharyltransferase, Oligosaccharyl transfer assay, N-glycosylation, SDS-PAGE, Fluorescent detection</p>
            <h2>Background</h2><p>Glycosylation of asparagine residues, which is also referred to as N-glycosylation, is one of the most ubiquitous protein modifications (Cherepanova <em>et al.</em>, 2016). N-glycosylation occurs not only in Eukarya, but also in most Archaea (Larkin and Imperiali, 2011). N-glycosylation is also found in some species in Eubacteria. In the entire complex N-glycosylation process, the transfer of an oligosaccharide chain from a sugar donor to asparagine residues in proteins is the defining event (Figure 1). This reaction is catalyzed by a membrane-bound enzyme, oligosaccharyltransferase (OST or OTase). The acceptor asparagine residues, in principle, reside in the consensus sequence Asn-X-Thr or Asn-X-Ser (or Asn-X-Cys in rare cases), where X can be any amino acid residue except Pro. Eubacterial OST uses an extended version of the 5-residue sequon, Asp-X-Asn-X-Ser/Thr or Glu-X-Asn-X-Ser/Thr (X â  Pro). Asp is preferable to Glu at the -2 position, even though the acidic residue at the -2 position is not absolutely required in some eubacterial species (Schwarz <em>et al.</em>, 2011). The sugar donor is a type of glycolipid, referred to as a lipid-linked oligosaccharide (LLO). The chemical structure of LLO is an oligosaccharide chain attached to a lipid-phospho carrier. The lipid part is dolichol in Eukarya and Archaea, and undecaprenol in Eubacteria. The phosphate group is basically diphosphate, but some archaeal species belonging to Euryarchaeota (an ancient phylum of Archaea) use monophosphate (Taguchi <em>et al.</em>, 2016).<br/><br/><img alt="" class="layerphoto" height="98" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190228/20190228004104_5752.jpg" width="200"/><br/><strong>Figure 1. Overview of the biosynthesis of N-glycans attached to proteins.</strong> The oligosaccharyltransferase catalyzes the transfer of the oligosaccharide chain from LLO to asparagine residues in nascent proteins on the luminal side of the eukaryotic ER. OST: oligosaccharyltransferase. LLO: lipid-linked oligosaccharide.<br/><br/>The <em>in vivo</em> oligosaccharyl transfer activity is measurable by analyzing the extent of oligosaccharide modification of a specified protein in cells. The amino-acid variants of model glycoproteins can be used to analyze the efficiency of the oligosaccharyl transfer reaction around the N-glycosylation site. However, it is difficult to analyze the effects of amino-acid variation on native oligosaccharyl transfer activity in cells, because the N-glycosylation is essential in eukaryotic cells and rather vital in prokaryotic cells. In general, in-cell studies are difficult to set up. In an exceptional case, the whole N-glycosylation system (the LLO biosynthetic pathway, the OST enzyme, and the substrate protein) of the eubacterium <em>Campylobacter</em> was transferred to <em>Escherichia coli</em> cells (Wacker <em>et al.</em>, 2002). <br/><br/>The reaction solution for <em>in vitro</em> oligosaccharyl transfer assay is a mixture of a detergent-solubilized OST enzyme, a sugar donor LLO, and a sugar acceptor peptide. Chromatographically purified LLO is preferred, but crude fractions after two-phase partitioning with a chloroform/methanol/water system are usable. In principle, OST strictly recognizes the chemical structure of LLO. As for eukaryotic OSTs, the chemical structure of LLO (Glc<sub>3</sub>Man<sub>9</sub>GlcNAc<sub>2</sub>-PP-dolichol) is common, and thus the human OST enzyme can utilize LLO isolated from yeast cells. In contrast, the archaeal LLOs are quite structurally diverse and must be isolated from the same or closely related species for the oligosaccharyl transfer assay. <br/><br/>One previous assay method is based on the lectin binding of radioactive glycopeptide products after the oligosaccharyl transfer reaction with a <sup>125</sup>I-labeled peptide substrate (Geetha-Habib <em>et al.</em>, 1990). Another assay method is based on the recovery of radioactive glycopeptide products in the upper water phase, in two-phase partitioning after the oligosaccharyl transfer reaction with <sup>14</sup>C-labeled LLO (Bause and Hettkamp, 1979; Sharma <em>et al.</em>, 1981). The two methods are problematic, in terms of the use of radioactive compounds and the cumbersome separation procedures. <br/><br/>Therefore, we have developed a new oligosaccharyl transfer assay method, which is radioisotope-free and highly efficient, by using PAGE as the separation mechanism (Figure 2) (Kohda <em>et al.</em>, 2007). The bulky oligosaccharide chain in glycopeptides decreases the rate of migration during electrophoresis, due to the sieving action of the polyacrylamide gel. The glycopeptide products are detected and quantified by fluorescent imaging of the PAGE (polyacrylamide gel electrophoresis) gels. A fluorescent dye for detection is attached to a peptide during peptide synthesis. The minimal detectable quantity of the glycopeptide is 1 fmol in routine assay conditions. The excellent linearity over three orders of magnitude for fluorescently labeled peptide amounts up to 1,000 fmol enabled us to quantify the glycopeptide product accurately, for the determination of the K<sub>m</sub> and V<sub>max</sub> values for peptide substrates (Igura and Kohda, 2011a). At 12 years after its development, the radioisotope-free oligosaccharyltransferase assay method is now widely used in many laboratories.<br/><br/><img alt="" class="layerphoto" height="105" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190228/20190228004240_4486.jpg" width="350"/><br/><strong>Figure 2. Radioisotope-free assay for the oligosaccharyltransferase activity.</strong> Oligosaccharyl transfer assay method using a fluorescently labeled peptide for detection and a separation mechanism based on PAGE (polyacrylamide gel electrophoresis). OST: oligosaccharyltransferase. LLO: lipid-linked oligosaccharide. Dol-PP: dolichol-pyrophosphate<br/></p>
        </body>

<p class="pptt" id="biaoti27146">## Materials and Reagents</p><ol class="npwone"><li>Labwares<ol class="npwtwo"><li>Pipette tips, 10 Î¼l and 200 Î¼l (Gilson, Diamond tips D10 and D200, catalog numbers: F161630 and F161930)</li><li>Conical polypropylene centrifuge tubes, 15 ml and 50 ml (Thermo Fisher, Nunc, catalog numbers: 339650 and 339652)</li><li>1.5 ml polypropylene tubes (Eppendorf, 3810X, catalog number: 0030125150)</li><li>Gloves (Ansell, Microflex, catalog number: NPG-INT)<br/><br/></li></ol></li><li>OST protein and two substrates<ol class="npwtwo"><li>Oligosaccharyltransferase enzyme, isolated from natural sources or recombinantly expressed, purified in the presence of detergent (see Notes for preparation)</li><li>Sugar donor LLO, (partially) purified from cultured cells (see Notes for preparation)</li><li>Sugar acceptor peptide containing the N-glycosylation sequon in the amino acid sequence and a fluorescent dye attached to the N- or C-terminus for detection (see Notes for design and preparation)<br/><br/></li></ol></li><li>Reagents, solutions, and buffers<ol class="npwtwo"><li>Precast PAGE gels, 15%-25% gradient gel (size 85 x 90 x 0.9 mm<sup>3</sup>) (Cosmo Bio, MULITGEL II Mini 15/25 (17W), catalog number: DCB-414923)</li><li>Trizma base [tris(hydroxymethyl)aminomethane] (Sigma-Aldrich, catalog number: T1503)<br/></li><li>HCl (Sigma-Aldrich, catalog number: H1758)</li><li>MgCl<sub>2</sub> hexahydrate (Sigma-Aldrich, catalog number: M2670)</li><li>MnCl<sub>2</sub> tetrahydrate (Sigma-Aldrich, catalog number: 221279)</li><li>Dithiothreitol (DTT) (Sigma-Aldrich, catalog number: 43819)</li><li>Tween 20 (Sigma-Aldrich, catalog number: P1379)</li><li>Lyophilized peptide powder (custom order, TORAY)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: X100)</li><li>N-dodecyl-Î²-D-maltoside (DDM) (DOJINDO, catalog number: D316)</li><li>Chloroform (Sigma-Aldrich, catalog number: C2432)</li><li>Methanol (Sigma-Aldrich, catalog number: 34860)</li><li>SDS (Sigma-Aldrich, catalog number: L3771)</li><li>Glycerol (Sigma-Aldrich, catalog number: G5516)</li><li>Bromophenol blue (Sigma-Aldrich, catalog number: B0126)</li><li>Glycine (Sigma-Aldrich, catalog number: G8898)</li><li>(Optional) EDTA (ethylenediaminetetraacetic acid) (Sigma-Aldrich, catalog number: EDS)</li><li>10:10:3 (v/v/v) chloroform:methanol:water (see Recipes)</li><li>5x SDS-PAGE sample loading buffer (see Recipes)</li><li>Running electrophoresis buffer (see Recipes or Sigma-Aldrich, 10x concentrate, catalog number: T7777-1L)</li></ol></li></ol><p class="pptt" id="biaoti27147">### Equipment</p><ol class="npwtwok"><li>Micropipettes (Gilson, Pipetman P2, P20, P200, catalog numbers: F144801, F123600, F123601)</li><li>Electronic pipettes (Gilson, Pipetman P10M, P100M, catalog numbers: F81022, F81013)</li><li>4 Â°C refrigerator (PHC, MPR-1014)</li><li>Refrigerated laboratory tabletop centrifuge (Eppendorf, Centrifuge 5418R, catalog number: 5401000030)</li><li>Water-bath type sonicator (Branson, Bransonic 2800 ultrasonic cleaner)</li><li>Aluminum heating/cooling block for 1.5 ml tubes (Sigma-Aldrich, catalog number: Z740272-1EA)</li><li>Incubator, natural convection oven or forced air flow convection oven (YAMATO, IC402, catalog number: 211195 or EYELA, NDO-420 or WFO-420, catalog numbers: 252240 or 252280)<br/><em>Note: Alternatively, a water bath-type incubator and PCR equipment can be used.</em> </li><li>Vacuum concentrator that withstands organic solvent (Thermo Fisher, Savant SpeedVac SPD120P1, catalog number: SPD120P1-115/230) </li><li>Cassette electrophoresis unit for PAGE (Cosmo Bio, Model DPE-1020 for Mini Gel, DCB-303111)</li><li>Power supply for PAGE (Bio-Rad, PowerPac Basic, catalog number: 1645050)</li><li>LAS-3000 multicolor gel image analyzer (Fuji Film)<ol class="npwthree"><li>LED (Green 520 nm EPI) illuminator (for TAMRA dye) and LED (Blue 460 nm EPI) illuminator (for 5-CF/5-FAM dye)</li><li>575DF20 filter (for TAMRA) and Y515-Di filter (for 5-CF/5-FAM)</li><li>Iris 0.85</li></ol><em>Note: Alternatively, other gel imagers can be used: for example, Amersham Imager 680 RGB system (GE) and Fusion Solo 7S with Spectra-Pad RGB option (Vilber).</em><br/></li><li>Air (gas) duster (Orientech, catalog number: AD400FL)</li><li>Fume hood (Yamato Scientific, catalog number: RFS-120SZ)<br/></li><li>-20 Â°C freezer (PHC, catalog number: MDF-MU500H-PJ)<br/></li></ol><p class="pptt" id="biaoti27148">### Software</p><ol class="npwtwok"><li>ImageGauge software (Fuji Film)<br/><em>Note: Alternatively, any software installed in other gel imagers can be used.</em><br/></li></ol><p class="pptt" id="biaoti27149">### Procedure</p><ol class="npwone"><li>Oligosaccharyl transfer reaction<ol class="npwtwo"><li>Preparation of reaction buffer solution<br/>Combine the following reagents in a 1.5 ml polypropylene tube:<br/>10 Î¼l of 1 M Tris-HCl, pH 7.5<br/>20 Î¼l of 100 mM MnCl<sub>2</sub><br/>2 Î¼l of 100 mM DTT<br/>168 Î¼l of 0.1% (v/v) Triton X-100 or 0.1% (w/v) DDM (n-dodecyl-Î²-D-maltoside)<br/>The total volume is 200 Î¼l<br/><em>Note: In the original protocol, 0.02% (v/v) Tween 20 was used (Kohda </em><em>et al.</em><em>, 2007). We now routinely use Triton X-100 or DDM for better LLO solubilization. MgCl</em><sub><em>2</em></sub><em> may be replaced by MnCl</em><sub><em>2</em></sub><em>. The solution becomes slightly brown upon the addition of DTT, due to the formation of manganese dioxide, but it poses no problem for the assay. The omission of DTT has an insignificant effect on the assay, but the inclusion of DTT may be expected to clarify the migration image by its denaturing effects on viscous contaminants in the reaction mixture solution.</em> </li><li>Solubilization of LLO<ol class="npwthree"><li>The volume of the LLO stock in 10:10:3 (v/v/v) chloroform:methanol:water is calculated by 10 Î¼l (corresponding to 0.1-10 pmol of LLO) x "the number of samples <em>plus</em> 1 extra for the plastic tubes". The recommended amount of LLO is 10 pmol per 10 Î¼l for quantitative experiments. Usual number of samples is in the range from 10 to 20, in consideration of the number of lanes of one PAGE gel. </li><li>Transfer the LLO solution to a 1.5 ml polypropylene tube and dry it in a SpeedVac concentrator. Temperature is set between 30 Â°C and 37 Â°C.</li><li>Add the reaction buffer solution to the dried LLO in the tube. The volume of the reaction buffer solution is calculated by 8 Î¼l x "the number of samples <em>plus</em> 1 extra for the plastic tubes". </li><li>Sonicate the solution in a bath-type sonicator in continuous mode for 5 min at room temperature, and vortex vigorously to dissolve the LLO. </li><li>Centrifuge at 16,000 <em>x g</em>, 5 min, at 4 Â°C to remove insoluble materials. </li><li>Transfer the supernatant to a new tube, and add the detergent-solubilized OST to the supernatant. The volume of the detergent-solubilized OST is calculated by 1 Î¼l (corresponding to 0.01-1 pmol of the enzyme) x "the number of samples <em>plus</em> 1 extra for the plastic tubes". </li><li>Nine Î¼l aliquots of the supernatant are pipetted into 1.5 ml polypropylene tubes, precooled on ice. The use of an electronic pipette is highly recommended for accurate dispensing.<br/></li></ol></li><li>Starting and stopping the reaction<ol class="npwthree"><li>Add 3 Î¼l (corresponding to 30 pmol) of 10 Î¼M TAMRA-acceptor peptide solution to each polypropylene tube. This allows time-course analyses and peptide library experiments to be performed. Centrifuge briefly to spin down the content, tap the tube with a finger a few times for mixing, and centrifuge again.</li><li>Place the tubes on ice. In the case of a large number of tubes, it is convenient to use an aluminum heating/cooling block for 1.5 ml tubes. </li><li>Transfer the tubes (in the aluminum block) to an incubator. Incubate at the desired temperature (30 Â°C for the yeast and <em>Campylobacter</em> OSTs, 65 Â°C for thermostable OSTs from thermophilic archaea). The incubation time should be chosen within a time frame corresponding to the linear part of the progress curve, to measure the initial rate of the reaction (Figure 3). Typical incubation time is 30 min to 2 h. At a high temperature, it is important to heat the tubes uniformly to prevent water evaporation and condensation inside the tube. The reaction in the dark is desirable to avoid fluorescence quenching, but not mandatory.</li><li>Use a natural convection or forced air flow convection oven, or PCR device. The reaction is stopped by adding 3 Î¼l of 5x SDS-PAGE sample loading buffer to each tube, and heating it at 95 Â°C for 1 min. </li><li>Centrifuge the tubes briefly to spin down the contents. Optionally, EDTA (ethylenediaminetetraacetic acid) is added at a final concentration of 5-10 mM, before the addition of the SDS-PAGE sample loading buffer, to stop the reaction immediately. Note that placing the tubes (in an aluminum block) on ice is sometimes inadequate to stop the reaction completely.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190228/20190228004446_8596.jpg" width="190"/><br/><strong>Figure 3. Time-course of glycopeptide production.</strong> A. Glycopeptide products were separated from unmodified peptides by SDS-PAGE and quantified by fluorescence imaging. The fluorescent dye is attached to the peptide. The positions of the protein standard and the dye front are indicated. The contrast was adjusted with a sigmoid tone curve. This gel image is adapted from (Matsumoto <em>et al.</em>, 2013). B. The change in the amount of the glycopeptide product with time should produce a straight line for quantitative measurement.<br/><br/></li></ol></li></ol></li><li>Electrophoresis<br/>The heating in the presence of SDS is not necessary for PAGE separation of (glyco)peptides, but may be useful to clarify the migration image by its denaturing effects on miscellaneous proteins in the reaction mixture solution. <br/>  Load the whole volume (15 Î¼l) of the samples on the SDS-PAGE gels. If the sample volume decreases for any reason, a small volume of water or buffer may be added, but keep in mind that the entire sample must be loaded in one of the gel wells. Perform electrophoresis at about 23 mA per gel in the constant current mode for about 75 min. The optimal constant current is lower than the standard current for the separation of polypeptides, 30-40 mA. The appropriate setting of the electric current is essential for good separation of glycopeptide products from unreacted peptides. As a guideline, the time of electrophoresis is set for the bromphenol blue (BPB) front line to reach about 1 cm above the bottom of the gel. The run out of gel should be avoided because unreacted peptides will migrate close to the dye front line. The concurrent running of prestained molecular weight markers on the gel is not recommended, because some of the staining dyes are fluorescent.<br/>  Use handcasted polyacrylamide gels with a 4% acrylamide(AA)/bisacrylamide(bis) stacking gel and a 20% AA/bis resolving gel, or precast polyacrylamide gels with a 4% AA/bis stacking gel and a 15%-25% gradient AA/bis resolving gel. The high AA/bis percentage is important for the separation between glycopeptides and peptides. In general, precast gels are preferable to handcasted gels, because of the more homogeneous quality of the gel. Gels sandwiched between glass plates should be used, because plastic plates make the fluorescent imaging difficult due to light absorption. <br/>  Laemmli-SDS-PAGE based on the Tris-Glycine buffer system is usually used. Alternatively, Tricine-SDS-PAGE based on the Tris-Tricine buffer system is effective for better separation of glycopeptides. In fact, a peptide with just one monosaccharide residue can be resolved from the unmodified peptide (Gerber <em>et al.</em>, 2013). One should take precautions, however, the mobility of glycopeptides could be affected by sugar composition and branching pattern.<br/><br/></li><li>Detection by in-gel fluorescent imaging<br/>(Glyco)peptides are visualized in-gel using a fluorescence imager. It is unnecessary to detach the glass plates from the gel.<ol class="npwtwo"><li>Immediately after the electrophoresis finishes, clean the glass plates with water and then with ethanol. </li><li>Blow off the residual ethanol with an air (gas) duster. Do not use paper towels or cloths, because traces of the wiping will remain on the glass plates. </li><li>Set up the fluorescence imager as described in the manual. In the case of the LAS-3000 multicolor image analyzer (Fuji Film) with the TAMRA fluorophore, use Precision mode, Standard sensitivity, green LED (Green 520 nm EPI) illuminator, a 575DF20 filter, and Iris 0.85. The typical exposure time is 1 to 60 s. In principle, the exposure time must be optimized for any glycosylated, and non-glycosylated bands should not be overexposed for the accurate quantification of the bands.</li><li>Possible negative controls include the omission of the detergent-solubilized OST or LLO in the reaction mixture. Instead, peptides containing non-functional sequons, NPT or NPS, can be used. By contrast, the preparation of a positive control is not easy, since the chemical synthesis of glycopeptides are difficult and expensive. Alternatively, after gel extraction from the product bands, mass analysis using MALDI-TOF-MS or ESI-MS is the best way to identify the glycosylation of the substrate peptides.</li></ol></li></ol><p class="pptt" id="biaoti27150">### Data analysis</p><p class="ppzz">A typical fluorescent image of an SDS-PAGE gel is shown in Figure 4. The magenta color indicates the saturation of the fluorescent intensity. A peptide library experiment with low efficient sequons requires a long exposure time to detect the glycopeptide product bands. Note the small variation in the migration rates of peptides and glycopeptides with different amino acid sequences during electrophoresis. <br/>  Select the band boundary of a (glyco)peptide to obtain the fluorescence intensity, and in parallel, select a blank boundary with the same area for subtraction of the background noise. The ImageGauge software (Fuji Film) has a special freehand tool to set band boundaries and blank boundaries with a free curve (Figure 4C). <br/>  For the determination of the absolute quantity of (glyco)peptides by fluorescent imaging, a standard calibration curve can be made, using any peptide with the TAMRA fluorophore. Caution must be exercised when preparing serial dilutions of the TAMRA-peptide to avoid adsorption on plastic tubes: it should be mixed with the SDS-PAGE sample loading buffer first, and then diluted. Alternatively, the addition of small amount of Tween 20 (0.02%) is effective to prevent adsorption on plastic tubes.<br/><br/><img alt="" class="layerphoto" height="98" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190228/20190228004601_5114.jpg" width="135"/><br/><strong>Figure 4. Effects of the amino acid substitutions at the second position of the N-glycosylation sequon on the efficiency of the oligosaccharide transfer reaction.</strong> A. The original fluorescent image of the peptide library experiment performed in (Igura and Kohda, 2011a). The asterisk shows the unidentified bands, which only appear when the TAMRA fluorophore is attached to the peptides through the thiol-maleimide reaction. When the TAMRA fluorophore is directly attached to the N-terminal amino group through amide bond formation, no extra bands appear (as shown in <em>lanes</em> marked with <em>m</em>). B. Part of Figure 1A in (Igura and Kohda, 2011a) is redisplayed for reference. C. For background correction, band boundaries and blank boundaries were set by the Freehand Tool option of the ImageGauge software (Fuji Film). The contrast was adjusted with a sigmoid tone curve.</p><p class="pptt" id="biaoti27151">### Notes</p><ol class="npwone"><li>Oligosaccharyltransferase<br/>The animal, plant, and fungal OSTs are multi-subunit protein complexes consisting of eight different membrane subunit proteins (Larkin and Imperiali, 2011). In contrast, the lower eukaryotic protozoan OST is a single-subunit enzyme consisting solely of the catalytic subunit, and is referred to as <em>Stt3</em>. The archaeal and eubacterial OSTs are also single-subunit enzymes, consisting of AglB and PglB, respectively. Even though the catalytic subunits have different names, they have a common ancestor and share several conserved short amino-acid sequence motifs. Each genome usually encodes multiple paralogous genes of the catalytic subunit (and other subunits). Thus, it is better to prepare a recombinant OST protein using appropriate host cells. For example, the OST proteins from <em>Pyrococcus</em> (Igura and Kohda, 2011b), <em>Archaeoglobus</em> (Matsumoto <em>et al.</em>, 2013), and <em>Campylobacter</em> (Maita <em>et al.</em>, 2010; Gerber <em>et al.</em>, 2013) were successfully expressed in the membrane fractions of <em>E. coli</em> cells. Exceptionally, the yeast genome encodes only one <em>Stt3</em> gene. The yeast OST complex (containing either the Ost3 or Ost6 subunit) can be prepared from yeast cells (Harada <em>et al.</em>, 2013). Alternatively, one paralogous OST protein from crude membrane fractions of a natural source, for example <em>Pyrococcus furiosus</em>, can be purified by polyclonal antibodies (Igura <em>et al.</em>, 2008). The polyclonal antibodies can be obtained by immunization of rabbits with a fragment corresponding to the C-terminal soluble domain expressed in <em>E. coli</em> cells.<br/>  To prepare a large amount of a glycopeptide product, the crude membrane fractions, which contain both wild-type OST and LLO, prepared from any cultured cells are usable. The detergent solubilization of the crude membrane fractions is effective to improve the storage life of the membrane fractions. Note that the choice of detergent is critical for LLO solubilization. For the preparation of yeast and archaeal membrane fractions, Triton X-100 is preferable to digitonin or DDM for the co-solubilization of OST and LLO. The incubation with an acceptor peptide readily produces glycopeptide products. Serious side reactions, such as O-glycosylation, do not occur since the water-soluble compounds, such as nucleotide sugars, are washed away from the membrane fractions. One side reaction we encountered is the deamination of the C-terminal Î±-amide group. The addition of protease inhibitors is recommended, even though the oligosaccharide chains on asparagine effectively protect the glycopeptide from protease digestion. Incorporation of a biotin group during peptide synthesis is helpful for the purification of large quantities of glycopeptides (Fujinami <em>et al.</em>, 2017). <br/><br/></li><li>Sugar donor LLO<br/>Crude LLO preparations were described for yeast (Kelleher <em>et al.</em>, 2001), protists (Samuelson <em>et al.</em>, 2005), <em>Pyrococcus</em> (Igura <em>et al.</em>, 2008), <em>Archaeoglobus</em> (Matsumoto <em>et al.</em>, 2013), <em>Haloferax</em> (Guan <em>et al.</em>, 2010), and <em>Campylobacter</em> (Maita <em>et al.</em>, 2010; Gerber <em>et al.</em>, 2013). Briefly, cultured cells harvested by centrifugation are disrupted by glass beads or sonication, and cell debris is removed by centrifugation. Optionally, the supernatant is ultracentrifuged at 100,000 <em>x g</em>, and the membrane pellet is resuspended in buffer. The two-phase partitioning of a chloroform:methanol:water system produces the crude LLO fractions. The extracted crude LLOs can be further purified by anion exchange chromatography in a chloroform:methanol:water solvent system, and subsequently by NP-HPLC (normal phase high-performance liquid chromatography) using a chloroform:methanol:water:NH<sub>4</sub>OH solvent system (Guan <em>et al.</em>, 2010; Taguchi <em>et al.</em>, 2016). The crude and purified LLO solubilized in chloroform:methanol:water is stored in a glass container at -20 Â°C until use.<br/><br/></li><li>Sugar acceptor peptide<br/><ol class="npwtwo"><li>Design of sugar acceptor peptides<br/>The sugar acceptor peptide is usually prepared by custom peptide synthesis. Typically, it consists of three to ten amino acid residues containing an N-glycosylation sequon, Asn-X-Thr or Asn-X-Ser (X â  Pro). Just three residues corresponding to the N-glycosylation sequon can serve as a substrate, if the N-terminal Î±-amino group and C-terminal Î±-carboxy group are both blocked by an acetyl group and an amide group, respectively. In the original report (Kohda <em>et al.</em>, 2007), (TAMRA)-Gly-<u>Asn-Ser-Thr</u>-Val-Thr-NH<sub>2</sub> was used, where the underlined sequence is the N-glycosylation sequon, for the oligosaccharyl transfer assays of the yeast and <em>Pyrococcus</em> OSTs. The underlined sequence was changed to Gln-Ser-Thr, Asn-Pro-Thr, or Asn-Ser-Ala to test the N-glycosylation sequon as negative controls. The eubacterial <em>Campylobacter</em> N-glycosylation sequon requires an acidic residue, Asp or Glu, at the -2 position. (TAMRA)-Ala-<u>Asp-Gln-Asn-Ala-Thr</u>-Tyr-Lys-COOH and (5-CF)-Gly-Ser-<u>Asp-Gln-Asn-Ala-Thr</u>-Phe-NH<sub>2</sub> were used for the <em>Campylobacter</em> OSTs (Maita <em>et al.</em>, 2010; Gerber <em>et al.</em>, 2013). The 5-CF stands for 5-carboxyfluorescein, alias 5-FAM. Considering the solubility in acidic aqueous solutions and the sensitive detection by RP (reverse phase)-HPLC-ESI-MS (electron spray ionization mass spectrometry), the incorporation of lysine and arginine residues is recommended in the peptide sequence. For example, (TAMRA)-Ala-Ala-Tyr-<u>Asn-Val-Thr</u>-Lys-Arg-(Lys-biotin)-COOH was used for the <em>Pyrobaculum</em> OST (Fujinami <em>et al.</em>, 2017). <br/>  To make the peptides better substrates, Thr is preferable to Ser at the +2 position, and Asp is preferable to Glu at the -2 position. Pro must be avoided at the +1 (X) position and should be avoided at the -1 and +3 positions. The choice of the amino acid residue at the +1 (X) position has a high impact on the oligosaccharyl transfer activity. Several amino acid residues should be tested for selecting the best acceptor peptide. For example, in <em>in vitro</em> assay conditions, Val is the most favored amino acid residue at the +1 position for the <em>Pyrococcus</em> OST (Igura and Kohda, 2011b), Glu and Gln are best for the <em>Archaeoglobus</em> OST (Matsumoto <em>et al.</em>, 2017), and Ala, Ser, and positively charged residues are ideal for the <em>Campylobacter</em> OST (Chen <em>et al.</em>, 2007). Similarly, experimental analyses of an individual N-glycosylation sequon in a particular protein showed the pronounced effects of amino acid variations at the +1 position on the glycosylation efficiency with the canine OST, in microsomes supplemented with a rabbit reticulocyte cell-free translation system (Shakin-Eshleman <em>et al.</em>, 1996; Kasturi <em>et al.</em>, 1997). In contrast, the strong preference at the +1 position disappeared when the peptide was cross-linked to the <em>Archaeoglobus</em> OST protein (Matsumoto <em>et al.</em>, 2017), and was not detected in comprehensive statistical analyses of the eukaryotic and archaeal glycosylation sites (Ben-Dor <em>et al.</em>, 2004; Petrescu <em>et al.</em>, 2004; Abu-Qarn and Eichler, 2007). <br/>  In the original report, TAMRA (carboxytetramethylrhodamine, absorption/emission, 555 nm/580 nm) was attached to the N-terminal Î±-amino group during custom peptide synthesis (Kohda <em>et al.</em>, 2007). Frequently, the TAMRA fluorophore is a mixture of two isomers, 5- and 6-carboxytetramethylrhodamines, and the 5- and 6-isomer carrying peptides split into two peaks in the RP-HPLC chromatogram. We found that the slowly eluting peak of the TAMRA peptide yielded a clearer SDS-PAGE background in the OST assay. If the background of the fluorescence in-gel image is high, then the other isomer-carrying peptide should be used, to avoid contamination with unwanted fluorescent impurities during the RP-HPLC purification. Instead of the incorporation of TAMRA or another fluorescent dye during peptide synthesis, an additional cysteine residue can be incorporated in the peptide sequence. The sulfhydryl (SH) group of the cysteine residue is useful as an anchoring point for fluorescent labeling by a thiol-maleimide reaction (Figure 4). </li><li>Preparation of acceptor peptide solutions<br/>Dissolve lyophilized peptide powder in water. The pH of the peptide solution is very acidic (pH &lt; 3). Neutralize the acidic solution by serial additions of a small volume of alkaline solution (<em>e.g.</em>, 1 M NaOH). The use of pH test paper is sufficient. Store the concentrated peptide solution (10 mM) in conveniently sized aliquots at -20 Â°C or -80 Â°C. To prepare the working stock solution, dilute to the concentration of the stock to 10-100 Î¼M with 0.02% Tween 20. The working stock solution can be stored at 4 Â°C in a refrigerator for up to several weeks. However, long-term refrigerated storage results in degradation, and may produce high background levels in the fluorescent images of the SDS-PAGE gels. <br/>  The concentration of a TAMRA-peptide is determined by the absorbance at 555 nm, with an extinction coefficient of 90,000 M<sup>-1</sup> cm<sup>-1</sup> in aqueous solution. For example, 10 Î¼M of the TAMRA fluorophore corresponds to an absorbance of 0.9 at 555 nm. The concentration of a 5-CF (<em>alias</em> 5-FAM)-peptide is determined by the absorbance at 495 nm, with an extinction coefficient of 79,000 M<sup>-1</sup> cm<sup>-1</sup> in aqueous solution.<br/></li></ol></li></ol><p class="pptt" id="biaoti27152">### Recipes</p><ol class="npwtwok"><li>10:10:3 (v/v/v) chloroform:methanol:water<ol class="npwthree"><li>Measure and mix all solvent in glass containers, since chloroform will dissolve plastic</li><li>Store the solution in a capped glass container to prevent concentration changes due to the evaporation of chloroform and methanol</li></ol><em>Note: Always wear gloves and use a fume hood when handling chloroform, because it is toxic.</em> </li><li>5x SDS-PAGE sample loading buffer<br/>0.25 M Tris-HCl, pH 6.8<br/>5% SDS<br/>50% (w/v) glycerol<br/>0.125% (w/v) bromophenol blue<br/>Store working aliquots at room temperature. The final percentage of SDS should not exceed 1% to avoid band broadening</li><li>Running electrophoresis buffer<br/>0.025 M Tris-HCl, pH 8.3<br/>0.192 M glycine<br/>0.1% SDS<br/></li></ol><p class="pptt" id="biaoti27153">### Acknowledgments</p><p class="ppzz">This work was supported by JSPS KAKENHI Grants JP15083205 and JP26119002 (to DK).</p><p class="pptt" id="biaoti27154">### Competing interests</p><p class="ppzz">The authors declare there are no potential conflicts of interest.</p><p class="pptt" id="biaoti27155">### References</p><ol class="npwtwok"><li>Abu-Qarn, M. and Eichler, J. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17350928" target="_blank">An analysis of amino acid sequences surrounding archaeal glycoprotein sequons.</a> <em>Archaea</em> 2(2): 73-81.</li><li>Bause, E. and Hettkamp, H. (1979). <a href="http://www.ncbi.nlm.nih.gov/pubmed/520572" target="_blank">Primary structural requirements for N-glycosylation of peptides in rat liver.</a> <em>FEBS Lett</em> 108(2): 341-344.</li><li>Ben-Dor, S., Esterman, N., Rubin, E. and Sharon, N. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14514714" target="_blank">Biases and complex patterns in the residues flanking protein N-glycosylation sites.</a> <em>Glycobiology</em> 14(2): 95-101.</li><li>Chen, M. M., Glover, K. J. and Imperiali, B. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17439157" target="_blank">From peptide to protein: comparative analysis of the substrate specificity of N-linked glycosylation in <em>C. jejuni</em>.</a> <em>Biochemistry</em> 46(18): 5579-5585.</li><li>Cherepanova, N., Shrimal, S. and Gilmore, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27085638" target="_blank">N-linked glycosylation and homeostasis of the endoplasmic reticulum.</a> <em>Curr Opin Cell Biol</em> 41: 57-65.</li><li>Fujinami, D., Taguchi, Y. and Kohda, D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28510654" target="_blank">Asn-linked oligosaccharide chain of a crenarchaeon,<em> Pyrobaculum</em> calidifontis, is reminiscent of the eukaryotic high-mannose-type glycan.</a> <em>Glycobiology</em> 27(8): 701-712.</li><li>Geetha-Habib, M., Park, H. R. and Lennarz, W. J. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2380180" target="_blank"><em>In vivo</em> N-glycosylation and fate of Asn-X-Ser/Thr tripeptides.</a> <em>J Biol Chem</em> 265(23): 13655-13660.</li><li>Gerber, S., Lizak, C., Michaud, G., Bucher, M., Darbre, T., Aebi, M., Reymond, J. L. and Locher, K. P. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23382388" target="_blank">Mechanism of bacterial oligosaccharyltransferase: <em>in vitro</em> quantification of sequon binding and catalysis.</a> <em>J Biol Chem</em> 288(13): 8849-8861.</li><li>Guan, Z., Naparstek, S., Kaminski, L., Konrad, Z. and Eichler, J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21091511" target="_blank">Distinct glycan-charged phosphodolichol carriers are required for the assembly of the pentasaccharide N-linked to the Haloferax volcanii S-layer glycoprotein.</a> <em>Mol Microbiol</em> 78(5): 1294-1303.</li><li>Harada, Y., Buser, R., Ngwa, E. M., Hirayama, H., Aebi, M. and Suzuki, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24062310" target="_blank">Eukaryotic oligosaccharyltransferase generates free oligosaccharides during N-glycosylation.</a> <em>J Biol Chem</em> 288(45): 32673-32684.</li><li>Igura, M. and Kohda, D. (2011a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21115605" target="_blank">Quantitative assessment of the preferences for the amino acid residues flanking archaeal N-linked glycosylation sites.</a> <em>Glycobiology</em> 21(5): 575-583</li><li>Igura, M. and Kohda, D. (2011b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21357684" target="_blank">Selective control of oligosaccharide transfer efficiency for the N-glycosylation sequon by a point mutation in oligosaccharyltransferase.</a> <em>J Biol Chem</em> 286(15): 13255-13260.</li><li>Igura, M., Maita, N., Kamishikiryo, J., Yamada, M., Obita, T., Maenaka, K. and Kohda, D. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18046457" target="_blank">Structure-guided identification of a new catalytic motif of oligosaccharyltransferase.</a> <em>EMBO J</em> 27(1): 234-243.</li><li>Kasturi, L., Chen, H. and Shakin-Eshleman, S. H. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9163332" target="_blank">Regulation of N-linked core glycosylation: use of a site-directed mutagenesis approach to identify Asn-Xaa-Ser/Thr sequons that are poor oligosaccharide acceptors.</a> <em>Biochem J</em> 323 (Pt 2): 415-419.</li><li>Kelleher, D. J., Karaoglu, D. and Gilmore, R. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11358881" target="_blank">Large-scale isolation of dolichol-linked oligosaccharides with homogeneous oligosaccharide structures: determination of steady-state dolichol-linked oligosaccharide compositions.</a> <em>Glycobiology</em> 11(4): 321-333.</li><li>Kohda, D., Yamada, M., Igura, M., Kamishikiryo, J. and Maenaka, K. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17693440" target="_blank">New oligosaccharyltransferase assay method.</a> <em>Glycobiology</em> 17(11): 1175-1182.</li><li>Larkin, A. and Imperiali, B. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21506607" target="_blank">The expanding horizons of asparagine-linked glycosylation.</a> <em>Biochemistry</em> 50(21): 4411-4426.</li><li>Maita, N., Nyirenda, J., Igura, M., Kamishikiryo, J. and Kohda, D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20007322" target="_blank">Comparative structural biology of eubacterial and archaeal oligosaccharyltransferases.</a> <em>J Biol Chem</em> 285(7): 4941-4950.</li><li>Matsumoto, S., Shimada, A., Nyirenda, J., Igura, M., Kawano, Y. and Kohda, D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24127570" target="_blank">Crystal structures of an archaeal oligosaccharyltransferase provide insights into the catalytic cycle of N-linked protein glycosylation.</a> <em>Proc Natl Acad Sci U S A</em> 110(44): 17868-17873.</li><li>Matsumoto, S., Taguchi, Y., Shimada, A., Igura, M. and Kohda, D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27997792" target="_blank">Tethering an N-glycosylation sequon-containing peptide creates a catalytically competent oligosaccharyltransferase complex.</a> <em>Biochemistry</em> 56(4): 602-611.</li><li>Petrescu, A. J., Milac, A. L., Petrescu, S. M., Dwek, R. A. and Wormald, M. R. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14514716" target="_blank">Statistical analysis of the protein environment of N-glycosylation sites: implications for occupancy, structure, and folding.</a> <em>Glycobiology</em> 14(2): 103-114</li><li>Samuelson, J., Banerjee, S., Magnelli, P., Cui, J., Kelleher, D. J., Gilmore, R. and Robbins, P. W. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15665075" target="_blank">The diversity of dolichol-linked precursors to Asn-linked glycans likely results from secondary loss of sets of glycosyltransferases.</a> <em>Proc Natl Acad Sci U S A</em> 102(5): 1548-1553.</li><li>Schwarz, F., Lizak, C., Fan, Y. Y., Fleurkens, S., Kowarik, M. and Aebi, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20847188" target="_blank">Relaxed acceptor site specificity of bacterial oligosaccharyltransferase <em>in vivo</em>.</a> <em>Glycobiology</em> 21(1): 45-54.</li><li>Shakin-Eshleman, S. H., Spitalnik, S. L. and Kasturi, L. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8626433" target="_blank">The amino acid at the X position of an Asn-X-Ser sequon is an important determinant of N-linked core-glycosylation efficiency.</a><em> J Biol Chem</em> 271(11): 6363-6366.</li><li>Sharma, C. B., Lehle, L. and Tanner, W. (1981). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7018901" target="_blank">N-Glycosylation of yeast proteins. Characterization of the solubilized oligosaccharyl transferase.</a> <em>Eur J Biochem</em> 116(1): 101-108.</li><li>Taguchi, Y., Fujinami, D. and Kohda, D. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27015803" target="_blank">Comparative analysis of archaeal lipid-linked oligosaccharides that serve as oligosaccharide donors for asn glycosylation.</a> <em>J Biol Chem</em> 291(21): 11042-11054.</li><li>Wacker, M., Linton, D., Hitchen, P. G., Nita-Lazar, M., Haslam, S. M., North, S. J., Panico, M., Morris, H. R., Dell, A., Wren, B. W. and Aebi, M. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12459590" target="_blank">N-linked glycosylation in <em>Campylobacter jejuni</em> and its functional transfer into <em>E. coli</em>.</a> <em>Science</em> 298(5599): 1790-1793.<br/></li></ol>