<body>
            <h1>Preparation of RNA 3â End Sequencing Libraries of Total and 4-thiouracil Labeled RNA for Simultaneous Measurement of Transcription,  RNA Synthesis and Decay in <em>S. cerevisiae</em></h1>
            <p><strong>Authors:</strong> Manfred Schmid, Agnieszka Tudek, Torben Heick Jensen</p>
            <h2>Abstract</h2>
            <p>Cellular RNA levels are determined by the rates of RNA transcription from the gene template and subsequent RNA stability. Knowledge about both transcription and RNA decay is, therefore, necessary to interpret RNA levels and gene expression, especially during cellular processes where these parameters change. Numerous experimental strategies have been developed to measure transcription and RNA decay rates. However, to our knowledge, none of those techniques can simultaneously interrogate transcription and RNA decay. The presented protocol allows this and provides a simple approach to simultaneously estimate total RNA levels, transcription and decay rates from the same RNA sample. It is based on brief metabolic labeling of RNA and subsequent concurrent sequencing of polyA<sup>+</sup> and polyA<sup>-</sup> RNA 3â ends. The protocol was developed in <em>S. cerevisiae</em> and should be broadly applicable.</p>
            <p><strong>Keywords:</strong> Transcription, RNA synthesis, RNA decay, Metabolic labeling, RNA-seq, RNA 3â ends</p>
            <h2>Background</h2><p>All cells express a multitude of RNA species and control of their RNA levels inside cells is a central function of life. Cells control levels of individual RNA species via regulation of their transcription and RNA decay rates. Hence, measuring those parameters has been a traditional focus area in molecular biology, and numerous techniques have been developed to interrogate them. Among the many methods to measure transcription, chromatin immunoprecipitation (ChIP) of RNA polymerase and its higher resolution variant ChIP-exo (Rhee and Pugh, 2011), global run-on sequencing (GRO-seq, Core <em>et al.</em>, 2008) and its successor âPrecision nuclear Run On sequencing (PRO-seq, Kwak <em>et al.</em>, 2013), and nascent elongating transcript sequencing (NET-seq, Churchman and Weissman, 2011) are all readily applicable to genome-wide analysis and have, arguably, received most attention in recent years. RNA stability is usually measured by distinct methodology including pulse-chase type experiments or by following RNA decline after transcription shut-off. More recently, pulse-only application of metabolic labeling (<em>i.e.</em>, with 4-thiouridine or 5-bromouridine) has been shown to allow for measurement of RNA stability without employing a chase (Miller <em>et al.</em>, 2011; Sun <em>et al.</em>, 2012, Neymotin <em>et al.</em>, 2014; Rabani <em>et al.</em>, 2014).<br/> <br/> A major limitation of the above-mentioned approaches is that they rely on significantly different experimental approaches and measuring transcription and RNA decay, therefore, requires two different lines of experimentation. Hence, obtained measurements for transcription and decay rates are not directly comparable. <br/><br/> Recently, strategies based on brief metabolic labeling times combined with fragmentation of labeled RNA before purification (termed TT-seq or iTC-seq) were shown to provide a transcription-like measure from metabolic labeling experiments (Riising <em>et al.</em>, 2014, Schwalb <em>et al.</em>, 2016). However, even such an optimized procedure does not directly distinguish transcription from early post-transcriptional effects occurring during the labeling period.<br/><br/> To overcome these limitations, we developed a new approach, which is based on existing 4tU based protocols (Barrass <em>et al.</em>, 2015), that allows to assay transcription together with total RNA levels and RNA decay rates (Schmid <em>et al.</em>, 2018). Our method employ<em>s</em> RNA 3â end sequencing instead of conventional RNA-seq to measure RNA levels of both total and labeled RNA. Obtained data can be used to measure total RNA levels from the amount of pA<sup>+</sup> RNA 3â ends originating from gene 3â ends, RNA production from the amount of pA<sup>+</sup> RNA 3â ends originating from gene 3â ends in the labeled RNA fraction. Transcription can be directly estimated using the amount of pA<sup>-</sup> RNA 3â ends within gene bodies. Finally, RNA decay rates can be derived from comparing RNA production to total RNA levels, or, alternatively, by comparing transcription and total RNA levels. The approach is simple, based on standard molecular biology techniques and does not require specialized equipment. Hence, it should be applicable to a wide range of experimental questions and model organisms. Here, we provide a detailed protocol for the RNA-seq library preparation part of the protocol.<br/></p>
        </body>

<p class="pptt" id="biaoti27181">## Materials and Reagents</p><ol class="npwtwok"><li>2 ml and 1.5 ml Eppendorf tubes (Hounisen, catalog numbers: 72.695.500 and 72.706)</li><li>15 ml Falcon tubes</li><li>Zeba Spin Desalting Columns 7KMWCO (Thermo Scientific, catalog number: 89890)</li><li>Oligonucleotides used<br/><ol class="npwthree"><li><em>S. pombe</em> spike-in<br/>ACT1_<em>S. pombe</em>_fwd CCATTCTTGCTTCTCTTTCTACTTTCC (Sigma-Aldrich, custom-made)<br/>ACT1_<em>S. pombe</em>_rev CGCTCTCATCATACTCTTGCT (Sigma-Aldrich, custom-made)</li><li><em>S. cerevisiae</em> pA<sup>+</sup> RNAs<br/>HHF1_fwd ACTGCCCGGTTTTTCTTCT (Sigma-Aldrich, custom-made)<br/>HHF1_rev CCTAAACCCGCTATAATACACTCAT (Sigma-Aldrich, custom-made)<br/>PMA1_fwd TACTGTCGTCCGTGTCTGGATCT (Sigma-Aldrich, custom-made)<br/>PMA1_rev CCTTCATTGGCTTACCGTTCA (Sigma-Aldrich, custom-made)<br/>TLC1_pA<sup>+</sup>_fwd TACTGTCGTCCGTGTCTGGATCT (Sigma-Aldrich, custom-made)<br/>TLC1_pA<sup>+</sup>_rev CCTTCATTGGCTTACCGTTCA (Sigma-Aldrich, custom-made)</li><li><em>S. cerevisiae</em> non-polyadenylated RNAs<br/>SCR1_fwd CCCGGCTATAATAAATCGATCT (Sigma-Aldrich, custom-made)<br/>SCR1_rev GCTGACGCTGGATAAAACT (Sigma-Aldrich, custom-made)</li></ol></li><li>4-thiouracil (Sigma-Aldrich, catalog number: 440736)</li><li>Dry ice </li><li>Phenol, pH 4.3 (Sigma-Aldrich, catalog number: P4682)</li><li>Chloroform (Sigma-Aldrich, catalog number: 32211)</li><li>Sodium acetate (Sigma-Aldrich, catalog number: S-8625)</li><li>Sodium citrate (Merck, catalog number: 1.06448)</li><li>Sodium chloride (Merck, catalog number: 1.06404)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375)</li><li>Tris (VWR, catalog number: 103156X)</li><li>EDTA (Merck, catalog number: 1.08418) </li><li>Disodium hydrogen phosphate (Merck, catalog number: 1.06575)</li><li>Sodium dihydrogen phosphate (Merck, catalog number: 1.06345)</li><li>SDS (AppliChem, Panreac, catalog number: A1112)</li><li>Magnesium chloride (Merck, catalog number: 1.05833)</li><li>Glycogen (20 mg/ml) (Sigma-Aldrich, catalog number: G1508) (store at -20 Â°C)</li><li>Î²-mercaptoethanol (Sigma-Aldrich, catalog number: M6250)</li><li>MTSEA-biotin (Biotium, catalog number: 90064)</li><li>Dimethylformamide (DMF) (Merck, catalog number: 1.03053)</li><li>TURBO DNase free kit (Ambion, catalog number: AM1907M)</li><li>SuperScript II (Invitrogen, catalog number: 18064-014)</li><li>Platinum SYBR Green qPCR Super-Mix-UDG kit (Invitrogen, catalog number: 11733-046)</li><li><em>Escherichia coli</em> poly(A) polymerase kit (Thermo Fisher, catalog number: AM1350)</li><li>PureLink micro RNA purification kit (Ambion, catalog number: 12183018A)</li><li>Ribo-Zero Gold rRNA Removal kit for yeast (Illumina, catalog number: MRZY1306)</li><li>RiboLock RNase Inhibitor (Thermo Scientific, catalog number: E00381)</li><li>Lexogen QuantSeq 3â mRNA-Seq Library Prep Kit REV (Lexogen, catalog number: 016.96)</li><li>Dynabeads MyOne Streptavidin C1 (Invitrogen, catalog number: 65002)</li><li>Ethanol 96% (VWR, catalog number: 83804)</li><li>KOH (Merck, catalog number: P5958)</li><li>SC amino acid mix (Merck, catalog number: Y1501; or home-made equivalent)</li><li>Yeast nitrogen base (Merck, catalog number: Y0626)</li><li>DMSO (Merck, catalog number: W387520)</li><li>YPAD growth medium (Merck, catalog number: Y1375; or home-made equivalent)</li><li>Acetic acid (Merck, catalog number: 1005706)</li><li>NaOH (Merck, catalog number: S8045)</li><li>HCl (Merck, catalog number: H1758)</li><li>URA<sup>-</sup> medium (synthetic growth medium without uracil) (see Recipes)</li><li>4tU stock (see Recipes)</li><li>TES (see Recipes)</li><li><em>S. pombe</em> spike-in RNA (see Recipes)</li><li>RNA storage solution (see Recipes)</li><li>3 M NaOAc (see Recipes)</li><li>MTSEA-biotin stock (see Recipes)</li><li>1 M HEPES-KOH, pH 7.5 (see Recipes)</li><li>0.5 M EDTA-NaOH, pH 8 (see Recipes)</li><li>1 M NaPi buffer, pH 6.8 (see Recipes)</li><li>10x NaMg (see Recipes)</li><li>1x NaMgPS (see Recipes)</li><li>TEN1000 (see Recipes)</li><li>0.7 M Î²-mercaptoethanol (see Recipes)</li><li>20 mg/ml Glycogen (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27182">### Equipment</p><ol class="npwtwok"><li>DynaMag-2 magnet (Thermo Fisher Scientific, catalog number: 12321D)</li><li>Incubator (30 Â°C, with agitation for 500 ml flasks)</li><li>Centrifuge (for 50 ml tubes)</li><li>Microcentrifuge (cooled, for Eppendorf tubes)</li><li>Thermomixer (for Eppendorf tubes, heating up to 65 Â°C) </li><li>Rotating wheel </li><li>Real-time PCR system (Agilent, model: AriaMX, catalog number: G8830A)</li><li>Bioanalyzer (Agilent, model: Bioanalyzer 2100)</li><li>Qubit 2.0 Fluorometer (Life Technologies)</li><li>NanoDrop UV spectrophotometer</li><li>Centrifuge bucket (or flask)</li><li>Autoclave</li><li>Vortexer</li><li>4 Â°C refrigerator</li><li>-20 Â°C freezer</li><li>-70 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti27183">### Procedure</p><ol class="npwone"><li>Metabolic labeling of <em>S. cerevisiae</em> culture<br/><ol class="npwtwo"><li>Prepare 96% ethanol for fast inactivation of cells. Prepare for each culture 1x culture volume of ethanol in a centrifuge bucket (or flask) that can accommodate 2.5x or more of culture volume. Pre-chill these flasks to around -70 Â°C in a box with plenty of dry ice. </li><li>Use 100 ml culture in URA<sup>-</sup> media for a standard experiment.</li><li>Grow cells to an optical density (OD<sub>600</sub>) of 0.4 to 0.8.</li><li>Add 1/10,000 culture volume of 100 mM 4tU (<em>i.e.</em>, final concentration of 10 ÂµM) and mix thoroughly. <br/><em>Note: If you process several cultures in parallel, prepare and follow an exact time schedule to be able to precisely adhere to the desired labeling times. Remember to prepare one culture that has not been treated with 4tU (negative control) and process it as the other samples.</em> </li><li>Continue to grow for desired metabolic labeling time (usually 2 min to enrich strongly for nascent RNA). <br/><em>Note: Time of labeling can be optimized by testing several time points, e.g., 1, 2, 5 and 10 min and comparing the enrichment of desired targets (see <strong>[Optional] Quality Control</strong> section).</em> </li><li>Quickly inactivate cells by pouring culture into 1x volume of deep-chilled ethanol prepared in Step A1. Leave on normal ice until all samples ready to centrifuge. <br/><em>Note: The idea behind this approach is that simply putting cells on ice or the like will allow metabolic activity for an uncontrollable amount of time, whereas pouring into deep-chilled ethanol will instantaneously stop all activity and allow for precise control over labeling time.</em> </li><li>Spin at 1,500 <em>x g</em> for 3 min at 4 Â°C.</li><li>Remove supernatant, resuspend in 3 ml of ice-cold ddH<sub>2</sub>O and split the suspension into 3 x 2 ml Eppendorf tubes.</li><li>Spin cells in Eppendorf tubes at full speed for 12 s, and wash cells another 2 rounds each with 1 ml ice cold ddH<sub>2</sub>O.</li><li>Carefully remove supernatant. Store pellet at -70 Â°C or continue with RNA extraction.</li></ol><br/></li><li>Extract total RNA<br/><em>Note: The 3 aliquots from above are handled separately, and all phenolizations and precipitation in this section are done in 2 ml tubes. You will need 15 tubes per sample.</em><br/><ol class="npwtwo"><li>Resuspend cells in 800 Âµl TES. Add 800 Âµl acidic phenol and shake vigorously for 1 h at 65 Â°C in a thermomixer.</li><li>Spin for 5 min at full speed and 4 Â°C. Transfer top phase to a new tube and phenolize with another 700 Âµl acidic phenol by shaking vigorously for 5 min at 65 Â°C in a heated block.</li><li>Spin for 5 min as above.</li><li>Take top phase to a new tube and extract with 700 Âµl chloroform by vortexing vigorously for 20 s at room temperature. </li><li>Spin for 1 min as above.</li><li>Take top phase to a new tube and extract with 600 Âµl chloroform by vortexing vigorously for 20 s at room temperature. </li><li>Spin for 1 min as above.</li><li>Take 500 Âµl of the top phase to a new tube and precipitate RNA by adding 50 Âµl 3 M NaOAc and ice-cold 1.4 ml 96% ethanol.</li><li>Spin immediately for 20 min at full speed at 4 Â°C.<br/></li><li>Wash pellet twice with 1 ml 70% ethanol each.</li><li>Resuspend RNA in 100 Âµl RNA storage solution per tube. <br/><em>Note: Be careful to avoid overdrying the pellets, as very dry pellets are hard to dissolve. This can be done by first removing most ethanol with a pipet, re-spin briefly, remove any remaining ethanol with a pipet P200 and then leave tubes open for ca. 1 min open at RT before resuspension. Resuspension should be fast and require only a bit of shaking and/or pipetting up and down.</em> </li><li>Combine RNA from 3 aliquotes and measure the RNA concentration of a 1:5 dilution using, <em>i.e.</em>, a Nanodrop. <br/><em>Note: RNA concentration of the undiluted sample should be &gt; 1 Âµg/Âµl.</em> </li></ol><br/></li><li>Biotinylate 4sU RNA<br/><ol class="npwtwo"><li>Prepare coupling reaction by buffering 500 Âµg RNA with 4 Âµl 1 M HEPES-KOH pH 7.5 and 0.8 Âµl 0.5 M EDTA and fill with RNA storage solution to 400 Âµl. <br/><em>Note: If 500 Âµg RNA is not available, the protocol is doable with 300 Âµg or even less.</em> </li><li>Optional: Add spike-in RNA. We usually use 3-5 Âµg (1/100 w/w) <em>S. pombe</em> spike-in RNA (see Recipes and Data analysis).</li><li>Add 10 Âµl of the 5 mg/ml MTSEA-biotin stock, vortex to mix and incubate for 30 min at 25 Â°C in the dark.</li><li>During this incubation prepare Zeba columns, place Zeba columns in 15 ml Falcon tubes. Drain and wash each column 3 x with 2 ml RNA storage solution each. For each wash: apply liquid and spin 2 min at 1,000 <em>x g</em>.</li><li>Apply MTSEA coupling reactions onto drained Zeba columns, let liquid soak in resin for 2 min, add then another 130 Âµl RNA storage solution, spin for 2 min at 1,000 <em>x g</em> in a fresh 15 ml Falcon and recover the flow-through which contains biotin-coupled RNA devoid of free MTSEA-biotin coupling reagent. Precipitate the flow-through (~500 Âµl) with 50 Âµl 3 M NaOAc and 1.4 ml 96% ethanol. Incubate for 30 min at 4 Â°C or overnight at -20 Â°C. Spin for 20 min at 4 Â°C, full speed.</li><li>Wash 2 x 1 h at RT each with 1 ml 80% ethanol in the dark. Spin for 15 min at 4 Â°C, 16,000 <em>x g</em> between washes.<br/><em>Note: As above, take care to void overdrying the pellet, </em><em>i.e.</em><em>, simply leave tubes with pellet open at RT for 1 min and then resuspend.</em> </li><li>Resuspend RNA in 250 Âµl RNA storage solution.</li><li>Take 10 Âµl of input (âTotal RNAâ) aliquot.</li></ol><br/></li><li>Isolation of the biotinylated RNA<br/><ol class="npwtwo"><li>Add to 240 Âµl RNA sample: 30 Âµl 10x NaMg, 30 Âµl 1 M NaPi, 3 Âµl 10% SDS.<br/><em>Note: This results in the RNA sample in ~300 Âµl with a solvent composition equivalent to 1x NaPSMg.</em> </li><li>Prepare magnetic streptavidin-coupled beads (adjust the amount depending on number of samples):<br/><ol class="npwthree"><li>Use 50 Âµl Dynabeads MyOne Streptavidin C1 slurry per sample.</li><li>Drain beads by applying on magnet and pipetting off the liquid.</li><li>Wash first 1 x with 400 Âµl 1x NaMgPS.</li><li>Block beads with 200 Âµl 1x NaMgPS <sup>+</sup> 10 Âµl 20 mg/ml glycogen.</li><li>Rotate for 20 min at RT.</li><li>Remove blocking solution from magnetic beads.</li><li>Wash 1 x with 400 Âµl NaMgPS.</li></ol></li><li>Drain beads and add RNA immediately.</li><li>Let the RNAs bind to beads for 30 min at room temperature by gentle agitation, <em>i.e.</em>, on a rotating wheel in the dark.</li><li>Remove supernatant from magnetic beads.</li><li>Wash 5 x with 400 Âµl 1x NaMgPS each.</li><li>Wash 1 x with 400 Âµl TEN1000.</li><li>Elute biotinylated RNA by incubating 2 x with 50 Âµl Î²-mercaptoethanol. Incubate for 5 min at RT before removing eluate at each step.</li><li>Combine eluates and precipitate RNA with 10 Âµl 3 M NaOAc, pH 5, 2.5 Âµl 20 mg/ml glycogen and 250 Âµl 96% ethanol.</li><li>Incubate on dry ice for 30 min or -20 Â°C overnight.</li><li>Spin for 30 min at full speed, 4 Â°C.</li><li>Wash RNA pellet 2 x with 75% ethanol, to remove Î²-mercaptoethanol as much as possible.</li><li>Resuspend in 20 Âµl RNA storage solution.</li><li>DNase-treat 2 Âµl (4 Âµg) of the total RNA aliquot and 17.6 Âµl (300 ng) of the 4tU IP samples using the TURBO DNase kit from Ambion, according to the manufacturerâs instructions. </li><li>Optional: Detect RNA quality and concentration using a Bioanalyzer and the RNA pico kit (Agilent), Qubit or similar (see Figure 1). </li><li>Optional: use 2 Âµl of elution RNA for quality control (see <strong>[Optional] Quality Control</strong> section).</li></ol><br/></li><li><em>In vitro</em> polyadenylation with E-PAP<br/><ol class="npwtwo"><li>To 5 Âµl of DNase-treated RNA, add 4 Âµl 5x E-PAP buffer, 2 Âµl 25 mM MnCl<sub>2</sub>, 0.2 Âµl E-PAP (2 U/Âµl) from the <em>Escherichia coli</em> poly(A) polymerase kit (Thermo Fisher), 2 Âµl 10 mM ATP, 0.2 Âµl RiboLock and 6.6 Âµl RNase-free ddH<sub>2</sub>O.</li><li>Mix and incubate for 30 min at 30 Â°C.</li><li>Purify RNA from polyadenylation reactions using PureLink RNA purification kit micro (Ambion) according to the manufacturerâs instructions and elute in 22 Âµl RNase-free ddH<sub>2</sub>O.</li></ol><br/></li><li>Ribo-depletion<br/><em>Note: The protocol below is based on RiboGold kit from Illumina, which is no longer commercially available. And it is also applicable to other similar products (e.g., RNA-seq kit).</em><br/><ol class="npwtwo"><li>Carry out ribo-depletion using Ribo-Zero Gold rRNA Removal Kit for Yeast (Illumina) essentially as per the manufacturerâs instruction but using a downscaled version.</li><li>Mix 18 Âµl of eluate with 2 Âµl reaction buffer, 2 Âµl of removal solution and 8 Âµl of RNase-free ddH<sub>2</sub>O. </li><li>Incubate for 10 min at 65 Â°C and cool to room temperature for 5 min.</li><li>Add 65 Âµl of rRNA magnetic beads to the reaction mix from Step F3 and first incubate for 5 min at room temperature and then 5 min at 50 Â°C. Transfer the unbound solution to a fresh tube containing 40 Âµl ddH<sub>2</sub>O and 10 Âµl 3 M NaOAc and 20 Âµg glycogen.</li><li>Add 300 Âµl ethanol, precipitate RNA for 20 min at -20 Â°C and spin for 30 min at 4 Â°C.</li><li>Wash pellet twice with 70% ethanol.</li><li>Resuspend pellet in 10 Âµl of RNA storage solution.</li></ol><br/></li><li>3â end RNA-seq library preparation and sequencing<br/><ol class="npwtwo"><li>Use 5 Âµl of IP or 1 Âµl of input aliquots as starting RNA material and prepare RNA-seq libraries using QuantSeq REV kit (Lexogen) according to the manufacturerâs instructions. </li><li>Sequence samples on Illumina HiSeq using single-read 50 nt reads or a similar machine with custom primers as defined by the QuantSeq REV specifications. Aim for around 5 million primary reads per each library to achieve necessary depth for <em>S. cerevisiae</em> experiments.<br/><br/><img alt="" class="layerphoto" height="113" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190310/20190310234614_3243.jpg" width="180"/><br/><strong>Figure 1. Example of Bioanalyzer traces of final sequencing libraries.</strong> Libraries shown are from samples that were not subjected to E-PAP treatment. Lanes 1-4 are from total RNA and lanes 5-11 from 10â 4tU IP samples. All samples were successfully used for Illumina sequencing.<br/></li></ol></li></ol><br/><p class="ppzz"><strong>(Optional) Quality control of 4tU labeled RNAs</strong><strong><br/></strong> </p><ol class="npwtwo" style="margin-left:20px;"><li>Use 2 Âµl of input and elution sample. </li><li>Perform reverse transcription using Superscript II or equivalent and an Oligo(dT)<sub>18</sub> according to manufacturersâ instructions.</li><li>Dilute RT reactions in 100 Î¼l nuclease-free water and use 2 Âµl if diluted RTs per 16 Âµl qPCR Platinum SYBR Green qPCR Super-Mix-UDG kit reactions according to manufacturersâ instructions. Prepare reactions with 1:10 diluted RT for oligos against mRNAs and RNAPIII non-adenylated transcript SCR1. <br/></li></ol><p class="pptt" id="biaoti27184">### Data analysis</p><p class="ppzz">RNA concentration equivalents of various targets are estimated from RT-qPCR values using the formula RNA<sub>conc</sub> = 2<sup>(-Ct)</sup>; optionally including a factor for correction with measured primer efficiency. Those values are then normalized relative to the values obtained for spike-in estimate reaction against <em>S. pombe</em><em> ACT1</em> mRNA. The use of external spike-ins for normalization permits direct comparison of IP efficiencies across samples and labeling times and allows for removal of technical noise (<em>i.e.</em>, IP efficiency) that may arise due to the complex nature of the experiment. Normalized values from negative control elutions (<em>i.e.</em>, from samples without 4tU) are used to estimate the background in elution samples. Several mRNAs should be significantly enriched above background, which indicates that the experiment has worked. Note, the goal is to enrich strongly for nascent (<em>i.e.</em>, pA<sup>-</sup>) 4tU-labeled RNAs, which is achieved with short labeling times. Enrichment of mature RNAs (pA<sup>+</sup> mRNA, full-length snRNAs) is most efficient with increased labeling times (see Figure 2). That is, for 10 min labeling the vast majority (&gt; 90%) of transcripts are enriched significantly above background. For 2 min labeling, the enrichment of pA<sup>+</sup> RNAs above background is less significant and the number of transcripts detected significantly above background varies significantly between technical repeats. Usually, at least ~20% of all transcripts are found significantly enriched. Hence, we usually use 2 min labeling times to enrich for nascent RNAs (where many, but not all mature RNAs are also significantly enriched above background) and 10 min labeling times when only measuring full-length RNAs. <br/>  To control <em>in vitro</em> polyadenylation reactions (E-PAP treatment), ratio of non-adenylated <em>SCR1</em> relative to mRNAs is compared. Those species should be enriched at least 10-fold relative to mRNAs in E-PAP treated samples.<br/><br/><img alt="" class="layerphoto" height="157" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190310/20190310234803_4665.jpg" width="180"/><br/><strong>Figure 2. Recovery of RNAs after 2 and 10 min labeling and efficiency of E-PAP treatment.</strong> A. Increase in labeling efficiency in function of 4tU labeling time. Wild-type cells were treated with 4tU for 0, 2 and 10 min, respectively. The graph depicts IP efficiencies (depicted as % input) from mock, 2â 4tU and 10â 4tU labeling times for spike in (SpACT1, for <em>S. pombe</em> <em>ACT1</em>) and <em>S. cerevisiae</em> mRNAs <em>PMA1</em> and <em>HHF1</em>, as well as for the polyadenylated precursor of telomerase RNA <em>TLC1</em>. B. RT-qPCR of total RNA before and after treatment with E-PAP. RNAs were reverse transcribed with Oligo(dT)<sub>18</sub> and quantified by qPCR with primers for <em>SCR1</em> (a non-polyadenylated RNAPIII product) and PMA1 (a polyadenylated mRNA). The non-polyadenylated <em>SCR1</em> RNA is detected &gt; 1,000 times more efficiently after E-PAP treatment, whereas <em>PMA1</em> signal is not affected.</p><p class="pptt" id="biaoti27185">### Recipes</p><ol class="npwtwok"><li>URA<sup>-</sup> medium (synthetic growth medium without uracil)<br/>0.2% (w/v) SC amino acid mix without uracil<br/>2% (w/v) glucose<br/>0.67 g (w/v) yeast nitrogen base<br/>Dissolve in ddH<sub>2</sub>O, autoclave and store at room temperature</li><li>4tU stock<br/>100 mM 4-thiouracil in DMSO, store at -20 Â°C. Warm to RT before use</li><li>TES<br/>10 mM Tris-HCl, pH 7.5<br/>1 mM EDTA, 1% (w/v) SDS<br/>Prepare from RNase-free stocks and keep at RT</li><li><em>S. pombe</em> spike-in RNA<br/><ol class="npwthree"><li>Grow standard <em>S. pombe</em> cells to exponential growth at 30 Â°C in YPAD growth medium and labeled for 15 min with 5 mM (w/v) final 4tU</li><li>Prepare total RNA as described for <em>S. cerevisiae</em> above and store aliquots in RNA storage solution at -20 Â°C</li></ol></li><li>RNA storage solution<br/>1 mM Na-citrate, pH 6.4<br/>Prepare from RNase-free stocks and keep at RT</li><li>3 M NaOAc<br/>3 M Sodium acetate adjusted to pH to 5 with acetic acid<br/>Prepare from RNase-free stocks and keep at RT</li><li>MTSEA-biotin stock<br/>5 mg/ml in DMF<br/>Store at -20 Â°C</li><li>1 M HEPES-KOH, pH 7.5<br/>1 M HEPES adjusted to pH 7.5 with KOH<br/>Prepare from RNase-free stocks and keep at RT</li><li>0.5 M EDTA-NaOH<br/>0.5 M EDTA adjusted to pH 8 with NaOH<br/>Prepare from RNase-free stocks and keep at RT</li><li>1 M NaPi buffer, pH 6.8<br/>Mix 1 M Na<sub>2</sub>HPO<sub>4</sub> with 1 M NaH<sub>2</sub>PO<sub>4</sub> to reach pH 6.8<br/>Prepare from RNase-free stocks and keep at RT</li><li>10x NaMg<br/>100 mM Tris-HCl, pH 7.0<br/>2 M NaCl<br/>250 mM MgCl<sub>2</sub> <br/>Prepare from RNase-free stocks and keep at RT</li><li>1x NaMgPS<br/>1x NaMg (1:10 from 10x)<br/>100 mM NaPi (1:10 from 1 M)<br/>0.1% SDS (1:100 from 10%)<br/>Prepare fresh before use in RNase-free H<sub>2</sub>O and keep at room temperature<br/><em>Note: Can form precipitate, which can be redissolved by gentle heating.</em> </li><li>TEN1000<br/>10 mM Tris, pH 7.5<br/>0.5 mM EDTA<br/>1 M NaCl<br/>Prepare from RNase-free stocks and keep at RT</li><li>0.7 M Î²-mercaptoethanol<br/>1:20 dilution of pure Î²-mercaptoethanol in RNA storage solution. Prepare fresh</li><li>20 mg/ml Glycogen<br/>Prepare in RNase-free H<sub>2</sub>O and keep at -20 Â°C<br/></li></ol><p class="pptt" id="biaoti27186">### Acknowledgments</p><p class="ppzz">We thank David Barrass and Jean Beggs for the initial 4tU labeling and IP protocol and Rikke Jespersen and Dorthe Caroline RiishÃ¸j for technical support. Work in the T.H.J. laboratory was supported by the Danish National Research Council, the Lundbeck Foundation (grant R171-2014-553) and the ERC (grant 339953).</p><p class="pptt" id="biaoti27187">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti27188">### References</p><ol class="npwtwok"><li>Barrass, J. D., Reid, J. E., Huang, Y., Hector, R. D., Sanguinetti, G., Beggs, J. D. and Granneman, S. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26679539" target="_blank">Transcriptome-wide RNA processing kinetics revealed using extremely short 4tU labeling.</a> <em>Genome Biol</em> 16: 282.</li><li>Churchman, L. S. and Weissman, J. S. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21248844" target="_blank">Nascent transcript sequencing visualizes transcription at nucleotide resolution.</a> <em>Nature</em> 469(7330): 368-373.</li><li>Core, L. J., Waterfall, J. J. and Lis, J. T. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19056941" target="_blank">Nascent RNA sequencing reveals widespread pausing and divergent initiation at human promoters.</a> <em>Science</em> 322(5909): 1845-1848.</li><li>Kwak, H., Fuda, N. J., Core, L. J. and Lis, J. T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23430654" target="_blank">Precise maps of RNA polymerase reveal how promoters direct initiation and pausing.</a> <em>Science</em> 339(6122): 950-953.</li><li>Miller, C., Schwalb, B., Maier, K., Schulz, D., Dumcke, S., Zacher, B., Mayer, A., Sydow, J., Marcinowski, L., Dolken, L., Martin, D. E., Tresch, A. and Cramer, P. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21206491" target="_blank">Dynamic transcriptome analysis measures rates of mRNA synthesis and decay in yeast.</a> <em>Mol Syst Biol</em> 7: 458.</li><li>Neymotin, B., Athanasiadou, R. and Gresham, D. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25161313" target="_blank">Determination of in vivo RNA kinetics using RATE-seq. </a><em>RNA</em> 20(10): 1645-1652.</li><li>Rabani, M., Raychowdhury, R., Jovanovic, M., Rooney, M., Stumpo, D. J., Pauli, A., Hacohen, N., Schier, A. F., Blackshear, P. J., Friedman, N., Amit, I. and Regev, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25497548" target="_blank">High-resolution sequencing and modeling identifies distinct dynamic RNA regulatory strategies.</a> <em>Cell</em> 159(7): 1698-1710.</li><li>Rhee, H. S. and Pugh, B. F. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22153082" target="_blank">Comprehensive genome-wide protein-DNA interactions detected at single-nucleotide resolution.</a> <em>Cell</em> 147(6): 1408-1419.</li><li>Riising, E. M., Comet, I., Leblanc, B., Wu, X., Johansen, J. V. and Helin, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24999238" target="_blank">Gene silencing triggers polycomb repressive complex 2 recruitment to CpG islands genome wide. </a><em>Mol Cell</em> 55(3): 347-360.</li><li>Schmid M, Tudek A, Jensen TH. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30157438" target="_blank">Simultaneous measurement of transcriptional and post-transcriptional parameters by 3' end RNA-seq.</a> <em>Cell Reports</em> 24(9): 2468-2478.</li><li>Schwalb, B., Michel, M., Zacher, B., Fruhauf, K., Demel, C., Tresch, A., Gagneur, J. and Cramer, P. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27257258" target="_blank">TT-seq maps the human transient transcriptome.</a> <em>Science</em> 352(6290): 1225-1228.</li><li>Sun, M., Schwalb, B., Schulz, D., Pirkl, N., Etzold, S., Lariviere, L., Maier, K. C., Seizl, M., Tresch, A. and Cramer, P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22466169" target="_blank">Comparative dynamic transcriptome analysis (cDTA) reveals mutual feedback between mRNA synthesis and degradation.</a> <em>Genome Res</em> 22(7): 1350-1359.<br/></li></ol>