<body>
            <h1>Quantification of Queuosine Modification Levels in tRNA from  Human Cells Using APB Gel and Northern Blot</h1>
            <p><strong>Authors:</strong> Zaneta Matuszek, Tao Pan</p>
            <h2>Abstract</h2>
            <p>Queuosine (Q) is a hypermodified base in the wobble anticodon position of tRNAs coding for the amino acids Tyr, His, Asn, and Asp. tRNA Q-modification is introduced by a queuine tRNA-ribosyltransferase (TGT) that replaces the guanine base at G34 at these tRNAs with the modified base. tRNA Q-modification is widely distributed among prokaryotic and eukaryotic organisms, but only bacteria synthesize Q-modified tRNA <em>de novo.</em> In mammals, tRNA Q-modifications strictly rely on the presence of gut microbiomes or diets to produce the queuine base. Despite decades of study, cellular roles of tRNA Q-modification are still not fully understood. Here we describe a method to quantify tRNA Q-modification levels in individual tRNAs from human cells based on the presence of a <em>cis</em>-diol in the Q modification. This <em>cis</em>-diol moiety slows modified tRNA migration through polyacrylamide gels supplemented with <em>N</em>-acryloyl-3-aminophenylboronic acid (APB) compared to the unmodified tRNA. This difference can be visualized by Northern blots using probes for specific tRNA.</p>
            <p><strong>Keywords:</strong> Queuosine, tRNA modification, tRNA, Queuine, Microbiome</p>
            <h2>Background</h2><p>Transfer RNAs (tRNA) contain nucleotide modifications that perform a wide range of roles in translation and in the generation of tRNA fragments that are functional small RNAs. Queuosine (Q, Figure 1) is a tRNA modification that occurs in the wobble anticodon position of tRNAs coding for amino acids Tyr, His, Asn, and Asp. All these tRNAs contain G<sup>34</sup>U<sup>35</sup>N<sup>36</sup> anticodon sequences. In bacteria, G34-to-Q34 modification can be synthesized <em>de novo</em> through a cascade of enzyme reactions (El Yacoubi <em>et al.</em>, 2012). In mammals, the G34 base in these tRNAs can be replaced with queuine (Figure 1) derived from either gut microbiome or diet by a two-protein complex enzyme QTRT1 (catalytic subunit) and QTRT2 encoded in the host genome. Q is widely distributed among prokaryotic and eukaryotic organisms, but its cellular roles are still not fully understood (Rakovich <em>et al.</em>, 2011; Fergus <em>et al.</em>, 2015; Tuorto <em>et al.</em>, 2018; Wang <em>et al.</em>, 2018). For example, tRNA Q-modification levels have been correlated with cancer progression and cellular metabolism.<br/><br/>To understand the role of Q-tRNA modification in mammalian cells, we performed comparative studies of cells having various levels of Q-modification (Wang <em>et al.</em>, 2018). We found that tRNAs in cultured cell lines are Q-modified at 5%-60% in standard culture medium. Using dialyzed serum medium which was deprived of small molecules like queuine and continued passage of cell divisions, tRNA Q-modification levels continuously decreased until they became undetectable (0Q cells). The addition of 0.1-1 ÂµM queuine to the media restored Q-modification to 100% levels within 12-24 h (100Q cells). To measure the level of tRNA Q-modification in individual tRNA, we used a previously developed method on the basis of the structure of queuosine (Igloi and Kossel, 1985; Zaborske <em>et al.</em>, 2014). <em>Cis</em>-diol moieties, such as the 3â ribose of a tRNA, slow migration through polyacrylamide gels supplemented with <em>N</em>-acryloyl-3-aminophenylboronic acid (APB). Since queuosine has an additional ribose moiety, the migration of Q-modified tRNA in APB gel is slower compared to unmodified tRNA. This differential migration of Q-modified versus unmodified tRNA can be determined by Northern blot analysis using probes specific for each tRNA. The mammalian tRNA<sup>Asp</sup> and tRNA<sup>Tyr</sup> are further glycosylated at the <em>cis</em>-diol of the modified base with mannose (man-Q) or galactose (gal-Q) which prevents the reaction between the Q-base and APB; our approach therefore only works for tRNA<sup>His</sup> and tRNA<sup>Asn</sup> in mammalian cells. Using this method and other techniques, we showed that Q-modification protects cognate tRNAs against ribonuclease cleavage <em>in vitro</em> and in cells which alters the abundance of tRNA fragments (tRF) in human cells (Wang <em>et al.</em>, 2018).<br/><br/>Previously, the Q-content in total tRNA in mammalian cells was evaluated by taking advantage of the irreversible nature of the N-glycosidic bond of the Q-base to the sugar-ribose backbone. Total tRNAs were isolated and the extent of Q-modified in total tRNA was measured by the exchange of <sup>3</sup>H-guanine with the guanine base of unmodified tRNAs catalyzed by the <em>E. coli</em> TGT enzyme. This exchange reaction was blocked by the Q-modification of the same tRNAs. Total Q-content was estimated based on the fraction of tRNAs that did not show exchange. This approach was first used to determine the Q-modification content in mammalian tissues and bacterial cells during different stages of development and aging (Singhal <em>et al.</em>, 1981). Although useful, this previous method could not measure Q-modification level of individual tRNA. In contrast, our APB-gel based method directly measures the Q-modification levels of individual tRNA species starting with as little as 1-2 Âµg of total RNA.<br/><br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190318/20190318004520_6413.jpg" width="200"/><br/> <strong>Figure 1. Queuosine (Q) tRNA modification in four tRNAs.</strong> Diagram of G34 to Q34 modifications in the wobble anticodon position of tRNA<sup>His/Asn/Asp/Tyr</sup>. Queuine corresponds to the nucleobase of Q-nucleotide. QTRT1 and QTRT2 (formerly QTRTD1) proteins are encoded in the human genome. Black dots represent the tRNA body; the QTRT1/QTRT2 complex introduces Q modification in the place of G using queuine as substrate. Q34 in tRNA<sup>Asp</sup> and tRNA<sup>Tyr</sup> are further glycosylated at the <em>cis</em>-diol of the modified base with mannose (man-Q) or galactose (gal-Q) in mammals. Our method measures tRNA<sup>His</sup> and tRNA<sup>Asn</sup>, but not tRNA<sup>Asp</sup> and tRNA<sup>Tyr</sup> because of their glycosylation.<br/></p>
        </body>

<p class="pptt" id="biaoti27203">## Materials and Reagents</p><p class="ppzz"><em>Note: All reagents were purchased from Thermo Fisher if not indicated otherwise and were stored at room temperature.</em> </p><ol class="npwtwok"><li>Pipette tips</li><li>Plastic foil</li><li>Syringe</li><li>Gel spacers 0.8 mm (Cole-Parmer, UX-28573-12)</li><li>Hybridization tubes 2 x 200 mm (VWR, catalog number: 32645-030)</li><li>Whatman paper (Whatman<sup>TM</sup>, 10427812 Grade)</li><li>Hybond-XL membrane (GE Healthcare, catalog number: 45-001-148) </li><li>5â <sup>32</sup>P-labeled oligonucleotide probes (see Procedure B)</li><li>[Î³-<sup>32</sup>P] ATP (PerkinElmer, catalog number: NEG035C005MC) (storage: -20 Â°C)</li><li>T4 Polynucleotide kinase (10 U/Âµl) (Thermo Fisher, catalog number: EK0032) (storage: -20 Â°C)</li><li>1x T4 PNK buffer</li><li>3-Acrylamidophenylboronic acid (Frontier Scientific, catalog number: A10094-5g) (storage: 4 Â°C)</li><li>40% acrylamide/bis solution, 29:1 (Bio-Rad, catalog number: 161-0146) </li><li>Acetic acid </li><li>DMEM (Thermo Fisher, catalog number: 11965092)</li><li>EDTA </li><li>Ethanol, 95.5% USP (Ricca, catalog number: R2916000-1A) </li><li>Fetal Bovine Serum dialyzed (Thermo Fisher, catalog number: 26400044)</li><li>Queuine Hydrochloride (Toronto Research Chemicals, catalog number: Q525000)</li><li>Sodium acetate </li><li>Sodium chloride (NaCl) </li><li>Sodium dodecyl sulfate (SDS) </li><li>Sodium phosphate monobasic (NaH<sub>2</sub>PO<sub>4</sub>) (Thermo Fisher, catalog number: BP329)</li><li>Sodium phosphate dibasic (Na<sub>2</sub>HPO<sub>4</sub>) (Thermo Fisher, catalog number: S374)</li><li>10x TAE buffer (Sigma-Aldrich, catalog number: 11666690001)</li><li>Tris (Bioland Scientific, catalog number: CT01-5kg)</li><li>Urea </li><li>Bromophenol blue</li><li>Xylene cyanol</li><li>10x TBE buffer (EMD Millipore, catalog number: 8850-OP)</li><li>Sodium acetate (KOAc)</li><li>Sodium chloride (KCl)</li><li>Nuclease-free water</li><li>2x Gel loading buffer (see Recipes)</li><li>10% denaturing polyacrylamide gel (see Recipes)</li><li>10% APB polyacrylamide gel (see Recipes)</li><li>Dialyzed serum medium (see Recipes)</li><li>Gel extraction buffer (see Recipes)</li><li>Hybridization buffer (see Recipes)</li><li>1 M Sodium phosphate buffer (pH 7.0) (see Recipes)</li><li>1 M Tris-HCl buffer (pH 9.0) (see Recipes)</li><li>Washing solution (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27204">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>Microcentrifuge (Eppendorf, model: 5415D)</li><li>Hybridization Oven (UVP, model: 95-0030-01)</li><li>Phosphoimager (Bio-Rad, Molecular Imager Pharosfx Plus System, 170-9460) </li><li>Phosphor screens (Bio-Rad or Fuji) </li><li>X-ray Cassette K 35 x 43 cm (Bio-Rad)</li><li>Polyacrylamide gel electrophoresis apparatus with glass plates 17 x 15 cm (Biometra, Model V16-2)</li><li>Thermoblock 24 x 1.5 ml (Eppendorf, 022670522)</li><li>Tube rotator (Cole-Parmer Ltd, model: SB3)</li><li>Vacuum gel dryer (Bio-Rad, Model 583 Gel Dryers)</li><li>Incubator </li><li>4 Â°C refrigerator</li><li>-20 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti27205">### Software</p><ol class="npwtwok"><li>Quantity One (Bio-Rad Laboratories, Inc., <a href="http://www.bio-rad.com/" target="_blank">www.bio-rad.com/</a>)</li></ol><p class="pptt" id="biaoti27206">### Procedure</p><ol class="npwone"><li>Human cell culture <br/><ol class="npwtwo"><li>Obtaining Q-deprived (0Q) human cells<br/>Culture your cells in dialyzed serum medium for at least five doublings. The Q-modification level is expected to decrease by a factor of 2 after each doubling.</li><li>Generating fully Q-modified (100Q) human cells<br/>Add queuine hydrochloride solution to the cell culture media at a final concentration of 0.1-1 ÂµM and incubate for at least 12 h.<br/><em>Note: Conditions provided here were verified for HEK293T cells, these need to be optimized separately for each cell line used.</em> </li><li>RNA extraction (described below)</li></ol><br/></li><li><sup>32</sup>P-labeling and purification of Northern probes<br/>Before proceeding to the Northern blot analysis, prepare 5â <sup>32</sup>P-labeled oligonucleotide probes that are complementary to tRNA species of interest. If needed, prepare probes for a tRNA not modified with Q (<em>e.g.</em>, tRNA<sup>Ala</sup>) as a negative control.<br/><em>Warning: [Î³-</em><sup><em>32</em></sup><em>P] ATP is highly radioactive. Follow all safety procedures required in handling radioactive material (</em><em>e.g.</em><em>, ordering, storage, personal protection, shielding, preventing and detecting contamination etc.). Contact the appropriate lab/department safety inspector for more information, also about available trainings.</em><br/>Oligonucleotide probe sequences for human tRNAs are: <br/>(tRNA<sup>His</sup>)5âTGCCGTGACTCGGATTCGAACCGAGGTTGCTGCGGCCACAACGCAGAGTACTAACCACTATACGATCACGGC; <br/>(tRNA<sup>Asn</sup>)5âCGTCCCTGGGTGGGCTCGAACCACCAACCTTTCGGTTAACAGCCGAACGCGCTAACCGATTGCGCCACAGAGAC; <br/>Oligonucleotides can be ordered from DNA synthesizing company (<em>e.g.</em>, IDT, gel purified is good enough).<br/><ol class="npwtwo"><li>Labeling the oligos with [Î³-<sup>32</sup>P] ATP by T4 PNK reaction<br/><ol class="npwthree"><li>Use 20 pmol of each oligo for labeling.</li><li>Set up the reaction with 1x T4 PNK buffer from the supplier, 6 U/Âµl T4 PNK, 28 ÂµCi/Âµl [Î³-<sup>32</sup>P] ATP and add water to the final volume of 10 Âµl.</li><li>Incubate for 30 min at 37 Â°C in a thermoblock.</li><li>Inactivate T4 PNK by adding 2x gel loading buffer.</li></ol></li><li>Gel purification of the probes<br/><ol class="npwthree"><li>Load the samples on 10% denaturing polyacrylamide gel.</li><li>Run the gel at 18 W until the first dye BPB is at the bottom of the gel (typically about 1 h). </li><li>Expose the gel to a phosphorimager screen for 1 min (this time can vary depending on the radioactivity of the probes). </li><li>Based on the signal, excise the bands corresponding to the probe. </li><li>Incubate crushed gel slices with gel extraction buffer and rotate the mixture at 4 Â°C overnight on a tube rotator. </li></ol></li><li>Precipitation of the probes<br/><ol class="npwthree"><li>Centrifuge the samples at maximum speed for 30 min and collect the solution from the gel slices above. </li><li>Precipitate the DNA using 3 volumes of 100% ethanol at -20 Â°C for 2 h. </li><li>Centrifuge the samples at maximum speed for 30 min and remove the supernatant.</li><li>Wash the pellet with 70% ethanol. </li><li>Centrifuge the samples at maximum speed for 5 min and remove the supernatant.</li><li>Resuspend the pellet in 400 Âµl of nuclease-free water. Use 1/4 of the prepared probe for each blot (described below).<br/><em>Note: Ideally, fluorescent probes could be used in place of <sup>32</sup>P-labeled probes. However, we have not optimized the use of fluorescent probes ourselves. The sensitivity of fluorescent probes at this time can be quite a bit lower than <sup>32</sup>P-probes, so more RNA samples might be needed.</em><em></em><br/></li></ol></li></ol><br/></li><li>APB gel separation of total RNA<br/><ol class="npwtwo"><li>Total RNA extraction<br/><ol class="npwthree"><li>Extract total RNA from mammalian cells (<em>e.g.</em>, Trizol method, see Reference 6)<br/><em>Note: If using RNA extraction kit, choose one that does not lose small RNAs.</em> </li><li>Measure the RNA concentration of your samples.</li><li>Aliquot 5 Âµg or more of RNA for each analyzed sample. If not available, as low as 1 Âµg total RNA may be used. Keep the total volume of the RNA sample low, so the whole sample can fit in one lane after adding 2x gel loading buffer (final volume: about 20 Âµl, Step B2c).</li></ol></li><li>Deacylation of RNA to remove amino acids attached to the 3â end of tRNAs<br/><ol class="npwthree"><li>Mix the RNA with 100 mM final concentration of Tris-HCl (pH 9.0).</li><li>Incubate for 30 min at 37 Â°C in a thermoblock. </li><li>Combine deacylated RNA samples with an equal volume of denaturing 2x gel loading buffer. (Store the samples at -20 Â°C if needed for future use) </li></ol></li><li>APB gel<br/><ol class="npwthree"><li>Prepare 10% urea denaturing polyacrylamide gel containing 5% 3-Acrylamidophenylboronic acid (adjust the volumes of the gel solution to the glass plates and gel spacers available in the lab). </li><li>Pre-run the gel in 1x TAE buffer for 30 min (<em>e.g.</em>, at constant 18 W using the electrophoresis apparatus listed in Equipment). </li><li>Wash the wells of the gel using a syringe and 1x TAE buffer. </li><li>Load your samples onto the gel. </li><li>Run gel electrophoresis in 1x TAE buffer at 18 W until the first dye (BPB) reaches the bottom of the gel. </li></ol></li></ol><br/></li><li>Northern blots<br/></li></ol><ol class="npwtwok"><li>Gel transfer<br/><ol class="npwthree"><li>After electrophoresis, take apart the glass plates.</li><li>Put a Hybond-XL membrane on the gel (mark the orientation of the gel).</li><li>Take the gel off the glass plate and create a âsandwichâ as follows (from the bottom): Whatman filter paper-membrane-gel-plastic foil. </li><li>Transfer RNA onto the membrane using vacuum gel dryer for 2 h at 80 Â°C.</li></ol></li><li>Pre-hybridization<br/><ol class="npwthree"><li>Remove the membrane from the gel using distilled water.</li><li>Wash the membrane twice for 30 min each in hybridization buffer at room temperature. </li></ol></li><li>Hybridization<br/><ol class="npwthree"><li>Put the membrane into hybridization tubes of appropriate size (adjusted to the size of your membrane). </li><li>Add 15 ml of hybridization buffer per one tube. </li><li>Add 100 Âµl of the 5â <sup>32</sup>P-labeled oligonucleotide probe prepared beforehand (described in Procedure A).</li><li>Incubate and rotate the tubes in a hybridization oven for 16 h at 60 Â°C. </li></ol></li><li>Washing the membrane <br/><ol class="npwthree"><li>Remove the hybridization buffer.</li><li>Wash the membranes twice for 30 min each at 60 Â°C in a washing solution. </li><li>Take the membranes out of the tubes and dry them until you donât see any bubbles from washing solution. </li></ol></li><li>Membrane exposure<br/>Put the membranes into a cassette, cover with plastic foil and expose to phosphor screen from 6 to 16 h.<br/></li><li>Reading the signal<br/>Use phosphoimager to scan and quantify the intensity of detected bands with the Imager software. An example of the result is shown in Figure 2.<br/><br/><img alt="" class="layerphoto" height="79" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190313/20190313040859_2221.jpg" width="180"/><br/><strong>Figure 2. Q-modification kinetics.</strong> Time course of Q modification for tRNA<sup>Asn</sup> starting with the addition of 1 Î¼M queuine to 0Q cell medium (5 Âµg total RNA loaded in each lane).<br/></li></ol><p class="pptt" id="biaoti27207">### Data analysis</p><p class="ppzz">Scanned gel image was analyzed with the use of Quantity One imager software. <br/>If procedure was performed correctly, the final image should show separate bands corresponding to the Q-modified and/or G-containing fractions of the intended tRNA species (Figure 2). If larger separation is desired, run the gel for longer time while preventing running the tRNAs out using the dye migration as a guide, or use a longer gel. While performing the transferring step, make sure to cut out the section of the gel containing the tRNA molecules of interest. <br/>  When quantifying results, assume that all signals in one lane (Q-tRNA + G-tRNA) represents total tRNA of this species loaded in this lane. Calculate the fraction of Q-modified tRNAs for each sample as [Q-band]/([Q-band] + [G-band]).<br/>  For negative controls, repeat the same procedure using a probe for a tRNA not modified with Q (<em>e.g.</em>, tRNA<sup>Ala</sup>), which should only show one band on the APB gel.<br/>  Another control is to perform peroxide oxidation of the RNA sample first which destroys the <em>cis</em>-diol of the Q-base and the 3â end of the RNA, so that the probed tRNA species runs faster than the G-band (Igloi and Kossel, 1985; Zaborske <em>et al.</em>, 2014).</p><p class="pptt" id="biaoti27208">### Recipes</p><p class="ppzz">All indicated concentrations are final concentrations of each chemical.</p><ol class="npwtwok"><li>2x Gel loading buffer<br/>9 M urea<br/>100 mM EDTA<br/>0.2% (w/v) bromophenol blue<br/>0.2% (w/v) xylene cyanol</li><li>10% denaturing polyacrylamide gel <br/>10% acrylamide/bis solution, 29:1<br/>9 M urea<br/>1x TBE</li><li>10% APB polyacrylamide gel <br/>10% acrylamide/bis solution, 29:1<br/>9 M urea<br/>1x TAE<br/>5% 3-Acrylamidophenylboronic acid</li><li>Dialyzed serum medium (HEK293T and HeLa)<br/>DMEM<br/>10% dialyzed FBS</li><li>Gel extraction buffer <br/>50 mM KOAc, pH 7.0<br/>200 mM KCl</li><li>Hybridization buffer <br/>20 mM sodium phosphate, pH 7.0<br/>300 mM NaCl<br/>1% sodium dodecyl sulphate (SDS)</li><li>1 M Sodium phosphate buffer (pH 7.0)<br/>390 mM NaH<sub>2</sub>PO<sub>4</sub><br/>610 mM Na<sub>2</sub>HPO<sub>4</sub> </li><li>1 M Tris-HCl buffer (pH 9.0)<br/>1 M Tris<br/>Adjust pH with HCl</li><li>Washing solution<br/>20 mM sodium phosphate, pH 7.0<br/>300 mM NaCl<br/>0.1% SDS<br/>2 mM EDTA<br/></li></ol><p class="pptt" id="biaoti27209">### Acknowledgments</p><p class="ppzz">This method was based on the protocol developed previously (Zaborske <em>et al.</em>, 2014; Wang <em>et al.</em>, 2018). We thank the support from DoD/CDMRP (BC160450 to T.P.).</p><p class="pptt" id="biaoti27210">### Competing interests</p><p class="ppzz">There are no conflicts of interest or competing interest.</p><p class="pptt" id="biaoti27211">### References</p><ol class="npwtwok"><li>El Yacoubi, B., Bailly, M. and de Crecy-Lagard, V. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22905870" target="_blank">Biosynthesis and function of posttranscriptional modifications of transfer RNAs.</a> <em>Annu Rev Genet</em> 46: 69-95. </li><li>Fergus, C., Barnes, D., Alqasem, M. A. and Kelly, V. P. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25884661" target="_blank">The queuine micronutrient: charting a course from microbe to man.</a> <em>Nutrients</em> 7(4): 2897-2929. </li><li>Igloi, G. L. and Kossel, H. (1985). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2414733" target="_blank">Affinity electrophoresis for monitoring terminal phosphorylation and the presence of queuosine in RNA. Application of polyacrylamide containing a covalently bound boronic acid.</a> <em>Nucleic Acids Res</em> 13(19): 6881-6898. </li><li>Rakovich, T., Boland, C., Bernstein, I., Chikwana, V. M., Iwata-Reuyl, D. and Kelly, V. P. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21487017" target="_blank">Queuosine deficiency in eukaryotes compromises tyrosine production through increased tetrahydrobiopterin oxidation.</a> <em>J Biol Chem</em> 286(22): 19354-19363. </li><li>Singhal, R. P., Kopper, R. A., Nishimura, S. and Shindo-Okada, N. (1981). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6165359" target="_blank">Modification of guanine to queuine in transfer RNAs during development and aging.</a> <em>Biochem Biophys Res Commun</em> 99(1): 120-126. </li><li>TRIzolTM Reagent RNA extraction protocol: <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/trizol_reagent.pdf" target="_blank">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/trizol_reagent.pdf.</a> </li><li>Tuorto, F., Legrand, C., Cirzi, C., Federico, G., Liebers, R., Muller, M., Ehrenhofer-Murray, A. E., Dittmar, G., Grone, H. J. and Lyko, F. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30093495" target="_blank">Queuosine-modified tRNAs confer nutritional control of protein translation.</a> <em>EMBO J</em> 37(18) pii: e99777.</li><li>Wang, X., Matuszek, Z., Huang, Y., Parisien, M., Dai, Q., Clark, W., Schwartz, M. H. and Pan, T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29970597" target="_blank">Queuosine modification protects cognate tRNAs against ribonuclease cleavage. </a><em>RNA</em> 24(10): 1305-1313. </li><li>Zaborske, J. M., DuMont, V. L., Wallace, E. W., Pan, T., Aquadro, C. F. and Drummond, D. A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25489848" target="_blank">A nutrient-driven tRNA modification alters translational fidelity and genome-wide protein coding across an animal genus.</a> <em>PLoS Biol</em> 12(12): e1002015. <br/><br/></li></ol>