<body>
            <h1>Detection of mRNA by Whole Mount <em>in situ</em> Hybridization and DNA  Extraction for Genotyping of Zebrafish Embryos</h1>
            <p><strong>Authors:</strong> Rachna  Narayanan, Andrew C. Oates</p>
            <h2>Abstract</h2>
            <p><em>In situ</em> hybridization is used to visualize the spatial distribution of gene transcripts in tissues and in embryos, providing important information about disease and development. Current methods involve the use of complementary riboprobes incorporating non-radioactive labels that can be detected by immunohistochemistry and coupled to chromogenic or fluorescent visualization. Although recent fluorescent methods have allowed new capabilities such as single-molecule counting, qualitative chromogenic detection remains important for many applications because of its relative simplicity, low cost and high throughput, and ease of imaging using transmitted light microscopy. A remaining challenge is combining high contrast signals with reliable genotyping after hybridization. Dextran sulfate is commonly added to the hybridization buffer to shorten development times and improve contrast, but this reagent inhibits PCR-based genotyping. This paper describes a modified protocol for <em>in situ</em> hybridization in fixed whole mount zebrafish embryos using digoxigenin (DIG) labeled riboprobes that are detected with alkaline phosphatase conjugated anti-DIG antibodies and nitroblue tetrazolium (NBT)/5-bromo-4-chloro-3-indolyl-phosphate (BCIP) chromogenic substrates. To yield embryos compatible with downstream genotyping after hybridization without sacrificing contrast of the signal, this protocol omits dextran sulfate and utilizes a lower hybridization temperature.</p>
            <p><strong>Keywords:</strong> <em>In situ</em>, Hybridization, WISH, Zebrafish, Staining, Qualitative assay</p>
            <h2>Background</h2><p><em>In situ</em> hybridization is a technique that enables the detection of RNA in single cells, tissues or in whole embryos (Schulte-Merker <em>et al.</em>, 1992; Hauptmann and Gerster, 1994; Schulte-Merker <em>et al.</em>, 1994). The technique has been widely employed in developmental biology to characterize the spatial and temporal distributions of target mRNAs in fixed embryos (Thisse <em>et al.</em>, 2001; Thisse and Thisse, 2004 and 2005). By allowing the comparison of mRNA distributions in wild-type, mutant and experimentally manipulated embryos, <em>in situ</em> hybridization continues to play a vital role in the investigation of gene function in development. <br/><br/>Hybridization is based on the principle of complementary base-pair binding of a specific probe oligonucleotide to a target mRNA. RNA oligonucleotides (riboprobes) are designed to pair with target mRNA molecules and are synthesized with nucleotides containing haptens, such as digoxigenin (DIG). In <em>in situ</em> hybridization, the riboprobes are incubated with previously fixed and permeabilized bulk tissue or intact embryos at an elevated temperature to favor the formation of hybrids with high sequence complementarity. The stable target-probe hybrids are then detected with a hapten-specific antibody. In chromogenic variations of the technique, the antibody is conjugated to alkaline phosphatase. This enzymatic activity results in the local generation of a colored precipitate upon the addition of a chromogenic substrate. NBT and BCIP are commonly used substrates yielding a purple-blue precipitate. Thus, the localization of target mRNA can be visualized with cellular resolution in the embryo or tissue by a colored signal in transmitted light microscopy.<br/><br/>The technique has been modified to compare the localization of two mRNAs by including different haptens on different probes (Hauptmann and Gerster, 1994) or combined with immunohistochemistry to compare the localization of mRNA and protein. Recent advances allow the fluorescent detection of multiple genes (Choi <em>et al.</em>, 2010; Gross-Thebing <em>et al.</em>, 2014), the sub-cellular localization of mRNAs (Jambor <em>et al.</em>, 2015) and the determination of the number of transcripts in a sample (Wang <em>et al.</em>, 2012; Little <em>et al.</em>, 2013; Stapel <em>et al.</em>, 2016; Trivedi <em>et al.</em>, 2018). Nevertheless, qualitative chromogenic detection remains important in the field because of its relative simplicity, low cost, high throughput and ease of imaging.<br/><br/>The signal to noise ratio of the colored signal of an <em>in situ</em> hybridization experiment is a function of the specificity of riboprobe, <em>i.e.</em>, the ability of the riboprobe to distinguish between different targets, and the stringency of the hybridization. High stringency requires the base pairing of all bases in the oligonucleotide while lower stringency allows some bases to be unpaired. The design of the riboprobe sets the specificity of hybridization. The reaction environment affects the stringencyâannealing of oligonucleotides and the thermal stability of the resulting hybrid is influenced by temperature, pH, the concentration of monovalent cations and the presence of organic solvents in the environment.<br/><br/>Riboprobes have to be designed to be maximally complementary to the target mRNA for high specificity. Additionally, maximally complementary RNA-RNA hybrids have the highest thermal stability as base pair mismatches lower the thermal stability. The length of the riboprobe will affect the hybridization rate and also influence the thermal stability of the resulting hybridâlonger probes have a higher hybridization rate and generate thermally stable hybrids (Wetmur, 1976). Riboprobes of lengths between 300-3,200 base pairs specific to the target can be used with the protocol described here. <br/><br/>The temperature used for hybridization is determined from the melting temperature of the designed oligonucleotide. Hybridization is performed 15-25 Â°C lower than the calculated melting temperature (Wetmur and Davidson, 1968)âthe higher this temperature, the greater the stringency of hybridization. However, prolonged incubation at typical RNA hybridization temperatures will degrade sample morphology. Therefore, organic solvents are added to reduce the thermal stability of the RNA duplexes, with formamide being the most common reagent (McConaughy <em>et al.</em>, 1969). Hybridization buffers used in <em>in situ</em> hybridization also contain monovalent sodium cations, from sodium chloride and sodium citrate, which interact electrostatically with the RNA-RNA hybrid to stabilize it. Low salt washes are subsequently performed to reduce background by preferentially reducing the stability of non-specific hybrids.<br/><br/>Typical <em>in situ</em> protocols for the qualitative detection of mRNA in whole mount in zebrafish recommend a hybridization temperature of 70 Â°C (Thisse and Thisse, 2008) for high stringency. Such high stringency conditions are necessary if a riboprobe is required to discriminate between two similar target sequences. We found that for riboprobes with high specificity, by performing hybridization at lower temperatures (55-60 Â°C), we could achieve a more rapidly developing, higher contrast stain. We could further reduce the development time and enhance the contrast of the stain by increasing the effective concentration of riboprobe by adding dextran sulfate to the hybridization buffer (Lauter <em>et al.</em>, 2011). Additionally, for probes that required prolonged development, the addition of polyvinyl alcohol to the NBT-BCIP staining solution reduces background (Kiyama and Emson, 1991). Making these modifications to the <em>in situ</em> hybridization protocol consistently accelerates the development and enhances the contrast of the stain.<br/><br/>A key part of an <em>in situ</em> experiment is the qualitative analysis of variations in mRNA distributions in mutant or otherwise experimentally manipulated embryos. However, differences in mRNA distributions may be evident before the genotype of the embryo can be ascertained by altered morphology. Further, such differences can be subtle, or variable, and may not fall into convenient Mendelian ratios complicating a straightforward assignment of wild-type and perturbed patterns. For these reasons, it is valuable to have a protocol compatible with reliable genotyping of embryos, or fragments of embryos, after hybridization and photographic documentation. We have observed that the presence of dextran sulfate in the hybridization buffer inhibits genotyping by PCR; therefore it is omitted if the embryos need to be genotyped after <em>in situ</em> hybridization.<br/> <br/>The protocol described here describes three linked procedures: (1) the generation of DIG-labeled riboprobes suitable for hybridization; (2) <em>in situ</em> hybridization using a single riboprobe to qualitatively detect an mRNA target of interest in developing zebrafish embryos and imaging of the embryos; and (3) DNA extraction that gives a PCR-ready extract, enabling the presence or absence of indel mutations to be identified in the genomic DNA of individual, previously imaged embryos. We have successfully tested the protocol in labs in two different countries. Combined, these procedures form a protocol sufficiently detailed to enable a relatively inexperienced experimentalist to achieve high quality and reliable analysis of mRNA distribution patterns with a range of experimental perturbations.<br/></p>
        </body>

<p class="pptt" id="biaoti27226">## Materials and Reagents</p><ol class="npwone"><li>Riboprobe synthesis<br/><ol class="npwtwo"><li>1.5 ml colorless microfuge tubes (Eppendorf, SafeLock, catalog number: 0030120086)</li><li>PCR tubes (Sarstedt, Multiply<sup>Â®</sup>-Î¼strip Pro 8-strip, catalog number: 72.991.002)</li><li>Restriction digest enzymes (Thermo Scientific, FastDigest) (store at -20 Â°C)</li><li>2x Phusion Hot Start II High-Fidelity PCR Master Mix (Thermo Scientific, catalog number: F-565) (store at -20 Â°C)</li><li>rNTP-DIG (Roche, catalog number: 11277073910) (store at -20 Â°C)</li><li>RNase Inhibitor (RNaseOUT, Invitrogen, catalog number: 100000840) (store at -20 Â°C)</li><li>RNA polymerases (1,000 U-20 U/Î¼l) (store at -20 Â°C)<br/><ol class="npwthree"><li>Sp6 (Roche, catalog number: 10810274001)</li><li>T3 (Roche, catalog number: 11031163001)</li><li>T7 (Roche, catalog number: 10881767001)</li></ol></li><li>Clean up kits (store at room temperature)<br/><ol class="npwthree"><li>Qiaquick PCR purification kit (QIAGEN, catalog number: 28104)</li><li>Promega SV Gel Wizard PCR purification kit (Promega, catalog number: A9281)</li><li>RNeasy Min Elute (QIAGEN, catalog number: 74204)</li></ol></li><li>Agarose (Sigma-Aldrich, catalog number: A9539) (store at room temperature)<br/></li><li>1x Tris-Acetate, EDTA (TAE) (store at room temperature, see Recipes)<br/><ol class="npwthree"><li>Tris Base (Sigma-Aldrich, BioXtra, catalog number: T6791) (store at room temperature)</li><li>Glacial acetic acid (Suprapur<sup>Â®</sup>, EMD Millipore, catalog number: 1.00066) (store at room temperature)</li><li>EDTA, 0.5 M (UltraPure, Invitrogen, catalog number: 15575-038) (store at room temperature)</li></ol></li><li>Hybridization buffer (Hyb) (store at -20 Â°C, see Recipes)<br/><ol class="npwthree"><li>Formamide, deionized (Roche, catalog number: 1814320) (store at 4 Â°C)</li><li>50 mg/ml Heparin stock (store at 4 Â°C, see Recipes)<br/>Heparin (Sigma-Aldrich, catalog number: H3400) (store at room temperature)</li><li>Torula-RNA (Sigma-Aldrich, catalog number: R6875) (store at -20 Â°C)</li><li>20x SSC (store at room temperate, see Recipes)<br/>Sodium chloride (Sigma-Aldrich, BioXtra, catalog number: S7653)<br/>Trisodium citrate dihydrate (Sigma-Aldrich, catalog number: C8532) (store at room temperature)</li><li>(Optional) Dextran sulfate sodium salt (Sigma-Aldrich, catalog number: D6001) (store at 4 Â°C)</li><li>10% Tween-20 (see Recipes)<br/>Tween-20 (Sigma-Aldrich, catalog number: P9416) (store at room temperature)</li></ol></li></ol><br/></li><li>Embryo culture<br/><ol class="npwtwo"><li>94 mm polystyrene Petri dish (Grenier Bio-One, catalog number: 633161)</li><li>100x PTU stock (store at room temperature, see Recipes)<br/><em>N</em>-Phenylthiourea (PTU, Sigma-Aldrich, catalog number: P7629) (store at room temperature)</li></ol><br/></li><li><em>In situ</em> hybridization<br/><ol class="npwtwo"><li>24-well plates (Grenier Bio-One, Cell Star, catalog number: 662160)</li><li>2 ml colorless microfuge tubes (Eppendorf, SafeLock, catalog number: 0030120094)</li><li>3 ml plastic non-sterile, macrograduated Pasteur pipettes (HuberLab, catalog number: 15.4051.11)</li><li>94 mm polystyrene Petri dish (Grenier Bio-One, catalog number: 633161)</li><li>Thin walled plastic container (Microwaveable plastic lunch boxâany brand)</li><li>Methanol (Fisher, catalog number: M/4000/15) (store at room temperature)</li><li>Phosphate buffered saline, 0.1% Tween (PBST) (store at room temperature, see Recipes)<br/><ol class="npwthree"><li>Sodium chloride (Sigma-Aldrich, BioXtra, catalog number: S7653) (store at room temperature)</li><li>Potassium chloride (Sigma-Aldrich, BioXtra, catalog number: P9333) (store at room temperature)</li><li>Sodium phosphate dibasic (Sigma-Aldrich, BioXtra, catalog number: S7907) (store at room temperature)</li><li>Potassium phosphate monobasic (Sigma-Aldrich, catalog number: P9791) (store at room temperature)</li><li>10% Tween-20 (see Recipes)</li></ol></li><li>4% Paraformaldehyde (PFA) in PBST (store at 4 Â°C) (see Recipes)<br/><ol class="npwthree"><li>PFA powder (Sigma-Aldrich, catalog number: P6148) or 16% PFA (Alfa Aesar, catalog number: 43368.9M) (store at 4 Â°C)</li><li>PBST (see Recipes)</li></ol></li><li>Proteinase K stock (store at -20 Â°C, see Recipes)<br/>Proteinase K (Merck, catalog number: 1.24568) (store at 4 Â°C)</li><li>Hybridization buffer (Hyb) (store at -20 Â°C, see Recipes)</li><li>Post-hybridization washes (see Recipes)<br/><ol class="npwthree"><li>Wash 1</li><li>Wash 2</li><li>Wash 3</li></ol></li></ol><br/></li><li>Detection<br/><ol class="npwtwo"><li>AP-conjugated anti-DIG antibody (Anti-Digoxigenin-AP, Fab fragments from sheep) (Roche, catalog number: 11093274910) (store at 4 Â°C)</li><li>Maleic acid buffer, 0.1% Tween (MABT) (store at room temperature, see Recipes)<br/><ol class="npwthree"><li>Maleic acid (Sigma-Aldrich, catalog number: M0375) (store at room temperature)</li><li>Sodium chloride (Sigma-Aldrich, BioXtra, catalog number: S7653) (store at room temperature)</li><li>Sodium hydroxide (Fisher Chemical, catalog number: S/4920/60) (store at room temperature)</li><li>10% Tween-20 (see Recipes)</li></ol></li><li>10% Blocking reagent in maleic acid buffer <br/><ol class="npwthree"><li>Blocking Reagent (Roche, catalog number: 11096176001) (store at room temperature)</li><li>Maleic acid buffer (see Recipes)</li></ol></li><li>2% Blocking reagent in MABT (2% Roche block) (see Recipes)</li><li>Staining buffer (see Recipes)<br/><ol class="npwthree"><li>Tris-HCl (Sigma-Aldrich, BioXtra, catalog number: T6666) (store at room temperature)</li><li>Magnesium chloride hexahydrate (Sigma-Aldrich, BioXtra, catalog number: M2670) (store at room temperature)</li><li>Sodium chloride (Sigma-Aldrich, BioXtra, catalog number: S7653) (store at room temperature)</li></ol></li><li>Staining solution (see Recipes)<br/><ol class="npwthree"><li>NBT (Roche catalog number: 11383213001) (aliquot and store at -20 Â°C)</li><li>BCIP (Roche, catalog number: 11282221001) (aliquot and store at -20 Â°C)</li><li>Staining buffer (see Recipes)</li><li>(Optional) Polyvinyl alcohol (PVA, Sigma-Aldrich, catalog number: P1763) (store at room temperature)</li></ol></li></ol><br/></li><li>Mounting<br/><ol class="npwtwo"><li>60 mm Petri dish</li><li>A human eyelash glued to a tooth pick/syringe needle or a fine pipette tip (GELoader Tips 0.5-20 Î¼l, 62 mm, Eppendorf, catalog number: 0030001222)</li><li>(Optional) Mold for embryos for photographic documentation in whole mount [the mold negative is made in our laboratory (Herrgen <em>et al.</em>, 2009)]<br/></li></ol><strong>Materials for flat mounting:</strong> </li><ol class="npwtwo" start="4"><li>(Optional) Dissection hook (Picker <em>et al.</em>, 2009) made with Tungsten wire (Hamilton, catalog number: 18306)</li><li>10 Î¼l pipette tip </li><li>Glass slides (Thermo Scientific, Superfrost, catalog number: 12372098)</li><li>Glass coverslip (VWR, Menzel GlÃ¤ser, #1.5 rectangular 22 x 40 mm, catalog number: 631-1370)</li><li>Silicone grease or petroleum jelly (such as Vaseline)</li><li>Transparent nail varnish (any brand)</li><li>87% glycerol in distilled water (store at room temperature, see Recipes)<br/>Glycerol (Fisher, catalog number: G/0650/08) (store at room temperature)</li></ol><br/><li>Genomic DNA extraction (store at room temperature)<br/><ol class="npwtwo"><li>50x base solution (see Recipes)<br/><ol class="npwthree"><li>Potassium hydroxide (Fisher Chemical, catalog number: P/5640/60)</li><li>EDTA, 0.5 M (UltraPure, Invitrogen, catalog number: 15575-038)</li></ol></li><li>1x base solution (see Recipes)</li><li>50x neutralization solution (see Recipes)<br/>Tris-HCl (Sigma-Aldrich, catalog number: T6666)</li><li>1x neutralization solution (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti27227">### Equipment</p><ol class="npwone"><li>Stock solution preparation<br/><ol class="npwtwo"><li>Magnetic stirrer (Heidolph Instruments, model: MR Hei-Standard, catalog number: 505-20000-00)</li><li>pH meter (Mettler Toledo, model: Seven compact and InLab Expert Pro-ISMï¼</li></ol><br/></li><li>Riboprobe synthesis<br/><ol class="npwtwo"><li>PCR machine (Eppendorf, model: Mastercycler pro S) </li><li>Thermoblock (Eppendorf, model: ThermoMixer C and SmartBlock 2.0 ml)</li><li>Gel electrophoresis (Bio-Rad, model: PowerPac Basic Power Supply; Wide Mini-Sub Cell GT Cell)</li><li>NanoDrop spectrophotometer (Thermo Fisher, model: NanoDrop<sup>TM</sup> 2000, catalog number: ND 2000)</li></ol><br/></li><li><em>In situ hybridization</em><br/><ol class="npwtwo"><li>Forceps (Dumont, model: 5-Inox-E)</li><li>Stereomicroscope (Olympus, model: SZ61)</li><li>Shaker (Heidolph, model: Duomax1030, catalog number: 543-32205-00)</li><li>(Optional) Vacuum set-up (Integra, Vacusafe aspiration system)</li><li>Water bath (GFL, model: 1003)</li><li>(Optional) Well plate stand (Diversified Biotech, catalog number: WPST-1000)</li></ol><br/></li><li>Documentation<br/><ol class="npwtwo"><li>RGB Camera (Olympus, model: DP22)</li><li>Ring light (Photonic Optics, model: LED 40 ring light)</li></ol><br/></li><li>Genomic DNA extraction<br/><ol class="npwtwo"><li>Micro knife (Fine Science Tools, catalog number: 10315-12)</li></ol></li></ol><p class="pptt" id="biaoti27228">### Software</p><ol class="npwtwok"><li>Fiji (Schindelin <em>et al.</em>, 2012, <a href="https://fiji.sc/" target="_blank">https://fiji.sc/</a>)</li><li>Snapgene (GSL Biotech LLC)<br/></li></ol><p class="pptt" id="biaoti27229">### Procedure</p><ol class="npwone"><li>Riboprobe synthesis<br/><em>Note: DNA templates for riboprobes can be either cDNA cloned into a plasmid backbone downstream of an RNA polymerase promoter or PCR product amplified from cDNA or genomic DNA. See Note 1.</em><br/><ol class="npwtwo"><li>Template preparation<br/><strong><em>Preparing a template from a plasmid</em></strong><br/><ol class="npwthree"><li>Linearize the plasmid by digestion with the appropriate restriction enzyme at 37 Â°C. Choose an enzyme such that the template is linearized immediately downstream of the cDNA insert. <br/>Set the reaction up as:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:200px;"><tbody><tr><td>Plasmid template<br/></td><td>5 Î¼g<br/></td></tr><tr><td>Restriction enzyme<br/></td><td>5 Î¼l<br/></td></tr><tr><td>10x restriction buffer<br/></td><td>10 Î¼l<br/></td></tr></tbody></table>Make up to 100 Î¼l with distilled water/nuclease-free water.</li><li>Purify the linearized plasmid with a commercial clean up kit. We recommend the Qiaquick PCR purification kit. Purify the plasmid as per the manufacturerâs instructions and elute the plasmid in 30 Î¼l of elution buffer.</li><li>Dilute 1 Î¼l of the eluent in 4 Î¼l of distilled water. Use 1 Î¼l to assess concentration on a spectrophotometer and 4 Î¼l to check the linearization with agarose gel electrophoresis. A single band corresponding to the linearized plasmid should be visible on the gel. Proceed with riboprobe synthesis if a single band corresponding to the fully linearized plasmid is visualized. <br/><br/></li></ol><strong><em>Preparing a template by PCR</em></strong><br/><ol class="npwthree"><li>Design primers to amplify the desired target from cDNA or genomic DNA. To make a riboprobe that is complementary to the target mRNA, add the RNA polymerase (T7/T3/Sp6) promoter sequence to the reverse primer as: 5â-RNA polymerase promoter sequence + template specific sequence-3â.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190318/20190318182440_8216.jpg" width="500"/><br/><br/></li><li>Carry out the PCR with the optimal cycling conditions. We recommend using a high fidelity DNA polymerase like Phusion.</li><li>Clean up the PCR product. We recommend the Promega SV Gel Wizard PCR purification kit. Elute in 30 Î¼l of the kit elution buffer. Dilute 1 Î¼l of the eluent in 4 Î¼l of distilled water. Use 1 Î¼l of this dilution to assess concentration on a NanoDrop and 4 Î¼l to assess product quality by gel electrophoresis. Proceed with riboprobe synthesis if a single band of the appropriate length is obtained. It is not necessary to sequence the PCR product to confirm sequence accuracy of the template (see Note 6).</li></ol></li><li><em>In vitro</em> transcription<br/><ol class="npwthree"><li>Thaw the reaction components (except the enzymes) and place on ice.</li><li>Set up each 20 Î¼l transcription reaction as:<br/><table border="0" bordercolor="#000000" cellpadding="9" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td>DNA Template<br/></td><td>1-1.5 Î¼g of linearized plasmid<br/>or 100-500 ng of PCR product<br/></td></tr><tr><td>10x Transcription buffer<br/></td><td>2 Î¼l<br/></td></tr><tr><td>rNTP-DIG<br/></td><td>2 Î¼l<br/></td></tr><tr><td>RNase Inhibitor<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>RNA polymerase<br/></td><td>2 Î¼l<br/></td></tr></tbody></table>Adjust the volume to 20 Î¼l with RNase-free water, if needed.</li><li>Transcribe for 2-4 h at 37 Â°C. The yield of the reaction is equivalent when the transcription time is between 2 h and 4 h.</li><li>Purify the riboprobe from the reaction with a commercial kit like QIAGEN RNeasy Min Elute. Follow the manufacturerâs instructions and elute the riboprobe in 30 Î¼l of RNase-free water. Dilute 1 Î¼l of the probe in 4 Î¼l of water. Use 1 Î¼l to assess the concentration on a spectrophotometer and 4 Î¼l to assess the quality of the transcription by agarose gel electrophoresis. The typical A<sub>280</sub>/A<sub>260</sub> ratio of the synthesized probe after clean up is between 1.85 and 2.00. To resolve the RNA riboprobe on a 1% agarose gel in 1x TAE, run it at 140 V for 10-15 min. The riboprobe should resolve as a distinct band. Occasionally, a longer band is also obtainedâthis is typically an electrophoresis artifact arising due to RNA secondary structure, if a high-quality PCR product or completely linearized plasmid template has been used.</li><li>Store the remainder of the synthesized riboprobe at -20 Â°C. Dilute the riboprobe in 5 ml of Hyb to perform <em>in situ</em> hybridization. See Note 2.</li><li>Test the probe by performing <em>in situ</em> hybridization and assessing the pattern of hybridization obtained. </li></ol></li></ol><br/></li><li><em>In situ</em> hybridization<br/><em>Notes:</em> <br/><ol class="npwthree"><li><em>This protocol has been optimized for embryos up to 48 h post fertilization (hpf) and is carried out in 24-well plates. Use 800 Î¼l for washes and 500 Î¼l for incubations. When exchanging media, the embryos being stained should not be allowed to dry out and should always remain covered with liquid. To ensure that they do not get stuck to the bottom or on to the sides of the wells, pipette gently and check after each solution change that embryos are covered by liquid.</em> </li><li><em>All incubation steps, unless specified, are performed at room temperature on a shaker. <br/><br/></em> </li></ol><ol class="npwtwo"><li>Fixation and dehydration of embryos (see Note 3).<br/><strong><em>Fixing embryos younger than 24 hpf</em><br/></strong> <ol class="npwthree"><li>Transfer embryos into 2 ml microfuge tubes (for long-term storage) or into wells of a 24-well plate (for immediate use for <em>in situ</em>) with a 3 ml plastic Pasteur pipette. Up to 100 embryos can be stored in a single tube and up to 30 embryos can be processed per well in a 24-well plate.</li><li>Remove as much embryo medium as possible. If the embryos are in their chorions, all the liquid can be removed. Add 4% PFA in PBST (fix)â1 ml per tube, 800 Î¼l per well. Incubate on a shaker for 5 min at room temperature. This step serves to remove any excess embryo medium.</li><li>Exchange the fix with fresh fixâup to 2 ml per tube, 800 Î¼l-1 ml per well. Incubate at room temperature for 1 to 4 h or overnight at 4 Â°C, with shaking.</li><li>Remove the fix and add PBSTâup to 2 ml per tube, 800 Î¼l-1 ml per well.</li><li>Transfer embryos into PBST in a 35 mm Petri dish with a 3 ml plastic Pasteur pipette and use forceps to dechorionate the embryos under a stereo microscope (Video 1). Transfer embryos back into the well plate or into a fresh tube/vial. The pipette should be rinsed by pipetting up and down in PBST, and the PBST should be exchanged between batches of embryos to prevent cross-contamination. </li></ol><br/><strong><em>Fixing embryos older than 24 hpf</em><br/></strong> <ol class="npwthree"><li>From 24 hpf onwards, add PTU to the embryo medium to a final concentration of 0.003% to inhibit pigment development in the embryos.</li><li>Dechorionate embryos in embryo medium in a 35 mm Petri dish under a stereomicroscope (Video 2) and transfer them with a 3 ml plastic Pasteur pipette to the well plate or microfuge tube vialâ30 for a well, 100 for a tube. The embryo medium should be exchanged between batches of embryos to prevent cross-contamination.</li><li>Remove as much embryo medium as possible. Add 4% PFA in PBST (fix)â1 ml per tube, 800 Î¼l per well. Incubate on a shaker for 5 min at room temperature. This step serves to remove any excess embryo medium.</li><li>Exchange with fresh fixâup to 2 ml per tube or vial, 800 Î¼l to 1 ml per well in the well plate. Incubate with shaking at room temperature for 1 to 4 h or overnight at 4 Â°C.</li><li>Briefly rinse the embryos by removing fix and adding PBSTâup to 2 ml per tube, 800 Î¼l-1 ml per well.</li><li>Exchange the PBST and wash the embryos in PBST 3 times, for a duration of 5 min each time, on a shaker at room temperature.</li></ol><br/><strong><em>Dehydration and permeabilization of embryos</em></strong><br/><ol class="npwthree"><li>Dehydrate the embryos in a graded methanol series: add 800 Î¼l of 30% methanol in PBST, then replace with 800 Î¼l of 50% methanol in PBST and then with 800 Î¼l of 70% methanol in PBST. Incubate on a shaker between each replacement for 5 min.</li><li>Incubate the embryos in 800 Î¼l of absolute methanol for 5 min on a shaker.</li><li>Replace the methanol and incubate embryos at -20 Â°C overnight to permeabilize the embryos. Use 1.1 ml of methanol per well or 2 ml per microfuge tube.<br/>The protocol can be stopped here if necessary as embryos can be stored at -20 Â°C in methanol indefinitely (see Note 3).<br/><br/><!--video3193-1å¼å§--><div class="videodiv" id="vd3193-1" style="height:480px;"></div><div class="videophoto" id="vp3193-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3193/3193-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Dechorionation of fixed somitogenesis stage embryos</strong> </div><!--video3193-1ç»æ--><br/><br/><!--video3193-2å¼å§--><div class="videodiv" id="vd3193-2" style="height:480px;"></div><div class="videophoto" id="vp3193-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3193/3193-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. Dechorionation of live 36 hours post fertilization (hpf) embryos</strong> </div><!--video3193-2ç»æ--></li></ol><br/></li><li>Enzymatic embryo permeabilization<br/><ol class="npwthree"><li>Rehydrate embryos in a graded methanol series: add 800 Î¼l of 70% methanol in PBST, then replace with 800 Î¼l of 50% methanol in PBST and then with 800 Î¼l of 30% methanol in PBST. Incubate on a shaker for 5 min at each step. </li><li>Perform two 5-min washes with PBST.</li><li>Digest embryos in Proteinase K (PKA). Dilute the PKA stock 1:200 for embryos younger than 24 hpf and 1:100 for embryos older than 24 hpf in PBST. Incubate in 500 Î¼l without shaking at room temperature. Digestion time is embryonic stage dependent; the timings listed below can be used as a guideline. Stages indicated here are as per Kimmel <em>et al.</em> (1995).<br/><br/><img alt="" class="layerphoto" height="229" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190318/20190318181803_0336.jpg" width="500"/><br/><br/></li><li>Stop the reaction by making a rapid exchange with PBST followed by a 20-min re-fixation with 500 Î¼l of fix at room temperature on a shaker.</li><li>Remove the fix; perform one rapid wash with PBST followed by two 5-min PBST washes.</li></ol></li><li>Hybridization<br/><em>Note: The temperature used for hybridization is probe-dependent, but a good starting temperature is 60 Â°C (see Note 4). Perform hybridization steps in a water bath set to the hybridization temperature. Use a thin-walled plastic container lined with moistened tissue to incubate the well plate in the water bath. The container and hybridization solutions need to be pre-warmed to the hybridization temperature in the water bath before use.<br/></em> <ol class="npwthree"><li>Pre-hybridize in 500 Î¼l of Hyb at the hybridization temperature for a minimum of 30 min. Embryos can be pre-hybridized for up to 3 h at the hybridization temperature. </li><li>Remove the Hyb and replace it with 500 Î¼l of labeled riboprobe diluted in Hyb. Hybridize overnight at the hybridization temperature. See Note 5.</li></ol></li><li>Post-hybridization washes<br/><em>Note: These washes are also performed in a water bath at the hybridization temperature. Prepare the wash solutions fresh and warm them in the water bath before use. Timings indicated in this step are criticalâdo not wash for shorter or longer than the time recommended.</em><br/><ol class="npwthree"><li>Remove and recycle probe. Store it at -20 Â°C. See Note 2.</li><li>Wash embryos 2 times, for 30 min each time in Wash 1.</li><li>Wash embryos for 15 min in Wash 2.</li><li>Wash embryos 2 times, for 30 min each time in Wash 3.</li></ol></li><li>Detection<br/><ol class="npwthree"><li>Make a quick rinse with MABT to remove any remaining Wash 3 from the embryos.</li><li>Add 500 Î¼l of 2% blocking reagent and incubate without shaking in the dark at room temperature for at least 30 min.</li><li>Dilute AP-conjugated anti-DIG antibody 1:2,000 in 2% blocking reagent. Remove the blocking reagent and add 500 Î¼l of the diluted antibody in blocking reagent to the embryos. Incubate for 2 h, without shaking, at room temperature in the dark.</li><li>Remove antibody solution and make 2 quick washes with MABT.</li><li>Wash in 10-20 min intervals over 1 to 2 h at room temperature; at least 6 exchanges of MABT should be made. Alternatively, embryos can be washed at 4 Â°C overnight in MABT, followed by three 5-min washes the next day.</li><li>Equilibrate embryos in freshly prepared staining buffer twice for 5 min each.</li><li>Add 500 Î¼l of the staining solution to the embryos. Stain in the dark at room temperature without shaking. See Note 5.</li><li>Control the staining reaction by eye under a stereomicroscope. Place the well plate on a white background and illuminate the plate from above. Monitor the development of the embryos in the first 5 min after the staining solution has been added and then in 10-15 min intervals till the stain saturates or the desired contrast is achieved.</li><li>Stop the staining reaction by performing three 5-min washes in PBST.</li><li>Incubate embryos in methanol at room temperature till the yolk is cleared and appears white, or overnight at 4 Â°C or -20 Â°C.<br/><em>Note: See Figures 2, 3B and 4B for stains we have made with this protocol. See Note 6 for notes on troubleshooting.</em></li></ol></li><li>Mounting and photography<br/><ol class="npwthree"><li>Remove the methanol from the embryos by making two 5-min washes in PBST.</li><li>Re-fix in 4% PFA in PBST for at least 15 min at room temperature.</li><li>Wash embryos twice each for 5 min in PBST.</li><li>Add 87% glycerol to the embryos and leave to equilibrate at 4 Â°C. When the embryos sink to the bottom of the well, they are completely equilibrated. This takes ~12 h. Embryos can be subsequently stored in 87% glycerol either at 4 Â°C or -20 Â°C.</li><li>Photograph the embryos: <br/><ol class="npwfour"><li>Photographing whole mount preparations: <br/>Photograph the embryos in 87% glycerol in a 35 mm Petri dish or an imaging mold (see Note 7) with an RGB camera coupled to a stereomicroscope. Illuminate the embryos from above using a ring light. Place a white sheet of paper below the dish to provide a white background for the image. An eyelash, tungsten wire or fine pipette tip can be used to manipulate the embryos. </li><li>Photographing flat mount preparations: <br/>See also the JoVE video protocol for de-yolking and flat mounting (Cheng <em>et al.</em>, 2014).<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:0px;"><div class="float-left">1)</div><div class="media-body" style="line-height:200%;">Transfer the embryos to be flat mounted onto a glass slide with a drop of 87% glycerol. De-yolk the embryos with an eyelash or sharpened tungsten wire (see Note 8) in the drop. Move the embryo to the edge of the drop so that a minimal volume of glycerol covers it. Remove the remaining yolk cells with the eyelash. Periodically rinse the loosened yolk cells off the embryo in the glycerol drop. Move the embryo to a fresh drop of glycerol after de-yolking.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:0px;"><div class="float-left">2)</div><div class="media-body" style="line-height:200%;">Place the embryonic axis, dorsal side up, on a clean glass slide with the wire or the eyelash. Place approximately 15-20 Î¼l total volume of 87% glycerol in small drops next to the embryos. Up to 10 de-yolked embryos can be flat mounted on a single slide.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:0px;"><div class="float-left">3)</div><div class="media-body" style="line-height:200%;">Make 4 small grease pillars with a 10 Î¼l pipette tip to hold the corners of a glass coverslip in place. Carefully lower the coverslip onto the slide (Figure 1).</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:0px;"><div class="float-left">4)</div><div class="media-body" style="line-height:200%;">Seal the 4 edges of the coverslip with transparent nail varnish.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:0px;"><div class="float-left">5)</div><div class="media-body" style="line-height:200%;">Photograph the embryos with transmitted light on a stereoscope with an RGB camera.</div></div><br/><img alt="" class="layerphoto" height="66" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190316/20190316050345_4964.jpg" width="180"/><br/><strong>Figure 1. Preparing a flat mount.</strong> A. Placing drops of glycerol next to de-yolked embryos (circled) and gently lowering a coverslip on the slide that will be held in place by grease pillars at the corners. B. The finished slide with flat mounted embryos (circled) and coverslip edges sealed with transparent nail varnish.</li></ol></li></ol></li></ol><br/></li><li>Genotyping<br/><em>Note: Genomic DNA can be extracted from intact embryos or from tissue fragments after <em>in situ</em> hybridization. Fragments of tissue are taken before de-yolking and flat mounting the embryos. Perform the following steps in PCR tubesâ1 embryo per tube.</em><br/><ol class="npwtwo"><li>Select the embryos to be genotyped and rinse them together in a small Petri dish or singly in PCR tubes with PBST to remove the glycerol. A few rapid exchanges followed by three 5-min washes on a shaker at room temperature are typically sufficient.</li><li>Transfer the whole embryo with forceps into a PCR tube. If DNA is being extracted from tissue fragments, transfer the embryo onto a glass slide into a 15-20 Î¼l drop of PBST. Cut one-third to one-half of the embryo that is not stained with a micro knife. Transfer the piece to a PCR tube. Ensure that no excess liquid is transferred. Add 87% glycerol to the tissue on the glass slide to continue with <u>de-yolking</u> and flat mounting. </li><li>Add 15 Î¼l of 1x base solution to the tube and incubate at 95 Â°C for 30 min.</li><li>Bring to room temperature and add 15 Î¼l of 1x neutralization solution.<br/><em>Note: This solution is PCR ready (see Note 9).</em> </li></ol></li></ol><p class="pptt" id="biaoti27230">### Data analysis</p><ol class="npwtwok"><li>Whole mount mRNA <em>in situ</em> hybridization coupled to chromogenic detection is a qualitative assay for gene expression in the embryo. In the whole mount format, the distribution of the mRNA of interest in specific tissue(s) at a developmental stage of interest can be assessed. Performing mRNA <em>in situ</em> hybridization for a target in embryos fixed at different developmental stages can help the investigator characterize temporal changes in the mRNA expression of the gene of interest over the course of development. <br/><em>In situ</em> hybridization can also be utilized to assess the effect of the loss or knock down of gene function on the expression of genes of interest. For example, <em>in situ</em> hybridization has been used in our laboratory to assay the wave patterns of cyclic genes arising from their transcriptional oscillations in zebrafish embryos during somitogenesis and to demonstrate that loss of function mutant cyclic gene alleles lead to the loss of these wave patterns (Figure 1, Lleras Forero <em>et al.</em>, 2018). Working from a set of microscope images of embryos, <em>in situ</em> hybridization can also be used as a semi-quantitative endpoint assay to measure the spatio-temporal effects of an experimental perturbation, for example to investigate the upregulation or downregulation of Wnt signaling (Figures 3, 5 and 6, Bajard <em>et al.</em>, 2014). mRNA <em>in situ</em> hybridization has also been used to perform a comparison between the spatial distribution of endogenous cyclic gene expression and a transgenic reporter (Figure S1, Soroldoni <em>et al.</em>, 2014). However, care must be taken in interpreting results that depend on the intensity of the chromogenic signal, either qualitatively, or after quantifying the intensity of the signal with a tool such as plot profile in Fiji (<a href="https://fiji.sc/" target="_blank">https://fiji.sc/</a>), as the intensity of the stain is initially development time-dependent, but the enzymatic reaction can later saturate.</li><li>Designing a genotyping strategy from the extracted genomic DNA will depend entirely on the nature of the specific alleles in the experiment. For some alleles, base transitions will lead to the gain or loss of a site for restriction digestion, allowing for a restriction digestion-based analysis of the PCR amplicon obtained from the genomic DNA using primers specific for the gene of interest. When such a strategy is not possible, the PCR amplicon can be sequenced. Analysis of genotyping sequence data generated can be done with open source or commercially available software (Snapgene, GSL Biotech LLC). For example, we detected base-pair transitions causative for a TALEN induced <em>her1</em> mutation in a <em>her7</em><sup><em>hu2124</em></sup> background initially by sequencing (Figure S1, Lleras Forero <em>et al.</em>, 2018). The mutation led to the loss of a restriction site and embryos were subsequently genotyped by the resistance of the <em>her1</em> mutant allele to restriction digestion. <br/></li></ol><p class="pptt" id="biaoti27231">### Notes</p><ol class="npwtwok"><li>Riboprobe design<br/>In general, using long probes provides a good stain. Long probes yield hybrids with high thermal stability and provide greater signal amplification. Therefore, a good general approach is to design a riboprobe that is complementary to the entire coding sequence of the mRNA. However, such an approach might not be suitable in all cases. For example, to distinguish between mRNAs of paralogs, designing a shorter riboprobe with high complementarity to the paralog of interest is the better approach. In our hands, probes of approximately 300-3,200 bp lengths have reliably yielded a good stain with this protocol (Figure 2). <br/><br/><img alt="" class="layerphoto" height="156" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190316/20190316050514_3807.jpg" width="180"/><br/><strong>Figure 2. Representative results of the <em>in situ</em> hybridization protocol using riboprobes of lengths ~300 and 3,200 base pairs.</strong> A. Stain for <em>paraxial protocadherin </em>(<em>papc/pcdh8</em>) using a riboprobe of 3,137 base pair length, antisense to the coding region of <em>papc</em> mRNA (Yamamoto <em>et al.</em>, 1998). B. Stain for <em>ripply1</em> using a 325 base pair riboprobe, antisense to the coding region of <em>ripply1</em> mRNA (Lleras <em>et al.</em>, 2018). Hybridization has been performed at 60 Â°C on 10-somite stage wild-type embryos. Flat mounted preparations are shown here oriented with the posterior towards the bottom of the panels. Scale bar is 50 Î¼m and applies to both panels. </li><br/><li>Probe concentration<br/>The final working concentration of the probe should be optimized experimentally. In general, concentrations of 0.3-1 ng/Î¼l produce a good <em>in situ</em> stain; however, working concentrations should be titrated every time a new probe is synthesized. The probe can be recycled and reused many times. Note down the initial volume, as the probe will dilute over time and fail to produce a high contrast stain.</li><li>Embryo fixation and storage<br/>Embryos can be fixed and dehydrated ahead of time. The quality of the fix will affect the signal to noise ratio of the stain. Freshly prepared fix gives the best results, but fix up to a week old can be used. Embryos should not be incubated in fix for more than 4 h at room temperature or 1 week at 4 Â°C. Fixed and dehydrated embryos can be stored in methanol at -20 Â°C indefinitely.</li><li>Hybridization temperature<br/>The hybridization temperature is probe-dependent and has to be experimentally determined for each probe. The choice of temperature also sets the stringency of the protocolâstringency can be increased by increasing the hybridization temperature and decreased by lowering it. When using probes that need to discriminate between targets with high sequence similarity, the stringency of the protocol needs to be raised to favor the formation of the desired RNA hybrids as a 1% base pair mismatch decreases the thermal stability of RNA-RNA duplexes by roughly 1 Â°C (Wetmur <em>et al.</em>, 1976).<br/>In our hands, a good starting temperature has been 60 Â°C for probes designed to detect the coding sequence of the target mRNA (Figures 2 and 3). For short probes and intronic probes, 55 Â°C has been used as the starting temperature. The hybridization temperature has then been optimized by changing the temperature in 2 Â°C steps.<br/> <br/><img alt="" class="layerphoto" height="139" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190316/20190316050555_9698.jpg" width="180"/><br/><strong>Figure 3. Hybridization at 60 Â°C is sufficiently stringent to achieve a high contrast final stain with a specific probe.</strong> <em>In situ</em> hybridization to detect <em>xirp2a</em> (Deniziak <em>et al.</em>, 2007) performed at 70 Â°C and 60 Â°C in 36 hpf wild-type embryos. A. Stain after 110 min of development when probe is hybridized at 70 Â°C. B. Stain after 50 min of development when hybridized at 60 Â°C. Riboprobe concentration used is 0.5 ng/Î¼l in hybridization buffer. Scale bar is 100 Î¼m and applies to both panels.</li><br/><li>Improving contrast of the stain<br/>Adding dextran sulfate to the hybridization buffer will yield high contrast stains with a relatively short development time, compared to stains produced without dextran sulfate (Figure 4). If long development times are required to detect the target mRNA, adding 10% PVA to the staining solution will decrease the rate at which background staining will develop.<br/><br/><img alt="" class="layerphoto" height="137" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190316/20190316050653_9615.jpg" width="180"/><br/><strong>Figure 4. Addition of dextran sulfate to the hybridization buffer enhances the contrast of the final stain.</strong> <em>In situ</em> hybridization to detect <em>xirp2a</em> performed at 60 Â°C in 36 hpf wild-type embryos. A. Stain after 25 min of development when probe is hybridized in hybridization buffer B. Stain after 25 mins of development when probe is hybridized in hybridization buffer with 5% dextran sulfate (DS). Riboprobe concentration used is 1 ng/Î¼l. Scale bar is 100 Î¼m and applies to both panels.</li><br/><li>Troubleshooting<br/><strong><em>No signal from an antisense riboprobe</em></strong><br/><em>In vitro</em> transcription has not been successful: The integrity of the plasmid could have been compromised. Re-prep the plasmid or generate a template by PCR.<br/><br/><strong><em>Sub-optimal signal to noise ratio <br/></em></strong> <ol class="npwthree"><li>Poor quality probe: Control for probe quality by performing agarose gel electrophoresis of templates and synthesized riboprobes. They should resolve as distinct bands. Templates can be sequenced to assess sequence accuracy.</li><li>Probe concentration is too high: Diluting the probe can reduce the background.</li><li>Background due to non-specific hybridization: Increase the stringency of the protocol by raising the hybridization temperature.</li><li>Background due to prolonged development: Reduce the hybridization temperature to lower the stringency of the protocol, add 10% PVA to the staining solution.</li></ol><br/><em><strong>Poor signal intensity</strong></em><br/><ol class="npwthree"><li>Probe is too dilute: Synthesize new probe, increase the concentration of the probe.</li><li>Probe design: Increase the length of the probe for better signal amplification.</li><li>Hybridization is too stringent: Decrease the hybridization temperature.</li><li>Add 5% dextran sulfate to the hybridization buffer. </li></ol></li><li>Mold for photography of whole mount preparations <br/>An imaging dish for photographing whole mount specimens can be made by casting a mold for the embryos in 1-1.5% agarose in distilled water using a polydimethylsiloxane (PDMS) negative. This negative is cast from a positive made from Perspex plastic machined with the conical depressions of the desired diameter and depth (Figure 4B, the positive and negative for the imaging mold are made by our laboratory and are described in Herrgen <em>et al.</em>, 2009). Add 87% glycerol to the dish. Transfer the stained embryos into the mold and orient them using the tip of a tungsten wire, an eyelash or a fine pipette tip such that their yolks fit in the conical depressions under a stereomicroscope (Figure 5).<br/><br/><img alt="" class="layerphoto" height="177" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190316/20190316050739_8349.jpg" width="180"/><br/><strong>Figure 5. Preparing to document the whole mount <em>in situ</em> hybridization preparations.</strong> A PDMS negative with an array of raised cones of 0.8 mm diameter and 0.4 mm height is used to cast the agarose mold with conical depressions. The yolk of the embryos fits into the depressions and embryos can be oriented laterally such that they lie flat on the surface of the agarose. A. Casting a mold to orient embryos in 1%-1.5% agarose. B. Embryos are oriented in the conical cavities of the mold in 87% glycerol. C-Câ. Polydimethylsiloxane (PDMS) negative used to cast the mold. D. Agarose mold for the embryos. E. Embryos oriented for photography. </li><br/><li>De-yolking the embryo<br/><ol class="npwthree"><li>We recommend referring to the video protocol for de-yolking and flat mounting (Cheng <em>et al.</em>, 2014).</li><li>A sharpened tungsten wire or âhookâ can be used instead of an eyelash for de-yolking embryos. To sharpen and shape the wire, fire-polish the end of the tungsten wire by holding the last 0.5 to 1 cm length of the wire in a Bunsen burner flame until it glows orange for 30-60 s. With a pair of forceps shape the flamed wire into a hook in 87% glycerol. Detailed instructions to make the tungsten hook and holder are provided in Picker <em>et al.</em>, 2009.</li></ol></li><li>Using the embryo extract for PCR<br/>A dilution series ranging from no dilution up to 1:5 in distilled water should be trialed to find the optimal volume of genomic DNA extract that is necessary for PCR for a given embryonic stage. Extracts from 8 to 10-somite stage whole embryos and embryo fragments typically do not need any further dilution and 1-2 Î¼l can be used per 10 Î¼l PCR reaction. Extracts from tissue of 24-36 hpf embryos typically need to be diluted 1:4.<br/></li></ol><p class="pptt" id="biaoti27232">### Recipes</p><p class="ppzz"><em>Note: This protocol does not require DEPC treated water or solutions or specific RNase free conditions. Distilled or ultrapure water, autoclaved glassware and single use polypropylene consumables such as microfuge tubes, centrifuge tubes, plastic Pasteur pipettes micropipette tips are sufficient.</em> </p><ol class="npwtwok"><li>1x TAE<br/>40 mM Tris-acetate<br/>20 mM Acetate<br/>1 mM EDTA<br/>pH 8.3<br/>Dilute 10x TAE 1:10 in ultrapure water<br/>Store at room temperature<br/><strong>To make 1 L of 10x TAE (0.4 M Tris-acetate, 0.2 M Acetate and 0.1 M EDTA):</strong><br/>48.5 g Tris base<br/>11.4 ml glacial acetic acid<br/>20 ml 0.5 M EDTA, pH 8.0<br/>Make up to 1 L with ultrapure water<br/>Autoclave</li><li>100x PTU stock (0.3% [weight by volume] in distilled water) (100 ml)<br/>Dissolve 0.3 g <em>N</em>-Phenylthiourea (PTU) in 100 ml of distilled water with heating and stirring (store in the dark at room temperature)</li><li>10% Tween-20 (dilute Tween-20 1:10 in distilled water) (100 ml)<br/>10 ml Tween-20<br/>Make up to 100 ml with distilled water<br/>Stir till a homogeneous solution is achieved <br/>Store at room temperature</li><li>PBST <br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub><br/>1.8 mM KH<sub>2</sub>PO<sub>4</sub> in distilled water, pH 7.4<br/>0.1% Tween-20<br/><strong>To make 1 L PBST:</strong><br/>8 g NaCl<br/>0.2 g KCl<br/>1.44 g Na<sub>2</sub>HPO<sub>4</sub><br/>0.24 g KH<sub>2</sub>PO<sub>4</sub><br/>Make up to 1 L with distilled water after adjusting pH and autoclave<br/>Add 10 ml of 10% Tween-20<br/>Store at room temperature</li><li>4% PFA in PBST (4% weight by volume in PBST) (100 ml)<br/><ol class="npwthree"><li>Pre-warm 100 ml PBS to 68 Â°C in a water bath<br/>Add 4 g PFA <br/>Incubate at 68 Â°C with periodic agitation for 50 â 60 mins<br/>Add 1 ml of 10% Tween-20 when the solution has cooled to room temperature <br/>Store for up to a week at 4 Â°C<br/><em>Note: The PFA should dissolve in 50 min to 1 h</em><br/>OR</li><li>Dilute 16% PFA 1:4 in PBST. Store for up to a week at 4 Â°C</li></ol></li><li>Proteinase K stock<br/>1 mg/ml in distilled water<br/>Store as aliquots at -20 Â°C</li><li>20x SSC<br/>3 M NaCl<br/>300 mM tri-sodium citrate<br/>pH 7.0-7.2<br/><strong>To make 1 L 20x SSC:</strong><br/>175.2 g NaCl<br/>88.2 g tri-sodium citrate<br/>Make up to 1 L with distilled water after adjusting pH<br/>Autoclave<br/>Store at room temperature</li><li>50 mg/ml Heparin stock<br/>Dissolve in Heparin in distilled water to a final concentration of 50 mg/ml<br/>Store at 4 Â°C</li><li>Hybridization buffer (Hyb)<br/>50% Formamide<br/>5x SSC<br/>50 Î¼g/ml Heparin<br/>0.5 mg/ml Torula-RNA<br/>0.1% Tween-20<br/>Optional: 5% weight by volume dextran sulfate<br/><strong>To make 50 ml Hyb:</strong> <br/>25 ml formamide<br/>12.5 ml 20x SSC<br/>50 Î¼l Heparin stock<br/>25 mg Torula-RNA<br/>2.5 g dextran sulfate<br/>Vortex well till a homogenous is achieved<br/>Add 500 Î¼l 10% Tween-20<br/>Make up to 50 ml with distilled water<br/>Store at -20 Â°C<br/><em>Note: Dextran sulfate is omitted from hyb used for pre-hybridization.</em> </li><li>Post hybridization washes<br/><strong>Wash 1: </strong>50% Formamide, 2x SSC, 0.1% Tween-20 in distilled water<br/>To make 50 ml: <br/>25 ml Formamide<br/>5 ml 20x SSC<br/>500 Î¼l 10% Tween-20<br/>Make up to 50 ml with distilled water<br/><strong>Wash 2:</strong> 2x SSC, 0.1% Tween-20 in distilled water<br/>To make 50 ml: <br/>5 ml 20x SSC<br/>500 Î¼l 10% Tween-20<br/>Make up to 50 ml with distilled water<br/><strong>Wash 3:</strong> 0.2x SSC, 0.1% Tween-20 in distilled water<br/>To make 50 ml: <br/>500 Î¼l 20x SSC<br/>500 Î¼l 10% Tween-20<br/>Make up to 50 ml with distilled water<br/><em>Note: Make the appropriate volume of wash solution fresh.</em> </li><li>MABT<br/>100 mM Maleic Acid <br/>150 mM NaCl in distilled water<br/>pH 7.5<br/>0.1% Tween-20<br/><strong>To make 1 L: </strong><br/>8.77 g maleic acid<br/>11.6 g NaCl<br/>Adjust pH with NaOH pellets<br/>Make up to 1 L with distilled water<br/>Autoclave<br/>Add 10 ml 10% Tween-20<br/>Store at room temperature</li><li>10% blocking reagent (Roche block)<br/>Prepare 10% blocking reagent in MAB as per the manufacturerâs instructions<br/>Store at -20 Â°C as aliquots</li><li>2% Blocking reagent in MABT (2% Roche block)<br/>Dilute 10% blocking reagent (Roche block) 1:5 in MABT<br/>Make fresh</li><li>Staining buffer<br/>100 mM Tris-HCl, pH 9.5<br/>100 mM NaCl<br/>50 mM MgCl<sub>2</sub> in distilled water<br/><strong>To make 50 ml:</strong><br/>5 ml 1 M Tris-HCl, pH 9.5<br/>5 ml 1 M NaCl<br/>2.5 ml 1 M MgCl<sub>2</sub><br/>500 Î¼l 10% Tween-20<br/><em>Note: Make fresh.</em> </li><li>Staining solution <br/>4.5 Î¼l of NBT and 3.5 Î¼l of BCIP per 1 ml of the staining buffer<br/>Vortex to mix<br/>Protect from light<br/><em>Note: Prepare the appropriate volume fresh.</em><br/>Optional: add PVA to a final concentration of 10%, weight by volume<br/><strong>To make 10 ml of staining solution with 10% PVA:</strong><br/>Add 1 g PVA to 10 ml of staining buffer (without Tween-20)<br/>Bring into solution with stirring and heating<br/>Cool to room temperature<br/>Add 45 Î¼l NBT and 35 Î¼l of BCIP<br/>Vortex to mix<br/>Protect from light<br/><em>Note: 10 ml is the minimum volume that can be made.</em> </li><li>87% glycerol in distilled water (weight by weight)<br/><strong>To make 100 g: </strong><br/>87 g glycerol in 13 g distilled water<br/>Stir till a homogenous solution is achieved<br/>Store at room temperature</li><li>50x base solution<br/>1.25 M KOH<br/>10 mM EDTA in distilled water<br/><strong>To make 200 ml: </strong><br/>14 g KOH<br/>4 ml 0.5 M EDTA<br/>Add distilled water to a final volume of 200 ml<br/>Store at room temperature</li><li>1x base solution<br/>Dilute 50x base solution 1:50 in distilled water<br/><em>Note: Prepare the appropriate volume fresh.</em> </li><li>50x neutralization solution <br/>2 M Tris-HCl in distilled water<br/><strong>To make 200 ml:</strong><br/>63 g Tris-HCl<br/>Add distilled water to a final volume of 200 ml<br/>Store at room temperature</li><li>1x neutralization solution<br/>Dilute 50x neutralization solution 1:50 in distilled water<br/><em>Note: Prepare the appropriate volume fresh.</em><br/></li></ol><p class="pptt" id="biaoti27233">### Acknowledgments</p><p class="ppzz">The protocol presented here is based on G. Hauptmannâs work (PMID: 11252185), which was implemented and continuously improved by D. Soroldoni. We acknowledge A. Faro and G. Gestri of the Wilson laboratory at UCL, UK who shared the genomic DNA extraction protocol with us. We also thank O. Venzin of the Oates lab for constructive feedback on the manuscript. <br/>  Funding: Institutional support from the Swiss Federal Institute of Technology in Lausanne (EPFL) (to ACO and RN); The Francis Crick Institute (receiving its core funding from Cancer Research UK, the Medical Research Council, and Wellcome) (to ACO and RN); Wellcome (WT098025MA to ACO); the Medical Research Council (MC_UP_1202/3 to ACO and RN).</p><p class="pptt" id="biaoti27234">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti27235">### References</p><ol class="npwtwok"><li>Bajard, L., Morelli, L. G., Ares, S., Pecreaux, J., Julicher, F. and Oates, A. C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24595291" target="_blank">Wnt-regulated dynamics of positional information in zebrafish somitogenesis.</a> <em>Development </em>141(6): 1381-1391.</li><li>Cheng, C. N., Li, Y., Marra, A. N., Verdun, V. and Wingert, R. A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25078510" target="_blank">Flat mount preparation for observation and analysis of zebrafish embryo specimens stained by whole mount <em>in situ</em> hybridization.</a> <em>J Vis Exp</em>(89). doi: 10.3791/51604.</li><li>Choi, H. M., Chang, J. Y., Trinh le, A., Padilla, J. E., Fraser, S. E. and Pierce, N. A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21037591" target="_blank">Programmable <em>in situ</em> amplification for multiplexed imaging of mRNA expression.</a> <em>Nat Biotechnol</em> 28(11): 1208-1212.</li><li>Deniziak, M., Thisse, C., Rederstorff, M., Hindelang, C., Thisse, B. and Lescure, A. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17123513" target="_blank">Loss of selenoprotein N function causes disruption of muscle architecture in the zebrafish embryo.</a> <em>Exp Cell Res</em> 313(1): 156-167.</li><li>Gross-Thebing, T., Paksa, A. and Raz, E. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25124741" target="_blank">Simultaneous high-resolution detection of multiple transcripts combined with localization of proteins in whole-mount embryos.</a> <em>BMC Biol</em> 12: 55.</li><li>Hauptmann, G. and Gerster, T. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7940754" target="_blank">Two-color whole-mount <em>in situ</em> hybridization to vertebrate and <em>Drosophila</em> embryos.</a> <em>Trends Genet</em> 10(8): 266.</li><li>Herrgen, L., Schroter, C., Bajard, L. and Oates, A. C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19378108" target="_blank">Multiple embryo time-lapse imaging of zebrafish development.</a> <em>Methods Mol Biol</em> 546: 243-254.</li><li>Jambor, H., Surendranath, V., Kalinka, A. T., Mejstrik, P., Saalfeld, S. and Tomancak, P. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25838129" target="_blank">Systematic imaging reveals features and changing localization of mRNAs in <em>Drosophila</em> development.</a> <em>Elife</em> 4: e05003.</li><li>Kimmel, C. B., Ballard, W. W., Kimmel, S. R., Ullmann, B. and Schilling, T. F. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8589427" target="_blank">Stages of embryonic development of the zebrafish.</a> <em>Dev Dyn</em> 203(3): 253-310.</li><li>Kiyama, H. and Emson, P. C. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1682362" target="_blank">An <em>in situ</em> hybridization histochemistry method for the use of alkaline phosphatase-labeled oligonucleotide probes in small intestine.</a> <em>J Histochem Cytochem</em> 39(10): 1377-1384.</li><li>Lauter, G., Soll, I. and Hauptmann, G. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21726453" target="_blank">Two-color fluorescent <em>in situ</em> hybridization in the embryonic zebrafish brain using differential detection systems.</a> <em>BMC Dev Biol</em> 11: 43.</li><li>Little, S. C., Tikhonov, M. and Gregor, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23953111" target="_blank">Precise developmental gene expression arises from globally stochastic transcriptional activity.</a> <em>Cell</em> 154(4): 789-800.</li><li>Lleras Forero, L., Narayanan, R., Huitema, L. F., VanBergen, M., Apschner, A., Peterson-Maduro, J., Logister, I., Valentin, G., Morelli, L. G., Oates, A. C. and Schulte-Merker, S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29624170" target="_blank">Segmentation of the zebrafish axial skeleton relies on notochord sheath cells and not on the segmentation clock.</a> <em>Elife</em> 7: e33843.</li><li>McConaughy, B. L., Laird, C. D. and McCarthy, B. J. (1969). <a href="https://www.ncbi.nlm.nih.gov/pubmed/4980190" target="_blank">Nucleic acid reassociation in formamide.</a> <em>Biochemistry</em> 8(8): 3289-3295.</li><li>Picker, A., Roellig, D., PourquiÃ©, O., Oates, A. C. and Brand, M. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/4980190" target="_blank">Tissue micromanipulation in zebrafish embryos.</a> <em>Methods Mol Biol</em> 546: 153-172.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682.</li><li>Schulte-Merker, S., Ho, R. K., Herrmann, B. G. and Nusslein-Volhard, C. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1295726" target="_blank">The protein product of the zebrafish homologue of the mouse T gene is expressed in nuclei of the germ ring and the notochord of the early embryo.</a> <em>Development</em> 116(4): 1021-1032.</li><li>Schulte-Merker, S., van Eeden, F. J., Halpern, M. E., Kimmel, C. B. and NÃ¼sslein-Volhard, C. (1994). <a href="https://www.ncbi.nlm.nih.gov/pubmed/7600949" target="_blank"><em>No tail</em> (<em>ntl</em>) is the zebrafish homologue of the mouse <em>T </em>(<em>Brachyury</em>) gene.</a> <em>Development</em> 120(4): 1009-1015.</li><li>Soroldoni, D., Jorg, D. J., Morelli, L. G., Richmond, D. L., Schindelin, J., Julicher, F. and Oates, A. C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25013078" target="_blank">Genetic oscillations. A Doppler effect in embryonic pattern formation.</a> <em>Science</em> 345(6193): 222-225.</li><li>Stapel, L. C., Lombardot, B., Broaddus, C., Kainmueller, D., Jug, F., Myers, E. W. and Vastenhouw, N. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26700682" target="_blank">Automated detection and quantification of single RNAs at cellular resolution in zebrafish embryos.</a> <em>Development</em> 143(3): 540-546.</li><li>Thisse, B. and Thisse, C. (2004). <a href="https://zfin.org/ZDB-PUB-040907-1" target="_blank">Fast release clones: a high throughput expression analysis.</a> <em>ZFIN Direct Data Submission</em> ZFIN ID: ZDB-PUB-040907-1.</li><li>Thisse, B., Pflumio, S., FÃ¼rthauer, M., Loppin, B., Heyer, V., Degrave, A., Woehl, R., Lux, A., Steffan, T., Charbonnier, X.Q. and Thisse, C. (2001). <a href="https://zfin.org/ZDB-PUB-010810-1" target="_blank">Expression of the zebrafish genome during embryogenesis (NIH R01 RR15402).</a> <em>ZFIN Direct Data submission</em> ZFIN ID: ZDB-PUB-010810-1.</li><li>Thisse, C. and Thisse, B. (2005). <a href="https://zfin.org/ZDB-PUB-051025-1" target="_blank">High throughput expression analysis of ZF-models consortium clones.</a> <em>ZFIN Direct Data submission</em> ZFIN ID: ZDB-PUB-051025-1.</li><li>Thisse, C. and Thisse, B. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18193022" target="_blank">High-resolution <em>in situ</em> hybridization to whole-mount zebrafish embryos.</a> <em>Nat Protoc</em> 3(1): 59-69.</li><li>Trivedi, V., Choi, H. M. T., Fraser, S. E. and Pierce, N. A. (2018). <a href="http://dev.biologists.org/content/145/1/dev156869" target="_blank">Multidimensional quantitative analysis of mRNA expression within intact vertebrate embryos.</a> <em>Development</em> 145(1): dev156869.</li><li>Wang, F., Flanagan, J., Su, N., Wang, L. C., Bui, S., Nielson, A., Wu, X., Vo, H. T., Ma, X. J. and Luo, Y. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22166544" target="_blank">RNAscope: a novel <em>in situ</em> RNA analysis platform for formalin-fixed, paraffin-embedded tissues.</a> <em>J Mol Diagn</em> 14(1): 22-29.</li><li>Wetmur, J. G. (1976). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7992" target="_blank">Hybridization and renaturation kinetics of nucleic acids.</a> <em>Annu Rev Biophys Bioeng</em> 5: 337-361.</li><li>Wetmur, J. G. and Davidson, N. (1968). <a href="http://www.ncbi.nlm.nih.gov/pubmed/5637197" target="_blank">Kinetics of renaturation of DNA.</a> <em>J Mol Biol</em> 31(3): 349-370.<br/></li><li>Yamamoto, A., Amacher, S. L., Kim, S. H., Geissert, D., Kimmel, C. B. and De Robertis, E. M. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9693142" target="_blank">Zebrafish paraxial protocadherin is a downstream target of spadetail involved in morphogenesis of gastrula mesoderm.</a> <em>Development</em> 125(17): 3389-3397.<br/></li></ol>