<body>
            <h1>Viral Chromosome Conformation Capture (V3C) Assays for Identifying Trans-interaction  Sites between Lytic Viruses and the Cellular Genome</h1>
            <p><strong>Authors:</strong> Kinjal  Majumder, Maria  Boftsi, David J Pintel</p>
            <h2>Abstract</h2>
            <p>The folding mechanisms of the mammalian genome package our genetic material into the nucleus, and in doing so, dictate its appropriate replication and expression. Chromosome conformation capture technology has enabled the dissection of the folding principles of the cellular genome. This has led to a better understanding of the role played by architectural proteins in forming and dissolving 3D-chromatin-structure. These assays are based on the principle of crosslinking distant cellular sites that are proximal to each other in 3D space using formaldehyde followed by digestion of formed hybrid DNA junctions. Invading viruses, such as the lytic parvovirus Minute Virus of Mice (MVM), establish distinct replication centers within the nuclear environment at cellular sites that preferentially undergo DNA damage, but do not integrate into the cellular DNA. We have adapted chromosome conformation capture technology to study the trans-interaction between MVM and the cellular genome, which we have dubbed V3C, which can be extended to a whole-genome analysis we term V3C-seq. This protocol describes the procedure for performing, as well as analyzing V3C-seq assays, and can be adapted for mapping the cellular interaction sites of any non-integrating DNA virus.</p>
            <p><strong>Keywords:</strong> Chromosome conformation capture, Parvovirus, DNA damage response, Topologically associating domains, Fragile sites, Host-pathogen genome interactions</p>
            <h2>Background</h2><p>Chromosome conformation capture technologies have helped us gain significant insights into the folding principles of the mammalian genome, helping identify the short-range promoter-enhancer loops that underlie epigenetic regulation of gene expression, as well as long-range multi-loop domains that form sub-chromosomal chromatin compartments (Dekker <em>et al.</em>, 2013; Denker and de Laat, 2016). While these technologies have helped define important rules related to the patterns of cis-folding of the cellular genome, their implementation in studying the trans-interaction between the genome of an invading DNA virus and its target cell remained largely unstudied. In this protocol, we describe the procedure for studying the interactome of the parvovirus Minute Virus of Mice (Figure 1). Owing to its 5 kilobase genome, the application of a modified Circular Chromosome Conformation Capture (4C) assay combined with high-throughput sequencing (Van De Werken <em>et al.</em>, 2012) allowed the generation of a high resolution map of the sites on cellular DNA where MVM localizes during infection (Majumder <em>et al.</em>, 2018).<br/><br/><img alt="" class="layerphoto" height="107" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190313/20190313012706_7280.jpg" width="200"/><br/><strong>Figure 1. Schematic of the V3C-seq assay with approximate timeline</strong></p>
        </body>

<p class="pptt" id="biaoti27280">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (Fisher, catalog numbers: 02-707-426, 02-707-403, 02-707-438)</li><li>15 ml Falcon tubes (Thermo Scientific, catalog number: 339650)</li><li>50 ml Falcon tubes (Thermo Scientific, catalog number: 339652)</li><li>Eppendorf tubes (Fisher, catalog number: 05-408-129)</li><li>Stericup-GP Sterile Vacuum Filtration System (Millipore, catalog number: SCGPU11RE)</li><li>Disposable Cell Scrapers (Fisher, catalog number: 08-100-241)</li><li>Murine A9 fibroblasts</li><li>Human NB324K kidney cells</li><li>MVMp virus</li><li>Primers (IDT)<br/>Inverse PCR primer:<br/>F: 5â-gaggcaaggttggtcactactt-3â<br/>R: 5â-caggaactttgccccattta-3â<br/>Nested Inverse PCR primer sequences:<br/>F: 5â-ggcaaggttggtcactactttt-3â<br/>R: 5â-gccccatttagcacagtagc-3â<br/>Primers for determining <em>Hin</em>dIII digestion efficiency:<br/>P1: 5â-agaaaattggcatcggtttg-3â<br/>P2: 5â-tgactgtagctcctccaaattgt-3â<br/>P3: 5â-ttttggcaggtgtccttttc-3â</li><li>iTaq Universal SYBR Green Supermix (Bio-Rad, catalog number: 1725121)</li><li>Liquid nitrogen (Airgas Mid-America)</li><li>37% formaldehyde (Fisher, catalog number: F79-4)</li><li>1 M glycine (Fisher, catalog number: BP381-5)</li><li>Protease Inhibitor Cocktail (MedChemExpress, catalog number: HY-K0010)</li><li>Sodium Dodecyl Sulfate (SDS; Fisher, catalog number: BP166-500)</li><li>Triton X-100 (Fisher, catalog number: BP151-100)</li><li>Proteinase K (Thermo Scientific, catalog number: AM2542)</li><li>RNase A (Sigma-Aldrich, catalog number: R4875-500MG)</li><li>Glycogen (Sigma-Aldrich, catalog number: 10901393001)</li><li>Isopropanol (Fisher, catalog number: A451SK-4)</li><li>70% ethanol (Fisher, catalog number: AC615090010)</li><li>Buffer EB (QIAGEN, catalog number: 19086)</li><li>PCR Purification Kit (QIAGEN, catalog number: 28106)</li><li><em>Nla</em>III (NEB, catalog number: R0125L) (store at -80 Â°C)</li><li><em>Hin</em>dIII (NEB, catalog number: R0104L)</li><li>10x NEB Restriction Enzyme buffer 2.1, Cutsmart buffer (supplied with <em>Hin</em>dIII and <em>Nla</em>III enzymes; see above for catalog numbers)</li><li>T4 DNA Ligase (NEB, catalog number: M0202L)</li><li>10x NEB T4 DNA Ligase buffer (NEB, catalog number: B0202S)</li><li>Platinum Taq DNA Polymerase High Fidelity (Thermo Fisher, catalog number: 11304029)</li><li>NEBNext Ultra II DNA Library Prep Kit for Illumina (NEB, catalog number: E7645S)</li><li>DMEM (Gibco, catalog number: 11965-092)</li><li>L-glut (Fisher, catalog number: BP379-100)</li><li>Fetal Calf Serum (FCS) (Fisher, catalog number: SH30073.03)</li><li>Gentamicin (Fresenius Kabi USA , catalog number: 63323-010-20)</li><li>Phosphate Buffered Saline (Fisher, catalog number: SH3025601)</li><li>Tris-base (Fisher, catalog number: BP152-5)</li><li>Sodium Chloride (NaCl) (Fisher, catalog number: BP358-10)</li><li>Calcium Chloride (Sigma-Aldrich, catalog number: C7902-500G)</li><li>Potassium Chloride (Fisher, catalog number: BP366-500)</li><li>Magnesium Sulfate (MgSO<sub>4</sub>) (Fisher, catalog number: BP213-1)</li><li>Sodium Dihydrogen Phosphate (Amresco, catalog number: 0571-500G)</li><li>Glucose (Fisher, catalog number: D16-1)</li><li>NP40 (Sigma-Aldrich, catalog number: CA-630)</li><li>L-arginine (Sigma-Aldrich, catalog number: A8094-25G)</li><li>L-cysteine (Sigma-Aldrich, catalog number: C7602-25G)</li><li>L-histidine (Sigma-Aldrich, catalog number: H6034-25G)</li><li>L-leucine (Sigma-Aldrich, catalog number: L8912-25G)</li><li>L-lysine (Sigma-Aldrich, catalog number: L8662-25G)</li><li>L-methionine (Sigma-Aldrich, catalog number: M5308-25G)</li><li>L-phenylalanine (Sigma-Aldrich, catalog number: P5482-25G)</li><li>L-threonine (Sigma-Aldrich, catalog number: T8441-25G)</li><li>L-tryptophan (Sigma-Aldrich, catalog number: T8941-25G)</li><li>L-tyrosine (Sigma-Aldrich, catalog number: T8566-25G)</li><li>L-valine (Sigma-Aldrich, catalog number: V0513-25G)</li><li>100x MEM Vitamin Solution (Thermo Fisher, catalog number: 11120052)</li><li>HEPES (Fisher, catalog number: BP410-500)</li><li>Sodium bicarbonate (Fisher, catalog number: BP328-500)</li><li>EDTA (Fisher, catalog number: S311-3)</li><li>1x Phenol:Chloroform:Isoamyl alcohol (Fisher, catalog number: BP1752I-400)</li><li>Chloroform (Fisher, catalog number: C606SK-4)</li><li>Phenol red</li><li>TE buffer (see Recipes)</li><li>Complete DMEM media (see Recipes)</li><li>3C Lysis buffer (see Recipes)</li><li>Earleâs 10x salts (see Recipes) </li><li>Isoleucine minus media (see Recipes)<br/></li></ol><em>Note: The Eppendorf tubes and Falcon tubes used in this protocol should be chloroform resistant.</em><br/><p class="pptt" id="biaoti27281">### Equipment</p><ol class="npwtwok"><li>Micro-Pipettes (P2, P20, P200, P1000)</li><li>Biorad T100 Thermal Cycler (Bio-Rad, catalog number: 1861096) </li><li>Illumina NextSeq 500 (Illumina, catalog number: SY-415-1001)</li><li>Bellco Rocker Platform (Marshall Scientific) </li><li>Vortex Genie 2 (Scientific Industries Inc., catalog number: SI-0236)</li><li>Refrigerated centrifuge (Eppendorf, catalog number: 5417C) </li><li>-80 Â°C freezer</li><li>-20 Â°C freezer</li><li>4 Â°C refrigerator </li><li>37 Â°C Shaker (Eppendorf Thermomixer, catalog number: 05-400-200) </li><li>NanoDrop spectrophotometer (Thermo Fisher, catalog number: ND-ONE-W)</li><li>Vacuum manifold (for aspirating supernatants) </li><li>CFX Connect Real-Time PCR Detection System (Bio-Rad, catalog number: 1855200)<br/></li></ol><p class="pptt" id="biaoti27282">### Software</p><ol class="npwtwok"><li>Bowtie 2 (Langmead and Salzberg, 2012)</li><li>SAMtools (Li <em>et al.</em>, 2009)</li><li>BEDTools (Quinlan and Hall, 2010)<br/></li></ol><p class="pptt" id="biaoti27283">### Procedure</p><ol class="npwone"><li>Parasynchronization and infection of cells<ol class="npwtwo"><li>Plate 5 x 10<sup>6</sup> cells at a concentration of 0.5 x 10<sup>6</sup> cells/ml in 10 ml Isoleucine-minus media in 15 cm dishes for 36-42 h at 37 Â°C and 5% CO<sub>2</sub>. Murine A9 fibroblasts and human NB324K kidney cells are permissive to MVMp infection and can be parasynchronized by isoleucine deprivation. </li><li>Release cells into G1/S-phase by aspirating the media and adding 10 ml complete DMEM media. Simultaneously, infect with MVMp virus at an MOI of 5 (25 Âµl of MVMp at 1 x 10<sup>6</sup> Pfu/Âµl) for 16 h. <br/><em>Note: MVMp replication starts as cells enter into S-phase, approximately 10 h post-release into complete DMEM.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Crosslinking and lysis<ol class="npwtwo"><li>Crosslink cells in the dish by adding 540 Âµl of 37% formaldehyde (final concentration of 2%). Mix well by pipetting and incubate at room temperature for 10 min on a rocker (12 oscillations/min). </li><li>Quench the crosslinking reaction by adding 1.425 ml of 1 M glycine (final concentration of 0.125 M). Mix well by pipetting and incubate at room temperature for 5 min on a rocker (12 oscillations/min).</li><li>Gently scrape cells using a disposable cell scraper (by moving horizontally, and vertically) and collect the cell suspension (in 12 ml media, formaldehyde and glycine) in 15 ml Falcon tubes. </li><li>Centrifuge at 500 <em>x g</em> for 5 min at 4 Â°C in a refrigerated centrifuge. Aspirate supernatant with a vacuum manifold or gently with a pipette.</li><li>Wash cells with 10 ml PBS.</li><li>Centrifuge at 500 <em>x g</em> for 5 min at 4 Â°C in a refrigerated centrifuge. Aspirate supernatant.</li><li>Suspend pellet in 5 ml cold 3C Lysis buffer (freshly supplemented with protease inhibitors) and incubate on ice for 10 min.</li><li>Centrifuge at 3,000 <em>x g</em> for 5 min at 4 Â°C. Aspirate the supernatant. A small white pellet should be visible at the bottom of the tube. This is mostly made up of cell nuclei. The tube can be snap frozen in liquid nitrogen at this point and stored at -80 Â°C. Thaw samples on ice if starting from this step.<br/><em>Note: Aspiration should be performed using a vacuum manifold. In its absence, pipettors may be used to carefully remove the supernatants without disturbing the cell (or nuclear) pellet.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Primary restriction digest<ol class="npwtwo"><li>Suspend nuclei in 500 Âµl 1.2x NEB buffer 2.1 (60 Âµl of 10x stock diluted in 440 Âµl ddH<sub>2</sub>O) and transfer to Eppendorf tube.</li><li>Add 7.5 Âµl of 20% SDS (final concentration of 0.3%) and place tube in a shaker at 37 Â°C at 900 rpm. <br/><em>Note: This process permeabilizes the nuclei. However, the formation of bubbles or too much SDS affects enzyme activity. Therefore, avoid forming bubbles, and ensure proper concentration of SDS solution.</em> </li><li>Incubate for 1 h at 37 Â°C while shaking at 900 rpm.</li><li>Add 50 Âµl of 20% Triton X-100 (final concentration of 2%). <br/><em>Note: Prepare 20% Triton solution well in advance, mix and keep at 56 Â°C to ensure homogeneous solution. Otherwise, SDS is inadequately sequestered, leading to premature inactivation of the restriction enzyme.</em> </li><li>Incubate for 1 h at 37 Â°C while shaking at 900 rpm.</li><li>Take out 10 Âµl aliquot as undigested gDNA Control. Store at -20 Â°C.</li><li>Add 20 Âµl (400 U) of <em>Hin</em>dIII to remaining sample and incubate overnight at 37 Â°C while shaking at 900 rpm.</li><li>Add 15 Âµl (300 U) of additional <em>Hin</em>dIII and incubate on a 37 Â°C shaker at 900 rpm for 4 h.</li><li>Take out 10 Âµl aliquot as digested DNA Control. Store at -20 Â°C. <br/></li><li>Reverse crosslinks and determine digestion efficiency as follows:<ol class="npwthree"><li>Add 2 Âµl of Proteinase K to the undigested DNA and digested DNA tubes, and incubate at 56 Â°C for 1 h. </li><li>Dilute the samples to 100 Âµl with 90 Âµl ddH<sub>2</sub>O. </li><li>Add 100 Âµl of phenol:chloroform:isoamyl alcohol (25:24:1) and vortex (setting: 10) for 10 s.</li><li>Centrifuge at 13,000 <em>x g</em> at room temperature for 5 min. </li><li>Transfer supernatant to new Eppendorf tube and add 1 ml isopropanol, and 2 Âµl glycogen. Vortex thoroughly for 10 s.</li><li>Centrifuge at 13,000 <em>x g</em> at room temperature for 5 min and aspirate the supernatant. A white pellet should be visible at the bottom of the tube.</li><li>Add 1 ml of 70% ethanol to the tube.</li><li>Centrifuge at 13,000 <em>x g</em> at room temperature for 5 min and aspirate the supernatant.</li><li>Resuspend pellet in 20 Âµl Buffer EB. </li><li>Perform qPCR (using Bio-Rad SYBR Green Supermix; annealing temp: 60 Â°C) with 2 Âµl of undigested and digested gDNA using the primer sets P1-P2 (Input) and P1-P3 (undigested). Set up each qPCR reaction in triplicate.</li><li>Digestion efficiency is calculated from the mean Cq value of each reaction in the digested gDNA sample using the formula:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190313/20190313030723_4852.jpg" width="410"/><br/></li></ol></li></ol></li><li>Ligation<ol class="npwtwo"><li>Add 40 Âµl of 20% SDS (final concentration of 1.6%) to the digested chromatin. </li><li>Incubate for 20-25 min at 65 Â°C.</li><li>Transfer digested nuclei to a 50 ml Falcon tube.</li><li>Add 6.125 ml of 1.15x Ligation buffer (10x concentrated buffer is diluted in ddH<sub>2</sub>O).</li><li>Add 375 Âµl of 20% Triton X-100 (final concentration of 1%).</li><li>Incubate for 1 h at 37 Â°C while shaking gently.</li><li>Add 1 Âµl T4 DNA Ligase (100 U total) and incubate for 4 h at room temperature.<br/><em>Note: Can also perform the ligation reaction at 16 Â°C.</em> </li><li>Add 15 Âµl of 20 mg/ml Proteinase K (final 300 Î¼g).</li><li>Incubate at 65 Â°C overnight to reverse crosslinks.<br/><em>Note: We also de-crosslink at 56 Â°C. Any temperature within this range reverses protein crosslinks without affecting the DNA.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>DNA purification<ol class="npwtwo"><li>Add 10 Âµl of 10 mg/ml RNase A (final 100 Âµg).</li><li>Incubate for 30-45 min at 37 Â°C.</li><li>Add 7 ml phenol-chloroform and mix by vortexing (setting: 10).</li><li>Centrifuge for 5 min at 2,000 <em>x g</em> at 4 Â°C.</li><li>Collect the upper aqueous phase, transfer to a new tube, add 7 ml chloroform, vortex (setting: 10).</li><li>Centrifuge for 5 min at 2,000 <em>x g</em> at 4 Â°C.</li><li>Transfer supernatant into a 50 ml tube, add 25 ml isopropanol and 5 Âµl glycogen (10 mg/ml). </li><li>Mix and place at -80 Â°C for 1 h.</li><li>Centrifuge for 20 min at 4,200 <em>x g</em> at room temperature.</li><li>Remove supernatant and add 10 ml 70% ethanol.</li><li>Centrifuge for 15 min at 4,200 <em>x g</em> at 4 Â°C.</li><li>Remove supernatant and dry pellet at room temperature.</li><li>Dissolve DNA pellet in 200 Âµl of Buffer EB.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Secondary restriction digest<ol class="npwtwo"><li>Set up <em>Nla</em>III digest on all of the 3C DNA overnight at 37 Â°C with 25 Âµl of 10x Cutsmart buffer and 15 Âµl of <em>Nla</em>III.</li><li>Inactivate <em>Nla</em>III by incubating at 65 Â°C for 20 min.</li><li>Add 1x Phenol:Chloroform:Isoamyl alcohol (250 Âµl); vortex, spin (10,000 <em>x g</em>), collect supernatant.</li><li>Add 250 Âµl chloroform, vortex, spin (10,000 <em>x g</em>), collect supernatant.</li><li>Add 700 Âµl of isopropanol and 2 Âµl glycogen (10 mg/ml), vortex, spin (10,000 <em>x g</em> to precipitate), aspirate supernatant.</li><li>Add 1 ml of 70% ethanol, spin (10,000 <em>x g</em>), aspirate supernatant.</li><li>Resuspend in 1 ml of ultrapure water.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Circularization<ol class="npwtwo"><li>Set up ligation reaction overnight with 1.5 ml of 10x ligase buffer, 12.5 ml of water, 1 ml resuspended <em>Nla</em>III digested DNA, 1 Âµl T4 DNA ligase in a 50 ml Falcon tube.</li><li>Add 15 ml Phenol:Chloroform:Isoamyl alcohol, vortex, spin (4,200 <em>x g</em>), collect supernatant.</li><li>Add 15 ml Chloroform, vortex, spin (4,200 <em>x g</em>), collect supernatant.</li><li>Add 25 ml Isopropanol and 5 Âµl glycogen (10 mg/ml) to precipitate DNA. Spin at 4,200 <em>x g</em>.</li><li>Wash with 10 ml of 70% ethanol; spin at 4,200 <em>x g</em> and aspirate supernatant.</li><li>Air dry the pellet at room temperature for 20 min. </li><li>Resuspend in 200 Âµl Buffer EB.<br/></li><li>Purify with QIAGEN PCR purification kit and elute twice in 100 Âµl Buffer EB.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Inverse PCR<ol class="npwtwo"><li>Quantify V3C-seq product by NanoDrop and load 200-600 ng of template DNA per PCR.</li><li>Set up inverse PCR using Platinum Taq DNA Polymerase High Fidelity from Thermo Scientific as follows:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:200px;height:200px;"><tbody><tr><td>Template DNA<br/></td><td>2 Âµl<br/></td></tr><tr><td>10x PCR buffer<br/></td><td>2.5 Âµl<br/></td></tr><tr><td>MgSO<sub>4</sub><br/></td><td>1 Âµl</td></tr><tr><td>dNTP<br/></td><td>0.5 Âµl<br/></td></tr><tr><td>Primer 1<br/></td><td>0.5 Âµl<br/></td></tr><tr><td>Primer 2<br/></td><td>0.5 Âµl<br/></td></tr><tr><td>HIFI Taq<br/></td><td>0.1 Âµl</td></tr><tr><td>ddH<sub>2</sub>O<br/></td><td>17.9 Âµl<br/></td></tr></tbody></table></li><li>Inverse PCR reaction conditions<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190313/20190313185523_0293.jpg" width="240"/><br/></li><li>Dilute PCR product 1:100 in TE buffer.</li><li>Set up nested inverse PCR reactions with diluted PCR product as template.</li><li>Purify nested inverse PCR product with PCR purification kit (QIAGEN) and elute in 50 Âµl Buffer EB.</li><li>NanoDrop inverse PCR product and generate sequencing library with NEBNext Ultra II DNA Library Prep Kit for Illumina.</li><li>Pool up to 12 samples per lane and sequence samples by high throughput sequencing using the Illumina NextSeq 500 platform.<br/></li></ol></li></ol><p class="pptt" id="biaoti27284">### Data analysis</p><ol class="npwtwok"><li>Align the sequencing reads using Bowtie 2<ol class="npwthree"><li>Module load Bowtie2/Bowtie2-2.2.9</li><li>Bowtie 2 --trim5 20 --very-sensitive -x mm10 -S mvm_1.sam mvm_1.fastq</li></ol></li><li>Convert .sam files into .bam files and sort .bam files using SAMtools<ol class="npwthree"><li>Module load SAMtools/SAMtools-1.3.1</li><li>SAMtools view -b -S -o mvm_1.sam &gt; aligned_mvm_1.bam</li><li>SAMtools sort -o aligned_mvm_1.bam sorted_aligned_mvm_1.bam</li></ol></li><li>Compute histograms using BEDTools<ol class="npwthree"><li>Module load BEDTools/BEDTools-2.26.0</li><li>GenomeCoverageBed -ibam sorted_aligned_mvm_1.bam -bg -trackline -split -g ... &gt; mvm_1.bedgraph</li></ol></li><li>Upload bedgraph file to UCSC genome browser<br/>In the âedit configurationâ field, type: track type=bedGraph name="mvm_1" description="mvm_1" priority=20<br/>A representative histogram of MVM interaction sites on mouse chromosome 17 is shown in Figure 2B (top panel). These sequencing tracks can be compared with ChIP-seq data, such as the DNA damage marker gamma-H2AX during MVM infection of A9 cells (shown in the bottom panel of Figure 2B).</li><li>The MVM interaction sites identified by V3C-seq can be validated using focused Taqman-based 3C-qPCR assays, such as the validation of MVM interaction with the 5â end of mouse chromosome 19 (Figure 2A, left column). In order to account for background interactions, these assays should include controls where the crosslinks are reversed prior to ligation (Figure 2A, middle column) and unligated controls (Figure 2A, right column). More information on primers and probes used for the 3C-qPCR assays can be found in Majumder <em>et al.</em>, 2018.<br/><br/><img alt="" class="layerphoto" height="102" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190313/20190313025629_2759.jpg" width="150"/><br/><strong>Figure 2. Quantification of MVM-host hybrid DNA formation using focused qPCR and high-throughput sequencing assays.</strong> A. Taqman qPCR analysis of MVM interaction with a cellular site on mouse chromosome 19, with the probe sequence complementary to the MVM genome. This control experiment shows that reversing the crosslinks prior to ligation (middle) or excluding the intramolecular ligation step (right) prevents the formation of virus-host hybrid junctions. The figure has been replotted from experiments published in Majumder <em>et al.</em> (2018). B. Representative histogram on chromosome 17 of V3C-seq in MVM infected murine A9 fibroblasts with the viewpoint on the 5â end of the MVM genome (top). The V3C-seq data can be compared with other high-throughput sequencing experiments. As an example, ChIP-seq for the DNA damage associated chromatin mark gamma-H2AX in MVM infected murine A9 cells at 20 h post infection is shown on the track below. For more details, refer to Majumder <em>et al.</em> (2018) and publicly available V3C-seq and ChIP-seq data in the GEO repository (Accession number GSE112957).<br/></li></ol><p class="pptt" id="biaoti27285">### Recipes</p><ol class="npwtwok"><li>TE Buffer<br/>10 mM Tris-HCl, pH 8.0<br/>1 mM EDTA, pH 8.0</li><li>Complete DMEM media (500 ml)<br/>DMEM (with L-glut)<br/>5% FCS<br/>500 Âµl Gentamicin (50 mg/ml)</li><li>3C Lysis buffer<br/>10 mM Tris-HCl, pH 7.5<br/>10 mM NaCl<br/>0.2% NP40</li><li>Earleâs 10x Salts (1L)<br/>2.645 g Calcium Chloride<br/>4 g Potassium Chloride<br/>2 g Magnesium Sulfate<br/>68 g Sodium Chloride<br/>1.22 g Sodium Dihydrogen Phosphate<br/>10 g Glucose<br/>0.1 g Phenol red</li><li>Isoleucine minus media<br/>Add in order ~350 ml ddH<sub>2</sub>O<br/>50 ml Earleâs 10x salts<br/>0.5 ml 1,000x L-arginine<br/>0.5 ml 1,000x L-cysteine<br/>0.5 ml 1,000x L-histidine<br/>0.5 ml 1,000x L-leucine<br/>0.5 ml 1,000x L-lysine<br/>0.5 ml 1,000x L-methionine<br/>0.5 ml 1,000x L-phenylalanine<br/>0.5 ml 1,000x L-threonine<br/>0.5 ml 1,000x L-tryptophan<br/>0.5 ml 1,000x L-tyrosine<br/>0.5 ml 1,000x L-valine<br/>0.5 ml 100x vitamins<br/>600 Âµl Gentamicin<br/>5 ml 1 M HEPES, pH 7.5<br/>Adjust pH with 7.5% sodium bicarbonate (until red). Bring up to 475 ml with ddH<sub>2</sub>O. Filter (0.22 Î¼m) sterilize and dialyze against 5% serum. Store at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti27286">### Acknowledgments</p><p class="ppzz">Work in the Pintel Laboratory is supported by NIH grants AI046458 and AI116595 to DJP. KM is supported by a Ruth L Kirschstein Postdoctoral Individual National Research Service Award (F32) AI131468. This procedure was modified from published 4C and 3C-seq studies (Dekker <em>et al.</em>, 2013; Denker and de Laat, 2016). Detailed procedures are published in Majumder <em>et al.</em> (2018) and sequencing data have been deposited in the GEO database under the accession number GSE112957.</p><p class="pptt" id="biaoti27287">### Competing interests</p><p class="ppzz">The authors declare no competing financial interest.</p><p class="pptt" id="biaoti27288">### References</p><ol class="npwtwok"><li>Dekker, J., Marti-Renom, M. A. and Mirny, L. A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23657480" target="_blank">Exploring the three-dimensional organization of genomes: interpreting chromatin interaction data.</a> <em>Nat Rev Genet</em> 14(6): 390-403.</li><li>Denker, A. and de Laat, W. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27340173" target="_blank">The second decade of 3C technologies: detailed insights into nuclear organization.</a> <em>Genes Dev</em> 30(12): 1357-1382.</li><li>Langmead, B. and Salzberg, S. L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" target="_blank">Fast gapped-read alignment with Bowtie 2.</a> <em>Nat Methods</em> 9(4): 357-359.</li><li>Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R. and Genome Project Data Processing Subgroup. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" target="_blank">The sequence alignment/map format and SAMtools.</a> <em>Bioinformatics</em> 25(16): 2078-2079.</li><li>Majumder, K., Wang, J., Boftsi, M., Fuller, M. S., Rede, J. E., Joshi, T. and Pintel, D. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30028293" target="_blank">Parvovirus minute virus of mice interacts with sites of cellular DNA damage to establish and amplify its lytic infection.</a> <em>Elife</em> 7: e37750.</li><li>Quinlan, A. R. and Hall, I. M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" target="_blank">BEDTools: a flexible suite of utilities for comparing genomic features.</a> <em>Bioinformatics</em> 26(6): 841-842.</li><li>van de Werken, H. J., Landan, G., Holwerda, S. J., Hoichman, M., Klous, P., Chachik, R., Splinter, E., Valdes-Quezada, C., Oz, Y., Bouwman, B. A., Verstegen, M. J., de Wit, E., Tanay, A. and de Laat, W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22961246" target="_blank">Robust 4C-seq data analysis to screen for regulatory DNA interactions.</a> <em>Nat Methods</em> 9(10): 969-972.<br/></li></ol>