<body>
            <h1>Delivering "Chromatic Bacteria" Fluorescent Protein Tags to Proteobacteria Using Conjugation</h1>
            <p><strong>Authors:</strong> Rudolf O Schlechter, Mitja NP Remus-Emsermann</p>
            <h2>Abstract</h2>
            <p>Recently, we published a large and versatile set of plasmids, the chromatic bacteria toolbox, to deliver eight different fluorescent protein genes and four combinations of antibiotic resistance genes to Gram-negative bacteria. Fluorescent tags are important tools for single-cell microbiology, synthetic community studies, biofilm, and host-microbe interaction studies. Using conjugation helper strain <em>E. coli</em> S17-1 as a donor, we show how plasmid conjugation can be used to deliver broad host range plasmids, Tn<em>5</em> transposons delivery plasmids, and Tn<em>7</em> transposon delivery plasmids into species belonging to the Proteobacteria. To that end, donor and recipient bacteria are grown under standard growth conditions before they are mixed and incubated under non-selective conditions. Then, transconjugants or exconjugant recipients are selected on selective media. Mutant colonies are screened using a combination of tools to ensure that the desired plasmids or transposons are present and that the colonies are not containing any surviving donors. Through conjugation, a wide range of Gram-negative bacteria can be modified without prior, often time-consuming, establishment of competent cell and electroporation procedures that need to be adjusted for every individual strain. The here presented protocol is not exclusive for the delivery of Chromatic bacteria plasmids and transposons, but can also be used to deliver other mobilizable plasmids to bacterial recipients.</p>
            <p><strong>Keywords:</strong> Fluorescent proteins, GFP, Plasmid, Tn<em>5</em> transposon, Tn<em>7</em> transposon, Conjugation, Mobilisable plasmids</p>
            <h2>Background</h2><p>Fluorescent proteins have become an important tool for the study of bacteria, <em>e.g.</em>, in biofilms, gene expression studies, or host-microbe interactions (Tomlin <em>et al.</em>, 2004; Monier and Lindow, 2005; Ma and Bryers, 2010; Remus-Emsermann <em>et al.</em>, 2011; Schada von Borzyskowski <em>et al.</em>, 2015; Remus-Emsermann <em>et al.</em>, 2016a; Remus-Emsermann and Schlechter, 2018; Schlechter <em>et al.</em>, 2018).<br/><br/>In a recently published study, we have described the construction of 96 plasmids, each different in fluorescent protein gene, antibiotic resistance marker combination, or plasmid backbone. The complete collection offers eight different fluorescent protein genes, each with its own unique excitation and emission spectrum, four different antibiotic resistance marker combinations, a broad host range plasmid, or delivery plasmids based on two different transposon systems, Tn<em>5</em> and Tn<em>7</em> (Schlechter <em>et al.</em>, 2018).<br/><br/>The broad-host plasmid backbone and the two transposon systems have different advantages and disadvantages: 1) The plasmid usually exists in several copies in the recipient cell, which results in multiple gene copies and thereby often higher fluorescent signals. However, plasmids may be lost more easily, leading to subpopulations of cells that might lose their ability to express fluorescent proteins (Bahl <em>et al.</em>, 2004). 2) The Tn<em>5</em> transposon inserts into Proteobacteria with high efficiency and, as it integrates into the bacterial genome, the chance of losing the fluorescent protein tag is minimal. However, the Tn<em>5</em> transposon randomly inserts DNA into the genome, thereby potentially disrupting genes that are important for bacterial fitness (de Lorenzo <em>et al.</em>, 1990). 3) The Tn<em>7</em> transposon integrates into the genome of the recipient bacterium in a site-specific manner, where no genes are disrupted and thereby should have minor impact on the bacterial fitness. However, the Tn<em>7</em> transposon only inserts in a narrow range of bacteria, <em>e.g.</em>, most Î³-Proteobacteria, some Î±-Proteobacteria, and Î²-Proteobacteria. Outside of the Î³-Proteobacteria, its insertion frequency may be low (McKenzie and Craig, 2006; Schlechter <em>et al.</em>, 2018).<br/><br/>In this protocol, we describe the procedure to equip a bacterial strain with fluorescent labels using <em>Pantoea eucalypti</em> 299R (Remus-Emsermann <em>et al.</em>, 2013; Tecon and Leveau, 2016) as an example recipient strain. <em>P. eucalypti</em> 299R is a competent recipient of all three different kinds of delivery systems, the broad host range plasmid and the two transposon systems. We could show that the broad-host plasmid as well as Tn<em>5</em> and Tn<em>7</em> transposon-based delivery of fluorescent proteins is feasible in this recipient strain (Schlechter <em>et al.</em>, 2018). Other Enterobacteriaceae, such as <em>E. coli</em>, <em>Erwinia amylovora</em> and <em>Pantoea vagans</em> were also successfully tested with all three systems as well as Pseudomonas strains such as <em>Pseudomonas citronellolis</em> P3B5 (Remus-Emsermann <em>et al.</em>, 2016b; Schlechter <em>et al.</em>, 2018). Additionally, bacteria from other taxa, such as the Î±-Proteobacterial genera <em>Sphingomonas</em> and <em>Methylobacterium</em> were modified at least with the Tn<em>5</em> transposon delivery system and in some cases with the broad-host-range plasmids and Tn<em>7</em> transposons.<br/><br/>The here presented protocol is based on bacterial mating using the <em>E. coli</em> S17-1 helper strain, which is able to transfer mobilizable plasmids to recipients. Conjugation is a promiscuous process that crosses strain, genus, and phylum level and even works across domains of life (Heinemann and Sprague, 1989; Waters, 2001). Transformation methods such as heat-shock and electroporation are often used when transferring genetic material into laboratory model hosts, however, newly isolated strains are usually recalcitrant to these methods and the establishment of transformation protocols is often tedious and time-consuming. By relying on conjugation, transformation can be avoided and can be as simple as scraping recipient bacterial colonies from an agar plate and mixing them with the donor bacterium.<br/></p>
        </body>

<p class="pptt" id="biaoti27300">##  Materials and Reagents</p><ol class="npwtwok"><li>Autoclaved, untreated, wooden toothpicks (local supermarket of choice)<br/></li><li>Pipette tips, 100-1,000 Î¼l</li><li>Pipette tips, 1-200 Î¼l</li><li>Pipette tips, 0.1-10 Î¼l<br/></li><li>1.5 ml Eppendorf tubes (Corning, Axygen<sup>Â®</sup>, catalog number: MCT-150-C)<br/></li><li>0.2 ml PCR tubes (Merck, Corning<sup>Â®</sup>, catalog number: CLS6531-960EA)<br/></li><li>Parafilm (Bemis, catalog number: PM996)<br/></li><li>Petri dishes 90 mm diameter (Merck, catalog number: P5731-500EA)<br/></li><li>Disposable cuvettes (Merck, catalog number: Z330418-1PAK)<br/></li><li>Disposable loops (Citotest Labware Manufacturing Co., catalog number: 2121-0001)<br/></li><li>Disposable spreaders (VWR, catalog number: 612-1560)<br/></li><li>15 ml centrifuge tubes (Electron Microscopy Sciences, catalog number: 64760-01)<br/></li><li>0.22 Âµm membrane filter (Merck, catalog number: GSTF02500)<br/></li><li>Donor strain <em>E. coli</em> S17-1 equipped with plasmids listed in Table 1 (strains can be acquired from Addgene, <a href="https://www.addgene.org/browse/article/28196767/" target="_blank">https://www.addgene.org/browse/article/28196767/</a>) <br/></li><li>Recipient strain (<em>e.g.</em>, <em>Pantoea eucalypti</em> 299R)<br/></li><li>Plasmids (Table 1) encoding for fluorescent proteins (Table 2)<br/><br/><strong>Table 1. Plasmids</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327205125_7249.jpg" width="500"/><br/>Source of all plasmids: Schlechter <em>et al.</em>, 2018, plasmids can be obtained at Addgene plasmid number 118484 to 118579. X is a placeholder for either 0, 1, 2, 3, 4, 5, 6, 7 representing mTagBFP2, mTurquoise2, sGFP2, sYFP2, mOrange2, mScarlet-I, mCardinal, or mClover3 encoding versions of the plasmids, respectively. Kan<sup>R</sup> = Kanamycin resistance; Cam<sup>R</sup> = Chloramphenicol resistance; Gent<sup>R</sup> = Gentamicin resistance; Tet<sup>R</sup> = Tetracycline resistance; Amp<sup>R</sup> = Ampicillin resistance.<br/><br/><strong>Table 2. Fluorescent protein properties*</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327205210_7093.jpg" width="550"/><br/><sup>*</sup>A more detailed spectrum figure can be found in Schlechter <em>et al.</em>, 2018<br/><br/></li><li>Lysogeny broth (Merck, catalog number: 110285)</li><li>Agar No. 1 (Oxoid, catalog number: LP0011)<br/></li><li>Tris free base (Sigma, catalog number: T6791-500G)<br/></li><li>Glacial acetic acid (Sigma, catalog number: 695092-500ML)<br/></li><li>Glycerol (Sigma, catalog number: G6279-500ML)<br/></li><li>Succinate (Sigma, catalog number: S9637-500G)<br/></li><li>DMSO (Sigma, catalog number: D8418-100ML)<br/></li><li>dNTPs (Thermo Fisher, catalog number: N8080260)<br/></li><li>Multi-purpose HyAgarose<sup>TM</sup> LE Agarose (HydraGene, catalog number: R9012LE-500g)<br/></li><li>Phusion polymerase (Thermo Fisher, catalog number: F530S)<br/></li><li>Oligonucleotides, synthesis amount 0.025 Âµmol, purified by desalting (see Table 3, Macrogen)<br/><br/><strong>Table 3. Primers</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327205307_9796.jpg" width="620"/><br/><br/></li><li>Nucleic acid stain RedSafe (Intron, catalog number: 21141)<br/></li><li>Ampicillin (Duchefa Biochemie, catalog number: A0104)<br/></li><li>Gentamicin (Duchefa Biochemie, catalog number: G0124)<br/></li><li>Kanamycin (Duchefa Biochemie, catalog number: K0126)<br/></li><li>Chloramphenicol (Duchefa Biochemie, catalog number: C0113)<br/></li><li>Tetracycline (Duchefa Biochemie, catalog number: T0150)<br/></li><li>Ethanol (Carl Roth, catalog number: T171.5)<br/></li><li>K<sub>2</sub>HPO<sub>4</sub> (Sigma, catalog number: 795496-100G)<br/></li><li>NaH<sub>2</sub>PO<sub>4</sub>Â·2H<sub>2</sub>O (Sigma, catalog number: 71505-250G)<br/></li><li>NH<sub>4</sub>Cl (Sigma, catalog number: A9434-500G)<br/></li><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma, catalog number: 63138-250G)<br/></li><li>Na<sub>2</sub>EDTAÂ·2H<sub>2</sub>O (Sigma, catalog number: E6635-500G)<br/></li><li>ZnSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma, catalog number: Z0251-100G)<br/></li><li>CoCl<sub>2</sub>Â·6H<sub>2</sub>O (Sigma, catalog number: C8661-25G)<br/></li><li>MnCl<sub>2</sub> (Sigma, catalog number: 244589-10G)<br/></li><li>H<sub>3</sub>BO<sub>3</sub> (Sigma, catalog number: 202878-10G)<br/></li><li>CaCl<sub>2</sub> (Sigma, catalog number: C5670-500G)<br/></li><li>Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O (Sigma, catalog number: 480967-25G)<br/></li><li>FeSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma, catalog number: F8263-500G)<br/></li><li>CuSO<sub>4</sub>Â·5H<sub>2</sub>O (Sigma, catalog number: 469130-50G)<br/></li><li>NaCl (Sigma, catalog number: S7653-1KG)<br/></li><li>KCl (Sigma, catalog number: P9333-500G)<br/></li><li>Na<sub>2</sub>HPO<sub>4</sub> (Sigma, catalog number: S7907-500G)<br/></li><li>KH<sub>2</sub>PO<sub>4</sub> (Sigma, catalog number: P5655-500G)<br/></li><li>NaOH (Sigma, catalog number: S8045-500G)<br/></li><li>Demineralized water<br/></li><li>Lysogeny broth (LB) (see Recipes)<br/></li><li>Lysogeny broth agar (LB agar) (see Recipes)<br/></li><li>Minimal medium (see Recipes)<ol class="npwthree"><li>10x ammonium solution </li><li>10x phosphate buffer solution </li><li>10x sulfate solution </li><li>1,000x trace elements solution </li><li>20% succinate solution </li><li>3% agar </li></ol></li><li>1x Phosphate buffered saline (1x PBS) (see Recipes)<br/></li><li>Antibiotic stock solutions and working concentrations (see Recipes)<ol class="npwthree"><li>Ampicillin 100 mg ml<sup>-1</sup> stock solution</li><li>Gentamicin 15 mg ml<sup>-1</sup> stock solution</li><li>Kanamycin 50 mg ml<sup>-1</sup> stock solution</li><li>Chloramphenicol 15 mg ml<sup>-1</sup> stock solution</li><li>Tetracycline 5 mg ml<sup>-1</sup> stock solution</li></ol></li><li>TAE buffer (see Recipes)<ol class="npwthree"><li>0.5 M EDTA stock solution </li><li>50x TAE stock solution</li><li>1x TAE working solution </li></ol></li></ol><p class="pptt" id="biaoti27301">###  Equipment</p><ol class="npwtwok"><li>Forceps</li><li>Pipettes (Eppendorf, P1000, catalog number: 3120000062; P200, catalog number: 3120000054; P10, catalog number: 3120000020)</li><li>Erlenmeyer flasks 125 ml (<em>e.g.</em>, Merck, catalog number: CLS4980125-12EA)</li><li>Automatic micropipettes (Eppendorf or Gilson)</li><li>Metal tweezers (VWR, catalog number: 232-0092)</li><li>Bunsen burner (Eisco, catalog number: CH0087A)</li><li>Vortex (Scientific Industries, catalog number: SI-0236)</li><li>Autoclave (Heidolph, catalog number: 023210745CA)</li><li>-20 Â°C Freezer (Fisher and Paykel, catalog number: E388LXFD1)</li><li>Sterile bench (Labconco, catalog number: 302389100)</li><li>Microcentrifuge for 1.5 ml centrifuge tubes with max speed &gt; 6,000 <em>x g</em> (Eppendorf, catalog number: 5418000017)</li><li>Centrifuge for 15 ml centrifuge tubes with max speed &gt; 6,000 <em>x g</em> (Eppendorf, catalog number: 5805000327)</li><li>Shaking incubators 30 Â°C and 37 Â°C (Infors, catalog number: Multitron Pro)</li><li>Incubators 30 Â°C and 37 Â°C (Binder, catalog number: BD 56)</li><li>Spectrophotometer (Biochrom WPA CO8000 Cell Density meter, catalog number: 80-3000-45)</li><li>PCR thermocycler (Eppendorf, catalog number: 6311000010)</li><li>Horizontal agarose gel electrophoresis setup including running chamber and power supply (VWR International, catalog number: 76196-448)</li><li>Blue light or UV transilluminator (optional: camera for documentation) (Fisher Scientific, catalog number: UV95042001)</li><li>Fluorescent microscope with appropriate filter sets for the excitation and emission spectra of the fluorescent protein tags and equipped with a 5x objective with a long working distance (Zeiss Axio Imager.M1 equipped with Objective A-Plan 5x/0.12 M27 [Zeiss, catalog number: 421030-9900-000])<br/></li></ol><p class="pptt" id="biaoti27302">###  Procedure</p><p class="ppzz">An overview of the workflow is shown in Figure 1. <br/><br/><img alt="" class="layerphoto" height="137" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327210020_8809.jpg" width="100"/><br/><strong>Figure 1. Flowsheet of protocol</strong><br/><br/><strong>Prepare or have ready media and stock solutions for the conjugation</strong> </p><ol class="npwtwo"><li>Prepare 2 lysogeny broth (LB) agar plates without antibiotics.</li><li>Prepare 1 LB plate containing kanamycin when working with a pMRE1XX plasmid or 1 LB plate containing ampicillin and chloramphenicol when working with transposon bearing plasmids (Ampicillin will select for the plasmid and chloramphenicol for the transposon).</li><li>Prepare 3 ml and 50 ml LB broth without antibiotics.</li><li>Prepare 3 ml and 50 ml LB broth containing kanamycin when working with a pMRE1XX plasmid or 3 ml and 50 ml LB broth containing ampicillin and chloramphenicol when working with transposon bearing plasmids. </li><li>Prepare ~10 minimal media plates containing succinate as the sole carbon source and antibiotics selecting for the plasmids or transposons as explained in Table 1. In case you are selecting for transposon insertions, <u>do not include ampicillin</u>, since it selects for the plasmid, not the transposon. Store at 4 Â°C and in the dark if the light-sensitive tetracycline is used in the plates.<br/><em>Note: Select a carbon source for minimal media plates that can be metabolized/utilized by the recipient strain.</em> </li><li>Prepare 100 ml sterile 1x PBS.</li></ol><br/><strong>Day 1</strong> <ol class="npwtwo"><li>Streak donor strain <em>E. coli</em> S17-1 containing pMRE1XX, pMRE-Tn<em>5</em>-1XX, or pMRE-Tn<em>7</em>-1XX on LB agar plates containing appropriate antibiotics from a glycerol stock using a disposable loop and incubate overnight at 37 Â°C or 30 Â°C when working with pMRE-Tn<em>7</em>-1XX plasmids. <br/><em>Note: pMRE-Tn</em><em>7</em><em>-1XX plasmids do not replicate above 32 Â°C. Thus, to maintain it in the bacterial culture, it must be grown at lower temperatures.</em> </li><li>Streak recipient strain <em>P. eucalypti</em> 299R on an LB agar plate from a glycerol stock using a disposable loop and incubate overnight at 30 Â°C.</li></ol><br/><strong>Day 2</strong> <ol class="npwtwo"><li>Inoculate 3 ml LB broth containing the same antibiotics as before with an individual colony of donor <em>E. coli</em> S17-1 containing pMRE1XX, pMRE-Tn<em>5</em>-1XX, or pMRE-Tn<em>7</em>-1XX using a sterile toothpick and ethanol flamed forceps. Incubate overnight at 200 rounds per minute (rpm) and 37 Â°C or 30 Â°C when working with pMRE-Tn<em>7</em>-1XX plasmids.</li><li>Inoculate 3 ml LB broth without antibiotics with an individual colony of <em>P. eucalypti</em> 299R using a sterile toothpick and ethanol flamed forceps. Incubate overnight at 30 Â°C and 200 rpm.</li></ol><br/><strong>Day 3</strong> <ol class="npwtwo"><li>Inoculate 50 ml LB broth in a 125 ml Erlenmeyer flask containing the same antibiotics as before with 500 Âµl overnight culture <em>E. coli</em> S17-1 containing pMRE1XX, pMRE-Tn<em>5</em>-1XX, or pMRE-Tn<em>7</em>-1XX using a sterile pipette tip and a P1000 pipette. Incubate at 200 rpm and 37 Â°C or 30 Â°C when working with pMRE-Tn<em>7</em>-1XX plasmids. Grow to an optical density at 600 nm (OD<sub>600nm</sub>) of ~0.5, which will take between 1.5 h and 3 h depending on the incubation temperature. <br/><em>Note: Measure OD</em><sub><em>600nm</em></sub><em> regularly using disposable cuvettes and a spectrophotometer and note down the final OD</em><sub><em>600nm</em></sub><em>.</em> </li><li>Inoculate 50 ml LB broth in a 125 ml Erlenmeyer flask containing no antibiotics with 500 Âµl overnight culture <em>P. eucalypti</em> 299R using a sterile pipette tip and a P1000 pipette. Incubate at 200 rpm and 30 Â°C. Grow until <em>P. eucalypti</em> 299R has reached late exponential/early stationary phase, which will take ~3-4 h (OD<sub>600nm</sub> of 0.7 or above). Measure OD<sub>600nm</sub> using disposable cuvettes and a spectrophotometer. Alternatively, recipient strain <em>P. eucalypti</em> 299R can be scraped off directly from the plate using a sterile disposable loop and resuspended in a 1.5 ml microcentrifuge tube containing 1 ml sterile 1x PBS by gentle vortexing. <br/><em>Note: </em><em>P. eucalypti</em><em> 299R is a fast growing strain with a similar growth rate to </em><em>E. coli</em><em> S17-1. For others, such as slow-growing strains, the inoculation volume or time of inoculation may vary accordingly. For other strains, growth rate, exponential growth phase and early/late stationary phase will have to be determined.</em> </li><li>Harvest 10 ml of both cultures by centrifugation at 2,000 <em>x g</em> for 5 min in a centrifuge. Discard the supernatants, resuspend each culture in 5 ml sterile 1x PBS by gentle vortexing and measure OD<sub>600nm</sub>.<br/><em>Note: Vortex gently for less than 5 s to avoid shearing of the donor which could impact conjugation efficiency. Mixing by inversion is also recommended.</em> </li><li>Mix the cultures to reach similar amounts of donor and recipient cells.<br/><em>Example: If the 10 ml </em><em>E. coli</em><em> S17-1 culture had an OD</em><sub><em>600nm</em></sub><em> of 0.5, it should be concentrated to OD</em><sub><em>600nm</em></sub><em> of 1. If the 10 ml </em><em>P. eucalypti</em><em> 299R culture had an OD</em><sub><em>600nm</em></sub><em> of 0.4, it should be concentrated to OD</em><sub><em>600nm</em></sub><em> of 0.8. The </em><em>P. eucalypti</em><em> 299R culture, therefore, contains only 80% of the cells that the </em><em>E. coli</em><em> S17-1 culture contains. Therefore, mix the 5 ml </em><em>P. eucalypti</em><em> 299R culture with 4 ml (80% of 5 ml) </em><em>E. coli</em><em> S17-1 culture.</em><br/><em> Note: For other strains, the number of CFU per OD</em><sub><em>600nm</em></sub><em> will have to be determined beforehand.</em> </li><li>Harvest the combined cultures by centrifugation at 2,000 <em>x g</em> for 5 min in a centrifuge, discard the supernatant and resuspend in 100 Âµl sterile 1x PBS using a vortex. <br/><em>Note: Vortex gently for less than 5 s to avoid shearing of donor pili which could impact conjugation efficiency. Mixing by inversion is also recommended.</em> </li><li>Dry an LB agar plate containing no antibiotics in a sterile bench for 15 min. In the sterile bench, dropspot the resuspended culture mix onto the agar using a sterile pipette tip and a 200 Âµl pipette. Keep the plate open and wait until all the liquid evaporates. Afterwards, close the plate and incubate overnight at 30 Â°C.</li></ol><br/><strong>Day 4</strong> <ol class="npwtwo"><li>Harvest the culture drop using a wire loop and transfer to a 1.5 ml microcentrifuge tube containing 1 ml sterile 1x PBS. Resuspend the bacteria using a vortex. Plate 10 Âµl, 100 Âµl and the rest of the bacterial suspensions on the minimal media plates containing antibiotics selecting for the used plasmid or transposon using a disposable or flame-sterilized spreader. Minimal media will select against the auxotrophic <em>E. coli</em> donor.</li><li>Incubate at 30 Â°C until visible colonies appear on the plates.</li></ol><br/><p class="ppzz"><strong>Days 5-8 </strong><br/><em>Note: Slow growing strains may appear weeks after the conjugation.</em></p><ol class="npwtwo"><li>Check morphology and fluorescence emission in emerging colonies. Fluorescence can often be seen using a transilluminator (<em>e.g.</em>, UV or Blue light) that is also used for agarose gel electrophoresis (<em>e.g.</em>, for most green, yellow, orange, and red emitting proteins see Figure 2). Otherwise, and additionally, it is useful to investigate emerging colonies with a fluorescent microscope equipped with a very low magnification and long working distance objectives. Use filters that have appropriate excitation and emission filters. Often colonies are made up of both, the donor and the recipient strain, for an example see Figure 3A. Select colonies that appear homogeneously fluorescent under the fluorescent microscope and streak them on minimal medium (as above) to individual colonies (Figure 3B). <br/><em>Note: A fun and beautiful example of a forced consortium. The recipient strain provides nutrients for the otherwise auxotroph </em><em>E. coli</em><em> and </em><em>E. coli</em><em> is providing antibiotic resistance to the recipient strain.</em> </li><li>Restreak the colonies at least three times using a disposable loop in the subsequent days on minimal media plates.<br/><br/><img alt="" class="layerphoto" height="112" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327210159_2991.jpg" width="145"/><br/><strong>Figure 2. Bacterial colonies expressing fluorescent proteins.</strong> A and B. Bacteria were streaked as crude streaks on agar plates. Expressed fluorescent proteins were visualized on a blue light gel reader. Green, yellow, orange, red and far-red fluorescent colonies were easily detectable on gel readers, blue and cyan fluorescent colonies were difficult to distinguish from non-fluorescent wild type colonies. C. Expressed fluorescent proteins were visualized using a UV light tray with attached gel documentation system. Fluorescent protein expressing colonies appear brighter than the wild type.<br/><br/><img alt="" class="layerphoto" height="106" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327210243_9844.jpg" width="266"/><br/><strong>Figure 3. Overlay of fluorescent and brightfield microscope channels.</strong> A. Example of unsuccessful conjugation that yielded colonies. The colonies are mixes of a <em>Sphingomonas</em> species and red fluorescent protein expressing <em>E. coli</em> S17-1. B. Clonal colony of <em>P. eucalypti</em> 299R::MRE-Tn<em>7</em>-145. Scale bars = 500 Âµm.</li></ol><br/><p class="ppzz"><strong>Day 9, or after subsequent subcultivations</strong><br/>At this stage, you will have to follow different procedures, depending on the delivery systems you choose:<br/><strong><em>pMRE1XX plasmids</em></strong> </p><ol class="npwtwo"><li>As final check, perform PCRs to test for <em>E. coli</em> contamination. For that, perform a crude DNA extraction from a freshly grown individual colony by picking a part of the colony with an autoclaved toothpick and dislodging the cells in a 0.2 ml PCR tube filled with 100 Âµl 0.05 M NaOH. </li><li>Boil the sample for 15 min at 95 Â°C in a PCR machine. Use 1 Âµl of crude DNA extract as template for a PCR using Phusion polymerase and primers FWD_<em>uidA</em> and REV_<em>uidA</em> following the recommendations of the manufacturer (for an example refer to Figure 4).<br/><em>Note: Presence of E. coli will yield a ~500 bp PCR product. To check for the presence of uidA in a bacterial strain, the sequence of the E. coli gene (</em><a href="https://www.ncbi.nlm.nih.gov/gene/946149" target="_blank"><em>https://www.ncbi.nlm.nih.gov/gene/946149</em></a><em>) can be blasted against the genome of the target strain.</em><br/><br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327210405_1053.jpg" width="170"/><br/><strong>Figure 4. Colony PCR of <em>uidA</em> for detection of <em>Escherichia coli</em>.</strong> <em>uidA</em> is an <em>E. coli</em> and Shigella sp. specific gene, thereby amplification of this gene fragment indicates mixed colony or false positive colony after restreak and colony isolation. Primers used were FWD_<em>uidA</em> and REV_<em>uidA</em> (Table 3) and yield a 500 bp product. Lane M: Molecular marker (HyperLadder<sup>TM</sup>, Bioline); Sample 1: <em>P. eucalypti</em> 299R::MRE-Tn<em>5</em>-145; Sample 2: <em>E. coli</em> (pMRE-Tn<em>5</em>-145); Sample 3: <em>P. eucalypti</em> 299R::MRE-Tn<em>7</em>-145; Sample 4: <em>E. coli</em> (pMRE-Tn<em>7</em>-145); C-: water control. In this example, samples #1 and #3 showed no detection of <em>E. coli</em>, while sample #2 and #4 showed presence of <em>uidA</em>.</li></ol><br/><strong><em>pMRE-Tn</em></strong><strong><em>5</em></strong><strong><em>-1XXâTn</em></strong><strong><em>5</em></strong><strong><em> transposon delivery plasmids</em></strong> <ol class="npwtwo"><li>Perform the <em>E. coli</em> contamination check as described above for pMRE1XX plasmids.</li><li>Additionally, use primers FWD_Tn<em>5</em>_gt, REV_Tn<em>5</em>_gt, FWD_Tn<em>5</em>/<em>7</em>_gt, and REV_Tn<em>5</em>/<em>7</em>_gt in a multiplex PCR. The primer combination tests of the presence of the delivery plasmid and for the presence of the fluorescent protein coding sequence in the sample (for an example refer to Figure 5). Samples containing the plasmid will yield two bands (~700 bp and ~400 bp), which suggests <em>E. coli</em> contamination. Samples containing only the fluorescent protein coding sequence yield only the larger band (~700 bp).<br/><br/><img alt="" class="layerphoto" height="101" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327210457_3139.jpg" width="136"/><br/><strong>Figure 5. Colony PCR to confirm Tn<em>5</em> insertions.</strong> Multiplex PCR result using primers FWD_Tn<em>5</em>_gt, REV_Tn<em>5</em>_gt, FWD_Tn<em>5</em>/<em>7</em>_gt, and REV_Tn<em>5</em>/<em>7</em>_gt. M: Molecular marker (HyperLadder, Bioline); Sample 1: <em>P. eucalypti</em> 299R::MRE-Tn<em>5</em>-145; Sample 2: <em>E. coli</em> (pMRE-Tn<em>5</em>-145); Sample 3: pMRE-Tn<em>5</em>-145; C-: water control.</li></ol><br/><strong><em>pMRE-Tn</em></strong><strong><em>7</em></strong><strong><em>-1XXâTn</em></strong><strong><em>7</em></strong><strong><em> transposon delivery plasmids</em></strong> <ol class="npwtwo"><li>Curing of plasmid (depending on the recipient strain)<br/>pMRE-Tn<em>7</em>-1XX plasmid series are conditional suicide plasmids in many, but not all, Enterobacteriaceae. <em>P. eucalypti</em> belongs to the Enterobacteriaceae, <em>i.e.</em>, the plasmid is able to replicate in <em>P. eucalypti</em> since the plasmids origin of replication pSC101 is functional in the strain. pSC101 is however heat instable and does not replicate at temperatures above 32 Â°C (McKenzie and Craig, 2006). Therefore, to cure the strain from the plasmid, it has to be grown at high temperatures. <br/><ol class="npwthree"><li>Inoculate a 5 ml LB broth culture using an individual fluorescent colony and grow it overnight at 37 Â°C. </li><li>Streak out the culture on an LB agar plate using an inoculation loop and incubate overnight at 37 Â°C. </li><li>Select 10 individual colonies and streak them each onto a new LB agar plate containing Amp and on an LB agar plate containing no antibiotic. Colonies that still contain the plasmid will be able to grow on ampicillin, while colonies where the transposition occurred will only grow on the LB agar plate without antibiotics.</li></ol></li><li>Perform the <em>E. coli</em> contamination check as described above for pMRE1XX plasmids.</li><li>Additionally, perform a PCR to identify the presence of Tn<em>7</em> insertion and/or Tn<em>7</em>-derived plasmid. Use FWD_Tn<em>5</em>/<em>7</em>_gt and REV_Tn<em>5</em>/<em>7</em>_g for the presence of the fluorescent protein coding sequence in the sample (~700 bp) and primers Tn<em>7</em>_gt1, Tn<em>7</em>_gt2, and Tn<em>7</em>_gt3 for the presence of the delivery plasmid (~1,000 bp). Tn<em>7</em>_gt1, Tn<em>7</em>_gt2, and Tn<em>7</em>_gt3 are needed because pMRE-Tn<em>7</em>-1XX plasmids were constructed by blunt-end cloning, so the insert orientation may vary between plasmids (Tn<em>7</em>_gt2, and Tn<em>7</em>_gt3 anneal to a flanking sequence of the insert in the plasmid backbone). For an example refer to Figure 6.<br/><br/><img alt="" class="layerphoto" height="107" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190403/20190403195736_2722.jpg" width="265"/><br/><strong>Figure 6. Colony PCRs to confirm Tn<em>7</em> insertions.</strong> PCR result using primers FWD_Tn<em>5</em>/<em>7</em>_gt and REV_Tn<em>5</em>/<em>7</em>_gt for the amplification of Tn<em>7</em>-delivered insert (MRE-Tn<em>7</em> insert), and PCR result using primers Tn<em>7</em>_gt1, Tn<em>7</em>_gt2, and Tn<em>7</em>_gt3 for amplification of a fragment of plasmid pMRE-Tn<em>7</em> backbone (MRE-Tn<em>7</em> backbone). M: Molecular marker. Sample 1: <em>P. eucalypti</em> 299R::MRE-Tn<em>7</em>-145; Lane 2: <em>E. coli</em> (pMRE-Tn<em>7</em>-145); Lane 3: pMRE-Tn<em>7</em>-145; C-: water control.</li></ol><p class="pptt" id="biaoti27303">###  Notes</p><ol class="npwtwok"><li>Depending on the recipient strain, addition of 100 Âµl drop spotted 1 M CaCl<sub>2</sub> onto the agar plate used to drop spot the bacterial mixture onto the CaCl<sub>2</sub> might increase conjugation efficiencies (Wang and Jin, 2014; McCully <em>et al.</em>, 2018).</li><li>Conjugation efficiency can be optimized by mixing donor and recipient strains in different ratios (<em>e.g.</em>, perform three conjugations by using a 1:3, 1:1, and 3:1 ratio of donor and recipient strain, respectively).<br/></li></ol><p class="pptt" id="biaoti27304">###  Recipes</p><ol class="npwtwok"><li>Lysogeny broth (LB)<br/>Add 25 g of lysogeny broth medium to 1 L of demineralized water<br/>Autoclave</li><li>Lysogeny broth agar (LB agar)<br/>Add 25 g of lysogeny broth medium and 15 g agar no. 1 to 1 L of demineralized water<br/>Autoclave</li><li>Minimal medium (Attwood and Harder, 1972)<br/><ol class="npwthree"><li>Prepare the following stock solutions:</li><ol class="npwfour"><li>10x ammonium solution <br/>Dissolve 16.2 g of NH<sub>4</sub>Cl in 1 L of demineralized water<br/>Autoclave</li><li>10x phosphate buffer solution<br/>Dissolve 15.9 g of K<sub>2</sub>HPO<sub>4</sub> and 18 g of NaH<sub>2</sub>PO<sub>4</sub>Â·2H<sub>2</sub>O in 1 L of demineralized water (pH should be 6.7 without adjustment)<br/>Autoclave</li><li>10x sulfate solution<br/>Dissolve 2 g of MgSO<sub>4</sub> in 1 L of demineralized water<br/>Filter sterilize using a 0.22 Âµm filter to avoid precipitation</li><li>1,000x trace elements solution (Harder and Veldkamp, 1968)<br/>Dissolve 15 g Na<sub>2</sub>EDTA<sub>2</sub>Â·2H<sub>2</sub>O, 4.5 g ZnSO<sub>4</sub>Â·7H<sub>2</sub>O, 3 g CoCl<sub>2</sub>Â·6H<sub>2</sub>O, 0.6 g MnCl<sub>2</sub>, 1 g H<sub>3</sub>BO<sub>3</sub>, 3 g CaCl<sub>2</sub>, 0.4 g Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O, 3 g FeSO<sub>4</sub>Â·7H<sub>2</sub>O, and 0.3 g CuSO<sub>4</sub>Â·5H<sub>2</sub>O in 1 L of demineralized water<br/>Filter sterilize using a 0.22 Âµm filter</li><li>20% succinate solution<br/>Dissolve 20 g succinate in 100 ml of demineralized water<br/>Filter sterilize using a 0.22 Âµm filter<br/><em>Note: </em><em>P. eucalypti</em><em> is able to grow on succinate as the sole carbon source, other strains may require other carbon source supplements to grow on minimal medium.</em> </li><li>3% agar<br/>Add 15 g Agar No. 1 in 500 ml of demineralized water<br/>Autoclave<br/><em>Note: 3% agar should be at a temperature below 60 Â°C, since higher temperatures might lead to salts precipitation. This also prevents boiling retardation. Pour quickly after thoroughly mixing the agar medium.</em> </li></ol><li>To prepare 1 L of minimal medium agar supplemented with 0.5% succinate as sole carbon source, combine 100 ml of 10x ammonium solution with 100 ml 10x phosphate buffer solution, 100 ml 10x sulphate solution, 1 ml of 1,000x trace element solution and 25 ml 20% succinate solution into 500 ml of molten 3% agar. Top up with sterile demineralized water to a final volume of 1 L</li></ol></li><li>1x Phosphate buffered saline (1x PBS) <br/>Dissolve 8 g NaCl, 0.24 g KCl, 1.42 g Na<sub>2</sub>HPO<sub>4</sub>, 0.24 g KH<sub>2</sub>PO<sub>4</sub> in 1 L of demineralized water and autoclave<br/>The buffer can alternatively be prepared as a 10x concentrated stock which is diluted to a 1x concentration with sterile water prior use</li><li>Antibiotic stock solutions and working concentrations<br/><img alt="" class="layerphoto" height="156" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190327/20190327210701_7336.jpg" width="600"/><br/><br/><ol class="npwthree"><li>Ampicillin stock solution<br/>Dissolve ampicillin in water to a concentration of 100 mg ml<sup>-1</sup> and filter sterilized using a 0.22 Âµm filter</li><li>Gentamicin stock solution<br/>Dissolve gentamicin in water to a concentration of 15 mg ml<sup>-1</sup> and filter sterilized using a 0.22 Âµm filter</li><li>Kanamycin stock solution<br/>Dissolve kanamycin in water to a concentration of 50 mg ml<sup>-1</sup> and filter sterilized using a 0.22 Âµm filter</li><li>Chloramphenicol stock solution<br/>Dissolve chloramphenicol in ethanol to a concentration of 15 mg ml<sup>-1</sup> </li><li>Tetracycline stock solution<br/>Dissolve tetracycline in ethanol to a concentration of 5 mg L-1</li></ol><em>Note: Store all antibiotic stock solutions in 1.5 ml aliquots at -20 Â°C.</em> </li><li>TAE buffer (40 mM Tris, 20 mM acetic acid, 1 mM EDTA)<br/><ol class="npwthree"><li>0.5 M EDTA stock solution<br/>Dissolve 93.05 g Na<sub>2</sub>EDTAÂ·2H<sub>2</sub>O in 400 ml of demineralized water and adjust the pH to 8.0 with NaOH<br/>Top up the solution to a final volume of 500 ml<br/><em>Note: EDTA will start dissolving when pH gets close to 8, use 10 M NaOH to adjust the pH.</em> </li><li>50x TAE stock solution<br/>Dissolve 242 g Tris base in 750 ml of demineralized water<br/>Add 57.1 ml of 100% glacial acetic acid and 100 ml of 0.5 M EDTA solution<br/>Adjust the volume to 1 L with demineralized water<br/>This solution can be stored at room temperature</li><li>1x TAE working solution<br/>Dilute 20 ml of 50x TAE stock solution in 980 ml of demineralized water</li></ol></li></ol><p class="pptt" id="biaoti27305">###  Acknowledgments</p><p class="ppzz">The protocol presented here was adapted from two previously published studies (Remus-Emsermann <em>et al.</em>, 2016a; Schlechter <em>et al.</em>, 2018) and majorly influenced by a study by McKenzie and Craig (2006). The authors thank MichaÅ Bernach, Hyunwoo Jun and the other authors of Schlechter <em>et al.</em>, 2018 for their help during the original study.<br/>  This work was funded by a seed grant of the Biomolecular Interaction Centre of the University of Canterbury to MR-E and the Royal Society of New Zealand Marsden Fast Start grant (UOC1704) to MR-E. RS was supported by an NZIDRS doctoral scholarship.</p><p class="pptt" id="biaoti27306">###  Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti27307">###  References</p><ol class="npwtwok"><li>Bahl, M. I., Sorensen, S. J. and Hestbjerg Hansen, L. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15019733" target="_blank">Quantification of plasmid loss in <em>Escherichia coli</em> cells by use of flow cytometry.</a> <em>FEMS Microbiol Lett</em> 232(1): 45-49.</li><li>de Lorenzo, V., Herrero, M., Jakubzik, U. and Timmis, K. N. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2172217" target="_blank">Mini-Tn<em>5</em> transposon derivatives for insertion mutagenesis, promoter probing, and chromosomal insertion of cloned DNA in gram-negative eubacteria.</a> <em>J Bacteriol</em> 172(11): 6568-6572.</li><li>Attwood, M. M. and Harder, W., (1972). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4561897" target="_blank">A rapid and specific enrichment procedure for <em>Hyphomicrobium</em> spp.</a> <em>Antonie van Leeuwenhoek</em> 38: 369â377.</li><li>Harder, W. and Veldkamp, H. (1968). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1365-2672.1968.tb00337.x" target="_blank">Physiology of an obligately psychrophilic marine <em>Pseudomonas</em> species.</a> <em>J Appl Microbiol</em> 31: 12-23.</li><li>Heinemann, J. A. and Sprague, G. F., Jr. (1989). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2666856" target="_blank">Bacterial conjugative plasmids mobilize DNA transfer between bacteria and yeast.</a> <em>Nature</em> 340(6230): 205-209.</li><li>Ma, H. and Bryers, J. D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20552252" target="_blank">Non-invasive method to quantify local bacterial concentrations in a mixed culture biofilm.</a> <em>J Ind Microbiol Biotechnol</em> 37(10): 1081-1089.</li><li>McCully, L. M., Bitzer, A. S., Seaton, S. C., Smith, L. M. and Silby, M. W. (2018). <a href="https://www.biorxiv.org/content/early/2018/04/06/296814" target="_blank">Social Motility: Interaction between two sessile soil bacteria leads to emergence of surface motility. </a><em>bioRxiv</em>: 296814.</li><li>McKenzie, G. J. and Craig, N. L. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16646962" target="_blank">Fast, easy and efficient: site-specific insertion of transgenes into enterobacterial chromosomes using Tn<em>7</em> without need for selection of the insertion event.</a> <em>BMC Microbiol</em> 6: 39.</li><li>Monier, J. M. and Lindow, S. E. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16151141" target="_blank">Spatial organization of dual-species bacterial aggregates on leaf surfaces.</a> <em>Appl Environ Microbiol</em> 71(9): 5484-5493.</li><li>Remus-Emsermann, M. N. P., de Oliveira, S., Schreiber, L. and Leveau, J. H. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22046169" target="_blank">Quantification of lateral heterogeneity in carbohydrate permeability of isolated plant leaf cuticles.</a> <em>Front Microbiol</em> 2: 197.</li><li>Remus-Emsermann, M. N. P., Gisler, P. and Drissner, D. (2016a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27445318" target="_blank">MiniTn<em>7</em>-transposon delivery vectors for inducible or constitutive fluorescent protein expression in <em>Enterobacteriaceae</em>.</a> <em>FEMS Microbiol Lett</em> 363(16).</li><li>Remus-Emsermann, M. N. P., Kim, E. B., Marco, M. L., Tecon, R. and Leveau, J. H. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23472227" target="_blank">Draft genome sequence of the phyllosphere model bacterium <em>Pantoea agglomerans</em> 299R.</a> <em>Genome Announc</em> 1(1).</li><li>Remus-Emsermann, M. N. P., Schmid, M., Gekenidis, M. T., Pelludat, C., Frey, J. E., Ahrens, C. H. and Drissner, D. (2016b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28300228" target="_blank">Complete genome sequence of <em>Pseudomonas citronellolis</em> P3B5, a candidate for microbial phyllo-remediation of hydrocarbon-contaminated sites.</a> <em>Stand Genomic Sci</em> 11: 75.</li><li>Remus-Emsermann, M. N. P. and Schlechter, R. O. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29504646" target="_blank">Phyllosphere microbiology: at the interface between microbial individuals and the plant host.</a> <em>New Phytol</em> 218(4): 1327-1333.</li><li>Schada von Borzyskowski, L., Remus-Emsermann, M., Weishaupt, R., Vorholt, J. A. and Erb, T. J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25105793" target="_blank">A set of versatile brick vectors and promoters for the assembly, expression, and integration of synthetic operons in <em>Methylobacterium</em> extorquens AM1 and other alphaproteobacteria.</a> <em>ACS Synth Biol</em> 4(4): 430-443.</li><li>Schlechter, R. O., Jun, H., Bernach, M., Oso, S., Boyd, E. F., MuÃ±oz-Lintz, D. A., Dobson, R. C., Remus, D. M. and Remus-Emsermann, M. (2018). <a href="https://www.biorxiv.org/content/early/2018/09/07/402172" target="_blank">Chromatic Bacteria - A broad host-range plasmid and chromosomal insertion toolbox for fluorescent protein expression in bacteria.</a> <em>bioRxiv</em>: 402172.</li><li>Tecon, R. and Leveau, J. H. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27534795" target="_blank">Symplasmata are a clonal, conditional, and reversible type of bacterial multicellularity.</a> <em>Sci Rep</em> 6: 31914.</li><li>Tomlin, K. L., Clark, S. R. and Ceri, H. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15003692" target="_blank">Green and red fluorescent protein vectors for use in biofilm studies of the intrinsically resistant <em>Burkholderia cepacia</em> complex.</a> <em>J Microbiol Methods</em> 57(1): 95-106.</li><li>Wang, X. K. and Jin, J. L. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24935520" target="_blank">Crucial factor for increasing the conjugation frequency in <em>Streptomyces netropsis</em> SD-07 and other strains.</a> <em>FEMS Microbiol Lett</em> 357(1): 99-103.</li><li>Waters, V. L. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11726922" target="_blank">Conjugation between bacterial and mammalian cells.</a> <em>Nat Genet</em> 29(4):375-6.<br/></li></ol>