<body>
            <h1>Identification of RNase-sensitive LINE-1 Ribonucleoprotein Interactions by Differential Affinity Immobilization</h1>
            <p><strong>Authors:</strong> Hua Jiang, Martin S. Taylor, Kelly R. Molloy, Ilya Altukhov, John LaCava</p>
            <h2>Abstract</h2>
            <p>Long Interspersed Nuclear Element-1 (LINE-1, L1) constitutes a family of autonomous, self-replicating genetic elements known as retrotransposons. Although most are inactive, copious L1 sequences populate the human genome. L1s proliferate in a âcopy-and-pasteâ fashion through an RNA intermediate; a full-length L1 transcript is ~6,000 nucleotides long and functions as a bicistronic mRNA that encodes and assembles <em>in cis</em> with two main polypeptides, ORF1p and ORF2p, forming a ribonucleoprotein (RNP); L1 RNPs also interact with a wide range of host factors in positive and negative regulatory capacities. The following protocol describes an approach to affinity enrich ectopically expressed L1 RNPs and, using RNases, release the fraction of protein that depends upon the presence of intact RNA for retention in the immobilized macromolecules.</p>
            <p><strong>Keywords:</strong> Retrotransposon, LINE-1, L1 RNP, Affinity capture, RNase treatment</p>
            <h2>Background</h2><p>In previous work, an inducible system for the ectopic expression of human L1 in human embryonic kidney (HEK) cells was established. This was achieved by (1) genomically integrating the reverse tetracycline-dependent transactivator (rtTA; Gossen <em>et al.</em>, 1995) from the Tet-On Advanced system (Clontech) into the cell line HEK-293T (Dai <em>et al.</em>, 2012); the Tet-On modified HEK cell line (referred to originally as HEK-293T-tet-on<sub>LD</sub>, and subsequently as HEK-293T<sub>LD</sub>) was then (2) transfected with episomal vectors (pCEP4 series, Invitrogen; Ostertag <em>et al.</em>, 2000; An <em>et al.</em>, 2009) containing a tetracycline response element promoter (pTRE-Tight, Clontech), which permitted (3) expression of the L1 cassette in response to the presence of doxycycline in the culture medium (Dai <em>et al.</em>, 2012; O'Donnell <em>et al.</em>, 2013). More recently, we have used this system to study the compositions of affinity isolated L1 RNPs (Taylor <em>et al.</em>, 2016)âincluding the interactomic consequences of treating them with a mixture of RNases A and T1 while immobilized on affinity medium, revealing which proteins become labile upon the treatment compared to a control (Taylor <em>et al.</em>, 2013 and 2018). The following protocol, detailed from Taylor <em>et al.</em> (2018), will enable users to obtain affinity enriched L1 RNPs suitable for the aforementioned RNase-sensitivity assay and can be easily adapted to include treatments with various enzymes not specifically described herein. A schematic diagram of the procedure is presented in Figure 1.<br/><br/><img alt="" class="layerphoto" height="102" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328001752_3180.jpg" width="520"/><br/><strong>Figure 1. Schematic diagram of this protocol.</strong> (i.) Cells will be grown in heavy and light SILAC media and will be transfected with a plasmid supporting (ii.) Inducible, ectopic L1 expression. From the moment the cells reach sufficient density to execute the transfection, (a.) it will be ~4 more days to harvest and freeze them; this is a natural stopping point in the protocol. These manipulations are described in Procedure A. (b.) An additional ~2 days will be needed to cryomill the cells to powder and carry out quality control checks by western blot and other protein assays, as well as to complete the peptide workup for mass spectrometry (MS), to determine the degree of heavy SILAC-label incorporation. The MS analysis could be completed as quickly as ~1 more day if done immediately. These manipulations are described in Procedure B. Pause until thorough SILAC labeling is confirmed. (iii.) Carry out affinity capture, followed by RNase and control treatments; these manipulations are described in Procedure C. (c.) Affinity capture and subsequent peptide workup (described in Procedure D) will require ~2 days. The time to complete (iv.) MS (Procedure D) and data analysis will depend on the number of samples but could be done within an additional ~3 days if the experiment is designed exactly as laid out in this protocol. Aside from the LC-MS/MS runs themselves, a substantial portion of the time is occupied by processing the RAW data files in MaxQuant; speed will vary according to the capabilities of the workstation.<br/><br/>To execute this protocol the user requires cryomilled, ORF2p-3xFLAG-tagged, LINE-1-expressing cells. HEK-293T<sub>LD</sub> cells and inducible tagged L1-expression plasmids may be obtained by request. With respect to the final readout of the below described procedure, RNase sensitivity can be assessed by numerous methods (Taylor <em>et al.</em>, 2013 and 2018). Among them, SDS-PAGE and general protein staining provide a way to broadly observe visibly changing bands in gels, and this approach is most useful if the identities of proteins within the bands are known (<em>e.g.</em>, labeled entities in Figure 4A). Western blotting provides a targeted and more sensitive way to assess the same, however, validated antibodies against the specific target(s) are required. In contrast, differential quantitative mass spectrometry (MS) provides a sensitive and hypothesis-free mode of analysis (<em>e.g.</em>, as in Figure 4D). Quantitative MS for these purposes may be conducted using a wide variety of methods, including the use of metabolic labeling, such as Stable Isotope Labeling by Amino acids in Cell culture (SILAC; Ong <em>et al.</em>, 2002), described below, or label-free approaches (such as those reviewed in Armean <em>et al.</em>, 2013).</p>
        </body>

<p class="pptt" id="biaoti27348">##  Materials and Reagents</p><p class="ppzz"><em>Note: Catalog numbers for specific suppliers we use are given for many of the reagents listed below; an equivalent quality reagent from an alternative supplier can typically be substituted with comparable results. If a supplier is not explicitly stated, we do not presume any particular source to be superior. Standard materials and reagents for mammalian cell culture are required and are not all explicitly listed below. For experimental procedures not related to cell culture (for which cell culture grade reagents should be used) molecular biology grade is suitable, unless an alternative is explicitly stated. LC-MS grade reagents are recommended for use in mass spectrometry sample workup. The protocols outlined involve work with liquid nitrogen; appropriate clothing and safety precautions are advised. The NuPAGE denaturing electrophoresis system (Thermo Fisher Scientific) is used in this protocol, but nearly any discontinuous or gradient, denaturing protein SDS-PAGE system will do. Simple solutions are described in this section; solutions containing multiple solutes are defined in the Recipes section, at bottom.</em> </p><br/><ol class="npwone"><li>Cell culture (Procedure A)<ol class="npwtwo"><li>Dialysis Tubing, 3,500 MWCO (Thermo Fisher Scientific, catalog number: 68035)</li><li>50 ml polypropylene conical tubes (Thermo Fisher Scientific, catalog number: 339652)</li><li>16 Ga needle</li><li>Luer-lock syringes (various sizes, 10-30 ml)</li><li>Syringe end caps (Bio-Rad Laboratories, catalog number: 7311660EDU)</li><li>Styrofoam box </li><li>Parafilm (Millipore Sigma, catalog number: P7793)</li><li>Serological pipettes (<em>e.g.</em>, from 1 ml to 50 ml)</li><li>Gloves<br/></li><li>pLD401 or other LINE-1 expression plasmid</li><li>Freestyle 293 Medium (Thermo Fisher Scientific, w/o lysine and arginineâcustom order)<br/></li><li>L-glutamine (200 mM; Thermo Fisher Scientific, catalog number: 25030081)</li><li>L-proline</li><li>L-arginine</li><li>L-lysine</li><li><sup>13</sup>C<sub>6</sub><sup>15</sup>N<sub>4</sub>-L-arginine (Arg10; Cambridge Isotope Laboratories, catalog number: CNLM-539-H-PK)</li><li><sup>13</sup>C<sub>6</sub><sup>15</sup>N<sub>2</sub>-L-lysine (Lys8; Cambridge Isotope Laboratories, catalog number: CNLM-291-H-PK)</li><li>Fetal bovine serum (FBS), tetracycline-free<br/><em>Note: Numerous suppliers can provide this. Many suppliers carry FBS products not labeled as tetracycline-free, but consulting the product specification sheet for a given lot may reveal that tetracycline has been tested for and found to be absent or below a ânegativeâ threshold (ideally in the single digit ng/ml range). Check with the supplier for the testing threshold and request a certificate of analysis for the lot to be purchased. Test a sample from that specific lot in case of doubt. In our hands, the performance of such lots has been identical to âcertifiedâ tetracycline-free FBS. </em> </li><li>Hybridoma SFM medium (Thermo Fisher Scientific, catalog number: 12045084)</li><li>Phosphate-buffered saline (PBS), pH 7.4 (Thermo Fisher Scientific, catalog number: 10010023)</li><li>PEI (polyethylenimine) Max (MW 40,000; Polysciences, catalog number: 24765-1)<br/><em>Note: Two grams is enough to transfect &gt; 600 L. To prepare 1 mg/ml PEI Max working solution: dissolve 100 mg PEI Max in 90 ml ddH</em><sub><em>2</em></sub><em>O, adjust pH to 7.0 using 1 M NaOH, adjust the volume to 100 ml, filter sterilize, and store at 4 Â°C. <u>Never freeze</u> PEI working solution; working solution can be used for up to 6 months if stored at 4 Â°C.</em> </li><li>Doxycycline (Millipore Sigma, catalog number: D9891)<br/><em>Note: Prepare a stock solution of 10 mg/ml in de-ionized water, sterile filter, and store at -20 Â°C, protected from light. Antibiotic stock solutions may lose activity over time; make them fresh at least every six months, but preferably every three (Okerman </em><em>et al.</em><em>, 2007). Thaw each stock only once; it may be stored a few days at 4 Â°C after thawing, and then should be discarded.</em> </li><li>Liquid nitrogen (LN<sub>2</sub>) and Dewar flask</li><li>Heavy SILAC medium (see Recipes)</li><li>Light SILAC medium (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Proteomics (Procedures B, C, and D)<br/><em>Note: For spectrophotometric protein concentration assay, such as the BCA or Bradford assay (Olson and Markwell, 2007), many ready-made kit products for such assays are available.</em> <ol class="npwtwo"><li>1.5 ml microcentrifuge tubes (Eppendorf, catalog number: 022363204)</li><li>2 ml safe-lock microcentrifuge tubes (Eppendorf, catalog number: 022363344)</li><li>5 ml or 15 ml tubes<br/><em>Note: See Step B3e iv for additional considerations regarding microcentrifuge tubes.</em> </li><li>OMIX C18 pipette tips, 10-100 Î¼l (Agilent, catalog number: A57003100)</li><li>Low binding tubes (<em>e.g.</em>, Sorenson, catalog numbers: 11300, 39640T); Eppendorf Protein LoBind, Eppendorf Safe-lock (<em>e.g.</em>, Eppendorf, catalog numbers: 145525, 145530)</li><li>2-[4-(2-hydroxyethyl)piperazin-1-yl]ethanesulfonic acid (HEPES)</li><li>Sodium chloride (NaCl)</li><li>Sodium hydroxide (NaOH)</li><li>NaH<sub>2</sub>PO<sub>4</sub>Â·H<sub>2</sub>O</li><li>Na<sub>2</sub>HPO<sub>4</sub> </li><li>Triton X-100 </li><li>Protease inhibitor cocktail, EDTA-free, Roche (Millipore Sigma, catalog number: 11873580001)</li><li>Ammonium sulfate ((NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>)</li><li>Mono- and di-sodium phosphate</li><li>Coomassie Brilliant Blue G (Millipore Sigma, catalog number: B0770)</li><li>Phosphoric acid</li><li>Methanol</li><li>Hydrochloric acid (HCl; Fisher Scientific, catalog number: A466)</li><li>Water (LC-MS grade; Fisher Scientific, catalog number: W6500)<br/><em>Note: Prepare solutions used for mass spectrometry sample workup and analyses with LC-MS grade water and the highest purity reagents available.</em> </li><li>Acetonitrile (LC-MS grade; Thermo Fisher Scientific, catalog number: 51101)</li><li>Ammonium Bicarbonate (use highest purity grade available; <em>e.g.</em>, Millipore Sigma, catalog number: 022363344)<br/><em>Note: Prepare 50 ml of a 100 mM solution with LC-MS grade water, filter through a sterile 0.22 Âµm syringe filter, and store at 4 Â°C in a clean polypropylene tube. Make fresh every couple weeks.</em> </li><li>Trypsin Gold, Mass Spectrometry Grade (Promega, catalog number: V5280)<br/><em>Note: Prepare trypsin stock solution at 100 ng/Âµl in 1 mM HCl; store in aliquots of up to 20-50 Âµl at -80 Â°C.</em> </li><li>Trifluoroacetic Acid (TFA) (sequencing grade, or better; <em>e.g.</em>, Thermo Fisher Scientific, catalog number: 28902)<br/><em>Note: Produce a 20% w/v TFA stock with LC-MS grade water.</em> </li><li>Acetic Acid (Fisher Scientific, catalog number: 60-046-912)</li><li>(Optional) Antibodies for Western blotting<br/><em>Note: Commercially available antibodies that we have used with success to identify ORF1p and ORF2p-3xFLAG by Western blotting.</em><br/><ol class="npwthree"><li>Î±-ORF1p, clone 4H1 (Millipore Sigma, catalog number: MABC1152), primary antibody, titerâ1:5,000 (original concentration 2 mg/ml; final concentration 0.4 Âµg/ml)</li><li>Anti-FLAG M2 antibody (Millipore Sigma, catalog number: F3165 or F1804), primary antibody, titerâ1:2,000 (original concentration 1 mg/ml; final concentration 0.5 Âµg/ml)</li><li>Î±-mouse IgG, HRP-linked whole antibody (GE Healthcare, catalog number: NA931-1ML), secondary antibody for enhanced chemiluminescence, titerâ1:10,000.</li></ol></li><li>Anti-FLAG M2 antibody (Millipore Sigma, catalog number: F3165 or F1804)</li><li>Dynabeads M-270 epoxy (Thermo Fisher Scientific, catalog number: 14302D)</li><li>Glycerol</li><li>Bovine Serum Albumin (BSA; Standard Ampules, 2 mg/ml, Thermo Fisher Scientific, catalog number: 23209)</li><li>1 M iodoacetamide stock solution<br/><em>Note: Best practice is to prepare iodoacetamide in water, fresh; iodoacetamide is light sensitive and chemically labile. In our experience, 1 M aliquots can be stored at -20 Â°C, protected from light, and are still functional for several months.</em> </li><li>RNase A/T1 Mix (Thermo Fisher Scientific, catalog number: EN0551)</li><li>4x LDS (lithium dodecyl sulfate) sample loading buffer (Thermo Fisher Scientific, catalog number: NP0007)</li><li>10x sample reducing agent (Thermo Fisher Scientific, catalog number: NP0004); or 500 mM dithiothreitol (DTT)</li><li>4%-12% Bis-Tris gels, 10-well, 1 mm (Thermo Fisher Scientific, catalog number: NP0321BOX)<br/><em>Note: Sample volumes loaded as described here may be up to 60-70 Âµl in volume, exceeding the standard well capacity. Thus, each sample is loaded twice (described in Step D4). After the first loading the sample is run just into the gel, and then the remainder is loaded into the same well and the run is completed. Alternatively, gels with sufficient well capacities for a single loading may be used, such as Bolt 4%-12% Bis-Tris Plus Gels, 10-well, 1 mm (Thermo Fisher Scientific, catalog number: NPNW04120BOX).</em> </li><li>20x MOPS (3-morpholinopropane-1-sulfonic acid) gel running buffer (Thermo Fisher Scientific, catalog number: NP0001) </li><li>10x PBS stock solution (see Recipes)</li><li>0.1 M sodium phosphate pH 7.4 (see Recipes)<br/></li><li>3 M Ammonium sulfate (see Recipes)</li><li>Magnetic Media Storage Solution (see Recipes)</li><li>Protein extraction solution (see Recipes)</li><li>Gel acidification solution (see Recipes)</li><li>Blue Silver stain (see Recipes)</li><li>Gel plug destaining solution (see Recipes)</li><li>Tip wetting solution (see Recipes)</li><li>Tip wash solution (see Recipes)</li><li>Elution 1 solution (see Recipes)</li><li>Elution 2 solution (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti27349">###  Equipment</p><p class="ppzz"><em>Note: Typical molecular biology lab equipment is required, not all explicitly listed below. catalog numbers are given for most of the equipment as examples; instruments from alternative manufacturers may be substituted provided equivalent functionality. Equipment has been listed approximately in order of use.<br/><br/></em> </p><ol class="npwone"><li>Cell culture (Procedure A)<ol class="npwtwo"><li>Pipettes</li><li>CO<sub>2</sub> incubator for mammalian cell culture</li><li>Scalpels (<em>e.g.</em>, size 11) or razor blades</li><li>New Brunswick Innova 2000 platform shaker (Eppendorf, catalog number: M1190-0002; or Thermo Fisher Scientific, model: Forma Model 416; or equivalent)<br/><em>Note: Any shaker installed in a mammalian cell culture incubator must be able to tolerate continuous high humidity (~90% relative humidity).</em> </li><li>Swing-bucket centrifuge rotor</li><li>1 L square PYREX glass bottles (Corning, catalog number: 1396-1L)<br/><em>Note: Plastic bottles inhibit transfection (polypropylene, polyethylene, PETG)</em> </li><li>Erlenmeyer flasks</li><li>Ice bucket</li><li>Hemocytometer</li><li>High speed centrifuge with appropriate rotor for pelleting cells from large volumes (<em>e.g.</em>, for ~250 or 500 ml bottles)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Proteomics (Procedures B, C, and D)<ol class="npwtwo"><li>Metal spatula</li><li>Milling balls, stainless steel, 20 mm (Retsch, catalog number: 05.368.0062)</li><li>50 ml stainless steel milling jar (Retsch, catalog number: 01.462.0149)</li><li>PTFE milling jar insulator (custom made, see LaCava <em>et al.</em>, 2016)<br/><em>Note: The use of a jar insulator is optional, but strongly recommended.</em> </li><li>Planetary Ball Mill PM 100 (Retsch, model: PM 100, catalog number: 20.540.0001) </li><li>Vacuum centrifugal concentrator (a.k.a. speed-vac)</li><li>Neodymium magnet microfuge tube rack (Thermo Fisher Scientific, catalog number: 12321D) </li><li>Thermomixer (Eppendorf, catalog number: 5355000.011; or equivalent) </li><li>Refrigerated microcentrifuge (capable of reaching ~20,000 RCF) </li><li>Microtip sonicator (<em>e.g.</em>, Qsonica, model: Q700) equipped with a low intensity 1/16â microtip probe (Qsonica, catalog number: 4717) </li><li>SDS-PAGE electrophoresis system (<em>e.g.</em>, Thermo Fisher Scientific NuPage)</li><li>Gel knife or chisel</li><li>Orbital shaker</li><li>Low-extractable borosilicate glass vials with chemically resistant caps (<em>e.g.</em>, Kimble, catalog number: 74515-20) <br/><em>Note: These are recommended for the storage of stock and working solutions used for MS</em><em>.</em> </li><li>Quadrupole-Orbitrap mass spectrometer (Thermo Fisher Scientific Q Exactive or Fusion series) interfaced with a nano-flow liquid chromatography (LC) system (<em>e.g.</em>, Thermo Scientific EASY-nLC 1200 or UltiMate 3000 RSLCnano)<br/><em>Note: Older model LC-MS systems may also be perfectly suitable; at the time this analysis was initiated we were using LTQ-Orbitrap models, see Step D7.</em> </li></ol></li></ol><p class="pptt" id="biaoti27350">###  Software</p><ol class="npwtwok"><li>MaxQuant</li><li>(Optional) R<br/></li></ol><p class="pptt" id="biaoti27351">###  Procedure</p><p class="ppzz">The protocol below anticipates that suspension-grown, SILAC-labeled cells will be used to enable a final readout by quantitative MS. In the case that SILAC labeling is not necessary for your experimental design, effective standard procedures for L1 expression in HEK-293T<sub>LD</sub> cells are thoroughly described in Taylor <em>et al.</em>, 2016; review of this reference is recommended in any case. In the case that adherent cells will be SILAC labeled, see Taylor <em>et al.</em>, 2013.</p><ol class="npwone"><li>Express ORF2p-3xFLAG-tagged L1 in SILAC labeled HEK-293T<sub>LD</sub> suspension cultures<br/>SILAC-labeled cells will be transfected in suspension using pLD401 plasmid DNA, and/or other 3xFLAG-tagged constructs such as pMT302 (Taylor <em>et al.</em>, 2013), and PEI (polyethylenimine); after transfection, L1 expression will be induced by the addition of doxycycline (Dox; typically 1 Î¼g/ml). High-quality endotoxin-free DNA is critical to success. pLD401 contains a synthetic, recoded human L1 sequence (<em>Orfeus</em>-Hs) encoding a carboxy-terminal 3xFLAG-tagged ORF2p, expressed under the control of the tetracycline response element promoter (An <em>et al.</em>, 2011; Taylor <em>et al.</em>, 2013). Suspension culture yields large human cell pellets with minimal work, waste, and cost when compared to monolayer adherent growth. We typically obtain ~1.5 g HEK-293T<sub>LD</sub> cells, wet cell weight, per 100 ml culture. In the past we have used square Corning Pyrex bottles in an orbital platform shaker at 130 rpm, installed inside of a 37 Â°C, humidified incubator, maintained at 5%-8% CO<sub>2</sub> (Muller <em>et al.</em>, 2005; Taylor <em>et al.</em>, 2016; Domanski and LaCava, 2017); however, more recently we have obtained comparable results using Erlenmeyer flasks at the same speed. For instructions on setting up the hardware for such a system, refer to Taylor <em>et al.</em>, 2016. In addition to procedures we describe, standard mammalian cell culturing techniques and best practices apply (Freshney, 2011; Uphoff and Drexler, 2013).<br/>  If starting from adherent HEK-293T<sub>LD</sub> cells, follow the HEK cell suspension adaptation protocol presented in Domanski and LaCava, 2017; however, unlike the cells described in that procedure, frozen stocks made from suspension-adapted HEK-293T<sub>LD</sub> cells can be used to directly inoculate new suspension cultures successfully. If starting from a frozen stock or recently suspension-conditioned HEK-293T<sub>LD</sub> cells, proceed as detailed below. Cells may grow to a density of ~10 million/ml but growth slows after ~5 million/ml and transfection efficiency drops when cells are not in logarithmic growth.<br/><strong><em>IMPORTANT:</em></strong> <em>We have observed that, over time, the Dox-inducibility of HEK-293T</em><sub><em>LD</em></sub><em> can markedly decline. This has been determined to be caused by loss of expression of rtTA. Therefore, these cells should be periodically re-selected for blasticidin resistance (the BSD gene [Kimura </em><em>et al.</em><em>, 1994]) was co-incorporated with rtTA), ensuring a transcriptionally active rtTA locus. Although keeping cells under continuous selection is the surest way to ensure a maximally Dox-responsive cell population, cells selected prior to making frozen stocks respond in sufficient proportion and uniformity for the work described below; it is also gentler on the cells overall. We therefore periodically re-select HEK-293T</em><sub><em>LD</em></sub><em> cell populations under challenge of 10 Âµg/ml blasticidin S prior making stocks, but do not maintain cells under continuous selection. </em><br/><strong><em>Practical note:</em></strong> <em>Before proceeding to full-scale expression in SILAC medium, we recommend optimization experiments in standard medium to verify transfection efficiency and Dox induction response within your setup and implementation. We recommend parallel transfections with a Tet-On GFP plasmid and a constitutively active GFP plasmid, with DNA prepared at the same time and in the same method as the pLD401 plasmid to be transfected. The ratio of PEI-max to DNA may be varied along with other parameters, see Longo </em><em>et al.</em><em>, 2013; Stuible </em><em>et al.</em><em>, 2018. Prior to culturing cells in suspension at full-scale, it may also be helpful to compare the per-cell expression of ORF1p and/or ORF2p to that of cells grown in adherent cultures, transiently transfected with either PEI-max or a commercial cationic lipid-based reagent, maintaining other variables comparably to suspension culture. Proper suspension culture produces indistinguishable per-cell expression from adherent culture transfected with either method (see Taylor </em><em>et al.</em><em>, 2018, Appendix 1âFigure 1). </em> <ol class="npwtwo"><li>Prepare heavy and light SILAC media (see Recipes).<ol class="npwthree"><li>Dialyze Tet-free FBS to remove amino acids for 24 h, at 4 Â°C, against 10-volumes of PBS with 2 buffer changes using 3500 MWCO dialysis tubing. <br/><em>Example: 500 ml FBS, dialyzed against 5 L PBS starting at ~noon (Day 1). Before leaving the lab in the evening change the buffer (5 L fresh PBS). First thing the next morning (Day 2) change the buffer again; retrieve dialyzed FBS ~noon. For this regime, it is helpful to have two 10 L buckets.</em> </li><li>For âheavyâ SILAC medium use Arg10 and Lys8; for âlightâ SILAC medium instead use naturally occurring L-arginine and L-lysine.</li><li>Sterilize the media by filtration.</li></ol></li><li>Inoculate otherwise identical heavy and light SILAC suspension cultures at a cell density of ~0.2 million cells per ml in 200 ml.</li><li>Grow to ~5 million/ml. <br/>Obtaining accurate cell counts can be challenging because HEK-293T<sub>LD</sub> tends to grow in small clumps of up to ~30 cells in suspension. Accurate counting requires dissociation of the clumps by gentle trituration using a pipette.<ol class="npwfour"><li>Using a 1 ml Serological pipette, aliquot 200 Î¼l of culture to a clean microcentrifuge tube.</li><li>Mix by flicking, and pipette 10 Î¼l onto one side of the hemocytometer (also called hemacytometer). <br/></li><li>With a 200 Î¼l pipette, set the volume to ~150 Î¼l and triturate 30 times to break up clumps. Try not to foam. </li><li>Pipette 10 Î¼l onto the remaining half of the hemocytometer. </li><li>Visualize the cells before and after cell clump dissociation by trituration. Shearing during trituration may lyse a small fraction of the cells. Increase or decrease trituration if appropriate to get accurate cell counts with minimal lysis. Properly triturated samples can also be counted in automated cell counters and leverage vital stains.</li></ol></li><li>Passage these cells, which have now doubled ~5 times, inoculating into fresh heavy and light SILAC suspension cultures at a cell density of ~0.2 million/ml in 200 ml.<br/><strong><em>IMPORTANT:</em></strong> <em>From the time of seeding cells in SILAC suspension, until the time of harvesting, the cells should have been permitted to double â¥ 10 times to assure thorough Arg10/Lys8 metabolic labeling of the proteome. Following the scheme presented here starting from Step A2, above, through Step A8, below, assures 11 doublings. Additional passages prior to induction, if convenient, provides similar results.</em> </li><li>Grow to ~2.5 million/ml in 200 ml medium (~4 doublings). We have successfully transfected at up to 4 million/ml but recommend this concentration to ensure logarithmic growth.</li><li>On <strong>Day 1</strong>âtransfect the cells in suspension<br/>Each 200 ml suspension culture will receive 1 Âµg/ml pLD401 DNA and 3 Âµg/ml PEI âMaxâ 40 kDa, prepared in ~1/20<sup>th</sup> the final culture volume hybridoma SFM medium. A master transfection mix for the two cultures is made. The following example assumes 2 x 200 ml SILAC culturesâone heavy, one lightâand transfection with pLD401 (ORF2p-3xFLAG). If working with a larger number, scale up accordingly. A parallel GFP plasmid may be used as a control for transfection efficiency, we recommend a Tet-On plasmid (in which case, add 1 Âµg/ml Dox to the control GFP culture at the time of transfection).<ol class="npwthree"><li>Warm 20 ml hybridoma-SFM medium to room temperature, 10 ml hybridoma-SFM per 200 ml culture to be transfected.</li><li>Add 400 Î¼g pLD401 to the hybridoma-SFM and mix well, <em>e.g.</em>, by vortexing.</li><li>Add 1.2 ml of 1 mg/ml PEI Max (pH 7.0) to the hybridoma-SFM/DNA solution and mix well. <br/>Final volume should be ~22-23 ml. </li><li>Incubate for 15 min at room temperature to allow DNA-PEI complexes to form. </li><li>Pipette half of the transfection mixture into each 200 ml culture and return to the incubator.</li></ol></li><li>On <strong>Day 2</strong>âsplit cells 1:2.5-1:3<br/>Transfer half (100 ml) of each transfected SILAC suspension culture to each of two 1 L bottles containing 150-200 ml of fresh SILAC medium, resulting in 2 x 250 ml or 2 x 300 ml cultures for both heavy and light.<br/><em>Note: If desired, before the decision to split the cells and use more SILAC media, check the efficiency of the GFP control transfection culture.</em> </li><li>On <strong>Day 3</strong>âinduce each culture by addition of dox at a final concentration of 1 Î¼g/ml.<br/>The cell density at the time of induction should be ~4 million/ml (~2 doublings, constituting ~11 total doublings to this point). Dilute accordingly or wait an additional day if needed. L1 expression should be induced for ~24 h, by 48 h, ORF2p expression declines (Taylor <em>et al.</em>, 2013); shorter induction times may be sufficient but should be verified.</li><li>On <strong>Day 4</strong>âharvest cells by centrifugation and extrude into liquid nitrogen.<ol class="npwthree"><li>Count cells; counts are useful for normalization and blotting. </li><li>Gather an ice bucket and clean centrifuge bottles. </li><li>Transfer a 1 ml aliquot(s) of each culture for Western blotting to a microcentrifuge tube. Spin at â¤ 1,000 RCF for 30 s, aspirate the media, and store on ice until freezing is convenient. <br/><strong><em>IMPORTANT: </em></strong><em><span id="__kindeditor_bookmark_start_188__" style="display:none;"></span>Check for comparable levels of ORF1p and ORF2p-3xFLAG expression in heavy and light labeled cell cultures by Western blot (antibodies and titers described in Materials and Reagents B25). If the expression levels are very different (greater than Â±20%), the samples may not be suitable for quantitative comparisons or will require thoughtful normalizations.</em> </li><li><em>Spin the cultures at â¤ 1,000 x g for 10 min at 4 Â°C to pellet.<span id="__kindeditor_bookmark_end_189__" style="display:none;"></span></em> <br/>This can be done in several 50 ml conical tubes for each batch, or in fewer, larger bottles. <br/><em>Note: The pellets should appear approximately uniform. If centrifuged too hard,</em><em> cells will be crushed, forming two different-colored layers. The goal here, and in subsequent centrifugations of these cells, is to obtain a relatively tight-packed cell pellet that maximally excludes buffer without breaking the cells. We consider 1,000 x g to be the maximum nominal centrifugal force to be used. We are commonly using a Sorvall T6000D or Beckman Allegra X-14R with swing-bucket centrifuge rotor at 2,000 rpm (~830-930 x g). One may want to establish their choice operating speeds in a preliminary experiment.</em> </li></ol></li><li>Resuspend the pellets in a minimal volume of PBS (approximately equal volume to the pellet) using a 25 ml pipette set to âslow.â One may pool resuspended cells. <br/><em>Note: Additional washes may be added if needed; however, we attempt to limit the amount of handling before freezing. Moreover, one may choose to handle the cells obtained from each suspension culture separately, </em><em>e.g.</em><em>, in case of potential differences in metabolic labeling efficiency (see below), transfection, or Dox induction efficiency between cultures. Powders produced from similar cell populations may be mixed after milling and quality control (Procedure B) if desired; when kept separate, each culture represents a biological replicate. On the other hand, pooling is often advantageous for sample handling and may be done at this point.</em> </li><li>Pellet cells inside syringes.<ol class="npwthree"><li>Select syringe(s) for cell pelleting. Syringes with capacities in the range of 10-30 ml are typically appropriate for suspension culture yields. </li><li>Remove the plungers and set aside. </li><li>Securely cap syringes with Luer-lock end caps at spout, Parafilm at plunger opening, and place inside 50 ml conical tubes. </li><li>Spin at â¤ 1,000 <em>x g</em> for 10 min at 4 Â°C. Transfer syringes to an ice bucket.<ol class="npwfour"><li>During the centrifugation, insert a conical tube rack in a Styrofoam box. Fill the box with LN<sub>2</sub> to at least the mid-point of the rack. </li><li>Pre-label 50 ml conical tubes. Punch a number of holes in the caps of the tubes using a 16 Ga needle. This will permit the LN<sub>2</sub> to be drained from the tubes later without loss of cell material. These caps will later be replaced with un-punched caps. To avoid plastic waste, punch caps taken from used 50 ml conical tubes destined for disposal (wash caps first) and save punched caps for future use. Each tube will hold approximately 15-20 g extruded cells, which we often refer to as âBBsâ because of their round shape.</li><li>Transfer the pre-labeled tubes to the rack in LN<sub>2</sub>. Fill the tubes with LN<sub>2</sub>. </li></ol></li><li>Aspirate the PBS, leaving only the wet cell pellet in the syringe.</li></ol></li><li>Extrude the cells into the LN<sub>2</sub> as follows. Remove the Luer lock cap. Positioning the bottom of the syringe over the 50 ml conical tube with LN<sub>2</sub>, insert the syringe plunger carefully. Extrude the cells gradually into the tube containing LN<sub>2</sub>. If injected too fast, they will form large clumps. If more LN<sub>2</sub> is needed, carefully pour more in. Use the pre-chilled spatula as needed to break apart any clumps. </li><li>Wearing appropriate protective gloves, cap the tube using the punched lid. Pressure may build up and cause LN<sub>2</sub> to spurt out through the punches. Exercise caution. </li><li>Decant LN<sub>2</sub> into dewar or Styrofoam box and replace the punched lid with an intact lid. Store the tubes at -80 Â°C until cryomilling.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cryomill L1-expressing HEK-293T<sub>LD</sub> cells and conduct quality control checks<ol class="npwtwo"><li>Cryogenically disrupt the cell pellets<br/>Cryo-milling L1 expressing HEK-293T<sub>LD</sub> cells has been previously described in intricate detail, see LaCava <em>et al.</em>, 2016; Taylor <em>et al.</em>, 2016. Briefly:<ol class="npwthree"><li>Pre-cool an insulator, milling jar, 2 x 20 mm grinding balls, a metal spatula, and a 50 ml tube by immersing in LN<sub>2</sub>. </li><li>Place the cell pellets inside the jar, jar inside the insulator, and assembly inside the Retsch PM 100 milling apparatus. </li><li>Set the counter balance and clamp the jar in place. Run the machine for 3 milling cycles of 3 min each (reverse rotation, 1 min interval, no break time) at 400 rpm. </li><li>Cool down the milling jar with LN<sub>2</sub> between cycles. </li><li>Recover the resulting cell powder using a spatula and transfer to 50 ml tube. Store at -80 Â°C.</li></ol></li><li>Assess the comparability of the cell powders by protein content<br/>Extract 25-50 mg of each cell powder into 9 parts (225-450 Âµl) protein extraction solution (see immediately below) and check by protein assay that each yields a similar protein content (typically Â±10%). Extracts may be produced more dilute if greater volume is advantageous. Save 10 Âµl for checking heavy amino acid incorporation (Step B3, below). The procedure essentially follows the rationale laid out in Shevchenko <em>et al.</em>, 2006.<ol class="npwthree"><li>Combine cell powder with your favorite protein extraction solution (<em>e.g.</em>, what you may use for immunoprecipitationâsee Recipes), vortex (and if necessary, briefly sonicate as per Step C2f) to homogenize, and then centrifuge the extract for 10 min at 20,000 <em>x g</em> to produce a clarified extract that can be measured by <em>e.g.</em>, Bradford or BCA assays. </li><li>Alternatively, extract proteins in a Tris-buffered solution containing 2% (w/v) SDS and measure protein content by BCA assayâSDS is not well tolerated by the Bradford assay (Zor and Selinger, 1996). SDS-based solutions tend to produce viscous cell extracts requiring a greater degree of sonication than typical IP solutions.</li></ol><em><strong>IMPORTANT:</strong> If comparable protein quantities are obtained from similar masses of cell powder, future experiments and comparisons may be calibrated to the mass of powder extracted. If the respective yields of protein from heavy and light SILAC cell powders made under identical extraction conditions are not comparable, then one source of material may have retained more liquid in the cell pellet before extrusion into LN</em><sub><em>2</em></sub><em> (</em><em>i.e.</em><em>, incomplete PBS removal) or may have accumulated excessive condensate due to storage at -80 Â°C with an improperly closed cap. Additional water weight will reduce the apparent yield of protein per mg of cell powder. In this case, future IP experiments using those materials should be calibrated by relative protein content per mass of powder.</em> </li><li>Assess the percentage of heavy amino acid incorporation.<br/><em><strong>IMPORTANT:</strong> In order to make quantitative comparisons by mass spectrometry using SILAC methods, heavy SILAC labeling should be thorough, ~95% or better. Provided (a) the heavy SILAC proteome is thoroughly labeled with Arg10 and Lys8, and both (b) target protein expression, as assessed by Western blotting (done in Step A9c, above), as well as (c) protein content per mass of cell powder, as assessed by </em><em>e.g.</em><em>, a colorimetric assay (Step B2, above), are both in good agreement between heavy and light materials, then these materials are ideal for quantitative mass spectrometry work. </em> <ol class="npwthree"><li>Conduct SDS-PAGE of clarified cell extracts from heavy-labeled cells.<ol class="npwfour"><li>Add DTT to 25 mM to the 10 Âµl sample(s) (Step B2a) and heat at 70 Â°C for 10 min, and then briefly chill samples on ice.</li><li>Add iodoacetamide to 100 mM and incubate at room temperature for 30 min in the dark.</li><li>Load a range of aliquotsâ<em>e.g.</em>, 1, 2, and 4 Âµl or 3 and 6 Âµlâon <em>e.g.</em>, a 10-well, 1 mm, 4-12% Bis-Tris gel.<br/><em>Notes:</em><br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-5px;"><div class="float-left"><em>1)</em> </div><div class="media-body"><em>Loading a range ensures the likely production of gel plugs with an appropriate quantity of protein, despite variations in extract efficiency, etc.</em> </div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-5px;"><div class="float-left"><em>2)</em> </div><div class="media-body"><em>Typical liquid chromatography columns attached to mass spectrometry systems have loading capacities of ~1 Âµg. In our hands, loading Â½ the peptides ultimately obtained from a 4 Âµl aliquot of extract and following the handling procedures described here, only approaches â¤ 20% of the column capacity upon loading. This is sufficient to estimate the H-label incorporation. More material may be loaded at the userâs discretion.</em> </div></div></li><li>Electrophorese the samples until they migrate ~6 mm into the gel as assessed by gel loading dye at the sample migration front.</li></ol></li><li>Fix and Coomassie stain the gel.<ol class="npwfour"><li>We use âblue silverâ (see Recipes), with the following modified pre-washing procedure for NuPAGE Bis-Tris gels. All steps that follow are carried out at room temperature until otherwise stated.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-4px;"><div class="float-left">1)</div><div class="media-body">After opening the gel cassette, with the gel adhered to one of the plates, use a clean gel knife/chisel to trim away the wells and the portion of the gel below the 6 mm mark (readily discerned by the position of the loading-dye front).</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-4px;"><div class="float-left">2)</div><div class="media-body">Transfer the upper portion of the gel from the plastic cassette to a clean plastic 15 cm diameter petri dish and incubate for 10 min in 40 ml of de-ionized water with gentle agitation on an orbital shaker.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-4px;"><div class="float-left">3)</div><div class="media-body">Remove the water and replace it with 40 ml of 0.1 M HCl; incubate for 10 min.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-4px;"><div class="float-left">4)</div><div class="media-body">Remove the 0.1 M HCl and replace it with 40 ml of water; incubate for 10 min.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-4px;"><div class="float-left">5)</div><div class="media-body">Remove the water and replace it with 40 ml of blue silver stain; incubate for 2 h to overnight. If the gel has been properly washed <strong style="display:inline !important;">in the prior 3 steps</strong>, the blue silver will remain a deep, translucent green color due to maintenance of the low pH dye solution. If the washing is incomplete, residual buffering capacity within the gel may turn the stain blue-ish, and/or, the gel may develop uneven background staining. Changing the stain and incubating several hours to overnight in fresh (green-hued) blue silver will typically resolve this problem and result in mild, uniform background staining with major bands clearly visible above background.</div></div></li><li>Destain background thoroughlyâseveral hours to overnight.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-4px;"><div class="float-left">1)</div><div class="media-body">For blue silver, simply use several changes of de-ionized water.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-4px;"><div class="float-left">2)</div><div class="media-body">You may choose to take an image of the gel. Always clean surfaces used to image gels that will contribute sample to mass spectrometry analyses.</div></div></li></ol></li><li>Cut the regions of gel containing the samples from just below the well to just above the dye front (referred to as a âgel plug;â will appear similar to those depicted in Figure 4C, but more protein rich)â<strong>Day 1</strong>. <br/><em>Note: If extracted 1:9 as suggested in Step B2a, the 2 or 4 Âµl samples should contain a sufficient quantity of protein for this analysis. The gel should be cut upon a clean surface. We use a rectangular glass plate placed over white paper, w</em><em>hich aids visibility of the gel and its pieces. Solutions used below should be prepared with âmass spectrometryâ grade reagents.</em> <ol class="npwfour"><li>Using a clean razor blade or scalpel, chop each one into 1 mm<sup>3</sup> cubes (about 25 pieces per gel plug), and transfer to a 1.5 ml microfuge tube. <br/><em>Note: The blade may be cleaned of residual gel fragments when necessary by agitating it in a beaker of molecular biology grade water; residual liquid can be wicked away by touching the tip of the blade to a clean low-lint tissue (âKimwipeâ).</em> </li><li>Add 500 Âµl of âgel plug destaining solutionâ to the gel pieces in the tubes (50 mM ammonium bicarbonate in 50% (v/v) acetonitrile; see Recipes). </li><li>Mix with the gel pieces by vortexing.<br/><em>Note: If the color of the solution changes to dark blue immediately, discard it and add another 500 Âµl of destaining solution. One milliliter may be used here instead of 500 Âµl; repeat multiple times if the solution becomes blue quickly.</em> </li><li>Incubate at 37 Â°C for 1 h with shaking in a Thermomixer (1,000-1,200 rpm); discard the supernatant. The gel pieces should be in motion, not settled at the bottom of the tube.</li><li>Add another 500 Âµl of destaining solution if the gel pieces are still blue.<br/><em>Note: Continue to destain as needed (this step can go overnight).</em> </li><li>Remove destain when the gel pieces no longer exhibit blue staining.<br/><em>Note: They should appear whitish and semi-transparent with no obvious blue remaining.</em> </li><li>Dehydrate gel pieces with 500 Âµl acetonitrile, vortex, let the tubes sit at room temperature for 2 min or more.<br/><em>Note: Gel pieces will appear opaque and thoroughly white.</em> </li><li>Remove acetonitrile from all samples.</li><li>Dry samples in a vacuum centrifuge (âspeed-vacâ) for 10 min.</li><li>Samples can be stored at -20 Â°C or proceed to trypsin digest.</li></ol></li><li>In-gel digestions of proteins to peptides using trypsinâ<strong>Day 2</strong>.<ol class="npwfour"><li>Prepare all tubes on ice.</li><li>Prepare trypsin working solution: 12.5 ng/Âµl in 50 mM ammonium bicarbonate. Our trypsin stock is 100 ng/Âµl in 1 mM HCl.</li><li>Add enough 12.5 ng/Âµl trypsin to cover the gel pieces. <br/><em>Note: This will vary by gel plug dimensions, but in our experience is ~38 Âµl for gel plugs obtained from 15-well, 1 mm thickness NuPAGE gel, and ~90 Âµl for a 10-well gel of the same.</em> </li><li>Let samples swell for 45 min on ice.</li><li>After swelling, add 50 mM ammonium bicarbonate to cover gel pieces if necessary. <br/><em>Note: Typically, ~12 Âµl for 15-well or ~30 Âµl for 10-well, 1 mm thickness gel plug.</em> </li><li>Move the tubes to a 37 Â°C incubator to digest overnight (or at least 6 h).</li></ol></li><li>Terminate the digest; collect and pool the peptidesâ<strong>Day 3</strong>.<ol class="npwfour"><li>Produce aliquots of 2% and 0.1% (w/v) TFA from the 20% stock.</li><li>Add 1/3<sup>rd</sup> the digestion volume of 2% (w/v) TFA to the digestion mix (0.5% w/v TFA final concentration) to stop digestion. </li><li>Incubate at room temperature for 5 min with shaking in a Thermomixer (1,000-1,200 rpm) or vortex mixer equipped with multi-tube holder (setting 2-3 on a vortexer with a max intensity of 10).</li><li>Remove the acidified digest solutions to microcentrifuge tubes (âprotein low-bindingâ optional); hold at room temperature or 4 Â°C while pieces are further extracted.<br/><strong><em>IMPORTANT:</em></strong> <em>Performance may vary, some considerations discussed here (Kraut </em><em>et al.</em><em>, 2009). Tubes that will be exposed to 40%+ acetonitrile (as below) may benefit from being pre-washed with pure acetonitrile. Some tubes may leak plasticizers or unincorporated (co)polymers when challenged with high concentrations of organic solvents. Some examples of polymer contaminated mass spectra are displayed in LaCava </em><em>et al.</em><em>, 2015, Figure 5. The best practice is to empirically test your tubes, and then repeat these tests periodically (or, when a problem occurs) because occasional (usually unannounced) manufacturing changes may alter performance. Pre-washing reduces the potential for contamination of the sample by components of tube manufacturing by first solubilizing them in </em><em>e.g.</em><em>, high acetonitrile and removing them with the wash. We routinely use ~100 Âµl acetonitrile to wash 0.65 ml tubes and ~300 Âµl to wash 1.5 ml tubes as a precaution (although it may not always be necessaryâtesting is advised for any doubt). Tubes we commonly use: Sorenson low binding (</em><em>e.g.</em><em>, #11300, #39640T), Eppendorf Protein LoBind, Eppendorf Safe-lock, Research Products International (</em><em>e.g.</em><em>, #145525, #145530). Importantly, the same rules apply for plastic pipette tips used for handling solvents! Use only high quality, solvent resistant tips. These are also available in low protein binding varieties if desired.</em> </li><li>Extract with 50 Âµl of 0.1% (w/v) TFA at room temperature for 45 min in a Thermomixer or vortex mixer equipped with multi-tube holder, speed setting as above.</li><li>Remove the extractants and combine them with the cognate <strong>digest solutions</strong>. The total volume may be ~100 Âµl for 15-well and ~150 Âµl for 10-well gel plugs.</li></ol></li><li>Desalt peptides using OMIX tips (C18, 100 Âµl) by centrifugation.<br/><strong><em>IMPORTANT:</em></strong> <em>The manufacturerâs protocol stipulates pipette-based desalting. We instead use a centrifugal approach because in our experience loading the samples on top of the C18 bed reduces the possibility of (albeit rare) micro-fragments of polyacrylamide gel (or other debris) remaining in the final desalted fractionâsuch debris may cause column clogging if applied to the mass spectrometerâs liquid chromatography column. This centrifugal approach is also beneficial when many samples are to be processed in parallel. If working with only few samples (as may be the case if assessing the heavy-label incorporation of only 1 or 2 cell cultures) you may implement the manufacturerâs pipette-based protocol using the solutions described belowâbut be mindful of potential debris carryover if column clogging issues are observed.</em> <ol class="npwfour"><li>Make the following before starting (solutions modified from manufacturerâs instructions; see Recipes). Calculate the amount of each solution needed based on the number of samples to be processed plus 10% volume.<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Make adaptors to hold the OMIX tips on top of the 2 ml collecting tubes (to collect waste) or 1.5 ml low binding tubes (to collect elution) during centrifugation steps. See Figure 2.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328032251_8326.jpg" width="160"/><br/><strong>Figure 2. Centrifugal device for solid phase extraction of tryptic peptides for LC-MS/MS.</strong> The individual parts are labeled on the left and shown as assembled on the right.<br/><br/></div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-10px;"><div class="float-left">2)</div><div class="media-body">Tip wetting solution: 200 Âµl per sample.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-2px;"><div class="float-left">3)</div><div class="media-body">Tip washing solution: 600 Âµl per sample.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-2px;"><div class="float-left">4)</div><div class="media-body">Elution 1 solution: 80 Âµl per sample.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-2px;"><div class="float-left">5)</div><div class="media-body">Elution 2 solution: 80 Âµl per sample.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-2px;"><div class="float-left">6)</div><div class="media-body">Carefully label each OMIX tip to help keep track of which sample is loaded on which tip during the following centrifugation steps. </div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-2px;"><div class="float-left">7)</div><div class="media-body">Similarly, label a set of clean 1.5 ml low binding tubes (pre-washed with 100% acetonitrile) for collecting eluted peptides.</div></div><div class="media media-x cnautokh cnautokh8" style="margin-top:-2px;"><div class="float-left">8)</div><div class="media-body">Put each tip in a 2 ml collecting tube and proceed.</div></div></li><li>Wet the tips by applying 2x 100 Âµl of tip wetting solution.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-2px;"><div class="float-left">1)</div><div class="media-body">Add 100 Âµl of wetting solution. </div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-2px;"><div class="float-left">2)</div><div class="media-body">Spin at 500 rpm for 1 min (27 <em>x g</em> in an Eppendorf centrifuge 5415).</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-2px;"><div class="float-left">3)</div><div class="media-body">Repeat the two steps above, once. <br/><em>Note: Check the volume of waste collected during centrifugation steps; make sure that the OMIX tip will not touch the waste in the collecting tube. Aspirate the liquid in the collecting tubes when needed.</em> </div></div></li><li>Wash tips by applying 2x 100 Âµl of tip washing solution.<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Add 100 Âµl of washing solution.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-2px;"><div class="float-left">2)</div><div class="media-body">Spin at 500 rpm (27 <em>x g</em>) for 1 min.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-2px;"><div class="float-left">3)</div><div class="media-body">Repeat the two steps above, once.</div></div></li><li>Bind the samples to C18 resin.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-2px;"><div class="float-left">1)</div><div class="media-body">Add the <strong>digest solutions</strong> from each gel plug (â¥ 100 Âµl) to the appropriate OMIX tips. (The tips should accept â¥ 200 Âµl.)</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-2px;"><div class="float-left">2)</div><div class="media-body">Spin at 500 (27 <em>x g</em>) for 1 min.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-2px;"><div class="float-left">3)</div><div class="media-body">Check each tip, make sure that the entire sample passes through the filter.</div></div></li><li>Wash the samples by applying 4x 100 Âµl of tip washing solution.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-2px;"><div class="float-left">1)</div><div class="media-body">Add 100 Âµl of washing solution.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-2px;"><div class="float-left">2)</div><div class="media-body">Spin at 500 rpm (27 <em>x g</em>) for 1 min.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-2px;"><div class="float-left">3)</div><div class="media-body">Repeat the two steps above, three times.</div></div></li><li>Elute the desalted peptides from the C18 resin.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-2px;"><div class="float-left">1)</div><div class="media-body">Transfer the OMIX tips from the 2 ml collecting tubes to the set of 1.5 ml sample collection tubes. (Make sure that the label on the OMIX tip matches the label of the tube.)</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-2px;"><div class="float-left">2)</div><div class="media-body">Elution #1: apply 80 Âµl <u>elution 1 solution</u> to the OMIX tips.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-2px;"><div class="float-left">3)</div><div class="media-body">Spin at 500 rpm (27 <em>x g</em>) for 1 min. (Check that all the solution has passed through.)</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-2px;"><div class="float-left">4)</div><div class="media-body">Elution #2: apply 80 Âµl <u>elution 2 solution</u> to the OMIX tips. Elutions #1 and #2 will be collected in the same 1.5 ml tube.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-2px;"><div class="float-left">5)</div><div class="media-body">Spin at 500 rpm (27 <em>x g</em>) for 1 min. (Check that all the solution has passed through. Refer to the combined fractions as <strong style="display:inline !important;">peptide solutions</strong>.)</div></div></li><li>Put the tubes containing the <strong>peptide solutions</strong> in a vacuum centrifuge and run until dry. (The time varies by vacuum strength.)</li></ol></li><li>Liquid Chromatography Tandem Mass Spectrometry (LC-MS/MS): MS methods will vary depending upon the liquid chromatography (LC) setup, the model of MS instrument, and the operator. We provide the following only as guidelines for conducting shotgun proteomic analyses of affinity enriched samples using an Orbitrap-based hybrid mass spectrometer for LC-MS/MS (Gillet <em>et al.</em>, 2016). We typically resuspend our dried samples in 10 Âµl of 0.5% (v/v) acetic acid (or, samples are initially concentrated to this volume). On a first run, ~1/2 or preferably ~1/3 of the sample will be loaded onto the LC column for MS analysis (permitting double the quantity to be used in a second run if needed). The column bound peptides are eluted in an acetonitrile gradient (typically reaching â¥ 40% (v/v) final concentration of acetonitrile over ~1 h), in the presence of an ion pairing reagent such as acetic acid (0.1 M) or formic acid (0.1% v/v) at a flow rate of 200-300 nl/min. Data dependent MS<sup>2</sup> fragmentation spectra (<em>e.g.</em>, of the top 20 most abundant MS<sup>1</sup> ions) are acquired via collision induced dissociation (CID) or high-energy collisional dissociation (HCD). The resulting mass spectra then require processing in a suitable software package to interpret the proteins present as well as relative quantities and specificities thereof. <br/><em>Note: MaxQuant (Cox and Mann, 2008) is a widely used, freely available MS processing software package with detailed protocols available (Tyanova </em><em>et al.</em><em>., 2016)âour procedure is summarized below.</em><br/></li><li>Data processing: Use MaxQuant to search the raw data against a database of human protein sequences (we typically us a www.uniprot.org fasta file), as well as a decoy database of reversed protein sequences (set in software), with Arg10 and Lys8 as potential heavy labels. Note that although ORF1p and ORF2p are now represented in Uniprot (entries Q9UN81 and O00370, respectively), you may want or need to add custom sequences for ectopically expressed L1s to your database. For the determination of heavy labeling, we do not typically use the âre-quantifyâ option; and a typical result with these settings will identify and quantify â¥ 300 proteins. The âpeptide.txtâ output file can be used to calculate the heavy amino acid incorporation (Geiger <em>et al.</em>, 2011). Peptides corresponding to known exogenous contaminants such as keratin and trypsin should be removed. Lysine- and arginine-containing peptides are considered separately. The âratio H/Lâ for each peptide is converted to H/(H+L) using the equation H/(H+L) = (âratio H/Lâ)/(1 + âratio H/Lâ). The median values of H/(H+L) for lysine- and arginine-containing peptides are then obtained. Note, these values underestimate incorporation, as a peptide âratio H/Lâ cannot be calculated by MaxQuant when there is no detectable signal at the mass-to-charge ratio of the light partner, as would be expected in the case of full incorporation of heavy amino acid. If the average of the median values of the lysine- and arginine-containing peptides is ~95% or better, it is indicative of thorough heavy labeling and suitability for quantitative comparisons by SILAC.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>Differential affinity immobilization<ol class="npwtwo"><li>Production of anti-FLAG magnetic medium. The procedure has been summarized below. We refer the reader to the following references: Cristea and Chait, 2011; Domanski <em>et al.</em>, 2012; and Taylor <em>et al.</em>, 2016 for comprehensive background on the procedure. Both the Cristea and Chait, and the Taylor <em>et al.</em> references contain detailed step-by-step procedures. However, one may also simply follow the manufacturerâs instructions (Thermo Fisher Scientific). Briefly:<br/><ol class="npwthree"><li>Prepare 20 Î¼l of 0.5 Î¼g/Î¼l antibody (anti-FLAG M2), in a sodium phosphate (pH 7.4) buffered 1 M ammonium sulfate solution, per mg of magnetic beads to be antibody-conjugated (<em>i.e.</em>, 10 Î¼g of antibody, in 20 Î¼l solution, will be used per mg of beads).<br/><em>Note: Millipore Sigma product F1804 contains 50% glycerol and will need to be buffer exchanged prior to epoxy coupling; product F3165 can be used directly (Domanski </em><em>et al.</em><em>, 2012).</em> </li><li>Combine the antibody solution and the magnetic beads and incubate overnight (16-24 h) at 30-37 Â°C with rotation. The mixing applied should be sufficient to ensure the beads remain in suspension for the duration of the incubation. For small batches, conjugation can be performed at 37 Â°C in a thermomixer (â¥ 1,200 rpm); however, evaporation and the formation of condensate away from the bulk solution should be avoided, so appropriately even heating or sufficiently thorough mixing are required.</li><li>After coupling, remove the excess antibody solution, wash the beads to remove residual uncoupled antibodies. <br/><em>Note: You may choose to analyze the antibody depletion during conjugation by running 1-2 Âµl of the antibody mix before (input) and after (flow-through) the coupling on an SDS-polyacrylamide gel and Coomassie staining.</em> </li><li>Finally resuspend the antibody-conjugated magnetic medium in âstorage solutionâ (Recipe B4): add 6.7 Î¼l of storage solution per 1 mg of medium (resulting in a slurry of ~15% w/v) and store at -20 Â°C. The slurry can be stored in this way for at least 1 year without any noticeable loss of performance.  </li></ol></li><li>Affinity purification in conjunction with on-the-beads RNase treatment <br/>A general procedure describing and demonstrating best practices for affinity purification using mammalian cell powder and magnetic media has been previously described (LaCava <em>et al.</em>, 2016); a procedure specifically for anti-FLAG affinity capture of ORF2p-3xFLAG-tagged L1 RNPs has also been detailed (Taylor <em>et al.</em>, 2016); the following implementation has been modified to test the effects of RNase A/T1 treatment on affinity immobilized L1 RNPs. Before carrying out a SILAC-based quantitative MS experiment (described below), preliminary studies should be carried out using general protein staining and Western blotting (<em>e.g.</em>, see Figure 4A, and Taylor <em>et al.</em>, 2013, Figure 6).<br/><ol class="npwthree"><li>Pre-cool a metal spatula and 8 x 2 ml safe-lock tubes in LN<sub>2</sub>. An aluminum block style tube rack is helpful for this step.</li><li>Dispense 4 x 200 mg of heavy and light HEK-293T<sub>LD</sub> cell powders, respectively (<em>i.e.</em>, 800 mg total of each), to pre-cooled 2 ml safe-lock tubes. <br/><em>Note: It is important to keep the cell powder frozen. Leave the tubes in LN</em><sub><em>2</em></sub><em> until the next step is initiated.</em> </li><li>Place the samples in the rack, open the caps, and let them stand at room temperature for 1 min. </li><li>Add 800 Âµl of room temperature protein extraction solution (see Recipes) to each tube.</li><li>Vortex at maximum speed to fully resuspend the cell powder (should not require more than 30 s), and immediately place the samples on ice. <br/><em>Note: Once the cell powder is resuspended, the sample should be held on ice between all manipulations throughout the procedure unless otherwise stated.</em> </li><li>Briefly sonicate, on ice or at 4 Â°C, to completely disperse and homogenize the cell powder. As a guide, we apply ~15-20 J energy per 100 mg cell powder extracted with 400 Âµl solution (LaCava <em>et al.</em>, 2016). <br/><em>Note: If clumps are visible, the sonication step should be repeated until no clumps are obvious by visual inspection. Likewise, once extracts appear homogenous upon visual inspection, no further sonication is needed.</em> </li><li>Clarify the extract by centrifugation at 20,000 <em>x g</em> for 10 min (4 Â°C). </li><li>While the samples are in the centrifuge, distribute 20 Î¼l anti-FLAG magnetic medium (slurry) into each of the four 2 ml microcentrifuge tubes and wash thrice each with 1 ml of the extraction solution of choice to thoroughly remove the storage solution. <br/><em>Note: We use 5 Î¼l of the beads slurry per 100 mg of the cell powderâthis has been titrated for the average expression of ORF2p in our hands, under the preceding conditions. To wash the beads, combine extraction solution with the slurry in a 2 ml microcentrifuge tube and vortex briefly to fully resuspend the beads. Pulse-spin the tube briefly in a mini-centrifuge to collect all the solution at the bottom and then place the tube in a magnetic tube rack until the beads are collected at the side of the tube. Remove the supernatant using a pipette or an aspirator and repeat the washing step twice more time. Perform washing steps at room temperature. After washing, the beads may be held on ice and are ready for use.</em> </li><li>Combine all heavy (with heavy) and light (with light) clarified extracts in 5 ml (or larger) tubes, and then dispense each equally into two new 2 ml tubes containing 20 Î¼l anti-FLAG beads resulting in four tubes total: 2 x 400 mg-scale heavy and 2 x 400 mg-scale light ORF2p-3xFLAG affinity isolations.</li><li>Incubate affinity isolations at 4 Â°C for 30 min with end-over-end mixing. </li><li>Wash 3 x 1 ml extraction solution. </li><li>Transfer the beads to fresh 1.5 ml tubes during 2<sup>nd</sup> wash.</li><li>After removing the final wash, organize the tubes for 4 reactions:<br/>Heavy-untreated: add 50 Î¼l of extraction solution + 2 Î¼l 2 mg/ml BSA<br/>Heavy-treated: add 50 Î¼l of extraction solution + 2 Î¼l 2 mg/ml RNase A/T1<br/>Light-untreated: add 50 Î¼l of extraction solution + 2 Î¼l 2 mg/ml BSA<br/>Light-treated: add 50 Î¼l of extraction solution + 2 Î¼l 2 mg/ml RNase A/T1</li><li>Incubate the reactions at room temperature for 30 min with gentle mixing (sufficient to keep the beads suspended: ~1,000 rpm).</li><li>Collect the supernatants after 30 min and save these samples aside. You may choose to analyze them by <em>e.g.</em>, analysis by SDS-PAGE (Figure 2A).</li><li>Wash the beads in each tube with 3 x 1 ml of ice-cold extraction solution.</li><li>Elute the materials retained on the affinity medium with 40 Î¼l of 1.1x LDS.<br/><em>Note: Elution for 10 min at room temperature with moderate agitation is typically sufficient to thoroughly release 3xFLAG-tagged proteins from anti-FLAG M2 epoxy magnetic beads. However, elution may also be achieved by incubation at 70 Â°C for 5 min with moderate agitationâwhich is more stringent and effective across a wide variety of affinity media. The latter has been observed to mildly increase the co-elution of immunoglobulin chains from anti-FLAG affinity medium prepared as described in this protocol but has not been observed to significantly contribute to eluted protein content: usually IgG chains are observed at only a bit above the limit of detection by Blue Silver stain (estimated in the ~1-5 ng range). Immunoglobulin chains release from other media may be drastic under these conditions and should be checked, case-by-case.</em> </li><li>Samples may be stored at -20 Â°C or colder.<br/><br/></li></ol></li></ol></li><li>Analysis by quantitative MS<ol class="npwtwo"><li>Combine elution samples as follows:<ol class="npwthree"><li>Light-treated with Heavy-treated (RNase<sup>light</sup> -with- RNase<sup>heavy</sup>; 10 Î¼l each)</li><li>Light-untreated with Heavy-untreated (BSA<sup>light</sup> -with- BSA<sup>heavy</sup>; 10 Î¼l each)</li></ol><em>Note: Samples <strong>a</strong> and<strong> b</strong> are controls: the expectation is that light and heavy samples should contribute the comparable protein constituents approximately equally to the mixtureâ</em><em>i.e.</em><em>, the metabolic labeling should not itself affect the result. Most proteins observed in these fractions should be ~50% heavy-labeled.</em><br/><ol class="npwthree" start="3"><li>Light-treated with Heavy-untreated (RNase<sup>light</sup> -with- BSA<sup>heavy</sup>; 30 Î¼l each)</li><li>Light-untreated with Heavy-treated (BSA<sup>light</sup> -with- RNase<sup>heavy</sup>; 30 Î¼l each)</li></ol><em>Notes:</em> <ol class="npwthree"><li><em>Samples <strong>c</strong> and <strong>d </strong>are the experimental samples representing label-swapped technical replicates comparing case (RNase) to control (BSA).</em> </li><li><em>Small aliquots (~10%) of the separate or combined fractions may be set aside for additional cross-checks by SDS-PAGE and staining or Western blotting.</em><br/></li></ol>The mixing of samples <strong>a-d</strong> is displayed schematically in Figure 4B and the resulting SDS-polyacrylamide gel is shown in Figure 4C.</li><li>Add DTT to 25 mM to all samples and heat at 70 Â°C for 10 min, and then briefly chill samples on ice.</li><li>Add iodoacetamide to 100 mM and incubate at room temperature for 30 min in the dark.</li><li>Load sample on a 10-well, 1 mm, 4-12% Bis-Tris gel.<ol class="npwthree"><li>Before assembling the gel in the electrophoresis apparatus, use a ruler to draw a horizontal line across the gel cassette, in indelible marker, at a position 6 mm below the bottom of the wells.</li><li>Samples <strong>c</strong> and <strong>d</strong> will be ~70 Î¼l each and may be loaded in two rounds into the same well if needed. First load 40 Î¼l and apply 200 V until the whole sample enters the gel (~30 s-1 min). Now load the remaining sample. Carefully layer the sample at the very bottom of the well using a gel-loading tip; residual glycerol in the well from the first loading will prevent the subsequent sample from âfallingâ into the well, so layering from the bottom of the well is crucial during the second round of sample loading.</li></ol></li><li>Electrophorese until all the loading dye reaches the 6 mm mark and stain (see Figure 4C), as per Step B3b (above).</li><li>Excise gel plugs, and process as per Steps B3c to B3f (above).</li><li>LC-MS/MS as per Step B3g (above)<br/>Details of the MS run as we conducted it are as follows: ~3 Âµl (<em>i.e.</em>, 1/3<sup>rd</sup>) were injected per LC-MS/MS analysis. Samples were loaded onto a PicoFrit column (New Objective, Woburn, MA) packed in-house with 6 cm of reverse-phase C18 material (YMC*Gel ODS-A, YMC, Allentown, PA). Peptides were gradient-eluted (Solvent A = 0.1 M acetic acid, Solvent B = 0.1 M acetic acid in 70% v/v acetonitrile, flow rate 200 nl/min) into an LTQ-Orbitrap-Velos (Thermo Fisher Scientific) acquiring data-dependent CID fragmentation spectra.</li></ol></li></ol><p class="pptt" id="biaoti27352">###  Data analysis</p><ol class="npwtwok"><li>Initial data processing in MaxQuant<ol class="npwthree"><li>Searches were performed against human protein sequences (UP000005640, April 2016) supplemented with custom L1 ORF1p and ORF2p protein sequences, common exogenous contaminants, and a decoy database of reversed protein sequences. <br/>Search parameters included: <br/><ol class="npwfour"><li>Fixed modification: carbamidomethyl (C).</li><li>Variable modifications: Arg10, Lys8, methionine oxidation.</li><li>Razor and unique peptides used for protein quantitation.</li><li>Requantify: enabled. </li></ol></li><li>Contaminants, low-scoring proteins, and proteins with only one razor + unique peptide were filtered out from the MaxQuant output file âproteingroups.txtâ. The list of contaminants was uploaded from the MaxQuant website (<a href="http://www.coxdocs.org/" target="_blank">http://www.coxdocs.org/</a>; âcontaminantsâ). Additionally, proteins with the âPOTENTIAL CONTAMINANTâ column value â+â were filtered out. </li><li>Proteins with at least two razor+unique peptides were retained for the analysis. H/(H + L)â<em>i.e.</em>, % heavy - and L/(H + L)â<em>i.e.</em>, % light - values were derived from unnormalized âratio H/Lâ values and were used in post-processing (below) prior to plotting the label-swapped RNase-sensitivity, shown in Figure 4D.</li></ol></li><li>Custom post-processing<br/><ol class="npwthree"><li>The RNase sensitivity data obtained from MaxQuant were rescaled and normalized such that proteins that did not change upon treatment with RNases were centered at the origin and those that were completely sensitive would give a value of 1.0. In a perfect experiment, unchanging proteins would yield a ratio of 0.5 when comparing the fraction of each protein present in the BSA-treated sample to the sum of both the BSA- and RNase-treated samples; <em>i.e.</em>, 1/(1 + 1). However, our data exhibit some variability, as shown in Figure 3A. Therefore, we normalized the data such that the peaks were re-centered at 0.5. From this set, 0.5 was subtracted from the data (centering insensitive proteins at the origin, and completely sensitive proteins at 0.5), followed by multiplication by two to expand the data to cover the range from 0 (insensitive) to 1 (completely sensitive); these latter two transformations are encompassed by the functions: <br/><br/>g(x) = x + b, and f[g(x)] = a (x + b) <br/><br/>where, a = 2; b = -0.5.<br/><br/></li><li>To determine if these data were normally distributed the distances from the (0, 0) point to protein coordinates were calculated. Proteins with distance less than two median distances were selected. The Shapiro-Wilk normality test (the null-hypothesis of this test is that the population is normally distributed) was applied for the distances (<em>P</em>-value = 0.29). The distribution of the distances was plotted as a histogram displaying the frequency (y-axis) versus RNase sensitivity (x-axis) of a simulation of normally distributed data (shown in black) and the actual data shown in blue; a Q-Q plot was also drawnâboth displayed in Figure 3B.</li><li>The mean value and standard deviation were calculated using the distribution of distances from the origin. The distance threshold for <em>P</em>-value = 0.001 was calculated using the R programming language. A circle with radius equal to the threshold was plotted and points with distances higher than the threshold were marked as black (Figure 2D).</li><li>The relevant code for the above post-processing can be obtained here: <a href="https://urldefense.proofpoint.com/v2/url?u=https-3A__bitbucket.org_altukhov_line-2D1_src_master_src_protein-5Fanalysis_RNAse.R&amp;d=DwMFaQ&amp;c=JeTkUgVztGMmhKYjxsy2rfoWYibK1YmxXez1G3oNStg&amp;r=S_fUBiCcviF6Fdue-UJckB9Wrp0Npp34W_7B69dw-pA&amp;m=WPGPstP-cVq68CjiTGsrBZJM-FZ8D3gEAba4vprTqNc&amp;s=iHYwiHzgNvlNfeDqePihCO7y49Wz4a0t56iTaXbO004&amp;e=" target="_blank">https://bitbucket.org/altukhov/line-1/src/master/src/protein_analysis/RNAse.R</a><br/><br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328032719_9214.jpg" width="300"/><br/><strong>Figure 3. Data treatment.</strong> A. Data re-centering as described in step 2a (Data analysis). B. Normality tests as described in step 2b (Data analysis). This figure reproduced from/modified from Taylor <em>et al.</em> (2018).</li></ol><br/>The prior I-DIRT analysis (Taylor <em>et al.</em>, 2013) is an important aspect of the presented analysis. SILAC as conducted hereâso-called mix after purification or MAP-SILAC (Wang and Huang, 2008)âdoes not reveal any information about which interactions are specific or non-specific; it only reveals what has changed between the two treatments (with RNases or BSA). We used I-DIRT, first, in order to establish which interactions are likely to have formed <em>in vivo</em>, versus which are likely to be post-extraction <em>in vitro</em> artifacts (Tackett <em>et al.</em>, 2005). Importantly, the I-DIRT data was collected using comparable experimental conditions (<em>i.e.</em>, protein extraction and affinity capture parameters) as for the SILAC data. By using the I-DIRT analysis as a specificity filter on the SILAC analysis, second, we were able to focus our attention on the differential behaviors of proteins most likely to be bona fide physical interactors within L1 RNPs (colored nodes, Figure 4D). Nevertheless, the proteins that were RNase sensitive and not I-DIRT-specific (black nodes, Figure 4D) proved informative: they are all RNA-binding proteins that one might expect to be released by RNase treatment. Thus, they served as a secondary indicator that the treatment is effective. Notably, PABPC1 and C4âpoly(A) RNA binding proteins appear insensitive to RNase treatment. This is most likely due to the fact that neither RNase A nor T1 cleave RNA at adenosine residues (Volkin and Cohn, 1953; Yoshida, 2001); hence poly(A) binding proteins may not be ready targets for release from direct RNA binding by the assay implemented here (or generally, using these ribonucleases) â see further discussion in (Taylor <em>et al.</em>, 2018). On the basis of these data, we concluded that UPF1, ZCCHC3, MOV10, and ORF1p are at least partly dependent upon the presence of intact RNA to remain stably associated within L1 RNPs.<br/><br/><img alt="" class="layerphoto" height="111" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328032844_5931.jpg" width="120"/><br/><strong>Figure 4. RNase sensitivity affinity capture</strong> (modified from Taylor <em>et al.</em>, 2018). A. Coomassie Blue G-250 stained SDS-polyacrylamide gel displaying the profiles of the RNase- or BSA-treated affinity immobilized fractions. This kind of gel would be run during preliminary studies of RNase (or other enzyme) treatments. Major changes can be observed visually. If the identities of bands visually observed change upon RNase treatment are not know, they may be cut from <em>e.g.</em>, the RNase supernatant lane (far left) and BSA elution lane (far right) and analyzed by mass spectrometry. More, subtle potential differences may be monitored by Western blotting. B. Mixing scheme for light and heavy SILAC-labeled fractions, as described in Step D1. C. Example image of Coomassie Blue G-250 stained gel plugs resulting from Step D5. Brackets indicate the gel areas excised and prepared for analysis. (1) Molecular mass marker. (2) Light-treated with Heavy-treated (RNase<sup>light</sup> -with- RNase<sup>heavy</sup>). (3) Light-untreated with Heavy-untreated (BSA<sup>light</sup> -with- BSA<sup>heavy</sup>). (4) Light-treated with Heavy-untreated (RNase<sup>light</sup> -with- BSA<sup>heavy</sup>). (5) BSAâ200 ng. (6) Light-untreated with Heavy-treated (BSA<sup>light</sup> -with- RNase<sup>heavy</sup>). D. Processed data displaying the degree of RNase-sensitivity (<em>i.e.</em>, differential affinity immobilization) exhibited by constituents of the SILAC-labeled fractions. Proteins requiring intact RNA to maintain stable interactions with immobilized ORF2p were released from the RNase-treated medium, while the BSA-treated sample controlled for the spontaneous release of proteins from the medium. Results from the assay have been graphed as the fraction of each detected protein present in the BSA-treated sample (RNase-sensitive proteins are more present in the BSA treated sample), normalized as described in step 2 (Data analysis). A cut-off of <em>P </em>= 10<sup>-3</sup> for RNase-sensitivity is indicated by a light gray circle; proteins that are RNase-sensitive with a statistical significance of <em>P</em> &lt; 10<sup>-3</sup> are outside the circle. Proteins previously ranked significant by I-DIRT analysis (Taylor <em>et al.</em>, 2013) are labeled and displayed in blue or magenta (as indicated); black nodes were RNase-sensitive but not significant by I-DIRT; gray, unlabeled nodes were neither RNase-sensitive nor significant by I-DIRT.<br/></li></ol><p class="pptt" id="biaoti27353">###  Recipes</p><ol class="npwone"><li>Cell culture media<br/><em>Note: Phenol red (final 0.001% w/v) may be added as a pH indicator. Antibiotics and/or antimycotics are not recommended in combination with PEI-based transfection; the combination of PEI and penicillin-streptomycin reduces cell viability and transfection efficiency. Most of our cultures are antibiotic free. Although the Freestyle-293 medium is designed to be used serum-free with the manufacturerâs cell lines, we have seen reduced growth rates and low viability at FBS below 1% (v/v) for HEK-293T</em><sub><em>LD</em></sub><em> and other 293 cells.</em> <ol class="npwtwo"><li>Heavy SILAC medium<br/>Supplement Freestyle-293 medium lacking arginine and lysine with: <br/>1% v/v dialyzed FBS<br/>2 mM L-glutamine<br/>200 mg/L L-proline<br/>800 mM <sup>13</sup>C<sub>6</sub><sup>15</sup>N<sub>4</sub>-L-arginine (Arg10)<br/>400 mM <sup>13</sup>C<sub>6</sub><sup>15</sup>N<sub>2</sub>-L-lysine (Lys8)</li><li>Light SILAC medium<br/>Supplement Freestyle-293 medium lacking arginine and lysine with: <br/>1% v/v dialyzed FBS<br/>2 mM L-glutamine<br/>200 mg/L L-proline<br/>800 mM L-arginine<br/>400 mM L-lysine</li></ol><br/></li><li>Magnetic bead coupling solutions<ol class="npwtwo"><li>10x PBS stock solution<br/>3.1 g of NaH<sub>2</sub>PO<sub>4</sub>Â·H<sub>2</sub>O, 10.9 g of Na<sub>2</sub>HPO<sub>4</sub>, and 87.8 g NaCl per liter; the final pH should be 7.4, confirm with pH indicator paper<br/>A 1x solution yields ~10 mM phosphate, 150 mM NaCl at pH 7.4</li><li>0.1 M sodium phosphate pH 7.4<br/>Follow a standard phosphate buffer chart, or <em>e.g.</em>, add 3.1 g of NaH<sub>2</sub>PO<sub>4</sub>Â·H<sub>2</sub>O and 10.9 g of Na<sub>2</sub>HPO<sub>4</sub> (anhydrous) to distilled H<sub>2</sub>O to make a volume of 1 L<br/>The pH of the final solution will be 7.4âconfirm with pH indicator paper</li><li>3 M Ammonium Sulfate<br/>39.6 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> combined with 0.1 M sodium phosphate pH 7.4 to a final volume of 100 ml</li><li>Magnetic Media Storage Solution<br/>1x PBS [final] containing 0.5 mg/ml BSA and 50% (v/v) glycerol</li></ol><br/></li><li>Protein extraction, staining, and destaining solutions<ol class="npwtwo"><li>Protein extraction solution<br/>20 mM HEPES-Na pH 7.4<br/>1% (v/v) Triton X-100<br/>500 mM NaCl<br/><em>Note: This solution is supplemented with protease inhibitors during protein extraction and is used without the addition of protease inhibitors during subsequent washes of the affinity medium. The frozen cell powders should be extracted with a room temperature solution, resulting in cold extracts that are subsequently handled on ice and/or at 4 Â°C; for subsequent washes of the affinity medium, after protein capture, the solution should be ice-cold. </em> </li><li>Gel acidification solution<br/>0.1 M HCl</li><li>Blue Silver Stain (modified from Candiano <em>et al.</em>, 2004)<br/>20% (w/v) H<sub>3</sub>PO<sub>4</sub><br/>10% (w/v) (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub><br/>0.12% (w/v) Coomassie Brilliant Blue G-250<br/>20% (v/v) methanol<ol class="npwthree"><li>To water (1/10<sup>th</sup> of the final volume), add first the phosphoric acid (we use double the published quantity). Note, commercially available concentrated phosphoric acid tends to be prepared at ~85% (w/w) with a density of ~1.69 g/ml. </li><li>Next, add the ammonium sulfate. When the ammonium sulfate has dissolved (incrementally add more water if this is taking excessively long), add the G-250 dye, and bring the water to 80% of the final volume. </li><li>To this solution, under vigorous stirring, add anhydrous methanol to 20% the final volume; once the solution is thoroughly mixed, adjust it to the final volume with water. </li><li>This solution should be kept in a brown or foil-covered bottle and is stable at room temperature for 6 months.</li></ol></li><li>Gel plug destaining solution<br/>50 mM ammonium bicarbonate in 50% (v/v) acetonitrile, prepared in LC-MS grade water<br/><em>Example: Combine one part 100 mM ammonium bicarbonate in LC-MS grade water with one part LC-MS grade acetonitrile. Prepare this solution fresh before use. One-hundred millimolar ammonium bicarbonate yields a solution of pH â¥ 8.5; 50 mM ammonium bicarbonate should produce a solution of pH ~8.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>OMIX C18 tip, peptide desalting solution<ol class="npwtwo"><li>Tip wetting solution<br/>0.1% (w/v) TFA<br/>50% (v/v) acetonitrile<br/>200 Âµl per sample</li><li>Tip wash solution<br/>0.1% (w/v) TFA<br/>600 Âµl per sample</li><li>Elution 1 solution<br/>0.5% (v/v) acetic acid<br/>40% (v/v) acetonitrile<br/>80 Âµl per sample</li><li>Elution 2 solution<br/>0.5% (v/v) acetic acid<br/>80% (v/v) acetonitrile<br/>80 Âµl per sample</li></ol></li></ol><p class="pptt" id="biaoti27354">###  Acknowledgments</p><p class="ppzz">We thank Professors Michael P. Rout, Brian T. Chait, Jef D. Boeke, and David FenyÃ¶ for their invaluable support of our research. This work was supported in part by the National Institutes of Health grants R01GM126170 and P41GM109824.</p><p class="pptt" id="biaoti27355">###  Competing interests</p><p class="ppzz">The authors declare no conflicting or competing interests.</p><p class="pptt" id="biaoti27356">###  References</p><ol class="npwtwok"><li>An, W., Dai, L., Niewiadomska, A. M., Yetil, A., O'Donnell, K. A., Han, J. S. and Boeke, J. D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21320307" target="_blank">Characterization of a synthetic human LINE-1 retrotransposon ORFeus-Hs.</a> <em>Mob DNA</em> 2(1): 2.</li><li>An, W., Davis, E. S., Thompson, T. L., O'Donnell, K. A., Lee, C. Y. and Boeke, J. D. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19481606" target="_blank">Plug and play modular strategies for synthetic retrotransposons.</a> <em>Methods</em> 49(3): 227-235.</li><li>Armean, I. M., Lilley, K. S. and Trotter, M. W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23071097" target="_blank">Popular computational methods to assess multiprotein complexes derived from label-free affinity purification and mass spectrometry (AP-MS) experiments.</a> <em>Mol Cell Proteomics</em> 12(1): 1-13.</li><li>Candiano, G., Bruschi, M., Musante, L., Santucci, L., Ghiggeri, G. M., Carnemolla, B., Orecchia, P., Zardi, L. and Righetti, P. G. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15174055" target="_blank">Blue silver: a very sensitive colloidal Coomassie G-250 staining for proteome analysis.</a> <em>Electrophoresis</em> 25(9): 1327-1333.</li><li>Cristea, I. M. and Chait, B. T. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21536766" target="_blank">Conjugation of magnetic beads for immunopurification of protein complexes.</a> <em>Cold Spring Harb Protoc.</em> doi:10.1101/pdb.prot5610.</li><li>Cox, J. and Mann, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19029910" target="_blank">MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification.</a> <em>Nat Biotechnol</em> 26(12): 1367-1372.</li><li>Dai, L., Taylor, M. S., O'Donnell, K. A. and Boeke, J. D. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22907758" target="_blank">Poly(A) binding protein C1 is essential for efficient L1 retrotransposition and affects L1 RNP formation.</a> <em>Mol Cell Biol</em> 32(21): 4323-4336.</li><li>Domanski, M. and LaCava, J. (2017). <a href="https://bio-protocol.org/e2238" target="_blank">Affinity purification of the RNA degradation complex, the exosome, from HEK-293 cells.</a> <em>Bio-protocol</em> 7(8): e2238.</li><li>Domanski, M., Molloy, K., Jiang, H., Chait, B. T., Rout, M. P., Jensen, T. H. and LaCava, J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22668517" target="_blank">Improved methodology for the affinity isolation of human protein complexes expressed at near endogenous levels.</a> <em>Biotechniques</em> 0(0): 1-6.</li><li>Freshney, R. I. (2011). <a href="https://www.wiley.com/en-us/Culture+of+Animal+Cells%3A+A+Manual+of+Basic+Technique+and+Specialized+Applications%2C+7th+Edition-p-9781118873656" target="_blank"><em>Culture of animal cells</em>.</a> 6th edition. John Wiley &amp; Sons.</li><li>Geiger, T., WiÅniewski, J. R., Cox, J., Zanivan, S., Kruger, M., Ishihama, Y. and Mann M. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21293456" target="_blank">Use of stable isotope labeling by amino acids in cell culture as a spike-in standard in quantitative proteomics.</a> <em>Nat Protoc</em> 6(2): 147-57.</li><li>Gillet, L. C., Leitner, A. and Aebersold, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27049628" target="_blank">Mass spectrometry applied to bottom-up proteomics: entering the high-throughput era for hypothesis testing.</a> <em>Annu Rev Anal Chem (Palo Alto Calif)</em> 9(1): 449-472.</li><li>Gossen, M., Freundlieb, S., Bender, G., Muller, G., Hillen, W. and Bujard, H. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7792603" target="_blank">Transcriptional activation by tetracyclines in mammalian cells.</a> <em>Science</em> 268(5218): 1766-1769.</li><li>Kimura, M., Takatsuki, A. and Yamaguchi, I. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7948022" target="_blank">Blasticidin S deaminase gene from <em>Aspergillus terreus (BSD)</em>: a new drug resistance gene for transfection of mammalian cells.</a> <em>Biochim Biophys Acta</em> 1219(3): 653-659.</li><li>Kraut, A., Marcellin, M., Adrait, A., Kuhn, L., Louwagie, M., Kieffer-Jaquinod, S., Lebert, D., Masselon, C. D., Dupuis, A., Bruley, C., Jaquinod, M., Garin, J. and Gallagher-Gambarelli, M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19397304" target="_blank">Peptide storage: are you getting the best return on your investment? Defining optimal storage conditions for proteomics samples.</a> <em>J Proteome Res</em> 8(7): 3778-3785.</li><li>LaCava, J., Jiang, H. and Rout, M. P. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28060343" target="_blank">Protein complex affinity capture from cryomilled mammalian cells.</a> <em>J Vis Exp</em> (118): e54518.</li><li>LaCava, J., Molloy, K. R., Taylor, M. S., Domanski, M., Chait, B. T. and Rout, M. P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25757543" target="_blank">Affinity proteomics to study endogenous protein complexes: pointers, pitfalls, preferences and perspectives.</a> <em>Biotechniques</em> 58(3): 103-119.</li><li>Longo, P. A., Kavran, J. M., Kim, M. S. and Leahy, D. J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24011049" target="_blank">Transient mammalian cell transfection with polyethylenimine (PEI).</a> <em>Methods Enzymol</em> 529: 227-240.</li><li>Muller, N., Girard, P., Hacker, D. L., Jordan, M. and Wurm, F. M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15619325" target="_blank">Orbital shaker technology for the cultivation of mammalian cells in suspension.</a> <em>Biotechnol Bioeng</em> 89(4): 400-406.</li><li>O'Donnell, K. A., An, W., Schrum, C. T., Wheelan, S. J. and Boeke, J. D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23818630" target="_blank">Controlled insertional mutagenesis using a LINE-1 (ORFeus) gene-trap mouse model.</a> <em>Proc Natl Acad Sci U S A</em> 110(29): E2706-2713.</li><li>Okerman, L., Van Hende, J. and De Zutter, L. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17386725" target="_blank">Stability of frozen stock solutions of beta-lactam antibiotics, cephalosporins, tetracyclines and quinolones used in antibiotic residue screening and antibiotic susceptibility testing.</a> <em>Anal Chim Acta</em> 586(1-2): 284-288.</li><li>Olson, B. J. and Markwell, J. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429326" target="_blank">Assays for determination of protein concentration.</a> <em>Curr Protoc Protein Sci</em> 48: 3.4.1-3.4.29.</li><li>Ong, S. E., Blagoev, B., Kratchmarova, I., Kristensen, D. B., Steen, H., Pandey, A. and Mann, M. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12118079" target="_blank">Stable isotope labeling by amino acids in cell culture, SILAC, as a simple and accurate approach to expression proteomics.</a> <em>Mol Cell Proteomics</em> 1(5): 376-386.</li><li>Ostertag, E. M., Prak, E. T., DeBerardinis, R. J., Moran, J. V. and Kazazian, H. H., Jr. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10684937" target="_blank">Determination of L1 retrotransposition kinetics in cultured cells.</a> <em>Nucleic Acids Res</em> 28(6): 1418-1423.</li><li>Shevchenko, A., Tomas, H., Havlis, J., Olsen, J. V. and Mann, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406544" target="_blank">In-gel digestion for mass spectrometric characterization of proteins and proteomes.</a> <em>Nat Protoc</em> 1(6): 2856-2860.</li><li>Stuible, M., Burlacu, A., Perret, S., Brochu, D., Paul-Roc, B., Baardsnes, J., Loignon, M., Grazzini, E. and Durocher, Y. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29886030" target="_blank">Optimization of a high-cell-density polyethylenimine transfection method for rapid protein production in CHO-EBNA1 cells.</a> <em>J Biotechnol</em> 281: 39-47.</li><li>Tackett, A. J., DeGrasse, J. A., Sekedat, M. D., Oeffinger, M., Rout, M. P. and Chait, B. T. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16212429" target="_blank">I-DIRT, a general method for distinguishing between specific and nonspecific protein interactions.</a> <em>J Proteome Res</em> 4(5): 1752-1756.</li><li>Taylor, M. S., Altukhov, I., Molloy, K. R., Mita, P., Jiang, H., Adney, E. M., Wudzinska, A., Badri, S., Ischenko, D., Eng, G., Burns, K. H., Fenyo, D., Chait, B. T., Alexeev, D., Rout, M. P., Boeke, J. D. and LaCava, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29309035" target="_blank">Dissection of affinity captured LINE-1 macromolecular complexes.</a> <em>Elife</em> 7: e30094.</li><li>Taylor, M. S., LaCava, J., Dai, L., Mita, P., Burns, K. H., Rout, M. P. and Boeke, J. D. (2016). Garcia-PerÃ©z, J. L. (Ed.). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26895062" target="_blank">Characterization of L1-ribonucleoprotein particles.</a> In: <em>Methods in Molecular Biology</em>. Springer New York<em>.</em> 1400: 311-338.</li><li>Taylor, M. S., LaCava, J., Mita, P., Molloy, K. R., Huang, C. R., Li, D., Adney, E. M., Jiang, H., Burns, K. H., Chait, B. T., Rout, M. P., Boeke, J. D. and Dai, L. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24267889" target="_blank">Affinity proteomics reveals human host factors implicated in discrete stages of LINE-1 retrotransposition.</a> <em>Cell</em> 155(5): 1034-1048.</li><li>Tyanova, S., Temu, T. and Cox, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27809316" target="_blank">The MaxQuant computational platform for mass spectrometry-based shotgun proteomics.</a> <em>Nat Protoc</em> 11(12): 2301-2319.</li><li>Uphoff, C. C. and Drexler, H. G. (2013). <a href="https://www.springer.com/kr/book/9781627031271" target="_blank"><em>Basic cell culture protocols</em>.</a> 4th edition. In: Helgason, C. D. and Miller, C. L. (Eds.). Humana Press.</li><li>Volkin, E. and Cohn, W. E. (1953). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13129256" target="_blank">On the structure of ribonucleic acids. II. The products of ribonuclease action.</a> <em>J Biol Chem</em> 205(2): 767-782.</li><li>Wang, X. and Huang, L. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17934176" target="_blank">Identifying dynamic interactors of protein complexes by quantitative mass spectrometry.</a> <em>Mol Cell Proteomics</em> 7(1): 46-57.</li><li>Yoshida, H. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11582784" target="_blank">The ribonuclease T1 family.</a> <em>Methods Enzymol</em> 341: 28-41.</li><li>Zor, T. and Selinger, Z. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8660509" target="_blank">Linearization of the Bradford protein assay increases its sensitivity: theoretical and experimental studies.</a> <em>Anal Biochem</em> 236(2): 302-308.<br/></li></ol>