<body>
            <h1>Quantification of Hepatitis B Virus Covalently Closed Circular DNA in Infected  Cell Culture Models by Quantitative PCR</h1>
            <p><strong>Authors:</strong> Bingqian Qu, Stephan Urban</p>
            <h2>Abstract</h2>
            <p>Persistence of the human hepatitis B virus (HBV) requires the maintenance of covalently closed circular (ccc)DNA, the episomal genome reservoir in nuclei of infected hepatocytes. cccDNA elimination is a major aim in future curative therapies currently under development. In cell culture based <em>in vitro</em> studies, both hybridization- and amplification-based assays are currently used for cccDNA quantification. Southern blot, the current gold standard, is time-consuming and not practical for a large number of samples. PCR-based methods show limited specificity when excessive HBV replicative intermediates are present. We have recently developed a real-time quantitative PCR protocol, in which total cellular DNA plus all forms of viral DNA are extracted by silica column. Subsequent incubation with T5 exonuclease efficiently removes cellular DNA and all non-cccDNA forms of viral DNA while cccDNA remains intact and can reliably be quantified by PCR. This method has been used for measuring kinetics of cccDNA accumulation in several <em>in vitro</em> infection models and the effect of antivirals on cccDNA. It allowed detection of cccDNA in non-human cells (primary macaque and swine hepatocytes, <em>etc</em>.) reconstituted with the HBV receptor, human sodium taurocholate cotransporting polypeptide (NTCP). Here we present a detailed protocol of this method, including a work flowchart, schematic diagram and illustrations on how to calculate âcccDNA copies per (infected) cellâ.</p>
            <p><strong>Keywords:</strong> Hepatitis B Virus, Covalently closed circular DNA, cccDNA, T5 exonuclease, Quantitative PCR, Copy number per cell</p>
            <h2>Background</h2><p>The Hepatitis B virus (HBV), a DNA virus belonging to the family <em>Hepadnaviridae</em>, is a human pathogen persisting in approximately 240 million people globally. HBV infection leads to higher risks of liver cirrhosis and hepatocellular carcinoma (Liang <em>et al.</em>, 2015). At present, chronic HBV infection is not curable as current treatments do not eradicate the replicative reservoir, covalently closed circular (ccc)DNA (Levrero <em>et al.</em>, 2016). In <em>in vitro</em> infected hepatocytes, cccDNA is formed by cellular repair of relaxed circular duplex (rc)DNA, the genomic form in virions (Guo <em>et al.</em>, 2007; Long <em>et al.</em>, 2017; Schreiner and Nassal, 2017). cccDNA serves as the viral template for pregenomic RNA which becomes encapsidated into nucleocapsids and is reverse transcribed to generate rcDNA and double-stranded linear (dsl)DNA by the viral polymerase (Bartenschlager and Schaller, 1992; Tu <em>et al.</em>, 2017). Of note, albeit the reconstitution of hepatoma cells with the receptor human sodium taurocholate cotransporting polypeptide (NTCP), high excess of inoculated virions with high multiple genome equivalents (mges) (mges &gt; 100) is required to achieve moderate infection rates (<em>e.g.</em>, &gt; 20% in HepG2<sup>NTCP</sup> cells) (Qu <em>et al.</em>, 2018). This raises the problem that cccDNA-containing samples taken early after inoculation with HBV contain large amounts of rcDNA from input virions.<br/><br/>Therefore, adequate methodologies for absolute or relative quantification of cccDNA are required. They are divided into two categories: (I) hybridization after separation by gel electrophoresis (Southern blot) and (II) PCR amplification (Li <em>et al.</em>, 2017). Southern blot is still a gold standard but not very sensitive, and demands multiple experimental processes and a high-copy load of cccDNA (&gt; 2 x 10<sup>6</sup> copies using <sup>32</sup>P-radioactive isotope/digoxigenin/biotin probe; &gt; 1 x 10<sup>4</sup> copies using branched DNA technique) (Yu <em>et al.</em>, 2015). Therefore, hybridization-based methods are complicated, time-consuming and not practical for a large number of samples (<em>e.g.</em>, &gt; 20) to be analyzed in parallel. PCR methods include real-time quantitative (q)PCR, nested qPCR (Xu <em>et al.</em>, 2011), digital-droplet PCR (Mu <em>et al.</em>, 2015), and rolling circle amplification (Margeridon <em>et al.</em>, 2008). Real-time qPCR is the fastest and the most robust method for almost all laboratories. However, unlike Southern blot which gel electrophoretically separates cccDNA, PCR methods are not strictly specific, especially when rcDNA and other HBV replicative intermediates are present in excess, such as in <em>in vitro</em> infection (<em>e.g.</em>, MGE &gt; 300), even when cccDNA-specific primer pairs are used (Nassal, 2015; Qu <em>et al.</em>, 2018).<br/><br/>To solve this problem, we have developed a qPCR assay using validated cccDNA selective primer pairs and a digestion step by T5 exonuclease, which removes cellular DNA and all HBV intermediates via its exonuclease activity targeting free ends of rcDNA and dslDNA but leaves cccDNA intact (Qu <em>et al.</em>, 2018). This assay allowed fast and specific quantification of cccDNA within one working day as shown in Figure 1 (2 h of total DNA extraction; 1.5 h of T5 exonuclease reaction and 2 h of qPCR), accurate calculation of âcccDNA copies/(infected) cellâ (Figure 2) and drug efficacy testing on cccDNA levels (Figure 3). The method also provided quantitative judgment on whether cccDNA is formed in new cell models and identified low amount of cccDNA in <em>in vitro</em> infection of non-primate hepatocytes (Lempp <em>et al.</em>, 2016 and 2017). This protocol is adapted from Qu <em>et al.</em> (2018) and herein more detailed information on this qPCR quantification after T5 exonuclease digestion is included, and different primer pairs are compared to address the applicability of HBV genotypes. Taken together, this protocol will facilitate studies on cccDNA and help clinicians, technicians and graduate students to analyze cccDNA in samples derived from <em>in vitro</em> infection.<br/></p>
        </body>

<p class="pptt" id="biaoti27390">##  Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (Neptune, 1,000 Î¼l, 200 Î¼l, 20 Î¼l, 10 Î¼l, DNase-/RNase-free &amp; Biozym, premium tips 1000 Î¼l, 200 Î¼l, 10 Î¼l)</li><li>1.5 ml microcentrifuge tube (Sarstedt AG &amp; Co.KG, SafeSeal tube, catalog number: 72.706)</li><li>0.2 ml microcentrifuge tube (Greiner Bio-one, Sapphire PCR tube, catalog number: 683271)</li><li>Hard-Shell PCR plates 96-well, thin-wall (Bio-rad, catalog number: HSP9601)</li><li>Primary human hepatocytes (PHH) (obtained from Hannover medical school or prepared in University Hospital Heidelberg) (Possible commercial vendors of PHH are Lonza, Biopredic, BioIVT, ThermoFisher, however susceptibility to HBV may vary), HepaRG<sup>NTCP</sup> and HepG2<sup>NTCP</sup> cell lines (Qu <em>et al.</em>, 2018)</li><li>Plasmid pSHH2.1 (Cattaneo <em>et al.</em>, 1983), available upon request from the corresponding author</li><li>Primers:<br/>Primer p1040: 5â-GTGGTTATCCTGCGTTGAT-3â<br/>Primer p1996: 5â-GAGCTGAGGCGGTATCT-3â<br/>p1578: 5â-CCGTGTGCACTTCGCTTCA-3â<br/>p1867: 5â-GCACAGCTTGGAGGCTTGA-3â<br/>p1583: 5â-TGCACTTCGCTTCACCT-3â<br/>p2301: 5â-AGGGGCATTTGGTGGTC-3â</li><li>Probe<br/>Probe p1085: 5â-FAM-AGTTGGCGAGAAAGTGAAAGCCTGC-TAMRA-3â</li><li>Trypsin [0.25% Trypsin-EDTA (1x)] (Invitrogen, Gibco, catalog number: 25200-056), 4 Â°C</li><li>NucleoSpin Tissue kit (Macherey-Nagel, catalog number: 740952.250)</li><li>Buffer T1 (Macherey-Nagel, catalog number: 740940.25)</li><li>Proteinase K (Macherey-Nagel, catalog number: 740506), -20 Â°C</li><li>Buffer B3 (Macherey-Nagel, catalog number: 740920)</li><li>Ethanol absolute (VWR chemicals, catalog number: 20821.330)</li><li>Buffer BW (Macherey-Nagel, catalog number: 740922)</li><li>Buffer B5 Concentrate (Macherey-Nagel, catalog number: 740921)</li><li>T5 exonuclease (New England Biolabs, catalog number: M0363), -20 Â°C</li><li>NEBuffer 4 (New England Biolabs, catalog number: B7004), -20 Â°C in aliquots</li><li>PerfeCTa qPCR Toughmix (Quanta Biosciences, catalog number: 95112-012), -20 Â°C in aliquots</li><li>SYBR Green Supermix (Bio-Rad, catalog number: 172-5121), -20 Â°C in aliquots</li><li>Myrcludex B (Bachem), available upon request from the corresponding author, -80 Â°C in aliquots</li><li>Recombinant human Interferon-Î±-2a (PeproTech, catalog number: 300-02AA), -80 Â°C in aliquots</li><li>Human Interferon-Î±-2a (PBL Assay Science, catalog number: 11100-1), -80 Â°C in aliquots</li><li>Nuclease-free water (B. Braun Melsungen, Aqua ad iniectabilia Braun)</li><li>NaCl (Carl Roth GmbH, catalog number: 9265.2)</li><li>KCl (Sigma-Aldrich, catalog number: 31248)</li><li>Na<sub>2</sub>HPO<sub>4</sub>Â·2H<sub>2</sub>O (Sigma-Aldrich, catalog number: 04272)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Sigma-Aldrich, catalog number: P9791)</li><li>Tris-base (Carl Roth GmbH, catalog number: 4855.2)</li><li>Dimethyl sulfoxide (Sigma-Aldrich, catalog number: 1.02950)</li><li>Fetal bovine serum gold (PAA Laboratories GmbH, catalog number: A15-151), -20 Â°C</li><li>Penicillin/Streptomycin (Thermo Fisher Scientific, catalog number: 15140-122), 4 Â°C</li><li>L-glutamine (Thermo Fisher Scientific, catalog number: 25030-024), 4 Â°C</li><li>MEM non-essential amino acids (Thermo Fisher Scientific, catalog number: 11140-035), 4 Â°C</li><li>Recombinant insulin (Sigma-Aldrich, catalog number: 91077C-1G), -20 Â°C in aliquots</li><li>Hydrocortisone 21-hemisuccinate sodium salt (Sigma-Aldrich, catalog number: H4881), -20 Â°C in aliquots</li><li>Culture medium (see Recipes)</li><li>10x PBS buffer (see Recipes)</li><li>5 mM Tris-HCl (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27391">###  Equipment</p><ol class="npwtwok"><li>Pipettes (Gilson P1000, P200, P20; Eppendorf P2.5)</li><li>Cell counter (Bio-Rad, TC20<sup>TM</sup> Automated Cell Counter, catalog number: 1450102)</li><li>Thermomixer (Eppendorf, ThermoMixer C, catalog number: 5382000015)</li><li>Microcentrifuge (Thermo Fisher Scientific, Heraeus<sup>TM</sup> Pico<sup>TM</sup> 21 Centrifuge, catalog number: 75002553)</li><li>Thermocycler (Labrepco, Biometra T3000 Thermocycler 48, catalog number: 050-723)</li><li>qPCR thermocycler (Bio-Rad, C1000 Touch<sup>TM</sup> Thermal cycler with 96-well Fast Reaction Module, catalog number: 1851196)<br/></li></ol><p class="pptt" id="biaoti27392">###  Software</p><ol class="npwtwok"><li>CFX96 Real-time System<br/>(Bio-Rad, <a href="http://www.bio-rad.com/en-us/product/cfx96-touch-real-time-pcr-detection-system" target="_blank">http://www.bio-rad.com/en-us/product/cfx96-touch-real-time-pcr-detection-system</a>)</li><li>CFX Manager<sup>TM</sup> Software <br/>(Bio-Rad, <a href="http://www.bio-rad.com/en-us/sku/1845000-cfx-manager-software?ID=1845000" target="_blank">http://www.bio-rad.com/en-us/sku/1845000-cfx-manager-software?ID=1845000</a>)<br/></li></ol><p class="pptt" id="biaoti27393">###  Procedure</p><p class="ppzz"><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331072831_4227.jpg" width="150"/><br/><strong>Figure 1. Working flowchart of this protocol.</strong> Susceptible cells (PHH, HepaRG<sup>NTCP</sup>, HepG2<sup>NTCP</sup>) infected with high mges (&gt; 300) of HBV are lysed. A. Lysates are incubated at 70 Â°C in the presence of proteinase K and later loaded on silica column and eluted. B. T5 exonuclease (5 units/1 h) removes host genomic DNA (black) and HBV replicative intermediates (rcDNA, dslDNA, <em>etc</em>.) (green) and preserves cccDNA (red) intact. C. cccDNA is amplified by p1040/p1996 and detected by probe p1085.</p><br/><ol class="npwone"><li>HBV DNA extraction from <em>in vitro</em> infected hepatocytes (Figure 1A)<br/><ol class="npwtwo"><li>For infection performed in a 24-well plate (culture area: 2 cm<sup>2</sup>), trypsinize and resuspend infected PHH, differentiated HepaRG<sup>NTCP</sup> or HepG2<sup>NTCP</sup> cells (infection procedure see Ni and Urban, 2017) in 1 ml of culture medium. Take 10 Î¼l of the resuspension for cell count by a cell counter (see Equipment). Calculate total cell numbers.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>At confluency, cells in one well of a 24-well plate approximately correspond to 180,000 PHH, 200,000 HepaRG<sup>NTCP</sup> and 500,000 HepG2<sup>NTCP</sup> cells.</em> </li><li><em>When smaller or larger plates are used, volumes of the respective buffers given in the protocol should be adjusted according to the respective culture area (cm<sup>2</sup>) of a well.</em> </li></ol></li><li>Spin down the cells (900 <em>x g</em>, 5 min) at room temperature. Wash cells with 1 ml of PBS twice. Spin down the cells and carefully aspirate PBS (Optional: freeze cell pellet at -20 Â°C for long-term storage). For DNA extraction, manufacturerâs manual of the NucleoSpin Tissue kit (740952.250) is adapted with a minor modification regarding the incubation time in Step A2 (1 h instead of 10~15 min) and the elution volume in Step A6 (50 Î¼l instead of 100 Î¼l). Add 200 Î¼l of lysis buffer T1, pipet up and down and incubate at room temperature for 10 min. Add 25 Î¼l of proteinase K and 200 Î¼l of lysis buffer B3, vortex and incubate the lysate at 70 Â°C for 1 h with shaking (500 rpm).<br/><em>Note: After proteinase K digestion, it is not necessary to re-centrifuge the lysates. Immediately perform the next step.</em> </li><li>Add 210 Î¼l of ethanol (96%-100%) to the sample and vortex vigorously for a few seconds. Remove buffer from the lid by a short centrifugation and apply the samples to the column. Centrifuge (11,000 <em>x g</em>, 1 min) at room temperature. Discard flow-through.</li><li>Add 500 Î¼l of wash buffer BW. Centrifuge (11,000 <em>x g</em>, 1 min) at room temperature. Discard flow-through.</li><li>Add 600 Î¼l of wash buffer B5 (pre-add indicated volume of ethanol to the Buffer B5 Concentrate) to the column. Centrifuge (11,000 <em>x g</em>, 1 min) at room temperature. Discard flow-through. Repeat this step once.</li><li>Remove residual ethanol from the column (13,000 <em>x g</em>, 1 min). Place the column into a 1.5 ml microcentrifuge tube. Add 50 Î¼l of pre-warmed (70 Â°C) elution buffer. Incubate at room temperature for 1 min. Centrifuge (11,000 <em>x g</em>, 1 min). Freeze eluted DNA samples at -20 Â°C until use or analyze them immediately.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Elution buffers are nuclease-free water or 5 mM Tris-HCl (pH = 8.0). Do not use buffers with EDTA that may inhibit enzymatic activity in the next step.</em> </li><li><em>DNA concentration in eluates is 200-1,000 ng/Î¼l.</em> </li><li><em>Preserve at least 10 Î¼l of the sample without T5 exonuclease digestion for quantification of Î²-globin (internal standard) for normalization use.</em> </li></ol></li></ol><br/></li><li>T5 exonuclease hydrolysis of total DNA elutes (Figure 1B)<br/><ol class="npwtwo"><li>Assemble the reaction components in a 0.2 ml microcentrifuge tube following the table as shown:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073520_2209.jpg" width="350"/><br/><br/></li></ol><p class="ppzz" style="margin-top:-10px;"><em>Note: Do not exceed unit and incubation time of T5 exonuclease since overdigestion leads to partial loss of cccDNA.</em></p> <ol class="npwtwo" start="2"><li>Incubate the reaction at 37 Â°C for 1 h and inactivate the enzyme at 70 Â°C for 20 min in a thermocycler with heat-lid supply.</li><li>Proceed quantification by real-time qPCR or freeze the products at -20 Â°C until use.<br/></li></ol><li>Quantification of cccDNA relative to human Î²-globin (single-copy gene) by real-time qPCR (Figure 1C)<br/><ol class="npwtwo"><li>Quantification of cccDNA<br/><ol class="npwthree"><li>Dilute primers and probe to 10 Î¼M:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190403/20190403003059_7446.jpg" width="500"/><br/>FAM: 6-carboxyfluorescein; TAMRA: 6-carboxytetramethylrhodamine<br/><br/></li><li>Prepare the following qPCR reactions in a 96-well Hard-Shell PCR plate:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073628_0473.jpg" width="500"/><br/><sup>#</sup>for absolute quantification, load 2 Âµl of each 1:10 serial-diluted plasmid (pSHH2.1: 10<sup>9</sup>, 10<sup>8</sup>, 10<sup>7</sup>, 10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 10<sup>2</sup> copies/Î¼l) as templates on the same plate.<br/><br/></li><li>Run qPCR program below:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073654_0080.jpg" width="460"/><br/><br/></li><li>Read absolute cccDNA copy numbers in the CFX Manager<sup>TM</sup> Software. Calculate cccDNA copies per well.</li></ol></li><li>Quantification of human Î²-globin prior to T5 exonuclease digestion<br/><ol class="npwthree"><li>Dilute primers at 10 Î¼M:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073737_8594.jpg" width="450"/><br/><br/></li><li>Prepare qPCR reactions in a 96-well plate:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073829_9051.jpg" width="460"/><br/><br/></li><li>Start qPCR program below:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073853_5743.jpg" width="460"/><br/><br/></li><li>Read levels of human Î²-globin. Normalize cccDNA copies per well using values of human Î²-globin. Calculate cccDNA copies per cell (copies per well divided by cell numbers per well). For instance, if cell numbers in a well of 24-well plate are 4.2 x 10<sup>5</sup> (as determined in Procedure A1) and cccDNA copies per well are 8.52 x 10<sup>5</sup> (as determined in Procedure C), cccDNA copies per cell are 2.03.</li></ol></li></ol></li></li></ol><p class="pptt" id="biaoti27394">###  Data analysis</p><p class="ppzz">cccDNA copy number per cell or infected cell is calculated according to Figure 2. The cccDNA level in <em>in vitro</em> infected cells can be determined similarly as two examples shown in Figure 3 and Qu <em>et al.</em>, 2018. Note that to discriminate signals from the inoculum, we highly recommend using an entry inhibitor (<em>e.g.</em>, Myrcludex B) as a control. Routinely Myrcludex B control gives a value below 0.05 copy per infected cell.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073917_4214.jpg" width="105"/><br/><strong>Figure 2. Schematic diagram of the calculation of absolute cccDNA copy numbers.</strong><span style="font-size:13.3333px;"> </span>A<span id="__kindeditor_bookmark_end_43__" style="display:none;"></span>. On the day of experiment, cell number in the well is determined. B. Total DNA samples are extracted as suggested in Procedure. C. Here shows a proper T5 exonuclease digestion as shown in Procedure. D. Absolute cccDNA copies and E level of human Î²-globin are quantified, respectively as shown in Procedure. âcccDNA copies/cellâ is âcccDNA copies/wellâ divided by âcell numbersâ. Optional: F. If calculation of copies per infected cell is required, additional wells have to be arranged in parallel on the same plate during the infection. On the day of DNA extraction, cells in the wells are fixed and subjected to an immunofluorescence assay (HBcAg visualization using DAKO B0586 antibody, <em>etc</em>.) to determine infectivity (%: number of HBcAg positive cells divided by number of total cells) (Qu <em>et al.</em>, 2018). âcccDNA copies/infected cellâ is the âcccDNA copies/cellâ divided by âinfectivity (%)â.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331073934_1877.jpg" width="105"/><br/><strong>Figure 3. cccDNA levels in three <em>in vitro</em>. models and upon IFN-Î±-2a and Myrcludex B treatment.</strong> A. PHH, differentiated HepaRG<sup>NTCP</sup> and HepG2<sup>NTCP</sup> cells were infected with HBV at mges/cell of 500. Myrcludex B (1 Î¼M), an entry inhibitor blocking cccDNA formation, was co-administered with HBV inoculum during infection. On Day 7 post infection, cccDNA copies per infected cell were determined. Data shown in triangles were collected from three independent experiments. B. HepG2<sup>NTCP</sup> cells were infected with HBV at mges/cell of 500 with mock treatment (untreated), co-treated with Myrcludex B (1 Î¼M) during infection, or co- and post-treated with IFN-Î±-2A at 100 ng/ml purchased from PeproTech and PBL Assay Science. On Day 7 post infection, cccDNA copies per infected cell were analyzed. Statistics: <em>P</em> &lt; 0.01 untreated versus IFN-treated; <em>P</em> = 0.828 PeproTech versus PBL.</p><p class="pptt" id="biaoti27395">###  Notes</p><p class="ppzz">The primer pair (pp1040-1996) is specific for the formed cccDNA in genotype D. Genotype D is the HBV genome in HepAD38 cell line. Users should consider this issue if they try patient-derived serum to perform <em>in vitro</em> infection using this protocol since HBV in patients can be any genotype and pp1040-1996 does not bind the formed cccDNA of other genotypes. <br/>   Genbank accession numbers of other genotypes: A (HE974370.1); B (AB540582.1); C (AB540584.1); E (HE974384.1); F (HE974369.1); G (AP007264.1); H (AB846650.1). pp1578-1867 and pp1583-2301 (Qu <em>et al.</em>, 2018) allowing quantification of eight HBV genotypes (A-H) are suggested below. (Table 1)</p><br/><p class="ppzz"><strong>Table 1. List of primer mismatches to HBV genotypes.</strong> The numbers of base-pair mismatch of each forward and reverse primers to eight HBV genotypes are summarized below. Green squares: perfect binding; yellow squares: one base-pair mismatch still allowing template binding; red squares: not binding.</p><p class="ppzz"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190331/20190331074014_2811.jpg" width="600"/><br/>F: forward primer; R: reverse primer. #: The probe of this protocol is not pan-genotypic and has mismatch number as shown: A(1), B(2), C(0), D(0), E(0), F(6), G(1), H(4).</p><p class="pptt" id="biaoti27396">###  Recipes</p><ol class="npwtwok"><li>Culture medium<br/>For HepG2<sup>NTCP</sup> cells: DMEM supplemented with 10% fetal calf serum (heat-inactivated), 100 U/ml penicillin, 100 Î¼g/ml streptomycin, 2 mM L-glutamine, 1% MEM non-essential amino acids. For infection, add 2% dimethyl sulfoxide<br/>For PHH and HepaRG<sup>NTCP</sup> cells: Williamâs medium E supplemented with 10% fetal calf serum (heat-inactivated), 100 U/ml penicillin, 100 Î¼g/ml streptomycin, 2 mM L-glutamine, 1% MEM non-essential amino acids, 5 Î¼g/ml recombinant insulin, 50 Î¼M hydrocortisone. For differentiation and infection, add 1.5% dimethyl sulfoxide</li><li>10x PBS<br/>Dissolve 80 g of NaCl, 2 g of KCl, 11.5 g of Na<sub>2</sub>HPO<sub>4</sub>Â·2H<sub>2</sub>O and 2 g of KH<sub>2</sub>PO<sub>4</sub> in 800 ml of H<sub>2</sub>O, adjust pH to 7.4, refill H<sub>2</sub>O to 1 L</li><li>Tris-HCl (1 M)<br/>Dissolve 121.14 g of Tris-base in 800 ml of H<sub>2</sub>O, adjust pH to 8.0 and refill H<sub>2</sub>O to 1 L<br/></li></ol><p class="pptt" id="biaoti27397">###  Acknowledgments</p><p class="ppzz">This protocol was adapted from Qu <em>et al.</em>, 2018. We herein acknowledge financial supports by Deutsche Forschungsgemeinschaft (DFG, German Research Foundation)âProject number 272983813âSFB/TRR179 (B.Q. and S.U.) and German Center for Infection Research (DZIF) TTU Hepatitis projects 05.704 (S.U.). We thank Pascal Mutz (DKFZ, Germany) and Florian WR Vondran (MHH, Germany) for providing PHH. We are grateful to Xue Li for critical reading of this manuscript.</p><p class="pptt" id="biaoti27398">###  Competing interests</p><p class="ppzz">Prof. Dr. Stephan Urban, the corresponding author, holds patents and intellectual property on Myrcludex B.</p><p class="pptt" id="biaoti27399">###  Ethics</p><p class="ppzz">Following written informed consent of the patients, PHH were isolated from liver specimens obtained after partial hepatectomy.</p><p class="pptt" id="biaoti27400">###  References</p><ol class="npwtwok"><li>Bartenschlager, R. and Schaller, H. (1992). <a href="https://www.ncbi.nlm.nih.gov/pubmed/1380455" target="_blank">Hepadnaviral assembly is initiated by polymerase binding to the encapsidation signal in the viral RNA genome.</a> <em>Embo j</em> 11(9): 3413-3420. </li><li>Cattaneo, R., Will, H., Darai, G., Pfaff, E. and Schaller, H. (1983). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC555053/" target="_blank">Detection of an element of the SV40 late promoter in vectors used for expression studies in COS cells.</a> <em>Embo j</em> 2(4): 511-514. </li><li>Guo, H., Jiang, D., Zhou, T., Cuconati, A., Block, T. M. and Guo, J. T. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17804499" target="_blank">Characterization of the intracellular deproteinized relaxed circular DNA of hepatitis B virus: an intermediate of covalently closed circular DNA formation.</a> <em>J Virol</em> 81(22): 12472-12484. </li><li>Lempp, F. A., Qu, B., Wang, Y. X. and Urban, S. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26865711" target="_blank">Hepatitis B virus infection of a mouse hepatic cell line reconstituted with human sodium taurocholate cotransporting polypeptide.</a> <em>J Virol</em> 90(9): 4827-4831. </li><li>Lempp, F. A., Wiedtke, E., Qu, B., Roques, P., Chemin, I., Vondran, F. W. R., Le Grand, R., Grimm, D. and Urban, S. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28195359" target="_blank">Sodium taurocholate cotransporting polypeptide is the limiting host factor of hepatitis B virus infection in macaque and pig hepatocytes.</a> <em>Hepatology</em> 66(3): 703-716. </li><li>Levrero, M., Testoni, B. and Zoulim, F. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27447092" target="_blank">HBV cure: why, how, when?</a> <em>Curr Opin Virol</em> 18: 135-143. </li><li>Li, X., Zhao, J., Yuan, Q. and Xia, N. (2017). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5490816/" target="_blank">Detection of HBV Covalently Closed Circular DNA.</a> <em>Viruses</em> 9(6): E139. </li><li>Liang, T. J., Block, T. M., McMahon, B. J., Ghany, M. G., Urban, S., Guo, J. T., Locarnini, S., Zoulim, F., Chang, K. M. and Lok, A. S. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26239691" target="_blank">Present and future therapies of hepatitis B: From discovery to cure.</a> <em>Hepatology</em> 62(6): 1893-1908. </li><li>Long, Q., Yan, R., Hu, J., Cai, D., Mitra, B., Kim, E. S., Marchetti, A., Zhang, H., Wang, S., Liu, Y., Huang, A. and Guo, H. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29287110" target="_blank">The role of host DNA ligases in hepadnavirus covalently closed circular DNA formation.</a> <em>PLoS Pathog</em> 13(12): e1006784. </li><li>Margeridon, S., Carrouee-Durantel, S., Chemin, I., Barraud, L., Zoulim, F., Trepo, C. and Kay, A. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18606836" target="_blank">Rolling circle amplification, a powerful tool for genetic and functional studies of complete hepatitis B virus genomes from low-level infections and for directly probing covalently closed circular DNA.</a> <em>Antimicrob Agents Chemother</em> 52(9): 3068-3073. </li><li>Mu, D., Yan, L., Tang, H. and Liao, Y. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26187315" target="_blank">A sensitive and accurate quantification method for the detection of hepatitis B virus covalently closed circular DNA by the application of a droplet digital polymerase chain reaction amplification system.</a> <em>B</em><em>iotech</em><em>nol Lett</em> 37(10): 2063-2073. </li><li>Nassal, M. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26048673" target="_blank">HBV cccDNA: viral persistence reservoir and key obstacle for a cure of chronic hepatitis B.</a> <em>Gut</em> 64(12): 1972-1984. </li><li>Ni, Y. and Urban, S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27975304" target="_blank">Hepatitis B Virus infection of HepaRG cells, HepaRG-hNTCP cells, and Primary Human Hepatocytes.</a> <em>Methods Mol Biol</em> 1540:15-25.</li><li>Qu, B., Ni, Y., Lempp, F. A., Vondran, F. W. R. and Urban, S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30232183" target="_blank">T5 exonuclease hydrolysis of hepatitis B virus replicative intermediates allows reliable quantification and fast drug efficacy testing of covalently closed circular DNA by PCR.</a> <em>J Vir</em><em>ol</em> 92(23). </li><li>Schreiner, S. and Nassal, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28531167" target="_blank">A Role for the Host DNA Damage Response in Hepatitis B Virus cccDNA Formation-and Beyond?</a> <em>Viruses</em> 9(5): E125.</li><li>Tu, T., Budzinska, M. A., Shackel, N. A. and Urban, S. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28394272" target="_blank">HBV DNA Integration: Molecular mechanisms and clinical implications.</a> <em>V</em><em>i</em><em>rus</em><em>es</em> 9(4): E75. </li><li>Xu, C. H., Li, Z. S., Dai, J. Y., Zhu, H. Y., Yu, J. W. and Lu, S. L. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21740808" target="_blank">Nested real-time quantitative polymerase chain reaction assay for detection of hepatitis B virus covalently closed circular DNA.</a> <em>Chin Med J </em><em>(Engl)</em> 124(10): 1513-1516. </li><li>Yu, M., Yang, H., Xie, X., Barauskas, O., Feng, J., Gamelin, L., Xu, S., Delaney, W. and Cheng, G. (2015). <a href="http://www.enyopharma.com/2015-international-hbv-meeting-the-molecular-biology-of-hepatitis-b-viruses/" target="_blank">2015 International HBV Meeting.</a> Poster number 43. October 4, 2015-October 8, 2015, Bad Nauheim, Germany.<br/></li></ol>