<body>
            <h1>Multiple Modification of Chromosomal Loci Using <em>URA5.3</em> Selection Marker in the Unicellular Red Alga <em>Cyanidioschyzon merolae</em></h1>
            <p><strong>Authors:</strong> Tokiaki Takemura, Sousuke Imamura, Yuki Kobayashi, Kan Tanaka</p>
            <h2>Abstract</h2>
            <p>The unicellular red alga <em>Cyanidioschyzon merolae</em> has been used as a eukaryotic photosynthetic model for various basic and applied studies. Although the nuclear genome of <em>C. merolae</em> can be modified by homologous recombination with exogenously introduced DNA, it has been difficult to modify multiple chromosome loci within the same strain because of the limited number of available positive selection markers. Recently, we reported a modified <em>URA5.3</em> gene cassette (<em>URA5.3T</em>), which can be used repeatedly for nuclear genome transformation using the pMKT plasmid vectors for epitope tagging (3x FLAG- or 3x Myc-) of nuclear-encoded proteins. In addition, these plasmid vectors can also be used to knock out multiple genes one by one. This report describes the construction of DNA fragments for transformation and the detailed transformation procedure.</p>
            <p><strong>Keywords:</strong> <em>Cyanidioschyzon merolae</em>, Epitope tagging, Gene knocking out, Homologous recombination, Selectable marker recycling, <em>URA5.3</em></p>
            <h2>Background</h2><p>The standard method of transformation in <em>C. merolae</em> uses the uracil-auxotrophic strains M4 (Minoda <em>et al.</em>, 2004) or T1 (Taki <em>et al.</em>, 2015), which harbor a frameshift or a complete deletion mutation in the gene <em>URA5.3</em> (<em>CMK046C</em>). The gene <em>URA5.3</em> encodes a protein composed of an orotate phosphor ribosyl transferase (OPRTase) domain and an orotidine-5'-phosphate decarboxylase (OMPdecase) domain, which synthesizes uridine-5'-monophosphate from orotate. The transformant is selected by introducing <em>URA5.3</em> as a selection marker into the uracil-auxotrophic strain. A recent study has reported the chloramphenicol resistance <em>CAT</em> gene as a second selectable marker gene for the nuclear transformation experiment in <em>C. merolae</em> (Fujiwara <em>et al.</em>, 2017). However, it is difficult to modify more than three genome loci. Thus, we developed the <em>URA5.3</em> selection marker gene recycling system using a modified <em>URA5.3</em> gene, named <em>URA5.3T</em> (Takemura <em>et al.</em>, 2018). The <em>URA5.3T</em> includes the <em>URA5.3</em> terminator sequence (462 bp) at the 5â end of the <em>URA5.3</em> promoter-terminator cassette to make homologous repeats, the epitope tag region which encodes the recognition sequence for three different proteases (TEV, Factor Xa and thrombin), and a 3x FLAG or 3x Myc tag-encoding sequence (Figure 1). The <em>URA5.3</em> promoter-terminator cassette is eliminated by frequently occurring DNA excision through the homologous repeat sequences located on both sides of <em>URA5.3T</em>. In this method, the <em>URA5.3</em> eliminated transformant is selected using 5-fluoroorotic acid (5-FOA), which is converted to cytotoxic 5-fluorouracil via <em>URA5.3</em> activity. The marker-eliminated transformant can be used for the next round transformation experiment using the <em>URA5.3T</em> marker gene (Figure 1). In this report, we present the procedure for construction of the transformation template DNA, which includes the recycling marker <em>URA5.3T</em>, for the purpose of gene tagging or gene knock out.<br/><br/><img alt="" class="layerphoto" height="112" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190329/20190329000428_5115.jpg" width="130"/><br/><strong>Figure 1. Outline from transformation to <em>URA5.3</em> marker elimination using <em>URA5.3T</em>.</strong> The DNA includes the homologous regions for the genome-specific locus (grey box) and the <em>URA5.3T</em>. The <em>URA5.3T</em> includes the epitope tag region, a homologous repeat sequence (red hatched box) and the <em>URA5.3</em> promoter-terminator cassette. The epitope tag region contains the recognition sequence for three different proteases (TEV, Factor Xa and thrombin) and a 3x FLAG or 3x Myc tag-encoding sequence.</p>
        </body>

<p class="pptt" id="biaoti27370">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (NIPPON Genetics Co., Ltd., catalog numbers: FG-102, FG-301, FG-401)</li><li>Toothpicks</li><li>Cuvettes (As One Corporation., catalog number: 1-2855-02)</li><li>1.5 ml Microtube (BM Equipment Co., Ltd., catalog number: NT-175)</li><li>50 ml Centrifuge tube (Corning Incorporated, catalog number: 352196)</li><li>24-well tissue culture test plate (BM Equipment Co., Ltd., catalog number: 92424)</li><li>Sterilized plastic plate (Î¦ 90 mm) (As One Corporation, catalog number: GD90-15)</li><li>AnaeroPouch (Mitsubishi Gas, catalog number: 2-3764-02)</li><li>AnaeroPack (Mitsubishi Gas, catalog number: 2-3765-01)</li><li><em>C. merolae</em> T1 strain (Taki <em>et al.</em>, 2015)</li><li><em>Escherichia coli</em> DH5Î± (Takara Bio, catalog number: 9057)</li><li>Wild type <em>C. merolae</em> DNA</li><li>Salmon sperm DNA (FUJIFILM Wako Pure Chemical Industries Corporation, catalog number: 047-17322)</li><li>Plasmid vector pMKT (Takemura <em>et al.</em>, 2018; available upon request to authors)</li><li>KOD-Plus-Neo DNA polymerase (Toyobo, catalog number: KOD-401)</li><li>GoTaq DNA polymerase (Promega, catalog number: M7123)</li><li><em>Stu</em>I (NEB, catalog number: R0187S)</li><li><em>Dpn</em>I (Takara Bio, catalog number: 1235A)</li><li>Specific primers [set for amplifying the Upstream and Downstream fragments with the homologous sequence for the <em>Stu</em>I-digested pMKT (see Figure 2A), ordered at Integrated DNA Technologies (IDT) Inc.]</li><li>Confirmation primer sets (annealing to the outside sequence of the homologous regions, see Figure 3, ordered at Integrated DNA Technologies (IDT) Inc.)</li><li>Primer sets for colony PCR (ordered at Integrated DNA Technologies (IDT) Inc.):<br/>Forward primer for up: 5â-CGAGGTCGACGGTATCGATAAGC-3â<br/>Reverse primer for up: 5â-GATCTAGTAACATAGATGACACCGC-3â<br/>Forward primer for down: 5â-GCTGCTAGGGATTGTGGCGCGA-3â<br/>Reverse primer for down: 5â-CCGCTCTAGAACTAGTGGATCCC-3'</li><li>In-fusion HD cloning kit (Takara Bio, catalog number: 639648)</li><li>Phenol, Crystals (FUJIFILM Wako Pure Chemical Corporation, catalog number: 169-22445)</li><li>Chloroform (FUJIFILM Wako Pure Chemical Corporation, catalog number: 035-02616)</li><li>Isoamyl alcohol (3-Methyl-1-butanol) (FUJIFILM Wako Pure Chemical Corporation, catalog number: 135-12015)</li><li>Trizma<sup>Â®</sup> base Primary Standard and Buffer, crystalline (Sigma-Aldrich, catalog number: T1503-500G)</li><li>HCl (FUJIFILM Wako Pure Chemical Corporation, catalog number: 081-03475)</li><li>Sodium dodecyl sulfate (SDS) (FUJIFILM Wako Pure Chemical Corporation, catalog number: 194-13985)</li><li>NaOH (FUJIFILM Wako Pure Chemical Corporation, catalog number:199-4895)</li><li>8-quinolinol (FUJIFILM Wako Pure Chemical Corporation, catalog number: 178-00752)</li><li>Agarose ME (Iwai Chemical Company, catalog number: 50013R)</li><li>Wizard SV gel and PCR clean-up system (Promega, catalog number: A9281)</li><li>Polyethylene glycol 4000 (FUJIFILM Wako Pure Chemical Corporation, catalog number: 5322-68-3)</li><li>Cornstarch (Kawamitsu-Bussan, catalog number: 4901 486 02701 6)</li><li>Top starch solution (Takemura <em>et al.</em>, 2019)</li><li>Gellan gum (FUJIFILM Wako Pure Chemical Corporation, catalog number: 073-03071)</li><li>Uracil (Sigma-Aldrich, catalog number: U0750-5G)</li><li>5-Fluoroorotic acid monohydrate (5-FOA) (FUJIFILM Wako Pure Chemical Corporation, catalog number: 066-03663)</li><li>Sodium acetate (FUJIFILM Wako Pure Chemical Corporation, catalog number: 190-01071)</li><li>Ethanol (FUJIFILM Wako Pure Chemical Corporation, catalog number: 055-00457)</li><li>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (FUJIFILM Wako Pure Chemical Corporation, catalog number: 019-03435)</li><li>MgSO<sub>4</sub>â7H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 138-00415)</li><li>H<sub>2</sub>SO<sub>4</sub> (FUJIFILM Wako Pure Chemical Corporation, catalog number: 192-04696)</li><li>H<sub>3</sub>BO<sub>3</sub> (FUJIFILM Wako Pure Chemical Corporation, catalog number: 021-02195)</li><li>MnCl<sub>2</sub>â4H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 133-00725)</li><li>ZnCl<sub>2</sub> (FUJIFILM Wako Pure Chemical Corporation, catalog number: 268-01022)</li><li>Na<sub>2</sub>MoO<sub>4</sub>â2H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 198-02471)</li><li>CoCl<sub>2</sub>â6H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 038-03681)</li><li>CuCl<sub>2</sub>â2H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 039-04135)</li><li>KH<sub>2</sub>PO<sub>4</sub> (FUJIFILM Wako Pure Chemical Corporation, catalog number: 169-04245)</li><li>CaCl<sub>2</sub>â2H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 038-19735)</li><li>FeCl<sub>3</sub>â6H<sub>2</sub>O (FUJIFILM Wako Pure Chemical Corporation, catalog number: 090-02802)</li><li>Na<sub>2</sub>EDTA (FUJIFILM Wako Pure Chemical Corporation, catalog number: 345-01865)</li><li>DNA solution (see Recipes)</li><li>PEG solution (see Recipes)</li><li>MA2 medium (Kobayashi <em>et al.</em>, 2010, see Recipes)<br/><ol class="npwthree"><li>MA2 solution I</li><li>A6 minor salts</li><li>MA2 solution II</li><li>MA2 solution III</li><li>MA2 solution IV</li></ol></li><li>MA2 solid gellan gum plate (see Recipes)</li><li>DNA extraction buffer (see Recipes)</li><li>Neutralized phenol (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27371">### Equipment</p><ol class="npwtwok"><li>Pipettes (Gilson, Inc., catalog numbers: F123600, F123601, F123602)</li><li>Refrigerated centrifuge (Koki Holdings Co., Ltd., model: CF16RXII)</li><li>Microcentrifuge (TOMY Seiko Co., Ltd., model: MX150)</li><li>500 ml flask (IWAI, catalog number: 4442FK500)</li><li>Heat Block (Chiyoda Science Co., Ltd., model: MiniT-100)</li><li>Bio incubator (TOMY Seiko Co., Ltd., model: CLE-303)</li><li>NanoDrop (Thermo Fisher Scientific Inc., model: ND-LITE)</li><li>Thermal Cycler (Bio-Rad Laboratories, Inc., model: T100 Thermal Cycler)</li><li>Electrophoresis apparatus (Atto Corporation, model: WSE-1710)</li><li>FAS gel imager (Nippon Genetics Co., Ltd., model: FAS-IV)</li><li>Spectrophotometer (Beckman coulter, Inc., model: DU730, A23616)<br/></li></ol><p class="pptt" id="biaoti27372">### Procedure</p><ol class="npwone"><li>Construction of the introducing DNA fragment<br/>Overview of the construction: PCR-amplified DNA fragments (Upstream and Downstream of homologous regions for the genome-specific locus) and <em>Stu</em>I-digested pMKT (including two fragments Digested vector L and S) are connected using the In-fusion system to generate the pMKT-UPandDOWN. The resultant plasmid is used as the template DNA for amplification of the introducing DNA fragment by PCR (Figure 2).<br/>PCR amplification of DNA fragments (Figure 2A):<ol class="npwtwo"><li>Specific primer sets are designed as given below to amplify the DNA fragments (500-1,500 bp each) for homologous recombination to the genome-specific locus. Upstream and Downstream DNA fragments do not need to share the same length. For tagging, the upstream DNA fragment should include an open reading frame devoid of the termination codon, which is fused in frame to the epitope tag. Each colored region (20 bp) is the homologous sequence of the end of relevant <em>Stu</em>I-digested pMKT (The colors correspond to Figure 2A) and each repeated âNsâ (20-25 bp) indicates the annealing sequence to amplify the Upstream and Downstream DNA fragments. The T<sub>m</sub> of the annealing sequence (âNsâ) of each primer should be designed to be above 63 Â°C.<br/>Upstream Forward (UF):<br/>5â-<span style="color:#E53333;">GACACGTGAATTTAAATAGG</span>NNNNNNNNNNNNNNNNNNNN-3â<br/>Upstream Reverse (UR):<br/>5â-<span style="color:#337FE5;">TGAAAATAAAGATTTTCAGG</span>NNNNNNNNNNNNNNNNNNNN-3â<br/>Downstream Forward (DF):<br/>5â-<span style="color:#009900;">AGGGATTGTGGCGCGACAGG</span>NNNNNNNNNNNNNNNNNNNN-3â<br/>Downstream Reverse (DR):<br/>5â-<span style="color:#FF9900;">GACACGTGAATTTAAATAGG</span>NNNNNNNNNNNNNNNNNNNN-3â</li><li>PCR reactions are carried out using KOD-Plus-Neo DNA polymerase under the following reaction conditions: 94 Â°C for 2 min, followed by 30 cycles of 10 s at 98 Â°C, 30 s/1 kb at 68 Â°C, and a final extension of 2 min/1 kb at 68 Â°C with specific primers sets (UF/UR for Upstream and DF/DR for Downstream) and 50 ng wild type <em>C. merolae</em> DNA as a template. PCR-amplified DNA fragments are purified using the Wizard SV Gel and PCR Clean-Up System following the manufacturerâs protocol, and the concentration of DNA is measured.<br/><em>Note: We routinely obtain reproducible amplification results by the above condition. The PCR results should be checked by gel electrophoresis to confirm the amplification.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>pMKT vector digestion by <em>Stu</em>I to obtain two DNA fragments, here named Digested vector L (Long, approximately 3.6 kbp) and S (Short, approximately 3.0 kbp, Figure 2B)<ol class="npwtwo"><li>Incubate 1 Î¼g of pMKT DNA in 50 Î¼l 1x CutSmart Buffer (<em>i.e.</em>, provided buffer of <em>Stu</em>I) containing 10 units of <em>Stu</em>I at 37 Â°C for 1 h.</li><li>Add 100 Î¼l of dH<sub>2</sub>O and 150 Î¼l of Phenol:Chloroform:Isoamyl alcohol = 25:24:1 solution and mix vigorously. Centrifuge at 12,000 <em>x g</em> for 10 min at room temperature. Take 150 Î¼l of supernatant and add 15 Î¼l of 3 M sodium acetate and 375 Î¼l of 100% EtOH. Mix vigorously and centrifuge at 12,000 <em>x g</em> for 30 min at 4 Â°C. Discard the supernatant, add 75% EtOH to the precipitate, and centrifuge at 12,000 <em>x g</em> for 5 min at 4 Â°C. Discard the supernatant, completely remove the remaining drops by pipette after flush centrifugation, and air dry the pellet briefly on the bench at room temperature. Then add 20 Î¼l of dH<sub>2</sub>O and dissolve the precipitated DNA. Measure the DNA concentration.<br/><em>Note: When using phenol-chloroform-isoamyl alcohol, personal protective equipment needs to be used according to local health and safety regulations. Always work under a fume hood.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>In-fusion reaction to generate the template plasmid (Figure 2C)<ol class="npwtwo"><li>Gently mix 5 Î¼l of DNA mixture [<em>Stu</em>I-digested pMKT (Digested vector L and S):upstream:downstream â 100 fmol:50 fmol:50 fmol] and 5 Î¼l of 2x In-fusion reaction mix. Incubate the reaction mixture at 50 Â°C for 15 min and then cool it down on ice.</li><li>Transform <em>Escherichia coli</em> DH5Î± with the reaction mixture by using a chemical transformation method as common use. Find the expected plasmid construct by colony PCR. PCR reactions are carried out using GoTaq DNA polymerase under the following reaction conditions: 95 Â°C for 3 min, followed by 30 cycles of 30 s at 95 Â°C, 30 s at 55 Â°C, 1 min/1 kb at 72 Â°C, and a final extension of 2 min at 72 Â°C with primers sets (Forward primer for up: 5â-CGAGGTCGACGGTATCGATAAGC-3â/Reverse primer for up: 5â-GATCTAGTAACATAGATGACACCGC-3â for the upstream sequence, and Forward primer for down: 5â-GCTGCTAGGGATTGTGGCGCGA-3â/Reverse primer for down: 5â-CCGCTCTAGAACTAGTGGATCCC-3â for the downstream sequence). Subsequently, the positive candidates are examined by DNA sequence analysis with primer sets for colony PCR. </li><li>Extract and purify the plasmid DNA (pMKT-UPandDOWN).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Amplification of DNA for transformation by PCR to prepare the linearized DNA fragment and completely remove the template circular plasmid DNA (Figure 2D)<br/><em>Note: Inclusion of the template plasmid DNA frequently may result in random integration of the plasmid into the genome. Thus, the template plasmid DNA should be carefully removed.</em><br/><br/><img alt="" class="layerphoto" height="144" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225057_3794.jpg" width="100"/><br/><strong>Figure 2. Overview of the template vector (pMKT-UPandDOWN) and DNA constructs for transformation.</strong> A. DNA regions (Upstream and Downstream) corresponding to the target gene are amplified by PCR with the specific primer sets having the 20 bp homologous sequence for the pMKT (shown as red, blue, green and yellow). B. Digestion of the pMKT vector with <em>Stu</em>I enzyme generates 2 DNA fragments named Digested vectors L and S after their lengths (L-long; S-short). The red hatched areas indicate the homologous repeat sequence (<em>i.e., URA5.3</em> terminator region). C. Four DNA fragments (Digested vector L, Digested vector S, Upstream and Downstream) are connected in one reaction tube using the In-fusion system, to make a plasmid pMKT-UPandDOWN). D. The DNA fragment for transformation is amplified by PCR with the primer set UF and DR using pMKT-UPandDOWN as the template.<br/><br/><ol class="npwtwo"><li>PCR is carried out as described in Step A2 with the specific primer sets (Upstream Forward: UF and Downstream Reverse: DR) and pMKT-UPandDOWN as the template. </li><li>To digest the plasmid DNA, treat the amplified PCR mix with 10 units of <em>Dpn</em>I at 37 Â°C for 1 h. </li><li>Purify the specific band using the Wizard SV Gel and PCR Clean-Up System after 1.2% agarose electrophoresis (applying a voltage of 100V for 30 min). Prepare 2-5 Î¼g of DNA fragments for transformation in 40 Î¼l of dH<sub>2</sub>O.</li></ol><br/></li><li>Transformation of <em>C. merolae</em> cells<br/>Transform the <em>C. merolae</em> T1 cells following the polyethylene glycol-mediated protocol as described by Ohnuma <em>et al.</em> (2008) and Fujiwara and Ohnuma (2018).<ol class="npwtwo"><li>When OD<sub>750</sub> reaches ~10, dilute the cell culture in 50 ml MA2 medium to yield an OD<sub>750</sub> of 0.4 and cultivate in the glass cylinders (3 cm in diameter and 35 cm in length) with cotton plug bubbled with 2% CO<sub>2</sub> in air under constant light (20 Î¼mol photons m<sup>-2</sup> s<sup>-1</sup>) at 40 Â°C for 20-24 h.</li><li>Harvest the cells (OD<sub>750</sub> &lt; 0.7) by centrifugation (1,200 <em>x g</em>, for 5 min at 40 Â°C), and discard the supernatant. Resuspend the pellet with 200-300 Î¼l of MA2 medium.</li><li>Combine 2-5 Î¼g of the DNA fragment in 40 Î¼l of dH<sub>2</sub>O and 60 Î¼l of DNA solution (see Recipes).</li><li>Add 25 Î¼l of the resuspended cells to the tube and mix gently.</li><li>Add 125 Î¼l of the PEG solution (see Recipes) immediately and mix the liquid by gently pipetting 4-5 times using the 200 Î¼l pipette.</li><li>Transfer the mixture to 30-40 ml of MA2 medium and incubate under the same condition as described in Step E1 for 20-24 h.</li><li>Harvest the cultured cells by centrifugation (1,200 <em>x g</em>, for 5 min at 40 Â°C) and spread the cells onto the MA2 solid gellan gum plate without uracil and 5-FOA by the top starch method to increase the plating efficiency (Takemura <em>et al.</em>, 2019). Incubate the plate at 40 Â°C supplemented with 5% CO<sub>2</sub> in AnaeroPack with AnaeroPouch. </li><li>After 2-4 weeks, colonies will appear on the surface of the MA2 solid gellan gum plate.</li><li>Pick up well-separated colonies (usually 20-24 candidates) using sterilized toothpicks, and inoculate 2 ml of MA2 liquid medium prepared in a 24-well tissue culture test plate.</li><li>After incubation for 1-2 weeks in the same incubator without agitation, harvest the 20-24 candidate colonies and extract the DNA as described below (Kobayashi <em>et al.</em>, 2010). To select the positive transformants, perform the PCR with the DNA using the ââConfirmation primer setsââ to anneal to the outside sequence of the homologous regions used to introduce the DNA fragment (Figure 3). PCR reactions are carried out using KOD-Plus-Neo DNA polymerase under the following reaction conditions: 94 Â°C for 2 min, followed by 30 cycles of 10 s at 98 Â°C, 30 s/1 kb at 68 Â°C, and a final extension of 2 min/1 kb at 68 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>DNA extraction from <em>C. merolae</em> cells<ol class="npwtwo"><li>Harvest <em>C. merolae</em> cells by centrifugation at 3,000 <em>x g</em> for 5 min at 4 Â°C, and discard the medium.</li><li>Resuspend the cell pellets with 400 Î¼l of DNA extraction buffer [50 mM Tris-HCl (pH = 8.0), 5 mM EDTA, 0.5% Sodium dodecyl sulfate].</li><li>Add 400 Î¼l of neutralized phenol.</li><li>Incubate at 65 Â°C for 10 min with shaking every 2 min by hand vigorously.<br/><em>Note: A vortex should not be used.</em> </li><li>Centrifuge at 12,000 <em>x g</em> for 5 min at room temperature. Transfer 375 Î¼l of the above aqueous phase into a fresh tube, and add 375 Î¼l of PCI (neutralized phenol:chloroform:isoamyl alcohol = 25:24:1). Invert the tube by hand vigorously.</li><li>Centrifuge at 12,000 <em>x g</em> for 5 min at room temperature. Transfer 350 Î¼l of the above aqueous phase into a fresh tube, and add 350 Î¼l of PCI. Invert the tube by hand vigorously.</li><li>Centrifuge at 12,000 <em>x g</em> for 5 min at room temperature. Transfer 300 Î¼l of the above aqueous phase into a fresh tube, and add 30 Î¼l of the 3 M sodium acetate and 750 Î¼l of 100% ethanol. Invert the tube by hand vigorously.</li><li>Centrifuge at 12,000 <em>x g</em> for 30 min at 4 Â°C. Discard the supernatant and add 200 Î¼l of 75% ethanol.</li><li>Centrifuge at 12,000 <em>x g</em> for 5 min at 4 Â°C. Discard the supernatant and let the liquid evaporate completely.</li><li>Dissolve the precipitate of DNA in distilled water.<br/><em>Note: When using phenol-chloroform-isoamyl alcohol, personal protective equipment needs to be used according to local health and safety regulations. Always work under a fume hood.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Elimination of <em>URA5.3</em> gene marker from the genome<ol class="npwtwo"><li>The transformant cell using <em>URA5.3T</em> as a gene selectable marker is cultured in 50 ml of MA2 medium until the OD<sub>750</sub> reaches 0.5-5.0.</li><li>Centrifuge 50 ml of the culture at 1,200 <em>x g</em> for 5 min at 40 Â°C.</li><li>Discard the supernatant, add 1.5 ml of MA2 medium and resuspend the precipitated cells.</li><li>Add 1 ml of the top starch solution (Takemura <em>et al.</em>, 2019) to 1 ml of the cell suspension, mix gently, and spread onto the MA2 solid gellan gum plate containing 0.5 mg/ml uracil and 0.8 mg/ml 5-FOA. </li><li>Add 1 ml of MA2 medium to the remaining cell suspension. Take 1 ml of the diluted cell suspension, and spread onto the MA2 solid gellan gum plate as in Step F4. </li><li>After 2-4 weeks, the colonies of the <em>URA5.3</em> eliminated cells appear on the MA2 solid gellan gum plate.</li><li>Pick up well-separated colonies (usually 20-24 candidates) using sterilized toothpicks, and inoculate each in 2 ml of MA2 liquid medium prepared in 24-well tissue culture test plate.</li><li>After incubation for 1-2 weeks in the same incubator, extract DNA from each green colored inoculate as described in Step E10. To confirm <em>URA5.3</em> elimination, perform the PCR with the DNA using the âConfirmation primer setsâ to anneal to the outside sequence of the homologous regions used to introduce the DNA fragment as described in Step E10 (Figure 3).</li><li>After confirmation of <em>URA5.3</em> elimination by PCR, the strain is ready for the next transformation experiment using <em>URA5.3T</em> as a gene selectable marker.<br/><br/><img alt="" class="layerphoto" height="101" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225200_5540.jpg" width="100"/><br/><strong>Figure 3. Schematic representation of confirmation of the transformation and </strong><em><strong>URA5.3</strong></em><strong> elimination by PCR.</strong> A. Schematic representation of the modified gene loci of the host strain, 1st transformant and <em>URA5.3</em> eliminated transformant. The arrows indicate the specific primer sets for confirmation of the transformation (named âConfirmation primer setsâ). These primers should be designed to anneal to the outside sequence of the homologous regions. B. Example of gel electrophoresis image of PCR to confirm the expected recombination events using confirmation primer sets. </li></ol></li></ol><p class="pptt" id="biaoti27373">### Recipes</p><ol class="npwtwok"><li>DNA solution<ol class="npwthree"><li>Mix 34 Î¼l of dH<sub>2</sub>O and 6 Î¼l of 10x MA2 solution I in a 1.5 ml microtube</li><li>Add 20 Î¼l of 10 mg/ml salmon sperm DNA to the above microtube</li><li>Incubate the microtube at 40 Â°C until use</li></ol><em>Note: This solution should be prepared at time of use under a fume hood.</em> </li><li>PEG solution<ol class="npwthree"><li>Take 0.6 g of polyethylene glycol 4000 in a 1.5 ml microtube</li><li>Add 470 Î¼l of 10x MA2 solution I to the microtube</li><li>Incubate the microtube at 65 Â°C for 10 min shaking every 2 min by hand</li></ol><em>Note: This solution should be prepared at the time of use under a fume hood.</em> </li><li>MA2 medium<ol class="npwthree"><li>Mix 100 ml of MA2 solution I, 10 ml of MA2 solution II, 1 ml of MA2 solution III and 885 ml of dH<sub>2</sub>O to a glass bottle and sterilize by autoclaving at 121 Â°C for 20 min</li><li>Add 4 ml of MA2 solution IV to the mixture<br/><br/><strong>MA2 solution I</strong><br/><img alt="" class="layerphoto" height="167" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225248_5539.jpg" width="600"/><br/><br/><strong>A6 minor salts</strong><br/><img alt="" class="layerphoto" height="220" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225323_0934.jpg" width="600"/><br/><br/><strong>MA2 solution II</strong><br/><img alt="" class="layerphoto" height="86" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225355_4711.jpg" width="600"/><br/><br/><strong>MA2 solution III</strong><br/><img alt="" class="layerphoto" height="86" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225436_3559.jpg" width="600"/><br/><br/><strong>MA2 solution IV</strong><br/><img alt="" class="layerphoto" height="138" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190328/20190328225518_9605.jpg" width="600"/><br/><em>Note: This MA2 solution IV should be sterilized by filtration and stored at 4 Â°C in the dark.</em><br/><br/></li></ol></li><li>MA2 solid gellan gum plate (for making 9-10 plates)<ol class="npwthree"><li>Mix 30 ml of MA2 solution I, 3 ml of MA2 solution II, 300 Î¼l of MA2 solution III, and 70 ml of dH<sub>2</sub>O in a 500 ml flask. Mix 200 ml of dH<sub>2</sub>O and 1.5 g gellan gum in another 500 ml flask. Sterilize these flasks by autoclaving</li><li>After autoclaving, mix each solution with 1.2 ml of MA2 solution IV in one flask. In case of 5-FOA selection, add directly 150 mg of uracil (final concentration: 0.5 mg/ml) and 240 mg of 5-FOA (final concentration: 0.8 mg/ml) in powder to the flask</li><li>Pour the mixed solution onto the sterilized plastic plates</li><li>Dry the plates at room temperature for 15 min</li><li>If plates are to be stored, they should be kept in the dark at 4 Â°C</li></ol></li><li>DNA extraction buffer<ol class="npwthree"><li>Dissolve 12.1 g of Trizma<sup>Â®</sup> base to 100 ml of dH<sub>2</sub>O in a glass bottle and adjust pH to 8.0 with HCl to make 1 M Tris-HCl (pH = 8.0) solution. Sterilize by autoclaving</li><li>Dissolve 10.0 g of sodium dodecyl sulfate to 100 ml of dH<sub>2</sub>O in a glass bottle to make 10% SDS solution</li><li>Dissolve 18.6 g of EDTAâ2Na to 100 ml of dH<sub>2</sub>O in a glass bottle and adjust pH to 8.0 with NaOH to make 0.5 M EDTA solution. Sterilize by autoclaving</li><li>Mix 5 ml of 1 M Tris-HCl (pH = 8.0) solution, 5 ml of 10% SDS solution, 1 ml of 0.5 M EDTA solution and 89 ml of dH<sub>2</sub>O in a glass bottle</li><li>Store at room temperature</li></ol></li><li>Neutralized phenol<ol class="npwthree"><li>Dissolve 100 g of phenol in a 200 ml glass bottle by heating at 65 Â°C</li><li>After dissolving phenol, add 0.1 g of 8-quinolinol to phenol</li><li>Add the mixture of 50 ml of 1 M Tris-HCl (pH = 8.0) solution and 50 ml of dH<sub>2</sub>O to the phenol and mix well for 5 min at room temperature</li><li>After mixing, incubate the phenol mixture for 10 min at room temperature</li><li>Remove the 80 ml of the upper phase (liquid layer)</li><li>Add the mixture of 50 ml of 1 M Tris-HCl (pH = 8.0) solution and 50 ml of dH<sub>2</sub>O to the phenol and mix well for 5 min at room temperature</li><li>After mixing, incubate the phenol mixture for 10 min at room temperature</li><li>Remove the 100 ml of the upper phase (liquid layer)</li><li>Store at 4 Â°C in the dark</li></ol><em>Note: When using phenol-chloroform-isoamyl alcohol, personal protective equipment needs to be used according to local health and safety regulations. Always work under a fume hood.</em><br/></li></ol><p class="pptt" id="biaoti27374">### Acknowledgments</p><p class="ppzz">This protocol was adapted from (Takemura <em>et al.</em>, 2018 and 2019). The authors thank the Biomaterials Analysis Division, Tokyo Institute for Technology for DNA sequence analysis. This study was supported by MEXT/JSPS KAKENHI (Grant numbers: 15K14539 to K.T., 17K07438 to S.I., 17K07439 to Y.K.) and by Advanced Low Carbon Technology Research and Development Program (ALCA) of Japan Science and Technology Agency (JST) to K.T.</p><p class="pptt" id="biaoti27375">### Competing interests</p><p class="ppzz">No competing interests declared.</p><p class="pptt" id="biaoti27376">### References</p><ol class="npwtwok"><li>Fujiwara, T., Ohnuma, M., Kuroiwa, T., Ohbayashi, R., Hirooka, S. and Miyagishima, S. Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28352279" target="_blank">Development of a double nuclear gene-targeting method by two-step transformation based on a newly established chloramphenicol-selection system in the red alga <em>Cyanidioschyzon merolae</em>.</a> <em>Front Plant Sci</em> 8: 343.</li><li>Kobayashi, Y., Ohnuma, M., Kuroiwa, T., Tanaka, K., and Hanaoka, M. (2010) <a href="https://zs.thulb.uni-jena.de/receive/jportal_jparticle_00209501" target="_blank">The basics of cultivation and molecular genetic analysis of the unicellular red alga <em>Cyanidioschyzon merolae</em>.</a> <em>Endocytobiosis Cell Res</em> 20: 53-61.</li><li>Minoda, A., Sakagami, R., Yagisawa, F., Kuroiwa, T. and Tanaka, K. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15215501" target="_blank">Improvement of culture conditions and evidence for nuclear transformation by homologous recombination in a red alga, <em>Cyanidioschyzon merolae</em> 10D.</a> <em>Plant Cell Physiol</em> 45(6): 667-671.</li><li>Fujiwara, T. and Ohnuma, M. (2018). <a href="https://link.springer.com/chapter/10.1007/978-981-10-6101-1_7" target="_blank">Procedures for transformation and their applications in <em>Cyanidioschyzon merolae</em>.</a> In: Kuroiwa, T., Miyagishima, S.Y., Matsunaga, S., Sato, N., Nozaki, H., Tanaka, K. and Misumi, O. (Eds.). <em>Cyanidioschyzon merolae</em>. Singapore, Springer Singapore: 87-103.</li><li>Ohnuma, M., Yokoyama, T., Inouye, T., Sekine, Y. and Tanaka, K. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18003671" target="_blank">Polyethylene glycol (PEG)-mediated transient gene expression in a red alga, <em>Cyanidioschyzon merolae</em> 10D.</a> <em>Plant Cell Physiol</em> 49(1): 117-120.</li><li>Takemura, T., Imamura, S., Kobayashi, Y. and Tanaka, K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30099537" target="_blank">Construction of a selectable marker recycling system and the use in epitope tagging of multiple nuclear genes in the unicellular red alga <em>Cyanidioschyzon merolae</em>.</a> <em>Plant Cell Physiol</em> 59(11): 2308-2316.</li><li>Takemura, T., Imamura, S., Kobayashi, Y., and Tanaka, K. (2019). <a href="https://doi.org/10.21769/BioProtoc.3172" target="_blank">Top starch plating method for the efficient cultivation of unicellular red alga <em>Cyanidioschyzon merolae</em>.</a> <em>Bio-protocol</em> 9(4): e3172.</li><li>Taki, K., Sone, T., Kobayashi, Y., Watanabe, S., Imamura, S. and Tanaka, K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26582291" target="_blank">Construction of a <em>URA5.3</em> deletion strain of the unicellular red alga <em>Cyanidioschyzon merolae</em>: A backgroundless host strain for transformation experiments.</a> <em>J Gen Appl Microbiol</em> 61(5): 211-214. <br/></li></ol>