<body>
            <h1>Improved HTGTS for CRISPR/Cas9 Off-target Detection</h1>
            <p><strong>Authors:</strong> Jianhang   Yin, Mengzhu Liu, Yang  Liu, Jiazhi Hu</p>
            <h2>Abstract</h2>
            <p>Precise genome editing is essential for scientific research and clinical application. At present, linear amplification-mediated high-throughput genome-wide translocation sequencing (LAM-HTGTS) is one of most effective methods to evaluate the off-target activity of CRISPR-Cas9, which is based on chromosomal translocation and employs a âbaitâ DNA double-stranded break (DSB) to capture genome-wide âpreyâ DNA DSBs. Here, we described an improved HTGTS (iHTGTS) method, in which size-selection beads were used to enhance reaction efficiency and a new primer system was designed to be compatible with Illumina Hiseq sequencing. Compared with LAM-HTGTS, iHTGTS is lower cost and has much higher sensitivity for off-target detection in HEK293T, K562, U2OS and HCT116 cell lines. So we believe that iHTGTS is a powerful method for comprehensively assessing Cas9 off-target effect.</p>
            <p><strong>Keywords:</strong> CRISPR-Cas9, Off-target activity, Chromosomal translocation, LAM-HTGTS, iHTGTS</p>
            <h2>Background</h2><p>The CRIPSR-Cas9 (clustered regularly interspaced short palindromic repeats and CRISPR-associated proteins) has been widely used as a powerful genome editing tool (Cong <em>et al.</em>, 2013; Jinek <em>et al.</em>, 2013; Mali <em>et al.</em>, 2013). However its off-target activity causes DNA DSBs at imperfectly matched loci. During the past few years, several methods based on next-generation sequencing have been published to detect off-target sites. LAM-HTGTS (Frock <em>et al.</em>, 2015; Hu <em>et al.</em>, 2016), which is based on chromosomal translocation, makes use of a âbaitâ DSB to capture âpreyâ DSBs to sensitively identify off-target hotspots; GUIDE-seq inserts a specific DNA oligo into break site and applies PCR to enrich DSBs (Tsai <em>et al.</em>, 2015). <em>In vitro </em>methods such as Digenome-seq (Kim <em>et al.</em>, 2015) and CIRCLE-seq (Tsai <em>et al.</em>, 2017) are more sensitive but often need to be verified <em>in vivo</em>. Though LAM-HTGTS is very sensitive, there is still room for improvement. <br/><br/>In the improved HTGTS (iHTGTS), we applied size-selection beads to deplete surplus biotinylated primer for the bridge adapter ligation efficiency enhancement. iHTGTS showed 4 times greater sensitivity than LAM-HTGTS. Also a new primer system was employed which can accommodate 150 bp x 2 Hiseq sequencing instead of 250 bp x 2 Miseq, so that the sequencing cost is much saved. Taken together, iHTGTS is a cost-effective and high efficient method. We believe iHTGTS can give researchers deeper insights into the off-target activity of CRIPSR/Cas9.</p>
        </body>

<p class="pptt" id="biaoti27629">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (Quality Scientific Plastics)</li><li>1.5 ml tube (Axygen, catalog number: MCT-150-C)</li><li>0.22 Î¼m syringe filter (MILLEX, catalog number: PR03683)</li><li>200 Î¼l PCR tubes (Axygen, catalog number: 14-222-261)</li><li>Agarose (Thermo Fisher, catalog number: R0492)</li><li>1 kb DNA plus ladder (Transgen Biotech, catalog number: BM211)</li><li>Streptavidin C1 beads (Thermo Fisher, catalog number: 65001)</li><li>AMPure XP beads (Axygen)</li><li>Protease K (Sigma, catalog number: P8044) </li><li>FastPfu (Transgen Biotech, catalog number: AP221-02)</li><li>dNTPs (Transgen Biotech, catalog number: AD101-11) </li><li>AxyPrep MAG PCR Clean-Up (Axygen, catalog number: MAG-PCR-CL)</li><li>NaCl (VWR Life Sciences, catalog number: 97061-266)</li><li>EDTA (Amresco, catalog number: BDH9232)</li><li>Tris (VWR Life Sciences, catalog number: 97062-420)</li><li>PEG 8000 (Sigma, catalog number: 89510-250G-F)</li><li>T4 ligase (Thermo Fisher, catalog number: EL0011)</li><li>EasyTaq (Transgen Biotech, catalog number: AP111-01)</li><li>Gel extraction kit (Thermo Fisher, catalog number: K0691)</li><li>75% Ethanol (Beijing Chemical Works)</li><li>Isopropyl (Beijing Chemical Works)</li><li>EDTA-Na<sub>2</sub>Â·2H<sub>2</sub>O (Amresco, catalog number: BDH9232)</li><li>NaOH (Sigma, catalog number: 221465)</li><li>HCl (Sigma, catalog number: 7647-01-0)</li><li>SDS (Sigma, catalog number: 72455)</li><li>TAE (see Recipes)</li><li>Proteinase K (see Recipes) </li><li>5 M NaCl (see Recipes)</li><li>0.5 M EDTA (pH 8.0) (see Recipes)</li><li>1 M Tris-HCl (pH 7.4) (see Recipes)</li><li>Cell lysis buffer (see Recipes)</li><li>TE buffer (see Recipes)</li><li>50% (wt/vol) PEG 8000 (see Recipes)</li><li>2x B&amp;W buffer (see Recipes)</li><li>Annealing buffer (see Recipes)</li><li>50 mM bridge adapter (see Recipes)</li></ol><p class="pptt" id="biaoti27630">### Equipment</p><ol class="npwtwok"><li>Foam floating tube rack</li><li>8-well magnet stand</li><li>Pipettes (GILSON)</li><li>Thermomixer C (Eppendorf)</li><li>PCR Thermal Cyclers (Applied Biosystems)</li><li>Covaris (M220 Focused-ultrasonicator)</li><li>Centrifuge (Eppendorf, model: 5418R)</li><li>NanoDrop (DeNOVIX, DS11)</li><li>Incubator (Thermo Fisher)</li><li>Autoclave<br/></li></ol><p class="pptt" id="biaoti27631">### Procedure</p><ol class="npwone"><li>Design the primers used for iHTGTS<br/>Choose Cas9-generated on-target site DSB as the âbaitâ to capture other genome-wide âpreyâ DSBs. To be compatible with 2 x 150 bp Hiseq sequencing, biotinylated primer for LAM-PCR was designed to bind 150 bp upstream of Cas9 binding site; nested primer was designed annealling to the downstream of the biotinylated primer about 90 bp away from the cut site. (Figure 1, Table 1)<br/><em>Note: Avoid designing primers at DNA repetitive region. <br/></em><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429011142_4355.jpg" width="150"/><br/><strong>Figure 1. Primer design for iHTGTS.</strong> The red arrow indicates the Cas9 on-target cutting site. The blue arrow indicates the nested primer and purple indicates the biotinylated primer. The distance to the cutting site is shown below.<br/><br/><strong>Table 1. The sequences of primers and bridge adapters.</strong> The Phor indicates the phosphorylation. MID indicates the index base for data demultiplex. REDPRIMER indicates nested primer, which varies between different loci.<br/><img alt="" class="layerphoto" height="152" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429011309_6517.jpg" width="600"/><br/><br/></li><li>Extract Genomic DNA (gDNA)<br/><ol class="npwtwo"><li>Forty-eight hours after transfecting HEK293T cells with the Cas9 plasmids, collect 10<sup>7</sup> transfected HEK293T cells (we have also tried K562, HCT116 and U2OS, all work well) in a 1.5 ml tube and add 500 Âµl cell lysis buffer. Incubate the tube in Thermomixer at 56 Â°C, 500<em> x g</em> for 10-18 h. <br/><em>Note: Protease K solution should be added into lysis buffer just before use (10 ng/ml final concentration).</em> </li><li>Add 500 Âµl isopropyl and mix thoroughly till you can see a white flocculent DNA pellet.</li><li>Using a pipet to transfer the pellet into another 1.5 ml tube with 1 ml 70% ethanol. Centrifuge at 13,000 <em>x g</em> for 5 min.</li><li>Discard the supernatant. Centrifuge again and deplete residual 70% ethanol. Add 500 Âµl TE and incubate the tube in Thermomixer at 56 Â°C, 500 <em>x g</em> for at least 2 h.<br/><em>Note: Adjust the volume of added TE to make sure the DNA concentration not less than 200 ng/Âµl.</em> </li><li>Quantify the DNA using NanoDrop. The A<sub>260</sub>/A<sub>280</sub> should be higher than 1.8.</li><li>Recommend 20 Âµg gDNA for iHTGTS library construction.</li></ol><br/></li><li>Fragment gDNA by sonication<br/><ol class="npwtwo"><li>Add 20 Âµg gDNA into a PCR tube. Set Covaris with the following parameters: PIP = 50 watts, DF = 30%, CPB = 200, Time = 60 s.</li><li>After sonication, take 200 ng DNA for 1% agarose page. The range of the DNA smear should be at 0.2-2 kb with a peak at 0.75 kb. (Figure 2)<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429011452_1400.jpg" width="100"/><br/><strong>Figure 2. DNA smear pattern after sonication.</strong> The smear ranges from 300 bp to 2 kb with the peak at about 750 bp.<br/><br/></li></ol></li><li>LAM-PCR<br/><ol class="npwtwo"><li>Set up the reaction in 4 x 50 Âµl PCR tubes as following (DNA template for each PCR reaction can be 1-10 Âµg):<br/><img alt="" class="layerphoto" height="195" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429011737_8292.png" width="300"/><br/><br/></li><li>Set up the PCR program: <br/>95 Â°C for 2 min<br/>[95 Â°C for 30 s, 58 Â°C for 30 s, 72 Â°C for 1.5 min] (80 cycles) <br/>72 Â°C for 2 min<br/>10 Â°C forever</li><li>Deplete surplus biotin primer using AMPure XP beads<br/><ol class="npwthree"><li>Add 50 Âµl AMPure XP beads into each PCR tube, mix gently and incubate the tubes at RT for 5 min.<br/><em>Note: This step aims to remove fragments less than 300 bp; the volume of the beads can be upscaled according to different batches.</em> </li></ol><br/><em>Note: All the following Steps (D3b-D3f) are operated at RT.</em><span style="font-size:10pt;"> <ol class="npwthree" start="2"><li>Put the tubes on an 8-well magnet stand for 5 min.</li><li>Remove the supernatant, add 200 Âµl 70% ethanol. After standing for 30 s, remove the supernatant.<br/></li><li>Repeat the Step D3c.</li><li>Add 50 Âµl dH2O, mix the beads completely using a pipette and incubate at RT for 2 min.</li><li>Put the tubes on the magnet for 2 min and pool the supernatants (about 200 Âµl) into a new 1.5 ml tube.</li></ol><br/></span> </li></ol></li><li>Streptavidin beads binding<br/><ol class="npwtwo"><li>Add 50 Âµl 5 M NaCl, 2.5 Âµl 0.5 M EDTA into the PCR product from the last step. Add 30 Âµl streptavidin beads and rotate for 4 h at RT. (The streptavidin beads should be washed twice with 1x B&amp;W buffer before use)</li><li>Put the beads against the 1.5 ml tube magnet stand and remove the supernatant. Wash the beads three times each using 400 Âµl 1x B&amp;W buffer. </li><li>Wash the beads with 400 Âµl dH<sub>2</sub>O and then resuspend the beads in 42.4 Âµl dH<sub>2</sub>O.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>On-beads ligation for bridge adapter<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203194509_4172.jpg" width="260"/><br/>Set the reaction in a 1.5 ml tube in a rotator and ligate overnight at RT.<br/><br/></li><li>Nested PCR<br/><ol class="npwtwo"><li>Add 80 Âµl 2x B&amp;W buffer and 160 Âµl 1x B&amp;W buffer. Put the beads against the 1.5 ml tube magnet stand and remove the supernatant. Wash the beads using 400 Âµl 1x B&amp;W buffer three times and 400 Âµl dH<sub>2</sub>O once. Resuspend the beads in 80 Âµl dH<sub>2</sub>O.<br/></li><li>Set up the PCR reaction in 2x PCR tubes as following:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203210607_9701.jpg" width="300"/><br/></li><li>Set up the PCR program: <br/>95 Â°C for 5 min <br/>[95 Â°C for 1 min, 58 Â°C for 30 s, 72 Â°C for 1 min] (15 cycles)<br/>72 Â°C for 10 min<br/>10 Â°C forever</li><li>Recycle the PCR products using AMpure XP beads as described in Step D3. Elute the PCR products with 35 Âµl dH<sub>2</sub>O for each PCR tube. Gather the PCR products together and measure the concentration.<br/><em>Note: This step aims to remove DNA fragments less than 500 bp, upscale the volume of the beads according to different batches.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><strong>Enzyme Blocking (Optional)<br/><ol class="npwtwo"><li>Add 8 Âµl 10x enzyme buffer, 10 U blocking enzyme, incubate at 37 Â°C for 1 h or longer.</li><li>Purify the DNA with GeneJET column, elute with 70 Âµl dH<sub>2</sub>O, and check the concentration.<br/><div style="white-space:nowrap;"><br/></div></li></ol></strong> </li><li>Tagged PCR<br/><ol class="npwtwo"><li>Set up the PCR reaction in 2 x PCR tubes as following:<br/><img alt="" class="layerphoto" height="174" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429012929_3157.png" width="300"/><br/><br/></li><li>Set up the PCR program:2.Set up the PCR program:<br/>95 Â°C for 3 min<br/>[95 Â°C for 20 s, 60 Â°C for 30 s, 72 Â°C for 1 min] (10-15 cycles)<br/>72 Â°C for 5 min<br/>10 Â°C forever<br/><em>Note: The PCR cycle number is dependent on the DNA concentration from the last step.</em><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" height="139" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429013127_2771.png" width="600"/><br/><br/></div></li></ol></li><li>Purified PCR products<br/>Pool the DNA together, run all the DNA on 1% agarose gel in TAE buffer, cut products between 500-900 bp (Figure 3), purify through a Gel extraction column, elute with 30 Âµl dH<sub>2</sub>O twice. Now the PCR product is ready for Hiseq sequencing.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190429/20190429013226_7082.jpg" width="145"/><br/><strong>Figure 3. DNA smear pattern for iHTGTS library.</strong> Cut the DNA within the range from 500 bp to 1 kb.<br/></li></ol><p class="pptt" id="biaoti27632">### Data analysis</p><p class="ppzz">The detailed operation and mechanism for data analysis is the same with LAM-HTGTS, which can be found in the step 48 to step 51 for Hu <em>et al.</em> (2016).</p><p class="pptt" id="biaoti27633">### Recipes</p><p class="ppzz"><em>Note: The reagents used in this protocol are almost the same with those in Hu et al. (2016). The following recipes are adapted from it.</em> </p><ol class="npwtwok"><li>TAE <br/>40 mM Tris-HCl<br/>20 mM Acetic Acid <br/>1 mM EDTA</li><li>Proteinase K <br/>Dissolve 0.1 g of proteinase K powder in 5 ml of H<sub>2</sub>O to make a 20 mg/ml stock<br/>Divide the solution into 0.5 ml aliquots and store them at -20 Â°C for up to 3 months</li><li>5 M NaCl <br/>Dissolve 292.5 g of NaCl in H<sub>2</sub>O, and adjust the total volume to 1 L<br/>Autoclave the solution and store it at room temperature (RT; 20-25 Â°C) for up to 1 year</li><li>0.5 M EDTA (pH 8.0) <br/>Dissolve 186.12 g of EDTA-Na<sub>2</sub>Â·2H<sub>2</sub>O in H<sub>2</sub>O, adjust the pH to 8.0 using 2.5 N NaOH and then adjust the total volume to 1 L<br/>Autoclave the solution and store it at RT for up to 1 year</li><li>1 M Tris-HCl (pH 7.4) <br/>Dissolve 121.14 g of Tris base in H<sub>2</sub>O, adjust the pH to 7.4 using HCl, and then bring the total volume to 1 L<br/>Autoclave the solution and store it at RT for up to 1 year</li><li>Cell lysis buffer <br/>200 mM NaCl<br/>10 mM Tris-HCl (pH 7.4)<br/>2 mM EDTA (pH 8.0)<br/>0.2% (wt/vol) SDS<br/>Store it at RT for up to 6 months</li><li>TE buffer <br/>10 mM Tris-HCl (pH 7.4)<br/>0.5 mM EDTA (pH 8.0)<br/>Store it at RT for up to 6 months</li><li>50% (wt/vol) PEG 8000<br/>Dissolve 5 g of PEG 8000 in H<sub>2</sub>O at 56 Â°C, and then adjust the total volume to 10 ml<br/>Filter the solution through a 0.22 Î¼m syringe filter, prepare 1 ml aliquots and store them at -20 Â°C for up to 1 year</li><li>2x B&amp;W buffer <br/>2 M NaCl<br/>10 mM Tris-HCl (pH 7.4)<br/>1 mM EDTA (pH 8.0)<br/>Dilute it with H<sub>2</sub>O to make 1x B&amp;W buffer. Store it at RT for up to 1 year</li><li>Annealing buffer <br/>25 mM NaCl<br/>10 mM Tris-HCl (pH 7.4)<br/>0.5 mM EDTA (pH 8.0)<br/>Store it at RT for up to 1 year</li><li>50 mM bridge adapter<br/><ol class="npwthree"><li>Dissolve the two DNA oligos (Table 1) in annealing buffer to a final concentration of 400 Î¼M</li><li>Mix equal volumes of the two dissolved oligos in a new 1.5 ml microtube, put the tube in 1 L of boiling water with a foam floating tube rack, boil it for 5 min and then cool it down slowly in water to ~30 Â°C on the bench (adapter concentration is 200 Î¼M). Alternatively, the oligos can be annealed on a PCR thermoblock</li><li>Dilute fourfold (concentration is 50 Î¼M) with H<sub>2</sub>O, prepare 100 Î¼l aliquots and store them at -20 Â°C for up to 2 months</li><li>Thaw the adapter on ice before use<br/></li></ol></li></ol><p class="pptt" id="biaoti27634">### Acknowledgments</p><p class="ppzz">Thanks for Thousand Talents Plan Youth Program to J. H. and PKU-TSU Center for Life Sciences.<br/>This protocol is adapted from Hu <em>et al.</em> (2016).</p><p class="pptt" id="biaoti27635">### Competing interests</p><p class="ppzz">The authors declare they have no conflict of interest or competing interests.</p><p class="pptt" id="biaoti27636">### References</p><ol class="npwtwok"><li>Cong, L., Ran, F. A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P. D., Wu, X., Jiang, W., Marraffini, L. A. and Zhang, F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718" target="_blank">Multiplex genome engineering using CRISPR/Cas systems.</a> <em>Science</em> 339(6121): 819-823.</li><li>Frock, R. L., Hu, J., Meyers, R. M., Ho, Y. J., Kii, E. and Alt, F. W. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25503383" target="_blank">Genome-wide detection of DNA double-stranded breaks induced by engineered nucleases.</a> <em>Nat Biotechnol </em>33(2): 179-186.</li><li>Hu, J., Meyers, R. M., Dong, J., Panchakshari, R. A., Alt, F. W. and Frock, R. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27031497" target="_blank">Detecting DNA double-stranded breaks in mammalian genomes by linear amplification-mediated high-throughput genome-wide translocation sequencing.</a> <em>Nat Protoc</em> 11(5): 853-871. </li><li>Jinek, M., East, A., Cheng, A., Lin, S., Ma, E. and Doudna, J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23386978" target="_blank">RNA-programmed genome editing in human cells.</a> <em>Elife</em> 2: e00471. </li><li>Kim, D., Bae, S., Park, J., Kim, E., Kim, S., Yu, H. R., Hwang, J., Kim, J. I. and Kim, J. S. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25664545" target="_blank">Digenome-seq: genome-wide profiling of CRISPR-Cas9 off-target effects in human cells.</a> <em>Nat Methods</em> 12(3): 237-243, 231 p following 243. </li><li>Mali, P., Yang, L., Esvelt, K. M., Aach, J., Guell, M., DiCarlo, J. E., Norville, J. E. and Church, G. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722" target="_blank">RNA-guided human genome engineering via Cas9.</a> <em>Science</em> 339(6121): 823-826. </li><li>Tsai, S. Q., Nguyen, N. T., Malagon-Lopez, J., Topkar, V. V., Aryee, M. J. and Joung, J. K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28459458" target="_blank">CIRCLE-seq: a highly sensitive <em>in vitro </em>screen for genome-wide CRISPR-Cas9 nuclease off-targets.</a> <em>Nat Methods</em> 14(6): 607-614. </li><li>Tsai, S. Q., Zheng, Z., Nguyen, N. T., Liebers, M., Topkar, V. V., Thapar, V., Wyvekens, N., Khayter, C., Iafrate, A. J., Le, L. P., Aryee, M. J. and Joung, J. K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25513782" target="_blank">GUIDE-seq enables genome-wide profiling of off-target cleavage by CRISPR-Cas nucleases.</a> <em>Nat Biotechnol </em>33(2): 187-197.</li></ol>