<body>
            <h1>TZA, a Sensitive Reporter Cell-based Assay to Accurately and Rapidly Quantify Inducible, Replication-competent Latent HIV-1 from Resting CD4<sup>+</sup> T Cells</h1>
            <p><strong>Authors:</strong> Anwesha Sanyal, Vatsala S. Rangachar, Phalguni Gupta</p>
            <h2>Abstract</h2>
            <p>The latent HIV-1 viral reservoir in resting CD4<sup>+</sup> (rCD4<sup>+</sup>) T cells represents a major barrier to an HIV-1 cure. There is an ongoing effort to identify therapeutic approaches that will eliminate or reduce the size of this reservoir. However, clinical investigators lack an assay to determine whether or not a decrease in the latent reservoir has been achieved. Therefore, it is critical to develop assays that can reproducibly quantify the reservoir size and changes therein, in participantâs blood during a therapeutic trial. Quantification of the latent HIV viral reservoir requires a highly sensitive, cost-effective assay capable of measuring the low frequency of rCD4<sup>+</sup> T cells carrying functional provirus. Preferably, such an assay should be such that it can be adopted for high throughput and could be adopted under conditions for use in large-scale clinical trials. While PCR-based assays are commonly used to quantify pro-viral DNA or intracellular RNA transcript, they cannot distinguish between replication-competent and defective proviruses. We have recently published a study where a reporter cell-based assay (termed TZA or TZM-bl based quantitative assay) was used to quantify inducible replication-competent latent HIV-1 in blood. This assay is more sensitive, cost-efficient, and faster than available technology, including the quantitative viral outgrowth assay or the Q-VOA. Using this assay, we show that the size of the inducible latent HIV-1 reservoir in virally suppressed participants on ART is approximately 70-fold larger than previous estimates. We describe here in detail an optimized method to quantitate latently infected cells using the TZA.</p>
            <p><strong>Keywords:</strong> TZM-bl cells, TZA assay, Latent reservoir, Latent HIV-1, Quantification of latent reservoir, Inducible virus, Replication competent virus</p>
            <h2>Background</h2><p>The ability to quantitate the latent HIV-1 viral reservoir in a combination ART suppressed individual requires a highly sensitive assay with the ability to measure low frequency of rCD4<sup>+</sup> cells carrying functional provirus. This assay should also be cost-effective and adaptable to be high throughput under different conditions and large-scale clinical applications. Currently, the frequency of the infected cells in latent condition is estimated using Poisson Statistics or by maximum likelihood analysis (Cillo <em>et al.</em>, 2014; Rosenbloom <em>et al.</em>, 2015). Most of these methods including the original Q-VOA method and its adaptations use limited dilution based techniques of PHA simulated CD4<sup>+</sup> T cells which either measures HIV-1 protein or RNA via ELISA or quantitative PCR. These are all labor-intensive, time-consuming, expensive, and requires the frequent addition of activated CD4<sup>+</sup> T cells as feeders (Chun <em>et al.</em>, 1997; Finzi <em>et al.</em>, 1999; Siliciano <em>et al.</em>, 2003; Siliciano and Siliciano, 2005). There has also been a modified version of Q-VOA assay reported which implements MOLT-4/CCR5 (Laird <em>et al.</em>, 2013) cell line for viral expansion instead of activated CD4<sup>+</sup> T cells followed by RNA measurement but this is still more time consuming and labor intensive. <br/><br/>We recently reported the development of a sensitive TZM-bl cell-based assay (termed TZA) (Sanyal <em>et al.</em>, 2017) to quantify the latent HIV-1 reservoir in blood. The present protocol consists of these main parts (i) isolation of resting T cells from the blood from HIV-1 positive and negative (control) donors (Steps A1-A4); (ii) activation of these resting CD4<sup>+</sup> T cells using a strong LRA like Anti-CD3/CD28 (Siliciano and Siliciano, 2004) antibodies and (iii) washing, counting and plating these activated cells on a TZM-bl reporter cells to quantitate replication-competent HIV-1 by measuring Î²-gal expression to quantify virus induced. This assay utilizes the TZM-bl cell line, which stably expresses CD4, CCR5, and CXCR4, and carries an integrated copy of the Î²-galactosidase (Î²-gal) gene under the control of an HIV-1 long terminal repeat (HIV-1 LTR) promoter that allows for the detection of inducible replication-competent HIV-1. By using TZM-bl cells, quantification of replication-competent HIV-1 can be achieved with high sensitivity (Ananworanich and Mellors, 2015). TZA has been used to calculate fraction of inducible latent virus (fPVE) (Cillo <em>et al.</em>, 2014; Sanyal <em>et al.</em>, 2017). The level of fPVE can be used to screen various LRAs <em>in-vitro</em> as well as help monitor their <em>in-vivo</em> efficacy especially during clinical trials. This assay reveals that the viral reservoir is likely much larger than previously predicted and estimated which can impact the ongoing therapeutic approaches to eradicate HIV-1. <br/><br/>This assay can be adapted and automated to a 384-well format further down the line to enable an efficient screening platform, which will be capable of handling multiple samples simultaneously. This will make it a high-throughput assay system. In addition, because of the low cell requirement in this system, the TZA may also be useful for quantification of replication-competent HIV-1 in the pediatric population as well as estimation of the reservoir in tissues. We are presently developing a protocol for determining the reservoir size in tissues using this assay.</p>
        </body>

<p class="pptt" id="biaoti27769">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips</li><li>15 ml centrifuge tubes (Fisherbrand, catalog number: 07-200-886)</li><li>50 ml centrifuge tubes (Fisherbrand, catalog number: 0553913)</li><li>Tissue culture plates, 24-well transparent (Falcon, catalog number: 353047)</li><li>White, 96-well plates, sterile with white lids (PerkinElmer, catalog number: 6005181)</li><li>EDTA tubes purple tops for blood collection: Vacutainer brand sterile (BD, catalog number: 366643) </li><li>CryoTube<sup>TM</sup> vials for freezing cells (Thermo Fischer Scientific, catalog numbers: 153779, 202209, 363401)</li><li>Tissue culture plates, 48-well transparent (Falcon, catalog number: 353078)</li><li>5 ml round-bottom polystyrene tubes (Corning, Falcon<sup>Â®</sup>, catalog number: 352054)</li><li>TZM-bl cells (NIH AIDS Reagent program, catalog number: 8129)</li><li>Animal serum complex (Gemini Bio Products, Fetal Plex<sup>TM</sup>, catalog number: 100-602), storage in a -18 Â°C freezer before opening bottle and then store at 4 Â°C for subsequent use</li><li>DMSO (Life Technologies Corp, catalog number: 20688)</li><li>FBS (HyClone, catalog number: SH3011803)</li><li>Beta-Glo Assay System (Promega, catalog number: E4780), storage in a -18 Â°C freezer</li><li>Custom ordered resting CD4<sup>+</sup> T cells negative selection kit (Stemcell Technologies, catalog number: 19309VK) with EasySep D Magnetic Particles (Stemcell Technologies, catalog number: 19250), storage at 4 Â°C</li><li>Dynabeads<sup>TM</sup> Human T-Activator CD3/CD28 (Thermo Fischer Scientific, Gibco<sup>TM</sup>, catalog number: 11131D)</li><li>Clinical grade Human Recombinant Interleukin-2: IL-2 Powder 22 MU VIAL (65483011607), storage in a -18 Â°C freezer where MU stands for medical units <br/><em>Note: Alternatively, human recombinant IL-2 can be purchased from Life Technologies Corp, catalog number: PHC-0023.</em> </li><li>Human Recombinant Interleukin-7: IL-7 Premium grade (Miltenyi Biotec, catalog number: 130-095-361), storage in a -18 Â°C freezer</li><li>Efaviranz (EFV) (Cayman Chemical, catalog number: 14412), need to be re-suspended in IMDM media and diluted according to need of the experimental procedure</li><li>Flow Monoclonal Antibodies <br/><em>Note: You will use 5 Î¼l of each antibody for staining. Use simple Flow staining protocol for staining the cells. This is just done to check for the purity of the resting T cells. Resting T cells are CD25-/CD69-/HLADR- cells.</em> <ol class="npwthree"><li>Mouse Anti-human CD4-AF700 (BD Biosciences, catalog number: 557922)</li><li>Mouse Anti-human CD3-V450 (BD Biosciences, catalog number: 560365)</li><li>Mouse Anti-human CD25-APC (BD Biosciences, catalog number: 340939)</li><li>Mouse Anti-human CD69-FITC (BD Biosciences, catalog number: 555530)</li><li>Mouse Anti-human HLADR-PE (BD Biosciences, catalog number: 347367)</li><li>Mouse Anti-human CD279-PeCy7 (BD Biosciences, catalog number: 561272) </li></ol><br/><em>Note: Isotype controls are used at the same concentration as the specific antibody. So, dilutions have to be determined according to the concentration of the matched antibody.</em> <ol class="npwthree" start="7"><li>PE-Cy<sup>TM</sup> 7 Mouse IgG<sub>1</sub>, Îº Isotype Control (BD Biosciences, catalog number: 557646)</li><li>FITC Mouse IgG<sub>1</sub>, Îº Isotype Control (BD Biosciences, catalog number: 556649)</li><li>V450 Mouse IgG<sub>1</sub>, Îº Isotype Control (BD Biosciences, catalog number: 561504)</li><li>Alexa Fluor<sup>Â®</sup> 700 Mouse IgG<sub>1</sub>, Îº Isotype Control (BD Biosciences, catalog number: 557882)</li><li>APC-H7 Mouse IgG<sub>1</sub>, Îº Isotype Control (BD Biosciences, catalog number: 561427)</li><li>PE Mouse IgG<sub>2a</sub>, Îº Isotype Control (BD Biosciences, catalog number: 558595) </li></ol></li><li>Dye-eFluor 506 (Invitrogen, catalog number: 65-0866-14)</li><li>DPBS (1x), Dulbeccoâs Phosphate-Buffered Saline (Corning Cellgro, catalog number: 21-031-CV), store at 4 Â°C for use after opening bottle</li><li>Dry Ice</li><li>HBSS (1x), Hanks Balanced Salt Solution (Thermo Fischer Scientific, Gibco<sup>TM</sup>, catalog number: 141775-095), store at 4 Â°C for use after opening bottle</li><li>IMDM (1x) (Thermo Fischer Scientific, Gibco<sup>TM</sup>, catalog number: 12440-053), store at 4 Â°C for use after opening bottle</li><li>Lymphocyte separation medium (Corning, catalog number: 25-072-CV), store at 4 Â°C for use after opening bottle</li><li>Penicillin/Streptomycin (Thermo Fischer Scientific, Gibco<sup>TM</sup>, catalog number: 15140-122), store at -18 Â°C freezer before opening bottle and then store at 4 Â°C for subsequent use</li><li>RPMI 1640 (1x) with L-glutamine and 25 mM HEPES (Corning Cellgro, catalog number: 10-041-CV)</li><li>0.05% Trypsin EDTA (1x) (Thermo Fischer Scientific, Gibco<sup>TM</sup>, catalog number: 25300-120), store at -20 Â°C freezer before opening bottle and then store at 4 Â°C for subsequent use</li><li>RoboSep<sup>TM</sup> Buffer (Stemcell Technologies, catalog number: 20104)</li><li>Formalin</li><li>10% RPMI (see Recipes)</li><li>10% IMDM (see Recipes)<br/></li></ol><p class="ppzz"><em>Note: This procedure can be done in a BSL2+ lab following safety procedures involving wearing a closed lab coat and double gloves. The blood and processing should all be done in a bio-safety hood with proper airflow.</em> </p><p class="pptt" id="biaoti27770">### Equipment</p><ol class="npwtwok"><li>Tissue culture flask vented caps, 70 ml (Falcon, catalog number: 353109)</li><li>Tissue culture flask vented caps, 250 ml (Falcon, catalog number: 353110)</li><li>Pipettes</li><li>Rocker</li><li>Incubator (Thermo Electron Corporation, NAPCO SERIES 8000 WJ, CO<sub>2</sub> Incubator) </li><li>Centrifuge: Sorvall Legend RT+ Centrifuge (Thermo Scientific)</li><li>Freezer</li><li>Luminometer (Promega GloMax Navigator, GM2000)</li><li>LN2 freezer</li><li>Magnets (Stem Cell Technologies, catalog number: 96290)</li><li>Light Microscope</li><li>4 Â°C refrigerator </li><li>Flow cytometer (BD, model: LSR II)</li><li>Water Bath (Fisher Scientific, model: 2223)</li><li>Haemocytometer 0.100 mm deep (Hausser Scientific)</li></ol><p class="pptt" id="biaoti27771">### Software</p><ol class="npwtwok"><li>FlowJo software (Tree Star)</li><li>BD FACSDiva<sup>TM</sup> software (BD Biosciences)</li><li>IUPMStats v1.0 Infection Frequency Calculator (Online) <a href="http://silicianolab.johnshopkins.edu/" target="_blank">http://silicianolab.johnshopkins.edu/</a> </li><li>Microsoft Excel <br/></li></ol><p class="pptt" id="biaoti27772">### Procedure</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwthree"><li><em>In order to get enough cells to keep the control for sets of experiment same, we leukophoresied control HIV-1 negative donors and also participants of the study, and froze those cells down at -140 Â°C. </em> </li><li><em>The TZA protocol may be adapted for fresh as well as frozen PBMCs.</em> </li><li><i>This procedure can be done in a BSL2+ lab following safety procedures involving wearing a closed lab coat and double gloves. The blood and processing should all be done in a bio-safety hood with proper airflow.<br/></i> <br/></li></ol><ol class="npwone"><li>Isolation of rCD4<sup>+</sup> T cells from patients and controls<ol class="npwtwo"><li>Collect blood from participants in EDTA tubes. This is usually done in the hospital by nurses and then sent over for processing. </li><li>Dilute the blood in DPBS (1:2). The total volume in a 40 ml tube should be 10 ml blood, 20 ml DPBS. </li><li>Then use 10 ml of lymphocyte separation medium or ficoll to separate the layers of cells in the blood. This is done by putting the pipette with ficoll at the bottom of the 50 ml tubes and slowly pushing the liquid in and a layer of clear ficoll will be at the bottom of the tube. The total volume of liquid will not become 50 ml in the tube.</li><li>Every reagent should be at room temperature before use. </li><li>Spin this in a centrifuge at 2,000 rpm (448 <em>x g</em>) for 20 min to get distinct PBMC layer which will appear in between the ficoll layer and the DPBS and serum mixture. RBC will appear at the bottom of the tube. </li><li>Collect the PBMCs layer and wash it again with DPBS spinning in a centrifuge at 1,000 rpm (112 <em>x g</em>) for 10 min.</li><li>Discard the supernatant and collect the pellet (PBMCs). At this point you may either carry on with the rest of the procedure or freeze your cells in a freezing media (10% DMSO in FBS) and store them at -140 Â°C for long term storage. </li><li>Count the cells recovered and then use resting CD4<sup>+</sup> T cell Isolation kit to separate out the rCD4<sup>+</sup> T cells from the PBMCs. It is always a good idea to not run less than 50 million cells at once since the yield in this kit is not high. It is a high purity but a low recovery kit. Also do not do two 5 min separation for less than 100 million cells, rather do one-time separation through the magnet. The details of the separation are listed in the protocol that is sent with the separation kit.</li><li>Count the cells one more time before proceeding to the next step.</li></ol><em>Notes: </em> <ol class="npwthree"><li><em>You will have to take some cells out and stain for resting T cells. This is important since it helps you determine whether the cells selected are purely resting T cells. </em> </li><li><em>If we are working with rCD4</em><sup><em>+</em></sup><em> T cells, you can expect about 5 to 15 million rCD4</em><sup><em>+</em></sup><em>T cells from thawing about 100 million PBMCs for patients and about 20-25 million rCD4</em><sup><em>+</em></sup><em> T cells from 100 million normal control PBMCs. This will not change irrespective of fresh or frozen cells. </em> </li><li><em>For thawing frozen cells, put the frozen cells dropwise slowly while moving the tube in circular motion in a 15 ml tube containing 10 ml of 10% IMDM. Then spin the cells down at 1,000 rpm (112 </em><em>x g</em><em>) for 10 min to wash it off from the freezing media.</em> </li><li><em>Follow the same Steps A1-A9 for cells from negative control participants. For every experiment you will need to do a control PBMCs with which the cells will be compared.</em> </li></ol><br/></li><li>Assess the purity of resting T cells using Flow cytometry staining. (Figure 1)<ol class="npwtwo"><li>First, transfer cells (2 x 10<sup>5</sup> to 2.5 x 10<sup>5</sup>) into a tube and centrifuge at 400 <em>x g</em> at 4 Â°C for 5 min and remove the supernatant. </li><li>Perform the viability staining: resuspend the cell pellets with 250 Î¼l of DPBS and 1 Î¼l of Viability Dye-eFluor506. Incubate for 30 min at 4 Â°C.</li><li>Wash the cells with DPBS and centrifuge the plate at 400 <em>x g</em> at 4 Â°C for 5 min and remove the supernatant.  </li><li>Resuspend the cell pellets first in 100 Î¼l of DPBS and then put this in a mixture of the antibodies in 5 ml round-bottom polystyrene tubes with 5 Î¼l of each antibody listed in Materials and Reagents and incubate for 30 min at 4 Â°C. </li><li>Wash the cells at 400 <em>x g</em> at 4 Â°C for 10 min and remove the supernatant. Do this two times.</li><li>Fix the cells in 1% Formalin to run your flow. You can run your flow in plates or tubes, but we prefer tubes. </li><li>First gate on singlet population and them on lymphocytes. Follow this up by gating on the live population. In the live population of cells look specifically at the CD4<sup>+</sup>/CD3<sup>+</sup> cells as these are the T cells.</li><li>Then the rest of the gating is from the CD4<sup>+</sup>/CD3<sup>+</sup> cells. Gate them for CD69, HLADR and CD25, which are all activation markers. PD1 is used for gating by us for determining cell death signal but it is not important to use if your follow up experiments do not require it.</li></ol><em>Note: You can choose antibodies of your choice for activation marker but we used CD-25, HLA-DR, CD-69 as activation markers. We also used PD1 as a secondary marker since activated cells have lower PD1. You will be looking for cells which are negative for CD-25, HLA-DR, CD-69. We gated on CD4 and CD3 positive cells since we were looking for resting T cells. 95 % or more purity if good for the resting T cells.</em><br/><br/><img alt="" class="layerphoto" height="166" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513033418_1156.jpg" width="100"/><br/><strong>Figure 1. Purity of resting T cells: the resting T cells should be CD25/HLADR/CD69<sup>-</sup>CD3</strong><sup><strong>+</strong></sup><strong>/CD4</strong><sup><strong>+</strong></sup><strong> cells.</strong> A. Gating on singlet population that is live and then to CD4<sup>+</sup>/CD3<sup>+</sup> cells. B. Gating on CD3<sup>+</sup>/CD4<sup>+</sup> cells to determine their activation status (CD25<sup>+</sup>/CD69<sup>+</sup>/HLADR<sup>+</sup>).<br/><br/></li><li>Activation of resting T cells<ol class="npwtwo"><li>For the next step you will need to first stimulate the cells for activation. Use Anti-CD3/CD28 Dynabeads in a concentration of 12.5 Î¼l of the beads per million cells you put in the culture and they will stay in the culture together for 6 days till you do the next phase of the experiment. </li><li>Wash the beads before using them for activation. Put beads in a 5 ml polystyrene tube, put RoboSep<sup>TM</sup>/EasySep<sup>TM</sup> buffer (1 ml) and put in magnet for 5 min. Then after decanting the liquid, resuspend the beads in 10% IMDM culture media exactly the amount you took out. For example, if you took out 100 Î¼l of beads for 8 million cells, you wash the beads, then resuspend it back in 100 Î¼l of 10% IMDM media and then out it in the final activation cocktail. (Figure 2).</li><li>The culture for stimulation is usually placed in 1.0 million cells/ml of media. Each well in a 24-well plate will usually have 2 million resting T cells, 2 ml of 10% IMDM media and 25 Î¼l of resuspended AntiCD3/CD28 Dynabead. After re-suspending the cells in media containing beads, put 300 nm of EFV per ml of the media to prevent cell-cell infection. Always prepare this fresh according to the amount of cell yield. </li><li>For best stimulation, the cells can be concentrated in smaller wells (48-well plates with 1 million/ml cells in one well) and then transferred to a 24-well plate next day. Plate 2 million/2 ml of cells per well in a 24-well plate (Figure 2).</li><li>After 24 h post activation, add 10 Î¼l of 10<sup>5</sup> units/ml IL-2 and 10 Î¼l of 10<sup>5</sup> units of IL-7per million cells as the secondary signaling cytokines to maintain the cells and help them proliferate. Do not remove the Anti-CD3/CD28 Dynabeads.</li><li>The cultures need to be monitored every day and if there is overcrowding which is usually when the media becomes very yellow in color and the cell in each well is more than 4 million cells. Put more base media in the wells and split it into two wells in such cases. </li><li>The base media during splitting should always contain IL-2, IL-7 (on Day 2) and EFV along with 10% IMDM. No beads are required during addition of media to the cells in culture (Figure 2).</li><li>These cells stay in culture for a total of 6 days (Figure 2).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513033449_0665.jpg" width="178"/><br/><strong>Figure 2. Processing latent cells for activation</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Harvesting, plating and reading Beta-gal activity of TZM-bl cells<ol class="npwtwo"><li>On Day 5 of the culture (one day before the T-cells are taken out of culture), seed 96-Well white PerkinElmer plates with 5 x 10<sup>4</sup> cells/well and rest of the plate with 3 x 10<sup>4</sup> cells/well of TZM-bl cells totaling 64-wells. Use 10% RPMI for media since TZM-bl cells grow in 10% RPMI. The total volume of media in each well should be 200 Î¼l. Keep this overnight so that cells adhere to the bottom of wells by next day.<br/><em>Note: Donât maintain TZM-bl cells in culture for more than 3 weeks and split the cells every three days. When splitting, do not use more than 1 million/10 ml of 10% RPMI. TZM-bl cells grow in 10% RPMI in a 250 ml vented Falcon flask and need to be trypsinized when ready to split. When a frozen cell culture is started, make sure it is done in a 70 ml Falcon vented flask and then expanded over time.</em> </li><li>On Day 6 from the start of culture, pull the activated T cells from the 24-well plates in 15 ml conical tubes and centrifuge at 1,000 rpm (112 <em>x g</em>) for 10 min to pellet the cells.<br/><em>Note: You can save the supernatants for RNA if you want to compare the RNA yield of these activated latently infected cells. This can be frozen in -80 Â°C freezer. You will have to count the cells and wash them two more times with DPBS. You can also save 0.5 million cells for DNA quantification from each patient. For this, just count the cells, put them in Eppendorf tubes, spin them down at 1,000 rpm (112 </em><em>x g</em><em>) for 10 min and pellet the cells. Then directly just freeze these pellets in -80 Â°C freezer. This is used if you want to calculate the fraction of provirus induced for the particular participant.</em> </li><li>With the rest of the activated T-cells, start to make a dilution series. Make a 4 fold dilution (1:3) starting from 1.25 x 10<sup>5</sup> cells/well in the highest dilution downward. (You can start as low as 6 x 10<sup>4</sup> cells/well on the top for this).<br/><em>Note: If you are starting from 1.25 x 10</em><sup><em>5</em></sup><em> cells/well in the first row, each well in the top row (8 wells) should have 1.25 x 10</em><sup><em>5</em></sup><em> cells/well in 200 Î¼l of 10% RPMI. The second row will have 1.25 x 10</em><sup><em>5</em></sup><em>/4 cells in all eight wells and so on downwards for five more dilutions. This should give you a total of 6 dilutions (Figure 3).</em> </li><li>After making the dilution series in 15 ml tubes, the media from the TZM-bl cell plates will have to be removed by aspiration before the patient cells are plated on it. Do not do it before you start to plate as it dries out the cells in the plate. The seventh row is usually not used and the eighth row is fresh media on the TZM-bl cells (Figure 3).</li><li>All the dilutions will have to be in 10% RPMI.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513033513_5271.jpg" width="131"/><br/><strong>Figure 3. Plating cells for reading in 96 wells using TZM-bl cells</strong><br/><br/></li><li>Keep this plate in a 37 Â°C incubator for 48 h after which it has to be read. This reading can be taken anywhere between 48 and 72 h.</li><li>Take the plates out and aspirate the media out without disturbing the cells. </li><li>Put back DPBS in the wells with multi-channel pipettes and lightly wash and take the washout and discard. Do this two times. Do not be very rigorous while washing as this could take the cells off from the plates.</li><li>Put 100 Î¼l of Promega Beta-Glo reagent on each well and incubate in the dark for 45 min-1 h. Try to do all these steps in very little light. This reagent is extremely sensitive to light.</li><li>Then read in a Promega Glomax machine to get the light units following software prompt. This is an automated machine where you will have to select any of the cell-titer protocols, which will be able to read your plate. Then you can name and save the plate reads as excel files.</li></ol><br/></li><li>Analysis of readings and IUPM calculation<ol class="npwtwo"><li>After the reading is taken, transport the Excel file out to analyze these results. First, calculate a mean (average) of the bottom media control row and deduct this value from all the other individual readings in the plate. This will be your blank. This should be done for both the control HIV-1 negative plate as well as the HIV-1 positive patient plate. This will also assure that you ideally have similar conditions for both the test and control cells. </li><li>In the control plate, which has the control HIV-1 negative T cells, calculate the average of each row with all the eight wells for each dilution. Do this for each dilution. Also, calculate the standard deviation for each dilution.</li><li>Add the average + 2 times the standard deviation (avg. + 2 SD) for each dilutions. This will give you a value for each dilution in the control well based on which you will calculate the positive wells in the test plate.</li><li>After you have the value of (avg. + 2 SD) for each dilution in the control plate, subtract it from each individual reading of wells for that particular dilution in the participant plate.</li><li>This will give you some wells with positive and some with negative values. The positive values can vary from 0/8 to 8/8.</li><li>Put these readings in the algorithm developed by Silicianos lab <a href="http://silicianolab.johnshopkins.edu/" target="_blank">http://silicianolab.johnshopkins.edu/</a> (Rosenbloom <em>et al.</em>, 2005) and calculate the Infectious unit per million cells or IUPM. </li><li>For DNA copies/million cells, use a standard DNA q-PCR (Cillo <em>et al.</em>, 2014) for integrated provirus in the cells and calculate the fraction of provirus that can be induced by the stimulation to form replication competent virus. This is calculated with the formula: fPVE = IUPM/DNA per million cells x 100.</li></ol></li></ol><p class="pptt" id="biaoti27773">### Data analysis</p><ol class="npwtwok"><li>For the purity estimation of the resting T cells, samples are acquired on an LSR II flow cytometer using FACSDiVa<sup>TM</sup> software before putting the cells in culture for activation. Flow cytometry data are analyzed using FlowJo to gate, quantify, and analyze the resting T cell population. </li><li>For the IUPM calculations, the maximum likelihood estimate was applied to determine the infectious unit per million (IUPM) cells for the TZA assay using online software, available at <a href="http://silicianolab.johnshopkins.edu/" target="_blank">http://silicianolab.johnshopkins.edu/</a>, developed by Rosenbloom <em>et al.</em> (2015).<br/></li></ol><p class="pptt" id="biaoti27774">### Notes</p><p class="ppzz">During the entire protocol, you have to make sure that you are doing exactly the same things for another set of cells which are HIV-1 negative or control cells. If you do not have your controls, your experiments will not work. You will have to make sure even the control cells are stimulated the same way as the patient cells so that ideally you have similar conditions for both the test and control cells.</p><p class="pptt" id="biaoti27775">### Recipes</p><ol class="npwtwok"><li>10% RPMI<br/>RPMI 1640<br/>1x medium with 10% Animal Serum Complex and 1% Penicillin-streptomycin<br/>Keep sterile and at room temperature during experiment and store at 4 Â°C</li><li>10% IMDM<br/>Mix 450 ml of IMDM w/L-glutamine with 50 ml FBS and 1% Penicillin-streptomycin (5 ml) from storage at -20 Â°C<br/>This will make a final volume of 10% FBS in IMDM<br/>This should always be stored at 4 Â°C for long term storage<br/></li></ol><p class="pptt" id="biaoti27776">### Acknowledgments</p><p class="ppzz">This protocol was adapted from our publication (Sanyal <em>et al.</em>, 2017). We thank the ex-members of the lab, Lori Caruso, Deena Ratner and Ming Ding (University of Pittsburgh, Pittsburgh) for their technical assistance; N. Sluis-Cremer for the DNA measurement and study design inputs; P. Tarwater (University of Texas, El Paso) and C. Shen (University of Pittsburgh, Pittsburgh) for statistical consultation and development of the assay statistics; W. Buchanan (University of Pittsburgh, Pittsburgh) for recruitment of the Multicenter AIDS Cohort Study participants for the study; and all the participants of the Pittsburgh portion of the MACS for donating blood for this study.<br/>  This work was supported by NIH grants R21AI138716 (Phalguni Gupta), U01-AI35041 (Charles R. Rinaldo.), R21-AI119117 (Nicolas P. Sluis-Cremer and NIH Fogarty training grant fellowship D43TW010039 (Phalguni Gupta).</p><p class="pptt" id="biaoti27777">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti27778">### Ethics</p><p class="ppzz">Blood was collected and handled according to protocols approved by the University of Pittsburgh institutional review board and was pulled from the patient in the hospital by nurses. The participants both HIV-1 negative and positive were recruited from the Multicenter AIDS Cohort Study.<br/>  Written consent was obtained from all the participants and the study was explained to them in simple terms before proceeding with it.</p><p class="pptt" id="biaoti27779">### References</p><ol class="npwtwok"><li>Ananworanich, J. and Mellors, J. W. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26425677" target="_blank">How much HIV is alive? The challenge of measuring replication competent HIV for HIV cure research.</a> <em>EBioMedicine</em> 2(8): 788-789. <br/></li><li>Chun, T. W., Carruth, L., Finzi, D., Shen, X., DiGiuseppe, J. A., Taylor, H., Hermankova, M., Chadwick, K., Margolick, J., Quinn, T. C., Kuo, Y. H., Brookmeyer, R., Zeiger, M. A., Barditch-Crovo, P. and Siliciano, R. F. (1997). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9144289" target="_blank">Quantification of latent tissue reservoirs and total body viral load in HIV-1 infection.</a> <em>Nature</em> 387(6629): 183-188. </li><li>Cillo, A. R., Sobolewski, M. D., Bosch, R. J., Fyne, E., Piatak, M., Jr., Coffin, J. M. and Mellors, J. W. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24706775" target="_blank">Quantification of HIV-1 latency reversal in resting CD4<sup>+</sup> T cells from patients on suppressive antiretroviral therapy.</a> <em>Proc Natl Acad Sci U S A</em> 111(19): 7078-7083. </li><li>Finzi, D., Blankson, J., Siliciano, J. D., Margolick, J. B., Chadwick, K., Pierson, T., Smith, K., Lisziewicz, J., Lori, F., Flexner, C., Quinn, T. C., Chaisson, R. E., Rosenberg, E., Walker, B., Gange, S., Gallant, J. and Siliciano, R. F. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10229227" target="_blank">Latent infection of CD4<sup>+</sup> T cells provides a mechanism for lifelong persistence of HIV-1, even in patients on effective combination therapy.</a> <em>Nat Med</em> 5(5): 512-517. </li><li>Laird, G. M., Eisele, E. E., Rabi, S. A., Lai, J., Chioma, S., Blankson, J. N., Siliciano, J. D. and Siliciano, R. F. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23737751" target="_blank">Rapid quantification of the latent reservoir for HIV-1 using a viral outgrowth assay.</a> <em>PLoS Pathog</em> 9(5): e1003398. </li><li>Rosenbloom, D. I., Elliott, O., Hill, A. L., Henrich, T. J., Siliciano, J. M. and Siliciano, R. F. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26478893" target="_blank">Designing and interpreting limiting dilution assays: General principles and applications to the latent reservoir for human immunodeficiency virus-1.</a> <em>Open Forum Infect Dis</em> 2(4): ofv123. </li><li>Sanyal, A., Mailliard, R. B., Rinaldo, C. R., Ratner, D., Ding, M., Chen, Y., Zerbato, J. M., Giacobbi, N. S., Venkatachari, N. J., Patterson, B. K., Chargin, A., Sluis-Cremer, N. and Gupta, P. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28553933" target="_blank">Novel assay reveals a large, inducible, replication-competent HIV-1 reservoir in resting CD4<sup>+</sup> T cells.</a> <em>Nat Med</em> 23(7): 885-889. </li><li>Siliciano, J. D., Kajdas, J., Finzi, D., Quinn, T. C., Chadwick, K., Margolick, J. B., Kovacs, C., Gange, S. J. and Siliciano, R. F. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12754504" target="_blank">Long-term follow-up studies confirm the stability of the latent reservoir for HIV-1 in resting CD4<sup>+</sup> T cells.</a> <em>Nat Med</em> 9(6): 727-728. </li><li>Siliciano, J. D. and Siliciano, R. F. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16061962" target="_blank">Enhanced culture assay for detection and quantitation of latently infected, resting CD4<sup>+</sup> T-cells carrying replication-competent virus in HIV-1-infected individuals.</a> <em>Methods Mol Biol</em> 304: 3-15. </li><li>Siliciano, J. D. and Siliciano, R. F. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15163657" target="_blank">A long-term latent reservoir for HIV-1: discovery and clinical implications.</a> <em>J Antimicrob Chemother</em> 54(1): 6-9.<br/></li></ol>