<body>
            <h1>Enterovirus Competition Assay to Assess Replication Fitness</h1>
            <p><strong>Authors:</strong> Valeria  Lulla, Andrew E. Firth</p>
            <h2>Abstract</h2>
            <p>In virology the difference between the fitness of two viruses can be determined by using various methods, such as virus titer, growth curve analysis, measurement of virus infectivity, analysis of produced RNA copies and viral protein production. However, for closely performing viruses, it is often very hard to distinguish the differences. <em>In vitro</em> competition assays are a sensitive tool for determining viral replication fitness for many viruses replicating in cell culture. Relative viral replication fitness is usually measured from multiple cycle growth competition assays. Competition assays provide a sensitive measurement of viral fitness since the viruses are competing for cellular targets under identical growth conditions. This protocol describes a competition assay for enteroviruses and contains two alternative formats for initial infections, which can be varied depending on specific goals for each particular experiment. The protocol involves infection of cells with competing viruses, passaging, RNA extraction from infected cells, RT-PCR and Sanger sequencing followed by comparative analysis of resulting chromatograms obtained under various initial infection conditions. The techniques are applicable to members of many virus families, such as alphaviruses, flaviviruses, pestiviruses, and other RNA viruses with an established reverse genetics system.</p>
            <p><strong>Keywords:</strong> Enterovirus, Competition assay, Virus titration, Dual infection, Pairwise growth competition assay, Virus passaging, Sanger sequencing</p>
            <h2>Background</h2><p>Enteroviruses comprise a large group of mammalian pathogens that includes poliovirus. Pathology in humans ranges from sub-clinical to acute flaccid paralysis, myocarditis and meningitis. In our recent paper (Lulla <em>et al.</em>, 2019) we reported that many enterovirus genomes harbor an upstream open reading frame (uORF) that encodes an additional viral protein, UP (upstream protein). Using the echovirus 7 (EV7) reverse genetics system, we created two UP knock-out mutants (EV7-Loop and EV7-PTC), which exhibited wtEV7-like growth properties. Therefore, we decided to perform a competition assay to determine possible subtle fitness defects.</p>
        </body>

<p class="pptt" id="biaoti27782">## Materials and Reagents</p><ol class="npwone"><li>Materials<ol class="npwtwo"><li>Pipette tips (Fisher, catalog numbers: 02-707-426, 02-707-403, 02-707-438)</li><li>1.5 ml microfuge tubes (STARLAB, catalog number: 51615-5500)</li><li>T175 tissue culture flasks (TPP<sup>Â®</sup> tissue culture flasks, Sigma, catalog number: Z707562)</li><li>12-well polystyrene culture plates (TPP<sup>Â®</sup> tissue culture plates, Sigma, catalog number: Z707775)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Viruses<br/>Echovirus 7 (EV7), derived from EV7 infectious clone<br/><em>Note: The cDNA of Echovirus 7 strain Wallace was sourced from Michael Lindberg (GenBank accession number AF465516, with the silent substitution <sup>1687</sup>G-to-A) and was cloned downstream of a T7 RNA promoter. Mutant viruses EV7-Loop and EV7-PTC were generated by mutagenesis of the original EV7 infectious clone and prepared exactly as wt EV7 (Lulla </em><em>et al.</em><em>, 2019).</em><br/><br/></li><li>Cell lines<br/>RD cells (human rhabdomyosarcoma cell line, ATCC, CCL-136), verified mycoplasma-free by next-generation sequencing<br/><br/></li><li>Reagents<ol class="npwtwo"><li>Dulbeccoâs modified Eagleâs medium (DMEM) (Sigma, catalog number: D6546)</li><li>Fetal bovine serum (FBS) (PAN Biotech, catalog number: P40-37500)</li><li>Penicillin-streptomycin (10,000 U/ml) (Life Technologies, catalog number: 15140-122)</li><li>200 mM L-Glutamine (Gibco, catalog number: 25030081)</li><li>Phosphate buffered saline (PBS) (Life Technologies, catalog number: 14190-144)</li><li>1 M HEPES buffer, pH range: 7.2-7.5 (Life Technologies, catalog number: 15630-080)</li><li>0.25% Trypsin (Gibco, catalog number: 15090046-100)</li><li>Bovine serum albumin (BSA) (PAN Biotech, catalog number: P06-1395500)</li><li>Direct-zol<sup>TM</sup> RNA MiniPrep Plus (Zymo Research, catalog number: R2072) </li><li>Phusion<sup>TM</sup> RT-PCR Kit (Thermo Scientific, catalog number: F-546S)</li><li>Complete DMEM (see Recipes)</li><li>Serum-free DMEM (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti27783">### Equipment</p><ol class="npwtwok"><li>-80 Â°C freezer</li><li>Vortexer</li><li>Pipettes: 1000 Âµl, 200 Âµl, 20 Âµl, 2 Âµl</li><li>CO<sub>2</sub> incubator</li><li>Haemocytometer</li><li>BSL2 cell culture cabinet</li><li>Rocking platform shaker</li><li>Phase-contrast inverted microscope (Nikon TMS)</li><li>4 Â°C refrigerator</li><li>Autoclave<br/></li></ol><p class="pptt" id="biaoti27784">### Software</p><ol class="npwtwok"><li>BioEdit (Free software, Ibis Therapeutics, <a href="http://www.mbio.ncsu.edu/BioEdit/bioedit.html" target="_blank">http://www.mbio.ncsu.edu/BioEdit/bioedit.html</a>)</li></ol><p class="pptt" id="biaoti27785">### Procedure</p><p class="ppzz"><em>IMPORTANT NOTE: All experiments with viruses (i.e., all pre-Trizol RNA isolation steps) should be performed inside a biosafety level 2 (BSL2) tissue culture laboratory according to the country and institution regulations and required permits regarding enterovirus strains handling and storage.<br/><br/></em> </p><ol class="npwone"><li>Infection and virus passaging<ol class="npwtwo"><li>Prepare cells for infection 1 day before by seeding 3 x 10<sup>5</sup> RD cells per well on 12-well plates (the aim is to achieve 5 x 10<sup>5</sup> RD cells per well by the next day, which corresponds to subconfluent RD monolayers).<ol class="npwthree"><li>RD cells seeded on T175 flasks and grown in complete DMEM until 80-90% confluency. Aspirate media from the cells and rinse with 10 ml of PBS. Trypsinize RD cells using 3 ml of 0.25% trypsin by gently tapping the flask.</li><li>Once cells are detached, add 10 ml of complete DMEM, ensure the cells are properly resuspended by pipetting up and down. Count cells in suspension using hemocytometer.</li><li>Dilute cell suspension with complete DMEM to get concentration corresponding to 3 x 10<sup>5</sup> cells per 1 ml. Seed 1 ml of cell suspension to each well of 12-well plates.</li><li>Incubate overnight in 37 Â°C, 5% CO<sub>2</sub> incubator.</li></ol></li><li>Check the cell monolayers using a microscope (equal healthy cell monolayers, no CPE). Remove media from cell monolayers and infect with a total multiplicity of infection (MOI) of 0.1, in 0.2 ml of serum-free DMEM. Use the following formula to calculate the amount of virus needed:<br/><br/><img alt="" class="layerphoto" height="41" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513195248_2952.jpg" width="450"/><br/><br/><em>Example: For 10</em><sup><em>9</em></sup><em> PFU/ml virus stock use, 5 x 10</em><sup><em>5</em></sup><em> cells x 0.1 PFU/cell x 10</em><sup><em>3</em></sup><em> Î¼l/ml/10</em><sup><em>9</em></sup><em> PFU/ml = 0.05 Î¼l.</em> </li><li>Mix using wt EV7 and mutant at either equal or 1:9 proportions, with a total MOI 0.1.<br/><em>Note: This assay can be completed with other viruses and cell lines. Different proportions and MOI can be used for this experiment depending on the virus growth kinetics and susceptibility of cells for infection. Using MOI 0.1 means that more than 90% of the cells remain uninfected providing enough space for competing viruses; any growth difference should also accumulate over passaging.</em> </li><li>Use triplicates for each infection, monoinfections for positive controls and mock (uninfected) cells as negative controls as suggested in Figure 1.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513195422_5836.jpg" width="283"/><br/><strong>Figure 1. Sample layout of two 12-well plates at either equal (Left) or 1:9 (Right) proportions.</strong> Every subsequent passage is performed on similarly aranged 12-well plates.<br/><br/></li><li>Incubate infected cells at room temperature on a gently rocking platform for 1 h. To remove unbound virus, aspirate infection media, wash each well with 1 ml of serum-free DMEM, and overlay cells with 0.5 ml of serum-free DMEM.</li><li>Incubate at 37 Â°C in a CO<sub>2</sub> incubator overnight (16-20 h) until complete CPE is observed.</li><li>Collect media from infected plates to a 1.5 ml tube, centrifuge at 9,600 <em>x g</em> for 5 min, then transfer to the new 1.5 ml tube and measure the volume.</li><li>Add 0.2% BSA to supernatant (10 Î¼l of 10% BSA per 0.5 ml of clarified supernatant).</li><li>Perform 5 blind passages (with estimated, but unknown titer) using 1:10,000 volume of obtained virus stock (corresponds to MOI 0.05-0.2). Use the same infection protocol as before (steps 1-8).<br/><em>Note: For less efficiently replicating viruses more than 5 passages could be needed.</em> </li><li>Store virus stocks at -80 Â°C.</li><li>Add 1 ml PBS to remaining cells and collect them by pipetting into a 1.5 ml tube (infected cells detach easily and do not require scraping). Centrifuge cells at 9,600 <em>x g</em> for 5 min, remove supernatant and proceed to RNA isolation from the infected cells.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA isolation from infected cells<ol class="npwtwo"><li>Isolate RNA from passages 1 and 5 using Direct-zol<sup>TM</sup> RNA MiniPrep Plus (Zymo Research) according to the manufacturerâs protocol (<a href="https://files.zymoresearch.com/protocols/_r2070t_r2070_r2071_r2072_r2073_direct-zol_rna_miniprep_plus_kit.pdf" target="_blank">Direct-zol<sup>TM</sup> RNA MiniPrep Plus INSTRUCTION MANUAL ver.1.10.1</a>).</li><li>Perform RT-PCR using Thermo Scientific<sup>TM</sup> Phusion<sup>TM</sup> RT-PCR Kit using virus-specific primers and manufacturerâs recommendations (<a href="http://www.ulab360.com/files/prod/manuals/201305/13/78737001.pdf" target="_blank">Thermo Scientific Phusion RT-PCR Kit manual</a>). The example design of the introduced mutations and sequencing primer is presented in Figure 2.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513195459_6658.jpg" width="406"/><br/><strong>Figure 2. Example design of mutant virus genomes and sequencing primer.</strong> Depending on sequencing facilities, a 300-1000 bp long PCR fragment should be optimal for this assay.<br/><br/></li><li>Perform Sanger sequencing of the PCR fragment containing the mutated region of the virus genome.<br/><em>Note: Please consider the nature of the virus (intracellular or budded virions) for choosing the correct sample to analyze. Sometimes analyzing RNA from media rather than from infected cells can be considered. For enteroviruses it is not a crucial step; therefore an easier approach (RNA isolation from infected cells) was chosen.</em> </li></ol></li></ol><p class="pptt" id="biaoti27786">### Data analysis</p><p class="ppzz">Compare chromatograms and evaluate based on three RT-PCR products from each analyzed sample (Figure 3). In the experimental conditions tested viruses do not differ in their growth properties. The chromatograms should shift if the viruses differ in their growth properties. Compare the peak areas from at least three different chromatograms for each sample and estimate the changes. If wt virus out-competes the mutant, the chromatogram of the fifth passage sample should match the wt virus chromatogram.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190513/20190513195538_2979.jpg" width="156"/> <br/><strong>Figure 3. Sample sequencing chromatograms of RT-PCR products for the competition.</strong> Experiment of EV7, EV7-PTC and EV7-Loop mutants at initial infection (P1) and after five passages (P5). The figure is reproduced from Supplementary Figure 3 in Lulla <em>et al.</em> (2019).</p><p class="pptt" id="biaoti27787">### Recipes</p><ol class="npwtwok"><li>Complete DMEM<br/>500 ml DMEM<br/>50 ml FBS<br/>5 ml 200 mM L-Glutamine<br/>5 ml penicillin-streptomycin (10,000 U/ml)<br/>10 ml 1 M HEPES pH 7.2-7.5 <br/>Store at 4 Â°C</li><li>Serum-free DMEM<br/>500 ml DMEM<br/>5 ml 200 mM L-Glutamine<br/>5 ml penicillin-streptomycin (10,000 U/ml)<br/>10 ml 1 M HEPES, pH 7.2-7.5<br/>Store at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti27788">### Acknowledgments</p><p class="ppzz">This work was supported by Wellcome Trust grant [106207] and European Research Council grant [646891] to A.E.F. The cDNA of Echovirus 7 strain Wallace was sourced from Michael Lindberg. We thank Hazel Stewart for critical reading of the protocol.</p><p class="pptt" id="biaoti27789">### Competing interests</p><p class="ppzz">The authors declare that they do not have any conflicts of interests or competing interests.</p><p class="pptt" id="biaoti27790">### References</p><ol class="npwtwok"><li>Lulla, V., Dinan, A. M., Hosmillo, M., Chaudhry, Y., Sherry, L., Irigoyen, N., Nayak, K. M., Stonehouse, N. J., Zilbauer, M., Goodfellow, I. and Firth, A. E. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30478287" target="_blank">An upstream protein-coding region in enteroviruses modulates virus infection in gut epithelial cells.</a> <em>Nat Microbiol</em> 4(2): 280-292.</li></ol>