<body>
            <h1>An Improved Bioassay to Study <em>Arabidopsis</em> Induced Systemic Resistance (ISR) Against Bacterial Pathogens and Insect Pests</h1>
            <p><strong>Authors:</strong> NicolÃ¡s M. Cecchini, Yi   Song, Suruchi   Roychoudhry, Jean  T.  Greenberg, Cara  H.  Haney</p>
            <h2>Abstract</h2>
            <p>The plant immune system is essential for plants to perceive and defend against bacterial, fungal and insect pests and pathogens. Induced systemic resistance (ISR) is a systemic immune response that occurs upon root colonization by beneficial microbes. A well-studied model for ISR is the association of specific beneficial strains of <em>Pseudomonas</em> spp. with the reference plant <em>Arabidopsis thaliana</em>. Here, we describe a robust, increased throughput, bioassay to study ISR against the bacterial pathogen <em>Pseudomonas cannabina</em> pv. <em>alisalensis</em> (formerly called <em>Pseudomonas syringae</em> <em>pv. maculicola</em>) strain ES4326 and the herbivore <em>Trichoplusia ni</em> by inoculating <em>Pseudomonas simiae</em> strain WCS417 (formerly called <em>Pseudomonas fluorescens</em> WCS417) on <em>Arabidopsis</em> plants grown in Jiffy-7<sup>Â®</sup> peat pellets. While most commonly used for <em>Pseudomonas</em>-triggered ISR on <em>Arabidopsis</em>, this assay is effective for diverse rhizosphere bacterial strains, plant species, pathogens and herbivores.</p>
            <p><strong>Keywords:</strong> Induced Systemic Resistance (ISR), <em>Pseudomonas simiae</em>, <em>Arabidopsis thaliana</em>, <em>Pseudomonas syringae</em>, <em>Pseudomonas cannabina</em>, <em>Trichoplusia ni</em></p>
            <h2>Background</h2><p>After plant infection or colonization by pathogenic or commensal microbes, a systemic defense response can ensue involving immunological âmemoryâ or âprimingâ (Parker, 2009; Fu and Dong, 2013; Martinez-Medina <em>et al.</em>, 2016). Depending on the microbe and the associated plant organ, different systemic resistance programs are induced (Pieterse <em>et al.</em>, 2014). When roots encounter specific commensal microbes, induced systemic resistance (ISR) in distal root and shoot tissue is observed (Pieterse <em>et al.</em>, 1996; van Loon <em>et al.</em>, 1998). Although the genetic and hormonal signaling mechanisms deployed during the ISR response are partially understood (Verhagen <em>et al.</em>, 2004; Pieterse <em>et al.</em>, 2014; Pangesti <em>et al.</em>, 2016), many questions still remain unanswered, such as the mechanisms involved in root-microbiome communication or the identity of systemic signal(s) (Pieterse <em>et al.</em>, 2014; Haney <em>et al.</em>, 2018; Cecchini <em>et al.</em>, 2019). <br/><br/>The inoculation of <em>Arabidopsis thaliana</em> roots with <em>Pseudomonas simiae</em> WCS417 has been used as a model system for studying ISR against bacteria, fungi and herbivore invaders (Pieterse <em>et al.</em>, 1996; van Loon <em>et al.</em>, 1998; Verhagen <em>et al.</em>, 2004; Haney <em>et al.</em>, 2015). Existing bioassays for <em>P. simiae</em>-<em>Arabidopsis</em> ISR make use of sterile sand-germinated <em>Arabidopsis</em> followed by seedling transplantation into a sand/soil mixture containing the rhizobacteria (Pieterse <em>et al.</em>, 1996; Pozo <em>et al.</em>, 2008; van Oosten <em>et al.</em>, 2008). Building on this method, we have recently developed ISR bioassays against the bacterial pathogen <em>Pseudomonas cannabina</em> pv. <em>alisalensis</em> (formerly called <em>Pseudomonas syringae</em> pv. <em>maculicola</em>) strain ES4326 (<em>Pma</em>) and the herbivore <em>Trichoplusia ni</em> (<em>T. ni</em>) using <em>Arabidopsis</em> plants germinated and grown in Jiffy-7Â® peat pellets. This is an effective system to robustly trigger and quantify ISR, primarily because peat pellets have a constant volume and low fluorescent pseudomonad content (Haney <em>et al.</em>, 2015 and 2018; Cecchini <em>et al.</em>, 2019). Moreover, avoiding the transplantation of seedlings reduces the experimental labor and stress to the seedlings, thereby improving assay efficiency. Here, we describe a step-by-step bioassay methodology for ISR experiments that can be potentially standardized across laboratories worldwide.</p>
        </body>

<p class="pptt" id="biaoti27693">## Materials and Reagents</p><ol class="npwtwok"><li>96-multiwell plates (Corning, Costar, catalog number: 2797)</li><li>Kimwipes (Fisher, catalog number: 06-666A)</li><li>15 ml and 50 ml centrifuge tubes (Midsci<sup>TM</sup>, catalog numbers: 15 ml-CT2715, 50 ml-CT2750)</li><li>Plastic domes (Hummert International, catalog number: 11-33480) and trays (Hummert International, catalog number: 11-33010)</li><li>9 cm round Petri dishes (Fisher, catalog number: FB0875712)</li><li>1.5 ml microfuge tubes (Midsci<sup>TM</sup>, catalog number: AVSS1700)</li><li>Plastic pestles for 1.5 ml microfuge tubes (Fisher, catalog number: 12-141-364)</li><li>3 or 5 mm metal beads (QIAGEN, catalog number: 69997 or 69989)</li><li>Trays for holding the Jiffy-7<sup>Â®</sup> peat pellets (Hummert International, catalog number: 11311000)</li><li>1 ml-syringes without needle (BD Biosciences, catalog number: 309659)</li><li>Mesh bags (can be made by sewing a semi-oval out of a fine washable mesh material such as bridal veil; see Figure 3 for schematic; alternatively, perforated cellophane bread bags can be used)</li><li>Tightly closing plastic container (with screwcap lid) </li><li>Damp paper towel </li><li>Fine bristled paint brush</li><li>96-well racked collection microtubes (optional, for 96-well tissuelyser format) (QIAGEN 19560)</li><li><em>Arabidopsis thaliana</em> (L. Heyhn.) ecotype Columbia (Col-0)</li><li><em>Pseudomonas cannabina</em> pv. <em>alisalensis</em> (formerly called <em>P. syringae</em> pv. <em>maculicola</em> ES4326 (Bull <em>et al.</em>, 2010) carrying an empty vector (<em>Pma</em>DG3/<em>Pma</em>) (Guttman and Greenberg, 2001) bacterial culture (stored in 15% glycerol at -80 Â°C)</li><li><em>Trichoplusia ni</em> (eggs; Benzon Research or Natural Resources Canada)</li><li><em>Pseudomonas simiae</em> strain WCS417 (formerly called <em>Pseudomonas fluorescens</em> WCS417)</li><li>Jiffy-7<sup>Â®</sup> peat pellets (Jiffy products, Canada, Hummert International, catalog number: 14-23700)</li><li>Glycerol (Sigma-Aldrich, catalog number: G7757)</li><li>K<sub>2</sub>HPO<sub>4</sub> (Fisher, catalog number: P288)</li><li>MgSO<sub>4</sub> (Fisher, catalog number: BP213)</li><li>Proteose peptone No. 3 (BD Biosciences-US, catalog number: 211693)</li><li>Antibiotics<ol class="npwthree"><li>Rifampicin (GoldBio, catalog number: R-120)</li><li>Kanamycin (GoldBio, catalog number: K-120) </li></ol></li><li>Agar (Fisher, catalog number: BP1423)</li><li>Sterile distilled and tap water</li><li>Bleach (Clorox Concentrated Germicidal Bleach)</li><li>Ethanol (Decon Labs Inc., catalog number: 2701)</li><li>Triton X-100 (Fisher, catalog number: BP151)</li><li>Tryptone (BD Biosciences-US, catalog number: 211705)</li><li>Sodium chloride (NaCl)</li><li>Yeast extract (BD Biosciences-US, catalog number: 212750)</li><li>70% ethanol (see Recipes)</li><li>25% bleach supplemented with 0.1% Triton X-100 (see Recipes)</li><li>0.1% agar (see Recipes)</li><li>King's medium B (KB) (see Recipes)</li><li>Luria-Bertani medium (LB) (see Recipes)</li><li>10 mM MgSO<sub>4</sub> (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27694">### Equipment</p><ol class="npwtwok"><li>Laboratory glassware</li><li>200 Âµl and 1 ml, and 20-200 Âµl multichannel micropipettes (Gilson)</li><li>Forceps (Grainger, catalog number: 4CR15)</li><li>1 L beaker (Pyrex, 1000)</li><li>Handheld electric drill (DeWALT, model: DWD110)</li><li>Cork borer 4 mm diameter</li><li>Autoclave (Primus Sterilizer Co. Inc. 1317. C.R.N: 09415.1256)</li><li>Microcentrifuge (Eppendorf, model: 5415D)</li><li>Freezer (-80 Â°C) (Panasonic VIP Plus, model: MDF-V76VC-PA)</li><li>Plant Growth chamber at 20-23 Â°C with 12 h light/12 h dark (~75-100 Î¼mol s<sup>-1</sup>m<sup>-2</sup>, cool white fluorescent; A1000 Conviron Growth Chamber with <em>Arabidopsis</em> Kit, or similar. Alternatively, a growth-room with ~135-145 Î¼mol s<sup>-1</sup>m<sup>-2</sup> -mix 50/50 of 400-watt sodium and metal halide bulbs or 75-100 Î¼mol s<sup>-1</sup>m<sup>-2</sup> cool white fluorescent bulbs can be used)</li><li>Shaker incubator at 28 Â°C (Barnstead Max, model: Q 5000)</li><li>Balance (Mettler Toledo, model: PB1501)</li><li>Spectrophotometer (Bio-Mini SHIMADZU)</li><li>Laminar flow hood (SterilGARD 3 Advance)</li><li>Incubator at 28 Â°C (VWR, 3020)</li><li>Analytical balance for weighing caterpillars </li><li>TissueLyser Beadmill (QIAGEN, catalog number: 85300) with 24 or 96 adapters (QIAGEN, catalog number: 69982 or 69984)<br/></li></ol><p class="pptt" id="biaoti27695">### Procedure</p><ol class="npwone"><li>Hydration of Jiffy-7<sup>Â®</sup> peat pellets and <em>Arabidopsis</em> seed germination and growth (Figures 1A-1C)<ol class="npwtwo"><li>In 1.5 ml Eppendorf tubes, surface-sterilize 100-200 <em>Arabidopsis</em> seeds by washing with 70% ethanol for 2 min followed by 5 min with 25% bleach supplemented with 0.1% Triton X-100 and three washes in sterile water. Resuspend the seeds in 1-1.5 ml of a sterile solution of 0.1% agar (prepared in water and sterilized by autoclaving) using a pipette. Alternatively, chlorine gas sterilization can be used to eliminate endophytic bacteria, particularly to avoid microbial contamination for rhizosphere microbiome studies (Haney <em>et al.</em>, 2015). </li><li>Prior to use, leave the seeds for 3-7 days at 4 Â°C to allow for their stratification. </li><li>Prepare 8-12 Jiffy-7<sup>Â®</sup> pellets for each genotype per bacterial infection experiment (or 25-30 pellets per treatment for <em>T. ni</em> assays, see Data analysis) by placing them in a germination tray.</li><li>Hydrate Jiffy-7<sup>Â®</sup> peat pellets (Jiffy Products International, Canada) by immersing the base of the pellet with tap water. Allow the pellets to stand in water for at least 1 h to allow them to hydrate.</li><li>Once the pellets are hydrated, use a pair of clean/sterile forceps to gently even/level the top surface of the pellet. </li><li>Sow 3-4 stratified seeds by pipetting them at the surface of each Jiffy-7<sup>Â®</sup> pellet and cover the trays with a plastic dome. Do not cover the seeds with soil.</li><li>Transfer the trays to a growth chamber/room under 12 h day and 12 h night conditions at 23-20 Â°C day/night temperature regime, ~75-100 Î¼mol s<sup>-1</sup>m<sup>-2</sup> of cool white fluorescent light at rosette level, 50%-70% relative humidity. Alternatively, ~135-145 Î¼mol s<sup>-1</sup>m<sup>-2</sup> of mix 50/50 of sodium and metal halide light can be used.<br/><em>Note: Intensities of cool white fluorescence &gt; 125 Î¼mol s</em><sup><em>-1</em></sup><em>m</em><sup><em>-2</em></sup><em> cause plants to bolt between 3 and 4 weeks and so should not be used.</em> </li><li>At 3-7 days post germination, carefully remove extra seedlings with clean/sterile forceps from the surface of the pellets leaving only a single seedling in each pellet (do not water seedlings during the first week after planting). </li><li>Two to three times per week, bottom water the Jiffy pellets by flooding the tray with tap water and pouring the excess water off after 30-60 min. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>P. simiae</em> suspension preparation and seedling/pellets inoculation (Figures 1D-1F)<ol class="npwtwo"><li>When seedlings are eight days old, streak <em>P. simiae</em> WCS417 strain from a frozen glycerol stock onto a Kingâs B (KB) solid medium plate supplemented with Rifampicin (100 Î¼g/ml). Allow it to grow for 24 h at 28 Â°C. Pick a single colony, streak it to a new plate and grow it for another 24 h.</li><li>Inoculate 15 ml of media supplemented with Rifampicin (100 Î¼g/ml) (in a 50 ml Falcon tube) with a loop of confluent bacteria from the plate. Grow in a 28 Â°C shaker incubator for 16-24 h. </li><li>Centrifuge the culture at 3,000 <em>x g</em> for 5 min, discard the supernatant and resuspend the pellet in 15 ml of sterile 10 mM MgSO<sub>4</sub> solution. Repeat the centrifugation and finally resuspend the pellet in 15 ml 10 mM MgSO<sub>4</sub> by pipetting up and down.</li><li>Make a 1/10 or 1/20 dilution of the bacterial suspension (900 Âµl 10 mM MgSO<sub>4</sub> + 100 Âµl bacterial suspension for 1:10 or 950 Âµl 10 mM MgSO<sub>4</sub> + 50 Âµl bacterial suspension for 1:20 dilution) in a 1.5 ml tube and measure optical density at 600 nm (OD<sub>600</sub>) in a spectrophotometer to estimate the OD<sub>600</sub> of the original <em>P. simiae</em> culture. (Divide the measured OD by the dilution factor). </li><li>In a clean (non-sterile) beaker, prepare at least 1 L of the <em>P. simiae</em> WCS417 inoculum by diluting the bacterial suspension culture in 10 mM MgSO<sub>4</sub> to a final OD<sub>600</sub> of 0.01. </li><li>Inoculate the ten-day-old seedlings with 6 ml of the bacterial solution per plant by gently pipetting on the top surface of the Jiffy-7<sup>Â®</sup> pellet, near the base of the seedling, taking care to avoid contact with the leaves or hypocotyl. Then, submerge the entire pellets for ~30 s in the same solution (using the 1 L beaker with the suspension) without allowing contact of the bacteria suspension with the seedling aerial tissues. One liter of bacterial solution can support the inoculation of at least 16 pellets. For the control/mock-treated plants follow the same steps but only using sterile MgSO<sub>4</sub>.<br/><em>Alternatively: If multiple strains are inoculated onto plants within the same flat, to avoid cross-contamination of bacterial strains between pellets, 2 ml of more concentrated inoculum (OD</em><sub><em>600</em></sub><em> = 0.02-0.1) can be applied to the Jiffy pellets. For these assays, the pellets should be fairly dry (generally not watered after the initial soaking) and 2 days should pass after inoculation before flood watering the plants. Using this method, multiple bacterial strains and buffer-treated controls can be grown within the same flat with no detectable cross-contamination between pellets. Additionally, significantly smaller starting volumes of inoculum can be prepared.</em> </li><li>After these treatments, bottom water the plants as before for an additional 15-25 days (3.5-5 week-old plants) for <em>Pma</em> assays or 20 days (4.5 week-old plants) for <em>T. ni</em> assays. <br/><em>Note: Different genotypes can be planted side by side in the same tray. Rotate different trays 2-3 times per week to minimize growth variation.</em> </li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514010832_9153.jpg" width="151"/><br/><strong>Figure 1. Schematic of <em>P. simiae</em> inoculation.</strong> A. Arrange Jiffy-7<sup>Â®</sup> peat pellets onto flats as shown. B. Flood the base of the trays with tap water for at least 60 min to allow the pellets to hydrate. C. Sow previously sterilized and stratified <em>Arabidopsis</em> seeds onto the surface of hydrated Jiffy<sup>Â®</sup> pellets. Water the pellets by flooding the base of the tray three times per week. D. Ten days after germination, inoculate 6 ml of <em>P. simiae</em> solution with OD<sub>600</sub> = 0.01 onto the surface of the pellet taking care to avoid contact with the seedling aerial tissues. E. Immerse for ~30 s the Jiffy<sup>Â®</sup> pellet into a 1 L solution of <em>P. simiae</em> of OD<sub>600</sub> = 0.01. One liter of bacterial solution can support inoculation of at least 16 pellets. F. Continue bottom-watering the pellets with tap water for a further 15 days (or 20 days for <em>T. ni</em> assay) until the plants are ready for plants infection with <em>Pma</em> or <em>T. ni</em>.<br/><br/></li><li>Infections and growth of <em>Pma</em> or <em>T. ni</em> to analyze the effects of <em>P. simiae</em> WCS417 ISR<br/><strong><em>To evaluate ISR against </em></strong><strong><em>Pma</em></strong><strong><em> (Figure 2)</em></strong><br/><u>Growing <em>Pma</em> and infection</u> <ol class="npwtwo"><li>Inoculate O/N cultures of <em>Pseudomonas cannabina</em> pv <em>alisalensis</em> carrying an empty vector (<em>Pma</em>) (Guttman and Greenberg, 2001) strain in KB-media supplemented with Kanamycin (50 Î¼g/ml).</li><li>The next morning, dilute the cultures back in KB medium supplemented with Kanamycin (1:5 ml) and grow for additional 3-4 h. </li><li>Spin the bacteria down at 3,000 <em>x g</em> for 3 min and resuspend in sterile 10 mM MgSO<sub>4</sub>. Repeat the wash an additional time. </li><li>Measure OD<sub>600</sub> of the suspension by diluting 1:20 as described above in section B, point 4. Dilute the culture to an OD<sub>600</sub> = 0.2 or 0.3 and dilute to a final OD<sub>600</sub> = 0.0002-0.0003 by performing three 10-fold serial dilutions in sterile 10 mM MgSO<sub>4</sub>. The final volume should be 10 ml (~0.1 ml inoculum/leaf is required).</li><li>Use a 1 ml syringe without a needle pressed up against the abaxial side of the 5<sup>th</sup> and 6<sup>th</sup> <em>Arabidopsis</em> leaves to infiltrate the bacteria into the apoplast of an entire leaf. Infect at least two leaves on eight different plants. Use a felt-tipped marker to mark the petiole of infected leaves for identification leading to the sampling and quantification next steps (see below).<br/><em>Note: Inoculate plants around 1 pm.</em> </li><li>With a Kimwipe, dry the excess bacterial suspension from each infiltrated leaf. Let the leaves dry for 1 h until the liquid in the apoplast is no longer visible. Then, cover the plants with a plastic dome to increase the humidity and reduce the variability in bacterial growth. Return plants to the growth chamber for 2-3 days.</li></ol><br/><u>Quantifying bacterial growth in leaf tissues</u> <ol class="npwtwo" start="7"><li>Two or three days after <em>Pma</em> inoculation, excise 8 leaf discs from different plants per treatment from infected leaves (ISR-induced vs. mock) using a cork borer (4 mm diameter). Take the discs from approximately the same leaf position for all the samples, 3-5 mm from the leaf tip. </li><li>Using forceps, place each leaf disc in a 1.5-ml microfuge tube containing 200 Âµl of 10 mM MgSO<sub>4</sub>.</li><li>Grind the samples using a small hand-held electric drill with a plastic pestle. Thoroughly macerate the tissue until pieces of intact leaf tissue are no longer visible to the eye (avoiding heating up the samples by performing repeated short bursts of ~2 s instead of grinding continuously for a long-time interval). We suggest grinding each sample for a similar duration to minimize variability during the experiment. Samples can also be ground with the plastic pestle by hand (without the drill). <br/><em>Alternatively: Use a bead mill or TissueLyser to homogenize the samples. To use a TissueLyser, place the leaf disk into a 2 ml Eppendorf tube with a 5 mm metal bead and 100 Âµl water. Alternatively, a 96-well plate format can be used with 3 mm beads. Homogenize tissue for 2 min at 25 Hz.</em> </li><li>Next, vortex the homogenate and remove 20 Âµl from each sample and dilute in 180 Âµl 10 mM MgSO<sub>4</sub> in 96-well multi-well plate. Use a multichannel pipette to repeat this process 4 times to have a serial 1:10 dilution series (10<sup>-1</sup> to 10<sup>-5</sup>) for each sample. </li><li>Plate 20 Âµl aliquots of the 1:10 serial dilutions (10<sup>-2</sup> to 10<sup>-5</sup>) on KB medium plates supplemented with Kanamycin (50 Î¼g/ml of media) and allowed to dry onto the surface. Divide each KB plate with a marker such that at least eight samples/dilutions per plate can be plated.<br/><em>Alternatively: By pouring media into rectangular plates, 10 Âµl can be spotted with a multichannel pipette, and all samples can be plated on a single plate (see Figure 2). </em> </li><li>Incubate the plates at 28 Â°C for 2-3 days until colony-forming units (CFU) can be counted. Calculate the number of CFU per leaf disc by multiplying by the corresponding dilution factor (see Data analysis).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514010915_5883.jpg" width="144"/><br/><strong>Figure 2. Images showing the key steps to evaluate ISR against </strong><em><strong>Pma</strong></em><strong> in Arabidopsis grown in Jiffy-7</strong><sup><strong>Â®</strong></sup><strong> peat pellets.</strong> A. Infiltrate the bacteria suspension into the abaxial surface of <em>Arabidopsis</em> leaves with a 1 ml-syringe without a needle (<em>Pma</em>, OD<sub>600</sub> = 0.0003) (<em>Note: Inoculate plants at ~1 p.m.</em>). B. Two-three days after <em>Pma</em> infection, excise leaf discs from the infiltrated leaves using a cork borer (4 mm diameter). C. Place each leaf disc in a 1.5-ml microfuge tube with 200 Âµl of 10 mM MgSO<sub>4</sub> solution. D. Grind the samples using a plastic pestle mounted in a hand-held electric drill (shown) or a bead beater/TissueLyser (not shown). E. Grind the leaf discs until the tissue is thoroughly homogenized and no visible pieces of tissue remain (<em>Note: Avoid heating up the samples by performing ~2 s bursts</em>). F. Remove 20 Âµl from each sample and dilute in 180 Âµl 10 mM MgSO<sub>4</sub> in a multi-well plate and repeat this process to have a 1:10 dilution series (10<sup>-1</sup> to 10<sup>-5</sup>) for each sample. Plate 20 Âµl aliquots of each dilution on KB medium plates or 10 Âµl aliquots onto rectangular plates.</li></ol><br/><strong><em>To evaluate ISR against </em></strong><strong><em>Trichoplusia ni</em></strong><strong><em> (Figure 3)</em></strong> <ol class="npwtwo"><li>Incubate <em>T. ni</em> eggs (Benzon Research or Natural Resources Canada) at 30 Â°C for 36 h with 12-h light days. The photoperiod of the chamber should be in synch with that of the plants so the caterpillars are entrained with the same photoperiod.</li><li>Using a minimum of 25 plants per treatment at 4-5 weeks of age, randomly choose one newly hatched caterpillar larva from the batch. Using a small paint brush, place one larva at the center of the rosette of a Jiffy-7<sup>Â®</sup> pellet grown <em>Arabidopsis</em> rosette pretreated with <em>Pseudomonas</em> or buffer control.</li><li>Cover each pellet/plant with a mesh bag (Figure 3), return to a growth chamber, and allow the caterpillars to feed for 7 days. Plants should be at a density of no more than 30 per flat.</li><li>On the 7<sup>th</sup> day, remove the mesh bag and find the caterpillar on the plant. Weigh the larva with a precision balance to the nearest tenth of a milligram. The weight of newly hatched larvae is negligible; thus, the final caterpillar weight correlates with how much the caterpillar ate and host plant susceptibility (Cui <em>et al.</em>, 2002).</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514010938_8051.jpg" width="332"/><br/><strong>Figure 3. <em>T. ni</em> herbivory assay.</strong> A. Mesh bags can be made by cutting mesh (see materials) in a half-oval of ~200 x 200 mm and then using a sewing machine to sew two half-ovals together (dashed line). B. Place cheese cloth with <em>T. ni</em> eggs in a tightly sealing container with a damp paper towel at the bottom. Place the container in an incubator at 30 Â°C with the same light regime as the plants until the larvae hatch (~24-36 h). C. Using a fine paint brush, transfer a small number of newly hatch <em>T. ni</em> larvae to a Petri plate or other small container. Reseal the original container. Using the paint brush, place 1 larva on the center of the rosette of a 4-5 week-old <em>Arabidopsis</em> plant. D. Cover each plant with a mesh bag and secure at the base with a rubber band. Return the plants to the flats and return flats to the incubator. Allow caterpillars to feed for 1 week. E. One week later, find and weigh each individually (to the nearest 0.1 mg).<br/></li></ol><p class="pptt" id="biaoti27696">### Data analysis</p><p class="ppzz"><strong><em>ISR-</em></strong><strong><em>Pma</em></strong><strong><em> data analysis</em></strong><br/>When the colonies are still small, count the CFUs for one of the dilutions (where 10-50 CFU can be counted and colonies are clearly distinct) for each disc from one leaf per plant taken and estimate the number of bacteria by multiplying with the corresponding dilution factor. Perform CFU counting for at least eight discs (12 discs provide greater statistical power and allows for increased confidence in smaller differences in bacterial growth). To increase confidence in possible differences in growth, replicate the entire experiment at least 3 independent times on different days and from different batches of inoculum. Use all the data collected from all independent experiments performed to calculate the average CFU per leaf disc plus/minus standard error. To determine if differences are statistically significant, perform analysis of variance (ANOVA) and a post-hoc test such Tukeyâs HSD test or Newman-Keuls (SNK) by using appropriate statistical software (Figure 4D and methods in Cecchini <em>et al.</em>, 2019). If only two conditions are being compared, a Studentâs <em>t</em>-test can be used. Plot the data on a log<sub>10</sub> scale. <br/><br/><strong><em>ISR-</em></strong><strong><em>T. ni</em></strong><strong><em> data analysis</em></strong><br/>Weigh at least 25 larvae/plants per treatment, per experiment and repeat the experiment a minimum of 3 independent times. Use all the data collected from all independent experiments performed to calculate the average weight per larvae plus/minus standard error. To determine if differences are statistically significant, perform analysis of variance (ANOVA) and a post hoc test such Tukeyâs HSD test or Newman-Keuls (SNK) by using appropriate statistical software. If only two conditions are being compared, a Studentâs <em>t</em>-test can be used. If the experiments are done in different chambers or with insects from different sources, a significant replicate effect has been observed (Haney <em>et al.</em>, 2018). As a result, data can be normalized to the buffer-treated Col-0 control data from a single experiment prior to averaging the data from at least 3 independent experiments.</p><p class="pptt" id="biaoti27697">### Notes</p><p class="ppzz">Using Jiffy-7<sup>Â®</sup> has allowed us to study interactions between diverse below and above ground commensals and pathogens. This setup has been used to study diverse rhizosphere bacteria that induce systemic resistance or susceptibility (Haney <em>et al.</em>, 2015 and 2018; Melnyk <em>et al.</em>, 2019). Additionally, it can be used to study below ground effects on diverse above ground pathogens including pathogenic <em>Pseudomonas</em> and <em>Xanthomonas</em> spp. (Haney <em>et al.</em>, 2018) and fungal pathogens such as <em>Botrytis</em>, and downy mildew.</p><p class="pptt" id="biaoti27698">### Recipes</p><ol class="npwtwok"><li>70% ethanol<br/>73.7 ml of 95% ethanol<br/>Add distilled water up to 100 ml</li><li>25% bleach supplemented with 0.1% Triton X-100<br/>25 ml bleach and 100 Âµl of Triton X-100<br/>Add distilled water to 100 ml</li><li>0.1% agar<br/>Dissolve 0.1 g agar in 100 ml sterile distilled water by autoclaving, swirl the solution while it cools</li><li>King's medium B (KB)<br/>20 g of proteose peptone No. 3<br/>10 g glycerol<br/>1.5 g MgSO<sub>4</sub><br/>1.2 g K<sub>2</sub>HPO<sub>4</sub><br/>For solid medium add 13 g agar<br/>Add distilled water up to 1 L<br/>Sterilize by autoclaving</li><li>Luria-Bertani medium (LB)<br/>10 g tryptone<br/>10 g sodium chloride (NaCl)<br/>5 g yeast extract<br/>Add distilled water up to 1 L<br/>Sterilize by autoclaving</li><li>10 mM MgSO<sub>4</sub> <br/>0.12 g of MgSO<sub>4</sub><br/>Add distilled water to 100 ml<br/>Sterilize by autoclaving<br/></li></ol><p class="pptt" id="biaoti27699">### Acknowledgments</p><p class="ppzz">We thank Elizabeth Baldo and Joanna Jelenska for helping identify catalog numbers for materials and equipment. <br/>  This work was supported by NSF grant IOS1456904 to JTG, by an NSERC Discovery Grant (NSERCRGPIN-2016-04121) awarded to CHH, a China Postdoctoral Science Foundation Fellowship awarded to YS and an Agencia Nacional de PromociÃ³n CientÃ­fica y TecnolÃ³gica grant (PICT-2017-0589) awarded to NMC. NMC is a Career Investigator of CONICET (Argentina).</p><p class="pptt" id="biaoti27700">### Competing interests</p><p class="ppzz">The authors declare that they have no conflict of interests.</p><p class="pptt" id="biaoti27701">### References</p><ol class="npwtwok"><li>Berendsen, R. L., van Verk, M. C., Stringlis, I. A., Zamioudis, C., Tommassen, J., Pieterse, C. M. and Bakker, P. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26198432" target="_blank">Unearthing the genomes of plant-beneficial <em>Pseudomonas</em> model strains WCS358, WCS374 and WCS417.</a> <em>BMC Genomics</em> 16: 539.</li><li>Bull, C. T., Manceau, C., Lydon, J., Kong, H., Vinatzer, B. A. and Fischer-Le Saux, M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20227217" target="_blank"><em>Pseudomonas cannabina</em> pv. <em>cannabina</em> pv. nov., and <em>Pseudomonas cannabina</em> pv. <em>alisalensis</em> (Cintas Koike and Bull, 2000) comb. nov., are members of the emended species <em>Pseudomonas cannabina</em> (ex Sutic &amp; Dowson 1959) Gardan, Shafik, Belouin, Brosch, Grimont &amp; Grimont 1999.</a> <em>Syst Appl Microbiol</em> 33(3): 105-115.</li><li>Cecchini, N. M., Roychoudhry, S., Speed, D. J., Steffes, K., Tambe, A., Zodrow, K., Konstantinoff, K., Jung, H. W., Engle, N. L., Tschaplinski, T. J. and Greenberg, J. T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30156481" target="_blank">Underground azelaic acid-conferred resistance to <em>Pseudomonas syringae </em>in <em>Arabidopsis</em>.</a> <em>Mol Plant Microbe Interact</em> 32(1): 86-94.</li><li>Cui, J., Jander, G., Racki, L. R., Kim, P. D., Pierce, N. E. and Ausubel, F. M. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12068100" target="_blank">Signals involved in <em>Arabidopsis</em> resistance to <em>Trichoplusia ni</em> caterpillars induced by virulent and avirulent strains of the phytopathogen <em>Pseudomonas syringae</em>.</a> <em>Plant Physiol</em> 129(2): 551-564.</li><li>Fu, Z. Q. and Dong, X. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23373699" target="_blank">Systemic acquired resistance: turning local infection into global defense.</a> <em>Annu Rev Plant Biol</em> 64: 839-863.</li><li>Guttman, D. S. and Greenberg, J. T. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11204777" target="_blank">Functional analysis of the type III effectors AvrRpt2 and AvrRpm1 of <em>Pseudomonas syringae</em> with the use of a single-copy genomic integration system.</a> <em>Mol Plant Microbe Interact</em> 14(2): 145-155.</li><li>Haney, C. H., Samuel, B. S., Bush, J. and Ausubel, F. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27019743" target="_blank">Associations with rhizosphere bacteria can confer an adaptive advantage to plants.</a> <em>Nat Plants</em> 1(6): pii: 15051.</li><li>Haney, C. H., Wiesmann, C. L., Shapiro, L. R., Melnyk, R. A., O'Sullivan, L. R., Khorasani, S., Xiao, L., Han, J., Bush, J., Carrillo, J., Pierce, N. E. and Ausubel, F. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29087012" target="_blank">Rhizosphere-associated <em>Pseudomonas</em> induce systemic resistance to herbivores at the cost of susceptibility to bacterial pathogens.</a> <em>Mol Ecol</em> 27(8): 1833-1847.</li><li>van Loon, L. C., Bakker, P. A. H. M., and Pieterse, C. M. J. (1998). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15012509" target="_blank">Systemic resistance induced by rhizosphere bacteria.</a> <em>Annu Rev Phytopathol</em> 36: 453-483. </li><li>Martinez-Medina, A., Flors, V., Heil, M., Mauch-Mani, B., Pieterse, C. M. J., Pozo, M. J., Ton, J., van Dam, N. M. and Conrath, U. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27507609" target="_blank">Recognizing plant defense priming.</a> <em>Trends Plant Sci</em> 21(10): 818-822.</li><li>Melnyk, R. A., Beskrovnaya, P., Liu, Z., Song, Y. and Haney, C. H. (2019). <a href="https://www.biorxiv.org/content/10.1101/517870v1" target="_blank">Bacterially produced spermidine induces plant systemic susceptibility to pathogens.</a> <em>bioRxiv</em>: 517870.</li><li>van Oosten, V. R., Bodenhausen, N., Reymond, P., van Pelt, J. A., van Loon, L. C., Dicke, M. and Pieterse, C. M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18533832" target="_blank">Differential effectiveness of microbially induced resistance against herbivorous insects in <em>Arabidopsis</em>.</a> <em>Mol Plant Microbe Interact</em> 21(7): 919-930.</li><li>Pangesti, N., Reichelt, M., van de Mortel, J. E., Kapsomenou, E., Gershenzon, J., van Loon, J. J., Dicke, M. and Pineda, A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27848154" target="_blank">Jasmonic acid and ethylene signaling pathways regulate glucosinolate levels in plants during rhizobacteria-induced systemic resistance against a leaf-chewing herbivore.</a> <em>J Chem Ecol</em> 42(12): 1212-1225.</li><li>Parker, J. E. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19436056" target="_blank">The quest for long-distance signals in plant systemic immunity.</a> <em>Sci Signal</em> 2(70): pe31.</li><li>Pieterse, C. M., Zamioudis, C., Berendsen, R. L., Weller, D. M., Van Wees, S. C. and Bakker, P. A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24906124" target="_blank">Induced systemic resistance by beneficial microbes.</a> <em>Annu Rev Phytopathol</em> 52: 347-375.</li><li>Pieterse, C. M., van Wees, S. C., Hoffland, E., van Pelt, J. A. and van Loon, L. C. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8776893" target="_blank">Systemic resistance in <em>Arabidopsis</em> induced by biocontrol bacteria is independent of salicylic acid accumulation and pathogenesis-related gene expression.</a> <em>Plant Cell</em> 8(8): 1225-1237.</li><li>Pozo, M. J., Van Der Ent, S., Van Loon, L. C. and Pieterse, C. M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18657213" target="_blank">Transcription factor MYC2 is involved in priming for enhanced defense during rhizobacteria-induced systemic resistance in <em>Arabidopsis thaliana</em>.</a> <em>New Phytol</em> 180(2): 511-523.</li><li>Verhagen, B. W., Glazebrook, J., Zhu, T., Chang, H. S., van Loon, L. C. and Pieterse, C. M. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15305611" target="_blank">The transcriptome of rhizobacteria-induced systemic resistance in <em>Arabidopsis</em>.</a> <em>Mol Plant Microbe Interact</em> 17(8): 895-908.<br/></li></ol>