<body>
            <h1>Biofilm Formation Assay in <em>Pseudomonas syringae</em></h1>
            <p><strong>Authors:</strong> Xiaolong Shao, Yingpeng Xie, Yingchao Zhang, Xin   Deng</p>
            <h2>Abstract</h2>
            <p><em>Pseudomonas syringae</em> is a model plant pathogen that infects more than 50 plant species worldwide, thus leading to significant yield loss. <em>Pseudomonas</em> biofilm always adheres to the surfaces of medical devices or host cells, thereby contributing to infection. Biofilm formation can be visualized on numerous matrixes, including coverslips, silicone tubes, polypropylene and polystyrene. Confocal laser scanning microscopy can be used to visualize and analyze biofilm structure. In this study, we modified and applied the current method of <em>P. aeruginosa</em> biofilm measurement to <em>P. syringae</em>, and developed a convenient protocol to visualize <em>P. syringae</em> biofilm formation using a borosilicate glass tube as the matrix coupled with crystal violet staining.</p>
            <p><strong>Keywords:</strong> Biofilm, <em>Pseudomonas syringae</em>, Plant pathogen, Borosilicate glass, Crystal violet, Visualization</p>
            <h2>Background</h2><p>Most <em>Pseudomonas</em> strains secrete exopolysaccharides, such as alginate, which is an important matrix molecule for biofilm formation (Hentzer <em>et al.</em>, 2001; Nivens <em>et al.</em>, 2001). Biofilm formed by the human pathogen <em>P. aeruginosa</em> plays important roles in its virulence and antibiotic resistance, and contributes to acute or chronic infections (Donlan and Costerton, 2002). <br/><br/>To date, various methods have been reported for biofilm characterization and quantification. Originally, biofilms were detected in microtiter plates made of polystyrene or polypropylene (O'Toole and Kolter, 1998; Merritt <em>et al.</em>, 2005). During the growth of <em>P. aeruginosa</em> on a surface, the expression of genes involved in extracellular polysaccharide synthesis is induced (Davies <em>et al.</em>, 1993; Davies <em>et al.</em>, 1995), which promotes the adherence of cells to the surface. Crystal violet specifically stains the bacterial cells, and has been developed as a widely used dye for bacterial biofilm (George <em>et al.</em>, 1998). Some recent studies have analyzed biofilm structure in a flow chamber coupled with confocal laser scanning microscopy (Sternberg and Tolker-Nielsen, 2006; Chua <em>et al.</em>, 2016). <br/><br/>Biofilms formed by <em>P. syringae</em> strains have also been found in plant tissues (Osman <em>et al.</em>, 1986; Fakhr <em>et al.</em>, 1999; Preston <em>et al.</em>, 2001). Alginate produced by <em>P. syringae</em> is an important polymer for <em>P. syringae</em> biofilm formation and contributes to its virulence and fitness, indicating its importance in plant-pathogen interaction (Preston <em>et al.</em>, 2001; Engl <em>et al.</em>, 2014). The formation of <em>P. syringae</em> and <em>P. fluorescens</em> biofilms can also be measured using crystal violet staining in microwell plates (Carezzano <em>et al.</em>, 2017; Zhu <em>et al.</em>, 2018; Patange <em>et al.</em>, 2019). <br/><br/>In this study, we modified and applied the current method of <em>P. aeruginosa</em> biofilm measurement to <em>P. syringae</em> (Kong <em>et al.</em>, 2015; Zhao <em>et al.</em>, 2016; Shao <em>et al.</em>, 2018). We present an economic, rapid and visual biofilm detection protocol that combines the use of borosilicate glass tubes and crystal violet staining methods, which have been efficiently used in our recent studies (Wang <em>et al.</em>, 2018; Wang <em>et al.</em>, 2019; Xie <em>et al.</em>, 2019) for visualizing the biofilm of the model plant pathogen <em>P. syringae</em>.<br/></p>
        </body>

<p class="pptt" id="biaoti27806">## Materials and Reagents</p><ol class="npwtwok"><li>10 ml Borosilicate glass tube (ISOLAB, catalog number: 077.02.003)</li><li>14 ml sterile tube (SPL Lifescience, catalog number: 40014)</li><li>Filter (PALL Lifesciences, catalog number: AP-4219)</li><li>Strains <em>P. syringae</em> pv. phaseolicola 1448A (<em>Psph</em>) (Xiao <em>et al.</em>, 2007) and <em>rhpS</em> deletion mutant (Î<em>rhpS</em>) (Xie <em>et al.</em>, 2019)</li><li>NaOH (UNI-CHEM, catalog number: 1310-73-2) </li><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Aladdin, catalog number: 10025-84-0)</li><li>K<sub>2</sub>HPO<sub>4</sub> (Aladdin, catalog number: 7758-11-4) </li><li>Bacto<sup>TM</sup> Proteose peptone No.3 (AOBOX, catalog number: 01-049)</li><li>Rifampin (Aladdin, catalog number: 13292-46-1)</li><li>Agar (MP Biomedicals, catalog number: 9002-18-0)</li><li>Crystal violet (Beijing Dingguo, catalog number: 548-62-9)</li><li>Glycerol (Beijing Bailingwei, catalog number: 262536)</li><li>100% ethanol (Honeywell, catalog number: 32221-2.5L)</li><li>King's B (KB) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27807">### Equipment</p><ol class="npwtwok"><li>1 ml pipette (Eppendorf, catalog number: 3123000063)</li><li>Benchtop shaking incubator (Labwit Scientific, model: ZWYR-240)</li><li>Constant temperature incubator (Labwit Scientific, model: ZXDP-B2120) </li><li>Evolution<sup>TM</sup> 350 UV-Vis Spectrophotometer (Thermo Fisher Scientific, catalog number: 912A0959)</li><li>Synergy<sup>TM</sup> 2 Multi-Mode Microplate Reader (BioTek)</li><li>Test tube stand (ISOLAB, catalog number: 079.01.005)</li><li>-80 Â°C freezer (Thermo Scientific, catalog number: 5IDTSX)<br/></li></ol><p class="pptt" id="biaoti27808">### Software</p><ol class="npwtwok"><li>Microsoft Office Excel 2016 and GraphPad Prism 8.0.2.</li></ol><p class="pptt" id="biaoti27809">### Procedure</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwtwo"><li><em>Select wild-type </em><em>P. syringae</em><em> pv. phaseolicola 1448A (</em><em>Psph</em><em>) as the model strain (Xiao </em><em>et al.</em><em>, 2007) and </em><em>rhpS</em><em> deletion mutant (Î</em><em>rhpS</em><em>) (Xie </em><em>et al.</em><em>, 2019) as the test strain.</em> </li><li><em>Perform the step-by-step protocol described in Figure 1.</em> </li><li><em>Perform the entire procedure gently to avoid damaging the biofilm.</em> </li><li><em>Collect the liquid crystal violet and ethanol waste liquor in specially labeled containers for professional disposal by trained staff, as per the safety regulations at City University of Hong Kong.</em><br/><br/><img alt="" class="layerphoto" height="135" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514193538_1604.jpg" width="100"/><br/><strong>Figure 1. Schematic step-by-step protocol for visualizing </strong><em><strong>P. syringae</strong></em><strong> biofilm formation.</strong> A. The <em>Psph</em> strain was activated on Kingâs B (KB) plate and cultured in liquid medium. B. Then the cultures were inoculated at 1:500 dilutions into KB liquid medium and incubated statically to sampling points. C. Then stained the biofilm by using 0.1% crystal violet. D. Measure the biofilm production at OD<sub>590nm</sub>. Two biological replicates were showed.<br/><br/></li></ol><ol class="npwone"><li>Bacterial growth<ol class="npwtwo"><li>Collect a <em>Psph</em> colony from the glycerol stock culture frozen at -80 Â°C and inoculate on a Kingâs B (KB) plate supplemented with rifampicin (25 Î¼g/ml). Incubate the KB plate at 28 Â°C for 36 h in a constant temperature incubator.</li><li>Collect a single colony from the cultured plates and inoculate into a sterile 10 ml tube containing 2 ml KB liquid medium supplemented with rifampicin (25 Î¼g/ml). Incubate the tube in a benchtop shaking incubator at 28 Â°C for 12 h with constant shaking at 220 rpm.</li></ol><br/></li><li>Biofilm formation<br/>Inoculate the <em>Psph</em> culture (1:500 dilutions) into 16 sterile 10 ml borosilicate glass tubes containing 2 ml KB liquid medium (supplemented with rifampicin) and ensure consistent initial dose. Incubate the <em>Psph</em>-containing glass tubes at 28 Â°C in a constant temperature incubator without shaking.<br/><br/></li><li>Biofilm visualization<ol class="npwtwo"><li>Harvest the biofilm samples at different time points. For <em>Psph</em> in our study, the biofilms were harvested at 24, 48, 72 and 96 h. Gently discard the planktonic cells with a 1 ml pipette and wash the tubes three times with sterile distilled water. Avoid damaging the biofilm formed on the tube wall.</li><li>Stain the biofilm forming bacteria with 2.5 ml 0.1% crystal violet for 20 min without shaking. Discard the dye and wash the tubes with sterile distilled water to remove the unbound dye. Dry the tubes and take photographs (Figure 2A).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Biofilm measurement<ol class="npwtwo"><li>Elute the biofilm with 2 ml 100% ethanol and shake the tubes at 220 rpm for 20 min to ensure that the dye has dissolved completely. Take photographs (Figure 2B).</li><li>Measure the eluted samples at OD<sub>590nm</sub> using a spectrophotometer (2 ml) or Synergy 2 Plate Reader (BioTek) (100 Î¼l). If the sample concentrations are too high, diluted them before measuring. Use an equal volume of 95%-100% ethanol as the blank control. <em>Psph</em> biofilm formation is shown in Figure 2C. The Î<em>rhpS</em> produces lower biofilm compared with <em>Psph</em> wild-type strain (Figure 3).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514193650_8557.jpg" width="154"/><br/><strong>Figure 2. Visualization and quantification of biofilm in <em>Psph</em>wild-type strain.</strong> A. Biofilm samples were grown from 24 to 96 h. Biofilm adhered to borosilicate glass tubes at different time points were stained with crystal violet. B. The crystal violet bound to the biofilm on the wall of the tubes was eluted by ethanol. C. The elution samples were measured at OD<sub>590nm</sub> by using spectrophotometer or Synergy 2 Plate Reader (BioTek). <em>Psph</em> wild-type strain produced more biofilm at 96 h than 24 h. * represents <em>P</em>-value &lt; 0.05. *** represents <em>P</em>-value &lt; 0.001. Error bars indicate S.D. among four biological replicates. Two biological replicates were shown. <br/><br/><img alt="" class="layerphoto" height="109" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514193714_4941.jpg" width="100"/><br/><strong>Figure 3. The Î<em>rhpS</em> strain produced less biofilm than did that in <em>Psph</em> wild-type strain.</strong> A. Biofilm produced by the <em>Psph</em> wild-type and the Î<em>rhpS</em> strain were visualized using borosilicate glass tubes and stained with crystal violet at 96 h. B. The crystal violet bound to the biofilm on tube wall was eluted by ethanol. C. The <em>Psph</em> wild-type strain produced more biofilm than the Î<em>rhpS</em> strain (<em>P</em>-value = 0.000136). *** represents <em>P</em>-value &lt; 0.001. Error bars indicate S.D. among four biological replicates. Two biological replicates were shown.</li></ol></li></ol><p class="pptt" id="biaoti27810">### Data analysis</p><p class="ppzz">Studentâs <em>t</em>-tests were performed using Microsoft Office Excel 2016. Quantitative data (OD<sub>590nm</sub>) were collected from four biological replicates in the figures and tables (Tables 1 and 2).<br/><br/><strong>Table 1. Biofilm production of </strong><em><strong>Psph</strong></em><strong> wild-type strain presented in Figure 2C.</strong> Two-sample equal variance was calculated by the following one-tailed Studentâs <em>t</em>-test formula in Excel = TTEST (array1, array2, tails, type). Array 1 is the first data set. Array 2 is the second data set. Tails show the number of distribution tails (1 for the one-tailed distribution, 2 for two-tailed distribution). Type is the kind of <em>t</em>-test to perform (1 for paired, 2 for two-sample equal variance, and 3 for two-sample unequal variance. For example, we used TTEST (B2:E2, B3:E3, 1, 2), TTEST (B2:E2, B4:E4, 1, 2) and TTEST (B2:E2, B5:E5, 1, 2) for biofilm production at 48 h, 72 h, 96 h, compared to 24 h respectively. The <em>P</em>-values were showed in Excel H3, H4 and H5 respectively. * represents <em>P</em>-value &lt; 0.05. ** represents <em>P</em>-value &lt; 0.01. *** represents <em>P</em>-value &lt; 0.001. Means and S.D. are shown in columns F and G. All experiments were repeated four times.<br/><img alt="" class="layerphoto" height="211" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514193803_8678.jpg" width="600"/><br/><br/><strong>Table 2. Biofilm production of the </strong><em><strong>Psph</strong></em><strong> wild-type and the Î</strong><em><strong>rhpS</strong></em><strong> strain presented in Figure 3C.</strong> Two-sample equal variance was calculated by the following Two-tailed Studentâs <em>t</em>-test formula in Excel = TTEST (B2:E2, B3:E3, 2, 2). The <em>P</em>-values were showed in Column H3. * represents <em>P</em>-value &lt; 0.05. ** represents <em>P</em>-value &lt; 0.01. *** represents <em>P</em>-value &lt;0.001. Means and S.D. are shown in Column F and G. All experiments were repeated four times.<br/><img alt="" class="layerphoto" height="154" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190514/20190514193835_7912.jpg" width="600"/><br/><br/>In sum, the results showed that the wild-type <em>Psph</em> strain produced more biofilm in 48 h (<em>P</em> &lt; 0.05), 72 h (<em>P</em> &lt; 0.001) and 96 h (<em>P</em> &lt; 0.001) than at 24 h (Figure 2 and Table 1). Besides, the Î<em>rhpS</em> strain produced less biofilm than <em>Psph</em> wild-type strain (<em>P</em> &lt; 0.001) at the same time point (96 h) (Figure 3 and Table 2).</p><p class="pptt" id="biaoti27811">### Recipes</p><ol class="npwtwok"><li>King's B (KB) (King <em>et al.</em>, 1954)<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:350px;"><tbody><tr><td>Bacto<sup>TM</sup> Proteose peptone No.3<br/></td><td>20.0 g/L<br/></td></tr><tr><td>K<sub>2</sub>HPO<sub>4</sub><br/></td><td>1.5 g/L<br/></td></tr><tr><td>Glycerol<br/></td><td>15 ml/L</td></tr></tbody></table>Dissolve in 993.75 ml ddH<sub>2</sub>O and adjust the pH to 7.2. Add 15 g/L agar for solidified media and autoclave<br/>Dissolve 24.637 g MgSO<sub>4</sub>Â·7H<sub>2</sub>O in 100 ml ddH<sub>2</sub>O and sterilize this stock solution (1 M) using sterile filter (0.45 Âµm)<br/>Then, add 6.25 ml MgSO<sub>4</sub>Â·7H<sub>2</sub>O (1 M) (final concentration 1.5 g/L) into the autoclaved King's B medium when the temperature drops to 40 Â°C-50 Â°C<br/></li></ol><p class="pptt" id="biaoti27812">### Acknowledgments</p><p class="ppzz">This project has been funded in Health Medical Research Fund [17160022 to X.D.]; National Natural Science Foundation of China Grants [31670127 to X.D.].<br/>  This protocol was adapted from O'Toole <em>et al.</em> (1998) Flagellar and twitching motility are necessary for <em>Pseudomonas aeruginosa</em> biofilm development. <em>Molecular Microbiology</em>, 30: 295-304.</p><p class="pptt" id="biaoti27813">### Competing interests</p><p class="ppzz">Conflict of interest statement: None declared.</p><p class="pptt" id="biaoti27814">### References</p><ol class="npwtwok"><li>Carezzano, M. E., Sotelo, J. P., Primo, E., Reinoso, E. B., Paletti Rovey, M. F., Demo, M. S., Giordano, W. F. and Oliva, M. L. M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28403565" target="_blank">Inhibitory effect of <em>Thymus vulgaris</em> and <em>Origanum vulgare</em> essential oils on virulence factors of phytopathogenic <em>Pseudomonas syringae</em> strains.</a> <em>Plant Biol (Stuttg)</em> 19(4): 599-607.</li><li>Chua, S. L., Yam, J. K., Hao, P., Adav, S. S., Salido, M. M., Liu, Y., Givskov, M., Sze, S. K., Tolker-Nielsen, T. and Yang, L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26892159" target="_blank">Selective labelling and eradication of antibiotic-tolerant bacterial populations in <em>Pseudomonas aeruginosa</em> biofilms.</a> <em>Nat Commun</em> 7: 10750.</li><li>Donlan, R. M. and Costerton, J. W. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11932229" target="_blank">Biofilms: survival mechanisms of clinically relevant microorganisms.</a> <em>Clin Microbiol Rev</em> 15(2): 167-193.</li><li>Davies, D. G., Chakrabarty, A. M. and Geesey, G. G. (1993). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8476292" target="_blank">Exopolysaccharide production in biofilms: substratum activation of alginate gene expression by <em>Pseudomonas aeruginosa</em>. </a><em>Appl Environ Microbiol</em> 59(4): 1181-1186.</li><li>Davies, D. G. and Geesey, G. G. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7793920" target="_blank">Regulation of the alginate biosynthesis gene algC in <em>Pseudomonas aeruginosa</em> during biofilm development in continuous culture.</a> <em>Appl Environ Microbiol</em> 61(3): 860-867.</li><li>Engl, C., Waite, C. J., McKenna, J. F., Bennett, M. H., Hamann, T. and Buck, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24846383" target="_blank">Chp8, a diguanylate cyclase from <em>Pseudomonas syringae</em> pv. Tomato DC3000, suppresses the pathogen-associated molecular pattern flagellin, increases extracellular polysaccharides, and promotes plant immune evasion.</a> <em>MBio</em> 5(3): e01168-01114.</li><li>Fakhr, M. K, Penaloza-Vazquez, A, Chakrabarty, A. M, Bender, C. L. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10348861" target="_blank">Regulation of alginate biosynthesis in <em>Pseudomonas syringae</em> pv. <em>syringae</em>.</a> <em>J Bacteriol</em> 181:3478-3485.</li><li>Hentzer, M., Teitzel, G. M., Balzer, G. J., Heydorn, A., Molin, S., Givskov, M. and Parsek, M. R. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11514525" target="_blank">Alginate overproduction affects <em>Pseudomonas aeruginosa</em> biofilm structure and function.</a> <em>J Bacteriol</em> 183(18): 5395-5401.</li><li>King, E. O., Ward, M. K. and Raney, D. E. (1954). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13184240" target="_blank">Two simple media for the demonstration of pyocyanin and fluorescin.</a> <em>J Lab Clin Med</em> 44(2): 301-307.</li><li>Kong, W., Zhao, J., Kang, H., Zhu, M., Zhou, T., Deng, X. and Liang, H. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26206672" target="_blank">ChIP-seq reveals the global regulator AlgR mediating cyclic di-GMP synthesis in <em>Pseudomonas aeruginosa</em>.</a> <em>Nucleic Acids Res</em> 43(17): 8268-8282.</li><li>Merritt, J. H., Kadouri, D. E. and O'Toole, G. A. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18770545" target="_blank">Growing and analyzing static biofilms.</a> <em>Curr Protoc Microbiol Chapter</em> 1: Unit 1B 1.</li><li>Nivens, D. E., Ohman, D. E., Williams, J. and Franklin, M. J. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11208804" target="_blank">Role of alginate and its O acetylation in formation of <em>Pseudomonas aeruginosa</em> microcolonies and biofilms.</a> <em>J Bacteriol</em> 183(3): 1047-1057.</li><li>Osman, S. F., Fett, W. F. and Fishman, M. L. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3957873" target="_blank">Exopolysaccharides of the phytopathogen <em>Pseudomonas syringae</em> pv. <em>glycinea</em>.</a> <em>J Bacteriol</em> 166(1): 66-71.</li><li>O'Toole, G. A. and Kolter, R. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9791175" target="_blank">Flagellar and twitching motility are necessary for <em>Pseudomonas aeruginosa</em> biofilm development.</a> <em>Mol Microbiol</em> 30(2): 295-304.</li><li>Patange, A., Boehm, D., Ziuzina, D., Cullen, P. J., Gilmore, B. and Bourke, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30711711" target="_blank">High voltage atmospheric cold air plasma control of bacterial biofilms on fresh produce.</a> <em>Int J Food Microbiol</em> 293: 137-145.</li><li>Preston, L. A., Bender, C. L. and Schiller, N. L. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11913796" target="_blank">Analysis and expression of algL, which encodes alginate lyase in <em>Pseudomonas syringae</em> pv. <em>syringae</em>.</a> <em>DNA Seq</em> 12(5-6): 455-461.</li><li>Shao, X., Zhang, X., Zhang, Y., Zhu, M., Yang, P., Yuan, J., Xie, Y., Zhou, T., Wang, W., Chen, S., Liang, H. and Deng, X. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29760208" target="_blank">RpoN-dependent direct regulation of quorum sensing and the Type VI secretion system in <em>Pseudomonas aeruginosa</em> PAO1.</a> <em>J Bacteriol</em> 200(16).</li><li>Sternberg, C. and Tolker-Nielsen, T. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18770573" target="_blank">Growing and analyzing biofilms in flow cells.</a> <em>Curr Protoc Microbiol </em>Chapter 1: Unit 1B 2.</li><li>Wang, J., Shao, X., Zhang, Y., Zhu, Y., Yang, P., Yuan, J., Wang, T., Yin, C., Wang, W., Chen, S., Liang, H. and Deng, X. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29717915" target="_blank">HrpS is a global regulator on Type III Secretion System (T3SS) and Non-T3SS genes in <em>Pseudomonas syringae</em> pv. <em>phaseolicola</em>.</a> <em>Mol Plant Microbe Interact</em> 31:1232-1243.</li><li>Wang, T., Cai, Z., Shao, X., Zhang, W., Xie, Y., Zhang, Y., Hua, C., Schuster, S. C., Yang, L. and Deng, X. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30850427" target="_blank">The pleiotropic effects of c-di-GMP content in <em>Pseudomonas syringae</em>. </a><em>Appl Environ Microbiol</em> pii: AEM.00152-19.</li><li>Xiao, Y., Lan, L., Yin, C., Deng, X., Baker, D., Zhou, J. M. and Tang, X. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17378425" target="_blank">Two-component sensor <em>rhpS</em> promotes induction of <em>Pseudomonas syringae</em> type III secretion system by repressing negative regulator RhpR.</a> <em>Mol Plant Microbe Interact</em> 20(3): 223-234.</li><li>Xie, Y., Shao, X., Zhang, Y., Liu, J., Wang, T., Zhang, W., Hua, C., Deng, X. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30890603" target="_blank"><em>Pseudomonas savastanoi</em> two-component system RhpRS switches between virulence and metabolism by tuning phosphorylation state and sensing nutritional conditions.</a> <em>mBio </em>10(2). pii: e02838-18.</li><li>Zhao, J., Yu, X., Zhu, M., Kang, H., Ma, J., Wu, M., Gan, J., Deng, X. and Liang, H. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27119725" target="_blank">Structural and molecular mechanism of CdpR involved in Quorum-sensing and bacterial virulence in <em>Pseudomonas aeruginosa</em>.</a> <em>PLoS Biol</em> 14(4): e1002449.</li><li>Zhu, J., Yan, Y., Wang, Y. and Qu, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30592126" target="_blank">Competitive interaction on dual-species biofilm formation by spoilage bacteria, <em>Shewanella baltica</em> and <em>Pseudomonas fluorescens</em>.</a> <em>J Appl Microbiol</em> 126(4): 1175-1186.<br/></li></ol>