<body>
            <h1>Improved Mammalian Mitochondrial RNA Isolation</h1>
            <p><strong>Authors:</strong> Jinliang  Huang, Geng   Wang</p>
            <h2>Abstract</h2>
            <p>Mitochondria have two sets of RNAs. One is encoded in mitochondrial genome, and the other that consists of imported RNAs within mitochondria and cytosolic RNAs associated with mitochondrial outer membrane is encoded in the nucleus. These mitochondrial RNAs play important roles in mitochondrion biosynthesis and signaling in and out of mitochondria. Isolation and analysis of mitochondrial RNAs can provide useful information on understanding the mitochondrial regulation of cellular processes. However, several ribonuclease activities have been found in mitochondria, which will degrade mitochondrial RNAs during the isolation process if they are not properly inactivated. Here, we describe an improved method to inactivate the ribonuclease activities prior to RNA extraction, and thus provide a reliable protocol to isolate mammalian mitochondrial RNAs for quantitative RT-PCR and other assays.</p>
            <p><strong>Keywords:</strong> Mitochondrial RNA, RNA isolation, RNA extraction, Ribonuclease inactivation, Quantitative RT-PCR, RNA turnover, RNA trafficking, Ribosomal RNA</p>
            <h2>Background</h2><p>Mitochondrial RNAs are more diverse than originally thought. Firstly, each strand of mitochondrial DNA is transcribed as a long polycistronic transcript and processed to mature mRNAs, rRNAs and tRNAs (HÃ¤llberg and Larsson, 2014). Secondly, some nucleus-encoded noncoding RNAs are imported into mitochondria (Entelis <em>et al.</em>, 2002; Wang <em>et al.</em>, 2010; Mercer <em>et al.</em>, 2011; Zhang <em>et al.</em>, 2014; Cheng <em>et al.</em>, 2018). In addition, cytosolic ribosomes and some mRNAs are found to be associated with mitochondrial outer membrane (Kellems and Butow, 1972; Williams <em>et al.</em>, 2014). Analysis of mitochondrion-related RNAs can provide important information on understanding the mitochondrial biosynthesis and mitochondrial regulation of cellular processes, such as protein translation (Huang <em>et al.</em>, 2018) and aging (Zheng <em>et al.</em>, 2019). However, isolation of mitochondrial RNAs can be problematic since there are several strong ribonucleases within mitochondria (Rorbach <em>et al.</em>, 2011; Bruni <em>et al.</em>, 2013; Levy <em>et al.</em>, 2016; Liu <em>et al.</em>, 2017). Some of these ribonucleases remain highly active even under harsh conditions, such as proteinase-K treatment (Liu <em>et al.</em>, 2017). To avoid partial degradation of mitochondrial RNAs after lysis of mitochondria, a reliable method to inactivate these ribonucleases during RNA isolation is needed. Most current mitochondrial RNAs isolation protocols, however, do not mention the potential consequences of incomplete RNase inactivation. Chloroform (Geiger and Dalgaard, 2018) or commercial kits (Sripada <em>et al.</em>, 2012) are routinely used, but neither could completely inactivate the mitochondrial ribonucleases. In one protocol (Mager-Heckel <em>et al.</em>, 2007), sodium dodecyl sulfate (SDS) and high temperature (100 Â°C) are used, but these conditions have a negative effect on RNA stability (Sidova <em>et al.</em>, 2015). Here, we describe an improved method to inactivate the ribonuclease activities prior to RNA extraction. In this method, SDS lysis coupled with a milder temperature (70 Â°C) incubation denatures most proteins without causing RNA instability and the subsequent proteinase K digestion degrade nearly all proteins, providing a reliable protocol to isolate mammalian mitochondrial RNA for quantitative RT-PCR and other assays.</p>
        </body>

<p class="pptt" id="biaoti27828">## Materials and Reagents</p><ol class="npwtwok"><li>1.5 ml microcentrifuge tubes (Quality Scientific Plastics, catalog number: 509-GRD-Q)</li><li>0.22 Î¼m filter (Merk, Millex-GP PES, catalog number: SLGP033RB)</li><li>Nuclease-free pipette tips (Quality Scientific Plastics, catalog numbers: T104RLS-Q, T090RLS-Q and T112NXLRLS-Q)</li><li>Tris (AMRESCO, catalog number: 0497-5KG)</li><li>Sodium dodecyl sulfate (SDS) (AMRESCO, catalog number: 0227-1KG)</li><li>EDTA, disodium salt, dihydrate (Na<sub>2</sub>EDTAÂ·2H<sub>2</sub>O) (AMRESCO, catalog number: 0105-1KG)</li><li>Proteinase K (AMRESCO, catalog number: 0706-100MG)</li><li>6x DNA Loading Dye (Thermo Fisher Scientific, catalog number: R0611)</li><li>Agarose (BIOWEST, Regular Agarose G-10, catalog number: 111860)</li><li>Nuclease-free water (AMRESCO, catalog number: E476-1L)</li><li>Double distilled water (ddH<sub>2</sub>O)</li><li>DNase I, RNase-free, supplied with 10x Reaction buffer with MgCl<sub>2</sub> (Thermo Fisher Scientific, catalog number: EN0521)</li><li>TRIzol (Life Technologies, catalog number: 15596018)</li><li>1-Bromo-3-chloropropane (Sigma-Aldrich, catalog number: B9673) </li><li>Isopropanol (AMRESCO, catalog number: 0918-1L)</li><li>Ethanol (Sigma-Aldrich, catalog number: 459836-2L)</li><li>HCl (Beijing Chemical Works, Analytic Reagent grade)</li><li>NaOH (sodium hydroxide pellets) (Shanghai Sangon Biotech, catalog number: A100173)</li><li>GeneRuler 1 kb DNA Ladder (Thermo Fisher Scientific, catalog number: SM0311) </li><li>HEPES free acid (AMRESCO, catalog number: 0511-1KG)</li><li>Mannitol (AMRESCO, catalog number: 0122-500G)</li><li>Sucrose (AMRESCO, catalog number: 0335-500G)</li><li>MitoPrep buffer (see Recipes)</li><li>10% (w/v) SDS (see Recipes)</li><li>1 M Tris-HCl (pH 7.4) (see Recipes) </li><li>0.5 M EDTA (pH 8.0) (see Recipes)</li><li>Proteinase K (1 mg/ml) (see Recipes)</li><li>Lysis buffer (see Recipes)</li><li>1x DNase buffer (see Recipes)</li><li>75% ethanol (v/v) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27829">### Equipment</p><ol class="npwtwok"><li>Pipettes (RAININ, Pipet-Lite XLS)</li><li>Two heat blocks (Hangzhou Allsheng Instruments, Product Name: dry bath incubator, catalog number: MK200-2)</li><li>NanoDrop instrument (Thermo Fisher Scientific, NanoDrop 2000c Spectrophotometer)</li><li>Power supply (Tanon, catalog number: EPS 300)</li><li>Gel imaging system (Tanon, catalog number: 1600)</li><li>Centrifuges (Thermo Fisher Scientific, models: Sorvall Legend Micro 21 and Micro 21R)</li><li>pH meter (Sartorius, catalog number: PB-10) </li><li>-80 Â°C freezer (Thermo Fisher Scientific)</li><li>Magnetic stir bars and magnetic stirrer</li><li>Fume hood </li><li>Laminar flow cabinet</li><li>Incubator</li><li>4 Â°C refrigerator</li><li>-20 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti27830">### Procedure</p><p class="ppzz">The flow sheet of this protocol is shown in Figure 1.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190528/20190528185111_8202.jpg" width="300"/><br/><strong>Figure 1. The flow sheet of steps in this protocol</strong> </p><br/><ol class="npwone"><li>Sample pretreatment<br/><em>Note: To examine sub-mitochondrially localized RNA, different treatments such as nuclease treatment or mitoplasting are needed. We focus here on inactivation of mitochondrial RNases. Other methods such as proximity labeling are also needed to separate mitochondrially localized RNA from RNA contamination from other cellular compartments.</em><br/><ol class="npwtwo"><li>Isolate mitochondria from tissue culture cells as described in Huang and Wang (2019) and measure mitochondrial protein concentration with a NanoDrop instrument after diluting 1 Î¼l of mitochondrial sample with 19 Î¼l of 0.6% SDS. Transfer 200 Î¼g of purified mitochondria (in 20 Î¼l of MitoPrep buffer) to a microcentrifuge tube and keep the tube on ice.<br/><em>Note: About 9 x 10<sup>6</sup> HEK293 cells could be harvested from a 10-cm cell culture dish of 90% cell density, for a yield of around 250 Î¼g mitochondria.</em> </li><li>Mix mitochondria with 100 Î¼l of fresh lysis buffer and heat the sample at 70 Â°C in a heat block for 5 min. Take out the sample, and let it cool down to room temperature (RT).</li><li>Add 1 Î¼l of 1 mg/ml proteinase K to the lysate and incubate it at 37 Â°C in a heat block for 5 min.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA extraction<br/><em>Note: Perform this part in a fume hood.</em> <ol class="npwtwo"><li>Add 400 Î¼l of TRIzol and 200 Î¼l of 1-Bromo-3-chloropropane to the pretreated lysate, shake the tube hard for 1 min, and then spin at 14,800 <em>x g</em> for 5 min at RT.</li><li>Transfer the colorless fraction in the upper layer to a new microcentrifuge tube, mix with an equal volume of isopropanol and spin at 21,000 <em>x g</em> for 10 min at 4 Â°C.</li><li>Discard the supernatant and save the RNA pellet. Add 600 Î¼l of ice-cold 75% ethanol to the tube, gently turn it upside down for a couple of times, and spin at 21, 000 <em>x g</em> for 5 min at 4 Â°C.<br/><em>Note: If a small amount of mitochondria (200 Î¼g or less) was used, the RNA pellet might be very small and hardly visible. </em> </li><li>Discard the supernatant and make the remaining ethanol as little as possible. Be careful to avoid losing the pellet.</li><li>Dry the RNA pellet in the laminar flow cabinet.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA treatment<br/><em>Note: If mitochondrial DNA would not interfere with downstream assays, DNase I treatment is not </em><em>required and RNA pellets can be directly dissolved in nuclease-free water.</em> <ol class="npwtwo"><li>Dissolve the RNA pellet in 40 Î¼l of 1x DNase buffer with 0.5 Î¼l of DNase I, incubate it at 37 Â°C for 10 min, vortex the tube, and then spin down the sample briefly. Incubate the sample again at 37 Â°C for 10 min.</li><li>Add 0.4 Î¼l of 0.5 M EDTA to the RNA sample and heat the sample at 70 Â°C in a heat block for 10 min to inactivate the DNase I.</li></ol></li></ol><p class="pptt" id="biaoti27831">### Data analysis</p><p class="ppzz"><strong>Representative data</strong> (Figure 2)<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190528/20190528185146_9049.jpg" width="200"/><br/><strong>Figure 2. Agarose gel image of the isolated mitochondrial RNA, which was treated with or without DNase I.</strong> Three hundred nanograms of the isolated RNA were treated with (+DNase I) or without (-DNase I) DNase I. After DNase I inactivation, RNA samples were separated on a 1.5% agarose gel stained with ethidium bromide. <br/><br/><strong>Data processing</strong><br/>The gel image was captured with Tanon 1600 Gel Image System (Tanon), and cropped using Photoshop. <br/><br/><strong>Data analysis</strong><br/>No statistical analysis was performed.</p><p class="pptt" id="biaoti27832">### Recipes</p><ol class="npwtwok"><li>MitoPrep buffer<br/>0.225 M mannitol<br/>0.075 M sucrose<br/>20 mM HEPES (pH 7.4)<br/>Dissolve 2.05 g of mannitol (Mw: 182.17), 1.28 g of sucrose (Mw: 342.3), and 0.2383 g of HEPES (Mw: 238.3) in ~45 ml ddH<sub>2</sub>O, adjust the pH to 7.4 with 1 M KOH, and add ddH<sub>2</sub>O to the final volume of 50 ml<br/>Filter through a 0.22 Î¼m filter, and store at 4 Â°C</li><li>10% (w/v) SDS<br/>Dissolve 10 g of SDS in 100 ml of ddH<sub>2</sub>O and stir until SDS is completely dissolved<br/>Store at RT. Dilute 10% SDS to 0.6% SDS with ddH<sub>2</sub>O</li><li>1 M Tris-HCl (pH 7.4)<br/>Dissolve 12.114 g of Tris (Mw: 121.14) in ~80 ml of ddH<sub>2</sub>O, stir until completely dissolved<br/>Adjust the pH to 7.4 with HCl, and make up to 100 ml with ddH<sub>2</sub>O<br/>Filter through a 0.22 Î¼m filter and store at RT</li><li>0.5 M EDTA (pH 8.0)<br/>Add 9.306 g of EDTA (Mw: 372.24) to ~40 ml of ddH<sub>2</sub>O, stir and slowly adjust the pH to 8.0 with NaOH, and make up to 50 ml with ddH<sub>2</sub>O<br/>Filter through a 0.22 Î¼m filter and store at 4 Â°C<br/><em>Note: EDTA will not fully dissolve until the pH is close to 8.0.</em> </li><li>Proteinase K (1 mg/ml)<br/>Dissolve 1 mg of proteinase K in 1 ml of nuclease-free water, aliquot and store at -20 Â°C</li><li>Lysis buffer<br/>The lysis buffer consists of 1% SDS, 10 mM EDTA and 10 mM Tris-HCl (pH 7.4)<br/><strong>For a 10 ml solution:</strong><br/>Mix 8.7 ml of nuclease-free water, 1 ml of 10% SDS, 0.2 ml of 0.5 M EDTA (pH 8.0), and 0.1 ml of 1 M Tris-HCl (pH 7.4) and make up to 10 ml with nuclease-free water</li><li>1x DNase buffer<br/>Dilute 10x DNase buffer (with MgCl<sub>2</sub>) to 1x with nuclease-free water</li><li>75% ethanol (v/v)<br/>For a 100 ml solution, add 25 ml of nuclease-free water to 75 ml of ethanol<br/>Mix thoroughly and store at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti27833">### Acknowledgments</p><p class="ppzz">This work was supported by Grant 2017YFA0504600 from the Priority Research Program of the Ministry of Science and Technology of China, by Grants 31371439 and 91649103 from the National Natural Science Foundation of the Peopleâs Republic of China, and by funds from the Ministry of Education of the Peopleâs Republic of China 1000 Talents Youth Program. This protocol was modified from Huang <em>et al.</em> (2018) and Liu <em>et al.</em> (2017).</p><p class="pptt" id="biaoti27834">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest with the contents of this article.</p><p class="pptt" id="biaoti27835">### References</p><ol class="npwtwok"><li>Bruni, F., Gramegna, P., Oliveira, J. M., Lightowlers, R. N. and Chrzanowska-Lightowlers, Z. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23741365" target="_blank">REXO2 is an oligoribonuclease active in human mitochondria.</a> <em>PLoS One</em> 8(5): e64670.</li><li>Cheng, Y., Liu, P., Zheng, Q., Gao, G., Yuan, J., Wang, P., Huang, J., Xie, L., Lu, X., Tong, T., Chen, J., Lu, Z., Guan, J. and Wang, G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30184494" target="_blank">Mitochondrial trafficking and processing of telomerase RNA TERC.</a> <em>Cell Rep</em> 24(10): 2589-2595.</li><li>Entelis, N., Kolesnikova, O., Kazakova, H., Brandina, I., Kamenski, P., Martin, R. P. and Tarassov, I. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12416306" target="_blank">Import of nuclear encoded RNAs into yeast and human mitochondria: experimental approaches and possible biomedical applications.</a> <em>Genet Eng</em> <em>(N Y)</em> 24: 191-213.</li><li>Geiger, J. and Dalgaard, L. T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29851010" target="_blank">Isolation and analysis of mitochondrial small RNAs from rat liver tissue and HepG2 cells. </a><em>Methods Mol Biol</em> 1782: 337-350.</li><li>HÃ¤llberg, B. M. and Larsson, N. G. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25088301" target="_blank">Making proteins in the powerhouse.</a> <em>Cell Metab</em> 20(2): 226-240.</li><li>Huang, J., Liu, P. and Wang, G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30385512" target="_blank">Regulation of mitochondrion-associated cytosolic ribosomes by mammalian mitochondrial ribonuclease T2 (RNASET2).</a> <em>J Biol Chem</em> 293(51): 19633-19644.</li><li>Huang, J. and Wang, G. (2019). <a href="http://www.bio-protocol.org/e3255" target="_blank">Organelle-associated rRNA Degradation.</a> <em>Bio-protocol</em> 9(11): e3255.</li><li>Kellems, R. E. and Butow, R. A. (1972). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4629740" target="_blank">Cytoplasmic-type 80 S ribosomes associated with yeast mitochondria. I. Evidence for ribosome binding sites on yeast mitochondria.</a> <em>J Biol Chem</em> 247(24): 8043-8050.</li><li>Levy, S., Allerston, C. K., Liveanu, V., Habib, M. R., Gileadi, O. and Schuster, G. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26826708" target="_blank">Identification of LACTB2, a metallo-Î²-lactamase protein, as a human mitochondrial endoribonuclease.</a> <em>Nucleic Acids Res</em> 44(4): 1813-1832.</li><li>Liu, P., Huang, J., Zheng, Q., Xie, L., Lu, X., Jin, J. and Wang, G. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28730546" target="_blank">Mammalian mitochondrial RNAs are degraded in the mitochondrial intermembrane space by RNASET2.</a> <em>Protein Cell</em> 8(10): 735-749.</li><li>Mager-Heckel, A. M., Entelis, N., Brandina, I., Kamenski, P., Krasheninnikov, I. A., Martin, R. P. and Tarassov, I. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18314730" target="_blank">The analysis of tRNA import into mammalian mitochondria.</a> <em>Methods Mol Biol</em> 372: 235-253.</li><li>Mercer, T. R., Neph, S., Dinger, M. E., Crawford, J., Smith, M. A., Shearwood, A. M., Haugen, E., Bracken, C. P., Rackham, O., Stamatoyannopoulos, J. A., Filipovska, A. and Mattick, J. S. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21854988" target="_blank">The human mitochondrial transcriptome.</a> <em>Cell</em> 146(4): 645-658.</li><li>Rorbach, J., Nicholls, T. J. and Minczuk, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21666256" target="_blank">PDE12 removes mitochondrial RNA poly(A) tails and controls translation in human mitochondria.</a> <em>Nucleic Acids Res</em> 39(17): 7750-7763.</li><li>Sidova, M., Tomankova, S., Abaffy, P., Kubista, M. and Sindelka, R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27077037" target="_blank">Effects of post-mortem and physical degradation on RNA integrity and quality.</a> <em>Biomol Detect Quantif</em> 5: 3-9.</li><li>Sripada, L., Tomar, D., Prajapati, P., Singh, R., Singh, A. K. and Singh, R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22984580" target="_blank">Systematic analysis of small RNAs associated with human mitochondria by deep sequencing: detailed analysis of mitochondrial associated miRNA.</a> <em>PLoS One</em> 7(9): e44873.</li><li>Wang, G., Chen, H. W., Oktay, Y., Zhang, J., Allen, E. L., Smith, G. M., Fan, K. C., Hong, J. S., French, S. W., McCaffery, J. M., Lightowlers, R. N., Morse, H. C., 3rd, Koehler, C. M. and Teitell, M. A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20691904" target="_blank">PNPASE regulates RNA import into mitochondria.</a> <em>Cell</em> 142(3): 456-467.</li><li>Williams, C. C., Jan, C. H. and Weissman, J. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25378625" target="_blank">Targeting and plasticity of mitochondrial proteins revealed by proximity-specific ribosome profiling.</a> <em>Science</em> 346(6210): 748-751.</li><li>Zhang, X., Zuo, X., Yang, B., Li, Z., Xue, Y., Zhou, Y., Huang, J., Zhao, X., Zhou, J., Yan, Y., Zhang, H., Guo, P., Sun, H., Guo, L., Zhang, Y. and Fu, X. D. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25083871" target="_blank">MicroRNA directly enhances mitochondrial translation during muscle differentiation.</a> <em>Cell</em> 158(3): 607-619.</li><li>Zheng, Q., Liu, P., Gao, G., Yuan, J., Wang, P., Huang, J., Xie, L., Lu, X., Di, F., Tong, T., Chen, J., Lu, Z., Guan, J. and Wang, G. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30788732" target="_blank">Mitochondrion-processed TERC regulates senescence without affecting telomerase activities.</a> <em>Protein Cell</em> doi: 10.1007/s13238-019-0612-5.<br/></li></ol>