<body>
            <h1>Minigene Assay to Evaluate CRISPR/Cas9-based Excision of Intronic  Mutations that Cause Aberrant Splicing in Human Cells</h1>
            <p><strong>Authors:</strong> David J. Sanz, Patrick T. Harrison</p>
            <h2>Abstract</h2>
            <p>The construction of Hybrid minigenes provides a robust and simple strategy to study the effects of disease-causing mutations on mRNA splicing when biological material from patient cells is not available. Hybrid minigenes can be used as splicing reporter plasmids allow RNA expression and heterologous splicing reactions between synthetic splicing signals in the vector and endogenous splicing signals in a cloned genomic DNA fragment that contains one or more introns and exons. Minigene-based assay has been used extensively to test the effect of mutations in the splicing of a target sequence. They can also be used to test the ability of CRISPR/Cas9 and one or more associated gRNAs to target specific sequences in the minigene, and determine the effect of these editing events on splicing. As an example, it is shown that CRISPR/Cas9-based, targeted excision of short intronic sequences containing mutations which create cryptic splice signals, can restore normal splicing in a <em>CFTR</em> Hybrid minigene.</p>
            <p><strong>Keywords:</strong> Cas9/CRISPR, Splicing mutations, Mini-gene, Gene editing, Cystic fibrosis</p>
            <h2>Background</h2><p>Our objective was to use CRISPR/Cas9 to correct the effect on splicing of three different intronic mutations in the <em>CFTR</em> gene which cause cystic fibrosis by creating strong alternative splice sites: c.1679+1634a&gt;g[1811+1.6Kba&gt;g], c.3718-2477c&gt;t[3849+10KBc&gt;t] and c.3140-26a&gt;g[3272-26a&gt;g] (Highsmith <em>et al.</em>, 1994; ChillÃ³n <em>et al.</em>, 1995; Beck <em>et al.</em>, 1999) and lead to the disruption of normal <em>CFTR</em> mRNA production.<br/><br/>A CRISPR/Cas9-Non-homologous end joining (NHEJ) approach using two gRNAs has been shown to successfully excise small intronic regions of <em>CFTR</em> gene in &gt; 25% of transfected cells (Hollywood <em>et al.</em>, 2016). We hypothesized that this approach could be used to remove the strong alternative splice sites described above with high efficiency in cells with these mutations and restore normal splicing of the <em>CFTR</em> mRNA. The efficiency of the NHEJ targeting approach is at least 10-fold higher than previously reported homology direct repair strategies to correct Cystic fibrosis causing mutations in the <em>CFTR</em> gene.<br/><br/>However, the lack of biological samples carrying these mutations of interest prompted us to ask if it is possible to construct hybrid minigene constructs that contain these mutations as a system to test our reagents, and determine if targeted excision could efficiently restore normal splicing. Once PCR was performed to amplify the <em>wt</em> sequences of interest from genomic DNA, these amplicons were cloned in the exon-trapping pSPL3 plasmid and directed mutagenesis was performed to obtain pSPL3 based minigenes containing the mutations of interest. The pSPL3 plasmid contains a small artificial gene composed of an SV40 promotor and exon-intron-exon sequence with splice donor and acceptor sites and a polyadenylation sequence. A multiple cloning site present in the intron allows cloning sequences of interest, and testing of putative heterologous splicing reactions between the plasmid splice sites and the splice sites contained in the cloned sequences (Sanz <em>et al.</em>, 2010).<br/><br/>HEK293T cells were transfected with both <em>wt</em> and mutation containing versions of the three minigenes to characterize by RT-PCR the aberrant splicing caused by these three mutations in the context of the minigenes.<br/><br/>HEK293T were also co-transfected with both the mutant minigene and a plasmid expressing both Cas9 and designed gRNAs. After DNA extraction, PCR across the target sequence was performed to determine the percentage excision caused in mutant minigenes by the CRISPR/Cas9 reagents.<br/><br/>RNA extraction from the same samples allows characterizing by RT-PCR the restoration of splicing due to the excision of the target sequences containing the splicing mutations.<br/><br/>In summary, construction of minigenes allows the analysis of how the sequences of interest are edited using designed CRISPR/Cas9 reagents with specific gRNAs, and how this editing changes the splicing process in the minigene, circumventing the need for primary cells or cell lines with the mutation of interest.<br/></p>
        </body>

<p class="pptt" id="biaoti27871">## Materials and Reagents</p><ol class="npwone"><li>PCR<ol class="npwtwo"><li>Pipette tips</li><li>0.2 ml PCR Tubes (SARSTEDT, catalog number: 72.737.002)</li><li>Total Human genomic DNA from Lung (Amsbio, catalog number: HG-601)</li><li>Phusion High-Fidelity DNA Polymerase (Thermo Fisher, catalog number: F530S)</li><li>dNTPs Mix (Thermo Fisher, catalog number: R0192)<br/><br/></li></ol></li><li>Gel electrophoresis<ol class="npwtwo"><li>TAE buffer (40 mM Tris base, 20 mM Acetic Acid, 1 mM EDTA)</li><li>Agarose (Sigma-Aldrich, catalog number: A9539)</li><li>Ethidium bromide (Sigma-Aldrich, catalog number: E7637)</li><li>1 kb DNA ladder (Thermo Fisher, catalog number: SM0311)<br/></li><li>100 bp DNA ladder (Thermo Fisher, catalog number: 15628019)</li><li>DNA Gel Loading Dye (Thermo Fisher, catalog number: R0611)<br/><br/></li></ol></li><li>Cloning<ol class="npwtwo"><li>1.5 ml Microtubes (SARSTEDT, catalog number: 72.706.700)</li><li>Petri dishes (SARSTEDT, catalog number: 82.1472.001)</li><li>15 ml polypropylene tubes (SARSTEDT, catalog number: 62.554.002)</li><li>NEB<sup>Â®</sup> 5Î± competent <em>E. coli</em> (NEB, catalog number: C2987I)</li><li>pSPL3 vector (Thermo Fisher, discontinued)</li><li>pSpCas9(BB)-2A-GFP PX458 (Addgene #48138)</li><li>NheI (Thermo Fisher, catalog number: ER0971)</li><li>XhoI (Thermo Fisher, catalog number: ER0691)</li><li>XbaI (Thermo Fisher, catalog number: ER0681)</li><li>BbsI (Thermo Fisher, catalog number: ER1011)</li><li>T4 DNA ligase (NEB, catalog number: M0202S)</li><li>High Pure PCR Product Purification kit (Sigma-Aldrich, catalog number: 11732668001)</li><li>StrataPrep DNA Gel Extraction kit (Agilent, catalog number: 400766)</li><li>QuikChange II Kit (Agilent, catalog number:200523)</li><li>LB Broth Lennox (Sigma-Aldrich, catalog number: L3022)</li><li>LB Broth with Agar Lennox (Sigma-Aldrich, catalog number: L2897)</li><li>Ampicillin Sodium salt (Sigma-Aldrich, catalog number: A0166)<br/><br/></li></ol></li><li>Mammalian cell culture<ol class="npwtwo"><li>T75 cell culture flasks (SARSTEDT, catalog number: 83.1813.002)</li><li>12-well plates (SARSTEDT, catalog number: 83.3921.005)</li><li>HEK293T cells, human embryonic kidney cell line (ATCC<sup>Â®</sup>, catalog number: CRL-3216)</li><li>CFTE cells, cystic fibrosis tracheal epithelial cell line (Gruenert <em>et al.</em>, 2004)<br/></li><li>Dubelccoâs modified Eagleâs Medium (DMEM, Sigma-Aldrich, catalog number: D5796)</li><li>Minimum Essential Medium Eagle (MEM, Sigma-Aldrich, catalog number: M2279)</li><li>Fetal Bovine Serum (Sigma-Aldrich, catalog number: F7524)</li><li>Phosphate-buffered saline PBS (Sigma-Aldrich, catalog number: P5493)</li><li>Penicillin-Streptomycin (Sigma-Aldrich, catalog number: P433)</li><li>Trypsin-EDTA (Sigma-Aldrich, catalog number: T3924)</li><li>Lipofectamine 2000 (Thermo Fisher, catalog number: 11668027)</li><li>Opti-MEM (Thermo Fisher, catalog number: 31985070)<br/><br/></li></ol></li><li>Plasmidic DNA and Total DNA extraction<ol class="npwtwo"><li>NucleoSpin Plasmid kit (MACHEREY-NAGEL, catalog number: 740588150)</li><li>DNeasy Blood and Tissue Kit (QIAGEN, catalog number: 69506)<br/><br/></li></ol></li><li>RNA extraction and RT-PCR<ol class="npwtwo"><li>0.5 ml PCR Tubes (SARSTEDT, catalog number: 70.704.400)</li><li>NucleoSpin RNA (MACHEREY-NAGEL, catalog number: 740955.150)</li><li>High-Capacity cDNA Reverse Transcription Kit (Thermo Fisher, catalog number: 15561175)</li><li>dNTPs Mix (Thermo Fisher, catalog number: R0192)</li><li>Phusion High-Fidelity DNA polymerase (Thermo Fisher, catalog number: F530S)</li></ol></li></ol><p class="pptt" id="biaoti27872">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>PCR Thermocycler Eppendorf Mastercycler</li><li>Gel Doc System (Bio-Rad)</li><li>Water bath</li><li>Bacterial Incubator and shaker</li><li>Mammalian cell incubator (37 Â°C, 5% CO<sub>2</sub>)</li><li>Bunsen Burner</li><li>NanoDrop 2000 spectrophotometer (Thermo Fisher Scientific, model: NanoDrop<sup>TM</sup> 2000)</li><li>Microcentrifuge</li><li>Electrophoresis tank and power pack</li><li>Autoclave</li><li>Biological safety cabinet<br/></li></ol><p class="pptt" id="biaoti27873">### Software</p><ol class="npwtwok"><li>Primer-blast (<a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/" target="_blank">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>)</li><li>CRISPR-Design (<a href="http://crispr.mit.edu/" target="_blank">http://crispr.mit.edu/</a>)</li><li>SnapGene Viewer (<a href="http://www.snapgene.com/products/snapgene_viewer/" target="_blank">http://www.snapgene.com/products/snapgene_viewer/</a>)</li><li>Tm Calculator (<a href="https://www.thermofisher.com/" target="_blank">https://www.thermofisher.com</a>)</li><li>Peak Scanner 2 (<a href="https://www.thermofisher.com/" target="_blank">https://www.thermofisher.com</a>)</li><li>Eurofins (Oligo synthesis, Sanger sequence and Fragment Length Analysis [FLA] service) (<a href="https://www.eurofinsgenomics.eu/" target="_blank">https://www.eurofinsgenomics.eu</a>)<br/></li></ol><p class="pptt" id="biaoti27874">### Procedure</p><ol class="npwone"><li>Minigene construction<ol class="npwtwo"><li>To construct <em>wt</em> minigenes, primers were designed (using Primer-Blast web, <a href="https://www.ncbi.nlm.nih.gov/tools/primer-blast/" target="_blank">https://www.ncbi.nlm.nih.gov/tools/primer-blast/</a>) to amplify the regions of interest from a <em>wt</em> DNA sequence. In our case, three intronic regions in the <em>CFTR</em> gene: First, a region of 1,522 bp between intron 12 and intron 13, which includes the nucleotide c.1679+1634. Second, a region of 2,012 bp between intron 18 and intron 20 which includes the nucleotide c.3140-26, and Third a region of 3224 between intron 22 and intron 23 which includes the nucleotide c.3718-2477. Restriction sites were included at the 5â² end of Fw and Rv primers to facilitate cloning into pSPL3 plasmid. We added XhoI sites to the three Fw primers and NheI sites to the first two Rv primers; XbaI was used in the Third one as an NheI site was already present in the genomic sequence. Add six additional nt upstream of the restriction sites to facilitate digestion by restriction enzymes.<br/><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td><strong>Minigenes cloning primers</strong><strong></strong><br/></td><td><br/></td></tr><tr><td>MGin12-in13fw<br/></td><td>CACACACTCGAGTGTGTTGTCCAGTTTTGGATGA<br/></td></tr><tr><td>MGin12-in13rv<br/></td><td>CTCACAGCTAGCACTGGTTTAGCATGAGGCGG<br/></td></tr><tr><td>MGin18-in20fw<br/></td><td>CTCACTCTCGAGTGACTAGGAATAGAATGGGGAGAG<br/></td></tr><tr><td>MGin18-in20rv<br/></td><td>CACACAGCTAGCACAATGGAAATTCAAAGAAATCACT<br/></td></tr><tr><td>MGin22-in23fw<br/></td><td>CACACACTCGAGACAGTACTGGATAGTCCTCTGA<br/></td></tr><tr><td>MGin22-in23rv<br/></td><td>CACACATCTAGAGCCTATGAGAAAACTGCACTGG<br/></td></tr></tbody></table><br/></li><li>Amplify Target regions of interest from a <em>wt</em> genomic DNA sample (Genomic DNA-Human adult normal tissue Lung, from a single donor, Amsbio, Abingdon, UK) using Phusion high-fidelity DNA polymerase (Thermo Fisher) and designed primers using the following program in a Mastercycler Thermocycler (Eppendorf).<br/><br/><strong>PCR Amplification of genomic DNA to construct hybrid minigenes</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>DNA template (20 ng/Î¼l)<br/></td><td>3 Î¼l<br/></td></tr><tr><td>5x Phusion Buffer<br/></td><td>4 Î¼l<br/></td></tr><tr><td>10 mM dNTPS <br/></td><td>0.4 Î¼l<br/></td></tr><tr><td>10 Î¼M FW primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>10 Î¼M RV primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Phusion polymerase<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Nuclease-free water<br/></td><td>Up to 20 Î¼l<br/></td></tr></tbody></table><br/><strong>PCR program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>35 cycles of<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>57 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>Final extension<br/></td><td>72 Â°C<br/></td><td>10 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Check successful amplification of the desired size by 1.5% agarose gel electrophoresis in TAE buffer, running 100 bp and 1 kb ladders as size standards and purify the PCR product using QIAquick PCR Purification Kit (QIAGEN).</li><li>Digest 1 Î¼g of each Purified product with selected endonucleases and purify by Gel extraction Bands of correct size after electrophoresis in a 1% agarose gel in TAE buffer using StrataPrep DNA Gel Extraction Kit (Agilent).</li><li>Digest 2 Î¼g of Plasmid pSPL3 with XhoI and NheI enzymes and excise and purify pSPL3 plasmid backbone (The bigger band of around 5 kb) from the agarose after electrophoresis (1% agarose in TAE buffer), using the StrataPrep DNA Gel Extraction kit (Agilent).</li><li>Prepare Ligation using 50 ng of each of the three digested amplified PCR products with 150 ng of purified XhoI/Nhei digested pSPL3 backbone using 10 units of T4 ligase (NEB) and incubate for 4 h at 16 Â°C.</li><li>Use 2 Î¼l of each of the three ligation reactions to transform 1 vial of 50 Î¼l of DH5Î± competent <em>E. coli</em> (NEB). Seed transformed <em>E. coli</em> in LB-agar Petri dishes containing 100 Î¼g/ml of ampicillin as selection marker for transformed colonies with pSPL3-derived plasmids. Incubate for 16 h at 37 Â°C</li><li>Pick and grow 5 colonies from each of the cloned amplicons in 5 ml of LB with ampicillin (100 Î¼g/ml) in 15 ml Polypropylene tubes and after 24 h purify plasmidic DNA from cultures using NucleoSpin Plasmid Kit (MACHEREY-NAGEL).</li><li>Sequence plasmid DNA (TubeSeq service, Eurofins) using selected primers SEQpspfw and SEQpsprv and the following PCR setup and program to confirm successful cloning of each <em>wt</em> amplified sequence into pSPL3 plasmid in at least one of the purified plasmids. We will refer to each of these plasmids as <em>wt</em> minigenes, pMG-in12-wt, pMG-in19-wt and pMG-in22-wt, as they contain sequences of the introns 12, 19 and 22 respectively.<br/><br/><strong>Sequencing pSPL3 primers</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>SEQpspfw<br/></td><td>TCACAGTCTATTATGGGGTACGG<br/></td></tr><tr><td>SEQpsprv<br/></td><td>AATTTCTGGGTCCCCTCCTGA<br/></td></tr></tbody></table><br/><strong>PCR reaction setup</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Extracted DNA (20 ng/Î¼l)<br/></td><td>3 Î¼l<br/></td></tr><tr><td>5x Phusion Buffer<br/></td><td>4 Î¼l<br/></td></tr><tr><td>10 mM dNTPs<br/></td><td>0.4 Î¼l<br/></td></tr><tr><td>10 Î¼M FW primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>10 Î¼M RV primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Phusion polymerase<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Nuclease-free water<br/></td><td>Up to 20 Î¼l<br/></td></tr></tbody></table><br/><strong>PCR program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>35 cycles of<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>57 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>Final extension<br/></td><td>72 Â°C<br/></td><td>5 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Perform site-directed mutagenesis to create mini-gene plasmids containing the desired mutations, site-directed mutagenesis is achieved by using the QuikChange II Kit (Agilent) and selected PCR primers pairs containing the desired mutations.<br/><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td><strong>Directed Mutagenesis</strong><strong></strong><br/></td><td><br/></td></tr><tr><td>MD1811+1.6kbA&gt;Gfw<br/></td><td>CCTATGTACTTGAGATGTAAGTAAGGTTACTATC<br/></td></tr><tr><td>MD1811+1.6kbA&gt;Grv<br/></td><td>GATAGTAACCTTACTTACATCTCAAGTACATAGG<br/></td></tr><tr><td>MD3272-26A&gt;Gfw<br/></td><td>GTGTTTATGTTATTTGCAGTGTTTTCTATGG<br/></td></tr><tr><td>MD3272-26A&gt;Grv<br/></td><td>CCATAGAAAACACTGCAAATAACATAAACAC<br/></td></tr><tr><td>MD3849+10kbC&gt;Tfw<br/></td><td>CTGTTGCAGTATTAAAATGGtGAGTAAGACACCCTGAAAG<br/></td></tr><tr><td>MD3849+10kbC&gt;Trv<br/></td><td>CTTTCAGGGTGTCTTACTCaCCATTTTAATACTGCAACAG<br/></td></tr></tbody></table><br/><strong>Site-Directed mutagenesis reaction setup</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>5x Reaction Buffer<br/></td><td>5 Î¼l<br/></td></tr><tr><td>Pfu Turbo DNA polymerase<br/></td><td>1 Î¼l<br/></td></tr><tr><td>10 Î¼M MD fw Primer<br/></td><td>1.25 Î¼l<br/></td></tr><tr><td>10 Î¼M MD rv Primer<br/></td><td>1.25 Î¼l<br/></td></tr><tr><td>Plasmid DNA (25 ng/Î¼l) <br/></td><td>1 Î¼l<br/></td></tr><tr><td>Nuclease-free water<br/></td><td>up to 50 Î¼l<br/></td></tr></tbody></table><br/><strong>Directed Mutagenesis Program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation<br/></td><td>95 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>16 cycles of<br/></td><td>95 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>56 Â°C<br/></td><td>1 min<br/></td></tr><tr><td><br/></td><td>68 Â°C<br/></td><td>6 min<br/></td></tr><tr><td>Final Extension<br/></td><td>68 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/><strong>Digestion of plasmid template</strong><br/>Add 1 Î¼l of DpnI enzyme directly to each PCR reaction; digest at 37 Â°C for 1 h.<br/><br/></li><li>Use 2 Î¼l of each site-directed mutagenesis reaction, that now contains mini-gene plasmids with the desired mutations, to transform 1 vial of 50 Î¼l of 5-alpha competent <em>E. coli</em> (NEB), and seed the transformed <em>E. coli</em> in LB-agar Petri dishes containing 100 Î¼g/ml of ampicillin as a selection marker for transformed colonies. Incubate for 16 h at 37 Â°C.</li><li>Select 5 colonies coming from each of the Directed mutagenesis reactions, and grow them in 5 ml of LB with ampicillin (100 Âµg/ml) for 24 h in 15 ml polypropylene tubes, then extract plasmid DNA using NucleoSpin Plasmid Kit (MACHEREY-NAGEL).</li><li>Sequence plasmidic DNA samples (TubeSeq service, Eurofins) using selected Screening primers to confirm desired mutated sequence on each target plasmid in at least one of the colonies.<br/>Screening Primers<br/><br/><strong>Screening Primers</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>in12Ufw<br/></td><td>AAGAACAGAGTGTGGGGAAGA<br/></td></tr><tr><td>in12Drv<br/></td><td>GCATTTTGTTCCTTGTAGTTTCAG<br/></td></tr><tr><td>in19Ufw<br/></td><td>ACTCCCAGTGGTAGCCAAGA<br/></td></tr><tr><td>in19Drv<br/></td><td>GCACGAAGTGTCCATAGTCCT<br/></td></tr><tr><td>in22Ufw<br/></td><td>TCATTCAGTGGGTATAAGCAGCA<br/></td></tr><tr><td>in22Drv<br/></td><td>GACATACCCTAAATCTAAGTCAGTG<br/></td></tr></tbody></table><br/></li></ol></li><li>Cloning of gRNA sequences to excise intronic sequences in <em>CFTR</em> <ol class="npwtwo"><li>Use the CRISPR design website tool (<a href="http://cripsr.mit.edu/" target="_blank">http://cripsr.mit.edu</a>) to identify and select gRNAs sequences in the intronic regions of gene of interest upstream and downstream to the position where the variants of interest may occur. We selected three different guide RNA target sequences upstream (U1-U3) and three different guide RNA sequences downstream (D1-D3) to each of the three intronic regions (in12, in19 and in22) we wanted to excise. We have chosen guide RNAs pairs which produce targeted excisions of a size that can be easily detected on a standard agarose gel. However, if a more sensitive detection method is used, such as Fragment Length Analysis (FLA), it is possible to choose guide RNAs that produce excisions of essentially any size. It should be noted that guide RNAs close to the canonical splice sites or the branch point should not be used. The CRISPR design website tool offers a score for each possible guide RNA in a particular region based on putative off-target effects that may assist in the selection of the best guide RNAs to use.</li><li>The CRISPR design website tool will also suggest the sequence of the sense and antisense oligos (Eurofins) to generate a dsDNA fragment encoding the desired gRNA sequence and overhangs to clone using Golden Gate strategy into the BbsI overhangs of the plasmid pSpCas9-(BB)-2A-GFP (Addgene.org PX458). Golden Gate cloning relies on the use of Type IIS restriction enzymes, which cleave outside of their recognition sequence, creating four base flanking overhangs. Oligos are designed to form compatible overhangs when annealed, so they can be cloned in the correct orientation into the plasmid when it is digested by BbsI enzyme. As successful ligation destroys the BbsI recognition site every cycle of BbsI digestion followed by T4 ligation will enrich the successful cloning.</li><li>Anneal sense and antisense oligo (1 Î¼g/Î¼l) by mixing 5 Î¼l of each with 85 Î¼l of nuclease-free water and 5 Î¼l of T4 ligase buffer (NEB) in 1.5 ml tubes, and heating the tubes in a block at 95 Â°C for 5 min. Turn off the heat block afterward to let the tubes cool down slowly.<br/><br/><strong>Oligos to express gRNAS to excise intron 12 region</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:900px;"><tbody><tr><td>gin12U1fw<br/></td><td>CACCGGAAACTGTGTACATTTTGAT<br/></td><td>gin12U1rv<br/></td><td>AAACATCAAAATGTACACAGTTTCC<br/></td></tr><tr><td>gin12U2fw<br/></td><td>CACCGAGTATGCAAGAGCTACATAA<br/></td><td>gin12U2rv<br/></td><td>AAACTTATGTAGCTCTTGCATACTC<br/></td></tr><tr><td>gin12U3fw<br/></td><td>CACCGGACTTTTAAAGTTTTGCCAT<br/></td><td>gin12U3rv<br/></td><td>AAACATGGCAAAACTTTAAAAGTCC<br/></td></tr><tr><td>gin12D1fw<br/></td><td>CACCGATGTACTTGAGATGTAAGTA<br/></td><td>gin12D1rv<br/></td><td>AAACTACTTACATCTCAAGTACATC<br/></td></tr><tr><td>gin12D2fw<br/></td><td>CACCGTTACTCATACTTTCCTTATT<br/></td><td>gin12D2rv<br/></td><td>AAACAATAAGGAAAGTATGAGTAAC<br/></td></tr><tr><td>gin12D3fw<br/></td><td>CACCGTTATCTCATTTCTATTAATA<br/></td><td>gin12D3rv<br/></td><td>AAACTATTAATAGAAATGAGATAAC<br/></td></tr></tbody></table><br/><strong>Oligos to express gRNAS to excise intron 19 region</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:900px;"><tbody><tr><td>gin19U1fw<br/></td><td>CACCGTTTACTTGGCTACCAGAGAT<br/></td><td>gin19U1rv<br/></td><td>AAACATCTCTGGTAGCCAAGTAAAC<br/></td></tr><tr><td>gin19U2fw<br/></td><td>CACCGAGTTACCCTCTTTTTTTACT<br/></td><td>gin19U2rv<br/></td><td>AAACAGTAAAAAAAGAGGGTAACTC<br/></td></tr><tr><td>gin19D1fw<br/></td><td>CACCGGTTATTTGCAGTGTTTTCTA<br/></td><td>gin19D1rv<br/></td><td>AAACTAGAAAACACTGCAAATAACC<br/></td></tr><tr><td>gin19D2fw<br/></td><td>CACCGTTTCTATGGAAATATTTCAC<br/></td><td>gin19D2rv<br/></td><td>AAACGTGAAATATTTCCATAGAAAC<br/></td></tr></tbody></table><br/><strong>Oligos to express gRNAS to excise intron 22 region<br/></strong> <table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:900px;"><tbody><tr><td>gin22U1fw<br/></td><td>CACCGCATTTTAATACTGCAACAGA<br/></td><td>gin22U1rv<br/></td><td>AAACTCTGTTGCAGTATTAAAATGC<br/></td></tr><tr><td>gin22U2fw<br/></td><td>CACCGCATCTGTTGCAGTATTAAAA<br/></td><td>gin22U2rv<br/></td><td>AAACTTTTAATACTGCAACAGATGC<br/></td></tr><tr><td>gin22U3fw<br/></td><td>CACCGCTTGATTTCTGGAGACCACA<br/></td><td>gin22U3rv<br/></td><td>AAACTGTGGTCTCCAGAAATCAAGC<br/></td></tr><tr><td>gin22D1fw<br/></td><td>CACCGTTGATCCAACATTCTCAGGG<br/></td><td>gin22D1rv<br/></td><td>AAACCCCTGAGAATGTTGGATCAAC<br/></td></tr><tr><td>gin22D2fw<br/></td><td>CACCGATCCAACATTCTCAGGGAGG<br/></td><td>gin22D2rv<br/></td><td>AAACCCTCCCTGAGAATGTTGGATC<br/></td></tr></tbody></table><br/></li><li>Mix 50 ng of each previously annealed gRNA-encoding oligo pair with 150 ng of plasmid pSpCas9-(BB)-2A-GFP, 2 Î¼l of 10x T4 DNA ligase buffer, 1 Âµl of T4 DNA ligase and 1 Î¼l of BbsI enzyme and water in a total volume of 20 Î¼l. To clone the annealed double strand fragment into the pSpCas9-(BB)-2A-GFP plasmid use the following thermocycler program for golden gate cloning:<br/><br/><strong>Golden gate cloning of Annealed oligos into pSpCas9-(BB)-2A-GFP plasmid</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>10 cycles of<br/></td><td>37 Â°C<br/></td><td>5 min<br/></td></tr><tr><td><br/></td><td>16 Â°C<br/></td><td>10 min<br/></td></tr><tr><td><br/></td><td>37 Â°C<br/></td><td>20 min <br/></td></tr><tr><td><br/></td><td>80 Â°C <br/></td><td>20 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Transform 50 Î¼l of competent DH5Î± <em>E. coli</em> (NEB) using 2 Î¼l of the previous reaction and plate them in LB agar Petri dishes with 100 Î¼g/ml ampicillin. Incubate for 16 h at 37 Â°C.</li><li>Pick and grow 5 colonies in 5 ml of selective LB medium containing ampicillin in 15 ml Polypropylene tubes and extract plasmid DNA after 24 h from the cultures.</li><li>Sequence plasmids (Tubeseq, Eurofins) using pcas9SEQfw and pcas9SEQrv primers to check for successful cloning of annealed oligos in at least one of the selected colonies.<br/><br/><strong>pcas9 sequence primers</strong><br/><div style="white-space:nowrap;"><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>pcas9SEQfw<br/></td><td>GAGGGCCTATTTCCCATGATTCC<br/></td></tr><tr><td>pcas9SEQrv<br/></td><td>GTCTGCAGAATTGGCGCAC<br/></td></tr></tbody></table><br/></div></li></ol></li><li>Cotransfection of CFTE cells with pairs of constructed gRNAs/Cas9 expressing plasmids<ol class="npwtwo"><li>Prior to transfection, CFTE cells were kept in culture and passaged twice a week with Trypsin-EDTA in a T75 cell culture flask with MEM + 10% FBS + 1% Penicillin/Streptomycin at 37 Â°C, 5% CO<sub>2</sub>.</li><li>Seed 100,000 CFTE cells in 12-well plates in free antibiotic MEM + 10% FBS medium 24 h prior to transfection. For other cell types seed enough cells to have around 90% confluency after 24 h. </li><li>Transfect cells with paired Cas9/gRNA expressing plasmids constructed in Procedure B using one plasmid that express a guideRNA upstream and other downstream of the mutation to get an excision of the region comprised between the two gRNA target sequences. It is worthwhile to test different combinations of gRNA pairs, as the excision efficiency of the different pairs may be different. We used 4 Î¼l of Lipofectamine 2000 and 0.8 Î¼g of each plasmid. A different transfection reagent or different transfection conditions may be necessary for other cell types.</li><li>Extract total DNA from transfected cells 48 h after transfection using DNeasy Blood and Tissue kit. Check Total Nucleic acid concentration using Nanodrop 2000.<br/><em>Note: RNase may be added as described in the DNeasy kit protocol to remove RNA from the sample for a more accurate measuring of DNA concentration.</em> </li><li>Perform PCR of the three different targeted intronic regions using the following set-up reaction and PCR program to detect excisions produced in genomic DNA from CFTE cells. Use the Screening primers described in Procedure A and the following PCR conditions.<br/><br/><strong>PCR reaction setup</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Extracted DNA (~20 ng/Î¼l)<br/></td><td>3 Î¼l<br/></td></tr><tr><td>5x Phusion Buffer<br/></td><td>4 Î¼l <br/></td></tr><tr><td>10 mM dNTPS<br/></td><td>0.4 Î¼l<br/></td></tr><tr><td>10 Î¼M FW primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>10 Î¼M RV primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Phusion polymerase<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Nuclease-free water<br/></td><td>Up to 20 Î¼l<br/></td></tr></tbody></table><br/><strong>PCR Program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>35 cycles of<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>57 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>1 min<br/></td></tr><tr><td>Final extension<br/></td><td>72 Â°C<br/></td><td>5 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Perform 1.5% agarose gel electrophoresis in TAE buffer to see products of smaller size corresponding to the different produced excisions (Figure 1). Select the most efficient plasmid pairs in producing excision on each of the three target intronic regions using densitometry analysis software (Bio-Rad) from the Bio-Rad Gel Doc System.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190529/20190529193000_1431.jpg" width="113"/><br/><strong>Figure 1. Excisions produced by different pairs of Cas9/gRNA expressing plasmids in intron 12 (Top), intron 19 (mid) or intron 22 (bottom) of genomic <em>CFTR</em>.</strong> gin12U2 and gin12D1 were identified as most efficient pair to excise selected sequence in intron 12, meanwhile gin19U1 and gin19D1 pair, and gin22U1 and gin22D1 pair were identified as the most efficient to excise target sequences in intron 19 and 22 respectively.<br/><br/></li></ol></li><li>(Optional) Construction of pTandem plasmids<br/>Once the most efficient pair of gRNAs that excise the particular intronic region of interest have been identified, it is possible to clone the expression cassettes from plasmids expressing gRNAs targeting the sequence upstream (gin12U2, gin19U1 and gin22U1) and sub-clone them into NheI/XbaI sites of plasmids expressing gRNAs targeting sequences downstream in the same intronic region (gin12D1, 19D1 and gin22D1) respectively, to obtain three plasmids (pTandem-in12, pTandem-in19 and pTandem-in22) that express the two gRNAs from each pair. This refinement is optional as the excision efficiency of using the two plasmids expressing one gRNA as a pair is similar in all the three regions of interest.<ol class="npwtwo"><li>PCR amplify the gRNA expression cassettes from the plasmids that express the upstream gRNA (gin12U2, gin19U1 and gin22U1) using primers that incorporate NheI at 5â² and XbaI at 3â² of each PCR product respectively.<br/><br/><strong>Oligos for p-Tandem plasmids construction</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>Pc9tand fwNheI<br/></td><td>CACACAGCTAGCGAGGGCCTATTTCCCATGATT<br/></td></tr><tr><td>Pc9tand rvXbaI<br/></td><td>CACACATCTAGATCTAGCTCTAAAACAAAAAAGC<br/></td></tr></tbody></table><br/><strong>PCR reaction setup to amplify gRNA expression cassettes</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>Plasmid (~20 ng/Î¼l)<br/></td><td>3 Î¼l <br/></td></tr><tr><td>5x Phusion Buffer<br/></td><td>4 Î¼l<br/></td></tr><tr><td>10 mM dNTPS<br/></td><td>0.4 Î¼l<br/></td></tr><tr><td>10 Î¼M Pc9tand fwNheI<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>10 Î¼M Pc9tand rvXbaI<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Phusion polymerase<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Nuclease free water<br/></td><td>Up to 20 Î¼l<br/></td></tr></tbody></table><br/><strong>PCR program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>35 cycles of<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>57 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>Final extension<br/></td><td>72 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Purify PCR product with High Pure PCR product purification kit (Sigma-Aldrich).</li><li>Digest 1 Î¼g of each of the three PCR products and 1 Î¼g of the plasmids expressing the corresponding downstream gRNA for that region (gin12D1, gin19D1 and gin22D1) with NheI and XbaI enzymes.<br/><br/><strong>Enzyme digestion</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>DNA (1 Î¼g/Î¼l) <br/></td><td>1 Î¼l<br/></td></tr><tr><td>XbaI (10 U/Î¼l)<br/></td><td>1 Î¼l<br/></td></tr><tr><td>NheI (10 U/Î¼l)<br/></td><td>1 Î¼l<br/></td></tr><tr><td>Tango buffer<br/></td><td>1 Î¼l<br/></td></tr><tr><td>Nuclease free water<br/></td><td>Up to 10 Î¼l<br/></td></tr></tbody></table><br/></li><li>Purify digested PCR and plasmids using High Pure PCR product purification kit (Sigma-Aldrich)</li><li>Set up Ligation reactions, each purified digested PCR product coming from each of the upstream gRNA expression plasmids (gin12U2, gin19U1 and gin22U1) with the corresponding digested downstream expression plasmid (gin12D1, gin19D1 and gin22D1) respectively. Keep reactions at 16 Â°C for 4 h.<br/><br/><strong>Ligation reaction setup</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td>Purified Digested PCR product<br/></td><td>1 Î¼l (50 ng/Î¼l)<br/></td></tr><tr><td>Purified Digested plasmid<br/></td><td>1 Î¼l (150 ng/Î¼l)<br/></td></tr><tr><td>T4 ligase buffer<br/></td><td>1 Î¼l<br/></td></tr><tr><td>T4 ligase<br/></td><td>1 Î¼l<br/></td></tr><tr><td>Nuclease free water<br/></td><td>Up to 10 Î¼l<br/></td></tr></tbody></table><br/></li><li>Transform a vial of 50 Î¼l of competent <em>E. coli</em> (NEB) with 3 Î¼l of each of the ligation reactions, and seed in LB agar dishes (100 Î¼g/ml of Ampicillin). Incubate for 16 h at 37 Â°C.</li><li>Pick five colonies for each transformation and grow in 5 ml of LB medium in 15 ml Polypropylene tubes (100 Î¼g/ml of Ampicillin) for 24 h. Extract plasmid DNA after that.</li><li>Sequence extracted plasmid DNA using specific primers to confirm the correct sequence of the cloned construct in at least one colony.<br/><br/><strong>Oligos for p-Tandem plasmids sequencing</strong><br/><div style="white-space:nowrap;"><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>pcas9tandemSEQfw<br/></td><td>TTTGTGATGCTCGTCAGGGG<br/></td></tr><tr><td>pcas9tandemSEQrv<br/></td><td>TGGAAAGTCCCTATTGGCGT<br/></td></tr></tbody></table><br/></div></li></ol></li><li>Cotransfection of HEK293T cells with <em>wt</em>, mutant minigenes and mutant minigenes and selected pairs of Cas9/gRNAs expressing plasmids<ol class="npwtwo"><li>Prior to transfection, HEK293T cells were kept in culture and passaged twice a week with Trypsin-EDTA in a T75 cell culture flask with DMEM + 10% FBS + 1% Penicillin/Streptomycin at 37 Â°C and 5% CO<sub>2</sub>.</li><li>Seed 1 x 10<sup>5</sup> HEK293T cells in 12-well plates in DMEM + 10% FBS medium without antibiotics 24 h prior to transfection.</li><li>For each intronic variant, transfect HEK293T cells with (i) just <em>wild type</em> minigene construct, (ii) just mutant minigene construct, or (iii) the mutant minigene construct and corresponding pTandem plasmid. Use 4 Î¼l of Lipofectamine 2000 and 1.6 Î¼g of total plasmid DNA, with an equimolar distribution when more than one plasmid is used. Table 1 shows a summary of the different experimental conditions.<br/><br/><strong>Table 1. Summary of different experimental conditions in minigenes assay</strong><br/><img alt="" class="layerphoto" height="228" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190529/20190529194701_4684.jpg" width="600"/><br/><br/></li><li>Forty-eight hours after transfection, Extract total DNA and total RNA from each transfected HEK293T sample using DNeasy Blood and Tissue Kit (QIAGEN) and NucleoSpin RNA (MACHEREY-NAGEL) respectively.</li><li>Extract total DNA from cells transfected with both mutant minigene and corresponding pTandem plasmid.</li><li>Set up the following PCR amplification to detect excisions produced in each mutant minigene by the corresponding pTandem plasmid. In order to have PCR products coming from excised and non-excised minigenes that can be distinguished in an agarose-based electrophoresis, use one sequencing primer of the vector (Step A9) and one screening primer of the corresponding cloned sequence (Step A13) as indicated:<br/><br/><strong>Primers used to detect excisions in mutant minigenes</strong><br/>In pMG-in12 minigene: pspSEQrv and in12Drv<br/>In pMG-in19 minigene: in19Ufw and pspSEQrv<br/>In pMG-in23 minigene: pspSEQfw and in22Drv<br/><strong>PCR reaction setup</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>Extracted DNA (~20 ng/Î¼l)<br/></td><td>3 Î¼l<br/></td></tr><tr><td>5x Phusion Buffer<br/></td><td>4 Î¼l<br/></td></tr><tr><td>10 mM dNTPS<br/></td><td>0.4 Î¼l<br/></td></tr><tr><td>10 Î¼M fw primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>10 Î¼M rv primer<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Phusion polymerase<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Nuclease free water<br/></td><td>Up to 20 Î¼l<br/></td></tr></tbody></table><br/><strong>PCR program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation <br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>35 cycles of<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>57 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>Final extension<br/></td><td>72 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Excisions produced by pTandem plasmids in mutant minigenes can be detected on a 1.5% agarose gel in TAE buffer, as a smaller amplification product (Figures 2F, 3F and 4F). Due to the existent homology between excised and non-excised PCR products, extra bands corresponding to heteroduplexes products may appear in the electrophoresis.</li><li>Produce cDNA using High-Capacity cDNA Reverse Transcription Kit (Thermo Fisher) from extracted RNA samples. Check produced cDNA concentration using Nanodrop 2000.</li><li>PCR amplify cDNA samples using specific primers for the exonic sequences in pSPL3.<br/><br/><strong>RT-PCR pSPL3 primers</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td>RTpspfw<br/></td><td>TCTGAGTCACCTGGACAACC<br/></td></tr><tr><td>RTpsprv<br/></td><td>ATCTCAGTGGTATTTGTGAGC<br/></td></tr><tr><td>5â² FAM- RTpspfw<br/></td><td>FAM-TCTGAGTCACCTGGACAACC<br/></td></tr></tbody></table><br/><strong>PCR reaction setup</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>cDNA (~20 ng/Î¼l)<br/></td><td>3 Î¼l <br/></td></tr><tr><td>5x Phusion Buffer<br/></td><td>4 Î¼l <br/></td></tr><tr><td>10 mM dNTPS <br/></td><td>0.4 Î¼l<br/></td></tr><tr><td>10 Î¼M RTpspfw<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>10 Î¼M RTpsprv<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Phusion polymerase<br/></td><td>0.5 Î¼l<br/></td></tr><tr><td>Nuclease free water<br/></td><td>Up to 20 Î¼l<br/></td></tr></tbody></table><br/><strong>PCR program</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>Initial Denaturation<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>35 cycles of<br/></td><td>98 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>57 Â°C<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>72 Â°C<br/></td><td>30 s<br/></td></tr><tr><td>Final extension<br/></td><td>72 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>Hold<br/></td><td>4 Â°C<br/></td><td><br/></td></tr></tbody></table><br/></li><li>Run 1.5% agarose gel electrophoresis of RT-PCR products, and gel extract and purify the different-sized fragments using StrataPrep kit (Agilent).</li><li>Sequence the extracted bands using RTpspfw and RTpsprv primers to determine the effect on splicing of the introduced mutations in the minigenes when compared to <em>wt</em> minigene, and to know the effect of Cas9/gRNA-based excision in restoring aberrant splicing on each mutant minigene.</li><li>Repeat RT-PCR as above but using a 5â² FAM RTpspfw labeled primer (Eurofins) and non-labeled standard RTpsprv to obtain 5â² FAM labeled PCR reactions.</li><li>Purify 5â² FAM labeled PCR reactions using high pure PCR product purification (Sigma-Aldrich) and perform fragment length analysis (Eurofins) to quantify the percentage of each RT-PCR product as the area of the peak. In fragment length analysis fluorescent-labeled PCR products are separated using capillary electrophoresis and sized by comparison to a size standard.</li></ol></li></ol><p class="pptt" id="biaoti27875">### Data analysis</p><p class="ppzz">Experiments were performed in triplicate.<br/>  Targeted excisions produced by the gRNA pairs in CFTE cells were quantified using GelDoc Software analysis (Bio-Rad) after PCR and Agarose gel Electrophoresis. The selected gRNA pairs were able to produce excisions in genomic <em>CFTR</em> gene with an efficiency of 33%, 30% and 47%, measured as the percentage of PCR products of reduced size (Figure 1).<br/>  Figures 2F, 3F and 4F show the percentage of amplicon of reduced size coming from the excised plasmid; this percentage was quantified using GelDoc Software analysis (Bio-Rad).<br/>  The FLA electropherograms in Figures 2-4 shows the size and percentage of the transcripts from (i) the <em>wt</em> minigene (2B, 3B, 4B), (ii) from the mutated minigene (2D, 3D, 4D), and (iii) the mutated minigene treated with the corresponding excision maker pTandem-In plasmid (2G, 3G, 4G). Those percentages were supplied by the FLA service and correspond to the area of the said peaks compared to the total area of the peaks in the electropherogram. All the three mutations of interest produce aberrant splicing with no production (2D, 3D) or partial production (4D) of normal transcript; this aberrant splicing is almost completely restored with up to 90% (2G, 3G, 4G) of normal transcript produced when the pTandem-in plasmids excise the intronic regions containing the target mutation.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190529/20190529201005_1198.jpg" width="216"/><br/><strong>Figure 2. Mini-gene analysis of c.1679+1634&gt;G mutation.</strong> A. Schematic of pMG-in12-WT plasmid comprising a contiguous ~1.5 kb region of <em>CFTR</em> containing part of intron 12, exon 13 and part of intron 13 cloned between the 5â² and 3â² exons of the pSPL3 vector. Exons are shown as boxes, introns as lines, and predicted transcript as thicker line shown underneath. Arrows below transcript represent pspRTfw and pspRTrv primers for RT-PCR. B. Electropherogram analysis of RT-PCR products (filled peaks) generated from HEK293T cells transfected with pMG-in12-WT and size markers (unfilled peaks). C. Schematic of pMG-in12-A&gt;G mutation which differs from pMG-in12-WT by the A&gt;G change shown with dotted arrow. The A&gt;G change creates a pseudo exon (Î¨-EX) which contains an in-frame stop codon (TAA). The triangles flanking the Î¨-EX indicate target sites of Cas9/gRNAs encoded by pTandem-in12 vector. Arrows above DNA represent pspSEQfw and in12Drv primers for PCR. D. Electropherogram analysis of RT-PCR products generated from HEK293T cells transfected with pMG-in12-A&gt;G. E. Schematic of pMG-in12-A&gt;G mutation following targeted excision and repair. F. Agarose gel electrophoresis analysis of targeted deletions in pMG-in12-A&gt;G measured by PCR. â-â lane is PCR products generated from HEK293T cells transfected with pMG-in12-A&gt;G only, â+â lane is PCR products generated from HEK293T cells transfected with pMG-in12-A&gt;G and pTandem-in12. The top band is generated from untargeted DNA, the bottom band is generated from DNA containing the targeted deletion, and the middle band is most likely heteroduplexes formed during the late stages of PCR. G. Electropherogram analysis of RT-PCR products generated from HEK293T cells co-transfected with pMG-in12-A&gt;G and pTandem-in12 vector.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190529/20190529201030_0357.jpg" width="256"/><br/><strong>Figure 3. Mini-gene analysis of c.3140-26A&gt;G mutation.</strong> A. Schematic of pMG-in19-WT plasmid comprising a contiguous ~2.0 kb region of <em>CFTR</em> containing part of intron 18, exon 19, all of intron 19, exon 20 and part of intron 20 cloned between the 5â² and 3â² exons of the pSPL3 vector. B. Electropherogram analysis of RT-PCR products generated from HEK293T cells transfected with pMG-in19-WT. C. Schematic of pMG-in19-A&gt;G mutation which differs from pMG-in19-WT by the A&gt;G change shown with dotted arrow. The A&gt;G change extends exon 20 by 25 bp which shifts the reading frame and eventually results in-frame stop codon (TGA). The triangles indicate target sites of Cas9/gRNAs encoded by pTandem-in19 vector. Arrows above DNA represent in19Ufw and pspSEQrv primers for PCR. D. Electropherogram analysis of RT-PCR products generated from HEK293T cells transfected with pMG-in19-A&gt;G. E. Schematic of pMG-in19-A&gt;G mutation following targeted excision and repair. F. Agarose gel electrophoresis analysis of targeted deletions in pMG-in19-A&gt;G measured by PCR. â-â lane is PCR products generated from HEK293T cells transfected with pMGin19-A&gt;G only, â+â lane is PCR products generated from HEK293T cells transfected with pMG-in19-A&gt;G and pTandem-in12. G. Electropherogram analysis of RT-PCR products generated from HEK293T cells co-transfected with pMG-in19-A&gt;G and pTandem-in19 vector.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190529/20190529201101_1692.jpg" width="211"/><br/><strong>Figure 4. Mini-gene analysis of c.3718-2477C&gt;T mutation.</strong> A. Schematic of pMG-in22-<em>WT</em> plasmid comprising a contiguous ~3.2 kb region of <em>CFTR</em> containing part of intron 22, exon 23, and part of intron 23 cloned between the 5â² and 3â² exons of the pSPL3 vector. B. Electropherogram analysis of RT-PCR products (filled peaks) generated from HEK293T cells transfected with pMG-in22-WT. C. Schematic of pMG-in22-C&gt;T mutation which differs from pMG-in22-<em>WT</em> by the C&gt;T change shown with dotted arrow. The C&gt;T change creates a pseudo exon (Î¨-EX) which contains an in-frame stop codon (TAA). The triangles flanking the Î¨-EX indicate target sites of Cas9/gRNAs encoded by pTandem-in22 vector. Arrows above DNA represent pspSEQfw and in22Drv primers for PCR. D. Electropherogram analysis of RT-PCR products generated from HEK293T cells transfected with pMGin22-C&gt;T. E. Schematic of pMG-in22-C&gt;T mutation following targeted excision and repair. F. Agarose gel electrophoresis analysis of targeted deletions in pMG-in22-C&gt;T measured by PCR. â-â lane is PCR products generated from HEK293T cells transfected with pMG-in22-C&gt;T only, â+â lane is PCR products generated from HEK293T cells transfected with pMG-in22-C&gt;T and pTandem-in22. G. Electropherogram analysis of RT-PCR products generated from HEK293T cells co-transfected with pMG-in22-C&gt;T and pTandem-in22 vector.</p><p class="pptt" id="biaoti27876">### Acknowledgments</p><p class="ppzz">This protocol was used for the work previously published in PLOS ONE (Sanz <em>et al.</em>, 2017). We want to thank Dieter Gruenert for the CFTE cell line and to Feng Zhang (Broad Institute, Boston, MA) for the pSpCas9(BB)-2A-GFP plasmid. Cystic Fibrosis Foundation (Bethesda, MD, USA) and Cystic Fibrosis Trust (London, UK) funded this work.</p><p class="pptt" id="biaoti27877">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti27878">### References</p><ol class="npwtwok"><li>Beck, S., Penque, D., Garcia, S., Gomes, A., Farinha, C., Mata, L., Gulbenkian, S., Gil-Ferreira, K., Duarte, A., Pacheco, P., Barreto, C., Lopes, B., Cavaco, J., Lavinha, J. and Amaral, M. D. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10425036" target="_blank">Cystic fibrosis patients with the 3272-26A--&gt;G mutation have mild disease, leaky alternative mRNA splicing, and CFTR protein at the cell membrane.</a> <em>Hum Mutat</em> 14(2): 133-144.</li><li>ChillÃ³n, M., Dork, T., Casals, T., Gimenez, J., Fonknechten, N., Will, K., Ramos, D., Nunes, V. and Estivill, X. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7534040" target="_blank">A novel donor splice site in intron 11 of the <em>CFTR</em> gene, created by mutation 1811+1.6kbA--&gt;G, produces a new exon: high frequency in Spanish cystic fibrosis chromosomes and association with severe phenotype.</a> <em>Am J Hum Genet</em> 56(3): 623-629.</li><li>Gruenert, D. C., Willems, M., Cassiman, J. J. and Frizzell, R. A. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15463957" target="_blank">Established cell lines used in cystic fibrosis research.</a> <em>J Cyst Fibros</em>. 3 Suppl 2:191-6.</li><li>Highsmith, W. E., Burch, L. H., Zhou, Z., Olsen, J. C., Boat, T. E., Spock, A., Gorvoy, J. D., Quittel, L., Friedman, K. J., Silverman, L. M. and et al. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7521937" target="_blank">A novel mutation in the cystic fibrosis gene in patients with pulmonary disease but normal sweat chloride concentrations.</a> <em>N Engl J Med</em> 331(15): 974-980.</li><li>Hollywood, J. A., Lee, C. M., Scallan, M. F. and Harrison, P. T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27557525" target="_blank">Analysis of gene repair tracts from Cas9/gRNA double-stranded breaks in the human <em>CFTR</em> gene.</a> <em>Sci Rep</em> 6: 32230.</li><li>Sanz, D. J., Acedo, A., Infante, M., Duran, M., Perez-Cabornero, L., Esteban-Cardenosa, E., Lastra, E., Pagani, F., Miner, C. and Velasco, E. A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20215541" target="_blank">A high proportion of DNA variants of BRCA1 and BRCA2 is associated with aberrant splicing in breast/ovarian cancer patients.</a> <em>Clin Cancer Res</em> 16(6): 1957-1967.</li><li>Sanz, D. J., Hollywood, J. A., Scallan, M. F. and Harrison, P. T. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28863137" target="_blank">Cas9/gRNA targeted excision of cystic fibrosis-causing deep-intronic splicing mutations restores normal splicing of <em>CFTR</em> mRNA.</a> <em>PLoS One</em> 12(9): e0184009.<br/></li></ol>