<body>
            <h1>Cross-linking, Immunoprecipitation and Proteomic Analysis to Identify Interacting Proteins in Cultured Cells</h1>
            <p><strong>Authors:</strong> Hao  Wang, Meiling  He, Belinda Willard, Qingyu   Wu</p>
            <h2>Abstract</h2>
            <p>Extracellular expression is essential for the function of secreted and cell surface proteins. Proper intracellular trafficking depends on protein interactions in multiple subcellular compartments. Co-immunoprecipitation and the yeast two-hybrid system are commonly used to investigate protein-protein interactions. These methods, however, depend on high-affinity protein interactions. In many glycoproteins, glycans are important for protein intracellular trafficking and extracellular expression. If glycoprotein interactions are transient and relatively weak, it may be challenging to use co-immunoprecipitation or the two-hybrid system to identify glycoprotein-binding partners. To circumvent this problem, protein cross-linking can be applied first to immobilize the transient and/or low-affinity protein interactions. Here we describe a protocol of protein cross-linking, co-immunoprecipitation, and proteomic analysis, which was used to identify endoplasmic reticulum (ER) chaperones critical for the folding and ER exiting of N-glycosylated serine proteases in human embryonic kidney (HEK) 293 cells. This approach can be used to identify other protein interactions in a variety of cells.</p>
            <p><strong>Keywords:</strong> Corin, Immunoprecipitation, N-glycosylation, Protein cross-linking, Protein-protein interaction, Proteomic analysis, Serine protease</p>
            <h2>Background</h2><p>Natriuretic peptides play a key role in body fluid homeostasis (Kuhn, 2016; Matsuo <em>et al.</em>, 2019). Corin is a transmembrane protease expressed on the surface of cardiac myocytes, where it activates the natriuretic peptides to regulate blood pressure and cardiac function (Yan <em>et al.</em>, 2000; Zhou and Wu, 2014). Corin is synthesized as a zymogen (Knappe <em>et al.</em>, 2003), which is converted to an active enzyme by proprotein convertase subtilisin/kexin-6 (PCSK6) on the cell surface (Chen <em>et al.</em>, 2015; Chen <em>et al.</em>, 2018). Naturally occurring mutations that impair corin cell surface expression and activation have been found in patients with hypertension and heart disease (Wang <em>et al.</em>, 2012; Dong <em>et al.</em>, 2013 and 2014; Zhang <em>et al.</em>, 2014 and 2017).<br/><br/>Human corin is a glycoprotein with 19 N-glycosylation sites (Yan <em>et al.</em>, 1999; Hooper <em>et al.</em>, 2000). N-glycosylation is essential for the cell surface expression and activation of corin (Liao <em>et al.</em>, 2007; Gladysheva <em>et al.</em>, 2008). In site-directed mutagenesis studies, a corin mutant without N-glycans in the protease domain had reduced levels on the cell surface due to impaired intracellular trafficking (Wang <em>et al.</em>, 2015 and 2018). Further immune staining experiments showed that the mutant protein was retained in the ER (Wang <em>et al.</em>, 2018). These results suggest that N-glycans on corin, particularly those in the protease domain, may interact with other ER proteins that are critical for corin intracellular trafficking (Wang <em>et al.</em>, 2018). Identification of such ER proteins should help to understand the cellular mechanism in regulating corin expression and function.<br/><br/>Co-immunoprecipitation and the yeast two-hybrid system are commonly used to analyze protein-protein binding and complex formation (Berggard <em>et al.</em>, 2007; Kaboord and Perr, 2008). These methods are suitable mostly for studying stable and/or high-affinity protein-protein interactions. In many cases, however, protein interactions in specific subcellular compartments are transient and unstable. In glycoprotein synthesis, for example, transient N-glycan-protein interactions are essential for glycoprotein folding and subsequent ER exiting (Ellgaard and Frickel, 2003; Lamriben <em>et al.</em>, 2016). The traditional methods such as protein co-immunoprecipitation and the two-hybrid system may not be suitable for studying such N-glycan-protein interactions. To circumvent this problem, protein cross-linking can be applied to immobilize the transient and/or weak protein interactions before co-immunoprecipitation proceeds. <br/><br/>In a recent study, we designed a protocol of protein cross-linking, co-immunoprecipitation, and proteomic analysis to examine the role of N-glycans in corin intracellular trafficking. We expressed corin wild-type (WT) and a mutant lacking the N-glycosylation site in the protease domain (N1022Q) in separate HEK293 cells. The cells were treated with dithiobis succinimidyl propionate (DSP), a cell membrane permeable cross-linker, which has an amine-reactive <em>N</em>-hydroxysuccinimide (NHS) ester at each end of a cleavable spacer (Mattson <em>et al.</em>, 1993). NHS esters react with primary amines in the side chain of lysine residues and the N-termini of proteins, thereby forming stable amide bonds connecting co-localized proteins (Mattson <em>et al.</em>, 1993). Proteins in HEK293 cells cross-linked to corin WT and the mutant were isolated by immunoprecipitation. After breaking the disulfide bond in the spacer of DSP under reducing conditions, proteins were separated by SDS-PAGE and analyzed by in-gel digestion and liquid chromatography-mass spectrum (LC-MS). By comparing proteins that were differentially bound to corin WT and the mutant, we identified calnexin as a key ER chaperone that mediates the N-glycan-dependent folding and ER exiting of corin and other N-glycosylated serine proteases such as enteropeptidase and prothrombin (Wang <em>et al.</em>, 2018).</p>
        </body>

<p class="pptt" id="biaoti27946">## Materials and Reagents</p><ol class="npwtwok"><li>Cell lifters (Corning, catalog number: 3008)</li><li>Surgical blades (any brand)</li><li>Pipette tips (any brand)</li><li>1.5 ml microcentrifuge tubes (any brand)</li><li>150 mm cell culture dishes (Corning, catalog number: 430599)</li><li>HEK293 cells (ATCC, catalog number: CRL-1573)</li><li>pcDNA 3.1/V5-His-based plasmids (Thermo Fisher, catalog number: K480001)</li><li>Dulbecco's modified Eagleâs medium (DMEM) (Lerner Research Institute Cell Culture Core, catalog number: 11-500)</li><li>Fetal bovine serum (FBS) (Corning, catalog number: 35-011-CV)</li><li>Phosphate buffered saline (PBS), 10x (Affymetrix, catalog number: 75889)</li><li>Dithiobis succinimidyl propionate (DSP) (Thermo Fisher, catalog number: 22585)</li><li>Dimethyl sulfoxide (DMSO) (Thermo Fisher, catalog number: 24600)</li><li>Glycine (Research Products International (RPI), catalog number: G36050)</li><li>Tris-base (Fisher Scientific, catalog number: 502-13-709)</li><li>Sodium chloride (RPI, catalog number: S23020)</li><li>Nonidet P-40 (Affymetrix, catalog number: 19628)</li><li>Protease inhibitor cocktail (Sigma-Aldrich, catalog number: P8340)</li><li>Protein assay dye reagent (Bio-Rad, catalog number: 500-0006)</li><li>Bovine serum albumin (BSA) (Sigma-Aldrich, catalog number: A9647)</li><li>Anti-V5 antibody (Thermo Fisher, catalog number: R96025)</li><li>Protein A-Sepharose (Thermo Fisher, catalog number: 10-1042)</li><li>Sodium dodecyl sulfate (SDS) (Fisher Scientific, catalog number: BP166500)</li><li>SDS-PAGE protein sample buffer (2x) (Bio-Rad, catalog number: 1610737)</li><li>Î²-mercaptoethanol (Fisher Scientific, catalog number: BP176-100)</li><li>Pre-stained protein ladder (Thermo Fisher, catalog number: 26616)</li><li>Tris-Glycine gel (4-20%) (Thermo Fisher, catalog number: XP04200BOX)</li><li>Silver staining kit (Thermo Fisher, catalog number: 24600)</li><li>Ethanol (any brand, HPLC grade)</li><li>Acetic acid (any brand, HPLC grade)</li><li>Acetonitrile (any brand, HPLC grade)</li><li>SilverQuest silver staining kit (Thermo Fisher, catalog number: LC607)</li><li>Dithiothreitol (DTT) (Thermo Fisher, catalog number: R0861)</li><li>Iodoacetamide (Thermo Fisher, catalog number: A39271)</li><li>Trypsin, sequencing grade (Thermo Fisher, catalog number: 90057)</li><li>Ammonium bicarbonate (Fisher Scientific, catalog number: A643-500)</li><li>Formic acid (Thermo Fisher, catalog number: 28905)</li><li>Fugene reagents (Promega, catalog number: E2311)</li><li>G418 (Teknova, catalog number: G5001)</li><li>DSP stock solution (see Recipes)</li><li>Cell lysis buffer (see Recipes)</li><li>SDS-PAGE buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27947">### Equipment</p><ol class="npwtwok"><li>Pipetman P20 (Gilson, catalog number: F144801)</li><li>Pipetman P200 (Gilson, catalog number: F144801)</li><li>Pipetman P1000 (Gilson, catalog number: F144801)</li><li>Humidified cell culture incubator (any brand)</li><li>Microcentrifuge (Thermo Fisher, catalog number: 75002446)</li><li>Benchtop rocker (any brand)</li><li>Mini gel apparatus (Thermo Fisher, catalog number: A25977)</li><li>Spectrophotometer or microplate reader (any brand with 595 nm wavelength)</li><li>Power supply (Bio-Rad, catalog number: 1645052)</li><li>Speedvac (Savant, AES1010-120)</li><li>Ultimate Nano-3000 HPLC (Dionex, RSLCnano 3000)</li><li>LTQ-Obitrap hybrid mass spectrometer system (Thermo Scientific; Elite)</li><li>Trapping column: Dionex 2 cm x 75 Î¼m id Acclaim Pepmap C18, 5 Î¼m, 100 Ã reversed-phase capillary (Thermo Fisher, catalog number: 164564)</li><li>Analytical column: Dionex 15 cm x 75 Î¼m id Acclaim Pepmap C18, 2 Î¼m, 100 Ã reversed-phase capillary chromatography column (Thermo Fisher, catalog number: 164534)</li><li>Ice bucket (any brand)<br/></li></ol><p class="pptt" id="biaoti27948">### Software</p><ol class="npwtwok"><li>Xcaliber version 2.2 (Thermo Scientific)</li><li>UniProtKB (<a href="https://www.uniprot.org/help/uniprotkb" target="_blank">https://www.uniprot.org/help/uniprotkb</a>), downloaded on June 29, 2016</li><li>Mascot version 2.3.0 (Matrix Science)</li><li>Proteome Discoverer 2.2 (Thermo Scientific)</li><li>Scaffold version 4.0.6.1 (Proteome Software)<br/></li></ol><p class="pptt" id="biaoti27949">### Procedure</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwtwo"><li><em>pcDNA 3.1/V5-His-based plasmids expressing human corin WT and the N1022Q mutant were described previously (Wang </em><em>et al.</em><em>, 2015). The corin proteins encoded by these plasmids contain a C-terminal V5 tag that is recognized by an anti-V5 antibody.</em> </li><li><em>The corin-expressing plasmids were transfected into HEK293 cells (authenticated by STR DNA profiling, no mycoplasma contamination) using Fugene reagents. The cells were cultured in DMEM with 10% FBS and 400 Î¼g/ml of G418 at 37 Â°C in a humidified incubator with 5% CO<sub>2</sub> to select stable corin-expressing cell clones. The experimental procedures were described previously (Wang </em><em>et al.</em><em>, 2018).</em><br/><br/></li></ol><ol class="npwone"><li>Protein cross-linking in cells<ol class="npwtwo"><li>Seed HEK293 cells stably expressing corin WT and the mutant N1022Q in two separate 150 mm dishes with DMEM and 10% FBS (~5 x 10<sup>6</sup> cells/dish in 25 ml medium).<br/><em>Note: In the mutant N1022Q, the N-glycosylation site at N1022 in the protease domain was mutated (Figure 1).</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190531/20190531001636_6058.jpg" width="264"/><br/><strong>Figure 1. Domain structures of corin WT and the mutant N1022Q.</strong> Corin consists of an N-terminal cytoplasmic tail, a transmembrane domain (TM) and an extracellular region containing two frizzled (Fz) domains, eight LDL receptor (LDLR) repeats, a scavenger receptor (SR) domain, and a serine protease domain. Y-shaped symbols indicate N-glycosylation sites. In the mutant N1022Q, the N-glycosylation site at N1022 in the protease domain is mutated. Recombinant corin proteins contain a C-terminal V5 tag (diamond shape) used for immunoprecipitation.<br/><br/></li><li>Culture the cells at 37 Â°C in a humidified incubator with 5% CO<sub>2</sub> until the cells reach ~90% of confluency (~36 h).</li><li>Remove the medium by suction and wash the cells once with PBS (pre-chilled on ice, 10 ml per dish).<br/><em>Note: Figure 2 illustrates the protocol flow chart in the experiments of protein cross-linking, immunoprecipitation, SDS-PAGE, in-gel digestion, and LC-MS proteomic analysis, which are described below.</em><br/><br/><img alt="" class="layerphoto" height="197" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190531/20190531001700_5813.jpg" width="100"/><br/><strong>Figure 2. Protocol flow chart.</strong> In protein cross-linking, recombinant corin protein containing a C-terminal V5 tag (diamond shape) in HEK293 cells was cross-linked with interacting proteins (brown), using dithiobis succinimidyl propionate (DSP). In immunoprecipitation, corin-associated proteins were isolated using an anti-V5 antibody and protein A-Sepharose beads. In proteomic analysis, corin-associated proteins were eluted from the beads under reducing conditions and analyzed by SDS-PAGE followed by in-gel trypsin digestion and LC-MS analysis. The experiments were done in parallel in HEK293 cells expressing corin WT and the mutant N1022Q. Proteins that were differentially associated with corin WT and the mutant N1022Q were selected for further biochemical and cellular studies.<br/><br/></li><li>Add DSP (0.8 mg/ml in PBS, 10 ml per dish) to the cells and incubate at 4 Â°C for 30 min.<br/><em>Note: The chemical structure of DSP is shown in Figure 3.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190531/20190531001730_1763.jpg" width="242"/><br/><strong>Figure 3. Chemical structure of DSP.</strong> The chemical structure of the homobifunctional cross-linker DSP is shown. The indicated disulfide bond will be broken upon reduction.<br/><br/></li><li>Stop the cross-linking reaction by adding 1 ml of PBS containing 0.2 M glycine at 4 Â°C and incubate for 15 min. No need to remove the DSP solution.</li><li>Discard the reaction solution and gently wash the cells with 10 ml pre-chilled PBS.</li><li>Use a cell lifter to scrap the cells off the dish in 1.5 ml PBS and transfer the cells in suspension to a 1.5 ml microcentrifuge tube.</li><li>Spin down the cells with a microcentrifuge (900 <em>x g</em> at 4 Â°C for 5 min) and discard the supernatant.<br/><br/></li></ol></li><li>Immunoprecipitation of bait (corin)-associated proteins<ol class="npwtwo"><li>Lyse the pelleted cells in a cell lysis buffer containing Nonidet P-40 (see Recipes) (750 Î¼l) at 4 Â°C for 30 min. <br/><em>Note: Tap the tube every 5 min to ensure that the cells are completely lysed.</em> </li><li>Centrifuge the cell lysate at 16,200 <em>x g</em>, 4 Â°C, for 10 min.</li><li>Collect the supernatant (cell lysate) and discard the insoluble cellular debris.</li><li>Take 1 Î¼l of the cell lysate sample to measure the protein concentration using the protein assay dye kit and a spectrophotometer.</li><li>Transfer the cell lysate (2,000 Î¼g of total proteins) to a new microcentrifuge tube and dilute the sample to 1 ml in the cell lysis buffer.</li><li>Immunoprecipitate corin proteins (bait) and associated proteins (prey) in the cell lysate by incubating with an anti-V5 antibody (1:1000) and protein A-Sepharose beads (50 Î¼l) at 4 Â°C for 2 h on a benchtop rocker (with gentle shaking).</li><li>Spin down the beads at 16,200 <em>x g</em>, 4 Â°C, for 1 min and remove the supernatant.</li><li>Wash the beads twice with pre-chilled PBS (1 ml, 5 min each).</li><li>Remove PBS by centrifugation (16,200 <em>x g</em>, 4 Â°C, 1 min).</li><li>Elute proteins from the beads and dissociate the bait-prey protein complexes with SDS-PAGE sample buffer (2x, 50 Î¼l) containing 5% Î²-mercaptoethanol at 37 Â°C for 30 min.</li><li>Spin down the beads at 16,200 <em>x g</em>, 4 Â°C, for 1 min and collect the supernatant.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Analysis of the eluted proteins by SDS-PAGE<ol class="npwtwo"><li>Load the entire eluted protein sample (50 Î¼l) onto a 4-20% gradient Tris-Glycine gel, together with PageRuler<sup>TM</sup> Prestained Protein Ladder (5 Î¼l).</li><li>Run the electrophoresis in SDS-PAGE buffer at a voltage â¤ 160 V.</li><li>Stop the electrophoresis before the dye front runs out of the gel.</li><li>Visualize proteins on SDS-PAGE gels using a commercial silver staining kit (Thermo Fisher), following the manufacturerâs instructions (Figure 4).<br/><br/><img alt="" class="layerphoto" height="205" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190531/20190531001752_2983.jpg" width="100"/><br/><strong>Figure 4. An example of silver-stained gel for proteomic analysis.</strong> Proteins eluted from Sepharose beads were analyzed by SDS-PAGE under reducing conditions followed by silver staining to visualize protein bands. The gel was cut horizontally into 11 slices, which were subjected to further in-gel trypsin digestion and LC-MS analysis.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>In-gel digestion of proteins<ol class="npwtwo"><li>Cut the stained gel into slices using a surgical blade (Figure 4).</li><li>Transfer each gel slice into a microcentrifuge tube. These gel pieces are then cut into 1 mm<sup>3</sup> pieces.</li><li>Wash gel pieces with 200 Î¼l deionized water and de-stain with SilverQuest silver staining kit according to manufacturerâs instructions. Wash away the destain reaction solution with 5 x 175 Î¼l deionized water.</li><li>Dehydrate gel pieces with 175 Î¼l acetonitrile for 5 min. Remove acetonitrile and dry the gel pieces in Speedvac for 3 min. Reduce proteins with 50 Î¼l DTT (5 mg/ml in 100 mM ammonium bicarbonate) for 30 min at room temperature. Remove DTT. Alkylate proteins with 50 Î¼l iodoacetamide (25 mg/ml in 100 mM ammonium bicarbonate) for 30 min at room temperature. Remove iodoacetamide.</li><li>Dehydrate gel pieces by adding 175 Î¼l acetonitrile and incubate for 5 min. Remove acetonitrile. Rehydrate gel pieces in 175 Î¼l 100 mM ammonium bicarbarbonate. Remove ammonium bicarbonate. Dehydrate gel pieces with 175 Î¼l acetonitrile for 5 min. Remove acetonitrile and dry gel pieces in Speedvac for 3 min.</li><li>Add 15 Î¼l of trypsin (10 ng/Î¼l in 50 mM ammonium bicarbonate) to the tube and incubate at room temperature overnight to ensure complete protein digestion. After overnight digestion, add 30 Î¼l extraction buffer (50% acetonitrile/5% formic acid). Mix and incubate for 10 min. Transfer the supernatant to an Eppendorf tube. Add a second 40 Î¼l aliquot of extraction buffer, mix and combine supernatants.</li><li>Reduce the volume of extract to &lt; 10 Î¼l in Speedvac.</li><li>Resuspend the extract in 1% acetic acid to a final volume of ~30 Î¼l for LC-MS analysis.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>LC-MS proteomic analysis<ol class="npwtwo"><li>Inject 5 Î¼l of the peptide extract into the LC-MS system and load onto the trapping column for 5 min at a flow rate of 10 Î¼l/min.</li><li>Elute the peptides from the trapping and reversed-phase columns at a flow rate of 0.3 Î¼l/min with a binary gradient starting at 95% A (0.1% formic acid) and 5% B (acetonitrile/0.1% formic acid) for 5 min followed by a linear increase from 2-40% B in 85 min (Figure 5). The column is washed by ramping to 80% B and holding for 5 min prior to re-equilibration of the column at 2% B for 15 min.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190531/20190531001818_6282.jpg" width="170"/><br/><strong>Figure 5. Illustration of a binary gradient in HPLC analysis</strong><br/><br/><em>Note: An example of peptide elution profile is shown in Figure 6.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190531/20190531001849_3827.jpg" width="142"/><br/><strong>Figure 6. A protein elution profile from one of the tryptic digests.</strong> Most of the tryptic peptides were eluted during the linear gradient from 2 to 40% acetonitrile.<br/><br/></li><li>The source conditions include a spray voltage of 2.0 kV and a capillary temperature of 220 Â°C. Other source parameters including tube lens, S-lens, and multipole voltages should be optimized regularly.</li><li>The Orbitrap Elite instrument is operated using the XCaliber Software package. Analyze the tryptic digests using a data-dependent acquisition method with the following parameters. Acquire full MS1 scans at a resolution of 60,000 in the Orbitrap MS. Select peptides for MS/MS analysis using a Top15 method with dynamic exclusion enabled (3 repeats in 50 s, exclusion duration of 90 s). Acquire MS/MS spectra in the ion trap with an isolation width of 3.0 Da and a collision energy of 35%.</li><li>Search the data using the programs Mascot and Sequest-HT which is bundled into Proteome Discoverer 1.4. To use Mascot, convert the RAW data files to mascot generic files (mgf files) using Proteome Discoverer 1.4. Search the data against the human UniProtKB database with both Mascot and Sequest using the following parameters: MS mass tolerance of 10 ppm, MS/MS mass tolerance of 0.6 Da, trypsin as the protease with full specificity, 2 missed cleavage sites, oxidized methionine as a variable modification, and carbamidomethylation as a fixed modification. Perform additional searches using the program Sequest specifically against the sequence of the bait protein to identify post-translational modifications such as phosphorylation, acetylation, and ubiquitination.</li><li>Perform protein and peptide validation by uploading the Proteome Discoverer (msf files) and Mascot (dat files) search results into the program Scaffold. Utilize Scaffold to perform additional database searches using X!Tandem with the same parameters as described above. Perform False Discover Rate analysis by searching the data against a reversed UniProtKB database and filter identifications based on a peptide level FDR of 0.1% and protein level FDR of 1%. Only consider proteins identified by two peptides, one of which is unique.</li></ol></li></ol><p class="pptt" id="biaoti27950">### Data analysis</p><p class="ppzz">The spectral counts in the LC-MS proteomic analysis (Step E) were examined. Proteins with high spectral counts were possible bait-interacting candidates. All proteins with spectral counts â¥ 10 were included for further analysis. To identify the proteins that differentially interacted with corin WT and the mutant, a ratio of â¥ 2-fold difference in spectral counts between WT and the mutant was used as a selection criterion. The selected candidates were analyzed for their subcellular expression patterns by searching the human UniProtKB database (<a href="http://www.uniprot.org/uniprot" target="_blank">www.uniprot.org/uniprot</a>), which includes information on protein subcellular locations. The proteins that are predominantly expressed in the targeted subcellular location, <em>i.e.</em>, ER in our study, were selected and tested in additional biochemical and cellular experiments (Wang <em>et al.</em>, 2018).<br/><em>Note: The criterion for protein spectral counts and the ratio between the control and the targeted protein may be modified depending on experimental settings.</em> </p><p class="pptt" id="biaoti27951">### Notes</p><ol class="npwtwok"><li>The protein expression system (in Step A1), including the plasmids and the cells, may be modified depending on the target, <em>i.e.</em>, bait, protein(s) of interest. The antibody used in immunoprecipitation (in Step B6) should also be modified based on the specific experimental design.</li><li>The LC-MS proteomic analysis was done at the Proteomic Core of the Lerner Research Institute of the Cleveland Clinic.<br/></li></ol><p class="pptt" id="biaoti27952">### Recipes</p><ol class="npwtwok"><li>DSP stock solution<br/>13 mM DSP in dimethyl sulfoxide<br/>Store at 4 Â°C for â¤1 week</li><li>Cell lysis buffer<br/>50 mM Tris-base (pH 8.0)<br/>150 mM sodium chloride<br/>1% Nonidet P-40 (vol/vol)<br/>1% protease inhibitor cocktail (vol/vol)</li><li>SDS-PAGE buffer<br/>25 mM Tris-base<br/>250 mM glycine<br/>3.5 mM SDS<br/></li></ol><p class="pptt" id="biaoti27953">### Acknowledgments</p><p class="ppzz">This work was supported in part by grants from the NIH (HL126697, HD093727, and 1S10RR031537-01), and Priority Academic Program Development of Jiangsu Higher Education Institutions.</p><p class="pptt" id="biaoti27954">### Competing interests</p><p class="ppzz">The authors have no competing interests to declare.</p><p class="pptt" id="biaoti27955">### References</p><ol class="npwtwok"><li>Berggard, T., Linse, S. and James, P. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17640003" target="_blank">Methods for the detection and analysis of protein-protein interactions.</a> <em>Proteomics</em> 7(16): 2833-2842.</li><li>Chen, S., Cao, P., Dong, N., Peng, J., Zhang, C., Wang, H., Zhou, T., Yang, J., Zhang, Y., Martelli, E. E., Naga Prasad, S. V., Miller, R. E., Malfait, A. M., Zhou, Y. and Wu, Q. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26259032" target="_blank">PCSK6-mediated corin activation is essential for normal blood pressure.</a> <em>Nat Med</em> 21(9): 1048-1053. </li><li>Chen, S., Wang, H., Li, H., Zhang, Y. and Wu, Q. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29180304" target="_blank">Functional analysis of corin protein domains required for PCSK6-mediated activation.</a> <em>Int J Biochem Cell Biol</em> 94: 31-39. </li><li>Dong, N., Fang, C., Jiang, Y., Zhou, T., Liu, M., Zhou, J., Shen, J., Fukuda, K., Qin, J. and Wu, Q. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23372161" target="_blank">Corin mutation R539C from hypertensive patients impairs zymogen activation and generates an inactive alternative ectodomain fragment.</a> <em>J Biol Chem</em> 288(11): 7867-7874. </li><li>Dong, N., Zhou, T., Zhang, Y., Liu, M., Li, H., Huang, X., Liu, Z., Wu, Y., Fukuda, K., Qin, J. and Wu, Q. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24828501" target="_blank">Corin mutations K317E and S472G from preeclamptic patients alter zymogen activation and cell surface targeting.</a> [Corrected]. <em>J Biol Chem</em> 289(25): 17909-17916. </li><li>Ellgaard, L. and Frickel, E. M. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14716078" target="_blank">Calnexin, calreticulin, and ERp57: teammates in glycoprotein folding.</a> <em>Cell Biochem Biophys</em> 39(3): 223-247. </li><li>Gladysheva, I. P., King, S. M. and Houng, A. K. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18549807" target="_blank">N-glycosylation modulates the cell-surface expression and catalytic activity of corin.</a> <em>Biochem Biophys Res Commun</em> 373(1): 130-135. </li><li>Hooper, J. D., Scarman, A. L., Clarke, B. E., Normyle, J. F. and Antalis, T. M. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11082206" target="_blank">Localization of the mosaic transmembrane serine protease corin to heart myocytes.</a> <em>Eur J Biochem</em> 267(23): 6931-6937. </li><li>Kaboord, B. and Perr, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18369874" target="_blank">Isolation of proteins and protein complexes by immunoprecipitation.</a> <em>Methods Mol Biol</em> 424: 349-364. </li><li>Knappe, S., Wu, F., Masikat, M. R., Morser, J. and Wu, Q. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14559895" target="_blank">Functional analysis of the transmembrane domain and activation cleavage of human corin: design and characterization of a soluble corin. </a><em>J Biol Chem</em> 278(52): 52363-52370. </li><li>Kuhn, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27030537" target="_blank">Molecular physiology of membrane guanylyl cyclase receptors.</a> <em>Physiol Rev</em> 96(2): 751-804. </li><li>Lamriben, L., Graham, J. B., Adams, B. M. and Hebert, D. N. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26676362" target="_blank">N-Glycan-based ER molecular chaperone and protein quality control system: The calnexin binding cycle.</a> <em>Traffic</em> 17(4): 308-326. </li><li>Liao, X., Wang, W., Chen, S. and Wu, Q. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17660514" target="_blank">Role of glycosylation in corin zymogen activation.</a><em> J Biol Chem</em> 282(38): 27728-27735. </li><li>Matsuo, A., Nagai-Okatani, C., Nishigori, M., Kangawa, K. and Minamino, N. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30120963" target="_blank">Natriuretic peptides in human heart: Novel insight into their molecular forms, functions, and diagnostic use.</a> <em>Peptides</em> 111: 3-17. </li><li>Mattson, G., Conklin, E., Desai, S., Nielander, G., Savage, M. D. and Morgensen, S. (1993). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8326953" target="_blank">A practical approach to crosslinking.</a> <em>Mol Biol Rep</em> 17(3): 167-183. </li><li>Wang, H., Li, S., Wang, J., Chen, S., Sun, X. L. and Wu, Q. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29889025" target="_blank">N-glycosylation in the protease domain of trypsin-like serine proteases mediates calnexin-assisted protein folding.</a> <em>Elife</em> 7: e35672. </li><li>Wang, H., Zhou, T., Peng, J., Xu, P., Dong, N., Chen, S. and Wu, Q. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25451932" target="_blank">Distinct roles of N-glycosylation at different sites of corin in cell membrane targeting and ectodomain shedding.</a><em> J Biol Chem</em> 290(3): 1654-1663. </li><li>Wang, W., Cui, Y., Shen, J., Jiang, J., Chen, S., Peng, J. and Wu, Q. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22987923" target="_blank">Salt-sensitive hypertension and cardiac hypertrophy in transgenic mice expressing a corin variant identified in blacks.</a> <em>Hypertension</em> 60(5): 1352-1358. </li><li>Yan, W., Sheng, N., Seto, M., Morser, J. and Wu, Q. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10329693" target="_blank">Corin, a mosaic transmembrane serine protease encoded by a novel cDNA from human heart.</a> <em>J Biol Chem</em> 274(21): 14926-14935. </li><li>Yan, W., Wu, F., Morser, J. and Wu, Q. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10880574" target="_blank">Corin, a transmembrane cardiac serine protease, acts as a pro-atrial natriuretic peptide-converting enzyme.</a> <em>Proc Natl Acad Sci U S A</em> 97(15): 8525-8529. </li><li>Zhang, Y., Li, H., Zhou, J., Wang, A., Yang, J., Wang, C., Liu, M., Zhou, T., Zhu, L., Zhang, Y., Dong, N. and Wu, Q. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25488193" target="_blank">A corin variant identified in hypertensive patients that alters cytoplasmic tail and reduces cell surface expression and activity.</a> <em>Sci Rep</em> 4: 7378. </li><li>Zhang, Y., Zhou, J., Fukuda, K., Qin, J., Dong, N. and Wu, Q. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28861913" target="_blank">Identification and functional analysis of CORIN variants in hypertensive patients.</a> <em>Hum Mutat</em> 38(12): 1700-1710. </li><li>Zhou, Y. and Wu, Q. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24407448" target="_blank">Corin in natriuretic peptide processing and hypertension.</a> <em>Curr Hypertens Rep</em> 16(2): 415. <br/></li></ol>