<body>
            <h1>A Detailed Protocol for Large-scale Recombinant Expression and Validation of Human FGFR2 with Its Transmembrane and Extracellular Domains in <em>Escherichia coli</em></h1>
            <p><strong>Authors:</strong> Adam Bajinting, Ho Leung Ng</p>
            <h2>Abstract</h2>
            <p>Receptor tyrosine kinases (RTKs) are an important class of transmembrane receptors that mediate some of the most crucial biochemical pathways essential to the growth, differentiation, and survival of a cell and thus, are highly involved in cancers. Due to the complexity of RTKs having biochemically different domains including a transmembrane domain, an intact crystal structure of any of these proteins remain elusive as it is difficult to produce milligram amounts of intact functional RTKs for crystallography studies. A heavily studied RTK is fibroblast growth factor receptor 2 (FGFR2), which plays a key role in fibroblast growth regulation, differentiation, and oncogenesis. Previous studies have focused on expressing FGFR2âs extracellular, transmembrane, and intracellular domains individually. For this protocol, we have focused on the extracellular and transmembrane domains of the FGFR2 protein. The function of the expressed protein is validated by demonstrating its ability to bind heparin and fibroblast growth factor 1 (FGF1). The primary contribution of our protocol is expressing two RTK domains together, including the transmembrane domain, in milligram quantities. Being able to express RTKs to define its crystal structures would enable pharmacologists to design cancer drugs that selectively target active conformations.</p>
            <p><strong>Keywords:</strong> Receptor tyrosine kinase, FGFR, Extracellular, Transmembrane, Recombinant expression, Fibroblast growth factor, Fibroblast growth factor receptor, Membrane proteins, FGFR2, FGF1, FGF, Kinases</p>
            <h2>Background</h2><p>Receptor tyrosine kinases have eluded scientists in the quest for an intact fully functional crystal structure. In general, transmembrane proteins have been highly challenging to conduct structural studies especially due to challenges with expressing large quantities of fully functional protein. While most studies of FGFR expression focus on crystallizing the extracellular receptor or tyrosine kinase domain separately, working with the transmembrane domain linked to either of the two domains can result in multiple biochemical challenges (Mohammadi <em>et al.</em>, 1996; Plotnikov <em>et al.</em>, 1999; Plotnikov <em>et al.</em>, 2000; Schlessinger <em>et al.</em>, 2000; Yeh <em>et al.</em>, 2002; Zhang <em>et al.</em>, 2009). These eukaryotic transmembrane proteins, when expressed in high quantities in <em>E. coli</em> are usually highly unstable and often precipitate out of solution or aggregate (Rosano and Ceccarelli, 2014b). Moreover, RTKs have three functionally and structurally distinct domains, making it difficult to optimize expression because each domain may function optimally in different conditions. Detergent must always be used to keep the protein soluble due to the hydrophobic nature of the transmembrane domain (Rosano and Ceccarelli, 2014a). Host cells heavily regulate the expression of protein kinases due to their roles in activating important pathways. As a result, overexpression of these proteins to produce significant quantities may be toxic to leading to inclusion bodies and degradation. We chose FGFR2, a receptor responsible for activating complex mechanisms that result in angiogenesis, skeletal formation, and cell proliferation, and growth, as the best candidate for structural studies based on preliminary small-scale expression experiments (Turner and Grose, 2011).<br/> <br/>A primary goal of expressing functional FGFR in significant quantities (milligrams) is for X-ray crystallography. Two to fifty milligrams of protein is typically necessary for X-ray crystallography (Dessau, and Modis, 2011). A crystal structure would greatly facilitate the design of that selectively target the receptor. While there are studies that have determined the crystal structure of the FGFR2 ECD, TM, and intracellular kinase domains separately, they do not tell us the mechanism by which the external signal is transduced to the intracellular kinase domain. While the approach described in this paper does not describe expression of the entire protein, it does provide an important starting point by describing the expression of two linked domains including the technically challenging transmembrane domain. Recently, there have been a few studies that have demonstrate successful recombinant expression of EGFR, EphA2, PDGFR, and Kit (Mi <em>et al.</em>, 2008 and 2011; Lu <em>et al.</em>, 2012; Paavilainen <em>et al.</em>, 2013; Opatowsky <em>et al.</em>, 2014; Chen <em>et al.</em>, 2015). However, none of these studies employed <em>E. coli</em> as a host and described production of protein quantities sufficient for X-ray crystallography studies. We hope our methods will contribute to the ongoing effort towards crystallizing the structure of an intact receptor tyrosine kinase.<br/><br/>The determination of the crystal structure for intact FGFR2 would greatly facilitate drug discovery. FGFR drugs may potentially be used to treat a multitude of cancers including breast, stomach, lung, bladder, osteosarcoma, and intrahepatic cholangiocarcinoma (Babina and Turner, 2017). In addition, there is the potential for pharmacological in-utero treatment of FGFR2 associated craniosynostosis syndromes such as Apert Syndrome, Pfieiffer syndrome, and Bear-Stevenson syndrome (Robin <em>et al.</em>, 2011). Overall, crystallizing an intact FGFR2 may also set a precedent for the crystallization of intact receptor tyrosine kinases, including other FGFRs, which are key drug targets. <br/></p>
        </body>

<p class="pptt" id="biaoti27978">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (VWR, Next Generation Low-Binding, catalog numbers: 89076-476, 89076-486)</li><li>Snakeskin<sup>TM</sup> membrane (Thermo Fisher Scientific, catalog numbers: 68035, 88244)</li><li>Superdex S200 10/300 GL size exclusion chromatography column (GE Healthcare)</li><li>HiTrap Heparin Affinity Chromatography Column (GE Healthcare, catalog number: 17040601)</li><li>SnakeSkin<sup>TM</sup> Dialysis Clips (Thermo Fisher Scientific, catalog number: 68011)</li><li>PVDF membrane (Thermo Fisher Scientific, catalog number: 88518)</li><li>1.5 ml centrifuge tubes (Fisher Scientific, catalog number: 05-408-129)</li><li>3 ml syringes, Luer lock (Becton Dickinson, catalog number: BD309657)</li><li>Petri Dishes for LB Plates, 100 x 10 mm (VWR, catalog number: 25384-324)</li><li>pSpeedET vector, DNASU plasmid repository (EvNO00285791)<br/><em>Note: pSpeedET must have N-terminal </em><em>E. coli</em><em> maltose binding protein (MBP) fusion tag.</em> </li><li>Lemo21 (DE3) Competent <em>E. coli</em> (New England Biolabs<sup>Â®</sup> Inc., catalog number: 2528J)</li><li>Primers<br/>31F: 5â ctgtacttccagggcACCACATTAGAGCCAGAAGGAGCAC 3â<br/>406R_noGFP: 5â gtttaattaagtcgcgtta CTTCTTGGTCGTGTTCTTCATTCGGC 3â</li><li>Phusion<sup>Â®</sup> High-Fidelity PCR Kit (New England Biolabs<sup>Â®</sup> Inc., catalog number: C2528J)</li><li>ThermoScientific GeneJET Gel Extraction Kit (Thermo Fisher Scientific<sup>TM</sup>, catalog number: K0691)</li><li>General Electric Healthcare illustra<sup>TM</sup> plasmidPrep Mini Spin Kit (GE Healthcare, catalog number: GE28-9042-69)</li><li>Dimethylsulfoxide (DMSO) (Sigma-Aldrich, catalog number: D650-5X5 ML)</li><li>Agarose (Lonza SeaKem LE, catalog number: 50004)</li><li>SOC recovery media (Thermo Fisher Scientific, catalog number: 15544034)</li><li>Kanamycin (Research Productions International, catalog number: K22000)</li><li>Chloramphenicol (Research Productions International, catalog number: C61000)</li><li>Carbenicillin (Research Productions International, catalog number: C46000)</li><li>Terrific broth (TB) (Sigma Aldrich, catalog number: T0918)</li><li>Beta-D-1-thiogalactopyranoside (IPTG) (Sigma-Aldrich, catalog number: l6758-1G)</li><li>DNase I (New England Biolabs, catalog number: M0303S)</li><li>CaCl<sub>2</sub> (Fisher Scientific, catalog number: AC349610250)</li><li>Benzamidine (Sigma-Aldrich, catalog number: 449709)</li><li>PMSF (Sigma-Aldrich, catalog number: 10837091001)</li><li>Protease Inhibitors E-64 (Sigma-Aldrich, catalog number: E3132)</li><li>Pepstatin (Sigma-Aldrich, catalog number: P5318-5MG)</li><li>Bestatin (Sigma-Aldrich, catalog number: B8385-1MG)</li><li>Î²-mercaptoethanol (BME) (Sigma-Aldrich, catalog number: M6250)</li><li>e2TAK polymerase (Takara Bio, catalog number: RF001A)</li><li>Rapid Transfer Buffer, 10x (VWR<sup>Â®</sup> Amresco<sup>Â®</sup> Life Science, catalog number: N789-1L)</li><li>Anti-MBP Antibody (New England Biolabs, catalog number: E-8038)</li><li>Tris-glycine gels 8-16% (NuSep, catalog number: NG15-816) </li><li>n-dodecyl-Î²-D-maltoside DDM (Anatrace, catalog number: D310)</li><li>KPL TMB Membrane Peroxidase Substrate System Kit (Sera Care, catalog number: 5420-0025)</li><li>Amresco RapidBlock<sup>TM</sup> solution (VWR, catalog number: 97064-124)</li><li>NaCl (Sigma-Aldrich, catalog number: S9888)</li><li>HEPES at pH 7.5 (VWR, catalog number: 97061-822)</li><li>MgSO<sub>4</sub> (Sigma-Aldrich, catalog number: M7506)</li><li>Glycerol (Sigma-Aldrich, catalog number: G9012)</li><li>Tris-HCl pH 7.5 (Sigma-Aldrich, catalog number: 10812846001)</li><li>Tween 20 (Sigma-Aldrich, catalog number: P9416)</li><li>Guanidinium-HCl (Sigma-Aldrich, catalog number: G3272)</li><li>Dithiothreitol (DTT) (Sigma Aldrich, catalog number: DTT-RO)</li><li>L-cysteine (Sigma-Aldrich, catalog number: 168149)</li><li>Imidazole (Alfa Aesar, catalog number: AA47274)</li><li>Gel Green<sup>Â®</sup> (Biotium, catalog number: 41005)</li><li>LB agar (VWR, catalog number: 97064-106)</li><li>NEB<sup>Â®</sup> 5-alpha Competent <em>E. coli</em> (High Efficiency) DH5Î± cells (New England Biolabs, catalog number: C2987l)</li><li>Coomassie Brilliant Blue (Thermo Fisher Scientific, catalog number: 20279)</li><li>FGF1 pMCSG7 plasmid (DNASU plasmid repository, catalog number: HsCD00343156)</li><li>HRP-conjugated anti-MBP antibody (New England Biolabs, catalog number: E8038)</li><li>Milk Powder </li><li>Lysis Buffer (see Recipes) </li><li>TBST (see Recipes)</li><li>Denaturing buffer dialysis solution #1 (see Recipes)</li><li>Dialysis Solution #2 (see Recipes)</li><li>Binding buffer for heparin affinity chromatography (see Recipes)</li><li>Elution buffer for heparin affinity chromatography (see Recipes)</li><li>Elution buffer for FGF1 (see Recipes)</li><li>Running Buffer for SEC (see Recipes)</li><li>Blocking Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti27979">### Equipment</p><ol class="npwtwok"><li>Baffled, 125 ml Erlenmeyer flasks (Sigma-Aldrich, catalog number: CLS4444125)</li><li>Pipettes (Gilson, Pipetman Classic)</li><li>Centrifuge for collecting liters of cell culture (Thermo Fisher, Sorvall, catalog number: ST16) </li><li>Tabletop microcentrifuge (Eppendorf, model: 5415)</li><li>PCR Thermal cycler (Thermo Scientific, model: Hybaid PX2 Gradient Thermal Cycler)</li><li>Sonicator (Thermo Fisher, model: 550 Sonic Dismembrator)</li><li>Akta Purifier or FPLC (GE Healthcare, <a class="blackNoUnder" target="_blank"> AKTA Purifier 100 </a> FPLC, Frac-950)</li><li>Refrigerated shaker/incubator for growing cells (New Brunswick, Innova 4230 Refrigerated Incubator)</li><li>Vortexer (Fisher Scientific, catalog number: 02215365)</li><li>JA20 Fixed-Angle Aluminum rotor Biosafety Lid- 8 x 50 ml, 20,000 rpm, 48,400 <em>x g</em> (Beckman Coulter, catalog number: 334831)</li><li>Magnetic stir bar (Fisher Scientific, catalog numbers: 14-513-83, 14-513-93, 14-513-95, 14-513-94)</li><li>Peristaltic pump (Gilson, Rabbit Plus)</li><li>Visible/UV spectrophotometer (Cary 50)</li><li>Test tube rotator (Scientific Equipment, model: 60448 Test Tube Rotator)</li><li>Western Blot apparatus (Bio-Rad)</li><li>Clare Dark Reader Transilluminator (Clare Chemicals, DR46)<br/></li></ol><p class="pptt" id="biaoti27980">### Software</p><ol class="npwtwok"><li>ApE (A plasmid Editor) program for nucleotide analysis (<a href="http://jorgensen.biology.utah.edu/wayned/ape/" target="_blank">http://jorgensen.biology.utah.edu/wayned/ape/</a>)</li><li>NCBI BLASTn for sequence comparison (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=blast2seq&amp;LINK_LOC=align2seq" target="_blank">https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE_TYPE=BlastSearch&amp;BLAST_SPEC=blast2seq&amp;LINK_LOC=align2seq</a>)<br/></li></ol><p class="pptt" id="biaoti27981">### Procedure</p><p class="ppzz"><strong>Part I: Large Scale Expression and Purification of FGFR2âs Extracellular and Transmembrane Domains<br/><br/></strong> </p><ol class="npwone" start="1"><li>Synthesizing FGFR2 ECD + TM construct via polymerase incomplete primer extension (PIPE) cloning<br/>Polymerase Incomplete Primer Extension (PIPE) cloning is a more efficient alternative to cloning with restriction and ligation enzymes. Instead of adding flanking restriction sites and using restriction enzymes to digest the sites followed by phosphorylation and ligation, PIPE cloning uses ligation independent cloning from annealing overhanging PCR ends and requires less time and components. Overhangs are added on to the sequences by using one half of the primers for the FGFR sequence and another half for the vector. The single-stranded overhangs that are now flanking the FGFR sequences are complementary to the vector sequence. As a result, by combining the two with subsequent transformation, a recombinant plasmid is formed (Klock and Lesley, 2009). <br/><br/><ol class="npwtwo"><li>PCR<ol class="npwthree"><li>Prepare the PCR reaction according to the instructions specified by the Phusion<sup>Â®</sup> High-Fidelity PCR kit using primers for generating the FGFR2 ECD + TM (31-406) construct.</li><li>Add 6% DMSO to each reaction along with the proper PCR components for 25 Î¼l reactions for the FGFR2 and vector pSpeedET + Maltose Binding Protein (MBP).</li><li>PCR Reaction components for a 25 Î¼l reaction are: <br/>FGFR/pSpeedET + MBP vector template: 20 ng each<br/>Phusion Hi Fidelity buffer: 5 Î¼l<br/>Phusion Hi Fidelity DNA polymerase: 0.25 Î¼l<br/>0.2 Î¼M dNTP (0.5 Î¼l of 10 mM dNTP)<br/>0.5 Î¼M forward and reverse primers (1.25 Î¼l of 10 Î¼M stocks)<br/>6% DMSO: 1.5 Î¼l of DMSO<br/>Type 1 nuclease-free water: add to a total volume of 25 Î¼l</li><li>Adjust the settings on the thermocycler to: <br/>98 Â°C for 6 min;<br/>Followed by a cycle of 35 times with the settings 98 Â°C for 30 s; <br/>50 Â°C for 30 s;<br/>72 Â°C for 2 min;<br/>Finally 72 Â°C for 10 min followed by a 4 Â°C holding period. </li><li>Run the PCR product on a 1% agarose gel. </li><li>Extract the PCR products of expected lengths using the Thermo Scientific GeneJET<sup>Â®</sup> Gel Extraction Kit. </li><li>Expected Lengths are:<br/><table border="1" bordercolor="#000000" cellpadding="2" cellspacing="0" style="width:400px;"><tbody><tr><td>Sequence in question<br/></td><td>Expected size (bp)<br/></td></tr><tr><td>FGFR2 31-406 (ECD + TM)<br/></td><td>783<br/></td></tr><tr><td>pSpeedET + MBP<br/></td><td>6642<br/></td></tr></tbody></table>FGFR2 Primers: PIPE overhangs are in lower case. Sequences matching FGFR2 are in upper case. <br/>pSpeedET-R2: cctcgagtgaagaacc TTCCTCATTGGTTGTGAGAGTGAGAATTCG; <br/>MBP-F: ctgtacttccagggc ACCACATTAGAGCCAGAAGGAGCAC<br/><br/></li></ol></li></ol>For the PIPE annealing reaction, use 20 ng of vector with 20 ng of purified PCR product containing the FGFR2 ECD + TM sequence in a total of 4 Î¼l for 10 min at room temperature. During this time, the overhangs of the FGFR sequences and the vector (with MBP) will anneal together creating a recombinant plasmid with the FGFR sequence and maltose binding protein. MBP was used as a fusion tag to increase the solubility of the construct and for protein identification during Western blot analysis. Because the FGFR sequences contain the hydrophobic transmembrane domain and the goal was to preserve the integrity of the transmembrane domain throughout expression studies, it was essential to use MBP (Figures 1 and 2).<br/><br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190602/20190602192732_8205.jpg"/><br/><strong>Figure 1. FGFR construct in pSpeedET plasmid.</strong> The TEV protease cleavage site is in the overhang for the forward primers for the FGFR2 construct.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190602/20190602192803_0594.jpg" width="227"/><br/><strong>Figure 2. Process of polymerase incomplete primer extension (PIPE) cloning</strong><br/><br/><ol class="npwtwo" start="2"><li>Transformation for cloningâImmediately after the PIPE annealing, transfer into <em>E. coli</em> DH5Î± cells for cloning<ol class="npwthree"><li>Thaw 1 vial of DH5Î± competent cells on ice. </li><li>Add 2 Î¼l of the PIPE reaction to the cells and incubate on ice for 10 min. </li><li>Heat shock the cells at 42 Â°C for 30 s and place the cells back on ice for 2 min. </li><li>Add 500 Î¼l of SOC recovery media (at RT) to the cells and place them in a 37 Â°C shaker for one hour. </li><li>Spread 100 Î¼l of the transformation on LB + kanamycin plates and incubate them at 37 Â°C overnight. Kanamycin final concentration is at 50 Î¼g/ml.</li></ol></li><li>Colony polymerase chain reaction, DNA extraction, and sequencingâVerify success of PIPE cloning by size and sequence using colony PCR and sequencing<ol class="npwthree"><li>Use a pipette tip to pick a colony and resuspend it in 13.7 Î¼l of nuclease-free water (the volume of water needed for a 25 Î¼l PCR reaction).</li><li>Add 13.7 Î¼l water, 2.5 Î¼l of 2.5 mM dNTPs, 2.5 Î¼l of 2 mM forward primer, 2.5 Î¼l of 2 Î¼M reverse primer, 0.3 Î¼l e2TAK polymerase (Takara Bio), and 5 Î¼l polymerase 5x Buffer. The thermocycler conditions are as follows: 94 Â°C for 6 min followed by 35 cycles of 94 Â°C for 30 s, 50 Â°C for 30 s, and 72 Â°C for 2 min, and finally 72 Â°C for 10 min followed by a 4 Â°C holding period.</li><li>Run on a 1% agarose gel to confirm the size of the expected insert.</li><li>Grow the selected colonies in 3 ml LB + kanamycin overnight at 37 Â°C in a shaker at 230 rpm in 5 ml culture tubes.</li><li>Isolate the recombinant plasmid using the General Electric Healthcare illustra plasmidPrep Mini Spin Kit.</li><li>Determine the presence and accuracy of the construct by sequencing.</li><li>Verify the sequence is correct by analysis with ApE program for nucleotide analysis and BLASTn comparison.<br/><br/></li></ol></li></ol></li><li>Large scale expression of the FGFR2 construct<ol class="npwtwo"><li>Transformation and inoculation for protein expression in Lemo21 DE3 cells<ol class="npwthree"><li>Using the protocol as specified from above (Step A2), positively select using LB + kanamycin agar plates.</li><li>Chose a colony and grow overnight between 12 h and 16 h in 100 ml of LB with 50 Î¼g/ml kanamycin + 25 Î¼g/ml chloramphenicol. </li><li>After 16 h, prepare the inoculation: 100 ml overnight culture, 100 Î¼g/ml kanamycin, 50 Î¼g/ml chloramphenicol, 0.5 mM L-rhamnose, and 2 mM of MgSO<sub>4</sub>. <br/><em>Note: The concentrations of the reagents are based on a volume of 1,000 ml of Terrific Broth (TB). </em> </li><li>Place the inoculated TB culture and antibiotics into the shaker at 37 Â°C. Monitor the OD<sub>600</sub> as it approaches 0.6. Once the culture reaches an OD<sub>600</sub> of 0.4-0.5, cool the cells to 18 Â°C in the shaker to slow the growth of cells. <br/><em>Note: OD should be checked every 10 min. The bacterial culture reaches OD = 0.6 after 30-45 min.</em> </li><li>Take 1 ml of each construct culture to serve as a negative control for later experiments. In addition, take a glycerol stock (50 Î¼l of 50% glycerol in ddH<sub>2</sub>O and 50 Î¼l of culture) for archival purposes. Store the glycerol stock at -80 Â°C. The glycerol should be sterile.</li><li>Add isopropyl beta-D-1-thiogalactopyranoside (IPTG) at a 0.1 mM final concentration to each culture to induce expression.</li><li>Incubate the cells in the shaker at 18 Â°C overnight at 230 rpm.</li><li>The next day, harvest the cells as followed: centrifuge the culture at 4 Â°C at 4,000 <em>x g</em> for 10 min.</li><li>Discard the supernatant and wash the pellet by resuspending it in 100 ml 4 Â°C lysis buffer (Recipe 1). </li><li>Centrifuge again at 4,000 <em>x g</em> for 10 min and discard the lysis buffer. </li></ol></li><li>Lysing cells<ol class="npwthree"><li>Resuspend the post induction Lemo21 cell in 40 ml lysis buffer by vortexing and pipetting up and down to ensure homogeneity in a 50 ml conical tube. This must all be done on ice.</li><li>Add 1 Î¼l of 2000 U/ml DNase I, 1 Î¼M CaCl<sub>2</sub>, and protease inhibitors (1 mM benzamidine, 1 mM PMSF, 1 Î¼M E-64, 0.5 mM TCEP, 1 Î¼M pepstatin, and 1 Î¼M bestatin) to lysis buffer prior to lysis by sonication. </li><li>Sonicate the cells. Sonication conditions: 6 cycles; 1 cycle: 20 s sonication ON followed by 40 s OFF.</li><li>Following sonication, place the suspension in a heavy-duty centrifuge tube and centrifuge at 48,384 <em>x g</em> using a JA20 rotor for 30 min. Separate and store both supernatant and cell pellet at -80 Â°C.<br/><em>Note: Optional stopping point. Ideally, better if the procedure is continued.</em> <br/><br/></li></ol></li></ol></li><li>Detergent extraction<ol class="npwtwo"><li>Prepare a lysis buffer that includes 1% n-dodecyl Î²-D-maltoside (DDM) detergent.</li><li>For every 100 Î¼g of cell pellet or 100 Î¼l of supernatant, add 500 Î¼l of 1% detergent solution and resuspend along with PMSF.</li><li>Invert the suspension continuously for 2 h at 4 Â°C.</li><li>Centrifuge the suspension at 23,708 <em>x g</em> for 15 min at 4 Â°C.</li><li>Separate the supernatant and the cell pellet and store both at -80 Â°C. (Both the cell pellet and supernatant should possess FGFR2, but the cell pellet should contain more).<br/><br/></li></ol></li><li>Running samples through NuSep 8-16% Tris-Glycine Gels via polyacrylamide gel electrophoresis (PAGE) to determine if the protein of interest is present<ol class="npwtwo"><li>Resuspend the pellet in 300 Î¼l MilliQ water and 100 Î¼l of 4x loading dye with Î²-mercaptoethanol (BME) while 100 Î¼l of the same loading dye was added to the 300 Î¼l supernatant/portion of cell pellet. </li><li>Centrifuge at 23,708 <em>x g</em> for 10 min in preparation for loading into a polyacrylamide gel. <br/><em>Note: Steps C1 and C2 are to ensure that the loaded sample is not too viscous for electrophoresis.</em> </li><li>Load the samples (20 Î¼l per well) along with a ladder and run at 200 V for about 45 min.</li><li>Run 2 gels so that one can be stained and the other can be used for Western blotting. <br/><br/></li></ol></li><li>Western blotting and antibody detection using anti-MBP monoclonal antibody HRP conjugated<ol class="npwtwo"><li>Western blot the gel of interest using the VWR Amresco<sup>Â®</sup> Buffers.</li><li>Soak the Tris-glycine gel in Amresco rapid transfer buffer for 10 min. Use just enough buffer to cover the entire gel. </li><li>Soak the PVDF membrane for 30 s in methanol, then subsequently soak in water for 5 min, then finally soak in rapid transfer buffer until it no longer floats. Use just enough solution to cover the membrane.</li><li>Assemble the Western block stack according to Amrescoâs wet transfer protocol.</li><li>Place the stack in the gel electrophoresis chamber with Amresco rapid transfer buffer and run at 75 V for 20 min. The rapid transfer buffer must be filled to the top of the chamber so that the stack is completely submerged in the buffer.</li><li>Disassemble the stack and incubate the membrane in Amresco RapidBlock<sup>TM</sup> solution for 5 min.</li><li>Incubate the PVDF membrane in 20 ml of blocking buffer for 1 h at room temperature. </li><li>Incubate the PVDF Membrane in 20 ml of blocking buffer with 10 Î¼l of anti-MBP monoclonal antibody (HRP Conjugated) from New England Biolabs overnight at 4 Â°C. </li><li>The next day, use 25 ml of TBST (Recipe 2) to wash the PVDF membrane three times for 5 min each. </li><li>Detect the presence of the protein using the KPL TMB Membrane Peroxidase Substrate System Kit. The volumes needed for the reaction to occur are listed in the kitâs instructions.<br/>  <br/></li></ol></li><li>Refolding FGFR2 by dialysis<ol class="npwtwo"><li>Washing inclusion bodies<br/>Resuspend the pellets in 0.5% guanidinium-HCl (with just enough volume to get the pellets resuspended homogenously), and subsequently centrifuge at 4,000 <em>x g</em> for 20 min at 4 Â°C. After washing, the inclusion bodies should have a colorless appearance.</li><li>Solubilization and denaturation of FGFR recombinant protein<ol class="npwthree"><li>Denature and solubilize the original inclusion bodies in 10 ml dialysis solution #1 (Recipe 3).</li><li>To facilitate solubilization, warm the cell pellet and dialysis solution mixture to 40 Â°C temporarily in a water bath followed by gentle vortexing. The solution should be colorless and clear. Incubate the mixture prior to dialysis for 30-40 min at 40 Â°C. <span style="font-size:10pt;"> </span> </li></ol></li><li>Assembly of apparatus and incubation<ol class="npwthree"><li>Load the solubilized inclusion bodies in the dialysis solution #1 into a snakeskin membrane with clips.</li><li>Suspend the filled membrane in a container with 700 ml dialysis solution #2 (Recipe 4).</li><li>Place the apparatus in the cold room (4 Â°C) with gentle constant stirring using a magnetic stir bar overnight.</li><li>The next day, place the sample from the snakeskin into heavy-duty centrifuge tubes and centrifuge at 48,384 <em>x g</em> for 30 min. Separate the pellet and supernatant and store at -80 Â°C. The protein of interest is in the supernatant.</li></ol></li></ol><br/></li><li>Purification and functionality testing via heparin binding chromatography<br/><ol class="npwtwo"><li>To test isolate and begin testing the functionality of FGFR2, use the 1 ml HiTrap<sup>TM</sup> Heparin Affinity Chromatography column (GE Healthcare) with a syringe pump.</li></ol><br/><em>Note: GE Healthcare provides instructions for how to use the column. The following is additional details for this procedure.</em><br/><ol class="npwtwo" start="2"><li>Load 10 column volumes (CVs) of binding buffer (Recipe 5) onto the column with a syringe to equilibrate the column at a rate of 1 ml/min. 1 CV = 1 ml</li><li>Syringe filter the FGFR2 supernatant from dialysis to prevent clogging and load it onto the column. <br/><em>Note: Make sure no air bubbles get into the column.</em> </li><li>Load the FGFR2 Supernatant unto the column.</li><li>Wash the column of any debris with 5 CVs of binding buffer. </li><li>Run 10 CVs of elution buffer (Recipe 6). Use sterile 1.5 ml centrifuge tubes to collect all flow through, wash, and elution fractions. The rate of loading and running is at 1 ml/min. The volume of each fraction should be 1 ml. 1 CV = 1 ml.</li><li>Run a PAGE and Western Blot analysis to confirm FGFR2 isolation. For Western blotting, use the anti-MBP HRP-conjugated antibody from New England Biolabs to detect FGFR2 (Figure 3).</li><li>Run size exclusion chromatography (SEC) of the purified FGFR2 combined with FGF1, heparin, and FGFR2 to verify dimerization (Figure 4). Follow by Western blotting of the SEC elution fractions with the anti-MBP antibody to identify the size of the eluted complex.</li></ol><em>Notes:</em> <ol class="npwthree"><li><em>The first step in testing functionality is to determine whether or not FGFR2 can bind to heparin and FGF1. This step is essential since heparin and FGF1 binding is a necessary step to ternary complex formation and dimerization. To do this, repeat Procedure F above along with loading 1 ml of FGF1âs elution from SEC, followed by 10 CVs (10 ml) of binding buffer between Steps F2 and F3.</em> </li><li><em>Critical notes have been included throughout the protocol at important steps. Overall, this is a lengthy procedure with few stopping points. As an additional note, we have included how we express, isolate and purify FGF1 to test for FGFR2 functionality.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190602/20190602192829_6115.jpg" width="225"/><br/><strong>Figure 3. Western blot shows elution of FGFR2 + MBP Construct (red rectangle) from HiTrap heparin affinity chromatography demonstrating recombinant proteinâs ability to bind to heparin.</strong> Lane 1: Protein standard. Lanes 2-3: Wash fractions. Lanes 4-5: Elution fraction.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190602/20190602192857_0176.jpg" width="210"/><br/><strong>Figure 4. Size exclusion chromatography profile.</strong> Elution fractions are in red and range from A9-12, followed by B12-B7. Elution fractions A11 and A12 correspond to inactive FGFR2 containing the ECD + TM domains in dimers in micelles. Elution fraction B11 corresponds to FGFR2 containing the ECD + TM domains. Elution fractions B11-B8 corresponds to FGF1. Each elution fraction has a volume of 1 ml. <br/><br/></li></ol></li></ol><strong>Part II: FGF1 Expression and Purification</strong><br/><br/><ol class="npwone"><li>FGF1 expression<ol class="npwtwo"><li>Transform DH5Î± cells using the FGF1 expression vector in pMCSG7 from the DNASU plasmid repository. To clone it, use the same procedure as detailed under Step A2 but use carbenicillin instead of kanamycin as the selection marker. </li><li>Grow a colony in 3 ml of LB with 100 Î¼g/ml carbenicillin overnight. </li><li>Isolate DNA from the culture using the GE Healthcare illustra plasmidPrep Mini Spin Kit. </li><li>Confirm FGF1 sequence using T7 forward and reverse primers. </li><li>Large scale expression<ol class="npwthree"><li>Transform the FGF1 expression plasmid into competent <em>E. coli</em> Lemo21 (DE3) cells and positively select with carbenicillin. </li><li>Pick a colony and grow in 100 ml LB with chloramphenicol and carbenicillin overnight. </li><li>The next day, use this culture to inoculate 1,000 ml of TB with 100 Î¼g/ml carbenicillin, 50 Î¼g/ml chloramphenicol, 0.5 mM rhamnose, and 2 mM of MgSO<sub>4</sub> following Steps B1c above. </li><li>Continue to cell harvesting, cell lysis by sonication and ultra-centrifugation as discussed above in Step B2, all at 4 Â°C. The protein will be in the supernatant.</li></ol></li></ol><br/></li><li>FGF1 Purification<br/><em>Note: Purify the FGF1 using a 1 ml HiTrap</em><sup><em>TM</em></sup><em> Column (GE Healthcare) with a syringe.</em> <ol class="npwtwo"><li>Fill the syringe with 1 ml of binding buffer which is the same lysis buffer used for the large-scale expression of FGFR2. Making sure the air bubbles are out of the syringe, run the same lysis buffer through the column at a rate of 1 ml/min. </li><li>Before reaching the end of the syringe, it was refilled with binding buffer, and again pushed through the syringe. This was done with 9 more CVs to equilibrate the column. </li><li>Run the entire FGF1 sample (supernatant from ultracentrifugation) through the column using the syringe. </li><li>Wash column with 10 CVs (10 ml total) of lysis buffer to elute any contaminating proteins that may have non-specifically bound to FGF1 and heparin. </li><li>Elute FGF1 with elution buffer (Recipe 7).</li><li>Determine the success of purification by PAGE and Western blot analysis as discussed above under Procedure C and Procedure D, but instead of using an anti-MBP antibody (HRP conjugated), an anti-His antibody (HRP conjugated) was used from Pierce. </li><li>Use size exclusion chromatography (SEC) to further purify the protein. Load 2 ml of the elution onto the column (this was the maximum volume allowed and its concentration was determined to be acceptable to subsequent functionality experiments with FGFR: 1.756 Î¼g/ml). The running buffer (Recipe 8) used for size exclusion chromatography (SEC) was 25 mM HEPES, 0.1% DDM, and 150 mM NaCl at pH 7.5.</li></ol></li></ol><p class="pptt" id="biaoti27982">### Recipes</p><ol class="npwtwok"><li>Lysis Buffer<br/>300 mM NaCl<br/>50 mM HEPES at pH 7.5<br/>0.1 mM MgSO<sub>4</sub><br/>5% glycerol</li><li>TBST<br/>20 mM Tris-HCl pH 7.5<br/>150 mM NaCl<br/>0.1% Tween 20</li><li>Denaturing buffer dialysis solution #1<br/>6 M Guanidinium-HCl<br/>0.1% DDM<br/>10 mM DTT<br/>1 Î¼M Protease inhibitors E-64<br/>1 mM benzamidine<br/>1 mM PMSF<br/>1 Î¼M bestatin<br/>1 Î¼M pepstatin at pH 8.0</li><li>Dialysis Solution #2<br/>25 mM HEPES<br/>150 mM NaCl<br/>10% glycerol<br/>1 mM L-cysteine at pH 7.5</li><li>Binding buffer for heparin affinity chromatography<br/>150 mM NaCl<br/>25 mM HEPES<br/>1 mM benzamidine<br/>0.1% DDM at pH 7.5</li><li>Elution buffer for heparin affinity chromatography<br/>25 mM HEPES<br/>1.5 M NaCl<br/>0.1% DDM<br/>1 mM benzamidine<br/>1 mM PMSF at pH 7.5</li><li>Elution buffer for FGF1<br/>1.0 M NaCl<br/>10% glycerol<br/>25 mM HEPES<br/>10 mM imidazole<br/>1 mM benzamidine at pH 7.5</li><li>Running Buffer for SEC<br/>25 mM HEPES<br/>0.1% DDM<br/>150 mM NaCl at pH 7.5</li><li>Blocking Buffer<br/>5% Milk Powder<br/>TBST<br/></li></ol><p class="pptt" id="biaoti27983">### Acknowledgments</p><p class="ppzz">This project was funded by NSF CAREER Award 1350555 (HLN), the University of Hawaii at Manoa, and the University of Hawaii Undergraduate Research Opportunities Program (AB).</p><p class="pptt" id="biaoti27984">### Competing interests</p><p class="ppzz">AB: No competing interests to disclose. HLN: No competing interests to disclose.</p><p class="pptt" id="biaoti27985">### References</p><ol class="npwtwok"><li>Chen, P. H., Unger, V. and He, X. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26463591" target="_blank">Structure of full-length human PDGFRÎ² bound to its activating ligand PDGF-B as determined by negative-stain electron microscopy.</a> <em>J Mol Biol </em>427(24): 3921-3934.</li><li>Dessau, M. A. and Modis, Y. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21304455" target="_blank">Protein crystallization for X-ray crystallography.</a> <em>J Vis Exp</em>(47).</li><li>Klock, H. E. and Lesley, S. A. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18988020" target="_blank">The Polymerase Incomplete Primer Extension (PIPE) method applied to high-throughput cloning and site-directed mutagenesis.</a> <em>Methods Mol Biol</em> 498: 91-103.</li><li>Klock, H. E., Koesema, E. J., Knuth, M. W. and Lesley, S. A. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18004753" target="_blank">Combining the polymerase incomplete primer extension method for cloning and mutagenesis with microscreening to accelerate structural genomics efforts.</a> <em>Proteins</em> 71(2): 982-994. </li><li>Lu, C., Mi, L. Z., Schurpf, T., Walz, T. and Springer, T. A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22988250" target="_blank">Mechanisms for kinase-mediated dimerization of the epidermal growth factor receptor.</a> <em>J Biol Chem </em>287(45): 38244-38253.</li><li>Mi, L. Z., Grey, M. J., Nishida, N., Walz, T., Lu, C. and Springer, T. A. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18771282" target="_blank">Functional and structural stability of the epidermal growth factor receptor in detergent micelles and phospholipid nanodiscs.</a> <em>Biochemistry</em> 47(39): 10314-10323.</li><li>Mi, L. Z., Lu, C., Li, Z., Nishida, N., Walz, T. and Springer, T. A. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21822280" target="_blank">Simultaneous visualization of the extracellular and cytoplasmic domains of the epidermal growth factor receptor. </a><em>Nat Struct Mol Biol</em> 18(9): 984-989.</li><li>Mohammadi, M., Schlessinger, J. and Hubbard, S. R. (1996). <a href="https://www.ncbi.nlm.nih.gov/pubmed/8752212" target="_blank">Structure of the FGF receptor tyrosine kinase domain reveals a novel autoinhibitory mechanism.</a> <em>Cell</em> 86(4): 577-587.</li><li>Opatowsky, Y., Lax, I., Tome, F., Bleichert, F., Unger, V. M. and Schlessinger, J. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24449920" target="_blank">Structure, domain organization, and different conformational states of stem cell factor-induced intact KIT dimers.</a> <em>Proc Natl Acad Sci U S A </em>111(5): 1772-1777.</li><li>Paavilainen, S., Grandy, D., Karelehto, E., Chang, E., Susi, P., Erdjument-Bromage, H., Nikolov, D. and Himanen, J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24036371" target="_blank">High-level expression of a full-length Eph receptor.</a> <em>Protein Expr Purif</em> 92(1): 112-118.</li><li>Plotnikov, A. N., Hubbard, S. R., Schlessinger, J. and Mohammadi, M. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10830168" target="_blank">Crystal structures of two FGF-FGFR complexes reveal the determinants of ligand-receptor specificity.</a> <em>Cell</em> 101(4): 413-424.</li><li>Plotnikov, A. N., Schlessinger, J., Hubbard, S. R. and Mohammadi, M. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10490103" target="_blank">Structural basis for FGF receptor dimerization and activation.</a> <em>Cell</em> 98(5): 641-650.</li><li>Rosano, G. L. and Ceccarelli, E. A. (2014a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25071752" target="_blank">Recombinant protein expression in microbial systems.</a> <em>Front Microbiol</em> 5: 341.</li><li>Rosano, G. L. and Ceccarelli, E. A. (2014b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24860555" target="_blank">Recombinant protein expression in <em>Escherichia coli</em>: advances and challenges.</a> <em>Front Microbiol</em> 5: 172. </li><li>Schlessinger, J., Plotnikov, A. N., Ibrahimi, O. A., Eliseenkova, A. V., Yeh, B. K., Yayon, A., Linhardt, R. J. and Mohammadi, M. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11030354" target="_blank">Crystal structure of a ternary FGF-FGFR-heparin complex reveals a dual role for heparin in FGFR binding and dimerization.</a> <em>Mol Cell</em> 6(3): 743-750.</li><li>Turner, N. and Grose, R. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20094046" target="_blank">Fibroblast growth factor signalling: from development to cancer.</a> <em>Nat Rev Cancer</em> 10(2): 116-129.</li><li>Yeh, B. K., Eliseenkova, A. V., Plotnikov, A. N., Green, D., Pinnell, J., Polat, T., Gritli-Linde, A., Linhardt, R. J. and Mohammadi, M. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12242295" target="_blank">Structural basis for activation of fibroblast growth factor signaling by sucrose octasulfate.</a> <em>Mol Cell Biol</em> 22(20): 7184-7192.</li><li>Zhang, F., Zhang, Z., Lin, X., Beenken, A., Eliseenkova, A. V., Mohammadi, M. and Linhardt, R. J. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19591432" target="_blank">Compositional analysis of heparin/heparan sulfate interacting with fibroblast growth factorÂ·fibroblast growth factor receptor complexes.</a> <em>Biochemistry</em> 48(35): 8379-8386.</li><li>Babina, I. S. and Turner, N. C. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28303906" target="_blank">Advances and challenges in targeting FGFR signalling in cancer.</a> <em>Nat Rev Cancer</em> 17 (5):318-332. </li><li>Robin, N. H., Falk, M. J., Haldeman-Englert, C. R. (1993). <a href="https://www.ncbi.nlm.nih.gov/books/NBK1455/" target="_blank"><em>FGFR-Related Craniosynostosis Syndromes.</em></a> In: Adam, M. P., Ardinger, H. H., Pagon, R. A. <em>et al.</em> (eds) GeneReviews ((R)). Seattle (WA).<br/></li></ol>