<body>
            <h1>Imaging Gene Expression Dynamics in <em>Pseudomonas fluorescens</em> In5 during Interactions with the Fungus <em>Fusarium graminearum</em> PH-1</h1>
            <p><strong>Authors:</strong> Rosanna  C. Hennessy, Peter  Stougaard, Stefan  Olsson</p>
            <h2>Abstract</h2>
            <p>Genomics, transcriptomics and metabolomics are powerful technologies for studying microbial interactions. The main drawback of these methods is the requirement for destructive sampling. We have established an alternative but complementary technique based on a microplate system combined with promoter fusions for visualizing gene expression in space and time. Here we provide a protocol for measuring spatial and temporal gene expression of a bacterial reporter strain interacting with a fungus on a solid surface. </p>
            <p><strong>Keywords:</strong> Microbial interactions, <em>Pseudomonas</em>, Fungi, Bioassay, Imaging, Spatial and temporal gene expression, Living cells</p>
            <h2>Background</h2><p>Microbial interactions underpin many important biotechnological applications spanning medicine, food development and processing, bioremediation and biocontrol. In order to study microbial interactions, a broad range of technologies exist including genomics, transcriptomics and mass spectrometry imaging to study gene expression and metabolites exchanged during interactions. While these methods advance our understanding of microbial interactions, they also have limitations namely the requirement for destructive sampling. We have developed a microplate reader-based system for visualizing gene expression dynamics in living bacterial cells in response to a fungus in space and real-time. <em>Pseudomonas fluorescens</em> In5 is a Gram-negative soil bacterium and potent producer of secondary metabolites with antifungal activity (Michelsen <em>et al.</em>, 2015; Hennessy <em>et al.</em>, 2015). We previously identified the LuxR-type regulator NunF as a key regulator for synthesis of the antifungal compounds nunamycin and nunapeptin (Hennessy <em>et al.</em>, 2017a). In this protocol, we detail how <em>in vivo</em> monitoring of target gene expression in living bacterial cells interacting with living fungal cells can be performed using a microplate-reader based technique (Hennessy <em>et al.</em>, 2017b). <em>P. fluorescens</em> In5 expressing the red fluorescent protein mCherry fused to the promoter region of a regulator gene <em>nunF</em> indicating activation of an antifungal secondary metabolite gene cluster was used as a reporter system. Time-lapse image recordings of the reporter red signal and a green signal from fluorescent metabolites naturally produced by the bacterium combined with microbial growth measurements showed that <em>nunF</em>-regulated gene transcription is switched on when the bacterium enters the deceleration growth phase and upon physical encounter with fungal hyphae. The established non-destructive method has many advantages notably the ability to provide detailed space and time information on the transcription of target genes in living organisms. Importantly, the technique is a fast and simple alternative and complementary tool to the many technologies already used for studying microbial interactions.<br/><br/>Here we present a detailed protocol for imaging microbial interactions. In this example, we describe the construction of a reporter strain of the antimicrobial isolate <em>Pseudomonas fluorescens</em> In5 coupled with imaging analysis of gene expression of the reporter strain during an interaction with the fungus <em>Fusarium graminearum</em> PH-1 on an agar surface.</p>
        </body>

<p class="pptt" id="biaoti33492">## Materials and Reagents</p><ol class="npwtwok"><li>Eclipse pipette refilling system (Labcon, catalog numbers: 1045-260-000, 1093-260-000, 1036-260-000)</li><li>Nunc<sup>TM</sup> OmniTray<sup>TM</sup> Single-Well Plate (Fischer Scientific, catalog number: 140156)</li><li>Gene Pulser<sup>Â®</sup> Cuvettes 0.2 cm gap (Bio-Rad, catalog number: 1652086)</li><li>2 ml Microcentrifuge tubes (Fischer Scientific, catalog number: 21-402-905)</li><li>NEB<sup>Â®</sup> 5-alpha Competent <em>E. coli</em> (Subcloning Efficiency) (NEB, catalog number: C2988J)</li><li>Restriction enzymes: <em>Bam</em>HI (NEB, catalog number: R3136S ), <em>Eco</em>R1 (NEB, catalog number: R0101S )</li><li>Gentra Puregene Yeast/Bact. kit (QIAGEN, catalog number: 158567)</li><li>Reporter plasmid pSEVA237R expressing the red fluorescent protein mCherry (Hennessy <em>et al.</em>, 2018)</li><li>Phusion high-fidelity polymerase (Fisher Scientific, catalog number: F530S)</li><li>Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System (Promega, catalog number: A9281)</li><li>Plasmid Mini Kit QIAprep Spin Miniprep Kit (QIAGEN, catalog number: 27104)</li><li>Gibson Assembly<sup>Â®</sup> Cloning Kit (NEB, catalog number: E5510S)</li><li>Bacto<sup>TM</sup> agar (Difco, catalog number: 214050)</li><li>Luria-Bertani broth (LB) (Difco, catalog number: 244610)</li><li>LB agar (LBA) (Difco, catalog number: 244520)</li><li>SOC outgrowth medium (NEB, catalog number: B9020S)</li><li>Defined Fusarium Medium (DFM) (Hennessy <em>et al.</em>, 2017b)</li><li>Kanamycin sulfate monohydrate (Duchefa Biochemie, catalog number: K0126)</li><li>Sucrose (Sigma-Aldrich, catalog number: S0389)</li><li>Bacto-tryptone (VWR Chemicals, catalog number: 84610)</li><li>Yeast extract (VWR Chemicals, catalog number: 84601)<br/></li><li>NaCl (Millipore, catalog number: 1.06404.1000)</li><li>Glucose (Sigma-Aldrich, catalog number: 49159)</li><li>Asparagine (Difco, catalog number: 214410)</li><li>MgSO<sub>4</sub> (VWR Chemicals, catalog number: 25164.265)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Sigma-Aldrich, catalog number: NIST200B)</li><li>KCl (Sigma-Aldrich, catalog number: P9333)</li><li>Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>Â·10H<sub>2</sub>O (Sigma-Aldrich, catalog number: S9640)</li><li>CuSO<sub>4</sub>Â·5H<sub>2</sub>O (Sigma-Aldrich, catalog number: C8027)</li><li>FeSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: 1270355)</li><li>MnSO<sub>4</sub>Â·H<sub>2</sub>O (Sigma-Aldrich, catalog number: M7634)</li><li>NaMoO<sub>2</sub>Â·2H<sub>2</sub>O (Sigma-Aldrich, catalog number: 331058)</li><li>ZnSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: Z0251)</li><li>300 mM sucrose (see Recipes)</li><li>LB medium (see Recipes)</li><li>DFM agar (see Recipes)</li><li>1,000x Trace solution (see Recipes)<br/></li></ol><p class="pptt" id="biaoti33493">### Equipment</p><ol class="npwtwok"><li>Pipettes (VWR, catalog numbers: 613-2696E, 613-2702E, 613-2704E)</li><li>Microplate reader (BMG LABTECH, catalog number: FLUO-star Omega)</li><li>Eppendorf<sup>Â®</sup> Mastercycler<sup>Â®</sup> (Sigma-Aldrich, catalog number: EP6313000018-1EA)</li><li>Gene Pulser<sup>Â®</sup> II (Bio-Rad, catalog number: 165-2105)</li><li>Sterile inoculation loops (10 Î¼l) (VWR, catalog number: 12000-812)</li><li>Cork borer (6 mm Ã) for making fungal plugs</li><li>Water bath or a heat block<br/></li></ol><p class="pptt" id="biaoti33494">### Software</p><ol class="npwtwok"><li>BMG Omega Mars Software (Omega V5.11; BMG LABTECH) </li><li>ImageJ analysis software (<a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>)</li><li>"Text image to stackâ macro (ImageJ, <a href="http://rsbweb.nih.gov/ij/macros/ImportTextImageSequence.txt" target="_blank">http://rsbweb.nih.gov/ij/macros/ImportTextImageSequence.txt</a>)</li><li>Bulk Rename Utility, (<a href="https://www.bulkrenameutility.co.uk/" target="_blank">https://www.bulkrenameutility.co.uk</a>)</li><li>Batch converter software converting BMG Omega Mars exported well scan text files to image text files for a rectangle of adjacent wells (MofM.exe)<br/></li></ol><p class="pptt" id="biaoti33495">### Procedure</p><ol class="npwone"><li>Selection of target gene and construction of reporter strains<br/>The first step of this protocol is to select a gene of interest (GOI) for gene expression analysis. Once selected, the promoter region located upstream of the GOI is PCR-amplified and cloned in front of a fluorescent protein-encoding gene. The promoter region (400 bp) upstream of the start codon is PCR-amplified using a high-fidelity polymerase and primers introducing overhangs to enable Gibson Assembly<sup>Â®</sup> cloning (Gibson <em>et al.</em>, 2009) into the target plasmid.<br/><br/></li><li>PCR to amplify promoter region located upstream of target gene<ol class="npwtwo"><li>Thaw the Phusion kit reagents and thoroughly mix after thawing.</li><li>Make a 2x Phusion High-Fidelity (HF) DNA Polymerase PCR master mix using the following recipe:<br/>20 Î¼l Phusion HF Buffer<br/>2 Î¼l 10 mM dNTPs<br/>1 Î¼l DMSO<br/>2 Î¼l forward primer (10 Î¼M) (5â²-CACAGGAGGCCGCCTAGGCCGCGGCCGCGCGAATTCGCCGACCGTTGGTCGGCTTTGTC-3â²)<br/>2 Î¼l reverse primer (10 Î¼M) (5â²-GCTTGCATGCCTGCAGGTCGACTCTAGAGGATCCAAGCCTGCATACCAAAATCGCTG-3â²)<br/>1 Î¼l Phusion HF Taq Polymerase<br/>70 Î¼l Nuclease-free water</li><li>Dispense 49 Î¼l of 2x Phusion PCR master mix into sterile 0.5 Î¼l PCR tubes and add 1 Î¼l genomic DNA (50 ng Î¼l<sup>-1</sup>) purified from <em>P. fluorescens</em> In5 as previously described (Dichmann <em>et al.</em>, 2018).</li><li>PCR program:<br/>a.98 Â°C 30 s<br/>b.98 Â°C 10 s<br/>c.62 Â°C 15 s<br/>d.72 Â°C 15 s<br/>e.Repeat steps b-d, 30x<br/>f.72 Â°C 10 min</li></ol><em>Note: Recommended running a water blank to ensure amplified bands are not due to contaminants in the PCR reagents.</em><br/><em> PAUSE POINT: PCR products can be stored at 4 Â°C or -20 Â°C.</em><br/><br/></li><li>Preparation of reporter plasmid for Gibson assembly<sup>Â®</sup> cloning<ol class="npwtwo"><li>Digest plasmid DNA of the reporter plasmid pSEVA237R (Hennessy <em>et al.</em>, 2017a) harboring a gene encoding the red fluorescent marker mCherry using the restriction enzymes <em>Bam</em>HI-<em>Eco</em>R1. Once digested, purify the digested plasmid DNA using the Promega Gel extraction kit according to the manufacturerâs instructions.</li><li>Set up a Gibson assembly<sup>Â®</sup> reaction on ice according to the manufacturerâs instructions. Incubated the Gibson assembly<sup>Â®</sup> reaction for 1 h at 50 Â°C without shaking. </li><li>Thaw 50 Î¼l of <em>E. coli</em> DH5Î± chemically competent cells on ice for 30 min and add 5 Î¼l of Gibson assembly<sup>Â®</sup> mix and incubate cells for further 30 min on ice. </li><li>Heat-shock cells using either a water bath for 45 s at 42 Â°C or a heat block warmed to 42 Â°C for 90 s. </li><li>Add 250 Î¼l of SOC outgrowth medium to the heat-shocked cells and incubate shaking 200 <em>x g</em> at 37 Â°C for 1 h.</li><li>Plate 100 Î¼l and 200 Î¼l on LBA supplemented with kanamycin (50 Î¼g/ml). </li><li>Isolate antibiotic resistant colonies into 10 ml LB supplemented with kanamycin (50 Î¼g/ml) and incubate shaking 200 <em>x g</em> overnight at 37 Â°C.</li><li>Purify plasmid DNA and digest using <em>Bam</em>HI-<em>Eco</em>R1 to identify positive clones.</li><li>Perform DNA sequencing to confirm the integrity of the DNA. Once the plasmid DNA is confirmed by sequencing, it can be transformed into <em>P. fluorescens</em> In5.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transformation of <em>P. fluorescens</em> In5 with the reporter plasmid pSEVA237R::<em>P<sub>GOI</sub></em> <ol class="npwtwo"><li>Inoculate <em>P. fluorescens</em> In5 in 10 ml LB broth shaking 200 <em>x g</em> overnight at 28 Â°C.</li><li>Centrifuge 6 ml of overnight culture at 4,000 <em>x g</em> for 5 min at 20 Â°C.</li><li>Wash cells three times each with 4 ml 300 mM sterile filtered sucrose.</li><li>Resuspend the washed bacterial pellet in 400 Î¼l of 300 mM sterile filtered sucrose.</li><li>Add 300-500 ng plasmid DNA from each construct: tester plasmid (pSEVA237R::<em>P<sub>GOI</sub></em>) and empty vector control (pSEVA237R) to 100 Î¼l competent cells.</li><li>Transfer mixture to 2 mm gap width electroporation cuvette. </li><li>Apply a pulse (25 Î¼F, 200 â¦, 2.5 kV) in the electroporation apparatus. </li><li>Add 1 ml LB at room temperature.</li><li>Transfer mixture to a 2 ml Eppendorf tube.</li><li>Shake mixture at 200 <em>x g</em> 2 h at 28 Â°C.</li><li>Isolate antibiotic resistant colonies into 10 ml LB supplemented with kanamycin (50 Î¼g/ml) and incubate shaking 200 <em>x g</em> overnight at 28 Â°C.</li><li>Purify plasmid DNA and digest using <em>Bam</em>HI-<em>Eco</em>R1 to identify positive clones.</li></ol><em>Note: Recommend adding no DNA to 100 Î¼l competent cells as a negative control. If a lawn appears on transformed plates but not on the negative control plate, streak cells onto a fresh LBA kanamycin (50 Î¼g/ml) plate and incubate overnight with no shaking at 28 Â°C to isolate single antibiotic-resistant colonies.</em><br/><br/></li><li>Pre-culturing of <em>F. graminearum</em> PH-1 and <em>P. fluorescens</em> In5<ol class="npwtwo"><li>Using a sterile cork borer, transfer a 6 mm Ã sized plug of <em>F. graminearum</em> PH-1 onto DFM media prepared as previously described (Hennessy <em>et al.</em>, 2017b) and incubate for 5 days at 28 Â°C.</li><li>Inoculate <em>P. fluorescens</em> In5 harboring either pSEVA237R::<em>P<sub>GOI</sub></em> or pSEVA237R in 10 ml LB broth supplemented with 50 Î¼g ml<sup>-1</sup> kanamycin (antibiotic selectable marker on pSEVA237R backbone) and incubate shaking 200 <em>x g</em> overnight at 28 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Establishment of bacterial-fungal interaction on solid surface<ol class="npwtwo"><li>Add 35 ml of DFM agar to a sterile Nunc<sup>TM</sup> OmniTray<sup>TM</sup>. Place 10 plugs of <em>F. graminearum</em> PH-1 (6 mm Ã) down the center of the plate in a straight line. </li><li>Using a 10 Î¼l sterile loop streak the bacterial reporter strains 3 cm away from the fungal plugs. Allow bacterial streaks to dry on the surface.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Experimental set-up<br/></li></ol><ol class="npwtwok"><li>The bacterial reporter strains and fungus are then co-cultured in the microplate reader, programmed to simultaneously measure red and green fluorescence respectively in addition to cell density (Figures 1 and 2). </li><li>For step-by-step instructions for defining plates and time-series recording of images using the plate reader software BMG Omega Mars Software see Hennessy <em>et al.</em> (2017b).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190613/20190613194431_5128.jpg" width="144"/><br/><strong>Figure 1. Schematic of experimental setup.</strong> A. Plate reader programmed to measure fluorescence and cell density and serves as an incubator to co-cultivate microbes at selected temperatures. B. 96-well microplates exchanged for OmniTray enabling the addition of agar medium providing a solid surface for interactions. C. Bacterial reporter strains are constructed whereby the promoter region located upstream of a gene of interest is cloned in front of a gene encoding a fluorescent protein (<em>e.g.</em>, mCherry) to measure gene transcription in living cells. D. Biological interaction is established where fungal plugs are placed down the center of the OmniTray and bacterial reporter strains streaked either side of the fungal plugs.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190613/20190613194501_8773.jpg" width="140"/><br/><strong>Figure 2. Schematic of scanning function.</strong> A script can be written to instruct the plate reader to measure multiple parameters simultaneously (A) and perform a scanning function âimagingâ a specific area of the plate measuring gene expression and growth. Bacteria streaked on the left-hand side of the fungal strains is the reporter strain (pSEVA237R::<em>P</em><sub><em>GOI</em></sub>) and the bacterial streak on the right-hand side of the fungal plugs is the control strain (pSEVA237R). âWell mapâ is shown using a rainbow (blue-green-yellow-red) representation of intensities of signals for one of the protocols from low to high (B). <br/></li></ol><p class="pptt" id="biaoti33496">### Data analysis</p><p class="ppzz">Once the runs are complete, a series of scans or images are obtained which must then be exported as text files prior to batch conversion of the files from Mars formatted well image text files into image text files for the whole plate. The text files can then be imported into the free image analysis software ImageJ. Once in ImageJ, the time-series image stack can be converted into an animation presentation and quantitative data can be extracted from specific areas of interest (Figures 3 and 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190613/20190613194646_6376.jpg" width="137"/><br/><strong>Figure 3. Schematic showing data processing. </strong>Upon completion of imaging, a time series of scans or images are (A) exported as MARS text files. (B) Batch converted into a time series of text image formatted files using the program MofM.exe. (C) The text image time series files are imported into the free image analysis software ImageJ (D) to make a TIF formatted time-series image stack retaining all the intensity levels of the original scans (32-bit TIF image stack used). Once in ImageJ, the time-series image stack can be converted into an animation presentation to visualize spatial and temporal gene expression or using the image analysis procedures to extract numerical data from the image stack (E).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190613/20190613194709_5618.jpg" width="148"/><br/><strong>Figure 4. System enables real-time imaging of transcription. </strong>Example of time-series stack converted into an animation video (available at <a href="https://www.nature.com/articles/s41598-017-00296-4#Sec9" target="_blank">https://www.nature.com/articles/s41598-017-00296-4#Sec9</a>). Top image shows a bacterial reporter strain emitting red fluorescence. Middle image shows fluorescent compounds produced by the bacterial reporter strains and a weak signal of fungal produced fluorescent compounds seen accompanying the expanding fungal mycelium growth. Bottom image shows fungal growth.</p><br/><ol class="npwone"><li>Export of imaging runs into text files<ol class="npwtwo"><li>Open the MARS Omega Software.</li><li>Import files (or scans) from the plate reader by selecting the .RUC files (note the plate reader files should reflect the number of parameters run <em>e.g.</em>, 141 files divided by 3 measurements [cell density, red fluorescence, green fluorescence] should give 47 files per run).</li><li>Once the files are imported select all the runs, select the set of stored test runs using the BMG Omega MARS program âOpen functionâ (use the program version V3.20 R2 that includes the new âExport multiple Ascii filesâ function).</li><li>Next sort the files in order (<em>i.e.</em>, mCherry runs, GFP runs <em>etc</em>.).</li><li>Click âsort according to nameâ (sort the test runs so that it is easy to select all test runs with the same protocol).</li><li>Mark the set of test runs and right-click. Select âExport multiple ASCII filesâ. Check settings by clicking the âSettingsâ button: Untick boxes in the âFile content optionsâ. In the âFile Name and Locations optionsâ do the following:<ol class="npwthree"><li>Select folder to store the set of output text files (there will be one file for each reading time). </li><li>File name: use the &lt; automatic filename creation &gt; option. </li><li>Select âFile Extensionâ= TXT âSeparatorâ=TAB and âIf file existâ=ârename old file adding date/timeâ. In âAuto mode and Manage Test Runs File Export optionsâ only tick the âExport microplate viewâ option. Then press the âOKâ button. </li></ol></li><li>Next, replace the âoâ s with â0â to ensure files are ordered correctly.</li></ol><em>Note: For step 3, export only files from one protocol (i.e., export mCherry files). In other words, for every protocol a new analysis has to be performed independently e.g., analysis 1 (mCherry), 2 (cell density) and so on.</em><br/><br/></li><li>Bulk rename utility tool for renaming files<ol class="npwtwo"><li>Download software <strong>Bulk Rename Utility</strong> tool (<a href="http://www.bulkrenameutility.co.uk/Main_Intro.php" target="_blank">http://www.bulkrenameutility.co.uk/Main_Intro.php</a>).</li><li>Open Bulk Rename Utility.</li><li>Open txt. Files.</li><li>Highlight only those to change (only 2 digit files <em>e.g.</em>, o98 not 100 or above).</li><li>Go into documents folder where .txt files saved and check that they have been changed from no<strong>o</strong>98 to no<strong>0</strong>98.</li><li>Prepare destination folder for these files by removing all spaces <em>e.g.</em>, Imaging Test Run to <strong>ImagingTestRun</strong> (necessary for converter program to function) and also find the location for this folder through my Computer: <strong>H:\Documents\ImagingTestRun</strong>. Then insert additional forward slashes for converter program to read: <strong>H:\\Documents\\ImagingTestRun</strong>. </li><li>Double-click the converter program MofM.exe (in supplementary file S2) that can be placed in any folder and need no installation. In the dialogue enter full path name (note that use double backslash \\ in the path name) of the folder containing the text files saved in step 6 (and eventually renamed by giving a prefix for the converted files. The program is now run and there is created a subfolder containing the converted files that can be used for text image series import to stack using the ImageJ Macro in supplementary file S3).</li><li>Copy location address by right-clicking in <span style="font-family:Courier New;">program &gt; edit &gt; paste</span> then press enter and enter a prefix, <em>e.g.</em>, pl (as the program is converting to a single well <strong>p</strong>late<strong> l</strong>ayout) &gt; enter. You should find a new sub-folder under TestImagingRun called â<strong>plfolder</strong>â with all the files.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Installation of ImageJ analysis software and macro retrieval<ol class="npwtwo"><li>Download and install ImageJ.</li><li>Go to the following link to retrieve the macro: <a href="http://rsbweb.nih.gov/ij/macros/ImportTextImageSequence.txt" target="_blank">http://rsbweb.nih.gov/ij/macros/ImportTextImageSequence.txt</a>.</li><li>Save the macro as a macro file (.ijm) in the macros folder of the ImageJ folder as âText to images. ijmâ.</li><li>Open ImageJ software as follows:<br/>&gt; plugins &gt; macros &gt; install &gt; select and open Text images to stack.ijm</li><li>The macro is now installed.</li></ol><em>Note: Recommend verifying the macro is correctly installed by the following: &gt; plugins &gt;macros &gt; âtext to images stackâ under the macros tab.</em><br/><br/></li><li>ImageJ data analysis<ol class="npwtwo"><li>First open ImageJ then click <span style="font-family:Courier New;">plugins &gt; macros &gt; text image to stack &gt; select plfolder</span>.</li><li>To reorder files, open <span style="font-family:Courier New;">image &gt; stacks &gt; tools &gt; reverse</span>.</li><li>This results in a pixelated image so click <span style="font-family:Courier New;">image &gt; adjust &gt; brightness and contrast &gt; auto</span>.</li><li>Once steps 1-3 are completed, the files are ready for analysis and the next step is extraction of data from Regions of Interest (ROI).</li><li>Enlarge the image using the magnifying glass and click on the image (right click enlarges, left click reduces).</li><li>Click on ROI and export data as a table at a specific slice (<em>i.e.</em>, time-point slice or image 31/47).</li><li>A picture shows the cumulative expression at a specific time over the whole area. To decide which time-point to investigate, scroll along <em>e.g.</em>, 24/27 and draw on the image.</li><li>To measure distance, the first step is to calibrate the image. For example, to measure the known distance between two streaks (<em>e.g.</em>, 5 cm) first press analyze, set scale. Calculate the distance in pixels. Known distance 50 mm.</li><li>To investigate expression over time on a plate reader, first mark the ROI, then <span style="font-family:Courier New;">select area &gt; image &gt; stacks &gt; plot Z axis profile &gt; save &gt; save as excel</span> (Figure 5).</li><li>Click <span style="font-family:Courier New;">Live &gt; File &gt; save as &gt; selection</span> (save ROI file <em>e.g.</em>, stack.roi).</li><li>Go to â<span style="font-family:Courier New;">Files ready for analysis</span>â and save as single TIFF file <em>e.g.,</em> stack. tif.</li><li>To open the file, click â<span style="font-family:Courier New;">File</span>â <span style="font-family:Courier New;">&gt; Open stack.tif &gt; select a time-point</span> <em>e.g.</em>, 34/47 <span style="font-family:Courier New;">&gt; File &gt; save as &gt;JPEG</span> to get an image.</li><li>For saving marked files (<em>i.e.</em>, marking a specific area): <span style="font-family:Courier New;">File &gt; save as selection &gt; stack.roi</span>. This step can be repeated to add a second ROI.</li></ol><em>Note: For step 10, remember to also take a background âcontrolâ value. Recap: highlight the area of interest then take a Z-plot and save the image for methods.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190613/20190613195903_9002.jpg" width="176"/><br/><strong>Figure 5. Extraction of quantitative data.</strong> Example of regions of interest (ROI) of two reporter strains: Tester (bacterium with mCherry behind <em>nunF</em> promoter) and the Control (bacterium with mCherry and no promoter). Using the free image analysis software ImageJ (<a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>) ROIs are selected and a z-axis profile is plotted to obtain quantitative data.<br/></li></ol><p class="pptt" id="biaoti33497">### Notes</p><ol class="npwtwo"><li><strong>How to add color:</strong> work of the TIFF file created above then click image &gt; look-up tables &gt; red. Note that if the file is saved in TIFF format, the color will not be red so remember to save the file in PNG format.</li><li><strong>How to export a video: </strong>save image as an AVI file then compress as a JPEG and define the frame rate <em>e.g.</em>, 7 fps (this can be manually decided based on the scrolling time through image).</li><li><strong>How to overlay images:</strong> The above method describes the data processing required for analyzing one set of files from the plate reader (<em>i.e.</em>, red fluorescence readings). Before overlaying, a TIFF file must be created for each parameter measured on the plate reader:<br/>3x TIFF files for each âchannelâ<ol class="npwthree"><li>Channel 1: red fluorescence signal stack to TIFF (color red)</li><li>Channel 2: blue fluorescence signal stack to TIFF (color blue)</li><li>Channel 3: cell density signal stack to TIFF (color grey or black/white)</li></ol></li><li>Click <span style="font-family:Courier New;">image &gt; colour &gt; merge channels &gt; create composite &gt; ok</span> </li><li>Save the super-imposed file.</li></ol><p class="ppzz"><em>Note: For âcreate compositeâ step, tick âcreate compositeâ and leave the other parameters blank. Note this type of image can only be used for distance or visualization. Red fluorescence e.g., mCherry signal cannot be measured, for this the single TIFF file must be used. It is also important to measure the distance between strains before and after the runs and to take a scan or photo of the plate.</em> </p><p class="pptt" id="biaoti33498">### Recipes</p><ol class="npwtwok"><li>300 mM sucrose (Filter sterilized)<br/>Add 102.69 g sucrose to 800 ml sterilized MilliQ water and stir until the sucrose is dissolved, and make the final volume to 1 L</li><li>LB<ol class="npwthree"><li>Add 10 g Bacto-tryptone, 5 g yeast extract, 10 g NaCl to 800 ml MilliQ water and stir until the reagents are dissolved</li><li>Autoclave at 121 Â°C for 15 min</li><li>To make LBA, add 15 g Bacto<sup>TM</sup> agar</li></ol></li><li>DFM agar<ol class="npwthree"><li>Add 20 g Bacto<sup>TM</sup> agar to 716 ml MilliQ water and autoclave at 121 Â°C for 15 min</li><li>Following autoclaving add the following: <br/>20 ml 20% w/v glucose <br/>1 ml of 1,000x Trace solution<br/>50 mM asparagin <br/>10 ml 0.21 M MgSO<sub>4</sub> <br/>10 ml (1.12 M KH<sub>2</sub>PO<sub>4</sub> + 0.7 M KCl), pH 6</li></ol></li><li>1,000x Trace solution<br/>Add 40 mg Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>Â·10H<sub>2</sub>O, 400 mg CuSO<sub>4</sub>Â·5H<sub>2</sub>O, 1.2 mg FeSO<sub>4</sub>Â·7H<sub>2</sub>O, 700 mg MnSO<sub>4</sub>Â·1H<sub>2</sub>O, 800 mg NaMoO<sub>2</sub>Â·2H<sub>2</sub>O, 10 g ZnSO<sub>4</sub>Â·7H<sub>2</sub>O to 800 ml sterilized MilliQ water and autoclave at 121 Â°C for 15 min<br/></li></ol><p class="pptt" id="biaoti33499">### Acknowledgments</p><p class="ppzz">This work was supported by the Villum Foundation grant VKR7310 (Microbial Communication-A Key to the Development of Novel Sustainable Agri- and Aquaculture Practices Using Biological Control Bacteria) awarded to P. Stougaard.</p><p class="pptt" id="biaoti33500">### Competing interests</p><p class="ppzz">None declared.</p><p class="pptt" id="biaoti33501">### References</p><ol class="npwtwok"><li>Hennessy, R. C., Christiansen, L., Olsson, S. and Stougaard, P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29203144" target="_blank">A broad-host range dual-fluorescence reporter system for gene expression analysis in Gram-negative bacteria.</a> <em>J Microbiol Methods</em> 144: 173-176.</li><li>Dichmann, S. I., Park, B., Pathiraja, D., Choi, I. G., Stougaard, P. and Hennessy, R. C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30533854" target="_blank">Draft genome sequence of a novel <em>Serratia</em> sp. strain with antifungal activity.</a> <em>Microbiol Resour Announc</em> 7(22) pii: e01340-18.</li><li>Gibson, D. G., Young, L., Chuang, R. Y., Venter, J. C., Hutchison, C. A., 3rd and Smith, H. O. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495" target="_blank">Enzymatic assembly of DNA molecules up to several hundred kilobases.</a> <em>Nat Methods</em> 6(5): 343-345.</li><li>Hennessy, R. C., Glaring, M. A., Michelsen, C. F., Olsson, S. and Stougaard, P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26607883" target="_blank">Draft genome sequence of <em>Pseudomonas</em> sp. strain in5 isolated from a greenlandic disease suppressive soil with potent antimicrobial activity.</a> <em>Genome Announc</em> 3(6) pii: e01251-15.</li><li>Hennessy, R. C., Phippen, C. B. W., Nielsen, K. F., Olsson, S. and Stougaard, P. (2017a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28782279" target="_blank">Biosynthesis of the antimicrobial cyclic lipopeptides nunamycin and nunapeptin by <em>Pseudomonas fluorescens</em> strain In5 is regulated by the LuxR-type transcriptional regulator <em>nunF</em>.</a> <em>Microbiologyopen</em> 6(6).</li><li>Hennessy, R. C., Stougaard, P. and Olsson, S. (2017b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28325928" target="_blank">A microplate reader-based system for visualizing transcriptional activity during <em>in vivo</em> microbial interactions in space and time.</a> <em>Sci Rep</em> 7(1): 281.</li><li>Michelsen, C. F., Jensen, H., Venditto, V. J., Hennessy, R. C. and Stougaard, P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26734508" target="_blank">Bioactivities by a crude extract from the Greenlandic <em>Pseudomonas</em> sp. In5 involves the nonribosomal peptides, nunamycin and nunapeptin.</a> <em>PeerJ </em>3: e1476.<br/></li></ol>