<body>
            <h1>Probing Conformational States of a Target Protein in <em>Escherichia coli</em> Cells by <em>in vivo</em> Cysteine Cross-linking Coupled with Proteolytic Gel Analysis</h1>
            <p><strong>Authors:</strong> Sujeet  Kumar, Natividad   Ruiz </p>
            <h2>Abstract</h2>
            <p>Transporters are dynamic membrane proteins that are essential to the physiology of cells. To function, transporters must cycle between various conformational states, so to understand their mechanistic details, it is critical to characterize how their structure changes during the transport cycle. One approach to studying the dynamics of transporters takes advantage of the chemistry of cysteine by using sulfhydryl-reactive, bi-functional cross-linkers to probe changes in the distance between two specific residues that have been substituted to cysteine. This approach is mostly used to study transporters <em>in vitro</em>, not in their natural cellular environment. Here we describe a protocol based on structure-guided cysteine cross-linking and proteolysis-coupled gel analysis to probe conformational changes of a target transporter in live <em>Escherichia coli</em> cells. Although cross-linking approaches have been used to probe the proximity between transmembrane segments in membrane proteins <em>in vivo</em>, to our knowledge this protocol is the first to be used to interrogate transporter dynamics in cells. The use of this protocol is optimal for proteins with known or modeled structures to guide the replacement of specific residues with cysteines and the selection of cross-linking agents with various spacer arm lengths. This protocol allows for discriminating easily cross-linked and uncross-linked species and does not require the often difficult or unavailable reconstitution of transport activity in an <em>in vitro</em> system. In addition, this protocol could be used to probe the conformation of transporters in cells treated with transport inhibitors in order to better understand their mechanism of action, and potentially dynamic interactions between domains in proteins that are not transporters.</p>
            <p><strong>Keywords:</strong> Conformational dynamics, <em>In vivo</em> cysteine cross-linking, Membrane transporters, Alternate-access mechanism, Maleimide homo-bifunctional crosslinkers</p>
            <h2>Background</h2><p>Transport of substrates across membranes is fundamental to all organisms. Membrane proteins that function as transporters constitute a very large and structurally diverse group of proteins that translocate a vast landscape of substrates through different mechanisms. Not surprisingly, approximately 25% of polypeptides encoded in fully sequenced genomes are predicted to be membrane proteins, and many of them are likely transporters (Elofsson and von Heijne, 2007). Their essential role in human health is underscored by the fact that genetic defects in specific transporters cause disease (<em>e.g.</em>, cystic fibrosis), and that some transporters are targeted by pharmaceuticals (<em>e.g.</em>, inhibitors of efflux pumps) (Meng <em>et al.</em>, 2018; Wiese and Stefan, 2019). Studying transporters is therefore important, but it is also challenging given their dynamic nature and the inherent difficulty of working with membrane proteins in biochemical and structural studies. Especially difficult is obtaining high-resolution structures, particularly those representing different states in the transport cycle. Even when available, static structures must be validated and their role in the transport cycle elucidated by complementary <em>in vitro</em> and <em>in vivo</em> assays that might not be available for many transporters (Mulligan and Mindell, 2017).<br/> <br/>Chemical biology approaches such as cysteine cross-linking can provide mechanistic details about transporters even in the absence of high-resolution structures (Mulligan and Mindell, 2017). Indeed, cysteine cross-linking has been used to probe the structure and conformational dynamics of many membrane proteins <em>in vitro</em> (Sun <em>et al.</em>, 1997; Wu <em>et al.</em>, 1999; Rimon <em>et al.</em>, 2002; Hennon and Dalbey, 2014; Radchenko <em>et al.</em>, 2016; Mulligan and Mindell, 2017). However, a concern when using this approach is that the <em>in vitro</em> system might not properly reflect the native environment of the target protein. The <em>in vitro</em> approach also requires the purification of multiple protein variants, which can be difficult for many transporters. More importantly, <em>in vitro</em> cysteine cross-linking relies on the functional reconstitution of the activity of purified membrane proteins, which can be extremely difficult or unavailable for many transporters. For example, developing an <em>in vitro</em> functional assay can be difficult for glycolipid transporters because their complex substrates are often not available. In our case, we study the bacterial transporter MurJ, which translocates the glycolipid lipid II from the inner leaflet to the outer leaflet of the cytoplasmic membrane, and for which we do not currently have an <em>in vitro</em> reconstitution assay to probe its function (Ruiz, 2008; Butler <em>et al.</em>, 2013 and 2014; Sham <em>et al.</em>, 2014; Ruiz, 2015 and 2016; Rubino <em>et al.</em>, 2018; Kumar <em>et al.</em>, 2019). As a result, we developed an <em>in vivo</em> cysteine cross-linking protocol that captures the different conformations the MurJ transporter adopts in growing <em>Escherichia coli</em> cells, when MurJ activity is required for viability (Kumar <em>et al.</em>, 2019).<br/> <br/>We designed this protocol to probe whether the V-shaped target transporter MurJ can adopt two states during the transport cycle in live bacteria: one in which the internal cavity is open to the cytoplasm (<em>i.e.</em>, inward-open conformation) and another in which the cavity is open to the periplasm (<em>i.e.</em>, outward-open conformation). Based on a crystal structure assumed to reflect a single state in the transport cycle, these conformations were predicted to result from the relative movement of two large domains or lobes that traverse the membrane and give the transporter its V shape (Kuk <em>et al.</em>, 2017; Zheng <em>et al.</em>, 2018). To test this model and obtain evidence that these states exist in cells, we first introduced a cysteine residue into either the periplasmic (out) or the cytoplasmic (in) side of each of the lobes (N-lobe and C-lobe) of our otherwise cysteine-less transporter. We then used homo-bifunctional cross-linking reagents of various lengths that could cross-link specific cysteine pairs when the two lobes adopted only one of the two predicted states, inward-open (Figure 1A) or outward-open (Figure 1B). To easily distinguish cross-linked and uncross-linked target proteins, we engineered a functional variant of the target protein containing a well-known protease site (thrombin) in the loop connecting the N and C lobes that could be cleaved after cells were treated with cross-linkers (Figure 2A). In addition to the protease site, a fusion tag (FLAG) was also added to the N terminus of the target protein (Figure 2A) for detection by immunoblotting (Figure 2B). Using this protocol, we demonstrated that, <em>in vivo</em>, the glycolipid MurJ transporter adopts both inward- and out-ward open conformations and hence uses the alternate-access mechanism. Furthermore, we also showed that the transporter stalls in the outward-open state in cells treated with ionophores that collapse the membrane potential (Kumar <em>et al.</em>, 2019). We therefore showed that our protocol can be used 1) to test the mechanism of transport by monitoring the distance between specific sites in a transporter during the transport cycle in live cells, 2) to validate high-resolution structures and <em>in silico</em> structural predictions, and 3) to test the effects of small-molecule inhibitors of transporters. It should also be noted that our protocol could also be used to probe dynamic interactions between domains of proteins that are not transporters (Kumar <em>et al.</em>, 2019).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617012132_0018.jpg" width="111"/><br/><strong>Figure 1. Selection of residues on target protein (<em>E. coli</em> MurJ) to be modified to cysteines for <em>in vivo</em> cross-linking.</strong> On the left, side view of the inward-open structure (panel A, Protein Data Bank ID code 6CC4) and outward-open homology model of MurJ (panel B). The N lobe (TM1-6) is colored pink and C lobe (TM7-14) is colored gold. The positions of amino acids to be substituted to cysteines in both periplasmic and cytoplasmic sides are denoted as blue spheres for those in the N lobe and green spheres for those in the C lobe. On the right, a cartoon representation of the structures showing that crosslinking (yellow line) can only occur in one conformation: between periplasmic cysteines closed enough to crosslink in the inward-open conformation (panel A) or between cytoplasmic cysteines closed enough to crosslink in the outward-open conformation (panel B). The membrane is represented with a grey box. PyMOL (version 2.0.1, Schrodinger) software was used to prepare this figure.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617012213_7792.jpg" width="182"/><br/><strong>Figure 2. Proteolytic analysis of target protein variant utilized for probing the conformational states <em>in vivo</em>.</strong> A. Cartoon representation of the target protein (<em>E. coli</em> MurJ) variant and its proteolytic analysis. A thrombin protease site (purple box) was introduced between the N lobe (pink) and C lobe (gold) of cysteine-less target protein (âCys). An N-terminal FLAG tag used for detection by immunoblotting is shown as a red flag on the N lobe. In the absence of thrombin treatment, the full-length âCys-thrombin variant should appear as a ~37 kDa band in a FLAG immunoblot. Thrombin treatment should cleave the two lobes, so only a fast-migrating band corresponding to the N lobe should be detected by FLAG immunoblotting. B. Result of thrombin treatment of full-length FLAG-MurJâCys (âCys) and FLAG-MurJâCys-thrombin (âCys-Thrombin) proteins. The small fragment resulting from thrombin digestion is not shown as it runs off the 10% SDS-polyacrylamide gel used for immunoblotting. We used this type of gel because it is optimal for the detection of the full-length target protein.</p>
        </body>

<p class="pptt" id="biaoti28104">## Materials and Reagents</p><ol class="npwtwok"><li>Nunc cryotube vials (Thermo Scientific, catalog number: 374502)</li><li>Bacteriological Petri plates (VWR, catalog number: 25384-092)</li><li>Sterile test tubes (17 x 100 mm, RPI, Research Products International, catalog number: 168599)</li><li>15-ml centrifuge tubes (VWR, catalog number: 10025-690)</li><li>50-ml centrifuge tubes (GeneMate, catalog number: C-33943-3)</li><li>1.7-ml microcentrifuge tubes (Axygen, catalog number: MCT-175-C)</li><li>Ultracentrifuge tubes (Beckman Coulter, catalog number: 343778)</li><li>Serological pipettes: 5-ml pipette (VWR, catalog number: 89130-896), 10-ml pipette (VWR, catalog number: 89130-910), 25-ml pipette (VWR, catalog number: 89130-910)</li><li>0.1-10 Î¼l pipette tips (GeneMate, catalog number: P-1236-10)</li><li>20-200 Î¼l pipette tips (GeneMate, catalog number: P-1236-200)</li><li>101-1,000 Î¼l pipette tips (GeneMate, catalog number: P-1236-1250)</li><li>0.2 Î¼m filter (Millipore, Model; Millex<sup>GP</sup>, catalog number: 202012)</li><li>Aluminum foil</li><li>All-purpose wrap (Fisher Brand, catalog number: 15-610)</li><li>Nitrile or latex gloves (VWR, catalog number: 89038-270)</li><li>Microscope slides (Corning, catalog number: 2947-75X25)</li><li>Thick Blot paper (Bio-Rad, catalog number: 1703965)</li><li><em>Escherichia coli</em> crosslinking strains carrying derivatives of plasmid pET23/42FLAGMurJâcysThrombin (Dr. Natividad Ruiz strain database) (Kumar <em>et al.</em>, 2019)<br/><em>Note: These strains will vary depending on the target protein and plasmids available.</em> </li><li>Immun-Blot PVDF membrane (Bio-Rad, catalog number: 1620177)</li><li>Trans-Blot SD semidry transfer cell (Bio-Rad, catalog number 1703848)</li><li>Ice</li><li>Distilled sterile water (dH<sub>2</sub>O)</li><li>LB Broth, Lennox (Fisher Scientific, catalog number: BP1427-2)</li><li>Difco-Agar Bacteriological (BD, catalog number: BD-214510)</li><li>Ampicillin sodium salt (Sigma-Aldrich, catalog number: A9518-5G) or other antibiotic appropriate for the selection of plasmids encoding other target proteins</li><li>1,6-bis(maleimido)hexane (BMH; Thermo Scientific Pierce, catalog number: 22330)</li><li>1,2-bis(maleimido)ethane (BMOE; Thermo Scientific Pierce, catalog number: 22323)</li><li><em>o</em>-PDM (Sigma-Aldrich, catalog number: 104590-5G)</li><li>L-cysteine (Sigma-Aldrich, catalog number: 168149-5G)</li><li>Lysozyme (Amresco, catalog number: 0663-5G)</li><li>Benzonase (Sigma-Aldrich, catalog number: E1014 5KU) </li><li>Thrombin from bovine plasma (Sigma-Aldrich, catalog number: T6634-100UN) </li><li>Ethylenediaminetetraacetic acid disodium dihydrate (EDTA) (Sigma-Aldrich, catalog number: E5134-1KG)</li><li>Calcium chloride (CaCl<sub>2</sub>) (Sigma-Aldrich, catalog number: 223506-500G)</li><li>Ammonium persulfate (Amresco, catalog number: 0486-100G)</li><li>N-Dodecyl-Î²-D-maltoside (DDM; Affymetrix, catalog number: D310S)</li><li>Magnesium chloride (MgCl<sub>2</sub>) (Sigma-Aldrich, catalog number: T6634-100UN)</li><li>Sodium citrate tribasic dihydrate (Sigma-Aldrich, catalog number: C8532-1KG)</li><li>Disodium hydrogen phosphate heptahydrate (Na<sub>2</sub>HPO<sub>4</sub>Â·7H<sub>2</sub>O) (Amresco, catalog number: 0348-1KG)</li><li>Sodium dihydrogen phosphate (NaH<sub>2</sub>PO<sub>4</sub>) (Fischer Scientific, catalog number: S397-500)</li><li>Sodium chloride (NaCl) (Sigma-Aldrich, catalog number: 59888-2.5KG)</li><li>PEG-8000 (Sigma-Aldrich, catalog number: 89510-250G)</li><li>Sucrose (Amresco, catalog number: 0335-1KG)</li><li>Dimethyl sulfoxide (DMSO) (Amresco, catalog number: 0231-500ml)</li><li>Tris (GoldBioCom, catalog number: T-4005-5)</li><li>Sodium dodecyl sulfate (SDS) (Sigma-Aldrich, catalog number: 62862-1KG)</li><li>Urea (Ameresco, catalog number: 0568-1KG)</li><li>Bromophenol blue sodium (Amresco, catalog number: 0312-50G)</li><li>Glycerol (Sigma, catalog number: G7893)</li><li>Î²-mercaptoethanol (Amresco, catalog number: 0482-250ml)</li><li>Acrylamide (Protogel) (National diagnostic, catalog number: EC-890) </li><li>Glycine (Fisher Scientific, catalog number: BP381-5)</li><li>Tetramethylethylenediamine (TEMED) (Bio-Rad, catalog number: 161-0800)</li><li>Instant non-fat dry milk powder</li><li>Anti-FLAG M2 (Sigma-Aldrich, catalog number: F3165)</li><li>Methanol (Fisher, catalog number: A412P-4)</li><li>Tween 20 (Amresco, catalog number: 0777-1L)<br/></li><li>ECL<sup>TM</sup> anti-mouse IgG, horseradish peroxide-linked whole antibody (from sheep) (GE healthcare, catalog number: NA931V)</li><li>Clarity Western ECL substrate (Bio-Rad, catalog number: 1705061)</li><li>Spheroplast buffer (see Recipes)</li><li>Thrombin-reaction buffer (see Recipes)</li><li>Thrombin-reconstitution buffer (see Recipes)</li><li>Benzonase dilution buffer (see Recipes)</li><li>2x AB Sample buffer (see Recipes)</li><li>Phosphate Buffered Saline (1x PBS; see Recipes)</li><li>10x SDS-PAGE running buffer (see Recipes)</li><li>1x Transfer buffer (see Recipes)</li><li>10x TBS Buffer (see Recipes)</li><li>1x TBS + 0.1% Tween 20 (1x TBST) (see Recipes)</li><li>5% Milk TBST (see Recipes)</li><li>1 M L-cysteine (see Recipes)</li><li>Maleimide homo-bifunctional cross-linkers (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28105">### Equipment</p><ol class="npwtwok"><li>Microscope (Nikon, model: Eclipse E200)</li><li>pH meter (Thermo scientific, model: STARA1110)</li><li>37 Â°C Incubator (Fisher Scientific, model: isotemp)</li><li>Roller drum (Eppendorf, New Brunswick<sup>TM</sup>, model: TC-7)</li><li>4 Â°C refrigerator</li><li>-20 Â°C freezer</li><li>-80 Â°C freezer</li><li>250-ml flasks (Pyrex USA, catalog number: 4980)</li><li>Platform shaker</li><li>Weighing balance (Mettler Toledo, model: ML32002E/03 and AE163)</li><li>0.5-10 Î¼l Pipettor (USA Scientific, model: Ergo One, catalog number: 7100-0510)</li><li>2-20 Î¼l Pipettor (USA Scientific, model: Ergo One, catalog number: 7100-0220)</li><li>20-200 Î¼l Pipettor (USA Scientific, model: Ergo One, catalog number: 7100-2200)</li><li>100-1,000 Î¼l Pipettor (USA Scientific, model: Ergo One, catalog number: 7110-1000)</li><li>PIPETMAN<sup>Â®</sup> (JENCONS, model: Powerpippette plus)</li><li>Table-top centrifuge (Eppendorf, model: 5810R)</li><li>High-speed microcentrifuge (Eppendorf, model: 5418)</li><li>Clay Adams Nutator (Marshal Scientific, catalaog number: CANU)</li><li>Ultracentrifuge (Beckman Coulter, model: OptimaMax-TL ultra)</li><li>Ultracentrifuge fixed angle rotor (Beckman Coulter, model: TLA120.2)</li><li>ChemiDoc<sup>TM</sup> XRS<sup>+</sup> Imaging system (Bio-Rad)</li><li>Biowave Personal Cell Density Meter (Biochrome, catalog number: 5300-06)</li><li>Vortex Genie 2 (Scientific Industries, catalog number: SI-0236)</li><li>Mini-PROTEAN<sup>Â®</sup> Tetra Handcast Systems (Bio-Rad, catalog number: 1658004)</li><li>PowerPac Basic power supply (Bio-Rad, catalog number: 1645050)<br/></li></ol><p class="pptt" id="biaoti28106">### Software</p><ol class="npwtwok"><li>ImageLab (Bio-Rad)</li><li>PyMOL (version 2.0.1, Schrodinger)<br/></li></ol><p class="pptt" id="biaoti28107">### Procedure</p><p class="ppzz"><em>Note: The full protocol is described in Figure 3.</em><br/><br/><img alt="" class="layerphoto" height="188" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617030033_8633.jpg" width="100"/><br/><strong>Figure 3. Schematic step-by-step protocol for <em>in vivo</em> cysteine cross-linking using maleimide homo-bifunctional cross-linkers</strong> </p><br/><ol class="npwone"><li>Growth of <em>E. coli</em> cross-linking strains<br/><em>Note: The crystal structure of the inward-open state and the in-silico homology model predicting the outward-open state of the target protein guided us in the selection of residues to be replaced with cysteine using site-directed mutagenesis. We selected pairs of residues so that the distance between their CÎ± positions would be within the arm-length of homo-bifunctional cross-linking reagents when the target protein adopted only one of the two conformational states. Based on this strategy, we built </em><em>E. coli</em><em> cross-linking strains, each harboring a plasmid encoding a double-cysteine variant of the target protein. The detailed experimental rationale for the selection of residues for cysteine substitution and homo-bifunctional cross-linking reagents, and the construction of double cysteine variants can be found in our recent study (Kumar </em><em>et al.</em><em>, 2019).</em> <ol class="npwtwo"><li>Streak frozen stocks stored at -80 Â°C of <em>E. coli</em> <em>cross-linking</em> strains, each harboring a plasmid encoding a double-cysteine variant of the target protein (pET23/42FLAGMurJâcysThrombin derivatives were used in this protocol) (Kumar <em>et al.</em>, 2019), on LB agar plates supplemented with 125 Î¼g/ml of ampicillin.<br/><em>Note: The antibiotic ampicillin selects for plasmid maintenance, so replace it according to the plasmid system used.</em> </li><li>Incubate plates overnight at 37 Â°C.</li><li>Use an individual colony to inoculate 5 ml of LB medium supplemented with 125 Î¼g/ml of ampicillin and incubate overnight at 37 Â°C in a roller drum spinning at the top speed of 56 rpm.</li><li>Inoculate 50 ml of LB medium (in a 250 ml Erlenmeyer flask) with 1 ml of culture from Step A2 and grow the culture at 37 Â°C shaking at 200 rpm until it reaches an OD<sub>600</sub> of 1.<br/><em>Note: Appropriate antibiotics can be used while growing this secondary culture to maintain the plasmid but they are not necessary because of the short incubation.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of samples for cross-linking with homo-bifunctional cross-linkers<ol class="npwtwo"><li>Transfer the required volume of <em>E. coli</em> culture from Step A4 into a 50-ml centrifuge tube.<br/><em>Note: The required volume of </em><em>E. coli</em><em> culture (in ml) should equal to 10 multiplied by the number of treatments, where the number of treatments represents the number of different cross-linkers used plus one to be used without any cross-linker treatment (negative control).</em> </li><li>Centrifuge culture samples for 5 min at 2,057 <em>x g</em> in a centrifuge at room temperature.</li><li>Discard the supernatant and resuspend cell pellet with a volume of LB broth corresponding to 500 Î¼l multiplied by the number of treatments (<em>i.e.</em>, number of cross-linkers plus one).</li><li>Vortex samples to ensure proper mixing and transfer 500 Î¼l of each resuspended cell pellet into a new 1.7-ml microcentrifuge tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Crosslinking with homo-bifunctional cross-linkers<br/><em>Note: Prior to the experiment, remove stocks of all the cross-linkers from -20 Â°C and let them warm up to room temperature. In our experiments, we used 1,6-bis(maleimido)hexane (BMH; which has a flexible 16-Ã spacer arm length), 1,2-bis(maleimido)ethane (BMOE; which has a flexible 8-Ã spacer arm length), and N,Nâ²-(o-phenylene)dimaleimide (o-PDM; which has a rigid 6-Ã spacer arm length). We used maleimide cross-linkers because they are reactive in aqueous environments such as the periplasm and cytoplasm, and form thioether bonds with sulfhydryl groups of cysteines that are resistant to reversal by reducing agents (Figure 4) (Mulligan </em><em>et al.</em><em>, 2017). Perform all steps at room temperature unless stated otherwise.</em> <ol class="npwtwo"><li>Add 10 Î¼l of DMSO to the microcentrifuge tube containing resuspended cells that is labeled ânegative controlâ.<br/><em>Note: DMSO is used in this case because DMSO is the solvent used to dissolve the cross-linkers.</em> </li><li>To each of the remaining microcentrifuge tubes containing resuspended cells, add 10 Î¼l of the 50 mM stock of a cross-linker (BMH, BMOE, <em>o</em>-PDM) for a final concentration of 1 mM.<br/><em>Note: The final concentration of cross-linkers should be optimized as it may vary depending on the target protein.</em> </li><li>Vortex all samples.</li><li>Place sample tubes on a nutator placed at room temperature and cover them with aluminium foil.<br/><em>Note: Samples are covered with foil because some cross-linkers are light sensitive.</em> </li><li>Incubate samples in the dark for 5 min rocking on the nutator at room temperature.<br/><em>Note: The incubation time with the cross-linkers should be optimized as it may vary depending on the target protein.</em> </li><li>Add 50 Î¼l of 0.1 M L-cysteine (10 mM final concentration) to each sample.<br/><em>Note: Providing cysteine in excess titrates any unused cross-linker. The L-cysteine stock solution stored at -20 Â°C should be thawed to room temperature within an hour prior to addition. </em> </li><li>Vortex all samples.</li><li>Incubate for 5 min rocking on a nutator at room temperature.</li><li>Add 1 ml of 1x PBS to each sample.</li><li>Centrifuge for 1 min at 15,777 <em>x g</em> in a tabletop microcentrifuge.</li><li>Discard the supernatant and resuspend each cell pellet in 500 Î¼l of 50 mM Tris (pH 8).</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617030104_8506.jpg" width="198"/><br/><strong>Figure 4. Reaction between a maleimide cross-linker and the sulfhydryl group of a cysteine on a target protein.</strong> The maleimide moiety of the cross-linker reacts with the sulfhydryl group (-SH) present on cysteine residues to form a stable thioether bond that cannot be reversed by reducing agents. The R group, which varies among cross-linking agents, includes a spacer arm and another maleimide group.<br/><br/></li><li>Isolation of cytoplasmic membranes containing the target protein<ol class="npwtwo"><li>For spheroplast formation, transfer each sample from Step C11 to a new 15-ml centrifuge tube.<br/><em>Note: During spheroplast formation, the outer membrane and peptidoglycan cell wall of </em><em>E. coli</em><em> cells are destroyed, but their cytoplasmic membrane remains intact.</em> </li><li>Add 500 Î¼l of spheroplast buffer.<br/></li><li>Add 2.5 Î¼l of 100 mg/ml lysozyme stock.</li><li>Add 1 ml of dH<sub>2</sub>O.</li><li>Incubate for 15 min at room temperature. Gently vortex at low setting a couple of times for uniform mixing during the 15-min incubation.</li><li>Add 50 Î¼l of 1 M MgCl<sub>2</sub>.</li><li>Transfer 3 Î¼l of each sample to microscope glass slides and view sample under 40x objective to confirm spheroplast formation.<br/><em>Note: Spheroplasts appear as large bright round cells. Uniform spheroplast formation is necessary. If not, the incubation time at room temperature should be optimized.</em> </li><li>Collect spheroplasts by centrifugation at 2,057 <em>x g</em> for 5 min at 4 Â°C.</li><li>Discard the supernatant and resuspend spheroplasts in 500 Î¼l of ice-cold 50 mM Tris (pH 8).</li><li>Transfer each of the resuspended spheroplast samples into separate pre-chilled ultracentrifuge tubes.<br/><em>Note: This step should be performed on ice to minimized degradation of the target protein.</em> </li><li>Add 1 Î¼l of 25 U/Î¼l Benzonase stock.</li><li>Mix well by carefully pipetting up and down.</li><li>Centrifuge at 434, 902 <em>x g</em> for 1 h at 4 Â°C in an Optima-MAX-TL ultracentrifuge (Beckman Coulter) using a TLA120.2 rotor to pellet membranes.</li><li>Discard the supernatant and resuspend membrane pellets in 150 Î¼l of thrombin reaction buffer. </li><li>Add 15 Î¼l of 10% DDM stock (final concentration 1%).<br/><em>Note: 1% DDM is used for solubilization of our target protein. However, choice of detergent can vary depending on the target protein.</em> </li><li>Transfer resuspended membrane preparations to a pre-chilled 1.7-ml microcentrifuge tube. </li><li>Centrifuge at 15,777 <em>x g</em> for 10 min at 4 Â°C in a tabletop microcentrifuge to remove unsolubilized or aggregated proteins.</li><li>For each sample, transfer supernatant to a separate microcentrifuge tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Thrombin treatment<ol class="npwtwo"><li>For each sample in Step D18, transfer 50 Î¼l into two separate microcentrifuge tubes.</li><li>Add 4 Î¼l of 0.1 U/Î¼l thrombin stock (final concentration 0.4 U) to one of the microcentrifuge tubes of each treatment and leave the other one untreated.<br/><em>Note: Lyophilized thrombin powder should be first reconstituted as 0.1 U/Î¼l in thrombin reconstitution buffer before using. Store the stock at -20 Â°C. </em> </li><li>Incubate all samples for 20 h at room temperature.<br/><em>Note: Incubation time and temperature should be optimized to ensure cleavage of all target proteins.</em> </li><li>Add 50 Î¼l of 2x AB buffer to all samples.<br/><em>Note: This step stops the proteolytic reaction and prepares samples for electrophoresis. We used 2x AB buffer for our protein samples; however, other types of SDS-PAGE loading buffers could be used depending on the target protein.</em> </li><li>Briefly centrifuge at 15,777 <em>x g</em> for 10 s at room temperature in a tabletop centrifuge to collect samples at the bottom of the microcentrifuge tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>SDS-PAGE and immunoblotting<ol class="npwtwo"><li>Assemble a 10% SDS-PAGE gel in a Mini-PROTEAN<sup>Â®</sup> Tetra system.<br/><em>Note: The % of SDS-PAGE gel should be decided based on the molecular weight of the target protein and its thrombin proteolytic cleave pattern.</em> </li><li>Load 25 Î¼l of each sample from Step E5 on a 10% SDS-PAGE gel. When doing so, load each thrombin treated/untreated pair adjacent to each other for each treatment with or without a particular cross-linker.<br/><em>Note: Equivalent amounts of samples should be loaded on the gel because equal numbers of cells were collected at Step B4 after using equal volumes of cultures that had reached the same OD<sub>600</sub>. However, methods to measure protein concentration could also be used for normalization before loading the samples on a gel.</em> </li><li>Start electrophoresis at 120 V using 1x SDS-PAGE running buffer and stop after the visible molecular weight markers have migrated the desired distance.</li><li>Transfer proteins from the gel to a polyvinylidene difluoride (PVDF) membrane at 10 V for 2.5 h using a semidry transfer apparatus (Bio-Rad).<br/><em>Note: Transfer time and voltage should be optimized according to the target protein and % SDS PAGE gel.</em> </li><li>Block the PVDF membrane in TBST with 5% milk for 30 min at room temperature on a platform shaker.</li><li>Probe the PVDF membrane with the primary antibody.<br/><em>Note: For our target protein, we probed with anti-FLAG M2 (1:10,000; Sigma-Aldrich) in TBST with 5% milk by incubation at 4 Â°C on a shaker overnight.</em> </li><li>Wash three times with 1x TBST with 5 min incubation on a shaker at room temperature.</li><li>Probe the PVDF membrane with a secondary antibody.<br/><em>Note: For our target protein, we probed with anti-mouse horseradish-peroxidase conjugated antibodies (1:10,000; GE Healthcare) in 1x TBST with 5% milk by incubating for 1 h on a shaker at room temperature.</em> </li><li>Wash three times with 1x TBST with 5 min incubation on a shaker at room temperature.</li><li>Develop the signal using the Clarity Western ECL substrate according to the manufacturer's instructions (Bio-Rad) and detected using Image Lab program in a ChemiDoc XRS+ system (Bio-Rad).</li></ol></li></ol><p class="pptt" id="biaoti28108">### Data analysis</p><p class="ppzz">The described <em>in vivo</em> cysteine cross-linking experiments using maleimide-containing cross-linkers with different spacer arms should probe the proximity of substituted cysteine residues in both periplasmic- and cytoplasmic-facing residues in the N-lobe and C-lobe of the target protein and test for the presence of inward- and outward-open conformations (Figures 5A and 5B). The results from the proteolytic gel analysis should be analyzed as follows:</p><ol class="npwtwo"><li>In any sample lacking protease treatment, only full-length protein variants should be detected.</li><li>In protease-treated samples in which cross-linking did not occur because either no cross-linking agent was added or the two cysteines were too distant to be cross-linked, the target protein should be cleaved into two fragments. This should be reflected by the absence of the band corresponding to full-length protein after immunoblotting (Figures 5C and 5D). A gel with higher acrylamide percentage could be used to confirm the detection of the smaller tagged fragment (Figure 2A).</li><li>In protease-treated samples prepared from cells in which cysteine pairs were cross-linked by homobifuntional crosslinking reagents, the full-length protein should be detected even after protease cleavage because the two lobes should remain covalently linked by the cross-linkers (Figures 5C and 5D). This result should be interpreted to mean that the two cysteine residues came within the cross-linkable distance of a particular cross-linker during the transport cycle. The intensity of the band corresponding to the full-length protein should be proportional to the amount of cross-linking. Cross-linking efficiency between two cysteines could vary between cross-linkers, since it will depend on the distance between these residues and the length of the spacer-arm of the various cross-linkers.</li></ol><p class="ppzz">In our experiment, the presence or absence of full-length protein after cross-linking and protease treatment of variants containing specific cysteine substitutions in the periplasmic or the cytoplasmic side demonstrated the existence of two conformations: inward- and out-ward open conformations during the transport cycle for the target protein.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617030139_3418.jpg" width="187"/><br/><strong>Figure 5. <em>In vivo</em> double-cysteine cross-linking results. </strong>The two cysteines each located in the N lobe and C lobe of the target protein facing either the periplasm (P) or the cytoplasm (C) are cross-linked with maleimide homo-bifunctional cross-linkers only in one of two conformations: inward-open conformation for cysteines facing the periplasm (panel A) or outward-open conformation for cysteines facing the cytoplasm (panel B). C. After thrombin treatment, the full-length protein will not be visible after immunoblotting if cross-linking did not occur (mock blot on the left). If cross-linking occurred (mock blot on the right), the full-length protein will be detected even after the thrombin treatment. D. Results of the cross-linking experiments of the target protein with dimethyl sulfoxide (DMSO) as a negative control and three different maleimide homo-bifunctional cross-linkers of various spacer arm lengths (1,6-bis(maleimido)hexane (BMH, which has a flexible 16-Ã spacer arm length), 1,2-bis(maleimido)ethane (BMOE, which has a flexible 8-Ã spacer arm length), and <em>N</em>,<em>N</em>â²-(<em>o</em>-phenylene)dimaleimide (<em>o</em>-PDM, which has a rigid 6-Ã spacer arm length).</p><p class="pptt" id="biaoti28109">### Notes</p><ol class="npwtwok"><li>Our protocol could be applied to other transporter proteins that utilize the alternate-access mechanism. Moreover, this protocol could also be used with proteins that are not transporters in order to probe dynamic interactions between domains that might be subject to regulation.</li><li>Demonstration of the functionality of all the target protein variants should be tested prior to use since it is crucial to validate the cross-linking results.</li><li>It is necessary to check the expression level of the target protein variants by suitable methods before cysteine cross-linking.</li><li>It is highly recommended that negative controls are included to validate the technical details of the procedure. For example, a cysteine-less variant of the target protein, use of mono-cysteine variants, and double cysteine mutants facing opposite side to each other are recommended (Kumar <em>et al.</em>, 2019).</li><li>Try to prepare the stocks of maleimide homo-bifunctional cross-linker fresh each time for best results (repeated freezing and thawing of cross-linker may impact results).</li><li>We engineered a functional variant of the target protein containing a well-known protease site (thrombin). Another protease site such as factor Xa protease (Sun and Kaback, 1997) could also be used. However, care should be taken that the protease treatment does not cause non-specific cleavage of the target protein.</li><li>To ensure reproducibility, assays should be performed in triplicate.</li><li>Our protocol could also be used to test the effects of small-molecule inhibitors of transporters as we did to study the effect of protonophores on MurJ (Kumar <em>et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti28110">### Recipes</p><ol class="npwtwok"><li>Spheroplast buffer <br/>50 mM Tris-HCl, pH 8.0<br/>1 M sucrose<br/>2 mM EDTA</li><li>Thrombin-reaction buffer(store the buffer at 4 Â°C)<br/>50 mM Tris-HCl, pH 8.0 <br/>150 mM NaCl<br/>2.5 mM CaCl<sub>2</sub> </li><li>Thrombin-reconstitution buffer <br/>50 mM sodium citrate, pH 6.5<br/>200 mM NaCl, 0.1%PEG-8000, 50% glycerol<br/><em>Note: Use this buffer to reconstitute thrombin by generating a 0.1 U/Î¼l thrombin stock (store stock at -20 Â°C).</em> </li><li>Benzonase dilution buffer (store the buffer at -20 Â°C)<br/>50 mM Tris-HCl, pH 8.8<br/>2 mM MgCl<sub>2</sub> <br/>50% glycerol<br/><em>Note: Dilute Benzonase in this buffer to 25 U/Î¼l before using.</em> </li><li>2x AB Sample buffer<br/>6.84 mM Na<sub>2</sub>HPO<sub>4</sub><br/>3.16 mM NaH<sub>2</sub>PO<sub>4</sub><br/>50 mM Tris-HCl, pH 6.8<br/>6 M urea<br/>1% Î²-mercaptoethanol <br/>3% SDS <br/>10% glycerol <br/>0.1% bromophenol blue</li><li>1x PBS<br/>8 mM Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O, 2 mM NaH<sub>2</sub>PO<sub>4</sub>Â·2H<sub>2</sub>O, 131 mM NaCl in dH<sub>2</sub>O</li><li>10x SDS-PAGE running buffer<br/>30.0 g of Tris base, 144.0 g of glycine, and 10.0 g of SDS in 1,000 ml of dH<sub>2</sub>O</li><li>1x Transfer buffer (1 L)<br/>5.82 g Tris<br/>2.93 g glycine<br/>200 ml methanol<br/>800 ml dH<sub>2</sub>O</li><li>10x TBS Buffer<br/>a.Dissolve 24 g Tris and 88 g NaCl in 900 ml distilled water<br/>b.Adjust pH to 7.6 with 12 N HCl<br/>c.Add dH<sub>2</sub>O to a final volume of 1 L<br/><em>Note: Dilute to 1x when using.</em> </li><li>1x TBST (1x TBS + 0.1% Tween 20) (1 L)<br/>50 ml 20x TBS<br/>1 ml Tween 20<br/>950 ml dH<sub>2</sub>O</li><li>5% Milk TBST (25 ml)<br/>1.25 g dry milk<br/>25 ml 1x TBS + 0.1% Tween 20</li><li>1 M L-cysteine<br/>121 mg L-cysteine in 1 ml dH<sub>2</sub>O<br/><em>Note: Store stock at -20 Â°C in aliquots that should not be refrozen after thawing.</em> </li><li>Maleimide homo-bifunctional cross-linkers<br/>50 mM BMH: 6.875 mg BMH in 500 Î¼l of DMSO<br/>50 mM BMOE: 5.5 mg BMOE in 500 Î¼l of DMSO<br/>50 mM <em>o</em>-PDM: 6.7 mg <em>o</em>-PDM in 500 Î¼l of DMSO<br/><em>Note: Store stocks solutions at -20 Â°C wrapped in aluminum foil.</em><br/></li></ol><p class="pptt" id="biaoti28111">### Acknowledgments</p><p class="ppzz">We thank Rebecca M. Davis for helping in the preparation of this manuscript. This work was supported by grant GM1000951 from the National Institute of General Medical Sciences to N.R. Parts of some figures are adapted and modified from our previous study (Kumar <em>et al.</em>, 2019). <br/>  This protocol is adapted from Kumar, S., Rubino, F. A., Mendoza, A. G., and Ruiz, N. (2019). The bacterial lipid II flippase MurJ functions by an alternating-access mechanism. <em>The Journal of biological chemistry</em> 294, 981-990.</p><p class="pptt" id="biaoti28112">### Competing interests</p><p class="ppzz">The authors declare there are no potential conflicts of interest.</p><p class="pptt" id="biaoti28113">### References</p><ol class="npwtwok"><li>Butler, E. K., Davis R. M., Bari, V., Nicholson, P. A. and Ruiz, N. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23935042" target="_blank">Structure-function analysis of MurJ reveals a solvent-exposed cavity containing residues essential for peptidoglycan biogenesis in <em>Escherichia coli</em>.</a> <em>J Bacteriol</em> 195(20): 4639-4649.</li><li>Butler, E. K., Tan, W. B., Joseph, H. and Ruiz, N. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25225268" target="_blank">Charge requirements of lipid II flippase activity in <em>Escherichia coli</em>.</a> <em>J Bacteriol </em>196(23): 4111-4119.</li><li>Elofsson, A. and von Heijne, G. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17579561" target="_blank">Membrane protein structure: prediction versus reality.</a> <em>Annu Rev Biochem</em> 76: 125-140.</li><li>Hennon, S. W. and Dalbey, R. E. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24801559" target="_blank">Cross-linking-based flexibility and proximity relationships between the TM segments of the <em>Escherichia coli</em> YidC.</a> <em>Biochemistry</em> 53(20): 3278-3286.</li><li>Kuk, A. C., Mashalidis, E. H. and Lee, S. Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28024149" target="_blank">Crystal structure of the MOP flippase MurJ in an inward-facing conformation.</a> <em>Nat Struct Mol Biol</em> 24(2): 171-176.</li><li>Kumar, S., Rubino, F. A., Mendoza, A. G. and Ruiz, N. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30482840" target="_blank">The bacterial lipid II flippase MurJ functions by an alternating-access mechanism</a>. <em>J Biol Chem</em> 294(3): 981-990.</li><li>Meng, X., Clews, J., Martin, E. R., Ciuta, A. D. and Ford, R. C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30154098" target="_blank">The structural basis of cystic fibrosis.</a> <em>Biochem Soc Trans </em>46(5): 1093-1098.</li><li>Mulligan, C. and Mindell, J. A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28779840" target="_blank">Pinning down the mechanism of transport: probing the structure and function of transporters using cysteine cross-linking and site-specific labeling.</a> <em>Methods Enzymol</em> 594: 165-202.</li><li>Radchenko, M., Nie, R. and Lu, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26975373" target="_blank">Disulfide cross-linking of a multidrug and toxic compound extrusion transporter impacts multidrug efflux.</a> <em>J Biol Chem</em> 291(18): 9818-9826.</li><li>Rimon, A., Tzubery, T., Galili, L. and Padan, E. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12475238" target="_blank">Proximity of cytoplasmic and periplasmic loops in NhaA Na<sup>+</sup>/H<sup>+</sup> antiporter of <em>Escherichia coli</em> as determined by site-directed thiol cross-linking.</a> <em>Biochemistry</em> 41(50): 14897-14905.</li><li>Rubino, F. A., Kumar, S., Ruiz, N., Walker, S., and Kahne, D. E. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29558128" target="_blank">Membrane potential is required for murJ function.</a> <em>Journal of the American Chemical Society</em> 140(13):4481-4484.</li><li>Ruiz, N. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18832143" target="_blank">Bioinformatics identification of MurJ (MviN) as the peptidoglycan lipid II flippase in <em>Escherichia coli</em>.</a> <em>Proc Natl Acad Sci U S A</em> 105(40): 15553-15557.</li><li>Ruiz, N. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26792999" target="_blank">Lipid flippases for bacterial peptidoglycan biosynthesis.</a> <em>Lipid Insights.</em> 8 (Suppl 1): 21-31. </li><li>Ruiz, N. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27449418" target="_blank">Filling holes in peptidoglycan biogenesis of <em>Escherichia coli</em>.</a> <em>Curr Opin Microbiol</em> 34: 1-6.</li><li>Sham, L. T., Butler, E. K., Lebar, M. D., Kahne, D., Bernhardt, T. G., and Ruiz, N. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25013077" target="_blank">Bacterial cell wall. MurJ is the flippase of lipid-linked precursors for peptidoglycan biogenesis.</a> <em>Science</em> 345(6193): 220-2.</li><li>Sun, J. and Kaback, H. R. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9305990" target="_blank">Proximity of periplasmic loops in the lactose permease of <em>Escherichia coli</em> determined by site-directed cross-linking.</a> <em>Biochemistry</em> 36(39): 11959-11965.</li><li>Wiese, M. and Stefan, S. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30941807" target="_blank">The A-B-C of small-molecule ABC transport protein modulators: From inhibition to activation-a case study of multidrug resistance-associated protein 1 (ABCC1).</a> <em>Med Res Rev</em>. doi: 10.1002/med.21573.</li><li>Wu, J., Hardy, D. and Kaback, H. R. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10026249" target="_blank">Site-directed chemical cross-linking demonstrates that helix IV is close to helices VII and XI in the lactose permease.</a> <em>Biochemistry</em> 38(6): 1715-1720.</li><li>Zheng, S., Sham, L. T., Rubino, F. A., Brock, K. P., Robins, W. P., Mekalanos, J. J., Marks, D. S., Bernhardt, T. G. and Kruse, A. C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29891673" target="_blank">Structure and mutagenic analysis of the lipid II flippase MurJ from <em>Escherichia coli</em>.</a> <em>Proc Natl Acad Sci U S A</em> 115(26): 6709-6714.<br/></li></ol>