<body>
            <h1>Visualizing Filamentous Actin in <em>Chlamydomonas reinhardtii</em></h1>
            <p><strong>Authors:</strong> Evan W. Craig, Prachee  Avasthi</p>
            <h2>Abstract</h2>
            <p>This protocol aims to visualize the filamentous actin network in <em>Chlamydomonas reinhardtii</em>. We improved fixed-cell labeling conditions using the F-actin probe, phalloidin. We created a <em>Chlamydomonas</em>-optimized protocol by halving the phalloidin incubation time, electing for optimal fixation conditions, and selecting for a healthy cell population. This phalloidin protocol is quick, effective, and is the only labeling method to date that allows for reliable actin filament detection in fixed vegetative <em>Chlamydomonas</em> cells. This method reveals previously unidentified actin structures in <em>Chlamydomonas</em> and novel insights into cytoskeletal dynamics. </p>
            <p><strong>Keywords:</strong> <em>Chlamydomonas reinhardtii</em>, Actin, Phalloidin, Cytoskeleton, Filaments</p>
            <h2>Background</h2><p><em>Chlamydomonas reinhardtii</em> is a single-celled green alga widely used in studying photosynthesis, cilia, chloroplast biology, and the synthesis of biofuels. It is a leading model system in these fields due to the organismâs well-characterized two apical flagella, inexpensive and simple culture conditions, and a fully sequenced haploid genome allowing for excellent genetic studies. Understanding the <em>Chlamydomonas</em> cytoskeleton will play an important role in advancing these areas of research. However, the filamentous actin network of <em>Chlamydomonas reinhardtii</em> remains uncharacterized, partly due to the difficulties in visualizing filaments in this eukaryotic alga.<br/> <br/>Recently, there has been some success in detecting filamentous actin in <em>Chlamydomonas</em>. Through the expression of the fluorescently-tagged actin binding peptide, LifeAct, live-cell imaging revealed an actin-based perinuclear structure (Avasthi <em>et al.</em>, 2014; Onishi <em>et al.</em>, 2016). However, there is increasing evidence that LifeAct alters cell morphology, actin organization, actin-dependent functions, and preferentially labels a subset of actin structures (Courtemanche <em>et al.</em>, 2016, Flores <em>et al.</em>, 2019). Further, achieving stable expression is difficult in <em>Chlamydomonas</em> due to epigenetic silencing of exogenous genes (Cerutti <em>et al.</em>, 1997) and the need to colocalize actin probes with fixed cell organelle markers is essential for understanding the revitalized field of <em>Chlamydomonas</em> actin biology. Thus, an efficient and specific fixed-cell method of actin visualization would lead to significant advancement in the field and offer novel insights into basic actin biology.<br/> <br/>Available actin antibodies do not discriminate between monomeric and filamentous actin. It was previously shown that the widely used fluorescent actin probe phalloidin was ineffective in labeling filamentous actin in vegetative <em>Chlamydomonas</em> cells (Harper <em>et al.</em>, 1992). However, phalloidin does label an actin-dense structure in gametic cells known as the fertilization tubule (Detmers <em>et al.</em>, 1985). By optimizing phalloidin staining conditions, we are now able to visualize the <em>Chlamydomonas</em> actin network with exquisite detail in vegetative cells (Figure 1). Specifically, reducing the staining incubation time from 30 min to 16 min was essential in achieving optimal signal to noise which allowed for detection above <em>Chlamydomonas</em>âs auto-fluorescent chloroplast. Usage of a brighter and more photostable fluorophore, Atto 488, was also critical in obtaining a high signal to noise ratio and uniform labeling. This protocol allows us to understand how cell body actin filaments are distributed in gametic and vegetative cells for the first time. The previously unidentified filamentous actin structures will be crucial in exploring actin-dependent behaviors and how co-expressed actin genes in <em>Chlamydomonas</em> function to regulate cytoskeletal dynamics. Development of these strategies may prove useful for the study of actin filaments in other protists for which actin visualization has also been challenging.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617190932_3389.jpg" width="134"/><br/><strong>Figure 1. Phalloidin staining in wild-type vegetative <em>Chlamydomonas reinhardtii</em>.</strong> A. Representative raw single widefield image of specific filamentous actin staining using our optimized protocol. B. The deconvolved maximum intensity projection of a Z-stack (0.3 Î¼m steps) of the field visualized in A. C. Overlay of brightfield and fluorescence channels with phalloidin signal indicated in green. D. Deconvolved maximum intensity projection of a Z-stack (0.3 Î¼m steps) of a single cell demonstrating effective phalloidin staining. Scale bars = 5 Î¼m.</p>
        </body>

<p class="pptt" id="biaoti28128">## Materials and Reagents</p><ol class="npwtwok"><li>Kimwipes</li><li>Parafilm</li><li>Hydrophobic Marker (Elite PAP Pen, Diagnostic Biosystems Ref: K039)</li><li>1.5 ml Microcentrifuge Tubes</li><li>Columbia Jars (DWK Life Sciences Wheaton<sup>TM</sup> Columbia Jars for Coverslips, catalog number: 02-912-637)</li><li>10 Î¼l inoculating loop</li><li>Culture tubes (Borosilicate Glass Tubes 13 x 100 mm item) (Globe Scientific, catalog number: 1510)</li><li>Micropipette tips</li><li>Microscope slides</li><li>Coverglass (22 x 22 mm square, 0.13 mm to 0.16 mm thick) (Corning, catalog number: 2845-22)</li><li>Disposable Petri Dishes 60 x 15 mm (VWR<sup>Â®</sup>, catalog number: 25384-164)</li><li>Whatman<sup>Â®</sup> Qualitative Filter Paper, Grade 1 (Whatman, catalog number: 1001 125)</li><li><em>Chlamydomonas</em> Strains<br/><em>Note: The wild-type strain CC-125 mating type + was obtained from the </em><em>Chlamydomonas</em><em> Resource Center (University of Minnesota).</em> </li><li>Atto 488 Phalloidin (Sigma, catalog number: 49409), store at -20 Â°C, shield from light</li><li>Poly-L-lysine solution (Sigma, catalog number: P8920)</li><li>PBS Tablet (Research Products International PBS 100 ml Tablets SKU P32080-100T)</li><li>PFA aqueous solution 16% (Electron Microscopy Sciences, catalog number: 15710)</li><li>HEPES (Corning, catalog number: 61-034-RM)</li><li>NaOH pellets (J.T. Baker, catalog number: 3722)</li><li>HCl (Fischer Chemical, catalog number: A481-212)</li><li>Acetone (Histological) (Fisher Chemical, catalog number: A16P-4)</li><li>Fluoromount-G<sup>TM</sup> (Invitrogen, Ref: 00-4958-02)</li><li>DI Water</li><li>BD Biosciences Difco<sup>TM</sup> Agar, Granulated (Ref: 214530)</li><li>Hunterâs Trace Elements (Can be ordered from the <em>Chlamydomonas</em> Resource Center)</li><li>Acetic Acid, Glacial (Certified ACS) (Fisher Chemical, catalog number: A35S-500)</li><li>Methanol</li><li>Tris Base (C<sub>4</sub>H<sub>11</sub>NO<sub>3</sub>), Ultra Pure</li><li>Ammonium Chloride (NH<sub>4</sub>Cl)</li><li>Magnesium Sulfate Heptahydrate (MgSO<sub>4</sub>Â·7H<sub>2</sub>O)</li><li>Calcium Chloride Dihydrate (CaCl<sub>2</sub>Â·2H<sub>2</sub>O)</li><li>Potassium Phosphate Dibasic Anhydrous (K<sub>2</sub>HPO<sub>4</sub>)</li><li>Potassium Phosphate Monobasic (KH<sub>2</sub>PO<sub>4</sub>)</li><li>1x PBS (see Recipes)</li><li>4% PFA in 7.5 mM HEPES Buffer (see Recipes)</li><li>Phalloidin Stock Preparation (see Recipes)</li><li>TAP liquid media (Tris Acetate Phosphate Media) (see Recipes)<ol class="npwthree"><li>100x Tris buffer (see Recipes)</li><li>100x TAP Salts (see Recipes)</li><li>1000x Phosphate Solution (see Recipes)</li></ol></li><li>1.5% TAP Agar Plates (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28129">### Equipment</p><ol class="npwtwok"><li>Chemical spatula</li><li>Graduated cylinder</li><li>Beaker</li><li>Stir plate</li><li>Stir rod</li><li>pH meter</li><li>Scale</li><li>Tweezers</li><li>Pipettes</li><li>Gusto<sup>Â®</sup> High-Speed Mini Centrifuge (Heathrow Scientific, catalog number: HEA10050)</li><li>Empty glove boxes, cover to shield samples from light</li><li>Microscope for image acquisition (We use a Nikon Eclipse Ti-S equipped with a QiIMAGING QICAM)</li><li>Roller Drum (Cel Gro Tissue Culture Rotator) (Thermo Scientific, catalog number: 1640Q)</li><li>-20 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti28130">### Software</p><ol class="npwtwok"><li>Huygens Essential (Scientific Volume Imaging)</li><li>ImageJ (Schneider <em>et al.</em>, 2012)<br/></li></ol><p class="pptt" id="biaoti28131">### Procedure</p><ol class="npwone"><li>Cell culture and experimental setup<ol class="npwtwo"><li>Grow two inoculating loops of CC-125 mating type + cells, taken from a Tris Acetate Phosphate (TAP) agar plate, in 2 ml of tris acetate phosphate (TAP) liquid media on a roller drum (40 rpm) overnight (~16 h) in growth lighting at room temperature.</li><li>The next day, set up a parafilm workspace (Figure 2). Use a hydrophobic marker to draw a circle on the coverslip where you will eventually pipette the cells and all future media used in the experiment. Make sure to label coverslips appropriately for each trial, we find it helpful to write âFâ in the corner of each coverslip to help denote front from back. Add 200 Î¼l of Poly-L-Lysine at room temperature to the hydrophobic circle and wash after 10 min by quickly dipping in distilled water. The Poly-L-Lysine will immediately fall off the coverslip once it is submerged in water, quickly place coverslip back onto parafilm and use a Kimwipe to remove any excess moisture around the edges of the coverslip. Cover your workspace to shield from debris that might adhere to the Poly-L-Lysine coated coverslips. Unused Poly-L-Lysine coverslips can be stored at room temperature for future use. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617195619_5846.jpg" width="105"/><br/><strong>Figure 2. Parafilm Workspace</strong><br/><br/></li><li>It is essential to select healthy cells by centrifuging 1 ml of cell culture at 1,800 rpm (203 <em>x g</em>) for 1.5 min. Discard the supernatant and resuspend cells in 600 Î¼l fresh TAP media. Slowly pipette media up and down a few times then let culture sit. Healthy, swimming cells should migrate to the top of the culture after about 10 min. Then pipette (from the top of the culture) 200 Î¼l of resuspended cells to Poly-L-Lysine-coated coverslips for 5 min and shield samples from light. The cells are shielded from the light because they phototax and swim upward towards the light, reducing the number of cells that adhere to the coverslip.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell fixation and permeabilization<ol class="npwtwo"><li>After 5 min, tilt off liquid from the coverslips and replace with 200 Î¼l 4% fresh paraformaldehyde (PFA) in 7.5 mM HEPES pH 7.4 to the middle of the hydrophobic circle. Allow cells to incubate in fixative for 15 min at room temperature. <br/><em>Note: If non-specific chloroplast or bright pyrenoid signal is present (Figures 3A and 3B), dilute PFA solution fresh. We do not make up PFA from powder, but find by simply diluting fixative to 4% from an unopened ampule of 16% PFA solution improves labeling. We find after 2-3 weeks of PFA use, labeling efficiency decreases and makes pyrenoid signal very prominent. Diluting fixative from a fresh ampule alleviates this issue.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617195646_4988.jpg" width="146"/><br/><strong>Figure 3. Phalloidin staining using non-optimal conditions.</strong> A. If cells incubate too long in staining solution or PFA has expired, cells will show strong pyrenoid signal (yellow arrow) and non-specific cytoplasmic staining. This type of signal can be easily distinguished from Atto 488 signal as there will be no filamentous perinuclear or apical staining, instead a dim hazy signal found throughout the entire cell body as pictured in A and B above. Scale bars = 5 Î¼m. C. A general diagram of a <em>Chlamydomonas</em> cell highlighting filamentous actin in pink, the pyrenoid in blue, and the chloroplast in green. <br/><br/></li><li>Tilt of PFA solution into a Kimwipe and place coverslips in a Columbia jar containing 1x PBS to wash for 3 min. Use enough PBS so that the coverslip is completely submerged in liquid. </li><li>For cell permeabilization, submerge coverslips in a Columbia jar containing 80% pre-cooled acetone (diluted in water and stored at -20 Â°C) and then incubate for 5 min at -20 Â°C.</li><li>Quickly transfer coverslips into a second Columbia jar containing 100% pre-cooled acetone and incubate for another 5 min at -20 Â°C.</li><li>Place coverslips back on parafilm and allow them to air dry for a minimum of 2 min or longer if needed.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Phalloidin Stain<ol class="npwtwo"><li>Rehydrate cells by transferring coverslips to a Columbia jar containing 1x PBS and incubate for 5 min.</li><li>Place coverslips back on parafilm. Stain coverslips with Atto 488 Phalloidin (Sigma) for 16 min in the dark. This shorter than recommended staining time significantly reduces background and increases the signal to noise ratio.<br/><em>Note: The Atto 488 Phalloidin reagent greatly enhances photostability and brightness compared to Alexa-488, which allows for more uniform and reproducible filament labeling. Due to the photostability, Atto 488-labeled slides can be reimaged.</em> </li><li>Quickly tilt of staining solution and wash cells by transferring coverslips to a Columbia jar containing 1x PBS. Wash once for 5 min.</li><li>Remove excess liquid from the coverslip with a Kimwipe, but be careful to not disturb the inside of the hydrophobic circle (where the cells are adhered). Mount coverslips with self-sealing Fluoromount-G<sup>TM</sup> (Invitrogen) as quickly as possible. This sealant is designed for staining experiments where the final step is aqueous. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Image acquisition<br/>Capture 0.3 Î¼m step Z-stacks in brightfield and widefield fluorescence channels (GFP filter set) using a Nikon Eclipse Ti-S equipped with a QiIMAGING QICAM. Deconvolve fluorescence images using Huygens Essential deconvolution software and format in ImageJ. <br/></li></ol><p class="pptt" id="biaoti28132">### Data analysis</p><p class="ppzz">Hyugens Essential deconvolution software (Scientific Volume Imaging) was used for post-image analysis. Images were deconvolved using the following parameters:</p><ol class="npwtwo"><li>Algorithm: Classic MLE</li><li>PSF <br/>Mode: Theoretical <br/>Max Iterations: 10</li><li>Iteration Mode: Optimized </li><li>Quality Change Thresh. (%): 0.001</li><li>Signal to Noise Ratio: 20</li><li>Background Mode: Auto</li><li>Background Estimation Radius: 0.7</li><li>Relative Background: 0.0</li><li>Background Correction: If Possible</li><li>Brick Mode: Auto</li><li>PSFs per Brick Mode: Auto</li><li>PSFs per Brick, Manual Mode: 1</li></ol><p class="pptt" id="biaoti28133">### Recipes</p><ol class="npwtwok"><li>1x PBS<ol class="npwthree"><li>Dissolve 1 tablet of PBS in 80 ml of purified water and add up to 100 ml for a 1x solution. Stir until dissolved</li><li>1 Tablet contains 137 mM Sodium Chloride, 2.7 mM Potassium Chloride, and 11.9 mM Phosphate Buffer</li></ol></li><li>4% PFA in 7.5 mM HEPES buffer pH 7.4<ol class="npwthree"><li>PFA aqueous solution 16% (Electron Microscopy Sciences)</li><li>HEPES Stock Solution 10x (0.1 M, pH 7.4) (Farhat, 2013)<ol class="npwfour"><li>For 100 ml of 0.1 M HEPES, pH 7.4, add 2.38 g of HEPES to an appropriate beaker (100-200 ml in this case)</li><li>Add 80 ml of deionized water to the beaker</li><li>Add a stir bar to the beaker and leave it on a stir plate until completely dissolved (~1 min)</li><li>Add one NaOH pellet to raise pH towards 7.4. Adding one pellet will bring the pH to around 7</li><li>Once the first pellet is fully dissolved, add a second NaOH pellet if necessary, to raise the pH to 7.4. Monitor carefully, and if the pH approaches 7.3/7.4 before the pellet is fully dissolved, carefully remove the NaOH pellet with a clean spatula<br/><em>Note: In our experience, about 1.5 pellets are just the right amount to raise the pH to 7.4, so retrieving the second pellet is necessary for achieving the right pH.</em> </li><li>If the pH goes too high, lower it back to a pH of 7.4 by carefully adding a little HCl, while monitoring the pH <br/><em>Caution: Wear gloves, eye protection and exercise extreme caution with this acid solution.</em> </li><li>Once the pH of the solution is 7.4, add enough deionized water to raise the volume to 100 ml</li><li>Filter (if Whatman filter paper is used here), if possible, and store in the refrigerator for up to 4 months or aliquot and freeze at -20 Â°C for future use</li><li>For 50 ml, add 37.5 ml of 1x HEPES solution to 12.5 ml fresh 16% PFA for final concentration of 4% PFA in 7.5 mM HEPES buffer. Vortex lightly and store at 4 Â°C</li></ol></li></ol></li><li>Phalloidin stock preparation<ol class="npwthree"><li>Follow the manufacturerâs instructions and resuspend lyophilized phalloidin reagent in 0.5 ml pre-cooled methanol and store at -20 Â°C. Shield from light</li><li>To prepare working stock, add 5 Î¼l of phalloidin stock to 200 Î¼l of 1x PBS per coverslip. For example, if you need to stain 4 coverslips, add 20 Î¼l of the stock to 800 Î¼l of 1x PBS. Keep phalloidin stocks shielded from light at all times</li></ol></li><li>TAP liquid media (1 L)<ol class="npwthree"><li>Put ~800 ml deionized water into a beaker, flask, or bottle</li><li>Add the following solutions:<ol class="npwfour"><li>10 ml 100x tris buffer<br/>Dissolve 24.2 g tris (free base, not tris HCl) in a final volume of 100 ml H<sub>2</sub>O</li><li>10 ml 100x TAP salts<br/>Dissolve 18.75 g NH<sub>4</sub>Cl, 5 g MgSO<sub>4</sub>Â·7H<sub>2</sub>O (heptahydrate), and 2.5 g CaCl<sub>2</sub>Â·2H<sub>2</sub>O (dihydrate) in a final volume of 500 ml dH<sub>2</sub>O</li><li>1 ml 1,000x Phosphate Solution<br/>Dissolve 21.6 g K<sub>2</sub>HPO<sub>4</sub> and 10.8 g KH<sub>2</sub>PO<sub>4</sub> in a final volume of 200 ml dH<sub>2</sub>O</li><li>1 ml Hunter trace elements</li><li>1 ml glacial acetic acid</li></ol></li><li>Add deionized water to a total volume of 1 L</li><li>Test the pH of the solution. It should be between 7 and 7.3. If required, adjust pH using acetic acid. If pH is below 7 initially, start the process over from scratch</li><li>Autoclave and let cool before use</li></ol></li><li>1.5% TAP Agar Plates<ol class="npwthree"><li>Follow Recipe 4 a-d for preparing TAP liquid media</li><li>For 1.5% TAP plates, add 15 g of Difco<sup>TM</sup> Agar, granulated to the 1 L TAP solution</li><li>Add a stir bar and autoclave</li><li>Remove agar solution from autoclave and allow solution to cool. Stir the agar so it is evenly distributed throughout the container</li><li>Pour 30-40 ml of agar into each Petri dish. 1 L of agar should make ~2 sleeves of 1.5% TAP agar plates which can be stored in a 4 Â°C refrigerator or at room temperature for short term use</li></ol></li></ol><p class="pptt" id="biaoti28134">### Acknowledgments</p><p class="ppzz">Thank you to members of the Avasthi Lab for troubleshooting and manuscript feedback. This work was funded through P20GM104936 and P20GM103418 (to P.A.).</p><p class="pptt" id="biaoti28135">### Competing interests</p><p class="ppzz">We have no conflict of interest or competing interests to declare.</p><p class="pptt" id="biaoti28136">### References</p><ol class="npwtwok"><li>Avasthi, P., Onishi, M., Karpiak, J., Yamamoto, R., Mackinder, L., Jonikas, M. C., Sale, W. S., Shoichet, B., Pringle, J. R. and Marshall, W. F. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25155506" target="_blank">Actin is required for IFT regulation in <em>Chlamydomonas reinhardtii</em>.</a> <em>Curr Biol</em> 24(17): 2025-2032.</li><li>Cerutti, H., Johnson, A. M., Gillham, N. W. and Boynton, J. E. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9212467" target="_blank">Epigenetic silencing of a foreign gene in nuclear transformants of <em>Chlamydomonas</em>.</a> <em>Plant Cell </em>9(6): 925-945.</li><li>Courtemanche, N., Pollard, T. D. and Chen, Q. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27159499" target="_blank">Avoiding artefacts when counting polymerized actin in live cells with LifeAct fused to fluorescent proteins.</a> <em>Nat Cell Biol</em> 18(6): 676-683.</li><li>Detmers, P. A., Carboni, J. M. and Condeelis, J. (1985). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2415259" target="_blank">Localization of actin in <em>Chlamydomonas</em> using antiactin and NBD-phallacidin.</a> <em>Cell Motil</em> 5(5): 415-430.</li><li>Farhat, Y. (2013). <a href="http://protocol-place.com/basic-lab-techniques/stock-solutions/hepes-stock-solution-0-1-m-ph-7-4/" target="_blank">Reagent Preparation: HEPES Stock Solution (0.1 M, pH 7.4).</a> <em>Protocol Place Dec</em> 2013.</li><li>Flores, L. R., Keeling, M. C., Zhang, X., Sliogeryte, K. and Gavara, N. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30824802" target="_blank">Lifeact-GFP alters F-actin organization, cellular morphology and biophysical behaviour.</a> <em>Sci Rep</em> 9(1): 3241.</li><li>Harper, J. D., McCurdy, D. W., Sanders, M. A., Salisbury, J. L. and John, P. C. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1378775" target="_blank">Actin dynamics during the cell cycle in <em>Chlamydomonas reinhardtii</em>.</a> <em>Cell Motil Cytoskeleton</em> 22(2): 117-126.</li><li>Onishi, M., Pringle, J. R. and Cross, F. R. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26715672" target="_blank">Evidence that an unconventional actin can provide essential F-Actin function and that a surveillance system monitors F-Actin integrity in <em>Chlamydomonas</em>.</a> <em>Genetics</em> 202:977-996.</li><li>Schneider, C. A., Rasband, W. S. and Eliceiri, K. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22930834" target="_blank">NIH Image to ImageJ: 25 years of image analysis.</a> <em>Nat Methods</em> 9(7): 671-675. <br/></li></ol>