<body>
            <h1>A High-throughput qPCR-based Method to Genotype the SOD1<sup>G93A</sup> Mouse  Model for Relative Copy Number</h1>
            <p><strong>Authors:</strong> Valerie R. Tassinari, Fernando G. Vieira</p>
            <h2>Abstract</h2>
            <p>The most commonly used mouse model in ALS preclinical research expresses multiple copies of the human SOD1 (G93A) transgene. During the course of breeding, successive generations of mice can lose copies of the transgene. Because shorter lifespan of these mice is dependent on transgene copy number, it is essential to ensure that no low-copy, and therefore longer-lived, mice are included in preclinical studies. Existing techniques for SOD1<sup>G93A</sup> mouse genotyping are broadly based on creating a standard curve using a reference gene and deducing the relative amount of SOD1 by comparison with the standard curve. This type of technique is used in Alexander <em>et al.</em> (2004), Vieira <em>et al.</em> (2017) and Maier <em>et al.</em> (2018). However, it is not described in detail (see Note 1). This paper provides a detailed protocol for determining the relative copy number of the human SOD1 transgene. Briefly, the protocol involves first the extraction of high-quality genomic DNA from mouse ear tissue, creation of a genomic DNA concentration-based standard curve, and qPCR analysis of up to 88 samples at once alongside the standard curve with Gapdh as a reference gene. Analysis involves the normalization of each unknown sample using the standard curve followed by determination of the copy number of the sample relative to the cohort median. This protocol has been optimized to produce high-quality genomic DNA and consistent results, and the relative copy number cutoffs have been optimized and validated empirically by comparison of relative copy number and mouse lifespan.</p>
            <p><strong>Keywords:</strong> ALS mouse model, Mouse genotyping, SOD1, Transgene copy number, Preclinical research, High-throughput genotyping, Relative copy number</p>
            <h2>Background</h2><p>The SOD1<sup>G93A</sup> mouse model (B6SJL-Tg(SOD1*G93A)1Gur/J) is currently the most commonly used mouse model for preclinical testing of therapies for amyotrophic lateral sclerosis (ALS), a progressive neurodegenerative disease with most patients living between two and five years after diagnosis (Morrice <em>et al.</em>, 2018). The human SOD1 transgene is present in multiple copies and mouse lifespan inversely correlates with copy number. Alexander <em>et al.</em> (2004) found that mice with 24 copies of the SOD1(G93A) gene lived to approximately 129 days, whereas mice with 34 copies lived to 99 days and mice with 20 copies lived to 150 days (Alexander <em>et al.</em>, 2004). Additionally, 3.6% of high-copy transgenic male breeders had offspring with low copy numbers over the course of four years, signifying that it is not enough to only test stud males for copy number (Alexander <em>et al.</em>, 2004). Although higher copy does correlate with shorter lifespan, spontaneous recombination tends toward dropped copies rather than increased copy number (Alexander <em>et al.</em>, 2004; Zwiegers <em>et al.</em>, 2014). Therefore, we focused our assay on identifying low-copy mice only.<br/><br/>In order to be sure that any measurable effects in a preclinical drug study are attributable to the treatment and not to an inherent difference in transgene copy, it is absolutely essential that no low-copy mice are included. While there are no SOD1 (G93A) genotyping methods published in a dedicated method or protocol journal, there are papers which briefly describe genotyping in the methods section. For example, Alexander <em>et al.</em> (2004) describe a qPCR-based method that uses change in cycle threshold (âCt) of SOD1 (G93A) and a reference gene to quantify copy number difference (Alexander <em>et al.</em>, 2004).<br/><br/>In our hands, we found that published genotyping methods prior to 2016 were not sensitive enough to exclude all low-copy animals. Figure 1 shows the age at death of all mice included in a non-treatment group of survival studies from August 2015 to January 2019. The x-axis is the start date of each study, and the data set for each start date is the death ages of the mice included in the untreated group of that study. The finalized genotyping protocol detailed in this paper was instated on May 9, 2016, and the first mouse genotyped with this protocol was included in the study with the start date of June 7, 2016. The graph was analyzed in GraphPad Prism using a Box-and-Whiskers Tukey Test, which highlights any value that is either greater than the 75<sup>th</sup> percentile plus 1.5 times the interquartile range (IQR) or less than the 25<sup>th</sup> percentile minus 1.5 times the IQR. These outliers are shown as large blue dots. Since the genotyping protocol was instated, the number of outliers and their distance away from the median tend to decrease.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617230922_8158.jpg" width="125"/><br/><strong>Figure 1. GraphPad Prism analysis of mouse death ages from August 2015 to January 2019.</strong> These mice were in untreated groups of survival studies. The dates shown on the x-axis indicate the start date of each study, and the data set associated with each date represents death ages of the mice in the untreated group of each study. Tukeyâs Test on box-and-whisker plots has been applied, with the blue dots showing which data points are outside the 75<sup>th</sup> percentile plus 1.5 times the interquartile range (IQR) or the 25<sup>th</sup> percentile minus 1.5 times the IQR.<br/><br/>This genotyping protocol has been used in several published studies since 2016, including Vieira <em>et al.</em> (2017), and provides a qPCR-based method to ascertain relative copy number in a cohort of up to 88 mouse ear tissue samples without the need to determine absolute gene copy. The protocol relies on a concentration-based Gapdh standard curve to normalize the samples. After normalization, relative copy numbers are assigned based on log difference between each sample and the cohort median. Along with developing a reliable method to screen for low-copy mice, we optimized tissue collection to avoid cross-contamination and optimized genomic DNA extraction for consistent yield and high quality (see Note 2). Additionally, we also performed a dilution qPCR of potential reference gene primer and probe sets to determine which correlated best with the custom SOD1 target primer/probe set, and to determine the concentration range of our standard curve (Figure 2). As shown with the green triangles, the copy-number Gapdh primer/probe set from Thermo Fisher had the most comparable slope and Ct values to the SOD1 primer/probe sets, and both performed best between 2.5 and 0.07 ng/Î¼l genomic DNA. To determine the safest relative copy number cutoff, we performed a survival study correlating copy number with lifespan. Mice with 16 or more copies showed typical lifespans for this mouse model, while mice with 15 or fewer copies were long-lived (Figure 3). Using this information we determined the safest cutoff to be 15 copies and fewer.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617231008_9089.jpg" width="183"/><br/><strong>Figure 2. A qPCR experiment performed with serial two-fold dilutions of SOD1<sup>G93A</sup> genomic DNA from mouse ear tissue, probed with our custom SOD1 primer/probe set as well as several off-the-shelf primer/probe assays from Thermo Fisher.</strong> Our custom SOD1 primer/probe set compares well with the off-the-shelf SOD1 assay, and the Gapdh reference assay is closest to the SOD1 target assays in both Ct and in slope of the linear region. The graph is most linear between 0.07 and 2.5 ng/Î¼l of genomic DNA, which gave us the optimal range for our experimental standard curve.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617231055_8655.jpg" width="181"/><br/><strong>Figure 3. This graph shows the survival of SOD1<sup>G93A</sup> mice compared to their relative copy number as assigned by the genotyping protocol.</strong> The mice within a tolerable range of death ages were those in the 16-23+ copy number groups. Therefore, we selected 15 as the cutoff for relative copy number.</p>
        </body>

<p class="pptt" id="biaoti28151">## Materials and Reagents</p><p class="ppzz"><em>Note: All storage is at room temperature unless otherwise specified.</em> </p><ol class="npwtwok"><li>QIAcube HT Plasticware (QIAGEN, catalog number: 950067)</li><li>Adhesive PCR Plate Seals (Thermo Fisher, Thermo Scientific, catalog number: AB0558)</li><li>Sterile Alcohol Prep Pads (Fisher Scientific, Fisherbrand, catalog number: 22-363-750)</li><li>15 ml Centrifuge Tube (Corning, catalog number: 430766)</li><li>96-well UV-Transparent Microplates (Corning, catalog number: 3635)</li><li>384-well qPCR plate (Thermo Fisher, Applied BioSystems, catalog number: 4309849) </li><li>Custom SOD1 MGB probe and primers (Thermo Fisher, Custom Genomics, -20 Â°C)<ol class="npwthree"><li>Probe: SOD1HumanV2-55T (VIC-ACTCTCTCCAACTTTG) with VIC fluorescent reporter chemistry, 6000 pmole (wet, desalt) </li><li>Forward primer: SOD1HumanV2-27F (GTAAATCAGCTGTTTTCTTTGTTCAGA), 80,000 pmole (dry, desalt)</li><li>Reverse primer: SOD1HumanV2-104R (TTCACTGGTCCATTACTTTCCTTTAA), 80,000 pmole (dry, desalt)</li></ol></li><li>QIAamp 96 QIAcube HT Kit (QIAGEN, catalog number: 51331)</li><li>PBS, Phosphate Buffered Saline, 10x Solution (Fisher BioReagents, Fisher Scientific, catalog number: BP3994)</li><li>Alcohol, Reagent (Denatured Alcohol), 70% (v/v) Aqueous Solution (Ricca Chemical Company, Fisher Scientific, catalog number: 25467032, room temperature flammables cabinet)</li><li>Neosporin First Aid Antibiotic Ointment (Hannaâs Pharm Supply Co, catalog number: 302-571-8761)</li><li>Cotton-Tipped Applicators (Fisher Scientific, Puritan, catalog number: 22-029-571)</li><li>50 ml Reagent Reservoir (Corning, Costar, catalog number: 4870)</li><li>DL-Dithiothreitol solution BioUltra for molecular biology, 1 M in H<sub>2</sub>O (Millipore Sigma, Sigma-Aldrich, catalog number: 43816), 4 Â°C</li><li>Ethanol, Absolute (200 Proof), Molecular Biology Grade (Fisher Scientific, catalog number: BP2818100), room temperature flammables cabinet</li><li>Nuclease-Free Water (Thermo Fisher, Invitrogen, catalog number: AM9932)</li><li>TaqMan Genotyping Master Mix (Thermo Fisher, Applied Biosystems, catalog number: 4371355), 4 Â°C</li><li>TaqMan Gapdh Copy Number Assay (Thermo Fisher Assay ID Mm00186822cn), -20 Â°C</li><li>Sample Prep solution (see Recipes)</li><li>Tissue Digestion solution (see Recipes)</li><li>Primer Dilution solution (see Recipes)</li><li>SOD1<sup>G93A</sup> Copy Number Genotyping qPCR Master Mix (see Recipes)</li><li>Primer/Probe solution (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28152">### Equipment</p><ol class="npwtwok"><li>Two sterilized vessels for tissue harvest ethanol washes</li><li>Surgery-quality tools to harvest ear tissue (forceps and scissors, or ear punch)</li><li>Shaker capable of 18 h of 200 rpm and 56 Â°C incubation (Eppendorf, New Brunswick, catalog number: M1352-0010) (see Note 3)</li><li>Qiagen QIAcube HT (QIAGEN, catalog number: 9001793) (see Note 3)</li><li>SpectraMax M5 Multi-Mode Microplate Reader (Molecular Devices, catalog number: M5) (see Note 3)</li><li>NanoDrop<sup>TM</sup> 8000 Spectrophotometer (Thermo Fisher, Thermo Scientific, catalog number: ND-8000-GL) (see Note 3)</li><li>Allegra X-22R Benchtop Centrifuge (Beckman Coulter, catalog number 392188) (see Note 3)</li><li>QuantStudio<sup>TM</sup> 7 Flex Real-Time PCR System, 384-well, desktop (Thermo Fisher, catalog number: 4485701) (see Note 3)<br/></li></ol><p class="pptt" id="biaoti28153">### Software</p><ol class="npwtwok"><li>Excel 2013 (Microsoft)</li><li>Prism 6 (GraphPad, <a href="http://www.graphpad.com/" target="_blank">www.graphpad.com</a>)<br/></li></ol><p class="pptt" id="biaoti28154">### Procedure</p><ol class="npwone"><li>Tissue collection<br/><em>Note: Although this protocol specifies extracting and analyzing genomic DNA from ear samples, 20 mg mouse tail tip samples may also be used.</em> <ol class="npwtwo"><li>Preparation<ol class="npwthree"><li>Create a plate map with samples organized top to bottom in columns (<em>i.e.</em>, Sample 1 is in spot A1, Sample 2 is in spot B1, <em>etc.</em>). Remember that in every experiment, 8 sample spots need to be reserved for a standard curve, so the most mice that can be analyzed in a single qPCR is 88 (Figure 4A).</li><li>Create Sample Prep solution (Recipe 1) with appropriate excess (at least 10). </li><li>Using an S-Block from the Qiagen QIAcube Plasticware kit, load 160 Î¼l of the Sample Prep solution into every well of every column needed for the experiment (Figure 4B).<br/><em>Note: The QIAcube HT works in column format, and every well of the columns that are used (regardless of whether they contain a sample) needs to be filled in order for the automated protocol to work properly.</em> </li><li>Seal the top of the S-block with an adhesive plate seal.<br/><br/><img alt="" class="layerphoto" height="120" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000329_1357.jpg" width="100"/><br/><strong>Figure 4. Sample collection plate example.</strong> A. Sample collection plate map for an 88-sample experiment, with placeholder wells for the standard curve. B. Qiagen S-Block.<br/><br/></li></ol></li><li>Sample Collection<ol class="npwthree"><li>Fill two clean vessels with enough 70% reagent alcohol to wash tools.</li><li>Pre-clean tools with 70% reagent alcohol.</li><li>Collect an ear sample from each mouse that is 20 mg or less using scissors or ear punch (Figure 5); using forceps, place sample into the appropriate well according to the plate map.</li><li>Apply antibiotic ointment to mouse ear using cotton-tipped applicator.<br/><em>Note: The antibiotic swab does not usually need to be cleaned between uses, but if the mouse bleeds, we replace the soiled swab.</em> </li><li>Swish forceps and scissors in first alcohol wash, wipe with alcohol prep pad to remove any solid material such as hairs, and then swish in second alcohol wash. Leave tools in the second wash until ready to use again; do not mix up washes because the first wash should function as a âdirty washâ and the second should function as a âclean washâ.</li><li>Repeat with remaining mice, changing the alcohol prep pad and cleaning gloves often.</li><li>Cover S-block with adhesive when youâre done and during any extended breaks.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000401_1848.jpg" width="133"/><br/><strong>Figure 5. An ear tissue sample that is approximately 20 mg</strong><br/><br/></li></ol></li><li>Sample digestion<ol class="npwthree"><li>In reagent reservoir, create Tissue Digestion solution (Recipe 2) with appropriate excess (at least 10), mix well, and aseptically load 40 Î¼l into sample wells in S-block; load recipe into any âblankâ wells too (wells in a used column that do not have a sample).</li><li>Ensure that every sample is submerged.</li><li>Seal S-block with a thermally stable adhesive plate cover; add extra security with lab tape so that the adhesive cover does not pop off during overnight digestion (samples will evaporate if this happens).</li><li>Secure in a thermal shaker set to 56 Â°C; shake at 200 rpm overnight or up to 18 h.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>Genomic DNA extraction<ol class="npwtwo"><li>QIAcube HT setup<ol class="npwthree"><li>After overnight sample digest, open QIAcube HTâs QIAamp 96 protocol.</li><li>Follow the setup wizard in the QIAamp 96 QIAcube HT protocol to input sample number and make the following customizations.<ol class="npwfour"><li>Make elution volume 150 Î¼l (Figure 6A). <br/><em>Note: In our hands, 150 Î¼l elution produces sample yield between 10-50 ng/Î¼l, which we found to be the best range for sample dilution. You may need to change elution volume to optimize yield range. </em> </li><li>Remove Top Elute (change volume to 0) (Figure 6B).<br/><em>Note: With experimentation, we found that using Top Elute is messy and does not improve sample yield or quality.</em> </li></ol></li><li>Set up deck according to the specifications on the software program.</li><li>Deionize the deck with an ionizing gun or fan prior to running the protocol.<br/><br/><img alt="" class="layerphoto" height="145" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000436_2876.jpg" width="100"/><br/><strong>Figure 6. QIAcube setup.</strong> A. Set Elution Buffer to 150 Î¼l. B. Set Top Elute volume to 0 Î¼l.<br/><br/></li></ol></li><li>Storage: once the protocol is finished, cover the elution plate with adhesive tape and lid, label, and store at 4 Â°C until ready to use.<br/><em>Note: The kit instructs the user to use the rubber tube caps, but we find that to be cumbersome and have an increased chance of liquid flicking into another well.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Genomic DNA Dilution<ol class="npwtwo"><li>Use a plate reader or MultiDrop to take 260 nm, 280 nm, and 320 nm readings of the DNA (if using a plate reader, dilute gDNA 1:10 in water in a UV plate).</li><li>Quantify the concentration and 260/280 ratio; ensure that 260/280 values are approximately 1.8.<br/><em>Note: Another ratio that indicates sample purity is 260/230. Ideal 260/230 values are at least 1.0, but we have not found that 260/230 values have a significant effect on qPCR results so it is not further addressed in this protocol.</em> </li><li>Find the samples with the highest concentration; choose one dilution factor for the whole plate that ensures that the highest concentration sample does not exceed 1 ng/Î¼l and the lowest concentration sample is not lower than 0.02 ng/Î¼l.</li><li>Dilute all samples in PCR-clean water in a new plate and mix well.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Standard curve setup<ol class="npwtwo"><li>Combine and mix several samplesâ worth of high-quality genomic DNA made from several different SOD1<sup>G93A</sup> mice to a total of approximate 100 Î¼l of mixed sample.</li><li>Find the concentration (ideally, use a NanoDrop for most accurate results); take a few replicates to make absolutely sure.</li><li>Dilute the sample down to 2 ng/Î¼l in 80 Î¼l volume with PCR-clean water; save remaining mixed gDNA stock at -20 Â°C for future standard curves.</li><li>Starting with 70 Î¼l of 2 ng/Î¼l gDNA, make 8 serial two-fold dilutions in PCR-clean water in one column of a PCR plate; this creates a standard curve where the most-concentrated sample is 2 ng/Î¼l and the least is 0.016 ng/Î¼l (Figure 7).<br/><em>Note: You can make more than one standard curve at a time if you use them a lot, and store them at 4 Â°C.</em> </li><li>Cover with adhesive plate sealer and store at 4 Â°C. Use aseptic technique every time you use the standard curve.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000509_4306.jpg" width="107"/><br/><strong>Figure 7. Standard curve setup in a 96-well mixer plate</strong><br/><br/></li></ol></li><li>qPCR setup<ol class="npwtwo"><li>Primer setup<ol class="npwthree"><li>Dilute dry primers according to the âPrimer Dilution solutionâ (Recipe 3); leave on the bench for 10 min, vortex, and spin down.</li><li>Aliquot into 100 Î¼l aliquots, label, and store at -20 Â°C.</li><li>Aliquot wet probes into 20 Î¼l aliquots, label, and store at -20 Â°C.</li></ol></li><li>qPCR Master Mix<ol class="npwthree"><li>Make master mix according to the âSOD1<sup>G93A</sup> Copy Number Genotyping qPCR Master Mix solutionâ (Recipe 4), using appropriate excess (be sure to include standard curve in sample number calculations and make enough for two genes, SOD1 and Gapdh).</li><li>Create SOD1 and Gapdh mixes according to the âPrimer/Probe solutionâ (Recipe 5), using appropriate excess. Remember when calculating excess that you can use at most half of the number of excess samples that you used for the master mix for the SOD1 and Gapdh mixes. Mix gently; store at 4 Â°C covered with foil until ready to use (not more than 1-2 h).</li><li>Label 2 96-well PCR plates âSOD1â and âGapdhâ, respectively.</li><li>Aliquot 4 Î¼l of diluted unknown gDNA into each plate.</li><li>Add 4 Î¼l of standard curve after the last column of samples on the Gapdh plate.</li><li>Aliquot 20 Î¼l of SOD1 master mix to each sample well of the SOD1 plate; pipette mix very well but do not introduce bubbles. Repeat with Gapdh plate and master mix.</li><li>Spin plates at 180 <em>x g</em> for 3 min.</li><li>Starting with the SOD1 plate, pipette 9.8 Î¼l duplicates across a 384-well qPCR plate with an 8-channel pipette so that the samples occupy rows A, C, E, G, <em>etc.</em> Repeat with Gapdh directly under the SOD1 rows so that Gapdh samples occupy rows B, D, F, H, <em>etc.</em>; make sure to include standard curve (Figure 8).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000535_8033.jpg" width="124"/><br/><strong>Figure 8. A few example columns of a qPCR plate map in a 384-well plate.</strong> Gapdh standard curve samples would be added to the final two columns in duplicate after all samples have been added.<br/><br/></li><li>Spin at 400 <em>x g</em> for 3 min.</li><li>Seal the plate with an optical plate sealer and run a standard-length TaqMan qPCR (Figure 9).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000605_5139.jpg" width="352"/><br/><strong>Figure 9. Standard-length genotyping qPCR cycling conditions for qPCR machine setup</strong> </li></ol></li></ol></li></ol><p class="pptt" id="biaoti28155">### Data analysis</p><ol class="npwone"><li>Organize data<ol class="npwtwo"><li>Export the following data from the qPCR machine into Microsoft Excel: Raw Ct, Sample Name, Target Name.</li><li>Find the average raw Ct of each set of duplicates.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Find interpolated concentrations of unknowns using standard curve<ol class="npwtwo"><li>Open an XY spreadsheet in GraphPad Prism; choose âEnter X valuesâ and âEnter and plot a single Y value for each pointâ.</li><li>Label the X column âConcentration, ng/Î¼lâ and the first Y column âGapdh Ctâ.</li><li>In the first 8 rows of the âConcentrationâ, put the concentration values of the standard curve (2, 1, 0.5, 0.25, 0.125, 0.0625, 0.03125, 0.015625).</li><li>In the first 8 rows of the âGapdh Ctâ column, paste the raw average Ct values for your Gapdh standard curve.</li><li>Click âAnalyzeâ and choose âNonlinear Regressionâ (If you need to choose again, choose semilog lineâX is log, Y is linearâ). Make sure âInterpolate unknowns from standard curveâ is checked.</li><li>Ensure that R-squared value is at least 0.99 in the Table of Results tab.</li><li>Starting at Row 9, paste in raw average Gapdh Cts under âGapdh Ctâ column; a new Result tab will appear called âInterpolated X Valuesâ.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Normalize SOD1 Cts using interpolated concentrations<ol class="npwtwo"><li>Create a table in Excel with the following headings: Sample, Target Ct, Gapdh Concentration, Normalization Value, Normalization Coefficient, Log, Normalized SOD1 Ct.</li><li>Use Figure 10A to put appropriate values into each column.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Find relative copy number<ol class="npwtwo"><li>In a separate sheet, paste the Samples and their Normalized Target Ct values. Rank them from lowest to highest Ct using the Sort function, and assign them rank numbers starting at 1. If there are wild-types in your samples (Cts approaching 40), do not include them in this table. </li><li>Create a table in a separate Excel document with the following headings: Rank, Sample, Ct, Median, Ct-Median, Fold Change, Relative Copies.</li><li>Use Figure 10B to put appropriate values into each column using your new ranked table.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000637_8515.jpg" width="179"/><br/><strong>Figure 10. Excel table setup for Ct normalization and relative copy number calculation.</strong> A. Normalization of SOD1 Cts using interpolated Gapdh concentration. B. Relative copy number calculation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Make genotype calls<ol class="npwtwo"><li>As shown in the Background section, we found that 16 ârelative copiesâ and above causes mice to have a normal lifespan. We make the âlow copyâ call for mice with 15 relative copies and fewer.<br/><em>Note: While we found that 15 copies were a safe threshold, you can be as strenuous as you want. For larger and higher-powered efficacy studies it may be worth it to change the threshold to 18 or 20 to make absolutely sure that no lower copy mice are put into the study.</em> </li><li>If you are analyzing stud tails, be more conservative because this affects the future of your colony. We use 18 as the cutoff for stud tails.</li><li>It is worth keeping track of stud males for each cohort; if there is a large number of low copies in a cohort, they may all be the progeny of one low copy male.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Representative Data<br/>Please see the attached Excel document â<a href="https://os.bio-protocol.org/attached/file/20190326/Representative Data Spreadsheet 08152018.xlsx" target="_blank">Representative Data Spreadsheet 08152018</a>â for a representative set of data, including Excel formulas.<br/></li></ol><p class="pptt" id="biaoti28156">### Notes</p><ol class="npwtwok"><li>Although no genotyping protocol is described in this paper, the mice were genotyped using this SOD1<sup>G93A</sup> genotyping protocol.</li><li>There are many ways to arrive at high-quality genomic DNA; this method is written about our specific method, which has been optimized and customized for minimal cross-contamination,</li><li>This protocol was written with this piece of equipment in mind; however, you can use another piece of equipment with comparable function.<br/></li></ol><p class="pptt" id="biaoti28157">### Recipes</p><ol class="npwtwok"><li>Sample Prep solution<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000739_2135.jpg" width="600"/><br/><br/></li><li>Tissue Digestion solution<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000823_4493.jpg" width="600"/><br/><br/></li><li>Primer Dilution solution<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000902_2938.jpg" width="600"/><br/><br/></li><li>SOD1<sup>G93A</sup> Copy Number Genotyping qPCR Master Mix<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618000939_9925.jpg" width="600"/><br/><br/></li><li>Primer/Probe solution<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618001019_1225.jpg" width="600"/><br/></li></ol><p class="pptt" id="biaoti28158">### Acknowledgments</p><p class="ppzz">We acknowledge Matt Ferola and Carlos Maya for diligent and experienced mouse husbandry; we acknowledge Shawn Sullivan for his contribution to the acquisition of historical data. We thank Augieâs Quest for generous support of these studies. We thank people living with ALS for inspiring this research.</p><p class="pptt" id="biaoti28159">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti28160">### Ethics</p><p class="ppzz">All experiments were conducted in accordance with the protocols described by the National Institutes of Health Guide for the Care and Use of Animals and were approved by ALS TDIâs institutional animal care and use committee (IACUC). The IACUC ID is 2016-011 and the validity period is December 2016 through December 2019.</p><p class="pptt" id="biaoti28161">### References</p><ol class="npwtwok"><li>Alexander, G. M., Erwin, K. L., Byers, N., Deitch, J. S., Augelli, B. J., Blankenhorn, E. P. and Heiman-Patterson, T. D. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15519671" target="_blank">Effect of transgene copy number on survival in the G93A SOD1 transgenic mouse model of ALS.</a> <em>Brain Res Mol Brain Res</em> 130(1-2): 7-15.</li><li>Maier, M., Welt, T., Wirth, F., Montrasio, F., Preisig, D., McAfoose, J., Vieira, F. G., Kulic, L., Spani, C., Stehle, T., Perrin, S., Weber, M., Hock, C., Nitsch, R. M. and Grimm, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30518612" target="_blank">A human-derived antibody targets misfolded SOD1 and ameliorates motor symptoms in mouse models of amyotrophic lateral sclerosis.</a> <em>Sci Transl Med</em> 10(470). </li><li>Morrice, J. R., Gregory-Evans, C. Y. and Shaw, C. A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30323119" target="_blank">Animal models of amyotrophic lateral sclerosis: A comparison of model validity.</a> <em>Neural Regen Res </em>13(12): 2050-2054.</li><li>Vieira, F. G., Hatzipetros, T., Thompson, K., Moreno, A. J., Kidd, J. D., Tassinari, V. R., Levine, B., Perrin, S. and Gill, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30135932" target="_blank">CuATSM efficacy is independently replicated in a SOD1 mouse model of ALS while unmetallated ATSM therapy fails to reveal benefits.</a> <em>IBRO Rep</em> 2: 47-53.</li><li>Zwiegers, P., Lee, G. and Shaw, C. A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25103619" target="_blank">Reduction in hSOD1 copy number significantly impacts ALS phenotype presentation in G37R (line 29) mice: implications for the assessment of putative therapeutic agents.</a> <em>J Negat Results Biomed</em> 13: 14.<br/></li></ol>