<body>
            <h1>Tandem Affinity Purification of SBP-CBP-tagged Type Three Secretion System Effectors</h1>
            <p><strong>Authors:</strong> Laura  Berneking, Marie  Schnapp, Theresa  Nauth, Moritz  Hentschke</p>
            <h2>Abstract</h2>
            <p>Identification of protein-protein interactions of bacterial effectors and cellular targets during infection is at the core to understand how bacteria manipulate the infected host to overcome the immune response. Potential interacting proteins might be identified by genetic methods, <em>i.e.</em>, two hybrid screens and could be verified by co-immunoprecipitation. The tandem affinity purification (TAP) method allows an unbiased screen of potential interaction partners of bacterial effectors in a physiological approach: target cells can be infected with a bacterial strain harboring the TAP-tagged bacterial effector protein which is translocated in the host similar as under physiological infection conditions. No transfection and overexpression of the bacterial protein in the eukaryotic host are needed. Therefore, also host target cells not easy to transfect can be analyzed by this method. Moreover, the two consecutive affinity tags Calmodulin-Binding-Peptide (CBP) and Streptavidin-Binding-Peptide (SBP) fused to the translocated bacterial protein allow an outstanding clear purification of protein complexes formed between the bacterial protein of interest and host cell proteins with less occurrence of contaminants. Mass spectrometry allows an unbiased identification of interacting eukaryotic proteins.</p>
            <p><strong>Keywords:</strong> <em>Yersinia</em>, Affinity purification, Interaction, Effector protein, Type Three secretion system, TTSS</p>
            <h2>Background</h2><p>The interaction of bacterial effectors with eukaryotic proteins during infection is fundamental to understand the processes initiated by the bacteria to manipulate the host cell responses in order to establish infection. The type three secretion system (TTSS) is a needle-like structure found in several gram-negative bacteria that facilities direct secretion of bacterial effector proteins into the host cell. The tandem affinity purification (TAP) of TAP-tagged bacterial effector proteins secreted directly into the host cell via the TTSS provides the invaluable possibility to identify eukaryotic proteins interacting with bacterial effector proteins in a physiological approach. No transfection and overexpression of the bacterial protein in the eukaryotic host is needed once the tandem affinity tag labeled bacterial protein is translocated in the host cell cytoplasm during infection with the manipulated bacteria. The use of two affinity tags increases the specificity of the protein-protein interaction and ensures a low level background of nonspecific interactions. <br/> <br/>This method can be adapted to a wide range of gram-positive or gram-negative bacterial species and various host target cell types. We have utilized this assay to identify new interaction partners of the <em>Yersinia</em> effector protein YopM. In an approach with TAP-tagged YopM being translocated into the macrophage cell line J774A.1 via the natural type three secretion system of <em>Yersinia</em> we verified the interaction of YopM with host kinases PKN2 and RSK1. Moreover, we detected association with additional PKN and RSK isoforms and identified a new interaction partner of YopM, the DEAD box Helicase DDX3. This assay can be adapted to other bacterial species and host cell types.</p>
        </body>

<p class="pptt" id="biaoti28218">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (sarstedt, catalog numbers: 70762010 [1,000 Î¼l]; 70762012 [100 Î¼l] and 701116 [10 Î¼l])</li><li>Cell culture dishes 100 mm (sarstedt, catalog number: 83.3902)</li><li>Falcon tubes 15 ml (Sarstedt, catalog number: 62.544.502)</li><li>Micro tubes 1.5 ml (Sarstedt, catalog number: 72.690.001)</li><li>Cell scraper (Sarstedt, catalog number: 83.1830)<br/></li><li>J774A.1 cells (ATCC# TIB-67)</li><li><em>Yersinia enterocolitica</em> strain deltaYopM (<em>Yersinia enterocolitica</em> Serotype O:8 strain WA-314 with the YopM gene replaced by a kanamycin resistance cassette Trulzsch <em>et al.</em>, 2004)</li><li><em>Yersinia enterocolitica</em> strain deltaYopM(pYopM-CBP-SBP) (deltaYopM strain complemented with the DNA construct YopM-CBP-SBP in pACYC184; Hentschke <em>et al.</em>, 2010)</li><li>RPMI1640 (Gibco, catalog number: 61870-010)</li><li>Fetal bovine serum (FCS, Gibco, catalog number: 26140-079) </li><li>Penicillin-Streptomycin (Gibco, catalog number: 15140122)</li><li>L-Glutamine (Gibco, catalog number: A2916801)</li><li>pACYC184 vector (NEB; Hentschke <em>et al.</em>, 2010)</li><li>InterPlay Mammalian TAP System (Agilent, catalog number: 240103)</li><li>Bromphenol Blue (Serva, catalog number: 15375.02)</li><li>Coomassie brilliant blue (Merck, catalog number: 1154440025)</li><li>Calmodulin Sepharose 4B (GE Healthcare, Life, catalog number: 17052901) </li><li>Streptavidin Sepharose High Performance (GE Healthcare, Life, catalog number: 17511301)</li><li>Complete Protease inhibitor (Roche Diagnostics)</li><li>Methanol (Roth, catalog number: 8388.1)</li><li>Trypsin, sequencing grade modified (Promega, catalog number: V5111)</li><li>a-SBP antibody (Millipore, clone 20, catalog number: MAB10764)</li><li>Luria-Bertani broth (Roth, catalog number: X986.2)</li><li>Acetic acid (Roth, catalog number: T179)</li><li>SDS (Roth, catalog number: CN30.3)</li><li>Molecular weight marker (Molecular weight range 10-180 kDa; PageRuler Prestained Protein Ladder, Thermo Scientific, catalog number 26616)</li><li>Glycerol (Calbiochem, catalog number: 356350)</li><li>Î²-Mercaptoethanol (Sigma, catalog number: M3148-25ml)</li><li>Tris (Sigma, catalog number: T1503-1KG)</li><li>EDTA (Roth, catalog number: 80432)</li><li>Dithiothreitol (DTT; Roth, catalog number: 6908.2 259)</li><li>Iodacetamid (Sigma, catalog number: I6125-25g)</li><li>NH<sub>4</sub>HCO<sub>3</sub> (Sigma, catalog number: 09830)</li><li>Acetonitrile (Thermo, catalog number: 51101)</li><li>Formic acid (Fluka, catalog number: 94318)</li><li>Trifluor acetic acid (TFA; Roth, catalog number: CN30.3)</li><li>Coomassie Brilliant Blue R-250 (Serva, catalog number: 17524.01)</li><li>Bromphenol blue (Merck, catalog number: 8122)</li><li>NaCl (Roth, catalog number: 9265.1)</li><li>KCl (Merck, catalog number: 1049361000) </li><li>Coomassie staining solution (see Recipes)</li><li>Coomassie destain solution (see Recipes)</li><li>SDS-PAGE sample buffer (2x) (see Recipes)</li><li>PBS (10x) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28219">### Equipment</p><ol class="npwtwok"><li>Pipettes (Eppendorf, Eppendorf Research Plus, catalog number: 3123000900)</li><li>Power supplies (Bio-Rad, Bio-Rad power PC 200) </li><li>SDS-PAGE electrophoresis cell (Bio-Rad Mini-Protean II) </li><li>Cell counting chamber (Hartenstein, Neubauer-cell counting chamber)</li><li>Mammalian cell culture incubator (Binder, CB Series)</li><li>Shaking incubator (Sartorius, model Certomat BS-1)</li><li>Rotary mixer (Hartenstein, catalog number LD26/L263)<br/></li><li>Centrifuge (Eppendorf, model: 5417R) </li><li>Vacuum dryer (Speed Vac SC110, Savant)</li><li>MALDI-TOF mass spectrometer (Bruker Daltonics, REFLEX IV)<br/></li></ol><p class="pptt" id="biaoti28220">### Procedure</p><ol class="npwone"><li>Cloning of the CBP-SBP-tagged bacterial effector protein for bacterial expression<br/>The tandem-affinity-tag containing the Calmodulin-Binding-Peptide (CBP; Stofko-Hahn <em>et al.</em>, 1992) and the Streptavidin-Binding-Peptide (SBP; Keefe <em>et al.</em>, 2001) can be derived from the pCTAP-vector commercially available from the Interplay<sup>TM</sup> Tandem Affinity Purification system (Agilent). The cloning is carried out with standard molecular biology techniques. Our cloning strategy to create YopM-CBP-SBP in pACYC184 is described in detail in the original publication (Hentschke <em>et al.</em>, 2010). Briefly, the cloning can be performed as follows:<ol class="npwtwo"><li>Remove the stop codon of your bacterial effector of interest by PCR amplification for insertion of a C-terminal CBP-SBP-tag. The resulting PCR product should contain the putative endogenous promoter region of the effector protein of interest to ensure physiological expression levels.</li><li>Digest the PCR product and the pCTAP with appropriate restriction enzymes.</li><li>Ligate the effector protein into pCTAP.</li><li>Digest the resulting pCTAP effector protein construct with the appropriate enzymes to ligate it in the next step into a plasmid.</li><li>Ligate the pCTAP effector protein construct from Step A4 into a low copy vector. In our case, we used pACYC184 (Hentschke <em>et al.</em>, 2010).<br/><em>Note: Use a low copy plasmid to ensure physiological expression levels of the bacterial effector protein.</em> </li><li>This results in a vector harboring the bacterial effector protein C-terminally tagged with the two consecutive affinity tags CBP and SBP from the pCTAP vector (Figure 1). <br/><em>Note: TAP-tag can be cloned to the N-terminus using the pNTAP vector provided in the kit.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618011344_0780.jpg" width="144"/><br/><strong>Figure 1. Schematic illustration of the cloning procedure described in the protocol.</strong> (1) Remove the stop codon of your bacterial effector of interest by PCR amplification. Digest the PCR product and the pCTAP with appropriate restriction enzymes (A and B). (3) Ligate the effector protein into pCTAP. (4) To express the CBP-SBP-tagged effector protein of interest in the (investigated) bacterium, digest pCTAP construct âeffector-CBP-SBPâ (left) and a low copy cloning plasmid of your choice (suitable for the investigated organism) with appropriate restriction enzymes (C and D). (5) Ligate the construct âeffector-CBP-SBPâ from step 4 into a low copy cloning plasmid. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Construction of the bacterial strain harboring CBP-SBP-tagged effector protein<ol class="npwtwo"><li>Remove the endogenous effector protein from the bacterial strain you desire to investigate. In our case the YopM protein on the <em>Yersinia</em> virulence plasmid pYV had been replaced by a kanamycin resistance cassette (as described in more detail in the original publication; Trulzsch <em>et al.</em>, 2004) resulting in YopM deletion strain deltaYopM.</li><li>The effector deletion strain can be complemented with the plasmid harboring the TAP-tagged effector protein. In our case the <em>Yersinia enterocolitica</em> Serotype 0:8 strain WA-314 (Heesemann and Laufs, 1983), lacking the YopM was complemented with the pACYC184 vector expressing the YopM-SBP-CBP resulting in a strain named WA-314ÎYopM (pYopM-CBP-SBP) (Hentschke <em>et al.</em>, 2010).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Infection of target cells <br/>Infect the cells with the strain harboring the CBP-SBP-tagged effector protein for an appropriate time period.<ol class="npwtwo"><li>Prepare ten 100 mm culture dishes of cells (in total 10<sup>8</sup> cells). </li><li>Grow J774A.1 cells overnight at 37 Â°C and 5% CO<sub>2</sub> in RPMI1640 supplemented with 10% FCS, 100 IU/ml Penicillin and 100 Î¼g/ml Streptomycin and additional 2 mmol Glutamine. </li><li>Prepare overnight bacterial pre-cultures one day prior to the experiment. Briefly, streak out a colony from a plate and suspend the colony in 2 ml Luria-Bertani broth (LB) supplemented with appropriate antibiotics. Shake <em>Yersinia</em> strains overnight at 27 Â°C. </li><li>Dilute overnight culture 1:20 in fresh LB (40 ml in total). Grow <em>Yersinia</em> for another 2 h at 37 Â°C in order to induce activation of the <em>Yersinia</em> type III secretion machinery and expression of Yops. </li><li>Centrifuge the bacteria culture (4 Â°C, 5,000 <em>x g</em>, 10 min) and resuspend in ice-cold PBS.</li><li>Adjust bacterial suspension to OD<sub>600</sub> = 0.36.</li><li>Upon infection, remove the medium of J774A.1 cells and change to antibiotic-free RPMI1640. </li><li>Infect eukaryotic cells by adding 1 ml of adjusted bacterial suspension prepared in the previous Step E6 per 100 mm culture dish. 1 ml per 100 mm culture dish equals a multiplicity-of-infection of about 50:1 (50 bacteria per cells, Hentschke <em>et al.</em>, 2010).</li><li>Culture cells at 37 Â°C and 5% CO<sub>2</sub> during the infection time.<br/><em>Note: The J774A.1 cells were infected with </em><em>Yersinia</em><em> for 90 min. Time of infection might differ from cell type to cell type.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparing the protein extracts <br/>All protein purification steps should be performed at 4 Â°C to prevent the interacting proteins from dissociating. All amounts are indicated for a lysate volume of 10 ml.<ol class="npwtwo"><li>Wash cells after infection three times with ice-cold PBS supplemented with 1x complete Protease inhibitor (Roche).</li><li>Harvest cells by scraping off the cells with a cell scraper (Sarstedt) in 10 ml lysis buffer (1 ml per plate) supplied with the Interplay<sup>TM</sup> Tandem Affinity Purification kit (Agilent) supplemented with 1x complete Protease inhibitor (Roche) and transfer them to a 15 ml Falcon tube. Combine lysates of all 10 dishes in one 15 ml Falcon tube.</li><li>Lyse the cells by three successive freeze-thaw rounds in a -80 Â°C freezer for 20 min and 10 min incubation in cold water.</li><li>Clear the cells by centrifugation at 16,000 <em>x g</em> for 10 min at 4 Â°C.</li><li>Collect the supernatant in a new 15 ml Falcon tube.</li><li>Store an aliquot of 5 Î¼l Lysate at -20 Â°C for further analysis by Western blot.</li><li>Add 4 Î¼l 0.5 M EDTA and 0.7 Î¼l 14.4 M Î²-mercaptoethanol provided in the Interplay<sup>TM</sup> Tandem Affinity Purification kit (Agilent) per ml lysate.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Purification of the protein complexes using Streptavidin sepharose beads<ol class="npwtwo"><li>Prepare the Streptavidin binding buffer (SBB) by adding 7 Î¼l of 14.4 M Î²-mercaptoethanol (provided in the kit) and 1x complete Protease inhibitor (Roche) to 10 ml of SBB. Mix the buffer by inverting the tube and keep it on ice. The prepared buffer may be stored at 4 Â°C for up to two weeks.</li><li>Wash the Strepavidin resin slurry: centrifugate 500 Î¼l of the provided streptavidin sepharose at 1,500 <em>x g</em> for 10 min at 4 Â°C. (Five-hundred microliters of provided sepharose has 250 Î¼l resin plus 250 Î¼l ethanol storage buffer resulting in 500 Î¼l 50% slurry.)<br/></li><li>Remove the supernatant with a pipette.</li><li>Discard the supernatant to remove the ethanol storage buffer.<br/></li><li>Resuspend the resin in 1 ml SBB.</li><li>Wash the resin twice each with 1 ml SBB buffer by centrifugation and resuspention as described in Steps E2-E5. Use a new 1.5 ml tube for each washing step.</li><li>Collect the resin in 250 Î¼l SBB in a new 1.5 ml tube resulting in 500 Î¼l 50% slurry.</li><li>Add 500 Î¼l of the washed streptavidin 50% slurry to the lysate (Figure 2-1).</li><li>Incubate for 2 h at 4 Â°C under constant rotation at 10 rpm on a rotary mixer (Hartenstein).</li><li>Collect the sepharose by centrifugation at 1,500 <em>x g</em> for 5 min and remove and store the supernatant for further Western blot analysis in a 1.5 ml tube at -20 Â°C.</li><li>Wash the sepharose twice each with 1 ml SBB (Figure 2-2).</li><li>Use a new 1.5 ml tube for each washing step.</li><li>Collect the washed sepharose by centrifugation at 1,500 <em>x g</em> for 5 min and discard the supernatant. </li><li>Supplement the Streptavidin elution buffer (SEB) with 7 Î¼l of 14.4 M Î²-mercaptoethanol (provided in the kit) and 1x complete Protease inhibitor (Roche) to 10 ml of SEB. The SEB buffer must be kept at 4 Â°C and should be protected from light.</li><li>Add 500 Î¼l Streptavidin elution buffer (SEB) to the sepharose beads.</li><li>For Elution of the protein complex incubate for 30 min at 4 Â°C rotating.</li><li>Collect the sepharose by centrifugation at 1,500 <em>x g</em> for 5 min at 4 Â°C.</li><li>Transfer the supernatant in a new 1.5 ml tube.</li><li>Store 10 Î¼l of the supernatant for later Western blot analysis.</li><li>Prepare the Calmodulin binding buffer (CBB) by adding 7 Î¼l of 14.4 M Î²-mercaptoethanol (provided in the InterPlay Mammalian TAP System kit) and 1x complete Protease inhibitor (Roche) to 10 ml of CBB. Mix the buffer by inverting the tube. Keep the prepared CBB on ice.</li><li>Add 10 Î¼l streptavidin supernatant supplement and then 4 ml CBB to the supernatant.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Second purification step using Calmodulin sepharose beads<ol class="npwtwo"><li>Wash the Calmodulin sepharose as described in Steps E2-E6 with CBB supplemented with Î²-mercaptoethanol (InterPlay Mammalian TAP System kit) and 1x complete Protease inhibitor (Roche) (Figure 2-3). </li><li>Add the washed calmodulin sepharose to the eluate (from Step E17).</li><li>Incubate overnight at 4 Â°C under constant rotation.</li><li>Wash the calmodulin sepharose twice with CBB.</li><li>Store the supernatant for further Western blot analysis.</li><li>Boil the washed beads for 10 min in 60-100 Î¼l 2x SDS-loading buffer (Figure 2-3).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618011419_5480.jpg" width="406"/><br/><strong>Figure 2. Schematic illustration of the Tandem affinity purification AP system.</strong> This figure is modified from InterPlay Mammalian TAP System protocol (<a href="https://www.agilent.com/cs/library/usermanuals/public/240101.pdf" target="_blank">https://www.agilent.com/cs/library/usermanuals/public/240101.pdf</a>.) (1) Add the streptavidin sepharose to the cell lysate of infected cells containing the SBP-CBP-tagged effector protein with interacting partners. (2) Wash contaminants with Streptavidin binding buffer and add calmodulin sepharose. (3) Wash contaminants with Calmodulin binding buffer and elute your effector protein with potential interacting partners by boiling the Calmodulin sepharose beads in 2x SDS loading buffer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Identification of protein complexes <br/>The eluate can be subjected to SDS-PAGE followed by Coomassie staining to determine purification efficiency and allow mass-spectrometry analysis of visible bands.<br/><ol class="npwtwo"><li>Load the lysate control, the streptavidin sepharose supernatant (Eluate-SBP) and the boiled calmodulin sepharose supernatant (Eluate-CBP) on a 10% SDS-PAGE (Figure 3).</li><li>Separate the eluted proteins from Step F6 (50 Î¼l) by SDS-PAGE.</li><li>Stain the SDS gel in 10 ml Coomassie staining solution for 1 h.</li><li>Destain the SDS gel in 10 ml Coomassie destain solution for 2 h or overnight.</li><li>Cut out visible bands with a scalpel. Gel pieces size should be as small as possible.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618011513_3138.jpg" width="165"/><br/><strong>Figure 3. Identification of interacting proteins of the bacterial effector protein.</strong> A. Representative Western blot analysis to determine the purification efficiency of each step of the Tandem affinity purification. The presence of the tagged effector was detected with an antibody against the SBP (Millipore clone 20 #MAB10764). In this case, Tandem affinity purification was performed with cell lysate of WA-314ÎYopM (pYopM-CBP-SBP) infected J774 cells. B. Representative Coomassie stained SDS-PAGE including purification steps of the Tandem affinity purification. Visible bands (number 1 to 5) of the boiled CBP-beads need to be excised for further analysis by mass spectrometry.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Analyze visible bands by mass spectrometry<br/>The mass spectrometry was carried out with standard procedure techniques. Details of the identification of interacting proteins of the <em>Yersinia</em> effector protein YopM can be found in Hentschke <em>et al.</em> (2010).<ol class="npwtwo"><li>Cut the gel bands into small pieces.</li><li>Reduce the proteins with freshly prepared 10 mM DTT solved in a 100 mM ammonium bicarbonate stock solution at 56 Â°C for 30 min. Add enough DTT solution to cover the gel pieces completely.</li><li>Modify the cysteine residues with 55 mM iodacetamid solved in a 100 mM ammonium bicarbonate stock solution at ambient temperature for 20 min in the dark. Gel pieces should be covered completely.</li><li>Digest the protein in eluate with trypsin (5 ng trypsin/Î¼l in 50 mM NH<sub>4</sub>HCO<sub>3</sub>, 37 Â°C, 16 h).</li><li>Extract the peptides from the gel pieces by adding extraction solution containing 50% acetonitrile (ACN)/5% formic acid (v/v) in water. The volume of the extraction solution should be twice as much as the volume of the gel pieces. Incubate for 30 min on a mixer at 37 Â°C. </li><li>Collect the supernatant. </li><li>Add 100% ACN and incubate for 15 min on a mixer at 37 Â°C. </li><li>Collect the supernatant and pool it with the supernatant collected in the previous Step H6. </li><li>Add enough H<sub>2</sub>O to cover the gel pieces and incubate again for 15 min on a mixer at 37 Â°C. </li><li>Pool the supernatant with the supernatant of previous Step H8.</li><li>Add extraction solution to the gel pieces in a ratio of 2:1 and incubate for 30 min on a mixer at 37 Â°C. Unite the supernatant with the others collected.</li><li>The volume of the extraction solution should be twice as much as the volume of the gel pieces.</li><li>Dry down the extracts in a vacuum concentrator using medium to high drying rate at ambient temperature. Depending on the volume you have that step can take several hours.</li><li>Dissolve the extracts in an appropriate volume of 5% methanol/5% formic acid. <br/><em>Note: Extracts should be fully covered with solution.</em> </li><li>Mix 0.5 Î¼l of sample with an equal volume of matrix solution (saturated solution of cyano-4-hydroxycinnamic acid (HCCA) in 65% water/35% acetonitrile/0.1% trifluor acetic acid (TFA).</li><li>Apply the mixture onto a MALDI target by the dried-droplet method. </li><li>Determine peptide mass fingerprint data on a MALDI-TOF mass spectrometer (Bruker Daltonics) in reflector mode. Settings are dependent on the mass spectrometer available to you.</li></ol></li></ol><p class="pptt" id="biaoti28221">### Data analysis</p><p class="ppzz">Data analysis of the obtained peptide mass fingerprints of MALDI-TOF analysis can be performed with the Mascot search algorithm version 2.2 (Matrix Sciences, London, UK).</p><ol class="npwtwo"><li>Use the following parameter: mass tolerance: 50 ppm, one missed tryptic cleavage allowed, fixed modification: carbamidomethyl cysteine, variable modification: monooxidized methionine.</li><li>Limit searches to organism of infected cells, <em>e.g.</em>, <em>mus musculus</em>.</li><li>As a result, the expectation values can be matched to specific protein sequences to identify interaction partners of the effector protein.</li></ol><p class="pptt" id="biaoti28222">### Notes</p><ol class="npwtwok"><li>If antibody against your effector protein is not available, use the commercially available anti-SBP-tag-antibody (<em>e.g.</em>, Millipore, clone 20, catalog number: MAB10764).</li><li>If failed to get visible bands using Coomassie staining method, increase the number of cells and try silver staining of the SDS-PAGE.</li><li>If you encounter problems with unspecific interactions or contamination, adding a negative control (<em>i.e.</em>, pCTAP plasmid without the effector) to infect the target cells, would facilitate identifying unspecific binding of contaminants.</li><li>We applied electroporation to transform the plasmid harboring the TAP-tagged effector protein in the <em>Yersinia enterocolitica</em> deletion strain. It is important that the bacteria are in the exponential growth phase and that they are washed carefully to ensure that they are salt-free. (Conchas and Carniel, 1990)<br/></li></ol><p class="pptt" id="biaoti28223">### Recipes</p><ol class="npwtwok"><li>Coomassie staining solution<br/>0.1% (w/v) Coomassie Brilliant Blue R-250 (Serva)<br/>25% (w/v) Methanol (Roth)<br/>10% (w/v) Acetic acid (Merck)<br/>ddH<sub>2</sub>O</li><li>Coomassie destain solution<br/>25% (w/v) Methanol (Roth)<br/>10% (w/v) Acetic acid (Merck)<br/>ddH<sub>2</sub>O<br/></li><li>SDS-PAGE sample buffer (2x)<br/>0.5 M Tris (Sigma)<br/>125 mM SDS (Roth)<br/>10% (w/v) Glycerol (Calbiochem)<br/>2% Î²-Mercaptoethanol (Sigma)<br/>0.1% Bromphenol blue (Merck)<br/>ddH<sub>2</sub>O adjust to pH 6.8</li><li>PBS (10x)<br/>137 mM NaCl (Roth)<br/>2.7 mM KCl (Roth)<br/></li></ol><p class="pptt" id="biaoti28224">### Acknowledgments</p><p class="ppzz">This protocol is an extended version of the one described in the original papers (Hentschke <em>et al.</em>, 2010; Berneking <em>et al.</em>, 2016). Details of the cloning procedure and identification of interacting proteins of the <em>Yersinia</em> effector protein YopM can be found in the original papers.<br/>    We thank the UKE Core Facility Mass Spectrometry and Fritz Buck for help with experimental setup and data analysis.<br/>  This work was supported by grants from the Deutsche Forschungsgemeinschaft DFG (HE 5875/1-1) to MH and MS. LB was supported by the Deutsche Forschungsgemeinschaft (GRK-1459).</p><p class="pptt" id="biaoti28225">### Competing interests</p><p class="ppzz">The authors declare that there is no conflict of interest.</p><p class="pptt" id="biaoti28226">### References</p><ol class="npwtwok"><li>Berneking, L., Schnapp, M., Rumm, A., Trasak, C., Ruckdeschel, K., Alawi, M., Grundhoff, A., Kikhney, A. G., Koch-Nolte, F., Buck, F., Perbandt, M., Betzel, C., Svergun, D. I., Hentschke, M. and Aepfelbacher, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27300509" target="_blank">Immunosuppressive <em>Yersinia</em> effector YopM binds DEAD box helicase DDX3 to control ribosomal S6 kinase in the nucleus of host cells.</a> <em>PLoS Pathog </em>12(6): e1005660.</li><li>Heesemann, J. and Laufs, R. (1983). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6874644" target="_blank">Construction of a mobilizable <em>Yersinia enterocolitica</em> virulence plasmid.</a> <em>J Bacteriol </em>155(2): 761-767.</li><li>Hentschke, M., Berneking, L., Belmar Campos, C., Buck, F., Ruckdeschel, K. and Aepfelbacher, M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20957203" target="_blank"><em>Yersinia</em> virulence factor YopM induces sustained RSK activation by interfering with dephosphorylation.</a> <em>PLoS One</em> 5(10): e13165.</li><li>Keefe, A. D., Wilson, D. S., Seelig, B. and Szostak, J. W. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11722181" target="_blank">One-step purification of recombinant proteins using a nanomolar-affinity streptavidin-binding peptide, the SBP-Tag.</a> <em>Protein Expr Purif</em> 23(3): 440-446.</li><li>Stofko-Hahn, R. E., Carr, D. W. and Scott, J. D. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1318232" target="_blank">A single step purification for recombinant proteins. Characterization of a microtubule associated protein (MAP 2) fragment which associates with the type II cAMP-dependent protein kinase.</a> <em>FEBS Lett</em> 302(3): 274-278.</li><li>Trulzsch, K., Sporleder, T., Igwe, E. I., Russmann, H. and Heesemann, J. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15322017" target="_blank">Contribution of the major secreted yops of <em>Yersinia enterocolitica</em> O:8 to pathogenicity in the mouse infection model.</a> <em>Infect Immun</em> 72(9): 5227-5234.</li><li>Conchas, R. F. and Carniel, E. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2332165" target="_blank">A highly efficient electroporation system for transformation of <em>Yersinia</em>.</a> <em>Gene</em> 87(1): 133-137.<br/></li></ol>