<body>
            <h1>Immunoprecipitation and Sequencing of Acetylated RNA</h1>
            <p><strong>Authors:</strong> Daniel  Arango, David  Sturgill, Shalini  Oberdoerffer</p>
            <h2>Abstract</h2>
            <p>Generation of the epitranscriptome through chemical modifications of protein-coding messenger RNAs (mRNAs) has emerged as a new mechanism of post-transcriptional gene regulation. While most mRNA modifications are methylation events, a single acetylated ribonucleoside has been described in eukaryotes, occurring at the N4-position of cytidine (N4-acetylcytidine or ac4C). Using a combination of antibody-based enrichment of acetylated regions and deep sequencing, we recently reported ac4C as a novel mRNA modification that is catalyzed by the N-acetyltransferase enzyme NAT10. In this protocol, we describe in detail the procedures to identify acetylated mRNA regions transcriptome-wide using acetylated RNA immunoprecipitation and sequencing (acRIP-seq).</p>
            <p><strong>Keywords:</strong> N4-acetylcytidine, ac4C, acRIP-seq, Epitranscriptome, mRNA modifications</p>
            <h2>Background</h2><p>Chemical modifications of RNA have emerged as a new mechanism for the post-transcriptional regulation of gene expression (Zhao <em>et al.</em>, 2017). In contrast to a handful of well-studied modified bases that constitute the DNA epigenome, over a hundred modified ribonucleosides are present in all types of RNA. This diversity offers new possibilities to modulate gene expression, significantly expanding the metabolic and regulatory functions of RNA (Zhao <em>et al.</em>, 2017). Mainly studied in abundant transfer RNA (tRNA) and ribosomal RNA (rRNA), ribonucleoside modifications have more recently been described in messenger RNAs (mRNAs), where they form the basis of the epitranscriptome (Zhao <em>et al.</em>, 2017). These epitranscriptomic modifications have the potential to control all steps of mRNA metabolism including structure, stability, location and translation.<br/> <br/>Several modifications, including N6-methyladenosine (m6A), N6,2â-O-dimethyladenosine (m6Am), N1-methyladenosine (m1A), pseudouridine (Î¨), inosine (I), 5-methylcytidine (m5C), 5-hydroxylmethylcytidine (hm<sup>5</sup>C), 7-methylguanosine (m7G) and 2â-O-methylnucleosides (Nm) are found in eukaryotic mRNA and can influence the metabolism and function of mRNA (Li <em>et al.</em>, 2016). While the majority of mRNA modifications are methylation events, only a single acetylated ribonucleoside has been described in eukaryotes, occurring at the N4-position of cytidine (N4-acetylcytidine, ac4C, Figure 1A). ac4C is one of the universally conserved RNA modifications present in all domains of life and had previously been observed in rRNA and tRNA (Ito <em>et al.</em>, 2014a and 2014b; Sharma <em>et al.</em>, 2015). More recently, ac4C was observed in RNA viruses and in polyadenylated RNA (poly(A)-RNA) of yeast and human cancer cells (Dong <em>et al.</em>, 2016; McIntyre <em>et al.</em>, 2018; Tardu <em>et al.</em>, 2018), raising the possibility that ac4C is present in mRNA. In all cases, ac4C production was attributed to the enzyme NAT10 (Figure 1A), or its homologs (Ito <em>et al.</em>, 2014a and 2014b; Sharma <em>et al.</em>, 2015), suggesting a non-redundant activity.<br/> <br/>To investigate the occurrence of ac4C in human mRNA, we generated tools for antibody-based enrichment and transcriptome-wide mapping, a method we named acRIP-seq (Arango <em>et al.</em>, 2018). Additionally, to assess ac4C regulation and function, we deleted NAT10 in HeLa cells. The sum of these studies confirmed NAT10-dependent acetylation of mRNAs (Arango <em>et al.</em>, 2018). Overall, acetylation was enriched within coding sequences and loss of ac4C through NAT10 ablation led to a specific decrease in the expression of defined ac4C targets. More specifically, we found that ac4C increases mRNA stability and enhances translation efficiency of mRNAs when the modification is present in coding sequences (Arango <em>et al.</em>, 2018).<br/> <br/>Here, we report a detailed protocol for acetylated RNA immunoprecipitation and sequencing, acRIP-seq (Figures 1B-1C). This method is used to identify acetylated mRNA regions transcriptome-wide.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617223855_3913.jpg" width="102"/><br/><strong>Figure 1. acRIP-seq identifies acetylated regions within mRNA transcriptome-wide.</strong> A. NAT10 catalyzes cytidine acetylation. B-C. Schematics of acRIP-seq. Fragmented poly(A) RNA is immunoprecipitated (IP) with anti-ac4C antibodies or Isotypic IgG control (B). RNA from acRIP, IgG-IP and Inputs are used to construct Illumina sequences using the NEBNext<sup>Â®</sup> Ultra<sup>TM</sup> II Directional RNA Library protocol followed by high-throughput sequencing and bioinformatical analysis to identify putative acetylated regions transcriptome-wide (C). USER: Uracil-Specific Excision Reagent.</p>
        </body>

<p class="pptt" id="biaoti28164">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (RNase-free)</li><li>0.2 ml PCR tubes (RNase-free)<br/></li><li>1.5 ml microcentrifuge tubes (RNase-free)</li><li>2.0 ml microcentrifuge tubes (RNase-free)</li><li>NEBNext<sup>Â®</sup> Magnesium RNA Fragmentation Module (NEB, catalog number: E6150)<br/></li><li>NEBNext<sup>Â®</sup> Ultra<sup>TM</sup> II Directional RNA Library Prep Kit (NEB, catalog number: E7760S)</li><li>NEBNext<sup>Â®</sup> Multiplex Oligos for Illumina<sup>Â®</sup> (Index Primers Set 1, NEB, catalog number: E7335S)</li><li>RNase inhibitor, murine (NEB, catalog number: M0314S)</li><li>Agilent RNA 6000 kit (Agilent, catalog number: 5067-1511)</li><li>Agencourt AMPure XP Beads (Beckman Coulter, catalog number: A63881)</li><li>Agencourt RNAclean XP Beads (Beckman Coulter, catalog number: A63987)</li><li>MAXIscript T7 Transcription Kit (Thermo Fisher Scientific, catalog number: AM1312)</li><li>Protein G-Magnetic Dynabeads (Thermo Fisher Scientific, catalog number:10003D)</li><li>Dynabeads<sup>TM</sup> oligo(dT)<sub>25</sub> (Thermo Fisher Scientific catalog number: 61002)</li><li>Linear acrylamide (Thermo Fisher Scientific, catalog number: AM9520)</li><li>Acid-Phenol:Chloroform, pH 4.5 (Thermo Fisher Scientific, catalog number: AM9722)</li><li>Proteinase K, RNase-free (Thermo Fisher Scientific, catalog number: AM2546)</li><li>EDTA, RNase-free (Thermo Fisher Scientific, catalog number: AM9260G)</li><li>Sodium Acetate, RNase-free (Thermo Fisher Scientific, catalog number: AM9740)</li><li>Nuclease-free H<sub>2</sub>O (Thermo Fisher Scientific, catalog number: AM9938)</li><li>Tris-HCl, RNase-free (Thermo Fisher Scientific, catalog number: 15567027)</li><li>NaCl, RNase-free (Thermo Fisher Scientific, catalog number: AM9759)</li><li>SDS, RNase-free (Thermo Fisher Scientific, catalog number: AM9823)</li><li>Bovine Serum Albumin (BSA, Thermo Fisher Scientific, catalog number: AM2616)</li><li>Ethanol (Fisher Scientific, catalog number: BP2818100)</li><li>Chloroform (Fisher Scientific, catalog number: BP1145-1)</li><li>Sodium Phosphate monobasic (Sigma-Aldrich, catalog number: S0751)</li><li>Sodium Phosphate dibasic (Sigma-Aldrich, catalog number: S3264)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: 93443)</li><li>Rabbit monoclonal IgG isotype control (Cell Signaling Technologies, catalog number: 3900S)</li><li>Rabbit monoclonal anti-N4-acetylcytidine (Abcam, catalog number: ab252215) </li><li>ac4CTP (Trilink, catalog number: C-0001)</li><li>Purified poly(A) RNA</li><li>Acetylated RNA (see section C: <em>In vitro</em> transcription of acetylated RNA)</li><li>LiCl (Sigma-Aldrich, catalog number: L7026)</li><li>10x acRIP buffer (see Recipes)</li><li>Proteinase K buffer (see Recipes)</li><li>Poly(A) RNA binding buffer (see Recipes)</li><li>Poly(A) RNA washing buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28165">### Equipment</p><ol class="npwtwok"><li>Pipette 0.2 to 2 Î¼l (Thermo Fisher Scientific, catalog number: 4642010)</li><li>Pipette 2 to 20 Î¼l (Thermo Fisher Scientific, catalog number: 4642060)</li><li>Pipette 20 to 200 Î¼l (Thermo Fisher Scientific, catalog number: 4642080)</li><li>Pipette 100 to 1000 Î¼l (Thermo Fisher Scientific, catalog number: 4642090)</li><li>-20 Â°C freezer (General Electric, model FUM 21SVCRWW)</li><li>-80 Â°C freezer (Thermo Fisher Scientific, model Ultima Plus)</li><li>Thermal Cycler (Thermo Fisher Scientific, catalog number: 4484073)</li><li>ThermoMixer (Eppendorf, catalog number: 2231000574)<br/></li><li>Refrigerated Microcentrifuge (Eppendorf, catalog number: 5404000138) </li><li>Rotator RotoFlex (Denville Scientific, catalog number: H5700)</li><li>Magnetic rack for 1.5 ml tubes DynaMag<sup>TM</sup>-2 Magnet (Thermo Fisher Scientific, catalog number: 12321D)</li><li>Magnetic rack for 0.2 ml tubes DiaMag02 (Diagenode, catalog number: B04000001)</li><li>Nanodrop Spectrophotometer (Nanodrop, catalog number: ND-1000)<br/></li><li>Agilent 2100 Bioanalyzer (Agilent, catalog number: G293BA)</li><li>HiSeq 2500 Illumina sequencer (Illumina, model HiSeq 2500)<br/></li></ol><p class="pptt" id="biaoti28166">### Procedure</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwtwo"><li><em>The acRIP-seq protocol starts with purified RNA. We used poly(A) RNA to specifically map acetylated regions in mRNA. However, this procedure can be extended to any RNA class of interest. </em> </li><li><em>The steps described below are for one biological replicate. However, acRIP-seq should be performed in at least two biological replicates. A negative control consisting of ac4C-depleted RNA should be included. This can be accomplished by using poly(A) RNA from NAT10<sup>-/-</sup> cells (Arango </em><em>et al.</em><em>, 2018).</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwone"><li>Poly(A) RNA purification<ol class="npwtwo"><li>For poly(A) RNA purification, we used two rounds of poly(A) selection using Dynabeads<sup>TM</sup> oligo(dT)<sub>25</sub>. Precautions should be taken to avoid ribonuclease contamination during the isolation procedure.</li><li>Resuspend 1.8 mg of total RNA, obtained using the TRizol method (Rio <em>et al.</em>, 2010), in 2.4 ml of Nuclease-free H<sub>2</sub>O at a concentration of 750 Î¼g/ml. This amount of starting material is enough to obtain ~20 Î¼g of poly(A) RNA. </li><li>Resuspend the Dynabeads<sup>Â®</sup> Oligo (dT)<sub>25</sub> and transfer 1.2 ml (6 mg) of beads to a 2.0 ml microcentrifuge tube.</li><li>Magnetically separate the Dynabeads using a magnetic stand.</li><li>Wash once with 1.2 ml poly(A) RNA washing buffer. </li><li>Resuspend beads in 1.2 ml poly(A) RNA binding buffer.</li><li>Pipette 600 Î¼l of total RNA from Step A2 into a 1.5 ml microcentrifuge tube and heat at 75 Â°C for 2 min in a thermomixer set at 1,200 rpm.</li><li>Immediately add the entire 600 Î¼l of heated RNA to 1.2 ml beads from Step A6 to achieve a total volume of 1.8 ml.</li><li>Mix thoroughly and anneal through continuous rotation for 5 min at room temperature.</li><li>Magnetically separate the Dynabeads using a magnetic stand and carefully remove the supernatant.</li><li>Remove the tube from the magnet and add 1.2 ml poly(A) RNA washing buffer. </li><li>Repeat Steps A10-A11 for a total of three washes. </li><li>Add 600 Î¼l of 10 mM Tris-HCl [pH 7.5] and heat at 75 Â°C for 2 min in a thermomixer set at 1,200 rpm.</li><li>Add 1.2 ml poly(A) RNA binding buffer.</li><li>Mix thoroughly and anneal through continuous rotation for 5 min at room temperature.</li><li>Magnetically separate the Dynabeads using a magnetic stand and carefully remove the supernatant.</li><li>Remove the tube from the magnet and add 1.2 ml poly(A) RNA washing buffer. </li><li>Repeat Steps A16-A17 for a total of three washes. </li><li>Add 50 Î¼l of 10 mM Tris-HCl [pH 7.5] and heat at 75 Â°C for 2 min in a thermomixer set at 1,200 rpm. </li><li>Quickly place the tube in a magnetic stand and transfer the supernatant to a new 1.5 ml microcentrifuge tube. </li><li>Add another 50 Î¼l of 10 mM Tris-HCl [pH 7.5] to beads and heat at 75 Â°C for 2 min in a thermomixer set at 1,200 rpm.</li><li>Quickly place the tube in a magnetic stand and transfer the supernatant to the same tube from Step A20. </li><li>Repeat Steps A6 to A22 for a total of four times using the same set of beads. This procedure will process 2.4 ml of total RNA prepared in Step A2. After this step, the volume of poly(A) RNA will be 400 Î¼l.</li><li>Add 40 Î¼l of 3 M sodium acetate, pH.5.5, 3 Î¼l linear acrylamide, a coprecipitant to aid recovery of RNA during alcohol precipitation, and 1.1 ml of 100% ethanol. </li><li>Incubate at -20 Â°C overnight.</li><li>Centrifuge at 20,000 <em>x g</em> for 20 min at 4 Â°C in a microcentrifuge.</li><li>Carefully remove supernatant.</li><li>Wash pellet with 750 Î¼l of 70% ethanol.</li><li>Centrifuge at 20,000 <em>x g</em> for 5 min at 4 Â°C in a microcentrifuge.</li><li>Carefully remove ethanol.</li><li>Air dry pellet for 5 min at room temperature to remove residual ethanol.</li><li>Resuspend in 50 Î¼l Nuclease-free H<sub>2</sub>O.</li><li>Evaluate poly(A) RNA quality using the Agilent RNA 6000 kit and Bioanalyzer (Figure 2A, top).</li></ol><em>Stopping point: Samples can be stored at -80 Â°C.</em><br/><br/></li><li>RNA fragmentation<br/>To narrow the mapping of acetylated regions, input poly(A) RNA is fragmented prior to immunoprecipitation. The optimal fragment size range of RNA is between 100-200 nucleotides. This range of RNA sizes is achieved by divalent metal cations such as Mg<sup>2+</sup> or Zn<sup>2+</sup> at high temperatures. Divalent cations attack the 2â hydroxyl group of ribose ultimately cleaving the RNA strand with no sequence specificity (Shelton and Morrow, 1991; AbouHaidar and Ivanov, 1999).<ol class="npwtwo"><li>Mix 20 Î¼g of poly(A) RNA in 180 Î¼l of Nuclease-free H<sub>2</sub>O in a sterile PCR tube. This amount of poly(A) RNA will be split between IgG and anti-ac4C immunoprecipitations.</li><li>Add 20 Î¼l of 10x RNA Fragmentation Buffer from the NEBNext<sup>Â®</sup> Fragmentation Module.</li><li>For better performance of the fragmentation reaction, divide the solution into four PCR tubes (50 Î¼l each).</li><li>Incubate at 94 Â°C for 5 min in a pre-heated thermal cycler with lid set at 105 Â°C.</li><li>Immediately place PCR tubes on ice.</li><li>Add 5 Î¼l of 10x RNA Fragmentation Stop Solution from the NEBNext<sup>Â®</sup> Fragmentation Module to each PCR tube.</li><li>Transfer the fragmentation RNA reactions into one 1.5 ml microcentrifuge tube and add the following components:<br/><img alt="" class="layerphoto" height="194" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618024708_3296.jpg" width="500"/><br/>*We recommend the use of linear acrylamide rather than other co-precipitants such as glycogen. We have observed that glycogen can bind non-specifically to anti-ac4C antibodies. </li><li>Incubate at -20 Â°C overnight.<br/></li><li>Centrifuge at 20,000 <em>x g</em> for 20 min at 4 Â°C in a microcentrifuge.</li><li>Carefully remove supernatant.</li><li>Wash pellet with 750 Î¼l of 70% ethanol.</li><li>Centrifuge at 20,000 <em>x g</em> for 5 min at 4 Â°C in a microcentrifuge.</li><li>Carefully remove ethanol.</li><li>Air dry pellet for 5 min at room temperature to remove residual ethanol.</li><li>Resuspend in 50 Î¼l Nuclease-free H<sub>2</sub>O.</li><li>Evaluate fragmentation efficiency and concentration using the Agilent RNA 6000 kit and Bioanalyzer (Figure 2A, bottom).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618024818_1277.jpg" width="198"/><br/><strong>Figure 2. Bioanalyzer profiles.</strong> A. Representative profiles of unfragmented (top) and fragmented (bottom) poly(A) RNA. B. Representative profiles of RNA isolated from immunoprecipitation with IgG control (top) or anti-ac4C antibodies (bottom). C. Representative profiles of DNA libraries from Input (top), IgG (middle) or acRIP (bottom).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>In vitro</em> transcription of acetylated RNA<br/>To evaluate acRIP efficiencies between different conditions, an <em>in vitro</em> transcribed acetylated mRNA is spiked into input RNA prior to immunoprecipitation. <em>In vitro</em> transcription is performed using the MAXIscript T7 Transcription Kit. We use a 177 nt probe derived from the mouse Î²-globin gene with a nucleotide length within the range of poly(A) RNA after fragmentation (100-200 nt). The instructions described below are for the 177 nt probe derived from the mouse Î²-globin gene. However, any DNA sequence can be used for the same purpose. <br/>The sequence of probe: Probe can be purchased as Ultramer from Integrated DNA Technologies (IDT)<br/><span style="background-color:#CCCCCC;">TAATACGACTCACTATAGGG</span>GAGGAAGTAGTGAAGAGTGTTAGAGGATG<strong><span style="color:#E53333;">C</span></strong>TTGT<strong><span style="color:#E53333;">C</span></strong>AT<strong><span style="color:#E53333;">C</span></strong>A<span style="color:#E53333;"><strong>CC</strong></span>GAAG<span style="color:#E53333;"><strong>CC</strong></span>TGATT<span style="color:#E53333;"><strong>CC</strong></span>GTAGAG<span style="color:#E53333;"><strong>CC</strong></span>A<strong> <span style="color:#E53333;">C</span></strong>A<strong><span style="color:#E53333;">CCC</span></strong>TGGTAAGGG<span style="color:#E53333;"><strong>CC</strong></span>AAT<strong><span style="color:#E53333;">C</span></strong>TG<strong><span style="color:#E53333;">C</span></strong>T<strong><span style="color:#E53333;">C</span></strong>A<strong><span style="color:#E53333;">C</span></strong>A<strong><span style="color:#E53333;">C</span></strong>AGGATAGAGAGGG<strong><span style="color:#E53333;">C</span></strong>AGGAG<strong><span style="color:#E53333;">C</span></strong><strong><span style="color:#E53333;">C</span></strong>AGGG<span style="color:#E53333;"><strong>C</strong></span>AGAG<strong><span style="color:#E53333;">C</span></strong>ATATAAGGTGAGGTAGGAT<strong><span style="color:#E53333;">C</span></strong>AGTTGGATGTGGGAGTTGTAAGGTAGAATGTG<br/>Grey: T7 promoter<br/>Red: cytidines<ol class="npwtwo"><li>Mix the following components:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618024905_6254.jpg" width="300"/><br/>*ac4CTP can be purchased from Trilink.<br/><br/></li><li>Incubate at 37 Â°C for 1 h.</li><li>Add 1 Î¼l of DNase I (provided in the MAXIscript T7 Transcription Kit, see Materials and Reagents) and incubate at 37 Â°C for 20 min. </li><li>Add 44 Î¼l of Agencourt RNAclean XP beads and mix well.</li><li>Incubate for 10 min at room temperature.</li><li>Spin and place the tube on a magnetic rack for 5 min. Discard the supernatant.</li><li>Wash twice each with 200 Î¼l of freshly prepared 80% ethanol while the tubes are on the magnetic rack.</li><li>Briefly spin the tube, and replace in the magnetic rack.</li><li>Completely remove the residual ethanol, and air dry beads for 5 min while the tube is on the magnetic rack with the lid open.</li><li>Elute RNA with 21 Î¼l Nuclease-free H<sub>2</sub>O. Mix well by pipetting up and down, incubate for 2 min at room temperature. Put the tube in the magnetic rack until the solution is clear.</li><li>Without disturbing the bead pellet, transfer 21 Î¼l of the supernatant to a clean PCR tube.</li><li>Assess the quality of the RNA fragment in a denaturing agarose gel.</li><li>Measure concentration using the Nanodrop.</li><li>Mix 16.2 Î¼g fragmented poly(A) RNA from Step B15 with 16.2 pg of <em>In vitro</em> transcribed acetylated RNA from Step C13.</li><li>Save 200 ng (~1.2%) of material from Step C14 as Input. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Coupling antibodies to protein G Dynabeads<br/>acRIP is achieved with a rabbit monoclonal antibody against ac4C (Sinclair <em>et al.</em>, 2017) and isotypic rabbit monoclonal IgG as control.<ol class="npwtwo"><li>Magnetically separate 20 Î¼l (10 Î¼l for IgG and 10 Î¼l for acRIP) of Protein G-Magnetic Dynabeads using a magnetic stand.</li><li>Wash twice with 200 Î¼l 1x acRIP Buffer each.</li><li>Resuspend Dynabeads in 200 Î¼l 1x acRIP Buffer.</li><li>Divide Dynabeads into two sterile 1.5 ml microcentrifuge tubes:<br/>Tube 1: rabbit monoclonal IgG; 100 Î¼l of bead solution.<br/>Tube 2: rabbit monoclonal anti-ac4C antibody; 100 Î¼l bead solution.</li><li>Add 1 Î¼g of monoclonal IgG to Tube 1. Add 1 Î¼g of rabbit monoclonal anti-ac4C antibody to Tube 2.</li><li>Incubate for 1 h at room temperature in a rotator.</li><li>Magnetically separate the Dynabeads using a magnetic stand.</li><li>Wash twice with 200 Î¼l 1x acRIP Buffer.</li><li>Resuspend Dynabeads in 200 Î¼l 1x acRIP Buffer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA Immunoprecipitation<ol class="npwtwo"><li>Mix the following components:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618024955_1005.jpg" width="500"/><br/><br/></li><li>Magnetically separate Dynabeads from Step D9, discard the supernatant and add 100 Î¼l of RNA solution from Step E1 into each tube.</li><li>Incubate at 4 Â°C for 4 h in a rotator.</li><li>Magnetically separate Dynabeads using a magnetic stand.</li><li>Collect supernatants in a sterile microcentrifuge tube and save at -80 Â°C (optional).</li><li>Add 500 Î¼l of cold 1x acRIP buffer. Quickly flick tubes to resuspend beads.</li><li>Magnetically separate Dynabeads on ice using a magnetic stand and discard the supernatant.</li><li>Repeat Steps E6-E7 for a total of three washes.</li><li>After last wash, resuspend Dynabeads in 100 Î¼l 1x proteinase K buffer.</li><li>Incubate at 37 Â°C for 30 min in a Thermomixer. Shake at 1,200 rpm.</li><li>Separate magnetic Dynabeads and transfer the supernatant to a sterile 1.5 ml microcentrifuge tube.</li><li>Add 300 Î¼l of Nuclease-free H<sub>2</sub>O to Dynabeads. Flick tubes to resuspend beads.</li><li>Magnetically separate Dynabeads and collect supernatant into the same microcentrifuge tube from Step E11.</li><li>Add 400 Î¼l Acid-Phenol:Chloroform, pH 4.5. Vortex.</li><li>Centrifuge at 20,000 <em>x g</em> at 4 Â°C for 15 min in a microcentrifuge.</li><li>Transfer the upper clear layer to a new 1.5 ml microcentrifuge tube.</li><li>Add 400 Î¼l chloroform.</li><li>Centrifuge at 20,000 <em>x g</em> for 1 min.</li><li>Transfer the upper layer to a new 1.5 ml microcentrifuge tube.</li><li>Add the following components:<br/><img alt="" class="layerphoto" height="111" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025033_4890.jpg" width="350"/><br/><br/></li><li>Incubate at -20 Â°C overnight.</li><li>Centrifuge at 20,000 <em>x g</em> for 20 min at 4 Â°C in a microcentrifuge.</li><li>Carefully remove supernatant.<br/></li><li>Wash pellet with 750 Î¼l of 70% ethanol.</li><li>Centrifuge at 20,000 <em>x g</em> for 5 min at 4 Â°C in a microcentrifuge.</li><li>Carefully remove ethanol.</li><li>Air dry pellet for 5 min at room temperature to remove residual ethanol.</li><li>Resuspend in 6 Î¼l Nuclease-free H<sub>2</sub>O.</li><li>Evaluate concentration using the Agilent RNA 6000 kit and Bioanalyzer (Figure 2). At least 1 ng of precipitated RNA should be obtained and used in library preparation.</li></ol><em>Stopping point: Samples can be stored at -80 Â°C.</em><br/><br/></li><li>Library preparation<br/>Libraries are constructed using the NEBNext<sup>Â®</sup> Ultra<sup>TM</sup> II Directional RNA Library Prep Kit for Illumina<sup>Â®</sup> and following the manufacturerâs suggestions (Figure 1C). All reagents are supplied with the kit.<ol class="npwtwo"><li>Samples:<br/>Input: Step C15<br/>IgG: Step E28<br/>acRIP: Step E28</li><li>First strand cDNA synthesis<ol class="npwthree"><li>Set up the following priming reaction in PCR tubes and mix by gentle pipetting. Use equivalent amounts of RNA (~1 ng) for all samples:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025118_3613.jpg" width="270"/><br/><br/></li><li>Incubate the samples in a thermal cycler as follows:<br/>5 min at 65 Â°C, with heated lid set at 105 Â°C.<br/>Hold at 4 Â°C.</li><li>To the RNA from Step F2b (6 Î¼l) add the following components and mix by gentle pipetting:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025151_5887.jpg" width="360"/><br/><br/></li><li>Incubate samples in a preheated thermal cycler (with the heated lid set at 105 Â°C) as follows: <br/>10 min at 25 Â°C<br/>15 min at 42 Â°C<br/>15 min at 70 Â°C<br/>Hold at 4 Â°C.</li></ol></li><li>Second strand cDNA synthesis<ol class="npwthree"><li>To the tubes from Step F2d (20 Î¼l) add the following components and mix by gentle pipetting:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025236_5033.jpg" width="410"/><br/><br/></li><li>Incubate samples in a thermal cycler (with the heated lid set at 37 Â°C) for 60 min at 16 Â°C.</li><li>Add 144 Î¼l (1.8x) of resuspended AMPure XP Beads to samples from Step F3b.</li><li>Incubate for 5 min at room temperature.</li><li>Spin, place the tube on the magnetic stand for 5 min. Carefully remove and discard the supernatant.<br/></li><li>Wash twice each with 200 Î¼l of freshly prepared 80% ethanol while the tubes are on the magnetic rack.</li><li>Air dry the beads for 5 min while the tube is on the magnetic rack with lid open. </li><li>Elute into 53 Î¼l 0.1x TE buffer. Mix well by pipetting up and down and incubate for 2 min at room temperature.</li><li>Place the tube in the magnetic rack until the solution is clear.</li><li>Remove 50 Î¼l of the supernatant and transfer to a clean nuclease free PCR tube.</li></ol></li><li>Perform end prep of cDNA library<ol class="npwthree"><li>Mix the following components in a PCR tube:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025320_8443.jpg" width="400"/><br/><br/></li><li>Mix well by pipetting up and down at least 10 times.</li><li>Incubate samples in a thermal cycler (with the heated lid set at 75 Â°C) as follows:<br/>30 min at 20 Â°C<br/>30 min at 65 Â°C<br/>Hold at 4 Â°C.</li></ol></li><li>Adaptor ligation<ol class="npwthree"><li>Dilute the NEBNext Adaptor for Illumina 30-fold with Adapter Dilution Buffer prior to use.</li><li>Add the following components directly to the End Prep Reaction. <br/><em>Caution: Do not pre-mix the components to prevent adaptor-dimer formation.</em><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025358_5652.jpg" width="400"/><br/>*The NEBNext adaptor is provided in the NEBNext<sup>Â®</sup> Multiplex Oligos for Illumina<sup>Â®</sup>.<br/><br/></li><li>Mix well by pipetting up and down at least 10 times.</li><li>Incubate the sample in a thermal cycler (with the heated lid set at 75 Â°C) for 15 min at 20 Â°C.</li><li>Add 3 Î¼l of USER (uracil-specific excision reagent) Enzyme to the ligation mixture resulting in a total volume of 96.5 Î¼l. USER enzyme is provided in the NEBNext<sup>Â®</sup> Ultra<sup>TM</sup> II Directional RNA Library Prep Kit. </li><li>Mix well and incubate at 37 Â°C for 15 min with the heated lid set to â¥ 45 Â°C.</li><li>Add 96.5 Î¼l (1x) resuspended AMPure XP Beads and mix well. </li><li>Incubate for 10 min at room temperature.</li><li>Quickly spin, place on a magnetic rack for 5 min. Discard the supernatant.</li><li>Wash twice with 200 Î¼l of freshly prepared 80% ethanol while the tubes are on the magnetic rack.</li><li>Briefly spin the tube, and put the tube back in the magnetic rack.</li><li>Completely remove the residual ethanol, and air dry beads for 5 min while the tube is on the magnetic rack with the lid open.</li><li>Elute DNA with 16 Î¼l 0.1x TE buffer. Mix well, incubate for 2 min at room temperature. Put the tube in the magnetic rack until the solution is clear.</li><li>Without disturbing the bead pellet, transfer 15 Î¼l of the supernatant to a clean PCR tube.</li></ol></li><li>PCR Enrichment of Adaptor Ligated DNA<ol class="npwthree"><li>Mix the following components:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025450_5790.jpg" width="400"/><br/>*The Universal PCR Primer and index primers for Illumina are provided in the NEBNext<sup>Â®</sup> Multiplex Oligos for Illumina<sup>Â®</sup>. Use a different index primer for each sample.<br/><br/></li><li>PCR Cycling Conditions:<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190618/20190618025531_9852.jpg" width="400"/><br/><br/></li><li>Add 50 Î¼l (1x) resuspended AMPure XP Beads and mix well.</li><li>Incubate for 5 min at room temperature.</li><li>Spin and place the tube on a magnetic rack for 5 min. Discard the supernatant.</li><li>Wash twice each with 200 Î¼l of freshly prepared 80% ethanol while the tubes are on the magnetic rack.</li><li>Briefly spin the tube, and put the tube back in the magnetic rack.</li><li>Completely remove the residual ethanol, and air dry beads for 5 min while the tube is on the magnetic rack with the lid open.</li><li>Elute DNA with 23 Î¼l of 10 mM Tris-HCl. Mix well by pipetting up and down, incubate for 2 min at room temperature. Put the tube in the magnetic rack until the solution is clear.</li><li>Without disturbing the bead pellet, transfer 21 Î¼l of the supernatant to a clean PCR tube.</li><li>Assess library quality on an Agilent Bioanalyzer DNA Chip (Figure 2).</li></ol></li><li>Sequence using an Illumina sequencer in paired-end mode read length 125 bp (2 x 125 bp).<br/></li></ol></li></ol><p class="pptt" id="biaoti28167">### Data analysis</p><ol class="npwtwok"><li>Pre-process raw reads to remove low-quality bases and adaptor sequences by dynamic trimming or fixed cropping using Trimmomatic (Bolger <em>et al.</em>, 2014).</li><li>Map reads to the human genome (hg19) with Tophat2 v.2.1.1 (Trapnell <em>et al.</em>, 2009). Parameters used are: (i) reporting at most one alignment per read (-g 1); (ii) allowing maximally five mismatches per read (&gt; 95% matching); (iii) supplying the Ensembl Release 75 gene annotation; and (iv) performing a post-alignment filter to remove alignments to mitochondrial DNA (chrM) and non-concordant mate pairs.</li><li>Since sites of acetylation in transcripts will be non-contiguous in genomic coordinates, and difficult to assign to alternative isoforms, a representative transcript reference can be used for continuous peak calling. For example, canonical transcript sequences can be obtained from the UCSC genome browser (Rosenbloom <em>et al.</em>, 2015) and used in the generation of a Bowtie2-based index (Langmead and Salzberg, 2012). Quantify the number of bases in this reference for use in step 5. Reads can then be aligned in local alignment mode with Bowtie2 (Langmead and Salzberg, 2012).</li><li>Perform separate alignments to the spiked-in mouse Î²-globin probe sequence (see section C) and to the 43 kb human ribosomal DNA complete repeating unit (GenBank U13369.1) using Bowtie2 (Langmead and Salzberg, 2012), to specifically analyze reads originating from those features.</li><li>Perform peak calling with MACS2 (Zhang <em>et al.</em>, 2008), with parameters selected to optimize performance with transcript mapped reads [<em>i.e.</em>, turning off the shifting model (-nomodel) and local lambda (-nolambda), and supplying transcript reference bases (with -gsize) as the genome size]. Input samples should be used as controls for peak calling. In addition to peak calling within replicates, a set of reference peaks can be generated on pooled data, on which to apply filtering criteria (step 7).</li><li>Note that in the MACS2 peak calling output, each multi-base-pair peak includes â¥ 1 ââsummit,ââ or local peak maxima, defined at a single base position. Each of these summits is a putative ââac4C site.ââ The acRIP-seq approach is not a base-resolution method, so these ac4C sites are not required to be cytidines. </li><li>Apply stringent filtering to enforce replicability and remove artifacts such as non-specific binding, with a routine like the following: (i) Compare peaks to select only those sites with a reduction in signal in <em>NAT10</em><sup><em>-/-</em></sup> (non-acetylated control [Arango <em>et al.</em>, 2018]) as compared to parental cells, following the expectation that non-artifactual ac4C sites would show diminished signal when the enzyme is reduced. To acquire an enrichment value for comparison, use the bedtools map function (Quinlan, 2014) to extract the value from the MACS2 pileup output at the position of the summit called in parental cells. Select peak summits with pileup values higher in parental cells than <em>NAT10</em><sup><em>-/-</em></sup> cells to pass this filtering step. (ii) Remove peaks that result from non-specific IgG binding, by intersecting ac4C peaks with peaks called in IgG-IP, and keep only those with no coordinate overlap (Bedtools, [Quinlan, 2014]). (iii) Enforce detection in replicate experiments, by requiring peaks called in your reference set (pooled data) to overlap with peaks called in each individual replicate. (iv) Investigate the possibility of mapping errors, by spot-checking peak calls against genomically aligned reads to confirm concordance. <br/></li></ol><p class="pptt" id="biaoti28168">### Recipes</p><ol class="npwtwok"><li>10x acRIP buffer<br/>100 mM Na-Phosphate monobasic pH 7.0<br/>100 mM Na-Phosphate dibasic<br/>1.4 M NaCl <br/>0.5% Triton X-100<br/>1% BSA</li><li>1x Proteinase K buffer<br/>100 mM Tris-HCl, pH 7.5<br/>150 mM NaCl<br/>12.5 mM EDTA<br/>2% SDS<br/>5 mg/ml Proteinase K</li><li>Poly(A) RNA binding buffer<br/>20 mM Tris-HCl, pH 7.5<br/>1 M LiCl<br/>2 mM EDTA</li><li>Poly(A) RNA washing buffer<br/>10 mM Tris-HCl, pH 7.5<br/>0.15 M LiCl<br/>1 mM EDTA<br/></li></ol><p class="pptt" id="biaoti28169">### Acknowledgments</p><p class="ppzz">This work was supported by the Intramural Research Program of the National Institutes of Health (NIH), NCI, Center for Cancer Research. We thank the members of the Center for Cancer Research Sequencing Facility at the National Cancer Institute (Frederick, MD) for providing Illumina sequencing services. This protocol utilized the Biowulf Linux cluster at the NIH, Bethesda, MD (<a href="https://hpc.nih.gov/" target="_blank">https://hpc.nih.gov</a>).</p><p class="pptt" id="biaoti28170">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti28171">### References</p><ol class="npwtwok"><li>AbouHaidar, M. G. and Ivanov, I. G. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10488562" target="_blank">Non-enzymatic RNA hydrolysis promoted by the combined catalytic activity of buffers and magnesium ions.</a> <em>Z Naturforsch C </em>54(7-8): 542-548. </li><li>Arango, D., Sturgill, D., Alhusaini, N., Dillman, A. A., Sweet, T. J., Hanson, G., Hosogane, M., Sinclair, W. R., Nanan, K. K., Mandler, M. D., Fox, S. D., Zengeya, T. T., Andresson, T., Meier, J. L., Coller, J. and Oberdoerffer, S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30449621" target="_blank">Acetylation of cytidine in mRNA promotes translation efficiency.</a> <em>Cell</em> 175(7): 1872-1886 e1824. </li><li>Bolger, A. M., Lohse, M. and Usadel, B. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" target="_blank">Trimmomatic: a flexible trimmer for Illumina sequence data.</a> <em>Bioinformatics</em> (Oxford, England) 30: 2114-2120. </li><li>Dong, C., Niu, L., Song, W., Xiong, X., Zhang, X., Zhang, Z., Yang, Y., Yi, F., Zhan, J., Zhang, H., Yang, Z., Zhang, L. H., Zhai, S., Li, H., Ye, M. and Du, Q. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27246735" target="_blank">tRNA modification profiles of the fast-proliferating cancer cells.</a> <em>Biochem Biophys Res Commun</em> 476(4): 340-345.</li><li>Ito, S., Akamatsu, Y., Noma, A., Kimura, S., Miyauchi, K., Ikeuchi, Y., Suzuki, T. and Suzuki, T. (2014a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25086048" target="_blank">A single acetylation of 18 S rRNA is essential for biogenesis of the small ribosomal subunit in Saccharomyces cerevisiae.</a> <em>J Biol Chem</em> 289(38): 26201-26212. </li><li>Ito, S., Horikawa, S., Suzuki, T., Kawauchi, H., Tanaka, Y., Suzuki, T. and Suzuki, T. (2014b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25411247" target="_blank">Human NAT10 is an ATP-dependent RNA acetyltransferase responsible for N4-acetylcytidine formation in 18 S ribosomal RNA (rRNA).</a> <em>J Biol Chem</em> 289(52): 35724-35730. </li><li>Langmead, B. and Salzberg, S. L. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22388286" target="_blank">Fast gapped-read alignment with Bowtie 2.</a> <em>Nat Methods</em> 9(4): 357-359.</li><li>Li, X., Xiong, X. and Yi, C. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28032622" target="_blank">Epitranscriptome sequencing technologies: decoding RNA modifications.</a> <em>Nat Methods </em>14(1): 23-31. </li><li>McIntyre, W., Netzband, R., Bonenfant, G., Biegel, J. M., Miller, C., Fuchs, G., Henderson, E., Arra, M., Canki, M., Fabris, D. and Pager, C. T. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29373715" target="_blank">Positive-sense RNA viruses reveal the complexity and dynamics of the cellular and viral epitranscriptomes during infection.</a> <em><span id="__kindeditor_bookmark_end_160__" style="display:none;"></span>Nucleic Acids Res </em>46(11): 4776-5791. </li><li>Quinlan, A. R. (2014). BEDTools: <a href="http://www.ncbi.nlm.nih.gov/pubmed/25199790" target="_blank">The Swiss-army tool for genome feature analysis.</a> <em>Curr Protoc Bioinformatics </em>47: 11.12.1-34.</li><li>Rio, D. C., Ares, M., Jr., Hannon, G. J. and Nilsen, T. W. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20516177" target="_blank">Purification of RNA using TRIzol (TRI reagent).</a> <em>Cold Spring Harb Protoc</em> 2010(6): pdb prot5439.</li><li>Rosenbloom, K. R., Armstrong, J., Barber, G. P., Casper, J., Clawson, H., Diekhans, M., Dreszer, T. R., Fujita, P. A., Guruvadoo, L., Haeussler, M., Harte, R. A., Heitner, S., Hickey, G., Hinrichs, A. S., Hubley, R., Karolchik, D., Learned, K., Lee, B. T., Li, C. H., Miga, K. H., Nguyen, N., Paten, B., Raney, B. J., Smit, A. F., Speir, M. L., Zweig, A. S., Haussler, D., Kuhn, R. M. and Kent, W. J. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25428374" target="_blank">The UCSC Genome Browser database: 2015 update.</a> <em>Nucleic Acids Res</em> 43(Database issue): D670-681. </li><li>Sharma, S., Langhendries, J. L., Watzinger, P., Kotter, P., Entian, K. D. and Lafontaine, D. L. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25653167" target="_blank">Yeast Kre33 and human NAT10 are conserved 18S rRNA cytosine acetyltransferases that modify tRNAs assisted by the adaptor Tan1/THUMPD1.</a> <em>Nucleic Acids Res </em>43(4): 2242-2258. </li><li>Shelton, V. M. and Morrow, J. R. (1991). <a href="https://pubs.acs.org/doi/abs/10.1021/ic00023a003" target="_blank">Catalytic transesterification and hydrolysis of RNA by zinc(II) complexes.</a> <em>Inorganic Chemistry</em> 30(23): 4295-4299. </li><li>Sinclair, W. R., Arango, D., Shrimp, J. H., Zengeya, T. T., Thomas, J. M., Montgomery, D. C., Fox, S. D., Andresson, T., Oberdoerffer, S. and Meier, J. L. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29039931" target="_blank">Profiling cytidine acetylation with specific affinity and reactivity.</a> <em>ACS Chem Biol</em> 12(12): 2922-2926. </li><li>Tardu, M., Lin, L. and Koutmou, K. S. (2018). <a href="https://www.biorxiv.org/content/10.1101/327585v2" target="_blank">N4-acetylcytidine and 5-formylcytidine are present in Saccharomyces cerevisiae mRNAs.</a> <em>bioRxiv</em>. doi.org/10.1101/327585.</li><li>Trapnell, C., Pachter, L. and Salzberg, S. L. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19289445" target="_blank">TopHat: discovering splice junctions with RNA-Seq.</a> <em>B</em><em>ioinformatics</em> 25(9): 1105-1111. </li><li>Zhang, Y., Liu, T., Meyer, C. A., Eeckhoute, J., Johnson, D. S., Bernstein, B. E., Nusbaum, C., Myers, R. M., Brown, M., Li, W. and Liu, X. S. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18798982" target="_blank">Model-based analysis of ChIP-Seq (MACS).</a> <em>Genome Biol</em> 9(9): R137. </li><li>Zhao, B. S., Roundtree, I. A. and He, C. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27808276" target="_blank">Post-transcriptional gene regulation by mRNA modifications.</a> <em>Nat Rev Mol Cell Biol</em> 18(1): 31-42. <br/></li></ol>