<body>
    <h1>Analysis of Functional Virus-generated PAMP RNAs Using IFNÎ±/Î² ELISA Assay</h1>
    <p><strong>Authors:</strong> Margit Mutso, Xiang Liu, Andres Merits, Suresh Mahalingam</p>
    <h2>Abstract</h2>
    <p>Virus-generated PAMP RNAs are key factors that activate host immune response. The PAMP RNAs are therefore usually
        closely related with viral disease pathogenesis. Quantitative real time PCR is a conventional method to assess
        RNA. However, it cannot be used for detecting short dsRNAs generated by viral replicase. This protocol was
        established to analyze the PAMP RNAs produced by viruses which are able to induce host immune response.
        Classical viral PAMP RNAs and non-classical viral PAMP RNAs are analyzed separately. Briefly, to access total
        viral PAMP RNAs, total RNA was extracted from the virus infected cells and then transfected into Cop5 cells.
        Whereas, to assess non-classical viral PAMP RNAs, the constructs expressing viral replicase are transfected into
        Cop5 cells. The amount of IFNÎ±/Î² produced by Cop5 cells, determined by ELISA, is correlated with the total and
        non-classical viral PAMP RNAs. Since this method is based on type I IFN response, it is therefore suitable for
        measuring the functional virus-generated PAMP RNAs and also for assessing the efficiency of these PAMP RNAs.</p>
    <p><strong>Keywords:</strong> Virus, Infection, PAMP RNAs, RNA isolation, IFNÎ±/Î² detection</p>
    <h2>Background</h2>
    <p>Under virus infection, the activation of host immune system usually starts with the recognition of viral
        pathogen-associated molecular patterns (PAMPs) by host pattern recognition receptors (PRRs) (Medzhitov and
        Janeway, 2002). Viral PAMPs, associated with virus invasion, include virus surface glycoproteins, viral DNA and
        RNAs (Mogensen and Paludan, 2005). In case of RNA virus infection, virus replication generated RNAs are
        essential PAMPs portion which is able to be recognized by the host intracellular PRRs (Alexopoulou <em>et
            al.</em>, 2001). Two possible candidates for the role of such viral-generated PAMP RNAs have been recently
        reported. First, classical viral PAMP RNAs are dsRNA replication forms that are recognized by RIG-I and MDA5 and
        non-capped positive-strand RNAs (polyA+) generated by alphaviruses (Sokoloski <em>et al.</em>, 2015) that may be
        recognized by RIG-I (Akhrymuk <em>et al.</em>, 2015). Second, non-classical viral PAMP RNAs are short
        non-polyadenylated dsRNAs with 5â triphosphate. They are IFN-inducing RNAs generated by viral replicase using
        cellular RNA templates (Nikonov <em>et al.</em>, 2013). In Semliki Forest virus (SFV) infected cells these RNAs
        represent the majority of PAMP RNAs (Nikonov <em>et al.</em>, 2013). <br /><br /> Given the essential role of
        viral PAMP RNAs to activate host immune system, a robust protocol to analyze different types of virus-generated
        PAMP RNAs is useful for virus-host interaction study. Quantitative real time PCR could be used to quantify
        classical viral dsRNAs by targeting negative strands. However, it is hard to differentiate between capped and
        non-capped positive strands. Furthermore, the approach is neither able to tell if the targeted PAMP RNA is
        functional nor able to access the PAMP efficiency and cannot detect the non-classical viral PAMP RNAs. To
        analyze the functional PAMP RNAs that are able to trigger host immune response, we established this protocol to
        analyze both classical and non-classical viral PAMP RNAs. To assess the total viral PAMP RNAs, total RNA from
        the infected cells were extracted for the assessment. Since all types of classical alphaviral PAMP RNAs have
        unpaired polyA tails and they are presumably the biggest proportion in the viral PAMP RNAs, they may overshadow
        other types of PAMP RNAs. Therefore, in a parallel experiment, we separated and examined the polyA- fraction
        from the total RNAs (Nikonov <em>et al.</em>, 2013). To assess the non-classical viral PAMP RNAs, viral
        replicase constructs were used for analysis because the system did not have replication competent viral RNA
        templates and therefore no classical viral PAMP RNAs were produced. Cop5 cells were used as IFNÎ±/Î² activation
        host for all PAMP RNAs analyses. The quantity of the IFNÎ±/Î² produced by the transfected PAMP RNAs can be
        determined by ELISA assay.<br /><br /> This protocol is a reliable method to analyze alphavirus-generated PAMP
        RNAs during the virus replication and, with exception of separation of polyA- fraction, applied to viruses that
        lack unpaired polyA tails. It is not only to measure the level of the PAMP RNAs but also is able to assess the
        PAMP RNAs efficiency. Therefore, this protocol is suitable for the study involving the interaction between RNA
        virus invasion and the host immune response.</p>
</body>

<p class="pptt" id="biaoti33504">## Materials and Reagents</p>
<ol class="npwone">
    <li>Materials<br />
        <ol class="npwtwo">
            <li>1.5 ml Eppendorf<sup>Â®</sup> DNA LoBind microcentrifuge tubes (Eppendorf, catalog number: 0030108051)
            </li>
            <li>100 mm Corning<sup>Â®</sup> tissue-culture treated culture dishes (Corning, catalog number: 430167)</li>
            <li>12-well Corning<sup>TM </sup>Costar<sup>TM</sup> flat bottom cell culture plates (Corning, catalog
                number: 3513)</li>
            <li>15 ml Falcon<sup>TM</sup> conical centrifuge tubes (Corning, catalog number: 352096)</li>
            <li>6-well plate</li>
        </ol><br />
    </li>
    <li>Cell lines<br />
        <ol class="npwtwo">
            <li>ATCC<sup>Â® </sup>BHK-21[C13] cells (ATCC, catalog number: CCL-10<sup>TM</sup>)</li>
            <li>Cop5 cells (Tyndall <em>et al.</em>, 1981)<br />
                <div style="white-space:nowrap;"><br /></div>
            </li>
        </ol>
    </li>
    <li>Viruses of interest<br /><em>Note: Prepare the viruses of interest from infectious clones. It is recommended
            that the virus titer should be prepared between 105 and 107 plaque forming units (PFU)/ml. The viruses
            listed below are for a demonstration. These viruses are derived from infectious clones, released (passage
            0), passaged in BHK-21 cells and stored at -80 Â°C. <br /></em>
        <ol class="npwtwo">
            <li>RRV-T48</li>
            <li>RRV-T48<sub>A534V</sub><br />
                <div style="white-space:nowrap;"><br /></div>
            </li>
        </ol>
    </li>
    <li>Viral replicase constructs<br /><em>Note: Prepare the constructs expressing the viral replicase (non-structural
            proteins) of the viruses of interest. The constructs listed below are for a demonstration. As shown in the
            construct map below (Figure 1), the regions encoding the replicase of RRV-T48, RRV-T48<sub>A534V</sub> and
            SFV4 (used as positive control [Nikonov et al., 2013]) were cloned into pMC-gtGTU2 vector, which contains
            cytomegalovirus early promoter, leader sequence from the thymidine kinase of Herpes Simplex virus (HSV) with
            an inserted synthetic intron and late polyadenylation signal from simian virus 40 (SV40).<br /></em>
        <ol class="npwtwo">
            <li>RRV-T48</li>
            <li>RRV-T48<sub>A534V</sub> </li>
            <li>SFV<br />
                <div style="white-space:nowrap;"><br /><img alt="" class="layerphoto" height="100" loading="lazy"
                        src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617005624_6857.jpg"
                        width="132" /><br /><strong>Figure 1. Viral replicase construct map</strong><br /><br /></div>
            </li>
        </ol>
    </li>
    <li>Reagents<br /><span id="__kindeditor_bookmark_end_80__" style="display:none;"></span>
        <ol class="npwtwo">
            <li>Dulbecco's modified Eagle medium (DMEM) (Sigma-Aldrich, catalog number: D5030)</li>
            <li>Iscoveâs Modified Dulbeccoâs Medium (IMDM) (Sigma-Aldrich, catalog number: 51471C)</li>
            <li>L-Glutamine (Sigma-Aldrich, catalog number: G6392)</li>
            <li>Fetal bovine serum (FBS) (Sigma-Aldrich, catalog number: F6178)</li>
            <li>100x Penicillin-streptomycin (Sigma-Aldrich, catalog number: P4333)</li>
            <li>Tryptose phosphate broth solution (TPB) (Sigma-Aldrich, catalog number: T8159)</li>
            <li>Phosphate buffered saline (PBS) (Sigma-Aldrich, catalog number: P5493)</li>
            <li>1 M HEPES buffer (Sigma-Aldrich, catalog number: 83264)</li>
            <li>0.25% Trypsin-EDTA solution (Sigma-Aldrich, catalog number: T4049)</li>
            <li>RNeasy Mini Kit (QIAGEN, catalog number: 74104)</li>
            <li>TRIzol<sup>Â®</sup> Reagent (Invitrogen, catalog number: 15596026)</li>
            <li>DNase I (Roche, catalog number: 04716728001)</li>
            <li>PolyATtract<sup>Â®</sup> mRNA Isolation Systems (Promega, catalog number: Z5210)</li>
            <li>Lipofectamine<sup>TM</sup> 2000 reagent (Thermo Fisher Scientific, catalog number: 11668019)</li>
            <li>VeriKine Mouse Interferon Beta ELISA Kit (PBL Assay Science, catalog number: 42400-1)</li>
            <li>pMC-gtGTU2 vector (Nikonov <em>et al.</em>, 2013) or any mammalian expression vector (such as pcDNA or
                pCMV series from Thermo Fisher Scientific)</li>
        </ol>
    </li>
</ol>
<p class="pptt" id="biaoti33505">### Equipment</p>
<ol class="npwtwok">
    <li>-80 Â°C freezer</li>
    <li>4 Â°C refrigerator</li>
    <li>Vortexer (Bio-Rad)</li>
    <li>Pipettes (Bio-Rad, catalog numbers: 166-0506, 166-0507 and 166-0508)</li>
    <li>CO<sub>2</sub> incubator (Bio-Rad, catalog number: 4110)</li>
    <li>Biosafety level 2 (BSL-2) cabinet</li>
    <li>Autoclave (Tuttnauer, ELV-D Line)</li>
    <li>UVC5000 Ultraviolet Crosslinker (Hoefer, code number: UVC5000-230V)</li>
    <li>Sunrise<sup>TM</sup> absorbance microplate reader (Tecan)</li>
    <li>Microfuge 16 Centrifuge (Beckman Coulter Life science, catalog number: A46473)</li>
</ol>
<p class="pptt" id="biaoti33506">### Software</p>
<ol class="npwtwok">
    <li>Microsoft Excel or Prism 5</li>
</ol>
<p class="pptt" id="biaoti33507">### Procedure</p>
<p class="ppzz">The procedure showing the analysis scheme for viral PAMP RNAs is summarized in Figure 2.<br /><br /><img
        alt="" class="layerphoto" height="100" loading="lazy"
        src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617005700_5406.jpg"
        width="250" /><br /><strong>Figure 2. Flow chart for viral PAMP RNAs analysis<br /><br /></strong> </p>
<ol class="npwone">
    <li>Analysis for total and classical viral PAMP RNAs <br />
        <ol class="npwtwo">
            <li>Preparation of total RNA samples from infected cells.<br /><strong>Day -1<br /></strong>
                <ol class="npwthree">
                    <li>Seed 8 x 10<sup>6</sup> BHK-21 cells on to 100 mm cell culture dish and culture in DMEM with 10%
                        FBS. Maintain the cells at 37 Â°C with 5% CO<sub>2</sub> overnight.<br /></li>
                </ol><strong> Day 0</strong>
                <ol class="npwthree" start="2">
                    <li>Discard the BHK-21 cell growth media and wash the cell monolayer once with PBS. Infect the cells
                        with the viruses of interest. As a demonstration, dilute RRV-T48 and RRV-T48<sub>A534V</sub>
                        separately in serum free DMEM for the infection. Infect the BHK-21 cell monolayer by the viruses
                        at Multiplicity of Infection (MOI) 1.0 with a minimum coverage volume (approximately 400 Î¼l for
                        each well in a 6-well plate). Incubate the cells for 1 h at 37 Â°C with 5% CO<sub>2</sub> and
                        then maintain the cells in DMEM with 10% FBS for another 6 h. </li>
                    <li>Extract the total RNA from the infected BHK-21 cells using TRIzol Reagent according to the
                        manufacturerâs protocol <a
                            href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Ftrizol_reagent.pdf&amp;title=VXNlciBHdWlkZTogVFJJem9sIFJlYWdlbnQ="
                            target="_blank">(Invitrogen TRIzol<sup>TM</sup> Reagent USER GUIDE)</a>. Use 1 U DNase I per
                        1 Î¼g total RNA to treat the extracted RNA for 60 min at 37 Â°C.</li>
                    <li><strong>Optional (viruses with non-paired polyA tail):</strong> Isolation of non-classical
                        PAMPs. Aliquot 100 Î¼g of the treated RNA into a clean Eppendorf<sup>Â®</sup> DNA LoBind
                        microcentrifuge tube (RNase free). Remove the polyA(+) fraction from the total RNA using
                        PolyATtract mRNA Isolation Systems Kit according to <a
                            href="https://www.promega.com.au/products/rna-purification-and-analysis/rna-purification/polyattract-mrna-isolation-systems/?catNum=Z5210"
                            target="_blank">the manufacturerâs protocol (Promega).</a> </li>
                    <li>Purify the total, polyA(+) and polyA(-) RNA samples with RNeasy Mini Kit according to <a
                            href="https://www.qiagen.com/us/products/discovery-and-translational-research/dna-rna-purification/rna-purification/cell-free-rna/rneasy-mini-kit/#resources"
                            target="_blank">the manufacturerâs protocol (Qiagen)</a> and elute the product within 20
                        Î¼l RNase free water.</li>
                    <li>UV inactivates the infectious viral RNAs, presented in the obtained samples, for 5 min in a
                        UVC5000 Ultraviolet Crosslinker at 2000 Î¼J/cmÂ². </li>
                    <li><strong>Optional: </strong>Store the RNA at -80 Â°C for longer period if needed.<br /></li>
                </ol>
            </li>
            <li>Analysis of viral PAMP RNAs<br /><strong>Day -1<br /></strong>
                <ol class="npwthree">
                    <li>Seed 1 x 10<sup>6</sup> Cop5 cells into 12-well plate and cultured in IMDM with 10% FBS and 2 mM
                        L-glutamine. Maintain the cells at 37 Â°C with 5% CO<sub>2</sub> overnight.</li>
                </ol><strong>Day 0</strong><br />
                <ol class="npwthree" start="2">
                    <li>Transfect 1 Î¼g of treated total, polyA(+) or polyA(-) RNAs respectively into confluent cultures
                        of Cop5 cells in 12-well plates using Lipofectamine2000 reagent. Briefly, dilute 3 Î¼l of the
                        Lipofectamine2000 reagent in 100 Î¼l serum free IMDM and incubate for 2-5 min at room
                        temperature. Dilute 1 Î¼g RNA samples in 100 Î¼l serum free IMDM and incubate for 2-5 min at
                        room temperature. Combine the 100 Î¼l diluted Lipofectamine2000 reagent with the 100 Î¼l diluted
                        RNA samples and incubate the RNA-lipid mixture for 5 to 10 min at room temperate to form the
                        RNA-lipid complex. Discard the BHK-21 cell growth media and wash the cell monolayer once with
                        PBS. Add 1 ml serum free IMDM to the cells. Add the RNA-lipid complex drop-wise on top of the
                        medium and gently rock the plate after adding to the mix. Incubate the transfected cells at 37
                        Â°C with 5% CO<sub>2</sub> for 24 h. <br /></li>
                </ol><strong>Day 1</strong>
                <ol class="npwthree" start="3">
                    <li>Collect the supernatant from the transfected Cop5 cells to a 1.5 ml tube. Centrifuge samples at
                        3,000 <em>x g</em> for 5 min at 4 Â°C to remove the cell debris. Collect the supernatant to a
                        new tube.</li>
                    <li>Determine the amount of IFN-Î² produced by the transfected Cop5 cells using VeriKine Mouse IFN
                        Beta ELISA kit according to the manufacturerâs protocol together with standard curve.</li>
                    <br />
                </ol>
            </li>
        </ol>
    </li>
    <li>Analysis for non-classical viral PAMP RNAs<br /><strong>Day -1</strong><br />
        <ol class="npwtwo">
            <li>Seed 1 Ã 10<sup>6</sup> Cop5 cells into a 12-well plate and culture in IMDM with 10% FBS and 2 mM
                L-glutamine. Maintain the cells at 37 Â°C with 5% CO<sub>2</sub> overnight.</li>
        </ol><strong>Day 0</strong><br />
        <ol class="npwtwo" start="2">
            <li>Transfect 1 Î¼g of the plasmids encoding for viral replicase of interest into the confluent cultures of
                Cop5 cells in 12-well plates using Lipofectamine2000 reagent. Use plasmid that expresses inactive
                replicase (active-site mutant) as a negative control. As a demonstration, transfect RRV-T48,
                RRV-T48<sub>A534V</sub> and SFV4 replicase constructs into the Cop5 cells for the following analysis.
            </li>
            <li>Incubate the transfected cells at 37 Â°C with 5% CO<sub>2</sub> for 48 h. </li>
        </ol><strong>Day 2</strong><br />
        <ol class="npwtwo" start="4">
            <li>Collect the supernatant from the transfected Cop5 cells to a 1.5 ml tube. Centrifuge samples at 3,000
                <em>x g</em> for 5 min at 4 Â°C to remove the cell debris. Collect the supernatant to a new tube.
            </li>
            <li>Determine the amount of IFN-Î² produced by the transfected Cop5 cells using VeriKine Mouse IFN Beta
                ELISA kit according to <a
                    href="https://www.pblassaysci.com/assay-kits/mouse-ifn-beta-elisa-kit-tcm-serum-42400#documentation"
                    target="_blank">the manufacturerâs protocol (pbl Assay science)</a> together with standard curve.
                For each independent experiment, normalized the IFN-Î² levels measured in supernatants from cells
                transfected with RRV replicase expression plasmids to the amount of IFN-Î² produced by cells transfected
                by plasmid expressing SFV replicase. </li>
        </ol>
    </li>
</ol>
<p class="pptt" id="biaoti33508">### Data analysis</p>
<p class="ppzz">Collect the data of the amount of IFN-Î² in Microsoft Excel or Prism 5. Analyze the data using
    Studentâs two-tailed unpaired <em>t</em>-test. As a demonstration, the standard curve for IFN-Î² ELISA assay was
    calculated in Table 1. The levels of IFN-Î² produced by Cop5 cells under the transfection of total viral PAMP RNAs
    (Figure 3), non-classic viral PAMP RNAs (Figure 4) and the replicase constructs (Figure 5) from RRV-T48 or
    RRV-T48<sub>A534V</sub> were plotted in bar graphs. The corresponding plotting raw data were demonstrated in <a
        href="https://os.bio-protocol.org/attached/file/20190616/3282%20Supplementary%20file%20Table%20S1%20%20S2%20%20S3.docx"
        target="_blank">Tables S1, S2 and S3.</a> <br /><br /><strong>Table 1. IFN-Î² ELISA assay standard
        curve.</strong> Briefly, seven concentrations of the mouse IFN-Î² standard were prepared according to the
    following table. One-hundred microliters per well of interferon standard was added to each well and incubate for 1 h
    at room temperature. The liquid was removed and washed with wash solution for three times. One-hundred microliters
    of diluted antibody solution was added to each well and incubated for 1 h at room temperature. The contents in the
    wells were removed and washed with wash solution for three times. One-hundred microliters of diluted HRP Solution
    was added to the wells and incubated for 1 h at room temperature. The contents in the wells were removed and washed
    with wash solution for three times. One-hundred microliters of the TMB substrate solution was added to the wells and
    incubated for 15 min at room temperature in the dark. 100 Î¼l of stop solution was added to each well to stop the
    reaction. The absorbance at 450 nm was determined within 5 min.<br /><img alt="" class="layerphoto" loading="lazy"
        src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617005751_6981.jpg"
        width="500" /><br /><br /><img alt="" class="layerphoto" height="128" loading="lazy"
        src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617005843_1447.jpg"
        width="100" /><br /><strong>Figure 3. IFN-Î² produced by Cop5 cells by the transfection of total viral PAMP RNAs
        from RRV-T48 or RRV-T48<sub>A534V</sub>.</strong> BHK-21 cells were infected with RRV-T48 and
    RRV-T48<sub>A534V</sub> separately at MOI 1.0. Cells were lysed and total RNA was isolated at 6 h post infection.
    One microgram of each RNA sample was used for transfection of Cop5 cells. The amount of IFN-Î² in the cell
    supernatant at 24 h post transfection was determined. IFN-Î² levels were expressed as the means Â±SEM from three
    experiments (*, <em>P</em> &lt; 0.05 using Studentâs two-tailed unpaired <em>t</em>-test).<br /><br /><img alt=""
        class="layerphoto" height="120" loading="lazy"
        src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617005915_1719.jpg"
        width="100" /><br /><strong>Figure 4. IFN-Î² produced by Cop5 cells by the transfection of non-classic viral
        PAMP RNAs from RRV-T48 or RRV-T48<sub>A534V</sub>.</strong> BHK-21 cells were infected with RRV-T48 and
    RRV-T48<sub>A534V</sub> separately at MOI 1.0. Cells were lysed and total RNA was isolated at 6 h post infection.
    Poly(A)- fraction was obtained from isolated RNAs by removal of poly(A)+ RNAs, including viral dsRNA replication
    forms. 1 Î¼g of each RNA sample was used for transfection of Cop5 cells. The amount of IFN-Î² in the cell
    supernatant at 24 h post transfection was determined. <br /><br /><img alt="" class="layerphoto" height="114"
        loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190617/20190617005944_5117.jpg"
        width="100" /><br /><strong>Figure 5. IFN-Î² produced by Cop5 cells by the transfection of replicase of RRV-T48
        or RRV-T48<sub>A534V</sub>.</strong> Cop5 cells were transfected with plasmids designed to express RRV-T48,
    RRV-T48<sub>A534V</sub> or SFV4. The amount of IFN-Î² in supernatant was determined at 48 h post transfection.
    Values obtained for cells transfected with RRV-T48 or RRV-T48<sub>A534V</sub> replicase expression plasmids were
    normalized to values obtained for cells transfected with plasmid expressing SFV4 replicase (taken as 100).
    Normalized values were shown as the means Â±SEM from three experiments (*, <em>P</em> &lt; 0.05 using Studentâs
    two-tailed unpaired <em>t</em>-test).</p>
<p class="pptt" id="biaoti33509">### Acknowledgments</p>
<p class="ppzz">This protocol was originally published in the paper by Nikonov <em>et al.</em>, 2013. This study was
    supported by grants from the Australian National Health and Medical Research Council to SM (APP1031024) and Estonian
    Research Council Grant to AM (IUT20-27). Suresh Mahalingam is the recipient of the NHMRC Senior Research Fellowship
    (ID: APP11544347).</p>
<p class="pptt" id="biaoti33510">### Competing interests</p>
<p class="ppzz">All authors declare no competing interests.</p>
<p class="pptt" id="biaoti33511">### References</p>
<ol class="npwtwok">
    <li>Akhrymuk, I., Frolov, I. and Frolova, E. I. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26550947"
            target="_blank">Both RIG-I and MDA5 detect alphavirus replication in concentration-dependent mode.</a><em>
            Virology</em> 487: 230-241.</li>
    <li>Alexopoulou, L., Holt, A. C., Medzhitov, R. and Flavell, R. A. (2001). <a
            href="http://www.ncbi.nlm.nih.gov/pubmed/11607032" target="_blank">Recognition of double-stranded RNA and
            activation of NF-ÎºB by Toll-like receptor 3.</a> <em>Nature</em> 413(6857): 732-738.</li>
    <li>Medzhitov, R. and Janeway, C. A., Jr. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11951031"
            target="_blank">Decoding the patterns of self and nonself by the innate immune system.</a> <em>Science</em>
        296: 298-300.</li>
    <li>Mogensen, T. H. and Paludan, S. R. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15635478"
            target="_blank">Reading the viral signature by Toll-like receptors and other pattern recognition
            receptors.</a> <em>J Mol Med</em> (Berl) 83(3): 180-192.</li>
    <li>Nikonov, A., Molder, T., Sikut, R., Kiiver, K., Mannik, A., Toots, U., Lulla, A., Lulla, V., Utt, A., Merits, A.
        and Ustav, M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24039580" target="_blank">RIG-I and MDA-5
            detection of viral RNA-dependent RNA polymerase activity restricts positive-strand RNA virus
            replication.</a> <em>P</em><em>LoS Pa</em><em>thog</em> 9(9): e1003610.</li>
    <li>Tyndall, C., La Mantia, G., Thacker, C.M., Favaloro, J., Kamen, R., (1981). <a
            href="https://www.ncbi.nlm.nih.gov/pubmed/6275353" target="_blank">A region of the polyoma virus genome
            between the replication origin and late protein coding sequences is required in cis for both early gene
            expression and viral DNA replication.</a> <em>Nucleic Acids Research</em> 9, 6231-6250.</li>
    <li>Sokoloski, K. J., Haist, K. C., Morrison, T. E., Mukhopadhyay, S. and Hardy, R. W. (2015). <a
            href="http://www.ncbi.nlm.nih.gov/pubmed/25833042" target="_blank">Noncapped alphavirus genomic RNAs and
            their role during infection.</a> <em>J Virol</em> 89(11): 6080-6092.<br /><br /></li>
</ol>