<body>
            <h1>Detection of Heteroplasmic Variants in the Mitochondrial Genome through Massive Parallel Sequencing</h1>
            <p><strong>Authors:</strong> Joke Mertens, Filippo Zambelli, Dorien  Daneels, Ben  Caljon, Karen  Sermon, Claudia  Spits</p>
            <h2>Abstract</h2>
            <p>Detecting heteroplasmies in the mitochondrial DNA (mtDNA) has been a challenge for many years. In the past, Sanger sequencing was the main option to perform this analysis, however, this method could not detect low frequency heteroplasmies. Massive Parallel Sequencing (MPS) provides the opportunity to study the mtDNA in depth, but a controlled pipeline is necessary to reliably retrieve and quantify the low frequency variants. It has been shown that differences in methods can significantly affect the number and frequency of the retrieved variants. <br/><br/> In this protocol, we present a method involving both wet lab and bioinformatics that allows identifying and quantifying single nucleotide variants in the full mtDNA sequence, down to a heteroplasmic load of 1.5%. For this, we set up a PCR-based amplification of the mtDNA, followed by MPS using Illumina chemistry, and variant calling with two different algorithms, mtDNA server and Mutect. <br/><br/> The PCR amplification is used to enrich the mitochondrial fraction, while the bioinformatic processing with two algorithms is used to discriminate the true heteroplasmies from background noise. The protocol described here allows for deep sequencing of the mitochondrial DNA in bulk DNA samples as well as single cells (both large cells such as human oocytes, and small-sized single cells such as human embryonic stem cells) with minor modifications to the protocol.<br/></p>
            <p><strong>Keywords:</strong> mtDNA, Massive parallel sequencing, Heteroplasmy, Long range PCR, Amplicon sequencing</p>
            <h2>Background</h2><p> In the past, the methods used for studying the mtDNA were amongst others PCR-RFLP (PCR-restriction fragment length polymorphism), Sanger sequencing and mitochondrial DNA re-sequencing using Affymetrixâs MitoChip. However, these methods are not able to accurately quantify the heteroplasmic load under 10%. Massive Parallel Sequencing (MPS) represents, in all likelihood, the best solution to investigate variants in the mitochondrial genome. However, when analyzing mtDNA there are two key factors that make mtDNA analysis less straightforward compared to the nuclear genome. First, the mtDNA contains regions with significant homology to regions dispersed within the nuclear genome called nuclear mitochondrial DNA sequences (NuMTS). Second, multiple mtDNA copies are present within a cell, and variants can be present at frequencies ranging between 0 and 100%. For this reason, when performing MPS analysis of the mtDNA different strategies might be applied to accurately identify and quantify single nucleotide variants (SNVs). The best approach to overcome the first problem is to enrich the sample for its mitochondrial genome only. This can be achieved by selectively amplifying the mtDNA by long-range PCR, by isolating the mtDNA using mtDNA enrichment kits, or by isolating the mitochondria themselves prior to DNA extraction (Just <em>et al</em>., 2015). The second issue is more challenging. Whilst MPS provides the ideal type of data to simultaneously identify SNVs and/or rearrangements and calculating their loads, the manner in which the data are generated and processed will not only determine the type of variants that can be detected, but also their lower threshold. If an SNV is present at a very low frequency, its signal will be undistinguishable from the systems sequencing errors (Bai and Wong, 2004; Rohlin <em>et al</em>., 2009; Zhang <em>et al</em>., 2012; Ye <em>et al</em>., 2014). Recently, many pipelines have been released to identify and quantify variants. However, bioinformatic processing does not represent the only critical step in these analyses. In our hands, we found that the initial amplification of the template is an extremely important step for the correct evaluation of SNV frequencies, such that a suboptimal PCR amplification leads to a gross overestimation of the retrieved frequencies (Zambelli <em>et al</em>., 2017). This is especially the case for PCR protocols with higher cycle numbers and primer sets that do not result in a linear amplification. The method we present here can be used to accurately detect and quantify single nucleotide variants at a low heteroplasmic load (as low as 1.5%) in both bulk DNA samples and single cells (Zambelli <em>et al</em>., 2017 and 2018). We here describe amplification conditions and bioinformatic processing for both bulk DNA and single cells with detailed information and screenshots about the bioinformatic steps.</p>
        </body>

<p class="pptt" id="biaoti28229">## Materials and Reagents</p><ol class="npwtwok"><li>96-well plate</li><li>200 Î¼l Eppendorf tubes</li><li>LongAmp Buffer (LongAmp Taq DNA Polymerase kit) (New England Biolabs, catalog number: M0323L), stored at -20 Â°C</li><li>Taq LongAmp (LongAmp Taq DNA Polymerase kit) (New England Biolabs, catalog number: M0323L), stored and kept at all times at -20 Â°C</li><li>dNTPâs (dNTP sets, Illustra<sup>TM</sup>) (VWR, catalog number: 28-4065-57), diluted to 2 mM and stored at -20 Â°C</li><li>Tricine (Sigma-Aldrich, catalog number: T9784), diluted to 200 mM and stored at 4 Â°C</li><li>DTT (DL-Dithiothreitol) (Sigma, catalog number: D-0632), stored at 4 Â°C</li><li>NaOH (Stock solution: 1 M)</li><li>TBE buffer (1x) (Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: 15581044)</li><li>Agarose (Agarose DNA Pure Grade, VWR, catalog number: 443666A)</li><li>PCR Purification Kit (AMPure XP for PCR purification, Beckman Coulter, catalog number: A63882)</li><li>Primer set 1 (5042f-1424r) (Integrated DNA Technologies, diluted to 10 Î¼mol, aliquoted and stored at -20 Â°C)<br/>Forward primer: 5â²-AGC AGT TCT ACC GTA CAA CC-3â²<br/>Reverse primer: 5â²-ATC CAC CTT CGA CCC TTA AG-3â²</li><li>Primer set 2 (528f-5789r) (Integrated DNA Technologies, diluted to 10 Î¼mol, aliquoted and stored at -20 Â°C)<br/>Forward primer: 5â²-TGC TAA CCC CAT ACC CCG AAC C-3â²<br/>Reverse primer: 5â²-AAG AAG CAG CTT CAA ACC TGC C-3â²<br/></li></ol><p class="ppzz"><em>Note: These two primer sets (Items 12 and 13) were selected because they were able to amplify large fragments of the mtDNA and tested negative when amplifying Rho Zero cells, indicating that they did not amplify NuMTS in the nuclear genome. These primer sets were also evaluated for their performance in low frequency heteroplasmy calling by performing spike-in experiments and were shown to give the better estimation of the low frequency variants (Zambelli et al., 2017).</em> </p><ol class="npwtwok" start="14"><li>Qubit<sup>TM</sup> dsDNA Broad Range Assay Kit (Invitrogen<sup>TM</sup>, catalog number: Q32853) </li><li>Ethanol (80%)</li><li>Tris-HCl solution (10 mM, pH 8.0)</li><li>Library Preparation kit (KAPA HyperPlus kit, Roche, catalog number: 07962436001)</li><li>Fragmentation Kit (HS NGS Fragment Kit, Agilent, catalog number: DNF-474)</li><li>Tween 20 (Sigma-Aldrich, catalog number: P9416)</li><li>MPS Reagent Kit (NovaSeq6000 S2 Reagent Kit [200 cycles]) (Illumina, catalog number: 20012861)</li><li>Gel electrophoresis (see Recipes)</li><li>Alkaline Lysis Buffer (see Recipes)</li><li>EBT (Elution Buffer with Tris) buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28230">### Equipment</p><ol class="npwtwok"><li>Agarose Gel electrophoresis apparatus</li><li>Power Supply (Electrophoresis Power SupplyâEPS 301) (GE Health Care, catalog number: 18113001) </li><li>Thermal Cycler (Veriti<sup>TM</sup> 96-Well, Applied Biosystems, catalog number: 4375786)</li><li>Magnet</li><li>Qubit Fluorometric Quantification (Invitrogen<sup>TM</sup>, catalog number: Q33238)</li><li>Fragment Analyzer (Agilent formerly Advanced Analytical Technologies, Agilent, <a href="https://www.aati-us.com/instruments/fragment-analyzer/" target="_blank">https://www.aati-us.com/instruments/fragment-analyzer/</a>) </li><li>Sequencing system (<em>e.g.</em>, NovaSeq6000, Illumina, catalog number: 20012850)<br/></li></ol><p class="pptt" id="biaoti28231">### Software</p><ol class="npwtwok"><li>bcl2fastq conversion software v2.19 script (Illumina, <a href="http://emea.support.illumina.com/downloads/bcl2fastq-conversion-software-v2-19.html" target="_blank">http://emea.support.illumina.com/downloads/bcl2fastq-conversion-software-v2-19.html</a>)</li><li>Seqtk (<a href="https://bioconda.github.io/recipes/seqtk/README.html" target="_blank">https://bioconda.github.io/recipes/seqtk/README.html</a>) </li><li>BWA-MEM </li><li>GATK v3.3 (Broad Institute, <a href="https://software.broadinstitute.org/gatk/" target="_blank">https://software.broadinstitute.org/gatk/</a>) </li><li>GATK v3.6 Mutect2 (Broad Institute, <a href="https://software.broadinstitute.org/gatk/documentation/tooldocs/3.7-0/org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.php" target="_blank">https://software.broadinstitute.org/gatk/documentation/tooldocs/3.7-0/org_broadinstitute_gatk_tools_walkers_cancer_m2_MuTect2.php</a>)</li><li>Mpileup (SAMtools, <a href="http://samtools.sourceforge.net/mpileup.shtml" target="_blank">http://samtools.sourceforge.net/mpileup.shtml</a>) </li><li>mtDNA server (<a href="https://mtdna-server.uibk.ac.at/index.html" target="_blank">https://mtdna-server.uibk.ac.at/index.html</a>)</li><li>MitoWheel (<a href="http://mitowheel.org/mitowheel.html" target="_blank">http://mitowheel.org/mitowheel.html</a>)</li><li>MitImpact (<a href="http://mitimpact.css-mendel.it/" target="_blank">http://mitimpact.css-mendel.it/</a>)</li><li>MutPred2 (<a href="http://mutpred.mutdb.org/" target="_blank">http://mutpred.mutdb.org/</a>)<br/></li></ol><p class="pptt" id="biaoti28232">### Procedure</p><ol class="npwtwok"><li>Extraction of bulk DNA: extract DNA from the sample of interest. <br/><ol class="npwthree"><li>The sample can be a bulk of tissue or cell lines. No special kit is required for bulk analysis, for instance, we routinely use either the DNeasy kit from Qiagen or a phenol/chlorophorm extraction followed by ethanol precipitation (Zambelli <em>et al.</em>, 2018). </li><li>For single cells, we manually collect them with the aid of a stereomicroscope under a horizontal flow in 2.5 Î¼l of alkaline lysis buffer (200 mM NaOH and 50 mM DTT) in 200 Î¼l Eppendorf tubes (Spits <em>et al.</em>, 2006) and store them at -20 Â°C. Before amplification, the single cells are incubated for 10 min at 65 Â°C for lysis, then prepared for amplification as under Step 3. </li></ol></li><li>Dilute the DNA sample to a working solution of 10 ng/Î¼l (only for bulk DNA).</li><li>Prepare the master mix as following:<br/><img alt="" class="layerphoto" height="267" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702050521_4123.jpg" width="600"/><br/>*Tricine is used in the PCR reaction to buffer the alkaline lysis buffer used in the single cell collection.</li><li>Aliquot 45 Î¼l of the master mix to PCR tubes and add 5 Î¼l of 10 ng/Î¼l DNA for bulk.<br/><em>Note: For single cells, add 47.5 Î¼l of the master mix to the Eppendorf tubes with the cells collected in the alkaline lysis buffer.</em> </li><li>Use the following PCR program:<ol class="npwthree"><li>step 1 (Initiation): 30 s at 94 Â°C</li><li>step 2 (8 cycles): 15 s at 94 Â°C<br/>30 s at 64 Â°C minus 0.4 Â°C per cycle (this is the touchdown step)<br/>11 min at 65 Â°C (for primer set 1) or 5 min at 65 Â°C (for primer set 2)</li><li>step 3 (22 cycles for bulk; 27 cycles for large single cells (human oocytes), 37 cycles for other single cells): <br/>15 s at 94 Â°C <br/>30 s at 61 Â°C <br/>11 min at 65 Â°C (for primer set 1) or 5 min at 65 Â°C (for primer set 2)</li><li>step 4 (Final): 11 min at 65 Â°C (for primer set 1) or 5 min at 65 Â°C (for primer set 2)</li><li>step 5: cooling step at 4 Â°C until storage.</li></ol></li><li>Keep the amplicons stored at 4 Â°C (for short-term storage) or -20 Â°C (long-term storage) until further processing.</li><li>Confirm successful amplification by running a gel-electrophoresis (1% for bulk, 1.5% for single cells) (see Recipes).</li><li>Load 9 Î¼l per sample on the gel and run the gel at 80 V for 1 h. Successful amplification should be as shown in Figure 1.<br/><br/><img alt="" class="layerphoto" height="176" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702050756_2132.jpg" width="200"/><br/><strong>Figure 1. Successful amplification of the two primer sets compared to a DNA ladder shown by gel-electrophoresis<br/></strong><br/></li><li>Clean up the samples with AMPure beads as follows.<br/><em>Note: This is usually done in batches of 96 samples in a 96-well plate.</em> <ol class="npwthree"><li>Add 54 Î¼l of AMPure beads to 30 Î¼l of each amplicon (the ratio is 1.8x).</li><li>Mix the samples by pipetting up and down and incubate the samples for 5 min at room temperature.</li><li>Place the samples on the magnet for 5 min or until the solution is clear.</li><li>Remove the supernatant without disturbing the beads.</li><li>Add 200 Î¼l ethanol (80%).</li><li>Incubate the samples for 30 s on the magnet.</li><li>Remove the supernatant without disturbing the beads.</li><li>Repeat Steps 9e-9g.</li><li>Let the beads dry for 5 min at room temperature.</li><li>Remove the samples from the magnet.</li><li>Add 30 Î¼l of Tris-HCl (10 mM, pH 8.0) and incubate the samples for 2 min at room temperature.</li><li>Place the samples on the magnet until the solution is clear.</li><li>Transfer the supernatant into a new 96-well plate.</li></ol></li><li>Quantify the DNA concentration of the samples using the Qubit according to the manufacturerâs instructions.</li><li>Pool the amplicons of primer sets 1 and 2 together per sample, and to maintain a uniform coverage, the amplicons need to be mixed in a 0.35/0.65 ratio (35% of the shorter amplicon generated by primer set 2 and 65% of the longer amplicon generated by primer set 1). Use a total of 500 ng DNA in 17.5 Î¼l Tris-HCl (10 mM, pH 8.0). <br/><em>Note: The procedure is also possible for each amplicon separately, again with a total of 500 ng.</em> </li><li>Prepare the library with the KAPA HyperPlus Kit using half of the reagent volumes as specified by the supplier. This is the protocol in summary:<br/><ol class="npwthree"><li>Prepare the fragmentation mix (on ice):<br/><img alt="" class="layerphoto" height="81" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702060323_2878.jpg" width="350"/><br/><br/></li><li>Vortex the fragmentation mix and spin down.</li><li>Add 7.5 Î¼l of the fragmentation mix to 17.5 Î¼l of the purified DNA sample and mix by pipetting (work on ice).</li><li>Incubate the samples in a thermocycler (lid heated at 50 Â°C).<br/>10 min at 4 Â°C<br/>15 min at 37 Â°C<br/>Keep the samples at 4 Â°C</li><li>Proceed immediately to the End Repair and A-tailing.</li><li>Prepare the End Repair and A-tailing mix as follows:<br/><img alt="" class="layerphoto" height="79" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702060437_6712.jpg" width="400"/><br/><br/></li><li>Mix the End Repair and A-tailing mix by vortexing and spin down.</li><li>Add 5 Î¼l of the End Repair and A-tailing mix to the fragmented DNA.</li><li>Incubate the samples in the thermocycler (put the lid on 85 Â°C).<br/>30 min at 65 Â°C<br/>Keep the samples at 4 Â°C.</li><li>Proceed immediately to the Ligation step.</li><li>Prepare the Ligation mix as follows:<br/><img alt="" class="layerphoto" height="92" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702060518_1809.jpg" width="400"/><br/><br/></li><li>Mix the Ligation mix by pipetting up and down.</li><li>Add 20 Î¼l of the Ligation mix to 30 Î¼l of the End Repaired and A-tailed DNA.</li><li>Add a unique adapter per sample: 5 Î¼l of 7.5 Î¼M Illumina TruSeq Unique Dual (UD) indexed adapter.<br/><em>Note: These adapters may be custom produced by an oligo supplier (e.g., Integrated DNA Technologies) or may be purchased as a kit (e.g., IDT for IlluminaâTruSeq DNA UD Indexes). Be aware that most suppliers deliver ready-made adapters at 15 Î¼M while in this protocol we use 7.5 Î¼M.</em> </li><li>Mix thoroughly by pipetting.</li><li>Incubate the samples in a thermocycler (no heated lid) for 15 min at 20 Â°C.</li><li>Proceed immediately with the bead sample clean up. </li></ol></li><li>Clean up the libraries with AMPure beads as follows:<br/><ol class="npwthree"><li>Add 44 Î¼l of AMPure beads to 35 Î¼l of the library (the ratio is 0.8x).</li><li>Mix the samples by pipetting up and down and incubate the samples for 5 min at room temperature.</li><li>Place the samples on a magnet for 5 min or until the solution is clear.</li><li>Remove the supernatant without disturbing the beads.</li><li>Add 200 Î¼l of ethanol (80%).</li><li>Incubate the samples for 30 s on the magnet.</li><li>Remove the supernatant without disturbing the beads.</li><li>Repeat Steps 13e-13g.</li><li>Let the beads dry for 5 min at room temperature.</li><li>Remove the samples from the magnet.</li><li>Add 27 Î¼l of Tris-HCl (10 mM, pH 8.0).</li><li>Incubate the samples for 2 min at room temperature.</li><li>Place the samples on the magnet until the solution is clear. </li><li>Transfer 25 Î¼l of the eluate to a new 96-well plate.</li></ol><em>Note: This can be a stopping point. Keep the samples in the fridge (4 Â°C) for 24 h, or for longer storage, keep them in the freezer (-20 Â°C).</em> </li><li>Size-select the libraries with AMPure beads as follows:<br/><ol class="npwthree"><li>Add 75 Î¼l of nuclease-free water to the purified libraries.</li><li>Add 50 Î¼l of AMPure beads to 100 Î¼l of the library (the ratio is 0.5x).</li><li>Mix the samples by pipetting up and down and incubate the samples for 5 min at room temperature.</li><li>Place the samples on the magnet for 5 min or until the solution is clear.</li><li>Transfer 140 Î¼l of the supernatant to a new 96-well plate.</li><li>Add 20 Î¼l of AMPure beads to 140 Î¼l of supernatant (the ratio is 0.7x).</li><li>Mix the samples by pipetting up and down and incubate the samples for 5 min at room temperature.</li><li>Place the samples on the magnet for 5 min or until the solution is clear.</li><li>Remove the supernatant without disturbing the beads.</li><li>Add 200 Î¼l of ethanol (80%).</li><li>Incubate the samples for 30 s on the magnet.</li><li>Remove supernatant without disturbing the beads.</li><li>Repeat Steps 14j-14I.</li><li>Let the beads dry for 5 min at room temperature.</li><li>Remove the samples from the magnet.</li><li>Add 15 Î¼l of Tris-HCl (10 mM; pH 8.0).</li><li>Incubate the samples for 2 min at room temperature.</li><li>Place the samples on the magnet until the solution is clear.</li><li>Transfer 13 Î¼l of the eluate to a new 96-well plate.</li></ol><em>Note: This can be a stopping point. Keep the samples in the fridge (4 Â°C) for 24 h or, for longer storage, keep them in the freezer (-20 Â°C).</em> </li><li>Check the quality of the final library (1/10 dilution of the prepared library) by electrophoresis on the AATI Fragment Analyzer using the HS NGS Fragment Kit. An example of successful libraries is shown in Figure 2. <br/><br/><img alt="" class="layerphoto" height="120" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702051753_8141.jpg" width="200"/><br/><strong>Figure 2. Successfully prepared libraries shown by electrophoresis on an AATI Fragment Analyzer<br/></strong><br/></li><li>Quantify all samples using the Qubit according to the supplierâs instructions.</li><li>Use the average size (smear analysis on the AATI Fragment Analyzer between 200 and 1500 base pairs) and the Qubit concentration, to calculate the molarity of the obtained library. Use the formula below:<br/><br/><img alt="" class="layerphoto" height="63" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702051919_3964.jpg" width="450"/><br/><br/></li><li>Dilute all libraries to 2 nM in EBT buffer.</li><li>Pool all the samples by combining equal volumes of each 2 nM library.</li><li>Load the samples on the Illumina platform of choice. For the Illumina NovaSeq6000, perform the library denaturation as mentioned in the NovaSeq6000 Sequencing System Guide (Document 1000000019358v11). For 96 samples, use 6% capacity (matching the denaturation of 9 Î¼l of 2 nM library pool) of the NovaSeq6000 S2 Reagent Kit (200 cycles).<br/></li></ol><p class="pptt" id="biaoti28233">### Data analysis</p><ol class="npwtwok"><li>Demultiplex the base call (.bcl) files to .fastq files with Illumina bcl2fastq v2.19 script.</li><li>Extract ad random 1.5 million reads from the .fastq files with the âseqtkâ tool.<br/><em>Note: This is to reduce the number of reads because a high number can cause computational problems in Step 6.</em> </li><li>Align the .fastq file to the reference genome (NC_012920.1) and generate a .bam file using BWA-MEM.</li><li>Realign for insertions and deletions and base recalibrations (from .bam to .bam) with GATK v3.3.</li><li>For variant calling, use the tool GATK v3.6Mutect2 (from .bam to .vcf).<br/><em>Note: The code for the previous steps 1-5 can be found in the <a href="https://os.bio-protocol.org/attached/file/20190624/3283 Supplementary File 1.docx" target="_blank">Supplementary File 1</a>.</em> </li><li>Upload the .bam file to mtDNA server (<a href="https://mtdna-server.uibk.ac.at/index.html" target="_blank">https://mtdna-server.uibk.ac.at/index.html</a>). </li><li>The mtDNA server report shows:<br/><ol class="npwthree"><li>Heteroplasmic variants (an example is shown in Figure 3).<br/><br/><img alt="" class="layerphoto" height="48" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702052130_8084.jpg" width="200"/><br/><strong>Figure 3. Report of the detected heteroplasmic variants by mtDNA server</strong><br/><br/></li><li>Homoplasmic variants (an example is shown in Figure 4).<br/><br/><img alt="" class="layerphoto" height="62" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702052550_2228.jpg" width="200"/><br/><strong>Figure 4. Report of the detected homoplasmic variants by mtDNA server</strong><br/><br/></li><li>Haplogroup and possible contamination with other haplogroups (an example is shown in Figure 5).<br/><br/><img alt="" class="layerphoto" height="58" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702052632_8618.jpg" width="200"/><br/><strong>Figure 5. Report of the haplogroup detected by mtDNA server</strong><br/><br/></li><li>Coverage plot (an example is shown in Figure 6).<br/><br/><img alt="" class="layerphoto" height="87" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702052700_8909.jpg" width="200"/><br/><strong>Figure 6. Example of a coverage plot provided by mtDNA server</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Open the Mutect file and sort the variants per frequency.<br/><ol class="npwthree"><li>Open the Mutect file and follow the process shown in Figure 7:<br/><br/><img alt="" class="layerphoto" height="154" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053016_5136.jpg" width="200"/><br/><strong>Figure 7. Sequential steps to open a Mutect file in a Microsoft Office Excel. </strong>Make sure that âDelimitedâ is indicated in step 1 and follow the standard settings.<br/><br/></li><li>When the file is opened, select the complete column below cell J39 as shown in Figure 8. <br/><br/><img alt="" class="layerphoto" height="131" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053050_4539.jpg" width="200"/><br/><strong>Figure 8. Lay-out when a Mutect file is opened and the complete column below cell J39 is selected</strong><br/><br/></li><li>Convert the selected area to columns by selecting the tool âConvert text to columnsâ as shown in Figure 9.<br/><br/><img alt="" class="layerphoto" height="130" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053130_8908.jpg" width="200"/><br/><strong>Figure 9. Conversion of text to columns in the complete column below cell J39<br/><br/></strong> </li><li>Follow these setting to complete the conversion as shown in Figure 10.<br/><br/><img alt="" class="layerphoto" height="164" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053244_1151.jpg" width="200"/><br/><strong>Figure 10. Sequential steps to complete the conversion from text to columns.</strong> Make sure that âDelimitedâ is indicated in step 1 and that the delimiter âotherâ is indicated with a semicolon as the delimiter. Follow the standard settings in step 3.<br/><br/></li><li>Select all rows from Row 39 and sort in âColumn Lâ for âLargest to smallestâ as shown in Figure 11.<br/><br/><img alt="" class="layerphoto" height="130" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053315_9410.jpg" width="200"/><br/><strong>Figure 11. Example of the selected rows from Row 39 and of the sorting from largest to smallest from all rows below Row 39 <br/><br/></strong> </li><li>The variants &gt; 0.015 in column L are selected for further analysis. Column K represents the forward and reverse reads of the position divided by a comma. Only variants with total reads (forward + reverse) above 1000 are considered (Figure 12).<br/><br/><img alt="" class="layerphoto" height="131" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053650_7521.jpg" width="200"/><br/><strong>Figure 12. The cells shown in yellow are the variants that are considered for further analysis</strong> </li></ol><br/></li><li>Crosscheck the frequency of the variants detected by MuTect with the variants of mtDNA server and report possible insertions/deletions.</li><li>If there is a discrepancy, calculate the frequency given by the coverage file as follows:<br/><br/><img alt="" class="layerphoto" height="55" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053832_4364.jpg" width="500"/><br/><br/></li><li>Annotate the variant by looking up the region on <a href="http://mitowheel.org/mitowheel.html." target="_blank">http://mitowheel.org/mitowheel.html.</a> An example is shown in Figure 13. <br/><br/><img alt="" class="layerphoto" height="102" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702053953_2306.jpg" width="200"/><br/><strong>Figure 13. Results of the annotation provided by mitoWheel for the position 7306</strong><br/><br/></li><li>If the variant is in a protein-coding region, identify the potential amino acid change on <a href="http://mitimpact.css-mendel.it/." target="_blank">http://mitimpact.css-mendel.it/.</a> An example is shown in Figure 14.<br/><br/><img alt="" class="layerphoto" height="101" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702054015_0758.jpg" width="200"/><br/><strong>Figure 14. Results for the variants that cause an amino acid change on the position 7306 provided by MitImpact2</strong><br/><br/></li><li>Check if the amino acid change is pathogenic by uploading the amino acid sequence and amino acid changes on <a href="http://mutpred.mutdb.org/." target="_blank">http://mutpred.mutdb.org/.</a> An example is shown in Figure 15.<br/><br/><img alt="" class="layerphoto" height="200" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702054034_5277.jpg" width="115"/><br/><strong>Figure 15. Results of the pathogenicity predictor MutPred2 for the amino acid changes M468T and M468K in the MT-CO1 gene</strong><br/><br/></li><li>Certain regions were excluded from the analysis. These excluded regions can be found in the supplementary data of Zambelli <em>et al.</em>, 2018 (<a href="https://www.cell.com/stem-cell-reports/fulltext/S2213-6711(18)30224-8#secsectitle0020" target="_blank">https://www.cell.com/stem-cell-reports/fulltext/S2213-6711(18)30224-8#secsectitle0020</a>).<br/></li></ol><p class="pptt" id="biaoti28234">### Recipes</p><ol class="npwtwok"><li>Gel electrophoresis<br/>50 ml of 1x TBE buffer<br/>0.5 g agarose</li><li>Alkaline Lysis Buffer<br/><img alt="" class="layerphoto" height="88" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702054340_4730.jpg" width="210"/><br/></li><li>EBT (Elution Buffer with Tris) buffer<br/>10 mM Tris-HCl pH 8.5<br/>1% Tween 20<br/></li></ol><p class="pptt" id="biaoti28235">### Acknowledgments</p><p class="ppzz">This work was funded by the Wetenschappelijke Fonds Willy Gepts (UZ Brussel), the Fonds voor Wetenschappelijk Onderzoek (FWO, 1506616N) and the Methusalem grant of the Vrije Universiteit Brussel to Prof. Karen Sermon.</p><p class="pptt" id="biaoti28236">### Competing interests</p><p class="ppzz">None.</p><p class="pptt" id="biaoti28237">### Ethics</p><p class="ppzz">In this protocol, we have used human mtDNA for optimization only. All donors gave their informed consent and all studies related to this protocol received approval of appropriate local ethical committees.</p><p class="pptt" id="biaoti28238">### References</p><ol class="npwtwok"><li>Bai, R. K. and Wong, L. J. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15073091" target="_blank">Detection and quantification of heteroplasmic mutant mitochondrial DNA by real-time amplification refractory mutation system quantitative PCR analysis: a single-step approach.</a> <em>Clin Chem</em> 50(6): 996-1001.</li><li>Just, R. S., Irwin, J. A. and Parson, W. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26009256" target="_blank">Mitochondrial DNA heteroplasmy in the emerging field of massively parallel sequencing.</a> <em>Forensic Sci Int Genet</em> 18: 131-139.</li><li>Rohlin, A., Wernersson, J., Engwall, Y., Wiklund, L., Bjork, J. and Nordling, M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19347965" target="_blank">Parallel sequencing used in detection of mosaic mutations: comparison with four diagnostic DNA screening techniques.</a> <em>Hum Mutat</em> 30(6): 1012-1020.</li><li>Spits, C., Le Caignec, C., De Rycke, M., Van Haute, L., Van Steirteghem, A., Liebaers, I. and Sermon, K. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17487184" target="_blank">Whole-genome multiple displacement amplification from single cells.</a> <em>Nat Protoc</em> 1(4): 1965-1970.</li><li>Ye, F., Samuels, D. C., Clark, T. and Guo, Y. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24859348" target="_blank">High-throughput sequencing in mitochondrial DNA research.</a> <em>Mitochondrion</em> 17: 157-163.</li><li>Zambelli, F., Mertens, J., Dziedzicka, D., Sterckx, J., Markouli, C., Keller, A., Tropel, P., Jung, L., Viville, S., Van de Velde, H., Geens, M., Seneca, S., Sermon, K. and Spits, C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29910126" target="_blank">Random mutagenesis, clonal events, and embryonic or somatic origin determine the mtDNA variant type and load in human pluripotent stem cells.</a> <em>Stem Cell Reports</em> 11(1): 102-114.</li><li>Zambelli, F., Vancampenhout, K., Daneels, D., Brown, D., Mertens, J., Van Dooren, S., Caljon, B., Gianaroli, L., Sermon, K., Voet, T., Seneca, S. and Spits, C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28832570" target="_blank">Accurate and comprehensive analysis of single nucleotide variants and large deletions of the human mitochondrial genome in DNA and single cells.</a> <em>Eur J Hum Genet</em> 25(11): 1229-1236.</li><li>Zhang, W., Cui, H. and Wong, L. J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22777720" target="_blank">Comprehensive one-step molecular analyses of mitochondrial genome by massively parallel sequencing.</a> <em>Clin Chem</em> 58(9): 1322-1331.<br/></li></ol>