<body>
            <h1>Plaque Assay to Determine Invasion and Intercellular Dissemination of <em>Shigella flexneri</em> in TC7 Human Intestinal Epithelial Cells</h1>
            <p><strong>Authors:</strong> Atin  Sharma, Andrea Puhar</p>
            <h2>Abstract</h2>
            <p><em>Shigella flexneri</em> invades the epithelial cells lining the gut lumen and replicates intracellularly. The specialized Type III Secretion System (T3SS) and its effector proteins, encoded on a large virulence plasmid, assist the bacterium to gain access to the cytosol. Thereafter <em>Shigella</em> disseminates to neighboring cells in an epithelial layer without further extracellular steps. Host cell lysis occurs when these bacteria have extensively replicated in the target cell cytosol. Here we describe a simple method to qualitatively as well as quantitatively study the capacity of <em>Shigella</em> to invade and disseminate within an epithelium by assessing the number and size of plaques representing the dead cells in a monolayer of TC7 cells. This classical protocol follows a simple approach of infecting the monolayers of epithelial cell lines with <em>Shigella</em> and visualizing the dead cells as plaques formed against a stained background.</p>
            <p><strong>Keywords:</strong> Intracellular bacteria, Intestinal epithelial cells, Giemsa stain, <em>Shigella</em>, Invasion, Dissemination</p>
            <h2>Background</h2><p>Shigellosis is a gastrointestinal infection caused by one of four species of <em>Shigella</em> and resulting in about 600,000 deaths annually (Reference 15). Although the infections are reported worldwide, children aged 1-4 years in low- or middle-income countries bear most of the burden of the disease (Kotloff <em>et al.</em>, 2018). The bacterium infects the colonic epithelium where it gains access to the cytoplasm and replicates. This intracellular multiplication is followed by dissemination to adjacent cells and modulation of the immune response, eventually leading to cell death and destruction of the colonic mucosa (Ashida <em>et al.</em>, 2015). Gene mutations followed by <em>in vitro</em> assays such as the gentamicin protection assay or the plaque formation assay, have led to identification of various virulence factors of <em>Shigella</em> that assist the bacterium in invasion and dissemination. Most of these virulence factors are found on a giant virulence plasmid. Notably, the virulence plasmid encodes the Type 3 Secretion System (T3SS) as well as its effectors and regulators (Puhar and Sansonetti, 2014). The T3SS system is necessary for invasion from the extracellular space. Once in the cytosol, <em>Shigella</em> moves via polarization of actin at one bacterial pole owing to the surface protein IcsA. These actin filaments (so-called âactin cometsâ) propel the bacteria into adjacent cells, which allows them to disseminate without further extracellular steps (Bernardini <em>et al.</em>, 1989). At the molecular level, dissemination is poorly understood, but it is thought to depend on the T3SS, similarly to invasion (Schroeder and Hilbi, 2008).<br/><br/>The plaque assay is a classical functional test to study two important steps in the <em>Shigella</em> infection cycle, invasion events and bacterial dissemination (Oaks <em>et al.</em>, 1985). Intracellular replication and dissemination can alternatively be observed by fluorescence microscopy. However, this approach is more laborious and mostly qualitative, not quantitative. Many virulence factors have been identified using the plaque assay as the primary evaluation of the bacterial ability to disseminate to adjacent cells. After initial bacterial entry, the infected cells are covered with an agarose overlay containing gentamicin to eliminate the external bacteria resulting from unproductive spreading into the extracellular medium instead of adjacent cells, thereby ensuring that every plaque corresponds to a single invasion event at the beginning of the experiment. After 48-72 h of incubation, dead cells will have detached and are washed away. The intact cell layer is counterstained with Giemsa, resulting in a clear plaque that is visible against the colored background. The number of plaques gives an estimate of the invasive capacity of the bacteria under study. For example, invasive-deficient <em>Shigella</em> such as the <em>mxiD</em> mutant (Allaoui <em>et al.</em>, 1993) or the virulence plasmid cured strain BS176 (Sansonetti <em>et al.</em>, 1982) will not form any plaques. On the other hand, the size of plaques reflects the bacteriaâs ability to spread within an epithelium. For example, a strain that is fully impaired in dissemination such as virG/icsA mutants (Makino <em>et al.</em>, 1986; Bernardini <em>et al.</em>, 1989) will show no plaque formation. In contrast, strains that spread more slowly such as the <em>ipgD</em> mutant will lead to the formation of smaller plaques with respect to wild-type bacteria (Puhar <em>et al.</em>, 2013; Koestler <em>et al.</em>, 2018). This protocol can be used to compare the ability of different <em>Shigella</em> strains, including mutants, to disseminate from the initial site of infection. The protocol, however, is less suitable to assess the invasiveness of bacterial strains, which is readily achieved with the less time-consuming gentamicin protection assay. This protocol is an improvement of the classic plaque assay protocol described by Oaks <em>et al.</em> (1985). <em>Shigella</em> is an enteric pathogen and one of its primary targets are intestinal epithelial cells. While Oaks <em>et al.</em> (1985) and most of the literature thereafter used HeLa cells, an epithelial cell line originating from a cervix cancer sample, TC7 cells are intestinal epithelial cells. The detailed step-by-step nature of this protocol makes it highly reproducible and it can be applied to other <em>Shigella</em> species. Further, it can be modified to extend to other intracellular bacteria such as <em>Salmonella</em> or <em>Listeria</em>, or with other epithelial cell lines.</p>
        </body>

<p class="pptt" id="biaoti28344">## Materials and Reagents</p><ol class="npwtwok"><li>Conical flask (Duran, Catalog number: 2121628)</li><li>Microfuge tubes (Eppendorf, Safe-Lock tubes, catalog numbers: 0030120086, 0030120094)</li><li>Centrifuge tubes (Sarstedt, catalog numbers: 62.554.502, 62.547.254)</li><li>Culture tube (TPP, catalog number: 91016)</li><li>Pipette tips (VWR, catalog numbers: 89041-404, 89041-412, 89041-400)</li><li>Serological pipettes (VWR, catalog numbers: 612-3702, 612-3700, 612-3698)</li><li>6-well tissue culture test plates (TPP, catalog number: 92406)</li><li>Cell counter slides (Countess Cell Counting Chamber Slides, Thermo Scientific, catalog number: C10228)</li><li>TC7 cells (Chantret <em>et al.</em>, 1994)</li><li><em>Shigella flexneri</em> M90T (Sansonetti <em>et al.</em>, 1982), can also be purchased from ATCC and available on request from the authors</li><li>Dulbecco's Modified Eagle Medium (DMEM) (Gibco, catalog number: 21885-025)</li><li>Penicillin-Streptomycin solution (10,000 U/ml) (Gibco, catalog number: 15140122)</li><li>Minimum Essential Medium Non-Essential Amino Acids solution (100x) (Gibco, catalog number: 11140-035)</li><li>Dulbeccoâs Phosphate Buffered Saline (PBS) solution (Gibco, catalog number: 14190-144)</li><li>Fetal Bovine Serum (Gibco, catalog number: 10500056)</li><li>HEPES 1 M solution (Gibco, catalog number: 15630-080)</li><li>Trypsin-EDTA (0.05%), phenol red (Gibco, catalog number: 25300-054)</li><li>Trypan Blue Stain (0.4%) (Thermo Scientific, catalog number: T10282)</li><li>Gentamicin sulfate (Sigma, catalog number: G1264)</li><li>Agarose (VWR Life Science, catalog number: 35-1020)</li><li>Tryptic Soy Broth (TSB) ready to use powder (Merck, catalog number: 105459)</li><li>Tryptic Soy Agar (TSA) ready to use powder (Merck, catalog number: 105458)</li><li>Congo red (Sigma, catalog number: C6277)</li><li>Ethanol (VWR chemicals, catalog number: 20821.558)</li><li>Giemsa's azur eosin methylene blue solution (Merck, catalog number: 109204)</li><li>Growth medium (see Recipes)</li><li>Infection buffer (see Recipes)</li><li>Agarose overlay (see Recipes)</li><li>5% (v/v) Giemsa solution (see Recipes)</li><li>Congo red solution (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28345">### Equipment</p><ol class="npwtwok"><li>Biosafety cabinet (Thermo Scientific, HERAsafe KS 18)</li><li>Spectrophotometer (Amersham, Ultrospec 2100 pro)</li><li>Pipetteman (VWR, Accurpette)</li><li>CO<sub>2</sub> Incubator (Thermo Scientific, Heracell VIOS 160i)</li><li>Benchtop centrifuge (VWR, Micro Star 17R)</li><li>Centrifuge (Eppendorf, 5810R with Rotor A-4-81 for plates)</li><li>Water bath (Grant, JBA 12)</li><li>Pipettes (Eppendorf, Research plus)</li><li>Inverted microscope (Motic, AE2000 Binocular)</li><li>Cell counter (Thermo Scientific, Countess II FL)</li><li>Imaging system (Bio-Rad, ChemiDoc XRS)</li><li>Microwave oven (Whirlpool, MD101)</li><li>Disposable scalpel (Swann-Morton, 0503)</li><li>Vacuum pump (VWR, Mini diaphragm vacuum pump VP 86)</li><li>Orbital shaker (Edmund BÃ¼hler, Swip SM25)<br/></li></ol><p class="pptt" id="biaoti28346">### Software</p><ol class="npwtwok"><li>Prism 7 (GraphPad, <a href="https://www.graphpad.com/" target="_blank">https://www.graphpad.com</a>)</li><li>ImageJ (<a href="https://imagej.nih.gov/ij/index.html" target="_blank">https://imagej.nih.gov/ij/index.html</a>)<br/></li></ol><p class="pptt" id="biaoti28347">### Procedure</p><p class="ppzz"><em>Note: The experiment should be carried out in a biosafety level 2 Lab.<br/><br/></em> </p><ol class="npwone"><li>Preparation of bacteria<br/><strong>Day 1</strong> <ol class="npwtwo"><li>Prepare or melt tryptic soy agar (TSA), as per manufacturerâs instructions, and let it cool down to about 45Â°C (<em>e.g.</em>, in a water bath already set at 45 Â°C). Add sterile Congo red solution (see Recipes) to a final concentration of 0.01%, <em>e.g.</em>, 5 ml of Congo red solution to 500 ml of TSA. Pour the plates (about 25 ml of TSA in each 10 cm diameter plate) next to an open flame or in a sterile cabinet and let them dry. Store the plates at 4Â°C and use them within 3 weeks.</li><li>In the afternoon, streak out <em>Shigella flexneri</em> from the glycerol stock (see Notes) on a TSA plate supplemented with Congo red. Grow overnight (approximately 18 h) at 37 Â°C.</li></ol><strong>Day 2</strong> <ol class="npwtwo" start="3"><li>Inoculate 5 ml of TSB with a single red colony (Figure 1) of <em>Shigella flexneri</em> and incubate overnight at 37 Â°C with 180 rpm agitation.</li></ol><strong>Day 3</strong> <ol class="npwtwo" start="4"><li>Dilute the overnight culture 200 times in fresh TSB in a conical flask. For instance, in a 125 ml flask, take 25 ml of TSB and add 125 Âµl of the overnight culture. Incubate at 37 Â°C with 180 rpm agitation and grow till mid log-phase (OD<sub>600nm</sub> = 0.3-0.4). The doubling time of <em>Shigella</em> during the exponential phase is about 40 min. It typically takes about 3 h to reach the mid log-phase.<br/><br/><img alt="" class="layerphoto" height="136" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701184528_1711.jpg" width="100"/><br/><strong>Figure 1. Characteristic red colonies of <em>Shigella flexneri</em> on TSA supplemented with 0.01% Congo red</strong><br/><div><br/></div></li></ol></li><li>Preparation of TC7 cells<br/><strong>Day 2</strong> <ol class="npwtwo"><li>Remove the growth medium (see Recipes) from a cell culture flask with a confluent monolayer of TC7 cells.</li><li>Wash the cells with sterile PBS three times.</li><li>Add 5 ml of pre-warmed Trypsin solution and incubate the flask at 37 Â°C for about 4 min.</li><li>Add 5 ml of growth medium (see Recipes) to inhibit the Trypsin and collect the trypsinized cells in a 50 ml centrifuge tube.</li><li>Centrifuge at 900 <em>x g</em> and discard the supernatant. Re-suspend the pellet in 5-10 ml of growth medium devoid of antibiotics (see Recipes) by gentle pipetting. </li><li>In a sterile 1.5 ml microcentrifuge tube, mix 10 Âµl of cell suspension with 10 Âµl of Trypan blue solution. Load 10 ml of this mixture on to the Countess II counting slide and using a Countess II cell counter, determine the number of cells.</li><li>Seed the required number of wells in a 6-well plate with 1 million cells (1 x 10<sup>6</sup>) per well, adding at least 2 ml of growth medium devoid of antibiotics (see Recipes) in each well. This number of cells is enough to form a fully confluent monolayer after one day of growth (Figure 2).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701184626_3730.jpg" width="103"/><br/><strong>Figure 2. Bright-field image of a monolayer of TC7 cells</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Infection<br/><strong>Day 3</strong> <ol class="npwtwo"><li>On Day 3, when the bacteria reach OD<sub>600nm</sub> = 0.3-0.4, pipette 1-1.5 ml culture into a microcentrifuge tube.</li><li>Spin the tube at 3,000 <em>x g</em> for 5 min. Aspirate the supernatant. Add an equal volume of PBS at room temperature and resuspend the pellet by pipetting up and down or by gentle vortexing.</li><li>Repeat Step C2 at least twice and finally resuspend the pellet in infection buffer at room temperature. Record the OD<sub>600nm</sub> of the bacterial suspension.<br/>The recommended multiplicity of infection (MOI) for this experiment is very low, 1 bacterium for every 150 cells. Since there are 1 x 10<sup>6</sup> cells in each well, the number of bacteria required per well is about 6,700.<br/>The number of <em>Shigella</em> cells at OD<sub>600nm</sub> = 1 is 0.5 x 10<sup>9</sup>/ml; hence at OD = z, the number of bacterial cells = z x 0.5 x 10<sup>9</sup>. Calculate the volume of bacterial suspension required for infecting each well using the following formula:<br/>Volume of bacterial suspension required (in ml) = 6,700/z x 0.5 x 10<sup>9</sup><br/>Additionally, make 10-fold serial dilutions of the bacterial suspension and spread 100 Âµl of each on TSA plates. Incubate the plates at 37 Â°C and count the colonies obtained the next day. This colony count of the inoculum will verify the number of bacteria used.</li><li>Aspirate the medium from the TC7 cells plate and add 2 ml of PBS at room temperature to each well.</li><li>Swirl the plate gently and carefully aspirate all the liquid. Wash the cells twice in a similar fashion and add 2 ml of infection buffer (see Recipes) at room temperature to each well.</li><li>Add the desired volume of bacterial suspension (calculated in Step C3) to each well of the 6-well plate. Mix well by gentle pipetting to ensure that the bacterial suspension is mixed uniformly. Centrifuge the plate (using the plate adaptors in the centrifuge rotor) at 180 <em>x g</em> for 10 min at room temperature.</li><li>Incubate the plate at 37 Â°C, 10% CO<sub>2</sub>, for 2 h.</li><li>After incubation, wash the wells with prewarmed PBS three times as in Steps C4 and C5.</li><li>Completely aspirate the liquid from the wells and cover cells with 2 ml of the cooled agarose overlay (see Recipes).</li><li>Put the plates back at 37 Â°C, 10% CO<sub>2</sub>. Examine the plates daily for appearance of plaques. Two days incubation can be enough for optimal plaque size (1-2 mm diameter). However, it is dependent on the bacterial strain and frequent observation is therefore required.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Visualization and enumeration of plaques<br/><strong>Day 5 or 6</strong> <ol class="npwtwo"><li>After 48-72 h of incubation, very carefully peel off the agarose overlay using a scalpel to detach the overlay from the walls and by turning around the plate.</li><li>Fix the cells by incubation in 95% ethanol for 5 min.</li><li>Remove the ethanol and stain the wells with 5% Giemsa solution (see Recipes) for 10 min.</li><li>Aspirate the stain and wash the wells twice with distilled water. Let the plate air-dry.</li><li>When the plate is dry, take a picture of the plate using an imaging system (Figure 3).</li><li>Count the number of plaques formed in each well with a marker pen.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701184656_9037.jpg" width="220"/><br/><strong>Figure 3. Visual assessment of the plaques formed on the cellular monolayer.</strong> A. Clear plaques representing dead cells in a well of 6-well plate with TC7 monolayer stained with Giemsa. B. Bright-field micrograph of a plaque edge. The clear zone is the plaque and the interface of plaque and the TC7 monolayer shows characteristic damage to the cells.</li></ol></li></ol><p class="pptt" id="biaoti28348">### Data analysis</p><ol class="npwtwok"><li>Using the ImageJ software determine the diameter of each plaque in the image.</li><li>Using the Prism software plot the number or the average diameter of plaques against the condition (<em>e.g.</em>, the strain) or the MOI used in particular well (Figure 4). For every condition tested, prepare at least duplicate samples in every experiment to serve as the technical replicates. For biological replicates, repeat the experiment on another day with the same number of bacteria and cells at least 3 times.</li><li>To compare only two sample values at a fixed time point, use Studentâs <em></em><em>t</em>-test. However, to assess the statistical significance of multiple samples across multiple time points, use two-way ANOVA with Tukeyâs <em>post-hoc</em> test.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701184722_9981.jpg" width="128"/><br/><strong>Figure 4. An example plot of the number of plaques at a particular MOI for different bacterial strains.</strong> Each dot represents an independent value obtained from a replicate.<br/></li></ol><p class="pptt" id="biaoti28349">### Notes</p><ol class="npwtwok"><li>To prepare glycerol stocks of <em>Shigella flexneri</em>, dilute the overnight culture 200 times in fresh TSB and incubate at 37 Â°C with 200 rpm shaking. Monitor the OD<sub>600</sub> of the culture periodically. At OD<sub>600</sub> = 0.6, withdraw 1 ml of culture in a 2 ml cryovial and add sterile glycerol to a concentration of 15-50% (v/v). Mix gently and store the tube at -80 Â°C. The frozen stocks are stable for years at -80 Â°C, however, multiple freeze-thaw cycles may reduce the shelf life.</li><li>Carefully select only the red colonies on Congo red agar for the experiment. The off-white colonies have lost the virulence plasmid and are invasion-deficient. If plates are not used immediately after overnight culture, but kept at 4 Â°C (which is possible for up to 3 weeks), white colonies need to be marked because they will turn red due to unspecific binding of Congo Red. Red staining of invasive colonies only develops during growth above 35 Â°C.</li><li>Congo red is able to induce secretion via the Type III Secretion System (T3SS) in <em>Shigella</em>. The dye binds to the bacterial colonies that have an active T3SS making them appear red in color. It is a simple and quick screening method to differentiate virulent <em>Shigella</em> colonies from the avirulent ones (due to curing of the virulence plasmid). Unfortunately, the mechanism of action of Congo red is not understood.</li><li>The TC7 cells are cultured and maintained in presence of Penicillin and Streptomycin (100 U/ml). However, infections are carried out in infection buffer that does not contain any antibiotics (which may inhibit the bacterial growth). Ideally, cells are seeded in medium devoid of antibiotics if used the next day or at least extensively washed at the beginning of the experiment. </li><li>The multiplicity of infection (MOI) for this experiment is to be kept very low, because neither the plaque number nor the plaque size can be determined properly when plaques are too close to each other. The MOI can be empirically determined by carrying out experiments with various dilutions of the bacterial suspension. We recommend one bacterial cell per 150 or 300 TC7 cells. </li><li>The 6-well plates are centrifuged after addition of bacteria to aid in binding as <em>Shigella</em> lacks adhesion factors.</li><li>The agarose overlay should not be hot while pouring. It may kill the cells.</li><li>Even during longer incubations, gentamicin wonât be able to penetrate the cells and kill the intracellular bacteria. The intracellular bacteria remain âprotectedâ all the time.</li><li>Keep all the buffers and reagents that are required at the initiation of invasion and thereafter, warmed up to 37 Â°C before using. The secretion of T3SS effectors mediating invasion is temperature dependent. It is active above 35 Â°C, but inactive at room temperature. In order to synchronize the invasion, it is important to keep the bacteria in buffers at room temperature before invasion.</li><li>All the reagents can be prepared on Day 1, except for the agarose overlay and Giemsa solution, which must be prepared freshly.</li><li>When using the pipette controller, gently dispense the solutions with gravity (âGâ setting in Accurpipette) or at the lowest flow speed into the 6-well plate to avoid detaching the cell monolayer.</li><li>Be careful while swirling the plate, washing the cells and during centrifugation, because the confluent cell monolayer detaches easily. </li><li>The average number and diameter of the plaque cannot be substituted with the measure of total plaque area. The number of plaques actually reflects the number of invasion sites. The diameter of the plaque reflects the capacity of the bacterium to disseminate. Therefore, total plaque area measurement wonât allow distinguishing between a bacterial strain that is poorly invasive but good at disseminating from a strain that is proficient in invasion but is poor at disseminating.</li><li>For further reading and details on basic methods that have not been described in this manuscript (such as bacterial inoculation, pouring the plates, <em>etc.</em>) the reader is referred to the following books: Microbiology: A Laboratory Manual (Cappuccino and Welsh, 2017) and Culture of Animal Cells: A Manual of Basic Technique and Specialized Applications (Freshney, 2010).<br/></li></ol><p class="pptt" id="biaoti28350">### Recipes</p><ol class="npwtwok"><li>Congo red solution<br/>1% (w/v) Congo red in water<br/>Weigh 1 g of Congo red dye and dissolve it in 70 ml of water. Bring the volume of the solution to 100 ml and filter sterilize the solution, using 0.2 Âµm membrane filters. The Congo red solution is stable for a long time at room temperature</li><li>Growth medium<br/>DMEM<br/>1x amino acid solution<br/>100 U/ml Pen-Strep solution<br/>10% heat inactivated FBS<br/>To 500 ml DMEM, add 5 ml of Pen-Strep solution, 5 ml of amino acid solution and 50 ml of FBS (inactivated by incubation at 55 Â°C for 30 min)<br/><em>Note: The growth medium must be stored at 4 Â°C and can be used for 3-4 weeks.</em> </li><li>Infection buffer<br/>DMEM (serum free)<br/>20 mM HEPES, pH 7.4<br/>To 50 ml of serum free DMEM add 1 ml of 1 M sterile HEPES pH 7.4<br/><em>Note: Infection buffer is stable at 4 Â°C for months.</em> </li><li>Agarose overlay<br/>0.5% (w/v) agarose in infection buffer<br/>5% FBS<br/>50 Âµg/ml gentamicin<ol class="npwthree"><li>Add 0.1 g of agarose to 20 ml of infection buffer</li><li>Melt the agarose in a microwave oven</li><li>When it cools down to about 50 Â°C, add 1 ml of heat-inactivated FBS and 20 Âµl of 50 mg/ml gentamicin solution</li><li>Allow the agarose to cool down to about 45 Â°C in a water bath, so that it is still liquid but not too hot</li></ol><em>Note: Agarose overlay needs to be prepared fresh every time before use and cannot be stored.</em> </li><li>5% (v/v) Giemsa solution <br/>Prepare a 20 ml working Giemsa solution by using 1 ml of stock Giemsa solution to 19 ml of PBS<br/>Keep it in a dark bottle/tube<br/><em>Note: Do not store it and always use it as a freshly prepared solution.</em><br/></li></ol><p class="pptt" id="biaoti28351">### Acknowledgments</p><p class="ppzz">We thank members of the Puhar lab for critical reading. A.S. is the recipient of a stipend from The MIMS Excellence by Choice Postdoctoral Programme under the patronage of Emmanuelle Charpentier. The programme is financed by the Kempe Foundations and the Knut and Alice Wallenberg Foundation. A.P. acknowledges generous funding from MIMS, UCMR, UmeÃ¥ University and the Wallenberg Academy Fellow programme. This protocol was used in Puhar <em>et al.</em> (2013).</p><p class="pptt" id="biaoti28352">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti28353">### References</p><ol class="npwtwok"><li>Allaoui, A., Sansonetti, P. J. and Parsot, C. (1993). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8437520" target="_blank">MxiD, an outer membrane protein necessary for the secretion of the <em>Shigella flexneri</em> lpa invasins.</a> <em>Mol Microbiol</em> 7(1): 59-68.</li><li>Ashida, H., Mimuro, H. and Sasakawa, C. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25999954" target="_blank"><em>Shigella</em> manipulates host immune responses by delivering effector proteins with specific roles.</a> <em>Front Immunol</em> 6: 219.</li><li>Bernardini, M. L., Mounier, J., d'Hauteville, H., Coquis-Rondon, M. and Sansonetti, P. J. (1989). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2542950" target="_blank">Identification of icsA, a plasmid locus of <em>Shigella flexneri</em> that governs bacterial intra- and intercellular spread through interaction with F-actin.</a> <em>Proc Natl Acad Sci U S A</em> 86(10): 3867-3871.</li><li>Cappuccino, J. G. and Welsh, C.T (2017). <em>Microbiology: A Laboratory Manual, 11th Edition.</em> ISBN: 978-0134298672.</li><li>Chantret, I., Rodolosse, A., Barbat, A., Dussaulx, E., Brot-Laroche, E., Zweibaum, A. and Rousset, M. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8175910" target="_blank">Differential expression of sucrase-isomaltase in clones isolated from early and late passages of the cell line Caco-2: evidence for glucose-dependent negative regulation.</a> <em>J Cell Sci</em> 107 (Pt 1): 213-225.</li><li>Freshney, R. I. (2010). <em>Culture of Animal Cells: A Manual of Basic Technique and Specialized Applications. Sixth Edition.</em> ISBN: 9780470528129.</li><li>Koestler, B. J., Fisher, C. R. and Payne, S. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30254126" target="_blank">Formate promotes <em>Shigella</em> intercellular spread and virulence gene expression.</a> <em>MBio</em> 9(5): e01777-18.</li><li>Kotloff, K. L., Riddle, M. S., Platts-Mills, J. A., Pavlinac, P. and Zaidi, A. K. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29254859" target="_blank">Shigellosis.</a> <em>Lancet</em> 391(10122): 801-812.</li><li>Makino, S., Sasakawa, C., Kamata, K., Kurata, T. and Yoshikawa, M. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3524856" target="_blank">A genetic determinant required for continuous reinfection of adjacent cells on large plasmid in <em>S. flexneri</em> 2a.</a> <em>Cell</em> 46(4): 551-555.</li><li>Oaks, E. V., Wingfield, M. E. and Formal, S. B. (1985). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3884506" target="_blank">Plaque formation by virulent <em>Shigella flexneri</em>.</a> <em>Infect Immun</em> 48(1): 124-129.</li><li>Puhar, A. and Sansonetti, P. J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25202865" target="_blank">Type III secretion system.</a> <em>Curr Biol</em> 24(17): R784-791.</li><li>Puhar, A., TronchÃ¨re, H., Payrastre, B., Nhieu, G. T. and Sansonetti, P. J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24332032" target="_blank">A <em>Shigella</em> effector dampens inflammation by regulating epithelial release of danger signal ATP through production of the lipid mediator PtdIns5P.</a> <em>Immunity</em> 39(6): 1121-1131.</li><li>Sansonetti, P. J., Kopecko, D. J. and Formal, S. B. (1982). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6279518" target="_blank">Involvement of a plasmid in the invasive ability of <em>Shigella flexneri</em>.</a> <em>Infect Immun</em> 35(3): 852-860.</li><li>Schroeder, G. N. and Hilbi, H. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18202440" target="_blank">Molecular pathogenesis of <em>Shigella</em> spp.: controlling host cell signaling, invasion, and death by type III secretion.</a> <em>Clin Microbiol Rev </em>21(1): 134-156.</li><li>Shigellosis-Chapter 3-2018 Yellow Book | Travelersâ Health | CDC. Available at: <a href="https://wwwnc.cdc.gov/travel/yellowbook/2018/infectious-diseases-related-to-travel/shigellosis" target="_blank">https://wwwnc.cdc.gov/travel/yellowbook/2018/infectious-diseases-related-to-travel/shigellosis.</a><br/></li></ol>