<body>
            <h1>Optogenetic Inactivation of Transcription Factors in the Early Embryo of <em>Drosophila</em></h1>
            <p><strong>Authors:</strong> Stephen L. McDaniel, Melissa M. Harrison</p>
            <h2>Abstract</h2>
            <p>The early embryo of <em>Drosophila melanogaster</em> exists as a rapidly dividing syncytium of nuclei that are transcriptionally silent. Maternally deposited factors are required to awaken the genome and assist in the transition from maternal to zygotic control of development. Because many of these essential factors are maternally deposited and the early nuclear divisions are so rapid, it has been difficult to assess the functional role of transcription factors at discrete points in early embryonic development. To address this issue, we have developed an optogenetic system that can rapidly and reversibly inactivate transcription factors with nuclear-cycle resolution. The temporal precision enabled by this technique will allow a mechanistic understanding of how transcription factors function together to control genome activation and patterning in the early embryo and is likely broadly applicable to factors throughout embryogenesis.</p>
            <p><strong>Keywords:</strong> Optogenetics, CRY2, <em>Drosophila</em>, Transcription factor, Gene expression, Zygotic genome activation, Embryo, RNA-seq</p>
            <h2>Background</h2><p>The early embryo is transcriptionally silent and development is driven by maternally deposited mRNAs and proteins (Hamm and Harrison, 2018; Schulz and Harrison, 2019). In <em>Drosophila</em>, the nuclei exist in a syncytium and undergo rapid nuclear divisions, consisting only of synthesis and mitosis phases (Foe and Alberts, 1983). During the 14<sup>th</sup> nuclear cycle (NC), the genome of the developing embryo is transcriptionally up-regulated during the major wave of zygotic genome activation (ZGA) (Hamm and Harrison, 2018; Schulz and Harrison, 2019). Prior to widespread ZGA at NC14, a small subset of genes, mostly encoding transcription factors, is transcribed during a minor wave of ZGA (McDaniel <em>et al.</em>, 2019). This entire process from egg laying to widespread ZGA takes less than three hours, which has made it technically challenging to dissect the role various transcription factors play in ZGA. Further, while genetic techniques are available to maternally deplete essential transcription factors, this removes these factors from the embryo throughout early development, making it impossible to tease apart their roles during the minor and major waves of ZGA. Other systems have used techniques like Anchor-Away or proteasome targeting to deplete nuclear proteins (Haruki <em>et al.</em>, 2008; Caussinus <em>et al.</em>, 2011), however, these methods take too long to inactivate their targets to work on the timescale necessary to study in the early embryo. Further, these techniques are not reversible on the timescales needed to study transcription-factor activity in the early embryo. To address these limitations, we developed an optogenetic system to rapidly and reversibly inactivate the essential transcription factor Zelda (Zld) in the early embryo of <em>Drosophila</em> with nuclear-cycle resolution. <br/><br/>Optogenetic protocols require the addition of a photo-responsive protein tag to the protein of interest, either at the endogenous locus or via a transgene. We note that both published examples of optogenetic transcription-factor inactivation in <em>Drosophila</em> have used an N-terminal tag, though it remains unclear if tagging at the N-terminus is required for inactivation (Huang <em>et al.</em>, 2017; McDaniel <em>et al.</em>, 2019). Using Cas9-mediated genome engineering, we endogenously tagged the N-terminus of Zld with the blue-light responsive CRY2 tag (McDaniel <em>et al.</em>, 2019). CRY2, originally characterized in <em>Arabidopsis</em>, is traditionally part of a two-component system, where CRY2 dimerizes with CIB1 upon exposure to blue light (Liu <em>et al.</em>, 2008). Both components are available for use in <em>Drosophila</em> (Guglielmi <em>et al.</em>, 2015). However, CRY2 alone has been demonstrated to be sufficient to inactivate transcription factors, albeit through different mechanisms; CRY2-tagged Bicoid remains on chromatin upon blue-light exposure, while CRY2-tagged Zelda dissociates from chromatin (Huang <em>et al.</em>, 2017; McDaniel <em>et al.</em>, 2019). The mechanism of inactivation is likely a conformational change upon blue-light exposure that disrupts the ability of the transcription factor to activate transcription. One benefit of the CRY2 system is its rapidity of inactivation; upon blue-light exposure, the CRY2-tagged protein is inactivated within seconds and, when the light is removed, refolds in ~5 min (Huang <em>et al.</em>, 2017). <br/><br/>While we utilized the one component CRY2 system for our experiments, this system depends on conformational changes that may be protein specific. Thus, inactivation must be confirmed experimentally. If the single-component system fails to inactivate protein function, the two-component system is available for long-term protein inactivation via cell-membrane sequestration (Guglielmi <em>et al.</em>, 2015). In the two-component system, CRY2 will dimerize with CIB1 within seconds, but takes over an hour to dissociate once the light is removed. In <em>Drosophila</em>, a cell membrane localized, GFP-tagged CIB1 has been developed and could be used to sequester a target protein away from chromatin using blue-light exposure when the nuclear envelope breaks down during mitosis (Guglielmi <em>et al.</em>, 2015). This would have a similar sequestration-mediated effect as the recently described Jabba-trap system (Seller <em>et al.</em>, 2019). Thus, depending on experimental needs, either the one or two-component system can be used.<br/><br/>Here, we will describe, in detail, how to use this versatile one-component system to reversibly inactivate a protein of choice with precise temporal resolution in the early embryo of <em>Drosophila</em> and harvest single embryos for downstream RNA-seq.</p>
        </body>

<p class="pptt" id="biaoti28379">## Materials and Reagents</p><p class="ppzz"><em>Note: Unless otherwise stated, materials/reagents are stored at room temperature.</em> </p><ol class="npwone"><li>Fly husbandry and embryo selection<ol class="npwtwo"><li>Microscope Slides (25 x 75 x 1 mm) (Fisher, catalog number: 125447)</li><li>Miracloth (Fisher, catalog number: 80058-394)</li><li>Glass Bottom Microwell Dishes (MatTek, catalog number: P35G-1.5-14-C)</li><li>Axygen 1.7 ml clear tubes (VWR, catalog number: 10011-722)</li><li>Bottles, 6 oz. bottles (VWR, catalog number: 75813-108)</li><li>Bottle plugs (VWR, catalog number: 89168-890)</li><li>Plates; 60 x 15 mm Petri dishes (Fisher, catalog number: FB0875713A)</li><li>Yeast (Lab Scientific, catalog number: FLY-8040-5F)</li><li>Molasses (Lab Scientific, catalog number: FLY-8008-16)</li><li>Bacteriological, Grade Agar (US Biological, catalog number: A0930)</li><li>Yellow Cornmeal (Lab Scientific, catalog number: FLY-8009-20)</li><li>p-hydroxy-benzoic acid methyl ester (Lab Scientific, catalog number: FLY-5501-5)</li><li>Bleach (5.25% Sodium Hypochlorite) (Staples, catalog number: COX02489)</li><li>Halocarbon oil 700 (Sigma, catalog number: H8898-50ML)</li><li>Ethyl Acetate (Fisher, catalog number: E1964)</li><li>Tegosept (Genesee Scientific, catalog number: 20-259)</li><li>Molasses Food (see Recipes)</li><li>Molasses plate (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA extraction and purification<ol class="npwtwo"><li>Parafilm (4 in. x 250 ft.) (VWR, catalog number: 52858-032)</li><li>Exel 27 G Hypodermic Disposable Needles (Fisher, catalog number: 148099)</li><li>Corning Barrier Tips, 1-20 Î¼l (Fisher, catalog number: 10320725)</li><li>Corning Barrier Tips, 1-200 Î¼l (Fisher, catalog number: 10320727)</li><li>Corning Barrier Tips, 100-1,000 Î¼l (Fisher, catalog number: 10320729)</li><li>RNase Zap (Fisher, catalog number: AM9780)</li><li>Trizol (Thermo Scientific, catalog number: 15596026) (store at 4 Â°C)</li><li>Ultra-Pure Glycogen (Fisher, catalog number: 10814-010) (store at -20 Â°C)</li><li>Chloroform (Fisher, ICN19400280)</li><li>Isopropanol (VWR, PL323ZA-4)</li><li>Ethanol, absolute (200 proof) (Fisher, catalog number: BP2818500)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Fly strains<br/><em>His2AvRFP (II)</em> (Bloomington <em>Drosophila</em> Stock Center (BDSC) #23651) and a CRY2-tagged line with His2Av-RFP in the background. A plasmid with the CRY2 tag can be obtained from Addgene (#26866). This sequence was used to generate a CRY2-containing donor vector that was injected along with a guide RNA into <em>w1118; PBac{y[+mDint2]=vas-Cas9}VK00027</em> (BDSC#51324) embryos by BestGene Inc. Edited alleles were selected using 3xP3 dsRED, and this cassette was subsequently removed. For details on the Cas9-mediated genome editing techniques used see (Hamm <em>et al.</em>, 2017). The RFP-tagged histone was used to visualize nuclear density during each nuclear division. If the gene of interest is on the second chromosome, His2AV-RFP on the third chromosome (Bloomington <em>Drosophila</em> Stock Center (BDSC) #23650) can be used. It is important to independently validate all lines that you generate or receive from outside sources. We fully sequenced the CRY2-ZLD locus to ensure that the repair event happened correctly and experimentally tested two independently derived CRY2-ZLD lines to ensure that they behaved identically in all experiments. All flies were grown on standard molasses food at 25 Â°C in an incubator on a 12-h light/dark cycle.<br/></li></ol><p class="pptt" id="biaoti28380">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>Embryo collection cage (Genesee Scientific, 59-100)</li><li>Dissection microscope (Nikon, SMZ1000)<br/></li><li>Nikon ECLIPSE Ti-E epiflurescence microscope with 470 Â± 20 nm and 572 Â± 35 nm filters</li><li>Embryo basket (Cantex, 5140104)</li><li>Pellet Pestles (Fisher, catalog number: 12-141-363)</li><li>Centrifuge</li><li>-20 Â°C freezer</li><li>-80 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti28381">### Procedure</p><ol class="npwone"><li>Fly husbandry and embryo selection<ol class="npwtwo"><li>Grow up 2-3 bottles of the <em>His2AvRFP (II)</em> and <em>CRY2; His2AvRFP (II)</em> flies on standard molasses food at 25 Â°C in incubators with 12-h light/dark cycles.<br/><em>Note: It is critically important to use the His2AvRFP (II) flies as a control for all optogenetic experiments. We controlled for three features causing non-specific changes in gene-expression: blue-light exposure alone, the presence of the CRY2 tag, and a slight developmental delay caused by extended blue-light exposure. Using His2AvRFP (II) flies exposed to identical blue-light regimens allowed us to identify gene sets that were changing from blue-light exposure rather than CRY2-mediated protein inactivation and exclude them from our downstream analyses.</em> </li><li>Empty the flies from 2-3 bottles into a small embryo collection cage with a molasses plate and a small dollop of wet yeast. Allow to recover for 24 h.</li><li>Allow the females to pre-lay on a fresh molasses plate for 1 h.</li><li>Carry out the appropriate time course to capture the timepoint of interest. For experiments focusing on NC10-14, we allowed the flies to lay for 60 min.</li><li>After the desired lay period, collect the embryos into a mesh embryo basket and thoroughly wash with deionized water.</li><li>Dechorionate the embryos for 3 min in 50% commercial bleach diluted with deionized water (final = 2.6% sodium hypochlorite solution).</li><li>Wash the dechorionated embryos with deionized water for 3 min.</li><li>Pick the embryos from the mesh basket into a drop of halocarbon oil 700 on a microscope slide using a small paint brush or other similar instrument.<br/><em>Note: Once dechorionated, the embryos are fragile. Be careful to treat them with care in all subsequent steps.</em> </li><li>Stage the embryos on a Nikon SMZ1000 (or similar) dissection microscope and select 5-7 embryos that are younger than the desired time points and move them to the coverslip of the glass bottom microwell dish along with a minimal amount of halocarbon oil 700. (We recommend using the Society for Developmental Biologyâs âThe Interactive Flyâ website <a href="https://www.sdbonline.org/sites/fly/aimain/2stages.htm" target="_blank">https://www.sdbonline.org/sites/fly/aimain/2stages.htm</a> as a resource for staging embryos.) Orient all but one embryo in the same orientation to make it easier to eventually pick the targeted embryo after blue-light exposure (Figure 1).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701231153_0246.jpg" width="141"/><br/><strong>Figure 1. An example of how to organize embryos on the coverslip for easy identification of the correct embryo after illumination.</strong> It is important to use a minimal amount of halocarbon oil 700 to mount the embryos. <br/><br/></li></ol></li><li>Optogenetic inactivation<ol class="npwtwo"><li>Move the mounted embryos to a Nikon ECLIPSE Ti-E (or similar) epifluorescence microscope for optogenetic inactivation. Using the RFP channel on the epifluorescence microscope, identify an embryo that is younger than the desired timepoint by determining the density of nuclei in a 2,500 Î¼m square (McDaniel <em>et al.</em>, 2019) (Table 1). <br/><em>Note: This step only applies to timepoints from NC10 and beyond as prior to NC10 the nuclei are in the interior of the embryo and cannot be easily visualized.</em><br/><br/><strong>Table 1. Staging </strong><em><strong>Drosophila</strong></em><strong> embryos by nuclear density.</strong> The length and nuclear density/2,500 Î¼m<sup>2</sup> for each NC between NC10 and NC14.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701232801_4701.jpg" width="300"/><br/><br/></li><li>Once an embryo has reached the desired nuclear cycle, start exposing the embryo to light filtered through a 470 Â± 20 nm filter. Every 30 s, switch to a 572 Â± 35 filter for a 30 ms exposure to visualize the His2Av-RFP signal. This method allows you to continually stage the embryo without allowing enough time in non-blue light for the CRY2-tagged protein to refold. Continue exposing the embryo to blue light with brief switches to image RFP until the embryo reaches the desired developmental stage. The length of time embryos are exposed to blue light will vary depending on experimental design.<br/><em>Note: It is helpful to have a motorized filter wheel to automate this process.</em> </li><li>At the completion of the time course, immediately pick the illuminated embryo onto a fresh molasses plate and, working quickly, roll it around briefly to remove any excess halocarbon oil 700 (Figure 2, Video 1).<br/><br/><img alt="" class="layerphoto" height="133" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701231231_3677.jpg" width="100"/><br/><strong>Figure 2. Removing excess oil from the embryo.</strong> Using your tool of choice, roll the embryo on a molasses plate until the excess Halocarbon 700 oil has been removed before moving the embryo to the Trizol/glycogen solution.<br/><br/><!--video3296-1å¼å§--><div class="videodiv" id="vd3296-1" style="height:480px;"></div><div class="videophoto" id="vp3296-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3296/3296-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Removal of excess Halocarbon 700 oil on a molasses plate</strong> </div><!--video3296-1ç»æ--><br/></li><li>Once the excess oil is removed, move the embryo into a 5 Î¼l drop of Trizol + 200 Î¼g/Î¼l glycogen on a piece of parafilm.</li><li>Poke the embryo once or twice with a 27-gauge needle. This is best performed under a microscope to visually ensure that the embryo and/or Trizol is not drawn up into the needle.<br/><em>Note: Be very careful here to orient the needle such that the embryo and/or Trizol does not get drawn up into the needle via capillary action (Figure 3, Video 2). Also be careful not to pierce through the parafilm. Any loss of Trizol at this step will reduce the complexity of the sequencing library downstream.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190701/20190701231258_6257.jpg" width="133"/><br/><strong>Figure 3. How to orient the needle when poking the embryo to release the RNA.</strong> Orient the needle at a high angle to prevent capillary action from drawing up Trizol or the embryo into the needle.<br/><br/><!--video3296-2å¼å§--><div class="videodiv" id="vd3296-2" style="height:480px;"></div><div class="videophoto" id="vp3296-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3296/3296-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. Lysing an embryo in Trizol </strong> </div><!--video3296-2ç»æ--><br/></li><li>Incubate the embryo for 2-3 min in the Trizol until the embryo is no longer visible.</li><li>Add an additional 5 Î¼l of the Trizol/glycogen solution to the parafilm and transfer the 10 Î¼l to a tube with an additional 30 Î¼l of Trizol/glycogen solution (for a total of 40 Î¼l). </li><li>Incubate the dissolved embryo in the Trizol/glycogen solution for 10 min at room temperature.</li><li>Store at -80 Â°C until all embryos are collected.<br/><em>Notes:</em> <ol class="npwthree"><li><em>For our experiments, we collected 4 individual embryos for each condition.</em> </li><li><em>To best control for blue-light mediated changes to gene expression, treat untagged His2AvRFP (II) embryos to the same blue-light exposure time course. This will allow for the identification of off-target gene-expression changes and exclude them from further analysis.</em> </li><li><em>We found it helpful to also carry out a âNo Lightâ control where we collected our CRY2-Zld embryos at the same developmental timepoint without any blue-light exposure. This allowed us to identify a small set of genes that were mis-expressed, likely due to the presence of the CRY2 tag.</em> </li><li><em>Because we were precisely staging our embryos, we wanted to ensure that each embryo was exposed to the same amount of blue light. The epifluorescence microscope used illuminates embryos beyond those being directly visualized. Thus, we extracted RNA from only a single embryo from the 5-7 that were mounted. The only exception was the âNo Lightâ control experiment as developmental stage was the only variable measured.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>RNA extraction and purification<br/>Before beginning, wipe down the pipettes and work surface with RNase Zap to remove all RNase from the work area. Only use barrier tips and RNase free reagents and tubes for this section. Also, pre-chill a tabletop centrifuge to 4 Â°C. This procedure is adapted from Lott <em>et al.</em> (2011).<ol class="npwtwo"><li>Thaw all embryo samples.</li><li>Grind with a disposable pellet pestle to ensure that the embryo is completely homogenized, and the RNA is completely released into solution.</li><li>Add 960 Î¼l of Trizol + 200 Î¼g/Î¼l glycogen solution to each sample. Incubate at room temperature for 5 min.<br/><em>Note: Pipette the new Trizol/glycogen solution directly onto the pestle to wash off any residual RNA into the tube.</em> </li><li>Add 200 Î¼l of chloroform. Mix vigorously for 15 s and incubate at room temperature for 5 min.</li><li>Centrifuge the samples at 12,000 <em>x g</em> at 4 Â°C for 15 min.</li><li>While the samples are spinning, prepare a new set of labeled tubes with 500 Î¼l of isopropanol.<br/><em>Note: After centrifugation, the samples will be separated into three layers. The red bottom layer contains the proteins. There will be a small interphase where the DNA resides and a clear top layer containing the RNA.</em> </li><li>Transfer the clear upper layer into the appropriate tube with isopropanol. There should be ~500 Î¼l of liquid in the upper layer. Invert the tubes 8-10 times to mix them thoroughly.</li><li>Move the tubes to a -20 Â°C freezer to precipitate the RNA. Incubate them at -20 Â°C for at least 30 min, but they can be left there for several hours to overnight if required.</li><li>Centrifuge the samples at 12,000 <em>x g</em> at 4 Â°C for 15 min.</li><li>Carefully remove the supernatant. You may not see an RNA pellet, so pipette very slowly to avoid removing any RNA accidentally.</li><li>Add 1 ml of 75% ethanol to each tube and invert the samples to mix.</li><li>Centrifuge the samples at 12,000 <em>x g</em> at 4 Â°C for 5 min.</li><li>Carefully remove the supernatant and wash again with 1 ml of 75% ethanol.</li><li>Centrifuge the samples at 12,000 <em>x g</em> at 4 Â°C for 5 min.</li><li>Carefully remove the supernatant and wash a third time with 1 ml of 75% ethanol.</li><li>Briefly centrifuge the tubes again to collect any residual ethanol and carefully remove it. </li><li>Allow the samples to dry for 5 min.</li><li>Add 50 Î¼l of RNase free water to each tube and refrigerate for 30-60 min to allow for the RNA to resuspend. You can then store these samples at -80 Â°C or proceed directly to library preparation.<br/><em>Notes:</em> <ol class="npwthree"><li><em>The amount of water used to resuspend the samples will vary depending on the library preparation kit being used. The desired input for the TruSeq RNA Sample Preparation v2 kit is 50 Î¼l. </em> </li><li><em>For our samples, we went directly from RNA extraction/purification into the first day of library preparation.</em> </li></ol></li></ol></li></ol><p class="pptt" id="biaoti28382">### Recipes</p><p class="ppzz"><em>Note: Ready made plates and bottles can be purchased if necessary.</em> </p><ol class="npwtwo"><li>Molasses Food<ol class="npwthree"><li>Ingredients:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td>Water<br/></td><td>2.66 L<br/></td></tr><tr><td>Yeast<br/></td><td>33 g<br/></td></tr><tr><td>Yellow cornmeal<br/></td><td>163 g<br/></td></tr><tr><td>Agar<br/></td><td>16 g<br/></td></tr><tr><td>Molasses<br/></td><td>200 ml<br/></td></tr><tr><td>10% p-Hydroxy benzoic acid methyl ester in 95% ethanol<br/></td><td>38 ml<br/></td></tr></tbody></table></li><li>Boil 80% of the water and add the agar and let it dissolve</li><li>In the remaining 20% of the water, add the yeast and let it dissolve and then stir in the cornmeal</li><li>Add the molasses to the agar solution and then stir in the yeast/cornmeal mixture</li><li>Boil the mixture and allow to simmer for 5-10 min</li><li>Turn off heat and allow to cool for at least 5 min</li><li>Stir in the 10% p-hydroxy-benzoic acid methyl ester</li><li>Pour into bottles</li></ol></li><li>Molasses plate<ol class="npwthree"><li>Ingredients:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>Bacteriological grade Agar<br/></td><td>22 g<br/></td></tr><tr><td>Molasses<br/></td><td>90 ml<br/></td></tr><tr><td>Water<br/></td><td>562 ml<br/></td></tr></tbody></table></li><li>Mix and add stir bar, autoclave to sterilize (Liquid cycle; 45 min)</li><li>Let the solution cool to 60 Â°C while stirring on a magnetic stir plate</li><li>Add 5 ml Ethyl Acetate</li><li>Add 5 ml 10% Tegosept (in Ethanol)</li><li>Mix well and pour into plates</li></ol></li></ol><p class="pptt" id="biaoti28383">### Acknowledgments</p><p class="ppzz">We would like to thank members of the Harrison lab for their critical reading and assistance in the creation of this protocol as well as Elle Grevstad from the UW-Madison Biochemistry Optical Core and Seth Blair for the use of his video capture capable dissection microscope.</p><p class="pptt" id="biaoti28384">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti28385">### References</p><ol class="npwtwok"><li>Caussinus, E., Kanca, O. and Affolter, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22157958" target="_blank">Fluorescent fusion protein knockout mediated by anti-GFP nanobody.</a> <em>Nat Struct Mol Biol</em> 19(1): 117-121.</li><li>Foe, V. E. and Alberts, B. M. (1983). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6411748" target="_blank">Studies of nuclear and cytoplasmic behaviour during the five mitotic cycles that precede gastrulation in <em>Drosophila</em> embryogenesis.</a> <em>J Cell Sci</em> 61: 31-70.</li><li>Guglielmi, G., Barry, J. D., Huber, W. and De Renzis, S. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26777292" target="_blank">An optogenetic method to modulate cell contractility during tissue morphogenesis.</a> <em>Dev Cell</em> 35(5): 646-660.</li><li>Hamm, D. C., Larson, E. D., Nevil, M., Marshall, K. E., Bondra, E. R. and Harrison, M. M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29261646" target="_blank">A conserved maternal-specific repressive domain in Zelda revealed by Cas9-mediated mutagenesis in <em>Drosophila melanogaster</em>.</a> <em>PLoS Genet</em> 13(12): e1007120.</li><li>Hamm, D. C. and Harrison, M. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30977698" target="_blank">Regulatory principles governing the maternal-to-zygotic transition: insights from <em>Drosophila melanogaster</em>.</a> <em>Open Biol</em> 8(12): 180183.</li><li>Haruki, H., Nishikawa, J. and Laemmli, U. K. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18922474" target="_blank">The anchor-away technique: rapid, conditional establishment of yeast mutant phenotypes.</a> <em>Mol Cell</em> 31(6): 925-932.</li><li>Huang, A., Amourda, C., Zhang, S., Tolwinski, N. S. and Saunders, T. E. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28691901" target="_blank">Decoding temporal interpretation of the morphogen Bicoid in the early <em>Drosophila</em> embryo.</a> <em>Elife</em> 6: e26258.</li><li>Liu, H., Yu, X., Li, K., Klejnot, J., Yang, H., Lisiero, D. and Lin, C. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18988809" target="_blank">Photoexcited CRY2 interacts with CIB1 to regulate transcription and floral initiation in <em>Arabidopsis</em>.</a> <em>Science</em> 322(5907): 1535-1539.</li><li>Lott, S. E., Villalta, J. E., Schroth, G. P., Luo, S., Tonkin, L. A. and Eisen, M. B. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21346796" target="_blank">Noncanonical compensation of zygotic X transcription in early <em>Drosophila melanogaster</em> development revealed through single-embryo RNA-seq.</a> <em>PLoS Biol</em> 9(2): e1000590.</li><li>McDaniel, S. L., Gibson, T. J., Schulz, K. N., Fernandez Garcia, M., Nevil, M., Jain, S. U., Lewis, P. W., Zaret, K. S. and Harrison, M. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30797686" target="_blank">Continued activity of the pioneer factor Zelda is required to drive zygotic genome activation.</a> <em>Mol Cell</em> 74(1): 185-195 e184.</li><li>Schulz, K. N. and Harrison, M. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30573849" target="_blank">Mechanisms regulating zygotic genome activation.</a> <em>Nat Rev Genet</em> 20(4): 221-234.</li><li>Seller, C. A., Cho, C. Y. and O'Farrell, P. H. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30808658" target="_blank">Rapid embryonic cell cycles defer the establishment of heterochromatin by Eggless/SetDB1 in <em>Drosophila</em>.</a> <em>Genes Dev</em> 33(7-8): 403-417.<br/></li></ol>