<body>
            <h1>Fluorescence Microscopy Assay to Measure HIV-1 Capsid Uncoating Kinetics <em>in vitro</em></h1>
            <p><strong>Authors:</strong> Chantal L. MÃ¡rquez, Derrick Lau, James Walsh, K. M. Rifat Faysal, Michael W. Parker, Stuart G. Turville, Till BÃ¶cking</p>
            <h2>Abstract</h2>
            <p>The stability of the HIV-1 capsid and the spatiotemporal control of its disassembly, a process called uncoating, need to be finely tuned for infection to proceed. Biochemical methods for measuring capsid lattice disassembly in bulk are unable to resolve intermediates in the uncoating reaction. We have developed a single-particle fluorescence microscopy method to follow the real-time uncoating kinetics of authentic HIV capsids <em>in vitro</em>. The assay utilizes immobilized viral particles that are permeabilized with the a pore-former protein, and is designed to (1) detect the first defect of the capsid by the release of a solution phase marker (GFP) and (2) visualize the disassembly of the capsid over time by âpaintingâ the capsid lattice with labeled cyclophilin A (CypA), a protein that binds weakly to the outside of the capsid. This novel assay allows the study of dynamic interactions of molecules with hundreds of individual capsids as well as to determine their effect on viral capsid stability, which provides a powerful tool for dissecting uncoating mechanisms and for the development of capsid-binding drugs.</p>
            <p><strong>Keywords:</strong> Retrovirus, HIV, Capsid stability assay, Uncoating, CypA, Viral particles, CA</p>
            <h2>Background</h2><p>The human immunodeficiency virus-1 (HIV-1) is a lentivirus that replicates in CD4-positive immune cells. HIV-1 copies its genomic RNA into DNA, which integrates into the host genome to establish a persistent infection within the host (Lusic and Siliciano, 2017). In the mature virion, the viral genome is enclosed within the capsid, a conical shell formed by the viral capsid protein (CA) organized into ~250 hexamers and exactly 12 pentamers (Briggs, <em>et al.</em>, 2003 and 2004). After fusion of the viral envelope with the host cell membrane, the capsid is released into the cytoplasm where it fulfills multiple essential functions, making capsid an attractive drug target. It not only protects the viral genome from degradation, but is also involved in influencing many viral post fusion events, including cytoplasmic transport, reverse transcription, nuclear import and integration site targeting (Ocwieja <em>et al.</em>, 2011; Schaller <em>et al.</em>, 2011; Lahaye <em>et al.</em>, 2013; Rasaiyaah <em>et al.</em>, 2013; Matreyek <em>et al.</em>, 2013; Jacques <em>et al.</em>, 2016; Dharan <em>et al.</em>, 2016; Sowd <em>et al.</em>, 2016; Burdick <em>et al.</em>, 2017). The viral capsid must ultimately disassemble in a gradual process termed uncoating to release the viral DNA. To regulate capsid stability and uncoating, HIV-1 capsid binds different sets of capsid-binding host molecules in the cytoplasm, at the nuclear pore complex and inside the nucleus. For example, the peptidylprolyl isomerase cyclophilin A (CypA) binds to multiple CypA bindings loops that are exposed on the outside of the capsid, which may stabilize the capsid during transport from the cell periphery to the nucleus. A roadblock to dissecting the order of events and precise effects of host factors and capsid-binding drugs on uncoating is the unavailability of methods to measure the kinetics of the uncoating reaction at the single-particle level. Most of the current biochemical methods to study HIV capsid stability, such as <em>in vitro</em> uncoating of isolated cores (Kotov <em>et al.</em>, 1999; Forshey <em>et al.</em>, 2002; Shah and Aiken, 2011), the fate of capsid assay (Stremlau <em>et al.</em>, 2006; Yang <em>et al.</em>, 2014), or cyclosporin A (CsA) washout assay (Perez-Caballero <em>et al.</em>, 2005; Hulme <em>et al.</em>, 2011; Hulme and Hope, 2014) rely on observing the average behavior of large numbers of viral cores, which obscures the identification of intermediates in the uncoating process. Fluorescence microscopy methods can resolve the uncoating of individual capsids in the cytoplasm (Campbell <em>et al.</em>, 2008; Pereira <em>et al.</em>, 2011; Ma <em>et al.</em>, 2016; Francis <em>et al.</em>, 2016; Mamede <em>et al.</em>, 2017; Francis and Melikyan, 2018), but complementary <em>in vitro</em> methods that allow higher throughput measurements and detailed kinetic studies under defined conditions are still missing.<br/><br/>We recently developed a novel fluorescence imaging method to follow the real-time uncoating kinetics of individual HIV capsids <em>in vitro</em> and applied this method to reveal how capsid disassembly is modulated by different host factors and small molecules (Mallery <em>et al.</em>, 2018; Marquez <em>et al.</em>, 2018). In our method, HIV-1 virus-like particles loaded with GFP as a content marker are immobilized on a coverslip for observation by total internal reflection (TIRF) microscopy. Upon permeabilization, the pool of GFP retained by the viral membrane is released while the pool of GFP trapped inside the capsid remains. Simultaneously, fluorescence-labeled CypA contained in the bulk solution can access and âpaintâ the capsid by binding to sites on its exterior surface. CypA binding is reversible, whereby the intensity of the CypA paint signal (<em>i.e.</em>, the number of CypA molecules bound at equilibrium) is proportional to the size of the CA lattice. At the concentrations used in this assay (up to 1 Î¼M, <em>i.e.</em>, about 10-fold lower than cytosolic concentrations), CypA binding has no measurable effect on capsid stability. The assay allows the measurement of the two processes involved in capsid uncoating for each particle in the field of view: (1) The sudden loss of the residual GFP signal pinpoints the precise time at which the first defect appears in the capsid, and (2) the gradual decay of the CypA paint signal reveals the kinetics of CA lattice disassembly thereafter. The method has several advantages: it studies authentic HIV capsids <em>in vitro</em>, utilizing small sample volumes in the low Î¼l range; hundreds of individual capsids are monitored in a single experiment and analyzed as single particles, allowing the identification of intermediates in the disassembly pathway that are averaged out in ensemble experiments; it allows screening of the effect of several molecules and proteins on capsid stability as well as the study of their binding kinetics. Our system has potential as a medium-throughput assay for screening compounds that target the stability of the HIV-1 capsid lattice. Furthermore, the approach can readily be applied to study capsid uncoating for other enveloped viruses.</p>
        </body>

<p class="pptt" id="biaoti28388">## Materials and Reagents</p><ol class="npwone"><li>Viral particle preparation<ol class="npwtwo"><li>T25 cell culture flask</li><li>Microwell plate</li><li>10 cm<sup>2</sup> culture dishes (BD Biosciences, catalog number: 353803)</li><li>Poly-L-lysine-coated glass-bottom (175 Î¼m thickness) 96-well plates (Greiner Sensoplate, Sigma, catalog number: M4187)</li><li>HEK-293T cells (ATCC, catalog number: CRL-3216)</li><li>Dulbeccoâs Modified Eagle Medium (DMEM) (Life Technologies, Invitrogen, catalog number: 11965-092)</li><li>Fetal bovine serum (FBS) (Sigma-Aldrich, catalog number: F2442-500ML)</li><li>1x PBS (Gibco, catalog number: 10010031)</li><li>1x trypsin-EDTA (Gibco, catalog number: 15400054)</li><li>Plasmid, psPAX2 (NIH AIDS Reagent Program, catalog number: 11348)</li><li>Plasmid, pNL4.3-iGFP-âEnv (Hubner <em>et al.</em>, 2007; Aggarwal <em>et al.</em>, 2012)</li><li>Polyethylenimine (PEI Max) reagent (Polysciences, catalog number: 9002-98-6)</li><li>0.9% w/v sodium chloride (Sigma-Aldrich, catalog number: S8776)</li><li>EZ-Link Sulfo-NHS-LC-LC-Biotin (Thermo Scientific, catalog number: 21338)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375-250G)</li><li>NaCl (Chem Supply, catalog number: SA046-5KG)</li><li>Cell culture media (see Recipes)</li><li>HBS pH 7.5 and 7.0 (see Recipes)<br/><br/></li></ol></li><li>AF568-CypA preparation<ol class="npwtwo"><li>0.22 Î¼m syringe filter</li><li>Amicon-15 Ultra centrifugal filtration device (10k MWCO, Merck, UFC901024)</li><li>BL21 (DE3) <em>E. coli</em> cells</li><li>Plasmid, pETMCSI-hCypA (pBH964) (Ozawa <em>et al.</em>, 2004)</li><li>Pierce Coomassie Plus (Bradford) Assay Kit (Thermo Scientific, catalog number: 23236)</li><li>Alexa-Fluor 568-C5-maleimide dye (Thermo Fisher Scientific, catalog number: A20341)</li><li>AF568-CypA (see Procedure)</li><li>Complete EDTA-free EASYpack, Protease Inhibitor Cocktail tablets (Roche, catalog number: 04693132001)</li><li>LB medium (MP Bio, catalog number: 113002041)</li><li>Isopropyl Î²-D-thiogalactopyranoside (IPTG)</li><li>Liquid nitrogen</li><li>Ampicillin powder (Thermo Scientific, catalog number: BP1760-25)</li><li>1% v/v acetic acid diluted from glacial stock (Ajax Finechem, catalog number: AJA1-2.5LPL)</li><li>BSA (Bovogen Biologicals, catalog number: BSAS-NZ 0.1)</li><li>DTT (Sigma-Aldrich, catalog number: 43815-25G)</li><li>TCEP (Sigma-Aldrich, catalog number: C4706-2G)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375-250G)</li><li>NaN<sub>3</sub> (Sigma-Aldrich, catalog number: S2002-500G)</li><li>NaCl (Chem Supply, catalog number: SA046-5KG)</li><li>Lysozyme (Sigma-Aldrich, catalog number: 62971-10G-F)</li><li>MOPS (Sigma-Aldrich, catalog number: M1254-1KG)</li><li>Tris (Ajax Finechem, catalog number: AJA2311-5KG)</li><li>Glycerol (Ajax Finechem, catalog number: AJA242-2.5LGL)</li><li>Lysis buffer (see Recipes)</li><li>AIEX buffer A (see Recipes)</li><li>AIEX buffer B (see Recipes)</li><li>CIEX buffer A (see Recipes)</li><li>CIEX buffer B (see Recipes)</li><li>CypA storage buffer (see Recipes)</li><li>AF568-CypA storage buffer (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Microfluidic device preparation<ol class="npwtwo"><li>Plastic weigh-boat (Sigma, or equivalent)</li><li>Tubing Intramedic 0.043 in. (Becton Dickinson, catalog number: 427406)</li><li>Disposable needles, 1 mm OD (Becton Dickinson, or equivalent)</li><li>Blu tack (Bostik, or equivalent)</li><li>Round glass coverslips, 25 mm diameter (Paul Marienfeld GmbH &amp; Co KG, catalog number: 0117650)</li><li>Biotinylated poly-L-lysine-g-poly-ethylene glycol (SuSoS AG, PLL-g-PEG-biotin, PLL(20)-g[3.5]-PEG(2)/PEG(3.4)-biotin(20%))</li><li>Streptavidin (Life Technologies Australia, catalog number: 434301)</li><li>Sylgard184 silicone elastomer kit (Dow Corning) </li><li>Absolute ethanol (Ajax Finechem, catalog number: AJA214-2.5LGL)</li><li>1 M NaOH (Ajax Finechem, catalog number: AJA482-500G)</li><li>Isopropyl alcohol (Thermo Scientific, catalog number: AJA214-2.5LGL)</li><li>Tris (Thermo Scientific, catalog number: AJA2311-5KG)</li><li>EDTA (Ajax Finechem, catalog number: AJA180-500G)</li><li>NaN<sub>3</sub> (Sigma-Aldrich, catalog number: S2002-500G)</li><li>Tween 20 (Sigma-Aldrich, catalog number: P5927-500ML)</li><li>BSA (Bovogen Biologicals, catalog number: BSAS-NZ 0.1)</li><li>Blocking buffer (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Imaging assay<ol class="npwtwo"><li>Bacterial pore-forming protein for viral membrane permeabilization<br/><strong>Option 1:</strong> Recombinant cysteine-less (C459A) perfringolysin O (PFO) with N-terminal His-tag was supplied by Michael Parker.<br/>The protocols for expression and purification of recombinant PFO C459A have been described previously (Shepard <em>et al.</em>, 1998; Wade <em>et al.</em>, 2015). Production and handling of PFO should be done in accordance with local occupational health and safety regulations for handling toxins. Store frozen in small aliquots at -80 Â°C. After thawing an aliquot, store at room temperature for up to two months. The protein loses activity when stored at 4 Â°C or during repeated freeze-thaw cycles.<br/><strong>Option 2:</strong> Alternatively, streptolysin O (SLO) (Sigma-Aldrich, catalog number: S5265) can be used for permeabilization of viral particles (unpublished data). <br/>Handling of SLO should be done in accordance with local occupational health and safety regulations for handling toxins. Dissolve the protein in PBS containing 2 mM TCEP to give a concentration of ~8 Î¼M, prepare small aliquots, freeze in liquid nitrogen and store frozen at -80 Â°C. Thaw a fresh aliquot on the day of use. Unlike PFO C459A, wild type SLO loses activity in solution as a result of oxidation (disulfide formation).</li><li>Protocatechuic acid (Sigma-Aldrich, catalog number: 37580-25G-F)</li><li>Protocatechuate-3,4-dioxygenase (Sigma-Aldrich, catalog number: P8279-25UN)</li><li>Trolox (Sigma-Aldrich, catalog number: 238813-1G)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>General<ol class="npwtwo"><li>Sharp blades (TechnoCut, or equivalent)</li><li>Serological pipettes, 5,10 and 25 ml (Corning, or equivalent)</li><li>Microcentrifuge tubes (Eppendorf, or equivalent)</li><li>Sterile 15- and 50-ml conical tubes (Falcon, or equivalent)</li><li>Sterile pipette tips (Eppendorf, or equivalent)</li><li>Disposable syringes, 1 and 3 ml (Becton Dickinson, or equivalent)</li><li>Milli-Q Water (from Millipore Milli-Q Integral 5 Water Purification System, or equivalent)</li></ol></li></ol><p class="pptt" id="biaoti28389">### Equipment</p><ol class="npwone"><li>Viral particle preparation<ol class="npwtwo"><li>10-ml super loop</li><li>Tissue culture incubator, humidity, temperature and CO<sub>2</sub> regulated (Thermo Fisher Scientific, model: 3110, or equivalent)</li><li>Biosafety cabinet (Thermo Fisher Scientific, model: 1323TS, or equivalent)</li><li>HiPrep 16/60 Sephacryl S-500 HR column (GE Healthcare, catalog number: 28-9356-06)</li><li>Fast protein liquid chromatography (FPLC) system including injector, one pump, UV-detector, fraction collector (GE Healthcare, <a class="blackNoUnder" target="_blank"> ÃKTA pure </a> , or equivalent)</li><li>4 Â°C refrigerator</li><li>-20 Â°C freezer</li><li>-80 Â°C freezer<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>AF568-CypA preparation<ol class="npwtwo"><li>5 ml HiTrap Q HP column (GE Healthcare Life Science, catalog number: 17115401)</li><li>5 ml HiTrap SP HP column (GE Healthcare Life Science, catalog number: 17115201)</li><li>Zeba desalting spin column (Thermo Fisher Scientific, catalog number: 89883)</li><li>CelluSep T1 regenerated tubular membrane MWCO 3,500 Da (CelluSep, catalog number: 501546)</li><li>Branson Sonifier 250 Probe Sonicator mounted with a tapered microtip<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Microfluidic set-up and capsid uncoating imaging<ol class="npwtwo"><li>TIRF Microscope, see specifications in Part IV</li><li>Magnetic chamber for 25 mm round coverslips (Chamlide chamber, model: CM-B25-1)</li><li>Syringe pump (New Era Pump System Inc., model: NE-1002X)</li><li>Mold (<em>e.g.</em>, prepared by photolithography on a silicon wafer or machined from aluminum) for making microfluidic devices; each device contains 5 flow cell channels; channel dimension 1,100 Î¼m x 800 Î¼m x 40 Î¼m (length x width x height)</li><li>Plasma cleaner (Harrick Plasma, catalog number: PDC-32G)</li><li>Ceramic coverslip holder</li><li>Vacuum desiccator</li><li>Harris Uni-Core 1.0 mm punch</li><li>Oven</li><li>Andor iXon 888 EMCCD cameras<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>General<ol class="npwtwo"><li>Pipettes (Gilson, catalog number: F167700, or equivalent)</li><li>Refrigerated tabletop centrifuge (Thermo Fisher Scientific, model: 75004524, or equivalent)</li><li>Tabletop centrifuge (Eppendorf Centrifuge, model: 5417R, or equivalent)</li><li>NanoDrop 1000 spectrophotometer or equivalent machine that can read UV-vis spectra</li><li>Rotator (Benchmark Scientific, model: R5010, or equivalent)</li></ol></li></ol><p class="pptt" id="biaoti28390">### Software</p><p class="ppzz"><em>Note: The following software package can be used individually or in combination for image analysis.</em> </p><ol class="npwtwo"><li>FIJI image analysis software (Schindelin <em>et al.</em>, 2012; <a href="https://imagej.net/Fiji/Downloads" target="_blank">https://imagej.net/Fiji/Downloads</a>)</li><li>Matlab (Mathworks, <a href="https://www.mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a>)</li><li>JIM Immobilized Microscopy analysis package (<a href="https://github.com/lilbutsa/JIM-Immobilized-Microscopy-Suite" target="_blank">https://github.com/lilbutsa/JIM-Immobilized-Microscopy-Suite</a>)</li></ol><p class="pptt" id="biaoti28391">### Procedure</p><p class="ppzz"><strong>Part I: HIV-1 viral particle preparation<br/><br/></strong> </p><ol class="npwone"><li>Produce HIV-1 viral particles (2.5 days)<br/><em>Note: The combination of plasmids used in this protocol results in the production of virus-like particles that lack envelope proteins and are non-infectious; procedures should be done in accordance with local biosafety regulations for producing virus-like particles.</em> <ol class="npwtwo"><li>In a 15 ml conical tube, prepare DNA solution for transfection by mixing 6.6 Î¼g of pNL4.3-iGFP-âEnv plasmid, 3.3 Î¼g of psPAX2 plasmid, 60 Î¼l of 1 mg/ml PEI max solution in a final volume of 500 Î¼l of 0.9% (w/v) sodium chloride. Incubate the mixture for 30 min at room temperature to allow formation of DNA:PEI complexes.</li><li>Split the HEK-293T cell culture as follows:<ol class="npwthree"><li>Remove the culture medium and wash the cell monolayer with 1x PBS. Add 1x trypsin-EDTA (typically 1 ml for a T25 cell culture flask) and incubate for 5 min at 37 Â°C. </li><li>Stop the trypsin digestion by adding new culture media and transfer the cell suspension to a 15 ml conical tube. </li><li>Take a sample to count cells and determine the total number of cells in the tube. Centrifuge the cell suspension at 300 <em>x g</em> for 5 min at room temperature to pellet the cells and discard the supernatant. Resuspend the cells in the appropriate volume of fresh culture media to obtain a concentration of 7 x 10<sup>6</sup> cells/ml.</li></ol></li><li>Gently add 1 ml of cell suspension to the DNA:PEI mix and incubate for 5 min at room temperature.</li><li>Plate the cells:DNA:PEI mixture drop by drop in a 10 cm<sup>2</sup> culture dish containing 6.5 ml of culture media. Slightly shake the dish to distribute the cells homogeneously. Incubate at 37 Â°C with 5% CO<sub>2</sub> for 48 h.</li><li>Collect the virus-containing supernatant in a 15 ml conical tube and centrifuge at 2,100 <em>x g</em> for 10 min at 4 Â°C to remove cellular debris.</li><li>Collect the supernatant and transfer it to a new conical tube. The final volume of the cleared virus-containing medium should be around 7 ml. </li></ol><em>Note: To verify the presence of fluorescent HIV particles in the supernatant, dilute 1 in 100 in PBS and then spinoculate 200 Î¼l of the virus preparation onto a poly-L-lysine-coated glass-bottom 96-well plate well at 1,200 </em><em>x g</em><em> for 60 min at 4 Â°C, and inspect for fluorescent dots with a 60x objective in a fluorescence microscope. The number of viral particles per surface area can be determined by counting the number of fluorescent dots in at least 4 fields of view. The particle concentration can then be obtained by multiplying the number of particles per surface area with the surface area of the well (to obtain the number of particles per well) and then dividing this number by the volume of supernatant added to the well (0.2 ml).</em><br/><br/></li><li>Biotinylate the HIV-1 viral particles (2 h)<ol class="npwtwo"><li>Dissolve ~1 mg of EZ-Link<sup>TM</sup> Sulfo-NHS-LC-LC-Biotin in 1 ml of HBS pH 7.5 in a microcentrifuge tube.</li><li>Add the biotinylation reagent to the virus-containing medium and incubate in the dark for 90 min at 4 Â°C with gentle rotation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Purify the biotinylated HIV-1 viral particles by size exclusion chromatography (1 day)<ol class="npwtwo"><li>Connect a HiPrep 16/60 Sephacryl S-500 HR size exclusion chromatography column to the FPLC system. All solutions used for FPLC should be passed through a filtration membrane with 0.2 Î¼m pore size for removal of particulates and to degas the solutions.</li><li>Replace the storage solution (usually 20% ethanol) with 2 column volumes (CV) of purified water and then equilibrate the column with 2 column volumes (CV) of HBS pH 7.5. Monitor the absorption at 280 nm. Flow rate: 1 ml/min. <br/><em>Note: This step should be started the day before collecting the viral particles as it takes ~8 h for the column to be equilibrated.</em> </li><li>Load the biotinylated viral particles (~6.5 ml) onto the column using a 10-ml super loop. Monitor the absorption at 280 nm. Flow rate: 0.5 ml/min.</li><li>Elute the sample with HBS pH 7.5 and collect 1 ml fractions until a total of 1.5 CV is reached (~6 h). A representative elution profile is shown in Figure 1.</li><li>Combine the fractions corresponding to the first small peak that contain the viral particles (typically around fractions 34-39 or C10-D3 on a microwell plate).</li><li>Purified biotinylated viral particles can be used within 7 days if they are stored at 4 Â°C, or you can make 200-500 Î¼l aliquots and store at -80 Â°C (no flash freezing in liquid nitrogen needed). Frozen samples can be thawed on ice for use.</li><li>Wash the column with 2 CV of purified water and then with storage solution.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020618_4689.jpg" width="136"/><br/><strong>Figure 1. Representative elution profile of biotinylated HIV-1 viral particles by size exclusion chromatography.</strong> Chromatographic separation of HIV-1 viral particles from culture media proteins. The first small peak (fractions C10-D3, marked with a red asterisk) corresponds to the HIV-1 viral particles.<br/><div style="white-space:nowrap;"><br/></div></li></ol><div style="white-space:nowrap;"><span style='color:#333333;font-family:"'></span> </div></li></ol><strong>Part II: Preparation AF568-CypA for lattice painting</strong><br/><br/><ol class="npwone"><li>Express and purify human CypA (3-4 days)<ol class="npwtwo"><li>Grow BL21 (DE3) <em>E. coli</em> cells expressing human CypA in a pET expression vector in LB medium supplemented with ampicillin (final 100 Î¼g/ml) at 37 Â°C, 180 rpm, overnight as a starter culture.</li><li>Inoculate 500 ml of fresh LB medium supplemented with ampicillin (final 100 Î¼g/ml) with 5 ml of starter culture.</li><li>Monitor the absorbance at 600 nm (OD<sub>600</sub>). Induce protein expression by addition of isopropyl Î²-D-thiogalactopyranoside (IPTG) to a final concentration of 1 mM when the OD<sub>600</sub> reaches a value of 0.6. Allow protein expression to proceed for 3 h at 37 Â°C, 180 rpm.</li><li>Pellet the bacterial cells by centrifugation (6,000 <em>x g</em>, 20 min at 4 Â°C) and discard the supernatant.</li><li>Resuspend the cell pellet in a cold lysis buffer (typically use 15 ml of buffer per 500 ml of culture pellet) and lyse cells by sonication on ice (50% duty cycle, setting 3, 15 s on/off per cycle, 12 cycles).<br/><em>Note: Cells can be lysed with any other sonicator or method for lysis of bacterial cells.</em> </li><li>Centrifuge the lysate at 43,000 <em>x g</em> for 30 min at 4 Â°C, recover the supernatant and filter it through a syringe filter (0.22 Î¼m).</li><li>Purify the sample by subtractive anion exchange chromatography using 2 x 5 ml HiTrap Q HP columns connected in tandem. First, wash the columns with 5 CVs of AIEX buffer A, followed by 5 CVs of AIEX buffer B and finally equilibrate with 5 CVs of AIEX buffer A at 5 ml/min. Load the sample and collect the flowthrough in 1 ml fractions. Keep eluting with extra ~5 ml of AIEX buffer A to recover the entire flowthrough. Flow rate: 1 ml/min.</li><li>Check each flowthrough fraction on reducing SDS-PAGE and combine the fractions that contain CypA. Adjust pH to 5.8 with 1% v/v acetic acid using a pH probe to measure the pH. A representative elution profile is shown in Figures 2A and 2C.</li><li>Remove aggregates by centrifugation (43,000 <em>x g</em>, 1 h, 4 Â°C) and collect the supernatant.</li><li>Purify CypA contained in the supernatant further using cation exchange chromatography using a 5 ml HiTrap SP HP column washed with 5 CVs of CIEX buffer A, followed by 5 CVs of CIEX buffer B, and finally equilibrated with 5 CVs of CIEX buffer A at 5 ml/min. Inject the sample onto the equilibrated column at 1 ml/min, then wash with 2 CVs of CIEX buffer A. Elute bound CypA with a 0-1 M linear gradient of NaCl (0%-100% CIEX buffer B) over 20 CVs at 5 ml/min. The CypA peak will elute at 20-25% CIEX buffer B. A representative elution profile is shown in Figures 2B and 2D.</li><li>Dialyze the fractions containing purified CypA against CypA storage buffer for 3 x 4 h using dialysis tubing (<em>e.g.</em>, CelluSep T1 regenerated membrane) with a molecular weight cut-off (MWCO) of 3,500 Da.</li><li>Determine the concentration of protein using the Bradford assay with BSA as a standard.</li><li>If necessary, concentrate the protein using an Amicon-15 Ultra centrifugal filtration device (10k MWCO). The final recommended concentration should be 100-300 Î¼M (1.8-5.4 mg/ml).</li><li>Use immediately or aliquot and flash freeze in liquid nitrogen for storage at -80 Â°C.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020654_5559.jpg" width="148"/><br/><strong>Figure 2. Purification of CypA by ion exchange chromatography.</strong> Elution profiles and corresponding SDS-PAGE analysis with Coomassie Blue staining of selected fractions from anion (A, C) and cation (B, D) exchange chromatography steps for CypA purification.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Fluorescence labeling of CypA (4 h)<ol class="npwtwo"><li>Exchange the buffer of purified recombinant CypA using a Zeba desalting spin column equilibrated with PBS containing 0.1 mM TCEP to remove the DTT contained in the CypA storage buffer.</li><li>Mix purified CypA with a 4-fold molar excess of Alexa Fluor 568-C5-maleimide dye in PBS pH 7.4 and incubate for 10 min at room temperature. A typical reaction for labeling of 130 Î¼l of 100 Î¼M recombinant CypA requires 5 Î¼l of 10 mM Alexa Fluor 568-C5-maleimide.</li><li>Add DTT solution (1 M) to a final concentration of 10 mM to quench unconjugated dye.</li><li>Remove the free dye by exchanging the buffer using a Zeba desalting spin column equilibrated with AF568-CypA storage buffer.</li><li>Measure a UV-visible absorption spectrum of AF568-CypA using a NanoDrop 1000 or equivalent instrument to determine the absorption at 280 nm (<em>A</em><sub>280</sub>) and the absorption at wavelength of maximum dye absorbance (<em>A</em><sub>dye</sub>, ~578 nm for Alexa Fluor 568). Calculate the protein concentration (<em>c</em><sub>CypA</sub>) using the following formula: <em>c</em><sub>CypA</sub> = (<em>A</em><sub>280</sub> - CF x <em>A</em><sub>dye</sub>)/<em>Îµ</em><sub>CypA</sub>, where CF is a dye-specific correction factor (CF = 0.45 for Alexa Fluor 568) and <em>Îµ</em><sub>CypA</sub> is the molar extinction coefficient of CypA at 280 nm (<em>Îµ</em><sub>CypA</sub> = 8730 M<sup>-1</sup>cm<sup>-1</sup>). Calculate the degree of labeling (DOL) using the following formula: DOL = (<em>A</em><sub>dye</sub>/<em>Îµ</em><sub>dye</sub>)/<em>c</em><sub>CypA</sub>, where Îµdye is the molar extinction coefficient of the dye at its absorbance maximum (<em>Îµ</em><sub>dye</sub> = 91300 M<sup>-1</sup>cm<sup>-1</sup> for Alexa Fluor 568). AF568-CypA prepared by this protocol should have a degree of labeling close to 1 fluorophore per protein.</li><li>Prepare small aliquots, flash freeze in liquid nitrogen and store at -40 Â°C. Typically CypA-AF568 are stored in 2-4 Î¼l at ~100 Î¼M.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol><strong>Part III: Microfluidic flow cells set-up</strong><br/><br/><ol class="npwone"><li>Prepare polydimethylsiloxane (PDMS) microfluidic devices (1.5 days)<ol class="npwtwo"><li>Prepare 30 ml of the transparent polymer mixture (elastomer/curing agent ratio 10:1 w/v) in a plastic weigh-boat.</li><li>Mix thoroughly (avoid formation of bubbles) and then pour over the silicon microlithography wafer mold (Figure 3A).</li><li>Place the silicon mold containing the PDMS mix in a vacuum desiccator and degas under reduced pressure for 20 min to remove all air bubbles before curing.</li><li>Cure the PDMS overnight at 70 Â°C in an oven. The cured PDMS should be firm to the touch.</li><li>Carefully remove the PDMS devices from the mold by using a sharp blade (scalpel). Make sure you cut them into square pieces with dimensions that fit into the magnetic chamber used for imaging (see below).</li><li>Place the PDMS block onto a clean surface with the channels facing up and punch holes for inlets and outlets at opposite ends of each channel by pushing a Harris Uni-Core 1.0 mm punch through the PDMS device. Remove polymer plugs from the holes.</li><li>Clean the PDMS devices by immersion in water, isopropyl alcohol (100%) and water (10 min each step).</li><li>Dry PDMS devices with lint-free tissue and store in a clean container with the channels facing up.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Clean glass coverslips (3 h)<ol class="npwtwo"><li>Place round glass coverslips in a coverslip holder and wash them by sonication in absolute ethanol for 30 min followed by sonication in 1 M NaOH for 30 min.</li><li>Rinse the coverslips by softly shaking the cover glass holder in a large beaker filled with ultrapure water. Repeat three times with fresh water each time. Handle carefully as the coverslips can stick together, fall out of the holder or break.</li><li>Wick away the excess of water from the coverslips in the holder with a lint-free tissue without touching the coverslip surfaces. Then place the holder with coverslips into a heated oven for at least 30 min for drying.</li><li>Use cleaned coverslips immediately or store them in a vacuum desiccator for use within a month.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Assemble the microfluidic flow cells (1-2 h)<br/>Depending on the number of experiments you are planning to do, assemble one or more PDMS devices. We recommend always preparing one extra device in case the glass coverslip breaks, the channels leak or something else happens.<ol class="npwtwo"><li>Clean the channels of a PDMS device with isopropanol using a lint-free tissue. Let the surface dry.</li><li>Place a coverslip (in a ceramic holder) and a PDMS inside a plasma cleaner and treat them with an air plasma for 3 min. The channels should be facing up.</li><li>Carefully mount the PDMS device on the center of the coverslip with the channels facing the glass. Apply gentle, uniform pressure with your hand to ensure a uniform seal but without breaking the glass slide. To keep the coverslip clean, always place it on top of a lint-free tissue. The assembled device is shown in Figure 3B.</li><li>Place the assembled microfluidic device in an oven at 70 Â°C for 15 min to improve bonding between the glass and the PDMS.</li><li>Use immediately or store the devices in a clean container for use within 2 weeks.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020736_6539.jpg" width="236"/><br/><strong>Figure 3. Microfluidic devices.</strong> A. Photo of the silicon microlithography wafer mold. B. Microfluidic device consisting of a PDMS block (with channels and holes for tubing) adhered to a glass coverslip.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Prepare inlet and outlet tubing (0.5 h)<ol class="npwtwo"><li>For inlets, cut ~5-7 cm long pieces of tubing. Do not reuse inlet tubing.</li><li>For outlets, cut ~10-13 cm long pieces of tubing. Connect a blunted needle (1 mm OD) to one of the ends. To reuse them, rinse the tubing with distilled water using a syringe. Dry by drawing air through the tubing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol><strong>Part IV: Imaging assay</strong><br/><br/><ol class="npwone"><li>Microscope set-up<br/>Images are acquired on a TIRF microscope. Our home-built system (designed by Philip Nicovich) is based on an ASI-RAMM frame (Applied Scientific Instrumentation) with a Nikon 100x CFI Apochromat TIRF (1.49 NA) oil immersion objective. Lasers were incorporated using the NicoLase system (Nicovich <em>et al.</em>, 2017). Images were captured on two Andor iXon 888 EMCCD cameras (Andor Technology Ltd) and 300 mm tube lenses were used to give a field of view of 88.68 Î¼m x 88.68 Î¼m at Nyquist sampling frequency (86 nm per pixel). The TIRF angle was adjusted to obtain ~200 nm penetration depth.<br/><em>Note: The relationship between TIRF angle and penetration depth and methods to empirically determine penetration depth are discussed in the following reference (Fish, 2009).</em><br/><br/></li><li>Capture viral particles onto the flow cell surface<ol class="npwtwo"><li>With a P20 micropipette, gently inject a solution of biotinylated poly-L-lysine-g-poly-ethylene glycol (1 mg/ml) into one of the ports of each flow channel until the solution reaches the other port (~3 Î¼l). Incubate for 30 min at room temperature.</li><li>Mount an assembled microfluidic device in a magnetic chamber and connect inlet and outlet tubing pieces to the opposite ports of each flow channel.</li><li>Place the inlet tubing of all channels into a microcentrifuge tube filled with ultrapure water (Figure 4A). Use removable adhesive to position the tube close to the device.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020806_2596.jpg" width="252"/><br/><strong>Figure 4. Setup for flow cell surface preparation.</strong> Picture of an assembled microfluidic device mounted on the magnetic holder in a (A) Petri dish or (B) on the microscope stage. Inlets (short tubing) are dipped into a microcentrifuge tube with HBS pH 7.5 while outlets (long tubing) are connected to a syringe.<br/><br/></li><li>Attach a 1 ml syringe to an outlet tubing and carefully rinse the channel by drawing distilled water through the tubing (~200 Î¼l each). Repeat for all channels. You can use the same syringe for all outlets (switching from one to the next) or connect a separate syringe to each outlet for ease of use.</li><li>Remove the inlets from the water and remove the water from the channels by drawing air through them.</li><li>Draw a solution of 0.2 mg/ml streptavidin in blocking buffer into the channel using syringe connected to the outlet tubing (as in Step B4) and incubate for 15 min. A volume of 100 Î¼l of solution is sufficient to fill all five channels. From this step onwards it is very important to avoid letting air into the channels as exposure to air could degrade the modified surface, so always keep the inlets in a solution.</li><li>Rinse the channel with HBS pH 7.5 to remove the streptavidin solution (~200 Î¼l per channel). Use modified flow channels on the day of preparation taking care to keep the channels filled with solution. </li><li>Place the microfluidic device onto the microscope stage and connect the outlet tubing of the channel to be imaged to a syringe pump. Operate the syringe pump in "withdraw" mode to pull solutions through the channels.<br/><em>Note: The flow rates can vary from the values suggested below without damaging the modified surface (typically in the range between 5 and 100 Î¼l/min), whereby slow flow rates allow for more time for particle to diffuse and bind to the surface (</em><em>e.g.</em><em>, in Step B9) while faster flow rates result in more efficient removal of non-specifically bound particles (</em><em>e.g.</em><em>, in Step B11) and faster solution exchange rates.</em> </li><li>Inject purified biotinylated viral particles in HBS pH 7.5 through the channel at a flow rate of 25 Î¼l/min until a density of ~1,000 viral particles per field of view is obtained (typically 30-100 Î¼l).</li><li>Place the objective at the center of the channel and focus on the surface. You should see the fluorescent viral particles bound to the surface as bright diffraction-limited spots.</li><li>Wash-out the unbound viral particles with 50-100 Î¼l of HBS pH 7.5 at a flow rate of 65 Î¼l/min.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Perform capsid uncoating assay via painting with CypA<ol class="npwtwo"><li>Prepare 40 Î¼l of CypA paint solution: HBS pH 7 containing PFO (200 nM), 0.5-1 Î¼M of AF568-CypA and an oxygen quenching system (2 mM trolox, 2.5 mM protocatechuic acid (PCA) and 0.25 U/ml protocatechuate-3,4-dioxygenase (PCD)) to reduce photobleaching during the dual color experiments. The pH of the imaging buffer is adjusted to pH 7 because PFO is considerably less efficient at higher pH (<em>e.g.</em>, pH 7.5).</li><li>Initiate the assay by injecting 30 Î¼l of CypA paint solution at a flow rate of 65 Î¼l/min. </li><li>At the same time start acquiring TIRF images sequentially with 488 nm and 561 nm laser excitation. Example images are shown in Figure 5 and a typical image stack is shown in Videos 1 and 2.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Laser power and exposure time should be chosen to obtain a signal-to-noise ratio that is sufficient to detect the level of GFP trapped inside the capsid (see below) while minimizing the energy that the sample is exposed to. High laser powers and long exposure times can lead to photochemically induced damage that affect the uncoating kinetics. On our system we typically use a power density of ~1 W cm<sup>-2</sup> (measured at the objective with the laser beam normal to the surface of the coverslip) and an exposure time of 20 ms. These settings avoid saturating the CCD camera in the GFP channel and result in a single molecule photobleaching rate of 0.02 s</em><sup><em>-1</em></sup><em> (half-life of 35 s) in the AF568-CypA channel. </em> </li><li><em>The frame rate and total imaging time depend on the experimental question; we typically record capsid opening traces by acquiring 800 frames, </em><em>e.g.</em><em>, at a frame rate of 1 frame/s (total imaging time of 13.3 min) for short experiments or at a frame rate of 1 frame/6 s (total imaging time of 80 min).</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020843_4108.jpg" width="577"/><br/><strong>Figure 5. Capsid uncoating assay via painting with CypA.</strong> Snapshot images from a time series of viral particles recorded in the GFP channel (content marker, top) and the AF568 channel (CypA paint, bottom) at 1 frame per second. Snapshots correspond to indicated time points from the experiment shown in Video 2. Scale bars = 1 Î¼m.<br/><br/><!--video3297-1å¼å§--><div class="videodiv" id="vd3297-1" style="height:480px;"></div><div class="videophoto" id="vp3297-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3297/3297-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Capsid uncoating assay via painting with CypA.</strong> Image stack of the GFP channel (content marker), AF568 channel (CypA paint), and merge of both videos of a typical capsid uncoating assay via painting with CypA (complete field of view, 1,024 x 1,024 pixels). Images were acquired at 1 frame per second.</div><br/><!--video3297-2å¼å§--><div class="videodiv" id="vd3297-2" style="height:480px;"></div><div class="videophoto" id="vp3297-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3297/3297-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. Capsid uncoating assay via painting with CypA.</strong> Image stack of the GFP channel (content marker), AF568 channel (CypA paint), and merge of both videos of a typical capsid uncoating assay via painting with CypA (80 x 60 pixel region of the field of view). Images were acquired at 1 frame per second.</div><!--video3297-2ç»æ--><br/></li><li>Repeat this procedure in the remaining flow channels of the microfluidic device.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Single molecule photobleaching of AF568-CypA<ol class="npwtwo"><li>Mount a clean coverslip in a magnetic chamber and add ~300 Î¼l of 0.02 mg/ml BSA to passivate it. Incubate for 2 min and remove.</li><li>Wash with HBS 7 and add a solution of 10 nM of AF568-CypA in HBS pH 7. Incubate for 3-5 min.<br/></li><li>Remove unbound molecules by washing with HBS 7 and mount the chamber on the microscope stage for imaging.</li><li>Acquire photobleaching movies in different fields of view with a 561 nm laser with the same laser power used during the uncoating assay and exposure times of 200 ms until most of the single molecules are photobleached (~150 frames).</li></ol></li></ol><p class="pptt" id="biaoti28392">### Data analysis</p><p class="ppzz">Analysis is performed in three stages:</p><ol class="npwtwo"><li>Creation of single particle traces using the Generate_Multi_Channel_Traces program in the JIM microscopy suite:<ol class="npwthree"><li>The two channels are aligned to each other and drift correction is calculated.</li><li>A subaverage of the start of the experiment is made to detect all particles.</li><li>Particles are detected using a threshold. Clusters of viral particles are excluded by setting the maximum eccentricity to 0.4. Background noise is excluded by setting the minimum number of pixels of a detected particle to 10.</li><li>The area around each particle is expanded by 4 pixels to ensure that all fluorescence is captured. A region with a radius of 20 pixels further surrounding the particle is used to calculate the background fluorescence.</li><li>The fluorescence intensity in each frame is calculated by taking the total fluorescence in each particle ROI minus the background fluorescence intensity.</li></ol></li><li>Trace analysis<ol class="npwthree"><li>Identify the time and the height of GFP release steps (decrease in signal intensity) in each GFP trace (BÃ¶cking <em>et al.</em>, 2011). This can be automated using a step fitting algorithm (<em>e.g.</em>, in MATLAB).</li><li>Sort the traces into different categories depending on the number of steps the signal has, according to the following classification criteria: (1) loss of entire GFP signal in one step (âleakyâ particles, Figure 6A); (2) loss of GFP intensity in one large (permeabilization) and one small (capsid opening) step (âopeningâ particles, Figure 6B); (3) loss of the majority of the GFP signal in one step with residual GFP signal persisting for the rest of the experiment (âclosedâ particles, Figure 6C); (4) no permeabilization; (5) uninterpretable traces. Sorting can be automated by defining thresholds for the different intensity levels. The assignment to classes should be verified by visual inspection of traces.</li><li>To plot lattice disassembly of leaky capsids, align the individual binding traces from class 1 at the time of membrane permeabilization. To plot lattice disassembly of capsids that undergo uncoating during the experiment, align traces from class 3 at the time of core opening.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020924_4372.jpg" width="503"/><br/><strong>Figure 6. Capsid uncoating assay via painting with CypA traces.</strong> Example traces of CypA binding (vermillion) to individual leaky (A), opening (B) and closed (C) capsids that retain or do not retain GFP (green) after permeabilization of the viral envelope.<br/><br/></li></ol></li><li>Single molecule photobleaching intensities are quantified using JIM.<ol class="npwthree"><li>Photobleaching traces are generated using the Generate_Single_Channel_Traces program.</li><li>Traces are stepfit to identify particles with single steps (representative of single molecules) using the Single_Molecule_Photobleaching method.</li><li>The step-heights of single step traces are fit to a gamma distribution and the modal intensity is taken as the single molecule intensity. A representative distribution of single step traces with fit is shown in Figure 7.</li><li>All traces from the CypA binding experiments are divided by this value to convert the traces from fluorescent intensity to the number of molecules.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190702/20190702020953_8578.jpg" width="258"/><br/><strong>Figure 7. Single molecule photobleaching of AF568-CypA.</strong> A. TIRF image of AF568-CypA molecules adhered to the coverslip surface; particles detected by the image analysis software are marked with a red dot and a yellow circle. B. Representative distribution of step height from single-molecule photobleaching traces. The black line represents a Gaussian fit of the distribution.<br/></li></ol></li></ol><p class="pptt" id="biaoti28393">### Recipes</p><ol class="npwtwok"><li>Cell culture media<br/>Dulbeccoâs Modified Eagle Medium (DMEM) supplemented with 10% fetal bovine serum (FBS)</li><li>HBS<br/>50 mM HEPES<br/>100 mM NaCl<br/>Adjust to pH 7.0 or 7.5</li><li>Lysis buffer<br/>25 mM HEPES, pH 7.6<br/>1 mM DTT<br/>0.02% w/v NaN<sub>3</sub><br/>1 mg/ml lysozyme<br/>Supplemented with "Complete" protease inhibitor</li><li>AIEX buffer A<br/>25 mM HEPES, pH 7.6<br/>1 mM DTT<br/>0.02% NaN<sub>3</sub> </li><li>AIEX buffer B<br/>AIEX buffer A<br/>1.5 M NaCl</li><li>CIEX buffer A<br/>25 mM sodium phosphate pH 5.8<br/>1 mM DTT<br/>0.02% NaN<sub>3</sub> </li><li>CIEX buffer B<br/>CIEX buffer A<br/>1 M NaCl</li><li>CypA storage buffer<br/>25 mM MOPS, pH 6.6<br/>1 mM DTT<br/>0.02% NaN<sub>3</sub> </li><li>AF568-CypA storage buffer<br/>50 mM Tris, pH 8<br/>20% v/v glycerol<br/>1 mM DTT</li><li>Blocking buffer<br/>20 mM Tris pH 7.5<br/>2 mM EDTA<br/>50 mM NaCl<br/>0.03% NaN<sub>3</sub><br/>0.025% Tween 20<br/>0.2 mg/ml BSA<br/></li></ol><p class="pptt" id="biaoti28394">### Acknowledgments</p><p class="ppzz">The construct for expression of CypA was supplied by Nick Dixon (University of Wollongong, Australia). Recombinant PFO was purified by Sara Lawrence (St. Vincentâs Institute of Medical Research and University of Melbourne, Australia). The TIRF microscope was designed and built by Philip Nicovich (current address: Allen Institute for Brain Science, Seattle, USA). CM and DL received an Australian Government Research Training Program Scholarship. TB received funding from the Australian Centre for HIV and Hepatitis Virology Research and from the National Health and Medical Research Council of Australia (NHMRC APP1100771). Funding from the Victorian Government Operational Infrastructure Support Scheme to St Vincentâs Institute is acknowledged. MWP is a National Health and Medical Research Council of Australia Research Fellow.</p><p class="pptt" id="biaoti28395">### Competing interests</p><p class="ppzz">There are no conflicts of interest or competing interest.</p><p class="pptt" id="biaoti28396">### References</p><ol class="npwtwok"><li>Aggarwal, A., Iemma, T. L., Shih, I., Newsome, T. P., McAllery, S., Cunningham, A. L. and Turville, S. G. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22685410" target="_blank">Mobilization of HIV spread by diaphanous 2 dependent filopodia in infected dendritic cells.</a> <em>PLoS Pathog</em> 8(6): e1002762.</li><li>BÃ¶cking, T., Aguet, F., Harrison, S. C. and Kirchhausen, T. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21278753" target="_blank">Single-molecule analysis of a molecular disassemblase reveals the mechanism of Hsc70-driven clathrin uncoating.</a> <em>Nat Struct Mol Biol </em>18(3): 295-301.</li><li>Briggs, J. A., Simon, M. N., Gross, I., Krausslich, H. G., Fuller, S. D., Vogt, V. M. and Johnson, M. C. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15208690" target="_blank">The stoichiometry of Gag protein in HIV-1.</a> <em>Nat Struct Mol Biol</em> 11(7): 672-675.</li><li>Briggs, J. A., Wilk, T., Welker, R., Krausslich, H. G. and Fuller, S. D. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12660176" target="_blank">Structural organization of authentic, mature HIV-1 virions and cores.</a> <em>EMBO J</em> 22(7): 1707-1715.</li><li>Burdick, R. C., Delviks-Frankenberry, K. A., Chen, J., Janaka, S. K., Sastri, J., Hu, W. S. and Pathak, V. K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28827840" target="_blank">Dynamics and regulation of nuclear import and nuclear movements of HIV-1 complexes.</a> <em>PLoS Pathog</em> 13(8): e1006570.<br/></li><li>Campbell, E. M., Perez, O., Anderson, J. L. and Hope, T. J. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18250195" target="_blank">Visualization of a proteasome-independent intermediate during restriction of HIV-1 by rhesus TRIM5Î±.</a> <em>J Cell Biol </em>180(3): 549-561.</li><li>Dharan, A., Talley, S., Tripathi, A., Mamede, J. I., Majetschak, M., Hope, T. J. and Campbell, E. M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27327622" target="_blank">KIF5B and Nup358 cooperatively mediate the nuclear import of HIV-1 during infection.</a> <em>PLoS Pathog</em> 12(6): e1005700.</li><li>Fish, K. N. (2009). <a href="https://europepmc.org/abstract/med/19816922" target="_blank">Total internal reflection fluorescence (TIRF) microscopy.</a> <em>Current Protocols in Cytometry.</em> Chapter 12, Unit12.18. https://doi.org/10.1002/0471142956.cy1218s50.</li><li>Forshey, B. M., von Schwedler, U., Sundquist, W. I. and Aiken, C. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11991995" target="_blank">Formation of a human immunodeficiency virus type 1 core of optimal stability is crucial for viral replication.</a> <em>J Virol </em>76(11): 5667-5677.</li><li>Francis, A. C. and Melikyan, G. B. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29649444" target="_blank">Single HIV-1 imaging reveals progression of infection through CA-dependent steps of docking at the nuclear pore, uncoating, and nuclear transport.</a> <em>Cell Host Microbe</em> 23(4): 536-548 e536.</li><li>Francis, A. C., Marin, M., Shi, J., Aiken, C. and Melikyan, G. B. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27322072" target="_blank">Time-resolved imaging of single HIV-1 uncoating <em>in vitro</em> and in living cells.</a> <em>PLoS Pathog</em> 12(6): e1005709.</li><li>Hubner, W., Chen, P., Del Portillo, A., Liu, Y., Gordon, R. E. and Chen, B. K. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17728233" target="_blank">Sequence of human immunodeficiency virus type 1 (HIV-1) Gag localization and oligomerization monitored with live confocal imaging of a replication-competent, fluorescently tagged HIV-1.</a> <em>J Virol</em> 81(22): 12596-12607.</li><li>Hulme, A. E. and Hope, T. J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24158812" target="_blank">The cyclosporin A washout assay to detect HIV-1 uncoating in infected cells.</a> <em>Methods Mol Biol</em> 1087: 37-46.</li><li>Hulme, A. E., Perez, O. and Hope, T. J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21628558" target="_blank">Complementary assays reveal a relationship between HIV-1 uncoating and reverse transcription.</a> <em>Proc Natl Acad Sci U S A</em> 108(24): 9975-9980.</li><li>Jacques, D. A., McEwan, W. A., Hilditch, L., Price, A. J., Towers, G. J. and James, L. C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27509857" target="_blank">HIV-1 uses dynamic capsid pores to import nucleotides and fuel encapsidated DNA synthesis.</a> <em>Nature</em> 536(7616): 349-353.</li><li>Kotov, A., Zhou, J., Flicker, P. and Aiken, C. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10482638" target="_blank">Association of Nef with the human immunodeficiency virus type 1 core.</a> <em>J Virol</em> 73(10): 8824-8830.</li><li>Lahaye, X., Satoh, T., Gentili, M., Cerboni, S., Conrad, C., Hurbain, I., El Marjou, A., Lacabaratz, C., Lelievre, J. D. and Manel, N. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24269171" target="_blank">The capsids of HIV-1 and HIV-2 determine immune detection of the viral cDNA by the innate sensor cGAS in dendritic cells.</a> <em>Immunity</em> 39(6): 1132-1142.</li><li>Lusic, M. and Siliciano, R. F. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27941817" target="_blank">Nuclear landscape of HIV-1 infection and integration.</a> <em>Nat Rev Microbiol</em> 15(2): 69-82.</li><li>Ma, Y., He, Z., Tan, T., Li, W., Zhang, Z., Song, S., Zhang, X., Hu, Q., Zhou, P., Wu, Y., Zhang, X. E. and Cui, Z. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27253587" target="_blank">Real-time imaging of single HIV-1 disassembly with multicolor viral particles.</a> <em>ACS Nano</em> 10(6): 6273-6282.</li><li>Mallery, D. L., Marquez, C. L., McEwan, W. A., Dickson, C. F., Jacques, D. A., Anandapadamanaban, M., Bichel, K., Towers, G. J., Saiardi, A., Bocking, T. and James, L. C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29848441" target="_blank">IP6 is an HIV pocket factor that prevents capsid collapse and promotes DNA synthesis.</a> <em>Elife</em> 7: e35335.</li><li>Mamede, J. I., Cianci, G. C., Anderson, M. R. and Hope, T. J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28784755" target="_blank">Early cytoplasmic uncoating is associated with infectivity of HIV-1.</a> <em>Proc Natl Acad Sci U S A </em>114(34): E7169-E7178.</li><li>Marquez, C. L., Lau, D., Walsh, J., Shah, V., McGuinness, C., Wong, A., Aggarwal, A., Parker, M. W., Jacques, D. A., Turville, S. and Bocking, T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29877795" target="_blank">Kinetics of HIV-1 capsid uncoating revealed by single-molecule analysis.</a> <em>Elife</em> 7: e34772.</li><li>Matreyek, K. A., Yucel, S. S., Li, X. and Engelman, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24130490" target="_blank">Nucleoporin NUP153 phenylalanine-glycine motifs engage a common binding pocket within the HIV-1 capsid protein to mediate lentiviral infectivity.</a> <em>PLoS Patho</em><em>g</em> 9(10): e1003693.</li><li>Nicovich, P. R., Walsh, J., Bocking, T. and Gaus, K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28301563" target="_blank">NicoLase-An open-source diode laser combiner, fiber launch, and sequencing controller for fluorescence microscopy.</a> <em>PLoS One</em> 12(3): e0173879.</li><li>Ocwieja, K. E., Brady, T. L., Ronen, K., Huegel, A., Roth, S. L., Schaller, T., James, L. C., Towers, G. J., Young, J. A., Chanda, S. K., Konig, R., Malani, N., Berry, C. C. and Bushman, F. D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21423673" target="_blank">HIV integration targeting: a pathway involving Transportin-3 and the nuclear pore protein RanBP2.</a> <em>PLoS Pathog</em> 7(3): e1001313.</li><li>Ozawa, K., Headlam, M. J., Schaeffer, P. M., Henderson, B. R., Dixon, N. E. and Otting, G. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15479237" target="_blank">Optimization of an Escherichia coli system for cell-free synthesis of selectively N-labelled proteins for rapid analysis by NMR spectroscopy.</a> <em>Eur J Biochem</em> 271(20): 4084-4093.</li><li>Pereira, C. F., Ellenberg, P. C., Jones, K. L., Fernandez, T. L., Smyth, R. P., Hawkes, D. J., Hijnen, M., Vivet-Boudou, V., Marquet, R., Johnson, I. and Mak, J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21347302" target="_blank">Labeling of multiple HIV-1 proteins with the biarsenical-tetracysteine system.</a> <em>PLoS One</em> 6(2): e17016.</li><li>Perez-Caballero, D., Hatziioannou, T., Zhang, F., Cowan, S. and Bieniasz, P. D. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16306627" target="_blank">Restriction of human immunodeficiency virus type 1 by TRIM-CypA occurs with rapid kinetics and independently of cytoplasmic bodies, ubiquitin, and proteasome activity.</a> <em>J Virol</em> 79(24): 15567-15572.</li><li>Rasaiyaah, J., Tan, C. P., Fletcher, A. J., Price, A. J., Blondeau, C., Hilditch, L., Jacques, D. A., Selwood, D. L., James, L. C., Noursadeghi, M. and Towers, G. J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24196705" target="_blank">HIV-1 evades innate immune recognition through specific cofactor recruitment.</a> <em>Nature</em> 503(7476): 402-405.</li><li>Schaller, T., Ocwieja, K. E., Rasaiyaah, J., Price, A. J., Brady, T. L., Roth, S. L., Hue, S., Fletcher, A. J., Lee, K., KewalRamani, V. N., Noursadeghi, M., Jenner, R. G., James, L. C., Bushman, F. D. and Towers, G. J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22174692" target="_blank">HIV-1 capsid-cyclophilin interactions determine nuclear import pathway, integration targeting and replication efficiency.</a> <em>PLoS Pathog</em> 7(12): e1002439.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682.</li><li>Shah, V. B. and Aiken, C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22105356" target="_blank"><em>In vitro</em> uncoating of HIV-1 cores.</a> <em>J Vis Exp</em>(57): 3384.</li><li>Shepard, L. A., Heuck, A. P., Hamman, B. D., Rossjohn, J., Parker, M. W., Ryan, K. R., Johnson, A. E. and Tweten, R. K. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9772185" target="_blank">Identification of a membrane-spanning domain of the thiol-activated pore-forming toxin <em>Clostridium perfringens</em> perfringolysin O: an Î±-helical to Î²-sheet transition identified by fluorescence spectroscopy.</a> <em>Biochemistry</em> 37(41): 14563-14574.</li><li>Sowd, G. A., Serrao, E., Wang, H., Wang, W., Fadel, H. J., Poeschla, E. M. and Engelman, A. N. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26858452" target="_blank">A critical role for alternative polyadenylation factor CPSF6 in targeting HIV-1 integration to transcriptionally active chromatin.</a> <em>Proc Natl Acad Sci U S A</em> 113(8): E1054-1063.</li><li>Stremlau, M., Perron, M., Lee, M., Li, Y., Song, B., Javanbakht, H., Diaz-Griffero, F., Anderson, D. J., Sundquist, W. I. and Sodroski, J. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16540544" target="_blank">Specific recognition and accelerated uncoating of retroviral capsids by the TRIM5Î± restriction factor.</a> <em>Proc Natl Acad Sci U S A</em> 103(14): 5514-5519.</li><li>Wade, K. R., Hotze, E. M., Kuiper, M. J., Morton, C. J., Parker, M. W. and Tweten, R. K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25646411" target="_blank">An intermolecular electrostatic interaction controls the prepore-to-pore transition in a cholesterol-dependent cytolysin.</a> <em>Proc Natl Acad Sci U S A</em> 112(7): 2204-2209.</li><li>Yang, Y., Luban, J. and Diaz-Griffero, F. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24158811" target="_blank">The fate of HIV-1 capsid: a biochemical assay for HIV-1 uncoating.</a> <em>Methods Mol Biol</em> 1087: 29-36.<br/></li></ol>