<body>
            <h1>A Protocol to Map the Spatial Proteome Using HyperLOPIT in <em>Saccharomyces cerevisiae</em></h1>
            <p><strong>Authors:</strong> Daniel J. H. Nightingale, Kathryn S. Lilley, Stephen G. Oliver</p>
            <h2>Abstract</h2>
            <p>The correct subcellular localization of proteins is vital for cellular function and the study of this process at the systems level will therefore enrich our understanding of the roles of proteins within the cell. Multiple methods are available for the study of protein subcellular localization, including fluorescence microscopy, organelle cataloging, proximity labeling methods, and whole-cell protein correlation profiling methods. We provide here a protocol for the systems-level study of the subcellular localization of the yeast proteome, using a version of hyperplexed Localization of Organelle Proteins by Isotope Tagging (hyperLOPIT) that has been optimized for use with <em>Saccharomyces cerevisiae</em>. The entire protocol encompasses cell culture, cell lysis by nitrogen cavitation, subcellular fractionation, monitoring of the fractionation using Western blotting, labeling of samples with TMT isobaric tags and mass spectrometric analysis. Also included is a brief explanation of downstream processing of the mass spectrometry data to produce a map of the spatial proteome. If required, the nitrogen cavitation lysis and Western blotting portions of the protocol may be performed independently of the mass spectrometry analysis. The protocol in its entirety, however, enables the unbiased, systems-level and high-resolution analysis of the localizations of thousands of proteins in parallel within a single experiment.</p>
            <p><strong>Keywords:</strong> <em>Saccharomyces cerevisiae</em>, Subcellular fractionation, Nitrogen cavitation, Quantitative proteomics, Yeast, Spatial proteomics</p>
            <h2>Background</h2><p>Knowledge of the subcellular localization of a protein is crucial in defining its functions within the cell and regulation of the spatial organization of the proteome is essential for cellular homeostasis. Subcellular protein localization allows proteins to act as members of higher-order complexes with binding partners such as proteins, metabolites and co-factors, and contribute to the functions of organelles. In some cases, the sequestration of enzymes within specific subcellular compartments is critical to ensure that intermediates are not exchanged between different metabolic pathways or that potentially detrimental reactions (<em>e.g.</em>, proteolysis) are confined to a âsafeâ space. Protein localization can be influenced by post-translational modification, such as the phosphorylation of proteins within signaling cascades that ultimately allow proteins such as transcription factors (TFs) to localize within the nucleus to exercise their gene regulatory functions, or allowing other proteins to traverse the nuclear membrane and regulate nuclear-localized transcription factors. One example is the case of the osmostress response pathway in yeast, where a signaling cascade is activated that results in the phosphorylation of the protein Hog1p, whereupon it can translocate to the nucleus and phosphorylate the TF Hot1p which, in turn, can recruit RNA polymerase II to the promoters of responsive genes (Alepuz <em>et al.</em>, 2003).<br/><br/>A growing body of evidence suggests that differential protein localization, even within the same cell type, may allow the protein to fulfil different roles depending on its respective localization within a cell, sometimes referred to as âmoonlightingâ (Gancedo <em>et al.</em>, 2016). In <em>Saccharomyces cerevisiae</em>, examples of such proteins include Gcn4p, which is the general transcription factor that controls amino acid biosynthesis (Hope and Struhl, 1986) and also acts as a ribonuclease (Nikolaev <em>et al.</em>, 2010); as well as Aco1p, which has roles in mitochondrial DNA stability in addition to its more widely known function as tricarboxylic acid cycle enzyme (Chen <em>et al.</em>, 2005). Isoforms of the same protein have further been observed with different localizations in the same cell type, suggesting that this differential localization may offer another level of regulation with regard to protein function. An example of this is the case of a DNA methyltransferase (Dnmt1) in murine stem cells, in which one isoform localized to chromatin and the other isoform is found in an alternative nuclear location (Christoforou <em>et al.</em>, 2016).<br/><br/>It is clear that there are many levels of complexity regarding protein subcellular localization. In order to fully understand the <em>in vivo</em> functions of proteins, it is vital to ascertain where, within the cell, they are located. Several methods exist that can be applied to approach this challenge, including fluorescence microscopy imaging, organelle proteomics, proximity biotinylation methods, and protein correlation profiling methods; all of which have been applied in <em>Saccharomyces cerevisiae</em> in different contexts to determine the subcellular localization of yeast proteins. Whole-cell, high-throughput studies based on fluorescence microscopy have been performed using GFP- or other fluorescence protein-tagged protein libraries (Huh <em>et al.</em>, 2003; Tkach <em>et al.</em>, 2012; Breker <em>et al.</em>, 2013; Denervaud <em>et al.</em>, 2013; Chong <em>et al.</em>, 2015; Weill <em>et al.</em>, 2018; Yofe <em>et al.</em>, 2016). Although powerful, these methods have documented problems in that the fluorescence signal of a particular protein of interest may not exceed the background level of cellular autofluorescence, particularly in yeast, and therefore it may not be possible to determine the location of such proteins. Additionally, some proteins may require their unmodified C- or N-terminus for correct localization (Weill <em>et al.</em>, 2018; Yofe <em>et al.</em>, 2016). As such, performing protein localization studies where proteins are tagged with a fluorescent protein either N- or C-terminally to the ORF in question may result in a given protein being reported with different localizations within the same biological system, depending on which strategy was used (Stadler <em>et al.</em>, 2013).<br/><br/>Organelle enrichment strategies have also been employed to define organelle proteomes, thereby contributing to our knowledge on protein localization. These strategies often exploit the characteristic buoyant densities of organelles in their enrichment. The relative ease with which mitochondria can be highly enriched, for example, has been exploited in many studies that have attempted to define the mitochondrial proteome, with varying levels of specificity, in <em>S. cerevisiae</em>. This includes some studies that used more crude approaches to attempt to define the mitochondrial proteome (Sickmann <em>et al.</em>, 2003; Reinders <em>et al.</em>, 2006) or specific sub-mitochondrial proteomes (Zahedi <em>et al.</em>, 2006; Vogtle <em>et al.</em>, 2012) in some cases without control for other organelle proteins that were co-enriched along with the mitochondrial proteins. Improvements have been made in recent studies that mapped the entirety of the mitochondrial sub-organellar proteome using more complex fractionation and proteome quantitation methods (Morgenstern <em>et al.</em>, 2017; Vogtle <em>et al.</em>, 2017). For organelles of more heterogeneous density, however, the difficulty of such studies can be compounded by the presence of contaminating membranes from other organelles that co-enrich with the organelle of interest (Wiederhold <em>et al.</em>, 2010). In focusing on a specific organelle, such methods fail to report on proteins that may be present at multiple locations <em>in vivo</em>. Further, there is no control for the proportion of an organelle that is lost upon enrichment.<br/><br/>Recently, new proximity labeling methods based on promiscuous biotinylation have shown promise in determining protein subcellular localization in organisms other than yeast. This includes the more detailed study of sub-organellar proteomes (Rhee <em>et al.</em>, 2013; Hung <em>et al.</em>, 2014), including cytoplasmic faces of organelles (Hung <em>et al.</em>, 2017). The BirA-based BioID method (Roux <em>et al.</em>, 2012) has been applied in <em>S. cerevisiae</em> to define the subcellular proteins that interact with a ribosomal scaffold protein (Opitz <em>et al.</em>, 2017) and the APEX2 method (Lam <em>et al.</em>, 2015) has been demonstrated to traverse the cell wall in proof-of-principle experiments in both <em>S. cerevisiae</em> and the fission yeast <em>Schizosaccharomyces pombe</em> (Hwang and Espenshade, 2016). Methods such as these focus only on the proximal interactors of a protein of interest, that are currently in the range of tens of nanometres (Rhee <em>et al.</em>, 2013; Kim <em>et al.</em>, 2014) and do not provide a whole-cell picture of the spatial organization of the proteome. <br/><br/>To overcome some of the inherent limitations of the methods described above, it is imperative to look at protein subcellular localization in the context of the whole cell. This has been commonly studied using whole-cell protein correlation profiling methods. Such methods are predicated on the observation that, when cell lysates are fractionated by some means, such as differential or density gradient centrifugation, proteins that localize to the same organelle will exhibit similar distributions when the protein distributions across the entire fractionation scheme are monitored (de Duve, 1971). This was originally monitored using enzyme assays that are representative of the organelles in question, but now employs quantitative mass spectrometry methods. Several such methods have been published, which offer differing levels of proteome coverage and subcellular resolution, employing different fractionation and mass spectrometry quantitation methods (Christoforou <em>et al.</em>, 2016; Itzhak <em>et al.</em>, 2016; Jean Beltran <em>et al.</em>, 2016; Mulvey <em>et al.</em>, 2017; Jadot <em>et al.</em>, 2017; Geladaki <em>et al.</em>, 2019). The hyperplexed Localization of Organelle Proteins by Isotope Tagging (hyperLOPIT) method uses density gradient centrifugation for subcellular fractionation, followed by multiplexed proteome quantitation using 10-plex TMT isobaric tags (Thompson <em>et al.</em>, 2003) and SPS-MS3 quantitation on an Orbitrap Fusion Lumos mass spectrometer. The exquisite subcellular resolution offered by the density gradient coupled with multiplexed quantitation, followed by sophisticated machine learning analysis methods means that hyperLOPIT allows unbiased protein localization determination of thousands of proteins in parallel in a single experiment. Indeed the method has afforded the highest resolution spatial map of any whole-cell correlation profile-based spatial proteome study to date (Thul <em>et al.</em>, 2017; Gatto <em>et al.</em>, 2019).<br/><br/>We recently published a yeast spatial proteome mapping study in which we mapped the proteome in high throughput using hyperLOPIT, under nitrogen-sufficient conditions (Nightingale <em>et al.</em>, 2019). In our study, we produced a dataset that provided localization data for 2,846 proteins. Only 936 of these proteins were found to map to a unique subcellular location, a number which corresponded to 32% of our observable spatial proteome. This suggests, in common with previous studies (Christoforou <em>et al.</em>, 2016; Thul <em>et al.</em>, 2017), that a large proportion of the proteome is dynamic or resides at multiple locations. <br/><br/>We provide here a protocol that describes how to perform nitrogen cavitation lysis (Hunter and Commerford, 1961; Simpson, 2010; Wang <em>et al.</em>, 2014) subcellular fractionation, and a version of the hyperLOPIT protocol that has been modified for use with the yeast <em>S. cerevisiae</em> (see Figure 1 for an overview of the protocol). We further draw the readerâs attention to the fact that the subcellular fractionation and Western blotting portion of this protocol may be performed as a standalone experiment without TMT labeling. Further to this, in cases where a mass spectrometer capable of resolution of TMT 10- or 11-plex tags (such as the Thermo Fusion series) is not available, isobaric tags with lower multiplexing capability may instead be used. This may be achieved using iTRAQ (Ross <em>et al.</em>, 2004) 4- or 8-plex, or TMT 6-plex, but, importantly, will result in reduced overall subcellular resolution relative to TMT 10- or 11-plex, as less of the subcellular fractionation gradient will be sampled in the experiment. HyperLOPIT and its predecessor, LOPIT, have been employed to great effect to map the subcellular proteomes of multiple different species and under numerous conditions (Sadowski <em>et al.</em>, 2006; Nikolovski <em>et al.</em>, 2012; Groen <em>et al.</em>, 2014; Christoforou <em>et al.</em>, 2016; Thul <em>et al.</em>, 2017). We envisage that this protocol will therefore be applicable and modifiable to study the spatial proteomes of other yeasts, including the industrial yeast <em>Komagataella phaffii</em> (syn. <em>Pichia pastoris</em>) and pathogenic yeasts such as <em>Candida albicans</em>.<br/><br/>We recently published a protocol in <em>Methods in Molecular Biology</em> (Nightingale <em>et al.</em>, 2018) that describes how to perform hyperLOPIT in <em>S. cerevisiae</em>. The lysis and subcellular fractionation portions of this protocol are substantially updated from a previous protocol by Wang and colleagues (2014). Here we provide a protocol with updated mass spectrometry data processing parameters using Proteome Discoverer 2.1 (Thermo Fisher Scientific, <a href="http://www.thermofisher.com/" target="_blank">www.thermofisher.com</a>), and with more focus on post-mass spectrometry informatics analysis. For a more in-depth and detailed informatics protocol than is presented here, we direct the reader to the publication of Breckels and colleagues (Breckels <em>et al.</em>, 2016b). The subcellular fractionation portion of the hyperLOPIT protocol is technically challenging, requiring an unbroken stretch of experimental work lasting from cell harvest to gradient fraction collection. We strongly recommend that experimental work not be paused until the gradient fractions have been collected.<br/><br/><img alt="" class="layerphoto" height="370" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190712/20190712011135_3644.jpg" width="100"/><br/><strong>Figure 1. Overview of the hyperLOPIT method for <em>Saccharomyces cerevisiae</em>.</strong> Also included in parentheses are approximate timescales for each step, and potential stopping points.<br/></p>
        </body>

<p class="pptt" id="biaoti28454">## Materials and Reagents</p><ol class="npwtwok"><li>96-well plates (Thermo Fisher Scientific, catalog number: 167008) <br/></li><li>Falcon tubes, 15 and 50 ml capacity (Corning, catalog numbers: 430791 and 430291)</li><li>Luer-lock syringes, 2 ml, 5 ml and 10 ml capacity (BD Plastipak, catalog numbers: 300185, 302187 and 300912, respectively)</li><li>Microcentrifuge tubes, 1.5 ml capacity (Eppendorf, catalog number: 0030120086)</li><li>Pipette tips (Rainin LTS 1,000 Î¼l, LTS 250 Î¼l and LTS 20 Î¼l; catalog numbers: 17001864, 17001863 and 17001865; or equivalent tips compatible with pipettes used for this protocol)</li><li>Polycarbonate thickwall ultracentrifuge tubes, 32 ml capacity (Beckman Coulter, catalog number: 355631)</li><li>Polypropylene OptiSeal<sup>TM</sup> ultracentrifuge tubes, 11.2 ml capacity (Beckman Coulter, catalog number: 362181) and associated tube adaptors (Beckman Coulter, catalog number: 362202)</li><li>Semi-micro cuvettes (Sarstedt, catalog number: 67.742)</li><li>Silicone tubing (1 mm inner diameter, 1 mm wall diameter) (Fisher Scientific, catalog number: 10430313)<br/></li><li>Stainless steel blunt-ended needle, 14 gauge (Sigma-Aldrich, catalog number: Z261408)</li><li>Total recovery glass vials and caps with pre-slit septa (Waters, catalog number: 186000385C)</li><li>Waste container</li><li>X-ray film (Fujifilm, catalog number: 4741019289)</li><li>Trans-Blot<sup>Â®</sup> Turbo<sup>TM</sup> polyvinylidene fluoride (PVDF) membranes (Bio-Rad, catalog number: 1704157)</li><li><em>Saccharomyces cerevisiae</em> strain appropriate to experimental aims (<em>e.g.</em>, BY4741 (Baker Brachmann <em>et al.</em>, 1998), which we used for our recent hyperLOPIT study; Nightingale <em>et al.</em>, 2019)</li><li>Acetic acid (Sigma-Aldrich, catalog number: 320099)</li><li>Acetone, analytical grade (Sigma-Aldrich, catalog number: 32201)</li><li>Acetonitrile, HPLC gradient-grade (Fisher Scientific, catalog number: A/0627/17)</li><li>Adenine hemisulfate (Sigma-Aldrich, catalog number: A3159)</li><li>Amersham<sup>TM</sup> ECL<sup>TM</sup> Prime Enhanced Chemiluminescent Western Blotting Detection Reagent (GE Healthcare, catalog number RPN2232)</li><li>Ammonium formate (Sigma-Aldrich, catalog number: 70221)</li><li>Ammonium hydroxide (Sigma-Aldrich, catalog number: 30501)</li><li>Arginine (Sigma-Aldrich, catalog number: A8094)</li><li>Difco<sup>TM</sup> Bacto<sup>TM</sup> agar (Fisher Scientific, catalog number: 10455513)</li><li>Difco Bacto peptone (Fisher Scientific, catalog number: DF0118-17-0)</li><li>Difco Bacto yeast extract (Fisher Scientific, catalog number: DF0127-17-9)</li><li>cOmplete<sup>TM</sup> Mini protease inhibitor tablets (Roche, catalog number: 11836170001)</li><li>cOmplete<sup>TM</sup> protease inhibitor tablets (Roche, catalog number: 11873580001)</li><li>D-(+)-Glucose (Sigma-Aldrich, catalog number: G7021)</li><li>Dibasic potassium phosphate (Sigma-Aldrich, catalog number: P3786)</li><li>DL-Dithiothreitol (DTT) (Melford Laboratories, catalog number: D11000)</li><li>Ethylenediaminetetraacetic acid (EDTA) (Sigma-Aldrich, catalog number: E5134)</li><li>Ficoll<sup>Â®</sup> PM-400 (Sigma-Aldrich, catalog number: F4375)</li><li>Formic acid (VWR, catalog number: 20318.297)</li><li>HEPES free acid (Melford Laboratories, H75030)</li><li>Histidine (Sigma-Aldrich, catalog number: H6034)</li><li>Hydrochloric acid, 37% (v/v) (Fisher Scientific, catalog number: H/1200/PB15)</li><li>Hydroxylamine 50% (w/v) solution (Thermo Fisher Scientific, catalog number: 90115)</li><li>Iodoacetamide (Sigma-Aldrich, catalog number: I6125)</li><li>Isoleucine (Sigma-Aldrich, catalog number: I5281)</li><li>Leucine (Sigma-Aldrich, catalog number: L8912)</li><li>Lysine (Sigma-Aldrich, catalog number: L5501)</li><li>Magnesium acetate (Sigma-Aldrich, catalog number: M5661)</li><li>Magnesium chloride (Sigma-Aldrich, catalog number: M8266)</li><li>Methionine (Sigma-Aldrich, catalog number: M5308)</li><li>Non-fat dry milk powder, such as Marvel<sup>TM</sup> </li><li>OptiPrep<sup>TM</sup> (Sigma-Aldrich, catalog number: D1556)</li><li>Phenylalanine (Sigma-Aldrich, catalog number: P5482)</li><li>Potassium acetate (Sigma-Aldrich, catalog number: P1190)</li><li>Potassium chloride (Sigma-Aldrich, catalog number: 31248)</li><li>Protein concentration estimation assay, such as Pierce<sup>TM</sup> BCA protein assay (Thermo Fisher Scientific, catalog number: 23225), DC<sup>TM</sup> protein assay (Bio-Rad, catalog number: 5000112) or Quick Start<sup>TM</sup> Bradford assay (Bio-Rad, catalog number: 5000202)</li><li>Sequencing Grade Modified Trypsin (Promega, catalog number: V5111)</li><li>Sodium dodecyl sulfate (SDS) (Sigma-Aldrich, catalog number: 436143)</li><li>Sodium hydroxide (Sigma-Aldrich, catalog number: 06203)</li><li>Sorbitol (Sigma-Aldrich, catalog number: W302902)</li><li>Sucrose (Sigma-Aldrich, catalog number: S0389)</li><li>Threonine (Sigma-Aldrich, catalog number: T8441)</li><li>TMT 10-plex or 11-plex isobaric tagging reagents (Thermo Fisher Scientific, catalog number: 90111 or A37725, respectively)</li><li>Trichloroacetic acid (TCA) (Fisher Scientific, catalog number: 421455000)</li><li>Trifluoroacetic acid (TFA) (Thermo Fisher Scientific, catalog number: 85183)</li><li>Tris base (Melford Laboratories, catalog number: T60040)</li><li>Tris-(2-carboxyethyl) phosphine (TCEP) (Generon, catalog number: GEN-TCEP)</li><li>Tryptophan (Sigma-Aldrich, catalog number: T8941)</li><li>Tween<sup>Â®</sup> 20 (Sigma-Aldrich, catalog number: P1379)</li><li>Tyrosine (Sigma-Aldrich, catalog number: T8566)</li><li>Uracil (Sigma-Aldrich, catalog number: U1128)</li><li>Valine (Sigma-Aldrich, catalog number: V4638)</li><li>Water, HPLC gradient-grade (Fisher Scientific, catalog number: W/0106/17)</li><li>Yeast nitrogen base without amino acids (Sigma-Aldrich, catalog number: Y0626)</li><li>Zymolyase 100-T (Nacalai-Tesque, catalog number: 07665-55)</li><li>Anti-Pgk1p antibody, for use as cytosol Western blotting marker (Abcam, catalog number: ab113687)</li><li>Anti-Dpm1p antibody, for use as endoplasmic reticulum Western blotting marker (ThermoFisher Scientific, catalog number: A-6429)</li><li>Anti-Pep1p antibody, for use as Golgi apparatus Western blotting marker (Abcam, catalog number: ab113690)</li><li>Anti-Cox4p antibody, for use as mitochondrial Western blotting marker (ThermoFisher Scientific, catalog number: 459150)</li><li>Anti-Hh3p antibody, for use as nuclear Western blotting marker (Abcam, catalog number: ab1791)</li><li>Anti-Pma1p antibody, for use as plasma membrane Western blotting marker (Abcam, catalog number: ab4645)</li><li>5x SDS-PAGE sample buffer (Jena Biosciences, catalog number: BU-117) </li><li>4-20% Mini-PROTEAN<sup>Â®</sup> TGX<sup>TM</sup> Precast Protein Gels (Bio-Rad, catalog number: 4561096)</li><li>TGS buffer 10x (Bio-Rad, catalog number: 1610732)</li><li>10x yeast nitrogen base (YNB) (see Recipes)</li><li>10x complete supplement mixture (see Recipes)</li><li>40% (w/v) glucose (see Recipes)</li><li>Synthetic medium with 2% glucose (see Recipes)</li><li>YPD agar medium (see Recipes)</li><li>TCEP reduction buffer (see Recipes)</li><li>Spheroplasting medium (see Recipes)</li><li>Zymolyase 100-T solution (see Recipes)</li><li>Spheroplast wash medium (see Recipes)</li><li>(Optional) Ficoll lysis buffer (Kizer <em>et al.</em>, 2006) (see Recipes)</li><li>(Optional) Buffer NP (Kizer <em>et al.</em>, 2006) (see Recipes)</li><li>1x LB (see Recipes)</li><li>6x LB (see Recipes)</li><li>Iodixanol working solution (IWS) (see Recipes)</li><li>18% (w/v) OptiPrep solution (see Recipes)</li><li>16% (w/v) OptiPrep solution (see Recipes)</li><li>Protein resolubilization buffer (see Recipes)</li><li>Digestion buffer (see Recipes)</li><li>5% (v/v) hydroxylamine (see Recipes)</li><li>TTBS (see Recipes)</li><li>Blocking solution (see Recipes)</li><li>Dithiothreitol (DTT) solution (see Recipes)</li><li>Iodoacetamide (IAA) solution (see Recipes)</li><li>Equilibration buffer (see Recipes)</li><li>Desalting buffer 1 (see Recipes)</li><li>Desalting buffer 2 (see Recipes)</li><li>Elution buffer (see Recipes)</li><li>RP mobile phase stock solution (see Recipes)</li><li>RP mobile phase A (see Recipes)</li><li>RP mobile phase B (see Recipes)</li><li>RP resolubilization buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28455">### Equipment</p><ol class="npwtwok"><li>4 Â°C refrigerator (Polestar cooling, model: MR 100 E)</li><li>-20 Â°C freezer (Hotpoint, model: FZS175)</li><li>-80 Â°C freezer (New Brunswick Scientific, model: U725-G) </li><li>Acquity Ultra Performance Liquid Chromatography (UPLC) system with photodiode array (PDA) detector (Waters)</li><li>Acquity UPLC BEH C<sub>18</sub> column (Waters, catalog number: 186002353)</li><li>Analytical balance ( Explorer, model: EX124)</li><li>Auto-Densiflow peristaltic pump with automatic meniscus detection (Labconco)</li><li>Avanti<sup>Â®</sup> JXN-26 floor-top preparative centrifuge (Beckman Coulter)</li><li>Bioruptor<sup>Â®</sup> Plus sonicator (Diagenode, catalog number: B01020001)</li><li>Centrifuge bottle assemblies, 500 ml capacity (Beckman Coulter, catalog number: 355649)</li><li>Cuvette spectrophotometer (Cecil Instruments, model: 2041)</li><li>Dounce homogenizer, 7 ml capacity (Wheaton, catalog number: 357542)</li><li>Erlenmeyer flasks, 250 ml, 500 ml and 2 L capacity (Fisher Scientific, catalog numbers: 15429103, 15439103 and 11383454)</li><li>Handheld analog refractometer (Bellingham + Stanley, E-line, catalog number: 44-803)</li><li>JLA-10.500 rotor (Beckman Coulter, catalog number: 369681)</li><li>Nitrogen cavitation vessel (Parr Instrument Company, model: 4639)</li><li>Optima<sup>TM</sup> L80-XP floor-top ultracentrifuge (Beckman Coulter)</li><li>Orbitrap Fusion<sup>TM</sup> Lumos<sup>TM</sup> Tribrid<sup>TM</sup> mass spectrometer (Thermo Fisher Scientific)</li><li>Oxygen-free nitrogen cylinder (BOC, Nitrogen [Oxygen-Free] 230bar Cylinder, size: W) </li><li>Pipettes (Rainin LTS L-1000 XLS, L-200 XLS, L-20 XLS and L-10 XLS, catalog numbers: 17014382, 17014391, 1704392 and 17014388; or equivalent pipettes capable of dispensing the same volumes)</li><li>Plate-reader spectrophotometer, capable of measuring wavelengths between 250 nm and 850 nm (Molecular Devices, Spectramax M2e)</li><li>Pre-column for mass spectrometer-coupled nanoLC (Thermo Fisher Scientific, catalog number: 160454)</li><li>Proxeon EASY-Spray column (Thermo Fisher Scientific, catalog number: ES803)</li><li>Refrigerated benchtop centrifuge, capable of centrifuging 15 ml and 50 ml Falcon tubes and running at 4,500 <em>x g</em> (Eppendorf, model: 5804R)</li><li>Refrigerated benchtop microcentrifuge, capable of centrifuging microfuge tubes and running at 16,100 <em>x g</em> (Eppendorf, model: 5415R)</li><li>Sep-Pak<sup>Â®</sup> tC<sub>18</sub> cartridges, 100 mg sorbent per cartridge (Waters, catalog number: WAT036820)</li><li>Shaking incubator for cell culture (Sartorius, Certomat, model: BS-1)</li><li>SW32Ti rotor (Beckman Coulter, catalog number: 369650)</li><li>Thermo Scientific<sup>TM</sup> Sterilin<sup>TM</sup> Standard 90mm Petri Dishes (Fisher Scientific, catalog number: 15370366)</li><li>Tube holder for microfuge and Falcon tubes (Starlab, catalog number: E2345-1000)</li><li>Trans-Blot<sup>Â®</sup> Turbo<sup>TM</sup> system (Bio-Rad, catalog number: 1704150)</li><li>Type 70 Ti rotor (Beckman Coulter, catalog number: 337922)</li><li>Ultimate RSLCnano chromatography system (Thermo Fisher Scientific)</li><li>Vacuum centrifuge (Labconco, CentriVap concentrator)</li><li>VTi65.1 rotor (Beckman Coulter, catalog number: 362759)</li><li>Electrophoresis system and associated power pack (Bio-Rad, model: Mini-PROTEAN<sup>Â®</sup> II)<br/></li></ol><p class="pptt" id="biaoti28456">### Software</p><ol class="npwtwok"><li>Proteome Discoverer version 2.1 (Thermo Fisher Scientific, <a href="http://www.thermofisher.com/" target="_blank">www.thermofisher.com</a>)</li><li>R programming language (R Core Team, 2018) (<a href="http://www.r-project.org/" target="_blank">www.r-project.org</a>)</li><li>Bioconductor (Gentleman <em>et al.</em>, 2004) MSnbase package (Gatto and Lilley, 2012) (<a href="http://bioconductor.org/packages/release/bioc/html/MSnbase.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/MSnbase.html</a>)</li><li>Bioconductor (Gentleman<em> et al.</em>, 2004) pRoloc package (Gatto <em>et al.</em>, 2014) (<a href="http://bioconductor.org/packages/release/bioc/html/pRoloc.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/pRoloc.html</a>)</li><li>Bioconductor (Gentleman <em>et al.</em>, 2004) pRolocGUI package (Breckels <em>et al.</em>, 2018) (<a href="http://bioconductor.org/packages/release/bioc/html/pRolocGUI.html" target="_blank">http://bioconductor.org/packages/release/bioc/html/pRolocGUI.html</a>)</li><li>RStudio R programming language editor (RStudio Team, 2016) (<a href="http://www.rstudio.com/" target="_blank">www.rstudio.com</a>)</li><li>Mascot Server (Matrix Science, <a href="http://www.matrixscience.com/" target="_blank">www.matrixscience.com</a>)</li><li>Microsoft Excel<br/></li></ol><p class="pptt" id="biaoti28457">### Procedure</p><ol class="npwone"><li>Cell culture<ol class="npwtwo"><li>At least 3 days before you plan to carry out the experiment, streak-plate an appropriate yeast strain from a cryostock onto a YPD agar plate. Incubate at 30 Â°C in a stationary incubator for approximately 2 days until colonies form and subsequently store at 4 Â°C for several weeks. </li><li>The following day, transfer the starter culture to a flask filled with fresh medium. For a typical hyperLOPIT experiment we recommend culturing 720 OD units of yeast, which equates to 1.2 L of culture at OD<sub>600</sub> of 0.6. For this volume of culture, we use 2 L Erlenmeyer flasks.</li><li>Dilute such that the cells undergo a minimum of 2 doublings prior to harvest so that the desired cell density is reached at a convenient time.<br/><em>Note: The amount of cells required for a new growth condition, which yields enough protein per collected density gradient fraction for downstream steps of the hyperLOPIT protocol (&gt; 50 Î¼g per fraction), should be empirically determined for each experiment in question. For experiments where the goal is not to perform hyperLOPIT, smaller cultures should be sufficient, but this should be determined by the investigator. </em> </li><li>When the required OD<sub>600</sub> has been reached, harvest cells by centrifugation at 3,000 <em>x g</em> for 5 min at room temperature and discard the supernatant. For this step, use 500 ml bottle assemblies and the JLA 10.500 rotor in the Avanti centrifuge. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell pre-treatment for lysis<ol class="npwtwo"><li>Resuspend the cell pellet, still in the 500 ml bottle assemblies, in TCEP reduction buffer at 5 OD units per ml and incubate for 5 min at room temperature without shaking. Centrifuge at 3,000 <em>x g</em> for 5 min in the Avanti centrifuge to harvest the cells and discard the supernatant.</li><li>Resuspend the cells in spheroplasting medium at 20 OD units per ml and transfer to 50 ml Falcon tubes. Add 1 Î¼l of zymolyase 100-T solution per OD unit of yeast used in the experiment.<br/><em>Note: Spheroplasting medium is composed of the same constituents as the medium that was used for cell culture, to maintain the same conditions as were used for culture. We do not, however, recommend the use of conditioned media for this step. The media should be made fresh and include 1.0-1.2 M sorbitol and 20 mM Tris-HCl, pH 7.5, to maintain osmotic support and the appropriate pH during spheroplasting.</em> </li><li>Withdraw a 10 Î¼l aliquot of the solution and dilute in 990 Î¼l of water. Measure the OD<sub>600</sub> after blanking the spectrophotometer with an equivalent dilution of spheroplasting medium in water.<br/><em>Note: Spheroplast conversion efficiency is conveniently monitored by spectrophotometry. Absorbance of the cell suspension at 600 nm is proportional to the number of intact cells that are present in the suspension. Spheroplasts are very fragile and sensitive to changes in the tonicity of a solution. As such, transferring them to an osmotically unsupported solution will result in lysis of the spheroplasts. Only the remaining intact cells contribute to the absorbance at 600 nm, indicating the amount of cells that have undergone conversion to spheroplasts.</em> </li><li>Incubate at 30 Â°C for 10 min with shaking at 200 rpm and measure OD<sub>600</sub> of a 1:100 dilution. The OD<sub>600</sub> should be around 10% of the pre-digestion OD<sub>600</sub>, indicating around 90% zymolyase digestion efficiency.<br/><em>Note: This step should not be allowed to continue for more than 10 min, as incubation in the presence of zymolyase for too long can result in uncontrolled cell lysis.</em> </li><li>Harvest cells, without transferring to new tubes, by centrifugation for 5 min at 1,500 <em>x g</em>, 4 Â°C and resuspend in spheroplast wash medium.<br/><em>Note: The pellet can be sticky and difficult to resuspend, but do not vortex or pipette vigorously as this risks lysis of the fragile spheroplasts. Resuspend with a trimmed 1 ml pipette tip, use a glass rod or gently invert to resuspend. Due to the risk of lysis, do not aim for a homogeneous suspension of cells at this point.</em> </li><li>Harvest the cells for 5 min at 1,500 <em>x g</em>, 4 Â°C and discard supernatant.<br/><br/></li></ol></li><li>Optional: Nuclear preparation <br/><em>Note: If greater nuclear resolution is not of particular interest, skip to Section D.</em> <ol class="npwtwo"><li>Optionally perform a nuclear preparation according to Kizer <em>et al.</em> (2006) with modifications described subsequently. For this protocol, we use approximately 120 OD units of spheroplasts for a nuclear preparation and 600 OD units for the main subcellular fractionation.<br/><em>Note: This step is optional and if other organelles are of special interest, enrichments for these organelles may be performed instead of a nuclear preparation, as was performed in two previous hyperLOPIT studies (Christoforou </em><em>et al.</em><em>, 2016; Thul </em><em>et al.</em><em>, 2017).</em> </li><li>Resuspend the spheroplasts in Ficoll lysis buffer and transfer to a Dounce homogenizer. Lyse with 20 up-down strokes on ice.<br/><em>Note: As in the original paper (Kizer </em><em>et al.</em><em>, 2006), we use the âtightâ Dounce pestle for the entirety of lysis, and do not advocate âpre-resuspensionâ with the âlooseâ pestle.</em> </li><li>Pre-clear the lysate for 10 min at 3,220 <em>x g</em>, 4 Â°C and transfer the supernatant to a 32 ml round-bottomed ultracentrifuge tube. Balance the opposite position of a Type 70 Ti rotor (Beckman Coulter) with a tube of the same mass and density and centrifuge at 50,000 <em>x g</em>, 4 Â°C for 35 min. Use âMAXâ acceleration and deceleration.</li><li>Discard the supernatant and resuspend pellet in buffer NP with vigorous pipetting. Store resuspended nuclear-enriched pellet at -80 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Lysis for density gradient centrifugation<ol class="npwtwo"><li>Resuspend the remaining 600 OD units of spheroplasts in 1x LB at 20 OD units per ml and transfer to the chamber of the nitrogen cavitation vessel.</li><li>Lyse using a method modified from Wang <em>et al.</em> (2014). Charge the vessel with oxygen-free nitrogen to 500 psi and incubate for 3 min.<br/><em>Note: The nitrogen within the cylinder is under pressure and if the inlet tap is opened too quickly, it can lead to the target pressure of 500 psi being overshot. Therefore, open the inlet tap slowly and closely monitor the pressure gauge to obviate the chance of this occurring.</em> </li><li>Release the pressure by opening the gas inlet valve, so that the pressure drops to 300 psi.</li><li>Allow a further 3 min of incubation and discharge the vessel by opening the outlet port at the base of the vessel. Collect the lysate at approximately 3 drops per second.<br/><em>Note: As the nitrogen is released from solution during lysis, the pressure within the nitrogen cavitation vessel will drop and consequently the lysate will be released more slowly. Therefore monitor the rate at which the lysate is released from the outlet port and, if necessary, open the port more to maintain a steady rate of approximately 3 drops per second.</em> </li><li>Clear the lysate of debris, aggregates, and unlysed cells and spheroplasts by centrifugation for 5 min at 1,000 <em>x g</em>, 4 Â°C. Subject the supernatant to another round of centrifugation for 10 min at 3,000 <em>x g</em>, 4 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Crude membrane preparation<ol class="npwtwo"><li>Prepare 18% (w/v) OptiPrep solution with the aid of a handheld refractometer and calibration curve of known OptiPrep concentrations and their refractive index measurements, measured in degrees Brix (Â°Bx) (see Note 1).</li><li>Transfer the cleared lysate into four round-bottomed ultracentrifuge tubes (~7.5 ml lysate per tube). Underlay the contents of each tube with 5 ml 18% (w/v) OptiPrep cushion solution, prepared in 1x LB, using a syringe attached to a blunt-ended, wide-bore needle.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Even though it is possible to fit the entirety of the lysate in a single round-bottomed centrifuge tube, we do not recommend this. If this is done, upon ultracentrifugation, the sheer amount of protein in the entirety of lysate will cause it to aggregate on top of the cushion solution. This may cause damage to organelle membranes and affect the results of the experiment as a whole. We find that if the lysate is split as described, this does not occur.</em> </li><li><em>When underlaying with 18% (w/v) OptiPrep cushion solution, take the utmost care to not to introduce any air bubbles into the cushion solution. If these are introduced it will result in mixing of the lysate with the OptiPrep solution, disrupting the crisp interface that should be present between the two solutions. Ultracentrifugation of a solution where this has occurred will prevent formation of a crude membrane interphase that would normally form between the two solutions, defeating the object of performing this step. Instead this will potentially cause the membranes to pellet and aggregate at the bottom of the tube. To prevent this happening, use the graduations on the syringe as a guide to the amount of OptiPrep solution withdrawn before underlaying. Before commencing the underlaying of the lysate, press down gently on the syringe plunger to allow a single drop of OptiPrep to be released from the wide-bore needle (to expel any air that may be present). Submerge the needle at the bottom of the ultracentrifuge tube and begin injection of the cushion solution, keeping a close eye on the level of the liquid within the syringe. As soon as the meniscus of the liquid drops to the level of the connection between the syringe and needle, stop injecting. Remove the needle and discard the remainder of the solution.</em> </li></ol></li><li>Balance the four tubes to within 10 mg of each other and place the tubes in opposing sides of an SW32Ti swinging-bucket rotor.</li><li>Centrifuge the tubes for 2 h at 28,000 rpm (96,300 <em>x g</em>), 4 Â°C. Select an acceleration profile of âMAXâ and a deceleration profile of â9â (the slowest deceleration that uses a brake). This minimizes disruption of the membrane interphase that forms upon ultracentrifugation.</li><li>Withdraw the cytosol-enriched supernatant from the tube, leaving between 1 and 2 cm of the supernatant above the membrane interphase, so as not to disturb the crude membrane interphase. Store the supernatant at -20 Â°C for further processing.</li><li>Withdraw the membrane-containing interphase using a trimmed 1 ml pipette tip into a separate tube, taking care not to withdraw too much solution from either above or below the interphase.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Isopycnic density gradient ultracentrifugation and fraction collection<ol class="npwtwo"><li>Adjust the concentration of OptiPrep in the collected membrane interphase solution to 16% (w/v), monitoring the concentration using a handheld refractometer (see Note 1).<br/><em>Notes:</em> <ol class="npwthree"><li><em>Using 600 OD units of culture typically yields around 2-3 ml of crude membranes at this step. The aim should be to adjust the refractive index of the crude membranes to the required value in as small a volume as possible and to make up the remainder of the 11.2 ml volume of the resolving gradient tube with pre-prepared 16% (w/v) OptiPrep. If this is not adhered to, it may result in the undesirable situation that the membranes, in the correct OptiPrep concentration, are split across multiple density gradients.</em> </li><li><em>We find that preparing a self-generating gradient using 16% (w/v) OptiPrep provides good separation of the major subcellular organelles in yeast (Figure 2). This step may be modified if a different gradient shape is desired or specific subcellular organelles are the main focus of the experiment.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190712/20190712011609_2269.jpg" width="349"/><br/><strong>Figure 2. Representative data obtained using the protocol described here.</strong> After performing hyperLOPIT in <em>S. cerevisiae</em> with the optional nuclear preparation, two-dimensional Principal Components Analysis reveals the resolution of 12 separate organelles and protein complexes within the <em>S. cerevisiae</em> spatial proteome map, across three principal components. These encompass the main subcellular organelles of the organism (data reproduced from a single replicate performed in our previous study [Nightingale <em>et al.</em>, 2019]). Within each plot each point represents a single protein group, points that are colored represent proteins determined by hyperLOPIT to localize to a specific organelle and light grey points represent protein that do not have a localization predicted by hyperLOPIT.<br/></li></ol><br/></li><li>Prepare a balance tube containing 16% (w/v) OptiPrep and use to fill a blank vertical rotor tube, which does not contain any membranes and acts as a balance tube.</li><li>Balance the tubes to within 10 mg of each other, using 16% (w/v) OptiPrep solution. Place the two tubes in opposing sides of a VTi65.1 rotor and float aluminum spacers on top of the tubes.</li><li>Screw the rotor lids on and centrifuge at 65,000 rpm (362,900 <em>x g</em>), for 4 h at 4 Â°C. Select an acceleration profile of âMAXâ and a deceleration profile of â9â (the slowest deceleration that uses a brake). This acts to minimize disruption to the density gradient that has formed on ultracentrifugation.</li><li>Remove the tubes from the VTi65.1 rotor. Remove and discard the plastic plugs.</li><li>Prepare a Labconco Auto-Densiflow peristaltic pump with meniscus tracking probe for fraction collection by connecting a piece of collection tubing approximately 40 cm long to the probe. Clamp on to the peristaltic pump.<br/><em>Note: If an Auto-Densiflow fraction collector is unavailable, fractions may be collected by puncturing the bottom of the tube and allowing the gradient fractions to drip out from the bottom and most dense, fraction to the top and least dense fraction. Alternative fractionators are available, from Brandel and Teledyne Isco, but these have not been tested for use with hyperLOPIT. We do NOT recommend collecting fractions by pipetting from the top of the gradient as this requires a high level of technical skill and is very susceptible to error, which will result in mixing of different parts of the gradient and ruin the entire experiment.</em> </li><li>Flush the tubing with distilled water into a waste container and follow this by flushing through with air. Flush through with 1x LB and then air.</li><li>Insert the gradient tube into the tube holder and set the probe direction to âdownâ until the probe finds the gradient meniscus, at which point the probe will stop moving.</li><li>Set the probe to âdepositâ and collect 23 fractions from the gradient into 1.5 ml microcentrifuge tubes, of which 22 should be 0.5 ml and one should be ~0.2 ml.<br/><em>Note: This step is not automated, so volumes collected per fraction should be monitored closely by eye until the required volume has been collected for the fraction in question. Store fractions at -80 Â°C until further processing.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Monitoring gradient shape<ol class="npwtwo"><li>Thaw gradient fractions and vortex well to ensure that the iodixanol content in the OptiPrep-containing fractions is uniformly suspended within the fraction.</li><li>Determine refractive index of each fraction using a handheld, analog refractometer. Use approximately 20 Î¼l of each fraction for this step.<br/><em>Note: If the refractive index of any fraction exceeds the maximum refractive index of the refractometer, dilute that fraction with water and re-measure. Do not use 1x LB. Multiply the resultant refractive index by the dilution factor to obtain the undiluted refractive index measurement.</em> </li><li>Monitor density gradient shape by plotting fraction number against the refractive index of that fraction. The result should be a slightly sigmoidal curve that is relatively flat in the middle of the gradient and becomes steeper in the last 2-3 fractions (Figure 3).</li><li>If monitoring organelle distribution by enzyme assay, take an aliquot here with which to do this, and perform enzyme assays.<br/><em>Note: We have not attempted such methods in </em><em>S. cerevisiae</em><em> but methods are available in the literature if this is desired.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190712/20190712011647_0827.jpg" width="163"/><br/><strong>Figure 3. Typical gradient shape achieved using the concentration of OptiPrep and the density gradient ultracentrifugation parameters detailed within this protocol.</strong> Fractions were collected using an Auto-Densiflow fraction collector and are numbered in ascending order from the least to the most dense fraction of the gradient.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Fraction processing<ol class="npwtwo"><li>Process all fractions (including separate organelle preparations and cytosol-enriched fraction) further by TCA precipitation, followed with acetone washing. For this add Â¼ of the volume of 100% (w/v) TCA, prepared in water and vortex well.</li><li>Incubate for 2 h at 4 Â°C to allow the protein content of each of the fractions to precipitate from solution.</li><li>Harvest precipitates by centrifugation at maximum speed in a benchtop microcentrifuge at 4 Â°C for 10 min. Discard supernatants.</li><li>Wash pellets in acetone that has been pre-chilled to -20 Â°C. Vortex extensively and subject to sonication cycles (30 s on and 30 s off) in a Diagenode BioRuptor Plus sonicator, until the pellets break down completely and become powdery.<br/><em>Notes:</em> <ol class="npwthree"><li><em>If the pellets do not break down completely, they may be difficult to resolubilize in downstream steps of this protocol.</em> </li><li><em>If a BioRuptor is not available, other sonicators may be used for this step.</em> </li></ol></li><li>Repeat centrifugation, acetone washing, and sonication twice more; each time discarding the supernatant.</li><li>Discard supernatants and allow pellets to dry briefly at room temperature until acetone evaporates (see Note 2).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sample resolubilization and protein concentration estimation<ol class="npwtwo"><li>Resolubilize each pellet in protein resolubilization buffer with extensive sonication. <br/><em>Note: Resolubilize in the minimum volume in which the entire protein content is soluble.</em> </li><li>For this step, begin with 50 Î¼l of protein resolubilization buffer per pellet and monitor for the presence of any pellet in the fraction by centrifugation, as in the TCA precipitation step (Section H, step 3). Add further aliquots of protein resolubilization buffer, sonicate, and spin down to monitor the presence of any protein pellet. As soon as no pellet is visible after centrifugation, this indicates that the protein content has been resolubilized to completion.</li><li>Estimate protein concentration in each fraction using a protein concentration estimation assay, preferably in 96-well plate format.<br/><em>Notes:</em> <ol class="npwthree"><li><em>If the recommended protein assay kits are not available, other protein estimation assays may be used, as long as they are compatible with 100 mM HEPES and 0.1% (w/v) SDS.</em> </li><li><em>We recommend the use of 96-well plate format assay as sample amounts in hyperLOPIT experiments can be extremely limited and this conserves the maximum amount of each sample for Western blotting and isobaric tag labeling.</em> </li></ol></li><li>Determine overall protein yield. The ideal yield per fraction for isobaric tag labeling should be &gt; 50 Î¼g, allowing for an extra 1 Î¼g per fraction used for downstream Western blotting. If fractionation is required without isobaric tag labeling, lower yields will be suitable.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Monitoring organelle resolution by Western blot<ol class="npwtwo"><li>Monitor organelle resolution using Western blotting against a panel of antibodies raised against marker proteins whose resolution is characteristic of the organelle which they represent (see Note 3) (Table 1).<br/><br/><strong>Table 1. Suggested antibodies for use with hyperLOPIT in </strong><em><strong>S. cerevisiae</strong></em><strong>.</strong> The suggested primary and secondary antibody dilutions are based on Western blots that are carried out using 1 Î¼g of total protein per lane of an SDS-PAGE gel.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190712/20190712012532_1802.jpg" width="600"/><br/><br/></li><li>Normalize the amounts of each fraction (including the nuclear preparation, cytosol-enriched fraction, and density gradient fractions) to 1 Î¼g of total protein and heat to 95 Â°C in the presence of SDS-PAGE sample buffer for 5 min. Load each fraction on a single well of a pre-cast, gradient SDS-PAGE gel.</li><li>Subject the samples to electrophoresis in SDS-PAGE minigels in TGS buffer and using the full length of the gels. Use a Mini-PROTEAN II electrophoresis system and power pack for this step and run the gel at 200 V. </li><li>Transfer to Trans-Blot Turbo polyvinylidene fluoride (PVDF) membranes, using the mixed molecular weight program on a Bio-Rad Trans-Blot Turbo apparatus.</li><li>Cut membranes into appropriate molecular weight ranges (see Note 3) and incubate each strip in blocking solution for 1 h at room temperature on a rotary mixer.</li><li>Probe each strip with the appropriate primary antibody, diluted in blocking solution (see Note 4). Incubate overnight at 4 Â°C, with shaking on a rotary mixer (see Table 1 for suggested dilutions).</li><li>The following day, place the membranes back at room temperature and wash three times for 5 min each in TTBS.</li><li>Probe with the appropriate horseradish peroxidase-conjugated secondary antibody diluted in blocking solution (see Table 1 for suggested dilutions), for 1 h at room temperature.</li><li>Discard secondary antibody and wash 3 times for ten minutes each in TTBS.</li><li>Proceed to detection of signal using enhanced chemiluminescent (ECL) substrate.</li><li>Expose each membrane with X-ray film until the desired level of signal has been reached (see Figure 4 for an example Western blot).</li><li>Assuming a suitable yield (see Note 5) has been obtained, proceed to the next section if it is desired to proceed with the hyperLOPIT protocol.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190712/20190712011722_1403.jpg" width="191"/><br/><strong>Figure 4. Representative Western blot typical of an experiment on <em>S. cerevisiae</em>, performed using the hyperLOPIT protocol described here, in the absence of a nuclear preparation and probing using the antibodies described in Table 1.</strong> The fractions are numbered in the order in which they were collected using the Labconco fraction collector, from the first and least dense fraction, to the twenty-third and most dense fraction. The separate cytosol-enriched fraction is denoted by âCâ. Modified from (Nightingale <em>et al.</em>, 2019).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Protein reduction, alkylation and digestion<ol class="npwtwo"><li>For experiments including nuclear preparations, select the nuclear preparation, cytosol-enriched fraction, and use the remainder of the isobaric tags to label fractions from the density gradient. In experiments that do not include nuclear preparations, select the cytosolic fraction and use the remainder of the tags to label gradient fractions (see Note 4). This recommendation is the same whether TMT 10- or 11-plex tags are being used, or isobaric tags of lower multiplexing capability.<br/><em>Note: Fractions should be chosen for labeling on the basis of relative enrichment of organelles of interest, and relative depletion of other contaminating organelles, as determined by Western blotting. It should be noted that, even though proteins from only 7 organelles are probed in this Western blotting scheme, we are able to resolve 12 subcellular locations within our final dataset (Figure 2). Indeed, a well-designed fractionation scheme will result in the resolution of subcellular locations within the final spatial proteome map that were not included within the Western blotting experiment, as they will resolve discretely within the density gradient.</em> </li><li>Normalize each fraction to the same amount of protein (50-100 Î¼g) and make up to the same volume (100 Î¼l) with protein resolubilization buffer.<br/><em>Note: The amount of total protein may be reduced if yields are not high enough to use 100 Î¼g of each fraction. We do not, however, recommend labeling &lt; 50 Î¼g of each of the selected fractions.</em> </li><li>Add 11.1 Î¼l of dithiothreitol (DTT) solution prepared in protein resolubilization buffer to each sample.</li><li>Incubate for 1 h at 56 Â°C to allow reduction of disulfide bonds.</li><li>Return to room temperature and add 12.3 Î¼l of iodoacetamide (IAA) solution to each sample. Incubate for 1 h at room temperature in the dark.</li><li>Add 1 ml of acetone to each fraction, vortex the fractions briefly and allow protein content to precipitate in acetone overnight at -20 Â°C.</li><li>The following day, centrifuge all precipitated fractions at maximum speed in a benchtop microcentrifuge for ten minutes at 4 Â°C.</li><li>Discard the supernatants and briefly air-dry the pellets (see Note 2).</li><li>Resuspend the pellets in 87.5 Î¼l of digestion buffer by vortexing.</li><li>Digest with trypsin at 1:20 (w/w) protease:protein ratio, at 37 Â°C for 16 h. Add trypsin in two aliquots (1:40 (w/w) protease:protein ratio) spaced 1 h apart, such that the final volume of the digest is 100 Î¼l.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>TMT labeling (modified from the TMT 10-plex kit user guide)<ol class="npwtwo"><li>This section describes labeling of digests with TMT tags. If using a different isobaric labeling chemistry, consult the relevant labeling protocol that comes with the kit.</li><li>The following day, clear the digests of any insoluble material by centrifugation for 15 min in a microcentrifuge at maximum speed.</li><li>Allow a TMT kit to equilibrate to room temperature. Before opening the vials, briefly pulse-centrifuge them.</li><li>Add 41 Î¼l of acetonitrile to each tag vial. Vortex well and allow to solubilize for 5 min.</li><li>Transfer each resolubilized tag to a different digest and incubate for 2 h at room temperature on a shaking platform.</li><li>Add 8 Î¼l of 5% (v/v) hydroxylamine, prepared in digestion buffer to each sample and incubate for 30 min on a shaking platform at room temperature.</li><li>Add 100 Î¼l of HPLC-grade water to each sample and incubate for 1 h at 4 Â°C.</li><li>Pool all labeled digests into a single tube and dry to completion in a vacuum centrifuge at 10 Â°C. Store at -80 Â°C until sample clean-up by solid-phase extraction (SPE).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sample clean-up by solid-phase extraction (SPE) (modified from a previous publication [Villen and Gygi, 2008])<ol class="npwtwo"><li>Equilibrate a Sep-Pak tC<sub>18</sub> cartridge with 1.8 ml of 100% (v/v) HPLC-grade acetonitrile (ACN).</li><li>Flush through with 600 Î¼l Equilibration Buffer.</li><li>Flush through with 1.8 ml of Desalting Buffer 1.</li><li>Resolubilize sample in 1 ml 0.4% TFA and ensure that pH &lt; 2. To achieve this, check pH using pH paper on a small aliquot (~10 Î¼l) of sample. If pH &gt; 2, keep adding aliquots of 10% TFA, checking pH after each addition until pH &lt; 2.<br/><em>Note: The buffering capacity of residual HEPES from the digestion and isobaric tag labeling steps can keep pH high, even in the presence of 0.4% TFA. Ensure that pH &lt; 2 before loading on the SPE cartridge, or the labeled peptides will not bind to the tC</em><sub><em>18</em></sub><em> material and will, instead, be lost.</em> </li><li>Load resolubilized peptides onto the cartridge and allow to bind to the packing material.</li><li>Desalt in 1.8 ml Desalting Buffer 1.</li><li>Flush with 180 Î¼l Desalting Buffer 2.</li><li>Elute the desalted peptides from the cartridge using 1 ml Elution Buffer and remove a small aliquot corresponding to approximately 5-10 Î¼g of labeled peptides.</li><li>Dry the samples to completion in a vacuum centrifuge at 10 Â°C.<br/><br/></li></ol></li><li>Pre-fractionation by high pH reversed phase (RP) chromatography<ol class="npwtwo"><li>Fractionate the sample using high pH reversed phase chromatography prior to mass spectrometric analysis, on a Waters Acquity UPLC system.</li><li>Resolubilize solid phase extracted sample in 100 Î¼l RP resolubilization buffer by vortex mixing and spin down in a benchtop microcentrifuge for 10 min at maximum speed.</li><li>Transfer supernatant to an Acquity autosampler vial and place in the autosampler of a Waters Acquity UPLC system, maintained at 10 Â°C.<br/><em>Note: If desired another chromatography system may be used, that is capable of running in normal flow.</em> </li><li>Inject the sample onto the system and run using the gradient parameters shown in Table 2. Monitor sample elution using a photodiode array (PDA) detector, scanning wavelengths from 210-400 nm. Collect fractions at 1-min intervals with the aid of a timed fraction collector and freeze to completion on dry ice.<br/><br/><strong>Table 2. Parameters for use during high pH reversed phase pre-fractionation of hyperLOPIT samples on a Waters Acquity UPLC system.</strong> The entire LC gradient is 75 min long. Within the gradient, 0-10 min, 62-67.5 and 67.6-75 min should be isocratic. The gradient between each of the switching steps that is documented within this Table should be linear (represented by â6â with the Waters UPLC Inlet Method software).<br/><img alt="" class="layerphoto" height="212" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190712/20190712012616_9900.jpg" width="600"/><br/><br/></li><li>Dry fractions to completion in a vacuum centrifuge at 10 Â°C. Store dried fractions at -80 Â°C.</li><li>Use the collected chromatographic data to determine the collected fractions that correspond to the peptide-containing chromatographic space.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>LC-SPS-MS3 analysis of samples<ol class="npwtwo"><li>Resuspend the dried samples in 0.1% (v/v) formic acid and pool equal amounts in the following scheme: Pool the first with the middle fraction, the second with the second-from-middle, and so on until the end of the peptide-containing elution space.<br/><em>Note: Pooling the samples in this fashion not only makes better use of the second dimension of peptide fractionation (low pH RP coupled in-line to the mass spectrometer) (Zhang </em><em>et al.</em><em>, 2011), but also reduces mass spectrometry analysis time by half.</em> </li><li>Load approximately 1 Î¼g of each pooled fraction into the autosampler of the Dionex Ultimate 3000 RSLCnano chromatography system.<br/><em>Note: Base this measurement either on the protein estimation that was performed prior to Western blotting and protein digestion or, perform a peptide estimation assay to determine the amount to load.</em> </li><li>Analyze each pooled fraction by liquid chromatography-mass spectrometry using 120-min gradient on an Orbitrap Fusion Lumos Tribrid mass spectrometer coupled in line to the Dionex Ultimate 3000 RSLC nanoUPLC system. Acquire data using an LC-SPS-MS3 method (McAlister <em>et al.</em>, 2014). For details about the methods used for LC and MS, see previous publications that have employed TMT 10-plex quantitation coupled with SPS-MS3 to perform hyperLOPIT studies (Christoforou <em>et al.</em>, 2016; Thul <em>et al.</em>, 2017; Mulvey <em>et al.</em>, 2017). Alternatively, in the absence of a Fusion series mass spectrometer, use an appropriate LC-MS/MS method on the mass spectrometer that is being used for analysis.<br/><em>Notes: </em> <ol class="npwthree"><li><em>We recommend using a Thermo Fisher Scientific Fusion series mass spectrometer, as the SPS-MS3 capability (McAlister </em><em>et al.</em><em>, 2014) offers greater quantitative accuracy and precision in the measurement of the abundance of each TMT 10- or 11-plex reporter ion in a specific scan, relative to the sole use of MS/MS. SPS-MS3 involves synchronous selection of multiple MS2 fragment ions, arising from the target precursor ion, for further fragmentation in an MS3 scan. This liberates TMT reporter ions that arise solely from the target precursor ion, in the absence of reporter ion signal arising from the fragmentation of contaminating ions that were co-selected in the MS1 scan. Other tandem mass spectrometers may be used for these experiments, but they will be more susceptible to issues (Ting </em><em>et al.</em><em>, 2011; Wenger </em><em>et al.</em><em>, 2011) with co-selection of contaminating precursor ions in addition to the ion targeted for fragmentation by MS/MS. This will result in a reduction in the overall quantitative accuracy and precision of quantitation in the experiment, a direct readout of which will be diminished organelle resolution in the final hyperLOPIT spatial map.</em> </li><li><em>Analysis of TMT labeled samples using mass spectrometers that are not of high enough resolution may preclude the use of 10- or 11-plex TMT tags due to an inability to resolve the isotopolog N and C tags, which will result in reporter ion coalescence (McAlister </em><em>et al.</em><em>, 2012) and, consequently, in inaccurate quantitation. In such cases, isobaric tags with lower multiplexing capability should be used.</em> </li></ol></li><li>Analyze the extra, non-fractionated, sample collected at the SPE step using the same mass spectrometry method as was used in Step O3 above. This will enable characterization of the isobaric tag labeling efficiency.<br/><em>Note: It is recommended that the isobaric labeling efficiency be characterized for every hyperLOPIT experiment, independently of the analysis of the pre-fractionated samples by LC-SPS-MS3 or other LC-MS/MS method. The test ensures that the efficiency of labeling of both peptide N-termini and Îµ-amino groups of lysine residues with isobaric tag reagents is high enough (i.e., essentially complete). Indeed, we observe that labeling efficiencies are typically &gt; 99% using this method. It should be noted that if either the pH or acetonitrile concentration is modified during isobaric tag labeling, it will result in diminished stability of the isobaric tag labeling reagents, hydrolysis of the reagents, or undesirable side-reactions with residues such as tyrosine, serine, and threonine. This will result in incomplete labeling of peptides, which will lead to inaccurate PSM-level and overall protein-level quantitation. This will result in non-representative and reduced overall organelle resolution in the final hyperLOPIT spatial map.</em> </li></ol></li></ol><p class="pptt" id="biaoti28458">### Data analysis</p><ol class="npwone"><li>Analysis of mass spectrometry data using Proteome Discoverer version 2.1<ol class="npwtwo"><li>Analyze the raw mass spectrometry data files using Proteome Discoverer 2.1 interfaced with an in-house Mascot server. Search the data against a canonical <em>S. cerevisiae</em> database, such as a UniProt or SwissProt database, either in the presence or absence of protein isoformal information. Alternatively download a protein sequence database from the Saccharomyces Genome Database (SGDâ<a href="http://www.yeastgenome.org/" target="_blank">www.yeastgenome.org</a>) (Cherry <em>et al.</em>, 2012). Search the non-fractionated test sample file separately from the remainder of the fractionated samples. Perform the search for the fractionated sample files together, such that there is a single results report for the entire group of hyperLOPIT experimental mass spectrometry runs.</li><li>Perform the labeling test search before carrying out the search of the fractionated hyperLOPIT raw data files. For the labeling test search, specify TMT (or other isobaric tags) labeling of both N-termini and lysine residues as dynamic modifications, but keep all other search parameters the same as described in remainder of this section. Export the Results files at peptide-spectral match (PSM) level and calculate the percentage of peptides that exhibit each isobaric tag modification, relative to all peptides that could theoretically exhibit that modification (<em>i.e.</em>, all peptides with a lysine Îµ-amino group that can be labeled, and all peptide N-termini that are not otherwise modified). This gives an estimate of the labeling efficiency for that particular form of labeling, which should be &gt; 99% using this method.</li><li>For the fractionated hyperLOPIT data, load each set of raw data files into Proteome Discoverer as a set of Fractions, and not Files. Perform the Mascot search allowing a peptide tolerance of Â±15 ppm and fragment match tolerance of Â±0.6 Da if using LC-SPS-MS3. If using another LC-MS/MS method on a different mass spectrometry platform, use peptide and fragment match tolerances that are appropriate to the mass analyzers being used in the MS1 and MS2 scans. Include the Percolator algorithm (Brosch <em>et al.</em>, 2009) node within the software, to allow filtering of peptide-spectral matches based on their respective false discovery rate.</li><li>Set carbamidomethylation of cysteines, isobaric tag labeling of lysines and N-termini as static modifications. Set oxidation of methionine, and deamidation of asparagine and glutamine as dynamic modifications.</li><li>For analyses that use LC-SPS-MS3, in the reporter ion quantitation node, use the sum of centroided peaks within a window of Â±2 mmu around m/z of each monoisotopic reporter ion for quantitation and set MS order for activation to MS3. For experiments using other tandem mass spectrometric analysis methods and/or isobaric tags, set MS order for activation to MS2 and adjust the window accordingly for summing around the m/z of the reporter ion. Set protein group-level quantitation to be equal to the median of all quantified PSMs for any given protein group and report quantitation using the S/N measurement option that is available in Proteome Discoverer version 2.1.</li><li>Export the protein group-level results information to an Excel workbook or tab-delimited text file. If exported to an Excel workbook, convert the workbook to CSV format before proceeding to the next section.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Data analysis using the MSnbase and pRoloc suite of packages<ol class="npwtwo"><li>Analyze the results files from Proteome Discoverer using an R programming language editor such as RStudio (RStudio Team, 2016) that is running the Bioconductor (Gentleman <em>et al.</em>, 2004) pRoloc (Gatto <em>et al.</em>, 2014), pRolocGUI (Breckels <em>et al.</em>, 2018) and MSnbase packages (Gatto and Lilley, 2012). For an exhaustive analysis protocol, we direct the reader to recent work published by Breckels and colleagues (Breckels <em>et al.</em>, 2016b).</li><li>Read the raw data into the R environment language, using the âreadMSnSet2()â function, where âecolâ are specified as the columns that contain the quantitation data.</li><li>Filter protein groups from the experiment that contain missing values for quantitation, using the âfilterNA()â function. Alternatively, allow a certain number of missing values for quantitation and impute them in some way. <br/><em>Note: The missing values may be set to zero, as has been performed previously (Christoforou </em><em>et al.</em><em>, 2016) or they may be imputed using one of the âimpute()â methods that are described in the MSnbase package (Gatto, and Lilley, 2012).</em> </li><li>Normalize the quantitation data for each row (<em>i.e.</em>, protein group) such that the summed ratios are equal to 1. Use the ânormalize ()â function and the âsumâ method. <br/><em>Note: Other normalization methods are available within the MSnbase package, which may be used to normalize the Proteome Discoverer quantitation data but we find the âsumâ method useful for this particular Data analysis.</em> </li><li>Observe the data using dimensionality reduction such as Principal Components Analysis (PCA) or t-Distributed Stochastic Neighbor Embedding (t-SNE) (Van Der Maaten and Hinton, 2008) plots in the absence of any annotation. Note any cluster resolution that is evident from the data.</li><li>Curate a list of marker proteins that are known to localize to subcellular organelles in <em>S. cerevisiae</em>. Alternatively, use the markers that are provided within the pRoloc package (using pRolocmarkers(âscerâ)).<br/><em>Note: Ensure that marker annotation is included for all organelles that are expected to resolve discretely within the experiment. When performing classification of proteins to organelle classes using supervised machine learning approaches, proteins can only be classified to one of the organelle classes that was defined prior to starting the analysis. Therefore, if annotation is omitted for a specific organelle or set of organelles, it will result in proteins from that organelle or set of organelles being subsumed into another organelle that resolved similarly to the organelle that has been omitted within the experiment. The net result will be not only lack of the organelle in question in the final dataset, but also failure to classify any proteins to that organelle.</em> </li><li>Append the marker annotation to the MSnSet and observe the resolution within the data at the level of subcellular organelles.</li><li>Optionally, perform automated protein marker annotation using the âaddGOAnnotation()â function within pRoloc, in which the user can add Gene Ontology (GO) (Ashburner <em>et al.</em>, 2000; The Gene Ontology Consortium, 2017) annotation from any GO name space to search for clustering of proteins based on common molecular function, biological process, or cellular component. The resolution of GO annotated clusters that resolve discretely after this analysis can be observed using the pRolocGUI package. This can further be used to augment the marker protein data used for classification of proteins of unknown localization to subcellular organelles.</li><li>Perform protein classification using a binary classifier to classify proteins of unknown localization to one of the pre-defined organelle classes. This may encompass use of a supervised machine learning method, such as a Support Vector Machine (SVM) (Vapnik, 1995) as has been widely used in multiple previous hyperLOPIT studies (Christoforou <em>et al.</em>, 2016; Thul <em>et al.</em>, 2017). Newer methods may instead be employed which include a semi-supervised machine learning method (â<em>phenoDisco</em>â) (Breckels <em>et al.</em>, 2013), which can predict presence of novel protein clusters without <em>a priori</em> knowledge of their existence. A variant method is available that uses additional, lower quality, auxiliary data in addition to the highly curated marker protein list to improve classification performance (Breckels <em>et al.</em>, 2016a). Alternatively, a new probabilistic classification method may be used, which is based on Bayesian statistics (Crook <em>et al.</em>, 2018) and is now available for use within the pRoloc package.</li><li>Observe resolution of all proteins after classification to organelles.</li><li>Export all classification data on an organelle-by-organelle basis and sort SVM scores in descending order for all proteins classified as belonging to a given organelle.</li><li>Consult the original literature to look for corroboration between protein localizations predicted by hyperLOPIT and low throughput studies. Allow a certain permitted percentage of false discoveries for protein localization to a given organelle, based on the literature (<em>e.g.</em>, 5% or 1%).<br/><em>Note: Due to the nature of such machine-learning methods, all proteins in the dataset will be classified to a single location that was pre-defined in the organelle marker list training data. This does not faithfully recapitulate what happens in vivo, where proteins can localize to multiple subcellular organelles or traffic between multiple organelles.</em> </li><li>Set the SVM score cut-off for each organelle at this threshold and use the resultant data as the final spatial proteome map. See Figure 2 for representative data from a single biological replicate performed in reference (Nightingale <em>et al.</em>, 2019), without any protein classification. Further, see the Bioconductor (Gentleman <em>et al.</em>, 2004) pRolocdata (Gatto <em>et al.</em>, 2014) package (version 1.19.4 onwards) and <a href="https://proteome.shinyapps.io/yeast2018/" target="_blank">https://proteome.shinyapps.io/yeast2018/</a> (Nightingale <em>et al.</em>, 2019) for a fully processed dataset comprising data from four biological replicate experiments, performed with and without optional nuclear preparations, that can be interactively explored.</li><li>Finally, optionally validate the localizations of a subset of proteins that have been predicted by the hyperLOPIT analysis in a low throughput manner and using an orthogonal method. This may be performed using fluorescence microscopy, affinity purification-mass spectrometry, subtractive proteomics, proximity labeling, or density gradient protein co-fractionation experiments.</li></ol></li></ol><p class="pptt" id="biaoti28459">### Notes</p><ol class="npwtwok"><li>Prepare this solution by mixing iodixanol working solution (IWS) with 1x LB. To increase the OptiPrep concentration and refractive index, add IWS. To decrease, add 1x LB.</li><li>Do not over-dry pellets as they will become very difficult to resolubilize in subsequent steps. Drying for 5 min at room temperature should enable most of the acetone to evaporate. If the protein pellet begins to change from white to colorless, it is a sign that it is starting to over-dry.</li><li>To minimize the amount of protein used for Western blotting, run as few gels as possible. Cut membranes, post-blotting, into strips of appropriate molecular weight range based on their reported molecular weight and probe each strip with a single antibody. We typically find that we can probe for all of the markers in Table 1 running only two SDS-PAGE gels.</li><li>Primary antibodies diluted in blocking solution may be stored at -20 Â°C after use and reused at least 5 times, with no appreciable loss of signal. Do not re-use horseradish peroxidase-conjugated secondary antibodies.</li><li>If it is necessary to obtain high enough protein yield, the protein contents of adjacent fractions can be pooled prior to labeling. If pooling is performed multiple times within the same gradient, however, it will diminish the resolving ability of the density gradient, meaning that overall organelle resolution within the experiment will be impacted.<br/></li></ol><p class="pptt" id="biaoti28460">### Recipes</p><ol class="npwone"><li>Cell culture<ol class="npwtwo"><li>10x yeast nitrogen base (YNB)<br/>Dissolve 6.7 g yeast nitrogen base without amino acids in 1 L water and filter-sterilize</li><li>10x complete supplement mixture<br/>0.03% (w/v) isoleucine<br/>0.15% (w/v) valine<br/>0.04% (w/v) adenine hemisulfate<br/>0.02% (w/v) arginine<br/>0.02% (w/v) histidine<br/>0.1% (w/v) leucine<br/>0.03% (w/v) lysine<br/>0.02% (w/v) methionine<br/>0.05% (w/v) phenylalanine<br/>0.2% (w/v) threonine<br/>0.04% (w/v) tryptophan<br/>0.03% (w/v) tyrosine<br/>0.02% (w/v) uracil<br/>Dissolve all components in water and filter sterilize</li><li>40% (w/v) glucose<br/>Dissolve 40 g glucose in 100 ml water and sterilize by autoclaving</li><li>Synthetic medium with 2% glucose (1.2 L per experiment)<br/>Mix 120 ml 10x YNB, 60 ml 40% (w/v) glucose and 120 ml 10x complete supplement mixture<br/>Make up to 1.2 L with sterile water</li><li>YPD agar medium<ol class="npwthree"><li>Suspend 4 g Bacto agar, 4 g Bacto peptone and 2 g Bacto yeast extract in 190 ml water</li><li>Sterilize by autoclaving</li><li>Whilst still molten, add 10 ml 40% (w/v) glucose and pour agar plates (approximately 20 ml per plate)</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell pre-treatment and spheroplast generation<ol class="npwtwo"><li>TCEP reduction buffer (150 ml per experiment)<br/>20 mM Tris-HCl, pH 7.5<br/>10 mM Tris-(2-carboxyethyl) phosphine (TCEP)</li><li>Spheroplasting medium (40 ml per experiment)<br/>Composed of the same constituents as synthetic medium, but with the addition of 20 mM Tris-HCl, pH 7.5 and 1.2 M sorbitol<br/>Freshly supplement with 1 Complete EDTA-free protease inhibitor tablet per 50 ml buffer</li><li>Zymolyase 100-T solution (700 Î¼l per experiment)<br/>5 Î¼g/ml zymolyase 100-T prepared in spheroplasting medium<br/>Prepare 1 Î¼l for every OD unit of yeast used in the experiment</li><li>Spheroplast wash medium (150 ml per experiment)<br/>Composed of the same constituents as spheroplasting medium, with the omission of the Tris-HCl buffer<br/>Freshly supplement with 1 Complete EDTA-free protease inhibitor tablet per 50 ml buffer<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>(Optional) Nuclear preparation <ol class="npwtwo"><li>Ficoll lysis buffer (Kizer <em>et al.</em>, 2006) (15 ml per experiment)<br/>18% (w/v) Ficoll PM-400<br/>20 mM dibasic potassium phosphate, pH 6.8<br/>1 mM magnesium chloride<br/>0.5 mM EDTA<br/>Freshly supplement with 1 Complete Mini EDTA-free protease inhibitor tablet per 10 ml buffer</li><li>Buffer NP (Kizer <em>et al.</em>, 2006) (300 Î¼l per experiment)<br/>340 mM sucrose<br/>20 mM Tris-HCl, pH 7.4<br/>50 mM potassium chloride<br/>5 mM magnesium chloride<br/>Freshly supplement with 1 Complete Mini EDTA-free protease inhibitor tablet per 10 ml buffer<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell lysis<ol class="npwtwo"><li>1x LB (75 ml per experiment)<br/>250 mM sucrose<br/>50 mM potassium acetate<br/>2 mM magnesium acetate<br/>1 mM EDTA, pH 8.0<br/>Freshly supplement with 1 Complete EDTA-free protease inhibitor tablet per 50 ml buffer</li><li>6x LB (4 ml per experiment)<br/>300 mM potassium acetate<br/>12 mM magnesium acetate<br/>6 mM EDTA, pH 8.0<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Density gradient centrifugation<ol class="npwtwo"><li>Iodixanol working solution (IWS) (18 ml per experiment)<br/>Mix 3 ml 6x LB with 15 ml OptiPrep</li><li>18% (w/v) OptiPrep solution (25 ml per experiment)<ol class="npwthree"><li>Mix 9 ml IWS with 16 ml 1x LB</li><li>Double-check refractive index against a standard curve of OptiPrep concentrations of known refractive index</li><li>Adjust if needed, using IWS to increase refractive index and 1 x LB to reduce refractive index</li></ol></li><li>16% OptiPrep solution (20 ml per experiment)<ol class="npwthree"><li>Mix 6.4 ml IWS with 13.6 ml 1x LB</li><li>Double check refractive index against a standard curve of OptiPrep concentrations of known refractive index</li><li>Adjust if needed, using IWS to increase refractive index and 1x LB to reduce refractive index</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Fraction processing, digestion and TMT labeling<ol class="npwtwo"><li>Protein resolubilization buffer (20 ml per experiment)<br/>100 mM HEPES-NaOH, pH 8.5<br/>0.1% (w/v) SDS</li><li>Digestion buffer (2 ml per experiment)<br/>100 mM HEPES-NaOH, pH 8.5</li><li>5% (v/v) hydroxylamine (120 Î¼l per experiment)<br/>Mix 108 Î¼l of digestion buffer with 12 Î¼l 50% (w/v) hydroxylamine solution<br/>Prepare just before use</li><li>TTBS (1 L per experiment)<br/>25 mM Tris-HCl, pH 7.6<br/>150 mM NaCl<br/>0.1% (v/v) Tween-20</li><li>Blocking solution (100 ml per antibody used)<br/>5% (w/v) non-fat milk powder, prepared in TTBS</li><li>Dithiothreitol (DTT) solution<br/>100 mM DL-dithiothreitol, prepared in protein resolubilization buffer</li><li>Iodoacetamide (IAA) solution<br/>250 mM iodoacetamide, prepared in protein resolubilization buffer<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sample clean-up by solid phase extraction (SPE)<ol class="npwtwo"><li>Equilibration buffer (600 Î¼l per experiment)<br/>50% (v/v) acetonitrile<br/>0.5% (v/v) acetic acid</li><li>Desalting buffer 1 (1.8 ml per experiment)<br/>0.1% (v/v) trifluoroacetic acid</li><li>Desalting buffer 2 (180 Î¼l per experiment)<br/>0.5% (v/v) acetic acid</li><li>Elution buffer (1 ml per experiment)<br/>80% (v/v) acetonitrile<br/>0.5% (v/v) acetic acid<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>High pH reversed-phase chromatography<ol class="npwtwo"><li>RP mobile phase stock solution (100 ml per experiment)<br/>200 mM ammonium formate, pH 10.0. Adjust pH with ammonium hydroxide.</li><li>RP mobile phase A (500 ml per experiment)<br/>20 mM ammonium formate, pH 10.0<br/>Prepare by mixing 50 ml RP mobile phase stock solution with 450 ml HPLC-grade water<br/>Re-adjust to pH 10.0 with ammonium hydroxide, if necessary</li><li>RP mobile phase B (500 ml per experiment)<br/>20 mM ammonium formate, pH 10.0<br/>80% (v/v) acetonitrile<br/>Prepare by mixing 50 ml RP mobile phase stock solution with 400 ml HPLC-grade ACN and 50 ml HPLC-grade water<br/>Do not adjust pH</li><li>RP resolubilization buffer (100 Î¼l per experiment)<br/>95% (v/v) RP mobile phase A<br/>5% (v/v) RP mobile phase B</li></ol></li></ol><p class="pptt" id="biaoti28461">### Acknowledgments</p><p class="ppzz">We acknowledge funding from the BBSRC (Strategic Longer and Larger grant awarded to KSL (award BB/L002817/1) and CASE studentship awarded to KSL and SGO (award BB/I016147/1)). We thank Owen Vennard for critically reading this protocol.</p><p class="pptt" id="biaoti28462">### Competing interests</p><p class="ppzz">There are no conflicts of interest or competing interest.</p><p class="pptt" id="biaoti28463">### References</p><ol class="npwtwok"><li>Alepuz, P. M., de Nadal, E., Zapater, M., Ammerer, G. and Posas, F. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12743037" target="_blank">Osmostress-induced transcription by Hot1 depends on a Hog1-mediated recruitment of the RNA Pol II.</a> <em>EMBO J</em> 22(10): 2433-2442.</li><li>Ashburner, M., Ball, C. A., Blake, J. A., Botstein, D., Butler, H., Cherry, J. M., Davis, A. P., Dolinski, K., Dwight, S. S., Eppig, J. T., Harris, M. A., Hill, D. P., Issel-Tarver, L., Kasarskis, A., Lewis, S., Matese, J. C., Richardson, J. E., Ringwald, M., Rubin, G. M. and Sherlock, G. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" target="_blank">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.</a> <em>Nat Genet </em>25(1): 25-29.</li><li>Brachmann, C. B., Davies, A., Cost, G. J., Caputo, E., Li, J., Hieter, P. and Boeke, J. D. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9483801" target="_blank">Designer deletion strains derived from <em>Saccharomyces cerevisiae</em> S288C: a useful set of strains and plasmids for PCR-mediated gene disruption and other applications.</a> <em>Yeast</em> 14(2): 115-132.</li><li>Breckels, L. M., Holden, S. B., Wojnar, D., Mulvey, C. M., Christoforou, A., Groen, A., Trotter, M. W., Kohlbacher, O., Lilley, K. S. and Gatto, L. (2016a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27175778" target="_blank">Learning from heterogeneous data sources: an application in spatial proteomics.</a> <em>PLoS Comput Biol</em> 12(5): e1004920.</li><li>Breckels, L. M., Mulvey, C. M., Lilley, K. S. and Gatto, L. (2016b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30079225" target="_blank">A Bioconductor workflow for processing and analysing spatial proteomics data.</a> <em>F1000Res</em> 5: 2926.</li><li>Breckels, L. M., Naake, T. and Gatto L. (2018). <a href="https://bioconductor.org/packages/release/bioc/vignettes/pRolocGUI/inst/doc/pRolocGUI.html" target="_blank">Interactive visualisation of spatial proteomics data.</a> R package version 1.14.0.</li><li>Breckels, L. M., Gatto, L., Christoforou, A., Groen, A. J., Lilley, K. S. and Trotter, M. W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23523639" target="_blank">The effect of organelle discovery upon sub-cellular protein localisation.</a> <em>J Proteomics</em> 88: 129-140.</li><li>Breker, M., Gymrek, M. and Schuldiner, M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23509072" target="_blank">A novel single-cell screening platform reveals proteome plasticity during yeast stress responses.</a> <em>J Cell Biol</em> 200(6): 839-850.</li><li>Brosch, M., Yu, L., Hubbard, T. and Choudhary, J. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19338334" target="_blank">Accurate and sensitive peptide identification with Mascot Percolator.</a> <em>J Proteome Res</em> 8(6): 3176-3181.</li><li>Cherry, J. M., Hong, E. L., Amundsen, C., Balakrishnan, R., Binkley, G., Chan, E. T., Christie, K. R., Costanzo, M. C., Dwight, S. S., Engel, S. R., Fisk, D. G., Hirschman, J. E., Hitz, B. C., Karra, K., Krieger, C. J., Miyasato, S. R., Nash, R. S., Park, J., Skrzypek, M. S., Simison, M., Weng, S. and Wong, E. D. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22110037" target="_blank">Saccharomyces Genome Database: the genomics resource of budding yeast.</a> <em>Nucleic Acids Res</em> 40(Database issue): D700-705.</li><li>Chen, X. J., Wang, X., Kaufman, B. A. and Butow, R. A. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15692048" target="_blank">Aconitase couples metabolic regulation to mitochondrial DNA maintenance.</a> <em>Science</em> 307(5710): 714-717.</li><li>Chong, Y. T., Koh, J. L., Friesen, H., Duffy, S. K., Cox, M. J., Moses, A., Moffat, J., Boone, C. and Andrews, B. J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26046442" target="_blank">Yeast proteome dynamics from single cell imaging and automated analysis.</a> <em>Cell</em> 161(6): 1413-1424.</li><li>Christoforou, A., Mulvey, C. M., Breckels, L. M., Geladaki, A., Hurrell, T., Hayward, P. C., Naake, T., Gatto, L., Viner, R., Martinez Arias, A. and Lilley, K. S. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26754106" target="_blank">A draft map of the mouse pluripotent stem cell spatial proteome.</a> <em>Nat Commun</em> 7: 8992.</li><li>Crook, O. M., Mulvey, C. M., Kirk, P. D. W., Lilley, K. S. and Gatto, L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30481170" target="_blank">A Bayesian mixture modelling approach for spatial proteomics.</a> <em>PLoS Comput Biol</em> 14(11): e1006516.</li><li>Denervaud, N., Becker, J., Delgado-Gonzalo, R., Damay, P., Rajkumar, A. S., Unser, M., Shore, D., Naef, F. and Maerkl, S. J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24019481" target="_blank">A chemostat array enables the spatio-temporal analysis of the yeast proteome.</a> <em>Proc Natl Acad Sci U S A</em> 110(39): 15842-15847.</li><li>de Duve, C. (1971). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19866786" target="_blank">Tissue fraction-past and present.</a> <em>J Cell Biol</em> 50(1): 20.</li><li>Gancedo, C., Flores, C. L. and Gancedo, J. M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27466281" target="_blank">The expanding landscape of moonlighting proteins in yeasts.</a> <em>Microbiol Mol Biol Rev</em> 80(3): 765-777.</li><li>Gatto, L. and Lilley, K. S. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22113085" target="_blank">MSnbase-an R/Bioconductor package for isobaric tagged mass spectrometry data visualization, processing and quantitation.</a> <em>Bioinformatics</em> 28(2): 288-289.</li><li>Gatto, L., Breckels, L. M., Wieczorek, S., Burger, T. and Lilley, K. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24413670" target="_blank">Mass-spectrometry-based spatial proteomics data analysis using pRoloc and pRolocdata.</a> <em>Bioinformatics</em> 30(9): 1322-1324.</li><li>Gatto, L., Breckels, L. M. and Lilley, K. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30711721" target="_blank">Assessing sub-cellular resolution in spatial proteomics experiments.</a> <em>Curr Opin Chem Biol</em> 48: 123-149.</li><li>Groen, A. J., Sancho-Andres, G., Breckels, L. M., Gatto, L., Aniento, F. and Lilley, K. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24344820" target="_blank">Identification of trans-golgi network proteins in <em>Arabidopsis thaliana</em> root tissue.</a> <em>J Proteome Res </em>13(2): 763-776.</li><li>Gentleman, R. C., Carey, V. J., Bates, D. M., Bolstad, B., Dettling, M., Dudoit, S., Ellis, B., Gautier, L., Ge, Y., Gentry, J., Hornik, K., Hothorn, T., Huber, W., Iacus, S., Irizarry, R., Leisch, F., Li, C., Maechler, M., Rossini, A. J., Sawitzki, G., Smith, C., Smyth, G., Tierney, L., Yang, J. Y. and Zhang, J. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15461798" target="_blank">Bioconductor: open software development for computational biology and bioinformatics.</a> <em>Genome Biol</em> 5(10): R80.</li><li>Geladaki, A., Kocevar Britovsek, N., Breckels, L. M., Smith, T. S., Vennard, O. L., Mulvey, C. M., Crook, O. M., Gatto, L. and Lilley, K. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30659192" target="_blank">Combining LOPIT with differential ultracentrifugation for high-resolution spatial proteomics.</a> <em>Nat Commun</em> 10(1): 331.</li><li>Hope, I. A. and Struhl, K. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3530496" target="_blank">Functional dissection of a eukaryotic transcriptional activator protein, GCN4 of yeast.</a> <em>Cell</em> 46(6): 885-894.</li><li>Huh, W. K., Falvo, J. V., Gerke, L. C., Carroll, A. S., Howson, R. W., Weissman, J. S. and O'Shea, E. K. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14562095" target="_blank">Global analysis of protein localization in budding yeast.</a> <em>Nature</em> 425(6959): 686-691.</li><li>Hung, V., Lam, S. S., Udeshi, N. D., Svinkina, T., Guzman, G., Mootha, V. K., Carr, S. A. and Ting, A. Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28441135" target="_blank">Proteomic mapping of cytosol-facing outer mitochondrial and ER membranes in living human cells by proximity biotinylation.</a> <em>Elife</em> 6: e24463.</li><li>Hung, V., Zou, P., Rhee, H. W., Udeshi, N. D., Cracan, V., Svinkina, T., Carr, S. A., Mootha, V. K. and Ting, A. Y. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25002142" target="_blank">Proteomic mapping of the human mitochondrial intermembrane space in live cells via ratiometric APEX tagging.</a> <em>Mol Cell</em> 55(2): 332-341.</li><li>Hunter, M. J. and Commerford, S. L. (1961). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13716896" target="_blank">Pressure homogenization of mammalian tissues.</a> <em>Biochim Biophys Acta</em> 47: 580-586.</li><li>Hwang, J. and Espenshade, P. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27274088" target="_blank">Proximity-dependent biotin labelling in yeast using the engineered ascorbate peroxidase APEX2.</a> <em>Biochem J</em> 473(16): 2463-2469.</li><li>Itzhak, D. N., Tyanova, S., Cox, J. and Borner, G. H. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27278775" target="_blank">Global, quantitative and dynamic mapping of protein subcellular localization.</a> <em>Elife</em> 5: e16950.</li><li>Jadot, M., Boonen, M., Thirion, J., Wang, N., Xing, J., Zhao, C., Tannous, A., Qian, M., Zheng, H., Everett, J. K., Moore, D. F., Sleat, D. E. and Lobel, P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27923875" target="_blank">Accounting for protein subcellular localization: A compartmental map of the rat liver proteome.</a> <em>Mol Cell Proteomics</em> 16(2): 194-212.</li><li>Jean Beltran, P. M., Mathias, R. A. and Cristea, I. M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27641956" target="_blank">A portrait of the human organelle proteome in space and time during cytomegalovirus infection.</a> <em>Cell Syst</em> 3(4): 361-373 e366.</li><li>Kim, D. I., Birendra, K. C., Zhu, W., Motamedchaboki, K., Doye, V. and Roux, K. J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24927568" target="_blank">Probing nuclear pore complex architecture with proximity-dependent biotinylation.</a> <em>Proc Natl Acad Sci U S A </em>111(24): E2453-2461.</li><li>Kizer, K. O., Xiao, T. and Strahl, B. D. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17101440" target="_blank">Accelerated nuclei preparation and methods for analysis of histone modifications in yeast.</a> <em>Methods</em> 40(4): 296-302.</li><li>Lam, S. S., Martell, J. D., Kamer, K. J., Deerinck, T. J., Ellisman, M. H., Mootha, V. K. and Ting, A. Y. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25419960" target="_blank">Directed evolution of APEX2 for electron microscopy and proximity labeling.</a> <em>Nat Methods</em> 12(1): 51-54.</li><li>McAlister, G. C., Huttlin, E. L., Haas, W., Ting, L., Jedrychowski, M. P., Rogers, J. C., Kuhn, K., Pike, I., Grothe, R. A., Blethrow, J. D. and Gygi, S. P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22880955" target="_blank">Increasing the multiplexing capacity of TMTs using reporter ion isotopologues with isobaric masses.</a> <em>Anal Chem</em> 84(17): 7469-7478.</li><li>McAlister, G. C., Nusinow, D. P., Jedrychowski, M. P., Wuhr, M., Huttlin, E. L., Erickson, B. K., Rad, R., Haas, W. and Gygi, S. P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24927332" target="_blank">MultiNotch MS3 enables accurate, sensitive, and multiplexed detection of differential expression across cancer cell line proteomes.</a> <em>Anal Chem</em> 86(14): 7150-7158.</li><li>Morgenstern, M., Stiller, S. B., Lubbert, P., Peikert, C. D., Dannenmaier, S., Drepper, F., Weill, U., Hoss, P., Feuerstein, R., Gebert, M., Bohnert, M., van der Laan, M., Schuldiner, M., Schutze, C., Oeljeklaus, S., Pfanner, N., Wiedemann, N. and Warscheid, B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28658629" target="_blank">Definition of a high-confidence mitochondrial proteome at quantitative scale.</a> <em>Cell Rep</em> 19(13): 2836-2852.</li><li>Mulvey, C. M., Breckels, L. M., Geladaki, A., Britovsek, N. K., Nightingale, D. J. H., Christoforou, A., Elzek, M., Deery, M. J., Gatto, L. and Lilley, K. S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28471460" target="_blank">Using hyperLOPIT to perform high-resolution mapping of the spatial proteome.</a> <em>Nat Protoc</em> 12(6): 1110-1135.</li><li>Nikolaev, Y., Deillon, C., Hoffmann, S. R., Bigler, L., Friess, S., Zenobi, R., Pervushin, K., Hunziker, P. and Gutte, B. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20505831" target="_blank">The leucine zipper domains of the transcription factors GCN4 and c-Jun have ribonuclease activity.</a> <em>PLoS One</em> 5(5): e10765.</li><li>Nightingale, D. J., Geladaki, A., Breckels, L. M., Oliver, S. G. and Lilley, K. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30503867" target="_blank">The subcellular organisation of <em>Saccharomyces cerevisiae</em>.</a> <em>Curr Opin Chem Biol</em> 48: 86-95.</li><li>Nikolovski, N., Rubtsov, D., Segura, M. P., Miles, G. P., Stevens, T. J., Dunkley, T. P., Munro, S., Lilley, K. S. and Dupree, P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22923678" target="_blank">Putative glycosyltransferases and other plant Golgi apparatus proteins are revealed by LOPIT proteomics.</a> <em>Plant Physiol</em> 160(2): 1037-1051.</li><li>Nightingale, D. J., Oliver, S. G. and Lilley, K. S. (2018). Mapping the <em>Saccharomyces cerevisiae</em> spatial proteome with high resolution using hyperLOPIT. In: <em>Methods in Molecular Biology</em>. In: Oliver. S. G. (Ed.). Humana Press.</li><li>Opitz, N., Schmitt, K., Hofer-Pretz, V., Neumann, B., Krebber, H., Braus, G. H. and Valerius, O. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28982715" target="_blank">Capturing the Asc1p/Receptor for Activated C Kinase 1 (RACK1) Microenvironment at the head region of the 40S ribosome with quantitative BioID in yeast.</a> <em>Mol Cell Proteomics</em> 16(12): 2199-2218.</li><li>Stadler, C., Rexhepaj, E., Singan, V. R., Murphy, R. F., Pepperkok, R., Uhlen, M., Simpson, J. C. and Lundberg, E. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23435261" target="_blank">Immunofluorescence and fluorescent-protein tagging show high correlation for protein localization in mammalian cells.</a> <em>Nat Methods </em>10(4): 315-323.</li><li>Sickmann, A., Reinders, J., Wagner, Y., Joppich, C., Zahedi, R., Meyer, H. E., Schonfisch, B., Perschil, I., Chacinska, A., Guiard, B., Rehling, P., Pfanner, N. and Meisinger, C. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14576278" target="_blank">The proteome of <em>Saccharomyces cerevisiae</em> mitochondria.</a> <em>Proc Natl Acad Sci U S A</em> 100(23): 13207-13212.</li><li>Reinders, J., Zahedi, R. P., Pfanner, N., Meisinger, C. and Sickmann, A. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16823961" target="_blank">Toward the complete yeast mitochondrial proteome: multidimensional separation techniques for mitochondrial proteomics.</a> <em>J Proteome Res</em> 5(7): 1543-1554.</li><li>Rhee, H. W., Zou, P., Udeshi, N. D., Martell, J. D., Mootha, V. K., Carr, S. A. and Ting, A. Y. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23371551" target="_blank">Proteomic mapping of mitochondria in living cells via spatially restricted enzymatic tagging.</a> <em>Science</em> 339(6125): 1328-1331.</li><li>Roux, K. J., Kim, D. I., Raida, M. and Burke, B. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22412018" target="_blank">A promiscuous biotin ligase fusion protein identifies proximal and interacting proteins in mammalian cells.</a> <em>J Cell Biol</em> 196(6): 801-810.</li><li>Ross, P. L., Huang, Y. N., Marchese, J. N., Williamson, B., Parker, K., Hattan, S., Khainovski, N., Pillai, S., Dey, S., Daniels, S., Purkayastha, S., Juhasz, P., Martin, S., Bartlet-Jones, M., He, F., Jacobson, A. and Pappin, D. J. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15385600" target="_blank">Multiplexed protein quantitation in <em>Saccharomyces cerevisiae</em> using amine-reactive isobaric tagging reagents.</a> <em>Mol Cell Proteomics</em> 3(12): 1154-1169.</li><li>R Core Team. (2018). <a href="https://www.scirp.org/(S(lz5mqp453edsnp55rrgjct55))/reference/ReferencesPapers.aspx?ReferenceID=2342186" target="_blank">R: A language and environment for statistical computing.</a> R Foundation for Statistical Computing, Vienna, Austria.</li><li>RStudio Team. (2016). RStudio: Integrated Development Environment for R.</li><li>Sadowski, P. G., Dunkley, T. P., Shadforth, I. P., Dupree, P., Bessant, C., Griffin, J. L. and Lilley, K. S. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17487160" target="_blank">Quantitative proteomic approach to study subcellular localization of membrane proteins.</a> <em>Nat Protoc </em>1(4): 1778-1789.</li><li>Simpson, R. J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21041386" target="_blank">Disruption of cultured cells by nitrogen cavitation.</a> <em>Cold Spring Harb Protoc</em> 2010(11): pdb prot5513.</li><li>Thompson, A., Schafer, J., Kuhn, K., Kienle, S., Schwarz, J., Schmidt, G., Neumann, T., Johnstone, R., Mohammed, A. K. and Hamon, C. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12713048" target="_blank">Tandem mass tags: a novel quantification strategy for comparative analysis of complex protein mixtures by MS/MS.</a> <em>Anal Chem</em> 75(8): 1895-1904.</li><li>Thul, P. J., Akesson, L., Wiking, M., Mahdessian, D., Geladaki, A., Ait Blal, H., Alm, T., Asplund, A., Bjork, L., Breckels, L. M., Backstrom, A., Danielsson, F., Fagerberg, L., Fall, J., Gatto, L., Gnann, C., Hober, S., Hjelmare, M., Johansson, F., Lee, S., Lindskog, C., Mulder, J., Mulvey, C. M., Nilsson, P., Oksvold, P., Rockberg, J., Schutten, R., Schwenk, J. M., Sivertsson, A., Sjostedt, E., Skogs, M., Stadler, C., Sullivan, D. P., Tegel, H., Winsnes, C., Zhang, C., Zwahlen, M., Mardinoglu, A., Ponten, F., von Feilitzen, K., Lilley, K. S., Uhlen, M. and Lundberg, E. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28495876" target="_blank">A subcellular map of the human proteome.</a> <em>Science</em> 356(6340).</li><li>The Gene Ontology Consortium. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27899567" target="_blank">Expansion of the Gene Ontology knowledgebase and resources.</a> <em>Nucleic Acids Res</em> 45(D1): D331-D338.</li><li>Ting, L., Rad, R., Gygi, S. P. and Haas, W. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21963607" target="_blank">MS3 eliminates ratio distortion in isobaric multiplexed quantitative proteomics.</a> <em>Nat Methods</em> 8(11): 937-940.</li><li>Tkach, J. M., Yimit, A., Lee, A. Y., Riffle, M., Costanzo, M., Jaschob, D., Hendry, J. A., Ou, J., Moffat, J., Boone, C., Davis, T. N., Nislow, C. and Brown, G. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22842922" target="_blank">Dissecting DNA damage response pathways by analysing protein localization and abundance changes during DNA replication stress.</a> <em>Nat Cell Biol</em> 14(9): 966-976.</li><li>Van Der Maaten, L. and Hinton, G. (2008). <a href="https://www.jmlr.org/papers/v9/vandermaaten08a.html" target="_blank">Visualizing Data using t-SNE.</a> <em>J Mach Learn Res</em> 9: 2579-2605.</li><li>Vapnik, V. (1995). <a href="https://www.springer.com/br/book/9780387987804" target="_blank">The nature of statistical learning theory.</a> <em>Springer</em>.</li><li>Villen, J. and Gygi, S. P. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18833199" target="_blank">The SCX/IMAC enrichment approach for global phosphorylation analysis by mass spectrometry.</a> <em>Nat Protoc</em> 3(10): 1630-1638.</li><li>Vogtle, F. N., Burkhart, J. M., Rao, S., Gerbeth, C., Hinrichs, J., Martinou, J. C., Chacinska, A., Sickmann, A., Zahedi, R. P. and Meisinger, C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22984289" target="_blank">Intermembrane space proteome of yeast mitochondria.</a> <em>Mol Cell Proteomics</em> 11(12): 1840-1852.</li><li>Vogtle, F. N., Burkhart, J. M., Gonczarowska-Jorge, H., Kucukkose, C., Taskin, A. A., Kopczynski, D., Ahrends, R., Mossmann, D., Sickmann, A., Zahedi, R. P. and Meisinger, C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28819139" target="_blank">Landscape of submitochondrial protein distribution.</a> <em>Nat Commun</em> 8(1): 290.</li><li>Wang, Y., Lilley, K. S. and Oliver, S. G. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24510422" target="_blank">A protocol for the subcellular fractionation of <em>Saccharomyces cerevisiae</em> using nitrogen cavitation and density gradient centrifugation. </a><em>Yeast</em> 31(4): 127-135.</li><li>Weill, U., Yofe, I., Sass, E., Stynen, B., Davidi, D., Natarajan, J., Ben-Menachem, R., Avihou, Z., Goldman, O., Harpaz, N., Chuartzman, S., Kniazev, K., Knoblach, B., Laborenz, J., Boos, F., Kowarzyk, J., Ben-Dor, S., Zalckvar, E., Herrmann, J. M., Rachubinski, R. A., Pines, O., Rapaport, D., Michnick, S. W., Levy, E. D. and Schuldiner, M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29988094" target="_blank">Genome-wide SWAp-Tag yeast libraries for proteome exploration.</a> <em>Nat Methods</em> 15(8): 617-622.</li><li>Wiederhold, E., Veenhoff, L. M., Poolman, B. and Slotboom, D. J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19955081" target="_blank">Proteomics of <em>Saccharomyces cerevisiae</em> organelles.</a> <em>Mol Cell Proteomics</em> 9(3): 431-445.</li><li>Wenger, C. D., Lee, M. V., Hebert, A. S., McAlister, G. C., Phanstiel, D. H., Westphall, M. S. and Coon, J. J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21963608" target="_blank">Gas-phase purification enables accurate, multiplexed proteome quantification with isobaric tagging.</a> <em>Nat Methods</em> 8(11): 933-935.</li><li>Yofe, I., Weill, U., Meurer, M., Chuartzman, S., Zalckvar, E., Goldman, O., Ben-Dor, S., Schutze, C., Wiedemann, N., Knop, M., Khmelinskii, A. and Schuldiner, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26928762" target="_blank">One library to make them all: streamlining the creation of yeast libraries via a SWAp-Tag strategy.</a> <em>Nat Methods</em> 13(4): 371-378.</li><li>Zahedi, R. P., Sickmann, A., Boehm, A. M., Winkler, C., Zufall, N., Schonfisch, B., Guiard, B., Pfanner, N. and Meisinger, C. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16407407" target="_blank">Proteomic analysis of the yeast mitochondrial outer membrane reveals accumulation of a subclass of preproteins.</a> <em>Mol Biol Cell</em> 17(3): 1436-1450.</li><li>Zhang, R., Sioma, C. S., Wang, S. and Regnier, F. E. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11721911" target="_blank">Fractionation of isotopically labeled peptides in quantitative proteomics.</a> <em>Anal Chem</em> 73(21): 5142-5149.<br/></li></ol>