<body>
            <h1>Total RNA Isolation from Separately Established Monolayer and Hydrogel Cultures of Human Glioblastoma Cell Line</h1>
            <p><strong>Authors:</strong> Manasi P Jogalekar, Elba E Serrano</p>
            <h2>Abstract</h2>
            <p>Astrocytoma is an invasive carcinoma occurring in the nervous system and currently lacks effective treatment options. A deeper understanding of the mechanisms of tumorigenesis and tumor progression is needed in order to develop novel therapeutic strategies. Recent advances in <em>in vitro</em> culture systems have demonstrated that the use of three-dimensional (3D) culture models could be more relevant for this purpose as compared to monolayer or two-dimensional (2D) models due to their resemblance to <em>in vivo</em> cancer pathology. High-throughput techniques such as RNA sequencing, microarray analyses and cloning could provide useful insights into the relevance of these systems to the native tissue. Previous studies have reported RNA extraction protocols needed for such applications. We have modified these protocols to suit the isolation of total RNA from monolayer and hydrogel cultures of astrocytoma established using basement membrane matrix, Geltrex<sup>TM</sup>. We have used this method to demonstrate the differences in the expression of genes involved in autophagy, a process deregulated in many cancer types, in monolayer and hydrogel cultures using quantitative polymerase chain reaction (qPCR). This protocol can be adopted by the researchers who wish to understand the molecular basis of gene expression in hydrogel cultures of normal as well as cancer cell lines.</p>
            <p><strong>Keywords:</strong> RNA, Monolayer, 3D, Hydrogel, Glioblastoma, Cancer, TRIzol, Cell recovery solution, Astrocytoma</p>
            <h2>Background</h2><p>Astrocytomas are central nervous system tumors of glial origin. Poor survival rates among patients within 5 years of diagnosis and almost inevitable relapse are the hallmarks of this life-threatening disease (Krex <em>et al.</em>, 2007). Our understanding of disease pathogenesis is limited, resulting in the lack of effective intervention strategies. <em>In vitro</em> culture systems could prove useful for studying cell-cell interactions and intracellular signaling events in astrocytoma, and provide platforms for testing potential therapeutic compounds (Reardon <em>et al.</em>, 2006; Baker and Chen, 2012). Three-dimensional (3D) <em>in vitro</em> culture platforms have especially gained popularity in recent years due to their biological relevance to <em>in vivo</em> tumors (Li <em>et al.</em>, 2007; Whiteside, 2008; Tibbitt and Anseth, 2009; Xu <em>et al.</em>, 2014; Jogalekar and Serrano, 2018). 3D cultures established using various tissue engineering scaffolds (<em>e.g.</em>, hydrogels) allow for multi-layered growth of cells in a fashion similar to that of native tissue, unlike two-dimensional (2D) cultures, making them better predictors of responses to drug candidates. <br/><br/>Hydrogel culture systems could be exploited in parallel with monolayer cultures to further our understanding of cell-cell and cell-matrix interactions with high-throughput applications that analyze cellular gene expression, such as RNA-sequencing, microarray analysis and cloning. To accomplish this, a pure, intact, high quality RNA needs to be isolated in parallel from separately cultured monolayer and hydrogel cultures so that both cultures are processed under identical conditions and are therefore comparable. Several protocols have been reported for extracting RNA from 3D cultures established using a variety of commonly used tissue engineering scaffolds such as Matrigel and collagen (Lee <em>et al.</em>, 2007; Mroue and Bissell, 2013). However, there is a paucity of information available with respect to RNA extraction from 3D cultures maintained in Geltrex<sup>TM</sup>, a relatively new basement membrane matrix derived from murine Engelbreth-Holm-Swarm tumors and manufactured by Invitrogen. <br/><br/>When we used existing protocols to carry out RNA isolation from multiple cell lines including astrocytoma, we found that our astrocytoma cell line (CCF-STTG1) was highly sensitive to the reagents that dissolve extracellular matrices used in 3D cultures and could not yield an intact RNA at the end of the procedure, although this procedure worked well with other mammalian and amphibian cell lines we used in our experiment. Therefore, we modified RNA isolation protocols available in the literature to extract total RNA from monolayer and hydrogel cultures of astrocytoma. We used this RNA to carry out gene expression analyses for key autophagy pathway genes, Beclin-1 (<em>BECN1</em>) and microtubule associated protein 1 light chain 3 beta (<em>MAP1LC3B</em>), using quantitative polymerase chain reaction (qPCR) (Jogalekar <em>et al.</em>, 2017). Our findings indicate a modest upregulation of these genes in hydrogel cultures of astrocytoma and are consistent with previous reports suggesting their upregulation in correlation with poor prognosis in glioma patients (Pirtoli <em>et al.</em>, 2009; Giatromanolaki <em>et al.</em>, 2014). </p>
        </body>

<p class="pptt" id="biaoti28478">## Materials and Reagents</p><ol class="npwtwok"><li>0.8 Î¼m 150 ml filter unit (Thermo Scientific Nalgene, catalog number: 125-0080)</li><li>Nunc<sup>TM</sup> Cell Culture Treated Flasks with Filter Caps (Nunc<sup>TM</sup>, catalog number: 136196)</li><li>Falcon<sup>Â®</sup> Cell Scrapers, Sterile (Corning<sup>Â®</sup>, catalog number: 353085)</li><li>Falcon<sup>Â®</sup> 50 ml High Clarity PP Centrifuge Tubes (Corning<sup>Â®</sup>, catalog number: 352098) </li><li>RNase-free Microfuge Tubes (1.5 ml) (Invitrogen<sup>TM</sup>, catalog number: AM12400)</li><li>5cc BD Luer-Lok<sup>TM</sup> Disposable Syringe (BD, catalog number: 309603)</li><li>21 G 1Â¼ needle (BD, catalog number: 305166)</li><li>10 ml serological pipette, sterile (USA Scientific, catalog number: 1071-0810)</li><li>Fisherbrand<sup>TM</sup> SureOne<sup>TM</sup> Aerosol Barrier Pipette Tips (Fisherbrand<sup>TM</sup> SureOne<sup>TM</sup>, catalog numbers: 02-707-404 [1,000 Î¼l], 02-707-430 [200 Î¼l], 02-707-432 [20 Î¼l])</li><li>CCF-STTG1 cell line (ATCC, catalog number: CRL-1718)</li><li>RPMI-1640 Medium (ATCC, catalog number: 30-2001)</li><li>Fetal Bovine Serum (FBS) (ATCC, catalog number: 30-2020)</li><li>LDEV-Free Reduced Growth Factor Basement Membrane Matrix, Geltrex<sup>TM</sup> (Invitrogen, catalog number: A1413202)</li><li>70% Ethanol</li><li>RNaseZap<sup>TM</sup> RNase Decontamination Solution (Ambion<sup>TM</sup>, catalog number: AM9780)</li><li>TRIzol<sup>TM</sup> Plus RNA Purification Kit (Ambion<sup>TM</sup>, catalog number: 12183555)</li><li>Cell Recovery Solution (Corning, catalog number: 354253)</li><li>Phosphate buffered saline (PBS) (Sigma-Aldrich, catalog number: P3813) </li><li>Chloroform (Sigma-Aldrich, catalog number: C2432)</li><li>Nuclease-Free Water (not DEPC-Treated) (Ambion<sup>TM</sup>, catalog number: AM9937)</li><li>DNA-free DNA Removal Kit (Ambion<sup>TM</sup>, catalog number: AM1906)</li><li>RNA 6000 Nano Kit (Agilent, catalog number: 5067-1511)</li><li>Ethyl alcohol, Pure (Sigma-Aldrich, catalog number: E7023)</li><li>RPMI-1640 complete medium (see Recipes)</li><li>Sterile 1x PBS (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28479">### Equipment</p><ol class="npwtwok"><li>Gilson<sup>TM</sup> PIPETMAN Classic<sup>TM</sup> Pipettes (Gilson, catalog numbers: F123600 [P20], F123601 [P200], F123602 [P1000], F144802 [P10], F144801 [P2])</li><li>2100 bioanalyzer instrument (Agilent, catalog number: G2939BA)</li><li>Drummond Pipet-Aid, Plain, 110V (Drummond, catalog number: DP-110)</li><li>Biosafety cabinet (Nuaire, catalog number: NU-425-600)</li><li>4 Â°C centrifuge (Beckman Coulter, catalog number: Allegra<sup>TM</sup> 21R) </li><li>Centrifuge rotors</li><li>Water bath at 37 Â°C (Sheldon Manufacturing Inc., catalog number: 1211)</li><li>AutoFlow Water Jacket CO<sub>2</sub> Incubator (Nuaire, catalog number: NU-4750) </li><li>-20 Â°C freezer <br/></li></ol><p class="pptt" id="biaoti28480">### Software</p><ol class="npwtwok"><li>2100 Expert (Agilent)</li><li>Photoshop CS6 (Adobe)<br/></li></ol><p class="pptt" id="biaoti28481">### Procedure</p><ol class="npwone"><li>Cell culture<ol class="npwtwo"><li>Establish 2D and 3D cultures separately, of CCF-STTG1 cells using RPMI-1640 complete medium. For 3D culture, coat T25 flasks with 100 Âµl per cm<sup>2</sup> Geltrex<sup>TM</sup> and allow it to solidify in a 37Â°C incubator for 30 min. Plate CCF-STTG1 cells on top of the solidified Geltrex<sup>TM</sup> at a density of 5 x 10<sup>4</sup> cells per cm<sup>2</sup>. For monolayer or 2D culture, plate equal number of cells directly in T25 flasks (do not coat these flasks with matrix prior to the addition of cells). Incubate the flasks at 37 Â°C and 5% CO<sub>2</sub>.</li><li>Aspirate the old media and add fresh media to culture flasks every other day. Monitor the cell growth periodically.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA isolation<br/><em>Note: Make sure that all the supplies used for the RNA isolation such as centrifuge tubes, pipette tips, plastic pipettes and microcentrifuge tubes are sterile, and DNase- and RNase-free.</em> <ol class="npwtwo"><li>Begin RNA isolation when the cells in 2D and 3D cultures are ~80-90% confluent (Figure 1). This usually takes about a week, although it may vary with the cell and culture type. Process both culture flasks on the same day in parallel.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190715/20190715002009_0241.jpg" width="267"/><br/><strong>Figure 1. Phase contrast images showing 80-90% confluent CCF-STTG1 cells in monolayer and hydrogel cultures.</strong> CCF-STTG1 cells were allowed to grow in A) Monolayer and B) Hydrogel cultures until they reached 80-90% confluency before beginning RNA extraction procedure.<br/><br/></li><li>Treat all surfaces with ethanol followed by RNase ZAP prior to UV-including both centrifuge rotors, pipettes, and bench surfaces outside the tissue culture room.</li><li>Turn on the centrifuge so it cools to 4 Â°C.</li><li>Turn on and set the water bath at 37 Â°C.</li><li>UV-sterilize the tissue culture hood and the required materials for 30 min before beginning the RNA extraction procedure. Also, have cold 1x PBS ready (for use with cell recovery solution). </li><li>Without removing the media, detach cells from the bottom of the 2D and 3D culture flasks using cell scrapers and transfer them to 50 ml centrifuge tubes. Centrifuge for 5 min at 200 <em>x g</em> at 4 Â°C. Trypsin was not used in this procedure since it alters gene expression (Chaudhry, 2008). <br/><em>Note: Cells can be lysed with TRIzol</em><sup><em>Â®</em></sup><em> reagent in a culture dish or flask directly if the dishes are made of materials that are compatible with TRIzol</em><sup><em>Â®</em></sup><em> reagent. Please check the product details and their compatibility before doing so. </em> </li><li>Decant the supernatant from the tube containing 3D cells. Please make sure to leave equal volume of media in the other tube containing monolayer cell pellet, since these tubes will be processed in parallel under identical conditions.</li><li>Separate the astrocytoma cells from the basement membrane matrix with one of the two waysâusing TRIzol or cell recovery solution. The method of choice is determined by analyzing the sensitivity of cell lines of interest to both matrix-dissolving reagents (by trial and error method). We found that the cell recovery solution worked well with some cell lines (<em>e.g.</em>, HCC70 cell line, Figure 2B), while others were found to be sensitive and died upon exposure (<em>e.g.</em>, CCF-STTG1 cell line, Figure 2A). Therefore, we decided to use TRIzol for these cell lines. Since 2D and 3D cultures must be processed identically, 2D or monolayer cultures were also exposed to equal volumes of TRIzol or cell recovery solution although they do not contain matrix.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190715/20190715002141_0529.jpg" width="144"/><br/><strong>Figure 2. Electropherograms showing the quality of total RNA extracted from monolayer and hydrogel cultures.</strong> The total RNA was extracted from monolayer and hydrogel cultures of astrocytoma cell line, CCF-STTG1. One microliter of total RNA per sample was analyzed using Agilent Bioanalyzer. A. RNA isolated from CCF-STTG1 cells using TRIzol (Top panel) and cell recovery solution (Bottom panel). B. RNA extracted from HCC70 (breast cancer cell line) cells used as control to show that cell recovery solution works well for HCC70 cells, but not for CCF-STTG1 cells. a: 18s peak, b: 28s peak.<br/><br/><ol class="npwthree"><li>Using TRIzol reagent:<ol class="npwfour"><li>Add TRIzol<sup>Â®</sup> reagent to both tubes so that the sample volume is 10% of the total volume and incubate the samples for 5 min. For example, if the sample volume is 1 ml, then 9 ml of TRIzol should be added. </li><li>Lyse cells by passing them through 21 G x 1Â¼" needle and 5cc syringe at least 5-6 times.</li><li>After cell lysis, add 0.2 ml of chloroform per ml of TRIzol<sup>Â®</sup> reagent, invert the tubes to mix the contents for 15 s followed by the incubation for 2-3 min at room temperature. </li><li>Centrifuge at 15,000 <em>x g</em> for 15 min at 4 Â°C.</li><li>Transfer colorless upper phase to the RNase-free tube. The volume of colorless upper phase varies depending upon the volumes of sample and TRIzol/chloroform added in previous steps. For example, in our case, the volume of matrix containing cells was 2.8 ml, so we added 2.8 x 9 ml = 25.2 ml of TRIzol plus 0.2 x 25.2 ml = 5.04 ml of chloroform. So the total volume of the solution was ~33 ml. Out of that, the volume of colorless upper phase came out to be ~50%, <em>i.e.</em>, 18 ml (Figure 3).</li><li>Add 70% ethanol (equal volume) to a final concentration of 35%. Use absolute or molecular biology grade alcohol to make 70% ethanol to reduce the chances of contamination.</li><li>Mix well by inverting the tube to disperse precipitate.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190715/20190715002207_6998.jpg" width="124"/><br/><strong>Figure 3. Colorless upper phase separated after centrifugation step.</strong> Followed by incubation with TRIzol and chloroform, the monolayer and hydrogel samples of CCF-STTG1 were centrifuged for 15 min at 15,000 <em>x g</em> at 4 Â°C to obtain colorless upper phase which contains RNA.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Using cell recovery solution:<ol class="npwfour"><li>Wash cells thrice with cold 1x PBS and add equal amounts of cell recovery solution (227Âµl per cm<sup>2</sup>) directly to T25 flasks. </li><li>Scrape the cells using cell scrapers and transfer to a cold 50 ml centrifuge tube. </li><li>Incubate on ice till Geltrex<sup>TM</sup> is dissolved (up to 1 h, varies for each cell line). Mix the suspension by inverting the tubes every 15 min.<br/><em>Note: Please pay attention to exposure times for cell recovery solution used to dissolve Geltrex</em><sup><em>TM</em></sup><em>. Some cell lines may be more sensitive to this reagent than others. Optimize treatment times for your cell line before conducting a major experiment.</em> </li><li>When the cells settle down at the bottom of the tube, centrifuge the mixture at 200 <em>x g</em> at 4 Â°C for 5 min.</li><li>Wash cells in cold 1x PBS twice. </li></ol></li></ol></li><li>Proceed with RNA extraction using the manufacturerâs (Ambion) protocol for isolating total RNA from the animal cells.</li><li>Use DNA-free<sup>TM</sup> kit to remove DNA contamination and store pure RNA at -80 Â°C until use. </li><li>Check the quality of extracted RNA integrity with Agilent bioanalyzer (Figure 2).<br/><em>Note: The samples with RNA integrity number (RIN) &gt; 8 were considered pure and high quality, and used for experiments. If RNA concentration in the sample is less than 25 ng/Î¼l, Agilent software may not show RIN values (Mueller </em><em>et al.</em><em>, 2004). In that case, it is essential to determine RNA concentration using another method like Nanodrop. It is also important to confirm RNA integrity by visually inspecting electropherograms provided by Agilent Software even though RIN values are not displayed.</em> </li></ol></li></ol><p class="pptt" id="biaoti28482">### Data analysis</p><ol class="npwtwok"><li>Determine the RNA quality using Agilent bioanalyzer 2100 and RNA 6000 Nano Kit. </li><li>Load RNA samples on the chip using the manufacturerâs (Agilent) instructions and read the chip using 2100 Bioanalyzer Expert Software (Figure 2). <br/></li></ol><p class="pptt" id="biaoti28483">### Recipes</p><ol class="npwtwok"><li>RPMI-1640 complete medium <br/>RPMI 1640 basal medium<br/>10% FBS<ol class="npwthree"><li>Mix both ingredients and filter sterilize the medium through a 0.8 Î¼m filter unit <br/><em>Note: We do not use antibiotics for cell culture in our laboratory but you can add them to the medium at the appropriate concentration (usually 1% of the total volume) depending upon your requirements.</em> </li><li>Store at 4 Â°C and use within a month</li></ol></li><li>Sterile 1x PBS<br/><ol class="npwthree"><li>Dissolve the powdered contents from one pouch of PBS to 1,000 ml of distilled water to get 1x PBS, pH 7.4</li><li>Autoclave the solution and store at room temperature</li></ol></li></ol><p class="pptt" id="biaoti28484">### Acknowledgments</p><p class="ppzz">We are grateful for the funding support provided by the New Mexico State University Manasse Chair Endowment to E. E. Serrano, and the National Institutes of Health (P50GM68762) grant. We thank V. B. Knight for providing technical assistance for this work. This protocol was modified from previously published RNA extraction methods (Lee <em>et al.</em>, 2007; Mroue and Bissell, 2013).</p><p class="pptt" id="biaoti28485">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti28486">### References</p><ol class="npwtwok"><li>Baker, B. M. and Chen, C. S. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22797912" target="_blank">Deconstructing the third dimension: how 3D culture microenvironments alter cellular cues.</a> <em>J Cell Sci</em> 125(Pt 13): 3015-3024.</li><li>Chaudhry, A. M. (2008). <a href="https://www.researchgate.net/publication/46031311_Induction_of_Gene_Expression_Alterations_by_Culture_Medium_from_Trypsinized_Cells" target="_blank">Induction of gene expression alterations by culture medium from trypsinized cells.</a> <em>J Biol Sci</em> 8(1): 81-87.</li><li>Giatromanolaki, A., Sivridis, E., Mitrakas, A., Kalamida, D., Zois, C. E., Haider, S., Piperidou, C., Pappa, A., Gatter, K. C., Harris, A. L. and Koukourakis, M. I. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25482944" target="_blank">Autophagy and lysosomal related protein expression patterns in human glioblastoma.</a> <em>Cancer Biol Ther</em> 15(11): 1468-1478.</li><li>Jogalekar, M. P., Cooper, L. G. and Serrano, E. E. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28589519" target="_blank">Hydrogel environment supports cell culture expansion of a grade IV astrocytoma.</a> <em>Neurochem Res</em> 42(9): 2610-2624.</li><li>Jogalekar, M. P. and Serrano, E. E. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29473000" target="_blank">Morphometric analysis of a triple negative breast cancer cell line in hydrogel and monolayer culture environments.</a> <em>PeerJ</em> 6: e4340.</li><li>Krex, D., Klink, B., Hartmann, C., von Deimling, A., Pietsch, T., Simon, M., Sabel, M., Steinbach, J. P., Heese, O., Reifenberger, G., Weller, M., Schackert, G. and German Glioma, N. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17785346" target="_blank">Long-term survival with glioblastoma multiforme.</a> <em>Brain</em> 130(Pt 10): 2596-2606.</li><li>Lee, G. Y., Kenny, P. A., Lee, E. H. and Bissell, M. J. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17396127" target="_blank">Three-dimensional culture models of normal and malignant breast epithelial cells.</a> <em>Nat Methods</em> 4(4): 359-365.</li><li>Li, H., Fan, X. and Houghton, J. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17226777" target="_blank">Tumor microenvironment: the role of the tumor stroma in cancer.</a> <em>J Cell Biochem</em> 101(4): 805-815.</li><li>Mroue, R. and Bissell, M. J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23097110" target="_blank">Three-dimensional cultures of mouse mammary epithelial cells.</a> <em>Methods Mol Biol</em> 945: 221-250.</li><li>Mueller, O., Lightfoot, S. and Schroeder, A. (2004). <a href="https://www.researchgate.net/publication/284654543_RNA_integrity_number_RIN-standardization_of_RNA_quality_control" target="_blank">RNA integrity number (RIN)âstandardization of RNA quality control.</a> Agilent application note: 5989-1165EN.</li><li>Pirtoli, L., Cevenini, G., Tini, P., Vannini, M., Oliveri, G., Marsili, S., Mourmouras, V., Rubino, G. and Miracco, C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19556884" target="_blank">The prognostic role of Beclin 1 protein expression in high-grade gliomas.</a> <em>Autophagy</em> 5(7): 930-936.</li><li>Reardon, D. A., Rich, J. N., Friedman, H. S. and Bigner, D. D. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16525180" target="_blank">Recent advances in the treatment of malignant astrocytoma.</a> <em>J Clin Oncol</em> 24(8): 1253-1265.</li><li>Tibbitt, M. W. and Anseth, K. S. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19472329" target="_blank">Hydrogels as extracellular matrix mimics for 3D cell culture.</a> <em>Biotechnol Bioeng</em> 103(4): 655-663.</li><li>Whiteside, T. L. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18836471" target="_blank">The tumor microenvironment and its role in promoting tumor growth.</a> <em>Oncogene</em> 27(45): 5904-5912.</li><li>Xu, X., Sabanayagam, C. R., Harrington, D. A., Farach-Carson, M. C. and Jia, X. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24447463" target="_blank">A hydrogel-based tumor model for the evaluation of nanoparticle-based cancer therapeutics.</a> <em>Biomaterials</em> 35(10): 3319-3330.<br/></li></ol>