<body>
            <h1><em>In vitro</em> RNA Cleavage Assays to Characterize IRE1-dependent RNA Decay</h1>
            <p><strong>Authors:</strong> G. Elif KaragÃ¶z, Jirka Peschek, Peter Walter, Diego Acosta-Alvear</p>
            <h2>Abstract</h2>
            <p>The kinase/RNase IRE1 is a key effector of the cellular response to endoplasmic reticulum stress. The RNase activity of IRE1 can be measured in cells or in the test tube. Here we describe a protocol for the <em>in vitro</em> cleavage and analysis of RNA substrates of IRE1. The method consists of the <em>in vitro</em> transcription, purification and re-folding of IRE1 substrate RNAs followed by their cleavage using recombinant cytosolic kinase/RNase domains of IRE1 and the separation of the resulting fragments by denaturing polyacrylamide gel electrophoresis. This protocol allows the study of the cleavage kinetics of IRE1âs RNA substrates <em>in vitro</em>.</p>
            <p><strong>Keywords:</strong> IRE1, Unfolded protein response, RNA cleavage, XBP1, Regulated IRE1-dependent decay, ER stress</p>
            <h2>Background</h2><p>Accumulation of un(mis)folded proteins in the endoplasmic reticulum (ER) causes ER stress and activates the unfolded protein response (UPR), an adaptive mechanism that maintains ER homeostasis (Reviewed in KaragÃ¶z <em>et al.</em>, 2019). IRE1, a transmembrane ER stress sensor/transducer with cytosolic kinase and RNase activities, governs the most conserved UPR signaling arm, and it is found from yeasts to metazoans (Mori, 2009). ER stress leads to IRE1 oligomerization and activation of its C-terminal cytosolic RNase domain. IRE1 preserves ER homeostasis in two ways. First, by its most studied mechanism, IRE1âs RNase domain cleaves an intron found in the <em>XBP1</em> mRNA to initiate an unconventional splicing event that activates the transcription factor <em>XBP1 </em>(Yoshida<em> et al.</em>, 2001; Calfon <em>et al</em>., 2002; Peschek<em> et al</em>., 2015). Active XBP1<sup>S</sup> (âSâ for spliced) controls the upregulation of genes that improve protein processing capacity of the ER (Lee <em>et al.,</em> 2003; Acosta-Alvear <em>et al.</em>, 2007). Second, IRE1 degrades ER-bound mRNAs in a process known as regulated IRE1-dependent decay (RIDD), to reduce the ER protein folding load or to initiate specific cytoprotective responses (Hollien and Weissman, 2006; Hollien<em> et al.</em>, 2009; Bae <em>et al.</em>, 2019). IRE1âs RNase output has been measured on a global scale through microarray-based gene expression analysis in wild-type and in IRE1-deficient cells (Hollien and Weissman, 2006; Han <em>et al</em>., 2009; Hollien <em>et al.,</em> 2009; So <em>et al.</em>, 2012). While this type of approach lends itself to the discovery of potential IRE1 substrates, it does not allow evaluating the direct engagement of IRE1 with potential substrates. Moreover, such global approaches can be costly and technically demanding. We and others have measured IRE1âs RNase output in mammalian cells using <em>XBP1</em> mRNA splicing and RIDD reporter constructs (Iwawaki<em> et al.</em>, 2004; Iwawaki and Akai, 2006; Sidrauski <em>et al.</em>, 2013; Mendez <em>et al.</em>, 2015; Moore and Hollien, 2015; Peschek <em>et al</em>., 2015), or by directly assessing the mRNA levels of endogenous spliced <em>XBP1</em> mRNA or of validated RIDD targets by TaqMan and/or RT-PCR assays (Acosta-Alvear<em> et al.</em>, 2007; Hollien <em>et al.</em>, 2009; Sidrauski <em>et al</em>., 2013; Mendez <em>et al.</em>, 2015; Peschek <em>et al.</em>, 2015). While useful, these techniques do not allow precise time-resolved measurements of RNA processing by IRE1. Precise kinetic measurements of IRE1âs RNase activity require examining RNA cleavage in the test tube. We and others have measured the kinetics of <em>XBP1</em> mRNA cleavage by IRE1 using short RNA substrates containing <em>XBP1</em> mRNA stem-loops recognized by IRE1 (Korennykh <em>et al.</em>, 2009; Lu <em>et al</em>., 2014; Mendez <em>et al</em>., 2015; Peschek <em>et al.,</em> 2015; Acosta-Alvear <em>et al.,</em> 2018). In cells, however, IRE1 is presented with cognate RNA structures embedded in full-length mRNAs. Since RNA folding is influenced by the size of the molecule and local sequence composition (Cheng <em>et al</em>., 2015), non-natural short RNA substrates containing putative IRE1 cleavage sites may contribute to false positive or false negative results in <em>in vitro</em> RNA cleavage assays. Radioactive <em>in vitro</em> cleavage assays for IRE1 RNA substrates of ~400-500 nucleotides in length have been performed (Lu <em>et al.</em>, 2014; Tam<em> et al</em>., 2014), and while undoubtedly useful, the use of radioactivity could be limiting for many laboratories. With these considerations in mind, we have developed a non-radioactive protocol for analyzing the cleavage of long RNA substrates of IRE1 <em>in vitro</em> (Figure 1). Our method can be performed in the presence and absence of purified ribosomes to account for IRE1 activity in IRE1-ribosome complexes (Acosta-Alvear<em> et al</em>., 2018), it can be deployed to study the RNase activity of IRE1 and its consequences in the context of maintaining cell viability (Lu <em>et al.</em>, 2014), and it can be useful for assessing the specificity of recently developed IRE1 inhibitors (Cross <em>et al.,</em> 2012; Harnoss <em>et al.</em>, 2018).<br/><br/><img alt="" class="layerphoto" height="200" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190714/20190714073805_9368.jpg" width="186"/><br/><strong>Figure 1. Protocol workflow </strong><br/></p>
        </body>

<p class="pptt" id="biaoti28498">## Materials and Reagents</p><ol class="npwone"><li>Purification of recombinant cytosolic domains of human IRE1<br/><ol class="npwtwo"><li>Superdex 200 10/300 GL size exclusion chromatography column (GE Healthcare, catalog number: 17517501)</li><li>50 ml screw-cap conical tubes (various suppliers)</li><li>5 ml HisTrap HP histidine-tagged protein purification column (GE Healthcare, catalog number: 17524801)</li><li>Size exclusion chromatography columns HiLoad<sup>Â®</sup> 16/600 Superdex<sup>Â®</sup> 200 (GE Healthcare, catalog number: 28989335), or similar</li><li>Amicon Ultra-15 centrifugal filters (30 kDa MW cut-off; Mfr. Millipore, catalog number: 903008)</li><li>SF21 cells (SFM adapted, Invitrogen, catalog number: 11497-013)</li><li>Sf-900<sup>TM</sup> II SFM (Gibco, Thermo Fisher Scientific, catalog number: 10902096)</li><li>Fetal bovine serum (FBS), heat inactivated (Gibco, Thermo Fisher Scientific, catalog number: 10082147)</li><li>Antibiotic-antimycotic solution (100x), stabilized (Sigma-Aldrich, catalog number: A5955)</li><li>Baculovirus expression construct encoding N-terminal hexahistidine tagged IRE1 of human origin bearing a 43 amino acid long portion of a linker domain located between the epitope tag and the kinase domain (IRE1-KR43, amino acids 521-977) as described in Li <em>et al</em>., 2010</li><li>cOmplete EDTA-free protease inhibitor cocktail tablets (Mfr. Roche. Sigma-Aldrich, catalog number: 11873580001)</li><li>TCEP-HCl (Sigma-Aldrich, catalog number: 75259)</li><li>Tobacco-etch virus (TEV) protease (recombinantly expressed and purified from <em>E. coli</em>)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375)</li><li>KCl, 1 M solution (Sigma-Aldrich, catalog number: 60142)</li><li>MgCl<sub>2</sub>, 2 M solution (Sigma-Aldrich, catalog number: 68475)</li><li>Imidazole (Sigma-Aldrich, catalog number: I5513)</li><li>SF21 cell lysis buffer (see Recipes)</li><li>IRE1 dialysis buffer (see Recipes)</li></ol><br/></li><li><em>In vitro</em> transcription, purification and refolding of IRE1 substrate RNAs<br/><ol class="npwtwo"><li>Plastic cling-wrap</li><li>Spin-X centrifuge tube filters, pore size 0.45 Î¼m (Mfr. Corning Costar. Sigma-Aldrich, catalog number: CLS8162)</li><li>Non-stick 1.5 ml RNase-free tubes (Mfr. Applied Biosystems., catalog number: AM12450)</li><li>Non-stick 0.5 ml RNase-free tubes (Mfr. Invitrogen., catalog number: AM12350)</li><li>Sterile No. 20 G syringe needles (Mfr. Beckton-Dickinson, catalog number: 305176)</li><li>pUC19 (Addgene plasmid #50005 or Thermo Scientific, catalog number: SD0061)</li><li>T7 oligonucleotide 5â²-TAATACGACTCACTATAG(N<sub>14-21</sub>)-3â²; where N corresponds to bases on the target gene of interest to amplify by PCR (Integrated DNA Technology)</li><li>Phusion high-fidelity DNA polymerase (New England Biolabs, catalog number: M0530S)</li><li>dNTPs solution (New England Biolabs, catalog number: N0447S)</li><li>RNaseZAP RNase decontamination solution (Invitrogen, catalog number: AM9780)</li><li>QIAquick PCR &amp; gel cleanup kit (QIAGEN, catalog number: 28506)</li><li>DNA clean and concentrator-5 kit (Zymo Research, catalog number: D4013)</li><li>HiScribe T7 high yield RNA synthesis kit (New England Biolabs, catalog number: E2040S)</li><li>RNase-free DNase I (New England Biolabs, catalog number: M0303S)</li><li>Novex 6% TBE-Urea PAGE gels (Invitrogen, catalog number: EC6865BOX)</li><li>Novex TBE-Urea 2x sample buffer (Invitrogen, catalog number: LC6876)</li><li>Novex TBE-Urea 5x running buffer (Invitrogen, catalog number: LC6675)</li><li>3 M NaOAc pH 5.5, RNase-free (Mfr. Invitrogen, catalog number: AM9740)</li><li>RiboRuler low range RNA ladder (Thermo Scientific, catalog number: SM1833)</li><li>80% ethanol in RNase-free water</li><li>70% ethanol</li><li>Pure isopropanol</li><li>RNase-free water (Ambion, catalog number: AM9930)</li><li>Recombinant RNasin RNase inhibitor (Promega, catalog number: N251A)</li><li>EDTA (Sigma-Aldrich, catalog number: EDS)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375)</li><li>NaCl (Sigma-Aldrich, catalog number: S3014)</li><li>MgCl<sub>2</sub>, 2 M solution (Sigma-Aldrich, catalog number: 68475)</li><li>RNA gel extraction buffer (see Recipes)</li><li>RNA re-suspension buffer (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>In vitro</em> cleavage assay of<em> in vitro</em> transcribed RNAs with purified recombinant IRE1 cytosolic domains and denaturing urea-PAGE of IRE1-generated RNA fragments<br/><ol class="npwtwo"><li>Non-stick 1.5 ml RNase-free tubes (Mfr. Applied Biosystems, catalog number: AM12450)</li><li>Non-stick 0.5 ml RNase-free tubes (Mfr. Invitrogen, catalog number: AM12350)</li><li>Purified recombinant IRE1 cytosolic kinase/RNase domains (IRE1-KR43, 50 nM to 5 ÂµM per reaction)</li><li>Purified, refolded substrate RNA (25-100 ng per reaction)</li><li>TCEP-HCl (Sigma-Aldrich, catalog number: 75259)</li><li>Proteinase K solution (Mfr. Invitrogen., catalog number: AM2546)</li><li>IRE1 inhibitor III, 4Âµ8C (CAS 14003-96-4; Calbiochem. Mfr. EMD Millipore. Sigma-Aldrich, catalog number: 212512)</li><li>Novex 6% TBE-Urea PAGE gels (Invitrogen, catalog number: EC6865BOX)</li><li>Novex TBE-Urea 2x sample buffer (Invitrogen, catalog number: LC6876)</li><li>Novex TBE-Urea 5x running buffer (Invitrogen, catalog number: LC6675)</li><li>SYBR Gold nucleic acid gel stain (Invitrogen, catalog number: S11494)</li><li>RiboRuler low range RNA ladder (Thermo Scientific, catalog number: SM1833)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375)</li><li>NaCl (Sigma-Aldrich, catalog number: S3014)</li><li>MgCl<sub>2</sub>, 2 M solution (Sigma-Aldrich, catalog number: 68475)</li><li>Glycerol (Sigma-Aldrich, catalog number: G5516)</li><li>Urea (Sigma-Aldrich, catalog number: 51456)</li><li>SDS (Sigma-Aldrich, catalog number: L3771)</li><li>EDTA (Sigma-Aldrich, catalog number: EDS)</li><li>Bromophenol blue (Sigma-Aldrich, catalog number: B8026)</li><li>Xylene cyanol FF (Sigma-Aldrich, catalog number: X4126)</li><li>2x RNA cleavage buffer (see Recipes)</li><li>Stop solution (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti28499">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>-80 Â°C freezer</li><li>Sterile No. 11 scalpels (Mfr. Graham-Field, Fisher Scientific, catalog number: 08-927-5B)</li><li>Cell homogenizer (Avestin Emulsiflex-C3, or similar)</li><li>Superspeed fixed angle rotor (SS-34, Mfr. Sorvall., Thermo Scientific, catalog number: 28020TS, or similar)</li><li>Refrigerated benchtop centrifuge (Beckman Coulter Allegra 6R, or similar) equipped with a horizontal swinging bucket rotor (Beckman GH-3.8 or similar), and rotor adaptors for 15 ml and 50 ml conical tubes </li><li>Benchtop microcentrifuge (Eppendorf 5424, or similar)</li><li>ÃKTA pure protein purification system (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 29018224 </a> ), or similar fast-performance liquid chromatography [FPLC] system)</li><li>Thermomixer C, equipped with a 1.5 ml tube block (Eppendorf, catalog number: 2231000574, or similar)</li><li>Thermocycler (BioRad C1000, S1000 or similar)</li><li>Microvolume spectrophotometer (Thermo Scientific, NanoDrop ND-2000; or similar)</li><li>XCell SureLock mini-cell electrophoresis system (Invitrogen, catalog number: EI0001)</li><li>Electrophoresis power supply (BioRad PowerPac basic power supply, catalog number: 1645050; or similar)</li><li>Gel imaging system (BioRad ChemiDoc XRS+, catalog number: 1708265; or similar)</li><li>UV transilluminator (UVP, catalog number: 95042001; or similar)<br/></li></ol><p class="pptt" id="biaoti28500">### Software</p><ol class="npwtwok"><li>ImageJ image processing and analysis software (National Institutes of Health and Laboratory of Optical and Computational Instrumentation, University of Wisconsin at Madison, Madison, WI, USA; <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>)</li></ol><p class="pptt" id="biaoti28501">### Procedure</p><ol class="npwone"><li>Purification of recombinant cytosolic domains of human IRE1<br/><ol class="npwtwo"><li>Grow SF21 cells in SF-900 II SFM media supplemented with 10% FBS and 1x antibiotic-antimycotic solution (100 U/ml penicillin, 100 Âµg/ml streptomycin, 250 ng/ml amphotericin B) at 28 Â°C in Erlenmeyer flasks, shaking at 150 rpm.</li><li>Infect SF21 cells at mid logarithmic growth phase at a density of 1 x 10<sup>6</sup>-2 x 10<sup>6</sup> cells per ml with high titer baculoviral stock.</li><li>Collect the SF21 cells 72-96 h after infection in 50 ml conical tubes and pellet them by centrifugation at 4 Â°C, at 160 <em>x</em> <em>g</em> for 5 min, using a refrigerated benchtop centrifuge (Beckman Coulter Allegra 6R or similar). At this point you can either flash freeze the cell pellet and store it at -80 Â°C, or continue with the purification of the protein.</li><li>Suspend the pellet from 250 ml of SF21 cells (around 5 ml dry volume) in 20 ml ice-cold SF21 cell lysis buffer. Keep the tube on ice.</li><li>Lyse the cells by passing them once through the homogenizer. Set the Emulsiflex homogenizer at 16,000 psi. Collect the homogenized cells in ice cooled tubes; keep the homogenates on ice.</li><li>Centrifuge the lysate at 30,500 <em>x</em> <em>g</em> for 40 min at 4 Â°C in SS-34 rotor to remove cell debris.</li><li>Equilibrate the 5 ml HisTrap HP column with SF21 cell lysis buffer using the FPLC system, which is kept in a cooling cabinet or cold room.</li><li>Load the clarified cell lysate onto the 5 ml HisTrap HP column at a flow rate of 2 ml/min to ensure efficient binding of the protein.</li><li>Wash the bound protein with 20 column volumes of lysis buffer at a flow rate of 5 ml/min.</li><li>Elute the protein with an imidazole gradient (0 to 500 mM) in 15 column volumes in the SF21 cell lysis buffer. Analyze a small sample of each collected fraction by SDS-PAGE to define the fractions in which the protein eluted. Pool the fractions containing the purified protein.</li><li>Incubate the purified IRE1-KR43 with TEV protease (250 Î¼l of 0.5 mg/ml) to remove the hexahistidine tag overnight (12-16 h) at 4 Â°C during dialysis against the IRE1 dialysis buffer.</li><li>Load the protein onto a HisTrap HP column equilibrated with the IRE1 dialysis buffer after TEV cleavage to remove impurities as well as the fraction of IRE1-KR43 still containing the hexahistidine tag.</li><li>Further purify the protein by running the HisTrap HP column eluate on a HiLoad 16/600 Superdex 200 column (GE Healthcare) which has been equilibrated with the dialysis buffer with 2 column volumes according to the manufacturerâs instructions.</li><li>Concentrate the purified IRE-KR43 protein using Amicon Ultra centrifugal filters to 50 Î¼M and flash freeze in 25 Î¼l aliquots. Store at -80 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>In vitro</em> transcription, purification and refolding of IRE1 substrate RNAs<br/><ol class="npwtwo"><li>Prepare templates for <em>in vitro</em> transcription. Assemble multiple parallel reactions to increase yield. The DNA templates can be either PCR products prepared with Phusion polymerase (a high-fidelity DNA polymerase that generates blunt ends), or linearized pUC19 plasmids cut at the 3â² end of the sequence encoding the RNA of interest. We routinely use 1 Î¼g of gel purified DNA templates for <em>in vitro</em> transcription. To account for loses during purification, start with at least 2 Î¼g of PCR products or 2 Î¼g of plasmid for digestion.<br/><em>Note: Design oligonucleotides to use as primers to amplify the sequence encoding your RNA of interest from high-quality cDNA obtained from the cell of your choice or from available cDNA clones. Use the T7 oligonucleotide 5â²-TAATACGACTCACTATAG(N<sub>14-21</sub>)-3â² as your forward primer and a gene-specific reverse primer. If you want to clone the templates, engineer restriction enzyme recognition sites compatible with the multiple cloning site of pUC19 into your oligonucleotides. Use recognition sites for enzymes that generate blunt-ends or 5â²-overhangs on the 3â²-end of your cloned DNA of interest. If you are planning on using linearized plasmids as templates for the in vitro transcription reaction, digest the plasmids to completion.</em> </li><li>Purify the templates from Step B1 above from 1% agarose gels. Use the QIAquick PCR &amp; gel cleanup kit following the manufacturerâs recommendations. Elute DNA from each column with 30 Î¼l of elution buffer.<br/><em>Note: Do not skip the gel purification step. Unincorporated oligonucleotides, off-target amplicons and circular plasmids all contribute to the synthesis of heterogenous RNAs.</em> </li><li>Concentrate and further clean-up the gel-purified DNA templates using the Zymo DNA clean and concentrator-5 kit following the manufacturerâs recommendations. Elute DNA from each column with 6 Î¼l of elution buffer. Quantify your template concentration and purity using a microvolume spectrophotometer and adjust the DNA concentration to 500 ng/Î¼l.<br/><em>Note: Highest transcription rates are obtained with highest purity templates. Expect A<sub>260</sub>/A<sub>280</sub> ratios of ~1.8 and A<sub>260</sub>/A<sub>230</sub> ratios &gt; 2.0.</em> </li><li>Transcribe RNA using 1 Î¼g of DNA template as input with the HiScribe T7 high yield RNA synthesis kit in 20 Î¼l reactions, for 2 h at 37 Â°C, following manufacturerâs recommendations.<br/><em>Notes: <br/><ol class="npwthree"><li>While the reactions are running, pre-run a Novex 6% TBE-Urea PAGE gel for 20 min in 1x Novex TBE-Urea running buffer.</li><li>Other pre-cast TBE-urea PAGE gels can be used as an alternative to Novex gels with compatible electrophoresis systems. While we routinely do not cast our own TBE-Urea gels, denaturing TBE-urea PAGE gels prepared in the laboratory could also be used. </li></ol></em> </li><li>Stop the reactions by adding 2 Î¼l DNase to the reactions, pipette gently to mix and incubate for an additional 15 min at 37 Â°C.</li><li>Add 1 volume of Novex TBE-Urea 2x sample buffer to the reactions, mix well by pipetting and heat at 80 Â°C for 3 min, then cool the samples immediately on ice. Load your samples on the Novex 6% TBE-Urea PAGE gel from Step B4a âNOTEâ. Load 2 Î¼l of diluted RiboRuler low range RNA ladder on the first well. Run the gel using the XCell SureLock mini-cell electrophoresis system at 100 V (constant voltage) for 50-60 min. For longer RNAs run the gel for a longer time (around 90 min) to achieve better separation. The dark blue bromophenol blue dye front corresponds to a fragment length of 25 nt according to the manufacturerâs instructions.<br/><em>Notes: <br/><ol class="npwthree"><li>Before loading your samples, pipette buffer vigorously into each well of the gel to remove urea buildup. Load your samples on the rinsed, clean wells. Use a single gel per transcription reaction; if using a 10-well gel, load ~5 Î¼l of sample per well. Do not overload the gel.</li><li>Dilute the RiboRuler low range RNA ladder in RNase free water to load 10-20 ng RNA per band. The undiluted RNA ladder contains 140 ng of each RNA transcript. </li></ol></em> </li><li>Disassemble the XCell SureLock mini-cell, remove the gel and dismantle the gel cassette carefully, leaving the gel mounted on one of the plastic plates. Stain the gel with SYBR Gold (diluted as per manufacturerâs recommendations in 1x TBE) by submerging the gel in the buffer for 20 min at room temperature. <br/><em>Note: Avoid touching the gel with gloved fingers as it will leave marks. We have found that staining the gel without agitation yields better results. Cover the gels to prevent exposure to light while staining.</em> </li><li>Carefully remove the gel from the backing plate and place it on plastic cling-wrap. Wipe the stage of the UV transilluminator with RNaseZAP RNase decontamination solution and follow by wiping the surface with 70% ethanol. Place the gel on the plastic cling wrap on the transilluminator and visualize the RNA. Use a sterile No. 11 scalpel to cut around the band of interest. Collect the band of interest in a non-stick 0.5 ml RNase-free tube. Use one tube for every 2-3 lanes of gel (max. gel slice per tube should not exceed 1 mm x 15 mm x 5 mm).<br/><em>Notes: <br/><ol class="npwthree"><li>It is possible to have too much RNA loaded on the gel, in which case expect to see âghostsâ around the band of interest (regions where the dye accumulates around the band). Such event does not compromise the RNA extraction downstream.</li><li>Do not drag the tip of the scalpel over the transilluminator stage. Instead place the blade edge parallel to the surface and press down around the band. No. 11 scalpels have straight blade edges.</li></ol></em> </li><li>Nest each 0.5 ml tube containing the gel fragments inside a non-stick 1.5 ml RNase-free tube. Carefully pierce the bottom of the 0.5 ml tube with a sterile 20 G syringe needle. Do not pierce the larger 1.5 ml tube. Spin in a benchtop microcentrifuge at maximum speed for 3 min to force the gel through the hole. Transfer any residual gel from the 0.5 ml tube to the holder tube. Discard the 0.5 ml tubes.</li><li>Add ~3 gel volumes of RNA gel extraction buffer to the crushed gel in the holder tubes. Incubate for one hour at room temperature with constant agitation at 1,000 rpm in a thermomixer.</li><li>Remove tubes from thermomixer. Vortex the gel slurry and recover it completely using a cut P1000 tip. Transfer the slurry to a Spin-X centrifuge tube filter. Be careful when transferring as gel pieces can clog the pipette tip. Spin the Spin-X centrifuge tube filters at max speed in a benchtop microcentrifuge for 3 min. The filtrate containing your RNA will be at the bottom of the filter tube and the gel pieces will stay on the filter.</li><li>Transfer the filtrate to a fresh non-stick nuclease free 1.5 ml microcentrifuge tube. Pool all filtrates for the same RNA (up to ~0.5 ml aliquots). Add 1/9 of the volume of 3 M NaOAc pH 5.5 and 1 volume of isopropanol and incubate for at least 30 min at -80 Â°C to precipitate the RNA.</li><li>Spin at maximum speed in a benchtop microcentrifuge at 4 Â°C for 30 min top pellet the RNA. Remove the supernatant being careful not to disturb the pellet. Wash the RNA pellet twice with 1 ml of 80% ice-cold ethanol prepared in RNase-free water.</li><li>Remove ethanol from the pellet. Spin the tubes to collect trace ethanol from the walls of the tube and remove as much as possible of the remaining ethanol with a pipette. Air-dry the RNA pellets for at least 10 min at room-temperature. Make sure all ethanol has evaporated before proceeding. Resuspend the pellets in 20 Î¼l RNase-free water or RNA resuspension buffer by gentle pipetting.</li><li>Fold the RNAs from Step B14 by incubating them for 5 min at 95 Â°C in a thermocycler and then slowly cool the samples (1 Â°C/min) until they reach 25 Â°C. Keep the samples on ice and use immediately for <em>in vitro </em>cleavage assays or store them at -80 Â°C for future use.<br/><em>Note: Before setting up the in vitro cleavage assays, measure the RNA concentration and purity using a microvolume spectrophotometer. Expect to obtain A<sub>260</sub>/A<sub>280</sub> ratios ~2.0 and A<sub>260</sub>/A<sub>230</sub> ratios &gt; 2</em><em>.0.</em> </li></ol><br/></li><li><em>In vitro </em>cleavage assay of <em>in vitro </em>transcribed RNAs with purified recombinant IRE1 cytosolic domains and denaturing urea-PAGE of IRE1-generated RNA fragments<br/><ol class="npwtwo"><li>Assemble the <em>in vitro </em>cleavage reactions in non-stick 1.5 ml RNase-free tubes on ice.</li><li>Assemble a master mix for your entire reaction time-series. Each time point has a final volume of 10 Î¼l. Add the following components (in order; per reaction):<br/><ol class="npwthree"><li>2x RNA cleavage buffer</li><li>15-100 ng of PAGE-purified <em>in vitro</em> transcribed RNA</li><li>50 nM to 5 Î¼M recombinant IRE1-KR43</li></ol></li><li>Place the tube containing the master mix in a thermomixer set at 30 Â°C* and incubate for predetermined amounts of time, ranging from 30 s to 90 min depending on the RNA you are studying. Do not agitate.<br/><em>Notes:<br/><ol class="npwthree"><li>Plan your time series. *You may use a dry bath or water bath set at 30 Â°C. </li><li>Include a control for cleavage specificity by supplementing one duplicate reaction (the one corresponding to the longest time point) with 4Î¼8C to a final concentration of 10 Î¼M. To inhibit IRE1, pre-incubate 10 Î¼M IRE1-KR43 with 20 Î¼M 4Î¼8C for 30 min on ice.</li></ol></em> </li><li>For each time point take out 7 Î¼l from the master mix and stop the reactions by adding 1 Î¼l of proteinase K solution (20 Î¼g) and incubating for 10 min at 37 Â°C. Add 1 reaction volume of stop solution, mix gently and heat the samples to 80 Â°C; incubate for an additional 5 min, then cool the samples immediately on ice to prevent re-folding or base-pairing of the denatured RNA.</li><li>Load 5 Î¼l of reaction for each data point (~1/3 of the total volume per time point) on a Novex 6% TBE-Urea PAGE gels. Load diluted RNA ladder on the first well of the gel as indicated in step B6 âNOTE bâ). Run the gel at 100 V (constant voltage) for 50-60 min (or longer if you are analyzing longer RNAs) using the XCell SureLock mini-cell electrophoresis system.<br/><em>Note: Before loading your samples, thoroughly rinse each well of the gel to remove urea buildup. </em><em>Load your samples on the rinsed, clean wells.</em> </li><li>Disassemble the XCell SureLock mini-cell, remove the gel and dismantle the cassette gel carefully, leaving the gel mounted on one of the plastic plates. Stain the gel with SYBR Gold (diluted as per manufacturerâs recommendations in 1x TBE) by submerging the gel in the buffer for 20 min at room temperature.<br/><em>Note: Avoid touching the gel with gloved fingers, as it will leave marks. Cover the gels to prevent exposure to light while staining.</em> </li><li>Place the stained gel on the stage of a gel imaging system and capture digital images using various exposure times. Record the exposure times for each photograph. Use the longer exposure time that gives the best contrast without saturated pixels. Representative examples of typical results are shown in Figure 2.<br/><br/><img alt="" class="layerphoto" height="126" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190715/20190715065243_0963.jpg" width="200"/><br/><strong>Figure 2. TBE-urea PAGE gels showing the IRE1-dependent cleavage of select IRE1 targets, BLOC1S1 (left) and SEC61A1 (right). </strong>5 Î¼M IRE1-KR43 was used to cleave 25 nM of RNA substrates for the indicated times. The IRE1 inhibitor 4Âµ8C was used as a control for specificity.</li></ol></li></ol><p class="pptt" id="biaoti28502">### Data analysis</p><p class="ppzz">The digital images of the gels can be densitometrically quantified and analyzed using ImageJ Software.</p><ol class="npwtwok"><li>Adjust the image brightness and contrast and remove background prior to the analysis.</li><li>Draw boxes of equal size around the bands corresponding to the uncleaved RNA and calculate the area under the curve (pixel density). </li><li>Normalize your numbers to the uncleaved control and calculate the fraction of remaining uncleaved RNA. To determine observed rate constants, plot these values as a function of time and calculate the best curve fit using your choice of scientific graphing and analysis software.<br/></li></ol><p class="pptt" id="biaoti28503">### Recipes</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwtwo"><li><em>For all recipes, use molecular biology grade reagents or pre-made solutions (available from multiple suppliers). Stock solutions should be prepared in RNase-free water.</em> </li><li><em>To pH a 0.5 M or 1 M HEPES stock solution, use KOH pellets, add slowly while stirring the solution and monitor the pH continuously.</em> </li><li><em>*We have not observed significant differences when using Mg(OAc)<sub>2</sub> as opposed to MgCl<sub>2</sub>.</em> </li></ol><p><br/></p><ol class="npwtwok"><li>SF21 cell lysis buffer<br/>20 mM HEPES pH 7.4<br/>600 mM KCl<br/>2 mM MgCl<sub>2</sub><br/>10% glycerol<br/>10 mM imidazole<br/>Supplement with freshly added cOmplete EDTA-free protease inhibitor cocktail </li><li>IRE1 dialysis buffer<br/>20 mM HEPES pH 7.4<br/>300 mM KCl<br/>2 mM MgCl<sub>2</sub><br/>5% glycerol<br/>1 mM TCEP-HCl</li><li>RNA gel extraction buffer<br/>300 mM NaOAc<br/>1 mM EDTA<br/>Supplemented with freshly added recombinant RNasin RNase inhibitor</li><li>RNA re-suspension buffer<br/>20 mM HEPES pH 7.4<br/>100 mM NaCl<br/>1 mM MgCl<sub>2</sub>*<br/><em>Note: We have not observed significant differences when re-folding RNA in this buffer versus refolding in RNase-free water. </em> </li><li>2x RNA cleavage buffer<br/>40 mM HEPES pH 7.4<br/>200 mM NaCl<br/>2 mM MgCl<sub>2</sub>* <br/>10% glycerol<br/>2 mM TCEP-HCl</li><li>Stop solution<br/>10 M urea<br/>0.1% SDS<br/>1 mM EDTA<br/>0.01% xylene cyanol<br/>0.01% bromophenol blue<br/><em>Note: Bromophenol blue can generate background noise/signal artifacts during imaging when the RNA concentration is low.</em><br/></li></ol><p class="pptt" id="biaoti28504">### Acknowledgments</p><p class="ppzz">DAA was supported by an Irvington Postdoctoral Fellowship of the Cancer Research Institute. JP was supported by a Human Frontier Science Program Postdoctoral Fellowship. PW is an Investigator of the Howard Hughes Medical Institute. We thank Aaron Mendez for his help with setting up the purification of human IRE1 Kinase/RNase domains.</p><p class="pptt" id="biaoti28505">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti28506">### References</p><ol class="npwtwok"><li>Acosta-Alvear, D., Karagoz, G. E., Frohlich, F., Li, H., Walther, T. C. and Walter, P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30582518" target="_blank">The unfolded protein response and endoplasmic reticulum protein targeting machineries converge on the stress sensor IRE1.</a> <em>Elife</em> 7: e43036.</li><li>Acosta-Alvear, D., Zhou, Y., Blais, A., Tsikitis, M., Lents, N. H., Arias, C., Lennon, C. J., Kluger, Y. and Dynlacht, B. D. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17612490" target="_blank">XBP1 controls diverse cell type- and condition-specific transcriptional regulatory networks.</a> <em>Mol Cell</em> 27(1): 53-66.</li><li>Bae, D., Moore, K. A., Mella, J. M., Hayashi, S. Y. and Hollien, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30787040" target="_blank">Degradation of Blos1 mRNA by IRE1 repositions lysosomes and protects cells from stress.</a> <em>J Cell Biol</em> 218(4): 1118-1127.</li><li>Calfon, M., Zeng, H., Urano, F., Till, J. H., Hubbard, S. R., Harding, H. P., Clark, S. G. and Ron, D. (2002).<a href="http://www.ncbi.nlm.nih.gov/pubmed/11780124" target="_blank"> IRE1 couples endoplasmic reticulum load to secretory capacity by processing the XBP-1 mRNA.</a> <em>Nature</em> 415(6867): 92-96.</li><li>Cheng, C. Y., Chou, F. C., Kladwang, W., Tian, S., Cordero, P. and Das, R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26035425" target="_blank">Consistent global structures of complex RNA states through multidimensional chemical mapping.</a> <em>Elife</em> 4: e07600.</li><li>Cross, B. C., Bond, P. J., Sadowski, P. G., Jha, B. K., Zak, J., Goodman, J. M., Silverman, R. H., Neubert, T. A., Baxendale, I. R., Ron, D. and Harding, H. P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22315414" target="_blank">The molecular basis for selective inhibition of unconventional mRNA splicing by an IRE1-binding small molecule.</a> <em>Proc Natl Acad Sci U S A </em>109(15): E869-878.</li><li>Han, D., Lerner, A. G., Vande Walle, L., Upton, J. P., Xu, W., Hagen, A., Backes, B. J., Oakes, S. A. and Papa, F. R. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19665977" target="_blank">IRE1alpha kinase activation modes control alternate endoribonuclease outputs to determine divergent cell fates.</a><em> Cell </em>138(3): 562-575.</li><li>Harnoss, J. M., Le Thomas, A., Marsters, S. A., Lawrence, D. A., Lu, M., Chen, Y. A., Qing, J., Totpal, K., Kan, D., Segal, E., Wallweber, H. A., Wang, W., Clark, K., Kaufman, S., Beresini, M., Sandoval, W., Lorenzo, M., Wu, J., Ly, J., De Bruyn, T., Heidersbach, A., Haley, B., Gogineni, A., Weimer, R., Lee, D., Braun, M., Rudolph, J., VanWyngarden, M. J., Sherbenou, D. W., Gomez-Bougie, P., Amiot, M., Acosta-Alvear, D., Walter, P., Ashkenazi, A. (2018). <a href="https://www.biorxiv.org/content/10.1101/495242v2.article-metrics" target="_blank">Disruption of IRE1Î± through its Kinase Domain Attenuates Multiple Myeloma.</a> <em>BioRxiv</em> doi: https://doi.org/10.1101/495242.</li><li>Hollien, J. and Weissman, J. S. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16825573" target="_blank">Decay of endoplasmic reticulum-localized mRNAs during the unfolded protein response.</a> <em>Science</em> 313(5783): 104-107.</li><li>Hollien, J., Lin, J. H., Li, H., Stevens, N., Walter, P. and Weissman, J. S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19651891" target="_blank">Regulated Ire1-dependent decay of messenger RNAs in mammalian cells.</a> <em>J Cell Biol</em> 186(3): 323-331.</li><li>Iwawaki, T., Akai, R., Kohno, K. and Miura, M. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14702639" target="_blank">A transgenic mouse model for monitoring endoplasmic reticulum stress.</a> <em>Nat Med</em> 10(1): 98-102.</li><li>Iwawaki, T., and Akai, R. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17026957" target="_blank">Analysis of the XBP1 splicing mechanism using endoplasmic reticulum stress-indicators. Biochem.</a> <em>Biochem Biophys Res Commun</em> 350(3): 709-15.</li><li>KaragÃ¶z, G. E., Acosta-Alvear, D. and Walter, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30670466" target="_blank">The unfolded protein response: detecting and responding to fluctuations in the protein-folding capacity of the endoplasmic reticulum.</a> <em>Cold Spring Harb Perspect Biol.</em> pii: a033886.</li><li>Korennykh, A.V., Egea, P.F., Korostelev, A.A., Finer-Moore, J., Zhang, C., Shokat, K.M., Stroud, R.M., and Walter, P. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19079236" target="_blank">The unfolded protein response signals through high-order assembly of Ire1.</a> <em>Nature</em> 457(7230): 687-93.</li><li>Lee, A. H., Iwakoshi, N. N. and Glimcher, L. H. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14559994" target="_blank">XBP-1 regulates a subset of endoplasmic reticulum resident chaperone genes in the unfolded protein response.</a> <em>Mol Cell Biol</em> 23(21): 7448-7459.</li><li>Li, H., Korennykh, A. V., Behrman, S. L. and Walter, P. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20798350" target="_blank">Mammalian endoplasmic reticulum stress sensor IRE1 signals by dynamic clustering.</a> <em>Proc Natl Acad Sci U S A</em> 107(37): 16113-16118.</li><li>Lu, M., Lawrence, D. A., Marsters, S., Acosta-Alvear, D., Kimmig, P., Mendez, A. S., Paton, A. W., Paton, J. C., Walter, P. and Ashkenazi, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24994655" target="_blank">Opposing unfolded-protein-response signals converge on death receptor 5 to control apoptosis.</a> <em>Science</em> 345(6192): 98-101.</li><li>Mendez, A. S., Alfaro, J., Morales-Soto, M. A., Dar, A. C., McCullagh, E., Gotthardt, K., Li, H., Acosta-Alvear, D., Sidrauski, C., Korennykh, A. V., Bernales, S., Shokat, K. M. and Walter, P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25986605" target="_blank">Endoplasmic reticulum stress-independent activation of unfolded protein response kinases by a small molecule ATP-mimic.</a> <em>Elife</em> 4: e05434.</li><li>Moore, K. and Hollien, J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26108623" target="_blank">Ire1-mediated decay in mammalian cells relies on mRNA sequence, structure, and translational status.</a> <em>Mol Biol Cell</em> 26(16): 2873-2884.</li><li>Mori, K. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19861400" target="_blank">Signalling pathways in the unfolded protein response: development from yeast to mammals.</a> <em>J Biochem</em> 146(6): 743-750.</li><li>Peschek, J., Acosta-Alvear, D., Mendez, A. S. and Walter, P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26483401" target="_blank">A conformational RNA zipper promotes intron ejection during non-conventional XBP1 mRNA splicing.</a><em> EMBO Rep</em> 16(12): 1688-1698. </li><li>Sidrauski, C., Acosta-Alvear, D., Khoutorsky, A., Vedantham, P., Hearn, B. R., Li, H., Gamache, K., Gallagher, C. M., Ang, K. K., Wilson, C., Okreglak, V., Ashkenazi, A., Hann, B., Nader, K., Arkin, M. R., Renslo, A. R., Sonenberg, N. and Walter, P. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23741617" target="_blank">Pharmacological brake-release of mRNA translation enhances cognitive memory.</a> <em>Elife </em>2: e00498.</li><li>So, J. S., Hur, K.Y., Tarrio, M., Ruda, V., Frank-Kamenetsky, M., Fitzgerald, K., Koteliansky, V., Lichtman, A.H., Iwawaki, T., Glimcher, L.H., <em>et al.</em> (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23741617" target="_blank">Silencing of lipid metabolism genes through IRE1Î±-mediated mRNA decay lowers plasma lipids in mice.</a> <em>Cell Metab</em> 16(4):487-99.</li><li>Tam, A. B., Koong, A. C., and Niwa, M. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25437541" target="_blank">Ire1 has distinct catalytic mechanisms for XBP1/HAC1 splicing and RIDD.</a> <em>Cell Rep</em> 9(3): 850-8.</li><li>Yoshida, H., Matsui, T., Yamamoto, A., Okada, T., and Mori, K. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11779464" target="_blank">XBP1 mRNA is induced by ATF6 and spliced by IRE1 in response to ER stress to produce a highly active transcription factor.</a> <em>Cell</em> 107(7): 881-91.<br/></li></ol>