<body>
            <h1>Axon-seq for in Depth Analysis of the RNA Content of Neuronal Processes</h1>
            <p><strong>Authors:</strong> Jik Nijssen, Julio Aguila, Eva Hedlund</p>
            <h2>Abstract</h2>
            <p>Neuronal processes have an RNA composition that is distinct from the cell body. Therefore, to fully understand neuronal biology in health and disease we need to study both somas, dendrites and axons. Here we describe a detailed protocol of a newly refined method, Axon-seq, for RNA sequencing of axons (and dendrites) grown in isolation using single microfluidic devices. We also detail how to generate motor neurons from mouse and human pluripotent stem cells for sequencing, but Axon-seq is applicable to any neuronal cell. In Axon-seq, the axons are recruited through a growth factor gradient, lysed and directly processed to cDNA without RNA isolation. A careful bioinformatic step ensures that any soma-contaminated samples are easily identified and removed.</p>
            <p><strong>Keywords:</strong> RNA sequencing, Motor neuron, Axons, Stem cells, Microfluidic devices, Bioinformatics</p>
            <h2>Background</h2><p>Neurons are highly polarized cells. Their processes, both dendrites and axons, need to be able to respond to changes in the microenvironment in a manner independent of the soma (Holt and Schuman, 2013). To fully understand neuron biology, it is therefore important to be able to study neurites in isolation, in addition to conducting analysis of the cell bodies. This appears particularly important for neurons where the axon and dendrites constitute the majority of the cellular volume, as is the case for spinal motor neurons where we estimate that they comprise approximately 99% of the cellular volume. To isolate neurites, there are excellent tools such as Campenot Chambers (Boyden, 1962; Campenot, 1977) or microfluidic chambers (Taylor <em>et al.</em>, 2005).<br/><br/>However, while neurites are separated out in such devices, it is important to bear in mind that cross-contamination between compartments can still occur. Thus, RNA sequencing of isolated axonal compartments (Minis <em>et al.</em>, 2014; Saal <em>et al</em>., 2014; Briese <em>et al.</em>, 2016; Rotem<em> et al</em>., 2017) can lead to incorrect results/conclusions when the purity and exclusion of somas is insufficiently examined. To ensure detailed and accurate investigation of motor axonal mRNA composition and its modulation in ALS we developed Axon-seq (Nijssen <em>et al.</em>, 2018). This is an application of our LCM-seq method for single-cell spatial RNA-sequencing (Nichterwitz <em>et al</em>., 2016).<br/><br/>In Axon-seq, we use microfluidic devices to separate axons from stem cell-derived motor neurons (mouse and human) from their somas. As motor neurons have axons that traverse far longer distances than their dendrites, we are able to analyze axons alone. This may not be possible for all neuronal subtypes, but the method still allows for analysis of neurites as an entity. <br/><br/>In contrast to previous methods, Axon-seq does not require an RNA isolation step, and it allows for high sensitivity and cost-efficient sequencing from a single microfluidic device.<br/><br/>Notably, Axon-seq effectively eliminates all samples with any somatic cross-contamination, as it uses a highly stringent and sensitive bioinformatic quality control step that identifies axonal samples containing trace levels of mRNA from undesired cell somas. Here we provide a detailed protocol for Axon-seq which can be applied to any cell containing longer processes.</p>
        </body>

<p class="pptt" id="biaoti28543">## Materials and Reagents</p><ol class="npwone"><li>Materials<br/><ol class="npwtwo"><li>Sterile 6 and 10 cm bacterial dishes (non-adhesive plastic) (Corning, catalog numbers: 351007 and 351029)</li><li>Pipette tips (0.5-10 Î¼l, 10-100 Î¼l, 100-1,000 Î¼l)</li><li>Sterile 15 and 50 ml Falcon tubes</li><li>Parafilm</li><li>70 Î¼m cell-strainer filter (VWR, catalog number: 732-2758P)</li><li>24 x 32 mm cover glasses (VWR, Menzel GlÃ¤ser, catalog number: 630-2873)</li><li>Soft pencil-type brush (Any soft brush type that does not have too strong hairs that could pierce/damage the PDMS-surface of which the microfluidic devices are made. Brushes, from a typical hardware/convenience store, with a diameter of 2-5 mm work best, laboratory grade is not required. Nylon/synthetic hairs are not an issue.)</li><li>Waterproof marker with fine (1 mm) or very fine (0.3 mm) tip</li></ol><br/></li><li>Biological material<br/><ol class="npwtwo"><li>Motor neurons from mouse embryonic stem cells</li><li>Motor neurons from human pluripotent stem cells<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Reagents<br/><ol class="npwtwo"><li>70% ethanol</li><li>Triton X-100 (Sigma-Aldrich, catalog number: X100)</li><li>PBS (Thermo Fisher, catalog number: 14190136)</li><li>Poly-L-ornithine, 0.01% solution (Sigma-Aldrich, catalog number: A-004-C)</li><li>Fibronectin, 1 mg/ml solution (Sigma-Aldrich, catalog number: F1056)</li><li>Laminin, 1 mg/ml solution (Sigma-Aldrich, catalog number: L2020)</li><li>TrypLe Express (Thermo Fisher, catalog number: 12604021)</li><li>Neurobasal (Thermo Fisher, catalog number: 21103049)</li><li>DMEM/F12 with GlutaMAX (Thermo Fisher, catalog number: 31331028)</li><li>B27 supplement, custom (Thermo Fisher, catalog number: 0080085SA)</li><li>Penicillin/Streptomycin (Thermo Fisher, catalog number: 15140122)</li><li>GlutaMAX supplement 100x (Thermo Fisher, catalog number: 35050061)</li><li>2-Mercaptoethanol (Sigma-Aldrich, catalog number: M6250)</li><li>Ascorbic acid, 100 mg powder (Sigma-Aldrich, catalog number: A4403), prepare as 200 mM stock solution by dissolving the powder in 2.84 ml of nuclease-free dH<sub>2</sub>O</li><li>Y-27632, 10 mg powder (Tocris, catalog number: 1254), prepare as 10 mM stock solution by dissolving the powder in 3.12 ml nuclease-free dH<sub>2</sub>O</li><li>LDN-193189 dihydrochloride, 10 mg powder (Tocris, catalog number: 6053), prepare as 10 mM stock solution by dissolving the powder in 2.09 ml DMSO, then prepare 1 mM working stocks (1:10 diluted) in DMSO</li><li>SB-431542, 10 mg powder (Tocris, catalog number: 1614), prepare as 10 mM stock solution by dissolving the powder in 2.60 ml DMSO</li><li>CHIR-99021, 10 mg powder (Tocris, catalog number: 4423), prepare as 3 mM stock solution by dissolving the powder in 7.16 ml DMSO<br/></li><li>DAPT, 10 mg powder (Tocris, catalog number: 2634), prepare as 10 mM stock solution by dissolving the powder in 2.31 ml DMSO<br/></li><li>SAG, 1 mg powder (Tocris, catalog number: 4366), prepare as 500 Î¼M stock solution by dissolving the powder in 4.08 ml DMSO</li><li>All-<em>trans</em> retinoic acid, 50 mg powder (Sigma-Aldrich, catalog number: R2625), prepare as 100 mM stock solution by dissolving the powder in 1.66 ml DMSO, then prepare 1 mM working stocks (1:100 diluted) in DMSO</li><li>Dimethyl sulfoxide (DMSO) (Sigma-Aldrich, catalog number: D2650)</li><li>Bovine serum albumin (BSA), 7.5% solution in dPBS (Sigma-Aldrich, catalog number: A8412)</li><li>Glial-derived neurotrophic factor (GDNF), 10 Î¼g powder (Peprotech, catalog number: 450-10), prepare as 10 Î¼g/ml stock solution by dissolving the powder in 1 ml dPBS with 0.1% BSA</li><li>Brain-derived neurotrophic factor (BDNF), 10 Î¼g powder (Peprotech, catalog number: 450-02), prepare as 10 Î¼g/ml stock solution by dissolving the powder in 1 ml dPBS with 0.1% BSA</li><li>RNase-inhibitor (Takara, catalog number: 2313A)</li><li>DTT (part of SuperScript II Reverse Transcriptase kit, Thermo Fisher, catalog number: 18064071)</li><li>Nuclease-free dH<sub>2</sub>O (Thermo Fisher, catalog number: 10977-035)</li><li>37% formaldehyde solution (Merck Millipore, catalog number: 818708)</li><li>Mouse MN medium (see Recipes)</li><li>N2 medium (see Recipes)</li><li>B27 medium (see Recipes)</li><li>Human EB medium (see Recipes)</li><li>Lysis/harvesting solution (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti28544">### Equipment</p><ol class="npwtwok"><li>Glass beaker (500 ml)</li><li>Sterile forceps (Dumont, catalog number: 11251-20, or other forceps of similar standard size)</li><li>Pipettes (0.5-10 Î¼l, 10-100 Î¼l, 100-1,000 Î¼l)</li><li>Magnetic stirrer</li><li>Sterile laminar flow hood</li><li>Humidified incubator (37 Â°C, 5% CO<sub>2</sub>)</li><li>Suction aspirator</li><li>Standard Neuron Device (Xona Microfluidics<sup>TM</sup>, SND75/150/450/900)</li><li>Low-speed orbital shaker (Elmi-tech, DOS-20S)</li><li>Inverted bright field microscope (&gt; 20x magnification is required to reliably see axons exiting the microgrooves. Examples are Laxco LMI-3000 or Carl Zeiss PrimoVert. Any similar setup is fine.)<br/></li></ol><p class="pptt" id="biaoti28545">### Software</p><ol class="npwtwok"><li>R (<a href="https://www.r-project.org" target="_blank">https://www.r-project.org</a>)</li><li>RStudio open source edition, Boston, MA, USA (<a href="https://www.rstudio.com" target="_blank">https://www.rstudio.com</a>)<br/></li></ol><p class="pptt" id="biaoti28546">### Procedure</p><ol class="npwone"><li>Storage and washing of microfluidic devices<br/>Commercial microfluidic devices from Xona Microfluidics<sup>TM </sup>(Standard Neuron Devices) with any groove length can be stored in dH<sub>2</sub>O and used multiple times. <br/><em>Note: The groove length of the devices could affect the proportion of axons versus dendrites that can cross over to the opposite chamber, as could the length of time of culture in the devices. For mouse motor neurons, we used devices with 150 Î¼m length grooves and saw a clear majority of axons being recruited during the 7 days of culture. For human motor axons, we used devices with 450 Î¼m grooves.<br/></em> <ol class="npwtwo"><li>Place devices in a glass beaker with 500 ml of 1% Triton X-100 in dH<sub>2</sub>O for 2 h and use a magnetic stirrer to keep devices agitated.</li><li>Use a soft brush to gently clean the microgroove surfaces and the chambers of the devices. Transfer devices into dH<sub>2</sub>O afterward.</li><li>Wash 3 x 10 min in fresh dH<sub>2</sub>O each time to remove any residues of Triton X-100. Then, sterilize devices in 70% ethanol for a minimum of 10 min.<br/><em>Note: The washing procedures can be done at any time and devices can be stored in 70% ethanol to be ready when needed. Start the device attachment (below) the day that the coating procedure is to be initiated.<br/><br/></em> </li></ol></li><li>Attachment of microfluidic devices to cell culture surface<br/><ol class="npwtwo"><li>Remove microfluidic devices from 70% ethanol using sterile forceps and allow the devices to air-dry in a sterile laminar flow hood for 20 min. Use a suction aspirator to remove any residual ethanol.<br/><em>Note: It is very important that devices are entirely dry at this stage, as any remaining liquid will negatively affect device attachment and can cause leakage in later stages.</em> </li><li>Place 4 drops of 4 Î¼l dH<sub>2</sub>O in a sterile 10 cm bacterial dish (Figure 1A).</li><li>Place sterile glass cover glasses (32 x 24 mm, thickness #1 or #1.5) on top of the water drops to ensure that the cover glasses do not move around within the dish (Figure 1B).</li><li>Use forceps to pick up one device at a time and gently press it, feature side down, onto the glass coverslip (Figure 1C). Apply gentle force, using the back of the forceps, to the center of the device (above the grooves), and all four corners. A microscope can be used to assess if all the grooves are in the same focal plane and thus attached to the coverslip. Devices attached in this way are stable for at least four weeks (and likely longer). Label the glass coverslips with a waterproof marker in one corner to define the future left-right orientation of the devices. The device forms the âceilingâ for the cells and creates grooves for the axons to grow through. Cells will attach to the glass surface of the coverslip.</li></ol><em>Notes: </em><br/><ol class="npwthree"><li><em>The final setup of devices (Step B4) is a suggestion. Devices can also be placed directly in tissue culture-grade plastic dishes (<em>i.e.</em>, directly onto the plastic, without cover glasses). For imaging purposes, however, placing them on glass coverslips is beneficial.</em> </li><li><em> In addition to the PDMS devices described here, Xona Microfluidics<sup>TM</sup> also offers pre-bonded devices for cell culture use (XonaChips). While these cost the same as regular devices, they are not optimal for reusing, as can be done with the regular devices (SND150). In the case of pre-bonded devices, the attachment step (Procedure B) can be skipped.</em><br/><br/><img alt="" class="layerphoto" height="83" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713081028_7842.jpg" width="200"/><br/><strong>Figure 1. Setup of 4 microfluidic devices in a 10 cm dish. </strong>A. First, place 4 drops of 4 Î¼l dH<sub>2</sub>O spaced apart evenly in the 10 cm dish. Then place glass cover glasses onto the drops to create a water seal between cover glasses and the plastic dish. B. When cover glasses are attached, place one microfluidic device with feature-side down on each cover glass. Press gently on the top of the device to ensure attachment. C. Final setup of four devices in one 10 cm dish. Other arrangements of devices (in terms of dish size) can be made if desired. </li></ol><br/></li><li>Coating of surfaces in microfluidic devices<br/><strong>Day 1:</strong><br/><ol class="npwtwo"><li>The primary surface coating consists of 0.001% poly-L-ornithine in dH<sub>2</sub>O. Prepare 0.5 ml solution per device.</li><li>Add 75 Î¼l solution into the two bottom wells of each device. Typically, the liquid will quickly fill the chambers through capillary forces. If this does not happen, use the aspirator with a mounted 1 ml pipette tip to gently suck the liquid through from the top. Do not aspirate all the liquid.</li><li>Once the liquid has flown through, add 150 Î¼l to the top left and 200 Î¼l to the top right compartment. Adding different volumes between the left and right chamber ensures that the coating is forced into the microgrooves over time.</li><li>Leave the coating on overnight at room temperature and shield it from light (<em>e.g.</em>, a drawer works well).</li></ol><strong>Day 2:</strong><br/><ol class="npwtwo" start="5"><li>Wash the devices twice using 1x PBS. To wash, aspirate the coating from the wells, then add PBS into the wells, 75 Î¼l in the bottom, then 150 Î¼l in the top.</li><li>The second coating consists of a solution of 2 Î¼g/ml fibronectin and 5 Î¼g/ml laminin in PBS. Add this last coating in a similar fashion to the firstâsee Steps C2-C4.</li></ol><br/></li><li>Specification of motor neurons from mouse embryonic stem cells<br/>Typically, a good starting point for differentiation is a 10 cm dish of stem cells at ~70-80% confluency. Cells are always maintained in a humidified incubator at 37 Â°C and 5% CO<sub>2</sub>.<br/><strong>Day 1:</strong><br/><ol class="npwtwo"><li>Prewarm TrypLE Express to be used for dissociation at 37 Â°C (<em>i.e.</em>, 2 ml for a 6 cm dish, 4 ml for 10 cm dish) in a tube. In addition, pre-warm a tube with 8 ml DMEM/F12-GlutaMAX (or any other plain MEM-based media).</li><li>Add the TrypLE Express to initiate the dissociation.</li><li>When cells have rounded up and are coming off the culture dish, use a pipette to dissociate them fully in the TrypLE Express, then transfer everything into the tube with pre-warmed DMEM.</li><li>Spin down for 4 min at 200 <em>x g</em>, at room temperature.</li><li>Remove supernatant and resuspend cells in 1 ml of mouse MN media (Recipe 1) and count.</li><li>Place cells in bacterial (non-adherent) dishes at a density of 0.5-1.5 million cells per ml (the number of cells is line dependent and needs to be tested out), and place dishes on a low-speed orbital shaker at 30 rpm overnight in the incubator.</li></ol><strong>Day 2:</strong><br/><ol class="npwtwo" start="7"><li>Remove dishes from the shaker and change to fresh mouse MN media. To change media on EBs, pipette the entire volume of media (including EBs) into a tube. Allow the EBs to sink for ~2 min and remove the supernatant. Resuspend EBs in 1 ml fresh media and place into a new culture dish.</li></ol><strong>Days 3-6:</strong><br/><ol class="npwtwo" start="8"><li>Change to fresh mouse MN media, supplemented with 100 nM all-<em>trans</em>-retinoic acid (RA) and 500 nM smoothened agonist (SAG). Repeat for 4 successive days. See Table 1.<br/><br/><strong>Table 1. Factors for motor neuron differentiation from mouse pluripotent stem cells</strong><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" height="156" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713081256_4796.jpg" width="530"/><br/><br/></div></li></ol><strong>Day 7:</strong><br/><ol class="npwtwo" start="9"><li>After 4 consecutive days of patterning, cells are ready to be dissociated and seeded into microfluidic devices (see below).</li></ol><br/></li><li>Specification of motor neurons from human pluripotent stem cells<br/>Typically, a good starting point for differentiation is a 10 cm dish of stem cells at ~70-80% confluency. Cells are always maintained in a humidified incubator at 37 Â°C and 5% CO<sub>2.</sub><br/><strong>Day 1:</strong><br/>The procedures on the first day (EB formation) are the same as described above for mouse ES cells. However, in the end, resuspend the cells in human EB media at a density of 0.5 M-0.75 M cells per ml (this density may require a degree of optimization between cell lines). Supplement the Day 1 human EB media with the required factors for Day 1 (for overview, see Table 2):<br/>5 Î¼M Y-27632 (ROCK-inhibitor)<br/>200 nM LDN-193189<br/>40 Î¼M SB-431542<br/>3 Î¼M CHIR-99021<br/>200 Î¼M ascorbic acid<br/><br/><strong>Table 2. Factors for motor neuron differentiation from human pluripotent stem cells</strong><br/><img alt="" class="layerphoto" height="348" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713081430_4725.jpg" width="700"/><br/><br/><strong>Days 2-10:</strong><br/>Perform daily media changes at approximate 24 h intervals with the corresponding factors for each day (Table 2).<br/>Note: During the RA-SAG stage of the protocol (days 3-10), it is not absolutely required that media changes follow 24 h intervals, and if necessary (but not recommended) one day of media changes can be omitted. However, it is critical that the dual-SMAD inhibition (using SB-431542 and LDN-193189) during the first two days is 48 h and not shorter.<br/><strong>Day 10:</strong><br/>Cells are ready to be dissociated and seeded into microfluidic devices (see below).<br/><br/></li><li>Cell dissociation and preparation for seeding<br/><ol class="npwtwo"><li>Transfer EBs into a sterile 15 ml Falcon tube. Allow the EBs to sink to the bottom for 1-5 min and aspirate the remaining media.</li><li>Add 1 ml pre-warmed (37 Â°C) TrypLE Express to the Falcon tube with EBs. The dissociation takes 10-20 min, depending on the size of the EBs.</li><li>Gently agitate the EBs every 5 min by tapping the tube or gently pipetting up and down using a 1 ml pipette.<br/><em>Note: If you resuspend EBs by pipette for the first time after 5 min in TrypLE Express, do not pipette them up into the pipette tip as they tend to stick on the inside. Instead, just pipette the liquid and ensure the EBs consequently are agitated slightly in the suspension.</em> </li><li>Once the EBs are beginning to break apart, pipette them up and down rather vigorously approximately 10 times to finalize the dissociation. Add 9 ml of pre-warmed DMEM to dilute the 1 ml of TrypLE Express.</li><li>Filter the resulting suspension through a 70 Î¼m cell-strainer filter to remove undissociated clumps/debris, then spin at 200<em> x g</em> for 4 min, at room temperature.</li><li>Resuspend the pellet in 1 ml of DMEM/F12-GlutaMAX (or any other plain MEM-based media). Count the cells.</li><li>Adjust the density to a final concentration of 2.5 x 10<sup>4</sup> cells per Î¼l (this will ensure that a total of 1 x 10<sup>5</sup> cells are seeded per microfluidic device later on). To do this, transfer 1 ml of the cell suspension into an Eppendorf tube and spin once more at 200 <em>x g</em> for 4 min. Resuspend cells in the required amount of media (<em>i.e.</em>, 1 Î¼l for every 2.5 x 10<sup>4</sup> cells). At this stage, the cells should be resuspended in the final culturing media, for human cells this is B27 media with the Day 10 factors (see Table 2), for mouse cells this is mouse MN media with Day 7 factors (see Table 1). <br/><em>Note: The exact composition of the final media can vary based on the type of neurons in culture. However, the addition of ascorbic acid and trophic factors (and ROCK-inhibitor for human cells) is highly recommended to improve survival upon plating in the confined environment of the microfluidic device.</em> </li><li>Prepare sufficient final media to fill all devices (0.5 ml per device). In the end, the device will hold media volumes as shown in Figure 2.</li><li>Take the required amount of media (see Figure 2) for the axonal compartment (top and bottom well) and transfer to a new tube. Add trophic factors to increase the concentration to 50 ng/ml. The higher concentration of trophic factors on the âaxonalâ side will improve axonal recruitment across the microgrooves.<br/><br/><img alt="" class="layerphoto" height="114" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713081546_8927.jpg" width="200"/><br/><strong>Figure 2. Media volumes and trophic factor concentrations in the microfluidic device at seeding.</strong> Note that somatic concentrations of the neurotrophic factors differ between human and mouse MNs. After axons have crossed the microgrooves into the other compartment, trophic factor concentrations can be equalized to 10 ng/ml on both sides, both for mouse and human.</li></ol><br/></li><li>Cell seeding<br/><ol class="npwtwo"><li>Start removing coating from the devices by aspiration (from one 10 cm dish with four devices at a time). </li><li>Add 4 Î¼l of the axonal media (containing 50 ng/ml BDNF/GDNF) into the chamber on the axonal side. Pipette close to the chamber entrance on the top-left (in the orientation of Figure 2, see Figure 3A for photograph) and allow capillary forces to pull the liquid into the chamber.</li><li>Add the cells in the same manner to the other compartment. Once cells are seeded in the chamber, place devices in the incubator and allow the cells to attach for approximately 15-20 min.</li><li>Start checking devices from 10 min after seeding the cells and onward to avoid excessive evaporation of the media. Once cells appear attached, add the final media amounts to the wells as indicated in Figure 2. Check if the cells remain attached and then place devices in the incubator overnight.<br/><em>Notes:<br/></em> <ol class="npwthree"><li><em>Occasionally cells do not attach well even after 15 min, maybe in part due to the flow between the compartments. If this occurs, we recommend adding only 40 Î¼l of media to all wells (keeping the right media types for each side), which will avoid excessive flow between compartments. This way, there is minimal flow and evaporation will not be an issue. Then keep the cells like this for an additional hour and finally add media to the recommended amounts in Figure 2. </em> </li><em> </em> <li><em>For human cells, the dissociation takes place on Day 10 of the protocol. In addition to everything described here, maintain the corresponding factors in the media for each day as described in Table 2. The trophic factor (BDNF/GDNF) concentrations described there are for the somatic compartment, the axonal compartments receive the trophic factor gradient for the first few days (see below).</em><br/><br/><img alt="" class="layerphoto" height="149" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713082053_5629.jpg" width="200"/><br/><strong>Figure 3. Seeding cells into microfluidic chambers and representative micrographs.</strong> A. Example photographs of the angle at which media/cells can be added into the chambers of the microfluidic device. Dashed lines show the location of the chambers, and chamber entrances. B. Micrographs of a good culture in the axonal compartment and the somatic compartment, where clusters of neuron somas are visible.<br/><br/></li></ol></li></ol></li><li>Culturing devices<br/><ol class="npwtwo"><li>After seeding neurons into microfluidic devices, change media daily to ensure that the trophic factor gradient stays intact. A bright field microscope can be used to keep track of the axonal growth.<br/><em>Note: When changing media, do NOT aspirate media out of the chambers as this will forcibly remove all cells/axons, only change media in the wells.</em><br/></li><li>Once axons can be seen crossing several microgrooves into the axonal compartment (<em>i.e. </em>&gt; 10 grooves with axons), the trophic factor gradient can be removed, and both compartments instead receive 10 ng/ml of GDNF and BDNF. At this point, media can be exchanged every other day. See Figure 3B for a successful micrograph of a microfluidic device with growing axons.</li></ol><br/></li><li>Compartmental harvesting for RNA sequencing<br/><ol class="npwtwo"><li>Prepare the lysis/harvesting solution. Prepare ~70 Î¼l solution per microfluidic device, or less if only one compartment is harvested (final recommended lysis volumes are 50 Î¼l for the somatic compartment and 10 Î¼l for the axonal compartment). The lower volume for axons ensures that a higher concentration of RNA is present in the final solution, which aids in downstream applications.</li><li>The lysis solution (see Recipes) is made up as a 2% Triton X-100 solution in nuclease-free dH<sub>2</sub>O. First, cool this solution on ice, then supplement with:<br/>0.5 Î¼l RNase-inhibitor per 100 Î¼l (final concentration of 0.2 U/Î¼l)<br/>1 mM dithiothreitol (DTT), required for RNase-inhibitor enzymatic activity<br/><em>Note: Always keep lysis solution on ice during preparation and use.</em> </li><li>Prepare nuclease-free Eppendorf tubes. Label them according to the samples that are being collected, and pre-cool them on ice.</li><li>Have dry ice ready to snap-freeze harvested samples immediately following the collection.</li><li>Optionally, before starting the harvesting, take bright field images along the grooves. If somatic contamination is detected in the downstream bioinformatics analysis of axon samples, these images can be scrutinized for evidence of any cross-contaminating somas.</li><li>Harvest one device at a time. Begin by washing both compartments once with pre-warmed (37 Â°C) PBS. Use at least 100 Î¼l PBS.</li><li>Remove PBS from the axonal wells and add 10 Î¼l lysis solution to the top of the chamber, as if seeding cells (the vastly higher volume of PBS on the somatic side prohibits any lysis solution from flowing through the grooves and accidentally lyse somas).</li><li>Wait for 10 s, then pipette up and down ~5 x in the bottom well, and extract as much solution as possible from the bottom of the chamber (can be &gt; 10 Î¼l). Transfer solution into a prepared Eppendorf tube and snap-freeze on dry ice.</li><li>Add at least 100 Î¼l PBS to the axonal side again.</li><li>Repeat Steps I7 and I8 above for the somatic side, but instead use 50 Î¼l lysis solution. When extracting the solution, try to extract as much as possible (&gt; 50 Î¼l). Snap-freeze on dry ice.</li><li>After collection and snap-freezing, samples for RNA sequencing can be stored at -80 Â°C until further used.</li></ol><em>Note: For detailed instructions, protocols and reagents for the preparation of RNA sequencing libraries without RNA-extraction, see Nichterwitz et al., 2018. To generate axonal and somatodendritic sequencing libraries, standard laboratory practices related to handling of RNA must be considered. Namely, keep the samples cold (4 Â°C) and surfaces clean. Work with RNAse/DNAse-free disposables and reagents certified for molecular biology grade. Consider establishing a dedicated working station (bench) and a thermocycler exclusive for library preparation purposes. Due to the low amount of material (RNA) per device, comparable to that of single cells, we employed the Smart-seq2 protocol, an established methodology for single-cell transcriptomics. Briefly, in the first step, samples are quickly thawed from -80 Â°C and 10 Î¼l of axonal lysate (5 Î¼l of somatodendritic lysate) are subjected to reverse transcription using an oligo-dT primer and a template switch LNA-oligo. In the second step, an enrichment PCR with 18 cycles is introduced to amplify the amount of cDNA and samples purified using magnetic beads. At this point, the quality and quantity of the cDNA libraries can be measured by bioanalyzer using a dsDNA high sensitivity chip (for representative examples, see Figure 4). Finally, cDNA libraries are tagmented (Nextera XT Kit, Illumina) and barcoded with Illumina indexes for RNA sequencing.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713082202_9895.jpg"/><br/><strong>Figure 4. Typical bioanalyzer profiles for RNA sequencing using Axon-seq. </strong>Axonal cDNA samples have a distinct profile compared to somatodendritic samples. An example of ready to sequence axonal sequencing library is also shown after tagmentation reaction and barcoding.<br/><br/></li><li>Fixation of cells in devices for staining purposes<br/><ol class="npwtwo"><li>Remove all media from the wells and wash both compartments once with PBS.</li><li>Add a solution of 4% formaldehyde in PBS. When adding the formaldehyde, maintain similar volumes in all wells as used during media changes (see Figure 2) to ensure a flow both through the compartments and across the microgrooves.</li><li>Leave the devices to fix at room temperature for 30 min.</li><li>Wash 3 times with PBS, dispose of formaldehyde waste.</li><li>Store devices in PBS (~200 Î¼l in all wells) until further use. If not used within a few days, wrap the 10 cm dish with parafilm.<br/><em>Note: Either before or after performing the desired staining method (immunocytochemistry/FISH/RNAscope), devices need to be removed along with coverslips from the 10 cm plastic dish.</em> </li><li>Using a 1 ml pipette, add a thin line of H<sub>2</sub>O around the edges of the cover glass. This will loosen it from the underlying surface and cause it to float after a few seconds/minutes.</li></ol></li></ol><p class="pptt" id="biaoti28547">### Data analysis</p><p class="ppzz"><em><strong>Bioinformatic quality control</strong></em> </p><ol class="npwtwok"><li>After RNA sequencing has been performed, a bioinformatic quality control (QC) step should be performed to exclude axonal samples with (traces of) soma contamination.<br/><em>Note: The following is a simplified version of the bioinformatic QC that is conducted using the programming language R (R Core Team, 2018). More in-depth knowledge of bioinformatics and R will be required to perform these analyses, but what follows is a general guide to exclude any soma-contaminated samples.<br/></em> <ol class="npwthree"><li>Ensure that the RNA sequencing data is processed to both a count table (raw read counts) and an RPKM table with normalized values (reads per kilobase per million mapped reads).</li><li>As a general QC guideline, samples with too few mapped reads or detected genes should be excluded. To perform this, extract the total number of counts (= mapped reads) per sample from the count table, as well as the total number of genes with counts.</li><li>Exclude samples with fewer than 2.5 x 10<sup>5</sup> mapped reads.</li><li>Exclude samples that have fewer than 2,500 genes with any counts.</li></ol></li><li>After this general QC, the data can be inspected for soma-contamination of axonal samples. Briefly, there are four indicators to investigate:<br/>The number of detected genes in axonal samples<br/>2D visualization plots (PCA/umap)<br/>Sample correlations in a correlation heatmap<br/>Hierarchical clustering based on all expressed genes (or a subset)<br/><ol class="npwthree"><li>Obtain the number of detected genes in the axonal samples, and plot these as a dot plot. As a rough guideline, axon samples contain fewer than ~8,000 genes. Investigate any outliers in this distribution. Soma-contaminated samples will have a strong increase in the number of detected genes.</li><li>Using the base R function <em>prcomp</em>, or alternatively the package <em>umap</em> (Konopka, 2018), plot a 2D visualization of your samples (first 2 principal components for <em>prcomp</em>, or <em>n_components = 2</em> for <em>umap</em>). Investigate the plot. Typically, axon and soma samples will separate strongly in 2D space. Contaminated axon samples will be significantly shifted towards the somatic samples (Figure 5).</li><li>Using the base R function <em>corr</em>, calculate the correlation between samples based on all expressed genes. This correlation matrix can be plotted as a heatmap using the function <em>pheatmap</em> from the package carrying the same name. Typically, axonal samples have a low correlation to somatic samples. However, contaminated samples will have an increased overall correlation to all somatic samples.</li><li>Using <em>pheatmap</em> (Kolde, 2019), a heatmap with corresponding hierarchical tree clustering can be generated based on all expressed genes in the dataset. Where axon and soma samples normally cluster in distinct branches, contaminated samples can cluster together with soma samples.<br/><br/><img alt="" class="layerphoto" height="165" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190713/20190713082412_9801.jpg" width="200"/><br/><strong>Figure 5. Example PCA-plot showing the shift of contaminated axon samples towards the somatic samples.</strong> They additionally have an increase in the number of detected genes per sample.</li></ol><br/></li><li>Using these four measures, one can obtain an idea of samples that are possibly soma contaminated. Note that cutoffs need to be set for each of the measures, and the decision to exclude samples depends on the severity of the contamination.<br/></li></ol><p class="pptt" id="biaoti28548">### Recipes</p><ol class="npwtwok"><li>Mouse MN medium (500 ml)<br/>240 ml Neurobasal<br/>240 ml DMEM/F12-GlutaMAX<br/>10 ml B27 supplement, custom (stock 50x, add the full supplement bottle)<br/>5 ml Penicillin/Streptomycin<br/>5 ml GlutaMAX supplement<br/>0.91 ml 2-Mercaptoethanol</li><li>N2 medium<br/>1 bottle (500 ml) of DMEM/F12-GlutaMax<br/>5 ml N2 supplement (stock 100x, add the full supplement bottle)<br/>5 ml Penicillin/Streptomycin</li><li>B27 medium<br/>1 bottle (500 ml) of Neurobasal<br/>10 ml B27 supplement, custom (stock 50x, add the full supplement bottle)<br/>5 ml Penicillin/Streptomycin</li><li>Human EB medium<br/>50/50 mixture of N2 medium and B27 medium (above)</li><li>Lysis/harvesting solution (~70 Î¼l per microfluidic device)<br/>2% Triton X-100 in nuclease-free dH<sub>2</sub>O. Cool on ice, then supplement with:<br/>0.5 Î¼l RNase-inhibitor per 100 Î¼l (final concentration of 0.2 U/Î¼l)<br/>1 mM dithiothreitol (DTT), required for RNase-inhibitor enzymatic activity<br/></li></ol><p class="pptt" id="biaoti28549">### Acknowledgments</p><p class="ppzz">This work was supported by grants from the Swedish Research Council [2016-02112 and 2018-03255]; EU Joint Programme for Neurodegenerative Disease (JPND) [529-2014-7500]; the Strategic Research Programme in Neuroscience (StratNeuro); Karolinska Institutet; Birgit Backmarkâs Donation to ALS Research at Karolinska Institutet in memory of Nils and Hans Backmark; ÃhlÃ©n-stiftelsen (mA1/h17 and mA1/h18); Ulla-Carin Lindquists stiftelse fÃ¶r ALS forskning; and Magnus Bergvalls stiftelse [2015-00783 and 2016-01531] to E.H. J.A.B. is supported by a postdoctoral fellowship from the Swedish Society for Medical Research (SSMF).<br/>  This protocol is based on our previous study published in Stem Cell Reports in 2018: Nijssen <em>et al.</em> (2018).</p><p class="pptt" id="biaoti28550">### Competing interests</p><p class="ppzz">The authors declare that there are no financial or non-financial competing interests.</p><p class="pptt" id="biaoti28551">### Ethics</p><p class="ppzz">All work was carried out according to the Code of Ethics of the World Medical Association (Declaration of Helsinki) and with national legislation and institutional guidelines. The use of human stem cell lines was approved by the regional ethical review board in Stockholm, Sweden (Regionala EtikprÃ¶vningsnÃ¤mnden, Stockholm, EPN).</p><p class="pptt" id="biaoti28552">### References</p><ol class="npwtwok"><li>Boyden, S. (1962). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13872176" target="_blank">The chemotactic effect of mixtures of antibody and antigen on polymorphonuclear leucocytes.</a> <em>J Exp Med</em> 115: 453-466.</li><li>Briese, M., Saal, L., Appenzeller, S., Moradi, M., Baluapuri, A. and Sendtner, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26464439" target="_blank">Whole transcriptome profiling reveals the RNA content of motor axons.</a> <em>Nucleic Acids Res</em> 44(4): e33.</li><li>Campenot, R. B. (1977). <a href="http://www.ncbi.nlm.nih.gov/pubmed/270699" target="_blank">Local control of neurite development by nerve growth factor.</a> <em>Proc Natl Acad Sci U S A</em> 74(10): 4516-4519.</li><li>Holt, C. E. and Schuman, E. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24183017" target="_blank">The central dogma decentralized: new perspectives on RNA function and local translation in neurons.</a> <em>Neuron</em> 80(3): 648-657.</li><li>Kolde R. (2019). <a href="https://cran.r-project.org/web/packages/pheatmap/pheatmap.pdf" target="_blank">R-package: pheatmap.</a> Pretty Heatmaps.</li><li>Konopka T. (2018). <a href="https://cran.r-project.org/web/packages/umap/umap.pdf" target="_blank">R-package: umap.</a> Uniform Manifold Approximation and Projection.</li><li>Nichterwitz, S., Chen, G., Aguila Benitez, J., Yilmaz, M., Storvall, H., Cao, M., Sandberg, R., Deng, Q. and Hedlund, E. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27387371" target="_blank">Laser capture microscopy coupled with Smart-seq2 for precise spatial transcriptomic profiling.</a> <em>Nat Commun</em> 7: 12139.</li><li>Nichterwitz, S., Benitez, J. A., Hoogstraaten, R., Deng, Q. and Hedlund, E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29130192" target="_blank">LCM-Seq: A Method for Spatial Transcriptomic Profiling Using Laser Capture Microdissection Coupled with PolyA-Based RNA Sequencing.</a> <em>Methods Mol Biol </em>1649: 95-110.</li><li>Nijssen, J., Aguila, J., Hoogstraaten, R., Kee, N. and Hedlund, E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30540963" target="_blank">Axon-Seq Decodes the Motor Axon Transcriptome and Its Modulation in Response to ALS.</a> <em>Stem Cell Reports</em> 11(6): 1565-1578.</li><li>R Core Team. (2018). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria. <a href="https://www.R-project.org/" target="_blank">https://www.R-project.org/.</a> </li><li>Rotem, N., Magen, I., Ionescu, A., Gershoni-Emek, N., Altman, T., Costa, C. J., Gradus, T., Pasmanik-Chor, M., Willis, D. E., Ben-Dov, I. Z., Hornstein, E. and Perlson, E. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28300211" target="_blank">ALS along the axonsâexpression of coding and noncoding RNA differs in axons of ALS models.</a> <em>Sci Rep</em> 7: 44500.</li><li>Saal, L., Briese, M., Kneitz, S., Glinka, M. and Sendtner, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25246652" target="_blank">Subcellular transcriptome alterations in a cell culture model of spinal muscular atrophy point to widespread defects in axonal growth and presynaptic differentiation.</a> <em>RNA</em> 20(11): 1789-1802.</li><li>Taylor, A. M., Blurton-Jones, M., Rhee, S. W., Cribbs, D. H., Cotman, C. W. and Jeon, N. L. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16094385" target="_blank">A microfluidic culture platform for CNS axonal injury, regeneration and transport.</a><em> Nat Methods</em> 2(8): 599-605.<br/></li></ol>