<body>
            <h1>Optimized Protocol for the Incorporation of FDAA (HADA Labeling) for <em>in situ</em> Labeling of Peptidoglycan </h1>
            <p><strong>Authors:</strong> Katharina Peters, Manuel Pazos, Michael S. VanNieuwenhze, Waldemar Vollmer</p>
            <h2>Abstract</h2>
            <p>The essential peptidoglycan (PG) layer surrounds the cytoplasmic membrane in nearly all bacteria. It is needed to maintain the shape of the cell and protect it from lysis due to high turgor. Growth of the PG layer is a complex process that involves the activities of PG synthases and hydrolases during elongation and cell division. PG growth sites can be labeled by the recently developed fluorescent D-amino acid (FDAA) probes in a range of different bacteria. FDAAs are incorporated into PG by DD-transpeptidases (Penicillin-binding proteins, PBPs) or, if present, LD-transpeptidase (LDTs). Long-pulse <em>in situ</em> labeling of <em>E. coli</em> cells with the FDAA 7-hydroxycoumarincarbonylamino-D-alanine (HADA) is expected to result in a uniform label at the side wall of cells and enhanced label at cell division sites due to the intense PG synthesis. However, we observed reduced label at mid-cell when labeling <em>E. coli </em>cells with HADA. We reasoned that probe incorporated at cell division sites may be removed by PG hydrolases and modified the labeling protocol to better preserve PG-incorporated HADA for fluorescence microscopy. Here, we report the optimized HADA-labeling protocol by which cells retain an enhanced HADA signal at the division septum.</p>
            <p><strong>Keywords:</strong> Fluorescence D-amino acid (FDAA), HADA, Peptidoglycan synthesis, Transpeptidase, Labeling</p>
            <h2>Background</h2><p>The peptidoglycan (PG) sacculus is a mesh-like, essential macromolecule that encases the cytoplasmic membrane in most bacteria. It is composed of glycan chains made of alternating Î²-1, 4-linked <em>N</em>-acetylglucosamine and <em>N</em>-acetylmuramic acid residues connected via short peptides and is required to maintain the shape and osmotic stability of a bacterial cell (Typas <em>et al.</em>, 2011). Growing and dividing bacteria synthesize new PG and incorporate it into their PG sacculus, and these processes are targeted by important antibiotics such as the Î²-lactams and glycopeptides. To visualize the growth sites on the PG sacculus Miguel de Pedro developed an elegant method in which D-cysteine (D-Cys) is first homogeneously incorporated into PG of growing bacteria (by then unknown enzymes), followed by a chase period in the absence of D-Cys. PG sacculi were isolated, followed by the selective biotinylation of D-Cys residues and the visualization of biotin with nano-gold labeled antibodies by electron microscopy (de Pedro <em>et al.</em>, 1997). PG growth sites were characterized by regions of reduced label (when D-Cys became 'diluted' upon incorporation of new, label free PG during the chase) or without label (a zone of exclusively new PG). Although this method was crucial to determine the modes of PG segregation in model species such as <em>Escherichia coli</em> (de Pedro <em>et al.</em>, 1997) and <em>Caulobacter crescentus</em> (Aaron <em>et al.</em>, 2007), it was relatively cumbersome and used mainly in specialist laboratories. Over the last years, Erkin Kuru, Michael VanNieuwenhze and Yves Brun developed an ever-increasing palette of fluorescent D-amino acid (FDAA) probes that covalently label PG in cells and can therefore be used to visualize PG by fluorescence microscopy (Kuru <em>et al.</em>, 2012 and 2015; Hsu <em>et al.</em>, 2017). New 'rotor probes' of this series, rFDAA, produce fluorescence only upon incorporation into PG, enabling real-time visualization of probe incorporation and making it unnecessary to wash cells after the labeling procedure (Hsu <em>et al.</em>, 2019). Presumably FDAAs are incorporated into PG by PBPs and LDTs, as are other D-amino acids (Cava <em>et al.</em>, 2011; Lupoli <em>et al.</em>, 2011). FDAAs efficiently label sites of active PG synthesis and have therefore been successfully used to track PG synthesis sites in different bacterial species (Kuru <em>et al.</em>, 2012; Radkov<em> et al.</em>, 2018). They are easy to use and reliably label the PG in many bacteria and they have contributed to major discoveries in a variety of bacterial species (Radkov <em>et al.,</em> 2018), for example, the discovery of PG in <em>Chlamydia trachomatis</em> and its visualization (Liechti <em>et al.</em>, 2014 and 2016), the tracking of septal PG synthesis dynamics in <em>B. subtilis</em> (Bisson-Filho <em>et al.</em>, 2017) and the visualization of preseptal PG synthesis in <em>E. coli</em> (Pazos <em>et al.</em>, 2018). <br/><br/>We recently showed that copper(II) inhibits LD-TPases <em>in vivo</em> and <em>in vitro</em> (Peters <em>et al.</em>, 2018). Within this study we performed long-pulse <em>in situ</em> labeling of <em>E. coli</em> cells with the FDAA 7-hydroxycoumarincarbonylamino-D-alanine (HADA), which was previously shown to uniformly label cells (Kuru <em>et al.</em>, 2012 and 2015). Indeed, using the published procedure wild-type cells showed homogeneous label along the lateral wall but, unexpectedly, constricting cells showed a reduced signal at the septum. No signal could be detected in a strain lacking all six LD-TPases (Peters <em>et al.</em>, 2018). During cell growth PG synthesis takes place at the division septum due to the activities of PG synthases (PBP1B and PBP3) (de Pedro <em>et al.</em>, 1997; Bertsche <em>et al.</em>, 2006). We reasoned that the HADA signal at mid-cell was low despite the enhanced PG synthesis because the incorporated HADA might be removed by PG hydrolases during the time the cells were harvested and washed at neutral pH, resulting in the loss of label at the division septum. Indeed, labeling of chlamydial PG with fluorescent dipeptide required the inactivation of DD-carboxypeptidases by amplicillin (Liechti <em>et al.</em>, 2014). Here we optimized the long-pulse HADA labeling protocol to reduce the potential loss of HADA label by PG hydrolases. Specifically, we stopped cell growth and label incorporation by adding sodium citrate buffer pH 2.25 to the growing cells and washed the cells once with sodium citrate buffer at pH 3.0, followed by two washing steps with phosphate-buffered saline (PBS, pH 7.4), prior to processing the samples for fluorescence microscopy. With this procedure cells of wild-type and several mutants were homogeneously labeled at the side-wall and had enhanced signal at the septum of constricting cells. We conclude that the rapid incubation and washes of cells at acidic pH preserves the HADA label by preventing the removal of incorporated HADA by PG hydrolases (Amanuma and Strominger, 1980; Stefanova <em>et al.</em>, 2002), resulting in improved signal detection at cell division sites.</p>
        </body>

<p class="pptt" id="biaoti28587">## Materials and Reagents</p><ol class="npwtwok"><li>Sterile Petri dishes 92 Ã 16 mm with cams (SARSTEDT, catalog number: 82.1473)</li><li>Pipette tips 10 Âµl, 200 Âµl and 1,000 Âµl (STARLAB, catalog numbers: S1111-3700, S1113-1700, S1111-6701)</li><li>1.5 ml micro-tubes (SARSTEDT, catalog number: 72.690.001)</li><li>Sterile serological pipettes (5, 10 and 25 ml) (SARSTEDT, catalog numbers: 86.1253.001, 86.1254.001 and 86.1685.001)</li><li>Sterile 50 ml CELLSTAR tubes with blue screw cap (Greiner BIO-ONE, catalog number: 227261)</li><li>Concentric Luer-Lok Syringe 50 ml (BD Plastipak, BD Luer Lok, catalog number: 10636531)</li><li>0.22 Âµm syringe filter (PES membrane, sterile) (STARLAB, catalog number: E4780-1226)</li><li>Diagnostic microscopic slides (12 wells, 5.2 mm numbered) (Thermo Scientific, catalog number: 10028210)</li><li>Microscope slides SuperFrost plus (25 Ã 75 Ã 1 mm) (R. Langenbrinck Labor + Medizintechnik, catalog number: 03-0060)</li><li>Cover glasses, square (22 Ã 22 mm, thickness 1.5) (VWR, catalog number: 631-0125)</li><li>Semi-micro cuvettes (polystyrene, 10 Ã 4 Ã 45 mm) (SARSTEDT, catalog number: 67.742)</li><li>Milli-Q quality water (ddH<sub>2</sub>O) and distilled water</li><li>Peptone from casein, enzymatic digest (Sigma-Aldrich, catalog number: 82303)</li><li>Peptone from soybean, enzymatic digest (Fluka Analytical, catalog number: 90765)</li><li>Sodium chloride (NaCl) (VWR, catalog number: 27810.295)</li><li>Di-Potassium hydrogen phosphate (K<sub>2</sub>HPO<sub>4</sub>) (VWR, catalog number: 26931.263)</li><li>D-Glucose (anhydrous) (Melford, catalog number: G1400)</li><li>Bacto Agar (BD Diagnostics, catalog number: 214030)</li><li>Bacto Tryptone (BD Diagnostics, catalog number: 211699)</li><li>Yeast extract powder (Ohly KAT, catalog number: FllOHLYKAT)</li><li>FDAA 7-hydroxycoumarincarbonylamino-D-alanine (HADA) provided by Michael S. VanNieuwenhze, Indiana University (Kuru <em>et al.</em>, 2015) </li><li>Dimethylsulfoxide anhydrous (DMSO) (Invitrogen, catalog number: D12345)</li><li>Citric acid (anhydrous) (Sigma-Aldrich, catalog number: C2404)</li><li>Sodium hydroxide (NaOH) (VWR, catalog number: 28245.298)</li><li>10Ã PBS buffer with 0.017 M KH<sub>2</sub>PO<sub>4</sub>, 0.05 M Na<sub>2</sub>HPO<sub>4</sub>, 1.5 M NaCl, pH 7.4 (phosphate saline) (Lonza Walkersville INC, AccuGENE, catalog number: 51226)</li><li>16% Paraformaldehyde (formaldehyde) aqueous solution (Electron Microscopy Science, catalog number: 15710-S)</li><li>SB Molecular Biology Grade Agarose (Severn Biotech LtD., catalog number: 30 10 50)</li><li>Tryptic soy broth (TSB) and Tryptic soy agar (TSA) (see Recipes)</li><li>Miller Luria-Bertani (LB) medium and LB agar (see Recipes)</li><li>HADA stock solution (see Recipes)</li><li>10Ã Sodium citrate buffer, pH 2.25 (see Recipes)</li><li>1Ã sodium citrate buffer, pH 3.0 (see Recipes)</li><li>1Ã Phosphate buffered saline pH 7.4 (PBS) (see Recipes)</li><li>3% Paraformaldehyde solution (see Recipes)</li><li>Agar slides for microscopic imaging (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28588">### Equipment</p><ol class="npwtwok"><li>Sterile, flexible and disposable Inoculating loops (VWR, catalog number: 612935P)</li><li>Glass beaker, 600 ml and 1,000 ml (PYREX, catalog numbers: 1000/18D and 1000/22D, respectively)</li><li>Premium single Cell Analytical Balance (Kern, model: ABT 120-4NM)</li><li>IKA RH basic 2 magnetic stirrer (IKA, catalog number: 0003339002)</li><li>pH meter (Cole-Parmer, Jenway, model: 3510, catalog number: 351001)</li><li>Pipettes (Gilson, catalog numbers: F167300 and F167500)</li><li>Ultra Low Temperature Freezer VIP<sup>TM</sup> Series -86 Â°C (SANYO, catalog number: MDF-U73V)</li><li>Comfort TP 1410 table-high, under worktop fridge (Liebherr, catalog number: 4016803034339)</li><li>Vortex (IKA, model: Minishaker MS2)</li><li>Microwave MS106 0.6 FT (Matsui)</li><li>50 L Laboratory Incubator with Digital Control (GenLab incubator: INC/50/DIG)</li><li>Microcentrifuge accuSpin Micro 17R, refrigerated (Fisher Scientific, catalog number: 11526873)</li><li>Nikon Eclipse Ti microscope (Nikon Plan Fluor Ã 100/1.30 Oil Ph3 DLL objective) equipped with a Photometrics/Cool SNAP HQ2CCD camera using the phase contrast and DAPI channel (filter set: Chroma 49000, excitation at 350/50 nm, emission 460/50 nm)</li><li>Autoclave<br/></li></ol><p class="pptt" id="biaoti28589">### Software</p><ol class="npwtwok"><li>ImageJ (Rasband W.S./U. S. NIH, Bethesda, Maryland, USA/<a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>) </li><li>MetaMorph<sup>Â®</sup> Version 7.0 (Microscopy Automation and Image Analysis Software/Molecular DevicesLLC/<a href="https://www.moleculardevices.com/products/cellular-imaging-systems/acquisition-and-analysis-software/metamorph-microscopy" target="_blank">https://www.moleculardevices.com/products/cellular-imaging-systems/acquisition-and-analysis-software/metamorph-microscopy</a>)<br/></li></ol><p class="pptt" id="biaoti28590">### Procedure</p><ol class="npwone"><li>Prepare the following solutions in advance<br/><ol class="npwtwo"><li>TSB (or LB) and TSB (LB) agar plates (see Recipes).</li><li>50 mM HADA stock solution (see Recipes).</li><li>10 and 1x sodium citrate buffers, 1x PBS and 3% paraformaldehyde (see Recipes).<br/><em>Note: Filter and cool 10x and 1x sodium citrate buffers, 1x PBS and 3% paraformaldehyde solution on ice prior to use. </em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Growing the cells<br/><ol class="npwtwo"><li>Defrost glycerol stock solutions of the bacterial strains on ice. Streak the strains BW25113 (Datsenko and Wanner, 2000), D456 (Edwards and Donachie, 1993) and CS703-1 (Meberg <em>et al.</em>, 2001) on TSB agar plates using an inoculation loop and incubate the plates overnight at 37 Â°C. </li><li>Inoculate 25 ml of TSB (or LB) media with a single bacterial colony and incubate overnight at 37 Â°C. </li><li>Dilute the overnight cultures to an OD<sub>578</sub> of 0.1 in TSB (or LB) media and grow at 37 Â°C until they reach an exponential OD<sub>578 </sub>of 0.4.</li></ol><br/></li><li>Labeling the cells<br/><ol class="npwtwo"><li>Dilute the cells to an OD<sub>578</sub> of 0.1 in a final volume of 500 Î¼l of pre-warmed (37 Â°C) TSB (or LB) in a sterile 1.5 ml micro-tube. </li><li>Add 2.5 Âµl of the 50 mM stock solution of fluorescent derivative of D-alanine (HADA) to the sample to achieve a final concentration of 250 Î¼M and incubate with shaking for 30 min at 37 Â°C. </li></ol><br/></li><li>Stop bacterial growth and washing steps to remove the excess of HADA<br/><ol class="npwtwo"><li>Add one-tenth of the final volume (50 Âµl) of ice cold 10x sodium citrate buffer (pH 2.25) to the growing cultures. Place the samples on ice until the centrifuge (step 2) is ready to use.</li><li>Centrifuge the samples for 2 min at 16,200 <em>x g</em> at 4 Â°C. </li><li>Remove the supernatant carefully and resuspend the cell pellet in 1.5 ml ice cold 1x sodium citrate buffer (pH 3.0) by gentle pipetting. </li><li>Centrifuge the samples for 2 min at 16,200 <em>x g</em> at 4 Â°C. Remove the supernatant carefully and resuspend the cells in 1.5 ml ice cold 1x PBS (pH 7.4) by gentle pipetting and centrifuge the samples again. </li><li>Repeat the washing step with ice cold 1x PBS (pH 7.4) for another time. Remove the supernatants of the samples carefully without touching the cell pellets.<br/><em>Note: It was reported that HADA is virtually non-fluorescent in acidic pH or non-buffered water and cells imaged at a pH above 7.0 showed the maximum brightness of HADA labeling (Kuru et al., 2015). Therefore, the last two washing steps with 1Ã PBS (pH 7.4) are crucial.</em> </li></ol><br/></li><li>Cell fixation<br/><ol class="npwtwo"><li>Resuspend the cell pellets in 12.5 Âµl 1Ã PBS and 12.5 Î¼l of 3% paraformaldehyde (see Recipes) for cell fixation. </li><li>Mix well by pipetting and store on ice or 4 Â°C until microscopic analysis. <br/><em>Note: Microscopic analysis of the labeled cells should be performed immediately.</em> </li></ol><br/></li><li>Microscopic analysis<br/><ol class="npwtwo"><li>Pipette 3 Î¼l of the fixed cells in a well of the prepared microscopic agarose slide (see Recipes) and place a square shaped cover glass carefully on top and press gently. </li><li>Analyze the labeled cells with a Nikon Eclipse Ti microscope (Nikon Plan Fluor Ã 100/1.30 Oil Ph3 DLL objective) equipped with a Photometrics/Cool SNAP HQ2CCD camera using the phase contrast and DAPI channel (filter set: Chroma 49000, excitation at 350/50 nm, emission 460/50 nm). </li><li>Use exposure times of 100 ms for phase contrast and 1 s for fluorescence images.<br/><em>Note: Using this optimized protocol, all analyzed cells were completely labeled, a stronger signal at the septum of constricting cells was revealed (see Figure 1). The addition of 10x sodium citrate buffer to the growing cells, the initial wash with 1x sodium citrate buffer at pH 3.0, and the rapid processing of the samples on ice improved the removal of unincorporated HADA and led to a decreased background fluorescence signal and improved signal detection.<br/></em> <br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190726/20190726015324_7981.jpg" width="100"/><br/><strong>Figure 1. PG labeling of E. coli cells using the fluorescent D-amino acid HADA.</strong> A. BW25113 (Datsenko and Wanner, 2000), D456 (Edwards and Donachie, 1993) and CS703-1 (Meberg <em>et al.</em>, 2011) cells were incubated with 250 ÂµM HADA for 30 min (long pulse labeling). One-tenth of the final volume of 10x sodium citrate buffer (pH 2.25) was added to the cell cultures prior to short washing steps performed at 4 Â°C. First the cells were washed once with 1x sodium citrate buffer (pH 3.0) followed by two washing steps with phosphate buffer saline (PBS, pH 7.4). The cells were fixed with 3% paraformaldehyde and visualized by phase contrast (left) and fluorescence microscopy using the DAPI channel (right). Scale bars, 2 Î¼m. B. Optional</li></ol></li></ol><p class="pptt" id="biaoti28591">### Data analysis</p><p class="ppzz">ImageJ was used to crop the images to prepare figures and MetaMorph (v7) was used to quantify the fluorescence signals at the septum and at the sidewalls of ~50 cells from at least two independent experiments. The fluorescence intensity per unit of area at septal and sidewall positions can be quantified, the background subtracted, and the ratio of the sidewall over the septum signal density calculated. The values should be given as mean Â± SD; the Studentâs t-test should be used to determine the significance of the results (Peters <em>et al.</em>, 2018).</p><p class="pptt" id="biaoti28592">### Notes</p><p class="ppzz">Experiments were performed in TSB and LB media without changing the reproducibility of the results.</p><p class="pptt" id="biaoti28593">### Recipes</p><ol class="npwtwok"><li>Tryptic soy broth (TSB) and Tryptic soy agar (TSA)<br/>To prepare 1 L of TSB weigh the following reagents:<br/>17 g Peptone from casein<br/>3 g Peptone from soybean<br/>5 g Sodium chloride <br/>2.5 g Di-Potassium hydrogen phosphate (K<sub>2</sub>HPO<sub>4</sub>) <br/>2.5 g D-Glucose (anhydrous)<br/><ol class="npwthree"><li>Dissolve all ingredients in a final volume of 1 L distilled water under gentle stirring using a magnetic stirrer</li><li>Adjust the pH to 7.3 and autoclave the solution for 15 min at 121 Â°C</li><li>In order to prepare TSA add 15 g/L of agar to 1 L of TSB before autoclaving, cool the solution to a temperature of 48 Â°C after autoclaving and pour the agarose plates under sterile conditions</li></ol></li><li>Miller Luria-Bertani (LB) medium and LB agar<br/>To prepare 1 L of LB, weigh the following reagents:<br/>10 g Bacto Tryptone<br/>10 g Sodium chloride<br/>5 g Yeast extract <br/><ol class="npwthree"><li>Dissolve all ingredients in a final volume of 1 L distilled water under gentle stirring using a magnetic stirrer</li><li>Adjust the pH to 7.3 and autoclave the solution for 30 min at 121 Â°C</li><li>In order to prepare LB agar add 15 g/L of agar to 1 L of LB medium before autoclaving, cool the solution to a temperature of 48 Â°C after autoclaving and pour the agarose plates under sterile conditions</li></ol></li><li>HADA stock solution<br/><ol class="npwthree"><li>Dilute 2.1 mg of powdered HADA (328.7 g/mol) in 127.77 Âµl DMSO to create a 50 mM stock solution</li><li>Mix the stock solution well by pipetting and aliquot it in 10 Âµl portions in sterile 1.5 ml micro-tubes</li><li>Sterilization of stock solutions is not required. Store at -80 Â°C and protected from light</li><li>Thaw the stock solution rapidly at room temperature shortly before use</li><li>Avoid excessive cycles of thawing and refreezing, or long exposure at room temperature and light<br/><em>Note: Dry powders of FDAAs are stable at -20Â°C or lower temperature for several months (Kuru et al., 2015). In our studies we stored HADA at -80Â°C, either as dry powder or as stock solution in DMSO for several months and without signs of decomposition or altered labeling efficacy.</em> </li></ol></li><li>10x Sodium citrate buffer, pH 2.25 (500 ml)<br/>805 mM citric acid<br/>410 mM NaOH<br/>1.19 M NaCl<br/><ol class="npwthree"><li>Weigh 77.5 g citric acid (anhydrous) and 34.9 g of sodium chloride</li><li>Add 206 ml of NaOH solution (<em>c</em> = 1 mol/L) and top the volume up to 500 ml with MilliQ water</li><li>Dissolve completely by gentle stirring and check the pH with a pH meter. It should be 2.25</li></ol></li><li>1x sodium citrate buffer, pH 3.0 (500 ml) <br/>80.5 mM citric acid<br/>41 mM NaOH<br/>119 mM NaCl<br/><ol class="npwthree"><li>Weigh 7.74 g citric acid (anhydrous) and 3.49 g sodium chloride</li><li>Add 206 ml NaOH solution (<em>c</em> = 0.1 mol/L) and top the volume up to 500 ml with MilliQ water</li><li>Dissolve completely by gentle stirring and check the pH with a pH meter. It should be 3.0</li></ol></li><li>1x Phosphate buffered saline pH 7.4 (PBS)<br/>To prepare 50 ml of 1x PBS:<br/><ol class="npwthree"><li>Pipette 5 ml of commercially available 10x PBS solution into a sterile Greiner tube</li><li>Add 45 ml of Milli Q water and mix well<br/><em>Note: Filter the prepared solutions (10x and 1x sodium citrate buffer and the 1Ã PBS) into sterile 50 ml Greiner tubes. Use sterile single use 50 ml syringes and 0.22 Âµm syringe filters (PES membrane) for filtration. Cool the solutions on ice prior to use.</em> </li></ol></li><li>3% Paraformaldehyde solution <br/>To prepare 1,000 Âµl of 3% paraformaldehyde for cell fixation:<br/><ol class="npwthree"><li>Pipette 187.5 Âµl of 16% paraformaldehyde solution in a sterile 1.5 ml micro-tube and add 812.5 Âµl filtered 1Ã PBS solution</li><li>Vortex and cool down on ice prior use</li></ol></li><li>Agar slides for microscopic imaging<br/>To prepare an agarose solution with a final concentration of 1% w/v:<br/><ol class="npwthree"><li>Dissolve 1 g agarose in 100 ml 1Ã PBS by microwaving</li><li>Bring the temperature of the solution to ~60 Â°C</li><li>Spot ~150 Î¼l agarose solution in every second of the 12 wells on a diagnostic microscopic slide</li><li>Place a microscope slide on top and press gently down in order to get a flat, even and thin agarose pad</li><li>Let the pad solidify before you carefully remove the top microscope slide</li><li>Air-dry the agarose pad for ~1 min prior to use</li></ol><em></em><em>Note: The cells do not settle on the agarose pad, if the agarose slide is not dried properly. This can make imaging of the cells more difficult. </em><em></em><br/></li></ol><p class="pptt" id="biaoti28594">### Acknowledgments</p><p class="ppzz">This work reports some of the methods used to demonstrate that copper inhibits peptidoglycan LD-transpeptidases suppressing Î²-lactam resistance due to bypass of penicillin-binding proteins (Peters <em>et al.,</em> 2018). This work was funded by Wellcome Trust Grant 101824/Z/13/Z; the United Kingdom Medical Research Council within the Antimicrobial Resistance Cross-Council Initiative Collaborative Grant MR/N002679/1 (to W.V.) and NIH Grant GM113172 (to M.S.V.).</p><p class="pptt" id="biaoti28595">### Competing interests</p><p class="ppzz">The authors declare that they have no conflict of interest or competing interest.</p><p class="pptt" id="biaoti28596">### References</p><ol class="npwtwok"><li>Aaron, M., Charbon, G., Lam, H., Schwarz, H., Vollmer, W. and Jacobs-Wagner, C. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17501919" target="_blank">The tubulin homologue FtsZ contributes to cell elongation by guiding cell wall precursor synthesis in <em>Caulobacter crescentus</em>.</a> <em>Mol Microbiol</em> 64(4): 938-952.</li><li>Amanuma, H. and Strominger, J. L. (1980). <a href="https://www.ncbi.nlm.nih.gov/pubmed/7002918" target="_blank">Purification and properties of penicillin-binding proteins 5 and 6 from <em>Escherichia coli</em> membranes.</a> <em>J Biol Chem</em> 255(23): 11173-11180.<br/></li><li>Bertsche, U., Kast, T., Wolf, B., Fraipont, C., Aarsman, M. E., Kannenberg, K., von Rechenberg, M., Nguyen-DistÃ¨che, M., den Blaauwen, T., HÃ¶ltje, J. V. and Vollmer, W. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16803586" target="_blank">Interaction between two murein (peptidoglycan) synthases, PBP3 and PBP1B, in <em>Escherichia coli</em>.</a> <em>Mol Microbiol</em> 61(3): 675-690.</li><li>Bisson-Filho, A. W., Hsu, Y. P., Squyres, G. R., Kuru, E., Wu, F., Jukes, C., Sun, Y., Dekker, C., Holden, S., VanNieuwenhze, M. S., Brun, Y. V. and Garner, E. C. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28209898" target="_blank">Treadmilling by FtsZ filaments drives peptidoglycan synthesis and bacterial cell division.</a> <em>Science</em> 355(6326): 739-743</li><li>Cava, F., de Pedro, M. A., Lam, H., Davis, B. M. and Waldor, M. K. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21792174" target="_blank">Distinct pathways for modification of the bacterial cell wall by non-canonical D-amino acids.</a> <em>EMBO J </em>30(16): 3442-3453.</li><li>Datsenko, K. A. and Wanner, B. L. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10829079" target="_blank">One-step inactivation of chromosomal genes in <em>Escherichia coli</em> K-12 using PCR products.</a> <em>Proc Natl Acad Sci U S A </em>97(12): 6640-6645.</li><li>de Pedro, M. A., Quintela, J. C., Holtje, J. V. and Schwarz, H. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9139895" target="_blank">Murein segregation in <em>Escherichia coli</em>.</a> <em>J Bacteriol </em>179(9): 2823-2834.</li><li>Edwards, D. H. and Donachie, W. D. (1993). Construction of a triple deletion of penicillin-binding proteins 4, 5, 6 in <em>Escherichia coli</em>. In: <em>Bacterial Growth and Lysis</em>. De Pedro, M. A., HÃ¶ltje, J. V. and LÃ¶ffelhardt, W. (Eds.). Boston, Springer: 369-374. Doi: https://doi.org/10.1007/978-1-4757-9359-8_44.</li><li>Hsu, Y. P., Hall, E., Booher, G., Murphy, B., Radkov, A. D., Yablonowski, J., Mulcahey, C., Alvarez, L., Cava, F., Brun, Y. V., Kuru, E. and VanNieuwenhze, M. S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30804500" target="_blank">Fluorogenic D-amino acids enable real-time monitoring of peptidoglycan biosynthesis and high-throughput transpeptidation assays.</a> <em>Nat Chem </em>11(4): 335-341.</li><li>Hsu, Y. P., Rittichier, J., Kuru, E., Yablonowski, J., Pasciak, E., Tekkam, S., Hall, E., Murphy, B., Lee, T. K., Garner, E. C., Huang, K. C., Brun, Y. V. and VanNieuwenhze, M. S. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28989665" target="_blank">Full color palette of fluorescent d-amino acids for <em>in situ</em> labeling of bacterial cell walls.</a> <em>Chem Sci </em>8(9): 6313-6321.</li><li>Kuru, E., Hughes, H. V., Brown, P. J., Hall, E., Tekkam, S., Cava, F., de Pedro, M. A., Brun, Y. V. and VanNieuwenhze, M. S. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23055266" target="_blank"><em>In situ</em> probing of newly synthesized peptidoglycan in live bacteria with fluorescent D-amino acids.</a><em> Angew Chem Int Ed Engl</em> 51(50): 12519-12523.</li><li>Kuru, E., Tekkam, S., Hall, E., Brun, Y. V. and Van Nieuwenhze, M. S. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25474031" target="_blank">Synthesis of fluorescent D-amino acids and their use for probing peptidoglycan synthesis and bacterial growth<em> in situ</em>.</a> <em>Nat Protoc</em> 10(1): 33-52.</li><li>Liechti, G. W., Kuru, E., Hall, E., Kalinda, A., Brun, Y. V., VanNieuwenhze, M. and Maurelli, A. T. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24336210" target="_blank">A new metabolic cell-wall labelling method reveals peptidoglycan in <em>Chlamydia trachomatis</em>.</a> <em>Nature</em> 506(7489): 507-510.</li><li>Liechti, G., Kuru, E., Packiam, M., Hsu, Y. P., Tekkam, S., Hall, E., Rittichier, J. T., VanNieuwenhze, M., Brun, Y. V. and Maurelli, A. T. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27144308" target="_blank">Pathogenic chlamydia lack a classical sacculus but synthesize a narrow, mid-cell peptidoglycan ring, regulated by MreB, for cell division.</a> <em>PLoS Pathog</em> 12(5): e1005590.</li><li>Lupoli, T. J., Tsukamoto, H., Doud, E. H., Wang, T. S., Walker, S. and Kahne, D. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21682301" target="_blank">Transpeptidase-mediated incorporation of D-amino acids into bacterial peptidoglycan.</a> <em>J Am Chem Soc</em> 133(28): 10748-10751.</li><li>Meberg, B. M., Sailer, F. C., Nelson, D. E. and Young, K. D. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11567017" target="_blank">Reconstruction of <em>Escherichia coli</em> mrcA (PBP 1a) mutants lacking multiple combinations of penicillin binding proteins.</a> <em>J Bacteriol</em> 183(20): 6148-6149.</li><li>Pazos, M., Peters, K., Casanova, M., Palacios, P., VanNieuwenhze, M., Breukink, E., Vicente, M. and Vollmer, W. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30504892" target="_blank">Z-ring membrane anchors associate with cell wall synthases to initiate bacterial cell division.</a> <em>Nat Commun</em> 9(1): 5090.</li><li>Peters, K., Pazos, M., Edoo, Z., Hugonnet, J. E., Martorana, A. M., Polissi, A., VanNieuwenhze, M. S., Arthur, M. and Vollmer, W. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30275297" target="_blank">Copper inhibits peptidoglycan LD-transpeptidases suppressing Î²-lactam resistance due to bypass of penicillin-binding proteins.</a> <em>Proc Natl Acad Sci U S A</em> 115(42): 10786-10791.</li><li>Radkov, A. D., Hsu, Y. P., Booher, G. and VanNieuwenhze, M. S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29596002" target="_blank">Imaging bacterial cell wall biosynthesis.</a><em> Annu Rev Biochem</em> 87: 991-1014.</li><li>Stefanova, M. E., Davies, C., Nicholas, R. A. and Gutheil, W. G. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12044907" target="_blank">pH, inhibitor, and substrate specificity studies on <em>Escherichia coli </em>penicillin-binding protein 5.</a> <em>Biochim Biophys Acta</em> 1597(2): 292-300.</li><li>Typas, A., Banzhaf, M., Gross, C. A. and Vollmer, W. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22203377" target="_blank">From the regulation of peptidoglycan synthesis to bacterial growth and morphology.</a> <em>Nat Rev Microbiol</em> 10(2): 123-136.<br/></li></ol>