<body>
            <h1>QUEEN-based Spatiotemporal ATP Imaging in Budding and Fission Yeast</h1>
            <p><strong>Authors:</strong> Masak Takaine</p>
            <h2>Abstract</h2>
            <p>Yeasts have provided an exceptional model for studying metabolism and bioenergetics in eukaryotic cells. Among numerous metabolites, adenosine triphosphate (ATP) is a major metabolite that is essential for all living organisms. Therefore, a clearer understanding of ATP dynamics in living yeast cells is important for deciphering cellular energy metabolism. However, none of the methods currently available to measure ATP, including biochemical analyses and ATP indicators, have been suitable for close examinations of ATP concentrations in yeast cells at the single cell level. Using the recently developed ATP biosensor QUEEN, which is suitable for yeasts and bacteria, a protocol was described herein to visualize ATP concentrations in living budding and fission yeast cells. This simple method enables the easy and reliable examination of ATP dynamics in various yeast mutants, thereby providing novel molecular insights into cellular energy metabolism.</p>
            <p><strong>Keywords:</strong> ATP, Budding yeast, Fission yeast, QUEEN, Biosensor, Energy metabolism, Mitochondria, Live cell imaging</p>
            <h2>Background</h2><p>Adenosine triphosphate (ATP) is one of the major and indispensable metabolites for all living organisms. In addition to being useful âenergy currencyâ for various cellular processes, ATP serves as an intracellular and extracellular signaling molecule as well as a phosphate donor for protein phosphorylation (Lehninger <em>et al.</em>, 2010). Recent studies suggested that ATP itself solubilizes protein as a âbiological hydrotropeâ (Patel <em>et al.</em>, 2017) and influences the aggregation state of intracellular proteins (Pu <em>et al.</em>, 2019; Sridharan <em>et al.</em>, 2019). Due to the obvious importance of ATP and its wide involvement in many cellular events, the spatial distributions and temporal dynamics of ATP in living organisms need to be clarified in order to elucidate energy metabolism and signal transductions at the molecular level. To date, a number of methods have been developed to measure ATP concentrations <em>in vitro</em>. However, these biochemical methods have only a limited ability to examine ATP dynamics <em>in vivo</em> because of their insufficient time resolution relative to the rapid turnover of ATP (less than one minute) (Mortensen <em>et al.</em>, 2011; Takaine <em>et al.</em>, 2019). Moreover, although these biochemical analyses detect the average ATP level of a cell population, they are incapable of showing the intracellular or intratissue distribution of ATP. <br/><br/>To overcome these difficulties, several types of ATP indicators have been developed for the direct visualization of ATP in living organisms (Dong and Zhao, 2016). Among them, genetically encodable ATP indicators are advantageous because their introduction into a cell population is non-invasive and highly specific. A series of fluorescence resonance energy transfer (FRET)-based genetically encoded ATP indicators, named âthe ATeamâ, was developed for the first time a decade ago by the pioneering work of Imamuraâs group (Imamura <em>et al.</em>, 2009). The ATeam comprises the ATP-binding domain of the Îµ subunit of bacterial F<sub>o</sub>F<sub>1</sub> ATP synthase and flanking cyan and yellow fluorescent proteins (FPs), and is now widely and successfully used to monitor intracellular ATP concentrations in various types of cells ranging from plants to mammals. However, the ATeam composed of two FPs may be susceptible to proteolysis and, thus, yield immature sensor molecules because of differences in the maturation times of the two FPs. These features manifest when the ATeam is expressed in fast-growing microorganisms, in which protein turnover is very rapid, such as yeasts or bacteria, and become problematic because the presence of a dysfunctional sensor inevitably affects the FRET signal. To circumvent the drawbacks associated with ATeam, the second-generation ATP indicator âQUEENâ (quantitative evaluator of cellular energy) was recently developed by Imamuraâs group (Yaginuma <em>et al.</em>, 2014). QUEEN is a dual-excitation ratiometric ATP biosensor composed of a single FP and, thus, is more resistant to degradation and matures more rapidly than ATeam, yielding the reliable quantification of ATP, even in actively proliferating bacterial cells.<br/><br/>Yeasts have provided an exceptional model for studying metabolism and bioenergetics in eukaryotic cells. Yeast carbon metabolism has long been studied because of its industrial importance for producing many valuable chemicals (yeast metabolites) (Borodina and Nielsen, 2014). Energy metabolism in yeasts also provides a tractable model for studying energy metabolism in tumor cells because they are similar in that they synthesize the majority of ATP through glycolysis and not oxidative phosphorylation, even in the presence of oxygen, and this is known as âaerobic glycolysisâ or âthe Warburg effectâ (Diaz-Ruiz <em>et al.</em>, 2011). Therefore, a close examination of ATP dynamics in living yeast cells has been of great significance, but waits to be embarked due to the absence of an appropriate ATP biosensor.<br/><br/>In a recent study, QUEEN was applied to both the budding yeast <em>Saccharomyces cerevisiae</em> and fission yeast <em>Schizosaccharomyces pombe</em> for the first time and intracellular ATP levels and its dynamicity were visualized in wild-type cells (Takaine <em>et al.</em>, 2019). The use of QUEEN enables the easy and reliable examination of ATP dynamics in living yeast cells, providing novel insights into cellular energy metabolism. The protocols used to observe and analyze the QUEEN fluorescence signal in yeast cells have been described herein. The constitutive and strong expression of QUEEN did not induce any noticeable defects in cell growth, meiosis, or sporulation. Thus, ATP concentrations in various yeast mutants under a number of conditions may be observed, as well as ATP in wild-type cells, using the same method. Moreover, this method employs a conventional wide-field fluorescence microscope and free image processing software for image calculations, and, thus, is accessible to a broad range of researchers.</p>
        </body>

<p class="pptt" id="biaoti36073">## Materials and Reagents</p><ol class="npwtwok"><li>35-mm glass-bottomed dish (IWAKI, catalog number: 3971-035, No. 1.5 thickness)</li><li>Cell culture dish, sterilized (SANSEI MEDICAL CO., LTD., catalog number: 01-013)</li><li>Pipette tips (Watson<sup>Â®</sup> Bio Lab, catalog numbers: 123R-755YS [200 Âµl], 123R-757CS [1,000 Âµl])</li><li>Sterile culture tube (16 ml, polypropylene) (EVERGREEN, catalog number: 222-2393-080)</li><li>Toothpicks (6.5-18 cm) (any brand)<br/><em>Note: Sterilize before use. Reusable until they break.</em> </li><li>Budding yeast cells expressing QUEEN-2m:<br/><em>e.g.</em>, MTY3261 (<em>MATa his3â1::3ÃpRS303-PTEF1-QUEEN-2m-TCYC1 leu2â0 ura3â0</em>) from Takaine <em>et al.</em>, 2019. This strain was derived from a slightly modified BY4741 parental strain (<em>MATa his3â1 leu2â0 ura3â0</em>).</li><li>Fission yeast cells expressing QUEEN-2m:<br/><em>e.g.</em>, KSP3769 (<em>h- leu1-32::Ptif51-QUEEN-2m::leu1+</em>) from Ito <em>et al.</em>, 2019.<br/><em>Note: Both yeast strains expressing QUEEN-2m and plasmids for expressing QUEEN-2m in budding yeast are available from the Yeast Genetic Resource Centre Japan (YGRC, </em><a href="http://yeast.nig.ac.jp/yeast/" target="_blank"><em>http://yeast.nig.ac.jp/yeast/</em></a><em>).</em> </li><li>Concanavalin A (Sigma-Aldrich, catalog number: C-7275)</li><li>Soybean lectin (Sigma-Aldrich, catalog number: L-1395)</li><li>Yeast nitrogen base w/o amino acids (YNB) (Invitrogen, catalog number: Q300-09)</li><li>Yeast extract (BD, catalog number: 212750)</li><li>Bacto-peptone (BD, catalog number: 211677)</li><li>D(+)-Glucose (FUJIFILM Wako, catalog number: 049-31165) </li><li>2-Deoxy-D-glucose (2DG) (FUJIFILM Wako, catalog number: 046-06483)<br/><em>Note: 2DG is a potent inhibitor of glycolysis and has been used to deplete ATP in yeast cells (Serrano, 1977; Xu and Bretscher, 2014; Takaine </em><em>et al.</em><em>, 2019).</em> </li><li>Agar (FUJIFILM Wako, catalog number: 010-08725)</li><li>Potassium hydrogen phthalate (FUJIFILM Wako, catalog number: 163-03822)</li><li>Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O (Kishida Chemical Co., Ltd., catalog number: 000-72535)</li><li>Ethanol (Kishida Chemical Co., Ltd., catalog number: 140-28553)</li><li>NH<sub>4</sub>Cl (FUJIFILM Wako, catalog number: 017-02995)</li><li>Uracil (FUJIFILM Wako, catalog number: 212-00062)</li><li>Myo-inositol (FUJIFILM Wako, catalog number: 096-00285)</li><li>L-Glutamic acid (FUJIFILM Wako, catalog number: 072-00501)</li><li>Adenine sulfate (FUJIFILM Wako, catalog number: 018-19613)</li><li>L-Leucine (PEPTIDE INSTITUTE INC., catalog number: 2713)</li><li>L-Histidine (FUJIFILM Wako, catalog number: 082-00683)</li><li>L-LysineÂ·HCl (PEPTIDE INSTITUTE INC., catalog number: 2714)</li><li>L-Methionine (PEPTIDE INSTITUTE INC., catalog number: 2715)</li><li>L-Phenylalanine (PEPTIDE INSTITUTE INC., catalog number: 2717)</li><li>L-Serine (PEPTIDE INSTITUTE INC., catalog number: 2719)</li><li>L-Threonine (PEPTIDE INSTITUTE INC., catalog number: 2720)</li><li>MgCl<sub>2</sub>Â·6H<sub>2</sub>O (FUJIFILM Wako, catalog number: 132-00175)</li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (FUJIFILM Wako, catalog number: 033-25035)</li><li>KCl (FUJIFILM Wako, catalog number: 163-03545)</li><li>Na<sub>2</sub>SO<sub>4</sub> (FUJIFILM Wako, catalog number: 195-03341)</li><li>H<sub>3</sub>BO<sub>3</sub> (FUJIFILM Wako, catalog number: 021-02195)</li><li>MnSO<sub>4</sub>Â·5H<sub>2</sub>O (FUJIFILM Wako, catalog number: 139-00825)</li><li>ZnSO<sub>4</sub>Â·7H<sub>2</sub>O (FUJIFILM Wako, catalog number: 268-00405)</li><li>FeCl<sub>3</sub>Â·6H<sub>2</sub>O (FUJIFILM Wako, catalog number: 095-00875)</li><li>Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O (FUJIFILM Wako, catalog number: 198-02471)</li><li>KI (FUJIFILM Wako, catalog number: 166-03971)</li><li>CuSO<sub>4</sub>Â·5H<sub>2</sub>O (FUJIFILM Wako, catalog number: 031-04411)</li><li>Citric acid (FUJIFILM Wako, catalog number: 038-05521)</li><li>Nicotinic acid (FUJIFILM Wako, catalog number: 142-01232)</li><li>Biotin (TOKYO CHEMICAL INDUSTRY CO., LTD., catalog number: B0463)</li><li>Sodium pantothenate (FUJIFILM Wako, catalog number: 198-05651)</li><li>100x Amino acids mix (see Recipes)</li><li>100x L-Leucine stock (see Recipes)</li><li>50x Uracil stock (see Recipes)</li><li>SC-His liquid medium (see Recipes)</li><li>YPD agar plate (see Recipes)</li><li>YELA agar plate (see Recipes)</li><li>100x 5 Low-supplements (see Recipes)</li><li>50x Salt stock (see Recipes)</li><li>1,000x Vitamins (see Recipes)</li><li>10,000x Minerals (see Recipes)</li><li>EMM liquid medium (see Recipes)</li><li>2-Deoxy-D-glucose (2DG) medium (see Recipes)</li><li>Glass-bottomed dish coated with concanavalin A (see Recipes)</li><li>Glass-bottomed dish coated with soybean lectin (see Recipes)<br/></li></ol><p class="pptt" id="biaoti36074">### Equipment</p><ol class="npwtwok"><li>Inverted fluorescent microscope (Nikon, Eclipse Ti-E) equipped with:<ol class="npwthree"><li>A 100x objective lens (Nikon, Apo TIRF 100x Oil DIC N2/NA 1.49)</li><li>A high-pressure mercury lamp (Nikon, Intensilight C-HGFIE 130W)</li><li>An electron multiplying charge-coupled device camera (Andor-Oxford Instruments, iXon3 DU897E-CS0-#BV80)</li><li>A FITC filter set (Nikon, Ex465-495/DM505/BA515-555)</li><li>A custom-made Ex409 filter set (Semrock, Ex393-425/DM506/BA516-556)</li><li>Neutral density (ND) filters (Nikon, ND4 [MBN21804] and ND8 [MBN21808])</li></ol></li><li>Bench-top aspirator (INTEGRA, VACUSIP)</li><li>Inverted routine microscope (Nikon, Eclipse Ts2) equipped with a 40Ã objective lens (Nikon, CFI Achromat LWD ADL 40XF/NA 0.55, catalog number: MRP46402)</li><li>Autoclave (TOMY SEIKO CO., LTD, LBS-325)</li><li>Pure water system (Merck Millipore, Milli-Q<sup>Â®</sup> Integral 15)</li><li>Erlenmeyer flasks (50 ml) (HARIO, catalog number: 82-0144)</li><li>Silicone stopper (for a 50-ml flask) (Shin-Etsu Polymer, SILICOSEN T22)</li><li>Incubator (TAITEC, INCUBATE BOX M-200F)</li><li>Culture rotator (TAITEC, RT-50)</li><li>Rotary shaker (NISSIN, NX-20)</li><li>Pipettes (BM EQUIPMENT Co., LTD, PipetPAL<sup>Â®</sup>, model: PAL-200 [20-200 Âµl], PAL-1000 [100-1,000 Âµl])<br/></li></ol><p class="pptt" id="biaoti36075">### Software</p><ol class="npwtwok"><li>NIS-Element AR ver. 4.30.01 (Nikon, <a href="https://www.microscope.healthcare.nikon.com/ja_JP/" target="_blank">https://www.microscope.healthcare.nikon.com/ja_JP/</a>)</li><li>Fiji ver. 2.0.0-rc-69/1.52n (Fiji contributors [Schindelin <em>et al.</em>, 2012], <a href="http://fiji.sc/" target="_blank">http://fiji.sc/#</a>)</li><li>Microsoft<sup>Â®</sup> Excel for Mac 2011 ver. 14.7.3 (Microsoft, <a href="https://products.office.com/ja-jp/excel" target="_blank">https://products.office.com/ja-jp/excel</a>)</li><li>KaleidaGraph ver. 4.5.1 (HULINKS, <a href="https://www.hulinks.co.jp/software/stat_graph/kaleida" target="_blank">https://www.hulinks.co.jp/software/stat_graph/kaleida</a>)<br/></li></ol><p class="pptt" id="biaoti36076">### Procedure</p><ol class="npwone"><li>Cell preparation<ol class="npwtwo"><li>Streak yeast cells from a frozen stock using a pipette tip on a YPD (for <em>S. cerevisiae</em>) or YELA (for <em>S. pombe</em>) agar plate, and incubate at 30 Â°C for 1-2 days.</li><li>Preparation of the preculture: <ol class="npwthree"><li>For budding yeast, inoculate 2 ml of SC-H with a loopful of cells in a 16-ml culture tube using a sterilized toothpick, and culture at 30 Â°C overnight with rotation at approximately 40 rpm using a rotator.</li><li>For fission yeast, inoculate 2 ml of EMM with a loopful of cells in a 16-ml culture tube using a sterilized toothpick, and culture at 30 Â°C overnight with shaking at 150 rpm using a rotary shaker.</li></ol></li><li>Preparation of the main culture:<ol class="npwthree"><li>For budding yeast, dilute 0.15 ml of the overnight culture with 2.85 ml SC-H (1:20) in a 16-ml culture tube, and culture at 30 Â°C with rotation at approximately 40 rpm.</li><li>For fission yeast, dilute 0.5 ml of the overnight culture with 9.5 ml EMM in a 50-ml flask (1:20), and culture at 30 Â°C with shaking at 150 rpm. </li></ol></li><li>Grow yeast cells to the mid-log phase. Approximately 3 h is needed for budding yeast and approximately 5 h for fission yeast under the experimental conditions described herein. The dilution ratio and incubation time may be optimized according to individual culture systems.</li><li>Dilute the cell culture with the same medium by 10-fold and place 100 Âµl of cells onto a 35-mm glass-bottomed dish coated by concanavalin A for <em>S. cerevisiae</em> or soybean lectin for <em>S. pombe</em> (see Recipes).</li><li>Wait for 5 min, remove most of the medium with gentle aspiration, and wash cells immobilized on the glass surface with 300 Âµl of fresh medium three times.</li><li>Observe the yeast cells immobilized on the glass-bottomed dish with a 40x objective lens using an inverted routine microscope to check cell density.</li><li>Repeat Steps A2-A3 if cells are too sparse.</li><li>Fill the dish with 4-5 ml of medium and incubate the dish in a room for fluorescent microscopy (maintained at 25 Â°C) for at least 30 min before microscopic observations.</li><li>Optional: To deplete intracellular ATP, replace the medium with 300 Âµl of 2DG medium three times. Acquire images of QUEEN fluorescence within 10 min.</li></ol><em>Note: The dish is filled with an excess amount of medium relative to the cell volume in order to minimize changes in the chemical composition of the medium by the consumption of nutrients or by secretory components during observations. It is important to maintain the medium in a dish at a constant and stable temperature because the affinity of QUEEN for ATP depends on temperature (Yaginuma </em><em>et al.</em><em>, 2014): higher temperatures result in lower QUEEN ratios, while lower temperatures lead to higher QUEEN ratios.</em><br/><br/></li><li>Image acquisition<ol class="npwtwo"><li>Power on the fluorescent microscope system at least 1 h before imaging to reach a stable and constant temperature of approximately 25 Â°C.</li><li>Observe the yeast cells immobilized on the glass-bottomed dish with a 100x objective lens using an inverted fluorescence microscope by bright-field imaging and focus the cells.</li><li>Take a bright-field image and two fluorescence images of QUEEN using FITC (ex480 image) and Ex409 (ex409 image) filter sets from a single <em>z</em>-plane.</li><li>Regarding time-lapse imaging, reduce the intensity of the excitation light and exposure times (see Table 1 for details) to prevent the photobleaching of QUEEN fluorescence and minimize damage to cells.</li><li>Save the combined images (bright-field and QUEEN) as an nd2 file.</li></ol><em>Note: The bright-field image is dispensable for calculating the QUEEN ratio, but is useful for examinations of cell viability or cell boundaries. Red fluorescence (emission wavelength &gt; 620 nm) is compatible with QUEEN fluorescence. In single time-point imaging, collect images of cells from several fields of view for each experimental condition, giving a sufficient sample size for a </em><em>quantitative analysis. Observations of 3-5 fields yields n &gt; 100 cells in total are appropriate. The parameters used for the fluorescence microscopy of QUEEN are summarized in Table 1.</em><br/><br/><strong>Table 1. Parameters for the fluorescence microscopy of QUEEN</strong><br/><img alt="" class="layerphoto" height="303" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190804/20190804230754_4588.jpg" width="600"/><br/>*Exposure time for the acquisition of images of QUEEN expressed in budding/fission yeast. <br/></li></ol><p class="pptt" id="biaoti36077">### Data analysis</p><p class="ppzz">In the following sections, menu commands and options in Fiji software are expressed in <strong>bold</strong> font.</p><ol class="npwone"><li>Calculation of the QUEEN ratio (single time-point imaging)<ol class="npwtwo"><li>Open an nd2 file using Fiji software as a hyperstack file (multi-channels and one time frame) and split the hyperstack into each channel image (one bright-field image, two QUEEN images): <strong>Image -&gt; Color -&gt; Split Channels</strong> (Figure 1A, raw image).</li><li>Subtract the background (pixel values in an area outside the cells) of QUEEN images using the rolling-ball algorithm (Figure 1A, background-subtracted): <strong>Process -&gt; Subtract Background</strong>. Set the <strong>rolling ball radius</strong> 20 (for <em>S. cerevisiae</em>) or 50 (for <em>S. pombe</em>) pixels and keep all the check boxes unchecked.</li><li>Convert QUEEN images to a signed 32-bit floating-point grayscale: <strong>Image -&gt; Type -&gt; 32-bit</strong> </li><li>Assess the maximum and minimum thresholds of the images using the modified IsoData algorithm (default algorithm): <strong>Image -&gt; Adjust -&gt; Threshold</strong>. Check the <strong>Dark background</strong> check box and click <strong>Auto</strong> and <strong>Apply</strong>. Check <strong>Set background pixels to NaN</strong> to set background (non-thresholded) pixels to the <em>NaN</em> (Not a Number) value (Figure 1A, thresholded).</li><li>Divide the pixel values of the ex409 image by those of the ex480 image to calculate the QUEEN ratio at each pixel: <strong>Process -&gt; Image calculator, Image1: ex409 image, Operation: Divide, Image2: ex480 image</strong>.</li><li>Apply the rainbow smooth look-up table (or any other table if preferable) to the ratio image to express the QUEEN ratio in pseudocolor: <strong>Image -&gt; Lookup Tables -&gt; Rainbow Smooth</strong>.</li><li>Adjust the display range to include the minimum and the maximum values of the QUEEN ratios in the image: <strong>Image -&gt; Adjust -&gt; Brightness/Contrast -&gt; Set</strong>. Set <strong>Minimum displayed value</strong> 0.5 and <strong>Maximum displayed value</strong> 3, respectively, for example (Figure 1B, ratio image).</li><li>Save the ratio image and the bright-field image as a tiff file: <strong>File -&gt; Save as...</strong> </li><li>Optional: steps 1-7 may be automated by running the following macro: <strong>Plugins -&gt; Macros -&gt; Installâ¦</strong>, select and open stack2ratio.jim macro. Open an nd2 file and apply <strong>Run</strong> on the macro window.<br/><br/></li></ol><strong>QUEEN ratio calculation macro (stack2ratio.jim)</strong><br/><span style="font-family:Courier New;">// </span><em><span style="font-family:Courier New;">Macro starts here</span></em><br/><span style="font-family:Courier New;"> title = getTitle();</span><br/><span style="font-family:Courier New;"> run("Split Channels");</span><br/><span style="font-family:Courier New;"> c1 = "C1-" + title;    // channel #1: bright-field image</span><br/><span style="font-family:Courier New;"> c2 = "C2-" + title;    // channel #2: ex480 image</span><br/><span style="font-family:Courier New;"> c3 = "C3-" + title;    // channel #3: ex409 image</span><br/><br/><span style="font-family:Courier New;"> selectWindow(c2);</span><br/><span style="font-family:Courier New;"> run("Subtract Background...", "rolling=20 stack"); </span><br/><span style="font-family:Courier New;"> run("32-bit");</span><br/><span style="font-family:Courier New;"> setAutoThreshold("Default dark"); </span><br/><span style="font-family:Courier New;"> run("NaN Background", "stack");</span><br/><br/><span style="font-family:Courier New;"> selectWindow(c3);</span><br/><span style="font-family:Courier New;"> run("Subtract Background...", "rolling=20 stack"); </span><br/><span style="font-family:Courier New;"> run("32-bit");</span><br/><span style="font-family:Courier New;"> setAutoThreshold("Default dark");</span><br/><span style="font-family:Courier New;"> run("NaN Background", "stack");</span><br/><br/><span style="font-family:Courier New;"> imageCalculator("Divide create stack",c3,c2);</span><br/><span style="font-family:Courier New;"> run("Rainbow Smooth");</span><br/><span style="font-family:Courier New;"> setMinAndMax(0.5, 3);// </span><br/><span style="font-family:Courier New;"> rename("Ratio-" + title);</span><br/><span style="font-family:Courier New;"> // Macro ends here</span><br/><br/><em>Note: The optimal rolling ball radius depends on the spatial resolution of the imaging system (0.16 Âµm/pixel in our system) and the average size of yeast cells examined. Therefore, readers need to adjust the rolling ball radius according to their own imaging system by trial and error.</em><br/><br/><img alt="" class="layerphoto" height="102" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190729/20190729200213_0843.jpg" width="100"/><br/><strong>Figure 1. Generation of a QUEEN ratio image.</strong> A. Image processing of images of budding yeast cells expressing QUEEN. Right panels show the line profile of pixel values along dotted lines indicated in the ex409 images. B The bright-field image and QUEEN ratio images calculated from the ex480 and ex409 images shown in (A) expressed in three different lookup tables. White scale bar = 5 Âµm.<br/><br/></li><li>Calculation of the QUEEN ratio (time-lapse imaging)<ol class="npwtwo"><li>Open the nd2 file using Fiji software as a hyperstack file (multi-channels and multiple time frames).</li><li>By choosing a <strong>Rectangle</strong> selection tool (see Figure 2A), draw the region of interest (ROI) to include a cell (and the progeny cells if necessary) throughout the time course.</li><li>Create a new hyperstack file composed of the ROI: <strong>Image -&gt; Duplicate</strong>. Check the <strong>Duplicate</strong> <strong>hyperstack</strong> check box and preserve all <strong>Channels (c)</strong> and <strong>Frames (t)</strong>.</li><li>Split the hyperstack into each channel stack (containing time frames): <strong>Image -&gt; Color -&gt; Split Channels</strong>.</li><li>Subtract the background using the rolling-ball algorithm as described in step A2. Process all the images in the stack.</li><li>Convert the QUEEN stacks to a signed 32-bit floating-point grayscale: <strong>Image -&gt; Type -&gt; 32-bit</strong> </li><li>Display the last time frame: <strong>Image -&gt; Stacks -&gt; Set Slice</strong>. Enter the last slice number.<br/><em>Note: Against a stack of 32-bit images, the maximum and minimum thresholds are calculated using the currently displayed image (slice), and not calculated for each image. Assessments of thresholds using the last time frame were appropriate under the experimental conditions described herein.</em> </li><li>Create thresholded stacks as described in step A4.</li><li>Calculate and save the QUEEN ratio stack as described in steps A5-A8.<br/><br/></li></ol></li><li>Measurement of the mean QUEEN ratio in individual cells (single time-point imaging)<ol class="npwtwo"><li>Open the QUEEN ratio image using Fiji software (Figure 2B).</li><li>By choosing the <strong>Rectangle</strong> or <strong>Polygon</strong> selection tool (Figure 2A), draw the ROI to include a living cell and register the ROI in the ROI manager: <strong>Analyze -&gt; Tools -&gt; ROI manager</strong>. Click <strong>Add</strong> or press <em>t</em> to register the ROI (Figure 2C).</li><li>Repeat step 2.</li><li>Set the measurement: <strong>Analyze -&gt; Set Measurements...</strong>, check the <strong>Mean Gray Value</strong> check box.</li><li>Select all ROI stored in the ROI manager and measure the mean of pixel values (QUEEN ratios) in ROIs: <strong>ROI manager -&gt; Deselect -&gt; Measure</strong> </li><li>Save the data table as a text file in the comma-separated value (csv) format: Activate the <strong>Results</strong> window (Figure 2D), then <strong>File -&gt; Save as...</strong><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190729/20190729200259_6204.jpg" width="145"/><br/><strong>Figure 2. Measurement of the mean QUEEN ratio in yeast cells.</strong> Measurements in fission yeast cells are shown as an example. A. <strong>Rectangle</strong> or <strong>Polygon</strong> selection tool. B. A QUEEN ratio image of fission yeast cells. Outlines of cells are drawn using the rectangle (cell #1-4) or polygon (cell #5-7) selection tool. C. <strong>ROI manager</strong>. D. <strong>Results</strong> window. White scale bar = 5 Âµm.<br/><br/></li></ol><em>Note: Since background pixels are assigned to NaN values and, thus, ignored in the calculation of the mean of pixel values, a precise drawing of the cell outline is not necessarily required. The mean QUEEN ratio in pixels corresponding to the inside of a cell represents the cytoplasmic ATP level of the cell.</em><br/><br/></li><li>Measurement of the mean QUEEN ratio in a cell (time-lapse imaging)<ol class="npwtwo"><li>Open the QUEEN ratio stack using Fiji software.</li><li>By choosing the <strong>Rectangle</strong> or <strong>Polygon</strong> selection tool, draw the ROI to include a cell and register the ROI in the ROI manager.</li><li>Repeat step 2 if necessary.</li><li>Select the ROI stored in the ROI manager and measure the mean of pixel values in the ROI throughout time series: <strong>ROI manager -&gt; More -&gt; Multi Measure</strong>. Check the <strong>Measure all the slices</strong> and <strong>One row per slice</strong> check boxes.</li><li>Save the result as described in step C6.<br/><br/></li></ol></li><li>Data analysis<ol class="npwtwo"><li>Import data into Microsoft<sup>Â®</sup> Excel, and calculate averages and standard deviations.</li><li>Calculate the <em>P</em>-value using an unpaired one-tailed Welchâs <em>t</em>-test in Excel software, and evaluate the significance of differences between two sets of data.</li><li>Plot data as a dot plot using KaleidaGraph software. An example of a dot plot is shown in Figure 3.<br/><em>Note: The presentation of data as a dot plot appears to be the most effective way to convey distribution characteristics in extensive detail. Data generally follow a normal distribution even though there are some outliers.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190729/20190729200324_4254.jpg" width="229"/><br/><strong>Figure 3. Examples of data analyses.</strong> A data set of measurements of the mean QUEEN ratio in budding yeast cells is represented in four different types of graphs. Sample size = 191 cells, the mean value = 1.67, variance = 0.0228, and the standard deviation (SD) = 0.151. A. Bar graph. Data are shown as the mean of the population Â± 1Ã SD (error bar). B. Dot plot. Each dot corresponds to the mean QUEEN ratio inside a cell. The horizontal bar indicates the mean of the population. C. Box plot. The plot shows the 75<sup>th</sup> and 25<sup>th</sup> percentiles of the data (interquartile range) as the upper and lower edges of the box. A horizontal bar in the box indicates the median of the population. Whiskers and circles indicate the 1.5Ã interquartile range and outliers, respectively. D. Histogram. Data are shown as probability density. A normal distribution with the same mean and variance to the data is shown as a red curve.</li></ol></li></ol><p class="pptt" id="biaoti36078">### Notes</p><ol class="npwtwok"><li>The mean QUEEN ratio in a cell is calculated based on (1) the intracellular ATP concentration, (2) the affinity of QUEEN for ATP, and (3) the settings and conditions of the imaging system. Thus, many factors affect the QUEEN ratio including the conditions of the cell culture, the batch of the culture medium, temperature (as described in Procedure A), the power output of the fluorescence excitation lamp, and the conditions of optical filters (excitation, emission, and neutral density) for fluorescence microscopy. These factors vary slightly day to day and are difficult to strictly control. Therefore, comparisons of data obtained from experiments performed on the same day and not pooling data from experiments performed on different days are recommended.</li><li>The readers need to optimize the imaging parameters of fluorescence microscopy for their own imaging systems. Since an excitation light of approximately 409 nm is cytotoxic and may affect the fluorescence emission spectrum of QUEEN, its intensity and exposure time need to be minimized. Adjustments to imaging parameters in order for the mean QUEEN ratio in cells depleted of ATP by 2DG, corresponding to the minimum mean QUEEN ratio, to be calculated as 0.7-0.8 are recommended. Under preferable conditions, the dynamic range of QUEEN-2m (the ratio between the maximum and minimum mean QUEEN ratios) is expected to reach 3.0-fold (Yaginuma <em>et al.</em>, 2014; Takaine <em>et al.</em>, 2019).</li><li>In addition to the calculation of the QUEEN ratio, measurements of the mean QUEEN ratio (the steps C1-C5) may also be automated by running a macro. Moreover, steps A1-A8 and C1-C5 in the Data analysis may be batch-processed: multiple nd2 files are automatically and sequentially processed by running a batch macro. However, the automatic drawing of cell outlines and batch processing are both beyond the scope of this study and, thus, were omitted. These macros are available from the author upon request.<br/></li></ol><p class="pptt" id="biaoti36079">### Recipes</p><ol class="npwtwok"><li>100x Amino acids mix<ol class="npwthree"><li>Mix L-glutamic acid 10 g, L-lysine 12 g, L-methionine 4 g, L-phenylalanine 5 g, L-serine 38 g, L-threonine 20 g, and Myo-inositol 0.6 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and store at room temperature</li></ol></li><li>100x Leucine stock<ol class="npwthree"><li>Dissolve L-leucine 11 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and store at room temperature</li></ol></li><li>50x Uracil stock<ol class="npwthree"><li>Dissolve uracil 2 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and store at room temperature</li></ol></li><li>SC-His (according to Hanscho <em>et al.</em>, 2012)<ol class="npwthree"><li>Mix YNB 3.4 g, glucose 20 g, 100x amino acid mix 10 ml, 100x leucine stock 10 ml, and 50x uracil stock 20 ml in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and store at room temperature</li></ol></li><li>YPD agar plate<ol class="npwthree"><li>Mix yeast extract 10 g, glucose 20 g, Bacto-peptone 20 g, and agar 20 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min </li><li>Pour the medium into a cell culture dish (approximately 20 ml/dish)</li><li>Solidify the plates for 2-3 days before use, wrap in plastic wrap, and store at room temperature</li></ol></li><li>YELA agar plate<ol class="npwthree"><li>Mix yeast extract 10 g, glucose 20 g, adenine sulfate 0.04 g, and agar 20 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min </li><li>Pour the medium into a cell culture dish (approximately 20 ml/dish)</li><li>Solidify the plates for 2-3 days before use, wrap in plastic wrap, and store at room temperature</li></ol></li><li>100x 5 Low-supplements<ol class="npwthree"><li>Mix adenine sulfate 10 g, L-lysine 3 g, L-leucine 6 g, L-histidine 2 g, and uracil 2 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and store at room temperature</li></ol></li><li>50x Salt stock<br/><ol class="npwthree"><li>Mix MgCl<sub>2</sub>Â·6H<sub>2</sub>O 53.3 g, CaCl<sub>2</sub>Â·2H<sub>2</sub>O 0.735 g, KCl 50 g, and Na<sub>2</sub>SO<sub>4</sub> 2 g in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and store at room temperature</li></ol></li><li>1,000x Vitamins<br/>Mix nicotinic acid 0.1 g, myo-inositol 0.1 g, biotin 0.1 mg, and sodium pantothenate 10 mg in 100 ml of absolute ethanol and store at -20 Â°C<br/><em>Note: These components do not completely dissolve.</em> </li><li>10,000x minerals<br/><ol class="npwthree"><li>Dissolve H<sub>3</sub>BO<sub>3</sub> 0.05 g, MnSO<sub>4</sub>Â·5H<sub>2</sub>O 0.057 g, ZnSO<sub>4</sub>Â·7H<sub>2</sub>O 0.04 g, FeCl<sub>3</sub>Â·6H<sub>2</sub>O 0.02 g, Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O 0.016 g, KI 0.01 g, CuSO<sub>4</sub>Â·5H<sub>2</sub>O 0.004 g, and citric acid 0.1 g in 100 ml of deionized water</li><li>Sterilize by filtration and store at room temperature</li></ol></li><li>EMM<ol class="npwthree"><li>Mix potassium hydrogen phthalate 3 g, Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O 5.54 g, glucose 20 g, 50x salt stock 20 ml, and NH<sub>4</sub>Cl 5 g, and 10,000x minerals 0.1 ml in 1,000 ml of deionized water</li><li>Autoclave at 120 Â°C for 20 min and cool down</li><li>Add 1,000x vitamins 1 ml and store at 4 Â°C</li><li>Aliquot and prewarm before use</li></ol></li><li>2DG medium<br/>The same as for SC-His or EMM except for the addition of 3.28 g of 2DG (final 20 mM) instead of glucose</li><li>Glass-bottomed dish coated with concanavalin A<br/><ol class="npwthree"><li>Dissolve concanavalin A with sterilized deionized water at 2 mg/ml and aliquot. The solution may be stored at -20 Â°C</li><li>Place 100 Âµl of 2 mg/ml of concanavalin A solution on the glass surface of a glass-bottomed dish, and keep at room temperature for 5 min </li><li>Aspirate concanavalin A and wash the glass surface with 100 Âµl of sterilized deionized water three times</li><li>Dry the glass surface for 10 min before use. The dish may be stored at 4 Â°C</li></ol></li><li>Glass-bottomed dish coated with soybean lectin<br/><ol class="npwthree"><li>Dissolve soybean lectin with sterilized deionized water at 2 mg/ml and aliquot. The solution may be stored at 4 Â°C</li><li>Place 5 Âµl of 2 mg/ml lectin solution on the glass surface of a glass-bottomed dish, and spread the solution using the pipette tip</li><li>Dry the glass surface for 10 min before use. The dish may be stored at 4 Â°C</li></ol></li></ol><p class="pptt" id="biaoti36080">### Acknowledgments</p><p class="ppzz">This work has been supported by JSPS grants to the author (15K18525, 16H04781, and 19K06654). This work was also supported by a joint research program of the Institute for Molecular and Cellular Regulation, Gunma University, Japan. The protocol has been adapted from Takaine <em>et al.</em> (2019).</p><p class="pptt" id="biaoti36081">### Competing interests</p><p class="ppzz">The author declares no competing interests.</p><p class="pptt" id="biaoti36082">### References</p><ol class="npwtwok"><li>Borodina, I. and Nielsen, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24677744" target="_blank">Advances in metabolic engineering of yeast <em>Saccharomyces cerevisiae</em> for production of chemicals.</a> <em>Biotechnol J</em> 9(5): 609-620.</li><li>Diaz-Ruiz, R., Rigoulet, M. and Devin, A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20804724" target="_blank">The Warburg and Crabtree effects: On the origin of cancer cell energy metabolism and of yeast glucose repression.</a> <em>Biochim Biophys Acta</em> 1807(6): 568-576.</li><li>Dong, J. and Zhao, M. (2016). <a href="https://www.sciencedirect.com/science/article/abs/pii/S0165993615300674" target="_blank"><em>In-vivo</em> fluorescence imaging of adenosine 5â-triphosphate.</a> <em>TrAc Trends Anal Chem</em> 80: 190-203.</li><li>Hanscho, M., Ruckerbauer, D. E., Chauhan, N., Hofbauer, H. F., Krahulec, S., Nidetzky, B., Kohlwein, S. D., Zanghellini, J. and Natter, K. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22780918" target="_blank">Nutritional requirements of the BY series of <em>Saccharomyces cerevisiae</em> strains for optimum growth.</a> <em>FEMS Yeast Res</em> 12(7): 796-808.</li><li>Imamura, H., Nhat, K. P., Togawa, H., Saito, K., Iino, R., Kato-Yamada, Y., Nagai, T. and Noji, H. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19720993" target="_blank">Visualization of ATP levels inside single living cells with fluorescence resonance energy transfer-based genetically encoded indicators.</a> <em>Proc Natl Acad Sci U S A</em> 106(37): 15651-15656.</li><li>Ito, H., Sugawara, T., Shinkai, S., Mizukawa, S., Kondo, A., Senda, H., Sawai, K., Ito, K., Suzuki, S., Takaine, M., Yoshida, S., Imamura, H., Kitamura, K., Namba, T., Tate, S. I. and Ueno, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30846209" target="_blank">Spindle pole body movement is affected by glucose and ammonium chloride in fission yeast.</a> <em>Biochem Biophys Res Commun</em> 511(4): 820-825.</li><li>Lehninger, A., Nelson, D. and Cox, M. (2010). Principles of Biochemistry. CBS Publisher and Distributors. Delhi.</li><li>Mortensen, S. P., Thaning, P., Nyberg, M., Saltin, B. and Hellsten, Y. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21300753" target="_blank">Local release of ATP into the arterial inflow and venous drainage of human skeletal muscle: insight from ATP determination with the intravascular microdialysis technique.</a> <em>J Physiol</em> 589(Pt 7): 1847-1857.</li><li>Patel, A., Malinovska, L., Saha, S., Wang, J., Alberti, S., Krishnan, Y. and Hyman, A. A. (2017). <a href="https://science.sciencemag.org/content/356/6339/753" target="_blank">ATP as a biological hydrotrope.</a> <em>Science</em> 356(6339): 753-756.</li><li>Pu, Y., Li, Y., Jin, X., Tian, T., Ma, Q., Zhao, Z., Lin, S. Y., Chen, Z., Li, B., Yao, G., Leake, M. C., Lo, C. J. and Bai, F. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30472191" target="_blank">ATP-dependent dynamic protein aggregation regulates bacterial dormancy depth critical for antibiotic tolerance.</a> <em>Mol Cell</em> 73(1): 143-156 e144.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682.</li><li>Serrano, R. (1977). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=21792" target="_blank">Energy Requirements for Maltose Transport in Yeast.</a> <em>European Journal of Biochemistry</em> 80(1): 97-102.</li><li>Sridharan, S., Kurzawa, N., Werner, T., Gunthner, I., Helm, D., Huber, W., Bantscheff, M. and Savitski, M. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30858367" target="_blank">Proteome-wide solubility and thermal stability profiling reveals distinct regulatory roles for ATP.</a> <em>Nat Commun</em> 10(1): 1155.</li><li>Takaine, M., Ueno, M., Kitamura, K., Imamura, H. and Yoshida, S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30858198" target="_blank">Reliable imaging of ATP in living budding and fission yeast.</a> <em>J Cell Sci</em> 132(8). doi: 10.1242/jcs.230649.</li><li>Xu, L. and Bretscher, A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25308080" target="_blank">Rapid glucose depletion immobilizes active myosin V on stabilized actin cables.</a> <em>Curr Biol</em> 24(20): 2471-2479.</li><li>Yaginuma, H., Kawai, S., Tabata, K. V., Tomiyama, K., Kakizuka, A., Komatsuzaki, T., Noji, H. and Imamura, H. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25283467" target="_blank">Diversity in ATP concentrations in a single bacterial cell population revealed by quantitative single-cell imaging.</a> <em>Sci Rep</em> 4: 6522.<br/></li></ol>