<body>
            <h1>Looking through Brains with Fast Passive CLARITY: Zebrafish, Rodents, Non-human Primates and Humans</h1>
            <p><strong>Authors:</strong> Farzad Mortazavi, Alexander J. Stankiewicz, Irina V. Zhdanova</p>
            <h2>Abstract</h2>
            <p>Recently developed CLARITY (Clear Lipid-exchanged Acrylamide-hybridized Rigid Imaging/Immunostaining/In situ-hybridization-compatible Tis-sue-hYdrogel) technique renders the tissue transparent by removing lipids in the tissue, while preserving and stabilizing the cellular and subcellular structures. This provides effective penetration of diverse labeling probes, from primary and secondary antibodies to complementary DNA and RNA strands. Followed by high-resolution 3D imaging of neuronal cells and their projections in thick sections, tissue blocks, whole brains, or whole animals, CLARITY allows for superior quantitative analysis of neuronal tissue. Here, we provide our detailed protocol for PACT (Passive Clarity Technique) in brain tissue of diverse species, including human, non-human primate, rodents, and zebrafish. We describe the six principal steps: (1) Tissue fixation and preparation, (2) Passive lipid removal, (3) Immuno-labeling, (4) Optical clearing, (5) Imaging, (6) 3D visualization and quantification.</p>
            <p><strong>Keywords:</strong> CLARITY, Immunohistochemistry, Brain, Imaging, Whole-tissue staining, Adult neurogenesis, Zebrafish, Rat, Mice, Rhesus monkey, Non-human primate, Human</p>
            <h2>Background</h2><p>Optimization of tissue imaging techniques had to overcome several inherent problems, including lack of optical tissue transparency and spatial limits on antibody penetration. The solutions to these problems have evolved over time. Major advances in microscopy provided for superior 2D and 3D image resolution (Richardson and Lichtman, 2015; Whitehead <em>et al.</em>, 2017). Thin sections, less than 40 microns in thickness, allowed most of the antibodies to reach their target proteins. Combined with diverse fluorescent tags, these antibodies revealed tissue complexities beyond those known before. However, evaluation of a tissue block or whole organ divided into many thin sections remained far from trivial. Following the time-consuming process of cutting and staining individual sections, an inherent discontinuity of specimens required the development of sophisticated imaging reconstruction techniques for accurate quantification of cells and ability to follow their individual projections, <em>e.g.</em>, micro-optical sectioning tomography (MOST) (Li <em>et al.</em>, 2010). Finally, to increase tissue transparency, a number of strategies to âclearâ the tissue has been proposed, first by Werner Spalteholz as early as 1914 (Spalteholz, 1914) and by many other researchers thereafter (Efimova and Anokhin, 2009; Hama <em>et al.</em>, 2011; ErtÃ¼rk <em>et al.</em>, 2012; Ke <em>et al.</em>, 2013; Susaki <em>et al.</em>, 2014; Fumoto <em>et al.</em>, 2016). Despite the effectiveness of these strategies, there were limitations in tissue types and species in which they performed best, with some causing tissue shrinking (for review, Mano <em>et al.</em>, 2018). Other technical difficulties include the rate and volume at which antibodies penetrated into a thick cleared tissue block or section, or whole organ specimen for labeling of proteins, and adverse effects of time and/or exposure to light on fluorescence emission. <br/><br/>In 2013, Stanford researchers Kwanghun Chung and Karl Deisseroth developed a novel approach called CLARITY (Clear Lipid-exchanged Acrylamide-hybridised Rigid Imaging/Immunostaining/In situ-hybridization-compatible Tis-sue-hYdrogel) (Chung <em>et al.</em>, 2013; Chung and Deisseroth, 2013). By simultaneously removing lipids and infusing the entire protein structure with a hydrogel, CLARITY preserved the tissue architecture, proteins and nucleic acid molecules, while making a large tissue block or an entire organ optically transparent. Importantly, the removal of lipids using this method enhanced antibody penetration into the preserved tissue, facilitating immunohistochemical staining, allowing for more efficient and accurate quantitative analysis. The success of CLARITY is highlighted by its increasing popularity among neuroscientists and biologists studying diverse tissues and organs (Azaripour <em>et al.</em>, 2016; Mortazavi <em>et al.</em>, 2016; Jensen and Berg, 2017; Vigouroux <em>et al.</em>, 2017; Du <em>et al.</em>, 2018; Yu <em>et al.</em>, 2018). <br/><br/>Active use of CLARITY technique resulted its further optimization, including PACT (passive CLARITY technique) and PARS (perfusion assisted agent released in situ), or ACT-PRESTO (active cleaning technique pressure related efficient and stable transfer of macromolecules into organs) (Yang <em>et al.</em>, 2014; Tomer <em>et al.</em>, 2014; Lee <em>et al.</em>, 2016). These methods proved to be applicable to a diverse array of tissues, including the peripheral organs such as the liver, kidney, intestine and lung (Lee <em>et al.</em>, 2014; Font-Burgada <em>et al.</em>, 2015; Neckel <em>et al.</em>, 2016; Saboor <em>et al.</em>, 2016). While there are common features in CLARITY methodology, the processing and imaging of diverse tissues, organs or whole animals may differ between model organisms. Human tissues also require special considerations due to the high lipid content of human brain tissue, and often the prolonged post-mortem interval (PMI) that can affect the quality of tissue, and its fixation. <br/><br/>Here we share our protocols for using CLARITY to visualize a number of proteins of interest in brain tissue of several species, including zebrafish, rat, mouse, rhesus monkey, and human. We find the technique to be relatively simple to execute, highly efficient in clarifying whole zebrafish, individual brains, large brain tissue blocks or thick sections. We also find that our CLARITY protocol allows for using lower than earlier reported antibody concentrations to effectively reveal target proteins, enabling high-quality 3D visualization. In addition to earlier proposed semi-quantitative analysis of CLARITY-processed whole-brain zebrafish samples, based on fluorescence Intensity (Lindsey and Kaslin, 2017), we show that 3D analytical tools (<em>e.g.</em>, Fiji or Imaris) can provide accurate counts and morphological parameters of labeled cells, axons, dendrites, or any other quantitative immunohistochemical labeling. Together, we find CLARITY to be an exceptional tool for 3D visualization and quantification of brain tissue constituents, which can further be used in studies of neurogenesis, connectivity, and pathological brain conditions.</p>
        </body>

<p class="pptt" id="biaoti37621">## Materials and Reagents</p><ol class="npwone"><li>Tissue Specimens tested in this Protocol<ol class="npwtwo"><li>Zebrafish (Danio rerio): whole animal, brain (compatible with any age) </li><li>Rat (Rattus or Mus musculus): brain (compatible with any age) </li><li>Non-Human Primate (<em>e.g.</em>, Macaca mulatta): brain (compatible with any age) </li><li>Human: brain (anatomical donation, compatible with any age) </li></ol><em>Note: For optimal results, use PFA perfused rat (mouse) or non-human primate brains.</em> <br/><br/></li><li>CLARITY Supplies and Reagents<ol class="npwtwo"><li>1.5 ml Eppendorfs (USA Scientific, catalog number: 1615-5500)</li><li>50 ml Falcon<sup>TM</sup> Tubes (Fisher Scientific, catalog number: 14-432-22) </li><li>DWK Life Sciences Kimble<sup>TM</sup> 7 ml Solvent-Saver<sup>TM</sup> Scintillation Vials (Fisher Scientific, catalog number: 03-340-128) </li><li>Thermo Scientific<sup>TM</sup> Nalgene<sup>TM</sup> Rapid-Flow<sup>TM</sup> Sterile Disposable Bottle Top Filters with PES Membrane, 0.45 Âµm (Fisher Scientific, catalog number: 09-740-64A) </li><li>Razorblades (Fisher Scientific, catalog number: 12-640)</li><li>Phosphate Buffered Saline (PBS) 10x, Fisher BioReagents, pH 7.4 (Fisher Scientific, catalog number: BP3994) (store at room temperature) </li><li>Sodium hydroxide solution (1 N) (NaOH) (Fisher Scientific, catalog number: SS2661) (store at room temperature)</li><li>Formalin solution, neutral buffered 10% (Sigma-Aldrich, catalog number: HT501128-4L) (store at room temperature)</li><li>200 mg/L MS222 (see Recipes)<br/>Tricaine (MS222) (Sigma-Aldrich, catalog number: E10521-10G) (store at room temperature)</li><li>0.1 M PBS with 0.02% sodium azide (pH 7.4) (see Recipes)<br/>Sodium Azide (Sigma-Aldrich, catalog number: S2002) (store at room temperature)</li><li>Paraformaldehyde (PFA) Fixing solution (4% PFA, 1x PBS,) pH 7.4 (see Recipes)<br/>Paraformaldehyde (PFA) (Sigma-Aldrich, catalog number: 252549) (store at room temperature)</li><li>CLARITY Solution (pH 8.5) (see Recipes)<br/><ol class="npwthree"><li>Boric Acid (Sigma-Aldrich, catalog number: B7901)</li><li>Sodium dodecyl sulfate (SDS) (Sigma-Aldrich, catalog number: L3771-500G)</li><li>Lithium hydroxide monohydrate (Sigma-Aldrich, catalog number: 254274)</li></ol><em>Note: Acrylamide free solution reduces clearing time (Xu </em><em>et al.</em><em>, 2017). With optimal transcardiac perfusion, we find that acrylamide is not a necessary component to maintain tissue structure. </em> </li><li>0.05 M TBS (pH 8.0) (see Recipes)<br/>Tris Buffer Saline (TBS) (Sigma-Aldrich, catalog number: T6664) (store at room temperature)</li><li>2 M Hydrochloric acid (HCI) (see Recipes)<br/>Hydrochloric acid, 37% for analysis (HCI) (Acros Organics, catalog number: 450550025) (store at room temperature)</li><li>0.1 M boric acid (pH 8.5) (see Recipes)<br/><br/></li></ol></li><li>Immunohistochemistry Reagents<ol class="npwtwo"><li>Click-iT<sup>TM</sup> EdU Alexa Fluor<sup>TM</sup> 488 Flow Cytometry Assay Kit (Thermo Fisher Scientific, catalog number: C10420)<br/><ol class="npwthree"><li>CuSO<sub>4</sub> (store at 4 Â°C)</li><li>Fluorescent dye azide (store at -20 Â°C)</li><li>Reaction Buffer Additive (store at -20 Â°C)</li></ol><em>Note: See Lindsey </em><em>et al.</em><em>, 2017 for alternative reagents, if the Click-iT<sup>TM</sup> kit is not available.</em> </li><li>0.05 M TBS (pH 8.0) with 1% Triton X-100 (see Recipes)<br/>Triton X-100 (Sigma-Aldrich, catalog number: X100) (store at room temperature)</li><li>Antibodies (Tables 1 and 2)<br/><br/><strong>Table 1. Primary antibodies used in this Protocol<br/><img alt="" class="layerphoto" height="497" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730185216_9269.jpg" width="600"/><br/></strong><br/><strong>Table 2. Secondary antibodies used with this Protocol</strong><br/><img alt="" class="layerphoto" height="317" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730030126_8377.jpg" width="600"/><br/><br/></li><li>Sodium citrate buffer (store at room temperature, see Recipes) <br/>Tri-sodium citrate (dihydrate) (Fisher Scientific, catalog number: 78-101-KG)</li><li>50% formamide/50% 2x SSC (see Recipes)<br/>a.Formamide Certified ACS, Fisher Chemical (Fisher Scientific, catalog number: 75-12-7) (store at 4 Â°C)<br/>b.20x SSC Buffer (Ambion, catalog number: AM9763) (store at room temperature)</li><li>EdU labeling solution (see Recipes)<br/><br/></li></ol></li><li>Imaging Supplies and Reagents<ol class="npwtwo"><li>Aluminum foil (Sigma-Aldrich, catalog number: 326852)</li><li>Paint Brush </li><li>15 ml polypropylene conical tubes (Falcon, Fisher Scientific, catalog number: 352096)</li><li>Blu Tack (pliable adhesive that can be readily formed into any shape, Amazon.com)</li><li>Pelco Black Wall Glass Bottom Dishes (Ted Pella, catalog number: 14032-120)</li><li>Camco 25573 Bullseye Level (Amazon)</li><li>80% glycerol solution in 0.1 M PBS (see Recipes)<br/>Glycerol (Sigma-Aldrich, catalog number: G7893) (store at room temperature) </li></ol></li></ol><p class="pptt" id="biaoti37622">### Equipment</p><ol class="npwtwok"><li>Microscopes: <br/><ol class="npwthree"><li>Upright Stereoscope Microscope (<em>e.g.</em>, Zeiss)</li><li>Confocal Microscope (<em>e.g.</em>, Leica SP8 using an HC Fluotar 25x/0.96 W VISIR objective)</li></ol></li><li>Stirring Hot plate (<em>e.g.</em>, Fisher Scientific, catalog number: 6796220) </li><li>Chemical hood </li><li>Vibratome (<em>e.g.</em>, Pelco 102 Vibratome Sectioning System) </li><li>Rocker (orbital or tube rocker)</li><li>Incubator (37 Â°C) </li><li>Forceps (<em>e.g.</em>, Fisher Scientific, catalog number: 12000122)<br/></li></ol><p class="pptt" id="biaoti37623">### Software</p><ol class="npwtwok"><li>ImageJ/Fiji (NIH, Bethesda, Maryland), Bitplane by Imaris, LasX (Leica Software)</li></ol><p class="pptt" id="biaoti37624">### Procedure</p><p class="ppzz"><strong>Tissue Processing and Immunolabeling for CLARITY</strong><br/><em>Notes: Common notes to processing tissue from all the species described here:</em> </p><ol class="npwtwo"><li><em>All washing, clearing, and staining procedures include placing a vial or tube containing tissue onto a tube rocker for the entire period listed in the protocol. If using an orbital shaker, rock vigorously. Sample should be fully submerged and freely floating in solution while on a rocker. Anytime a fluorophore is present, cover sample in aluminum foil.</em> </li><li><em>Some tissues are more prone to volumetric changes (expansion) following clearing and glycerol incubation steps of the Protocol (Figure 1A). Measure volume of tissue/organ via volumetric displacement, pre- and post-CLARITY, if the dimension variables are of importance in the final analysis.</em> </li><li><em>All animal procedures described here are in accordance with international ethical standards. Prior to initiating the experiments, receive an approval from your Institutional Animal Care and Use Committee.</em><br/><br/></li></ol><p class="ppzz"><strong><em>Zebrafish</em></strong><br/>The optically clear zebrafish larvae have provided an outstanding vertebrate model for developmental genetics, neuroscience, cancer research, studies of metabolic and infectious diseases (Brothers <em>et al.</em>, 2011; Wolman and Granato, 2012; Palha <em>et al.</em>, 2013; Fior <em>et al.</em>, 2017; Kamel and Ninov, 2017; Migault <em>et al.</em>, 2018; Vanwalleghem <em>et al.</em>, 2018). As the larvae continue to mature, they develop pigment cells which limit the transparency and interfere with imaging. The use of transgenic animals lacking pigment (White <em>et al.</em>, 2008; Antinucci <em>et al.</em>, 2016) and the CLARITY technique allows for visualization in the whole adult zebrafish or its individual organs (Cronan <em>et al.</em>, 2015; FrÃ©taud <em>et al.</em>, 2017; Lindsey <em>et al.</em>, 2017; Kaufman <em>et al.</em>, 2018). <br/>  We use adult zebrafish to study adult neurogenesis. Each day, thousands of neural stem or progenitor cells are undergoing cell division in 16 neurogenic niches of adult zebrafish brain, as well as in the spinal cord and gut. We found that, in the brain, the cell division cycle follows a circadian pattern (Akle <em>et al.</em>, 2017). The majority of newborn cells eventually differentiate into specialized neurons (Zupanc <em>et al.</em>, 2005; Grandel <em>et al.</em>, 2006; Kaslin <em>et al.</em>, 2009; Chapouton <em>et al.</em>, 2010). CLARITY has proved to be an optimal strategy for quantification of stem cells, progenitor cells, and mature neurons, in the whole brain (see Video 1).<br/>  Zebrafish represents an excellent model to study the kinetics of adult neurogenesis and its changes in normal and pathological aging (Stankiewicz <em>et al.</em>, 2019). To follow the kinetics of stem cells, neural progenitors and their progeny, we use a pulse-chase technique, with BrdU pulse followed by EdU chase. The protocol below describes a procedure for triple labeling for the S-phase markers (BrdU and EdU) and M-phase marker (pH3). If labeling EdU-only, follow Day 1 Steps 1 and 7 and conduct a wash in 0.05 TBS (2 x 15 min) the next day. For BrdU only, follow Day 1 Steps 1-6 in and Days 2-9 Steps 3-7.<br/><br/><!--video3321-1å¼å§--></p><div class="videodiv" id="vd3321-1" style="height:480px;"></div><div class="videophoto" id="vp3321-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3321/3321-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. The 3D confocal imaging of S-phase marker, EdU, within a cleared adult zebrafish brain.</strong> Following a 2 h EdU (green) pulse (Dimensions: 2,000 Î¼m x 3,600 Î¼m x 1,100 Î¼m). <br/><em>Note: Green autofluorescence in the vasculature, especially prominent in the optic tectum and cerebellum.</em> </div><!--video3321-1ç»æ--><p><br/></p><ol class="npwone"><li>Preparation of zebrafish tissue for CLARITY<ol class="npwtwo"><li>Euthanize fish with overdose of MS222 (see Recipe 1) or place in ice-cold water until operculum movements cease. </li><li>This protocol can be used for imaging either whole adult zebrafish without dissection or individual organs and tissue sections. The timing of each procedure described below would increase based on the thickness of the specimen. To process the brain tissue, cut heads behind the gills with a new razor-blade. Avoid any compression or tearing of the CNS. </li><li>Fix heads overnight in 4% PFA in 1.5 ml Eppendorfs at 4 Â°C (see Recipe 2). </li><li>Dissect out brains under a stereomicroscope using forceps, as per Gupta and Mullins, 2010. </li><li>Transfer brains into a 1.5 ml Eppendorf tube containing 0.1 M PBS with 0.02% sodium azide (see Recipe 3) and store at 4 Â°C, until used. Change storage solution every 2 weeks. This way, samples can be stored for up to 6 months.<br/> <br/></li></ol></li><li>CLARITY processing of zebrafish brain (Figure 1A)<ol class="npwtwo"><li>Remove brains from the buffer (0.1 M PBS with 0.02% sodium azide) and transfer up to 6 brains into a 7 ml scintillation vial containing 5 ml of 0.05 M TBS (see Recipe 4). Wash for 1 h. </li><li>Replace TBS with CLARITY solution (see Recipe 5), using the same scintillation vial. </li><li>Incubate brains at 37 Â°C for 30 min. Keep on a rocker at room temperature for 4-6 days, until the tissue is completely transparent (Figures 1A). <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023203_1285.jpg" width="242"/><br/><strong>Figure 1. Optical tissue clearing in brains of diverse species.</strong> A. Zebrafish brain following fixation, clearing and glycerol incubation. B. Rat brain hemisphere, sagittal cut 1 mm thick, pre- (left) and post-clearing (right). C. Non-human primate (NHP) occipital lobe section (primary visual area, V1), 1 mm thick, pre- (left) and post-clearing (right). A, B in millimeters and C, inches on top. Brain orientations: A, anterior; P, posterior; D, dorsal; V, ventral. Note volumetric changes post clearing. <br/><br/></li></ol></li><li>Immunolabeling of proteins in CLARITY processed Zebrafish tissue<br/><strong><em>Day 1: BrdU/EdU Immunohistochemistry of whole zebrafish brain</em></strong><br/><em> Note: Brains will turn opaque again after the following steps, but will regain transparency later in the protocol, as per below.</em><br/><ol class="npwtwo"><li>Wash in 0.05 M TBS (2 x 1 h) to completely remove CLARITY solution from the tissue.</li><li>Incubate in 50% formamide/50% 2x SSC (see Recipe 6) for 1h45 min at 37 Â°C. </li><li>Wash in 0.05 M TBS (2 x 15 min).</li><li>Incubate in 2 M HCl (see Recipe 7) at 37 Â°C for 30 min. </li><li>Rinse in 0.1 M boric acid (see Recipe 8) for 8 min. </li><li>Wash in 0.05 M TBS (2 x 15 min).</li><li>Incubate in EdU Click-iT<sup>Â®</sup> reaction cocktail covered in aluminum foil for 30 min at room temperature and then 4 Â°C overnight (see Recipe 9).</li></ol><em>Note: For those antibodies that, unlike BrdU, do not require antigen retrieval (Steps 2-5), follow only Day 1 step 1 and immediately proceed to Days 2-9 protocol below.</em><br/><br/><strong><em>Days 2-9:</em></strong><em><strong> Antibody immunohistochemistry of whole zebrafish brain</strong></em><br/><ol class="npwtwo"><li>Wash in 0.05 M TBS (2 x 15 min).</li><li>Add primary antibody: Anti-BrdU Alexa Fluor 647 (Marker for S-phase proliferation; 1:200). </li><li>Incubate in the solution for 4 days at 4 Â°C. The sample should be fully submerged and freely floating in the solution during agitation on a rocker. Keep sample covered in aluminum foil. </li><li>Wash in 0.05 M TBS (2 x 1 h). </li><li>Incubate in secondary antibody donkey anti-rabbit Alexa Fluor 555 (1:750) in 0.05M TBS overnight at room temperature. </li><li>Wash in 0.05 M TBS at room temperature for 2 days to reduce non-specific staining. </li><li>For long term storage, keep in 0.05 M TBS at 4 Â°C and covered in aluminum foil.</li></ol>This basic protocol can be used with several antibodies in parallel. For example, we use Anti-phospho Histone H3 (Ser10) (pH3) (marker for mitosis; 1:500) or HuC/HuD (marker for new neurons; 1:500) antibodies simultaneously with staining for BrdU and EdU. To achieve this, we add pH3 antibody (or HuC/D antibody) at Days 2-9 Step 5. <br/><br/></li></ol><p class="ppzz"><em><strong>Mammals</strong></em><br/>While studying neurogenesis in mammalian embryo and in neurogenic niches of adult brain, lack of tissue transparency is the major obstacle (Stankiewicz <em>et al.</em>, 2019). CLARITY allows for deep tissue fluorescent imaging of numerous cells in whole brain specimens (Chung <em>et al.</em>, 2013). Importantly, CLARITY also allows for visualization of dendrite and axonal morphology, including changes in neuronal processes associated with neurological diseases, as well as with normal aging. For example, in Alzheimerâs disease, large scale cell death results in shorter, less branched apical dendrites (Buell and Coleman 1981) and the number of dendritic spines is reduced (el Hachimi and Foncin, 1990; Ferrer and Gullotta, 1990). Similarly, in Parkinsonâs disease, striatal neurons have truncated dendrites and a reduced number of dendritic spines (McNeill <em>et al.</em>, 1988). In contrast, moderate grades of Huntingtonâs disease show an increase in both dendritic branching and the number of spines, and in later stages of the disease truncated dendritic arbors and spine loss is observed (Ferrante <em>et al.</em>, 1991). In normal aging, <em>e.g.</em>, in older rhesus monkeys, dendrites of long-projecting neurons in Brodmann Area 46 become shorter and less complex, and dendrites in prefrontal cortex lose spines and synapses (Coskren <em>et al.</em>, 2015). Visualizing such changes in dendrite structure provides insights into the nature of the pathological conditions. However, quantification of dendrite structure from digital tissue images is currently a highly labor-intensive process, requiring extensive manual marking, and is extremely challenging to accomplish in large-scale projects (Helmstaedter <em>et al.</em>, 2013). CLARITY-processed tissues allow for optimizing 3D imaging and precise quantification of dendrite structure and morphology. As a result, large regions of interest or even whole brain structures can be analyzed in their entirety, while avoiding issues typical of sectioned brains, <em>e.g.</em>, interrupting axonal projections, turn or branches, or double counting of cells. Here we describe the protocol we followed while studying axonal morphology and connections in rodents (Figures 2 and 3) and microcolumnar structures in the brains of non-human primates (NHPs) (Figures 4 and 5).<br/><br/><strong><em>Rodents</em></strong> </p><ol class="npwone"><li>Preparation of rodent brain for CLARITY<br/><ol class="npwtwo"><li>Conduct standard transcardiac perfusion, first with chilled 0.1 M PBS (pH 7.4), followed by 4% PFA solution at 37 Â°C. <br/><em>Note: Chilled PBS is used to clear the blood throughout the vascular system. Thereafter, 37 Â°C PFA is used for vasodilation, allowing for more effective and even fixation of the brain tissue, improving the overall quality of immunohistochemistry (Fix </em><em>et al.</em><em>, 2000). </em> </li><li>Remove brain from the skull and post-fix for 24 h in 4% PFA solution at 4 Â°C, as per Gage <em>et al.</em>, 2012. </li><li>Transfer brains into 0.1 M PBS with 0.02% sodium azide and store at 4 Â°C, until used (see Recipe 3). We typically use one hemisphere blocks.</li><li>Change storage solution every 2 weeks. This way, samples can be stored for up to 6 months.<br/><br/></li></ol></li><li>CLARITY processing of rodent tissue (Figure 1B)<br/><ol class="npwtwo"><li>Remove one brain from the buffer and transfer into a 50 ml Falcon tube with 0.05 M TBS. Keep on a rocker for 5 h at room temperature. </li><li>Process rat brain as a whole or dissect into smaller cubes, or cut thick sections on Vibratome, 0.5 mm-5 mm. </li><li>Incubate brain tissue at 37 Â°C on a rocker in CLARITY solution in a 50 ml Falcon tube for 4-10 days (depending on the thickness of the tissue), until the tissue is transparent (Figures 1B). For adult rat brain, one hemisphere clears in approximately 6 days. Doubling the thickness of tissue typically requires doubling the time of incubation.<br/><br/></li></ol></li><li>Immunolabeling of proteins in CLARITY processed rodent tissue (Figures 2 and 3). <br/><em>Note: Antibody quality and rate of penetration into the tissue are variable. Proteins that are expressed in relatively large quantities require less time for incubation in our experience.</em><br/><ol class="npwtwo"><li>Wash tissue in 0.01 M PBS for 24-48 h in order to completely remove CLARITY solution. </li><li>Transfer tissue into a scintillation vial or fresh falcon tube and add a primary antibody(s) solution: anti-MAP2 (marker for dendrites; 1:500) and anti-NeuN (neuronal marker 1:500) in 0.05 M TBS with 1% Triton X-100 (see Recipe 10). </li><li>Incubate for 12 h at room temperature and then for 4 days at 4 Â°C. </li><li>Rinse the tissue in 0.05 M TBS for 12 h at room temperature, on a rocker. </li><li>Incubate in secondary antibody(s) solution: <em>e.g.</em>, anti-rabbit Alexa Fluor 488 (1:500), anti-mouse Alexa Fluor 568 (1:500) in 0.05 M TBS with 1% Triton X-100. Keep on a rocker at room temperature overnight, then for 2 days at 4 Â°C.</li><li>Wash in 0.05 M TBS at room temperature for 2 days to reduce non-specific staining. For long term storage, keep in 0.05 M TBS at 4 Â°C covered in aluminum foil for up to a month.<br/></li></ol><br/><img alt="" class="layerphoto" height="110" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023311_6052.jpg" width="100"/><br/><strong>Figure 2. CLARITY-processed mouse hippocampus.</strong> A. Mouse hippocampal dentate gyrus (Dg) reconstruction where axons are labeled with SMI-312 (green) and neurons with NeuN (red). The block dimensions: 1,700 Î¼m x 700 Î¼m x 500 Î¼m. Individual stacks at high resolution were first acquired, then stitched for 3D rendering and visualization. Images were acquired on a Leica SP8 confocal, with a 25x water immersion objective. The white box in A is enlarged in B (axons, green) and C (neurons, red), with their overlap shown in D. Bifurcation of axons within the neurons of dentate gyrus is visible. (Scale bars: 60 Î¼m in B, C and D)<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023339_2156.jpg" width="239"/><br/><strong>Figure 3. CLARITY-processed tissue from rat brain, revealing axonal morphology in striatum and cortex. </strong>Coronal blocks of rat brain stained with NF-H, in green. A. Axonal fascicles (SMI-312, green) in the striatum (Str) and fimbria (Fi). The septum can be observed in the upper left corner. B. Basal surface of the cerebral cortex, where projection fibers below the corpus callosum can be observed descending into the striatum. C. Axonal morphology of the folia of rat cerebellum. Arrows show axons surrounding Purkinje cells. Difference in axonal morphology in the granular layer (Gl) and the molecular layer (Ml) can be observed, as well as the axons in white matter (Wm). Images were acquired on a Leica SP8 confocal, with a 25x water immersion objective. (Scale bars: 1,000 Î¼m in A, 200 Î¼m in B and C). </li></ol><br/><strong><em>PRIMATES: Non-human Primates (NHP) and Humans</em></strong> <br/><ol class="npwone"><li>Preparation of primate tissue for CLARITY<ol class="npwtwo"><li><u>NHP</u>: Conduct standard transcardiac perfusion with chilled 0.1 M PBS (1 L), followed by warm 4% PFA solution (8 L) at 37 Â°C. <br/><em>Note: Chilled PBS is used to efficiently clear the blood throughout the vascular system. Thereafter, 37 Â°C PFA is used to provide vasodilation, allowing for more effective and even fixation of the brain tissue, improving the overall quality of subsequent immunohistochemical processing (Fix and Garman, 2000). </em> </li><li><u>NHP</u>: Remove brain from the skull and post-fix the entire brain for 24 h in 4% PFA solution at 4 Â°C, as per Mortazavi <em>et al.</em>, 2018.</li><li><u>Human</u>: Brain tissue maintained in 10% formalin solution.<br/><br/></li></ol><em>Note: The rest of the procedures are common to all primates, NHP and Human.</em><br/><ol class="npwtwo" start="4"><li>Dissect brain into blocks or sections based on anatomical regions of interest. We typically prepare blocks or sections of one hemisphere, 0.5-1 mm in thickness using a Vibratome. </li><li>Transfer brain blocks or sections into 50 ml Falcon tubes containing 0.1 M PBS with 0.02% sodium azide and store at 4 Â°C, until used (see Recipe 3). Change storage solution every 2 weeks. This way, samples can be stored for up to 6 months.<br/><br/></li></ol></li><li>CLARITY Processing of primate brain tissue (Figure 1C)<ol class="npwtwo"><li>Remove the brain tissue from buffer and transfer into a 50 ml Falcon tube with 0.05 M TBS. Keep on a rocker for 5 h at room temperature. </li><li>Incubate tissue at 37 Â°C on a rocker in a 50 ml Falcon tube containing CLARITY solution for 4-15 days (depending on the thickness of the tissue) until the tissue is transparent (Figures 1C). Due to increased lipid content in the brain of non-human primates (NHP) and especially humans (Rouser <em>et al.</em>, 1972), when compared to zebrafish or rodent brain, a 0.5 mm section clears in about 4 days and a 1 mm thick section in about 6-8 days. A 5 mm block/section takes around 1 month to clear.<br/><br/></li></ol></li><li>Immunolabeling of proteins in CLARITY processed primate tissue (Figures 4 and 5)<br/><em>Note: High lipid content in primate brain tissue and factors such as length of fixation, the type of fixative, the region of interest, and block size affect the rate of tissue clearing. Certain proteins may be masked by long-term fixation in formaldehyde. To increase antigenicity, use sodium citrate buffer Step C2 (see Recipe 11). Antibody quality and rate of penetration into the tissue are variable. Proteins that are expressed in relatively large quantities require less time for incubation in our experience.</em><br/><ol class="npwtwo"><li>Wash tissue in 0.01 M PBS for 48 h in order to completely remove CLARITY solution.</li><li>Formalin fixed brain tissues, especially those that remain in fixative for prolonged periods of time, require antigen retrieval to increase antigenicity. To achieve this, transfer the sample into a scintillation vial or fresh falcon tube and add sodium citrate buffer (see Recipe 11). Incubate for 20-24 h at room temperature, on a rocker. Wash tissue in 0.01 M TBS for 12 h, on a rocker.</li><li>Incubate in primary antibody(s) solution, <em>e.g.</em>, anti-MAP2 (marker for dendrites; 1:500) and anti-NeuN (neuronal marker 1:500) in 0.05 M TBS with 1% Triton X-100. </li><li>Incubate in primary antibody (or several antibodies) solution for 12 h at room temperature, and then for 4 days at 4 Â°C. </li><li>Rinse the tissue in 0.05 M TBS for 12 h at room temperature.</li><li>Incubate the tissue in secondary antibody(s) solution, <em>e.g.</em>, anti-rabbit Alexa Fluor 488 (1:500), anti-mouse Alexa Fluor 568 (1:500) in 0.05 M TBS with 1% Triton X-100. Keep on a rocker at room temperature overnight, then for 4 days at 4 Â°C covered in aluminum foil. </li><li>Wash in 0.05 M TBS at room temperature for 2 days to reduce non-specific staining. </li><li>For long term storage, keep in 0.05 M TBS at 4 Â°C.<br/></li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023425_1628.jpg" width="121"/><br/><strong>Figure 4. CLARITY-processed rhesus monkey brain tissue, showing neurons, dendrites, and axons.</strong> Thick block from the cingulate cortex triple-labeled for MAP2 (dendrites, red), Neun (neurons, magenta) and NF-H (axons, green) (Dimensions: 1,700 Î¼m x 3,200 Î¼m x 500 Î¼m). A. Cortical minicolumns in association with their local dendrites (red) and axons (axons). B and C. B shows layers 2, 3, and dense layer 4 of the cingulate cortex, and at higher magnification in C. where dendritic bundles can be resolved individually (white arrows). Images were acquired on a Leica SP8 confocal, with a 25x water immersion objective. (Scale bars: 100 Î¼m in B, 20 Î¼m in C). <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023457_2905.jpg" width="224"/><br/><strong>Figure 5. CLARITY-processed human cerebellum and motor cortex, showing neurons and axonal projections.</strong> A. Human cerebellum, thick section: NeuN (granular cell neurons, red) with SMI-312 (pan-axonal, green) where axonal morphology can be observed in the folia; arrows point to the Purkinje cell layer (dimensions: 600 Î¼m x 600 Î¼m x 200 Î¼m). B. Human motor cortex, thick section: image of a 3D stack from cortical layer 3 stained with MAP2 (red); arrows point to the pyramidal neurons (350 Î¼m x 350 Î¼m x 150 Î¼m).<br/><br/></li></ol><p class="ppzz"><strong>Stripping Antibodies and Restaining tissue</strong> (common to all species)</p><p class="ppzz">Antibodies and EdU Click-iT<sup>Â®</sup> can be destained (stripped), allowing for multiple labels to be assessed in the same tissue (Murray <em>et al.</em>, 2015; Phillips <em>et al.</em>, 2016).</p><ol class="npwtwo"><li>Place tissue in CLARITY solution for 9-18 days at 37 Â°C on an orbital shaker or tube rocker. </li><li>Check whether the stain has been completely eliminated, starting day 10 of incubation, and every 2 days thereafter. To do this, remove one representative sample from the rocker and image under confocal microscope to document the progression of the destaining process. </li><li>Once complete destaining is achieved, wash tissue in TBS for 4 h at room temperature on an orbital shaker or tube rocker. </li><li>Repeat the immunohistochemical protocol, as described above.<br/><div style="white-space:nowrap;"><br/></div></li></ol><p class="ppzz"><strong>Image Aquisition and Quantification in CLARITY-processed Tissues</strong> (common to all species) (Figure 2, Video 1)<br/><em>Note: Cleared tissue becomes somewhat opaque following the immunohistochemistry protocol. To improve the refractive index prior to confocal imaging, incubate tissue in 80% glycerol solution in 0.1 M PBS (Recipe 12), until transparent (Figure 1A). A whole zebrafish brain takes about 10-15 min to clear. For a 0.5 mm brain section, it takes 30-60 min for rodent tissue and 1-2 h for NHP or human tissue to be ready for imaging. The time increases proportionally to section thickness, with a 1 mm thick human brain section taking up to 4 h to clear. </em> </p><ol class="npwone"><li>Tissue preparation for imaging<ol class="npwtwo"><li>Invert a Pelco Black Wall Glass Bottom Dish and roll out Blu Tack in a circle to fully enclose tissue (see Figure 6A). </li><li>Add around 500 Î¼l of 80% glycerol solution (see Recipe 12) within the circle and use a brush/forceps to position the tissue in the center of the well. </li><li>Secure a Pelco Black Wall Glass Bottom Dish on top. Be careful not to flatten brain tissue in order to maintain 3D representation and ensure there are no bubbles. Seal well to prevent tissue dehydration during imaging. </li><li>To avoid glare during image acquisition, assure a horizontal position of the Pelco dish sandwich using a circular leveler (see Imaging Supplies; Figures 6B and 6C).</li><li>Place Pelco dish sandwich onto the microscope stage. </li><li>Add water to the top of Pelco dish for the immersion objective. </li><li>Acquire z-stack tiles that encompass the entire region of interest.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023630_4745.jpg" width="408"/><br/><strong>Figure 6. Pelco dish for microscopy.</strong> A. Inverted Pelco dish with Blu Tack in a circle (left) and upright Pelco dish (right). B. Pelco dish sandwich with a circular leveler in the middle, top view. C. Pelco dish sandwich, side view. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Post-processing data reconstruction and quantification (Figure 7)<br/><ol class="npwtwo"><li>Stitch z-stack tiles, <em>e.g.</em>, using the Leica native software or FIJIâJAVA 6 with the 3D stitching plugin (Preibisch <em>et al.</em>, 2009). </li><li>Transfer the resulting file to a computer running the Bitplane-Imaris program. </li><li>Isolate part of the z-stack corresponding to a structure containing the region of interest, and save as a separate file. </li><li>Open the file on Fiji counting software. </li><li>Run the Fiji 3D object counter and separate out channels.</li><li>Eliminate false positive counts based on voxel volumes (pixel<sup>3</sup>).</li><li>Confirm colocalization <br/><em>Note: Co-localized pixels will have the same x, y, z coordinates. </em> </li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730023656_3561.jpg" width="101"/><br/><strong>Figure 7. 3D cell counting and physical analysis using Imaris by Bitplane.</strong> Left panels, A and C represent raw images of stained cells. Right panels, B and D show program-created yellow spheres annotating each visible cell in A and C. In A CLARITY-processed human brain tissue section (500 Î¼m thick), with neurons labeled with NeuN (red). Arrows in A correspond to the annotation spheres pointed out in B. C. Zebrafish cerebellum in CLARITY-processed whole brain preparation. Dividing cells are labeled with BrdU (red) and EdU (green), following a pulse-chase protocol (Stankiewicz <em>et al.</em>, 2019). Arrows in C and D correspond to the same EdU-labeled cells or their annotation by the program, respectively. Apart from the cellsâ position in the stack (x, y, z coordinates), the program provides data on cell diameter, volume, intensity, and distance between cells.</li></ol><p class="pptt" id="biaoti37625">### Recipes</p><ol class="npwtwok"><li>MS222 (200 mg/L) (1 L)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>MS222<br/></td><td>200 mg<br/></td></tr><tr><td>Tank Water<br/></td><td>1 L<br/></td></tr></tbody></table>Prepare fresh<br/></li><li>Paraformaldehyde (PFA) Fixing solution (4% PFA, 1x PBS,) pH 7.4<br/><em>Note: Prepare within 24 h of use for rodent and NHP tissues. It is typical to use stored frozen PFA, however, long-term storage of 4% PFA increases conversion to formaldehyde. As such, it is recommended to always use freshly prepared PFA. </em><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:320px;"><tbody><tr><td>PBS (10x)<br/></td><td>100 ml<br/></td></tr><tr><td>PFA<br/></td><td>40 g<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>~900 ml<br/></td></tr></tbody></table><ol class="npwthree"><li>Prepare under the chemical hood</li><li>Heat 800 ml of 1x PBS to 60 Â°C on a stirring hot plate</li><li>Add PFA to the 1x PBS slowly, stir while on hot plate</li><li>Raise pH by adding NaOH dropwise until PFA dissolves</li><li>Cool solution and then filter-sterilize (0.45 Âµm)</li><li>Bring the volume to 1 L with 1x PBS</li><li>Adjust pH to 7.4 with HCI</li><li>Aliquot in 1.5 ml eppendorfs and store at -20 Â°C for up to 1 year (only for use in zebrafish)</li></ol></li><li>0.1 M PBS with 0.02% sodium azide (pH 7.4) (Store at 4 Â°C for up to 4 weeks)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>PBS (10x)<br/></td><td>100 ml<br/></td></tr><tr><td>Sodium azide<br/></td><td>200 mg<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>900 ml<br/></td></tr></tbody></table>Adjust pH using NaOH or HCI</li><li>0.05 M TBS (pH 8.0) (Store at room temperature for up to 2 weeks)<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>TBS<br/></td><td>16 g<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>1 L<br/></td></tr></tbody></table></li><li>CLARITY Solution (pH 8.5) (Store at room temperature for up to 6 months) <br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td>0.2 M boric acid<br/></td><td>12.37 g<br/></td></tr><tr><td>4% Sodium dodecyl sulfate (SDS)<br/></td><td>40 g<br/></td></tr><tr><td>20 mM Lithium Hydroxide<br/></td><td>0.84 g<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>up to 1 L<br/></td></tr></tbody></table>Adjust pH using NaOH or HCI</li><li>50% formamide/50% 2x SSC (prepare fresh each time) <br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>Formamide<br/></td><td>25 ml<br/></td></tr><tr><td>20x SSC<br/></td><td>2.5 ml<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>22.5 ml<br/></td></tr><tr><td>Final Volume<br/></td><td>50 ml<br/></td></tr></tbody></table></li><li>2 M Hydrochloric acid (HCI) (prepare fresh each time)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>HCl<br/></td><td>10 ml<br/></td></tr><tr><td>20x SSC<br/></td><td>40 ml<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>50 ml<br/></td></tr></tbody></table>Prepare under the chemical hood, adding acid to water.</li><li>0.1 M boric acid (pH 8.5) (store at room temperature for up to 6 months) <br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>Boric Acid<br/></td><td>2.5 g<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>400 ml<br/></td></tr></tbody></table>Prepare under chemical hood, adding acid to water<br/>Adjust pH to 8.5 using NaOH or HCI</li><li>EdU labeling solution (prepared fresh each time) <br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>PBS (1x)<br/></td><td>875 Î¼l<br/></td></tr><tr><td>CuSO<sub>4</sub><br/></td><td>20 Î¼l<br/></td></tr><tr><td>Fluorescent dye azide<br/></td><td>5 Î¼l<br/></td></tr><tr><td>Reaction Buffer Addictive<br/></td><td>100 Î¼l<br/></td></tr></tbody></table></li><li>0.05 M TBS (pH 8.0) with 1% Triton X-100 (Store at room temperature for up to 2 weeks)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>TBS<br/></td><td>16 g<br/></td></tr><tr><td>Triton X-100<br/></td><td>10 ml<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>1 L<br/></td></tr></tbody></table></li><li>Sodium citrate buffer (10 mM Sodium citrate, 0.5% Triton X-100, pH 9) (store at 4 Â°C for 4 weeks)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>Tri-sodium citrate (dihydrate)<br/></td><td>2.94 g<br/></td></tr><tr><td>Triton X-100<br/></td><td>5 ml<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>1,000 ml<br/></td></tr></tbody></table>Adjust pH using NaOH or HCI</li><li>80% glycerol solution in 0.1 M PBS (store at room temperature for up to 4 weeks)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>PBS (10x)<br/></td><td>10 ml<br/></td></tr><tr><td>Glycerol<br/></td><td>80 g<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>90 ml<br/></td></tr></tbody></table></li></ol><p class="pptt" id="biaoti37626">### Acknowledgments</p><p class="ppzz">This work was supported by a grant from the Chaikin-Wile Foundation and NSF PHY 1505000. This protocol was modified from Chung <em>et al.</em>, 2013.</p><p class="pptt" id="biaoti37627">### Competing interests</p><p class="ppzz">The authors declare no competing financial interests.</p><p class="pptt" id="biaoti37628">### Ethics</p><p class="ppzz">Tissues were obtained as part of other ongoing studies in zebrafish (#14366), rodents and NHP (#14291). Human tissue was obtained from the Anatomical Donor Program at Boston University Medical Campus (BUMC). Animals were housed at the Laboratory Animal Science Center of BUMC. The facility is fully accredited by the Association for the Assessment of the Laboratory Animal Care where all procedures were strictly conformed to the National Institute of Health Guidelines, the Institute of the Laboratory Animal Resources Commission on Life Sciences and approved by BUMC Institutional Animal Care and Use Committee.</p><p class="pptt" id="biaoti37629">### References</p><ol class="npwtwok"><li>Akle, V., Stankiewicz, A. J., Kharchenko, V., Yu, L., Kharchenko, P. V. and Zhdanova, I. V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28087763" target="_blank">Circadian kinetics of cell cycle progression in adult neurogenic niches of a diurnal vertebrate.</a> <em>J Neurosci</em> 37(7): 1900-1909.</li><li>Antinucci, P. and Hindges, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27381182" target="_blank">A crystal-clear zebrafish for <em>in vivo</em> imaging.</a> <em>Sci Rep </em>6: 29490.</li><li>Azaripour, A., Lagerweij, T., Scharfbillig, C., Jadczak, A. E., Willershausen, B. and Van Noorden, C. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27142295" target="_blank">A survey of clearing techniques for 3D imaging of tissues with special reference to connective tissue.</a> <em>Prog Histochem Cytochem</em> 51(2): 9-23.</li><li>Brothers, K. M., Newman, Z. R. and Wheeler, R. T. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21551247" target="_blank">Live imaging of disseminated candidiasis in zebrafish reveals role of phagocyte oxidase in limiting filamentous growth.</a> <em>Eukaryot Cell</em> 10(7): 932-944.</li><li>Buell, S. J. and Coleman, P. D. (1981). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7237164" target="_blank">Quantitative evidence for selective dendritic growth in normal human aging but not in senile dementia.</a> <em>Brain Res</em> 214(1): 23-41.</li><li>Chapouton, P., Skupien, P., Hesl, B., Coolen, M., Moore, J.C., Madelaine, R., Kremmer, E., Faus-Kessler, T., Blader, P., Lawson, N.D., Bally-Cuif, L. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20534844" target="_blank">Notch activity levels control 806 the balance between quiescence and recruitment of adult neural stem cells.</a><em> J Neurosci</em> 807 (30): 7961-7974.</li><li>Chung, K. and Deisseroth, K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23722210" target="_blank">CLARITY for mapping the nervous system.</a> <em>Nat Methods</em> 10(6): 508-513.</li><li>Chung, K., Wallace, J., Kim, S. Y., Kalyanasundaram, S., Andalman, A. S., Davidson, T. J., Mirzabekov, J. J., Zalocusky, K. A., Mattis, J., Denisin, A. K., Pak, S., Bernstein, H., Ramakrishnan, C., Grosenick, L., Gradinaru, V. and Deisseroth, K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23575631" target="_blank">Structural and molecular interrogation of intact biological systems.</a> <em>Nature</em> 497(7449): 332-337.</li><li>Coskren, P. J., Luebke, J. I., Kabaso, D., Wearne, S. L., Yadav, A., Rumbell, T., Hof, P. R. and Weaver, C. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25527184" target="_blank">Functional consequences of age-related morphologic changes to pyramidal neurons of the rhesus monkey prefrontal cortex.</a> <em>J Comput Neurosci</em> 38(2): 263-283.</li><li>Cronan, M. R., Rosenberg, A. F., Oehlers, S. H., Saelens, J. W., Sisk, D. M., Jurcic Smith, K. L., Lee, S. and Tobin, D. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26449262" target="_blank">CLARITY and PACT-based imaging of adult zebrafish and mouse for whole-animal analysis of infections.</a> <em>Dis Model Mech</em> 8(12): 1643-1650.</li><li>Du, H., Hou, P., Zhang, W. and Li, Q. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30186373" target="_blank">Advances in CLARITY-based tissue clearing and imaging.</a> <em>Exp Ther Med</em> 16(3): 1567-1576. </li><li>Efimova, O. I. and Anokhin, K. V. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Efimova%2C+O.I.%2C+Anokhin%2C+K.V.+(2009)." target="_blank">Enhancement of optical transmission capacity of isolated structures in the brain of mature mice.</a> <em>Bull Exp Biol Med.</em> 147(1):3-6.</li><li>el Hachimi, K. H. and Foncin, J. F. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2125849" target="_blank">Loss of dendritic spines in Alzheimer's disease.</a> <em>C R Acad Sci III</em> 311(11): 397-402.</li><li>ErtÃ¼rk, A., Becker, K., Jahrling, N., Mauch, C. P., Hojer, C. D., Egen, J. G., Hellal, F., Bradke, F., Sheng, M. and Dodt, H. U. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23060243" target="_blank">Three-dimensional imaging of solvent-cleared organs using 3DISCO.</a> <em>Nat Protoc</em> 7(11): 1983-1995.</li><li>Ferrante, R. J., Kowall, N. W. and Richardson, E. P., Jr. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1836019" target="_blank">Proliferative and degenerative changes in striatal spiny neurons in Huntington's disease: a combined study using the section-Golgi method and calbindin D28k immunocytochemistry.</a> <em>J Neurosci</em> 11(12): 3877-3887.</li><li>Ferrer, I. and Gullotta, F. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2141748" target="_blank">Down's syndrome and Alzheimer's disease: dendritic spine counts in the hippocampus.</a> <em>Acta Neuropathol</em> 79(6): 680-685.</li><li>Fior, R., Povoa, V., Mendes, R. V., Carvalho, T., Gomes, A., Figueiredo, N. and Ferreira, M. G. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28835536" target="_blank">Single-cell functional and chemosensitive profiling of combinatorial colorectal therapy in zebrafish xenografts.</a> <em>Proc Natl Acad Sci U S A</em> 114(39): E8234-E8243.</li><li>Fix, A. S. and Garman, R.H. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Practical+aspects+of+neuropathology%3A+a+technical+guide+for+working+with+the+nervous+system." target="_blank">Practical aspects of neuropathology: a technical guide for working with the nervous system.</a> <em>Toxicol Pathol</em>. 28(1):122-31.</li><li>FrÃ©taud, M., Riviere, L., Job, E., Gay, S., Lareyre, J. J., Joly, J. S., Affaticati, P. and Thermes, V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28211501" target="_blank">High-resolution 3D imaging of whole organ after clearing: taking a new look at the zebrafish testis.</a> <em>Sci Rep </em>7: 43012.</li><li>Font-Burgada, J., Shalapour, S., Ramaswamy, S., Hsueh, B.,Rossell, D., Umemura, A., Taniguchi, K., Nakagawa, H.,Valasek, M., Ye, L., Kopp, J., Sander, M., Carter, H.,Deisseroth, K., Verma, I. and Karin, M. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Hybrid+periportal+hepatocytes+regenerate+the+injured+liver+without+giving+rise+to+cancer" target="_blank">Hybrid periportal hepatocytes regenerate the injured liver without giving rise to cancer.</a> <em>Cell</em> 162(4): 766-779. </li><li>Fumoto, S., Nishimura, K., Nishida, K. and Kawakami, S. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26824850" target="_blank">Three-Dimensional imaging of the intracellular fate of plasmid DNA and transgene expression: ZsGreen1 and tissue clearing method CUBIC are an optimal combination for multicolor deep imaging in murine tissues.</a> <em>PLoS One</em> 11(1): e0148233. </li><li>Gage, G. J., Kipke, D. R. and Shain, W. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=53.%09Gage%2C+G.+J.%2C+Kipke%2C+D.+R.%2C+Shain+Whole+Animal+Perfusion+Fixation+for+Rodents" target="_blank">Whole Animal Perfusion Fixation for Rodents.</a> <em>J Vis Exp</em> (65): e3564. </li><li>Grandel, H., Kaslin, J., Ganz, J., Wenzel, I. and Brand, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16682018" target="_blank">Neural stem cells and neurogenesis in the adult zebrafish brain: origin, proliferation dynamics, migration and cell fate. </a><em>Dev Biol</em> 295(1): 263-277. </li><li>Gupta, T. and Mullins, M. C. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20203557" target="_blank">Dissection of organs from the adult zebrafish.</a> <em>J Vis Exp </em>37: 1717.</li><li>Hama, H., Kurokawa, H., Kawano, H., Ando, R., Shimogori, T., Noda, H., Fukami, K., Sakaue-Sawano, A. and Miyawaki, A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21878933" target="_blank">Scale: a chemical approach for fluorescence imaging and reconstruction of transparent mouse brain.</a> <em>Nat Neurosci</em> 14(11): 1481-1488.</li><li>Helmstaedter, M., Briggman, K. L., Turaga, S. C., Jain, V., Seung, H. S. and Denk, W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23925239" target="_blank">Connectomic reconstruction of the inner plexiform layer in the mouse retina.</a> <em>Nature</em> 500(7461): 168-174.</li><li>Jensen, K. H. R. and Berg, R. W. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28728966" target="_blank">Advances and perspectives in tissue clearing using CLARITY. </a><em>J Chem Neuroanat</em> 86: 19-34.</li><li>Kamel, M. and Ninov, N. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28888214" target="_blank">Catching new targets in metabolic disease with a zebrafish.</a> <em>Curr Opin Pharmacol</em> 37: 41-50.</li><li>Kaslin, J., Ganz, J., Geffarth, M., Grandel, H., Hans, S. and Brand, M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19439592" target="_blank">Stem cells in the adult zebrafish cerebellum: initiation and maintenance of a novel stem cell niche.</a> <em>J Neurosci</em> 29(19): 6142-6153.</li><li>Kaufman, J. A., Castro, M. J., Sandoval-Skeet, N. and Al-Nakkash, L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28967147" target="_blank">Optical clearing of small intestine for three-dimensional visualization of cellular proliferation within crypts.</a> <em>J Anat</em> 232(1): 152-157.</li><li>Ke, M. T., Fujimoto, S. and Imai, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23792946" target="_blank">SeeDB: a simple and morphology-preserving optical clearing agent for neuronal circuit reconstruction.</a> <em>Nat Neurosci</em> 16(8): 1154-1161.</li><li>Lee, H., Park, J., Seo, I., Park, S., and Kim, S. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Improved+application+of+the+electrophoretic+tissue+clearing+technology%2C+CLARITY%2C+to+intact+solid+organs+including+brain%2C+pancreas%2C+liver%2C+kidney%2C+lung%2C+and+intestine." target="_blank">Improved application of the electrophoretic tissue clearing technology, CLARITY, to intact solid organs including brain, pancreas, liver, kidney, lung, and intestine.</a> <em>BMC Dev Biol</em> 14: 48.</li><li>Lee, E., Choi, J., Jo, Y., Kim, J. Y., Jang, Y. J., Lee, H. M., Kim, S. Y., Lee, H. J., Cho, K., Jung, N., Hur, E. M., Jeong, S. J., Moon, C., Choe, Y., Rhyu, I. J., Kim, H. and Sun, W. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26750588" target="_blank">ACT-PRESTO: Rapid and consistent tissue clearing and labeling method for 3-dimensional (3D) imaging.</a> <em>Sci Rep</em> 6: 18631.</li><li>Li, A., Gong, H., Zhang, B., Wang, Q., Yan, C., Wu, J., Liu, Q., Zeng, S. and Luo, Q. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21051596" target="_blank">Micro-optical sectioning tomography to obtain a high-resolution atlas of the mouse brain.</a> <em>Science</em> 330(6009): 1404-1408.</li><li>Lindsey, B. W. and Kaslin, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28296621" target="_blank">Optical Projection Tomography as a Novel Method to Visualize and Quantitate Whole-Brain Patterns of Cell Proliferation in the Adult Zebrafish Brain.</a> <em>Zebrafish</em> 14(6): 574-577.</li><li>Lindsey, B. W., Douek, A. M., Loosli, F. and Kaslin, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29386991" target="_blank">A Whole Brain Staining, Embedding, and Clearing Pipeline for Adult Zebrafish to Visualize Cell Proliferation and Morphology in 3-Dimensions.</a> <em>Front Neurosci</em> 11: 750. </li><li>Mano, T., Albanese, A., Dodt, H. U., Erturk, A., Gradinaru, V., Treweek, J. B., Miyawaki, A., Chung, K. and Ueda, H. R. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30381424" target="_blank">Whole Brain Analysis of Cells and Circuits by Tissue Clearing and Light-Sheet Microscopy.</a> <em>J Neurosci</em> 38(44): 9330-9337.</li><li>McNeill, T. H., Brown, S. A., Rafols, J. A. and Shoulson, I. (1988). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3416180" target="_blank">Atrophy of medium spiny I striatal dendrites in advanced Parkinson's disease.</a> <em>Brain Res</em> 455(1): 148-152.</li><li>Migault, G., van der Plas, T. L., Trentesaux, H., Panier, T., Candelier, R., Proville, R., Englitz, B., Debregeas, G. and Bormuth, V. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30449666" target="_blank">Whole-brain calcium imaging during physiological vestibular stimulation in larval zebrafish.</a> <em>Curr Biol</em> 28(23): 3723-3735.e6.</li><li>Mortazavi, F., Wedeen, V. J. and Rosene, D. L. (2016). Chapter 17: <a href="https://www.sciencedirect.com/science/article/pii/B9780128013939000177" target="_blank">Neuroanatomical Techniques for Analysis of Axonal Trajectories in the Cerebral Cortex of the Rhesus Monkey.</a> In: <em>Axons and Brain Architecture</em>. In: Rockland, K. S. (Ed.). Elsevier. Cambridge, MA. 349-368.</li><li>Mortazavi, F., Oblak, A.L., Morrison, W.Z., Schmahmann, J.D., Stanley, H.E., Wedeen, V.J., Rosene, D.L. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Geometric+Navigation+of+Axons+in+a+Cerebral+Pathway%3A+Comparing+dMRI+with+Tract+Tracing+and+Immunohistochemistry" target="_blank">Geometric Navigation of Axons in a Cerebral Pathway: Comparing dMRI with Tract Tracing and Immunohistochemistry.</a> <em>Cereb Cortex </em>28(4):1219-1232.</li><li>Murray, E., Cho, J. H., Goodwin, D., Ku, T., Swaney, J., Kim, S. Y., Choi, H., Park, Y. G., Park, J. Y., Hubbert, A., McCue, M., Vassallo, S., Bakh, N., Frosch, M. P., Wedeen, V. J., Seung, H. S. and Chung, K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26638076" target="_blank">Simple, Scalable Proteomic Imaging for High-Dimensional Profiling of Intact Systems.</a> <em>Cell</em> 163(6): 1500-1514.</li><li>Neckel, P. H., Mattheus, U., Hirt, B., Just, L., and Mack, A. F. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Large-scale+tissue+clearing+(PACT)%3A+Technical+evaluation+and+new+perspectives+in+immunofluorescence+histology%2C+and+ultrastructure" target="_blank">Large-scale tissue clearing (PACT): Technical evaluation and new perspectives in immunofluorescence histology, and ultrastructure.</a> <em>Sci Rep</em> 6(1): e34331.</li><li>Palha, N., Guivel-Benhassine, F., Briolat, V., Lutfalla, G., Sourisseau, M., Ellett, F., Wang, C. H., Lieschke, G. J., Herbomel, P., Schwartz, O. and Levraud, J. P. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24039582" target="_blank">Real-time whole-body visualization of Chikungunya Virus infection and host interferon response in zebrafish.</a> <em>PLoS Pathog</em> 9(9): e1003619.</li><li>Phillips, J., Laude, A., Lightowlers, R., Morris, C. M., Turnbull, D. M. and Lax, N. Z. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27181107" target="_blank">Development of passive CLARITY and immunofluorescent labelling of multiple proteins in human cerebellum: understanding mechanisms of neurodegeneration in mitochondrial disease.</a> <em>Sci Rep</em> 6: 26013.</li><li>Richardson, D. S. and Lichtman, J. W. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26186186" target="_blank">Clarifying Tissue Clearing.</a> <em>Cell</em> 162(2): 246-257.</li><li>Rouser, G., Kritchevsky, G., Yamamoto, A. and Baxter, C. F. (1972). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Lipids+in+the+nervous+system+of+different+species+as+a+function+of+age%3A+brain%2C+spinal+cord%2C+peripheral+nerve%2C+purified+whole+cell+preparations%2C+and+subcellular+particulates%3A+regulatory+mechanisms+and+membrane+structure." target="_blank">Lipids in the nervous system of different species as a function of age: brain, spinal cord, peripheral nerve, purified whole cell preparations, and subcellular particulates: regulatory mechanisms and membrane structure.</a> <em>Adv Lipid Res</em>. 10:261-360. </li><li>Saboor, F., Reckmann, A. N., Tomczyk, C. U., Peters, D. M.,Weissmann, N., Kaschtanow, A., Schermuly, R. T., Michurina, T. V., Enikolopov, G., MÃ¼ller, D., Mietens, A. and Middendorff, R. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Nestin-expressing+vascular+wall+cells+drive+development+of+pulmonary+hypertension." target="_blank">Nestin-expressing vascular wall cells drive development of pulmonary hypertension.</a> <em>Eur Respir J</em> 47(3): 876-888.</li><li>Spalteholz, W. (1914). Uber das durchsichtigmachen von menschlichen und tierischen prÃ¤paraten und seine theoretischen bedingungen, nebst anhang: Uber KnochenfÃ¤rbung. Leipzig: S. Hirzel.</li><li>Stankiewicz, A. J., Mortazavi, F., Kharchenko, P. V., McGowan, E. M., Kharchenko, V. and Zhdanova, I. V. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30737307" target="_blank">Cell kinetics in the adult neurogenic niche and impact of diet-induced accelerated aging.</a> <em>J Neurosci</em> 39(15): 2810-2822.</li><li>Susaki, E. A., Tainaka, K., Perrin, D., Kishino, F., Tawara, T., Watanabe, T. M., Yokoyama, C., Onoe, H., Eguchi, M., Yamaguchi, S., Abe, T., Kiyonari, H., Shimizu, Y., Miyawaki, A., Yokota, H. and Ueda, H. R. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24746791" target="_blank">Whole-brain imaging with single-cell resolution using chemical cocktails and computational analysis.</a> <em>Cell</em> 157(3): 726-739.</li><li>Tomer, R., Ye, L., Hsueh, B. and Deisseroth, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24945384" target="_blank">Advanced CLARITY for rapid and high-resolution imaging of intact tissues.</a> <em>Nat Protoc</em> 9(7): 1682-1697.</li><li>Vanwalleghem, G. C., Ahrens, M. B. and Scott, E. K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29486425" target="_blank">Integrative whole-brain neuroscience in larval zebrafish.</a> <em>Curr Opin Neurobiol</em> 50: 136-145</li><li>Vigouroux, R. J., Belle, M. and Chedotal, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28728585" target="_blank">Neuroscience in the third dimension: shedding new light on the brain with tissue clearing.</a> <em>Mol Brain</em> 10(1): 33.</li><li>White, R. M., Sessa, A., Burke, C., Bowman, T., LeBlanc, J., Ceol, C., Bourque, C., Dovey, M., Goessling, W., Burns, C. E. and Zon, L. I. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18371439" target="_blank">Transparent adult zebrafish as a tool for <em>in vivo</em> transplantation analysis.</a> <em>Cell Stem Cell</em> 2(2): 183-189.</li><li>Whitehead, L. W., McArthur, K., Geoghegan, N. D. and Rogers, K. L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28446796" target="_blank">The reinvention of twentieth century microscopy for three-dimensional imaging.</a> <em>Immunol Cell Biol</em> 95(6): 520-524.</li><li>Wolman, M. and Granato, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22328273" target="_blank">Behavioral genetics in larval zebrafish: learning from the young.</a> <em>Dev Neurobiol</em> 72(3): 366-372.</li><li>Xu, N., Tamadon, A., Liu, Y., Ma, T., Leak, R.K., Chen, J., Gao, Y. and Feng, Y. (2017). <a href="https://www.nature.com/articles/s41598-017-10204-5" target="_blank">Fast free-of-acrylamide clearing tissue (FACT) an optimized newprotocol for rapid, high-resolution imaging of three-dimensionalbrain tissue.</a> <em>Sci Rep</em> 7(1): 9895.</li><li>Yang, B., Treweek, J. B., Kulkarni, R. P., Deverman, B. E., Chen, C. K., Lubeck, E., Shah, S., Cai, L. and Gradinaru, V. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25088144" target="_blank">Single-cell phenotyping within transparent intact tissue through whole-body clearing.</a> <em>Cell</em> 158(4): 945-958.</li><li>Yu, T., Qi, Y., Gong, H., Luo, Q. and Zhu, D. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29024450" target="_blank">Optical clearing for multiscale biological tissues.</a> <em>J Biophotonics</em> 11(2).</li><li>Zupanc, G. K., Hinsch, K. and Gage, F. H. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15952170" target="_blank">Proliferation, migration, neuronal differentiation, and long-term survival of new cells in the adult zebrafish brain.</a> <em>J Comp Neurol</em> 488(3): 290-319.<br/></li></ol>