<body>
            <h1><em>Ex vivo</em> Analysis of DNA Repair Capacity of Human Peripheral Blood Mononuclear  Cells by a Modified Host Cell Reactivation Assay</h1>
            <p><strong>Authors:</strong> Katja Matt, JÃ¶rg Bergemann</p>
            <h2>Abstract</h2>
            <p>The ability of humans to repair DNA damages decreases with increasing age. In order to be able to repair daily occurring DNA damages, it becomes more and more important to preserve repair capability of cells with aging. The preservation of DNA repair processes contributes to preventing DNA mutations and subsequently the onset of age-related diseases such as cancer. For the determination of DNA repair of human cells, mostly <em>in vitro</em> cell cultures are used. However, an <em>ex vivo</em> approach can provide a more accurate result compared with <em>in vitro</em> cell cultures, since the DNA repair ability is measured directly without the influence of prolonged culture time. Published protocols use <em>in vitro</em> cultured cells with a single reporter plasmid or a luciferase reporter. Our modified host cell reactivation assay enables the measurement of DNA repair capacity (nucleotide excision repair) of <em>ex vivo</em> isolated human peripheral blood mononuclear cells (PBMCs). For this purpose, PBMCs are isolated out of human anticoagulated blood by density gradient centrifugation. Directly after isolation, the PBMCs are co-transfected with two plasmids, one being previously damaged by UVC irradiation and one remaining undamaged. PBMCs are incubated for 24 h and subsequently analyzed by fluorescence activated cell sorting (FACS). The ability of cells to repair the DNA damages leads to a functional reactivation of the reporter gene. The assay presented here provides a solution to determine human DNA repair capacity <em>ex vivo</em> directly out of the human body. Furthermore, it can be used to research the <em>ex vivo</em> influence of different substances on DNA repair capacity of humans.</p>
            <p><strong>Keywords:</strong> Modified host cell reactivation assay, DNA repair capacity, Human peripheral blood mononuclear cells, <em>Ex vivo</em>, Nucleotide excision repair, Aging</p>
            <h2>Background</h2><p>Maintaining the integrity of the human genome is a prerequisite for delaying or preventing the development of age-related diseases. Every day, up to 50,000 DNA-damaging events occur in every cell (reviewed in Shrinivas <em>et al.</em>, 2017). Such damages arise because of exogenous influences like radiation or chemicals, or via the normal cellular metabolism. For this reason, functional DNA repair mechanisms are crucial for the survival of organisms. One of the most important DNA repair pathways is nucleotide excision repair (NER), which detects and eliminates bulky DNA lesions and helix-distorting DNA adducts like cyclobutane pyrimidine dimers (CPDs) and pyrimidine-(6-4)-pyrimidone photoproducts (6-4PPs) (for review see Nouspikel, 2009).<br/><br/>The modified host cell reactivation assay (mHCRA) is a reliable and reproducible method for the research of NER capacity of human cells based on the restoration of a previously damaged reporter gene. The principle of reporter gene restoration was first described in 1985 (ProtiÄ-SabljiÄ <em>et al.</em>, 1985), subsequently adjusted by Athas <em>et al.</em> for an application with human lymphocytes (Athas <em>et al.</em>, 1991) and further refined by using fluorescent reporter proteins (Roguev and Russev, 2000). Up to now, few researchers use the mHCRA, although it is a highly reproducible and reliable method. Qiao <em>et al.</em> used the modified HCRA with a luciferase reporter on human lymphocytes to study DNA repair (Qiao <em>et al.</em>, 2002). In 2010, Burger <em>et al.</em> published an advanced method, which describes the mHCRA for studies on human skin cells. They used two reporter plasmids and fluorescence-activated cell sorting to determine DNA repair capacity (Burger <em>et al.</em>, 2010). Furthermore, Mendez <em>et al.</em> researched DNA repair by carrying out nucleofection of cryopreserved human lymphocytes with one reporter plasmid (Mendez <em>et al.</em>, 2011). <br/><br/>Our mHCRA enables the measurement of NER capacity of PBMCs as an <em>ex vivo</em> approach. By using two different plasmids, transfection efficiencies can be normalized by a transfected control plasmid. This procedure allows for comparing experiments with different transfection efficiencies with each other. We used the method described here to reveal the influence of calorie reduction on the DNA repair capacity of humans (Matt <em>et al.</em>, 2016). Moreover, the use of this assay can provide a more accurate assessment of positive or negative effects of substances like dietary supplements or pharmaceutical agents on DNA repair ability of human cells than the use of <em>in vitro</em> cell cultures.</p>
        </body>

<p class="pptt" id="biaoti28676">## Materials and Reagents</p><ol class="npwtwok"><li>Centrifuge tubes 15, 50 ml (SARSTEDT, catalog numbers: 62.554.001, 62.547.254)</li><li>1.5 ml reaction tube (SARSTEDT, catalog number: 72.706)</li><li>Petri dishes (SARSTEDT, catalog number: 83.3902)</li><li>Pipette tips 10 Î¼l, 20 Î¼l, 200 Î¼l, 1,000 Î¼l (SARSTEDT, catalog numbers: 70.1130, 70.116, 70.760.002, 70.762)</li><li>Serological pipettes 5 ml and 10 ml (SARSTEDT, catalog numbers: 86.1253.001 and 86.1254.001)</li><li>Leucosep<sup>TM</sup>, with porous barrier, 50 ml, pre filled with Leucosep<sup>TM</sup> separation medium (Greiner Bio-One GmbH, catalog number: 227288)</li><li>Sterile filter 0.2 Î¼m (SARSTEDT, catalog number: 83.1826.001)</li><li>Cell culture plate, 6-well, Cell+ (SARSTEDT, catalog number: 83.3920.300)</li><li>0.4 cm electroporation cuvettes (Molecular BioProducts, Inc., purchased via Thermo Fisher Scientific GmbH, catalog number: 5540-11)</li><li>FACS tubes 5 ml and 0.5 ml (SARSTEDT, catalog numbers: 55.1579, 55.673)</li><li>Plasmid-DNA solution pEGFP-N1 (Clontech, catalog number: 6085-1) and pDsRedExpress-N1 (Clontech, catalog number: 632429), each 900 ng/Âµl in Qiagen EB buffer (storage temperature for longer periods: -80 Â°C, otherwise keep at 4 Â°C)</li><li>Human venous blood, anticoagulated</li><li>KCl (Carl Roth, catalog number: 6781.3)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Carl Roth, catalog number: 3904.2)</li><li>Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O (Carl Roth, catalog number: N350.1)</li><li>NaCl (Carl Roth, catalog number: 3957.3)</li><li>S-Monovette<sup>Â®</sup> K3 EDTA 4.9 ml and 7.5 ml (SARSTEDT, catalog numbers: 04.1931.001 and 01.1605.001)</li><li>RPMI 1640 without Phenol Red (Gibco<sup>Â®</sup> by Life Technologies, purchased via Fisher Scientific GmbH, catalog number: 11835-063, storage temp. 4 Â°C)</li><li>Fetal Bovine Serum (Gibco<sup>Â®</sup> by Life Technologies, chased via Fisher Scientific GmbH, catalog number: 10270-106)</li><li>EB Buffer, part of the QIAGEN Plasmid Plus Giga Kit (Qiagen GmbH, Hilden, catalog number: 12991)</li><li>BD FACS Sheath Solution with Surfactant (BD Biosciences, catalog number: 336911)</li><li>1x PBS (see Recipes)</li><li>0.9% NaCl solution (see Recipes)</li><li>PBMC medium (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28677">### Equipment</p><ol class="npwtwok"><li>Pipettes, Eppendorf Research<sup>Â®</sup> Plus 10 Î¼l, 20 Î¼l, 200 Î¼l, 1,000 Î¼l (Eppendorf, catalog numbers: 3123000020, 3123000039, 3123000055, 3123000063)</li><li>Stratalinker<sup>Â®</sup> UV Crosslinker (Stratagene, catalog number: 400072, model: Model 1800)</li><li>Centrifuge with swinging bucket rotor (Eppendorf, catalog numbers: 5805000010, 5804709004)</li><li>Neubauer counting chamber improved (Carl Roth, catalog number: PC72.1)</li><li>Gene Pulser II with Capacitance extender plus and Pulse controller plus (Bio-Rad Laboratories, Inc., catalog numbers: 165-2106, 165-2108, 165-2110)</li><li>CO<sub>2</sub> Incubator Hera cell 150 (Kendro Laboratory Products, catalog number: 51013568)</li><li>BD FACSCalibur Flow Cytometry System with 488 nm laser (Becton, Dickinson and Company)</li><li>-80 Â°C freezer HFU600TV (Thermo Fisher Scientific<sup>TM</sup>, catalog number: 11670823)<br/></li></ol><p class="pptt" id="biaoti28678">### Software</p><ol class="npwtwok"><li>CellQuest Pro (Becton, Dickinson and Company, version 4.0.2)</li><li>Prism (GraphPad Software, Inc.)<br/></li></ol><p class="pptt" id="biaoti28679">### Procedure</p><p class="ppzz"><strong>Day 1</strong> </p><ol class="npwtwo"><li>Prepare plasmid solutions<ol class="npwthree"><li>pDsRedExpress-N1: Adjust concentration to a maximum of 900 ng/Âµl using EB buffer. <br/><em>Note: The higher the plasmid concentration is, the more likely plasmids lie above each other during irradiation and irradiation becomes less effective. This results in higher DNA repair capacities in the end. Irradiate half of the volume by pipetting droplets of 40 Âµl into a Petri dish (10 cm diameter). Irradiate with 5 kJ/m<sup>2</sup> UVC (254 nm) without lid of the Petri dish using Stratalinker</em><sup><em>Â®</em></sup><em> UV Crosslinker. After irradiation, collect droplets in a 1.5 ml reaction tube. The other half of the plasmid solution remains non-irradiated.</em> </li><li>pEGFP-N1 needs no further treatment as it serves as transfection control. The concentration of plasmid solution should be &gt; 600 ng/Âµl in order to avoid too high volumes. The ratio of absorbance at 260 nm and 280 nm should be ~1.8.</li><li>For the measurement of DNA repair, mix 10 Âµg of pEGFP-N1 and 30 Âµg of irradiated pDsRedExpress-N1. For calculation of the correction factor, mix 10 Âµg of pEGFP-N1 and 30 Âµg of non-irradiated pDsRedExpress-N1 for each electroporation. Note that the volumes needed depend on the concentration of the plasmid solutions. Adjust the volume of the mixture to 100 Âµl using EB buffer. Prepare enough plasmid-DNA mixtures for the entire study planned. Store mixtures at 4 Â°C for short-term storage or at -80 Â°C for longer periods.</li></ol></li><li>Isolate PBMCs from 12 ml of human blood using Leucosep<sup>TM</sup> tubes according to the manufacturerâs instructions as follows:<br/><ol class="npwthree"><li>Dilute 12 ml anticoagulated blood 1:2 using 0.9% NaCl solution (24 ml final volume) and transfer into Leucosep<sup>TM</sup> tubes.</li><li>Centrifugate at 1,000 <em>x g</em> for 10 min in a swinging bucket rotor and switch off brakes. Figure 1A shows the Leucosep<sup>TM</sup> tube before centrifugation and Figure 1B shows the tube after centrifugation.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731013505_6896.jpg" width="155"/><br/><strong>Figure 1. Leucosep<sup>TM</sup> tube before and after centrifugation.</strong> A. Leucosep<sup>TM</sup> tube filled with anticoagulated blood, which was previously diluted 1:2 with 0.9% NaCl solution. B. Leucosep<sup>TM</sup> tube after density gradient centrifugation. The different layer can be seen clearly from top to bottom: plasma, enriched cells, separation medium, the porous barrier, again separation medium and finally erythrocytes and granulocytes.<br/><br/></li><li>Pipet approximately 10 ml of plasma from the top in order to reach the enriched cell fraction more easily.</li><li>Harvest the enriched cell fraction using a 1,000 Âµl Eppendorf pipette and transfer cells into a 50 ml centrifuge tube.</li><li>Wash cells with 10 ml PBS and centrifuge at 250 <em>x g</em> for 10 min.</li><li>Repeat the washing step two times each with 5 ml PBS. When adding 5 ml for the second time, count cells using a Neubauer counting chamber.</li></ol></li><li>For each electroporation, fill 3 ml PBMC medium per well into a 6-well cell culture plate.</li><li>Centrifuge 2 x 10<sup>6</sup> cells at 250 <em>x g</em> for 10 min in a 15 ml centrifuge tube for each electroporation. Prepare two tubes in order to be able to calculate DNA repair capacity of one sample properly. One tube will contain the irradiated pDsRedExpress-N1 and pEGFP-N1 and the other one the non-irradiated pDsRedExpress-N1 and pEGFP-N1. Resuspend pellet in 150 Âµl RPMI media without phenol red.</li><li>Add 100 Âµl of the previously prepared plasmid mixture to the cell suspension, so that the suspension contains 10 Âµg pEGFP-N1 and 30 Âµg pDsRedExpress-N1 (either irradiated or non-irradiated).</li><li>Transfer 250 Âµl of the cell suspension to electroporation cuvettes with 4 mm gap, place it into the shocking chamber of Gene Pulser II and start electroporation at 400 V and 500 ÂµF. <br/><em>Note: Always electroporate one irradiated batch and one non-irradiated batch directly one after the other. These two will be used to calculate the DNA repair capacity of one sample.</em> </li><li>Transfer electroporation batches in the order of electroporation into the prepared 6-well cell culture plate containing RPMI medium.</li><li>Incubate for 24 h at 37 Â°C, 5 % CO<sub>2</sub> and relative humidity of 95%.<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong>Day 2</strong> <ol class="npwtwo" start="9"><li>Prepare one 1.5 ml reaction tube for each electroporation approach.</li><li>Mix cells in the wells of the cell culture plate using a 1,000 Âµl Eppendorf pipette before transferring 1.5 ml of the cell suspension into the 1.5 ml reaction tubes prepared previously. </li><li>Centrifuge at 250 <em>x g</em> for 10 min. </li><li>Remove 1.4 ml of the supernatant, resuspend cells in the remaining 100 Âµl, and transfer into 0.5 ml FACS tubes.</li><li>Place the 0.5 ml tube into a 5 ml tube and carry out FACS analysis using 488 nm laser.</li></ol><p class="pptt" id="biaoti28680">### Data analysis</p><p class="ppzz">FACS analysis was carried out using FACSCalibur, CellQuest Pro software and the 488 nm laser. Gating of the cells of interest (lymphocytes) was carried out with the control sample. For analysis of the sample containing the irradiated pDsRedExpress-N1 the same gates were used. Gating of the transfected control samples and the samples containing the irradiated pDsRedExpress-N1 are shown in Figures 2 and 3 respectively.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731013540_2997.jpg" width="193"/><br/><strong>Figure 2. FACS analysis of the control sample.</strong> A. Control sample containing the non-irradiated pDsRedExpress-N1 with gated lymphocytes. B. Transfected cells gated with gate R1. The area lower right (R2) displays cells expressing only GFP, the area upper left (R3) displays cells expressing only DsRed and the area upper right (R4) displays cells expressing GFP as well as DsRed.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731013610_4668.jpg" width="193"/><br/><strong>Figure 3. FACS analysis of the samples containing the irradiated pDsRedExpress-N1.</strong> A. Sample containing irradiated pDsRedExpress-N1 with gated lymphocytes. B. Transfected cells gated with gate R1. The area lower right (R2) displays cells expressing only GFP, the area upper left (R3) displays cells expressing only expressing DsRed and the area upper right (R4) displays cells expressing GFP as well as DsRed.<br/><br/>For calculation of DNA repair capacity, percentages of cells expressing GFP and DsRed are added up as follows. All GFP-expressing cells are the cells in gates R2 and R4. All DsRed-expressing cells are the cells in gates R3 and R4. <br/>  For the correction of repair capacity with regard to efficacy of transfection, the correction factor F is needed. F is always calculated from the percentages of GFP and DsRed of the control sample. Hence, a control is needed for each transfection with irradiated pDsRedExpress-N1. F is the ratio of percentages of GFP-expressing cells to the DsRed-expressing cells of the control sample. DNA repair capacity is calculated according to Matt <em>et al.</em> (2016) as follows:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731014014_3468.jpg" width="600"/><br/><br/>For statistical analyses, any version of Prism (GraphPad Software, Inc.) can be used. We recommend triplicates for the calculation of DNA repair capacity.</p><p class="pptt" id="biaoti28681">### Notes</p><ol class="npwtwok"><li>Pay attention to the time constants during electroporation. In our experiments, time constants are usually between 20 and 25 ms. Furthermore, too high time constants resulted in low transfection efficiencies or cell death.</li><li>To allow the laser to warm up, switch on flow cytometer approximately 30 min before FACS analysis.</li><li>To compensate for the non-optimal excitation of DsRed by the 488 nm laser, three times the amount of DsRed compared to GFP is used for electroporation. </li><li>We recommend triplicates for each sample (three control approaches and three matching approaches containing the irradiated pDsRedExpress-N1).<br/></li></ol><p class="pptt" id="biaoti28682">### Recipes</p><ol class="npwtwok"><li>1x PBS<br/>8 g NaCl<br/>0.20 g KCl<br/>2.88 g Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O<br/>1.24 g KH<sub>2</sub>PO<sub>4</sub><br/>Add 1 L ddH<sub>2</sub>O, pH 7.4<br/></li><li>0.9% NaCl solution<br/>9 g NaCl<br/>Add 1 L ddH<sub>2</sub>O, sterile filter solution using 0.2 Âµm sterile filter</li><li>PBMC medium<br/>RPMI 1640 without phenol red<br/>20% fetal bovine serum<br/>Store at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti28683">### Acknowledgments</p><p class="ppzz">The authors thank Dr. med. Karin Rupprecht (Center for Traditional Chinese MedicineâTCM Sigmaringen) and Dr. med. Adrian Schulte (F. X. Mayr Bodensee Center Ãberlingen) for drawing of the blood samples and informing of the volunteers.</p><p class="pptt" id="biaoti28684">### Competing interests</p><p class="ppzz">One of the authors has consulting contracts with MSE Pharmazeutika GmbH, Bad Homburg, Germany.</p><p class="pptt" id="biaoti28685">### Ethics</p><p class="ppzz">All experiments were conducted in accordance with the declaration of Helsinki as well as approved by the ethics committee of the LandesÃ¤rztekammer Baden-WÃ¼rttemberg, Germany. The volunteers were informed before collection of the samples and gave their written consent to the use of their blood samples.</p><p class="pptt" id="biaoti28686">### References</p><ol class="npwtwok"><li>Athas, W. F., Hedayati, M. A., Matanoski, G. M., Farmer, E. R. and Grossman, L. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1933849" target="_blank">Development and field-test validation of an assay for DNA repair in circulating human lymphocytes.</a> <em>Cancer Res</em> 51(21): 5786-5793.</li><li>Burger, K., Matt, K., Kieser, N., Gebhard, D. and Bergemann, J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20569452" target="_blank">A modified fluorimetric host cell reactivation assay to determine the repair capacity of primary keratinocytes, melanocytes and fibroblasts.</a> <em>BMC Biotechnol</em> 10: 46.</li><li>Matt, K., Burger, K., Gebhard, D. and Bergemann, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26879629" target="_blank">Influence of calorie reduction on DNA repair capacity of human peripheral blood mononuclear cells.</a> <em>Mech Ageing Dev</em> 154: 24-29.</li><li>Mendez, P., Taron, M., Moran, T., Fernandez, M. A., Requena, G. and Rosell, R. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21546323" target="_blank">A modified host-cell reactivation assay to quantify DNA repair capacity in cryopreserved peripheral lymphocytes.</a> <em>DNA Repair (Amst)</em> 10(6): 603-610.</li><li>Nouspikel, T. (2009). <a href="https://link.springer.com/article/10.1007/s00018-009-8737-y" target="_blank">DNA repair in mammalian cells-Nucleotide excision repair: variations on versatility.</a> <em>Cell Mol Life Sci</em> 66(6): 994â1009.</li><li>ProtiÄ-SabljiÄ, M., Whyte, D., Fagan, J., Howard, B. H., Gorman, C. M., Padmanabhan, R. and Kraemer, K. H. (1985). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2991746" target="_blank">Quantification of expression of linked cloned genes in a simian virus 40-transformed xeroderma pigmentosum cell line. </a><em>Mol Cell Biol</em> 5(7): 1685-1693.</li><li>Qiao, Y. W., Spitz, M. R., Guo, Z. Z., Hadeyati, M., Grossman, L., Kraemer, K. H. and Wei, Q. Y. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12427537" target="_blank">Rapid assessment of repair of ultraviolet DNA damage with a modified host-cell reactivation assay using a luciferase reporter gene and correlation with polymorphisms of DNA repair genes in normal human lymphocytes.</a> <em>Mutat Res</em> 509(1-2): 165-174.</li><li>Roguev, A. and Russev, G. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11112279" target="_blank">Two-wavelength fluorescence assay for DNA repair.</a> <em>Anal Biochem</em> 287(2): 313-318.</li><li>Shrinivas, S. A., Shanta, S. H., Prajakta, B. B. (2017). <a href="https://juniperpublishers.com/gjpps/GJPPS.MS.ID.555613.php" target="_blank">DNA: damage and repair mechanisms in humans.</a> <em>Glob J Pharmaceu Sci</em> 3(2): 555613.<br/></li></ol>