<body>
            <h1>Cell Type-specific mRNA Purification in <em>Caenorhabditis elegans</em> via Translating Ribosome Affinity Purification</h1>
            <p><strong>Authors:</strong> Ian G. McLachlan, Steven W. Flavell</p>
            <h2>Abstract</h2>
            <p>Cell type-specific molecular profiling is widely used to gain new insights into the diverse cell types that make up complex biological tissues. Translating ribosome affinity purification (TRAP) is a method in which the cell type-specific expression of epitope-tagged ribosomal subunits allows one to purify actively translating mRNAs without the need for cell sorting or fixation. We adapted this method for use in <em>C. elegans</em> to identify novel transcripts in single cell types or to identify the effects of environmental changes on the transcriptomes of larger cohorts of cells. In this protocol, we describe methods to generate transgenic animals bearing tagged ribosomes in cells of interest, prepare these animals for immunoprecipitation, purify ribosome-mRNA complexes, and obtain purified mRNA for next-generation sequencing. </p>
            <p><strong>Keywords:</strong> RNA purification, Expression profiling, RNA-seq, <em>Caenorhabditis elegans</em>, Ribosome profiling</p>
            <h2>Background</h2><p>With the advent of next-generation sequencing technologies, transcriptome profiling of defined cell populations and single cells has reached new heights of popularity. Many new techniques to isolate mRNA from select cells have arisen as a consequence. Translating Ribosome Affinity Purification (TRAP), or ribotagging, is a method to purify polysomal RNA from genetically-defined cell types (Heiman <em>et al</em>., 2008; Sanz <em>et al</em>., 2009; Heiman <em>et al</em>., 2014). In this method, epitope-tagged ribosomes are expressed in cells of interest, then tissue is extracted in the presence of cycloheximide, which stalls the ribosomes on the mRNAs that they are translating and thereby preserves a snapshot of translation. Tagged ribosomes are then affinity purified and the isolated RNA can be analyzed via quantitative PCR, microarrays, or RNA sequencing. This technique is advantageous because it does not require fixation or sorting of cells, has a high degree of sensitivity, and detects only the translated pool of mRNAs. <br/><br/>We recently adapted the TRAP methodology for use in the nematode <em>Caenorhabditis elegans</em>. We have used this method to identify novel transcripts that are enriched in the serotonergic neurosecretory motor (NSM) neuron (Rhoades <em>et al</em>., 2019) and to identify transcripts that vary with environmental changes in the entire nervous system (unpublished results). TRAP is particularly valuable in <em>C. elegans</em> because it is relatively easy to generate new transgenic lines for any cell type of interest, and because it can provide a snapshot of gene expression following an experimental manipulation. TRAP complements previous methods for neuron-specific mRNA isolation in larval and adult <em>C. elegans</em>, such as fluorescence-activated cell sorting (FACS) of dissociated cells (Spencer <em>et al</em>., 2014, Kaletsky <em>et al</em>., 2018) and affinity purification with epitope-tagged poly(A) binding protein (Roy <em>et al</em>., 2002, Takayama <em>et al</em>., 2010). <br/><br/>A TRAP protocol has previously been described in <em>C. elegans</em> (Gracida and Calarco, 2017). Our protocols share many similarities as both are derived from original protocols developed for mouse tissue and cell culture. Some key differences are the use of a distinct ribosomal subunit (tagged <em>rpl-1</em> versus <em>rpl-22</em> here) and epitope tag (GFP versus HA here). Both of these approaches have been successfully used in mammalian systems and appear to be sensitive and robust in <em>C. elegans</em>. The method that we describe here has successfully isolated mRNA from single <em>C. elegans</em> neuronal cell types (Rhoades <em>et al</em>., 2019).</p>
        </body>

<p class="pptt" id="biaoti28714">## Materials and Reagents</p><ol class="npwtwok"><li>15 cm Petri dishes (Corning, catalog number: 351058)</li><li>RNase-free pipette tips<br/></li><li>Glass slides (ThermoFisher, catalog number: 3011-002)</li><li>1.5 ml low-adhesion RNAse-free microcentrifuge tubes (USA Scientific, catalog number: 1415-2600)</li><li>50 ml conical centrifuge tubes (Corning, catalog number: 352098)</li><li>15 ml conical centrifuge tubes (Corning, catalog number: 352097) </li><li> RNase-free 20 Âµl filter tips (Rainin, catalog number: RT-20F)</li><li> RNase-free 200 Âµl filter tips (Rainin, catalog number: RT-200F)</li><li>RNase-free 1,000 Âµl filter tips (Rainin, catalog number: RT-1000F)</li><li>RNase-free disposable serological pipettes (Corning, catalog number: EF20361E)</li><li>RNAse-free disposable pellet pestles with tubes (Thermo Fisher Scientific, catalog number: 12-141-368)</li><li><em>Caenorhabditis elegans</em> (100 young adult animals)</li><li><em>rpl-22</em>-3xHA plasmid (Addgene, catalog number: 122203)</li><li>FseI and AscI restriction enzymes (New England Biolabs, catalog numbers: R0588, R0558)</li><li>Liquid nitrogen</li><li>1,2-diheptanoyl-sn-glycero-3-phosphocholine (DHPC; Avanti, catalog number: 850306P)</li><li>Dynabeads Protein G (Thermo Fisher Scientific, catalog number: 10004D)</li><li> Monoclonal anti-HA antibody, purified, clone HA-7 (Sigma-Aldrich, catalog number: H3663-200UL)</li><li>1 M MgCl<sub>2</sub> (Ambion, catalog number: 9503G)</li><li>2 M KCl (Ambion, catalog number: 9604G)</li><li>Nuclease-free H<sub>2</sub>O (1 L: Ambion, catalog number: 4387936; 100 ml: Ambion, catalog number: 9939)</li><li>RNAse Zap (Ambion, catalog number: AM9782)</li><li>RNAsin RNAse inhibitor (Promega, catalog number: N2515)</li><li>Complete Mini, EDTA-free Protease Inhibitor Tablets (Roche, catalog number:4693124)</li><li>DTT (Sigma-Aldrich, catalog number:D9779-5G)</li><li>Cycloheximide (Sigma-Aldrich, catalog number: C7698-5G)</li><li>Igepal Ca-630 (NP-40) (Sigma-Aldrich, catalog number: I8896-50ML)</li><li>Tetramethyl Sulfolane (for use with Absolutely RNA Nanoprep Kit) (Sigma-Aldrich, catalog number: T2209-100G)</li><li>Absolutely RNA Nanoprep Kit (Agilent, catalog number: 400753). Kit includes the beta-mercaptoethanol (BME).</li><li>Ribonucleoside Vanadyl Complex (VRC) (New England Biolabs, catalog number: S1402S)</li><li>1 M HEPES, pH 7.3 (Thermo Fisher Scientific, catalog number: J16924AE)</li><li>Methanol (Sigma-Aldrich, catalog number: 494437)</li><li>Ethanol (for use with Absolutely RNA Nanoprep Kit) (Sigma-Aldrich, catalog number: E7023)</li><li>Standard laboratory reagents for <em>C. elegans</em> maintenance, molecular cloning, and transgenesis</li><li>Sodium Hypochlorite Solution (Fisher, catalog number: SS290-1)<br/></li><li>Enriched-peptone nematode growth medium (NGM) plates (see Recipes)</li><li>Liquid NGM (standard NGM without agar) (see Recipes)</li><li>100 mg/ml cycloheximide (see Recipes)</li><li>300 mM DHPC stock (see Recipes)</li><li>Homogenization buffer (stock) (see Recipes)</li><li>Homogenization buffer with fresh ingredients (see Recipes)</li><li>150 mM KCl wash buffer (stock) (see Recipes)</li><li>150 mM KCl wash buffer with fresh ingredients (see Recipes)</li><li>350 mM KCl wash buffer (stock) (see Recipes)</li><li>350 mM KCl wash buffer with fresh ingredients (see Recipes)</li></ol><p class="pptt" id="biaoti28715">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>-80 Â°C freezer</li><li>Benchtop microcentrifuge (Thermo Fisher Scientific, catalog number: 75002411 or equivalent)</li><li>Benchtop centrifuge (Thermo Fisher Scientific, catalog number: 75004504 or equivalent)</li><li>Dynamag-2 magnetic rack for 1.5 ml microcentrifuge tubes (Thermo Fisher Scientific, catalog number: 12321D)</li><li>Dynamag-15 magnetic rack for 15 ml conical tubes (optional; Thermo Fisher Scientific, catalog number: 12301D)</li><li>Nanodrop One Spectrophotometer (Thermo Fisher Scientific, model: ND-ONE-W) </li><li>Cold room or refrigerator with access to electrical outlet (4 Â°C)</li><li>Forceps (Fine Science Tools)</li><li>Autoclave</li><li>Standard laboratory equipment for <em>C. elegans</em> maintenance, molecular cloning, and transgenesis<br/></li></ol><p class="pptt" id="biaoti28716">### Procedure</p><p class="ppzz"><em>Note: This protocol assumes basic familiarity with </em><em>C. elegan</em><em>s</em><em> maintenance and transgenesis.<br/></em><br/><img alt="" class="layerphoto" height="300" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730211213_6218.jpg" style="white-space:normal;" width="308"/><br/><b style="white-space:normal;">Figure 1. Workflow for the described protocol.</b> This protocol consists of several steps, each described in detail in the main text.<br/><br/><span style='color:#333333;font-family:"'></span> </p><ol class="npwone"><li>Prepare a transgenic line expressing the tagged ribosome in cell(s) of interest<ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li><em>C. elegans</em> TRAP requires expressing a tagged ribosomal subunit (<em>rpl-22</em>-3xHA cDNA) in cells of interest (Figure 2). Using standard molecular cloning techniques, generate a promoter::<em>rpl-22</em>-3xHA plasmid with any cell-specific promoter of interest. <br/><em>A tph-1p</em>::<em>rpl-22</em>-3xHA expression plasmid that drives expression of RPL-22-3xHA in serotonergic neurons is available via Addgene. In this plasmid, the <em>tph-1</em> promoter is flanked by a 5â² FseI site and 3â² AscI site, so another promoter of choice can easily be swapped in with a FseI/AscI restriction digest. <br/><br/><img alt="" class="layerphoto" height="25" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730211332_6205.jpg"/><br/><strong>Figure 2. Schematic of the <em>rpl-2</em><em>2</em>-3xHA expression vector.</strong> The promoter can be replaced by amplifying a cell-specific promoter of interest by PCR with a 5â² FseI restriction site and 3â² AscI restriction site.<br/><br/></li><li>Inject the promoter::<i style="font-size:10pt;">rpl-22</i>-3xHA plasmid into adult worms with promoter::GFP as a co-injection marker to generate transgenic lines carrying an extrachromosomal array.<ol class="npwthree" style="font-size:13.3333px;"><li>Inject the plasmid at half the normal concentration typically used for the promoter (if known) to reduce the likelihood of off-target expression in other tissues.</li><li>Using a plasmid that drives GFP expression under the same promoter as the <em>rpl-22</em> construct as the sole co-injection marker eliminates the possibility of off-target expression of <em>rpl-22</em> in other tissues as a consequence of plasmid recombination.</li></ol></li><li>Integrate one line by UV and backcross 5x to remove background mutations. We have typically selected transgenic lines based on co-injection marker expression, but anti-HA immunostaining or immunoblot analysis could also be conducted if necessary. We have not attempted TRAP on transgenic animals with extra-chromosomal arrays or MosSCI insertions, but it is possible that these may work.<br/><br/></li></ol></li><li>Grow age-synchronized animals to the appropriate developmental stage<ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Pour 15 cm enriched-peptone NGM plates and allow them to dry at room temperature overnight. These plates allow for robust bacterial growth that can support a very dense nematode culture.</li><li>Seed enriched-peptone NGM plates (one per condition) with 1 ml OP50 and allow to dry. Drip the OP50 over as much of the plate surface as possible, but do not spread it.</li><li>Pick 100 young adult animals onto each plate and allow them to grow for 3 days, until there are many adult progeny and eggs on the plate. We typically prepare four parallel plates to recover enough animals for single-cell TRAP (150,000 animals; see below).</li><li>Wash each plate with 6 ml M9 and transfer animals into a 15 ml conical tube with a serological pipette. To bleach-synchronize animals, add 3 ml bleach and 1 ml 5 N NaOH, then invert occasionally for 5 min until adult animals are broken up.</li><li>Centrifuge at room temperature at 1,300 <i style="font-size:10pt;">x g</i> for 1 min to pellet eggs, aspirate the solution without disturbing the pellet, and wash with 10 ml M9. Repeat wash 2 x and resuspend in a small volume of M9.</li><li>Eggs obtained from the above procedure can be plated directly on a freshly-seeded enriched-peptone plate. We plate all of the eggs recovered from one bleached 15 cm plate on to a single freshly-seeded enriched-peptone 15 cm NGM plate. <br/><em>Note: We typically avoid plating arrested L1s due to concerns that developmental arrest could affect gene expression. However, animals are not perfectly synchronized if L1 starvation is omitted. If perfect synchronization is required, we suggest including this L1 starvation step. </em> </li><li>Allow worms to grow at 20 Â°C tothe desired developmental stage. Animals will reach adulthood roughly 60 h after plating. Ensure that animals are not starved on the day of the experiment, unless this is desired as part of the experimental design.<br/><br/></li></ol></li><li>Wash, freeze, and lyse animals in the presence of cycloheximid<ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>At least one day in advance, prepare liquid NGM, homogenization buffer stock, and wash buffer stocks (see Recipes). Liquid NGM can be stored at room temperature, and homogenization and wash buffers are stable at 4 Â°C for one month.</li><li>On the day of harvesting, add fresh reagents (DTT, protease inhibitor, RNasin, and VRC) to an aliquot of homogenization buffer (1 ml per plate).</li><li>Prepare cycloheximide stock and add the stock to the following solutions at a final concentration of 0.8 mg/ml (125x dilution): liquid NGM (20ml per plate), homogenization buffer (10 ml per plate), and homogenization buffer with fresh reagents (1 ml per plate). Please note that cycloheximide is hazardous and should be handled with proper precautions.</li><li>Clean your workstation, equipment, and gloves thoroughly with RNase ZAP.</li><li>Place the cardboard insert from a microcentrifuge tube cryo box at the bottom of the ice bucket, and place RNase-free tubes (six tubes per plate) from the mortar/pestle set in the insert (Figure 3).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190730/20190730211459_0766.jpg" style="font-size:13.3333px;" width="100"/><br/><strong>Figure 3. Photograph of set-up for grinding frozen pellets.</strong> Fill an ice bucket and conical tube with liquid nitrogen. Cool pestle in a microcentrifuge tube and use it to grind the frozen sample. To cool the sample, use forceps to grip a microcentrifuge tube, and use it to transfer liquid nitrogen from the conical tube to the sample.<br/><br/></li><li>Add liquid nitrogen to the ice bucket, filling the RNase-free tubes and the bottom of the bucket. Do not fill the bucket beyond the height of the tubes; only add enough extra liquid nitrogen to cool the outside of the tubes and prevent the evaporation of the liquid nitrogen in the tubes.</li><li>Wash worms off each plate in 10 ml liquid NGM + cycloheximide and transfer them into a 15 ml conical tube with a serological pipette. Take care not to disrupt the bacterial lawn during the wash to avoid transferring excess bacteria to the tube.</li><li>Spin at room temperature at 1,300 <em>x g</em> for 30 s, aspirate solution, then wash worms with 10 ml liquid NGM + cycloheximide.</li><li>Spin at 1,300 <em>x g</em> for 30 s, aspirate solution, then wash worms with 10 ml homogenization buffer + cycloheximide and transfer the solution to a fresh 15 ml conical tube.</li><li>Spin at 1,300 <em>x g</em> for 30 s, aspirate solution, and add 1 ml homogenization buffer with cycloheximide and fresh reagents.</li><li>Using an RNase free pipette, drip the worm suspension into the RNase free tubes filled with liquid nitrogen, splitting the worm suspension evenly across five tubes. Place a small drop in a sixth tube that will be later used to estimate worm concentration. The worm suspension will bead up and freeze immediately upon contact with the liquid nitrogen. You do not need to record the volumes of these drops, as this will be estimated in a subsequent step.<br/><i>Note: Do not close the tubes until all of the liquid nitrogen has evaporated or been poured off.</i><br/></li><li>Store tubes at -80 Â°C.<br/><br/></li></ol></li><li>Estimation of worm concentration<ol class="npwtwo"><li>To estimate worm concentration, thaw the extra tube of frozen worms, then dilute it 10x and pipet 10 Âµl onto a glass slide. </li><li>Count the number of worms present in the droplet. </li><li>Repeat this 2 x and calculate the average to estimate the number of worms per Âµl in each frozen sample.<br/><br/></li></ol></li><li>Grind the frozen worm pellets using the RNase-free pestles<ol class="npwtwo"><li>Add liquid nitrogen to an RNase-free ice bucket with a cardboard insert, as before. Fill a 50 ml RNase-free conical tube with liquid nitrogen to act as a reservoir (Figure 3).</li><li>Cool a pestle by dipping it into an RNase-free microcentrifuge tube filled with liquid nitrogen. </li><li>Remove one tube from the -80 Â°C freezer and place it in the bucket. Grind the sample thoroughly with a pestle, ensuring that it remains frozen and will be ground up into a fine powder. Whenever the sample begins to thaw slightly, cool it again by pouring liquid nitrogen over it. <br/><em>Notes:<ol class="npwthree"><li>It is easiest to cool the sample by using forceps to dip an RNase-free microcentrifuge tube into the reservoir of liquid nitrogen, then pouring it over the sample. Continually cool the pestle as well. <br/></li><li>This process can take quite some time; be patient and continue to cool the sample, ensuring that it does not thaw. Avoid overly aggressive grinding, as this could break the tube or cause the sample to fly out of the tube. </li><li>Repeat for each tube until all samples have been ground into powder. </li><li>Place the tubes back at -80 Â°C for storage. Samples can be stored at -80 Â°C for weeks before proceeding to immunoprecipitation step.<br/><br/></li></ol></em> </li></ol></li><li>Stock buffer preparation<br/>Prepare the liquid NGM, homogenization buffer stock, 150 mM KCl wash buffer stock, and 350 mM KCl wash buffer stock at least one day before performing the immunoprecipitation. The homogenization buffer and wash buffers can be stored for up to one month at 4 Â°C<br/><br/></li><li>Perform immunoprecipitation<ol class="npwtwo"><li>Clean your workstation and all equipment thoroughly with RNAse Zap. All of the following steps should be performed on ice, keeping the samples cold throughout. </li><li>Thaw the worm powder by diluting it into cold homogenization buffer (with all fresh ingredients added) to a concentration of 8 worms/Âµl and transfer it to a 15 ml conical tube. Based on the total volume of extract used, we estimate that our typical starting material for single cell type pull downs is â¼150,000 animals (typically pooled from several plates). Ensure that samples remain cold to prevent RNA degradation.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Use a small known volume of homogenization buffer to gently resuspend the worm powder in each microcentrifuge tube. Once all powder is transferred to the 15 ml tube, measure the displacement with a serological pipette to determine the amount of powder added to the tube, then add extra homogenization buffer to bring the solution to the correct concentration. </em> </li><li><em>This dilution is very important, high concentrations of animals can lead to substantial RNA degradation, presumably due to endogenous RNases. </em> </li></ol></li><li>Add 1/10 volume of 10% NP-40 and 1/9 volume of 300 mM DHPC to the 15 ml tube. Incubate on ice for 10 min, inverting occasionally. </li><li>Transfer the solution to RNase-free microcentrifuge tubes and spin for 12 min at 15,000 x g at 4 Â°C. Transfer the supernatant into fresh tubes, then centrifuge again, and pool the supernatant into a fresh 15 ml RNase-free conical tube. </li><li>While centrifuging the samples, wash the Protein G beads in 150 mM KCl Wash Buffer. Add 1 ml of 150 mM KCl Wash Buffer to 150 Âµl beads (per IP; use 300 Âµl beads per IP if pre-clearing), collect the beads with a magnetic rack, aspirate wash buffer, and repeat 2 x. </li><li><em>(OPTIONAL)</em> To pre-clear the lysate, add 150 Âµl washed Protein G beads to the lysate and rotate at 4 Â°C for 30 min, then collect the beads with the magnetic rack and transfer the cleared supernatant to a fresh 15 ml conical tube. We have typically applied this step when performing TRAP on single cells, but have skipped it when performing TRAP on large numbers of cells (for example, during pan-neuronal TRAP).</li><li>Collect a 100 Âµl âinputâ sample at this point and mix with 100 Âµl of lysis buffer from the Agilent Absolutely RNA Nanoprep Kit. Ensure that beta-mercaptoethanol (BME) has been added to the lysis buffer, as per kit protocol. Vortex, let sit at room temperature for 10 min, vortex again, then store at -80 Â°C. </li><li>Add 3.5 Âµl monoclonal anti-HA antibody to pre-cleared lysate and rotate at 4 Â°C for 30 min. For large numbers of cells (pan-neuronal TRAP), we have used 10 Âµl of antibody. </li><li>Add 150 Âµl Protein G beads to the lysate and rotate at 4 Â°C for 30 min. </li><li>Place the 15 ml conical tube on the magnetic rack and collect a 100 Âµl âflowthroughâ sample and process in the same manner as the âinputâ material. This sample is valuable to check if there was any RNA degradation during the incubation with antibody and beads.</li><li>Wash twice with 350 mM KCl Wash Buffer, each time gently inverting the tube then placing it on the magnetic rack to aspirate supernatant. Resuspend the beads in 350 mM KCl Wash Buffer and transfer to a clean RNase-free microcentrifuge tube. Wash once more with 350 mM KCl Wash Buffer, place on the magnetic rack, and aspirate supernatant (in total, the beads should be washed 4 x with 350 mM KCl Wash Buffer).<br/></li><li>Elute RNA by adding 100Âµl lysis buffer with BME (from the Agilent Absolutely RNA Nanoprep Kit), vortex, and let the sample sit for 10 min at room temperature with occasional agitation. Place the tube on the magnetic rack, collect liquid in a fresh, chilled RNase-free microcentrifuge tube, and store at -80 Â°C. Beads can be discarded after elution.<br/><br/></li></ol></li><li>RNA purification<br/>Purify RNA with the Agilent Absolutely RNA Nanoprep Kit as per the kit protocol, including the on-column DNase treatment. Typical RNA concentrations (measured by nanodrop) are 30-200 ng/Âµl for input samples and 4-10 ng/Âµl for IP samples in a volume of 10 Âµl.<br/></li></ol><p class="pptt" id="biaoti28717">### Data analysis</p><ol class="npwtwok"><li>Check for RNA degradation by running the input, flowthrough, and IP samples on an Agilent Bioanalyzer chip (nano or pico, depending on concentration). RNA integrity number (RIN) values should be greater than 7.5 to proceed, and values greater than 8.5 are typical in experienced hands. We find that the IP samples can show relatively lower RNA degradation regardless of the quality of the input, so pay particular attention to input and flowthrough quality to ensure that the procedure went well and that the IP sample is representative.</li><li>Quantify positive control genes via RT-qPCR. Reverse transcribe RNA with a Superscript II kit with random hexamers to create a cDNA sample for qPCR. For low RNA amounts, we have also used Nugen Ovation RNA amplification system v2 to perform RT reactions. Use a 2x dilution of this cDNA for qPCR. We have used <em>cdc-42</em> and <em>unc-37</em> as ubiquitously-expressed control genes, and <em>rab-3</em> and <em>tag-168</em> as pan-neuronal genes. Ensure that you see enrichment for known genes in your IP sample relative to the input before proceeding to RNA sequencing. </li></ol><p class="pptt" id="biaoti28718">### Notes</p><ol class="npwtwok"><li>The details of preparing and analyzing RNA-seq data sets are beyond the scope of this protocol, but we typically prepare libraries with a Takara SMART-Seq Ultra Low Input RNA Kit for sequencing on an Illumina platform.</li><li>We and other laboratories have found that standard oligo-dT priming is not sufficient to deplete ribosomal RNA from <em>C. elegans</em> libraries. Nevertheless, we typically recover enough non-rRNA reads to have sufficient sequencing depth for most purposes (&gt; 20 million non-rRNA reads per library). </li></ol><p class="pptt" id="biaoti28719">### Recipes</p><ol class="npwtwok"><li>Enriched-peptone nematode growth medium (NGM)<ol class="npwthree" style="white-space:normal;"><li>Per 1 L of NGM, add to a 2 L or larger Erlenmeyer flask:<br/>22 g agar<br/>20 g peptone<br/>3 g NaCl<br/>975 ml H<sub>2</sub>O<br/></li><li>Autoclave, allow to cool to 55 Â°C, and add:<br/>1 ml 1M CaCl<sub>2</sub><br/>1 ml 1M MgSO<sub>4</sub><br/>25 ml 1M KPO<sub>4</sub><br/>1 ml 5 mg/ml cholesterol in ethanol<br/></li><li>Pour 50 ml per 15 cm Petri dish and allow to set completely. Store inverted at 4 Â°C</li></ol></li><li>M9 Solution<br/>3 g KH<sub>2</sub>PO<sub>4</sub><br/>6 g Na<sub>2</sub>HPO<sub>4</sub><br/>5 g NaCl<br/>Suspend in 1 L of H<sub>2</sub>O and autoclave<br/>After cooling, add 1 ml of sterile 1 M MgSO<sub>4</sub> </li><li>Liquid NGM (standard NGM without agar)<ol class="npwthree" style="font-size:13.3333px;white-space:normal;color:#666666;"><li>Per 1 L of liquid NGM, add to a capped bottle:<br/>2.5 g peptone<br/>3 g NaCl<br/>975 ml H<sub>2</sub>O</li><li>Autoclave, allow to cool to 55 Â°C, and add: <br/>1 ml 1 M CaCl<sub>2</sub><br/>1 ml 1 M MgSO<sub>4</sub><br/>25 ml 1 M KPO<sub>4</sub><br/>1 ml 5 mg/ml cholesterol in ethanol</li><li>Store at room temperature</li></ol></li><li>100mg/ml cycloheximide<ol class="npwthree" style="white-space:normal;"><li>Add 500 Âµl methanol to 50 mg cycloheximideb.</li><li>Make a fresh stock each day, do not store</li></ol></li><li>300 mM DHPC stock<ol class="npwthree"><li>Add 1.38 ml nuclease-free H<sub>2</sub>O to 200 mg DHPC, and allow &gt; 60 min to fully resuspend</li><li>Store at 4 Â°C up to one month</li></ol></li><li>Homogenization buffer (stock)<ol class="npwthree" style="white-space:normal;"><li>Remove 9 ml from a 100 ml bottle of nuclease-free H<sub>2</sub>O</li><li>Add: <br/>1 ml 1 M HEPES, pH 7.3 (10 mM final conc.)<br/>7.5 ml 2 M KCl (150 mM final conc.)<br/>0.5 ml 1 M MgCl<sub>2</sub> (5 mM final conc.)</li><li>Store at 4 Â°C up to one month</li></ol></li><li>Homogenization buffer with fresh ingredients<br/>Add to 10 ml homogenization buffer stock:<br/>5 Âµl 1 M DTT (0.5 mM final conc.)<br/>100 Âµl RNAsin RNAse inhibitor (note that this is a higher concentration than wash buffers)<br/>0.5 ml Ribonucleoside Vanadyl Complex (VRC)<br/>10 Âµl 100 mg/ml cycloheximide (100 Âµg/ml final conc.) <br/><em>Note: We use a higher concentration of cycloheximide (0.8 mg/ml final concentration) in the liquid NGM and homogenization buffer applied to live animals on the day of freezing. Then, we use a lower concentration of cycloheximide (100 Âµg/ml final concentration) in subsequent lysis and immunoprecipitation steps.</em><br/>1 protease inhibitor tablet</li><li>150 mM KCl wash buffer (stock) <ol class="npwthree"><li>Remove 19 ml from a 100 ml bottle of nuclease-free H<sub>2</sub>O</li><li> Add:<br/>1 ml 1 M HEPES, pH 7.3 (10 mM final conc.)<br/>7.5 ml 2M KCl (150 mM final conc.)<br/>0.5 ml 1M MgCl<sub>2</sub> (5 mM final conc.)<br/>10 ml 10% NP-40 (1% final conc.)</li><li>Store at 4 Â°C up to one month</li></ol></li><li>150 mM KCl wash buffer with fresh ingredients<br/>Add to 10 ml 150 mM KCl wash buffer stock:<br/>5 Âµl 1 M DTT (0.5 mM final conc.)<br/>10 Âµl RNAsin RNase inhibitor<br/>10 Âµl 100 mg/ml cycloheximide (100 Î¼g/ml final conc.)<br/><em>Note: Prepare just before use.</em> </li><li>350 mM KCl wash buffer (stock) <ol class="npwthree"><li>Remove 29 ml from a 100 ml bottle of nuclease-free H<sub>2</sub>O</li><li>Add:<br/>1 ml 1 M HEPES, pH 7.3 (10 mM final conc.)<br/>17.5 ml 2M KCl (350 mM final conc.)<br/>0.5 ml 1M MgCl<sub>2</sub> (5 mM final conc.)<br/>10 ml 10% NP-40 (1% final conc.)</li><li>Store at 4 Â°C up to one month</li></ol></li><li>350 mM KCl wash buffer with fresh ingredients<br/>Add to 10 ml 350 mM KCl wash buffer stock:<br/>5 Âµl 1 M DTT (0.5 mM final conc.)<br/>10 Âµl RNAsin RNase inhibitor<br/>10 Âµl 100 mg/ml cycloheximide (100 Âµg/ml final conc.) <br/><em>Note: Prepare just before use.</em> </li></ol><p class="pptt" id="biaoti28720">### Acknowledgments</p><p class="ppzz">We thank Cori Bargmann, Eric Schmidt, Ivo Spiegel, and Myriam Heiman for help during early phases of developing this protocol. We also thank Max Heiman for discussions about effective cycloheximide concentrations in <em>C. elegans</em>. Howard Hughes Medical Institute supported early phases of this project. I.G.M. was supported by a JPB Postdoctoral Fellowship. S.W.F. acknowledges support from the JPB Foundation, PIIF, and PNDRF.</p><p class="pptt" id="biaoti28721">### Competing interests</p><p class="ppzz">There are no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti28722">### References</p><ol class="npwtwok"><li>Gracida, X. and Calarco, J. A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28648677" target="_blank">Cell type-specific transcriptome profiling in <em>C. elegans</em> using the Translating Ribosome Affinity Purification technique.</a> <em>Methods</em> 126: 130-137. </li><li>Heiman, M., Schaefer, A., Gong, S., Peterson, J. D., Day, M., Ramsey, K. E., Suarez-Farinas, M., Schwarz, C., Stephan, D. A., Surmeier, D. J., Greengard, P. and Heintz, N. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19013281" target="_blank">A translational profiling approach for the molecular characterization of CNS cell types.</a> <em>Cell</em> 135(4): 738-748.</li><li>Heiman, M., Kulicke, R., Fenster, R. J., Greengard, P. and Heintz, N. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24810037" target="_blank">Cell type-specific mRNA purification by translating ribosome affinity purification (TRAP).</a> <em>Nat Protoc</em> 9(6): 1282-1291. </li><li>Kaletsky, R., Yao, V., Williams, A., Runnels, A. M., Tadych, A., Zhou, S., Troyanskaya, O. G. and Murphy, C. T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30096138" target="_blank">Transcriptome analysis of adult <em>Caenorhabditis elegans</em> cells reveals tissue-specific gene and isoform expression.</a> <em>PLoS Genet</em> 14(8): e1007559.</li><li>Rhoades, J. L., Nelson, J. C., Nwabudike, I., Yu, S. K., McLachlan, I. G., Madan, G. K., Abebe, E., Powers, J. R., ColÃ³n-Ramos, D. A. and Flavell, S. W. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30580965" target="_blank">ASICs mediate food responses in an enteric serotonergic neuron that controls foraging behaviors.</a> <em>Cell</em> 176:85-97.e14.</li><li>Roy, P. J., Stuart, J. M., Lund, J. and Kim, S. K. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12214599" target="_blank">Chromosomal clustering of muscle-expressed genes in <em>Caenorhabditis elegans</em>.</a> <em>Nature</em> 418:975-979.</li><li>Sanz, E., Yang, L., Su, T., Morris, D. R., McKnight, G. S. and Amieux, P. S. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19666516/?otool=None" target="_blank">Cell-type-specific isolation of ribosome-associated mRNA from complex tissues.</a> <em>Proc Natl Acad Sci U S</em> <em>A</em> 106:13939-13944.</li><li>Spencer, W. C., McWhirter, R., Miller, T., Strasbourger, P., Thompson, O., Hillier, L. W., Waterston, R. H., and Miller, D. M. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25372608/?otool=None" target="_blank">Isolation of specific neurons from <em>C. elegans</em> larvae for gene expression profiling. </a><em>PLoS ONE</em> 9: e112102.</li><li>Takayama, J., Faumont, S., Kunitomo, H., Lockery, S.R. and Iino, Y. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1093%2Fnar%2Fgkp868" target="_blank">Single-cell transcriptional analysis of taste sensory neuron pair in <em>Caenorhabditis elegans</em>.</a> <em>Nucleic Acids Res</em> 38: 131-142.<br/><br/></li></ol><p class="pptt" id="biaoti_no">### </p><span id="__kindeditor_bookmark_start_0__" style="display:none;"></span>