<body>
            <h1>Strand-specific Single-stranded DNA Sequencing (4S-seq) of <em>E. coli</em> genomes</h1>
            <p><strong>Authors:</strong> Takahiro Masuda, Nobuaki Kono, Masaru Tomita, Kazuharu Arakawa</p>
            <h2>Abstract</h2>
            <p>Most bacterial genomes have biased nucleotide composition, and the asymmetry is considered to be caused by a single-stranded DNA (ssDNA) deamination arising from the bacterial replication machinery. In order to evaluate the relationship experimentally, the position and frequency of ssDNA formed during replication must be verified clearly. Although many ssDNA detection technologies exist, almost all methods have been developed for eukaryotic genomes. To apply these to bacterial genomes, which harbor a smaller amount of DNA than those of eukaryotes, more efficient, new methods are required. Therefore, we developed a novel strand-specific ssDNA sequencing method, called 4S-seq, for the bacterial genome. The 4S-seq method enriches ssDNA in the extracted genomic DNA by a dsDNA-specific nuclease and implements a strand-specific library using a biotin label with a customized tag. As a result, the 4S-seq is able to calculate the ssDNA content in each strand (Watson/Crick) at each position of the genome efficiently.</p>
            <p><strong>Keywords:</strong> Bacterial genome, Single-stranded DNA, Nascent DNA, Strand-specific sequencing, Library preparation</p>
            <h2>Background</h2><p>Nascent and template strand tracking is an important approach to investigate bacterial genome evolution. Since biased mutations in bacterial genome arise from asymmetric replication machinery, DNA strands that have been exposed to a single-stranded state for a long time will accumulate more mutations. Several strand-specific sequencing technologies are available for eukaryotic genomes, with various selection strategies for nascent DNA (Hyrien, 2015). In general, the nascent DNA is caught by immunoprecipitation with BrdU (Karnani <em>et al</em>., 2010) or a combination of antibodies against strand-specific binding proteins (Yu <em>et al</em>., 2014). Other protocols utilize a customized label (Smith and Whitehouse, 2012) or an agarose gel trap (Mesner <em>et al</em>., 2011) to select nascent DNA of a certain size. However, these strand-specific sequencing technologies for eukaryotic genomes are not readily applicable to bacterial genomes because the BrdU incorporation efficiency or the amount of DNA is overwhelmingly low in bacteria. Therefore, we developed a novel 4S-seq method for the bacterial genome. Overall, the 4S-seq method is broadly divided into ssDNA enrichment and strand-specific sequencing steps (Figure 1). The ssDNA is enriched in the fragmented genomic DNA using a duplex-specific nuclease. The strand-specific sequencing is implemented using a biotinylated adapter with an exclusively unique designed tag. At this time, the library is prepared using a low input library preparation kit because a low amount of enriched ssDNA is expected. Sequenced reads are analyzed with a custom Perl script, and the ssDNA location and strand are calculated. The new 4S-seq method for the bacterial genome described here does not require specialized equipment and can be applied from plasmid to mega-scale genomes. This 4S-seq was applied to research investigating the relationship between a base compositional bias in a bacterial genome and a spontaneous mutation in ssDNA, and revealed that there is a strong correlation between ssDNA frequency and the mutation rate (Kono <em>et al</em>., 2018). We show the protocol optimized for <em>Escherichia coli</em>.â</p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731221149_0167.jpg"/><br/><strong>Figure 1. Overview of the 4S-seq protocol.</strong> Steps A-C: purification of the sheared genomic DNA from E. coli cultures; Steps D-F: preparation of dephosphorylated ssDNA and the phosphorylated adapter; Steps G-J: purification of tagged ssDNA fragments using streptavidin beads; Step K: preparation of the library for the Illumina sequencer.</p><p class="pptt" id="biaoti28725">## Materials and Reagents</p><ol class="npwtwok"><li>1.5 ml tubes (Eppendorf, catalog number: 30108051)</li><li>0.2 ml tubes (NIPPON GENE CO., LTD., catalog number: BPB79211)</li><li>1000 ml pipette tips (Sorenson, catalog number: 34000)</li><li>200 ml pipette tips (NIPPON GENE CO., LTD., catalog number: FG-302RS)</li><li>10 ml pipette tips (Sorenson, catalog number: 37660)</li><li>TapeStation D1000 Screen Tape (Agilent Technologies, catalog number: 5067-5582)</li><li>14 ml tube (CORNING, catalog number: 352057)</li><li>50 ml tube (Greiner Bio-One, catalog number: 227281)</li><li>milliTUBE 1 ml AFA Fiber (Covaris, catalog number: 520131)</li><li>PCR Clean-up Column</li><li><em>E. coli</em> strains to be analyzed</li><li>Bacto<sup>TM</sup> Tryptone (Thermo Fisher Scientific, catalog number: 211705)</li><li>Bacto<sup>TM</sup> Yeast Extract (Thermo Fisher Scientific, catalog number: 212750)</li><li>NaCl (FUJIFILM Wako Pure Chemical Corporation, catalog number: 192-13925)</li><li>Appropriate antibiotics</li><li>Milli-Q water</li><li>10x PBS (NIPPON GENE CO., LTD., catalog number: 314-90185)</li><li>NucleoSpin<sup>Â®</sup> Gel and PCR Clean-up (MACHEREY-NAGEL, catalog number: 74069.250)<br/><ol class="npwthree" style="white-space:normal;"><li>Binding Buffer NTI<div style="white-space:nowrap;"></div></li><li>Wash Buffer NT3</li><li>Elution Buffer NE</li><li>NucleoSpin<sup>Â®</sup> Gel and PCR Clean-up Columns</li><li>2 ml Collection Tube</li></ol></li><li>Buffer NTC (MACHEREY-NAGEL, catalog number: 740654.100)</li><li>Qubit<sup>TM</sup> dsDNA BR Assay Kit (Thermo Fisher Scientific, catalog number: Q32850)</li><li>Duplex-specific nuclease (Evrogen, catalog number: EA001)<ol class="npwthree" style="white-space:normal;"><li>DSN enzyme</li><li>DSN storage buffer</li><li>10x DSN master buffer</li></ol></li><li>TapeStation D1000 Reagents (Agilent Technologies, catalog number: 5067-5583)<br/></li><li>0.5 M EDTA (FUJIFILM Wako Pure Chemical Corporation, catalog number: 311-90075)</li><li>Agencourt AMPure XP (BECKMAN COULTER Life Sciences, catalog number: A63881)</li><li>EtOH (FUJIFILM Wako Pure Chemical Corporation, catalog number: 054-7225)</li><li>Sucrose (FUJIFILM Wako Pure Chemical Corporation, catalog number: 196-00015)</li><li>Lysozyme (FUJIFILM Wako Pure Chemical Corporation, catalog number: 123-06721)</li><li>100 mg/ml RNase A (QIAGEN, catalog number: 19101)</li><li>1 M Tris-HCl (pH 8.0) (NIPPON GENE CO., LTD., catalog number: 312-90061)</li><li>1 M Tris-HCl (pH 7.5) (NIPPON GENE CO., LTD., catalog number: 312-90221)</li><li>FastAP Thermosensitive Alkaline Phosphatase (Thermo Fisher Scientific, catalog number: EF0651)<ol class="npwthree" style="white-space:normal;"><li>FastAP Thermosensitive Alkaline Phosphatase</li><li>10x FastAP Buffer</li></ol></li><li>Oligo nucleotides<ol class="npwthree"><li>Adapter_N8B 5â²-GGGAANNNNNNNNTAGGGATAACAGGGTAATAGGAGGA-3â² (Eurofins Genomics, custom made)</li><li>1stAdapter_primer 5â²-TCCTCCTATTACCCTGTTATCCCTA-3â² (Eurofins Genomics, custom made)</li></ol></li><li>T4 Polynucleotide Kinase (T4 PNK) and 10x T4 PNK buffer (NEW ENGLAND BioLab, catalog number: M0201S)</li><li>100 mM ATP (TAKARA BIO INC., catalog number: 4041)</li><li>T4 DNA ligase and 10x T4 DNA ligase buffer (Enzymatics, catalog number: L6030-HC-L)</li><li>Dynabeads MyOne Streptavidin C1 (Invitrogen, catalog number: DB65001)</li><li>Tks Gflex DNA Polymerase (Gflex) and 2x Gflex PCR Buffer (TAKARA BIO INC., catalog number: R060A)</li><li>I-Sce I and 10x CutSmart buffer (NEW ENGLAND BioLab, catalog number: R0694S)</li><li>SMARTer ThruPLEX DNA-Seq Kit (TAKARA BIO INC., catalog number: RB4674)</li><li>(Optional) Low input Library Prep Kit (Clontech Laboratories, Inc., catalog number: 634947)</li><li>Qubit ssDNA Assay Kit (Thermo Fisher Scientific, Invitrogen, catalog number: Q10212)</li><li>NextSeq 500 High Output Kit (75 cycles) (Illumina, Inc., catalog number: TG-160-2005)</li><li>Luria-Bertani broth (LB) (see Recipes)</li><li>TE buffer, pH 8.0 (see Recipes)<br/></li><li>Lysis buffer (see Recipes)</li><li>80% EtOH (see Recipes)</li><li>2x Binding and Washing (B&amp;W) buffer (see Recipes)</li><li>1x B&amp;W buffer (see Recipes)</li></ol><p class="pptt" id="biaoti28726">### Equipment</p><ol class="npwtwok"><li>200 ml flask (Sansyo Co., ltd, catalog number: 82-0086)</li><li>Pipettes (GILSON, catalog number: F167380)</li><li>Digital photometer (TAITEC, catalog number: 0040889-000)</li><li>Milli-Q water purification system (Merck, model: Direct-Q UV 5)</li><li>Autoclave (TOMY, model: LSX-300)</li><li>Centrifuge (TOMY, model: MX-307)</li><li>Incubator (TAIYO COMPANY CO., LTD, catalog number: 2-6795-02)</li><li>Covaris<sup>TM</sup> M220 Focused-ultrasonicator<sup>TM</sup> Instrument (Thermo Fisher Scientific, catalog number: 4482277)</li><li>Qubit 3.0 Fluorometer (Thermo Fisher Scientific, catalog number: Q33216)</li><li>Heat block (WAKENBTECH CO., LTD, catalog number: WKN-9603)</li><li>Thermal Cycler (LifeECO ver2.0) (NIPPON Genetics Co., Ltd, catalog number: TC-96GHbC)</li><li>Agilent 2200 TapeStation (Agilent Technologies, catalog number: G2965AA)</li><li>Magnetic stand for 0.2 ml tube (NIPPON Genetics Co., Ltd, catalog number: FG-SSMAG2)</li><li>Magnetic stand for 1.5 ml tube (Thermo Fisher Scientific, catalog number: 12321D)</li><li>NextSeq 500 instrument (Illumina, Inc., catalog number: SY-415-1001)<br/></li></ol><p class="pptt" id="biaoti28727">### Software</p><ol class="npwtwok"><li>bcl2fastq (v. 2.18.0.12) (<a href="https://support.illumina.com/sequencing/sequencing_software/bcl2-fastq-conversion-software.html" target="_blank">https://support.illumina.com/sequencing/sequencing_software/bcl2-fastq-conversion-software.html</a>)</li><li>BWA (v. 0.7.1-r1034) (<a href="http://bio-bwa.sourceforge.net/" target="_blank">http://bio-bwa.sourceforge.net</a>)</li><li>Samtools (v. 1.2) (<a href="http://samtools.sourceforge.net/" target="_blank">http://samtools.sourceforge.net</a>)</li><li>FastQC (v. 0.10.1) (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>)<br/></li></ol><p class="pptt" id="biaoti28728">### Procedure</p><ol class="npwone"><li>Cell collection<ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li><em>E. coli</em> strains are inoculated by picking a single colony and cultured in 2 ml of LB with appropriate antibiotics using 14 ml tubes until they reach an optical density (OD) at 600 nm of 0.4 with a digital photometer.</li><li>Fifty microliters of culture is inoculated to 50 ml of LB with appropriate antibiotics in a 200 ml flask and cultured until they reach an OD at 600 nm of 0.4 with a digital photometer.</li><li>Transfer 50 ml of culture to a 50 ml tube and collect cells by centrifugation at 4,000 <em>x g</em> for 5 min at 4 Â°C, then remove the supernatant.</li><li>After washing the cell pellet twice each with chilled 1 ml of PBS (pH 8.0), resuspend the cell pellet in 2 ml of Lysis buffer and incubate at 37 Â°C for 30 min.</li><li>Collect cells by centrifugation at 8,000 <em>x g</em> for 5 min and gently remove the supernatant.<br/><br/></li></ol></li><li>DNA shearing<ol class="npwtwo"><li>Resuspend the cell pellet in 1 ml of TE buffer (pH 8.0) and transfer the cell lysate to a milliTUBE 1 ml AFA Fiber.</li><li>Transfer the milliTUBE into a Covaris<sup>TM</sup> M220 Focused-ultrasonicator<sup>TM</sup> Instrument and conduct the following shearing protocol [peak power (W): 75.0; duty factor (%): 5.0; cycles/burst: 200; treatment (sec): 1200; temperature (Â°C): 9.0 (maximum), 7.0 (setpoint), 5.0 (minimum)] to dissociate the genomic DNA into 300-400 bp fragments.<br/><br/></li></ol></li><li>Purification of sheared genomic DNA<ol class="npwtwo"><li>Dispense a 125 Î¼l aliquot of the total 1 ml sheared DNA solution into eight 1.5 ml <br/><em>Note: The capacity of the NucleoSpin Gel is limited (â¤ 25 Î¼g, 50 bp-20 kb). Purification of sheared genomic DNA is performed at room temperature.</em><em></em><br/></li><li>Mix sheared DNA solution with 250 Î¼l Binding Buffer NTC.</li><li>Place a NucleoSpin<sup>Â®</sup> Gel and PCR Clean-up Column into a 2 ml Collection Tube and load sample. </li><li>Centrifuge for 30 s at 11,000 <em>x g</em>.</li><li>Discard flow-through and place the column back into the collection tube.</li><li>Add 700 Î¼l Wash Buffer NT3 buffer to the NucleoSpin<sup>Â®</sup> Gel and PCR Clean-up Column.</li><li>Centrifuge for 30 s at 11,000 <em>x g</em>.</li><li>Discard flow-through and place the column back into the collection tube.</li><li>Centrifuge for 1 min at 11,000 <em>x g </em>to remove Wash Buffer NT3.</li><li>Place a NucleoSpin<sup>Â®</sup> Gel and PCR Clean-up Column into a new 1.5 ml tube.</li><li>Add 20 Î¼l Elution Buffer NE and incubate at room temperature (18-25 Â°C) for 1 min.</li><li>Centrifuge for 1 min at 11,000 <em>x g</em>.</li><li> Pool all eight eluted DNA solutions into a new 1.5 ml tube.<br/><br/></li></ol></li><li>Duplex-specific nuclease reaction<ol class="npwtwo"><li>Determine the DNA concentration of purified genomic DNA with Qubit<sup>TM</sup> dsDNA BR Assay Kit according to the manufacturer's instructions.</li><li>Add 10x DSN master buffer and DSN solution (final conc. 0.05 U for 1 Î¼g of purified DNA fragments) to purified DNA fragments, and adjust the total reaction volume to 200 Î¼l.<br/><em>Note: The efficiency of the dsDNA cleaving is very high. We have demonstrated that 0.05 U is enough to cleave the 2 Î¼g of dsDNA completely (Figure 2). In addition, since quantification of ssDNA with the fluorescent dye of the Qubit ssDNA Assay Kit (Thermo Fisher Scientific, Invitrogen) might not be absolutely accurate, in this protocol, treated DNA samples are qualitatively assessed.</em><em><br/></em><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731210333_9458.jpg"/><br/><strong>Figure 2. DSN reaction efficiency.</strong> dsDNA cleaving reaction from 2 Î¼g of DNA pool including 70-mer ssDNA was either mock-treated or treated with 0.05, 0.1, or 0.3 U of DSN and analyzed by TapeStation with the D1000 ScreenTape.<br/><br/></li><li>Incubate the reactions at 65 Â°C for 20 min to specifically cleave the dsDNA on the heat block.</li><li>To inactivate the DSN reaction, add 1.5 Î¼l of 0.5 M EDTA (final concentration: 5 mM) and incubate at room temperature for 5 min.</li><li>Add 604.5 Î¼l AMPure XP (3.0x) and mix by pipetting.<br/><em>Note: Because the enriched ssDNA size is expected to be about 100 bp, a little bit more AMPure is used to recover the whole amount as much as possible.</em><em></em><br/></li><li>After incubation at room temperature for 5 min, place the reaction tube into a magnetic stand for 2 min to separate the beads from the solution.</li><li>Aspirate the supernatant from the reaction tube and discard.</li><li>Dispense 200 Î¼l of 80% EtOH and incubate at room temperature for at least 30 s.</li><li>Aspirate out the ethanol and discard.</li><li>Repeat Steps D7-D8 again.</li><li>Take out the tube from the magnetic stand and incubate at room temperature within 5 min to remove EtOH completely.</li><li>Add 17 Î¼l of 10 mM Tris-HCl (pH 8.0) and mix by pipetting.</li><li>Incubate at room temperature for 2 min.</li><li>Place the reaction tube into a magnetic stand for 1 min to separate beads from solution.</li><li>Transfer purified product to a new 0.2 ml tube.</li><li>Examples of purified ssDNA before and after DSN treatment are shown in Figure 3. One microliter of sample is used in this demonstration.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731210413_0033.jpg"/><br/><strong>Figure 3. Distribution of ssDNA after the ssDNA enrichment by DSN treatment.</strong> Sixty micrograms of genomic DNA was used in a volume of 300 Î¼l for 20 min with 1.5 U of DSN, and the reaction was inactivated by 5 mM EDTA. The enriched ssDNA was observed by the TapeStation with the D1000 ScreenTape.<br/><br/></li></ol></li><li>Dephosphorylation of ssDNA<ol class="npwtwo"><li>Add 3 Î¼l of 10x FastAP buffer and 2 Î¼l of FastAP to the purified ssDNA and adjust the total reaction volume to 30 Î¼l.</li><li>Transfer the tube to the thermal cycler and incubate at 37 Â°C for 10 min.</li><li>Inactivate and denature the dephosphorylated ssDNA solution by incubating the reaction at 95 Â°C for 2 min, quickly transfer the tube onto ice for 2-3 min, and then subsequently transfer it to room temperature (products: 50 Î¼l of [deP-ssDNA]).<br/><em>Note: [deP-ssDNA] can be stored at -20 Â°C overnight.<br/><br/></em><em></em> </li></ol></li><li>Phosphorylation of 1stAdapter_BIOTEG<ol class="npwtwo"><li>Combine the following reagents in Table 1:<br/><br/><strong>Table 1. Reagents used for phosphorylation of adapter</strong><strong></strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731220228_5803.jpg" width="350"/><br/><br/></li><li>Incubate the reactions at 37 Â°C for 1 h, inactivate at 95 Â°C for 5 min on the thermal cycler, and transfer to room temperature (products: 5 Î¼M of [P-Adapter_N8B]).<br/><em>Note: [P-adapter_N8B] can be stored at -20 Â°C overnight.<br/></em><em></em><br/></li></ol></li><li><span style="font-size:13.3333px;">L</span>igation<ol class="npwtwo"><li>Combine the following reagents in Table 2.<br/><br/><strong>Table 2. Reagents used for ligation of ssDNA and adapter</strong><strong></strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731220310_8743.jpg" width="400"/><br/><br/></li><li>Incubate the reactions at 16 Â°C overnight (products: [ligation product])<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Immobilization of the biotinylated ssDNA fragment<ol class="npwtwo"><li>Resuspend the Dynabeads MyOne Streptavidin C1 beads in the vial by vortexing.</li><li>Transfer the 25 Î¼l of beads (10 Î¼g/Î¼l) to a 0.2 ml tube.</li><li>Add an equal volume of 1x B&amp;W Buffer and resuspend.</li><li>Place the tube on a magnetic stand for 1 min and discard the supernatant.</li><li>Remove the tube from the magnetic stand and resuspend the washed beads in 25 Î¼l of 1x B&amp;W Buffer.</li><li>Repeat Steps H4-H5 twice, for a total of 3 washes and resuspend the washed beads in 50 Î¼l of 2x B&amp;W Buffer (5 Î¼g/Î¼l).</li><li>Add an equal volume of [ligation product]. </li><li>Incubate for 15 min at room temperature using gentle rotation.</li><li>Separate the [ligation product] coated beads on a magnetic stand for 2-3 min.</li><li>Wash the coated beads 2 times with 1x B&amp;W Buffer and discard 1x B&amp;W Buffer (product: [ssDNA-Adapter-C1beads]).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Primer annealing and extension<ol class="npwtwo"><li>Combine the following reagents in Table 3:<br/><br/><strong>Table 3. Reagents used for primer annealing</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731220535_4545.jpg" width="400"/><br/><br/></li><li>Incubate the reactions at 65 Â°C for 2 min and place on ice for 1 min.</li><li>Transfer the tube to a precooled thermal cycler at 15 Â°C (leave the lid open) and add 1 Î¼l of Gflex.</li><li>Increase the temperature by 0.1 Â°C/s, from 15 Â°C to 68 Â°C, and maintain the reaction at 68 Â°C for 5 min.</li><li>Wash the beads with 1x B&amp;W buffer (product: [dsDNA-beads]).<br/><br/></li></ol></li><li>I-SceI restriction digest to release the product from the beads<ol class="npwtwo"><li>Add 1 Î¼l of I-SceI and 5 Î¼l of 10x CutSmart buffer to [dsDNA-beads], and bring the volume up to 50 Î¼l.</li><li>Incubate at 37 Â°C for 1 h, and transfer the supernatant to a new tube.</li><li>Add 150 Î¼l AMPure XP (3.0x) and mix by pipetting.<br/><em>Note: The fragment size is expected to be about 100 bp, a little bit more AMPure is used to recover the whole amount as much as possible.</em><em></em><br/></li><li>After incubation at room temperature for 5 min, place the reaction tube into a magnetic stand for 2 min to separate the beads from the solution.</li><li>Aspirate the supernatant from the reaction tube and discard.</li><li>Dispense 200 Î¼l of 80% EtOH and incubate at room temperature for at least 30 s.</li><li>Aspirate out the ethanol and discard.</li><li>Repeat Steps J6-J7 again.</li><li>Take out the tube from the magnetic stand and incubate at room temperature within 5 min to remove EtOH completely.</li><li>Add 10 Î¼l of 10 mM Tris-HCl (pH 8.0) and mix by pipetting.</li><li>Incubate at room temperature for 2 min.</li><li>Place the reaction tube into a magnetic stand for 1 min to separate the beads from solution.<br/></li><li>Transfer purified product to a new 0.2 ml tube.<br/><br/></li></ol></li><li>Library preparation and sequencing<ol class="npwtwo"><li>The library is prepared using the ThruPLEX DNA-seq Kit according to the manufacturer's instructions.<br/><em>Note: Because a low amount of adapter-ligated DNA fragment is expected, the low input library preparation kit (ThruPLEX DNA-seq Kit) is used. Of course, other kits may be used (such as Low input Library Prep Kit, Clontech Laboratories, Inc.). Using this protocol we successfully prepared the library with 14 cycle amplification.</em><em></em><br/></li><li>Quality check of the library is performed by the TapeStation with D1000 ScreenTape (Figure 4).</li><li>Sequencing is performed on the NextSeq 500 instrument using a NextSeq 500 High Output Kit (75 bp single-end read).<br/><em>Note: The read length should be 70 bp or more in consideration of the adapter size. The optimal amount of sequence read is 1 million for E. coli genome. The library is pooled according to the amount of read required to analysis. In the case of NextSeq 500 High Output Kit (75 bp single-end read), you can pool approximately 1% of the total.</em><em><br/></em><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731210457_5830.jpg"/><br/><strong>Figure 4. Final library of ThruPLEX with 14 cycle amplification was analyzed by the TapeStation with the D1000 ScreenTape assay.</strong> The electropherogram shows a peak with a maximum at 212 bp.</li></ol></li></ol><p class="pptt" id="biaoti28729">### Data analysis</p><ol class="npwtwo"><li>Perform a demultiplex of sequenced raw data to fastq format using bcl2fastq.</li><li>To assess the sequence read quality, run a FastQC (v. 0.10.1).</li><li>Align sequence reads to a reference genome using BWA-MEM (v. 0.7.1-r1034; Li and Durbin, 2009), and create SAM format file without unmapped read using samtools (v. 1.2).</li><li>The 4S-seq reads contain a molecular tag (5â-GGGAANNNNNNNNTAGGG-3â) in addition to the adapter, and this tag sequence can be used to determine whether the read originated from the Watson or Crick strand.<br/><em>Note: The ssDNA skew, represented as (Watson - Crick)/(Watson + Crick), can be calculated by a custom Perl script (available at 4Spgm.pl, <a href="https://github.com/nkono/4S-seq" target="_blank">https://github.com/nkono/4S-seq</a>). Figure 5 represents the example commands of the analysis pipeline and output.</em><em><br/></em><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190731/20190731210540_1799.jpg"/><br/><strong>Figure 5. The example of 4Spgm.</strong> You can obtain the ssDNA skew data just by using a reference genome file and a SAM file.</li></ol><p class="pptt" id="biaoti28730">### Recipes</p><ol class="npwtwok"><li>LB<br/>2 g of Bacto<sup>TM</sup> Tryptone (Thermo Fisher SCIENTIFIC)<br/>1 g of Bacto<sup>TM</sup> Yeast Extract (Thermo Fisher SCIENTIFIC)<br/>1 g of NaCl (FUJIFILM Wako Pure Chemical Corporation)<br/>in 200 ml of milli-Q water<br/>After sterilizing with autoclave and adding appropriate antibiotics, store at room temperature<br/></li><li>TE buffer<br/>1 ml of 1 M Tris-HCl (pH 8.0, 10 mM final concentration) (NIPPON GENE CO., LTD)<br/>200 Î¼l of 0.5 M EDTA (1 mM final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>in 100 ml of Milli-Q water<br/>Store at room temperature</li><li>Lysis buffer<br/>1 ml of 1 M Tris-HCl (pH 8.0, 10 mM final concentration) (NIPPON GENE CO., LTD)<br/>2 ml of 0.5 M EDTA (10 mM final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>2 g of Sucrose (20% [w/v] final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>292.2 mg of NaCl (50 mM final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>20 mg of Lysozyme (200 Î¼g/ml final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>10 Î¼l of 100 mg/ml RNase A (10 Î¼g/ml final concentration) (QIAGEN)<br/>in 100 ml of Milli-Q water <br/>Freshly prepare before use</li><li>80% (v/v) EtOH<br/>8 ml of EtOH (FUJIFILM Wako Pure Chemical Corporation) in 10 ml of Milli-Q water<br/>Store at room temperature</li><li>2x B&amp;W buffer<br/>1 ml of 1 M Tris-HCl (pH 7.5, 10 mM final concentration) (NIPPON GENE CO., LTD)<br/>200 Î¼l of 0.5 M EDTA (1 mM final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>11.688 g of NaCl (2 M final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>in 100 ml of Milli-Q water<br/>Store at room temperature</li><li>1x B&amp;W buffer<br/>500 Î¼l of 1 M Tris-HCl (pH 7.5, 5 mM final concentration) (NIPPON GENE CO., LTD)<br/>100 Î¼l of 0.5 M EDTA (500 Î¼M final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>5.844 g of NaCl (1 M final concentration) (FUJIFILM Wako Pure Chemical Corporation)<br/>in 100 Î¼l of milli-Q water<br/>Store at room temperature</li></ol><p class="pptt" id="biaoti28731">### Acknowledgments</p><p class="ppzz">This protocol was adapted from our established publication (Kono, <em>et al.</em>, 2018). We thank Nozomi Abe and Yuki Takai for technical assistance, and James Fleming for the English language proof-reading. This work was supported by research funds from the Yamagata Prefectural Government and Tsuruoka City, Japan.</p><p class="pptt" id="biaoti28732">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti28733">### References</p><ol class="npwtwok"><li>Hyrien, O. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25601401" target="_blank">Peaks cloaked in the mist: the landscape of mammalian replication origins.</a> <em>J Cell Biol</em> 208(2): 147-160.</li><li>Karnani, N., Taylor, C. M., Malhotra, A. and Dutta, A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19955211" target="_blank">Genomic study of replication initiation in human chromosomes reveals the influence of transcription regulation and chromatin structure on origin selection.</a> <em>Mol Biol Cell</em> 21(3): 393-404.</li><li>Kono, N., Tomita, M. and Arakawa, K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30371772" target="_blank">Accelerated Laboratory Evolution Reveals the Influence of Replication on the GC Skew in <em>Escherichia coli</em>.</a> <em>Genome Biol Evol</em> 10(11): 3110-3117.</li><li>Li, H. and Durbin, R. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" target="_blank">Fast and accurate short read alignment with Burrows-Wheeler transform.</a> <em>Bioinformatics</em> 25(14): 1754-1760.</li><li>Mesner, L. D., Valsakumar, V., Karnani, N., Dutta, A., Hamlin, J. L. and Bekiranov, S. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21173031" target="_blank">Bubble-chip analysis of human origin distributions demonstrates on a genomic scale significant clustering into zones and significant association with transcription.</a> <em>Genome Res</em> 21(3): 377-389.</li><li>Smith, D. J. and Whitehouse, I. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22419157" target="_blank">Intrinsic coupling of lagging-strand synthesis to chromatin assembly.</a> <em>Nature</em> 483(7390): 434-438.</li><li>Yu, C., Gan, H., Han, J., Zhou, Z. X., Jia, S., Chabes, A., Farrugia, G., Ordog, T. and Zhang, Z. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25449133" target="_blank">Strand-specific analysis shows protein binding at replication forks and PCNA unloading from lagging strands when forks stall.</a> <em>Mol Cell</em> 56(4): 551-563.<br/></li></ol>