<body>
            <h1>Three-dimensional Reconstruction and Quantification of Proteins and mRNAs at the Single-cell Level in Cultured Cells</h1>
            <p><strong>Authors:</strong> Xueer Jiang, Ingrid Brust-Mascher, Li-En Jao</p>
            <h2>Abstract</h2>
            <p>Gene expression is often regulated by the abundance, localization, and translation of mRNAs in both space and time. Being able to visualize mRNAs and protein products in single cells is critical to understand this regulatory process. The development of single-molecule RNA fluorescence <em>in situ</em> hybridization (smFISH) allows the detection of individual RNA molecules at the single-molecule and single-cell levels. When combined with immunofluorescence (IF), both mRNAs and proteins in individual cells can be analyzed simultaneously. However, a precise and streamlined quantification method for the smFISH and IF combined dataset is scarce, as existing workflows mostly focus on quantifying the smFISH data alone. Here we detail a method for performing sequential IF and smFISH in cultured cells (as described in Sepulveda <em>et al.</em>, 2018) and the subsequent statistical analysis of the smFISH and IF data via three-dimensional (3D) reconstruction in a semi-automatic image processing workflow. Although our method is based on analyzing centrosomally enriched mRNAs and proteins, the workflow can be readily adapted for performing and analyzing smFISH and IF data in other biological contexts.</p>
            <p><strong>Keywords:</strong> Single-molecule imaging, Centrosome, Image processing, Quantification, mRNA distribution, 3D reconstruction, Imaris, MATLAB</p>
            <h2>Background</h2><p>smFISH is a technique to visualize individual RNA molecules using multiple short fluorescently-labeled DNA oligonucleotides (âprobesâ) complementary to the target RNA (Femino <em>et al.</em>, 1998; Raj <em>et al.</em>, 2008). In this technique, when an ensemble of short fluorescent DNA probes is bound at the target RNA, robust signals are produced as opposed to the weak signals produced by a single probe. This feature enhances the signal-to-noise ratio to reveal the location of the target RNAs, even if a single probe may have off-target binding. smFISH provides information about the RNA abundance and subcellular localization of a given RNA at the single-molecule and single-cell levels. Furthermore, combined with IF to visualize proteins, both RNAs and proteins of interest can be analyzed simultaneously in the same cell. However, tools for analyzing both smFISH and IF data are not broadly available. A widely used tool for analyzing smFISH data is FISH-quant (Mueller <em>et al.</em>, 2013; Tsanov <em>et al.</em>, 2016). It is a freely available software package that greatly streamlines the image analysis and mRNA spot identification. However, most quantification tools (<em>e.g.</em>, Mueller <em>et al.</em>, 2013; Lee <em>et al.</em>, 2016; Tsanov <em>et al.</em>, 2016), including FISH-quant, focus on the quantification of the smFISH data and do not intergrade the analysis of the IF data in the pipeline. Here, we detail a streamlined workflow for performing, acquiring, and analyzing sequential IF and smFISH data via 3D reconstruction in Imaris software, followed by quantifications using MATLAB and R scripts. We use co-translational targeting of pericentrin (PCNT) polysomes to the centrosome during mitosis in adherent cultured cells as an example (Sepulveda <em>et al.</em>, 2018) to demonstrate how to use Imaris software to reconstruct the <em>PCNT</em> mRNAs and proteins in 3D confocal z-stacks and to apply MATLAB and R scripts to quantify their intensities, volumes, and relations (<em>e.g.</em>, spatial distribution of molecules and overlapping between signals) in a semi-automatic manner. Our protocol can be readily applied to other sequential IF and smFISH experiments to precisely quantify RNAs and proteins in 3D space.</p>
        </body>

<p class="pptt" id="biaoti28768">## Materials and Reagents</p><ol class="npwtwok"><li>1-1/2 micro circular glass coverslipâ12 mm Diameter (Electron Microscopy Sciences, Hatfield, PA, catalog number: 72230-01)</li><li>Cellstar<sup>Â®</sup> 24-well cell culture multiwell plate (Greiner Bio-one North America, Inc., Monroe, NC, catalog number: 662160)</li><li>15-cm Petri dish (Corning, New York, catalog number: 351058)</li><li>Parafilm (Parafilm M, Bemis, Fisher Scientific, Waltham, MA, catalog number: 13-374-12)</li><li>Paper towel (Fisher Scientific, Waltham, MA, catalog number: 19-040-898)</li><li>HeLa cells (ATCC, CCL-2) (a gift from Susan Wente, Vanderbilt University, Nashville, TN; RRID: CVCL_0030)</li><li>UltraPure<sup>TM</sup> distilled water (DNase and RNase-Free) (Invitrogen, Carlsbad, CA, catalog number: 10977-015)</li><li>Formaldehyde, Para (PFA) (Fisher Chemical, Waltham, MA, catalog number: O4042-500)</li><li>Phosphate Buffered Saline (PBS), pH 7.4 (Fisher BioReagents, Pittsburgh, PA, catalog number: BP399-1)</li><li>Triton<sup>TM</sup> X-100 (Fisher BioReagents, Pittsburgh, PA, catalog number: BP151-500)</li><li>Rabbit anti-PCNT antibody (Abcam, Cambridge, MA, catalog number: ab4448)</li><li>Goat anti-Rabbit IgG (H+L) cross-adsorbed secondary antibody, Alexa Fluor 488 (Invitrogen, Carlsbad, CA, catalog number: A-11008</li><li>37% formaldehyde solution (Fisher BioReagents, Pittsburgh, PA, catalog number: BP531-500)</li><li>Formamide (Sigma-Aldrich, Burlington, MA, catalog number: F9037)</li><li>Prolong Gold antifade mountant (Life Technologies, Carlsbad, CA, catalog number: P36930)</li><li>Stellaris<sup>Â®</sup> RNA FISH Hybridization Buffer (Biosearch Technologies, Petaluma, CA, catalog number: SMF-HB1-10)</li><li>Stellaris<sup>Â®</sup> RNA FISH Wash Buffer A (Biosearch Technologies, Petaluma, CA, catalog number: SMF-WA1-60)<br/></li><li>Stellaris<sup>Â®</sup> RNA FISH Wash Buffer B (Biosearch Technologies, Petaluma, CA, catalog number: SMF-WB1-20)</li><li>Stellaris<sup>Â®</sup> RNA FISH probes labeled with Quasar 670 (Biosearch Technologies, Petaluma, CA)</li><li>4â²,6-Diamidino-2-phenylindole dihydrochloride (DAPI) (Sigma-Aldrich, Burlington, MA, catalog number: D9542)<br/></li></ol><p class="pptt" id="biaoti28769">### Equipment</p><ol class="npwtwok"><li>Leica DMi8 with 63x/1.40 or 100x/1.40 HC PL APO objectives equipped with a spinning disk confocal microscope system (Dragonfly, Andor Technology, Belfast, UK)</li><li>iXon Ultra 888 EMCCD camera (Andor Technology, Belfast, UK) <br/></li></ol><p class="pptt" id="biaoti28770">### Software</p><ol class="npwtwok"><li>Fusion Software (Andor Technology, Belfast, UK)<br/></li><li>Imaris 8.3.1 (Bitplane, Belfast, UK; Imaris, RRID: SCR_007370)</li><li>MATLAB (MathWorks, Natick, MA; RRID:SCR_001622)</li><li>Microsoft Excel (Microsoft, Redmond, WA; RRID:SCR_016137)</li><li>R (R Project for Statistical Computing; RRID: SCR_001905)</li><li>ImageJ (Image Processing and Analysis in Java; <a href="RRID: SCR_003070" target="_blank">RRID: SCR_003070</a>)<br/></li></ol><p class="pptt" id="biaoti28771">### Procedure</p><ol class="npwone"><li>Sequential immunofluorescence (IF) and single-molecule RNA fluorescence <em>in situ</em> hybridization (smFISH)<br/><em>Note: We follow the sequential IF and smFISH protocol described by the manufacturer of smFISH probes (Biosearch Technologies, Petaluma, CA) with some modifications detailed below.<br/><br/><strong>IF<br/></strong></em><em>Note: Use nuclease-free water for preparing all reagents for the sequential IF and smFISH experiment to limit potential RNA degradation.</em> <br/><strong> <ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Place a 12-mm circular coverslip to each well of a 24-well plate. Seed about 5 x 10<sup>4</sup> cells to each well. Allow cells to attach onto the coverslips.</li><li>When the cells reach 75-90% confluency in each well (12-18 h for most human cultured cells), fix cells with 4% paraformaldehyde in phosphate buffered saline (1x PBS) for 10 min at room temperature (RT), followed by two quick washes with 1x PBS.<br/><em>Note: To perform quick washes, gently transfer 1 ml of PBS to each well and then aspirate the solution immediately.</em> </li><li>Permeabilize cells with 0.1% Triton X-100 in 1x PBS for 5 min at RT, followed by one quick wash with 1x PBS.</li><li>Incubate cells (on a 12-mm circular coverslip) with 70 Âµl of primary antibody solution for 3 h at RT (<em>e.g.</em>, 1:1,000 dilution of rabbit anti-PCNT antibody in 1x PBS in our example).</li><li>Perform three 5-min washes with 1x PBS and incubate the cells with 70 Âµl of secondary antibody solution overnight in the dark at 4 Â°C (<em>e.g.</em>, 1:500 diluted anti-rabbit Alexa Fluor 488). <br/><em>Note: The dilution factor and incubation time of the primary and secondary antibodies need to be empirically determined; the sample condition described is for detecting PCNT proteins. To perform 5-min washes, gently transfer the coverslip to a well in a 24-well plate containing 1 ml of 1x PBS. Incubate for 5 min at RT without shaking the plate.</em> </li><li>Perform three 5-min washes with 1x PBS and post-fix cells with 3.7% formaldehyde in 1x PBS for 10 min at RT.</li></ol></strong><strong><em><br/>smFISH</em></strong><strong><em></em><br/><ol class="npwtwo"><li>Wash cells with Wash Buffer A for 5 min and incubate with 67 Âµl of Hybridization Buffer containing 125 nM smFISH DNA probe mix for 6 h at 37 Â°C in the dark.<br/><em>Note: We use the online Stellaris Probe Designer from Biosearch Technologies to design the FISH probes. We generally use a set of 48 to 96 20-mer DNA probes for each RNA target. For best results, add 10% formamide into 1x Wash Buffer A and Hybridization Buffer immediately before use.</em><em></em><br/></li><li>Incubate cells with Wash Buffer A for 30 min at 37 Â°C, Wash Buffer A containing 0.05 Âµg/ml DAPI for 30 min at 37 Â°C, and Wash Buffer B for 3 min at RT.</li><li>Mount the coverslip with enough Prolong Gold antifade mountant (<em>e.g</em><em>.</em>, 5 Âµl for a 12-mm circular coverslip). Let the mountant cure overnight in the dark at RT and seal the edge of the coverslip with nail polish before imaging. <br/><p class="ppzz" style="margin-left:-20px;"><em>Note: For the best results, perform the incubations of antibody and smFISH probes in a humidified container (<em>e.g.</em>, a 15-cm Petri dish with a piece of parafilm and wet paper towel inside, wrapped with aluminum foil to block light).</em></p><br/></li></ol></strong> </li><li>Image acquisition and deconvolution<br/><strong><em>Imaging </em></strong><br/><em>Note: When using multiple fluorophores, it is important to carefully select the wavelength ranges, emission filters, and dichroic mirrors to avoid signal bleed-through. Use settings to minimize photobleaching but also maximize the signal-to-noise ratio. A camera with high sensitivity is preferred to obtain good smFISH signals (e.g., an electron multiplying CCD camera). We recommend acquiring images sequentially, channel by channel, starting from the longest wavelength. Below are the general IF and smFISH acquisition settings using the Dragonfly spinning confocal system in the Jao lab.</em><br/><ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Objective: 100x NA 1.4 (as well as placing a 1.5x or 2x magnification lens before the camera)</li><li>Laser power: between 5-30% of 50-100 mW lasers</li><li>Exposure time: 100-250 ms</li><li>Gain: 200 for iXon Ultra 888 EMCCD camera</li><li>Pinhole: 40 Âµm</li><li>Z-stack interval: 0.3 Âµm</li><li>For assessing colocalization between different fluorescent signals, acquire all channels for each z-stack.<br/><br/></li></ol><em><strong>Deconvolution </strong></em><br/><em>Note: It is important to avoid potential artifacts introduced during the deconvolution process by comparing the pre- and post-deconvolved images. In general, the higher the number of iterations, the more likely image artifacts will be produced. Below are our typical deconvolution settings using the built-in deconvolution algorithm in the Fusion software of the Dragonfly system.</em><br/><ol class="npwtwo"><li>Deconvolution preview mode: ON</li><li>Algorithm: Robust (Iterative)</li><li>Pre-sharpening: OFF</li><li>Edge artifact reduction: none</li><li>Denoising filter size: 0.7</li><li>Denoising frequency: 4</li><li>Number of iterations: 15</li><li>Minimum intensity removal: ON</li><li>Initial denoising: ON</li><li>Normalization: OFF</li><li>Iteration acceleration: OFF</li></ol></li></ol><p class="pptt" id="biaoti28772">### Data analysis</p><ol class="npwone"><li>3D reconstruction and image rendering in Imaris<br/><ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Prior to image processing, in âFile/Preference/Statisticsâ, turn on relevant statistical parameters. Below are the parameters we routinely use.<br/><ol class="npwthree"><li>Surfaces: intensity max, intensity mean, intensity sum; position x, position y, position z; total number of surfaces; volume.</li><li>Spots: intensity max, intensity mean, intensity sum; position x, position y, position z; total number of spots; volume.</li><li>Volume<br/><p class="ppzz" style="margin-left:-20px;"><em>Note: The term âStatisticsâ in Imaris refers to different quantitative descriptions associated with each object (e.g., area, volume, intensity, etc.). It does not refer to a statistical analysis of the data.</em></p></li></ol></li><li>Extract and mask a 3D cell outline with the following steps in Imaris:<br/><ol class="npwthree"><li>Open the deconvolved image.</li><li>Click âEdit/Add Channelsâ: Select the corresponding original image.</li><li>In the menu, select âSurpass/Add new surfacesâ (Figure 1A); select âSkip automatic creation, edit manuallyâ (Figure 1B).</li><li>Move to the slice position with the clearest outline.</li><li>Switch pointer to select mode and select âDrawâ option to mark the cell outline (Figures 1C and 1D).<br/><em>Note: The cell outline in our analyses was marked based on the visualization of the background noise by adjusting the channel max intensity under âdisplay adjustmentâ. Users may use other membrane or cytoplasmic markers to visualize the cell boundary.</em><br/></li><li>Copy and paste the contour lines to the first and last image planes (Figure 1D).</li><li>Select manual âResolutionâ and adjust it to the largest size (Figure 1D). Click âCreate Surfaceâ.</li><li>Make cell mask (Figures 1E and 1F): Select âMask Allâ. The box âDuplicate channel before applying maskâ should be selected. Set âvoxels outside surfaceâ to â0â and âinside surfaceâ to a high value within the bit depth of your image (<em>e.g.</em>, â30,000â in this example). This creates a binary mask of the cell of interest.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815194604_0140.jpg"/><br/><strong>Figure 1. Cell outlining in Imaris.</strong> A. Add new surfaces. B. Skip automatic creation and edit manually. C and D. Draw contours lines to mark the cell outline. E and F. Mask channel of the cell outline and set voxels.<br/></li></ol><br/></li><li>Create 3D Surface object(s) for protein(s) of interest in Imaris:<br/><ol class="npwthree"><li>Add a new Surface object (Figure 1A). Click the blue arrow to continue to the next page (Figure 2A). For âSource Channelâ, select the deconvolved channel of protein of interest. Check the box âSmoothâ, and the value of âSurfaces Area Detail Levelâ is automatically calculated and kept as the default. Select âAbsolute Intensityâ for thresholding (Figure 2B).<br/><em>Note: The âSmoothâ command applies a Gaussian filter to the image to facilitate the identification of surface objects for noisy images. It reduces the noise and detail included in the rendered surface. If âSmoothâ is selected, the âSurface Area Detail Levelâ parameter is automatically determined from the image. Users can alter this parameter to remove more or less surface details. The higher this parameter, the less detail this surface rendering will include.</em><br/></li><li>âThreshold (Absolute Intensity)â has an automatic value by Imaris (Figure 2C). Increase or lower this threshold to include only the signal of interest. Use the same value across images to be compared.</li><li>There is an automatic filter based on voxel size. This can be left at the default of 10 or adjusted depending on the size of interest.</li><li>Add Filters: Select âIntensity Max of the masked channel number of the cell outlineâ generated in step 2. This step restricts the fitting of any protein signals within the outlined cell (Figure 2D).</li><li>Click âFinishâ to complete the rendering of protein surface (Figure 2D). Examples are in Figure 4.<br/><p class="ppzz" style="margin-left:-20px;"><em>Note: If there are multiple proteins of interest per cell, repeat step 3 to generate a surface rendering for each protein. </em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815200320_2888.jpg"/><br/><strong>Figure 2. 3D reconstruction of protein surfaces.</strong> A and B. Select the source channel and set surface area detail level. C. Adjust absolute intensity threshold to capture all signals. D. Add filters to restrict fitting of protein signals within the outlined cell.</p></li></ol><br/></li><li>Create 3D spot object(s) for RNA of interest in Imaris:<br/><ol class="npwthree"><li>Add a new spot object (Figure 3A).</li><li>Select âDifferent Spot Size (Region Growing)â (Figure 3B).</li><li>For âSource Channelâ, select the deconvolved channel of RNA of interest (Figure 3C).</li><li>For âSpot Detectionâ, enter âEstimated XY Diameter and Z Diameterâ of RNA of interest (Figure 3C). <br/><em>Note: Determine the XY diameter by measuring the average size of several RNA smFISH foci in the original confocal image via ImageJ with reference to a scale bar. The Z diameter is set two times of the XY diameter by Imaris as the default.</em><br/></li><li>Imaris calculates a âspot qualityâ (Figure 3D) based on intensity differences and shapes for spot rendering. This value should be adjusted to include the signal of interest.<br/><em>Note: We found that the spot quality by the default calculation is often inaccurate in marking the true RNA spots. Users need to manually increase or lower the threshold to ensure that the true RNA spots are marked, but the noises (e.g., small dot-like background signals) are excluded.</em><br/></li><li>Add Filter: Select âIntensity Max = masked channel number of the cell outlineâ generated in step 2. This step restricts the fitting of any RNA signals within the outlined cell (Figure 3D).</li><li>Select âSpot Region from Absolute Intensityâ (Figure 3E).</li><li>Spot Region Threshold is pre-calculated by Imaris (Figure 3F).<br/><em>Note: Users may need to manually increase or decrease the region threshold to ensure that all the RNA signals are rendered in the final step.</em><br/></li><li>Click âFinishâ to complete the rendering of RNA spots (Figure 3F). Examples are in Figure 4.<br/><br/><img alt="" class="layerphoto" height="110" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815194655_9623.jpg" style="color:#666666;font-family:Arial, Helvetica, sans-serif;text-align:justify;" width="100"/><br/><strong>Figure 3. 3D reconstruction of RNA spots.</strong> A and B. Create a new spot and select different spot sizes (region growing). C and D. Select source channel and set spot detection. (D) Adjust thresholds and add filters. E and F. Select absolute intensity and adjust region threshold.<br/><br/><img alt="" class="layerphoto" height="109" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815193430_0443.jpg" style="color:#666666;font-family:Arial, Helvetica, sans-serif;text-align:justify;" width="100"/><br/><strong>Figure 4. Examples of before (Confocal) and after (3D rendered) 3D reconstruction of the confocal images from a sequential IF and smFISH experiment</strong><strong></strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>This is an additional process to determine if the 3D reconstructed protein surfaces and RNA spots are overlapped.<br/><ol class="npwthree"><li>Click the protein surface object generated in step 3 and mask channel of the protein surface (Figures 5A and 5B): Select âMask Allâ. Choose the deconvolved channel of the protein surface as Source Channel, and check the box âDuplicate channel before applying maskâ. Set âvoxels outside surfaceâ to â0â and inside surface to the highest voxel values of your imaging system (<em>e.g.</em>, â30000â in this example). This will generate a masked channel of protein surface (<em>e.g.</em>, Channel number 8 in Figure 5F).</li><li>Click the RNA spot object generated in step 4. Add Filter (Figure 5C) and select âIntensity Max [channel number] = masked [channel number] of protein surfaceâ generated in step 5a; this is a binary channel with values either 0 or the high number selected earlier. Apply the upper threshold to select for overlapping RNA spots. Click âDuplicate Selection to new Spotsâ (Figure 5D).<br/><em>Note: If one or more pixels of an RNA spot overlaps with the masked protein surface, it is counted as an overlapping spot.</em><br/></li><li>Repeat the process (step 5b) but apply the low threshold (<em>i.e.</em>, spots with a value of 0 in the masked channel) to select for non-overlapping RNA spots. Click âDuplicate selection to new Spotsâ (Figure 5E).<br/><em>Note: If no pixel of an RNA spot overlaps with the masked protein surface, it is counted as a non-overlapping spot.</em><br/></li></ol></li><li>Repeat steps 1 to 5 for each image. Export the processed images to a folder.<br/><em>Note: During image processing, keep the order consistent for generation of cell outlines, protein surfaces, and RNA spots in Imaris (Figure 5F) so that the corresponding numbers of channels, protein surfaces, and RNA spots are identical for a given set of images. This thus allows a consistent definition of protein(s) or RNA(s) of interest when running the MATLAB scripts.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815193537_6294.jpg" style="font-style:italic;"/><br/><strong>Figure 5. Detection of the RNA spots that overlap with the protein signals.</strong> A and B. Mask protein surface of interest. C-E. Filtering overlapping and non-overlapping RNA spots. F. Illustration of channel numbers, protein surface numbers, and RNA spot numbers.<br/><br/></li></ol></li><li>Process and export data from Imaris using MATLAB scripts<br/><em>Note: To set up MATLAB, do the following first:</em><br/><ol class="npwtwo" style="color:#666666;font-style:italic;white-space:normal;"><li>Open MATLAB software; select âHOME/Set Pathâ: Add MATLAB search path as the location of the image folder</li><li>Prior to using ImarisReader for data analysis in MATLAB, download the ImarisReader .zip from the MATLAB Central File Exchange. Save the files within the MATLAB search path.</li></ol><br/><em>Processing note: Below are the examples of using MATLAB and R scripts to quantify the intensities (RNA or protein, total or centrosomal) and RNA distribution relative to the nearest centrosome after rendering of the RNA and centrosomal protein signals in Imaris. See Sepulveda et al., 2018 for examples of the final plots.</em><br/></li></ol><ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><ol class="npwtwo"><li>Quantification of total RNA and protein intensities<br/><ol class="npwthree"><li>Open â<a href="https://os.bio-protocol.org/attached/file/20190429/total protein and RNA intensity.zip" target="_blank">total_protein_and_RNA_intensity.m</a>â (Supplementary files) in MATLAB.</li><li>Define âfpathâ as the location of the images (<em>e.g.</em>, line 14 in Figure 6).</li><li>Define protein and RNA of interest for quantification. These are the âprotein_surface_numberâ in line 18 and âspot_numberâ in line 19 in Figure 6. <br/><em>Note: For example, in Figure 5F, âProtein surface number 2â is the Surface object created for the protein of interest. You will thus define âprotein_surface_number = 2â in line 18 in Figure 6. âRNA spot number 1â is the Spot object created for the RNA of interest. You will thus define âspot_number = 1â in line 19 in Figure 6.</em><br/></li><li>Define the original channel numbers for protein and RNA of interest. These are âori_protein_channel_numberâ in line 23 and âori_RNA_channel_numberâ in line 24 in Figure 6. <br/><em>Note: For example, in Figure 5F, the original channel of protein and RNA of interest are channel 5 and channel 4, respectively. You thus define âori_protein_channel_number = â5ââ in line 23 and âori_RNA_channel_number = â4ââ in line 24 in Figure 6.</em><br/></li><li>Run the script. The original protein and RNA intensities within the rendered protein surface and RNA spots will be exported into a .csv file in the image folder, which can be opened by Microsoft Excel.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815193721_5269.jpg"/><br/><strong>Figure 6. Quantification of total protein and mRNA intensities from the original (pre-deconvolved) confocal images.</strong> A highlight of the MATLAB script lines. See <a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank">Supplementary files for the complete script</a>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Quantification of protein intensity at the centrosomes<br/><em>Note: This is an example of quantifying protein intensity in a subcellular area of interest. Here we quantify the protein intensity from the two centrosomes during mitosis. The script isolates the two centrosomes from the rest of the cell by defining them as the first and second largest protein surface volumes of the anti-PCNT signals.</em><br/><ol class="npwthree"><li>Open â<a href="https://os.bio-protocol.org/attached/file/20190429/centrosomal protein intensity.zip" target="_blank">centrosomal_protein_intensity.m</a>â (Supplementary files) in MATLAB.</li><li>Define âfpathâ as the location of the images (<em>e.g.</em>, line 14 in Figure 7).</li><li>Define the surface number for protein of interest for quantification (<em>e.g.</em>, line 17 in Figure 7).</li><li>Define the original and deconvolved channel numbers for protein of interest (<em>e.g.</em>, in lines 21 and 22, respectively, in Figure 7).</li><li>Run the script to obtain a .csv file in the image folder. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815193919_6896.jpg" style="color:#666666;font-family:Arial, Helvetica, sans-serif;text-align:justify;"/><br/><strong>Figure 7. Quantification of the protein intensity at the centrosomes from the original (pre-deconvolved) confocal images.</strong> A highlight of the MATLAB script lines. See <a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank">Supplementary files for the complete script</a>.<br/><div><br/></div></li></ol></li><li>Quantification of mRNA distribution relative to the centrosome<br/><em>Note: This is an example of quantifying the distribution of mRNA molecules to the centrosome as a function of distance. The intensity of the mRNA signal in each spot is assumed to be proportional to the amount of mRNA in each spot and is used in lieu of mRNA units. The distance from each mRNA spot to each centrosomeâs center of mass is calculated, and the mRNA signal is assigned to the closest centrosome. The mRNA spots were binned in 0.5 Âµm intervals to the centrosome, and the spot intensities in each bin were added as a measure of the amount of mRNA at that distance. This is calculated for each cell and then averaged over all the cells of a given condition and graphed as average mRNA intensity as a function of distance to the closest centrosome within the cell in 3D voxels.</em><br/><ol class="npwthree"><li>Open â<a href="https://os.bio-protocol.org/attached/file/20190429/mRNA distribution.zip" target="_blank">mRNA_distribution.m</a>â (Supplementary files) in MATLAB. </li><li>Define âfpathâ as the location of the images (<em>e.g.</em>, line 14 in Figure 8).</li><li>Define the surface numbers for protein and mRNA for quantification (<em>e.g.</em>, line 18 for protein surface and line 19 for RNA spots in Figure 8).<br/></li><li>Define the deconvolved channel number of RNA (<em>e.g.</em>, line 22 in Figure 8).<br/><em>Note: We use RNA intensities from the deconvolved channel to eliminate the noise or background signals.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815194026_1173.jpg"/><br/><strong>Figure 8. Quantification of mRNA distribution relative to the centrosome.</strong> A highlight of the MATLAB script lines. See <a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank">Supplementary files for the complete script</a>.<br/><div><br/></div></li><li>Run the script to obtain .csv files in the image folder. In each .csv file, each row represents the distance relative to the centrosome, from 0 Âµm (first row) to 20 Âµm (last row) with 0.5 Âµm intervals (Figure 9). Each column represents the amount of mRNA at each distance relative to the nearest centrosome (Figure 9).<br/><br/><img alt="" class="layerphoto" height="279" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815194108_6281.jpg" width="100"/><br/><strong>Figure 9. An example of the data in the columns and rows of a .csv file generated by the âmRNA_distribution.mâ script<br/></strong><br/><em>Note: Below are the steps to use an R script to combine all .csv files from multiple cells into a single spreadsheet for further quantification (e.g., calculation of the normalized intensity as the ratio of each data point over the total intensity in each column).</em><br/></li><li>Group all .csv files into a new folder.</li><li>Open â<a href="https://os.bio-protocol.org/attached/file/20190429/column combination and normalization.zip" target="_blank">column_combination_and_normlization</a>â (Supplementary files) in R. </li><li>Specify the working directory and the location of the .csv files (<em>e.g.</em>, in lines 11 and 12, respectively, in Figure 10). </li><li>Run the R script to obtain two .csv files within the new folder: (1) combined original dataset; (2) combined normalized dataset.<br/><br/><img alt="" class="layerphoto" height="50" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815194200_0567.jpg" style="color:#666666;font-family:Arial, Helvetica, sans-serif;text-align:justify;"/><br/><strong>Figure 10. Using an R script to combine and normalize the data in multiple .csv files.</strong> A highlight of the R script lines. See <a href="https://os.bio-protocol.org/mgn/html/Default.aspx?wzid=3330" target="_blank">Supplementary files for the complete script</a>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Quantification of the RNA signals that overlap with the protein signals within a defined distance to the centrosome.<br/><em>Note: This is an example of quantifying how much of the PCNT mRNA overlaps with the anti-PCNT N-terminus protein signals within a defined distance from the centrosome; the overlap of these two signals outside of the centrosome suggests that these mRNAs are undergoing active translation. The distance from each overlapping or non-overlapping mRNA spot (separated in Imaris as explained above) to each centrosomeâs center of mass is calculated, and the mRNA signal is assigned to the closest centrosome. The mRNA spots were binned in 0.5 Âµm intervals from the centrosome, and the intensities of the overlapping or non-overlapping spots in each bin were added as a measure of the amount of mRNA at that distance. Percentage of overlapping mRNA spots with protein signals was calculated in each distance per cell. This was then averaged over all the cells of a given condition and graphed as the overlapping mRNA percentage as a function of distance to the closest centrosome within the cell in 3D voxels.<span id="__kindeditor_bookmark_end_193__" style="display:none;"></span></em><a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank"><br/></a> <ol class="npwthree"><a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank"></a> <li><a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank">Open â</a><a href="https://os.bio-protocol.org/attached/file/20190429/column combination and normalization.zip" target="_blank">overlapping_mRNA_percentage.m</a>â (Supplementary files) in MATLAB.</li><li>Define âfpathâ as the location of the images (<em>e.g.</em>, line 14 in Figure 11).</li><li>Define centrosome protein surface number, overlapping RNA spot number, and non-overlapping RNA spot number (<em>e</em><em>.g.</em>, in lines 18, 19, and 20, respectively, in Figure 11).<br/><em>Note: In Figure 5F, RNA spot number 2 and 3 are the Spot objects created for overlapping and non-overlapping RNA spots, respectively. You will thus define âoverlapping_RNA_spot_number = 2â in line 19 and ânonoverlapping_RNA_spot_number = 3â in line 20 in Figure 11.<em><br/></em></em> </li><li>Define the deconvolved channel number for RNA (<em>e.g.</em>, line 23 in Figure 11)</li><li>Run the script to obtain .csv files in the image folder. The first column represents the distance relative to the centrosome, from 0 Âµm (first row) to 20 Âµm (last row) with 0.5 Âµm intervals. The second column represents the percentage of RNA signals that overlaps with the protein surfaces within a given distance to the centrosome.<em><em><br/></em></em><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815194310_7807.jpg"/><br/><strong>Figure 11. Quantification of the RNA signals that overlap with the protein signals.</strong> A highlight of the MATLAB script lines. See <a href="https://os.bio-protocol.org/attached/file/20190429/Supplementary files complete script.zip" target="_blank">Supplementary files for the complete script</a>.<br/><div style="white-space:nowrap;"></div></li></ol></li></ol><p class="pptt" id="biaoti28773">### Notes</p><ol class="npwtwok"><li>Use the same parameters or thresholds for each image during Imaris processing if users aim to compare the absolute intensity or distribution of proteins or RNAs between different experimental conditions.</li><li>If MATLAB indicates errors about ImarisReader, run the script of âCellsReader.mâ to make sure that ImarisReader is functional before performing data analyses.</li></ol><p class="pptt" id="biaoti28774">### Acknowledgments</p><p class="ppzz">We thank the Jao lab members for discussions and support. This protocol was adapted from the previous work (Sepulveda <em>et al.</em>, 2018). Experiments and analyses were performed in part through the use of UC Davis Health Sciences District Advanced Imaging Facility. The work was supported by the New Faculty Startup Funds from University of California, Davis (to L.J).</p><p class="pptt" id="biaoti28775">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt" id="biaoti28776">### References</p><ol class="npwtwok"><li>Femino, A. M., Fay, F. S., Fogarty, K. and Singer, R. H. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9554849" target="_blank">Visualization of single RNA transcripts <em>in situ</em>.</a> <em>Science</em> 280(5363): 585-590.</li><li>Lee, C., Roberts, S. E. and Gladfelter, A. S. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26690072" target="_blank">Quantitative spatial analysis of transcripts in multinucleate cells using single-molecule FISH.</a> <em>Methods</em> 98: 124-133.</li><li>Mueller, F., Senecal, A., Tantale, K., Marie-Nelly, H., Ly, N., Collin, O., Basyuk, E., Bertrand, E., Darzacq, X. and Zimmer, C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23538861" target="_blank">FISH-quant: automatic counting of transcripts in 3D FISH images.</a> <em>Nat Methods</em> 10(4): 277-278.</li><li>Raj, A., van den Bogaard, P., Rifkin, S. A., van Oudenaarden, A. and Tyagi, S. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18806792" target="_blank">Imaging individual mRNA molecules using multiple singly labeled probes.</a> <em>Nat Methods</em> 5(10): 877-879.<br/></li><li>Sepulveda, G., Antkowiak, M., Brust-Mascher, I., Mahe, K., Ou, T., Castro, N. M., Christensen, L. N., Cheung, L., Jiang, X., Yoon, D., Huang, B. and Jao, L. E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29708497" target="_blank">Co-translational protein targeting facilitates centrosomal recruitment of PCNT during centrosome maturation in vertebrates.</a> <em>Elife</em> 7: e34959.</li><li>Tsanov, N., Samacoits, A., Chouaib, R., Traboulsi, A. M., Gostan, T., Weber, C., Zimmer, C., Zibara, K., Walter, T., Peter, M., Bertrand, E. and Mueller, F. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27599845" target="_blank">smiFISH and FISH-quant - a flexible single RNA detection approach with super-resolution capability.</a> <em>Nucleic Acids Res</em> 44(22): e165.<em><em><br/></em></em> </li></ol></ol>