<body>
            <h1>Construction of Viral Vectors for Cell Type-specific CRISPR Gene Editing in the Adult Mouse Brain</h1>
            <p><strong>Authors:</strong> Hiroshi Yamaguchi, Luis de Lecea</p>
            <h2>Abstract</h2>
            <p>Recently developed gene editing technologies based on engineered CRISPR/Cas9 systems enables researchers to disrupt genes in a cell type-specific manner in the adult mouse brain. Using these technologies, we recently showed that the dopamine beta-hydroxylase gene in Locus Coeruleus (LC) norepinephrine neurons plays a vital role in the maintenance of wakefulness. Our method consists of four steps, (1) crossing Cre-dependent spCas9 knockin mice with a Cre-driver mouse line to express spCas9 in the target neural populations, (2) cloning of sgRNA, (3) construction of an AAV (adeno associated virus) vector expressing dual sgRNA, and (4) virus packaging and stereotaxic injection of the virus into the target brain area. Here, we describe a detailed protocol of AAV vector construction for cell type-specific CRISPR gene editing in the adult mouse brain. The method adopts a dual-sgRNA strategy for efficient disruption of the target gene. At first, a few different sgRNAs targeting the same gene are cloned into a plasmid expressing spCas9. After evaluation of the sgRNAs by a T7 endonuclease assay, the two most efficient sgRNAs are cloned in tandem into an AAV vector using the Gibson Assembly method.</p>
            <p><strong>Keywords:</strong> CRISPR/Cas9, AAV, Adult mouse brain, Gibson Assembly, dual-sgRNA</p>
            <h2>Background</h2><p>Interrogation of gene functions in specific cell subtypes in the brain remains a challenge. The conventional techniques such as RNAi-based methods lack cellular specificity and efficiency. On the other hand, recently developed gene editing techniques using the CRISPR/Cas9 system allows researchers efficient gene disruption in a cell type-specific manner. A research group led by Feng Zhang showed the disruption of multiple genes in the adult mouse brain by AAV-mediated delivery of sgRNA (Swiech <em>et al.</em>, 2015). Also, the group established cre-dependent spCas9 knockin mice (Platt <em>et al.</em>, 2014). In a recent study, we used an AAV vector carrying dual sgRNA to improve the efficiency of bi-allelic gene targeting (Yamaguchi <em>et al.</em>, 2018). We here describe a protocol for the construction of dual sgRNA AAV vector (Figure 1).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815030952_8597.jpg" width="110"/><br/><strong>Figure 1. Schematic representation of all the procedure.</strong> Our method consists of four steps: (1) crossing Cre-dependent spCas9 knockin mice with a Cre-driver mouse line to express spCas9 in the target neural populations, (2) cloning of sgRNA into the spCas9 expressing plasmid, (3) construction of an AAV vector tandemly expressing dual sgRNA, and (4) AAV packaging and stereotaxic viral injection into the target brain area.</p>
        </body>

<p class="pptt" id="biaoti28812">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips</li><li>24-well plate (coated) (Thermo Fisher Scientific, catalog number: 142475)</li><li>1 ml plastic tube</li><li>NEB10-beta Competent <em>E. coli</em> (New England Biolabs, catalog number: C3019)</li><li>pX458 (Addgene, catalog number: 48138)</li><li>pAAV EF1Î± DIO mCherry (Addgene, catalog number: 20299)</li><li>BbsI-HF (New England Biolabs, catalog number: R3539)</li><li>MluI-HF (New England Biolabs, catalog number: R3198)</li><li>DpnI (New England Biolabs, catalog number: R0176)</li><li>Alkaline Phosphatase, Calf Intestinal (CIP) (New England Biolabs, catalog number: M0290)</li><li>Guide oligonucleotides (see Procedure A)</li><li>Quick ligation kit (New England Biolabs, catalog number: M2200)</li><li>Gibson Assembly Master Mix (New England Biolabs, catalog number: E2611)</li><li>PrimeSTAR HS DNA polymerase (Takara, catalog number: R010)</li><li>dNTP Mix 10 mM each (the mix of 10 mM of dTTP, dCTP, dGTP and dATP) (GenScript, catalog number: C01689)</li><li>QIAquick Gel Extraction kit (QIAGEN, catalog number: 28704)</li><li>QIAprep Spin Miniprep kit (QIAGEN, catalog number: 27104)</li><li>Tris-EDTA, (1x Solution) (Fisher BioReagents, catalog number: BP2473)</li><li>LB Broth EZMix Powder (Sigma, catalog number: L7658)</li><li>LB Agar (Fisher BioReagents, catalog number: BP1425)</li><li>Ampicillin (Gemini Bio-Products, catalog number: 400-130P)</li><li>Primer to verify the cloning of sgRNA into pX458: 5â-gactatcatatgcttaccgt-3â</li><li>Primer to verify the tandem cloning of sgRNAs into pAAV: 5â-actgacgggcaccggagcca-3â</li><li>Primers for Gibson Assembly:<br/>Primer S1: 5â-taggggttcctgcggccgcacgcgtgagggcctatttc-3â<br/>Primer AS1: 5â-ataggccctctctagaaaaaaagcaccgactc-3â<br/>Primer S2: 5â-tttttctagagagggcctatttcccatg-3â<br/>Primer AS2: 5â-atccatctttgcaaagcttacgcgtaaaaaagcaccgac-3â<br/><br/><p class="ppzz" style="margin-left:-30px;"><strong>Optional (see Procedure C)</strong> </p></li><li>NIH3T3 cell (ATCC, catalog number: CRL-1658)</li><li>AAV vector (pX458-sgRNA or pX458 empty as negative control )</li><li>Primers: You can design the primers to amplify the target exons by CHOPCHOP (see Procedure C)</li><li>PrimeSTAR HS DNA polymerase</li><li>FuGENE6 Transfection Reagent (Promega, catalog number: E2693)</li><li>DMEM (1x) (Gibco, catalog number: 11995-065)</li><li>Fetal Bovine Serum (Corning, catalog number: 35-015-CV)</li><li>Penicillin/Streptomycin (Gibco, catalog number: 15070-063)</li><li>PBS, pH 7.4 (Gibco, catalog number: 10010-023)</li><li>TrypLE Select (1x) (Gibco, catalog number: 12563-011)</li><li>QuickExtract DNA Extraction Solution (Epicentre, catalog number: QE0905T)</li><li>T7 Endonuclease I (New England Biolabs, catalog number: M0302)<br/><div style="white-space:nowrap;"></div></li></ol><p class="pptt" id="biaoti28813">### Equipment</p><ol class="npwtwok"><li>Pipettes (Gilson, Pipetman classic P10, P20, P200)</li><li>Centrifuge (Eppendorf, Centrifuge 5424)</li><li>Incubator (VWR, incubating 5000IR orbital shaker)</li><li>Thermocycler (Applied Biosystems, GeneAmp PCR system 2700)</li><li>Electrophoresis chamber (Labnet, ENDURO GEL XL)</li><li>(Optional) CO<sub>2</sub> Incubator (Binder, CB 170)</li><li>(Optional) Clean bench (Labconco, Purifier classII biosafety cabinet)</li><li>(Optional) Centrifugation (Thermo Scientific, Soryall legend XTR)<br/></li></ol><p class="pptt" id="biaoti28814">### Procedure</p><ol class="npwone"><li>Design of sgRNA<br/>Single guide RNAs (sgRNAs) with high efficiency and specificity can be designed by using gRNA design webtools. We recommend CHOPCHOP (<a href="http://chopchop.cbu.uib.no/" target="_blank">http://chopchop.cbu.uib.no</a>) because of its user-friendly interface (Labun <em>et al.</em>, 2016). Also, you may find the guide design resources on the website of Feng Zhangâs lab (<a href="https://zlab.bio/guide-design-resources" target="_blank">https://zlab.bio/guide-design-resources</a>). The sgRNA should target early exons (near transcriptional start site) or essential exons of gene function. We usually choose 4 to 6 target sites on early exons which do not have the start codon because frameshift mutations near the start codon can lead to translation from other in-frame ATGs, resulting in the expression of a functional protein. If the target sequence is 5â-CGCAGGACCAGACCCCATAATGG-3â [Protospacer adjacent motif (PAM) is labeled in blue], synthesize 5â-caccgCGCAGGACCAGACCCCATAA-3â (sense) and 5â-aaacTTATGGGGTCTGGTCCTGCGc-3â (anti-sense) to clone the guide sequence into pX458 (Ran <em>et al.</em>, 2013) sgRNA scaffold (Figure 3A). cacc (lower case) in the first oligo and aaac (lower case) in the second oligo are overhangs for cloning into BbsI-site in pX458. We also add âgâ between cacc and the target sequence in the first oligo because the human U6 promoter prefers âgâ at the transcription start site for strong expression. Also, you may design primers using CHOPCHOP to amplify the target site for further evaluation of sgRNAs using the T7 endonuclease I assay (see Procedure C).<br/><br/></li><li>Cloning of sgRNAs into the spCas9 expressing plasmid (pX458)<ol class="npwtwo"><li>Digestion of pX458 with BbsI:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>X Î¼l<br/></td><td>pX458 (10 Î¼g)<br/></td></tr><tr><td>5 Î¼l<br/></td><td>10x NEB cut smart<br/></td></tr><tr><td>1 Î¼l<br/></td><td>BbsI-HF<br/></td></tr><tr><td>24 - X Î¼l<br/></td><td>MilliQ water<br/></td></tr><tr><td>30 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Incubate at 37 Â°C overnight.</li><li>Gel purify the plasmid using QIAquick Gel extraction kit. Elute the plasmid with 50 Î¼l of 1x Tris-EDTA buffer.</li><li>Annealing of oligonucleotides encoding sgRNAs:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td>1 Î¼l<br/></td><td>oligonucleotide, sense (100 Î¼M)<br/></td></tr><tr><td>1 Î¼l<br/></td><td>oligonucleotide, antisense (100 Î¼M)<br/></td></tr><tr><td>8 Î¼l<br/></td><td>1x Tris-EDTA buffer<br/></td></tr><tr><td>10 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Incubate at 95 Â°C for 5 min. Then cool down to room temperature by switching off the thermocycler (30 min).</li><li>Prepare the ligation reaction as following:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td>1 Î¼l<br/></td><td>diluted (1:100 with 1x Tris-EDTA buffer) annealed oligonucleotides<br/></td></tr><tr><td>1 Î¼l <br/></td><td>BbsI-digested pX458 (50 ng/Î¼l)<br/></td></tr><tr><td>5 Î¼l<br/></td><td>2x ligation buffer<br/></td></tr><tr><td>1 Î¼l<br/></td><td>Quick ligase<br/></td></tr><tr><td>2 Î¼l <br/></td><td>MilliQ water<br/></td></tr><tr><td>10 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Incubate at room temperature for 10 min.</li><li>Add 5 Î¼l of the ligation reaction to 50 Î¼l of NEB10-beta.</li><li>Next place on ice for 10 min.</li><li>Heat shock at 42 Â°C for 30 s.</li><li>Transfer onto ice for 3 min.</li><li>Plate the reaction on an LB-Agar plate containing ampicillin (50 Î¼g/ml).</li><li>Incubate at 37 Â°C overnight.</li><li>Inoculate 2 ml LB broth containing ampicillin (50 Î¼g/ml) from a single colony and incubate at 37 Â°C overnight with agitation. <br/><em>Note: 60-70% of colonies have a correct insertion in our experience. </em> </li><li>Purify the plasmid from 1 ml of the overnight culture using QIAprep Spin Miniprep kit.<br/>Elute the plasmid with 50 Î¼l of buffer EB and determine the concentration</li><li>Verify the plasmid by sequencing with a primer (5â-gactatcatatgcttaccgt-3â).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><strong>(Optional) </strong>Evaluation of sgRNA by T7 endonuclease I assay<ol class="npwtwo"><li>Plate NIH3T3 cells at the density of 2.5 x 10<sup>4 </sup>cells (500 Î¼l/well, DMEM/10% FBS/1% Penicillin /1% Streptomycin) onto a well of 24-well plate.<br/></li><li>Incubate in a 37 Â°C CO<sub>2</sub> incubator overnight.</li><li>Transfection<br/><ol class="npwthree"><li>Add 1.5 Î¼l of FuGENE6 reagent to 50 Î¼l of serum-free DMEM in a 1 ml plastic tube and mix by vortexing.</li><li>Incubate the mixture for 5 min at room temperature.</li><li>Add 0.5 Î¼g of pX458-sgRNA or control vector to the mixture and mix by vortexing.</li><li>Incubate the mixture for 30 min at room temperature.</li><li>Add the mixture to each well of a 24-well plate containing cells.</li><li>Incubate in a 37 Â°C CO<sub>2</sub> incubator for 24 h.</li><li>Replace the cell culture media with fresh one (DMEM/10% FBS/1% Penicillin /1% Streptomycin).</li><li>Repeat Steps C3d-C3e. <br/><em>Note: Repeated transfection is essential for high expression of Cas9 and sgRNA to edit the target gene in NIH3T3 cells.</em> </li><li>Incubate in a 37 Â°C CO<sub>2</sub> incubator for 48 h.</li></ol></li><li>Remove the supernatant of cells by pipetting and wash cells with 500 Î¼l of 1x PBS.<br/></li><li>Add 100 Î¼l of QuickExtract to each well and collect the lysate containing genomic DNA to a 1 ml plastic tube.</li><li>Incubate at 65 Â°C for 15 min and then at 98 Â°C for 10 min.</li><li>Centrifuge at 20,000 <em>x g</em> for 10 min.</li><li>Collect 60 Î¼l of the supernatant (can be stored at -20 Â°C).</li><li>PCR to amplify 200-300 bp fragments around the CRISPR binding site. The primers can be designed by CHOPCHOP.<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:800px;"><tbody><tr><td>6 Î¼l<br/></td><td>lysate<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>dNTP (the mixture of dTTP, dCTP, dGTP and dATP, each at a final concentration of 10 mM)<br/></td></tr><tr><td>12 Î¼l<br/></td><td>5x buffer<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>primer S (10 Î¼M) <br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>primer AS (10 Î¼M)<br/></td></tr><tr><td>1 Î¼l<br/></td><td>PrimeSTAR HS DNA polymerase<br/></td></tr><tr><td>36.5 Î¼l<br/></td><td>MilliQ water<br/></td></tr><tr><td>60 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Start PCR with the following settings:<br/><ol class="npwthree"><li>Denaturation at 95 Â°C for 2 min.</li><li>35 cycles of<br/>95 Â°C for 10 s<br/>55 Â°C for 5 s<br/>72 Â°C for 45 s</li><li>Final elongation at 72 Â°C for 2 min.</li><li>Store at 4 Â°C.</li></ol></li><li>Gel purify PCR product using QIAquick Gel extraction kit. Elute PCR product with 50 Î¼l of buffer EB.<br/><ol class="npwthree"><li>Denaturarion and annealing of PCR product:<br/>5 Î¼l PCR product (200 ng)<br/>2 Î¼l 10x NEB buffer 2<br/>12 Î¼l MilliQ water<br/>19 Î¼l in total</li><li>95 Â°C for 5 min. </li><li>Cool down to room temperature by switching off the thermocycler (30 min).</li></ol></li><li>Add 1 Î¼l of diluted (1:5) T7 endonuclease I (2.5 U/Î¼l).<br/><em>Note: Dilution of T7 endonuclease I is essential because an excess amount of the enzyme will result in non-specific cleavage of DNA.</em> </li><li>Incubate at 37 Â°C for 30 min.</li><li>Run on 2% agarose gel and verify bands in sgRNA transfected-wells compared with control wells (Figure 2).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815210726_7577.jpg" width="191"/><br/><strong>Figure 2. Agarose gel electrophoresis of T7 endonuclease I-digested PCR products derived from NIH3T3 cells transfected with sgRNA and Cas9.</strong> The target exon was PCR-amplified and gel purified. After the denaturation and annealing, the PCR product was digested by T7 endonuclease I and run on 2% agarose gel. The asterisk indicates cleaved DNA fragment. In this experiment, we found only one shorter fragment. However, you may find bands of two different sizes after the cleavage if you carefully design the primers. It can be useful to design PCR primers such that the indel resulting from genome editing is not in the middle of the amplicon, ultimately resulting in two cleaved bands of different sizes.<br/><br/>You can quantify the efficiency of gene modification based on the result of T7 endonuclease I assay. The percentage of cleavage on a gel could be converted to the percentage of gene modification by using an equation below. <br/><br/><div style="text-align:center;"><span style="font-size:10pt;">% gene modification = 100 x (1 - (1 - fraction cleaved)</span><sup>1/2</sup><span style="font-size:10pt;"> )(Guschin <em>et al.</em>, 2010)</span> </div><br/>Also, you may evaluate the efficiency of gene editing by a webtool called Synthegoâs ICE (<a href="https://ice.synthego.com/#/" target="_blank">https://ice.synthego.com/#/</a>). Target exons from edited or unedited (control) cells are PCR-amplified and Sanger sequenced. You may submit the results of sequencing to the webtool and ICE compares these sequence traces to give a detailed analysis of gene editing. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Tandem cloning of two sgRNAs into pAAV EF1Î± DIO mCherry<br/>In Procedure A, you clone several sgRNAs targeting the same gene. For the efficient gene disruption, you tandemly clone two different sgRNAs driven by independent human U6 promoters into an AAV vector (Figures 3B and 3C) using the Gibson Assembly method. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190815/20190815210809_5388.jpg" width="230"/><br/><strong>Figure 3. Construction strategy of a dual sgRNA viral vector.</strong> A. Cloning of guide oligonucleotides. Oligonucleotides with 20-nt guide sequence and overhangs for cloning into BbsI site in pX458 were annealed and cloned into BbsI-digested pX458. B. Cloning of U6 promoter-driven sgRNAs. U6 promoter followed by sgRNA1 or sgRNA2 were PCR amplified, respectively, by using primers with overhangs for Gibson Assembly. C. Gibson Assembly of the dual sgRNA virus vector. U6-gRNA1 and U6-gRNA2 were cloned into MluI-digested pAAV EF1ï¡ DIO mCherry using Gibson Assembly.<br/><br/><ol class="npwtwo"><li>Digestion, dephosphorylation and purification of pAAV EF1Î± DIO mCherry with MluI.<br/><ol class="npwthree"><li>Digestion:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td>X Î¼l <br/></td><td>pAAV EF1Î± DIO mCherry (10 Î¼g)<br/></td></tr><tr><td>5 Î¼l<br/></td><td>10x NEB cut smart<br/></td></tr><tr><td>1 Î¼l<br/></td><td>MluI-HF<br/></td></tr><tr><td>24 - X Î¼l<br/></td><td>MilliQ water<br/></td></tr><tr><td>30 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Incubate at 37 Â°C overnight.</li><li>Add 1 Î¼l of CIP, and then incubate at 37 Â°C for 60 min.</li><li>Gel purify the plasmid using QIAquick Gel extraction kit. </li><li>Elute the plasmid with 50 Î¼l of buffer EB.</li></ol></li><li>Amplification of U6 promoter plus sgRNA#1 or sgRNA#2.<br/><ol class="npwthree"><li>PCR reaction#1 (Expected size is 393 bp):<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td>3 Î¼l <br/></td><td>pX458-sgRNA#1 (10 ng/Î¼l)<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>dNTP, 10 mM each<br/></td></tr><tr><td>12 Î¼l<br/></td><td>5x buffer<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>primer S1 (10 Î¼M); 5â- taggggttcctgcggccgcacgcgtgagggcctatttc -3â<br/></td></tr><tr><td>1.5 Î¼l <br/></td><td>primer AS1 (10 Î¼M); 5â- ataggccctctctagaaaaaaagcaccgactc -3â<br/></td></tr><tr><td>1 Î¼l <br/></td><td>PrimeSTAR HS DNA polymerase<br/></td></tr><tr><td>39.5 Î¼l<br/></td><td>MilliQ water<br/></td></tr><tr><td>60 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>PCR reaction#2 (Expected size is 388 bp):<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td>3 Î¼l<br/></td><td>pX458-sgRNA#2 (10 ng/Î¼l)<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>dNTP, 10 mM<br/></td></tr><tr><td>12 Î¼l<br/></td><td>5x buffer<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>primer S2 (10 Î¼M); 5â- tttttctagagagggcctatttcccatg -3â<br/></td></tr><tr><td>1.5 Î¼l<br/></td><td>primer AS2 (10 Î¼M); 5â- atccatctttgcaaagcttacgcgtaaaaaagcaccgac -3â<br/></td></tr><tr><td>1 Î¼l<br/></td><td>PrimeSTAR HS DNA polymerase<br/></td></tr><tr><td>39.5 Î¼l<br/></td><td>MilliQ water<br/></td></tr><tr><td>60 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li></ol></li><li>Start PCR with the following settings.<br/><ol class="npwthree"><li>Denaturation at 95 Â°C for 2 min.</li><li>35 cycles of<br/>95 Â°C for 10 s<br/>62 Â°C for 5 s<br/>72 Â°C for 45 s</li><li>Final elongatation at 72 Â°C for 2 min.</li><li>Store at 4 Â°C.</li></ol></li><li>Run on a 2% agarose gel.</li><li>Gel purify the PCR product using QIAquick Gel extraction kit. Elute the PCR product with 50 Î¼l of MilliQ water.</li><li>Digestion of the residual template plasmid:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:350px;"><tbody><tr><td>45 Î¼l<br/></td><td>PCR amplicon#1 or #2<br/></td></tr><tr><td>5 Î¼l<br/></td><td>10x NEB cut smart<br/></td></tr><tr><td>1 Î¼l<br/></td><td>DpnI<br/></td></tr><tr><td>51 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Incubate at 37 Â°C for 30 min, 80 Â°C for 30 min.</li><li>Gibson assembly:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td>1 Î¼l<br/></td><td>CIP-treated MluI-digested pAAV EF1Î± DIO mCherry (100 ng/Î¼l)<br/></td></tr><tr><td>3 Î¼l<br/></td><td>PCR amplicon#1<br/></td></tr><tr><td>3 Î¼l<br/></td><td>PCR amplicon#2<br/></td></tr><tr><td>3 Î¼l<br/></td><td>MilliQ water<br/></td></tr><tr><td>10 Î¼l<br/></td><td>Gibson Assembly mix<br/></td></tr><tr><td>20 Î¼l<br/></td><td>in total<br/></td></tr></tbody></table></li><li>Incubate at 50 Â°C for 15 min.</li><li>Add 2 Î¼l of ligation reaction to 50 Î¼l of NEB10-beta.</li><li>Follow Steps B9-B15.</li><li>Verify the plasmid by sequencing with a primer (<em>e.g.</em>, 5â-actgacgggcaccggagcca-3â).</li><li>Proceed to virus packaging.<br/><em>Note: We usually request virus packaging services from </em><a href="https://neuroscience.stanford.edu/research/programs/community-labs/neuroscience-gene-vector-and-virus-core" target="_blank"><em>Stanford Neuroscience Gene Vector and Virus core</em></a><em>.</em> </li><li>Keep the virus stock at -80 Â°C.<br/><em>Note: This strategy can disrupt genes in a cell type-specific manner. You can use the virus not only for adult mouse brain but also other organs.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti28815">### Recipes</p><p class="ppzz">All materials and reagents are commercially available.</p><p class="pptt" id="biaoti28816">### Acknowledgments</p><p class="ppzz">H.Y. was supported by Uehara memorial foundation research fellowship. L.d.L was supported by National Institute of Health Grants AG047671, MH087592, MH102638. This protocol was adapted from Yamaguchi <em>et al.</em> (2018) and includes a recent improvement of the method of our lab.</p><p class="pptt" id="biaoti28817">### Competing interests</p><p class="ppzz">Competing interestsThe authors declare no competing financial interests.</p><p class="pptt" id="biaoti28818">### References</p><ol class="npwtwok"><li>Labun, K., Montague, T. G., Gagnon, J. A., Thyme, S. B. and Valen, E. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27185894" target="_blank">CHOPCHOP v2: a web tool for the next generation of CRISPR genome engineering.</a> <em>Nucleic Acids Res</em> 44(W1): W272-276.</li><li>Platt, R. J., Chen, S., Zhou, Y., Yim, M. J., Swiech, L., Kempton, H. R., Dahlman, J. E., Parnas, O., Eisenhaure, T. M., Jovanovic, M., Graham, D. B., Jhunjhunwala, S., Heidenreich, M., Xavier, R. J., Langer, R., Anderson, D. G., Hacohen, N., Regev, A., Feng, G., Sharp, P. A. and Zhang, F. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25263330" target="_blank">CRISPR-Cas9 knockin mice for genome editing and cancer modeling.</a> <em>Cell</em> 159(2): 440-455.</li><li>Ran, F. A., Hsu, P. D., Wright, J., Agarwala, V., Scott, D. A. and Zhang. F. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24157548" target="_blank">Genome engineering using the CRISPR-Cas9 system.</a> <em>Nat Protoc</em> 8(11): 2281-2308.</li><li>Swiech, L., Heidenreich, M., Banerjee, A., Habib, N., Li, Y., Trombetta, J., Sur, M. and Zhang, F. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25326897" target="_blank"><em>In vivo</em> interrogation of gene function in the mammalian brain using CRISPR-Cas9.</a> <em>Nat Biotechnol</em> 33(1): 102-106.</li><li>Yamaguchi, H., Hopf, F. W., Li, S. B. and de Lecea, L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30523254" target="_blank"><em>In vivo</em> cell type-specific CRISPR knockdown of dopamine beta hydroxylase reduces locus coeruleus evoked wakefulness.</a> <em>Nat Commun</em> 9(1): 5211.</li><li>Guschin, D. Y., Waite, A. J., Katibah, G. E., Miller, J. C., Holmes, M. C. and Rebar, E. J. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20680839" target="_blank">A rapid and general assay for monitoring endogenous gene modification.</a> <em>Methods Mol Biol</em> 649: 247-256.<br/></li></ol>