<body>
            <h1>Measurement of the Length of the Integrated Donor DNA during <em>Bacillus subtilis</em>  Natural Chromosomal Transformation</h1>
            <p><strong>Authors:</strong> Ester Serrano, BegoÃ±a Carrasco</p>
            <h2>Abstract</h2>
            <p>For natural transformation to occur, bacterial cells must first develop a programmed physiological state called competence. Competence in <em>Bacillus subtilis</em>, which occurs only in a fraction of cells, is a transient stress response that allows cells to take up DNA from the environment. During natural chromosomal transformation, the internalized linear single-stranded (ss) DNA recombines with the identical (homologous) or partially identical (homeologous) sequence of the resident duplex. The length of the integrated DNA, which can be measured, depends on the percentage of sequence divergence between the donor (internalized) and the recipient (chromosomal) DNAs.<br/><br/>The following protocol describes how to induce the development of competence in <em>B. subtilis</em> cells, how to transform them with donor DNAs representing different percentages of sequence divergence compared with the recipient chromosomal DNA, how to calculate the chromosomal transformation efficiency for each of them, and how to amplify the chromosomal DNA from the transformants in order to measure the length in base pairs (bp) of the integrated donor DNA.</p>
            <p><strong>Keywords:</strong> Natural chromosomal transformation, <em>Bacillus subtilis</em> competent cells, Divergent DNA sequences, Integration length</p>
            <h2>Background</h2><p>Natural transformation is a bacterial programmed mechanism mediated by natural competence systems encoded in the genomes of many bacteria (Chen and Dubnau, 2004; Chen <em>et al.</em>, 2005; Takeuchi <em>et al.</em>, 2014; Johnston <em>et al.</em>, 2014). Competence development allows cells to take up DNA from the environment (Chen and Dubnau, 2004; Chen <em>et al.</em>, 2005; Thomas and Nielsen, 2005; Kidane <em>et al.</em>, 2012; Yadav <em>et al.</em>, 2012; Yadav <em>et al.</em>, 2013; Takeuchi <em>et al.</em>, 2014; Johnston <em>et al.</em>, 2014). <br/><br/>In <em>Bacillus subtilis</em> transient natural competence is induced in a subset of cells by starving them of critical nutrients (Kidane <em>et al.</em>, 2012; Chen and Dubnau, 2004). In the competent subpopulation, DNA replication is halted, expression of a set of genes is induced, and the DNA uptake apparatus is built at one cell pole (Chen and Dubnau, 2004; Kidane <em>et al.</em>, 2012). The DNA uptake apparatus binds environmental double-stranded (ds) DNA, degrades one of the strands, transports the other strand (independently of its nucleotide sequence and polarity) through the cell wall and the membrane into the cytosol. During chromosomal transformation, the incoming single-stranded (ss) DNA is integrated into the recipient dsDNA replacing homologous (or partially homologous) chromosomal sequences (Kidane <em>et al.</em>, 2012).<br/><br/>It has been already shown that the divergence between the sequences of the internalized and the recipient DNAs, provides a barrier to chromosomal transformation and decreases the length of integration of donor DNA (Carrasco <em>et al.</em>, 2016; Carrasco <em>et al.</em>, 2019). <br/><br/>In this protocol, we describe how to transform <em>B. subtilis</em> competent cells to finally measure the length of the integrated DNA. The experimental procedure described here to prepare <em>B. subtilis</em> competent cells is an adaptation of the previous described by Dubnau <em>et al.</em>, 1973 and Alonso <em>et al.</em>, 1988. </p>
        </body>

<p class="pptt" id="biaoti28858">## Materials and Reagents</p><ol class="npwtwok"><li>Petri dishes 90 mm (VWR, catalog number: 391-0440)</li><li>Sterile wood toothpicks </li><li>Sterile pipette tips<br/></li><li>50- and 100-ml Flasks (Duran, catalog numbers: 21-216-17 and 21-216-24, respectively)</li><li>1.5- and 2-ml tubes (Sarstedt, catalog number: 72.690.001; Fisherbrand, catalog number: 11393613)<br/></li><li>Polystyrol/Polystyrene cuvettes 10 x 4 x 45 mm (SARSTEDT, catalog number: 67742)</li><li>Semi-log paper</li><li>Glass beads 3 mm (Sigma-Aldrich, catalog number: Z143928-1EA)</li><li><em>Bacillus subtilis</em> cells (wild-type strain [<em>B. subtilis</em> 168] or derivative deletion mutants)</li><li>Primers (Sigma-Aldrich)<br/>Forward primer: 5â²-CATATAATACGCATGATTTGAGGGG<br/>Reverse primer: 5â²-GTCCGTCGTAAAGCACTGTTTTG</li><li>Plasmids: pCB980; pCB981; pCB982; pCB983; pCB984; pCB1054; pCB985; pCB1056<br/><em>Note: These plasmids have been previously reported in Carrasco </em><em>et al.</em><em>, 2016 and Carrasco </em><em>et al.</em><em>, 2019.</em> </li><li><em>Escherichia coli</em> 2,710-bp pUC57 vector</li><li>Bi-distilled H<sub>2</sub>O (ddH<sub>2</sub>O)</li><li>Glycerol (Sigma-Aldrich, catalog number: G6279-4L)</li><li>Rifampicin (Sigma-Aldrich, catalog number: 13292-46-1)</li><li>Ampicillin sodium salt (Sigma-Aldrich, catalog number: 69-52-3)</li><li>Plasmid purification kit (Qiagen plasmid mini kit, catalog number: 27106)</li><li>Bacto tryptone pancreatic digest of casein (BD Biosciences, catalog number: 211705)</li><li>Bacto yeast extract (Extract of autolyzed yeast cells) (BD Biosciences, catalog number: 211750)</li><li>European bacteriological agar (Pronadisa, catalog number: 1800)</li><li>NaCl (VWR, catalog number: 7647-14-5) </li><li>Agarose D1 medium EEO (Pronadisa, catalog number: 8021)</li><li>PCR purification kit (Speedtools PCR clean up kit, Biotools, catalog number: 740609.50.205571)</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Calbiochem, Merck, catalog number: 7558-79-4)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Calbiochem, Merck, catalog number: 7778-77-0)</li><li>K<sub>2</sub>HPO<sub>4</sub> (Calbiochem, Merck, catalog number: 16788-57-1) </li><li>D-(+)-Glucose (Sigma-Aldrich, catalog number: 50-99-7) </li><li>Tri-Sodium citrate dihydrate (Na<sub>3</sub>C<sub>6</sub>O<sub>7</sub>Â·2H<sub>2</sub>O) (Merck, catalog number: 6132-04-3) </li><li>Ammonium sulfate (MP Biomedical, catalog number: 7783-20-2)</li><li>Titriplex III (EDTA) (Merck, catalog number: 6381-92-6)</li><li>Acetic acid glacial (AppliChem PanReac, catalog number: 141008.1214) </li><li>Casein hydrolysate (EMD, catalog number: 65072-00-6) </li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (Merck, catalog number: 10035-04-8)</li><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Merck, catalog number: 10034-99-8)</li><li>Tryptophan (Merck, catalog number: 54-12-6)</li><li>Methionine (Merck, catalog number: 63-68-3)</li><li>Tris-HCl (AppliChem PanReac, catalog number: A1086)</li><li>1 M Tris-HCl, pH 7.5 (see Recipes) </li><li>Luria Bertani (LB) broth (see Recipes)</li><li>LB-agar (see Recipes)</li><li>GM1 (see Recipes)</li><li>GM2 (see Recipes)</li><li>SBase10x (see Recipes)</li><li>TAE 50x (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28859">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>30 and 37 Â°C incubators</li><li>Amersham Biosciences Ultrospec 3100 Pro UV/Visible Spectrophotometer (GE Healthcare)</li><li>Heating block</li><li>Microcentrifuge </li><li>Thermal cycler (VWR, Doppio, catalog number: 732-2551) </li><li>NanoDrop<sup>TM</sup> 1000 Spectrophotometer (Thermo Fisher Scientific)</li><li>Autoclave</li><li>-80 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti28860">### Software</p><ol class="npwtwok"><li>Sequence data compilation<br/>Sequences of <em>rpoB</em> genes from different <em>Bacillus</em> species or subspecies need to be downloaded from relevant databases. A recommended database is <a href="https://www.ncbi.nlm.nih.gov/gene/?term=rpob%20and%20bacillus" target="_blank">NCBI</a>. A single C to T transition mutation at codon 482 (position 1,446) in the house-keeping <em>rpoB</em> gene, confers resistance to rifampicin (Rif<sup>R</sup>). To generate the <em>rpoB</em>482 sequence variants, a C to T change is required at the position 1,446 of each sequence (see Figure 1A, asterisk).</li><li>Online tool (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a>) to perform the nucleotide alignment<br/></li></ol><p class="pptt" id="biaoti28861">### Procedure</p><ol class="npwone"><li>Preparation of <em>Bacillus subtilis</em> competent cells<ol class="npwtwo"><li>Streak <em>B. subtilis</em> cells onto LB-agar plate.</li><li>Incubate the plate at 37 Â°C overnight (O/N) or room temperature (RT) for 2-3 days.</li><li>Using a sterile toothpick of wood, inoculate 5 ml of GM1 with a single colony from the O/N plate in a 50-ml flask and incubate the culture O/N without aeration at 30 Â°C.</li><li>Dilute the overgrown culture in 15 ml of GM1 to reach an OD<sub>560</sub> of 0.05.</li><li>Divide the culture into two parts: 100-ml flask containing 10 ml of culture to measure the OD<sub>560</sub> and 50-ml flask containing 5 ml to freeze the cells. The division of the culture in two flasks is recommended to avoid the contamination of the culture that it will be frozen. The possible contamination might occur due to the multiple OD<sub>560</sub> measurements.</li><li>Incubate the cultures at 37 Â°C in an incubator for flasks at 250 rpm. Two hours later start measurements (from the 100-ml flask for measurements) at OD<sub>560</sub> every 30 min. Plot the data in a semi-log paper to predict the point of inflection between the exponential phase of growth and the stationary phase (OD<sub>560</sub> of approximately 1).</li><li>One hour after having reached the point of inflection in the growth curve, cells have become competent. At this point, there are two options (always use the 5 ml culture from the 50-ml flask that does not stop shaking): transform the cells (see Step B3) or freeze the cells to use them else when. To freeze the cells, add glycerol (20% final concentration) and, as soon as possible, divide the culture into 1 ml aliquots in 2-ml tubes and put them on dry ice. Keep the tubes at -80 Â°C.<br/><em>Note: This section of the protocol is an adaptation of the previously described methods by Dubnau </em><em>et al.</em><em>, 1973 and Alonso </em><em>et al.</em><em>, 1988.</em> </li></ol><br/></li><li><em>Bacillus subtilis</em> chromosomal transformation (see Figure 1B)<ol class="npwtwo"><li>Add the frozen cells (1 ml) into 10 ml GM2 medium (see Recipes) in a 100-ml flask.</li><li>Incubate the culture for 1.5-2 h at 37 Â°C with shaking speed as aforementioned.</li><li>Transfer 1 ml of competent cells to 2-ml tubes:<br/><ol class="npwthree"><li>One tube without DNA (negative control) (âTube [-]â).</li><li>One tube containing 1,000 ng of <em>rpoB</em>482 DNAs (pCB980-1056) (see Table 1 and Figure 1A) (âTube [+]â). DNA concentration should be measured using a Nanodrop.</li></ol><br/><strong>Table 1. Plasmid-borne </strong><em><strong>rpoB</strong></em><strong>482 variants from different </strong><em><strong>Bacillus</strong></em><strong> species or subspecies cloned into <em>E. coli</em> pUC57 vector.</strong> The house-keeping <em>rpoB</em> gene encodes the essential Î² subunit of RNA polymerase. A single C to T transition at codon 482 (position 1446) in the <em>rpoB</em> gene confers resistance to rifampicin (Rif<sup>R</sup>) (Nicholson and Maughan, 2002). This mutation was introduced in the <em>rpoB</em> region of the indicated <em>Bacillus</em> species or subspecies leading to the <em>rpoB</em>482 mutant variants for selection of the Rif<sup>R</sup> transformants (see Figure 1A). Homologous regions at 5â²- and 3â²-ends of the different <em>rpoB</em>482 genes were selected to eliminate variants in the PCR amplification efficiencies. The 2,997-bp segment of the different <em>rpoB</em>482 genes was <em>in vitro</em> synthesized and cloned into <em>Escherichia coli</em> 2,710-bp pUC57 vector (Ampicillin resistance). The accuracy of the <em>in vitro</em> synthesized DNAs was confirmed by nucleotide sequence (GeneWiz, London, UK).<br/><img alt="" class="layerphoto" height="278" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816004428_8268.jpg" width="650"/><br/><br/></li><li>Incubate the tubes at 37 Â°C for 1 h with shaking in a heating block at 250 rpm.</li><li>Serial dilutions and plating.<br/><em>Note: Depending on the </em><em>rpoB</em><em>482 DNAs (see Table 1) and the mutant competent cells used, preliminary experiments should be performed to determine the range of dilutions required for plating.</em><br/><ol class="npwthree"><li>For plating the total number of cells, create 1:10 serial dilutions of the âTube (-)â (negative control without DNA, see Step B3a) to a 10<sup>-5</sup> dilution in LB, and plate 100 Î¼l of desired dilutions onto LB-agar plates using glass beads.</li><li>For plating the transformants (see Figure 1C), create 1:10 serial dilutions of the âTube (+)â (with DNA, see Step B3b) to a 10<sup>-2</sup> dilution in LB and plate 100 Î¼l of desired dilutions onto LB-agar plates supplemented with rifampicin (8 Î¼g/ml) using glass beads.<br/><em>Note: When the chromosomal transformation efficiency is impaired, 10<sup>0</sup> and 10<sup>1</sup> dilutions could be necessary. In this case, harvest cells by centrifugation for 3 min at 900 x g at room temperature (RT). There should be a visible pellet at the bottom of the microfuge tube. Resuspend cells in 100 Î¼l LB and plate the whole volume onto LB-agar plates supplemented with rifampicin (8 Î¼g/ml) using glass beads.</em> </li><li>For plating the spontaneous rifampicin resistant (Rif<sup>R</sup>) cells, plate the âTube (-)â (negative control without DNA, see Step B3a) at the same dilution as the most concentrated dilution of transformants (see Step B5b) onto LB-agar plates supplemented with rifampicin (8 Î¼g/ml) using glass beads.</li><li>Incubate plates at 37 Â°C overnight, or leave at RT for 2-3 days.</li><li>Count colonies on the most appropriate plate (the one that contains around 100-1000 colonies). The colonies on each plate must be multiplied by their dilution factor to obtain colony forming units (CFUs)/ml.<br/><em>Note: In order to reduce the variability between different experiments, count 100-500 CFUs per plate from the proper dilution.</em> </li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>PCR amplification of <em>rpoB</em>482 DNA from the Rif<sup>R</sup> transformants (see Figure 1C)<br/><ol class="npwtwo"><li>Lyse the transformant cells<ol class="npwthree"><li>Pick up a single colony with a sterile toothpick and insert it into a 1.5 ml tube containing 20 Î¼l ddH<sub>2</sub>O and incubate the tube for 5 min at 95 Â°C.<br/><em>Note: It is necessary to amplify by PCR and send for sequencing several examples (several single colonies) resulted from the transformation with each DNA. This is because the recombination point can vary (from one to another) so that the end points and the length of integration also varies. Finally, we determine the length of integration as the average of lengths from different examples of each condition.</em> </li><li>Centrifuge for 5 min at 20,000 <em>x g</em> at RT.</li><li>Use 2 Î¼l of supernatant per PCR reaction as DNA template.</li></ol></li><li>Perform a PCR 20 Î¼l per reaction.<ol class="npwthree"><li>Set up the PCR, by adding the following component into a PCR tube on ice:<br/>DNase-free ddH<sub>2</sub>O 11.5 Î¼l<br/>dNTP mix (2.5 Î¼M each) 2 Î¼l<br/>Forward primer (10 Î¼M) (see Table 2) 2 Î¼l<br/>Reverse primer (10 Î¼M) (see Table 2) 1 Î¼l<br/>PCR buffer (10x) 2 Î¼l<br/>Taq DNA polymerase (250 Units) 2 Î¼l<br/>Supernatant lysates (DNA template) 2 Î¼l<br/><br/><strong>Table 2. Primers.</strong> These primers hybridize with the region upstream and downstream of the 2,997-bp <em>rpoB</em>482 DNAs. These primers are used in this protocol for amplifying and sequencing <em>rpoB</em>482 genes from the Rif<sup>R</sup> transformants.<br/><img alt="" class="layerphoto" height="78" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816004804_3364.jpg" width="350"/><br/><br/></li><li>Run PCR:<table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td>1 cycle<br/></td><td>94 Â°C, 10 min<br/></td></tr><tr><td>30 cycles<br/></td><td>94 Â°C, 1 min; 55 Â°C, 1 min and 30 s; 72 Â°C, 2 min and 45 s<br/></td></tr><tr><td>1 cycle<br/></td><td>72 Â°C, 10 min<br/></td></tr></tbody></table></li><li>Load 2 Î¼l of PCR products on a 0.8% agarose gel in 1x TAE stained with EtBr. Run the gel in 1x TAE buffer at 5 to 8 V/Cm for 1 h at RT (bands expected of 2,997 bp). Visualize the gel under UV light.</li><li>Purify the PCR products using the PCR purification kit (see Materials and Reagents).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816004927_0575.jpg" width="140"/><br/><strong>Figure 1. <em>Bacillus subtilis</em> natural chromosomal transformation and detection of mean integration.</strong> A. Distribution of sequence divergence among different <em>Bacillus</em> species. A single C-to-T transition mutation at codon 482 (indicated with an asterisk) in the essential <em>rpoB</em> gene renders the <em>rpoB</em>482 gene, which confers rifampicin-resistant (Rif<sup>R</sup>). The <em>rpoB</em>482 DNA was derived from <em>B. subtilis</em> 168 (<em>Bsu</em> 168), <em>B. subtilis</em> W23 (<em>Bsu</em> W23), <em>B. atrophaeus</em> 1942 (<em>Bat</em> 1942), <em>B. amyloliquefaciens</em> DSM7 (<em>Bam</em> DSM7), <em>B. licheniformis</em> DSM13 (<em>Bli</em> DSM13), <em>B. gobiensis</em> FJAT-4402 (<em>Bgo</em> FJAT4402), <em>B. thuringiensis</em> MC28 (<em>Bth</em> MC28) and <em>B. smithii</em> DSM4512 (<em>Bsm</em> DSM4216). Mismatches between Rif<sup>R</sup> donor and the corresponding <em>rpoB</em> in the rifampicin sensitive recipient strain are indicated by vertical red bars, and insertions/deletions by vertical black bars. Bar thickness represents the number of mismatches in a particular area. B. Chromosomal transformation. RecA-mediated homeologous recombination between the <em>rpoB</em>482 donor DNAs (in red) and the recipient <em>rpoB</em> gene (in black). C. Measurement of the length of the integrated donor DNA into the recipient. PCR amplification of the Rif<sup>R</sup> transformants using DNA primers hybridizing upstream and downstream the <em>rpoB</em> gene (blue arrows). The mean length detection between the first predicted and the first observed mismatch define the end-points region.<br/><br/></li><li>Sequencing<br/>Send for sequencing each PCR product mixed with Forward and/or Reverse primer (see Table 2) (following the instructions of the company).<br/><br/></li><li>Computational analysis<br/>Run two different nucleotide BLASTs:<ol class="npwtwo"><li>BLAST-1 (see Figure 2A): to visualize the bases that are not aligned (defined here as mismatches) between the Rif<sup>R</sup> donor DNAs (see Table 1) and the corresponding <em>rpoB</em> in the rifampicin sensitive recipient strain, run a nucleotide BLAST using each of the <em>rpoB</em>482 variants sequences (donor DNAs) (see Table 1) as query sequence against the house-keeping <em>rpoB</em> gene sequence (recipient DNA) as subject sequence.<br/><em>Note: The presence or the absence of the mismatches between the donor and the recipient DNA will be used to determine the overall integration length.</em> </li><li>BLAST-2 (see Figure 2B): to visualize the bases from the Rif<sup>R</sup> transformants with donor DNA origin, run a nucleotide BLAST using the house-keeping <em>rpoB</em> gene sequence (recipient DNA) as query sequence against each of the Rif<sup>R</sup> transformants sequences (see Procedure D) as subject sequences.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816005006_2471.jpg" width="241"/><br/><strong>Figure 2. Analysis of the integrated sequences and measurement of the integration length.</strong> A. Schematic representation of the BLAST-1 using <em>rpoB</em>482 donor DNA (with different <em>Bacillus</em> origins) (red line) as query sequence against the house-keeping <em>rpoB</em> gene sequence (recipient DNA) (black line) as subject sequence. Black dotted lines indicate the âFirst mismatches predictedâ. B. Schematic representation of the BLAST-2 using the house-keeping <em>rpoB</em> gene sequence (recipient DNA) (black line) as query sequence against the PCR product got from the Rif<sup>R</sup> transformants DNA (blue line) as subject. Black dotted lines indicated the âFirst mismatches observedâ. End-points are defined as the average region between the first predicted and first observed mismatches and represent the recombination points. The integration length is calculated as the subtraction between both end-points. Black dots represent DNA bases and blue vertical lines represent the bases that are aligned between both sequences. The mutation that confers resistant to rifampicin is represented with an asterisk.</li></ol></li></ol><p class="pptt" id="biaoti28862">### Data analysis</p><ol class="npwone"><li>Chromosomal transformation efficiency<br/>Colonies on each plate must be multiplied by their dilution factor to obtain CFUs/ml (see âStep B5eâ). For data analysis, transformation efficiency is calculated using the formula: Chromosomal transformation efficiency = T-E/Nf, where âTâ corresponds to the transformants (see Step B5b), âEâ corresponds to the spontaneous Rif<sup>R</sup> cells (see Step B5c) and âNfâ is the total number of cells (see âStep B5eâ).<br/><br/></li><li>Analysis of the integrated sequences<ol class="npwtwo"><li>Go to the resulting BLAST-2 (see Step E2 and Figure 2B) and visualize the fragment of donor DNA that is integrated into the recipient DNA by transformation (see Figures 1C and 2B).<em> </em> </li><li>Go to the resulting BLAST-2 (see Step E2 and Figure 2B) and search for the first mismatch between the query (recipient DNA) and the subject (Rif<sup>R</sup> transformant DNA) sequences, to identify the âFirst mismatch observedâ.<br/><em>Note: The bases aligned between these two sequences correspond to the part of the recipient DNA, whereas the mismatches represent the bases with donor origin in the Rif<sup>R</sup> transformant sequence.</em><br/><br/></li></ol></li><li>Map of integration end-points<br/>We define end-points as the borders of the donor DNA sequence integrated into the recipient DNA.<ol class="npwtwo"><li>Knowing the position of the âFirst mismatch observedâ (see Data analysis B), go to that position in the resulting BLAST-1 (see Step E1 and Figure 2A) and search upstream into the sequence, the first mismatch between the query sequence (recipient DNA) and the subject sequence of interest (depending on the donor DNA used) to identify the âFirst mismatch predictedâ.</li><li>Calculate the end-point using the formula: end-point= Fp-Fo/2, where âFpâ corresponds to the âFirst mismatch predictedâ and âFoâ corresponds to the âFirst mismatch observedâ.<br/><em>Note: We assume that the recombination events have occurred between the end-points at the borders of the donor DNA sequence integrated into the recipient one. It is necessary to identify the âFirst predictedâ and the âFirst observedâ mismatches from both borders for determining the two end-points (see Figure 2).</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Determination of the integration length<br/>Calculate the integration length as the subtraction between both end-points (see Figure 2).<br/></li></ol><p class="pptt" id="biaoti28863">### Recipes</p><ol class="npwtwok"><li>1 M Tris-HCl, pH 7.5<br/>Dilute 121.1 g Tris in 1 L of ultrapure water and adjust pH to 7.4 with HCl</li><li>Luria Bertani (LB) broth<br/>10 g/L Bacto tryptone<br/>5 g/L Bacto yeast extract<br/>10 g/L NaCl<br/>Sterilize by autoclaving</li><li>LB-agar<br/>LB medium<br/>12 g/L Agar<br/>Sterilize by autoclaving</li><li>GM1 (Table 3)<br/><em>Note: Filter the GM1 medium using vacuum filtration and store at 4 Â°C.</em><br/><br/><strong>Table 3. GM1 medium recipe</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816005108_4248.jpg" width="450"/><br/><br/></li><li>GM2 (Table 4)<br/><em>Note: GM2 medium should be prepared before use.</em><br/><br/><strong>Table 4. GM2 medium recipe</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816005206_1245.jpg" width="400"/><br/><br/></li><li>SBase 10x (Table 5)<br/><br/><strong>Table 5. SBase 10x recipe</strong><br/><img alt="" class="layerphoto" height="114" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816005303_4662.jpg" width="400"/><br/><br/></li><li>TAE 50x (Table 6)<br/><br/><strong>Table 6. TAE 50x recipe and preparation</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190816/20190816005406_8448.jpg" width="420"/><br/></li></ol><p class="pptt" id="biaoti28864">### Acknowledgments</p><p class="ppzz">This work was supported by the Ministerio de Ciencia, InnovaciÃ³n y Universidades (MICINN)/ FEDER PGC2018-097054-B-I00 to J. C. Alonso. ES thanks MINECO (BES-2013-063433) for the fellowship and B.C thanks Programa Centros de Excelencia Severo Ocho-CNB for fellowship.</p><p class="pptt" id="biaoti28865">### Competing interests</p><p class="ppzz">The authors have no conflicts of interest or competing interests with respect to this work.</p><p class="pptt" id="biaoti28866">### References</p><ol class="npwtwok"><li>Alonso, J. C., Tailor, R. H. and Luder, G. (1988). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3133357" target="_blank">Characterization of recombination-deficient mutants of <em>Bacillus subtilis</em>.</a> <em>J Bacteriol</em>. 170(7): 3001-3007.</li><li>Carrasco, B., Serrano, E., Martin-Gonzalez, A., Moreno-Herrero, F. and Alonso, J. C. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30814990" target="_blank"><em>Bacillus subtilis</em> MutS modulates RecA-mediated DNA strand exchange between divergent DNA sequences.</a> <em>Front Microbiol</em> 10: 237.</li><li>Carrasco, B., Serrano, E., Sanchez, H., Wyman, C. and Alonso, J. C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26786319" target="_blank">Chromosomal transformation in <em>Bacillus subtilis</em> is a non-polar recombination reaction.</a> <em>Nucleic Acids Res</em> 44(6): 2754-2768.</li><li>Chen, I., Christie, P. J. and Dubnau, D. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16322448" target="_blank">The ins and outs of DNA transfer in bacteria.</a> <em>Science</em> 310(5753): 1456-1460.</li><li>Chen, I. and Dubnau, D. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15083159" target="_blank">DNA uptake during bacterial transformation.</a> <em>Nat Rev Microbiol</em> 2(3): 241-249.</li><li>Dubnau, D., Davidoff-Abelson, R., Scher, B. and Cirigliano, C. (1973) <a href="http://www.ncbi.nlm.nih.gov/pubmed/4349031" target="_blank">Fate of transforming deoxyribonucleic acid after uptake by competent <em>Bacillus subtilis</em>: phenotypic characterization of radiation-sensitive recombination-deficient mutants.</a> <em>J Bacteriol</em> 114(1): 273-286.</li><li>Johnston, C., Campo, N., Berge, M. J., Polard, P. and Claverys, J. P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24508048" target="_blank"><em>Streptococcus pneumoniae</em>, le transformiste.</a> <em>Trends Microbiol</em> 22(3): 113-119.</li><li>Kidane, D., Ayora, S., Sweasy, J. B., Graumann, P. L. and Alonso, J. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23046409" target="_blank">The cell pole: the site of cross talk between the DNA uptake and genetic recombination machinery.</a> <em>Crit Rev Biochem Mol Biol</em> 47(6): 531-555.</li><li>Nicholson, W. L., and Maughan, H. (2002). <a href="https://jb.asm.org/content/184/17/4936" target="_blank">The spectrum of spontaneous rifampin resistance mutations in the <em>rpoB</em> gene of <em>Bacillus subtilis</em> 168 spores differs from that of vegetative cells and resembles that of <em>Mycobacterium tuberculosis</em>.</a> <em>Journal of bacteriology</em>. 184(17): 4936-4940.</li><li>Takeuchi, N., Kaneko, K. and Koonin, E. V. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24347631" target="_blank">Horizontal gene transfer can rescue prokaryotes from Muller's ratchet: benefit of DNA from dead cells and population subdivision. </a><em>G3 (Bethesda)</em> 4(2): 325-339.</li><li>Thomas, C. M. and Nielsen, K. M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16138099" target="_blank">Mechanisms of, and barriers to, horizontal gene transfer between bacteria.</a> <em>Nat Rev Microbiol</em> 3(9): 711-721.</li><li>Yadav, T., Carrasco, B., Hejna, J., Suzuki, Y., Takeyasu, K. and Alonso, J. C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23779106" target="_blank"><em>Bacillus subtilis</em> DprA recruits RecA onto single-stranded DNA and mediates annealing of complementary strands coated by SsbB and SsbA.</a> <em>J Biol Chem</em> 288(31): 22437-22450.</li><li>Yadav, T., Carrasco, B., Myers, A. R., George, N. P., Keck, J. L. and Alonso, J. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22373918" target="_blank">Genetic recombination in <em>Bacillus subtilis</em>: a division of labor between two single-strand DNA-binding proteins.</a> <em>Nucleic Acids Res</em> 40(12): 5546-5559.<br/></li></ol>