<body>
            <h1>Assessment of Metacaspase Activity in Phytoplankton</h1>
            <p><strong>Authors:</strong> Dina Spungin, Ilana Berman-Frank</p>
            <h2>Abstract</h2>
            <p>Programmed cell death (PCD) is an irreversible, genetically-controlled form of cell suicide in which an endogenous biochemical pathway leads to morphological changes and ultimately, cellular demise. PCD is accompanied by <em>de-novo</em> protein synthesis of a family of proteases-"caspases" that are often used as a diagnostic marker of PCD. Although phytoplankton do not contain true caspases, caspase-like activity (hypothetical proteins with analogous activity) has been traditionally used as a diagnostic marker of PCD in marine phytoplankton. Increased caspase-like proteolytic activity was demonstrated when synthetic fluorogenic activity substrates specific for caspases (with an Asp at the P1 position) were applied upon PCD induction. Metacaspases, cysteine proteases, share structural properties with those of caspases, yet they are highly specific for Arg and Lys cleavage site at the P1 position implying that caspase specific substrates are not indicative of metacaspase catalytic activity. This method specifically tests direct metacaspase activity in phytoplankton by the cleavage of the fluorogenic metacaspase substrate Ac-VRPR-AMC. Metacaspase activity was tested by the addition of a metacaspase specific peptide that is conjugated to the fluorescent reporter molecule. The cleavage of the peptide by the metacaspase releases the fluorochrome that, when excited by light, emits fluorescence. The level of metacaspase enzymatic activity in the cell lysate is directly proportional to the fluorescence signal detected. The use of specific standards in this test enables the quantification of the fluorescence results. This assay directly allows monitoring the metacaspase cleavage products and thereby tracing evidence for programmed cell death. </p>
            <p><strong>Keywords:</strong> Metacaspase activity, Phytoplankton, Cyanobacteria, Programmed cell death, Cell extracts, Fluorescence</p>
            <h2>Background</h2><p>PCD is an irreversible, genetically-controlled form of cell suicide which is a fundamental feature of prokaryotic and eukaryotic microbial life essential for the regulation of cellular and tissue homeostasis in metazoans (Aravind <em>et al.</em>, 1999). In phytoplankton, PCD has been defined as a form of autocatalytic cell suicide in which an endogenous biochemical pathway leads to morphological changes and ultimately, cellular dissolution (Bidle and Falkowski, 2004). PCD has been demonstrated in diverse evolutionary lineages of prokaryotic and eukaryotic phytoplankton including cyanobacteria, coccolithophores, diatoms, and dinoflagellates (Vardi <em>et al.</em>, 1999; Segovia <em>et al.</em>, 2003; Berman-Frank <em>et al.</em>, 2004; Bidle and Falkowski, 2004; Bidle and Bender, 2008). In phytoplankton, environmental stresses, such as cell age, nutrient stress, excessive salt concentrations or oxidative stress and viral attacks may initiate PCD (Berman-Frank <em>et al.</em>, 2004; Bidle and Falkowski, 2004; Spungin <em>et al.</em>, 2019). This process is accompanied by <em>de novo</em> protein synthesis of a family of proteases-the caspases, (cysteine aspartic proteases) cleaving specifically at Asp residues at the P1 position. Caspases are often used as a diagnostic marker of PCD. Classic caspases are unique in metazoans and in several viruses (Minina <em>et al.</em>, 2017). Higher plants, unicellular protists, fungi, and bacteria lack true caspases but contain metacaspases (Uren <em>et al.</em>, 2000), cysteine-proteases that share structural properties, specifically a histidine-cysteine catalytic dyad in the predicted active site (Tsiatsiani <em>et al.</em>, 2011). Metacaspases are widespread among prokaryotic and eukaryotic phytoplankton genomes. It has been indicated that prokaryotic phytoplankton, as well as evolving eukaryotic lineages, not only express PCD that involves caspase-like activity, but also a core set of proteins that are orthologues of metazoan caspases (Bidle and Falkowski, 2004; Bidle, 2016; Koonin and Aravind 2002).<br/><br/>Although phytoplankton do not contain true caspases, upon PCD induction caspase proteolytic activity is demonstrated when synthetic fluorogenic activity substrates are applied with an Asp at the P1 position (Berman-Frank <em>et al.</em>, 2004 and 2007; Bidle and Bender, 2008; Thamatrakoln <em>et al.</em>, 2012; Bar-Zeev <em>et al.</em>, 2013). Unlike caspases, metacaspases lack Asp specificity and cleave their peptide substrates after Arg or Lys in the P1 position (Tsiatsiani <em>et al.</em>, 2011). This implies that caspase specific substrates are not indicative of metacaspase catalytic activity. Thus, it has been suggested to use substrates with Arg or Lys residues at the P1 position to detect metacaspase activities in cellular extracts (Tsiatsiani <em>et al.</em>, 2011; KlemenÄiÄ <em>et al.</em>, 2015). Recently developed substrates applied to investigate the activity of specific metacaspases (Tsiatsiani <em>et al.</em>, 2011; KlemenÄiÄ <em>et al.</em>, 2015) have enabled further examination into the roles of metacaspases in organisms possessing metacaspase genes. <br/><br/>The method we detail herewith specifically tests direct metacaspase activity in phytoplankton by examining the cleavage of the fluorogenic metacaspase substrate Ac-VRPR-AMC (Ac-Val-Arg-Pro-Arg-AMC). Metacaspase activity tests and method calibrations were performed specifically in the marine cyanobacteria <em>Trichodesmium</em> under conditions that induce PCD (<em>e.g.</em>, iron starvation and oxidative stress) and in samples collected from the South West Pacific Ocean (Spungin <em>et al.</em>, 2018; Spungin <em>et al.</em>, 2019). In this method, <em>Trichodesmium</em> cell lysate was tested for protease activity by the addition of a metacaspase specific peptide that is conjugated to the fluorescent reporter molecule, for example, 7-amino-4-methyl coumarin (AMC). The cleavage of the peptide by the metacaspase releases the fluorochrome that, when excited by light at 380 nm wavelength, emits fluorescence at 460 nm. Enzymatic activity in the cell lysate is directly proportional to the fluorescence signal detected with a fluorimeter or a fluorescent microplate reader.</p>
        </body>

<p class="pptt" id="biaoti28878">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips</li><li>Polycarbonate filters (<em>e.g.</em>, PC 25 mm 5 Âµm, Whatman 110613)</li><li>Microcentrifuge tubes (<em>e.g.</em>, Axygen, catalog number: 3110451)</li><li>Ice</li><li>96-well plate-black (<em>e.g.</em>, SPL, Cell Culture Plate, PS, 96-well, Black, catalog number: 30496)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375) </li><li>Sodium chloride (<em>e.g.</em>, Merck, catalog number: 105404) </li><li>Sucrose (<em>e.g.</em>, AMRESCO, catalog number: 0335)</li><li>CHAPS (3-(3-cholamidopropyl)-dimethylammonio-1-propanesulfonate) (Sigma-Aldrich, catalog number: C9426)</li><li>Dithiothreitol (DTT) (Bio-Rad, catalog number: 1610611)</li><li>Dimethyl Sulfoxide (DMSO) (Sigma-Aldrich, catalog number: 154938)</li><li>Fluorogenic metacaspase substrate, Ac-VRPR-AMC, Mw = 725.85 <img alt="" class="layerphoto" height="35" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819005840_0674.jpg" style="margin-top:10px;" width="27"/> (BACHEM, catalog number: 4048494), store at -20 Â°C</li><li>7-amino-4-methyl coumarin (AMC) standard (Sigma-Aldrich, catalog number: A9891), Mw = 175.18 <img alt="" class="layerphoto" height="35" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819005840_0674.jpg" style="margin-top:10px;" width="27"/>, store at -20 Â°C</li><li>Pierce<sup>TM</sup> BCA protein assay kit (Thermo, catalog number: 23225)</li><li>Liquid nitrogen </li><li>5x Lauber buffer (see Recipe 1)</li><li>1 M AMC standard stock (see Recipe 2)</li><li>3 M Fluorogenic metacaspase substrate, Ac-VRPR-AMC (see Recipe 3)</li><li>Working solution (Fluorogenic metacaspase aubstrate) (see Recipe 4)<br/></li></ol><p class="pptt" id="biaoti28879">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>Ice bucket </li><li>Forceps</li><li>-20 Â°C freezer</li><li>-80 Â°C freezer</li><li>Ultra-cell disruptor (<em>e.g.</em>, Sonic Dismembrator, Fisher Scientific, Waltham, MA)</li><li>Microcentrifuge (<em>e.g.</em>, Thermo Heraeus Pico 17 Microcentrifuge, Thermo Fisher, catalog number: 75002410)</li><li>Vortex (<em>e.g.</em>, Labnet, model: VX100 Vortex Mixer, catalog number: S-0100)</li><li>Microplate reader equipped with fluorescence detection capabilities, kinetic reads and shake options (<em>e.g.</em>, Synergy4 BioTek, Winooski, VT)<br/></li></ol><p class="pptt" id="biaoti28880">### Procedure</p><ol class="npwone"><li>Preparation of cell extracts (for cell collection and volumes refer to Notes 1 and 2)<br/><ol class="npwtwo"><li>Prepare 1x Lauber buffer (see Recipe 1).</li><li>Add 1000 Âµl 1x Lauber buffer to the cells (Note 3). </li><li>Place the tubes in a bucket of ice.</li><li>Sonicate each sample for 30 s (short pulses). Repeat this step 4 times (each sample 4 x 30 s) (Note 4). </li><li>Centrifuge at room temperature, maximum speed (14,000<em> x g</em>) for 5 min.</li><li>Transfer 500 Âµl of supernatant to a fresh tube containing 5 Âµl DTT (1 M) and place on ice for immediate assay or store at -80 Â°C for future use. The remaining sample in the tube can be stored at -80 Â°C for backup and for total protein concentration analysis.</li></ol><br/></li><li>Preparation of Standards (Note 5)<br/>Use Table 1 as a guide to prepare a set of standards by using two-fold dilutions.<ol class="npwtwo"><li>Prepare a stock of 1 M AMC (see Recipe 2). </li><li>Prepare 15 tubes.</li><li>To each tube add the diluent (1x Lauber buffer). </li><li>To tube 1 add 1 Âµl AMC standard stock (1 M) to a final concentration of 1,000 ÂµM.</li><li>Vortex the tube and remove 500 Âµl from tube 1 to tube 2 (2 fold dilution), thus in tube 2 the final concentration will be 500 ÂµM, vortex the tube and remove 500 Âµl from tube 2 to tube 3, to a final concentration of 250 ÂµM in tube 3. Repeat this to all tubes. Tube number 15 leave with 1x Lauber buffer only (0 ÂµM).</li></ol><br/><strong>Table 1. Preparation of diluted AMC standards</strong><br/><img alt="" class="layerphoto" height="415" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819002635_6549.jpg" width="600"/><br/><br/></li><li>Loading of the plate (Table 2)<ol class="npwtwo"><li>Additional controls that should be included in this assay are a) Sample blank (no substrate),  b) Substrate blank (no cell lysate, substrate blanks will ensure the substrate is not spontaneously degraded over time) and c) DMSO control (as it is used as a solvent). The total reaction volume must be kept constant and therefore 1x Lauber buffer can be used to replace the volume normally occupied by either the cell lysate or the substrate reagent.<br/></li><li>Prepare working solution (WS) of substrate (1 mM) sufficient for all samples, standards and substrate blanks (see Recipe 4).</li><li>Load the 96-well plate according to the content and amounts presented in Table 2 and table plan example presented in Figure 1. </li><li>Cover the plate with a lid and go to the plate reader (Note 6).<br/><br/><strong>Table 2. Description of the amounts and contents required for loading of the plate</strong><br/><img alt="" class="layerphoto" height="252" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819002731_8228.jpg" width="600"/><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819002828_1223.jpg" width="149"/><br/><strong>Figure 1. Example of 96-well plate samples, standards and control loading plan.</strong> Std- standard sample; SB- sample blank; S- sample; Sub blk- substrate blank; DMSO- Dimethyl Sulfoxide. <br/><br/></li><li>Plate reader<br/>Substrate cleavage (release of AMC) is performed here by continuously monitoring the release over 2-3 h. Alternatively, end point analysis can be performed by measuring AMC after a defined time (30-40 min), but it is advisable to first monitor continuously to ensure that the time chosen is within the linear phase of the reactions.<br/>Read the plate with the next parameters:<ol class="npwthree"><li>Temperature: room temperature.<br/></li><li>Kinetics: 2-3 h with a read-every 10 min.</li><li>Mix: all wells should be mixed (medium mix, 5 s) before each read.</li><li>Ex: 380 nm, Em: 460 nm.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Calculations<ol class="npwtwo"><li>Standard curve calculation:<br/>Prepare a graph of standard curve (Fluorescence unit [FU] against standard concentration) and calculate the standard curve equation. See Table 3, and Figure 2 for example.<br/><br/><strong>Table 3. Example of fluorescence units (FU) obtained for the different AMC standard concentrations.</strong> Only standard tubes 7-15 are read (Note 5).<br/><img alt="" class="layerphoto" height="444" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819002920_4365.jpg" width="600"/><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003012_7975.jpg" width="100"/><br/><strong>Figure 2. Example of AMC standard curve obtained from the fluorescence reads against AMC concentrations (ÂµM)</strong><br/><br/></li><li>Sample calculations:<br/><ol class="npwthree"><li>If background controls (sample blanks or substrate blanks) give a substantial reading, these values should be subtracted from the experimental results.Subtract the sample blanks read from the sample results reads.</li><li>Using the standard curve equation insert the sample fluorescence values obtained in the standard equation, to obtain AMC concentrations (Table 4).</li><li>Plot the data on a graph (AMC concentration against time) (Figure 3).</li><li>For each sample calculate the slope, obtained from the linear time points. This is an indicator for change in concentration over time (ÂµM AMC min<sup>-1</sup>).</li><li>Multiply the slope (ÂµM min<sup>-1</sup>) by the filtration volume and divide by the resuspension volume (1x Lauber buffer added) (Table 5).</li><li>Protein normalization: it is suggested to normalize each kinetic rate to its total protein amount. This will also allow comparing between the different samples. Protein concentration can be determined from the remaining sample (see Step A6). The protein content of the cell lysate can be estimated using a protein determination assay that is compatible with detergents present in the Lauber Buffer where no DTT is added (<em>e.g.</em>, Pierce<sup>TM</sup> BCA protein assay kit).</li><li>Final units for metacaspase activity rate (cleavage of substrate rate) are ÂµM AMC min<sup>-1</sup>. If results are normalized to mg L<sup>-1</sup> protein, final units are Âµmol AMC mg protein<sup>-1</sup> min<sup>-1</sup> (For unit equation refer to Note 7, unit equation 1). </li><li>Average the 3 results obtained from the 3 technical replicates (Table 5). </li></ol><br/><strong>Table 4. Example of FU obtained for sample 1 (3 technical sample replicates 1a, 1b and 1c) in 10 min</strong><strong> time intervals with a total reaction time of 110 min and values obtained after inserting the FU in the standard curve equation (ÂµM AMC).</strong><br/><img alt="" class="layerphoto" height="329" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003104_5798.jpg" width="600"/><br/><br/><strong>Table 5. Data calculation example, after the slope is obtained.</strong> The slope (ÂµM AMC min<sup>-1</sup>) is multiplied by the filtration volume (ml) and divided by resuspension volume (ml) for normalization. Normalization to biomass (<em>e.g.</em>, protein) is advised. Technical repeats are then averaged, to a final result of metacaspase activity, representing the rate of AMC release min<sup>-1</sup> mg protein<sup>-1</sup>.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003202_6767.jpg" width="1500"/><br/><br/><img alt="" class="layerphoto" height="105" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003242_9214.jpg" width="100"/><br/><strong>Figure 3. Example of curves obtained from sample 1, and sample 2, representing the change in AMC concentration (ÂµM) over time.</strong> The 3 curves represent 3 technical reads of each sample (1a, 1b and 1c or 2a, 2b and 2c). </li></ol></li></ol><p class="pptt" id="biaoti28881">### Notes</p><ol class="npwtwok"><li>Collection of cells by filtration: Filtration process is necessary only when no other way (<em>e.g.</em>, centrifuge) of cell isolation is suitable. For filtration process follow the next steps:<br/><ol class="npwthree"><li>Filter cells on a polycarbonate filter (For <em>Trichodesmium</em> 5-10 ÂµM pore size).<br/>Filtration volume depends on the source of the samples and concertation. For dense cultures of <em>Trichodesmium</em> (~1-5 Âµg ml<sup>-1</sup> total protein) 100-200 ml are filtered. For natural marine samples, where concentrations are low (~0.02 Âµg ml-1 total protein) 2,000-4,000 ml should be filtered. Filtration volume may vary between different cells; a specific test should be done to verify filtration volumes.</li><li>Insert the filter in a microcentrifuge tube.</li><li>Quick freeze the filter in liquid nitrogen. Use forceps and immerse the tube in the liquid nitrogen for ~15 s. </li><li>Shake the tube or gently vortex until cells are released from the filter, then take the filter out of the tube.</li><li>Proceed to the sonication step (Step A4).<br/>For further information regarding, sample collection of <em>Trichodesmium</em> during our experiments, refer to Spungin et al., 2016, 2018 and 2019.</li></ol></li><li>After freezing the sample in liquid nitrogen, samples can be stored in -80 Â°C until further analysis. For long term storage it is advisable to proceed to the next steps, of buffer addition and cell extraction and then store the samples at -80 Â°C. </li><li>It is advised to study different cell contractions to ensure that no saturation effects occur during the assay. The amount of suspension volume can be changed according to cell concentration, thus 1,000 Âµl is suggested, but if low cell concentrations are detected, reduce the suspension volume. </li><li>Take at least 10-15 s break between each sonication per sample as the sonication process heats the samples. Let the sample rest on ice between each sonication round. </li><li>As metacaspase activity rate may vary in different organisms or under different cell concentrations, the range of the standard curve should suit the range of the activity rate (the amount of AMC substrate released via metacaspase activity in the sample). Thus, it is important to specifically test the suitable standard range. For analyses we use standard in tubes 7-15 (concentration ranging between 0 to 15.6 ÂµM. However, a test should be done to ensure the appropriate range).</li><li>Reaction occurs instantaneously at room temperature so fluorescence should be read right after the loading of the plate.</li><li>Final units for metacaspase activity rate (cleavage of substrate rate) are ÂµM AMC min<sup>-1</sup>. If results are normalized to mg L<sup>-1</sup> protein, final units are Âµmol AMC mg protein<sup>-1</sup> min<sup>-1</sup> (Unit equation 1).<br/><br/><img alt="" class="layerphoto" height="50" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003343_8081.jpg" width="535"/><br/></li></ol><p class="pptt" id="biaoti28882">### Recipes</p><ol class="npwtwok"><li>1.5x Lauber buffer<br/>50 mM HEPES (5.96 g in 100 ml DDW)<br/>100 mM NaCl (2.92 g in 100 ml DDW)<br/>10% Sucrose (50 g in 100 ml DDW)<br/>0.1% CHAPS (0.5 g in 100 ml DDW)<br/>10 mM DTT (10 Î¼l of 1 M DTT in 1 ml Lauber buffer, will yield a final concentration of 10 mM DTT)- for metacaspase activity assay prepare 5x Lauber buffer with no DTT (DTT will be added during assay procedure, see Step A6) <br/>For preparing 1x Lauber buffer: Dilute 5x Lauber buffer 1:4 with DDW.</li><li>1 M AMC standard stock<br/>7-amino-4-methylcoumarin (AMC) standard (Sigma)<br/><br/><img alt="" class="layerphoto" height="35" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003416_2448.jpg"/><br/><br/>Prepare stock of 1 M:<br/>For 1 M AMC stock, add 0.175 mg to 1 ml DMSO<br/>Divide to aliquots and store at -20 Â°C</li><li>3 M Fluorogenic metacaspase substrate, Ac-VRPR-AMC (BACHEM)<br/>Amount in tube: 1 mg= 0.001 g<br/><br/><img alt="" class="layerphoto" height="35" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190819/20190819003447_1070.jpg"/><br/><br/>Prepare stock of 3 mM:<br/>For 3 mM substrate stock, add 460 Âµl DMSO to stock<br/>Following reconstitution, aliquot and freeze (-20 Â°C). Stock solutions are stable for up to 6 months at -20 Â°C</li><li>Working solution (Fluorogenic metacaspase aubstrate)<br/>For assay we use working solution of 1 mM, thus dilute the amount needed for the assay: 1:2 with 1x Lauber buffer<br/></li></ol><p class="pptt" id="biaoti28883">### Acknowledgments</p><p class="ppzz">This work was supported by the United States-Israel Binational Science Foundation (BSF) grant 2008048 to IBF and KDB, a collaborative grant from MOST Israel and the High Council for Science and Technology (HCST)-France to IBF, and GIF funding No. 1133/2011 to IBF. This protocol was adapted and modified from our previous studies (Spungin <em>et al.</em>, 2018 and 2019).</p><p class="pptt" id="biaoti28884">### Competing interests</p><p class="ppzz">The authors declare no competing financial interests.</p><p class="pptt" id="biaoti28885">### References</p><ol class="npwtwok"><li>Aravind, L., Dixit, V. M. and Koonin, E. V. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10098397" target="_blank">The domains of death: evolution of the apoptosis machinery.</a> <em>Trends Biochem Sci</em> 24(2): 47-53.</li><li>Bar-Zeev, E., Avishay, I., Bidle, K. D. and Berman-Frank, I. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23887173" target="_blank">Programmed cell death in the marine cyanobacterium <em>Trichodesmium</em> mediates carbon and nitrogen export.</a> <em>ISME J</em> 7(12): 2340-2348. </li><li>Berman-Frank, I., Bidle, K. D., Haramaty, L. and Falkowski, P. G. (2004). <a href="https://aslopubs.onlinelibrary.wiley.com/doi/abs/10.4319/lo.2004.49.4.0997" target="_blank">The demise of the marine cyanobacterium, <em>Trichodesmium</em> spp., via an autocatalyzed cell death pathway.</a> <em>Limnol Oceanogr</em> 49(4): 997-1005.</li><li>Berman-Frank, I., Rosenberg, G., Levitan, O., Haramaty, L. and Mari, X. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17504479" target="_blank">Coupling between autocatalytic cell death and transparent exopolymeric particle production in the marine cyanobacterium <em>Trichodesmium</em>.</a> <em>Environ Microbiol</em> 9(6): 1415-1422.</li><li>Bidle, K. D. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27404255" target="_blank">Programmed cell death in unicellular phytoplankton.</a> <em>Curr Biol</em> 26(13): R594-R607.</li><li>Bidle, K. D. and Bender, S. J. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18039944" target="_blank">Iron starvation and culture age activate metacaspases and programmed cell death in the marine diatom <em>Thalassiosira pseudonana</em>.</a> <em>Eukaryot Cell</em> 7(2): 223-236.</li><li>Bidle, K. D. and Falkowski, P. G. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15263899" target="_blank">Cell death in planktonic, photosynthetic microorganisms.</a> <em>Nat Rev Microbiol</em> 2(8): 643-655.</li><li>KlemenÄiÄ, M., Novinec, M. and Dolinar, M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26114948" target="_blank">Orthocaspases are proteolytically active prokaryotic caspase homologues: the case of <em>Microcystis aeruginosa</em>.</a> <em>Mol Microbiol</em> 98(1): 142-150.</li><li>Koonin, E. V. and Aravind, L. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11965492" target="_blank">Origin and evolution of eukaryotic apoptosis: the bacterial connection.</a> <em>Cell Death Differ</em> 9(4): 394-404.</li><li>Minina, E. A., Coll, N. S., Tuominen, H. and Bozhkov, P. V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28234356" target="_blank">Metacaspases versus caspases in development and cell fate regulation.</a> <em>Cell Death Differ</em> 24(8): 1314-1325.</li><li>Segovia, M., Haramaty, L., Berges, J. A. and Falkowski, P. G. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12746516" target="_blank">Cell death in the unicellular chlorophyte <em>Dunaliella tertiolecta</em>. A hypothesis on the evolution of apoptosis in higher plants and metazoans.</a> <em>Plant Physiol</em> 132(1): 99-105.</li><li>Spungin, D., Belkin, N., Foster, R.A., Stenegren, M., Caputo, A., Pujo-Pay, M., Leblond, N., Dupouy, C., Bonnet, S. and Berman-Frank, I. (2018). <a href="https://www.biogeosciences.net/15/3893/2018/" target="_blank">Programmed cell death in diazotrophs and the fate of organic matter in the western tropical South Pacific Ocean during the OUTPACE cruise.</a> <em>Biogeosciences</em> 15: 3893-3908.</li><li>Spungin, D., Bidle, K. D. and Berman-Frank, I. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30585394" target="_blank">Metacaspase involvement in programmed cell death of the marine cyanobacterium <em>Trichodesmium</em>.</a> <em>Environ Microbiol</em> 21(2): 667-681.</li><li>Spungin, D., Pfreundt, U., Berthelot, H., Bonnet, S., AlRoumi, D., Natale, F., Hess, W. R., Bidle K. D. and Berman-Frank, I. (2016). <a href="https://www.biogeosciences.net/13/4187/2016/bg-13-4187-2016.html" target="_blank">Mechanisms of <em>Trichodesmium</em> demise within the New Caledonian lagoon during the VAHINE mesocosm experiment.</a> <em>Biogeosciences</em> 13: 4187-4203.</li><li>Thamatrakoln, K., Korenovska, O., Niheu, A. K. and Bidle, K. D. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21453404" target="_blank">Whole-genome expression analysis reveals a role for death-related genes in stress acclimation of the diatom <em>Thalassiosira pseudonana</em>.</a> <em>Environ Microbiol</em> 14(1): 67-81. </li><li>Tsiatsiani, L., Van Breusegem, F., Gallois, P., Zavialov, A., Lam, E. and Bozhkov, P. V. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21597462" target="_blank">Metacaspases.</a> <em>Cell Death Differ</em> 18(8): 1279-1288.</li><li>Uren, A. G., O'Rourke, K., Aravind, L. A., Pisabarro, M. T., Seshagiri, S., Koonin, E. V. and Dixit, V. M. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11090634" target="_blank">Identification of paracaspases and metacaspases: two ancient families of caspase-like proteins, one of which plays a key role in MALT lymphoma.</a> <em>Mol Cell</em> 6(4): 961-967. </li><li>Vardi, A., Berman-Frank, I., Rozenberg, T., Hadas, O., Kaplan, A. and Levine, A. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10508616" target="_blank">Programmed cell death of the dinoflagellate <em>Peridinium gatunense</em> is mediated by CO2 limitation and oxidative stress.</a> <em>Curr Biol</em> 9(18): 1061-1064.<br/></li></ol>