<body>
            <h1>Genotyping of the OATP1B1 c. 521 T>C Polymorphism from the Formalin-Fixed Paraffin-Embedded (FFPE) Tissue Specimens: An Optimized Protocol</h1>
            <p><strong>Authors:</strong> Alexandra R Crowe, Jonathan  Miller, Wei  Yue</p>
            <h2>Abstract</h2>
            <p>Organic anion transporting polypeptide (OATP) 1B1 is a liver-specific transport protein that plays an important role in hepatic drug disposition. It transports many drugs from the blood into the liver, including lipid-lowering statins. The c. 521 T&gt;C polymorphism of OATP1B1 has reduced transport activity and is associated with statin-induced myopathy. Formalin-fixed paraffin-embedded (FFPE) liver tissues can be an enriched source for genotyping of this clinically significant OATP1B1 polymorphism in retrospective studies. The successfulness of genotyping using Sanger-sequencing of a PCR product from FFPE tissue relies on a successful PCR amplification using genomic DNA extracted from the FFPE tissues. Such PCR amplification is often limited by the quality of DNA extracted from the FFPE tissue. An optimized method for high-quality DNA extraction and efficient PCR amplification is highly needed in order to genotype polymorphisms such as the c. 521 T&gt;C polymorphism using FFPE tissues. The current study established an optimized and reproducible method for a Sanger-sequencing-based genotyping method using FFPE human liver tissues that is applicable to even small FFPE tissues such as needle-core biopsy specimens.</p>
            <p><strong>Keywords:</strong> Membrane transport proteins, Organic anion transporting polypeptide (OATP), Hepatic drug disposition, Genotyping, Formalin-fixed paraffin-embedded tissue</p>
            <h2>Background</h2><p>Organic anion transporting polypeptide (OATP) 1B1 is a liver-specific transport protein that plays an important role in hepatic drug disposition (Shitara <em>et al.</em>, 2013; Khurana <em>et al.</em>, 2014; Lee <em>et al.</em>, 2015). It transports many drugs from blood into the liver including lipid-lowering statins, anti-cancer and anti-hepatitis C virus drugs (Hsiang <em>et al.</em>, 1999; Hirano <em>et al.</em>, 2004; Simonson <em>et al.</em>, 2004; Nozawa <em>et al.</em>, 2005; Feng <em>et al.</em>, 2009; Oostendorp <em>et al.</em>, 2009; Hartkoorn <em>et al.</em>, 2010). The c. 521 T&gt;C polymorphism of OATP1B1 has reduced transport activity and is associated with altered toxicity of drugs that are OATP1B1 substrates such as statins. Genotyping the c. 521 T&gt;C of OATP1B1 using archived liver FFPE tissue allows retrospective correlation of the transport activity of OATP1B1 due to genetic variation with toxicity and efficacy of drugs that are OATP1B1 substrates, indicating its significant clinical relevance. The Sanger-sequencing-based genotyping method using FFPE tissues is often limited by the lack of an efficient PCR product using DNA template extracted from the FFPE tissues. A recent publication reported that the DNA template from the FFPE tissue itself can inhibit the PCR efficiency (Dietrich <em>et al.</em>, 2013). Based on this finding, we optimized the DNA template amount used for PCR and suggest an upper limit of DNA that should not be exceeded for PCR in FFPE tissue. We also identified an efficient kit for DNA purification for the FFPE tissue. We report an optimized, reproducible protocol for PCR using FFPE DNA as template.</p>
        </body>

<p class="pptt" id="biaoti28898">## Materials and Reagents</p><ol class="npwtwok"><li>1.7 ml Eppendorf tubes (VWR, catalog number: 87003-294)</li><li>0.65 ml Eppendorf tubes (VWR, catalog number: 87003-290)</li><li>20 Î¼l ART<sup>TM</sup> Barrier Low-Retention Pipette tips (Thermo Fisher Scientific, catalog number: 02-707-432)</li><li>200 Î¼l ART<sup>TM</sup> Barrier Low-Retention Pipette tips (Thermo Fisher Scientific, catalog number: 21-402-486)</li><li>Parafilm (VWR, catalog number: 52858-000)</li><li>Micro-Tube Cap Locks (Research Products International, catalog number: 145063)</li><li>Formalin-Fixed Paraffin-Embedded (FFPE) tissue<br/>The FFPE tissue blocks from surgical resection (1.5-2.5 x 1.5-2.0 x 0.2-0.3 cm) and needle-core biopsy specimens (1.4-2.0 x 0.1-0.2 x 0.2-0.2 cm) were sliced into paraffin rolls (~2 Î¼m) at the University of Oklahoma Health Sciences Center Stephenson Cancer Center Tissue Pathology Shared Resource facilities. Each Eppendorf tube contains 2-3 tissue rolls. Paraffin rolls should be stored at -20 Â°C. </li><li>Formalin-Fixed Paraffin-Embedded (FFPE) DNA Isolation Kit (Zymo Research, catalog number: D3067). <br/><em>Note: Store all kit components at room temperature except proteinase K and storage buffer after mixing (-20 Â°C) and resuspended RNase A (4 Â°C).</em> </li><li>Zymoclean Gel DNA Recovery Kit (Zymo Research, catalog number: D4007),store kit at room temperature</li><li>QIAamp DNA FFPE Tissue Kit (QIAGEN, catalog number: 56404) (Tissue from liver donors 1-3 were used with the QIAmp kit to isolate DNA) </li><li>100% Ethanol (VWR, catalog number: 71001-630), store at room temperature</li><li>Isopropanol (IBI Scientific, catalog number: IB15735), store at room temperature</li><li>Nuclease-free water (Thermo Fisher Scientific, catalog number: NC075301)<br/><em>Note: Water was aliquoted (1 ml per Eppendorf tube) in cell culture hood and stored at -20 Â°C. A new aliquot was used each time to avoid cross contamination. </em> </li><li>Phusion High-Fidelity DNA Polymerase (100 units) (Thermo Fisher Scientific, catalog number: F530S), store at -20 Â°C</li><li>5x Phusion High-Fidelity Buffer (Thermo Fisher Scientific, catalog number: F-518), store at -20 Â°C</li><li>10 mM dNTP Mix (Invitrogen, catalog number: 18427-013), store at -20 Â°C</li><li>Agarose (Thermo Fisher Scientific, catalog number: BP160-100), store at room temperature</li><li>Ethidium Bromide (Thermo Fisher, catalog number: 15585011), store at 4 Â°C</li><li>Tris Base (Thermo Fisher, catalog number: BP152-5), store at room temperature</li><li>Boric Acid (Sigma, catalog number: B0394), store at room temperature</li><li>100 base pair (bp) DNA ladder (Promega, catalog number: G210A), store at -20 Â°C</li><li>6x Blue/Orange loading dye (Promega, catalog number: G190A), store at -20 Â°C</li><li>Forward and Reverse primers (Sigma). Primers were synthesized at 0.05 Î¼M synthesis scale and pre-diluted to 100 Î¼M in water with desalt purification, store at -20 Â°C<br/>Forward Primer: AAATTACCCAGTCTCAGGTATG<br/>Reverse Primer: TGTCCTTCTTTAGCGAAATC</li><li>Ethylenediaminetetra-acetic acid (EDTA) (Sigma, catalog number: EDS-100g), store at room temperature</li><li>10x Tris-Borate EDTA (TBE) Buffer (see Recipes)</li><li>0.5x Tris-Borate EDTA (TBE) Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28899">### Equipment</p><ol class="npwtwok"><li>Pipettes: Research Plus<sup>Â®</sup> 4-pack (0.1-2.5 Î¼l, 2-20 Î¼l, 20-200 Î¼l, 100-1,000 Î¼l) (Eppendorf, manufacture ID: 2231300004)</li><li>Heat block (55-94 Â°C) (VWR, catalog number: 75838-318)</li><li>Centrifuge (Eppendorf, model: 5424R)</li><li>Vortex (Vortex Genie 2, 110V, VWR, catalog number: 100370-856)</li><li>Thermocycler (Techne, Cole-Parmer Scientific Experts, TC-312)</li><li>NanoDrop Spectrophotometer (NanoDrop Technologies, ND-1000 UV/Vis)</li><li>3730xl DNA Analyzer for sequencing (Applied Biosystems, 3730xl)</li><li>BioRad Molecular XRS Imager (BioRad, Hercules, CA)<br/></li></ol><p class="pptt" id="biaoti28900">### Software</p><ol class="npwtwok"><li>Finch TV Software (Geospiz, Inc., <a href="https://digitalworldbiology.com/FinchTV" target="_blank">https://digitalworldbiology.com/FinchTV</a>)</li><li>Image Lab software (BioRad, Hercules, CA)<br/></li></ol><p class="pptt" id="biaoti28901">### Procedure</p><ol class="npwone"><li>Extraction of DNA from formalin-fixed, paraffin-embedded (FFPE) liver tissue using FFPE DNA Isolation Kit #D3067 (Zymo Research, Irvine, CA)<ol class="npwtwo"><li>Preparation of buffers<br/><ol class="npwthree"><li>Prepare buffers according to the manufacturerâs instructions (<a href="https://files.zymoresearch.com/protocols/_d3067_quick-dna_ffpe_miniprep.pdf" target="_blank">Quick-DNA<sup>TM</sup> FFPE Kit</a>).</li><li>Add 260 Î¼l of Proteinase K Storage Buffer to Proteinase K included in the kit to reconstitute lyophilized Proteinase K to a final concentration of 20 mg/ml. Vortex the solution and store at -20 Â°C.</li><li>Add 48 ml of 100% ethanol to 12 ml of the Genomic DNA Wash 2 concentrate.</li><li>Resuspend lyophilized RNase A in 300 Î¼l ddH<sub>2</sub>O to a final concentration of 6.66 mg/ml and store at 4 Â°C.</li></ol></li><li>Deparaffinization of tissue<br/><ol class="npwthree"><li>Warm heat block up to 55 Â°C prior to starting experiment.</li><li>Add 400 Î¼l of Deparaffinization solution to each Eppendorf tube containing tissue. Tubes are tapped multiple times to make sure tissue was completely in solution.</li><li>Incubate tissue with Deparaffinization solution for 1 min at 55 Â°C. Vortex tube briefly after the 1 min incubation. </li><li>Centrifuge down Eppendorf tubes in order to separate tissue sample from the Deparaffinization solution at 15,000 <em>x g</em> (1 min, 23 Â°C). Carefully, pipette out the solution, making sure to not remove any tissue.</li></ol></li><li>Digestion of tissue<br/><ol class="npwthree"><li>Prepare the master mix. The final volume is 100 Î¼l (Table 1). A bulk stock of the master mix can be made up for multiple tissues.<br/><br/><strong>Table 1. Master Mix Reagents</strong><strong></strong><br/><img alt="" class="layerphoto" height="113" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190817/20190817065453_2057.png" width="450"/><br/><br/></li><li>Add 100 Î¼l of Master Mix to each Eppendorf tube containing deparaffinized tissue and incubate for at least 3 h or overnight at 55 Â°C for tissue digestion. Incubation time of deparaffinized tissue in the Master Mix is subjective to the size of the tissue. For larger tissues, it is best to leave in Master Mix overnight to ensure complete digestion of tissue.</li><li>At the end of incubation, centrifuge the Eppendorf tube containing samples briefly (~1 min) to collect any condensed vapor from the incubation on the lid.</li><li>Set the heat block temperature to 94 Â°C and add water in each hole of the heat block in order to equilibrate the temperature of the heat block. </li><li>Add micro-tube cap locks to each Eppendorf tube to prevent accidental expansion of tube and incubate samples at 94 Â°C for 20 min.</li><li>Allow samples to cool down to room temperature (~5 min) and briefly centrifuge down.</li><li>Add 5 Î¼l of reconstituted RNase directly to the solution (RNase final concentration of 0.32 mg/ml). Pipette tips are changed between samples to prevent any DNA contamination. <br/></li><li>Vortex sample tubes briefly (~10 s) to make sure the solution is well mixed and incubate at room temperature for 5 min.</li></ol></li><li>Purification of DNA<br/><ol class="npwthree"><li>Add 350 Î¼l of Genomic Lysis Buffer (350 Î¼l) to each tube and vortex the tubes briefly (~10 s).</li><li>After vortexing, spin down the sample tubes for 1 min.</li><li>Add 135 Î¼l of isopropanol to each tube. Mix the DNA solution and isopropanol thoroughly by inverting the tube multiple times.</li><li>Centrifuge down the isopropanol and DNA mixture at 12,000 <em>x g</em> (1 min) to spin down any insoluble debris in the Eppendorf tube. <br/></li><li>Transfer the supernatant containing the DNA to a Zymo-Spin Column attached to a collection tube and centrifuge down the column and collection tube at 10,000 <em>x g</em> (1 min).<br/></li><li>Transfer the spin column to a new collection tube and add 400 Î¼l Genomic DNA wash 1 buffer to the column and centrifuge down the column and collection tube at 10,000 <em>x g</em> (1 min). <br/></li><li>Transfer the spin column to a new collection tube. Add 700 Î¼l of Genomic DNA wash 2 buffer to the spin column and centrifuge down the spin column and collection tube at 12,000 <em>x g</em> (1 min). <br/></li><li>Decant the flow-through from Step A4g and add 200 Î¼l of Genomic DNA wash 2 buffer to the spin column and centrifuge down the spin column and collection tube at 12,000 <em>x g</em> (1 min). Transfer the spin column containing the DNA to a new Eppendorf tube after centrifugation. </li><li>In order to elute DNA from column, add 50 Î¼l of DNA elution buffer to the spin column and incubate at room temperature for 5 min.</li><li>Centrifuge down the spin column containing the DNA elution buffer for 30 s at 21,130 <em>x g</em>.<br/></li><li>Measure the DNA concentration by NanoDrop (ND-1000). Store the DNA at -20 Â°C in a screw-capped tube or in tubes sealed with parafilm. The expected yield of genomic DNA should be around 15 and 2 Î¼g from tissue blocks and biopsy tissues, respectively (Figure 1).<br/><br/><img alt="" class="layerphoto" height="96" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190817/20190817063311_4924.jpg" width="300"/><br/><strong>Figure 1. Comparing the yield and purity of DNA using tissue blocks and biopsy samples. </strong>A. Yield of DNA (Î¼g) measured in liver tissue blocks (n = 44) and liver biopsy samples (n = 38). Data represent mean Â± standard error of mean. The concentration of DNA was measured by a NanoDrop. * indicates a <em>P</em>-value &lt; 0.05 by Studentâs <em>t</em>-test. B. Comparison of 260/280 ratios between liver tissue blocks (n = 44) and liver biopsy samples (n = 38). Data represent mean Â± standard error of mean. The 260/280 ratios were determined at the same time the DNA concentration was measured on the NanoDrop.<br/><br/></li></ol></li></ol></li><li>PCR using DNA from FFPE tissues<ol class="npwtwo" start="1"><li>Amplify DNA using PCR <br/><ol class="npwthree"><li>Dilute the DNA extracted from the FFPE tissue to a concentration of 50 ng/Î¼l in nuclease-free water.</li><li>A 20 Î¼l volume will be used for each PCR reaction. Make up a fresh bulk PCR stock of the forward and reverse primers, Phusion Buffer, Phusion DNA polymerase, dNTPs and nuclease-free water. DNA polymerase should be added last to this bulk PCR mix. To a 0.65 ml empty Eppendorf tube, add 19 Î¼l of the bulk PCR mix to each tube. Add 1 Î¼l of the 50 ng/Î¼l (50 ng total) template DNA to the Eppendorf tube (Table 2). <br/><br/><strong>Table 2. Consists of one PCR reaction</strong><br/><img alt="" class="layerphoto" height="253" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190817/20190817065518_7197.png" width="800"/><br/><br/></li><li>PCR is carried out in a thermocycler using the conditions below:<br/><ol class="npwfour"><li>Initial denaturation: 94 Â°C, 5 min</li><li>40 cycles: <br/>Denaturation: 94 Â°C, 30 s<br/>Annealing: 54 Â°C, 2 min<br/>Extension: 72 Â°C, 1 min<br/>Final extension: 72 Â°C, 10 min</li></ol></li><li>Store the PCR product at -20 Â°C or continue on to the next step.<br/></li><li>Cast a 2% agarose gel containing ethidium bromide in 0.5x Tris-Borate-EDTA (TBE) buffer. After casting the gel, allow the gel to sit at room temperature for at least 60 min before using for sufficient polymerization. <br/></li><li>Add 4 Î¼l of the 6x blue/orange loading dye to the 20 Î¼l PCR reaction (final concentration 1x). Visualize the PCR products on a 2% agarose gel in 0.5x Tris-Borate-EDTA (TBE) buffer stained with ethidium bromide. <br/></li><li>Excise the DNA band of interest (<em>i.e.</em>, 339 bp for our experiment) for each FFPE tissue sample (Figure 2) with a sterile razor blade and place the band in an empty Eppendorf tube. Make sure to change blades between each PCR to avoid cross contamination of samples.<br/><br/><img alt="" class="layerphoto" height="273" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190817/20190817063435_2062.jpg" width="300"/><br/><strong>Figure 2. Optimizing template DNA amount for PCR.</strong> A range of DNA template concentrations (10-200 ng) from two different surgical resection liver tissues were used in PCR. PCR products and the 100 base pair (bp) DNA ladder were run on a 2% agarose gel containing ethidium bromide and visualized with the BioRad Molecular XRS Imager. The white box indicates our 339 bp DNA band of interest.<br/><br/></li></ol></li><li>Recovery of PCR product using Zymoclean Gel DNA Recovery Kit #D4007 (Zymo Research, Irvine, CA)<br/><ol class="npwthree"><li>Add three volumes (350 Î¼l) of ADB buffer to the Eppendorf tube containing the DNA agarose band and incubate at 55 Â°C for 20-30 min to completely dissolve the agarose.</li><li>Transfer the dissolved agarose DNA to a Zymo-Spin Column attached to a collection tube and centrifuge for 1 min at 13,000 <em>x g</em> at 23 Â°C.</li><li>Discard the flow-through from the collection tube and wash the spin column two times with DNA wash buffer (200 Î¼l) (95% ethanol). Centrifuge at 13,000 <em>x g</em> for 30 s at 23 Â°C for each wash. The flow-through in the collection tube can be discarded after each wash. <br/></li><li>To elute purified DNA, add 6 Î¼l of DNA elution buffer directly to the column with an Eppendorf tube attached and centrifuge for 1 min at 13,000 <em>x g</em> at 23 Â°C. The flow-through contains the purified DNA in the Eppendorf tube attached to column. DNA can be stored at -20 Â°C.</li></ol></li><li>Sequence purified DNA product<br/><ol class="npwthree"><li>To measure the concentration of the purified PCR product, use a NanoDrop (Figure 1).</li><li>Add 1 Î¼l of 2.3 pmol/Î¼l reverse primer to 40 ng of PCR product in a 0.65 ml Eppendorf tube. Add additional nuclease-free water to make the final volume 4 Î¼l. Wrap PCR samples in parafilm. These conditions were used by the OUHSC Laboratory for Molecular Biology and Cytometry Research DNA sequencing core.</li><li>Use Finch TV software (Geospiza, Inc.) to view the sequencing results. An example of sequencing results can be seen in Figure 3 for c.521 T&gt;C single nucleotide polymorphism.<br/><br/><img alt="" class="layerphoto" height="296" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190817/20190817063452_4260.jpg" width="300"/><br/><strong>Figure 3. Sequencing for OATP1B1 c. 521 T&gt;C polymorphism.</strong> The 339 bp amplification was subjected to Sanger-sequencing. Shaded area indicates the position of the c. 521 T&gt;C. Representative sequencing results for C. 521 TT (A), TC (B) and CC (C) are shown. <br/></li></ol></li></ol></li></ol><p class="pptt" id="biaoti28902">### Data analysis</p><p class="ppzz">Studentâs <em>t</em>-test was used to determine statistical significance in Figures 1 and 4B. A two-sided <em>P</em>-value of &lt; 0.05 denoted statistical significance.<br/><br/><img alt="" class="layerphoto" height="175" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190817/20190817063509_9281.jpg" width="300"/><br/><strong>Figure 4. Comparison of DNA extracted using two different kits.</strong> A. Three different FFPE liver tissues with similar sizes were extracted by the QIAamp DNA FFPE Tissue Kit (Liver 1-3) or FFPE DNA Isolation Kit (Zymo Research) (Liver 27-29). DNA concentration was measured by NanoDrop. DNA (100 ng) from each liver tissue was run on a 2% agarose gel containing ethidium bromide at 150V. The agarose gel was imaged using the BioRad Molecular XRS Imager (BioRad, Hercules, CA). B. Densitometry of DNA â¥ 400 bp was conducted using Image Lab software (BioRad, Hercules, CA) and expressed as fold change vs. the Zymo Kit. *, <em>p</em> &lt; 0.05 by Studentâs <em>t</em>-test. Data represent mean Â± standard deviation.</p><p class="pptt" id="biaoti28903">### Notes</p><ol class="npwtwok" start="1"><li>From a recent publication, the DNA extracted from the FFPE tissue itself can inhibit the real-time PCR efficiency (Dietrich <em>et al.</em>, 2013). The amount of template DNA from FFPE used in PCR is critical for the successfulness of PCR. "The more the better" is not better due to possible inhibition of the PCR reaction by excess DNA. We optimized the amount of the extracted FFPE DNA used as template in PCR. In a 20 Î¼l reaction volume containing dNTPs (0.5 mM each), the primer pair (0.5 Î¼M each), 1x Phusion buffer, and 2 units of Phusion DNA polymerase (Thermo Fisher Scientific, Carlsbad, CA), PCR efficiency was compared among reactions containing different amounts of template DNA of 10, 50, 100 or 200 ng. As shown in Figure 2, for the liver FFPE tissues (#4 and #5), 50 and 100 ng template DNA per 20 Î¼l reaction yielded a much higher amount of amplification of the expected 339 bp band compared to 10 ng template DNA. Our findings suggest that for successful PCR using FFPE tissues and current Phusion DNA polymerase, the template DNA should not exceed 100 ng in a 20 Î¼l reaction volume. It is important to use an optimized amount of template DNA from FFPE tissue to ensure the successfulness of an efficient PCR, which is a prerequisite for sequencing. </li><li>DNA extraction efficiency was compared between FFPE tissues from surgical resection tissues (1.5-2.5 x 1.5-2.0 x 0.2-0.3 cm) and needle-core biopsy specimens (1.4-2.0 x 0.1-0.2 x 0.1-0.2 cm). As the former has a greater size than the latter, as expected, the DNA yields are significantly higher from resection tissue than from the biopsies (Figure 1A). The OD<sub>260/280</sub> values for DNA from the two types of tissue were comparable, and were both ranging 1.6-1.9, suggesting there is high purity of DNA (Desjardins and Conklin, 2010) (Figure 1B). The ratio of the OD values of 260 and 280 are used to assess the purity of DNA and RNA, where a ratio of ~1.8 and ~2 is considered to be pure DNA and RNA, respectively. Similar as shown in Figure 2, using DNA (50 ng) from biopsy FFPE tissues in a 20 Î¼l volume also yielded successful amplification of the 339 bp PCR product (data not shown). Sequencing of the expected 339 bp PCR products from different tissues was able to distinguish between all three genotypes, c. 521 TT, TC and CC (Figure 3 and also as shown in the original publication [Crowe <em>et al.</em>, 2019]). The FFPE tissues from both surgical resection and biopsy specimens are all suitable for genotyping using the current method. This method has a high success rate. DNA was extracted, amplified by PCR, sequenced and genotyped for 100% of the FFPE tissues (n = 47 for tissue block, n = 24 for biopsy specimens) using this optimized method.</li><li>There are multiple FFPE genomic DNA extraction kits on the market. We have tried another kit from Qiagen (QIAamp DNA FFPE Tissue Kit), as shown in Figures 4A and 4B. When comparing FFPE liver tissues of similar size, apparently, there was less DNA â¥ 400 bp extracted using the Qiagen kit compared to the Zymo Kit. PCR amplification from DNA extracted from the Qiagen kit was unsuccessful using 34 ng template DNA in a 20 Î¼l reaction, similar as described above. We believe the Zymo FFPE DNA Isolation Kit (Zymo Research, #D3067) is an optimal reagent for our protocol. </li><li>The current protocol reports that the Zymo DNA extraction kit (D3067) is suitable for extracting DNA from the FFPE tissues. Though this protocol was only used with FFPE liver tissue, we anticipate this protocol to be suitable for other FFPE tissues and is not liver-specific. Also, this method optimized the amount of template DNA from FFPE tissue suitable for PCR. This protocol is reproducible and requires a small amount of tissue, even from biopsy specimens, to yield sufficient PCR products for Sanger-sequencing-based genotyping of OATP1B1 c. 521 T&gt;C. This method has also been successfully used for genotyping of another polymorphism of OATP1B1 c. 388 A&gt;G (Crowe<em> et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti28904">### Recipes</p><ol class="npwtwok" start="1"><li>10x Tris-Borate EDTA (TBE) Buffer<br/><ol class="npwthree"><li>Dissolve 108 g of Tris-Base and 55 g of boric acid in 900 ml deionized water</li><li>Add 9.3 g of 0.5 M EDTA to the solution and adjust the volume to 1 L with deionized water</li><li>Buffer can be stored at room temperature</li></ol></li><li>0.5x Tris-Borate EDTA (TBE) Buffer<br/>100 ml of 10x TBE buffer and 1,900 ml of deionized water<br/></li></ol><p class="pptt" id="biaoti28905">### Acknowledgments</p><p class="ppzz">This research was supported by NIH R01 GM094268 [W. Y] and American Foundation of Pharmaceutical Education pre-doctoral Award [A. C.]. Research reported in this publication was supported in part by the National Cancer Institute Cancer Center Support Grant P30CA225520 awarded to the University of Oklahoma Stephenson Cancer Center and used the Molecular Biology Shared Resource. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. Alexandra Crowe is an American Foundation of Pharmaceutical Education pre-doctoral Fellow. We would like to thank Dr. Allison Gillaspy for designing the PCR primers. We thank the Laboratory for Molecular Biology and Cytometry Research at OUHSC for the use of the core facility which provided the DNA sequencing service. We thank Drs. Wei Zheng, Kar-Ming Fung, Feng Yin and Erin Rubin for providing the FFPE tissues for this research.<br/>  The initial publication where this method is published is Crowe <em>et al.</em>, 2019.</p><p class="pptt" id="biaoti28906">### Competing interests</p><p class="ppzz">No competing financial interests for this study.</p><p class="pptt" id="biaoti28907">### Ethics</p><p class="ppzz">Seventy-nine formalin-fixed, paraffin-embedded (FFPE) archived human liver (42 from surgical resection and 37 from liver biopsy) and normal kidney tissue blocks were obtained from OUHSC Stephenson Cancer Center Biospecimen Acquisition Core and Bank from the Department of Pathology at the University of Oklahoma Health Sciences Center. Use of human tissues was approved by the Institutional Review Board at the University of Oklahoma Health Sciences Center (IRB number: 6347).</p><p class="pptt" id="biaoti28908">### References</p><ol class="npwtwok" start="1"><li>Crowe, A., Zheng, W., Miller, J., Pahwa, S., Alam, K., Fung, K. M., Rubin, E., Yin, F., Ding, K. and Yue, W. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31093828" target="_blank">Characterization of Plasma Membrane Localization and Phosphorylation Status of Organic Anion Transporting Polypeptide (OATP) 1B1 c.521 T&gt;C Nonsynonymous Single-Nucleotide Polymorphism.</a> <em>Pharm Res</em> 36(7): 101.</li><li>Dietrich, D., Uhl, B., Sailer, V., Holmes, E. E., Jung, M., Meller, S. and Kristiansen, G. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24155973" se_prerender_url="complete" target="_blank">Improved PCR performance using template DNA from formalin-fixed and paraffin-embedded tissues by overcoming PCR inhibition.</a> <em>PLoS One</em> 8(10): e77771.</li><li>Desjardins, P. and Conklin, D. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21189466" target="_blank">NanoDrop microvolume quantitation of nucleic acids.</a> <em>J Vis Exp </em>(45): 2565.</li><li>Hartkoorn, R. C., Kwan, W. S., Shallcross, V., Chaikan, A., Liptrott, N., Egan, D., Sora, E. S., James, C. E., Gibbons, S., Bray, P. G., Back, D. J., Khoo, S. H. and Owen, A. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20051929" se_prerender_url="complete" target="_blank">HIV protease inhibitors are substrates for OATP1A2, OATP1B1 and OATP1B3 and lopinavir plasma concentrations are influenced by SLCO1B1 polymorphisms.</a> <em>Pharmacogenet Genomics</em> 20(2): 112-120.</li><li>Feng, B., Xu, J. J., Bi, Y. A., Mireles, R., Davidson, R., Duignan, D. B., Campbell, S., Kostrubsky, V. E., Dunn, M. C., Smith, A. R. and Wang, H. F. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19223659" se_prerender_url="complete" target="_blank">Role of hepatic transporters in the disposition and hepatotoxicity of a HER2 tyrosine kinase inhibitor CP-724,714.</a> <em>Toxicol Sci</em> 108(2): 492-500.</li><li>Hirano, M., Maeda, K., Shitara, Y. and Sugiyama, Y. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15159445" target="_blank">Contribution of OATP2 (OATP1B1) and OATP8 (OATP1B3) to the hepatic uptake of pitavastatin in humans.</a> <em>J Pharmacol Exp Ther</em> 311(1): 139-146.</li><li>Hsiang, B., Zhu, Y., Wang, Z., Wu, Y., Sasseville, V., Yang, W. P. and Kirchgessner, T. G. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10601278" se_prerender_url="complete" target="_blank">A novel human hepatic organic anion transporting polypeptide (OATP2). Identification of a liver-specific human organic anion transporting polypeptide and identification of rat and human hydroxymethylglutaryl-CoA reductase inhibitor transporters.</a> <em>J Biol Chem</em> 274(52): 37161-37168.</li><li>Khurana, V., Minocha, M., Pal, D. and Mitra, A. K. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24643910" se_prerender_url="complete" target="_blank">Role of OATP-1B1 and/or OATP-1B3 in hepatic disposition of tyrosine kinase inhibitors.</a> <em>Drug Metabol Drug Interact</em> 29(3): 179-190.</li><li>Lee, H. H., Leake, B. F., Teft, W., Tirona, R. G., Kim, R. B. and Ho, R. H. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25695959" se_prerender_url="complete" target="_blank">Contribution of hepatic organic anion-transporting polypeptides to docetaxel uptake and clearance.</a> <em>Mol Cancer Ther</em> 14(4): 994-1003. <br/></li><li>Nozawa, T., Minami, H., Sugiura, S., Tsuji, A. and Tamai, I. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15608127" se_prerender_url="complete" target="_blank">Role of organic anion transporter OATP1B1 (OATP-C) in hepatic uptake of irinotecan and its active metabolite, 7-ethyl-10-hydroxycamptothecin: <em>in vitro</em> evidence and effect of single nucleotide polymorphisms.</a> <em>Drug Metab Dispos</em> 33(3): 434-439.</li><li>Oostendorp, R. L., van de Steeg, E., van der Kruijssen, C. M., Beijnen, J. H., Kenworthy, K. E., Schinkel, A. H. and Schellens, J. H. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19139163" target="_blank">Organic anion-transporting polypeptide 1B1 mediates transport of Gimatecan and BNP1350 and can be inhibited by several classic ATP-binding cassette (ABC) B1 and/or ABCG2 inhibitors.</a> <em>Drug Metab Dispos</em> 37(4): 917-923.</li><li>Shitara, Y., Maeda, K., Ikejiri, K., Yoshida, K., Horie, T. and Sugiyama, Y. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23115084" target="_blank">Clinical significance of organic anion transporting polypeptides (OATPs) in drug disposition: their roles in hepatic clearance and intestinal absorption.</a> <em>Biopharm Drug Dispos</em> 34(1): 45-78.</li><li>Simonson, S. G., Raza, A., Martin, P. D., Mitchell, P. D., Jarcho, J. A., Brown, C. D., Windass, A. S. and Schneck, D. W. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15289793" target="_blank">Rosuvastatin pharmacokinetics in heart transplant recipients administered an antirejection regimen including cyclosporine.</a> <em>Clin Pharmacol Ther</em> 76(2): 167-177.<br/></li></ol>
