<body>
            <h1>Application of a Modified Smart-seq2 Sample Preparation Protocol for Rare Cell Full-length Single-cell mRNA Sequencing to Mouse Oocytes</h1>
            <p><strong>Authors:</strong> Rebecca S. Treger, Scott D. Pope, Xiaojun Xing, Akiko Iwasaki</p>
            <h2>Abstract</h2>
            <p>Endogenous retroviruses (ERV) are transposable retroelements that form ~10% of the murine genome and whose family members are differentially expressed throughout embryogenesis. However, precise regulation of ERV in germ cells remains unclear. To investigate ERV expression in oocytes, we adapted a single-cell mRNA-sequencing library preparation method to generate bulk sequencing libraries from growing oocytes in a time- and cost-efficient manner. Here, we present a modified Smart-seq2 protocol that yields full-length cDNA libraries from purified RNA obtained from low numbers of pooled immature or mature oocytes. Using this method, RNA-sequencing libraries can be generated from any rare or difficult-to-isolate populations for subsequent sequencing and retroelement expression analysis.</p>
            <p><strong>Keywords:</strong> Oocyte, Transposable element, Endogenous retrovirus, mRNA-sequencing, Smart-seq2</p>
            <h2>Background</h2><p>Endogenous retroviruses (ERV) are retrovirus-like mobile DNA elements that form ~10% of the murine genome, and are expressed in both healthy tissues and disease states (Kazazian and Moran, 2017; Johnson, 2019). Expression of ERV families is tissue-specific (Faulkner <em>et al.</em>, 2009), and many families of ERV are temporally and differentially regulated throughout embryogenesis (Peaston <em>et al.</em>, 2004; Rowe and Trono, 2011). We observed that proviral ERV are globally dysregulated in C57BL/6N, but not C57BL/6J, mice, secondary to loss of two transcriptional repressors, suppressor of non-ecotropic ERV (SNERV)-1, and SNERV-2, that promote heterochromatinization at ERV loci (Treger <em>et al.</em>, 2019). We observed that in the absence of SNERVs in C57BL/6N mice, ERV are upregulated in a variety of immune cells, as well as in murine embryonic fibroblasts and embryonic stem cells. To identify whether ERV dysregulation is similarly present in gametes, we needed a cost-effective method to successfully generate cDNA sequencing libraries from pooled samples of growing oocytes. <br/><br/>While primordial oocytes can now be efficiently isolated from fetal and neonatal ovaries by flow cytometry (Stewart <em>et al.</em>, 2015), antral germinal vesicle (immature) and metaphase II-arrested (mature) oocytes require a technique employing microdissection of superovulated ovaries (Seli <em>et al.</em>, 2005; Duselis and Vrana, 2007; Litscher and Wassarman, 2010; Stein and Schindler, 2011; Guzeloglu-Kayisli <em>et al.</em>, 2012;). While superovulation increases the yield of immature and mature oocytes, only ~20-30 ovulated oocytes per mouse can be reliably obtained through this method (Litscher and Wassarman, 2010). Prior studies have generated mRNA sequencing libraries from ~700-1,500 oocytes ( Smallwood <em>et al.</em>, 2011; Veselovska <em>et al.</em>, 2015; Gahurova <em>et al.</em>, 2017), whose isolation represents a significant investment of both time and money.<br/><br/>The transcriptomes of immature (GV) and mature (MII) oocytes have been successfully sequenced by single-cell RNA-seq (scRNA-sesq) technologies (Tang <em>et al.</em>, 2010; Xue <em>et al.</em>, 2013). scRNA-seq permits the identification of cell-to-cell heterogeneity in a sample by detecting high-abundance cellular transcripts (Liu and Trapnell, 2016; Picelli, 2016; Choi and Kim, 2019), and circumvents the need to isolate large numbers of oocytes for bulk sequencing library preparation. However, our aim was to sensitively detect even low-abundance retroelement transcripts in growing oocytes and to detect differential expression between substrains. For this goal, scRNA-seq was not a suitable technique. Additionally, a high percentage of repeat element sequencing reads map to multiple locations in the genome and transcriptome, and their mapping accuracy is improved by using longer sequencing read lengths and paired-end reads (Treangen and Salzberg, 2011). Importantly, while many scRNA-seq methods amplify only the 3â² end of polyadenylated transcripts, the Smart-seq2 method is a low-cost protocol that generates full-length cDNA from mRNA transcripts (Picelli <em>et al.</em>, 2014; Haber <em>et al.</em>, 2017; Ziegenhain <em>et al.</em>, 2017) and could further improve mappability of repeat-associated reads. For these reasons, we developed a modified Smart-seq2 protocol to generate full-length cDNA libraries from low numbers of pooled oocytes for bulk sequencing to permit subsequent analysis of transposable element expression.<br/><br/>This Smart-seq2-based protocol provides a cost-effective method to generate libraries for bulk mRNA-sequencing of oocytes, which results in data suitable for downstream analysis of population-level differences in retroelement expression. This protocol can be further modified to generate sequencing libraries from any rare or difficult-to-isolate population, with broad applications ranging from analysis of transduced primary cells with low transduction efficiency to investigation of transcriptional differences in scarce tissue-resident immune cell populations.</p>
        </body>

<p class="pptt" id="biaoti28923">## Materials and Reagents</p><ol class="npwtwok"><li>30 gauge needles (BD, catalog number: 305128)</li><li>100 mm Petri dish (Falcon, catalog number: 351029)</li><li>1.5 ml autoclaved sterile microcentrifuge tubes (USA Scientific, catalog number: 1615-5500)</li><li>SHARP Precision barrier tips (Denville, catalog numbers: P1120, P1121, P1122, P1126)</li><li>Twintec PCR 96-well PCR plates, colorless (Eppendorf, catalog number: 951020401)</li><li>Microseal B PCR Plate Sealing Film (Bio-Rad, catalog number: MSB1001)</li><li>Qubit Assay tubes (ThermoFisher, catalog number: Q32856)</li><li>200 Î¼l pipette tip (Fisherbrand, catalog number: 02-681-147 or equivalent)</li><li>Pregnant mare serum gonadotropin (Biovendor, catalog number: RP1782721000)</li><li>Human chorionic gonadotropin (Sigma, catalog number: CG10-10VL)</li><li>M2 medium (Millipore, catalog number: MR-015P-5F)</li><li>Hyaluronidase (Sigma, catalog number: H4272)</li><li>Milrinone (Sigma, catalog number: M4659)</li><li>RNase Away (Ambion, catalog number: 10328011)</li><li>Î²-mercaptoethanol (Sigma, catalog number: M6250)</li><li>RLT buffer (Qiagen, catalog number: 79216)</li><li>Dry ice</li><li>RNeasy Micro Kit (Qiagen, catalog number: 74004)</li><li>Ultrapure water (AmericanBio, catalog number: AB02128)</li><li>RNAClean XP beads (Agencourt, catalog number: A63987)</li><li>80% ethanol</li><li>10 mM dNTPs (Thermo Fisher, catalog number: R1092)</li><li>RNase Inhibitor (Takara, catalog number: 2313A)</li><li>Maxima RNase H-minus RT 5x Buffer (Thermo Fisher, catalog number: EP0751)</li><li>Maxima RNase H-minus RT (Thermo Fisher, catalog number: EP0751)</li><li>Betaine (Sigma, catalog number: 61962)</li><li>1 M MgCl<sub>2</sub> (Sigma, catalog number: M1028)</li><li>Kapa HiFi HotStart ReadyMix (Kapa Biosystems, catalog number: KK2601)</li><li>AMPure XP beads (Agencourt, catalog number: A63882)</li><li>70% ethanol</li><li>TE buffer (AmericanBio, catalog number: AB14033)</li><li>Qubit dsDNA HS Assay Kit (Thermo Fisher, catalog number: Q32851)</li><li>High Sensitivity DNA Kit (Agilent, catalog number: 5067-4626)</li><li>Illuminaâs Nextera XT Kit</li><li>Oligos<br/><ol class="npwthree"><li>3â² RT primer<br/>5â²-AAGCAGTGGTATCAACGCAGAGTACT30VN-3â² (Sigma)<br/><em>Note: T30VN corresponds to 30 Tâs followed by a âVâ (A, C, or G), followed by an âNâ (any base).</em><br/></li><li>Template Switching Oligo<br/>5â²-AAGCAGTGGTATCAACGCAGAGTACATrGrG+G-3â² (Exiqon)<br/><em>Note: ârGâ stands for riboguanosine, while â+Gâ indicates that the last deoxyguanosine is modified to be a locked nucleic acid.</em><br/></li><li>ISPCR<br/>5â²-AAGCAGTGGTATCAACGCAGAGT-3â² (Sigma)<br/><em>Note: All oligos should be ordered as desalted and diluted to 100 Î¼M using 0.1x TE and stored at -20 Â°C prior to use.</em> </li></ol></li><li>Master Mix 1 (see Recipes)</li><li>Master Mix 2 (see Recipes)</li><li>Master Mix 3 (see Recipes)<br/></li></ol><p class="pptt" id="biaoti28924">### Equipment</p><ol class="npwtwok"><li>Dissecting scissors (Fine Science Tools, catalog number: 91460-11 or equivalent)</li><li>Curved tip forces (Fine Science Tools, catalog number: 11273-20 or equivalent)</li><li>Watchmakerâs forceps (Fine Science Tools, catalog number: 11251-20 of equivalent)</li><li>Dissecting microscope (Laxco, LMS-S2103-LS200 or equivalent)</li><li>1.0 ml pipette controller (Hamilton, catalog number: 0010PC)</li><li>Aspirator tube assemblies for calibrated microcapillary pipettes (Sigma, catalog number: A5177-5EA)</li><li>Capillaries without filament (World Precision Instruments, catalog number: 1B100-6)</li><li>Vortex machine (Scientific Industries, catalog number: 00-SI-0236)</li><li>Swinging bucket centrifuge (Eppendorf, model: 5810R)</li><li>96-well magnet plate (Alpaqua Magnum, model: FLX A000400 or equivalent)</li><li>96-well heated thermal cycler (Bio-Rad, model: S1000 or equivalent)</li><li>Qubit Fluorometer (Invitrogen, model: Q33226 or equivalent)</li><li>Agilent 2100 Bioanalyzer System (Agilent, G2939BA, G2953CA) <br/></li></ol><p class="pptt" id="biaoti28925">### Procedure</p><p class="ppzz"><strong>Part I: Oocyte isolation</strong><br/><br/>This protocol should yield ~80-120 mature oocytes and ~200-300 immature oocytes per pooled sample (from four mice).<br/><em>Note: Prior to beginning this protocol, assemble the Hamilton pipette-based micropipette oocyte handling device as described in Xing (2010). Videos that detail the following procedures for oviduct removal and dissection can be found in the Journal of Visualized Experiments (Duselis and Vrana, 2007; Stein and Schindler, 2011).<br/><br/></em> </p><ol class="npwone"><li>RLT buffer preparation<ol class="npwtwo"><li>Clean pipettes and workspace thoroughly with RNase Away to avoid RNase contamination.</li><li>Using PCR-rated filter tips, add 10 Î¼l of Î²-mercaptoethanol to 1 ml of RLT buffer in a 1.5 ml Eppendorf tube and pipette to mix well.<br/><em>Note: 350 Î¼l of the RLT buffer containing Î²-mercaptoethanol will be needed per RNA sample. Increase volumes proportionally to obtain the desired final volume. RLT buffer containing Î²-mercaptoethanol can be stored for up to 1 month at room temperature.</em> </li><li>Transfer 350 Î¼l of the RLT buffer containing Î²-mercaptoethanol to each of two 1.5 ml Eppendorf tubes.<br/><br/></li></ol></li><li>Mice superovulation and tissue harvesting<ol class="npwtwo" start="1"><li>Mature oocyte generation and oviduct isolation<br/><ol class="npwthree"><li>Inject four 3-week-old female mice intraperitoneally with 5 IU of pregnant mare serum gonadotropin (PMSG) to induce oocyte growth.</li><li>Forty-eight hours later, inject these same mice intraperitoneally with 5 IU of human chorionic gonadotropin (hCG) to induce oocyte ovulation.</li><li>Euthanize the mice by cervical dislocation fourteen hours after hCG injection.</li><li>Use sterilized fine forceps and fine scissors to remove the oviducts with attached uterine horn from each mouse (total of 8). Place tissues into a droplet of pre-warmed room temperature M2 media on an untreated 100 mm Petri dish.<br/></li><li>Transfer one oviduct-uterine horn sample at a time to a new 300 Î¼l drop of pre-warmed M2 media on a new untreated 100 mm Petri dish.<br/></li><li>Under a dissecting microscope, use two pairs of watchmakerâs forceps to tear the ampulla of the oviduct and release the cumulus-oocyte complexes into the surrounding media.<br/></li><li>Add 300 Î¼l of 0.6 mg/ml hyaluronidase to the drop of media (final concentration of 0.3 mg/ml).</li></ol><em>Note: In order to optimize RNA quality, proceed directly to Procedure C prior to isolating immature oocytes from mice (Step B2).</em> </li><li>Immature oocyte generation and ovary isolation<br/><ol class="npwthree"><li>Inject four 3-week-old female mice intraperitoneally with 5 IU of pregnant mare serum gonadotropin to induce superovulation.</li><li>Euthanize the mice by cervical dislocation forty-four hours after PMSG injection.</li><li>Use sterilized fine forceps and fine scissors to remove the ovaries from each mouse (total of 8). Place tissues into a droplet of pre-warmed room temperature M2 media on an untreated 100 mm Petri dish.</li><li>Transfer one ovary at a time to a new 300 Î¼l drop of pre-warmed M2 media supplemented with 10 Î¼m milrinone on a new 100 mm Petri dish. Milrinone helps to ensure metaphase arrest of the oocytes (Stein and Schindler, 2011).<br/></li><li>Add 300 Î¼l of 0.6 mg/ml hyaluronidase to the drop of media (final concentration of 0.3 mg/ml).</li></ol><br/></li></ol></li><li>Oocyte isolation <br/><em>Note: During the processing of immature oocytes, use M2 media supplemented with 10 Î¼m milrinone.</em><br/><ol class="npwtwo" start="1"><li>Under a dissecting microscope, puncture and gently dissociate the tissue with two 30-gauge needles to aid the release of oocytes into the surrounding media.</li><li>Detach adherent cumulus cells from the oocytes by gently and repeatedly pipetting the cumulus-oocyte complexes using the micropipette.</li><li>Once denuded of cumulus cells, wash oocytes three times by transferring the oocytes into new droplets of pre-warmed M2 media on the untreated 100 mm Petri dish using the micropipette.</li><li>Count the total number of oocytes as they are drawn up into the micropipette. Aspirate the minimal amount of media possible (less than 6 Î¼l), as large volumes will dilute the RLT buffer.</li><li>Transfer the counted immature or mature oocytes into the corresponding Eppendorf tube containing previously prepared RLT buffer.</li><li>Vortex the tube at high speed for 45 s to homogenize the sample. Place tube on dry ice until it can be stored at -80 Â°C.<br/></li></ol></li></ol><p class="ppzz" style="padding-left:20px;"><br/></p><p class="ppzzs"><strong>Part II: Oocyte RNA library preparation<br/><br/></strong> </p><ol class="npwone"><li>RNA isolation<ol class="npwtwo"><li>Thaw RLT samples on ice.</li><li>Isolate RNA from the thawed samples using the RNeasy Micro Kit, according to the manufacturerâs instructions.</li><li>Elute RNA in 14 Î¼l of ultra-pure water and store at -80 Â°C until ready to proceed with library preparation.<br/><br/></li></ol></li><li>cDNA generation<br/><em>Note: The heated lid should be set to 105 Â°C for all thermal cycler programs. Preheat the PCR machine before running each program.</em><br/><ol class="npwtwo" start="1"><li>Allow RNAClean XP beads to equilibrate to room temperature for 30 min before use in Step B4.</li><li>Prepare Master Mix 1 and place on ice until ready to be used.</li><li>Transfer 10 Î¼l of each RNA sample to a corresponding well of a 96-well skirted PCR plate.</li><li>Add 22 Î¼l (2.2x) RNAClean XP beads to each sample and pipette gently to mix. Allow to incubate for 10 min at room temperature. The RNA is now bound to the beads.</li><li>Place the 96-well plate containing beads and RNA on a 96-well magnet plate and incubate for 5 min. The magnetic beads will aggregate along the sidewalls of the wells.</li><li>Use a multichannel pipette to carefully remove the supernatant without disturbing the beads.</li><li>Wash the beads by gently pipetting 100 Î¼l of 80% ethanol over the beads. Let stand for 1 min and then remove the supernatant without disturbing the beads.</li><li>Repeat Step B7 for a total of two ethanol washes.</li><li>Allow the beads to air dry on the magnet for 5 min.<br/></li><li>Remove the 96-well plate from the magnet stand and resuspend each well of dried beads in 8 Î¼l of Master Mix 1.</li><li>Seal the plate with a Microseal B film, quick-spin the plate in a swinging bucket centrifuge to remove liquid from the sidewalls, and incubate the plate for 3 min at 72 Â°C. This step denatures the RNA.<br/></li><li>Immediately place the sealed 96-well plate on ice for 1 min. This step minimizes RNA secondary structure formation.</li><li>Prepare Master Mix 2 and place on ice until ready to be used.</li><li>Add 14 Î¼l of Master Mix 2 to each well for a total volume of 22 Î¼l and gently pipette to mix.</li><li>Seal the plate with a Microseal B film and quick-spin the plate in a swinging bucket centrifuge to remove liquid from the sidewalls. Place the plate in a preheated thermal cycler with the heated lid set to 105 Â°C and run PCR Program 1 to perform reverse transcription of the RNA.<br/><strong>PCR Program 1:</strong><br/>50 Â°C for 90 min<br/>85 Â°C for 5 min</li><li>Prepare Master Mix 3 and place on ice until ready to be used.</li><li>Following reverse transcription, quick-spin the plate in a swinging bucket centrifuge to remove liquid from the sidewalls and remove the adhesive seal.</li><li>Add 28 Î¼l of Master Mix 3 to each well for a total volume of 50 Î¼l and pipette gently to mix.</li><li>Seal the plate with a Microseal B film and quick-spin the plate in a swinging bucket centrifuge to remove liquid from the sidewalls. Place the plate in a preheated thermal cycler with the heated lid set to 105 Â°C and run PCR Program 2 to amplify cDNA.<br/><em>Note: Because of the difficulty of isolating oocytes in large numbers, coupled with the high cost of library preparation and sequencing, we chose a PCR amplification cycle number in Step B19 based upon Qubit results we obtained in Step B31 using equivalent input numbers of splenocytes. Ideally, the PCR amplification cycle number will be determined empirically using test samples of the same cell type and input cell number being sequenced. An ideal final concentration of cDNA is ~0.5 ng/Î¼l, which permits further dilution of the cDNA sample to 0.2 ng/Î¼l. </em><br/><strong>PCR Program 2:</strong><br/>98 Â°C for 3 min<br/>Then 12 cycles of:<br/>98 Â°C for 15 s <br/>67 Â°C for 20 s<br/>72 Â°C for 6 min<br/>Followed by:<br/>72 Â°C for 5 min</li><li>Allow AMPure XP beads to equilibrate to room temperature for 30 min before use in Step B22.</li><li>Quick-spin the plate in a swinging bucket centrifuge to remove liquid from the sidewalls and remove the adhesive seal.\</li><li>Add 35 Î¼l (0.7x) AMPure XP beads to each sample and pipette gently to mix. Allow to incubate for 5 min at room temperature. The cDNA is now bound to the beads.</li><li>Place the 96-well plate on a 96-well magnet plate and incubate for 6 min.</li><li>Use a multichannel pipette to carefully remove the supernatant without disturbing the beads.</li><li>Wash the beads by gently pipetting 100 Î¼l of 70% ethanol over the beads. Let stand for 1 min and then remove the supernatant without disturbing the beads.</li><li>Repeat Step B25 for a total of two ethanol washes.</li><li>Allow the beads to air dry on the magnet for 5 min.</li><li>Remove the 96-well plate from the magnet stand and resuspend each well of dried beads in 40 Î¼l of 1x TE Buffer to elute the cDNA from the beads for 5 min.</li><li>Place the 96-well plate on a 96-well magnet plate and incubate for 5 min. The magnetic beads will aggregate along the sidewalls of the wells. The cDNA is contained in the supernatant.</li><li>Use a multichannel pipette to carefully transfer 36 Î¼l of supernatant from each well into a new well of the same 96-well plate (one well per pooled sample).</li><li>Quantify the cDNA concentration of each sample using a Qubit dsDNA HS kit, according to the manufacturerâs instructions.</li><li>The cDNA quality can be analyzed using a 2100 Bioanalyzer with the high sensitivity DNA kit, according to the manufacturerâs instructions. The average size of the PCR-amplified cDNA library should be 1.5-2 kb, as described alongside representative Bioanalyzer traces (Picelli <em>et al.</em>, 2014).</li><li>Normalize the cDNA concentration of each well to 0.2 ng/Î¼l using 1x TE Buffer.</li><li>Proceed to RNA-seq library generation using Illuminaâs Nextera XT Kit with 1 ng of cDNA as input.<br/></li></ol></li></ol><p class="pptt" id="biaoti28926">### Data analysis</p><p class="ppzz">cDNA was generated from total RNA isolated from immature and mature C57LB/6J oocytes as described in the above protocol. Sequencing libraries were then constructed using tagmentation with Illuminaâs Nextera XT kit and paired-end 75bp reads were sequenced on the NextSeq500. Sequencing reads were mapped to 680 LTR families using the RepEnrich (Criscione <em>et al.</em>, 2014) alignment strategy and are plotted below. Detailed methods for data analysis have been previously published (Treger <em>et al.</em>, 2019). <br/><br/>We observe that expression of 197 of the 680 LTR families is significantly increased or decreased more than two-fold in mature oocytes when compared to immature oocytes (Figure 1). The 191 of these 197 LTR families are downregulated in mature oocytes, and the expression of only 6 LTR familiesâall of which correspond to virus-like 30 (VL30) elements or VL30-associated LTRsâis increased in mature oocytes (Figure 1). VL30 elements are proviral-like retroelements that lack envelope-coding regions and do not encode intact open reading frames (Markopoulos <em>et al.</em>, 2016). VL30 expression is induced by both epidermal growth factor and steroid hormones such as estradiol and progesterone <em>in vitro</em> (Courtney <em>et al.</em>, 1982; Tzavaras <em>et al.</em>, 2003). Transcription of VL30-type LTRs may be directly induced by growth factors and/or hormones as the germinal vesicle oocytes mature and arrest at metaphase II. Alternatively, these LTR elements may be coordinately regulated by the transcriptional activation of nearby cellular genes that are activated during oocyte maturation.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190818/20190818033624_6937.jpg"/><br/><strong>Figure 1. While VL30 LTR family members are upregulated, the expression of other LTR elements are globally decreased in mature oocytes.</strong> Volcano plot of significantly differentially expressed long-terminal repeat (LTR) families from mRNA sequencing comparing immature and mature oocytes. The log<sub>2</sub> fold-change in expression is graphed against the negative log<sub>10</sub> of the adjusted P-value calculated by DESeq2 (Love <em>et al.</em>, 2014). Positive fold-change values indicate LTR families whose expression is increased in mature oocytes, while negative fold-change values indicate LTR families whose expression is decreased in mature oocytes. mRNA-sequencing libraries were generated using the modified Smart-seq2-based protocol presented in this paper.</p><p class="pptt" id="biaoti28927">### Recipes</p><ol class="npwtwok" start="1"><li>Master Mix 1<br/>2.5 Î¼M 3â² RT primer<br/>2.5 mM dNTPs<br/>1 unit/Î¼l RNase inhibitor<br/><strong>For making 8 Î¼l (per well) Master Mix 1:</strong><br/>0.2 Î¼l of 100 Î¼M 3â² RT primer<br/>2 Î¼l of 10 mM dNTPs<br/>0.2 Î¼l of 40 U/Î¼l RNase inhibitor<br/>5.6 Î¼l of Ultrapure water</li><li>Master Mix 2<br/>1.4x Maxima RNase H-minus RT Buffer <br/>1.4 M Betaine, 12.9 mM MgCl<sub>2</sub><br/>1.4 Î¼M Template Switching Oligo<br/>1.4 units/Î¼l RNAase inhibitor <br/>2.9 units/Î¼l Maxima RNase H-minus RT<br/><strong>For making 14 Î¼l (per well) Master Mix 2:</strong><br/>4 Î¼l of 5x Maxima RNase H-minus RT Buffer<br/>4 Î¼l of 5 M Betaine<br/>0.18 Î¼l of 1 M MgCl<sub>2</sub><br/>0.2 Î¼l of 100 Î¼M Template Switching Oligo<br/>0.5 Î¼l of 40 U/Î¼l RNase inhibitor<br/>0.2 Î¼l of 200 U/Î¼l Maxima RNase H-minus RT<br/>4.92 Î¼l of Ultrapure water</li><li>Master Mix 3 <br/>0.4 Î¼M ISPCR primer<br/>1.8x Kapa HiFi HotStart ReadyMix<br/><strong>For making 28 Î¼l (per well) Master Mix 2:</strong><br/>25 Î¼l of 2x Kapa HiFi HotStart ReadyMix<br/>0.1 Î¼l of 100 Î¼M ISPCR primer<br/>2.9 Î¼l of Ultrapure water<br/></li></ol><p class="pptt" id="biaoti28928">### Acknowledgments</p><p class="ppzz" style="padding-left:20px;">This work was supported in part by Howard Hughes Medical Institute (to A.I.) and by NIH award R01 AI054359 and R01 AI127429 (to A.I.). R.T. was supported by NIH training grant (5-T32-GM00720540) and F30 (5-F30-AI129265-02). An abbreviated version of this protocol was originally published in âThe Lupus Susceptibility Locus <em>Sgp3</em> Encodes the Suppressor of Endogenous Retrovirus Expression SNERVâ (Treger <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti28929">### Competing interests</p><p class="ppzz" style="padding-left:20px;">The authors declare no competing interests.</p><p class="pptt" id="biaoti28930">### Ethics</p><p class="ppzz" style="padding-left:20px;">Mice were housed in SPF conditions and care was provided in accordance with Yale University IACUC guidelines (protocol #10365).</p><p class="pptt" id="biaoti28931">### References</p><ol class="npwtwok" start="1"><li>Choi, Y. H. and Kim, J. K. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30764602" target="_blank">Dissecting cellular heterogeneity using single-cell RNA sequencing.</a> <em>Mol Cells</em> 42(3): 189-199.</li><li>Courtney, M. G., Schmidt, L. J. and Getz, M. J. (1982). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6275981" target="_blank">Organization and expression of endogenous virus-like (VL30) DNA sequences in nontransformed and chemically transformed mouse embryo cells in culture.</a> <em>Cancer Res </em>42(2): 569-576.</li><li>Criscione, S. W., Zhang, Y., Thompson, W., Sedivy, J. M. and Neretti, N. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25012247" target="_blank">Transcriptional landscape of repetitive elements in normal and cancer human cells.</a> <em>BMC Genomics</em> 15: 583-583.</li><li>Duselis, A. R. and Vrana, P. B. (2007). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2535842/" target="_blank">Retrieval of mouse oocytes.</a> <em>JoVE </em>(3): 185-185.</li><li>Faulkner, G. J., Kimura, Y., Daub, C. O., Wani, S., Plessy, C., Irvine, K. M., Schroder, K., Cloonan, N., Steptoe, A. L., Lassmann, T., Waki, K., Hornig, N., Arakawa, T., Takahashi, H., Kawai, J., Forrest, A. R., Suzuki, H., Hayashizaki, Y., Hume, D. A., Orlando, V., Grimmond, S. M. and Carninci, P. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19377475" target="_blank">The regulated retrotransposon transcriptome of mammalian cells.</a> <em>Nat Genet</em> 41(5): 563-571.</li><li>Gahurova, L., Tomizawa, S.-I., Smallwood, S. A., Stewart-Morgan, K. R., Saadeh, H., Kim, J., Andrews, S. R., Chen, T. and Kelsey, G. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28507606" target="_blank">Transcription and chromatin determinants of de novo DNA methylation timing in oocytes.</a> <em>Epigenetics Chromatin</em> 10: 25-25.</li><li>Guzeloglu-Kayisli, O., Lalioti, M. D., Aydiner, F., Sasson, I., Ilbay, O., Sakkas, D., Lowther, K. M., Mehlmann, L. M. and Seli, E. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22621333" target="_blank">Embryonic poly(A)-binding protein (EPAB) is required for oocyte maturation and female fertility in mice.</a> <em>Biochemical J</em> 446(1): 47-58.</li><li>Haber, A. L., Biton, M., Rogel, N., Herbst, R. H., Shekhar, K., Smillie, C., Burgin, G., Delorey, T. M., Howitt, M. R., Katz, Y., Tirosh, I., Beyaz, S., Dionne, D., Zhang, M., Raychowdhury, R., Garrett, W. S., Rozenblatt-Rosen, O., Shi, H. N., Yilmaz, O., Xavier, R. J. and Regev, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29144463" target="_blank">A single-cell survey of the small intestinal epithelium.</a> <em>Nature</em> 551(7680): 333-339.</li><li>Johnson, W. E. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30962577" target="_blank">Origins and evolutionary consequences of ancient endogenous retroviruses.</a> <em>Nat Rev Microbiol</em> 17(6): 355-370.</li><li>Kazazian, H. H., Jr. and Moran, J. V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28745987" target="_blank">Mobile DNA in health and disease.</a> <em>N Engl J Med</em> 377(4): 361-370.</li><li>Litscher, E. S. and Wassarman, P. M. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20691861" target="_blank">Chapter 5: Isolation and manipulation of mouse gametes and embryos.</a> In: <em>Methods in Enzymology</em>. Wassarman, P. M. and Soriano, P. M. (Eds.). Academic Press. 73-84.</li><li>Liu, S. and Trapnell, C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26949524" target="_blank">Single-cell transcriptome sequencing: recent advances and remaining challenges.</a> <em>F1000Res</em> 5.</li><li>Love, M. I., Huber, W. and Anders, S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25516281" target="_blank">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2.</a> <em>Genome Biol</em> 15(12): 550.</li><li>Markopoulos, G., Noutsopoulos, D., Mantziou, S., Gerogiannis, D., Thrasyvoulou, S., Vartholomatos, G., Kolettas, E. and Tzavaras, T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27158269" target="_blank">Genomic analysis of mouse VL30 retrotransposons.</a> <em>Mob DNA</em> 7: 10.</li><li>Peaston, A. E., Evsikov, A. V., Graber, J. H., de Vries, W. N., Holbrook, A. E., Solter, D. and Knowles, B. B. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15469847" target="_blank">Retrotransposons regulate host genes in mouse oocytes and preimplantation embryos.</a> <em>Dev Cell</em> 7(4): 597-606.</li><li>Picelli, S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27442339" target="_blank">Single-cell RNA-sequencing: The future of genome biology is now.</a> <em>RNA Biol</em> 14(5): 637-650.</li><li>Picelli, S., Faridani, O. R., Bjorklund, A. K., Winberg, G., Sagasser, S. and Sandberg, R. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24385147" target="_blank">Full-length RNA-seq from single cells using Smart-seq2.</a> <em>Nat Protoc</em> 9(1): 171-181.</li><li>Rowe, H. M. and Trono, D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21251689" target="_blank">Dynamic control of endogenous retroviruses during development.</a> <em>Virology</em> 411(2): 273-287.</li><li>Seli, E., Lalioti, M.D., Flaherty, S.M., Sakkas, D., Terzi, N., and Steitz, J.A. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15630085" target="_blank">An embryonic poly(A)-binding protein (ePAB) is expressed in mouse oocytes and early preimplantation embryos.</a> <em>PNAS</em> 102, 367-372.</li><li>Smallwood, S. A., Tomizawa, S., Krueger, F., Ruf, N., Carli, N., Segonds-Pichon, A., Sato, S., Hata, K., Andrews, S. R. and Kelsey, G. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21706000" target="_blank">Dynamic CpG island methylation landscape in oocytes and preimplantation embryos.</a> <em>Nat Genet</em> 43(8): 811-814.</li><li>Stein, P. and Schindler, K. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21808228" target="_blank">Mouse oocyte microinjection, maturation and ploidy assessment.</a> <em>J Vis Exp </em>(53). pii: 2851. doi: 10.3791/2851.</li><li>Stewart, K. R., Veselovska, L., Kim, J., Huang, J., Saadeh, H., Tomizawa, S., Smallwood, S. A., Chen, T. and Kelsey, G. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26584620" target="_blank">Dynamic changes in histone modifications precede de novo DNA methylation in oocytes.</a> <em>Genes Dev</em> 29(23): 2449-2462.</li><li>Tang, F., Barbacioru, C., Nordman, E., Li, B., Xu, N., Bashkirov, V. I., Lao, K. and Surani, M. A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20203668" target="_blank">RNA-Seq analysis to capture the transcriptome landscape of a single cell.</a> <em>Nat Protoc</em> 5(3): 516-535.</li><li>Treangen, T. J. and Salzberg, S. L. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22124482" target="_blank">Repetitive DNA and next-generation sequencing: computational challenges and solutions.</a> <em>Nat Rev Genet</em> 13(1): 36-46.</li><li>Treger, R. S., Pope, S. D., Kong, Y., Tokuyama, M., Taura, M. and Iwasaki, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30709743" target="_blank">The lupus susceptibility locus Sgp3 encodes the suppressor of endogenous retrovirus expression SNERV.</a> <em>Immunity</em> 50(2): 334-347 e339.</li><li>Tzavaras, T., Eftaxia, S., Tavoulari, S., Hatzi, P. and Angelidis, C. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12964010" target="_blank">Factors influencing the expression of endogenous reverse transcriptases and viral-like 30 elements in mouse NIH3T3 cells.</a> <em>Int J Oncol</em> 23(4): 1237-1243.</li><li>Veselovska, L., Smallwood, S. A., Saadeh, H., Stewart, K. R., Krueger, F., Maupetit-Mehouas, S., Arnaud, P., Tomizawa, S., Andrews, S. and Kelsey, G. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26408185" target="_blank">Deep sequencing and de novo assembly of the mouse oocyte transcriptome define the contribution of transcription to the DNA methylation landscape.</a> <em>Genome Biol</em> 16: 209.</li><li>Xing, X. (2010). An embryo handling device with a hamilton pipette (International Society for Transgenic Technologies at <a href="https://www.transtechsociety.org/index.php" target="_blank">https://www.transtechsociety.org/</a>) (Membership user ID and password are required for access.)</li><li>Xue, Z., Huang, K., Cai, C., Cai, L., Jiang, C. Y., Feng, Y., Liu, Z., Zeng, Q., Cheng, L., Sun, Y. E., Liu, J. Y., Horvath, S. and Fan, G. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23892778" target="_blank">Genetic programs in human and mouse early embryos revealed by single-cell RNA sequencing.</a> <em>Nature</em> 500(7464): 593-597.</li><li>Ziegenhain, C., Vieth, B., Parekh, S., Reinius, B., Guillaumet-Adkins, A., Smets, M., Leonhardt, H., Heyn, H., Hellmann, I. and Enard, W. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28212749" target="_blank">Comparative analysis of single-cell RNA sequencing methods.</a> <em>Mol Cell</em> 65(4): 631-643 e634.<br/></li></ol>
