<body>
            <h1>PEG-mediated, Stable, Nuclear and Chloroplast Transformation of <em>Cyanidioschizon merolae</em></h1>
            <p><strong>Authors:</strong> Maksymilian  Zienkiewicz, Tomasz Krupnik, Anna DroÅ¼ak, Kinga Kania</p>
            <h2>Abstract</h2>
            <p>The ability to achieve nuclear or chloroplast transformation in plants has been a long standing goal, especially in microalgae research. Over past years there has been only little success, but transient and stable nuclear transformation has been achieved in multiple species. Our newly developed method allows for relatively simple transformation of <em>Cyanidioschizon merolae</em> in both nuclear and chloroplast genome by means of homologous recombination between the genome and a transformation vector. The use of chloramphenicol resistance gene as the selectable marker allows for plate-based efficient selection of mutant colonies. Overall, the method allows the generation of mutant strains within 6 months.</p>
            <p><strong>Keywords:</strong> <em>Cyanidioschizon merola</em>, Chloroplast transformation, Nuclear transformation, Chloramphenicol selection, Mutant selection, PEG mediated vector delivery</p>
            <h2>Background</h2><p><em>Cyanidioschizon merolae</em> 10D is a small single-cell eukaryotic red algae, that inhabits acidic hot springs. Lately, all three of its genomes were sequenced: nucleus (Matsuzaki <em>et al</em>., 2004), mitochondrion (Ohta <em>et al</em>., 1998), and plastid (Ohta <em>et al</em>., 2003). Its simple cell anatomy, the presence of a single mitochondrion and chloroplast, a highly reduced spliceosome (Stark <em>et al</em>., 2015), almost deprived of introns nuclear genome make this organism a suitable candidate for a model organism in broad range of plant research, from cell and plastid anatomy, through physiology and up to genetic engineering (Kuroiwa <em>et al</em>., 1998; Nozaki <em>et al</em>., 2003). Its simplicity makes it particularly useful for genetic manipulations. Thus, many genetic techniques have been applied and optimized, for this organism, including antisense suppression, transient expression, gene disruption, and stable insertion into the nuclear genome (Newell, 2000; Walker <em>et al</em>., 2005; Hallmann, 2007; Minoda <em>et al</em>., 2004; Ohnuma <em>et al</em>., 2008; Imamura <em>et al</em>., 2009; Ohnuma <em>et al</em>., 2009; Imamura <em>et al</em>., 2010; Watanabe <em>et al</em>., 2011, Zienkiewicz <em>et al</em>., 2017a). Recently, also stable transformation of the chloroplast genome has been achieved (Zienkiewicz <em>et al</em>., 2017b; Zienkiewicz <em>et al</em>., 2018) in our laboratory. The developed method takes advantage of double homologs recombination in <em>C. merolae</em>, allowing for integration of foreign DNA into the genome in specific regions. A gene knock-out mutation replaces or disrupts the gene of interest with the <em>cat</em> (chloramphenicol acetyltransferase; Zienkiewicz <em>et al</em>., 2017a) gene under the original promoter or any familiar nuclear or chloroplast promoter (Krupnik <em>et al</em>., 2018). Promoters driving the transcription of the <em>cat</em> gene should exhibit at least moderate to significant constitutive activity; therefore application of promoters already tested is advised (Krupnik <em>et al</em>., 2018). By introducing <em>Diphtheria</em> toxin (fragment A-DTA) under a constitutive promoter into the transformation vector we managed to virtually exclude the possibility of random, illegitimate recombination events (Zienkiewicz <em>et al</em>., 2018). A single recombination event, random integration or otherwise unspecific incorporation of the transformation vector into the nuclear genome would likely cause expression of the A-DTA protein and in effect results in cell lysis. However, the heterogeneity of chloroplast transformation still remains a challenge. We have shown that continued growth of chloroplast mutants in increasing concentration of chloramphenicol allows in principal for reduction of heterogeneity and selection of homogenous mutants. Chloramphenicol is the only antibiotic that is toxic to <em>C. merolae</em> (out of c.a. 10 tested, Zienkiewicz <em>et al</em>., 2017a). This method is just as simple as application of lipofectamine for DNA delivery and is as efficient as the gene-gun method but without its drawbacks and complexity (Zienkiewicz <em>et al</em>., 2017b). </p>
        </body>

<p class="pptt" id="biaoti36359">## Materials and Reagents</p><ol class="npwtwok"><li>Syringe filters (0.45 Î¼m, sterile, PTFE, VWR, Germany, catalog number: 514-0073)</li><li>1.5 ml microtubes (GenoPlast, Poland, GP30623B/80-1500)</li><li>250 ml Erlenmeyer glass flasks (Simax, Germany)</li><li>50 ml centrifuge tubes FALCON type (GenoPlast, Poland, GP440PSK)</li><li>Petri dishes (12 cm diameter)</li><li>Parafilm</li><li>Cell culture of <em>C. merolae</em>, 10D (NIES-3377 Unialgal, Clonal and axenic or NIES-1332, Unialgal, Clonal and Non-axenic) strain, was obtained from Microbial Culture Collection (mcc.nies.go.jp, Tsukuba, Japan) and was used throughout this study<br/><em>Note: Cells were grown in MA2 liquid medium (Minoda et al., 2004) in a glass vessel under continuous white light (50 Î¼mol photon m-2 s-1; measured with Quantitherm PAR/Temp Sensor ) at 42 Â°C or on Petri dishes filled with MA2 medium, solidified by addition of 1% (w/v) agar (Blirt, Poland).</em> </li><li>Tansformation vector<br/>Construction of the transformation vector depends entirely on the method of DNA integration or the general construction of the intended experiment. For example, a gene can be knocked out by means of double homologous recombination. For that purpose, a transformation vector should consist of two homologous flanks of 500-1000 bp length, at both 3â and 5â ends of the sequence of the gene of interest. Both flanks should be connected with a sequence of a selectable marker gene (catâChloramphenicol acetyltransferase) under a constitutive promoter, <em>e.g.</em>, (P<sub>apc</sub>) (Zienkiewicz <em>et al</em>., 2018).</li><li>Agar (LE Standard Blirt, Poland, catalog number: AG41)</li><li>PEG (Polyethylene glycol, Fluka, Germany, catalog number: BCBC0873)</li><li>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (Chempur, Poland, CAS: 7783-20-2,  WE: 231-984-1)</li><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Chempur, Poland)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Chempur, Poland, CAS: 7778-77-0,  WE: 231-913-4)</li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (Chempur, Poland CAS: 10035-04-8,  WE: 233-140-8)</li><li>EDTAÂ·2Na (Carl Roth, Germany, CAS: 6381-92-6,  EG-Nr. 205-358-3)</li><li>FeCl<sub>3</sub>Â·6H<sub>2</sub>O (Chempur, Poland, CAS: 10025-77-1,  WE: 231-729-4)</li><li>H<sub>3</sub>BO<sub>3</sub> (Chempur, Poland, CAS: 10043-35-3,  WE: 233-139-2)</li><li>MnCl<sub>2</sub>Â·4H<sub>2</sub>O (Chempur, Poland, CAS: 13446-34-9,  WE: 231-869-6)</li><li>ZnCl<sub>2</sub> Chempur (Poland, CAS: 7646-85-7,  WE: 231-592-0)</li><li>Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O (Chempur, Poland, CAS: 10102-40-6,  WE: 231-551-7)</li><li>CoCl<sub>2</sub>Â·6H<sub>2</sub>O (Chempur, Poland CAS: 7791-13-1,  WE: 231-589-4)</li><li>CuCl<sub>2</sub>Â·2H<sub>2</sub>O (Chempur, Poland CAS: 10125-13-0,  WE: 231-210-2)</li><li>PEG 4000 (Fluka, Germany, CAS: 25322-68-3)</li><li>Chloramphenicol (BioShop, Canada, CAS: 56-75-7, Lot No.:8E55095)</li><li>Chloramphenicol (see Recipes)</li><li>MA-1 (see Recipes)</li><li>MA2 (Allen medium) and 10x MA2 (Minoda <em>et al</em>., 2004, see Recipes)</li><li>Agar-solidified plates (see Recipes)</li><li>A2 Fe-EDTA solution (see Recipes)</li><li>A2 trace element stock solution (see Recipes)</li><li>PEG 60% (see Recipes)<br/></li></ol><p class="pptt" id="biaoti36360">### Equipment</p><ol class="npwtwok"><li>Temperature-regulated centrifuges (<em>e.g.</em>, Eppendorf Centrifuge, Germany, models: 5810 R and 5415 R)</li><li>Spectrophotometer (<em>e.g.</em>, Shimadzu, Japan, model: UV-1800)</li><li>NanoDrop (<em>e.g.</em>, NanoDrop Lite, Thermo, USA)</li><li>Quantitherm PAR/Temp Sensor (Hanzatech, UK)</li><li>Growth chamber/incubator (sustainable conditions of 42 Â°C and 50 Î¼moles m<sup>-2</sup> s<sup>-1</sup> of light)</li><li>Orbital shaker<br/></li></ol><p class="pptt" id="biaoti36361">### Procedure</p><ol class="npwone"><li><em>C. merolae</em> standard growth conditions<br/><ol class="npwtwo"><li>Standard growth medium is MA2 Allen medium (Recipe 2). We suggest preparation of 10x concentrated stock solution of MA2 with A2 trace elements, without A2 Fe-EDTA and autoclave it. For every new preparation of culture medium, dilute an appropriate volume. After autoclaving add A2 Fe-EDTA (Recipe 4).</li><li>The cultures should be grown in a transparent sterile vessel, of at least five times larger volume than the cell culture (<em>e.g.</em>, 50 ml cultures in 250 ml flasks).</li><li>Cultures must be gently agitated (30-50 rpm on an orbital shaker).</li><li>Cultures must be sterile so any culture vessel must be closed off but it must allow for gas exchange. Atmospheric CO<sub>2</sub> is sufficient for growth, but up to 5% CO<sub>2</sub> in air can be applied.</li><li>The light intensity should be in the range between 50 and 100 Âµmoles photons m<sup>-2</sup> s<sup>-1</sup> of continuous illumination. </li><li>Growth rates are similar in the temperature range of 37-42 Â°C.</li><li>OD of <em>C. merolae</em> should be kept between 0.01 and 1 at most. We advise to refresh every culture weekly or biweekly by inoculation of fresh culture out of 1/100 of the volume of the old culture. </li></ol><br/></li><li>PEG transformation of <em>C. merolae</em> cells<br/><ol class="npwtwo"><li>Set up at least (work in duplicates) 2x 50 ml culture of OD<sub>750</sub> = 0.1 a week prior to transformation and allow it to grow in the standard conditions up to OD<sub>750</sub> = 0.4.</li><li>Two days before transformation, introduce a photoperiod of 8/16 ho light/dark.</li><li>One hour before transformation expose to standard light conditions (if the dark period isnât completed).</li><li>Spin down 100 ml of culture at 40 Â°C and 2,000 <em>x g</em> for 5 min.</li><li>Resuspend the pellet in 50 ml of MA-1 solution at 40 Â°C and spin down again at 40 Â°C and 2,000 <em>x g</em> for 5 min.</li><li>Resuspend the pellet in 0.5 ml of MA-1 solution at 40 Â°C (now ready for transformation).</li><li>Produce large quantity (above 50 Î¼g and high concentration (above 200 ng/Î¼L, measure precisely on NanoDrop) of transformation vector.</li><li>Dilute the transformation vector to 20 Î¼g in total 400 Î¼l MA-1 buffer.</li><li>Add 100 Î¼l of the cell suspension to 400 Î¼l of DNA and gently mix it by pipetting it up and down 2-3 times.</li><li>Add 500 Î¼l of 60% sterile PEG and gently mix it with the tip (the final concentration of PEG should be 30%).</li><li>Incubate for 5 min at room temperature.</li><li>Transfer the cell suspension to 50 ml MA2 and spin down at 40 Â°C and 2,000 <em>x g</em> for 5 min, repeat one more time.</li><li>Resuspend in 50 ml of MA2 medium and apply standard growth conditions.</li><li>Allow growing under continuous light for 3 days, after that introduce selectable conditionsâchloramphenicol at 150 Î¼g/ml form freshly prepared stock solution of 100 mg/ml.</li><li>Culture should fade (slowly turn yellow/white) and then regain growth (by appearance of green color).</li><li>After 2-3 weeks, transfer 500 Î¼l of the culture into 50 ml of MA2 medium in 250 ml flask.</li><li>Plate 50 Î¼l of culture on 1% (w/v) agar-solidified MA2 Petri dishes, seal 3 times with parafilm to avoid excessive evaporation and drying out of the medium matrix.</li><li>Allow growing in standard conditions until colonies appear.</li><li>Pick 20-50 colonies and transfer into a 250 ml flask with 50 ml of MA2 containing 100 Î¼g/ml chloramphenicol.</li><li>After it grows up to OD<sub>750</sub> = 0.4, then inoculate into MA2 media with150, 200 and 250 Î¼g/ml of chloramphenicol, respectively.</li><li>Allow it to grow for ~2 weeks up to OD<sub>750</sub> ~0.5.</li><li>Inoculate new cultures from the successful, highest-concentration-of-chloramphenicol-grown surviving culture. </li><li>Repeat as long as necessary to achieve and confirm homogeneity by appropriate genetic means (<em>e.g.</em>, by qPCR specific to WT and modified genomes, Southern Blott The time needed for the entire procedure may vary and typically takes about 6 months). </li></ol></li></ol><p class="pptt" id="biaoti36362">### Data analysis</p><p class="ppzz">Due to 2-3 week-long period of selectable growth, followed by plating of the cells on solid medium it is impossible to approximate the efficiency of transformation. However, in our earlier attempts (Zienkiewicz <em>et al</em>., 2017a) we have obtained viable colonies in 10 out of 11 attempts. The introduced mutation supplied the cell with the resistance gene, without disrupting any native gene. When introducing any mutation of a functional gene, a lower efficiency should be expected, depending on the gene of interest. Homogeneity may be achieved by multiple passaging of cultures in stepwise increasing (typically, increase the chloramphenicol concentration by 50 Âµg/ml for every passage) concentration of the selectable marker. Due topresence of multiple copies of the chloroplast genome, it is more difficult to produce homogenous chloroplast mutants. In our experience, it takes 3 to 6 months longer and it requires higher concentrations of chloramphenicol. Moreover, the presence of WT and modified multiple copies of the chloroplasts genomes may lead to optimum heterogeneity, where a cell must maintain both versions to endure specific maximal concentration of chloramphenicol. The more important is the gene of interest, the more likely it is to occur.</p><p class="pptt" id="biaoti36363">### Recipes</p><ol class="npwtwok"><li>MA-1 (100 ml) (Table 1)<br/><br/><strong>Table 1. MA-1 buffer components </strong><br/><img alt="" class="layerphoto" height="111" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830041058_4121.jpg" width="500"/><br/><em>Note: Donât adjust pH. Autoclave and store sterile at RT, before transformation warm up to 40 Â°C.<br/><br/></em> </li><li>MA2 (Allen medium) (Table 2)<br/><br/><strong>Table 2. MA-2 buffer components</strong><br/><img alt="" class="layerphoto" height="243" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190902/20190902003945_7380.jpg" width="500"/><br/><em>Note: Set pH to 2.5, autoclave and store sterile at RT.</em><br/><br/></li><li>Agar-solidified plates (Table 3)<br/><br/><strong>Table 3. Agar-solidified plates components</strong><br/><img alt="" class="layerphoto" height="157" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190902/20190902005042_7139.jpg" width="500"/><br/><em>Note: Autoclave agar separately in water, combine with MA2 solution in sterile conditions when both solutions are cooled down to ca. 60 Â°C.</em><br/> </li><li>A2 Fe-EDTA solution (Table 4)<br/><br/><strong>Table 4. A2 Fe-EDTA solution components</strong><br/><img alt="" class="layerphoto" height="128" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190902/20190902005107_1072.jpg" width="500"/><br/><em>Note: First dissolve EDTA and set pH to 7, then add FeCl3, filter sterilize, store at 4 Â°C.</em><br/><br/></li><li>A2 trace element stock solution (Table 5)<br/><br/><strong>Table 5. A2 trace element stock solution components</strong><br/><img alt="" class="layerphoto" height="245" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190902/20190902005129_2999.jpg" width="500"/><br/><em>Note: Autoclave, store at 4 Â°C.</em><br/><br/></li><li>PEG 60% (w/v) (Table 6)<br/><br/><strong>Table 6. PEG 60% component</strong><br/><img alt="" class="layerphoto" height="96" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190902/20190902005153_4130.jpg" width="500"/><br/><em>Note: Prepare PEG at least a day earlier and allow to dissolve overnight, filter the solution with a syringe filter (0.2 Âµm). Store at RT.</em><br/><br/></li><li>Chloramphenicol (100 mg/ml) stock solution (Table 7)<br/><br/><strong>Table 7. Chloramphenicol (100 mg/ml) stock solution components</strong><br/><img alt="" class="layerphoto" height="95" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190902/20190902005216_4921.jpg" width="500"/><br/><em>Note: Prepare fresh every time, do not store.</em><br/></li></ol><p class="pptt" id="biaoti36364">### Acknowledgments</p><p class="ppzz">This protocol was adapted from Zienkiewicz <em>et al</em>. (2017a and 2017b). This study was supported by grant Opus 5 (DEC-2013/09/B/NZ1/00187) awarded by the Polish National Science Centre to M.Z.</p><p class="pptt" id="biaoti36365">### Competing interests</p><p class="ppzz">No competing interests declared.</p><p class="pptt" id="biaoti36366">### References</p><ol class="npwtwok"><li>Hallmann, A. (2007). <a href="https://pub.uni-bielefeld.de/record/1870248" target="_blank">Algal transgenics and biotechnology.</a> <em>Transgenic Plant J</em> 1(1): 81-98.</li><li>Imamura S, Terashita M, Ohnuma M, Maruyama S, Minoda A, Weber AP, Inouye T, Sekine Y, Fujita Y, Omata T and Tanaka K. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20375110" target="_blank">Nitrate assimilatory genes and their transcriptional regulation in a unicellular red alga <em>Cyanidioschyzon merolae</em>: genetic evidence for nitrite reduction by a sulfite reductase-like enzyme.</a> <em>Plant Cell Physiol</em> 51(5): 707-717.</li><li>Krupnik, T., Wasilewska, W., DroÅ¼ak, A., Romanowska, E. and Zienkiewicz, M. (2018). <a href="https://www.e-algae.org/journal/view.php?number=2877" target="_blank">Application of chloroplast promoters of <em>Cyanidioschyzon merolae</em> for exogenous protein expression.</a> <em>ALGAE</em> 33(4): 351-358. </li><li>Kuroiwa, T., Kuroiwa, H., Sakai, A., Takahashi, H., Toda, K. and Itoh, R. (1998). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9522454" target="_blank">The division apparatus of plastid and mitochondria.</a> <em>Int Rev Cytol</em> 181:1-41.</li><li>Matsuzaki, M., Misumi, O., Shin, I. T., Maruyama, S., Takahara, M., Miyagishima, S. Y., Mori, T., Nishida, K., Yagisawa, F., Nishida, K., Yoshida, Y., Nishimura, Y., Nakao, S., Kobayashi, T., Momoyama, Y., Higashiyama, T., Minoda, A., Sano, M., Nomoto, H., Oishi, K., Hayashi, H., Ohta, F., Nishizaka, S., Haga, S., Miura, S., Morishita, T., Kabeya, Y., Terasawa, K., Suzuki, Y., Ishii, Y., Asakawa, S., Takano, H., Ohta, N., Kuroiwa, H., Tanaka, K., Shimizu, N., Sugano, S., Sato, N., Nozaki, H., Ogasawara, N., Kohara, Y. and Kuroiwa, T. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15071595" target="_blank">Genome sequence of the ultrasmall unicellular red alga <em>Cyanidioschyzon merolae</em> 10D.</a> <em>Nature</em> 428(6983): 653-657.</li><li>Minoda, A., Sakagami, R., Yagisawa, F., Kuroiwa, T. and Tanaka, K. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15215501" target="_blank">Improvement of culture conditions and evidence for nuclear transformation by homologous recombination in a red alga, <em>Cyanidioschyzon merolae</em> 10D.</a> <em>Plant Cell Physiol</em> 45(6): 667-671.</li><li>Newell, C. A. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11098468" target="_blank">Plant transformation technology. Developments and applications.</a> <em>Mol Biotechnol</em> 16(1): 53-65.</li><li>Nozaki, H., Matsuzaki, M., Takahara, M., Misumi, O., Kuroiwa, H., Hasegawa, M., Shin-i, T., Kohara, Y., Ogasawara, N. and Kuroiwa, T. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12664168" target="_blank">The phylogenetic position of red algae revealed by multiple nuclear genes from mitochondria-containing eukaryotes and an alternative hypothesis on the origin of plastids.</a> <em>J Mol Evol</em> 56(4): 485-497.</li><li>Ohnuma, M., Misumi, O., Fujiwara, T., Watanabe, S., Tanaka, K. and Kuroiwa, T. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19533298" target="_blank">Transient gene suppression in a red alga, <em>Cyanidioschyzon merolae</em> 10D.</a> <em>Protoplasma</em> 236(1-4): 107-112.</li><li>Ohnuma, M., Yokoyama, T., Inouye, T., Sekine, Y. and Tanaka, K. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18003671" target="_blank">Polyethylene glycol (PEG)-mediated transient gene expression in a red alga, <em>Cyanidioschyzon merolae</em> 10D.</a> <em>Plant Cell Physiol</em> 49(1): 117-120.</li><li>Ohta, N., Matsuzaki, M., Misumi, O., Miyagishima, S. Y., Nozaki, H., Tanaka, K., Shin, I. T., Kohara, Y. and Kuroiwa, T. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12755171" target="_blank">Complete sequence and analysis of the plastid genome of the unicellular red alga <em>Cyanidioschyzon merolae</em>.</a> <em>DNA Res</em> 10(2): 67-77.</li><li>Ohta, N., Sato, N. and Kuroiwa, T. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9801318" target="_blank">Structure and organization of the mitochondrial genome of the unicellular red alga <em>Cyanidioschyzon merolae</em> deduced from the complete nucleotide sequence.</a> <em>Nucleic Acids Res</em> 26(22): 5190-5198.</li><li>Stark, M. R., Dunn, E. A., Dunn, W. S., Grisdale, C. J., Daniele, A. R., Halstead, M. R., Fast, N. M. and Rader, S. D. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25733880" target="_blank">Dramatically reduced spliceosome in <em>Cyanidioschyzon merolae</em>.</a> <em>Proc Natl Acad Sci U S A</em> 112(11): E1191-1200.</li><li>Walker, T. L., Collet, C. and Purton, S. (2005). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/j.1529-8817.2005.00133.x" target="_blank">Algal transgenics in the genomic era.</a> <em>J Phycol</em> 41(6): 1077-1093. </li><li>Watanabe, S., Ohnuma, M., Sato, J., Yoshikawa, H. and Tanaka, K. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21478650" target="_blank">Utility of a GFP reporter system in the red alga <em>Cyanidioschyzon merolae</em>.</a> <em>J Gen Appl Microbiol</em> 57(1): 69-72.</li><li>Zienkiewicz, M., Krupnik, T., Drozak, A., Golke, A. and Romanowska, E. (2017a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26715590" target="_blank">Chloramphenicol acetyltransferase-a new selectable marker in stable nuclear transformation of the red alga <em>Cyanidioschyzon merolae</em>.</a> <em>Protoplasma</em> 254(1): 587-596.</li><li>Zienkiewicz, M., Krupnik, T., Drozak, A., Golke, A. and Romanowska, E. (2017b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27796719" target="_blank">Transformation of the <em>Cyanidioschyzon merolae</em> chloroplast genome: prospects for understanding chloroplast function in extreme environments.</a> <em>Plant Mol Biol</em> 93(1-2): 171-183.</li><li>Zienkiewicz, M., Krupnik, T., Drozak, A., Wasilewska, W., Golke, A. and Romanowska, E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29196904" target="_blank">Deletion of psbQ' gene in <em>Cyanidioschyzon merolae</em> reveals the function of extrinsic PsbQ' in PSII.</a> <em>Plant Mol Biol</em> 96(1-2): 135-149.<br/></li></ol>