<body>
            <h1>Yeast Single-cell RNA-seq, Cell by Cell and Step by Step</h1>
            <p><strong>Authors:</strong> Mariona Nadal-Ribelles, Saiful Islam, Wu Wei, Pablo Latorre, Michelle Nguyen, EulÃ lia de Nadal, Francesc Posas, Lars M. Steinmetz</p>
            <h2>Abstract</h2>
            <p>Single-cell RNA-seq (scRNA-seq) has become an established method for uncovering the intrinsic complexity within populations. Even within seemingly homogenous populations of isogenic yeast cells, there is a high degree of heterogeneity that originates from a compact and pervasively transcribed genome. Research with microorganisms such as yeast represents a major challenge for single-cell transcriptomics, due to their small size, rigid cell wall, and low RNA content per cell. Because of these technical challenges, yeast-specific scRNA-seq methodologies have recently started to appear, each one of them relying on different cell-isolation and library-preparation methods. Consequently, each approach harbors unique strengths and weaknesses that need to be considered. We have recently developed a yeast single-cell RNA-seq protocol (yscRNA-seq), which is inexpensive, high-throughput and easy-to-implement, tailored to the unique needs of yeast. yscRNA-seq provides a unique platform that combines single-cell phenotyping via index sorting with the incorporation of unique molecule identifiers on transcripts that allows to digitally count the number of molecules in a strand- and isoform-specific manner. Here, we provide a detailed, step-by-step description of the experimental and computational steps of yscRNA-seq protocol. This protocol will ease the implementation of yscRNA-seq in other laboratories and provide guidelines for the development of novel technologies.</p>
            <p><strong>Keywords:</strong> Yeast, Single-cell RNA-seq, Transcriptomics, Transcript isoforms, Noncoding RNA</p>
            <h2>Background</h2><p>The appearance of single-cell-omics has revolutionized our understanding of many biological processes and is a field that is rapidly evolving at both the experimental and computational level. Over the last decade, there has been a rapid increase in the number of features that can be measured within individual cells. Single-cell RNA-seq (scRNA-seq) has pioneered this endeavor, and it has become a routine experiment to perform in higher eukaryotes. While very close to the entirety of the repertoire of bulk experiments in higher eukaryotes now have single-cell counterparts, single-cell tools for microorganisms (beyond fluorescent reporter-based methods) are scarce. This technological gap is mainly due to technical limitations imposed by the intrinsic nature of yeast and other microorganisms.<br/><br/>Unlike the mammalian cells, in which scRNA-seq was developed, yeast cells are smaller in cell, genome and transcriptome size. The presence of a rigid cell wall, which needs to be removed prior to library preparation, has been one of the major technical challenges for single cell analysis. On top of that, the yeast genome is highly condensed and pervasively transcribed. Over 85% of their genome is expressed from both strands with extensive transcript isoform diversity per gene (David <em>et al</em>., 2006; Pelechano <em>et al</em>., 2013, 2014 and 2015), and this requires a sensitive, quantitative and strand-specific approach.<br/><br/>As was the case for higher eukaryotes, initial studies exploring scRNA-seq in yeast focused on the development and optimization of the method. Recently, three yeast single-cell RNA-seq approaches have been reported (Gasch <em>et al</em>., 2017; Nadal-Ribelles <em>et al</em>., 2019; Saint <em>et al</em>., 2019). Interestingly, each one of them uses a different cell-isolation strategy and library-preparation protocol. Cell isolation is one of the main differences across these studies that range from microfluidics (Gasch <em>et al</em>., 2017), micromanipulation (Saint <em>et al</em>., 2019) and index sorting (Nadal-Ribelles <em>et al</em>., 2019), the later described here. Microfluidic and micromanipulation-based approaches require specialized equipment and are labor intensive, but allow imaging of individual cells by microscopy as a way to measure phenotype. The protocol described here for yeast single-cell RNA-seq (yscRNA-seq), relies on index sorting as a strategy for recording phenotypic information such as cell morphology and any Fluorescent-Activated Cell-Sorting-compatible feature (fluorescent proteins, antibodies, <em>etc.</em>). Index sorting provides a seamless strategy for merging techniques for individual phenotyping and transcriptional profiling in cells.<br/><br/>A major tipping point for single-cell isolation was the appearance of droplet-based methodologies (Zhang <em>et al</em>., 2019). These technologies dramatically increase potential throughput by generating and isolating cDNA in barcoded gel emulsions, which encapsulate single cells on the order of thousands of cells per run, and thus the cost significantly decreases. Interestingly, a preliminary droplet-based study in yeast is underway (Gresham <em>et al</em>., 2019). While cell capturing boosts cell numbers to an unprecedented scale, the number of genes detected per cell tends to be much lower (Skinnider <em>et al</em>., 2019) and phenotypic information of individual cells is lost. In addition, customization of library preparation is complex and labor intensive in droplet based-approaches, which will most likely be bypassed as technologies are developed.<br/><br/>While cell-isolation strategies determine the number of cells and phenotypic information that can be recorded, the choice of library-preparation approach defines the kind of transcriptional profiling. Due to the intertwined nature of the yeast transcriptome, and the diversity of isoforms per gene, library preparation requires both gene- and strand-isoform-specific resolutions. YscRNA-seq is based on STRT-seq (Islam <em>et al</em>., 2014), and its library design bypasses the technical limitations of working with yeast, fulfilling the requirements for a high-resolution transcriptome profiling method. First, unique molecular identifiers (UMI) are incorporated during cDNA synthesis via a biotinylated template-switching oligo (TSO) at the 5â²-end of the molecule to digitally count the absolute abundance of a gene, along with its transcription start site (TSS) position. Second, the use of a homemade Tn5 enzyme to incorporate cell-specific adaptors provides a cost-effective strategy that greatly reduces experimental costs, compared to commercial Tn5 (Hennig <em>et al</em>., 2018). Finally, biotinylated primers allow selectively recovering 5â² ends. Moreover, the high affinity between streptavidin and biotin leads to a sequestration of the biotinylated strand after a brief denaturing step, which releases the non-biotinylated strand into the supernatant for sequencing. These features make yscRNA-seq one of the most sensitive methods available, and the most sensitive method for yeast reported to date (Nadal-Ribelles <em>et al</em>., 2019).<br/><br/>We anticipate that the development of novel methodologies with different aims will soon start to emerge, and yscRNA-seq will be extended to any yeast species or microorganism. The protocol described here, along with future optimizations, could serve as a starting point for the development of new methods. There are many unanswered biological questions that influence our understanding of human health, such as the emergence of drug-resistant microbial phenotypes. Generation of solid frameworks to profile microorganisms at single-cell resolution promises to expand our understanding, and perhaps answer some of these questions once and for all.<br/></p>
        </body>

<p class="pptt" id="biaoti29156">## Materials and Reagents</p><p class="ppzz"><em>Note: Reagents can be from different suppliers as far as they are only used for single-cell RNA-seq protocols and nuclease free. The ones listed here were used to develop the protocol. The regular molecular biology reagents are assumed to be already present in each laboratory (e.g., water, Tris, or NaCl). For plastic labware (filter tips and tubes), we have used several brands, with identical results as far as the material is certified to be nuclease-free and low binding.</em> </p><br/><ol class="npwtwok"><li>Eppendorf LoBind 1.5 ml (Eppendorf, catalog number: 30108051)</li><li>Break-away plates (Thomas Scientific, catalog number: EK-75118)</li><li>96-well plates (Thomas Scientific, catalog number: EK-75012)</li><li>Filter Tips (Mettler Toledo, catalog numbers: 17007954, 17014973, 17014973)</li><li>qPCR plates (Applied Biosystems, catalog number: 4309849)</li><li>Universal PCR plate seal (Sigma-Aldrich, catalog number: Z742420-100EA)</li><li>UMI_Oligo dT_T31 (100 Î¼M) (Integrated DNA Technologies)</li><li>UMI_TSO6 (Integrated DNA Technologies)</li><li>STRT-adaptors 96 different oligos (96 well plate Integrated DNA Technologies)</li><li>UMI PCR (96 well plate Integrated DNA Technologies</li><li>dNTP 25 mM (Thermo Fisher, catalog number: R0181)</li><li>50x Advantage 2 Polymerase (Takara, catalog number: 639202)</li><li>PvuI (Cutsmart 10x provided) (New England Biolabs, catalog number: R3150L)</li><li>Dynabeads<sup>TM</sup> MyOne<sup>TM</sup> Streptavidin C1 (Thermo Fisher, catalog number: 65001)</li><li>Zymolyase 100T (100 mg/ml) (US Bio, catalog number:37340-57-1)</li><li>RNase Zap (Thermo Fisher, catalog number: AM9780)</li><li>RNase Inhibitor (40 U/ml) (Takara, catalog number: 2313A) </li><li>RNase-nuclease free water (Thermo Fisher, catalog number: 10977035)</li><li>ERCC RNA spike ins (Thermo Fisher, catalog number: 4456740)</li><li>TAPS (Sigma-Aldrich, catalog number: T5316)</li><li>DMF (Sigma-Aldrich, catalog number: 74438)</li><li>5x Superscript First strand buffer (Thermo Fisher, catalog number: 18064014)</li><li>MgCl<sub>2</sub> (1 M) (Thermo Fisher, catalog number: AM9530G)</li><li>Betaine (5 M) (Sigma-Aldrich, catalog number: 61962)</li><li>Superscript II (Thermo Fisher, catalog number: 18064014)</li><li>10x Advantage 2 PCR Buffer (Takara, catalog number: 639202)</li><li>Tris (Sigma-Aldrich, catalog number: T2319)</li><li>Tween-20 (Sigma-Aldrich, P9416-50ML)</li><li>Glycerol (Sigma-Aldrich, catalog number: G5516)</li><li>KAPA library quantification (Kapa Biosystems, catalog number: KR0405)</li><li>Ampure beads (Beckman Coulter, catalog number: A63881)</li><li>TE 10x (Sigma-Aldrich, catalog number: T9285)</li><li>NaCl (Sigma-Aldrich, catalog number: S5150)</li><li>EDTA (Thermo Fisher, catalog number: AM9261)</li><li>Elution Buffer (EB) (Qiagen, catalog number: 19086)</li><li>PB Buffer (PB) (Qiagen, catalog number: 19066)</li><li>Sybergreen 2x mastermix (Invitrogen, catalog number: 4309155)</li><li>LNA primers (Exiqon)</li><li>Primers for qPCR<br/><ol class="npwthree"><li>SOMN17 Fw_TDH3_probe: TCGTCAAGTTGGTCTCCTGG</li><li>SOMN18 Rv_TDH3_probe: GGCAACGTGTTCAACCAAGT</li><li>SOMN21 Fw_ADH1_probe: TGGTGCCAAGTGTTGTTCTG</li><li>SOMN22 Rv_ADH1_probe: GGCGAAGAAGTCCAAAGCTT</li><li>SOMN310 Fw_5_ERCC_00130: CGGAAAAGTACTGACCAGCG</li><li>SOMN311 Rv_5_ERCC_00130: TGCCAATGACTTCAGCTGACæ ·å¼</li></ol></li><li>Optical lids for qPCR (Applied Biosystems, catalog number: 4311971)</li><li>DNA High Sensitivity CHIP (Agilent, catalog number: 5067-4626)</li><li>1% Triton X-100 (Sigma-Aldrich, catalog number: X100-1L)</li><li>100 mM DTT (Thermo Fisher, catalog number: 18064014)</li><li>Propidium Iodide (PI) (Sigma Aldrich, P4170-10MG)</li><li>Cell capturing and lysis solution (see Recipes)</li><li>2x BWT Buffer (see Recipes)</li><li>TNT Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti29157">### Equipment</p><ol class="npwtwok"><li>96-well plate magnet (Thermo Fisher, catalog number: 12331D)</li><li>1.5-2 ml tube magnet (Thermo Fisher, catalog number: 12303D)</li><li>Bioanalyzer (Agilent)</li><li>HiSeq 2000 (Illumina)</li><li>FACS (BD Influx, or Aria II, nozzle 70 and 100 microns)</li><li>qPCR (Applied Biosystems) </li><li>Qubit (Thermo Fisher, catalog number: Q32854)</li><li>Thermocycler (any vendor)</li><li>Thermomixer (any vendor)</li><li>Multichannel pipettes (any vendor)</li><li>Plate centrifuge (any vendor)<br/></li></ol><p class="pptt" id="biaoti29158">### Software</p><ol class="npwtwok"><li>Novocraft (Novocraft Technologies Sdn Bhd, <a href="http://www.novocraft.com/" target="_blank">http://www.novocraft.com/</a>)</li><li>Samtools v1.3.1 (Li <em>et al</em>., 2009, <a href="http://samtools.sourceforge.net/" target="_blank">http://samtools.sourceforge.net/</a>)</li><li>R programming language v.3.5.0 (R Core Team, 2019, <a href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a>)</li><li>Genomic Alignments R package v.1.18.1 (Lawrence <em>et al</em>., 2013)<br/></li></ol><p class="pptt" id="biaoti29159">### Procedure</p><p class="ppzz"><em>Note: If possible, generate a single-cell library preparation space in the laboratory. If not, wash the surface of the bench and all the material with RNase Zap at the beginning and end of each library. All material and reagents are handled with gloves and the standard precautions for RNA work should be taken (RNase free material and filter tips).</em> </p><br/><ol class="npwone"><li>Cell growth<br/>On the day before sorting, grow the desired pre-inoculum of your desired yeast strain in their corresponding media overnight (O/N). To profile exponentially growing cells, we recommend the initial culture not to grow over optical density OD<sub>660</sub> = 1.<br/></li><br/><li>Cell sorting<br/><ol class="npwtwo"><li>The next morning (sorting day), dilute cells to OD<sub>660</sub> = 0.05 in the corresponding media and allow for at least 2 cell divisions (3 h approximately for wild type strains) prior to cell isolation.</li><li>Prepare 96- or 384-well plates containing 5 Âµl absolute ethanol in each well to fix cells immediately for sorting.<br/><em>Notes: <br/><ol class="npwthree"><li>During protocol optimization, we recommend using break-away plates. These plates allow breaking 96 well plates by rows, and thus several tests can be done using the same plate. Check with your facility the compatibility of the plates.</li><li>We have obtained the same results sorting cells directly into 5 Âµl of âCell capturing and lysis solutionâ (see below). If doing so, prepare plates right before sorting and keep them on 4 Â°C ice.</li></ol></em> </li><li>Dilute cells prior to sorting to OD = 0.05 in 3 ml of growth media and vortex vigorously to separate cell clumps.<br/><em>Note: At this step, propidium iodide (PI) (4 Âµg/ml) can be added to check for cell viability. Adjust the culture volume to your needs. In our experience, 3 ml is enough to sort at least 10 plates.</em> </li><li>At the FACS facility, filter cells with Cell Strainer Tubes (check with your facility which tubes they prefer) and put cells in the appropriate sorting tube for live single-cell sorting.</li><li>Check the alignment of the plate with the sorter. For example, this can be done by sorting a drop into a covered plate, and ensuring that the droplet would fall into the center of each well.</li><li>Sort live single yeast (propidium iodide (PI) negative) into each well of the plates, being sure to leave one well as a negative control. We index-sorted from the population using the forward and side scatter (FSC and SSC respectively). <br/><em>Note: To include a positive control, sort 100 cells into one well.</em> </li><li>Cover plates with Universal PCR plate seal.</li><li>Quick spin plates to collect cells at the bottom of each well (short spin to collect all liquid to the bottom of the wells).</li><li>Let the ethanol evaporate in a sterile environment (<em>i.e.</em>, sterile hood) for no more than 45 min.</li><li>Once the ethanol is completely evaporated, add 5 Âµl yeast âCell capturing and lysis solutionâ. Spin down and freeze immediately.<br/><em>Note: Regardless of whether the cells are sorted into ethanol or directly into âCell capturing and lysis solutionâ (see Recipes), frozen plates can be stored at -80 Â°C for at least 6 months. ERCCs are spike in RNAs that provide an accurate measure of technical noise, while we recommend using them, they can be excluded.</em> </li><li>Perform the following lysis cycle from freshly-sorted or frozen plates (Table 1).<br/><br/><strong>Table 1. Temperature conditions for cell wall digestion and cell lysis</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830010757_6410.jpg" width="420"/><br/><em>Note: Digestion can be extended up to 30 min.</em><br/><br/><div><span style="font-size:10pt;"></span> </div></li><li>Immediately proceed to add the RT reaction for first strand cDNA synthesis. Add 5 Âµl Reverse Transcription mix (RT mix) (Table 2).<br/><br/><strong>Table 2. Master mix reagents for first strand cDNA synthesis</strong><strong></strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005613_5472.jpg" width="420"/><br/><br/></li><li>Spin down the plate and perform the following cycles (Table 3):<br/><br/><strong>Table 3. Incubation temperatures for cell lysis</strong><strong></strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005630_6526.jpg" width="420"/><br/><br/></li><li>Add 15 Âµl PCR mix for library amplification to each well (Table 4).<br/><br/><strong>Table 4. Master mix reagents for library amplification</strong><strong></strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005643_4240.jpg" width="480"/><br/><br/></li><li>Spin down the plate and do the following cycles (Table 5):<br/><br/><strong>Table 5. PCR cycling conditions for library amplification</strong><br/><div><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005656_0788.jpg" width="480"/><br/></div><em>Note: This is a safe stopping point (at 4 Â°C O/N, or frozen -20 Â°C for 1-2 months).</em><em></em><br/><br/></li><li>(Optional) Dilute the amplified library 1:20 and check the percentage of positive cells from Step B15 (Figure 1; Tables 6 and 7).<br/><em>Note: Use primers against a housekeeping gene (we use TDH3 but we have obtained same results with ADH1) to measure the number of positive cells per well. Use primers against ERCC as a positive control for amplification that should be even across all reactions. This step is especially useful during the protocol set up as it allows inspecting the samples/libraries before moving forward (check for the number of positive libraries). See <a href="https://os.bio-protocol.org/attached/file/20190808/Table%201%20primers%20Protocol%201902586.xlsx" target="_blank">Table S1</a> for primer sequences.</em><br/><div><br/></div><strong>Table 6. Master mix for qPCR assessment</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005710_1605.jpg" width="400"/><br/><em>Note: Primer mix refers to the mixture of Fw and Rv primers (10 ÂµM each) diluted in TE 1x.</em><br/><br/><strong>Table 7. PCR cycling conditions for qPCR<br/></strong><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005736_8968.jpg" width="500"/><br/><br/>Primer sequences for qPCR:<br/>SOMN17 Fw_TDH3_probe: TCGTCAAGTTGGTCTCCTGG<br/>SOMN18 Rv_TDH3_probe: GGCAACGTGTTCAACCAAGT <br/>SOMN21 Fw_ADH1_probe: TGGTGCCAAGTGTTGTTCTG<br/>SOMN22 Rv_ADH1_probe: GGCGAAGAAGTCCAAAGCTT<br/>SOMN310 Fw_5_ERCC_00130: CGGAAAAGTACTGACCAGCG<br/>SOMN311 Rv_5_ERCC_00130: TGCCAATGACTTCAGCTGAC<br/><em>Note: A good plate will have around 70% positive wells considering the Ct values of the housekeeping gene. Plates with less than 50% positive cells are rare. To determine if low efficiency is due to sorting or due reaction efficiency, perform a qPCR using a 1:10 dilution of the cDNA library against ERCCs. Failure to amplify ERCCs, or uneven amplification (represented by wildly different Ct values), is indicative of incorrect library preparation. In the case of low number of positive cells per plate, you can generate a new plate by combining positive from different plates into a new plate before proceeding to tagmentation and try to improve sorting efficiency.</em><br/><div><br/></div><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830004712_9475.jpg" width="200"/><br/><strong>Figure 1. Representative example of Ct Values obtained by qPCR.</strong> The scatter plot represents the cycle amplification (Ct value) of a yscRNA-seq 96-well plate. Each dot represents the value obtained using a yeast housekeeping gene (<em>TDH3</em>, x-axis) as a function of the index-sorting value for cell size (Forward Scatter (FSC), y-axis). Dotted line (Ct &gt; 25) displays the threshold used to discriminate positive and negative wells. The label displays the Ct value for the not sorted well H<sub>12 </sub>(which is used as a negative control). <br/><br/></li><li>Add 15 Âµl of room temperature equilibrated and well homogenized Ampure XP beads (1:0.6 sample/ bead ratio) to each well.<br/><em>Note: Do not increase the volume of beads in the purification step above the 1:1 ratio. A less-than-standard amount of beads ensures that primer dimer carryover is minimal.</em> </li><li>Mix by pipetting up and down ten times, or until the solution appears to be homogeneous. Transfer solutions to a 96-well plate with compatible magnet stand.</li><li>Incubate the mixture for 10 min at room temperature to let the DNA bind to Ampure XP beads.</li><li>Place the 96-well plate on the magnetic stand for 5 min, or until the solution is clear and beads have been collected.</li><li>While samples are on the magnet, carefully remove the liquid without disturbing the beads.</li><li>Wash magnet-bound beads with 200 Âµl of 80% (vol/vol) ethanol solution. Incubate the samples for 30 s and then remove the ethanol with the tube in the magnet, do not overdry the beads.<br/><em>Note: It is important that the ethanol solution is freshly prepared every time, as ethanol absorbs moisture from the environment, thus changing the final concentration. Read and follow the manufacturerâs instructions.</em> </li><li>Repeat the ethanol wash one more time (repeat Steps B20-B22).</li><li>Remove any trace of ethanol and let beads dry completely by leaving the plate at room temperature for 5 min or until ethanol evaporates.<br/><em>Note: Cover the plate during this step or protect it from any possible source of contamination or airflows.</em> </li><li>Once there is no ethanol left, elute dscDNA libraries from Ampure XP beads with 16.5 Âµl elution buffer at room temperature (EB buffer Qiagen). </li><li>Remove the plate from the magnet and mix vigorously by pipetting up and down three times or until the solution becomes homogeneous.</li><li>Place the plate on a magnetic stand and leave it for 2 min, or until the solution appears clear. </li><li>Recover 15 Î¼l of supernatant from each well and transfer to a new plate. Label the plate correctly, as it will be stored.<br/><em>Safe stopping point: cDNA libraries can be stored at -20 Â°C before proceeding to tagmentation for up to 2 weeks.</em> </li></ol><br/></li><li>Full-length cDNA library quality check<br/><ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Run 1 Âµl of dscDNA libraries (Step B28) to check the size distribution and estimate of concentrations using a High Sensitivity DNA ChIP (2100 Bioanalyzer). If available, use the qPCR results from Step B16 to guide the selection of wells for Bioanalyzer (Figure 2).<br/><br/><img alt="" class="layerphoto" height="51" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005302_9141.jpg" width="300"/><br/><strong>Figure 2. Representative Bioanalyzer traces of full-length cDNA obtained with yscRNA-seq (Step C1).</strong> cDNA libraries obtained from step (Step B28) were run on a DNA High Sensitivity CHIP (Agilent 5067-4626) for validation. Library concentration was also measured by Qubit High Sensitivity (Thermo Fisher). Left panel (A1) represents the lower limit of library quality that we sequenced while middle (A5) and right (B2) panel represent average libraries (Figure adapted from Nadal-Ribelles <em>et al</em>., 2019)<br/><br/><strong></strong><p class="ppzz" style="margin-left:-20px;"><strong>Adaptor annealing:</strong> In order to generate cell-barcoded libraries, Tn5 needs to be loaded with double stranded DNA (dsDNA) adapters. To do so, 96 different dsDNA adapters need to be annealed.</p></li><li>Thaw the plate with the 96 STRT barcodes (100 ÂµM) in ice.</li><li>In a new plate, mix 5 Î¼l UMI-TN5-U (100 ÂµM) and 5 Î¼l UMI-TN5_1 to 96 (ÂµM) in TE 1x to final concentration 50 ÂµM (each), a 1:1 dilution.</li><li>Anneal primers by heating the mix at 95 Â°C for 3 min and gradually cool down to room temperature (0.5 Â°C/s). Label the plate as âAnnealed cell-specific adaptersâ plate. <br/><em>Note: Label plates with the amount of annealed primer and the annealing date. Plates can be prepared in advance and stored at -20 Â°C for up to six months. The amount of annealed adaptors depends on the number of plates that need to be tagmented and the frequency of usage. </em> </li><li>To load cell specific adapters to Tn5, prepare a new plate and label it as âloaded Tn5â. In each well, mix the following reagents. We recommend to make a mix with all reagents and add annealed cell-specific adapters individually (Table 8). <br/><br/><strong>Table 8. Master mix to generate 10x transposome</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005756_9168.jpg" width="550"/><br/><br/></li><li>Load Tn5 by incubating at 37 Â°C for one hour and immediately cool to 4 Â°C. Freeze the âloaded Tn5â plate at -20 Â°C.<br/><em>Note: Loaded Tn5 plate can be safely stored for 1-2 weeks at -20 Â°C. Caution, store the plate immediately after use leaving the loaded plate on ice 4 Â°C will significantly reduce Tn5 activity and will result in inefficient tagmentation.</em><br/><strong><br/><p class="ppzz" style="margin-left:-20px;">Tagmentation</p></strong> </li><li>Prepare the following mix in a new plate in ice (Table 9).<br/><br/><strong>Table 9. Master mix for library tegmantation</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830014910_7774.jpg" width="420"/><br/><br/></li><li>Incubate at 55 Â°C for 5 min and 3 min at 85 Â°C to inactivate Tn5, then cool to 4 Â°C in a thermocycler. Tagmentation time depends on Tn5 purification efficiency and activity. We have observed that the enzyme loses activity over time. Titration of each batch of Tn5 is strongly suggested by tagmenting the same amount cDNA (Steps B28 and C1) with increasing concentration of loaded Tn5 (Step C6) and check fragmentation profile in a Bioanalyzer.</li><li>Prepare a 1:20 dilution of <em>MyOne</em> C1 Streptavidin per each sample.</li><li>Wash <em>MyOne</em> beads 2 x with 2x BWT buffer and dilute with 20x more volume than the original volume of beads with 2x BWT. <br/><em>Example: 20 Âµl beads for 20 samples will be finally diluted with 400 Âµl 2x BWT.</em> </li><li>Add 20 Âµl of diluted <em>MyOne</em> C1 Streptavidin beads (step C10) to each well and incubate at RT for 5 min at room temperature.</li><li>Pool all samples per plate (up to 96) into a single collecting tube (1.5 or 2 ml).</li><li>Place collecting tube in magnetic rack and allow enough time for the solution to be completely clear.</li><li>While on the magnet, wash beads once with 100 Âµl of TNT buffer do not mix the beads.</li><li>While on the magnet, wash <em>MyOne</em> C1 Streptavidin once with 100 Âµl PB Buffer and discard the supernatant.</li><li>While on the magnet, wash beads 3 x with 100 Âµl TNT buffer again and the discard supernatant.<br/><br/><strong><p class="ppzz" style="margin-left:-30px;">Remove 3Â´ ends</p></strong> </li><li>Add 100 Âµl of the following mix to the washed beads from Step C14 (Table 10). <br/><br/><strong>Table 10. Mastermix for 3Â´ end removal</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005836_2281.jpg" width="450"/><br/><br/></li><li>Incubate mix for 1 h at 37 Â°C in a thermomixer. Mix every 2 min for 30 s at 1,000 rpms, to avoid bead sedimentation.</li><li>Wash beads 3 x with 100 Âµl of TNT buffer. <br/><br/><strong><p class="ppzz" style="margin-left:-30px;">Elute single stranded cDNA</p></strong> </li><li>Resuspend in 30 Âµl Nuclease-free water.</li><li>Incubate 10 min at 70 Â°C, 850 rpm mix in a thermomixer.</li><li>Immediately bind beads to the magnet and transfer the supernatant to a new tube, which contains the single strand cDNA library in the supernatant (the other strand remains bound to the streptavidin beads).<br/><br/><strong></strong><p class="ppzz" style="margin-left:-30px;"><strong>Single-strand cDNA cleanup</strong> </p></li><li>Add 54 Âµl of room temperature Ampure XP beads to 30 Âµl sscDNA library.</li><li>Incubate for 10 min at RT.</li><li>Bind beads to the magnet for 1 min or until the solution is completely clear and discard supernatant (keep the beads).</li><li>Wash once with 200 Âµl fresh 80% ethanol for 20-30 s. Perform this step with the beads bound to the magnet.</li><li>Air dry beads for approximately 2 min.</li><li>Resuspend in 30 Âµl EB buffer and incubate for 5 min at RT.</li><li>Bind beads for 2 min, or until the solution is clear, and transfer the supernatant to a new tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Assess library concentration<ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Set up a KAPA quantification reaction with a 1:100, 1:1,000 and 1:10,000 dilutions of the eluted cDNA library (Step C27). All regents except for your DNA library are provided in the kit (Table 11).<br/><em>Note: This kit can be substituted by your favorite quantification method or by a qPCR using P5-P7 primer pairs with known standards (PhiX is strongly recommended) SYBRGreen 2x mastermix.</em><br/><strong><br/>Table 11. Master mix for qPCR library quantification</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830010000_6928.jpg" width="450"/><br/><br/></li><li>qPCR cycling conditions for KAPA and homemade Sybergreen (Table 12).<br/><strong><br/>Table 12. PCR cycling conditions for qPCR<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830010020_4309.jpg" width="400"/><br/></strong><br/></li><li>Use the qPCR to calculate library quantification using the template provided by KAPA biosystems or the instructions provided from your manufacturer.<br/><em>Note: We have used KAPA, NEB and homemade systems with similar results.</em><br/></li><li>Set up a separate PCR to run a bioanalyzer to determine the final size distribution. Prepare the following mix, one separate reaction per each library to be loaded (Table 13).<br/><br/><strong>Table 13. Master mix for to assess library size after tagmentation</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830010035_9742.jpg" width="450"/><br/><br/></li><li>Run the same PCR as in Step D2 but for 11 cycles.</li><li>Run 1 Âµl into a High sensitivity DNA CHIP to obtain an average library size based on the Bioanalyzer profile (Figure 3). <br/><br/><img alt="" class="layerphoto" height="73" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005416_0662.jpg" width="300"/><br/><strong>Figure 3. Representative Bioanalyzer traces obtained from yscRNA-seq.</strong> Two representative samples obtained from approximately 80 cells. Concentration of each library is shown and was determined by qPCR (Step D3). Figure from Nadal-Ribelles <em>et al</em>., 2019. <br/><br/></li><li>Sequence the library on the HiSeq 2000 High output using the custom <em>Read 1</em> primer and UMI-TN5-U as the <em>Index read</em> primer.</li><li>To run the libraries on the HiSeq rapid run, use lock nucleic acid (LNA) primers. Spike in the primer at 0.5 ÂµM.<br/>Index 1 primer into HP8<br/>Read 1 primer into HP9<br/><em>Notes:<br/><ol class="npwthree"><li>Double-check this information with your sequencing kit/instrument and/or sequencing core facility.</li><li>For a High Output Run, custom primers are required as well, but without LNA due to differences in sequencing chemistry.<br/></li></ol><br/></em>UMI_PCR_read1: +GAATGA+TACGGCG+ACCA +CCGA+T â custom 250 nmole. DNA oligo, HPLC Purification <br/>Index1: CTGT+CT+CTT+ATA+CA +CA+TCTGA+CG+C â custom 250 nmole DNA oligo, HPLC Purification<br/><br/></li><li>Load around 8-14 pmol of each library per lane. Libraries are single-stranded DNA, thus, no denaturing is required.<br/><em>Note: Once the protocol is optimized, there is no need to run a PhiX control, if PhiX is loaded take into consideration a denaturing step for the double-stranded PhIX control, which is not required for yscRNA-seq libraries.<br/></em><br/></li></ol>A schematic representation of the plate processing is shown in Figure 4 as a reference. <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005443_4396.jpg" width="200"/><br/><strong>Figure 4. Schematic representation of yscRNA-seq.</strong> Images from Smart Medical server (Les Laboratoires Servier, <u><a href="https://smart.servier.com/" target="_blank">SMART Servier Medical Art.</a></u>).</li></ol><p class="pptt" id="biaoti29160">### Data analysis</p><p class="ppzz">The initial steps of yscRNA-seq data analysis are very similar to those applied for bulk RNA-seq (Conesa <em>et al</em>., 2016), but with certain particularities. YscRNA-seq generates two reads per sequenced cDNA molecule. The first read (Index 1) contains the cell-specific barcode that associates a given molecule with a specific cell. The second read (Read 1) contains the UMI, and maps to the TSS of a specific transcript, allowing for absolute molecule-counting and transcript identification, respectively. To properly estimate gene expression from yscRNA-seq data, both reads have to be computationally associated to their respective references. The former association is performed by default in Illuminaâs <em>BaseSpace</em> demultiplexing tool. This generates individual FASTQ files (one per each yeast cell) that are the starting point of yscRNA-seq analysis. After this, we further pre-process the FASTQ files to identify the UMIs and perform the alignment and quantification to obtain the gene expression table (Figure 5). Here, we detail the steps that we follow for analyzing yscRNA-seq data, starting from FASTQ files: <br/><em>Note: The analysis pipeline was executed in a Linux 64 bit machine with 32 Gb of RAM and 6 CPUs. We recommend to have at least 12 Gb of RAM. A machine with several computing cores is also preferable for speeding up the processes.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830005516_8638.jpg" width="200"/><br/><strong>Figure 5. Schematic representation of analysis pipeline of yscRNA-seq<br/></strong><strong><br/></strong> </p><ol class="npwone"><li>Read pre-processing<br/>To extract the UMI from the beginning of the reads, we used a custom JavaScript program (<a href="http://steinmetzlab.embl.de/yscRNASeq/code/deBCSCell.tar.gz" target="_blank">deBCSCell</a>). <br/><br/><u>Command</u>: <span style="font-family:Courier New;"><em>$java -cp directoryOfdeBCSCell rootNameOfdeBCSCell F1=fastqFile BL="â</em></span><br/><br/>This script performs the following actions: <br/><ol class="npwtwo"><li>Extracts the first 6nt (UMI) and incorporates them into the read name for later use.</li><li>Extracts and counts Nâs and Gâs following the UMI, up to 14. The read is discarded if it contains 14 or more Nâs or Gâs.</li></ol><br/></li><li>Alignment<br/><p class="ppzz"><em>Note: In our case, the reference sequences (referenceSequencesName.fa) are a combination of the S. cerevisiae genome (Saccer3, SGD R64 version; <a href="http://www.yeastgenome.org/" target="_blank"><u>www.yeastgenome.org</u></a>) and the ERCC control sequences. The ERCC control sequences provided by the manufacturer do not include the restriction sites used during the cloning of the sequences downstream of the T7 promoter. Since yscRNA-seq is TSS-specific, this fact considerably decreases the mapping rate due to misalignments at the beginning of the read. To overcome that, we generated a new ERCC reference that includes these sequences. These modified references could be downloaded from the Gene Expression Omnibus (under <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122392" target="_blank">GSE122392</a> accession number). <br/></em></p><em> </em> <ol class="npwtwo"><li><em>In our case, we have sequenced each library to a median depth of ~720,000 reads. We assessed, with down-sampling, that this depth is substantially above the needed to obtain a high resolution of the yeast transcriptome. In fact, at ~500,000 reads per cell in the number of transcripts detected saturation is reached (Supplementary Figure 2C [Nadal-Ribelles <em>et al.</em>, 2019]). In addition, we observe a median of ~74% uniquely mapped reads for all sequenced libraries. These values are useful reference values using this protocol.</em> </li><li><em>We used Novocraft for the alignment of the reads, but open-source aligners such as <a href="https://ccb.jhu.edu/software/hisat2/index.shtml" target="_blank">HISAT2</a> (Kim <em>et al.</em>, 2015) or <a href="https://github.com/alexdobin/STAR" target="_blank">STAR</a> (Dobin <em>et al.</em>, 2013) can be used without obtaining major differences in the results. </em><br/><div style="white-space:nowrap;"><br/></div></li></ol>Reads were aligned with <em>Novocraft</em> (<a href="http://www.novocraft.com/" target="_blank">http://www.novocraft.com/</a>) using default parameters.<br style="color:#666666;font-family:Arial, Helvetica, sans-serif;font-size:13.3333px;text-align:justify;white-space:normal;"/><ol class="npwtwo" style="font-size:13.3333px;color:#666666;white-space:normal;"><li>Create the index for Novocraft using novoindex command:<br/><br/><span style="font-size:10pt;"><em><span style="font-family:Courier New;font-size:16px;">$novoindex referenceSequencesName.nix referenceSequencesName.fa</span></em><br/><br/></span> <div style="text-align:center;"><span style="font-size:10pt;"> </span> </div></li><li>Align reads using novoalign command:<br/><br/><em><span style="font-family:Courier New;">$novoalign -f fastqFileUMIproccesed.fastq -d referenceSequencesName -o SAM |</span></em><em><span style="font-family:Courier New;"></span></em><br/><span style="font-size:10pt;"><br/></span> <div style="text-align:center;"><span style="font-size:10pt;"> </span> </div></li><li>Convert to BAM format and sort it using Samtools view and Samtools sort: (piped from the previous command)<br/><br/><span style="font-family:Courier New;"><em>| samtools view -bS - | samtools sort - fileName.sorted.bam.</em></span><span style="font-family:Courier New;"><em></em></span><br/><div style="text-align:center;"><br/></div></li></ol></li><li>Quantification<br/>Although we are aware of the existence of public software for quantifying UMI-based single-cell data (such as <a href="https://github.com/CGATOxford/UMI-tools" target="_blank"><em>UMI-tools</em></a> [Smith <em>et al.</em>, 2017]), we performed quantification using R custom scripts to have full flexibility and control over data inspection and data filtering. The starting point of the quantification is the sorted BAM files obtained in the previous step, one for each sequenced cell. <br/><ol class="npwtwo"><li>Process the BAM files.<br/><u>Script:</u><span style="font-family:Courier New;"> readAlignUMI.R. $<em>Rscript readAlignUMI.R</em></span><em><span style="font-family:Courier New;"></span></em><br/><br/>Each BAM file is loaded into R using the <em>Genomic Alignments</em> package. Then, low quality mapping reads (MAPQ &lt; 30) and reads that map with soft-clipping in the 5â² end are filtered out. Finally, reads that map to the same position and that have the same UMI are grouped for collapsing them in the next steps. </li><li>Process UMI grouped data. <br/><br/><u>Script:</u> <u><span style="font-family:Courier New;"><em></em></span></u><span style="font-family:Courier New;">bard2rds.R. $<em>Rscript bard2rds.R</em></span><span style="font-family:Courier New;"><em></em></span><br/><br/>At this step, UMIs supported with less than 3 reads are filtered out since they could represent sequencing errors. Then, different UMIs that map to the same genomic position are grouped. Different UMIs mapping to the same position represent different molecules of the same transcript. This information is stored into a <em>RangedData</em> object. </li><li>Overlap with annotation and count table generation. <br/><br/><u>Script:</u> <span style="font-family:Courier New;">txAnno.R (#link). $<em>Rscript txAnno.R.</em></span><span style="font-family:Courier New;"><em></em></span><br/><br/>If there is overlap, each molecule is assigned to a genomic feature from an annotation. In this study, we used the annotation from (Xu <em>et al</em>., 2009) since it includes a comprehensive categorization of genes into different classes (coding, CUTs, SUTs and others). The output obtained is an absolute gene expression table with genes as rows and cell as columns.</li></ol><br/></li><li>(Optional) Filtering<br/>To avoid the presence of low-quality cells (dead, stressed or apoptotic), we decided to keep only those with more than 500,000 sequenced reads, and those in which we detect more than 1,000 different transcripts. In our case, this filter eliminates cells with a high ratio of mitochondrial RNA, which has been associated with low quality cells (Ilicic <em>et al.</em>, 2016). However, this filter is not imperative and could be tuned depending on the type of experiment and on the biological question under study. </li></ol><p class="pptt" id="biaoti29161">### Recipes</p><p class="ppzz"><em>Note: All reagents and water used must be nuclease-free and only used for single cell protocols.</em> </p><ol class="npwtwo"><li>Cell capturing and lysis solution (Table 14)<br/><br/><strong>Table 14. Composition of âCell capturing and lysis solutionâ</strong><em><em><br/><img alt="" class="layerphoto" height="265" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190830/20190830012457_0098.jpg" width="500"/><br/><br/></em></em> </li><li>2x BWT Buffer<br/>10 mM Tris-HCl pH 7.5<br/>1 mM EDTA<br/>2 M NaCl<br/>0.02% Tween-20</li><li>TNT Buffer<br/>20 mM Tris pH 7.5 <br/>50 mM NaCl<br/>0.02% Tween<br/></li></ol><p class="pptt" id="biaoti29162">### Acknowledgments</p><p class="ppzzs"><br/></p><p class="ppzz">The authors would like to thank Sten Linnarsson for kindly providing reagents during the initial tests with yscRNA-seq. We thank the Protein Expression and Purification Core Facility at EMBL, Bianca Hennig and Lars Velten for kindly providing in-house purified Tn5. We thank Derek Caetano-AnollÃ©s for editing and refining the manuscript. M.N.R. was a recipient of an EMBO long-term fellowship (Stanford University) and later of a Maria de Maeztu Postdoctoral Fellowship (Doctores Banco de Santander-MarÃ­a de Maeztu at Universitat Pompeu Fabra). P.L. is a recipient of an FI Predoctoral Fellowship (Generalitat de Catalunya). This work was supported by the National Institutes of Health (NIH) and a European Research Council Advanced Investigator Grant (AdG-294542) to L.M.S. and the National Key Research and Development Program of China (2017YFC0908405) to W.W. The study was also supported by grants from the Spanish Ministry of Economy and Competitiveness (PGC2018-094136-B-I00 and FEDER to F.P.; BFU2017-85152-P and FEDER to E.N.), the Catalan Government (2017 SGR 799), and the Unidad de Excelencia Maria de Maeztu, MDM-2014-0370. We gratefully acknowledge institutional funding from the Spanish Ministry of Economy, Industry and Competitiveness (MINECO) through the Centres of Excellence Severo Ochoa award, and from the CERCA Programme of the Catalan Government. F.P. is recipient of an ICREA AcadÃ¨mia (Generalitat de Catalunya).</p><p class="pptt" id="biaoti29163">### Competing interests</p><p class="ppzz">The authors declare no financial or non-financial interests.</p><p class="pptt" id="biaoti29164">### References</p><ol class="npwtwok"><li>Conesa, A., Madrigal, P., Tarazona, S., Gomez-Cabrero, D., Cervera, A., McPherson, A., Szczesniak, M. W., Gaffney, D. J., Elo, L. L., Zhang, X. and Mortazavi, A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813401" target="_blank">A survey of best practices for RNA-seq data analysis.</a> <em>Genome Biol</em> 17: 13.</li><li>David, L., Huber, W., Granovskaia, M., Toedling, J., Palm, C. J., Bofkin, L., Jones, T., Davis, R. W. and Steinmetz, L. M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16569694" target="_blank">A high-resolution map of transcription in the yeast genome. </a><em>Proc Natl Acad Sci U S A</em> 103(14): 5320-5325.</li><li>Dobin, A., Davis, C. A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M. and Gingeras, T. R. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1093%2Fbioinformatics%2Fbts635" target="_blank">STAR: ultrafast universal RNA-seq aligner.</a> <em>Bioinformatics</em> 29(1):15-21.</li><li>Gasch, A. P., Yu, F. B., Hose, J., Escalante, L. E., Place, M., Bacher, R., Kanbar, J., Ciobanu, D., Sandor, L., Grigoriev, I. V., Kendziorski, C., Quake, S. R. and McClean, M. N. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29240790" target="_blank">Single-cell RNA sequencing reveals intrinsic and extrinsic regulatory heterogeneity in yeast responding to stress.</a> <em>PLoS Biol</em> 15(12): e2004050.</li><li>Gresham, D., Bonneau, R., Jackson, C. A., Castro, D. M. and Saldi, G. A. (2019). <a href="https://www.biorxiv.org/content/early/2019/04/17/581678.full.pdf" target="_blank">Gene regulatory network reconstruction using single-cell RNA sequencing of barcoded genotypes in diverse environments.</a> <em>bioRxiv</em> doi: https://doi.org/10.1101/581678.</li><li>Hennig, B. P., Velten, L., Racke, I., Tu, C. S., Thoms, M., Rybin, V., Besir, H., Remans, K. and Steinmetz, L. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29118030" target="_blank">Large-scale low-cost NGS library preparation using a Robust Tn5 purification and tagmentation protocol.</a> <em>G3 (Bethesda)</em> 8(1): 79-89.</li><li>Ilicic, T., Kim, J. K., Kolodziejczyk, A. A., Bagger, F. O., McCarthy, D. J., Marioni, J. C. and Teichmann, S. A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26887813" target="_blank">Classification of low quality cells from single-cell RNA-seq data.</a> <em>Genome Biol</em> 17: 29.</li><li>Islam, S., Zeisel, A., Joost, S., La Manno, G., Zajac, P., Kasper, M., Lonnerberg, P. and Linnarsson, S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24363023" target="_blank">Quantitative single-cell RNA-seq with unique molecular identifiers.</a> <em>Nat Methods</em> 11(2): 163-166.</li><li>Kim, D., Langmead, B. and Salzberg, S. L. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25751142" target="_blank">HISAT: a fast spliced aligner with low memory requirements.</a> <em>Nat Methods</em> 12(4): 357-360.</li><li>Lawrence, Michael, Wolfgang Huber, HervÃ© PagÃ¨s, Patrick Aboyoun, Marc Carlson, Robert Gentleman, Martin T. Morgan, and Vincent J. Carey. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23950696/" target="_blank">Software for Computing and Annotating Genomic Ranges.</a> <em>PLoS Computational Biology</em> 9(8):e1003118.</li><li>Li, Heng, Bob Handsaker, Alec Wysoker, Tim Fennell, Jue Ruan, Nils Homer, Gabor Marth, Goncalo Abecasis, and Richard Durbin. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19505943" target="_blank">The Sequence Alignment/Map Format and SAMtools.</a> <em>Bioinformatics</em> 25(16):2078â79.</li><li>Nadal-Ribelles, M., Islam, S., Wei, W., Latorre, P., Nguyen, M., de Nadal, E., Posas, F. and Steinmetz, L. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30718850" target="_blank">Sensitive high-throughput single-cell RNA-seq reveals within-clonal transcript correlations in yeast populations.</a> <em>Nat Microbiol</em> 4(4): 683-692.</li><li>Pelechano, V., Wei, W. and Steinmetz, L. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23615609" target="_blank">Extensive transcriptional heterogeneity revealed by isoform profiling.</a> <em>Nature</em> 497(7447): 127-131.</li><li>Pelechano, V., Wei, W. and Steinmetz, L. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26046441" target="_blank">Widespread co-translational RNA decay reveals ribosome dynamics.</a> <em>Cell</em> 161(6): 1400-1412.</li><li>Pelechano, V., Wei, W., Jakob, P. and Steinmetz, L. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24967623" target="_blank">Genome-wide identification of transcript start and end sites by transcript isoform sequencing.</a> <em>Nat Protoc</em> 9(7): 1740-1759.</li><li>Saint, M., Bertaux, F., Tang, W., Sun, X. M., Game, L., Koferle, A., Bahler, J., Shahrezaei, V. and Marguerat, S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30718845" target="_blank">Single-cell imaging and RNA sequencing reveal patterns of gene expression heterogeneity during fission yeast growth and adaptation.</a> <em>Nat Microbiol</em> 4(3): 480-491.</li><li>Skinnider, M. A., Squair, J. W. and Foster, L. J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30962620" target="_blank">Evaluating measures of association for single-cell transcriptomics.</a> <em>Nat Methods</em> 16(5): 381-386.</li><li>Smith, T., Heger, A. and Sudbery, I. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28100584" target="_blank">UMI-tools: modeling sequencing errors in Unique Molecular Identifiers to improve quantification accuracy.</a> <em>Genome Res</em> 27(3): 491-499.</li><li>Xu, Z., Wei, W., Gagneur, J., Perocchi, F., Clauder-Munster, S., Camblong, J., Guffanti, E., Stutz, F., Huber, W. and Steinmetz, L. M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19169243" target="_blank">Bidirectional promoters generate pervasive transcription in yeast.</a> <em>Nature</em> 457(7232): 1033-1037.</li><li>Zhang, X., Li, T., Liu, F., Chen, Y., Yao, J., Li, Z., Huang, Y. and Wang, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30472192" target="_blank">Comparative analysis of droplet-based ultra-high-throughput single-cell RNA-seq systems.</a> <em>Mol Cell</em> 73(1): 130-142 e5.<br/></li></ol>