<body>
            <h1>A Widely Applicable Urea-based Fluorescent/Colorimetric mRNA <em>in situ</em> Hybridization Protocol</h1>
            <p><strong>Authors:</strong> Chiara  Sinigaglia</p>
            <h2>Abstract</h2>
            <p><em>In situ</em> hybridization methods are routinely employed to detect nucleic acid sequences, allowing to localize gene expression or to study chromosomal organization in their native context. These methods rely on the pairwise binding of a labeled probe to the target endogenous nucleic acid sequenceâthe hybridization step, followed by detection of annealed sequences by means of fluorescent or colorimetric reactions. Successful hybridization requires permeabilization of tissues, followed by denaturation of nucleic acids strands, which is usually carried out in a formamide-based buffer and at high temperatures. Such reaction conditions, besides posing a health hazard (both concerning manipulation and waste disposal), can be excessively harsh for the delicate tissues of some species or developmental stages. We detail here an alternative method for <em>in situ</em> hybridization, where the toxic formamide is replaced with a urea solution. This substitution improved both tissues preservation and signal-to-noise detection, in several animal species. The protocol described here, originally developed for the hydrozoan jellyfish <em>Clytia hemisphaerica</em>, provides guidelines for adapting formamide-based traditional protocols to the urea variant. Urea-based protocols have already been successfully applied to diverse invertebrate and vertebrate species, showing the ease of such a modification, and providing the scientific community with a promising, safer and versatile tool.</p>
            <p><strong>Keywords:</strong> mRNA <em>in situ</em> hybridization, Urea, Formamide, Health and safety, Signal-to-noise</p>
            <h2>Background</h2><p>The pairwise complementary binding of single strand nucleic acid sequences inspired the development of a powerful technique, <em>in situ</em> hybridization, which allows researchers to visualize the location of DNA or RNA strands within their cellular and tissue contexts (Pardue and Gall, 1969). Numerous variants have since been developed, differing for the labeling of the exogenous complementary probes (<em>e.g.</em>, radioactive or hapten-based), in the type of samples (<em>e.g.</em>, for whole embryos or tissue sections) and in the detection method (<em>e.g.</em>, fluorescent or colorimetric signals). <br/><br/>Precise annealing of probes depends on the accessibility of the target sequences, achieved through appropriate permeabilization of tissues and cell membranes (which allows penetration of reagents), and denaturation of nucleic acid strands (which exposes the complementary target sequences). Achieving adequate sensitivity of detection, specificity of annealing, and preservation of sample morphology often requires a time-consuming optimization of reaction conditions. Efficient denaturation and annealing of nucleic acids strands are usually obtained by means of elevated temperatures and the addition of a denaturing reagent in the hybridization buffer. <br/><br/>Reaction temperatures should ideally be adapted to the base composition of the sequences of interest: hybridization temperatures are about 25 Â°C below denaturation temperatures (Tm; Marmur and Doty, 1961). 50-70% formamide is usually employed as the denaturing reagent, this choice remaining largely unchallenged since the â80sâwith few exceptions, such as the smFISH (single-molecule Fluorescence <em>In Situ</em> Hybridization) approach where 10-15% formamide is used (Haimovich and Gerst, 2018).<br/><br/>Elevated temperatures increase evaporation of the hybridization buffer, compounding the health risk already posed by formamide, an irritating, embryotoxic and teratogenic solvent (Gleich, 1974; Stula and Krauss, 1977; Merkle and Zeller, 1980; Kennedy and Short, 1986; Fail <em>et al</em>., 1998; George <em>et al</em>., 2000 and 2002; see also Sinigaglia <em>et al</em>., 2018 for further information). Hybridizing samples and waste need to be appropriately handled (CICAD 31), thus limiting the application of standard formamide-based protocols in sensitive contexts, such as pregnancy or in classrooms.<br/><br/>In the early years, a diversity of solvents was found to effectively destabilize nucleic acid strands, with urea being particularly efficient (Herskovits, 1963). Indeed, urea and formamide have similar properties, and are employed equivalently in a number of applications, such as fluorescent <em>in situ</em> hybridization on bacteria (Fontenete <em>et al</em>., 2016), protein denaturation, and clearing agents for imaging (reviewed in Azaripour <em>et al</em>., 2016). Urea can effectively replace formamide in Northern and Southern blot experiments, in particular at concentrations in the 2 M-4 M range (Simard <em>et al</em>., 2001). The mechanism of action of urea is still relatively poorly understood. Urea lowers the Tm of DNA of about 2 Â°C per mole of urea (Hutton, 1977), and is thus slightly less efficient than formamide, which achieves a 2.4-2.9 Â°C reduction per mole. Urea molecules bind weakly to RNA, disrupting its base-pair interactions and ultimately destabilizing the structure of RNA molecules (Herskovits and Bowen, 1974; Priyakumar <em>et al</em>., 2009; Lambert and Draper, 2012). <br/><br/>Urea, in powder or crystal form, is water soluble, and 8 M stock solutions can be readily prepared at 20 Â°C. Solutions tend to be viscous (Hutton, 1977), which might explain why high concentrations of urea are less effective (Simard <em>et al</em>., 2001). Such increased viscosity demands care during the hybridization step of the <em>in situ</em> protocol, to ensure that samples are efficiently immersed, and evaporation minimized. <br/><br/>The urea-based protocol presented here has been shown to be more efficient than the standard formamide-based ones in diverse metazoan species and at different developmental stages (see Sinigaglia <em>et al</em>., 2018). Detection sensitivity was improved, allowing visualization of unsuspectedly complex patterns of gene expression (Sinigaglia <em>et al</em>., 2018). This increased sensitivity of detection might be explained by an additional permeabilizing action of urea on tissues (Lim <em>et al</em>., 2009; Huang <em>et al</em>., 2011), likely due to increased osmotic pressure within cells (reviewed in Tainaka <em>et al</em>., 2016). The hyperhydration of tissues might also partially account for the improved morphologies of delicate samples, shown in the original research paper (Sinigaglia <em>et al</em>., 2018). <br/><br/>The protocol detailed here was originally developed for the hydrozoan medusa <em>Clytia hemisphaerica</em> (Quiroga Artigas <em>et al</em>., 2018; Sinigaglia <em>et al</em>., 2018, LeclÃ¨re <em>et al</em>., 2019). Following those guidelines, urea-alternative protocols have already been developed for a diversity of metazoan species and developmental stages, including priapulid and brachiozoan embryos (Thiel <em>et al</em>., 2017; Sinigaglia <em>et al</em>., 2018), the acoel <em>Hofstenia miamia</em> (L. Ricci, personal communication), the scyphozoan jellyfish Aurelia aurita (M. Manuel and T. Condamine, personal communication), and in the paddlefish (M. Minarik, personal communication). Substituting formamide with urea has also been applied to DNA-FISH on mouse oocytes (Manil-Segalen <em>et al</em>., 2018), further showing the versatility of this approach. <br/><br/>Such versatility stems from the simplicity of the key implementation, the substitution of formamide with an equal volume of urea solution (at a final concentration of 4 M). When adapting previous protocol to novel species or tissues, it is recommended to start by simply switching formamide to urea, leaving the rest of the procedure unchanged. Guidelines for eventual troubleshooting are also provided, offering general recommendations for successful <em>in situ</em> hybridization.</p>
        </body>

<p class="pptt" id="biaoti29167">## Materials and Reagents</p><ol class="npwone"><li>Materials and reagents used in multiple steps<br/><ol class="npwtwo"><li>Gloves</li><li>RNase free tubes (<em>e.g.</em>, 1.5 ml Eppendorf tubes, or equivalent)</li><li><em>In situ</em> baskets (Sample container, as in Figure 1A)<br/>They could be either the Netwell baskets (Corning), or homemade baskets constructed from nylon mesh and microcentrifuge tubes, as explained in Sive <em>et al</em>. (2007). Baskets size depends on the type of sample; they usually fit within the wells of a 6-, 12- or 24-well plate. Performing <em>in situ</em> hybridization in baskets (instead of tubes) is recommended: <br/><ol class="npwthree"><li>To reduce loss and damages to samples, due to pipetting.</li><li>To accelerate exchange of solutions (baskets can be simply transferred with the help of forceps to a multiwell plate with the new solution (Figure 1B)âin this case tap gently the basket against the walls of the well to ensure removal of older solution, then gently immerse it in the new well).</li><li>Baskets can be recycled multiple times, after being thoroughly washed (<em>e.g.</em>, rinse with MilliQ H<sub>2</sub>O, then immerse in a 1 M NaOH solution to neutralize RNases, and rinse several times with RNase-free MilliQ H<sub>2</sub>Oâsimply placing them in a beaker on a shaker will assure thorough swirling).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031156_8169.jpg" width="300"/><br/><strong>Figure 1. Sample carrier.</strong> A. Example of three baskets with mesh-bottom. B. A 24-well plate used for <em>in situ</em> hybridization, with four baskets. C. The same plate, closed and wrapped in plastic film to avoid evaporation during hybridization step.<br/><br/></li></ol></li><li>Multiwell plate (For fitting <em>in situ</em> baskets, as in Figure 1B); (<em>e.g.</em>, 24-well cell culture plate, sterile, with lid, from CellStar (Greiner Bio-one, catalog number: 662-160))</li><li>Ice and ice container</li><li>MilliQ H<sub>2</sub>O (Storage: RT)</li><li>Tween-20 (Sigma-Aldrich, catalog number: P1379), storage: RT (Mild detergent. Given its high viscosity, it is recommended to prepare a 10% working solution)</li><li>Na<sub>2</sub>HPO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: 431478), storage: RT (For preparing 10X PBS solution)</li><li>NaCl (Sigma-Aldrich, catalog number: S9888), storage: RT (For preparing 10X PBS solution)</li><li>KCl (Sigma-Aldrich, catalog number: P9333), storage: RT (For preparing 10X PBS solution)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Sigma-Aldrich, catalog number: P9791), storage: RT (For preparing 10X PBS solution)</li><li>10x PBS (Phosphate-Buffered Saline) pH 7.4 (see Recipes, or from tablets from Calbiochem, catalog number: 524650), storage: RT (Helps maintaining a constant pH)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Fixation<br/><ol class="npwtwo"><li>37% formaldehyde (Sigma-Aldrich, catalog number: 252549), storage: RT (Crosslinking)</li><li>8% glutaraldehyde (Electron Microscopy Sciences, catalog number: 16020), storage: 4 Â°C (Crosslinking)</li><li>EGTA (Ethylene glycol-bis(2-aminoethylether)-N,N,N',N'-tetraacetic acid) (Sigma-Aldrich, catalog number: E3889), storage: RT [Calcium/magnesium chelator, which reduces enzyme activity (such as RNase); reagent of HEM buffer]</li><li>HEPES (Sigma-Aldrich, catalog number: H4034), storage: RT (Reagent of HEM buffer)</li><li>MgSO<sub>4</sub> (Sigma-Aldrich, catalog number: M7506), storage: RT (Reagent of HEM buffer)</li><li>Tween-20 (Sigma-Aldrich, catalog number: P1379), storage: RT</li><li>Methanol (<em>e.g.</em>, Sigma-Aldrich, catalog number: 322415), storage: RT</li><li>1x PBS (see Recipes)</li><li>1x PBST (see Recipes)</li><li>CISH fixation buffer (see Recipes)</li><li>FISH fixation buffer (see Recipes)</li><li>Dehydration buffer (see Recipes)</li><li>Rehydration buffer (see Recipes)</li><li>Re-fixation buffer (see Recipes)</li></ol><br/></li><li>Acetylation<br/><ol class="npwtwo"><li>TEA (Triethylamine) (Sigma-Aldrich, catalog number: T0886), storage: RT (For reducing the non-specific binding of negatively-charged probe to tissues)</li><li>Acetic anhydride (Sigma-Aldrich, catalog number: A6404), storage: 4 Â°C (For reducing the non-specific binding of negatively-charged probe to tissues)</li><li>1x PBST (see Recipes)</li><li>10x TEA stock solution (see Recipes)</li><li>0.1 M TEA (see Recipes)</li><li>0.25% Acetic anhydride in 0.1 M TEA (see Recipes)</li></ol><br/></li><li>Permeabilization<br/><ol class="npwtwo"><li>Proteinase K (Sigma-Aldrich, catalog number: P4850), storage: 4 Â°C or -20 Â°C (Digests proteins, including nucleases. Optimum pH is 7.5-9.0)</li><li>Glycine (Sigma-Aldrich, catalog number: G8898), storage: RT (For neutralizing the proteinase K)</li><li>1x PBST (see Recipes)</li></ol><br/></li><li>Hybridization<br/><ol class="npwtwo"><li>Plastic film (For wrapping sample container during hybridization step)</li><li>Urea (Sigma-Aldrich, catalog number: U5378), storage: RT (Denaturing agent)</li><li>Dextran sulfate sodium salt, Mr ~200''00 (Sigma-Aldrich, catalog number: 67578-25G), storage: RT [Increases molecular crowding and can therefore accelerate the kinetics of hybridization (locally increases probe concentration)]</li><li>tRNA, from bakerâs yeast (Sigma-Aldrich, catalog number: 10109509001), storage: -20 Â°C (Blocks non-specific probe binding)</li><li>Salmon sperm (Thermo Fisher, catalog number: 15632011), storage: -20 Â°C (Blocks non-specific probe binding)</li><li>Heparin sodium (Thermo Fisher, catalog number: 10239840), storage: -20 Â°C (Helps to reduce background, by binding to proteins that bind to DNA)</li><li>20% SDS (Sodium Dodecyl Sulfate) (Fisher, catalog number: BP131), storage: RT (Detergent. Permeabilizes membranes, facilitating probe distribution)</li><li>NaCl (Sigma-Aldrich, catalog number: S9888), storage: RT (Salt. For preparing 20x SSC stock buffer, see Recipes)</li><li>Trisodium citrate (Na<sub>3</sub>C<sub>6</sub>H<sub>5</sub>O<sub>7</sub>) (For preparing 20x SSC stock buffer, see Recipes)</li><li>Tween-20 (Sigma-Aldrich, catalog number: P1379), storage: RT</li><li>10x DIG labeling mix RNA (Roche, catalog number: 11277073910), storage: -20 Â°C (For probe synthesis)</li><li>RNasin Ribonuclease Inhibitor (Promega, catalog number: N2111 or N2511), storage: -20 Â°C (For probe synthesis)</li><li>DTT (Promega, catalog number: P1171), storage: -20 Â°C (For probe synthesis)</li><li>Transcription Optimized 5x Buffer (Promega, catalog number: P1181), storage: -20 Â°C (For probe synthesis)</li><li>T7/SP6/T3 RNA polymerase (Promega, catalogue number, respectively: P2075/ P1085/ P2083), storage: -20 Â°C (For probe synthesis)</li><li>RQ1 RNase-free DNase (Promega, catalog number: M6101), storage: -20 Â°C (For probe synthesis)</li><li>Illustra ProbeQuant G-50 Micro Columns (GE Healthcare, catalog number: 28-9034-08) (For probe synthesis)</li><li>1x PBST (see Recipes)</li><li>8 M Urea stock solution (see Recipes)</li><li>20x SSC (Sodium Saline Citrate or Standard Saline Citrate) (see Recipes, or from Sigma-Aldrich, catalog number: S6639), storage: RT [Controls stringency (annealing of probe to target)]</li><li>Wash I (see Recipes)</li><li>Wash II (see Recipes)</li><li>Wash III (see Recipes)</li><li>Urea-based 100% Hybridization Buffer (see Recipes)</li><li>Urea-based 50% Hybridization Buffer (see Recipes) </li></ol><br/></li><li>Antibody incubation<br/><ol class="npwtwo"><li>Anti-Digoxigenin-AP, Fab fragments (Roche Applied Sciences, Sigma-Aldrich, catalog number: 11093274910), storage: -20 Â°C [Recognizes the DIG hapten, conjugated to Alkaline Phosphatase (AP)]</li><li>Anti-Digoxigenin-POD, Fab fragments (Roche Applied Sciences, Sigma-Aldrich, catalog number: 11207733910), storage: -20 Â°C [Recognizes the DIG hapten, conjugated to horseradish Peroxidase (POD)]</li><li>Blocking reagent (Sigma-Aldrich, catalog number: 11096176001), storage: 20-25 Â°C (To decrease non-specific binging of antibodies)</li><li>Maleic acid (Sigma Aldrich, catalog number: M0375), storage: RT (For preparing MAB)</li><li>NaCl (Sigma-Aldrich, catalog number: S9888), storage: RT (Salt. For preparing MAB)</li><li>NaOH (Sigma-Aldrich, catalog number: 221465), storage: RT (For preparing MAB)</li><li>10x blocking reagent stock solution (see Recipes), storage: -20 Â°C</li><li>10x MAB (Maleic Acid Buffer) stock solution (see Recipes) (For diluting blocking buffer, used after SSC washes)</li><li>1x MABT buffer (see Recipes)</li><li>10x blocking reagent stock solution (see Recipes), storage: -20 Â°C</li><li>1x Blocking buffer (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Detection<br/><ol class="npwtwo"><li>Aluminum foil (For protecting samples during staining step) </li><li>BCIP (5-Bromo-4-Chloro-3-Indolyl-Phosphate)/NBT (Nitro Blue Tetrazolium) Color Development Substrate (Promega, catalog number: S3771), storage: 4 Â°C or -20 Â°C [Substrate of alkaline phosphatase which generates a dark (blue/violet) precipitate]</li><li>NaCl (Sigma-Aldrich, catalog number: S9888), storage: RT (Salt. Component of NTMT buffer)</li><li>MgCl<sub>2</sub> (Sigma-Aldrich, catalog number: M8266), storage: RT (Component of NTMT buffer, MgCl<sub>2</sub> is a necessary cofactor for alkaline phosphatase)</li><li>Tris-HCl (Tris hydrochloride) (Sigma, catalog number: T3253), storage: RT (pH buffering component of NTMT)</li><li>Tween-20 (Sigma-Aldrich, catalog number: P1379), storage: RT</li><li>Levamisole (Agilent Dako, catalog number: X302130-2), storage: 2-8 Â°C (Alkaline Phosphatase inhibitor)</li><li>TSA Plus Fluorescence Amplification kit Cyanine 3/5 (Perkin Elmer, Waltham, MA; catalog number: NEL752001KT), storage: 4 Â°C [Tyramide Signal Amplification (TSA) kit uses Horseradish Peroxidase (HRP or Horseradish POD) to catalyze the deposition of a fluorophore-labeled radicals]</li><li>H<sub>2</sub>O<sub>2</sub> (Sigma-Aldrich, catalog number: 516813), storage: 4 Â°C (Substrate for horseradish peroxidase)</li><li>Filters for syringe (Millex-GS, catalog number: SLGS033SS) (For filtering NTMT buffer, in order to avoid precipitates)</li><li>Disposable syringes (<em>e.g.</em>, 50 ml from Thermo Scientific, catalog number: S7510-50) (For filtering NTMT buffer, in order to avoid precipitates)</li><li>5 M NaCl stock solution (For preparing NTMT buffer, see Recipes), storage: RT</li><li>1 M Tris-HCl, pH 9.5 stock solution (For preparing NTMT buffer, see Recipes), storage: RT</li><li>1 M MgCl<sub>2</sub> stock solution (For preparing NTMT buffer, see Recipes), storage: RT</li><li>NTMT buffer (see Recipes)</li><li>NTMT <em>minus</em> buffer (see Recipes)</li><li>NBT/BCIP in NTMT buffer (see Recipes)</li></ol><br/></li><li>Storage &amp; Mounting<br/><ol class="npwtwo"><li>Glycerol (Sigma-Aldrich, catalog number: G5516), storage: RT (Mounting medium)</li><li>Citifluor-AF1 (Citifluor, catalog number: AF1-100), storage: RT [Mounting medium for fluorescent samples (on-hardening antifading/anti-bleaching)]</li><li>Slide (<em>e.g.</em>, 76 x 26 mm, with frosted end (practical for labeling), from Knittel Glass (Mounting)</li><li>Coverslide (<em>e.g.</em>, 20 x 20 mm, thickness 1, from Knittel Glass) (Mounting)</li><li>Nailpolish (For sealing mounted specimens; any commercial nailpolish can be used, however prefer a transparent one, and test first on lesser important samples, in case it contains too much solventâwhich might damage samples)</li></ol></li></ol><p class="pptt" id="biaoti29168">### Equipment</p><p class="ppzz"><em>Note: No specific brand/model is provided; this equipment is commonly available in laboratories.</em> </p><ol class="npwtwo"><li>Pipettes </li><li>Heating block (For probe denaturation, <em>e.g.</em>, featuring a temperature range 10-100 Â°C)</li><li>Metallic forceps (For transferring baskets, <em>e.g.</em>, from Fine Science Tools)</li><li>Waterbath/Hybridization oven (Hybridization and stringent washes)</li><li>Glass bottles with high-temperature resistant screw caps (<em>e.g.</em>, 250 ml bottle from Fisherbrand (catalogue number: FB-800-250); 500 ml bottle from VWR [borosilicate 3.3; catalogue number: 215-1594); 1000 ml bottle from Scott Duran (Original GL 45 series)]</li><li>pH meter (For adjusting pH of solutions)<br/></li></ol><p class="pptt" id="biaoti29169">### Procedure</p><p class="ppzz"><em> General notes on samples handling:</em> </p><ol class="npwtwo"><li><em>Sample fixation is best performed in tightly sealed RNase free containers, such as Eppendorf or Falcon tubes. </em> </li><li><em>Once samples are re-hydrated, they can be transferred to RNase free in situ hybridization baskets (shown in Figure 1A).</em> </li><li><em>The density of samples might affect the in situ hybridization results: overfilled vials/baskets often yield poorly stained/damaged samples, and too little amount of sample (<em>e.g.</em>, one embryo) might become over-stained (possibly due to unbalanced ratio probe/target).<br/></em> <br/></li></ol><ol class="npwone"><li>Sample fixation/dehydration (Table 1)<br/><p class="ppzz"><em>Notes:<br/></em> </p><ol class="npwtwo"><li><em>Fixatives should be handled and disposed of according to laboratory safety guidelines.</em> </li><li><em>Fixation (choice of fixative, duration of fixation) must be adapted to sample types. </em> </li><li><em>Two alternative types of fixation are provided, for colorimetric in situ hybridization (CISH), and for fluorescent in situ (FISH).</em><br/><br/><strong>Table 1. Fixationa<br/></strong> <p class="ppzz" style="margin-left:-15px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031458_8401.jpg" width="650"/> </p><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Rehydration (Table 2)<br/><br/><strong>Table 2. Rehydration</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031529_0117.jpg" width="550"/><br/><br/></li><li><em>(Optional)</em> Permeabilization; proteinase K treatment permeabilize tissues (Table 3)<br/><em>Notes:<br/><ol class="npwthree"><li>Treatment time should be titrated for each new batch of enzyme, as strength might vary. A useful rule of thumb is treating a test sample until tissue damage appears, and halving the resulting time (see Figure 3 in Notes section for further details). </li><li>This step might prove unnecessary for very delicate tissues.<br/><div style="white-space:nowrap;"><br/></div></li></ol></em><strong>Table 3. Permeabilization</strong><strong><br/></strong> <p class="ppzz" style="margin-left:-10px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031548_5535.jpg" width="650"/> </p><br/></li><li>(Optional) Acetylation (Table 4)<br/><em>Note: This step might help reducing non-specific signal, due to probes binding to positively charged surfaces.</em><br/><br/><strong>Table 4. Acetylation</strong><strong></strong><br/><p class="ppzz" style="margin-left:-5px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031609_1149.jpg" width="650"/> </p><br/></li><li>(<em>Optional</em>) Re-fixation (Table 5)<br/><em>Notes:<br/></em> <ol class="npwtwo"><li><em>Depending on sample type (e.g., fragile invertebrate embryos), an additional re-fixation step prior to hybridization might be required. </em> </li><li><em>Generally, it is advisable to re-fix after proteinase K treatment. </em> </li><li><em>Glutaraldehyde might be omitted: it is recommended to test several options while setting up the protocol for the first time. Glutaraldehyde penetrates tissues slowly, and its use is preferable for samples thinner than 1 mm (Eltoum et al., 2001).</em> </li></ol><br/><strong>Table 5. Re-fixation<br/></strong> <p class="ppzz" style="margin-left:-10px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031630_5509.jpg" width="600"/> </p><br/></li><li>Hybridization (Table 6)<br/><em>Notes:<br/></em> <ol class="npwtwo"><li><em>Do not let the temperature drop during the pre-hybridization/hybridization phase. Transfer of baskets or addition of probe might take long time. When working on multiple samples, set up manageable batches.</em> </li><li><em>The urea-based hybridization buffer tends to be viscous, and evaporation can be an important issue, leading to the formation of crystals that will damage the samples. Seal vials/multi-well plate carefully (Figure 1C).</em> </li><li><em>Be aware that some sample types might become temporarily transparent while in heated hybridization buffer. In this phase, they will be also more fragile.</em> </li></ol><br/><strong>Table 6. Hybridization</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190903/20190903233248_7084.jpg" width="750"/><br/><br/><p><br/></p></li><li>Blocking (Table 7)<br/><br/><strong></strong><strong>Table 7. Blocking</strong><strong><br/></strong><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829032711_4689.jpg" width="650"/><br/><br/></li><li>Signal detection <br/><strong><em>AP antibody, colorimetric reaction</em></strong> (Table 8)<br/><br/><strong>Table 8. Colorimetric signal detection<br/></strong> <img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190903/20190903235129_2670.jpg" width="650"/><br/><br/><strong><em>Fluorescent detection (POD antibody, signal amplification)</em></strong> (Table 9)<br/><br/><strong>Table 9. Fluorescent signal detection</strong><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031944_4593.jpg" width="650"/><br/><br/></div></li><li>CISH mounting (Table 10)<br/><br/><strong>Table 10. Colorimetric samples- mounting</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829032010_4273.jpg" width="650"/><br/><br/></li><li>FISH mounting (Table 11)<br/><br/><strong>Table 11. Fluorescent samples- mounting<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829032037_9753.jpg" width="650"/></strong><strong></strong><br/></li></ol><p class="pptt" id="biaoti29170">### Data analysis</p><p class="ppzz">The urea-variant <em>in situ</em> hybridization protocol generates reliable results. Sample morphology is well preserved, and the good ratio signal-to-background allows for detecting of gene expression with cellular resolution. Figure 2 depicts the results of colorimetric (CISH) and fluorescent (FISH) protocols performed on a <em>Clytia</em> medusa.<br/>  Please refer to the original paper (Sinigaglia <em>et al</em>., 2018) for a further comparison of the formamide-based and urea protocol, for examples from diverse species, and for a more detailed analysis of results.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031234_2566.jpg" width="100"/><br/><strong>Figure 2. Detection of gene expression in medusae.</strong> <em>RFamide</em> (accession number for <em>Clytia</em> gene: KX496951) is expressed in nervous system cells. The urea- <em>in situ</em> hybridization protocol detects cells <em>RFamide</em>+, scattered thorough the medusa body; here a detail is shown, including (left to right): tentacle, tentacle bulb, circular canal running at the periphery of the umbrella, and umbrella. Tissues are well preserved, in particular the fragile umbrella. A. CISH result, the focus is on the scattered cells in the tentacle and in the circular canal. Image taken with a Zeiss Axio Imager A2. B. FISH results, maximum projection (done with Fiji) of a z-stack image taken with a Leica SP8 confocal microscope. Both images were cropped; scale bars = 50 Î¼m.</p><p class="pptt" id="biaoti29171">### Notes</p><ol class="npwone"><li>Controls<br/><ol class="npwtwo"><li>It is highly recommended to perform both a negative control (<em>e.g.</em>, using sense probes, or a probe of similar length for a gene not expressed in the sample being studied), and a positive control (using a reliable, known antisense probe), in particular when testing any new probe or optional step.</li><li>When testing for the first time this urea-based protocol, it is recommended to perform a parallel <em>in situ</em> hybridization with an eventual previous formamide-based protocol (if one is already in use in the laboratory).</li></ol><br/></li><li>mRNA probes<br/><ol class="npwtwo"><li>Riboprobes (mRNA probes) can be synthesized from either a linearized template plasmid, or from a PCR product. The DIG-labeled RNA probes shown in the original research paper (Sinigaglia <em>et al</em>., 2018), including the <em>RFamide</em> probe employed in the present protocol, were synthesized using one of the RNA polymerase kits from Promega (T7, SP6 or T3 polymerases, according to the orientation of insert). For probe synthesis, add in order: 2 Î¼l of 10x DIG labeling mix RNA (Roche); 0.5 Î¼l of RNasin Ribonuclease Inhibitor (Promega); 2 Î¼l of 100 mM DTT (Promega); 4 Î¼l of Transcription Optimized 5x Buffer (Promega); 1-2 Î¼g of DNA template (calculate the appropriate volume); RNase-free H<sub>2</sub>O to a 20 Î¼l of total volume; 0.5 Î¼l of T7/SP6/T3 RNA polymerase (Promega). Run reaction for 2-5 h at 37 Â°C, then arrest by digesting the DNA template with 1.5 Î¼l of RQ1 RNase-free DNase (Promega), at 37 Â°C for 30 min. Elute by adding 30 Î¼l of RNase free H<sub>2</sub>O, purify with the illustra ProbeQuant G-50 Micro Columns (GE Healthcare). Check probe integrity by running 1 Î¼l of purified probe on 1% agarose gel, and quantify with nanodrop. Probes can be stored at -20 Â°C or -80 Â°C, addition of 50% (v/v) RNase-free formamide helps preserving RNA integrity.</li><li>It is recommended to synthesize always both the antisense (complementary to mRNA) probe and the sense probe, to be used as a control for the specificity of binding. </li><li>Probes are usually designed on the coding sequence of a gene, but they can also target the 3' UTR region (this might be helpful for example when discriminating highly similar genes). Clone the desired region from cDNA, possibly deriving from mRNA extracted from the stage/body region of interest (different stages/tissues might express differently-spliced forms of the same gene).</li><li>Recommended probe length ranges between 500 and 800, up to 1,500 bp. Long probes can be fragmented, <em>e.g.</em>, by alkaline hydrolysis.<br/><strong>Alkaline hydrolysis</strong> (Cox <em>et al</em>., 1984): Hydrolyze probe by adding 1:1 volume of hydrolysis buffer (40 mM NaHCO<sub>3</sub>/60 mM Na<sub>2</sub>CO<sub>3</sub>, pH 10.2) freshly prepared in RNase-free water. Incubate at 60 Â°C, for a length of time (expressed in minutes) calculated as: <br/><br/><div style="text-align:center;"><img align="center" alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829034019_6661.jpg" style="font-size:10pt;" width="200"/> </div><br/>Where, L<sub>0</sub> is original transcript length (expressed in kb) and Lf is the desired fragment length (in kb). <br/>Neutralize by adding sodium acetate (to 0.1 M final, at pH 6.0), and glacial acetic acid (to 0.5%, v/v), and precipitate with ethanol.</li></ol><br/></li><li>Adapting the protocol/Variants<br/><ol class="npwtwo"><li>This protocol is based on <em>Clytia</em> <em>in situ</em> hybridization (Sinigaglia <em>et al</em>., 2018); it is however meant to represent a general guideline. When adapting it to new study organisms/tissues/cells, it is recommended to start by simply substituting the formamide in the original hybridization buffer, with a urea solution, to a final concentration of 4 M.</li><li>Variants to the protocol are possible, in particular the fluorescent <em>in situ</em> hybridization has already been successfully coupled to immunostaining or to EdU reactions (data not shown).</li><li>This protocol might be used also on tissue sections (frozen or paraffin-embedded), in which case further adjustments might be necessary (<em>e.g.</em>, reducing detergents or urea concentration in the hybridization buffer).</li><li>Urea-based hybridization buffer might also be tested with smFISH.</li><li>Proteinase K treatment is optional. Proteinase K digests proteins, thus permeabilizing tissues, and aiding the penetration of probes and reagents. Too strong treatment might damage tissue morphology, while insufficient digestion might result in poor detection of signal. Both incubation time and enzyme concentration should be assessed for each new batch of enzyme.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190829/20190829031250_9458.jpg" width="100"/><br/><strong>Figure 3. Titration of proteinase K treatment.</strong> Images show the progressive digestion of a fixed <em>Clytia</em> medusa, treated with 10 Î¼g/ml of proteinase K in 1x PBST. Visible damages (yellow asterisk) start to appear after 30 min of treatment, and by 40 min the medusa is severely damaged (notice the detached velum and gonad). According to the guideline provided in the main protocol (see Table 3), the proteinase K treatment should last for 15-20 min during <em>in situ</em> hybridization. </li></ol><br/></li><li>Troubleshooting<br/><ol class="npwtwo"><li>Damaged samples<br/><ol class="npwthree"><li>Make sure that samples are always covered by solution, and never let them dry. </li><li>Verify that hybridization buffer did not evaporate.</li><li>Add a second fixation step prior to hybridization, after rehydration.</li><li>Remove permeabilization step, or eventually reduce duration of treatment.</li><li>Handle samples and baskets carefully. Samples might be more fragile while in methanol and in hybridization solution.</li><li>If damages are due to crystals forming on samples, try rapidly washing samples with MilliQ H<sub>2</sub>O.<br/><div style="white-space:nowrap;"></div></li></ol></li><li>Absent/poor signal<br/><ol class="npwthree"><li>Verify probe quality (check for degradation) and that correct antisense probe is being used. Long probes usually perform worse, the optimal length is however species- and gene-specific (aim for 500-800 bp). Probes can be fragmented (<em>e.g.</em>, by alkaline hydrolysis, see point B4 of Notes section); this will improve both penetration and recognition of target. </li><li>Add permeabilization step/re-test/renew proteinase K batch.</li><li>Increase/reduce probe concentration in hybridization mix.</li><li>In case of CISH: extend the signal detection step (renew frequently the NBT/BCIP mix, while keeping samples at 4 Â°C). In case of FISH: extend the signal detection reaction, or repeat it.</li><li>In case of CISH: use freshly prepared staining solution. Freshly prepare also NTMT (do not use if older than a day, keep at 4 Â°C while not in use). </li><li>In case of FISH: prepare buffer just before use.</li><li>Increase/reduce urea concentration in the hybridization mix (8 M to 2 M): in general, higher concentrations will have a stronger permeabilizing effect, and are recommended for difficult/hard tissues.</li><li>Verify the pH of solutions, in particular of TEA and of SSC stock solutions.</li><li>Samples might be over-fixed, verify. It might be advisable to remove glutaraldehyde, in case it was added to the fixative solution.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Non-specific signal<br/><ol class="npwthree"><li>In case of CISH: add levamisole to NTMT and staining mix.</li><li>In case of CISH: extend incubation time with NTMT, prior to NBT/BCIP color reaction.<br/></li><li>In case of CISH: protect NBT/BCIP mix from light. Verify that the stock solution did not turn purple or black and that it appears homogeneous. </li><li>In case of CISH: try performing FISH instead (allows for more precise observation, avoids eventual issues with non-specific deposition of precipitate).</li><li>Pre-hybridize overnight.</li><li>Add acetylation step.</li><li>Make sure that acetic anhydride solution (Steps 1 and 2 in Procedure D) gets efficiently mixed, and that samples get well in contact with it.</li><li>Increase hybridization temperature.</li><li>Reduce probe concentration. Recycle probe (store the Hybridization mix at -20 Â°C).</li><li>Increase duration/stringency of post-hybridization washes. Stringency is modulated by changing the concentration of SSC buffer (5-6x for low stringency, 2-0.1x for high stringency).</li><li>Make sure that no dust/dirt is attached to specimens before starting the color reaction: particles might derive from insufficient washing of recycled baskets, from solutions (filter them, in particular NTMT solution), from clothes. If present, remove them manually. <br/>In general, particles attached to specimens most commonly derive from old solutions: renew them, in particular the staining reagents, and adjust pH accurately. A step in NTMT <em>minus</em> buffer (filtered) prior to staining might also help</li><li>In case of CISH: extend Steps 9 in Procedure H (Colorimetric detection) and Procedure I (CISH mounting), keeping samples at 4 Â°C.</li><li>Verify the pH of solutions, in particular of TEA and of SSC stock solutions.<br/><br/></li></ol></li><li>Signal too strong/rapid<br/><ol class="npwthree"><li>Reduce probe concentration.</li><li>For CISH: dilute staining solution. Reduce staining time. Add levamisole to staining solution and/or carry out staining reaction at 4 Â°C, in order to slow down AP reaction.</li><li>In case of CISH: try performing FISH instead (allows for more precise observation, avoids eventual issues with non-specific deposition of precipitate).</li><li>Increase the stringency of washes.</li><li>Add acetylation step.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti29172">### Recipes</p><ol class="npwtwok"><li>10x PBS (pH 7.4) stock solution<br/><ol class="npwthree"><li>Weigh and add: 25.6 g of Na<sub>2</sub>HPO<sub>4</sub>Â·7H<sub>2</sub>O, 80 g of NaCl, 2 g of KCl, 2 g of KH<sub>2</sub>PO<sub>4</sub> </li><li>Bring to 1 L with MilliQ H<sub>2</sub>O</li><li>Autoclave</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>1x PBST<br/><ol class="npwthree"><li>Add 0.1% Tween-20 to 1x PBS solution</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated) </li></ol></li><li>HEM buffer <br/>0.1 M HEPES, pH 6.9<br/>50 mM EGTA, pH 7.2<br/>10 mM MgSO<sub>4</sub><br/>Prepare freshly</li><li>CISH fixation buffer<br/>3.7% formaldehyde plus<br/>0.4% glutaraldehyde <br/>1x PBS (pH 7.4)<br/>Prepare freshly</li><li>FISH fixation buffer <br/>3.7% formaldehyde in HEM buffer <br/>Prepare freshly</li><li>Dehydration buffer<br/>50% 1x PBST<br/>50% Methanol<br/>Prepare freshly</li><li>Rehydration buffer <br/>50% Methanol<br/>50% 1x PBST <br/>Prepare freshly</li><li>Permeabilization buffer <br/>10 Î¼g/ml proteinase K in 1x PBST <br/>Prepare freshly</li><li>Neutralization buffer<br/>2 mg/ml glycine in 1x PBST <br/>Prepare freshly</li><li>10x TEA stock solution<br/><ol class="npwthree"><li>Prepare 1 M solution in H<sub>2</sub>O</li><li>Adjust the pH to 7.8</li><li>Do not autoclave</li><li>Handle TEA with care (do not shake)</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated) </li></ol></li><li>0.1 M TEA<br/>Dilute 10x TEA stock solution with 1x PBST, to 0.1 M Triethanolamine (TEA)<br/>Prepare freshly</li><li>0.25% Acetic anhydride in 0.1 M TEA<br/>Add 0.25% acetic anhydride (v/v) to 0.1 M TEA solution, and mix thoroughly<br/>Prepare freshly</li><li>Re-fixation buffer <br/>3.7% formaldehyde plus 0.2% glutaraldehyde in 1x PBST<br/>Prepare freshly</li><li>8 M Urea stock solution<br/><ol class="npwthree"><li>Dissolve urea in MilliQ H<sub>2</sub>O, calculating the required amounts (MW = 60.06 g/mol)</li><li>Be aware that the reaction is endothermic</li><li>Prepare freshly and sterilize by filtration if necessary</li><li>The solution can be used for both hybridization and post-hybridization steps, in this case keep at 4 Â°C</li></ol></li><li>20x SSC (sodium saline citrate) stock buffer (1 L)<br/><em>Note: 20x concentration of buffer prevents fungal and bacterial growth.</em><br/><ol class="npwthree"><li>Dissolve 175.3 g NaCl and 88.2 g of trisodium citrate (Na<sub>3</sub>C<sub>6</sub>H<sub>5</sub>O<sub>7</sub>) in 900 ml MilliQ H<sub>2</sub>O</li><li>Adjust the pH to 7.0, top to 1 L with MilliQ H<sub>2</sub>O</li><li>Autoclave to sterilize</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>20% SDS (Sodium Dodecyl Sulfate)<br/>Precipitation can occur at low temperatures. In this case, just let the buffer warm up before use</li><li>Urea-based Hybridization Buffer <br/>Final concentration:<br/>5x SSC<br/>1% dextran<br/>4 M urea<br/>50 Î¼g/ml of heparin<br/>50 Î¼g/ml of tRNA (or salmon sperm)<br/>1% SDS<br/><strong>To prepare (10 ml) add in the following order, prepare freshly: </strong><br/>2.5 ml of 20x SSC<br/>0.1 g of dextran powder (dissolve well)<br/>5 ml of 8 M urea stock solution<br/>25 Î¼l of heparin (20 mg/ml stock)<br/>10 Î¼l tRNA<br/>5.1 ml of MilliQ H<sub>2</sub>O<br/>500 Î¼l of 20% SDS<br/><em>Notes:<br/><ol class="npwthree"><li>Be aware that premature addition of SDS might cause a precipitate to appear.</li><li>Do not agitate vigorously, to avoid bubbles.</li><li>Hybridization buffer can also be aliquoted and stored at -20 Â°C, in this case avoid multiple thawing/freezing cycles.</li></ol></em> </li><li>50% Hybridization Buffer <br/>50% Urea-based Hybridization Buffer <br/>50% 1x PBST<br/>Prepare freshly</li><li>Wash I <br/>Final concentration:<br/>4 M urea<br/>0.1% Tween-20<br/>5x SSC<br/>Add MilliQ H<sub>2</sub>O to volume<br/>Prepare freshly; or shortly beforehand (<em>e.g.</em>, the evening before use)</li><li>Wash II <br/>Final concentration:<br/>2 M urea<br/>0.1% Tween-20<br/>2x SSC <br/>Add MilliQ H<sub>2</sub>O to volume<br/>Prepare freshly; or shortly beforehand (<em>e.g.</em>, the evening before use)</li><li>Wash III<br/>Final concentration:<br/>0.1% Tween-20 <br/>2x SSC<br/>Add MilliQ H<sub>2</sub>O to volume<br/>Prepare freshly; or shortly beforehand (<em>e.g.</em>, the evening before use)</li><li>10x MAB buffer stock solution<br/><ol class="npwthree"><li>Add to 700 ml of MilliQ H<sub>2</sub>O: 116 g of maleic acid, 87 g of NaCl, 40 g of NaOH</li><li>Adjust carefully the pH to 7.5, and top to 1 L with MilliQ H<sub>2</sub>O</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>1x MABT buffer<br/><ol class="npwthree"><li>Dilute stock solution 1:10 with MilliQ H<sub>2</sub>O, add 0.1% Tween-20</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>10x blocking reagent stock solution<br/><ol class="npwthree"><li>Dissolve in maleic acid buffer to a final concentration of 10% (w/v), shaking and heating to 60 Â°C</li><li>Stock solution should be autoclaved, aliquoted and stored at -20 Â°C</li></ol></li><li>1x Blocking buffer</li><li>5 M NaCl<br/><ol class="npwthree"><li>Dissolve 292 g of NaCl in 800 ml of MilliQ H<sub>2</sub>O<br/></li><li>Adjust the volume to 1 L with MilliQ H<sub>2</sub>O</li><li>Autoclave for sterilizing<br/></li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>1 M Tris-HCl, pH 9.5<br/><ol class="npwthree"><li>Dissolve 121.1 g of Tris-base in 800 ml of MilliQ H<sub>2</sub>O</li><li>Adjust the pH to 9.5 by adding HCl (let the solution reach room temperature for accurate adjustment of pH)</li><li>Adjust the volume to 1 L with MilliQ H<sub>2</sub>O</li><li>Autoclave for sterilizing</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>1 M MgCl<sub>2<br/></sub> <ol class="npwthree"><li>Dissolve 203.3 g of MgCl<sub>2</sub>Â·6H<sub>2</sub>O in 800 ml of MilliQ H<sub>2</sub>O</li><li>Adjust the volume to 1 L with MilliQ H<sub>2</sub>O</li><li>Autoclave</li><li>Store at room temperature, discard if precipitate/particles appear (the bottle might have been contaminated)</li></ol></li><li>NTMT buffer<br/>Final concentration:<br/>100 mM NaCl<br/>100 mM Tris-HCl <br/>50 mM of MgCl<sub>2</sub><br/><em><strong>For 100 ml, add:</strong></em><br/>2 ml of 5 M NaCl<br/>10 ml of 1 M Tris-HCl (pH 9.5)<br/>5 ml of 1 M MgCl<sub>2</sub><br/>1 ml of Tween-20<br/>MilliQ H<sub>2</sub>O to volume <br/>Prepare freshly</li><li>NTMT <em>minus</em> buffer<br/>Final concentration:<br/>100 mM NaCl<br/>100 mM Tris-HCl<br/><strong><em>For 100 ml add:</em></strong><br/>2 ml of 5 M NaCl<br/>10 ml of 1 M Tris-HCl (pH 9.5)<br/>1 ml of Tween-20<br/>MilliQ H<sub>2</sub>O to volume<br/>Prepare freshly</li><li>NBT/BCIP in NTMT buffer<br/><ol class="npwthree"><li>Add to NTMT buffer:<br/>0.08 mg/ml of Nitro Blue Tetrazolium<br/>0.1 mg/ml of 5-Bromo-4-Chloro-3-Indolyl-Phosphate</li><li>Prepare freshly and protect from light</li></ol></li></ol><p class="pptt" id="biaoti29173">### Acknowledgments</p><p class="ppzz">I thank Lucas LeclÃ¨re and Evelyn Houliston for comments on the manuscript and discussions, and the other members of the Houliston/Momose group, in particular Sandra Chevalier, for discussion and assistance. I also thank all the people who contributed to testing this protocol on their organism of study, in particular Gonzalo Quiroga-Artigas, Lorenzo Ricci, Daniel Thiel. I thank the anonymous reviewers for their helpful comments on the manuscript. I also thank Enrique Arboleda for comments on the final version of the manuscript. Funding: Agence National de la Recherche, grant âMEDUSEVOâ ANR-13-PDOC-0016.</p><p class="pptt" id="biaoti29174">### Competing interests</p><p class="ppzz">The author declares there are no competing interests.</p><p class="pptt" id="biaoti29175">### References</p><ol class="npwtwok"><li>Azaripour, A., Lagerweij, T., Scharfbillig, C., Jadczak, A. E., Willershausen, B. and Van Noorden, C. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27142295" target="_blank">A survey of clearing techniques for 3D imaging of tissues with special reference to connective tissue.</a> <em>Prog Histochem Cytochem</em> 51(2): 9-23.</li><li>CICAD 31: N,N-Dimethylformamide, 2001. World Health Organization Geneva.</li><li>Cox, K. H., DeLeon, D. V., Angerer, L. M. and Angerer, R. C. (1984). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6692991" target="_blank">Detection of mRNAs in sea urchin embryos by <em>in situ</em> hybridization using asymmetric RNA probes.</a> <em>Dev Biol</em> 101(2): 485-502.</li><li>Eltoum, I., Fredenburgh, J., Myers, R. B. and Grizzle, W. E. (2001). <a href="https://doi.org/10.1179/his.2001.24.3.173" target="_blank">Introduction to the theory and practice of fixation of tissues.</a> J<em>ournal of Histotechnology</em> 24(3): 173-190.</li><li>Fail, P. A., George, J. D., Grizzle, T. B. and Heindel, J. J. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9628555" target="_blank">Formamide and dimethylformamide: reproductive assessment by continuous breeding in mice.</a> <em>Reprod Toxicol</em> 12(3): 317-332.</li><li>Fontenete, S., Carvalho, D., Guimaraes, N., Madureira, P., Figueiredo, C., Wengel, J. and Azevedo, N. F. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26969040" target="_blank">Application of locked nucleic acid-based probes in fluorescence <em>in situ</em> hybridization.</a> <em>Appl Microbiol Biotechnol</em> 100(13): 5897-5906.</li><li>George, J. D., Price, C. J., Marr, M. C., Myers, C. B. and Jahnke, G. D. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11006358" target="_blank">Evaluation of the developmental toxicity of formamide in Sprague-Dawley (CD) rats.</a> <em>Toxicol Sci</em> 57(2): 284-291.</li><li>George, J. D., Price, C. J., Marr, M. C., Myers, C. B. and Jahnke, G. D. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12215671" target="_blank">Evaluation of the developmental toxicity of formamide in New Zealand white rabbits.</a> <em>Toxicol Sci</em> 69(1): 165-174.</li><li>Gleich, J. (1974). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4276548" target="_blank">Proceedings: The influence of simple acid amides on fetal development of mice.</a> <em>Naunyn Schmiedebergs Arch Pharmacol</em> 282(Suppl): suppl 282:R225.</li><li>Haimovich, G. and Gerst, J. E. (2018). <a href="https://en.bio-protocol.org/e3070" target="_blank">Single-molecule fluorescence <em>in situ</em> hybridization (smFISH) for RNA detection in adherent animal cells.</a> <em>Bio-protocol</em> 8(21): e3070.</li><li>Herskovits, T. T. (1963). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13954139" target="_blank">Nonaqueous solutions of DNA; denaturation by urea and its methyl derivatives</a>. <em>Biochemistry</em> 2: 335-340.</li><li>Herskovits, T. T. and Bowen, J. J. (1974). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4549016" target="_blank">Solution studies of the nucleic acid bases and related model compounds. Solubility in aqueous urea and amide solutions.</a> <em>Biochemistry</em> 13(27): 5474-5483.</li><li>Huang, E., Talukder, S., Hughes, T. R., Curk, T., Zupan, B., Shaulsky, G. and Katoh-Kurasawa, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21810415" target="_blank">BzpF is a CREB-like transcription factor that regulates spore maturation and stability in <em>Dictyostelium</em>.</a> <em>Dev Biol</em> 358(1): 137-146.</li><li>Hutton, J. R. (1977). <a href="http://www.ncbi.nlm.nih.gov/pubmed/928068" target="_blank">Renaturation kinetics and thermal stability of DNA in aqueous solutions of formamide and urea.</a> <em>Nucleic Acids Res</em> 4(10): 3537-3555.</li><li>Kennedy, G. L. and Short, R. D. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3530639" target="_blank">Biological effects of acetamide, formamide, and their monomethyl and dimethyl derivatives.</a> <em>Crit Rev Toxicol</em> 17(2): 129-182.</li><li>Lambert, D. and Draper, D. E. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23088364" target="_blank">Denaturation of RNA secondary and tertiary structure by urea: simple unfolded state models and free energy parameters account for measured m-values.</a> <em>Biochemistry</em> 51(44): 9014-9026.</li><li>LeclÃ¨re, L., Horin, C., Chevalier, S., LapÃ©bie, P., Dru, P., Peron, S., Jager, M., Condamine, T., Pottin, K., Romano, S., Steger, J., Sinigaglia, C., Barreau, C., Quiroga Artigas, G., Ruggiero, A., Fourrage, C., Kraus, J. E. M., Poulain, J., Aury, J. M., Wincker, P., Queinnec, E., Technau, U., Manuel, M., Momose, T., Houliston, E. and Copley, R. R. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30858591" target="_blank">The genome of the jellyfish Clytia hemisphaerica and the evolution of the cnidarian life-cycle.</a> <em>Nat Ecol Evol </em>3(5):801-810.</li><li>Lim, W. K., Rosgen, J. and Englander, S. W. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19196963" target="_blank">Urea, but not guanidinium, destabilizes proteins by forming hydrogen bonds to the peptide group.</a> <em>Proc Natl Acad Sci U S A</em> 106(8): 2595-2600.</li><li>Marmur, J. and Doty, P. (1961). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14470100" target="_blank">Thermal renaturation of deoxyribonucleic acids.</a> <em>J Mol Biol</em> 3: 585-594.</li><li>Manil-Segalen, M., Luksza, M., Kanaan, J., Marthiens, V., Lane, S. I. R., Jones, K. T., Terret, M. E., Basto, R. and Verlhac, M. H. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30082296" target="_blank">Chromosome structural anomalies due to aberrant spindle forces exerted at gene editing sites in meiosis.</a> <em>J Cell Biol</em> 217(10): 3416-3430.</li><li>Merkle, J. and Zeller, H. (1980). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7193024" target="_blank">Studies on acetamides and formamides for embryotoxic and teratogenic activities in the rabbit (author's transl).</a> <em>Arzneimittelforschung</em> 30(9): 1557-1562.</li><li>Pardue, M. L. and Gall, J. G. (1969). <a href="http://www.ncbi.nlm.nih.gov/pubmed/5261036" target="_blank">Molecular hybridization of radioactive DNA to the DNA of cytological preparations.</a> <em>Proc Natl Acad Sci U S A</em> 64(2): 600-604.</li><li>Priyakumar, U. D., Hyeon, C., Thirumalai, D. and Mackerell, A. D., Jr. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19919063" target="_blank">Urea destabilizes RNA by forming stacking interactions and multiple hydrogen bonds with nucleic acid bases.</a> <em>J Am Chem Soc</em> 131(49): 17759-17761.</li><li>Quiroga Artigas, G., Lapebie, P., LeclÃ¨re, L., Takeda, N., Deguchi, R., Jekely, G., Momose, T. and Houliston, E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29303477" target="_blank">A gonad-expressed opsin mediates light-induced spawning in the jellyfish <em>Clytia</em>.</a> <em>Elife</em> 7. e29555.</li><li>Simard, C., Lemieux, R. and Cote, S. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11545392" target="_blank">Urea substitutes toxic formamide as destabilizing agent in nucleic acid hybridizations with RNA probes.</a> <em>Electrophoresis</em> 22(13): 2679-2683.</li><li>Sinigaglia, C., Thiel, D., Hejnol, A., Houliston, E. and LeclÃ¨re, L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29197505" target="_blank">A safer, urea-based <em>in situ</em> hybridization method improves detection of gene expression in diverse animal species.</a> <em>Dev Biol</em> 434(1): 15-23.</li><li>Sive, H. L., Grainger, R. M. and Harland, R. M. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21357142" target="_blank">Baskets for <em>in situ</em> hybridization and immunohistochemistry.</a> <em>CSH Protoc</em> 2007: pdb prot4777.</li><li>Stula, E. F. and Krauss, W. C. (1977). <a href="http://www.ncbi.nlm.nih.gov/pubmed/898190" target="_blank">Embryotoxicity in rats and rabbits from cutaneous application of amide-type solvents and substituted ureas.</a> <em>Toxicol Appl Pharmacol</em> 41(1): 35-55.</li><li>Thiel, D., Bauknecht, P., Jekely, G. and Hejnol, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28835571" target="_blank">An ancient FMRFamide-related peptide-receptor pair induces defence behaviour in a brachiopod larva.</a> <em>Open Biol</em> 7(8). pii: 170136.</li><li>Tainaka, K., Kuno, A., Kubota, S. I., Murakami, T. and Ueda, H. R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27298088" target="_blank">Chemical principles in tissue clearing and staining protocols for whole-body cell profiling.</a> <em>Annu Rev Cell Dev Biol</em> 32: 713-741.<br/></li></ol>