<body>
            <h1>Production of Quantum Dots-containing Influenza Virus Particles for Studying Viral Uncoating Processes</h1>
            <p><strong>Authors:</strong> Chong Qin, Xiaowei Zhang, Zongqiang   Cui</p>
            <h2>Abstract</h2>
            <p>The genome of influenza A virus (IAV) comprises eight pinlike genomic segments called vRNPs enclosed in viral capsid. During infection, uncoating is the key step for viral replication and represents an antiviral therapeutic target, but it is difficult to observe the transient and dynamic event in detail. Here, we report a protocol for production of quantum dots-containing influenza virus particles by encapsulating quantum dot-conjugated vRNPs during viral assembly. These labeled virions can be used for monitoring viral trafficking in real time and studying viral uncoating processes.</p>
            <p><strong>Keywords:</strong> Influenza A virus, vRNP, Quantum dots, Viral assembly, Uncoating</p>
            <h2>Background</h2><p>Influenza A virus (IAV) is one of the most important human pathogens. Viral entry before virus-endosome fusion has been widely studied by various methods, including single-particle tracking. However, the viral uncoating process following endocytosis remains elusive. Robust live imaging strategies that offer high temporal and spatial resolution are essential to study this dynamic and transient event. However, the influenza RNA genome is intolerant to insertion of large genetic materials, and rescue of viruses using fluorescent protein fused to viral genomic core proteins have been limited (Lakdawala <em>et al.</em>, 2014). This has delayed progress in the field of influenza-uncoating and vRNPs-dynamics live-imaging studies. Although purified vRNPs have been dye-labeled and microinjected into cells to track their intracellular transportation and nuclear import process in living cells in real time (Babcock <em>et al.</em>, 2004), this method cannot capture the real infection process of incoming virions or determine vRNP behaviors during uncoating. Fluorescence <em>in situ</em> hybridization (FISH) and colocalization analyses with single-molecule resolution have also been used to study the behavior of vRNPs in infected cells (Chou <em>et al.</em>, 2013), but the fixation procedures are not compatible with imaging vRNP dynamics in living cells. Semiconductor quantum dots (QDs) have unique optical properties, such as remarkable brightness and superior photostability, and are well-suited to single-particle tracking of viral infection. Single-particle tracking of virions with QDs-labeled genetic material provides an opportunity for tracking viral uncoating and post-uncoating genetic behaviors in real time. As a subunit of polymerase complex, PA protein participates in the composition of vRNP-genetic core of IAV. In PNAS, Qin <em>et al.</em> (2019) developed a nanotechnology that labels IAV viral ribonucleoprotein complexes (vRNPs) with QDs. Briefly, the biotin acceptor peptide (AP tag) was genetically fused to the PA protein to construct a recombinant PR8 strain. The rescued rPR8-PA-AP virus was then cultured in BirA-expressing MDCK cells in the presence of biotin. Streptavidin-coated QDs (SA-QDs) were introduced into these cells by lipofection to allow noncovalent binding of the QDs to biotinylated PA. QDs-containing influenza virus particles were thus generated. This approach will advance the mechanistic understanding of influenza virus uncoating using live fluorescence microscopy.</p>
        </body>

<p class="pptt" id="biaoti29295">## Materials and Reagents</p><ol class="npwone"><li>Materials <br/><ol class="npwtwo"><li>1.5 ml graduated microcentrifuge tubes (QSP, catalog number: 509-GRD-Q)</li><li>2 ml graduated microcentrifuge tubes (QSP, catalog number: 508-GRD-Q)</li><li>SW32 Ultra-Clear Tubes (Beckman Coulter, catalog number: 344058)</li><li>SW41 Ultra-Clear Tubes (Beckman Coulter, catalog number: 344059)</li><li>0.1-10 Âµl pipette tips (QSP, catalog number: T104RS-Q)</li><li>10-200 Âµl pipette tips (QSP, catalog number: TTW110RS-Q)</li><li>100-1250 Âµl pipette tips (QSP, catalog number: T112NXLRS-Q)</li><li>12-well plates (Corning Costar, catalog number: 3513)</li><li>6-well plates (Thermo Scientific, catalog number: 140675)</li><li>V-bottom 96-well plates (Solarbio, catalog number: YA0590)</li><li>Scalpel Blade (Hong Yue, catalog number: GC-HY00388)<br/>(link: <a href="https://www.casmart.com.cn/product-details/page/103/11044449" target="_blank">https://www.casmart.com.cn/product-details/page/103/11044449</a>)</li><li>100 mm Nunc EasYDishes (Thermo Scientific, catalog number: 150464)</li><li>75 cm<sup>2</sup> Nunc EasYFlasks (Thermo Scientific, catalog number: 156472)</li><li>25 cm<sup>2</sup> Nunc EasYFlasks (Thermo Scientific, catalog number: 156340)</li><li>0.45 Î¼m filter (Millex-HV, catalog number: SLHU033RB)</li><li>Filter paper (Biorad, catalog number: 1703932)</li><li>3 ml Pasteur pipettes (Nest, catalog number: 318314)</li><li>1 ml syringes (Gemtier, catalog number: JT-001)</li><li>10 ml syringes (Gemtier, catalog number: JT-010)</li><li>15 ml centrifuge tubes (Nest, catalog number: 601052)</li><li>50 ml centrifuge tubes (Nest, catalog number: 602052)<br/><br/></li></ol></li><li>Cell lines<ol class="npwtwo" start="1"><li>293T cells (Boster, catalog number: CX0008; ATCC, catalog number: ACS-4500) </li><li>MDCK cells (Boster, catalog number: CX0206; ATCC, catalog number: CRL-2936)<br/><br/></li></ol></li><li>Plasmids<br/><em>Note: All plasmids are available from our lab. BirA is an enzyme that catalyzes ligation of Biotin and AP tag.</em><br/><ol class="npwtwo" start="1"><li>Eight plasmids (pHW2000) each containing individual cDNA segment of the influenza virus strain A/Puerto Rico/8/34 (H1N1) (Hoffmann <em>et al.</em>, 2000)</li><li>pcDNA3.1(+)-BirA<br/><br/></li></ol></li><li>Primers<br/>Linearized Vector-F: GTGCTACTATTTGCTATCCATACTGTCC<br/>Linearized Vector-R: ACATCTTCTCCAATCTCATCGAGCTCTATC<br/>Insert-F1: GAAAATCGAATGGCACGAATAGTCTAGAAAACTGCTTCTTATCGTTCAGG<br/>Insert-R1: TGGACAGTATGGATAGCAAATAGTAGCAC<br/>Insert-F2: GATATCTTCGAAGCTCAGAAAATCGAATGGCACGAATAGTCT<br/>Insert-R2: TGGACAGTATGGATAGCAAATAGTAGCAC<br/>Insert-F3: GAGTGGTTCAGGAGGCGGTCTGAACGATATCTTCGAAGCTCAGAAAATCGAAT<br/>Insert-R3: TGGACAGTATGGATAGCAAATAGTAGCAC<br/>Insert-F4: GATAGAGCTCGATGAGATTGGAGAAGATGT<br/>Insert-R4: GTTCAGACCGCCTCCTGAACCACTCAATGCATGTGTAAGGAAGGAGTTG<br/>Insert-F5: GATAGAGCTCGATGAGATTGGAGAAGATGT<br/>Insert-R5: TGGACAGTATGGATAGCAAATAGTAGCAC<br/>PA-F: AGCGAAAGCAGGTACTGATCCAAAATG<br/>PA-R: AGTAGAAACAAGGTACTTTTTTGGACAGTATG<br/><em>Note: All primers were synthesized by Sangon Biotech.</em><br/><br/></li><li>Reagents<ol class="npwtwo" start="1"><li>SPF embryonated chicken eggs (Merial-vital)</li><li>Competent cells DH5Î± (TIANGEN, catalog number: CB101-01)<br/>(Link: <a href="http://www.tiangen.com/?productShow/t1/6/id/160.html" target="_blank">http://www.tiangen.com/?productShow/t1/6/id/160.html</a>)</li><li>70% ethanol (Amresco, catalog number: E505)</li><li>2 kb Plus DNA ladder (TransGen, catalog number: BM121)</li><li>Ampicillin (MDBio, catalog number: 69-52-3)</li><li>SA-QD625 (Invitrogen, catalog number: A10196)</li><li>KOD FX DNA polymerase enzyme (TOYOBO, catalog number: KFX-101)</li><li>T4 DNA polymerase (NEB, catalog number: M0203S)</li><li>50x TAE buffer (Sangon Biotech, catalog number: B548101-0500)</li><li>Agarose G-10 (BIOWESTE, catalog number: 111860)</li><li>Omega Cycle-Pure Kit (OMEGA, catalog number: D6492-01)</li><li>Omega Viral RNA Kit (OMEGA, catalog number: R6874-02)</li><li>Omega gel extraction kit (OMEGA, catalog number: D2500-02)</li><li>Omega Plasmid Mini kit (OMEGA, catalog number: D6943-02)</li><li>One Step SYBR PrimeScript PLUS RT-PCR Kit (TaKaRa, catalog number: RR096A)</li><li>DMEM (Gibco, catalog number: 11965118)</li><li>MEM (Genom, catalog number: GNM41500)</li><li>DMEM Powder (Gibco, catalog number: 12100061)</li><li>Low gelling temperature Agarose (Sigma-Aldrich, catalog number: A9045)</li><li>FBS (Gibco, catalog number: 1787602)</li><li>7.5% BSA (Solarbio, catalog number: H1130)</li><li>TPCK-trypsin (Sigma-Aldrich, catalog number: T1426)</li><li>1x PBS (Gibco, catalog number: 10010049)</li><li>Lipofectamine 2000 (Invitrogen, catalog number: 11668019)</li><li>Biotin (Sigma-Aldrich, catalog number: B4639)</li><li>Sucrose (HUSHI, catalog number: 10021418) <br/>(Link: <a href="https://www.reagent.com.cn/goodsDetail/94d364146ee0495d955089dbd33dade3" target="_blank">https://www.reagent.com.cn/goodsDetail/94d364146ee0495d955089dbd33dade3</a>)</li><li>NaCl (HUSHI, catalog number: 10019318) <br/>(Link: <a href="https:/www.reagent.com.cn/goodsDetail/d8517dae2fd14870aeeaddbb8b0a13f4" target="_blank">https://www.reagent.com.cn/goodsDetail/d8517dae2fd14870aeeaddbb8b0a13f4</a>)</li><li>Tryptone (OXOID, catalog number: LP0042)</li><li>Yeast extract (OXOID, catalog number: LP0021)</li><li>LB medium with 100 Î¼g/ml ampicillin (see Recipes)</li><li>LB agar plate with 100 Î¼g/ml ampicillin (see Recipes)</li><li>Cell culture medium (see Recipes)</li><li>Infection medium (see Recipes)</li><li>0.5% Chicken red blood cell (RBC) (see Recipes)</li><li>2x plaque medium (see Recipes)</li><li>1.6% agar (see Recipes)</li><li>Sucrose solution (see Recipes)<br/></li></ol></li></ol><p class="pptt" id="biaoti29296">### Equipment</p><ol class="npwtwok"><li>Eppendorf Pipettes (0.1-10 Âµl, 10-200 Âµl, 100-1250 Âµl)</li><li>6 W UV transilluminator (UVP, model: UVLM-26)</li><li>Rotating incubator (FUMA, model: QYC-200)</li><li>Centrifuge (Eppendorf, model: 5810R)</li><li>Ultracentrifuge (Beckman, model: OPTIMA XE-100)</li><li>Water bath (YIHENG17, model: DK-8D)</li><li>Beckman SW32 rotor</li><li>Beckman SW41 rotor</li><li>PCR machine (GnenAmp, PCR System 9700)</li><li>100 kV Transmission electron microscope (Hitachi, model: H-7000 FA)<br/></li></ol><p class="pptt" id="biaoti29297">### Procedure</p><ol class="npwone"><li>Construction of pHW2000-PA-AP tag<br/>The PA-AP segment was constructed by inserting a sequence encoding the short linker GSGG (Nucleotide sequence: GGTTCAGGAGGC), a 15-amino acid biotin AP tag (Nucleotide sequence: GGTCTGAACGATATCTTCGAGCTCAGAAAATCGAATGGCACGAA), and a duplication of 165 base pairs of the 3Ê¹ end of the PA-coding region after the PA ORF (stop codon deleted) using one-step sequence and ligation-independent cloning method (Jeong <em>et al.</em>, 2012). Schematic of pHW2000 PA-AP tag is shown in Figure 1.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914042923_8544.jpg"/><br/><strong>Figure 1. Schematic of the constructed plasmid pHW2000 PA-AP</strong><br/><br/><ol class="npwtwo"><li>PCR amplification and gel purification of linearized vector fragment<br/><ol class="npwthree"><li>The linearized vector fragment (pHW2000: 2,980 bp, partial PA sequence: 1,368 bp) is amplified using the KOD FX DNA polymerase enzyme in a PCR machine, according to the manufacturerâs protocol. Reaction system (Table 1) and parameters (Table 2) are shown as follows:<br/><br/><strong>Table 1. The PCR reaction mixture used for the linearized vector fragment</strong><br/><img alt="" class="layerphoto" height="245" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043002_0582.png" width="520"/><br/><br/><strong>Table 2. The PCR reaction parameters used for the linearized vector fragment</strong><br/><img alt="" class="layerphoto" height="229" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043237_0894.png" width="520"/><br/><br/></li><li>Prepare a 1% agarose-TAE gel (Add 3 Î¼l of 1 mg/ml EB into 30 ml solution for a gel).</li><li>Add 1/10 volume of 10x DNA loading dye to each PCR sample and load 50 Î¼l reaction onto the gel. In addition, load the 2 kb Plus DNA ladder for size verification.</li><li>Run the gel in 1x TAE until the bands of the loading dye run to approximately three quarters of the way down the gel. <br/></li><li>Visualize the PCR product on a UV transilluminator. Cut the ~4,300 bp band using a clean blade. Place the gel slice into a clean 1.5 ml microcentrifuge tube.<br/></li><li>Purify the PCR product using the Omega gel extraction kit as per the manufacturer's protocol. Elute the linearized vector fragment PCR product with 50 Î¼l elution buffer.</li></ol></li><li>PCR amplification and gel purification of insert fragment<br/><ol class="npwthree"><li>Acquire the insert fragment (partial PA sequence: 895 bp, linker-AP tag: 45 bp, and duplicated packaging sequence of PA gene: 205 bp) through a 5-steps PCR procedure. For a flow chart of these steps see Figure 2.<br/><ol class="npwfour"><li>Firstly, amplify duplicated packaging fragment from PA gene with Insert-F1 and Insert-R1 primers.</li><li>Then duplicated packaging fragment is used as the template to amplify AP tag-duplicated packaging fragment with Insert-F2 and Insert-R2 primers.</li><li>The same linker-AP tag-duplicated packaging fragment can be acquired in the third step.</li><li>Partial PA fragment is acquired from PA gene using Insert-F4 and Insert-R4 primers in the fourth step. <br/></li><li>Finally, the linker-AP tag-duplicated packaging fragment and partial PA fragment are fused together and amplified using Insert-F5 and Insert-R5 primers by overlap extension PCR.</li></ol><br/><img alt="" class="layerphoto" height="210" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043427_2661.jpg" width="300"/><br/><strong>Figure 2. Flow chart diagram of the insert fragment acquisition.</strong> A. Schematic diagram of pHW2000 PA-AP. B. Flow chart diagram of the 5-steps PCR procedure.<br/><br/></li><li>PCR amplification and gel purification are performed as described above with reference to linearized vector fragments. Elute the Purified PCR product (~1,200 bp) with 50 Î¼l elution buffer.</li></ol></li><li>One-step sequence- and ligation-independent cloning<br/><ol class="npwthree"><li>Mix the linearized vector fragment and insert fragment at a molar ratio of 1:2 in a 1.5 ml tube. Reaction system is shown in Table 3.<br/><br/><strong>Table 3. Reaction system of One-Step sequence- and ligation-independent cloning</strong><br/><img alt="" class="layerphoto" height="170" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043455_6422.png" width="660"/><br/><br/></li><li>Add 0.5 Î¼l of T4 DNA polymerase (3 U/Î¼l, NEB) to the mixture and incubate at room temperature for 2.5 min. Then, put the reaction mixture on ice immediately and incubate for 10 min. At the same time, thaw chemically competent cells DH5Î± (200 Î¼l) on ice for ~10 min.</li><li>Gently mix the cells with 10 Î¼l of the reaction mixture and incubate the cells on ice for 30 min. Heat shock the cells at 42 Â°C for 90 s. Incubate the cells on ice for 2 min. Add 800 Î¼l of LB medium to the cells and transfer the cells to 15 ml glass tube. Incubate the cells at 37 Â°C for 1 h. Then plate the cells on agar plates containing ampicillin antibiotics. Normally, we spread 200 Î¼l of culture onto an agar plate. Incubate the plates upside down at 37 Â°C for 16 h.</li><li>Pick six colonies and inoculate each into LB medium supplemented with ampicillin in 15 ml glass tubes. Grow overnight in a 37 Â°C rotating incubator. Extract plasmid DNA from the above cultures using the Omega Plasmid Mini kit as per the manufacturerâs protocol.<br/></li><li>Sequence clones with PA-F and PA-R sequencing primers to confirm the proper insertion of linker-AP tag-duplicated packaging sequence into the pHW2000-PA backbone.<br/></li><li>The resulting plasmid is herein referred to as pHW2000-PA-AP tag.</li></ol><br/></li></ol></li><li>Rescue of infectious recombinant rPR8-PA-AP<ol class="npwtwo" start="1"><li>Transfect the cells with rPR8-PA-AP rescue plasmids.<br/><ol class="npwthree"><li>Transfect 293T cells in Opti-MEM as per the manufacturerâs protocol for 6-well plate, and the following per well:<br/>2 x 10<sup>6</sup> 293T cells<br/>1 Î¼g pHW2000-PB2<br/>1 Î¼g pHW2000-PB1<br/>1 Î¼g pHW2000-HA<br/>1 Î¼g pHW2000-NP<br/>1 Î¼g pHW2000-NA<br/>1 Î¼g pHW2000-M<br/>1 Î¼g pHW2000-NS<br/>1 Î¼g pHW2000-PA-AP tag<br/>16 Î¼l Lipofectamine 2000<br/>1.5 ml Opti-MEM<br/></li><li>Incubate the transfected cells for 6 h at 37 Â°C, 5% CO<sub>2</sub>. Carefully aspirate the Opti-MEM/lipofectamine mix from the transfected cells, and replace with 1.5 ml of fresh Opti-MEM. After 48 h, transfer the supernatant from the transfected cells into a 2.0 ml microcentrifuge tube. Centrifuge the supernatant for 1 min, 19,419 <em>x g</em> to remove cell debris.</li></ol></li><li>Rescue rPR8-PA-AP from transfection supernatants<br/><strong><em>Infection of 10-day-old chicken embryonated eggs </em></strong><br/><ol class="npwthree"><li>Candle the 10-day-old eggs using a light egg candler in a darkroom to mark the interface between the air sac and the allantoic cavity. Make a hole with a 10 ml syringe needle in the eggshell. Inoculate each egg with 200 Î¼l of the tissue culture supernatants using a 1 ml syringe. Cover the hole in the eggshell with melted wax, then incubate the infected eggs at 37 Â°C for 3 days.</li><li>Prior to harvesting the allantoic fluid, incubate the chicken eggs overnight at 4 Â°C to kill the embryo and coagulate the blood.</li><li>Clean the eggshells with 70% ethanol and open the embryo over the air cavity by tapping with a spoon. Remove the broken eggshell and allantoic membrane with forceps. Stabilize the chicken embryo and collect as much allantoic fluid as possible into a 15 ml centrifuge tube on ice by a 1 ml pipette. Centrifuge for 5 min, 19,419 <em>x g</em> at 4 Â°C and transfer the allantoic fluid to fresh 15 ml centrifuge tubes. Store the allantoic fluid at 4 Â°C until they are checked for the presence of rescued virus with a hemagglutination (HA) assay.</li></ol><em>Note: For more information about this process, please watch the jove video (</em><a href="https://www.jove.com/video/2057/generation-of-recombinant-influenza-virus-from-plasmid-dna" target="_blank"><em>https://www.jove.com/video/2057/generation-of-recombinant-influenza-virus-from-plasmid-dna</em></a><em>). Chicken eggs are SPF-level eggs. Infected-eggs are discarded in autoclavable bags and autoclaved following standard procedures. (MartÃ­nez-Sobrido and Adolfo GarcÃ­a-Sastre, 2010)</em><br/><br/><strong><em>HA assays</em></strong><br/><ol class="npwthree"><li>HA assays are carried out in V-bottom 96-well plates. Add 50 Î¼l of 1x PBS into each well of the V-bottom 96-well plate.</li><li>Add 50 Î¼l of allantoic fluid to the first well and make 2-fold serial dilutions for the following wells. Discard the extra 50 Î¼l from the last well.</li><li>Add 50 Î¼l of 1.0% chicken red blood cells (prepared in 1x PBS) to each well.</li><li>Incubate the V-bottom 96-well plate for 30 min (until a red dot is visible in the bottom) at RT. A positive viral rescue can be confirmed by HA assay (Figure 3).</li></ol><em>Note: For more information about haemagglutination inhibition testing, please see Manual for the Laboratory Diagnosis and Virological Surveillance of Influenza, WHO Part 2.E.</em><br/><em> <strong><br/>Plaque purification of rescued rPR8-PA-AP</strong></em><br/><ol class="npwthree"><li>Seed 2x 10<sup>5</sup> MDCK cells/well to 90% confluency in 12-well plates using 1.5 ml growth medium per well.</li><li>Dilute HA positive allantoic fluid and perform eleven 1:10 dilutions using infection medium.</li><li>Heat 1.2% agar until dissolved and place it in a 56 Â°C water bath. Add TPCK-trypsin to the 2x plaque assay medium at concentration of 2 Î¼g/ml and place it in a 37 Â°C water bath.</li><li>Aspirate the growth medium from the MDCK cells, and wash thrice in 1x PBS.<br/></li><li>Add the diluted samples (200 Î¼l) to each 12-well plate, then incubate the well plate for 1 h at 37 Â°C, 5% CO<sub>2</sub> and shake the plate every 15 min.<br/></li><li>Carefully aspirate the inoculums from the MDCK cells, and wash the cells once in 1x PBS.<br/></li><li>Mix 2x plaque assay medium plus TPCK-trypsin (7 ml) with 1.2% agar (7 ml) with a ratio of 1:1 and immediately overlay each well with 1 ml mixture, and solidify at room temperature for 30 min.<br/></li><li>Invert the plate and incubate for 48-72 h at 37 Â°C, 5% CO<sub>2</sub> until the plaques are visible (Figure 4A).<br/></li><li>Outline the individual plaques and pick about 10 plaques. Using a 200 Î¼l pipet tip, stab the plaque and scrape the cells forming the plaque. Inoculate each plaque into 500 Î¼l of infection medium.<br/></li><li>Detect and sequence PA-AP gene from the plaques by RT-PCR using the OMEGA Viral RNA kit and TaKaRa one step SYBR PrimeScript PLUS RT-PCR Kit with PA-F and PA-R primers following the manufacturerâs protocol. The correct clonal stocks are amplified in chicken eggs for further use.</li></ol><em>Note: For more information about plaque assay protocol, please see Immunoplaque Assay (Influenza Virus), Bio-protocol (Tse et al., 2013).</em><br/><br/></li></ol></li><li>Production of IAV-QDs<br/><ol class="npwtwo"><li>Seed 5x 10<sup>6</sup> MDCK cells/well to reach 80% confluency in 100 mm dishes using 10 ml growth medium per dish.</li><li>Transfect MDCK cells in Opti-MEM as per the manufacturerâs protocol.<br/>15 Âµg pcDNA3.1(+)-BirA<br/>37.5 Âµl Lipofectamine 2000<br/>6 ml Opti-MEM containing 50 ÂµM biotin<br/></li><li>Incubate the transfected cells for 6 h at 37 Â°C, 5% CO<sub>2</sub>. Aspirate the Opti MEM/ lipofectamine mix from the transfected cells and wash twice in 1x PBS.</li><li>Incubate with rPR8 PA-AP viruses at a multiplicity of infection (MOI) of 10 about 1 ml for 1 h at 37 Â°C, 5% CO<sub>2</sub>. Rock the dishes every 15 min to keep the inoculums evenly distributed.<br/></li><li>Remove the inoculums from the MDCK cells, wash twice in 1x PBS and add 10 ml infection medium containing 50 ÂµM biotin.<br/></li><li>After a 2-h incubation, deliver SA-QD625 into MDCK cells by liposomal transfection:<br/>4 Î¼l SA-QD625<br/>10 Î¼l Lipofectamine 2000<br/>1 ml Opti-MEM<br/></li><li>Harvest cell culture supernatants every 12 h after virus infection and supplement with fresh infection medium containing 50 ÂµM biotin until 3 days post-infection or ~80% CPE.<br/><em>Note: The purpose of adding biotin during the 6 h incubation after the transfection of the BirA is to internalize biotin into cells before PA protein expression. Biotinylation of PA protein is dependent on BirA catalyzing the ligation of biotin to AP tag. Quantum dots introduced into cells are coupled with biotinylated PA proteins during virus infection.</em> </li><li>Centrifuge the collected supernatants at 4 Â°C, 19,419<em> x g</em> for 10 min and filter through a 0.45 Î¼m filter to remove cell debris.</li><li>Centrifuge at 4 Â°C, 153,125 <em>x g</em> for 2.5 h with Beckman SW32 rotor on 5 ml of a 30% (w/v) sucrose cushion in PBS to concentrate the supernatants. Then layer the pellet onto a 20-60% sucrose density gradient and centrifuge at 4 Â°C, 245,853 <em>x g</em> for 3.5 h with SW41 rotor.</li><li>Collect fractions containing QD625 under UV excitation (Figure 5), and centrifuge at 4 Â°C, 161,560 <em>x g</em> for 1.5 h to remove sucrose. </li><li>Resuspend purified virus in PBS (200-500 Âµl) to acquire quantum dots-containing influenza virus particles (IAV-QDs).<br/></li></ol><em>Note: For more information about concentration of virus by ultracentrifugation, please see Influenza Virus Methods in Molecular Biology part 5: Purification and Proteomics of Influenza Virions (Hutchinson </em><em>et al.</em><em>, 2018).</em><br/><br/></li><li>Transmission electron microscopy of IAV-QDs<br/><ol class="npwtwo"><li>Load IAV-QDs sample onto carbon-coated copper grids, adsorb for 5 to 10 min, and remove the excess liquid using filter papers.</li><li>Wash the grids once with PBS and negatively stain with 10 Î¼l of 1% phosphotungstate (PTA) for 20 s at room temperature.</li><li>Remove the excess phosphotungstate using filter papers and let the copper grids dry naturally.</li><li>Examine the prepared copper grids under a transmission electron microscope.</li></ol><em>Note: For more information about transmission electron microscopy for virus detection, please see Application of transmission electron microscopy to the clinical study of viral and bacterial infections: Present and future (</em><a href="https://doi: 10.1016/j.micron.2005.10.001" target="_blank"><em>https://doi: 10.1016/j.micron.2005.10.001</em></a><em>) (Curry et al., 2006).</em><br/></li></ol><p class="pptt" id="biaoti29298">### Data analysis</p><ol class="npwtwok" start="1"><li>For virus rescue, read the results as indicated in Figure 3 Virus rescue causes hemagglutination of RBC, while no rescue of virus causes the formation of a red pellet of RBC in the bottom of the well.<br/><br/><img alt="" class="layerphoto" height="104" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043537_6556.jpg" width="300"/><br/><strong>Figure 3. Hemagglutinin assay.</strong> A representative result from an HA assay is shown with no detectable levels of virus (top) or presence of virus (bottom).<br/><br/></li><li>For plaque purification, we picked 10 plaques, as shown in Figure 4A, 5 small plaques (white arrow) and 5 large plaque (black arrow). A single band of the expected size was detected from the purified rPR8-PA-AP by RT-PCR, which is larger than the wild-type PA gene due to insertion of the foreign sequence (Figure 4B).<br/><br/><img alt="" class="layerphoto" height="167" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043604_6842.jpg" width="300"/><br/><strong>Figure 4. Virus purification.</strong> A. Viral plaques appear as white round spots. White arrow indicates small plaque, black arrow indicates large plaque. B. RT-PCR detection of PA-AP gene of purified rPR8-PA-AP.<br/><br/></li><li>For IAV-QDs production, after sucrose density gradient centrifugation, the labeled virus band shows red color under UV excitation and is located in the 50% sucrose layer, as shown in Figure 5.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043626_4203.jpg" width="100"/><br/><strong>Figure 5. Sucrose density gradient ultracentrifugation.</strong> Arrow indicates IAV-QD625 band under UV excitation.<br/><br/></li><li>For TEM imaging of IAV-QDs, we tested different times and concentration gradients to determine the optimal conditions for IAV-QDs staining. We set staining times from 5 s to 3 min and PTA concentration from 0.25% to 2%. Under the condition of staining with 1% PTA for 20 s, we get the highest-resolution images, as shown in Figure 6. TEM images show the structure of virus as well as dark quantum dots clearly.<br/><br/><img alt="" class="layerphoto" height="284" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190914/20190914043658_3875.jpg" width="300"/><br/><strong>Figure 6. TEM images of IAV-QD625.</strong> The dark round dots indicate QD625s, Scale bar: 50 nm.<br/></li></ol><p class="pptt" id="biaoti29299">### Recipes</p><ol class="npwtwok" start="1"><li>LB medium with 100 Î¼g/ml ampicillin<br/>10 g NaCl<br/>10 g Tryptone<br/>5 g yeast extract<br/>1 L ddH<sub>2</sub>O<br/>Autoclave at 121 Â°C, 15 min and let cool to RT<br/>1 ml of 100 mg/ml ampicillin </li><li>LB agar plate with 100 Î¼g/ml ampicillin<br/>2 g NaCl<br/>2 g Tryptone<br/>1 g yeast extract<br/>3 g agar<br/>200 ml ddH<sub>2</sub>O<br/>Autoclave at 121 Â°C, 15 min and let cool to 55 Â°C<br/>200 Î¼l of 100 mg/ml ampicillin</li><li>Cell culture medium<br/>445 ml DMEM<br/>50 ml FBS<br/>5 ml penicillin/streptomycin</li><li>Infection medium<br/>429 ml MEM<br/>66 ml 7.5% BSA<br/>5 ml penicillin/streptomycin<br/>0.5 Î¼g/ml TPCK-trypsin (final concentration)</li><li>0.5% Chicken red blood cell (RBC)<br/>0.5 ml RBC<br/>100 ml physiological saline solution</li><li>2x plaque medium<br/>100 ml 2x DMEM<br/>1 Î¼g/ml TPCK-trypsin</li><li>1.6% agar<br/>1.6 g low gelling temperature agarose<br/>100 ml ddH<sub>2</sub>O<br/>Autoclave at 121 Â°C, 15 min</li><li>Sucrose solution<br/>20% (w/v) sucrose: 20 g sucrose, 100 ml 1x PBS<br/>30% (w/v) sucrose: 30 g sucrose, 100 ml 1x PBS<br/>40% (w/v) sucrose: 40 g sucrose, 100 ml 1x PBS<br/>50% (w/v) sucrose: 50 g sucrose, 100 ml 1x PBS<br/>60% (w/v) sucrose: 60 g sucrose, 100 ml 1x PBS<br/></li></ol><p class="pptt" id="biaoti29300">### Acknowledgments</p><p class="ppzz" style="padding-left:20px;">This research was supported by the Strategic Priority Research Program of the Chinese Academy of Sciences (Grant XDB29050201), the National Key Research and Development Program of China (Grant 2018ZX10301405), the National Natural Science Foundation of China (Grants 31470269, 21727816, 91743108, and 31470837), and the Youth Innovation Promotion Association of the Chinese Academy of Sciences. This protocol is mainly derived from the experimental procedures used in our previous work (Qin <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti29301">### Competing interests</p><p class="ppzz" style="padding-left:20px;">The authors declare that they do not have any conflicts of interests or competing interests.</p><p class="pptt" id="biaoti29302">### References</p><ol class="npwtwok" start="1"><li>Babcock, H. P., Chen, C. and Zhuang, X. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15454466" target="_blank">Using single-particle tracking to study nuclear trafficking of viral genes.</a> <em>Biophys J</em> 87(4): 2749-2758.</li><li>Chou, Y. Y., Heaton, N. S., Gao, Q., Palese, P., Singer, R. H. and Lionnet, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23671419" target="_blank">Colocalization of different influenza viral RNA segments in the cytoplasm before viral budding as shown by single-molecule sensitivity FISH analysis.</a> <em>PLoS Pathog</em> 9(5): e1003358.</li><li>Curry, A., Appleton, H. and Dowsett, B. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16361103" target="_blank">Application of transmission electron microscopy to the clinical study of viral and bacterial infections: present and future.</a> <em>Micron</em> 37(2): 91-106.</li><li>Hoffmann, E., Neumann, G., Kawaoka, Y., Hobom, G. and Webster, R. G. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10801978" target="_blank">A DNA transfection system for generation of influenza A virus from eight plasmids.</a> <em>Proc Natl Acad Sci U S A</em> 97(11): 6108-6113.</li><li>Hutchinson E.C. and Stegmann M. (2018). <a href="https://www.researchgate.net/publication/327265589_Purification_and_Proteomics_of_Influenza_Virions_Methods_and_Protocols" target="_blank">Purification and proteomics of influenza virions: methods and protocols.</a> Humana Press, New York, NY.</li><li>Jeong, J. Y., Yim, H. S., Ryu, J. Y., Lee, H. S., Lee, J. H., Seen, D. S. and Kang, S. G. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22610439" target="_blank">One-step sequence- and ligation-independent cloning as a rapid and versatile cloning method for functional genomics studies.</a> <em>Appl Environ Microbiol</em> 78(15): 5440-5443.</li><li>Lakdawala, S. S., Wu, Y., Wawrzusin, P., Kabat, J., Broadbent, A. J., Lamirande, E. W., Fodor, E., Altan-Bonnet, N., Shroff, H. and Subbarao, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24603687" target="_blank">Influenza a virus assembly intermediates fuse in the cytoplasm.</a> <em>PLoS Pathog</em> 10(3): e1003971.</li><li>Qin, C., Li, W., Li, Q., Yin, W., Zhang, X., Zhang, Z., Zhang, X. E. and Cui, Z. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30626642" target="_blank">Real-time dissection of dynamic uncoating of individual influenza viruses.</a> <em>Proc Natl Acad Sci U S A</em> 116(7): 2577-2582.</li><li>Martinez-Sobrido, L. and Garcia-Sastre, A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20729804" target="_blank">Generation of recombinant influenza virus from plasmid DNA.</a> <em>J Vis Exp</em>(42): e2057.</li><li>Tse, L. V., Zhang, Y., Whittaker, G. R. (2013). <a href="https://en.bio-protocol.org/e959" target="_blank">Immunoplaque assay (Influenza virus).</a> <em>Bio-protocol</em> 3(21): e959.</li><li>WHO global influenza surveillance network. (2011). <a href="http://citeseerx.ist.psu.edu/viewdoc/download;jsessionid=745F6DA176D40E384C74AE1A25E5A210?doi=10.1.1.644.5378&amp;rep=rep1&amp;type=pdf" target="_blank">Manual for the Laboratory Diagnosis and Virological Surveillance of Influenza.</a><br/></li></ol>
