<body>
            <h1>A PhoA-STII Based Method for Efficient Extracellular Secretion and Purification of Fab from <em>Escherichia coli</em></h1>
            <p><strong>Authors:</strong> Ziyan Wang, Yang Gao, Manyu Luo, Cedric Cagliero, Hua Jiang, Yueqing Xie, Jianwei Zhu, Huili   Lu</p>
            <h2>Abstract</h2>
            <p> In comparison with full-length IgGs, antigen binding fragments (Fabs) are smaller in size and do not require the complexed post-translational modification. Therefore, Fab can be cost-effectively produced using an <em>Escherichia coli</em> (<em>E. coli</em>) expression system. However, the disulfide-bonds containing exogenous protein, including Fab, tend to form insoluble inclusion bodies in <em>E. coli</em>, which has been the bottleneck for exogenous protein expressions using this system. The secretory expression of proteins in periplasm or extracellular medium are promising strategies to prevent the formation of inclusion bodies to improve the efficiency to produce Fabs from <em>E. coli</em>. The extracellular expression is of particularly interest since it releases the product into the medium, while periplasmic expression yield is limited to the periplasm space. In addition, the extracellular expression allows for the direct harvesting of proteins from the culture supernatant, sparing the procedures of cell lysis and reducing contamination of host cell protein or DNA. Using anti-VEGF Fab as an example, here we provide a protocol based on the alkaline phosphatase (phoA) promoter and the heat-stable enterotoxin II (STII) leader sequence. Using phosphate starvation to induce the secretory expression, the protocol could be generally used for the efficient production of Fabs. </p>
            <p><strong>Keywords:</strong> Fab, PhoA, STII, Extracellular production, Secretion</p>
            <h2>Background</h2><p>Due to its clear genetic background, easy manipulation, and cost-effective production, <em>E. coli</em> is widely employed for exogenous gene expression, especially those of lower molecular weight and simpler conformational structures (Gupta and Shukla, 2017). The expression of foreign proteins in <em>E. coli</em> is mainly divided into three categories, inclusion bodies expression, intracellular soluble expression in periplasmic space, and extracellular secretion into medium (Jalalirad, 2013; Gupta and Shukla, 2017; Zhou <em>et al.</em>, 2018). At present, most of mammalian-sourced foreign proteins are expressed in <em>E. coli</em> intracellular region, either in the form of inclusion bodies or soluble. But both forms have their respective disadvantages for subsequent processes. The inclusion bodies need to be tediously denatured and renatured to recover the target proteins with correct refolded structures, and the processes tend to cause reduction in the biological activity and yield (Panda <em>et al.</em>, 2003; Nelson and Reichert, 2009). The periplasmic space of <em>E. coli</em> can provide an oxidative environment conducive to the formation of disulfide bonds, but the yield is usually limited by the capacity of the periplasmic space as well as the leading capability of signal peptides (Lobstein <em>et al.</em>, 2012; Ellis <em>et al.</em>, 2017). Compared with the two processes above, extracellular protein expression is not restricted to the intracellular space, allows convenient enrichment of target proteins with right structures, thus simplifies the downstream purification processes (Zhou <em>et al.</em>, 2018).<br/><br/>Fab fragment is of smaller size than full-length antibody, and does not require post-translational modification such as glycosylation modifications, it is particularly suitable to be produced in <em>E. coli</em> (Walsh and Jefferis, 2006; Rezaie <em>et al.</em>, 2017). We recently reported a method for efficient extracellular expression and purification of Fabs from <em>E. coli</em>, which has been optimized for many parameters (Luo <em>et al.</em>, 2019a). Here, we provide a detailed protocol using anti-VEGF Fab as a model protein, since the anti-VEGF Fab Ranibizumab was the first approved Fab drug on the market (Danyliv <em>et al.</em>, 2017). The process consists of three main parts: A) construction of pPhoA-Fab expression vector and transfection of host strain BL21(DE3); B) secretory soluble expression of Fab in <em>E. coli</em>; and C) affinity chromatography purification of Fab. We investigated the combinations with different promoters (phoA and T7) and leader peptides (STII and pelB), among them phoA-STII showed the highest yield in mass and secretion efficiency. The vector was a previously engineered pRSF (Augustine <em>et al.</em>, 2016) plasmid containing phoA promoter, hereinafter referred to as the pPhoA plasmid. The secretory expression was induced by phosphate starvation to stimulate the function of phoA promoter (Wang <em>et al.</em>, 2005). For affinity purification of Fab, the resin should be selected according to species and light chain types (Kappa or Lambda chain). For anti-VEGF Fab, which has human Kappa light chain, we used a prepacked Capto L column to purify it (Ulmer <em>et al.</em>, 2019). After purification, the final product could be further analyzed for purity, yield, and bioactivities. <br/><br/>Generally, secretory expression in <em>E. coli</em> is the most suitable process for producing disulfide bond-containing antibody fragments such as Fab, Fabâ, and (Fabâ)<sub>2</sub> (Ellis <em>et al.</em>, 2017). In comparison with previous studies about extracellular expression of Fabs, our work is superior for a considerably reduced time and cost and simplified purification process, due to the use of different expression cassette designs (two separate expression cassettes, phoA promoter, STII leader sequence), host strains (BL21[DE3]) and fermentation conditions (phosphate starvation, low temperature). We have applied the protocol to prepare five Fab fragments which have been marketed successfully (anti IGF1R, anti-Her2, anti-VEGF, anti-RANKL and anti-PD-1) with different types of IgG1/IgG2 or human/humanized structures to cover as wide as possible range of Fab fragments. The results demonstrated that they were all expressed in soluble expression, and the fractions in culture medium were more than the intracellular soluble fractions or inclusion bodies content. By one-step affinity chromatography, the purity of the Fabs reached above 94%, and all products were of correct molecular weight as well as full bioactivity against their antigens. To the best of our knowledge, the current protocol is a universal technique for efficient extracellular expression, secretion and purification of Fabs in <em>E. coli</em>.</p>
        </body>

<p class="pptt" id="biaoti29247">## Materials and Reagents</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwthree"><li><em>All the reagents could be of other brands.</em> </li><li><em>Resins could be used for both AKTA equipment and manual purification, which is depend on the sample volume to handle. For example, when the sample is more than 100 ml, it saves hand-harbor using AKTA than manual purification.</em> </li></ol><br/><ol class="npwtwo"><li>Inoculating loop</li><li>2 L Erlenmeyer flask</li><li>250 ml centrifuge bottles</li><li>Pipettes</li><li>200 Î¼l PCR tubes</li><li>1.5 ml Eppendorf tubes</li><li>0.22 Î¼m filter</li><li>Pipette tips</li><li><em>E. coli</em> strains DH5Î± (Shanghai Weidi Biotechnology, catalog number: DL1001)</li><li><em>E. coli</em> strain BL21(DE3) (Shanghai Weidi Biotechnology, catalog number: EC1002)</li><li><em>Bgl</em> II (concentration: 10 units/Î¼l) (NEB, catalog number: R0144S), store at -20 Â°C</li><li><em>Nde</em> I (concentration: 20 units/Î¼l ) (NEB, catalog number: R0111S), store at -20 Â°C</li><li>T4 DNA Ligase (concentration: 350 units/Î¼l) (Takara, catalog number:2011A), store at -20 Â°C</li><li>NaCl (Sinopharm Chemical Reagent, catalog number: 10019318)</li><li>Tryptone (Oxoid, catalog number: 2336957)</li><li>Yeast (Oxoid, catalog number: 210408)</li><li>Agar (Oxoid, catalog number: LP0011B)</li><li>Kanamycin (Sinopharm Chemical Reagent, catalog number: xw253899403)</li><li>PIPES (Sigma-Aldrich, catalog number: P1851-100G)</li><li>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (Sinopharm Chemical Reagent, catalog number: 10002917)</li><li>MgSO<sub>4</sub> (Sinopharm Chemical Reagent, catalog number: 20025117)</li><li>D(+)-Glucose (Sinopharm Chemical Reagent, catalog number: 63005518)</li><li>Tris (hydroxymethyl) aminomethane (Amresco, catalog number: 252859-100G)</li><li>Methanol (Sinopharm Chemical Reagent, catalog number:1001418)</li><li>Isopropanol (Sinopharm Chemical Reagent, catalog number: 80109218)</li><li>Absolute alcohol (Sinopharm Chemical Reagent, catalog number: 10009218)</li><li>Enhanced BCA Protein Assay Kit (Beyotime, catalog number: P0009)</li><li>NaH<sub>2</sub>PO<sub>4</sub> (Sinopharm Chemical Reagent, catalog number: 20040818) </li><li>Na<sub>2</sub>HPO<sub>4</sub> (Sinopharm Chemical Reagent, catalog number: 20040617)</li><li>K<sub>2</sub>HPO<sub>4</sub> (Sinopharm Chemical Reagent, catalog number: 20032117)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Sinopharm Chemical Reagent, catalog number: 10017618)</li><li>NaOH (Sinopharm Chemical Reagent, catalog number: 10019762)</li><li>Glycerol (Sinopharm Chemical Reagent, catalog number: 10010618)</li><li>Citric acid (BBI, catalog number: A610055-0500) </li><li>B-PER (Thermo Fisher, catalog number: 78243)</li><li>Millipore ECL (Millipore, catalog number: WBKLS0010), store at 2-8 Â°C</li><li>Capto L resins (GE Healthcare) </li><li>KappaSelect prepacked column 1 ml (GE, catalog number: 17545811)</li><li>Prestained 2x FastTaq PCR SuperMix (Miozyme, catalog number: AX111), store at -20 Â°C</li><li>PrimeSTAR Max DNA Polymerase (Takara, catalog number: R045A), store at -20 Â°C</li><li>AxyPrep<sup>TM</sup> PCR Cleanup Kit (Axygen, catalog number: AP-PCR-250G)<br/></li><li>AxyPrep<sup>TM</sup> Plasmid Miniprep Kit (Axygen, catalog number: AP-MN-P-250G)</li><li>AxyPrep<sup>TM</sup> DNA Gel Extraction Kit (Axygen, catalog number: AP-GX-250G)</li><li>PLM medium (see Recipes)</li><li>Buffer A (see Recipes)</li><li>Buffer B (see Recipes)</li><li>TAE buffer (see Recipes)</li></ol><p class="pptt" id="biaoti29248">### Equipment</p><ol class="npwtwok"><li>BS 210S electronic balance (Sartorius)</li><li>Electronic analytical balance (Mettler Toledo)</li><li>Vertical protein electrophoresis tank (Bio-Rad)</li><li>Tanon GIS Gel Imaging Analysis System (Tanon)</li><li>BECKMAN COULTER Avanti J-E Centrifuge (BECKMAN COULTER)</li><li>Centrifuge (Eppendorf, Model: Centrifuge 5418R)</li><li>LRH-70F biochemical incubator (TENSUC)</li><li>TS-200B desktop constant temperature oscillation culture shaker (TENSUC)</li><li>HH-4 constant temperature water bath (Youlian)</li><li>SW-CJ-IFD type ultra-clean workbench (Sujing)</li><li>HYCD-205 refrigerator freezer (Haier)</li><li>-80Â°C cryopreservation box (Haier)</li><li>Benchtop pH meter FE20K (Mettler Toledo)</li><li>SS325 autoclave (Tomy)</li><li>TS-2 type bleaching shaker (Qilinbeier)</li><li>Milli-Q Advantage A10 UltraPure Water Meter (Millipore)</li><li>Electric blast drying oven (Yiheng)</li><li>Thermal Cyclers for PCR (Bio-Rad)</li><li>Multi-function microplate reader Infinite M200 PRO (Tecan)</li><li><a class="blackNoUnder" target="_blank"> AKTA Avant 150 </a> (GE)<br/></li></ol><p class="pptt" id="biaoti29249">### Software</p><ol class="npwtwok"><li>Primer 5.0</li></ol><p class="pptt" id="biaoti29250">### Procedure</p><ol class="npwone"><li>Construct of anti-VEGF Fab expressing plasmids and transformation of <em>E. coli</em> host strain<br/><em>Notes:</em> <ol class="npwthree"><li><em>According to the Tm of the primers and the polymerases, set appropriate PCR reaction conditions, and select proper restriction enzymes depending on the restriction sites of the plasmids.</em> </li><li><em>The DNA sequence can be amplified from a template or synthesized by companies such as General Biosystems (Chuzhou, Anhui, China) we used.</em> </li><li><em>The backbone plasmid pPhoA can be derived from commercial plasmids such as pRSF-Duet by inserting dual phoA promoters followed by STII signal peptides.</em> </li><li><em>The transformation approach adapts to all the transformation mentioned in this protocol.</em> </li></ol><ol class="npwtwo"><li>According to the process shown in Figure 1, synthesize the entire DNA fragment encoding VL-CL, phoA promoter, STII peptide and VH-CH1 of anti-VEGF Fab expression cassette according to the published sequences. </li><li>Analyze the sequences of backbone vector and Fab fragments to determine the sites for double restriction enzymes digestion and ligation. <br/><em>Note: If there are no appreciated sites, recombinase mediated homology infusion method can be used.</em> </li><li>Design the primers with the selected restriction enzymes using Primer 5.0 software. To simplify the insertion of Fab fragments into the vector, itâs optimal to synthesize STII-VL-CL-phoA-STII-VH-CH1 by a company (for example General Biosystems, Chuzhou, Anhui, China).<br/><em>Note: To insert the target fragment into the backbone vector, we firstly determined the restriction enzymes to produce pairing sticky ends. We chose </em><em>Nde</em><em> I which holds the translational starting codon ATG as the upstream site. For the downstream site, itâs fine to choose any one listed in the multiple clone sites of the vector. In our study, we chose </em><em>Bgl</em><em> II, and the vector could be digested by </em><em>Nde</em><em> I and </em><em>Bgl</em><em> II to get the linear backbone. However, the target fragment is rather long (about 2 kb), and there are another two </em><em>Bgl</em><em> II sites, making it impossible to digest the target fragment with </em><em>Bgl</em><em> II. Therefore, we used Bsa I enzyme and designed its sticky ends the same with that of </em><em>Bgl</em><em> II. Then we designed the forward primer with an </em><em>Nde</em><em> I site and the reverse primer with a Bsa I site to amplify the fragment from the synthesized template. Finally the </em><em>Nde</em><em> I and Bsa I enzymes digested fragment was ligated with the </em><em>Nde</em><em> I and </em><em>Bgl</em><em> II enzymes digested backbone to construct the recombinant plasmid expressing anti-VEGF Fab.</em> </li><li>Extract backbone vector.</li><li>Prepare target fragments of STII-VL-CL-phoA-STII-VH-CH1.<ol class="npwthree"><li>Amplify STII-VL-CL-phoA-STII-VH-CH1 fragments with above-designed primers (Table 1) by PCR.<ol class="npwfour"><li>Add the reaction reagents in 200 Î¼l PCR tubes as shown in Table 2.<br/><br/><strong>Table 1. Sequences of primers used in PCR or sequencing</strong><br/><img alt="" class="layerphoto" height="172" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917194857_2876.jpg" width="600"/><br/>*Primers M1 and M2 hold <em>Nde</em> I and <em>Bsa</em> I digestion sites, respectively.<br/><br/><strong>Table 2. PCR reaction recipes for amplifying STII-VL-CL-phoA-STII-VH-CH1 fragments</strong><br/><img alt="" class="layerphoto" height="102" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917194923_1117.jpg" width="350"/><br/><br/></li><li>Set the PCR program to be: denaturation at 98 Â°C for 10 s, annealing at 55 Â°C for 5 s (according to the manual of PrimeSTAR Max, the annealing temperature could be set at 55 Â°C when the calculated Tm values are around 60 Â°C) and extension at 72 Â°C for 30 s (the extension time depends on the polymerase as well as the length of the target fragment, here the target fragment is about 2 kb), recycling for 35 cycles.</li></ol></li><li>Digest the STII-VL-CL-phoA-STII-VH-CH1 PCR products with <em>Nde</em> I and <em>Bsa</em> I, and digest the backbone vectors with double enzymes <em>Bgl</em> II and <em>Nde</em> I to get desired sticky ends.<ol class="npwfour"><li>Add the reaction reagents in a 1.5 ml Eppendorf tubes as Table 3.<br/><br/><strong>Table 3. Double enzyme digestion reaction recipes</strong><br/><img alt="" class="layerphoto" height="116" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917194950_2503.jpg" width="350"/><br/><br/></li><li>Incubate the mixture in a 37 Â°C water bath for 1 h.</li></ol></li><li>Separate the digested PCR fragments or backbone vectors by a 1% DNA agarose gel electrophoresis.</li><li>Retrieve the target bands from the gel with the AxyPrep<sup>TM</sup> DNA Gel Extraction Kit. </li></ol></li><li>Ligate the fragments onto the backbone vector and then transform them into <em>E. coli</em>.<ol class="npwthree"><li>Add the reaction reagents in 200 Î¼l PCR tubes as listed in Table 4.<br/><br/><strong>Table 4. Ligation reaction recipes</strong><br/><img alt="" class="layerphoto" height="161" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917195018_9308.jpg" width="300"/><br/><br/></li><li>Incubate the mixture in the thermal cycler at 16 Â°C for 30 min.</li><li>Transform the <em>E. coli</em> strain DH5Î± with the ligated mixture following the procedures as described below.</li><li>Place the <em>E. coli</em> competent cells stored at -80 Â°C on ice to thaw.</li><li>Add 10 Î¼l of ligated products into the melted <em>E. coli</em> competent cells, incubate on ice for 20 min.</li><li>Heat the mixture in a preheated 42 Â°C water bath for 80 s, followed by another 2 min incubation on ice.</li><li>Add 600 Î¼l of fresh LB broth to the tube in a clean bench.</li><li>Incubate the bacteria at 37 Â°C, 220 rpm for 40 min in a constant temperature shaker.</li><li>Centrifuge the tube at 2,150 <em>x g</em> for 5 min, discard most of the supernatant, retain about 200 Î¼l of the liquid, and re-suspend the bacteria in the clean bench.</li><li>Then spread it on a pre-prepared LB plate (include 100 Î¼g/ml kanamycin).</li><li>Incubate the plate upside-down in a 37 Â°C incubator overnight (Placing the plate upside down can prevent condensation from dripping onto the medium, avoiding contamination, and facilitating the growth, reproducibility and colony counting), then verify the clones with correct insertion.<br/><ol class="npwfour"><li>Pick a single clone in 5 Î¼l sterilized ddH<sub>2</sub>O for colony PCR verification using the primers listed in Table 1.</li><li>Add the reaction system in 200 Î¼l PCR tubes as Table 5.<br/><br/><strong>Table 5. Colony PCR reaction recipes</strong><br/><img alt="" class="layerphoto" height="116" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917195055_9325.jpg" width="350"/><br/><br/></li><li>Set the PCR program to be: denaturation at 95 Â°C for 10 s, annealing at 54 Â°C for 5 s and extension at 72 Â°C for 30 s and together for 35 cycles. The primers were listed in Table 1.</li><li>Use 1% DNA agarose gel electrophoresis using TAE buffer (Recipe 4) to verify the PCR products.</li><li>The remaining of the clones with positive PCR fragments were further cultured in LB broth for sequencing to confirm the correct insertion of desired Fab sequences. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Confirm the correct insertion of STII-VL-CL-phoA-STII- VH-CH1 fragments by sequencing, and get the recombinant pPhoA-Fab plasmid as shown in Figure 1. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917195140_8604.jpg" width="227"/><br/><strong>Figure 1. Construction process of pPhoA-Fab plasmid.</strong> STII-VL-CL-phoA-STII- VH-CH1 fragment was synthesized, PCR amplified, digested by <em>Nde</em> I and <em>Bsa</em> I, and ligated with backbone vector digested by <em>Nde</em> I and <em>Bgl</em> II, to obtain the recombinant plasmid pPhoA-Fab.<br/><br/></li><li>Culture the DH5Î± clone that sequenced to be with the correct insert overnight to extract recombinant plasmid pPhoA-Fab as shown in Figure 1 by the AxyPrep<sup>TM</sup> Plasmid Miniprep Kit.</li><li>Using the constructed pPhoA-Fab to further transform BL21 (DE3) using the same heat shock procedures (Steps A6e to A6l), and the successfully transformed clone is used as the expression host strain for anti-VEGF Fab expression.</li><li>Make the transformed bacteria DH5Î± and BL21 (DE3) aliquots, add glycerol to a final concentration of 20% and store at -80Â°C for future use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>Extracellular expression of anti-VEGF Fab<ol class="npwtwo"><li>Take a vial of BL21 (DE3) containing pPhoA-Fab from -80 Â°C, put it on ice for 5 min.</li><li>Transfer the thawed BL21 (DE3) to an LB agar plate (containing 100 Î¼g/ml kanamycin) using inoculating loop, incubate at 37 Â°C overnight.</li><li>Pick a single clone from the LB plate, and inoculate it into a tube containing 5 ml of LB broth (containing 100 Î¼g/ml kanamycin). <br/><em>Notes: </em><br/><ol class="npwthree"><li><em>It takes about 12-14 h for preculture, so itâs best to carry out this step in the evening to ensure using fresh E.coli to inoculate. </em> </li><li><em>The ingredients of LB media of various brands are consistent, so any qualified brand of LB medium can be used.</em> </li></ol></li><li>Culture the bacteria in the tube in a shaking incubator at 37 Â°C, 220 rpm for about 12-14 h. No more than 16 h.</li><li>Inoculate 5 ml culture into 800 ml of fresh LB broth containing 100 Î¼g/ml kanamycin. <br/><em>Note: Use a 2 L Erlenmeyer flask. The LB medium volume should not occupy over 40% of the full capacity of the container.</em> </li><li>Culture the bacteria at 37 Â°C, 220 rpm until OD<sub>600 </sub>reaches 0.8-1.0.</li><li>Transfer the bacteria to a sterilized 250 ml centrifuge bottles to pellet the bacteria at room temperature by centrifugation at 4,000 <em>x g</em> for 10 min. <br/><em>Note: This step should be performed in the clean bench.</em> </li><li>Discard the supernatant, re-suspend the bacterial pellet with 800 ml sterilized PLM medium. <br/><em>Note: PLM medium is chemically modified and contains no phosphate. This step should be performed in the clean bench. Here take 1 ml of culture for analysis, marked as âPreâ. </em> </li><li>Incubate the bacteria in PLM medium overnight at a temperature of 20 Â°C with a shaking speed of 200 rpm for about 12-14 h. The absence of phosphate in PLM medium promotes protein induction.</li><li>After induction, quickly raise the incubation temperature to 59 Â°C in the water bath and incubate for 1 h to improve Fab yields. <br/><em>Note: This step can improve the extracellular yield for some level, it is also OK to skip it.</em> </li><li>Then, quickly cool down the culture to room temperature in a water bath and centrifuge at 3,500 <em>x g</em> for 10 min to separate media and bacterial pellet, the expressed anti-VEGF Fab should mostly remain in the medium. </li><li>Collect the culture medium and further centrifuge at 12,000 x g for 10 min to further remove the contaminants.<br/><em>Note: Immediately before centrifugation, take 2 aliquots of culture for analysis, 1 ml for each, marked as âIndâ.</em><br/><br/></li></ol></li><li>Purification of anti-VEGF Fab<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>The Capto L binds to kappa light chains of type I, III, and IV of human IgG but not type II. </em> </li><li><em>KappaSelect resin can be used to purify protein with a type II IgG kappa light chain. The purification process is similar to that of Capto L, but using PBS buffer of pH 7.4 as Buffer A, and 0.1 M glycine buffer of pH 2.7 as Buffer B.</em> </li></ol><ol class="npwtwo"><li>Filter the expression medium through a 0.22 Î¼m filter for subsequent column chromatography purification steps.</li><li>Prepare AKTA Avant 150 by washing the A, B pumps and tubes with 20% ethanol at 2 ml/min until the UV280 reaches baseline. </li><li>Assemble the prepacked Capto L column with a column volume (CV) of 1 ml at a flow rate of 0.5 ml/min, and continue to wash the system with 20% ethanol at 2 ml/min until the UV 280 nm and Cond baselines are stable.</li><li>Then, repeat the washing with ddH<sub>2</sub>O with the same procedure until the UV 280 nm and Cond baselines are stable.</li><li>Place the B1 tube head in Buffer B and fill the tube with Buffer B.</li><li>Place the A1 tube head in Buffer A, and equilibrate the system and column for 10-20 CV until the UV 280 nm and Cond reach baseline.</li><li>Load the sample onto the column at a flow rate of 0.5 ml/min while collecting flow through. <br/><em>Note: Collect 1 ml of flow through for analysis, marked as âFLâ.</em> </li><li>After loading, wash the column with Buffer A until the UV 280 nm reaches the baseline. </li><li>Set the B pump solution ratio to 100%, elute the target protein by Buffer B with a flow rate of 1.0 ml/min, and collect the target protein when there are UV280 peaks.</li><li>Immediately adjust the pH of the collected sample with 1 M Tris buffer (pH 9.0) to prevent protein inactivation. <br/><em>Note: Take eluted protein samples for analysis, marked as âEâ.</em> </li><li>After the baseline of the elution is stable, flush the tubing with 10 to 20 CV ddH<sub>2</sub>O until the baselines of UV and Cond are stable. </li><li>Then, rinse the tubing and column with 15 mM NaOH to clean the column. After the UV280 baseline is stabilized, wash the system, column, and tubes with ddH<sub>2</sub>O and 20% ethanol successively. </li><li>Finally, fill the pipeline with 20% ethanol and disassemble the pre-packed column, which should also be kept in 20% ethanol. </li><li>Test the collected protein by SDS-PAGE and Western blot (WB).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917195214_2414.jpg" width="147"/><br/><strong>Figure 2. Scheme of the expression and purification processes for Fab production</strong> (Luo <em>et al.</em>, 2019a)  </li></ol></li></ol><p class="pptt" id="biaoti29251">### Data analysis</p><ol class="npwtwok"><li>Take one aliquot of the collected sample âIndâ and centrifuge at 3,500 <em>x g</em> for 10 min to separate medium (marked as âMediâ), and use B-PER to incubate the pellet for 15 min at room temperature to release the intracellular soluble proteins.</li><li>Centrifuge at 12,000 <em>x g</em> for 30 min at 4 Â°C to separate intracellular soluble (marked as âSupâ) and insoluble (marked as âIBâ) fractions. </li><li>To investigate the distribution of anti-VEGF Fab expression, analyze all samples Pre, Ind, Medi, Sup, and IB by SDS-PAGE (Figure 3A) and WB (Figure 3B). Use a goat anti-human kappa light chains antibody as the capture antibody, and rabbit anti-goat IgG antibody conjugated with HRP as the second antibody. </li><li>The purification process (Figure 4A) can be analyzed by SDS-PAGE of samples Medi, FL, and E (Figure 4B).</li><li>For the purity estimation of the Fab antibody, the product can be determined by size exclusion high performance liquid chromatography (SEC-HPLC) using TSK G2000SWXL column (5 Î¼m, 0.78 x 300 mm) at an Agilent 1260 HPLC system. The purity of each Fab can be calculated as the area percentage of the corresponding peak detected.</li><li>The molecular weight of the Fab can be accurately measured using an ultraperformance liquid chromatography-quadrupole time-of-flight mass spectrometer (MALDI-TOF MS). </li><li>The affinity between the Fab and the specific antigen can be tested using the ForteBio Octet RED96e system. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917195235_7554.jpg" width="172"/><br/><strong>Figure 3. Expression of anti-VEGF Fab in <em>E. coli</em>. </strong>A. SDS-PAGE; B. Western blot. Most of anti-VEGF Fab was secreted into the culture medium. Lane Pre: bacteria before induction; Ind: bacteria after induction; Medi: Medium; Sup: intracellular soluble fraction; IB: Inclusion bodies. The samples charged in the same volume and all samples were non-reducing. The star (â) indicates the predicted size of Fab at about 40 kDa (Luo et al., 2019b). <br/><br/>The samples charged in the same volume and all samples were non-reducing. The star (â) indicates the predicted size of Fab at about 40 kDa (Luo <em>et al.</em>, 2019b).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917195254_2084.jpg" width="198"/><br/><strong>Figure 4. Purification of anti-VEGF Fab by Capto L affinity chromatography. </strong>A. Eluted chromatography spectrum. Red curve: UV absorbance at 280 nm; Blue curve: Conductivity. B. SDS-PAGE. Lane Medi: Pre-loading sample (medium); FL: flow through; E(NR): non-reducing elutant; E(R): reduced elutant. The star (â) indicates the non-reducing Fab; the diamond (â) indicates the educed fragments of LC(â²) and HC(â²) (Luo <em>et al.</em>, 2019b).<br/></li></ol><p class="pptt" id="biaoti29252">### Recipes</p><ol class="npwtwok"><li>PLM medium<br/>7.5 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub><br/>0.4 mM MgSO<sub>4</sub> <br/>11 mM glucose<br/>111 mM PIPES <br/>Adjust to pH 7.0<br/><em>Note: PLM medium should be steam-sterilized or filter-sterilized immediately after preparation.</em> </li><li>Buffer A<br/>50 mM citric acid<br/>50 mM sodium citrate<br/>Adjust to pH 6.5<br/><em>Note: The buffer should be filtered through a 0.22 Î¼m filter and used within one week. </em> </li><li>Buffer B<br/>50 mM citric acid<br/>50 mM sodium citrate<br/>Adjust to pH 2.3<br/><em>Note: The buffer should be filtered through a 0.22 Î¼m filter and used within one week. </em> </li><li>TAE buffer<br/>40 mM Tris-acetate, pH 8.5<br/>1 mM EDTA <br/></li></ol><p class="pptt" id="biaoti29253">### Acknowledgments</p><p class="ppzz">This work was in collaboration with Jecho Labs, and a US patent was applied jointly (CAGLIERO, Cedric; BURNETTE, Andrew; XIE, Yueqing; JIANG, Hua; ZHU, Jianwei; LU, Huili; LUO, Manyu. Method for Preparing Recombinant Protein from Bacterium and Composition Containing the Same. United States Patent Application, Date: Dec. 31, 2018, Application No. 16/237265). The work was also supported in part by the National Natural Science Foundation of China (No. 81773621 to Zhu J.), the Science and Technology Commission of Shanghai Municipality (No. 17431904500 &amp; 17ZR1413700 to Lu H.). We thank all the authors of our original research article (Luo <em>et al.</em>, 2019a <em>Appl Microbiol Biotechnol</em>).</p><p class="pptt" id="biaoti29254">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest with the contents of this article.</p><p class="pptt" id="biaoti29255">### References</p><ol class="npwtwok"><li>Augustine, R. C., York, S. L., Rytz, T. C. and Vierstra, R. D. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27208252" target="_blank">Defining the SUMO system in maize: SUMOylation is Up-Regulated during endosperm development and rapidly induced by stress.</a><em> Plant Physiol</em> 171(3): 2191-2210.</li><li>Danyliv, A., Glanville, J., McCool, R., Ferreira, A., Skelly, A. and Jacob, R. P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28188433" target="_blank">The clinical effectiveness of ranibizumab treat and extend regimen in nAMD: systematic review and network Meta-Analysis.</a> <em>Adv Ther</em> 34(3): 611-619.</li><li>Ellis, M., Patel, P., Edon, M., Ramage, W., Dickinson, R. and Humphreys, D. P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27790865" target="_blank">Development of a high yielding <em>E. coli</em> periplasmic expression system for the production of humanized Fab' fragments.</a> <em>Biotechnol Prog</em> 33(1): 212-220.</li><li>Gupta, S. K. and Shukla, P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27387055" target="_blank">Microbial platform technology for recombinant antibody fragment production: A review.</a> <em>Crit Rev Microbiol</em> 43(1): 31-42.</li><li>Jalalirad, R (2013). <a href="https://www.researchgate.net/publication/269585521_Production_of_antibody_fragment_Fab_throughout_Escherichia_coli_fed-batch_fermentation_process_Changes_in_titre_location_and_form_of_product" target="_blank">Production of antibody fragment (Fab) throughout <em>Escherichia coli</em> fed-batch fermentation process: Changes in titre, location and form of product.</a> <em>Electron J Biotechn</em> 16(3). DOI: 10.2225/vol16-issue3-fulltext-15.</li><li>Lobstein, J., Emrich, C. A., Jeans, C., Faulkner, M., Riggs, P. and Berkmen, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22569138" target="_blank">SHuffle, a novel <em>Escherichia coli</em> protein expression strain capable of correctly folding disulfide bonded proteins in its cytoplasm.</a> <em>Microb Cell Fact</em> 11: 56.</li><li>Luo, M., Zhao, M., Cagliero, C., Jiang, H., Xie, Y., Zhu, J., Yang, H., Zhang, M., Zheng, Y., Yuan, Y., Du, Z. and Lu, H. (2019a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30887174" target="_blank">A general platform for efficient extracellular expression and purification of Fab from <em>Escherichia coli</em>.</a> <em>Appl Microbiol Biotechnol</em> 103(8): 3341-3353.</li><li>Luo, M., Gao, Y., Zhu, J. and Lu, H. (2019b). Extracellular expression and purification of ranibizumab in <em>E. coli</em> and its structural characterization. Chinese Journal of Pharmaceuticals . Submitted.</li><li>Nelson, A. L. and Reichert, J. M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19352366" target="_blank">Development trends for therapeutic antibody fragments.</a> <em>Nat Biotechnol</em> 27(4): 331-337.</li><li>Panda, A. K. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12930093" target="_blank">Bioprocessing of therapeutic proteins from the inclusion bodies of <em>Escherichia coli</em>.</a> <em>Adv Biochem Eng Biotechnol</em> 85: 43-93.</li><li>Rezaie, F., Davami, F., Mansouri, K., Agha Amiri, S., Fazel, R., Mahdian, R., Davoudi, N., Enayati, S., Azizi, M. and Khalaj, V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28482126" target="_blank">Cytosolic expression of functional Fab fragments in <em>Escherichia coli</em> using a novel combination of dual SUMO expression cassette and EnBase((R)) cultivation mode.</a> <em>J Appl Microbiol</em> 123(1): 134-144.</li><li>Ulmer, N., Ristanovic, D. and Morbidelli, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31169346" target="_blank">Process for Continuous Fab Production by Digestion of IgG.</a> <em>Biotechnol J</em>: e1800677.</li><li>Walsh, G. and Jefferis, R. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17033665" target="_blank">Post-translational modifications in the context of therapeutic proteins.</a> <em>Nat Biotechnol</em> 24(10): 1241-1252.</li><li>Wang, Y. G., Ding, H. Z., Du, P., Gan, R. B. and Ye, Q. (2005). <a href="https://www.sciencedirect.com/science/article/pii/S1359511305001261" target="_blank">Production of phoA promoter-controlled human epidermal growth factor in fed-batch cultures of <em>Escherichia coli</em> YK537 (pAET-8).</a> <em>Process Biochem</em> 40(9): 3068-3074.</li><li>Zhou, Y., Lu, Z., Wang, X., Selvaraj, J. N. and Zhang, G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29270732" target="_blank">Genetic engineering modification and fermentation optimization for extracellular production of recombinant proteins using <em>Escherichia coli</em>.</a> <em>Appl Microbiol Biotechnol</em> 102(4): 1545-1556.<br/></li></ol>