<body>
            <h1><em>In situ</em> Hybridization of Plant-parasitic Nematode <em>Globodera pallida</em> Juveniles to Detect Gene Expression</h1>
            <p><strong>Authors:</strong> Joanna  Kud, Nejra  Solo, Allan  Caplan, Joseph C. Kuhl, Louise-Marie  Dandurand, Fangming  Xiao</p>
            <h2>Abstract</h2>
            <p>In this study, we describe a standard whole mount <em>in situ</em> hybridization method which is used to determine the spatial-temporal expression pattern of genes from <em>Globodera</em> spp. Unlike more invasive radioactive labeling approaches, this technique is based on a safe, highly specific enzyme-linked immunoassay where a Digoxigenin (DIG)-tagged anti-sense probe hybridized to a target transcript is detected by anti-DIG antibodies conjugated with alkaline phosphatase enzyme (AP) (anti-DIG-AP). The hybrid molecules are visualized through an AP-catalyzed color reaction using as the substrate 5-bromo-4-chloro-3-indolyl phosphate (BCIP) and nitro blue tetrazolium chloride (NBT). This method can be applied to both free-living pre-parasitic juveniles and early endoparasitic stages of cyst nematodes.</p>
            <p><strong>Keywords:</strong> <em>In situ</em> hybridization, DIG-labeling, Cyst nematodes, <em>Globodera pallida</em>, Esophageal glands (dorsal and subventral), Effector, RHA1B</p>
            <h2>Background</h2><p>Potato cyst nematodes (PCNs), <em>Globodera pallida</em> and <em>G. rostochiensis</em>, are a global threat to a potato production, causing in excess of 80% yield loss in infested fields (Brodie, 1989). PCNs are highly specialized sedentary endoparasites undergoing a complex life cycle with six developmental stages: egg, four juvenile stages, and male or female adult. The second stage juveniles (J2) hatched from eggs in soil are free-living mobile pre-parasitic nematodes that recognize and invade plant roots. After reaching cells in the root vascular cylinder, J2s become sedentary, feed and molt into third and fourth parasitic stage (J3 and J4, respectively). To complete their life cycle, nematodes undergo sexual differentiation into males and females. Following sexual reproduction, eggs are laid within the female body which eventually becomes a cyst. One way to understand this complex life cycle, is to use the <em>in situ</em> hybridization technique to monitor <em>in vivo</em> spatial gene expression at different nematode developmental stages to gain insight into the function of those genes. The <em>in situ</em> hybridization is a highly specific and sensitive assay based on the immunodetection of DIG-tagged probes hybridized to a target transcript. First, the polymerase chain reaction (PCR) is used to generate labeled probes through randomly incorporating DIG-coupled dUTP during enzymatic amplification of the cDNA template. Second, biological samples are fixed, mechanically cut, enzymatically permeabilized, and incubated with DIG-labeled probes for hybridization. Finally, hybridized probes are selectively detected by anti-DIG antibodies conjugated with alkaline phosphatase enzyme (anti-DIG-AP). AP catalyzes a color reaction using the substrate BCIP and NBT to visualize targeted hybrid molecules. Described here, the <em>in situ</em> hybridization protocol has been adapted from de Boer <em>et al.</em> (1998) and optimized for <em>Globodera</em> spp. Although this method has been routinely used to confirm esophageal gland expression of nematode effector genes, it can be applied to detect the expression pattern of any other nematode gene (Jones <em>et al.</em>, 2002).</p>
        </body>

<p class="pptt" id="biaoti29340">## Materials and Reagents</p><ol class="npwtwok"><li>Laboratory gloves</li><li>RNaseZap<sup>Â®</sup> RNase Decontamination Wipes (Thermo Fisher Scientific, catalog number: AM9786)</li><li>15 ml glass tubes</li><li>0.5 ml and 1.5 ml nonstick microcentrifuge tubes (<em>e.g.</em>, VWR, catalog numbers: 20170-315 and 20170-650)</li><li>0.2 ml PCR tubes (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: E0030124707) </li><li>Microscope slides and coverslips (<em>e.g.</em>, Thermo Fisher Scientific, catalog numbers: 12-550-A3 and 10-016-24)</li><li>Razor blades (<em>e.g.</em>, Thermo Fisher Scientific, catalog number:12-640)</li><li>Filtered DNase free tips (<em>e.g.</em>, Mettler-Toledo, Rainin, catalog numbers: 17007957, 17002927, 17014361)</li><li>Nuclease-free water (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: AM9937)</li><li><em>Taq</em> PCR polymerase (New England BioLabs, catalog number: M0273S)</li><li>Deoxynucleotides (dNTPs) (Thermo Fisher Scientific, catalog number: 10297117)</li><li>Forward and reverse primers for <em>in situ</em> hybridization probes (<em>e.g.</em>, Sigma-Aldrich)</li><li>PCR Purification Kit (<em>e.g.</em>, ZYMO RESEARCH, catalog number: D4033)</li><li>Digoxigenin (DIG) DNA Labeling Mix (Roche Diagnostics, catalog number: 11277065910)</li><li>Proteinase K 20 mg/ml (Roche Diagnostics, catalog number: 03115887001)</li><li>Boehringer blocking reagent (Roche Diagnostics, catalog number: 11096176001)</li><li>Anti-Digoxigenin-AP-Fab fragments (Roche Diagnostics, catalog number:11093274910)</li><li>5-bromo-4-chloro-3-indolyl-phosphate, 4-toluidine salt BCIP (Roche Diagnostics, catalog number: 11383221001)</li><li>4-Nitro blue tetrazolium chloride NBT (Roche Diagnostics, catalog number: 11383213001)</li><li>DNA sodium salt from salmon testes (Sigma-Aldrich, catalog number: D1626)</li><li>tRNA from bakerâs yeast (Sigma-Aldrich, catalog number: R8759, type X-SA)</li><li>Denhardtâs solution 50x (Sigma-Aldrich, catalog number: D2532)</li><li>Dry ice</li><li>Sucrose (<em>e.g.</em>, Sigma-Aldrich, catalog number: S0389)</li><li>Agarose (<em>e.g.</em>, VWR, catalog number: 0710)</li><li>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) (<em>e.g.</em>, Sigma-Aldrich, catalog number: P9791)</li><li>Sodium phosphate dibasic (Na<sub>2</sub>HPO<sub>4</sub>) (<em>e.g.</em>, Sigma-Aldrich, catalog number: S7907)</li><li>Sodium chloride (NaCl) (Sigma-Aldrich, catalog number: S7653)</li><li>Sodium citrate (Na<sub>3</sub>C<sub>6</sub>H<sub>5</sub>O<sub>7</sub>) (Sigma-Aldrich, catalog number: 1613859)</li><li>Magnesium chloride hexahydrate (MgCl<sub>2</sub>Â·6H<sub>2</sub>O) (Sigma-Aldrich, catalog number: 63138)</li><li>0.5 M EDTA pH 8 (Sigma-Aldrich, catalog number: 324504)</li><li>Tris-base (Sigma-Aldrich, catalog number: T1503)</li><li>Tween-20 (Sigma-Aldrich, catalog number: P1379)</li><li>Acetone (Sigma-Aldrich, catalog number: 650501)</li><li>Methanol (Sigma-Aldrich, catalog number: 34860)</li><li>37% formaldehyde solution (Sigma-Aldrich, catalog number: F15587)</li><li>Maleic acid (Sigma-Aldrich, catalog number: M0375)</li><li>SDS (Sigma-Aldrich, catalog number: 1614363)</li><li>Formamide deionized (Sigma-Aldrich, catalog number: F9037)</li><li>Hydrochloric acid (HCl) (Sigma-Aldrich, catalog number: H1758)</li><li>Transparent nail polish (<em>e.g.</em>, Pure Ice)</li><li><em>Globodera pallida</em>âpre-parasitic J2s (10,000) or parasitic stages (&gt; 100)</li><li>M9 buffer (see Recipes)</li><li>Fixation Buffer (see Recipes)</li><li>20x SSC (see Recipes)</li><li>Hybridization Buffer (see Recipes)</li><li>Washing Buffer A (see Recipes)</li><li>Washing Buffer B (see Recipes)</li><li>Maleic Acid Buffer (see Recipes)</li><li>Blocking Buffer (see Recipes)</li><li>Detection Buffer (see Recipes)<br/></li></ol><p class="ppzz" style="margin-left:0px;"><em>Note: Recipes for </em><em>in situ</em><em> hybridization buffers can be found at the end of this protocol.</em> </p><p class="pptt" id="biaoti29341">### Equipment</p><ol class="npwtwok"><li>Pipets (<em>e.g.</em>, Rainin, models: P2, P20, P200, P1000)</li><li>Sieve of 2.8 mm/500 Î¼m/250 Î¼m/90 Î¼m/25 Î¼m/20 Î¼m (<em>e.g.</em>, Humboldt, catalog numbers: No. 7, 35, 60, 170, 500, 635)</li><li>PCR thermocycler (<em>e.g.</em>, Bio-Rad, model: T100)</li><li>DNA electrophoresis system (<em>e.g.</em>, Bio-Rad) and imaging system (<em>e.g.</em>, Azure Biosystems, model: c300)</li><li>NanoDrop (<em>e.g.</em>, Thermo Fisher Scientific, model: 2000 Spectrophotometer)</li><li>Centrifuge for 15 ml glass tubes (<em>e.g.</em>, Eppendorf, model: 5804 R)</li><li>Laboratory blender (<em>e.g.</em>, Waring, model: WF2211214)</li><li>Microcentrifuge for 0.5 ml and 1.5 ml tubes (<em>e.g.</em>, Eppendorf, model: 5424)</li><li>Tube rotator (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: 88881001)</li><li>4 Â°C Fridge</li><li>-20 Â°C Freezer</li><li>Hybridization oven (<em>e.g.</em>, VWR, model: 230402V)</li><li>Mini block heater (<em>e.g.</em>, VWR, model: 10153-318)</li><li>Light Inverted Microscope (Leica, model: DMi8)<br/></li></ol><p class="pptt" id="biaoti29342">### Software</p><ol class="npwtwok"><li>Microscope imagining softwareâLAS V4.12 (Leica, <a href="https://www.leica-microsystems.com/products/microscope-software/p/leica-application-suite/" target="_blank">https://www.leica-microsystems.com/products/microscope-software/p/leica-application-suite/</a>)<br/></li></ol><p class="pptt" id="biaoti29343">### Procedure</p><ol class="npwone"><li>Generate sense and anti-sense cDNA probes<ol class="npwtwo"><li>Identify a unique 200-250 bp region in the cDNA sequence of the gene of interest that will be detected by <em>in situ</em> probe and design standard PCR primers for that DNA fragment.</li><li>Obtain nematode RNA and synthesize cDNA as described (Casavant <em>et al.</em>, 2017).</li><li>Follow the standard PCR protocol for the DNA polymerase (<em>e.g.</em>, <em>Taq</em> polymerase) to amplify desired gene fragment from nematode cDNA. Verify a single PCR product by running a small amount on a 1% DNA agarose gel and purify it using commercially available PCR purification kit.</li><li>Set up two independent asymmetric PCR reactions with DIG DNA Labeling Mix (1 mM dATP, 1 mM dCTP, 1 mM dGTP, 0.65 mM dTTP, 0.35 mM DIG-dUTP) to separately synthesize sense (negative control) and anti-sense cDNA probes as follows: <br/>X Âµl     PCR template from A3 (20 ng)<br/>4 Âµl     5x <em>Taq</em> Buffer<br/>2 Âµl     DIG DNA Labeling Mix<br/>4 Âµl     10 ÂµM Forward primer (sense probe) OR Reverse primer (anti-sense probe)<br/>1 Âµl    <em>Taq</em> polymerase<br/>X Âµl     ddH<sub>2</sub>O<br/>Total volume 20 Âµl<br/><br/>Run the following program in a PCR cycler as:<br/><ol class="npwthree"><li>95 Â°C â 30 s</li><li>95 Â°C â 15 s</li><li>60 Â°C â 30 s (temperature might be adjusted depending on the primers) </li><li>68 Â°C â 90 s (repeat b-c 35 times)</li><li>68 Â°C â 5 min </li></ol><br/></li><li>Check your DIG-labeling PCR step, by running a small amount of labeled probes on the 1% DNA agarose gel site-by-site with the original PCR template. The labeled probes should show an increase in molecular mass compare with unlabeled DNA due to the incorporated DIG (Figure 1).<br/><br/><img alt="" class="layerphoto" height="143" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917210633_2547.jpg" width="100"/><br/><strong>Figure 1. DNA 1% agarose gel to check DIG-labeling PCR step. </strong>Lane 1âPCR template, Lane 2âDIG-labelled sense negative probe synthesized with Forward primer, and Lane 3âDIG-labeled anti-sense probe synthesized with Reverse primer. Both DIG-labeled probes have increased molecular mass due to incorporation of DIG.<br/><br/></li><li>Purify DIG-labeled probes with commercially available PCR purification kit and store them at  -20 Â°C until needed.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>FixationâDay 1<ol class="npwtwo"><li>Collect nematode juveniles<ol class="npwthree"><li>For pre-parasitic juveniles<ol class="npwfour"><li>Hatched J2s nematodes as described previously (Casavant <em>et al.</em>, 2017).</li><li>Spin down freshly hatched J2s nematodes by centrifugation in 15 ml glass tubes (150 <em>x g</em> for 5 min, 4 Â°C) and remove supernatant.</li></ol></li><li>For parasitic juveniles<ol class="npwfour"><li>Infect potato plants with <em>G. pallida</em> cysts (~ 10 cysts/6-inch pot) as described previously (Dandurand and Knudsen, 2016). </li><li>At the desired infection stage, gently pull the infected root system from the soil and rinse it with water to remove larger soil particles.<br/></li><li>Cut roots into 2 cm long sections, place in a blender, cover with water, and blend 5 times for 2 s at the low setting.</li><li>To separate nematode juveniles from root debris, pour the blended root-nematode suspension through a series of sieves (2.8 mm, 500 Î¼m, 250 Î¼m, 90 Î¼m, 20 Î¼m). Nematodes will be collected on the last 20 Î¼m sieve.<br/><em>Note: Re-blend larger roots and repeat this step if needed.</em> </li><li>To remove small soil and root residue, purify the nematodes by sucrose gradient centrifugation.<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Prepare 50% w/v sucrose solution and cool at 4 Â°C.<br/><em>Note: Sucrose solution can be made a day earlier and stored overnight at 4 Â°C.</em> </div></div><p class="ppzz" style="margin-top:-15px;"><div class="media media-x cnautokh cnautokh2"><div class="float-left">2)</div><div class="media-body">In 15 ml glass tubes add 5 ml of cold 50% sucrose, then gently overlay it with 5 ml of the collected nematode solution.</div></div></p><p class="ppzz" style="margin-top:-15px;"><div class="media media-x cnautokh cnautokh3"><div class="float-left">3)</div><div class="media-body">Spin down tubes for 10 min at 1,250 <em>x g</em> at 4 Â°C (acceleration and break set to âzeroâ). Nematodes will accumulate at the water-sucrose solution border, whereas, small soil particles and root residues will pellet on the bottom of the tube (Figure 2).</div></div></p><p class="ppzz" style="margin-top:-15px;"><div class="media media-x cnautokh cnautokh4"><div class="float-left">4)</div><div class="media-body">Collect the nematode layer with P1000 pipette and wash nematodes 5 times with water on the 25 Î¼m sieve to remove sucrose. <br/></div></div></p><br/><img alt="" class="layerphoto" height="95" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917210714_1529.jpg" width="100"/><br/><strong>Figure 2. Purification of parasitic nematodes through sucrose gradient centrifugation</strong><br/></li></ol></li></ol></li><br/><li>Transfer nematodes (10,000 pre-parasitic J2s or &gt; 100 parasitic worms) to a 1.5 ml nonstick microcentrifuge tube.</li><li>Wash twice by resuspending nematodes in 1 ml of water and then centrifuge them at 6,000 <em>x g</em> for 2 min.</li><li>Remove as much water as possible and resuspend nematodes in 1 ml of Fixation Buffer.</li><li>Fixate nematodes by lying tubes flat for 18 h at 4 Â°C, then move samples to room temperature and incubate for additional 4 h.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cutting of nematodes, permeabilization and hybridizationâDay 2<ol class="npwtwo"><li>Wipe everything down with RNase wipes to protect RNA.</li><li>Centrifuge nematodes (6,000 <em>x g</em> for 2 min), remove supernatant and resuspend nematode pellet with 200 Î¼l of 10-fold diluted (in M9 buffer) Fixation Buffer.</li><li>Spread ~70 Î¼l of the nematode suspension onto a glass slide and cut them using a flat razor blade until ~80% of nematodes are cut (inspect under microscope) (Figure 3).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917210738_6369.jpg" width="311"/><br/><strong>Figure 3. Cutting fixed nematodes.</strong> A. Cutting station. B. Nematodes before cutting. C. Nematodes after cutting. Scale bars = 100 Î¼m.<br/><br/></li><li>Collect chopped nematodes into a 1.5 ml nonstick microcentrifuge tube by gently rinsing a glass slide with 10-fold diluted Fixation Buffer, and repeat Step C3 until all nematodes are cut.</li><li>Spin down nematodes (6,000 <em>x g</em> for 2 min) and remove supernatant.</li><li>Wash nematodes twice by resuspending in 500 Î¼l of M9 buffer and then centrifuging at 6,000 <em>x g</em> for 2 min.</li><li>Add 500 Î¼l of proteinase K solution (0.5 mg/ml in M9 buffer) and carry out permeabilization on a rotator for 30 min at room temperature. For nematodes of parasitic stages, permeabilization time should be increased to 45-60 min.</li><li>Spin down the sample and wash the nematode pellet twice with 500 Î¼l of M9 buffer (6,000 <em>x g</em> for 2 min). After the second wash, remove as much M9 buffer as possible.</li><li>Freeze the nematode pellet on dry ice for 15 min.</li><li>Re-suspend nematodes in 1 ml of cold methanol (stored at -20 Â°C) and incubate nematode pellet on dry ice for an additional 30 s.</li><li>Spin down the nematodes at 21,000 <em>x g</em> for 30 s (room temperature) and remove the methanol.</li><li>Re-suspend the nematodes in 1 ml of cold acetone (stored at -20 Â°C) and incubate the nematode pellet on dry ice for 1 min.</li><li>Spin down the nematodes at 21,000 <em>x g</em> for 30 s and discard all but 100 Î¼l of the acetone.</li><li>Slowly, drop-by-drop, rehydrate the sample by adding 100 Î¼l of H<sub>2</sub>O.</li><li>Pre-heat the Hybridization Buffer at 50 Â°C.</li><li>Spin down the sample and wash the nematode pellet with 500 Î¼l of the Hybridization Buffer (6,000 <em>x g</em> for 2 min at room temperature). Discard the supernatant.</li><li>Add 300 Î¼l of the pre-heated Hybridization Buffer and transfer 150 Î¼l aliquots to two 0.5 ml nonstick microcentrifuge tubes (for sense and anti-sense sample).</li><li>Pre-hybridize samples for 15 min at 50 Â°C in the hybridization oven.</li><li>MEANWHILE: denature probes for 10 min at 100 Â°C and place them on ice immediately to prevent re-annealing.</li><li>Add separately sense and anti-sense probes to two nematode samples.</li><li>Hybridize by rotating in a hybridization oven at 50 Â°C overnight.<br/><em>Note: Hybridization temperature might be adjusted depending on length and probe specificity.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Washing and stainingâDay 3<ol class="npwtwo"><li>Spin down the nematodes (6,000 <em>x g</em> for 2 min at room temperature) and remove the supernatant.</li><li>Wash the samples 3 times each with 100 Î¼l of Wash Buffer 1 by rotating 15 min in hybridization oven at 50 Â°C followed by centrifugation at 6,000 <em>x g</em> for 2 min.</li><li>Wash the samples 3 times each with 100 Î¼l of Wash Buffer 2 by rotating 20 min in a hybridization oven at 50 Â°C followed by centrifugation at 6,000 <em>x g</em> for 2 min (room temperature).</li><li>Wash samples by resuspending them in 100 Î¼l of Maleic Acid Buffer and then centrifuging at 6,000 <em>x g</em> for 2 min.</li><li>Add 100 Î¼l of Blocking Buffer and incubate for 30 min by rotating.</li><li>Spin down samples (6,000 <em>x g</em> for 2 min), resuspend nematode pellet in 100 Î¼l of Blocking Buffer containing the anti-DIG-AP (1 Î¼l anti-DIG-AP/1,000 Î¼l Blocking Buffer), and incubate while rotating for 2 h at room temperate.</li><li>Wash the samples 3 times with 100 Î¼l of Maleic Acid Buffer each by rotating for 15 min followed by centrifugation at 6,000 <em>x g</em> for 2 min.</li><li>Wash the samples briefly with 100 Î¼l Detection Buffer (6,000 <em>x g</em> for 2 min). </li><li>Add 150 Î¼l of freshly made AP Substrate Solution (3.5 Î¼l BCIP and 3.4 Î¼l NBT in 1,000 Î¼l Detection Buffer) and incubate overnight without agitation at 4 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>ImagingâDay 4<ol class="npwtwo"><li>Wash the nematodes twice with 100 Î¼l 0.01% Tween-20 for 1 min (6,000 <em>x g</em> for 2 min) and remove the supernatant.</li><li>Resuspend the nematode pellet with 50 Î¼l 0.01% Tween-20 and apply 10 Î¼l/slide.</li><li>Cover with coverslip and seal edges with nail polish.</li><li>Examine stained nematode sections under microscope.</li></ol></li></ol><p class="pptt" id="biaoti29344">### Data analysis</p><p class="ppzz">The stained nematode sections were analyzed under Leica Microscope and images were captured using LAS V4.12 imaging software. The representative results are described in the example of dorsal glad specific expression of <em>Globodera pallida</em> effector RHA1B (Kud <em>et al.</em>, 2019) (Figure 4). The positive <em>in situ</em> hybridization signal in a parasitic-J2 is visualized as dark violet/brown coloring of dorsal glands (Figure 4A). In contrast, no signal is detected for the same gene in a pre-parasitic J2 (Figure 4B), showing that parasitic/developmental stage-dependent gene expression directly impacts <em>in situ</em> hybridization results. Positive results are typically accompanied by a negative control, sense probe-incubated sample with no signal detected (Figures 4C and 4D), to confirm signal specificity. The staining efficiency may vary from experiment to experiment; therefore, this assay should be repeated several times with similar results. At least 20 stained nematodes should be checked for each assay to ensure consistency.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190917/20190917210820_3039.jpg" width="243"/><br/><strong>Figure 4. Representative results showing <em>in situ</em> hybridization results for a dorsal gland expression of G. pallida RHA1B effector.</strong> A. Sample with RHA1B anti-sense probe in a para-J2âstrong signal confirming glad specific expression at this parasitic stage. B. Sample with RHA1B anti-sense probe in a pre-J2sâno signal detected due to very low expression or RHA1B at this stage. C. Negative control with RHA1B sense probe in a parasitic J2 (para-J2). D. Negative control with RHA1B sense probe in a pre-parasitic J2. Scale bars = 50 Î¼m. sâstylet and dgâdorsal glands. This figure has been modified from Kud <em>et al.</em> (2019).</p><br/><ol class="npwtwo"><li>Because even a small amount of RNase can compromise RNA integrity, it is recommended to wear gloves, use only RNase-free plastic- and glassware, and frequently wipe lab surfaces with commercially available decontamination reagents, such as RNaseZap wipes, to ensure RNase and DNase-free environment.</li><li>Most of hybridization buffers can be purchased as premade stock solutions. </li><li>Make sure to always use fresh deionized formamide for Fixation Buffer.</li><li>The probe length is a tradeoff between sensitivity and penetrability, where a shorter probe gets into nematode tissue, but a longer probe results in better detection/hybridization to the tested transcript.</li><li>If a strong background signal is observed, a few troubleshooting steps can be applied. 1) Designing a probe that spans two exons will reduce its affinity to DNA. 2) In our hands, hybridization temperature of 50 Â°C is a good starting point. Depending on length and probe specificity, the perfect hybridization temperature might be optimized experimentally (range of 45 Â°C to 55 Â°C). 3) Although generally the wash temperature should match the hybridization temperature, slightly increasing that temperature and/or SSC salt concentration can reduce background signal. </li><li>To additionally validate <em>in situ</em> hybridization results, the second <em>in situ</em> probe aligning with a different region of transcript of interest can be used.</li></ol><p class="pptt" id="biaoti29345">### Recipes</p><ol class="npwtwok"><li>M9 buffer (pH 7)<br/>22 mM KH<sub>2</sub>PO<sub>4</sub><br/>42 mM Na<sub>2</sub>HPO<sub>4</sub><br/>86 mM NaCl<br/>Sterilize by autoclaving</li><li>Fixation Buffer<br/>2% formaldehyde<br/>M9 buffer</li><li>20x SSC (pH 7)<br/>3 M NaCl<br/>0.3 M sodium citrate<br/>Sterilize by autoclaving</li><li>Hybridization Buffer<br/>50% Deionized formamide<br/>4x SSC<br/>1% Blocking reagent<br/>2% SDS<br/>1x Denhardtâs solution<br/>1 mM EDTA, pH 8<br/>200 Î¼g/ml DNA sodium salt from salmon testes<br/>3.125 U/ml Yeast tRNA</li><li>Washing Buffer A<br/>4x SSC</li><li>Washing Buffer B <br/>0.1x SSC<br/>0.1% SDS</li><li>Maleic Acid Buffer (pH 7.5)<br/>0.1 M maleic acid<br/>0.15 M NaCl <br/>Sterilize by autoclaving</li><li>Blocking Buffer<br/>1% Boehringer blocking reagent<br/>Maleic Acid Buffer<br/>Sterilize by autoclaving</li><li>Detection Buffer (pH 9.5)<br/>0.1 M Tris-HCl<br/>0.1 M NaCl<br/>50 mM MgCl<sub>2</sub>Â·6H<sub>2</sub>O<br/></li></ol><p class="pptt" id="biaoti29346">### Acknowledgments</p><p class="ppzz">This work was supported by the Agriculture and Food Research Initiative competitive grant (2015-69004-23634, 2017-67014-26197; 2017-67014-26591) of the USDA National Institute of Food and Agriculture, USDA-NIFA Farm Bill, Northwest Potato Consortium, and ISDA Specialty Crop.<br/>  Described here <em>in situ</em> hybridization protocol has been adapted from de Boer <em>et al.</em> (1998). We would like to thank Dr. John Jonesâ and Dr. Vivian Blockâs lab at The James Hutton Institute for providing useful tips to optimize this method.</p><p class="pptt" id="biaoti29347">### Competing interests</p><p class="ppzz">The authors declare no financial and non-financial competing interests.</p><p class="pptt" id="biaoti29348">### References</p><ol class="npwtwok"><li>Brodie, B. B. (1989). <a href="https://www.annualreviews.org/doi/abs/10.1146/annurev.py.27.090189.002303?journalCode=phyto" target="_blank">Control of the golden nematode in the United States.</a> <em>Annu. Rev. Phytopathol</em> 27: 443-461.</li><li>Casavant, N. C., Kuhl, J. C., Xiao, F., Caplan, A. B. and Dandurand, L. M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28512373" target="_blank">Assessment of <em>Globodera pallida</em> RNA extracted from solanum roots.</a> <em>J Nematol</em> 49(1): 12-20.</li><li>Dandurand, L.-M. and Knudsen, G. R. (2016). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/aab.12295" target="_blank">Effect of the trap crop <em>Solanum sisymbriifolium</em> and two biocontrol fungi on reproduction of the potato cyst nematode, <em>Globodera pallida</em>.</a> <em>Annals of Applied Biology</em> 169(2): 180-189.</li><li>de Boer, J. M., Yan, Y., Smant, G., Davis, E. L. and Baum, T. J. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19274224" target="_blank"><em>In-situ</em> hybridization to messenger RNA in <em>Heterodera glycines</em>.</a> <em>J Nematol</em> 30(3): 309-312.</li><li>Jones, J., Blok, V. and Smant, G. (2002). <a href="https://brill.com/view/journals/nemy/2/8/article-p887_7.xml?lang=en" target="_blank">SXP/RAL-2 proteins of the potato cyst nematode <em>Globodera rostochiensis</em>: secreted proteins of the hypodermis and amphids.</a> <em>Nematology</em> 2(8): 887-893. </li><li>Kud, J., Wang, W., Gross, R., Fan, Y., Huang, L., Yuan, Y., Gray, A., Duarte, A., Kuhl, J. C., Caplan, A., Goverse, A., Liu, Y., Dandurand, L. M. and Xiao, F. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30978251" target="_blank">The potato cyst nematode effector RHA1B is a ubiquitin ligase and uses two distinct mechanisms to suppress plant immune signaling.</a> <em>PLoS Pathog</em> 15(4): e1007720.<br/></li></ol>