<body>
            <h1>Isolation and Transcriptomic Profiling of Single Myofibers from Mice</h1>
            <p><strong>Authors:</strong> Francesco Chemello, Enrico Alessio, Lisa Buson, Beniamina Pacchioni, Caterina Millino, Gerolamo Lanfranchi, Stefano   Cagnin</p>
            <h2>Abstract</h2>
            <p> Skeletal muscle is composed of different cells and myofiber types, with distinct metabolic and structural features. Generally, transcriptomic analysis of skeletal muscle is performed using whole muscle, resulting in average information as all cells composing the organ contribute to the expression value detected for each gene with the loss of information about the distinctive features of each specific myofiber type. Since myofibers are the smallest complete contractile system of skeletal muscle influencing its contraction velocity and metabolism, it would be beneficial to have fiber-specific information about gene expression. Here, we describe a protocol for the isolation and the transcriptomic analysis of single individual myofibers. The protocol was set up using single myofibers isolated from <em>soleus</em> and <em>Extensor Digitorum Longus</em> (EDL) muscles, but it can be applied to all skeletal muscles. Briefly, muscles are enzymatically dissociated and individually collected. Long RNAs (&gt; 200 nt) and short RNAs (&lt; 200 nt) are separately purified from each myofiber and used to produce libraries for microarray or sequencing analysis. Through this approach, myofiber-specific transcriptional profiles can be produced, free from transcripts from other non-contractile cell types, in order to identify mRNA-miRNA-lncRNA regulatory networks specific for each myofiber type.</p>
            <p><strong>Keywords:</strong> Single myofiber, Transcriptomic analysis, Skeletal muscle, Non-coding RNAs, miRNAs, Regulatory network</p>
            <h2>Background</h2><p>Skeletal muscle is a complex organ and heterogenous tissue. It is composed by different cell types such as those from blood, vessels, nerves, connective tissue, and myofibers, that are the smallest complete contractile system of skeletal muscle influencing both its mechanical performance and metabolism. In addition, myofibers are classified into different types. In mouse, type I myofibers display slow-twitch contraction and oxidative metabolism, type IIA and IIX myofibers fast-twitch contraction and oxidative metabolism, and type IIB myofibers fast-twitch contraction and glycolytic metabolism (Schiaffino and Reggiani, 2011). The preferential metabolism of each muscle is defined by the different proportion of myofiber types. <em>Soleus</em> muscle is mainly composed by type I, type IIA, and type 2X oxidative myofibers, and preferentially consumes lipids, whereas EDL muscle by type IIB glycolytic myofibers, and primarily uses carbohydrates as energy substrate (Augusto <em>et al.</em>, 2004). To investigate differences in muscle metabolism and contraction at transcriptomic level, previous expression profiles were obtained using <em>soleus</em> and EDL whole muscles (Campbell <em>et al.</em>, 2001; Wu <em>et al.</em>, 2003). Results described in these works are influenced by the different contributions of diverse cells composing whole muscle (Figure 1A) and lose the transcriptional specificity of myofiber types (Figure 1B). We demonstrated the feasibility of scaling down the transcriptomic analysis of skeletal muscle at single isolated myofiber level for mRNA, long non-coding RNA (lncRNA) and microRNA (miRNA) populations (Figure 1C). This approach allowed the identification of expression profiles, free from non-myofiber cells (Chemello <em>et al.</em>, 2011), and allowed the generation of mRNA-miRNA-lncRNA regulatory networks specific for each myofiber type (Alessio <em>et al.</em>, 2019; Chemello <em>et al.</em>, 2019). Moreover, it is easier to characterize perturbations that exclusively affect myofibers (Chemello <em>et al.</em>, 2015; Mammucari <em>et al.</em>, 2015), and to improve the description and comprehension of muscle atrophy (Alessio <em>et al.</em>, 2019). In fact, muscle atrophy induces significant systemic metabolic modifications (Celegato <em>et al.</em>, 2006) also in a myofiber-specific manner (Wang and Pessin, 2013). </p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927020952_3872.jpg" width="313"/><br/><strong>Figure 1. Complexity of skeletal muscle.</strong> A. Structure of the skeletal muscle with the different tissues and cells that make up its structure. Image modified from (Biga <em>et al.</em>, 2019). B. Succinate Dehydrogenase Staining (SDH) shows different myofiber types in a transversal muscle cross-section. Scale bar is for 100 Âµm. C. Images of single isolated myofibers from EDL and <em>soleus</em> muscles. Scale bar is for 250 Âµm. From each single myofibers is possible to profile the expression of long and short RNAs.</p><p class="pptt" id="biaoti29795">## Materials and Reagents</p><ol class="npwtwok"><li>200 Âµl tips</li><li>DNA LoBind tubes (Eppendorf, catalog number: 0030108051 or Serstedt, catalog number: 72.706.700)</li><li>Spin-X centrifuge tubes (Corning-Costar, catalog number: CLS8160-96EA)</li><li>RNase/DNase free Eppendorf microcentrifuge tube</li><li>0.2 ml PCR tube</li><li>Hybridization gasket slides, 8 microarrays/slide (Agilent, catalog number: G2534-60014)</li><li>Cell culture plates: 24-well and 6-well plates </li><li>Plastic Pasteur pipettes</li><li>Slide-staining dishes</li><li>Adult wild-type mice, weight: 33-35 g<br/><em>Note: We used CD1 mouse strain.</em> </li><li>High-glucose Dulbecco's modified Eagle medium (DMEM) (Millipore Sigma, catalog number: D5796)</li><li>2,3-Butanedione monoxime (BDM) (Sigma-Aldrich, catalog number: B0753)</li><li>Agencourt AMPure XP beads (Beckman Coulter, catalog number: A63881)</li><li>Collagenase from <em>Clostridium histolyticum</em>, type I (Millipore Sigma, catalog number: C0130)</li><li>Fetal bovine serum (FBS) (Millipore Sigma, catalog number: F2442)</li><li>Phosphate-buffered saline (PBS) (Thermo Fisher Scientific, catalog number: 20012019)</li><li>TRIzol Reagent (Thermo Fisher Scientific, catalog number: 15596026)</li><li>Nuclease-free water (Thermo Fisher Scientific, catalog number: 10977035)</li><li>Nuclease-free water (Ambion, catalog number: AM9937)</li><li>Chloroform (PanReac AppliChem, catalog number: 121252)</li><li>70% v/v ethanol (dilute ethanol in nuclease-free water)</li><li>75% v/v ethanol (dilute ethanol in nuclease-free water)</li><li>80% v/v ethanol (dilute ethanol in nuclease-free water)</li><li>Absolute ethanol</li><li>miRNeasy kit (Qiagen, catalog number: 217004)</li><li>RNeasy micro kit (Qiagen, catalog number: 74004)</li><li>Complete Whole Transcriptome Amplification Kit (Millipore Sigma, catalog number: WTA2)</li><li>GenElute PCR Clean-Up Kit (Millipore Sigma, catalog number: NA1020)</li><li>SureTag DNA Labeling Kit (Agilent, catalog number: 5190-3400)</li><li>Gene Expression Hybridization Kit (Agilent, catalog number: 5190-0404)</li><li>Gene Expression Wash Buffer Kit (Agilent, catalog number: 5188-5327)</li><li>SurePrint G3 Mouse GE 8x60K Microarray Kit (Agilent, catalog number: G4852A)</li><li>Poly(A) Tailing Kit (Thermo Fisher Scientific, catalog number: AM1350)</li><li>Sodium Acetate 3 M (pH 5.5) (Thermo Fisher Scientific, catalog number: AM9740)</li><li>SuperScript II Reverse Transcriptase (Thermo Fisher Scientific, catalog number: 18064014)</li><li>Oligo-dT-Ion P1 adapter primer (5â²-CCTCTCTATGGGCAGTCGGTGATCCTCAGC[dT]<sub>20</sub>VN-3â²)</li><li>SMART primer (5â²-CACACACAATTAACCCTCACTAAAggg-3â²)</li><li>Ion Xpress Plus gDNA Library (Thermo Fisher Scientific, catalog number: 4471269)</li><li>PGM HI-Q OT2 kit (Thermo Fisher Scientific, catalog number: A27739)</li><li>Platinum Taq DNA Polymerase High Fidelity (Thermo Fisher Scientific, catalog number: 11304011)</li><li>E-Gel SizeSelect Gels (Thermo Fisher Scientific, catalog number: G661012)</li><li>DNA ladder 50 bp (Thermo Fisher Scientific, catalog number: 10416014)</li><li>High Sensitivity DNA kit (Agilent, catalog number: 5067-4626)</li><li>Ion OneTouch Template Kit (Thermo Fisher Scientific, catalog number: A29900)</li><li>1x Low TE buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti29796">### Equipment</p><ol class="npwtwok"><li>Micropipette (P10, P20)</li><li>-80 Â°C freezer</li><li>-20 Â°C freezer</li><li>Forceps and scissors for microdissection</li><li>37 Â°C incubator</li><li>Stereo-microscope</li><li>Refrigerated microcentrifuge</li><li>SpeedVac concentrator</li><li>Thermocycler</li><li>Nanodrop spectrophotometer</li><li>Heat block</li><li>Centrifuge</li><li>Agilent SureHyb chamber (Agilent, catalog number: G2534A)</li><li>Hybridization oven (Agilent, model: G2545A)</li><li>Hybridization oven rotator (Agilent, model: G2530-60029)</li><li>Agilent scanner system (Agilent, model: G2505C)</li><li>DynaMag (Thermo Fisher Scientific, catalog number: 12321D)</li><li>Ion Torrent, or Ion S5 System (Thermo Fisher Scientific)</li><li>E-Gel<sup>TM</sup> Electrophoresis Device (Thermo Fisher Scientific)</li><li>Agilent 2100 Bioanalyzer (Agilent)</li><li>Ion OneTouch Instrument (Thermo Fisher Scientific)</li><li>Ion PGM sequencing instrument (Thermo Fisher Scientific)<br/></li></ol><p class="pptt" id="biaoti29797">### Software</p><ol class="npwtwok"><li>Bioanalyzer software (Agilent, <a href="https://www.agilent.com/en/product/automated-electrophoresis/bioanalyzer-systems/bioanalyzer-instrument/2100-bioanalyzer-laptop-228251 Company or Developer/Provider/Supplier" target="_blank">https://www.agilent.com/en/product/automated-electrophoresis/bioanalyzer-systems/bioanalyzer-instrument/2100-bioanalyzer-laptop-228251 Company or Developer/Provider/Supplier</a>, web address)</li><li>Feature Extraction Software (Agilent, <a href="https://www.agilent.com/en/product/mirna-microarray-platform/mirna-microarray-software/feature-extraction-software-228496" target="_blank">https://www.agilent.com/en/product/mirna-microarray-platform/mirna-microarray-software/feature-extraction-software-228496</a>)</li><li>R statistical software (<a href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a>)</li><li>MultiExperiment Viewer (<a href="https://sourceforge.net/projects/mev-tm4/" target="_blank">https://sourceforge.net/projects/mev-tm4/</a>)</li><li>miRDeep software (<a href="https://sourceforge.net/projects/mirdeepstar/" target="_blank">https://sourceforge.net/projects/mirdeepstar/</a>)<br/></li></ol><p class="pptt" id="biaoti29798">### Procedure</p><p class="ppzz">See Figure 2 for a summary of principal steps for the analysis of transcriptional profiling of single myofibers.<br/><br/><img alt="" class="layerphoto" height="111" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927021239_3759.jpg" width="100"/><br/><strong>Figure 2. Workflow of transcriptomic analysis of single myofibers.</strong> The scheme summarizes principal steps for transcriptomic analysis of single myofibers. On the side of each box, it is reported the corresponding part of the protocol.<br/><br/><strong>Part I: Purification of long RNAs (mRNAs and lncRNAs) and short RNAs (miRNAs) from single isolated myofibers</strong> </p><br/><ol class="npwone"><li>Myofiber isolation<br/><em>Note: Here we will describe myofiber isolation from </em><em>soleus</em><em> and </em><em>Extensor Digitorum Longus</em><em> (EDL) hindlimb muscles of 3 months old mice, but the same procedure can be used for other types of skeletal muscles optimizing the timing of collagenase dissociation. In principle, younger mice have smaller muscles, we suggest decreasing the time of collagenase incubation and the repetitions of triturating process with muscles of young mice, to avoid damage of myofibers. On the contrary, muscles of older mice have more connective tissue, it could be necessary increase the time of collagenase incubation and the repetitions of triturating process to obtain a sufficient number of single isolated myofibers for the following analyses. In any case, it is important to limit this step to avoid the alteration of gene expression and consequently cell pathways. Sacrifice mice according to the local animal welfare act.</em> <ol class="npwtwo"><li>Collect <em>soleus</em> or EDL muscles from hindlimb mouse, <u>taking care to handle them only by their tendons</u> to minimize mechanical damage of myofibers (Figure 3). We used 3 months old CD1 mice. Store them in 1 ml of cold DMEM 3 mM BDM. Muscles collection should take no more than half an hour. Below, step by step procedure to collect <em>soleus</em> and EDL muscles.</li><li>Peel off the leg skin to expose the hindlimb muscles.</li><li>Gently remove the fascia of the hindlimb muscles.</li><li>Position the mouse supine and expose the tendons of tibialis anterior (TA) and EDL.</li><li>Cut the distal TA tendon and use it to remove the TA muscle by cutting it at its proximal attachment.</li><li>Cut the distal EDL tendon and remove it through cutting the proximal EDL tendon.</li><li>Place the muscle in a tube containing 1 ml of DMEM 3 mM BDM.</li><li>Repeat Steps A5-A7 to collect the EDL of the other leg.</li><li>Place the mouse in prone position.</li><li>Cut the tendon of gastrocnemius muscle and use it to peel off the muscle until the exposure of the <em>soleus</em> tendon.</li><li>Cut the <em>soleus</em> tendon and use it to remove the <em>soleus</em> cutting the other end.</li><li>Place the muscle in a tube containing 1 ml of DMEM 3 mM BDM.</li><li>Repeat to collect the <em>soleus</em> of the other leg.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927021358_3236.jpg" width="309"/><br/><strong>Figure 3. Muscles collection.</strong> A. EDL muscle isolated from hindlimb mouse muscles. B. Collection of one muscle in 1 ml of DMEM 3 mM BDM. C. <em>Soleus</em> muscle isolated from hindlimb mouse muscles.<br/><br/></li><li>Digest the muscles for 45 min at 37 Â°C in 1 ml high-glucose DMEM 3 mM BDM containing 10 mg type I collagenase in one well of a 24-well plate.</li><li>Sequentially rinse the muscles for 2 min in 3 ml of DMEM and 3 mM BDM, 3 ml DMEM and 3 mM BDM supplemented with 10% FBS and 3 ml of DMEM and 3 mM BDM in different wells of a 6-well plate (Figures 4A, 4B and 4C). Handle muscle with care and control under the microscope to avoid losing myofibers during these washing steps.</li><li>Transfer the muscles into a well of 6-well plate with 3 ml of DMEM with 10% FBS at about 25 Â°C to inactivate collagenase (Figure 4D).</li><li>Liberate myofibers by gentle physical trituration (pipetting up and down) using a wide-mouth plastic Pasteur pipette (about 4 mm diameter), rinsed in FBS to prevent sticking.</li><li>Repeat the triturating process several times, eventually transferring non-dissociated muscle in new wells, until about 100 intact fibers are obtained. After each physical trituration, transfer the muscles into a new well, to get rid of collagen wisps and hyper-contracted fibers (Figures 4E, 4F, and 4G).</li><li>Using a stereo-microscope and a micropipette (200 Âµl tips), pick one by one intact and well-isolated fibers and wash first in a well with DMEM 3 mM BDM 10% FBS and then in a well with PBS (Figure 4H). </li><li>If the researcher chose to avoid the myofiber characterization via MyHC protein expression (possible on a piece of myofiber [Chemello <em>et al.</em>, 2011]), after washing, transfer the single myofiber into an RNase/DNase free Eppendorf microcentrifuge tube. The expression of specific genes [Myosin Heavy Chain 4 (Myh4), Myosin Light Chain 3 (Myl3), Myosin Heavy Chain 7 (Myh7), Tropomyosin 3 (Tpm3), actinin alpha 3 (Actn3), myosin heavy chain 2 (Myh2)] will be useful for the myofiber characterization (Chemello <em>et al.</em>, 2019). See Figure 5 for the flowchart and timing specifications for each step.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927021422_2689.jpg" width="229"/><br/><strong>Figure 4. Steps for single myofiber dissociation and isolation.</strong> A. After 45 min of incubation with collagenase the muscle is transferred from a 24-well plate (on the left of the picture) into a well of a 6-well plate containing 3 ml of DMEM 3 mM BDM. After 2 min of incubation, the whole muscle is transferred into the second well and then in the third well, waiting 2 min for the incubation in each well. B. Representation of the third transfer of the muscle. C. Scheme of the plate to use for the muscle rinsing after collagenase digestion (L: muscle form the left hindlimb; R: muscle from the right hindlimb; 1, 2, 3: first, second, and third wash). D. Scheme of the plate for collagenase inactivation and muscle dissociation (1, 2, 3: steps for releasing single myofibers from muscle). E. Using a 6-well plate filled as described in D dissociate muscle during collagenase inactivation. F. Transfer the muscle not dissociated in the second well to leave free myofibers in the first one. G. Transfer the muscle not dissociated in the third well to leave free myofibers in the second well and liberate eventually other myofibers. Recover free myofibers from all the wells and transfer them into a new 24-well plate filled as described in H. H. Scheme of the plate used to wash single myofibers (I: first wash; II: second wash; 1, 2, 3: several wells to wash different myofibers. It is not necessary to use a well per myofiber). After the second wash, single myofibers can be collected independently in different Eppendorf tubes or can be cut into two pieces; one used for their characterization using MyHC proteins and the other for RNA extraction and transcriptional analyses.<br/><br/><img alt="" class="layerphoto" height="109" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927021450_2499.jpg" width="100"/><br/><strong>Figure 5. Workflow for the isolation of single myofibers from hindlimb mouse muscles.</strong> Each box describes work steps and timing and is associated with the protocol points.<br/><em>Note: During the isolation of single myofibers it is mandatory to follow exactly the same procedure and time points when processing the same muscle types derived from different mice to avoid introducing errors due to tissue processing. Moreover, it is important to avoid recovering contracted fibers to avoid having a different gene expression due to their peculiar condition. There are no stopping points during the myofiber collection procedure. Only when myofibers are lysed in TRIzol for RNA analysis, in Laemmli buffer (Tris pH 6.8 62.5 mM, glycerol 10%, SDS 2%, Î²-mercaptoethanol 5%) for MyHC analysis (Chemello </em><em>et al.</em><em>, 2011), flash freeze in nitrogen and then the tube can be stored at -80 Â°C. In Figure 1C, it is represented how isolated myofibers appear under microscope.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Long RNA and short RNA separation using RNeasy kits (Qiagen)<br/>To proceed with RNA purification, it is better to avoid working with too many myofibers. We suggest working with 4-6 myofiber samples.<ol class="npwtwo"><li>Lyse each myofiber in 250 Âµl of TRIzol Reagent (Thermo Fisher Scientific) pipetting up and down the solution.</li><li>Add 50 Âµl of chloroform, mix well by inverting the tube, and incubate on ice for 15 min.</li><li>Centrifuge the sample for 15 min at 12,000 <em>x g</em> at 4 Â°C.</li><li>Collect and transfer the aqueous upper phase (about 130 Âµl) into a new tube.</li><li>Add 1 volume of 70% ethanol and mix.</li><li>Transfer the solution onto a miRNeasy column (Qiagen).</li><li>Spin for 15 s at 12,000 <em>x g</em>. </li><li>Long RNAs are retained by the column, whereas short RNAs are collected in the eluate. <u>Retain the eluate</u> and process as described in Section D in Part I if you are interested in miRNA components.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Long RNA purification<ol class="npwtwo"><li>Add 700 Âµl of RWT Buffer (Qiagen) to the column containing long RNAs (Section B in Part I).</li><li>Spin for 15 s at â¥ 8,000 <em>x g</em> and discard the eluate.</li><li>Add 500 Âµl of RPE Buffer (Qiagen) to the column.</li><li>Spin for 15 s at â¥ 8,000 <em>x g</em> and discard the eluate.</li><li>Add again 500 Âµl of RPE Buffer (Qiagen) to the column.</li><li>Spin for 15 s at â¥ 8,000 <em>x g</em> and discard the eluate.</li><li>Place the column in a new collection tube.</li><li>Add 50 Âµl of nuclease-free water to the center of the column and incubate at room temperature for 2 min.</li><li>Spin for 1 min at â¥ 8,000 <em>x g</em> to eluate long RNAs. Eventually, store the eluate at -80 Â°C in the eluted volume.</li><li>Use SpeedVac concentrator to reduce the volume to 14 Âµl. It is better to control the volume during its reduction to avoid drying the sample. </li><li>Store at -80 Â°C until use. To avoid RNA binding to the plastic tube it is strongly suggested to use LoBind tubes.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>miRNA purification using miRNeasy kit (Qiagen)<ol class="npwtwo"><li>Add 0.65 volumes of absolute ethanol (about 165 Âµl) to the eluate produced after loading the aqueous phase of the TRIzol extraction into the miRNeasy columns (Section B in Part I) and mix well.</li><li>Transfer the solution onto an RNeasy micro column (Qiagen).</li><li>Spin for 15 s at â¥ 8,000 <em>x g</em> and discard the eluate.</li><li>Add 500 Âµl of RPE Buffer (Qiagen) to the column.</li><li>Spin for 15 s at â¥ 8,000 <em>x g</em> and discard the eluate.</li><li>Add 500 Âµl of 80% ethanol to the column.</li><li>Spin for 2 min at â¥ 8,000 <em>x g</em> and discard the eluate.</li><li>Spin again for 5 min at â¥ 8,000 <em>x g</em> with the cap opened.</li><li>Transfer the column to a new collection tube.</li><li>Add 14 Âµl of nuclease-free water to the center of the column and incubate at room temperature for 2 min. </li><li>Spin for 1 min at â¥ 8,000 <em>x g</em> to eluate miRNAs. Eventually, store the eluate at -80 Â°C in the eluted volume.</li><li>Use SpeedVac concentrator to reduce the volume to 6.5 Âµl. If you dry the solution, resuspend miRNAs in nuclease-free water and preferably use LoBind tubes for the SpeedVac step.</li><li>Store at -80 Â°C until use.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol><strong>Part II: Transcriptomic profiling of long RNAs from single isolated myofibers by microarray</strong><br/><br/><ol class="npwone"><li>Preparation of the cDNA library of long RNAs for microarray hybridization using Complete Whole Transcriptome Amplification Kit (Sigma)<ol class="npwtwo"><li>Add 2.5 Âµl of Library Synthesis Solution to the 14 Âµl of long RNA solution prepared in Section C in Part I.</li><li>Incubate at 70 Â°C for 5 min and cool down the temperature by putting the Eppendorf on ice (~4 Â°C) for 5 min.</li><li>Add the mix indicated below to the solution previously prepared (Steps A1 and A2 in Part II) (Table 1).<br/><br/><strong>Table 1. Library synthesis enzyme mix</strong><br/><img alt="" class="layerphoto" height="135" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927031936_9209.jpg" width="350"/><br/><br/></li><li>Incubate in the thermocycler using the parameters shown in (Table 2).<br/><br/><strong>Table 2. Parameters for library synthesis</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032022_3989.jpg" width="350"/><br/><br/></li><li>Remove tubes from the thermocycler and spin briefly.</li><li>Add the mix indicated into the solution (Table 3).<br/><br/><strong>Table 3. Library amplification solution </strong><br/><img alt="" class="layerphoto" height="163" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032120_3049.jpg" width="400"/><br/><br/></li><li>Divide previously prepared mix into five Eppendorf each containing 75 Âµl of solution (for the last aliquot the reaction volume is &lt; 75 Âµl).</li><li>Incubate in the thermocycler using the following parameters (Table 4).<br/><br/><strong>Table 4. Parameters for Library amplification</strong><br/><img alt="" class="layerphoto" height="168" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032153_0001.jpg" width="400"/><br/><br/></li><li>Store the WTA DNA at -20 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>WTA DNA purification using GenElute PCR Clean-Up Kit (Sigma)<ol class="npwtwo"><li>Purify separately each mix (each Eppendorf contains 75 Âµl of solution prepared as described in Section A of Part II).</li><li>Add 375 Âµl of Binding Solution and transfer the solution into the binding column.</li><li>Spin for 1 min at 12,000 <em>x g</em> and discard the eluate.</li><li>Add 500 Âµl of Wash Solution to the column.</li><li>Spin for 1 min at 12,000 <em>x g</em> and discard the eluate.</li><li>Spin for 2 min at maximum speed without any additional wash.</li><li>Transfer the column to a new collection tube.</li><li>Add 50 Âµl of nuclease-free water to the center of the column and incubate at room temperature for 2 min</li><li>Spin for 1 min at 12,000 <em>x g</em>.</li><li>Quantify DNA using Nanodrop spectrophotometer.</li><li>Store the purified WTA DNA at -20 Â°C until use.</li><li>WTA DNA can be labeled for microarray hybridization (Section C in Part II) or can be used for NGS library preparation (Section F in Part II).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Labeling WTA DNA for microarray hybridization using SureTag DNA Labeling Kit (Agilent)<ol class="npwtwo"><li>Prepare 2 Âµg of WTA DNA in 13 Âµl of nuclease-free water.</li><li>Add 2.5 Âµl of Random Primers.</li><li>Incubate at 98 Â°C for 5 min. </li><li>Move onto ice and incubate for 5 min.</li><li>Spin for 1 min at 12,000 <em>x g</em>.</li><li>Add the mix indicated to the solution (Table 5).<br/><br/><strong>Table 5. DNA labeling solution</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032238_9257.jpg" width="350"/><br/><br/></li><li>Incubate in the thermocycler using the parameters shown in Table 6.<br/><br/><strong>Table 6. Parameters for DNA labeling</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032316_3369.jpg" width="350"/><br/><br/></li><li>Spin for 1 min at 10,000 <em>x g</em>.</li><li>Add 430 Âµl 1x TE (pH 8.0).</li><li>Transfer the solution onto a purification column provided with the SureTag DNA Labeling Kit (Agilent).</li><li>Spin for 10 min at 14,000 <em>x g</em> and discard the eluate.</li><li>Add 480 Âµl of 1x TE (pH 8.0) to the column.</li><li>Spin for 10 min at 14,000 <em>x g</em> and discard the eluate.</li><li>Invert the column into a new tube.</li><li>Spin for 1 min at 1,000 <em>x g</em> to collect the purified sample. The volume per sample will be approximatively 20-32 Âµl.</li><li>Measure DNA quantity and labeling using Nanodrop spectrophotometer. DNA yield will be about 4 Âµg and the specific activity 30 pmol of Cy3 per Âµg of cDNA.</li><li>Store the labeled DNA at -20 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Hybridization on Agilent microarrays<br/><em>Note: If you are interested in measuring the expression of lncRNAs make sure that the microarray platform you are using contains probes for this class of transcripts.</em> <ol class="npwtwo"><li>Prepare 800 ng of labeled DNA in 20 Âµl of nuclease-free water.</li><li>Add 5 Âµl of 10x Blocking Agent.</li><li>Incubate at 95 Â°C for 2 min.</li><li>Add 25 Âµl of 2x GEx Hybridization Buffer HI-RPM.</li><li>Dispense 40 Âµl of the solution on hybridization gasket slide.</li><li>Put the 8x60K microarray platform slide facing down the hybridization gasket slide.</li><li>Load the microarray sandwich into the Agilent SureHyb chamber.</li><li>Load the assembled chamber into the oven rotator rack.</li><li>Hybridize at 65 Â°C for 17 h at 10 rpm.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Scan of the microarray slides<ol class="npwtwo"><li>Disassemble the hybridization chamber.</li><li>Disassemble the microarray sandwich in slide-staining dish filled with Wash Buffer 1.</li><li>Wash the microarray slide in Wash Buffer 1 for 1 min.</li><li>Wash the microarray slide in 37 Â°C pre-warmed Wash Buffer 2 for 1 min.</li><li>Slowly remove the microarray slide, to minimize droplets on the slide.<br/></li><li>Put the microarray slide in a slide holder.</li><li>Scan the slide using G2505C scanner at 3 Âµm resolution to obtain single myofiber mRNA and lncRNA expression profile.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>NGS of RNA extracted from a single myofiber<ol class="npwtwo"><li>The bulk of SeqPlex amplification product (WTA DNA) ranges in size from 200-400 base pairs and therefore it is not needed DNA fragmentation for its sequencing. Here we will describe the sequencing protocol to use Ion Torrent or Proton sequencers but changing adapters, amplified RNA (WTA DNA) can also be sequenced with Illumina sequencer.</li><li>The terminus of each amplicon possesses a 5â²-phosphate and 2-base 3â²-over-hang.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sequencing library preparation<ol class="npwtwo"><li>In a 0.2 ml PCR tube, prepare Adapters mix as indicated in the table below (non-barcoded), and mix by pipetting up and down. Reagents are in the Ion Xpress Plus gDNA Library preparation (Thermo Fisher Scientific) (Table 7).<br/><br/><strong>Table 7. Adapters mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032440_9723.jpg" width="350"/><br/><br/></li><li>Prepare the following mix (Table 8).<br/><br/><strong>Table 8. DNA repair and adapters ligation mix</strong><br/><img alt="" class="layerphoto" height="249" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032517_7844.jpg" width="400"/><br/><br/></li><li>Incubate in thermal cycler following the program below (Table 9).<br/><br/><strong>Table 9. Parameters for DNA repair and adapters ligation </strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032556_8305.jpg" width="350"/><br/><br/></li><li>Purify adapter-ligated DNA using Agencourt AMPure XP beads.<br/>According to the desired reads length it will be used different amounts of Agencourt AMPure XP beads: add 100 Âµl of Agencourt AMPure XP beads to 100 Âµl of reaction from Step G3 in Part II for 400-600 bp reads length, 120 Âµl of Agencourt AMPure XP beads to 100 Âµl of reaction from Step G3 in Part II for 200-300 bp reads length.</li><li>Mix well by pipetting up and down and avoid bubble formation. Then place the tube in a magnetic rack (<em>e.g.</em>, DynaMag) for 3 min and wait until the solution becomes clear.</li><li>Discard the supernatant and avoid disturbing the pellet.</li><li>Add 500 Âµl of freshly prepared 70% ethanol leaving the tube on the magnet.</li><li>Incubate for 30 s rotating the tube around the vertical axis 3 times and maintaining the tube on the magnet to agitate the beads avoiding that they go in the solution. After the solution is clear, remove the supernatant avoiding disturbing the pellet.</li><li>Wash the beads with 70% ethanol for a second time as indicated in the previous step.</li><li>Remove residual of ethanol centrifuging the tube, placing it on the magnet and pipetting out the supernatant using a tip for a P10 micropipette.</li><li>Keep the cap-opened tube on the magnet for 5 min to air dry the beads (avoid over dry).</li><li>Remove the tube from the magnet, add 20 Âµl of Low TE on the beads and mix by pipetting up and down the solution. Vortex the sample for 10 s.</li><li>Spin the solution for a few seconds and place the tube on the magnet for at least 1 min.</li><li>Transfer the supernatant into a new LoBind Eppendorf without disturbing the pellet.</li><li>Store at -30 Â°C to -10 Â°C.</li><li>Before performing emulsion-PCR, amplification library can or cannot be selected in a gel according to DNA size. If the range selected with Agencourt AMPure XP beads it is fine for NGS researcher to go throughout emPCR. For method about size selection see Section F in Part III. For emulsion-PCR, see Section H in Part III.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol><strong>Part III: Transcriptomic profiling of miRNAs from single isolated myofibers by NGS</strong><br/><br/><ol class="npwone"><li>Purification of miRNAs from a single myofiber (see Section D in Part I)<br/><br/></li><li>Polyadenylation of miRNAs using Poly(A) Tailing Kit (Thermo Fisher Scientific) <ol class="npwtwo"><li>Add the mix indicated in Table 10 to the solution of miRNAs purified as in Section D in Part I (6.5 Âµl of miRNA solution).<br/><br/><strong>Table 10. Poly(A) tailing mix</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032636_4408.jpg" width="350"/><br/><br/></li><li>Incubate at 37 Â°C for 1 h.</li><li>Add 1 Âµl NaOAc 3 M (pH 5.5).</li><li>Add 44 Âµl of absolute ethanol and mix well.</li><li>Precipitate at -20 Â°C overnight.</li><li>Spin for 20 min at maximum speed and carefully remove the supernatant without disturbing the pellet.</li><li>Wash with 50 Âµl of 75% ethanol. </li><li>Spin for 10 min at maximum speed and carefully remove the supernatant without disturbing the pellet.</li><li>Resuspend in 3.2 Âµl of nuclease-free water.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Synthesis of the first stranded cDNA from polyadenylated miRNAs (from Section B in Part III)<ol class="npwtwo"><li>Prepare the mix A in a 0.2 ml tube as indicated in Table 11.<br/><br/><strong>Table 11. Retrotranscription mix A</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032705_6128.jpg" width="400"/><br/><br/></li><li>Prepare the mix B in a 0.2 ml tube as indicated below (Table 12).<br/><br/><strong>Table 12. Retrotranscription mix B</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032742_0698.jpg" width="550"/><br/><br/></li><li>Incubate the mix B at 65 Â°C for 5 min in a thermoblock or thermocycler without heated lid. </li><li>Spin the mix B and incubate it on ice for 5 min.</li><li>Mix the mix A and B. You should have a final volume of 10 Âµl. Mix well the solution by pipetting up and down.</li><li>Incubate at 42 Â°C for 60 min using a thermocycler without heated lid.</li><li>Inactivate the enzyme at 70 Â°C for 15 min.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Synthesis of the second stranded cDNA<br/>Second strand cDNA is synthesized by a PCR step avoiding reaching the plateau phase. In fact, it is important to maintain the original ratios of miRNAs. This step allows the production of double stranded DNA that, after its purification, can be sequenced because presenting on the 5â² and 3â² end sequencing primers for the Ion Torrent platform. It is important to use a High-Fidelity Taq DNA polymerase to avoid the introduction of errors during PCR amplification. For a scheme of the SMART protocol see Figure 6.<br/><ol class="npwtwo"><li>Prepare the following mix to perform two PCR reactions per sample (each of 50 Âµl) (Table 13).<br/><br/><strong>Table 13. PCR amplification of poly(A) tagged miRNAs</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032828_8976.jpg" width="550"/><br/><br/></li><li>Incubate in the thermocycler according to the following cycle (Table 14).<br/><br/><strong>Table 14. Parameters for PCR amplification of poly(A) tagged miRNAs</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927032904_7910.jpg" width="350"/><br/><br/><img alt="" class="layerphoto" height="128" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927033002_8456.jpg" width="100"/><br/><strong>Figure 6. Scheme for miRNA amplification.</strong> miRNAs extracted from a single myofiber are polyadenylated using the PAP enzyme (1, and Section B in Part III). After that, using the SMART technique (Wellenreuther <em>et al.</em>, 2004) miRNAs are retrotranscribed. The primer used for the retrotranscription is an oligo-d(T) that includes the Ion P1 adapter sequence at the 5â² end (2). When the Superscript II reaches the 5â² end of miRNA it adds a stretch of no template C to the cDNA that serves for the binding of the SMART primer (3). In the SMART primer, the three terminal g are RNA nucleotides. After the binding of the SMART primer to the cDNA it is used from the Superscript II for the synthesis of the complementary sequence (4, and Section C in Part III). Using this method, two anchors at the 3â² and 5â² end of the miRNAs are produced. The two anchors can be used for the miRNA amplification by PCR reaction. During PCR amplification (Section D in Part III) a specific primer it is used to include the A adapter in the 5â² end of miRNAs (5) that will be useful for miRNA sequencing in a strand specific way. After 10-12 cycles of PCR amplification (6, 7, 8), sufficient quantity of double stranded DNA is obtained with P1 and A adaptors on the miRNA 3â² and 5â² end, respectively to allow emulsion-PCR and following sequencing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>DNA purification using GenElute PCR Clean-Up Kit (Sigma)<br/><ol class="npwtwo"><li>Pool the two PCRs (50 + 50 Âµl). If you have more than one sample remember to maintain them separated.</li><li>Add 500 Âµl of Binding Solution to the pooled PCRs and transfer the solution into the binding column.</li><li>Spin for 1 min at 12,000 <em>x g</em> and discard the eluate.</li><li>Add 500 Âµl of Wash Solution to the column.</li><li>Spin for 1 min at 12,000 <em>x g</em> and discard the eluate.</li><li>Spin for 2 min at maximum speed without any additional wash.</li><li>Transfer the column to a new collection tube.</li><li>Add 50 Âµl of nuclease-free water to the center of the column and incubate at room temperature for 2 min.</li><li>Spin for 1 min at 12,000 <em>x g</em>.</li><li>Quantify DNA using Nanodrop spectrophotometer.</li><li>Store the purified PCR at -20 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Size selection for NGS using the E-Gel SizeSelect Gels (Thermo Fisher Scientific)<br/>For DNA sequencing it is important to select the size range of DNA to be sequenced. In the case of long RNAs dimensional purification it was described in the Step G4 in Part II. Retained reads to identify miRNAs are 18-35 nt long (He <em>et al.</em>, 2019) or purified small RNAs for miRNA libraries covers the same dimensional range ( Krishna <em>et al.</em>, 2013; Fu <em>et al.</em>, 2018) and adapters added during the library construction (Sections D and E in Part III) are ~107 nucleotides (Primer A is 57 nt and Oligo-dT-Ion P1 50 nt ) (Figure 6). We, therefore, selected the smear between 125 and 142 nt. See below for the full procedure.<ol class="npwtwo"><li>Use a 2% agarose gel.</li><li>Each well of the gel should be loaded with no more than 20-25 Âµl and 500 ng of DNA.</li><li>If DNA prepared from Section E in Part III needed to be diluted use Low TE to dilute it.</li><li>Dilute DNA ladder (50 bp) to 25 ng/Âµl with Low TE and load central well of the E-Gel with 10 Âµl of diluted ladder.</li><li>Load sample wells avoiding using adjacent wells for different samples (same samples split in different volumes can be loaded in adjacent wells).</li><li>Empty wells have to be filled with 25 Âµl of nuclease-free water. </li><li>Run E-Gel SizeSelect instrument (Thermo Fisher Scientific) selecting the correct program (SizeSelect 2%).</li><li>Visualize the run using the instrument and stop it when desired DNA dimension filled recovery well. </li><li>Recover the solution from the recovery well using a P20 pipette without touching the gel. Usually, 12-15 Âµl can be recovered.</li><li>Fill the recovery well with nuclease-free water replacing the withdrawn volume.</li><li>Start again the instrument and repeat Step F9 in Part III until all the desired dimensional range is recovered.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Dimensional analysis and library quantification (Agilent Bioanalyzer analysis)<br/>Run purified samples according to the High Sensitivity DNA Chip (Agilent). <br/><ol class="npwtwo"><li>Thaw High Sensitivity DNA kit at room temperature for at least 30 min.</li><li>Prepare gel dye mix by adding 15 Âµl of High Sensitivity DNA dye to High Sensitivity DNA gel matrix.</li><li>Mix by vortexing and spin down.</li><li>Put the solution onto Spin-X Centrifuge tubes 0.22 ÂµM pore diameter (Corning Costar) and centrifuge at 2,200 <em>x g</em> for 10 min at room temperature.</li><li>Load the High Sensitivity DNA chip as described in the table below (Table 15).<br/><br/><strong>Table 15. Agilent Bioanalyzer chip loading</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927033046_8058.jpg" width="600"/><br/><br/></li><li>From the Bioanalyzer analysis used to assess the library size distribution, determine the molar library concentration in pmol/L using the Bioanalyzer software. </li><li>Determine the dilution factor that results in a concentration of the library of ~100 pM.</li><li>This concentration is suitable for template preparation using Ion OneTouch Template Kit (Thermo Fisher Scientific). Use the following formula: Dilution factor = (Library concentration in pM)/100 pM.</li><li>Dilute the library in a new LoBind Eppendorf using a small amount of DNA library and necessary Low TE to yield approximately a concentration of 100 pM. Use this library dilution for template preparation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Emulsion-PCR (template preparation for DNA sequencing) using the Ion OneTouch system (Thermo <br/>I.Fisher Scientific)<br/><em>Note: Described material is included in the Ion PGM HiâQ OT2 Kit (Thermo Fisher Scientific).</em> <ol class="npwtwo"><li>Set up the Ion OneTouch system.</li><li>Prepare the amplification solution as follow (Table 16).<br/><br/><strong>Table 16. DNA library dilution </strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927033205_2661.jpg" width="350"/><br/><br/></li><li>Vortex for 5 s and spin for 2 s.</li><li>Prepare the Ion PGM Hi-Q ISPs and mix thoroughly by vortexing the tube for 1 min and spinning for 2 s.</li><li>Prepare the following mix in a 2-ml LoBind tube (Table 17).<br/><br/><strong>Table 17. OneTouch reaction mix</strong><br/><img alt="" class="layerphoto" height="199" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927033256_7562.jpg" width="400"/><br/><br/></li><li>Load the Ion OneTouch Reaction Filter with the solution prepared in Step H5 of Part III, install the filter onto the Ion OneTouch Instrument and start the run.</li><li>At the end of the run, enrich for the template-positive IPSs following the Ion OneTouch instructions and load the sequencing chip. </li></ol></li></ol><p class="pptt" id="biaoti29799">### Data analysis</p><ol class="npwone"><li>Microarray data analysis for the expression analysis of long RNAs<ol class="npwtwo"><li>Data pre-processing<br/>Probes features are extracted using Feature Extraction Software (Agilent Technologies). We published the following description of data treatment in (Alessio <em>et al.</em>, 2019; Chemello <em>et al.</em>, 2019). Data have to be intra-array and inter-array normalized. For intra-array normalizations, we suggest using the Feature Extraction Software while for the inter-array the quantile normalization (normalizeQuantiles, limma R package). For filtering out bad expression values we used Found and/or Well Above Background flags calculated from the Feature Extraction Software (Agilent Technologies). If the chosen filter was not positive, we assign to the particular expression value the value of NA. This allows maintaining in the expression matrix only transcripts that have a defined expression value in a sufficient number of biological replicates. For example, in (Chemello <em>et al.</em>, 2019) we considered only gene/probes with at least 8 available values out of a total of 10 biological replicates.</li><li>Statistical analyses of microarray data<br/>Microarray data can be analyzed using the R statistical suite or a more user-friendly platform such as the MultiExperiment Viewer, a tool of TM4 Microarray Software Suite (Saeed <em>et al.</em>, 2003). To identify the differentially expressed probes among myofibers, we performed one-way analysis of variance (ANOVA), but different alternatives can be explored, such as the identification of differentially expressed probes using the False Discovery Rate implemented in the Significance Analysis of Microarrays (SAM) algorithm (Tusher <em>et al.</em>, 2001). </li><li>Gene ontology<br/>Differentially expressed coding genes can be categorized according to gene ontology (GO) using different web tools (<em>e.g.</em>, the Functional Annotation Clustering method, implemented in the DAVID database [Huang da <em>et al.</em>, 2009] or according to pathway analyses [<em>e.g.</em>, (Sales <em>et al.</em>, 2013)] that implement different algorithms for the pathway analysis). Regarding lncRNAs, there are no enrichment analyses that can be performed. To infer their function, it can be performed a gene expression correlation with coding genes such as in (Guo <em>et al.</em>, 2017; Fang <em>et al.</em>, 2018; Cagnin <em>et al.</em>, 2019).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>NGS expression analysis<br/>In all sequencing experiments, raw reads have to be trimmed for the presence of adaptors and then filtered out for their length and quality. In Chemello <em>et al.</em> (2019) reads longer than 18 nucleotides were used for the following analyses: The expression of miRNAs was quantized using miRDeep software (Friedlander <em>et al.</em>, 2008). The processed reads were mapped to the known mouse miRNA precursors from miRBase database (Ver. 19) using the mapper module of miRDeep with default values. Basically, in this process equal reads are counted and collapsed. Reads that mapped more than 5 different locations are automatically excluded. Quantize module in miRDeep was used to normalize read counts of mature miRNAs. For a discussion about problems, software used in the analysis (normalization, read count, clustering) of single cell data see (Risso <em>et al.</em>, 2018a; Risso <em>et al.</em>, 2018b; Chen <em>et al.</em>, 2019; Cole <em>et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti29800">### Recipes</p><ol class="npwtwok"><li>1x Low TE buffer<br/>TE buffer is a buffer solution containing Tris, a common pH buffer, and EDTA, a molecule that chelates cations like Mg<sup>2+</sup>. It is used to solubilize DNA or RNA, to protect them from degradation<br/>To make 1x Low TE buffer, see Table 18.<br/><br/><strong>Table 18. Recipe for making 1x Low TE buffer</strong><br/><img alt="" class="layerphoto" height="107" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190927/20190927033348_6949.jpg" width="400"/><br/></li></ol><p class="pptt" id="biaoti29801">### Acknowledgments</p><p class="ppzz">Founding sources: CARIPARO Foundation (FIBRE-GEXP) and the University of Padova (CPDA139317) to S.C. and G.L.; the Cariplo Foundation (2016-1006) to S.C. and G.L.; the Italian Ministry of Health (GR-2011-02346845) to S.C.; the AIRC Foundation (Italy project IG2015-ID17773) to G.L.; Agenzia Nazionale di Valutazione del Sistema Universitario e della Ricerca (ANVUR) [FFABR- 2017 to S.C.]; Bando Grandi Attrezzature 2015 University of Padova (Italy) (Prot. 5086).<br/>  This protocol was modified from our previous works by Alessio <em>et al.</em>, 2019 and Chemello <em>et al.</em>, 2019. We thank all authors of the previous works.</p><p class="pptt" id="biaoti29802">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti29803">### Ethics</p><p class="ppzz">All aspects of animal care and experimentation were performed in accordance with the Guide for the Care and Use of Laboratory Animals published by the National Institutes of Health (NIH Publication No. 85-23, Revised 1996) and the Italian regulations (DL 116/92) concerning the maintenance and use of laboratory animals. Experimental procedures were approved by the he Italian Ministry of Health (authorization N. 305/2017-PR), and Ethical Committee of the University of Padova. All efforts were made to minimize animal suffering.</p><p class="pptt" id="biaoti29804">### References</p><ol class="npwtwok"><li>Alessio, E., Buson, L., Chemello, F., Peggion, C., Grespi, F., Martini, P., Massimino, M. L., Pacchioni, B., Millino, C., Romualdi, C., Bertoli, A., Scorrano, L., Lanfranchi, G. and Cagnin, S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30649422" target="_blank">Single cell analysis reveals the involvement of the long non-coding RNA Pvt1 in the modulation of muscle atrophy and mitochondrial network.</a> <em>Nucleic Acids Res</em> 47(4): 1653-1670.</li><li>Augusto, V., Padovani, C.R., and Campos, G.E.R. (2004). <a href="http://pesquisa.bvsalud.org/portal/resource/pt/lil-406360" target="_blank">Skeletal muscle fiber types in C57BL6J mice.</a> <em>Braz J Morphol Sci</em> 21(2): 89-94.</li><li>Biga, L. M., Dawson, S., Harwell, A., Hopkins, R., Kaufmann, J., LeMaster, M., Matern, P., Morrison-Graham, K., Quick, D. and Runyeon, J. (2019). <em><a href="http://library.open.oregonstate.edu/aandp/" target="_blank">Anatomy &amp; Physiology,</a></em> Open Oregon State, Oregon State University.gaiw</li><li>Cagnin, S., Chemello, F., Alessio, E. and Lanfranchi, G. (2019). <a href="https://www.elsevier.com/books/single-cell-omics/barh/978-0-12-814919-5" target="_blank">Single-cell transcriptomics and proteomics of skeletal muscle.</a> <em>Technology and Applications,</em> Elsevier.</li><li>Campbell, W. G., Gordon, S. E., Carlson, C. J., Pattison, J. S., Hamilton, M. T. and Booth, F. W. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11245591" target="_blank">Differential global gene expression in red and white skeletal muscle.</a> <em>Am J Physiol Cell Physiol</em> 280(4): C763-768.</li><li>Celegato, B., Capitanio, D., Pescatori, M., Romualdi, C., Pacchioni, B., Cagnin, S., Vigano, A., Colantoni, L., Begum, S., Ricci, E., Wait, R., Lanfranchi, G. and Gelfi, C. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17013991" target="_blank">Parallel protein and transcript profiles of FSHD patient muscles correlate to the D4Z4 arrangement and reveal a common impairment of slow to fast fibre differentiation and a general deregulation of MyoD-dependent genes.</a> <em>Proteomics</em> 6(19): 5303-5321.</li><li>Chemello, F., Bean, C., Cancellara, P., Laveder, P., Reggiani, C. and Lanfranchi, G. (2011). <a href="file://C:\Users\eowan\Dropbox\Files part time åå­¦åäº«æä»¶\20190701 Reformatting\1902646 Stefano Cagnin 760176\Chemello, F., Bean, C., Cancellara, P., Laveder, P., Reggiani, C. and Lanfranchi, G. (2011). Microgenomic analysis in skeletal muscle: expression signatures of individual fast and slow myofibers. PLoS One 6(2): e16807." target="_blank">Microgenomic analysis in skeletal muscle: expression signatures of individual fast and slow myofibers.</a> <em>PLoS One</em> 6(2): e16807.</li><li>Chemello, F., Grespi, F., Zulian, A., Cancellara, P., Hebert-Chatelain, E., Martini, P., Bean, C., Alessio, E., Buson, L., Bazzega, M., Armani, A., Sandri, M., Ferrazza, R., Laveder, P., Guella, G., Reggiani, C., Romualdi, C., Bernardi, P., Scorrano, L., Cagnin, S. and Lanfranchi, G. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30917329" target="_blank">Transcriptomic analysis of single isolated myofibers identifies miR-27a-3p and miR-142-3p as regulators of metabolism in skeletal muscle.</a> <em>Cell Rep</em> 26(13): 3784-3797 e3788.</li><li>Chemello, F., Mammucari, C., Gherardi, G., Rizzuto, R., Lanfranchi, G. and Cagnin, S. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26484227" target="_blank">Gene expression changes of single skeletal muscle fibers in response to modulation of the mitochondrial calcium uniporter (MCU).</a> <em>Genom Data</em> 5: 64-67.</li><li>Chen, G., Ning, B. and Shi, T. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31024627" target="_blank">Single-cell RNA-Seq technologies and related computational data analysis.</a> <em>Front Genet</em> 10: 317. </li><li>Cole, M. B., Risso, D., Wagner, A., DeTomaso, D., Ngai, J., Purdom, E., Dudoit, S. and Yosef, N. (2019). <a href="file://C:\Users\eowan\Dropbox\Files part time åå­¦åäº«æä»¶\20190701 Reformatting\1902646 Stefano Cagnin 760176\Cole, M. B., Risso, D., Wagner, A., DeTomaso, D., Ngai, J., Purdom, E., Dudoit, S. and Yosef, N. (2019). Performance Assessment and Selection of Normalization Procedures for Single-Cell RNA-Seq. Cell Syst 8(4): 315-328 e318." target="_blank">Performance assessment and selection of normalization procedures for Single-Cell RNA-Seq.</a> <em>Cell Syst</em> 8(4): 315-328 e318.</li><li>Fang, L., Wang, H. and Li, P. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29082457" target="_blank">Systematic analysis reveals a lncRNA-mRNA co-expression network associated with platinum resistance in high-grade serous ovarian cancer.</a> <em>Invest New Drugs</em> 36(2): 187-194.</li><li>Friedlander, M. R., Chen, W., Adamidi, C., Maaskola, J., Einspanier, R., Knespel, S. and Rajewsky, N. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18392026" target="_blank">Discovering microRNAs from deep sequencing data using miRDeep.</a> <em>Nat Biotechnol </em>26(4): 407-415.</li><li>Fu, Y., Wu, P. H., Beane, T., Zamore, P. D. and Weng, Z. (2018). <a href="file://C:\Users\eowan\Dropbox\Files part time åå­¦åäº«æä»¶\20190701 Reformatting\1902646 Stefano Cagnin 760176\Fu, Y., Wu, P. H., Beane, T., Zamore, P. D. and Weng, Z. (2018). Elimination of PCR duplicates in RNA-seq and small RNA-seq using unique molecular identifiers. BMC Genomics 19(1): 531." target="_blank">Elimination of PCR duplicates in RNA-seq and small RNA-seq using unique molecular identifiers.</a> <em>BMC Genomics</em> 19(1): 531.</li><li>Guo, X., Chang, Q., Pei, H., Sun, X., Qian, X., Tian, C. and Lin, H. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28757056" target="_blank">Long non-coding RNA-mRNA correlation analysis reveals the potential role of HOTAIR in pathogenesis of sporadic thoracic aortic aneurysm.</a> <em>Eur J Vasc Endovasc Surg</em> 54(3): 303-314.</li><li>He, P., Wei, P., Chen, X., Lin, Y. and Peng, J. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31029056" target="_blank">Identification and characterization of microRNAs in the gonad of Trachinotus ovatus using Solexa sequencing.</a> <em>Comp Biochem Physiol Part D Genomics Proteomics</em> 30: 312-320.</li><li>Huang da, W., Sherman, B. T. and Lempicki, R. A. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19131956" target="_blank">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources.</a> <em>Nat Protoc</em> 4(1): 44-57.</li><li>Krishna, S., Nair, A., Cheedipudi, S., Poduval, D., Dhawan, J., Palakodeti, D. and Ghanekar, Y. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23166307" target="_blank">Deep sequencing reveals unique small RNA repertoire that is regulated during head regeneration in Hydra magnipapillata.</a> <em>Nucleic Acids Res</em> 41(1): 599-616.</li><li>Mammucari, C., Gherardi, G., Zamparo, I., Raffaello, A., Boncompagni, S., Chemello, F., Cagnin, S., Braga, A., Zanin, S., Pallafacchina, G., Zentilin, L., Sandri, M., De Stefani, D., Protasi, F., Lanfranchi, G. and Rizzuto, R. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25732818" target="_blank">The mitochondrial calcium uniporter controls skeletal muscle trophism <em>in vivo</em>.</a> <em>Cell Rep</em> 10(8): 1269-1279.</li><li>Risso, D., Perraudeau, F., Gribkova, S., Dudoit, S. and Vert, J. P. (2018a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29348443" target="_blank">A general and flexible method for signal extraction from single-cell RNA-seq data.</a> <em>Nat Commun</em> 9(1): 284.</li><li>Risso, D., Purvis, L., Fletcher, R. B., Das, D., Ngai, J., Dudoit, S. and Purdom, E. (2018b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30180157" target="_blank">clusterExperiment and RSEC: A Bioconductor package and framework for clustering of single-cell and other large gene expression datasets.</a> <em>PLoS Comput Biol</em> 14(9): e1006378.</li><li>Saeed, A. I., Sharov, V., White, J., Li, J., Liang, W., Bhagabati, N., Braisted, J., Klapa, M., Currier, T., Thiagarajan, M., Sturn, A., Snuffin, M., Rezantsev, A., Popov, D., Ryltsov, A., Kostukovich, E., Borisovsky, I., Liu, Z., Vinsavich, A., Trush, V. and Quackenbush, J. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12613259" target="_blank">TM4: a free, open-source system for microarray data management and analysis.</a> <em>Biotechniques</em> 34(2): 374-378.</li><li>Sales, G., Calura, E., Martini, P. and Romualdi, C. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23666626" target="_blank">Graphite Web: Web tool for gene set analysis exploiting pathway topology.</a> <em>Nucleic Acids Res</em> 41(Web Server issue): W89-97.</li><li>Schiaffino, S. and Reggiani, C. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=schiaffino+reggiani+2011" target="_blank">Fiber types in mammalian skeletal muscles.</a> <em>Physiol Rev</em> 91(4): 1447-1531.</li><li>Tusher, V. G., Tibshirani, R. and Chu, G. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11309499" target="_blank">Significance analysis of microarrays applied to the ionizing radiation response.</a> <em>Proc Natl Acad Sci U S A</em> 98(9): 5116-5121.</li><li>Wang, Y., and Pessin, J.E. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23493017" target="_blank">Mechanisms for fiber-type specificity of skeletal muscle atrophy.</a> <em>Curr Opin Clin Nutr Metab Care</em> 16, 243-250.</li><li>Wellenreuther, R., Schupp, I., Poustka, A., Wiemann, S., and German c, D.N.A.C. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=SMART+amplification+combined+with+cDNA+size+fractionation+in+order+to+obtain+large+full-length+clones." target="_blank">SMART amplification combined with cDNA size fractionation in order to obtain large full-length clones.</a> <em>BMC Genomics</em> 5, 36.</li><li>Wu, H., Gallardo, T., Olson, E.N., Williams, R.S., and Shohet, R.V. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=Transcriptional+analysis+of+mouse+skeletal+myofiber+diversity+and+adaptation+to+endurance+exercise." target="_blank">Transcriptional analysis of mouse skeletal myofiber diversity and adaptation to endurance exercise.</a> <em>J Muscle Res Cell Motil</em> 24, 587-592.<br/></li></ol>