<body>
            <h1>ATAC-seq on Sorted Adult Mouse Neurons</h1>
            <p><strong>Authors:</strong> Erin A. Clark, Yasuyuki   Shima, Sacha Nelson</p>
            <h2>Abstract</h2>
            <p>Transcription regulation is a key aspect of cellular identity established during development and maintained into adulthood. Molecular and biochemical assays that probe the genome are critical tools in exploring mechanisms of transcription regulation and cell type identity. The mammalian brain is composed of a huge diversity of cell types with distinct properties and functions. To understand these specific roles, it is necessary to selectively target cell populations for study. However, the need to selectively study restricted cell populations poses a challenge in neurobiology. It is often difficult to collect sufficient cellular input for many standard biochemical and molecular assays. Recently, important advances have been made to scale assays down, opening up new frontiers to explore molecular mechanisms in neurons. Concurrently, methodologies for preparing neurons for such assays has advanced taking into consideration specific methods to preserve the cell biology meant to be assayed. Here we describe a method for preparing live neurons from adult brain tissue for the Assay for Transposase Accessible Chromatin (ATAC).</p>
            <p><strong>Keywords:</strong> ATAC, Neuron, Chromatin, Tn5, Transposase</p>
            <h2>Background</h2><p>The Assay for Transposase Accessible Chromatin coupled with next generation sequencing (ATAC-seq) is becoming a widely used method to assess genome-wide chromatin accessibility (Buenrostro <em>et al.</em>, 2013; Corces <em>et al.</em>, 2017). A major advantage of ATAC-seq over older methods such as DNase I hypersensitivity mapping, is that it can produce high-quality data from significantly smaller input. This allows chromatin and transcriptional regulatory mechanisms to be explored in previously inaccessible populations of rare or sparse cell types. This is particularly useful in the nervous system where cellular diversity is incredibly high and cells of interest are post-mitotic and often sparse. While single cell methods play an important role in identifying heterogeneity and defining certain characteristics of cell types, they are often not suitable for deeper, reproducible characterization, or targeted manipulation and comparison of cell types. The growing number of marker genes and other genetic strategies to label restricted neuronal populations (Shima <em>et al.</em>, 2016) is increasing the depth and breadth of molecular and biochemical assays, including ATAC-seq, used to study diverse types of neurons. However, isolation of healthy, high-quality neurons from live tissue requires special considerations. Here we adapt an earlier protocol for isolating neurons from live, adult brain tissue (Hempel <em>et al.</em>, 2007) to generate ATAC-seq profiles from sorted populations of 20,000-50,000 cells expressing a fluorescent marker. We successfully used this method to generate ATAC-seq profiles from a diverse range of adult neuronal types (Sugino <em>et al.</em>, 2019). </p>
        </body>

<p class="pptt" id="biaoti33951">## Materials and Reagents</p><p class="ppzzs"><strong><em>Materials</em></strong> </p><ol class="npwtwo"><li>Tube with cell strainer cap (35 Âµm mesh, Thermo Fisher Scientific, catalog number: 08-771-23)</li><li>Cell strainer (nylon mesh with 40, 70, and 100 Âµm micron pores, Thermo Fisher Scientific, catalog number: 08-771-19)</li><li>Filter-Steriflip, 50 ml, membrane tube device (Thermo Fisher Scientific, catalog number: SCGP00525)</li><li>Tools and supplies for animal dissection and incubation of brain slices (Hempel <em>et al.</em>, 2007) </li><li>100-mm glass Petri dish</li><li>Glass Pasteur pipettes with tips fire-polished to diameters of ~600, 400-300 and, ~200 Âµm (measured under the microscope with a fine-grade ruler or stage micrometer)</li><li>Gas Diffuser (VWR, catalog number: 80073-710)<br/><br/></li></ol><p class="ppzzs"><strong><em>Reagents</em></strong><br/><strong><em> For Cutting Solution (CS) and modified ACSF (mACSF)</em></strong> </p><ol class="npwtwo"><li>95% O<sub>2</sub>/5% CO<sub>2</sub> compressed gas (for solution oxygenation)</li><li>N-Methyl-D-glucamine (NMDG) (Sigma, catalog number: M2004-500G)</li><li>D-Trehalose Dihydrate (Sigma-Aldrich, catalog number: T5251)</li><li>Sodium Hydroxide (NaOH) (Thermo Fisher Scientific, catalog number: S318-1)</li><li>Choline chloride (Thermo Fisher, catalog number: AC110295000)</li><li>Sodium chloride (Thermo Fisher, catalog number: S271-500)</li><li>Potassium chloride (Thermo Fisher, catalog number: P217-500)</li><li>NaH<sub>2</sub>PO<sub>4</sub>Â·H<sub>2</sub>O (Thermo Fisher, catalog number: S369-500)</li><li>NaHCO<sub>3</sub> (Thermo Fisher, catalog number: S233-500)</li><li>HEPES (Thermo Fisher, catalog number: BP310-500)</li><li>Na ascorbate (AppliChem, catalog number: A5048)</li><li>Thiourea (Sigma-Aldrich, catalog number: T8656-500G)</li><li>Na pyruvate (AppliChem, catalog number: A4859)</li><li>N-acetyl-L-cystein (Sigma-Aldrich, catalog number: A7250-100G)</li><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Thermo Fisher, catalog number: M65-500)</li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (Thermo Fisher, catalog number: BP510-250)</li><li>Tris-HCl (Sigma-Aldrich, catalog number: T2819)</li><li>EDTA (Sigma-Aldrich, catalog number: E5134-500G)</li><li>Ketamine 100 mg/ml (Webster Vet Supply, catalog number: 07-803-6637)</li><li>Kynurenic acid (Sigma-Aldrich, catalog number: K3375-1G)</li><li>Tetrodotoxin (TTX) (Tocris, catalog number: 1069)</li><li>TTX 0.1 mM (see Recipes)</li><li>Ketamine 500 mM (see Recipes)</li><li>Cutting Solution (CS) (see Recipes)</li><li>Modified Artificial Cerebral Spinal Fluid (mACSF) (see Recipes)<br/><br/></li></ol><p class="ppzzs"><strong><em>For tissue preparation and single cell suspension preparation</em></strong> </p><ol class="npwtwo" start="26"><li>Loctite 40104 (Construction Equipment Plus, catalog number: 82525940)</li><li>Pronase (Sigma-Aldrich, catalog number: P5147-1G) </li><li>Fetal bovine serum (FBS) (Atlanta Biological, catalog number: S11550)</li><li>DNase I (Thermo Fisher, catalog number: AM2222)</li><li>Percoll (Sigma-Aldrich, catalog number: P1644-100ML)<br/><br/></li></ol><p class="ppzzs"><strong><em>For nuclei preparation and tagmentation</em></strong> </p><ol class="npwtwo" start="31"><li>Digitonin (Millipore, catalog number: 300410-250MG)</li><li>Tween-20 (Thermo Fisher, catalog number: BP337-100)</li><li>Dimethyl Formamide (Sigma-Aldrich, catalog number: 270547-100ML)</li><li>100% acetic acid (Thermo Fisher, catalog number: A38500)</li><li>Tn5 <br/><em>Note: We used a discontinued kit (Illumina, catalog number: FC-121-1030). However, Tn5 can be purchased from several manufactures specifically for ATAC-seq and there are published protocols for producing and purifying Tn5 in house (Picelli et al., 2014).</em> </li><li>IGEPAL CA-630 (Sigma-Aldrich, catalog number: I8896-50ML)</li><li>1% (100x) Digitonin (see Recipes)</li><li>10% (100x) Tween-20 (see Recipes)</li><li>2x TD buffer (see Recipes)</li><li>Lysis buffer (see Recipes)<br/><br/></li></ol><p class="ppzzs"><strong><em>For library preparation</em></strong> </p><ol class="npwtwo" start="41"><li>Zymo DNA Clean and Concentrator-5 Kit (Zymo, catalog number: D4014)</li><li>1x Low TE buffer (see Recipes)</li><li>NEBNext High-Fidelity 2x PCR Master Mix (New England Labs, catalog number: M0541)</li><li>SYBR Green 10,000x (Thermo Fisher, catalog number: S7567)</li><li>DMSO (Sigma-Aldrich D8418-50ML)</li><li>Primers (Tables 1 and 2)<br/><br/><strong>Table 1. Library amplification primers</strong> (HPLC purified) (24097267)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:900px;"><tbody><tr style="border-top:1;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>name</strong><strong></strong><br/></td><td><strong>sequence</strong><strong></strong><br/></td></tr><tr><td>Ad1_noMX<br/></td><td>AATGATACGGCGACCACCGAGATCTACACTCGTCGGCAGCGTCAGATGTG<br/></td></tr><tr><td>Ad2.1_TAAGGCGA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTCGCCTTAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.2_CGTACTAG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCTAGTACGGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.3_AGGCAGAA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTTCTGCCTGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.4_TCCTGAGC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGCTCAGGAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.5_GGACTCCT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAGGAGTCCGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.6_TAGGCATG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCATGCCTAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.7_CTCTCTAC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGTAGAGAGGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.8_CAGAGAGG <br/></td><td>CAAGCAGAAGACGGCATACGAGATCCTCTCTGGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.9_GCTACGCT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAGCGTAGCGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.10_CGAGGCTG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCAGCCTCGGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.11_AAGAGGCA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTGCCTCTTGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.12_GTAGAGGA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTCCTCTACGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.13_GTCGTGAT <br/></td><td>CAAGCAGAAGACGGCATACGAGATATCACGACGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.14_ACCACTGT <br/></td><td>CAAGCAGAAGACGGCATACGAGATACAGTGGTGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.15_TGGATCTG <br/></td><td>CAAGCAGAAGACGGCATACGAGATCAGATCCAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.16_CCGTTTGT<br/></td><td>CAAGCAGAAGACGGCATACGAGATACAAACGGGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.17_TGCTGGGT<br/></td><td>CAAGCAGAAGACGGCATACGAGATACCCAGCAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.18_GAGGGGTT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAACCCCTCGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.19_AGGTTGGG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCCCAACCTGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.20_GTGTGGTG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCACCACACGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.21_TGGGTTTC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGAAACCCAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.22_TGGTCACA <br/></td><td>CAAGCAGAAGACGGCATACGAGATTGTGACCAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr><td>Ad2.23_TTGACCCT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAGGGTCAAGTCTCGTGGGCTCGGAGATGT<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Ad2.24_CCACTCCT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAGGAGTGGGTCTCGTGGGCTCGGAGATGT<br/></td></tr></tbody></table><br/><strong>Table 2. QC qPCR primers<br/></strong> <table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr style="border-top:1;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td>Uvrag_F <br/></td><td>CTAAACCTGGGGCGTAACTT<br/></td><td>(TSS primer)<br/></td></tr><tr><td>Uvrag_R<br/></td><td>CTAGAGCAATAGCCCACGAG <br/></td><td><br/></td></tr><tr><td>Usp5_F<br/></td><td>ATGGAAGGAAGTCGTGGTCG<br/></td><td>(TSS primer)<br/></td></tr><tr><td>Usp5_R<br/></td><td>CCGGGTCCATAAAGACGAGT <br/></td><td><br/></td></tr><tr><td>Ap2a1_F<br/></td><td>AGTACCGGGCTATTTCCGTT<br/></td><td>(TSS primer)<br/></td></tr><tr><td>Ap2a1_R<br/></td><td>GGAGCTGGTAGTCCATCCAA <br/></td><td><br/></td></tr><tr><td>ATACbg1_F<br/></td><td>CTAGTGGGGTAAATTGGCAGG<br/></td><td>(background primer)<br/></td></tr><tr><td>ATACbg1_R<br/></td><td>AACGGCACACAAATAGGCTC<br/></td><td><br/></td></tr><tr><td>ATACbg2_F<br/></td><td>ATATGCCCAGAACTACCGAC<br/></td><td>(background primer)<br/></td></tr><tr><td>ATACbg2_R<br/></td><td>TTCTTTTCAGAAACAGGCGG<br/></td><td><br/></td></tr><tr><td>ATACbg3_F<br/></td><td>TTCTGCACTTTTCATGGCAG<br/></td><td>(background primer)<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>ATACbg3_R<br/></td><td>ACAGATGCAGGACCCATAAG<br/></td><td><br/></td></tr></tbody></table><em>Note: These primers are used to screen libraries post-amplification but prior to sequencing. Primers should amplify 3-4 regions at the TSS of genes that are highly accessible in your samples and 3-4 background regions that are typically not accessible in your samples. The primers listed above worked well for mouse neurons isolated from multiple different tissues.</em><br/><br/></li></ol><ol class="npwtwo" start="47"><li>100x SYBR Green (see Recipes)<br/><br/></li></ol><p class="ppzzs"><strong><em>For library quality assessment</em></strong> </p><ol class="npwtwo" start="48"><li>KAPA Library Quantification kit (Roche, catalog number: KK4854)</li><li>High Sensitivity DNA kit (Agilent, catalog number: 5067-4626)</li><li>SSO Evagreen Master mix (Bio-Rad, catalog number: 1725203)<br/></li></ol><p class="pptt" id="biaoti33952">### Equipment</p><ol class="npwtwok"><li>Vibratome (such as Leica VT1000S)</li><li>Flow Cytometer (such as BD FACSAria)</li><li>Dissection stereomicroscope, such as Leica MZ16FA (Leica) equipped with fluorescence and transmitted light (such as Leica MZ16FA).</li><li>Osmometer (such as Advanced Instruments MicroOmometer 3300)</li><li>Real-time PCR thermocycler (such as Corbett Research RotorGene 6000)</li><li>Bioanalyzer (such as Agilent 2100)</li><li>Centrifuges (such as Sorvall Legend RT+ Centrifuge and Eppendorf Centrifuge, model: 5415 D) </li><li>Laboratory water bath (such as Fisher Scientific Isotemp 110 Water Bath)<br/></li></ol><p class="pptt" id="biaoti33953">### Software</p><ol class="npwtwok"><li>FastQC (Andrews, 2010)</li><li>bowtie2 (Langmead and Salzberg, 2012)</li><li>picard-tools-1.123 (Broad Institute, 2019)<br/></li></ol><p class="pptt" id="biaoti33954">### Procedure</p><p class="ppzzs"><strong>Procedure Overview</strong> </p><ol class="npwtwo"><li>Harvest and section brain tissue (1-2 h depending on the number of mice).</li><li>Micro dissection and tissue digestion (1-1.5 h).</li><li>Trituration and Percoll separation (1-1.5 h depending on the amount of tissue).</li><li>Sorting (variable depending on machine and sample size, typically more than 2 h).</li><li>Preparing nuclei and tagmentation (~1 h).<br/><em>Pause point: Tagmented nuclei can be stored at -20 Â°C for up to two weeks.</em> </li><li>Library preparation (2-3 h depending on the number of samples).<br/><em>Pause point: Amplified DNA can be stored at -20 Â°C.</em> </li><li>Quality assessment (2-3 h).<br/><br/></li></ol><p class="ppzzs"><strong>Harvest and Section Brain Tissue</strong> </p><ol class="npwtwo"><li>Add the following reagents to Cutting Solution (CS) and Modified Artificial Cerebral Spinal Fluid (mACSF):<br/>Ketamine (0.05 mM final concentration) or 0.05 mM APV<br/>TTX (100 nM final concentration) <br/>Kynurenic acid (0.8 mM final concentration) or 0.02 mM DNQX<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>These inhibitors are important for high quality ATAC from neurons. They prevent excitotoxicity as tissue is sliced and dissociated.</em> </li><li><em>The amount of CS and mACSF needed will depend on the number and size of animals perfused and the set-up used. For reference, when perfusing four young adult mice typically weighing ~20 mg, 300 ml of CS and 100 ml of mACSF are prepared. If working with a cell type that requires multiple rounds of Percoll centrifugation (Step 26), extra mACSF may be needed. Prior to attempting the protocol, we recommend calculating the volume needed by measuring and summing the volumes of liquid needed for each step. To allow for the possibility of poor perfusions requiring additional volume (Step 4d), prepare 50 ml extra of CS.</em> </li></ol></li><li>To mACSF, add 5% (w/v) Trehalose.<br/><em>Note: This is a protectant for tissue dissociation (Saxena et al., 2012). The effects of Trehalose are a bit controversial (Lee et al., 2018), but we found it increases the number of intact neurons after trituration.</em> </li><li>Stir at room temp (RT) and oxygenate using gas diffuser until Kynurenic acid dissolves.<br/><em>Note: mACSF can remain at room temp.</em><br/><ol class="npwthree"><li>Move enough CS to a new container to oxygenate and suspend one or more cell strainers.<br/><ol class="npwfour"><li>Place container in a water bath at 34 Â°C.</li><li>After sectioning, brain slices will recover at 34 Â°C (Ting <em>et al.</em>, 2014).</li></ol></li><li>Move remaining CS to 4 Â°C and oxygenate.</li></ol></li><li>Harvest brain.<br/><ol class="npwthree"><li>Make an ice bath around the vibratome buffer tray.</li><li>Fill vibratome buffer tray with cold CS and oxygenate.</li><li>Administer lethal dose of appropriate anesthetic.</li><li>Ideally before heart stops beating, perfuse cold CS via trans-cardiac perfusion through the left ventricle exiting the right atrium until blood has been completely replaced with CS and tissues such as the liver and lungs are no longer red (Typically 20-30 ml depending on mouse size). If the liver has red patches this indicates poor perfusion of tissues. Continue perfusing until the liver color is evenly white/tan and only clear CS is seen exiting the heart. For a detailed example of trans-cardiac perfusion technique see Gage <em>et al.</em> (2012).<br/></li><li>Quickly remove brain and place in oxygenating CS filled vibratome buffer tray. <br/><em>Note: For dissection details, see Papouin and Haydon (2018).</em><br/></li><li>If slicing multiple brains repeat (c)-(f) for each animal.<br/></li><li>Make sure brain(s) in tray do not come in direct contact with O<sub>2</sub> bubbles.<br/></li><li>Allow the last brain to equilibrate for 5 min.</li></ol></li><li>Mount brains for slicing.<br/><ol class="npwthree"><li>Place wet blot paper with oxygenated cold CS inside a 100-mm petri dish.</li><li>Place brain(s) on wetted paper.</li><li>Remove unwanted tissue and cut to create a flat surface to affix the brain.<br/><ol class="npwfour"><li>Examples of two options for mounting the brain are shown in Video 1. The first brain is mounted by removing the cerebellum so that rostral structures will be sliced first. The second brain is oriented in the opposite direction so that caudal structures may be sliced. Both orientations will produce coronal slices.</li><li>Use a fine scalpel to minimize tissue distortion that can damage neurons.<br/><br/><!--video3382-1å¼å§--><div class="videodiv" id="vd3382-1" style="height:480px;"></div><div class="videophoto" id="vp3382-1"><a href="http://www.bio-protocol.org/e3382" target="_blank"> </a><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3382/3382-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Mounting brains on vibratome platform.</strong> (Animal experimentation: All experiments were conducted in accordance with the requirements of the Institutional Animal Care and Use Committees at Brandeis University (protocol #17001).) <br/><br/></div></li></ol></li><li>Attach brains to platform with Loctite. <br/>Affix the brain in an orientation such that cells of interest are away from the Loctite.</li><li>Move platform back to oxygenated CS. <br/></li></ol><em>Note: There is no need to wait for Loctite to air dry.</em><br/><br/></li></ol><p class="ppzzs"><strong>Micro Dissection and Tissue Digestion</strong> </p><ol class="npwtwo" start="6"><li>Slice mounted brain(s) into 300 Âµm sections through the region of interest.<br/><ol class="npwthree"><li>Set speed to 3 (on a scale of 0-10) and frequency between 6 and 8 (on a scale of 0-10). <br/><em>Note: Settings should be optimized on individual equipment to minimize tissue distortion/compression. If brains tend to come loose during slicing, then the speed may be too fast or the frequency too low. </em><br/></li><li>Reducing tissue deformation/compression will increase yield. We recommend always using a fresh blade.</li></ol></li><li>Using a wide mouth pipette or a plastic transfer pipette that has been cut to make the opening larger gently move slices to cell strainers in oxygenated CS at 34 Â°C<br/><ol class="npwthree"><li>The strainer protects slices from coming into direct contact with oxygenation bubbles.</li><li>If working with cells expressing a fluorescent marker, protect samples from light as much as possible from here on to minimize photobleaching.</li><li>Incubate for 10-12 min.</li></ol></li><li>Move strainer containing slices to oxygenated mACSF at RT.<br/><ol class="npwthree"><li>Ensure slices are submerged.</li><li>Allow equilibrating for 5-10 min.</li></ol></li><li>Dissolve pronase (10% w/v) in RT oxygenated mACSF (20-40 ml) and pour into a 100 mm petri dish.</li><li>Transfer the strainer with slices to Petri dish with pronase solution. Use a wide mouth pipette or cut plastic transfer pipette to move slices from the strainer and into the solution. Incubate for 1-1.5 h at RT.</li><li>While slices digest, micro-dissect region(s) of interest under a stereomicroscope.<br/><ol class="npwthree"><li>If dissecting a large number of slices, it is ideal to work in batches using several dishes so that the tissue waiting to be dissected or post-dissection can be oxygenated in the pronase solution.</li><li>When finished dissection, oxygenate tissue pieces in pronase solution for any remaining time on the digest incubation.</li><li>This step should be optimized to reduce non-target tissue. Enriching the cells of interest will reduce flow cytometry sort times.</li></ol></li><li>Prepare 30-50 ml of oxygenated mACSF + 1% FBS (mACSF + FBS), filter with Steriflip and keep on ice</li><li>Add 1 U/ml DNase to 5-10 ml of filtered mACSF + FBS (mACSF + FBS + DNase), keep on ice.</li><li>Move tissue pieces to 1 ml of mACSF + FBS.<br/>Wash tissue 2-3 x with mACSF + FBS to quench and remove protease by allowing tissue to settle to the bottom of the tube and removing most of the supernatant. Add fresh mACSF + FBS and repeat. </li><li>Suspend tissue pieces in mACSF + FBS + DNase and divide between Eppendorf tubes such that the volume of tissue does not exceed 0.1% of the volume (Figure 1).<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>DNase prevents genomic DNA released by lysed cells from clumping and blocking the pipette opening during trituration.</em> </li><li><em>Trituration with too dense cell solution results in low yield of live cells.</em><br/><br/><img alt="" class="layerphoto" height="343" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190928/20190928063828_3524.jpg" width="300"/><br/><strong>Figure 1. Example of tissue amount for trituration in 1 ml</strong><br/><br/></li></ol></li><li>Keep tissue on ice.</li><li>With the 600 Î¼m fire-polished Pasteur pipette, gently triturate the sample ten times.<br/><ol class="npwthree"><li>Pre-wet the pipette to prevent tissue from sticking to the inside walls.</li><li>Avoid making bubbles as this can damage cells.</li><li>The pipette mouth should not be held against the wall or floor of the tube.</li><li>The solution should turn cloudy with large chunks remaining.</li></ol></li><li>Incubate tube(s) on ice allowing chunks to settle for a few minutes.</li><li>Filter supernatant using tube with cell strainer cap.<br/><em>Note: You may need to prewet the strainer with mACSF + FBS to start gravity flow.</em> </li><li>Add 1 ml fresh mACSF + FBS + DNase I to remaining tissue chunks.<br/><em>Note: If working with precious small amounts of tissues, rinse the cell strainer cap with the 1 ml and transfer any chunks left in the top back to the Eppendorf for the next trituration step.</em> </li><li>Triturate ten times with the 400 Î¼m fire-polished Pasteur pipette as described in Step 17.<br/><em>Note: The solution should become cloudy again, and remaining chunks be smaller.</em> </li><li>Filter supernatants (as in step 19) into the same tube and add fresh mACSF + FBS + DNase (as in Step 20).<br/><br/></li></ol><p class="ppzz"><strong>Trituration and Percoll Separation</strong> </p><ol class="npwtwo" start="23"><li>Triturate ten times with the 200 Î¼m fire-polished Pasteur pipette. Again, filter supernatant into the same tube.</li><li>If chunks remain you may repeat the 200 Î¼m trituration on leftover chunks.</li><li>Transfer the combined, filtered cell suspension to a 15 ml falcon tube.</li><li>Centrifuge at 4 Â°C through a Percoll solution. Always load ~25 Î¼l of 100% Percoll at the bottom of the tube underneath the Percoll solution to act as a cushion for your cell pellet. <br/><em>Note: This step should be optimized for the cell type of interest. The goal is to separate cell bodies from smaller debris to make sorting more efficient. We proved example conditions from three types of neurons that are small, medium, and large in size for guidance in optimizing. </em><br/><ol class="npwthree"><li>Example conditions<br/><strong><em>Percoll solution</em></strong><br/><ol class="npwfour"><li>For cerebellar Purkinje neurons, which are large, the cell suspension was brought to a final concentration of 5% (v/v) Percoll in 5-10 ml.</li><li>For cortical pyramidal neurons, the cell suspension was brought to a final concentration of 10% (v/v) Percoll in 5-10 ml.</li><li>For cerebellar granule neurons, which are small, the cell suspension was brought to a final concentration of 20% (v/v) Percoll in 5-10 ml.</li></ol><br/><strong><em>Centrifugation</em></strong><br/><ol class="npwfour"><li>For cerebellar Purkinje neurons in 10 ml, 54 <em>x g</em> for 15 min is used.</li><li>For cortical pyramidal neurons in 10 ml, 54 <em>x g</em> for 15 min is used.</li><li>For cerebellar granule neurons in 10 ml, 91 <em>x g</em> for 20 min is used.<br/><em>Note: The first cerebellar granule cell pellet typically contains too much debris. To clean it up further the pellet is resuspended in another 5 ml of 20% Percoll, and centrifuged again.</em> </li></ol></li><li>When optimizing, move the supernatant to a new tube and keep on ice.<br/><ol class="npwfour"><li>~25 Î¼l of the supernatant can be placed on slide or small dish. Once cells have settled for 1-2 min the droplet can be examined under a fluorescent microscope to estimate the number of cell bodies that failed to pellet.</li><li>If needed, you may add a Percoll cushion to the bottom of the tube containing the supernatant and centrifuge again. This typically results in a pellet with additional cells, but often contains more debris than the first pellet.</li></ol></li><li>Considerations<br/><ol class="npwfour"><li>Pelleting cell bodies through the Percoll solution is a balance between speed, time, the concentration of Percoll (and volume, to lesser degree). Less force and less time spent centrifuging typically produced higher quality samples. Optimize for the minimal Percoll concentration that removes enough debris to significantly shorten sorting on the flow cytometer. It is not necessary to fully fractionate the sample.</li><li>An additional option is to layer the cell suspension on top of the Percoll solution. This has the disadvantage of making it difficult to do additional spins on the supernatant but may allow better fractionation of some samples.</li></ol></li></ol></li><li>Suspend cell pellet in 1-3 ml cold mACSF + FBS immediately after removing supernatant.<br/><em>Note: Avoid leaving cells pelleted for long periods of time.</em> </li><li>Filter through a clean tube with cell strainer cap.<br/><br/></li></ol><p class="ppzzs"><strong>Sorting</strong> </p><ol class="npwtwo" start="29"><li>Sort on a flow cytometer<br/>During optimization we recommend sorting ~100 cells onto a slide or small dish and examine the droplet under a fluorescent microscope to confirm cells of interest were isolated and estimate debris contamination using fluorescent illumination and brightfield. <br/><ol class="npwthree"><li>Gating strategies are highly cell type and flow cytometer dependent. The goal is to capture individual healthy cells containing high levels of fluorescence signal. Our gates typically capture the top 1-10% of fluorescent events and subset to capture events with side scatter and forward scatter indicative of single cells and excluding doublets/clumps and debris.</li><li>Adding a small amount of DNA dye such as Hoechst be useful in distinguishing small cell bodies containing a nucleus from debris when working with small neurons such as granule cells.<br/><em>Notes: </em><br/><ol class="npwfour"><li><em>Hoechst will penetrate live cells. If using other dyes such as DAPI or propidium iodide Triton x 100 will need to be added to permeabilize cells.</em> </li><li><em>With the bright mCitrine+ cells from Nelson enhancer trap lines (Shima et al., 2016) we often observe cell lysis under the microscope. A cell becomes dim and the fluorescent protein can be seen diffusing away. Therefore, simply estimating the number of cells that lack fluorescence by eye as a measure of contamination would likely give an overestimate and should not be used for this purpose.</em> </li></ol></li><li>Sort bulk samples into a tube containing cold mACSF + FBS.</li><li>Optional: to increase the quality of difficult samples tagment the sample in ~10,000 cell batches.<br/><ol class="npwfour"><li>For this, collect ~10,000 cells and follow Steps 29-39 while the next batch of cells is collected. Scale the tagmentation reaction in Step 36 by 1/5.</li><li>Batching the sample reduces the overall time early sorted cells sit in suspension. Batches can be pooled later for library prep as needed, prioritizing batches collected earlier in the sort as these tended to give higher quality ATAC.<br/><br/></li></ol></li></ol></li><li>Pellet sorted cells at 4 Â°C.<br/><ol class="npwthree"><li>Centrifuge sorted cells at 500 <em>x g</em> for 2 min.</li><li>Rotate tube 180Â° in holder, centrifuge again at 700 <em>x g</em> for 1 min.</li><li>Rotate tube 180Â° again and centrifuge at 500 <em>x g</em> for 2 min.<br/><em>Note: Alternating the force angle helps collect cells off the tube wall and reduces cell loss when removing the supernatant.</em> </li><li>The pellet is often not visible for samples &lt; 20,000 cells.<br/><br/></li></ol></li></ol><p class="ppzzs"><strong>Preparing Nuclei and Tagmentation Solution</strong> </p><ol class="npwtwo" start="31"><li>Remove the supernatant leaving ~2 Î¼l and immediately resuspend the cell pellet by gentle flicking.</li><li>Add 100 Î¼l of Lysis Buffer and mix gently.</li><li>Incubate on ice for 2-5 min.<br/><em>Note: The goal is to lyse the cell membrane while leaving the nucleus mostly intact.</em> </li><li>Quench lysis by diluting with 1 ml of mACSF+FBS.<br/>This reduces the concentration of Triton X-100 allowing 2-3 Î¼l to be left behind at Step 37 without inhibiting tagmentation later on.</li><li>Pellet nuclei at 4 Â°C.<br/><ol class="npwthree"><li>Centrifuge nuclei at 700 <em>x g</em> for 2 min.</li><li>Rotate tube 180Â°, centrifuge at 1,000 <em>x g</em> for 1 min.</li><li>Rotate tube 180Â°, centrifuge at 700 <em>x g</em> for 2 min.</li><li>The pellet is often not visible for samples &lt; 40,000 nuclei.<br/></li></ol></li><li>Prepare 1x tagmentation solution:<br/>For 50,000 cells:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>25 Î¼l <br/></td><td>2x TD<br/></td></tr><tr><td>0.5 Î¼l <br/></td><td>1% digitonin<br/></td></tr><tr><td>0.5 Î¼l <br/></td><td>10% Tween-20<br/></td></tr><tr><td>(variable) Î¼l <br/></td><td>PBS for final volume of 50 Î¼l* <br/></td></tr><tr><td>(variable) Î¼l<br/></td><td><span style="font-family:'&quot;font-size:small;"></span>Tn5 transposase (100 nM final)**<br/></td></tr></tbody></table><em>Notes:</em><br/><ol class="npwthree"><li><em>*Accounting for the 2-3 Î¼l of supernatant left in Step 37 becomes more important with smaller tagmentation reactions where all volumes are scaled down except Step 37)</em> </li><li><em>**The volume of Tn5 will vary depending on the enzyme source. 100 nM is recommended as a starting point for further optimization. This concentration worked well for most samples but had to be doubled for cerebellar granule neurons, likely due to significant debris contamination.</em> </li></ol></li><li>Remove most of the supernatant from pelleted nuclei.<br/><em>Note: It can be difficult to remove all of the volume without disturbing the pellet. Leave 2-3 Î¼l and resuspend nuclei by gentle flicking in this volume. </em> </li><li>Add 1x tagmentation solutionand flick tube gently to mix.</li><li>Incubate at 37 Â°C with agitation for 30 min.<br/><ol class="npwthree"><li>Agitation option 1: Incubate stationary and gently flick tube every ~5 min.</li><li>Agitation option 2: Incubate in a large bacterial flask incubator shaker set to ~250 RPM.</li></ol><em>Note: This protocol typically produced libraries with fragment sizes ranging from di-nucleosome to subnucleosomal (Figure 2). If nucleosome positioning is an important analytical goal, incubation time and/or tagmentation reaction may need to be adjusted.</em> </li><li>Add 5 volumes of DNA Binding Buffer from Zymo DNA Clean and Concentrator-5 Kit. <br/><em>Pause point: Sample can be stored at -20 Â°C for up to two weeks.</em><br/><br/></li></ol><p class="ppzzs"><strong>Library Preparation</strong> </p><ol class="npwtwo" start="41"><li>If starting from frozen samples, allow samples to warm to room temp.</li><li>Purify tagmented DNA.<br/><ol class="npwthree"><li>Load and wash columns according to Zymo kit instructions.</li><li>Elute in a volume to obtain ~2,500 cells worth of material/Î¼l.<br/><em>Note: ~1 Î¼l is typically not recovered from the column. Therefore, an extra 1 Î¼l can be added when calculating volumes. </em><br/></li><li>To increase recovery of DNA from the column the volume can be split between two elutions.<br/><ol class="npwfour"><li>For example, 40,000 cells worth of tagmented material can be eluted in 8.5 Î¼l twice to give ~16 Î¼l total.</li><li>The kit recommends using a minimum volume of 6 Î¼l for elution. For samples where two 6 Î¼l elutions would dilute the sample below 2,500 cells/Î¼l, elute 2 times with 6 Î¼l using H<sub>2</sub>O or diluted EB buffer* and rotovap to concentrate to ~2,500 cells worth of material/Î¼l.</li><li>*1x EB buffer comes with the Zymo DNA Clean kit. Dilute with H<sub>2</sub>O such that it will be 1x in the target final volume after rotovap concentration. For example, if the target final volume is 6 Î¼l, rotovaping will concentrate the sample that was eluted twice with 6 Î¼l by 2-fold, so a 1:2 dilution is needed.</li></ol></li></ol></li><li>Prepare PCR Amplification Mix.<br/>For 50,000 cells worth of material :<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:350px;"><tbody><tr><td>25 Î¼l<br/></td><td>NEBNext Master Mix<br/></td></tr><tr><td>2.5 Î¼l<br/></td><td>25 Î¼M Primer Ad1<br/></td></tr><tr><td>2.5 Î¼l<br/></td><td>25 Î¼M Primer Ad2 (barcode)<br/></td></tr><tr><td>0.5 Î¼l<br/></td><td><span style="font-family:'&quot;font-size:small;"></span>100x SYBR Green (in DMSO)<br/></td></tr></tbody></table><em>Note: If final libraries contain significant primer dimer contamination, the primer concentration can be cut in half.</em> </li><li>Add PCR Amplification Mix to the tube containing eluted DNA, mix by pipetting up and down 5-10 times then transfer to appropriate PCR tube(s) for amplification.<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>The original ATAC and Omni-ATAC protocols suggest amplifying a small fraction of the sample in a qPCR thermocycler to determine the optimal number of PCR cycles and then amplify the rest of the sample in a different thermocycler appropriate for the larger volume.</em> </li><li><em>We found the optimal number of cycles varied for a single master mix aliquoted in different volumes as well as between different thermocyclers. To ensure the whole sample was amplified optimally and consistently, the entire sample was amplified in a qPCR thermocycler and monitored in real time to observe which cycle the fluorescent signal began to increase. Therefore, depending on the machine and sample volume, it may be necessary to divide the sample between multiple tubes. Aliquot the sample into the minimum number of PCR tubes (to reduce sample loss) without exceeding the recommended maximum volume for the qPCR machine.</em> </li></ol></li><li>Amplify samples on a real-time PCR thermocycler with the cycling protocol indicated below: <br/>Cycling Protocol<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:320px;"><tbody><tr><td>Step<br/></td><td>Temp (Â°C)<br/></td><td>Time<br/></td></tr><tr><td>1</td><td>72</td><td>5 min<br/></td></tr><tr><td>2</td><td>98</td><td>30 s<br/></td></tr><tr><td>3</td><td>98</td><td>10 s<br/></td></tr><tr><td>4</td><td>63</td><td>30 s<br/></td></tr><tr><td>5</td><td>repeat steps 3-4 for &lt; 12 cycles<br/></td><td><br/></td></tr></tbody></table></li><li>Stop the amplification 2-3 cycles after the fluorescent signal begins to rise. <br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Good quality libraries typically show an increase in fluorescent signal in &lt; 6 cycles.</em> </li><li><em>Libraries that require &gt; 12 cycles typically have low complexity.</em><br/><em> Note: The 2x NEBNext Master Mix is sensitive to freeze thaw cycles. Best practice is to aliquot, store in an enzyme box, and protect from temperature swings while in the freezer. We recommend tracking the number of cycles need for libraries over time. If cycles creep up over batches, it may indicate the need to replace 2x Master Mix and further optimize storage.</em><br/></li><li><em>If obtaining ATAC libraries with a large fraction of PCR duplicates (e.g., &gt; 50%), reduce the number of cycles by 1.</em> </li></ol></li><li>Purify amplified libraries using Zymo DNA Clean and Concentrator-5 Kit.<br/><ol class="npwthree"><li>Pool replicate tubes from the same sample.</li><li>Optional: If concerned about yield, rinse PCR tubes with DNA Binding Buffer.</li><li>Elute in 6 Î¼l of low TE twice (total volume will be ~11 Î¼l).<br/><br/></li></ol></li></ol><p class="ppzzs"><strong>Quality Assessment</strong> </p><ol class="npwtwo" start="48"><li>Move 1 Î¼l to a new tube and dilute 1:2 with low TE. Use this aliquot for quality control testing and quantification.<br/><ol class="npwthree"><li>Run 1 Î¼l on a High Sensitivity Bioanalyzer Chip to examine the size distribution of fragments <br/><em>Note: Figure 2A shows examples of the size distribution diversity from libraries that gave high quality sequencing data. While nucleosome laddering can indicate a high-quality library, good libraries do not always have this pattern.</em> </li><li>Use 0.5 Î¼l for qPCR with primers against genomic regions as in Table 2.<br/><em>Notes:</em><br/><ol class="npwfour"><li><em>Libraries with low complexity may show strong signal from 1-2 TSS regions but lack signal at 1-2 other TSS regions.</em> </li><li><em>High quality libraries typically showed strong signal from at least 3 different TSS regions.</em> </li><li><em>Libraries with good signal to noise at open regions typically gave a mean (TSS): mean (background) ratio &gt; 50. This ratio may need to be established empirically if using other regions, species or tissue types. </em> </li></ol></li><li>Use 0.5 Î¼l to quantify using Kapa Library Quantification Kit according to manufacturerâs instructions. Samples are typically 5-50 nM.<br/><br/><img alt="" class="layerphoto" height="236" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190928/20190928063856_8175.jpg" width="300"/><br/><strong>Figure 2. Examples of fragment size distributions from high quality ATAC libraries that deviated from standard nucleosome laddering.</strong> A. 0.5 Î¼l of sample was run on a High Sensitivity DNA chip. B. Unique non-mitochondrial reads per million from the samples in A. Transcription start site (TSS) enrichment calculated following ENCODE guidelines for all seven samples was â¥ 20. Examples are from multiple different cell types.<br/><br/></li></ol></li><li>Store the remaining 10 Î¼l at -20 Â°C until sequenced.</li><li>Sequence libraries.<br/><ol class="npwthree"><li>The protocol used for sequencing depends on the project goals. Paired-end sequencing is most commonly used with ATAC. This provides data on the total length of the fragment generated by transposition. Filtering data for certain sized fragments can be useful for certain analyses.</li><li>When considering fragments versus reads it is important to keep in mind that the ends of fragments or one end of a read represents the location of Tn5 access to the genome while the fragment itself is the intervening DNA between two transposition events.</li></ol></li></ol><p class="pptt" id="biaoti33955">### Data analysis</p><p class="ppzz">There are numerous options for analysis of ATAC-seq data that depend on the project goals. As with any project involving bioinformatic analysis it is strongly advised to consult with a statistician or bioinformatician to assess the data needed to achieve project goals. This section is meant to provide basic examples of early analytical steps to aid in visualizing data quality, as in Figure 2B. The parameters of each step should be assessed for their appropriateness to individual datasets. A useful resource is the ENCODE project pipeline (<a href="https://www.encodeproject.org/pipelines/ENCPL792NWO/" target="_blank">https://www.encodeproject.org/pipelines/ENCPL792NWO/</a>).</p><ol class="npwtwok"><li>Check sequence quality with FastQC (Andrews, 2010). Assess the read quality by importing data from FastQ files.</li><li>Trim adaptor sequences using cutadapt (version 1.9.1) (Martin, 2015). Using cutadapt (version 1.9.1) on paired end reads:<br/><br/><span style="font-family:Courier New;">&gt;cutadapt -a CTGTCTCTTATACACATCT -A CTGTCTCTTATACACATCT -q 30 --minimum-length 36 -o trimmed_output_1.fastq -p trimmed_output_2.fastq input_file_name_1.fastq.gz input_file_name_2.fastq.gz</span><br/><br/></li><li>Map reads to genome allowing only paired and concordant alignments with a maximum fragment length of 2,000 bp using bowtie2 (Langmead and Salzberg, 2012). Reference genomes can be obtained from iGenome.<br/><br/><span style="font-family:Courier New;">&gt; bowtie2 -t -X2000 --no-mix --no-discordant -x igenome/UCSC_mm10/Sequence/Bowtie2Index/genome -1 trimmed_output_1.fastq -2 trimmed_output_2.fastq -S mapped _output.sam</span><br/><span style="font-family:Courier New;">&gt; samtools view -bS mapped_output.sam &gt; mapped_output.bam</span><br/><span style="font-family:Courier New;">&gt; samtools sort mapped_output.bam -o sorted_output.bam</span><br/><span style="font-family:Courier New;">&gt; samtools index sorted_output.bam</span><br/><br/></li><li>Remove mitochondrial reads using samtools (Li <em>et al.</em>, 2009). Mitochondrial reads can be large contaminant in ATAC data but varies between cell types.<br/><br/><span style="font-family:Courier New;">&gt; samtools idxstats sorted_output.bam | cut -f 1 | grep -v chrM | xargs samtools view -b sorted_output.bam &gt; noMT_output.bam</span><br/><br/></li><li>Remove duplicate reads using picard-tools-1.123 (Broad Institute, 2019).<br/><br/><span style="font-family:Courier New;">&gt; java -jar -Xmx20g MarkDuplicates.jar METRICS_FILE=metrics.txt INPUT= noMT_output.bam OUTPUT=noDupM.bam REMOVE_DUPLICATES=TRUE</span><br/><br/></li><li>Shift unique non-mitochondrial reads to account for Tn5 protein offset using BEDTools-Version-2.16.2 (Quinlan and Hall, 2010).<br/><br/><span style="font-family:Courier New;">&gt; bamToBed -i noDupMT.bam &gt; noDupMT.bed</span><br/><span style="font-family:Courier New;">&gt; cat noDupMT.bed | awk -F $'\t' 'BEGIN {OFS = FS}{if ($6 == "+") {$2 = $2 + 4} else if ($6 == "-") {$3 = $3 - 5}print $0}' &gt; shifted_output.bed</span><br/><div><br/></div></li><li>Make coverage and bigwig files for visualization using BEDTools-Version-2.16.2 and UCSC tools (Kent <em>et al.</em>, 2010).<br/><br/><span style="font-family:Courier New;">&gt; genomeCoverageBed -g igenome/UCSC_mm10/genome_size.tab.txt -i shifted_output.bed -bg output_cov.bed</span><br/><span style="font-family:Courier New;">&gt; wigToBigWig output_cov.bed igenome/UCSC_mm10/genome_size.tab.txt output.bw</span><br/></li></ol><p class="pptt" id="biaoti33956">### Notes</p><ol class="npwtwok"><li>We recommend optimizing the steps of Percoll separation (Step 26), flow cytometry sorting (Step 29), and tagmentation (Steps 36 and 39) for each cell type. We provide examples of three specific cell types throughout the protocol for reference in optimizing. </li><li>For optimizing Steps 26 and 29, the goal of centrifuging the cell suspension through a Percoll solution is to separate cell bodies from debris by density. Removing debris can make sorting by flow cytometry more efficient, yielding a larger number and higher quality nuclei. This is especially true if using one of the Nelson Lab enhancer trap lines (Shima <em>et al.</em>, 2016) where mCitrine is expressed at high levels including in neuronal projections. MCitrine<sup>+</sup> debris from these cells is often too bright to remove by simple intensity gaiting on the flow cytometer. In particular, small cells such as cerebellar granule cells pose a significant challenge to distinguish from debris. While projections from the cells of interest may not be problematic for assays such as RNA-seq, it is a problem for tagmentation where debris, which does not contain genomic DNA, can reduce Tn5 efficiency. You may optimize around this issue by reducing contaminating debris in the final sample and/or titrating the amount of Tn5 to compensate for debris.<br/></li></ol><p class="pptt" id="biaoti33957">### Recipes</p><ol class="npwtwok"><li>TTX 0.1 mM (1,000x)<br/><ol class="npwthree"><li>Dilute TTX in molecular grade H<sub>2</sub>O</li><li>Aliquot and store at -20 Â°C for up to six months</li></ol></li><li>Ketamine 500 mM (1,000x)<br/><ol class="npwthree"><li>Dilute in molecular grade H<sub>2</sub>O</li><li>Store at 4 Â°C for up to six months</li></ol></li><li>1% (100x) Digitonin (supplied at 2% in DMSO)<br/><ol class="npwthree"><li>Dilute 1:1 with molecular grade H<sub>2</sub>O</li><li>Aliquot and store at -20 Â°C for up to 6 months or &lt; 5 freeze thaw cycles</li></ol></li><li>10% (100x) Tween-20 <br/><ol class="npwthree"><li>Diluted in molecular grade H<sub>2</sub>O</li><li>Store at 4 Â°C for up to six months</li></ol></li><li>1x Low TE buffer (1x)<br/>10 mM Tris-HCl (pH 8.0)<br/>0.1 mM EDTA</li><li>100x SYBR Green <br/><ol class="npwthree"><li>Dilute in DMSO</li><li>Aliquot and store at -20 Â°C for up to six months</li></ol></li><li>2x TD buffer<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:250px;"><tbody><tr><td><br/></td><td>Final [ ]<br/></td></tr><tr><td>1 M Tris-HCl pH 7.6<br/></td><td>20 mM<br/></td></tr><tr><td>1 M MgCl<sub>2</sub><br/></td><td>10 mM<br/></td></tr><tr><td>Dimethyl Formamide*<br/></td><td>20%</td></tr></tbody></table>*Adjust the pH to 7.6 with 100% acetic acid<br/>Aliquot and store at -20 Â°C for up to six months</li><li>Lysis buffer<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:250px;"><tbody><tr><td><br/></td><td>Final [ ]<br/></td></tr><tr><td>Tris-HCl, pH 7.4<br/></td><td>10 mM<br/></td></tr><tr><td>NaCl <br/></td><td>10 mM<br/></td></tr><tr><td>MgCl<sub>2</sub><br/></td><td>3 mM<br/></td></tr><tr><td>IGEPAL CA-630<br/></td><td>0.1%<br/></td></tr></tbody></table>Aliquot and store at 4 Â°C for up to six months</li><li>Cutting Solution (CS)<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:250px;"><tbody><tr><td><br/></td><td>Final [mM]<br/></td></tr><tr><td>Choline chloride<br/></td><td>92<br/></td></tr><tr><td>KCl<br/></td><td>2.5<br/></td></tr><tr><td>NaH<sub>2</sub>PO<sub>4</sub>Â·H<sub>2</sub>O<br/></td><td>1.2<br/></td></tr><tr><td>NaHCO<sub>3</sub><br/></td><td>30<br/></td></tr><tr><td>HEPES<br/></td><td>20<br/></td></tr><tr><td>Na ascorbate<br/></td><td>5<br/></td></tr><tr><td>Thiourea<br/></td><td>2<br/></td></tr><tr><td>Na pyruvate<br/></td><td>3<br/></td></tr><tr><td>N-acetyl-L-cystein<br/></td><td>12<br/></td></tr><tr><td>MgSO<sub>4</sub>Â·7H<sub>2</sub>O<br/></td><td>10<br/></td></tr><tr><td>CaCl<sub>2</sub>Â·2H<sub>2</sub>O<br/></td><td><span style="font-family:'&quot;font-size:small;"></span>0.5</td></tr></tbody></table></li><li>Modified Artificial Cerebral Spinal Fluid (mACSF)<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:250px;"><tbody><tr><td><br/></td><td>Final [mM]<br/></td></tr><tr><td>NaCl <br/></td><td>92</td></tr><tr><td>KCl<br/></td><td>2.5</td></tr><tr><td>NaH<sub>2</sub>PO<sub>4</sub>Â·H<sub>2</sub>O<br/></td><td>1.2</td></tr><tr><td>NaHCO<sub>3</sub><br/></td><td>30</td></tr><tr><td>HEPES<br/></td><td>20</td></tr><tr><td>Na ascorbate<br/></td><td>5</td></tr><tr><td>Thiourea<br/></td><td>2</td></tr><tr><td>Na pyruvate<br/></td><td>3</td></tr><tr><td>N-acetyl-L-cystein<br/></td><td>12</td></tr><tr><td>MgSO<sub>4</sub>Â·7H<sub>2</sub>O<br/></td><td>10</td></tr><tr><td>CaCl<sub>2</sub>Â·2H<sub>2</sub>O<br/></td><td><span style="font-family:'&quot;font-size:small;"></span>0.5<br/></td></tr></tbody></table><ol class="npwthree"><li>After dissolving all solutes in CS and mACSF, oxygenate with stirring for ~30 min at room temp to stabilize the working pH</li><li>Bring the pH of CS to 7.3-7.4 using NMDG<ol class="npwfour"><li>CS should have very little sodium to enhance neuroprotection</li><li>Bring the pH of mACSF to 7.3-7.4 using NaOH</li></ol></li><li>For both CS and mACSF add enough trehalose or glucose (typically 2-6 g for 2 L of solution) to bring the osmolarity to 300-310 mOsm as measured on an osmometer<br/><em>Note: Trehalose added here is separate and does not need to be considered in step 2 of the procedure.</em><br/></li><li>If preparing more than 24 h in advance of use, filter sterilize and store at 4 Â°C</li></ol></li></ol><p class="pptt" id="biaoti33958">### Acknowledgments</p><p class="ppzz">We thank the National Eye Institute (EY022360), the National Institute of Mental Health (MH105949) and the National Institute of Neurological Disorders and Stroke (NS075007) for funding support.<br/>  This protocol was adapted from previous work (Hempel <em>et al.</em>, 2007; Buenrostro <em>et al.</em>, 2013; Corces <em>et al.</em>, 2017).</p><p class="pptt" id="biaoti33959">### Competing interests</p><p class="ppzz">The authors have no competing interests to report.</p><p class="pptt" id="biaoti33960">### Ethics</p><p class="ppzz">Animal experimentation: All experiments were conducted in accordance with the requirements of the Institutional Animal Care and Use Committees at Brandeis University (protocol #17001).</p><p class="pptt" id="biaoti33961">### References</p><ol class="npwtwok"><li>Andrews, S. (2010). <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc" target="_blank">FastQC: a quality control tool for high throughput sequence data.</a> Available online.</li><li>Broad Institute. (2019). <a href="http://broadinstitute.github.io/picard/" target="_blank">Picard Toolkit.</a> Available on line, GitHub; Broad Institute</li><li>Buenrostro, J. D., Giresi, P. G., Zaba, L. C., Chang, H. Y. and Greenleaf, W. J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24097267" target="_blank">Transposition of native chromatin for fast and sensitive epigenomic profiling of open chromatin, DNA-binding proteins and nucleosome position.</a> <em>Nat Methods</em> 10(12): 1213-1218.</li><li>Corces, M. R., Trevino, A. E., Hamilton, E. G., Greenside, P. G., Sinnott-Armstrong, N. A., Vesuna, S., Satpathy, A. T., Rubin, A. J., Montine, K. S., Wu, B., Kathiria, A., Cho, S. W., Mumbach, M. R., Carter, A. C., Kasowski, M., Orloff, L. A., Risca, V. I., Kundaje, A., Khavari, P. A., Montine, T. J., Greenleaf, W. J. and Chang, H. Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28846090" target="_blank">An improved ATAC-seq protocol reduces background and enables interrogation of frozen tissues.</a> <em>Nat Methods</em> 14(10): 959-962.</li><li>Gage, G. J., Kipke, D. R., Shain, W. (2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3476408/" target="_blank">Whole animal perfusion fixation for rodents.</a> <em>J Vis Exp</em> 65: 3564.</li><li>Hempel, C. M., Sugino, K. and Nelson, S. B. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18007629" target="_blank">A manual method for the purification of fluorescently labeled neurons from the mammalian brain.</a> <em>Nat Protoc</em> 2(11): 2924-2929.</li><li>Kent, W. J., Zweig, A. S., Barber, G., Hinrichs, A. S. and Karolchik, D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20639541" target="_blank">BigWig and BigBed: enabling browsing of large distributed datasets.</a> <em>Bioinformatics</em> 26(17): 2204-2207.</li><li>Langmead, B. and Salzberg, S. L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" target="_blank">Fast gapped-read alignment with Bowtie 2.</a> <em>Nat Methods</em> 9(4): 357-359.</li><li>Lee, H. J., Yoon, Y. S. and Lee, S. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29907758" target="_blank">Mechanism of neuroprotection by trehalose: controversy surrounding autophagy induction.</a> <em>Cell Death Dis</em> 9(7): 712.</li><li>Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R. and Genome Project Data Processing, S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" target="_blank">The sequence alignment/map format and SAMtools.</a> <em>Bioinformatics</em> 25(16): 2078-2079.</li><li>Martin M. (2015). <a href="https://journal.embnet.org/index.php/embnetjournal/article/view/200" target="_blank">Cutadapt removes adapter sequences from high-throughput sequencing reads.</a> <em>EMBnet Journal</em> 17(1): 10-12.</li><li>Picelli, S., Bjorklund, A. K., Reinius, B., Sagasser, S., Winberg, G. and Sandberg, R. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25079858" target="_blank">Tn5 transposase and tagmentation procedures for massively scaled sequencing projects.</a> <em>Genome Res</em> 24(12): 2033-2040.</li><li>Papouin, T. and Haydon, P. G. (2018). <a href="https://bio-protocol.org/e2699" target="_blank">Obtaining acute brain slices.</a> <em>Bio-protocol</em> 8(2): e2699.</li><li>Quinlan, A. R. and Hall, I. M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" target="_blank">BEDTools: a flexible suite of utilities for comparing genomic features.</a> <em>Bioinformatics</em> 26(6): 841-842.</li><li>Saxena, A., Wagatsuma, A., Noro, Y., Kuji, T., Asaka-Oba, A., Watahiki, A., Gurnot, C., Fagiolini, M., Hensch, T. K. and Carninci, P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22668417" target="_blank">Trehalose-enhanced isolation of neuronal sub-types from adult mouse brain.</a> <em>Biotechniques</em> 52(6): 381-385.</li><li>Shima, Y., Sugino, K., Hempel, C. M., Shima, M., Taneja, P., Bullis, J. B., Mehta, S., Lois, C. and Nelson, S. B. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26999799" target="_blank">A Mammalian enhancer trap resource for discovering and manipulating neuronal cell types.</a> <em>Elife</em> 5: e13503.</li><li>Sugino, K., Clark, E., Schulmann, A., Shima, Y., Wang, L., Hunt, D. L., Hooks, B. M., Trankner, D., Chandrashekar, J., Picard, S., Lemire, A. L., Spruston, N., Hantman, A. W. and Nelson, S. B. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30977723" target="_blank">Mapping the transcriptional diversity of genetically and anatomically defined cell populations in the mouse brain.</a> <em>Elife</em> 8: e38619.</li><li>Ting, J. T., Daigle, T. L., Chen, Q. and Feng, G. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25023312" target="_blank">Acute brain slice methods for adult and aging animals: application of targeted patch clamp analysis and optogenetics.</a> <em>Methods Mol Biol</em> 1183: 221-242.<br/></li></ol>