<body>
            <h1>Experimental Setup for a Diffusion Bioreactor to Isolate Unculturable Soil Bacteria</h1>
            <p><strong>Authors:</strong> Dhiraj Kumar Chaudhary, Jaisoo  Kim</p>
            <h2>Abstract</h2>
            <p>Unculturable bacteria are those bacteria which proliferate in their native habitat but unable to grow or thrive in the normal laboratory media and conditions. The molecular techniques have revealed the significance of these uncultured bacteria in terms of their functional diversity and potential to produce secondary metabolites. To achieve these benefits, scientists have attempted to isolate and cultivate unculturable bacteria in the laboratory using transwell plates, optical tweezers, laser microdissection, microbioreactors, and diffusions bioreactors. However, these techniques are still inadequate to resolve the difficulties of cultivating unculturable bacteria. Therefore, it is essential to develop new cultivation method that enables growth of diverse range of bacteria in the laboratory conditions. Diffusion bioreactor is a membrane bound chamber which allows microbes to proliferate in their native environment by providing the excess to naturally occurring nutrients and signaling compounds. This paper presents efficient and reliable protocol to construct a diffusion bioreactor and its utilization to isolate and cultivate unculturable soil bacteria in laboratory.</p>
            <p><strong>Keywords:</strong> Diffusion bioreactor, Unculturable bacteria, Microbial ecology, Cultivation technique, Soil</p>
            <h2>Background</h2><p>The metagenomic analysis has shown an overwhelming number of bacterial populations which are beyond the cultivation in the normal laboratory conditions (Gans <em>et al.</em>, 2005; Pham and Kim, 2012). These bacteria which are not-yet cultured and not able to grow on synthetic media are known as unculturable bacteria. The âunculturableâ does not imply that these bacteria can never be cultured, but instead, it indicates the absence of information on their natural habitat and growth requirements (Stewart, 2012; Chaudhary <em>et al.</em>, 2019). The molecular analysis has reported that uncultured bacteria possess novel biological and chemical properties, produce secondary metabolites, and comprise a diverse functional role (Ferrari <em>et al.</em>, 2005; Vartoukian <em>et al.</em>, 2010; van Dorst <em>et al.</em>, 2016). The secondary metabolites such as novel antibiotics, enzymes, toxins, alkaloids have potential use for human welfares. Due to these beneficial aspects, several studies have attempted to explore and cultivate the hidden uncultured bacteria from the environments (Sait <em>et al.</em>, 2002; Ferrari <em>et al.</em>, 2005; Davis <em>et al.</em>, 2005; George <em>et al.</em>, 2011; van Dorst <em>et al.</em>, 2016).<br/><br/>There are several difficulties in cultivating unculturable species in the laboratory setups. The unculturable bacteria fail to replicate in laboratory conditions due to lack of natural growth environment and inappropriate incubation time, temperature, and nutrients (Stewart, 2012). Next, traditional approach of cultivation is mainly based on nutrient-rich media which favors only rapid growers (Chaudhary <em>et al.</em>, 2019). In the natural habitat, microbe-microbe interaction and their metabolism are crucial aspects for survival and growth. These microbial behaviors depend on several physiological parameters which include pH, osmotic conditions, temperature, and nutrients (Sait <em>et al.</em>, 2002). Therefore, without addressing above-mentioned limitations, it is not possible to cultivate missing bacterial species. Strategies such as simulating natural habitat, modifying nutrients composition, optimizing growth conditions, co-culturing with helper agents, and prolonging the incubation time during cultivation of uncultured domains have reported good results (Kaberlein <em>et al.</em>, 2002; Joseph <em>et al.</em>, 2003; Pham <em>et al.</em>, 2012; Stewart, 2012; Chaudhary <em>et al.</em>, 2019). Some recent advancement has been made in cultivation techniques using diffusion chambers, transwell plates, microbioreactors, optical tweezers and laser microdissection (Pham <em>et al.</em>, 2012). However, the available techniques are still inadequate to address the limitation of cultivation of unculturable bacteria. Any attempt to develop novel cultivation technique for unculturable bacteria accounts a great significance in this field.<br/><br/>Enrichment cultivation utilizing diffusion bioreactor is a promising strategy to cultivate previously uncultured microbes. It helps to enrich microbial species by creating natural environment and allows the microbes with the access to growth components and signaling compounds (Bollmann <em>et al.</em>, 2007). In addition, it also provides environment for bacterial interaction (Stewart, 2012; Chaudhary <em>et al.</em>, 2019). Diffusion bioreactors are also suitable to cultivate various microbial species from varieties of samples such as soil, sediments, and water samples (Kakumanu <em>et al.</em>, 2012; Chaudhary <em>et al.</em>, 2019). This paper describes a novel cultivation technique for uncultured soil bacteria using newly designed diffusion bioreactor. Diffusion bioreactor permits the passage of essential elements from the soil into the inner chamber mimicking the natural environments (Kaberlein <em>et al.</em>, 2002; Bollmann <em>et al.</em>, 2007; Kakumanu and Williams, 2012). Here, we present an effective protocol for experimental setup using diffusion bioreactors to isolate and cultivate the unculturable soil bacteria.</p>
        </body>

<p class="pptt" id="biaoti29741">## Materials and Reagents</p><ol class="npwtwok"><li>2- mm mesh sieve (Fisher Scientific, catalog number: 04-881-10G)</li><li>Whatman filter paper, No. 1 (GE Healthcare, catalog number: 1001-150)</li><li>Mixed cellulose ester filter membrane, 0.22 Âµm (Merck Millipore, catalog number: GSWP04700)</li><li>Polycarbonate membrane, 0.45 Âµm (GE Healthcare, catalog number: RPN303B)</li><li>2 L Conical flasks (DURAN, catalog number: 212176303)</li><li>Petri dish (SPL Life Science, catalog number: 10090)</li><li>15 ml sterile tube (SPL Life Science, catalog number: 50015)</li><li>Inoculating loop (SPL Life Science, catalog number: 90010)</li><li>Plastic containers with sizes of 2 L (for inner chamber) and 4 L (for outer chamber) (Berry Plastic, catalog number: T51368CP)</li><li>R2A broth (KisanBio, catalog number: MB-R2230)</li><li>Nutrient broth (NB, KisanBio, catalog number: MB-N1037)</li><li>Tryptic Soy Broth (TSB, KisanBio, catalog number: MB-T1053)</li><li>Luria-Bertani broth (LB, Oxoid, catalog number: CM0996)</li><li>Agar (Bio Basic, catalog number: MD01)</li><li>Gelatin (Sigma-Aldrich, catalog number: G2500)</li><li>Cycloheximide (Sigma-Aldrich, catalog number: C7698)</li><li>DNA isolation kit (Instagene Matrix, Bio-Rad, catalog number: 7326030)</li><li>Multiscreen PCR<sub>384</sub> Filter Plate kit (Millipore, catalog number: S384PCR10)</li><li>BigDye Terminator Cycle Sequencing Kit (Applied Biosystems, catalog number: 4337455)</li><li>FastDNA SPIN kit for soil (MP Biomedicals, catalog number: 116560-200)</li><li>QIAQuick PCR Purification Kit (Qiagen, catalog number: 28104)</li><li>Beef extract</li><li>Casein</li><li>Enzymatic digest of soya bean </li><li>Glucose</li><li>Peptone</li><li>Sodium pyruvate</li><li>Soluble starch</li><li>Yeast extract</li><li>NaCl</li><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O</li><li>CaCo<sub>3</sub> </li><li>K<sub>2</sub>HPO<sub>4</sub> </li><li>KNO<sub>3</sub> </li><li>FeCl<sub>2</sub>Â·7H<sub>2</sub>O</li><li>(NH<sub>2</sub>)<sub>2</sub>HPO<sub>4</sub> </li><li>HCl</li><li>CoCl<sub>2</sub>Â·6H<sub>2</sub>O </li><li>CuCl<sub>2</sub>Â·2H<sub>2</sub>O </li><li>FeCl<sub>2</sub>Â·4H<sub>2</sub>O </li><li>NaBO<sub>3</sub> </li><li>MnCl<sub>2</sub>Â·4H<sub>2</sub>O </li><li>Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O </li><li>NiCl<sub>2</sub>Â·6H<sub>2</sub>O </li><li>ZnCl<sub>2</sub> <br/></li><li>NaOH</li><li>Na<sub>2</sub>SeO<sub>3</sub>Â·5H<sub>2</sub>O </li><li>Na<sub>2</sub>WO<sub>4</sub>Â·2H<sub>2</sub>O </li><li>Soil extract (SE) (see Recipes)</li><li>Trace element SL-10 (see Recipes)</li><li>Selenite tugstate (see Recipes)<br/></li></ol><p class="pptt" id="biaoti29742">### Equipment</p><ol class="npwtwok"><li>Orbital shaker (FINEPCR, model: SH30)</li><li>Vortex (Scientific Industries, model: 2E-179213)</li><li>Multi-room incubator (Han Baek Scientific, model: HB-103-4)</li><li>Twin cooling system Refrigerator (4 Â°C to -18 Â°C) (Samsung Zipel, model: C23442BS300886)</li><li>Centrifuge (Hanil Science Industrial, model: UNION 32R)</li><li>Magnetic stirrer (Global Lab, model: GLMS-G)<br/></li><li>MaestroNano spectrophotometer (MaestroGen, model: MN-913)</li><li>MultiGene OptiMax Thermal Cycler (Labnet International, catalog: TC9600-G-230V)</li><li>3730xl DNA analyzer (Applied Biosystems, catalog: A41046)<br/></li></ol><p class="pptt" id="biaoti29743">### Software</p><ol class="npwtwok"><li>Microsoft Office Excel 2013 and OriginPro 8.5</li><li>MEGA 6.0.6 (For phylogenetic analysis)</li><li>Lasergene SeqMan Pro, version 7.1.0 (44.1) (DNASTAR Inc.)<br/></li></ol><p class="pptt" id="biaoti29744">### Procedure</p><ol class="npwone"><li>Soil sampling<ol class="npwtwo"><li>Collect soil samples (4 kg per experimental set-up) from the uppermost layer (0-10 cm deep) in a sterile zippered bag.</li><li>Sieve the collected soil samples using a 2-mm mesh sieve.</li><li>Utilize the sieved soil for DNA extraction, cultivation, and preparation of soil extract (SE).<br/><br/></li></ol></li><li>Preparation of culture media<ol class="npwtwo"><li>Prepare R2A, TSA, LB, and NB culture media according to the instructions provided by the manufacturer (Table 1).</li><li>Prepare 50% diluted R2A by mixing equal amount of normal R2A media and distilled water (d/w) (For example 1 L normal R2A media + 1 L d/w).</li><li>Prepare R2A-SE by using SE instead of d/w. (Recipe 1).</li><li>Prepare starch-casein agar-SE (SCA-SE) and J26-SE media by using SE and the composition as mentioned in Table 1 (see Recipe 2). J26-SE media is laboratory formulated media and refer to Table 1 for its preparation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Construction of the diffusion bioreactor<ol class="npwtwo"><li>Take 2 L plastic container to prepare it as the inner chamber of the diffusion bioreactor.</li><li>Make holes (6 mm in diameter) in the wall of the inner chamber as shown in Figure 1. To make holes, use electric drill machine or screw driver (with heating) of defined size. Maintain the space of 8 mm in length between the holes.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190925/20190925212638_2206.jpg" width="120"/><br/><strong>Figure 1. Design of the diffusion bioreactor.</strong> A. Plastic container (2 L in size) perforated throughout with holes; B. Inner chamber (2L in size) glued with polycarbonate membrane; C. Schematic representation of the diffusion bioreactor; D. Overall experimental setup using the newly developed diffusion bioreactor (Figure adapted from Chaudhary <em>et al.</em>, 2019).<br/><br/></li><li>Stick polycarbonate membrane (0.45 Âµm pore size) on the outer side of the inner chamber.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Experimental setup with constructed diffusion bioreactor<ol class="npwtwo"><li>Place the inner chamber inside the outer chamber (4 L plastic container).</li><li>Fill the gap between the walls of the two chambers with soil.</li><li>Mix 3 g of soil sample with 300 ml of respective culture medium properly and then add this mixture in the inner chamber. Also, place a magnetic stir inside the inner chamber. (Figures 1 and 2).<br/><br/><img alt="" class="layerphoto" height="144" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190925/20190925212711_4029.jpg" width="100"/><br/><strong>Figure 2. Experimental setup for diffusion bioreactor</strong> (Chaudhary <em>et al.</em>, 2019)<br/><br/></li><li>Close and seal tightly the lid of the inner chamber with the sealing tape.</li><li>Keep stirring (at 60-80 rpm) the diffusion bioreactors on the magnetic stirrer and incubate at room temperature for 4 weeks.</li><li>Set up a similar experiment for each of the assayed culture media (R2A, TSB, LB, NB, 50% R2A, R2A-SC, J26-SE, and SCA-SE).</li><li>After incubation, perform the sampling by taking 1 ml of culture broth from diffusion bioreactor and serially dilute the sample. For serial dilution, use freshly prepared normal saline.</li><li>Spread 100 Âµl of serially diluted samples from each dilution (10<sup>-1</sup>-10<sup>-7</sup>) onto the agar plates for each respective medium. Perform each spreading in duplicates.</li><li>Incubate aerobically at 25 Â°C for 4 weeks.</li><li>After incubation, select the isolated colonies from each plate.<br/></li><li>Perform subculture repeatedly (at least twice) to obtain pure colonies.</li><li>Identify pure colonies by 16S rRNA gene sequence analysis.<br/><br/></li></ol></li><li>PCR amplification and 16S rRNA gene sequencing<ol class="npwtwo"><li>Extract genomic DNA from the pure colonies using DNA isolation kit.</li><li>Perform PCR amplification for 16S rRNA gene using the following primer set and conditions:<br/><strong>Primers</strong> (Frank <em>et al.</em>, 2008)<br/>Forward (27F), 5â-AGAGTTTGATCMTGGCTCAG-3â<br/>Reverse (1492R), 5â-TACGGYTACCTTGTTACGACTT-3â<br/><strong>Reaction mixture</strong><br/>30 Âµl reaction mixture containing 20 ng of genomic DNA<br/><strong>PCR conditions</strong><br/>95 Â°C, 2 min<br/>35 cycles: 95 Â°C, 1 min; 55 Â°C, 1 min; 72 Â°C, 1 min<br/>72 Â°C, 10 min</li><li>Purify the PCR products using a Multiscreen PCR384 Filter Plate kit.</li><li>Sequence the purified PCR products using a BigDye Terminator Cycle Sequencing Kit v.3.1 (Applied Biosystems) in a 3730XL DNA analyzer (Applied Biosystems).<br/>Use the following primer set for sequencing reactions:<br/>Forward (518F), 5â²-CCAGCAGCCGCGGTAATACG-3â² <br/>Reverse (800R), 5â²-TACCAGGGTATCTAATCC-3â²</li><li>Compile short sequence of 16S rRNA gene using SeqMan Pro (DNASTAR) software.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Phylogenetic analysis<ol class="npwtwo"><li>Compare the sequenced 16S rRNA genes of all the isolates to find the closest phylogenetic neighbors using BLAST (<a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>) and the EZBioCloud server (<a href="https://www.ezbiocloud.net/" target="_blank">https://www.ezbiocloud.net</a>) (Yoon <em>et al.</em>, 2017).</li><li>Retrieve the 16S rRNA gene sequences of closely related species from the GenBank.</li><li>Construct the phylogenetic tree (Figure 3) in Mega 6.0.6 software using maximum-likelihood algorithm (Felsenstein, 1981; Tamura <em>et al.</em>, 2013).<br/><br/><img alt="" class="layerphoto" height="141" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190925/20190925212824_8849.jpg" width="100"/><br/><strong>Figure 3. Phylogenetic tree.</strong> The phylogenetic tree was constructed using the maximum likelihood algorithm based on nearly complete 16S rRNA gene sequences. It shows the taxonomic affiliations of previously uncultured bacteria using the newly developed diffusion bioreactor with representatives of different established phylogenetic groups (Figure adapted from Chaudhary <em>et al.</em>, 2019).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Extraction of DNA from soil<ol class="npwtwo"><li>Extract DNA from soil samples using FastDNA SPIN kit (MP Biomedicals) following the manufacturerâs instructions.</li><li>Assess the quality of DNA with 1.5% agarose gel electrophoresis.</li><li>Determine the purity and concentration of extracted DNA using MaestroNano spectrophotometer (MaestroGen).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>PCR amplification and pyrosequencing<ol class="npwtwo"><li>Perform PCR amplification for V1-V3 regions of 16S rRNA gene using the following primer set and PCR conditions:<br/><strong>Primers</strong> (Hur <em>et al.</em>, 2011; Okubo <em>et al.</em>, 2012)<br/>27F: 5â²-CCTATCCCCTGTGTGCCTTGGCAGTC-TCAG-AC-GAGTTTGATCMTGGCTCAG-3â²<br/>518R: 5â²-CCATCTCATCCCTGCGTGTCTCCGAC-TCAG-X-ACWTTACCGCGGCTGCTGG-3â²; where âXâ represents unique barcode for each subject.<br/><strong>PCR conditions</strong><br/>30 Â°C, 5 min<br/>30 cycles: 95 Â°C, 30 s; 72 Â°C, 30 s; 72 Â°C, 30 s<br/>72 Â°C, 10 min</li><li>Assess the PCR amplicons by 1.5% agarose gel electrophoresis.</li><li>Purify the PCR amplicons using QIAQuick PCR Purification Kit (Qiagen)</li><li>Perform the pyrosequencing of the purified PCR products using Roche/454 GS FLX Titanium platform following the manufacturerâs instructions. </li></ol></li></ol><p class="pptt" id="biaoti29745">### Data analysis</p><ol class="npwtwok"><li>Statistical analysis can be conducted using Microsoft Office Excel 2013 and OriginPro 8.5. The significant difference (p &lt; 0.05) is estimated by two-way ANOVA analysis.</li><li>The following threshold values can be used to infer novel taxa: species (99.0% â¥ x â¥96.5%), genus (96.5% &gt; x â¥ 90%), family (90% &gt; x â¥ 81.7%), order (81% &gt; x â¥ 70%), and class (70% &gt; x â¥ 65%), where âxâ represents the sequence similarity value (Yarza <em>et al.</em>, 2008; Kim <em>et al.</em>, 2014).<br/></li></ol><p class="pptt" id="biaoti29746">### Notes</p><ol class="npwtwok"><li>For comparison of the effectiveness of diffusion bioreactors, set up traditional experiment of cultivation in similar conditions as illustrated in Figure 2.</li><li>Sterilize all the components used to prepare diffusion bioreactors in 70% ethanol followed by drying under UV-chamber in a clean bench.</li><li>This diffusion bioreactor relies on the type of soil sample to recover a diverse range of bacteria and other microbes. Selective uncultured bacteria can be cultivated in this bioreactor by using soil sampled from a specific area. For example, uncultured oil-contaminated bacteria can be recovered by using oil-contaminated soil.<br/></li></ol><p class="pptt" id="biaoti29747">### Recipes</p><ol class="npwtwok"><li>Soil extract (SE)<ol class="npwthree"><li>Mix 1 kg soil in 2 L sterile d/w</li><li>Shake in an orbital shaker with 120 rpm shaking speed at room temperature overnight</li><li>Stand for 2 h for settling</li><li>Decant and collect supernatant</li><li>Filter the supernatant through a 0.22 Âµm membrane filter</li></ol></li><li>Cultivation media (Table 1)<br/><br/><strong>Table 1. Composition of media used for the cultivation</strong> (Chaudhary <em>et al.</em>, 2019)<br/><img alt="" class="layerphoto" height="558" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190925/20190925213304_1916.jpg" width="600"/><br/>*see Recipe 3; #see Recipe 4.<br/><br/><ol class="npwthree"><li>Heat to boiling until completely dissolved. </li><li>Adjust the pH to 7.2</li><li>Sterilize at 121 Â°C for 15 min; pH= 7.2 Â± 0.2 at 25 Â°C.</li></ol></li><li>Trace element SL-10<br/>HCl, 10 ml<br/>CoCl<sub>2</sub>Â·6H<sub>2</sub>O, 190 mg <br/>CuCl<sub>2</sub>Â·2H<sub>2</sub>O, 2 mg <br/>FeCl<sub>2</sub>Â·4H<sub>2</sub>O, 1.5 g <br/>NaBO<sub>3</sub>, 6 mg <br/>MnCl<sub>2</sub>Â·4H<sub>2</sub>O, 100 mg <br/>Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O, 36 mg <br/>NiCl<sub>2</sub>Â·6H<sub>2</sub>O, 24 mg <br/>ZnCl<sub>2</sub>, 70 mg <br/>Distilled water, 990 ml<br/>Filter through a 0.22 Âµm filter for sterilization<br/></li><li>Selenite tugstate<br/>NaOH, 0.5 g<br/>Na<sub>2</sub>SeO<sub>3</sub>Â·5H<sub>2</sub>O, 3 mg <br/>Na<sub>2</sub>WO<sub>4</sub>Â·2H<sub>2</sub>O, 4 mg <br/>Distilled water, 1,000 ml<br/>Filter through a 0.22 Âµm filter for sterilization<br/></li></ol><p class="pptt" id="biaoti29748">### Acknowledgments</p><p class="ppzz">This research was supported by Basic Science Research Program through the National Research Foundation of Korea (NRF-2019R1F1A1058501). This protocol was adapted from Chaudhary <em>et al.</em>, (2019) Development of a novel cultivation technique for uncultured soil bacteria. <em>Scientific Reports</em>, 9: 6666.</p><p class="pptt" id="biaoti29749">### Competing interests</p><p class="ppzz">The authors declare there are no conflicts of interest.</p><p class="pptt" id="biaoti29750">### References</p><ol class="npwtwok"><li>Bollmann, A., Lewis, K. and Epstein, S. S. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17720826" target="_blank">Incubation of environmental samples in a diffusion chamber increases the diversity of recovered isolates.</a> <em>Appl Environ Microbiol</em> 73(20): 6386-6390.</li><li>Chaudhary, D. K., Khulan, A. and Kim, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31040339" target="_blank">Development of a novel cultivation technique for uncultured soil bacteria.</a> <em>Sci Rep</em> 9(1): 6666.</li><li>Davis, K. E., Joseph, S. J. and Janssen, P. H. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15691937" target="_blank">Effects of growth medium, inoculum size, and incubation time on culturability and isolation of soil bacteria.</a> <em>Appl Environ Microbiol</em> 71(2): 826-834.</li><li>Felsenstein, J. (1981). <a href="https://www.ncbi.nlm.nih.gov/pubmed/7288891" target="_blank">Evolutionary trees from DNA sequences: a maximum likelihood approach.</a> <em>J Mol Evol</em> 17(6): 368-376.</li><li>Ferrari, B. C., Binnerup, S. J. and Gillings, M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16332866" target="_blank">Microcolony cultivation on a soil substrate membrane system selects for previously uncultured soil bacteria.</a> <em>Appl Environ Microbiol</em> 71(12): 8714-8720. </li><li>Frank, J. A., Reich, C. I., Sharma, S., Weisbaum, J.S., Wilson, B. A. and Olsen, G. J. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18296538" target="_blank">Critical evaluation of two primers commonly used for amplification of bacterial 16S rRNA gene.</a> <em>Appl Environ Microbiol</em> 74(8): 2461-2470.</li><li>Gans, J., Wolinsky, M. and Dunbar, J. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16123304" target="_blank">Computational improvements reveal great bacterial diversity and high metal toxicity in soil.</a> <em>Science</em> 309(5739): 1387-1390.</li><li>George, I. F., Hartmann, M., Liles, M. R. and Agathos, S.N. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21948845" target="_blank">Recovery of as-yet-uncultured soil <em>Acidobacteria</em> on dilute solid media.</a> <em>Appl Environ Microbiol</em> 77(22): 8184-8188.</li><li>Hur, M., Kim, Y., Song H. R., Kim, J. M., Choi Y. I. and Yi, H. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21890678" target="_blank">Effect of genetically modified poplars on soil microbial communities during the phytoremediation of waste mine tailings.</a> <em>Appl Environ Microbial</em> 77(21): 7611-7619.</li><li>Joseph, S. J., Hugenholtz, P., Sangwan, P., Osborne, C. A. and Janssen, P. H. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/14660368" target="_blank">Laboratory cultivation of widespread and previously uncultured soil bacteria.</a> <em>Appl Environ Microbiol</em> 69(12): 7210-7215.</li><li>Kaeberlein, T., Lewis, K. and Epstein, S. S. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12004133" target="_blank">Isolating âuncultivableâ microorganisms in pure culture in a simulated natural environment.</a> <em>Science</em> 296(5570): 1127-1129.</li><li>Kakumanu, M. L. and Williams, M. A. (2012). <a href="https://search.proquest.com/openview/e2a9147a02b86329bb8e64ac6651f611/1?pq-origsite=gscholar&amp;cbl=40901" target="_blank">Soil diffusion system enriches the growth of diverse and previously uncultivated bacterial taxa.</a> <em>Soil Biol Biochem</em> 76(2): 463-474.</li><li>Kim. M., Oh, H. S., Park, S. C. and Chun, J. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24505072" target="_blank">Towards a taxonomic coherence between average nucleotide identity and 16S rRNA gene sequence similarity for species demarcation of prokaryotes.</a> <em>Int J Syst Evol Microbiol</em> 64(Pt 2): 346-351.</li><li>Okubo, T., Ikeda, S., Yamashita, A., Terasawa, K. and Minamisawa, K. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22791055" target="_blank">Pyrosequencing read length of 16S rRNA gene affects phylogenetic assignment of plant-associated bacteria.</a> <em>Microbes Environ</em> 27(2): 204-208.</li><li>Pham, V. H. and Kim, J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22770837" target="_blank">Cultivation of unculturable soil bacteria.</a> <em>Trends Biotechnol</em> 30(9): 475-484.</li><li>Sait, M., Hugenholtz, P. and Janssen, P. H. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12460273" target="_blank">Cultivation of globally distributed soil bacteria from phylogenetic lineages previously only detected in cultivation-independent surveys.</a> <em>Environ Microbiol</em> 4 (11): 654â666.</li><li>Stewart, E. J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22661685" target="_blank">Growing unculturable bacteria.</a><em> J Bacteriol</em> 194(16): 4151-4160.</li><li>Tamura, K., Stecher, G., Peterson, D., Filipski, A. and Kumar, S. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24132122" target="_blank">MEGA6: Molecular evolutionary genetics analysis version 6.0.</a> <em>Mol Biol Evol</em> 30(12): 2725-2729.</li><li>van Dorst, J. M., Hince, G., Snape, I. and Ferrari, B. C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27827405" target="_blank">Novel culturing techniques select for heterotrophs and hydrocarbon degraders in a Subantarctic soil.</a> <em>Sci Rep</em> 6: 36724.</li><li>Vartoukian, S. R., Palmer, R. M. and Wade, W. G. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20487025" target="_blank">Strategies for culture of âunculturableâ bacteria.</a> <em>FEMS Microbiol Lett</em> 309(1): 1-7.</li><li>Yarza, P., Richter, M., Peplies, J., Euzeby, J., Amann, R., Schleifer, K. H., Ludwig, W., GlÃ¶ckner F. O. and RossellÃ³-MÃ³ra, R. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18692976" target="_blank">The All-Species Living Tree Project: A 16S rRNA-based phylogenetic tree of all sequenced type strains.</a> <em>Syst Appl Microbiol</em> 31(4): 241-250.</li><li>Yoon, S. H., Ha, S. M., Kwon, S., Lim, J., Kim, Y., Seo, H. and Chun, J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28005526" target="_blank">Introducing EzBioCloud: A taxonomically united database of 16S rRNA and whole genome assemblies.</a> <em>Int J Syst Evol Microbiol</em> 67(5): 1613-1617.<br/></li></ol>