<body>
            <h1>Analyzing the Functionality of Non-native Hsp70 Proteins in <em>Saccharomyces cerevisiae</em></h1>
            <p><strong>Authors:</strong> Laura E. Knighton, Lizbeth P. Saa, Adam M. Reitzel, Andrew W. Truman</p>
            <h2>Abstract</h2>
            <p>Yeast are an ideal system to study Heat Shock Protein 70 (Hsp70) function in a cellular context. This protocol was generated to analyze the function of non-native Hsp70 proteins by expressing them as the sole cytosolic Hsp70 in yeast. As an initial step, Hsp70 variants (such as Ssa1 point mutants and non-yeast versions such as <em>Nematostella vectensis</em> NvHsp70A, B and D) are cloned into an appropriate expression plasmid. Next, these plasmids are transformed into <em>ssa1-4</em>â yeast [expressing native Ssa1 from an uracil-based (<em>URA3</em>) plasmid] which are subsequently cured of the original yeast on 5-Fluroorotic Acid (5-FOA). The resulting cells can be screened for a variety of phenotypes which match to the activity of well-studied cellular pathways.</p>
            <p><strong>Keywords:</strong> Hsp70, Molecular chaperones, Nematostella, Plasmid swap</p>
            <h2>Background</h2><p>cHsp70 is a molecular chaperone that plays a role in protein folding of newly synthesized and misfolded proteins (Rosenzweig <em>et al.</em>, 2019). It also controls the activity of regulatory proteins that contribute to cell cycle progression, protein degradation, apoptosis and resistance to anticancer therapeutics. Hsp70 is highly conserved throughout nature and is essential for cell viability. Organisms can express several highly similar Hsp70 isoforms. For example, the budding yeast <em>Saccharomyces cerevisiae</em> expresses 4 cytosolic isoforms, Ssa1-4 (Lotz <em>et al.</em>, 2019). In order to characterize Hsp70 function, we can express different Hsp70 isoforms, paralogs, point mutations and truncations as the sole cytosolic Hsp70 protein in <em>S. cerevisiae</em> using a 5-FOA plasmid swap strategy (Boeke <em>et al.</em>,1987). <br/><br/>We were recently able to use this methodology to express <em>Nematostella vectensis</em> Hsp70 isoforms (NvHsp70A, B and D) in yeast and asses their functionality through phenotypic screens, growth assays and analysis of interaction partners (Waller <em>et al.</em>, 2018; Knighton <em>et al.</em>, 2019). These results contribute to a better understanding of Hsp70 isoform function and the potential basis of local adaptation in populations of <em>N. vectensis</em>. </p>
        </body>

<p class="pptt" id="biaoti36742">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (10 Î¼l, 300 Î¼l, 1000 Î¼l) (Neptune, catalog number: 2347; US Scientifica, catalog number: 1110-9700; Neptune, catalog number: 2167)</li><li>Eppendorf 1.5 ml tubes (VWR, catalog number: 87003-294)</li><li>15 ml Falcon tubes (VWR, catalog number: 525-0449)</li><li>50 ml Falcon tubes (VWR, catalog number: 525-0447)</li><li>Petri dishes (Thermo Fisher, catalog number: FB0875713)</li><li>96-well microplate (Greiner bio-one, catalog number: 655101)</li><li>pAG415GPD-ccdB vector (Addgene plasmid # 14146; <a href="http://www.addgene.org/" target="_blank">http://n2t.net/addgene</a>:14146; RRID: Addgene_14146)</li><li>Q5<sup>Â®</sup> High-Fidelity 2x Master Mix (NEB, catalog number: M0492S)</li><li><em>ssa1-4</em>â yeast strain (Jaiswal <em>et al.</em>, 2011)</li><li>Top 10 <em>E. coli</em> cells (Invitrogen, catalog number: C404003)<br/></li><li>Single-stranded carrier DNA (salmon sperm DNA, Solarbio, catalog number: D8030)</li><li>Tryptone (US Biological Life Sciences, catalog number: C16050360)</li><li>Ampicillin (Thermo Fisher, catalog number: 1159027)</li><li>Yeast extract (US Biological Life Sciences, catalog number: C16091364)</li><li>Peptone (HIMEDIA, catalog number: RM001)</li><li>Glucose (VWR Amresco Life Science, catalog number: 0188)</li><li>Agar (IBS Scientific, catalog number: IB49171)</li><li>Adenine hemisulfate salt (Acros Organics, catalog number: 163631000)</li><li>Yeast nitrogenous base (US Biological Life Sciences, catalog number: C16121501)</li><li>Drop out mix minus leucine, methionine, uracil (US Biological Life Sciences, catalog number: C15050101)</li><li>Drop out mix minus leucine (US Biological Life Sciences, catalog number: C15121094)</li><li>Lithium Acetate (LiAc) (Sinopharm Chemical Reagent, catalog number: 30109760)</li><li>Polyethylene glycol (PEG) (Sigma-Aldrich, catalog number: P3640)<br/></li><li>0.05 g uracil (Arcos organics, catalog number: XLE-1000)</li><li>5-FOA (US biological life sciences, catalog number: 16052511)</li><li>Hydroxyurea (HU) (Chem-implex IntâL INC, catalog number: 24533)</li><li>Methyl methanesulfonate MMS (Acos organics, catalog number: 156891000)</li><li>Sodium chloride (NaCl) (VWR, amrescolife science, catalog number: 0188)</li><li>Cadmium chloride (CdCl<sub>2</sub>) (Acros organics, catalog number: 42350500)</li><li>Hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>) (Sigma-Aldrich, catalog number: 216763)</li><li>Copper Chloride (CuCl<sub>2</sub>) (Sigma-Aldrich, catalog number: 222011)</li><li>2 ml of methionine (1 g/100 ml) (Arcos organics, catalog number: AC166160250) (see Recipes)</li><li>YPDA Media (1 L) (see Recipes)</li><li>Synthetic drop-out plates/media (1 L) (see Recipes)</li><li>5-FOA media (1 L) (see Recipes)</li><li>Stock solution of 100 mM CuCl<sub>2</sub> (see Recipes) </li><li>Stock solution of CdCl<sub>2</sub> (see Recipes)</li><li>Chemical plates (see Recipes)</li><li>1 M LiAc (500 ml) (see Recipes)</li><li>100 mM LiAc (500 ml) (see Recipes)<br/></li><li>50% (w/v) PEG (500 ml) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti36743">### Equipment</p><ol class="npwtwok"><li>UV radiator (Spectroline UV Crosslinker, model: XLE-1000)</li><li>30 Â°C Incubator (VWR, catalog number: 89511-422)</li><li>37 Â°C incubator (VWR, catalog number: 89511-428)</li><li>Heraeus Pico 21 Microcentrifuge (Thermo Fisher scientific, catalog number: 75002416)</li><li>Heraeus Multifuge X1 Centrifuge Series (Thermo Fisher scientific, catalog number: 75004211)</li><li>30 Â°C Incubator Shaker (New Brunswick Excella E25)</li><li>Thermomixer (Eppendorf, catalog number: 5382EJ909137)</li><li>Heat block (Thermo Scientific, catalog number: 88870000)</li><li>Thermal cycler (Bio-Rad, catalog number: 621BR26941)</li><li>Replica plater for 96-well plate 8X6 Array (Sigma-Aldrich, catalog number: R2383-1EA) </li><li>Autoclave (Steris Amsco Cnetury, catalog number: SG-120)</li><li>pH reader (RPI, catalog number: 850063)<br/></li></ol><p class="pptt" id="biaoti36744">### Procedure</p><ol class="npwone"><li>Cloning of <em>Nematostella vectensis</em> Hsp70 isoforms into yeast expression plasmids<ol class="npwtwo"><li>There are three <em>Nematostella vectensis</em> Hsp70 cytosolic isoforms; <em>NvHsp70A</em>, <em>NvHsp70B</em>, <em>NvHsp70D</em>.</li><li>We assembled the open reading frame from each Hsp70 (A, B, D) using sequence resources available through <em>Nematostella</em> JGI genome portal (<a href="https://genome.jgi.doe.gov/portal/" target="_blank">https://genome.jgi.doe.gov/portal/</a>).</li><li>NvHsp70A, B, and D were amplified from cDNA synthesized from RNA isolated from <em>Nematostella vectensis</em> originating from Massachusetts coast. <br/></li><li>The published protocol for Q5<sup>Â®</sup> High-Fidelity 2x Master Mix (<a href="https://www.neb.com/protocols" target="_blank">https://www.neb.com/protocols</a>) was used with an annealing temperature of 60 Â°C and extension time of 1 min. </li><li>The following primers were used (Table 1):<br/><br/><strong>Table 1. Primer sequences used for cloning NvHsp70 isoforms</strong><br/><img alt="" class="layerphoto" height="208" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190925/20190925232118_7680.jpg" width="600"/><br/><br/></li><li>Infusion cloning was used to integrate PCR products into yeast expression plasmid pAG415GPD-ccdB vector</li><li>The main yeast cytosolic Hsp70, Ssa1 was used as a control.</li><li>All inserts were sequence confirmed with Sanger sequencing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>Ssa1-4</em>â yeast strain <br/><em>S. cerevisiae</em> contains four isoforms of cytosolic Hsp70 (Ssa1-4), at least one of the isoforms is required in order to remain viable. The <em>ssa1</em>â <em>ssa2</em>â <em>ssa3</em>â <em>ssa4</em>â (<em>ssa1-4</em>â) pYCPlac33-SSA1 strain was generated using the <em>loxP-kanMX-loxP</em> gene disruption cassette. In order to keep the <em>ssa1-4</em>â strain, a plasmid-borne pYCPlac33-SSA1 (<em>URA3</em>) is needed. For more strain details, please see (Jaiswal <em>et al.</em>, 2011).<br/><br/></li><li>Transformation of Hsp70 expression plasmids into <em>ssa1-4</em>â yeast cells<ol class="npwtwo"><li>Inoculate 1 colony in 10 ml of YPD (See Recipes) and incubate in a shaker overnight at 30 Â°C. </li><li>The following morning, inoculate 100 ml of YPD with the 10 ml overnight culture and incubate at 30 Â°C (shaking at 220 rpm) until an OD600<sub>nm</sub> of 0.5 is achieved. </li><li>Harvest the culture in a sterile 50 ml centrifuge tube at 3,000 <em>x g</em> for 5 min.</li><li>Pour off medium, re-suspend the cells in 25 ml of sterile H<sub>2</sub>O and centrifuge again.</li><li>Pour off the H<sub>2</sub>O, re-suspend the cells in 1.0 ml of 100 mM LiAc and transfer the suspension to a sterile 1.5 ml microfuge tube.</li><li>Pellet the cells at top speed for 5 s and remove the LiAc with a micropipette.</li><li>Re-suspend the cells to a final volume of 500 Î¼l which is about 400 Î¼l of 100 mM LiAc.</li><li>Boil a 1.0 ml sample of single stranded carrier DNA for 5 min and quickly chill in ice water.</li><li>Vortex the cell suspension and pipette 50 Î¼l samples into labeled microfuge tubes. Pellet the cells and remove the LiAc with a micropipette.</li><li>The basic âtransformation mixâ consists of the following ingredients; carefully add them in order listed:<br/><ol class="npwthree"><li>240 Î¼l of PEG (50%w/v)</li><li>36 Î¼l of 1.0 M LiAc</li><li>25 Î¼l of single stranded carrier DNA (2.0 mg/ml)</li><li>50 Î¼l of H<sub>2</sub>O and cloned NvHsp70 plasmid DNA (0.1-10 Î¼g)</li></ol></li><li>Vortex each tube vigorously until the cell pellet has been completely mixed. This usually takes about one minute.</li><li>Incubate the cell suspension for 30 min at 30 Â°C shaking at 220 rpm.</li><li>Heat shock the cell suspension for 20-25 min in a water bath at 42 Â°C.</li><li>Incubate on ice for 2 min.</li><li>Microfuge at 5,000 <em>x g</em> for 15 s and remove the transformation mix with a micropipette.</li><li>Pipette 0.2-1.0 ml of sterile H<sub>2</sub>O into each tube and re-suspend the pellet by pipetting it up and down gently.<br/><br/></li></ol></li><li>Screening the positive yeast colonies on selective media<ol class="npwtwo"><li>Warm and dry appropriate plates in a 30 Â°C incubator for at least 30 min prior to plating cells.</li><li>After transformation, spread the yeast cells on synthetic dropout media (SD, Recipe 2) supplemented with the appropriate nutrients to select for plasmids and incubate at 30 Â°C for 2 days.</li><li>Streak the colonies to fresh SD plates and incubate at 30 Â°C for at least 1 day.</li><li>Streak yeast cells from previous to SD media containing 5-fluoro-orotic acid (5-FOA, Recipe 3) to counter-select for the <em>URA3</em>-based covering vector. </li><li>If the mutated or non-native Hsp70 is insufficient to keep the cells alive as the sole isoform, the cells will not grow on plates containing 5-FOA.</li><li>At this point, cells may be re-streaked and kept on YPD media. It is not required to keep them on SD-Leu as the Hsp70 plasmid is essential for viability.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Testing functionality of Hsp70 under different treatments<ol class="npwtwo"><li>If the non-native Hsp70 is able to provide essential function as the sole Hsp70 isoform of the cell at normal conditions (30 Â°C), we can further characterize the functionality of Hsp70 by observing the cellular resistance to different stressors including heat stress (37 Â°C), DNA damage (Hydroxyurea), Oxidative stress (H<sub>2</sub>O<sub>2</sub>), NaCl (osmotic stress), UV radiation, and CdCl<sub>2</sub> and CuCl<sub>2</sub> (heavy metal exposure).</li><li><em>Ssa1-4</em>â cells expressing NvHsp70 isoforms as their sole cytosolic Hsp70 are grown to mid-logarithmic phase (OD600<sub>nm</sub> = 0.5) in 10 ml of YPDA in a 50 ml tube.</li><li>Dilute cells 10-fold serially (full concentration, 1/10 concentration, 1/100 concentration, and 1/1,000 concentration) in a clear 96-well plate.</li><li>Cells are replica plated onto solid YPDA media containing the aforementioned chemicals or normal YPDA media and then exposed to abiotic stressors including heat stress at 37 Â°C and UV radiation (150 Jm<sup>-2</sup>, 200 Jm<sup>-2</sup>).</li><li>Concentration of drugs used are as follows; HU (200 mM and 300 mM), MMS (0.04%, 0.08%), NaCl (0.8 M,1 M), CdCl<sub>2</sub> (30 mM, 45 mM), CuCl<sub>2</sub> (3 mM, 4 mM, 5 mM), H<sub>2</sub>O<sub>2</sub> (0.8 mM, 1 mM).</li><li>After cells have dried, all plates (except for the 37 Â°C plate) are kept upside-down in a 30 Â°C incubator for 3 days. </li><li>A control plate of cells replica plates on normal YPDA (kept at 30 Â°C) should be used to compare the ânormal growth rateâ of the non-native isoforms in yeast to that on the chemical plates, see Figure 1. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20190925/20190925232346_9037.jpg" width="339"/><br/><strong>Figure 1. Expressing NvHsp70 in yeast</strong> (Waller <em>et al.</em>, 2018). NvHsp70 isoforms provide essential Hsp70 function in yeast. <em>Ssa1â4</em>â cells were transformed with control plasmid pAG415-ccDB, Ssa1-expressing, or NvHSP70 isoform-expressing plasmids and then serially diluted onto media lacking leucine or containing 5-FOA. Growth of cells on 5-FOA demonstrates the ability of NvHSP70 isoforms to provide essential function when expressed as the sole Hsp70 in the cell. Plates were incubated for 3 days at 30 Â°C and then photographed.</li></ol></li></ol><p class="pptt" id="biaoti36745">### Data analysis</p><p class="ppzz">Photographs of the plates should be taken after a pattern of cell growth can be observed, typically after 3 days (see Figure 1).</p><p class="pptt" id="biaoti36746">### Notes</p><p class="ppzz">Although this protocol describes the expression and analysis of Hsp70 isoforms from <em>Nematostella vectensis</em>, it can also be used to analyze Hsp70 phosphorylation site function as in (Truman <em>et al.</em>, 2012) or allow analysis of global interactions of Hsp70 using epitope tagged versions as in (Truman <em>et al.</em>, 2015a and 2015b; Knighton <em>et al.</em>, 2019)</p><p class="pptt" id="biaoti36747">### Recipes</p><ol class="npwtwok"><li>50% (w/v) PEG (500 ml) <br/>Dissolve 250g of polyethylene glycol in 300 ml of ddH<sub>2</sub>O<br/>Add ddH<sub>2</sub>O until final volume is 500 ml<br/>Autoclave at 121.0 Â°C on 1-hour liquid cycle </li><li>1 M LiAc (500 ml) <br/>Dissolve 51 g of Lithium Acetate in 300 ml of ddH<sub>2</sub>O<br/>Add ddH<sub>2</sub>O until final volume is 500 ml<br/>pH to 7.5<br/>Autoclave at 121.0 Â°C on 1-hour liquid cycle </li><li>100 mM LiAc (500 ml) <br/>Dilute 1 M LiAc by 10x<br/>Autoclave at 121.0 Â°C on 1-hour liquid cycle </li><li>YPDA Media (1 L)<br/>800 ml ddH<sub>2</sub>O<br/>10 g Yeast Extract<br/>20 g Peptone<br/>20 g glucose<br/>40 mg of Adenine<br/>Add 20 g of Agar (for plates only)<br/>Add ddH<sub>2</sub>O to make the volume up to 1 L<br/>Pour into appropriate size bottles<br/>Autoclave at 121.0 Â°C on 1-hour liquid cycle <br/>Let cool and dry overnight at room temperature </li><li>Synthetic drop-out plates/media (1 L)<br/>800 ml ddH<sub>2</sub>O<br/>6.7 g YNB<br/>20 g glucose<br/>1.62 g Dropout mix -leu<br/>0.05 g of adenine <br/>Adjust the pH to 6.0 with sodium hydroxide (if lower than 6)<br/>Add 20 g of Agar (for plates only)<br/>Add RO water to get to 1 L<br/>Pour into appropriate size bottles<br/>Autoclave at 121.0 Â°C on 1-hour liquid cycle <br/>Let cool and dry overnight at room temperature </li><li>5-FOA media (1 L)<br/>6.7 g YNB<br/>20 g glucose<br/>1.47 g of -ura, -leu, -met dropout<br/>0.05 g uracil <br/>2 ml of methionine<br/>Add 20 g agar and make up to 1 L<br/>Autoclave at 121.0 Â°C on 1-hour liquid cycle <br/>After autoclaving, let it cool slightly while stirring and then add 1 g of FOA<br/>Pour plates <br/>Let cool and dry overnight at room temperature </li><li>Stock solution of 100 mM CuCl<sub>2</sub> <br/>Measure 0.166 g of CuCl<sub>2</sub> and mix with 15 ml of water</li><li>Stock solution of CdCl<sub>2</sub> <br/>Measure 1.67 g of CdCl<sub>2</sub> and mix with 15 ml of water</li><li>Chemical plates <br/><strong><em>Hydroxyurea (HU)</em></strong> <ol class="npwthree"><li>Measure 0.76 g of HU and mix gently with 50 ml of liquid YPDA (cool to the touch) and pour into two Petri dishes.</li><li>Let cool and dry overnight at room temperature<br/><br/></li></ol><strong><em>MMS</em></strong> <ol class="npwthree"><li>Measure 0.02 ml of MMS and mix gently with 50 ml of liquid YPDA (cool to the touch) and pour into two Petri dishes to make 0.04% MMS</li><li>Measure 0.04 ml of MMS and mix gently with 50 ml of liquid YPDA (cool to the touch) and pour into two Petri dishes to make 0.08% MMS</li><li>Let cool and dry overnight at room temperature</li></ol><br/><strong><em>CuCl</em></strong><sub><strong><em>2</em></strong></sub> <ol class="npwthree"><li>From 100 mM CuCl<sub>2</sub> stock solution measure out 1.5 Î¼l and mix with 50 ml of liquid YPDA (cool to the touch) to make a 3.0 mM concentration and pour into two Petri dishes</li><li>From 100 mM CuCl<sub>2</sub> stock solution measure out 2 Î¼l and mix with 50 ml of liquid YPDA (cool to the touch) to make a 4.0 mM concentration and pour into two Petri dishes</li><li>From 100 mM CuCl<sub>2</sub> stock solution measure out 2.5 Î¼l and mix with 50 ml of liquid YPDA (cool to the touch) to make a 5.0 mM concentration and pour into two Petri dishes</li><li>Let cool and dry overnight at room temperature</li></ol><br/><strong><em>CdCl</em></strong><sub><strong><em>2</em></strong></sub> <ol class="npwthree"><li>From the CdCl<sub>2</sub> stock solution measure out 15 Î¼l and mix with 50 ml of liquid YPDA (cool to the touch) and pour into Petri dishes</li><li>From the CdCl<sub>2</sub> stock solution measure out 22.5 Î¼l and mix with 50 ml of liquid YPDA (cool to the touch) and pour into Petri dishes </li><li>Let cool and dry overnight at room temperature<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong><em>H</em></strong><sub><strong><em>2</em></strong></sub><strong><em>O</em></strong><sub><strong><em>2</em></strong></sub> <ol class="npwthree"><li>Add 40 Î¼l of H<sub>2</sub>O<sub>2</sub> and mix with 50 ml of YPDA (cool to the touch) to make 0.8 mM concentration and pour into two Petri dishes. And let it cool overnight</li><li>Add 50 Î¼l of H<sub>2</sub>O<sub>2</sub> and mix with 50 ml of YPDA (cool to the touch) to make 1 mM concentration and pour into two Petri dishes. And let it cool overnight</li><li>Add 75 Î¼l of H<sub>2</sub>O<sub>2</sub> and mix with 50 ml of YPDA (cool to the touch) to make 1.5 mM concentration and pour into two Petri dishes. And let it cool overnight</li><li>Let cool and dry overnight at room temperature<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong><em>NaCl</em></strong> <ol class="npwthree"><li>Measure 2.34 g NaCl and mix with 50 ml of YPDA (cool to the touch) to make a 0.8 M concentration and pour into two Petri dishes</li><li>Measure 2.92 g NaCl and mix with 50 m of YPDA (cool to the touch) to make a 1.0 M concentration and pour into two Petri dishes</li><li>Let cool and dry overnight at room temperature</li></ol></li></ol><p class="pptt" id="biaoti36748">### Acknowledgments</p><p class="ppzz">This research was supported by the National Institutes of Health NCI R15CA208773 (AWT) and National Science Foundation NSF DEB1545539 and REU #1359271 (AMR). We thank our colleagues in both the Truman and Reitzel labs who provided assistance, insight, and comments that greatly assisted both the research and the manuscript. This protocol was adapted from our previous work (Waller <em>et al.</em>, 2018).</p><p class="pptt" id="biaoti36749">### Competing interests</p><p class="ppzz">The authors of this manuscript declare no competing interests.</p><p class="pptt" id="biaoti36750">### References</p><ol class="npwtwok"><li>Boeke, J. D., Trueheart, J, Natsoulis, G. and Fink, G. R. (1987). <a href="https://www.ncbi.nlm.nih.gov/pubmed/3323810" target="_blank">5-Fluoroorotic acid as a selective agent in yeast molecular genetics.</a> <em>Method Enzymol</em> 154:164-175.</li><li>Jaiswal, H., Conz, C., Otto, H., Wolfle, T., Fitzke, E., Mayer, M. P. and Rospert, S. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21245388" target="_blank">The chaperone network connected to human ribosome-associated complex.</a> <em>Mol Cell Biol</em> 31(6): 1160-1173.</li><li>Knighton, L. E., Nitika, Waller, S. J., Strom, O., Wolfgeher, D., Reitzel, A. M. and Truman, A. W. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31233900" target="_blank">Dynamic remodeling of the interactomes of <em>Nematostella vectensis</em> Hsp70 isoforms under heat shock.</a> <em>J Proteomics</em> 206: 103416.</li><li>Lotz, S. K., Knighton, L. E., Nitika, Jones, G. W. and Truman, A. W. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31020385" target="_blank">Not quite the SSAme: unique roles for the yeast cytosolic Hsp70s.</a> <em>Curr Genet</em> 65(5):1127-1134.</li><li>Rosenzweig, R., Nillegoda, N. B., Mayer, M. P. and Bukau, B. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31253954" target="_blank">The Hsp70 chaperone network.</a> <em>Nat Rev Mol Cell Biol.</em> doi: 10.1038/s41580-019-0133-3.</li><li>Truman, A. W., Kristjansdottir, K., Wolfgeher, D., Hasin, N., Polier, S., Zhang, H., Perrett, S., Prodromou, C., Jones, G. W. and Kron, S. J. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23217712" target="_blank">CDK-dependent Hsp70 Phosphorylation controls G1 cyclin abundance and cell-cycle progression.</a> <em>Cell</em> 151(6): 1308-1318.</li><li>Truman, A. W., Kristjansdottir, K., Wolfgeher, D., Ricco, N., Mayampurath, A., Volchenboum, S. L., Clotet, J. and Kron, S. J. (2015a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25452130" target="_blank">Quantitative proteomics of the yeast Hsp70/Hsp90 interactomes during DNA damage reveal chaperone-dependent regulation of ribonucleotide reductase.</a><em> J Proteomics</em> 112: 285-300.</li><li>Truman, A. W., Kristjansdottir, K., Wolfgeher, D., Ricco, N., Mayampurath, A., Volchenboum, S. L., Clotet, J. and Kron, S. J. (2015b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26217697" target="_blank">The quantitative changes in the yeast Hsp70 and Hsp90 interactomes upon DNA damage.</a> <em>Data Brief</em> 2: 12-15.</li><li>Waller, S. J., Knighton, L. E., Crabtree, L. M., Perkins, A. L., Reitzel, A. M. and Truman, A. W. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29696514" target="_blank">Characterizing functional differences in sea anemone Hsp70 isoforms using budding yeast.</a> <em>Cell Stress Chaperones</em> 23(5): 933-941.<br/></li></ol>