<body>
            <h1>Immunofluorescence-based Determination of Centrosome Number in Tissue Samples</h1>
            <p><strong>Authors:</strong> Mengdie Wang, Gregory C. Rogers, Anne E. Cress</p>
            <h2>Abstract</h2>
            <p>Centrosome numerical abnormalities have been reported in a variety of tumors. Centrosome numbers in cancer cells display both inter-tumor and intra-tumor heterogeneity. The over production of centrosomes (centrosome amplification) is unique in cancer cells and is a promising target for therapy. Thus, a method to quantify centrosome numbers on a single cell level is needed. Here, we describe a protocol to quantify centrosome numbers in formalin fixed paraffin embedded (FFPE) tissue samples by multiplexing antibodies to define <em>bona fide</em> centrosomes and cell borders. Centrosomes in single cells are identified using high resolution immunofluorescent microscopy with Z-sectioning. This protocol is easy to perform and has been used to quantify centrosome numbers on a single cell level in a variety of human tissue samples.</p>
            <p><strong>Keywords:</strong> Centrosome, FFPE, Cancer, Tissue, High resolution immunofluorescent microscopy</p>
            <h2>Background</h2><p>The centrosome is the major microtubule-organizing center in cells and is crucial in defining mitotic spindle poles and forming cilia (Conduit <em>et al.</em>, 2015). A centrosome consists of a pair of centrioles that serve as the duplicating elements of the centrosome and organize surrounding layers of pericentriolar material (PCM), a protein matrix with microtubule-nucleating activity (Bettencourt-Dias and Glover, 2007; Lawo <em>et al.</em>, 2012). Centrosome numerical abnormalities, especially centrosome amplification, have been reported to be commonly present in a variety of human tumors and are correlated with advanced tumor grade, metastasis, recurrence, and poor survival (Bettencourt-Dias <em>et al.</em>, 2011; Chan, 2011; Godinho and Pellman, 2014; Gonczy, 2015). Furthermore, centrosome numbers display both inter-tumor and intra-tumor heterogeneity in a broad range of tumors (Chan, 2011). Thus, a method to accurately quantify centrosome numbers on a single cell level in human tissue sample is useful to identify cancer cells with centrosome amplification, to study tumor heterogeneity, and to identify patients that might benefit from therapies that target cells with centrosome amplification. <br/><br/>Human centrioles are approximately 450-500 nm in length with a diameter of 200-250 nm (Gonczy, 2012; Greenan <em>et al.</em>, 2018). Currently, centrosome quantification mainly relies on microscopy. Electron microscopy has been used as an accurate method to quantify centrosome numbers (Lingle <em>et al.</em>, 1998). Super-resolution microscopy is another powerful way to study human centrosome structures (Sonnen <em>et al.</em>, 2012). However, both electron microscopy and super-resolution microscopy require careful sample preparation and can be too laborious for quantifying centrosomes in a large number of cells and clinical tissue samples. Thus, most studies use high resolution microscopy like confocal and deconvolution microscopy. Further, the selection of centrosome markers is critical for accurate quantification. Most previous studies rely on PCM proteins like Î³-Tubulin and pericentrin as centrosome markers, and centrosome numbers were quantified by counting PCM foci. However, PCM proteins are highly dynamic during centrosome assembly and can form assemblies or fragmentation without centrioles; they are not representative as <em>bona fide</em> centrosomes (Woodruff <em>et al.</em>, 2014). Therefore, quantifying accentriolar PCM foci can lead to false positive results as centrosome amplification. On the other hand, cancer cells with centrosome amplification often undergo centrosome clustering during mitosis (Ganem <em>et al.</em>, 2009; Silkworth <em>et al.</em>, 2009). This leads multiple centrosomes in close proximity to each other and to appear as single PCM foci, which can result in false negative detection of centrosome numbers. <br/><br/>We have established a method using both centriolar protein CEP135 and PCM protein Î³-Tubulin together as centrosome markers. We multiplexed centrosome markers with cell-cell junction markers to quantify centrosomes on a single cell level by high resolution microscopy (Wang <em>et al.</em>, 2019a). We have tested different centriolar and cell-cell junction markers. Cell-cell junction proteins like E-Cadherin and Î±6 integrin are used to distinguish single cells from the neighbor cells and to identify the whole cell volume (Figure 1A). Foci with both centriolar proteins and PCM proteins are considered as centrosomes. Using this method, we have successfully performed centrosome quantification on a variety of normal and cancer tissues (Wang <em>et al.</em>, 2019a; Wang <em>et al.</em>, 2019b). Further, this method can be used to detect cells with centrosome amplification in both interphase and mitotic cells (Figure 1B). Here, we describe a detailed protocol to perform this method.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015012342_7328.jpg" width="200"/><br/><strong>Figure 1. Centrosomes in human normal and cancer tissue. </strong>A. A human normal tonsil FFPE tissue sample stained with CEP135 (red) and E-Cadherin (green). Nuclei in blue. Scale bars: 10 Î¼m. B. A human triple-negative breast cancer FFPE tissue sample stained with CEP135 (red), Î³-Tubulin (green) and E-Cadherin (green). Nuclei in blue. Scale bars: 10 Î¼m. Arrowheads indicate centrosomes.</p>
        </body>

<p class="pptt" id="biaoti29896">## Materials and Reagents</p><ol class="npwtwok"><li>Glass Coverslips 24 x 50 mm, No.1.5 (Globe scientific <em>inc</em>., catalog number: 1415-15)</li><li>Forceps </li><li>Humidified chamber (Biolegend, catalog number: SIG-31031)</li><li>Coplin Jar</li><li>Slide Staining Dish (IHC WORLD, catalog number: IW-2511)</li><li>Slide Staining Rack (IHC WORLD, catalog number: IW-2512)</li><li>Xylene (Fisher Scientific, catalog numbers: HC7001GAL, 22-110-676 and X16-4)</li><li>Isopropanol (Fisher Scientific, catalog number: A416-4)</li><li>EnVision FLEX Target Retrieval Solution, High pH (50x) (Dako Omnis, Code No: GV804)</li><li>Tris-HCl (Sigma-Aldrich, catalog number: T5941)</li><li>NaCl (Sigma-Aldrich, catalog number: S9888)</li><li>Tween 20 (Sigma-Aldrich, catalog number: BP337)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: X100)</li><li>NaN<sub>3</sub> (Fisher Scientific, catalog number: S2271-25)</li><li>Normal Goat Serum (Sigma-Aldrich, catalog number: G9023)</li><li>Bovine serum albumin (BSA) (Sigma-Aldrich, catalog number: A7030)</li><li>Tris Base (Fisher Scientific, catalog number: BP152-5)</li><li>Anti- Î³-tubulin (1:100, Sigma-Aldrich, catalog number: T5326)</li><li>Anti-Pericentrin (1:200, Abcam, catalog number: 28144)</li><li>Anti-CEP135 (1:50, Abcam, catalog number: ab75005)</li><li>Anti-Centrin-3 (1:100, Abnova, catalog number: H00001070-M01)</li><li>Anti-CP110 (1:100, proteintech<sup>TM</sup>, catalog number: 12780-I-AP)</li><li>Anti-E-Cadherin (1:100, Abcam, catalog number: ab76055 M168)</li><li>Secondary antibodies, purchased from Jackson Immunoresearch and used at 1:200</li><li>Hoechst 33342 (1:1,000; Invitrogen, catalog number: H3570)</li><li>ProLong<sup>Â®</sup> Diamond Antifade Mountant (Thermo Fisher Scientific, catalog number: P36970) </li><li>EDTA (Fisher Scientific, catalog number: BP120-500)</li><li>Washing buffer for FFPE tissue (see Recipes) </li><li>Blocking buffer for FFPE tissue (see Recipes)</li><li>Antigen retrieval buffer (see Recipes)</li><li>Tris-EDTA Buffer for Antigen Retrieval (pH 9.0) (see Recipes)</li></ol><p class="pptt" id="biaoti29897">### Equipment</p><ol class="npwtwok"><li>Hot air oven (Labnet ProBlot<sup>TM</sup> 6 Hybridization Oven)</li><li>Fume hood</li><li>Decloaking chamber or pressure cooker (Biocare medical, DC0228)</li><li>High-resolution fluorescence microscope (DeltaVision Core system (GE Healthcare Bio-Sciences, Pittsburgh, United States of America) equipped with Olympus IX71 Camera (CoolSNAP HQ2; Photometrics)<br/><em>Note: Because centrioles are small (0.5 Î¼m in length) and can localize at close proximity depending on cell cycle, high resolution images at a resolution around 200 nm are required to accurately separate each centrosome. High resolution microscopy like confocal and deconvolution microscopy can be used. We have successfully detected individual centrosomes using Deconvolution microscopy (DeltaVision) with a 60x, NA 1.42 objective. Because the positions of centrosomes vary depending on the tissue type, Z-section images through the whole cell volumes are required to detect and measure most centrioles in different z-sections within individual cells.</em> </li></ol><p class="pptt" id="biaoti29898">### Software</p><ol class="npwtwok"><li>Image acquisition software (softWoRx v1.2 software [Applied Science])</li><li>Adobe photoshop (<a href="https://www.adobe.com/products/photoshopfamily.html" target="_blank">https://www.adobe.com/products/photoshopfamily.html</a>)</li><li>ImageJ or Fiji (Schindelin <em>et al.</em>, 2012; Schneider <em>et al.,</em> 2012)</li></ol><p class="pptt" id="biaoti29899">### Procedure</p><ol class="npwone"><li>Staining FFPE tissue samples<br/><em>Notes:  <ol class="npwtwo"><li>Tissue sections should be sliced at 5 to 10 Î¼m thickness. Sectioning the tissue too thin can result in loss of cell volume and false negative detection of centrosomes. The optimal thickness should be determined according to the specific tissue samples used.</li><li>For each section, we recommend the use of an H&amp;E stained section as a guide slide to identify regions of interest.</li></ol></em> <ol class="npwtwo"><li>Tissue preparation. Timing: ~1 h. <br/>Bake tissues 1 h or overnight at 65 Â°C in a hot air oven. <br/><em>Note: This can vary according to the experiments based on tissue thickness.</em> </li><li>Deparaffinization. Timing: ~25 min.<ol class="npwthree"><li>Immerse tissue slides in a Coplin jar containing 100% fresh Xylene for 7 min at room temperature in a fume hood. </li><li>Repeat Step 2a twice, 3 times total.</li></ol></li><li>Rehydration. Timing: ~45 min.<ol class="npwthree"><li>Incubate slides in a Coplin jar containing 100% Isopropanol for 7 min at room temperature in a fume hood twice.</li><li>Incubate slides in a Coplin jar containing 70% Isopropanol for 7 min at room temperature in a fume hood. </li><li>Incubate slides in a Coplin jar containing 50% Isopropanol for 7 min at room temperature in a fume hood. </li><li>Incubate slides in a Coplin jar containing ddH<sub>2</sub>O for 7 min at room temperature twice.</li></ol><em>Note: The slides can be kept at room temperature in ddH<sub>2</sub>O overnight before proceeding.</em> </li><li>Antigen retrieval. Timing: ~1 h. <br/><em>Note: This can vary depending on the device used. </em> <ol class="npwthree"><li>Add 500 ml deionized or filtered water into a decloaking chamber or a pressure cooker.</li><li>Add 200 ml freshly prepared antigen retrieval buffer into a slide staining dish and immerse the slides into antigen retrieval buffer. </li><li>Balance the decloaking chamber using another slide staining dish with 200 ml ddH<sub>2</sub>O in it.</li><li>Close the lid of the decloaking chamber and rotate it to make sure the decloaking chamber is appropriately sealed. Turn on the decloaking chamber to incubate the slides for 20 min at its full pressure. For Decloaking chamber (Biocare Medical, DC0228), it is set to approximately 103 kPa/15 psi, and it will achieve a temperature of approximately 120 Â°C at full pressure. </li><li>Open the decloaking chamber after the pressure is completely released and the decloaking chamber is cooled down.</li><li>Transfer slides to a humidified chamber and incubate slides in washing buffer by placing a droplet of washing buffer on top of each slide to cover the tissue for 5 min at room temperature.<br/><em>Notes:<ol class="npwfour"><li>Declocking chamber at full pressure is dangerous. The pressure must be completely released before opening. </li><li>From this step on, it is very important that the slides do not dry out. Make sure to perform all the following steps in a humidified chamber.</li></ol></em> </li></ol></li><li>Blocking. Timing: ~30 min<ol class="npwthree"><li>Gently tap the slides by holding the slides horizontally with label facing the investigator and tap on the bottom edge to remove excess liquid. </li><li>Cover the tissue section with 100 Î¼l of blocking buffer. </li><li>Gently place a piece of coverslip on top of the tissue to evenly spread the liquid and incubate slides for 30 min at room temperature.<br/><em>Note: The slides can be kept in blocking buffer at 4 Â°C overnight before proceeding.</em> </li></ol></li><li>Primary antibody incubation. Timing: Overnight. <br/><em>Note: This can vary according to the experiment.</em> <ol class="npwthree"><li>Remove coverslips and gently tap the slides to remove excess liquid. </li><li>Cover the tissue section with 100 Î¼l of diluted antibodies in blocking buffer. </li><li>Gently place a piece of coverslip on top of the tissue to evenly spread the liquid and incubate slides overnight at 4 Â°C. Alternatively, tissue slides can be incubated in primary antibodies for 1 h at room temperature. <br/><em>Note: The dilutions for primary antibodies can vary depending on the targets. Optimal antibody concentrations should be determined on a case-by-case basis. We have successfully stained different centrosomal markers. The primary antibodies and dilutions can be found in Materials and Reagents. To quantify centrosome numbers per cell, investigators should choose centrosomal markers and cell border markers accordingly. To simultaneously stain different targets, investigators should choose the antibodies that are raised in different species. For example, we have successfully co-stained CEP135 (rabbit) to mark centrioles and E-Cadherin (mouse) to mark epithelial cell borders in different tissue types (Figure 1A).</em> </li></ol></li><li>Secondary antibody incubation. Timing: 45 min to 1 h.<ol class="npwthree"><li>Remove coverslips and wash slides three times in washing buffer by placing a droplet of washing buffer on top of each slide to cover the tissue, 5 min each at room temperature.</li><li>Dilute secondary antibodies in the blocking buffer. For each tissue slide, 100 Î¼l blocking buffer is recommended. </li><li>Remove coverslips and gently tap the slides to remove excess liquid.</li><li>Cover the tissue section with 100 Î¼l of diluted antibodies in blocking buffer. </li><li>Gently place a piece of coverslip on top of the tissue to evenly spread the liquid and incubate for 30 min to1 h at room temperature.<br/><em>Note: For co-staining different markers, investigators should choose the secondary antibodies conjugated to different fluorophores according to the primary antibodies used and the filter sets on the microscope available. For example, image E-Cadherin (mouse), CEP135 (rabbit) and DNA together using a microscope equipped with FITC, TRITC, and DAPI filter sets. We used Alexa-488 Goat anti-mouse for E-Cadherin (mouse), and Alexa-568 Goat anti-rabbit for CEP135 (rabbit). The dilutions for secondary antibodies can vary. We have successfully used Alexa<sup>TM</sup> conjugated, highly cross-absorbed secondary antibodies from Invitrogen used at a 1:200 dilution. Samples stained with secondary antibodies only can be used as negative controls to increase the robustness of the analysis.</em> </li></ol></li><li>Counterstain. Timing: ~30 min.<ol class="npwthree"><li>Remove coverslip and wash slides three times in washing buffer, 5 min each at room temperature.</li><li>Counterstain with Hoechst (1:1,000; 10 Âµg/ml) in PBS for 20 min at room temperature. </li><li>Wash slides twice in washing buffer by placing a droplet of washing buffer on top of each slide to cover the tissue, 5 min each at room temperature.</li></ol></li><li>Mount. Timing: 24 h.<ol class="npwthree"><li>Mount tissue sections using ProLong<sup>Â®</sup> Diamond Antifade Mountant and store in the dark at room temperature overnight for mountant to cure. <br/><em>Note: Avoid bubbles in the tissue area. You can add a drop of mountant on coverslips and invert the coverslip to cover the tissue section. </em> </li><li>Seal with nail polish after cure. <br/><em>Note: Image the sample next day after the mountant has cured, or store at -20 Â°C for future analysis. Protect the samples from light. The samples can be stored at -20 Â°C, protected from light for more than a year.</em> </li></ol></li></ol><br/></li><li>Imaging<ol class="npwtwo"><li>Use low magnification objective (<em>e.g.</em>, 10x, N.A 0.7) to find the region of interest.</li><li>Use high magnification objective (<em>e.g.</em>, 60x, N.A 1.42 oil) to image the region of interest defined from Step B1 by Z-sections. Use a cell-cell junction marker (<em>e.g.</em>, E-Cadherin) to find the whole cell volume and set the Z-section thick enough to cover the whole tissue section. We have successfully separated centrosomes at different focal planes using Z-sectioning with a 0.2 Î¼m interval on DeltaVision Core system with a 60x objective, NA 1.42. Export images to .tiff format before performing proceeding to dara analysis.<br/><em>Note: High resolution microscope systems like confocal or deconvolution microscopy can be used to image the region of interest. Note that a confocal microscope uses a pinhole to block out-of-focus light in the z-plane while deconvolution microscope uses computational methods to remove the out-of -focus light after capturing the image. Since each deconvolution method should be generated specifically for each objective on the imaging system, we only used deconvolution files generated on the DeltaVision core system using softWoRx.</em><br/><em>Representative graphs using the protocol to display the quantification of centrosome numbers in tissue was recently published (Wang et al., 2019a, Wang et al., 2019b). They can be found in figure 2 in our published paper (Wang et al., 2019b).</em> </li></ol></li></ol><p class="pptt" id="biaoti29900">### Data analysis</p><p class="ppzz">Use the Z-sectioning images taken from high-resolution microscope for quantifying centrosome number per cell.</p><ol class="npwtwok"><li>Use ImageJ or Fiji to open Z-section images using âstacksâ under âimageâ. <br/></li><li>If imported as multichannel RGB images, separate each channel using âimageâââcolorââ âsplit channelsâ.</li><li>Adjust contrast and brightness for each channel to reduce the background and increase the signal intensity. Merge all the channels after adjusting each channel.<br/><em>Note: Adjustment of brightness and contracts should be linear, and the changes should be cautiously used without losing or oversaturating signals.</em> </li><li>Generate a maximum intensity projection using command âimageââ âstacksââ âZ projectâ. Use this maximum intensity projection image to find regions with centrosomes and check if cells overlap in the Z dimension. If there are no cells or centrosomes overlapping in the image, investigators can use maximum projection to quantify centrosomes per cell. If overlap occurs, go to step 5.</li><li>Go through Z-section to quantify centrosome numbers per cell manually using the centrosome markers (<em>e.g.</em>, CEP135 foci) and cell border marker (<em>e.g.</em>, E-Cadherin). <br/><em>Note: Depending on the tissue type and sample sectioning, cells in tissue can overlap on different focal planes. If the cells overlapping with each other at different focal planes, maximum projection images from Z-section images should not be used to quantify centrosomes for individual cells.</em> </li></ol><p class="pptt" id="biaoti29901">### Notes</p><ol class="npwtwok"><li>Tissue fixation is critical for the staining results. Do not use over-fixed tissue, or re-fix tissue samples that previously were frozen. </li><li>To practice the procedure and to test specific antibodies, we recommend using human tonsil tissue because it contains many different normal cell types.</li><li>The thickness of tissue sections is critical for accurately quantifying centrosome numbers per cell accurately. Sectioning tissue too thin could result in loss of cell volume and false negative results. Too thick of the tissue samples results in cells overlapping. Depending on the tissue quality and tissue type, we recommend carefully testing the appropriate tissue section thickness to avoid loss of tissue or the complication of overlapping cells. A thickness of 5 or 10 Î¼m is a good starting point.</li><li>Keep all the reagents sterile.</li></ol><p class="pptt" id="biaoti29902">### Recipes</p><ol class="npwtwok"><li>Washing buffer for FFPE tissue <br/>0.1 M Tris-HCl<br/>0.3 M NaCl<br/>0.1% Tween 20<br/>7.7 mM NaN<sub>3</sub><br/>Adjust pH to 7.6 at 25 Â°C. <br/><strong>To make a 50 ml stock solution:</strong> <ol class="npwthree"><li>Add 5 ml of 1 M Tris-HCl, 5 ml of 5 M NaCl, 50 ul of Tween 20 and 0.025 g of NaN3 into a beaker, and ddH<sub>2</sub>O</li><li>Adjust pH to 7.6 at room temperature<br/><em>Note: Tween 20 is viscous. When adding Tween 20, pipette slowly to avoid air bubbles entering the pipette tip. One can also cut the end of the pipette tip to facilitate the procedure. Washing buffer can be stored at room temperature indefinitely. </em> </li></ol></li><li>Blocking buffer for FFPE tissue <br/>5% (v/v) Normal Goat Serum (NGS)<br/>0.1 M Tris-HCl<br/>0.15 M NaCl<br/>7.7 mM NaN<sub>3</sub><br/>Adjust pH to 7.6 at 25 Â°C<br/><strong>To make a 50 ml stock solution:</strong> <ol class="npwthree"><li>Add 5 ml of 1 M Tris-HCl, 5 ml of 5 M NaCl, 2.5 ml of NGS and 0.025 g of NaN<sub>3</sub> into beaker, and ddH<sub>2</sub>O.</li><li>Adjust pH to 7.6 at room temperature. Blocking buffer can be stored at room temperature indefinitely. Warm up to room temperature before use<br/><em>Note: NGS can be substituted with 5% (w/v) bovine serum albumin (BSA) depending on the primary antibody one is using. If the primary antibody was generated in goat, one needs to change the blocking reagent to BSA.</em> </li></ol></li><li>Antigen retrieval buffer<br/>Dilute 50x EnVision FLEX Target Retrieval Solution into ddH<sub>2</sub>O<br/><em>Note: For staining less than 24 slices (the number of slides can vary depending on the slide rack one uses), one can add 4 ml 50x EnVision FLEX Target Retrieval Solution into 200 ml of ddH<sub>2</sub>O in a Slide Canister. Unused diluted solution may be stored at 2-8 Â°C for one month. </em> </li><li>Tris-EDTA Buffer Antigen Retrieval (pH 9.0) (1x)<br/><em>Note: This buffer is an alternative antigen retrieval buffer for this protocol.</em><br/>10 mM Tris Base<br/>1 mM EDTA Solution<br/>0.05% Tween 20<br/>Adjust pH to 9.0 at 25 Â°C <br/><strong>To make 100 ml 10x stock solution:</strong><ol class="npwthree"><li>Add 1.21 g Tris Base, 0.37 g EDTA, and 0.5 ml Tween 2, and 100 ml ddH<sub>2</sub>O into a beaker</li><li>Mix well and adjust pH to 9.0 at room temperature. Store this solution at 4 Â°C</li></ol></li></ol><p class="pptt" id="biaoti29903">### Acknowledgments</p><p class="ppzz">We are grateful to National Cancer Institute (NCI) P30CA23074, which provided the research support services of the Tissue Acquisition and Molecular Analysis Service. This work was also supported by National Institutes of Health (NIH) Grants R01GM110166 and R01GM126035 to G.C.R., NIH-NCI RO1CA159406 to A.E.C., and a Tim and Diane Bowden Cancer Biology Research Fellowship to M.W. The protocol expands the details of the procedure from our previous work (PMID: 30699045) on a method of quantifying centrosomes at the single-cell level in human normal and cancer tissue.</p><p class="pptt" id="biaoti29904">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti29905">### Ethics</p><p class="ppzz">We accessed the tissue through the tissue acquisition and molecular analysis service research core (TACMASR) that was provided by the UA Cancer Center. The UA Cancer Center is an NCI designated comprehensive center and supported by P30 CA 23074 that utilizes best practices for management of the core services in accordance with NCI policy. The specimens/data are unidentifiable and obtained from a provider that is prohibited from releasing identifiers by established regulations or policies with oversight and approval from the Institutional Review Board.</p><p class="pptt" id="biaoti29906">### References</p><ol class="npwtwok"><li>Bettencourt-Dias, M. and Glover, D. M. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17505520" target="_blank">Centrosome biogenesis and function: centrosomics brings new understanding.</a> <em>Nat Rev Mol Cell Biol</em> 8(6): 451-463.</li><li>Bettencourt-Dias, M., Hildebrandt, F., Pellman, D., Woods, G. and Godinho, S. A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21680046" target="_blank">Centrosomes and cilia in human disease.</a> <em>Trends Genet</em> 27(8): 307-315.</li><li>Chan, J. Y. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22043171" target="_blank">A clinical overview of centrosome amplification in human cancers.</a> <em>Int J Biol Sci</em> 7(8): 1122-1144.</li><li>Conduit, P. T., Wainman, A. and Raff, J. W. (2015). <span id="__kindeditor_bookmark_start_18__" style="display:none;"></span><a href="http://www.ncbi.nlm.nih.gov/pubmed/26373263" target="_blank">Centrosome function and assembly in animal cells.</a> <em>Nat Rev Mol Cell Biol</em> 16(10): 611-624.</li><li>Ganem, N. J., Godinho, S. A. and Pellman, D. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19506557" target="_blank">A mechanism linking extra centrosomes to chromosomal instability.</a> <em>Nature</em> 460(7252): 278-282.</li><li>Gonczy, P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26493645" target="_blank">Centrosomes and cancer: revisiting a long-standing relationship.</a> <em>Nat Rev Cancer</em> 15(11): 639-652.</li><li>Godinho, S. A. and Pellman, D. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25047621" target="_blank">Causes and consequences of centrosome abnormalities in cancer.</a> <em>Philos Trans R Soc Lond B Biol Sci</em> 369(1650).</li><li>Gonczy, P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22691849" target="_blank">Towards a molecular architecture of centriole assembly.</a> <em>Nat Rev Mol Cell Biol</em> 13(7): 425-435.</li><li>Greenan, G. A., Keszthelyi, B., Vale, R. D. and Agard D. A. (2018). <a href="https://www.researchgate.net/publication/326860487_Insights_into_centriole_geometry_revealed_by_cryoTomography_of_doublet_and_triplet_centrioles" target="_blank">Insights into centriole geometry revealed by cryotomography of doublet and triplet centrioles.</a> <em>Elife</em> 7. doi:10.7554/elife.36851.</li><li>Lawo, S., Hasegan, M., Gupta, G. D. and Pelletier, L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23086237" target="_blank">Subdiffraction imaging of centrosomes reveals higher-order organizational features of pericentriolar material.</a> <em>Nat Cell Biol </em>14(11): 1148-1158.</li><li>Lingle, W. L., Lutz, W. H., Ingle, J. N., Maihle, N. J. and Salisbury, J. L. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9501196" target="_blank">Centrosome hypertrophy in human breast tumors: implications for genomic stability and cell polarity.</a> <em>Proc Natl Acad Sci U S A</em> 95(6): 2950-2955.<br/></li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682.</li><li>Schneider, C. A., Rasband, W. S. and Eliceiri, K. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22930834" target="_blank">NIH Image to ImageJ: 25 years of image analysis.</a> <em>Nat Methods</em> 9(7): 671-675.</li><li>Silkworth, W. T., Nardi, I. K., Scholl, L. M.and Cimini, D. (2009). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2719800/" target="_blank">Multipolar spindle pole coalescence is a major source of kinetochore mis-attachment and chromosome mis-segregation in cancer cells.</a> <em>PLoS One</em> 4(8): e6564.</li><li>Sonnen, K. F., Schermelleh, L, Leonhardt H.and Nigg , E. A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3507176/" target="_blank">3D-structured illumination microscopy provides novel insight into architecture of human centrosomes.</a> <em>Biol Open. </em>1(10): 965-976.</li><li>Wang, M., Knudsen, B. S., Nagle, R. B., Rogers, G. C. and Cress, A. E. (2019a). <a href="https://www.researchgate.net/publication/330746112_A_method_to_quantify_centrosomes_at_single-cell_level_in_human_normal_and_cancer_tissue" target="_blank">A method of quantifying centrosomes at the single-cell level in human normal and cancer tissue.</a> <em>Mol Biol Cell</em> 30(7): 811-819.<br/></li><li>Wang, M., Nagle, R. B., Knudsen, B. S., Cress, A. E. and Rogers, G. C. (2019b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31477840" target="_blank">Centrosome loss results in an unstable genome and malignant prostate tumors.</a> <em>Oncogene.</em> doi: 10.1038/s41388-019-0995-z.</li><li>Woodruff, J. B., Wueseke, O. and Hyman, A. A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25047613" target="_blank">Pericentriolar material structure and dynamics.</a> <em>Philos Trans R Soc Lond B Biol Sci</em> 369(1650).</li></ol>