<body>
            <h1>Non-aqueous Fractionation (NAF) for Metabolite Analysis in Subcellular Compartments of <em>Arabidopsis</em> Leaf Tissues</h1>
            <p><strong>Authors:</strong> David B. Medeiros, StÃ©phanie  Arrivault, Jessica Alpers, Alisdair  R. Fernie, Fayezeh  Arabi</p>
            <h2>Abstract</h2>
            <p>The accurate determination of metabolite distribution in subcellular compartments is still challenging in plant science. Various methodologies, such as fluorescence resonance energy transfer-based technology, nuclear magnetic resonance spectroscopy and protoplast fractionation allow the study of metabolite compartmentation. However, large changes in metabolite levels occur during such procedures. Therefore, the non-aqueous fractionation (NAF) technique is currently the best method for the study of <em>in-vivo</em> metabolite distribution. Our protocol presents a detailed workflow including the NAF procedure and quantification of compartment-specific markers for three subcellular compartments: ADP glucose pyrophosphorylase (AGPase) as plastidic marker, phosphoenolpyruvate carboxylase (PEPC) as cytosolic marker, and nitrate and acid invertase as vacuolar markers. </p>
            <p><strong>Keywords:</strong> <em>In-vivo</em> metabolite distribution, Non-aqueous fractionation (NAF), Compartment-specific markers</p>
            <h2>Background</h2><p>Metabolic activity and functionality require precise compartmentation of metabolism in eukaryotic cells. Despite knowing the spatial organization of metabolism within the plant cells, deciphering localization of the metabolites is still challenging due to metabolite redundancy, leakage, rapid turnover, futile cycling, and sophisticated transport systems and storage (Lunn, 2007; Klie <em>et al.</em>, 2011; Sweetlove and Fernie, 2013). Several methods have been established to investigate metabolite compartmentation, among them non-aqueous fractionation (NAF) has been widely used for decades to determine the <em>in-vivo</em> subcellular metabolite distributions in eukaryotic cells. Firstly developed in animal sciences (Elbers <em>et al.</em>, 1974), NAF was subsequently applied to plant leaf material from spinach (Gerhardt and Heldt, 1984 and 1987), bean (Sharkey and Vanderveer, 1989), maize (Weiner and Heldt, 1992) or barley (Winter <em>et al.</em>, 1993). NAF is also successfully applied to rose petals (Yamada <em>et al.</em>, 2009) and potato tubers (Farre <em>et al.</em>, 2001). More recently, NAF has also been optimized to study metabolite compartmentation in the model plant <em>Arabidopsis thaliana</em> (Fettke <em>et al.</em>, 2005; Krueger <em>et al.</em>, 2009 and 2011; Klie <em>et al.</em>, 2011; Arrivault <em>et al.</em>, 2014; Shapiguzov <em>et al.</em>, 2019). The NAF procedure prevents modification of metabolite levels (for example due to enzymatic activity) by having all steps performed at extremely low temperatures or under anhydrous conditions (Gerhardt and Heldt, 1984). After quenching plant material in liquid nitrogen, the frozen material is dehydrated by freeze drying, homogenized, and then applied to a gradient made of anhydrous organic solvents prior to centrifugation and subsequent collection of fractions. Subcellular compartments are separated on the basis of density in the centrifugation step and the fractions collected are enriched in organelles (such as chloroplasts, cytosol or vacuole; Gerhardt and Heldt, 1984; Lunn, 2006). By correlating the compartment-specific marker distributions with metabolite distributions over the gradient, it is possible to calculate compartment-specific metabolite distributions (Gerhardt and Heldt, 1984; Klie <em>et al.</em>, 2011; Krueger <em>et al.</em>, 2011). The software tool BestFit enables such calculations (Krueger <em>et al.</em>, 2011 and 2014; Klie <em>et al.</em>, 2011). Recently, a benchtop NAF method has been established and is adapted for a relatively small amount of tissue material (FÃ¼rtauer <em>et al.</em>, 2016). This benchtop NAF method can be used as an alternative to the conventional NAF procedure. However, we still believe that the conventional NAF procedure is the method of choice for the determination of <em>in vivo</em> metabolite distribution among cellular compartments as (i) the plastidic and vacuolar compartments are more clearly separated on the density gradient used, and (ii) the higher amount of starting material leads to more material in each obtained fraction, giving the advantage to increase the number of sub-aliquots per fraction (thus increasing the range of measurements which can be done) and also to apply less sensitive quantification methods than mass spectrometry-based methods. A good example of the use of the conventional NAF procedure is a study performed by Arrivault <em>et al.</em> (2014) in which the distribution of about 1,000 proteins and 70 metabolites, including 22 phosphorylated intermediates in <em>Arabidopsis thaliana</em> rosette leaves were analyzed, using the conventional NAF combined with MS-based approaches.<br/><br/>The protocol presented here is based on the method described by Krueger <em>et al.</em> (2014) with minor modifications. In our method, we provide a complete workflow for the NAF procedure in <em>Arabidopsis thaliana</em> and detailed information for the analysis of compartmentâspecific marker enzymes and metabolites which can be assigned to three subcellular compartments: ADP glucose pyrophosphorylase (AGPase) as plastidic marker, phosphoenolpyruvate carboxylase (PEPC) as cytosolic marker and nitrate and acid invertase as vacuolar markers.</p>
        </body>

<p class="pptt" id="biaoti29932">## Materials and Reagents</p><ol class="npwone"><li>NAF<ol class="npwtwo"><li>Plastic lids (Zinsser Analytic, catalog number: 3071400)</li><li>20 ml Polyvialâone per sample (Zinsser Analytic, catalog number: 3071400) </li><li>Steel balls:<br/>3 mm (Askubal, catalog number: 3 mm G100 1.4034)<br/>5 mm (Askubal, catalog number: 5 mm G100 1.4034)<br/>7 mm (Altmann, catalog number: KU.7G80-1.3541) <br/><em>Note: We recommend using steel balls of three different diameters 3, 5, and 7 mm (two of each size in each 20 ml Polyvial).</em> </li><li>Steel caps (custom made, please check Figure 1)</li><li>50 ml Falcon tubesâ10 per sample (SARSTEDT, catalog number: 62.554.502)</li><li>Paper tissue (KimWipes)</li><li>pH-indicator paper (Merck, catalog number: 1.10962)</li><li>Polyallomer centrifuge tubes (Beckman Coulter, catalog number: 326823) </li><li>Nylon mesh filter 20 Âµm pore size (SEFAR Nitex<sup>Â®</sup>, catalog number: 03-20/14)</li><li>1.5 ml screw cap tubesâ11 per sample (SARSTEDT, catalog number: 72.692)</li><li>Activated molecular sieve 4 Ã (Carl Roth, catalog number: 1318-02-1)</li><li>Plastic Pasteur pipette (NeoLab, catalog number: 2600111)</li><li>Glass pipettes (10 ml, 20 ml)</li><li>Glass funnel</li><li>Metal rack</li><li><em>Arabidopsis thaliana</em> leaf material</li><li>Liquid nitrogen</li><li>n-Heptane (Carl Roth, catalog number: 142-82-5)</li><li>Tetrachloroethylene (Carl Roth, catalog number: 127-18-4)</li><li>Liquid nitrogen (~5 L per sample)<br/><em>Note: the required liquid nitrogen for harvesting (1 L), grinding (2 L), and aliquoting (2 L) samples suggested here is a rough estimate, since the amount of nitrogen needed will depend on several factors. Please consider that while planning your experiments.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Measurement of metabolite markers<ol class="npwtwo"><li>96 Deep-well plate (Sarstedt, catalog number: 82.1970.002)</li><li>Steel balls (5 mm)</li><li>Tubes holder for the Retsch mill (Retsch, catalog number: 22.008.0008)</li><li>Ethanol (VMR Chemicals, catalog number: 20821.330)</li><li>HEPES (CAS: 7365-45-9)</li><li>NaOH (Carl Roth, catalog number: 6771.1)</li><li>Potassium phosphate buffer 1 M, pH 7.5<br/><ol class="npwthree"><li>K<sub>2</sub>HPO<sub>4</sub> (Sigma-Aldrich, CAS: 7758-11-4)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Sigma-Aldrich, CAS: 7778-77-0)</li></ol></li><li>NADPH (Roche, CAS: 2646-71-1)</li><li>Nitrate reductase (Roche, catalog number: 10981249001)</li><li>Phenazine ethosulfate (PES) (Sigma-Aldrich, CAS: 10510-77-7)</li><li>Sulfanilamide (Sigma-Aldrich, CAS: 63-74-1)</li><li>N-(1-Naphthyl)ethylenediamine dihydrochloride (NNEDA) (Sigma-Aldrich, CAS: 1465-25-4)</li><li>KNO<sub>3</sub> (Sigma-Aldrich, CAS: 7757-79-1)</li><li>Nitrate Standards (see Recipes)<br/>0, 0.2, 0.4, 0.8, and 1.6 mM in 70% ETOH with 10 mM MES pH 5.9 <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Measurement of enzyme markers<br/>For the full description and order details of the reagents used in the measurements of enzyme activities, please see <a href="https://os.bio-protocol.org/attached/file/20190718/Supplemental Table 1.xlsx" target="_blank">Supplemental Table 1</a>.<ol class="npwtwo"><li>PVPP</li><li>Deionized water</li><li>MgCl<sub>2</sub> </li><li>EDTA</li><li>EGTA</li><li>Benzamidine</li><li>Îµ-aminocapronic acid</li><li>BSA (protease free)</li><li>Leupeptin </li><li>DTT </li><li>PMSF </li><li>Triton X-100</li><li>Glycerol</li><li>NaF</li><li>3-phosphoglycerate</li><li>AGPase assay mix</li><li>PPi</li><li>Glycerokinase</li><li>ADP-glucose</li><li>GPOX</li><li>GDH</li><li>NaHCO<sub>3</sub> </li><li>PEP</li><li>Malate dehydrogenase</li><li>MTT</li><li>Alcohol Dehydrogenase</li><li>Acid invertase assay Buffer</li><li>Glucose oxidase</li><li>Horse Radish Peroxidase</li><li>Amplex Red</li><li>DMSO</li><li>Nitrate standards (see Recipes)</li><li>Extraction buffers (see Recipes)<br/><ol class="npwthree"><li>Extraction Buffer 10x</li><li>Extraction Buffer 1x</li></ol></li><li>AGPase activity (see Recipes)<br/><ol class="npwthree"><li>AGPase assay buffer</li><li>AGPase assay mix</li><li>AGPase determination mix</li></ol></li><li>PEPC activity (see Recipes)<br/><ol class="npwthree"><li>PEPC assay Buffer<br/></li><li>PEPC assay Mix</li><li>PEPC determination mix</li></ol></li><li>Acid Invertase activity (see Recipes)<br/><ol class="npwthree"><li>Acid invertase assay Buffer</li><li>Acid invertase Assay Mix</li><li>Acid invertase Determination mix</li></ol></li></ol></li></ol><p class="pptt" id="biaoti29933">### Equipment</p><ol class="npwone"><li>NAF<ol class="npwtwo"><li>Capper/decapper machine (custom made, see Figure 1)</li><li>Retsch mill (Retsch<sup>Â®</sup>, model: MM400)</li><li>Freeze dryer (Christ, model: Alpha 2-4)</li><li>Sonicator (BANDELIN SONOPULS HD 2070)</li><li>Peristaltic gradient pump (Bio-Rad Econo Gradient Pump, catalog number: 731-9002)</li><li>Gradient mixer (Bio-Rad Gradient mixer, catalog number: 731-8323)</li><li>Centrifuge (Beckman Coulter, model: Allegra X-15R)</li><li>Rotor (Beckman Coulter, model: SX4750A)</li><li>Ultracentrifuge (Beckman Coulter, model: Optima<sup>TM</sup> L80 XP)</li><li>Swinging bucket rotor (Beckman Coulter, model: SW32Ti)</li><li>Desiccator</li><li>Vacuum pump<br/><br/></li></ol></li><li>Measurement of metabolite markers<ol class="npwtwo"><li>Thermomixer (Eppendorf, model: Compact 5350)</li><li>Centrifuge (Eppendorf, model: 5430)</li><li>Microplate Spectrophotometer (BioTeK<sup>Â®</sup>, Epoch 2)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Measurement of enzyme markers<ol class="npwtwo"><li>Microplate Spectrophotometer (BioTeK<sup>Â®</sup>, ELx808)</li><li>Microplate Fluorescence reader (BioTeK<sup>Â®</sup>, Synergy HT)</li></ol></li></ol><p class="pptt" id="biaoti29934">### Software</p><ol class="npwtwok"><li>BestFit (Klie <em>et al.</em>, 2011; <a href="http://www.csbdb.de/csbdb/bestfit/bestfit.html" target="_blank">http://www.csbdb.de/csbdb/bestfit/bestfit.html</a>)</li></ol><p class="pptt" id="biaoti29935">### Procedure</p><ol class="npwone"><li>Harvesting, grinding and preparation of leaf material<ol class="npwtwo"><li>Harvest <em>Arabidopsis thaliana</em> leaf material and place into a 20 ml Polyvial (containing steel balls of three different diameters 3, 5, and 7 mm, two of each size in each 20 ml Polyvial) pre-cooled in liquid N<sub>2</sub> and snap-freeze in liquid N<sub>2</sub>.<br/><em>Note: If the metabolites of interest have extremely fast turn-over time, plant material should be harvested by cutting rosettes and quenching them instantaneously in a bath of liquid N</em><sub><em>2</em></sub><em> under the prevailing irradiance. The quenched material is then transferred into a 20 ml Polyvial (containing steel balls) pre-cooled in liquid N</em><sub><em>2</em></sub><em>. For an optimal grinding process, fill the vial to a maximum of 2/3 of its volume with the plant material. For NAF, 4 g of material is necessary, but it is recommended to harvest slightly more in order to have extra material to perform additional analysis in this material (such as quantification of metabolites, see section Data analysisâ4).</em> </li><li>For homogenization, replace the plastic lids of the Polyvials by steel caps and place the Polyvials containing the leaf material into a pre-cooled steel grinding adapter for the Retsch mill (Figures 1A-1D).<br/><em>Note: Our grinding adapters and steel caps are custom-made for the 20 ml Polyvials. Alternatively, the grinding of the leaf material can be performed by using a pre-cooled mortar and pestle. Make sure that the leaf material is kept frozen during the whole procedure.</em> </li><li>Homogenize the tissue for 1 min at 25 Hz (two times) in the Retsch mill (Figures 1E and 1F).<br/><em>Note: Make sure that the ground tissue is very well homogenized (thin powder). We suggest grinding the leaf material twice for 1 min each at 25 Hz. Cool down the vials and adapters in between with liquid N</em><sub><em>2</em></sub><em> to avoid thawing of the material.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191014/20191014200909_3477.jpg" width="335"/><br/><strong>Figure 1. Grinding procedure.</strong> A. Cool down the Polyvials, steel grinding adapter, and steel caps in liquid nitrogen. B. Remove the plastic lid of the Polyvials and place the steel cap frozen. C. Use a capper/decapper machine (custom-made) to close the vial with the steel cap. D. Closed vial. E. Place the Polyvial inside steel grinding adapter. F. Place the grinding adapter in the Retsch mill and proceed as in Step A3. After grinding use the same capper/decapper machine for replacing the steel caps for the plastic lids. The samples can be stored at -80 Â°C or proceed to Step A4. Note that to avoid overheating of the vial, steel caps and grinding adapter deep them in liquid nitrogen or make use of dried ice while preparing a second sample.<br/><br/></li><li>Weigh ~4 g of frozen tissue into a 50 ml Falcon tube pre-cooled in liquid N<sub>2</sub>.The samples can be stored at -80 Â°C for up to three months. Prepare as many Falcon tubes as necessary (each one will be used for one NAF).<br/><em>Note: Label the Falcon tubes and their corresponding lids. Record the weights of the empty Falcon tubes (with lids) and the precise amount of plant material weighed.</em> </li><li>Freeze dry the homogenate. To avoid losing leaf material during this process, remove the lid and cover the Falcon tubes with paper tissue (KimWipes) fixed with an elastic band. Place the tubes in a metal rack into the freeze drier at 0.02 bar and -80 Â°C for 5 days. Close the tubes with their corresponding lids immediately after removing them from the freeze drier. The dried material can be stored under vacuum in a desiccator protected from light and humidity up to 6 months. <br/><em>Note: In order to avoid thawing, place the Falcon tubes in a polystyrene box containing dry ice and replace the lids by the paper tissue while the Falcon tubes are in the box. </em> </li><li>Determine the plant material dry weight by weighing the closed Falcon tubes after the freeze drying process.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of solutions and filters <ol class="npwtwo"><li>Prepare the solutions for the NAF three days before starting the fractionation.</li><li>Add the activated molecular sieve 4 Ã to the Heptane (C<sub>7</sub>H<sub>16</sub>) and Tetrachloroethylene (C<sub>2</sub>C<sub>l4</sub>) solutions to ensure that no residual water is present.</li><li>Prepare a mixture of Tetrachloroethylene/Heptane 66:34 (v/v); density = 1.3 g cm<sup>-3</sup>. Add the activated molecular sieve 4 Ã and store the mixture protected from light in a brown glass bottle.<br/><em>Note: We usually use the ~100 ml of activated molecular sieve in a 1 L bottle. For 10 samples it is needed: a) 1 L of the Tetrachloroethylene/Heptane mixture 66:34 (v/v): 660 ml of Tetrachloroethylene + 340 ml of Heptane, b) 2 L of Heptane, c) 500 ml of Tetrachloroethylene.</em> </li><li>Before use always check the pH of the solutions with a pH paper, especially Tetrachloroethylene. pH should be neutral. If Tetrachloroethylene solution turns acidic, the dried leaf material will become brown during gradient centrifugation and the NAF will fail.</li><li>Cut 15 cm x 15 cm the nylon filters (one per NAF), weigh them and store them in clean 50 ml Falcon tubes. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Fractionation procedure<br/><em>Note: The whole procedure should be performed under a fume hood. Only dried aliquots (after Step C17) can be processed on regular bench. For pouring solutions, use glass pipettes.</em><br/><ol class="npwtwo"><li>Resuspend the dried leaf material in 20 ml of the Tetrachloroethylene/Heptane = 66:34 (v/v) mixture.</li><li>Ultrasonicate the suspension for 2 min, with 6 x 10 cycles, 65% of power (Figure 2). To avoid overheating the suspension during the sonication process, keep the Falcon tube in an ice bath.<br/><em>Note: Place the sonicator tip inside the solution, but avoid touching the tube wall. Make sure the sonication is successful. This is a critical step in the fractionation, since insufficient sonication can produce aggregates that do not pass through nylon mesh.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191014/20191014200956_6584.jpg" width="128"/><br/><strong>Figure 2. Setup for the ultrasonication procedure.</strong> Note that to avoid overheating the suspension during the process, the Falcon tube should be kept in an ice bath.<br/><br/></li><li>Fold a nylon mesh into a glass funnel placed on top of a 50 ml Falcon tube and pour the suspension through the nylon mesh. Wash the nylon mesh 3 x with 10 ml of heptane. <br/><em>Note: The first two washing steps are performed by slowly pouring heptane on the nylon mesh and collecting the flow through in the Falcon tube below. For the third washing step, remove all the residual solution in the nylon mesh by squeezing it (while wearing adequate gloves). Place the nylon mesh back in a 50 ml Falcon tube and continue at Step C16. Be careful that all liquid go through the nylon mesh. If itÂ´s the case, filter the whole obtained mixture again through a fresh nylon mesh. </em> </li><li>Centrifuge for 10 min at 3,200 <em>x g</em> and 4 Â°C, using a swing-out rotor. Discard the supernatant (by pouring it into a waste glass bottle) and resuspend the pellet in 5 ml of Tetrachloroethylene/Heptane = 66:34 (v/v) mixture.</li><li>Transfer 100 Âµl of the well homogenized resuspended pellet into a 1.5 ml screw cap tube. This aliquot is named fraction 0 (F0) and corresponds to the unfractionated material. Prepare three aliquots, add 900 Âµl of C<sub>7</sub>H<sub>16</sub> to each, and keep protected from the light and continue at Step C15. <br/><em>Note: Use a cut 200 Âµl pipette tip to collect F0. Homogenize well the mixture before taking each aliquot.</em> </li><li>Prepare the NAF gradient (Figure 3A): Linear gradient (30 ml, density from 1.43 to 1.62 g cm<sup>-3</sup>)<br/><ol class="npwthree"><li>With a Pasteur pipette fill the 50 ml Falcon tube A with Tetrachloroethylene/Heptane mixture and the 50 ml Falcon tube B with Tetrachloroethylene.</li><li>The dispensing needle is placed above a waste container. <br/><em>Note: Ensure that there are no air bubbles in any tubing by purging thoroughly the whole system.</em> </li><li>Set the peristaltic pump at a flow rate of 1.15 ml min<sup>-1</sup> and the following program:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Step</strong><br/></td><td><strong>ml</strong><br/></td><td><strong>Solvent A</strong><br/></td><td><strong>Solvent B</strong><br/></td></tr><tr><td>1* <br/></td><td>0</td><td>70%<br/></td><td>30%<br/></td></tr><tr><td>2**<br/></td><td>3</td><td>70% <br/></td><td>30%<br/></td></tr><tr><td>3</td><td>28</td><td>0%</td><td>100%</td></tr><tr><td>4</td><td>33</td><td>0%</td><td>100%</td></tr></tbody></table><em>Notes:</em><br/><ol class="npwfour"><li><em>*After 3 ml, pause the program. This step is used to ensure that the first mixture reaches the end of the dispensing needle before starting the actual gradient.</em> </li><li><em>**Remove the waste container and place the Polyallomer tube at this step. The needle should be placed slightly over the bottom of the Polyallomer tube and slightly touching the side of the tube (Figure 3B). Re-start the program.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191014/20191014201537_4979.jpg" width="112"/><br/><strong>Figure 3. Setup of the gradient pump and the mixer.</strong> A. Overview of the setup for preparing the NAF gradient. Solution A (Tetrachloroethylene/Heptane mixture) and solution B (Tetrachloroethylene). B. Detail of the bottom-up preparation of the gradient. Tube holder is a 50 ml falcon tube with paper support on the bottom to help holding the gradient tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>Take the needle carefully out from the gradient and place<em> the Polyallomer</em> tube in a rack. Apply the suspension from Step C4 to the gradient, very carefully with a Pasteur pipette. Touch the wall of the tube with the pipette slightly above the gradient and release carefully to avoid disturbing the gradient.</li><li>Place the gradients in the appropriate tube buckets from the ultracentrifuge. Verify their weights and to counterbalance, adjust the weight by adding 100% C<sub>2</sub>Cl<sub>4</sub>.</li><li>Centrifuge for 50 min at 5,000 <em>x g</em>, and 4 Â°C using a swinging bucket rotor. <br/><em>Note: We always use the Ultracentrifuge Beckman Coulter Optima</em><sup><em>TM</em></sup><em> L80 XP and Swinging bucket rotor Beckman Coulter SW32Ti. Set the acceleration/deceleration on three.</em> </li><li>Place the gradient tubes in a rack and mark eight fractions. Figure 4 illustrates how we usually separate the fractions based on the color.<br/><br/><img alt="" class="layerphoto" height="214" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191014/20191014201608_4734.jpg" width="100"/><br/><strong>Figure 4. NAF gradient after centrifugation.</strong> Usually, we separate eight fractions based on their color. The dark green layer is enriched in chloroplast and usually divided into two fractions (F2 and F3) to avoid metabolite overloading. Note that fraction 8 is a pellet, which is the vacuolar-enriched fraction.<br/><br/></li><li>Transfer carefully each fraction into clearly labeled clean 50 ml Falcon tube by using Pasteur pipette. The fractions are collected starting from the upper part (F1) to the bottom (F8). After F7 is removed, the pellet is resuspended in 20 ml heptane and the mixture transferred to a clean 50 ml Falcon tube. Use a new pipette for each fraction.<br/><em>Note: Transferring the fractions is a critical point for the method, since at this step cross contamination of the fractions should be avoided as much as possible. Therefore, always transfer the fractions from the very top of each layer.</em> </li><li>Add heptane up to 20 ml to the Falcon tubes containing F1 to F8 and mix carefully. Centrifuge all Falcon tubes for 10 min at 3,200 <em>x g</em> and at room temperature. </li><li>Discard the supernatant (by carefully pouring it into a waste glass bottle) and resuspend the pellet with 7 ml of Heptane.<br/><em>Note: The volume depends on how many aliquots you will need for your analysis. We usually prepare seven sub-aliquots per fraction. This number is enough for various metabolite analysis and determination of the compartment-specific markers.</em> </li><li>Transfer 6 x 1 ml from each suspension into 1.5 ml screw caps tubes. The pellet can easily re-aggregate at this step. Keep mixing by hand the solution while transferring into the tubes in order to obtain homogeneous aliquots from the same fraction.<br/><em>Note: Use a cut 1000 Âµl pipette tip to collect sub-aliquots from each fraction. As adding 7 ml generally does not allow to obtain seven aliquots of 1 ml, we generally take 0.8 ml for the seventh aliquot and clearly label the corresponding tube.</em> </li><li>Centrifuge for 10 min at 14,000 <em>x g</em> and at room temperature. Discard excess of supernatant (<em>c.a.</em> 900 Âµl) carefully, using a 1000 Âµl pipette tip. Process the F0 aliquots obtained in Step C5 in the same manner. </li><li>Place the screw caps tubes containing the sub-aliquots open in racks inside of a desiccator connected to a vacuum pump. Also, place the open Falcon tubes containing the filters from Step C3. Keep the aliquots under vacuum and protected from light overnight.</li><li>Close the tubes and store them at -80 Â°C until further process. </li><li>Weigh the dried filters.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Measurements of metabolite markers<ol class="npwtwo"><li>Extraction<ol class="npwthree"><li>Add to the dried aliquots a steel ball and 250 Âµl 80% ethanol/10 mM HEPES pH 7 to the samples.</li><li>Place the tubes in a cooled (4 Â°C) holder for the ball mill and disrupt the pellet for 1 min at 25 Hz.</li><li>Incubate in a thermomixer for 30 min at 95 Â°C and 1000 rpm.</li><li>Centrifuge for 10 min at 20,800 <em>x g</em> and 4 Â°C.</li><li>Transfer the supernatant to a 96-deep well plate and keep it protected from light and on ice.</li><li>Add to the Pellet 150 Âµl 80% ethanol/10 mM HEPES pH 7.</li><li>Incubate in a thermomixer for 30 min at 95 Â°C 1000 rpm.</li><li>Centrifuge for 10 min at 20,800 <em>x g</em> and 4 Â°C. </li><li>Transfer and combine with the previous supernatant. </li><li>Add to the Pellet 250 Âµl 50% ethanol/10 mM HEPES pH 7.</li><li>Incubate in a thermomixer for 30 min at 95 Â°C and 1,000 rpm.</li><li>Centrifuge for 10 min at 20,800 <em>x g</em> and 4 Â°C. </li><li>Transfer and combine with the previous two supernatants. It can be stored at -20 Â°C until further use.</li></ol></li><li>Nitrate (vacuolar marker)<br/>Nitrate content is determined following Cross <em>et al.</em> (2006). Nitrate can be used as a vacuolar marker in addition or alternatively to Acid Invertase.<br/><em>Note: Nitrite content must be determined and subtracted from the Nitrate values. For measuring nitrite content, nitrate reductase is replaced by water in the assay mix.</em> <br/><strong>Solutions:</strong> For storage recommendations, the solutions for nitrate measurement can be stored as follow:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td><strong>Solution:</strong><br/></td><td><strong>Storage temperature</strong><br/></td></tr><tr><td>Potassium phosphate buffer 1 M pH 7.5<br/></td><td>-20 Â°C<br/></td></tr><tr><td>NADPH 50 mM in NaOH 20 mM<br/></td><td>-80 Â°C<br/></td></tr><tr><td>Nitrate reductase (5 U ml<sup>-1</sup> in Potassium phosphate buffer 0.1 M)<br/></td><td>-80 Â°C<br/></td></tr><tr><td>PES 0.25 mM</td><td>-80 Â°C</td></tr><tr><td>Sulfanilamide 1% (w/v) in phosphoric acid 5%<br/></td><td>4 Â°C<br/></td></tr><tr><td>NNEDA 0.02% (w/v)<br/></td><td>4 Â°C<br/></td></tr><tr><td>Nitrate standards<br/></td><td>-80 Â°C<br/></td></tr></tbody></table>Nitrate Standards: 0, 0.2, 0.4, 0.8, and 1.6 mM in 70% ETOH with 10 mM MES pH 5.9 <br/><strong>Protocol:</strong> <ol class="npwthree"><li>For one reaction, dispense in a 96-well plate 95 Âµl of the assay mix:<br/>10.0 Âµl phosphate buffer<br/>0.5 Âµl NADPH<br/>2.0 Âµl Nitrate reductase<br/>82.5 Âµl deionized water</li><li>Add 5 Âµl of standard or extract (1 to 10 diluted extract).</li><li>Mix and incubate for 30 min at 25 Â°C, protected from light.</li><li>Add 15 Âµl of PES.</li><li>Mix and incubate for 20 min at 25 Â°C, protected from light.</li><li>Add 50 Âµl of Sulphanilamide and 50 Âµl of NNEDA.</li><li>Mix and incubate for 10 min at 25 Â°C, protected from light.</li><li>Read the plate at 540 nm in a microplate reader.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>Measurements of enzyme markers<ol class="npwtwo"><li>Enzyme extraction<ol class="npwthree"><li>Add to the dried extract ~2 mg of PVPP (one little spoon), one steel ball (5 mm diameter), and 500 Âµl of buffer 1x (see Recipe 1b). Keep the samples on ice.</li><li>Place the tubes in a cooled (4 Â°C) holder for the ball mill and disrupt the pellet for 1 min at 25 Hz. Make sure that the entire pellet is dissolved. If not, repeat the pellet disruption procedure.</li><li>Centrifuge for 10 min at 20,800 <em>x g</em> and 4 Â°C. Transfer the enzyme extract (supernatant) to a 96 deep-well plate. Keep on ice and proceed to the enzyme assays. If required, the supernatant can be stored at -80 Â°C.</li></ol></li><li>Enzyme assays<br/>All the three enzyme markers are measured according to Gibon <em>et al.</em> (2004).<ol class="npwthree"><li>AGPase activity (plastid marker)<ol class="npwfour"><li>Dispense in a 96-well plate placed on ice:<br/>14 Âµl of AGPase assay mix (see Recipe B2b)<br/>6 Âµl of standard or extract.</li><li>Mix and incubate for 20 min at 25 Â°C.</li><li>Add 20 Âµl of 0.5 M HCl in 100 mM Tricine/KOH pH 8.</li><li>Mix and incubate for 10 min at room temperature.</li><li>Add 20 Âµl of 0.5 M NaOH.</li><li>Mix, add 50 Âµl of AGPase Determination mix (see Recipe B2c).</li><li>Mix, spin down, and read at wavelength of 340 nm and 30 Â°C until the maximum rate of reaction stabilizes.</li></ol></li><li>PEPC (Cytosol marker)<ol class="npwfour"><li>Dispense in a 96-well plate placed on ice.<br/>18 Âµl PEPC Assay Mix (see Recipe B3b)<br/>2 Âµl of standard or extract</li><li>Mix and incubate for 20 min at 25 Â°C.</li><li>Add 20 Âµl of 0.5 M HCl in 100 mM Tricine/KOH pH 9.</li><li>Mix, spin down, and incubate at 95 Â°C for 5 min. Cool and spin down. Keep the plate on ice.</li><li>Add 20 Âµl 0.5 M NaOH.</li><li>Mix, add 45 Âµl of PEPC Determination mix (see Recipe 3c, protect from light).</li><li>Add 5 Âµl of 4 mM PES (protect from light).</li><li>Mix and read at wavelength of 570 nm and 30 Â°C until the maximum rate of reaction stabilizes.</li></ol></li><li>Acid Invertase (Vacuole marker)<ol class="npwfour"><li>Dispense in a black 96-well microplate:<br/>10 Âµl Acid invertase Assay mix (see Recipe B4b)<br/>5 Âµl of standard or enzyme extract</li><li>Mix and incubate at 25 Â°C for 5 and 40 min.</li><li>Add 10 Âµl 0.5 M NaOH.</li><li>Mix and incubate at R.T. for 10 min.</li><li>Add 10 Âµl 0.5 M HCl in 100 mM Tricine/KOH pH 8.</li><li>Mix and add 50 Âµl of Acid invertase Determination mix (see Recipe B4c).</li><li>Mix and read the fluorescence for 10-15 min using the following setting:<br/>Excitation at 530 nm, emission at 590 nm, temperature: 30 Â°C, sensitivity at 25. Use the rate of reaction for calculations. We use the Synergy fluorescence meter.</li></ol></li></ol></li></ol></li></ol><p class="pptt" id="biaoti29936">### Data analysis</p><ol class="npwtwok"><li>For evaluation of subcellular metabolite distributions, we recommend the statistical software BestFit. The BestFit is a C-language command line tool that allows calculation and evaluation of subcellular distributions from NAF data. The software requires; (i) the distribution of specific markers for each subcellular compartment analyzed throughout the different fractions (Figure 5) and (ii) the distribution of the metabolites of interest throughout the different fractions.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191014/20191014203453_3857.jpg" width="163"/><br/><strong>Figure 5. Distribution in percentage of subcellular compartment-specific markers throughout the gradient showed in Figure 4.</strong> ADP glucose pyrophosphorylase (AGPase) as plastidic marker, phosphoenolpyruvate carboxylase (PEPC) as cytosolic marker and nitrate and acid invertase as vacuolar markers. <br/><br/></li><li>For the values of either markers and metabolites of interest in the input file, we recommend using percentage, although absolute measurements can also be used (note that in this case values must be â¥ 0). </li><li>For more details on how to use BestFit, we suggest reading the documentation pdf file within the BestFit Folder (<a href="http://www.csbdb.de/csbdb/bestfit/bestfit.html" target="_blank">http://www.csbdb.de/csbdb/bestfit/bestfit.html</a>). Alternatively, you can also find details regarding data input and use of BestFit in Krueger <em>et al.</em> (2014).</li><li>BestFit outputs the subcellular-distribution of the metabolites of interest in percentage. In order to calculate which amounts this represents, metabolites are also measured in ground material not processed through NAF (see Note in A1). Various recovery calculations can be done by using the following information: (i) material fresh weight used for NAF (determined in step A4), (ii) corresponding dry weight (determined in step A6), (iii) how much material was applied to the gradient (determined in step C18 by weighing the residual material in the filter), (iv) F0 obtained step C6. We recommend always recording these data and collecting F0 for each NAF. <br/></li></ol><p class="pptt" id="biaoti29937">### Recipes</p><ol class="npwone"><li>Measurement of metabolite markers<ol class="npwtwo"><li>Nitrate standards<br/>0, 0.2, 0.4, 0.8, and 1.6 mM in 70% ETOH with 10 mM MES pH 5.9 <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Measurement of enzyme markers<ol class="npwtwo"><li>Extraction buffers<ol class="npwthree"><li>Extraction Buffer 10x<br/>500 mM HEPES <br/>100 mM MgCl<sub>2</sub> <br/>10 mM EDTA <br/>10 mM EGTA <br/>10 mM Benzamidine <br/>10 mM Îµ-aminocapronic acid <br/>2.5% w/v BSA (protease-free)<br/>Bring volume to c.a. 40 ml, adjust pH to 7.5 with KOH 10 M, and then adjust the volume to 50 ml<br/>The extraction buffer 10x can be stored at -20 Â°C<br/><strong>Prepare the following solutions and store accordingly:</strong><br/>Leupeptin 2 mM (100x), freeze in liquid nitrogen and then store at -80 Â°C<br/>DTT 500 mM (1,000x), freeze in liquid nitrogen and then store at -80 Â°C<br/>PMSF 100 mM in isopropanol (100x), store at 4 Â°C, protect from light, stable for 1 month<br/>Triton X-100 10% v/v (10x), store at RT<br/>Glycerol (87% v/v), store at RT</li><li>Extraction Buffer 1x<br/>Prepare only prior the extraction combining as the following:<br/><div style="white-space:nowrap;"><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:550px;"><tbody><tr><td><strong>[Final]       </strong><br/></td><td><strong>Reagent</strong><br/></td><td><strong>[stock]</strong><br/></td><td><strong>For 1,000 Âµl</strong><br/></td></tr><tr><td>1x<br/></td><td>Buffer 10x (see Recipe 1a)<br/></td><td>10x<br/></td><td>100 Âµl<br/></td></tr><tr><td>10 ÂµM<br/></td><td>Leupeptin<br/></td><td>2 mM<br/></td><td>5 Âµl<br/></td></tr><tr><td>1 mM<br/></td><td>DTT<br/></td><td>500 mM<br/></td><td>2 Âµl<br/></td></tr><tr><td>1 mM<br/></td><td>PMSF<br/></td><td>100 mM in isopropanol <br/></td><td>10 Âµl<br/></td></tr><tr><td>1%<br/></td><td>Triton X-100 <br/></td><td>10% v/v <br/></td><td>100 Âµl<br/></td></tr><tr><td>20%<br/></td><td>Glycerol <br/></td><td>87% (v/v) <br/></td><td>230 Âµl<br/></td></tr><tr><td><br/></td><td>H<sub>2</sub>O <br/></td><td><br/></td><td>553 Âµl<br/></td></tr></tbody></table></div></li></ol></li><li>AGPase activity<ol class="npwthree"><li>AGPase assay buffer (can be stored at -20 Â°C)<br/>0.25 M HEPES/KOH pH 7.5<br/>7.5 mM NaF<br/>25 mM MgCl<sub>2</sub><br/>5 mM 3-phosphoglycerate<br/>0.25% (v/v) Triton X-100</li><li>AGPase assay mix<br/>7.6 Âµl water<br/>4 Âµl AGPase assay buffer (see Recipes B2a)<br/>2 Âµl 20 mM PPi<br/>0.2 Âµl Glycerokinase (200 U ml-1 in 200 mM Tricine/KOH pH 8, 10 mM MgCl<sub>2</sub>) <br/>0.2 Âµl ADP-glucose 0 mM (Blank) or 100 mM (maximum)</li><li>AGPase determination mix<br/>37.8 Âµl water<br/>10 Âµl 1 M Tricine/KOH pH 8<br/>0.2 Âµl 1 M MgCl<sub>2</sub><br/>0.5 Âµl GDH (200 U ml-1 in 200 mM Tricine/KOH pH 8, 10 mM MgCl<sub>2</sub>)<br/>0.5 Âµl GPOX (500 U ml-1 in 200 mM Tricine/KOH pH 8, 10 mM MgCl<sub>2</sub>)<br/>1 Âµl 66 mM NADH</li></ol></li><li>PEPC activity<ol class="npwthree"><li>PEPC assay Buffer (can be stored at -20 Â°C)<br/>0.5 M Tricine/KOH pH 8<br/>100 mM MgCl<sub>2</sub> <br/>50 mM NaHCO<sub>3</sub> <br/>0.25% Triton X-100 </li><li>PEPC assay Mix<br/>11.4 Âµl H<sub>2</sub>O<br/>4 Âµl PEPC assay Buffer (see Recipe B3a)<br/>2 Âµl 0 (Blank) or 20 mM PEP (maximum) <br/>0.4 Âµl 5 mM NADH<br/>0.2 Âµl Malate dehydrogenase (100 U ml<sup>-1</sup> in 200 mM Tricine/KOH pH 8, 10 mM MgCl<sub>2</sub>)</li><li>PEPC determination mix (protect from light)<br/>18 Âµl water<br/>10 Âµl 1M Tricine/KOH pH 9.0<br/>10 Âµl 10 mM MTT <br/>4 Âµl 200 mM EDTA<br/>2 Âµl 50% Ethanol <br/>1 Âµl Alcohol Dehydrogenase (2,000 U ml<sup>-1</sup> in 200 mM Tricine/KOH pH 9, 10 mM MgCl<sub>2</sub>)</li></ol></li><li>Acid Invertase activity<ol class="npwthree"><li>Acid invertase assay Buffer (can be stored at -20 Â°C)<br/>0.25 M Acetate/KOH pH 5 </li><li>Acid invertase Assay Mix: for 1 reaction<br/>10 Âµl Acid invertase assay buffer (see Recipe B4a)<br/>35 Âµl water</li><li>Acid invertase Determination mix (protect from light): for 1 reaction<br/>42 Âµl water<br/>5 Âµl 1 M HEPES/KOH pH 7<br/>1 Âµl Glucose oxidase (200 U ml<sup>-1</sup> in 200 mM HEPES/KOH pH 7.0)<br/>1 Âµl Horse Radish Peroxidase (0.2 U ml<sup>-1</sup> in 200 mM HEPES/KOH pH 7.0)<br/>1 Âµl 20 mM Amplex Red in DMSO</li></ol></li></ol></li></ol><p class="pptt" id="biaoti29938">### Acknowledgments</p><p class="ppzz">We thank Deutsche Forschungsgemeinschaft (DFG TRR 175 The Green Hub â Central Coordinator of Acclimation in Plants; FA and ARF) for funding.<br/>  We also would like to thank the authors from Krueger <em>et al.</em> (2014), the previous work describing the NAF method, which we made the modification presented in the present protocol.</p><p class="pptt" id="biaoti29939">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti29940">### References</p><ol class="npwtwok"><li>Arrivault, S., Guenther, M., Florian, A., Encke, B., Feil, R., Vosloh, D., Lunn, J. E., Sulpice, R., Fernie, A. R., Stitt, M. and Schulze, W. X. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24866124" target="_blank">Dissecting the subcellular compartmentation of proteins and metabolites in <em>Arabidopsis</em> leaves using non-aqueous fractionation.</a> <em>Mol Cell Proteomics</em> 13(9): 2246-2259.</li><li>Cross, J. M., von Korff, M., Altmann, T., Bartzetko, L., Sulpice, R., Gibon, Y., Palacios, N. and Stitt, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17085515" target="_blank">Variation of enzyme activities and metabolite levels in 24 <em>Arabidopsis</em> accessions growing in carbon-limited conditions.</a> <em>Plant Physiol </em>142(4): 1574-1588.</li><li>Elbers, R., Heldt, H. W., Schmucker, P., Soboll, S. and Wiese, H. (1974). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4154892" target="_blank">Measurement of the ATP/ADP ratio in mitochondria and in the extramitochondrial compartment by fractionation of freeze-stopped liver tissue in non-aqueous media. </a><em>Hoppe Seylers Z Physiol Chem </em>355(3): 378-393.</li><li>Farre, E. M., Tiessen, A., Roessner, U., Geigenberger, P., Trethewey, R. N. and Willmitzer, L. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11598242" target="_blank">Analysis of the compartmentation of glycolytic intermediates, nucleotides, sugars, organic acids, amino acids, and sugar alcohols in potato tubers using a nonaqueous fractionation method.</a> <em>Plant Physiol </em>127(2): 685-700. </li><li>Fettke, J., Eckermann, N., Tiessen, A., Geigenberger, P. and Steup, M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16098110" target="_blank">Identification, subcellular localization and biochemical characterization of water-soluble heteroglycans (SHG) in leaves of <em>Arabidopsis thaliana</em> L.: distinct SHG reside in the cytosol and in the apoplast.</a> <em>Plant J </em>43(4): 568-585.</li><li>Furtauer, L., Weckwerth, W. and Nagele, T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28066469" target="_blank">A benchtop fractionation procedure for subcellular analysis of the plant metabolome.</a> <em>Front Plant Sci </em>7: 1912.</li><li>Gerhardt, R. and Heldt, H. W. (1984). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16663663" target="_blank">Measurement of subcellular metabolite levels in leaves by fractionation of freeze-stopped material in nonaqueous media.</a> <em>Plant Physiol </em>75(3): 542-547.</li><li>Gerhardt, R., Stitt, M. and Heldt, H. W. (1987). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16665257" target="_blank">Subcellular metabolite levels in spinach leaves: regulation of sucrose synthesis during diurnal alterations in photosynthetic partitioning. </a><em>Plant Physiol </em>83(2): 399-407.</li><li>Gibon, Y., <em>et al.</em> (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15548738" target="_blank">A Robot-based platform to measure multiple enzyme activities in Arabidopsis using a set of cycling assays: comparison of changes of enzyme activities and transcript levels during diurnal cycles and in prolonged darkness. </a><em>The Plant Cell </em>16(12): 3304-3325.</li><li>Klie, S., Krueger, S., Krall, L., Giavalisco, P., Flugge, U. I., Willmitzer, L. and Steinhauser, D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22645541" target="_blank">Analysis of the compartmentalized metabolome - a validation of the non-aqueous fractionation technique. </a><em>Front Plant Sci </em>2: 55.</li><li>Krueger, S., Giavalisco, P., Krall, L., Steinhauser, M. C., Bussis, D., Usadel, B., Flugge, U. I., Fernie, A. R., Willmitzer, L. and Steinhauser, D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21423574" target="_blank">A topological map of the compartmentalized <em>Arabidopsis thaliana</em> leaf metabolome. </a><em>PLoS One </em>6(3): e17806.</li><li>Krueger, S., Niehl, A., Lopez Martin, M. C., Steinhauser, D., Donath, A., Hildebrandt, T., Romero, L. C., Hoefgen, R., Gotor, C. and Hesse, H. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19143986" target="_blank">Analysis of cytosolic and plastidic serine acetyltransferase mutants and subcellular metabolite distributions suggests interplay of the cellular compartments for cysteine biosynthesis in <em>Arabidopsis</em>. </a><em>Plant Cell Environ </em>32(4): 349-367.</li><li>Krueger, S., Steinhauser, D., Lisec, J. and Giavalisco, P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24057387" target="_blank">Analysis of subcellular metabolite distributions within <em>Arabidopsis thaliana</em> leaf tissue: a primer for subcellular metabolomics. </a><em>Methods Mol Biol </em>1062: 575-596.</li><li>Lunn, J. E. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17030538" target="_blank">Compartmentation in plant metabolism. </a><em>J Exp Bot </em>58(1): 35-47.</li><li>Shapiguzov, A., Vainonen, J. P., Hunter, K., Tossavainen, H., Tiwari, A., Jarvi, S., Hellman, M., Aarabi, F., Alseekh, S., Wybouw, B., Van Der Kelen, K., Nikkanen, L., Krasensky-Wrzaczek, J., Sipari, N., Keinanen, M., Tyystjarvi, E., Rintamaki, E., De Rybel, B., Salojarvi, J., Van Breusegem, F., Fernie, A. R., Brosche, M., Permi, P., Aro, E. M., Wrzaczek, M. and Kangasjarvi, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30767893" target="_blank"><em>Arabidopsis</em> RCD1 coordinates chloroplast and mitochondrial functions through interaction with ANAC transcription factors.</a> <em>Elife</em> 8. pii:e43284.</li><li>Sharkey, T. D., and Vanderveer, P. J. (1989). <a href="http://www.plantphysiol.org/content/91/2/679.short" target="_blank">Stromal phosphate concentration is low during feedback limited photosynthesis.</a> <em>Plant Physiol</em>. 91: 679-684.</li><li>Sweetlove, L. J. and Fernie, A. R. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23330793" target="_blank">The spatial organization of metabolism within the plant cell.</a> <em>Annu Rev Plant Biol</em> 64: 723-746. </li><li>Weiner, H., and Heldt, H. W. (1992). <a href="https://link.springer.com/article/10.1007/BF00201946" target="_blank">Intercellular and intracellular-distribution of amino acids and other metabolites in maize (<em>Zea-Mays</em> L) leaves.</a> <em>Planta</em> 187(2),:242-246.</li><li>Winter, H., Robinson, D. G.and Heldt, H. W. (1993). <a href="https://link.springer.com/article/10.1007/BF00199748" target="_blank">Subcellular volumes and metabolite concentrations in barley leaves.</a> <em>Planta</em> 191(2): 180-190.</li><li>Yamada, K., Norikoshi, R., Suzuki, K., Imanishi, H. and Ichimura, K. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20183924" target="_blank">Determination of subcellular concentrations of soluble carbohydrates in rose petals during opening by nonaqueous fractionation method combined with infiltration-centrifugation method.</a> <em>Planta</em> 230(6): 1115-1127.<br/></li></ol>