<body>
            <h1>Measuring Small-molecule Inhibition of Protein Interactions in Live Cells Using FLIM-FRET</h1>
            <p><strong>Authors:</strong> James M. Pemberton, Qian Liu, David W. Andrews</p>
            <h2>Abstract</h2>
            <p>This protocol was designed to quantitatively measure small-molecule displacement of proteins in live mammalian cells using fluorescence lifetime imaging microscopyâFÃ¶rster resonance energy transfer (FLIM-FRET). Tumour cell survival is often dependent on anti-apoptotic proteins, which bind to and inhibit pro-apoptotic proteins, thus preventing apoptosis. Small-molecule inhibitors that selectively target these proteins (termed BH3-mimetics) are therefore a promising avenue for the treatment of several cancers. Previous techniques used to study the efficacy of these drugs often use truncated versions of both pro- and anti-apoptotic proteins, as they are membrane bound and hydrophobic in nature. As a result, the true efficacy of these drugs to displace full-length pro-apoptotic proteins in their native environment within a cell is poorly understood. This protocol describes FLIM-FRET methods to directly measure the displacement (or lack of displacement) of full-length Bcl-2 family proteins in live mammalian cells.</p>
            <p><strong>Keywords:</strong> FLIM, FRET, BH3-mimetic, Apoptosis, Bcl-2, Bcl-XL, Bad, Displacement, Resistance, Live cell</p>
            <h2>Background</h2><p>Tumour cell survival is dependent on anti-apoptotic proteins such as Bcl-2 and Bcl-XL, which maintain the integrity of the mitochondrial outer-membrane, thus preventing programmed cell death. Small-molecule inhibitors that selectively target these anti-apoptotic proteins (termed BH3-mimetics) are therefore a promising avenue for the treatment of several cancers. However, in order to study the efficacy of BH3-mimetics, tools to measure Bcl-2 family interactions are required. Previous techniques used to study the efficacy of these drugs often used truncated versions of Bcl-2 and Bcl-XL, as they are membrane bound and contain hydrophobic sequences that make purification difficult. However, the use of truncated proteins <em>in vitro</em> fails to recapitulate the whole story, since Bcl-2 family proteins often behave differently on membrane surfaces (Lovell <em>et al.</em>, 2008; PÃ©cot <em>et al.</em>, 2016). Moreover, the use of detergents in immunoprecipitation of endogenous proteins can artificially promote or dissociate Bcl-2 family protein interactions (Hsu and Youle, 1997; Billen <em>et al.</em>, 2008). Our approach is to use fluorescence lifetime imaging microscopyâFÃ¶rster resonance energy transfer (FLIM-FRET), to measure full-length protein interactions in living cells (Osterlund <em>et al.</em>, 2015; Liu <em>et al.</em>, 2019). This is accomplished by expressing two proteins of interest in a desired cell type, with each protein fused to appropriate fluorescent proteins. As a donor fluorescent protein, we use mCerulean3 (mCer3) and as the acceptor, Venus. During a protein-protein interaction, the fluorescent proteins are in close proximity, enabling the donor fluorescent protein to non-radiatively transfer energy to the acceptor. Small molecule inhibitors that inhibit the studied protein interaction, result in a decrease in energy transfer due to a reduction in protein binding. The advantage of this technique is that it enables the use of full-length proteins interacting in their native environment within the cell. Additionally, with the use of proper controls, FLIM-FRET can distinguish true protein interactions from random collisions that occur on confined organelle-membrane surfaces (Osterlund <em>et al.</em>, 2015). Here we describe a detailed protocol on how to collect and analyze FLIM-FRET data, as in our most recent publication (Liu <em>et al.</em>, 2019).</p>
        </body>

<p class="pptt" id="biaoti36753">## Materials and Reagents</p><ol class="npwtwok"><li>10 cm culture dish (Sarstedt, catalog number: 83.3902)</li><li>1.7 ml graduated microcentrifuge tubes, autoclaved/sterile (FroggaBio Scientific Solutions, catalog number: LMCT1.7B)</li><li>CellCarrier-384 Ultra microplate, tissue culture treated (PerkinElmer, catalog number: 6057300)</li><li>Mirus TransIT-X2<sup>Â®</sup> (Mirus, catalog number: MIR6003, store at -20 Â°C)</li><li>Opti-MEM<sup>TM</sup> (Thermo Fisher Scientific, catalog number: 51985091)</li><li>DMEM (Gibco, 12800-082)</li><li>Fetal Bovine Serum (FBS) (Sigma-Aldrich, catalog number: 17B500)</li><li>Non-essential amino acids (NEAA) (Thermo Fisher Scientific, catalog number: 11140050)</li><li>Phosphate Buffered Saline (PBS) (Thermo Fisher Scientific, catalog number: 10010031)</li><li>Trypsin (10x) (Gibco, catalog number: 15400-054)</li><li>ABT-263 (Navitoclax) (Selleck Chemicals, catalog number: S1001. Dissolve in DMSO, store at -20 Â°C)</li><li>DMSO (Thermo Fisher Scientific, catalog number: 85190)</li><li>NaOH (BioShop, catalog number: SHY700.1)</li><li>BMK Bax<sup>-/-</sup>Bak<sup>-/-</sup> cells expressing mCer3-Bcl-XL (Liu <em>et al.</em>, 2019)</li><li>EGFP constructs for mCer3-Bcl-XL, Venus-Bad, Venus-Bad(4E) (Liu <em>et al.</em>, 2012; Osterlund <em>et al.</em>, 2015; Liu <em>et al.</em>, 2019). Please contact the corresponding author for constructs</li><li>Fluorescein (Thermo Fisher Scientific, catalog number: F1300)</li><li>10 nM Fluorescein standard in NaOH (see Recipes)</li><li>Media for cells (see Recipes)<br/></li></ol><p class="pptt" id="biaoti36754">### Equipment</p><ol class="npwtwok"><li>Biological safety cabinet (Thermo Fisher Scientific, 1300 Series A2, catalog number: 1323TS)</li><li>Tissue culture incubator (Thermo Fisher Scientific, Heracell<sup>TM</sup> 150i CO<sub>2</sub> incubator, catalog number: 512026282)</li><li>Vortex (Scientific Industries, Vortex Genie 2, catalog number: SI-0236)</li><li>Hemocytometer (Millipore Sigma, Bright-Line<sup>TM</sup>, catalog number: Z359629-1EA)</li><li>ISS-Alba FLIM/FCS Confocal Microscope (ISS) <br/><em>Note: This microscope is custom built from ISS. For full specifications, please refer to (Osterlund </em><em>et al.</em><em>, 2015).</em><br/></li></ol><p class="pptt" id="biaoti36755">### Software</p><ol class="npwtwok"><li>VistaVision 4.1 (or higher) (ISS, <a href="http://www.iss.com/microscopy/software/vistavision.html" target="_blank">http://www.iss.com/microscopy/software/vistavision.html</a>)</li><li>ImageJ bundled with 64-bit Java 1.8.0 (NIH, <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>)</li><li>ImageJ ROI Macro (Script attached to key resource table of Liu <em>et al.</em>, 2019) (<a href="https://github.com/DWALab/Liu_et_al_2018_eLife/blob/master/ROILifetimeAnalysis.ijm" target="_blank">https://github.com/DWALab/Liu_et_al_2018_eLife/blob/master/ROILifetimeAnalysis.ijm</a>)</li><li>Excel (Microsoft, <a href="https://www.microsoft.com/en-ca/" target="_blank">https://www.microsoft.com/en-ca/</a>)</li><li>GraphPad Prism 7 (GraphPad, <a href="https://www.graphpad.com/" target="_blank">https://www.graphpad.com/</a>)<br/></li></ol><p class="pptt" id="biaoti36756">### Procedure</p><ol class="npwone"><li>Reverse transfection and plating of cells<br/>The presented example is an experiment to determine if 10 Î¼M of the BH3-mimetic ABT-263 can displace the BH3-only protein, Bad, from the anti-apoptotic protein Bcl-XL in live cells (Figure 1). The acceptor fluorescent protein is fused to the N-terminus of Bad (Venus-Bad), while the donor fluorescence protein is fused to the N-terminus of Bcl-XL (mCer3-Bcl-XL). Our cell line of choice is baby mouse kidney (BMK) cells genetically deficient for the pro-apoptotic proteins Bax and Bak (BMK-dko cells, Liu <em>et al.</em>, 2019) stably expressing mCer3-Bcl-XL. These cells are resistant to apoptotic stimuli, and thus tolerate higher expression of pro-apoptotic proteins and higher concentrations of apoptotic inducing agents like BH3-mimetics. It is worth noting that a wide range of different cell lines and proteins can be used with this protocol. Our laboratory has previously used MCF-7 and HCT-116 cell lines. Additionally, we use a wide range of different proteins, both pro-apoptotic (Bad, Bim, tBid, Puma, Noxa, <em>etc.</em>) and anti-apoptotic (Bcl-XL, Bcl-2, Bcl-w, and Mcl-1).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015002603_4097.jpg" width="303"/><br/><strong>Figure 1. Overview of FLIM-FRET in live cells.</strong> The donor fluorescence protein (mCer3) alone has a measured lifetime of 3.8 ns. In close proximity to the acceptor fluorescent protein, Venus, the lifetime for mCer3 decreases, indicative of FÃ¶rster resonance energy transfer. The BH3-mimetic ABT-263 binds to Bcl-XL, displacing and/or preventing protein interactions. When fully displaced, the fluorescence lifetime for mCer3 returns to 3.8 ns. Partial displacement results in intermediate fluorescence lifetimes enabling estimation of relative binding affinities.<br/><br/><ol class="npwtwo"><li>The following conditions are for the transfection of 2 wells in a 384-well format plate (~10,000 cells per well). In a sterile tissue-culture hood, put 10 Î¼l of Opti-MEM<sup>TM</sup> into a sterile 1.7 ml microcentrifuge tube, then add 200 ng of plasmid DNA with the Venus-Bad gene. Vortex the mixture, ensuring the small volume of 10 Î¼l falls back to the bottom of the tube.<br/><em>Note: Other plate types can be used, such as 6-well dishes and whole mount slides. However, we prefer using 384-well plates as less reagents are required per experiment. For all plate types, itâs critical that there is no background fluorescence from the plate and that the plate or chamber slide must be compatible with cell culture.</em> </li><li>Add 0.2 Î¼l of Mirus TransIT-X2<sup>Â®</sup> then immediately vortex the mixture. Ensure a thorough vortex for up to 20 s. After mixing, ensure the small volume falls back to the bottom of the tube. Let the reaction sit at room temperature for approximately 10 min.</li><li>While the transfection reaction sits, we often use this time to split and count the BMK-dko cells stably expressing our donor fluorescent protein (mCer3-Bcl-XL), as follows:<ol class="npwthree"><li>Remove a 10 cm culture dish containing cells that are ~80% confluent from the incubator, and place it in the sterile tissue culture hood.</li><li>Aspirate the DMEM media in the dish.</li><li>Wash the cells with 10 ml of PBS and then remove the PBS using vacuum aspiration. </li><li>Add 1 ml of 1x-Trypsin. Using your hands, tilt the 10 cm dish to ensure the 1 ml of Trypsin is evenly distributed. Place the 10 cm dish back into the 37 Â°C incubator, periodically checking on the plate (every few minutes) until it becomes visually apparent that the majority of cells have detached from the dish.</li><li>When the cells are detached from the dish, add 4 ml of fresh DMEM media to the dish and gently pipette the cells up and down (~10 times) to disrupt large cell aggregates. Then place the suspended cells into a 15 ml falcon tube.</li><li>Using a hemocytometer, count the number of suspended cells and express this value as (number of cells) per (1 Î¼l). In our example experiment, we counted a total of 520 cells/Î¼l.</li></ol></li><li>Since the prepared transfection is for 2-wells (of a 384 well plate), 2 x 10<sup>4</sup> cells in a total volume of 80 Î¼l are required (this is 1 x 10<sup>4</sup> cells in a volume of 40 Î¼l per well). Dilute a small fraction of your counted cell suspension to a final concentration of 250 cells/Î¼l (2 x 10<sup>4</sup> cells divided by 80 Î¼l volume).</li><li>Take 80 Î¼l of your suspended cells (at a concentration of 250 cells/Î¼l) and when the 10 min incubation (Step A2) is complete, mix with the transfection reaction. Add 45 Î¼l of this mixture to a single well of a PerkinElmer cellcarrier ultra 384-well plate (2 wells in total).</li><li>Allow the cells to settle to the bottom of the plate by leaving the plate to rest flat in the tissue culture hood for approximately 10 min. Afterward, the plate should be placed in the 37 Â°C incubator.</li><li>Approximately four hours after Step 6, remove the plate from the incubator and place in the sterile tissue culture hood.</li><li>Remove all media from the wells, and replace with desired media containing the treatment of choice. In our example experiment, either media containing DMSO, or media containing 10 Î¼M ABT-263 is added back to the wells of the 384-well plate.</li><li>Return the plate back to the incubator, and wait 20 h.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Calibrating the ISS-Alba system<ol class="npwtwo"><li>Turn the ISS-Alba instrument on in the following order: computer, climate control unit (set at 25 Â°C), TI-PS100W power supply, Nikon TI-S-CON motorized stage control, Nikon Intensilight C-HGFIE, excitation boxes for 445 and 514 nm lasers (Ext. 20 MHz), Detector box, Nikon Eclipse Ti microscope, A320 FastFLIM (91070 2012), on the computer load VistaVision Suite software, A512 3-Axis DAC controller, A403 scanning mirror driver (91032 1009), signal conditioning unit (98018 3001). Allow the entire apparatus to warm up for 30 min.<br/><em>Note: This protocol is not limited to the ISS-Alba microscope. Several other commercial and custom-built instruments can be used, but with differences in downstream image processing (Osterlund </em><em>et al.</em><em>, 2019; Hirmiz </em><em>et al.</em><em>, 2019). We recommend using confocal microscopy as many Bcl-2 family interactions occur on mitochondrial membranes, confocal microscopy allows you to achieve sub-cellular resolution interaction maps. For more accurate estimation of fluorescence lifetime, we also recommend single photon counting detectors. Single photon avalanche photo-diodes (SPADs) with single photon time correlated or time-gating capabilities are ideal. Frequency domain FLIM measurements can also be utilized but the data interpretation is less straight forward (Sun </em><em>et al.</em><em>, 2012).</em> </li><li>Take the plate out of the incubator and add to an empty well ~60 Î¼l of fluorescein at a concentration of 10 nM in 0.1 M NaOH (see Recipe below).<br/><em>Note: We often find day to day variability in the photon count, despite not making any changes to laser power or filters. As a result, we use this fluorescein sample as a reference point to ensure we consistently achieve the same photon counts as previous experiments. Additionally, we measure the mono-exponential lifetime of fluorescein as a control for the instrument.</em> </li><li>Lower the objective, and apply a small bead of water. Ensure the water resupply tube is in close proximity to the droplet. Place the 384-well plate into the holder on the microscope, and begin raising the objective until the water droplet makes contact with the bottom of the plate.</li><li>Move the stage to a well containing cells, and use the brightfield to focus on the cells.</li><li>Once focused on the cells, select âz-resetâ on the microscope, and move 100 Î¼m above the cells to ensure the focal plane in within the solution. Move the stage to the well containing fluorescein, turn off brightfield, cover the stage with a black dark fabric, and select L100 on the microscope to connect the light path to the detectors.</li><li>Within the VistaVision software, select âTD-FLIMâ under âMeasurementâ on the left hand side, this is time-domain FLIM using the method of time-correlated single photon counting (Osterlund <em>et al.</em>, 2015).<br/><em>Note: Use Figure 2 as a reference for the correct buttons or location of certain settings within VistaVision. The red numbers indicate which step itâs referring to.</em><br/><br/><img alt="" class="layerphoto" height="118" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015002655_7323.jpg" width="100"/><br/><strong>Figure 2. Screenshot of VistaVision software for acquiring FLIM-FRET data.</strong> Note that red numbers within the figure refer to the corresponding step in the protocol.<br/><br/></li><li>Next, set the 445 nm and 514 nm lasers to ~70%. We often cycle through the filters under âMotor Alignmentâ to ensure they are in their correct positions. Set Channel-2 to the 479/40 nm filter and Channel-1 to the 542/27 nm filter. Filter #1 is a multi-band filter which allows the light of three different wavelengths to pass, enabling the examination of the three most common fluorescent proteins (cerulean, green and red fluorescent protein). Filter #4 will pass the emission of Venus and reflect the light of mCer3, resulting in the signal of Venus for Channel-1 (Ch1) and mCer3 for Channel-2 (Ch2).</li><li>Under the âMulti-well Plateâ tab, select the correct plate-map configuration as your experiment. For ours, we are using a 384 well plate from Corning as it works for the PerkinElmer plate.</li><li>Under âDevice Monitorâ, select the small microscope icon, and select Alba to direct light from the microscope to the instrument.</li><li>Select the green circle directly below to open the shutters. You should now be able to see the photon counts from Ch1 and Ch2 directly below. We often strive to achieve photon counts between 1 x 10<sup>5</sup> â 1.5 x 10<sup>5</sup> for both channels by adjusting the laser power or using the âMotor Alignmentâ to adjust the pin hole location. Using either one of these tools to achieve consistent photon counts between experiments means the data collected from multiple experiments can be directly comparable. Record the photon counts for Ch1 and Ch2 then uncheck the green circle to close the shutters.</li><li>The next step is to ensure we are measuring the correct lifetime for fluorescein. Under the âScan Settingsâ tab, change the pixel dwell time to 0.5 (ms), and averaging mode set to ânoneâ. Also, uncheck both âPosition Seriesâ and âMacro Position seriesâ directly below.</li><li>Click âfilenameâ and enter the name of âreferenceâ.</li><li>Finally click âStartâ. A new window should appear on the right-hand side. After image acquisition, please follow Steps A3-A6 under Data Analysis, A. VistaVision Analysis, to check if the instrument can correctly measure fluoresceinâs lifetime (4 +/- 0.1 ns). In addition to lifetime, we also use this fluorescein image to check for homogeneity of intensity. Being a pure solution of fluorescein, we expect the intensity to be homogenous, or at the very least, well centered. If itâs not, you may consider changing the X and Y position for the field of view as seen in step 16 of Figure 1.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Live cell image acquisition on ISS<br/><em>Note: From this step we enter the process to collect data for cells and regions of interest.</em> <ol class="npwtwo" start="14"><li>Next, on the microscope turn on the brightfield, reset the stage from 100 back to 0 (focal height of cells), and move to your most top-left well containing your cells (Since the stage moves left to right, up to down to collect data). Confirm you are focused on your cells in the most top-left view within the most top-left well using the eyepiece and brightfield. Press âZero resetâ on the microscope, select âScanner Auto-Focusâ to lock the focal plane. Finally, press âL100â to switch from the brightfield to the detector.</li><li>Within VistaVision, select your wells of interest under âMacro Position Seriesâ (always refer your starting well as A1) then your fields of view within each well under âPosition seriesâ.</li><li>Select Z-reset under the âStage and scannersâ tab, then check âUse Scanner Auto-Focusâ.</li><li>Finally, change the file name to a name that makes sense to store your data (for example the name of the cells and experiment) and click âstartâ. By the end of the experiments, a â. issâ file will be automatically stored in your selected path.</li><li>Be sure to check the water supply has enough reservoir for the time of image acquisition.</li></ol></li></ol><p class="pptt" id="biaoti36757">### Data analysis</p><ol class="npwone"><li>VistaVision analysis<ol class="npwtwo"><li>The three files we need for downstream analysis are the intensities from Ch1 (Venus), Ch2 (mCer3), and the lifetime for mCer3 (Tau). Ch1 and Ch2 tiff files have been automatically extracted and saved with image acquisition. The Tau file can be extracted using VistaVision offline version (which means not necessarily connected with the microscope). Open VistaVision software, and make sure to select the offline mode under âDevice Configurationâ if you are using a separate computer. Click continue.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015002744_8182.jpg" width="161"/><br/><strong>Figure 3. Screenshot of VistaVision software for the analysis of Tau files.</strong> The Red numbers within the figure refer to the corresponding step in the protocol.<br/><br/></li><li>Next, drag and drop the saved ISS file into VistaVision. Under file type, it should say (ISS Fluorescence Lifetime Image Package).</li><li>Your ISS data should look very similar to Figure 3. Click the âSet intensity threshold for lifetime analysisâ button on the left hand side, and input a minimum of 80 for Ch-2. Click okay. Pixels with a Ch2 intensity less than 80 will be discarded from the lifetime data (this variable may change based on your own fluorescence data).<br/><em>Note: Use Figure 3 as a reference for the following steps. The red number indicates which step it refers to.</em> </li><li>Next, click the âlifetime fitâ button found on the left column.</li><li>On the right-hand side under the lifetime fitting select the following from their respective dropdown menus; select Ch2 since we are concerned with mCer3 lifetime, select series to apply settings to all images in this stack, select Bin of 3 (this means photons from neighbor pixels will be binned together to fit lifetime). Finally, input T1 and T2 gates as 12 and 96 respectively.<br/><em>Note: T1 and T2 gates are correlated with the PIE time setting for this microscope. We limit the gate scope after mCer3 excitation and before the disappearance of the emission.</em> </li><li>Directly below, set Comps to 1, and click the green âplayâ button.<br/><em>Note: If measuring your fluorescein reference, select âFrameâ instead of âseriesâ, the lifetime should appear directly below under âTauâ. Fluorescein lifetime should be 4ns (+/- 0.1 ns) for both Ch1 and Ch2.</em> </li><li>Once finished, click âExport Dataâ, select âExport Frame Analysis Dataâ, and select âComp-1 Tauâ and âChi-squareâ. Click the save button, and select the same folder where Ch1 and Ch2 tiff files are currently saved. Click okay.<br/><em>Note: The âChi-squareâ file is our quality record, which could be useful to troubleshoot if we notice abnormal data in later steps.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>ImageJ Macro<ol class="npwtwo"><li>For file organization, each transfection/treatment group should have its own designated folder. In this example, and as seen in Figure 4, all data is stored under the file named â<sup>V</sup>Badâ. Within this folder, you need to make four additional ones named âanalysisâ, âCh1â, âCh2â, âtauâ. Within each folder place the respective data renamed as the following; For Ch1 and Ch2 tiff image stacks, name them as â1_ch1â and â1_ch2â. If you have additional files from additional wells, they can be named as â2_ch1â and â2_ch2â. The lifetime tiff should be renamed to â1tâ, again additional files from additional wells can be renamed to â2tâ.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015002817_0654.jpg" width="143"/><br/><strong>Figure 4. This diagram illustrates how to store your files for the ImageJ script.</strong> Please note the correct spelling and capitalization of characters.<br/><br/></li><li>Next, open ImageJ software. Under âPluginsâ select âNewâ and then âMacroâ.</li><li>Copy the ImageJ macro found in the key resource table of Liu <em>et al.</em> (2019)., or found at this link: <a href="https://github.com/DWALab/Liu_et_al_2018_eLife/blob/master/ROILifetimeAnalysis.ijm" target="_blank">https://github.com/DWALab/Liu_et_al_2018_eLife/blob/master/ROILifetimeAnalysis.ijm</a>.<br/><em>Note: This ImageJ macro selects ROI (regions of interest) based on Ch2. As a result, the ROIs selected have different ratios of Ch1 to Ch2 intensities. ROIs with intensities too high or low are excluded from the downstream analysis as they may lead to inaccurate lifetime fitting.</em> </li><li>Within the open Macro window, click âMacroâ and then âRun Macroâ.</li><li>The macro will prompt you to âChoose Source Directoryâ. Select your â<sup>V</sup>Badâ folder (or whichever folder contains your data).</li><li>Input the number of images contained within your tiff stacks. Once you input this value, the script will automatically start to run. When the analysis is done, a new window will appear stating âAnalysis completeâ. Click okay and close ImageJ. <br/><em>Note: To figure out the number of images per stack, open either 1_ch1 or 1_ch2 within ImageJ, the number of images should be displayed on top of the image stack as 1/(number of images in stack), this is also the number of fields you imaged per well).</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Organizing/binning data in Microsoft Excel<ol class="npwtwo"><li>Within the folder â<sup>V</sup>Badâ, open the folder called âanalysisâ. The ImageJ script should have automatically saved your data into three separate excel files named; âCh1â, âCh2â and âtauâ. Copy the mean (column C) from each excel file, and paste into a new Excel workbook called âBin_analysisâ. Paste Ch1 data into Column A, Ch2 data into Column B, and tau data into Column C. Use row 1 for each column as a space to label the data. For each of these columns of data, each row represents the average values for a ROI.</li><li>Create a new Column D named âA/Dâ, which represents the acceptor to donor ratio. More specifically, the value obtained from taking the mean intensity for Venus divided by the mean intensity of mCer3 for each ROI. The acceptor/donor ratio acts as a surrogate for the relative amount of protein concentrations used to plot binding curves. In column D (âA/Dâ) take Ch1 data and divide it by Ch2 (<em>i.e.</em>, In cell D2, type: = A2/B2), and extend this to the bottom of the Excel workbook. </li><li>Now, in cell E1, assign the name âBinâ. This column âBinâ assigns each ROI to a different acceptor/donor ratio bin for averaging and plotting data. To do this, copy and paste the following IF statement into cell E2 and extend it to the bottom of the data [= IF(D2 &gt; 0.01,IF(D2 &gt; 0.05,IF(D2 &gt; 0.1,IF(D2 &gt; 0.2,IF(D2 &gt; 0.3,IF(D2 &gt; 0.4,IF(D2 &gt; 0.5,IF(D2 &gt; 0.75,IF(D2 &gt; 1,IF(D2 &gt; 1.5,IF(D2 &gt; 2,IF(D2 &gt; 2.5,3,2.5),2),1.5),1),0.75),0.5),0.4),0.3),0.2),0.1),0.05),0.01)].</li><li>Finally, in cell F1, name this column âFRETâ as it will contain the calculated FLIM-FRET, the average value of energy transfer that occurred for each ROI. This value is calculated by the following formula: <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015020720_0355.jpg" width="150"/><br/><br/>where,<br/>E is FLIM-FRET efficiency, <em>ti</em> is the mean lifetime for the ROI, and <em>t0</em> is the average lifetime of mCer3 in all ROIs not expressing Venus (acceptor protein). In cell F2, copy and paste the following formula: [= (1-(C2/3.8)) x 100], and extend this to the bottom of the data.</li><li>In a new sheet (sheet 2), select well A1 and insert âPivot tableâ. Select all the data from the first sheet (Columns A-F). Under Rows select âBinsâ. Drag âFretâ to the Values box, select the dropdown tab, and Value field settings. From here you can select the âAverageâ value. This will then display the average FRET value for each bin.</li><li>Repeat step 5 but this time select âCountâ instead of âaverageâ, to have an idea how many ROIs contribute to each Bin.</li><li>Repeat step 5 again, but this time select âStdDevâ, to know the standard deviation of FRET values in each bin.</li><li>Finally, in cell E1 (of sheet 2 beside the pivot table), label this column as âSEMâ. We plot our FLIM-FRET data using standard error of the mean (SEM), a value not available in Excelâs pivot tables. In cell E2 type the following formula: [= D2/(SQRT(C2))]. Extend this down to the bottom of the pivot table.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Plotting data with GraphPad Prism<ol class="npwtwo"><li>Open GraphPad Prism version 7.04. Select âXYâ under "New table &amp; graphâ. Under âEnter/import dataâ select âX: numbersâ and âY: enter and plot values calculated elsewhere; mean, SEM, Nâ. Click âcreateâ.</li><li>On the left hand side, Data table 1 is automatically selected for you. Input your Acceptor/donor ratio bin data (Column A on your Excel pivot table) under column âXâ in Graphpad. Copy and paste mean FLIM-FRET efficiency (column B of your pivot table) into âMeanâ column next to âXâ in GraphPad. Column E in Excel is your calculated standard error of the mean âSEMâ for FRET values. Copy Column E and paste it under the âSEMâ labeled column in GraphPad. Finally, copy Column C, the number of ROIs per bin, and paste this data under the column labeled âNâ in GraphPad.</li><li>Within GraphPad on the left hand side, select Data1 under the âGraphsâ folder. Your data should appear on a simple XY-plot with unlabeled axes. Label the X-axis by selecting it and typing âRatio of Venus to mCer3â. Label the Y-axis by selecting it and typing âFLIM-FRET Efficiency %â.</li><li>The next step is to fit the data. Under the âAnalysisâ tab at the top, select âFit a curve with nonlinear regressionâ. The new window that appears is called âParameters: Nonlinear Regressionâ. Within this new window select the tab called âFitâ and select âOne site-Specific binding with Hill slopeâ.</li><li>Next, select the tab called âCompareâ and select the question âDo the best-fit values of selected unshared parameters differ between data sets?â. Below a new box should appear labeled âChoose one or more parametersâ. Select the âBmaxâ box. <br/><em>Note. This step should only be used when all data sets are from the same protein [or slightly changed mutants like </em><sup><em>V</em></sup><em>Bad-2A which has 2 amino acids changed via site-directed mutagenesis (Liu </em><em>et al.</em><em>, 2019), since we expect the Bmax value to be the same for all.</em> </li><li>Under the âConfidenceâ tab, select âCalculate CI of parametersâ and select the confidence interval of 95%. Below under âConfidence or prediction bandsâ, check the box to âPlot confidence/prediction bandsâ, and again select the 95% value. Also ensure the box for âShow SE of parametersâ is checked. Finally, click okay to fit your data.</li><li>Your data should now be fitted with the Hill slope equation and display the 95% confidence interval for this fit. A new sheet is created on the left hand side under âResultsâ and should be labeled as âNonlin fit of Data1â. If you go back to the âData tableâ and input the value of 0.5 under the âXâ column, an additional folder will appear under âResultsâ and be labeled as âInterpolated Y valuesâ.</li><li>Repeat all the above steps under âData Analysisâ for your other FLIM-FRET groups, plotting their data within the same GraphPad file. In our example, our groups were Venus-Bad (<sup>V</sup>Bad), <sup>V</sup>Bad + 10 Î¼M ABT-263, and <sup>V</sup>Bad-4E. Mutating the BH3 domain of <sup>V</sup>Bad (specifically the key hydrophobic residues h1-h4) has previously been reported to significantly inhibit the binding with Bcl-XL and Bcl-2 in live cells (Aranovich <em>et al.</em>, 2012; Liu <em>et al.</em>, 2019). Importantly, this mutation does not disrupt the membrane binding domain of Bad, making <sup>V</sup>Bad-4E an excellent control to account for the FLIM-FRET efficiency measured due to random collisions on subcellular membranes (Wilfling <em>et al.</em>, 2012). As seen in Figure 5, <sup>V</sup>Bad-4E has significantly reduced FLIM-FRET efficiency compared to <sup>V</sup>Bad (Green line). The addition of 10 Î¼M ABT-263 to <sup>V</sup>Bad (Blue line) reduces the FLIM-FRET efficiency, however, it remains higher than <sup>V</sup>Bad-4E. A reduction in FLIM-FRET efficiency from the addition of BH3-mimetic suggests a reduction in protein binding with mCer3-Bcl-XL. Since the FLIM-FRET efficiency remains higher than the non-binding mutant (<sup>V</sup>Bad-4E), we define the remaining percentage of FLIM-FRET efficiency as resistance to displacement from the specified 10 Î¼M of ABT-263. <br/><em>Note: For FLIM-FRET experiments, itâs imperative to have negative controls. Bcl-2 family proteins localize to mitochondrial membranes, and as a result, have locally high concentrations. This increases the frequency of random collisions that can contribute to the measured FLIM-FRET efficiency. Because </em><sup><em>V</em></sup><em>Bad-4E binds to subcellular membranes, but no longer binds to Bcl-XL, it is an excellent negative control. </em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015020836_9638.jpg" width="138"/><br/><strong>Figure 5. Final result plotting data in GraphPad Prism.</strong> Note that the addition of 10 Î¼M ABT-263 reduces the FLIM-FRET efficiency (%) (compare green to blue lines). In addition, introducing four glutamic residues to the BH3-domain of Bad (<sup>V</sup>Bad-4E) completely abrogates all binding with mCer3-Bcl-XL (Red line) as shown in Liu <em>et al.</em> (2019).<br/><br/></li><li>To quantify resistance to displacement (Resistance (%)), we compare the FLIM-FRET efficiency (%) between <sup>V</sup>Bad, and <sup>V</sup>Bad + 10 Î¼M ABT-263 at the ratio of Venus to mCer3 of 0.5. To find the FLIM-FRET efficiency (%) value at this acceptor/donor ratio, input the value of 0.5 in the âdata tableâ (in the ratio column below all raw data), then select the tab under âResultsâ called âInterpolate Y valuesâ. The FLIM-FRET efficiency (%) value will be calculated automatically according to the fit model.<br/><em>Note: The standard error of the mean describes the accuracy of our measurement while the 95% confidence interval describes how well our data fits the model.</em> </li><li>To calculate Resistance (%), we use the following formula:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015020916_7091.jpg" width="150"/><br/><br/>where, R denotes Resistance (%), A is the interpolated mean FLIM-FRET efficiency value for <sup>V</sup>Bad + DMSO, B is the interpolated mean FLIM-FRET efficiency value for <sup>V</sup>Bad + 10 Î¼M ABT-263, and C is the interpolated mean FLIM-FRET efficiency value for <sup>V</sup>Bad-4E.</li><li>In our example, we calculated a resistance of 63% at the dose used of 10 Î¼M ABT-263.</li></ol></li></ol><p class="pptt" id="biaoti36758">### Recipes</p><ol class="npwtwok"><li>10 nM Fluorescein standard in NaOH<ol class="npwthree"><li>Dissolve 0.3765 g fluorescein disodium salt in 0.1 M NaOH up to 5 ml (0.2 Î¼M fluorescein stock)</li><li>Next, dilute this stock 1/20 for imaging. <em>i.e.</em>, Take 1 ml of this 0.2 Î¼M stock and mix with 19 ml 0.1 M NaOH to achieve 10 nM solution</li><li>Store these stocks at 4 Â°C, wrapped in tinfoil or tape to avoid light. Stocks can be stored up to 6 months</li></ol></li><li>Media for cells<ol class="npwthree"><li>Follow the manufacturerâs instructions for resuspending DMEM powder, then autoclave the solution</li><li>Next, to 450 ml of autoclaved DMEM media add 50 ml of Fetal Bovine Serum (FBS), achieving a final concentration of 10%</li><li>Finally, add 5 ml of non-essential amino acids (NEAA). Mix well, and store at 4 Â°C</li></ol></li></ol><p class="pptt" id="biaoti36759">### Acknowledgments</p><p class="ppzz">Research was funded by Canadian Institutes of Health Foundation grant FRN 12517 to DWA who also holds a Tier 1 Canada Research Chair in membrane biogenesis. The protocol described here is based on research previously published in Osterlund <em>et al.</em> (2015) and Liu <em>et al.</em> (2019).</p><p class="pptt" id="biaoti36760">### Competing interests</p><p class="ppzz">The authors declare that they have no financial or non-financial competing interests.</p><p class="pptt" id="biaoti36761">### References</p><ol class="npwtwok"><li>Aranovich, A., Liu, Q., Collins, T., Geng, F., Dixit, S., Leber, B. and Andrews, D. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22464442" target="_blank">Differences in the mechanisms of proapoptotic BH3 proteins binding to Bcl-XL and Bcl-2 quantified in live MCF-7 cells.</a><em> Mol Cell </em>45(6): 754-763.</li><li>Billen, L. P., Kokoski, C. L., Lovell, J. F., Leber, B. and Andrews, D. W. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18547146" target="_blank">Bcl-XL inhibits membrane permeabilization by competing with Bax.</a> <em>PLoS Biol </em>6(6): e147.</li><li>Hirmiz, N., Tsikouras, A., Osterlund, E. J., Richards, M., Kun, J., Andrews, D. W. and Fang, Q.(2019). <a href="https://www.spiedigitallibrary.org/conference-proceedings-of-spie/11076/1107617/Investigating-Bcl-2-family-protein-protein-interactions-using-a-high/10.1117/12.2526906.short?SSO=1" target="_blank">Investigating Bcl-2 family protein-protein interactions using a high-speed multiplexing confocal FLIM microscope.</a> In<em> Proc.SPIE.</em> Vol. 11076. </li><li>Hsu, Y. T. and Youle, R. J.( 1997). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9153240" target="_blank">Nonionic detergents induce dimerization among members of the Bcl-2 family. </a><em>J Biol Chem </em>272 (21): 13829-13834. </li><li>Liu, Q., Leber, B. and Andrews, D. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22895112" target="_blank">Interactions of pro-apoptotic BH3 proteins with anti-apoptotic Bcl-2 family proteins measured in live MCF-7 cells using FLIM FRET. </a><em>Cell Cycle </em>11(19): 3536-3542.</li><li>Liu, Q., Oesterlund, E. J., Chi, X., Pogmore, J., Leber, B. and Andrews, D. W. (2019). <a href="https://cdn.elifesciences.org/articles/37689/elife-37689-v2.pdf" target="_blank">Bim escapes displacement by BH3-Mimetic anti-cancer drugs by double-bolt locking both Bcl-XL and Bcl-2.</a> <em>ELife</em> 8:e37689.</li><li>Lovell, J. F., Billen, L. P., Bindner, S., Shamas-Din, A., Fradin, C., Leber, B. and Andrews, D. W. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19062087" target="_blank">Membrane binding by tBid initiates an ordered series of events culminating in membrane permeabilization by Bax.</a> <em>Cell</em> 135(6): 1074-1084.</li><li>Osterlund, E. J., Hirmiz, N, Tardif, C. and Andrews, D. W. (2019). <a href="https://link.springer.com/protocol/10.1007/978-1-4939-8861-7_19" target="_blank">Rapid imaging of BCL-2 family interactions in live cells using FLIM-FRET.</a> In <em>Methods in Molecular Biology.</em> Vol. 1877: 305-335.</li><li>Osterlund, E., J., Liu, Q. and Andrews, D. W. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25631031" target="_blank">The Use of Flim-Fret for the detection of mitochondrial protein-protein interactions.</a> <em>Methods Mol Biol</em> 1264: 395-419. </li><li>PÃ©cot, J., Maillet, L., Le Pen, J., Vuillier, C., TrÃ©cesson, S. C., FÃ©tiveau, A., Sarosiek, K. A., Bock, F. J., Braun, F., Letai, A., Tait S. W. G. ,Gautier, F. and Juin, P. P. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28009301" target="_blank">Tight sequestration of BH3 proteins by BCL-XL at subcellular membranes contributes to apoptotic resistance.</a> <em>Cell Rep </em>17 (12): 3347-3358. </li><li>Sun, Y., Hays, N. M., Periasamy, A., Davidson, M. W. and Day, R. N. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22264545" target="_blank">Monitoring protein interactions in living cells with fluorescence lifetime imaging microscopy. </a><em>Methods Enzymol </em>504: 371-391.</li><li>Wilfling, F., Weber, A., Potthoff, S., VÃ¶gtle, F. N., Meisinger, C., Paschen, S. A. and HÃ¤cker., G. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22343714" target="_blank">BH3-only proteins are tail-anchored in the outer mitochondrial membrane and can initiate the activation of Bax. </a><em>Cell Death Differ</em>19 (8): 1328-1336. <br/></li></ol>