<body>
            <h1>Quantification of HIV-2 DNA in Whole Blood</h1>
            <p><strong>Authors:</strong> ZsÃ³fia  Szojka, Sara  Karlson, Marianne Jansson, Patrik Medstrand</p>
            <h2>Abstract</h2>
            <p>Time to AIDS infection is longer with HIV-2, compared to HIV-1, but without antiretroviral therapy both infections will cause AIDS-related mortality. In HIV-2 infection, monitoring of antiretroviral treatment (ART) efficacy is challenging since a large proportion of HIV-2-infected individuals displays low or undetectable plasma RNA levels. Hence, quantification of cellular DNA load may constitute an alternative method for monitoring ART efficacy. Moreover, sensitive HIV-2 DNA quantification protocols are also important for the characterization of the HIV-2 reservoirs, and ultimately for the development of HIV-2 cure strategies. We have developed a sensitive and robust HIV-2 DNA quantification protocol based on whole blood as DNA source, including normalization of leukocyte cell numbers using parallel quantification of the single copy porphobilinogen deaminase gene. The specificity and sensitivity of the assay was 100%. The limit of detection was 1 copy and limit of quantification was 5 copies. When applying this protocol to HIV-2 infected, it was found that HIV-2 viral DNA was detectable in individuals in whom viral RNA was undetectable or under quantification level. Thus, this method provides a sensitive approach to HIV-2 DNA viral quantification from whole blood of HIV-2 infected patients.</p>
            <p><strong>Keywords:</strong> HIV-2, DNA load, Quantification, qPCR, Whole Blood</p>
            <h2>Background</h2><p>Acquired immunodeficiency syndrome (AIDS) can be caused by two viruses, either human immunodeficiency virus (HIV) type 1 (HIV-1) or type 2 (HIV-2) (Azevedo-Pereira and Santos-Costa, 2016). Although these two retroviruses have similar life cycles, they differ in that HIV-2 displays lower pathogenicity, translating into longer progression time to AIDS and reduced transmissibility (Marvin <em>et al.</em>,2015; Azevedo-Pereira and Santos-Costa, 2016). Another distinct feature of HIV-2 is that the plasma viral RNA load of infected individuals is significantly lower, compared with those infected with HIV-1 (Berry <em>et al.</em>, 1998; Popper <em>et al.</em>, 1999; Andersson <em>et al.</em>, 2000). The cellular viral DNA load in peripheral blood has, instead, been suggested to be similar between HIV-1 and HIV-2 infections (Ariyoshi <em>et al.</em>, 1996; Popper <em>et al.</em>, 2000; Damond <em>et al.</em>, 2001). However, others have indicated that also the DNA load is lower in HIV-2, compared with HIV-1, infection (Gueudin <em>et al.</em>, 2008). Overall, HIV-2 DNA load quantification studies are relatively few, and most studies have not considered a protocol for a house-keeping gene to quantify the number of cells used in the reaction (Ariyoshi <em>et al.</em>, 1996; Gomes <em>et al.</em>, 1999; Popper <em>et al.</em>, 2000; Damond <em>et al.</em>, 2001; Soares <em>et al.</em>, 2006; Gueudin <em>et al.</em>, 2008; Bertine <em>et al.</em>, 2017).<br/><br/>Despite the fact that the estimated time to AIDS development in HIV-2 infected individuals is longer than for individuals with HIV-1, it is clear that HIV-2 will cause AIDS-related mortality without antiretroviral therapy (ART) in infected individuals with long follow-up (Esbjornsson <em>et al.</em>, 2018). The monitoring of treatment efficacy is, however, challenging since a large proportion of ART-naÃ¯ve HIV-2 infected individuals display low or undetectable plasma RNA levels (Berry <em>et al.</em>, 1998; Popper <em>et al.</em>, 1999; Andersson <em>et al.</em>, 2000; Buggert <em>et al.</em>, 2016; Honge <em>et al.</em>, 2018). Hence, quantification of cellular DNA load may constitute an alternative method for monitoring of ART efficacy.<br/><br/>For simplicity, robustness and the requirement of small volumes of blood, viral DNA quantification protocols based on whole blood, as the source of DNA, are desirable. Currently there are, however, a limited number of publications where HIV-2 DNA quantification has been based on whole blood (Bertine <em>et al.</em>, 2017), and the need for protocols combining sensitivity with robustness is still important.<br/><br/> In a recent study (Esbjornsson <em>et al.</em>, 2018), we showed that both HIV-1-infected and HIV-2-infected individuals have a high probability of developing AIDS without antiretroviral treatment.<br/><br/>In HIV-1 infection it has been shown that viral DNA load is an independent marker of disease progression and is well correlated with the number of latently HIV-1 infected cells, that comprises the viral reservoir (Parisi <em>et al.</em>, 2012; Rouzioux and Avettand-Fenoel, 2018). Comprehensive studies on the HIV-2 reservoirs are, however, lacking. Thus, sensitive HIV-2 DNA quantifications protocols are also important for the characterization of the HIV-2 reservoirs, and ultimately for the development of HIV-2 cure strategies. We therefore decided to develop a sensitive and robust cellular HIV-2 DNA quantification method based on whole blood as DNA source, including normalization of leukocyte numbers using parallel quantification of the single copy porphobilinogen deaminase gene (PBDG) (Raich <em>et al.</em>, 1986; Mbisa <em>et al.</em>, 2009).</p>
        </body>

<p class="pptt" id="biaoti29989">## Materials and Reagents</p><ol class="npwone"><li>DNA extraction<ol class="npwtwo"><li>Filter tips (1,000 Î¼l filter tip: Sarstedt, catalog number: 70.762.211; 200 Î¼l filter tip: Sarstedt, catalog number: 70.760.211; 20 Î¼l filter tip: Sarstedt, catalog number: 70.760.213)</li><li>Low Binding Eppendorf tubes (DNA LoBind Snap Cap PCR Tube, 1.5 ml) (Eppendorf, catalog number: 022431021)<br/></li><li>PM1 cells (Lusso <em>et al.</em>, 1995)</li><li>Whole Blood specimen of HIV-2 infected individuals, storage in -70 Â°C freezer. Collect samples in EDTA tubes (BD Life Sciences, Vacutainer spray-coated K2EDTA tubes, catalog number: 367841)</li><li>Whole Blood specimen of uninfected individuals, storage in -70 Â°C freezer. Collect samples in EDTA tubes (BD Life Sciences, Vacutainer spray-coated K2EDTA tubes, catalog number: 67841)</li><li>Carrier RNA (poly A) (1350 Î¼g) (Qiagen, catalog number: 1017647) store at room temperature before use, then store at -20 Â°C</li><li>Gibco<sup>TM</sup> RPMI 1640 Medium (Thermo Fisher Scientific, Fisher Scientific, catalog number: 11875093), store at 4 Â°C before opening the bottle</li><li>Gibco<sup>TM</sup> Fetal Bovine Serum (FBS) (Thermo Fisher Scientific, Fisher Scientific, catalog number: 26140079), storage in -20 Â°C freezer before heat inactivation, then portion it and store at -20 Â°C</li><li>Gibco<sup>TM</sup> Penicillin-Streptomycin (10,000 U/ml) (Thermo Fisher Scientific, Fisher Scientific, catalog number: 15140122), store at -20 Â°C</li><li>QIAamp DNA Mini Kit (Qiagen, catalog number: 51304), store at room temperature</li><li>QIAamp Blood DNA Mini Kit (Qiagen, catalog number: 51104), store at room temperature</li><li>Gibco<sup>TM</sup> Phosphate buffered saline (PBS, pH = 7.4) (Thermo Scientific, Fisher Scientific catalog number: 10010023), store at room temperature</li><li>Complete RPMI-1640 medium (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of Standards<ol class="npwtwo"><li>Filter tips (1,000 Î¼l filter tip: Sarstedt, catalog number: 70.762.211; 200 Î¼l filter tip: Sarstedt, catalog number: 70.760.211; 20 Î¼l filter tip: Sarstedt, catalog number: 70.760.213)</li><li>Pipette Tips (1,000 Î¼l filter tips: Sarstedt, catalog number: 70.762.100; 200 Î¼l filter tips: Sarstedt, catalog number: 70.760.502; 10 Î¼l tips: Sarstedt, catalog number: 70.1130)</li><li>Low Binding Eppendorf tubes (DNA LoBind Snap Cap PCR Tube, 1.5 ml) (Eppendorf, catalog number: 022431021)</li><li>MicroAmp<sup>TM</sup> Optical 8-Tube Strip, 0.2 ml (Thermo Scientific, Applied Biosystems<sup>TM</sup>, catalog number: 4316567)</li><li>MicroAmp<sup>TM</sup> Optical 8-Cap Strips (Thermo Scientific, Applied Biosystems<sup>TM</sup>, catalog number: 4323032)</li><li>HIV-2 (NIH-Z Strain) Purified Virus and electron microscopy counted (Advanced Biotechnologies Inc, catalog number: 10-127-000), storage in -70 Â°C freezer</li><li>Qiagen miRNeasy micro Kit (Qiagen, catalog number: 217084), store at 4 Â°C</li><li>DNA oligonucleotides primers were obtained from Invitrogen and listed in Table 1, store oligonucleotides at -20 Â°C</li><li>SuperScript<sup>TM</sup> III One-Step RT-PCR System with Platinum<sup>TM</sup> <em>Taq</em> DNA Polymerase (Thermo Fisher Scientific, Invitrogen, catalog number: 12574018), storage in -20 Â°C freezer. SuperScript<sup>TM</sup> III One-Step RT-PCR System is supplied with the following:<br/><ol class="npwthree"><li>SuperScript<sup>Â®</sup> III RT/Platinum<sup>Â®</sup> <em>Taq</em> Mix (50 Î¼l)</li><li>2x Reaction Mix (containing 0.4 mM of each dNTP, 3.2 mM MgSO<sub>4</sub>) (1 ml)</li><li>5 mM Magnesium Sulfate (500 Î¼l)</li></ol></li><li>Platinum<sup>TM</sup> <em>Taq</em> DNA Polymerase High Fidelity (Thermo Fisher Scientific, Invitrogen, catalog number: 11304011), storage in -20 Â°C freezer. Platinum<sup>Â®</sup> Taq DNA Polymerase High Fidelity (20 Î¼l, at 5 U/Î¼l) is supplied with the following:<ol class="npwthree"><li>10x High Fidelity Buffer [600 mM Tris-SO<sub>4</sub> (pH 8.9), 180 mM (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>] (1.25 ml)</li><li>50 mM MgSO<sub>4</sub> (1 ml)</li></ol></li><li>Agarose (Thermo Fisher Scientific, Invitrogen, catalog number: 16500500), store at room temperature</li><li>Milli-Q quality water (RNase, DNase free water [dH<sub>2</sub>O]) (VWR, catalog number: SH30538.02), store at room temperature</li><li>AccuGENE<sup>TM</sup> 50x TAE Buffer (Lonza, catalog number: 51216), store at room temperature</li><li>Gel Red (VWR International, catalog number: 41003), store at room temperature</li><li>QIAquick PCR purification Kit (Qiagen, catalog number: 28104), store at room temperature<br/><br/><strong>Table 1. Primers used for HIV and </strong><em><strong>PBDG</strong></em><strong> standard preparation</strong><br/><img alt="" class="layerphoto" height="488" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015210715_8020.jpg" width="600"/><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>qPCR for HIV-2 DNA detection<ol class="npwtwo"><li>Pipette tips (1,000 Î¼l filter tips: Sarstedt, catalog number: 70.762.100; 200 Î¼l filter tips: Sarstedt, catalog number: 70.760.502)</li><li>96-well qPCR plates (Applied Biosystems<sup>TM</sup>, catalog number: 4346906)</li><li>Optical Adhesive Film (Applied Biosystems<sup>TM</sup>, catalog number: 4360954)</li><li>HyClone<sup>TM</sup> Water, Molecular Biology Grade (RNase, DNase, proteinase free) (Thermo Fisher Scientific, Fisher Scientific, GE Healthcare HyClone SH30538.FS, catalog number: 10247783), store at room temperature</li><li>HIV-1 infected U937 Cells (U1) (Folks <em>et al.</em>, 1987)</li><li>Maxima probe qPCR Master Mix (2x) (Thermo Scientific, catalog number: K0231), storage in -20 Â°C freezer. qPCR Master Mix contains the following:<br/><ol class="npwthree"><li>2x Maxima Probe/ROX qPCR Master Mix containing Maxima Hot Start <em>Taq</em> DNA Polymerase and dNTPs (1.25 ml)</li><li>Nuclease-Free Water (1.25 ml)</li></ol></li><li>Primer and probe sequences for HIV-2 was obtained from Damond <em>et al.</em> (2002) (PMID: 12354861). Storage in -20 Â°C freezer</li><li>Primer and probe sequences of the housekeeping human gene PBGD was obtained from Mbisa <em>et al.</em> (2009) (PMID 19020818). Storage in -20 Â°C freezer</li><li>Agarose (Thermo Fisher Scientific, Invitrogen, catalog number: 16500500), store at room temperature</li><li>Milli-Q quality water (RNase, DNase free water [dH<sub>2</sub>O]) (VWR, catalog number: SH30538.02), store at room temperature</li><li>AccuGENE<sup>TM</sup> 50x TAE Buffer (Lonza, catalog number: 51216), store at room temperature</li><li>Gel Red (VWR International, catalog number: 41003), store at room temperature</li><li>1 Kb Plus DNA Ladder (Thermo Fisher Scientific, Invitrogen, catalog number: 10787018), storage in -20 Â°C freezer</li></ol></li></ol><p class="pptt" id="biaoti29990">### Equipment</p><ol class="npwtwok"><li>EVE<sup>TM</sup> Automated Cell Counter, NanoEnTek (VWR, Avantor, catalog number: 10027-452)</li><li>EVE<sup>TM</sup> Cell Counting Slides with Trypan Blue (VWR, Avantor, catalog number: 10027-446)</li><li>Eppendorf Microcentrifuge, model: 5415D, with rotor F 45-24-11, AC/DC input 230 V AC, 50-60 Hz (Sigma-Aldrich, catalog number: Z604062)</li><li>Life Eco Thermal Cycler, 96 well gradient, 100-240V, 50-60Hz, 600W, (Bioer Technology Co., catalog number: BTC42096)</li><li>Gilson single-channel/adjustable-volume pipettes: 0.1-2 Î¼l, 2-20 Î¼l, 20-200 Î¼l, 100-1,000 Î¼l (Eppendorf, catalog number: F167380)</li><li>Applied Biosystems StepOnePlus Real-Time PCR System (Thermo Scientific, Applied Biosystems<sup>TM</sup>, catalog number: 4376600)</li><li>NanoDrop ND-1000 spectrophotometer (Thermo Scientific, catalog number: ND-2000C)</li><li>Mini ReadySub-Cell<sup>TM</sup> electrophoresis system (Bio-Rad Laboratories, catalog number: 1704467)</li><li>UV-Transparent gel tray (Bio-Rad Laboratories, catalog number: 1704435)</li><li>8-well Comb (Bio-Rad Laboratories, catalog number: 1704463)</li><li>Gel Doc<sup>TM</sup> XR+ Gel Documentation System (Bio-Rad Laboratories, catalog number: 1708195)<br/></li></ol><p class="pptt" id="biaoti29991">### Software</p><ol class="npwtwok"><li>Image Analysis Software, Image Lab<sup>TM</sup> Software (Life Sciences, Bio-Rad)</li><li>Excel 2016 (Microsoft)</li><li>GraphPad Prism 7.0.<br/></li></ol><p class="pptt" id="biaoti29992">### Procedure</p><p class="ppzz"><strong>Main steps of preparation of HIV-2 qPCR:</strong> </p><ol class="npwtwo"><li>Extract DNA from PM1 cells as outlined below Steps A1 and A2.</li><li>Prepare <em>PBDG</em>, cellular controls as indicated in Procedure B âPreparation of <em>PBDG</em> standardâ by using extracted DNA of PM1 cells.</li><li>Extract viral RNA -from HIV-2 isolate- to prepare HIV-2 standards, as mentioned in Procedure B âPreparation of HIV-2 standardâ.</li><li>Extract viral DNA from whole blood as indicated in Step A3.</li><li>Set up HIV-2 qPCR as outlined below Procedure C.<br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwone"><li>DNA extraction<ol class="npwtwo"><li>Collect 5 x 10<sup>6</sup> PM1 cells (Lusso <em>et al.</em>, 1995) in 1.5 ml Eppendorf tubes and centrifuge for 5 min at 200 <em>x g</em>. Discard supernatant and wash pellets twice in 1 ml of sterile PBS. Store dried pellet at -20 Â°C until DNA extraction.<br/><em>Notes:</em> <ol class="npwthree"><li><em>PM1 cell line is a modified CD4+ T-cell clone, which expresses co-receptors such as CCR5 and CXCR4 (Lusso </em><em>et al.</em><em>, 1995).</em> </li><li><em>Maintain PM1 cells in complete RPMI-1640 (Invitrogen) (see Recipes) until passage number 3, after defreezing. Count cells by using Automated Cell Counter.</em> </li><li><em>U1s are subclones of U937 chronically HIV-1 infected cells (Folks </em><em>et al.</em><em>, 1987). These cells are carrying integrated HIV-1 in their genome, thus extracted DNA can be used as a negative control, and to detect the specificity.</em> </li><li><em>For negative control, collect 5 x 10</em><sup><em>6</em></sup><em> U1 cells (Folks </em><em>et al.</em><em>, 1987) as the same way as it is mentioned above for PM1 cells. And count cells by using Automated Cell Counter.</em> </li></ol></li><li>To extract cellular DNA of PM1 cells use Qiagen QIAamp DNA Mini Kit according to the manufacturerâs instructions. A single elution of 50 Î¼l buffer AE is used.<br/><em>Notes:</em> <ol class="npwthree"><li><em>DNA extractions should be performed in a cell biology-grade clean hood equipped with separate air-handling mechanisms or under a DNA-free hood. Add extra centrifugation step before elution to decrease the EtOH contamination: 20,000 x g for 1 min. For elution use prewarmed Buffer AE (56 Â°C) and incubate it for 2 min at room temperature.</em> </li><li><em>For negative control, extract DNA from collected U1 cells (Folks </em><em>et al.</em><em>, 1987) by using Qiagen QIAamp DNA Mini Kit according to the manufacturerâs instructions, then elute in 50 Î¼l buffer AE.</em> </li></ol></li><li>Extract viral DNA from 200 Î¼l of whole blood in the presence of 5 Î¼g carrier RNA using Qiagen QIAamp Blood DNA Mini Kit according to the manufacturerâs instruction. Load DNA to spin column and elute in 50 Î¼l elution buffer.<br/><em>Notes</em> <ol class="npwthree"><li><em>DNA extractions should be performed in a biosafety cabinet.</em> </li><li><em>Whole Blood sample should be collected in EDTA tubes and stored at -70 Â°C until DNA extraction.</em> </li><li><em>To reduce degradation of DNA, avoid freezing/thawing of the whole blood samples.</em> </li><li><em>Extraction should be performed by using low-binding Eppendorf tubes.</em> </li></ol></li><li>Store DNA extracts at -20 Â°C for up to a few weeks.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of Standards<br/><strong><em>Preparation of HIV-2 standard</em></strong> <ol class="npwtwo"><li>Prepare the HIV-2 standard by extracting RNA from 1.44 x 10<sup>11</sup> RNA-copies/ml of electron microscopy counted HIV-2 particles using Qiagen miRNeasy micro Kit according to the manufacturerâs instructions with minor modifications.<br/><em>Notes: </em> <ol class="npwthree"><li><em>HIV-2 control stock solution has a concentration of 7.2 x 10<sup>10</sup> viral particles/ml. As two RNA genomes are packed in each viral particle the final stock contains: 1.44 x 10<sup>11</sup> RNA-copies/ml = 1.44 x 10<sup>8</sup> copies/Î¼l (7.2 x 10<sup>10</sup>multiply with two). Extract a sample (HIV-2) with 5 Î¼l from the stock + 95 Î¼l PBS. Extracted RNA should be eluted in 30 Î¼l. In this case, the concentration of sample is 2.4 x 10<sup>7</sup> copies/Î¼l ((1.44 x 10<sup>8</sup> x 5)/30). Use 10-fold dilution series to reach 5,000 copies/Î¼l RNA concentration.</em> </li><li><em>By using miRNeasy micro Kit purification gives a better yield and allows elution in smaller volume.</em> </li><li><em>To increase the yield of extracted RNA: increase the amount or qiazol reagent from 700 Î¼l to 1000 Î¼l. Double incubation time (10 min instead of 5 min). Use 200 Î¼l of chloroform instead of 140 Î¼l during extraction.</em> </li><li><em>To improve the yield of extraction add 5 Î¼l carrier RNA (1 Î¼g/ Î¼l) before and after transferring the aqueous phase.</em> </li><li><em>Wash it twice by using 350 Î¼l RWT buffer and between washing steps add DNase to decrease the amount of DNA contamination.</em> </li><li><em>Elute RNA in 30 Î¼l of RNase-free water.</em> </li></ol></li><li>Perform cDNA synthesis directly after RNA extraction by using SuperScript III One-Step RT-PCR System and Platinum <em>Taq</em> DNA polymerase. Prepare cDNA synthesis in 25 Î¼l reaction volume containing 2x reverse transcription buffer (0.4 mM of each dNTP, 3.2 mM MgSO<sub>4</sub>), 1 Î¼l of eluted total RNA, 100 ng forward and reverse primers, SuperScript<sup>TM</sup> III One-Step RT-PCR System with Platinum<sup>TM</sup> <em>Taq</em> DNA Polymerase and RNase/DNaseâfree sterile water up to the final volume (see Table 2).<br/><br/><strong>Table 2. Preparation of cDNA synthesis by reverse transcription and first round nested PCR</strong><br/><img alt="" class="layerphoto" height="232" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015210838_5975.jpg" width="600"/><br/><br/></li><li>Quantify the HIV-2 <em>gag</em> fragment of RT-PCR (850 base pairs) using the following program: initial cDNA synthesis for 30 min at 50 Â°C, followed by amplification: initial denaturation for 2 min at 94 Â°C, 35 cycles of 15 s at 94 Â°C, 30 s at 50 Â°C, 45 s at 68 Â°C and a final elongation step for 5 min at 68 Â°C.<br/><em>Note: By using primer pair BEN_740F and BEN_1640R plus strand of HIV-2 </em><em>gag</em><em> can be amplified. The estimated size of the amplified fragment is 850 base pairs.</em> </li><li>Visualize PCR products on 1% agarose gels stained with Gel Red, run at 110 V for 45 min in 1x TAE buffer.<br/><ol class="npwthree"><li>Purify PCR Product by using QIAquick PCR purification Kit according to the manufacturerâs instruction. Load DNA to spin column and elute in 50 Î¼l sterile water. Quantify the concentration of PCR product by using NanoDrop ND-1000 spectrophotometer and visualize it on 1% agarose gel with the same running conditions (see Figure 1).</li><li>Purified HIV-2 <em>gag</em> should be stored at -20 Â°C until needed.<br/><br/><img alt="" class="layerphoto" height="277" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015210926_6624.jpg" width="100"/><br/><strong>Figure 1. Detection of cleaned PCR product of HIV-2 </strong><em><strong>gag</strong></em><strong> PCR.</strong> An example of successful of PCR purification is visualized on 1% Agarose gel stained with Gel red. L = 1 KB DNA ladder, 1 and 2 = HIV-2 PCR products, with the estimated size of 850 bp.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol><strong><em>Preparation of </em></strong><strong><em>PBDG</em></strong><strong><em> standard</em></strong> <ol class="npwtwo"><li>Perform host gene standard (<em>PBDG</em> fragment) amplification in two steps (outer followed by nested PCR). Prepare outer PCR in 25 Î¼l solution containing: 10 ng eluted DNA of PM1 cells, Dream <em>Taq</em> DNA polymerase, 5 mM dNTP mix and 100 ng/Î¼l primers (PBGD-CF1 and PBGD-CR2) (see Table 3). Visualize PCR product on 1% agarose gel stained with Gel Red, run at 110 V for 45 min in 1x TAE buffer.</li><li>Reaction of nested PCR solution should contain: 1 Î¼l of outer PCR product, Dream <em>Taq</em> DNA polymerase, 5 mM dNTP mix and 100 ng/Î¼l primers (PBGD-CF1 and PBGD-CR1). Detect PCR product on 1% agarose gel stained with Gel Red at the same conditions as aforementioned.<br/><br/><strong>Table 3. Host gene performed by 2-step PCR</strong><br/><img alt="" class="layerphoto" height="294" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211012_4962.jpg" width="600"/><br/><br/></li><li>Outer and Nested PCR cycling parameters: an initial denaturation 2 min at 94 Â°C followed by 35 cycles of 30 s at 94 Â°C, 30 s at 55 Â°C, 45 s at 72 Â°C and a final elongation step for 5 min at 72 Â°C.</li><li>Purify PCR product by using QIAquick PCR purification Kit according to the manufacturerâs instruction. Load DNA to spin column, elute in 50 Î¼l sterile water and quantify the concentration of PCR product by using NanoDrop ND-1000 spectrophotometer and detect the product on 1% agarose gel with the same running conditions as mentioned before.<br/><em>Notes:</em> <ol class="npwthree"><li><em>For RNA elution use RNase/ DNase/ proteinase free water.</em> </li><li><em>cDNA synthesis and One-Step PCR reactions are conducted in 0.2 ml, MicroAmp</em><sup><em>TM</em></sup><em> Optical 8-Tubes.</em> </li><li><em>Stock and dilutions must be kept on ice after thawing and mixing, and frozen at -20 Â°C when not in use. Pipette mix or flick mix samples and dilutions, do not vortex to mix.</em> </li><li><em>RNA extractions, dilution of RNA and DNA, PCR setups must be done in the amplicon-free room and should be performed using filtered pipette tips.</em> </li><li><em>Modified PCR protocol of Damond </em><em>et al.</em><em> (2002) is used for HIV-2 qPCR.</em> </li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>qPCR for HIV-2 DNA detection<br/>Main steps of the qPCR (see Figure 2 for an example of experiment layout): 1) Prepare Real-Time PCR reaction of HIV-2 from extracted DNA of infected and uninfected individuals and with sterile water instead of DNA, as mentioned in Steps C1, C2 and C3. 2) Prepare PCR reaction for HIV-2 standard from HIV-2 <em>gag</em> fragment corresponding to 10<sup>6</sup> to 1 DNA copies as mentioned in Step C5. 3) Prepare Real-Time PCR reaction of <em>PBDG</em> from extracted DNA of infected and uninfected individuals and with sterile water instead of DNA as mentioned in Step C6. 4) Prepare PCR reaction for <em>PBDG</em> standard from purified housekeeping gene fragment corresponding to 10<sup>6</sup> to 100 DNA copies as mentioned in Step C7. 5) All reactions need to be performed using a 96-well plate. 6) Carry out qPCR reaction in the conditions mentioned in Step C10. 7) Calculate the concentration of samples by using the formula in Step C11.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211107_9730.jpg" width="183"/><br/><strong>Figure 2. Example of the experimental layout.</strong> Blue colors indicate the qPCR of HIV-2 <em>gag</em>, while yellow colors stand for Real-Time PCR of <em>PBDG</em> gene. Standard curves are generated from eight (for HIV-2) (10<sup>6</sup> to 1 copies) and from five dilutions (for <em>PBDG</em>) (10<sup>6</sup> to 100 copies) dilutions. Each sample and control are prepared in duplicates on a 96-well qPCR plate. DNA of U1 cells stands for negative and specificity controls (as these cells are carrying an integrated HIV-1 genome, but no HIV-2 genome).<br/><br/><ol class="npwtwo"><li>Prepare the following Real-Time PCR reactions for HIV-2 in 30 Î¼l solution (quantities per sample) containing 15 Î¼l Thermo Fisher Maxima probe qPCR Master Mix, 4 Î¼M of Taqman Probe, 100 ng/Î¼l of each primer and 8 Î¼l of template DNA.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Sequences of primers and probe were obtained from Damond </em><em>et al.</em><em> (2002) (PMID:12354861). </em> </li><li><em>Extract sample DNA from 200 Î¼l of whole blood as mentioned in Step A3. To improve the yield of extraction use carrier RNA during viral DNA extraction.</em> </li><li><em>Since carrier RNA was used in the extraction, it is not possible to quantify the DNA reliably since spectrophotometric quantification will detect both RNA and DNA in the solution.</em> </li></ol></li><li>As a negative control, add 8 Î¼l of sterile water instead of DNA.</li><li>Set up the same reaction by using DNA from uninfected individuals to ensure that the primers are specific for viral DNA detection.<br/><em>Note: Extract DNA from 200 Î¼l of whole blood of uninfected individuals in the same way as it was performed for infected samples (see Step A3).</em> </li><li>Also, prepare the reaction in the presence of 1 Î¼g PM-1 cellâs DNA to exclude whether or not human DNA contamination has an effect on the sensitivity of real-time PCR.</li><li>For quantification, use a series of ten-fold dilutions of the HIV-2 <em>gag</em> fragment corresponding to 10<sup>6</sup> to 1 DNA copies of the standard, and include it in each experiment in order to generate an external standard curve. Each sample and standard should be quantified in duplicate (see Figures 3 and 4).<br/><em>Note: The most critical step of HIV-2 qPCR is the preparation of standards. Prepare the dilutions by using 1.5 ml low-binding Eppendorf tubes. Mix every tube by pipetting up and down, then centrifuge it before preparing the next dilution.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211143_6815.jpg" width="211"/><br/><strong>Figure 3. The serial dilution scheme for HIV-2 standard.</strong> Prepare serial dilution in 1.5 ml Eppendorf tubes by pipetting 5 Î¼l from 10<sup>9</sup> âstockâ to 10<sup>8</sup> copies/Î¼l tube and add 45 Î¼l water. Pipette up and down to mix and shortly centrifuge it. Using a new tip, pipette 5 Î¼l from tube 10<sup>8</sup> to tube 10<sup>7</sup>. Pipette up and down to mix and centrifuge it for a few seconds. Repeat the same process for the other tubes. Yellow highlighted shows dilutions used to perform HIV-2 standard curve.<br/><br/><img alt="" class="layerphoto" height="118" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211230_3374.jpg" width="100"/><br/><strong>Figure 4. Standard curve of HIV-2 </strong><em><strong>gag</strong></em><strong> fragment.</strong> A. Sensitivity test for TaqMan Probe for gene of interest. Amplification plot showed the testing results of the tenfold dilution series of host gene (10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 100, 10 and 1 DNA copies per reaction in duplicate). B. The linear regression analysis between the quantities of host gene (10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 100, 10 and 1 DNA copies per reaction in duplicate) and Ct values. Regression equations were calculated with y = -3.305 + 38.692, R<sup>2</sup> = 0.999. The red box represents Ct values.<br/><br/></li><li>Normalize HIV-2 DNA to expression of the housekeeping human gene, <em>PBGD</em>. Prepare the following reaction: 12.5 Î¼l Thermo Fisher Maxima probe qPCR Master Mix, 50 ng/Î¼l primers, 2 Î¼M of probe and 3 Î¼l of template DNA.<br/><em>Notes: </em> <ol class="npwthree"><li><em>Primer and probe sequences of PBGD gene were obtained from Mbisa </em><em>et al.</em><em> (2009) (PMID 19020818).</em> </li><li><em>Use extracted DNA of uninfected and infected individuals as a template DNA. For preparation of DNA, see Step A3.</em> </li></ol></li><li>For each run, a standard curve should be generated from purified <em>PBDG</em> fragment ranging from 10<sup>6</sup> to 100 DNA copies (see Figure 5).<br/><em>Notes: </em> <ol class="npwthree"><li><em>Prepare the same reaction for </em><em>PBDG</em><em> standard as mentioned in Step C6. </em> </li><li><em>Similar to HIV-2 standard preparation, dilutions of </em><em>PBDG</em><em> should be performed by using 1.5 ml low-binding tubes. Mixing by pipetting up and down, then centrifugation is necessary before every step of dilution.</em> </li></ol></li><li>Distribute the reaction mix in the wells of the 96-well qPCR plate and avoid air bubbles. An example of a plate layout is provided in Figure 2.</li><li>Cover the qPCR plate with an adhesive plastic film, centrifuge it briefly.</li><li>Carry out qPCR with an initial denaturation step at 95 Â°C for 10 min, followed by 40 cycles of denaturation at 95 Â°C for 15 s, annealing at 60 Â°C for 30 s and final extension at 72 Â°C for 30 s.</li><li>HIV-2 DNA concentrations should be first reported as HIV-2 DNA copies/Î¼l obtained from the qPCR reaction. The numbers of copies of HIV-2 DNA are then normalized to the host gene. The final results are reported as the number of copies/10<sup>5</sup> cells. The formula to convert the results is: (HIV-2 DNA copies per Î¼l)/(<em>PBDG</em> copies per Î¼l/2 chromosomes per cell) x 10<sup>5</sup> cells = HIV-2 DNA copies per 10<sup>5</sup> cells.<br/><em>Note: Perform quantitative real-time PCR (qPCR) by using the Applied Biosystems StepOnePlus Real-Time PCR System. Extracted DNA from U1 cells, HIV negative and HIV-1 infected individuals can be used as controls.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211337_3669.jpg" width="106"/><br/><strong>Figure 5. Standard curve for <em>PBDG</em> (host gene).</strong> A. Sensitivity test for TaqMan Probe for <em>PBDG</em> host gene. Amplification plot showed the testing results of the tenfold dilution series of host gene (10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup> and 100 DNA copies per reaction in duplicate). B. The linear regression analysis between the quantities of host gene (10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup> and 100 DNA copies per reaction in duplicate) and Ct values. Regression equations were calculated with y = -3.483 + 39.893, R<sup>2</sup> = 0.999. The red box represents Ct values.<br/></li></ol></li></ol><p class="pptt" id="biaoti29993">### Data analysis</p><ol class="npwtwok"><li>First export the obtained copy numbers from AB StepOnePlus Real-Time PCR System. Calculate for each sample the average of copy number of duplicate measurements.<br/><em>Note: To determine HIV-2 DNA load in whole blood, express HIV-2 DNA copies per cell number using normalization to the porphobilinogen deaminase (</em><em>PBDG</em><em>) gene copy number.</em> </li><li>First present DNA levels as copies per Î¼l obtained in the PCR reaction, after that normalize the level of HIV-2 DNA to the <em>PBDG</em> copies per Î¼l obtained in the PCR reaction. Set up duplicates of samples and standards in each experiment. Use the following formula to convert the results: (HIV-2 DNA copies per Î¼l)/(<em>PBDG</em> copies per Î¼l)/2 chromosomes per cell) x 10<sup>5</sup> cells = HIV-2 DNA copies per 10<sup>5</sup> cells. An example of calculation is provided in Figure 6.<br/><br/><img alt="" class="layerphoto" height="129" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211419_7624.jpg" width="100"/><br/><strong>Figure 6. Calculation of HIV-2 DNA copies of an unknown sample.</strong> A. Standard curve of HIV-2 <em>gag</em> fragment with the copy numbers of unknown samples. The red box represents Ct values of HIV-2 <em>gag</em> fragment. Linear regression analysis between the quantities of HIV-2 <em>gag</em> gene 10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 100, 10 and 5 DNA copies per reaction in duplicate) and Ct values. The blue box represents HIV-2 <em>gag</em> Ct values of unknown samples. B. Standard curve for <em>PBDG</em> (housekeeping gene) and unknown sample. The linear regression analysis between the quantities of <em>PBDG</em> gene (10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 100 and 1 DNA copies per reaction in duplicate) and Ct values. The red box represents Ct values of <em>PBDG</em> and blue box represents Ct values of <em>PBDG</em> of unknown sample. C. Formula used to convert the results to HIV-2 DNA copies per 10<sup>5</sup> cells. <br/><br/></li><li>For quantification, the standard with dilutions (see Figure 3) was analyzed in the presence or absence of 1 Î¼g of HIV-2 negative PM1 cell DNA (background DNA). The two curves revealed no differences in their linear range of quantification or PCR efficiency. Moreover, in the presence of 1 Î¼g background DNA, no inhibitory effect was detected (see Figure 7).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191015/20191015211501_9456.jpg" width="138"/><br/><strong>Figure 7. Standard of HIV-2 </strong><strong>gag</strong><strong> fragment in the presence of 1 Î¼g human DNA.</strong> To exclude that background human DNA has an inhibitory effect or not, qPCR was performed in the presence of 1 Î¼g DNA of PM1 cells. The linear regression analysis between the quantities of HIV-2 <em>gag</em> gene (10<sup>6</sup>, 10<sup>5</sup>, 10<sup>4</sup>, 10<sup>3</sup>, 100, 10 and 5 DNA copies per reaction in duplicate) and Ct values in the presence of DNA of PM1 cells. Regression equations were calculated with y = -3.305 + 38.692, R<sup>2</sup> = 0.999. The red box represents Ct values of standard without human DNA. The blue box represents Ct values in the presence of PM1 cellsâ DNA.<br/><br/></li><li>Sensitivity of the assay was 100% at five copies/reaction (detected in 14/14 runs, 0.69 log<sub>10</sub> with SD 0.44), while 78% at one copy/reaction (11/14 runs). Limit of detection was 1 copies/reaction (11/14 runs), the median correlation coefficient was 0.999 (range: 0.998-1.000) and limit of quantification was 5 copies/reaction at CT 33 (SD: 0.9). Median slope was -3.15 (range: -3.00--3.37) and median efficiency was 97.0% (range: 98.4% to 98.5%).</li><li>LOD and LOQ was calculated by using a formula described by Schwarz <em>et al.</em> (2004). Detection limit of proviral DNA was 0.5 copies/10<sup>5</sup> leukocytes.</li><li>To exclude that this method is specific only for HIV-2, run qPCR on extracted DNA of U1 (subclone of U937, which has been chronically infected with HIV-1) cells (Folks <em>et al.</em>, 1987), and negative controls, such as extracted DNA from whole blood of HIV negative individuals and extracted DNA from HIV-1 infected patients. As none of the HIV-negative samples, nor HIV-1 infected samples gave positive results, specificity was 100%.<br/></li></ol><p class="pptt" id="biaoti29994">### Recipes</p><ol class="npwtwok"><li>Complete RPMI-1640 medium (pre-warmed at 37 Â°C before use)<br/>RPMI-1640 medium <br/>2.0 mM L-glutamine<br/>10% FBS (Heat inactivate at 56 Â± 2 Â°C for 30 Â± 2 min in a water-bath, and portion at 50 ml/centrifuge tubes, and store at -20 Â°C until use)<br/>1% penicillin/streptomycin solution<br/></li></ol><p class="pptt" id="biaoti29995">### Acknowledgments</p><p class="ppzz">This work was supported by Physicians Against AIDS Research Foundation, Sweden, and the Swedish Research Council, Sweden (no. 321-2012-3274 for P.M., no. 2016-02285 for M.J.), Crafoord Foundation, Ãsterlund Foundation and by EFOP-3.6.3-VEKOP-16-2017-00009 for ZsSz.<br/>  We thank the members of the Sweden Guinea-Bissau Cohort Research (SWEGUB CORE) group, including: Babetida NâBuna, Antonio Biague, Ansu Biai, Cidia Camara, Joakim EsbjÃ¶rnsson, Marianne Jansson, Sara Karlson, Jacob Lopatko Lindman, Patrik Medstrand, Fredrik MÃ¥nsson, Hans Norrgren, Angelica A. Palm, GÃ¼lsen Ãzkaya Sahin, and Zacarias Jose da Silva.<br/>  The following reagents were obtained through the NIH AIDS Reagent Program, Division of AIDS, NIAID, NIH: PM1 from Dr. Marvin Reitz and HIV-1 infected U937 Cells (U1) from Dr. Thomas Folks.</p><p class="pptt" id="biaoti29996">### Competing interests</p><p class="ppzz">The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript. Authors have no conflict of interest.</p><p class="pptt" id="biaoti29997">### Ethics</p><p class="ppzz">The study was approved by the ethical committees of the National Health Ethics Committee in Guinea-Bissau (Ref 038/CNES/INASA/2016) and the Regional Ethical Review Board, Lund University, Sweden (Dnr 2016/426). All participants received information about the study before inclusion and provided oral and written informed consent. To ensure confidentiality, all study data was managed under code.</p><p class="pptt" id="biaoti29998">### References</p><ol class="npwtwok"><li>Andersson, S., Norrgren, H., da Silva, Z., Biague, A., Bamba, S., Kwok, S., Christopherson, C., Biberfeld, G. and Albert, J. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11088091" target="_blank">Plasma viral load in HIV-1 and HIV-2 singly and dually infected individuals in Guinea-Bissau, West Africa: significantly lower plasma virus set point in HIV-2 infection than in HIV-1 infection. </a><em>Arch Intern Med</em> 160(21): 3286-3293.</li><li>Ariyoshi, K., Berry, N., Wilkins, A., Ricard, D., Aaby, P., Naucler, A., Ngom, P. T., Jobe, O., Jaffar, S., Dias, F., Tedder, R. S. and Whittle, H. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8537668" target="_blank">A community-based study of human immunodeficiency virus type 2 provirus load in rural village in West Africa.</a> <em>J Infect Dis </em>173(1): 245-248.</li><li>Azevedo-Pereira, J. M. and Santos-Costa, Q. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26936760" target="_blank">HIV Interaction With Human Host: HIV-2 As a Model of a Less Virulent Infection.</a> <em>AIDS Rev </em>18(1): 44-53.</li><li>Berry, N., Ariyoshi, K., Jaffar, S., Sabally, S., Corrah, T., Tedder, R. and Whittle, H. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10195267" target="_blank">Low peripheral blood viral HIV-2 RNA in individuals with high CD4 percentage differentiates HIV-2 from HIV-1 infection. </a><em>J Hum Virol</em> 1(7): 457-468.</li><li>Bertine, M., Gueudin, M., Melard, A., Damond, F., Descamps, D., Matheron, S., Collin, F., Rouzioux, C., Plantier, J. C. and Avettand-Fenoel, V. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28701422" target="_blank">New Highly Sensitive Real-Time PCR Assay for HIV-2 Group A and Group B DNA Quantification.</a> <em>J Clin Microbiol </em>55(9): 2850-2857.</li><li>Buggert, M., Frederiksen, J., Lund, O., Betts, M. R., Biague, A., Nielsen, M., Tauriainen, J., Norrgren, H., Medstrand, P., Karlsson, A. C., Jansson, M. and group, S. C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27525551" target="_blank">CD4+ T cells with an activated and exhausted phenotype distinguish immunodeficiency during aviremic HIV-2 infection.</a> <em>AIDS</em> 30(16): 2415-2426.</li><li>Damond, F., Descamps, D., Farfara, I., Telles, J. N., Puyeo, S., Campa, P., Lepretre, A., Matheron, S., Brun-Vezinet, F. and Simon, F. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11724830" target="_blank">Quantification of proviral load of human immunodeficiency virus type 2 subtypes A and B using real-time PCR.</a> <em>J Clin Microbiol </em>39(12): 4264-4268.</li><li>Damond, F., Gueudin, M., Pueyo, S., Farfara, I., Robertson, D. L., Descamps, D., Chene, G., Matheron, S., Campa, P., Brun-Vezinet, F. and Simon, F. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12354861" target="_blank">Plasma RNA viral load in human immunodeficiency virus type 2 subtype A and subtype B infections.</a> <em>J Clin Microbiol </em>40(10): 3654-3659.</li><li>Esbjornsson, J., Mansson, F., Kvist, A., da Silva, Z. J., Andersson, S., Fenyo, E. M., Isberg, P. E., Biague, A. J., Lindman, J., Palm, A. A., Rowland-Jones, S. L., Jansson, M., Medstrand, P., Norrgren, H., Sweden and Guinea-Bissau Cohort Research, G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30392769" target="_blank">Long-term follow-up of HIV-2-related AIDS and mortality in Guinea-Bissau: a prospective open cohort study. </a><em>Lancet HIV.</em> doi: 10.1016/S2352-3018(18)30254-6.</li><li>Folks, T. M., Justement, J., Kinter, A., Dinarello, C. A. and Fauci, A. S. (1987). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3313729" target="_blank">Cytokine-induced expression of HIV-1 in a chronically infected promonocyte cell line.</a> <em>Science</em> 238(4828): 800-802.</li><li>Gomes, P., Taveira, N. C., Pereira, J. M., Antunes, F., Ferreira, M. O. and Lourenco, M. H. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9889242" target="_blank">Quantitation of human immunodeficiency virus type 2 DNA in peripheral blood mononuclear cells by using a quantitative-competitive PCR assay. </a><em>J Clin Microbiol </em>37(2): 453-456.</li><li>Gueudin, M., Damond, F., Braun, J., Taieb, A., Lemee, V., Plantier, J. C., Chene, G., Matheron, S., Brun-Vezinet, F. and Simon, F. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18097223" target="_blank">Differences in proviral DNA load between HIV-1- and HIV-2-infected patients. </a><em>AIDS</em> 22(2): 211-215.</li><li>Honge, B. L., Jespersen, S., Medina, C., Te, D. S., da Silva, Z. J., Christiansen, M., Kjerulff, B., Laursen, A. L., Wejse, C., Krarup, H., Erikstrup, C. and Bissau, H. I. V. C. s. g. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29573304" target="_blank">The challenge of discriminating between HIV-1, HIV-2 and HIV-1/2 dual infections. </a><em>HIV Med </em>19(6): 403-410.</li><li>Lusso, P., Cocchi, F., Balotta, C., Markham, P. D., Louie, A., Farci, P., Pal, R., Gallo, R. C. and Reitz, M. S., Jr. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7745720" target="_blank">Growth of macrophage-tropic and primary human immunodeficiency virus type 1 (HIV-1) isolates in a unique CD4<sup>+</sup> T-cell clone (PM1): failure to downregulate CD4 and to interfere with cell-line-tropic HIV-1.</a> <em>J Virol</em> 69(6): 3712-3720.</li><li>Marvin, S., Reitz, Jr. and Robert, C. G. (2015). <a href="https://www.sciencedirect.com/science/article/pii/B9781455748013001715" target="_blank">Human Immunodeficiency Viruses.</a> In: Mandell, Douglas, and Bennett's Principles and Practice of Infectious Diseases (Eighth Edition). <em>Elsevier</em> 2: 2054-2065.</li><li>Mbisa, J. L., Delviks-Frankenberry, K. A., Thomas, J. A., Gorelick, R. J. and Pathak, V. K. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19020818" target="_blank">Real-time PCR analysis of HIV-1 replication post-entry events.</a> <em>Methods Mol Biol </em>485: 55-72.</li><li>Parisi, S. G., Andreis, S., Mengoli, C., Scaggiante, R., Ferretto, R., Manfrin, V., Cruciani, M., Giobbia, M., Boldrin, C., Basso, M., Andreoni, M., Palu, G. and Sarmati, L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22135262" target="_blank">Baseline cellular HIV DNA load predicts HIV DNA decline and residual HIV plasma levels during effective antiretroviral therapy.</a> <em>J Clin Microbiol </em>50(2): 258-263.</li><li>Popper, S. J., Sarr, A. D., Gueye-Ndiaye, A., Mboup, S., Essex, M. E. and Kanki, P. J. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10627569" target="_blank">Low plasma human immunodeficiency virus type 2 viral load is independent of proviral load: low virus production <em>in vivo</em>. </a><em>J Virol </em>74(3): 1554-1557.</li><li>Popper, S. J., Sarr, A. D., Travers, K. U., Gueye-Ndiaye, A., Mboup, S., Essex, M. E. and Kanki, P. J. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10479138" target="_blank">Lower human immunodeficiency virus (HIV) type 2 viral load reflects the difference in pathogenicity of HIV-1 and HIV-2.</a> <em>J Infect Dis </em>180(4): 1116-1121.</li><li>Raich, N., Romeo, P. H., Dubart, A., Beaupain, D., Cohen-Solal, M. and Goossens, M. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2875434" target="_blank">Molecular cloning and complete primary sequence of human erythrocyte porphobilinogen deaminase.</a> <em>Nucleic Acids Res </em>14(15): 5955-5968.</li><li>Rouzioux, C. and Avettand-Fenoel, V. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29615133" target="_blank">Total HIV DNA: a global marker of HIV persistence. </a><em>Retrovirology</em> 15(1): 30.</li><li>Schwarz, G., Baumler, S., Block, A., Felsenstein, F. G. and Wenzel, G. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14960708" target="_blank">Determination of detection and quantification limits for SNP allele frequency estimation in DNA pools using real time PCR. </a><em>Nucleic Acids Res </em>32(3): e24.</li><li>Soares, R., Foxall, R., Albuquerque, A., Cortesao, C., Garcia, M., Victorino, R. M. and Sousa, A. E. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17035326" target="_blank">Increased frequency of circulating CCR5<sup>+</sup> CD4<sup>+</sup> T cells in human immunodeficiency virus type 2 infection. </a><em>J Virol</em> 80(24): 12425-12429.<br/></li></ol>