<body>
            <h1>Probabilistic Models for Predicting Mutational Routes to New Adaptive Phenotypes</h1>
            <p><strong>Authors:</strong> Eric  Libby, Peter A. Lind</p>
            <h2>Abstract</h2>
            <p>Understanding the translation of genetic variation to phenotypic variation is a fundamental problem in genetics and evolutionary biology. The introduction of new genetic variation through mutation can lead to new adaptive phenotypes, but the complexity of the genotype-to-phenotype map makes it challenging to predict the phenotypic effects of mutation. Metabolic models, in conjunction with flux balance analysis, have been used to predict evolutionary optimality. These methods however rely on large scale models of metabolism, describe a limited set of phenotypes, and assume that selection for growth rate is the prime evolutionary driver. <br/><br/>Here we describe a method for computing the relative likelihood that mutational change will translate into a phenotypic change between two molecular pathways. The interactions of molecular components in the pathways are modeled with ordinary differential equations. Unknown parameters are offset by probability distributions that describe the concentrations of molecular components, the reaction rates for different molecular processes, and the effects of mutations. Finally, the likelihood that mutations in a pathway will yield phenotypic change is estimated with stochastic simulations.<br/><br/>One advantage of this method is that only basic knowledge of the interaction network underlying a phenotype is required. However, it can also incorporate available information about concentrations and reaction rates as well as mutational biases and mutational robustness of molecular components. The method estimates the relative probabilities that different pathways produce phenotypic change, which can be combined with fitness models to predict evolutionary outcomes.</p>
            <p><strong>Keywords:</strong> Evolutionary forecasting, Mathematical modeling, Adaptation, Mutation, Evolution, Genotype-to-phenotype map</p>
            <h2>Background</h2><p>The ability to forecast evolution would not only make evolutionary biology more predictive but could have translational impact with applications in biotechnology (<em>e.g.</em>, synthetic biology or biofuels) or medicine (<em>e.g.</em>, antibiotic resistance or cancer). Most previous success in evolutionary forecasting has been in asexual microbial populations subject to strong selection and relies on extensive historical sequence data to estimate the fitness of different strains (Luksza and Lassig 2014; Neher <em>et al.</em>, 2014). Such models are not based on the mechanistic details of how mutations translate into fitness differences. The models also assume that selection is the main determinant of evolutionary outcomes which is a limiting assumption because evolutionary routes can be significantly influenced by factors other than selection. For example, if the rate of phenotypic production differs across evolutionary routes then more likely mutations that confer small gains in fitness may be observed more often than rare mutations that confer large fitness gains; being first may be more important than being best. In such cases, accurate prediction of the mutational routes to adaptive phenotypes requires knowledge of their relative rates of phenotypic production. <br/><br/>The rates of phenotypic production can be shaped by a variety of factors. For instance, a higher rate of phenotypic production can be caused by mutational hotspots, which are sections of DNA that increase the frequency of mutational events in nearby genes. Another possibility is that some genes have a greater capacity to translate mutation into phenotypic change, <em>i.e.</em>, have a larger number of sites that can be mutated with functional effects. This can due to the properties of the individual protein (Lind <em>et al.</em>, 2017) or ncRNA, in terms of mutational robustness, but also because proteins and ncRNAs have different functions in the molecular interaction networks underpinning adaptive phenotypes (Lind <em>et al.</em>, 2015 and 2019). A simple example of the latter case would be when expression of a gene is controlled by both a negative and a positive regulator, activation of that gene occurs more often through mutations in the negative regulator simply due to loss-of-function mutations being more likely than gain-of-function mutations. <br/><br/>To account for the factors affecting the rate of phenotypic production, mechanistic information is needed. An arena in which mechanistic information has been used to successfully predict evolution is in metabolism (Edwards and Palsson 2000; O'Brien <em>et al.</em>, 2015). There is a wealth of data on the biochemical reactions involved in central metabolism in different microbial species. This data can be used to form explicit metabolic models that in conjunction with a technique known as flux balance analysis can predict the growth rate of an organism in simple environments. By manipulating which reactions are present in the metabolic model, the phenotypic effects, <em>i.e.</em>, growth rates, of mutational knockouts can be predicted. Similar metabolic models have also been used to predict how organisms will interact in different environments via the exchange of biochemical compounds including spatio-temporal interactions (Liu <em>et al.</em>, 2015; Bocci <em>et al.</em>, 2018). While the metabolic models have been successful in certain evolutionary predictions, they are limited in their applicability: they focus mainly on growth phenotypes in environments in which organisms are assumed to be actively reproducing. Moreover, the growth phenotypes are computed using large-scale models with hundreds or thousands of reactions and typically involve optimization of a âbiomassââ function of some 50+ variables. The models also do not usually incorporate any type of regulation, <em>e.g.</em>, transcriptional/translational control or protein modifications, but see Chandrasekaran and Price, 2010 for one such example. Thus, they are not well suited to generalizations outside of metabolism. <br/><br/>Here we describe a method to predict the rates that mutational changes to different molecular networks produce phenotypic change. One advantage of this method is that only knowledge of the general architecture of the molecular networks is needed to make a prediction. However information about reaction rates, concentrations, mutation rates and gene size, mutational robustness of components can be included if known. The method can incorporate different types of reactions, for example conformational changes and enzymatic reactions, and the phenotypes predicted are not limited to optimization for growth rate. Predictions generated by the method could potentially be combined with origin-fixation models (McCandlish and Stoltzfus, 2014) in order to predict the mutation rate to an adaptive phenotype in the absence of unbiased experimental data that is often difficult to obtain. Information about mutational biases (Lind <em>et al.</em>, 2019) and the molecular effects of mutations on protein function from different prediction methods (Capriotti <em>et al.</em>, 2005; Bromberg and Rost, 2007; Kumar <em>et al.</em>, 2009; Dehouck <em>et al.</em>, 2011; Capriotti <em>et al.</em>, 2013; Celniker <em>et al.</em>, 2013; Yates <em>et al.</em>, 2014; Choi and Chan, 2015) can be incorporated into the model described here to adjust the rates of disabling and enabling mutation in different genes. The method described here is also useful for providing null models in order to test the causes of repeated evolution (Lind, 2018; Lind <em>et al.</em>, 2019). It could also be one component in understanding the molecular bases of complex genetic diseases and for evolutionary forecasting of antibiotic resistance and cancer, especially when experimental data is incomplete.</p>
        </body>

<p class="pptt" id="biaoti30025">### Equipment</p><ol class="npwtwok"><li>Computer<br/>Any desktop computer that fulfills the system requirements for the programming language(s) used. Here a desktop computer with the following configuration was used: iMac with a 3.6 GHz Intel Core i7 processor and 16 GB 2400 MHz DDR4 memory<br/></li></ol><p class="pptt" id="biaoti30026">### Software</p><ol class="npwtwok"><li>MATLAB (MathWorks<sup>Â®</sup>, <a href="http://www.mathworks.com/" target="_blank">http://www.mathworks.com/</a>)<br/>MATLAB was used to solve the differential equations, analyze the resulting data, and generate figures. However, other programming language(s) would have sufficed so long as they can be used to numerically solve differential equations and perform simple mathematical operations, <em>e.g.</em>, Python, Julia, Octave, or Mathematica. We note that in the associated paper (Lind <em>et al.</em>, 2019) a combination of languages was used but for simplicity here we have put everything into a single language. MATLAB code for the procedures RunModelComparison and PlotModelComparison is available at <a href="http://www.mathworks.com/" target="_blank">https://github.com/ericlibby/BioProtocol</a>.<br/></li></ol><p class="pptt" id="biaoti30027">### Procedure</p><p class="ppzz">This protocol describes a method for computing the relative likelihood that a mutational change will translate into a phenotypic change in two molecular pathways. The pathways do not have to produce the same phenotype but there should be a way of determining what the phenotype is, based on the interactions between pathway components. Thus, the protocol assumes that the mechanistic details of the molecular pathways are understood; however, information may be missing concerning the reaction kinetics, the concentrations of molecular compounds, and/or the effects of mutations. This protocol outlines a procedure that randomly samples many models of the pathways and compares the likelihood that mutations in the pathwayâthat affect the reaction kineticsâalter the phenotype(s). The steps of the procedure are illustrated in Figure 1 using the same notation as in the text below. Figure 2 and Figure 3 presents the procedure in a pseudocode format and the full MATLAB code is available at <a href="http://www.mathworks.com/" target="_blank">https://github.com/ericlibby/BioProtocol</a>.</p><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191016/20191016012701_9951.jpg" width="300"/><br/><strong>Figure 1. Overview of modeling methodology.</strong> Details are explained in the text below using the same notation.</p><br/><ol class="npwone"><li>Build mathematical models of the molecular pathways that enable comparison<ol class="npwtwo"><li>Identify the set of biochemical reactions in each molecular pathway that determine expression of a particular phenotype. The types of reactions can vary depending on the pathway modeled and can include, for example, enzymatic reactions, conformational changes in proteins, oligomerization, and transcription initiation. The reactions should link the expression of the phenotype to some indicator, <em>e.g.</em>, a biochemical compound (or function of compounds). For example, if the phenotype is the color of an organism then an appropriate indicator might be the amount of a pigment compound. In the article where this method was first used (Lind <em>et al.</em>, 2019) the indicators were three diguanylate cyclase enzymes that could each be activated by mutations in their regulatory networks.</li><li>For each set of biochemical reactions, formulate a system of differential equations that describes the reaction kinetics of the indicator (see Figure 1 for example). Each pathway is likely affected by biochemical compounds whose dynamics are governed by external pathways. It is often better to treat these compounds as constants rather than include more differential equations for their dynamics. For instance, if a pathway affects an indicator through phosphorylation then the intracellular pool of phosphate groups may be held as a constant rather than modeled through an additional set of differential equations. In general, it also helps to keep a similar level of detail/abstraction across the models of the pathways for better comparison. As an example, in (Lind <em>et al.</em>, 2019) the models of the different pathways focused on the interactions between proteins and ignored processes involving transcription.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191016/20191016012729_2426.jpg" width="300"/><br/><strong>Figure 2. Modeling of the AWS network.</strong> The reactions of the AWS regulatory network from (Lind <em>et al.</em>, 2019) can be illustrated by an interaction diagram. The mathematical model of AWS is a system of ordinary differential equations describing all reactions and rates. In this case, the active dimer RR is the indicator of phenotypic change.<br/></li></ol><br/></li><li>Choose parameter distributions to solve the differential equations<br/>We assume that the systems of differential equations used to model the pathways cannot be solved analytically. To solve each system of differential equations numerically, three key pieces of information are needed:<ol class="npwthree"><li>Initial concentrations of the molecular compounds.</li><li>Reaction rates for the interactions between compounds.</li><li>Time interval over which to solve the differential equations.</li></ol>It is possible that all pieces of information are known for a model in which case there will be only one numerical solution for each pathway model. In most cases, however, information is missing and must be estimated. <br/><br/><ol class="npwtwo"><li>If there is any information concerning a) or b) that is known or can be estimated from empirical data then use this to constrain the sampling procedure. For unknown concentrations or reaction rates, choose a distribution for each that seems reasonable based on empirical data or at least encompasses a broad spectrum of likely dynamics. For instance, in Lind <em>et al.</em> (2019) there was no information concerning reaction rates or concentrations, so the authors chose the distribution for the reaction rates to be a uniform distribution on log space (<em>i.e.</em>, 10<sup>U[â2,2]</sup>) and the distribution for initial concentrations to be a uniform distribution U[0,10]. These choices were in line with other pathway reaction kinetic models and allowed for numerically stable solutions. Importantly, whatever distributions are chosen, it is useful to see the effects of different choices on the results of the pathway comparison as part of a post-analysis parameter sensitivity study.</li><li>The time interval for numerical solving the differential equations depends on the particular molecular system and the dynamics of the indicator compound. If the indicator reaches a steady state then that might set the time interval; however, in some systems transient, short-term behavior may be more biologically relevant. A distribution could be chosen for the time to solve the differential equations; however, in practice it is often faster and easier to evaluate the results when a fixed time is chosen and used for all modelsâespecially when all other parameters are sampled randomly.</li></ol><br/></li><li>Choose distributions for the effects of mutations and the amount of change in the indicator compound that corresponds to phenotypic change<br/>The prior steps allow for the computation of the indicator compound in each pathway, as a baseline for comparison. The choices made in this step will allow numerical solution of a pathwayâs differential equation model following a mutation (or set of mutations) and determination of whether the phenotype has changed.<ol class="npwtwo"><li>We assume that the relevant mutations to a pathway are those that alter reaction rates, <em>i.e.</em>, change the internal dynamics of a pathway, and we classify mutations as enabling or disabling based on whether they increase/decrease reaction rates. Thus, we need a way of determining new reaction rate(s) in the differential equation model of a pathway, following a mutation (or a set of mutations). Ideally, one would have information on all possible mutations and their effects on reaction rates. However, this is unlikely to be available. In the absence of empirical data, a distribution should be picked such that given a reaction rate and whether a mutation increases or decreases reaction rates, a new reaction rate is determined. For example, in (Lind <em>et al.</em>, 2019) enabling mutations were assumed to affect reaction rates multiplicatively and thus new reaction rates were computed as the product of the original reaction rate and a multiplicative factor sampled from 10<sup>U[0,2]</sup> (similarly a factor of 10<sup>U[â2,0]</sup>) was used for disabling mutations. </li><li>The differential equation models for each pathway link the concentration of an indicator compound to a phenotype. Since the sampling distributions in Step B may result in different baseline concentrations of indicator compound, there needs to be a way of assessing whether a mutation (or set of mutations) produces a phenotypic change. In the absence of empirical data, a simple approach is to establish that if the indicator increases in concentration above some threshold then the phenotype changes. The actual threshold will depend on the biological system and how much of a change in the indicator is biologically relevant. Without this information, the threshold should be high enough to avoid issues of numerical precision and will depend on the differential equation solver and its parameters (<em>e.g.</em>, step sizes in Runge-Kutta methods). For simplicity, we used a threshold of 0.000001 which was similar in magnitude to the numerical tolerances used in the differential equation solver.</li></ol><br/></li><li>Run simulation routine to compute the likelihood of a phenotypic change given all combinations of mutations<br/>Use the code âRun Model Comparisonâ (<a href="https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m" target="_blank">https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m</a>) to generate the data. The pseudocode for âRun Model Comparisonâ is included below as Algorithm 1 (Figure 3). âRun Model Comparisonâ use the procedure âPhenotype Changeâ for each pathway to compute the probability that mutation in a pathway causes a phenotypic change. The pseudocode for âPhenotype Changeâ is shown in Figure 3. âPhenotype Changeâ uses several procedures to set the parameter distributions (Procedure B, C above). The pseudocode for these are shown in Figure 4. To visualize the results, use the code âPlot Model Comparisonâ (<a href="https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m" target="_blank">https://github.com/ericlibby/BioProtocol/blob/master/PlotModelComparison.m</a>) to create a contour plot that shows the log<sub>2</sub> ratio of relative probabilities between two pathways with axes denoting the probabilities of enabling and disabling change.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191016/20191016012910_8848.jpg" width="300"/><br/><strong>Figure 3. Pseudocode for the âRunModelComparisonâ and âPhenotypeChangeâ procedures.</strong> âPhenotypeChangeâ compute the probability that mutation in a pathway causes a phenotypic change. âRunModelComparisonâ computes the relative likelihood that Pathway 1 is used relative to Pathway 2. MATLAB code for RunModelComparison is available at <a href="https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m" target="_blank">https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m.</a><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191016/20191016013000_5669.jpg" width="300"/><br/><strong>Figure 4. Pseudocode for procedures used by âPhenotypeChangeâ procedure shown in Figure 3.</strong> The SolveModel procedure is not listed as it is assumed to be a numerical solver available as a built-in function or from a library/package.<br/></li></ol><p class="pptt" id="biaoti30028">### Data analysis</p><p class="ppzz">The result of running the codes âRun Model Comparisonâ (<a href="https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m" target="_blank">https://github.com/ericlibby/BioProtocol/blob/master/RunModelComparison.m</a>) and âPlot Model Comparisonâ (<a href="https://github.com/ericlibby/BioProtocol/blob/master/PlotModelComparison.m" target="_blank">https://github.com/ericlibby/BioProtocol/blob/master/PlotModelComparison.m</a>) is a contour plot that shows the ratio of likelihoods that the pathways produce a phenotypic change for different values of the probabilities of enhancing and disabling mutations (the vertical and horizontal axes, respectively). In terms of statistical analyses, it depends on the conclusions that are drawn. The contour plot only shows the ratio of the average likelihood for each pathway. Further analyses can investigate the standard deviation or maximum/minimum of the likelihoods. We recommend that additional analyses be performed to validate any conclusions. In particular, the analyses can be performed again with changes to the probability distributions to test for parameter sensitivity. Alternatively, the number of iterations can be increased/decreased by some factor to evaluate whether the contour plot remains stable. Examples of these analyses can be found in Figure 5 and associated methods of Lind <em>et al.</em>, 2019.</p><p class="pptt" id="biaoti30029">### Acknowledgments</p><p class="ppzz">This protocol was adapted from (Lind <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti30030">### Competing interests</p><p class="ppzz">The authors declare no competing financial interest.</p><p class="pptt" id="biaoti30031">### References</p><ol class="npwtwok"><li>Bocci, F., Suzuki, Y., Lu, M. and Onuchic, J. N. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29610314" target="_blank">Role of metabolic spatiotemporal dynamics in regulating biofilm colony expansion.</a> <em>Proc Natl Acad Sci U S A</em> 115: 4288-4293.</li><li>Bromberg, Y. and Rost, B. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17526529" target="_blank">SNAP: predict effect of non-synonymous polymorphisms on function.</a> <em>Nucleic Acids Res</em> 35(11): 3823-3835.</li><li>Capriotti, E., Calabrese, R., Fariselli, P., Martelli, P. L., Altman, R. B. and Casadio, R. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23819482" target="_blank">WS-SNPs&amp;GO: a web server for predicting the deleterious effect of human protein variants using functional annotation.</a> <em>BMC Genomics</em> 14 Suppl 3: S6.</li><li>Capriotti, E., Fariselli, P. and Casadio, R. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15980478" target="_blank">I-Mutant2.0: predicting stability changes upon mutation from the protein sequence or structure.</a> <em>Nucleic Acids Res </em>33(Web Server issue): W306-310.</li><li>Celniker, G., Nimrod, G., Ashkenazy, H., Glaser, F., Martz, E., Mayrose, I., Pupko, T. and Ben-Tal, N. (2013). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ijch.201200096" target="_blank">ConSurf: using evolutionary data to raise testable hypotheses about protein function.</a> <em>Isr J Chem</em> 53: 199-206.</li><li>Chandrasekaran, S. and Price, N. D. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20876091" target="_blank">Probabilistic integrative modeling of genome-scale metabolic and regulatory networks in <em>Escherichia coli</em> and <em>Mycobacterium tuberculosis</em>.</a> <em>Proc Natl Acad Sci U S A</em> 107: 17845-17850. </li><li>Choi, Y. and Chan, A. P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25851949" target="_blank">PROVEAN web server: a tool to predict the functional effect of amino acid substitutions and indels.</a> <em>Bioinformatics</em> 31(16): 2745-2747.</li><li>Dehouck, Y., Kwasigroch, J. M., Gilis, D. and Rooman, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21569468" target="_blank">PoPMuSiC 2.1: a web server for the estimation of protein stability changes upon mutation and sequence optimality.</a> <em>BMC Bioinformatics</em> 12: 151. </li><li>Edwards, J. S. and Palsson, B. O. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11001586" target="_blank">Metabolic flux balance analysis and the in silico analysis of <em>Escherichia coli</em> K-12 gene deletions.</a> <em>BMC Bioinformatics</em> 1: 1.</li><li>Kumar, P., Henikoff, S. and Ng, P. C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19561590" target="_blank">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.</a> <em>Nat Protoc</em> 4(7): 1073-1081. </li><li>Lind, P. A. (2018). <a href="https://www.biorxiv.org/content/10.1101/342261v1" target="_blank">Evolutionary forecasting of phenotypic and genetic outcomes of experimental evolution in <em>Pseudomonas</em>.</a> <em>bioRxiv.</em> 342261.</li><li>Lind, P. A., Arvidsson, L., Berg, O. G. and Andersson, D. I. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28025272" target="_blank">Variation in mutational robustness between different proteins and the predictability of fitness effects.</a> <em>Mol Biol Evol</em> 34(2): 408-418.</li><li>Lind, P. A., Farr, A. D. and Rainey, P. B. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25806684" target="_blank">Experimental evolution reveals hidden diversity in evolutionary pathways.</a> <em>Elife</em> 4: e07074.</li><li>Lind, P. A., Libby, E., Herzog, J. and Rainey, P. B. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30616716" target="_blank">Predicting mutational routes to new adaptive phenotypes.</a> <em>Elife</em> 8: e38822.</li><li>Liu, J., Prindle, A., Humphries, J., Gabalda-Sagarra, M., Asally, M., Lee, D. Y., Ly, S., Garcia-Ojalvo, J. and Suel, G. M. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26200335" target="_blank">Metabolic co-dependence gives rise to collective oscillations within biofilms.</a> <em>Nature</em> 523(7562): 550-554. </li><li>Luksza, M. and Lassig, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24572367" target="_blank">A predictive fitness model for influenza.</a> <em>Nature</em> 507(7490): 57-61.</li><li>McCandlish, D. M. and Stoltzfus, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25195318" target="_blank">Modeling evolution using the probability of fixation: history and implications.</a> <em>Q Rev Biol</em> 89(3): 225-252.</li><li>Neher, R. A., Russell, C. A. and Shraiman, B. I. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25385532" target="_blank">Predicting evolution from the shape of genealogical trees.</a> <em>Elife</em> 3: e03568.</li><li>O'Brien EJ, Monk JM, Palsson BO. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26000478" target="_blank">Using Genome-scale Models to Predict Biological Capabilities.</a> <em>Cell</em> 161:971-987.</li><li>Yates, C. M., Filippis, I., Kelley, L. A. and Sternberg, M. J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24810707" target="_blank">SuSPect: enhanced prediction of single amino acid variant (SAV) phenotype using network features.</a> <em>J Mol Biol</em> 426(14): 2692-2701.</li></ol>