<body>
            <h1>Generation of CoilR Probe Peptides for VIPER-labeling of Cellular Proteins</h1>
            <p><strong>Authors:</strong> Julia K. Doh, Savannah J. Tobin, Kimberly E. Beatty</p>
            <h2>Abstract</h2>
            <p>Versatile Interacting Peptide (VIP) tags are a new class of genetically-encoded tag designed for imaging cellular proteins by fluorescence and electron microscopy. In 2018, we reported the VIPER tag (Doh <em>et al.</em>, 2018), which contains two elements: a genetically-encoded peptide tag (<em>i.e.</em>, CoilE) and a probe peptide (<em>i.e.</em>, CoilR). These two peptides deliver contrast to a protein of interest by forming a specific, high-affinity heterodimer. The probe peptide was designed with a single cysteine residue for site-specific modification via thiol-maleimide chemistry. This feature can be used to attach a variety of biophysical reporters to the peptide, including bright fluorophores for fluorescence microscopy or electron-dense nanoparticles for electron microscopy. In this Bio-Protocol, we describe our methods for expressing and purifying recombinant CoilR. Additionally, we describe protocols for making fluorescent or biotinylated probe peptides for labeling CoilE-tagged cellular proteins. This protocol is complemented by two other Bio-Protocols outlining the use of VIPER (Doh <em>et al.</em>, 2019a and 2019b).</p>
            <p><strong>Keywords:</strong> Peptide, Bioconjugation, Genetic tag, Microscopy, Chemical biology, Fluorescent</p>
            <h2>Background</h2><p>Fluorescence microscopy (FM), electron microscopy (EM), and correlative light and EM (CLEM) enable investigations into the multi-protein complexes and macromolecular interactions that mediate normal and disease-associated cellular functions. However, multiscale microscopy is restricted by the shortage of methods for attaching FM-, EM-, and CLEM-compatible reporter chemistries to target proteins. Additionally, there are few methods for protein labeling that facilitate switching between imaging systems. As a result, most multiscale imaging studies obtain protein-specific contrast with immunolabeling. However, there are known drawbacks to immunolabeling. The large size of antibodies reduces localization precision, and labeling protocols can disrupt cellular ultra-structure (Schnell <em>et al.</em>, 2012). Scarce proteins and rare interactions can elude detection unless immunolabeling is efficient (Griffiths and Hoppeler, 1986; Schnell <em>et al.</em>, 2012; Griffiths and Lucocq, 2014). Many antibodies have poor target specificity and cross-reactivity (Berglund <em>et al.</em>, 2008; Bordeaux <em>et al.</em>, 2010; Baker, 2015; Bradbury and Pluckthun, 2015), which can result in misleading observations. <br/><br/>The central obstacle that has limited progress in multiscale microscopy is the shortage of genetic tags for labeling proteins. Most tags were developed for FM (Liu <em>et al.</em>, 2015), with the most commonly used tags being fluorescent proteins [<em>e.g.</em>, GFP] (Tsien, 1998; Cranfill <em>et al.</em>, 2016; Rodriguez <em>et al.</em>, 2017). By comparison, there are few genetic tags for EM or CLEM (Ellisman <em>et al.</em>, 2012). We saw this as an opportunity to create a new class of genetically-encoded peptide tags for multiscale microscopy (Zane <em>et al.</em>, 2017; Doh <em>et al.</em>, 2018). We named this technology Versatile Interacting Peptide (VIP) tags (Figure 1). VIP tags consist of a heterodimeric coiled-coil between a genetically-encoded peptide tag and a reporter-conjugated peptide (âprobe peptideâ). Binding is driven by a hydrophobic interface and inter-strand salt bridges between the two coils. Initially we reported VIP Y/Z, which was used to label cellular proteins with fluorophores and Qdots (Zane <em>et al.</em>, 2017). This pair consists of a heterodimeric CoilY-CoilZ pair with a reported dissociation constant (K<sub>D</sub>) of less than 15 nM (Reinke <em>et al.</em>, 2010). Either CoilY or CoilZ could serve as the genetically-encoded tag. In 2018, we reported the VIPER tag, which enables high-affinity labeling of proteins for imaging by FM and CLEM (Doh <em>et al.</em>, 2018). Binding between the CoilE tag and the CoilR probe peptide to form VIPER is specific and nearly irreversible [K<sub>D</sub> ~10<sup>-11</sup> M] (Moll <em>et al.</em>, 2001).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191028/20191028194150_7586.jpg" width="177"/><br/><strong>Figure 1. Versatile interacting peptide (VIP) tags are a new technology for imaging proteins by FM, EM or CLEM.</strong> VIPER labeling of transferrin receptor 1 (TfR1) is mediated by heterodimer formation between the CoilE tag and a fluorescent CoilR probe peptide. Fluorescent micrograph: VIPER-tagged TfR1 labeled with CoilR-Cy5 (magenta) and colocalized with fluorescent transferrin (Tf-AF488; green) at the cell surface of transfected CHO TRVb cells (63x magnification). Magenta-green signal overlap appears white and nuclei are blue.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191028/20191028194224_2389.jpg" width="159"/><br/><strong>Figure 2. VIP tags are a versatile technology for multi-scale microscopy.</strong> After a target protein is tagged, it can be labeled using a variety of probe peptides selected for the particular application.<br/><br/>For VIP tags, the versatility is imparted by the customizable probe peptide. After introduction of the CoilE tag onto a target protein, the protein can be labeled with one of many different reporters attached to CoilR (Figure 2). For example, we imaged the transmembrane receptor, TfR1-CoilE, with CoilR-BODIPY, CoilR-Cy5 (see Figure 1), and CoilR-biotin (Doh <em>et al.</em>, 2018). In other words, the probe peptide can be customized for different studies or imaging systems without changing the genetic tag. This is possible because CoilR encodes a single cysteine residue for site-specific modification via thiol-maleimide chemistry. The CoilR probe peptide can be bioconjugated to a variety of probes, including fluorophores, small molecules (<em>e.g.</em>, biotin), or nanoparticles. Many companies sell thiol-reactive probes, which makes this conjugation reaction accessible to labs without synthetic chemistry expertise. For more information on bioconjugation reactions, we recommend reading Hermansonâs <em>Bioconjugate Techniques</em> (Hermanson, 2013).<br/><br/>In this Bio-Protocol, we provide methods for making CoilR probe peptides that can be used for VIPER-labeling of cellular proteins for imaging by FM or EM. The CoilR peptide and the CoilE tag sequences are provided in Table 1. As described in prior work (Doh <em>et al.</em>, 2018), we used gene assembly PCR to enable the recombinant expression of probe peptides in <em>E. coli</em>. The method for peptide expression is described in Procedure A. CoilR was designed to interact with CoilE via an optimized alpha-helical coil-coil, as originally described by Vinson and coworkers (Moll <em>et al.</em>, 2001). We included a hexahistidine tag at the C-terminus of CoilR for purification by immobilized metal affinity chromatography (IMAC) (Hochuli <em>et al.</em>, 1987); this is described in Procedure B. <br/><br/><strong>Table 1. Sequences of CoilR and CoilE</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191028/20191028194544_4645.jpg" width="750"/><br/><span style="font-family:Arial;font-size:12px;"><sup>Â§</sup><em>Italics</em>: Linker sequence; <strong>Bold</strong>: Peptide coil; <strong><u>C</u></strong>: Cysteine (conjugation site).</span><br/><span style="font-family:Arial;font-size:12px;"><sup>â¡</sup>Heptad position: Residues <em>a</em> and <em>d</em> form a hydrophobic interface, residues at <em>e</em> and <em>g</em> form inter-strand salt bridges. </span><br/><br/>Procedures C and D describe thiol-maleimide reactions to label CoilR with a small molecule reporter. In Procedure C, we describe the method that we used to generate probe peptides in our prior work (Doh <em>et al.</em>, 2018). In Procedure D, we adapted a method described by Weiss and coworkers for solid state-based labeling of peptides (Kim <em>et al.</em>, 2008). Lastly, we include methods for purifying fluorophore-labeled (Procedure E) or biotinylated (Procedure F) probe peptide. This Bio-Protocol is accompanied by two companion articles, which include detailed methods for imaging VIPER-labeled cellular proteins by FM (Doh <em>et al.</em>, 2019a) and CLEM (Doh <em>et al.</em>, 2019b) (Figure 3). <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191028/20191028194719_2679.jpg" width="120"/><br/><strong>Figure 3. A decision tree for implementing VIPER.</strong> Procedures are color-coded by the publication in which they appear. Methods in this publication are color-coded purple. Methods in Doh <em>et al.</em>, 2019a are yellow and methods in Doh <em>et al.</em>, 2019b are orange. Publication 1: this article; Publication 2: Doh <em>et al.</em>, 2019a; Publication 3: Doh <em>et al.</em>, 2019b).</p>
        </body>

<p class="pptt" id="biaoti30095">## Materials and Reagents</p><p class="ppzz"><em>Note: â*â indicates a brand that is critical to the success of the experiment.</em><br/><strong><em>Materials</em></strong> </p><ol class="npwtwo"><li>Universal pipette tips (USA Scientific TipOne<sup>TM</sup>, catalog numbers: 1112-1770, 1163-1730, and 1121-3812)</li><li>Microcentrifuge tubes, 1.5 ml (Thermo Scientific, catalog number: 02-682-002)</li><li>Sterile serological pipettes (Thermo Scientific, catalog number: 13-678-11D + E)</li><li>Sterile 14 ml culture tubes (Corning, Falcon<sup>TM</sup>, catalog number: 352059)</li><li>Disposable polystyrene spectrophotometer cuvettes (Thermo Scientific, catalog number: 14-955-127)</li><li>Conical 50 ml tubes (Thermo Scientific, Nunc<sup>TM</sup>, catalog number: 12-565-270)</li><li>Chromatography column (Bio-Rad, Econo-Pac<sup>TM</sup>, catalog number: 7321010)</li><li>Ring stand (Fisher, catalog number: 11-474-207)</li><li>Adjustable ring stand clamps (United Scientific Supplies, catalog number: CLHD03)</li><li>Molecular weight cut off (MWCO) 3 kDa filters (Sigma-Aldrich, Amicon Ultra<sup>TM</sup>, catalog number: UFC900324)</li><li>Quartz 10.00 mm cuvette (Hellma Analytics, Ultra-Micro Cell, catalog number: 105-250-15-40)</li><li>Pipettes (<em>e.g.</em>, Rainin Pipet-Lite<sup>TM</sup> XLS, catalog numbers: 17014407, 17014411, 17014412, and 17014413)</li><li>Glass 2 L Erlenmeyer flask (Corning, Pyrex<sup>TM</sup>, catalog number: 49802L)</li></ol><br/><strong><em> Reagents</em></strong> <ol class="npwtwo"><li>Anti-biotin HRP antibody (Jackson Immunoresearch, catalog number: 200-032-211) </li><li>Streptavidin-HRP (Thermo Scientific, catalog number: ENN100)</li><li>pET28b(+)_CoilR [Available by MTA from OHSU or made as published (Doh <em>et al.</em>, 2018)]</li><li>BL21 (DE3) <em>E. coli</em> (New England Biolabs, catalog number: C2527I)</li><li>Glycine (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP381-500)</li><li>SOC outgrowth media (New England Biolabs, catalog number: C2527I)</li><li>Miller Luria-Bertani (LB) agar (BD Difco<sup>TM</sup>, catalog number: 244520)</li><li>Miller LB broth (BD Difco<sup>TM</sup>, catalog number: BD 244610)</li><li>2X YT (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP9743500)</li><li>Kanamycin sulfate (Thermo Scientific, Fisher Chemical, catalog number: BP906-5)</li><li>IPTG (GoldBio, catalog number: I2481C5)</li><li>*Ni-NTA agarose (Qiagen, catalog number: 30230)</li><li>*Pierce Monomeric Avidin Agarose (Thermo Scientific Pierce<sup>TM</sup>, catalog number: 20228)</li><li>Sodium Phosphate Monobasic Anhydrous (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP329-500)</li><li>Urea (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: U15 3)</li><li>Tris Base (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP152 5)</li><li>Tris HCl (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP153 1)</li><li>NaCl (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP358-1)</li><li>Glycerol (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP229-1)</li><li>Imidazole (ACROS Organics, catalog number: AC39674-1000)</li><li>Coomassie Brilliant Blue R-250 (Thermo Scientific, catalog number: 20278)</li><li>Methanol (Thermo Scientific, Fisher Chemical, catalog number: A412)</li><li>Acetone (Thermo Scientific, Fisher Chemical, catalog number: A18)</li><li>Nitrogen gas</li><li>Ammonium sulfate (EMD Millipore, catalog number: AX1385-1)</li><li>TCEP-HCl (GoldBio, catalog number: TCEP10)</li><li>Dithiothreitol (DTT) (Thermo Scientific, Molecular Probes<sup>TM</sup>, catalog number: D1532)</li><li>TC-grade DMSO (Sigma-Aldrich, catalog number: D2650-5X10ML)</li><li>*Sulfo-Cy5-maleimide (Lumiprobe, catalog number: 23380)</li><li>*Biotin-PEG2-maleimide (Thermo Scientific, catalog number: 21901BID)</li><li>D-Biotin (Ark Pharma, catalog number: AK-44010)</li><li>Pierce BCA assay kit (Thermo Fisher Scientific, catalog number: 23227)</li><li>12% Bis-Tris polyacrylamide protein gels (Bio-Rad Criterion<sup>TM</sup> XT, catalog number: 3450119)</li><li>MES (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP300-100)</li><li>NaH<sub>2</sub>PO<sub>4</sub> (Sigma-Aldrich, catalog number: S3139-250G)</li><li>Ponceau Red (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP103-10)</li><li>NaOH (Thermo Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP359-500)</li><li>Buffer B (Ni-NTA peptide purification) (see Recipes)</li><li>Buffer C (Ni-NTA peptide purification) (see Recipes)</li><li>Buffer E (Ni-NTA peptide purification) (see Recipes)</li><li>MES running buffer (see Recipes)</li><li>TCEP/SDS Loading Dye (5x) (see Recipes)</li><li>Coomassie Stain (see Recipes)</li><li>Destain Solution (see Recipes)</li><li>Tris-Buffered Saline (TBS) (see Recipes)</li><li>TBS Urea (see Recipes)</li><li>0.5 M TCEP (see Recipes)</li><li>TBS Urea Binding Buffer (see Recipes)</li><li>Solid state-based labeling (SSL) Buffer, pH 7.5 (see Recipes)</li><li>1 M DTT (see Recipes)</li><li>TBS Urea Imidazole (see Recipes)</li><li>Biotin Buffer (see Recipes)</li><li>Regeneration buffer (see Recipes)</li></ol><p class="pptt" id="biaoti30096">### Equipment</p><ol class="npwtwok"><li>Electronic pipettor (Eppendorf Easypet<sup>TM</sup>, catalog number: 4430000018)</li><li>-20 Â°C freezer (Thermo Scientific, Revco<sup>TM</sup>, catalog number: 13 990 206)</li><li>Incubator and shaker (New Brunswick Excella<sup>TM</sup> E24, catalog number: M1352-0010)</li><li>Spectrophotometer (Eppendorf, Biophotometer Plus, catalog number: 6132)</li><li>Rotisserie (Thermo Scientific, catalog number: 400110Q)</li><li>Sonifier (Branson Ultrasonics<sup>TM</sup>, catalog number: 101063198R)</li><li>Sonifier 1/8 inch micro-tip (Branson Ultrasonics<sup>TM</sup>, catalog number: 22-309796)</li><li>Refrigerated centrifuge (Thermo Scientific, Sorvall Legend XTR Centrifuge, catalog number: 75211731)</li><li>Microcentrifuge (Eppendorf, catalog number: 022620304)</li><li>Heat block (Fisher, Isotemp<sup>TM</sup>, catalog number: 88-860-022)</li><li>Electrophoresis cell (Bio-Rad Criterion<sup>TM</sup>, catalog number: 165-6001)</li><li>Power supply (Bio-Rad PowerPac<sup>TM</sup> HC, catalog number: 1645052)</li><li>Plate reader (Tecan Infinite M200 Pro, catalog number: 30050303)</li><li>(Optional) Fluorescence and western blot imager (<em>i.e.</em>, GE Healthcare Amersham<sup>TM</sup> Typhoon 5 multimode scanner, catalog number: <a class="blackNoUnder" target="_blank"> 29187191 </a> or Protein Simple, FluorChem Q)<br/></li></ol><p class="pptt" id="biaoti30097">### Procedure</p><ol class="npwone"><li>Expression of recombinant CoilR<br/>CoilR is generated by recombinant expression in <em>E. coli</em>. The growth and purification of CoilR follows standard protocols for making and purifying histidine-tagged peptides under inducible expression. For detailed background, protocols, and troubleshooting, we recommend referring to the <em>Qiaexpressionist</em> handbook (Qiagen) (Morimoto-Tomita <em>et al.</em>, 2003).<ol class="npwtwo"><li>Obtain or generate a plasmid encoding the CoilR peptide (<em>i.e.</em>, pET28b(+)_CoilR) (Doh <em>et al.</em>, 2018). The amino acid sequence of CoilR expressed from pET28b(+)_CoilR is provided in Table 1.<br/><em>Note: The pET28b(+)_CoilR plasmid encodes kanamycin resistance.</em> </li><li>Transform the plasmid into <em>E. coli</em> BL21 (DE3) competent cells, following NEBâs instructions for product C2527. <ol class="npwthree"><li>Plate cells on LB/agar/kanamycin (50 Î¼g/ml) and grow overnight at 37 Â°C. </li><li>Pick single colonies and inoculate 5 ml starter cultures (one colony per 5 ml culture) in LB supplemented with kanamycin (50 Î¼g/ml) in sterile 14 ml culture tubes. </li><li>Grow overnight in a shaking incubator (225 rpm, 37 Â°C). We grow several starter cultures in case of variation in growth is observed (<em>e.g.</em>, a culture grows slowly).</li></ol></li><li>Use one overnight culture to inoculate (2.5 ml, 1:200 dilution) 500 ml of 2X YT sterile media in a 2 L Erlenmeyer flask supplemented with kanamycin (50 Î¼g/ml). Grow at 225 rpm, 37 Â°C until the OD<sub>600</sub> reaches 0.8 to 1.0.<ol class="npwthree"><li>Monitor growth by measuring OD<sub>600</sub> of the culture in a disposable cuvette on a spectrophotometer.</li><li>It will take approximately 2-4 h for the culture to reach this OD<sub>600</sub>.</li><li>Prior to induction, take a 1 ml sample of the uninduced culture for peptide expression analysis by SDS-PAGE.<ol class="npwfour"><li>For each sample: Pellet 1 ml of bacterial culture in a microcentrifuge tube (10,000 <em>x g</em>, 2 min).</li><li>Resuspend the pellet in Buffer B (Recipe 1). Normalize the sample by adding Buffer B to the pellet. Use the equation: volume = OD<sub>600</sub> x 100 Î¼l Buffer B.<br/></li><li>Freeze at -20 Â°C.</li></ol></li></ol></li><li>Lower the temperature of the incubator/shaker to 25 Â°C and induce peptide expression for 2-4 h by addition of 0.1 mM IPTG.<br/><em>Note: The peptide will degrade if the induction is done at 37 Â°C, reducing the overall yield.</em> <ol class="npwthree"><li>Monitor expression by taking 1 ml samples every hour during the induction. Normalize as described in Step A3c.</li></ol></li><li>Harvest cells by centrifugation in a refrigerated centrifuge (5,000 <em>x g</em>, 15 min, 4 Â°C).<ol class="npwthree"><li>Discard the supernatant.</li><li>Transfer the pelleted bacteria to a tared 50 ml conical tube to obtain the weight of the wet pellet.</li><li>Store the pellet frozen (-20 Â°C). The pellet can be stored for several months at -20 Â°C.</li></ol><em>Note: If desired, cell lysis (Step B4) can be performed on the same day as the peptide expression. However, we typically freeze the pellet before proceeding to purification the next day.</em> </li><li>Analyze peptide expression by SDS-PAGE. See Figure 4 for a representative SDS-PAGE analysis of CoilR expression and purification.<br/><ol class="npwthree"><li>Thaw protein samples from time-points collected during induction. Freezing and thawing in the presence of Buffer B will partially lyse the cells. <br/><em>Note: If lysis is incomplete, freeze-thaw the samples several times to break open cells.</em> </li><li>Pellet insoluble debris (10 min, 10,000 <em>x g</em>). </li><li>Transfer the clarified lysate to a new microcentrifuge tube.</li><li>Add 5x TCEP-SDS loading dye (Recipe 5). Boil samples for 5 min, pellet by centrifugation, and then load 10-15 Î¼l sample/well onto the protein gel.</li><li>Analyze peptide expression by SDS-PAGE.<ol class="npwfour"><li>We recommend analyzing on a BioRad Criterion Bis-Tris Gel (12%; 26-well) run in MES running buffer (Recipe 4) at constant voltage (180 V). </li><li>Run until the loading dye reaches the bottom of the gel, approximately 35 min.</li><li>After electrophoresis, stain the protein gel with Coomassie stain (Recipe 6) and then destain (Destain solution, Recipe 7) before imaging.</li><li>CoilR will migrate on the gel as a monomer (7.5 kDa) and a dimer (15 kDa).<br/></li></ol></li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191028/20191028235909_4977.jpg" width="118"/><br/><strong>Figure 4. Peptide expression in </strong><em><strong>E. coli</strong></em><strong> and denaturing purification of CoilR.</strong> Samples were collected from uninduced (UI) and induced (2 h and 4 h) cells, lysed, and resolved by SDS-PAGE. CoilR was purified from clarified lysate by IMAC (<em>i.e.</em>, on Ni-NTA resin) under denaturing conditions. CL = clarified lysate; FT = flow-through (unbound fraction). Wash 1: Buffer B (pH 8) with 10 mM imidazole. Washes 2-4: Buffer C (pH 6.8) with 10 mM imidazole. Elutions 1-3: Buffer E (pH 4.3). The CoilR peptide (MW = 7502.35 Da) migrates as a monomer at ~6 kDa and an apparent dimer at ~14 kDa (gray arrowheads). <br/><br/></li></ol></li><li>Purification of recombinant CoilR by IMAC<ol class="npwtwo"><li>Prior to peptide purification, measure and adjust the pH of all purification buffers.<ol class="npwthree"><li>Buffer B: 8 M Urea, 100 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Tris-Cl, pH 8 (Recipe 1).</li><li>Buffer C: 8 M Urea, 100 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Tris-Cl, pH 6.5 (Recipe 2).</li><li>Buffer E: 8 M Urea, 100 mM NaH<sub>2</sub>PO<sub>4</sub>, 10 mM Tris-Cl, pH 4.5 (Recipe 3).</li></ol></li></ol><em>Note: Unless otherwise noted, all steps should be done on ice with pre-chilled buffers.</em><br/><ol class="npwtwo" start="2"><li>Thaw the <em>E. coli</em> pellet (from Step A5) on ice.</li><li>Resuspend the pellet in Buffer B. Use 5 ml Buffer B per gram of wet weight.</li><li>Lyse by sonication on ice. <br/><ol class="npwthree"><li>We use a Branson sonifier fitted with a 1/8â Branson microtip to lyse bacteria. We lysed cells at 40% duty cycle, output: 4. The sample was pulsed for 30 s and then left to rest for 1 min on ice for 8 cycles.</li><li>Avoid foaming of the sample, which will cause protein loss.</li><li>Alternatively, cells can be lysed by other methods (<em>e.g.</em>, freeze-thaw or French press).</li></ol></li><li>Clarify the lysate by centrifugation in a refrigerated centrifuge (10,000 <em>x g</em>, 30 min, 4 Â°C).<br/><em>Note: Keep a sample of the clarified lysate for analysis by SDS-PAGE.</em> </li><li>Incubate the clarified lysate with Ni-NTA agarose resin (Qiagen) for 1 h at 4 Â°C on a rotisserie.<ol class="npwthree"><li>Buffer B should be supplemented with 10-20 mM imidazole to reduce non-specific protein binding to the resin.</li><li>Use 1 ml of resin per 1 gram of pellet (wet weight).</li><li>Alternatively, bind at 4 Â°C overnight.</li></ol><em>Note: If you observe CoilR peptide in the flow-through and initial washes, then the resin was overloaded. Use more resin in the binding step.</em> </li><li>Load the lysate-resin mixture onto a clean, fritted chromatography column.</li><li>Collect the flow-through. Save a sample for analysis by SDS-PAGE.</li><li>Wash the resin with 5 column volumes (CV) of Buffer B.</li><li>Wash the resin with 10-50 CV of Buffer C.<ol class="npwthree"><li>The CoilR peptide will elute at ~pH 6. The wash buffer should be between pH 6.3 and pH 7.0.</li><li>Save a sample from each wash step to analyze by SDS-PAGE.</li><li>Wash until no further impurities elute.</li></ol></li><li>Elute the CoilR peptide in Buffer E. Elute in 5 fractions of 2 CV each. <ol class="npwthree"><li>Most of the peptide will elute in the first 3 elutions.</li><li>Elute in additional volume if you detect incomplete elution (see Figure 4).</li></ol></li><li>Monitor the purification by SDS-PAGE. Analyze samples from the clarified lysate, flow-through, washes, and elution. <ol class="npwthree"><li>Gel running conditions are the same as from Step A6d-A6e.</li></ol></li><li>Combine fractions containing the purified peptide based on results from Step B12.</li><li>Concentrate and exchange the purified CoilR into the desired buffer using a 3 kDa MWCO filter.<ol class="npwthree"><li>We recommend exchanging into TBS Urea (Recipe 9) using the MWCO filter. This buffer is compatible with the thiol-maleimide conjugation reaction (Procedure C).</li><li>Keep the peptide concentration between 0.5 mg/ml and 2 mg/ml to avoid issues with solubility. If a white precipitate is observed in the peptide solution, the peptide is too concentrated.</li></ol></li><li>Quantify the peptide concentration using the Pierce BCA assay kit (Thermo Fisher Scientific).<ol class="npwthree"><li>The protein concentration can also be estimated by measuring the absorbance at 280 nm. However, the extinction coefficient of CoilR is low (2,980 LÂ·mol<sup>-1</sup>Â·cm<sup>-1</sup>); so the concentration will be more accurate if determined by BCA assay.</li><li>Include replicates and dilutions to obtain an accurate concentration.</li></ol></li><li>Add 5-10% glycerol to the concentrated peptide, aliquot (200 Î¼l/tube), and freeze (-20 Â°C). CoilR peptide can be stored frozen for several months.</li></ol><br/></li><li>Generation of CoilR probe peptides by thiol-maleimide conjugation in solution<br/>VIPER-labeling is specific and efficient in living and fixed cells expressing CoilE-tagged protein. However, the quality of labeling is directly related to the quality of the probe peptide. This is because unlabeled CoilR and labeled CoilR will both dimerize with CoilE-tagged cellular proteins. We recommend using peptides that are &gt; 50% labeled with the reporter chemistry.<br/>  We have found that the efficiency of the thiol-maleimide bioconjugation reaction is variable. Therefore we have included two approaches for modifying CoilR: Procedure C and Procedure D. We have used both successfully to label CoilR with reporters, with the preferred protocol being dependent on the researcher. Procedure C describes a conventional thiol-maleimide conjugation reaction in solution; this is the method used to generate probe peptides described in our 2018 publication (Doh <em>et al.</em>, 2018). This method can be used to attach a fluorescent probe, such as Sulfo-Cyanine5 (Cy5)-maleimide, or to biotinylate CoilR. <ol class="npwtwo"><li>Prepare buffers, TCEP, and a stock solution of the reactive maleimide. If these stocks are already made, then proceed to Step C2.<ol class="npwthree"><li>We recommend labeling in TBS Urea (Recipe 9). The reaction should be done in a thiol-free buffer between pH 7 and pH 7.5.</li><li>Degas the buffer before using. This can be done by bubbling a stream of nitrogen gas through the buffer or by vacuum degassing.</li><li>Prepare 0.5 M TCEP (Recipe 10).</li></ol></li><li>Prepare a concentrated stock solution of the maleimide probe at 20-100 mg/ml in anhydrous DMSO. For Cy5-maleimide or other fluorophores, protect the solution from light. <ol class="npwthree"><li>More concentrated stocks are preferable to limit the amount of DMSO in the reaction.</li><li>Stocks can be stored at -20 Â°C.</li><li>The maleimide will hydrolyze in water, so storage in DMSO is recommended.</li></ol></li><li>Thaw the purified CoilR peptide on ice. The concentrated peptide stock should be in TBS Urea and degassed by nitrogen.<ol class="npwthree"><li>The reaction will proceed better if the peptide is concentrated. We recommend using a stock that is 2 mg/ml (~270 ÂµM).</li><li>A typical labeling reaction will include 50 to 200 nmoles of CoilR peptide.</li><li>If the peptide is not in an appropriate buffer, transfer into a different buffer at this point using a 3 kDa MWCO filter and degas before proceeding to Step C4.</li></ol></li><li>Reduce the peptide by the addition of a 10-fold molar excess of TCEP. Incubate for 30 min at 50 Â°C.</li><li>Initiate the conjugation reaction by adding at least 20-fold molar excess of the maleimide probe. Mix well. Incubate for 2 h at room temperature or at 4 Â°C overnight on a rotisserie.<ol class="npwthree"><li>For fluorophore-labeling, protect the reaction from light.</li></ol></li><li>After labeling, add TBS Urea Binding Buffer (Recipe 11) to a total volume of 15 ml.<ol class="npwthree"><li>Save a sample of the crude reaction mixture for analysis by SDS-PAGE.</li></ol></li><li>Concentrate and buffer exchange the crude reaction on a 3 kDa MWCO filter to remove unreacted probe. Buffer exchange into TBS Urea Binding Buffer which is compatible with Ni-NTA purification. Save a sample for analysis by SDS-PAGE.<ol class="npwthree"><li>For CoilR-fluorophores, continue the buffer exchange until the filtrate becomes colorless or stops changing color with subsequent buffer exchanges. Then proceed to Procedure E.</li><li>For biotinylated CoilR, a 40 ml wash is sufficient to remove most of the free biotin moieties. Then proceed to Procedure F.<br/><br/></li></ol></li></ol></li><li>Generation of CoilR probe peptides by thiol-maleimide chemistry using solid state-based labeling (SSL)<br/>In 2008, Weiss and coworkers described a new method for modifying proteins using thiol-maleimide chemistry (Kim <em>et al.</em>, 2008). In that work, the protein was first precipitated with ammonium sulfate and reduced with DTT before fluorophore conjugation. They named this method solid state-based labeling (SSL). The advantage of this approach is that it is easy to do, efficient (70-90% labeled) and thiol-specific (Kim <em>et al.</em>, 2008). We currently use both SSL and solution-based labeling to generate probe peptides. Procedure D is adapted from Weiss and coworkers published method (Kim <em>et al.</em>, 2008).<ol class="npwtwo"><li>Prepare SSL Buffer (Recipe 12) and the reducing agents (1 M DTT [Recipe 13] and 0.5 M TCEP). If these stocks are already made, then proceed to Step D2.</li><li>Thaw the purified CoilR peptide on ice.<ol class="npwthree"><li>A typical labeling reaction will include 50 to 200 nmoles of CoilR peptide.</li><li>The volume should be less than 1 ml, but the peptide will become concentrated by precipitation in Step D4.</li></ol></li><li>Reduce the peptide by the addition of 10 mM DTT, from a 1 M stock. Incubate for 30 min at 4 Â°C on a rotisserie.<br/></li><li>Precipitate the reduced peptide by slow addition of ammonium sulfate powder to a final concentration of 70-75%. <ol class="npwthree"><li>For an overview of protein precipitation, see the 1998 publication by Wingfield (Wingfield, 2001).</li><li>Encor Biotechnology has a useful online tool for calculating the amount of ammonium sulfate to add; see: <a href="http://www.encorbio.com/protocols/AM-SO4.htm" target="_blank">http://www.encorbio.com/protocols/AM-SO4.htm</a>.<br/><em>Example: For a 500 Âµl peptide solution at 4 Â°C, add 0.23 g of ammonium sulfate to get a 70% saturated solution.</em> </li></ol></li><li>After a precipitate forms, add 10 mM DTT and reduce for 2 h at 4 Â°C on a rotisserie.</li><li>Wash the reduced peptide slurry with ice-cold SSL Buffer to remove DTT.<ol class="npwthree"><li>Pellet the slurry by centrifugation (4 min, 14,000 <em>x g</em>, 4 Â°C). Discard the supernatant.</li><li>Add 1 ml SSL Buffer and invert the sample several times.</li><li>Repeat Steps D6a and D6b 3-5 times to remove all excess DTT.</li><li>After the last centrifugation step, resuspend the pellet in 100 Âµl SSL Buffer.</li></ol><em>Note: Any residual DTT will react with the maleimide probe so it is critical to wash the peptide pellet several times.</em> </li><li>Perform the thiol-maleimide conjugation on reduced peptide in the solid state.<ol class="npwthree"><li>Add 10- to 30-fold molar excess probe to the reduced peptide. Mix by inverting the tube several times.<br/><ol class="npwfour"><li>Use a concentrated stock (20-100 mg/ml) of maleimide probe (<em>e.g.</em>, Cy5-maleimide) in anhydrous DMSO.</li></ol></li><li>Mix the reaction on a rotisserie for 15 min at 4 Â°C.</li><li>Add 5- to 10-fold molar excess TCEP. Mix and continue to incubate for 45 min at 4 Â°C on a rotisserie.<ol class="npwfour"><li>For fluorophore labeling, protect the tube from light.</li><li>Keep the amount of the maleimide higher than the amount of TCEP in the reaction because the maleimide probe can undergo a side-reaction with TCEP (Kim <em>et al.</em>, 2008).</li><li>The reaction can be incubated overnight.</li></ol></li></ol></li><li>After labeling, we recommend washing the reaction mixture with SSL Buffer to remove excess maleimide.<ol class="npwthree"><li>Pellet the reaction by centrifugation (4 min, 14,000 <em>x g</em>, 4 Â°C). Discard the supernatant.</li><li>Resuspend in SSL Buffer (1 ml).</li><li>Pellet by centrifugation (4 min, 14,000 <em>x g</em>, 4 Â°C). Discard the supernatant.</li></ol></li><li>Resuspend the pellet from Step D8 in Buffer B.<br/><em>Notes: </em> <ol class="npwthree"><li><em>High concentrations of EDTA will strip nickel from the Ni-NTA agarose used in Procedure E. Add enough Buffer B to ensure that the final concentration of EDTA is less than 1 mM.</em> </li><li><em>Save a sample of the crude reaction mixture for analysis by SDS-PAGE.</em> </li></ol></li><li>Proceed to purification, following Procedure E (for fluorescent peptides) or Procedure F (for biotinylated peptides).<br/><br/></li></ol></li><li>Purification of CoilR-Fluorophore probe peptide<br/>This procedure removes excess unreacted free dye from fluorophore-labeled CoilR (<em>i.e.</em>, CoilR-Cy5) while also purifying the peptide. This section additionally describes the method used to quantify the fluorophore labeling of the peptide. <br/><em>Note: Protect the peptide from light and keep the peptide on ice, unless otherwise noted.</em> <ol class="npwtwo"><li>Bind the labeled CoilR peptide to Ni-NTA agarose resin for 1 h at 4 Â°C.<br/><ol class="npwthree"><li>For a typical labeling reaction (50-200 nmol CoilR), we recommend using 0.5 ml Ni-NTA resin and binding in a large volume (20-40 ml) of TBS Urea Binding Buffer.</li></ol></li><li>Load the lysate-resin mixture onto a clean, fritted chromatography column.</li><li>Collect the flow-through and save a sample for analysis by SDS-PAGE.<br/></li><li>Wash the resin with 20 column volumes (CV) of TBS Urea Binding Buffer.<br/><ol class="npwthree"><li>Continue washing until fractions are colorless.</li><li>Save washes for analysis by SDS-PAGE.</li></ol></li><li>Optional step: Wash the resin with 10 CV of TBS Urea Binding Buffer supplemented with 20% ethanol. The addition of ethanol can help remove free fluorophore.</li><li>Elute the CoilR peptide with 5-10 CV of TBS Urea Imidazole (Recipe 14).<br/><ol class="npwthree"><li>Fractions should be dark blue for Cy5-labeled peptide.</li><li>Continue to elute until the fractions are nearly colorless before proceeding to the next step.</li><li>Alternatively, elute in a low pH buffer (<em>e.g.</em>, Buffer E).</li></ol></li><li>Analyze the purification by SDS-PAGE, following Steps A6d-A6e. For a representative analysis see Figure 5.<br/><ol class="npwthree"><li>Analyze the crude reaction, samples from the purification (flow-through, washes, elutions), and the concentrated elution.</li><li>Image the gel on a fluorescence scanner to detect labeled peptide. A representative 2-color scan acquired using a Protein Simple imaging system is provided in Figure 5A. Alternatively, we recommend imaging on a GE Amersham<sup>TM</sup> Typhoon multimode scanner using the appropriate detection settings (<em>i.e.</em>, Cy5: ex: 635 nm, em: 670/30 nm).</li><li>After fluorescence imaging, stain the protein gel with Coomassie stain, destain, and image to detect total protein.</li></ol></li><li>Concentrate and buffer exchange the elutions containing labeled peptide into a storage buffer of choice using a 3 kDa MWCO filter.<br/><em>Note: We recommend storing the peptide in TBS urea.</em> </li><li>Determine the degree of labeling (moles of fluorophore per mole of protein). We recommend following the protocol published by Thermo Scientific [Tech Tip #31: Calculate dye:protein (F/P) molar ratios] (Reference 22).<ol class="npwthree"><li>Determine the amount of fluorophore in the solution by measuring the absorption at the fluorophoreâs absorbance maximum (Abs<sub>max</sub>) and using the published extinction coefficient (Îµ<sub>FL</sub>) (Table 2).</li><li>Determine the amount of peptide in the solution by measuring the absorbance at 280 nm.<ol class="npwfour"><li>The fluorophore will also absorb at 280 nm and a correction factor (CF) must be used (<em>e.g.</em>, CF for Cy5 = 0.04).</li><li>The extinction coefficient (Îµ) of the CoilR peptide is 2,980 LÂ·mol<sup>-1</sup>Â·cm<sup>-1</sup> (<a href="http://www.expasy.org/" target="_blank">www.expasy.org</a>).</li></ol></li><li>Calculate the molarity of the peptide and the degree of labeling using the following equations:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029000343_9684.jpg" width="250"/><br/><br/></li><li>We offer the following recommendations:<ol class="npwfour"><li>Absorbance readings are only accurate in the linear range of the spectrophotometer (between 0.1 and 1.0).</li><li>We suggest preparing several dilutions of the peptide and replicates to obtain more accurate results.</li><li>We measured absorbance in a quartz cuvette on a Tecan Infinite M200 Pro with a cuvette port.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029000446_3360.jpg" width="110"/><br/><strong>Figure 5. Analysis of CoilR probe peptides by SDS-PAGE.</strong> CoilR was labeled with sulfo-Cyanine3 (CoilR-Cy3; 60% labeled), AlexaFluor-488 (CoilR-AF488; 45% labeled), or BODIPY-FL (CoilR-BDPY-FL: 40% labeled). The crude reaction (CR) was purified on Ni-NTA resin to remove free dye. Samples were resolved by SDS-PAGE and the gel was scanned for green (ex: 488 nm, em: 525/50 nm) and red (ex: 532 nm, em: 570/20 nm) fluorescence (A). The same gel was subsequently stained for total protein with Coomassie (B). CR = diluted crude reaction (pre-column), FT = flow-through (unbound protein/fluorophore), W = wash (TBS Urea Binding Buffer), E = elution (TBS Urea Imidazole). Unreacted CoilR peptide (15 and 30 Âµg) was included for reference and CoilR is indicated by a gray arrowhead. <br/><br/><strong>Table 2. Values for quantifying CoilR labeling with Cy5-maleimide<sup>â¡</sup></strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029000714_6843.jpg" width="550"/><br/><sup>â¡</sup>Values provided on the Lumiprobe website: <a href="http://www.lumiprobe.com/" target="_blank">www.lumiprobe.com</a>.<br/><br/></li></ol></li></ol></li><li>Store the fluorophore-labeled peptide in 5-10% glycerol.<br/><ol class="npwthree"><li>Aliquot (100 Î¼l/tube) and freeze (-20 Â°C). The peptide can be stored (frozen and protected from light) for several months.</li><li>Final stocks should be between 1-50 Î¼M for experimental convenience.</li><li>To minimize freeze-thaw cycles, a thawed aliquot can be divided into smaller single-use volumes (<em>e.g.</em>, 10 Î¼l) and re-frozen.</li></ol><br/></li></ol></li><li>Purification of biotinylated probe peptide (CoilR-biotin)<br/>This procedure is intended for purifying CoilR peptide that was biotinylated using Procedure C or D. For an overview of avidin-based affinity chromatography and a troubleshooting guide, refer to the Pierce<sup>Â®</sup> Monomeric Avidin Agarose instructions, available online (Reference 18). After purification and elution from the monomeric avidin resin, the CoilR-biotin peptide is assumed to be 100% biotinylated.<ol class="npwtwo"><li>Prepare the buffers and equilibrate them to room temperature.<ol class="npwthree"><li>TBS: 20 mM Tris, 150 mM NaCl pH 7 (Recipe 8).</li><li>Biotin Buffer: 2 mM biotin in DPBS pH 7.4 (Recipe 15).<br/></li><li>Regeneration Buffer: 0.1 M glycine, pH 2.8 (Recipe 16).</li></ol></li><li>Add the Pierce Monomeric Avidin Agarose to a clean, fritted chromatography column and drain. <ol class="npwthree"><li>For 100 nmoles of CoilR peptide, use 1 ml of resin.</li></ol></li><li>Block non-reversible biotin binding sites on the resin:<ol class="npwthree"><li>Wash with 5 column volumes (CV) of TBS.</li><li>Wash with 5 CV of Biotin Buffer to block any non-reversible biotin binding sites.</li><li>Wash with 5 CV of Regeneration Buffer to remove biotin bound to reversible biotin-binding sites on the resin.</li><li>Wash with 5 CV of TBS to re-equilibrate the column.</li><li>Plug the column to prevent flow; the resin is now ready to be used.</li></ol></li><li>Dilute the biotinylated peptide sample to approximately 5 ml in TBS. Apply to the column.</li><li>Incubate the sample with the resin for 30 min at room temperature.</li><li>Unplug the column and collect the flow-through. Save a sample of the flow-through and all subsequent wash and elution steps for analysis by SDS-PAGE.</li><li>Wash the resin twice with 5 CV of TBS.</li><li>Elute the biotinylated protein in 5 CV of Biotin Buffer. Collect 1 ml fractions.</li><li>Elute in 5 CV of Regeneration Buffer. Collect 1 ml fractions. This elution step is included because some peptides do not elute with excess biotin.</li><li>Regenerate the resin. Wash with 5 CV of Regeneration Buffer. Collect and analyze to ensure that this wash does not contain biotinylated peptide.</li><li>Analyze all fractions by SDS-PAGE (see Steps A6d-A6e).<br/><em>Note: In our experience, CoilR-biotin elutes in both the Biotin Buffer and the Regeneration Buffer, with more eluting in the Biotin Buffer.</em> </li><li>Analyze all fractions by Western blot (using your preferred method) to detect biotinylated proteins. For example, we detect biotinylated proteins using either an anti-biotin HRP antibody (Jackson Immunoresearch) or using a streptavidin-HRP (Thermo Scientific).</li><li>Combine fractions containing biotinylated peptide based on the analysis in Steps F11-F12.</li><li>Concentrate and exchange the biotinylated peptide into desired buffer using a 3 kDa MWCO filter.<ol class="npwthree"><li>We recommend exchanging into TBS Urea.</li><li>Keep the peptide concentration between 0.5 mg/ml and 2 mg/ml.</li></ol></li><li>Quantify the protein yield of the purification using the Pierce BCA assay kit.<ol class="npwthree"><li>The crude reaction will contain unmodified and biotinylated peptide. The amount of CoilR-biotin retrieved after monoavidin-based purification is thus expected to be less than the amount of CoilR used in the reaction.</li><li>The biotinylation (%) of the peptide in the crude reaction mixture can be estimated by dividing the nmoles of CoilR-biotin obtained from the monoavidin purification by the nmoles of CoilR in the labeling reaction.</li><li>The CoilR-biotin obtained from this procedure is presumed to be 100% biotinylated once it is eluted from the monoavidin column because unreacted peptide (<em>i.e.</em>, CoilR) will not bind to the resin.</li></ol></li><li>Store the biotin-labeled peptide in 5-10% glycerol.<ol class="npwthree"><li>Aliquot (100 Î¼l/tube) and freeze (-20 Â°C). The peptide can be stored for several months.</li><li>Final stocks should be between 1 Î¼M and 50 Î¼M for experimental convenience.</li><li>To minimize freeze-thaw cycles, a thawed aliquot can be divided into smaller single-use volumes (<em>e.g.</em>, 10 Î¼l) and re-frozen.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti30098">### Recipes</p><p class="ppzz"><em>Notes:</em> </p><ol class="npwthree"><li><em>Buffers are made in autoclaved DI water unless otherwise stated.</em> </li><li><em>The pH of Tris buffers changes with temperature.</em> </li><li><em>The pH of urea-containing buffers (Buffer B, Buffer C, and Buffer E) should be checked and adjusted immediately prior to use.</em> </li><li><em>The pH of DPBS is 7.0.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwtwo"><li>Buffer B (Ni-NTA peptide purification)<br/>8 M Urea<br/>100 mM NaH<sub>2</sub>PO<sub>4</sub><br/>10 mM Tris-Cl pH 8.0</li><li>Buffer C (Ni-NTA peptide purification)<br/>8 M Urea<br/>100 mM NaH<sub>2</sub>PO<sub>4</sub><br/>10 mM Tris-Cl pH 6.5</li><li>Buffer E (Ni-NTA peptide purification)<br/>8 M Urea<br/>100 mM NaH<sub>2</sub>PO<sub>4</sub><br/>10 mM Tris-Cl pH 4.5</li><li>MES running buffer<br/>50 mM MES<br/>50 mM Tris pH 7.3<br/>1 mM EDTA<br/>0.1% (w/v) SDS</li><li>TCEP/SDS Loading Dye (5x)<br/>300 mM Tris pH 6.8<br/>50 mM TCEP<br/>10% (w/v) SDS<br/>65% (v/v) glycerol<br/>0.025% (v/v) Ponceau Red</li><li>Coomassie Stain<br/>45% (v/v) methanol<br/>0.3% (w/v) Coomassie Brilliant Blue R-250<br/>10% v/v acetic acid</li><li>Destain Solution<br/>20% (v/v) methanol<br/>10% (v/v) acetic acid</li><li>Tris-Buffered Saline (TBS)<br/>20 mM Tris pH 7.4<br/>150 mM NaCl </li><li>TBS Urea<br/>20 mM Tris pH 7.4<br/>150 mM NaCl<br/>2 M Urea </li><li>0.5 M TCEP <br/>Dissolve the TCEP and then adjust the pH to 7 by the addition of 10 M NaOH <br/><em>Note: Single-use aliquots of TCEP can be stored at -20 Â°C.</em> </li><li>TBS Urea Binding Buffer<br/>20 mM Tris pH 8.0<br/>150 mM NaCl<br/>2 M Urea </li><li>Solid state-based labeling (SSL) Buffer, pH 7.5<br/>125 mM NaH<sub>2</sub>PO<sub>4</sub><br/>200 mM NaCl<br/>1.25 mM EDTA <br/>4.6 M Ammonium Sulfate (75% saturated solution)</li><li>1 M DTT<br/>Dissolve the DTT in autoclaved DI water<br/><em>Note: Single-use aliquots can be prepared and stored at -20 Â°C.</em> </li><li>TBS Urea Imidazole<br/>20 mM Tris pH 7.4<br/>150 mM NaCl<br/>2 M Urea <br/>500 mM imidazole</li><li>Biotin Buffer<br/>2 mM biotin in DPBS </li><li>Regeneration buffer<br/>0.1 M glycine<br/>pH 2.8</li></ol><p class="pptt" id="biaoti30099">### Acknowledgments</p><p class="ppzz">KEB is grateful for support from the OHSU School of Medicine and the National Institutes of Health (R01 GM122854). JKD was partially funded by the Portland Chapter of Achievement Rewards for College Scientists (ARCS). The protocols described herein were originally described in two prior publications (Zane <em>et al.</em>, 2017; Doh <em>et al.</em>, 2018). We are grateful to our colleagues at OHSU, particularly Drs. Hannah Zane and Jonathan White, for their contributions to the development of the VIP tags.</p><p class="pptt" id="biaoti30100">### Competing interests</p><p class="ppzz">The authors declare no financial or non-financial competing interests. A patent application is pending on the VIP technology (PCT/US17/60609).</p><p class="pptt" id="biaoti30101">### References</p><ol class="npwtwok"><li>Baker, M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25993940" target="_blank">Reproducibility crisis: Blame it on the antibodies.</a> <em>Nature</em> 521(7552): 274-276.</li><li>Berglund, L., Bjorling, E., Oksvold, P., Fagerberg, L., Asplund, A., Szigyarto, C. A., Persson, A., Ottosson, J., Wernerus, H., Nilsson, P., Lundberg, E., Sivertsson, A., Navani, S., Wester, K., Kampf, C., Hober, S., Ponten, F. and Uhlen, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18669619" target="_blank">A genecentric Human Protein Atlas for expression profiles based on antibodies.</a> <em>Mol Cell Proteomics</em> 7(10): 2019-2027.</li><li>Bordeaux, J., Welsh, A., Agarwal, S., Killiam, E., Baquero, M., Hanna, J., Anagnostou, V. and Rimm, D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20359301" target="_blank">Antibody validation.</a> <em>Biotechniques</em> 48(3): 197-209.</li><li>Bradbury, A. and Pluckthun, A.(2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25652980" target="_blank">Reproducibility: Standardize antibodies used in research.</a> <em>Nature</em> 518(7537):27-29.</li><li>Cranfill, P. J., Sell, B. R., Baird, M. A., Allen, J. R., Lavagnino, Z., de Gruiter, H. M., Kremers, G. J., Davidson, M. W., Ustione, A. and Piston, D. W. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27240257" target="_blank">Quantitative assessment of fluorescent proteins.</a> <em>Nat Methods</em> 13(7): 557-562.</li><li>Doh, J. K., Enns, C. A. and Beatty, K. E. (2019a). <a href="https://bio-protocol.org/e3413" target="_blank">Implementing VIPER for imaging cellular proteins by fluorescence microscopy.</a> <em>Bio-protocol</em> 9(21): e3413.</li><li>Doh, J. K., Chang, Y. H., Enns, C. A., Lopes, C. S. and Beatty, K. E. (2019b). <a href="https://bio-protocol.org/e3414" target="_blank">Imaging VIPER-labeled cellular proteins by correlative light and electron microscopy. </a><em>Bio-protocol</em> 9(21): e3414.</li><li>Doh, J. K., White, J. D., Zane, H. K., Chang, Y. H., LÃ³pez, C. S., Enns, C. A. and Beatty, K. E. (2018). <a href="https://www.researchgate.net/publication/329428091_VIPER_is_a_genetically_encoded_peptide_tag_for_fluorescence_and_electron_microscopy" target="_blank">VIPER is a genetically encoded peptide tag for fluorescence and electron microscopy.</a> <em>Proc Natl Acad Sci U S A</em> 115(51): 12961-12966.</li><li>Ellisman, M. H., Deerinck, T. J., Shu, X. and Sosinsky, G. E. (2012) <a href="http://europepmc.org/articles/PMC3532940" target="_blank">Chapter 8 - Picking faces out of a crowd: Genetic labels for identification of proteins in correlated light and electron microscopy imaging.</a> In <em>Methods in cell biology</em>. <em>Academic Press</em> 111: 139-155.</li><li>Griffiths, G. and Hoppeler, H. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3534077" target="_blank">Quantitation in immunocytochemistry: correlation of immunogold labeling to absolute number of membrane antigens.</a> <em>J Histochem Cytochem </em>34(11): 1389-1398.</li><li>Griffiths, G. and Lucocq, J. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25151300" target="_blank">Antibodies for immunolabeling by light and electron microscopy: not for the faint hearted.</a> <em>Histochem Cell Biol</em> 142(4): 347-360.</li><li>Hermanson, G. T. (2013). <a href="https://www.sciencedirect.com/book/9780123822390/bioconjugate-techniques" target="_blank"><em>Bioconjugate Techniques</em>.</a> 3rd ed. Academic Press. ISBN: 978-0-12-382239-0.</li><li>Hochuli, E., Dobeli, H. and Schacher, A. (1987). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3443622" target="_blank">New metal chelate adsorbent selective for proteins and peptides containing neighbouring histidine residues.</a> <em>J Chromatogr</em> 411: 177-184.</li><li>Kim, Y., Ho, S. O., Gassman, N. R., Korlann, Y., Landorf, E. V., Collart, F. R. and Weiss, S. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18275130" target="_blank">Efficient site-specific labeling of proteins via cysteines.</a> <em>Bioconjug Chem</em> 19(3): 786-791.</li><li>Liu, Z., Lavis, L. D. and Betzig, E. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26000849" target="_blank">Imaging live-cell dynamics and structure at the single-molecule level.</a> <em>Mol Cell </em>58(4): 644-659.</li><li>Moll, J. R., Ruvinov, S. B., Pastan, I. and Vinson, C. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11344333" target="_blank">Designed heterodimerizing leucine zippers with a ranger of pIs and stabilities up to 10(-15) M.</a> <em>Protein Sci</em> 10(3): 649-655.</li><li>Morimoto-Tomita, M., Uchimura, K. and Rosen, S. D. (2003). <a href="https://www.researchgate.net/publication/270543927_Novel_Extracellular_Sulfatases_Potential_Roles_in_Cancer" target="_blank">Novel extracellular sulfatases: Potential roles in cancer.</a> <em>Trends Glycosci Glycotechnol</em> 15(83):159-164.</li><li>PierceÂ® Monomeric Avidin Agarose. Thermo Scientific Inc., 2011. (Accessed 12-Oct, 2019, at <a href="https://www.thermofisher.com/order/catalog/product/20228." target="_blank">https://www.thermofisher.com/order/catalog/product/20228.</a>)</li><li>Reinke, A. W., Grant, R. A. and Keating, A. E. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20387835" target="_blank">A synthetic coiled-coil interactome provides heterospecific modules for molecular engineering.</a> <em>J Am Chem Soc</em> 132(17): 6025-6031.</li><li>Rodriguez, E. A., Campbell, R. E., Lin, J. Y., Lin, M. Z., Miyawaki, A., Palmer, A. E., Shu, X., Zhang, J. and Tsien, R. Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27814948" target="_blank">The growing and glowing toolbox of fluorescent and photoactive proteins.</a> <em>Trends Biochem Sci</em> 42(2): 111-129.</li><li>Schnell, U., Dijk, F., Sjollema, K. A. and Giepmans, B. N. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22290187" target="_blank">Immunolabeling artifacts and the need for live-cell imaging.</a> <em>Nat Methods</em> 9(2): 152-158.</li><li>TECH TIP #31 Calculate dye:protein (F/P) molar ratios. Thermo Scientific Inc., 2011. (Accessed 12-Oct, 2019, at <a href="https://assets.thermofisher.com/TFS-Assets/LSG/brochures/TR0031-Calc-FP-ratios.pdf" target="_blank">https://assets.thermofisher.com/TFS-Assets/LSG/brochures/TR0031-Calc-FP-ratios.pdf.</a>)</li><li>Tsien, R. Y. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9759496" target="_blank">The green fluorescent protein.</a> <em>Annu Rev Biochem</em> 67: 509-544.</li><li>Wingfield, P.(2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429073" target="_blank">Protein precipitation using ammonium sulfate.</a> <em>Curr Protoc Protein Sci</em> Appendix 3:Appendix 3F.</li><li>Zane, H. K., Doh, J. K., Enns, C. A. and Beatty, K. E. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28052473" target="_blank">Versatile Interacting Peptide (VIP) tags for labeling proteins with bright chemical reporters.</a> <em>Chembiochem</em> 18(5): 470-474.<br/></li></ol>