<body>
            <h1>A Mismatch-tolerant RT-LAMP Method for Molecular Diagnosis of Highly Variable Viruses</h1>
            <p><strong>Authors:</strong> Yingxue  Li, Yi Zhou, Yingying Ma, Rong Xu, Xia Jin, Chiyu Zhang</p>
            <h2>Abstract</h2>
            <p>Loop-mediated isothermal amplification (LAMP) has been widely used in the detection of pathogens. However, there are usually numerous variants in one viral pathogen and primers employed in LAMP can hardly match all these variants. The mismatches between the primers and the viral genomes, especially those at the 3â²-end of the primers, hinder LAMP reactions, leading to failure of the detection. Here, we present a mismatch-tolerant RT-LAMP protocol, which utilizes the 3â²-5â² exonuclease activity of the Q5 high-fidelity DNA polymerase to remove potential mismatched bases at the 3â²-end of the primers during LAMP amplification. Using HIV-1 as a proof-of-principle, we showed that this protocol could represent a promising tool for accurate detection of genetically unstable viruses in laboratory, hospital and field.</p>
            <p><strong>Keywords:</strong> Mismatch-tolerant RT-LAMP, Highly variable viruses, Mutant, HIV-1, Visual detection, High-fidelity DNA polymerase</p>
            <h2>Background</h2><p>Emerging and re-emerging infectious diseases are serious threats to global public health (Mehand <em>et al.</em>, 2018). Many outbreaks and epidemics have been caused by viruses, such as HIV-1, HCV, MERS-CoV, Ebola virus, A/H7N9 influenza virus, and more recently Zika virus. These viral diseases have led to high morbidity and mortality that disproportionally impacted on low-income countries (Van Doorn, 2017; Fenollar and Mediannikov, 2018; Waldman and Balskus, 2018). Rapid and accurate diagnosis of viral pathogens is crucial for the prevention and control of viral infectious diseases (Mehand <em>et al.</em>, 2018). <br/><br/>Isothermal amplification techniques represent a promising direction for the development of point-of-care testing (POCT) diagnostic tools especially in the low-income countries or resource-limited settings (de Paz <em>et al.</em>, 2014). Loop-mediated isothermal amplification (LAMP) is the most widely used isothermal amplification technology in biomedical research (Notomi <em>et al.</em>, 2000). Its principle is auto-cycling strand displacement DNA amplification reaction using Bst DNA polymerase with high strand displacement activity under isothermal condition. LAMP generally uses three pairs of primers, and two inner primers (FIP and BIP) are responsible for initiating the self-primed DNA synthesis of the dumbbell form DNA. However, the biggest challenge for the detection of viruses using LAMP is the high genetic diversity of some viral genomes, which exist in the forms of genotypes, subtypes, and/or quasispecies (Sanjuan <em>et al.</em>, 2010; Domingo and Perales, 2018). These diverse forms can easily cause mismatches with primers during their amplification, thereby resulting in a low sensitivity of detection and a limited spectrum of detection (Zhang <em>et al.</em>, 2017; Li <em>et al.</em>, 2019; Zhou <em>et al.</em>, 2019). It is virtually impossible to detect all variants or serotypes in one LAMP assay as the conserved regions in the genomes are usually too short to completely match the long (approximately 40 nt) inner primers (FIP and BIP) of the assay. Therefore, an underestimate of viral load or even failure of detection is common using RT-LAMP method, especially for highly variable RNA viruses (Waldman and Balskus, 2018; Zhou <em>et al.</em>, 2019). This may be the most important reason that limits the commercial application of LAMP in the diagnosis of viral infectious diseases (Wong <em>et al.</em>, 2018). To overcome this problem, we recently developed a mismatch-tolerant RT-LAMP method that contains a minuscule amount of high-fidelity DNA polymerase and utilizes its 3â²-5â² exonuclease activity to remove potential mismatched bases at the 3â²-end of the primers during amplification (Zhou <em>et al.</em>, 2019). The new method was demonstrated to be especially suited for the detection of highly variable viruses (Zhou <em>et al.</em>, 2019). In this paper, we provide a detailed protocol of the mismatch-tolerant RT-LAMP method using HIV-1 detection as an example (Table 1). </p>
        </body>

<br/><p class="ppzz"><strong>Table 1. Primers used for the HIV-1 RT-LAMP assay</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029234821_1582.jpg" width="700"/> </p><p class="pptt" id="biaoti30153">## Materials and Reagents</p><ol class="npwtwok"><li>Axygen<sup>Â®</sup> MicroVolume Extended-Length Filtered Pipet Tips (Axygen, catalog number: TXLF10)</li><li>Axygen<sup>Â®</sup> Universal Fit 100 Î¼l Filtered Pipet Tips (Axygen, catalog number: TF100RS)</li><li>Axygen<sup>Â®</sup> Universal Fit 200 Î¼l Filtered Pipet Tips (Axygen, catalog number: TF200RS)</li><li>Axygen<sup>Â®</sup> Universal Fit 1,000 Î¼l Filtered Pipet Tips (Axygen, catalog number: TF1000LRS)</li><li>Axygen 1.5 ml Snaplock Microtubes (Axygen, catalog number: MCT150CS)</li><li>Axygen 0.2 ml PCR<sup>Â®</sup> Tubes (Axygen, catalog number: PCR02C)</li><li>LightCycler<sup>Â®</sup> 480 Multiwell Plate 96,white (Roche, catalog number: 4729692001ï¼</li><li>LightCycler<sup>Â®</sup> 8-Tube Strips, white (Roche, catalog number: 6612601001)</li><li>DreamTaq<sup>TM</sup> Green PCR Master Mix (2x) (Thermo Fisher Scientific, catalog number: k1081)</li><li><p style="word-break: break-all;">pUC57-IN (containing partial fragment of HIV-1 integrase gene: AF033819.3. the sequence is 5â-ACGGTTAGGGCCGCCTGTTGGTGGGCGGGAATCAAGCAGGAATTTGGAATTCCCTACAATCCCCAAAGTCAAGGAGTAGTAGAATCTATGAATAAAGAATTAAAGAAAATTATAGGACAGGTAAGAGATCAGGCTGAACATCTTAAGACAGCAGTACAAATGGCAGTATTCATCCACAATTTTAAAAGAAAAGGGGGGATTGGGGGGTACAGTGCAGGGGAAAGAATAGTAGACATAATAGCAACAGACATACAAACTAAAGAATTACAAAAACAAATTACAAAAATTCAAAATTTTCGGGTTTATTACAGGGACAGCAGAAATCCACTTTGGAAAGGACCAGCAAAGCTCCTCTGGAAAGGTGAAGGGGCAGTAGTAATACAAGATAATAGTGACATAAAAGTAGTGCCAAGAAGAAAAGCAAAGATCATTAGGGATTATGGAAAACAGATGGCAGGTGATGATTGTGTGGCAAGTAGACAGGATGAGGATTAGAACATGGAAAAGTTTAGTAAAACACCATATGTATGTTTCAGGGAAAGCTAGGGGATGGTTTTATAGACATCACTATGAAAGCCCTATCGGATCCCGGGCCCGTCGACTG-3â²) (Synthesized by Shanghai BioSune Biotechnology Co., Ltd.)</p></li><li>Qubit<sup>TM</sup> RNA HS Assay Kit RNA (Life Technologies, catalog number: Q32855)</li><li>WarmStart RTx Reverse Transcriptase (NEB, catalog number: M0380L)</li><li>Q5<sup>Â®</sup> High-Fidelity DNA Polymerase (NEB, catalog number: M0491L)</li><li><em>Bst</em> 2.0 DNA Polymerase (NEB, catalog number: M0537L)</li><li>Fast Mutagenesis System (Transgen, catalog number: FM111)</li><li>QIAgen Viral RNA Mini Kit (Qiagen, catalog number: 52906)</li><li>QIAquick<sup>Â®</sup> Gel Extraction Kit (Qiagen, catalog number: GC-28706)</li><li>HiScribe T7 High Yield RNA Synthesis Kit (NEB, catalog number: E2040S)</li><li>WarmStart<sup>Â®</sup> Colorimetric LAMP 2x Master Mix (DNA &amp; RNA) (with cresol red) (NEB, catalog number: M1800S)</li><li>Isothermal Amplification Buffer Pack (NEB, catalog number: B0537S)</li><li>Magnesium Sulfate (MgSO<sub>4</sub>) Solution (NEB, catalog number: B1003S)</li><li>dNTP Set, 100 mM Solutions (Thermo Fisher, catalog number: R0186)</li><li>SYTO<sup>TM</sup> 9 Green Fluorescent Nucleic Acid Stain (Invitrogen, catalog number: S34854)</li><li>Nuclease-Free Water (not DEPC-Treated) (Ambion<sup>TM</sup>, catalog number: AM9938)</li><li>Agarose (Biowest, catalog number: BY-R0100) </li><li>Ultra-pure water (Genview, catalog number: GU3313-500)</li><li>GelRed Nucleic Acid Straining Dye (10000x) (TOROIVD, catalog number: RSD100-25)</li><li>Plasma samples (Previous samples from our laboratory)</li><li>Yeast extract (Oxoid, catalog number: LP0021)</li><li>Tryptone (Oxoid, catalog number: LP0042)</li><li>Agar (Shangxiang, catalog number: 120420)</li><li>Sodium chloride (HuShi, catalog number: 10019318)</li><li>Ampicillin trihydrate (Solarbio, catalog number: A7490-5)</li><li>50x TAE buffer (Meilunbio, catalog number: MA0004)</li><li>2000 DNA Marker (Yeasen, catalog number: 10501ES60)</li><li>5000 DNA Marker (Yeasen, catalog number: 10504ES60)</li><li>TIANprep Mini Plasmid Kit (Tiangn, catalog number: DP103-03)</li><li>Liquid LB medium (see Recipes)</li><li>100 mg/ml ampicillin solution (see Recipes)</li><li>Ampicillin-resistant solid medium (see Recipes)</li><li>Ampicillin-resistant liquid LB medium (see Recipes)</li><li>2% agarose gel (see Recipes)</li><li>1% agarose gel (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30154">### Equipment</p><ol class="npwtwok"><li>0.5-10 Î¼l Eppendorf Research<sup>Â®</sup> plus Adjustable Volume Pipettes (Eppendorf, catalog number: I32693E)</li><li>10-100 Î¼l Eppendorf Research<sup>Â®</sup> plus Adjustable Volume Pipettes (Eppendorf, catalog number: 251596Z)</li><li>20-200 Î¼l Eppendorf Research<sup>Â®</sup> plus Adjustable Volume Pipettes (Eppendorf, catalog number: 4830359)</li><li>100-1,000 Î¼l Eppendorf Research<sup>Â®</sup> plus Adjustable Volume Pipettes (Eppendorf, catalog number: 4847859)</li><li>NanoDrop<sup>TM</sup> 2000 Spectrophotometer (Thermo Fisher, model: NanoDrop<sup>TM</sup> 2000, catalog number: ND-2000)</li><li>LightCycler<sup>Â®</sup> 96 System (Roche, catalog number: 05815916001)</li><li>Bio-Rad CFX96<sup>TM</sup> Real-Time PCR System (Bio-Rad, catalog number: 785BR18555)</li><li>Eppendorf Centrifuge 5417R (Eppendorf, model: 5417R)</li><li>Eppendorf Mastercycler nexus (Eppendorf, catalog number:6325ZK904949)</li><li>Tanon Gel Image System (Tanon, model: 2500)</li><li>Tanon EPS 300 (Tanon, model: 300)</li><li>Tanon electrophoresis tank (Tanon, model: 400)<br/></li></ol><p class="pptt" id="biaoti30155">### Software</p><ol class="npwtwok"><li>LightCycler<sup>Â®</sup> 96 System software (Roche) </li><li>Bio-Rad CFX Manager 3.1 software (Bio-Rad)<br/></li></ol><p class="pptt" id="biaoti30156">### Procedure</p><ol class="npwone"><li>Preparation of HIV-1 wild type and mutant RNA standards<ol class="npwtwo"><li>Construct two mutant pUC57-IN plasmids using the fast mutagenesis system according to the following steps. Primers for construction of the mutant plasmids are in Table 2.<ol class="npwthree"><li>Prepare the PCR reaction mixes (Table 3), PCR cycling condition: Enzyme activation and pre-denaturation at 94 Â°C for 3 min, 25 cycles of denaturation at 94 Â°C for 20 s, annealing at 55 Â°C for 20 s and extension at 72 Â°C for 1 min, followed by extending at 72 Â°C for 10 min.<br/><br/><strong>Table 2. Primers for construction of two HIV-1 mutant plasmids</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029234913_2621.jpg" width="550"/><br/><em>Note: Introduced mutant is shown in Red font. F: forward primer; R: reverse primer; Mu: mutant.</em><br/><br/><strong>Table 3. PCR reaction mixes of the construction of two HIV-1 mutants</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029234940_3098.jpg" width="550"/><br/><em>Note: F: forward primer; R: reverse primer; Mu: mutant. </em><br/><br/></li><li>Electrophoresis detection: measure 10 Î¼l PCR product using 1% agarose gel electrophoresis at a constant voltage (140 V) for about 30 min.</li><li>PCR product digestion: add 1 Î¼l of DMT enzyme to the remaining PCR product, mix and incubate for 1 h at 37 Â°C.</li><li>Add 5 Î¼l of DMT digestion product to 50 Î¼l of competent cells, mix and place on ice for 30 min. Then place the mixture at 42 Â°C for 45 s, and on ice for 2 min.</li><li>Add 250 Î¼l of room temperature LB medium (without antibiotics) to the mixture, 200 rpm, 37 Â°C for 1 h.</li><li>Spread 100 Î¼l of the bacterial solution evenly on a plate containing 100 Î¼g/ml ampicillin and incubate overnight in a 37 Â°C incubator. </li><li>Pick the mono-clones into 3 ml LB medium containing 100 Î¼g/ml ampicillin, for shaking culture (200 rpm), at 37 Â°C for 10-12 h. </li><li>Extract the plasmid DNA for Sanger sequencing (Shanghai Platinum Company) and verify the presence of the mutation. Store the mutated plasmids at -20 Â°C for subsequent experiments.<br/><div style="white-space:nowrap;"></div></li></ol></li><li>Amplify the HIV-1 integrase segment using T7 promotor-containing primer pair (HIV T7-F: TAATACGACTCACTATAGACGGTTAGGGCCGCCTGT and HIV R: CAGTCGACGGGCCCGGGA) with the wild-type and mutant HIV-1 plasmids as templates. <ol class="npwthree"><li>In a 0.2 ml PCR<sup>Â®</sup> tube with cap, a PCR reaction mix includes 25 Î¼l 2x DreamTaq<sup>TM</sup> Green PCR Master Mix, 2 Î¼l of 10 Î¼M of each primer (HIV T7-F and HIV R primer) and 2 Î¼l HIV-1 plasmid in a final volume of 50 Î¼l. </li><li>The cycling condition is enzyme activation and pre-denaturation at 95 Â°C for 3 min, followed by 40 cycles of denaturation at 95 Â°C for 30 s, annealing at 55 Â°C for 30 s and extension at 72 Â°C for 1 min, fully extending at 72 Â°C for 10 min.</li></ol></li><li>Measure the PCR product using 2% agarose gel electrophoresis at a constant voltage (140 V) for about 30 min. </li><li>After electrophoresis, cut the specific PCR product from the gel under UV control and purify according to the manufacturerâ²s instruction using the QIAquick<sup>Â®</sup> Gel Extraction Kit. </li><li>Obtain RNAs through <em>in vitro</em> transcription with the purified DNA as templates using the HiScribe T7 High Yield RNA Synthesis Kit. </li><li>Quantify the obtained RNA using NanoDrop<sup>TM</sup> 2000 Spectrophotometer, and calculate the RNA copy number using the following formula: <br/><br/>RNA copies/ml = [RNA concentration (g/ml)/(nt transcript length Ã 340)] Ã 6.022 Ã 10<sup>23</sup><br/><br/></li><li>Dilute RNA to 10<sup>4</sup> copies/ul, aliquot to 40 Î¼l in a 0.2 ml PCR<sup>Â®</sup> tube with cap and store at -80 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Plasma sample viral RNA extraction<ol class="npwtwo"><li>Extract viral RNA from 140 Î¼l of plasma using the QIAgen Viral RNA Mini Kit following the manufacturerâ²s instruction.</li><li>Elute the extracted RNA in 50 Î¼l of nuclease-free water, aliquot to 25 Î¼l per tube, and store at -80 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Qualitative detection of HIV-1 by the real-time mismatch-tolerant HIV-1 RT-LAMP assay<ol class="npwtwo"><li>Preparation of mixed primer sets (Table 4)<br/>Prepare the mixed primer set 1 by adding 5 Î¼l 100 Î¼m each of primers AceIN-F3, AceIN-B3a, and AceIN-B3b into 85 Î¼l nuclease-free water; the mixed primer set 2 includes 20 Î¼l 100 Î¼m each of primers AceIN-FIPf and AceIN-FIPe, 40 Î¼l 100 Î¼m AceIN-BIP, and 20 Î¼l nuclease-free water; the mixed primer set 3 includes 20 Î¼l 100 Î¼m each of primers AceIN-LF and AceIN-LB, and 60 Î¼l nuclease-free water. </li><li>Preparation of the real-time RT-LAMP mix (Table 5)<br/>In a 96-well PCR plate or 8-tube PCR strip, each 25 Î¼l reaction mix includes 3 Î¼l RNA template (RNA standard or RNA extracts from clinical sample), 2.5 Î¼l 10x isothermal amplification buffer, 1 Î¼l 100mM MgSO<sub>4</sub>, 3.5 Î¼l 10 mM dNTPs, 0.075 Î¼l 2 units of Q5 high-fidelity DNA polymerase, 1 Î¼l 8 units of <em>Bst</em> 2.0 DNA polymerase, 0.5 Î¼l 15 units of warmstart RTx reverse transcriptase, 1 Î¼l 1 mM SYTO 9, and 1 Î¼l each mixed primer set (mixed primer set 1-3 in Table 4).<br/><br/><strong>Table 4. Mixed primer sets preparation of the mismatch-tolerant RT-LAMP assay</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029235105_9944.jpg" width="620"/><br/><em>Note: The regular assay is recommended when no degenerate primers is used. </em><br/><br/></li><li>RT-LAMP cycling condition: Perform the reaction at 62 Â°C for 1 min by 60 cycles. Collect fluorescence signal at each cycle.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Colorimetric RT-LAMP detection (Table 5)<ol class="npwtwo"><li>Preparation of the colorimetric RT-LAMP mix (Table 5) <br/>In an 8-tube PCR strip, each 25 Î¼l reaction mix includes 12.5 Î¼l warmstart colorimetric LAMP 2x master mix, 1 Î¼l each mixed primer set (mixed primer sets 1-3 in Table 4), 0.075 Î¼l 2 units of Q5 high-fidelity DNA polymerase, and 3 Î¼l of RNA template (RNA standard or RNA extracts from clinical sample). </li><li>Colorimetric RT-LAMP cycling condition: Perform the reaction at 62 Â°C for 50 min. Observe the color change at 20-, 30-, 40-, and 50-min time points by naked eyes.</li></ol></li></ol><p class="pptt" id="biaoti30157">### Data analysis</p><ol class="npwtwok"><li>The result of the real-time mismatch-tolerant RT-LAMP assay can be seen with LightCycler<sup>Â®</sup> 96 system software, Bio-Rad CFX manager 3.1 software (Figure 1) or softwares implemented in other real-time PCR machines. Positive results (with the presence of HIV-1 templates) show clear "S" type amplification curves, and negative results (without HIV-1 template) have no amplification curve. The Ct values (time to appearance of the amplification curve) are negatively related to the amount of template input.<br/><br/><img alt="" class="layerphoto" height="127" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029235222_1633.jpg" width="100"/><br/><strong>Figure 1. Amplification curves of the real-time mismatch-tolerant HIV-1 RT-LAMP.</strong> Amplification curves using LightCycler<sup>Â®</sup> 96 system (A) or Bio-Rad CFX96 TM Real-Time PCR system (B). HIV-1 RNA standard used as a positive control is shown in red, clinical samples from HIV-positive subjects are shown in cyan, and samples from HIV-negative subjects or that do not have template are shown in black. <br/><br/></li><li>In the Colorimetric RT-LAMP detection assay, the results can be judged by naked eyes. A clear color change from burgundy to orange or yellow is considered as HIV-1 positive at 30-50 min which is dependent on the set cut-off (Figure 2). <br/><br/><strong>Table 5. Real-time and colorimetric reaction mixes of the mismatch-tolerant RT-LAMP assay</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029235403_7726.jpg" width="620"/><br/><em>Note: Obtain 0.2 U/Î¼l Q5 high-fidelity DNA polymerase by ten-fold dilution of 2 U/Î¼l Q5 enzyme stock. </em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029235447_1536.jpg" width="240"/><br/><strong>Figure 2. Colorimetric RT-LAMP detection of HIV-1.</strong> The color change from burgundy to orange or yellow is considered as positive (+). NTC: no template control. The mutated base is marked in red. AceIN-F3 is an outer forward primer.<br/></li></ol><p class="pptt" id="biaoti30158">### Notes</p><ol class="npwtwok"><li>The LAMP reaction is very sensitive. Attention should be paid to avoid contamination during the operation, and a stringent laboratory compartmentalization is strongly recommended for LAMP and other amplification assays. </li><li>To avoid potential contamination, agarose gel electrophoresis of LAMP products is not encouraged. </li><li>Because only a small amount of Q5 high-fidelity DNA polymerase is used in each reaction, the enzyme stock can be diluted to a lower concentration (<em>e.g.</em>, 0.2 U/Î¼l) to reduce pipetting error. </li><li>Other high-fidelity DNA polymerases can also be used instead of Q5 enzyme in the reaction. The recommended optimal concentration of high-fidelity DNA polymerase is between 0.1 and 0.3 units per 25 Î¼l LAMP reaction.</li><li>The exact amount of Q5 high-fidelity DNA polymerase per 25 Î¼l LAMP reaction needs to be optimized for detection of each specific virus. </li><li>If there are no degenerate primers to be used, the primer mix can be prepared based on the standard assay (Table 2).</li><li>The amount of RNA template per 25 Î¼l LAMP reaction can be adjusted between the range of 0-11.75 Î¼l for the real-time version, and 0-8.75 Î¼l for the colorimetric version (Table 3). <br/></li></ol><p class="pptt" id="biaoti30159">### Recipes</p><ol class="npwtwok"><li>Liquid LB medium<br/>10 g of tryptone,5 g of yeast extract and 10 g of sodium chloride in 1 L of ultra-pure water<br/>Mix and autoclave it</li><li>100 mg/ml ampicillin solution<br/>1 g of ampicillin powder in 5 ml of ultra-pure water<br/>After it was dissolved, add ultra-pure water to make it to 10 ml<br/>Dispense the 100 mg/ml ampicillin solution into a 1.5 ml EP tube and store at -20 Â°C until use</li><li>Ampicillin-resistant solid medium<br/>10 g of tryptone, 5 g of yeast extract, 10 g of sodium chloride and 15 g of agar in 1 L of ultra-pure water<br/>Mix and autoclave it<br/>Add 1 ml of 100 mg/ml ampicillin solution and mix it<br/>Pour about 10 ml on each plate, and store at 4 Â°C after solidification</li><li>Ampicillin-resistant liquid LB medium<br/>Add 50 Î¼l of 100 mg/ml ampicillin solution in 50 ml liquid LB medium and mix it</li><li>2% agarose gel<br/>2 g of agarose in 100 ml of 1x TAE and 10 Î¼l 10000x GelRed</li><li>1% agarose gel<br/>1 g of agarose in 100 ml of 1x TAE and 10 Î¼l 10000x GelRed<br/></li></ol><p class="pptt" id="biaoti30160">### Acknowledgments</p><p class="ppzz">This protocol was modified from our original method published previously (Zhou <em>et al.</em>, 2019). This study was supported by grants from the National Natural Science Foundation of China (81672033 and U1302224), the National Science and Technology Major Project of China (2017ZX10103009-002), the âOne Belt One Roadâ Project (153831KYSB20170043) of Chinese Academy of Sciences, and the 133 projects of Institute Pasteur of Shanghai, CAS. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p><p class="pptt" id="biaoti30161">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti30162">### References</p><ol class="npwtwok"><li>de Paz, H. D., Brotons, P. and MuÃ±oz-Almagro, C. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25052202" target="_blank">Molecular isothermal techniques for combating infectious diseases: towards low-cost point-of-care diagnostics.</a> <em>Expert Rev Mol Diagn </em>14(7): 827-843. </li><li>Domingo, E. and Perales, C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29397419" target="_blank">Quasispecies and virus.</a> <em>Eur Biophys J</em> 47(4): 443-457.</li><li>Fenollar, F. and Mediannikov, O. (2018). <a href="https://doi.org/10.1016/j.nmni.2018.09.004" target="_blank">Emerging infectious diseases in Africa in the 21st century.</a> <em>New Microbes New Infect</em> 26: S10-S18.</li><li>Li, Y., Wan, Z., Hu, Y., Zhou, Y., Chen, Q. and Zhang, C. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31050303" target="_blank">A mismatch-tolerant RT-quantitative PCR: application to broad-spectrum detection of respiratory syncytial virus.</a> <em>Biotechniques</em> 66(5): 225-230. </li><li>Mehand, M. S., Al-Shorbaji, F., Millett, P. and Murgue, B. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30261226" target="_blank">The WHO R&amp;D Blueprint: 2018 review of emerging infectious diseases requiring urgent research and development efforts.</a> <em>Antiviral Res</em> 159: 63-67.</li><li>Notomi, T., Okayama, H., Masubuchi, H., Yonekawa, T., Watanabe, K., Amino, N. and Hase, T. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10871386" target="_blank">Loop-mediated isothermal amplification of DNA.</a> <em>Nucleic Acids Res</em> 28(12): E63.</li><li>Ocwieja, K. E., Sherrill-Mix, S., Liu, C., Song, J., Bau, H. and Bushman, F. D. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25675344" target="_blank">A reverse transcription loop-mediated isothermal amplification assay optimized to detect multiple HIV subtypes.</a> <em>PLoS One</em> 10(2): e0117852.</li><li>Sanjuan, R., Nebot, M. R., Chirico, N., Mansky, L. M. and Belshaw, R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20660197" target="_blank">Viral mutation rates.</a> <em>J Virol</em> 84(19): 9733-9748.</li><li>Van Doorn, H. R. (2017). <a href="https://doi.org/10.1016/j.mpmed.2017.09.002" target="_blank">Emerging infectious diseases.</a> <em>Medicine</em> 45: 798-801.</li><li>Waldman, A. J. and Balskus, E. P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29207239" target="_blank">The human microbiota, infectious disease, and global health: challenges and opportunities.</a> <em>ACS Infect Dis</em> 4(1): 14-26.</li><li>Wong, Y. P., Othman, S., Lau, Y. L., Radu, S., and Chee, H. Y. (2018). <a href="https://doi.org/10.1111/jam.13647" target="_blank">Loop-mediated isothermal amplification (LAMP): a versatile technique for detection of micro-organisms.</a> <em>J Appl Microbiol</em> 124(3): 626-643.</li><li>Zhang, M., Liu, K., Hu, Y., Lin, Y., Li, Y., Zhong, P., Jin, X., Zhu, X. and Zhang, C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28871131" target="_blank">A novel quantitative PCR mediated by high-fidelity DNA polymerase.</a> <em>Sci Rep</em> 7(1): 10365.</li><li>Zhou, Y., Wan, Z., Yang, S., Li, Y., Li, M., Wang, B., Hu, Y., Xia, X., Jin, X., Yu, N. and Zhang, C. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31139171" target="_blank">A mismatch-tolerant reverse transcription loop-mediated isothermal amplification method and its application on simultaneous detection of all four serotype of dengue viruses.</a> <em>Front Microbiol</em> 10: 1056.<br/></li></ol>