<body>
            <h1>High Throughput Traction Force Microscopy for Multicellular Islands on Combinatorial Microarrays</h1>
            <p><strong>Authors:</strong> Ian C. Berg, Gregory H. Underhill</p>
            <h2>Abstract</h2>
            <p>The composition and mechanical properties of the cellular microenvironment along with the resulting distribution of cellular devolved forces can affect cellular function and behavior. Traction Force Microscopy (TFM) provides a method to measure the forces applied to a surface by adherent cells. Numerous TFM systems have been described in literature. Broadly, these involve culturing cells on a flexible substrate with embedded fluorescent markers which are imaged before and after relaxion of cell forces. From these images, a displacement field is calculated, and from the displacement field, a traction field. Here we describe a TFM system using polyacrylamide substrates and a microarray spotter to fabricate arrays of multicellular islands on various combinations of extra cellular matrix (ECM) proteins or other biomolecules. A microscope with an automated stage is used to image each of the cellular islands before and after lysing cells with a detergent. These images are analyzed in a semi-automated fashion using a series of MATLAB scripts which produce the displacement and traction fields, and summary data. By combining microarrays with a semi-automated implementation of TFM analysis, this protocol enables evaluation of the impact of substrate stiffness, matrix composition, and tissue geometry on cellular mechanical behavior in high throughput. </p>
            <p><strong>Keywords:</strong> Traction force microscopy, TFM, Multicellular island, Microarray, Substrate stiffness, Microenvironment, Mechanobiology, Cellular forces</p>
            <h2>Background</h2><p>Cellular generated forces are transmitted to the extracellular matrix (ECM) through integrins, and to surrounding cells by cadherins. The signaling associated with the transduction of these forces plays an important role in the regulation of proliferation, differentiation, morphogenesis, and homeostasis. Further, the magnitude and distribution of cellular forces are impacted by the conditions of the microenvironment such as ECM composition, geometry, substrate stiffness, and the presence of other ligands or drugs (Discher <em>et al.</em>, 2009). We recently showed how ECM composition and substrate stiffness work together to affect liver progenitor cell differentiation and contractility (Kourouklis <em>et al.</em>, 2016), and that cell contractility and traction stress distribution can facilitate spatial patterning of liver progenitor cell differentiation (Kaylan <em>et al.</em>, 2018). Investigating the ways that the conditions of microenvironment such as ECM composition, geometry, substrate stiffness, and the presence of other ligands or drugs, mediate biomechanical and biochemical responses requires engineered systems that enable systematic control of the environment along with a way to assay the resulting behavior. <br/><br/>A variety of techniques have merged to measure cell-generated forces (Polacheck and Chen, 2016). Traction force microscopy (TFM) is a well-established method used to measure the forces applied by cells to the surface they are adhered to (Butler <em>et al.</em>, 2002). While this concept has been expanded to three-dimensional cell culture, this protocol with only address TFM in two dimensional monolayers. Since its establishment, many authors have implemented TFM systems using various approaches and algorithms (Schwarz and Soine, 2015; Kulkarni <em>et al.</em>, 2018). Broadly, in TFM, a flexible cell culture substrate is prepared with embedded fluorescent markers, often polymer beads. This substrate is modified such that cells will adhere, often by attaching a suitable ECM protein. Cells are then cultured on these substrates. Using an inverted fluorescence microscope, an image is taken the substrate surface, and the fluorescent markers, under each cell area of interest. The cells lysed using a detergent or enzyme, resulting in any forces applied to the surface by the cells to be relaxed. The same region is imaged once again. From these images, a displacement field is calculated using specialized software, based on the movement of the fluorescent markers after cell lysing. From the displacement field, a traction stress field is the calculated, using the known mechanical characteristics of the substrate.<br/><br/>In the MATLAB code provided with this protocol, the displacement field is calculated using a publicly available digital image correlation (DIC) algorithm (Landauer <em>et al.</em>, 2018). Traction forces calculation is performed using an available Fourier transform traction cytometry (FFTC) algorithm (Sabass <em>et al.</em>, 2008; Han <em>et al.</em>, 2015). These methods were chosen based on their relatively low computational expense and time and relatively few required user inputs. This supports the high throughput nature of the analysis and goal to allow âoff the shelfâ use for labs who do not focus on TFM as a core competency. However, the code was intended to enable users to substitute alternative displacement or traction field algorithms based on the needs and computational resources of the user and application. <br/><br/>Performing TFM often involves bulk preparations of adhesive substrates which requires relatively large amounts of adhesive proteins, decreasing the practicality of performing TFM on multiple ECM proteins or surface bound ligands. Here we use contact printed microarrays which allows multiple ECM/ligand combinations, with replicates, to be included on a single substrate with low material usage (Flaim <em>et al.</em>, 2005). TFM is often implemented either on single cells, or randomly distributed small colonies. In the microarray system, we can assess the mechanical behavior of multicellular islands where forces are transmit to the substrate as well as neighboring cells, resulting in collective behavior (Mertz <em>et al.</em>, 2013). Further, these multi cellular islands are of consistent size and shape. This permits analysis of average mechanical behavior of these islands as a function of the environmental conditions. Island diameter can be tuned by using differently sized microarray pins, adding geometry as an additional parameter to investigate. The MATLAB code provided includes techniques to identify island boundaries and align data from replicate islands to enable analysis of these average behaviors. These cellular micro-arrays can be assayed in parallel using immunocytochemistry, which has been described in more detailed elsewhere (Kaylan <em>et al.</em>, 2017), to allow correlation of mechanical and phenotypic behavior. <br/><br/>Our application has focused on the relationship between spatial patterns of mechanical behavior and differentiation of liver progenitor cells. This method can be applied to other stem cell systems especially where spatial patterning and collective mechanical behavior is of interest. It can also be applied to investigations of how the mechanical behavior of cancer relates to its microenvironment and further impacts proliferation or response to drugs. Although here we used contact microarray printing, this protocol can easily be applied to other micro patterned systems such as those with non-contact printing (Romanov <em>et al.</em>, 2014) and Polydimethylsiloxane (PDMS) stamped substrates (Kane <em>et al.</em>, 1999). In this application, we utilize the XY position within the array to know the condition of that island. This protocol could be extended to other systems where spatial position of cells on their substrate is linked to an experimental variable or substrates such as with gradients of stiffness (Hadden <em>et al.</em>, 2017) or biomolecules (Dertinger <em>et al.</em>, 2002). </p>
        </body>

<p class="pptt" id="biaoti30186">## Materials and Reagents</p><ol class="npwtwok"><li>Aluminum foil</li><li>Forceps</li><li>Pipettes</li><li>1 ml syringe</li><li>6-well polystyrene microplates (Fisher Scientific, catalog number: 08-772-1B) <br/><em>Note: Other 6-well microplates may be suitable, however the 35 mm glass bottom Petri dishes fit well into these plates and do not fit into all 6-well plates.</em> </li><li>Far-red fluorescent beads, 0.2% v/v (Life Technologies, catalog number: F-8816)</li><li>12 mm glass coverslip (Electron Microscopy Sciences, catalog number: 72231-01)</li><li>384-well polypropylene plate (USA Scientific, catalog number: 1823-8400)</li><li>35 mm glass bottom dish with 13 mm well and #1.5 German cover glass (Cell E&amp;G, catalog number: GBD00002-200)<br/></li><li>Ligands<br/>Fc-recombinant DLL1, mouse (R&amp;D Systems, catalog number: 5026-DL-050)<br/>Fc-recombinant DLL4, mouse (AdipoGen, catalog number: AG-40A-0145-C050)<br/>Fc-recombinant JAG1, rat (R&amp;D Systems, catalog number: 599-JG-100)<br/>Human IgG (R&amp;D Systems, catalog number: 1â001-A)<br/>Protein A/G (Life Technologies, catalog number: 21186)<br/><em>Note: Cell-cell signaling ligands can be arrayed on the surface in combination with ECM proteins. Retention of smaller molecules in the gel may differ, therefore it may be necessary to conjugate ligands to an additional molecule. We used Fc-recombinant notch ligands conjugated to protein A/G, and IgG as a control. If using this strategy, ligands should be combined with protein A/G at a molar ratio of 6:1 or greater. Ligands should be reconstituted per manufacturer recommendations and stored at -80 Â°C. </em> </li><li>Extracellular Matrix proteins<br/>Collagen I, rat tail (EMD Millipore, catalog number: 08-115MI), stored at 4 Â°C<br/>Collagen III, human (EMD Millipore, catalog number: CC054)<br/>Collagen IV, human (EMD Millipore, catalog number: CC076)<br/>Fibronectin, human (Sigma-Aldrich, catalog number: F2006)<br/>Laminin, mouse (EMD Millipore, catalog number: CC095)<br/><em>Note: Numerous ECM proteins are suitable for this process. Choice of ECM protein will depend on the application. Here we have listed those used in our in various studies, which were prepared at 1 mg/ml, and stored at -20 Â°C unless otherwise noted. </em> </li><li>3-TPM (Sigma-Aldrich, catalog number: 440159-100ML) <br/><em>Note: Exposure of 3-TPM to air could negatively impact silanization of glass substrates and should be stored per manufacturer instructions. The 3-TPM described here is packaged in a Sure/SealTM container, which requires a syringe and needle for use.</em> </li><li>NaOH (Sigma-Aldrich, catalog number: 415413-1L)</li><li>Ethanol 200 Proof (Decon Labs, catalog number: UN1170)</li><li>Acrylamide (Sigma-Aldrich, catalog number: A3553-100G)</li><li>Bis-acrylamide (Sigma-Aldrich, catalog number: M7279-25G)</li><li>Irgacure 2959 (BASF Corporation, catalog number: 55047962)</li><li>Methanol (Sigma-Aldrich, catalog number: 179957-1L)</li><li>Sodium acetate (Sigma-Aldrich, catalog number: S2889-250G)</li><li>Ethylenediaminetetraacetic (EDTA) (Sigma-Aldrich, catalog number: ED-100G)</li><li>Glycerol (Sigma-Aldrich, catalog number: M6145-25ML)</li><li>Glacial acetic acid (Sigma-Aldrich, catalog number: 695092-500ML)</li><li>Phosphate-buffered saline (PBS) (Fisher Scientific, catalog number: SH3001302)</li><li>3-[(3-Cholamidopropyl) dimethylammonio]-1-propanesulfonate (CHAPS) (Sigma-Aldrich, catalog number: C3023-1G)</li><li>Dextran-rhodamine (Life Technologies, catalog number: D-1841)</li><li>100x penicillin-streptomycin solution (Fisher Scientific, catalog number: SV30010)</li><li>Bovine serum albumin (Sigma-Aldrich, catalog number: A2153-100G)</li><li>Sodium dodecyl sulfate (Fisher Scientific, catalog number: BP166-100)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: X100-1L)</li><li>2x ECMP Printing Buffer (see Recipes)</li><li>2x Growth Factor (GF) Printing Buffer (see Recipes)</li><li>Polyacrylamide Prepolymer Solution (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30187">### Equipment</p><ol class="npwtwok"><li>Vacuum Desiccator (Fisher Scientific, catalog number: 08-642-5)</li><li>OmniGrid Micro arrayer (Digilab) <br/><em>Note: Other microarrayers with similar capabilities can be used instead. Follow manufacturerâs instructions to operate arrayer.</em> </li><li>Stealth or solid pins (ArrayIt, catalog number: SMP3 or SSP015) <br/><em>Note: Other pin types may be suitable for use ArrayIt, SMP3 prints spots approximately 150 Î¼m in diameter. SSP015 pins print spots of approximately 600 Î¼m in diameter. Select pins that produce an appropriate island size for your application.</em> </li><li>UV crosslinker, UVP, CL-1000</li><li>Vortex</li><li>Fluorescence microscope, Zeiss Axiovert 200M with a computer running Zeiss Zen image acquisition software<br/><em>Note: Here we use a Zeiss Axiovert 200M, however other fluorescent microscopes are suitable. The microscope should be equipped with a robotic stage and the ability to mark and find locations. We also advise environmental controls (i.e., 37 Â°C and 5% CO</em><sub><em>2</em></sub><em> since TFM involves live cell imaging. The provided MATLAB code is designed to directly load and read metadata from the Zeiss proprietary format (.czi). However, because it utilizes tools from Bio-Formats (Linkert </em><em>et al.</em><em>, 2010), files from most common microscopes can be loaded with little or no modifications.</em><br/></li></ol><p class="pptt" id="biaoti30188">### Software</p><ol class="npwtwok"><li>MATLAB version 2018b, Mathworks </li><li>MATLAB scripts from <a href="https://github.com/UnderhillLab/High-Throughtput-TFM" target="_blank">https://github.com/UnderhillLab/High-Throughtput-TFM</a><br/><em>Note: The provided scripts are intended to be able to be used by those with basic MATLAB experience, however adjusting the scripts for the additional needs of a given user may require more advanced MATLAB expertise.</em><br/></li></ol><p class="pptt" id="biaoti30189">### Procedure</p><ol class="npwone"><li>Substrate fabrication<br/>Polyacrylamide gels are prepared as adapted from protocols in literature (Tse and Engler, 2010; Wen <em>et al.</em>, 2014).<br/><ol class="npwtwo"><li>Silanization of glass bottom dishes. This functionalizes the glass surface so that the polyacrylamide hydrogel will covalently bind to it. <ol class="npwthree"><li>Add 250 Î¼l of a 0.2 N NaOH solution to the glass area of each glass bottom dish. Ensure NaOH solution coats entire glass area but not the plastic surface as shown in Figure 1A.</li><li>Incubate on a gently rocking shaker for 1 h.</li><li>Rinse dishes 3 times with deionized water (dH<sub>2</sub>O). </li><li>Use compressed, filtered air to dry the dishes. Bake the dishes at 80 Â°C on aluminum foil on a hot plate until completely dry. Overheating the dishes may cause adhesive on the glass to fail. </li><li>Add 0.2 ml of 3-(Trimethoxysilyl) Propyl Methacrylate (3-TPM) to 9.8 ml of 100% ethanol.</li><li>Apply 100 ul of 3-TPM solution to the glass area of each dish. Ensure 3-TPM solution coats entire glass area but not the plastic surface as shown in Figure 1A.</li><li>Incubate on a gently rocking shaker for 30 min.</li><li>Add 3 ml of ethanol to each dish. Incubate on a gently rocking shaker for 5 min. </li><li>Remove ethanol. Use compressed air to dry the dishes. </li><li>Bake the dishes at 80 Â°C on aluminum foil on a hot plate until completely dry. <br/><em>Note: At this stage, silanized glass bottom dishes can be stored for up to one month in a dark, dry location.</em> <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030203954_5646.jpg" width="149"/><br/><strong>Figure 1. Glass bottom dish silanization and gel fabrication.</strong> A. Proper coverage for NaOH and 3-TPM solutions. B. Droplet of prepolymer solution. C. Coverslip floated on prepolymer droplet. D. Inverted dish to allow beads to settle. E. Dish filled with deionized water following UV exposure. F. Dish with coverslip removed and gel dehydrated.<br/><br/></li></ol></li><li>Polyacrylamide gel fabrication. The polyacrylamide gel provides the deformable hydrogel substrate for cells to bind to as part of TFM preparation.<br/><ol class="npwthree"><li>Prepare a polyacrylamide pre-polymer solution based on desired stiffness (Recipe 3). You will need 20 Î¼l of solution for each dish. </li><li>Add 0.1 g of Irgacure 2959 (light-sensitive) to 0.5 ml 100% methanol to produce a 20% w/v Irgacure solution. Vortex until solution is clear.</li><li>Combine the pre-polymer solution and Irgacure solution at a 9:1 ratio. Prepare enough. </li><li>Sonicate the bead suspension for 15 min.</li><li>Add beads to the working solution at a 1:500 ratio (<em>i.e.</em>, 1 Î¼l bead solution to 500 Î¼l working solution). Vortex to ensure complete mixing. </li><li>Degas the full working solution in a vacuum desiccator for 15 min. </li><li>Pipette 20 Î¼l of the working solution into the center a glass bottom dish. Avoid bubbles in the droplet as shown in Figure 1B.</li><li>Carefully float a 12 mm coverslip onto the droplet. The solution should spread the edges of the droplet as shown in Figure 1C. </li><li>Repeat with the remaining dishes.</li><li>Invert the dishes, as shown in Figure 1D, and let stand for 15 min to allow beads to migrate towards the surface.</li><li>Expose dishes to 365 nM UVA for 15 min in the UV crosslinker. </li><li>Fill dishes with deionized water as shown in Figure 1E. Let sit for 3-24 h. </li><li>Carefully remove coverslips using forceps. Avoid twisting or shearing the coverslip which may cause ripping in the gel surface.<br/><em>Note: Use of clean coverslips and careful removal after 24 h of soaking in dH</em><sub><em>2</em></sub><em>O typically results in successful removal of the coverslip without damaging the gel. For very soft gels, it may be necessary to further passivate the coverslip surface. Strategies include wiping the surface with RainâXÂ® using a delicate task wiper, or vapor deposition of Dichlorodimethylsilane.</em> </li><li>Dehydrate gels at 50 Â°C on a hot plate until all water has evaporated from the gel. A dehydrated gel is shown in Figure 1A. Gel dishes can be stored for up to one month in a dark, dry location. </li></ol><br/></li></ol></li><li>Microarray printing <br/>This process prints an array of circular spots of biomolecules onto the polyacrylamide substrates. These array spots are where cells will adhere forming circular islands.<ol class="npwtwo"><li>If needed, prepare a suitable 2x printing buffer (Recipe 1) <br/><em>Note: ECM protein printing (ECMP) buffer is appropriate for most ECM molecules. Growth factor (GF) printing buffer is suitable for other classes of molecules such as growth factors or ligands, where a low pH could cause issues.</em> </li><li>Prepare a microarraying source plate.<br/><ol class="npwthree"><li>In a 384-well V-bottom microplate, prepare printing solutions which should include equal volumes of 2x printing buffer with biomolecule solution at double the target concentration. For many ECM proteins, 250 Âµg/m is a suitable concentration. Optimal concentrations will vary depending on the molecule, its retention, and its function. The total volume in each well should be between 5 and 15 Âµl. </li><li>In your source plate, you should also include a solution with a fluorescent marker which will be used to convey the orientation of the array to determine the locations of each condition. We recommend rhodamine-conjugated dextran at a final concentration of 2.5 mg/ml. The source plate configuration will differ based on the arrayer, pin configuration, and desired array layout.</li><li>Mix each well thoroughly by pipetting. Take care to avoid generating bubbles. Centrifuge the source microplate for 1 min at 1,000 <em>x g</em>. Source plates can be used immediately or stored at 4 Â°C for 1 day before until microarray fabrication. If storing, cover source plate with an adhesive seal. </li><li>Clean and prepare pins according to the manufacturer's instructions. Load cleaned pins into the microarrayer printhead.</li><li>Prepare the microarrayer and arraying program using the manufacturerâs software. The set-up and programming will differ based on the arrayer and desired array layout. The program should be devised such that the array orientation is unambiguous, and the locations of each arrayed conditions are known and could be determined by the location relative to the fluorescent marker in any orientation. An example of this is provided in Figure 2.<br/><em>Notes:</em><br/><ol class="npwfour"><li><em>The example array layout shown was designed for printing 6 replicates of 4 conditions, which were simultaneously printed by four SSP015 pins which produce 600 Î¼m diameter islands. The number of islands and/or conditions can be scaled up when using pins that produce smaller islands.</em> </li><li><em>Be sure to record how the layout corresponds to the arrayed conditions. Include wash steps using both water and dimethyl sulfoxide (DMSO) between each condition in order to prevent carry-over and cross-contamination.</em> </li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204041_3704.jpg" width="161"/><br/><strong>Figure 2. Example layout of an array of 4 ECM conditions with dextran rhodamine markers.</strong> When the islands and dextran rhodamine spots are visible, the orientation of the array can be discerned whether rotated or if viewed from above or below.<br/></li></ol></li><br/><ol class="npwthree" start="6"><li>Turn on the humidifier and ensure it is adjusted to 65% RH (non-condensing). Wait until the rheometer matches the set point. </li><li>Place the source plate in the arrayer in an appropriate adaptor.</li><li>Dehydrate hydrogel substrates at 50 Â°C for 15 min. Protect from light due to the presence of the fluorescent beads. </li><li>Place dishes into an appropriate adaptor. If the arrayer can fit a standard multiwell plate, the 6-well plate is suitable to hold the dishes.<br/><em>Note: See the note under Materials and Reagents #5. </em> </li><li>Begin array fabrication. Check frequently that the humidity has not dropped below 65% RH (non-condensing). </li><li>When the program is complete, store fabricated arrays covered with aluminum foil at room temperature and 65% RH (non-condensing) overnight. While the array spots are visible, it is helpful to visibly mark the top or bottom of the array so the orientation is known when placing on the microscope. For some hydrogel and pin combinations, it may be necessary to store arrays at ambient temperature and humidity for an additional two days to ensure arrays have dried completely. Arrays can be stored for up to 7 days before use. <br/><br/></li></ol></ol></li><li>Seeding cells on microarrays<br/>Here cells are transferred from their normal culture condition on to the microarrayed hydrogel substrates for TFM.<ol class="npwtwo"><li>To sterilize the gels, add 3 ml of PBS with 1% v/v penicillin/streptomycin. Expose to UV C for 30 min. Exchange penicillin/streptomycin solution for cell culture media. </li><li>Collect and count cells following the cell appropriate procedure. Resuspend cells in culture media at an appropriate concentration for seeding. This will differ based on cell type but will likely range between 170 x 10<sup>3</sup> and 7 x 10<sup>5</sup> cells/ml. Add 3 ml of cell suspension to each dish. Incubate dishes at 37 Â°C and 5% CO<sub>2</sub> for 2-24 h, or until confluent cell islands have formed. Seeding density and time may need to be optimized for your cells and application. Agitation of the dishes every 15-60 min may also aid in forming consistent, confluent islands.</li><li>Once islands have formed, rinse arrays twice with 3 ml prewarmed media. At this stage, add any experimental treatments such as growth factors or inhibitors. Change media every 1-2 days until time to perform TFM, or as your cell culture protocol suggests, maintaining any treatment concentrations at each exchange. Figure 3 shows an example of an array with cellular islands.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204121_5683.jpg" width="134"/><br/><strong>Figure 3. 35 mm glass bottom dish with cellular island array after 24 h of culture</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Acquiring images for traction force microscopy<ol class="npwtwo"><li>Prepare a solution of 1% v/v bovine serum albumin (BSA) and 1% v/v sodium dodecyl sulfate (SDS) in PBS. This will be used to dissociate cells during image acquisition. </li><li>The microscopy process will vary based on the microscope used, but the following steps should broadly apply. Some additional details are provided for use with Zeiss Zen software. The microscope should have a robotic stage and a chamber which provides incubation and humidity control which have been appropriately set and allowed to come to equilibrium. Place one dish on the stage of the inverted fluorescent microscope. It is helpful to use the mark provided after array fabrication to place the dish with the arrayâs XY axis approximately aligned along that of the microscope stage. Secure the dish in place so that it does not shift when the stage moves.</li><li>Using the appropriate fluorescent channel, locate and mark (in the software) the XY positions of the fluorescent arrayed markers. In Zeiss Zen Blue 2, this is accomplished in the acquisition section by checking the tiles box and adding a single position for each island, as shown in Figure 4.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204143_2920.jpg" width="123"/><br/><strong>Figure 4. Zeiss Zen Blue 2 environment.</strong> To mark locations, in acquisition, mark Tiles and enter advanced setup. Set individual island positions as shown by the yellow markers. The verify positions tool can be used to move through positions to set the Z position without altering the XY position. <br/><br/></li><li>Using phase contrast or bright field, locate and mark the XY positions of the cell islands. Either imaging modality may be suitable as long as the image can provide the boundary of the cell island in subsequent analysis. Individually find and save the Z plane focus of each island. An example image phase contrast is shown in Figure 5A. The verify positions tool can be used to move through positions to set the Z position without altering the XY position. The saved XY positions of the islands can be saved and reused for subsequent dishes.</li><li>Begin the automated imaging of the phase contrast images of the cell islands. Save this file with a suitable name that notes the experiment details as well as that it is the phase contrast image.</li><li>Switch to the appropriate fluorescent channel for the beads. Individually find and save the Z plane focus of the top surface of the gel under each island. Take care to avoid changing the XY positions. Save this file with a suitable name that notes the experiment details as well as that it is the pre-dissociation image. An example image is shown in Figure 5B.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204207_5622.jpg" width="247"/><br/><strong>Figure 5. Example images from TFM data collection. </strong>A. Phase contract image of a cellular island. Image should be clear enough to define island boundaries. B. Fluorescence image of the beads at the surface of the substrate under the island. Scale bar: 200 Î¼m.<br/><br/></li><li>Begin the automated imaging in the fluorescent channel of the pre-dissociation bead images. Save this file with the same experiment details as well as that it is the pre-dissociation image.</li><li>Carefully add 150 Âµl of the SDS solution to the dish, taking care to not bump or move the dish. Monitor dissociation of the cell islands using the phase contrast channel. Wait until the islands have completely dissociated from the substrate at which time there the island locations, when viewed in phase contrast channel, should appear mostly blank. <br/><em>Note: Some cells may require addition of more SDS solution, or higher concentration. Additionally, Triton-X may be used instead. </em> </li><li>Return to the island positions and using the fluorescent channel check that the beads at the top of the surface are still in focus. If the beads have moved out-of-plane due to relaxion of the cell induced deformation, correct and save the Z-coordinate of the focus plane but do not alter the XY positions. Repeat automated imaging of the islands to capture the post-dissociation images of the beads. The displacement of the beads will likely be difficult to discern, meaning the post-dissociation fluorescent bead image should appear very similar to the pre-dissociation image. Save this file with a suitable name that notes the experiment details as well as that it is the post-dissociation image.</li><li>Repeat Steps D2-D9 for each dish. </li></ol></li></ol><p class="pptt" id="biaoti30190">### Data analysis</p><ol class="npwone"><li>Process images to estimate displacement fields and calculate traction fields<br/><em>Note: Images collected per the previous section may be exported and used with available TFM code packages. Here we provide the TFM code we have implemented, and its use is explained in the following sections. The TFM and analysis are designed to be used with arrays of cells with the same geometry to enable assessing replicates of the same condition. These scripts allow the user to generate TFM data and create basic visualizations. Further manipulation and analysis of the data for a userâs specific application may require generation of additional scripts. </em><br/><br/><ol class="npwtwo"><li>Ensure the provided MATLAB code has been saved to an appropriate location. Navigate to the folder where this directory has been saved. </li><li>Make sure all image files have been saved or transferred to a folder available from the computer to be used for analysis. </li><li>From the command window, run the function <em>run_island_tfm</em> with no inputs.</li><li>You will be prompted to select the file with the phase contrast image. Navigate to and select the file with the phase contrast image.</li><li>A new file selection window will open in the same directory as the selected file. Now select the file with the pre-dissociation fluorescent image. Finally, when prompted, select the post-dissociation image.</li><li>A GUI will appear for entering the experimental information for the image files. This GUI is shown in Figure 6. The graph panel shows the XY position of each image collected. The numbers correspond to how the images are indexed in the image file. Enter all relevant experiment information listed above the âData file nameâ field. The pixel size will be read from the metadata of the image but can be changed. The ânumber of arrayed conditionsâ refers to the number of different conditions in the imaged array. For the example layout in this protocol, there are 4 arrayed conditions. This number is used to group the islands into their conditions.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204237_2648.jpg" width="150"/><br/><strong>Figure 6. GUI for entering experiment and dish information.</strong> Chart panel shows XY position of each island, labeled by the index of the island in the file. Chart title is the image file name. Pixel size is pulled from the image metadata but can be set manually. All other fields are set by the user.<br/><br/></li><li>Once these fields are completed, click âSet Info.â This will populate the âData file nameâ field using the entered information which will be the file name of the output file. This field can be changed manually. Two additional fields will appear. Enter a name for the first condition in the âCondition 1â field. Enter the numbers of all the islands assigned this condition in the âIslands with Condition 1â field as a list of numbers. Click âSet Condition.â If there are additional conditions, repeat these steps for each of the remaining conditions. Once all conditions have been set, the âDoneâ button will appear. Confirm that all experiment information is correct, then click âDone.â All islands assigned a condition in this step will be analyzed. To analyze only some islands, set the number of conditions to the number conditions represented by the islands to be analyzed, and only list those islands in the âIslands with Conditionâ field.</li><li>The script will now cycle through the conditions and islands. It will first run a script to correct frame shifts between the before and after dissociation. <br/><em>Note: The correct shift function is to remove full frame shifts between the images caused stage drift between collecting the images. This is done by performing digital image correlation on regions of the image far enough from the cells where there is no expected deformation. The script automatically selects four regions near the four corners of the image. To manually select these regions, change the ânâ to a âyâ in the frame_shift function call.</em><br/>A GUI will now appear to aid in drawing a boundary around the cell island. This GUI is shown in Figure 7. The software will attempt to draw a boundary around the cell island, which is plotted in red. The slides can be used to adjust the parameters of the trace which will cause the trace to rerun in real time. You can also change the values of these parameters by entering numbers into the text boxes and then hit the âRerunâ button. Alternatively, you can draw the boundary manually by clicking the âdraw manualâ button. Left click around the island until the boundary is closed. Double click inside the boundary to create the shape. To avoid sharp corners, after the polygon is drawn, a blurring and rounding is applied using the current value in the âBlurâ field. To avoid this rounding, set this field to 1. The manual draw function can be used to trace multiple areas. To reset the manual boundaries, click âClear boundaries,â or click âRerunâ to repeat the automated tracing. When satisfied with the boundary, click âDone.â<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204258_4568.jpg" width="146"/><br/><strong>Figure 7. GUI for selecting cell island boundary.</strong> Parameters can be set with the sliders which will update the boundary in real time, or by typing in values and clicking Rerun. You can also manually draw the boundary. <br/><br/></li><li>This will bring up the next island. Repeat this process for all islands in the file selected for analysis. </li><li>When the boundary trace has been completed for each island, the data will be saved. The program will then move to calculate the displacement and traction fields for each island. This process can take up to several hours depending on the number of images, image size, and processors on the computer. Data is saved after each island to limit data loss in the case of issues during analysis. Data is saved in the folder âdata out.â </li><li>Most of the computation time is the displacement field estimation. To rerun the traction field calculation with different settings on a file which already has the boundaries identified and displacement field calculated, run the command <em>run_island_tfm</em>(â<em>rerun</em>â). You will be prompted to choose a data file which will be reanalyzed. </li></ol><br/></li><li>View data and generate summaries<ol class="npwtwo"><li>In the code provided, the output is saved in the folder âdata outâ with the file name established during analysis. To explore data from a single file, data can be loaded into the workspace by double clicking in the Current Folder explored or using the load function.</li><li>This loads the cell array âall_cell_dataâ into the workspace. Each cell corresponds to a single island from the analysis. The output is intended to provide easy access to all relevant data for the user to explore and analyze in MATLAB or export to other programs for analysis as appropriate for their application. Table 1 provides the organization of the data stored within the output file. Data structure elements can be accessed using dot notation of the form structName.fieldName.<br/><em>Note: See </em><a href="https://www.mathworks.com/help/matlab/matlab_prog/access-data-in-a-structure-array.html" target="_blank"><em>https://www.mathworks.com/help/matlab/matlab_prog/access-data-in-a-structure-array.html</em></a><em> for more information on accessing data in structures.</em> </li><li>A set of functions are provided to view and analyze data. Example code to use these functions is provided in <em>data_analysis_examples</em>. To view the results from a single island, use <em>v</em><em>iew_one_island</em>. See code for further documentation. An example of the output is shown in Figure 8.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030204351_2938.jpg" width="131"/><br/><strong>Figure 8. Example output from <em>view_one_island</em>.</strong> Island information is displayed. Vector fields are plotted over the magnitudes of the displacement filed (in Î¼m) and traction field (in Pa). The island boundary is plotted in black. The phase contrast image is also displayed, with the boundary plotted in yellow. The best fit ellipse is shown in red, with the major and minor axes.<br/><br/></li><li>Using the boundary traced of the island, the script finds a best fit ellipse. This ellipse can be used to align compare many replicates of islands with the same geometry. <br/><em>Note: The provided scripts assume the islands can be represented by an ellipse. This can be adapted to other geometries by altering how the center and angle of rotation are identified. </em><br/>Use view_one_island_rotandcen to view the data centered and rotated according to the best fit ellipse.<em></em><span></span> </li><li>The function collect_island_data is provided to collect data from multiple islands across multiple TFM runs. The output of this function is a table with data and information on each of the islands loaded from the files (see Table 1), and a structure holding the displacement and traction data, indexed according to the âsumm_indâ field of the summary table This table can be exported to excel or similar. When the function is run without an input, you will be prompted to select data files to load and consolidate. <br/><br/><strong>Table 1. Structure of data output file and description of saved data</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:1;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><span style="font-size:12px;"><strong>Data Field</strong></span><br/></td><td><br/></td><td><br/></td><td><span style="font-size:12px;"><strong>Description</strong></span><br/></td></tr><tr><td><span style="font-size:12px;">cell_info.</span><br/></td><td><span style="font-size:12px;">young</span><br/></td><td><br/></td><td><span style="font-size:12px;">Youngs Modulus of substrate</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">pois</span><br/></td><td><br/></td><td><span style="font-size:12px;">Poisson Ratio of substrate</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">pixelsize</span><br/></td><td><br/></td><td><span style="font-size:12px;">Size of original image pixels in micron</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">experiment</span><br/></td><td><br/></td><td><span style="font-size:12px;">Description of experiment</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">date_ID</span><br/></td><td><br/></td><td><span style="font-size:12px;">Identifier of experiment or run</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">soluble</span><br/></td><td><br/></td><td><span style="font-size:12px;">Soluble factors or treatments added to dish</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">dish_num</span><br/></td><td><br/></td><td><span style="font-size:12px;">Identifier of dish, <em>e.g.</em>, if replicates of a unique condition</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">time</span><br/></td><td><br/></td><td><span style="font-size:12px;">Time from seeding and TFM data collection</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">arrayed_condition</span><br/></td><td><br/></td><td><span style="font-size:12px;">ECM or other condition of the island within the array</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">Xposition</span><br/></td><td><br/></td><td><span style="font-size:12px;">X position of island within array</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">Yposition</span><br/></td><td><br/></td><td><span style="font-size:12px;">Y position of island within array</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">image_number</span><br/></td><td><br/></td><td><span style="font-size:12px;">Index of image within the original file</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">file</span><br/></td><td><br/></td><td><span style="font-size:12px;">file containing original phase/brightfield image</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">dm</span><br/></td><td><br/></td><td><span style="font-size:12px;">Final spacing, in pixels, of displacement/traction field output</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><br/></td><td><br/></td></tr><tr><td><span style="font-size:12px;">Images.</span><br/></td><td><span style="font-size:12px;">Images{1}</span><br/></td><td><br/></td><td><span style="font-size:12px;">Pre-dissociation fluorescent image</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">Images{2}</span><br/></td><td><br/></td><td><span style="font-size:12px;">Post-dissociation fluorescent image</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">Images{3}</span><br/></td><td><br/></td><td><span style="font-size:12px;">Phase/Brightfield image</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><br/></td><td><br/></td></tr><tr><td><span style="font-size:12px;">cell_boundaries.</span><br/></td><td><span style="font-size:12px;">mask</span><br/></td><td><br/></td><td><span style="font-size:12px;">mask of regions determined within island boundary</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">boundary_points</span> </td><td><br/></td><td><span style="font-size:12px;">Cell containing the XY coordinates of each of the boundaries traced</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">cny</span><br/></td><td><br/></td><td><span style="font-size:12px;">Sensitivity factor used to trace boundary</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">sigma</span><br/></td><td><br/></td><td><span style="font-size:12px;">Blur factor used to trace boundary</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">di</span><br/></td><td><br/></td><td><span style="font-size:12px;">Erosion factor used to trace boundary</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">ellipse_fit.</span><br/></td><td><span style="font-size:12px;">phi            </span><br/></td><td><span style="font-size:12px;">Angle of roation of the fit ellipse with respect to the major axis being horizontal</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">X_center</span><br/></td><td><span style="font-size:12px;">center at the X axis of the non-rotated ellipse</span><br/></td></tr><tr><td><br/></td><td></td><td><span style="font-size:12px;">Y_center</span> </td><td><span style="font-size:12px;">center at the Y axis of the non-rotated ellipse</span> </td></tr><tr><td><span style="font-size:12px;">      </span> </td><td></td><td><span style="font-size:12px;">X0</span><br/></td><td><span style="font-size:12px;">center at the X axis of the rotated ellipse</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">Y0</span><br/></td><td><span style="font-size:12px;">center at the Y axis of the rotated ellipse</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">major_axis</span><br/></td><td><span style="font-size:12px;">major axis of ellipse (in pixels)</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">minor_axis  </span><br/></td><td><span style="font-size:12px;">minor axis of ellipse (in pixels)</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">rotated_boundary_points         </span><br/></td><td><br/></td><td><span style="font-size:12px;">Cell containing the XY coordinates of each of the boundaries traced, rotated and centered according to the fit ellipse</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><br/></td><td><br/></td></tr><tr><td><span style="font-size:12px;">cell_displacements.        </span><br/></td><td><span style="font-size:12px;">raw_displacements.</span> </td><td><span style="font-size:12px;">raw_displacements{1}      </span> </td><td><span style="font-size:12px;">X component of raw estimated displacement field in pixels</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">raw_displacements{2}         </span><br/></td><td><span style="font-size:12px;">Y component of raw estimated displacement field in pixels</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">raw_displacements{3}    </span><br/></td><td><span style="font-size:12px;">Magnitude of raw estimated displacements in pixels</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">Displacements.</span><br/></td><td><span style="font-size:12px;">displacements{1}</span><br/></td><td><span style="font-size:12px;">X component of displacement field in micron</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">displacements{2}</span><br/></td><td><span style="font-size:12px;">Y component of displacement field in micron</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">displacements{3}       </span><br/></td><td><span style="font-size:12px;">Magnitudes of displacements field in micron</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">rotated_displacements</span><br/></td><td><br/></td><td><span style="font-size:12px;">displacements which have been rotated and centered using the fit ellipse</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><br/></td><td><br/></td></tr><tr><td><span style="font-size:12px;">cell_tractions.</span><br/></td><td><span style="font-size:12px;">Tractions.</span><br/></td><td><span style="font-size:12px;">tractions{1}</span><br/></td><td><span style="font-size:12px;">X component of tractions in Pa</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">tractions{2}</span><br/></td><td><span style="font-size:12px;">Y component of tractions in Pa</span><br/></td></tr><tr><td><br/></td><td><br/></td><td><span style="font-size:12px;">tractions{3}</span><br/></td><td><span style="font-size:12px;">Magnitudes of tractions in Pa</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">RMS</span><br/></td><td><br/></td><td><span style="font-size:12px;">Root mean square of traction field</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">RMS_int</span><br/></td><td><br/></td><td><span style="font-size:12px;">Root mean square of traction field within cell island boundary</span><br/></td></tr><tr><td><br/></td><td><span style="font-size:12px;">Strain_Energy</span><br/></td><td><br/></td><td><span style="font-size:12px;">Strain Energy</span><br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td><br/></td><td><span style="font-size:12px;">rotated_tractions</span><br/></td><td><br/></td><td><span style="font-size:12px;">Tractions which have been rotated and centered using the fit ellipse</span><br/></td></tr></tbody></table><br/></li><li>The output of the <em>collect_island_data</em> function can further be used to create summary data. The function <em>summarize_islands</em> plots and outputs averaged displacement and traction fields, see Figure 9 for an example. This function uses the aligned the data from each island based on the best fit ellipse. The islands to include in the averaging can be selected by choosing a subset of set of islands of the summary_table by indexing on one or more variables. See <em>data_analysis_examples</em> for an example of using this function. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030234333_4335.jpg" width="236"/><br/><strong>Figure 9. Example of average displacement and traction vector fields produced by <em>summarize_islands</em></strong><br/><br/></li><li>The circular islands described in this protocol and used in our work (Kaylan <em>et al.</em>, 2018) have radial symmetry. Therefore, it was informative to analyze the traction a function of a single radial dimension as shown in Figure 10. The function <em>summarize_islands_1D</em> is provided to perform this analysis. Here, the XY position of each data point is converted to a radial coordinate, and the radial position is normalized by the measured radius of the island. The traction data is binned by radial coordinate, and a mean is taken. The data used for this analysis can also be selected using the summary table as discussed previously. The function also outputs the data table with the peak traction of each island amended to the relevant lines. See <em>data_analysis_examples</em> for an example of using this function. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191030/20191030234350_7275.jpg" width="320"/><br/><strong>Figure 10. Tractions from 6 replicates of 4 conditions in one TFM dish plotted as a function of normalized radial coordinate.</strong> Dashed line shows standard deviation. In all conditions, traction peaks near the island periphery, where R = 1.<br/></li></ol></li></ol><p class="pptt" id="biaoti30191">### Recipes</p><ol class="npwtwok"><li>2x ECMP Printing Buffer<ol class="npwthree"><li>Add 164 mg sodium acetate and 37.2 mg EDTA to 5.9 ml dH<sub>2</sub>O </li><li>Once dissolved, add 50 Âµl warmed Triton X-100 and 4 ml glycerol. Vortex until combined</li><li>Add approximately 80 Âµl of glacial acetic acid, titrating to adjust the pH to 4.8</li><li>Store at 4 Â°C until arraying</li></ol></li><li>2x Growth Factor (GF) Printing Buffer<ol class="npwthree"><li>Add 105.5 mg sodium acetate and 37.2 mg EDTA to 6 ml PBS</li><li>Once dissolved, add 0.1 g CHAPS and 3.8 ml glycerol</li><li>Store at 4 Â°C until arraying</li></ol></li><li>Polyacrylamide Prepolymer Solution<ol class="npwthree"><li>Polyacrylamide solutions are prepared according to the desired stiffness. An example set of stiffness recipes is adapted from (Wen <em>et al.</em>, 2014)<br/>4 kPa Youngâs modulus: 0.400 g acrylamide and 0.040 g bis-acrylamide<br/>13 kPa Youngâs modulus: 0.600 g acrylamide and 0.045 g bis-acrylamide<br/>30 kPa Youngâs modulus: 0.800 g acrylamide and 0.055 g bis-acrylamide</li><li>Add appropriate amount of acrylamide and bis-acrylamide to 10 ml dH<sub>2</sub>O</li><li>Vortex until the solution is clear. Filter solution with 0.2 Âµm syringe </li><li>Store solution at 4 Â°C for up to one month</li></ol></li></ol><p class="pptt" id="biaoti30192">### Acknowledgments</p><p class="ppzz">We acknowledge the Carl R. Woese Institute for Genomic Biology, University of Illinois at UrbanaâChampaign Core Facility for assistance with microscopy. Research reported in this publication was supported by the National Institute of Biomedical Imaging and Bioengineering of the National Institutes of Health under Award Number T32EB019944. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health.</p><p class="pptt" id="biaoti30193">### Competing interests</p><p class="ppzz">The authors declare that they have no competing financial interests.</p><p class="pptt" id="biaoti30194">### References</p><ol class="npwtwok"><li>Butler, J. P., Tolic-Norrelykke, I. M., Fabry, B. and Fredberg, J. J. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11832345" target="_blank">Traction fields, moments, and strain energy that cells exert on their surroundings.</a> <em>Am J Physiol Cell Physiol </em>282(3): C595-C605. </li><li>Dertinger, S. K., Jiang, X., Li, Z., Murthy, V. N. and Whitesides, G. M. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12237407" target="_blank">Gradients of substrate-bound laminin orient axonal specification of neurons.</a> <em>Proc Natl Acad Sci U S A</em> 99(20): 12542-12547. </li><li>Discher, D. E., Mooney, D. J. and Zandstra, P. W. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19556500" target="_blank">Growth factors, matrices, and forces combine and control stem cells.</a> <em>Science</em> 324(5935): 1673-1677. </li><li>Flaim, C. J., Chien, S. and Bhatia, S. N. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15782209" target="_blank">An extracellular matrix microarray for probing cellular differentiation.</a> <em>Nat Methods</em> 2(2): 119-125. </li><li>Hadden, W. J., Young, J. L., Holle, A. W., McFetridge, M. L., Kim, D. Y., Wijesinghe, P., Taylor-Weiner, H., Wen, J. H., Lee, A. R., Bieback, K., Vo, B. N., Sampson, D. D., Kennedy, B. F., Spatz, J. P., Engler, A. J. and Choi, Y. S. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28507138" target="_blank">Stem cell migration and mechanotransduction on linear stiffness gradient hydrogels.</a> <em>Proc Natl Acad Sci U S A</em> 114(22): 5647-5652. </li><li>Han, S. J., Oak, Y., Groisman, A. and Danuser, G. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26030446" target="_blank">Traction microscopy to identify force modulation in subresolution adhesions.</a> <em>Nat Methods</em> 12(7): 653-656. </li><li>Kane, R. S., Takayama, S., Ostuni, E., Ingber, D. E. and Whitesides, G. M. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10614942" target="_blank">Patterning proteins and cells using soft lithography.</a> <em>Biomaterials</em> 20(23-24): 2363-2376. </li><li>Kaylan, K. B., Berg, I. C., Biehl, M. J., Brougham-Cook, A., Jain, I., Jamil, S. M., Sargeant, L. H., Cornell, N. J., Raetzman, L. T. and Underhill, G. H. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30589410" target="_blank">Spatial patterning of liver progenitor cell differentiation mediated by cellular contractility and Notch signaling.</a> <em>Elife</em> 7: e38536.</li><li>Kaylan, K. B., Kourouklis, A. P. and Underhill, G. H. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28287589" target="_blank">A high-throughput cell microarray platform for correlative analysis of cell differentiation and traction forces.</a> <em>J Vis Exp</em> (121). </li><li>Kourouklis, A. P., Kaylan, K. B. and Underhill, G. H. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27235994" target="_blank">Substrate stiffness and matrix composition coordinately control the differentiation of liver progenitor cells.</a> <em>Biomaterials</em> 99: 82-94.</li><li>Kulkarni, A. H., Ghosh, P., Seetharaman, A., Kondaiah, P. and Gundiah, N. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29740649" target="_blank">Traction cytometry: regularization in the Fourier approach and comparisons with finite element method.</a> <em>Soft Matter</em> 14(23): 4687-4695. </li><li>Landauer, A. K., Patel, M., Henann, D. L. and Franck, C. (2018). <a href="https://link.springer.com/article/10.1007/s11340-018-0377-4" target="_blank">A q-Factor-Based digital image correlation algorithm (qDIC) for resolving finite deformations with degenerate speckle patterns.</a> <em>Exp Mech</em> 58(5): 815-830. </li><li>Linkert, M., Rueden, C. T., Allan, C., Burel, J. M., Moore, W., Patterson, A., Loranger, B., Moore, J., Neves, C., Macdonald, D., Tarkowska, A., Sticco, C., Hill, E., Rossner, M., Eliceiri, K. W. and Swedlow, J. R. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20513764" target="_blank">Metadata matters: access to image data in the real world.</a> <em>J Cell Biol</em> 189(5): 777-782. </li><li>Mertz, A. F., Che, Y., Banerjee, S., Goldstein, J. M., Rosowski, K. A., Revilla, S. F., Niessen, C. M., Marchetti, M. C., Dufresne, E. R. and Horsley, V. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23277553" target="_blank">Cadherin-based intercellular adhesions organize epithelial cell-matrix traction forces.</a> <em>Proc Natl Acad Sci U S A</em> 110(3): 842-847.</li><li>Polacheck, W. J. and Chen, C. S. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27123817" target="_blank">Measuring cell-generated forces: a guide to the available tools.</a> <em>Nature Methods</em> 13(5): 415-423. </li><li>Romanov, V., Davidoff, S. N., Miles, A. R., Grainger, D. W., Gale, B. K. and Brooks, B. D. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24479125" target="_blank">A critical comparison of protein microarray fabrication technologies.</a> <em>Analyst</em> 139(6): 1303-1326. </li><li>Sabass, B., Gardel, M. L., Waterman, C. M. and Schwarz, U. S. (2008). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2134850/" target="_blank">High resolution traction force microscopy based on experimental and computational advances.</a> <em>Biophys J</em> 94(1): 207-220.</li><li>Schwarz, U. S. and Soine, J. R. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26026889" target="_blank">Traction force microscopy on soft elastic substrates: A guide to recent computational advances.</a> <em>Biochim Biophys Acta</em> 1853(11 Pt B): 3095-3104. </li><li>Tse, J. R. and Engler, A. J. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20521229" target="_blank">Preparation of hydrogel substrates with tunable mechanical properties.</a> <em>Curr Protoc Cell Biol Chapter</em> 10: Unit 10 16. </li><li>Wen, J. H., Vincent, L. G., Fuhrmann, A., Choi, Y. S., Hribar, K. C., Taylor-Weiner, H., Chen, S. and Engler, A. J. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25108614" target="_blank">Interplay of matrix stiffness and protein tethering in stem cell differentiation.</a> <em>Nat Mater</em> 13(10): 979-987. <br/></li></ol>