<body>
            <h1>Purification and HPLC Analysis of Cell Wall Muropeptides from <em>Caulobacter crescentus</em></h1>
            <p><strong>Authors:</strong> Gabriele Stankeviciute, Eric A. Klein</p>
            <h2>Abstract</h2>
            <p>The peptidoglycan sacculus, or cell wall, is what defines bacterial cell shape. Cell wall composition can be best characterized at the molecular level by digesting the peptidoglycan murein polymer into its muropeptide subunits and quantifying the abundance of muropeptides using high-pressure liquid chromatography. Certain features of the cell wall including muropeptide composition, glycan strand length, degree of crosslinking, type of crosslinking and other peptidoglycan modifications can be quantified using this approach. Well-established protocols provide us with highly-resolved and quantitatively reproducible chromatographic data, which can be used to investigate bacterial cell wall composition under a variety of environmental or genetic perturbations. The method described here enables the purification of muropeptide samples, their quantification by HPLC, and fraction collection for peak identification by mass spectrometry. Although the methods for peptidoglycan purification and HPLC analysis have been previously published, our method includes important details on how to re-equilibrate the column between runs to allow for automated analysis of multiple samples.</p>
            <p><strong>Keywords:</strong> LD-crosslinks, Transpeptidation, Muropeptides, High-pressure liquid chromatography, Gram-negative bacteria, Bacterial cell wall, Consecutive LC samples</p>
            <h2>Background</h2><p>Most bacteria are encased in a rigid mesh-like protective layer called peptidoglycan (PG) that is synthesized during cell growth and division. PG is a continuous polymer network made of glycan strands consisting of alternating <em>N</em>-acetylglucosamine (Glc<em>N</em>Ac) and <em>N</em>-acetylmuramic acid (Mur<em>N</em>Ac) sugars linked by Î² (1â4) bonds. Peptide stems attached to the Mur<em>N</em>Ac sugars are the site of PG crosslinking. While the glycan sugars are largely-conserved in Gram-negative and Gram-positive species, the sugar backbone can be modified via N-deacetylation, N-glycolylation and O-acetylation. These modifications provide resistance to host defense mechanisms against infections, namely lysozyme resistance (Vollmer, 2008). By contrast, the compositions of the peptide strands vary significantly. During pentapeptide synthesis the bacterial family of Mur-ligases can introduce a variety of amino acids at each position of the pentapeptide stem. For example, at the third position of the pentapeptide most Gram-negative species incorporate <em>meso</em>-diaminopimelic acid (mDAP), while most Gram-positives insert L-lysine, although other variations have also been reported (Vollmer, 2008). Additionally, the peptide stem can be modified after synthesis. For instance, Corynebacteriales are reported to modify the third residue <em>meso</em>-Diaminopimelic acid via amidation once peptide synthesis has occurred (Levefaudes <em>et al.</em>, 2015). Another aspect of peptidoglycan variation is the type of crosslinking used to connect multiple glycan strands (Schleifer and Kandler, 1972). Transpeptidation by DD-transpeptidases creates a 3-4 cross-linkage between third-position mDAP and fourth-position D-alanine. LD-transpeptidases, by contrast, form 3-3 crosslinks between two mDAP residues. <br/><br/>The sacculus, which provides the cell its architectural framework, also plays a large number of physiological roles in activating the immune system, aiding in colonization, evading antibiotics and host-responses, as well as inter-species recognition (Clarke and Weiser, 2011; Royet <em>et al.</em>, 2011; Mesnage <em>et al.</em>, 2014). Using high-resolution imaging (cryo-electron tomography and atomic force microscopy) and computational modeling, studies have begun to explain how cell wall composition effects the biophysical properties of cells (Tocheva <em>et al.</em>, 2013; de Pedro and Cava, 2015). Genetic and cell biology approaches have demonstrated the role of LD-transpeptidation in mediating lysozyme resistance and bacterial virulence (Schleifer and Kandler, 1972; Schoonmaker <em>et al.</em>, 2014; Stankeviciute <em>et al.</em>, 2019). Thus, HPLC is a powerful tool to connect PG composition and structure to its physiological impact.<br/></p>
        </body>

<p class="pptt" id="biaoti30229">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml polypropylene conical tubes (Nunc, catalog number: 339653)</li><li>Micro magnetic stir bar (Science ware, catalog number: F37121-0012)</li><li>2.0 ml Safe-lock microcentrifuge tubes (Eppendorf, catalog number: 022363352)</li><li>1.7 ml microcentrifuge tubes (VWR, catalog number: 87003-294)</li><li>MColorpHast pH test strips (pH 0-6) (Millipore Sigma, catalog number: 1095310001)</li><li>1 ml syringe (Becton Dickinson, catalog number: 309597)</li><li>Millex-GV syringe filter unit, 0.22 Âµm (Millipore Sigma, catalog number: SLGV033R)</li><li>12 x 32 mm screw neck HPLC vials (Waters, catalog number: 186000273)</li><li>Bottle top vacuum filter (VWR, catalog number: 10040-468)</li><li><em>Caulobacter crescentus</em> strain NA1,000 (Laboratory strain, available upon request)</li><li>Bactopeptone (BD Biosciences, catalog number: 211677)</li><li>Yeast extract (BD Biosciences, catalog number: 212750)</li><li>Magnesium sulfate heptahydrate (MgSO<sub>4</sub>Â·7H<sub>2</sub>O) (Fisher, catalog number: BP213)</li><li>Calcium chloride dihydrate (CaCl<sub>2</sub>Â·2H<sub>2</sub>O) (Fisher, catalog number: BP510)</li><li>Ethylenediaminetetraacetic acid (EDTA) (Fisher, catalog number: BP118)</li><li>Zinc sulfate heptahydrate (ZnSO<sub>4</sub>Â·7H<sub>2</sub>O) (Millipore Sigma, catalog number: Z0251)</li><li>Iron sulfate heptahydrate (FeSO<sub>4</sub>Â·7H<sub>2</sub>O) (ACROS Organics, catalog number: 423730050)</li><li>Manganese sulfate hydrate (MnSO<sub>4</sub>Â·7H<sub>2</sub>O) (Millipore Sigma, catalog number: M7634)</li><li>Copper (II) sulfate pentahydrate (CuSO<sub>4</sub>Â·5H<sub>2</sub>O) (Millipore Sigma, catalog number: 209918)</li><li>Cobalt (II) nitrate hexahydrate (Co(NO<sub>3</sub>)<sub>2</sub>Â·6H<sub>2</sub>O) (Millipore Sigma, catalog number: 239267)</li><li>Sodium tetraborate decahydrate (Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>Â·10H<sub>2</sub>O) (Millipore Sigma, catalog number: B9876)</li><li>Sulfuric acid (Millipore Sigma, catalog number: 320501)</li><li>Nitrilotriacetic acid (C<sub>6</sub>H<sub>9</sub>NO<sub>6</sub>) (Millipore Sigma, catalog number: N9877)</li><li>Ammonium heptamolybdate tetrahydrate (NH<sub>4</sub>Mo<sub>7</sub>O<sub>24</sub>) (Millipore Sigma, catalog number: 09878)</li><li>Sodium phosphate dibasic (Na<sub>2</sub>HPO<sub>4</sub>) (Fisher, catalog number: BP331)</li><li>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) (Fisher, catalog number: P285)</li><li>Imidazole (Millipore Sigma, catalog number: I2399)</li><li>Glucose (Fisher, catalog number: D16500)</li><li>Sodium glutamate (Millipore Sigma, catalog number: 49621)</li><li>Ammonium chloride (NH<sub>4</sub>Cl) (Fisher, catalog number: A661)</li><li>Sodium dodecyl sulfate (SDS) (Millipore Sigma, catalog number: L3771)</li><li>Pronase E (VWR, catalog number: VE629)</li><li>Tris base (Fisher, catalog number: BP152)</li><li>Sodium chloride (NaCl) (Fisher, catalog number: BP358)</li><li>Mutanolysin from <em>Streptomyces globisporus</em> ATCC 21553 (Millipore Sigma, catalog number: M9901)</li><li>Phosphoric acid (H<sub>3</sub>PO<sub>4</sub>) (Millipore Sigma, catalog number: 79607)</li><li>Sodium hydroxide (NaOH) (Fisher, catalog number: S318)</li><li>Boric acid (H<sub>3</sub>BO<sub>3</sub>) (Millipore Sigma, catalog number: B6768)</li><li>Sodium borohydride (NaBH<sub>4</sub>) (Millipore Sigma, catalog number: 452882)</li><li>HPLC-grade methanol (Millipore Sigma, catalog number: 34860)</li><li>Sodium azide (NaN<sub>3</sub>) (Fisher, catalog number: BP9221)</li><li>Peptone-Yeast Extract (PYE) media (see Recipes)</li><li>Hutner-Imidazole-Glucose-Glutamate (HIGG) growth media (see Recipes) </li><li>Concentrated Hutner base (per 1 L) (see Recipes)</li><li>0.5 M Phosphate buffer, pH 7.0 (see Recipes)</li><li>HIGG media (see Recipes)</li><li>Pronase E buffer (see Recipes)</li><li>50 mM phosphate buffer (pH 4.9) (see Recipes)</li><li>500 mM borate buffer (pH 9) (see Recipes)</li><li>HPLC Buffer A (see Recipes)</li><li>HPLC Buffer B (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30230">### Equipment</p><ol class="npwtwok"><li>ELGA PURELAB Flex 3 water purification system (ELGA LabWater)</li><li>Magnetic hot plate-stirrer (Thermo Scientific, catalog number: SP195025)</li><li>Sorvall Lynx 6000 Centrifuge (Thermo Scientific)</li><li>50 ml conical tube centrifuge rotor (Thermo Scientific, catalog number: Fiberlite F14-14x50cy)</li><li>pH meter (Fisher, catalog number: 13-636-AB150) </li><li>Sorvall WX 80+ ultracentrifuge (Thermo Scientific, catalog number: 75000080)</li><li>Swinging bucket rotor TH-641 (Thermo Scientific, catalog number: 54295)</li><li>14 x 89 mm thickwall polycarbonate tube for TH-641 rotor (Seton Scientific 2013)</li><li>Fixed angle rotor T-1270 (Thermo Scientific, catalog number: 08259)</li><li>16 x 76 mm thickwall polycarbonate tube for T-1270 rotor (Seton Scientific 2004)</li><li>Digital dry bath (Fisher, catalog number: 88-871-001)</li><li>6 x 1.5 ml block for dry bath (Fisher, catalog number: 88-871-103)</li><li>6 x 2.0 ml block for dry bath (Fisher, catalog number: 88-871-104)</li><li>Microcentrifuge (Eppendorf, model: 5424)</li><li>Glass culture tubes (13 x 100 mm) (Fisher, catalog number: 14-961-27)</li><li>Microspatula (Fisher, catalog number: S50823)</li><li>Vortexer (Fisher, catalog number: 02-215-414)</li><li>Agilent 1260 Infinity II HPLC system including:<ol class="npwthree"><li>Agilent 1260 Infinity II Quaternary pump</li><li>Agilent 1260 Infinity II Multisampler</li><li>Agilent 1260 Infinity II Multicolumn thermostat</li><li>Agilent 1260 Infinity II Diode array detector (DAD)</li><li>Agilent 1260 Infinity II Quaternary Pump</li><li>Agilent 1260 Infinity II Analytical-scale fraction collector </li></ol></li><li>Hypersil ODS C18 HPLC column (3 Âµm particles, 4.6 mm diameter, 250 mm length) (Thermo Scientific, catalog number: 30103-254630)</li><li>Uniguard guard cartridge holder (Thermo Scientific, catalog number: 850-00)</li><li>Hypersil ODS C18 guard cartridge (3 Âµm particles, 4 mm diameter, 10 mm length) (Thermo Scientific, catalog number: 30103-014001)</li><li>Vacufuge concentrator (Eppendorf, model: 022820109)<br/></li></ol><p class="pptt" id="biaoti30231">### Software</p><ol class="npwtwok"><li>Matlab R2018b (Mathworks)</li><li><em>Chromanalysis</em> v. 1.0 (Desmarais <em>et al</em>., 2015)<br/></li></ol><p class="pptt" id="biaoti30232">### Procedure</p><p class="ppzz">This procedure is largely based on previously described protocols (Glauner, 1988; Desmarais <em>et al</em>., 2014). We include additional details for muropeptide separation by HPLC as well as methods for running consecutive samples with column cleaning and re-equilibration steps.</p><br/><ol class="npwone"><li>Inoculation and growth of bacterial cultures<ol class="npwtwo"><li>Inoculate 2.5 ml PYE media with a single colony of <em>Caulobacter crescentus</em> from an agar plate or directly from a freeze-down culture and grow overnight at 30 Â°C with shaking at 200 rpm.</li><li>The next day, back-dilute the culture into 500 ml of HIGG media (containing the desired phosphate concentration). Incubate this culture for 48 h at 30 Â°C with 200 rpm shaking.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Muropeptide purification<br/><strong>Day 1âCell lysis</strong> <ol class="npwtwo"><li>Bring 700 ml of water to a boil in a glass beaker on a stirrer-hot plate set to 450 Â°C. For each culture to be processed, prepare a 50 ml conical tube containing 6 ml of 6% (w/v) SDS and a micro stir bar, place the tube in the boiling water, and turn on the magnetic stirrer to 300 rpm (Figure 1). <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029020400_2277.jpg" width="200"/><br/><strong>Figure 1. Cell lysis.</strong> Cells in 4% SDS are boiled with stirring for 3 h. The arrow indicates the micro stir bar.<br/><br/></li><li>Harvest the cells after 48 h by centrifugation at 12,000 <em>x g</em> for 10 min (4 Â°C). Decant the supernatant. The starved cells are very buoyant and some of the pellet will slip away.</li><li>Resuspend the pellet in 3 ml of HIGG media and quickly transfer to the preheated 50 ml conical tubes to yield a final concentration of 4% SDS (w/v).</li><li>Boil the samples for 3 h with continuous stirring, periodically adding water to the beaker when necessary. Do not remove from heat too early as this step is critical for complete cell lysis as well as the inactivation of cell wall altering enzymes. After 3 h, turn off the heat and continue to stir the lysed cultures overnight.</li></ol><br/><strong>Day 2âEnzymatic digestions</strong> <ol class="npwtwo"><li>Check whether the SDS has precipitated overnight; if so, boil the sample for one hour to redissolve the SDS. Collect the sacculi by centrifugation at 276,000 <em>x g</em> for 1 h in a TH-641 rotor. Spin at 20 Â°C to avoid SDS precipitation. Resuspend the pellet in 6.5 ml ultrapure water and centrifuge the sample at 329,738 <em>x g</em> for 30 min (20 Â°C) in a T-1270 rotor. </li><li>Repeat this washing two times. If bubbles are still seen in the sample, add an extra wash to fully remove SDS (Figure 2).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029020425_9333.jpg" width="200"/><br/><strong>Figure 2. SDS removal.</strong> Samples washed in ultra-pure water should be transparent (left tube) and have lost the sudsy appearance with multiple washes (right tube).<br/><br/></li><li>Prepare 1 mg/ml Pronase E in 10 mM Tris-HCl (pH 7.2) + 0.06% w/v NaCl. Activate the Pronase E by heating the stock solution at 60 Â°C in a heat block for a minimum of 30 min (Figure 3).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029020510_4638.jpg" width="200"/><br/><strong>Figure 3. Enzymatic digestions.</strong> The cell envelope is digested with Pronase E to eliminate lipoproteins linking the outer membrane to the peptidoglycan. The intact sacculus is digested into muropeptide fragments that cleave along the glycan backbone.<br/><br/></li><li>Resuspend each sample pellet in 900 Âµl of 10 mM Tris-HCl (pH 7.2) + 0.06% w/v NaCl and transfer to a safe-lock microcentrifuge tube. Add 100 Âµl of the activated Pronase E to the tube and incubate for a minimum of 2 h on a 60 Â°C heat block. Pronase E is a protease that will digest lipoproteins associated with the peptidoglycan.</li><li>Add 200 Âµl of 6% SDS (w/v) to the digested samples and boil them in a 100 Â°C dry bath for 30 min to stop the proteolysis.</li><li>Resuspend the sacculi in 6.5 ml ultrapure water and centrifuge them at 329,738 <em>x g</em> for 30 min (20 Â°C) in a T-1270 rotor to wash out the SDS. Repeat the washing two times as before (or as long as necessary to eliminate any bubbles). </li><li>Prepare 1 mg/ml mutanolysin stocks in 50 mM phosphate buffer (pH 4.9), aliquot 40-50 Âµl into microcentrifuge tubes, and store the tubes at -20 Â°C.</li><li>After the last wash, resuspend the peptidoglycan pellet in 75-200 Âµl of 50 mM sodium phosphate buffer (pH 4.9) and transfer the sample to a 1.7 ml microcentrifuge tube. This volume will depend on how much peptidoglycan material is in the sample. For example, sacculi from a 500 ml <em>C. crescentus</em> culture at OD<sub>660</sub> of 0.8-1.0 were resuspended in 200 Âµl. Add mutanolysin to the sample to a final concentration of 40 Âµg/ml and digest overnight in a 37 Â°C dry bath (at least 10 h).</li></ol><br/><strong>Day 3âSample preparation for HPLC</strong> <ol class="npwtwo"><li>Inactivate the muramidase by boiling the samples in a 100 Â°C dry bath for 5 min.</li><li>Remove undigested material by centrifugation at 16,000 <em>x g</em> at room temperature for 10 min. Carefully transfer the supernatant into a 13 x 100 mm glass tube without disturbing the pellet.</li><li>Add 500 mM Borate buffer (pH 9) to the supernatant to a final concentration of 100 mM.</li><li>Add approximately half a micro-spatula amount (~ 6 mg) of sodium borohydride into each sample (Figure 4). Vigorous bubbling should be seen immediately, after which small bubbles will continue to rise in the solution. Reduce the muropeptides at room temperature for a minimum of 30 min. Vortex the tube gently to ensure complete mixing of the sodium borohydride. <br/><em>Caution: If too much sodium borohydride is added, it will precipitate during the pH titration and this will ruin the muropeptide purification.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029020533_4539.jpg" width="200"/><br/><strong>Figure 4. Reduction of muropeptides with sodium borohydride.</strong> Approximately 0.6 mg of sodium borohydride (half a micro-spatula) is added to the sample for muropeptide reduction.<br/><br/></li><li>Acidify the reduced muropeptides to pH 3.0-4.0 (the isoelectric point of the muropeptides is approximately 3.5). Start by adding 15 Âµl of 50% phosphoric acid (vigorous bubbling will be seen) and gently vortex the tube to ensure thorough mixing. Test the pH by placing a 1 Âµl drop of sample onto the pH indicator paper. Continue to add phosphoric acid in increments of 5 Âµl until the pH is 5.5-6.0 (typically 1-2 additions). Continue adding phosphoric acid in increments of 1 Âµl until reaching a pH between 3.0 and 4.0.</li><li>Transfer the samples to a 1 ml syringe and filter them through a Millex-GV syringe filter directly into a 12 x 32 mm HPLC vial. Samples can be stored long-term at -20 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>HPLC set-up and sample analysis<ol class="npwtwo"><li>While directing flow into the waste container, wash all 4 pump lines of the Agilent HPLC system with ultra-pure water for a minimum of 10 min. Close the waste valve and wash the internal tubing by flushing with ultra-pure water for an additional 10 min.</li><li>Connect pump A with Solvent A (Recipe 9), pump B with Solvent B (Recipe 9), pump C with HPLC-grade methanol and pump D with ultra-pure water. While directing flow to the waste container, flush each line with its respective solvent. </li><li>Set the temperature of the column incubator to 55 Â°C. Close the waste valve and set the flow-rate to 0.5 ml/min from Line D (100% Ultra-pure water). </li><li>Prepare the HPLC column by first attaching the Uniguard column containing a guard cartridge. Attach the column to the HPLC system by first allowing 5-6 drops of water to go into the upstream end of the column. Firmly tighten the upstream connector and wait for the water to drip from the downstream end before connecting the other end of the column. </li><li>Wash the column with ultrapure water for 20 min and ensure that the column pressure does not exceed 150 bar. Set the DAD detector to collect the absorbance at 205 nm.</li><li>Equilibrate the column with Solvent A at 0.5 ml/min for 50 min. The column pressure and UV absorbance will stabilize during this step.</li><li>Prepare the HPLC samples as follows. Place a 100 Âµl insert into a clean HPLC vial and add 50-90 Âµl of sample to the insert. Prepare 2 vials with ultrapure water which will be used to blank the system as well as during cleanup. Arrange the vials in the autosampler unit.</li><li>If collecting peaks for further analysis by mass spectrometry, enable time-based fraction collection to collect the desired peaks. Place the appropriate number of collection tubes in the fraction collector according to the number of collected peaks. Each tube will collect approximately 100-450 Âµl of sample depending on the duration of the fraction collection window (10-50 s). Since the elution time can vary slightly from run to run, we recommend collecting extra fractions before and after the expected elution time to ensure recovery of the desired peak.</li><li>The HPLC method consists of a 135 min linear gradient (0.5 ml/min) from 100% Solvent A to 100% Solvent B. To re-equilibrate the column between samples, continue flowing 100% Solvent B (0.5 ml/min) for 5 min, followed by a 25 min linear gradient to 100% Solvent A, and finally an additional 25 min of 100% Solvent A. At the end of this washing protocol, the pressure and absorbance at 205 nm should be stabilized.</li><li>Prior to running muropeptide samples, inject 50 Âµl of ultra-pure water into the HPLC system to serve as a blank for background subtraction. For the muropeptide samples, inject 40-80 Âµl; be certain not to inject more than the total volume of the sample to ensure that an air bubble is not introduced into the HPLC system.</li><li>After the last sample has completed, wash the system and the column by injecting 20 Âµl of ultra-pure water and running the following washing method. Begin with a 30 min flush with 100% water (0.2 ml/min), followed by a 10 min linear gradient to 100% methanol. Wash with 100% methanol for 60 min followed by a 10 min linear gradient to 10% methanol/90% water. Flush the column with 10% methanol/90% water for an additional 10 min before removing the column for storage.</li><li>If fractions were collected, combine the fractions containing the peak of interest and remove the solvent using a vacufuge concentrator. Set the vent mode to âaqueousâ and spin the samples in the vacufuge concentrator until dry, typically 30-60 min at 30 Â°C. A small translucent, white pellet may be seen on the bottom of the tube. These samples are suitable for LC/MS analysis.</li><li>The HPLC trace can be quantified using the Matlab-based <em>Chromanalysis</em> software as previously described (Desmarais <em>et al</em>., 2015).</li></ol></li></ol><p class="pptt" id="biaoti30233">### Data analysis</p><p class="ppzz">The protocol above can be used to compare the muropeptide compositions of bacteria growing in different nutrient environments. For <em>C. crescentus</em>, phosphate starvation leads to an increase in LD-transpeptidation (Figure 5). <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191029/20191029020607_1326.jpg" width="200"/><br/><strong>Figure 5. Muropeptide analysis of <em>C. crescentus</em> grown in different phosphate concentrations. </strong>Muropeptides were purified from <em>Caulobacter crescentus</em> grown in HIGG media with 1 mM or 1 ÂµM phosphate and analyzed via HPLC. The arrow indicates a peak uniquely present in the 1 ÂµM phosphate chromatogram. Major peaks were labeled based on previous analysis of <em>C. crescentus</em> muropeptides (Takacs <em>et al</em>., 2010). LD-crosslinks were labeled based on previous analysis of <em>E. coli</em> muropeptides (Glauner, 1988) and confirmed by LC/MS. This figure is reprinted with permission from John Wiley and Sons (Stankeviciute <em>et al</em>., 2019).</p><p class="pptt" id="biaoti30234">### Notes</p><ol class="npwtwok"><li>The protocol for operating the HPLC system may vary depending on your particular instrumentation. For an overview of HPLC operation, we recommend the article and accompanying video from the JoVE Science Education Database (<a href="https://www.jove.com/science-education/10156/high-performance-liquid-chromatography-hplc" target="_blank">https://www.jove.com/science-education/10156/high-performance-liquid-chromatography-hplc</a>).</li><li>Prior to peak quantification, perform background subtraction using the blank-run (water sample).<br/></li></ol><p class="pptt" id="biaoti30235">### Recipes</p><ol class="npwtwok"><li>Peptone-Yeast Extract (PYE) media (1 L)<br/>2.0 g peptone<br/>1.0 g yeast extract <br/>1 mM MgSO<sub>4</sub><br/>0.5 mM CaCl<sub>2</sub> </li><li>Hutner-Imidazole-Glucose-Glutamate (HIGG) media (Poindexter, 1978)<br/>There are several stock solutions required for this media:<ol class="npwthree"><li>Metals â44â (per 100 ml)<br/>1.3 ml 0.5 M disodium EDTA<br/>1,095.0 mg ZnSO<sub>4</sub>Â·7H<sub>2</sub>O<br/>500.0 mg FeSO<sub>4</sub>Â·7H<sub>2</sub>O<br/>154.0 mg MnSO<sub>4</sub>Â·H<sub>2</sub>O<br/>39.2 mg CuSO<sub>4</sub>Â·5H<sub>2</sub>O<br/>24.8 mg Co(NO<sub>3</sub>)<sub>2</sub>Â·6H<sub>2</sub>O<br/>17.7 mg Na<sub>2</sub>B<sub>4</sub>O<sub>7</sub>Â·10H<sub>2</sub>O<br/>Bring volume to 100 ml with distilled water<br/>Add 2-3 drops of sulfuric acid to prevent precipitation</li><li>Concentrated Hutner base (per 1 L)<br/>10.0 g Nitrilotriacetic acid <br/>29.59 g MgSO<sub>4</sub>Â·7H<sub>2</sub>O<br/>3.335 g CaCl<sub>2</sub>Â·2H<sub>2</sub>O<br/>9.25 mg (NH<sub>4</sub>)<sub>6</sub>Mo<sub>7</sub>O<sub>24</sub>Â·4H<sub>2</sub>O<br/>99.0 mg FeSO<sub>4</sub>Â·7H<sub>2</sub>O<br/>50.0 ml Metals â44â<br/>Distilled water to 1,000 ml<ol class="npwfour"><li>Before adding all reagents, dissolve nitrilotriacetic acid and neutralize with KOH; approximately 5.6 g KOH will be required </li><li>Add the rest of the components and adjust the pH to 6.6-6.8 before adjusting the volume </li><li>Bring the solution to 1,000 ml with distilled water </li><li>Sterilize the solution by filtration (do not autoclave)</li></ol></li><li>0.5 M Phosphate buffer, pH 7.0<br/>Mix 61 ml of 0.5 M Na<sub>2</sub>HPO<sub>4</sub> with 39 ml of 0.5 M KH<sub>2</sub>PO<sub>4</sub> and sterilize by autoclaving</li><li>1 M imidazole, pH 7.0 (autoclaved)</li><li>20% (w/v) glucose (sterile filtered)</li><li>20% (w/v) sodium glutamate (sterile filtered)</li><li>50 mM CaCl<sub>2</sub> (autoclaved)</li><li>1 M NH<sub>4</sub>Cl (autoclaved)</li></ol>To prepare the HIGG media, autoclave the desired amount of distilled water and add the following components to the final concentrations listed:<br/>5 mM imidazole, pH 7.0<br/>2% (v/v) Hutnerâs base<br/>1 mM CaCl<sub>2</sub><br/>0.15% (w/v) glucose<br/>0.15% (w/v) sodium glutamate<br/>8.9 mM NH<sub>4</sub>Cl<br/>1 Î¼M-1,000 Î¼M phosphate</li><li>Pronase E buffer<br/>10 mM Tris, pH 7.2<br/>0.06% (w/v) NaCl</li><li>50 mM phosphate buffer, pH 4.9 (100 ml)<ol class="npwthree"><li>Dissolve 0.2 g NaOH in ~90 ml water</li><li>Adjust the pH of the solution to 4.9 with phosphoric acid</li><li>Bring the total volume to 100 ml with water</li></ol></li><li>500 mM borate buffer, pH 9 (500 ml)<ol class="npwthree"><li>Dissolve 15.46 g boric acid in ~480 ml water</li><li>Adjust the pH to 9.0 with NaOH</li><li>Bring the total volume to 500 ml with water</li></ol></li><li>HPLC buffers<br/><strong>Buffer A</strong>: 50 mM NaPO<sub>4</sub> pH 4.35, 0.4% NaN<sub>3</sub> <ol class="npwthree"><li>Dissolve 2.0 g NaOH into ~950 ml ultrapure water</li><li>Adjust the pH to 4.35 with phosphoric acid</li><li>Add 200 Âµl 2% (w/v) NaN<sub>3</sub> </li><li>Bring the total volume to 1,000 ml with ultrapure water</li><li>Re-adjust the pH to 4.35</li><li>Sterile filter before use<br/><br/></li></ol><strong>Buffer B:</strong> 75 mM NaPO<sub>4</sub> pH 4.95, 15% methanol<ol class="npwthree"><li>Dissolve 3.0 g NaOH into ~800 ml ultrapure water</li><li>Adjust the pH to 4.95 with phosphoric acid</li><li>Bring the total volume to 850 ml with ultrapure water</li><li>Add 150 ml HPLC-grade methanol</li><li>Re-adjust the pH to 4.95</li><li>Sterile filter before use</li></ol></li></ol><p class="pptt" id="biaoti30236">### Acknowledgments</p><p class="ppzz">We thank Amanda Miguel and K.C. Huang (Stanford University) for their guidance in muropeptide purification. We have generally adapted their protocol for UPLC-based separation (Desmarais <em>et al.</em>, 2014) for use on an HPLC system. This work was supported by a grant from the National Science Foundation (MCB-1553004) to E.A.K.</p><p class="pptt" id="biaoti30237">### Competing interests</p><p class="ppzz">The authors have no competing interests to declare.</p><p class="pptt" id="biaoti30238">### References</p><ol class="npwtwok"><li>Clarke, T. B. and Weiser, J. N. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21884164" target="_blank">Intracellular sensors of extracellular bacteria.</a> <em>Immunol Rev</em> 243(1): 9-25.</li><li>de Pedro, M. A. and Cava, F. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26005443" target="_blank">Structural constraints and dynamics of bacterial cell wall architecture.</a> <em>Front Microbiol</em> 6: 449.</li><li>Desmarais, S. M., Cava, F., de Pedro, M. A. and Huang, K. C. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24457605" target="_blank">Isolation and preparation of bacterial cell walls for compositional analysis by ultra performance liquid chromatography.</a> <em>J Vis Exp</em> 83: e51183.</li><li>Desmarais, S. M., Tropini, C., Miguel, A., Cava, F., Monds, R. D., de Pedro, M. A. and Huang, K. C. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26468288" target="_blank">High-throughput, highly sensitive analyses of bacterial morphogenesis using ultra performance liquid chromatography.</a> <em>J Biol Chem</em> 290(52): 31090-100.</li><li>Glauner, B. (1988). <a href="https://www.ncbi.nlm.nih.gov/pubmed/3056100" target="_blank">Separation and quantification of muropeptides with high-performance liquid chromatography.</a> <em>Anal Biochem</em> 172(2): 451-64.</li><li>Levefaudes, M., Patin, D., de Sousa-d'Auria, C., Chami, M., Blanot, D., Herve, M., Arthur, M., Houssin, C. and Mengin-Lecreulx, D. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25847251" target="_blank">Diaminopimelic acid amidation in Corynebacteriales: new insights into the role of LtsA in peptidoglycan modification.</a> <em>J Biol Chem</em> 290(21): 13079-94.</li><li>Mesnage, S., Dellarole, M., Baxter, N. J., Rouget, J. B., Dimitrov, J. D., Wang, N., Fujimoto, Y., Hounslow, A. M., Lacroix-Desmazes, S., Fukase, K., Foster, S. J. and Williamson, M. P. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24978025" target="_blank">Molecular basis for bacterial peptidoglycan recognition by LysM domains.</a> <em>Nat Commun</em> 5: 4269.</li><li>Poindexter, J. S. (1978). <a href="https://www.ncbi.nlm.nih.gov/pubmed/690072" target="_blank">Selection for nonbuoyant morphological mutants of <em>Caulobacter crescentus</em>.</a> <em>J Bacteriol</em> 135(3): 1141-5.<br/></li><li>Royet, J., Gupta, D. and Dziarski, R. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22076558" target="_blank">Peptidoglycan recognition proteins: modulators of the microbiome and inflammation.</a> <em>Nat Rev Immunol</em> 11(12): 837-51.</li><li>Schleifer, K. H. and Kandler, O. (1972). <a href="https://www.ncbi.nlm.nih.gov/pubmed/4568761" target="_blank">Peptidoglycan types of bacterial cell walls and their taxonomic implications.</a> <em>Bacteriol Rev</em> 36(4): 407-77.</li><li>Schoonmaker, M. K., Bishai, W. R. and Lamichhane, G. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24464457" target="_blank">Nonclassical transpeptidases of <em>Mycobacterium tuberculosis</em> alter cell size, morphology, the cytosolic matrix, protein localization, virulence, and resistance to beta-lactams.</a> <em>J Bacteriol </em>196(7): 1394-402.</li><li>Stankeviciute, G., Miguel, A. V., Radkov, A., Chou, S., Huang, K. C. and Klein, E. A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30614079" target="_blank">Differential modes of crosslinking establish spatially distinct regions of peptidoglycan in <em>Caulobacter crescentus.</em></a> <em>Mol Microbiol</em> 111(4): 995-1008.</li><li>Takacs, C. N., Poggio, S., Charbon, G., Pucheault, M., Vollmer, W. and Jacobs-Wagner, C. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20023035" target="_blank">MreB drives de novo rod morphogenesis in <em>Caulobacter crescentus</em> via remodeling of the cell wall.</a> <em>J Bacteriol</em> 192(6): 1671-84.</li><li>Tocheva, E. I., Lopez-Garrido, J., Hughes, H. V., Fredlund, J., Kuru, E., Vannieuwenhze, M. S., Brun, Y. V., Pogliano, K. and Jensen, G. J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23531131" target="_blank">Peptidoglycan transformations during <em>Bacillus subtilis</em> sporulation.</a> <em>Mol Microbiol</em> 88(4): 673-86.</li><li>Vollmer, W. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18070068" target="_blank">Structural variation in the glycan strands of bacterial peptidoglycan.</a> <em>FEMS Microbiol Rev</em> 32(2): 287-306.<br/><br/></li></ol>