<body>
            <h1>Isolation and Quantification of Metabolite Levels in Murine Tumor Interstitial Fluid by LC/MS </h1>
            <p><strong>Authors:</strong> Mark R Sullivan, Caroline A Lewis, Alexander Muir</p>
            <h2>Abstract</h2>
            <p>Cancer is a disease characterized by altered metabolism, and there has been renewed interest in understanding the metabolism of tumors. Even though nutrient availability is a critical determinant of tumor metabolism, there has been little systematic study of the nutrients directly available to cancer cells in the tumor microenvironment. Previous work characterizing the metabolites present in the tumor interstitial fluid has been restricted to the measurement of a small number of nutrients such as glucose and lactate in a limited number of samples. Here we adapt a centrifugation-based method of tumor interstitial fluid isolation readily applicable to a number of sample types and a mass spectrometry-based method for the absolute quantitation of many metabolites in interstitial fluid samples. In this method, tumor interstitial fluid (TIF) is analyzed by liquid chromatography-mass spectrometry (LC/MS) using both isotope dilution and external standard calibration to derive absolute concentrations of targeted metabolites present in interstitial fluid. The use of isotope dilution allows for accurate absolute quantitation of metabolites, as other methods of quantitation are inadequate for determining nutrient concentrations in biological fluids due to matrix effects that alter the apparent concentration of metabolites depending on the composition of the fluid in which they are contained. This method therefore can be applied to measure the absolute concentrations of many metabolites in interstitial fluid from diverse tumor types, as well as most other biological fluids, allowing for characterization of nutrient levels in the microenvironment of solid tumors.</p>
            <p><strong>Keywords:</strong> Nutrients, Metabolomics, Microenvironment, Interstitial fluid, Cancer metabolism, Mass spectrometry</p>
            <h2>Background</h2><p>Cell division requires the duplication of the biomass of the mother cell prior to division. As a result, growing cells must be able to utilize the nutrients available in their environments to synthesize the macromolecules required to divide. To sustain cancerous proliferation, tumors often exhibit altered metabolism (DeBerardinis and Chandel, 2016). In many cases, tumor metabolism is driven by cell-intrinsic processes such as oncogenic activation (Cairns <em>et al.</em>, 2011; Nagarajan <em>et al.</em>, 2016). However, recent work has highlighted the importance of cell-extrinsic factors in dictating cancer cell metabolism (Anastasiou, 2017; Bi <em>et al.</em>, 2018; Muir <em>et al.</em>, 2018). The importance of the extracellular environment in shaping cancer metabolism is perhaps unsurprising, as the nutrient environment in which a cancer cell exists constrains which metabolic reactions are possible within that cell. Since cell-extrinsic metabolite levels can play a role in determining the behavior of tumor cells, it is critical to examine tumor cell metabolism under physiological conditions. However, our understanding of the metabolic composition of the tumor microenvironment is lacking.<br/><br/>The nutrient environment that a cancer cell has access to is predominantly composed of interstitial fluid (Wiig and Swartz, 2012). Understanding the nutrient content of tumor interstitial fluid would provide insight into the metabolic constraints imposed upon tumor cells by their environment. There exist multiple methodologies for isolating interstitial fluid from normal organs and from tumors (Wiig <em>et al.</em>, 2010). However, early attempts to measure the nutrient content of interstitial fluid were limited by their inability to measure multiple metabolites, and consequently our knowledge of nutrient availability in tumors is restricted to a few metabolites in a limited number of animal tumor models (Burgess and Sylven, 1962; Gullino <em>et al.</em>, 1964). The advent of mass spectrometry has allowed for detection of many metabolites simultaneously. However, despite technological advances, metabolomics studies are complicated by the fact that components present in biological fluids can suppress or enhance the detection of specific metabolites. These discrepancies in detection of metabolites between different biological fluids are termed âmatrix effects,â and are a major confounding factor in comparing metabolite concentrations between different biological fluids and in accurately quantitating metabolites in those fluids (Panuwet <em>et al.</em>, 2016; Sullivan <em>et al.</em>, 2019).<br/><br/>Here we demonstrate a method for centrifugation-based isolation of tumor interstitial fluid and the subsequent absolute quantitation of numerous metabolites within that fluid using stable isotope dilution, a technique in which stable isotope-labeled internal standards for metabolites of interest are added to experimental samples. These stable isotope internal standards are subject to the same matrix effects as the corresponding metabolite in the sample and can be distinguished by their increased mass compared to the metabolites in the sample. To measure many metabolites simultaneously, we first quantitate the concentrations of <sup>13</sup>C metabolites from an extract of polar metabolites from yeast that are cultured with <sup>13</sup>C isotopically labeled glucose as the sole carbon source. This quantitated yeast extract is then used as an internal standard that allows for reliable quantification of targeted metabolites in biological samples while minimizing systematic error from matrix effects. This approach provides a robust method to quantitate polar metabolites in biological fluids and complements similar existing isotope dilution based methods, such as the commercially available Biocrates AbsoluteIDQ kits (Gieger <em>et al.</em>, 2008) that primarily quantify non-polar lipids in biological samples.<br/><br/>The absolute quantitation of metabolite levels enabled by this protocol can allow for direct comparison of interstitial fluid composition in diverse tumor types, providing the opportunity to systematically interrogate nutrient availability in animal models of diverse cancers and human tumor samples. Further, the absolute quantification of metabolite levels in interstitial fluid allows for the generation of tissue culture media that mimics physiological conditions found in a tumor, thus expanding the range of <em>in vitro</em>/ <em>ex vivo</em> experiments that can be carried out under physiological nutrient conditions. Most broadly, this protocol provides a method to absolutely quantify many metabolites simultaneously in complex biological fluids, which can be used to study the metabolic composition of any biological material.</p>
        </body>

<p class="pptt" id="biaoti30401">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml conical tube (Falcon, catalog number: 14 959 49A)</li><li>Lab tape (Thermo Fisher, catalog number: 15-901-20H)</li><li>EDTA-coated plasma collection tubes (Sarstedt, catalog number: 41.1395.105)</li><li>1 ml 25G TB syringe (Becton Dickinson, catalog number: 309625)</li><li>20 Î¼M mesh nylon filter (Spectrum Labs, catalog number: 148134)</li><li>Whatman paper (Thermo Fisher, catalog number: 88600)</li><li>Petri dishes (Corning, catalog number: 07 202 011)</li><li>Eppendorf tubes (Corning, catalog number: 14 222 168)</li><li>LC/MS sample vials (Thermo Fisher, catalog number: C4000-11)</li><li>LC/MS vial caps (Thermo Fisher, catalog number: C5000-54B)</li><li>Glassware (bottles, graduated cylinders, conical flasks) reserved for LCMS (not washed by central glass washing services, as this can introduce surfactant and other contaminants into your system)</li><li>Pipettes (Gilson, catalog number: F167380)</li><li>Mouse surgical kit (Thermo Fisher, catalog number: 14516249)</li><li>Blood bank saline (Azer Scientific, catalog number: 16005-092)</li><li>70% Ethanol (Thermo Fisher, catalog number: 04-355-305)</li><li>Acetonitrile LC/MS Optima 4 L (Fisher Scientific, catalog number: A955-4)</li><li>Methanol LC/MS Optima 4 L (Fisher Scientific, catalog number: A456-4)</li><li>Pierce formic acid (99%, LC/MS grade, Life Technologies, catalog number: 28905)</li><li>Water LC/MS Optima 4 L (Fisher Scientific, catalog number: W64)</li><li>Ammonium carbonate (Sigma-Aldrich, catalog number: 379999)</li><li>Ammonium hydroxide solution (28%, Sigma-Aldrich, catalog number: 338818)</li><li>Metabolomics amino acid standard mix (Cambridge Isotope Laboratories, Inc., catalog number: MSK-A2-1.2)</li><li><sup>13</sup>C isotopically labeled yeast metabolite extract (Cambridge Isotope Laboratories, Inc., catalog number: ISO1)</li><li><sup>13</sup>C<sub>3</sub> lactate (Sigma-Aldrich, catalog number: 485926)</li><li><sup>13</sup>C<sub>3 </sub>glycerol (Cambridge Isotope Laboratory, catalog number: CLM-1510)</li><li><sup>13</sup>C<sub>6</sub> <sup>15</sup>N<sub>2</sub> cystine (Cambridge Isotope Laboratory, catalog number: CNLM-4244)</li><li><sup>2</sup>H<sub>9</sub> choline (Cambridge Isotope Laboratory, catalog number: DLM-549)</li><li><sup>13</sup>C<sub>4</sub> 3-hydroxybutyrate (Cambridge Isotope Laboratory, catalog number: CLM-3853)</li><li><sup>13</sup>C<sub>6</sub> glucose (Cambridge Isotope Laboratory, catalog number: CLM-1396)</li><li><sup>13</sup>C<sub>2</sub> <sup>15</sup>N taurine (Cambridge Isotope Laboratory, catalog number: CNLM-10253)</li><li><sup>2</sup>H<sub>3</sub> creatinine (Cambridge Isotope Laboratory, catalog number: DLM-3653)</li><li><sup>13</sup>C<sub>5</sub> hypoxanthine (Cambridge Isotope Laboratory, catalog number: CLM-8042)</li><li><sup>13</sup>C<sub>3</sub> serine (Cambridge Isotope Laboratory, catalog number: CLM-1574)</li><li><sup>13</sup>C<sub>2</sub> glycine (Cambridge Isotope Laboratory, catalog number: CLM-1017)</li><li>Chemical standard library pool 1<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>Alanine<br/></td><td>Sigma<br/></td><td>A7469<br/></td></tr><tr><td>Arginine<br/></td><td>Sigma<br/></td><td>A6969<br/></td></tr><tr><td>Asparagine<br/></td><td>Sigma<br/></td><td>A7094<br/></td></tr><tr><td>Aspartate<br/></td><td>Sigma<br/></td><td>A7219<br/></td></tr><tr><td>Carnitine<br/></td><td>Sigma<br/></td><td>C0283<br/></td></tr><tr><td>Citrulline<br/></td><td>Sigma<br/></td><td>C7629<br/></td></tr><tr><td>Cystine<br/></td><td>Sigma<br/></td><td>C7602<br/></td></tr><tr><td>Glutamate<br/></td><td>Sigma<br/></td><td>G8415<br/></td></tr><tr><td>Glutamine<br/></td><td>Sigma<br/></td><td>G3126<br/></td></tr><tr><td>Glycine<br/></td><td>Sigma<br/></td><td>G7126<br/></td></tr><tr><td>Histidine<br/></td><td>Sigma<br/></td><td>53319<br/></td></tr><tr><td>Hydroxyproline<br/></td><td>Sigma<br/></td><td>H54409<br/></td></tr><tr><td>Isoleucine<br/></td><td>Sigma<br/></td><td>I7403<br/></td></tr><tr><td>Leucine<br/></td><td>Sigma<br/></td><td>L8912<br/></td></tr><tr><td>Lysine<br/></td><td>Sigma<br/></td><td>L8662<br/></td></tr><tr><td>Methionine<br/></td><td>Sigma<br/></td><td>M5308<br/></td></tr><tr><td>Ornithine<br/></td><td>Sigma<br/></td><td>75469<br/></td></tr><tr><td>Phenylalanine<br/></td><td>Sigma<br/></td><td>P5482<br/></td></tr><tr><td>Proline<br/></td><td>Sigma<br/></td><td>P5607<br/></td></tr><tr><td>Serine<br/></td><td>Sigma<br/></td><td>S4311<br/></td></tr><tr><td>Taurine<br/></td><td>Sigma<br/></td><td>T0625<br/></td></tr><tr><td>Threonine<br/></td><td>Sigma<br/></td><td>T8441<br/></td></tr><tr><td>Tryptophan<br/></td><td>Sigma<br/></td><td>T8941<br/></td></tr><tr><td>Tyrosine<br/></td><td>Sigma<br/></td><td>T8566<br/></td></tr><tr><td>Valine<br/></td><td>Sigma<br/></td><td>V0513<br/></td></tr><tr><td>Lactate<br/></td><td>Sigma<br/></td><td>L7022<br/></td></tr><tr><td>Glucose<br/></td><td>Sigma<br/></td><td>G7528</td></tr></tbody></table></li><li>Chemical standard library pool 2<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>2-hydroxybutyric acid<br/></td><td>Sigma<br/></td><td>220116<br/></td></tr><tr><td>2-aminobutyric acid<br/></td><td>Sigma<br/></td><td>162663-25G<br/></td></tr><tr><td>AMP<br/></td><td>Sigma<br/></td><td>A1752<br/></td></tr><tr><td>Argininosuccinate<br/></td><td>Sigma<br/></td><td>A5707-50MG<br/></td></tr><tr><td>Betaine<br/></td><td>Sigma<br/></td><td>61962<br/></td></tr><tr><td>Biotin<br/></td><td>Sigma<br/></td><td>B4639<br/></td></tr><tr><td>Carnosine<br/></td><td>Sigma<br/></td><td>C9625-5G<br/></td></tr><tr><td>Choline<br/></td><td>Sigma<br/></td><td>C7017<br/></td></tr><tr><td>CMP <br/></td><td>Sigma<br/></td><td>C1006<br/></td></tr><tr><td>Creatine<br/></td><td>Sigma<br/></td><td>C0780-50G<br/></td></tr><tr><td>Cytidine <br/></td><td>Sigma<br/></td><td>C4654<br/></td></tr><tr><td>dTMP <br/></td><td>Sigma<br/></td><td>T7004-100MG<br/></td></tr><tr><td>Fructose<br/></td><td>Sigma<br/></td><td>F0127<br/></td></tr><tr><td>Glucose-1-phosphate<br/></td><td>Sigma<br/></td><td>G6750<br/></td></tr><tr><td>Glutathione<br/></td><td>Sigma<br/></td><td>G4251<br/></td></tr><tr><td>GMP <br/></td><td>Sigma<br/></td><td>G8377<br/></td></tr><tr><td>IMP <br/></td><td>Sigma<br/></td><td>57510-5G<br/></td></tr><tr><td>O-phosphoethanolamine<br/></td><td>Sigma<br/></td><td>P0503-1G<br/></td></tr><tr><td>Pyridoxal <br/></td><td>Sigma<br/></td><td>P9130-500MG<br/></td></tr><tr><td>Thiamine <br/></td><td>Sigma<br/></td><td>T4625<br/></td></tr><tr><td>trans-Urocanate<br/></td><td>Cayman<br/></td><td>16228<br/></td></tr><tr><td>UMP <br/></td><td>Sigma<br/></td><td>U6375-1G<br/></td></tr><tr><td>Xanthine <br/></td><td>Sigma<br/></td><td>X7375-10G<br/></td></tr></tbody></table></li><li>Chemical standard library pool 3<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>3-hydroxybutyric acid<br/></td><td>Sigma<br/></td><td>H6501<br/></td></tr><tr><td>Acetylalanine <br/></td><td>Sigma<br/></td><td>A4625-1G<br/></td></tr><tr><td>Acetylaspartate <br/></td><td>Sigma<br/></td><td>00920-5G<br/></td></tr><tr><td>Acetylcarnitine <br/></td><td>Sigma<br/></td><td>A6706<br/></td></tr><tr><td>Acetylglutamine <br/></td><td>Sigma<br/></td><td>A9125-25G<br/></td></tr><tr><td>ADP <br/></td><td>Sigma<br/></td><td>A5285<br/></td></tr><tr><td>Allantoin <br/></td><td>Sigma<br/></td><td>05670-25G<br/></td></tr><tr><td>CDP <br/></td><td>Abcam<br/></td><td>ab146214-100 mg<br/></td></tr><tr><td>CDP-choline<br/></td><td>Alfa<br/></td><td>J64161-10 g<br/></td></tr><tr><td>Coenzyme A<br/></td><td>Sigma<br/></td><td>C4282<br/></td></tr><tr><td>Creatinine <br/></td><td>Sigma<br/></td><td>C4255-10MG<br/></td></tr><tr><td>gamma-aminobutyric acid<br/></td><td>Sigma<br/></td><td>A2129-10G<br/></td></tr><tr><td>GDP <br/></td><td>Sigma<br/></td><td>G7127<br/></td></tr><tr><td>Glutathione disulfide<br/></td><td>Sigma<br/></td><td>G4376<br/></td></tr><tr><td>Glycerate<br/></td><td>Sigma<br/></td><td>367494<br/></td></tr><tr><td>Hypoxanthine <br/></td><td>Sigma<br/></td><td>H9377<br/></td></tr><tr><td>myo-Inositol<br/></td><td>Sigma<br/></td><td>I5125<br/></td></tr><tr><td>NAD+<br/></td><td>Sigma<br/></td><td>N1511<br/></td></tr><tr><td>p-aminobenzoate<br/></td><td>Sigma<br/></td><td>A9878<br/></td></tr><tr><td>Phosphocholine<br/></td><td>Sigma<br/></td><td>P0378-5G<br/></td></tr><tr><td>Sorbitol<br/></td><td>Sigma<br/></td><td>W302902<br/></td></tr><tr><td>UDP <br/></td><td>Sigma<br/></td><td>94330-100MG<br/></td></tr><tr><td>UDP-glucose<br/></td><td>Sigma<br/></td><td>U4625-100MG<br/></td></tr></tbody></table></li><li>Chemical standard library pool 4<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>Phenylacetylglutamine <br/></td><td>Cayman <br/></td><td>16724-25mg<br/></td></tr><tr><td>Acetylglutamate <br/></td><td>Sigma<br/></td><td>855642<br/></td></tr><tr><td>Acetylglycine <br/></td><td>Sigma<br/></td><td>A16300-5G<br/></td></tr><tr><td>Acetylmethionine <br/></td><td>Sigma<br/></td><td>01310-5G<br/></td></tr><tr><td>Asymmetric dimethylarginine<br/></td><td>Cayman<br/></td><td>80230<br/></td></tr><tr><td>ATP<br/></td><td>Sigma<br/></td><td>A2383<br/></td></tr><tr><td>CTP<br/></td><td>Sigma<br/></td><td>C1506<br/></td></tr><tr><td>dATP<br/></td><td>Sigma<br/></td><td>D6500<br/></td></tr><tr><td>dCTP<br/></td><td>Sigma<br/></td><td>D4635<br/></td></tr><tr><td>Deoxycytidine <br/></td><td>Sigma<br/></td><td>D3897<br/></td></tr><tr><td>Folic acid<br/></td><td>Sigma<br/></td><td>F8758<br/></td></tr><tr><td>GTP <br/></td><td>Sigma<br/></td><td>G8877<br/></td></tr><tr><td>Hypotaurine <br/></td><td>Sigma<br/></td><td>H1384-100MG<br/></td></tr><tr><td>Methionine sulfoxide<br/></td><td>Sigma<br/></td><td>M1126-1G<br/></td></tr><tr><td>Methylthioadenosine<br/></td><td>Sigma<br/></td><td>D5011-25MG<br/></td></tr><tr><td>Phosphocreatine<br/></td><td>Sigma<br/></td><td>P7936-1G<br/></td></tr><tr><td>Pyridoxine <br/></td><td>Sigma<br/></td><td>P9755<br/></td></tr><tr><td>Ribose-5-phosphate<br/></td><td>Sigma<br/></td><td>83875<br/></td></tr><tr><td>SAH <br/></td><td>Sigma<br/></td><td>A9384-25MG<br/></td></tr><tr><td>Thymidine <br/></td><td>Sigma<br/></td><td>T9250<br/></td></tr><tr><td>Trimethyllysine<br/></td><td>Sigma<br/></td><td>T1660-25MG<br/></td></tr><tr><td>Uridine <br/></td><td>Sigma<br/></td><td>T1660-25MG<br/></td></tr><tr><td>Uridine <br/></td><td>Sigma<br/></td><td>U3003<br/></td></tr><tr><td>UTP <br/></td><td>Sigma<br/></td><td>U6625<br/></td></tr></tbody></table></li><li>Chemical standard library pool 5<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>3-phosphoglycerate<br/></td><td>Sigma<br/></td><td>P8877<br/></td></tr><tr><td>cis-aconitic acid<br/></td><td>Sigma<br/></td><td>A3412-1G<br/></td></tr><tr><td>Citrate <br/></td><td>Sigma<br/></td><td>754<br/></td></tr><tr><td>DHAP <br/></td><td>Sigma<br/></td><td>51269<br/></td></tr><tr><td>Fructose-1,6-bisphosphate<br/></td><td>Sigma<br/></td><td>F6803<br/></td></tr><tr><td>Fumarate <br/></td><td>Sigma<br/></td><td>240745<br/></td></tr><tr><td>Glucose-6-phosphate<br/></td><td>Sigma<br/></td><td>G7879<br/></td></tr><tr><td>Glycerol-3-phosphate<br/></td><td>Cayman<br/></td><td>20729-100 mg<br/></td></tr><tr><td>Guanidinoacetate <br/></td><td>Sigma<br/></td><td>G11608<br/></td></tr><tr><td>Kynurenine <br/></td><td>Sigma<br/></td><td>K8625<br/></td></tr><tr><td>Malate <br/></td><td>Sigma<br/></td><td>2288<br/></td></tr><tr><td>NADP+<br/></td><td>Sigma<br/></td><td>N0505<br/></td></tr><tr><td>Niacinamide <br/></td><td>Sigma<br/></td><td>72340<br/></td></tr><tr><td>2-oxoglutarate<br/></td><td>Sigma<br/></td><td>75890-25G<br/></td></tr><tr><td>Phosphoenolpyruvate <br/></td><td>Sigma<br/></td><td>P3637<br/></td></tr><tr><td>Pyruvate <br/></td><td>Sigma<br/></td><td>P5280<br/></td></tr><tr><td>Succinate <br/></td><td>Sigma<br/></td><td>S3674<br/></td></tr><tr><td>Uracil<br/></td><td>Sigma<br/></td><td>U0750<br/></td></tr></tbody></table></li><li>Chemical standard library pool 6<table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>3-hydroxyisobutyric acid<br/></td><td>Adipogen <br/></td><td>CDX-H0085-M250<br/></td></tr><tr><td>2-hydroxyglutarate<br/></td><td>Sigma<br/></td><td>H8378<br/></td></tr><tr><td>Aminoadipate <br/></td><td>Sigma<br/></td><td>A0637<br/></td></tr><tr><td>beta-alanine<br/></td><td>Sigma<br/></td><td>14064<br/></td></tr><tr><td>Carbamoylaspartate <br/></td><td>Alfa<br/></td><td>A17166-10 g<br/></td></tr><tr><td>Cystathionine <br/></td><td>Cayman <br/></td><td>16061-50 mg<br/></td></tr><tr><td>Cysteic acid<br/></td><td>Santa Cruz<br/></td><td>sc-485621<br/></td></tr><tr><td>FAD <br/></td><td>Sigma<br/></td><td>F6625<br/></td></tr><tr><td>Glycerophosphocholine <br/></td><td>Sigma<br/></td><td>G5291-50MG<br/></td></tr><tr><td>Inosine <br/></td><td>Sigma<br/></td><td>I4125<br/></td></tr><tr><td>Orotate <br/></td><td>Sigma<br/></td><td>O2875<br/></td></tr><tr><td>Pantothenate <br/></td><td>Sigma<br/></td><td>P5155<br/></td></tr><tr><td>Phosphoserine <br/></td><td>Fluka<br/></td><td>79710<br/></td></tr><tr><td>Riboflavin <br/></td><td>Sigma<br/></td><td>R9504<br/></td></tr><tr><td>UDP-GlcNAc<br/></td><td>Sigma<br/></td><td>U4375<br/></td></tr><tr><td>Uric acid<br/></td><td>Sigma<br/></td><td>U2625-25G</td></tr></tbody></table></li><li>Chemical standard library pool 7<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td><strong>Metabolite name</strong><br/></td><td><strong>Manufacturer</strong><br/></td><td><strong>Part#</strong><br/></td></tr><tr><td>Itaconic acid<br/></td><td>Sigma<br/></td><td>I29204<br/></td></tr><tr><td>Homocysteine <br/></td><td>TCI<br/></td><td>H0159<br/></td></tr><tr><td>2-oxobutyric acid<br/></td><td>Sigma<br/></td><td>K401<br/></td></tr><tr><td>2-hydroxybutyric acid<br/></td><td>Sigma<br/></td><td>220116<br/></td></tr><tr><td>Ascorbate <br/></td><td>Sigma<br/></td><td>A7506<br/></td></tr><tr><td>Sarcosine <br/></td><td>Sigma<br/></td><td>131776-100G<br/></td></tr><tr><td>Dimethylglycine <br/></td><td>Sigma<br/></td><td>D1156-5G<br/></td></tr><tr><td>N6-acetyllysine<br/></td><td>Sigma<br/></td><td>A4021-1G<br/></td></tr><tr><td>Pipecolate <br/></td><td>Sigma<br/></td><td>P45850-25G<br/></td></tr><tr><td>Indolelactate <br/></td><td>Sigma<br/></td><td>I5508-250MG-A<br/></td></tr><tr><td>Picolinate <br/></td><td>Sigma<br/></td><td>P42800-5G<br/></td></tr><tr><td>3-methyl-2-oxobutyrate<br/></td><td>Sigma<br/></td><td>198994-5G<br/></td></tr><tr><td>3-methyl-2-oxopentanoic acid<br/></td><td>Sigma<br/></td><td>198978-5G<br/></td></tr><tr><td>Formyl-methionine<br/></td><td>Sigma<br/></td><td>F3377-250MG<br/></td></tr><tr><td>2-aminobutyric acid<br/></td><td>Sigma<br/></td><td>A2536-1G<br/></td></tr><tr><td>Homocitrulline <br/></td><td>Santa Cruz<br/></td><td>sc-269298-100 mg<br/></td></tr><tr><td>gamma-glutamyl-alanine<br/></td><td>Sigma<br/></td><td>483834-500MG<br/></td></tr><tr><td>Mannose <br/></td><td>Sigma<br/></td><td>M6020-25G<br/></td></tr><tr><td>Cysteine-glycine (dipeptide)<br/></td><td>Sigma<br/></td><td>C0166-100MG<br/></td></tr></tbody></table></li><li>Mobile Phase A (see Recipes)</li><li>Mobile Phase B and Needle Wash (see Recipes)</li><li>Rear Seal Wash (see Recipes)</li><li>80% methanol containing <sup>13</sup>C-<sup>15</sup>N labeled amino acid mix (see Recipes)</li><li>Extraction Buffer with isotopically labeled internal standards (EB) (see Recipes)</li><li>Chemical standard library preparation (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30402">### Equipment</p><ol class="npwtwok"><li>Sorvall Legend X1R Refrigerated Centrifuge (Thermo Fisher, catalog number: 75004260)</li><li>Sorvall Legend Micro 21 Refrigerated Centrifuge (Thermo Fisher, catalog number: 75002447)</li><li>Mixer Mill (Retsch, catalog number: MM301)</li><li>50 ml Mixing Jar (Retsch, catalog number: 01.462.0216)</li><li>5 mM Grinding Balls (Retsch, catalog number: 05.368.0034)</li><li>LP Vortex Mixer (Thermo Fisher, catalog number: 11676331)</li><li>Analytical balance (Mettler Toledo, catalog number: AL54)</li><li>Dionex Ultimate 3000 UHPLC equipped with RS Binary Pump, RS column oven and RS autosampler (Thermo Fisher Scientific, San Jose, CA)</li><li>QExactive hybrid quadrupole-Orbitrap benchtop mass spectrometer equipped with an Ion Max ion source and HESI-II probe (Thermo Fisher Scientific, San Jose, CA)</li><li>SeQuant<sup>Â®</sup> ZIC<sup>Â®</sup>-pHILIC 5 Î¼m 150 x 2.1 mm polymeric analytical PEEK HPLC column (Millipore Sigma, catalog number: 1504600001)</li><li>SeQuant<sup>Â®</sup> ZIC<sup>Â®</sup>-pHILIC 5 Î¼m 20 x 2.1 mm PEEK Guard Kit with coupler (3 pc) (Millipore Sigma, catalog number: 15043800001)<br/></li></ol><p class="pptt" id="biaoti30403">### Software</p><ol class="npwtwok"><li>Thermo Scientific Xcalibur 4.1 SP1 (Thermo Fisher Scientific)</li><li>Microsoft Excel<br/></li></ol><p class="pptt" id="biaoti30404">### Procedure</p><ol class="npwone"><li>Study design<ol class="npwtwo"><li>How many biological replicates are recommended?<br/><ol class="npwthree"><li>The number of biological replicates needed for studies will depend on the variability between samples. For animal studies, where a large number of variables can be controlled (<em>i.e.</em>, tumor genetics, tumor size, animal genetics, animal diet, time of interstitial fluid isolation), variability will likely be smaller than for human samples. Additionally, the number of replicates required will depend on the intended purpose of the experiment to be performed. For example, if the intended purpose is to determine if there is a nutritional difference in the interstitial fluid between two tumor types, it is important to determine an effect size between the groups in addition to variability between samples in order to estimate sample sizes needed. We recommend generating pilot data or utilizing previously published data on TIF composition differences (Sullivan <em>et al.</em>, 2019) to estimate effect sizes and utilizing power analysis software in Metaboanalyst (Chong <em>et al.</em>, 2018) or other statistical analysis software to estimate the number of biological replicates needed.</li><li>Additionally, when determining the number of biological replicates needed, it is important to note that not every tumor will necessarily yield interstitial fluid. In our experience, roughly 75% of murine pancreatic adenocarcinomas yielded tumor interstitial fluid (TIF) with volumes ranging from 5 to 180 Î¼l of fluid (Sullivan <em>et al.</em>, 2019). Therefore, additional samples may be required to achieve the required number of replicates determined from power analysis.</li></ol></li><li>We recommend harvesting TIF at the same time from all animals involved in a study. Circadian rhythm and food intake can alter plasma metabolite levels and therefore TIF metabolite levels, introducing additional variability (Dallmann <em>et al.</em>, 2012; Abbondante <em>et al.</em>, 2016). If TIF must be harvested from animals on different days, we recommend harvesting TIF at the same time during the day.</li><li>We recommend two people work together to isolate TIF and cardiac blood to increase the speed of TIF harvest to prevent alterations in TIF composition due to prolonged periods of ischemia occurring between euthanasia and TIF harvest. In our own experiments, dissection was completed in ~2 min. and we found limited evidence of ischemia altering tumor metabolite levels (Sullivan <em>et al.</em>, 2019).</li><li>We have successfully isolated TIF using the protocol described in Procedure B from multiple genetically engineered and implantation mouse models of breast, lung, prostate, pancreas and melanoma cancers and others have isolated TIF from murine models of melanoma and breast cancer (Ho <em>et al.</em>, 2015; Eil et al., 2016; Spinelli <em>et al.</em>, 2017; Zhang <em>et al.</em>, 2017) using similar protocols. Additionally, similar protocols have been used to isolate TIF from renal (Siska <em>et al.</em>, 2017) and ovarian carcinomas (Haslene-Hox <em>et al.</em>, 2011). Thus, we anticipate the TIF isolation protocol can be used successfully for a variety of tumor types, of mouse and human origin.</li><li>The analysis described in Procedure C uses 7 separate chemical standard pools as described in (Sullivan <em>et al.</em>, 2019) that enable the quantification of 149 metabolites commonly measured in biofluids (Lawton <em>et al.</em>, 2008; Evans <em>et al.</em>, 2009; Mazzone <em>et al.</em>, 2016; Cantor <em>et al.</em>, 2017). However, depending on experimental goals, the full analysis utilizing all 7 standard pools may not be required. Subsets of the chemical standard pools can be used that cover analytes of interest if the full analysis is not needed. Note though that individual metabolites in the standard pools provided in this protocol have been carefully selected so as to avoid metabolites with the same m/z (isomeric and isobaric species) being in the same pool. In addition, metabolites that could be generated by in-source fragmentation from larger metabolites have been separated.</li><li>The description of the liquid chromatography-mass spectrometry analysis in this protocol (Procedure D) is a rough guideline for experienced operators of such instruments to perform the analysis described. Successful mass spectrometry analysis of the samples will require a trained UHPLC and Thermo Scientific hybrid quadrupole-Oribtrap mass spectrometer operator.<br/><br/></li></ol></li><li>Isolation of TIF and plasma from tumor bearing animals<ol class="npwtwo"><li>Prepare a TIF isolation tube (Figure 1 A).<br/><ol class="npwthree"><li>Take a nylon filter and place it over the top of a 50 ml conical tube.</li><li>Tape the filter down using lab tape. Make sure the filter is affixed somewhat loosely to the top of the conical tube, such that the tumor can push the filter down slightly into the tube.<br/><br/><img alt="" class="layerphoto" height="137" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191117/20191117221207_6240.jpg" width="300"/><br/><strong>Figure 1. Collecting TIF using nylon mesh filters attached to conical tubes.</strong> A. The filter is loosely affixed to the top of the conical tube using laboratory tape, so that the filter will sag slightly into the conical tube when a sample is placed on top of the filter. B. A sample on top of the filter and conical tube. C. After adding the sample to the filter, the lid of the conical tube is placed on top, but not screwed onto the conical. Instead, it is taped using laboratory tape in place. D. ~30 Î¼l of tumor interstitial fluid (colored blue to here for contrast) collected in the conical tube after centrifugation.<br/><br/></li></ol></li><li>Prepare materials in advance to allow for rapid mouse dissection.<br/><ol class="npwthree"><li>Put pre-chilled (4 Â°C) saline (~25 ml) into a Petri dish for washing the tumor.</li><li>Make a ~4 cm square piece of Whatman paper for drying the tumors after the saline rinse.</li><li>Have a 25G TB syringe ready for cardiac blood collection.</li><li>Label and pre-chill one EDTA-coated plasma collection tubes on ice for 10 min prior to mouse dissection.</li></ol></li><li>Euthanize the mouse by cervical dislocation.</li><li>Spray around the incision site with 70% ethanol to prevent contamination of samples with hair.</li><li>Quickly and cleanly dissect the tumor away from the animal. The exact procedure for the dissection will depend on the anatomical location of the tumor. One person should continue with the following steps while the other person can start on plasma isolation (Step B6).<br/><ol class="npwthree"><li>Place the tumor into the saline containing Petri dish to rinse the tumor.</li><li>Blot the tumor dry on Whatman paper. Take care at this step to ensure all saline is blotted from the tumor, so that saline does not contaminate the TIF.<br/>If concerned about saline contamination during TIF isolation, we suggest the following additional step: add a non-natural metabolite such as norvaline to the saline. Continue with the remainder of the protocol as normal. Subsequently, when analyzing TIF samples, determine if the non-natural metabolite is present in the TIF sample. If the non-natural metabolite is detected this indicates saline contamination, whereas if the metabolite is not detected saline contamination is unlikely.<br/></li><li>Optional: Weigh the tumor if this information is needed. Tumor volume can also be determined by caliper measurements if needed.</li><li>Place the tumor on top of the nylon mesh filter that is affixed to the conical tube (Figure 1B). Place the 50 ml conical lid over the tumor and tape the lid in place (Figure 1C).<br/></li><li>Centrifuge the tumor at 4 Â°C for 10 min at 106<span style="font-size:7.5px;"> </span><em>x g</em> using a refrigerated clinical centrifuge (<em>e.g.</em>, Sorvall Legend X1R).<br/></li><li>Remove the 50 ml conical tube. If the isolation worked, there will be 10-50 Î¼l of fluid in the bottom of the conical tube (Figure 1D). Keep the tube on ice.<br/><em>Note: Human tumors have been found to yield 5-150 Î¼l of fluid per gram of tumor (Haslene-Hox et al., 2011). Similarly, we isolated 10-50 Î¼l of fluid from murine pancreatic tumors weighing ~0.3-2.5 g (Sullivan et al., 2019). There is not an exact correlation between tumor size and TIF volume, as many factors likely influence interstitial volume. However, larger tumors are more likely to yield TIF in larger volumes. Tumors with large fluid filled cysts can give hundreds of Î¼l of fluid per gram of tumor. Disregard these from analysis as it is unclear if the cystic fluid is representative of interstitial fluid.</em> </li><li>Remove the isolate fluid to a fresh Eppendorf tube. The fluid can be extracted directly for LC/MS analysis or frozen and stored at -80 Â°C for future analysis.<br/>We have analyzed TIF samples both before and after 2 months of storage at -80 Â°C (avoiding freeze-thaw cycles) and detected similar metabolite concentrations after storage (Sullivan <em>et al.</em>, 2019). Thus, TIF samples can be stored for at least 2 months without freeze-thaw cycles prior to analysis.</li><li>The tumor from which TIF was isolated can be used for additional analysis using other appropriate protocols.<br/><em>Note: We have successfully used tumors from which TIF was isolated for immunohistochemical, immunoblotting and flow cytometric analyses.</em> </li></ol></li><li>Use the 25G TB syringe to isolate blood from the mouse heart by cardiac puncture. Cardiac puncture can be a difficult technique to perform. Detailed protocols with video documentation of cardiac puncture blood isolation have been previously published (Schroeder, 2019). If unfamiliar with this technique, we recommend utilizing these resources for more detailed information on isolating blood in this manner.<br/><ol class="npwthree"><li>Dissect open the thoracic cavity.</li><li>Insert the syringe into the ventricle.</li><li>Slowly withdraw blood to prevent collapse of the heart.</li><li>Remove the needle from the syringe to prevent cell lysis when expelling the cells from the syringe.<br/></li><li>Expel the blood into the EDTA-coated plasma collection tube.<br/></li><li>Keep plasma on ice.<br/></li><li>Spin the EDTA-coated plasma collection tubes at 845 <em>x g</em> in benchtop centrifuge (<em>e.g.</em>, Sorvall Legend Micro 21) for 15 min at 4 Â°C.<br/></li><li>Remove the plasma from the pelleted blood cells and put into fresh Eppendorf tube.<br/></li><li>Extract this plasma directly for LC/MS analysis or freeze and store at -80 Â°C for future analysis.<br/><em>Note: Previous studies have found that plasma samples can be stored at -80 Â°C (without freeze-thaw cycles) for up to 30 months without significant alterations in the levels of many metabolites (Stevens </em><em>et al.</em><em>, 2019). Thus, plasma samples can be stored for many months prior to analysis.</em><br/><br/></li></ol></li></ol></li><li>Extraction of metabolites from TIF and plasma<ol class="npwtwo"><li>Prepare libraries of pooled chemical standards that include the metabolites to be quantified. See Recipes section and Tables 5-11 for details on how libraries were compounded in (Sullivan <em>et al.</em>, 2019).</li><li>Prepare metabolite extraction buffer (EB) (Recipe 5) with appropriate isotopically labeled internal standards. See Recipes section for details on making EB with isotopic standards as described in (Sullivan <em>et al.</em>, 2019).<br/><em>Note: Make enough EB for the number of samples and standards you have plus an additional 10%, so as not to run out of EB before extracting metabolites from every sample. 45 Î¼l of EB is needed for each sample and standard pool dilution. Isotopically labeled metabolite standards in the EB are not indefinitely stable. Prepare EB fresh prior to each experiment.</em> </li><li>Prepare dilutions of chemical standard libraries in HPLC grade water. The highest concentration should be 5 mM.</li><li>Next, make dilution series of each standard pool in HPLC grade water as listed below in Step C5. Keep these on ice prior to extraction. Note that multiple dilutions of the chemical standard libraries are required for construction of standard curves relating known metabolite concentration to LC/MS response, which is necessary for downstream analysis of metabolite concentrations. Below we recommend a scheme to generate 8-point standard curves covering physiological concentrations of metabolites, but variations are possible.</li><li>The 5 mM pool will not be always in solution for each pool, therefore vortex vigorously and immediately pipette from this mixture in order to prevent error from settling particles:<br/><ol class="npwthree"><li>Take 20 Î¼l of 5 mM stock and dilute into 80 Î¼l HPLC grade water to yield a 1 mM solution.</li><li>Take 30 Î¼l of 1 mM stock and dilute into 70 Î¼l HPLC grade water to yield a 300 Î¼M solution.</li><li>Take 33.33 Î¼l of 300 Î¼M stock and dilute into 66.67 Î¼l HPLC grade water to yield a 100 Î¼M solution.</li><li>Take 30 Î¼l of 100 ÂµM stock and dilute into 70 Î¼l HPLC grade water to yield a 30 Î¼M solution.<br/></li><li>Take 33.33 Î¼l of 30 Î¼M stock and dilute into 66.67 Î¼l HPLC grade water to yield a 10 Î¼M solution.<br/></li><li>Take 30 Î¼l of 10 Î¼M stock and dilute into 70 Î¼l HPLC grade water to yield a 3 ÂµM solution.<br/></li><li>Take 33.33 Î¼l of 3 Î¼M stock and dilute into 66.67 Î¼l HPLC grade water to yield a 1 Î¼M solution.</li></ol></li><li>Thaw the TIF and plasma samples on ice.</li><li>Add 5 Î¼l of each TIF sample, plasma sample and chemical standard library dilution (Recipe 6) to a fresh Eppendorf tube. Keep on ice.</li><li>Add 45 Î¼l of EB to each sample/standard. Keep on ice.</li><li>Vortex all the samples for 10 min at maximum speed at 4 Â°C.</li><li>Spin down all samples for 10 min at 21,000 <em>x g</em> at 4 Â°C.</li><li>Take 20 Î¼l of the mixtures from the Eppendorf tube and add to an LC/MS sample vial. Cap the vial. <br/><em>Note: A minimum of 15 Î¼l is needed in the LC/MS vial to ensure correct and accurate injection by the autosampler. The vials described in Materials and Reagents contain fused inserts. Vials without inserts will require larger volumes. </em> </li><li>Freeze the remaining sample in the Eppendorf tubes and store at -80 Â°C. This sample can be run later if the initial LC/MS is not successful.<br/><br/></li></ol></li><li>Liquid chromatography-mass spectrometry analysis of extracted metabolites<ol class="npwtwo"><li>Start off with a clean system (use appropriate LC cleaning methods in place in your lab).</li><li>Calculate the amount of Mobile Phase A (Recipe 1) required and prepare fresh on the day of analysis. Store this for no more than one week.<br/><em>Note: Depending on your system, you will use ~2 ml per injection and require an additional 50-100 ml in the bottle. Do not forget to make enough Mobile Phase A for additional injection types such as solvent blanks and system suitability tests that must be run in addition to the samples.</em> </li><li>Calculate the amount of Mobile Phase B (Recipe 2) required and prepare more if needed. Depending on your system, you will need ~2 ml per injection plus an additional 50-100 ml in the bottle.</li><li>Check the level of rear seal wash (Recipe 3) and top up if needed.</li><li>If using an UHPLC system that has a separate needle wash, fill this with acetonitrile.</li><li>Connect a SeQuant<sup>Â®</sup> ZIC<sup>Â®</sup>-pHILIC 5 Î¼m 150 x 2.1 mm analytical column to the Guard column using the connector supplied in the Guard kit.</li><li>Connect the column and guard to your UHPLC system using standard techniques.</li><li>Set the column oven temperature to 25 Â°C. </li><li>Set the autosampler temperature to 4 Â°C.</li><li>Set initial conditions: set the flow rate to 0.150 ml/min with 80% B. Record initial pressure value. <br/><em>Note: ZIC-pHILIC columns cannot tolerate such high back pressures and injection volumes as typical reverse phase columns. Keep an eye on the back pressure and do not let it exceed the maximum pressure recommended by the manufacturer. It is good practice to set a maximum pressure in your method that is below that set by the manufacturer to avoid damage to the column.</em> </li><li>Equilibrate the column with starting conditions (80% B) for 30 min prior to running anything on the system. </li><li>Check the mass calibration on the mass spectrometer. If the mass has not been calibrated within the last week, or if it fails the mass check, recalibrate using the standard calibration mixes recommended by the manufacturer. In addition, perform a custom low-mass calibration by spiking glycine and aspartate into the calibration mix, or as recommended by the manufacturer. </li><li>Ensure your entire LCMS system performance is acceptable by running system suitability tests, such as injecting a mixture of amino acids onto your column and into the MS. Check for signal intensity as well as peak shape and separation.</li><li>Use the conditions shown in Table 1 below for the UHPLC gradient:<br/><br/><strong>Table 1. LC parameters</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Time (min)</strong><br/></td><td><strong>Flow rate (ml/min)</strong><br/></td><td><strong>%B</strong><br/></td></tr><tr><td>0.00<br/></td><td>0.150<br/></td><td>80<br/></td></tr><tr><td>20.0<br/></td><td>0.150<br/></td><td>20<br/></td></tr><tr><td>20.5<br/></td><td>0.150<br/></td><td>80<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>28.0 <br/></td><td>0.150<br/></td><td>80<br/></td></tr></tbody></table><br/></li><li>Operate the mass spectrometer in full-scan, polarity-switching mode, with a scan range of 70-1000 m/z. Include an additional narrow-range scan from 220 to 700 m/z in negative mode to improve detection of nucleotides. Use the parameters shown below in Tables 2 and 3 for the MS:<br/><br/><strong>Table 2. MS source parameters </strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Parameter</strong><br/></td><td><strong>Setting</strong><br/></td></tr><tr><td>Spray voltage<br/></td><td>3.0 kV (pos); 3.1 kV (neg)<br/></td></tr><tr><td>Heated capillary<br/></td><td>275 Â°C<br/></td></tr><tr><td>HESI probe<br/></td><td>350 Â°C<br/></td></tr><tr><td>Sheath gas<br/></td><td>40 units<br/></td></tr><tr><td>Aux gas <br/></td><td>15 units<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Sweep gas<br/></td><td>1 unit<br/></td></tr></tbody></table><br/><strong>Table 3. MS scan parameters</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Parameter </strong><br/></td><td><strong>Setting</strong><br/></td></tr><tr><td>Resolution <br/></td><td>70,000<br/></td></tr><tr><td>AGC target<br/></td><td>1E6<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Max IT<br/></td><td>20 ms<br/></td></tr></tbody></table><br/></li><li>Note that in this particular study, we had previously collected MS/MS data for each metabolite being quantified, and used this to confirm retention times using a library of chemical standards. If adding new metabolites to your standard pools, collect MS/MS data on the standard itself, as well as on a pooled biological sample to help confirm peak identification. </li><li>Write your sequence (sample run order) using Thermo Scientific Xcalibur Sequence Setup View.<br/><em>Note: Given the extremely low volume of samples used in this method, the sequence differs from typical sequences which will include column conditioning injections, as well as quality control pooled samples. As multiple standard curves are run and each sample includes <sup>13</sup>C-labeled internal standards, we chose to forgo using precious sample to create QC pools and instead determine linearity and consistency of metabolite detection using the standard curves and the <sup>13</sup>C internal standards.</em><br/><ol class="npwthree"><li>Start off by injecting several water blanks to ensure system is clean from carry over and contaminants.</li><li>Include solvent blanks, using the 75/25/0.1 acetonitrile/methanol/formic acid mix that was used to make the extraction mix.</li><li>Follow with a system suitability test (SST) injection. We use 80% methanol containing <sup>13</sup>C-<sup>15</sup>N labeled amino acid mix (Recipe 4).</li><li>Add the samples to your sequence and follow with the standard curves, starting with the lowest concentration and working up to the highest concentration for each curve. Separate each curve with solvent blank and check for carry-over.<br/></li><li>Insert additional SST injections every 8-10 samples. These will be used as QCs to ensure no loss of signal over time.<br/></li><li>Set the injection volume to 2 Î¼l for each injection type.<br/></li><li>Set the instrument method to the appropriate method.<br/></li><li>Save the sequence file.<br/></li><li>Randomize sample running order to decrease the chance of signal loss over time. Export the sequence as a .csv file. Open in Microsoft Excel, cut and paste the samples into a new tab, leaving behind the blanks, STTs and standard curves. Add an additional column and use the = rand () function to create random numbers for each of the samples. Now sort the samples from smallest to largest using the random number values. Cut and paste back into the previous sequence containing the blanks, SSTs and the standard curves. Save the .csv file with ârandomâ in the file name. Import the new randomized sequence back into Xcalibur Sequence View and save using ârandomâ in the file name.<br/></li><li>Place your sample vials in the autosampler in the vial positions according to the sequence. Use the <em>non-randomized</em> sequence to check vial positions for your samples.</li><li>Ensure the solvent blank vials and SST vials contain enough volume for multiple injections. <br/></li></ol></li><li>Run the sequence. <br/><em>Note: For examples of expected outputs from the LC/MS analysis of TIF and plasma samples, LC/MS data from (Sullivan et al., 2019) is available at </em><a href="https://www.metabolomicsworkbench.org/data/DRCCMetadata.php?Mode=Project&amp;ProjectID=PR000750." target="_blank"><em>https://www.metabolomicsworkbench.org/data/DRCCMetadata.php?Mode=Project&amp;ProjectID=PR000750</em><em>.</em></a><br/></li></ol></li></ol><p class="pptt" id="biaoti30405">### Data analysis</p><ol class="npwone"><li>Identify metabolite peaks. This protocol will describe peak identification in Thermo Scientific Xcalibur, but could be adapted with any other peak identification method.<ol class="npwtwo"><li>Generate a processing method file that will be used to identify peaks for each metabolite of interest:<br/><ol class="npwthree"><li>Create a new processing method.</li><li>Load a .raw file containing LC/MS data derived from an external standard sample that contains the metabolite of interest as well as a <sup>13</sup>C internal standard for that metabolite.<br/><em>Note: Typically using an external standard sample that is in the middle of the standard curve works best; sometimes high concentrations points on the standard curve have poor quality peaks.</em><br/></li><li>Calculate the exact mass of the metabolite of interest.<br/><em>Notes: </em><br/><ol class="npwfour"><li><em>Exact mass is determined by summing the masses of the most abundant isotopes of each element in a compound. For instance, the exact mass of CO<sub>2</sub> would be the summed masses of a carbon-12 atom (12.000) + two oxygen-16 atoms (15.995 + 15.995) = 43.990. This exact mass of a compound will differ from its molecular weight; the molecular weight of an element is derived by averaging the masses of each of the isotopes of that element, weighted by the abundance of each isotope in nature.</em> </li><li><em>If using Thermo Scientific Xcalibur, calculate the exact mass using the Isotope Simulation tab in the QualBrowser module. Enter the chemical formula, ensure the âadductâ check box in unchecked and select âNewâ. Ensure that the software global settings are set to a mass tolerance of 5 ppm and mass precision is set to 5 decimal places. Ideally, the peak integration software that you are using should calculate this for you.</em> </li></ol></li><li>Calculate the mass to charge ratio (m/z) of the metabolite of interest in positive and negative ionization mode.<br/><em>Notes: </em><br/><ol class="npwfour"><li><em>For analysis of small polar metabolites, the most common ions will be those that have gained or lost a single proton and most molecules will have a charge state of 1.</em> </li><li><em>If using Thermo Scientific Xcalibur, calculate the m/z using the Isotope Simulator in the QualBrowser module by entering the formula, checking the âadductâ box and selecting a charge of either +1 or -1, depending on whether you are calculating the m/z in positive or negative mode, respectively.</em> </li><li><em>There are a variety of online tools available that will provide exact mass information, as well as calculate m/z for a variety of different adducts. The most comprehensive is the Metlin data base (Guijas et al., 2018): </em><a href="https://metlin.scripps.edu/landing_page.php?pgcontent=mainPage" target="_blank"><em>https://metlin.scripps.edu/landing_page.php?pgcontent=mainPage.</em></a> </li></ol></li><li>In the processing method, select either positive ionization mode or negative ionization mode depending on whether you will be searching for a positively charged ion or a negatively charged ion.<br/><em>Note: Some metabolites are more easily detected in positive or negative mode. A list of recommendations for which mode to use for a variety of metabolites is located in Supplementary File 1 of (Sullivan et al., 2019). If no recommendations are available, empirically determine which method gives better detection by trying both.</em> </li><li>Identify and validate the retention time of each metabolite:<br/><ol class="npwfour"><li>Search for the exact mass of the ion of interest.</li><li>Note the retention time of any peaks that match the exact mass of the ion of interest within 5 ppm.</li><li>Open a .raw file of a different external standard sample with the metabolite of interest at a lower concentration.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:5px;"><div class="float-left">1)</div><div class="media-body">Note which peaks that match the exact mass of the ion of interest decrease in area.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:5px;"><div class="float-left">2)</div><div class="media-body">Repeat with each of the external standard samples that contain the metabolite of interest, checking which peak areas track with the expected amount of the metabolite.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:5px;"><div class="float-left">3)</div><div class="media-body">Refer to MS/MS data to confirm peak identification.</div></div></li><li>Open a .raw file that does not contain the metabolite of interest. Ensure that any candidate peaks are not present in this .raw file.<br/></li><li>Search for the exact mass of the <sup>13</sup>C labeled version of the ion of interest.<br/><em>Note: This peak should be approximately the same area in all samples.</em> </li><li>Check that the retention time of the <sup>13</sup>C labeled standard peak exactly matches that of the candidate peak.</li><li>Repeat for all metabolites of interest.<br/></li></ol></li><li>Assign <sup>13</sup>C labeled standards as internal standards for their corresponding <sup>12</sup>C metabolites.<br/>For metabolites with no <sup>13</sup>C internal standard, assign a <sup>13</sup>C metabolite with a similar retention time as the internal standard.</li></ol></li><li>Use the processing method to pick and validate peaks for all metabolites in all LC/MS data files (both experimental samples and external standards):<br/><ol class="npwthree"><li>Once all peaks have been automatically picked, manually inspect every peak for each metabolite and for each sample to ensure that all peaks have been correctly identified. Some common examples of errors that occur with automatic peak picking algorithms:<br/><ol class="npwfour"><li>Incorrect peak was picked: this can occur for isobaric compounds with similar retention times, such as leucine and isoleucine.</li><li>Peak was not fully picked from baseline to baseline.</li><li>Peak was picked but overlaps with a second peak: this occasionally happens where biological samples have an overlapping peak that was not present in the external standards. If this is the case, this metabolite should not be quantitated using this LC/MS method and an alternative method of chromatographic separation should be identified.</li></ol></li><li>Export the ratio of peak areas for the sample versus the <sup>13</sup>C internal standard to Microsoft Excel or your data processing software of choice.<br/><br/></li></ol></li></ol></li><li>Determine the relationship between relative peak area and concentration of metabolite in external standards.<ol class="npwtwo"><li>Calculate the exact concentration of each metabolite in each point on the external standard curve based on the amount that was weighed out.</li><li>Generate a graph of metabolite concentration in each external standard sample versus the relative peak area of the metabolite.</li><li>Check if the relative peak area of the metabolite increases linearly with concentration:<br/><ol class="npwthree"><li>Fit a linear regression to the graph.<br/></li><li>The R<sup>2</sup> value for the linear regression should be â¥ 0.995. <br/>Metabolites often respond non-linearly at high concentrations. If the standard curve has points that are much higher than the concentrations present in experimental samples, the highest points on the standard curve can be removed. Just ensure that the relative peak areas for all samples fall within the linear range of the standard curve.<br/></li><li>Non-linear metabolites should be excluded from quantitative analysis, as this lack of linearity will prevent accurate quantitation by isotope dilution.</li></ol><br/></li></ol></li><li>Determine the concentrations of the internal standards that were added to all samples.<br/>Solve for the concentration of <sup>13</sup>C internal standard present in each external standard sample using the following relationship:<br/><br/><img alt="" class="layerphoto" height="55" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191117/20191117224605_2092.png" width="600"/><br/><br/><em>Note: This relationship can be used to calculate the concentration of the <sup>13</sup>C internal standard in each of the external standard samples; the same concentration should be present in each. To derive the most accurate value for the concentration of the <sup>13</sup>C internal standard, average the concentrations derived from the external standard points most similar in concentration to the experimental samples.</em><br/><br/></li><li>Calculate the concentration of each metabolite in the experimental samples by isotope dilution. Solve for the concentration of the <sup>12</sup>C metabolite using the same relationship defined in step C.<br/><br/></li><li>Calculate the semi-quantitative concentration of all other analytes using the external standard curves. These values are considered semi-quantitative as they are subject to matrix effects arising from the biological samples being compared to external standards dissolved in water. These matrix effects can be substantial (Sullivan <em>et al.</em>, 2019).<br/><ol class="npwtwo"><li>Calculate the slope and intercept of the linear regression calculated in step B3a.</li><li>Use this slope and intercept to calculate the semi-quantitative value approximating the concentration of the metabolite.</li><li>Manually evaluate the concentrations derived from this calculation:<br/><ol class="npwthree"><li>Check that the value of each metabolite is zero in the external standard samples that do not contain that metabolite.</li><li>Any data that shows a negative concentration should be removed.</li></ol><br/></li></ol></li><li>Perform statistical analysis of the data using Metaboanalyst (<a href="https://www.metaboanalyst.ca/MetaboAnalyst/faces/home.xhtml" target="_blank">https://www.metaboanalyst.ca/MetaboAnalyst/faces/home.xhtml</a>) (Chong <em>et al.</em>, 2018) or another program for statistical analysis.<ol class="npwtwo"><li>Auto-scale the data (mean-center and divide by the standard deviation of each concentration).</li><li>To broadly compare if there are differences in the metabolites present in two sample types, perform principal component analysis or hierarchical clustering.</li><li>To identify specific metabolites that differ in concentration between sample types, generate a volcano plot in which a raw <em>P</em>-value of 0.01 and a fold change of 1.5 are used to identify significantly altered metabolites.<br/></li></ol></li></ol><p class="pptt" id="biaoti30406">### Recipes</p><ol class="npwtwok"><li>Mobile Phase A<br/>20 mM ammonium carbonate<br/>0.1% ammonium hydroxide (pH 9.4-9.6)<br/>Optima LC/MS water</li><li>Mobile Phase B and Needle Wash<br/>Optima LC/MS acetonitrile</li><li>Rear Seal Wash<br/>10% Optima LC/MS methanol<br/>Optima LC/MS water</li><li>80% methanol (containing <sup>13</sup>C-<sup>15</sup>N amino acid standard mix) (200 ml)<br/>160 ml Optima LC/MS methanol<br/>40 ml Optima LC/MS water<br/>40 Î¼l Metabolomics Amino Acid Standard Mix</li><li>Extraction Buffer with isotopically labeled internal standards (EB) (Table 4)<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>This is the recipe containing isotopically labeled internal standards for analysis of metabolites as in (Sullivan et al., 2019). If analysis of other metabolites using stable isotope internal standards is desired, purchase or synthesize the desired isotopically labeled metabolite and add it to the Extraction Buffer. Quantification with isotopically labeled standards performs best when the abundance of the isotopically labeled metabolite is similar to the unlabeled metabolite to be quantified. Therefore, when adding isotopically labeled internal standards, add the isotope such that it will be at roughly a similar abundance as the unlabeled metabolite when the sample is diluted in the Extraction Buffer.</em> </li><li><em>This recipe is for 180 samples at 45 Î¼l per sample for a total of 8,100 Î¼l in volume. Adjust the volumes accordingly as needed for the number of samples you intend to analyze. After adding all components, vortex briefly to ensure EB is well mixed, and store on ice while in use. Make fresh prior to each experiment. Remember to include extra 75/25/0.1 acetonitrile/methanol/formic acid for use as solvent blanks in your calculations.</em><br/><br/><strong>Table 4. Extraction Buffer (EB) composition</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Component</strong><br/></td><td><strong>Volume added   </strong><br/></td><td><strong>Final concentration</strong><br/></td></tr><tr><td>HPLC grade acetonitrile<br/></td><td>5771.25 Î¼l<br/></td><td>71.25%<br/></td></tr><tr><td>HPLC grade methanol<br/></td><td>1923.75 Î¼l<br/></td><td>23.75%<br/></td></tr><tr><td>HPLC grade formic acid<br/></td><td>15.39 Î¼l<br/></td><td>1.9%<br/></td></tr><tr><td>~15 mg of isotopically labeled yeast extract (ISO1) dissolved in 1.5 ml of HPLC grade water.<br/><em>Note: After adding water to the yeast extract, dissolve the yeast extract by vortexing and/or <br/>rocking the yeast extract and water at 4 Â°C for approximately 30 min. Solution can be stored <br/>at -80 Â°C although some metabolites will degrade over time (see manufacturerâs instructions)</em><br/></td><td>405 Î¼l<br/></td><td>5%<br/></td></tr><tr><td>2 mM solution of <sup>2</sup>H<sub>9</sub> choline prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>4.03 Î¼l<br/></td><td>1 Î¼M<br/></td></tr><tr><td>50 mM solution of <sup>13</sup>C<sub>4</sub> 3-hydroxybutyrate prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>0.81 Î¼l<br/></td><td>5 Î¼M<br/></td></tr><tr><td>200 Î¼M solution of <sup>13</sup>C<sub>6</sub> <sup>15</sup>N<sub>2</sub> cystine prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>81 Î¼l<br/></td><td>2 Î¼M<br/></td></tr><tr><td>100 mM solution of <sup>13</sup>C<sub>3</sub> lactate prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>16.2 Î¼l<br/></td><td>200 Î¼M<br/></td></tr><tr><td>57.3 mM solution of <sup>13</sup>C<sub>6</sub> glucose prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>7.05 Î¼l<br/></td><td>50 Î¼M<br/></td></tr><tr><td>100 mM solution of <sup>13</sup>C<sub>3</sub> serine prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>1.62 Î¼l<br/></td><td>20 Î¼M<br/></td></tr><tr><td>750 mM solution of <sup>13</sup>C<sub>2</sub> glycine prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>1.62 Î¼l<br/></td><td>150 Î¼M<br/></td></tr><tr><td>2 mM solution of <sup>13</sup>C<sub>5</sub> hypoxanthine prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>2.02 Î¼l<br/></td><td>0.5 Î¼M<br/></td></tr><tr><td>200 mM solution of <sup>13</sup>C<sub>2</sub> <sup>15</sup>N taurine prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>2.02 Î¼l<br/></td><td>50 Î¼M<br/></td></tr><tr><td>60 mM solution of <sup>13</sup>C<sub>3</sub> glycerol prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>2.02 Î¼l<br/></td><td>15 Î¼M<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>4 mM solution of <sup>2</sup>H<sub>3</sub> creatinine prepared in HPLC grade water (stored at -20 Â°C)<br/></td><td>2.02 Î¼l<br/></td><td>1 Î¼M</td></tr></tbody></table><br/></li></ol></li><li>Chemical standard library preparation<br/>Below are recipes for the preparation of chemical standard libraries described in (Sullivan <em>et al.</em>, 2019). To prepare these chemical libraries, purchase the chemicals from the listed supplier and weigh them out as indicated, placing each metabolite into a 50 ml mixing mill jar. Mix the combined metabolites using a Mixer Mill MM301 with five 5 mm diameter stainless steel grinding balls. Perform 6 cycles of 1 min mixing at 25 Hz followed by 3 min resting. Store the now mixed chemical standard library powder stocks at -20 Â°C prior to use. For use, resuspend each mixed chemical library in HPLC grade water at 5 mM concentration as indicated for each library below.<br/>Custom chemical standard libraries can be produced by acquiring desired pure chemical standards and mixing the pure chemical standards in equimolar amounts. When generating libraries, it is important to ensure that each library will not contain metabolites that have the same exact mass, as it is not then possible to determine the correct retention time for both metabolites when compounded into the same library. Consider putting these metabolites into separate pooled libraries (Tables 5-11).<br/><br/><strong>Table 5. Chemical standard library pool 1</strong><br/><span id="__kindeditor_bookmark_start_148__" style="display:none;"></span> <table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:980px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong><strong style='font-family:"font-size:12px;white-space:normal;'>Metabolite name</strong><span id="__kindeditor_bookmark_end_149__" style="display:none;"></span></strong><br/></td><td><strong>Molecular weight of metabolite</strong><strong></strong><br/></td><td><strong>Molecular weight of chemical standard</strong><strong></strong><br/></td><td><strong>Amount to weigh (mg)</strong><strong></strong><br/></td></tr><tr><td>Alanine<br/></td><td>89.09<br/></td><td>89.09<br/></td><td>429.99<br/></td></tr><tr><td>Arginine <br/></td><td>174.2<br/></td><td>210.66<br/></td><td>1016.75<br/></td></tr><tr><td>Asparagine <br/></td><td>132.12<br/></td><td>150.13<br/></td><td>724.60<br/></td></tr><tr><td>Aspartate <br/></td><td>133.11<br/></td><td>133.11<br/></td><td>642.45<br/></td></tr><tr><td>Carnitine <br/></td><td>161.199<br/></td><td>197.66<br/></td><td>954.00<br/></td></tr><tr><td>Citrulline <br/></td><td>175.2<br/></td><td>175.2<br/></td><td>845.60<br/></td></tr><tr><td>Cystine <br/></td><td>240.3<br/></td><td>240.3<br/></td><td>1159.80<br/></td></tr><tr><td>Glutamate <br/></td><td>147.13<br/></td><td>147.13<br/></td><td>710.12<br/></td></tr><tr><td>Glutamine <br/></td><td>146.14<br/></td><td>146.14<br/></td><td>705.34<br/></td></tr><tr><td>Glycine <br/></td><td>75.066<br/></td><td>75.066<br/></td><td>362.30<br/></td></tr><tr><td>Histidine <br/></td><td>155.1546 <br/></td><td>155.1546 <br/></td><td>748.85<br/></td></tr><tr><td>Hydroxyproline <br/></td><td>131.13<br/></td><td>131.13<br/></td><td>632.89<br/></td></tr><tr><td>Isoleucine <br/></td><td>131.1729 <br/></td><td>131.1729 <br/></td><td>633.10<br/></td></tr><tr><td>Leucine <br/></td><td>131.17<br/></td><td>131.1729<br/></td><td>633.10<br/></td></tr><tr><td>Lysine <br/></td><td>146.19<br/></td><td>182.65<br/></td><td>881.56<br/></td></tr><tr><td>Methionine <br/></td><td>149.21<br/></td><td>149.21<br/></td><td>720.16<br/></td></tr><tr><td>Ornithine <br/></td><td>132.16<br/></td><td>168.62 <br/></td><td>813.84<br/></td></tr><tr><td>Phenylalanine <br/></td><td>165.19 <br/></td><td>165.19 <br/></td><td>797.28<br/></td></tr><tr><td>Proline <br/></td><td>115.13<br/></td><td>115.13<br/></td><td>555.67<br/></td></tr><tr><td>Serine <br/></td><td>105.09 <br/></td><td>105.09 <br/></td><td>507.21<br/></td></tr><tr><td>Taurine <br/></td><td>125.15<br/></td><td>125.15<br/></td><td>604.03<br/></td></tr><tr><td>Threonine <br/></td><td>119.119<br/></td><td>119.119<br/></td><td>574.92<br/></td></tr><tr><td>Tryptophan <br/></td><td>204.225<br/></td><td>204.225<br/></td><td>985.69<br/></td></tr><tr><td>Tyrosine <br/></td><td>181.19<br/></td><td>181.19<br/></td><td>874.51<br/></td></tr><tr><td>Valine <br/></td><td>117.151<br/></td><td>117.151<br/></td><td>565.42<br/></td></tr><tr><td>Lactate<br/></td><td>90.09<br/></td><td>112.06<br/></td><td>540.85<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Glucose <br/></td><td>180.1559<br/></td><td>180.1559<br/></td><td>869.52<br/></td></tr></tbody></table><em>Note: Dissolve this pool at 20.19 mg/ml for 5 mM solution.</em><br/><br/><strong>Table 6. Chemical standard library pool 2</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:980px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Metabolite name</strong><br/></td><td><strong>Molecular weight of metabolite</strong><br/></td><td><strong>Molecular weight of chemical standard</strong><br/></td><td><strong>Amount to weigh (mg)</strong><br/></td></tr><tr><td>2-hydroxybutyric acid<br/></td><td>104.1<br/></td><td>126.09<br/></td><td>12.61<br/></td></tr><tr><td>2-aminobutyric acid<br/></td><td>103.12<br/></td><td>103.12<br/></td><td>10.31<br/></td></tr><tr><td>AMP <br/></td><td>347.2212<br/></td><td>347.22<br/></td><td>34.72<br/></td></tr><tr><td>Argininosuccinate <br/></td><td>290.273<br/></td><td>334.24<br/></td><td>33.42<br/></td></tr><tr><td>Betaine <br/></td><td>117.1463 <br/></td><td>117.15<br/></td><td>11.71<br/></td></tr><tr><td>Biotin <br/></td><td>244.31<br/></td><td>244.31<br/></td><td>24.43<br/></td></tr><tr><td>Carnosine <br/></td><td>226.2324<br/></td><td>226.23<br/></td><td>22.62<br/></td></tr><tr><td>Choline <br/></td><td>104.1708<br/></td><td>139.62<br/></td><td>13.96<br/></td></tr><tr><td>CMP <br/></td><td>323.1965<br/></td><td>367.16<br/></td><td>36.72<br/></td></tr><tr><td>Creatine <br/></td><td>131.133<br/></td><td>131.13<br/></td><td>13.11<br/></td></tr><tr><td>Cytidine <br/></td><td>243.2166<br/></td><td>243.22<br/></td><td>24.32<br/></td></tr><tr><td>dTMP <br/></td><td>320.1926 <br/></td><td>366.17<br/></td><td>36.62<br/></td></tr><tr><td>Fructose <br/></td><td>180.16<br/></td><td>180.16<br/></td><td>18.02<br/></td></tr><tr><td>Glucose-1-phosphate<br/></td><td>260.135<br/></td><td>336.32<br/></td><td>33.63<br/></td></tr><tr><td>Glutathione <br/></td><td>307.3235 <br/></td><td>307.32<br/></td><td>30.73<br/></td></tr><tr><td>GMP <br/></td><td>363.22<br/></td><td>407.18<br/></td><td>40.72<br/></td></tr><tr><td>IMP <br/></td><td>348.206<br/></td><td>392.17<br/></td><td>39.22<br/></td></tr><tr><td>O-phosphoethanolamine<br/></td><td>141.063<br/></td><td>141.06<br/></td><td>14.11<br/></td></tr><tr><td>Pyridoxal <br/></td><td>167.16<br/></td><td>203.62<br/></td><td>20.36<br/></td></tr><tr><td>Thiamine <br/></td><td>265.35<br/></td><td>337.23<br/></td><td>33.72<br/></td></tr><tr><td>trans-Urocanate<br/></td><td>137.118<br/></td><td>137.12<br/></td><td>13.71<br/></td></tr><tr><td>UMP <br/></td><td>324.1813 <br/></td><td>368.15<br/></td><td>36.82<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Xanthine <br/></td><td>152.11 <br/></td><td>152.11<br/></td><td>15.21<br/></td></tr></tbody></table><em>Note: Dissolve this pool at 28.54 mg/ml for 5 mM solution.</em><br/><br/><strong>Table 7. Chemical standard library pool 3</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Metabolite name</strong><br/></td><td><strong>Molecular weight <br/>of metabolite</strong><br/></td><td><strong>Molecular weight <br/>of chemical standard</strong><br/></td><td><strong>Amount to weigh (mg)</strong><br/></td></tr><tr><td>3-hydroxybutyric acid<br/></td><td>104.1045<br/></td><td>126.09<br/></td><td>252.18<br/></td></tr><tr><td>Acetylalanine <br/></td><td>131.1299<br/></td><td>131.13<br/></td><td>262.26<br/></td></tr><tr><td>Acetylaspartate <br/></td><td>175.139<br/></td><td>175.14<br/></td><td>350.28<br/></td></tr><tr><td>Acetylcarnitine <br/></td><td>203.2356<br/></td><td>239.70<br/></td><td>479.40<br/></td></tr><tr><td>Acetylglutamine <br/></td><td>188.183<br/></td><td>188.18<br/></td><td>376.36<br/></td></tr><tr><td>ADP <br/></td><td>427.203<br/></td><td>501.32<br/></td><td>1002.64<br/></td></tr><tr><td>Allantoin <br/></td><td>158.121 <br/></td><td>158.121<br/></td><td>316.24<br/></td></tr><tr><td>CDP <br/></td><td>403.177<br/></td><td>403.20<br/></td><td>806.40<br/></td></tr><tr><td>CDP-choline<br/></td><td>489.332<br/></td><td>510.31<br/></td><td>1020.62<br/></td></tr><tr><td>Coenzyme A<br/></td><td>767.535<br/></td><td>767.53<br/></td><td>1535.06<br/></td></tr><tr><td>Creatinine <br/></td><td>113.12<br/></td><td>113.12<br/></td><td>226.24<br/></td></tr><tr><td>gamma-aminobutyric acid<br/></td><td>103.12<br/></td><td>103.12<br/></td><td>206.24<br/></td></tr><tr><td>GDP <br/></td><td>443.201<br/></td><td>443.20<br/></td><td>886.40<br/></td></tr><tr><td>Glutathione disulfide<br/></td><td>612.631<br/></td><td>612.63<br/></td><td>1225.26<br/></td></tr><tr><td>Glycerate <br/></td><td>106.0773<br/></td><td>286.25<br/></td><td>572.50<br/></td></tr><tr><td>Hypoxanthine <br/></td><td>136.1115 <br/></td><td>136.11<br/></td><td>272.22<br/></td></tr><tr><td>myo-Inositol<br/></td><td>180.16<br/></td><td>180.16<br/></td><td>360.32<br/></td></tr><tr><td>NAD+<br/></td><td>663.43<br/></td><td>663.43<br/></td><td>1326.86<br/></td></tr><tr><td>p-aminobenzoate <br/></td><td>137.138<br/></td><td>137.14<br/></td><td>274.28<br/></td></tr><tr><td>Phosphocholine <br/></td><td>184.152<br/></td><td>329.73<br/></td><td>659.46<br/></td></tr><tr><td>Sorbitol <br/></td><td>182.17<br/></td><td>182.17<br/></td><td>364.34<br/></td></tr><tr><td>UDP <br/></td><td>404.1612<br/></td><td>448.12<br/></td><td>896.24<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>UDP-glucose<br/></td><td>566.302<br/></td><td>610.27<br/></td><td>1220.54<br/></td></tr></tbody></table><em>Note: Dissol</em><em>ve this pool at 37.23 mg/ml for 5 mM solution.</em><br/><br/><strong>Table 8. Chemical standard library pool 4</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Metabolite name</strong><br/></td><td><strong>Molecular weight of metabolite</strong><br/></td><td><strong>Molecular weight of chemical standard</strong><br/></td><td><strong>Amount to weigh (mg)</strong><br/></td></tr><tr><td>Phenylacetylglutamine <br/></td><td>264.3<br/></td><td>264.3<br/></td><td>17.17<br/></td></tr><tr><td>Acetylglutamate <br/></td><td>189.1659<br/></td><td>189.1659<br/></td><td>12.29<br/></td></tr><tr><td>Acetylglycine <br/></td><td>117.1033<br/></td><td>117.1033<br/></td><td>7.61<br/></td></tr><tr><td>Acetylmethionine <br/></td><td>191.245<br/></td><td>191.245<br/></td><td>12.43<br/></td></tr><tr><td>Asymmetric dimethylarginine<br/></td><td>202.25<br/></td><td>275.2<br/></td><td>17.88<br/></td></tr><tr><td>ATP</td><td>507.18<br/></td><td>551.14<br/></td><td>35.82<br/></td></tr><tr><td>CTP</td><td>483.1563<br/></td><td>527.12<br/></td><td>34.26<br/></td></tr><tr><td>dATP <br/></td><td>491.2<br/></td><td>535.15<br/></td><td>34.78<br/></td></tr><tr><td>dCTP <br/></td><td>467.2<br/></td><td>511.12<br/></td><td>33.22<br/></td></tr><tr><td>Deoxycytidine <br/></td><td>227.2172<br/></td><td>227.2172<br/></td><td>14.76<br/></td></tr><tr><td>Folic acid<br/></td><td>441.3975<br/></td><td>441.3975<br/></td><td>28.69<br/></td></tr><tr><td>GTP <br/></td><td>523.2<br/></td><td>523.18<br/></td><td>34.00<br/></td></tr><tr><td>Hypotaurine <br/></td><td>109.1475<br/></td><td>109.1475<br/></td><td>7.09<br/></td></tr><tr><td>Methionine sulfoxide<br/></td><td>165.21<br/></td><td>165.21<br/></td><td>10.73<br/></td></tr><tr><td>Methylthioadenosine <br/></td><td>297.3335<br/></td><td>297.3335<br/></td><td>19.32<br/></td></tr><tr><td>Phosphocreatine <br/></td><td>211.114<br/></td><td>255.08<br/></td><td>16.58<br/></td></tr><tr><td>Pyridoxine <br/></td><td>169.18<br/></td><td>205.64<br/></td><td>13.36<br/></td></tr><tr><td>Ribose-5-phosphate<br/></td><td>230.11<br/></td><td>310.1<br/></td><td>20.15<br/></td></tr><tr><td>SAH <br/></td><td>384.4<br/></td><td>384.41<br/></td><td>24.98<br/></td></tr><tr><td>Thymidine <br/></td><td>242.2286<br/></td><td>242.2286<br/></td><td>15.74<br/></td></tr><tr><td>Trimethyllysine <br/></td><td>189.279 <br/></td><td>224.73 <br/></td><td>14.60<br/></td></tr><tr><td>Uridine <br/></td><td>244.2014<br/></td><td>244.2014<br/></td><td>15.87<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>UTP <br/></td><td>484.1411<br/></td><td>559.09<br/></td><td><span style="font-family:'&quot;font-size:small;"></span>36.34<br/></td></tr></tbody></table><em>Note: Dissolve this pool at 36.75 mg/ml for 5 mM solution.</em><br/><br/><strong>Table 9. Chemical standard library pool 5</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Metabolite name</strong><br/></td><td><strong>Molecular weight of metabolite</strong><br/></td><td><strong>Molecular weight of chemical standard</strong><br/></td><td><strong>Amount to weigh (mg)</strong><br/></td></tr><tr><td>3-phosphoglycerate<br/></td><td>186.06<br/></td><td>230.02<br/></td><td>57.51<br/></td></tr><tr><td>cis-aconitic acid<br/></td><td>174.108<br/></td><td>174.11<br/></td><td>43.53<br/></td></tr><tr><td>Citrate <br/></td><td>192.124<br/></td><td>294.10<br/></td><td>73.53<br/></td></tr><tr><td>DHAP <br/></td><td>170.06 <br/></td><td>180.19<br/></td><td>45.05<br/></td></tr><tr><td>Fructose-1,6-bisphosphate<br/></td><td>340.1157<br/></td><td>406.06<br/></td><td>101.52<br/></td></tr><tr><td>Fumarate <br/></td><td>116.07 <br/></td><td>116.07 <br/></td><td>29.02<br/></td></tr><tr><td>Glucose-6-phosphate<br/></td><td>260.135<br/></td><td>282.12<br/></td><td>70.53<br/></td></tr><tr><td>Glycerol-3-phosphate<br/></td><td>172.0737<br/></td><td>370.40<br/></td><td>92.60<br/></td></tr><tr><td>Guanidinoacetate <br/></td><td>117.1066<br/></td><td>117.11<br/></td><td>29.28<br/></td></tr><tr><td>Kynurenine <br/></td><td>208.2139<br/></td><td>208.21 <br/></td><td>52.05<br/></td></tr><tr><td>Malate <br/></td><td>134.0874<br/></td><td>134.09<br/></td><td>33.52<br/></td></tr><tr><td>NADP+<br/></td><td>744.413<br/></td><td>765.39 <br/></td><td>191.35<br/></td></tr><tr><td>Niacinamide <br/></td><td>122.12<br/></td><td>122.12 <br/></td><td>30.53<br/></td></tr><tr><td>2-oxoglutarate<br/></td><td>146.11 <br/></td><td>146.11 <br/></td><td>36.53<br/></td></tr><tr><td>Phosphoenolpyruvate <br/></td><td>168.042<br/></td><td>267.22<br/></td><td>66.81<br/></td></tr><tr><td>Pyruvate <br/></td><td>88.06<br/></td><td>110.04<br/></td><td>27.51<br/></td></tr><tr><td>Succinate <br/></td><td>118.09<br/></td><td>118.09<br/></td><td>29.52<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Uracil <br/></td><td>112.0868<br/></td><td>112.09<br/></td><td>28.02<br/></td></tr></tbody></table><em>Note: Dissolve this pool at 20.77 mg/ml for 5 mM solution.</em><br/><br/><strong>Table 10. Chemical standard library pool 6</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Metabolite name</strong><br/></td><td><strong>Molecular weight of metabolite</strong><br/></td><td><strong>Molecular weight of chemical standard</strong><br/></td><td><strong>Amount to weigh (mg)</strong><br/></td></tr><tr><td>3-hydroxyisobutyric acid <br/></td><td>104.1045<br/></td><td>126.09<br/></td><td>22.07<br/></td></tr><tr><td>2-hydroxyglutarate <br/></td><td>148.114<br/></td><td>192.10<br/></td><td>33.62<br/></td></tr><tr><td>Aminoadipate <br/></td><td>161.156<br/></td><td>161.16<br/></td><td>28.20<br/></td></tr><tr><td>beta-alanine<br/></td><td>89.093<br/></td><td>89.09<br/></td><td>15.59<br/></td></tr><tr><td>Carbamoylaspartate <br/></td><td>176.128<br/></td><td>176.13 <br/></td><td>30.82<br/></td></tr><tr><td>Cystathionine <br/></td><td>222.263<br/></td><td>222.26<br/></td><td>38.90<br/></td></tr><tr><td>Cysteic acid<br/></td><td>169.16<br/></td><td>169.16<br/></td><td>29.60<br/></td></tr><tr><td>FAD <br/></td><td>785.5497<br/></td><td>829.51<br/></td><td>145.16<br/></td></tr><tr><td>Glycerophosphocholine <br/></td><td>258.231 <br/></td><td>257.22<br/></td><td>45.01<br/></td></tr><tr><td>Inosine <br/></td><td>268.229 <br/></td><td>268.23<br/></td><td>46.94<br/></td></tr><tr><td>Orotate <br/></td><td>156.1<br/></td><td>194.19<br/></td><td>33.98<br/></td></tr><tr><td>Pantothenate <br/></td><td>219.23<br/></td><td>238.27 <br/></td><td>41.70<br/></td></tr><tr><td>Phosphoserine <br/></td><td>185.07<br/></td><td>185.07<br/></td><td>32.39<br/></td></tr><tr><td>Riboflavin <br/></td><td>376.369<br/></td><td>376.37<br/></td><td>65.86<br/></td></tr><tr><td>UDP-GlcNAc<br/></td><td>607.3537<br/></td><td>651.32<br/></td><td>113.98<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Uric acid<br/></td><td>168.1103<br/></td><td>168.11<br/></td><td>29.42<br/></td></tr></tbody></table><em>Note: Dissolve this pool at 21.52 mg/ml for 5 mM solution.</em><br/><br/><strong>Table 11. Chemical standard library pool 7</strong><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1100px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Metabolite name</strong><br/></td><td><strong>Molecular weight of metabolite</strong><br/></td><td><strong>Molecular weight of chemical standard</strong><br/></td><td><strong>Amount to weigh (mg)</strong><br/></td></tr><tr><td>Itaconic acid<br/></td><td>130.0987<br/></td><td>130.10<br/></td><td>52.04<br/></td></tr><tr><td>Homocysteine <br/></td><td>135.185<br/></td><td>135.19<br/></td><td>54.07<br/></td></tr><tr><td>2-oxobutyric acid<br/></td><td>102.0886<br/></td><td>102.09<br/></td><td>40.84<br/></td></tr><tr><td>2-hydroxybutyric acid<br/></td><td>104.1045<br/></td><td>126.09<br/></td><td>50.44<br/></td></tr><tr><td>Ascorbate <br/></td><td>176.1241 <br/></td><td>198.11<br/></td><td>79.24<br/></td></tr><tr><td>Sarcosine <br/></td><td>89.0932<br/></td><td>89.09<br/></td><td>35.64<br/></td></tr><tr><td>Dimethylglycine <br/></td><td>103.1198<br/></td><td>103.12<br/></td><td>41.25<br/></td></tr><tr><td>N6-acetyllysine<br/></td><td>188.2242 <br/></td><td>188.22<br/></td><td>75.29<br/></td></tr><tr><td>Pipecolate <br/></td><td>129.157<br/></td><td>129.16<br/></td><td>51.66<br/></td></tr><tr><td>Indolelactate <br/></td><td>205.2099 <br/></td><td>205.21<br/></td><td>82.08<br/></td></tr><tr><td>Picolinate <br/></td><td>123.1094 <br/></td><td>123.11 <br/></td><td>49.24<br/></td></tr><tr><td>3-methyl-2-oxobutyrate<br/></td><td>116.1152<br/></td><td>138.10<br/></td><td>55.24<br/></td></tr><tr><td>3-methyl-2-oxopentanoic acid<br/></td><td>130.1418<br/></td><td>152.12<br/></td><td>60.85<br/></td></tr><tr><td>Formyl-methionine <br/></td><td>177.221 <br/></td><td>177.22<br/></td><td>70.89<br/></td></tr><tr><td>2-aminobutyric acid<br/></td><td>103.1198<br/></td><td>103.12<br/></td><td>41.25<br/></td></tr><tr><td>Homocitrulline <br/></td><td>189.2123<br/></td><td>189.21<br/></td><td>75.68<br/></td></tr><tr><td>gamma-glutamyl-alanine<br/></td><td>217.2224 <br/></td><td>218.21<br/></td><td>87.28<br/></td></tr><tr><td>Mannose<br/></td><td>180.16<br/></td><td>180.16<br/></td><td>72.06<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Cysteine-glycine (dipeptide)<br/></td><td>178.21 <br/></td><td>178.21 <br/></td><td>71.28<br/></td></tr></tbody></table><em>Note: Dissolve this pool at 14.33 mg/ml for 5 mM solution.</em><br/></li></ol><p class="pptt" id="biaoti30407">### Acknowledgments</p><p class="ppzz">This protocol is based on our previously published study (Sullivan <em>et al</em><em>.</em>, 2019). This work has been supported by a grant to AM from the NIH (F32CA213810). MRS was supported by T32GM007287 and acknowledges additional support from an MIT Koch Institute Graduate Fellowship.</p><p class="pptt" id="biaoti30408">### Competing interests</p><p class="ppzz">The authors report no competing interests. CAL is a paid consultant for ReviveMed.</p><p class="pptt" id="biaoti30409">### Ethics</p><p class="ppzz">This study was performed in accordance with the recommendations in the Guide for the Care and Use of Laboratory Animals. All animal experiments were performed using protocols (#1115-110-18) that were approved by the MIT Committee on Animal Care (IACUC). All surgeries were performed using isoflurane anesthesia administered by vaporizer and every effort was made to minimize suffering.</p><p class="pptt" id="biaoti30410">### References</p><ol class="npwtwok"><li>Abbondante, S., Eckel-Mahan, K. L., Ceglia, N. J., Baldi, P. and Sassone-Corsi, P. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26644470" target="_blank">Comparative circadian metabolomics reveal differential effects of nutritional challenge in the serum and liver.</a> <em>J Biol Chem</em> 291(6): 2812-2828.</li><li>Anastasiou, D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28006817" target="_blank">Tumour microenvironment factors shaping the cancer metabolism landscape.</a> <em>Br J Cancer</em> 116(3): 277-286.</li><li>Bi, J., Wu, S., Zhang, W. and Mischel, P. S. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29775654" target="_blank">Targeting cancer's metabolic co-dependencies: A landscape shaped by genotype and tissue context.</a> <em>Biochim Biophys Acta Rev Cancer</em> 1870(1): 76-87.</li><li>Burgess, E. A. and Sylven, B. (1962). <a href="http://www.ncbi.nlm.nih.gov/pubmed/13874819" target="_blank">Glucose, lactate, and lactic dehydrogenase activity in normal interstitial fluid and that of solid mouse tumors.</a> <em>Cancer Res</em> 22: 581-588.</li><li>Cairns, R. A., Harris, I. S. and Mak, T. W. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21258394" target="_blank">Regulation of cancer cell metabolism.</a> <em>Nat Rev Cancer</em> 11(2): 85-95.</li><li>Cantor, J. R., Abu-Remaileh, M., Kanarek, N., Freinkman, E., Gao, X., Louissaint, A. Jr., Lewis, C. A., and Sabatini, D.M. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28388410" target="_blank">Physiologic Medium Rewires Cellular Metabolism and Reveals Uric Acid as an Endogenous Inhibitor of UMP Synthase.</a> <em>Cell</em> 169(2): 258-272 e217.</li><li>Chong, J., Soufan, O., Li, C., Caraus, I., Li, S., Bourque, G., Wishart, D. S. and Xia, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29762782" target="_blank">MetaboAnalyst 4.0: towards more transparent and integrative metabolomics analysis.</a> <em>Nucleic Acids Res</em> 46(W1): W486-W494.</li><li>Dallmann, R., Viola, A. U., Tarokh, L., Cajochen, C. and Brown, S. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22308371" target="_blank">The human circadian metabolome.</a> <em>Proc Natl Acad Sci U S A</em> 109(7): 2625-2629.</li><li>DeBerardinis, R. J. and Chandel, N. S. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27386546" target="_blank">Fundamentals of cancer metabolism.</a> <em>Sci Adv</em> 2(5): e1600200.</li><li>Eil, R., Vodnala, S. K., Clever, D., Klebanoff, C. A., Sukumar, M., Pan, J. H., Palmer, D. C., Gros, A., Yamamoto, T. N., Patel, S. J., Guittard, G. C., Yu, Z., Carbonaro, V., Okkenhaug, K., Schrump, D. S., Linehan, W M., Roychoudhuri, R. and Restifo, N. P. (2016). <a href="https://www.nature.com/articles/nature19364" target="_blank">Ionic immune suppression within the tumour microenvironment limits T cell effector function.</a> <em>Nature</em> 537: 539-543.</li><li>Evans, A. M., DeHaven, C. D., Barrett, T., Mitchell, M. and Milgram, E. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19624122" target="_blank">Integrated, nontargeted ultrahigh performance liquid chromatography/electrospray ionization tandem mass spectrometry platform for the identification and relative quantification of the small-molecule complement of biological systems.</a> <em>Anal Chem</em> 81(16): 6656-6667.</li><li>Gieger, C., Geistlinger, L., Altmaier, E., HrabÃ© de Angelis, M., Kronenberg, F., Meitinger, T., Mewes, H. W., Wichmann, H. E., Weinberger, K. M., Adamski, J., Illig, T. and Suhre, K. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19043545" target="_blank">Genetics meets metabolomics: a genome-wide association study of metabolite profiles in human serum.</a> <em>PLoS Genet</em> 4(11): e1000282.</li><li>Gullino, P. M., Clark, S. H. and Grantham, F. H. (1964). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14190544" target="_blank">The interstitial fluid of solid tumors.</a> <em>Cancer Res</em> 24: 780-794.</li><li>Guijas, C., Montenegro-Burke, J. R., Domingo-Almenara, X., Palermo, A., Warth, B., Hermann, G., Koellensperger, G., Huan, T., Uritboonthai, W., Aisporna, A. E., Wolan, D. W., Spilker, M. E., Benton, H. P. and Siuzdak, G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29381867" target="_blank">METLIN: a technology platform for identifying knowns and unknowns.</a> <em>Anal Chem</em> 90(5): 3156-3164.</li><li>Haslene-Hox, H., Oveland, E., Berg, K. C., Kolmannskog, O., Woie, K., Salvesen, H. B., Tenstad, O. and Wiig, H. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21541282" target="_blank">A new method for isolation of interstitial fluid from human solid tumors applied to proteomic analysis of ovarian carcinoma tissue.</a> <em>PLoS One</em> 6(4): e19217.</li><li>Ho, P. C., Bihuniak, J. D., Macintyre, A. N., Staron, M., Liu, X., Amezquita, R., Tsui, Y. C., Cui, G., Micevic, G., Perales, J. C., Kleinstein, S. H., Abel, E. D., Insogna, K. L., Feske, S., Locasale, J. W., Bosenberg, M. W., Rathmell, J. C., Kaech, S. M. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26321681" target="_blank">Phosphoenolpyruvate is a metabolic checkpoint of anti-tumor T cell responses.</a> <em>Cell</em> 162(6): 1217-1228.</li><li>Lawton, K. A., Berger, A., Mitchell, M., Milgram, K. E., Evans, A. M., Guo, L., Hanson, R. W., Kalhan, S. C., Ryals, J. A. and Milburn, M. V. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18384253" target="_blank">Analysis of the adult human plasma metabolome.</a> <em>Pharmacogenomics</em> 9(4): 383-397.</li><li>Mazzone, P. J., Wang, X. F., Beukemann, M., Zhang, Q., Seeley, M., Mohney, R., Holt, T. and Pappan, K. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26762741" target="_blank">Metabolite Profiles of the Serum of Patients with Non-Small Cell Carcinoma.</a> <em>J Thorac Oncol</em> 11(1): 72-78</li><li>Muir, A., Danai, L. V. and Vander Heiden, M. G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30104199" target="_blank">Microenvironmental regulation of cancer cell metabolism: implications for experimental design and translational studies.</a> <em>Dis Model Mech</em> 11(8).</li><li>Nagarajan, A., Malvi, P. and Wajapeyee, N. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27822561" target="_blank">Oncogene-directed alterations in cancer cell metabolism.</a> <em>Trends Cancer</em> 2(7): 365-377.</li><li>Panuwet, P., Hunter, R. E., Jr., D'Souza, P. E., Chen, X., Radford, S. A., Cohen, J. R., Marder, M. E., Kartavenka, K., Ryan, P. B. and Barr, D. B. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25562585" target="_blank">Biological matrix effects in quantitative tandem mass spectrometry-based analytical methods: advancing biomonitoring.</a> <em>Crit Rev Anal Chem</em> 46(2): 93-105.</li><li>Schroeder, K. S. V. A. (2019). <a href="https://www.jove.com/science-education/10246/blood-withdrawal-i" target="_blank">Blood Withdrawal I.</a> JoVE.</li><li>Siska, P. J., Beckermann, K. E., Mason, F. M., Andrejeva, G., Greenplate, A. R., Sendor, A. B., Chiang, Y. J., Corona, A. L., Gemta, L. F., Vincent, B. G., Wang, R. C., Kim, B., Hong, J., Chen, C. L., Bullock, T. N., Irish, J. M., Rathmell, W. K. and Rathmell, J. C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28614802" target="_blank">Mitochondrial dysregulation and glycolytic insufficiency functionally impair CD8 T cells infiltrating human renal cell carcinoma.</a> <em>JCI Insight</em> 2(12).</li><li>Spinelli, J. B., Yoon, H., Ringel, A. E., Jeanfavre, S., Clish, C. B. and Haigis, M. C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29025995" target="_blank">Metabolic recycling of ammonia via glutamate dehydrogenase supports breast cancer biomass.</a> <em>Science</em> 358(6365): 941-946.</li><li>Stevens, V. L., Hoover, E., Wang, Y. and Zanetti, K. A. (2019). <a href="https://www.mdpi.com/2218-1989/9/8/156" target="_blank">Pre-analytical factors that affect metabolite stability in human urine, plasma, and serum: A review.</a> <em>Metabolites</em> 9(8):156.</li><li>Sullivan, M. R., Danai, L. V., Lewis, C. A., Chan, S. H., Gui, D. Y., Kunchok, T., Dennstedt, E. A., Vander Heiden, M. G. and Muir, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30990168" target="_blank">Quantification of microenvironmental metabolites in murine cancers reveals determinants of tumor nutrient availability.</a> <em>Elife</em> 8: e44235.</li><li>Wiig, H. and Swartz, M. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22811424" target="_blank">Interstitial fluid and lymph formation and transport: physiological regulation and roles in inflammation and cancer.</a> <em>Physiol Rev</em> 92(3): 1005-1060.</li><li>Wiig, H., Tenstad, O., Iversen, P. O., Kalluri, R. and Bjerkvig, R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20653943" target="_blank">Interstitial fluid: the overlooked component of the tumor microenvironment?</a> <em>Fibrogenesis Tissue Repair</em> 3: 12.</li><li>Zhang, Y., Kurupati, R., Liu, L., Zhou, X. Y., Zhang, G., Hudaihed, A., Filisio, F., Giles-Davis, W., Xu, X., Karakousis, G. C., Schuchter, L. M., Xu, W., Amaravadi, R., Xiao, M., Sadek, N., Krepler, C., Herlyn, M., Freeman, G. J., Rabinowitz, J. D. and Ertl, H. C. J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28898698" target="_blank">Enhancing CD8<sup>+</sup> T cell fatty acid catabolism within a metabolically challenging tumor microenvironment increases the efficacy of melanoma immunotherapy.</a> <em>Cancer Cell</em> 32(3): 377-391.<br/></li></ol>