<body>
            <h1>Labeling and Isolation of Fluorouracil Tagged RNA by Cytosine Deaminase Expression</h1>
            <p><strong>Authors:</strong> Harihar Basnet, Joan  Massague</p>
            <h2>Abstract</h2>
            <p>Tissues are comprised of different cell types whose interactions elicit distinct gene expression patterns that regulate tissue formation, regeneration, homeostasis and repair. Analysis of these gene expression patterns require methods that can capture as closely as possible the transcriptomes of cells of interest in their tissue microenvironment. Current technologies designed to study <em>in situ</em> transcriptomics are limited by their low sensitivity that require cell types to represent more than 1% of the total tissue, making it challenging to transcriptionally profile rare cell populations rapidly isolated from their native microenvironment. To address this problem, we developed fluorouracil-tagged RNA sequencing (Flura-seq) that utilizes cytosine deaminase (CD) to convert the non-natural pyrimidine fluorocytosine to fluorouracil. Expression of <em>S. cerevisiae</em> CD and exposure to fluorocytosine generates fluorouracil and metabolically labels newly synthesized RNAs specifically in cells of interest. Fluorouracil-tagged RNAs can then be immunopurified and used for downstream analysis. Here, we describe the detailed protocol to perform Flura-seq both <em>in vitro</em> and <em>in vivo</em>. The robustness, simplicity and lack of toxicity of Flura-seq make this tool broadly applicable to many studies in developmental, regenerative, and cancer biology.</p>
            <p><strong>Keywords:</strong> <em>In situ</em> transcriptomics, Cytosine deaminase, Uracil phosphoribosyl transferase, Fluorouracil, Nascent RNA</p>
            <h2>Background</h2><p>In an organ, cells interact with tissue microenvironment that includes organ-specific resident cells, immune cells, perivascular niches, extracellular matrix, cytokines, metabolites, and an oxygen concentration range. These interactions dictate the expression of specific set of genes in the cells that often have functional significances. The transcriptional responses associated with these interactions are usually dynamic and can only be observed if the interactions are preserved. Thus, it is critical to maintain the tissue microenvironment to ensure the capture of true transcriptional state of cells under the physiological conditions.<br/><br/>Current techniques to study cell-type specific transcriptomes have limitations that preclude their effective application in studying rare and under-represented cell populations. Single-cell RNA sequencing (scRNA-seq) with or without an intervening fluorescence activated cell sorting (FACS) step, requires extensive physical and enzymatic processing of the tissue, which not only disrupts the effects of the host microenvironment on cells but also exerts stress on these cells, severely compromising the ability to discern the impact of the host stroma from the transcriptome of the isolated cells. As many as 7,500 genes have been reported to be altered by more than two-fold during the FACS processing of muscle stem cells for transcriptomic analysis (Machado <em>et al.</em>, 2017). <em>In situ</em> transcriptomic profiling obviate these problems but lack the necessary sensitivity for cell populations that represent less than 1% of the tissue. For example, translating ribosome affinity purification and mRNA sequencing (TRAP-Seq) (Heiman <em>et al.</em>, 2008) is not suitable to analyze cells that constitute less than 1% of the total population (Bertin <em>et al.</em>, 2015; Obenauf <em>et al.</em>, 2015). Direct enzyme based metabolic tagging of RNA with thiouracil (TU) and ethynyl cytosine (EC) in the cells of interest are limited in sensitivity and specificity due to collateral tagging of RNA in cells lacking the enzymes, and requires additional <em>in vitro</em> biotinylation steps (Cleary <em>et al.</em>, 2005; Miller <em>et al.</em>, 2009; Gay <em>et al.</em>, 2013 and 2014; Hida <em>et al.</em>, 2017). TU tagging has a sensitivity limit of 5% (Gay <em>et al.</em>, 2013). Thiol (SH)-linked alkylation of the metabolic labeling of RNA in tissue (SLAM-ITseq) eliminates the noise associated with the purification of RNAs that are not thiol tagged in TU-tagging method (Matsushima <em>et al.</em>, 2018), but undesired TU tagging through endogenous enzymes in cells lacking UPRT expression remains a limitation. We described the development of Flura-seq, a cytosine deaminase (CD)-based method for <em>in situ</em> transcriptomic profiling of rare cell populations that represent as little as 0.003% of an organ (Basnet <em>et al.</em>, 2019). Flura-seq requires exogenous expression of CD, a key enzyme of the pyrimidine salvage pathway in fungi and prokaryotes (Mullen <em>et al.</em>, 1992) and UPRT from <em>T. gondii</em>. CD is absent in mammalian cells, which instead use cytidine deaminase for the same purpose (Mullen <em>et al.</em>, 1992). In addition to converting cytosine to uracil, CD can also convert 5-fluorocytosine (5-FC), a non-natural pyrimidine, to 5-flourouracil (5-FU). 5-FU is endogenously converted to fluorouridine triphosphate (F-UTP), which is then incorporated into RNA. The co-expression of UPRT limits the labeling of RNA in the cells expressing CD (Basnet <em>et al.</em>, 2019). Flura-seq is applicable to <em>in vitro</em> co-culture experiments as well as <em>in vivo</em> experiments where cells of interest, expressing CD-UPRT, are present in the intact tissue. Flura-seq is technically very simple that only requires very basic molecular biology skills, and the whole procedures can be completed within 24 hours. Here, we describe the detailed protocol for labeling and isolation of RNA for Flura-seq for both <em>in vitro</em> and <em>in vivo</em> experiments. </p>
        </body>

<p class="pptt" id="biaoti30315">## Materials and Reagents</p><ol class="npwtwok"><li>5 ml Polystyrene Round-Bottom tube (Corning, catalog number: 352058)</li><li>60 mm Tissue culture plates (Corning, catalog number: 353004)</li><li>Oligo (dT)<sub>25</sub> magnetic beads (New England Biolabs, catalog number: S1419S)</li><li>Protein G Dynabeads (Thermo Fisher Scientific, catalog number: 10004D)</li><li>6-8 weeks old athymic nude female mice (Envigo, catalog number: 069)</li><li>Doxycycline diet (625 mg/kg, Envigo, catalog number: TD.07383)</li><li>BrdU antibody (Abcam, catalog number: ab6326)</li><li>CD-UPRT (Addgene, plasmid number: 126677)</li><li>rtTA3 (Addgene, plasmid number: 26730)</li><li>Doxycycline (Sigma-Aldrich, catalog number: D9891)</li><li>5-Fluorocytosine (5-FC) (Sigma-Aldrich, catalog number: F7129)</li><li>5-Bromo-2â-deoxyuridine (BrdU) (Millipore Sigma, catalog number: B5002)</li><li>Thymine (Sigma-Aldrich, catalog number: T0376)</li><li>RNeasy MinElute Cleanup kit (Qiagen, catalog number: 74204)</li><li>cDNA kit-First Strand Transcriptor (Roche, catalog number: 043790-12001)<br/></li><li>RNeasy Mini Kit (Qiagen, catalog number: 74106)</li><li>Ethylenediaminetetraacetic acid solution (EDTA) (Sigma-Aldrich, catalog number: 03690)</li><li>Tween 20 (Sigma Millipore, catalog number: P1379)</li><li>Lithium dodecyl sulfate (LiDS) (Sigma Millipore, catalog number: L4632)</li><li>Lithium chloride (LiCl) (Sigma Millipore, catalog number: 203637)</li><li>Ultrapure BSA (BSA) (Thermo Fisher Scientific, catalog number: AM2616)</li><li>Glycogen (Thermo Fisher Scientific, catalog number: R0551)</li><li>RNAlater (Sigma-Aldrich, catalog number: R0901)</li><li>20x UltraPure SSPE Buffer (Thermo Fisher Scientific, catalog number: 15591043)</li><li>PBS</li><li>Tris base</li><li>Tris-HCl</li><li>Ethanol (Decon Labs, catalog number: 2716)</li><li>DTT (Sigma-Aldrich, catalog number: 3483-12-3)</li><li>Cell lysis buffer (see Recipes)</li><li>mRNA Wash Buffer I (see Recipes)</li><li>mRNA Wash Buffer II (see Recipes)</li><li>mRNA Wash Buffer III (see Recipes)</li><li>Elution Buffer (see Recipes)</li><li>SSPET buffer (see Recipes)</li><li>TE buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30316">### Equipment</p><ol class="npwtwok"><li>PRO 200 grinder from PRO Scientific Inc (Homogenizer) (PRO Scientific, catalog number: 01-01200) </li><li>Magnetic rack (ThermoFisher Scientific, catalog number: 12321D)</li><li>Thermo scientific Rotator (ThermoFisher Scientific, catalog number: 88881001)</li><li>Eppendorf Thermomixer (ThermoFisher Scientific, catalog number: 05-400-200)</li><li>Centrifuge<br/></li></ol><p class="pptt" id="biaoti30317">### Procedure</p><ol class="npwone"><li>Making CD-UPRT expressing stable cells<ol class="npwtwo"><li>Generate CD-UPRT expressing cell line of interest (for example, MDA-MB-231) by transducing cells with CD-UPRT (Addgene) and rtTA3 (Addgene) lentivirus using standard protocol.<br/><br/></li></ol></li><li>Labeling of RNA <em>in vitro</em><br/><strong>Day 1</strong><br/><ol class="npwtwo"><li>Plate 1,000 cells (for example, MDA-MB-231) expressing Doxycycline inducible CD-UPRT (cells of interest) with 1 million 4T1 cells (or other cells (optional)) in 60 mm plates in 3 ml volume (Doxycycline inducible system is optional).</li></ol><strong>Day 2</strong> <ol class="npwtwo" start="2"><li>Add 1 Î¼g/ml Doxycycline.</li></ol><strong>Day 3</strong> <ol class="npwtwo" start="3"><li>24 h post Doxycycline treatment, add 5-FC and thymine to 250 Î¼M (50 mM stock) and 125 Î¼M (25 mM stock) final concentration, respectively. 5-FC and thymine stock solutions are made in PBS. </li><li>Harvest cells after 2-12 h using cell lysis buffer. Wash the cells 1x with 3 ml PBS before adding 1 ml of cell lysis buffer on the plate. Scrape the cells in 1.5 ml eppendorf tube. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Labeling of RNA in mice<br/><strong>Day 1</strong> <ol class="npwtwo"><li>Prepare 500,000/ml CD-UPRT expressing stable cells (described above in A.1) in PBS.</li><li>Inject 100 Î¼l (50,000 cells) in mouse through tail vein.</li></ol><strong>Day 28</strong> <ol class="npwtwo" start="3"><li>Change the mice diet to Doxycycline diet, 3 days before 5-FC injection, if cells are expressing Doxycycline inducible CD-UPRT. Skip this step if cells are constitutively expressing CD-UPRT.</li></ol><strong>Day 31</strong> <ol class="npwtwo" start="4"><li>Inject the mice with 250 mg/kg 5-FC in PBS (12.5 Î¼g/ml stock) intraperitoneally, and 125 mg/kg thymine in PBS (6.25 Î¼g/ml stock) subcutaneously.</li><li>After 4-12 h, euthanize the mice by CO<sub>2</sub> asphyxiation, harvest the organs, and keep the organ in RNAlater buffer.</li><li>Put the organ of interest (for example, lungs) in 2 ml cell lysis buffer (Recipe 1) in 5 ml Polystyrene Round-Bottom tube, and homogenize the organ with a tissue grinder.</li><li>Divide the lysate equally into two tubes and add 3 ml of cell lysis buffer in each tube (The lysate can be flash frozen in liquid nitrogen for future use).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li> Isolation of mRNA<ol class="npwtwo"><li>Add 1x volume of cell lysis buffer (4 ml) in the lysate from C.7, rotate at room temperature (RT) for 5 min. For <em>in vitro</em> cells, add appropriate volume of lysis buffer (1 ml for confluent 60 mm plates), and scrape the cells.</li><li>Put the lysate into 1.5 ml Eppendorf tube/s, and centrifuge at 16, 000 <em>x g</em> for 5 min at RT, and keep supernatant.</li><li>Prepare oligo dT<sub>25</sub> beads (â¥ 100 Î¼l beads/~25 mg of lungs/brain).<br/><ol class="npwthree"><li>Place 250 Î¼l beads/Eppendorf tube (1.5 ml) in magnetic rack.</li><li>Remove the storage buffer.</li><li>Wash the beads 1x with 1 ml cell lysis buffer by pipetting up and down five times.</li></ol></li><li>Take out the supernatant (1 ml/250 Î¼l beads) from D.2 and add it to the freshly prepared oligo dT<sub>25</sub> beads.</li><li>Incubate at RT for 10 min in a rotator.</li><li>Discard the supernatant.</li><li> Wash the beads 2x with 1 ml mRNA wash buffer I (Recipe 2) at RT by pipetting up and down seven times.</li><li>Wash the beads 2x with 1 ml mRNA wash buffer II (Recipe 3) at RT by pipetting up and down seven times.</li><li>Wash the beads 1x with 1 ml mRNA wash buffer III (Recipe 4) at RT by pipetting up and down seven times.<br/></li><li>Wash the beads 1x with 1 ml TE buffer (Recipe 7) at RT by pipetting up and down seven times.</li><li>Add 50 Î¼l of Elution buffer (Recipe 5), and elute the RNA in Eppendorf Thermomixer at 85 Â°C for 2 min shaking at 750 rpm.</li><li>Place the beads in magnetic rack and keep the Eluate. The mRNA can be stored at -80 Â°C for future use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Purification of 5-FU tagged mRNA<ol class="npwtwo"><li>Prepare 70 Î¼l of protein G beads per sample at RT. Place the beads in magnetic rack, and remove the storage buffer. Upto 350 Î¼l (for 5 samples) of beads can be transferred to one 1.5 ml Eppendorf tube.</li><li>Wash the beads with 1 ml of 0.5x SSPET buffer (Recipe 6) at RT by pipetting up and down five times.</li><li>Block the beads with 1 ml of 0.5x SSPET buffer containing 10 Î¼g/ml BSA and 20 Î¼g/ml glycogen for 1 h at 4 Â°C.</li><li> Wash the beads 1x with 1 ml of 0.5x SSPET buffer at RT by pipetting up and down five times.</li><li>Add 35 Î¼l of beads in 750 Î¼l of 0.5x SSPET buffer containing BrdU antibody (1 Î¼g for RT-PCR, and 5 Î¼g for RNA-seq) in 1.5 ml Eppendorf tube. Rotate at 4 Â°C overnight.</li><li>Wash the beads 1x with 1 ml of 0.5x SSPET buffer at RT by pipetting up and down five times.</li><li>Add 35 Î¼l of beads in mRNA (400 Î¼l in this case) from Step D12 diluted with the same volume (400 Î¼l) of 1x SSPET buffer. Total volume will be about 835 Î¼l.</li><li>Rotate for 2 h using ThermoScientific Rotator at RT.<br/></li><li>Wash 2x with 1 ml of 0.5x SSPET buffer at RT by pipetting up and down seven times.<br/></li><li>Wash 2x with 1 ml of 1x SSPET buffer at RT by pipetting up and down seven times.</li><li>Wash 1x with 1 ml of TE buffer at RT by pipetting up and down seven times.</li><li>Elute the bound mRNA with 200 Î¼l of 100 Î¼g/ml BrdU in TE buffer in Eppendorf Thermomixer at 900 rpm for 45 min at RT.</li><li>Keep the supernatant (5-FU tagged mRNA).</li><li>Mix the 200 Î¼l supernatant with 700 Î¼l of RLT buffer from RNeasy kit.</li><li>Add 500 Î¼l of 100% ethanol, and mix well.</li><li>Pass the solution through RNeasy column.</li><li>Wash 1x with 500 Î¼l of RPE buffer from RNeasy kit.</li><li>Wash 1x with 500 Î¼l of 80% ethanol.</li><li>Spin at 13,000 <em>x g</em> for 2 min at RT.</li><li>Elute the RNA with 200 Î¼l of RNase free water.</li><li>Add 600 Î¼l of 0.5x SSPET to the elute.</li><li>Wash the Protein G Dynabeads incubated with BrdU antibody as described in Step E6.</li><li>Re-immunoprecipitate 5-FU tagged mRNA as described in Steps E7-E13.</li><li>Purify RNA as described in Steps E14-E19 using RNeasy MiniElute Cleanup kit, and elute the RNA with 12.5 Î¼l RNase free water.</li><li>Proceed with cDNA synthesis for RT-PCR or RNA-seq.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Labeling and Purification of 5-Fluorouracil (5-FU) tagged mRNA from co-cultured cells and from mouse lungs in xenograft model<br/>1,000 human MDA-MB-231 cells transduced with CD-UPRT (Addgene) and rtTA3 (Addgene) (MDA231-CDUPRT) were co-cultured with 1 million mouse 4T1 cells. RNAs were labeled as described in Section B, and 5-FU tagged mRNAs were purified as described in Section D and E. Enrichment of 5-FU tagged mRNAs of representative human and mouse genes relative to non immuno-purified mRNA were measured by RT-PCR (Figure 1A). <br/>  5 x 10<sup>4</sup> MDA231-CDUPRT cells were injected in the mouse lungs through tail vein. After 28 days, CD UPRT expression was induced by feeding mice with doxycycline diet for 3 days, and RNA was labeled as described in Section C, and 5-FU tagged mRNA was purified as described in Section D and E. Enrichment of 5-FU tagged mRNAs of representative human and mouse genes relative to non immuno-purified mRNA were measured by RT-PCR (Figure 1B).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114015648_3074.jpg" width="251"/><br/><strong>Figure 1. Labeling and isolation of 5-FU tagged RNA by cytosine deaminase expression.</strong> A. 1,000 human MDA-MB-231 cells expressing CD-UPRT were co-cultured with 1 million mouse 4T1 cells, and 5-FU tagged mRNA from human cells were isolated as described in the protocol, and enrichment of representative human genes and mouse genes relative to their corresponding 2% inputs (non immuno-purified mRNA) are shown (n = 3, Â± SEM). B. 50,000 MDA-MB-231 cells expressing CD-UPRT were injected to the lungs through tail vein, and after 28 days, 5-FU tagged mRNA was isolated as described in the protocol, and the enrichment of representative human and mouse genes relative to 1% inputs (non immune-purified mRNAs) were measured by RT-PCR (n = 6, Â± SEM). âhâ indicates human and âmâ indicates mouse.<br/></li></ol><p class="pptt" id="biaoti30318">### Notes</p><ol class="npwtwok"><li>For <em>in vitro</em> experiments, confluency of cells does not affect 5-FU tagging (Step B1) and the method is likely to be applicable to all cell types (adherent as well as suspension cells). We have successfully applied the method in MDA-MB-231, HCC1954, H2087, 293T and 4T1 cells. </li><li>Thymine does not dissolve completely in PBS. Filter the solution before use (Step B3).</li><li>Longer incubation of 5-FC and thymine increases labeling of RNA, and yields higher signal-to-noise ratio. 12 h of labeling is recommended for very small number of cells (&lt; 0.01% or &lt; 1,000 cells). For conditions consisting of more than 1 x 10<sup>5</sup> cells, 4 h of RNA labeling is sufficient. (Steps B4 and C4). <em>In vivo</em> experiments have been successfully performed with 4 h and 12 h of labeling, and <em>in vitro</em> experiments have been performed with 2-12 h of labeling.</li><li>In mice experiments, doxycycline can also be administered through drinking water (Step C3).</li><li>This protocol has been successfully used in mouse lungs, brain and mammary fat pad. Other organs have not been tested yet. </li><li>For <em>in vivo</em> experiments, the growth curve of different cell lines can vary. Hence, it is important to pick the time for 5-FC injection based on desired size of the tumors. For example, mouse cancer cell lines grow much faster than MDA-MB-231 cells, so earlier time point might be suitable for mouse cancer cell lines (Step C4).</li><li>The strain, sex and age of the mice can vary for different cell lines. For example, different mice strain might be suitable for syngeneic experiments.<br/></li></ol><p class="pptt" id="biaoti30319">### Recipes</p><ol class="npwtwok"><li>Cell lysis buffer<br/>20 mM Tris-HCl pH 7.5<br/>500 mM LiCl<br/>1% LiDS<br/>1 mM EDTA<br/>5 mM DTT (DTT is added freshly)</li><li>mRNA Wash Buffer I<br/>20 mM Tris-HCl pH 7.5<br/>500 mM LiCl<br/>0.1% LiDS<br/>1 mM EDTA<br/>5 mM DTT (DTT is added freshly)</li><li>mRNA Wash Buffer II<br/>20 mM Tris-HCl pH 7.5<br/>500 mM LiCl<br/>1 mM EDTA</li><li>mRNA Wash Buffer III<br/>20 mM Tris-HCl pH 7.5<br/>200 mM LiCl<br/>1 mM EDTA</li><li>Elution Buffer<br/>20 mM Tris-HCl pH 7.5<br/>1 mM EDTA</li><li>SSPET buffer<br/>1x SSPE (Sodium Chloride-Sodium Phosphate-EDTA)<br/>0.05% Tween 20</li><li>TE buffer<br/>10 mM Tris pH 7.5<br/>1 mM EDTA<br/></li></ol><p class="pptt" id="biaoti30320">### Acknowledgments</p><p class="ppzz">H.B. was supported by a Damon Runyon Postdoctoral Fellowship. This work was supported by NIH grants P01-CA094060 (J.M.), P30-CA008748 (MSKCC), and a DOD Innovator award W81XWH-12-0074 (J.M.). The protocol is derived from Basnet <em>et al.</em>, 2019.</p><p class="pptt" id="biaoti30321">### Competing interests</p><p class="ppzz">H.B and J.M have filed for patent for Flura-seq method (PCT/US18/22092). J.M serves in the scientific advisory board and owns company stock in Scholar Rock.</p><p class="pptt" id="biaoti30322">### Ethics</p><p class="ppzz">Mouse experiments were performed following the protocol (99-09-032) approved by the MSKCC Institutional Animal Care and Use Committee (IACUC) that is valid until 06/19/2020.</p><p class="pptt" id="biaoti30323">### References</p><ol class="npwtwok"><li>Basnet, H., Tian, L., Ganesh, K., Huang, Y. H., Macalinao, D. G., Brogi, E., Finley, L. W. and MassaguÃ©, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30912515" target="_blank">Flura-seq identifies organ-specific metabolic adaptations during early metastatic colonization.</a> <em>Elife</em> 8: e43627.</li><li>Bertin, B., Renaud, Y., Aradhya, R., Jagla, K. and Junion, G. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26381166" target="_blank">TRAP-rc, translating ribosome affinity purification from rare cell populations of <em>Drosophila</em> embryos.</a> <em>J Vis Exp</em>(103).</li><li>Cleary, M. D., Meiering, C. D., Jan, E., Guymon, R. and Boothroyd, J. C. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15685165" target="_blank">Biosynthetic labeling of RNA with uracil phosphoribosyltransferase allows cell-specific microarray analysis of mRNA synthesis and decay.</a> <em>Nat Biotechnol</em> 23(2): 232-237.</li><li>Gay, L., Karfilis, K. V., Miller, M. R., Doe, C. Q. and Stankunas, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24457332" target="_blank">Applying thiouracil tagging to mouse transcriptome analysis.</a> <em>Nat Protoc</em> 9(2): 410-420.</li><li>Gay, L., Miller, M. R., Ventura, P. B., Devasthali, V., Vue, Z., Thompson, H. L., Temple, S., Zong, H., Cleary, M. D., Stankunas, K. and Doe, C. Q. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23307870" target="_blank">Mouse TU tagging: a chemical/genetic intersectional method for purifying cell type-specific nascent RNA. </a><em>Genes Dev</em> 27(1): 98-115.</li><li>Heiman, M., Schaefer, A., Gong, S., Peterson, J. D., Day, M., Ramsey, K. E., SuÃ¡rez-FariÃ±as, M., Schwarz, C., Stephan, D. A., Surmeier, D. J., Greengard, P. and Heintz, N. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19013281" target="_blank">A translational profiling approach for the molecular characterization of CNS cell types. </a><em>Cell</em> 135(4): 738-748.</li><li>Hida, N., Aboukilila, M. Y., Burow, D. A., Paul, R., Greenberg, M. M., Fazio, M., Beasley, S., Spitale, R. C. and Cleary, M. D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28641402" target="_blank">EC-tagging allows cell type-specific RNA analysis. </a><em>Nucleic Acids Res </em>45(15): e138.</li><li>Machado, L., Esteves de Lima, J., Fabre, O., Proux, C., Legendre, R., Szegedi, A., Varet, H., Ingerslev, L. R., Barres, R., Relaix, F. and Mourikis, P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29141227" target="_blank"><em>In situ</em> fixation redefines quiescence and early activation of skeletal muscle stem cells. </a><em>Cell Rep</em> 21(7): 1982-1993.</li><li>Matsushima, W., Herzog, V. A., Neumann, T., Gapp, K., Zuber, J., Ameres, S. L. and Miska, E. A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29945865" target="_blank">SLAM-ITseq: sequencing cell type-specific transcriptomes without cell sorting. </a><em>Development</em> 145(13). pii: dev164640.</li><li>Miller, M. R., Robinson, K. J., Cleary, M. D. and Doe, C. Q. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19430475" target="_blank">TU-tagging: cell type-specific RNA isolation from intact complex tissues.</a> <em>Nat Methods</em> 6(6): 439-441.</li><li>Mullen, C. A., Kilstrup, M., and Blaese, R. M. (1992). <a href="https://www.ncbi.nlm.nih.gov/pubmed/1729703" target="_blank">Transfer of the bacterial gene for cytosine deaminase to mammalian cells confers lethal sensitivity to 5-fluorocytosine: a negative selection system.</a> <em>Proc Natl Acad Sci U S A</em> 89(1): 33-37.</li><li>Obenauf, A. C., Zou, Y., Ji, A. L., Vanharanta, S., Shu, W., Shi, H., Kong, X., Bosenberg, M. C., Wiesner, T., Rosen, N., Lo, R. S. and MassaguÃ©, J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25807485" target="_blank">Therapy-induced tumour secretomes promote resistance and tumour progression.</a> <em>Nature</em> 520(7547): 368-372.<br/></li></ol>