<body>
            <h1>Method for Assessing Virulence of <em>Colletotrichum higginsianum</em> on <em>Arabidopsis thaliana</em> Leaves Using Automated Lesion Area Detection and Measurement</h1>
            <p><strong>Authors:</strong> Ayako Tsushima, Pamela  Gan, Ken Shirasu</p>
            <h2>Abstract</h2>
            <p>The plant pathogenic fungus, <em>Colletotrichum higginsianum</em> is widely used to understand infection mechanisms, as it infects the model plant <em>Arabidopsis thaliana</em>. To determine the virulence of <em>C. higginsianum</em>, several methods have been developed, such as disease reaction scoring, lesion measurement, entry rate assays, and relative fungal biomass assays using real-time quantitative PCR. Although many studies have taken advantage of these methods, they have shortcomings in terms of objectivity, time, or cost. Here, we show a lesion area detection method applying ImageJ color thresholds to images of <em>A. thaliana</em> leaves infected by <em>C. higginsianum</em>. This method can automatically detect multiple lesions in a short time without the requirement for special equipment and measures lesion areas in a standardized way. This high throughput technique will aid better understanding of plant immunity and pathogenicity and contribute to reproducibility of assays.</p>
            <p><strong>Keywords:</strong> <em>Colletotrichum higginsianum</em>, Fungal phytopathogen, <em>Arabidopsis thaliana</em>, Infection assay, ImageJ</p>
            <h2>Background</h2><p><em>Colletotrichum</em> fungi cause anthracnose disease in a broad range of plants, including important crops, and have a serious economic impact (Cannon <em>et al.</em>, 2012). Among <em>Colletotrichum</em> species, <em>C. higginsianum</em> infects Brassicaceae plants, including the model plant <em>Arabidopsis thaliana</em> (OâConnell <em>et al.</em>, 2004). Thus, this pathogen has been used for various studies that have revealed molecular mechanisms of plant immunity and pathogenicity (Birker <em>et al.</em>, 2009; Narusaka <em>et al.</em>, 2009; Kleemann <em>et al.</em>, 2012; Takahara <em>et al.</em>, 2016; Plaumann <em>et al.</em>, 2018). Moreover, recently released high-quality genome assemblies of <em>C. higginsianum</em> provide genomic resources that should increase the use of this fungus in functional studies by facilitating the identification of candidate genes of interest (Zampounis <em>et al.</em>, 2016; Tsushima <em>et al.</em>, 2019). In order to determine the virulence of <em>C. higginsianum</em>, initial studies employed disease reaction scores based on visual observation and subjective judgement (OâConnell <em>et al.</em>, 2004; Narusaka <em>et al.</em>, 2004). The necessity for more objective approaches has led to the development of alternative methods including lesion diameter measurement, entry rate assays, and measurement of relative fungal biomass using real-time quantitative PCR (Narusaka <em>et al.</em>, 2010; Hiruma and Saijo, 2016). These methods are commonly employed in studies using <em>C. higginsianum</em>, yet they have disadvantages, such as the variability of lesion shapes leading to differences in measured values between observers, time-consuming steps, or the economic burden of purchasing reagents. ImageJ has been used as an open source tool for the analysis of scientific images (Schneider <em>et al.</em>, 2012). In the field of plant pathology, many studies have utilized ImageJ to analyze pathogenicity or resistance, for instance by measuring lesion sizes (Dagdas <em>et al.</em>, 2016; Kumakura <em>et al.</em>, 2019). However, these procedures can be further automated. Therefore, we developed an objective, automated, and affordable method to detect and measure lesion areas from images of <em>A. thaliana</em> leaves infected by <em>C. higginsianum</em> using ImageJ. In this article, we describe how to perform infection assays to study the <em>A. thaliana</em>-<em>C. higginsianum</em> interaction and how to detect and measure lesions using ImageJ macros to record multiple lesion areas at one time. This method enables us to reproducibly perform infection assays with the <em>A. thaliana</em>-<em>C. higginsianum</em> pathosystem and may be adapted to other pathosystems. For example, ImageJ color thresholds were also used to measure lesion areas on <em>C. shisoi</em>-infected <em>A. thaliana</em> (Gan <em>et al.</em>, 2019). This protocol will allow us to perform objective and high throughput analyses to gain further insights into plant-microbe interactions.</p>
        </body>

<p class="pptt" id="biaoti30326">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml conical tube (<em>e.g.</em>, Falcon 50 ml Conical Centrifuge Tubes, Corning Inc., catalog number: 352070)</li><li>Nylon mesh (100 Î¼m pore size) cut into 11 x 11 cm square</li><li>Surgical tape </li><li>1.5 ml microcentrifuge tube (<em>e.g.</em>, 1.5 ml Sampling Tubes, Round Bottom, FUKAE KASEI Co., Ltd., catalog number: 131-615C)</li><li>Petri dish<br/></li><li>Paper towel</li><li>Permanent marker</li><li><em>A. thaliana</em> plants grown at 22 Â°C with a 10-h photoperiod for 4 weeks<br/><em>Note: We recommend including genotypes known to show clear resistant and susceptible phenotypes as controls. In the case of infection assays using </em><em>C. higginsianum</em><em> MAFF 305635, Ws-2 and Ler-0 can be used as controls for resistance and susceptible phenotypes, respectively.</em> </li><li><em>C. higginsianum</em> strains</li><li>Potato dextrose agar (Potato Dextrose Agar, Nissui Pharmaceutical Co., Ltd., catalog number: 05709) prepared in Petri dishes</li><li>Sterilized water<br/></li></ol><p class="pptt" id="biaoti30327">### Equipment</p><ol class="npwtwok"><li>Airtight transparent plastic container with a rubber seal lid</li><li>Painting brush (autoclaved) (<em>e.g.</em>, Nylon watercolor brush (brown hair) flat, Artec Co., Ltd., catalog number: 10629)<br/><em>Note: Any painting brush can be used. We autoclave painting brushes to eliminate any contaminants from previous experiments and reuse them.</em> </li><li>Funnels (autoclaved)<br/><em>Note: We autoclave funnels for the same reason of painting brushes.</em> </li><li>Centrifugal machine (<em>e.g.</em>, High speed refrigerated micro centrifuge, TOMY SEIKO Co., Ltd., catalog number: MX-307)</li><li>Incubator (dark condition, 24 Â°C) (<em>e.g.</em>, Cool incubator, Mitsubishi Electric Corporation, catalog number: CN-25C)</li><li>Growth chamber with BLB light (light/dark = 12 h/12 h, 24 Â°C) (<em>e.g.</em>, Growth cabinet, SANYO Electric Co., Ltd., catalog number: MLR-350 and Blacklight blue, Hitachi, Ltd., catalog number: FL40SBLB)</li><li>Hemacytometer (<em>e.g.</em>, Reichert Bright-Line, Hausser Scientific Co., catalog number: 1492)</li><li>Light microscope (<em>e.g.</em>, Binocular Microscope, Olympus Corporation, catalog number: CX31)</li><li>Pipette that can measure 5 Î¼l (<em>e.g.</em>, PIPETMAN Classic P20, Gilson Inc., catalog number: F123600)</li><li>Spray bottle (500 ml)</li><li>Digital camera (<em>e.g.</em>, EOS Kiss X6i, Canon Inc., catalog number: 6557B001)</li><li>Camera stand (<em>e.g.</em>, Copy stand CS-A4, LPL Co., Ltd., catalog number: L18142)</li><li>Scissors</li><li>Forceps</li><li>White plastic tray</li><li>Scale<br/></li></ol><p class="pptt" id="biaoti30328">### Software</p><p class="ppzz">1.ImageJ v1.52p (Rasband, W.S., ImageJ, U. S. National Institutes of Health, Bethesda, Maryland, USA, <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>, 1997-2018)</p><p class="pptt" id="biaoti30329">### Procedure</p><ol class="npwone"><li>Preparation of <em>A. thaliana</em> plants and <em>C. higginsianum</em> culture<ol class="npwtwo"><li>Culture fungal strains on potato dextrose agar at 24 Â°C under 12-h black-light blue fluorescent bulb light/12-h dark conditions for 1 week (Figure 1A).</li><li>Set up at least 8 replicate <em>A. thaliana</em> plants for each genotype to be tested in airtight transparent plastic containers with about 1 cm of water (Figure 1B).<br/><em>Note: The lid of this transparent plastic container should have a rubber seal to maintain 100% humidity conditions during infection.</em> </li><li>Mark 3 petioles of fully expanded leaves per plant using a permanent marker (Figure 1C).<br/><em>Note: We recommend completing all the steps given above before starting to prepare conidia suspensions.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114173407_6968.jpg" width="307"/><br/><strong>Figure 1. Preparation of <em>A. thaliana</em> plants and <em>C. higginsianum</em> culture.</strong> A. 7-day-old culture of <em>C. higginsianum</em> MAFF 305635 on potato dextrose agar incubated at 24 Â°C under 12-h black-light blue fluorescent bulb light/12-h dark conditions. B. Airtight transparent plastic container with <em>A. thaliana</em> plants. C. Marked petioles using a permanent marker (White arrows).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Inoculation with <em>C. higginsianum</em> <ol class="npwtwo"><li>Place an autoclaved filter funnel in a 50 ml conical tube and fix a folded piece of nylon mesh to the funnel using a piece of surgical tape (Figure 2A).</li><li>Add 10 ml of room temperature sterilized water to the fungal culture in a Petri dish and gently scrape the surface using an autoclaved painting brush to release conidia (Figure 2B).</li><li>Pour the conidial suspension through the nylon mesh in the funnel to separate mycelia from conidia (Figure 2C).<br/><em>Note: Conidia should pass through the nylon mesh, but mycelia should not.</em> </li><li>After centrifugation at 4,500 <em>x g</em> for 5 min at room temperature, remove the supernatant by decanting (Figure 2D).</li><li>Resuspend the conidial pellet in 2 ml of distilled water.</li><li>Prepare a 1/100 diluted conidia suspension in distilled water in an Eppendorf<sup>Â®</sup> microcentrifuge tube and determine the concentration of the original suspension using a hemacytometer (Figure 2E).<br/><em>Note: Depending on the conidial concentration, you can prepare different dilution series to obtain ~100 conidia/mm<sup>2</sup> on a hemacytometer.</em> </li><li>Prepare at least 1 ml of 5 x 10<sup>5</sup> conidia/ml suspension in an Eppendorf<sup>Â®</sup> microcentrifuge tube.</li><li>Open the lid of the transparent plastic container and inoculate 5 Î¼l of the conidia suspension per leaf on the surface of the selected leaves using a pipette (Figure 2F).<br/><em>Note: We recommend inoculating plants where they will be grown during pathotests to minimize the risk of the inoculated conidia suspension running off from the leaf surface during transportation.</em> </li><li>Spray tap water on the backside of a lid of a transparent plastic container using a spray bottle. Then, seal the lid of the container and carefully place the container in the growth chamber.<br/><em>Note: Do not open the lid until 5 days post-infection to maintain 100% humidity during infection.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114173430_2162.jpg" width="150"/><br/><strong>Figure 2. Inoculation with <em>C. higginsianum</em>. </strong>A. Setup to separate mycelia from conidia. B. Scraping the fungal culture using a painting brush to obtain conidia suspension. C. Filtering the conidia suspension through nylon mesh. D. Conidial pellet after centrifugation. E. Conidia on a hemacytometer (Black arrows). Bar = 100 Î¼m. F. Inoculation of conidia on the leaf surface using a pipette.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparing images of symptoms<ol class="npwtwo"><li>Set up a digital camera on a camera stand and set magnification, aperture, shutter speed, and ISO.<br/><em>Note: Adjust your camera and exposure settings to minimize shadows and reflections in the images. Make sure to use the same settings for reproducibility when you repeat experiments. For this reason, we also recommend taking photos in a room without windows to avoid the effect of sunshine.</em> </li><li>Open the lid of the transparent plastic container. If necessary, gently wipe water droplets on the leaf surface using a paper towel to avoid capturing water-reflected light.</li><li>Cut the marked petioles, put leaves on a white tray and take photos.<br/><em>Note: The white tray provides a uniform white background for the images. Make sure to include a ruler in images of inoculated leaves.</em> </li></ol></li></ol><p class="pptt" id="biaoti30330">### Data analysis</p><p class="ppzz"><em>Note: For data analysis using ImageJ, you can also refer to the video tutorials provided in this article (Videos 1 and 2).</em><br/><br/><!--video3434-1å¼å§--></p><div class="videodiv" id="vd3434-1" style="height:480px;"></div><div class="videophoto" id="vp3434-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3434/3434-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. How to analyze images of infected leaves using ImageJ</strong> </div><!--video3434-1ç»æ--><br/><!--video3434-2å¼å§--><div class="videodiv" id="vd3434-2" style="height:480px;"></div><div class="videophoto" id="vp3434-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3434/3434-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. How to visualize obtained data in a beeswarm boxplot using R</strong> </div><!--video3434-2ç»æ--><p><br/></p><ol class="npwtwo"><li>Install ImageJ in your environment following the instructions at <a href="https://imagej.nih.gov/ij/download.html" target="_blank">https://imagej.nih.gov/ij/download.html</a>.<br/><em>Note: The procedure described below has been tested under Windows10 version 1903 and macOS Mojave 10.14.5. </em> </li><li>Download ImageJ macros <a href="https://os.bio-protocol.org/attached/file/20191104/ijm files.zip" target="_blank">lesion.ijm and lesion_loop.ijm</a> and save them in the same directory.</li><li>Open a photo of infected leaves in ImageJ. File &gt; Open &gt; Select your photo or drag-and-drop your image file on to the ImageJ main menu bar.</li><li>Open the ROI Manager. Analyze &gt; Tools &gt; ROI Manager.</li><li>Set each leaf as a region of interest (ROI) on the photo using the rectangle tool and add this ROI by pressing âAddâ in the ROI Manager (Figure 3A).<br/><em>Note: To add each ROI to the ROI Manager, you can use the short cut key. (In default settings, it is [t].)</em> </li><li>Save ROIs as a .zip file in your preferred directory. In the ROI Manager: More &gt;Save &gt; Select the destination directory.<br/><em>Note: You can load the ROIs by drag-and-drop of your .zip file on to the ImageJ main menu bar.</em> </li><li>Open the Threshold Color menu and set your color space as HSB. Image &gt; Adjust &gt; Color Threshold &gt; Select the color space as HSB.</li><li>Find the thresholds to define lesions in the photo by changing HSB thresholds (Figure 3B). After confirming the appropriate thresholds, click âOriginalâ and close the Threshold Color menu.<br/><em>Note: In this step, you can define your thresholds by referring to images of control genotypes that are known to show resistance and susceptible phenotypes. The settings used in Tsushima </em><em>et al.</em><em> (2019) are as follows: hue, 0-255; saturation, 110-140; and brightness, 0-255. We normally adjust only the saturation values from the default settings to define lesion areas. After customizing the settings that work for your images, make sure to use the same settings when you repeat analyses.</em><br/></li><li>Open lesion.ijm in a text editor, edit the thresholds according to the settings identified in Step 8 and save. In lesion.ijm, min[0] and max[0], min[1] and max[1], and min[2] and max[2] indicate the minimum and maximum thresholds for hue, saturation, and brightness, respectively (Figure 3C).</li><li>Install lesion.ijm and lesion_loop.ijm in your ImageJ. Plugins &gt; Macros &gt; Install &gt; Select lesion.ijm or lesion_loop.ijm</li><li>Measure 10 mm on the scale in your photo by using the straight tool. Then, set the scale in the dialog box. Analyze &gt; Set Scale &gt; Enter a known distance (10) and the unit of length (mm).<br/><em>Note: If you take your photos with the same magnification, you can tick âGlobalâ in the dialog box and apply this scale to all subsequent images analyzed.</em> </li><li>Open the âSet Measurementsâ dialog box and tick âAreaâ, âMean gray valueâ and âLimit to thresholdâ. Analyze &gt; Set Measurements &gt; tick in the dialog box.</li><li>Run the ImageJ macro. Plugins &gt; Macros &gt; Run &gt; Select lesion_loop.ijm.<br/><em>Note: You can assign a short cut key for running the macro. Plugins &gt; Shortcuts &gt; Add Shortcut.</em> </li><li>Save results as a comma-delimited (.csv) file. In the Results: More &gt; Save &gt; Select the directory where the results should be saved.</li><li>Repeat to open the next photo, set ROIs, run lesion_loop.ijm and save the data until you have processed all images. </li><li>Analyze the detected lesion areas, for example, using R or Excel. The following describes an example of how to visualize the obtained data in a beeswarm boxplot using R.<br/><em>Note: We provide sample files in </em><a href="https://os.bio-protocol.org/attached/file/20191104/sample files.zip" target="_blank"><em>sample_files.zip</em></a><em> to test all data analysis steps. This includes images of </em><em>C. higginsianum</em><em>-infected </em><em>A. thaliana</em><em> leaves (Ler-0_ChWT.JPG and Ws-2_ChWT.JPG), ROI files (Ler-0_ChWT.zip and Ws-2_ChWT.zip), raw data of detected lesion areas obtained from ImageJ (Ler-0_ChWT.csv and Ws-2_ChWT.csv), lesion area data reformatted for use as input for R analysis (Ws_Ler.csv), R code to generate the beeswarm plots in Figure 3D (beeswarm.Rmd), and the expected output of running the R code (Lesion_area.pdf).</em> </li><li>Prepare a .csv file with the lesion area data.<br/><em>Note: Please format the file as in Ws_Ler.csv</em> </li><li>Open RStudio and install âbeeswarmâ and âggsciâ packages by typing install.packages ("beeswarm", dependencies = TRUE) and install.packages ("ggsci", dependencies = TRUE).<br/><em>Note: Once you install these two packages, you do not need to install them from next time.</em> </li><li>Set the directory containing the .csv file as your working directory by typing setwd ("PATH_TO_DIRECTORY") in the console.</li><li>In RStudio, open beeswarm.Rmd included in sample_files.zip. and change the .csv file name in Line 21 if necessary. Then, click âRun Current Chunkâ. This will create the âLesion_area.pdfâ file in your working directory. Figure 3D shows the expected result using the sample files provided in this article.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114173622_7135.jpg" width="149"/><br/><strong>Figure 3. Data analysis using color thresholds in ImageJ.</strong> A. Setting regions of interest (ROIs) on a photo. Yellow rectangles indicate ROIs that will be measured with color thresholds. Arrow heads indicate the rectangle tool and the straight tool. B. The Threshold Color menu of ImageJ. Red-colored parts in ROIs will be detected as lesions. C. Lesion.ijm opened in a text editor indicating the lines specifying the minimum and maximum thresholds of hue, saturation, and brightness, respectively. D. An expected result using the sample files provided in this article.<br/></li></ol><p class="pptt" id="biaoti30331">### Acknowledgments</p><p class="ppzz">This protocol was derived from our published work (Tsushima <em>et al.</em>, 2019). This work was supported in part by JSPS Grant-in-Aid for JSPS Research Fellow to A.T. (17J02983) and KAKENHI (17H06172 to K.S. and 19K15846 to P.G.).</p><p class="pptt" id="biaoti30332">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti30333">### References</p><ol class="npwtwok"><li>Birker, D., Heidrich, K., Takahara, H., Narusaka, M., Deslandes, L., Narusaka, Y., Reymond, M., Parker, J. E. and OâConnell, R. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19686535" target="_blank">A locus conferring resistance to <em>Colletotrichum higginsianum</em> is shared by four geographically distinct <em>Arabidopsis</em> accessions.</a> <em>Plant J</em> 60(4): 602-613.</li><li>Cannon, P. F., Damm, U., Johnston, P. R. and Weir, B. S. (2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3458418/" target="_blank"><em>Colletotrichum</em> - current status and future directions.</a> <em>Stud Mycol</em> 73: 181-213.</li><li>Dagdas, Y. F., Beihaj, K., Maqbool, A., Chaparro-Garcia, A., Pandey, P., Petre, B., Tabassum, N., Cruz-Mireles, N., Hughes, R. K., Sklenar, J., Win, J., Menke, F., Findlay, K., Banfield, M. J., Kamoun, S. and Bozkurt, T. O. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26765567" target="_blank">An effector of the irish potato famine pathogen antagonizes a host autophagy cargo receptor.</a> <em>ELife</em> 5: e10856.</li><li>Gan, P., Tsushima, A., Hiroyama, R., Narusaka, M., Takano, Y., Narusaka, Y., Kawaradani, M., Damm, U. and Shirasu, K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31527702" target="_blank"><em>Colletotrichum shisoi</em> sp. nov., an anthracnose pathogen of Perilla frutescens in Japan: molecular phylogenetic, morphological and genomic evidence.</a> <em>Sci Rep</em> 9(1): 13349.</li><li>Hiruma, K. and Saijo, Y. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26867633" target="_blank">Plant inoculation with the fungal leaf pathogen <em>Colletotrichum higginsianum</em>.</a> <em>Methods Mol Biol</em> 1398: 313-318.</li><li>Kleemann, J., Rincon-Rivera, L. J., Takahara, H., Neumann, U., van Themaat, E. V. L., van der Does, H. C., Hacquard, S., StÃ¼ber, K., Will, I., Schmalenbach, W., Schmelzer, E. and OâConnell, R. J. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22496661" target="_blank">Sequential delivery of host-induced virulence effectors by appressoria and intracellular hyphae of the phytopathogen <em>Colletotrichum higginsianum</em>.</a> <em>PLoS Pathog</em> 8(4): e1002643.</li><li>Kumakura, N., Ueno, A. and Shirasu, K. (2019). <a href="https://onlinelibrary.wiley.com/doi/full/10.1111/mpp.12766" target="_blank">Establishment of a selection marker recycling system for sequential transformation of the plant-pathogenic fungus <em>Colletotrichum</em> <em>orbiculare</em><em>.</em></a> <em>Mol Plant Pathol </em>20(3): 447-459.</li><li>Narusaka, M., Shiraishi, T., Iwabuchi, M. and Narusaka, Y. (2010). <a href="https://link.springer.com/article/10.1007/s10327-009-0211-z" target="_blank">Monitoring fungal viability and development in plants infected with <em>Colletotrichum higginsianum</em> by quantitative reverse transcription-polymerase chain reaction.</a> <em>J General Plant Pathol</em> 76(1): 1-6.</li><li>Narusaka, M., Shirasu, K., Noutoshi, Y., Kubo, Y., Shiraishi, T., Iwabuchi, M. and Narusaka, Y. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19519800" target="_blank"><em>RRS1</em> and <em>RPS4</em> provide a dual <em>Resistance</em>-gene system against fungal and bacterial pathogens.</a> <em>Plant J</em> 60(2): 218-226.</li><li>Narusaka, Y., Narusaka, M., Park, P., Kubo, Y., Hirayama, T., Seki, M., Shiraishi, T., Ishida, J., Nakashima, M., Enju, A., Sakurai, T., Satou, M., Kobayashi, M. and Shinozaki, K. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15242169" target="_blank"><em>RCH1</em>, a locus in <em>Arabidopsis</em> that confers resistance to the hemibiotrophic fungal pathogen <em>Colletotrichum higginsianum</em>.</a> <em>Mol Plant-Microbe Interact</em> 17(7): 749-762.</li><li>OâConnell, R., Herbert, C., Sreenivasaprasad, S., Khatib, M., EsquerrÃ©-TugayÃ©, M.-T. and Dumas, B. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15000394" target="_blank">A novel <em>Arabidopsis</em>-<em>Colletotrichum</em> pathosystem for the molecular dissection of plant-fungal interactions.</a> <em>Mol Plant-Microbe Interact</em> 17(3): 272-282.</li><li>Plaumann, P.-L., Schmidpeter, J., Dahl, M., Taher, L. and Koch, C. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29867895" target="_blank">A dispensable chromosome is required for virulence in the hemibiotrophic plant pathogen <em>Colletotrichum higginsianum</em>.</a> <em>Front Microbiol</em> 9: 1005.</li><li>Schneider, C. A., Rasband, W. S. and Eliceiri, K. W. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22930834" target="_blank">NIH Image to ImageJ: 25 years of image analysis.</a> <em>Nat Methods</em> 9(7): 671-675.</li><li>Takahara, H., Hacquard, S., Kombrink, A., Hughes, H. B., Halder, V., Robin, G. P., Hiruma, K., Neumann, U., Shinya, T., Kombrink, E., Shibuya, N., Thomma, B. P. H. J. and OâConnell, R. J. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27174033" target="_blank"><em>Colletotrichum higginsianum</em> extracellular LysM proteins play dual roles in appressorial function and suppression of chitin-triggered plant immunity.</a> <em>New Phytol</em> 211(4): 1323â1337.</li><li>Tsushima, A., Gan, P., Kumakura, N., Narusaka, M., Takano, Y., Narusaka, Y. and Shirasu, K. (2019). <a href="https://academic.oup.com/gbe/article/11/5/1487/5480666" target="_blank">Genomic plasticity mediated by transposable elements in the plant pathogenic fungus <em>Colletotrichum higginsianum</em>.</a> <em>Genome Biol Evol</em> 11(5): 1487-1500.</li><li>Zampounis, A., PignÃ©, S., Dallery, J., Wittenberg, A. H. J., Zhou, S., Schwartz, D. C., Thon, M. R. and OâConnell, R. J. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27540062" target="_blank">Genome sequence and annotation of <em>Colletotrichum higginsianum</em>, a causal agent of crucifer anthracnose disease.</a> <em>Genome Announc</em> 4(4): e00821-16.<br/></li></ol>