<body>
            <h1>Determination of Chromatin Accessibility in <em>Drosophila</em> Midgut Enterocytes by <em>in situ</em> 5mC Labeling</h1>
            <p><strong>Authors:</strong> Eliya Bitman-Lotan, Hector Rincon-Arano, Gal  Raz, Amir Orian</p>
            <h2>Abstract</h2>
            <p>Regulation of gene expression involves dynamic changes in chromatin organization, where in many cases open chromatin structure correlates with gene activation. Several methods enable monitoring changes in chromatin accessibility, including ATAC-seq, FAIRE-seq, MNase-seq and DNAse-seq methods, which involve Next-generation-sequencing (NGS). Focusing on the adult <em>Drosophila</em> differentiated gut enterocytes (ECs) we used a sequencing-free method that enables visualizing and semi-quantifying large-scale changes in chromatin structure using <em>in vitro</em> methylation assay with the bacterial CpG Methyltransferase, M. Sssl, that determine chromatin accessibility. In brief, as CpG methylation is minimal in differentiated somatic <em>Drosophila</em> cells, we used the bacterial M. SssI enzyme to methylate CpG dinucleotides <em>in situ</em> depending on their chromatin accessibility. The methylated dinucleotides are detected using 5mCytosine monoclonal antibody and nuclei are visualized microscopically. Thus, the 5mC method enables to monitor large-scale chromatin changes in heterogenic cellular tissue focusing on the cell type of interest and without the need for cell purification or NGS. </p>
            <p><strong>Keywords:</strong> Chromatin 5-methyl-Cytosine (5mC), M. SssI, Nuclear organization, <em>Drosophila</em>, Midgut, Enterocytes</p>
            <h2>Background</h2><p>The regulation of the differentiated state of cells requires active supervision and is established and maintained by âidentity supervisorsâ (Natoli, 2010; Holmberg and Perlmann, 2012). In part, these supervisors are transcription factors (TFs) that together with chromatin and architectural/scaffold proteins establish and maintain large-scale organization of the differentiated nucleus (Bitman-Lotan and Orian, 2018). In both mammals and <em>Drosophila</em>, the adult gut is a highly dynamic tissue where intestinal stem cells (ISCs) proliferate to self-renew or differentiate to give rise to mature differentiated gut polyploid enterocytes (ECs) (Jiang and Edgar, 2012; Buchon <em>et al.</em>, 2013; Guo <em>et al.</em>, 2016). In the differentiated ECs, the transcription factor Hey together with the <em>Drosophila</em> type A lamin, Lamin-C, co-regulate enhancers activity and large-scale nuclear organization that prevents the expression of progenitor and non-relevant programs (Flint-Brodsly <em>et al.</em>, 2019). Genetic ablation of Hey in young ECs, or due to its decline during aging lead to a failure to express EC programs, including a decrease in LamC protein. Loss of LamC resulted in the ectopic expression of non-relevant gene programs and re-organization of the nucleus. These large-scale changes in chromatin organization can be visualized using the 5mC method as shown in Figures 1A and 1B. Moreover, the 5mC method was also used to monitor changes in chromatin accessibility in other <em>Drosophila</em> tissues such as the ovary upon loss of upSETR, a chromatin regulator (Rincon-Arano <em>et al.</em>, 2012). Please note that while the method provides a qualitative approach to address chromatin organization in complex tissues, NGS-based approaches should be considered to confirm and fine map structural changes.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114182250_3288.jpg" width="500"/><br/><strong>Figure 1. Loss of Hey in ECs results in increased chromatin accessibility in polyploid cells.</strong>Confocal images of midguts expressing the indicated transgenes: A. Control midgut stained for 5mC without M. SssL-I enzyme. B. Control midgut with enzyme treatment. C. Midguts where Hey was knockdown in ECs. 5mC methylation is shown in red, DAPI marks DNA, scale bar = 10 Î¼M, dashed squares point to cells shown in the inset. D. Quantification of 5mC positive polyploid cells in similar setting to B and C. ** = P&lt;0.01 (adopted from Flint-Brodsly <em>et al.</em>, 2019).<br/></p><p class="ppzz"><br/><strong>General description of the method:</strong> The M. SssI methyltransferase modifies all cytosine residues (C5) within the double-stranded dinucleotide recognition sequence CG, and it has been used for nucleosome mapping from isolated nuclei, during transcription (Fatemi <em>et al.</em>, 2005; Bell <em>et al.</em>, 2010). As the <em>Drosophila</em> genome has only minimal endogenous 5-cytosine methylation (5mC), we established conditions that enable efficient modification of mCpG dinucleotides by M. SssI methyltransferase depending on chromatin accessibility in <em>Drosophila</em> guts <em>in situ</em>. After a mild fixation and efficient permeabilization of the guts, M. SssI-based CpG methylation is performed at different time points and the DNA is denatured to expose the modified DNA. The resulting methylation is observed using a 5m Cytosine specific monoclonal antibody (Bell <em>et al.</em>, 2010), and is low or absent in untreated ECs in wild-type tissues (Not shown). M. SssI-based 5mC methylation is only minimally observed in wildtype polyploid ECs (Figure 1A). However, it dramatically increases in polyploid cells resembling ECs upon loss of Hey (Figures 1B and 1C; Flint-Brodsly <em>et al.</em>, 2019). Quantification of the visual signal demonstrated that the 5mC signal was observed in 11% of control ECs, and in 24% of Hey-targeted ECs (Figure 1C, n = 395, 820 respectively, P&lt; 0.01). <br/><br/>The 5mC method described below (Outlines in Figure 2) is simple to use, reproducible, enables focusing on specific cells of interest and can be combined with linage-tracing methods such as G-TRACE (Evans <em>et al.</em>, 2009).<br/><br/>Moreover, it does not involve cell purification or NGS, and is semi-quantitative. However, it requires the appropriate controls, including scaling the duration of the methylation reaction, fixation, permeabilization conditions and enzyme adjustments depending on the tissue of choice. Additional semi-quantitative evaluation can be performed by extracting the DNA from treated tissues and using nuclei acid-specific ELISA or DNA-based dot blot to estimate the dynamic range of the 5mC methylation. Moreover, unlike NGS-based methods (Tsompana and Buck, 2014), it does not point to the genomic location of the accessible regions. <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114182331_9455.jpg" width="250"/><br/><strong>Figure 2. 5mC method flow chart</strong></p>
        </body>

<p class="pptt" id="biaoti30336">## Materials and Reagents</p><ol class="npwtwok"><li>Aluminum foil</li><li>Eppendorf tubes</li><li>Pipettes range of 0-10 Î¼l, 10-200 Î¼l, 200-1,000 Î¼l</li><li>Microscope mounting Slides (Sigma-Aldrich, catalog number S8902) </li><li>Cover slips (Thermo Fisher, catalog number 102455)</li><li>Albumin Bovine, Fraction V (MP biomedicals, catalog number: 0216006980)</li><li>CpG Methyltransferase (M. SssI) (Supplied with Buffer and S-adenosylmethionine [SAM])- (New England Bio-labs (NEB), catalog number: M0226L)</li><li>Alexa Fluor 568 goat anti-mouse IgG (H + L) (Invitrogen, catalog number: A11031)</li><li>5-Methylcytosine (5-mC) antibody (mAb) - clone 33D3 (Active Motif, catalog number: 39649)</li><li>Schneider's <em>Drosophila</em> Medium (Biological Industry, catalog number: 01-150-1A)</li><li>Dulbecco's Phosphate Buffered Saline (DPBS) (10x) (Biological industries, catalog number: 02-023-5A)</li><li>Pierce<sup>TM</sup> 16% Formaldehyde (w/v), Methanol-free (Thermo Scientific<sup>TM</sup>, catalog number: 28906)</li><li>Heptane (Sigma-Aldrich, catalog number: 246654)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: X100)</li><li>32% Hydrochloric acid (Bio-Lab, catalog number: 084605)</li><li>Borax anhydrous (Sigma-Aldrich, catalog number: 71997)</li><li>DAPI-for nucleic acid staining (Sigma-Aldrich, catalog number: D9542-1MG)</li><li>Fluoromount-G (Southern Biotech, catalog number: 0100-01)</li><li>HCl</li><li>Fix-I solution (1 ml) (see Recipes)</li><li>PBX (see Recipes)</li><li>PAT (see Recipes)</li><li>Pre-reaction Buffer (1 ml) (see Recipes)</li><li>M. SssI Reaction Buffer-100 Î¼l (25 U) (see Recipes)</li><li>Fix-II solution (1 ml) (see Recipes)</li><li>PBT (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30337">### Equipment</p><ol class="npwtwok"><li>Incubator for rearing <em>Drosophila</em> at consistent temperature of 29 Â°C</li><li>Stereomicroscope with light source for dissection (Leica Microsystems, model: MZ75)</li><li>Dumont tweezers #5 Biology 0.05 x 0.02 mm Inox (magnetic, FST11252-20)</li><li>Watch Glasses square for dissection (Carolina, catalog number: 742300) </li><li>Analog Orbital Shaker (ELMI S-3.02 10 L) </li><li>Confocal microscope (Zeiss, model: LSM-700)<br/></li></ol><p class="pptt" id="biaoti30338">### Procedure</p><ol class="npwone"><li>Flies collection<br/><em>Drosophila</em> flies are reared at 22 Â°C on standard medium. To prevent gender differences, collect only female flies at relevant age. For collecting young adults, we used 3-10 days old flies that were transferred to 29 Â°C for two days for activating the GAL4/Gal80 system and UAS-RNAi for gene knockdown. For more details regarding the GAL4/GAL80 system, see Salmeron <em>et al.</em> (1990) and Brand and Perrimon (1993). </li><br/><li>5mC methylation assay<ol class="npwtwo"><li>Dissect guts in watch glasses squares filled with cold Schneider medium. Transfer dissected guts to an Eppendorf tube containing 0.5 ml Schneider medium. Eppendorf tubes with dissected guts are kept on ice. Collect 8-10 guts per Eppendorf tube. For detailed gut dissection, see an excellent midgut protocol (Nawrot-Esposito <em>et al.</em>, 2017). </li><li>Remove Schneider media and add 1 ml cold DPBS to wash dissected guts (replace media only without incubation time).</li><li>Fix: Replace DPBS with 1 ml of freshly made "Fix-I solution" (Recipe 1), this will form two phases of liquids: the upper phase is Heptane and lower phase is the formaldehyde/DPBS solution at the bottom (See Figure 3 âFixâ). Rotate for 20 min on an orbital shaker at RT.<br/><em>Note: Over-crosslinking can cause reduced diffusion and accessibility of the enzyme to the nuclei. </em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191114/20191114182739_5823.jpg" width="150"/> <br/><strong>Figure 3. Typical Eppendorf tube after formation of the two phases</strong><br/><br/></li><li>Gently remove all the liquid solutions (the 2 phases) leaving only the dissected tissue at the bottom. Be sure to remove all heptane, also from the cap.</li><li>Wash guts three times with 1 ml DPBS, in each wash rotate the guts for 15 min at RT. </li><li>Permeabilization: Permeabilize the tissue by incubating with 500 Âµl of 0.5% Triton X-100 in DPBS for 1h on an orbital shaker in Eppendorf tubes, at RT (we recommend evaluating different permeabilization conditions [<em>e.g.</em>, 1 h, 2 h and triton concentrations (0.5%, 0.6%, 0.7%)]).</li><li>Gently remove Triton solution, and wash for three times with DPBS, in each wash rotate the guts for 15 min at RT.</li><li>Block in 1 ml "PAT solution" (Recipe 2) overnight at 4 Â°C on an orbital shaker.</li><li>Wash twice with freshly prepared 250 Î¼l of "Pre-reaction Buffer "(Recipe 3), 15 min each wash. </li><li>Methylation reaction: Gently remove "Pre-Reaction Bufferâ and resuspend in 50 Âµl of M. SssI reaction buffer (Recipe 4) and incubate for 1 h at 25 Â°C, on an orbital shaker.<br/><em><strong>Important: </strong>Titer the reaction conditions time (e.g., 1 h, 5 h, 24 h) and enzyme amount (25 U, 50 U) for optimal results.</em> </li><li>Termination: Add 1 ml of 2 N HCl and rotate on an orbital shaker at room temperature for 30 min. This step stops the M. SssI reaction and denatures the DNA.</li><li>Remove HCl solution and neutralize with 0.5 ml of freshly prepared 100 mM Borax (dissolved in water) and rotate on an orbital shaker for 5 min at RT.</li><li>Remove Borax and wash twice with 1 ml DPBS, rotate the guts for 15 min at RT each wash.</li><li>Fix with freshly prepared "Fix-II solution" (Recipe 5) and rotate for 10 min. </li><li>Remove Fix-II and wash twice with 1 ml "PBX solution" (Recipe 6), rotate the guts for 15 min at RT each wash. <br/><em>Note: Evaluate different time points (10, 15, 20 min) as over-crosslinking could affect antibody penetration.</em> </li><li>Remove PBX and block with 1 ml "PAT solution" (Recipe 2), for 30 min rotating at RT. </li><li>Remove blocking solution and add diluted anti-5mC primary antibody in "PAT solution" (Recipe 2), to a concentration of 1 Î¼g/ml (1:1,000). Incubate overnight at 4 Â°C on an Orbital shaker. </li><li>Remove primary antibody and wash three times with "PBT solution" (Recipe 7), rotating for 15 min each wash at RT.</li><li>Remove PBT and incubate in diluted Secondary Fluorescent anti-mouse (1:1,000) together with DAPI (1:1,000) (for DNA staining) in "PAT solution" (Recipe 2), cover the samples with aluminum foil or in a light protected box, and rotate on orbital shaker for 1 h in the dark. </li><li>Remove Secondary antibody and DAPI solution and wash three times with "PBT solution" (recipe 7), 15 min each wash while rotating in light protected environment (box or aluminum foil) at RT.</li><li>Wash twice with 1 ml DPBS, rotate the guts for 15 min at RT each wash.</li><li>Mount in a Fluoromount-G (25 ml) on slide and keep at 4 Â°C until confocal microscopic analysis.</li><li>Visualize under a confocal microscope, signal intensity in cells can be quantified. For quantification we suggest counting several hundreds of cells derived from different biological repeats, and in each scoring 5mC positive polyploid cells. </li></ol></li></ol><p class="pptt" id="biaoti30339">### Recipes</p><p class="ppzz">Prepare each solution fresh and keep no longer than a week at 4 Â°C or on Ice.</p><ol class="npwtwok"><li>Fix-I solution (1 ml)<br/>75 Î¼l 16% EM-grade formaldehyde <br/>12.5 Î¼l 10x DPBS<br/>112.5 Î¼l distilled water <br/>600 Î¼l Heptane</li><li>PAT<br/>1% BSA in PBX</li><li>Pre-reaction Buffer (1 ml) <br/>899.5 Î¼l distilled water <br/>100 Î¼l 10x NEB buffer (New England Biolab MsslI buffer)<br/>0.5 Î¼l SAM (S-adenosyl methionine from NEB, supplied with the enzyme)</li><li>M. SssI Reaction Buffer-100 Î¼l (25 U)<br/>6.25 Î¼l M. SssI <br/>83.25 Î¼l distilled water <br/>10 Î¼l 10x NEB<br/>0.5 Î¼l SAM</li><li>Fix-II solution (1 ml)<br/>50 Î¼l 16% EM-grade formaldehyde<br/>150 Î¼l 1x DPBS </li><li>PBX<br/>0.1% Triton X-100 in 1x DPBS</li><li>PBT<br/>0.1% BSA in PBX<br/></li></ol><p class="pptt" id="biaoti30340">### Acknowledgments</p><p class="ppzz">This original protocol was used in Rincon-Arano <em>et al.</em> (2012). The research was supported by Israel Science Foundation (ISF) (Grant 739/15), and the Rappaport Institute for Research in the Medical Sciences and Flinkman-Marandi cancer research grant to AO.</p><p class="pptt" id="biaoti30341">### Competing interests</p><p class="ppzz">We declare that there is no financial or non-financial conflict of interest, or competing interests related to this work to any of the authors.</p><p class="pptt" id="biaoti30342">### References</p><ol class="npwtwok"><li>Bell, O., Schwaiger, M., Oakeley, E. J., Lienert, F., Beisel, C., Stadler, M. B. and Schubeler, D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20562853" target="_blank">Accessibility of the <em>Drosophila</em> genome discriminates PcG repression, H4K16 acetylation and replication timing.</a> <em>Nat Struct Mol Biol</em> 17(7): 894-900.</li><li>Bitman-Lotan, E. and Orian, A. (2018). <a href="https://www.sciencedirect.com/science/article/pii/S245231001830012X" target="_blank">Chromatin, Nuclear Lamins, and maintenance of the differentiated identity.</a> <em>Curr Opin System Biol</em> 11: 1-8.</li><li>Brand, A. H. and Perrimon, N. (1993). <a href="https://www.ncbi.nlm.nih.gov/pubmed/8223268" target="_blank">Targeted gene expression as a means of altering cell fates and generating dominant phenotypes.</a> <em>Development</em> 118(2): 401-415.</li><li>Buchon, N., Osman, D., David, F. P., Fang, H. Y., Boquete, J. P., Deplancke, B. and Lemaitre, B. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23643535" target="_blank">Morphological and molecular characterization of adult midgut compart-mentalization in <em>Drosophila</em>.</a> <em>Cell Rep</em> 3(5): 1725-1738.</li><li>Evans, C. J., Olson, J. M., Ngo, K. T., Kim, E., Lee, N. E., Kuoy, E, Patananan, A. N., Sitz, D., Tran, P., Do, M. T., Yackle, K., Cespedes, A., Hartenstein, V., Call, G. B. and Banerjee, U. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19633663" target="_blank">G-TRACE: Rapid Gal4-based cell lineage analysis in <em>Drosophila</em>.</a> <em>Nat Methods</em> 6(8): 603-605.</li><li>Fatemi, M., Pao, M. M., Jeong, S., Gal-Yam, E. N., Egger G, Weisenberger, D. J. and Jones, P. A. (2005). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1292996/" target="_blank">Footprinting of mammalian promoters: use of a CpG DNA methyltransferase revealing nucleosome positions at a single molecule level.</a> <em>Nucleic Acids Res</em> 33(20): e176.</li><li>Flint-Brodsly, N., Bitman-Lotan, E., Boico, O., Shafat, A., Monastirioti, M., Gessler, M., Delidakis, C., Rincon-Arano, H. and Orian, A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31310235" target="_blank">The transcription factor Hey and nuclear lamins specify and maintain cell identity.</a> <em>Elife</em> 8. pii: e44745.</li><li>Guo, Z., Lucchetta, E., Rafel, N. and Ohlstein, B. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27392294" target="_blank">Maintenance of the adult <em>Drosophila</em> intestine: All roads lead to homeostasis.</a> <em>Curr Opin Genet Dev </em>40: 81-86. </li><li>Holmberg, J. and Perlmann, T. (2012). <a href="https://www.nature.com/articles/nrg3209" target="_blank">Maintaining differentiated cellular identity.</a> <em>Nat Rev Genet</em> 13: 429-439. </li><li>Jiang, H. and Edgar, B. A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22608824" target="_blank">Intestinal stem cell function in <em>Drosophila</em> and mice.</a> <em>Curr Opin Genet Dev</em> 22(4): 354-360.</li><li>Nawrot-Esposito, M., Loudhaief, R. and Gallet, A. (2017). <a href="https://bio-protocol.org/e2560" target="_blank">Protease activity assay in fly intestines.</a> <em>Bio-protocol </em>7(18): e2560.</li><li>Natoli, G. (2010). <a href="https://www.cell.com/immunity/abstract/S1074-7613(10)00248-7" target="_blank">Maintaining cell identity through global control of genomic organization.</a> <em>Immunity</em> 33: 12-24. </li><li>Rincon-Arano, H., Halow, J., Delrow, J. J., Parkhurst, S. M., and Groudine, M. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23177352" target="_blank">UpSET recruits HDAC complexes and restricts chromatin accessibility and acetylation at promoter regions.</a> <em>Cell</em> 151(6): 1214-1228.</li><li>Salmeron-Jr, J. M., Leuther, K. K. and Johnston, S. A. (1990). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1204005/" target="_blank">GAL4 mutations that separate the transcriptional activation and GAL80-interactive functions of the yeast GAL4 protein.</a> <em>Genetics</em> 125(1): 21-27.</li><li>Tsompana, M. and Buck, M. J. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25473421" target="_blank">Chromatin accessibility: a window into the genome.</a> <em>Epigenetics Chromatin</em> 7(1): 33. <br/></li></ol>