<body>
            <h1>Mapping the MechanomeâA Protocol for Simultaneous Live Imaging and Quantitative  Analysis of Cell Mechanoadaptation and Ingression</h1>
            <p><strong>Authors:</strong> Vina D. L. Putra, Iman Jalilian, Madeline Campbell, Kate  Poole, Renee  Whan, Florence  Tomasetig, Melissa L. Knothe Tate</p>
            <h2>Abstract</h2>
            <p>Mechanomics, the mechanics equivalent of genomics, is a burgeoning field studying mechanical modulation of stem cell behavior and lineage commitment. Analogous to mechanical testing of a living material as it adapts and evolves, mapping of the mechanome necessitates the development of new protocols to assess changes in structure and function in live stem cells as they adapt and differentiate. Previous techniques have relied on imaging of cellular structures in fixed cells and/or live cell imaging of single cells with separate studies of changes in mechanical and biological properties. Here we present two complementary protocols to study mechanobiology and mechanoadaptation of live stem cells in adherent and motile contexts. First, we developed and tested live imaging protocols for simultaneous visualization and tracking of actin and tubulin mechanoadaptation as well as shape and volume of cells and their nuclei in adherent model embryonic murine mesenchymal stem cells (C3H/10T1/2) and in a neuroblastoma cell line. Then we applied the protocol to enable quantitative study of primary human mesenchymal stem cells in a motile state, <em>e.g.</em>, ingression in a three-dimensional, <em>in vitro</em> cell culture model. Together, these protocols enable study of emergent structural mechanoadaptation of the cell's own cytoskeletal machinery while tracking lineage commitment using phenotypic (quantitative morphology measures) and genotypic (<em>e.g.</em>, reverse transcription Polymerase Chain Reaction, rtPCR) methods. These tools are expected to facilitate the mapping of the mechanome and incipient mechanistic understanding of stem cell mechanobiology, from the cellular to the tissue and organ length scales.</p>
            <p><strong>Keywords:</strong> Live cell imaging, Mechanoadaptation, Mechanomics, Cell ingression, Tubulin, Actin, Cytoskeletal dynamics, Dynamic cytoskeletal adaptation, Nucleus and cell shape and volume</p>
            <h2>Background</h2><p>Collectively referred to as <em>mechanomics</em>, the study of how mechanical cues modulate stem cell behavior has grown rapidly in the past decade (Figure 1). Experimental and computational mechanomics studies have demonstrated the profound influence of mechanical environment on cell motility (Knothe Tate <em>et al.</em>, 2008; Blanchoin <em>et al.</em>, 2014; Aubry <em>et al.</em>, 2015; Knothe Tate <em>et al.</em>, 2016; De Pascalis and Etienne-Manneville, 2017; Ladoux and Mege, 2017), stem cell niche quiescence (Yu <em>et al.</em>, 2017; Ni et al, 2019) and lineage commitment (Anderson <em>et al.</em>, 2006; Anderson and Knothe Tate 2007a and 2007b; Knothe Tate <em>et al.</em>,2008; McBride and Knothe Tate, 2008; McBride <em>et al.</em>, 2008; Song <em>et al.</em>, 2010; 2012 and 2013; Zimmermann <em>et al.</em>, 2011; Chang and Knothe Tate, 2011; Earls <em>et al.</em>, 2013; Heo <em>et al.</em>, 2015; Nimmo <em>et al.</em>, 2015; Steward and Kelly, 2015; Le <em>et al.</em>, 2016; Stumpf <em>et al.</em>,2017; Galarza <em>et al.</em>, 2018). Mechanical testing of live cells, as they adapt and differentiate, necessitates the development and testing of mechanobiological tools incorporating imaging and quantitative measures of shape, volume, and architecture changes (<em>e.g.</em>, of cells, as well as their nuclei and cytoskeletons) for controlled loading scenarios, with relevant spatial and temporal resolution (Anderson <em>et al.</em>, 2006; Anderson and Knothe Tate 2007a and 2007b; McBride and Knothe Tate, 2008; McBride <em>et al.</em>, 2008; Song <em>et al.</em>, 2010; 2012 and 2013; Zimmermann and Knothe Tate, 2011; Chang and Knothe Tate, 2011). <br/><br/>During development and postnatal healing, the mechanical cues to which stem cells are subjected exert a profound role in the cells' capacity to self-assemble structure (Figure 1), which manifests in several ways. The cell's internal structures such as the cytoskeleton self-assemble and adapt in response to the mechanical environment. Cells themselves self-assemble into multicellular constructs. Cells also create tissue architectures through transcription, assembly and post-translational modification of extracellular matrix (Knothe Tate <em>et al.</em>, 2016). <br/><br/>Prior to lineage commitment, stem cells act as sensors and actuators, transducing mechanical signals from the local environment to the nucleus, where gene transcription is up- and down-regulated, resulting in genesis of tissue templates that grow and mature over time (Knothe Tate <em>et al.</em>, 2008; Knothe Tate <em>et al.</em>, 2016; Ng <em>et al.</em>, 2017). Embryonic mesenchymal stem cells exhibit intrinsic sensitivity to mechanical stress; these cells change their baseline gene expression in response to subtle mechanical cues three orders of magnitude smaller than those to which <em>e.g.</em>, trigger changes in baseline gene expression of adult chondrocytes of the knee joint (McBride and Knothe Tate, 2008; McBride <em>et al.</em>, 2008; Song <em>et al.</em>, 2012). In addition, conditionally knocking out specific constituents of stem cellsâ mechanosensing apparatus causes stem cells to lose their capacity to self-assemble structure and hence tissues (Knothe Tate <em>et al.</em>, 2010). <br/><br/>The geometric arrangement of cells in space and time results in patterning of the organismâs template during prenatal development as well as the injured and/or missing tissue template during postnatal healing. The process is gated by cell motility and adherence (with adherence defined by the lack of capacity to move) (Knothe Tate <em>et al.</em>, 2008; Evans <em>et al.</em>, 2013). Until recently, mechanistic and dynamic study of unfolding cell fate and tissue template genesis has been stymied by the lack of methods to study these processes <em>in situ</em>, in live multicellular constructs. This protocol presents methods to study cell motility in model constructs as well as mechanoadaptation of the cytoskeleton within the cell itself. Pilot study data demonstrates the utility of the methods.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128005903_3966.jpg" width="149"/><br/><strong>Figure 1. Mapping the mechanome at the interface of genomics and mechanics of materials.</strong> Cell behavior is modulated by biophysical cues including volume (dilatationalâcompression and tension) and shape changing (deviatoricâshear) stresses. A. Mechanical loading throughout life literally shapes the structure and function of cells. Cells sense mechano-chemical stimuli and prototype tissue templates via up and downregulation of structural protein transcription, secretion into the extracellular matrix, and post-translational modification. This figure depicts characteristic magnitudes and time domains of mechanical signals applied in studies of multipotent cell differentiation, with nascent lineage commitment depicted by the shape of the data points. Tissue genesis and adaptation represents a continuum in space and time, over the life cycle of the individual organism, from development of the body template <em>in utero</em> (depicted 11.5 days after fertilization at the first stages of skeletogenesis in the mouse) and in the adult human. Image <em>after</em> (Ng <em>et al.</em>, 2017) as adapted from (Anderson and Knothe Tate 2017b; Song <em>et al.</em>, 2013) <em>and used with permission</em>. B. Using paired, live imaging and computational modeling approaches from our consortium, previous published studies mapped the mechanome <strong>retrospectively</strong>, depicting real stress state data from the coupled computational modeling and imaging studies in relation to 95% confidence intervals (shaded ovals) of nascent lineage commitment data (yellow, pink, violet), measured using rtPCR (Song <em>et al.</em>, 2010; 2012 and 2013; Ng <em>et al.</em>, 2017; Anderson and Knothe Tate, 2017b). Current and future approaches implement stress states from non-overlapping fates to <strong>prospectively</strong> test fate guidance through delivery of mechanical cues (B, <em>e.g.</em>, <strong>X</strong> indicates chondrogenesis, in yellow; +haematapoesis, in purple; +osteogenesis in fuschia) that induce volume (<strong>dilatational</strong>) and shape (<strong>deviatoric</strong>) changing stresses at cell surfaces. This is akin to conducting a mechanical test on a stem cell during the process of lineage commitment. <em>Adapted from original with permission</em> (Song <em>et al.</em>, 2013). C. An equivalent classical mechanics Mohrâs Circle diagram graphically represents the stress tensor obtained by performing a stress analysis on a material body such as a cell. Classical continuum mechanicsâ principal limitation in biology is that it cannot adequately address biological behavior of live materials that evolve over time, <em>e.g.</em>, via motility, mechanoadaptation, and differentiation (Knothe Tate <em>et al.</em>, 2011 and 2016). <br/><br/><strong>Overview of Approach</strong><br/> Live imaging methods were first developed for simultaneous tracking of actin and tubulin transcription and dynamic cytoskeletal mechanoadaptation, as well as changes in nucleus shape and volume, in adherent cells (Figure 2). This model system was designed to enable study of emergent structural mechanoadaptation of the cell's own cytoskeletal machinery while measuring emergent lineage commitment using phenotypic (morphology) and genotypic (rtPCR) methods. The C3H/10T1/2 cell line was used for the intracellular mechanoadaptation studies to insure cell-to-cell standardization. Derived from the mesenchyme, the C3H/10T1/2 murine pluripotent embryonic cells do not show phenotypic drift and are capable of differentiating along osteogenic, chrondogenic, adipogenic, smooth muscle, and endothelial lineage paths. This established cell line was used previously by our research consortium in a series of mechanoadaptation studies (McBride and Knothe Tate, 2008; McBride <em>et al.</em>, 2008; Song <em>et al.</em>, 2010; 2012 and 2013; Chang and Knothe Tate, 2011; Zimmermann and Knothe Tate, 2011). In addition, we tested the protocol on the rat neuroblastoma B35 cell line (Tpm3.1 neuroblastoma), an immortalized and well characterized cell line that displays prominent actin stress fibers (Bryce <em>et al.</em>, 2003). Finally, use of these two different cell lines enabled us to optimize the protocol, accurately and efficiently, for use in primary cell cultures as well as cell lines.<br/><br/>Thereafter, an <em>in vitro</em> model was developed for dynamic, live imaging and tracking of stem cells ingressing from a seeded surface to the interior of an idealized tissue template, mimicking an epithelial to mesenchymal transition (EMT) (Li <em>et al.</em> 2014) in a model tissue template (<em>Anlage</em>) during development or postnatal healing (Figure 2). To maximize clinical relevance, we tested the tissue ingression protocol using primary mesenchymal stem cells isolated from adult human periosteum (periosteum-derived stem cells, PDCs; Human ethics protocol approved) (Chang and Knothe Tate, 2011; Zimmermann and Knothe Tate, 2011) as well as bone marrow derived stem cells (BMSCs, adult human) from a commercial vendor.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128010207_5452.jpg" width="139"/><br/><strong>Figure 2. Protocol for mapping the mechanome at the interface of genomics and mechanics of materials, on cells adherent to tissue template scaffolds or functionalized coverslips (Part 1) or ingressing into idealized tissue templates (Part 2).</strong> Protocols steps are depicted in context of the aim to map the mechanome throughout cell and tissue genesis, in developmental and postnatal healing contexts where cells exist in adherent or motile states. <strong>Part 1. Mapping the mechanome</strong> by tracking changes in cell/nucleus shape &amp; volume and actin and tubulin as a function of gene transcription and mechanoadaptation, similar to a mechanical test of a living material as it evolves and adapts. <strong>Part 2. Energy of Mechanomics</strong> by live imaging and tracking of stem cell ingression from a seeded surface to the interior of an idealized tissue template as a function of cell metabolism, enabling assessment of the energy of mechanomics. The approach measures metabolic energy expended by the cell over time (power) as a function of distance covered by the motile cell.</p>
        </body>

<p class="pptt" id="biaoti30589">## Materials and Reagents</p><p class="ppzz"><em>Note: </em><sup><em>+</em></sup><em>Examples of items available from multiple vendors.<br/><br/></em> </p><ol class="npwtwok"><li>Eppendorf tube</li><li>T-75 culture flasks (<sup>+</sup>Corning, catalog number: 430641U)</li><li>25-mm plain glass coverslip</li><li>6-well plate (<sup>+</sup>Corning, catalog number: CLS 3516)</li><li>24-well plate (<sup>+</sup>Corning, catalog number: CLS 3524)</li><li>Petri dish (<sup>+</sup>Corning, catalog number: CLS 430166)</li><li>Serological pipette (5, 10, 25 ml) (<sup>+</sup>Corning, catalog numbers: CLS 4487, 4488, 4489)</li><li>Specimen jars (<sup>+</sup>Sardstedt, catalog number: SAR75.9922.534)</li><li>Falcon<sup>TM</sup> cell strainer (<sup>+</sup>Fischer Scientific)</li><li>Cryovials/cryogenic vials/cryotubes (<sup>+</sup>Fisher Scientific, catalog number: 12-565-169N)</li><li>Glass bottom dish (Cellvis, catalog number: D35-10-0-N)</li><li>Ovine femora (fresh from butcher)</li><li>Cells tested in the current protocol<br/><ol class="npwthree"><li>C3H/10T1/2 murine embryonic mesenchymal progenitor cell line (CCL-226, ATCC)</li><li>Rat neuroblastoma B35 cell line (Tpm3.1 neuroblastoma) cell line (after Bryce <em>et al.</em>, 2003; Bach <em>et al.</em>, 2010; Jalilian <em>et al.</em>, 2015)</li><li>PDSCs, Primary Periosteum Derived mesenchymal Stem Cells (after Knothe Tate, 2011; Evans <em>et al.</em>, 2013; Chang <em>et al.</em>, 2014)</li><li>BMSCs, Bone marrow derived Mesenchymal Stem Cells (Lonza, catalog number: PT 2501, lot no. 0000636886)<br/></li></ol></li><li>Fetal bovine serum (<sup>+</sup>Gibco, catalog number: 26400044)</li><li>L-glutamine (Sigma, catalog number: G7513-100ML)</li><li>Penicillin/streptomycin (<sup>+</sup>Sigma, catalog number: P4333-100ML)</li><li>DMSO (<sup>+</sup>Sigma, catalog number: 472301-100ML)</li><li>Trypsin-EDTA (<sup>+</sup>Sigma, catalog number: T4049-500ml)</li><li>Cellular Lights<sup>TM</sup> actin-GFP (CellLight<sup>Â®</sup> Actin-RFP, BacMam 2.0, Molecular Probes/Life Technologies, catalog number: C10583) </li><li>CellLight<sup>Â®</sup> Tubulin-GFP, BacMam 2.0 (Molecular Probes/Life Technologies, catalog number: C10613)</li><li>PBS (Phosphate-buffered saline) (UNSW Upper campus store, catalog number: UCS-BIO-0040)</li><li>Hanks buffer (Hoechst 33342, Trihydrochloride, Trihydrate) (Thermo Fisher Scientific, catalog number: H3570)</li><li>Collagenase II (Gibco, catalog number: 17101015)</li><li>Î±-minimal essential medium (Î±-MEM) with GlutaMAX (<sup>+</sup>Life Technologies, catalog number: 32561037)</li><li>Antibiotic-antimycotic (AMAB) (Gibco, catalog number: 15240062)</li><li>0.4% (wt/vol) trypan blue salt solution (<sup>+</sup>Sigma, catalog number: T8154-100 ML)</li><li>Liquid nitrogen</li><li>Matrigel<sup>Â®</sup>, phenol red free (Corning, catalog number: 356237)</li><li>Calcein-AM (<sup>+</sup>Sigma-Aldrich, catalog number: 17783-1MG)</li><li>MTT (Thiazolyl Blue Tetrazolium Bromide) Metabolic Assay (Sigma-Aldrich, catalog number: M5655-100MG)</li></ol><p class="pptt" id="biaoti30590">### Equipment</p><ol class="npwtwok"><li>Centrifuge</li><li>Humidified incubator</li><li>Hemocytometer</li><li>Perfusion and imaging chamber (<sup>+</sup>ProFlow Chamber, Warner Apparatus, PFC-1) </li><li>Leica SP8 DLS confocal laser scanning microscope or other fluorescent microscopes with tunable excitation and emission wavelengths</li><li>Plate reader (Perkin Elmer Victor 3, 570 nm filter)<br/></li></ol><p class="pptt" id="biaoti30591">### Software</p><ol class="npwtwok"><li>Statistics and graphical analysis software (<sup>+</sup>Prism 7 software, GraphPad)</li><li>Image analysis software (<sup>+</sup>ImageJ, Reference 36)<br/><br/></li></ol><p class="ppzzs"><strong>Part 1âTracking change in cell/nucleus shape &amp; volume, cytoskeletal transcription &amp; mechanoadaptation</strong><br/>Here we implemented multiplexed imaging and tracking of changes in the nucleus, actin filaments and microtubules, in live cells and at high resolution using a viral gene delivery system to insert the desired gene into the host's genome (Shoji <em>et al.</em>, 1997; Airenne <em>et al.</em>, 2003; Ho <em>et al.</em>, 2005; Kost <em>et al.</em>, 2005; Salminen <em>et al.</em>, 2005). The main advantage of the technique is that both actin and tubulin monomers are fluorescently tagged through the cellular transcriptional system and then individual monomers get incorporated into cytoskeletal structures by cells. This method offers rapid, safe, easy and convenient steps for delivery of desired gene to different mammalian cell lines but may not be efficient in certain cell types, including primary ovine Periosteum Derived Stem Cells, PDSCs (Chang and McBride, MechBio Team unpublished data). Here we optimized the method for the aforementioned C3H/10T1/2 murine embryonic stem cell line and the immortalized Tpm3.1 neuroblastoma cell lines. These two cell lines were chosen specifically because the transduction of two different cell types let us measure the amount of transduction efficiency and protein expression in a quantitative and robust way.</p><p class="pptt" id="biaoti30592">### Procedure</p><ol class="npwone"><li>Cell expansion (please refer to specific details recommended for specific cell lines of relevance to your particular study; the following are representative for the cell lines tested in the current protocol).<ol class="npwtwo"><li>Passage cells in Basal medium eagle supplemented with 10% fetal bovine serum, 1% L-glutamine, and 1% penicillin/streptomycin (Invitrogen) in T-75 culture flasks (Corning) and incubated at 37 Â°C at 5% CO<sub>2</sub> in a humidified incubator until passage 3 (P3). </li><li>At P3, freeze and store cells in cryovials at -80 Â°C in culture medium with 40% fetal bovine serum and 10% DMSO. </li><li>After storage, an additional passage allows cells to proliferate and enables all experiments to be conducted on P5 cells. </li><li>Wash cells once with PBS and detach cells using 0.25% trypsin-EDTA (Invitrogen, Carlsbad, CA) for 5 min.</li><li>As cell detachment is observed, add standard culture medium to the flask and transfer this cell suspension to a falcon tube. Centrifuge cells at 300 <em>x g</em> for 5 min and resuspend cells in fresh culture medium, and seed on glass coverslips (see B), placing one sterilized 25-mm treated plain glass coverslip or coated (<em>e.g.</em>, with gelatin or Poly L lysine) glass coverslip (Fisher Scientific, Hampton, NH) in each well of a six-well plate (Becton Dickinson, Frankin Lakes, NJ). <br/><em>Note: Our collaborative team has previously used radio frequency glow discharge (RFGD) to improve cell adhesion on plain glass coverslips in the absence of extracellular matrix proteins which can affect biological and mechanical outcome measures (Song <em>et al.</em>, 2010, 2012 and 2013). </em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell seeding and BacMam and viral transduction<ol class="npwtwo"><li><strong>At the time of cell seeding</strong>, fluorescent tagging of cell actin and tubulin is initiated using Cellular Lights<sup>TM</sup> actin-GFP (CellLight<sup>Â®</sup> Actin-RFP, BacMam 2.0, Molecular Probes/Life Technologies) and CellLight<sup>Â®</sup> Tubulin-GFP, BacMam 2.0 (Molecular Probes/Life Technologies). Add reagents simultaneously to the growth medium prior to seeding of the cells. Per manufacturer recommendations, 10-40 particles should be used per cell; however, based on the cell type and incubation time, the number of particles could be increased or reduced. The ratio of viral particles per cell can be calculated using the following equation:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203020038_8631.jpg" width="650"/><br/><br/></li><li>Seed cells at desired densities, measured using a hemocytometer. Our previous studies implemented different densities to mimic targeted developmental contexts, <em>e.g.</em>, low density (5,000 cells/cm<sup>2</sup>), high density (35,000 cells/cm<sup>2</sup>), and very high density (70,000 cells/cm<sup>2</sup>), either seeded at or proliferated to target density (McBride and Knothe Tate, 2008; McBride <em>et al.</em>, 2008; Zimmerman and Knothe Tate, 2011).</li><li>Add growth medium with cytoskeleton tagging agents to each well of the plate and gently mix. </li><li>Place in an incubator at 37 Â°C and 5% CO<sub>2</sub> overnight. Then, remove the solution from the well, followed by 2x wash with PBS and addition of fresh medium alone (without tagging agents) for another 24-72 h.<br/><br/></li></ol></li><li>Nucleus staining in live cells<ol class="npwtwo"><li>45-60 min prior to live imaging under the microscope, stain live cell nuclei with Hoechst stain, 1 Âµl/ml diluted in Hanks buffer (Hoechst 33342, Trihydrochloride, Trihydrate, Thermo Fisher Scientific). Gently mix the Hoechst stain with the growth medium and return the plate to the incubator (37 Â°C and 5% CO<sub>2</sub>) for another 45 min. </li><li>Wash each well with 4 ml of sterile PBS (2 x 2 ml). </li><li>Remove the PBS after the second wash.</li><li>Remove the cell-seeded coverslip and place in a perfusion and imaging chamber (Warner Apparatus, PFC-1, ProFlow Chamber) or other live imaging system that delivers controlled mechanical cues such as pressure, shear and normal stresses at fluid-cell interfaces, tension, <em>etc.</em> <br/><br/></li></ol></li><li>Fluorescent microscopy for live imaging of cell mechanoadaptation<ol class="npwtwo"><li>Using a high resolution laser scanning confocal or multiphoton microscope, cells, their nuclei and their respective actin and tubulin cytoskeletons are imaged in three dimensions, prior to, during, and/or after exposure to changes in mechanical environment within the flow chamber (ProFlow, Warner Apparatus). We used the Leica SP8 DLS confocal laser scanning microscope, tuned for the respective excitation and emission wavelengths specific to the fluorescent tag used for each cell feature of interest. </li><li>For cell and nucleus volume, surface area, and shape measurements, one or more randomly chosen fields of view were imaged per coverslip (on at least five coverslips to insure adequate sample size, as determined by power calculations), at 40x magnification (HCX APO L U-V-I 40 x 80 W). In previous studies, cell and nuclear volume as well as surface area were measured, enabling calculation of shape (Zimmerman and Knothe Tate, 2011). To measure shape independent of cell volume the surface area: volume is normalized to the ratio for a sphere with the same volume. The normalized SA/V gives a measure of the shape of a cell compared to a perfect sphere (1) with larger values indicating flatter, more spread cell shapes. Cell height can also serve as a surrogate for cell shape independent of volume and surface area measurements (Zimmerman and Knothe Tate, 2011).</li><li>For studies of actin and tubulin mechanoadaptation, images are acquired at time intervals appropriate for time scales of interest (thirty minute intervals in our previous separate studies of actin or tubulin adaptation; Zimmermann and Knothe Tate, 2011; Chang and Knothe Tate, 2011), using respective excitation and emission wavelengths for the fluorescent probes. Previous studies acquired images at 1024 x 1024 pixel resolution and 0.2 Î¼m intervals between focal planes.</li><li>The intensity of the fluorescence-tagged tubulin and actin provide a measure of concentration of actin and tubulin in space and time and adaptation of the cytoskeleton to mechanical environment; as noted previously (Chang and Knothe Tate, 2011), fluorescence intensity gives a measure of cytoskeletal quality and architecture akin to spatial and temporal patterns of mineral density in the bony skeleton, which provides a measure of mechanical adaptation (Knothe <em>et al.</em>, 2011; McBride <em>et al.</em>, 2011).</li><li>In this way, by accounting for cell and nucleus volume and shape as well as spatial distribution of actin and tubulin, adaptation of the cell can be measured in response to shifts in the mechanical environment brought about by volume and shape changing stresses. These environmental changes may be designed to mimic those occurring physiologically or may also be controlled in context of a targeted mechanical test of a cell and/or cellular construct, where a controlled force or stress is applied and the mechanical response of the cell is measured.<br/></li></ol></li></ol><p class="pptt" id="biaoti30593">### Data analysis</p><p class="ppzz"><strong>Representative data and protocol validation</strong><br/>Given the ubiquitous role of the cytoskeleton in cellular behavior, we first examined transduction efficiency for each of the cell lines as well as the impact of BacMam viral particle on their respective growth rates, implemented simultaneously previously tested protocols (Song <em>et al.</em>, 2010; 2012 and 2013; Chang and Knothe Tate, 2011; Zimmermann and Knothe Tate, 2011). The cytoskeleton, and in particular actin filaments, control the stiffness and hence modulate the mechanoadaptation of the cells. Since the baculovirus uses the actin filaments for intracellular transport, it was important to check for potential effects of viral transduction on the cell stiffness. To assess the effect of the viral transduction on the mechanical properties of cells, the stiffness of the cells was examined using atomic force microscopy (AFM) <em>per</em> our previous methods (Jalilian <em>et al.</em>, 2015).<br/>  The transduction of the cytoskeletal tagging agents was qualitatively apparent (Figure 3). The efficiency of transduction was proportional to growth rate, indicating that incubation time for efficient transduction should be increased in cells that have lower growth rates. For example, under the same conditions, the efficiency of viral transduction is higher in neuroblatoma cells that the C3H cells due to their higher growth rate (48 h for neuroblatoma cells compared to 72 h for C3H cells) (Figure 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128013746_4338.jpg" width="65"/><br/><strong>Figure 3. C3H/10T1/2 and neuroblastoma cells were transduced with Actin- and Tubulin-BacMam particles (20-30 particles per cell).</strong> (A) Merged, high resolution confocal image of live cells enlarged from (B), showing distinct actin and microtubule structures in both cell lines, with nuclei in blue.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128013920_3155.jpg" width="237"/><br/><strong>Figure 4. Representative growth curves for transduced C3H/10T1/2 (left) and neuroblastoma cells (right).</strong> Transduced cells were cultured for 7 days and their proliferation rate was measured against their relative controls; n = 3 independent experiments.<br/><br/>No significant differences were observed between the growth rate of transduced cells and their controls, indicating no significant effect of BacMam viral transduction on the physiological growth rate of the transduced cells (Figure 4). Furthermore, no significant differences were observed in the Youngâs modulus of the transduced cells compared to their controls, indicating that BacMam viral transduction had no significant effect on the mechanical properties of the cells in the time periods studied (Figure 5).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128014118_6562.jpg" width="157"/><br/><strong>Figure 5. Young's modulus for transduced C3H/10T1/2 (left) and neuroblastoma (right) cells and baseline controls.</strong> Each point represents a measurement from a unique single cell. Between 15 and 20 cells for each cell line were examined from n = 3 independent experiments.<br/><strong><br/></strong> </p><p class="ppzzs"><strong>Part 2: Live imaging and tracking of stem cell ingression from a seeded surface to the interior of an idealized tissue template</strong><br/>Whereas Part 1 allows the equivalent of mechanical testing of cells and cellular constructs as they adapt in space and time, Part 2 of the protocol focuses on cell movement in space and time. This provides a means to measure the energy or power (energy use over time) of cell motility. The protocol describes live imaging and tracking of stem cell ingression from a seeded surface to the interior of an idealized tissue template as a function of cell metabolism, enabling assessment of the energy of mechanomics. The approach measures metabolic energy expended by the cell over time (power) as a function of distance covered by the motile cell.</p><p class="pptt" id="biaoti30594">### Procedure</p><ol class="npwone"><li>Periosteum harvest and cell isolation (the procedure can alternatively be completed using ovine periosteum, fresh from the butcher)<ol class="npwtwo"><li>Source periosteum explant tissue from human patients, <em>e.g.</em>, following total joint replacement surgery (Human Research Ethics Committee (HREC) approval 16/203) or from other source such as fresh ovine femora (Knothe Tate <em>et al.</em>, 2011; Chang and Knothe Tate 2011 and 2012; Chang <em>et al.</em>, 2014). <br/><em>Note: The PDC donor in this feasibility study was a 52-year-old female with osteoarthritis of the knee. </em> </li><li>Lift periosteum off the underlying bone with a periosteal elevator and immediately transfer, double-bagged and on ice, to the laboratory. </li><li>To transfer the tissue from hospital to laboratory, keep the explant tissue in a specimen jar and place the jar in a sealed transport box with ice packs. </li><li>In tissue culture laboratory, isolate periosteum in a sterile environment inside the biosafety cabinet. Using periosteal elevator, lift periosteum off the underlying bone and immediately transfer the tissue pieces on to a 60 x 15 mm Petri dish.</li><li>To isolate the PDCs (Figure 6), suspend the tissue on a Petri dish in 3 mg/ml Collagenase II (Gibco) solution in Î±-minimal essential medium (Î±-MEM) with GlutaMAX (Invitrogen) with 1% Antibiotic-antimycotic (AMAB) (Invitrogen) overnight <em>per</em> previous protocols (Knothe Tate <em>et al.</em>, 2011; Chang <em>et al.</em>, 2014).</li><li>After 24 h, observe the presence and growth of cells around the tissue on the Petri dish under the microscope. PDCs appear fibroblastic in shape. Old medium can be replaced with fresh medium containing Collagenase II.</li><li>When adequate cell growth is observed on the dish, wash cells with 3 ml 1x PBS. Then remove PBS and add 2 ml of 0.25% trypsin-EDTA. Then incubate cells at 37 Â°C for 5 min.</li><li>When cell detachment is observed, add 7 ml of standard culture medium (Î±-MEM with 1% AMAB and 10% FBS) and transfer the cell suspension to a 50 ml falcon tube.</li><li>Place the tube in a centrifuge (<em>e.g.</em>, Thermo Fischer Heraus<sup>TM</sup> Multifuge<sup>TM</sup> centrifuge) and spin for 5 min at 300 <em>x g</em>.</li><li>Remove supernatant and resuspend pellet in 10 ml of standard culture medium.</li><li>To determine the cellularity of the tissue, take 10 Âµl of the cell suspension and mix with 10 Âµl 0.4% wt/vol trypan blue salt solution in a separate Eppendorf tube. Prepare a glass coverslip on a hemocytometer. Add about 10 Âµl of the mixture into a chamber on a hemocytometer. Under the microscope, count the viable cells present on the four quadrants of the chamber to get the number of cells present per ml of the cell suspension. Under an inverted microscope, count only viable cells that appear bright white. Dead cells appear blue as their membrane disrupted, allowing trypan blue staining to enter the cells.</li><li>To remove the undigested tissue, filter the remaining cell suspension in falcon tube using a 100 Âµm cell strainer.</li><li>Depending on the number of viable cells, transfer the eluted cells to a T75 culture flask for further expansion (<em>e.g.</em>, if 1 x 10<sup>6</sup> viable cells are present, cell suspension can be divided into 2 or 3 T75 flasks). Maintain cells in an incubator with 5% CO<sub>2</sub> at 37 Â°C. Replace medium every 2-3 days.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128020259_3613.jpg" width="75"/><br/><strong>Figure 6. Schematic diagram outlining overview of protocol, part 2.</strong> A. Periosteum isolation from the knee tissue obtained from a patient undergoing knee replacement surgery. The knee tissue often has a small amount of bone proximal or distal to the knee, where periosteum can be isolated. The tissue piece can be placed on to Petri dish and suspended in 3 mg/ml Collagenase II containing medium to digest the tissue. Periosteum-derived cells (PDCs) growth can be observed after 24 h. B. The expanded PDCs are harvested and seeded on the Matrigel<sup>Â®</sup> in a glass-bottomed well of thickness 1 mm. At a pre-determined time point, cells are stained and migration through the gel can be visualized via confocal microscopy.<br/></li></ol><br/></li><li>Choice of a cell cohort control<br/>Validated BMSCs, acquired from Lonza (Catalog number: PT 2501, lot no. 0000636886), serve as a comparative control. <br/><em>Note: Analogous to the cohort from a previously published study (Chang <em>et al.</em>, 2014), the BMSCs </em><em>were isolated from a 44-year-old male donor and were passaged twice prior to cryopreservation. <br/></em><br/></li><li>Cell culture and cryopreservation<ol class="npwtwo"><li>Culture primary PDCs for a few days in a T75 flask until 80-90% confluent, then expand and passage them 1-2 times prior to use. Some of the PDCs with early passage number can be cryopreserved for future use.</li><li>To passage cell, remove medium and wash cells with 1x PBS once. Then, remove PBS and add 3 ml 0.25% trypsin-EDTA. Keep the flask in an incubator at 37 Â°C for 5 min until cell detachment is observed. </li><li>Then, add 7 ml of standard culture medium to the flask and transfer the cell suspension to a 50 ml falcon tube.</li><li>Centrifuge cells at 300 <em>x g</em> for 5 min. Then, remove the supernatant and resuspend pellet in 10 ml standard culture medium.</li><li>For 1 in 10 dilution passage, transfer 1 ml of this suspension to a new T75 flask for further expansion, with an addition of 14 ml standard culture medium.</li><li>To cryopreserve the cells, harvest cells as per protocol in Step 2, then centrifuge the suspension at 300 <em>x g</em> for 5 min. Remove the supernatant and resuspend the pellet in freezing medium (Î±-MEM with 40% FBS and 10% Dimethyl sulfoxide [DMSO]). A confluent T75 flask can be split into 3 cryovials, with each has a volume of 1 ml.</li><li>Transfer 1 ml of the cell suspension in freezing medium into a cryovial and store at -80 Â°C overnight. </li><li>Transfer cryovials to a liquid nitrogen tank at -196 Â°C for long term storage. </li><li>Culture all cells in standard medium, and replace medium every 2-3 days. </li><li>Perform experiments at passage 2 for PDCs after primary culture establishment and passage 2 for BMSCs after cryopreserved acquisition.<br/><br/></li></ol></li><li>Preparation of tissue template (<em>Anlage</em>) model using Extracellular-Matrix-based hydrogel<ol class="npwtwo"><li>Prepare Matrigel<sup>Â®</sup> at a concentration of 6 mg/ml through dilution with Î±-MEM, 20% FBS and 1% AMAB. Keep Matrigel<sup>Â®</sup> on ice or at a temperature below 10 Â°C during the preparation as it polymerizes at room temperature.</li><li>Load 110 Âµl of the thus prepared Matrigel into the well of a glass-bottom dish (Cellvis, D35-10-0-N) to a thickness of 1.18 mm. </li><li>To prepare BMSCs and PDCs for seeding, wash the cell culture twice each with 5 ml of 1x phosphate-buffered saline (PBS).</li><li>Incubate cells for 5 min with 3 ml 0.25% trypsin-EDTA (Sigma-Aldrich) and add 7 ml standard culture medium to harvest cells. Transfer the cell suspension to a falcon tube and centrifuge at 300 <em>x g</em> for 5 min.</li><li>Resuspend pellet in 10 ml standard culture medium. Take 10 Âµl of this suspension and mix with 10 Âµl of 0.4% (wt/vol) trypan blue. Load 10 Âµl of this mixture into a chamber on a hemocytometer with coverslip. Count the cells on the four quadrants to get the cell number per ml.</li><li>Adjust the cells' number by dilution. Seed 40,000 cells onto the Matrigel<sup>Â®</sup> in a glass-bottomed dish with 2 ml of FBS-reduced medium comprising Î±-MEM, 5% FBS, 1% AMAB. Keep the cells in the incubator at 37 Â°C with 5% CO<sub>2</sub> until predetermined time points designated for live imaging to assess cell migration (t = 3 and 7 days in our studies). Change medium every 2-3 days.<br/><br/></li></ol></li><li>Live cell staining<ol class="npwtwo"><li>At predetermined time points, carefully wash the Matrigel/cell culture twice with 1x PBS. </li><li>Prepare a 2 mM stock solution of Calcein-AM (995 g/mole) (Sigma-Aldrich) by dissolving 1 mg of Calcein-AM in 0.5 ml DMSO. </li><li>From this, make a staining solution by diluting 1 Âµl of Calcein-AM stock for every 1 ml Î±-MEM to give 2 ÂµM staining solution. </li><li>Stain the cells on Matrigel with 2 ml Calcein-AM solution and incubate in the dark for 30 min at 37 Â°C.</li><li>Remove the staining solution and replace with 2 ml FBS-reduced medium. </li><li>Maintain the stained culture in the dark at 37 Â°C prior to imaging.<br/><br/></li></ol></li><li>Fluorescent microscopy and data analysis for live imaging of cell motility<ol class="npwtwo"><li>Using fluorescent microscopy, track the respective cells' spatiotemporal migration behavior and dynamics. We used the Leica SP8 DLS confocal laser scanning microscope to image live PDC and BMSC migration using the 10x dry objective and 488 nm laser. </li><li>Acquire Z-stacks of optical slices and process using ImageJ. </li><li>Either reconstruct image stacks into 3D images using 3D Viewer Plugin to visualize the dynamics of migrating cells or compress them into a 2D image to observe cell behavior. </li><li>Quantify cell migration distance by measuring the z-position of cells as they migrate from the top to the bottom of the 3D idealized tissue template model. We measured in random locations of the Matrigel (n = 6) at Days 3 and 7. Two-way ANOVA statistical analysis and Tukeyâs multiple comparisons test was performed with Prism 7 software (GraphPad Software, La Jolla, USA), where data were presented as mean Â± standard error of the mean (SEM) with a 99% confidence interval. <br/><em>Note: The measurement of cell migration distance takes advantage of the Leica SP8 confocal microscope which can capture images of 3D samples in x, y and z directions. This microscope also enables time-lapse imaging of 3D samples (xyzt mode), a feature that can be explored further to study temporal aspects of cell migration. To perform this, a more stable, permanent fluorophore should be used to label the cells, e.g., GFP expression construct.</em><br/><br/></li></ol></li><li>MTT (Thiazolyl Blue Tetrazolium Bromide) Metabolic Assay<ol class="npwtwo"><li>Retrieve PDCs and BMSCs grown in culture at early passage (up to passage 6 or less). Alternatively, PDCs and BMSCs with early passage number can be retrieved from liquid nitrogen. These cells are thawed and grown in standard culture medium. After thawing, the cells are passaged 1-2 times prior to use for experiment.</li><li>PDCs and BMSCs are washed with PBS and trypsinized with 3 ml 0.25% trypsin-EDTA for 5 min at 37 Â°C. After cell detachment is observed, 7 ml of standard culture medium is added and cell suspension is transferred to a Falcon tube.</li><li>Centrifuge the suspension at 1500 rpm or 300 <em>x g</em> for 5 min. Then, remove the supernatant and resuspend the pellet in 10 ml standard culture medium. Take 10 Âµl of the cell suspension and mix with 10 Âµl of trypan blue for cell counting using a hemocytometer.</li><li>Cell number in the suspension is adjusted for seeding into a 24-well plate at a density of 10,000 cells per well (5,000 cells/cm<sup>2</sup>), with a volume of 1 ml per well.</li><li>Prior to seeding, prepare Matrigel<sup>Â® </sup>at a concentration of 6 mg/ml by diluting with standard culture medium containing 10% FBS and 1% AMAB. Then, add 300 Âµl of the prepared Matrigel to each well to coat the wells of 24-well plate. </li><li>After the gel coat has polymerized at room temperature, seed PDCs and BMSCs in each well at a density of 5,000 cells/cm<sup>2</sup> (n = 4). Keep cells in well plate in the incubator at 37 Â°C with 5% CO<sub>2</sub>.</li><li>Retrieve the 3D cell culture on Day 3 and Day 6 after seeding for MTT assay. </li><li>Prepare MTT stock solution by diluting 5 mg MTT (Sigma-Aldrich) in 5 ml 1x PBS. </li><li>Dilute 1 ml of this stock solution in 8 ml medium (1:8). </li><li>Remove the medium in the well plate and wash the cells with 1x PBS. </li><li>Add 300 Âµl of MTT solution to each well and incubate the cells in the dark at 37 Â°C for 1 h. </li><li>After incubation, purple formazan can be observed which is the result of metabolized MTT by the cellsâ mitochondrial dehydrogenase. Remove the MTT solution from the wells and solubilize the coating gel containing the purple formazan in DMSO (300 Âµl per well). Then transfer the content in a well to an Eppendorf tube. Centrifuge for 5 min at 1500 rpm or 300 <em>x g</em> to separate gel and liquid layer. </li><li>Discard the liquid DMSO layer and solubilize the gel again with DMSO.</li><li>Centrifuge at 300 <em>x g</em> to solubilize the formazan in the gel, whereby the gel becomes a pellet.</li><li>Transfer the upper liquid layer containing formazan to a well plate and read the absorbance using a plate reader (Perkin Elmer Victor 3, 570 nm filter).</li></ol></li></ol><p class="pptt" id="biaoti30595">### Data analysis</p><p class="ppzz"><strong><em>Representative data and protocol validation</em></strong><br/>In the model tissue template, PDCs were observed to migrate significantly faster than BMSCs, covering a 1.4-fold and 1.5-fold greater respective distance on Day 3 and Day 7 of culture (<em>P</em> â¤ 0.01) (Figure 7A). Relative differences in migrating PDCs' and BMSCs' patterns, as well as cell morphologies and numbers were qualitatively apparent as well. While PDCs exhibited long cellular protrusions to form extensive networks within the Matrigel<sup>Â®</sup>, migrating BMSCs tended to form defined clusters of cells dispersed throughout the Matrigel. The number of PDCs migrating and forming networks increased visibly from Day 3 to 7 (Figures 7B PDC<sub>B</sub>, PDC<sub>C</sub> and BMSC<sub>B</sub>, BMSC<sub>C</sub>) compared to BMSCs, while BMSC cluster size increased (Figures 7B BMSC<sub>B</sub> and BMSC<sub>C</sub>) in the same period. PDCs and BMSCs exhibited similar metabolic activity (Figure 8) after 3 and 7 days in culture.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128021400_3928.jpg" width="61"/><br/><strong>Figure 7. Human PDCs and BMSCs exhibit distinct migration behavior and dynamics in an EMT model.</strong> A. In pilot testing of the model, a two-way ANOVA (n = 6) showed that PDCs from a single patient migrated significantly further and exhibited higher migration rate than that of BMSCs from a single donor on Days 3 and 7. Error bars represent standard error of the mean (SEM) and asterisks (*) indicate significant differences at <em>P</em> â¤ 0.01. B and C. Representative stack images of PDCs and BMSCs on (B) Day 3 and (C) on Day 7. The individual image represents the different 3D views of the distance covered by both cell types, as well as the cell behavior and distribution throughout the thickness of the Matrigel<sup>Â®</sup>. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191128/20191128021529_9012.jpg" width="145"/><br/><strong>Figure 8. Proliferation of PDCs and BMSCs quantified as MTT absorbance and normalized to that of the control.</strong> PDCs and BMSCs show a comparable increase in cell number over ten days in culture, suggesting the comparable proliferation rate between the two cell lines. No statistically significant differences are observed between MTT absorbance of PDCs or BMSCs at Days 0, 3, 7, or 10 (significance defined as <em>P</em> &lt; 0.05).<br/><br/><strong><em>Discussion </em></strong><br/>To probe the mechanome, the mechanics equivalent to the genome, and thereby unravel relationships between the local mechanical environment of the stem cell (SC), SC mechanoadaptation and lineage commitment, we developed and tested two integrated cell and tissue model culture and imaging platforms. Our objective was to develop and test the two model platforms to enable quantitative study of mechanoadaptations and mechanomics at the tissue and cellular length scales, and in contexts emulating physiological cell and tissue environments (Sorkin <em>et al.</em>, 2004). First, through simultaneous implementation of nuclear and cytoskeletal tagging, we integrated methods used previously in separate applications, successfully tracking cell and nucleus volume and shape changes as well as cytoskeletal actin and tubulin architectural adaptation within individual, adherent model embryonic murine mesenchymal stem cells. Then, a tissue template (<em>Anlage</em>) model was developed on a platform enabling ingression of cells seeded in a monolayer into a three-dimensional extracellular matrix protein comprised tissue template. The model was tested successfully using human mesenchymal stem cells including PDCs and BMSCs and showed promising initial results, enabling comparative measures of motility rates and directions. The model lends itself for refinement and expansion to emulate EMT-METs tissue and cell contexts of interest to a variety of research groups. While the two model systems have not yet been tested together in a relevant cell model, doing so would present an elegant platform for cross scale imaging of emergent structure and function in mechanobiology. <br/>  Given the spatial and temporal complexity of mechanobiology of single cells, there is an acute need to quantify effects of biomechanical stimuli. There is an imperative to measure effects of physical forces on the nucleus shape, cell volume, spatiotemporal organization of the actin filaments and tubulins, as well as gene expression associated with unfolding lineage commitment; previous measures have been carried out in both fixed and live cells (McBride and Knothe Tate, 2008; McBride<em> et al.</em>, 2008; Song <em>et al.</em>, 2010, 2012 and 2013; Chang and Knothe Tate, 2011). Although different protocols have been optimized and used for imaging of actin and microtubules separately in live cells (Wang <em>et al.</em>, 2003; Suresh, 2007; Na <em>et al.</em>, 2008; Mendez <em>et al.</em>, 2014), this is the first protocol to our knowledge that has demonstrated simultaneous, multiplexed imaging of actin, tubulin and nucleus in live cells. Application of this protocol to live stem cells enabled simultaneous multi-color time lapse imaging of actin and tubulin, cytoskeletal adaptation, as well as changes in nucleus volume and shape. The protocol lends itself for time lapse studies of cellular mechanoadaptation during application of controlled mechanical stimuli, like a mechanical test of a living material that evolves (adapts structure and/or mechanical properties and/or biological phenotype) during the testing procedure. The protocol was tested and optimized for both stem cells and neuroblastoma cells, demonstrating its utility for different cell types. In addition, this protocol can be used to image the organization of both actin and tubulin in migrating cells in real time and in a way that has not been possible before. <br/>  The model tissue <em>Anlage</em> can be further tuned to include extracellular matrix constituents typical for specific tissues, from tissue templates during development to postnatal tissue healing. Experimental design can be honed to include molecular gradients specific to mechanistic pathways of interest. Of particular interest in this first feasibility study, we wanted to determine whether the rate of ingression could be measured in a standardized context as a currently underappreciated factor in regenerative medicine and intrinsic healing capacities of resident stem cell populations (Yu <em>et al.</em>, 2017; Ni <em>et al.</em>, 2019). The model system also lends itself well for future studies aiming to elucidate regulation of stem cell niche quiescence (Yu <em>et al.</em>, 2017; Ni <em>et al.</em>, 2019).<br/>  Increasingly, systems biology approaches combined with engineering innovations are leading to quantification and mechanistic elucidation of common paradigms across disparate tissues, organs, organisms and even kingdoms. In mechanomics, cell shape and fate are intrinsic expressions of form and function (Song <em>et al.</em>, 2012 and 2013; Wang <em>et al.</em>, 2014; Knothe Tate <em>et al.</em>, 2011 and 2016). Through coupling of multiscale imaging and mechanical modeling, and performing mechanical tests on SCs as they adapt and differentiate, it may be possible to begin to formulate a first Law of Biology, establishing a quantitative basis and a predictive model for the relationship between stress distribution in a cell and the unfolding of cell fate. <br/><br/><strong><em>Comparison with other methods</em></strong><br/>A number of studies have used different protocols to visualize cytoskeletal de-/polymerization, especially of actin filaments, in live cells. Some inject fluorescently labeled actin directly into living cells (Riedl <em>et al.</em>, 2008 and 2010). In addition, direct observation of the actin dynamics through actin-GFP fusions or through fluorescent-tagged actin-associated protein had limited success mainly due to interference of the tagged protein in actin polymerization and depolymerization processes which impacts natural behavior of the filaments. Excessive fluorescent background because of the unbound G-actin has also been reported as a major problem (Riedl <em>et al.</em>, 2008 and 2010). Despite these limitations, methods have been developed to successfully tag actin filaments without any significant effects on actin kinetics and assembly (Riedl <em>et al.</em>, 2008; LukinaviÄius <em>et al.</em>, 2014). New techniques enable imaging of actin filaments and tubulins in live cells and at high resolution. However, the imaging of the actin filaments and tubulins simultaneously has not been demonstrated previously to our knowledge. In our previous studies, we have developed a live cell imaging technique to observe cytoskeleton spatiotemporal organization and cell mechanoadaptation during stem cell differentiation and in near real time (Zimmermann and Knothe Tate, 2011; Chang and Knothe Tate, 2011). However, these experiments were performed separately on actin and tubulin as well as the cell and its nucleus. <br/><br/><strong><em>Limitations and Context</em></strong><br/>The BacMam protocol was shown to be efficient and efficacious for two cell lines (model embryonic murine mesenchymal stem cell line and immortalized neuroblastoma cells) but was not effective for transduction in ovine PDCs [unpublished data, MechBio Team]. Similarly, while the tissue template model platform worked well with human PDCs and BMSCs, it would need to be validated for other cell types and extracellular matrix constituents. While each refinement will require additional time investment, the presentation of the methods in their current form is intended to facilitate the process. Furthermore, the model systems require additional validation for different systems of interest and/or cell types. <br/>  Both model systems presented in this manuscript represent idealizations of actual stem cell biology <em>in situ</em> in living organisms. Nonetheless, they provide valuable tools for quantifying intracellular adaptation in response to controlled mechanical cues and live imaging of stem cell ingression. From an engineering perspective, the model platforms provide defined control volumes in which the boundary and initial conditions are set and hence known by the researcher, enabling quantitative spatial and temporal study of mechanomics using traditional engineering problem solving rubrics, as well as implementation of governing equations for outcome measures of interest. The systems are designed to intersect between engineering, systems biology and cell biology approaches to enable mechanistic study of mechanical properties and biological behaviors in the same system and at the same time. The platforms can be specialized for various systems of biological or physiological interest similar to microfluidics (Shemesh <em>et al.</em>, 2015) and organ-on-a-chip approaches but are specifically designed to emulate fundamental events such as ingression towards or away from tissue templates, like MET-EMTs, that are postulated to play a key role in stem cell fate decisions.<br/><br/><strong><em>Conclusions</em></strong><br/>The mechanisms underpinning the stem cells' innate capacity to adapt to mechanical stimuli and the role of mechanoadaptation in lineage commitment are unknown. An understanding of SC mechanoadaptation is key to deciphering lineage commitment, during prenatal development, postnatal wound healing, and the engineering of tissues. Cell shape and fate are intrinsic expressions of form and function in the most basic building element of tissues. Just as the development of experimental and theoretical mechanics, including tools to visualize and measure displacements and forces on surfaces and interiors of structural elements, led to a fundamental understanding of mechanics of materials, the development of mechanomics tools for live imaging of cell motility cell mechanoadaptation are expected to define the equivalent of a Mohrâs circle of lineage commitment (referred to herein as mapping the mechanome, Figure 1).</p><p class="pptt" id="biaoti30596">### Acknowledgments</p><p class="ppzz">The authors would like to acknowledge the collaborative efforts of the MechBio Team in the Graduate School of Biomedical Engineering and infrastructure and collaboration of the Mark Wainwright Analytical Centre, in particular the Biomedical Imaging Facility at the University of New South Wales.<br/><br/><strong><em>Author contributions</em></strong><br/>IJ refined and tested the multiplexed live cytoskeletal tracking approach with MLKT and carried out related experiments with imaging advice from RW. VP and MC developed under mentorship of MLKT and KP and RW the tissue template model system and carried out related experiments with imaging expertise from FT. We acknowledge with gratitude the extensive previous methods and protocols developed by MechBio Team members and collaborators and their publications, upon which the current protocol has been developed and tested (Sorkin <em>et al.</em>, 2004; Anderson <em>et al.</em>, 2006; 2007a and 2007b; Knothe Tate <em>et al.</em>, 2008; 2010 and 2016; McBride and Knothe Tate, 2008; McBride <em>et al.</em>, 2008; Song <em>et al.</em>, 2010; 2012 and 2013; Chang and Knothe Tate, 2011; Zimmerman and Knothe Tate, 2011; Evans <em>et al.</em>, 2013; Jalilian <em>et al.</em>, 2015; Putra <em>et al.</em>, 2019).<br/>  As noted above, parts of the protocol were used in previous publications (Putra <em>et al.</em>, 2019; Ng <em>et al.,</em> 2019). The actin and tubulin tagging protocols with controlled loading were also used separately in previous publications (Zimmerman and Knothe Tate, 2011; Chang and Knothe Tate, 2011).</p><p class="pptt" id="biaoti30597">### Competing interests</p><p class="ppzz">In context of full disclosure, the live imaging studies of nucleus shape and volume changes and cytoskeletal adaptation were carried out using an imaging and perfusion chamber developed by Professor Knothe Tate's MechBio Team and later commercialized through a nonexclusive license agreement with Harvard Apparatus, Warner Instruments (<a href="https://www.warneronline.com/proflow-shear-flow-chamber-pfc-1" target="_blank">https://www.warneronline.com/proflow-shear-flow-chamber-pfc-1</a>). The study design and outcomes were conducted without consultation or involvement by Warner Instruments and other perfusion chamber and/or microfluidics-based platforms could be implemented with these protocols (Shemesh <em>et al.</em>, 2015). <br/>  The studies were made possible through the generous support of the U.S. National Institutes of Health (DD and MLKT), U.S. National Institutes of Health Training Grant (recipients: SM-G, MJS), U.S. National Science Foundation (MLKT), Australian National Health and Medical Research Council (MLKT), the Paul Trainor Foundation (MLKT), and Gold and Silver Star grants from UNSW for "near miss funding" of Australian Research Council grant proposals.</p><p class="pptt" id="biaoti30598">### Ethics</p><p class="ppzz">The University of New South Wales human ethics committee approved the described experiment. Informed consent was obtained from all subjects.</p><p class="pptt" id="biaoti30599">### References</p><ol class="npwtwok"><li>Airenne, K. J., MaHonen, J., A. and Laitinen, H. O. (2003) <a href="https://www.taylorfrancis.com/books/e/9780429131936/chapters/10.1201/9781420002126-17" target="_blank">Baculovirus-mediated gene transfer: an evolving new concept, p. 181-197. In N. S. Templeton (ed.).</a> <em>Gene therapy: therapeutic mechanisms and strategies,</em> 2nd ed. Marcel Dekker, Inc., New York, N.Y.</li><li>Anderson, E. J., Falls, T. D., Sorkin, A. M. and Knothe Tate, M. L. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16672051" target="_blank">The imperative for controlled mechanical stresses in unraveling celular mechanisms of mechanotransduction.</a> <em>Biomed Eng Online</em> 5: 27.</li><li>Anderson, E. J. and Knothe Tate, M. L. (2007a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17822359" target="_blank">Design of tissue engineering scaffolds as delivery devices for mechanical and mechanically modulated signals.</a> <em>Tissue Eng</em> 13(10): 2525-2538.</li><li>Anderson, E. J. and Knothe Tate, M. L. (2007b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18053207" target="_blank">Open access to novel dual flow chamber technology for in vitro cell mechanotransduction, toxicity and pharamacokinetic studies. </a><em>Biomed Eng Online</em> 6: 46.</li><li>Aubry, D., Gupta, M., Ladoux, B. and Allena, R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25884335" target="_blank">Mechanical link between durotaxis, cell polarity and anisotropy during cell migration.</a> <em>Phys Biol</em> 12(2): 026008.</li><li>Bach, C.T., Schevzov, G., Bryce, N.S., Gunning, P.W., and O'Neill, G.M. (2010). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2900618/" target="_blank">Tropomyosin isoform modulation of focal adhesion structure and cell migration.</a> <em>Cell Adh Migr</em> 4(2): 226-34. </li><li>Blanchoin, L., Boujemaa-Paterski, R., Sykes, C. and Plastino, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24382887" target="_blank">Actin dynamics, architecture, and mechanics in cell motility.</a> <em>Physiol Rev</em> 94(1): 235-263.</li><li>Bryce, N. S., Schevzov, G., Ferguson, V., Percival, J. M., Lin, J. J., Matsumura, F., Bamburg, J. R., Jeffrey, P. L., Hardeman, E. C., Gunning, P. and Weinberger, R. P. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12631719" target="_blank">Specification of actin filament function and molecular composition by tropomyosin isoforms.</a> <em>Mol Biol Cell</em> 14(3): 1002-1016. </li><li>Chang, H. and Knothe Tate, M. L. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22338708" target="_blank">Structure-function relationships in the stem cell's mechanical world B: emergent anisotropy of the cytoskeleton correlates to volume and shape changing stress exposure.</a> <em>Mol Cell Biomech</em> 8(4): 297-318.</li><li>Chang, H. and Knothe Tate, M. L. (2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23197852" target="_blank">The periosteum: tapping into a reservoir of clinically useful progenitor cells.</a><em> Stem Cells Transl Med</em> 1(6): 480-491.</li><li>Chang, H., Docheva, D., Knothe, U. R. and Knothe Tate, M. L. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24477075" target="_blank">Arthritic periosteal tissue from joint replacement surgery as an autologous source of stem cells.</a> <em>Stem Cells Transl Med</em> 3(3): 308-17.</li><li>De Pascalis, C. and Etienne-Manneville, S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28684609" target="_blank">Single and collective cell migration: the mechanics of adhesions.</a> <em>Mol Biol Cell</em> 28(14): 1833-1846.</li><li>Earls, J. K., Jin, S. and Ye, K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23472616" target="_blank">Mechanobiology of human pluripotent stem cells.</a> <em>Tissue Eng Part B Rev</em> 19(5): 420-430.</li><li>Evans, S. F., Docheva, D., Bernecker, A., Colnot, C., Richter, R. and Knothe Tate, M. L. (2013) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23237517" target="_blank">Solid-supported lipid bilayers as a novel platform to engineer emergence of stem cell fate and tissue architecture using periosteum derived progenitor cells.</a> <em>Biomaterials</em> 34(8): 1878-1887.</li><li>Galarza Torre, A., Shaw, J. E., Wood, A., Gilbert, H. T. J., Dobre, O., Genever, P., Brennan, K., Richardson, S. M. and Swift, J. (2018) <a href="https://www.nature.com/articles/s41598-018-27346-9" target="_blank">An immortalised mesenchymal stem cell line maintains mechano-responsive behaviour and can be used as a reporter of substrate stiffness.</a> <em>Sci Rep</em> 8(1): 8981.</li><li>Heo, S. J., Thorpe, S. D., Driscoll, T. P., Duncan, R. L., Lee, D. A. and Mauck, R. L. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26592929" target="_blank">Biophysical regulation of chromatin architecture instills a mechanical memory in mesenchymal stem cells.</a> <em>Sci Rep</em> 5: 16895.</li><li>Ho, Y. C., Chung, Y. C., Hwang, S. M., Wang, K. C. and Hu, Y. C. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15712348" target="_blank">Transgene expression and differentiation of baculovirus-transduced human mesenchymal stem cells.</a> <em>J Gene Med</em> 7: 860â868.</li><li>Jalilian, I., Heu, C., Cheng, H., Freittag, H., Desouza, M., Stehn, J. R., Bryce, N. S., Whan, R. M., Hardeman, E. C., Fath, T., Schevzov, G. and Gunning, P. W. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25978408" target="_blank">Cell elasticity is regulated by the tropomyosin isoform composition of the actin cytoskeleton.</a> <em>PLoS One</em> 10(5): e0126214.</li><li>Kost, T. A., Condreay, J. P. and Jarvis, D. L. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15877075" target="_blank">Baculovirus as versatile vectors for protein expression in insect and mammalian cells.</a> <em>Nat Biotechnol</em> 23(5): 567-575.</li><li>Knothe, U., Dolejs, S., Miller, R.M., and Knothe Tate, M.L. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20673900" target="_blank">Effects of mechanical loading patterns, bone graft and proximity to periosteum on bone defect healing.</a> <em>J. Biomechanics</em> 43(14): 2728-37.</li><li>Knothe Tate, M. L., Falls, T. D., McBride, S. H., Atit, R. and Knothe, U. R. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18620888" target="_blank">Mechanical modulation of osteochondroprogenitor cell fate.</a><em> Int J Biochem Cell Biol</em> 40(12): 2720-2738.</li><li>Knothe Tate, M. L., Falls, T. D., McBride, S. H. and Atit, R. (2010). <a href="https://www.researchgate.net/publication/307718401_Engineering_an_ecosystem_Taking_cues_from_nature's_paradigm_to_build_tissue_in_the_lab_and_the_body" target="_blank">Engineering an ecosystem: taking cues from natureâs paradigm to build tissue in the lab and the body.</a> <em>Fields Institute for Mathematics in Biology monograph series on New Perspectives in Mathematical Biology</em> 57: 113-134.</li><li>Knothe Tate, M. L. (2011) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21146825" target="_blank">Top down and bottom up engineering of bone.</a><em> J Biomechanics</em> 44(2): 304-12.</li><li>Knothe Tate, M. L., Moore, S., Chang, H. and Knothe, U. (2011) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22174873" target="_blank">Surgical membranes as directional delivery devices to generate tissue in critical sized defects.</a> <em>PLoS one</em> 6(12): e28702.</li><li>Knothe Tate, M. L., Gunning, P. W. and Sansalone, V. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27739911" target="_blank">Emergence of form from function - mechanical engineering approaches to probe the role of stem cell mechanoadaptation in sealing cell fate.</a> <em>Bioarchitecture</em> 6(5): 85-103.</li><li>Ladoux, B. and Mege, R. M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29115298" target="_blank">Mechanobiology of collective cell behaviours.</a> <em>Nat Rev Mol Cell Biol</em> 18(12): 743-757.</li><li>Le, H. Q., Ghatak, S., Yeung, C. Y., Tellkamp, F., GÃ¼nschmann, C., Dieterich, C., Yeroslaviz, A., Habermann, B., Pombo, A., Niessen, C. M. and WickstrÃ¶m, S. A. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27398909" target="_blank">Mechanical regulation of transcription controls Polycomb-mediated gene silencing during lineage commitment.</a> <em>Nat Cell Biol</em> 18(8): 864-75.</li><li>Li, X., Pei, D. and Zheng, H. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24805308" target="_blank">Transitions between epithelial and mesenchymal states during cell fate conversions.</a> <em>Protein Cell</em> 5(8): 580-91.</li><li>LukinaviÄius, G., Reymond, L., D'Este, E., Masharina, A., Gottfert, F., Ta, H., Guther, A., Fournier, M., Rizzo, S., Waldmann, H., Blaukopf, C., Sommer, C., Gerlich, D. W., Arndt, H. D., Hell, S. W. and Johnsson, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24859753" target="_blank">Fluorogenic probes for live-cell imaging of the cytoskeleton.</a> <em>Nat Methods</em> 11(7): 731-733.</li><li>Mendez, M. G., Restle, D. and Janmey, P. A. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25028873" target="_blank">Vimentin enhances cell elastic behavior and protects against compressive stress.</a> <em>Biophys J</em> 107: 314-23.</li><li>McBride, S. H. and Knothe Tate, M. L. (2008a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18774910" target="_blank">Modulation of stem cell shape and fate A: the role of density and seeding protocol on nucleus shape and gene expression.</a> <em>Tissue Eng Part A</em> 14(9): 1561-1572.</li><li>McBride, S. H., Falls, T. and Knothe Tate, M. L. (2008b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18774911" target="_blank">Modulation of stem cell shape and fate B: mechanical modulation of cell shape and gene expression.</a> <em>Tissue Eng Part A</em> 14(9): 1573-1580.</li><li>McBride, S. H., Dolejs, S., Knothe, U. and Knothe Tate, M. L. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21353676" target="_blank">Major and minor centroidal axes serve as objective, automatable reference points to test mechanobiological hypotheses using histomorphometry.</a> <em>J Biomech</em> 44(6): 1205-1208.</li><li>Na, S., Collin, O., Chowdhury, F., Tay, B., Ouyang, M., Wang, Y. and Wang, N. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18456839" target="_blank">Rapid signal transduction in living cells is a unique feature of mechanotransduction.</a> <em>Proc Natl Acad Sci</em> <em>USA</em> 105:6626â6631.</li><li>Ng, J. L., Kersh, M. E., Kilbreath, S. and Knothe Tate, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28634452" target="_blank">Establishing the basis for mechanobiology-based physical therapy protocols to potentiate cellular healing and tissue regeneration.</a> <em>Front Physiol</em> 8: 303.</li><li>Ni, F., Yu, W. M., Wang, X., Fay, M. E., Young, K. M., Qiu, Y., Lam, W. A., Sulchek, T. A., Cheng, T., Scadden, D. T. and Qu, C. K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30880025" target="_blank">Ptpn21 controls hematopoietic stem cell homeostasis and biomechanics.</a> <em>Cell Stem Cell</em> 24(4): 608-620 e606.</li><li>Ng, J. L., Putra, V. D. L. and Knothe Tate, M. L. (2019). <a href="http://www.sciencedirect.com/science/article/pii/S1751616119308264" target="_blank"><em>In vitro</em> biocompatibility and biomechanics study of novel, Microscopy Aided Designed and ManufacturEd (MADAME) materials emulating natural tissue weaves and their intrinsic gradients.</a> <em>J Mech Behav Biomed </em>: 103536.<br/></li><li>Nimmo, R. A., May, G. E. and Enver, T. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26004869" target="_blank">Primed and ready: understanding lineage commitment through single cell analysis.</a> <em>Trends Cell Biol</em> 25(8): 459-467.</li><li>Putra, V., Song, M.J., McBride-Gagyi, S., Chang, H., Poole, K., Whan, R., Dean, D., Sansalone, V. and Knothe Tate, M.L. (2019). Mechanomics approaches to understand cell behavior in context of tissue neogenesis, during prenatal development and postnatal healing. <em>Frontiers Dev</em> (Unpublished).</li><li>Rasband, W.S., ImageJ, U. S. National Institutes of Health, Bethesda, Maryland, USA, <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/.</a> 1997-2018.</li><li>Riedl, J., Crevenna, A. H., Kessenbrock, K., Yu, J. H., Neukirchen, D., Bista, M., Bradke, F., Jenne, D., Holak, T. A., Werb, Z., Sixt, M. and Wedlich-Soldner, R. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18536722" target="_blank">Lifeact: a versatile marker to visualize F-actin.</a> <em>Nat Methods</em> 5(7): 605-607.</li><li>Riedl, J., Flynn, K. C., Raducanu, A., Gartner, F., Beck, G., Bosl, M., Bradke, F., Massberg, S., Aszodi, A., Sixt, M. and Wedlich-Soldner, R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20195247" target="_blank">Lifeact mice for studying F-actin dynamics. </a><em>Nat Methods</em> 7(3): 168-169.</li><li>Salminen, M., Airenne, K. J., Rinnankoski, R., Reimari, J., Valilehto, O., Rinne, J., Suikkanen, S., Kukkonen, S., Yla-Herttuala, S., Kulomaa, M. S. and Vihinen-Ranta, M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15708991" target="_blank">Improvement in nuclear entry and transgene expression of baculoviruses by disintegration of microtubules in human hepatocytes.</a> <em>J Virol</em> 79(5): 2720-2728.</li><li>Shemesh, J., Jalilian, I. H. P., Shi, A., Knothe Tate, M. L., Yeoh, G. H., and Ebrahimi Warkiani, M. (2015) <a href="https://pubs.rsc.org/en/content/articlelanding/2015/lc/c5lc00633c#!divAbstract" target="_blank">Flow induced stress on adherent cells in microfluidic devices.</a> <em>Lab on a Chip</em> 15(21): 4114-27.</li><li>Steward, A. J. and Kelly, D. J. (2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25382217" target="_blank">Mechanical regulation of mesenchymal stem cell differentiation.</a> <em>J Anat</em> 227(6): 717-31.</li><li>Shoji, I., Aizaki, H., Tani, H., Ishii, K., Chiba, T., Saito, I., Miyamura, T. and Matsuura, Y. (1997) <a href="https://www.ncbi.nlm.nih.gov/pubmed/9349488" target="_blank">Efficient gene transfer into various mammalian cells,including non-hepatic cells, by baculovirus vectors.</a> <em>J Gen Virol</em> 78: 2657â2664.</li><li>Song, M. J., Brady-Kalnay, S. M., McBride, S. H., Phillips-Mason, P., Dean, D. and Knothe Tate, M. L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22970134" target="_blank">Mapping the mechanome of live stem cells using a novel method to measure local strain fields in situ at the fluid-cell interface.</a> <em>PLoS One</em> 7(9): e43601.</li><li>Song, M. J., Dean, D. and Knothe Tate, M. L. (2010).<a href="http://www.ncbi.nlm.nih.gov/pubmed/20862249" target="_blank"> In situ spatiotemporal mapping of flow fields around seeded stem cells at the subcellular length scale.</a> <em>PLoS One</em> 5(9).</li><li>Song, M. J., Dean, D. and Knothe Tate, M. L. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23660249" target="_blank">Mechanical modulation of nascent stem cell lineage commitment in tissue engineering scaffolds.</a> <em>Biomaterials</em> 34(23): 5766-5775.</li><li>Sorkin, A., Dee, KC, and Knothe Tate, M.L. (2004) <a href="https://www.physiology.org/doi/full/10.1152/ajpcell.00059.2004" target="_blank">âCulture shockâ from the bone cellâs perspective: emulating physiologic conditions for mechanobiological investigation.</a> <em>Am J Physiol Cell Physiol</em> 287(6): C1527-36.</li><li>Stumpf, P. S., Smith, R. C. G., Lenz, M., Schuppert, A., MÃ¼ller, F. J., Babtie, A., Chan, T. E., Stumpf, M. P. H., Please, C. P., Howison, S. D., Arai, F. and MacArthur B. D.(2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28957659" target="_blank">Stem cell differentiation as a non-Markov stochastic process.</a> <em>Cell Syst</em> 5(3): 268-282.</li><li>Suresh, S (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17540628" target="_blank">Biomechanics and biophysics of cancer cells.</a> <em>Acta Biomater</em> 3: 413-38.</li><li>Wang, J., Chen, H., Seth, A. and McCulloch, C. A. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12842814" target="_blank">Mechanical force regulation of myofibroblast differentiation in cardiac fibroblasts.</a> <em>Am J Physiol Heart Circ Physiol</em> 285(5): H1871-1881.</li><li>Wang, J., LÃ¼, D., Mao, D. and Long, M. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24756566" target="_blank">Mechanomics: an emerging field between biology and biomechanics.</a> <em>Protein Cell</em> 5: 518-31.</li><li>Yu, N. Y., O'Brien, C. A., Slapetova, I., Whan, R. M. and Knothe Tate, M. L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28170186" target="_blank">Live tissue imaging to elucidate mechanical modulation of stem cell niche quiescence.</a> <em>Stem Cells Transl Med</em> 6(1): 285-292.</li><li>Zimmermann, J. A. and Knothe Tate, M. L. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22338707" target="_blank">Structure-function relationships in the stem cell's mechanical world A: seeding protocols as a means to control shape and fate of live stem cells.</a> <em>Mol Cell Biomech</em> 8(4): 275-296.</li></ol>