<body>
            <h1>An Optimized Method to Isolate Human Fibroblasts from Tissue for <em>Ex Vivo</em> Analysis</h1>
            <p><strong>Authors:</strong> Sara   Waise, Rachel  Parker, Matthew J. J. Rose-Zerilli, David M. Layfield, Oliver  Wood, Jonathan  West, Christian H. Ottensmeier, Gareth J. Thomas, Christopher J. Hanley</p>
            <h2>Abstract</h2>
            <p>Despite their involvement in many physiological and pathological processes, fibroblasts remain a poorly-characterized cell type. Analysis of primary fibroblasts while maintaining their <em>in vivo</em> phenotype is challenging: standard methods for fibroblast isolation require cell culture <em>in vitro</em>, which is known to alter phenotypes. Previously-described protocols for the dissociation of primary tissues fail to extract sufficient numbers of fibroblasts, instead largely yielding immune cells. Here, we describe an optimized method for generating a fibroblast-enriched single-cell suspension from human tissues using combined mechanical and enzymatic dissociation. This allows analysis of <em>ex vivo</em> fibroblasts without the need for culture <em>in vitro</em>.</p>
            <p><strong>Keywords:</strong> Tissue disaggregation, Stroma, Fibroblasts, Lung</p>
            <h2>Background</h2><p>Fibroblasts are almost ubiquitous in human tissues, and the most common cell type in the stroma of a number of solid tumours, where they are referred to as cancer-associated fibroblasts (CAFs) (Ishii <em>et al.</em>, 2016; Kalluri and Zeisberg, 2006; Rupp <em>et al.</em>, 2014; Servais and Erez, 2013). CAFs are associated with multiple hallmarks of malignancy (Ishii <em>et al.</em>, 2016; Tao <em>et al.</em>, 2017) and correlate with poor prognosis in multiple solid tumours (Hanley <em>et al.</em>, 2018). <br/><br/>Given these tumour-promoting effects, and their genetic stability relative to cancer cells (Ishii <em>et al.</em>, 2016), it is unsurprising that fibroblasts are an attractive therapeutic target. However, clinical trials targeting CAFs have so far yielded disappointing results (Hofheinz <em>et al.</em>, 2003; Narra <em>et al.</em>, 2007). This may, in part, be due to variation within the fibroblast population: these cells are known to be heterogeneous in both normal and disease states (DesmouliÃ¨re <em>et al.</em>, 2004; Sugimoto <em>et al.</em>, 2006; Anderberg and Pietras, 2009; Servais and Erez, 2013; Witowski <em>et al.</em>, 2015; Kalluri, 2016; Mellone <em>et al.</em>, 2017). However, this heterogeneity remains poorly-characterised and it is not yet clear how many subtypes are present within a given tissue or tumour type, or the nature of functional differences between groups (Herrera <em>et al.</em>, 2013; Servais and Erez, 2013; Ishii <em>et al.</em>, 2016).<br/><br/>Characterising heterogeneity of cell populations within human tissues often requires analysis at a single-cell level. Single-cell RNA sequencing is a valuable platform for characterising multicellular ecosystems. However, fibroblasts are embedded within extracellular matrix and are particularly difficult to isolate: this has led to under-representation in, for example, single-cell RNA sequencing datasets (Lambrechts <em>et al.</em>, 2018). Unlike many murine models, there is no standardized disaggregation protocol for human solid tissues. A number of different immune cell populations have been successfully isolated and analyzed directly from tissues (Holt <em>et al.</em>, 1986; Perrot <em>et al.</em>, 2007; Grange <em>et al.</em>, 2011; Quatromoni <em>et al.</em>, 2015; Ganesan <em>et al.</em>, 2017). However, epithelial and non-immune stromal cells are usually cultured <em>in vitro</em> prior to analysis (Lurton <em>et al.</em>, 1999; Koumas <em>et al.</em>, 2003; Comhair <em>et al.</em>, 2012; Barkauskas <em>et al.</em>, 2013; Mackay <em>et al.</em>, 2013). Culture <em>in vitro</em> has been shown to change fibroblast phenotypes (Ãhlund <em>et al.</em>, 2017; Waise <em>et al.</em>, 2019); thus, how and whether the functional differences described <em>in vitro</em> are maintained <em>in vivo</em> is not yet known (Lurton <em>et al.</em>, 1999). <br/><br/>Here, we describe an optimized protocol for the isolation of fibroblasts from primary human tissues, allowing immediate analysis without the need for culture <em>in vitro</em>. In brief, primary samples undergo mechanical and enzymatic dissociation, followed by incubation with TrypLE and red cell lysis buffer (to disrupt intercellular adhesions and remove red blood cells, respectively). Use of this approach yields a single-cell suspension consisting of multiple cell types, with approximately a 4-fold greater proportion of fibroblasts compared to other disaggregation strategies (Waise <em>et al.</em>, 2019). We describe use of this protocol for the <em>ex vivo</em> analysis of fibroblasts in both normal and disease states using single-cell RNA sequencing, and highlight alternative downstream applications. In addition, this approach may have applications in the analysis of other cell types (<em>e.g.</em>, epithelial cells).</p>
        </body>

<p class="pptt" id="biaoti30578">## Materials and Reagents</p><ol class="npwtwok"><li>5 ml syringes (BD Plastipak, catalog number: 307731)</li><li>10 ml syringes (BD Plastipak, catalog number: 307736)</li><li>Pasteur pipette (Scientific Laboratory Supplies, catalog number: PIP4210)</li><li>50 ml Falcon tube (Sarstedt, Brand, model/catalog number: 114 x 28 mm, 62.547.004)</li><li>15 ml Falcon tube (Sarstedt, Brand, model/catalog number: 120 x 17 mm, 62.554.002)</li><li>Sterile scalpel #21 blade (Swann-Morton, catalog number: 0507)</li><li>Polystyrene cell culture dish (Sarstedt, catalog number: 83.3902)</li><li>Syringe filtration unit Filtropur S 0.2 (Sarstedt, catalog number: 83.1826.001)</li><li>Scissors</li><li>Collagenase P from <em>Clostridium histolyticum </em>(Merck, Roche, catalog number: 11213857001). Reconstitute in PBS to 150 U/ml, store 100 Î¼l aliquots at -20 Â°C</li><li>TrypLE Express Enzyme (no phenol red; Thermo Fisher, catalog number: 12604013). Store at room temperature protected from light for up to 2 years</li><li>Fetal bovine serum (Biosera, catalog number: FB-1001/500). Store at -20 Â°C for up to 60 months</li><li>Deoxyribonuclease I from bovine pancreas (Merck, Sigma-Aldrich, catalog number: D4263). Reconstitute in 1 ml PBS (2000 U/ml), store 40 Î¼l aliquots at -20 Â°C</li><li>Dulbeccoâs Modified Eagle Medium (Merck, Sigma-Aldrich, catalog number: D5671-500ML). Store at 4 Â°C</li><li>L-glutamine (Merck, Sigma-Aldrich, catalog number: G7513-100ML). Store at -20 Â°C for up to 2 years</li><li>Penicillin-streptomycin (Merck, Sigma-Aldrich, catalog number: P4333-100ML). Store at -20 Â°C for up to 2 years</li><li>Phosphate-buffered saline (PBS)</li><li>Amphotericin B (250 Î¼g/ml; Gibco, catalog number: 15290-018). Store at -20 Â°C for 1 year</li><li>Sterile double-distilled H<sub>2</sub>O</li><li>Red blood cell lysis buffer (10x; BioLegend, catalog number: 420301). Store at 4 Â°C</li><li>DNase stock solutions</li><li>âCompleteâ DMEM (see Recipes)</li><li>âEmptyâ DMEM (see Recipes)</li></ol><p class="pptt" id="biaoti30579">### Equipment</p><ol class="npwtwok"><li>Orbital shaker-incubator (<em>e.g.</em>, Grant-bio Orbital Shaker-Incubator ES-20)</li><li>EASYStrainer 40 Î¼m (Grener Bio-One, catalog number: 542040)</li><li>Centrifuge<br/></li></ol><p class="pptt" id="biaoti30580">### Procedure</p><ol class="npwone"><li>Tissue dissociation<ol class="npwtwo"><li>Sample collection<br/>Transport the tissue sample in âemptyâ DMEM (Recipe 2) on ice</li><li>Prepare working solutions<ol class="npwthree"><li>In a 50 ml Falcon tube, add 100 Î¼l Collagenase P and 40 Î¼l DNase stock solutions to 5 ml âcompleteâ DMEM (Recipe 1)</li><li>Make PBS-A: add 1 Î¼l Amphotericin to 10 ml PBS in a 15 ml Falcon tube </li></ol></li><li>Mechanical dissociation<br/>In a polystyrene Petri dish, incise the tissue 10-12 times to relax the tissue</li><li>Wash sample in PBS-A<ol class="npwthree"><li>Add 5 ml PBS-A to incised tissue, leave at room temperature for 5 min</li><li>This process may be repeated if the sample is particularly congested</li></ol></li><li>Enzymatic dissociation<ol class="npwthree"><li>Using scissors, cut the Pasteur pipette bulb at a 45-degree angle to create a scoop. Use this to remove the tissue from the PBS-A</li><li>Transfer to the 50 ml Falcon tube containing the Collagenase P/DNase solution</li><li>Transfer the 50 ml Falcon tube to the orbital shaker </li><li>Incubate at 37 Â°C with agitation (200 rpm) for 15 min</li><li>After 15 min, remove from the orbital shaker and sequentially pipette with 50 ml, 10 ml and 5 ml pipette tips to promote dissociation</li><li>Return to the orbital shaker for a further 15 min, then repeat sequential pipetting</li><li>Return to the orbital shaker for a further 30 min, then repeat sequential pipetting</li></ol></li><li>TrypLE treatment<ol class="npwthree"><li>Centrifuge the sample at 450 <em>x g</em> for 5 min</li><li>Remove the supernatant and re-suspend the resulting pellet in 1 ml of undiluted TrypLE </li><li>Incubate at 37 Â°C for 10 min</li></ol></li><li>Removing non-digested tissue fragments<ol class="npwthree"><li>Add tissue strainer to a new 50 ml Falcon tube on ice </li><li>Strain tissue/enzyme suspension, pressing through with plunger of 5 ml syringe, simultaneously washing with âemptyâ DMEM</li><li>Keep the sample on ice from this point</li></ol></li><li>Red blood cell lysis<ol class="npwthree"><li>Centrifuge the sample at 450 <em>x g</em> for 5 min at 4 Â°C</li><li>Aspirate the medium</li><li>Re-suspend the pellet in 1 ml red blood cell lysis solution</li><li>Incubate at 4 Â°C for 10 min</li><li>Centrifuge at 450 <em>x g</em> for 5 min at 4 Â°C</li></ol></li><li>Sample collection<br/><ol class="npwthree"><li>Aspirate the red cell lysis buffer</li><li>Re-suspend the pellet in 1 ml of âcompleteâ DMEM</li><li>The cells are now ready for quantification (if necessary) and use <br/></li></ol></li></ol></li></ol><p class="pptt" id="biaoti30581">### Notes</p><ol class="npwtwo"><li>The single-cell suspension may be used for a number of downstream applications, including single-cell RNA sequencing, establishing primary fibroblast cultures, and isolating fibroblast subpopulations. For single-cell RNA sequencing with the Drop-seq platform, re-suspend the cells in double-distilled H<sub>2</sub>O supplemented with 9% Optiprep (Sigma-Aldrich), 1% PBS and 0.1% BSA, and perform as per Macosko <em>et al.</em> (2015) with the following modifications: 500 ng cDNA for PCR, 15 PCR cycles. The number of cells it is feasible to use will depend on the yield from the microfluidic step.</li><li>To establish primary cell cultures, plate 1 x 10<sup>5</sup> cells/cm<sup>2</sup> to tissue culture plates in âCompleteâ DMEM. Incubate in a humidified incubator at 37âÂ°C and 5% CO<sub>2</sub> for 2âh to allow cells to adhere, before washing three times with PBS to remove non-adherent cells. This typically yields a 99.1% pure fibroblast (CD45-EpCAM-CD31-CD90<sup>+</sup>) culture, as determined by flow cytometry analysis. However, users should note that these culture conditions will not maintain <em>in vivo</em> fibroblast phenotypes (Waise <em>et al.</em>, 2019).</li><li>Antibodies directed to specific fibroblast surface markers (<em>e.g.</em>, PDGFR-Î± [Erez <em>et al.</em>, 2010]), in combination with fluorescence-activated or magnetic cell sorting, can be used for isolation of fibroblast subpopulations. It is of note when employing these methods that enzymatic disaggregation can alter surface marker expression (Gray <em>et al.</em>, 2002; Grange <em>et al.</em>, 2011; Quatromoni <em>et al.</em>, 2015), and that no single surface marker will reliably identify or differentiate all fibroblast populations (Sugimoto <em>et al.</em>, 2006; Lambrechts <em>et al.</em>, 2018).</li></ol><p class="pptt" id="biaoti30582">### Recipes</p><ol class="npwtwok"><li>âCompleteâ DMEM<br/>Dulbeccoâs Modified Eagle Medium (Sigma-Aldrich)<br/>10% (v/v) fetal calf serum (Biosera)<br/>1% (v/v) L-glutamine (Sigma-Aldrich)<br/>1% (v/v) penicillin-streptomycin (Sigma-Aldrich)</li><li>âEmptyâ DMEM<br/>Dulbeccoâs Modified Eagle Medium (Sigma-Aldrich) only<br/></li></ol><p class="pptt" id="biaoti30583">### Acknowledgments</p><p class="ppzz">This protocol was derived from previously-published data <sup>28</sup>. This work was supported by Cancer Research UK and Medical Research Council Clinical Research Training Fellowships and a Pathological Society Traineeâs Small Grant to SW. Implementation of Drop-seq was supported by a Medical Research Council Discovery award (MC_PC_15078) and a Southampton Cancer Research UK Centre Development Fund Award to MJJRZ, CHO, JW, CJH &amp; GJT. RP was supported by a John Goldman Fellowship for Future Science (2016/JGF/0003; Leuka Charity) awarded to MJJRZ. The authors thank Evan Macosko, Melissa Goldman and Steve McCarroll for their helpful advice, Dr. Serena Chee (University Hospital Southampton), Benjamin Johnson, Carine Fixmer and Maria Lane (TargetLung Clinical Trials Associates) for enabling access to clinical samples, and the patients involved in this study.</p><p class="pptt" id="biaoti30584">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti30585">### Ethics</p><p class="ppzz">Lung samples were received fresh from patients undergoing surgery at Southampton General Hospital (TargetLung study; approved by NRES Committee South Central: Hampshire A, REC number 14/SC/0186). All research was performed in accordance with the appropriate regulations. Informed consent was obtained from patients or their legal guardians.</p><p class="pptt" id="biaoti30586">### References</p><ol class="npwtwok"><li>Anderberg, C. and Pietras, K. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19395855" target="_blank">On the origin of cancer-associated fibroblasts.</a> <em>Cell Cycle </em> 8(10): 1461-1462.</li><li>Barkauskas, C. E., Cronce, M. J., Rackley, C. R., Bowie, E. J., Keene, D. R., Stripp, B. R., Randell, S. H., Noble, P. W. and Hogan, B. L. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23921127" target="_blank">Type 2 alveolar cells are stem cells in adult lung.</a> <em>J Clin Invest </em>123(7): 3025-3036.</li><li>Comhair, S. A., Xu, W., Mavrakis, L., Aldred, M. A., Asosingh, K. and Erzurum, S. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22427538" target="_blank">Human primary lung endothelial cells in culture.</a> <em>Am J Respir Cell Mol Biol </em>46(6): 723-730.</li><li>Desmouliere, A., Guyot, C. and Gabbiani, G. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15349825" target="_blank">The stroma reaction myofibroblast: a key player in the control of tumor cell behavior.</a> <em>Int J Dev Biol </em>48(5-6): 509-517.</li><li>Erez, N., Truitt, M., Olson, P., Arron, S. T. and Hanahan, D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20138012" target="_blank">Cancer-associated fibroblasts are activated in incipient neoplasia to orchestrate tumor-promoting inflammation in an NF-kappaB-Dependent manner.</a> <em>Cancer Cell </em>17(2): 135-147.</li><li>Ganesan, A. P., Clarke, J., Wood, O., Garrido-Martin, E. M., Chee, S. J., Mellows, T., Samaniego-Castruita, D., Singh, D., Seumois, G., Alzetani, A., Woo, E., Friedmann, P. S., King, E. V., Thomas, G. J., Sanchez-Elsner, T., Vijayanand, P. and Ottensmeier, C. H. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28628092" target="_blank">Tissue-resident memory features are linked to the magnitude of cytotoxic T cell responses in human lung cancer. </a><em>Nat Immunol </em>18(8): 940-950.</li><li>Gray, D. H., Chidgey, A. P. and Boyd, R. L. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11792372" target="_blank">Analysis of thymic stromal cell populations using flow cytometry.</a> <em>J Immunol Methods </em>260(1-2): 15-28.</li><li>Grange, C., Letourneau, J., Forget, M. A., Godin-Ethier, J., Martin, J., Liberman, M., Latour, M., Widmer, H., Lattouf, J. B., Piccirillo, C. A., Cailhier, J. F. and Lapointe, R. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21782822" target="_blank">Phenotypic characterization and functional analysis of human tumor immune infiltration after mechanical and enzymatic disaggregation. </a><em>J Immunol Methods </em>372(1-2): 119-126.</li><li>Hanley, C. J., Mellone, M., Ford, K., Thirdborough, S. M., Mellows, T., Frampton, S. J., Smith, D. M., Harden, E., Szyndralewiez, C., Bullock, M., Noble, F., Moutasim, K. A., King, E. V., Vijayanand, P., Mirnezami, A. H., Underwood, T. J., Ottensmeier, C. H. and Thomas, G. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28922779" target="_blank">Targeting the myofibroblastic cancer-associated fibroblast phenotype through inhibition of NOX4.</a> <em>J Natl Cancer Inst </em>110(1).</li><li>Herrera, M., Islam, A. B., Herrera, A., Martin, P., Garcia, V., Silva, J., Garcia, J. M., Salas, C., Casal, I., de Herreros, A. G., Bonilla, F. and Pena, C. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24052018" target="_blank">Functional heterogeneity of cancer-associated fibroblasts from human colon tumors shows specific prognostic gene expression signature.</a> <em>Clin Cancer Res</em> 19: 5914-5926.</li><li>Hofheinz, R. D., al-Batran, S. E., Hartmann, F., Hartung, G., Jager, D., Renner, C., Tanswell, P., Kunz, U., Amelsberg, A., Kuthan, H. and Stehle, G. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12624517" target="_blank">Stromal antigen targeting by a humanised monoclonal antibody: an early phase II trial of sibrotuzumab in patients with metastatic colorectal cancer.</a> <em>Onkologie</em> 26(1): 44-48.</li><li>Holt, P. G., Robinson, B. W., Reid, M., Kees, U. R., Warton, A., Dawson, V. H., Rose, A., Schon-Hegrad, M. and Papadimitriou, J. M. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3026698" target="_blank">Extraction of immune and inflammatory cells from human lung parenchyma: evaluation of an enzymatic digestion procedure.</a> <em>Clin Exp Immunol</em> 66(1): 188-200.</li><li>Ishii, G., Ochiai, A. and Neri, S. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26278673" target="_blank">Phenotypic and functional heterogeneity of cancer-associated fibroblast within the tumor microenvironment.</a> <em>Adv Drug DelivRev</em> 99(Pt B): 186-196.</li><li>Kalluri, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27550820" target="_blank">The biology and function of fibroblasts in cancer.</a> <em>Nat Rev Cancer</em> 16(9): 582-598.</li><li>Kalluri, R. and Zeisberg, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16572188" target="_blank">Fibroblasts in cancer.</a> <em>Nat Rev Cancer</em> 6(5): 392-401.</li><li>Koumas, L., Smith, T. J., Feldon, S., Blumberg, N. and Phipps, R. P. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14507638" target="_blank">Thy-1 expression in human fibroblast subsets defines myofibroblastic or lipofibroblastic phenotypes.</a> <em>Am J Pathol</em> 163(4): 1291-1300.</li><li>Lambrechts, D., Wauters, E., Boeckx, B., Aibar, S., Nittner, D., Burton, O., Bassez, A., Decaluwe, H., Pircher, A., Van den Eynde, K., Weynand, B., Verbeken, E., De Leyn, P., Liston, A., Vansteenkiste, J., Carmeliet, P., Aerts, S. and Thienpont, B. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29988129" target="_blank">Phenotype molding of stromal cells in the lung tumor microenvironment.</a> <em>Nat Med</em> 24(8): 1277-1289.</li><li>Lurton, J., Rose, T. M., Raghu, G. and Narayanan, A. S. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9922225" target="_blank">Isolation of a gene product expressed by a subpopulation of human lung fibroblasts by differential display.</a> <em>Am J Respir Cell Mol Biol </em>20(2): 327-331.</li><li>Mackay, L. S., Dodd, S., Dougall, I. G., Tomlinson, W., Lordan, J., Fisher, A. J. and Corris, P. A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23425195" target="_blank">Isolation and characterisation of human pulmonary microvascular endothelial cells from patients with severe emphysema.</a> <em>Respir Res</em> 14: 23.</li><li>Macosko, E. Z., Basu, A., Satija, R., Nemesh, J., Shekhar, K., Goldman, M., Tirosh, I., Bialas, A. R., Kamitaki, N., Martersteck, E. M., Trombetta, J. J., Weitz, D. A., Sanes, J. R., Shalek, A. K., Regev, A. and McCarroll, S. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26000488" target="_blank">Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets.</a> <em>Cell</em> 161(5): 1202-1214.</li><li>Mellone, M., Hanley, C. J., Thirdborough, S., Mellows, T., Garcia, E., Woo, J., Tod, J., Frampton, S., Jenei, V., Moutasim, K. A., Kabir, T. D., Brennan, P. A., Venturi, G., Ford, K., Herranz, N., Lim, K. P., Clarke, J., Lambert, D. W., Prime, S. S., Underwood, T. J., Vijayanand, P., Eliceiri, K. W., Woelk, C., King, E. V., Gil, J., Ottensmeier, C. H. and Thomas, G. J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27992856" target="_blank">Induction of fibroblast senescence generates a non-fibrogenic myofibroblast phenotype that differentially impacts on cancer prognosis.</a> <em>Aging (Albany NY)</em> 9(1): 114-132.</li><li>Narra, K., Mullins, S. R., Lee, H. O., Strzemkowski-Brun, B., Magalong, K., Christiansen, V. J., McKee, P. A., Egleston, B., Cohen, S. J., Weiner, L. M., Meropol, N. J. and Cheng, J. D. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18032930" target="_blank">Phase II trial of single agent Val-boroPro (Talabostat) inhibiting Fibroblast Activation Protein in patients with metastatic colorectal cancer.</a> <em>Cancer Biol Ther</em> 6(11): 1691-1699.</li><li>Ohlund, D., Handly-Santana, A., Biffi, G., Elyada, E., Almeida, A. S., Ponz-Sarvise, M., Corbo, V., Oni, T. E., Hearn, S. A., Lee, E. J., Chio, II, Hwang, C. I., Tiriac, H., Baker, L. A., Engle, D. D., Feig, C., Kultti, A., Egeblad, M., Fearon, D. T., Crawford, J. M., Clevers, H., Park, Y. and Tuveson, D. A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28232471" target="_blank">Distinct populations of inflammatory fibroblasts and myofibroblasts in pancreatic cancer.</a> <em>J Exp Med</em> 214(3): 579-596.</li><li>Perrot, I., Blanchard, D., Freymond, N., Isaac, S., Guibert, B., Pacheco, Y. and Lebecque, S. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17312119" target="_blank">Dendritic cells infiltrating human non-small cell lung cancer are blocked at immature stage.</a> <em>J Immunol</em>, 178(5): 2763-2769.</li><li>Quatromoni, J. G., Singhal, S., Bhojnagarwala, P., Hancock, W. W., Albelda, S. M. and Eruslanov, E. 2015. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25359999" target="_blank">An optimized disaggregation method for human lung tumors that preserves the phenotype and function of the immune cells.</a> <em>J Leukoc Biol</em> 97(1): 201-209.</li><li>Rupp, C., Scherzer, M., Rudisch, A., Unger, C., Haslinger, C., Schweifer, N., Artaker, M., Nivarthi, H., Moriggl, R., HengstschlÃ¤ger, M., Kerjaschki, D., Sommergruber, W., Dolznig, H. and Garin-Chesa, P. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24632618" target="_blank">IGFBP7, a novel tumor stroma marker, with growth-promoting effects in colon cancer through a paracrine tumorâstroma interaction.</a> <em>Oncogene</em>, 34(7): 815-825.</li><li>Servais, C. and Erez, N. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22996812" target="_blank">From sentinel cells to inflammatory culprits: cancer-associated fibroblasts in tumour-related inflammation.</a> <em>J Pathol</em> 229(2): 198-207.</li><li>Sugimoto, H., Mundel, T. M., Kieran, M. W. and Kalluri, R. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17106243" target="_blank">Identification of fibroblast heterogeneity in the tumor microenvironment.</a> <em>Cancer Biol Ther</em> 5(12): 1640-1646.</li><li>Tao, L., Huang, G., Song, H., Chen, Y. and Chen, L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28927027" target="_blank">Cancer associated fibroblasts: An essential role in the tumor microenvironment.</a> <em>Oncol Lett</em> 14(3): 2611-2620.</li><li>Waise, S., Parker, R., Rose-Zerilli, M. J. J., Layfield, D. M., Wood, O., West, J., Ottensmeier, C. H., Thomas, G. J. and Hanley, C. J. (2019). <a href="https://www.researchgate.net/publication/334196118_An_optimised_tissue_disaggregation_and_data_processing_pipeline_for_characterising_fibroblast_phenotypes_using_single-cell_RNA_sequencing" target="_blank">An optimised tissue disaggregation and data processing pipeline for characterising fibroblast phenotypes using single-cell RNA sequencing.</a> <em>Sci Rep</em> 9(1): 9580.</li><li>Witowski, J., Kawka, E., Rudolf, A. and Jorres, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26495280" target="_blank">New developments in peritoneal fibroblast biology: implications for inflammation and fibrosis in peritoneal dialysis.</a> <em>Biomed Res</em> Int 2015: 134708.<br/></li></ol>