<body>
            <h1>Localizing Genome Segments and Protein Products of a Multipartite Virus in Host Plant Cells</h1>
            <p><strong>Authors:</strong> Marie-Stephanie Vernerey, Elodie Pirolles, Stephane Blanc, Anne Sicard</p>
            <h2>Abstract</h2>
            <p>A founding paradigm in virology is that the spatial unit of the viral replication cycle is an individual cell. This concept applied to multipartite virusesâwhich have a genome composed of two or more nucleic acid segments, each individually encapsulatedâimplies that all segments constituting a viral genome need to coinfect the same host cell for replication to occur. Would this requirement be verified, it would constitute a major cost for extreme cases of multipartition such as the <em>Faba bean necrotic stunt virus</em> (FBNSV, nanovirus) whose genome is composed of eight complementary segments, each encoding a single gene (Grigoras <em>et al.</em>, 2009). To address this question, we followed the distribution of the FBNSV genome segments by fluorescence <em>in situ</em> hybridization combined to immunolocalization of the replication-controlling viral protein within the cells of the host plant: <em>Vicia Faba.</em><br/><br/>A rapid and efficient protocol to localize viral transcripts in plant and insect hosts has been developed earlier (Ghanim <em>et al.</em>, 2009). We here improve this method by using random-primed labeled probes and apply it to the detection and quantification of the individual segments composing the FBNSV genome. Moreover, we combine this technique with immunolocalization so that both viral segments and proteins can be visualized within the same samples.<br/></p>
            <p><strong>Keywords:</strong> <em>In situ</em>  hybridization (ISH), Immunolocalization, Plant, Virus, Multipartite virus, FBNSV, Fluorescence microscopy, Nanovirus, <em>Vicia Faba</em></p>
            <h2>Background</h2><p>Fluorescence <em>in situ</em> hybridization (FISH) and immunolocalization are essential techniques to localize specific nucleic acids and proteins respectively. These techniques have been successfully used either separately or in combination in a number of studies. Generally, <em>in situ</em> hybridization protocols include numerous steps, resulting in a cumbersome and time-consuming technique. The protocol developed by Ghanim <em>et al.</em> has considerably simplified this technique (Ghanim <em>et al.</em>, 2009). When applied to our pathosystem (FBNSV, <em>Vicia Faba</em>), however, this protocol yielded poor and/or non-specific signals. Although FBNSV is a circular single-stranded DNA phytovirus like the <em>Tomato yellow leaf curl virus</em> (TYLCV) on which Ghanim and coauthors tested their protocol, the FBNSV genome is composed of eight molecules of around 1 kb each, sharing common regions which sometimes correspond to the almost quasi-totality of their non-coding regions (Grigoras <em>et al.</em>, 2010). We thus sought to improve this protocol for the detection of individual FBNSV genomic segments by using, instead of fluorescent oligonucleotides, random primed probes generally used for <em>in situ</em> localization experiments on chromosomes. We were able to couple it with immunolocalization for visualization of nucleic acid and proteins <em>i.e.</em>, covisualization of genetic information and translation products within the same cells. <br/><br/>By combining these two methods, we showed (i) that the FBNSV genomic segments do not colocalize in individual host cells, and (ii) that the viral protein involved in the replication of all segments is often found within cells lacking the segment encoding it; suggesting that viral expression products are able to move away from the cells where they are produced. We believe that our methodology could easily be adapted to locate and quantify (single-stranded) nucleic acids in different tissues with great specificity. For instance, this protocol has also been successfully used to locate the FBNSV segments within gut cells of its aphid vector and TYLCV genome within host plants.</p>
        </body>

<p class="pptt" id="biaoti30614">## Materials and Reagents</p><p class="ppzz"><em>Note: Some of the reagents and materials described below are optional depending on the targeted sequence and tissue (see Procedure below for further details).</em> </p><ol class="npwtwok"><li>Microscopy slides (Dutscher, catalog number: 068763)</li><li>Coverslips (Dutscher, catalog number: 100031)</li><li>Nylon membrane 0.45 Âµm pore size (Sigma, catalog number: GERPN203T) </li><li>Whatman paper</li><li>24-well plate (Dutscher, catalog number: 353226) </li><li>Microcentrifuge tube (Dutscher, catalog number: 033297)</li><li>Embryo dish 30 mm (Delta microscopies, Ems, catalog number: 70543-30)</li><li>Wizard SV Gel PCR Clean-Up System (Promega, catalog number: A9282)</li><li>GoTag DNA polymerase kit (Promega, catalog number: M3001)</li><li>Paraformaldehyde (Delta microscopies, Ems, catalog number: 15713), store at room temperature</li><li>PBS buffer 10x pH 7.4 (Dutscher, biosolve, catalog number: 91549), store at room temperature</li><li>Tween 20 (Merck, Sigma-Aldrich, catalog number: P1379), store at room temperature</li><li>Glycine (Merck, Sigma-Aldrich, catalog number: G8898), store at room temperature</li><li>Ethanol (AnalyticLab, Honeywell, catalog number: 32221), store at room temperature</li><li>Chloroform (Merck, Sigma-Aldrich, catalog number: 32211), store at room temperature</li><li>Acetic acid (Merck, Sigma-Aldrich, catalog number: 33209), store at room temperature</li><li>Low melting agarose (Ozyme, Lonza, catalog number: LON50080), store at room temperature</li><li>RNase (Merck, Sigma-Aldrich, catalog number: R6513), store at -20 Â°C</li><li>Tris (Carl roth, Pufferan, catalog number: AE15.3), store at room temperature</li><li>NaCl 5 M (Merck, Sigma-Aldrich, catalog number: S3014), store at room temperature</li><li>Sodium dodecyl sulfate (SDS) 10% (Dutscher, Biosolve, catalog number: 19812323), store at room temperature</li><li>Deionized Formamide (Eurobio, catalog number: GHYFOR01-01), aliquot and store at -20 Â°C after opening</li><li>BioPrime DNA labeling system (Fisher Scientific, Invitrogen, catalog number: 18094-011), store at -20 Â°C</li><li>Chromatide Alexa Fluor 488-5-dUTP (Fisher Scientific, Molecular probes, catalog number: 10173952), store at -20 Â°C</li><li>Chromatide Alexa Fluor 568-5-dUTP (Fisher Scientific, Molecular probes, catalog number: 10413222), store at -20 Â°C</li><li>Herring sperm DNA (Sigma, catalog number: D3159), store at -20 Â°C</li><li>Bovine serum albumin (Carl Roth, catalog number: 8076.4), store at 4 Â°C</li><li>Goat anti rabbit Alexa Fluor 488 IgG (H+L) (Fisher Scientific, Invitrogen, catalog number: 10082502), store in 50 % glycerol at -20 Â°C</li><li>Goat anti rabbit Alexa Fluor 594 IgG Conjugate (Fisher Scientific, Invitrogen, catalog number: 10266352), store in 50 % glycerol at     -20 Â°C</li><li>Vectashield mounting medium with DAPI (Eurobio, Vector, catalog number: H-1200), store at 4 Â°C</li><li>Carnoy 6:3:1 (Recipe 1)</li><li>RNase solution 100 Âµg/ml (Recipe 2)</li><li>dNTP fluo mix for the Bioprime DNA labeling (Recipe 3)</li><li>Hybridisation buffer (Hb Buffer) (Recipe 4)</li><li>1 M Tris-HCl, pH 8 (Recipe 5)</li><li>5 M NaCl (Recipe 6)</li><li>0.1 M glycine in PBS 1x (Recipe 7)</li><li>5% Bovine Serum Albumin (BSA) (Recipe 8)<br/></li></ol><p class="pptt" id="biaoti30615">### Equipment</p><ol class="npwtwok"><li>Vibratome (MICROM, model HM650V)</li><li>Confocal microscope (ZEISS, model LSM700)</li><li>Phosphorimager (<a class="blackNoUnder" target="_blank"> Typhoon FLA 9000 </a> )</li><li>UV light (254 nm) (UVllite, UVItec CL-E508.G)</li><li>37 Â°C oven</li><li>Water bath</li><li>Rotator<br/></li></ol><p class="pptt" id="biaoti30616">### Software</p><ol class="npwtwok"><li>Zeiss Efficient Navigation (ZEN) (ZEISS, version 2009) </li><li>Image J (National Institutes of Health, version 1.50 c, <a href="https://imagej.nih.gov/ij/download.html" target="_blank">https://imagej.nih.gov/ij/download.html</a>)</li><li>JMP (version 13.2.0, <a href="https://www.jmp.com/fr_fr/home.html" target="_blank">https://www.jmp.com/fr_fr/home.html</a>)<br/></li></ol><p class="pptt" id="biaoti30617">### Procedure</p><ol class="npwone"><li>Samples preparation<ol class="npwtwo"><li>Samples fixation<ol class="npwthree"><li>Collect samples (see the note* below for the choice of samples) in PBS 1x and add paraformaldehyde (PFA) right after sampling to obtain a solution of PFA at 4% in PBS 1x with 0.2% Tween-20**. We recommend taking at least one sample from at least 10-12 individual plants coming from three different inoculation trials. For comparison purposes, samples should be collected from plants at the same development and infection stages. Incubate under agitation overnight at 4 Â°C. <br/><em>Notes:<ol class="npwfour"><li>*Choose the samples to collect carefully by thinking in which parts you are most likely to find virus accumulations. In our case, we choose to work on petioles.</li><li>**Pay attention to use a volume of fixator high enough to completely cover the fragment collected: use at least 2/3 volume of fixator for 1/3 volume of sample.</li></ol></em> </li><li>On the next day, neutralize PFA with a PBS 1x solution containing 0.1 M glycine (Recipe 7). Incubate for 15 min at room temperature (RT).<br/><em>Note: The volume of âPBS 1x solution containing 0.1 M glycineâ should be adapted to the sample you are working with (petioles, leaves, rootsâ¦); samples must always be entirely covered by the liquid.</em> </li><li>Rinse the samples for 5 min with PBS 1x. Replace PBS with a 70% ethanol solution. Samples can be stored in this solution at 4 Â°C for up to a month.</li></ol></li><li>Samples sectioning<br/><em>Note: Depending on the localization of the nucleic acid of interest, and on the tissue or organ analyzed, this step could be optional. </em> <ol class="npwthree"><li>Embed the fixed samples in 8% low melting agarose (diluted in either distilled water or PBS 1x). Pour the liquid agarose in a 24-well plate, or anything else convenient for the samples, until the agarose completely covers the sample. Place the tissue fragments straight in the middle of the well and let the agarose solidify at 4 Â°C for at least 4 h (Figure 1).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191204/20191204193658_9836.jpg" width="350"/><br/><strong>Figure 1. Embedded samples.</strong> <em>Vicia faba</em> petioles embedded in 8% low melting agarose in a 24-well plate before sectioning.<br/><br/></li><li>Proceed to the sectioning with a vibratome. For instance, the following parameters were used to section <em>Vicia faba</em> petioles using the MICROM vibratome:<br/>thickness: 100 Âµm<br/>frequency: 100 Hertz<br/>program: 50<br/>amplitude: 0.6 mm<br/>method: one cut after another<br/>speed: 23 mm/s<br/>All of the above parametersâincluding the percentage of agaroseâshould be adjusted according to the hardness of the sample. For example, in the case of a softer sample, decrease the percentage of agarose. Based on our experience, 8% is around the limit of agarose solubility. </li></ol></li><li>Samples discoloration<br/>For plant samples, bleach them by incubating them under gentle agitation at RT in a Carnoy 6:3:1 solution (Recipe 1). As soon as the samples have totally lost their green color (step unnecessary for tissues such as roots explants), stop the reaction by replacing the Carnoy solution with a PBS 1x solution. For instance, bleaching of <em>Vicia faba</em> petioleâs sections takes an hour (Figure 2). <br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191204/20191204193720_6283.jpg" width="400"/><br/><strong>Figure 2. Effect of Carnoy 6:3:1 discoloration on sectioned samples.</strong> <em>Vicia faba</em> sectioned petioles before (left) and after (right) discoloration using the Carnoy solution. <br/><br/></li><li>RNase or DNase treatment<br/>Depending on the nature of the genetic material you want to localize, you can proceed to an RNase or DNase treatment. Since FBNSV is a single-stranded DNA virus and because we were interested in localizing its genomic segments only, we RNase-treated our samples.<ol class="npwthree"><li>Immerge the samples in 500 Âµl of a 100 Âµg/ml RNase solution (Recipe 2) for 45 min at 37 Â°C. </li><li>Rinse 1 time for 5 min in PBS 1x.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Probes preparation<br/>DNA amplicons (Table 1) corresponding to the coding regions of each genomic segments were amplified by PCR using the GoTaq Polymerase kit (Promega) and the conditions summarized in Table 2. PCR products were then run on a 1% agarose gel and gel purified using the Wizard SV Gel and PCR Clean-Up System (Promega). These amplicons were then used as templates to produce segment-specific probes. DNA labeling was performed using the Bioprime DNA labeling Kit of Invitrogen. Two modifications were made to the kit protocol based on the recommendations of the Plant molecular cytogenetic platform of Rennes (INRA) and of the Cytogenetic and the imaging center of UMR AGAP, Montpellier CIRAD (Personal communications). First, for direct labeling, Biotine molecules were replaced with Alexa Fluor enabling a direct labeling. Second, the incubation time was increased to allow enhanced labeling. <br/>  Below is the kit labeling procedure with the two changes made (in bold font). All of the reagents are included in the kitâexcept for the dNTP Fluo mix (Recipe 3). <ol class="npwtwo"><li>Dissolve 25 to 500 ng (we used 200 ng of PCR product) of DNA in 5-20 Î¼l of dilute buffer in a microcentrifuge tube. On ice, add 20 Î¼l of 2.5x Random Primers Solution. Denature by heating for 10 min in a boiling water bath. Immediately cool on ice for 15 min. <ol class="npwthree"><li><strong>10 Î¼l of dNTP Fluo mix (instead of the 10x Biotine dNTP mixture, see Recipe 3 for preparation). </strong> </li><li>And distilled Water to a total volume of 49 Î¼l. </li></ol></li><li>On ice, add (to the DNA and primers mixture prepared in Step B1): </li><li>Mix briefly by vortexing or pipetting.</li><li>Add 1 Î¼l of Klenow Fragment. Mix gently but thoroughly. Centrifuge at maximum speed (no specific recommendation is included in the manufacturer notice, when we proceed to it, we do it at maximum speed of a bench centrifuge) for 15-30 s.</li><li><strong>Incubate at 37 Â°C for at least 12 h.</strong> </li><li>Add 5 Î¼l of Stop Buffer and mix.<br/>The probes can be stored at -20 Â°C for months. The probes chosen must be specific of the DNA or RNA sequence you want to localize. The specificity of the probe can be checked following the procedure detailed in the next section. <br/><br/><strong>Table 1. Primers used to prepare segment-specific fluorescent probes</strong> (from Sicard <em>et al.</em>, 2019)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" frame="hsides" style="width:100%;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td>Name<br/></td><td>Primer sequence<br/></td><td> Size<br/></td><td>   Segment<br/></td></tr><tr></tr></tbody><tbody><tr><td>NV C-297 F<br/></td><td>ATGGGTCTGAAATATTTCTC<br/></td><td>510nt <br/></td><td>   C<br/></td></tr><tr><td>NV C-806 R <br/></td><td>TTAATTAATTACAATCTCC<span> </span><br/></td><td><br/></td><td><br/></td></tr><tr><td>M ORF For2<br/></td><td>GCTGCGTATCAAGACGAC<br/></td><td>268nt<br/></td><td>   M<br/></td></tr><tr><td>M ORF Rev2<br/></td><td>TTCTAGCATCCCAATTCC<br/></td><td><br/></td><td><br/></td></tr><tr><td>N ORF For<br/></td><td>TGGCAGATTGGTTTTCTAGT<br/></td><td>450nt<br/></td><td>   N<br/></td></tr><tr><td>N ORF Rev<br/></td><td>TTCTGAGTGAATGTACAATAAACATTT<br/></td><td><br/></td><td><br/></td></tr><tr><td>R ORF For<br/></td><td>ACATTAAATAATCCTCTCTCTCCTA<br/></td><td>810nt<br/></td><td>   R<br/></td></tr><tr><td>R ORF Rev<br/></td><td>CCTATCATCACTAAACATGCC<br/></td><td><br/></td><td><br/></td></tr><tr><td>S ORF For<br/></td><td>AAATGGTGAGCAATTGGAA<br/></td><td>350nt <br/></td><td>   S<br/></td></tr><tr><td>S ORF Rev<br/></td><td>GCCTATGATAGTAATCATATCTTGACA<br/></td><td><br/></td><td><br/></td></tr><tr><td>U1 ORF For<br/></td><td>TTGGTCGATTATTTGTTGGTT<br/></td><td>455nt<br/></td><td>   U1<br/></td></tr><tr><td>U1 ORF Rev<br/></td><td>AATATCTCATTAGCATTAATTACATTTGAA<br/></td><td><br/></td><td><br/></td></tr><tr><td>U2 ORF For<br/></td><td>TTATGGATGCCGGCTTT<br/></td><td>361nt<br/></td><td>   U2<br/></td></tr><tr><td>U2 ORF Rev<br/></td><td>CATGAAGTATTAGAATAACGAACTTGA<span> </span><br/></td><td><br/></td><td><br/></td></tr><tr><td>U4 ORF For<br/></td><td>AGCAGGTTATGGAATGTAGG<br/></td><td>313nt<br/></td><td>   U4<br/></td></tr><tr><td>U4 ORF Rev<br/></td><td>ATAGATTCCCACAATCGCT<br/></td><td><br/></td><td><br/></td></tr><tr><td>R1 For<br/></td><td>ACATTAAATAATCCTCTCTCTCCTA<br/></td><td>282nt<br/></td><td>   R<br/></td></tr><tr><td>R1 Rev<br/></td><td>CTCTAAGCGTACTTGCAG<br/></td><td><br/></td><td><br/></td></tr><tr><td>R2 For<br/></td><td>GAAGAAAGCTATTGCTACTTGG<br/></td><td>406nt<span> </span><br/></td><td><br/></td></tr><tr><td>R2 Rev <br/></td><td>CCTATCATCACTAAACATGCC<br/></td><td><br/></td><td><br/></td></tr><tr><td>Nmys2-Red<br/></td><td>TTACATCATAAGTGCTGTAAGTATAATAAGGCACAACTAA<br/></td><td>40 nt<br/></td><td>   N<br/></td></tr><tr><td>Nmys7-Green<br/></td><td>TTACATCATATTATACCAACTGTTGCTATCCCACAACTAA<br/></td><td>40 nt<br/></td><td>   N<br/></td></tr><tr><td>Smys1-Red<br/></td><td>GTTTTTCATTAACTTCTCACTATAATAACTGCCCACAGTG<br/></td><td>40 nt<br/></td><td>   S<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Smys8-Green<br/></td><td>GTTTTTCATTAATATAAGTCATAATTCCTGCTCCACAGTG<br/></td><td>40 nt<br/></td><td>   S</td></tr></tbody></table><table border="0" bordercolor="#000000" class="ke-zeroborder" rules="none" style="width:100%;"></table><br/><strong>Table 2. PCR conditions followed to prepare the segment-specific fluorescent probes.</strong> Note that these conditions need to be adjusted according to the polymerase used, the primers and the size of the amplicons. <br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" frame="hsides" style="width:800px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Reagents</strong><br/></td><td><br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;"><strong>Procedure</strong><br/></td><td><br/></td><td><br/></td></tr><tr><td>Components<br/></td><td>Final concentrations<br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;">Step<br/></td><td>Temperature<br/></td><td>Time<br/></td></tr><tr><td>GoTaq reaction buffer<br/></td><td>1x<br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;">Initial Denaturation<br/></td><td>95 Â°C<br/></td><td>2 min<br/></td></tr><tr><td>GoTaq DNA polymerase<br/></td><td>0.5 unit<br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;">40 Cycles <br/></td><td>95 Â°C<br/></td><td>45 s<br/></td></tr><tr><td>dNTP 0.2 mM each<br/></td><td>0.2 mM each<br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;"><br/></td><td>59 Â°C<br/></td><td>45 s<br/></td></tr><tr><td>Primers Forward + Reverse<br/></td><td>0.5 Î¼M<br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;"><br/></td><td>72 Â°C<br/></td><td>45 s<br/></td></tr><tr><td>Template DNA <br/></td><td><br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;"><br/></td><td>72 Â°C<br/></td><td>5 min<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Nuclease-free water<br/></td><td><br/></td><td style="border-top:0pt;border-left:1pt;border-bottom:0px;border-right:0;border-style:solid;">Hold<br/></td><td>4-10 Â°C <br/></td><td><br/></td></tr></tbody></table></li></ol></li><br/><li>Specificity of the probe<ol class="npwtwo"><li>First denature samples (mandatory when starting with double-stranded nucleic acid) by incubating them at 95 Â°C (or in boiling water) for 10 min immediately followed by placing them on ice. </li><li>Each of the denatured samples is pipetted on a nylon membrane and allowed to dry. The edges and names of each drop are penciled directly on the membrane (Figure 3).<br/>The samples may include DNA extracted from the tissue of interest containing or not the targeted nucleic acid as well as some purified DNA containing (or not) the targeted nucleic acid (<em>e.g.</em>, a plasmid containing the gene of interest). For instance, we spotted 1 Î¼g of each of the eight plasmids, each containing one of the eight FBNSV segments, separately on the membrane. 1 Î¼l of DNA extract from FBNSV infected and uninfected plants were also added to the membrane. Each sample should contain at least 45 ng of the target in order to be detected. </li><li>Fix the DNA onto the membrane using ultraviolets for a few seconds (UV linker, UVllite, UVItec). </li><li>Pre-hybridize the nylon membranes for 2 h at 37 Â°C on a rotator using Hb buffer (Recipe 4) containing 1% (w/v) of BSA and 2 mg of denatured herring sperm DNA.</li><li>Add 10 Î¼l of the denatured probe previously obtained (Procedure B). Denature samples by heating for 10 min at 95 Â°C (or in boiling water) followed by an immediate cooling on ice for 15 min.</li><li>Incubate for at least 2 h at 37 Â°C on a rotator. This incubation step can also be done overnight.</li><li>Rinse the membranes three times using hybridization buffer, for 5 min each, and once with PBS 1x for 5 min. Do the rinse steps at room temperature.</li><li>Dry the membranes with Whatman paper.</li><li>Results can be visualized using a phosphorimager.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191204/20191204193732_3964.jpg" width="400"/><br/><strong>Figure 3. Results obtained using the R1 probe (upper picture) and the R2 probe (lower picture) after revelation with the phosphorimager.</strong> 1 Î¼g of each purified plasmid DNA containing one of the eight FBNSV genomic segments (C, M, N, R, S, U1, U2, U4) were spotted on a nylon membrane. 200 ng of PCR purified product corresponding to two different regions of the segment R (R1 and R2) were also spotted on the membrane. Finally, 1 Î¼l of DNA extracted from a healthy plant (Vfs) and 1 Î¼l of DNA extracted from an FBNSV-infected plant (Vfi) were used. The upper and lower membranes were incubated with the R1 and R2 probes respectively. The results obtained confirm the high specificity of each probe. Note that no signal was obtained here for the FBNSV-infected plant sample when using either of the two R probes due to the very low concentration of the segment R in the amount of plant DNA used.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>In situ</em> hybridization<ol class="npwtwo"><li>Rinse the samples obtained in Procedure A for 5 min, 3 times with Hybridisation Buffer (Hb buffer, Recipe 4). </li><li>Dilute 20 Âµl of labeled DNA probe (with Alexa Fluor 488 or/and Alexa Fluor 568) in Hb buffer in a total volume of 600 Âµl. Adjust the volume to the amount of material. For example, we used 300 Âµl of this solution in an embryo dish of 30 mm.</li><li>Since the labeled probes obtained by Random priming (Procedure B) are double-stranded, denature the solution for 10 min at 95 Â°C or in boiling water and cool immediately for 15 min on ice. Incubate the samples with the probe solution overnight at 37 Â°C in a dark place (<em>e.g.</em>, incubator). As a general rule, since the fluorochromes are light-sensitive, avoid prolonged exposure to light. </li><li>Recover the probe solution. In our hands, this solution could be used up to 4 times with no significant decrease of the intensity of the signal. Store the solution at -20 Â°C in a tube protected from light.</li><li>Rinse the samples for 5 min 3 times in Hb Buffer (Recipe 4).</li><li>Rinse 2 times more with PBS 1x for 5 min.<br/><em>Note: If no other experiment is foreseen, prepare slides for microscopy observations (Procedure F).</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Immunolocalization <br/>Immunolocalization protocol can immediately follow <em>in situ</em> hybridization (Figure 4).<ol class="npwtwo"><li>Incubate the samples in BSA 5% in PBS 1x solution (Recipe 8) for 1.5 h at room temperature (RT) to block aspecific sites. </li><li>Split samples into two batches. Incubate one batch with a fresh PBS-BSA 5% solution containing the primary antibody (diluted at the ad hoc concentration). For the second batch, simply replace the PBS-BSA 5 % with a fresh solution (this represents the negative control). Incubate overnight at 4 Â°C. <br/><em>Note: Regarding the immunolocalization experiment, you can first try using the primary antibody at a concentration 10x higher than the one used in Western blots. Then, adjust accordingly.</em> </li><li>Rinse all samples 3 times each for 10 min in PBS 1x + 0.05% Tween-20.</li><li>Replace the solution by a PBS-BSA 5% solution containing the secondary antibodyâdiluted at the required concentration (see manufacturer instruction). In our experiments, we used a goat anti-rabbit IgG Conjugate Alexa Fluor 488 at a concentration of 2 Âµg/ml and a goat anti-rabbit IgG Conjugate Alexa Fluor 594 at 5 Âµg/ml. Incubate for 1 h at 37 Â°C.</li><li>Rinse 3 times each for 10 min in PBS 1x + 0.05% Tween-20.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191204/20191204193745_1691.jpg" width="400"/><br/><strong>Figure 4. Confocal microscopy pictures of double labeling (FISH/Immuno) experiment obtained following our protocol.</strong> Cross sections of infected faba bean petioles stained with DAPI (in blue) and hybridized with the S-A488 probe in green and M-Rep protein antibody in red. A. Green channel (S probe = part of the segment encoding the capsid protein) merged with the blue channel (DAPI). B. Red channel (M-Rep protein localization = immunolabeling of the movement protein) merged with the blue channel (DAPI). C. Merged image of the three channels. D. Merged picture of the three channels of the control sample (petiole of non-infected faba bean plant) taken in the same conditions as Figure 4C; only autofluorescence of the tissue and blue signal of the DAPI nuclei staining could be observed.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of microscopy slides<br/>For microscopy observations, mount samples in Vectashield with DAPI (if interested in localizing cell nuclei) or any other mounting media (<em>e.g.</em>, glycerolâ¦, Figure 5). <br/><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191204/20191204193759_6776.jpg" width="400"/><br/><strong>Figure 5. Microscopy slide ready to be observed and mounted as described in the note below.</strong> A Vaseline chamber was made prior to adding the mounting medium and the cross section to avoid movement of the preparation during long image captures.<br/><em>Note: For microscopy slides, depending on the thickness of your samples, you can use Vaseline to make a kind of âincubation chamberâ on the slide before adding the mounting media and the coverslip. </em> </p><div><br/></div></li><li>Microscopy observations<br/>All observations were made using an LSM700 Confocal Microscope (ZEISS) with the ZEN Software. Parameters were adjusted to obtain sufficient resolution and fluorescence intensity signal recovery-in a chosen series of infected plant exhibiting a high intensity of fluorescence-without saturation points. Once these parameters were set, all of the images were acquired using the same parameters so that they could easily be compared with one another. <br/>    In our case, images were taken with a 40x water immersion objective with a resolution of at least 512 x 512 and with a pinhole aperture of 1 Airy Unit so as to work in confocal mode. <br/>    We set on a configuration with 3 sequential tracks, one for each fluorochrome used.<ol class="npwtwo"><li>Excitation parameters were as followed:<br/>laser 405 nm for DAPI <br/>laser 488 nm for Alexa Fluor 488<br/>laser 555 nm for Alexa Fluor 568<br/></li></ol><p "="" class="ppzz">Fluorescence emission was collected on the photomultiplicator 1 and the variable secondary dichroic beam splitter was set so as to recover fluorescence up to:</p></li><ol class="npwtwo" start="2"><li>490 nm for DAPI</li><li>535 nm for Alexa 488</li><li>615 nm for Alexa 568<br/></li></ol><p class="pptt" id="biaoti30618">### Data analysis</p><p class="ppzz">Analyses were run using maximum intensity projections so that all the fluorescence emitted in the whole nuclei was accounted for. These microscopic images were analyzed using Image J software.</p><ol class="npwtwok"><li>First of all, in order to minimize the signal caused by the autofluorescence of the tissuesâeither natural or induced by fixation, grey level intensities coming from the green and red tracks were decreased independently on each photo until no green or red fluorescence could be visually detected in tissues where the FBNSV is known to be absent (<em>i.e.</em>, in xylem and mesophyll; the FBNSV being a phloem restricted virus). This computer process was done in the exact same way for all pictures and applied equally to the whole surface of the image (Figure 6).<br/><em>Note: The aim of this step was to simply distinguish infected cells from non-infected ones with a high level of background autofluorescence. As a consequence, only cells in which green or red fluorescence levels exceeded the level of background fluorescence were considered for further analysis. As we were mainly interested in the cells in which our ssDNA virus can replicate, we focused our analyses on phloem nucleated cells; nuclei stained using DAPI. </em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191204/20191204193813_2076.jpg" width="400"/><br/><strong>Figure 6. Confocal microscopy pictures obtained following our protocol.</strong> Cross sections of infected faba bean petioles stained with DAPI (in blue) and hybridized with the R-A488 probe in green and N-A568 probe in red. A. Green channel (R probe = part of the segment encoding the replication protein) merged with the blue channel (DAPI). B. Red channel (N probe = part of the segment encoding the nuclear shuttle protein) merged with the blue channel (DAPI). C. Merged image of the three channels. D. Merged picture of the three channels of the control sample (petiole of non-infected faba bean plant) taken in the same conditions as picture C; only autofluorescence of the tissue and blue signal of the DAPI nuclei staining could be observed.<br/><br/></li><li>On the images, we manually delineated the DAPI stained area (corresponding to the nucleus in each cell) using a cursor and quantified intensity of grey level values for each pixel using the ImageJ software. The intensity of grey level was also determined for both green and red fluorescence. </li><li>We then calculated the average pixel fluorescence intensity value for each nucleus and each fluorochrome. We thus obtained an average intensity value (in arbitrary fluorescence unit) for both green and red fluorescence for each selected individual cell within an infected petiole (the FBNSV replicate and accumulate in the nucleus of infected cells).</li><li>In order to check the reliability of our analysis, we produced two probes respectively targeting the two moieties of the same segment R: âprobe-R1â and âprobe-R2â, one labeled green and the other red. In such a control experiment, we could confirm that absolutely all nuclei with a significant red signal also had a significant green signal. When quantifying these green and red fluorescence, we observed a highly significant correlation, logically showing that the accumulation of one part of a genome segment within a cell is clearly correlated to that of the other part of the same segment. Around 50 infected cells were analyzed by condition.</li><li>Images acquired from the same slide with different resolutions (512 x 512 and 1024 x 1024) and at different times post labeling (2 weeks between the two-time points) were analyzed and led to the same results.</li><li>Linear regression analyses were performed with the JMP software.<br/>All details of the data analysis can be found in the original research article (Sicard <em>et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti30619">### Notes</p><ol class="npwtwok"><li>Always use both infected tissues and uninfected tissues as controls.</li><li>A minimum of 3 repetitions per condition should be run. <br/></li></ol><p class="pptt" id="biaoti30620">### Recipes</p><ol class="npwtwok"><li>Carnoy 6:3:1<br/>6 ml chloroform<br/>3 ml ethanol<br/>1 ml acetic acid</li><li>100 Âµg/ml RNase solution <br/>Dilute your RNase solution with PBS 1x so as to obtain a 100 Âµg/ml solution</li><li>dNTP fluo mix for the Bioprime DNA labeling<br/>Dilute dNTP solutions in H<sub>2</sub>O so as to obtain 1 mM solution of dATP, dGTP, dCTP and 0.5 mM of dTTP and dUTP-Alexa<br/>For example, mix:<br/>5 Âµl of 10 mM dATP<br/>5 Âµl of 10 mM dGTP<br/>5 Âµl of 10 mM dCTP<br/>2.5 Âµl of 10 mM dTTP<br/>25 Âµl of 1 mM dUTP-Alexa (Alexa 488, 568 or others depending on the microscope equipment)<br/>7.5 Âµl of H<sub>2</sub>O</li><li>Hb Buffer (Ghanim <em>et al.</em>, 2009)<br/>20 mM Tris HCl, pH 8<br/>0.9 M NaCl<br/>0.01% (w/v) SDS<br/>30 % (v/v) deionized formamide<br/>pH = 8<br/><strong>For a volume of 100 ml:</strong><br/>2 ml Tris HCl pH8 1 M<br/>18 ml NaCl 5 M<br/>100 Âµl SDS 10%<br/>30 ml deionized formamide<br/>50 ml H<sub>2</sub>O</li><li>1 M Tris-HCl, pH 8<br/>121.1 g of Tris base<ol class="npwthree"><li>Complete with water to 800 ml</li><li>Adjust to pH 8 by adding concentrated HCl (around 42 ml)</li><li>Adjust the volume of the solution to 1 L with H<sub>2</sub>O</li></ol></li><li>5 M NaCl<br/>292.2 g NaCl<br/>Complete with water to 1 L</li><li>0.1 M glycine in PBS 1x<br/>Dissolve 7.5 g of glycine in PBS 1x to a final volume of 1 L </li><li>Bovine Serum Albumin (BSA) 5%<br/>Dissolve 5 g of BSA in 10 ml of PBS 1x<br/></li></ol><p class="pptt" id="biaoti30621">### Acknowledgments</p><p class="ppzz">We are thankful to the PHIV platform (CIRAD, Montpellier, France) for sharing their immunolocalization protocol.</p><p class="pptt" id="biaoti30622">### Competing interests</p><p class="ppzz">No competing interests to declare.</p><p class="pptt" id="biaoti30623">### References</p><ol class="npwtwok"><li>Ghanim, M., Brumin, M. and Popovski, S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19406154" target="_blank">A simple, rapid and inexpensive method for localization of Tomato yellow leaf curl virus and Potato leafroll virus in plant and insect vectors.</a> <em>J Virol Methods</em> 159(2): 311-314.</li><li>Grigoras, I., Timchenko, T., Katul, L., Grande-Perez, A., Vetten, H. J. and Gronenborn, B. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19656882" target="_blank">Reconstitution of authentic nanovirus from multiple cloned DNAs.</a> <em>J Virol</em> 83(20): 10778-10787.</li><li>Grigoras, I., Timchenko, T., Grande-Perez, A., Katul, L., Vetten, H. J. and Gronenborn, B. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20592071" target="_blank">High variability and rapid evolution of a nanovirus.</a> <em>J Virol</em> 84(18): 9105-9117.</li><li>Sicard, A., Pirolles, E., Gallet, R., Vernerey, M. S., Yvon, M., Urbino, C., Peterschmitt, M., Gutierrez, S., Michalakis, Y. and Blanc, S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30857590" target="_blank">A multicellular way of life for a multipartite virus.</a> <em>Elife</em> 8: e43599.<br/></li></ol></ol>