<body>
            <h1>DNase I Chromatin Accessibility Analysis</h1>
            <p><strong>Authors:</strong> Brook S. Nepon-Sixt, Mark G. Alexandrow</p>
            <h2>Abstract</h2>
            <p>Chromatin consists of compacted DNA in complex with proteins and contributes to the organization of DNA and its stability. Furthermore, chromatin plays key roles in regulating cellular processes such as DNA replication, transcription, DNA repair, and mitosis. Chromatin assumes more compact (inaccessible) or decondensed (accessible) conformations depending on the function that is being supported in the genome, either locally or globally. The activity of nucleases has been used previously to assess the accessibility of specific genomic regions <em>in vitro</em>, such as origins of replication at varying points in the cell cycle. Here, we provide an assay to determine the accessibility of specific human genomic regions (example used herein: Lamin B2 origin of DNA replication) by measuring the effect of DNase I nuclease on qPCR signal from the studied site. This assay provides a powerful method to interrogate the molecular mechanisms that regulate chromatin accessibility, and how these processes affect various cellular functions involving the human genome that require manipulation of chromatin conformation.</p>
            <p><strong>Keywords:</strong> Chromatin access, Origin of DNA replication, DNase I, qPCR, Lamin B2, Chromatin conformation</p>
            <h2>Background</h2><p>Studies aiming to elucidate the molecular underpinnings of genomic processes influenced by chromatin have utilized nuclease sensitivity to assess the accessibility of chromatin. This is a powerful concept for interrogating the regulation and timing of chromatin conformational changes that contribute to cellular processes such as DNA replication and transcription. For example, it has been shown that chromatin is more sensitive to micrococcal nuclease at two distinct sites within the regions encompassing origins of DNA replication in Chinese hamster ovary cells during G<sub>1</sub> versus S-phase, suggesting that chromatin is more accessible at these origins in G<sub>1</sub> phase (Pemov <em>et al.</em>, 1998). However, the approach used in this latter study required laborious techniques, Southern blotting, large-scale and complex genomic DNA collections and restriction digests, and cumbersome gel-electrophoresis and membrane-transfer methods. In addition, the ability to interpret the results was limited, and only allowed one to observe changes to a single site in a larger locus, with no other changes in nuclease (chromatin) accessibility being detectable throughout the region. Older more classical techniques, such as nuclease-protection footprinting assays, can give more insight into specific loci in terms of chromatin changes. For example, the positioning or rearranging of nucleosomes can be detected by such techniques in a small region. However, these footprinting techniques also require laborious methods by the investigator, and are not amenable to asking questions regarding large-scale changes to chromatin accessibility in specific regions of interest in the genome.<br/><br/>To overcome many of the limitations of older techniques, our group developed a more simple and efficient method of detecting differences in large-scale chromatin accessibility for any site in the genome that an investigator wishes to assess. Our approach relies on the accessibility of DNase I to perform an initial limited digestion of genomic DNA in the context of chromatin prior to using such DNA as a substrate for an efficient quantitative real-time PCR (qPCR) analysis. No radioactivity or cumbersome gel-electrophoretic methods are required, and isolation of chromatin/DNA for nuclease digestion is straightforward. Analysis is not complicated, and the results are obtained using the rationale that the more DNase I can pre-digest the chromatin/DNA sample, the less substrate DNA there will be for qPCR relative to chromatin/DNA not exposed to DNase I (total uncut DNA). Less substrate DNA due to DNAse I digestion will require more PCR cycles to amplify, and this will be interpreted as indicating that the chromatin was more accessible for nuclease digestion. The investigator can manipulate the cells using various methods prior to isolating chromatin, allowing quick and efficient quantitative comparisons of chromatin accessibility changes to a specific region under different experimental conditions. The main limitations of our approach are knowing the genomic DNA sequences of the region under investigation, for primer design, and access to a real-time PCR machine.<br/><br/>In practice, our group has used this DNase I chromatin accessibility technique to assess changes at the human Lamin B2 origin of DNA replication in multiple studies. We demonstrated that chromatin at this origin was less accessible to DNase I (more condensed) in S phase (Wong <em>et al.</em>, 2010) or in the absence of endogenous Myc protein specifically in late-G<sub>1</sub> (Nepon-Sixt <em>et al.</em>, 2019), since the qPCR signal from the Lamin B2 origin increased (required fewer cycles) under both of these conditions. While this assay has been utilized to determine chromatin accessibility at a specific origin of mammalian DNA replication under various conditions, chromatin accessibility at other genomic sites (<em>e.g.</em>, promoters) can be evaluated by using suitable primers during qPCR designed against other sites of interest.<br/></p>
        </body>

<p class="pptt" id="biaoti30544">## Materials and Reagents</p><ol class="npwtwok"><li>Tissue culture plates (Sarstedt, catalog number: 83.3902)</li><li>Cell lifter (Fisher, catalog number: 8100240)</li><li>Microtubes (Axygen, catalog number: MCT-150-C)</li><li>Multiplate PCR Plates, 96-well, clear (Bio-Rad, catalog number: MLL9601)</li><li>Microseal âBâ seal Seals (Bio-Rad, catalog number: MSB1001)</li><li>Human cell line (<em>e.g.</em>, HaCaTâimmortalized human keratinocytes)</li><li>Appropriate tissue culture medium (<em>e.g.</em>, Dulbeccoâs modification of Eagleâs medium + 10% fetal bovine serum)</li><li>Aprotinin (Fisher, catalog number: 61-637-010MG)</li><li>Leupeptin (Fisher, catalog number: 50550413)</li><li>PMSF (RPI Corp., catalog number: P20270-5.0)</li><li>DNase I Enzyme (Promega, catalog number: M610A)</li><li>EDTA (Fisher, catalog number: BP120-500)</li><li>EGTA (Calbiochem, catalog number: 4100-50GM)</li><li>SDS (Sigma-Aldrich, catalog number: L5750-5KG)</li><li>NaCl (Fisher, catalog number: BP358-10)</li><li>RNase A (Sigma-Aldrich, catalog number: R4875-100MG)</li><li>Proteinase K (Fisher, catalog number: BP1700-100)</li><li>Sterile deionized H<sub>2</sub>O<br/></li><li>Chloroform (Fisher, catalog number: C298-500)</li><li>EtOH (Pharmco-Aaper, catalog number: 111ACS200)</li><li>5x Colorless GoTaq Reaction Buffer (Promega, catalog number: M792A)</li><li>10 mM dNTPs (Promega, catalog number: U151B)</li><li>GoTaq DNA Polymerase (Promega, catalog number: M300B)</li><li>Forward Primer (see Procedure)</li><li>Reverse Primer (see Procedure)</li><li>FAM-labeled probe (see Procedure)</li><li>KCl (Fisher, catalog number: P217-500)</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Fisher, catalog number: S374-500)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Fisher, catalog number: P285-500)</li><li>NaOH (Fisher, catalog number: BP359-500)</li><li>MgCl<sub>2</sub> (Sigma-Aldrich, catalog number: M0250-1KG)</li><li>CaCl<sub>2</sub> (Fisher, catalog number: BP510-500)</li><li>Sucrose (Fisher, catalog number: BP220-1)</li><li>Triton X-100 (Fisher, catalog number: BP151-100)</li><li>Phenol crystals (Fisher, catalog number: A92-100)</li><li>8-hydroxyquinoline (Fisher, catalog number: AC121900250)</li><li>Tris Base (Fisher, catalog number: BP152-5)</li><li>1x PBS Solution, kept in 4 Â°C fridge (called ice cold) (see Recipes)</li><li>Lysis Buffer (see Recipes)</li><li>DNase Buffer (see Recipes)</li><li>Tris-saturated Phenol (see Recipes)</li><li>TE Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30545">### Equipment</p><ol class="npwtwok"><li>Pipettes (VWR) </li><li>Centrifuge (Eppendorf mini-centrifuge, model: 5424, catalog number: 022620401)<br/><em>Note: All ârpmâ listed below also include RCF in g, for those who need to convert for use in a different model of centrifuge.</em> </li><li>Isotemp Water Bath (Fisher, catalog number: 2223)</li><li>Bioruptor 300 Plus Sonication Device w/ Water cooler system (Diagenode, catalog numbers: B01020001 and B02010003)</li><li>CFX96 Real-Time PCR Detection System including C1000 Thermal Cycler Chassis (Bio-Rad, catalog number: 185-5096)</li><li>Autoclave (Steris, model: SG-120 Scientific Gravity Sterilizer)<br/></li></ol><p class="pptt" id="biaoti30546">### Software</p><ol class="npwtwok"><li>MS Excel for Mac 2011 Version 14.6.0</li><li>CFX Manager Version 3.1 (Bio-Rad, catalog number: 184-5000)<br/></li></ol><p class="pptt" id="biaoti30547">### Procedure</p><ol class="npwone"><li>Genomic DNA collection <ol class="npwtwo"><li>Add protease inhibitors fresh to Lysis Buffer and DNase Buffer to final concentrations: 5 Âµg/ml Aprotinin, 5 Âµg/ml Leupeptin, and 1 mM PMSF. Store buffers on ice.</li><li>Collect human cell line from 1 x 10 cm plate (per experimental condition; ~5 x 10<sup>6</sup> cells/plate):<br/><ol class="npwthree"><li>Aspirate cell culture medium.</li><li>Wash cells with 4 ml of ice cold 1x PBS solution and aspirate. Repeat once.</li><li>Add 500 Âµl ice cold 1x PBS to the plate.</li><li>Scrape cells with cell lifter and collect cell suspension into a microtube and keep on ice. </li></ol></li><li>Centrifuge cells for 5 s at 10,000 rpm (~9,400 x g) to pellet, then carefully remove and discard the 1x PBS supernatant with a pipette. Should be performed in a cold room.</li><li>Resuspend cell pellet in 300 Âµl Lysis Buffer with protease inhibitors, and incubate on ice for 5 min. This step creates the chromatin/DNA to be analyzed below.</li><li>Centrifuge lysate at 4,000 rpm (~1,500 x g) for 5 min at 4 Â°C.</li><li>Carefully remove and discard supernatant with a pipette, and then resuspend DNA pellet in 300 Âµl DNase Buffer with protease inhibitors.</li><li>Centrifuge lysate at 4,000 rpm (~1,500 x g) for 5 min at 4 Â°C.</li><li>Resuspend DNA pellet in 300 Âµl DNase Buffer with protease inhibitors. No need to quantify the DNA. </li><li>Aliquot 100 Âµl of the lysate to each of two separate microtubes (one for âDNase Iâ, one for âNo DNase Iâ). Note here that equal amounts of chromatin/DNA for each condition are being assessed, and normalization to uncut input DNA (<em>i.e.</em>, No DNase I) will be used in the Data Analysis below. Be accurate in your pipetting. The DNA is not viscous here because these are chromatin samples, where DNA is still on nucleosomes, etc. The Lysis Buffer does not disrupt the nuclear structure (on purpose) so that chromatin can be interrogated by the investigator using DNAse I in Procedure B of the protocol.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>DNase I reaction and substrate preparation (per experimental condition)<ol class="npwtwo"><li>Add 3 units (3 Âµl) DNase I Enzyme to one of the microtubes only (âDNase Iâ), mix by tapping, and incubate both microtubes at room temperature for 7-8 min. <br/><em>Note: Incubation time is determined by trial and error (empirically) using distinct 10 cm plates of cells to test specific incubation durations for DNAse I exposure. Too short can limit nuclease effectiveness, and too long can tip activity beyond a linear range of detectable differences between experimental conditions. Chromatin will vary greatly for different loci, and DNase I will have different effectiveness at different sites, which makes it very difficult to pre-determine the DNase I duration for chromatin analysis with an optimization step on naked total input DNA. As an example, for the results obtained in Nepon-Sixt </em><em>et al.</em><em> (2019) (graphs in Figure 8c&amp;d) we determined by empirical testing that 7 min of DNase I produced an acceptable partial digestion of chromatin that was not under-digested, nor exhaustive, to allow DNase I sensitivity differences to be measured for the Lamin B2 locus under the experimental conditions that were assessed. Your empirical testing would follow this same protocol to perform initial determinations of DNase I time durations needed for your particular chromatin site under investigation.</em> </li><li>Add 4.25 Âµl of 0.25 M EDTA pH 8.0 and 2.1 Âµl of 100 mM EGTA to each microtube (10 mM and 2 mM final concentrations, respectively), mix by tapping, and incubate in a 65 Â°C water bath for 10 min to stop reaction.</li><li>Add 5.75 Âµl of 10% SDS (0.5% final concentration) to each tube, and mix.</li><li>Lightly sonicate with Bioruptor 300 for an 8 min duration, 30 s on/30 s off.</li><li>Add 20 Âµl of 2.5 M NaCl (~500 mM final concentration) to each microtube.</li><li>Add 1 Âµl of 10 mg/ml RNase A (50 Âµg/ml final concentration) to each microtube, mix by pipetting, and incubate in a 37 Â°C water bath for 30 min.</li><li>Add 3 Âµl of 10 mg/ml Proteinase K (250 Âµg/ml final concentration) to each microtube, mix by pipetting, and incubate in a 42 Â°C water bath overnight.</li><li>Quick spin microtubes in centrifuge to remove condensation from the lid.</li><li>Add ~350 Âµl of sterile deionized H<sub>2</sub>O (dH<sub>2</sub>O) to each tube, and purify DNA with phenol/chloroform extractions:<br/><ol class="npwthree"><li>Add 250 Âµl Tris-saturated phenol and 250 Âµl chloroform to each microtube, and invert 8-10 times to mix well.</li><li>Centrifuge tubes at 12,000 rpm (~13,500 x g) for 5 min.</li><li>Move aqueous layer (top layer) to new microtubes, respectively.</li><li>Add 500 Âµl chloroform to each tube.</li><li>Centrifuge tubes at 12,000 rpm (~13,500 x g) for 5 min.</li><li>Move aqueous layer (top layer) to new microtubes, respectively.</li><li>Fill tubes with ice cold 100% EtOH, shake, and incubate at -20 Â°C for 1 h.</li></ol></li><li>Centrifuge tubes at 12,000 rpm (~13,500 <em>x g</em>) at 4 Â°C for 25 min to pellet DNA.</li><li>Discard supernatant and dry DNA pellets by inverting microtubes in a rack and incubating at room temperature for 10 min.</li><li>Resuspend each DNA pellet in 50 Âµl TE Buffer to make DNA substrates. Again, no need to quantify DNA, but care must be taken in pipetting accuracy.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>qPCR analysis of DNA substrates<ol class="npwtwo"><li>Create master mix for qPCR analysis containing the following (enough for quadruplicate 10 Âµl reactions per experimental condition, each with âDNase Iâ and âNo DNase Iâ respectively):<br/><ol class="npwthree"><li>Sterile deionized H<sub>2</sub>O: 35 Âµl</li><li>5x Colorless GoTaq Reaction Buffer:10 Âµl</li><li>10 mM dNTPs:1 Âµl</li><li>GoTaq Polymerase: 1 Âµl</li><li>Forward Primer (10 ÂµM):1 Âµl (<em>e.g.</em>, sequence used for Lamin B2 ori accessibility analysis: 5â²-GCTACACTAGCCAGTGACCTT-3â²)</li><li>Reverse Primer (10 ÂµM): 1 Âµl (<em>e.g.</em>, sequence for used for Lamin B2 ori accessibility analysis: 5â²-GTTCTGCCTCTGAGTTTATTCCTG-3â²)</li><li>FAM-labeled Probe (10 ÂµM): 0.5 Âµl (<em>e.g.</em>, sequence used for Lamin B2 ori accessibility analysis: 5â²-/56-FAM/-TTAGACATCCGCTTCATTAGGGCAGAGG-/3BHQ_1/-3â²)</li></ol></li><li>Aliquot 49.5 Âµl master mixes into two microtubes each. </li><li>Add 0.5 Âµl âDNase Iâ DNA substrate to one tube, and 0.5 Âµl âNo DNase Iâ DNA substrate to the other, for each experimental condition, and mix well.</li><li>Aliquot 10 Âµl of each reaction mix into 4 wells of a 96-well multiplate. <br/><em>Note: Try to avoid using the outer wells on the edge of the 96-well plate, if possible. In our experience use of the outer regions can lead to non-reproducible variability in results, likely due to temperature issues.</em> </li><li>Cover the 96-well plate with seal (Microseal âBâ Seals) and press to secure. </li><li>Put the 96-well plate in CFX96 Real-Time PCR Detection System and run the following program:<br/><ol class="npwthree"><li>95 Â°C for 10 min</li><li>95 Â°C for 15 s </li><li>60 Â°C for 1 min</li><li>Repeat steps b and c 40 x </li></ol></li><li>Analyze qPCR results with CFX Manager to obtain quantitation cycle (Ct) values using FAM.</li></ol></li></ol><p class="pptt" id="biaoti30548">### Data analysis</p><p class="ppzz">For each experimental condition, analyze results according to the formula: 100/2<sup>Ct(DNase I)âCt(no DNase I</sup>), which normalizes the data to input DNA (âno DNase Iâ). This is performed separately for each experimental condition that the investigator chooses to study. The changes in the ability of DNase I to digest DNA contained in chromatin are then plotted on graphs to determine if a tested experimental condition rendered a locus more sensitive to DNase I digestion (and is therefore interpreted as being more open in chromatin accessibility). Examples of how the final data will appear in a graph can be seen Figure 8c&amp;d in Nepon-Sixt <em>et al.</em>, (2019). Using our experiment from Figure 8c in ref. 1 as an example, we next describe how to mathematically analyze the data obtained using our tested experimental conditions, allowing graphical plotting of the final results.<br/>  As an example for comparing a single locus for two experimental conditions [<em>e.g.</em>, with normal <em>Myc</em> expression (si-Control; <em>e.g.</em>, Luciferase control, si-Luc), and with <em>Myc</em> si-RNA suppression; see ref. 1, Figure 8c, left pair of bars]: If one obtains Ct values of 24 for âno DNase Iâ and 26 for the âDNase Iâ sample, then the calculation would be 100/2<sup>26-24</sup> = 25 [note, the whole numbers chosen here are not the exact same as what we obtained for Figure 8c in ref. 1, but are used here for simplicity in illustrating how to perform the mathematics needed]. This means that using the substrate with DNase I treatment gave 25% of the signal relative to the sample without DNase I under the one experimental condition (<em>e.g.</em>, with normal <em>Myc</em>; si-Luc/Control). This is also referred to as â25% untreated input chromatin/DNAâ as shown on the Y-axis in Fig 8c in ref. 1. This is then performed on the samples from the other experimental condition (<em>e.g.</em>, with <em>Myc</em> si-RNA exposure and loss of <em>Myc</em> expression; si-<em>Myc</em> in Figure 8c from ref. 1). If for example the latter gives Ct values of 24 for âno DNase Iâ and 25 for the âDNase Iâ sample, then the calculation would be 100/2<sup>25-24</sup> = 50. This means that using the substrate with DNase I treatment gave 50% of the signal relative to the sample without DNase I for this experimental condition (<em>e.g.</em>, loss of <em>Myc</em>, si-Myc). The interpretation would be that loss of <em>Myc</em> expression by si-RNA renders the locus tested to be less sensitive to DNase I digestion (relative to si-Luc control), and therefore more compact from a chromatin standpoint (less accessible to the added DNase I enzyme). Thus, loss of <em>Myc</em> expression caused the locus to become more condensed in its chromatin state. Such data are then plotted on graphs after analyzing technical replicates (separate samples analyzed from same day and plating of cells). Further validity to the interpretations will need to be achieved with biological replicates giving the same results (samples analyzed from different plating of cells collected on a different day).</p><p class="pptt" id="biaoti30549">### Notes</p><ol class="npwtwok"><li>If an investigator wants to normalize their data for a locus in question to an internal control/comparison, such as a gene or promoter that is known to have no chromatin changes between the conditions tested (will need to be verified first in separate testing), then separate analyses would need to be performed on that locus (internal control/comparison) and the locus in question. The changes between the results of the locus in question will then be normalized to any changes in the presumptively non-changing control/comparison locus. However, it is important to understand that many loci will change when one subjects cells to various conditions (such as global effects due to drugs, or si-RNA loss of important transcription factors), so it is perhaps better to use biological and technical replicate testing to achieve statistically valid interpretations.</li><li>This approach is not designed to compare the same locus in two different cell types, but instead has been validated for comparing chromatin changes occurring to one locus in the same cell type that is exposed to different experimental conditions (see Nepon-Sixt <em>et al.</em> [2019] for examples). <br/></li></ol><p class="pptt" id="biaoti30550">### Recipes</p><ol class="npwtwok"><li>1x PBS Solution; kept in 4 Â°C fridge (called ice cold)<br/>Make 10x PBS solution (below) and dilute 1:10 to 1x<br/>80 g of NaCl (1.4 M)<br/>2 g of KCl (26.8 mM)<br/>14.4 g of Na<sub>2</sub>HPO<sub>4</sub> (101.4 mM)<br/>2.4 g of KH<sub>2</sub>PO<sub>4</sub> (17.6 mM)<br/>Add sterile deionized H<sub>2</sub>O (dH<sub>2</sub>O, used below) to 800 ml<br/>Adjust pH to 7.4 by 10 N NaOH<br/>Add dH<sub>2</sub>O to 1 L<br/>Autoclave and cool to room temperature</li><li>Lysis Buffer<br/>10 mM Tris pH 7.5<br/>5 mM MgCl<sub>2</sub><br/>1 mM CaCl<sub>2</sub><br/>10 mM KCl<br/>300 mM Sucrose<br/>0.1% Triton X-100</li><li>DNase Buffer <br/>10 mM Tris pH 7.5<br/>5 mM MgCl<sub>2</sub><br/>1 mM CaCl<sub>2</sub><br/>10 mM KCl<br/>300 mM Sucrose</li><li>Tris-Saturated Phenol (can use commercially available Tris-Phenol if desired, but our method here allows investigator control over phenol preps and saturation levels)<br/><ol class="npwthree"><li>Dissolve 100 g phenol crystals with 100 ml dH<sub>2</sub>O in 37 Â°C water bath</li><li>Add 0.1 g 8-hydroxyquinoline (0.1% w/vol) and mix by shaking</li><li>Saturate aliquots with 1:1 volume of 0.5 M Tris pH 8.0 by shaking and allow separation. Discard Tris solution as phenol waste (Repeat 1 x)</li><li>Saturate with 1:1 volume of 0.2 M Tris pH 8.0 by shaking and allow separation. Discard most Tris solution as phenol waste, leaving a small layer</li></ol></li><li>TE Buffer pH 7.5<br/>10 mM Tris pH 7.5<br/>1 mM EDTA pH 8.0<br/></li></ol><p class="pptt" id="biaoti30551">### Acknowledgments</p><p class="ppzz">This protocol was adapted from Nepon-Sixt <em>et al.</em> (2019). The Alexandrow lab was supported by NIH grants R01-CA130865 and R21-CA155393.</p><p class="pptt" id="biaoti30552">### Competing interests</p><p class="ppzz">All authors acknowledge that they have no competing financial or non-financial interests in the development of this manuscript.</p><p class="pptt" id="biaoti30553">### References</p><ol class="npwtwok"><li>Nepon-Sixt, B. S., Bryant, V. L. and Alexandrow, M. G. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30911685" target="_blank"><em>Myc</em>-driven chromatin accessibility regulates Cdc45 assembly into CMG helicases.</a> <em>Commun Biol</em> 2: 110.</li><li>Pemov, A., Bavykin, S. and Hamlin, J. L. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9843962" target="_blank">Attachment to the nuclear matrix mediates specific alterations in chromatin structure.</a> <em>Proc Natl Acad Sci U S A </em>95(25): 14757-14762.</li><li>Wong, P. G., Glozak, M. A., Cao, T. V., Vaziri, C., Seto, E. and Alexandrow, M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20980834" target="_blank">Chromatin unfolding by Cdt1 regulates MCM loading via opposing functions of HBO1 and HDAC11-geminin.</a> <em>Cell Cycle</em> 9(21): 4351-4363.<br/></li></ol>