<body>
            <h1>Mitochondrial Respiratory Measurements in Patient-derived Fibroblasts</h1>
            <p><strong>Authors:</strong> Prashant Mishra, Ting Zhang, Ming Guo, David Chan</p>
            <h2>Abstract</h2>
            <p>Mitochondrial dysfunction is associated with a number of human diseases. As an example, we recently established <em>in vivo Drosophila</em> models of IBMPFD (Inclusion body myopathy, Paget disease, and frontotemporal dementia), and uncovered that human disease mutations of the <em>p97/VCP</em> (Valosin Containing Protein) gene behave as hyperactive alleles associated with mitochondrial defects. Pharmacologic inhibition of <em>VCP</em> strongly suppressed disease and mitochondrial pathology in these animal models. In this protocol, we describe a method to evaluate mitochondrial respiratory function in IBMPFD patient-derived fibroblasts, as well as investigate the role of pharmacologic treatments. These experiments complement work done in animal models by investigating mitochondrial biology and the pharmacologic response in a human cell-based model of the disease. In principle, this technique can be used to investigate mitochondrial respiratory function for any disease in which patient-derived fibroblasts are available. </p>
            <p><strong>Keywords:</strong> Patient-derived fibroblasts, Mitochondrial respiration, Seahorse XF assay, Inclusion body myopathy, Paget disease and frontotemporal dementia (IBMPFD), <em>VCP/p97</em></p>
            <h2>Background</h2><p>The mitochondrial electron transport chain (ETC) constitutes a series of protein complexes (complex I, II, III, IV and V) mediating the production of ATP coupled to the consumption of oxygen (O<sub>2</sub>). Briefly, the oxidation of nutrients (glucose, fatty acids, amino acids, <em>etc.</em>) by the organelle provides a pool of electrons which are sequentially transferred by ETC complexes between redox-active molecules located at the mitochondrial inner membrane, including ubiquinone, cytochrome c and O<sub>2</sub>. Early electron acceptors (<em>e.g.</em>, ubiquinone, cytochrome c) are first reduced and then recycled to their oxidized forms, while O<sub>2</sub>, the terminal electron acceptor, is converted to H<sub>2</sub>O and not re-generated. For this reason, the net consumption rate of O<sub>2</sub> (the Oxygen Consumption Rate or âOCRâ) at complex IV serves as a proxy for mitochondrial ETC function in cells.<br/><br/>Measurement of the OCR in cultured cells has traditionally relied on detecting declines in the O<sub>2</sub> levels in media surrounding cells. The advent of the Seahorse Extracellular Flux (Seahorse XF) analyzer has improved the throughput and sensitivity of these measurements, by allowing rapid and simultaneous measurements of the OCR from a small number of cells (typically, 5,000-100,000 cells per well) in an adherent 24-well or 96-well format. In brief, the XF analyzer forms a transient microchamber consisting of a small volume (~2 Î¼L) of media directly above the cell monolayer, and utilizes fluorescent probes to follow [O<sub>2</sub>] over time. For a more complete description of the principle of XF assays, see <a href="https://www.agilent.com/en/products/cell-analysis/how-seahorse-xf-analyzers-work" target="_blank">https://www.agilent.com/en/products/cell-analysis/how-seahorse-xf-analyzers-work</a>.<br/><br/>Defects in mitochondrial ETC function directly underlie a set of genetic diseases affecting components of the ETC complexes (<em>e.g.</em>, Leighâs syndrome and others). In addition, mitochondrial ETC dysfunction can be indirectly associated with genetic diseases secondary to mutations in non-mitochondrial proteins, and potentially contribute to disease pathology in these settings. Assessment of mitochondrial ETC function in the disease setting has the potential to both implicate organelle dysfunction in the pathophysiology, as well as assess the impact of pharmacologic treatments. In this protocol, we describe the measurement of oxygen consumption rates in immortalized human fibroblasts derived from disease patients and age-matched controls. As a specific example, we consider disease-associated alleles of the <em>p97/VCP</em> gene. Missense mutations of <em>p97/VCP</em> are causative for inclusion body myopathy, Paget disease and frontotemporal dementia (IBMPFD) (Watts <em>et al.</em>, 2004), a rare autosomal dominant disease associated with muscle weakness, bone pain and neurocognitive decline. <em>VCP</em> mutations are also associated with other neuropathic diseases [sporadic amyotrophic lateral sclerosis (ALS), hereditary spastic paraplegia and Charcot-Marie-Tooth 2 neuropathy] (Abramzon <em>et al.</em>, 2012; de Bot <em>et al.</em>, 2012; Gonzalez <em>et al.</em>, 2014). The <em>VCP</em> gene encodes a highly conserved and abundant member of the AAA+ (ATPases associated with diverse cellular activities) protein family, which participates in multiple cellular processes, and potentially impacts mitochondrial biology (Meyer <em>et al.</em>, 2012). Using <em>in vivo</em> IBMPFD models in <em>Drosophila</em>, we found that <em>VCP</em> disease mutants behave as hyperactive alleles, and cause significant mitochondrial defects including fragmentation and broken cristae. Feeding of <em>VCP</em> inhibitors relieve the pathology in these models (Zhang <em>et al.</em>, 2017). To substantiate our findings in <em>Drosophila</em>, we investigated mitochondrial ETC function in IBMPFD cell-based models derived from patient fibroblasts. Disease-associated fibroblasts exhibited reduced oxygen consumption rates (as compared with age-matched controls), and were responsive to multi-day treatment regimens with <em>VCP</em> inhibitors (Zhang <em>et al.</em>, 2017). Thus, the investigation of mitochondrial pathology present in animal disease models can be complemented by measurements of mitochondrial ETC function in a human cell-based model of the disease.</p>
        </body>

<p class="pptt" id="biaoti37766">## Materials and Reagents</p><ol class="npwtwok"><li>Cell lines: human primary IBMPFD patient fibroblasts harboring the <em>VCP<sup>R155H/+</sup></em> mutation (GM21752) and age-matched controls (GM00024) were purchased from the Coriell Institute for Medical Research (<a href="https://catalog.coriell.org/" target="_blank">https://catalog.coriell.org</a>). Primary fibroblasts were immortalized by infection with retrovirus expressing hTERT. Infected cells were selected in puromycin (1.5 Âµg/ml) for 1 week and maintained in 1 Âµg/ml puromycin</li><li>pBABE-puro-hTERT (Addgene plasmid, catalog number: 1771)</li><li>Liquid DME medium (Sigma-Aldrich, catalog number: D6429)</li><li>Fetal Bovine Serum (Sigma-Aldrich, catalog number: F0926)</li><li>Penicillin/Streptomycin (Sigma-Aldrich, catalog number: P0781)</li><li>PBS (Sigma-Aldrich, catalog number: D8537)</li><li>Trypsin-EDTA (Sigma Aldrich, catalog number: T4049)</li><li>NMS-873 (Selleckchem, catalog number: S7285)</li><li>ML240 (Sigma-Aldrich 1346527-98-7)</li><li>DMSO (Sigma-Aldrich, catalog number: D8418)</li><li>Seahorse XFe96 FluxPak (Agilent Technologies, catalog number: 102416-100), including sensor cartridge, calibration utility plate, 96-well culture plate, loading guides, calibrant solution</li><li>Bicarbonate-free DMEM (Sigma-Aldrich, catalog number: D5030)</li><li>L-glutamine (Sigma-Aldrich, catalog number: G7513)</li><li>Sodium pyruvate (Sigma-Aldrich, catalog number: S8636)</li><li>D-glucose (Sigma-Aldrich, catalog number: G8270)</li><li>Oligomycin (Sigma-Aldrich, catalog number: O4876)</li><li>CCCP (Sigma-Aldrich, catalog number: C2759)</li><li>Antimycin A (Sigma-Aldrich, catalog number: A8674)</li><li>Trichloroacetic Acid (Sigma-Aldrich, catalog number: T9159)</li><li>Acetic Acid (Sigma-Aldrich, catalog number: A6283)</li><li>Sulforhodamine B (SRB, Sigma-Aldrich, catalog number: 230162)</li><li>Tris base (Sigma-Aldrich, catalog number: T1503)</li><li>Culture media (see Recipe 1)</li><li>Seahorse Assay Media (see Recipe 2)</li><li>Stock solutions (see Recipe 3)<br/><ol class="npwthree"><li>1 M glucose</li><li>200 mM glutamine </li><li>100 mM sodium pyruvate</li><li>50 mM Oligomycin</li><li>100 mM CCCP</li><li>50 mM Antimycin A</li><li>Trichloroacetic Acid solution </li><li>Acetic Acid solution </li><li>Sulforhodamine B solution </li><li>10 mM Tris solution</li><li>1 mM NMS-873 stock solution</li><li>1 mM ML240 stock solution</li></ol></li><li>Bicarbonate-free DMEM medium (see Recipe 4)<br/></li></ol><p class="pptt" id="biaoti37767">### Equipment</p><ol class="npwtwok"><li>Pipettes (0.5-10 Î¼l, 2-20 Î¼l, 20-200 Î¼l, 100-1,000 Î¼l)</li><li>8-channel multichannel pipettes (2-20 Î¼l, 20-200 Î¼l)</li><li>Seahorse Biosciences Extracellular Flux Analyzer (Model: XF96 or XFe96)</li><li>Hemocytometer (Hausser Scientific, catalog number: VWR 15170-168)</li><li>37 Â°C, non-CO<sub>2</sub> humidified incubator (<em>e.g.</em>, Seahorse PrepStation or equivalent)</li><li>Plate reader for absorbance measurements (<em>e.g.</em>, Tecan Infinite 200 or equivalent)<br/></li></ol><p class="pptt" id="biaoti37768">### Software</p><ol class="npwtwok"><li>WAVE (Agilent Technologies Seahorse Wave Desktop software, version 2.6 or equivalent)</li></ol><p class="pptt" id="biaoti37769">### Procedure</p><p class="ppzz">See Figure 1 for an overview of the complete protocol. The overall procedure involves plating human fibroblasts into a Seahorse XF96 cell culture plate at low density, followed by 4-6 days of pharmacological treatment. After treatment, a Seahorse XF assay is first run to collect data on oxygen consumption rates. After completion, cell quantities are normalized on a per well basis to account for differences in cell viability and growth.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202190504_4463.jpg" width="163"/><br/><strong>Figure 1. Timing and overview of protocol.</strong> On Day 1, cells are plated and allowed to attach overnight (Step 1). Drug treatment occurs for 4-6 days, starting on Day 2 (Step 2). A 6-day treatment is depicted in this figure. On the last day of treatment (Day 7), the Seahorse sensor cartridge is hydrated overnight (Step 3). The Seahorse assay is performed on Day 8 (Steps 4-6). Cells are fixed after assay completion on Day 8, and normalization is performed on Days 8-10 (Step 7). Step numbers refer to the main text.</p><br/><ol class="npwtwo"><li>Plate human fibroblasts into a Seahorse XF96 cell culture plate (Step 1)<br/><em>Note: Perform all steps using sterile technique.</em> <ol class="npwthree"><li>Pre-warm culture media, PBS, and Trypsin-EDTA solution to 37 Â°C. </li><li>Wash adherent, cultured fibroblasts with PBS. Add trypsin-EDTA solution. Wait for cells to detach (3-5 min at 37 Â°C). Add an equal volume of culture media to neutralize trypsin. Resuspend cells in culture media/trypsin-EDTA mix, and transfer to a 15 ml conical tube.</li><li>(Optional Step): Wash cells by pelleting cell suspension at 1,000 <em>x g</em> (5 min, 20 Â°C), removing supernatant, and resuspending the cell pellet in 2-3 ml of culture media.</li><li>Count cells on a hemocytometer in triplicate. Resuspend cells at a final concentration of 5,000 cells per 80 Âµl of culture media.</li><li>Plate 80 Âµl of cell suspension per well into a Seahorse XFe96 cell culture plate. Leave the 4 corners without cells (<em>i.e.</em>, wells A1, A12, H1, H12 are filled with 80 Î¼l of culture media with no cells).</li><li>To promote even plating, let the culture plate sit for 30-60 min in the tissue culture hood without being disturbed.</li><li>Transfer the plate to a 37 Â°C, 5% CO<sub>2</sub> incubator overnight.</li></ol></li><li>Drug treatment (Step 2)<br/><em>Note: Perform all steps using sterile technique.</em> <ol class="npwthree"><li>Prepare fresh culture media containing drug (<em>e.g.</em>, 1 nM NMS-873, 50 nM ML240) or vehicle (equivalent amount of DMSO). Prepare enough volume for 80 Âµl/well.</li><li>With a vacuum, remove media from each well, and replace with fresh culture media containing drug or vehicle.</li><li>Return plate to 37 Â°C, 5% CO<sub>2</sub> incubator overnight.</li><li>Repeat Steps 2a-2c every 24 h for the duration of the drug treatment (<em>e.g.</em>, 4-6 days).</li></ol></li><li>Hydrate sensor cartridge and stabilize instrument (Step 3)<br/><em>Note: Perform one day prior to respiration measurements.</em> <ol class="npwthree"><li>Open a Seahorse XFe96 FluxPak. Remove the sensor cartridge (green), from the 96-well utility plate. <br/><em>Note: Make sure not to scratch the printed fluorophores on the sensor cartridge by placing the cartridge upside down on a flat surface.</em> </li><li>Using a 8-channel multichannel pipette, add 200 Âµl calibrant solution to each well of the utility plate.</li><li>Carefully return sensor cartridge to the utility plate. Make sure not to scratch the fluorophores.</li><li>Incubate Sensor cartridge and utility plate overnight in a 37 Â°C, non-CO<sub>2</sub> incubator.</li><li>Turn on Seahorse XF96 instrument and start up WAVE software. Ensure computer is connected to the instrument, and the temperature is set to 37 Â°C. Leave instrument on overnight to maintain temperature stability.</li></ol></li><li>Wash cells into Seahorse assay media (Step 4)<br/><em>Note: Perform approximately 60 min prior to start of assay.</em> <ol class="npwthree"><li>Using a vacuum, remove media from each well of the Seahorse XFe96 culture plate. Using an 8-channel multi-channel pipette, add back 150 Âµl of warm Seahorse assay media. Repeat three times.</li><li>On the last wash, remove as much media as possible, and add 150 Âµl of assay media.</li><li>Place the plate in a 37 Â°C, non-CO<sub>2</sub> incubator for approximately 60 min prior to measurements.</li></ol></li><li>Load cartridge with desired compounds (oligomycin, CCCP, antimycin A) (Step 5)<br/><em>Note: Final drug concentrations will be 5 ÂµM oligomycin, 10 ÂµM CCCP, and 1 ÂµM antimycin A after each sequential injection.</em> <ol class="npwthree"><li>Prepare the following solutions (with drugs diluted from frozen stocks):<br/>2 ml of Seahorse assay media containing 80 ÂµM oligomycin<br/>2 ml of Seahorse assay media containing 170 ÂµM CCCP<br/>2 ml of Seahorse assay media containing 18 ÂµM antimycin A</li><li>Using a 2-20 Âµl multichannel pipette, load 10 Âµl of oligomycin solution into port A of each well. </li><li>Using a 2-20 Âµl multichannel pipette, load 10 Âµl of CCCP solution into port B of each well.</li><li>Using a 2-20 Âµl multichannel pipette, load 10 Âµl of antimycin A solution into port C of each well.</li><li>Tap down sensor cartridge gently to ensure solution reaches bottom of each port.</li></ol></li><li>Run Assay (Step 6)<ol class="npwthree"><li>Within the WAVE software, set up the following protocol:<br/>Calibration<br/>Equilibration<br/>Basal measurements: 4 cycles of Mix (3 min) and Measure (3 min)<br/>Injection (port A), followed by 4 cycles of Mix (3 min) and Measure (3 min)<br/>Injection (port B), followed by 4 cycles of Mix (3 min) and Measure (3 min)<br/>Injection (port C), followed by 4 cycles of Mix (3 min) and Measure (3 min)</li><li>Start assay and insert sensor cartridge (loaded with drugs) into instrument on top of utility plate containing calibrant solution. <strong>Ensure no lids or drug loading guides are present</strong>.</li><li>Allow instrument to calibrate (15-20 min).</li><li>After the instrument has completed calibration, remove the utility plate and insert Seahorse XFe96 cell plate. <strong>Ensure no lids are present</strong>.</li></ol></li><li>Normalization (Step 7)<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>Normalization of Seahorse data can be performed using multiple methods, including cell counting, cell staining, and protein measurements. Here, we describe a Sulforhodamine B staining protocol for normalization (Vichai </em><em>et al.</em><em>, 2006), based on the ability of sulforhodamine B to bind protein-components present in fixed cells.</em> </li><li><em>Use a multichannel pipette for each addition (#1-5) of the normalization process.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwtwo"><li>After assay is complete, fix cells by adding 50% TCA solution directly to each well, to a final concentration of 10%. Incubate plate at 4 Â°C for 1-24 h.</li><li>Wash cell culture plate 5 times with distilled water to completely remove all TCA solution. Dry plate completely (overnight at room temperature).</li><li>Stain cells by adding 25 Âµl per well of SRB (Sulforhodamine B) solution. Incubate at room temperature for 30 min, with rocking.</li><li>Wash plate 5 times with 1% acetic acid to remove any residual SRB. Dry plate completely (overnight at room temperature in the dark).</li><li>Add 100 Âµl of 10 mM Tris solution per well to solubilize stain. Incubate plate at room temperature for 30 min with rocking.</li><li>Measure absorbance at 510 nm, using a microplate reader. Signals for each well should be subtracted from absorbance signals for the background wells (A1, A12, H1, H12). Background-corrected absorbance values can be input into WAVE software to normalize rate measurements.</li><li>Calculation of basal oxygen consumption rates (Basal OCR; Figure 2) is as follows:<br/><br/><img alt="" class="layerphoto" height="52" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202190948_2028.jpg" width="450"/><br/><br/>The residual OCR (post injection of antimycin A) is considered the background or non-mitochondrial OCR.</li><li>Calculation of maximal oxygen consumption rates (Max OCR; Figure 2) is as follows:<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202191034_6971.jpg" width="600"/><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202191116_8972.jpg" width="207"/><br/><strong>Figure 2. Schematic of sample oxygen consumption data.</strong> Normalized Oxygen consumption rates (OCRs) are plotted versus time over the course of a typical Seahorse experiment (Step 6). Drugs (red) are injected at the indicated times, including oligomycin, CCCP, and antimycin A. Basal OCR and Maximal OCR are calculated as depicted (Step 7). Basal oxygen consumption rate (Basal OCR) is the average OCR of the first 4 measurements (prior to injection of oligomycin) minus the average OCR of the last 4 measurements (post injection of antimycin A). Maximal OCR is the max OCR of the four measurements post injection of CCCP minus the average OCR of the last 4 measurements (post injection of antimycin A). </li></ol></li></ol><p class="pptt" id="biaoti37770">### Recipes</p><ol class="npwtwok"><li>Culture media (DMEM + 10% FBS)<br/>445 ml Liquid DMEM<br/>50 ml Fetal Bovine Serum<br/>5 ml penicillin/streptomycin<br/>Filter/sterilize through a 0.2 Âµm filter into an autoclaved glass bottle<br/>Store at 4 Â°C<br/>Warm to 37 Â°C before use</li><li>Seahorse Assay Media (prepare fresh on day of assay)<br/>8 ml of FBS (final concentration: 10%)<br/>0.8 ml of 1 M glucose (final concentration: 10 mM)<br/>0.8 ml of 200 mM glutamine (final concentration: 2 mM)<br/>0.8 ml of 100 mM pyruvate (final concentration: 1 mM)<br/>0.8 ml of penicillin/streptomycin (final concentration: 1%) <br/>68.8 ml of bicarbonate-free DMEM<br/>Warm solution to 37 Â°C, and adjust pH to 7.4<br/>Filter/sterilize through a 0.2 Âµm filter<br/>Store at 37 Â°C until use</li><li>Stock solutions<br/><ol class="npwthree"><li>1 M glucose<br style="white-space:normal;"/>Dissolve in ddH<sub>2</sub>O<br style="white-space:normal;"/>Aliquot and store at -20 Â°C</li><li>200 mM glutamine <br style="white-space:normal;"/>Dissolve in ddH<sub>2</sub>O. Filter/sterilize through a 0.2 Âµm filter</li><li>100 mM sodium pyruvate<br style="white-space:normal;"/>Dissolve in ddH<sub>2</sub>O. Filter/sterilize through a 0.2 Âµm filter</li><li>50 mM Oligomycin<br style="white-space:normal;"/>Dissolve in DMSO, aliquot and store at -20 Â°C</li><li>100 mM CCCP<br style="white-space:normal;"/>Dissolve in DMSO, aliquot and store at -20 Â°C</li><li>50 mM Antimycin A<br style="white-space:normal;"/>Dissolve in DMSO, aliquot and store at -20 Â°C</li><li>Trichloroacetic Acid solution <br style="white-space:normal;"/>50% (w/v)in ddH<sub>2</sub>O<br style="white-space:normal;"/><em>Note: Prepare fresh on the day of use, store at 4 Â°C.</em> </li><li>Acetic Acid solution <br style="white-space:normal;"/>1% (v/v) in ddH<sub>2</sub>O<br style="white-space:normal;"/>Store at room temperature in glass bottle</li><li>Sulforhodamine B solution <br style="white-space:normal;"/>0.057% (w/v) in 1% acetic acid solution<br style="white-space:normal;"/>Store at 4 Â°C, in the dark</li><li>10 mM Tris solution<br style="white-space:normal;"/>pH 10.5 in ddH<sub>2</sub>O<br style="white-space:normal;"/>Store at room temperature</li><li>1 mM NMS-873 stock solution<br style="white-space:normal;"/>Dissolve in DMSO and store at -20 Â°C</li><li>1 mM ML240 stock solution<br style="white-space:normal;"/>Dissolve in DMSO and store at -20 Â°C</li></ol></li><li>Bicarbonate-free DMEM medium<br/>Dissolve 8.3 g powder in 1 L of ddH<sub>2</sub>O, filter-sterilize through a 0.2 Âµm filter<br/>Store at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti37771">### Acknowledgments</p><p class="ppzz">We are grateful to the generous support from the National Institute of Health (National Institute on Aging), Glenn Foundation for Medical Research, the Natalie R. and Eugene S. Jones Fund in Aging and Neurodegenerative Disease Research, Kenneth Glenn Family Foundation, funds from the UCLA Laurie and Steven Gordon Commitment to Cure Parkinsonâs Disease, and Renee and Meyer Luskin Family Fund.</p><p class="pptt" id="biaoti37772">### Competing interests</p><p class="ppzz">We have no competing interests.</p><p class="pptt" id="biaoti37773">### References</p><ol class="npwtwok"><li>Abramzon, Y., Johnson, J. O., Scholz, S. W., Taylor, J. P., Brunetti, M., Calvo, A., Mandrioli, J., Benatar, M., Mora, G., Restagno, G., Chio, A. and Traynor, B. J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22572540" target="_blank">Valosin-containing protein (<em>VCP</em>) mutations in sporadic amyotrophic lateral sclerosis.</a> <em>Neurobiol Aging </em>33(9): 2231 e2231-2231 e2236.</li><li>de Bot, S. T., Schelhaas, H. J., Kamsteeg, E. J. and van de Warrenburg, B. P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22991237" target="_blank">Hereditary spastic paraplegia caused by a mutation in the <em>VCP</em> gene.</a> <em>Brain</em> 135(Pt 12): e223; author reply e224.</li><li>Gonzalez, M. A., Feely, S. M., Speziani, F., Strickland, A. V., Danzi, M., Bacon, C., Lee, Y., Chou, T. F., Blanton, S. H., Weihl, C. C., Zuchner, S. and Shy, M. E. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25125609" target="_blank">A novel mutation in <em>VCP</em> causes Charcot-Marie-Tooth Type 2 disease. </a><em>Brain</em> 137(Pt 11): 2897-2902.</li><li>Meyer, H., Bug, M. and Bremer, S. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22298039" target="_blank">Emerging functions of the <em>VCP</em>/<em>p97</em> AAA-ATPase in the ubiquitin system.</a> <em>Nat Cell Biol</em> 14(2): 117-123.</li><li>Vichai, V. and Kirtikara, K. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406391" target="_blank">Sulforhodamine B colorimetric assay for cytotoxicity screening.</a> <em>Nat Protoc </em>1(3): 1112-1116.</li><li>Watts, G. D., Wymer, J., Kovach, M. J., Mehta, S. G., Mumm, S., Darvish, D., Pestronk, A., Whyte, M. P. and Kimonis, V. E. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15034582" target="_blank">Inclusion body myopathy associated with Paget disease of bone and frontotemporal dementia is caused by mutant valosin-containing protein.</a> <em>Nat Genet </em>36(4): 377-381.</li><li>Zhang, T., Mishra, P., Hay, B. A., Chan, D. and Guo, M. (2017). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5360448/" target="_blank">Valosin-containing protein (VCP/p97) inhibitors relieve Mitofusin-dependent mitochondrial defects due to <em>VCP</em> disease mutants. </a><em>eLife</em> 6: e17834.<br/></li></ol>