<body>
            <h1>Detection of <em>in vivo</em> Protein Interactions in All Bacterial Compartments by FÃ¶rster Resonance Energy Transfer with the Superfolder mTurquoise2 ox-mNeongreen FRET Pair</h1>
            <p><strong>Authors:</strong> Nils Y. Meiresonne, Elisa Consoli, Laureen M.Y. Mertens, Tanneke den Blaauwen</p>
            <h2>Abstract</h2>
            <p>This protocol was developed to qualitatively and quantitatively detect protein-protein interactions in all compartments of <em>Escherichia coli</em> by FÃ¶rster Resonance Energy Transfer (FRET) using the Superfolder mTurquoise2 ox-mNeonGreen FRET pair (sfTq2<sup>ox</sup>-mNG). This FRET pair has more than twice the detection range for FRET interaction studies in the cytoplasm or periplasm of <em>E. coli</em> compared to other pairs to date. These protein-interaction studies can be performed <em>in vivo</em> because fluorescent proteins can be genetically encoded as fusions to proteins of interest and expressed in the cell. sfTq2<sup>ox</sup> and mNG fluorescent protein fusions are co-expressed in bacterial cells and the fluorescence emission spectra are measured. By also measuring reference spectra for the background, sfTq2<sup>ox</sup>-only and mNG-only samples, expected emission spectra can be calculated. Sensitized emission for mNG above the expected spectrum can be attributed to FRET and quantified by spectral unmixing. This bio-protocol discusses the sfTq2<sup>ox</sup>-mNG FRET pair and provides a practical guide in preparing the protein fusions, setting up and running the FRET experiments, measuring fluorescence spectra and gives the tools to analyze the collected data.</p>
            <p><strong>Keywords:</strong> Bacteria, Protein interactions, FRET, Cytoplasm, Periplasm, sfTq2ox, mNG</p>
            <h2>Background</h2><p>Determining how and which proteins interact to sustain life is at the core of molecular biology research. Many <em>in vitro</em> methods exist but may result in false positives as the interactions are taken out of their biological context. An <em>in vivo</em> method is crucial for the verification of <em>in vitro</em> obtained results. Moreover, <em>in vivo</em> methods allow for the observation of the often, dynamic processes that take place in the cell. Fluorescent proteins (FP)s are ideal for <em>in vivo</em> experiments as they can be fused to proteins of interest, making it possible to microscopically track the localization and dynamics of the fused proteins in living cells. Energy can be transferred between FPs by FÃ¶rster Resonance Energy Transfer (FRET) provided they have compatible physical properties. The emission spectrum of a donor FP must overlap with the excitation spectrum of an acceptor FP and the physical distance between the chromophores within the folded FPs should be less than 10 nm. The larger the spectral overlap and the smaller the distance between the donor and acceptor FP the more FRET can occur. The stringent distance dependence for FRET is ideal to detect direct protein-protein interactions as they also occur in the nanometer range, whereas indirect protein interactions are usually on a larger distance scale and not detectable by FRET. For this reason, it gives very few false positives but due to the physical requirements of the involved fluorophores, negative results could be false negatives. The advantage of using FPs for FRET measurements is that the chromophore is enclosed in a protein barrel and therefore much better protected from the environment than chemical chromophores. The fluorescence of the latter is extremely sensitive to environmental changes such as pH, salt, and solvents. Consequently, they often cannot be used reliably to measure FRET in a complicated environment like the living cell. A drawback of the 3-4 nm in diameter protein-barrel that encloses the chromophore in FPs is that it reduces the detectable distance between proteins to 1-6 nm. Since 5-7 nm is the typical size of proteins, it usually does not pose a problem, but can again result in false negatives. For a recent perspective on FRET as a biomolecular research tool see Alger <em>et al.</em> (2019). We have used a spectroscopy based FRET method to analyze the interaction between proteins in the cytoplasm and in the periplasm of <em>Escherichia coli</em> (Alexeeva <em>et al.</em>, 2010; Fraipont <em>et al.</em>, 2011; van der Ploeg <em>et al.</em>, 2013 and 2015; Meiresonne <em>et al.</em>, 2017 and 2019) and showed that the technique is suitable for mode of action studies of antibiotics (van der Ploeg <em>et al.</em>, 2015) and medium throughput screening (Meiresonne <em>et al.</em>, 2017 and 2019) using various FP FRET pairs. Building on our experience with bacterial FRET and fluorescent proteins in challenging compartments, we preferentially use the FRET-pair that gives the highest rates of energy transfer in the cytoplasm as well as in the periplasm.<br/><br/>This bio-protocol describes how to perform our sfTq2<sup>ox</sup>-mNG FRET assay in bacteria to determine protein-protein interactions by spectrally unmixing fluorescence signal into its individual components. Although this protocol is written for <em>E. coli</em>, the method of spectral unmixing is suitable for any organism, provided fusion proteins can be expressed in the species and fluorescence spectra can be collected. The protocol describes how to perform FRET experiments by 2 approaches; 1) Measurements on fixed cell using a fluorometer, the initial determination of protein interactions. 2) Measurements on fixed cells in the plate reader for the faster detection of already confirmed protein interactions that provide clear fluorescence signals.</p>
        </body>

<p class="pptt" id="biaoti37457">## Materials and Reagents</p><ol class="npwtwok"><li>Sterile straight neck culturing flasks (DWK Life Sciences, DURAN, catalog number: 2177124) with aluminum anodized metal blue cap (DWK Life Sciences, DURAN, catalog number: 2901324)</li><li>50 ml sterile Falcon tubes (SARSTEDT, catalog number: <a href="https://www.sarstedt.com/en/products/laboratory/reagent-centrifuge-tubes/tubes/product/62.547.254/" target="_blank">62.657.254</a>)</li><li>2 ml sterile Eppendorf tubes (Greiner Bio One International, catalog number: <a href="https://shop.gbo.com/en/row/products/distribution/5_0060/5_0060_0130/5_0060_0130_0030/5_0060_0130_0030_0010/623201.html" target="_blank">623201</a>)</li><li>1.5 ml sterile Eppendorf tubes (Greiner Bio One International, catalog number: <a href="https://shop.gbo.com/en/row/products/bioscience/tubes-beakers/tubes/reaction-tubes/616201.html" target="_blank">616201</a>)</li><li>Disposable pipet tips 200 Âµl (Ultratip, Greiner Bio One International, catalog number: <a href="https://shop.gbo.com/en/row/products/bioscience/liquid-handling/pipette-tips/100-200ul-pipette-tips/739290.html" target="_blank">739290</a>)</li><li>Disposable pipet tips 1,000 Âµl (Ultratip, Greiner Bio One International, catalog number: <a href="https://shop.gbo.com/en/row/products/bioscience/liquid-handling/pipette-tips/100ul-pipette-tips/686290.html" target="_blank">686290</a>)</li><li>Wash bottles (Thermo Fisher Scientific, Thermo Scientific<sup>TM</sup>, catalog number: <a href="https://www.thermofisher.com/search/results?query=2401-0500&amp;focusarea=%E6%90%9C%E7%B4%A2%E5%85%A8%E9%83%A8" target="_blank">2401-0500</a>)</li><li>Glass-bottomed black walled 96 wells plate (Porvair Sciences, catalog number: 324002)</li><li>Black walled clear bottomed 96 wells plastic plate (Greiner Bio One International, catalog number: <a href="https://shop.gbo.com/en/row/products/bioscience/microplates/96-well-microplates/96-well-half-area-microplates/675096.html" target="_blank">675096</a>)</li><li>Stirred Quartz cuvettes (Hellma, catalog number: <a href="http://www.hellma-analytics.com/kuevetten/136/en/pg_id,41$pg_id,41$g_id,12$item_id,129$product_type,$app,$sg,12$sr,$lp,$vol,/fluorescence-cells.html" target="_blank">119004F-10-40</a>)</li><li>Lens cleaning tissue (Fischer Scientific, Fisherbrand<sup>TM</sup>, catalog number: <a href="https://www.fishersci.co.uk/shop/products/grade-913-lens-cleaning-tissue/10605955?searchHijack=true&amp;searchTerm=10605955&amp;searchType=RAPID&amp;matchedCatNo=10605955" target="_blank">10605955</a>)<br/><em>Note: This product has been discontinued.</em> </li><li>Filtropur S 0.2 Âµm filter (SARSTEDT, catalog number: <a href="https://www.sarstedt.com/en/products/laboratory/cell-tissue-culture/filtration/product/83.1826.001/" target="_blank">83.1826.001</a>)</li><li>Plasmids containing cytosolic and periplasmic controls for the sfTq2<sup>ox</sup>-mNG FRET pair as well as additional information are available on: <a href="https://www.addgene.org/browse/article/28196682/" target="_blank">https://www.addgene.org/browse/article/28196682/</a> </li><li>Ethanol 70% in lab wash bottle (VWR, Technisolv<sup>Â®</sup>, catalog number: <a href="https://uk.vwr.com/store/product/2995538/vwrc83801" target="_blank">83801.360</a>)</li><li>Ethanol 96% in lab wash bottle (VWR, Technisolv<sup>Â®</sup>, catalog number: <a href="https://uk.vwr.com/store/product/2357969/ethanol-96-v-v-technisolv-pure" target="_blank">83804.360</a>)</li><li>H<sub>2</sub>O in lab wash bottle</li><li>1 M Î²-D-1-thiogalactopyranoside (IPTG) (Duchefa Biochemie, catalog number: <a href="https://www.duchefa-biochemie.com/product/details/number/I1401" target="_blank">I1401</a>)</li><li>Hellmanex III (Hellma, catalog number: <a href="http://www.hellma-analytics.com/text/197/en/hellmanex%C2%AE-iii.html" target="_blank">9-307-011-4-507</a>)</li><li>Bacto tryptone (Duchefa Biochemie, catalog number: <a href="https://www.duchefa-biochemie.com/product/details/number/T1332" target="_blank">T1332</a>)</li><li>Yeast extract (Duchefa Biochemie, catalog number: <a href="https://www.duchefa-biochemie.com/product/details/number/Y1333" target="_blank">Y1333</a>)</li><li>Sodium chloride (NaCl) (Merck, catalog number: <a href="http://www.merckmillipore.com/INTL/en/product/Sodium-chloride,MDA_CHEM-106404" target="_blank">106404</a>)</li><li>Glucose (Duchefa Biochemie, catalog number: <a href="https://www.duchefa-biochemie.com/product/details/number/G0802" target="_blank">G0802</a>)</li><li>Thiamine hydrochloride (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sial/t4625?lang=en®ion=US" target="_blank">T4625-25G</a>)</li><li>L-Lysine monohydrochloride (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/l8662?lang=en®ion=US" target="_blank">L8662</a>)</li><li>Ampicillin stock solution 100 mg/ml (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sial/a9518?lang=en®ion=US" target="_blank">A9518</a>) in 50% ethanol</li><li>Chloramphenicol stock solution 25 mg/ml (Duchefa Biochemie, catalog number: <a href="https://www.duchefa-biochemie.com/product/details/number/C0113" target="_blank">C0113</a>) in 50% ethanol</li><li>di-Potassium hydrogen phosphate (K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O) (VWR, catalog number: <a href="https://us.vwr.com/store/product/9902214/potassium-phosphate-dibasic-acs" target="_blank">26932.290</a>)</li><li>Potassium dihydrogen phosphate (KH<sub>2</sub>PO<sub>4</sub>) (Merck, catalog number: <a href="http://www.merckmillipore.com/CN/zh/search/104873?search=&amp;TrackingSearchType=SB+-+Search+Result+Search+Box&amp;SearchContextPageletUUID=&amp;SearchTerm=104873&amp;SearchTermHistory=" target="_blank">104873</a>)</li><li>Ammonium sulfate, (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/a4418" target="_blank">A4418</a>)</li><li>Magnesium sulfate heptahydrate (MgSO<sub>4</sub>Â·7H<sub>2</sub>O) (Carl Roth, catalog number: <a href="https://www.carlroth.com/en/en/Chemicals/A-Z-Chemicals/M/Magnesium-sulphate-heptahydrate/Magnesium-sulphate-heptahydrate/p/000000010000a42c00020023_en" target="_blank">T888.1</a>)</li><li>Iron(II) sulfate heptahydrate (FeSO<sub>4</sub>Â·7H<sub>2</sub>O) (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigald/215422?lang=en®ion=US" target="_blank">215422</a>)</li><li>Calcium nitrate tetrahydrate, Ca(NO<sub>3</sub>)<sub>2</sub>Â·4H<sub>2</sub>O (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigald/c1396?lang=en®ion=US" target="_blank">C1396</a>)</li><li>L-Arginine (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/a8094?lang=en®ion=US" target="_blank">A8094</a>)</li><li>L-Glutamine (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/g8540?lang=en®ion=US" target="_blank">G8540</a>)</li><li>Thymidine (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/t1895?lang=en®ion=US" target="_blank">T1895</a>)</li><li>Uracil (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/u1128?lang=en®ion=US" target="_blank">U1128</a>)</li><li>Potassium chloride (KCl) (VWR, catalog number: <a href="https://us.vwr.com/store/product/8086681/potassium-chloride-kcl" target="_blank">71003-522</a>)</li><li>di-Sodium hydrogen phosphate dihydrate (Na<sub>2</sub>HPO<sub>4</sub>Â·2H<sub>2</sub>O) (Merck, catalog number: <a href="http://www.merckmillipore.com/CN/zh/search/106580?search=&amp;TrackingSearchType=SB+-+Search+Result+Search+Box&amp;SearchContextPageletUUID=&amp;SearchTerm=106580&amp;search=&amp;SearchTermHistory=" target="_blank">106580</a>)</li><li>Glutaraldehyde 25% (Merck, catalog number: <a href="http://www.merckmillipore.com/DE/en/product/Glutardialdehyde-solution-250/0,MDA_CHEM-104239" target="_blank">1042390250</a>)</li><li>Formaldehyde â¥ 36.0% (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/47608?lang=en®ion=US" target="_blank">47608</a>)</li><li>Lysine (Sigma-Aldrich, catalog number: <a href="https://www.sigmaaldrich.com/catalog/product/sigma/l8662?lang=en®ion=US" target="_blank">L8662</a>)</li><li>PBS buffer (pH 7.2)</li><li>TY medium (see Recipes)</li><li>20% glucose solution (see Recipes)</li><li>GB1 medium (see Recipes)</li><li>Stock solution (see Recipes)<br/><ol class="npwthree"><li>MM1-10x</li><li>MM2-10x</li><li>MM3-100x</li><li>MM4-100x<br/></li><li>Vitamin B1 stock solution (4 mg/ml)<br/></li><li>Lysine stock solution (20 mg/ml)</li></ol></li><li>Phosphate buffered saline (PBS 1x) (see Recipes)</li><li>Formaldehyde and glutaraldehyde (FAGA) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti37458">### Equipment</p><ol class="npwtwok"><li>P20, P200 &amp; P1,000 Pipetman pipettes (Gilson, catalog number: <a href="http://www.gilson.com/en/Pipette/Products/90.256/Default.aspx?d=381#.WgawttOCyUl" target="_blank">F167300</a>)</li><li>Gyrotory water bath shaker Model G76 (Eppendorf, New Brunswick Scientific, model: Model G76 or GFL-Gesellschaft fÃ¼r Labortechnik, catalog number: 1092) with holders for culturing flasks </li><li>Photospectrometer (Biochrom, model: <a href="http://www.biochrom.co.uk/product/48/biochrom-libra-s22-uv-vis-spectrophotometer.html" target="_blank">Libra S22</a>)</li><li>Compressed air source</li><li>Water purification system (Purelab flex with ELGA LC197-Biofilter from ELGA LabWater, Laner End, HP14 3BY, UK) <br/><em>Note: All H<sub>2</sub>O in the protocol indicates water of at least 18.0 MÎ© from Water purification system.</em> </li><li>Centrifuge for 50 ml Falcon tubes (Eppendorf, model: <a href="https://www.eppendorf.com/product-media/doc/en/244964/Centrifugation_Certificate-EC-Conformity-Declaration_Centrifugen-5804-R_5810-R.pdf" target="_blank">5804 R</a>)</li><li>Centrifuge for 1.5 and 2.0 ml Eppendorf tubes (Eppendorf, model: <a href="https://www.eppendorf.com/product-media/doc/en/145293_Certificate/Centrifugation_Certificate-Compliance_Centrifuge-5424-R.pdf" target="_blank">5424 R</a>)</li><li>Spectrofluorometer Quantamaster 2000-4 (Photon Technology International, NJ) with a red-optimized set-up: R928P PMT tube (185-900 nm), 500 nm blaze (1,200 lines/mm) both excitation and emission gratings<br/><em>Note: A Photomultiplier tube (PMT) that will reliably detect fluorescence spectra up until 700 nm will be suitable for the described protocol.</em> </li><li>Optical filters</li><li>500/10 nm BrightLine<sup>Â®</sup> single-band bandpass filter (Semrock, catalog number: <a href="https://www.semrock.com/filterdetails.aspx?id=ff01-500/10-25" target="_blank">FF01-500/10</a>)</li><li>ET510LP (Chroma, catalog number: <a href="https://cn.chroma.com/products/parts/et510lp" target="_blank">ET510lp</a>)</li><li>435/40 nm BrightLine<sup>Â®</sup> single-band bandpass filter (Semrock, catalog number: <a href="https://www.semrock.com/FilterDetails.aspx?id=FF02-435/40-25" target="_blank">FF02-435/40</a>)</li><li>458LP EdgeBasicâ¢ best-value long-pass edge filter (Semrock, catalog number: <a href="https://www.semrock.com/filterdetails.aspx?id=blp01-458r-25" target="_blank">BLP01-458R</a>)</li><li>Fluorescence scanning plate reader (BioTek Instruments, model: <a href="https://www.biotek.com/products/detection?Product_Interest_Source=PPC%20-%20Adwords&amp;source=PPC%20-%20Adwords&amp;Lead_Campaign_Source=7011O000002tr2g&amp;Campaign_Source=7011O000002tr2g&amp;gclid=EAIaIQobChMI1aa_nuys5QIVjRitBh3aMQYLEAAYAyAAEgIpQvD_BwE" target="_blank">Synergy<sup>TM</sup> Mx</a>, catalog number: SMATBC)</li><li>Autoclave (Sanyo, catalog number: MLS-3780) <br/></li></ol><p class="pptt" id="biaoti37459">### Software</p><ol class="npwtwok"><li>Fluorescence spectrophotometer software (PTI acquisition software FeliX32 version 1.2 Build 56)</li><li>Microplate reader software Gen5 version 2.00 (Biotek)</li><li>Excel (Microsoft)<br/></li></ol><p class="pptt" id="biaoti37460">### Procedure</p><p class="ppzz">The FRET method from beginning to end encompasses several steps, which are discussed per subject. The procedure and sections are outlined in Figure 1. This protocol describes our FRET assay with sfTq2<sup>ox</sup> and mNG, for a full description of sections A and B we refer to <a href="https://bio-protocol.org/e2697" target="_blank">https://bio-protocol.org/e2697.</a><br/><br/><img alt="" class="layerphoto" height="155" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202083259_5576.jpg" width="300"/><br/><strong>Figure 1. Workflow of the bacterial spectral-based FRET method.</strong> The labeling of the panels corresponds to the section describing the specific protocols. A. Choosing the suitable FRET-pair and creating expression plasmids. B. Testing the constructed plasmids for functionality. C. Growing cultures steady state, inducing expression of the FRET constructs and harvesting cells. D. Sample preparation and data acquisition. E. Analysis of the data.</p><br/><ol class="npwone"><li>Choosing a suitable FRET pair for your experiment and creating FP fusions<ol class="npwtwo"><li>Choosing a suitable FRET pair for your experiment<br/>Before starting FRET experiments, the most suitable FRET-pair should be selected. To date the FRET-pair that yields the highest rates of energy transfer in the cytoplasm and the periplasm of <em>E. coli</em> is super folder mTurquoise2 ox (sfTq2<sup>ox</sup>) (Meiresonne <em>et al.</em>, 2019)âmNeonGreen (mNG) (Shaner <em>et al.</em>, 2013). The high FRET efficiency of the sfTq2<sup>ox</sup>-mNG pair is related to the substantial overlap of their respective emission and excitation spectra, as well as their considerable quantum yield and extinction coefficient (Figure 2, Table 1). It is conceivable that a FRET pair is required with lower intrinsic background fluorescence. For this, the monomeric Kusabira-Orange (mKO) (Karasawa <em>et al.</em>, 2004)âmCherry (mCh) (Shaner <em>et al.</em>, 2004) could be attempted. The mKO-mCh FRET protocol as well as other alternative FRET pairs is described in <a href="https://bio-protocol.org/e2697" target="_blank">https://bio-protocol.org/e2697.</a><br/><br/><img alt="" class="layerphoto" height="121" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202083409_8534.jpg" width="300"/><br/><strong>Figure 2. Spectral overlap and Forster distance of the mTq2-mNG FRET pair.</strong> On the left, spectral overlap between emission spectrum of mTq2 and absorbance spectrum of mNG. SfTq2<sup>ox</sup> has to date not been isolated and therefore its spectrum is not available. However, since the spectral properties of mTq2 and sfTq2<sup>ox</sup> in cells are identical, the shown mTq2 spectrum is identical to that of sfTq2<sup>ox</sup>. On the right, plotted theoretical FRET efficiencies as a function of chromophore distance for the mTq2-mNG, FRET pairs using the spectral properties in Table 1, the mNG-mCh FRET pair is shown for comparison. R<sub>0</sub> is the distance between the fluorophores that will yield 50% energy transfer.<br/><br/><strong>Table 1. Information on the FRET pairs described in this bio-protocol </strong><br/><img alt="" class="layerphoto" height="298" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202083453_3161.png" width="800"/><br/>*As described in the original papers (Meiresonne <em>et al.</em>, 2017 and 2019).<br/><br/><em>Notes: </em><br/><ol class="npwthree"><li><em>For the R<sub>0</sub> values presented in Table 1, the originally published extinction coefficient of 72,000 M</em><sup><em>-1</em></sup><em> cm</em><sup><em>-1</em></sup><em> of mCh was used (Shaner </em><em>et al.</em><em>, 2004). Variation occurs in the description of FP extinction coefficients based on detection accuracy and sample conditions. The ever-improving field of FP research shows that the extinction coefficient of mCh is more in the range of 85,000 M<sup>-1</sup> cm<sup>-1</sup>(Cranfill et al., 2016; Bindels et al., 2017).</em> </li><li><em>The quantum yield of sfTq2<sup>ox</sup> (and sfTq2) was approximated by fluorescence lifetime to not differ from mTq2 (Goedhart et al., 2012; Meiresonne et al., 2019).</em> </li><li><em>Different FP pairs, have been successfully used for bacterial FRET. For more information, for reference see </em><a href="https://bio-protocol.org/e2697" target="_blank"><em>https://bio-protocol.org/e2697.</em></a> </li></ol><a href="https://bio-protocol.org/e2697" target="_blank"></a><br/></li><li>Creating FP fusions<br/>Bacterial genes encoding a protein of interest are fused to a gene encoding one of the described fluorescent proteins and expressed from an inducible expression vector. Any plasmid expression system can be used, provided that the plasmids are compatible (different origins of replication and different antibiotic resistance markers) and the expression level is low (typically 2,000 copies of the protein) to avoid nonspecific interactions. To give the FP and the protein of interest some freedom of movement and reduce steric hindrance, it can be useful to add a linker between the FP and the protein. GGS repeats are often used but we also have good results with ARNNNN and other amino acid sequences. Another consideration is the choice for an N- or C-terminal fusion. For instance, N-terminal fusions of proteins that encode signal sequences for downstream transport or processing should be avoided or circumvented. Additionally, the N- or C-terminus of a protein may be essential for its normal localization or functionality. Procedure B discusses testing the created fusions. The sequence of the plasmid should be confirmed before further experiments. A description of cloning and transforming <em>E. coli</em> with plasmids is beyond the scope of this protocol but other bio-protocols like â[BIO101] Standard cloningâ and â[BIO101] The Inoue Method for Preparation and Transformation of Competent <em>E. coli</em>: âUltra Competentâ Cellsâ (He, 2011; Im, 2011) may be useful. <br/><br/></li></ol></li><li>Testing the FP fusions<br/>Ideally, the FP-fusions should be fully functional and expressed at levels close to their native levels while still resulting in measurable fluorescence intensities. Once the fusions are created on a plasmid, the expressed products should be tested for their effects on phenotype, fluorescence and correct cellular localization. If no fluorescence is observed, degradation of the FP may be assessed by Western blot analysis (anti-GFP for sfTq2<sup>ox</sup>, Invitrogen <a href="https://www.thermofisher.com/antibody/product/GFP-Antibody-Polyclonal/A-6455" target="_blank">A6455</a> and anti-mNG, Chromogen <a href="https://www.chromotek.com/products/detail/product-detail/mneongreen-antibody-32f6/" target="_blank">32F6</a>). Perhaps the fusion disrupts normal localization, functionality and interactions of the protein of interest. This may be amended by modifying the length of the linker or switching the position of the FP. Some proteins may neither be functional as N nor as C-terminal fusion but will work as a sandwich fusion (van der Ploeg <em>et al.</em>, 2015). An FP-fusion should complement the loss of function from a deletion or depletion strain of that same protein. Complementation testing will depend on whether the loss of the gene results in a phenotype. If no complementation experiments are possible, FRET experiments can be done in either a wild-type or deletion background but it should be kept in mind that the fusion protein may not be functional and results could be misinterpreted.<br/><em>Note: Plasmid-based FP-fusions are a great way of quickly testing localization or the interaction of proteins of interest. However, plasmid-based complementation is not always possible due to stoichiometry imbalance. In such cases, the FP-fusion could be inserted in the chromosome at the original locus. In fact, chromosomal insertions of well-established FRET partners would be preferred for easy assays. However, this would also require chromosomal references in an isogenic background because bacteria can respond differently to plasmid and chromosomal based protein expression with respect to growth differences in and possible autofluorescence due to plasmid maintenance. </em><br/><br/><strong><em>Preparation for the FRET experiment </em></strong><br/>When the FP fusion proteins are ready for use, competent cells can be transformed with the donor and acceptor expressing plasmids. Any <em>E. coli</em> strain should be suitable. Because we are interested in proteins that affect morphology, we prefer to use wild-type strains like MC4100 or BW25113 that are morphogenetically homogenous. <br/><em>Note: If deletion strains are used the references and controls should be in the wild-type variant. The FRET assay described, requires at least three reference samples that are expressed in cells of the same genetic background as the assessed interaction. Strongly advised is the use of positive and negative controls consisting of a plasmid expressing sfTq2<sup>ox</sup> fused to mNG and two plasmids expressing the sfTq2<sup>ox</sup> or mNG fused to unrelated proteins that are known not to interact, respectively. The best controls are expressed in the same compartment as where the protein-protein interaction takes place to be able to assess molecular crowding effects. It is worthwhile to also switch donor and acceptor fusions (i.e., protein 1 is fused to sfTq2<sup>ox</sup> or mNG and protein 2 has the opposite FP). This is to rule out biased results due to differences of expression due to plasmid copy number etc. and the found FRET values should be similar. A list of samples for a minimal sfTq2<sup>ox</sup>-mNG FRET experiment are shown in Table 2.</em><br/><br/><strong>Table 2. Samples and plasmids required for a minimal FRET experiment</strong><br/><img alt="" class="layerphoto" height="267" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202083611_2644.png" width="800"/><br/><sup>a</sup>To detect the interaction between two proteins of interest at least three references are required. Positive and negative controls are advised. The described protocol is for two compatible plasmids that have a p15 origin of replication combined with chloramphenicol resistance and a ColE1 origin of replication combined with ampicillin resistance. Both plasmids have a weakened <em>trc</em>99A promoter and two terminators.<br/>*Several positive control plasmids for cytoplasmic and periplasmic expression are available on <a href="https://www.addgene.org/browse/article/28196682/" target="_blank">https://www.addgene.org/browse/article/28196682/.<br/></a> <br/></li><li>Steady state growth, induction and harvesting cells<br/>Bacterial cells that are grown in rich medium often contain a high concentration of autofluorescing molecules that decrease the signal to noise ratio. For that reason, the cells are grown in minimal medium and FRET experiments performed in rich medium are discouraged. To allow comparison of the various bacterial cultures expressing FRET constructs the cells are grown to steady state in about 20 generations before the expression of the proteins is induced (Figure 3). This assures identical autofluorescence backgrounds in all samples. After induction of expression for at least 2 mass doublings the cells are fixed and harvested. <br/><ol class="npwtwo"><li>Prepare starter cultures of all strains required to run your FRET experiment in 5 ml TY + ampicillin 100 Âµg/ml + chloramphenicol 25 Âµg/ml + 0.5% glucose medium at 37 Â°C while shaking until bacterial growth becomes visible (<strong>Day 0</strong>).<br/><em>Note: Glucose is added to the medium to repress the expression of the lacIq controlled gene from the plasmids.</em> </li><li>Use the starter cultures to inoculate 25 ml (1:250) of Gb1 medium + ampicillin + chloramphenicol and continue overnight growth at 28 Â°C while shaking. </li><li><strong>Day 1</strong><strong>.</strong> Dilute the overnight cultures 1:250 in fresh Gb1 medium + ampicillin + chloramphenicol and continue growth at 28 Â°C while shaking. <br/><em>Note: From now on periodically measure and note OD<sub>450</sub> values of the cultures and do not let them reach above 0.2. When the cultures approach this value dilute with fresh pre-warmed medium. </em> </li><li>Calculate the doubling time (TD) for all growing cultures using the provided Excel sheet â<a href="https://os.bio-protocol.org/attached/file/20191022/protocol 1902854 Growth Calculation.xlsx" target="_blank"><strong>Growth calculation.xlsx</strong></a>â and use it to calculate the dilution factor to reach OD<sub>450</sub> values of 0.1 the next morning when the experiment is continued. </li><li><strong>Day 2.</strong> Repeat the growing in fresh Gb1 medium + ampicillin + chloramphenicol while keeping the OD<sub>450</sub> values below 0.2 and dilute for the next morning like in Step C4.</li><li><strong>Day 3.</strong> Repeat the growing in fresh Gb1 medium + ampicillin and chloramphenicol while keeping the OD<sub>450</sub> values below 0.2 and dilute for the next morning like in Step C4 but this time aim for an OD<sub>450</sub> value of 0.025 in 35 ml fresh Gb1 + ampicillin + chloramphenicol.</li><li><strong>Day 4.</strong> Measure OD<sub>450</sub> values of the cultures and if they are around 0.03-0.04 induce expression from the plasmids by adding IPTG at an appropriate concentration (<em>e.g.</em>, 10-20 ÂµM) and continue growth.</li><li>Before the cultures approach OD<sub>450</sub> values of ~0.2, prepare a sufficient amount of FAGA fixative (Recipe 5).<br/><em>Note: The total volume of FAGA needed differs depending on the total volume of the to-be-fixed cultures, 1 ml of FAGA is required to fix 12.5 ml of culture.</em> </li><li>When the cells reach OD<sub>450</sub> values of 0.2, fix the cells for 10 min in the shaking water bath at 28 Â°C with FAGA, which amount depends on how much of the cultureâs volume is left.</li><li>Decant the fixed cultures in 50 ml Falcon tubes and pellet the cells by centrifugation for 10 min at 6,300 <em>x g</em> at room temperature (Eppendorf 5804 R).</li><li>Discard the supernatant and wash 3 x with 1 ml PBS in a 1.5 ml Eppendorf tube centrifuging for 5 min at 4,600 <em>x g</em> at room temperature in a tabletop centrifuge (Eppendorf 5424 R).</li><li>Resuspend the last pellet in 1 ml of PBS and store the samples overnight at 4 Â°C and measure the next day.<br/><em>Note: sfTq2<sup>ox</sup> and mNG fold fast and their chromophore matures quickly so direct measurements could be considered, but it is more practical to do the spectral measurements a day after.</em><br/><br/><img alt="" class="layerphoto" height="124" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202083655_8864.jpg" width="300"/><br/><strong>Figure 3. Example of steady state growth.</strong> Flask culture growth curves of MC4100 based <em>E. coli</em> cultures were grown to steady state in Gb1 at 28 Â°C expressing FP-fusions. The circles represent the time points at which overnight dilutions were made. At Day 4 the cultures were induced with IPTG (arrow) to induce production of FP-fusions. The EV (Empty Vector) culture does not express any FP and serves as a control. The X-axis is given in OD<sub>450</sub> to signify exponential growth. Cells were fixed and harvested (asterisk) after at least two mass doublings after induction. This figure was adapted from (Meiresonne <em>et al.</em>, 2017).<br/><br/></li></ol></li><li>Sample preparation and data acquisition<br/><ol class="npwtwo"><li>Pellet the cells as described in Step C10 and resuspend in 1 ml PBS.</li><li>Prepare at least 1,300 Âµl of samples of with OD<sub>450</sub> values of 1.00 Â± 0.005 by diluting the cells with fresh PBS and add 1,200 Âµl to the stirred quartz cuvette.<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>The dilution can be made inside the photospectrometer; the more equal the OD<sub>450</sub> values are; the more reliable will be the results.</em> </li><li><em>Do not forget to add the magnetic stirrer to the cuvette before measuring.</em> </li><li><em>In theory plastic cuvettes could be used for measurements of spectra at visible wavelengths, however, that was not attempted for the experiments described in this protocol and should be tested first. The cuvettes should be identical for each sample to avoid differences in background.</em> </li></ol></li><li>Measure the mNG emission spectrum using the settings and optical filters indicated in Table 3 and described in â<strong>Measuring spectra using a fluorometer</strong>âFluorometer settings and practical setupâ.<br/><br/><strong>Table 3. Filter setup for fluorometer spectral measurements</strong><br/><img alt="" class="layerphoto" height="114" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202083741_6257.png" width="600"/><br/><sup>a</sup>Filters with similar transmission properties can be used.<br/><br/></li><li>Measure the donor emission spectrum using the settings and optical filters indicated in Table 3 and described in â<strong>Measuring spectra using a fluorometer</strong>âFluorometer settings and practical setupâ.<br/><em>Note: It is advisable to measure the mNG and sfTq2</em><sup><em>ox</em></sup><em> spectra of each sample in the same order to prevent possible differences in FP bleaching. </em> </li><li>Repeat Steps D2-D4 for each sample and also measure a blank PBS only sample at the start of the measurement and the end of all measurements. This allows you to verify any change in the instrument response.</li><li>Save the data in any convenient file format that can be exported to Microsoft Excel.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Before loading a new sample in the quartz cuvette, it needs to be thoroughly cleaned by performing the steps outlined in the cleaning protocol below.</em> </li><li><em>By saving the OD<sub>450</sub> = 1.00 samples they can directly be reused for spectral measurements with the plate reader described in â<strong>Measuring spectra with the plate reader</strong>â. </em> </li></ol></li></ol><br/><em><strong>Measuring spectra using a fluorometer</strong>âFluorometer settings and practical setup</em><br/>Emission spectra are measured using a fluorometer. The fluorometer should excite a sample with a narrow band wavelength, which then emits fluorescence that is measured. A schematic overview of a fluorometer is shown in Figure 4. The light source emits white light, which passes a slit to the excitation monochromator where it is separated into its wavelength components. The light passes a slit before passing the excitation filter and hits the sample suspension in the quartz cuvette. The cells in the sample will be excited and emit light in every direction but excitation light also scatters from them. At an angle of 90Â°, the scattered light is filtered by an emission filter and the fluorescence emission passes a slit entering the emission monochromator. Here, the emission wavelengths are scanned and pass the last slit before entering a photomultiplier where quanta of photons are being transformed into a measurable electrical current. The resulting emission spectra are the raw data needed to calculate FRET. The filter and measurement settings are shown in Table 3. Slit widths are usually set to 6 nm and generally give a good result if the fluorescent proteins are expressed well. Smaller slit widths result in more specific measurements but will also let less light pass to be detected so slits should only be more closed when fluorescent signals are very high. If the fluorescent signals are low, the slits could be widened but this will result in less specific signals. When background fluorescence is an issue, this will increase in a proportional manner.<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>The white light source has to heat up before the light output will be stable. Thirty minutes after turning it on, check whether the power of the lamp is 75 W and if not adjust. Other fluorometers will have different requirements.</em> </li><li><em>The use of other white light sources or equipment may require different preparations.</em> </li></ol><br/><img alt="" class="layerphoto" height="197" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202084006_3236.jpg" width="300"/><br/><strong>Figure 4. Schematic overview of measuring a sample in the fluorometer.</strong> The double black and gray circles at the monochromators represent adjustable slits.<br/><br/><strong>Protocol cleaning quartz cuvettes</strong><br/><ol class="npwthree"><li>Profusely rinse the inside and outside of the cuvette with H<sub>2</sub>O.</li><li>Carefully shake off the excess H<sub>2</sub>O.</li><li>Profusely rinse the inside and outside with 70% ethanol.</li><li>Carefully shake off the excess 70% ethanol.<br/></li><li>Repeat steps a-d (Protocol cleaning quartz cuvettes), three times.<br/></li><li>Profusely rinse the inside and outside with 96% ethanol.<br/></li><li>Carefully shake off the excess ethanol 96%.<br/></li><li>Blow dry the cuvette with compressed air.<br/></li><li>Polish the cuvette with lens tissue to get rid of accidental smudges like for instance fingerprints. Now the cuvettes are properly cleaned and can hold a sample for measurement.<br/><em>Notes: </em><br/><ol class="npwfour"><li><em>The magnetic stirrers beans are washed by the same rinsing protocol in the palm of your hand.</em> </li><li><em>After the last measurement, wash all cuvettes and stirrer beans one more time and then submerge them in 2% Hellmanex III for storage. If cuvettes are not needed for longer time periods, store them clean and dry.</em> </li><li><em>Quartz cuvettes should be cared for meticulously as they are fragile and expensive. Only touch the cuvettes with your fingers close to the opening to prevent finger prints on the important part of the quartz. Carry the cuvette between your index finger and thumb while supporting the bottom with your pinky or ring finger as shown in Figure 5.</em> </li></ol><br/><img alt="" class="layerphoto" height="243" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202084207_9608.jpg" width="300"/><br/><strong>Figure 5. Cartoon of how to hold quartz cuvettes properly</strong> (<em>i.e.</em>, between thumb and index finger supported by the little finger)<br/></li></ol><br/><strong>Measuring spectra with the plate reader</strong><br/>A convenient way of measuring the emission spectra of your samples is by using a fluorescence plate reader. A word of caution is in order as the sensitivity of most fluorescence plate readers is lower than that of a dedicated fluorometer and provide weaker fluorescence signals. If fluorescence is low, the fluorometer should be used. This bio-protocol describes the use of the BioTek synergy MX plate reader, which has excitation and emission monochromators without the possibility for additional optical filtering. In addition, this plate reader has slits with a minimal opening of 9 nm. The, on average, lower signal to noise and less specific signals from a plate reader may therefore lead to unmixing problems. It is thus recommended to first firmly establish FRET interactions using a fluorometer and then confirming them using the plate reader. If results are reproducible, experiments can be done with the plate reader only. <br/><div><br/><strong><em>Protocol plate reader FRET for fixed cells</em></strong><br/><ol class="npwthree"><li>Prepare samples with OD<sub>450</sub> values of 1.00 Â± 0.005 like described in Steps D1-D2.</li><li>Dispense 200 Âµl replicates of the samples in a glass bottomed black walled 96-well plate. If eight replicates are made, there is space for 11 prepared samples and 1 PBS measurement in a 96-well plate.</li><li>Measure the donor and acceptor emission spectra of each well using the settings shown in Table 4.</li><li>Export the emission data in an Excel compatible format and plot the replicates to check for obvious outlying data.<br/></li><li>Exclude the possible outliers based on large and obvious deviations from the other replicates and average the acceptor and donor emission spectra. These data will be used for spectral unmixing and FRET calculations.<br/><div><span style="font-size:10pt;"></span> </div></li></ol><br/></div><strong>Table 4. Settings for spectral measurements using the plate reader</strong><br/><img alt="" class="layerphoto" height="189" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202084302_7444.png" width="520"/><br/>*Optimal gain may differ between experiments. It is better to have slightly lower signals than to oversaturate the detector.<br/><em>Note: Measuring the OD<sub>450</sub> values in the plate reader before the spectral measurements will give an indication how similar the samples are to each other.</em><br/><div><br/></div></li><li>Data analysis<br/>When the acceptor and donor emission spectra of all samples are measured either by the fluorometer or by the plate reader, they can be unmixed. Spectral unmixing dissects the emission data into its components based on the references that were provided. The sfTq2<sup>ox</sup> channel is composed of background emission, sfTq2<sup>ox</sup>-only emission, directly excited mNG emission and possible FRET mNG emission (Figure 6). The mNG channel is composed of background emission and mNG-only emission. From this spectrum, the amount of mNG in the sample can be calculated using the background and reference spectra by unmixing. Knowing the amount of mNG fluorescence, it can be calculated how much fluorescence is expected from the mNG due to cross-excitation of the donor and the acceptor in the sample that contains both. Using the reference spectra and the knowledge on the amount of mNG in the sample, the emission spectrum for a sample can be dissected into the background spectrum, the sfTq2<sup>ox</sup> spectrum and the mNG spectrum. Remaining is a spectrum of identical shape as the mNG spectrum that cannot be accounted for if no sensitized emission (energy transfer) had occurred. This extra mNG signal corresponds to the amount of energy transfer. The supplementary data of (Alexeeva <em>et al.</em>, 2010) gives a detailed description of the calculations based on biophysical work beyond the scope of a protocol (Clegg, 1992; Clegg <em>et al.</em>, 1992; Wlodarczyk <em>et al.</em>, 2008; Gadella, 2009). In the supplementary information, ready-to-use Excel sheets are provided to calculate FRET efficiencies from sfTq2<sup>ox</sup>-mNG experiments measured with the fluorometer or plate reader <a href="https://os.bio-protocol.org/attached/file/20191205/protocol 1902854 sfTq2ox mNG.xlsx" target="_blank">âsfTq2ox-mNG.xlsxâ</a> <a href="https://os.bio-protocol.org/attached/file/20191205/protocol 1902854 sfTq2ox mNG platereader.xlsx" target="_blank">âsfTq2ox-mNG platereader.xlsxâ</a>.<br/><br/><img alt="" class="layerphoto" height="330" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202084547_7378.jpg" width="300"/><br/><strong>Figure 6. Principle of the sfTq2ox-mNG FRET assay.</strong> A. Excitation and emission spectra of mTq2 and mNG, indicating the wavelengths used to measure the mNG channel (left) or the sfTq2<sup>ox</sup> channel (right). The bold dotted line represents the used excitation wavelengths. The hatched blue area represents the spectral overlap between the mTq2 emission spectrum and the mNG excitation spectrum. The gray shaded area represents the wavelengths for which the emission was measured. B. Samples measured to calculate periplasmic FRET. References of background, mNG, and sfTq2<sup>ox</sup> are needed to calculate their contributions to the FRET within a sample. C. Unmixing of the tandem FRET sample, showing the measured spectrum as black dots (tandem) and the calculated spectrum as a solid red line (Unmix. Calc.). The measured spectrum for the mNG channel is composed of the background fluorescence (Unmixed Empty cells) and the amount of mNG (Unmixed Acceptor) present in the sample excited at 504 nm. The direct excitation of mNG is used to determine the amount of mNG present in the sample. This amount is then used to calculate how much extra mNG (Unmixed SE) is detected in the sfTq2<sup>ox</sup> excited sample. The measured spectrum for the sfTq2<sup>ox</sup> channel contains the background (Unmixed Empty cells), sfTq2<sup>ox</sup> (Unmixed Donor), and mNG (Unmixed Acceptor) fluorescence and sensitized emission (Unmixed SE), which is the extra fluorescence that the unmixing algorithm cannot attribute to direct excitation of mNG. D. The difference between the measured and calculated spectra represents signal that cannot be attributed to the reference components and is a measure of the quality of unmixing.<br/><br/><strong><em>Protocol using the unmixing sheets </em></strong><br/>An overview of how the unmixing process would appear on the computer screen is shown in Figure 7. High resolution screenshots are available in the <a href="https://os.bio-protocol.org/attached/file/20191022/protocol 1902845 Unmixing screens.pptx" target="_blank">supplementary data.</a><br/><em>Note: Be aware that the raw data may have different output formats depending on the equipment used and adjust accordingly.</em><br/><div><br/><ol class="npwtwo"><li>Copy the spectral data in the appropriate excel sheet 1 âRaw dataâ.</li><li>Subtract the PBS measurement from all samples in sheet 2 âMinus PBSâ.</li><li>Copy the empty vector, acceptor reference and the donor reference in the indicated places in sheet 3 âDataâ. This will automatically subtract the background signal from the acceptor and donor reference spectra.</li><li>Copy sample data to the indicated place in sheet 3 âDataâ the sheet will unmix these spectra based on the given references. <br/><em>Note: The reference and sample spectra are plotted at the bottom of sheet 3 for inspection.</em> </li><li>Inspect the unmixing overview on sheet 4 âResultâ as described further below.</li><li>Copy paste the results to sheet 5 âSummariesâ.</li><li>Repeat steps 4-6 for the remaining samples.<br/><br/><img alt="" class="layerphoto" height="250" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191202/20191202084708_3666.jpg" width="300"/><br/><strong>Figure 7. The unmixing process for the calculation of FRET efficiencies as it would appear on the screen.</strong> Raw donor and acceptor unmixing calculations that will not be directly used are shown in the high-resolution screenshots in Supplementary file <a href="https://os.bio-protocol.org/attached/file/20191022/protocol 1902845 Unmixing screens.pptx" target="_blank">âUnmixing screens.pptxâ.</a><br/><br/></li></ol></div><strong><em>What determines a good measurement? </em></strong><br/>Spectral unmixing is highly dependent on clear fluorescent signals that should be measured under identical conditions. The donor and acceptor reference spectra are obtained by subtracting the PBS and empty cells data from their respective emission measurements as described in â<strong>Protocol using the unmixing sheets</strong>â. This should yield a clean emission spectrum of sfTq2<sup>ox</sup> only or mNG only. Sometimes spectra that do not resemble the measured FP anymore or even negative can be seen. This can be caused by strong levels of autofluorescence in the empty cells sample and low FP signals in the reference samples. If one of the references is not good, the unmixing of the spectra will not give trustworthy results. Alternatively, reference spectra of a previous measurement may be used to still get a rough indication of the FRET values of the samples. It is not advised to use these values as definite and the experiment needs to be repeated to yield trustworthy data. When the experiment results in good references, sample data can be unmixed. Emission signal that could not be attributed to the mNG spectrum is called the unmixing difference. The unmixing difference is a good measure for the quality of the measured FRET spectra and in an ideal experiment it should be zero. More realistically, it should be a small percentage of the measured spectra as shown in Figure 6D.</li></ol><p class="pptt" id="biaoti37461">### Data analysis</p><p class="ppzz">This bio-protocol describes how to prepare, perform and analyze <em>in vivo</em> bacterial FRET experiments. At the relevant sections (Procedure E), information is provided on data analysis and references to in-depth technical information are provided. If the protocol is rigorously followed every time, the resulting data will be highly similar. The significance of the obtained FRET values can be calculated using standard confidence interval statistical approaches. The FRET values should be significantly higher than the values obtained from non-interacting samples due to crowding. For this analysis, standard <em>t</em>-tests can be used.</p><p class="pptt" id="biaoti37462">### Recipes</p><ol class="npwtwok"><li>TY medium (1 L)<br/>10 g Bacto tryptone<br/>5 g yeast extract<br/>5 g NaCl<br/>Add H<sub>2</sub>O up to 1 L and autoclave for 15 min at 121 Â°C</li><li>20% glucose solution (200 ml)<br/>40 g glucose<br/>Add H<sub>2</sub>O up to 200 ml and filter sterilize with Filtropur S 0.2 Âµm filter</li><li>Stock solution<br/><ol class="npwthree"><li>MM1-10x (1 L)<br/>63.3 g K<sub>2</sub>HPO<sub>4</sub>Â·3H<sub>2</sub>O <br/>29.5 g KH<sub>2</sub>PO<sub>4</sub><br/>Add H<sub>2</sub>O up to 1 L and autoclave for 15 min at 121 Â°C<br/></li><li>MM2-10x (1 L)<br/>10.5 g (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub><br/>1 g MgSO<sub>4</sub>Â·7H<sub>2</sub>O<br/>Add H<sub>2</sub>O up to1 L and autoclave for 15 min at 121 Â°C<br/></li><li>MM3-100x (1 L)<br/>28 mg FeSO<sub>4</sub>Â·7H<sub>2</sub>O<br/>710 mg Ca(NO<sub>3</sub>)<sub>2</sub>Â·4H<sub>2</sub>O<br/>Add H<sub>2</sub>O up to 1 L and autoclave for 15 min at 121 Â°C<br/><em>Note: After sterilization, the MM3 will show a red precipitation that should be allowed to deposit and not be touched when pipetting.</em><br/></li><li>MM4-100x (50 ml)<br/>250 mg L-arginine<br/>250 mg L-glutamine<br/>10 mg thymidine<br/>100 mg uracil<br/>Add H<sub>2</sub>O up to 50 ml and filter sterilize with Filtropur S 0.2 Âµm filter</li><li>Vitamin B1 stock solution (4 mg/ml) (50 ml)<br/>200 mg thiamine hydrochloride <br/>Add H<sub>2</sub>O up to 50 ml and filter sterilize with Filtropur S 0.2 Âµm filter</li><li>Lysine stock solution (20 mg/ml) (50 ml)<br/>1 g lysine<br/>Add H<sub>2</sub>O up to 50 ml and filter sterilize with Filtropur S 0.2 Âµm filter<br/></li></ol></li><li>Gb1 medium (1 L) (using aseptic technique)<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:300px;"><tbody><tr><td>100 ml <br/></td><td>MM1-10x <br/></td></tr><tr><td>100 ml <br/></td><td>MM2-10x <br/></td></tr><tr><td>10 ml <br/></td><td>MM3-100x <br/></td></tr><tr><td>10 ml <br/></td><td>MM4-100x <br/></td></tr></tbody></table><em>Note: MM4-100x has to be added to GB1 medium for BW25113, TB28 and MG1655 based E. coli strains.</em><br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>20 ml<br/></td><td>20% glucose <br/></td></tr><tr><td>1 ml <br/></td><td>4 mg/ml thiamine hydrochloride <br/></td></tr><tr><td>2.5 ml <br/></td><td>20 mg/ml L-lysine monohydrochloride <br/></td></tr><tr><td>1 ml <br/></td><td>100 mg/ml ampicillin <br/></td></tr><tr><td>1 ml <br/></td><td>25 mg/ml chloramphenicol <br/></td></tr></tbody></table> Add sterile H<sub>2</sub>O up to 1 L</li><li>Phosphate buffered saline (PBS 1x)<br/>PBS buffer (pH 7.2)<br/>140 mM NaCl<br/>27 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub>Â·2H<sub>2</sub>O<br/>2 mM KH<sub>2</sub>PO<sub>4</sub> </li><li>Formaldehyde and glutaraldehyde (FAGA) fixative<br/>Per 1 ml of formaldehyde add 21 Âµl of glutaraldehyde<br/></li></ol><p class="pptt" id="biaoti37463">### Acknowledgments</p><p class="ppzz">This bio-protocol is a sfTq2<sup>ox</sup>-mNG specific adaptation of <a href="https://bio-protocol.org/e2697" target="_blank">https://bio-protocol.org/e2697</a>, which was co-authored by Svetlana Alexeeva and RenÃ© van der Ploeg. Adaptations of this bio-protocol have been used in the following publications (Alexeeva <em>et al.</em>, 2010; Fraipont <em>et al.</em>, 2011; van der Ploeg <em>et al.</em>, 2013 and 2015; Meiresonne <em>et al.</em>, 2017 and 2019). NYM was supported by the NWO, ALW open program (822.02.019) and HFSP Program RGP0034/2013. E.C. and L.M.Y.M. were supported by the European Commission via the International Training Network Train2Target (721484).</p><p class="pptt" id="biaoti37464">### References</p><ol class="npwtwok"><li>Alexeeva, S., Gadella, T. W., Jr., Verheul, J., Verhoeven, G. S. and den Blaauwen, T. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20497333" target="_blank">Direct interactions of early and late assembling division proteins in <em>Escherichia coli</em> cells resolved by FRET.</a> <em>Mol Microbiol</em> 77(2): 384-398.</li><li>Algar, W. R., Hildebrandt, N., Vogel, S. S. and Medintz, I. L.(2019) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31471616" target="_blank">FRET as a biomolecular research toolâunderstanding its potential while avoiding pitfalls.</a> <em>Nat Methods</em> 16(9): 815â829.</li><li>Bindels, D. S., Haarbosch, L., van Weeren, L., Postma, M., Wiese, K. E., Mastop, M., Aumonier, S., Gotthard, G., Royant, A., Hink, M. A. and Gadella, T. W., Jr. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27869816" target="_blank">mScarlet: a bright monomeric red fluorescent protein for cellular imaging.</a> <em>Nat Methods</em> 14(1): 53-56.</li><li>Clegg, R. M. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1406315" target="_blank">Fluorescence resonance energy transfer and nucleic acids.</a> <em>Methods Enzymol</em> 211: 353-388.</li><li>Clegg, R. M., Murchie, A. I., Zechel, A., Carlberg, C., Diekmann, S. and Lilley, D. M. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1591245" target="_blank">Fluorescence resonance energy transfer analysis of the structure of the four-way DNA junction.</a> <em>Biochemistry</em> 31(20): 4846-4856.</li><li>Cranfill, P. J., Sell, B. R., Baird, M. A., Allen, J. R., Lavagnino, Z., de Gruiter, H. M., Kremers, G. J., Davidson, M. W., Ustione, A. and Piston, D. W. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27240257" target="_blank">Quantitative assessment of fluorescent proteins.</a> <em>Nat Methods</em> 13(7): 557-562.</li><li>Fraipont, C., Alexeeva, S., Wolf, B., van der Ploeg, R., Schloesser, M., den Blaauwen, T. and Nguyen-Disteche, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20847002" target="_blank">The integral membrane FtsW protein and peptidoglycan synthase PBP3 form a subcomplex in <em>Escherichia coli</em>.</a> <em>Microbiology</em> 157(Pt 1): 251-259.</li><li>Gadella T. W. J. (2009). <a href="https://www.elsevier.com/books/fret-and-flim-techniques/gadella/978-0-08-054958-3" target="_blank">FRET and FLIM techniques.</a> Elsevier.</li><li>Goedhart, J., von Stetten, D., Noirclerc-Savoye, M., Lelimousin, M., Joosen, L., Hink, M. A., van Weeren, L., Gadella, T. W., Jr. and Royant, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22434194" target="_blank">Structure-guided evolution of cyan fluorescent proteins towards a quantum yield of 93%.</a> <em>Nat Commun</em> 3: 751.</li><li>He, F. (2011). <a href="https://bio-protocol.org/bio101/e52" target="_blank">Standard DNA cloning.</a> <em>Bio-101</em>: e52.</li><li>Im, H. (2011). <a href="https://bio-protocol.org/bio101/e143" target="_blank">The inoue method for preparation and transformation of competent <em>E. coli</em>: "Ultra competent" cells.</a> <em>Bio-101</em>: e143.</li><li>Karasawa, S., Araki, T., Nagai, T., Mizuno, H. and Miyawaki, A. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15065984" target="_blank">Cyan-emitting and orange-emitting fluorescent proteins as a donor/acceptor pair for fluorescence resonance energy transfer.</a> <em>Biochem J</em> 381(Pt 1): 307-312.</li><li>Meiresonne, N. Y., Consoli, E., Mertens, L. M. Y., Chertkova, A. O., Goedhart, J. and den Blaauwen, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30648295" target="_blank">Superfolder mTurquoise2(ox) optimized for the bacterial periplasm allows high efficiency <em>in vivo</em> FRET of cell division antibiotic targets.</a> <em>Mol Microbiol</em> 111(4): 1025-1038.</li><li>Meiresonne, N. Y., van der Ploeg, R., Hink, M. A. and den Blaauwen, T. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28900026" target="_blank">Activity-related conformational changes in d,d-Carboxypeptidases revealed by <em>in vivo</em> periplasmic forster resonance energy transfer assay in <em>Escherichia coli</em>.</a> <em>MBio</em> 8(5).</li><li>Shaner, N. C., Campbell, R. E., Steinbach, P. A., Giepmans, B. N., Palmer, A. E. and Tsien, R. Y. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15558047" target="_blank">Improved monomeric red, orange and yellow fluorescent proteins derived from Discosoma sp. red fluorescent protein.</a> <em>Nat Biotechnol</em> 22(12): 1567-1572.</li><li>Shaner, N. C., Lambert, G. G., Chammas, A., Ni, Y., Cranfill, P. J., Baird, M. A., Sell, B. R., Allen, J. R., Day, R. N., Israelsson, M., Davidson, M. W. and Wang, J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23524392" target="_blank">A bright monomeric green fluorescent protein derived from <em>Branchiostoma lanceolatum</em>.</a> <em>Nat Methods</em> 10(5): 407-409.</li><li>van der Ploeg, R., Goudelis, S. T. and den Blaauwen, T. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26263980" target="_blank">Validation of FRET assay for the screening of growth inhibitors of <em>Escherichia coli</em> reveals elongasome assembly dynamics.</a> <em>Int J Mol Sci</em> 16(8): 17637-17654.</li><li>van der Ploeg, R., Verheul, J., Vischer, N. O., Alexeeva, S., Hoogendoorn, E., Postma, M., Banzhaf, M., Vollmer, W. and den Blaauwen, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23387922" target="_blank">Colocalization and interaction between elongasome and divisome during a preparative cell division phase in <em>Escherichia coli</em>.</a> <em>Mol Microbiol</em> 87(5): 1074-1087.</li><li>Wlodarczyk, J., Woehler, A., Kobe, F., Ponimaskin, E., Zeug, A. and Neher, E. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17921223" target="_blank">Analysis of FRET signals in the presence of free donors and acceptors.</a> <em>Biophys J</em> 94(3): 986-1000.<br/></li></ol>