<body>
            <h1>Proximity Ligation Assay for the Investigation of the Intramolecular Interaction of ELMO1</h1>
            <p><strong>Authors:</strong> Wai Wa Ray Chan, Dik Long Dennis Chau, Wen  Li, Kwok-Fai  Lau</p>
            <h2>Abstract</h2>
            <p>Intramolecular interaction is a common mechanism that regulates protein activities. Conventionally, such interactions are investigated by classical <em>in vitro</em> biochemical assays. Here, we describe a protocol for studying the intramolecular interaction of cell motility and engulfment 1 (ELMO1) in mammalian cells by using proximity ligation assay (PLA). PLA is a specific and sensitive method that allows the observation of interacting proteins by target-specific antibody detection coupled to rolling circle amplification. ELMO1 is the regulatory subunit of ELMO1-dedicator of cytokinesis 180 (DOCK180) bipartite Rac1 guanine nucleotide exchange factor (GEF) which adopts a closed autoinhibitory conformation via an intramolecular interaction of its N-terminal ELMO inhibitory domain (EID) and C-terminal ELMO autoregulatory domain (EAD). In the assay, PLA signals are detected in cells transfected with ELMO1<sup>1-315</sup> and ELMO1<sup>315-727</sup> fragments. Moreover, overexpression of FE65, a neuronal adaptor which has been shown to disrupt ELMO1 intramolecular interaction, reduces the PLA signals of the two ELMO1 fragments significantly. Together, our results demonstrate that PLA can be employed for studying protein intramolecular interaction. </p>
            <p><strong>Keywords:</strong> Autoinhibition, Engulfment and cell motility 1, Fluorescence, Intramolecular interaction, PLA</p>
            <h2>Background</h2><p>The activities of proteins in cells can be regulated by various mechanisms including intramolecular interaction (Miernyk and Thelen, 2008). Several protein-protein interaction assays have been adopted for studying intramolecular interaction such as co-immunoprecipitation and protein complementation assays (PCAs). Despite being widely employed, there are known limitations of these conventional assays. For instance, co-immunoprecipitation is not ideal for detecting low-affinity interaction. Even though PCAs provide good sensitivity, the reporter fragments may hinder the interaction of interest (Ohad <em>et al.</em>, 2007; Miernyk and Thelen, 2008). Proximity ligation assay (PLA) is an innovative technology that allows <em>in situ</em> detection of protein-protein interactions. The protein targets are first labeled by their specific primary antibodies from different species, followed by the recognition of species-specific secondary antibodies conjugated to oligonucleotides (PLA probes). Proximity ligation of the oligonucleotides generates the templates for single-stranded rolling circle amplification. The resultant products are detected with fluorescence-labeled complementary oligonucleotide detection probes. PLA offers exceptional specificity and sensitivity as target-specific antibodies and single-stranded rolling circle amplification are involved. The resultant products are detected with fluorescence-labeled complementary oligonucleotide detection probes, and PLA signals can be visualized and quantified by using fluorescence microscopy (Weibrecht <em>et al.</em>, 2010). Although the technology has been available for over a decade, the possibility of using PLA for investigating protein intramolecular interaction has not been explored. <br/><br/>Engulfment and cell motility 1 (ELMO1) is the regulatory subunit of the bipartite ELMO1-dedicator of cytokinesis 180 (DOCK180) guanine nucleotide exchange factor (GEF) for the activation of the small GTPase Rac1. ELMO1 possesses several distinct domains: a Ras binding domain (RBD), ELMO inhibitory domain (EID), ELMO domain, pleckstrin homology (PH) domain, ELMO autoregulatory domain (EAD) and a PXXP motif (Figure 1). Similar to many other GEFs, the activity of ELMO1 is modulated by the intramolecular interaction between its EID and EAD which allows ELMO1 to adopt a closed autoinhibitory conformation at the basal state (Patel <em>et al.</em>, 2010) (Figure 1). The relief of ELMO1 autoinhibition is required for the targeting of the complex of ELMO1-DOCK180 to the plasma membrane where it stimulates Rac1 (Patel <em>et al.</em>, 2011). However, the mechanism that alleviates the autoinhibition of ELMO1 remains elusive. We have recently reported that the neuronal adaptor protein FE65 interacts with ELMO1 EAD and to disrupt the interaction of ELMO1<sup>1-315</sup> (comprises of ELMO1 amino acid residues 1-315) and ELMO1<sup>315-727</sup> (comprises of ELMO1 amino acid residues 315-727) fragments, which possess the EID and EAD respectively, by using both classical protein binding assay and PLA (Li <em>et al.</em>, 2018). Therefore, we demonstrate for the first time that PLA can be applied for examining protein intramolecular interaction.<br/><br/>Here, we describe how to investigate the intramolecular interaction of ELMO1 EID and EAD by using PLA. The method may also be adopted for the investigation of other protein intramolecular interactions.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203002128_2954.jpg" width="300"/><br/><strong>Figure 1. Schematic diagram shows the subdomains and the intramolecular interaction of ELMO1.</strong> At basal stage, the EAD and EID interact intramolecularly (indicated with double headed arrow) to form an autoinhibitory conformation. The disruption of the intramolecular interaction alleviates the autoinhibition for the subsequence Rac1 activation. Numbers denote the amino acid residue number in ELMO1. RBD: Ras-binding domain; EID: ELMO inhibitory domain; ELMO: ELMO domain; PH: Pleckstrin homology domain; EAD: ELMO autoregulatory domain; PXXP: PXXP motif.</p>
        </body>

<p class="pptt" id="biaoti30490">## Materials and Reagents</p><ol class="npwtwok"><li>Sterilin<sup>TM</sup> Standard 90 mm Petri Dish (Thermo Scientific, catalog number: 101VIRR), stored at room temperature</li><li>12-well cell culture plate (SPL, catalog number: 30012), stored at room temperature</li><li>Cover glass 18-mm circles (Thermo-Menzel, catalog number: CB00180RA1), stored at room temperature</li><li>Hemocytometer (HBG, catalog number: 9010-01), stored at room temperature</li><li>Microscope slides (Thermo Scientific, catalog number: 6776214), stored at room temperature</li><li>HEK293 cells (ATCC, catalog number: CRL-1573), stored in liquid nitrogen</li><li>Goat anti-ELMO1 antibody (Santa Cruz Biotechnology, catalog number: sc-21651), stored at 4 Â°C</li><li>Mouse anti-FLAG antibody (Sigma-Aldrich, catalog number: F1804), stored at -20 Â°C</li><li>Rabbit anti-Î²-tubulin antibody (Abcam, catalog number: AB6046)</li><li>Donkey anti-rabbit IgG (H + L) highly cross-adsorbed secondary antibody, Alexa Fluor 488 (Invitrogen, catalog number: A21206), stored at 4 Â°C</li><li>Fetal bovine serum (HyClone, catalog number: SV30160.03), stored at -20 Â°C</li><li>pCMV-Tag2B vector (Aligent, catalog number: 211172), stored at -20 Â°C</li><li>T4 DNA Ligase (1 U/Î¼l) (Thermo Scientific, catalog number: 15224025), stored at -20 Â°C</li><li>BamHI (10 U/Î¼l) (Thermo Scientific, catalog number: ER0055), stored at -20 Â°C</li><li>XhoI (10 U/Î¼l) (Thermo Scientific, catalog number: ER0691), stored at -20 Â°C</li><li>TIANprep Mini Plasmid Kit (TIANGEN, catalog number: DP103), stored at room temperature</li><li>QuikChange II Site-Directed Mutagenesis Kit (Agilent, catalog number: 200523), stored at -20 Â°C</li><li>Phusion High-Fidelity PCR Kit (Thermo Scientific, catalog number: F553S), stored at -20 Â°C</li><li>UltraPure<sup>TM</sup> Agarose (Invitrogen, catalog number: 16500500), stored at room temperature</li><li>TAE Buffer (Tris-acetate-EDTA) (50x) (Thermo Scientific, catalog number: B49), stored at room temperature</li><li>UltraPure<sup>TM</sup> Ethidium Bromide, 10 mg/ml (Invitrogen, catalog number: 15585011), stored at room temperature</li><li>1 kb Plus DNA Ladder (New England Biolabs, catalog number: N3200), stored at -20 Â°C</li><li>Luria Broth Base, powder (Invitrogen, catalog number: 12795084), stored at room temperature</li><li>Clear nail polish</li><li>LB Agar, powder (Invitrogen, catalog number: 22700041), stored at room temperature</li><li>Ampicillin sodium salt (Sigma-Aldrich, catalog number: A0166), stored at 4 Â°C</li><li>Kanamycin sulfate (Sigma-Aldrich, catalog number: 60615), stored at 4 Â°C</li><li>Poly-D-lysine hydrobromide (Sigma-Aldrich, catalog number: P7280), stored at -20 Â°C</li><li>DMEM (HyClone, catalog number: SH30021.01), stored at 4 Â°C</li><li>Trypsin-EDTA 0.05% (Gibco, catalog number: 25300062), stored at -20 Â°C</li><li>Opti-MEM reduced serum medium (Gibco, catalog number: 31985088), stored at 4 Â°C</li><li>X-tremeGENE<sup>TM</sup> 9 DNA transfection reagent (Sigma-Aldrich, catalog number: 6365779001), stored at 4 Â°C</li><li>PBS (Gibco, catalog number: 10010023), stored at room temperature</li><li>Paraformaldehyde (Sigma-Aldrich, catalog number: P6148), stored at 4 Â°C</li><li>Triton X-100 (Anatrace, catalog number: T1001500ML), stored at room temperature</li><li>Duolink<sup>Â®</sup> In Situ Red Starter Kit Mouse/Goat (Sigma-Aldrich, catalog number: DUO92103), stored at -20 Â°C</li><li>1x TAE buffer (see Recipes)</li><li>Poly-D-lysine solution (see Recipes)</li><li>4% paraformaldehyde (freshly prepared) (see Recipes)</li><li>1x Ligation buffer (see Recipes)</li><li>1x Amplification red (see Recipes)</li><li>Wash buffer A (see Recipes)</li><li>Wash buffer B (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30491">### Equipment</p><ol class="npwtwok"><li>Water bath (Labnet, model: W1106)</li><li>Heat block (Labnet, model: D1301-230V)</li><li>UV transilluminator (Accuris, model: E3000)</li><li>T100<sup>TM</sup> Thermal Cycler (Bio-rad) </li><li>Heratherm<sup>TM</sup> Compact Microbiological Incubators (37 Â°C, Thermo Scientific)</li><li>Forma<sup>TM</sup> Series II 3110 Water-Jacketed CO<sub>2</sub> Incubators (37 Â°C, 5% CO<sub>2</sub>, Thermo Scientific)</li><li>Fluorescence microscope (Nikon, model: Eclipse Ni-U) equipped with QI2 high-resolution microscope camera (Nikon)<br/></li></ol><p class="pptt" id="biaoti30492">### Software</p><ol class="npwtwok"><li>NIS-Elements research basic (Nikon)</li><li>Prism (<a href="https://www.graphpad.com/scientific-software/prism/" target="_blank">https://www.graphpad.com/scientific-software/prism/</a>)<br/></li></ol><p class="pptt" id="biaoti30493">### Procedure</p><ol class="npwone"><li>Generation of mammalian expression construct of ELMO1<sup>1-315</sup> <ol class="npwtwo"><li>Perform site-directed mutagenesis to introduce a stop codon after ELMO1 residue 315 in pcDNA3-N-Myc-Elmo1 by using QuikChange II Site-Directed Mutagenesis Kit (Figure 2). The mutagenesis primers used are listed in Table 1.<br/><br/><img alt="" class="layerphoto" height="97" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203004109_0023.jpg" width="300"/><br/><strong>Figure 2. Schematic diagram of ELMO1 fragment constructs.</strong> Site-directed mutagenesis is performed to create a stop codon after ELMO1 residue 315 in pcDNA3-N-Myc-Elmo1 to generate ELMO1<sup>1-315</sup> fragment. cDNA of ELMO1<sup>315-727</sup> is amplified from pcDNA3-N-Myc-Elmo1 by using PCR and the product is inserted to pCMV-Tag2B for the expression of N-terminal FLAG tagged protein. The asterisk indicates the mutation site. Numbers denote the amino acid residue number in ELMO1.<br/><br/><strong>Table 1. Primers used for generating ELMO1 fragment constructs.</strong> Primers 1 and 2 are used for the site-directed mutagenesis to introduce a stop codon to the pcDNA3-N-Myc-Elmo1. Primers 3 and 4 are used for the amplification of ELMO1<sup>315-727</sup> fragment by using pcDNA3-N-Myc-Elmo1 as a template.<br/><img alt="" class="layerphoto" height="142" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203004129_1360.png" width="600"/><br/><br/></li><li>Transform the mutagenesis product to DH5Î± competent <em>E. coli</em> cells by heat shock for 1 min at 42 Â°C.</li><li>Plate the transformants on LB agar plates with 100 Î¼g/ml ampicillin. </li><li>Incubate the plates overnight in 37 Â°C incubator.</li><li>Inoculate isolated colonies in 5 ml LB with 100 Î¼g/ml ampicillin and grow overnight at 37 Â°C.</li><li>Isolate the plasmid DNA by using TIANprep Mini Plasmid Kit.</li><li>Perform DNA sequencing to confirm the introduction of the stop codon.<br/><br/></li></ol></li><li>Generation of mammalian expression construct of FLAG-ELMO1<sup>315-727</sup> <ol class="npwtwo"><li>Perform polymerase chain reaction (PCR) to obtain cDNA encoded for ELMO1<sup>315-727</sup> fragment by using Phusion High-Fidelity PCR Kit with pcDNA3-N-Myc-Elmo1 as a template. Primers used are listed in Table 1.</li><li>Resolve the PCR product by a 1% agarose gel with 0.2 Î¼g/ml ethidium bromide in 1x TAE buffer. </li><li>Visualize the PCR product by using a UV transilluminator and excise the PCR product (~1.2 kb) with a razor blade after electrophoresis.</li><li>Purify the PCR product from gel by using MEGAquick-spin<sup>TM</sup> Plus Total Fragment DNA Purification Kit.</li><li>Digest the purified ELMO1<sup>315-727</sup> PCR product and pCMV-Tag2B, a vector with an N-terminal FLAG tag, with BamHI and XhoI for 2 h in a 37 Â°C water bath.</li><li>Purify the digestion products by using MEGAquick-spin<sup>TM</sup> Plus Total Fragment DNA Purification Kit.</li><li>Ligate the digested ELMO1<sup>315-727</sup> fragment and linearized pCMV-Tag2B vector with T4 DNA ligase overnight in a 16 Â°C water bath.</li><li>Transform the ligation product as stated in Steps A2-A6. 50 Î¼g/ml Kanamycin should be used for selection.</li><li>Perform DNA sequencing to confirm the successful insertion of the ELMO1<sup>315-727</sup> encoding cDNA into pCMV-Tag2B.<br/><br/></li></ol></li><li>Cell preparation and transfection<ol class="npwtwo"><li>Coat 18-mm cover glasses with 5 Î¼g/ml poly-D-lysine solution in 12-well cell culture plate overnight in a 37 Â°C incubator.<br/><em>Note: Sterilize the cover glass before use.</em> </li><li>Wash the cover glasses once with distilled water.</li><li>Seed HEK293 cells on the pre-coated 18-mm cover glasses at 0.4 x 10<sup>5</sup> cells in 1 ml DMEM medium supplemented with 10% FBS.</li><li>After 24 h, transfect the cells with mammalian expression constructs of myc-ELMO1<sup>1-315</sup> and FLAG-ELMO1<sup>315-727</sup> together with either empty vector (EV), FE65 or FE65 K48A/R51A (FE65m, an ELMO1 binding defective FE65 mutant) using X-tremeGENE<sup>TM</sup> 9 DNA transfection reagent in Opti-MEM reduced serum medium according to the manufacturerâs instruction (Table 2).<br/><br/><strong>Table 2 Transfection combinations for studying the effect of FE65 in ELMO1 intramolecular interaction.</strong> HEK293 cells are co-transfected with either EV, FE65 or FE65m together with ELMO1<sup>1-315</sup> and ELMO1<sup>315-727</sup> constructs in 1:1:1 ratio. All transfections receive the same amount of DNA. â+â denotes the presence of the plasmid while â-â denotes the absence of the plasmid.<br/><img alt="" class="layerphoto" height="112" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203004216_4993.png" width="620"/><br/><br/></li></ol></li><li>Cell fixing and proximity ligation assay<br/><em>Note: Wash Buffer A, Wash Buffer B, PLA probe anti-goat MINUS, PLA probe anti-mouse PLUS, Antibody Diluent, Ligase, 5x Ligation Buffer, Polymerase, 5x Amplification Red and Duolink In Situ Mounting Media with DAPI are provided within Duolink<sup>Â®</sup> In Situ Red Starter Kit Mouse/Goat. Please see Figure 3 for the workflow of PLA.</em><br/><br/><img alt="" class="layerphoto" height="143" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203004245_6320.jpg" width="300"/><br/><strong>Figure 3. Timeline illustrates the workflow of PLA of ELMO1 fragments using Duolink<sup>Â®</sup> <em>In Situ</em> Red Starter Kit Mouse/Goat</strong><br/><br/><ol class="npwtwo"><li>After 24 h transfection, wash the cells once with PBS and fix the cells with 4% paraformaldehyde for 10 min at room temperature.</li><li>Wash the fixed cells three times with PBS.</li><li>Permeabilize the cells with 0.1% Triton X-100/PBS for 15 min at room temperature.</li><li>Block the cells with 5% FBS/PBS for 30 min at room temperature.</li><li>Dilute the goat anti-ELMO1 (recognizes myc-ELMO1<sup>1-315</sup>) (1:100), mouse anti-FLAG (recognizes FLAG-ELMO1<sup>315-727</sup>) (1:1000) and rabbit anti-Î²-tubulin (1:200) antibodies in 5% FBS/PBS and stain the cells with 80 Î¼l of diluted antibody mix for 1 h at room temperature.<br/><em>Note: The concentrations of primary antibodies used are critical for obtaining high-quality PLA images and should be optimized. </em> </li><li>Wash the cells five times with PBS.</li><li>Dilute the Alexa Fluor 488 conjugated Donkey anti-rabbit IgG (1:1000) antibody in 5% FBS/PBS and stain the cells for 1 h in the dark at room temperature.</li><li>Wash the cells five times with PBS.</li><li>Wash the cells twice with Wash Buffer A for 5 min in the dark at room temperature.</li><li>Add 80 Î¼l PLA probe reaction mix to each cover glass and incubate in a humidified 37 Â°C incubator for 1 h.<br/><em>Note: For each reaction, mix 16 Î¼l of PLA probe anti-goat MINUS (Donkey anti-goat secondary antibody conjugated to oligonucleotide MINUS) with 16 Î¼l of PLA probe anti-mouse PLUS (Donkey anti-mouse secondary antibody conjugated to oligonucleotide PLUS) in 48 Î¼l of Antibody Diluent and incubate for 20 min at room temperature before use.</em> </li><li>Wash the cover glasses twice with Wash Buffer A for 5 min in the dark at room temperature.</li><li>Add 80 Î¼l Ligation reaction mix to each cover glass and incubate in a humidified 37 Â°C incubator for 30 min.<br/><em>Note: For each reaction, add 2 Î¼l Ligase to 78 Î¼l 1x Ligation buffer.</em> </li><li>Wash the cover glasses twice with Wash Buffer A for 5 min in the dark at room temperature.</li><li>Add 80 Î¼l Amplification reaction mix to each cover glass and incubate in humidified 37 Â°C incubator for 100 min.<br/><em>Note: For each reaction, add 1 Î¼l Polymerase to 79 Î¼l 1x Amplification Red.</em> </li><li>Wash the cover glasses twice with Wash Buffer B for 10 min in the dark at room temperature.</li><li>Wash the cover glasses once with 0.01x Wash Buffer B for 1 min in the dark at room temperature.</li><li>Mount the cover glasses on microscope slides with a minimal amount of Duolink <em>In Situ</em> Mounting Media with DAPI.</li><li>Seal the edge of the mounted cover glasses with clear nail polish.</li><li>Capture the cell images with a 60x water immersion lens using a Nikon fluorescent microscope. <br/><em>Note: Store the slides in the dark at 4 Â°C if necessary.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti30494">### Data analysis</p><p class="ppzz">Microscope slides are imaged by a Nikon fluorescent microscope with a 60x water immersion lens. FITC and TRITC filter sets are used to visualize Î²-tubulin and PLA signals, respectively. The number of discrete fluorescent PLA spots inside the cells is counted. The average numbers of PLA signals per cell in different transfections are compared to investigate the change of interaction between the two fragments. The PLA signals from at least 60 cells are counted, and three independent experiments are performed. Statistical analysis is performed by using one-way analysis of variance (ANOVA) with Bonferroni post-hoc test.<br/><br/><strong><em>Representative data</em></strong><br/>Recently, we demonstrated that the interaction of ELMO1<sup>1-315</sup> and ELMO1<sup>315-727</sup>, which contain its EID and EAD, respectively, in cells by PLA. Moreover, the number of PLA signals reduced significantly in cells co-transfected with the wildtype FE65 but not the binding defective mutant (<em>i.e.</em>, FE65m) (Figure 4). These findings from the PLAs are in line with the result from the classical protein binding assay (Li <em>et al.</em>, 2018).<br/><br/><img alt="" class="layerphoto" height="191" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191203/20191203004310_6158.jpg" width="300"/><br/><strong>Figure 4. Representative images and quantification of PLA signals.</strong> HEK293 cells were transfected with myc-ELMO1<sup>1-315</sup> and FLAG-ELMO1<sup>315-727</sup> together with either empty vector (EV), FE65 or FE65m. After 24 h transfection, the cells were fixed and subjected to PLA. The number of PLA signals from at least 60 cells was counted in each transfection and three independent experiments were performed. The bar chart shows the relative PLA signal (fold change to EV) in different transfections. *<em>P</em> &lt; 0.001. Error bars are SEM. Significant reduction in PLA signals was observed only in FE65-transfected cells but not in EV- or FE65m-transfected cells. Î²-tubulin and nucleus were stained to serve as a morphological and nuclear marker, respectively. Scale bar = 10 Î¼m. (This research was originally published in the Journal of Biological Chemistry. Li W, Tam KMV, Chan WWR, Koon AC, Ngo JCK, Chan HYE and Lau KF. Neuronal adaptor FE65 stimulates Rac1-mediated neurite outgrowth by recruiting and activating ELMO1. J. Biol. Chem. 2018; 293(20): 7674-7688. © the American Society for Biochemistry and Molecular Biology)</p><p class="pptt" id="biaoti30495">### Recipes</p><ol class="npwtwok"><li>1x TAE buffer<br/>Dilute the 50x stock solution to 1x with distilled water</li><li>Poly-D-lysine solution<br/>5 mg poly-D-lysine<br/>5 ml distilled water<br/>Dissolve 5 mg poly-D-lysine in 5 ml distilled water to obtain a stock solution (200x)<br/>Store the stock solution in aliquots at -20 Â°C</li><li>4% paraformaldehyde (freshly prepared)<br/>0.4 g paraformaldehyde powder<br/>10 ml PBS<br/><ol class="npwthree"><li>Add 0.4 g paraformaldehyde powder to 10 ml PBS</li><li>Incubate in a 70 Â°C water bath until the powder dissolved</li><li>Cooldown the solution at room temperature</li><li>Filter the solution with a 0.22 Î¼m syringe filter</li></ol></li><li>1x Ligation buffer<br/>For each reaction, dilute 16 Î¼l 5x Ligation buffer with 64 Î¼l distilled water</li><li>1x Amplification red<br/>For each reaction, dilute 16 Î¼l 5x Amplification Red with 64 Î¼l distilled water</li><li>Wash buffer A<br/>0.01 M Tris <br/>0.15 M NaCl <br/>0.05% Tween 20<br/>pH 7.4</li><li>Wash buffer B<br/>0.2 M Tris <br/>0.1 M NaCl<br/>pH 7.5<br/></li></ol><p class="pptt" id="biaoti30496">### Acknowledgments</p><p class="ppzz">This work was supported by funds from the Research Grants Council Hong Kong, Health and Medical Research Fund (Hong Kong), CUHK direct grant scheme and the United College endowment fund. This protocol was adapted from previous work. We thank Prof Jean-FranÃ§ois CÃ´tÃ© (Montreal Clinical Research Institute) for the mammalian expression construct of pcDNA3-N-Myc-Elmo1.</p><p class="pptt" id="biaoti30497">### Competing interests</p><p class="ppzz">The authors declare that they have no conflicts of interest with the contents of this article.</p><p class="pptt" id="biaoti30498">### References</p><ol class="npwtwok"><li>Li, W., Tam, K. M. V., Chan, W. W. R., Koon, A. C., Ngo, J. C. K., Chan, H. Y. E. and Lau, K. F. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29615491" target="_blank">Neuronal adaptor FE65 stimulates Rac1-mediated neurite outgrowth by recruiting and activating ELMO1.</a> <em>J Biol Chem</em> 293(20): 7674-7688. </li><li>Miernyk, J. A. and Thelen, J. J. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18269571" target="_blank">Biochemical approaches for discovering protein-protein interactions.</a> <em>Plant J</em> 53(4): 597-609. </li><li>Ohad, N., Shichrur, K. and Yalovsky, S. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18056859" target="_blank">The analysis of protein-protein interactions in plants by bimolecular fluorescence complementation.</a> <em>Plant Physiol</em> 145(4): 1090-1099. </li><li>Patel, M., Margaron, Y., Fradet, N., Yang, Q., Wilkes, B., Bouvier, M., Hofmann, K. and Cote, J. F. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21035343" target="_blank">An evolutionarily conserved autoinhibitory molecular switch in ELMO proteins regulates Rac signaling.</a> <em>Curr Biol</em> 20(22): 2021-2027. </li><li>Patel, M., Pelletier, A. and Cote, J. F. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22292130" target="_blank">Opening up on ELMO regulation: New insights into the control of Rac signaling by the DOCK180/ELMO complex.</a> <em>Small GTPases</em> 2(5): 268-275. </li><li>Weibrecht, I., Leuchowius, K. J., Clausson, C. M., Conze, T., Jarvius, M., Howell, W. M., Kamali-Moghaddam, M. and Soderberg, O. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20536310" target="_blank">Proximity ligation assays: a recent addition to the proteomics toolbox.</a> <em>Expert Rev Proteomics</em> 7(3): 401-409. <br/></li></ol>