<body>
            <h1>Enhanced-<em>ice</em>-COLD-PCR for the Sensitive Detection of Rare DNA Methylation Patterns in Liquid Biopsies</h1>
            <p><strong>Authors:</strong> Florence Mauger, JÃ¶rg Tost</p>
            <h2>Abstract</h2>
            <p>In the context of precision medicine, the identification of novel biomarkers for the diagnosis of disease, prognosis, predicting treatment outcome and monitoring of treatment success is of great importance. The analysis of methylated circulating-cell free DNA provides great promise to complement or replace genetic markers for these applications, but is associated with substantial challenges. This is particularly true for the detection of rare methylated DNA molecules in a limited amount of sample such as tumor released hypermethylated molecules in the background of DNA fragments from normal cells, especially lymphocytes.<br/><br/>Technologies for the sensitive detection of DNA methylation have been developed to enrich specifically methylated DNA or unmethylated DNA using among other methods: enzymatic digestion, methylation-specific PCR (often combined with TaqMan like oligonucleotide probes (MethyLight)) and co-amplification at lower denaturation temperature PCR (COLD-PCR). <br/><br/>E-<em>ice</em>-COLD-PCR (Enhanced-improved and complete enrichment-COLD-PCR) is a sensitive method that takes advantage of a Locked Nucleic Acid (LNA)-containing oligonucleotide probe to block specifically unmethylated CpG sites allowing the strong enrichment of low-abundant methylated CpG sites from a limited quantity of input. E-<em>ice</em>-COLD-PCRs are performed on bisulfite-converted DNA followed by Pyrosequencing analysis. The quantification of the initially present DNA methylation level is obtained using calibration curves of methylated and unmethylated DNA. The E-<em>ice</em>-COLD-PCR reactions can be multiplexed, allowing the analysis and quantification of the DNA methylation level of several target genes. In contrast to the above-mentioned assays, E-<em>ice</em>-COLD-PCR will also perform in the presence of frequently occurring heterogeneous DNA methylation patterns at the target sites. The presented protocol describes the development of an E-<em>ice</em>-COLD-PCR assay including assay design, optimization of E-<em>ice</em>-COLD-PCR conditions including annealing temperature, critical temperature and concentration of LNA blocker probe followed by Pyrosequencing analysis.</p>
            <p><strong>Keywords:</strong> DNA methylation, Enrichment, Bisulfite conversion, E-<em>ice</em>-COLD-PCR, LNA, Blocker probe, Pyrosequencing, Circulating cell-free DNA, Liquid biopsy</p>
            <h2>Background</h2><p>In the context of precision medicine, the discovery of novel non-invasive biomarkers in clinical samples such as circulating cell-free DNA (ccfDNA) is crucial for improving the diagnosis, prognosis, predicting and monitoring the response to a patient-tailored treatment regimen against cancer and other diseases (Siravegna <em>et al.</em>, 2017, Wan <em>et al.</em>, 2017; Cabel <em>et al.</em>, 2018; Barlebo Ahlborn and Ãstrup, 2019). The analysis of ccfDNA is challenging because the outcome depends on multiple pre-analytical steps and requires the development of methods for the detection of genetic and epigenetic variations that are present in limited quantity in highly fragmented samples (Lewis <em>et al.</em>, 2015; Kumar <em>et al.</em>, 2018).<br/><br/>Especially, methylated ccfDNA could be a promising non-invasive biomarker for cancer and potentially other diseases (Lehmann-Werman <em>et al.</em>, 2016; Warton <em>et al.</em>, 2016). In comparison to mutations and other genetic changes, DNA methylation changes occur early during carcinogenesis and are also present in a number of other complex non-neoplastic human diseases. Furthermore, DNA methylation changes are often restricted to a genomic region of limited size, for example the promoter associated CpG island or the transcription start site in contrast to genetic mutations that might be present all along the gene. The most commonly used methods for the analysis of DNA methylation patterns are based on a bisulfite reaction that converts cytosine bases into uracil bases while 5-methylcytosine bases are not converted (Frommer <em>et al.</em>, 1992). Bisulfite conversion thus translates a DNA methylation difference into a sequence change. DNA methylation patterns are then read-out using next generation sequencing or microarrays for genome-wide analysis or Pyrosequencing, methylation-sensitive or methylation-specific PCR (MSP) methods for locus-specific analysis (Tost, 2016). <br/><br/>Different DNA methylation enrichment approaches for the analysis of low DNA methylation levels have been developed based on digestion or oligonucleotide probes (Liu <em>et al.</em>, 2017; Campan <em>et al.</em>, 2018; Distler, 2019). For example, the HeavyMethyl method which is based on enrichment of methylated DNA by blocker probes competing with the amplification primers on the target site, is used for a commercially available non-invasive blood based test (MolnÃ¡r <em>et al.</em>, 2015; Jung <em>et al.</em>, 2018).<br/><br/>While MSP and MethyLight will specifically amplify methylated molecules in the presence of a 1,000 to 10,000 fold excess of unmethylated molecules, the efficient amplification relies on a pre-defined consistent methylation profile underlying the bindings sites for the amplification primers and the hydrolysis probe (for MethyLight), usually completely methylated molecules (Candiloro <em>et al.</em>, 2011). However, in the presence of heterogeneous DNA methylation patterns, the assays fail to amplify partially methylated molecules and significantly underestimate the proportion of methylated samples (Alnaes <em>et al.</em>, 2015). <br/><br/>Co-amplification at lower denaturation temperature PCR (COLD-PCR) approaches have been developed to enrich and analyze rare mutations including <em>full</em>, <em>fast</em>, <em>ice</em> (improved and complete enrichment), E-<em>ice</em> (Enhanced-<em>ice</em>) and temperature tolerant COLD-PCR and have been combined with several read-out technologies (Mauger <em>et al.</em>, 2017). These protocols use a critical temperature (Tc) to selectively denature wild-type mutant heteroduplexes to allow the enrichment of rare mutations. More recently, <em>fast</em>-COLD-MS-PCR and E-<em>ice</em>-COLD-PCR, have been shown to allow the enrichment of unmethylated DNA and methylated DNA, respectively, from bisulfite-converted DNA (Castellanos-Rizaldos <em>et al.</em>, 2014; Mauger <em>et al.</em>, 2018). <br/><br/>E-<em>ice</em>-COLD-PCR uses a blocker probe that contains several LNA bases which selectively hybridizes to and thereby blocks the amplification of wild-type alleles and enriches mutant alleles followed by a Pyrosequencing assay for a sequence based read-out at single nucleotide resolution (How-Kit and Tost, 2015). For DNA methylation analysis, LNA blocker probes are designed to block unmethylated CpG sites on bisulfite-converted DNA enriching thus all other DNA methylation patterns except for completely unmethylated molecules (Mauger <em>et al.</em>, 2018).<br/><br/>Therefore, the design of E-<em>ice</em>-COLD-PCR assays is fundamentally different from the more widely used MSP assays as it impedes the amplification of the normal, unmethylated state, but does not make any requirements on the degree or the patterns of DNA methylation in the amplified target region. All molecules different from the blocked pattern will be amplified and the subsequent sequencing-based analysis gives detailed information on the molecules that have been enriched. E-<em>ice</em>-COLD-PCR reaction can be easily multiplexed and the level of mutations or DNA methylation can be quantified using standard curves for the analysis of very low input material such as ccfDNA (Mauger <em>et al.</em>, 2016 and 2018; Sefrioui <em>et al.</em>, 2017).<br/><br/>The current protocol describes the development and implementation of an E-<em>ice</em>-COLD-PCR assay for the enrichment of low-abundant methylated DNA from a limited quantity of input sample and thus applicable to non-invasive blood-based tests. After bisulfite conversion, the E-<em>ice</em>-COLD-PCR reaction is performed followed by Pyrosequencing analysis. The optimization of E-<em>ice</em>-COLD-PCR includes assay design, optimization of the reaction and analysis of the amplification product.<br/><br/>While the presented protocol focusses on the technically more demanding detection of hypermethylation, blocker probes could also be used to block methylated DNA and enable the enrichment of unmethylated DNA in the presence of an excess of methylated DNA.</p>
        </body>

<p class="pptt" id="biaoti30456">## Materials and Reagents</p><ol class="npwtwok"><li>DNA LoBind and PCR Tubes 2 ml (Eppendorf, catalog number: 022431048) or similar</li><li>LightCycler<sup>Â®</sup> 480 Mutiwell Plate 96, white (Roche, catalog number: 04729692001)</li><li>LightCycler<sup>Â®</sup> 480 Sealing Foil (Roche, catalog number: 04729757001)</li><li>Pyrosequencing plate (PyroMark<sup>Â®</sup> Gold Q96 Plate, Qiagen, catalog number: 979101)</li><li>PyroMark<sup>Â®</sup> Q96 HS Capillary Tips (Qiagen, catalog number: 979104)</li><li>PyroMark<sup>Â®</sup> Q96 HS Reagents Tips (Qiagen, catalog number: 979102)</li><li>PCR Plate 96-well (Thermo Fisher, catalog number: AB0800) or similar</li><li>Adhesive PCR Plate (Thermo Fisher, catalog number: AB0558) or similar</li><li>Polyethylene adhesive PCR Plate (Corning, catalog number: 6524) or similar</li><li>Aluminium Adhesive PCR Plate (Corning, catalog number: 6569) or similar</li><li>10/20 Âµl filter tips (Rainin, catalog number: 17002429) or similar </li><li>200/250 Âµl filter tips (Rainin, catalog number: 17002428) or similar</li><li>1,000 Âµl filter tips (Rainin, catalog number: 17002426) or similar</li><li>10 Âµl tips (Rainin, catalog number: 17005091) or similar</li><li>250 Âµl tips (Rainin, catalog number: 17005093) or similar</li><li>1,000 Âµl tips (Rainin, catalog number:17005089) or similar</li><li>EpiTect control DNA methylated (Qiagen, catalog number: 59655)</li><li>Unmethylated and methylated DNA standards (Zymo Research, catalog number: D5014) </li><li>Qubit dsDNA HS Assay (Thermo Fisher, catalog number: Q33226)</li><li>Epitect Fast DNA bisulfite kit (Qiagen, catalog number:59826)</li><li>EZ DNA Methylation-Gold kit (Zymo Research, catalog number: D5005) or similar</li><li>Primers, blocker probes and Pyrosequencing primers, HPLC-purified grade (TIBMOLBIOL)</li><li>Nuclease Free water (Invitrogen, catalog number: AM9937)</li><li>HotStar Taq Polymerase (provided with the respective HotStar Taq buffer, Qiagen, catalog number: 203203)</li><li>UltraPure<sup>TM</sup> Agarose (Thermo Fisher, catalog number 16500500) </li><li>dNTP Mix (Thermo Fisher, catalog number: R0192)</li><li>SYTO<sup>TM</sup> 9 Green Fluorescent Nucleic Acid Stain (Thermo Fisher, catalog number: S34854)</li><li>PyroMark<sup>Â®</sup> Gold SQA Q96 Reagents (Qiagen, catalog number: 972812)</li><li>Streptavidin Sepharose beads (GE Healthcare, catalog number: 17-5113-01)</li><li>Denaturing Buffer (0.2 M NaOH)</li><li>70% Ethanol</li><li>MgCl<sub>2</sub> </li><li>Tris-HCl</li><li>Tris base</li><li>NaCl</li><li>EDTA</li><li>Tween-20</li><li>Acetic acid</li><li>Binding Buffer (see Recipes)</li><li>Annealing Buffer (see Recipes)</li><li>Wash Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30457">### Equipment</p><ol class="npwtwok"><li>A pre-PCR room with Laminar flow cabinets and a post-PCR room</li><li>20, 200 and 1,000 Âµl and multichannel pipettes (Rainin) or similar and filter tips in pre-PCR room</li><li>Vortex for tubes</li><li>Centrifuge for plates and microtubes (Eppendorf) or similar</li><li>Refrigerant block for 96 plates and ice bucket</li><li>Qubit Fluorometer (Thermo Fisher) or similar</li><li>QIAcube (Qiagen) or similar</li><li>LightCycler<sup>Â®</sup> 480 Instrument (Roche LifeScience) or similar </li><li>LightCycler<sup>Â®</sup> 96 Instrument (Roche LifeScience) or similar</li><li>Thermocycler (Eppendorf) or similar</li><li>Thermomixer (Eppendorf) or similar</li><li>PyroMark Q96 Vacuum Workstation (Qiagen, catalog number: 9001529)</li><li>Pyrosequencer (PyroMark Q96 MD System, Qiagen) or similar<br/></li></ol><p class="pptt" id="biaoti30458">### Software</p><ol class="npwtwok"><li>Design of PCR primers using MethPrimer (<a href="http://www.urogene.org/methprimer/" target="_blank">http://www.urogene.org/methprimer/</a>) (Li and Dahiya, 2002)</li><li>Nucleic Acid Sequence Massager (<a href="http://www.attotron.com/cybertory/analysis/ seqMassager.htm" target="_blank">http://www.attotron.com/cybertory/analysis/ seqMassager.htm</a>)</li><li>Design of Pyrosequencing primers can be performed manually or using the commercial PyroMark assay design software (Qiagen)</li><li>Manual design of LNA blocker probe and calculation of Tc using the LNA oligo Tm prediction tool (<a href="http://www.exiqon.com/ls/homeoflna/Oligo-tools/tm-prediction-tool.htm" target="_blank">http://www.exiqon.com/ls/homeoflna/Oligo-tools/tm-prediction-tool.htm</a>)</li><li>LightCycler 480 Software (Roche LifeScience) or similar</li><li>Gradient LightCycler 96 Software (Roche LifeScience) or similar</li><li>PyroMark<sup>Â®</sup> CpG Software (Qiagen) or similar</li><li>Statistical or graphical software such as Excel<sup>Â®</sup> or similar <br/></li></ol><p class="pptt" id="biaoti30459">### Procedure</p><p class="ppzz">The development of an E-<em>ice</em>-COLD-PCR assay includes the design of the assay, the optimization of the E-<em>ice</em>-COLD-PCR reaction, the analysis of the samples of interest as well as the data analysis for the E-<em>ice</em>-COLD-PCR assay (Figure 1).</p><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191201/20191201191337_0179.jpg" width="450"/><br/><strong>Figure 1. Overview of the development of an E-<em>ice</em>-COLD-PCR assay composed of three main steps.</strong> 1) Assay design on bisulfite converted DNA, 2) optimization of the E-<em>ice</em>-COLD-PCR conditions and 3) analysis of E-<em>ice</em>-COLD-PCR to quantify the presence and level of DNA methylation in the samples of interest.</p><br/><ol class="npwone"><li>Assay design<br/>The assay design including the design of PCR and Pyrosequencing primers for the targeted methylated sequence as well as the design of the LNA blocker probe for the optimal enrichment of methylated CpGs, which strongly depends on the frequency and spacing of CpG sites present in the targeted region, is probably the most critical step of the presented protocol. <br/><strong><em>Design of PCR primers</em></strong> <ol class="npwtwo"><li>We recommend designing at least 2 PCR primer sets. </li><li>Design PCR primers for the targeted region on <em>in-silico</em> bisulfite converted DNA using the MethPrimer software. Optimal PCR primers size should be around 20 bases with a melting temperature (Tm) around 60 Â°C. The size of the amplification product for the E-<em>ice</em>-COLD-PCR amplicons should be kept relatively small (100-180 base pairs maximum). </li><li>Place PCR primers in a region containing four or more cytosines that have been converted to ensure complementary to bisulfite-converted DNA. They should preferentially not contain CpG sites or limited to one CpG site. If CpG sites cannot be completely avoided, they should not be present in the last five bases of the 3' end to avoid preferential amplification of a specific DNA methylation pattern.</li><li>PCR primers should not contain palindromes, complementary sequences, degenerated bases or inosine. </li><li>Check for the presence of single nucleotide polymorphisms (SNPs) within the amplification product using sequence alignment tools such as BLAST or BLAT and associated dbSNP resources. Genetic variants in the sequences targeted by the amplification primers that have a significant frequency in the population of interest must be avoided and amplification primers should be redesigned.</li><li>If no primers are found on the forward strand, create the reverse complement strand using Nucleic Acid Sequence Massager and repeat the design process.</li><li>One PCR primer needs to be biotinylated on the 5' end for the Pyrosequencing reaction later on and the primer to be biotinylated depends on the sense of the Pyrosequencing primer designed in the next step.</li></ol><br/><em><strong>Design of Pyrosequencing primers</strong></em> <ol class="npwtwo"><li>Pyrosequencing primers can be designed manually or using the commercial PyroMark Assay Design Software. We recommend designing at least 2 Pyrosequencing primers to ensure optimal assay performance. </li><li>Pyrosequencing primers should not overlap with potentially variable positions and must not produce primer dimers or a hairpin structure. </li><li>Position the Pyrosequencing primer at least five bases from the end of the amplification product. The last consecutive 4-5 bases at the 3'-end of the Pyrosequencing primer should be complementary to a unique sequence in the amplification product to avoid background signals due to non-specific annealing during the sequencing-by-synthesis reaction. </li><li>Design the Pyrosequencing primer in a non-polymorphic region next to the variable region of interest. A number of nucleotide dispensations can be used to approach the region of interest to ensure specific annealing if necessary. </li><li>Indicate polymorphic positions in the dispensation order with degenerated bases.</li></ol><br/><strong><em>Blocker probe design</em></strong><br/>The blocker probe for E-<em>ice</em>-COLD-PCR contains Locked Nucleic Acid (LNA) nucleobases in order to anneal very strongly to the complementary sequence. For the detection and analysis of hypermethylated molecules, the LNA bases will be complementary to unmethylated CpGs, <em>i.e.</em>, to the thymines generated through PCR amplification after bisulfite conversion. Methylated CpGs (cytosines following bisulfite conversion and PCR amplification) will generate mismatches (heteroduplexes) and the blocker probe will dissociate from the target strand at the critical temperature allowing the amplification and preferential enrichment of methylated bisulfite-converted CpGs while inhibiting amplification of unmethylated bisulfite-converted CpGs.<ol class="npwtwo"><li>We recommend designing at least 3 LNA blocker probes for evaluation for each target region. </li><li>LNA blocker probes should contain more than two closely neighbored LNA bases at the unmethylated CpG sites in the center of the probe and a phosphate group at the 3' end to avoid any elongation of the blocker probe during the E-<em>ice</em>-COLD-PCR reaction.<br/><em>Note: LNA blocker probe could also be designed to block methylated bisulfite-converted DNA to enrich specifically unmethylated bisulfite-converted DNA molecules. The enrichment of unmethylated DNA is thermodynamically favored as unmethylated DNA has a lower melting temperature compared to methylated DNA after bisulfite conversion. Therefore, a greater flexibility on the assay design is possible for the enrichment of unmethylated DNA.</em> </li><li>The LNA blocker probe should have a length of between 40 and 65 bases and overlap five to six nucleotides at the 3' end with one of the amplification primers. </li><li>If it is not possible to cover all CpG sites present in the amplification product, the LNA blocker probe should cover a maximum of consecutive CpG sites representing at least 45-50% of the total CpG sites of the amplification product. The blocker probes should contain LNA bases at each unmethylated bisulfite-converted CpG sites underlying the probe sequence. <br/><em>Note: The blocker probe sequence should be complementary to the DNA methylation patterns which are present in the non-disease relevant cell populations (e.g., white blood cells). If specific CpG sites are highly methylated in samples of control individuals, this should be incorporated in the design to avoid enrichment of uninformative methylated molecules.</em> </li><li>Do not incorporate LNA bases in the last six nucleotides from the 5' or the 3' ends of the blocker sequence. </li><li>Make sure that LNA blocker probe and PCR primers cannot generate primer dimers. </li><li>Verify the prediction of the Tm of the blocker probe using the LNA oligo Tm prediction tool. The Tm should be above 70 Â°C. The Tm of the LNA blocker probe can be modulated by increasing the length and/or the number of LNA bases. If necessary, incorporate LNA bases at both bases of an unmethylated bisulfite-converted CpG site (<em>i.e.</em>, the A, which is complementary to the T of an unmethylated cytosine after bisulfite treatment, and the C complementary to the G of the CpG dinucleotide). This strategy might prove especially useful if there are less than three CpGs sites underlying the LNA blocker probe. <br/><em>Note: The calculated Tm is only an estimation of temperature, which might differ from its actual behavior during E-ice-COLD-PCR experiments. If it is not possible to increase the length of the blocker probe or the number of LNA bases within the blocker probe, a predicted temperature between 68 Â°C and 70 Â°C can be used.</em> </li></ol><br/></li><li>Bisulfite conversion<br/>Following the design, the next step of the protocol is the bisulfite conversion of the samples to be analyzed. <ol class="npwtwo"><li>Work in a pre-PCR room with filter tips. </li><li>Quantify the DNA content of the samples to be analyzed using a Qubit Fluorometer according to the manufacturerâs instructions. <br/><em>Note: This method is currently used in our laboratory and allows the quantification of circulating cell-free DNA (Mauger <em>et al.</em>, 2015). Several commercially available methods could also be used to quantify DNA as well as qPCR-based assays, which will also yield information on the fragment size of the cell-free DNA (Mauger <em>et al.</em>, 2015).</em> </li><li>Use unmethylated and methylated DNA standards to control for the efficiency of the bisulfite conversion (see Note 1). </li><li>If at least 200 ng of DNA quantified by fluorescence are available, perform bisulfite conversion using the EpiTect Fast DNA Bisulfite Kit on a QIAcube automated system (Qiagen) according to the manufacturerâs instructions in an elution volume of 40 Âµl. </li><li>If only a few ng of circulating cell-free DNA is available, perform bisulfite conversion using the EZ DNA Methylation-Gold Kit according to the manufacturerâs instructions with an elution volume of 30 Âµl (see Note 1). </li><li>Prepare serial dilutions of known bisulfite-converted methylated and unmethylated standard DNA: 100%, 75%, 50%, 20%, 10%, 5%, 2.5%, 1%, 0.5%, 0.25%, 0.1% and 0%. <br/><em>Note: Commercially available standard bisulfite-converted methylated and unmethylated DNA should be used for the optimization of E-ice-COLD-PCR and to establish standard curves for the enrichment during E-ice-COLD-PCR.</em> </li></ol><br/></li><li>E-<em>ice</em>-COLD-PCR reaction<br/>The optimization of the E-<em>ice</em>-COLD-PCR reaction consists of three main steps: the determination of the annealing temperature (Ta) of the PCR primer set, the determination of the critical temperature (Tc) for the LNA blocker probe and a concentration gradient of the designed LNA blocker probes (Figure 1, Figure 2 and Figure 3).<ol class="npwtwo"><li>Evaluate at least three different LNA blocker probes as the enrichment of E-<em>ice</em>-COLD-PCR depends on the optimal combination of an amplification primer set, the critical temperature and concentration of LNA blocker probe.</li><li>Order all PCR primers, LNA blocker probe and Pyrosequencing primers with HPLC purification grade. We recommend using the same lot of synthesized primers for the analysis of all samples in the project. For a new synthesis of LNA blocker probe, verify the enrichment by using a concentration gradient of the LNA blocker probe. New syntheses might show a slightly different enrichment efficiency.</li><li>Work in a dedicated pre-PCR room with Laminar flow cabinets and filter tips. </li><li>E-<em>ice</em>-COLD-PCR is optimized for low input of samples starting with 2 ng of bisulfite-converted DNA (corresponding to ~700 DNA molecules). For higher input of DNA (â¤ 25 ng), we recommend reducing the number of cycles of the reaction and/or the quantity of the PCR primers. </li><li>Typical E-<em>ice</em>-COLD-PCR reaction conditions are 1x HotStar Taq buffer (Qiagen) supplemented with 1.6 mM MgCl<sub>2</sub>, 200 ÂµM of each dNTP, 2.0 U of HotStar Taq polymerase, 200 nM of forward and reverse primers 2 ÂµM of SYTO9, 0-100 nM of blocker probe in a 25 Âµl volume. <br/><em>Note: HotStar Taq Polymerase is currently used in our laboratory for both E-ice-COLD-PCR analysis and standard bisulfite Pyrosequencing analysis. Other commercially available enzymes for DNA methylation could be used, but the efficiency of the enzyme for the amplification of the E-ice-COLD-PCR product as well as its compatibility with bisulfite converted DNA should be verified.</em> </li><li>We recommend analyzing each sample in triplicate and using negative and positive controls. </li><li>The E-<em>ice</em>-COLD-PCR cycling conditions include an initial denaturation step for 10 min at 95 Â°C, followed by 6 cycles of 30 s at 95 Â°C, 20 s at the Ta Â°C (see â<em>Determination of annealing temperature</em>â in Procedure C) and 10 s at 72 Â°C, followed by 52 cycles of 20 s at 95 Â°C, 30 s at 70 Â°C, 20 s at Tc Â°C (see â<em>Determination of the critical temperature</em>â in Procedure C), 20 s at Ta Â°C and 10 s at 72 Â°C, followed by a melting curve analysis with 20 acquisition per degree from 65 to 95 Â°C and a final cooling step to 40 Â°C (Figure 2). <br/><em>Note: Determination of the required number of cycles of the E-ice-COLD-PCR reaction should be performed especially for higher quantity of input sample.</em> </li><li>We recommend verifying the E-<em>ice</em>-COLD-PCR reaction using a melting curve analysis prior to proceeding to the Pyrosequencing analysis (Figure 4A).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191201/20191201191433_0453.jpg" width="450"/><br/><strong>Figure 2. Schematic overview of the different steps of an E-<em>ice</em>-COLD-PCR assay composed of three major steps: Initial denaturation, standard PCR amplification and E-<em>ice</em>-COLD-PCR amplification</strong> </li></ol><br/><em><strong>Determination of annealing temperature</strong></em><br/>The optimal annealing temperature should be determined in a temperature gradient thermocycler using bisulfite-converted DNA controls of 100%, 50%, 0% of DNA methylation as well as a negative control.<ol class="npwtwo"><li>The PCR reaction conditions are the same as used for E-<em>ice</em>-COLD-PCR without adding the LNA blocker probe. </li><li>The gradient PCR program consists of a denaturing step of 15 min at 95 Â°C, followed by 52 cycles of 30 s at 95 Â°C, 20 s at a gradient Ta and 10 s at 72 Â°C, with a final extension at 72 Â°C. </li><li>In a post-PCR room, deposit 10 Âµl of the PCR product as well as a positive and negative control on a 2% agarose gel.</li><li>The optimal annealing temperature is selected taking the expected size and the intensity of the PCR product band as well as specificity of the reaction into account. Although not absolutely necessary for E-<em>ice</em>-COLD-PCR, the absence of a strong amplification bias during PCR reaction should be verified by Pyrosequencing. <br/><em>Note: Due to the difference in GC content between completely unmethylated and methylated molecules following bisulfite conversion, a preferential amplification in most cases of unmethylated molecules is commonly observed. This preferential amplification might be attenuated using different primer pairs and/or enzymes for the PCR amplification, but persists in many cases (Warnecke <em>et al.</em>, 1997).</em> </li><li>Analyze the PCR product by Pyrosequencing to verify the correct sequence has been amplified and the Pyrosequencing primer is annealing uniquely and does not create any background noise. The optimal Pyrosequencing primer should yield signals (Pyrosequencing peaks) of expected height of the bisulfite-converted DNA standards and no observable signals on negative controls (no template, genomic DNA).</li></ol><br/><em><strong>Determination of the critical temperature (Tc)</strong></em> <ol class="npwtwo"><li>Identify the critical temperature using a temperature gradient quantitative PCR between 70 Â°C and 90 Â°C. <br/><em>Note: The use of a gradient-enabled qPCR cycler allows for the rapid determination of the critical temperature and associated blocker probe concentration. If no gradient qPCR thermocycler is available, several independent reactions using different temperatures have to be performed. Alternatively, optimization can be performed using a standard thermocycler and analyzing PCR products on an agarose gel followed by a sequencing reaction for reaction conditions yielding a PCR product.</em> </li><li>We recommend for the first test using concentrations of 0, 25 and 50 nM of LNA blocker probe.</li><li>Control the reaction by using serial dilutions of bisulfite-converted methylated DNA controls into bisulfite converted unmethylated DNA yielding a DNA methylation degree of 0%, 10%, 25%, 50% and 100% and a negative control. </li><li>We recommend analyzing samples in triplicates especially if quantification of the DNA methylation level is performed as some variation between individual amplifications exists.</li><li>Quantify the enrichment of DNA methylation using Pyrosequencing by comparing the E-<em>ice</em>-COLD-PCR reaction to a standard qPCR reaction without adding the LNA blocker probe (Figure 4C). </li><li>For some combinations (LNA blocker probes, concentration of LNA blocker probe and/or Tc of one LNA blocker probe) no enrichment, no PCR product, or presence of primer dimers might be encountered. In this case, decrease the concentration of blocker probe for Tc determination.</li><li>The enrichment of DNA methylation should clearly be obtained for all replicates, different % of methylated DNA as input (except for unmethylated DNA) and for several concentrations of the same LNA blocker probe. At the optimal Tc a high enrichment should be obtained but yield a high quality Pyrogram (without dispensation artefacts and sufficient intensity of peaks) and it should avoid non-specific amplification of the E-<em>ice</em>-COLD-PCR reaction (see Note 2).<br/><em>Note: It is important to avoid a complete blocking of unmethylated DNA, as in this case an amplification failure due to low quality samples will be indistinguishable from a completely unmethylated sample. For all samples, a PCR amplification product should be obtained, with an enrichment of methylated molecules in those which contain methylated molecules.</em> </li></ol><br/><strong><em>Determination of concentration of LNA blocker probe</em></strong> <ol class="npwtwo"><li>Use a concentration gradient of the blocker probe at 0, 20, 40, 60, 80 and 100 nM. </li><li>Use a serial dilution of standard bisulfite-converted methylated DNA of 0%, 0.5%, 1%, 5%, 10%, 20 % and 100% into unmethylated DNA as well as a negative control.</li><li>Analyze samples in triplicates.</li><li>Analyze the melting curve of each reaction for E-<em>ice</em>-COLD-PCR, amplification and enrichment of unmethylated and/or methylated DNA, primer dimer formation and reproducibility of replicates (Figure 4A).</li><li>The optimal concentration of the blocker probe should be obtained by analyzing the Pyrograms selecting the concentration with the highest enrichment without any non-specific reactions. </li><li>Another gradient of concentration should be tested with a narrower concentration range around the pre-defined concentration to find the optimal concentration of LNA blocker probe.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191201/20191201191500_0946.jpg" width="450"/><br/><strong>Figure 3. Enrichment of DNA methylation levels by E-<em>ice</em>-COLD-PCR detected by pyrosequencing in function of the input of methylated DNA for various LNA blocker probe concentrations.</strong> The example shows enrichment of DNA methylation at the <em>AIM2</em> gene promoter (Mauger <em>et al.</em>, 2018) and demonstrates that the enrichment depends on the concentration of LNA blocker probe.</li></ol><br/><em><strong>Quantification of DNA methylation by E-ice-COLD-PCR standard curve</strong></em><br/>E-<em>ice</em>-COLD-PCR is an enrichment method for the detection of low-abundant methylated DNA and thus yields mainly a qualitative result on the presence of DNA methylation, but it can also be used to quantify the level of DNA methylation using standard curves.<ol class="npwtwo"><li>Control optimal conditions of E-<em>ice</em>-COLD-PCR by comparing standard curves of CpG sites to those obtained using standard qPCR without the blocker probe.</li><li>Analyze serial dilutions of standards with a known degree of methylation (100%, 75%, 50%, 20%, 10%, 5%, 2.5%, 1%, 0.5%, 0.25%, 0.1% and 0%) in sextuplicates (see Note 2).</li><li>Analyze samples in triplicates.</li></ol><br/><strong><em>Multiplex E-ice-COLD-PCR</em></strong><br/>If the amount of sample available is limited such as for the analysis of circulating cell-free DNA, E-<em>ice</em>-COLD-PCR reaction can be multiplexed analyzing several target regions in parallel (Sefrioui <em>et al.</em>, 2017; Mauger <em>et al.</em>, 2018).<ol class="npwtwo"><li>E-<em>ice</em>-COLD PCR conditions for multiplex assays should be the same as for the simplex assay except that the concentration of each PCR primer should be decreased (example: half the quantity for a duplex) (Mauger <em>et al.</em>, 2018).</li><li>Multiplex E-<em>ice</em>-COLD-PCR reactions using the same Ta and Tc.<br/><em>Note: The critical temperature of the different assays might not be identical, in that case a critical temperature should be chosen, that allows the best enrichment of all targets included in the multiplex assay (Mauger <em>et al.</em>, 2016).</em> </li><li>Optimize the multiplex PCR by comparing the enrichment obtained with the simplex assay and standard qPCR varying the blocker concentration and/or critical temperature slightly.</li><li>Analyze the multiplexing experiment using a melting curve analysis. Amplifications should not yield primer dimers.</li></ol><br/></li><li>Pyrosequencing assay<br/>E-<em>ice</em>-COLD-PCR reaction is analyzed by Pyrosequencing (Tost and Gut, 2007; Busato <em>et al.</em>, 2018). Prior to the Pyrosequencing reaction on the PyroMark Q96 MD System, a sample preparation step is performed which includes the purification and the denaturation of E-<em>ice</em>-COLD-PCR product to obtain single-stranded product followed by hybridization of the Pyrosequencing primer. <br/><em>Notes:</em><br/><em> <ol class="npwthree"><li>E-ice-COLD-PCR reaction is currently analyzed by Pyrosequencing in our laboratory as it allows for short-turnaround time, but the amplification products could be analyzed by Sanger or NGS sequencing or if no detailed information on individual CpG positions is requiredâsimply by the included high-resolution melting analysis.</li><li>This protocol uses the PyroMark Q96 MD System, but the Pyrosequencing analysis can also be performed on the more recent PyroMark Q24 Advanced System according to the manufacturerâs instructions.</li></ol></em><br/><em><strong>Sample preparation</strong></em> <ol class="npwtwo"><li>We recommend working in a dedicated post-PCR room. </li><li>Prepare the PCR plate by transferring 10 Âµl of E-<em>ice</em>-COLD-PCR reaction, adding 40 Âµl of binding buffer and 2 Âµl of sepharose beads and completing to 80 Âµl with water in a new PCR plate. <br/><em>Note: The different buffers used for Pyrosequencing sample preparation are also commercially available from Qiagen. We commonly prepare them ourselves.</em> </li><li>Seal the PCR plate and incubate the reaction for 10 min at room temperature under agitation (1,400 rpm) allowing the sepharose beads to capture the biotinylated amplification products.</li><li>During the incubation, prepare the Pyrosequencing plate by diluting 4 pmol of the Pyrosequencing primer into 12 Âµl of annealing buffer per well. Several Pyrosequencing primers can be used in the same plate.<br/><em>Note: For multiplexing experiments, it might be necessary to re-amplify each enriched PCR product separately in a standard PCR reaction as Pyrosequencing is performed at low </em><em>temperature due to the thermal instability of the enzyme mix and sequencing primers might anneal to several of the amplified regions simultaneously leading to a high background signal.</em> </li><li>Prepare the vacuum workstation by adding approximately 180 mI of 70% Ethanol, 180 ml of denaturing solution, 200 ml of wash buffer and 200 ml of water. </li><li>Turn on the vacuum of the workstation to create a vacuum in the aspiration device. </li><li>Clean the tips of the filters by immersion in water for several seconds. </li><li>Remove the PCR plate from the plate shaker and the seal from the PCR plate. Aspirate the remaining binding buffer from the PCR plate. The beads remain on the filters of the tips.</li><li>Immerse the tips of the filters in successive baths of 70% Ethanol for 5 s, denaturing solution for 5 s and wash buffer for 10 s. Turn over the tool and release the vacuum.</li><li>Immerse the tips of the filters in the annealing mix of the Pyrosequencing plate and shake gently to release the beads into the wells.</li><li>Incubate the Pyrosequencing plate at 80 Â°C for 2 min on a thermoplate placed on a heating device. </li><li>Let the Pyrosequencing plate cool down at room temperature to allow annealing of the sequencing primer.</li></ol><br/><em><strong>Pyrosequencing reaction</strong></em> <ol class="npwtwo"><li>During the cooling of the Pyrosequencing plate, program the Pyrosequencing run on the Pyrosequencer. </li><li>Calculate the amount of reagents for the Pyrosequencing run using the software. Dispense the reagent in the appropriate tips and place the tips in the cartridge avoiding bubble formation. </li><li>Place the reagentâs cartridge and the sealed Pyrosequencing test plate into the Pyrosequencer. Perform the dispensation test to be sure that the dispensing tips are working properly. Droplets should be clearly visible and homogenous.</li><li>Deposit the Pyrosequencing plate into the Pyrosequencer and start the Pyrosequencing run. The length of a run is proportional to the number of dispensations (1 per min).</li><li>In some cases, the signals corresponding to part of the sequence may be missing. Run a dispension test to check if some tips are blocked.</li><li>After the end of the Pyrosequencing run, remove the cartridge and clean the tips by rinsing with pure water. </li></ol></li></ol><p class="pptt" id="biaoti30460">### Data analysis</p><ol class="npwtwok"><li>Analyze E-<em>ice</em>-COLD-PCR reactions using the LightCycler 480 Software.</li><li>Compare the melting curves of each sample with serial dilutions of bisulfite-converted unmethylated and methylated DNA standards and a negative control to verify the amplification by E-<em>ice</em>-COLD-PCR and to check for the absence of primer dimers. </li><li>In multiplex E-<em>ice</em>-COLD-PCR reactions, the peak of unmethylated and/or methylated DNA can be obtained for each reaction (Figure 4A).</li><li>Analyze the Pyrosequencing data using the PyroMark MD Software. </li><li>Export the results to be treated with statistical or graphical software such as Excel<sup>Â®</sup>. </li><li>Calculate the % of DNA methylation for each CpG site using the intensity of the peak of cytosine and thymine of the CpG site for each sample (Figure 4B).</li><li>Establish standard curves for each CpG site using serial dilutions of standard bisulfite-converted unmethylated and methylated DNA. Calculate the equation and the standard deviation of each standard curve (see Note 3) (Figure 4C). </li><li>Quantify the % of DNA methylation of each sample using the standard curves and the % of DNA methylation obtained by Pyrosequencing (Figures 4B and 4C). </li><li>Calculate, for each sample, the average % of DNA methylation of all CpG sites analyzed.</li><li>Calculate the standard deviation of the % of the DNA methylation using triplicates for each sample.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191201/20191201191520_7779.jpg" width="450"/><br/><strong>Figure 4. Analysis of DNA methylation by E-<em>ice</em>-COLD-PCR.</strong> A. Example of a melting curve of duplex E-<em>ice</em>-COLD-PCR reaction analyzing methylation in the <em>AIM2</em> and <em>RARB2</em> genes that shows unmethylated and methylated peaks for each amplicon. B. Example of a Pyrogram of <em>RARB2</em> following E-<em>ice</em>-COLD-PCR. C. Example of E-<em>ice</em>-COLD-PCR and qPCR standard curves for 5 CpGs sites in the <em>RARB2</em> promoter.<br/></li></ol><p class="pptt" id="biaoti30461">### Notes</p><ol class="npwtwok"><li>We currently use the EZ DNA Methylation-Gold (Zymo Research) and the Epitect Fast DNA bisulfite kits (Qiagen) in our laboratory. Several commercially available kits can be used but they could have different efficiencies for different sample types (Worm Ãrntoft <em>et al.</em>, 2017). </li><li>The standard curve of the qPCR reaction (without blocker) can also be used to estimate the enrichment by E-<em>ice</em>-COLD-PCR (Figure 4C).</li><li>E-<em>ice</em>-COLD PCR standard curves can saturate at high levels of DNA methylation due to the enrichment by E-<em>ice</em>-COLD-PCR reaction. Consequently, the quantification using these standard curves cannot be performed for high DNA methylation level. In this case, qPCR reaction without enrichment can be performed to quantify high levels of DNA methylation (Figure 4C) or another E-<em>ice</em>-COLD-PCR reaction can be performed using a lower concentration of the LNA blocker probe.<br/></li></ol><p class="pptt" id="biaoti30462">### Recipes</p><ol class="npwtwok"><li>Binding Buffer <br/>10 mM Tris-HCl<br/>2 mM NaCl<br/>1 mM EDTA<br/>0.1% Tween-20, pH 7.6</li><li>Annealing Buffer<br/>20 mM Tris-acetate<br/>2 mM Mg-acetate, pH 7.6 adjusted with acetic acid 4 M</li><li>Wash Buffer<br/>10 mM Tris-acetate, pH 7.6 adjusted with acetic acid 4 M<br/></li></ol><p class="pptt" id="biaoti30463">### Acknowledgments</p><p class="ppzz">This work was supported by the institutional budget of the Centre National de Recherche en GÃ©nomique Humaine (CNRGH). This protocol was adapted from previous work (How-Kit and Tost <em>et al.</em>, 2015; Busato <em>et al.</em>, 2018; Mauger <em>et al.</em>, 2018).</p><p class="pptt" id="biaoti30464">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti30465">### Ethics</p><p class="ppzz">Use of breast cancer samples in the framework of the original publication (Mauger <em>et al.</em>, 2018) was approved by the Norwegian Regional Committee for Medical Research Ethics (approval numbers 429-04148 and 2014/895), and patients have given written informed consent for the use of material for research purposes. No additional samples were used for the preparation of this manuscript.</p><p class="pptt" id="biaoti30466">### References</p><ol class="npwtwok"><li>Alnaes, G. I., Ronneberg, J. A., Kristensen, V. N. and Tost, J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26393654" target="_blank">Heterogeneous DNA methylation patterns in the GSTP1 promoter lead to discordant results between assay technologies and impede its implementation as epigenetic biomarkers in breast cancer.</a> <em>Genes (Basel)</em> 6(3): 878-900.</li><li>Barlebo Ahlborn, L. and Ãstrup, O. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30784124" target="_blank">Toward liquid biopsies in cancer treatment: application of circulating tumor DNA.</a> <em>APMIS</em> 127(5): 329-336.</li><li>Busato, F., Dejeux, E., El Abdalaoui, H., Gut, I. G. and Tost, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224157" target="_blank">Quantitative DNA methylation analysis at single-nucleotide resolution by pyrosequencing<sup>Â®</sup>.</a> <em>Methods Mol Biol</em> 1708: 427-445.</li><li>Cabel, L., Proudhon, C., Romano, E., Girard, N., Lantz, O., Stern, M. H., Pierga, J. Y. and Bidard, F. C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30050094" target="_blank">Clinical potential of circulating tumour DNA in patients receiving anticancer immunotherapy. </a><em>Nat Rev Clin Oncol</em> 15(10): 639-650.</li><li>Campan, M., Weisenberger, D. J., Trinh, B. and Laird, P. W. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224160" target="_blank">MethyLight and digital methyLight.</a> <em>Methods Mol Biol</em> 1708: 497-513.</li><li>Candiloro, I. L., Mikeska, T. and Dobrovic, A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21913071" target="_blank">Closed-tube PCR methods for locus-specific DNA methylation analysis.</a> <em>Methods Mol Biol</em> 791: 55-71.</li><li>Castellanos-Rizaldos, E., Milbury, C. A., Karatza, E., Chen, C. C., Makrigiorgos, G. M. and Merewood, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24728321" target="_blank">COLD-PCR amplification of bisulfite-converted DNA allows the enrichment and sequencing of rare un-methylated genomic regions.</a> <em>PLoS One</em> 9(4): e94103.</li><li>Distler, J. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18987825" target="_blank">Quantification of methylated DNA by HeavyMethyl duplex PCR.</a> <em>Methods Mol Biol</em> 507: 339-46.</li><li>Frommer, M., McDonald, L. E., Millar, D. S., Collis, C. M., Watt, F., Grigg, G. W., Molloy, P. L. and Paul, C. L. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1542678" target="_blank">A genomic sequencing protocol that yields a positive display of 5-methylcytosine residues in individual DNA strands.</a> <em>Proc Natl Acad Sci U S A</em> 89(5): 1827-1831.</li><li>How-Kit, A. and Tost, J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26103893" target="_blank">Pyrosequencing<sup>Â®</sup>-based identification of low-frequency mutations enriched through enhanced-ice-COLD-PCR.</a> <em>Methods Mol Biol</em> 1315: 83-101.</li><li>Jung, M., Kristiansen, G. and Dietrich, D. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224167" target="_blank">DNA methylation analysis of free-circulating DNA in body fluids.</a> <em>Methods Mol Biol</em> 1708: 621-641.</li><li>Kumar, M., Choudhury, Y., Ghosh, S. K. and Mondal, R. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29484962" target="_blank">Application and optimization of minimally invasive cell-free DNA techniques in oncogenomics.</a> <em>Tumour Biol</em> 40(2): 1010428318760342.</li><li>Lehmann-Werman, R., Neiman, D., Zemmour, H., Moss, J., Magenheim, J., Vaknin-Dembinsky, A., Rubertsson, S., Nellgard, B., Blennow, K., Zetterberg, H., Spalding, K., Haller, M. J., Wasserfall, C. H., Schatz, D. A., Greenbaum, C. J., Dorrell, C., Grompe, M., Zick, A., Hubert, A., Maoz, M., Fendrich, V., Bartsch, D. K., Golan, T., Ben Sasson, S. A., Zamir, G., Razin, A., Cedar, H., Shapiro, A. M., Glaser, B., Shemer, R. and Dor, Y. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26976580" target="_blank">Identification of tissue-specific cell death using methylation patterns of circulating DNA.</a> <em>Proc Natl Acad Sci U S A</em> 113(13): E1826-1834.</li><li>Lewis, J. M., Heineck, D. P. and Heller, M. J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26189641" target="_blank">Detecting cancer biomarkers in blood: challenges for new molecular diagnostic and point-of-care tests using cell-free nucleic acids.</a> <em>Expert Rev Mol Diagn</em> 15(9): 1187-1200.</li><li>Li, L. C. and Dahiya, R. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112" target="_blank">MethPrimer: designing primers for methylation PCRs.</a> <em>Bioinformatics</em> 18(11): 1427-1431. </li><li>Liu, Y., Song, C., Ladas, I., Fitarelli-Kiehl, M. and Makrigiorgos, G. M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27903892" target="_blank">Methylation-sensitive enrichment of minor DNA alleles using a double-strand DNA-specific nuclease. </a> <em>Nucleic Acids Res</em> 45(6): e39.</li><li>Mauger, F., Daunay, A., Deleuze, J. F., Tost, J. and How-Kit, A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27335078" target="_blank">Multiplexing of E-<em>ice</em>-COLD-PCR assays for mutation detection and identification.</a> <em>Clin Chem</em> 62(8): 1155-1158.</li><li>Mauger, F., Dulary, C., Daviaud, C., Deleuze, J. F. and Tost, J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26123439" target="_blank">Comprehensive evaluation of methods to isolate, quantify, and characterize circulating cell-free DNA from small volumes of plasma.</a> <em>Anal Bioanal Chem</em> 407(22): 6873-6878. </li><li>Mauger, F., How-Kit, A. and Tost, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28101802" target="_blank">COLD-PCR technologies in the area of personalized medicine: methodology and applications.</a> <em>Mol Diagn Ther</em> 21(3): 269-283.</li><li>Mauger, F., Kernaleguen, M., Lallemand, C., Kristensen, V. N., Deleuze, J. F. and Tost, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29697281" target="_blank">Enrichment of methylated molecules using enhanced-<em>ice</em>-co-amplification at lower denaturation temperature-PCR (E-<em>ice</em>-COLD-PCR) for the sensitive detection of disease-related hypermethylation.</a> <em>Epigenomics</em> 10(5): 525-537.</li><li>MolnÃ¡r, B., TÃ³th, K., BartÃ¡k, B. K. and Tulassay, Z. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25429690" target="_blank">Plasma methylated septin 9: a colorectal cancer screening marker.</a> <em>Expert Rev Mol Diagn</em> 15(2): 171-184.</li><li>Sefrioui, D., Mauger, F., Leclere, L., Beaussire, L., Di Fiore, F., Deleuze, J. F., Sarafan-Vasseur, N. and Tost, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27940131" target="_blank">Comparison of the quantification of KRAS mutations by digital PCR and E-<em>ice</em>-COLD-PCR in circulating-cell-free DNA from metastatic colorectal cancer patients.</a> <em>Clin Chim Acta</em> 465: 1-4. </li><li>Siravegna, G., Marsoni, S., Siena, S. and Bardelli, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28252003" target="_blank">Integrating liquid biopsies into the management of cancer.</a> <em>Nat Rev Clin Oncol</em> 14(9): 531-548.</li><li>Tost, J. and Gut, I. G. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17853883" target="_blank">DNA methylation analysis by pyrosequencing.</a> <em>Nat Protoc</em> 2(9): 2265-75. </li><li>Tost, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27826846" target="_blank">Current and emerging technologies for the analysis of the genome-wide and locus-specific DNA methylation patterns.</a> <em>Adv Exp Med Biol</em> 945: 343-430.</li><li>Wan, J. C. M., Massie, C., Garcia-Corbacho, J., Mouliere, F., Brenton, J. D., Caldas, C., Pacey, S., Baird, R. and Rosenfeld, N. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28233803" target="_blank">Liquid biopsies come of age: towards implementation of circulating tumour DNA.</a> <em>Nat Rev Cancer</em> 17(4): 223-238.</li><li>Warnecke, P. M., Stirzaker, C., Melki, J. R., Millar, D. S., Paul, C. L. and Clark, S. J. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9336479" target="_blank">Detection and measurement of PCR bias in quantitative methylation analysis of bisulphite-treated DNA.</a> <em>Nucleic Acids Res</em> 25(21): 4422-4426. </li><li>Warton, K., Mahon, K. L. and Samimi, G. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26764421" target="_blank">Methylated circulating tumor DNA in blood: power in cancer prognosis and response.</a> <em>Endocr Relat Cancer</em> 23(3): R157-171.</li><li>Worm Ãrntoft, M. B., Jensen, S. Ã., Hansen, T. B., Bramsen, J. B. and Andersen, C. L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28557629" target="_blank">Comparative analysis of 12 different kits for bisulfite conversion of circulating cell-free DNA.</a> <em>Epigenetics</em> 12(8): 626-636.</li></ol>