<body>
            <h1>Lysosome Targeting RedGreen-assay: Selective Autophagy Sensing Assay for Mammalian Cells</h1>
            <p><strong>Authors:</strong> Victoria Riccio, Miluska Vissa, Angus McQuibban, Peter Kijun Kim</p>
            <h2>Abstract</h2>
            <p>The process of autophagy is an essential cellular mechanism, required to maintain general cell health through the removal of dysfunctional organelles, such as the ER, peroxisomes and mitochondria, as well as protein aggregates, and bacteria. Autophagy is an extremely dynamic process, and tools are constantly being developed to study the various steps of this process. This protocol details a method to study the end steps of autophagy-lysosomal fusion and the formation of the autolysosome. Many techniques have been used to study the various steps of the autophagy process. Here we describe the RedGreen-assay (RG-assay), an immunofluorescence-based technique used to visualize the targeting of substrates to the autolysosome in live cells. This technique takes advantage of the low lysosomal pH and over-expression of a tandem GFP-mCherry tagged protein targeted to an organelle of interest. While in the neutral cytosol or autophagosome, both GFP and RFP will fluoresce. However, within the autolysosome, the GFP signal is quenched due to the low pH environment and the RFP emission signal will predominate. This technique is readily quantifiable and amenable to high throughput experiments. Additionally, by tagging the GFP-RFP tandem fluorescent protein with organelle specific targeting sequences, it can be used to measure a wide range of substrates of autophagy. </p>
            <p><strong>Keywords:</strong> Autophagy, Pexophagy, Mitophagy, Lysosome, Autolysosome, Microscopy, Live-cell imaging</p>
            <h2>Background</h2><p>Macroautophagy (herein referred to as autophagy) is an essential cellular process, required for the removal of large cellular material. It can be both general and selective, either removing bulk material or specific cytoplasmic material, respectively. The method described in this protocol can be used in the study of either process, but the focus here will be on selective autophagy. In brief, selective autophagy in mammalian cells begins with the tagging of damaged organelles or other cellular material, generally with the small protein ubiquitin. The collection of ubiquitinated proteins on the substrate allows the recruitment of autophagy receptors which interact with the ubiquitin protein and with the ATG8 family of autophagic factors on developing phagophores through their LC3 interacting region (LIR). This interaction results in the engulfment of the cargo by the phagophore, producing an autophagosome. Finally, the autophagosome fuses with the lysosome, producing an autolysosome and degrading the cargo within. Selective autophagy is responsible for degrading cargo such as peroxisomes (pexophagy), mitochondria (mitophagy), protein aggregates (aggrephagy) and bacteria (xenophagy). For a review, refer to Lahiri <em>et al.</em>, 2019 and Morishita and Mizushima, 2019. As autophagy proceeds in many steps, different assays have been developed to study the various stages. Historically, the most popular assay to detect changes in autophagy induction is the activation of the ATG8 family of proteins, particularly LC3B. Another common assay for detecting targeting of substrate to the autolysosome is the co-localization between the cargo with either the autophagosome marker LC3, or with various lysosomal markers such as Lamp1. Co-localization studies are technically easy to perform, can be performed in a high throughout manner and can detect specific cargo within the lysosome easily through the use of fluorescent tagging. However, it is limited by the microscope resolution, and the cargo can be difficult to detect due to degradation. For a review describing these and other methods for detecting autophagy, refer to Klionsky <em>et al.</em>, 2016.<br/><br/>The protocol described here, lysosome targeting Red-Green (RG) assay, can be used to study the end stages of autophagy, namely the localization of the cargo to the autolysosome. This assay takes advantage of the acidic pH of the lysosome (~4.6) and its ability to quench GFP (pKa ~6) fluorescence without affecting RFP (pKa = 4.5) fluorescence. Through the use of a tandem RFP-GFP fusion protein targeted to an organelle of choice, cargo that is cytosolic or within an autophagosome, and therefore exposed to neutral pH, will appear yellow when using pseudo colors green for GFP and red for RFP due to the fluorescence of both GFP and RFP. Upon lysosomal fusion, the labeled cargo will be exposed to an acidic pH, quenching GFP fluorescence and resulting in mainly RFP fluorescence (Figure 1). Again, this can be detected by microscopy and the ratio of cargo with RFP-only fluorescence can be compared to the total RFP fluorescence, to obtain a measure of autophagy. This method was first used to study aggrephagy (Pankiv <em>et al.</em>, 2007). By tagging LC3 with a tandem GFP-mCherry, autophagic vesicles and autolysosomes were able to be readily visualized and distinguished. We have adopted the principle used with LC3 to visualize the targeting of peroxisomes to autolysosomes. This is done by tagging the peroxisome targeting transmembrane domain of PEX26 with the tandem fluorophore, allowing it to be localized to the cytosolic face of the peroxisomal membrane (Deosaran <em>et al.</em>, 2013; Riccio <em>et al.</em>, 2019). This allows for the visualization of pexophagy specifically. Others have also used a similar technique to study pexophagy, where the fluorescent proteins were targeted to the peroxisome matrix through the use of a matrix targeting sequence (Nazarko <em>et al.</em>, 2014). Importantly, our method, targeting the fluorophore to the cytosolic face of the peroxisome membrane, allows for increased sensitivity of the tandem fluorophore to the acidic environment of the lysosome. Additionally, our quantification method differs, taking into account the relative GFP and mCherry fluorescence in order to determine the quantity of peroxisomes targeted to the lysosome. In addition to our use of the tandem fluorophore in pexophagy, we and others have used this method to quantify mitophagy (Allen <em>et al.</em>, 2013; Kim <em>et al.</em>, 2014; Wang <em>et al.</em>, 2015). <br/><br/>We have shown that the use of the tandem fluorophore is a reproducible method for observing and quantifying selective autophagy. As displayed above, the tandem mCherry-GFP lysosome targeting assay is extremely flexible, and can be used to measure various forms of selective autophagy, as well as general autophagy. Through the pH sensitivity of the fluorophores, it is a reliable method to detect lysosomal targeting. Additionally, because GFP and mCherry are expressed at equal levels, comparing the fluorescent intensities of mCherry to GFP allows for a reproducible method to quantify pexophagy.</p>
        </body>

<br/><p class="ppzz"><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191212/20191212011044_8266.jpg" width="228"/><br/><strong>Figure 1. Schematic of the lysosome targeting RG assay.</strong> Peroxisomes expressing the PEX26<sub>TM</sub>-RG protein will appear yellow within the cytosol or the autophagosome due to fluorescence of both GFP and mCherry. When targeted to the lysosome, GFP fluorescence will be quenched and the peroxisome will appear red.</p><p class="pptt" id="biaoti30661">## Materials and Reagents</p><ol class="npwtwok"><li>Stericup Quick Release Millipore Express PLUS 0.22 Î¼m PES filter, 500 ml (Millipore Sigma, catalog number: S2GPU05RE)</li><li>T25 flask (Corning Life Sciences, catalog number: C353109)</li><li>4-well Nunc<sup><sub>TM</sub></sup> Lab-Tek<sup><sub>TM</sub></sup> Chambered Coverglass (Thermo Fisher Scientific, catalog number: 155383)</li><li>Lipofectamine 2000 (Life Technologies, catalog number: 11668-019)</li><li>E-64 (Enzo Life Sciences, catalog number: BML-PI107-0001)</li><li>Leupeptin (Bioshop, catalog number: LEU001)</li><li>HeLa human cervical cancer cell line (ATCC CCL-2)</li><li>GFP-mCherry (RG)-PEX26<sub><sub>TM</sub></sub> plasmid</li><li>Dulbeccoâs Modified Eagleâs Medium (Hyclone, catalog number: SH3008101)</li><li>L-Glutamine (Hyclone, catalog number: SH30034.01)</li><li>Fetal Bovine Serum (Life Technologies, catalog number: A12617)</li><li>OptiMEM Reduced Serum Medium (Thermo Fisher, catalog number: 31985062)</li><li>Trypsin-EDTA (0.05%), phenol red (Thermo Fisher, catalog number: 25300062)</li><li>CO<sub>2</sub> independent medium (Life Technologies, catalog number: 18045)</li><li>Hankâs Balanced Salt Solution (HBSS) (Thermo Fisher, catalog number: 14025092)</li></ol><p class="pptt" id="biaoti30662">### Equipment</p><ol class="npwtwok"><li>Zeiss LSM710 equipped with a 63x 1.4 NA oil immersion objective and the appropriate lasers (488 nm on a tuneable Argon 458/477/488/514 nm at 30 mM with a 493-565 nm bandpass filter and 561 nm DPSS laser with a 600-700 nm bandpass)</li><li>Forma Steri-Cycle CO<sub>2</sub> Incubator (Thermo Fisher) <br/></li></ol><p class="pptt" id="biaoti30663">### Software</p><ol class="npwtwok"><li>Volocity (Perkin Elmer, version 6.3)</li><li>Zeiss Zen (Zeiss, <a data-ke-src="https://www.zeiss.com/microscopy/int/products/microscope-software/zen.h&lt;sub&gt;TM&lt;/sub&gt;l" href="https://www.zeiss.com/microscopy/int/products/microscope-software/zen.h&lt;sub&gt;TM&lt;/sub&gt;l" target="_blank">https://www.zeiss.com/microscopy/int/products/microscope-software/zen.html</a>)</li><li>Microsoft<sup>Â®</sup> Office Excel (Microsoft Office 365)<br/></li></ol><p class="pptt" id="biaoti30664">### Procedure</p><ol class="npwone"><li>Gene cloning<br/>GFP-mCherry-PEX26<sub>TM</sub> was constructed in several steps as described in Deosaran <em>et al.</em>, 2013. First, the PEX26 ORF from pSPORT-PEX26 (Invitrogen) was cloned into the EcoRI and SalI sites of pmEGFP-C1, to insert the entire ORF of PEX26 at the C-terminus of GFP. Next, the entire PEX26 but its C-terminus tail-anchor domain (TM) was removed by PCR using two primers containing a BglII site. Finally, primers containing a BglII or BamHI sites were used to PCR amplified mCherry. Following digestion with BglII and BamHI, the PCR product was inserted into the BglII site on the pmEGFP-PEX26<sub>TM</sub> vector to generate the GFP-mCherry-PEX26<sub>TM</sub> (RG-PEX26<sub>TM</sub>) vector. Refer to Figure 2 for a schematic. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191212/20191212012115_0650.jpg" width="146"/><br/><strong>Figure 2. Schematic diagram of the construction of the RG-PEX26<sub>TM</sub> construct.</strong> Restriction enzymes used are indicated. Small arrows indicate the location of primer used. <br/><br/></li><li>Cell seeding <br/>Here we will describe work done on HeLa cells which were grown in Dulbecco's Modified Eagle Media (DMEM) supplemented with 10% FBS and 5% CO<sub>2</sub>. Media was filter sterilized with FBS. However, this protocol can be adapted to most transfectable cell lines. We have tested this protocol in a number of transfectable cells such as MEFs, Cos7, U2OS, Hek293 and MDCKs. <ol class="npwtwo"><li>To seed cells, first warm sterile media and PBS to 37 Â°C. Thaw trypsin and warm to room temperature. </li><li>Remove cells from the incubator and aspirate growth media. Wash with PBS (about 5 ml) and aspirate. Add trypsin to the flask at an appropriate volume to ensure cells will not dry (<em>i.e.</em>, 1 ml trypsin in a T25 flask).</li><li>Incubate cells in trypsin at 37 Â°C for 3-5 min.</li><li>Once cells are no longer adhered to the flask, add media to dilute and inactivate the trypsin (about 5 ml). Collect media containing cells in a 15 ml tube and centrifuge at 1,000 <em>x g</em> for 5 min to pellet cells.</li><li>Following centrifugation, aspirate media and resuspend the cells in fresh media.</li><li>Count the cells using a hemocytometer or other cell counting device.</li><li>Seed cells in a 4-well Lab-Tek chamber (or equivalent coverslip system that allows for live cell imaging) at a density of 25,000 cells/well. Total volume within the well should be 0.5 ml. For other cell types or longer procedures, seed cells at an appropriate density for transfection.</li><li>Allow cells to adhere and rest for 24 h before transfection in the 37 Â°C incubator. <br/><br/></li></ol></li><li>Cell transfection<ol class="npwtwo"><li>On Day 2, transfect with the RG-PEX26<sub>TM</sub> construct (and any other construct required for the experiment). Use 1-2 Î¼g of DNA per well and Lipofectamine2000 transfection reagent according to the manufacturerâs instructions. In brief, combine 0.5 Î¼l of Lipofectamine2000 with 50 Î¼l of Opti-mem medium in a 1.5 ml tube. Allow it to sit for 5 min. In another 1.5 ml tube, combine 50 Î¼l of Opti-mem medium and DNA. Combine the lipofectamine and DNA containing mixtures and let sit for 20 min. Do this for each transfected well. Add complete mixture (100 Î¼l) to each well.</li><li>Let cells grow for 24-36 h.</li><li>Change media to DMEM + FBS or HBSS without FBS. HBSS induces mTORC1 related pexophagy due to amino acid starvation. Both are supplemented with 2 Î¼M E-64 and 0.5 mM leupeptin to prevent degradation of protein inside lysosomes and allow detection of mCherry signal. Incubate at 37 Â°C for 4-24 h. Incubation at 37 Â°C is accomplished through the use of a CO<sub>2</sub> dependent incubator. <br/><br/></li></ol></li><li>Image acquisition<ol class="npwtwo"><li>For imaging, the cells should be imaged under the appropriate conditions (<em>i.e.</em>, 37 Â°C and 5% CO<sub>2</sub>). If using a microscope system without CO<sub>2</sub>, the media should be replaced with appropriate CO<sub>2</sub> independent media supplemented with 0.5 mM leupeptin and 2 Î¼M E-64.</li><li>Ensure microscope is powered on and warmed up to 37 Â°C before starting.</li><li>For each set of experiments, calibrate settings such that the mean GFP and mCherry fluorescence intensities (I<sub>G/R</sub>) are approximately equal (<em>i.e.</em>, I<sub>GFP</sub>/I<sub>mCherry</sub> = 1). Calibration will depend on the microscope used, however in short, the laser, gain and speed should be set to have the background close to zero intensity while most of the signal for the fluorophores are not saturated. The ratio of the mean intensity for RFP over GFP for each cell should be close to 1 for the control cells. Ensuring that GFP and mCherry signals are approximately equal allows for optimal detection of changes in the fluorescence intensity ratio upon pexophagy induction. For each trial, all conditions should be imaged on the same day. Either 3D z-sectioning or a single slice can be used for quantification but ensure to keep the imaging method consistent throughout trials. Refer to Figure 3 for sample images. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191212/20191212013014_7465.jpg" width="178"/><br/><strong>Figure 3. HeLa cells transfected with RG construct.</strong> Control cells have been incubated in DMEM supplemented with E-64 and leupeptin as above. Starved cells were grown in HBSS for 12 h before imaging, also supplemented with E-64 and Leupeptin. Both cells are expressing GFP-mCherry-PEX26<sub>TM</sub>. Control cell (Top) shows almost equal GFP and mCherry pixel intensities. Cells where pexophagy has been induced by amino acid starvation (Bottom) show increased 'red-only' signal indicating peroxisomes have been targeted to lysosomes. Note, an increase in âred-onlyâ signal is more easily observable as a decrease in green fluorescent intensity. Red fluorescent intensity should not change. Scale bar = 10 Î¼m</li></ol></li></ol><p class="pptt" id="biaoti30665">### Data analysis</p><p class="ppzz"><em>Note: Please refer to Figure 3 for a detailed spreadsheet.</em> </p><ol class="npwtwok"><li>Using Volocity software or its equivalent, first determine the cell area by drawing a region of interest (ROI) around an individual cell expressing the RG construct (Figure 4A).</li><li>Next, identify peroxisomes by using the mCherry signal only (Figure 4B, column 1). In Volocity, this can be done in several different ways, however for most images with very high signal to noise in the ROI is the easiest. Most other imaging analysis software with similar algorithms can be used to find objects within Z-stack images. Measure the mean fluorescent intensity of GFP and mCherry signal per puncta as well as the volume of each puncta, and input into a spreadsheet (Figure 4B, columns 2-4).</li><li>Next, identify the proportion of lysosomally targeted peroxisomes (âred onlyâ peroxisomes) by calculating the ratio of mCherry to GFP fluorescence per puncta (Figure 4B, column 5). A puncta is considered âred-onlyâ if the mean mCherry fluorescent signal is at least 3 times higher than the mean GFP fluorescent signal. This is denoted in the spreadsheet as a value of 1 (lysosomally targeted) or 0 (not lysosomally targeted) (Figure 4B, column 6A). Using this information, the volume of lysosomally targeted peroxisomes per cell can be tabulated efficiently by Excel (Figure 4B, column 6B).</li><li>Add the volumes of all lysosomally targeted peroxisomes per cell (Figure 4B, column 7). Identify the area of all peroxisomes in the cell by adding the volume of all mCherry positive puncta (Figure 4B, column 8). To identify the proportion of lysosomally targeted peroxisomes per cell, divide the volume of the lysosomally targeted peroxisomes by the total peroxisome volume (Figure 4B, column 9).</li><li>Repeat this process for every cell, quantifying at least 50 cells per trial. Calculate an average of proportion of lysosomally targeted puncta per tested condition and plot on a graph.</li><li>Once a minimum of 3 independent trials has been completed, perform statistical analysis and graph. For a sample graph, refer to Figure 4C. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191212/20191212012702_2572.jpg" width="123"/><br/><strong>Figure 4. Sample cell ROI generation and calculation spreadsheet.</strong> A. ROI generation for cells imaged in Figure 2 (leftâcell incubated for 12 h in DMEM, rightâcell incubated for 12 h in HBSS). ROI is denoted by the white dotted line. Merged images are shown. B. Sample calculation spreadsheet. Quantification of 10 peroxisomes from each cell imaged in A is shown. Cell 1 was incubated in DMEM (control), while Cell 2 was incubated in HBSS (pexophagy induced) for 12 h. Please refer to the protocol for detailed column explanations. Note, red only peroxisomes (column 6) are those peroxisomes which have a mean red fluorescent intensity at least 3 times greater than the mean green fluorescent intensity. C. Sample data from B graphed.<br/></li></ol><p class="pptt" id="biaoti30666">### Notes</p><ol class="npwtwok"><li>This experiment must be imaged live, as fixation may alter the mCherry and GFP pixel intensity.</li><li>If additional staining is required, for example to determine if a construct is expressed, cells may be fixed following imaging, stained normally and re-imaged, to determine what proportion of cells expressing the RG construct are also expressing your construct of interest.</li><li>Laser intensity or gain must not be altered between images or samples within the same trial. <br/></li></ol><p class="pptt" id="biaoti30667">### Acknowledgments</p><p class="ppzz">Funding Sources: This work was supported by the Canada Institute of Health Research grants (PJT156196 to P.K. Kim and MOP-130281 to G.A. McQuibban) and Natural Sciences and Engineering Research Council of Canada Discovery grants (RGPIN-2015-04077 to P.K. Kim and RGPIN-2015-05969 to G.A. McQuibban). This protocol was adapted from our previous work (Deosaran <em>et al.</em>, 2013; Wang <em>et al.</em>, 2015; Riccio <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti30668">### Competing interests</p><p class="ppzz">There are no significant competing financial, professional, or personal interests that might have influenced the performance or presentation of the work described in this manuscript.</p><p class="pptt" id="biaoti30669">### References</p><ol class="npwtwok"><li>Allen, G. F., Toth, R., James, J. and Ganley, I. G. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24176932" target="_blank">Loss of iron triggers PINK1/Parkin-independent mitophagy.</a> <em>EMBO Rep</em> 14(12): 1127-1135.</li><li>Deosaran, E., Larsen, K. B., Hua, R., Sargent, G., Wang, Y., Kim, S., Lamark, T., Jauregui M., Law, K., Lippincott-Schwartz, J., Brech, A., Johansen, T. and Kim, P. K. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23239026" target="_blank">NBR1 acts as an autophagy receptor for peroxisomes.</a> <em>J Cell Sci</em> 126(Pt 4): 939-52.</li><li>Klionsky, D. J., Abdelmohsen, K., Abe, A., Abedin, M. J., Abeliovich, H., <em>et al.</em> (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26799652" target="_blank">Guidelines for the use and interpretation of assays for monitoring autophagy (3rd edition).</a> <em>Autophagy</em> 12(1): 1-222.</li><li>Kim, S. J., Syed, G. H., Khan, M., Chiu, W. W., Sohail, M. A., Gish, R. G. and Siddiqui, A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24733894" target="_blank">Hepatitis C virus triggers mitochondrial fission and attenuates apoptosis to promote viral persistence.</a> <em>Proc Natl Acad Sci USA</em> 111(17): 6413-6418.</li><li>Lahiri, V., Hawkins, W. D. and Klionsky, D. J. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30943392" target="_blank">Watch what you (self-) eat: Autophagic mechanisms that modulate metabolism.</a> <em>Cell Metab</em> 29(4): 803-826.</li><li>Morishita, H. and Mizushima, N. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31283377" target="_blank">Diverse cellular roles of autophagy.</a> <em>Annu Rev Cell Dev Biol</em> 35: 453-475.</li><li>Nazarko, T. Y., Ozeki, K., Till, A., Ramakrishnan, G., Lotfi, P., Yan, M. and Subramani, S. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24535825" target="_blank">Peroxisomal Atg37 binds Atg30 or palmitoyl-CoA to regulate phagophore formation during pexophagy.</a> <em>J Cell Biol</em> 204(4): 541-557.</li><li>Pankiv, S., Clausen, T. H., Lamark, T., Brench, A., Bruun, J. A., Outzen, H., Overvatn, A., Bjorkoy, G. and Johansen, T. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17580304" target="_blank">p62/SQSTM1 binds directly to ATG8/LC3 to facilitate degradation of ubiquitinated protein aggregates by autophagy.</a> <em>J Biol Chem</em> 282(33): 24131-24145.</li><li>Riccio, V., Demers, N., Hua, R., Vissa, M., Cheng, D. T., Strilchuk, A. W., Wang, Y., McQuibban, G. A. and Kim, P. K. (2019). <a href="http://jcb.rupress.org/content/218/3/798.abstract" target="_blank">Deubiquitinating enzyme USP30 maintains basal peroxisome abundance by regulating pexophagy.</a> <em>J Cell Biol</em> 218(3): 798-807.</li><li>Wang, Y., Serricchio, M., Jauregui, M., Shanbhag, R., Stoltz, T., Di Paolo, C. T., Kim, P. K. and McQuibban, G. A. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25915564" target="_blank">Deubiquitinating enzymes regulate PARK2-mediated mitophagy.</a> <em>Autophagy</em> 11(4): 595-606.</li></ol>