<body>
            <h1>cDNA Display Mediated Immuno-PCR (cD-IPCR): A Novel PCR-based Antigen Detection Method </h1>
            <p><strong>Authors:</strong> Chathuni Jayathilake, Takuya  Terai, Naoto Nemoto</p>
            <h2>Abstract</h2>
            <p>Immuno-PCR (IPCR) is a powerful method in antigen detection where a PCR-amplifiable DNA reporter is conjugated to a specific antibody or an aptamer for the target molecule. In the development and application of IPCR, successful conjugation of a protein (an antibody) with a reporter DNA becomes challenging. To address this issue, we recently demonstrated the feasibility of IPCR based on cDNA display, a 1:1 covalent complex of a polypeptide and its encoding cDNA at the single molecule level. The cDNA display molecule for IPCR is generated first by transcribing the DNA that encodes the detection antibody into an mRNA by <em>in vitro</em> transcription. A puromycin DNA linker is then ligated to the mRNA and then <em>in vitro</em> translation and reverse-transcription are performed to generate the cDNA display molecule. The molecule is then directly used in antigen detection and subsequent qPCR. This method can be applied to detect various antigens in biological samples, if sequences of their single-domain antibodies (VHHs) or peptide aptamers are known.</p>
            <p><strong>Keywords:</strong> Immuno-polymerase chain reaction, Immuno-detection, cDNA display, qPCR, VHH</p>
            <h2>Background</h2><p>Immuno-PCR, often abbreviated as IPCR, is an extremely powerful method in detection and quantification of low abundance biomarkers that exist in biological samples (<em>e.g.</em>, serum and urine). It acts as a bridge between immuno-reaction and signal amplification. In most cases, IPCR relies on the use of a detection antibody which has been conjugated to a reporter oligonucleotide, followed by the quantification of the analytes using real-time PCR of the reporter (Sano <em>et al.</em>, 1992). However, in the development and application of IPCR, the difficulty associated with appropriately conjugating antibodies to oligonucleotides has become the most challenging issue (van Buggenum <em>et al.</em>, 2016). Strategies available to conjugate the antibody and the DNA include non-covalent conjugation such as coupling via biotin-streptavidin (Sano <em>et al.</em>, 1992) and covalent conjugation using chemical crosslinkers (Hendrickson <em>et al.</em>, 1995). However, the tetrameric nature of streptavidin leads to the formation of heterogeneous DNA-antibody conjugates, which may decrease the reproducibility of IPCR. At the same time, conventional crosslinking chemistry reacts with the cysteine/lysine in the antibody, and the modification may compromise binding affinity of the antibody.<br/><br/>To avoid these problems, Guo <em>et al.</em> developed a new immuno-assay protocol with the aid of phage display: phage display mediated immuno-PCR (PD-IPCR) (Guo <em>et al.</em>, 2006). In PD-IPCR, an engineered recombinant phage particle which expresses a single-chain antibody for the analyte is used as a ready reagent for IPCR instead of a monoclonal antibody (mAb) and chemically bound DNA. However, in PD-IPCR, the specificity of the DNA-protein linkage is compromised due to the difficulty in controlling of the number of exposed engineered coat proteins on the phage surface. Further, since these bacteriophages are genetically engineered viruses, experiments must be performed under strict legal regulations, which may be a hurdle for real-world applications.<br/><br/>cDNA display is one of the simplest genotype-phenotype linking methods, where cDNA is covalently fused to its coding polypeptide (which can act as the affinity probe for the target) via a uniquely designed puromycin linker (Yamaguchi <em>et al.</em>, 2009; Mochizuki <em>et al.</em>, 2011). So far, we and others have successfully applied cDNA display for <em>in vitro</em> selection of high affinity peptides/proteins against various targets (Suzuki <em>et al.</em>, 2018; Terai <em>et al.</em>, 2019). Stimulated by PD-IPCR, we envisioned that cDNA display molecules can also be regarded as antibody-DNA conjugates that are used for IPCR. Thus, we recently demonstrated the feasibility of IPCR based on cDNA display (cD-IPCR, Anzai <em>et al.</em>, 2019). The cD-IPCR, which takes advantage of the structural characteristics of cDNA display, proved to work in detection of target proteins both in direct and sandwich-type formats. The cD-IPCR method offers several merits over the conventional IPCR and PD-IPCR, which include 1) based on a covalent homogeneous DNA-antibody conjugation, 2) non-viral, and 3) almost a ready to use reagent, 4) easy to perform by biologists without expertise in organic chemistry. The overall procedures for cD-IPCR are shown in Figure 1. Refer to the original paper "Anzai <em>et al.</em> (2019)" for further clarifications about the newly developed method. Here in this article, we describe step by step preparation of cDNA display molecule and its application in detecting analytes in a sandwich-type manner, which should be most general. To demonstrate the above steps, sandwich-type detection of green fluorescent protein (GFP) using anti-GFP VHH (Single variable domain on a heavy chain antibody) (Fridy <em>et al.</em>, 2014) and commercially available polyclonal anti-GFP antibody is explained here. Of course, the readers can detect other target proteins with this method if appropriate VHH or peptide aptamers as well as their immobilization antibodies are available. Current article takes advantage from our previously published "Anzai <em>et al.</em> (2019)" but different from original paper as this article includes all the experimental details that even novices could easily reproduce the experiment. We hope this article will help to ensure the high reproducibility of our newly developed protocol, and also lend a helping hand to researchers who are interested in the procedures and want to use themselves.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218002944_6087.jpg" width="300"/><br/><strong>Figure 1. Schematic representation of cDNA displays mediated immuno-PCR (cD-IPCR).</strong> A. Preparation of cDNA display (schematic representation). B. Schematic diagram of cD-IPCR (sandwich-type detection). A target protein in a biological sample is captured on solid phase using a capture antibody. After washing, cDNA display of a polypeptide/VHH that has affinity to the target is added. Unreacted display molecules are washed away, and resulting cDNA is quantified by qPCR.</p>
        </body>

<p class="pptt" id="biaoti30781">## Materials and Reagents</p><p class="ppzz"><em>Note: All reagents should be of molecular biology grade to avoid contamination of ribonuclease.</em> </p><ol class="npwtwok"><li>Polystyrene microtiter plate (MICROLON, 96 Well Single-Break Strip Plate, PS) (Greiner Bio-One, catalog number: 705071)</li><li>PCR tubes (End point PCR tubes; TreffLab Laboratory Consumables, catalog number: 96.09852.9.01, qPCR strip tubes; SSIBio, catalog number: 3248-00)</li><li>cnvK-rG Puromycin-linker, store at -20 Â°C<br/><em>Note: The components of cnvK-rG puromycin-linker can be obtained from custom DNA synthesis service. Please refer (Nemoto et al., 2018 or Terai et al., 2019) for the detailed protocol of formation.</em> </li><li>DNA oligos can be obtained from custom DNA synthesis service. Eurofins Genomics (Ota-ku, Tokyo, Japan), Tsukuba Oligo Service (Tsuchiura, Ibaraki, Japan), and Hokkaido System Science (Sapporo, Hokkaido, Japan), store at -20 Â°C</li><li>Nuclease-free ultra-pure distilled water (UPDW) (various suppliers)</li><li>PrimeSTAR HS DNA polymerase (Takara, catalog number: R010A), store at -20 Â°C</li><li>FavorPrep PCR Clean-Up Mini Kit (Favorgen), store at room temperature</li><li>RiboMAX<sup>TM</sup> Large-Scale RNA Production System T7 (Promega, catalog number: P1300), store at -20 Â°C</li><li>RNA Clean-Up Kit (Favorgen, catalog number: FAPCK001), store at 4 Â°C</li><li>Nuclease treated Rabbit Reticulocyte Lysate System (Promega, catalog number: L4960). Amino acid mixtures for translation are included. Store at -80 Â°C</li><li>RNasin<sup>Â®</sup> Ribonuclease Inhibitor (Promega; catalog number: N2111), store at -20 Â°C</li><li>Dynabeads MyOne streptavidin C1 (VERITAS; catalog number: DB65002), store at 4 Â°C</li><li>ReverTra Ace<sup>Â®</sup> (Toyobo, catalog number: TRT-101). 5x RT Buffer and 2.5 mM each dNTP mix are included. Store at -20 Â°C</li><li>His Mag Sepharose Ni (GE Healthcare, catalog number: 2896390)</li><li>RNase T1 (1,000 U/L) (Thermo Fisher Scientific, catalog number: EN0541)</li><li>RNase H (Takara, catalog number: 2150A, 10 U) and 10x NE buffer 2 (NEB)</li><li>DL-Dithiothreitol â¥ 99.0% (RT) (Sigma, catalog number: 4381)</li><li>Quick-Load 100 bp DNA Ladder (Biolabs, catalog number: N0467), NEB</li><li>SYBR<sup>TM</sup> Gold Nucleic Acid Gel Strain (Invitrogen, catalog number: S11494)</li><li>THUNDERBIRD SYBR qPCR Mix (Toyobo, catalog number: QPS-201)</li><li>10x PBS (Phosphate-buffered saline-) (Wako, catalog number: 163-25265)</li><li>Anti-GFP (Green Fluorescent Protein) pAb (MBL Life Science, catalog number: 598) </li><li>Skim milk (Wako, catalog number: 190-12865)</li><li>NaCl (Wako, catalog number: 191-01665)</li><li>Tris-HCl (Wako, catalog number: 208-14691)</li><li>KCl (Wako, catalog number: 163-03545)</li><li>MgCl<sub>2</sub> (Wako, catalog number: 136-03995)</li><li>EDTA (Invitrogen, catalog number: 15575-038)</li><li>Urea (Wako, catalog number: 217-01215)</li><li>Bromophenol Blue (BPB) (Wako, catalog number: 021-02911)</li><li>Xylene Cyanol (XC) (Wako, catalog number: 244-00461)</li><li>Sucrose (Wako, catalog number: 196-00015)</li><li>UltraPure DNase/RNase-Free Distilled Water (UPDW) (Invitrogen, catalog number: 10977-015)</li><li>Ammonium persulfate (Wako, catalog number: 012-20503)</li><li>Acrylamide (Nacalai Tesque, catalog number: 00807-05)</li><li>Tetramethylethylenediamine (TEMED) (Wako, catalog number: 205-06313)</li><li>Boric acid (Wako, catalog number: 021- 02195)</li><li>Tween20 (Sigma-Aldrich, catalog number: P9416)</li><li>Dibasic sodium phosphate (Wako, catalog number: 042-29445)</li><li>Imidazole (Wako, catalog number: 095-05392)</li><li>Glycine (Wako, catalog number: 077-00735)</li><li>Hydrochloric acid (HCl) (Sigma-Aldrich, catalog number: 320331)</li><li>Tris-borate-EDTA (TBE) Buffer (10x) (Invitrogen, catalog number: B52)</li><li>Tris-EDTA (TE) buffer (Invitrogen, catalog number: AM9849)</li><li>Tris base (Wako, catalog number: 011-20095)</li><li>SDS (Wako, catalog number: 194-13985)</li><li>APS (Wako, catalog number: 016-08021)</li><li>BIS (N, Nâ-Methylene-bis (acrylamide) (Wako, catalog number: 130-06031)</li><li>1x SYBR Gold Nucleic Acid Gel Strain (stored at 4 Â°C, see Recipes) </li><li>2x loading dye (see Recipes)</li><li>4% Denaturing urea polyacrylamide gel electrophoresis (PAGE) 10 ml (see Recipes)</li><li>5x TBE buffer 1 L (see Recipes)</li><li>2x Binding buffer (see Recipes)</li><li>Ni-NTA binding/wash buffer (see Recipes)</li><li>Ni-NTA elution buffer (see Recipes)</li><li>SDS-page gel (prepare and allow to set 2-3-hour prior to the experiment) (see Recipes)<br/>10 ml of 6% separating gel and 5 ml of 4% stacking gel</li><li>0.2 M Glycine-HCl buffer (see Recipes)</li><li>40% (w/v) Acrylamide (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30782">### Equipment</p><ol class="npwtwok"><li>Thermal cycler (Biometra, Model: TRIO48)</li><li>Biomolecular imager (GE Healthcare, Model: <a class="blackNoUnder" target="_blank"> Typhoon FLA9500 </a> )</li><li>NanoDrop Spectrophotometer (Thermo Fisher Scientific, Model: 1000 V3.3)</li><li>NanoDrop Spectrophotometer (Thermo Fisher Scientific, Model: 3300 V2.7, FITC) </li><li>Heat block (ANATECH, Model: cool stat 5200)</li><li>UV Crosslinker (UVP, Model: CL-1000)</li><li>Vortex mixer (IWAKI, Model: TM 2000)</li><li>Magnetic separator (Invitrogen, Model: 12320D)</li><li>Thermo block rotator (NISSIN, Model: SNP 24B)</li><li>StepOne Real-Time PCR System (Thermo Fisher Scientific, Model: 4376600)</li><li>High speed refrigerated microcentrifuge (Tomy Tech, Model: KITMAN MX-301)</li><li>Gel electrophoresis apparatus (PAGE/SDS-PAGE) (ATTO, Model: AE 6510)</li><li>Pipettes (Gilson Pipetman, Model P2- P1000)<br/></li></ol><p class="pptt" id="biaoti30783">### Software</p><ol class="npwtwok"><li>Quantity One 1-D Analysis Software (Bio-Rad, Version 4.6.6)</li><li>Primer Express (Thermo Fisher Scientific, Version 3.01)<br/></li></ol><p class="pptt" id="biaoti30784">### Procedure</p><ol class="npwone"><li>Synthesis of cnvK-rG puromycin-linker (Figure 2)<br/>Please refer (Nemoto <em>et al.</em>, 2018 or Terai <em>et al.</em>, 2019) for the detailed protocol of formation of the cnvK-rG puromycin-linker.<br/><br/><img alt="" class="layerphoto" height="125" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218012312_6313.jpg" width="300"/><br/><strong>Figure 2. Schematic diagram of puromycin-linker DNA.</strong> The puromycin-linker DNA comprises four parts: a photo cross-linking site for mRNA (cnvK), a primer region for reverse transcription, a biotin moiety for immobilization onto SA beads for purification, and a restriction site for release from SA beads. In addition, the linker has puromycin moiety and FITC moiety for detection.<br/><br/></li><li>Construction anti-GFP VHH DNA templates and their PCR amplification<ol class="npwtwo"><li>Prepare the Anti-GFP VHH coding DNA construct as follows. If you want to use other VHHs, please change âAnti-GFP VHHâ part. The DNA construct can be chemically synthesized as an artificial gene on a plasmid or overlap extension PCR from several fragments. Please ensure that there is no stop codon before cnvK NewYtag region.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>14-33<br/></td><td><span style="color:#E53333;">T7</span><br/></td></tr><tr><td>34-36<br/></td><td><span style="color:#666666;">5â² cap</span><br/></td></tr><tr><td>37-107<br/></td><td><span style="color:#9933E5;">Î©</span><br/></td></tr><tr><td>110-114 <br/></td><td><span style="color:#009900;">kozak</span><br/></td></tr><tr><td>118-498<br/></td><td><span style="color:#337FE5;">Anti-GFP VHH</span><br/></td></tr><tr><td>499-522<br/></td><td><span style="color:#B8D100;">GGGSGGGS</span><br/></td></tr><tr><td>523-540<br/></td><td><span style="color:#E56600;">His Tag</span><br/></td></tr><tr><td>541-549<br/></td><td><span style="color:#DFC5A4;">GGS</span><br/></td></tr><tr><td>550-571<br/></td><td><span style="color:#FFE500;">cnvK NewYtag</span> </td></tr></tbody></table>GATCCCGCGAAAT<span style="color:#E53333;">TAATACGACTCACTATAGGG</span><span style="color:#999999;">GAA</span><span style="color:#9933E5;">GTATTTTTACAACAATTACCAACAACAACAACAAACAACAACAACATTACATTTTACATTCTACA<br/>ACTACAA</span><span style="color:#009900;">GCCACC</span>ATG<span style="color:#337FE5;">GCCCAGGTGCAGCTGGTTGAAAGCGGTGGCCGTCTGGTGCAGGCGGGTGATAGCCTGCGTCTGAGCTGTGCCGCAAGCG<br/>GTCGCACCTTTAGCACCAGCGCCATGGCATGGTTTCGTCAGGCCCCGGGCCGTGAACGCGAATTTGTGGCGGCCATTACCTGGACCGTTGGTAACAC<br/>CATCCTGGGCGATAGCGTGAAAGGTCGTTTTACCATTAGCCGTGATCGCGCCAAAAACACCGTGGATCTGCAGATGGATAATCTGGAACCGGAAGATA<br/>CCGCGGTTTATTATTGTAGCGCCCGTAGCCGCGGTTATGTGCTGAGCGTTCTGCGCAGCGTTGATAGCTATGATTATTGGGGTCAGGGCACCCAGGTT<br/>ACGGTCAGC</span><span style="color:#B8D100;">GGCGGCGGCTCGGGCGGCGGTTCC</span><span style="color:#E56600;">CATCATCATCATCATCAT</span><span style="color:#DFC5A4;">GGCGGAAGC</span><span style="color:#FFE500;">AGGACGGGGGGCGGCG</span>T<span style="color:#FFE500;">GGAAA</span><span style="color:#FFE500;"></span><br/><br/></li><li>Perform PCR amplification to generate appropriate DNA concentrations as follows: Mix 2 Âµl of DNA sample (~50 ng), 1 Âµl of 10 ÂµM Forward primer, 1 Âµl of 10 ÂµM Reverse primer, 4 Âµl of 2.5 mM dNTP Mix, 10 Âµl of 5x PrimeSTAR buffer (Mg<sup>2+</sup>), 0.5 Âµl of PrimeSTAR HS DNA polymerase (2.5 U/Âµl), and UPDW up to 50 Âµl in PCR tube.</li><li>The PCR program is as follows: 98 Â°C for 1 min, 98 Â°C for 10 s, 64 Â°C for 5 s, 72 Â°C for 37 s (extension time is dependent on the DNA length), repeat 25 cycles of these three steps, followed by 72 Â°C for 2 min. Forward primer: 5â²-GATCCCGCGAAATTAATACGACTCACTATAGGG-3â and Reverse primer: 5â²-TTTCCACGCCGCCCCCCGTCCT-3â².</li><li>Analyze the amplification of DNA on 4% denaturing urea polyacrylamide gel electrophoresis (PAGE) using a gel imager as follows.<br/><em>Note: Ensure there are no sub bands during amplification and if so, adjust the PCR cycles or cut the correct gel band and purify the DNA.</em> </li><li>Mix 0.5 Âµl of DNA sample, 1.5 Âµl of UPDW and 2 Âµl of 2x loading buffer. Use 100 bp DNA Ladder as the marker (Recipe 1).<br/></li><li>Denature the DNA by heating at 85 Â°C for 5 min before loading.</li><li>Perform PAGE analyses for 20 min at 60 Â°C, 200 V using gels (1 mm thickness) containing 8 M urea (Recipe 2), with 0.5x TBE as a running buffer (Recipe 3). </li><li>Purify the DNA using FavorPrep DNA purification kit. Use UPDW as eluent. </li><li>Measure the DNA concentration using NanoDrop.<br/><br/></li></ol></li><li>Synthesis of cDNA display molecules<ol class="npwtwo"><li><em>In vitro</em> transcription<br/><ol class="npwthree"><li>Prepare the transcription reaction solution as follows: mix 4 Âµl of 5x T7 transcription buffer, 6 Âµl of 25 mM each rNTP mix [mix equal volumes of four individual 100 mM rNTPs (rATP, rCTP, rGTP, and rUTP)], 200 ng of the template DNA, 2 Âµl of T7 Enzyme Mix, and add nuclease-free water up to 20 Âµl.</li><li>ncubate at 37 Â°C for 2-4 h on a heat block.<br/><em>Note: The amount of mRNA will depend on the incubation time.</em><br/></li><li>Add 2 Âµl of RQ-1 RNase-free DNase and incubate at 37 Â°C for 15 min on a heat block.</li><li>Purify the transcript (mRNA) with RNA Clean-Up kit.<br/></li><li>Determine its concentration by absorbance at 260 nm using NanoDrop.<br/><em>Note: The concentration of mRNA should be over 2 pmol/Âµl and the ratio of 260/280 nm should be over 1.8.</em><br/></li><li>Store the mRNA at -80 Â°C until use.</li></ol></li><li>Photo-crosslinking between mRNA and cnvK-rG puromycin-linker<br/><ol class="npwthree"><li>Prepare the photo-crosslinking reaction solution as follows: mix 4 Âµl of 1 M NaCl, 4 Âµl of 0.25 M Tris-HCl (pH 7.5), 20 pmol of cnvK-rG puromycin-linker, 20 pmol of mRNA, and nuclease-free water up to 20 Âµl.<br/><em>Note: It is convenient to prepare working solution of the ligation product (mRNA-puromycin-linker) as 1 pmol/Âµl.</em><br/></li><li>Incubate at 90 Â°C for 1 min followed by lowering the temperature to 70 Â°C at a rate of 0.4 Â°C/s and incubate for 1 min and then cool down to 25 Â°C at a rate of 0.08 Â°C/s using a thermocycler.</li><li>Irradiate (in the same tube) with UV light at 365 nm for about 30 s using a UV cross linker.</li><li>Prepare the samples for gel electrophoresis the same as described above in Procedure B.<br/></li><li>Analyze the photo-crosslinked product on a 4% denaturing PAGE containing 8 M urea and visualize the FITC motif in Puromycin-linker using a fluorescence gel imager (Data analysis A).<br/><em>Note: Binding efficiency can be estimated by analyzing the FITC band intensity ratios of bound and unbound puromycin-linker.</em><br/>After that, stain the same gel with SYBR Gold (Recipe 9) for ~1 min, wash with water, and take an image.</li><li>Store the ligated sample at -80 Â°C until use. Keep 0.5 Âµl of the sample separately for subsequent SDS-PAGE analysis (Procedure D).<br/></li></ol></li><li><em>In vitro</em> translation (synthesis of mRNA-VHH fusion molecule)<br/><ol class="npwthree"><li>Prepare the translation mixture as follows: mix 6 pmol/6 Âµl of photo-crosslinked product, 35 Âµl of Rabbit Reticulocyte Lysate, 1 Âµl of Amino Acid Translation Mixture, 1 Âµl of RNasin<sup>Â®</sup> Ribonuclease Inhibitor, and nuclease-free water up to 50 Âµl.<br/><em>Note: As the working solution of the mRNA-Puromycin-linker is 1 pmol/Âµl, 6 Âµl equals to 6 pmol. The reaction volume should be less than 50 Âµl, because of higher translation efficiency. If the reaction needs to be scaled up, prepare 50 Âµl of the reaction solution in several test tubes.</em><br/></li><li>Incubate the mixture at 30 Â°C for 20 min on a heat block.</li><li>Add 24 Âµl of 3 M KCl and 6 Âµl of 1 M MgCl<sub>2</sub> to each tube. Incubate at 37 Â°C for 60 min.</li><li>Add 18 Âµl of 0.5 M EDTA and 98 Âµl (same volume to the mixture) of 2x binding buffer (Recipe 4). Incubate at 37 Â°C for 5 min.<br/></li><li>Collect 16.3 Âµl of translation reaction sample into a separate tube for subsequent SDS-PAGE analysis (Procedure D).<br/></li></ol></li><li>Immobilization of mRNA-VHH fusion molecule on streptavidin magnetic beads<br/><ol class="npwthree"><li>Take 10 Âµl suspension of streptavidin (SA) magnetic beads for 1 pmol of the photo-crosslinked product (<em>i.e.</em>, 60 Âµl for 6 pmol) and wash the beads twice with 200 Âµl of 1x binding buffer.<br/><em>Note: All washing steps should be performed at room temperature.</em><br/></li><li>Resuspend the beads with rest of the translation reaction sample.<br/><em>Note: If the reaction volume is too large for one tube, use several tubes with beads. It is not necessary to cut tips during pipetting beads.</em><br/></li><li>Incubate the mixture in a thermo block rotator at 25 Â°C for 30 min.<br/><em>Note: Keep the beads in suspension during the incubation to avoid the aggregation of beads.</em><br/></li><li>Place the tube on a magnetic separator for 1 min and collect 16.3 Âµl of supernatant into a separate tube for subsequent SDS-PAGE analysis (Procedure D).<br/></li><li>Discard the rest of the supernatant and wash the beads with 200 Âµl of 1x binding buffer for three times.</li></ol></li><li>Reverse transcription to form cDNA display molecules.<br/><ol class="npwthree"><li>Wash the mRNA-VHH fusion molecules immobilized on SA beads (from above procedures) with 100 Âµl of 1x RT Buffer.</li><li>Prepare the reverse transcription (RT) reaction solution for 60 Âµl of SA beads as follows: mix 10 Âµl of 5x RT Buffer, 25 Âµl of 2 mM dNTP mix, 14 Âµl of nuclease-free water, and 1 Âµl of ReverTra Ace<sup>Â®</sup> (100 U/Âµl).</li><li>Resuspend the SA beads in RT reaction solution and incubate at 42 Â°C for 30 min using a thermo block rotator.<br/><em>Note: Keep the beads in suspension during the incubation to avoid the aggregation of beads.</em> </li><li>Place the tube on a magnetic separator for 1 min and discard the supernatant, wash the beads twice with 200 Âµl of 1x binding buffer and once with 100 Âµl of 1x Ni-NTA binding/wash buffer (Recipe 5).<br/></li><li>Resuspend the above beads with 29.8 Âµl of 1x Ni-NTA binding/wash buffer and add 0.2 Âµl of RNase T1 (1,000 U/Âµl). Incubate using the rotator at 37 Â°C for 15 min.<br/><em>Note: Keep the beads in suspension during the incubation to avoid the aggregation of beads.</em><br/></li><li>Place the tube on a magnetic separator for 1 min and collect the supernatant containing cDNA display molecules.<br/></li><li>Save 2.7 Âµl of the aliquots for SDS-PAGE analysis (Procedure D).<br/></li></ol></li><li>His-tag affinity purification of cDNA display<br/><ol class="npwthree"><li>Take 20 Âµl of His Mag Sepharose Ni beads into a tube and wash twice with 100 Âµl of the 1Ã Ni-NTA binding/wash buffer.</li><li>Resuspend the beads in mRNA/cDNA-VHH fusion molecule solution (supernatant of RNase T1 cleavage) and incubate at 25 Â°C for 120 min using the rotator.<br/><em>Note: Adjust the incubation temperature accordingly with incubation time to prevent DNA degradation.</em><br/></li><li>Place the tube on a magnetic separator for 1 min and collect 2.7 Âµl of supernatant into a separate tube for subsequent SDS-PAGE analysis (Procedure D).</li><li>Discard the rest of the supernatant and wash the beads twice with 100 Âµl of 1x Ni-NTA binding/wash buffer.<br/></li><li>Resuspend the beads in 25 Âµl Ni-NTA elution buffer (Recipe 6) and incubate using the rotator at 25 Â°C for 20 min.<br/><em>Note: Elution volume can be adjusted as needed and can be used for immunoassay after appropriate dilutions.</em><br/></li><li>Place the tube on a magnetic separator for 1 min and collect the supernatant containing purified cDNA display molecules.<br/></li><li>Save 2.5 Âµl of the aliquots for SDS-PAGE analysis (Procedure D).</li></ol><br/></li></ol></li><li>Confirmation of cDNA display formation by SDS-PAGE<ol class="npwtwo"><li>Aliquots collected during the above process (Procedure C) correspond to 0.5 pmol of molecules each.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:500px;"><tbody><tr><td>(1) mRNA-puromycin linker <br/></td><td>0.5 Âµl<br/></td></tr><tr><td>(2) Input (mRNA-VHH fusion molecules after translation)<br/></td><td>16.3 Âµl<br/></td></tr><tr><td>(3) Supernatant of SA-beads immobilization <br/></td><td>16.3 Âµl<br/></td></tr><tr><td>(4) Elution of RNase T1 treatment (crude cDNA display)<br/></td><td>2.7 Âµl<br/></td></tr><tr><td>(5) Supernatant of Ni beads immobilization<br/></td><td>2.7 Âµl<br/></td></tr><tr><td>(6) Ni-NTA elution (purified cDNA display)<br/></td><td>2.5 Âµl</td></tr></tbody></table></li><li>Incubate sample (4), (5) and (6) with 0.2 Âµl RNase H, 1 Âµl 10x NE buffer 2 and nuclease-free water (adjust the final volume up to 10 Âµl) at 37 Â°C for 30 min to digest the RNA/cDNA duplex.</li><li>For sample (1), add 1.5 Âµl of UPDW and 2 Âµl of 2x SDS sample buffer</li><li>For samples (4), (5) and (6) after RNase H incubation, add 10 Âµl of 2x SDS sample buffer.</li><li>For samples (2) and (3), add 16.3 Âµl of 2x SDS sample buffer.</li><li>Incubate all samples for 5 min at 95 Â°C in a thermoblock before loading on a gel.</li><li>Analyze the samples in SDS-PAGE containing 8 M urea (4% stacking-6% separating gel, 0.02 A, 2 h) and visualize the FITC motif in puromycin-linker using a fluorescence scanner (Recipe 7).</li><li>Calculate the band intensity ratio of (1) and (6) to estimate the cDNA display formation efficiency (Data analysis B).<br/><br/></li></ol></li><li>Sandwich cD-IPCR for target protein (sfGFP) detection<ol class="npwtwo"><li>Coat a polystyrene microtiter plate with 100 Âµl of anti-GFP pAb [anti-GFP pAb in PBS (dilution = 1:450)] and keep at 4 Â°C overnight.</li><li>Wash the plates 3 times with 200 Âµl of PBS each.</li><li>Add 200 Âµl of 1% skim milk in PBS (w/v) and incubate for 2 h at 25 Â°C to block the wells.</li><li>Washed out the plates with 200 Âµl of PBS.</li><li>Prepare 10-fold serial dilutions of GFP protein in PBS from 100 Âµg/ml to 1 ng/ml as well as the negative control (no GFP). It is recommended that all the samples should be prepared in triplicate or more.<br/><em>Note: In this study, we used superfolder GFP (sfGFP) as the target protein. The amino acid sequence and the methods of expression and purification of the protein is described in the original paper (PÃ©delacq et al., 2006).</em> </li><li>Add 100 Âµl of each dilution of GFP protein to the wells and incubate at 25 Â°C for 2 h.</li><li>Wash with 200 Âµl of PBS, 8 times.</li><li>Dilute the cDNA display coding anti-GFP VHH (prepared as described in Procedure C) up to 65-fold with PBS-T buffer.<br/><em>Note: The dilution value should be changed and decided empirically depending on the target.</em> </li><li>Add 100 Âµl of cDNA display (corresponded to ~1.3 fmol/sample) to wells and incubate at 25 Â°C for 2 h.<br/><em>Note: It is not necessary to shake the plates during incubation.</em> </li><li>Wash with 200 Âµl of PBS-T, 8 times. (0.05% Tween 20 in PBS)</li><li>Elute the bound cDNA display with 100 Âµl of 0.2 M glycine/HCl buffer (pH 2.2) for 15 min at 25 Â°C (Recipe 8).</li><li>Neutralize the elute DNA with 20 Âµl of 1 M Tris-base solution and purify the cDNA display using a Favorprep kit.</li><li>Use 2-8 Âµl of the purified cDNA display as template for the following qPCR.</li><li>Perform the qPCR. The step program for PCR was as follows: 95 Â°C for 1 min, followed by 40 cycles of 95 Â°C for 15 s and 66 Â°C for 30 s. Forward primer: AACACCATCCTGGGCGATAG, Reverse primer: GTGTTTTTGGCGCGATCAC. The PCR mixture was prepared as follows: 10 Âµl of 2x reaction buffer (including enzyme and dNTP), 0.6 Âµl of each primer (10 ÂµM), 2 Âµl of template, 0.4 Âµl of reference dye (ROX) and 6.4 Âµl of water.<br/><em>Note: These primers are specific to anti-GFP VHH. So, if another VHH or peptide aptamer is used, design an appropriate primer set using software such as Primer Express 3.01.</em> </li><li>When performing qPCR, use anti-GFP VHH DNA to prepare the standard curve (100 pM, 1 pM, 10 fM,100 aM), and include a primer only control to verify the quality of amplification.</li><li>Calculate the difference of Ct values between the samples and the negative control containing no GFP (Data analysis C).<br/></li></ol></li></ol><p class="pptt" id="biaoti30785">### Data analysis</p><ol class="npwone"><li>Photo-crosslinking of mRNA and cnvK-rG puromycin-linker<ol class="npwtwo"><li>Save the image data and open the file with Quantity One 1-D Analysis Software. Representative images of a PAGE gel of photo-ligation samples are shown in Figure 3. <br/><em>Note: The digital images should not be over-exposed for optimal quantification.</em><br/><br/><img alt="" class="layerphoto" height="289" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218020106_0665.jpg" width="300"/><br/><strong>Figure 3. Visualization of photo-crosslinked product on denaturing PAGE using the FITC motif in puromycin-linker (right) and after SYBR Gold staining (left)</strong><br/><br/></li><li>Confirm the position of ligated mRNA (linker + mRNA, it should appear at slightly higher position than intact mRNA) and that no sub band in SYBR Gold-stained gel.</li><li>Quantify the band intensity of ligated mRNA and unbound linker in FITC-imaged gel. If more than about 80% of the linker is ligated, proceed to the next step (<em>in vitro</em> translation). If not, change the stoichiometry of mRNA and linker (in the range between 1:2 and 2:1) and perform photo-ligation again.<br/><em>Note: It is recommended to have more than 80% of the linker to be ligated to mRNA because remaining free linkers in the protein translation solution may end up inefficient cDNA display formation.</em><br/><br/></li></ol></li><li>Formation of cDNA display<ol class="npwtwo"><li>Visualize the gel of SDS-PAGE with FITC filter set. Save the data and open the file with Quantity One 1-D Analysis Software. A representative image is shown in Figure 4A. Please note that it corresponds to the gel of a different model protein, because formation efficiency of anti-GFP VHH is low and hard to analyze.<br/><em>Note: If the readers want to check the data of anti-GFP VHH, please refer to Anzai et al., 2019.</em> </li><li>Calculate the band intensity percentage of each band (with appropriate background subtraction) in the gel using the software. Use âvolume rectangleâ function to indicate regions of interest and âvolume analysis reportâ function to show the results. Since all samples are corresponding to the same amount of moles, you can directly compare the band intensities with each other. Band intensities of a model protein is given in Figure 4B for better understanding.<br/><em>Note: âBackgroundâ is the free gel space without any band.</em> </li><li>The formation efficiency of cDNA display (<em>i.e.</em>, yield of cDNA display from mRNA-linker) is the percentage value of band intensity ratio between sample (1) and (6). For example, when we compare the (1) mRNA-puromycin linker (whose intensity occupies 65.55% of all bands) and (6) Ni-NTA elution (purified cDNA display, 5.15%) intensities, the cDNA display formation efficiency is calculated to be 7.85%. Generally, the formation efficiencies of cDNA display for small polypeptides are approximately 5-25%.<br/><br/><img alt="" class="layerphoto" height="201" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218020132_8204.jpg" width="300"/><br/><strong>Figure 4. Confirmation of cDNA display formation of VHH.</strong> A. Example of an image of SDS-PAGE gel using FITC filter set. Upper bands are responsible for the correctly formed cDNA display. Note that the lower band corresponds to the mRNA-linker that was not conjugated to a peptide. Red rectangles indicate the ROIs used for intensity calculation. B. Corresponding band intensities calculated from data of Figure 4A. (1) mRNA-puromycin linker (65.55%), (2) Input (mRNA-VHH fusion molecules after translation) (21.88%), (3) Supernatant of SA-beads immobilization (2.59%), (4) Elution of RNase T1 treatment (crude cDNA display) (6.49%), (5) Supernatant of Ni beads immobilization (0.32%), (6) Ni-NTA elution (purified cDNA display) (5.15%).<br/><br/></li></ol></li><li>Sandwich CD-IPCR detection of target protein GFP<ol class="npwtwo"><li>After performing a qPCR experiment, the software attached to the instrument automatically prepare amplification plot and calculate the Ct (threshold cycle) value of each sample (Figure 5). Representative data of Ct values are shown in Table 1.<br/><br/><strong>Table 1. Ct values from qPCR reaction</strong><br/><img alt="" class="layerphoto" height="285" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218020201_5285.png" width="700"/><br/><br/><img alt="" class="layerphoto" height="404" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218020230_9539.jpg" width="300"/><br/><strong>Figure 5. Standard curve and amplification plot for the qPCR reaction</strong><br/><br/></li><li>Calculate the ÎCt values by substracting the Ct value of 0 Âµg/ml (no analyte) from sample Ct value for each GFP concentration (see Table 1).</li><li>Compare the ÎCt values statistically using <em>t</em>-test. The ÎCt value is gradually increased with the GFP concentration (from 0.01 Âµg/ml to 100 Âµg/ml), and 0.01 Âµg/ml is appeared as the lowest detected concentration according to the statistical analysis (Figure 6).<br/><br/><img alt="" class="layerphoto" height="174" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191218/20191218020253_5901.jpg" width="300"/><br/><strong>Figure 6. ÎCt values for sandwich CD-IPCR detection of target protein GFP.</strong> *** indicates <em>P</em> &lt; 0.05, ** indicates <em>P</em> &lt; 0.001. n = 3 (error bars indicate S.D.).<br/></li></ol></li></ol><p class="pptt" id="biaoti30786">### Recipes</p><ol class="npwtwok"><li>2x Loading dye (store at room temperature)<br/>Urea 24 g<br/>1% BPB 2.5 ml<br/>1% XC 2.5 ml<br/>0.5 M EDTA 0.1 ml<br/>10x TBE buffer 1 ml<br/>Sucrose 4 g<br/>Up to 50 ml by UPDW</li><li>4% Denaturing urea polyacrylamide gel electrophoresis (PAGE) 10 ml (prepare and allow to set 2-3-h prior to the experiment)<br/>4.8 g 8 M Urea<br/>1 ml 5x TBE buffer<br/>1 ml 40% Polyacrylamide and UPDW up to 10 ml<br/>Microwave the content for proper dissolving and let cool<br/>Add 25 Î¼l of 20% Ammonium Persulfate (APS) and 10 Î¼l of Tetramethylethylenediamine (TEMED) immediately before pouring into gel mold</li><li>5x TBE buffer 1 L (store at room temperature)<br/>54 g of Tris base<br/>27.5 g of Boric acid<br/>20 ml of 0.5 M EDTA (pH 8.0)<br/>The 0.5x working solution is 45 mM Tris-borate/1 mM EDTA</li><li>2x Binding buffer (store at 4 Â°C)<br/>20 mM Tris-HCl, pH 8.0<br/>2 mM EDTA<br/>2 M NaCl<br/>0.2% Tween20<br/>Prepare with nuclease-free water</li><li>Ni-NTA binding/wash buffer (store at 4 Â°C)<br/>20 mM Sodium phosphate dibasic, pH 7.4, 50 mM Tris-Cl (pH 8.0)<br/>0.5 M NaCl<br/>5 mM Imidazole<br/>0.05% Tween20</li><li>Ni-NTA elution buffer (store at 4 Â°C)<br/>20 mM Sodium phosphate dibasic, pH 7.4, 50 mM Tris-Cl (pH 8.0)<br/>500 mM NaCl<br/>250 mM Imidazole<br/>0.05% Tween20</li><li>SDS-page gel (prepare and allow to set 2-3-hour prior to the experiment)<br/><strong>10 ml of 6% separating gel</strong><br/>4.8 g of urea (final 8 M)<br/>1.5 ml of acrylamide (40 %)<br/>2.5 ml of 1.5 M Tris (pH 8.8)<br/>100 Âµl of SDS (10%)<br/>25 Âµl of APS (20%)<br/>5 Âµl of TEMED<br/>Up to 10 ml with UPDW<br/><strong>5 ml of 4% stacking gel</strong><br/>0.5 ml of acrylamide (40 %)<br/>0.5 ml of 1 M Tris (pH 6.8)<br/>50 Âµl of SDS (10%)<br/>15 Âµl of APS (10%)<br/>5 Âµl of TEMED<br/>Up to 5 ml with UPDW</li><li>0.2 M Glycine-HCl buffer (pH 2.2) (store at room temperature)<br/>25 ml of 0.2 M Glycine (15.01 g in 1 L distilled water)<br/>22 ml of 0.2 M HCl<br/>Up to total of 100 ml distilled water</li><li>1x SYBR Gold Nucleic Acid Gel Strain (store at 4 Â°C)<br/>5 Âµl of 10,000x SYBR<sup>TM</sup> Gold Nucleic Acid Gel Stain<br/>50 ml of 1x Tris-EDTA (TE) buffer</li><li>40% acrylamide<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:350px;"><tbody><tr><td>Compound<br/></td><td><strong>for PAGE</strong><strong></strong><br/></td><td><strong>for SDS-PAGE</strong><strong></strong><br/></td></tr><tr><td>Acrylamide<br/></td><td>190 g<br/></td><td>194.8 g<br/></td></tr><tr><td>Bis-acrylamide<br/></td><td>10 g<br/></td><td>5.2 g<br/></td></tr><tr><td>UPDW<br/></td><td>500 ml<br/></td><td>500 ml<br/></td></tr></tbody></table></li></ol><p class="pptt" id="biaoti30787">### Acknowledgments</p><p class="ppzz">This work was in part supported by KAKENHI (15H05723 and 17K19471 to N.N., 16J01710, 18H04535, and 18H02103 to T.T.). T.T. was also supported by a JSPS fellowship. This protocol was adapted from our recent article (Anzai <em>et al.</em>, 2019). We especially thank Mr. Hiroki Anzai and Mr. Takeru Suzuki for conducting the parts of the original work and also other members in Nemoto laboratory, Saitama University for valuable advice and support on this work.</p><p class="pptt" id="biaoti30788">### Competing interests</p><p class="ppzz">The authors have filed a patent for cD-IPCR.</p><p class="pptt" id="biaoti30789">### References</p><ol class="npwtwok"><li>Anzai, H., Terai, T., Jayathilake, C., Suzuki, T. and Nemoto, N. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31028717" target="_blank">A novel immuno-PCR method using cDNA display.</a> <em>Anal Biochem</em> 578: 1-6.</li><li>Fridy, P. C., Li, Y., Keegan, S., Thompson, M. K., Nudelman, I., Scheid, J. F., Oeffinger, M., Nussenzweig, M. C., FenyÃ¶, D., Chait, B. T. and Rout, M. P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25362362" target="_blank">A robust pipeline for rapid production of versatile nanobody repertoires.</a> <em>Nat Methods</em> 11(12): 1253-1260.</li><li>Guo, Y. C., Zhou, Y. F., Zhang, X. E., Zhang, Z. P., Qiao, Y. M., Bi, L. J., Wen, J. K., Liang, M. F. and Zhang, J. B. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16682441" target="_blank">Phage display mediated immuno-PCR.</a> <em>Nucleic Acids Res</em> 34(8): e62. </li><li>Hendrickson, E. R., Truby, T. M., Joerger, R. D., Majarian, W. R. and Ebersole, R. C. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7885849" target="_blank">High sensitivity multianalyte immunoassay using covalent DNA-labeled antibodies and polymerase chain reaction.</a> <em>Nucleic Acids Res</em> 23(3): 522-529.</li><li>Mochizuki, Y., Biyani, M., Tsuji-Ueno, S., Suzuki, M., Nishigaki, K., Husimi, Y. and Nemoto, N. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21766868" target="_blank">One-pot preparation of mRNA/cDNA display by a novel and versatile puromycin-linker DNA.</a> <em>ACS Comb Sci</em> 13(5): 478-485. </li><li>Nemoto, N., Kumachi, S. and Arai, H. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30196502" target="_blank"><em>In vitro</em> selection of single-domain antibody (VHH) using cDNA display.</a> <em>Methods Mol Biol</em> 1827: 269-285. </li><li>PÃ©delacq, J. D., Cabantous, S., Tran, T., Terwilliger, T. C. and Waldo, G. S. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16369541" target="_blank">Engineering and characterization of a superfolder green fluorescent protein.</a> <em>Nat Biotechnol</em> 24(1): 79-88.</li><li>Sano, T., Smith, C. L. and Cantor, C. R. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1439758" target="_blank">Immuno-PCR: very sensitive antigen detection by means of specific antibody-DNA conjugates.</a> <em>Science</em> 258(5079): 120-122.</li><li>Suzuki, T., Mochizuki, Y., Kimura, S., Akazawa-Ogawa, Y., Hagihara, Y. and Nemoto, N. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30119893" target="_blank">Anti-survivin single-domain antibodies derived from an artificial library including three synthetic random regions by <em>in vitro</em> selection using cDNA display.</a> <em>Biochem Biophys Res Commun</em> 503(3): 2054-2060. </li><li>Terai, T., Anzai, H. and Nemoto, N. (2019). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2515554/" target="_blank">Selection of peptides that associate with Dye-conjugated solid surfaces in a pH-dependent manner using cDNA display.</a> <em>ACS Omega</em> 4(4): 7378-7384</li><li>van Buggenum, J. A. G. L., Gerlach, J. P., Eising, S., Schoonen, L., van Eijl, R. A. P. M., Tanis, S. E. J., Hogeweg, M., Hubner, N. C., van Hest, J. C., Bonger, K. M. and Mulder, K. W. (2016). <a href="https://www.nature.com/articles/srep22675" target="_blank">A covalent and cleavable antibody-DNA conjugation strategy for sensitive protein detection via immuno-PCR.</a> <em>Sci Rep</em> 6: 22675.</li><li>Yamaguchi, J., Naimuddin, M., Biyani, M., Sasaki, T., Machida, M., Kubo, T., Funatsu, T., Husimi, Y. and Nemoto N. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19528071" target="_blank">cDNA display: a novel screening method for functional disulfide-rich peptides by solid-phase synthesis and stabilization of mRNA-protein fusions.</a> <em>Nucleic Acids Res</em> 37(16):e108.<br/></li></ol>