<body>
            <h1>Analysis of Monosaccharides from <em>Arabidopsis</em> Seed Mucilage and Whole Seeds Using HPAEC-PAD</h1>
            <p><strong>Authors:</strong> Gillian H. Dean, Kresimir  Sola, Faride Unda, Shawn D. Mansfield, George W. Haughn</p>
            <h2>Abstract</h2>
            <p><em>Arabidopsis</em> seed coat epidermal cells deposit a significant quantity of mucilage, composed of the cell wall components pectin, hemicellulose, and cellulose, into the apoplast during development. When mature seeds are hydrated, mucilage extrudes to form a gelatinous capsule around the seed. Determining the monosaccharide composition of both extruded mucilage and whole seeds is an essential technique for characterizing seed coat developmental processes and mutants with altered mucilage composition. This protocol covers growth of plants to produce seeds suitable for analysis, extraction of extruded mucilage using water and sodium carbonate (used for mutants with impaired mucilage release), and extraction of alcohol insoluble residue (AIR) from whole seeds. The prepared polysaccharides are then hydrolyzed using sulfuric acid, which hydrolyses all polysaccharides including cellulose. Sensitive and reproducible quantification of the resulting monosaccharides is achieved using high-performance anion exchange chromatography coupled with pulsed amperometric detection (HPAEC-PAD).</p>
            <p><strong>Keywords:</strong> <em>Arabidopsis</em>, Seed mucilage, Seed alcohol insoluble residue (AIR), Monosaccharide analysis, HPAEC-PAD</p>
            <h2>Background</h2><p>In <em>Arabidopsis</em>, fertilization of the ovule initiates growth and differentiation of the ovule integuments into the seed coat (Beeckman <em>et al.</em>, 2000; Western <em>et al.</em>, 2000; Windsor <em>et al.</em>, 2000). During differentiation, the outermost layer of integument cells begins to secrete large quantities of pectinaceous mucilage into the apoplastic space. This secretion is targeted to the point where the outer primary cell wall meets the radial cell wall, and leads to the formation of a donut-shaped pocket of mucilage surrounding a column of cytoplasm. After mucilage secretion is complete, the cytoplasmic column is replaced by a cellulosic secondary cell wall called the columella and the cells undergo programmed cell death prior to seed desiccation. When mature <em>Arabidopsis</em> seeds are hydrated, the mucilage expands and ruptures the primary cell wall. The extruded mucilage forms two distinct layers, a non-adherent layer that is easily detached, and an adherent layer that is strongly attached to the seed surface (reviewed in Å ola <em>et al.</em>, 2019a). Mucilage polysaccharides include pectin, hemicellulose, and cellulose. Ninety percent of the mucilage is pectin, of which 90% is unbranched Rhamnogalacturonan I (RG-I; reviewed in Voiniciuc <em>et al.</em>, 2015).<br/> <br/>The synthesis of seed coat mucilage is not necessary for plant viability under laboratory conditions (Western <em>et al.</em>, 2001). In addition, seed mucilage is readily accessible in large quantities. This has allowed isolation of viable mutants and development of transgenic lines with altered seed coat mucilage. The analysis of these mutants and transgenic lines can be used to understand the biosynthesis, deposition, regulation and function of mucilage polysaccharides and serves as a model system for studying various aspects of cell wall biology (Arsovski <em>et al.</em>, 2010; Haughn and Western, 2012; Western, 2012; North <em>et al.</em>, 2014; Francoz <em>et al.</em>, 2015; Voiniciuc <em>et al.</em>, 2015; Griffiths and North, 2017; Francoz <em>et al.</em>, 2018; Golz <em>et al.</em>, 2018; Å ola <em>et al.</em>, 2019a).<br/> <br/>An essential technique for analyzing mucilage is rapid, reliable, and simple quantification of monosaccharides. We have developed an HPAEC-PAD technique that has been employed successfully for over a decade (Dean <em>et al.</em>, 2007; Huang <em>et al.</em>, 2011; Mendu <em>et al.</em>, 2011; Voiniciuc <em>et al.</em>, 2013; Griffiths <em>et al.</em>, 2014; Shi <em>et al.</em>, 2019; Å ola <em>et al.</em>, 2019b). Our protocol for seed coat mucilage extraction includes modifications for mutants such as <em>mucilage modified2</em> (<em>mum2</em>) that produce mucilage with altered properties that impair hydration and extrusion (Dean <em>et al.</em>, 2007; Macquet <em>et al.</em>, 2007). We also established a method for preparation of whole seed alcohol insoluble residue (AIR). As previous analysis in our lab has shown that much of the rhamnose (Rha) and galacturonic acid (GalA) in whole seed is derived from mucilage RG-I (Western <em>et al.</em>, 2001), this analysis can be used to determine whether the total amount of mucilage synthesized is altered. The release of monosaccharides from mucilage polysaccharides and whole seed AIR is based on the secondary acid hydrolysis derived from the Klason procedure (Coleman <em>et al.</em>, 2009). This protocol uses sulfuric acid instead of the trifluoracetic acid (TFA) used by other mucilage analysis protocols (<em>e.g.</em>, Voiniciuc and GÃ¼nl, 2016) to improve digestion of the cellulose in seed coat mucilage. Analysis of the resulting monosaccharides by HPAEC-PAD permits sensitive and reproducible detection of both neutral and acidic sugars. This protocol could also be adapted for use in other mucilage-producing species, including Plantago and flax (Western, 2012; Yang <em>et al.</em>, 2012; Phan and Burton, 2018).</p>
        </body>

<p class="pptt" id="biaoti30717">## Materials and Reagents</p><ol class="npwtwok"><li>35 mm Petri dishes (Thermo Fisher Scientific, catalog number: FB0875712)</li><li>Serological pipettes, 50 ml</li><li>1 L autoclavable bottles for making and storing medium</li><li>Plastic wrap</li><li>Micropore tape (3M, catalog number: 1530-0)</li><li>Plant pots, 12 cm</li><li>Forceps</li><li>Bamboo barbeque skewers, 30 cm</li><li>Aluminum foil</li><li>Small sieve OR Whatman<sup>TM</sup> 3MM Chr Chromatography Paper (GE Healthcare, catalog number: 3030917) for removing chaff from seeds</li><li>2 ml boil-proof microcentrifuge tubes (VWR International, catalog number: 76332-080)</li><li>1.5 ml microcentrifuge tubes (VWR International, catalog number: 87003-294)</li><li>15 ml sterile polypropylene conical centrifuge tubes (<em>e.g.</em>, Corning Life Sciences, catalog number: 431470)</li><li>50 ml sterile polypropylene conical centrifuge tubes (<em>e.g.</em>, Corning Life Sciences, catalog number: 431472)</li><li>Plastic rack for 1.5 ml and 2 ml tubes</li><li>Plastic Pasteur pipettes, 5.8 ml (VWR International, catalog number: 612-4494)</li><li>Plastic box to contain tubes on orbital shaker</li><li>Tips for micropipettes</li><li>Chemical resistant gloves (<em>e.g.</em>, VWR International nitrile examination gloves, catalog number: 82026-426)</li><li>Glass microscope slides with cavity (<em>e.g.</em>, Ted Pella, catalog number: 260241)</li><li>Face shield</li><li>Safety glasses</li><li>Ice bucket</li><li>Bubble wrap</li><li>Small box for vortexing samples (A tip box lid is ideal)</li><li>12 ml 16 x 100 mm KIMAX tubes with PTFE-faced rubber lined caps (VWR International, catalog number: SCERSP45066A-16100)</li><li>Test tube brush for KIMAX tubes</li><li>Porcelain pestles and mortars, 50 ml capacity (<em>e.g.</em>, Coorstek, catalog number: 60310)</li><li>2 mm and 1 mm zirconia (zirconium dioxide) beads</li><li>2 ml screw-capped tubes (<em>e.g.</em>, Precellys 2 ml Empty Reinforced Tubes with O-ring, catalog number: 9000538)</li><li>UV syringe filters, 4 mm diameter, nylon, 0.45 Î¼m pore size (Chromatographic Specialties, catalog number: CSUV04N451)</li><li>1 ml BD tuberculin syringes (VWR International, catalog number: 309659)</li><li>Kimwipes (Kimtech, catalog number: 34120)</li><li>200 Î¼l screw top microvials (Chromatographic Specialties, catalog number: C581311)</li><li>Open top caps for HPLC vials (Chromatographic Specialties, catalog number: C223711M)</li><li>8 mm, 0.060â natural PTFE/silicone septa (Chromatographic Specialties, catalog number: C243M)</li><li>Dry ice</li><li>Ruthenium red, for microscopy (Sigma-Aldrich, catalog number: 00541)</li><li>Sodium carbonate, anhydrous (Thermo Fisher Scientific, catalog number: S263)<br/></li><li>Sodium bicarbonate (Baking soda)</li><li><em>meso</em>-Erythritol (Sigma-Aldrich, catalog number: E7500)</li><li>L-(-)-Fucose (Sigma-Aldrich, catalog number: F2252)</li><li>L-(+)-Arabinose (Sigma-Aldrich, catalog number: A3256)</li><li>L-Rhamnose (Sigma-Aldrich, catalog number: R3875)</li><li>D-(+)-Galactose (Sigma-Aldrich, catalog number: G6404)</li><li>D-(+)-Glucose (Sigma-Aldrich, catalog number: G8270)<br/></li><li>D-(+)-Mannose (Sigma-Aldrich, catalog number: M2069)</li><li>D-(+)-Xylose (Sigma-Aldrich, catalog number: X3877)</li><li>D-(+)-Galacturonic Acid (Sigma-Aldrich, catalog number: 48280)</li><li>Sulfuric Acid, Certified ACS Plus (Thermo Fisher Scientific, catalog number: A300-212)</li><li>95% ethanol</li><li>70% ethanol (v/v) in ultrapure water</li><li>HPLC-grade methanol (Sigma-Aldrich, catalog number: 34885)</li><li>80% HPLC-grade methanol (v/v) in ultrapure water</li><li>HPLC-grade acetone (Sigma-Aldrich, catalog number: 439126)</li><li>Sodium hydroxide solution (50% w/w Certified NaOH, Thermo Fisher Scientific, catalog number: SS254-4)</li><li>Anhydrous sodium acetate (CH<sub>3</sub>COONa, Thermo Fisher Scientific, catalog number: S210-500)</li><li>Ultrapure water (18.2 MÎ© cm at 25 Â°C)</li><li>Agar (Thermo Fisher Scientific, catalog number: BP1423)</li><li>1 M KOH for adjusting pH of medium</li><li>Sunshine Mix #5 (Sungro) or growth medium of your choice (see Notes)</li><li>AT minimal medium, with or without agar (see Recipes)</li><li>20 mM sodium carbonate solutions (see Recipes)</li><li>Ruthenium red solution (see Recipes)</li><li>100 mM monosaccharide stock solution (see Recipes), stored at -20 Â°C</li><li>72% (w/w) sulfuric acid (see Recipes)</li><li>Dilution series for monosaccharide standards (see Recipes)</li><li><em>meso</em>-Erythritol solution, 5 mg/ml (see Recipes), stored at -20 Â°C</li><li>1 M NaOH (HPAEC-PAD eluent) (see Recipes)</li><li>200 mM NaOH (HPAEC-PAD eluent) (see Recipes)</li><li>1 M sodium acetate (HPAEC-PAD eluent) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30718">### Equipment</p><ol class="npwtwok"><li>Micropipettes for volumes ranging from 5 Î¼l to 1 ml</li><li>5 ml, 10 ml, 100 ml, 1.5 L and 2 L volumetric flasks</li><li>100 ml beakers</li><li>3 x 2 L vacuum side arm flasks with stoppers and vacuum tubing for degassing HPAEC eluents</li><li>pH meter</li><li>Magnetic stirrer and stir bars</li><li>Autoclave</li><li>Growth chamber with light level and temperature controls (for plates, and growth of plants to maturity)</li><li>Analytical balance</li><li>Microcentrifuge</li><li>Lab balance</li><li>Vortex mixer with 3-inch platform (Scientific Industries, Vortex-Genie 2, catalog number: SI-0236)</li><li>Orbital shaker (New Brunswick Scientific, Innova 2000 platform shaker)</li><li>Dissecting or compound microscope with camera (<em>e.g.</em>, Zeiss Axioscop2 with Leica DFC450C and Leica application suite 4.2)</li><li>Fume hood</li><li>Reacti-Vap evaporator (Thermo Fisher Scientific, catalog number: TS-18825) with Reacti-Therm heating module (Thermo Fisher Scientific, catalog number: TS-18822)</li><li>Nitrogen (N<sub>2</sub>) gas (Praxair, NI M-T), connected to Reacti-Vap</li><li>Ice machine </li><li>Access to vacuum line (<em>e.g.</em>, integral building vacuum) for degassing HPAEC-PAD solutions</li><li>Ion chromatography (IC): Dionex DX600 system equipped as follows:<ol class="npwthree"><li>AS50 autosampler</li><li>GP50 gradient pump</li><li>ED50 electrochemical detector</li><li>CarboPac PA1 standard bore guard (4 x 50 mm) (Thermo Fisher Scientific, catalog number: 043096)</li><li>CarboPac PA1 analytical column (4 x 250 mm) (Thermo Fisher Scientific, catalog number: 035391)</li></ol></li></ol><p class="pptt" id="biaoti30719">### Software</p><ol class="npwtwok"><li>Chromeleon 6.8 chromatography data system software (Thermo Fisher Scientific, Dionex)</li><li>Python 2.7 (<a href="https://www.python.org/download/releases/2.7/" target="_blank">https://www.python.org/download/releases/2.7/</a>)</li><li>Scientific Python (SciPy) package (<a href="https://www.scipy.org/install.html" target="_blank">https://www.scipy.org/install.html</a>)<br/></li><li>Notepad ++ (Don Ho, <a href="https://notepad-plus-plus.org/" target="_blank">https://notepad-plus-plus.org/</a>)</li><li>Microsoft Excel (Microsoft)<br/></li></ol><p class="pptt" id="biaoti30720">### Procedure</p><ol class="npwone"><li>Growth of seeds for HPAEC analysis<br/>Growing seeds correctly for HPAEC analysis is critical as mucilage release is impaired if seeds do not complete development. It is very important to use standard growth conditions, grow wild type and mutants together, and water until seed development is complete as described below. In addition, see the Notes for details on preparation of biological replicates.<ol class="npwtwo"><li>Sow seeds of all genotypes onto <em>Arabidopsis</em> AT medium with agar in Petri dishes. Seeds do not need to be sterilized as this medium does not contain sugar.</li><li>Seal plates with micropore tape.</li><li>Place in growth chamber 20 Â°C under continuous light (100-150 Î¼mol/(s m<sup>2</sup>) photosynthetically active radiation, PAR).</li><li>Transplant the seedlings after 7-10 days.<ol class="npwthree"><li>Prepare one 12 cm pot of soil mix for each genotype (see Notes).</li><li>Add 30 ml of AT medium without agar to the top of the soil.</li><li>Use forceps to transplant 8 seedlings from each genotype into one pot of soil.</li><li>Cover with plastic wrap to maintain humidity.</li><li>Return to the growth chamber.</li><li>Remove plastic wrap after 3-4 days.</li></ol></li><li>Once the plants have bolted (3-4 weeks), gather the inflorescence stems of all eight plants in each pot and tie them to a bamboo skewer stake using aluminum foil.</li><li>Water plants until the last siliques are yellow and beginning to dry out.</li><li>Harvest seeds, and pool seeds of the same genotype together. Be sure to remove all chaff from the seeds. This can be done using a small sieve (being careful to clean it between different seed batches) or by collecting seeds on rough paper (<em>e.g.</em>, Whatman 3MM paper) then tilting and gently shaking the paper so that healthy seeds roll onto a second piece of paper leaving the chaff and undeveloped seeds behind. Seeds can be transferred several times between the two pieces of paper until the seeds are free of chaff.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of <em>Arabidopsis</em> seed extruded mucilage. Wear chemical-resistant gloves and a lab coat<ol class="npwtwo"><li>Determine optimal mucilage release for mature seeds hydrated in aqueous solution.<ol class="npwthree"><li>Place approximately 50 seeds per genotype (including wild type) into separate 2 ml tubes.</li><li>Add 1 ml of ultrapure water, and vortex on maximum speed for 5 s to wet seeds.</li><li>Place tubes in a box large enough for the tubes to roll around freely and place on an orbital shaker at 130 rpm.</li><li>After 1 h shaking, remove the tubes, place upright in a tube rack and allow the seeds to settle for 1-2 min.</li><li>Using a plastic Pasteur pipette, remove and discard as much water as possible without disturbing the seeds.</li><li>Add 1 ml of 0.01% ruthenium red solution and return the tubes to the box on the shaker at 130 rpm.</li><li>After 1 h, remove the tubes to the rack, allow the seeds to settle for 1-2 min, then remove and discard the ruthenium red as before.<br/><em>Note: Do not let the seeds dry out as once they are stained, the mucilage will not rehydrate to form a halo around the seed.</em> </li><li>Add 0.5 ml of ultrapure water and transfer the seeds to cavity slide (no need for cover slips).</li><li>View the seeds using the compound microscope at low magnification (5x) and determine if mucilage is efficiently released using water by comparing halo size between wild-type controls and genotypes of interest [Figure 1, compare wild type (A) with <em>mum2</em> (B)].</li><li>If mucilage is not released efficiently using water, repeat the entire process using sodium carbonate solution instead of water. Many genotypes <em>e.g.</em>, <em>mum2</em>, will release more mucilage in sodium carbonate than in water [Figure 1, compare <em>mum2</em> in water (B) with <em>mum2</em> in sodium carbonate (D). Wild type treated with sodium carbonate (C) is included for comparison]. It is important to determine the minimum concentration of sodium carbonate that will allow efficient mucilage release, as sodium ions interfere with HPAEC-PAD detection of monosaccharides. Mucilage is released from <em>mum2</em> using 20 mM sodium carbonate. It may also be useful to compare the composition of mucilage released using water with mucilage released with sodium carbonate.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223351_7875.jpg" width="131"/><br/><strong>Figure 1. Mucilage release after treatment with water in wild type (A) and <em>mum2</em> (B), and after treatment with 20 mM sodium carbonate in wild type (C) and <em>mum2</em></strong> (D). Scale bars = 500 Î¼m.<br/><em>Note: Non-adherent mucilage has been removed during the staining procedure.</em><br/><br/></li><li>Once optimal release of mucilage has been determined, proceed to either Step B2. (Preparation of extruded mucilage in water) or B3 (preparation of extruded mucilage in sodium carbonate) below.</li></ol></li><li>Preparation of extruded mucilage in water. Use seeds grown as described in A, and record the batch.<ol class="npwthree"><li>Prepare 4 replicates for each genotype by weighing out approximately 20 mg of seed per replicate. Record the exact weight of each replicate.</li><li>Place each replicate into a 2 ml tube and add 1.4 ml of ultrapure water. Make sure all seeds are suspended in the solution rather than floating on the surface by vortexing for 5 s.</li><li>Add 5 Î¼l of 5 mg/ml <em>meso</em>-Erythritol internal standard.</li><li>Prepare the samples for vortexing by taping a plastic box onto a vortex mixer 3-inch platform and fill it with tubes laid on their sides, with all tubes in the same orientation. Fill the box with bubble wrap and tape it in place securely to ensure the tubes do not move. See Notes for further details on mucilage removal.</li><li>Vortex for 2 h on the âvortex #3â setting of the vortex mixer.</li><li>While the samples are vortexing, prepare the standards (see Notes for important technical considerations). New standards must be prepared, processed, and run with each set of samples.<br/><ol class="npwfour"><li>Dilute the 100 mM monosaccharide stock to 2 mM and then make a serial dilution to give 1 mM, 0.5 mM, 0.25 mM, 0.125 mM, 0.0625 mM solutions, to give a total of 6 sugar standards (see Recipes).</li><li>Transfer 0.5 ml of each sugar standard to a KIMAX tube.</li><li>Add 5 Î¼l 5 mg/ml <em>meso</em>-Erythritol to each standard.</li></ol></li><li>Remove the samples from the vortex mixer, place in a tube rack and allow seeds to settle for 1-2 min. Do not centrifuge as mucilage will pellet.</li><li>Remove 1 ml of the mucilage-containing supernatant to a KIMAX tube. See Notes for proper cleaning of tubes.</li><li>After mucilage removal, stain the seeds with ruthenium red, as described in Steps B1e-B1i, and photograph to record how much mucilage was removed.</li><li>Using the Reacti-Vap, dry the samples and the standards (in batches) under a stream of N<sub>2</sub> gas at 60 Â°C. Once dry, the tubes can be capped and stored at -20 Â°C if required.</li><li>In the fume hood, wearing safety glasses, add 17.4 Î¼l 72% sulfuric acid to each sample and standard, and cap the tubes.</li><li>Incubate samples and standards on ice for 2 h, mixing every 30 min by vortexing on maximum for 5-10 s.</li><li>In the fume hood, add 482.6 Î¼l of ultrapure water to give a final concentration of 4% sulfuric acid.</li><li>Tighten the caps and autoclave at 121 Â°C (15 psi/1 Bar/1 ATM) for 1 h.</li><li>Remove from autoclave and allow to cool.</li><li>In the fume hood, transfer samples into clean 1.5 ml tubes, and centrifuge at 16,000 <em>x g</em> for 1 min to pellet debris.</li><li>Filter the samples and standards into clean 1.5 ml tubes using 1 ml syringes and 0.45 Î¼m nylon syringe filters. Wrap the junction between the syringe and the filter with a Kimwipe in case it âpopsâ.<br/><em>Note: If the sample is very difficult to filter then the filter is blocked and should be replaced.</em> </li><li>Load 200 Î¼l of each sample into HPLC vials with caps and septa. Samples and standards can be stored at -20 Â°C in HPLC vials or 1.5 ml tubes to prevent growth of microbes.</li></ol></li><li>Preparation of extruded mucilage in sodium carbonate. The protocol described here is for extrusion of mucilage in 20 mM sodium carbonate. If a different concentration is used, the amount of 72% sulfuric added should be to be adjusted, as some carbonate will react with sulfuric acid and alter the final concentration. See Notes for more details. Use seeds grown as described in A, and record the batch.<ol class="npwthree"><li>Prepare 4 replicates for each genotype by weighing out approximately 20 mg of seed per replicate. Record the exact weight of each replicate.</li><li>Place each replicate into a 2 ml tube and add 1.4 ml 20 mM sodium carbonate. Make sure all seeds are suspended in the solution rather than floating on the surface by vortexing for 5 s.</li><li>Add 5 Î¼l of 5 mg/ml <em>meso</em>-Erythritol internal standard.</li><li>Prepare the samples for vortexing by taping a plastic box onto a vortex mixer 3-inch platform and fill it with tubes laid on their sides, with all tubes in the same orientation. Fill the box with bubble wrap and tape it in place securely to ensure the tubes do not move. See Notes for further details on mucilage removal.</li><li>Vortex for 2 h on the âvortex #3â setting of the vortex mixer</li><li>While the samples are vortexing, prepare the standards (see Notes for important technical considerations). New standards must be prepared, processed, and run with each set of samples.<br/><ol class="npwfour"><li>Dilute the 100 mM monosaccharide stock to 2 mM and then make a serial dilution to give 1 mM, 0.5 mM, 0.25 mM, 0.125 mM, 0.0625 mM solutions, to give a total of 6 sugar standards (see Recipes).</li><li>Transfer 0.5 ml of each sugar standard to a KIMAX tube.</li><li>Add 5 Î¼l 5 mg/ml <em>meso</em>-Erythritol and 1 ml 20 mM sodium carbonate. If a different concentration of sodium carbonate is used for mucilage extraction, the amount added here should be adjusted to be the same as in the 1 ml of sample removed in Step B3h below.</li></ol></li><li>Remove the samples from the vortex mixer, place in a tube rack and allow seeds to settle for 1-2 min. Do not centrifuge as mucilage will pellet.</li><li>Remove 1 ml of the mucilage-containing supernatant to a KIMAX tube. See Notes for proper cleaning of tubes.</li><li>After mucilage removal, stain the seeds with ruthenium red, as described in Steps B1e-B1i to record how much mucilage was removed.</li><li>Using the Reacti-Vap, dry the samples and the standards (in batches) under a stream of N<sub>2</sub> gas at 60 Â°C. Once dry, the tubes can be capped and stored at -20 Â°C if required.</li><li>In the fume hood, wearing safety glasses, add 19.07 Î¼l of 72% sulfuric acid to each sample and standard, and cap the tubes.<br/></li><li>Incubate samples and standards on ice for 2 h, mixing every 30 min by vortexing on maximum for 5-10 s.<br/></li><li>In the fume hood, add 480.9 Î¼l of ultrapure water to give a final concentration of 4% sulfuric acid.</li><li>Tighten the caps and autoclave at 121 Â°C (15 psi/1 Bar/1 ATM) for 1 h.</li><li>Remove from autoclave and allow to cool.</li><li>In the fume hood, transfer samples into clean 1.5 ml tubes, and centrifuge at 16,000 <em>x g</em> for 1 min to pellet debris.</li><li>Filter the samples and standards into clean 1.5 ml tubes using 1 ml syringes and 0.45 Î¼m nylon syringe filters. Wrap the junction between the syringe and the filter with a Kimwipe in case it âpopsâ.<br/><em>Note: If the sample is very difficult to filter then the filter is blocked and should be replaced.</em> </li><li>Load 200 Î¼l of each sample into HPLC vials with caps and septa. Samples and standards can be stored at -20 Â°C in HPLC vials or 1.5 ml tubes to prevent growth of microbes.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of Alcohol Insoluble Residues (AIR) from mature whole seeds. Use seeds grown as described in A, and record the batch<ol class="npwtwo"><li>Prepare 4 replicates for each genotype by weighing out approximately 20 mg of seed per replicate. Record the exact weight of each replicate.</li><li>Homogenize seeds with one of the following techniques:<br/><ol class="npwthree"><li>Place each replicate into a 1.5 ml tube and freeze the seeds in 1.5 ml tubes in dry ice. Cool pestles and mortars with dry ice, and grind the frozen seeds to a fine powder, then transfer the frozen powder back to a 1.5 ml tube and store on dry ice. This procedure gives very good results but is time-consuming.</li><li>Place each replicate into a 2 ml screw-capped tube with three to five 2 mm zirconia beads, freeze on dry ice and disrupt in a tissue homogenizer at 6,500 rpm for 20 s. It is important to choose tubes that will not break in the tissue homogenizer.</li></ol></li><li>Add 1 ml 70% (v/v) ethanol to the tubes containing the frozen, ground seeds, and resuspend the tissue fully by vortexing. If grinding was done with a pestle and mortar, add three to five 2 mm zirconia beads to the tube to help to resuspend the tissue.</li><li>Incubate at 65 Â°C (use the heat block in the Reacti-Vap) for 10 min.</li><li>Vortex the samples in a box attached to the vortex mixer (as described in Step B2d) for 10-15 min to solubilize metabolic sugars.</li><li>Centrifuge at 16,000 <em>x g</em> for 15 s. Do not spin for longer or the pellet will become hard to resuspend.</li><li>Remove the supernatant, and replace with 1 ml fresh 70% (v/v) EtOH in ultrapure water.</li><li>Vortex for 10-15 min and centrifuge again, as described above.</li><li>Remove the supernatant, and replace with 1 ml 80% (v/v) HPLC-grade MeOH in ultrapure water.</li><li>Vortex for 10-15 min and centrifuge again, as described above.</li><li>Remove the supernatant, and replace with 1 ml 100% HPLC-grade MeOH.</li><li>Vortex for 10-15 min and centrifuge again, as described above.</li><li>Remove the supernatant, and replace with 1 ml fresh 100% HPLC-grade MeOH.</li><li>Vortex for 10-15 min and centrifuge again, as described above.</li><li>Remove the supernatant, and replace with 1 ml 100% HPLC-grade acetone.</li><li>Vortex for 10-15 min and centrifuge again, as described above.</li><li>Remove the supernatant, and replace with 1 ml fresh 100% HPLC-grade acetone.</li><li>Vortex for 10-15 min and centrifuge again, as described above. Remove most of the acetone, leaving a small amount which can be used to resuspend the pellet by vortexing briefly.</li><li>Using a micropipette, transfer the resuspended pellet (in acetone) to a clean 2 ml tube, leaving the beads behind.</li><li>Dry the AIR by leaving the tubes open in the fume hood. Acetone will evaporate quickly, leaving a light brown AIR pellet.</li><li>Weigh the resulting AIR (there should be over 3 mg of AIR) and record the exact weight, and then transfer the AIR to a KIMAX tube.</li><li>Add 20-30 1 mm zirconia beads to each tube, cap the tubes, and vortex briefly to break the pellet into powder. The beads will also help with mixing samples in sulfuric acid.</li><li>Add 500 Î¼l of 100% HPLC-grade acetone and 20 Î¼l of 5 mg/ml <em>meso</em>-Erythritol to each sample and dry under a stream of N<sub>2</sub> gas at 60 Â°C using the Reacti-Vap. Acetone helps to evaporate the water, leaving a less clumped pellet than evaporating 20 Î¼l of water alone.</li><li>Prepare the standards (see Notes for important technical considerations). New standards must be prepared, processed, and run with each set of samples.<br/><ol class="npwthree"><li>Dilute the 100 mM monosaccharide stock to 2 mM and then make a serial dilution to give 1 mM, 0.5 mM, 0.25 mM, 0.125 mM, 0.0625 mM solutions, to give a total of 6 sugar standards (see Recipes).</li><li>Transfer 0.5 ml of each sugar standard to a KIMAX tube.</li><li>Add 5 Î¼l of 5 mg/ml <em>meso</em>-Erythritol to each tube.</li><li>Using the Reacti-Vap, dry standards under a stream of N<sub>2</sub> gas at 60 Â°C using the Reacti-Vap.</li></ol></li><li>Once dry, the tubes containing standards and samples can be capped and stored at -20 Â°C if required.</li><li>In the fume hood, wearing safety glasses, add 70 Î¼l of 72% sulfuric acid to each sample, and 17.4 Î¼l of 72% sulfuric acid to each standard, and cap the tubes.</li><li>Incubate samples and standards on ice for 2 h, mixing every 30 min by vortexing for 5-10 s.</li><li>In the fume hood, add 1.93 ml of ultrapure water to the samples, and 482.6 Î¼l to the standards, to give a final concentration of 4% sulfuric acid.</li><li>Tighten the caps and autoclave at 121 Â°C (15 psi/1 Bar/1 ATM) for 1 h.</li><li>Remove from autoclave and allow to cool.</li><li>In the fume hood, transfer samples into clean 1.5 ml tubes, and centrifuge at 16,000 <em>x g</em> for 1 min to pellet debris.</li><li>Filter the samples and standards into clean 1.5 ml tubes using 1 ml syringes and 0.45 Î¼m nylon syringe filters. Wrap the junction between the syringe and the filter with a Kimwipe in case it âpopsâ.<br/><em>Note: If the sample is very difficult to filter then the filter is blocked and should be replaced.</em> </li><li>Load 200 Î¼l of each sample into HPLC vials with caps and septa. Samples and standards can be stored at -20 Â°C in HPLC vials or 1.5 ml tubes to prevent growth of microbes.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Separation and quantification of monosaccharides using high performance anion-exchange chromatography coupled with pulsed amperometric detection (HPAEC-PAD) using a CarboPac PA1 Guard and Analytical column at 25 Â°C<ol class="npwtwo"><li>Separation of acid sugars.<ol class="npwthree"><li>Use a constant eluant flow rate of 0.4 ml/min.</li><li>Equilibrate the column with 10% 1 M NaOH, 1% 1 M sodium acetate, and 89% ultrapure water for 10 min.</li><li>Independently inject 10 Î¼l of each sample and standard.<br/></li><li>Separate acid sugars using a 30-minute linear gradient reaching 40% 1 M sodium acetate, and keeping the 1 M NaOH constant (Figure 2).</li><li>Clean the column with a linear gradient of 1 M NaOH that reaches 30% over 10 min, while decreasing 1 M sodium acetate to 1% (Figure 2).</li><li>Re-equilibrate the column for 10 min with the eluents at their starting concentrations for 10 min (10% 1 M NaOH, 1% 1 M sodium acetate, and 89% ultrapure water) before injection of the next sample (Figure 2).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223455_2307.jpg" width="187"/><br/><strong>Figure 2. Eluent gradient for the separation of acid sugars</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Separation of neutral sugars.<ol class="npwthree"><li>Use a constant eluant flow of 1 ml/min.</li><li>Equilibrate the column with 100% ultrapure water for 10 min, and start the post column pump which supplies 200 mM NaOH to the detector for post-column separation of monomeric neutral sugars.</li><li>Inject 10 Î¼l of each sample.</li><li>Elute the neutral sugars with 100% ultrapure water for 35 min with post column detection.</li><li>Clean the column for 10 min with 20% 1 M NaOH.</li><li>Re-equilibrate the column with a 10-min rinse of 100% ultrapure water before next injection.</li></ol></li><li>Peak integration and data export from Chromeleon.<ol class="npwthree"><li>View the files produced in Chromeleon (see examples in Figures 3 and 4).</li><li>Manually integrate chromatographs, keeping the limits of the peaks in the samples at similar retention times as the limits of the peaks in the standards. Baselines for the <em>meso</em>-Erythritol internal standard and peak of interest (GalA) should have a flat baseline; adjust baseline with the vertical/horizontal tool from the Chromeleon tool ribbon. Save changes to peak integrations.</li><li>Export values for area under the curve from the integration tab in the Chromeleon software to Excel.</li><li>Before running further analysis, make sure that the standards show linear response, <em>i.e.</em>, for any given monosaccharide, the peak area for the 1 mM standard should be exactly half of the 2 mM standard, and so on. This can be tested by plotting peak area vs. concentration of the standards in Excel and performing linear regression. The intercept should be close to 0 and R<sup>2</sup> close to 1 to ensure accurate quantification.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223521_6433.jpg" width="192"/><br/><strong>Figure 3. Separation of neutral sugars (Ara, Rha, Gal, Glu, Xyl, Man) with the CarboPac PA1 analytical column.</strong> Fuc is not detectable in this sample but has a retention time of approximately 5.5 min.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223541_1468.jpg" width="188"/><br/><strong>Figure 4. Separation of acid sugars (GalA) with the CarboPac PA1 analytical column</strong> </li></ol></li></ol></li></ol><p class="pptt" id="biaoti30721">### Data analysis</p><ol class="npwone"><li>Install Python and SciPy, if not already installed on your workstation <em>e.g.</em>, as follows:<ol class="npwtwo"><li>Install Python 2.7.16 (<a href="https://www.python.org/downloads/release/python-2716/" target="_blank">https://www.python.org/downloads/release/python-2716/</a>) and make a note of the directory where Python is installed.</li><li>Open the Command Prompt and type in âcdâ (without the quotation marks).</li><li>Open File Explorer and navigate to the Python27 folder.</li><li>Drag and drop the Python27 folder into the Command Prompt and press Enter. The Command Prompt should look as shown in Figure 5.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223615_5070.jpg" width="299"/><br/><strong>Figure 5. Command prompt after setting up Python27</strong><br/><br/></li><li>Go to scipy.org/install.html, and copy the code highlighted in dark grey shown in Figure 6:<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223637_3137.jpg" width="421"/><br/><strong>Figure 6. Code to install SciPy</strong><br/><br/></li><li>Paste the code into the Command Prompt and press Enter.</li><li>The SciPy package will be installed.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Customize the MonosaccharideComposition.py Python code provided as part of this protocol. The code must be changed between mucilage and AIR data processing, as the final sample volumes are different<ol class="npwtwo"><li>Use Notepad++ to open the âMonosaccharideComposition.pyâ Python file (see <a href="https://os.bio-protocol.org/attached/file/20191211/Supplemental 1.zip" target="_blank">Supplemental 1</a>).</li><li>Edit the final sample volumes. Mucilage samples are prepared in a final volume of 0.5 ml, and AIR samples are prepared in a final volume of 2 ml. To adjust different final volumes that are employed during sample preparation, change line 162 of the code: âfinalCor = finalCorrections (calibrated, corrected, <span style="color:#E53333;">6</span>, <span style="color:#E53333;"><strong>0.5</strong></span>)â, where the last number, shown here in red and bold, represents volume in ml.</li><li>Edit the number of standards used. Go to line number 161 and 162 and find the code that reads âcalibrated = calibration (corrected, <strong>6</strong>)â and âfinalCor = finalCorrections (calibrated, corrected, <span style="color:#E53333;"><strong>6</strong></span>, <strong><span style="color:#E53333;">0.5</span></strong>)â. Replace the number 6 (Colored red) in the code with the number of standards used, if different.</li><li>Save the customized Python file.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Prepare HPAEC data for analysis<ol class="npwtwo"><li>Paste integrated peak areas obtained from Chromeleon into the template Excel spreadsheet. Prepare separate spreadsheets for mucilage and whole seed AIR samples as the final sample volumes are different.<ol class="npwthree"><li>Acid and neutral monosaccharides are analyzed separately, and therefore need to be processed as separate files; the template Excel spreadsheet has separate sheets for inputting acid and neutral sugar data (see <a href="https://os.bio-protocol.org/attached/file/20191211/supplemental 2.xlsx" target="_blank">Supplemental 2</a>).</li><li>Make sure to use monosaccharide abbreviations: Ara for arabinose, Xyl for xylose, Fuc for fucose, Rha for rhamnose, GalA for galacturonic acid, Gal for galactose, Man for mannose, and Glc for glucose, as they will be used to determine correction coefficients for hydrolysis.</li><li>Enter the mass, in mg, of either seeds used for mucilage extraction, or of AIR used for whole seeds.</li><li>Do not include spaces in sample and standard names.</li><li>Add âmMâ after standard concentrations. This is required so that standards can be distinguished from samples during processing.</li><li>For each genotype, mark replicates by adding â#â followed by replicate number after the genotype name. This must be included so that replicates are grouped together.</li></ol></li><li>Copy the data from Excel, paste it into Notepad++, and save as a .txt file. Make sure there is no extra row at the end of the .txt file underneath the data or it will not run properly.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Run the customized Monosaccharide Composition.py Python code saved in B<ol class="npwtwo"><li>Drag and drop the Monosaccharide Composition.py file into the Command Prompt, and press Enter.</li><li>A popup window will open: navigate to and select the .txt file of integrated peak areas.</li><li>A popup window will open: choose a file name and a location for the processed data, and press Enter. The processed data will be saved.</li><li>Open the processed tab-delimited data file with Excel. The data for each sample and sugar can be cut and pasted into Excel for processing. Calculate average nmol/mg and standard deviation for each sugar for each genotype, and plot graphs.</li></ol></li></ol><p class="pptt" id="biaoti30722">### Notes</p><ol class="npwtwok"><li><em>Arabidopsis</em> prefers a light peaty soil and previously we have used 1:1:1 peat: perlite: vermiculite fertilized with AT medium, but have found that prepared peat-rich medium such as Sunshine Mix #5 (Sungro) works well and is more convenient. As Sunshine Mix #5 does not contain fertilizer, we water once before planting with liquid AT minimal medium (see Recipes) as described in Procedure A. We recommend that plants be grown according to your usual procedure. It is critical to water plants until the last siliques are turning yellow as described in Procedure A. It is also very important to grow several independent sets of seeds, including wild type, at different times to capture any variation caused by slight differences in growth conditions. We recommend growing 3 independent replicates of seeds as described in Procedure A, and performing monosaccharide analysis independently on each one. Please note that within one HPAEC experiment (as outlined in Procedure B), the 4 replicates prepared are biological and not technical replicates. In our hands, there is very little variation among replicates from the same seed batch that are prepared and analyzed together.</li><li>Mucilage removal from seeds can be increased by splitting the 20 mg of seeds used for extraction into 4 x 5 mg aliquots (exact weights recorded), and extracting each 5 mg aliquot with 0.35 ml water/sodium carbonate and 1.25 Î¼l of 5 mg/ml <em>meso</em>-Erythritol. After vortexing as described, recombine the 4 aliquots, then take 1 ml and continue with the standard protocol.</li><li>Proper cleaning of the KIMAX tubes used for acid hydrolysis is important to prevent contamination. Wash the tubes with HPLC-grade methanol to remove any soluble sugars and dry in the fume hood. After sample processing, first prepare a solution of sodium bicarbonate (baking soda) in water. Wearing safety glasses, lab coat, and chemical resistant gloves, fill the KIMAX tubes containing debris and residual sulfuric acid with the baking soda solution. Leave to soak for at least 2 h, then scrub the debris from the tubes using a test tube brush and wash thoroughly with tap water. Finally, either wash in a dishwasher or rinse thoroughly with ultrapure water before air drying.</li><li>Careful preparation of standard curves to convert peak area to amount of monosaccharide is essential, as is accurate addition of the <em>meso</em>-Erythritol internal standard. Test micropipette accuracy using an analytic balance (1 ml of ultrapure water weighs 1 g). The sample values detected by the PAD detector need to be within the calibration range of the external standards. If the samples have higher reads than the highest concentration standard (2 mM), repeat the procedure with lower amounts of tissue or higher standard concentrations.</li><li>If a different concentration of sodium carbonate is used for mucilage extrusion, then the amount of 72% sulfuric acid added has to be adjusted as the sodium carbonate will react with the acid, decreasing the concentration. The reaction between sodium carbonate and sulfuric acid is as follows:<br/>H<sub>2</sub>SO<sub>4</sub> + Na<sub>2</sub>CO<sub>3</sub> â Na<sub>2</sub>SO<sub>4</sub> + CO<sub>2</sub> + H<sub>2</sub>O<br/>1 mole of sulfuric acid reacts with 1 mole of sodium carbonate. If 20 mM sodium carbonate is used, then 0.02 mmoles of sodium carbonate is present in the 1 ml mucilage sample, and 0.02 mmoles of sulfuric acid is required for neutralization. The molarity of 72% sulfuric acid is 12 M (specific gravity 1.622 g/cm<sup>3</sup>), hence 1.67 Î¼l of 72% sulfuric acid is required for neutralization. Therefore, the total volume of sulfuric acid required is 1.67 Î¼l + 17.4 Î¼l = 19.07 Î¼l.<br/></li></ol><p class="pptt" id="biaoti30723">### Recipes</p><ol class="npwtwok"><li>AT minimal medium<br/>We use an AT minimal medium which was developed in our lab (Haughn and Somerville, 1986), but the <em>Arabidopsis</em> growth medium of your choice may be substituted. For AT minimal medium, begin by making the stock solutions and autoclaving them in 1 L bottles (see Table 1). To prepare the medium, add the correct amount of each nutrient stock solution to 800 ml of ultrapure water (see Table 1). Add stocks to water to prevent precipitation of insoluble calcium phosphate complexes. Make volume up to 1 L.<br/><br/><strong>Table 1. AT minimal medium</strong><br/><img alt="" class="layerphoto" height="364" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223757_3172.jpg" width="600"/><br/><br/><em>Note: We water our soil mix with AT minimal medium without agar, but this may not be necessary depending on your choice of soil mix (see Notes). AT minimal medium that will be used for watering does not need to be autoclaved, but should be stored in dark to prevent growth of algae.</em><br/>For plates, adjust the pH of the medium to 5.8 with KOH, add 7 g/L of agar and autoclave. After autoclaving, medium can be stored in bottles at room temperature, and plates stored wrapped in plastic wrap at 4 Â°C.</li><li>Ruthenium red solution<br/><strong>Stock solution of 0.1% (w/v) ruthenium red</strong><br/>Dissolving 50 mg of ruthenium red into 50 ml of ultrapure water<br/>Store in a 50 ml conical tube at 4 Â°C<br/><strong>Working solution of 0.01% (w/v) ruthenium red</strong><br/>Dilute 5 ml into 50 ml of ultrapure water in a 50 ml conical tube and store at 4 Â°C<br/><em>Note: These solutions are good as long as the color is not purple or brown. Once discolored, they should be discarded and fresh solutions prepared.</em> </li><li>Sodium carbonate solutionï¼20 mM)<ol class="npwthree"><li>Dissolve 212 mg of anhydrous sodium carbonate in 80 ml ultrapure water</li><li>Make up to 100 ml with ultrapure water using a volumetric flask</li><li>Store at room temperature</li></ol></li><li>Sulfuric acid, 72% (w/w), specific gravity 1.634 g/cm<sup>3</sup><br/><em>Note: Prepare in the fume hood with face shield and chemical-resistant gloves, while wearing a lab coat.</em><br/>Slowly add 665 ml of concentrated sulfuric acid (specific gravity 1.84 g/cm<sup>3</sup>, 96% solution) to 333 ml ultrapure water while mixing slowly and continuously<br/>Formula for calculations:<br/>Volume<sub>1 </sub>Ã Specific Gravity<sub>1 </sub>Ã % Solution<sub>1</sub>=Volume<sub>2 </sub>Ã Specific Gravity<sub>2</sub> Ã % Solution<sub>2</sub> </li><li>Monosaccharide stock solution, 100 mM<ol class="npwthree"><li>To prepare a 100 mM monosaccharide stock solution containing 100 mM each of L-(-)-Fucose, L-(+)-Arabinose, L-Rhamnose, D-Galactose, D-Glucose, D-(+)-Mannose, D-(+)-Xylose, D-(+)-Galacturonic acid, weigh the appropriate amount of each sugar in Table 2 below</li><li>Dissolve in ultrapure water and make up to 5 ml in a volumetric flask</li><li>Store as 1 ml aliquots in sterile 1.5 ml tubes at -20 Â°C<br/><br/><strong>Table 2. Preparation of monosaccharide stock solution</strong><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" height="282" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223839_8595.jpg" width="400"/><br/><br/></div></li></ol></li><li>Dilution series for monosaccharide standards<br/>Prepare a dilution series as shown in Table 3. Use ultrapure water for the dilutions.<br/><br/><strong>Table 3. Preparation of monosaccharide standard dilution series</strong><br/><img alt="" class="layerphoto" height="220" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191216/20191216223917_8718.jpg" width="600"/><br/><br/></li><li><em>meso</em>-Erythritol solution, 5 mg/ml<ol class="npwthree"><li>To prepare a 5 mg/ml stock solution, weigh out 50 mg of <em>meso</em>-Erythritol. Dissolve in ultrapure water and make up to 10 ml in a volumetric flask</li><li>Transfer to a sterile 15 ml conical tube</li><li>Store at -20 Â°C</li></ol></li><li>1 M NaOH (HPAEC-PAD eluent)<ol class="npwthree"><li>Weigh out 120 g of 50% (w/w) NaOH into a clean beaker using a plastic 50 ml serological pipette and transfer into a 1.5 L volumetric flask</li><li>Using ultrapure water, rinse the beaker into the volumetric flask and make up to volume</li><li>Transfer to a vacuum flask with a magnetic stir bar and degas by vacuum for at least 2 h</li></ol></li><li>200 mM NaOH (HPAEC-PAD eluent)<ol class="npwthree"><li>Weigh out 32 g of 50% (w/w) NaOH into a clean beaker using a plastic 50 ml serological pipette and transfer into a 2 L volumetric flask</li><li>Using ultrapure water, rinse the beaker into the volumetric flask and make up to volume</li><li>Transfer to a vacuum flask with a magnetic stir bar and degas by vacuum for at least 2 h</li></ol></li><li>1 M sodium acetate (HPAEC-PAD eluent)<ol class="npwthree"><li>Weigh out 82.03 g of anhydrous sodium acetate</li><li>Dissolve in ultrapure water and make up to 1 L in a volumetric flask</li><li>Transfer to a vacuum flask with a magnetic stir bar and degas by vacuum for at least 2 h</li></ol></li></ol><p class="pptt" id="biaoti30724">### Acknowledgments</p><p class="ppzz">This protocol was derived from the original protocol in Dean <em>et al.</em> (2007) with minor modifications in Å ola <em>et al.</em> (2019b). This work was supported by Natural Sciences and Engineering Research Council of Canada (NSERC) Discovery Grants to G.W.H. and S.D.M.</p><p class="pptt" id="biaoti30725">### Competing interests</p><p class="ppzz">The authors have no financial or non-financial competing interests.</p><p class="pptt" id="biaoti30726">### References</p><ol class="npwtwok"><li>Arsovski, A. A., Haughn, G. W. and Western, T. L. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20505351" target="_blank">Seed coat mucilage cells of <em>Arabidopsis</em> <em>thaliana</em><em> as a model for plant cell wall research.</em></a> <em>Plant Signal Behav</em> 5(7): 796-801.</li><li>Beeckman, T., Rycke, R. De., Viane, R. and InzÃ©, D. (2000). <a href="https://link.springer.com/article/10.1007/PL00013924" target="_blank">Histological study of seed coat development in <em>Arabidopsis thaliana</em>.</a> <em>J Plant Res</em> 113: 139-148.<br/></li><li>Coleman, H. D., Yan, J. and Mansfield, S. D. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19625620" target="_blank">Sucrose synthase affects carbon partitioning to increase cellulose production and altered cell wall ultrastructure.</a> <em>Proc Nati Acad Sci U S A </em>106(31): 13118-13123.</li><li>Dean, G. H., Zheng, H., Tewari, J., Huang, J., Young, D. S., Hwang, Y. T., Western, T. L., Carpita, N. C., McCann, M. C., Mansfield, S. D. and Haughn, G. W. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18165329" target="_blank">The <em>Arabidopsis MUM2</em> gene encodes a Î²-galactosidase required for the production of seed coat mucilage with correct hydration properties.</a> <em>Plant Cell</em> 19(12): 4007-4021.</li><li>Francoz, E., Lepiniec, L. and North, H. M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30056618" target="_blank">Seed coats as an alternative molecular factory: thinking outside the box.</a> <em>Plant Reprod</em> 31: 327-342.</li><li>Francoz, E., Ranocha, P., Burlat, V. and Dunand, C. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25998090" target="_blank"><em>Arabidopsis</em> seed mucilage secretory cells: Regulation and dynamics.</a> <em>Trends Plant Sci</em> 20: 515-524.<br/></li><li>Golz, J. F., Allen, P. J., Li, S. F., Parish, R. W., Jayawardana, N. U., Bacic, A. and Doblin, M. S. (2018). <a href="https://www.sciencedirect.com/science/article/pii/S0168945218302097" target="_blank">Insights into the role of transcription factors controlling mucilage production in the <em>Arabidopsis</em> seed coat.</a> <em>Plant Sci</em> 272: 179-192.</li><li>Griffiths, J. S. and North, H. M. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28191645" target="_blank">Sticking to cellulose: exploiting <em>Arabidopsis</em> seed coat mucilage to understand cellulose biosynthesis and cell wall polysaccharide interactions. </a><em>New Phytol</em> 214(3): 959-966.</li><li>Griffiths, J. S., Tsai, A. Y., Xue, H., Voiniciuc, C., Å ola, K., Seifert, G. J., Mansfield, S. D. and Haughn, G. W. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24808103" target="_blank">SALT-OVERLY SENSITIVE5 mediates <em>Arabidopsis</em> seed coat mucilage adherence and organization through pectins.</a> <em>Plant Physiol</em> 165(3): 991-1004.</li><li>Haughn, G. W. and Somerville, C. (1986). <a href="https://link.springer.com/article/10.1007/BF00331020" target="_blank">Sulfonylurea-resistant mutants of <em>Arabidopsis thaliana</em>.</a> <em>Mol Gen Genet </em>204: 430-434.</li><li>Haughn, G. W. and Western, T. L. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22645594" target="_blank"><em>Arabidopsis</em> seed coat mucilage is a specialized cell wall that can be used as a model for genetic analysis of plant cell wall structure and function. </a><em>Front Plant Sci </em>3: 64.</li><li>Huang, J., DeBowles, D., Esfandiari, E., Dean, G., Carpita, N. C. and Haughn, G. W. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21518777" target="_blank">The <em>Arabidopsis</em> transcription factor <em>LUH/MUM1</em> is required for extrusion of seed coat mucilage. </a><em>Plant Physiol</em> 156(2): 491-502.</li><li>Macquet, A., Ralet, M. C., Loudet, O., Kronenberger, J., Mouille, G., Marion-Poll, A. and North, H. M. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18165330" target="_blank">A naturally occurring mutation in an <em>Arabidopsis</em> accession affects a Î²-D-galactosidase that increases the hydrophilic potential of rhamnogalacturonan I in seed mucilage.</a> <em>Plant Cell</em> 19(12): 3990-4006.</li><li>Mendu, V., Griffiths, J. S., Persson, S., Stork, J., Downie, A. B., Voiniciuc, C., Haughn, G. W. and DeBolt, S. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21750228" target="_blank">Subfunctionalization of cellulose synthases in seed coat epidermal cells mediates secondary radial wall synthesis and mucilage attachment.</a> <em>Plant Physiol</em> 157(1): 441-453.</li><li>North, H. M., Berger, A., Saez-Aguayo, S., and Ralet, M. C. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24607722" target="_blank">Understanding polysaccharide production and properties using seed coat mutants: future perspectives for the exploitation of natural variants.</a> <em>Ann Bot</em> 114(6): 1251-1263.</li><li>Phan, J. L. and Burton, R. A. (2018). <a href="https://onlinelibrary.wiley.com/doi/10.1002/9781119312994.apr0606" target="_blank">New insights into the composition and structure of seed mucilage. </a><em>Annual Plant Reviews </em>1: 1-41.</li><li>Shi, L., Dean, G. H., Zheng, H., Meents, M. J., Haslam, T. M., Haughn, G. W. and Kunst, L. (2019). <a href="http://www.plantphysiol.org/content/181/3/901" target="_blank">ECERIFERUM11/C-TERMINAL DOMAIN PHOSPHATASE-LIKE2 Affects secretory trafficking.</a> <em>Plant physiol</em> 181: 901-915.</li><li>Å ola, K., Dean, G. H. and Haughn, G. W. (2019a). <a href="https://onlinelibrary.wiley.com/doi/10.1002/9781119312994.apr0691" target="_blank"><em>Arabidopsis</em> seed mucilage, a specialized extracellular matrix that demonstrates the structure-function versatility of cell wall polysaccharides.</a> <em>Annual Plant Reviews</em> 2(4).</li><li>Å ola, K., Gilchrist, E. J., Ropartz, D., Wang, L., Feussner, I., Mansfield, S. D., Ralet, M. C. and Haughn, G. W. (2019b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30852555" target="_blank">RUBY, a putative galactose oxidase, influences pectin properties and promotes cell-to-cell adhesion in the seed coat epidermis of <em>Arabidopsis thaliana</em>.</a> <em>Plant Cell</em> 31(4): 809-831.</li><li>Voiniciuc, C., Dean, G. H., Griffiths, J. S., Kirchsteiger, K., Hwang, Y. T., Gillett, A., Dow, G., Western, T. L., Estelle, M. and Haughn, G. W. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23482858" target="_blank">FLYING SAUCER1 is a transmembrane RING E3 ubiquitin ligase that regulates the degree of pectin methylesterification in <em>Arabidopsis</em> seed Mucilage.</a> <em>Plant Cell</em> 25: 944-959.</li><li>Voiniciuc, C. and GÃ¼nl, M. (2016). <a href="https://bio-protocol.org/e1801" target="_blank">Analysis of monosaccharides in total mucilage extractable from <em>Arabidopsis</em> seeds.</a> <em>Bio-protocol</em> 6: e1801.</li><li>Voiniciuc, C., Yang, B., Schmidt, M. H., GÃ¼nl, M. and Usadel, B. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25658798" target="_blank">Starting to gel: how <em>Arabidopsis</em> seed coat epidermal cells produce specialized secondary cell walls.</a> <em>Int J Mol Sci</em> 16: 3452-3473.</li><li>Western, T. L. (2012). <a href="https://www.researchgate.net/publication/259418199_The_sticky_tale_of_seed_coat_mucilages_Production_genetics_and_role_in_seed_germination_and_dispersal" target="_blank">The sticky tale of seed coat mucilages: production, genetics, and role in seed germination and dispersal.</a> <em>Seed Sci Res</em> 22: 1-25.</li><li>Western, T. L., Burn, J., Tan, W. L.,. Skinner, D. J., Martin-McCaffrey, L., Moffatt, B. A. and Haughn, G. W. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11706181" target="_blank">Isolation and characterization of mutants defective in seed coat mucilage secretory cell development in <em>Arabidopsis</em>.</a> <em>Plant Physiol</em> 127: 998-1011.</li><li>Western, T. L., Skinner, D. J. and Haughn, G. W. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10677428" target="_blank">Differentiation of mucilage secretory cells of the <em>Arabidopsis</em> seed coat.</a> <em>Plant Physiol</em> 122: 345-356.</li><li>Windsor, J. B., Symonds, V. V., Mendenhall, J. and Lloyd, A. M. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10886768" target="_blank"><em>Arabidopsis</em> seed coat development: morphological differentiation of the outer integument.</a> <em>Plant J</em> 22: 483-493.</li><li>Yang, X., Baskin, J. M., Baskin, C. C. and Huang, Z. (2012). <a href="https://www.sciencedirect.com/science/article/abs/pii/S1433831912000546" target="_blank">More than just a coating: Ecological importance, taxonomic occurrence and phylogenetic relationships of seed coat mucilage.</a> <em>Perspect Plant Ecol</em> 14(6): 434-442.<br/></li></ol>