<body>
            <h1>Protocol for Ribosome Profiling in Bacteria</h1>
            <p><strong>Authors:</strong> Fuad Mohammad, Allen R. Buskirk</p>
            <h2>Abstract</h2>
            <p>Ribosome profiling provides information on the position of ribosomes on mRNA on a genomic scale. Although this information is often used to detect changes in gene expression under different conditions, it also has great potential for yielding insight into the mechanism and regulation of protein synthesis itself. First developed in yeast, ribosome profiling involves the isolation and sequencing of ribosome-protected mRNA fragments generated by nuclease treatment. Since the application of ribosome profiling in bacteria has been problematic, we report here a systematically optimized protocol for <em>E. coli</em> that we have used with success for other bacteria as well. Cells are harvested by flash-freezing cultures directly in liquid nitrogen. After lysis, translation is arrested by high magnesium concentration without the use of antibiotics. These improvements eliminate artifacts induced by harvesting cells by centrifugation or filtration and by use of chloramphenicol to arrest translation. These improvements are especially appropriate for studies where the exact position of the ribosome is critical, and not merely the number of ribosomes per message, such as studies aimed at monitoring differences in local elongation rates.</p>
            <p><strong>Keywords:</strong> Ribosome profiling, Ribosome pausing, Translational efficiency, Gene expression, Bacteria</p>
            <h2>Background</h2><p>mRNAs are translated at varying efficiencies depending on properties inherent to the mRNA such as initiation rates, mRNA structure, and codon usage. In addition, physiological changes in the cell can alter how efficiently mRNAs are translated. Ribosome profiling was developed to monitor gene expression at the level of protein synthesis, making it possible to ask global questions about which regions of the genome are translated, when they are translated, and how translation is regulated. First developed by Ingolia and Weissman in yeast (Ingolia <em>et al.</em>, 2009), ribosome profiling utilizes nuclease digestion to degrade regions of mRNA that are not occupied by ribosomes, generating ribosome-protected mRNA fragments that are isolated, cloned, and sequenced. When the reads are mapped back to the genome, they reveal the position of ribosomes throughout the transcriptome, providing a powerful tool to monitor translation and its regulation <em>in vivo</em>. Our focus here is the application of ribosome profiling to bacteria, first reported by Weissman and Bukau (Oh <em>et al.</em>, 2011; Li <em>et al.</em>, 2012; Becker <em>et al.</em>, 2013). Our protocol is derived from these pioneering studies and incorporates recent improvements described by Ingolia for yeast and mammalian cells (McGlincy and Ingolia, 2017).<br/><br/>Ribosome profiling has been used in bacteria to identify which regions of the genome are translated, revealing novel small proteins and internal initiation sites in annotated ORFs (Shell <em>et al.</em>, 2015; Meydan <em>et al.</em>, 2019; Weaver <em>et al.</em>, 2019). Many studies report changes in gene expression that occur upon a change in the environment, the imposition of an external stress, or the depletion of a regulatory factor (Liu <em>et al.</em>, 2013; Subramaniam <em>et al.</em>, 2013; Guo <em>et al.</em>, 2014; Haft <em>et al.</em>, 2014; Zhang <em>et al.</em>, 2018). These studies are powerful because in exponentially growing bacteria the level of translation correlates strongly with steady-state protein levels (Li <em>et al.</em>, 2014). When supplemented with sequencing of total RNA by conventional RNA-seq, ribosome profiling data provide estimates of translational efficiency (the number of ribosomes per message). These estimates have yielded important conclusions about how the translation of proteins that form a complex occurs in the correct stoichiometry, even from a single mRNA (Li <em>et al.</em>, 2014; Lalanne <em>et al.</em>, 2018), and identified mRNA structure as one of the primary determinates of translational efficiency (Burkhardt <em>et al.</em>, 2017).<br/><br/>In addition to these studies of gene expression, ribosome profiling can also be used to probe features of the translational machinery and the distinct phases of translation. Mechanisms of action of elongation inhibitors can be studied <em>in vivo</em> at codon resolution (Kannan <em>et al.</em>, 2014; Marks <em>et al.</em>, 2016). Factors involved in regulating translation can also be studied in detail using profiling (Balakrishnan <em>et al.</em>, 2014; Elgamal <em>et al.</em>, 2014; Woolstenhulme <em>et al.</em>, 2015; Baggett <em>et al.</em>, 2017). Another successful approach has been immunoprecipitation of factors of interest that associate with ribosomes, followed by ribosome profiling to determine where on mRNAs these factors are engaged (Oh <em>et al.</em>, 2011). This approach has led to insights into protein folding and complex-formation during translation.<br/><br/>We are primarily interested in using ribosome profiling to measure local elongation rates which vary as a result of ribosome pausing at mRNA structures, rare codons, or troublesome nascent peptide sequences. Sites where ribosomes pause are expected to yield a higher number of reads than sites where elongation is fast, leading to enriched ribosome density. For these studies which rely on high resolution, it is important that the procedure to capture ribosome footprints truly reflects ribosomes in their native environment.<br/><br/>In our efforts to optimize the protocol for bacteria, where no standard protocol has emerged, we identified artifacts that interfere with capturing the native distribution of ribosome footprints (Mohammad <em>et al.</em>, 2019). First, ribosome footprints in bacteria vary widely in size from as short as 15 nt to as long as 40 nt. It is essential to capture all of these footprints to have an accurate view of translation. Second, the way that cells are harvested can alter translation and induce artifacts: addition of chloramphenicol before centrifugation induces strong pauses and leads to pile-ups of ribosomes at the 5â-end of ORFs. Harvesting by filtration also induces pauses at specific codons due to rapid depletion of aminoacyl-tRNAs. Third, translation continues after cell lysis, even in the presence of chloramphenicol. To address these problems, we developed a method of directly freezing the culture and immediately inhibiting translation with a high concentration of MgCl<sub>2</sub>. These improvements capture ribosomes as they were under growth conditions and will enable studies to more accurately detect ribosome position <em>in vivo</em>. For studies primarily concerned with the number of ribosomes per message (and not their exact position) we also provide the earlier filtration protocol which is simpler and useful for preparation of samples for RNA-seq.</p>
        </body>

<p class="pptt" id="biaoti30760">## Materials and Reagents</p><ol class="npwtwok"><li>Whatman cellulose nitrate 0.45 Î¼m filters (VWR, catalog number: 7184-009)</li><li>Corning<sup>Â®</sup> Costar<sup>Â®</sup> Spin-X<sup>Â®</sup> centrifuge tube filters (Sigma-Aldrich, catalog number: CLS8163-100EA)</li><li>Corning<sup>Â®</sup> Costar Cell Lifter (plastic spatula) (Sigma-Aldrich, catalog number: CLS3008)</li><li>3M Overhead Sheets (PP2500)</li><li>Razor blades</li><li>Saran wrap</li><li>50 ml conical tube</li><li>Ti 70 polycarbonate tube</li><li>18 G needle</li><li>P1000 tips</li><li>Aluminum foil</li><li>CircLigase<sup>TM</sup> ssDNA Ligase (Epicentre, catalog number: CL4115K)<br/></li><li>SuperScript<sup>Â®</sup> III Reverse Transcriptase (Invitrogen, catalog number: 18080-044)</li><li>T4 RNA Ligase 2, truncated (NEB, catalog number: M0242L)</li><li>T4 Polynucleotide Kinase (PNK) (NEB, catalog number: M0201L)</li><li>Ambion SUPERaseÂ·In<sup>TM</sup> (Invitrogen Life Technologies, catalog number: AM2696)</li><li>Nuclease S7 Micrococcal nuclease (MNase) (Roche, catalog number: 10107921001)</li><li>RNase Free DNase I (Roche, catalog number: 04716728001)</li><li>Phusion<sup>Â®</sup> High-Fidelity DNA Polymerase (NEB, catalog number: M0530L)</li><li>OâRangeRuler 10 bp Ladder (ThermoFisher, catalog number: SM1313)</li><li>Primers<ol class="npwthree"><li>Universal miRNA cloning linker (New England BioLabs, catalog number: S1315S)</li><li>Oligonucleotides from IDT:<br/>Size control 15 mer (100 nmol) (5â-AUGUACACGGAGUCG)<br/>Size control 45 mer (100 nmol):<br/>(5â-AUGUACACGGAGUCGACCCGCAACGCGAUGUACACGGAGUCGACC) </li><li> RT primer NI-NI-9 (250 nmol, DNA oligo, 96 bases, PAGE purified Ultramer)<br/>(/5Phos/AGATCGGAAGAGCGTCGTGTAGGGAAAGAGTGTAGATCTCGGTGGTCGC/iSP18/CACTCA/iSp18/TTCAGACGTGTGCTCTTCCGATCTATTGATGGTGCCTACAG-3)</li><li>PCR primer Forward (100 nmol; not gel purified):<br/>(5â-AATGATACGGCGACCACCGAGATCTACAC)</li><li>PCR primer Reverse (100 nmol; not gel purified): <br/>(5â-CAAGCAGAAGACGGCATACGAGAT<u>NNNNNN</u>GTGACTGGAGTTCAGACGTGTGCTCTTCCG)</li></ol><em>Note: The underlined region of PCR primer Reverse is the sequence that corresponds to the reverse complement of the barcode.</em><em></em><br/></li><li>Isopropanol</li><li>Ethanol</li><li>Dry ice</li><li>EGTA</li><li>MilliQ H<sub>2</sub>O </li><li>Sodium acetate</li><li>PEG 8000</li><li>Chloroform </li><li>NaOH</li><li>MgCl<sub>2</sub> </li><li>NH<sub>4</sub>Cl</li><li>CaCl<sub>2</sub> </li><li>Triton X-100</li><li>NP-40</li><li>SDS</li><li>EDTA</li><li>DTT</li><li>NaOAc</li><li>MOPS EZ Rich Defined Medium (Teknova, catalog number: M2105)</li><li>Phenol, Saturated pH 4.5 (VWR, catalog number: 97064-714)</li><li>Oligo Clean and Concentration kit (Zymo, catalog number: D4061)</li><li>Ribo-Zero<sup>TM</sup> Magnetic Kit (Gram-Negative Bacteria) (Epicentre, catalog number: MRZGN126)</li><li>Agilent High Sensitivity DNA Kit (Bio Analyzer) (Agilent Technologies, catalog number: 5067-4626)</li><li>GlycoBlue (Invitrogen Life Technologies, catalog number: AM9516)</li><li>SYBR<sup>Â®</sup> Gold Nucleic Acid Gel Stain (Invitrogen Life Technologies, catalog number: S-11494)</li><li>1 M Tris, pH 8 (Life Technologies, catalog number: AM9856)</li><li>D-Sucrose (Fisher, catalog number: BP220-1)</li><li>15% TBE Urea 1.0 mm (12 + 2) 45 Âµl gel (Bio-Rad, catalog number: 345-0091) (2-3 per experiment)</li><li>10% TBE Urea 1.0 mm (12 + 2) 45 Âµl gel (Bio-Rad, catalog number: 345-0088) (2-3 per experiment)</li><li>10% TBE Urea 1.0 mm (18 + 2) 30 Âµl gel (Bio-Rad, catalog number: 345-0089) (1 per experiment)</li><li>10% TBE Native 1.0 mm (12 + 2) 45 Âµl gel (Bio-Rad, catalog number: 345-0051) (4-6 per experiment)</li><li>10x Lysis Buffer (50 ml) (see Recipes)</li><li>Pelleting Buffer (see Recipes)</li><li>Resuspension Buffer (see Recipes)</li><li>10% Sucrose Gradient Buffer (see Recipes)</li><li>50% Sucrose Gradient Buffer (see Recipes)</li><li>RNA Elution Buffer (50 ml) (see Recipes)</li><li>DNA Elution Buffer (50 ml) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30761">### Equipment</p><ol class="npwtwok"><li>Kontes 90 mm filtration apparatus</li><li>Spex Sampleprep 6870 Freezer Mill</li><li>Beckman Coulter Optima L-80 XP Ultracentrifuge</li><li>SW41 Ti rotor</li><li>Type 70 Ti rotor</li><li>Polyclear Centrifuge tubes (Seton, catalog number: 7030)</li><li>Polycarbonate bottle with cap for Ti 70</li><li>Biocomp Gradient Maker and Fractionator</li><li>Thermocycler</li><li>Eppendorf Thermomixer R</li><li><a class="blackNoUnder" target="_blank"> GE Typhoon Imager </a> </li><li>Agilent 2100 BioAnalyzer<br/></li></ol><p class="pptt" id="biaoti30762">### Software</p><ol class="npwtwok"><li>Analysis of sequenced ribosome footprints was done using a custom python pipeline found in the link below: <a href="https://github.com/greenlabjhmi/2018_Bacterial_Pipeline_riboseq" target="_blank">https://github.com/greenlabjhmi/2018_Bacterial_Pipeline_riboseq</a> </li></ol><p class="pptt" id="biaoti30763">### Procedure</p><p class="ppzz">This protocol details every step of ribosome profiling from culturing cells up to PCR amplification of ribosome footprints in preparation for next-generation sequencing (NGS). A general outline of the workflow is described below:<br/><br/><strong>Day 1</strong><br/>â¢Growth of bacterial cultures <br/>â¢Harvest by whole culture freeze<br/>â¢Harvest by filtration (for total RNA extraction for RNA-seq)<br/>â¢Cell lysis by freezer mill<br/><br/><strong>Day 2</strong><br/>â¢Pellet ribosomes over sucrose cushion<br/>â¢Nuclease digestion<br/>â¢Sucrose gradient and monosome isolation<br/><br/><strong>Day 3</strong><br/>â¢Phenol chloroform extraction<br/>â¢Size selection gel<br/><br/><strong>Day 4</strong> (can split into 2 days)<br/>â¢Gel extraction<br/>â¢Dephosphorylation <br/>â¢Linker ligation<br/>â¢rRNA depletion<br/>â¢Reverse transcription<br/>â¢RT gel<br/><br/><strong>Day 5</strong><br/>â¢RT gel extraction <br/>â¢Circularization<br/>â¢Pilot PCR<br/>â¢Pilot PCR gel<br/>â¢Preparative PCR<br/>â¢Preparative PCR gel<br/><br/><strong>Day 6</strong><br/>â¢Preparative PCR gel extraction <br/>â¢Sequencing and data analyses<br/><br/><strong><em>Day 1</em></strong> </p><ol class="npwone"><li>Growth of bacterial cultures<br/>Bacterial cultures should be grown in the appropriate medium and growth conditions. When preparing cultures, consider the volume necessary to obtain around 1 mg of total RNA. The following conditions are used for <em>E. coli</em> during exponential growth:<ol class="npwtwo"><li>Inoculate an overnight starter culture (25 ml of MOPS EZ Rich media) from a single colony.</li><li>The next day, prepare 300 ml of MOPS EZ Rich media and warm at 37 Â°C for 1 h prior to inoculation.</li><li>Inoculate 300 ml of MOPS media with 1 ml of overnight culture.</li><li>Grow to an OD<sub>600</sub> = 0.3-0.5 (~2 h).<br/><br/></li></ol></li><li>Harvest by whole culture freeze<br/>This method of harvesting captures ribosomes without inducing artifacts from stress seen from harvesting by filtration or centrifugation. Under the growth conditions above, only 100 ml of the 300 ml will be flash frozen. The rest can be filter harvested to collect RNA for RNA-seq or frozen as backup.<ol class="npwtwo"><li>While cells are growing, prepare a liquid nitrogen bath:<ol class="npwthree"><li>Line an ice bucket with 2-3 layers of aluminum foil (see Figure 1A).</li><li>Fill bucket halfway with liquid nitrogen.</li></ol></li><li>When cells are at the correct OD<sub>600</sub>, use a 50 ml serological pipet and spray 50 ml of the culture directly into the liquid nitrogen bath. Make sure not to spray in one area. Instead make circles in the N<sub>2</sub> (see Figures 1B and 1C).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191215/20191215232145_7318.jpg" width="300"/><br/><strong>Figure 1. Liquid nitrogen bath used to freeze cultures.</strong> A. Line a standard lab ice bucket with 2-3 layers of aluminum foil. B. Add 2 inches of liquid nitrogen, then drip 100 ml of the bacterial culture into the liquid nitrogen. C. Drips should form ice pellets about 0.25 to 0.5 inches in diameter.<br/><br/></li><li>Repeat Step B2 to collect a total of 100 ml of culture.</li><li>If collecting cells for RNA-seq, immediately run the rest of the culture (200 ml) through the filtration harvest protocol below to collect cell pellets.</li><li>Lift the aluminum foil and pour off the liquid nitrogen from the sample. Scrape the frozen pellets into several 50 ml conical tubes on dry ice. Avoid thawing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Harvest by filtration (for total RNA extraction for RNA-seq)<br/><em>Note: This step is needed to create a sample matched RNA-seq library and can be skipped if RNA-seq is not necessary.</em> <ol class="npwtwo"><li>Set up the Kontes 90 mm filtration apparatus with Whatman cellulose nitrate 0.45 Âµm filters during cell growth.</li><li>Pour ~200 ml of culture into the Kontes 90 mm filtration apparatus and turn on the vacuum.</li><li>While the media is filtering, gently but quickly scrape with a plastic spatula along the surface until you get a good clump of cells. Freeze in liquid nitrogen.<br/><em>Note: You should be able to do this twice in about 60 s. The sample should never go dryâthe dryness and cold that comes with losing all the media stresses the cells.</em> </li><li>Place pellets in a 50 ml conical tube and store at -80 Â°C, do not allow any thawing.</li><li>RNA can be extracted using RNA extraction kits or phenol extraction from the cell pellet.</li><li>Extracted RNA can then be used to crate RNA-seq using commercially available kits.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell lysis by freezer mill<br/>Lysis is done through mechanical grinding of the frozen culture. The lysis buffer used contains 150 mM MgCl<sub>2</sub> to inhibit translation in the lysate.<ol class="npwtwo"><li>Pre-Chill Mixer Mill grinding cylinders and 3 x 50 ml conical tubes on dry ice.</li><li>Measure 50 g of frozen culture, making sure to avoid thawing. Add 5.5 ml of 10x lysis buffer to large grinding cylinder and allow it to freeze.</li><li>Transfer 50 g frozen culture into the grinding cylinder.</li><li>Grind using the following cycles: 10 cycles, 10 Hz, 5 min precool, 1 min run, 1 min cool.<br/><em>Note: The grinding protocol used here was optimized to maximize RNA yield while maintaining polysome integrity visualized by sucrose gradient sedimentation. We recommend revalidating grinding settings if using a different model Mixer Mill or a different organism.</em> </li><li>Transfer the pulverized cells into the pre-chilled 50 ml conical tubes.</li><li>Store at -80 Â°C for later use, or continue.</li></ol></li></ol><br/><strong><em>Day 2</em></strong> <ol class="npwone" start="5"><li>Pelleting ribosomes over sucrose cushion<ol class="npwtwo"><li>Thaw lysate on benchtop until thawed (1-2 h, mixing every 10 min). Place on ice immediately after thawed.</li><li>Pre-clear lysate by spinning for 10 min at 9,000 <em>x g</em> at 4 Â°C to remove cellular debris.</li><li>Prepare two sucrose cushions per sample:<ol class="npwthree"><li>Transfer 3 ml of pre-chilled pelleting buffer into a Ti 70 polycarbonate tube x 2.</li><li>Layer 22 ml lysate gently on top of cushion x 2.</li><li>Spin at 370,000 <em>x g</em> (60,000 rpm) for 1.5 h at 4 Â°C in a Ti 70 rotor to pellet ribosomes.</li></ol></li><li>Remove supernatant from the ribosome pellet and rinse each pellet with 150 Âµl of pre-chilled Resuspension Buffer without disturbing pellet.</li><li>To resuspended pellet:<ol class="npwthree"><li>Add 100 Âµl of Resuspension Buffer to each pellet.</li><li>Gently vortex for 5 min, making sure the ribosome pellet is fully dissolved.</li><li>Combine the 2 tubes for a total of 200 Âµl resuspended pellet.</li></ol></li><li>Measure A<sub>260</sub> of a 1:100 dilution using water.</li><li>Calculate the amount needed for 12.5 AU (roughly 0.5 mg RNA), where AU = 1 ml solution with a given A<sub>260</sub>.<br/><em>Note: If sample is less than 12.5 AU, lyse and pellet the other 50 ml of frozen culture. Resuspend with the first 200 Âµl collected. As little as 5 AU can be used for library preparation, but this is not recommended.</em> <ol class="npwthree"><li>12.5 AU/(A<sub>260 </sub>* 100/1,000) = Âµl needed (this takes into account the 100-fold dilution)</li><li>Use the calculation above to calculate the volume needed for 12.5 AU, and add Resuspension Buffer to bring volume to 200 Âµl.</li></ol></li><li>Freeze at -80 Â°C to store or continue.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Nuclease digestion <br/>Use 12.5 AU of sample in 200 Âµl that was pelleted for nuclease digestion.<br/><em>Note: If sample is less than 12.5 AU, scale the MNase accordingly.</em> <ol class="npwtwo"><li>Add 6 Âµl of SUPERase-In.</li><li>Add 2 Âµl of 375 U/Âµl MNase (total of 750 U).</li><li>Incubate at 25 Â°C for 1 h shaking at 1,400 rpm (thermomixer).</li><li>Add 2 Âµl of 0.5 M EGTA pH 8 to quench the reaction (EGTA chelates calcium to quench MNase, a calcium dependent enzyme).</li><li>Return to ice until next step.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sucrose gradient and monosome isolation<ol class="npwtwo"><li>Fill SW41 Polyallomer tubes with 6 ml of 50% Sucrose Gradient Buffer and layer on top ~6 ml of 10% Sucrose Gradient Buffer (all the way to the top of the tube). </li><li>Put on rubber caps carefully to allow any air bubbles to escape through the hole in the cap.</li><li>Rotate the tubes on the Biocomp gradient maker using the following settings:<br/>Time = 1:48 s<br/>Angle = 81.5Â°<br/>Speed = 17 rpm</li><li>Remove rubber caps and 150 Âµl off the top of the gradient to make room for the sample.</li><li>Gently layer 200 Âµl of the sample on top of gradient and balance in rotor tubes (sample should be approximately 210 Âµl).</li><li>Spin in ultracentrifuge at 201,000 <em>x g</em> (35,000 rpm) for 2.5 h at 4 Â°C using acceleration and deceleration setting of 7.</li><li>Prepare sucrose gradient fraction collector following the manufacturer's instructions.</li><li>Set fraction collection so that each fraction has approximately 350 Âµl.</li><li>Fractionate the sucrose gradient. Keep ~3 monosome fractions (~1,050 Âµl) (Figure 2A).</li><li>Freeze fractions in liquid nitrogen and store at -80 Â°C.</li></ol></li></ol><br/><strong><em>Day 3</em></strong> <ol class="npwone" start="8"><li>Phenol chloroform extraction<br/><em>Note: Use screwcap tubes and pipette in the fume hood. Spin steps are done on a tabletop centrifuge at &gt; 10,000 x g at room temperature (RT) unless otherwise specified.</em> <ol class="npwtwo"><li>Pre-warm 750 Âµl of phenol, saturated pH 4.5 at 65 Â°C in thermomixer.</li><li>Add 57.1 Âµl of 20% SDS to 1 ml of sample.</li><li>Add sample to heated phenol.</li><li>Incubate and shake for 5 min at 65 Â°C at 1,400 rpm on the thermomixer. Occasionally pull the tubes during this period and vortex on a full speed bench vortexer.</li><li>Chill samples on ice for 5 min.</li><li>Spin for 2 min.</li><li>Keep the <strong>bottom aqueous layer</strong> and add to a tube containing 700 Âµl of phenol, saturated pH 4.5 at room temperature.<br/><em><strong>Important:</strong> Due to the high concentration of sucrose, there is layer inversion and the aqueous layer is on the bottom.</em> </li><li>Vortex for 5 min at room temperature.</li><li>Spin for 2 min.</li><li>Keep the <strong>bottom aqueous layer</strong> and add to 600 Âµl of chloroform.</li><li>Vortex for 30 s.<br/><em>Note: If the samples become cloudy, warm to ~30 Â°C. This is likely the SDS falling out of solution.</em> </li><li>Spin for 2 min.</li><li>Keep the <strong>top aqueous layer</strong> (~1 ml) and place in a 15 ml conical tube.</li><li>Add 1 volume (1 ml) of MilliQ H<sub>2</sub>O to dilute the sucrose.</li><li>Add 3 M of sodium acetate (pH 5.5) to a final concentration of 0.3 M (~200 Âµl).</li><li>Add an equal volume of isopropanol and mix well.</li><li>Precipitate the RNA (30 min on dry ice, or overnight -20 Â°C).</li><li>Centrifuge in Eppendorf 5804R at 12,000 <em>x g</em> (~9,000 rpm) for 30 min. Set temperature to 4 Â°C.</li><li>Remove as much liquid as possible without disturbing the pellets.</li><li>Air dry samples in the hood.</li><li>Resuspend in 40 Âµl of MilliQ H<sub>2</sub>O or 10 mM Tris pH 8 and transfer to a 1.7 ml tube.</li><li>Measure A<sub>260.</sub> </li><li>You will run approximately 20 Âµg on the size selection gel.</li><li>Store samples at -80 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Size selection gel<ol class="npwtwo"><li>Prepare samples:<br/>20 Âµg Sample RNA + 2x RNA Loading Dye (NEB)<br/>Ladder (oligos: 15 nt and 45 nt) = Mix 3 Âµl (10 ÂµM stock) of each marker size + 6 Âµl 2x RNA Loading Dye per lane. Load 6 Âµl in one lane (enough for 2 gels)</li><li>Heat samples and ladder at 80 Â°C for 2 min, then place on ice.</li><li>Prepare<strong> 15% TBE Urea 1.0 mm (12 + 2) 45 Âµl gel</strong>.</li><li>Load samples carefully with at least one empty lane between samples.</li><li>Run gel(s) until bromophenol blue is Â¾ down the gel.<br/>If running at 15 W:<br/>1 gel = 17 min<br/>2 gels = 30 min</li><li>Remove gels from plates and place in SYBR-gold stain (100 ml of 1x TBE + 10 Âµl of SYBR-gold) for 5 min.</li><li>Remove from stain and place on transparency film, then cover with saran wrap.</li><li>Scan on the GE Typhoon Imager using the fluorescence setting for SYBR Gold. Use a 350-400 PMT.</li><li>Print the gel as actual size and use printout as a cutting guide to cut sections out of the gel. Cut from the top of the upper marker (45 nt) to bottom of lower marker (15 nt) (Figure 2B).</li><li>Cut out 15 nt and 45 nt markers and treat as a control sample. This can be used to assess proper ligation and reverse transcription later on.</li><li>Rescan gel to confirm proper cutting.</li><li>For elution, crush gel by doing the following:<ol class="npwthree"><li>Place gel cutout into a 0.5 ml tube with holes poked through the bottom (18 G needle).</li><li>Place the 0.5 ml tube in a 1.7 ml tube to collect the gel.</li><li>Spin for 5 min, gel should extrude from the 0.5 ml tube into the bottom 1.7 ml tube.</li><li>Make sure to collect any remaining gel from the 0.5 ml tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li></ol><strong><em>Day 4</em></strong> (can be split into 2 days)<ol class="npwone" start="10"><li>Gel extraction<ol class="npwtwo"><li>Add 500 Âµl of RNA Elution Buffer to each sample.</li><li>Add 2.5 Âµl of SUPERase-Inhibitor to each sample.</li><li>Shake overnight in a thermomixer, 1,400 rpm at 4 Â°C.</li><li>The next day, briefly spin samples.</li><li>Transfer gel slurry into Spin-X columns (using P1000 tips with the ends cut off).</li><li>Spin for 3 min at 20,000<em> x g</em> at room temperature.</li><li>Put eluate into a new 1.7 ml tube.</li><li>Add 2 Âµl of GlycolBlue to each sample.</li><li>Add 650 Âµl of isopropanol.</li><li>Precipitate (40 minâdry ice), spin for 30 min at &gt; 10,000 <em>x g</em>, 4 Â°C.</li><li>Remove supernatant, and gently wash with ~100 Âµl of 80% cold ethanol.</li><li>Dry pellets by speed vac (10 min).</li><li>Resuspend pellet in 5 Âµl of MilliQ H<sub>2</sub>O.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Dephosphorylation<br/>Reaction Mix (per sample)<br style="color:#666666;font-family:Arial, Helvetica, sans-serif;font-size:13.3333px;text-align:justify;white-space:normal;"/>1 Âµl of 10x PNK buffer<br style="color:#666666;font-family:Arial, Helvetica, sans-serif;font-size:13.3333px;text-align:justify;white-space:normal;"/>1 Âµl of SUPERase-In<br style="color:#666666;font-family:Arial, Helvetica, sans-serif;font-size:13.3333px;text-align:justify;white-space:normal;"/>2 Âµl of MilliQ H<sub>2</sub>O<br/><br/><ol class="npwtwo"><li>Heat samples collected from J13 at 80 Â°C for 2 min and then place on ice.</li><li>Add 4 Âµl of Reaction Mix to each sample.</li><li>Add 1 Âµl of PNK to each sample (10K U/Âµl).</li><li>Incubate at 37 Â°C for 1 h.</li></ol><br/></li><li>Linker ligation<br/>Reaction Mix (per sample):<br/>1 Âµl of 10x Ligation Buffer<br/>0.4 Âµl of 50 ÂµM Universal miRNA cloning linker<br/>7.6 Âµl of 50% PEG 8000<br/><em>Note: Ligation buffer and PNK buffer are nearly identical, so half of the required ligation buffer is used.</em><br/><br/><ol class="npwtwo"><li>Add 9 Âµl of Reaction Mix to each sample. Any white precipitate is assumed to be glycogen coming out as a result of the PEG.</li><li>Add 1 Âµl of T4 RNA ligase 2 truncated to each sample.</li><li>Incubate at 37 Â°C for 3 h.</li><li>Clean up with oligo and concentration kit and elute in 15 Âµl of MilliQ H<sub>2</sub>O.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>rRNA depletion<br/>Deplete rRNA for all samples except the 15 nt and 45 nt controls.<ol class="npwtwo"><li>Follow Ribo-Zero kit instructions for rRNA depletion.</li><li>Clean up with oligo and concentration kit and elute in 15 Âµl of MilliQ H<sub>2</sub>O.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Reverse transcription (RT)<br/>Reaction Mix (per sample):<br/>4 Âµl of 5x RT buffer<br/>1 Âµl of 10 mM dNTPs<br/>1 Âµl of 0.1 M DTT<br/>1 Âµl of SUPERase-In<br/><br/><ol class="npwtwo"><li>Use 10.75 Âµl of sample for RT, save the other 4.25 Âµl.</li><li>Add 1.25 Âµl of RT primer NI-NI-9 (10 ÂµM stock) and heat sample to 80 Â°C for 2 min, then chill on ice.</li><li>Add 7 Âµl of Reaction mix to each sample.</li><li>Add 1 Âµl of Superscript III RT to each sample (Invitrogen 56575).</li><li>Incubate at 48 Â°C for 30 min.</li><li>Add 2.2 Âµl of 1 M NaOH to each sample.</li><li>Heat to 98 Â°C for 20 min.</li><li>Add 28 Âµl of MilliQ H<sub>2</sub>O.</li><li>Clean up with oligo and concentration kit (Zymo) and elute in 10 Âµl of MilliQ H<sub>2</sub>O.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RT Gel<ol class="npwtwo"><li>Prepare Samples:<br/>10 Âµl of sample cDNA + 10 Âµl of 2x RNA Loading Dye<br/>5 Âµl of 10 bp O'RangeRuler ladder<br/><em>Note: The sample containing the 15 and 45 nt markers will be used to indicate proper ligation and reverse transcription, and act as size markers for gel excision.</em> </li><li>Heat to 80 Â°C for 2 min and then put on ice. </li><li>Prepare <strong>10% TBE Urea 1.0 mm (18 + 2) 30 Âµl gel</strong>.</li><li>Load samples carefully with at least one empty lane between samples.</li><li>Run gel(s) until xylene cyanol is run off the gel.<br/>If running at 15 W:<br/>1 gel = 40 min<br/>2 gels = 60 min</li><li>Remove gels from plates and place in SYBR-gold stain (100 ml 1x TBE + 10 Âµl SYBR-gold) for 5 min.</li><li>Remove from stain and place on transparency film, then cover with saran wrap.</li><li>Scan on the GE Typhoon Imager using the fluorescence setting for SYBR Gold. Use a 350-400 PMT.</li><li>Print the gel as actual size and use printout as a cutting guide to cut sections out of the gel. Cut from the top of the upper marker (45 nt) to bottom of lower marker (15 nt) (Figure 2C).<br/><em><strong>Important:</strong> Avoid excess RT primer band (band below 15 nt marker, 110 nt in size determined by the O'RangeRuler ladder).</em> </li><li>Rescan to confirm proper cutting.</li><li>For elution, crush gel by doing the following:<ol class="npwthree"><li>Place gel cutout into a 0.5 ml tube with holes poked through the bottom (18 G needle).</li><li>Place the 0.5 ml tube in a 1.7 ml tube to collect the gel.</li><li>Spin for 4 min, gel should extrude from the 0.5 ml tube into the bottom 1.7 ml tube.</li></ol></li></ol><div style="white-space:nowrap;"><br/></div></li></ol><strong><em>Day 5</em></strong> <ol class="npwone" start="16"><li>RT Gel Extraction<ol class="npwtwo"><li>Add 500 Âµl <strong><u>DNA Elution Buffer</u></strong> to each sample.</li><li>Shake overnight in thermomixer 1,050 rpm at 25 Â°C (room temperature).</li><li>The next day, briefly spin samples.</li><li>Transfer gel slurry into Spin-X columns (using P1000 tips with the ends cut off).</li><li>Spin for 3 min.</li><li>Put eluate in new 1.7 ml tube.</li><li>Add 2 Âµl of GlycoBlue to each sample.</li><li>Add 650 Âµl of isopropanol.</li><li>Precipitate (40 minâdry ice), spin for 30 min at &gt; 10,000 <em>x g</em>, 4 Â°C.</li><li>Remove supernatant, and gently wash with ~100 Âµl 80% cold ethanol.</li><li>Dry pellets by speed vac (10 min).</li><li>Resuspend pellets in 15 Âµl of 10 mM Tris pH 8.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Circularization<br/>Reaction mix (per sample):<br/>2 Âµl of CircLigase<sup>TM</sup> buffer<br/>1 Âµl of 1 mM ATP (from CircLigase<sup>TM</sup> Kit)<br/>1 Âµl of 50 mM MnCl<sub>2</sub> (from CircLigase<sup>TM</sup> Kit)<br/><br/><ol class="npwtwo"><li>Add 4 Âµl of Reaction Mix to each sample</li><li>Add 1 Âµl of Circ ligase to each sample.</li><li>Incubate at 60 Â°C for 60 min.</li><li>Incubate at 80 Â°C for 10 min, then put on ice.</li></ol><br/></li><li>Pilot PCR<br/>The goal of the PCR step is to amplify the sample while minimizing PCR amplification bias and over-amplification. To estimate the cycles of PCR needed for proper amplification, several pilot cycles are run to visualize the point at which over-amplification occurs.<br/><br/><strong>Reaction Mix (6 samples)</strong><br/>40 Âµl of HF Buffer (F-518)<br/>4 Âµl of 10 mM dNTPs<br/>1 Âµl of 100 ÂµM PCR primer Forward<br/>2 Âµl of Phusion polymerase (F-530)<br/>133 Âµl of MilliQ H<sub>2</sub>O<br/><br/><ol class="npwtwo"><li>Prepare Reaction mix.</li><li>Combine in a 1.7 ml tube:<br/>27 Âµl Reaction mix<br/>1.5 Âµl barcoded PCR primer Reverse (10 ÂµM)<br/>1.5 Âµl Sample</li><li>Divide this into 3 x 9 Âµl aliquots into 0.5 ml tubes.</li><li>Run on PCR protocol:<br/>Step 1: 98 Â°Câ30 s<br/>Step 2: 98 Â°Câ10 s<br/>Step 3: 65 Â°Câ10 s<br/>Step 4: 72 Â°Câ5 s<br/>Step 5: go to step 2â8 times<br/>Step 6: 4 Â°Câforever<br/>End<br/>Each sample will have 3 aliquots in the PCR machine. Take one aliquot from each sample out of the PCR machine at the end of 5, 7, and 9 PCR amplification cycles.<br/><em>Notes:<ol class="npwthree"><li>Pause the machine at 72 Â°C, wait for 5 s.</li><li>Quickly remove tubes and place directly on ice.</li><li>Quickly close the lid and continue the program.</li></ol></em> </li><li>Each sample will now have 3 PCR products made from 3 different PCR cycle numbers.</li><li>Proceed to Pilot PCR gel.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Pilot PCR gel<ol class="npwtwo"><li>Prepare Samples:<br/>9 Âµl of PCR sample + 2 Âµl of 6x DNA Loading Dye<br/>5 Âµl of 10 bp O'RangeRuler ladder</li><li>Prepare <strong>10% TBE Native 1.0 mm (12 + 2) 45 Âµl gel</strong>.</li><li>Load samples carefully, grouping each sample together with the cycle numbers in ascending order.</li><li>Run gel(s) until xylene cyanol is run off the gel.<br/>If running at 15 W:<br/>1 gel = 40 min<br/>2 gels = 60 min</li><li>Remove gels from plates and place in SYBR-gold stain (100 ml of 1x TBE + 10 Âµl of SYBR-gold) for 5 min.</li><li>Remove from stain and place on transparency film, then cover with saran wrap.</li><li>Scan on the GE typhoon Imager using the fluorescence setting for SYBR Gold. Use a 350-400 PMT (Figure 2D).</li><li>Determine how many cycles are needed to obtain a good band around 150-190 bp without any larger molecular weight products.<br/><em>Note: The ideal number of cycles gives you a nice product band but no trace of higher molecular weight products. Be very careful to avoid this because when you see higher molecular weight products, bias is likely being introduced. After determining the appropriate number of cycles from the Pilot PCR, run the Preparative PCR at 1 fewer cycles to ensure that no high weight bands (usually a smear) emerge.</em> </li><li>Proceed to Preparative PCR.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparative PCR<br/>Determine the proper cycle count for each sample. Follow the same steps as the pilot PCR, but do not divide the reaction.<ol class="npwtwo"><li>Combine in 1.7 ml tube:<br/>27 Âµl of Reaction mix<br/>1.5 Âµl of barcoded PCR primer (10 ÂµM) (can use either Ni or IDX primers)<br/>1.5 Âµl of Sample </li><li>Run PCR using the cycle count determined from the pilot PCR.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparative PCR gel<ol class="npwtwo"><li>Prepare Samples:<br/>30 Âµl of PCR sample + 6 Âµl of 6x DNA Loading Dye <br/>5 Âµl of 10 bp O'RangeRuler ladder</li><li>Prepare <strong>10% TBE Native 1.0 mm (12 + 2) 45 Âµl gel</strong>.</li><li>Load samples carefully, leaving one lane empty between samples.</li><li>Run gel(s) until xylene cyanol is run off the gel.<br/>If running at 15 W:<br/>1 gel = 40 min<br/>2 gels = 60 min</li><li>Remove gels from plates and place in SYBR-gold stain (100 ml of 1x TBE + 10 Âµl of SYBR-gold) for 5 min.</li><li>Remove from stain and place on transparency film, then cover with saran wrap.</li><li>Scan on the GE Typhoon Imager using the fluorescence setting for SYBR Gold. Use a 350-400 PMT.</li><li>Print the gel as actual size and use printout as a cutting guide to cut sections out of the gel. Cut from the 150 bp marker to the 200 bp marker (Figure 2E).<br/><em><strong>Important:</strong> Avoid the excess RT primer band (runs at 140 bp mark).</em> </li><li>Rescan to confirm proper cutting.</li><li>For elution, crush gel by doing the following:<ol class="npwthree"><li>Place gel cutout into a 0.5 ml tube with holes poked through the bottom (18 G needle).</li><li>Place the 0.5 ml tube in a 1.7 ml tube to collect the gel.</li><li>Spin for 5 min, gel should extrude from the 0.5 ml tube into the bottom 1.7 ml tube.</li></ol></li></ol></li></ol><br/><strong><em>Day 6</em></strong> <ol class="npwone" start="22"><li>Preparative PCR gel extraction<ol class="npwtwo"><li>Add 500 Âµl of DNA Elution Buffer to each sample.</li><li>Shake overnight in thermomixerâ1,050 rpm at 25 Â°C (room temperature).</li><li>The next day, briefly spin samples.</li><li>Transfer gel slurry into Spin-X columns (using P1000 tips with the ends cut off).</li><li>Spin for 3 min at 20,000 <em>x g</em> at RT.</li><li>Put eluate in new 1.7 ml tube.</li><li>Add 2 Âµl of GlycoBlue to each sample.</li><li>Add 650 Âµl of isopropanol.</li><li>Precipitate (40 minâdry ice), spin for 30 min at &gt; 10,000 <em>x g</em>, 4 Â°C.</li><li>Remove supernatant, and gently wash with ~100 Âµl of 80% cold ethanol.</li><li>Dry pellets by speed vac (10 min).</li><li>Resuspend pellets in 10 Âµl of 10 mM Tris pH 8.</li><li>Check size and concentration using a BioAnalyzer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sequencing and data analyses<ol class="npwtwo"><li>Dilute an aliquot of each library to 2 nM and combine together to multiplex.</li><li>Submit 10 Âµl of 2 nM multiplexed sample for 50 cycles, single-end sequencing on the Illumina HiSeq 2500.</li><li>Process the sequencing data with custom python 2.7 scripts available here:<br/><a href="https://github.com/greenlabjhmi/2018_Bacterial_Pipeline_riboseq" target="_blank">https://github.com/greenlabjhmi/2018_Bacterial_Pipeline_riboseq</a>.</li><li>In broad strokes, the Jupyter notebook Github-Ribo_Density.ipynb.<ol class="npwthree"><li>Remove the linker CTGTAGGCACCATCAATAGATCGGAAGAGCACACGTCTGAACTCCA GTCA from the 3â-end of reads using Skewer version 0.2.2.</li><li>Use Bowtie version 0.12.7 to align reads to indices including ladder oligos, tRNA, and rRNA; reads that do not map to these indices are then mapped to the <em>E. coli</em> genome (NC_000913.2) allowing two mismatches and requiring unique mapping to a single site.</li><li>Write out ribosome density as a WIG file, as a binary file, and a size-separated dictionary using the 3â-end of reads to assign ribosome position.</li></ol></li><li>The Jupyter notebook Github-Ribo-Analysis.ipynb provides further analyses including plots of average ribosome density at start or stop codons, the calculation of pause scores at specific codons, reading frame, and the degree of drop-off across genes (their asymmetry scores).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191215/20191215232029_5672.jpg" width="400"/><br/><strong>Figure 2. Ribosome profiling sucrose gradients and gels.</strong> A. Sucrose gradient trace from Step G9. A volume of ~1,050 Âµl was collected (highlighted in red) containing 70S ribosomes. B. Size selection gel from Step I9. Lane 1: 15 nt and 45 nt size markers. Lanes 2 and 3: RNA from fractionated monosomes. The red box indicates the excised region containing ribosome footprints (FP). C. Reverse transcription gel from Step O9. Lane 1: ligated and reverse transcribed 15 nt and 45 nt size markers. Lanes 2 and 3: ligated and reverse transcribed ribosome footprints. The red box indicates excised region which avoids the excess RT primer band. D. Pilot PCR gel from Step S7. Lanes 1-3: PCR amplified products after 5, 7, and 9 PCR cycles. Note the appearance of unwanted high molecular weight products after 7 and 9 PCR cycles. E. Preparative PCR gel from Step U8. The red box indicates excised region which avoids the RT primer band. FP = ribosome footprints. L = OâRangeRuler 10 bp ladder.<br/></li></ol></li></ol><p class="pptt" id="biaoti30764">### Recipes</p><p class="ppzz"><em>Note: All concentrations listed are final concentrations.</em> </p><ol class="npwtwok"><li>10x Lysis Buffer (50 ml)<br/>200 mM Tris pH 8 <br/>1.5 M MgCl<sub>2</sub> <br/>1 M NH<sub>4</sub>Cl <br/>50 mM CaCl<sub>2 </sub><br/>4% Triton X-100<br/>1% NP-40<br/>MilliQ H<sub>2</sub>O</li><li>Pelleting Buffer<br/>1.1 M Sucrose <br/>20 mM Tris pH 8<br/>500 mM NH<sub>4</sub>Cl<br/>10 mM MgCl<sub>2</sub><br/>0.5 mM EDTA pH 8<br/>MilliQ H<sub>2</sub>O</li><li>Resuspension Buffer <br/>20 mM Tris pH 8 <br/>15 mM MgCl<sub>2</sub><br/>100 mM NH<sub>4</sub>Cl<br/>5 mM CaCl<sub>2</sub><br/>MilliQ H<sub>2</sub>O</li><li>10% Sucrose Gradient Buffer<br/>Each sample uses 6 ml<br/>20 mM Tris pH 8 <br/>15 mM MgCl<sub>2</sub><br/>100 mM NH<sub>4</sub>Cl<br/>2 mM DTT<br/>10% sucrose <br/>MilliQ H<sub>2</sub>O</li><li>50% Sucrose Gradient Buffer <br/>Each sample uses 6 ml<br/>20 mM Tris pH 8 <br/>15 mM MgCl<sub>2</sub><br/>100 mM NH<sub>4</sub>Cl<br/>2 mM DTT<br/>50% sucrose <br/>MilliQ H<sub>2</sub>O</li><li>RNA Elution Buffer (50 ml) <br/>300 mM NaOAc pH 5.5<br/>1 mM EDTA pH 8<br/>MilliQ H<sub>2</sub>O</li><li>DNA Elution Buffer (50 ml) <br/>300 mM NaCl<br/>1 mM EDTA pH 8 <br/>10 mM Tris pH 8 <br/>MilliQ H<sub>2</sub>O<br/></li></ol><p class="pptt" id="biaoti30765">### Acknowledgments</p><p class="ppzz">Our work on ribosome profiling was supported by NIH grant GM110113. This protocol was adapted from the early bacterial profiling studies by Bukau and Weissman (Oh <em>et al.</em>, 2011; Li <em>et al.</em>, 2012; Becker <em>et al.</em>, 2013) and incorporates recent improvements described by Ingolia for use in yeast and mammalian cells (McGlincy and Ingolia, 2017).</p><p class="pptt" id="biaoti30766">### Competing interests</p><p class="ppzz">There are no conflicts of interest or competing interest.</p><p class="pptt" id="biaoti30767">### References</p><ol class="npwtwok"><li>Baggett, N. E., Zhang, Y. and Gross, C. A. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28301469" target="_blank">Global analysis of translation termination in <em>E. coli</em>.</a> <em>PLoS Genet</em> 13(3): e1006676. </li><li>Balakrishnan, R., Oman, K., Shoji, S., Bundschuh, R. and Fredrick, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25378333" target="_blank">The conserved GTPase LepA contributes mainly to translation initiation in <em>Escherichia coli</em>.</a> <em>Nucleic Acids Res</em> 42(21): 13370-13383. </li><li>Becker, A. H., Oh, E., Weissman, J. S., Kramer, G. and Bukau, B. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24136347" target="_blank">Selective ribosome profiling as a tool for studying the interaction of chaperones and targeting factors with nascent polypeptide chains and ribosomes.</a> <em>Nat Protoc</em> 8(11): 2212-2239. </li><li>Burkhardt, D. H., Rouskin, S., Zhang, Y., Li, G. W., Weissman, J. S. and Gross, C. A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28139975" target="_blank">Operon mRNAs are organized into ORF-centric structures that predict translation efficiency.</a> <em>Elife</em> 6: e22037.</li><li>Elgamal, S., Katz, A., Hersch, S. J., Newsom, D., White, P., Navarre, W. W. and Ibba, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25144653" target="_blank">EF-P dependent pauses integrate proximal and distal signals during translation.</a> <em>PLoS Genet</em> 10(8): e1004553. </li><li>Guo, M. S., Updegrove, T. B., Gogol, E. B., Shabalina, S. A., Gross, C. A. and Storz, G. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25030700" target="_blank">MicL, a new sigmaE-dependent sRNA, combats envelope stress by repressing synthesis of Lpp, the major outer membrane lipoprotein.</a> <em>Genes Dev</em> 28(14): 1620-1634.</li><li>Haft, R. J., Keating, D. H., Schwaegler, T., Schwalbach, M. S., Vinokur, J., Tremaine, M., Peters, J. M., Kotlajich, M. V., Pohlmann, E. L., Ong, I. M., Grass, J. A., Kiley, P. J. and Landick, R. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24927582" target="_blank">Correcting direct effects of ethanol on translation and transcription machinery confers ethanol tolerance in bacteria.</a> <em>Proc Natl Acad Sci U S A</em> 111(25): E2576-E2585.</li><li>Ingolia, N. T., Ghaemmaghami, S., Newman, J. R. and Weissman, J. S. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19213877" target="_blank">Genome-wide analysis <em>in vivo</em> of translation with nucleotide resolution using ribosome profiling.</a> <em>Science</em> 324(5924): 218-223.</li><li>Kannan, K., Kanabar, P., Schryer, D., Florin, T., Oh, E., Bahroos, N., Tenson, T., Weissman, J. S. and Mankin, A. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25349425" target="_blank">The general mode of translation inhibition by macrolide antibiotics.</a> <em>Proc Natl Acad Sci U S A</em> 111(45): 15958-15963.</li><li>Lalanne, J. B., Taggart, J. C., Guo, M. S., Herzel, L., Schieler, A. and Li, G. W. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29606352" target="_blank">Evolutionary convergence of pathway-specific enzyme expression stoichiometry.</a> <em>Cell</em> 173(3): 749-761 e38. </li><li>Li, G. W., Burkhardt, D., Gross, C. and Weissman, J. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24766808" target="_blank">Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources.</a> <em>Cell</em> 157(3): 624-635. </li><li>Li, G. W., Oh, E. and Weissman, J. S. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22456704" target="_blank">The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.</a> <em>Nature</em> 484(7395): 538-541.</li><li>Liu, X., Jiang, H., Gu, Z. and Roberts, J. W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23812753" target="_blank">High-resolution view of bacteriophage lambda gene expression by ribosome profiling.</a> <em>Proc Natl Acad Sci U S A </em>110(29): 11928-11933.</li><li>Marks, J., Kannan, K., Roncase, E. J., Klepacki, D., Kefi, A., Orelle, C., Vazquez-Laslop, N. and Mankin, A. S. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27791002" target="_blank">Context-specific inhibition of translation by ribosomal antibiotics targeting the peptidyl transferase center.</a> <em>Proc Natl Acad Sci U S A</em> 113(43): 12150-12155.</li><li>McGlincy, N. J. and Ingolia, N. T. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28579404" target="_blank">Transcriptome-wide measurement of translation by ribosome profiling.</a> <em>Methods</em> 126: 112-129. </li><li>Meydan, S., Marks, J., Klepacki, D., Sharma, V., Baranov, P. V., Firth, A. E., Margus, T., Kefi, A., Vazquez-Laslop, N. and Mankin, A. S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30904393" target="_blank">Retapamulin-Assisted Ribosome Profiling Reveals the Alternative Bacterial Proteome.</a> <em>Mol Cell</em> 74(3):481-493.</li><li>Mohammad, F., Green, R. and Buskirk, A. R. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30724162" target="_blank">A systematically-revised ribosome profiling method for bacteria reveals pauses at single-codon resolution.</a> <em>Elife</em> 8: 42591.</li><li>Oh, E., Becker, A. H., Sandikci, A., Huber, D., Chaba, R., Gloge, F., Nichols, R. J., Typas, A., Gross, C. A., Kramer, G., Weissman, J. S. and Bukau, B. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22153074" target="_blank">Selective ribosome profiling reveals the cotranslational chaperone action of trigger factor <em>in vivo</em>.</a> <em>Cell</em> 147(6): 1295-1308.</li><li>Shell, S. S., Wang, J., Lapierre, P., Mir, M., Chase, M. R., Pyle, M. M., Gawande, R., Ahmad, R., Sarracino, D. A., Ioerger, T. R., Fortune, S. M., Derbyshire, K. M., Wade, J. T. and Gray, T. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26536359" target="_blank">Leaderless Transcripts and Small Proteins Are Common Features of the Mycobacterial Translational Landscape.</a> <em>PLoS Genet</em> 11(11): e1005641.</li><li>Subramaniam, A. R., Deloughery, A., Bradshaw, N., Chen, Y., O'Shea, E., Losick, R. and Chai, Y. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24347549" target="_blank">A serine sensor for multicellularity in a bacterium.</a> <em>Elife</em> 2: e01501.</li><li>Weaver, J., Mohammad, F., Buskirk, A. R. and Storz, G. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30837344" target="_blank">Identifying small proteins by ribosome profiling with stalled initiation complexes.</a> <em>MBio</em> 10(2).</li><li>Woolstenhulme, C. J., Guydosh, N. R., Green, R. and Buskirk, A. R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25843707" target="_blank">High-precision analysis of translational pausing by ribosome profiling in bacteria lacking EFP.</a> <em>Cell Rep</em> 11(1): 13-21.</li><li>Zhang, Y., Burkhardt, D. H., Rouskin, S., Li, G. W., Weissman, J. S. and Gross, C. A. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29628307" target="_blank">A stress response that monitors and regulates mRNA structure is central to cold shock adaptation.</a> <em>Mol Cell</em> 70(2): 274-286 e7. <br/></li></ol>