<body>
            <h1>A Yeast Chromatin-enriched Fractions Purification Approach,  yChEFs, from <em>Saccharomyces cerevisiae</em></h1>
            <p><strong>Authors:</strong> Abel Cuevas-BermÃºdez, Ana I. Garrido-Godino, Francisco GutiÃ©rrez-Santiago, Francisco  Navarro</p>
            <h2>Abstract</h2>
            <p>We have adapted a previous procedure and improved an approach that we named yChEFs (yeast Chromatin Enriched Fractions) for purifying chromatin fractions. This methodology allows the easy, reproducible and scalable recovery of proteins associated with chromatin. By using yChEFs, we bypass subcellular fractionation requirements involved when using zymolyase to obtain the spheroplast, which is employed in many other procedures. Employing small amount of culture cells and small volumes of solutions during the yChEFs procedure is very useful to allow many samples to be handled at the same time, and also reduces costs and efforts. The purified proteins associated with chromatin fractions obtained by yChEFs can be analyzed by Western blot (Figure 1) or combined with mass spectrometry for proteomic analyses.</p>
            <p><strong>Keywords:</strong> Chromatin, Proteins, Yeast, Mass spectrometry, <em>Saccharomyces cerevisiae</em></p>
            <h2>Background</h2><p>DNA-protein macrocomplexes are usually insoluble and lost while obtaining chromatin fractions (Lambert <em>et al.</em>, 2009). To maximize DNA-protein recovery, cellular fractionation has been used in several organisms, such as <em>Xenopus</em> (Liang and Stillman, 1997; Khoudoli <em>et al.</em>, 2008), <em>Caenorhabditis elegans</em> (Chu <em>et al.</em>, 2006), humans (Du <em>et al.</em>, 2006; Foltz <em>et al.</em>, 2006; Foltz <em>et al.</em>, 2009), rice (Tan <em>et al.</em>, 2007) or budding yeast (Frei and Gasser, 2000; Kubota <em>et al.</em>, 2011). In most cases, these studies have been accompanied by proteomic analyses conducted by mass spectrometry, including quantitative mass spectrometry (Kubota <em>et al.</em>, 2011; Lambert <em>et al.</em>, 2012). The chromatin immunoprecipitation method (ChIP) emerged as an alternative to the above-mentioned approaches and allows interactions to be identified between protein and genomic DNA regions (Aparicio <em>et al.</em>, 2004). However, the use of formaldehyde for <em>in vivo</em> crosslinking renders this method less efficient for proteomic analyses (Metz <em>et al.</em>, 2004). ChIP combined with mass spectrometry has also been used in mouse embryonic stem cells or other model organisms (Lambert <em>et al.</em>, 2010; Engelen <em>et al.</em>, 2015). Notably a method based on ChIP, termed mChIP, has also been developed including protein affinity purification from chromatin that is most useful for mass spectrometry analyses (Lambert <em>et al.</em>, 2009 and 2010).<br/><br/>In budding yeast, many procedures use cell fractionation by including additional steps of zymolyase treatment to lyse cell wall of viable yeast cells and then, to obtain the spheroplast before nucleus and cytoplasm separation (Liang and Stillman, 1997; Frei and Gasser, 2000; Kubota <em>et al.</em>, 2011). Finally, a method to obtain chromatin fractions to isolate nascent RNA has been recently developed in <em>S. cerevisiae</em> which, without following a prior zymolyase treatment, allows chromatin fractions depleted of rRNA and tRNA to be obtained (Carrillo Oesterreich <em>et al.</em>, 2010).<br/><br/>However, the need of easy and scalable procedure to obtain highly chromatin-enriched fractions in <em>S. cerevisiae</em> led us to adapt and improve this methodology termed yChEFs for yeast Chromatin Enriched Fractions. This procedure does away with the need to perform the cell fractionation required to obtain the spheroplast by using zymolyase incubation and to achieve nucleus isolation, which greatly facilitates the procedure and makes chromatin isolation more efficient. In addition, yChEFs allows work to be done with small solution volumes, very useful if many samples need to be manipulated. This method, allows proteins to be isolated from non-crosslinked chromatin by avoiding later sonication steps after chromatin isolation, and can be combined with mass spectrometry for proteomic analyses.</p>
        </body>

<p class="pptt" id="biaoti30814">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml centrifuge FALCON-type tubes</li><li>1.5 ml microtubes</li><li>Cuvettes</li><li>Glass beads (Retsch, catalog number: 22.222.0004)</li><li>anti-Histone H3 (Abcam, catalog number: ab1791)</li><li>anti-Pgk1 (Invitrogen, catalog number: 459250)</li><li>Anti-Rabbit IgG (Sigma-Aldrich, catalog number: A0545)</li><li>Ani-Mouse IgG (Bio-Rad, catalog number: 170-6516)</li><li>Cell culture media:<br/>Peptone (FORMEDIUM, catalog number: PEPCFG)<br/>Glucose (FORMEDIUM, catalog number: GLUCFG)<br/>Galactose (FORMEDIUM, catalog number: GALCFG)<br/>Yeast extract (FORMEDIUM, catalog number: YEACFG)<br/>Yeast nitrogen base (CONDA, catalog number: 1553)<br/>Ammonium sulfate (PanReac AppliChem, catalog number: 141140)<br/>Uracil (Sigma-Aldrich, catalog number: U-1128)<br/>L-Methionine (FORMEDIUM, catalog number: DOC0168)<br/>L-Tryptophan ((FORMEDIUM, catalog number: DOC0187)<br/>Adenine sulphate (FORMEDIUM, catalog number: DOC0229)<br/>L-Leucine (SIGMA ALDRICH, catalog number: L8912)<br/>L-Lysine monohydrate (FORMEDIUM, catalog number: DOC0159)<br/>L-Histidine (FORMEDIUM, catalog number: DOC0144)</li><li>MilliQ H<sub>2</sub>O</li><li>HEPES (iNtRON BIOTECHNOLOGY, catalog number: 1017365)</li><li>KCl (PanReac AppliChem, catalog number: 131494)</li><li>NaCl (PanReac AppliChem catalog number: 121659)</li><li>MgCl<sub>2</sub> (Sigma-Aldrich, catalog number: M9272)</li><li>CaCl<sub>2</sub> (PanReac AppliChem catalog number: 131232)</li><li>Triton X-100 (PanReac AppliChem, catalog number: 142314)</li><li>Sucrose (Sigma-Aldrich, catalog number: S9378)</li><li>Spermidine (Sigma-Aldrich, catalog number: S2626)</li><li>Spermine (Sigma-Aldrich, catalog number: S3256)</li><li>EDTA (PanReac AppliChem, catalog number: 131669)</li><li>Glycerol (PanReac AppliChem, catalog number: 141339)</li><li>IGEPAL CA-630 (NP-40) (Sigma-Aldrich, catalog number: I8896)</li><li>Urea (Sigma-Aldrich, catalog number: U-6504)</li><li>DTT (FORMEDIUM, catalog number: DTTCFG)</li><li>PMSF (Sigma-Aldrich, catalog number: P7626)</li><li>Tris (PanReac AppliChem, catalog number: 141940)</li><li>SDS (BioPure, catalog number: 8030S)</li><li>Bromophenol blue (Sigma-Aldrich, catalog number: B5525)</li><li>Î-Mercaptoethanol (Sigma-Aldrich, catalog number: M3148)</li><li>N-butyric acid (Sigma-Aldrich, catalog number: B103500)</li><li>Buffer 1 and Buffer 2 (see Recipes)</li><li>Sample Buffer (see Recipes)</li><li>Cell culture media (see Recipes)<ol class="npwthree"><li>Yeast complete medium YPD</li><li>Synthetic dropout medium (SD)</li></ol></li></ol><p class="pptt" id="biaoti30815">### Equipment</p><ol class="npwtwok"><li>500 ml Erlenmeyer glass flasks</li><li>Refrigerated-regulated centrifuges (for 50 ml tubes: Hettich, model: Universal 32R; for 1.5 ml tubes: Beckman Coulter, model: Allegra 21R)</li><li>Spectrophotometer: Ultrospec 10 Cell Density Meter (Amersham Biosciences)</li><li>Vortex (Genie 2 with TurboMix Attachment, a multisample device)</li><li>Orbital shaker/ incubator for yeast culture: Multitron Standard (Infors AG)</li></ol><p class="pptt" id="biaoti30816">### Procedure</p><ol class="npwone"><li><em>Saccharomyces cerevisiae</em> standard culture conditions<ol class="npwtwo"><li><em>Saccharomyces cerevisiae</em> can be grown in yeast complete medium (YPD) (Recipe 3a) or in Synthetic dropout medium (SD) (Recipe 3b) supplemented with strain-specific requirements.</li><li>Inoculate 150 ml of culture medium (containers of 500 ml) with <em>S. cerevisiae</em> cells from a 5 ml preculture growing exponentially, and incubate them at 30 Â°C (or at the required temperature). Cultures must be shaken at 180-200 rpm on an orbital shaker.</li><li>Let cultures grow until an OD<sub>600</sub> ~0.7-0.8 is reached.</li><li>Transfer cultures to 50 ml centrifuge tubes (x3) and harvest them by centrifugation (2,500 <em>x g</em> for 2 min at 4 Â°C). Discard the supernatant.</li><li>In this step, pellets can be stored at -80 Â°C or can be used to continue the procedure.<br/><br/></li></ol></li><li>Chromatin purification protocol<ol class="npwtwo"><li>All the subsequent steps are performed at 4 Â°C.</li><li>Wash cells with 5 ml of sterile MilliQ H<sub>2</sub>O, centrifuge (2,500 <em>x g</em> for 2 min) and resuspend the pellet in 1 ml of MilliQ H<sub>2</sub>O.</li><li>Transfer to 1.5 ml microtubes and centrifuge them at 2,500 <em>x g</em> for 1 min at 4 Â°C. Discard the supernatant.</li><li>Resuspend the pellet in 200 Âµl of Buffer 1 (Recipe 1 Table 1) and add 200 Âµl of glass beads (size: 425-600 Âµm).</li><li>Lyse cells by continuous vigorous vortexing for 4 min at maximal speed using a Genie 2 vortex with a TurboMix Attachment (a multisample device) at 4 Â°C.</li><li>Centrifuge at 500 <em>x g</em> for 5 min to spin down the glass beads and intact cells. Transfer the supernatant to a new 1.5 ml microtube. Be careful to not take anything from the pellet to efficiently reduce cytoplasmic contamination during subsequent steps.</li><li>Centrifuge the supernatant at 500 <em>x g</em> for 5 min to discard the residual intact cells. Transfer the supernatant to a new 1.5 ml microtube. The pelletâs contamination in this step leads to cytoplasmic contaminants in the final purified chromatin fraction.</li><li>Keep 5-10 Âµl of supernatant as the input control.</li><li>Centrifuge the rest of the supernatant at high speed (20,000 <em>x g</em>) for 20 min.</li><li>Discard the supernatant using a pipette and resuspend the visible pellet (P1) in 200 Âµl of Buffer 1 (Table 1).</li><li>Centrifuge at high speed (20,000 <em>x g</em>) for 20 min.</li><li>Discard the supernatant and resuspend the visible pellet (P2) in 200 Âµl of Buffer 2 (Recipe 1 Table 2).</li><li>Centrifuge at high speed (20,000 <em>x g</em>) for 20 min.</li><li>Discard the supernatant. The visible final pellet (P3) constitutes the chromatin-enriched fraction.</li><li><strong>For SDS-PAGE and Western blots,</strong> resuspend P3 in 20 Âµl of 1x Tris-Glycine SDS Sample Buffer (Recipe 2) and incubate it for 10 min at 100 Â°C. After boiling, spin down at 30,000 <em>x g</em> for 1 min. Use the supernatant for gel electrophoresis.</li><li><strong>For the mass spectrometry analysis,</strong> gently resuspend the chromatin-enriched fractions (P3) in 20 Âµl of Tris-HCl 50 mM, pH 7.5, at room temperature. It can be frozen at -20 or -80 Â°C for further analyses.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Scale-up conditions<ol class="npwtwo"><li>To scale up the protocol, 600 ml of <em>S. cerevisiae</em> cultures at OD<sub>600</sub> ~0.7-0.8, must be used and the complete protocol must be performed at 4 Â°C.</li><li>Harvest the culture into three centrifuge tubes and centrifuge at 2,500 <em>x g</em> for 2 min. Under these conditions, each tube contains a cell pellet corresponding to 200 ml of the culture cells.</li><li>Wash each cellâs pellet with 5 ml of sterile MilliQ H<sub>2</sub>O, centrifuge at 2,500 <em>x g</em> for 2 min and resuspend each pellet in 1 ml of MilliQ H<sub>2</sub>O.</li><li>Transfer to 1.5 ml microtubes and centrifuge at 2,500 <em>x g</em> for 1 min. Discard the supernatant.</li><li>Resuspend each pellet in 200 Âµl of Buffer 1 (Table 1) and add 200 Âµl of glass beads (425-600 Âµm).</li><li>Lyse cells by vigorous continuous vortexing for 4 min (Genie 2 with the TurboMix Attachment, a multisample device, at maximal speed).</li><li>Centrifuge at 500 <em>x g</em> for 5 min to spin down the glass beads and intact cells. Transfer the three supernatants corresponding to the same cell culture to a new single 1.5 ml microtube.</li><li>Subsequent steps are similar to the standard protocol. In Steps B10 and B12 of the standard protocol, scale up buffers to 600 Âµl.</li><li><strong>For SDS-PAGE and Western blots,</strong> resuspend P3 in 60 Âµl of 1x Tris-Glycine SDS Sample Buffer (Recipe 2; Table 3) and incubate the desired amount for 10 min at 100 Â°C. After boiling, spin down at 30,000 <em>x g</em> for 1 min. Use the supernatant for gel electrophoresis.</li><li><strong>For the mass spectrometry analysis,</strong> gently resuspend the chromatin-enriched fractions (P3) in 20 Âµl (or that required) of Tris-HCl 50 mM, pH 7.5, at room temperature. It can be frozen at -20 or -80 Â°C for further analyses.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Scale-down conditions<ol class="npwtwo"><li>To scale down the protocol, 50 ml of <em>S. cerevisiae</em> cultures at OD<sub>600</sub> ~0.7-0.8, are used and all the protocol must be performed at 4 Â°C.</li><li>When scaling down the protocol, the solution volumes used for the standard procedures (200 Î¼l) are maintained to easily manipulate samples.</li><li><strong>For SDS-PAGE and Western blots,</strong> resuspend P3 in 20 Âµl of 1x Tris-Glycine SDS Sample Buffer (Recipe 2; Table 3) and incubate the desired amount for 10 min at 100 Â°C. After boiling, spin down at 30,000 <em>x g</em> for 1 min. Use the supernatant for gel electrophoresis.</li></ol></li></ol><p class="pptt" id="biaoti30817">### Data analysis</p><p class="ppzz">The protein analysis of the different fractions from the yChEFs procedure is shown in Figure 1.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191221/20191221231857_7467.jpg" width="214"/><br/><strong>Figure 1. Protein analysis of the different fractions from the yChEFs procedure.</strong> Histone H3 and Pgk1, as controls, were analyzed by Western blot (Anti-Rabbit IgG as a secondary antibody for anti-H3 primary antibody and Anti-mouse IgG as a secondary antibody for anti-Pgk1 primary antibody). Note that chromatin was properly isolated as no cytoplasmic Pgk1 protein was detected. WCE: whole-cell (lysate). P1-P3: Pellet 1-3. S1-S3: Supernatant 1-3. The 1/10 volumes from each fraction were loaded, except for the whole P3 fraction.</p><p class="pptt" id="biaoti30818">### Recipes</p><ol class="npwtwok"><li>Buffer 1 and Buffer 2 (Tables 1 and 2)<br/><br/><strong>Table 1. Buffer 1 composition and preparation</strong><br/><img alt="" class="layerphoto" height="315" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191221/20191221231702_6424.jpg" width="340"/><br/><strong>*FC: </strong>Final concentration<br/><br/><strong>Table 2. Buffer 2 composition and preparation</strong><br/><img alt="" class="layerphoto" height="325" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191221/20191221231752_7313.jpg" width="340"/><br/><strong>*FC:</strong> Final concentration<br/><br/></li><li>Sample buffer (Table 3)<br/><br/><strong>Table 3. Sample buffer composition and preparation</strong><br/><img alt="" class="layerphoto" height="184" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191221/20191221231821_9173.jpg" width="340"/><br/><strong>*FC:</strong> Final concentration<br/><br/></li><li>Cell culture media<ol class="npwthree"><li>Yeast complete medium (YPD)<br/>2% (w/v) peptone<br/>2% (w/v) glucose (or galactose)<br/>1% (w/v) yeast extract</li><li>Synthetic dropout medium (SD)<br/>0.17% (w/v) yeast nitrogen base<br/>0.5% (w/v) ammonium sulfate<br/>2% (w/v) glucose (or galactose)<br/>Supplemented requirements:<br/>L-Histidine 20 mg/L<br/>L-Lysine monohydrate 30 mg/L<br/>L-Leucine 150 mg/L<br/>L-Methionine 20 mg/L<br/>L-Tryptophan 20 mg/L<br/>Adenine sulphate 50 mg/L<br/>Uracil 20 mg/L</li></ol></li></ol><p class="pptt" id="biaoti30819">### Acknowledgments</p><p class="ppzz">This protocol is described in our previous report (Cuevas-BermÃºdez <em>et al.</em>, 2019). This work has been supported by grants from the Spanish Ministry of Economy and Competitiveness, MINECO and FEDER funds (BFU2016-77728-C3-2-P to F.N.) and the Junta de AndalucÃ­a (BIO258 to F. N.). A.C-B. is a recipient of an FPI predoctoral contract from MINECO. A.I.G-G was a recipient of MEC and a postdoctoral fellowship from the University of JaÃ©n. F.G-S is a recipient of a predoctoral fellowship from the University of JaÃ©n.</p><p class="pptt" id="biaoti30820">### References</p><ol class="npwtwok"><li>Aparicio, O., Geisberg, J. V. and Struhl, K. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18228445" target="_blank">Chromatin immunoprecipitation for determining the association of proteins with specific genomic sequences <em>in vivo</em>.</a> <em>Curr Protoc Cell Biol</em> Chapter 17: Unit 17.7.</li><li>Carrillo Oesterreich, F., Preibisch, S. and Neugebauer, K. M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21095587" target="_blank">Global analysis of nascent RNA reveals transcriptional pausing in terminal exons.</a> <em>Mol Cell</em> 40(4): 571-581.</li><li>Cuevas-BermÃºdez, A., Garrido-Godino, A. I. and Navarro, F. (2019). <a href="https://www.sciencedirect.com/science/article/pii/S2452014419300925" target="_blank">A novel yeast chromatin-enriched fractions purification approach, yChEFs, for the chromatin-associated protein analysis used for chromatin-associated and RNA-dependent chromatin-associated proteome studies from <em>Saccharomyces cerevisiae</em>.</a> <em>Gene Rep</em> 16: e100450.</li><li>Chu, D. S., Liu, H., Nix, P., Wu, T. F., Ralston, E. J., Yates, J. R., 3rd and Meyer, B. J. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16943775" target="_blank">Sperm chromatin proteomics identifies evolutionarily conserved fertility factors.</a> <em>Nature</em> 443(7107): 101-105.</li><li>Du, Y. C., Gu, S., Zhou, J., Wang, T., Cai, H., Macinnes, M. A., Bradbury, E. M. and Chen, X. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16522924" target="_blank">The dynamic alterations of H2AX complex during DNA repair detected by a proteomic approach reveal the critical roles of Ca<sup>2+</sup>/calmodulin in the ionizing radiation-induced cell cycle arrest.</a> <em>Mol Cell Proteomics</em> 5(6): 1033-1044.</li><li>Engelen, E., Brandsma, J. H., Moen, M. J., Signorile, L., Dekkers, D. H., Demmers, J., Kockx, C. E., Ozgur, Z., van IJcken, W. F., van den Berg, D. L. and Poot, R. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25990348" target="_blank">Proteins that bind regulatory regions identified by histone modification chromatin immunoprecipitations and mass spectrometry. </a><em>Nat Commun</em> 6: 7155.</li><li>Foltz, D. R., Jansen, L. E., Bailey, A. O., Yates, J. R., 3rd, Bassett, E. A., Wood, S., Black, B. E. and Cleveland, D. W. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19410544" target="_blank">Centromere-specific assembly of CENP-a nucleosomes is mediated by HJURP.</a> <em>Cell</em> 137(3): 472-484.</li><li>Foltz, D. R., Jansen, L. E., Black, B. E., Bailey, A. O., Yates, J. R., 3rd and Cleveland, D. W. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16622419" target="_blank">The human CENP-A centromeric nucleosome-associated complex.</a> <em>Nat Cell Biol</em> 8(5): 458-469.</li><li>Frei, C. and Gasser, S. M. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10640278" target="_blank">The yeast Sgs1p helicase acts upstream of Rad53p in the DNA replication checkpoint and colocalizes with Rad53p in S-phase-specific foci.</a> <em>Genes Dev</em> 14(1): 81-96.</li><li>Khoudoli, G. A., Gillespie, P. J., Stewart, G., Andersen, J. S., Swedlow, J. R. and Blow, J. J. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18514518" target="_blank">Temporal profiling of the chromatin proteome reveals system-wide responses to replication inhibition.</a> <em>Curr Biol</em> 18(11): 838-843.</li><li>Kubota, T., Hiraga, S., Yamada, K., Lamond, A. I. and Donaldson, A. D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21505101" target="_blank">Quantitative proteomic analysis of chromatin reveals that Ctf18 acts in the DNA replication checkpoint.</a> <em>Mol Cell Proteomics</em> 10(7): M110 005561.</li><li>Lambert, J. P., Fillingham, J., Siahbazi, M., Greenblatt, J., Baetz, K. and Figeys, D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21179020" target="_blank">Defining the budding yeast chromatin-associated interactome.</a> <em>Mol Syst Biol</em> 6: 448.</li><li>Lambert, J. P., Mitchell, L., Rudner, A., Baetz, K. and Figeys, D. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19106085" target="_blank">A novel proteomics approach for the discovery of chromatin-associated protein networks.</a> <em>Mol Cell Proteomics</em> 8(4): 870-882.</li><li>Lambert, J. P., Pawson, T. and Gingras, A. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22611019" target="_blank">Mapping physical interactions within chromatin by proteomic approaches.</a> <em>Proteomics</em> 12(10): 1609-1622.</li><li>Liang, C. and Stillman, B. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9407030" target="_blank">Persistent initiation of DNA replication and chromatin-bound MCM proteins during the cell cycle in cdc6 mutants.</a> <em>Genes Dev</em> 11(24): 3375-3386.</li><li>Metz, B., Kersten, G. F., Hoogerhout, P., Brugghe, H. F., Timmermans, H. A., de Jong, A., Meiring, H., ten Hove, J., Hennink, W. E., Crommelin, D. J. and Jiskoot, W. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14638685" target="_blank">Identification of formaldehyde-induced modifications in proteins: reactions with model peptides.</a> <em>J Biol Chem</em> 279(8): 6235-6243.</li><li>Tan, F., Li, G., Chitteti, B. R. and Peng, Z. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18022940" target="_blank">Proteome and phosphoproteome analysis of chromatin associated proteins in rice (<em>Oryza sativa</em>).</a> <em>Proteomics</em> 7(24): 4511-4527.</li></ol>