<body>
            <h1>Unbiased and Tailored CRISPR/Cas gRNA Libraries by Synthesizing Covalently-closed-circular (3Cs) DNA</h1>
            <p><strong>Authors:</strong> Martin  Wegner, Koraljka Husnjak, Manuel  Kaulich</p>
            <h2>Abstract</h2>
            <p>Simplicity, efficiency and versatility of the CRISPR/Cas system greatly contributed to its rapid use in a broad range of fields. Applications of unbiased CRISPR/Cas screenings are increasing and thus there is a growing need for unbiased and tailored CRISPR/Cas gRNA libraries. Conventional methods for gRNA library generation apply PCR and cloning techniques, thus coupling library diversity with distribution. Here, we provide additional technical expertise to apply our covalently-closed-circular synthesized (3Cs) gRNA library generation technology for the generation of high-quality CRISPR/Cas gRNA libraries. F1-origin of replication-containing plasmid DNA is transformed into CJ236 bacteria for single colony outgrow followed by M13KO7 bacteriophage superinfection for the production and preparation of circular dU-containing ssDNA. dU-ssDNA is annealed with homology- and gRNA-encoding DNA oligonucleotides for their T7 DNA polymerase-mediated extension to form hetero-duplexed CCC-dsDNA (3Cs-dsDNA). 3Cs-dsDNA is electroporated for the selected amplification of the newly synthesized, gRNA-containing strand. To remove wild-type plasmid remnants, the purified plasmid DNA is digested with restriction enzymes targeting the gRNA-placeholder sequence in the template DNA. Undigested plasmid is electroporated for the extraction of the final 3Cs gRNA library. Due to the absence of PCR amplification and conventional cloning steps, the 3Cs technology uncouples sequence diversity from sequence distribution, thereby generating gRNA libraries with near-uniform distribution in diversities being only limited by electroporation efficiencies.</p>
            <p><strong>Keywords:</strong> CRISPR/Cas, gRNA library, 3Cs technology, 3Cs gRNA, Gene editing, CRISPR screen</p>
            <h2>Background</h2><p>CRISPR/Cas gene editing has developed to be the gold standard for tailored and unbiased high-throughput screening (Doench, 2018). Unbiased pooled screening enables the simultaneous testing of thousands of individual genetic perturbations in a single experiment. However, a prerequisite is the presence of a gRNA library, in which each individual construct defines a specific genomic target site (gRNA sequence), at which the gene perturbation is desired, with a libraryâs gRNA distribution and diversity dictating the experimental scale (Sanson <em>et al.</em>, 2018). Classically, protocols to generate CRISPR/Cas gRNA libraries for arrayed or pooled applications contain T4 ligase or recombination-based ligations steps, requiring DNA polymerase-dependent amplification of the gRNA-encoding oligonucleotides coupled with restriction enzyme-digested plasmid DNA for subsequent gRNA cloning (Koike-Yusa <em>et al.</em>, 2014; Shalem <em>et al.</em>, 2014; Wang <em>et al.</em>, 2014; Schmidt <em>et al.</em>, 2015; Vidigal and Ventura, 2015; Arakawa, 2016; Ong <em>et al.</em>, 2017). Due to these technical limitations, the resulting gRNA libraries contain undesired PCR and cloning-dependent sequence biases in their gRNA sequence distribution that directly affects the experimental scale needed for robust hit calling (Shalem <em>et al.</em>, 2014; Wang <em>et al.</em>, 2014). Unbiased and tailored CRISPR/Cas gRNA screening is becoming more ubiquitous, highlighting the need for novel protocols to generate gRNA libraries that can readily be established in non-expert laboratories.<br/><br/>Our recently described protocol to generate covalently-closed-circular synthesized (3Cs) gRNAs avoids DNA polymerase-dependent amplification steps of gRNA-encoding oligonucleotides, as well as enzyme-digested plasmid DNA (Figure 1, Wegner <em>et al.</em>, 2019), thereby circumventing the occurrence of PCR-dependent gRNA sequence bias, as well as cloning artifacts within the final gRNA library, resulting in the decoupling of sequence distribution from sequence diversity and leading to nearly uniformly distributed gRNA libraries. As a consequence, the quality and distribution of the gRNA sequences within the original gRNA-encoding oligonucleotide pool is very important and particular caution should be paid to only use high-quality oligonucleotide pools.</p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191221/20191221235512_8182.jpg" width="277"/><br/><strong>Figure 1. Covalently-closed-circular (3Cs) synthesized gRNA reagents.</strong> Amplification of f1-ori-containing plasmids in M13KO7-infected <em>dut<sup>-</sup>/ung<sup>-</sup> E. coli</em> CJ236 bacteria and subsequent purification yields uracilated single-stranded DNA (dU-ssDNA). Annealing of gRNA-encoding oligonucleotides, extension with T7 polymerase, and ligation with T4 ligase produces a heteroduplex dU-dsDNA plasmid library. Electroporation into <em>dut<sup>-</sup>/ung<sup>-</sup> E. coli</em> yields the final library. The gRNA-encoding oligonucleotide is designed to anneal 5â² and 3â² relative to the I-SceI placeholder site in the library backbone and acts as a primer for T7 polymerase in the synthesis step of the 3Cs protocol.</p><p class="pptt" id="biaoti30823">## Materials and Reagents</p><ol class="npwtwok"><li>2 mm electroporation cuvette (BTX, catalog number: 45-0125)</li><li>Conical centrifuge tube, 50 ml (Falcon, catalog number: 352070)</li><li>Safe-lock tubes, 1.5 ml (Eppendorf, catalog number: 0030120086)</li><li>Bacteria (<em>E. coli</em>) CJ236 (TaKaRa, catalog number: 9053 or Lucigen, catalog number: 60701), storage -80 Â°C</li><li>Bacteria (<em>E. coli</em>) 10-beta, electrocompetent (New England Biolabs, catalog number: C3020K), storage -80 Â°C</li><li>Bacteria (<em>E. coli</em>) SS320 (MC1061 Fâ), electrocompetent (Lucigen, catalog number: 60512)</li><li>dNTP mix (Carl Roth, catalog number: 0178), storage -20 Â°C</li><li>T4 polynucleotide kinase (PNK) (New England Biolabs, catalog number: M0201), storage -20 Â°C</li><li>T4 DNA ligase(New England Biolabs, catalog number: M0202), storage -20 Â°C</li><li>T7 DNA polymerase (unmodified) (New England Biolabs, catalog number: M0274), storage -20 Â°C</li><li>Restriction enzyme I-SceI (New England Biolabs, catalog number: R0694), storage -80 Â°C</li><li>Restriction enzyme EcoRV-HF (New England Biolabs, catalog number: R3195), storage -20 Â°C</li><li>Helper phage M13KO7 (New England Biolabs, catalog number: N0315)</li><li>LB broth with agar (Lennox) (Sigma-Aldrich, catalog number: L2897)</li><li>LB medium (Lennox) (Carl Roth, catalog number: X964)</li><li>2x YT medium (Carl Roth, catalog number: 6676)</li><li>S.O.C. medium (Thermo Fisher, catalog number: 15544034)</li><li>Polyethylene glycol 8000 (PEG 8000) (Carl Roth, catalog number: 263)</li><li>Sodium chloride (NaCl) (Sigma-Aldrich, catalog number: 31434-M)</li><li>Dulbeccoâs phosphate-buffered saline (PBS) (Sigma-Aldrich, catalog number: D8662)</li><li>E.Z.N.A. M13 DNA Mini Kit (Omega Bio-Tek, catalog number: D69001-01)</li><li>GeneJET Gel Extraction Kit (Thermo Fisher, catalog number: K0692)</li><li>GeneJET Plasmid Miniprep Kit (Thermo Fisher, catalog number: K0503)</li><li>DNA Clean and Concentrator-5 Kit (Zymo Research, catalog number: D4013)</li><li>Plasmid Midi Kit (Qiagen, catalog number: 12143)</li><li>Plasmid Maxi Kit (Qiagen, catalog number: 12163 or Promega, catalog number: 12163)</li><li>Ampicillin sodium salt (Amp) (Carl Roth, catalog number: K0292)</li><li>Chloramphenicol (Cmp) (Carl Roth, catalog number: 3886)</li><li>Kanamycin sulphate (Kan) (Carl Roth, catalog number: T832)</li><li>Adenosine 5'-triphosphate (ATP) (New England Biolabs, catalog number: P0756), storage -20 Â°C</li><li>Dithiothreitol (DTT) (Cell Signaling Technology, catalog number: 7016), storage -20 Â°C</li><li>Agarose, standard (Applichem, catalog number: 732-2789)</li><li>SYBR Safe DNA gel stain (Thermo Fisher, catalog number: S33102)</li><li>KCl (Carl Roth, catalog number: P017)</li><li>CaCl<sub>2</sub> (Carl Roth, catalog number: CN93)</li><li>MgCl<sub>2</sub> (Carl Roth, catalog number: KK36)</li><li>Tris base (Carl Roth, catalog number: A411)</li><li>Tris-HCl (Carl Roth, catalog number: 9090)</li><li>EDTA (Applichem, catalog number: A5097,0500)</li><li>Acetic acid (Sigma-Aldrich, catalog number: 33209)</li><li>KCM buffer (see Recipes)</li><li>20% PEG/NaCl (see Recipes)</li><li>10x TM buffer (see Recipes)</li><li>50x TAE buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30824">### Equipment</p><ol class="npwtwok"><li>Bacterial shaker (Eppendorf, model: New Brunswick Innova 44/44R, catalog number: M1282-0012; or similar)</li><li>Gene Pulser electroporation system (Bio-Rad, catalog number: 164-2076, or similar system for electroporation of bacteria)</li><li>Mini-sub cell GT horizontal electrophoresis system, 7 x 10 cm (Bio-Rad, catalog number: 1704467, or similar)</li><li>Centrifuge Avanti J-E (Beckman, catalog number: 369003, or similar)</li><li>Rotor JA-12 (fixed-angle aluminum rotor- 12 x 50 ml, 12,000 rpm, 23,200 <em>x g</em>, Beckman, catalog number: 360993)</li><li>NanoDrop (Thermo Fisher, catalog number: ND-1000, or similar)<br/></li></ol><p class="pptt" id="biaoti30825">### Software</p><ol class="npwtwok"><li>bCl<sub>2</sub>fastq (Illumina, <a href="https://support.illumina.com/" target="_blank">https://support.illumina.com/</a>)</li><li>cutadapt (<a href="https://cutadapt.readthedocs.io/en/stable/" target="_blank">https://cutadapt.readthedocs.io/en/stable/</a>; Martin, 2011)</li><li>Bowtie2 (<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a>; Langmead and Salzberg, 2012)</li><li>PinAPL-Py (<a href="http://pinapl-py.ucsd.edu/" target="_blank">http://pinapl-py.ucsd.edu/</a>; Spahn <em>et al.</em>, 2017) <br/></li></ol><p class="pptt" id="biaoti30826">### Procedure</p><ol class="npwone"><li>Preparation of plasmid single-stranded DNA (ssDNA)<ol class="npwtwo"><li>Transform 100-200 ng of plasmid DNA into 25-50 Î¼l competent <em>E. coli</em> CJ236 bacteria. Use either heat-shock (1 min, 42 Â°C, heat-block) or KCM buffer (use the same volume of bacteria and a mix of 5x KCM buffer diluted 1:5 with water, ice/15 min, room temperature/10 min) for transformation. Add 50-200 Î¼l S.O.C. medium and plate on LB plates containing 25 Î¼g/ml Cmp and additional antibiotic matching plasmid resistance (usually 100 Î¼g/ml Amp*, depending on plasmid), leave overnight at 37 Â°C.<br/><em>*Note: In theory, any plasmid containing f1-origin of replication can be used, such as lentiCRISPR v2 (Addgene 52961). The plasmid should have an antibiotic resistance gene other than Amp, Cmp and Kan.</em> </li><li>Next day, pick several single colonies of <em>E. coli</em> CJ236 with a sterile pipette tip, resuspend each into 1 ml of 2x YT medium supplemented with Amp* (100 Î¼g/ml), Cmp (25 Î¼g/ml) and 1 Î¼l helper phage. Shake for 2 h (37 Â°C, 200 rpm); add 100 Î¼g/ml Kan and continue shaking for 6-10 h (37 Â°C, 200 rpm).</li><li>Transfer 1 ml of initial culture into 30 ml of 2x YT medium containing Amp* (100 Î¼g/ml) and Kan (100 Î¼g/ml) and shake 16-20 h (37 Â°C, 200 rpm).</li><li>Transfer 30 ml bacterial culture into a 50 ml Falcon tube; centrifuge (10 min, 12,000 <em>x g</em>, 4 Â°C).</li><li>Transfer phage-containing supernatant (Figure 2A) into 6 ml of sterile-filtered PEG/NaCl buffer, invert 4-6 times, incubate for 30-60 min at room temperature; centrifuge (10 min at 12,000 <em>x g</em>, 4 Â°C).</li><li>Decant supernatant and aspirate the remaining supernatant.</li><li>Resuspend phage pellet (Figure 2B) in 1 ml 1x PBS and transfer to a new 1.5 ml reaction tube.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222000820_2873.jpg" width="138"/><br/><strong>Figure 2. Preparation of ssDNA from bacterial culture.</strong> <em>E. coli</em> CJ236 bacterial culture before (A) and after (B) centrifugation (pellet contains bacteria, whereas supernatant contains phage). C. Phage pellet following PEG/NaCl precipitation and subsequent centrifugation. D. Phage resuspended in 1x PBS buffer (transparent solution). E. After addition of ssDNA lysis buffer (turbid solution, turbidity reflects amount of phage).<br/><br/></li><li>Centrifuge for 5 min at 16,100 <em>x g</em> to remove cell debris, transfer the supernatant to a new 1.5 ml reaction tube and store at 4 Â°C.</li><li>Purify ssDNA from phage supernatant according to the Omega Bio-Tek E.Z.N.A M13 DNA Mini Kit protocol.</li><li>Determine ssDNA concentration using NanoDrop.</li><li>Analyze ssDNA in a 0.8% TAE/agarose gel using 500 ng ssDNA and 250 ng of the respective dsDNA. The ssDNA should appear as a predominant single band (Figure 3A).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222000747_2242.jpg" width="223"/><br/><strong>Figure 3. Quality control of ssDNA and 3Cs reaction products.</strong> A. ssDNA appears as a single band on an agarose gel and usually exhibits higher electromobility than its dsDNA plasmid counterpart. #1, #2, and #3 refer to example plasmids of different sizes. Low quality ssDNA exhibits multiple bands indicating phage-derived recombination products that will lead to poor performance during 3Cs (!). B. The 3Cs reaction product typically exhibits three bands on an agarose gel. All three electrophoreses were run on 0.8% 1x TAE agarose for 30 min at 110 V.<br/><br/></li></ol></li><li>Oligonucleotide phosphorylation with T4 polynucleotide kinase<br/>In a 1.5 ml reaction tube, combine 600 ng oligonucleotide pool with 2 Î¼l 10x TM buffer, 1 mM ATP, 5 mM DTT and 20 units T4 polynucleotide kinase in a total volume of 20 Î¼l; incubate for 2 h at 37 Â°C.<br/><em>Note: Oligonucleotide pools can be ordered from various commercial sources. Hand-pooled oligonucleotides generally lead to libraries with very variable gRNA distribution and should therefore be avoided.<br/><br/></em> </li><li>Annealing of the oligonucleotides to the template<br/>Mix 20 Î¼g freshly prepared ssDNA with 25 Î¼l 10x TM buffer (final concentration 1x TM) and 20 Î¼l phosphorylated oligonucleotides in a total volume of 250 Î¼l; incubate in a thermoblock as follows: 90 Â°C (3 min), 50 Â°C (5 min), room temperature (5 min).<br/><em>Note: Molar ratio between oligonucleotides and vector ssDNA should be determined experimentally and might be different for a particular plasmid. Usually, ratio ranges of 3:1, 5:1 and 7:1 are optimal for 3Cs reactions.<br/><br/></em> </li><li>Enzymatic synthesis of 3Cs-dsDNA<br/>Add the following components to the previous mixture: 10 Î¼l of 10 mM ATP, 10 Î¼l of 100 mM dNTPs, 15 Î¼l of 100 mM DTT, 3 Î¼l of T4 DNA ligase (stock: 400,000 units/ml) and 5 Î¼l of T7 DNA polymerase (stock: 10,000 units/ml); mix carefully by pipetting up and down or flicking the tube; incubate for 16 h at 22 Â°C.<br/><br/></li><li>Purification of 3Cs-dsDNA with DNA Clean and Concentrator-5 Kit<ol class="npwtwo"><li>Mix the 3Cs reaction with 2x volume of DNA binding buffer and incubate for 10 min at room temperature.</li><li>Divide the 3Cs reaction on 2 columns.</li><li>Spin down (1 min, 2,300 <em>x g</em>, room temperature); re-spin each flow through twice.</li><li>Wash column twice each with 200 Î¼l wash buffer (1 min, 2,300 <em>x g</em>, room temperature).</li><li>Discard flowthrough and dry the column (1 min, 7,400 <em>x g</em>, room temperature).</li><li>Incubate column with preheated 10 Î¼l of elution buffer (10 min, 50 Â°C), spin down at maximum speed (2 min), re-spin each flow through twice.</li><li>Repeat elution with 20 Î¼l flow through (final volume will be around 40 Î¼l).</li><li>Run a gel to control the quality of dsDNA alongside to ssDNA (Figure 3B).</li><li>Determine concentration by Nanodrop.<br/><br/></li></ol></li><li>Electroporation of electrocompetent <em>E. coli</em> bacteria with 3Cs dsDNA<ol class="npwtwo"><li>Prepare 20 ml pre-warmed S.O.C. medium at 37 Â°C in a small Erlenmeyer flask.</li><li>Incubate the entire purified 3Cs dsDNA and a 2 mm electroporation cuvette on ice.</li><li>Thaw 400 Î¼l of electrocompetent <em>E. coli</em> strain SS320 or NEB 10-beta bacteria on ice (20 min).</li><li>Add the cells to the pre-chilled and purified 3Cs DNA and mix slowly by pipetting, avoid generating bubbles.</li><li>For electroporation, use the following settings: 2.5 kV field strength, 25 ÂµF capacitance, 200 ohms resistance (optimal duration: 3.8-4.2 ms).</li><li>Place the dried cuvette into the electroporator holder.</li><li>Transfer the mixture gently (without bubbles) into the cuvette and tap the cuvette (avoid air bubbles that lead to arcing!).</li><li>Tightly close the holder, put the lid on the cuvette.</li><li>Electroporation: press both buttons on the power supply at once, release once beeping starts (usually 3.8-4.2 ms).</li><li>Immediately rescue the electroporated cells by rinsing in 2x 1 ml pre-warmed S.O.C. medium. Any delay will significantly decrease the electroporation efficiency.</li><li>Incubate Erlenmeyer flask (with 20 ml S.O.C. medium and electroporated bacteria) for 30 min (37 Â°C, 140 rpm).</li><li>Remove 40 Î¼l of bacterial culture, keep for preparing serial dilutions.<br/><em>Note: In a 96-well plate prepare 3 columns (i.e., 3 x 8 wells) with 90 Î¼l 1x PBS per library. Add 10 Î¼l of bacterial culture into the first 90 Î¼l 1x PBS (prepare in triplicate), use it for subsequent serial dilutions (1:10). Plate 5 Î¼l of diluted bacteria on Amp* LB plate, incubate at 37 Â°C overnight. Calculate library diversity as follows: count colonies in a dilution where separate colonies are nicely visible and calculate the mean number of colonies per 1 Î¼l diluted bacterial culture. Knowing the final volume of the bacterial culture (20.4 ml) and dilution from which colonies were counted (for example, lane 3 dilution in a 96-well plate corresponds to 10<sup>3</sup> dilution), calculate the library diversity: Diversity = (number of bacterial colonies in 1 Î¼l diluted bacterial culture) x 10<sup>(dilution)</sup> x bacterial culture volume (Î¼l).</em> </li><li>Transfer the remaining content of the small Erlenmeyer flask into 180 ml of pre-warmed 2x YT medium (with 100 Î¼g/ml Amp*), shake overnight (37 Â°C, 140 rpm).</li><li>Next day prepare a mini-prep from 5 ml of the overnight culture and use it to check library quality (see Steps G1-G3).</li><li>Spin the remaining culture down (4 Â°C, 6,000 <em>x g</em>, 20 min) for subsequent midi-prep.<br/><br/></li></ol></li><li>Analysis of the pre-library by restriction enzymes<ol class="npwtwo"><li>Digest 0.5 Î¼g pre-library dsDNA with the appropriate restriction enzymes; always use original vector as cleavage control as well.</li><li>Load undigested and digested pre-library and original vector on 0.5-0.8% 1x TAE agarose gel.</li><li>Check the quality of the pre-library and compare it with the original vector (Figure 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222000713_0185.jpg" width="157"/><br/><strong>Figure 4. Removal of wild-type remnants from pre-library preparation and quality control of final library.</strong> Restriction enzyme digestion of the wild-type plasmid, the pre-library, and the final library indicates the ratio of wild-type and 3Cs plasmids at each step. As control, the wild-type plasmid (wt plasmid) is linearized with either I-SceI or EcoRV. A double digest with both enzymes removes a fragment of approximately 2,500 bp as both enzymes cut the plasmid (!). In the pre-library pool, the incomplete digestion with I-SceI and the faint band resulting from the double-digest indicate the majority of DNA being 3Cs products. The double digest reveals a band at ~2,500 bp with a significantly lower intensity than in the wild-type plasmid pool. In the final library, the I-SceI digest reveals no visible linearized fragment. The double digest linearizes the library and shows no visible band at ~2,500 bp, indicating the final library to be free of wild-type plasmid. Electrophoresis was run on 0.8% 1x TAE agarose for 30 min at 110 V.<br/><br/></li></ol></li><li>Digestion of pre-library library prior final electroporation<ol class="npwtwo"><li>Digest 10-20 Î¼g of the pre-library with the appropriate enzyme and purify by Clean &amp; Concentrator Kit.</li><li>Transform and process 5-10 Î¼g of the cleaved pre-library into 400 Î¼l electrocompetent bacteria as described under Steps F1-F12.</li><li>Grow 200-1,000 ml of 2x YT-Amp* (100 Âµg/ml) bacterial culture (volume depends on library size, future needs and amount of remaining oligonucleotides) at 37 Â°C overnight.</li><li>Next day spin down bacteria (prepare multiple aliquots of bacterial pellets from either 50, 100 or 150 ml bacterial culture, depending on the maxi-prep kit that you are using and size of your library).</li><li>Prepare a maxi-prep from one bacterial pellet, keep the remaining pellets at -20 Â°C until needed.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Analysis of the final library by restriction enzymes<br/>Follow Steps G1-G3 (Figure 4).<br/><br/></li><li>Sequence analysis of the final library by Sanger sequencing<br/>Send appropriate amount (usually 700-1,200 ng) of the final library for Sanger sequencing. Choose a sequencing primer that is near your gRNA cassette, for example U6 primer. Representative data is presented in Figure 5.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222000643_8463.jpg" width="665"/><br/><strong>Figure 5. SANGER sequencing of final library.</strong> As a first quality control step, SANGER sequencing of the final library with the U6-primer reveals randomization at the I-SceI placeholder side. In this example, digestion of the final optimized truly genome-wide library (oTGW) from Wegner <em>et al.</em> (2019) results in a randomized nucleotide pattern resembling the intended sequence composition of that library.<br/><br/></li><li>Sequence analysis of the final library guide distribution by next-generation sequencing (NGS)<br/>For a detailed description of data processing and data analysis we refer to Wegner <em>et al.</em>, 2019, a brief overview of the procedures is described below.</li></ol><p class="pptt" id="biaoti30827">### Data analysis</p><p class="ppzz">A detailed description of data processing and data analysis can be found in Wegner <em>et al.</em>, 2019. To quality control 3Cs reagents, multiplexed Next Generation Sequencing (NGS) of several different plasmid libraries is possible when using barcoded Illumina adapter sequences. Raw NGS data can be demultiplexed using bCl<sub>2</sub>fastq from Illumina, preferably in the most recent version. To count individual reads of a library, use a designated tool like PinAPL-Py (Spahn <em>et al.</em>, 2017) or design a customized pipeline that performs read trimming and sequence alignments according to your needs using, <em>e.g.</em>, cutadapt and bowtie2.<br/>  Adapter trimming can be performed using cutadapt by trimming the read to the 20 gRNA-encoding nucleotides (Martin, 2011, <a href="https://cutadapt.readthedocs.io/en/stable/" target="_blank">https://cutadapt.readthedocs.io/en/stable/</a>). Alignment of the trimmed reads against the library can then be performed with bowtie2 or custom scripts. The final library should contain all desired gRNA sequences. To assess the distribution of the library, generate Lorenz curves of gRNA representation and compute the area under the resulting curve according to (Wegner <em>et al.</em>, 2019) (Figure 6A). This will allow you to compare different libraries. A histogram of the frequencies of the gRNA reads and calculating the ratio between the 90% and 10% percentiles reveals the skew of a library (Figure 6B). For the analyses of gene perturbation screens, we refer to the relevant literature as the experimental settings and read out determine the analyses techniques.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222000930_7059.jpg" width="237"/><br/><strong>Figure 6. NGS analysis and visualization of expected sequence distributions.</strong> A. gRNA counts of three gRNA libraries are visualized as Lorenz plots, displaying the cumulative fraction of represented NGS reads against the gRNAs ranked by abundance. The ideal distribution would be uniform and resembles a diagonal line with an area under the curve (AUC) of 0.5 (green curve, ideal). A well distributed library contains every intended gRNA sequence and exhibits an AUC that differs only slightly from 0.5 (blue curve, good). An increased AUC value indicates an unevenly distributed library (orange curve, bad). Note that the slope of this curve decreases to 0 near x = 1.0, indicating that there are gRNA sequences missing in that library. B. Plotting the gRNA distributions from (A) as histograms visualizes the skew or symmetry of a library. A well distributed library exhibits a low skew near 1 (blue, good), indicating symmetry. A skewed library is asymmetrically distributed. The bad example is skewed towards higher gRNA read counts as revealed by the long right tail in the histogram (orange, bad).</p><p class="pptt" id="biaoti30828">### Notes</p><ol class="npwtwok"><li>Oligonucleotides are usually 55-65 nucleotides in length, having a gRNA-encoding region in the middle, and plasmid sequence homology regions (3Cs homology) at its 5â² and 3â² ends. Homology of 18 nucleotides (on each side of the gRNA) and annealing temperature above 45 Â°C are optional for 3Cs reaction. The oligonucleotides should not have any modifications at either end as the efficiency of the ligation will decrease.<br/><em>Important: The template plasmid must contain an f1-origin of replication. The </em><em>E. coli</em><em> CJ236 strain is Cmp-resistant and dut<sup>-</sup>/ung<sup>-</sup>.</em> </li><li>Phage suspended in PBS can be stored for months at 4 Â°C.</li><li>Once isolated, ssDNA is stable up to a few days or weeks. We recommend using freshly prepared ssDNA.</li><li>To confirm the quality of your ssDNA, you can mix 1 Î¼l of phage prep with KCM-competent bacteria and 200 Î¼l S.O.C. medium. Incubate at 37 Â°C for 1 h and then dilute to 5 ml LB medium complemented with Amp* (100 Î¼g/ml). Shake overnight (37 Â°C, 200 rpm), isolate dsDNA using a mini-prep kit and analyse dsDNA by using the same approach as in Procedure (Steps G1-G3). Use original plasmid dsDNA as positive control (compare size and restriction pattern in agarose gel).</li><li>Use only freshly phosphorylated oligonucleotides for 3C reaction.</li><li>High-quality electrocompetent <em>E. coli</em> strain SS320 or NEB 10-beta bacteria can also be prepared in the lab.</li><li>The removal of dU-containing DNA in <em>dut<sup>+</sup>/ung<sup>+</sup> E. coli</em> is not 100% efficient, therefore, it can be beneficial to introduce restriction sites (such as I-SceI) in the area of plasmid where the gRNAs will be introduced (gRNA placeholder sequence). Such an approach ensures that all the original sequences (still containing I-SceI site) will be depleted upon pre-library cleavage by I-SceI and subsequent electroporation. Importantly, make sure that the restriction recognition sequence does not resemble an active gRNA sequence in the genome of interest to avoid unintended editing events. A guide sequence containing an I-SceI restriction site is safe to use in the human genome. Also ensure that the restriction site is not a part of any of the gRNA sequences in the library to avoid unintended removal of a gRNA from the final pool.</li><li>For pLentiGuide (Addgene 52963) and pLentiCRISPRv2 (Addgene 52961) plasmids, use a forward primer (5â²-GGGCCTATTTCCCATGATTCCTTCATATTTGC-3â²) that binds in the U6-promoter for Sanger sequencing.</li></ol><p class="pptt" id="biaoti30829">### Recipes</p><ol class="npwtwok"><li>KCM buffer<br/><em>Note: KCM renders the </em><em><em>dut-/ung- </em><em>E. coli</em></em><em> CJ236 cells competent.</em><br/>0.5 M KCl<br/>0.15 M CaCl<sub>2</sub><br/>0.25 M MgCl<sub>2</sub> </li><li>20% PEG/NaCl<br/><em>Note: PEG/NaCl is used to precipitate phage particles containing dU-ssDNA.</em><br/>20 % polyethylene glycol<br/>2.5 M NaCl</li><li>10x TM buffer<br/><em>Note: Serves together with ATP and DTT as a reaction buffer for polynucleotide kinase T4 in the phosphorylation step of the 3Cs protocol.</em><br/>0.1 M MgCl<sub>2</sub><br/>0.5 M Tris-HCl<br/>pH 7.5</li><li>50x TAE buffer<br/>2 M Tris base<br/>0.5 M EDTA<br/>1 M acetic acid</li></ol><p class="pptt" id="biaoti30830">### Acknowledgments</p><p class="ppzz">We thank Alkmini Kalousi for providing images related to bacteria and phage pellets. We further thank all members of the Frankfurt CRISPR/Cas Screening Center (FCSC) and the Kaulich laboratory who contributed to the original research paper from which this protocol is derived from (Wegner <em>et al.</em>, 2019). This work was supported by the Hessian Ministry for Science and the Arts (HMWK, LOEWE-CGT, IIIL5-518/17.004), the German Research Foundation (DFG; CEF-MC - EXC115/2; ECCPS - EXC147/2) and in part by the LOEWE Center Frankfurt Cancer Institute (FCI) funded by the Hessen State Ministry for Higher Education, Research and the Arts (IIIL5-519/03/03.001-(0015)).</p><p class="pptt" id="biaoti30831">### Competing interests</p><p class="ppzz">The Goethe University Frankfurt has filed a patent application related to this work on which Martin Wegner and Manuel Kaulich are inventors (WO2017EP84625). The Goethe University provides an exclusive license of the 3Cs technology to Vivlion GmbH for which Manuel Kaulich is co-founder, shareholder and chief scientific officer.</p><p class="pptt" id="biaoti30832">### References</p><ol class="npwtwok"><li>Arakawa, H. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27574704" target="_blank">A method to convert mRNA into a gRNA library for CRISPR/Cas9 editing of any organism.</a> <em>Sci Adv</em> 2(8): e1600699.</li><li>Doench, J. G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29199283" target="_blank">Am I ready for CRISPR? A user's guide to genetic screens.</a> <em>Nat Rev Genet</em> 19(2): 67-80.</li><li>Koike-Yusa, H., Li, Y., Tan, E. P., Velasco-Herrera Mdel, C. and Yusa, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24535568" target="_blank">Genome-wide recessive genetic screening in mammalian cells with a lentiviral CRISPR-guide RNA library.</a> <em>Nat Biotechnol</em> 32(3): 267-273.</li><li>Langmead, B. and Salzberg, S. L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" target="_blank">Fast gapped-read alignment with Bowtie 2.</a> <em>Nat Methods</em> 9(4): 357-359.</li><li>Martin, M. (2011). <a href="https://journal.embnet.org/index.php/embnetjournal/article/view/200" target="_blank">Cutadapt removes adapter sequences from high-throughput sequencing reads.</a> <em>EMBnet J</em> 17(1): 10-12.</li><li>Ong, S. H., Li, Y., Koike-Yusa, H. and Yusa, K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28785007" target="_blank">Optimised metrics for CRISPR-KO screens with second-generation gRNA libraries.</a> <em>Sci Rep</em> 7(1): 7384.</li><li>Sanson, K. R., Hanna, R. E., Hegde, M., Donovan, K. F., Strand, C., Sullender, M. E., Vaimberg, E. W., Goodale, A., Root, D. E., Piccioni, F. and Doench, J. G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30575746" target="_blank">Optimized libraries for CRISPR-Cas9 genetic screens with multiple modalities.</a> <em>Nat Commun</em> 9(1): 5416.</li><li>Schmidt, T., Schmid-Burgk, J. L. and Hornung, V. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26446710" target="_blank">Synthesis of an arrayed sgRNA library targeting the human genome.</a> <em>Sci Rep</em> 5: 14987.</li><li>Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelson, T., Heckl, D., Ebert, B. L., Root, D. E., Doench, J. G. and Zhang, F. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571" target="_blank">Genome-scale CRISPR-Cas9 knockout screening in human cells.</a> <em>Science</em> 343(6166): 84-87.</li><li>Spahn, P. N., Bath, T., Weiss, R. J., Kim, J., Esko, J. D., Lewis, N. E. and Harismendy, O. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29158538" target="_blank">PinAPL-Py: A comprehensive web-application for the analysis of CRISPR/Cas9 screens.</a> <em>Sci Rep</em> 7(1): 15854.</li><li>Vidigal, J. A. and Ventura, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26278926" target="_blank">Rapid and efficient one-step generation of paired gRNA CRISPR-Cas9 libraries.</a> <em>Nat Commun</em> 6: 8083.</li><li>Wang, T., Wei, J. J., Sabatini, D. M. and Lander, E. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569" target="_blank">Genetic screens in human cells using the CRISPR-Cas9 system.</a> <em>Science</em> 343(6166): 80-84.</li><li>Wegner, M., Diehl, V., Bittl, V., de Bruyn, R., Wiechmann, S., Matthess, Y., Hebel, M., Hayes, M. G., Schaubeck, S., Benner, C., Heinz, S., Bremm, A., Dikic, I., Ernst, A. and Kaulich, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30838976" target="_blank">Circular synthesized CRISPR/Cas gRNAs for functional interrogations in the coding and noncoding genome.</a> <em>Elife</em> 8: 42549.<br/></li></ol>