<body>
            <h1>Isolation and Characterization of Live Yeast Cells from Ancient Clay Vessels</h1>
            <p><strong>Authors:</strong> Tzemach Aouizerat, Aren M. Maeir, Yitzhak  Paz, Yuval  Gadot, Amir Szitenberg, Sivan Alkalay-Oren, Shunit Coppenhagen-Glazer, Michael Klutstein, Ronen Hazan</p>
            <h2>Abstract</h2>
            <p>Ancient fermented food has been studied mainly based on residue analysis and recipes and reconstruction attempts were performed using modern domesticated yeast. Furthermore, microorganisms which participated in fermentation were studied using ancient-DNA techniques. In a recent paper, we presented a novel approach based on the hypothesis that enriched yeast populations in fermented beverages could have become the dominant species in storage vessels and their descendants could be isolated and studied today. Here we present a pipeline for isolation of yeast from clay vessels uncovered in archeological sites and transferred to the microbiology lab where they can be isolated and characterized. This method opens new avenues for experimental archeology and enables attempts to recreate ancient food and beverages using the original microorganisms.</p>
            <p><strong>Keywords:</strong> Ancient pottery vessels, Yeast, Bio-archeology, Experimental archaeology, Food, Beverages, Fermentation</p>
            <h2>Background</h2><p>Food ways and dietary habits, offer significant insight into subsistence strategies, connectivity and cultural and ideological choices of societies (Beaudry and Metheny, 2015). This direction of research is rather restricted when applied to ancient societies, as only residues of food have survived the absence of preservatives. Fragmentary evidence has impeded research of ancient fermented alcoholic beverages such as beer, wine, mead (honey wine) and kvass, which played a major role in daily life as well as in religious, social and political rituals of ancient societies (Joffe, 1998; Myers <em>et al.</em>, 2000; Dietler, 2006; Legras <em>et al.</em>, 2007; McGovern, 2009; SchiefenhÃ¶vel and Macbeth, 2011; Hornsey, 2012; Parker and McCool, 2015).<br/><br/>Reconstruction of ancient fermented beverages has, until now, consisted of brewing modern ingredients according to ancient recipes or based on organic residue found in ancient containers or breweries. To achieve fermentation, previous efforts have used <em>Saccharomyces cerevisiae</em>, the domesticated commercial yeast species (McGovern, 2009; Stika, 2011). To the best of our knowledge, the study of ancient microorganisms, including bacteria (Donoghue <em>et al.</em>, 2004; Andam <em>et al.</em>, 2016), viruses (Muhlemann <em>et al.</em>, 2018) and yeast (Cavalieri <em>et al.</em>, 2003), was based on ancient DNA studies; no attempt has been made to isolate and reproduce microorganisms from containers used for the production of fermented beverages.<br/><br/>Hypothesizing that yeast can form stable colonies in clay nanopores, we were recently able to isolate live yeast cells from ancient containers thought to have contained beverages and oil, using the current protocol (Aouizerat <em>et al.</em>, 2019). Following the isolation, we characterized the growth of the yeast, sequenced and analyze their full genome and used them to produce and analyze beer and mead (Aouizerat <em>et al.</em>, 2019).<br/><br/>Thus, our research suggests that ancient microorganisms can be isolated from archaeological finds, facilitating the identification of additional ancient foods and beverages. It also promotes the field of âexperimental archeologyâ: the recreation of the past, mainly of ancient fermented food (such as cheese and pickles) and beverages.<br/><br/>The following text presents a two-part protocol for the collection of sherds/vessels from archeological dig sites, as well as the isolation and characterization of microorganisms. Part I details guidelines for handling vessels intended for microorganismsâ isolation. Part II describes the isolation process. While this protocol was applied to yeast, it can be easily modified to apply to other microorganisms, such as bacteria.</p>
        </body>

<p class="ppzzs"><strong><br/><span style="font-size:24px;">Part I: Guidelines for the excavation and collection of clay vessels for microorganismsâ isolation (field work)</span><br/></strong> </p><p class="pptt" id="biaoti30835">## Materials and Reagents</p><ol class="npwtwok"><li>Lab gloves</li><li>50 ml Falcon tubes (MINIPLAST EIN SHEMER, catalog number: 835-050-21-111)</li><li>Clean plastic container</li><li>Aluminum foil</li><li>Tube markers<br/></li></ol><p class="pptt" id="biaoti30836">### Equipment</p><ol class="npwtwok"><li>Folding table</li><li>Recording device/s including camera, tablet, laptop, and cellular phones</li></ol><p class="pptt" id="biaoti30837">### Procedure</p><p class="ppzz"><em>Note: In sites where containers intended for isolation of microorganisms are expected, do not allow any kind of food or beverage in the excavation area (primarily fruit, vegetables, beer, yogurt, bread, pickles, cheese but also others).</em> </p><br/><ol class="npwone"><li>Before and while excavating<ol class="npwtwo"><li>Define vessels that are to be collected for microorganismsâ isolation. For example:<ol class="npwthree"><li>Vessels that are clearly meant for storing, serving and drinking of beverages or food or, vessels carrying beverage-related specific features, such as strainers (Figure 1).</li><li>Vessels suggested to have been previously used for the preparation of fermented beverages and food such as dairy products (<em>e.g.</em>, churns).</li><li>For more examples see also Aouizerat <em>et al.</em> (2019).</li></ol></li><li>Increase the awareness of the team so that they are prepared to collect vessels for microorganismsâ isolation based on the context of the site. For instance, in the case of the discovery of a drinking set <em>in situ</em>, or a possible production site.</li><li>Make sure that no fermented food of any kind, including unfiltered beer, sourdough and wine are being consume in the site. In addition, it is recommended that workers wash their hands and faces before starting.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222005127_8864.jpg" width="75"/><br/><strong>Figure 1. Putative beer jug.</strong> An example of a vessel thought to be a beer jug due to its shape and strainer.</li></ol></li><br/><li>Excavation and sample handling in the field<ol class="npwtwo"><li>Identify vessels suitable for microorganismsâ isolation as soon as you can, <em>in situ</em>, before they are fully uncovered.</li><li>Visually record the excavation process, and fully document the vessel (drawing, photography, textual description) within its context, prior to its extraction from the soil.</li><li>Set up a folding table near the vessel and cover the table with clean aluminum foil.</li><li>From now on, work on this vessel using disposable lab or cotton gloves.</li><li>When the identified vessel is uncovered, decide if and how much of the vessel can be broken based on the policy of the excavators and government regulations.</li><li>Immediately after the initial uncovering, take small pieces (5-6 cm) of the vessel and transfer them to 50 ml tubes using gloves at all times.<br/><em>Note: Try to minimize vessel exposure time before the initial sampling.</em> </li><li>Label the tubes with a tube marker and record the time, place, specific context, excavator, and type of sherd.</li><li>Carefully, using laboratory gloves, excavate the vessels according to the excavationâs regular protocol.<br/><em>Note: Only dedicated and guided personnel, preferably only one, should touch the sherds to avoid possible contamination.</em> </li><li>While uncovering, identify parts of the vessel that are more likely to hold microorganisms such as the base, strainer and the side of the curve towards the neck. These parts of the vessel will be the focus of efforts from now on.</li><li>Transfer the collected fragments to the table, keep them in plastic containers and label them accordingly.<br/><em>Note: Do not wash the pieces meant for microorganismsâ isolation.</em> </li><li>Label the other fragments of the uncovered vessel that are not to be sent to the lab with the same ID, to keep them for restoration and other analyses.</li><li>Always keep some fragments unwashed in case the lab would need more fragments.</li><li>Transfer the plastic containers with the sherds for microorganism analysis to the microbiology lab (see below).<br/><em>Note: In specific cases, an entire vessel can be chosen to be taken to the lab. In that case, try to minimize the time of the vessel exposure and excavate it, record and transfer it to a plastic container as soon as possible.</em><br/> <br/><p class="ppzz" style="margin-left:-30px;"><em><strong>Control vessels and control soil samples:</strong></em><br/><em>Note: Non-related controls are a key component in the analysis, to ensure that the isolated microorganisms originated in the vessel and are not an environmental contamination.</em> </p></li><li>Using gloves, collect as many samples as possible of: sediments, stones, plants, and roots from the vesselâs surroundings. Store them in 50 ml plastic tubes.</li><li>Identify vessels which in high probability are unrelated to microorganisms, such as burned cooking pots, small vessels/juglets (<em>i.e.</em>, putative perfume containers), and other vessels, and process them as described above for the putative fermented food containers.<br/><em>Note: Containers suspected to have held oily products should not be considered as controls, since yeast are major inhabitants of oil (Aouizerat et al., 2019).</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Delivery to the lab<ol class="npwtwo"><li>Using gloves at all times, transfer the tubes, containers and their documentation to the lab.</li><li>Store in-doors at room temperature.<br/><br/></li></ol></li></ol><p class="ppzzs"><strong><span style="font-size:24px;">Part II: Isolation and characterization of yeast from environmental samples (in the lab)</span></strong> </p><p class="pptt" id="biaoti30838">## Materials and Reagents</p><ol class="npwone"><li>For yeast isolation<ol class="npwtwo"><li>50 ml sterile Falcon tubes (MINIPLAST EIN SHEMER, ref: 835-050-21-111)</li><li>15 ml sterile Falcon tubes (MINIPLAST EIN SHEMER, ref: 835-015-40-111)</li><li>Lab strain of <em>S. cerevisiae </em>to serve as a control. We used Sefale S-04 (Fermentis)</li><li>YPD broth (Difco, catalog number: bbl242820)</li><li>Agar (Bacto<sup>TM</sup> Agar, BD, catalog number: 214010)</li><li>Selective agar plates: Sabouraux dextrose agar supplemented with chloramphenicol and gentamicin 4 mg/ml (NOVAmed Israel, catalog number: BA-114)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>For yeast DNA purification<ol class="npwtwo"><li>AMPure XP beads (New England BioLabs, catalog number: E6270)</li><li>Acid washed glass beads (Sigma-Aldrich, catalog number: G4649)</li><li>Ice cubes</li><li>RNase A (10 mg/ml DNase- and Protease-free RNase, Thermo Fisher Scientific, catalog number: EN0531)</li><li>Sterile ddH<sub>2</sub>O</li><li>Phenol:chlorophorm:Isoamyl alcohol in ratios of is 25:24:1 respectively, saturated with Tris (10 mM) and EDTA (1 mM) at pH 8 (Sigma-Aldrich, catalog number: P3803)</li><li>Tris-HCl, pH 8.0 (10 mM), EDTA, pH 8.0 (TE buffer, Thermo Fisher Scientific, catalog number: AM9858)</li><li>Ammonium acetate (Sigma-Aldrich, catalog number: A1542)</li><li>Ethanol (GADOT, Israel)</li><li>Nextera XT DNA kit (Illumina, catalog number: FC-131-1096)</li><li>Malt extract (Chen shmuel chemicals, Israel)</li><li>Hops (Hopsteiner, Hop pellets 90)</li><li>Glycerol (50%) (Bio-Lab)</li><li>PBS buffer</li><li>âSmash and Grabâ lysis solution (Harju <em>et al.</em>, 2004) (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti30839">### Equipment</p><ol class="npwtwok"><li>Lab Glass flasks</li><li>Pipettes of all sizes (Eppendorf)</li><li>Incubator that can heat to 30 Â°C (QSR Technologies)</li><li>Incubator Shaker (New Branswick scientific, Exella E24 incubator shaker)</li><li>Spectrophotometer (VWR, v-1200 spectrophotometer)</li><li>Centrifuge (Eppendorf, centrifuge 5430R)</li><li>Plate readerâSynergy<sup>TM </sup>2 Multi-Mode Microplate Reader (BioTek)</li><li>DNA quantification device such as Qubit (Invitrogen, Qubit<sup>Â®</sup> 2.0 fluorometer)</li><li>-80 Â°C freezer (Thermo Scientific<sup>TM</sup>, catalog number: 5IDTSX)</li><li>Vortex (Fisher Scientific, Fisher STD mini vortexer 230v)</li><li>Fume hood of any kind for the work with the Phenol:chlorophorm:Isoamyl alcohol</li><li>Laminar hood (Thermo Scientific<sup>TM</sup>, MSC-Advantage<sup>TM</sup>)</li><li>PCR hood (Lab Companion)</li><li>PCR thermocycler (QSR Technologies, Thermal cycler GE48529)</li><li>Microscope (ZEISS, AXIO Lab.A1)</li><li>Microscope camera attached to PC (ZEISS, AXIOCAM 105 color)<br/></li></ol><p class="pptt" id="biaoti30840">### Software</p><ol class="npwtwok"><li>96-well plate reader software (BioTek, Gen5 2.07)</li><li>GraphPad Prism 8.0.2 (<a href="https://www.graphpad.com/" target="_blank">https://www.graphpad.com/</a>)</li><li>Geneious Pro 2019 (<a href="https://www.geneious.com/" target="_blank">https://www.geneious.com/</a>)</li><li>Microscope software (ZEISS, ZEN2 blue edition)</li><li>Trimmomatic 0.36 (Bolger <em>et al.</em>, 2014)</li><li>FastQC (<a href="http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc" target="_blank">http://www.bioinformatics.bbsrc.ac.uk/projects/fastqc</a>)</li><li>Celera assembler 8.3rc2 (Myers <em>et al.</em>, 2000)</li><li>BlobTools V1 (Laetsch and Blaxter, 2017)</li><li>R (<a href="https://www.r-project.org" target="_blank">https://www.r-project.org</a>) with "growthcurverâ package (<a href="https://cran.r-project.org/web/packages/growthcurver/vignettes/Growthcurver-vignette.html" target="_blank">https://cran.r-project.org/web/packages/growthcurver/vignettes/Growthcurver-vignette.html</a>)<br/></li></ol><p class="pptt" id="biaoti30841">### Procedure</p><p class="ppzz"><em>Note: Work routinely with gloves and lab safety equipment (<em>i.e.</em>, lab-coat, protective eye goggles). The beginning steps do not need sterile work conditions. However, after the initial isolation of the yeast, work in a laminar hood or near a burner.</em> </p><ol class="npwone"><li>Yeast isolation from vessels and control samples<ol class="npwtwo"><li>Put the sherds in a plastic container and cover them with sterile liquid YPD medium.</li><li>Incubate at room temperature without shaking for 7 days.</li><li>Spread 0.1 ml samples from the medium on selective agar plates for fungal isolation and incubate at 30 Â°C for 12 to 48 h.</li><li>Inspect the plates frequently for the appearance of yeast colonies (Figure 2A).<br/><em>Note: Yeast colonies usually grow faster than molds and filamentous fungi. Do not wait for more than 48 h or the plate will be covered by molds and filamentous fungi.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222010502_6984.jpg" width="122"/><br/><strong>Figure 2. Yeast colonies and cells.</strong> Yeast have typical colonies and cell morphology both of which are distinct from molds and bacteria. A. several types of yeast colonies isolated from ancient vessels (Aouizerat <em>et al.</em>, 2019). In general, yeast colonies are relatively large with a shiny white color in contrast to bacterial colonies which are usually small and matte. B. Yeast cells observed under light microscopy (60x). Note that yeast cells are usually bigger than typical bacteria and many kinds of yeast divide by budding rather than fission. C. Upper panel, a selective agar plate for fungal isolation was plated as described in the Procedure A of Part II. Note the low number of bacterial colonies due to the antibiotics (blue circle), structure of molds (green circle) and distinctive shiny colony of yeast (red circle). In the bottom panel, a yeast colony which was picked and re-streaked on a YPD plate (red arrow).<br/></li><br/><li>Collect any colony which seems to be a yeast colony, observe for typical yeast cells using light microscopy (Figure 2B) and plate it on solid YPD agar plates, containing 2% w/v Bacto-agar (Figure 2C).</li><li>Incubate the plates at 30 Â°C for 24-48 h until yeast colonies are clearly observed.<br/><em>Note: If a mixture of molds and yeast grow, re-pick a separate yeast colony to a new YPD plate and repeat Step A6.</em> </li><li>Inspect the colonies visually and under light microscope (60x magnification) for typical yeast cells which are distinct from bacteria colonies (Figure 2).</li><li>To make a frozen stock of the yeast, pick a single colony to 5 ml YPD broth and incubate it for 24 h at 30 Â°C, in a shaker incubator under aerobic conditions with agitation (250-300 rpm).</li><li>Mix 0.5 ml of the culture with sterile 0.5 ml glycerol (50%), vortex to mix, aliquot and store at -80 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Yeast Characterization<ol class="npwtwo"><li>Yeast DNA isolation<br/><em>Note: Yeast cell DNA isolation was performed as previously described (Holm <em>et al.</em>, 1986).</em> <ol class="npwthree"><li>Select a yeast containing tube from the -80 Â°C and allow to defrost at room temperature. When thawed, perform isolation streak on a YPD plate and incubate at 30 Â°C until colonies are observed (about 48 h).</li><li>Transfer a single colony to 10 ml YPD broth in a sterile 100 ml flask. Grow the yeast overnight at 30 Â°C with 200 rpm shaking.</li><li>Transfer the culture to 50 ml Falcon tube and centrifuge at 1,000 <em>x g</em> for 5 min. Pour out the supernatant, re-suspend the pellet in sterile water, re-centrifuge and discard the supernatant.</li><li>In a fume hood, add 200 Âµl of Phenol:chlorophorm:Isoamyl alcohol, 0.3 g acid washed glass beads and 200 Âµl of Smash and Grab solution and vortex for 3 min in order to lyse the cells.</li><li>Add 0.2 ml TE buffer and centrifuge the cells at 1,000 <em>x g</em> for 5 min at room temperature.</li><li>Transfer the aqueous layer containing the DNA to 1 ml ethanol and incubate it for 20 min at -80 Â°C.</li><li>Wash by centrifugation at 10,000 <em>x g</em> for 5 min and re-suspend in 0.4 ml TE buffer.<br/><em>Note: DNA pellet is transparent and can be difficult to seeâavoid losing the pellet.</em> </li><li>Add 1 Âµl of RNase A.</li><li>Incubate the solution at 37 Â°C for 5 min.</li><li>Add 10 Âµl of ammonium acetate (4 M) and 1ml of 70% ethanol and incubate for 20 min at -80 Â°C.</li><li>Centrifuge at maximal speed for 10 min (at least 15K <em>x g</em>) and discard the supernatant.</li><li>Re-suspend the pellet in 50 Âµl of TE buffer.</li><li>Quantify the DNA using Qubit, it should be 10-20 Âµg/ml.</li><li>Store the extracted DNA at -20 Â°C.</li></ol></li><li>Yeast identification by internal transcribed spacer (ITS)<br/><em>Note: For rapid and cheap identification you can use sanger sequencing of the ITS region. Keep in mind that the specific yeast species might not be represented in the database and thus full genome sequencing is preferable.</em> <ol class="npwthree"><li>Amplify the ITS region using Standard PCR kit and program with Illuminaâs primers: ITS1F; ACACTGACGACATGGTTCTACACTTGGTCATTTAGAGGAAGTAA and ITS2R; TACGGTAGCAGAGACTTGGTCTGCTGCGTTCTTCATCGATGC.</li><li>Send the PCR fragments for Sanger sequencing.</li><li>BLAST the sequence results against the ISHAM barcoding database (<a href="http://its.mycologylab.org" target="_blank">http://its.mycologylab.org</a>) and the NCBI database (<a href="https://blast.ncbi.nlm.nih.gov/ Blast.cgi" target="_blank">https://blast.ncbi.nlm.nih.gov/ Blast.cgi</a>).</li></ol></li><li>Full genome sequencing and assembly<br/><em>Note: DNA sequencing can be performed using any common NGS technology. Our libraries were prepared using a Nextera XT DNA kit (Illumina, San Diego, CA), and the DNA was amplified by a limited-cycle PCR followed by purification using AMPure XP beads. The DNA libraries were normalized, pooled, and tagged in a common flow cell at 2 x 250 base-paired-end reads using the NextSeq platform. For more detailed protocols see Aouizerat <em>et al.</em> (2019). The assembly also can be performed using numerous NGS analysis tools. Here we describe the analysis we performed (Aouizerat <em>et al.</em>, 2019).</em> <ol class="npwthree"><li>Remove Illumina adaptors using Trimmomatic 0.36.</li><li>Determine the quality of the reads using FastQC.</li><li>Perform De-novo assembly with the Celera assembler 8.3rc2.</li><li>Exclude non-target-species scaffolds using BlobTools V1.</li></ol></li><li>Growth curve analysis (in wort)<br/><em>Note: This procedure is used for the comparison of various growth parameters of yeast or bacteria in liquid under various conditions such as varying media, temperatures and addition of different compounds. Here, the growth of yeast in beer wort was performed, in order to test whether the isolated yeast might have been selected for in the past under such conditions. As </em><em>a positive control we used a modern beer-producing strain of <em>S. cerevisiae </em>which is commercially available and as a negative control the pathogen yeast Candida parapsilosois which does not produce beer.<br/></em> <br/><strong><em>Wort Preparation</em></strong><br/><em>Note: The wort is prepared according to a standard recipe (Kunze and Manger, 2004).</em> <ol class="npwthree"><li>Heat water (5 L) to a pasteurization temperature of 72 Â°C.</li><li>Add malt extract to a final concentration of 100 g/L, while thoroughly stirring, and allow to infuse together for 30 min in temperatures between 63 and 67 Â°C.</li><li>Increase the solution temperature to 100 Â°C.</li><li>Add 1 g/L of hops.</li><li>Boil the solution for 45 min.</li><li>Meanwhile prepare 5 L ice-cold water by mixing water with ice cubes.</li><li>Add 1 g/L of hops once more.</li><li>Let boil for an additional minute.</li><li>Add the previously prepared ice-cold water (from Section f) to the mixture.</li><li>Transfer the wort to a sterile flask and keep at 4 Â°C.<br/><br/></li></ol><em><strong>Growth curve comparison</strong></em> <ol class="npwthree"><li>Perform isolation streak of yeast from frozen stock on YPD agar plate.</li><li>Incubate the plate at 30 Â°C for 48 h.</li><li>Transfer a colony to 20 ml liquid YPD and incubate it at 30 Â°C, under aerobic conditions with agitation (250-300 rpm).</li><li>Centrifuge the yeast at 1,000 <em>x g</em> for 5 min, wash in sterile PBS buffer and re-suspend the pellet in wort.</li><li>Dilute the solution 1:100 in wort.</li><li>Add 200 Î¼l of the diluted solution/well of 96-well plate in triplicates or more repeats.</li><li>Incubate the plate in a plate reader at 30 Â°C with linear shaking of 5 s every 20 min followed by an optical density (OD<sub>600</sub>) reading for 48-96 h.</li><li>Export the results to a text file (Figure 3A).</li><li>Fit a logistic equation using R package âgrowthcurverâ (Figure 3B).</li><li>Compare the statistical significance of the differences between the various parameters using Student's <em>t</em>-test.</li><li>Determine the relative distance between the samples using Principal Components Analysis (PCA, R prcomp () command) (Figure 3C).</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191222/20191222011544_9869.jpg" width="84"/><br/><strong>Figure 3. Example of growth curve analysis.</strong> A. Growth curves were read as output from the plate reader. B. A logistic curve was fitted to each curve using R growthcurver. C. A PCA representation of the distance between the various parameters of the logistic equations of each sample. These results were adapted from Aouizerat <em>et al.</em>, 2019.</li></ol></li></ol><p class="pptt" id="biaoti30842">### Notes</p><p class="ppzz">For DNA sequencing we enlisted the help of the interdepartmental equipment unit of the Ein-Kerem Campus, Hebrew University of Jerusalem.</p><p class="pptt" id="biaoti30843">### Recipes</p><ol class="npwtwok"><li>âSmash and Grabâ lysis solution<br/>Triton X-100 (2%)<br/>SDS (1%)<br/>NaCl (100 mM)<br/>Tris-HCl, pH 8.0 (10 mM)<br/>EDTA, pH 8.0 (1 mM)<br/></li></ol><p class="pptt" id="biaoti30844">### Acknowledgments</p><p class="ppzz">This protocol was adapted from Aouizerat <em>et al.</em> (2019). Isolation and characterization of live yeast cells from ancient vessels as a tool in bio-archeology. <em>mBio</em> 10(2): e00388-19. We thank Prof. Christina Warinner from University of Jena for her helpful suggestions.</p><p class="pptt" id="biaoti30845">### Competing interests</p><p class="ppzz">Conflict of interest statement: None declared.</p><p class="pptt" id="biaoti30846">### References</p><ol class="npwtwok"><li>Andam, C. P., Worby, C. J., Chang, Q. and Campana, M. G. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27618404" target="_blank">Microbial Genomics of Ancient Plagues and Outbreaks.</a> <em>Trends Microbiol</em> 24(12): 978-990.</li><li>Aouizerat, T., Gutman, I., Paz, Y., Maeir, A. M., Gadot, Y., Gelman, D., Szitenberg, A., Drori, E., Pinkus, A., Schoemann, M., Kaplan, R., Ben-Gedalya, T., Coppenhagen- Glazer, S., Reich, E., Saragovi, A. S., Lipschits, O., Klutstein, M. and Hazan, R. (2019). <a href="https://www.researchgate.net/publication/331122266_Isolation_and_characterization_of_live_yeast_cells_from_ancient_vessels_as_a_tool_in_bio-archeology" target="_blank">Isolation and characterization of live yeast cells from ancient vessels as a tool in bio-archaeology.</a> <em>mBio</em> 10(2): e00388-19.</li><li>Beaudry, M. C. and Metheny, K. B.(2015). <em><a href="https://www.researchgate.net/publication/322641117_Archaeology_of_Food_An_Encyclopedia" target="_blank">Archaeology of food: an encyclopedia.</a></em> Rowman &amp; Littlefield.</li><li>Bolger, A. M., Lohse, M. and Usadel, B. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" target="_blank">Trimmomatic: a flexible trimmer for Illumina sequence data.</a> <em>Bioinformatics</em> 30(15): 2114-2120.</li><li>Cavalieri, D., McGovern, P. E., Hartl, D. L., Mortimer, R. and Polsinelli, M. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15008419" target="_blank">Evidence for <em>S. cerevisiae </em>fermentation in ancient wine.</a> <em>J Mol Evol 57 Suppl</em> 1: S226-232.</li><li>Dietler, M. (2006). <a href="https://www.annualreviews.org/doi/abs/10.1146/annurev.anthro.35.081705.123120" target="_blank">Alcohol: anthropological/archaeological perspectives.</a> <em>Annu Rev Anthropol</em> 35: 229-249.</li><li>Donoghue, H. D., Spigelman, M., Greenblatt, C. L., Lev-Maor, G., Bar-Gal, G. K., Matheson, C., Vernon, K., Nerlich, A. G. and Zink, A. R. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15336226" target="_blank"><em>Tuberculosis</em>: from prehistory to Robert Koch, as revealed by ancient DNA.</a> <em>Lancet Infect Dis</em> 4(9): 584-592.</li><li>Harju, S., Fedosyuk, H. and Peterson, K. R. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15102338" target="_blank">Rapid isolation of yeast genomic DNA: Bust n' Grab.</a> <em>BMC Biotechnol</em> 4: 8.</li><li>Hornsey, I. S. (2012). <a href="https://pubs.rsc.org/en/content/ebook/978-1-84973-161-4" target="_blank"><em>Alcohol and its role in the evolution of human society.</em> </a>Royal Society of Chemistry.</li><li>Holm, C., Meeks-Wagner, D. W., Fangman, W. L. and Botstein, D. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3015730" target="_blank">A rapid, efficient method for isolating DNA from yeast.</a> <em>Gene</em> 42(2): 169-173.</li><li>Joffe, A. H. (1998). <a href="https://www.jstor.org/stable/10.1086/204736?seq=1" target="_blank">Alcohol and social complexity in ancient western Asia.</a> <em>Curr Anthropol</em> 39: 297-322.</li><li>Kunze, W. and Manger, H. J. (2004). <a href="https://www.worldcat.org/title/technology-brewing-and-malting/oclc/491820300" target="_blank">Technology brewing and malting.</a> VLB, Berlin, Germany.</li><li>Laetsch, D. R. and Blaxter, M. L. (2017). <a href="https://www.researchgate.net/publication/318805284_BlobTools_Interrogation_of_genome_assemblies" target="_blank">BlobTools: interrogation of genome assemblies.</a> <em>F1000 Res</em> 6: 1287.</li><li>Legras, J. L., Merdinoglu, D., Cornuet, J. M. and Karst, F. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17498234" target="_blank">Bread, beer and wine: <em>Saccharomyces</em> <em>cerevisiae</em> diversity reflects human history<em>.</em></a> <em>Mol Ecol</em> 16(10): 2091-2102.</li><li>McGovern, P. E. (2009). <em><a href="https://www.jstor.org/stable/10.1525/j.ctt7zw1wq" target="_blank">Uncorking the past: the quest for wine, beer, and other alcoholic beverages.</a></em> University of California Press.</li><li>Muhlemann, B., Jones, T. C., de Barros Damgaard, P., Allentoft, M. E., Shevnina, I., Logvin, A., Usmanova, E., Panyushkina, I. P., Boldgiv, B., Bazartseren, T., Tashbaeva, K., Merz, V., Lau, N., Smrcka, V., Voyakin, D., Kitov, E., Epimakhov, A., Pokutta, D., Vicze, M., Price, T. D., Moiseyev, V., Hansen, A. J., Orlando, L., Rasmussen, S., Sikora, M., Vinner, L., Osterhaus, A., Smith, D. J., Glebe, D., Fouchier, R. A. M., Drosten, C., Sjogren, K. G., Kristiansen, K. and Willerslev, E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30108364" target="_blank">Author Correction: Ancient hepatitis B viruses from the Bronze Age to the Medieval period. </a><em>Nature</em> 562(7726): E4.</li><li>Myers, E. W., Sutton, G. G., Delcher, A. L., Dew, I. M., Fasulo, D. P., Flanigan, M. J., Kravitz, S. A., Mobarry, C. M., Reinert, K. H., Remington, K. A., Anson, E. L., Bolanos, R. A., Chou, H. H., Jordan, C. M., Halpern, A. L., Lonardi, S., Beasley, E. M., Brandon, R. C., Chen, L., Dunn, P. J., Lai, Z., Liang, Y., Nusskern, D. R., Zhan, M., Zhang, Q., Zheng, X., Rubin, G. M., Adams, M. D. and Venter, J. C. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10731133" target="_blank">A whole-genome assembly of <em>Drosophila</em>.</a> <em>Science</em> 287(5461): 2196-2204.</li><li>Parker, B. J. and McCool, W. (2015). <a href="https://www.journals.uchicago.edu/doi/10.3998/jar.0521004.0071.303?mobileUi=0&amp;" target="_blank">Indices of household maize beer production in the Andes: an ethnoarchaeological investigation.</a> <em>J Anthropol Res</em> 71(3): 359-400.</li><li>SchiefenhÃ¶vel, W. and Macbeth, H. (2011). <a href="https://www.jstor.org/stable/j.ctt9qd5gz" target="_blank">Liquid bread: beer and brewing in cross-cultural perspective.</a> <em>Berghahn Books</em>.</li><li>Stika, H. P. (2011). <a href="https://link.springer.com/article/10.1007/s12520-010-0049-5" target="_blank">Early Iron age and late mediaeval malt finds from Germanyâattempts at reconstruction of early Celtic brewing and the taste of Celtic beer.</a> <em>Archaeol Anthrop Sci</em> 3(1): 41-48.<br/></li></ol>