<body>
            <h1>Laser Capture Micro-dissection (LCM) of Neonatal Mouse Forebrain for RNA Isolation</h1>
            <p><strong>Authors:</strong> Achira  Roy, Mei Deng, Kimberly A. Aldinger, Ian A. GLass, Kathleen J. Millen</p>
            <h2>Abstract</h2>
            <p>Precise and reproducible isolation of desired cell types or layers from heterogeneous tissues is crucial to analyze specific gene profiles and molecular interactions <em>in vivo</em>. Forebrain is the core site of higher functions, like cognition and memory consolidation. It is composed of heterogeneous and distinct cell types, interconnected to form functional neural circuits. Any alteration in the development or function often leads to brain disorders with profound consequences. Thus, precise molecular understanding of forebrain development in normal and diseased scenarios is important. For quantitative studies, most traditional analytical methods require pooling of large cell populations, that results in loss of <em>in vivo</em> tissue integrity and of spatial, molecular and cellular resolution. Laser capture microdissection (LCM) is a fast and extremely precise method of obtaining uncontaminated, homogeneous sets of specific cell types and layers. Our current procedure involves cryo-sectioning and laser microdissection of fresh-frozen mouse forebrains, that are genetically modified and treated with small-molecule therapeutics. Using LCM, specific regions of interest, such as neural layers, cells from adjacent yet distinct subregions within a tissue layer, are obtained under RNase-free conditions. These small cellular cohorts are further used for downstream, high-throughput genomic or transcriptomic assays. Here, we have introduced break-points at multiple stages throughout our protocol. This makes our method simpler and more user-friendly to follow, without compromising on the quality. The current protocol can easily be adapted for different brain regions, as well as for other model organisms/human tissue.</p>
            <p><strong>Keywords:</strong> Mouse, Forebrain, Laser capture microdissection, RNA isolation, Neural progenitors, Microscopy, RNA sequencing, Hippocampus</p>
            <h2>Background</h2><p>To understand the molecular mechanisms underlying normal and diseased neural processes, it is critical to analyze the interactions <em>in vivo</em>, within a specific tissue microenvironment. It is quite challenging to precisely access different areas of the developing brain, especially in deep areas like hippocampus. The level of complication increases further due to the existence of cellular heterogeneity even within a single neural layer. Traditional quantitative methods like microarray <em>etc.</em>, using gross tissue lysates or various primary and organotypic culture systems, rely heavily on pooling gene expression data from thousands of cells and reporting a population-wide average (Levsky and Singer, 2003). Thus, these procedures fail to capture subtle differences within distinct cell subsets as well as to maintain <em>in vivo</em> tissue/cell integrity. Moreover, there remains a high risk of tissue contamination from neighboring areas during sample collection. Laser-capture microdissection (LCM) combines the benefits of light microscopy and precision of laser beam to procure homogeneous subpopulations of cells from heterogeneous tissue under direct microscopic visualization (Emmert-Buck <em>et al.</em>, 1996). The laser cutting is usually less than 1 Î¼m wide. This allows target cells to remain unaffected by the laser beam. LCM technology can selectively and routinely harvest regions of interest, ranging from specific tissue layers to single cells and chromosomes (<a href="https://www.leica-microsystems.com/products/light-microscopes/details/product/leica-lmd7/" target="_blank">https://www.leica-microsystems.com/products/light-microscopes/details/product/leica-lmd7/</a>). This enables the user to obtain reproducible, targeted starting material for biologically relevant data analyses.<br/><br/>Data using the described LCM protocol was published in Roy <em>et al.</em> (2019). In this study, we found that activating mutations in the phosphoinositide 3-kinase (PI3K) pathway results in focal disruption of cell-cell adhesion, proliferation and apical tissue integrity, leading to cortical gyrification (in both neocortex and hippocampus) and developmental hydrocephalus in mice. Conventional quantitative analytical methods were not precise enough to distinguish cellular/molecular heterogeneity within a neural layer, between adjacent gyral and sulcal zones in the mutant mouse brain. Hence, we used LCM to specifically dissect selective portions of the developing ventricular zone from P0 control and mutant littermates, followed by RNA sequencing. This data was then analyzed to study the effect of the <em>PI3K</em> mutation on neural progenitors. Further, we performed the same experiment after administering a small-molecule drug <em>in utero</em>, to compare the effect of drug treatment on the gene expression profile between controls and mutants.<br/><br/>Here, we present detailed step-wise instructions including experimental âdoâs and donâtsâ, and also illustrate key components and steps using photographs, to aid new researchers in setting up this useful procedure. We have introduced a number of stopping steps throughout the protocol. This makes our method simple and more user-friendly, without making any compromise on the quality, quantity and integrity of the end product (RNA, DNA <em>etc.</em>). This method can be used with fresh-frozen and fixed tissues.<br/><br/>Some other applications of this protocol are as follows:<br/>1.Absolute identification of possible mosaic mutations with cells of interest, in mouse models or human patients.<br/>2.Downstream analyses like selective quantitative Real-Time PCR, protein assays like Western blot analysis, chromatin immuno-precipitation, genomics (DNA assays), as well as metabolomics.<br/>3.Live cell culture, cloning and genetic manipulations.<br/>4.Isolation of small quantity of specific tissue material, highly useful for plant biology, forensic science and prenatal diagnostics.</p>
        </body>

<p class="pptt" id="biaoti30858">## Materials and Reagents</p><ol class="npwtwok"><li>Sterile gloves</li><li>DNase-free, RNase-free micropipette tips</li><li>Fisherbrand Permanent fine tip marking pen (Fisher Scientific, catalog number: 13-379-4), to label embedding molds, and tubes for genotyping, if required</li><li>Fisherbrand Petri Dish with Clear Lid 100 mm x 15 mm, sterile (Fisher Scientific, catalog number: FB0875713)</li><li>Bucket of ice</li><li>Embedding molds (Thermo Scientific, catalog number: 70182)</li><li>Cryogenic Fiberboard Freezer Box, 2ââ (VWR International, catalog number: 89214-750)</li><li>Regular duty single edge blades (Personna, catalog number: 94-115-71) </li><li>Low Profile Microtome Blades (Sakura, Accu-Edge, catalog number: 4689)</li><li>Fisherbrand Superfrost Plus microscope slides, precleaned (Fisher Scientific, catalog number: 12-550-15)</li><li>MembraneSlides, nuclease and human nucleic acid-free polyethylene naphthalate (PEN)-Membrane 2 Î¼m (PEN slides) (Leica Microsystems, catalog number: 11505189)</li><li>Pencil, to label slides</li><li>New plastic 5-slide mailerâopen top (Electron Microscopy Sciences, catalog number: 71548-01), for storage of PEN slides</li><li>Sterile 50 ml Polypropylene conical tube (Falcon, catalog number: 352070) for staining or storage of PEN slides</li><li>Kimwipes (Kimberly-Clark Professional, catalog number: 34155 or 34120)</li><li>Samco<sup>TM</sup> Graduated Transfer Pipettes, sterile individually wrapped (Thermo Fisher Scientific, catalog number: 274-1S) </li><li>Tissue collection tubes: 0.5 ml Tubes with Flat Cap, certified DNase/RNase-free (Thermowell Tubes, catalog number: 6530)</li><li>High quality bags from VacUpack (for example: 4 Mil Vacuum Bags, size: 8âX10â, U.S. Vacuum Packaging and Wrapping LLC., catalog number: SS-VB4-0810)</li><li>Small biohazard bags, to dispose of mouse carcasses (Medline, catalog number: DYND30261)</li><li>RNase-free 1.7 ml tubes (Olympus Plastics, Genesee Scientific, catalog number: 24-282A)</li><li>Aqui-Pad Laboratory Mat (Therapak, catalog number: 10324G)</li><li><strong>Animal:</strong> Mouse, neonatal pups (P0-P6)<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Male and female mice are mated, and embryos are staged considering the morning a vaginal plug is detected as embryonic day (E)0.5. In utero drug treatment can be done using this staging. Specifically, the types of mating used in (Roy <em>et al.</em>, 2019) are described in detail in the articleâs Methods section.</em> </li><li><em>The day the pups are born is considered as P0. For FVB strain, the gestation period is E18-E19.</em> </li><li><em>No alterations to the protocol are required if different mouse strains are used.</em> </li></ol></li><li>RNasin Plus RNase Inhibitor (Promega, catalog number: 5067-1513)</li><li><em>Desiccants:</em> Indicator blue Drierite<sup>TM</sup> desiccant (W.A. Hammond Drierite Company Ltd., mesh size: 8, stock number: 23005); regular Drierite<sup>TM</sup> desiccant (W.A. Hammond Drierite Company Ltd., mesh size: 8, stock number: 12001)</li><li>RNaseZap RNase Decontamination Solution (Invitrogen by Thermo Fisher Scientific, catalog number: AM9780)</li><li>Phosphate Buffer Saline PBS pH 7.4 (1x) (Gibco, catalog number: 10010-023)</li><li>Dry Ice</li><li>Optimum Cutting Temperature (OCT) Embedding Compound (Electron Microscopy Sciences, TissueTek #4583, catalog number: 62550-01)</li><li>Cresyl violet powder (Sigma-Aldrich, Harleco, catalog number: 190-12), storage: 15 Â°C to 30 Â°C<br/><em>Note: Keep the bottle RNase-free.</em> </li><li>RNeasy Plus Micro Kit (50) (Qiagen, catalog number: 74034, storage: RNeasy MinElute Spin Columns at 2-8 Â°C, remaining components at room temperature (15-25 Â°C)</li><li>Agilent RNA 6000 Pico Kit (Agilent Technologies, catalog number: 5067-1513), storage: 4 Â°C, RNA ladder at -80 Â°C</li><li>Molecular Biology Grade Water (Corning, catalog number: 46-000-CM)</li><li>Ethanol 200 Proof Anhydrous (Decon Labs, Inc., catalog number: 2716)</li><li>2-Mercaptoethanol (Î²-mercaptoethanol) (Bio-Rad, catalog number: 161-0710)</li><li>70% ethanol (100 ml) (see Recipes)</li><li>95% ethanol (100 ml) (see Recipes)</li><li>RNaseZap/70% ethanol mixture (100 ml) (see Recipes)</li><li>75% ethanol (freshly-prepared) (see Recipes)</li><li>Nissl stain for RNA research (50 ml) (see Recipes)</li></ol><p class="ppzz"><em>Note: If the experiment requires comparison of mouse tissue between control and genetically modified mutant models and/or Â±drug treatment, reagents to perform mouse tissue genotyping are required as briefly mentioned in Roy <em>et al.</em> (2015 and 2019).</em> </p><p class="pptt" id="biaoti30859">### Equipment</p><ol class="npwtwok"><li>Clean micropipettes</li><li>16oz Rubber Mallet with Wood Handle, to crush dry ice (Olympia Tools, catalog number: 60661116)</li><li>Dissection instruments:<br/><ol class="npwthree"><li>Dumont fine forceps #5 (Fine Science Tools, catalog number: 11252-20)</li><li>Dumont fine forceps #55 (Fine Science Tools, catalog number: 11255-20)</li><li>Dumont coarse forceps AA (Fine Science Tools, catalog number: 11210-20)</li><li>Fine ScissorsâSharp (Fine Science Tools, catalog number: 14060-10)</li></ol></li><li>Fisherbrand<sup>TM</sup> Stainless Steel Micro Chattaway Spatulas (Fisher Scientific, catalog numbers: 11543482 and 14-373)</li><li>Autoclave</li><li>Cryostat (Leica Microsystems, model: CM1850)</li><li>Brightfield Leica EZ4 Stereo Microscope (Leica Microsystems, catalog number: 10447197)</li><li>-80 Â°C freezer</li><li>-20 Â°C freezer</li><li>4 Â°C refrigerator</li><li>Polyethylene ethanol wide-mouth wash bottle with polypropylene cap; 500 ml (Bel-Art SP Scienceware, catalog number: F11816-0019)</li><li>Laser Microdissection Microscope (Leica Microsystems, model: LMD7)</li><li>Agilent 2100 Bioanalyzer (Agilent Technologies, catalog number: G2939BA)</li><li>Chip priming station (Agilent Technologies, catalog number: 5065-4401)</li><li>IKA MS3 vortex mixer for Agilent 2100 Bioanalyzer</li><li>NanoDrop 2000 microvolume spectrophotometer (Thermo Scientific, catalog number: ND-2000)</li><li>Vacuum Desiccator (Fisher Scientific, catalog number: 08-648-107)<br/><em>Optional: Desiccator, home-made using stacked mixture of regular Drierite and Indicator blue Drierite desiccant, in a large glass bottle.</em> </li><li>Microcentrifuge (Eppendorf, catalog number: 5417R)</li><li>Vortex (Scientific Industries, model: Vortex Genie-2, catalog number: SI-T236)</li><li>Analytical electronic balance (Reshy, catalog number: JM-B)</li><li>UV transilluminator (one used here is from Fotodyne, catalog number: FD33500; should preferably be used only RNase-free and without any chemical contamination)</li><li>Rocking shaker (Reliable Scientific Inc., model: 55), for solution preparation and staining of slides.</li><li>Vacuum Sealing System with Starter Kit (Foodsaver, catalog number: V2244)</li></ol><p class="ppzz"><em>Note: If the experiment requires mouse tissue genotyping, appropriate set of equipment should be available.</em> </p><p class="pptt" id="biaoti30860">### Software</p><ol class="npwtwo"><li>Laser Microdissection System (Leica Microsystems, version: 8.2.0.6739)</li><li>Agilent 2100 Expert Software (Agilent Technologies, version: B.02.08.SI648(SR2))</li><li>NanoDrop software (Thermo Scientific, version used: 1.6.198)</li><li><em>Optional:</em> Microsoft PowerPoint<br/></li></ol><p class="ppzzs"><em>Note: For genotyping, Image Lab<sup>TM</sup> Software (Bio-Rad, 1708265) is required for gel documentation.</em> </p><p class="pptt" id="biaoti30861">### Procedure</p><p class="ppzz"><em>Note: Prior to starting laser microdissection, the user should be well trained in cryo-sectioning and have sufficient knowledge of microscopy and histopathology to sample the correct tissue or cells. To minimize quality damage, the whole procedure of tissue handling should be carried out quickly, yet carefully.</em> </p><br/><ol class="npwone"><li>Preparation of solutions and set-up for brain dissection<br/><em>Notes:<br/></em> <ol class="npwthree"><li><em>Wear sterile disposable gloves and change frequently at every step, from preparing reagents till RNA analysis etc.</em> </li><li><em>Use clean, RNase-free instruments. Before handling the next specimen, spray the instruments with RNaseZap, rinse with RNase-free water and wipe with clean Kimwipes.</em> </li><li><em>Use RNase-free or Nuclease-free solutions, glassware and plastic ware.</em> </li><li><em>Use RNaseZap or similar product to clean the equipment or the surface of the working area.</em><em></em><br/></li><li><em>Freeze tissues immediately after surgery to obtain high-quality RNA. This minimizes gradual RNA degradation by ubiquitous RNases or heat.</em><em></em><br/></li></ol><br/><ol class="npwtwo"><li>Prepare working solutions, as mentioned in the Recipes section.</li><li>Label the cryomolds at room temperature using the Fisherbrand permanent marker, prior to the tissue dissection experiment, preferably on the side-wall opposite to the notched side (for easier orientation).</li><li>Autoclave the dissection instruments and spatula for sterilization, prior to the experiment.<br/><em>Note: The dissection tools should have never seen any fixative etc.; wipe them with RNaseZap, then RNase-free DNase-free water and dry with clean Kimwipes before starting the experiment.</em> </li><li>Wipe the tissue dissection area thoroughly with 100% RNaseZAP solution, let the area dry before placing the laboratory mat for dissection.</li><li>Assemble all reagents and equipment before tissue dissection, as shown in Figure 1.</li><li>Crush some dry-ice using the mallet to generate a soft powdered area for future placement of the embedding mold (Figure 2A).</li><li>Pour OCT slowly and carefully along one edge of the sterile embedding mold till it is half-filled (Figures 2B and 2F).<br/><em>Note: Avoid air bubbles in the OCT while pouring. Air bubbles disrupt the tissue integrity.</em><br/><br/><img alt="" class="layerphoto" height="139" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228084130_8093.jpg" width="300"/><br/><strong>Figure 1. Materials needed for dissection of neonatal mouse forebrain.</strong> Reagents and equipment needed for mouse brain dissection are labeled. 1x PBSâPhosphate buffer saline, working solution.<br/><br/></li></ol></li><li>Neonatal mouse brain dissection<ol class="npwtwo"><li>Euthanize neonatal mouse pups of the desired age (P0-P6) using procedures permitted by your Institutional guidelines. In Roy <em>et al</em>. (2019), as per the approved animal protocol (see Ethics statement), euthanasia was performed by inducing hypothermia anesthesia, followed by decapitation. For this procedure, place the newborn pups on the lid of a Petri dish and set on top of crushed ice for 2-3 min, leading to their unconsciousness. Confirm anesthesia in pups by lack of response to firm toe pinch, once they stop moving. Decapitate using sharp scissors.</li><li>Transfer the heads to a sterile 100 mm Petri dish containing sterile 1x PBS.</li><li>At this stage, if genotyping of the pups is required, take a tiny piece of the tail (â¤ 2 mm) of each pup and place them in labeled 1.7 ml microcentrifuge tubes for further processing.</li><li>Dispose of the carcasses using labeled biohazard bags.</li><li>Under a dissection microscope, dissect out the brains using fine and coarse forceps (Figure 2C). Dispose of the remaining carcasses using the biohazard bags.<br/><em>Note: If the Petri dish becomes murky with blood, you can transfer the brains at this stage to another Petri dish containing sterile 1x PBS.</em> </li><li>After removing the brain, sever the hindbrain and midbrain, retaining only the forebrain including the olfactory bulbs (Figures 2D-2E). The presence of the olfactory bulb helps to position the telencephalic hemisphere in the correct orientation in the embedding molds.<br/><br/></li></ol></li><li>RNase-free brain mounting<ol class="npwtwo"><li>With the help of the spatula and coarse forceps (if needed), take the brain out of the buffer and transfer to the pre-labeled embedding mold half-filled with OCT (Figures 2F-2G).<br/><em>Note: Try to remove liquid from the brain surface as much as possible without damaging the brain (you may carefully use a finely rolled Kimwipe if needed), before immersing it in the OCT.</em> </li><li>Orient the mouse forebrain such that it is parallel to or on the floor of the embedding mold.</li><li>For coronal sectioning (perpendicular to the antero-posterior axis of the forebrain), slowly orient the anterior side of the brain (olfactory bulb side) directly perpendicular to the notched side of the mold. Avoid air bubbles in the process (Figures 2G and 2K).</li><li>Once the brain is properly oriented, slowly pour OCT along one edge to fill the mold (or at least enough to cover the entire brain). Take precaution that the brain orientation remains intact and air bubbles do not develop in between OCT layers.</li><li>Carefully place the mold within the powdered dry ice and support the mold from the sides by pieces of dry ice (Figures 2H-2I).</li><li>Once the mold, with brain embedded, becomes completely opaque (Figure 2J), transfer it to a new, clean freezer box and immediately store at -80 Â°C freezer.<br/><br/><img alt="" class="layerphoto" height="321" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228084154_1116.jpg" width="300"/><br/><strong>Figure 2. Steps of dissecting and embedding a neonatal mouse forebrain.</strong> A. Crushed dry ice prepared for freezing the embedding mold. B. OCT poured along an edge of the embedding mold to avoid air bubbles. C-E. Dissection of neonatal mouse forebrain, in sterile 1x PBS. F-G. Forebrain carefully transferred to and then oriented in the embedding mold filled with OCT. H-J. Mold with OCT-covered oriented mouse forebrain placed in dry ice, until OCT freezes to opaque form. K. Schematic of how forebrain was oriented, to be used for coronal sectioning (Roy <em>et al.</em>, 2019).<br/><br/></li></ol></li><li>Preparation before tissue sectioning<ol class="npwtwo"><li>Clean cryostat, microtome blade and tools with a 1:1 mixture of RNaseZap and 70% ethanol (RNaseZap/70% ethanol) and then with sterile 95% ethanol (Recipes 1-3). Wipe dry with Kimwipes.</li><li>Incubate PEN membrane slides for 30 min under UV light (on the UV transilluminator) to enhance the binding of the tissue to the membrane slide.</li><li>Get a bucket of dry ice for tissue/embedded mold transportation.</li><li>Prepare solutions with RNase-free water and high-quality ethanol:<br/><ol class="npwthree"><li>75% ethanol (freshly prepared, Recipe 4).</li><li>Nissl stain (Recipe 5, prepare at least 3 days before use and store at 4 Â°C). Aliquot the amount needed for current use, by roughly estimating the total number of slides to be stained (10-15 drops of the solution is adequate to rapid stain each slide).</li></ol></li><li>Set temperature of cryostat in the range of -15 Â°C to -18 Â°C.</li><li>Label PEN slides with brain code/genotype using a pencil.<br/><br/></li></ol></li><li>Cryo-sectioning of brain tissue<ol class="npwtwo"><li>Wear new sterile disposable RNase-free gloves. Change gloves frequently between steps to maintain the RNase-free status.</li><li>Transfer an embedded brain mold, placed securely on dry ice, from -80 Â°C freezer to the cryostat.</li><li>Use new sterile, single-edge blade to tear the plastic embedding mold inside the cryostat. Trim the OCT-embedded block as needed.</li><li>For coronal staining, attach the block on the cryostat chuck (specimen clamp) such that the anterior side of the brain (notched side of mold, Figure 2K) faces perpendicular to the microtome blade inside the cryostat.</li><li>Place embedded brain inside the cryostat at least 20 min prior to sectioning, in order to allow it to come to the cryostat temperature.</li><li>Use a new sterile microtome blade for each experiment (Figure 3A).</li><li>Slice a few sections from the surface of the mold to avoid any sort of contamination, prior to the collection of actual sections to be mounted on slides.</li><li>Cut 8-10 Î¼m sections and mount on regular charged plus slides (on the positively-charged coated side). To identify your region of interest, rapid stain with Nissl staining solution or other dye for about 1 min, swivel gently, and observe slides under stereo microscope.<br/><em>Note: Staining procedure is explained in Step E14b.</em> </li><li>Once you start getting sections of your region of interest, change the slicing thickness to 16-18 Î¼m and mount on the PEN membrane slides. You can mount multiple sections on one slide. For example, ~12-14 sections were mounted on each PEN slide in Roy <em>et al.</em> (2019).<br/><em>Note: Mount the sections within the extent on the PEN-coated membrane (Figure 3B). Please take care that the coating does not get any scratches or leave the RNase-free area.</em> </li><li>Optimally, place the PEN slides on the desiccant during the section collection.</li><li>Clean the blade and cryo-sectioning stage with brush or Kimwipe in between sections.</li><li>Collect one section of every 5-10 sections and mount on the regular charged slide for histologic staining.</li><li>Air-dry the PEN slides at room temperature for about 2 min on desiccant. Make sure the collected sections are completely dry before staining.</li><li>After air-drying, PEN slides with collected sections can be:<br/><ol class="npwthree"><li>Stored in plastic slide mailer in a vacuumed bag, containing desiccant, at -80 Â°C, for a few months or longer, until user is ready to perform LCM.</li><li>Immediately transferred to 100% ethanol at room temperature for dehydration, followed by Nissl staining.<br/>The Nissl staining protocol is as follows [a quicker staining method, modified from Grundemann<em> et al. </em>(2008) and Leica LMD Protocol Guide <em>(</em><a href="https://www.leica-microsystems.com/products/light-microscopes/details/product/leica-lmd7/" target="_blank"><em>https://www.leica-microsystems.com/products/light-microscopes/details/product/leica-lmd7/</em></a><em>)</em>]:<br/><ol class="npwfour"><li>Stain slides one at a time using 50 ml tube or simultaneously using a big container.</li><li>Immerse the slides in fresh 100% ethanol, 5 min x 4 times, and place on the rocking shaker to remove any OCT attached.</li><li>Immerse slides in freshly-prepared 75% ethanol (Recipe 4) for ~1 min.</li><li>Place drops of Nissl stain (Recipe 5) directly onto the sections, using plastic transfer pipette, and incubate for 1 min. Swivel the slides gently. On average, 10-15 drops of the solution are adequate per slide.<br/></li><li>Quickly rinse the slides with fresh 100% ethanol, 5 s x 3 times.<br/></li><li>Finally, immerse the slides in fresh 100% ethanol for 1 min, to allow the stain to bind onto the sections.<br/></li><li>Observe sections under a brightfield dissecting microscope.<br/></li><li>Allow the slides to stand for at least 30 min in order to make them completely dry before storage or use.</li></ol></li></ol><em>Note: Optimally, finish staining of sections strictly within 10 min in total.</em><br/></li><li>Store completely dried PEN slides with mounted sections at -80 Â°C, either before or after Nissl staining. Neither procedure will negatively affect the RNA quality. Keep the slides at -80 Â°C until you are ready to start LCM.<br/><br/><img alt="" class="layerphoto" height="139" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228084340_2233.jpg" width="300"/><br/><strong>Figure 3. Preparation for tissue sectioning and laser microdissection.</strong> A. New sterile microtome blades used for brain sectioning. B. PEN slides: mount sections onto the inner film, which finally gets laser-cut. C. Home-made desiccator using Drierite to dry slides prior to LCM. D. Representative image of a Nissl-stained coronal hemi-section of P0 mouse forebrain, used in microdissection and data analysis (Roy <em>et al.</em>, 2019), showing how region of interest is traced. In this context, ventricular zone of P0 mouse hippocampal CA1 region (CA1 vz) was traced.<br/><br/></li></ol></li><li>Laser microdissection<br/><em>Notes:</em><br/><ol class="npwtwo"><li><em>All steps of laser microdissection should be performed in a designated RNase-free zone, in a clean, steady air-conditioned room.</em> </li><li><em>Be cautious to slowly adjust frozen slides to room temperature before laser microdissection. This is critical for RNA quality since water (moisture) activates RNases.</em> </li><li><em>Wipe the work area with Kimwipes dipped in RNase-free water, followed by 100% ethanol.</em> </li><li><em>Clean the stage, substage and UV shield of the LMD machine sequentially with 95% and 100% ethanol.</em><br/><br/><img alt="" class="layerphoto" height="263" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228084424_3691.jpg" width="300"/><br/><strong>Figure 4. Flowchart showing different steps of laser microdissection of neonatal mouse forebrain tissue.</strong> Different stepsâtissue sectioning, laser-capture microdissection, downstream applications. Some images are obtained from Roy <em>et al</em>. (2019), where we compared the genetic profiles of the hippocampal CA1 ventricular zone (CA1 vz) between P0 control and activating <em>Pik3ca</em> mutant Â± drug, using RNA sequencing.<br/><br/></li></ol><ol class="npwtwo"><li>Take the PEN slides (in the vacuumed bag) out of -80 Â°C freezer and allow it to slowly reach room temperature, for at least 60 min, before opening the bag. This is to avoid formation of water condensation on the sections. This step is critical for preserving RNA quality. Another method of drying the slide is to use the vacuum desiccator or a home-made RNase-free desiccator (Figure 3C) and dry the slides on the same day.</li><li>Switch the power of the Leica LMD equipment on.</li><li>Clean the collect tube caps (0.5 ml) with RNaseZap, followed by a rinse with RNase-free water.</li><li>Stain the section collected on regular slide (see Step E12) to confirm the area of interest, before performing LCM on serial sections mounted on PEN slides.<br/><em>Alternate method: If one studying the scientific question and one executing the laser microdissection are separate personnel, working at separate places/time zones, a crude method to designate the region of interest can be followed:</em><br/><ol class="npwthree"><li><em>Image the Nissl-stained forebrain sections mounted on the regular plus slides, 1 for every 10 sections mounted on PEN slides.</em> </li><li><em>Copy and paste each image as a separate slide in Microsoft PowerPoint.</em> </li><li><em>Carefully trace the region of interest, using PowerPoint tools, as shown in Figure 3D.</em> </li><li><em>Using these representative tracings, successfully perform LCM on the related PEN slides having the respective serial sections.</em> </li></ol></li><li>If not already done, stain sections on the PEN slides with Nissl stain, as described in the previous section, just prior to microdissection.</li><li>Dry the slides post-staining, in the desiccator.</li><li>Place PEN slides upside down in the stack-slide holder.</li><li>Create a sample overview under 1.25x lens, using LED/halogen illumination.</li><li>Mount 0.5 ml RNase-free, DNase-free collection tubes in the LMD tube holder. Add 20-40 Î¼l of a mixture of RLT buffer and 2-mercaptoethanol to the cap of each tube (10 Î¼l 2-mercaptoethanol in 1 ml RLT buffer from Qiagen RNeasy Plus Micro Kit).</li><li>Set the LMD software to Transmitted light-brightfield (TL-BF) mode.</li><li>Adjust software parameters to get suitable cutting including more accurate and less damage of tissue. Software parameters used in Roy <em>et al</em>. (2019): Draw shapes using âclose lineâ. Use the âMove + Cutâ mode for cutting respected tissue zones/cells. Use âFineâ X-Y-Z precision mode.</li><li>Guide the laser beam with mouse or touchscreen over your specific region of interest. The dissected tissue drops into the caps of collection tube by force of gravity without contamination. In Roy <em>et al</em>. (2019), ventricular zone of P0 hippocampal CA1 field (CA1 vz) was collected from each coronal hemisphere (Figure 4).<br/><em>Note: Please load a new collection tube every 2 h if longer cutting time needed.</em> </li><li>Once all sections of the first slide are laser-cut, move to the next slide and repeat Steps F9-F12. You can store the used slides for future inspection and documentation.<br/><em>Note: Depending on the size and density of the laser-dissected samples, you can decide whether to pool together samples obtained from each brain for RNA isolation and further processing. In Roy et al. (2019), our region of interest (ventricular zone of P0 mouse CA1 region) was both low in volume and in density. Hence, total RNA was isolated from LCM-enriched samples pooled across ~6 slides per genotype.</em> </li><li>After microdissection, carefully retrieve the collect tubes, close the cap upside down and vortex for 1 min.</li><li>Give the tubes a brief spin in the centrifuge in order to collect sample at the bottom of the tube. Store tubes at -80 Â°C for further analysis.<br/><em>Note: If more than one cell type or cell population is needed from one slide, collect one type of tissue from all sections at the same time and make sure that the cut tissue had been detached from the section before starting to collect others. Load the collection tube for the next cell/tissue type in a different hole to avoid possible tissue contamination.</em> </li><li>Store unfinished PEN slides (where further LCM is possible) in the vacuum desiccator with desiccant at room temperature up to 5 days.<br/><br/></li></ol></li><li>RNA isolation<br/><ol class="npwtwo"><li>Pool all the collected RLT buffer (with micro-dissected samples of each cell/tissue type of each biological replicate) into a new, RNase-free 1.7 ml tube.</li><li>Vortex the tubes for 30 s.</li><li>Add an equal amount of 70% ethanol (RNase-free).</li><li>If the total volume is &gt; 700 Î¼l, transfer ~700 Î¼l mixture each time to an RNeasy MinElute spin column and repeat this step using the same column, until the entire volume of mixture is used.<br/></li><li>Follow the instruction given in the manual of RNeasy Micro Kit to isolate RNA from the LCM-ed samples.<br/></li><li>Elute the RNA with 15 Î¼l RNase-free water contained RNase Inhibitor (1 U/Î¼l).<br/></li><li>Repeat Step H6. If necessary, use the same tube for RNA collection.<br/></li><li>Take 2 Î¼l RNA elution into 2-3 Î¼l RNase-free water for NanoDrop and RNA bioanalysis.</li><li>Proceed with measuring the concentration and purity of obtained RNA, as part of the data analysis.<br/></li></ol><em>Note: You may distribute each tube/genotype/sample into 3 technical replicates for further processing (for example, RNA sequencing).</em> </li></ol><p class="pptt" id="biaoti30862">### Data analysis</p><p class="ppzz"><strong>Measure RNA concentration and RIN score</strong> </p><ol class="npwtwo"><li>After RNA is isolated by the RNAeasy kit, measure the concentration of RNA per area of interest, per genotype, per biological replicate, using the NanoDrop.</li><li>Calculate RNA quality by measuring the RNA Integrity Number (RIN) as well as RNA concentration (more precise than NanoDrop) for each RNA sample, using the Agilent Bioanalyzer 6000 Pico Kit.<br/><em>Note: RNA sample of RIN &gt; 5.0 is considered usable (where maximum score can be 10.0).</em> </li><li>Follow the manual available with the Agilent Kit for the consecutive steps. Briefly, (<a href="https://www.agilent.com/cs/library/usermanuals/Public/G2938-0049_RNA6000Pico_QSG.pdf" target="_blank">https://www.agilent.com/cs/library/usermanuals/Public/G2938-0049_RNA6000Pico_QSG.pdf</a>):<br/><ol class="npwthree"><li>Thaw RNA ladder on ice (avoid excessive warmth).</li><li>Prepare RNA gel matrix prior to the experiment (maximum: 4 weeks) and store at 4 Â°C.</li><li>Prepare the gel-dye mixture (components available within the kit) such that it is used within a day.</li><li>Load the gel-dye mixture carefully into each well of a new, sterile RNA Pico chip, with the help of the priming station (Figures 5A-5B).<br/></li><li>Load RNA conditioning solution and RNA marker.<br/></li><li>Load the thawed RNA ladder and samples. Vortex the Chip using the IKA vortex mixer (Figure 5C) before placing the former in the Agilent Bioanalyzer equipment for analysis.<br/></li><li>Analyze RNA concentration and integrity using the Agilent software (Figure 5D).<br/><br/><img alt="" class="layerphoto" height="161" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228084550_9332.jpg" width="300"/><br/><strong>Figure 5. Components used to analyze isolated RNA.</strong> A. RNA Pico Chip, where the small quantity of RNA samples is loaded. B. Chip priming station, to be obtained separately to load the gel-dye mix into each sample-well of the Pico Chip. C. IKA vortex mixer is used to vortex the sample-loaded Chip before running it in the Agilent BioAnalyzer instrument. D. Representative image demonstrating different types of RNA integrity (modified from <a href="https://www.agilent.com/cs/library/applications/5989-1165EN.pdf" target="_blank">https://www.agilent.com/cs/library/applications/5989-1165EN.pdf</a>).<br/><br/></li></ol></li><li>With good quality of RNA thus obtained, proceed for different downstream analyses, as needed. These can include RNA sequencing (tissue, single cell), quantitative Real-Time polymerase chain reaction (RT-PCR), protein assays (Figure 4).<br/></li></ol><p class="ppzz"><em>Note: In our experience, RNA concentration of 1.5-7 ng/Âµl and integrity of &gt; 7.0 was sufficient to proceed to successful RNA sequencing (see Methods, Figure 4 and Figure supplements 1-3, Source files 1-2 in Roy et al., 2019).</em> </p><p class="pptt" id="biaoti30863">### Recipes</p><p class="ppzz"><em>Note: Use 100% RNase-free molecular grade water in the following recipes.</em> </p><ol class="npwtwo"><li>70% ethanol (100 ml)<br/><ol class="npwthree"><li>Add 70 ml RNase-free absolute (100%) ethanol and 30 ml water to make 100 ml of 70% ethanol</li><li>Store at -20 Â°C</li></ol></li><li>95% ethanol (100 ml)<br/><ol class="npwthree"><li>Add 95 ml RNase-free 100% ethanol and 5 ml water to make 100 ml of 95% ethanol</li><li>Store at -20 Â°C</li></ol></li><li>RNaseZap/70% ethanol (100 ml)<br/><ol class="npwthree"><li>Mix RNaseZap and 70% ethanol (Recipe 1) in 1:1 ratio. For 100 ml, measure 50 ml of RNaseZap and 50 ml of 70% ethanol into a clean, RNase-free wash bottle</li><li>Mix the two liquids by rocking the bottle sideways for 4-5 min</li></ol>Clean the cryostat, microtome blade and other tools with this mixture<br/></li><li>75% ethanol (freshly-prepared)<br/><ol class="npwthree"><li>Add 75 ml RNase-free 100% ethanol and 25 ml water to make 100 ml of 75% ethanol</li><li>Store at -20 Â°C</li></ol></li><li>Nissl stain for RNA research (50 ml)<br/><ol class="npwthree"><li>Measure 0.5 g cresyl violet powder in a 50 ml RNase-free tube</li><li>Add 100% ethanol (RNase-free), final concentration: 1% solution (weight/volume)</li><li>Shake the solution for a few hours and store at 4 Â°C, sealed air-tight and dark for up to 6 months. This solution should be prepared at least 3 days prior to usage. Aliquot as required and discard the used solution</li></ol></li></ol><p class="pptt" id="biaoti30864">### Acknowledgments</p><p class="ppzz">This work was performed using the funding sources National Institutes of Health 1R01NS099027 (KJM), Seattle Childrenâs Hydrocephalus Research Guild (KJM) and Hydrocephalus Association Innovator Award 2018 (KJM). This protocol was derived from the original research paper (Roy <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti30865">### Competing interests</p><p class="ppzz">The authors declare that no competing interests exist.</p><p class="pptt" id="biaoti30866">### Ethics</p><p class="ppzz">Animal experimentation: All animal experimentation was conducted in accordance with the guidelines laid down by the Institutional Animal Care and Use Committees (IACUC) of Seattle Childrenâs Research Institute, Seattle, WA, USA (protocol ID: IACUC00006; last day of triennial approval period: 11<sup>th</sup> July 2021).</p><p class="pptt" id="biaoti30867">### References</p><ol class="npwtwok"><li>Emmert-Buck, M. R., Bonner, R. F., Smith, P. D., Chuaqui, R. F., Zhuang, Z., Goldstein, S. R., Weiss, R. A. and Liotta, L. A. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8875945" target="_blank">Laser capture microdissection.</a> <em>Science</em> 274(5289): 998-1001.</li><li>Grundemann, J., Schlaudraff, F., Haeckel, O. and Liss, B. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18332041" target="_blank">Elevated alpha-synuclein mRNA levels in individual UV-laser-microdissected dopaminergic substantia nigra neurons in idiopathic Parkinson's disease.</a> <em>Nucleic Acids Res</em> 36(7): e38.</li><li>Levsky, J. M. and Singer, R. H. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12480334" target="_blank">Gene expression and the myth of the average cell.</a> <em>Trends Cell Biol</em> 13(1): 4-6.</li><li>Roy, A., Murphy, R. M., Deng, M., MacDonald, J. W., Bammler, T. K., Aldinger, K. A., Glass, I. A. and Millen, K. J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31094678" target="_blank">PI3K-Yap activity drives cortical gyrification and hydrocephalus in mice.</a> <em>eLife</em> 8: e45961.</li><li>Roy, A., Skibo, J., Kalume, F., Ni, J., Rankin, S., Lu, Y., Dobyns, W. B., Mills, G. B., Zhao, J. J., Baker, S. J. and Millen, K. J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26633882" target="_blank">Mouse models of human PIK3CA-related brain overgrowth have acutely treatable epilepsy.</a> <em>eLife</em> 4: e12703.<br/></li></ol>