<body>
            <h1>RNA Interactome Identification via RNA-BioID in Mouse Embryonic Fibroblasts</h1>
            <p><strong>Authors:</strong> Joyita Mukherjee, Mirita Franz-Wachtel, Boris MaÄek, Ralf-Peter Jansen</p>
            <h2>Abstract</h2>
            <p>Cytoplasmic localization of mRNAs is common to all organisms and serves the spatial expression of genes. Cis-acting RNA signals (mostly found in the mRNAâs 3'-UTR), called zipcodes recruit trans-acting RNA-binding proteins that facilitate the localization of the mRNA. UV-cross-linking or affinity purification has been applied to identify such proteins but suffer from the need for stable RNA-protein binding or direct contact of protein and RNA. To identify stably or transiently interacting proteins that directly or indirectly associated with the localization elements and the body of the mRNA, we developed an <em>in vivo</em> proximity labeling method we call RNA-BioID. In RNA-BioID, we tether a fusion of the BirA* biotin ligase and the MS2 coat protein (MCP) at the 3'-UTR of MS2-tagged Î²-actin mRNA <em>in vivo</em>. Exposing BirA* expressing cells to biotin in the media and induces biotinylation of Î²-actin mRNA-associated proteins that can be isolated with streptavidin beads. This technique allowed us to identify by mass spectrometry analysis the Î²-actin mRNA 3'-UTR-interacting proteome in fibroblasts. The protocol can be useful to identify the interacting proteome of any mRNA in mammalian cells.</p>
            <p><strong>Keywords:</strong> RNA-BioID, Proximity labeling, Biotinylation, RNA-binding protein, RNA tagging</p>
            <h2>Background</h2><p>Localization of mRNAs to specific subcellular sites is a widespread phenomenon and has been observed from bacteria to humans (Bovaird <em>et al.</em>, 2018; Fei and Sharma, 2018). Diverse human pathologies of the neural system are linked to defects in mRNA localization (Tolino <em>et al.</em>, 2012; Bovaird <em>et al.</em>, 2018). Additionally, this mechanism is essential for developmental processes including mesoderm formation in the clawfrog <em>Xenopus laevis</em> or the determination of the embryonic body axes of the fruit fly <em>Drosophila melanogaster</em> (Jansen, 2001). To achieve intracellular asymmetry of transcripts, mRNAs are incorporated into motor-protein containing particles that can move along the cytoskeleton to distinct cellular sites, where the transcripts are locally translated (Marchand <em>et al.</em>, 2012; Buxbaum <em>et al.</em>, 2015). Such particles containing mRNA and RNA-binding proteins (RBPs) are beginning to assemble during transcription and undergo several changes upon maturation in the cytoplasm (Holt <em>et al.</em>, 2019). Localized mRNAs differ from non-localized ones by containing cis-acting motifs termed localization elements or zip codes that are recognized by specific RBPs (Bovaird <em>et al.</em>, 2018). Zipcodes can be positioned in the coding sequence but are mainly found in the 3' untranslated region (3'-UTR; Bovaird <em>et al.</em>, 2018).<br/><br/>A well-studied example is the 54-nt long zip code in the 3'-UTR of Î²-actin mRNA that is essential for targeting the mRNA to the leading edge of fibroblasts as well as to growth cones and dendrites in neurons (Ross <em>et al.</em>, 1997; Eom <em>et al.</em>, 2003; Buxbaum <em>et al.</em>, 2015). The zip code is recognized by the RBP ZBP1 (aka IGF1 or IGF2BP1) that is not only essential for the localization of Î²-actin mRNA (Ross <em>et al.</em>, 1997; Oleynikov and Singer, 2003) but also represses the translation of the mRNA until it has arrived at the target site (HÃ¼ttelmaier <em>et al.</em>, 2005). However, additional RBPs and supplementary factors are involved in splicing, translational regulation, stabilization, transport, and decay of Î²-actin mRNA.<br/><br/>Obtaining a complete set of the proteins that associate with any mRNA remains challenging, despite a large range of high-throughput methods available so far that include diverse pulldown approaches as well as cross-linking and immunoprecipitation (CLIP) (for an overview see Ramanathan <em>et al.</em>, 2019). A specific challenge that remains is the identification of transient interactors that are not directly contacting the RNA. To solve this, we recently introduced a proximity biotinylation method-based on tethering the BioID biotin ligase BirA (BirA*) via MS2 coat binding protein (MCP) at the 3'-UTR of MS2-tagged Î²-actin mRNA (Park <em>et al.</em>, 2014). In the presence of biotin, BirA* generates AMP-biotin (âactivated biotinâ), which reacts with accessible lysine residues in its vicinity (10-20 nm). After lysis, biotinylated proteins can be isolated via streptavidin affinity purification and identified using standard mass spectrometry techniques. The RNA-BioID method can in principle be applied to any MS2-tagged mRNA and allows the identification of its associated proteins as well as probing its environment by indirect or transient interactions.</p>
        </body>

<p class="pptt" id="biaoti30870">## Materials and Reagents</p><ol class="npwtwok"><li>Beaker (500 ml, 1 L)</li><li>50 ml conical tube</li><li>21 G needle</li><li>T175 flasks (Corning, catalog number: CLS431080)</li><li>10 cm culture dishes (Corning, catalog number: 430167)</li><li>6-well dishes (Corning, catalog number: CLS3516)</li><li>FACS tubes (Falcon, catalog number: 352235)</li><li>Reprosil-Pur C18-AQ, 1.9 Î¼m resin (Dr. Maisch GmbH)</li><li>25 cm fused silica emitters with an inner diameter of 75 Î¼m (DNU-MS GbR)</li><li>Bench cleaner (terralin<sup>Â®</sup> PAA, catalog number: 126203, store at room temperature)</li><li>1.5 ml safe seal pyrogen-free tubes (Sarstedt, catalog number: 72.706.201)</li><li>0.45 Î¼m filter (Sartorius, catalog number: 10109180)</li><li>pH strip 0.0-6.0; 6.0-10.0 (MN, Part No: 1138922; 1178895)</li><li>PVDF membrane (GE Healthcare Amersham<sup>TM</sup> Hybond<sup>TM</sup>-P Membranes, catalog number:10471085, store at room temperature)</li><li>Phage-ubc-nls-2xmcp-egfp-BirA* (Addgene Plasmid, catalog number: 131132) or phage-ubc-nls-2xmcp-egfp-cMyc BirA* (Addgene Plasmid, catalog number: 131133) or phage-ubc-nls-2xmcp-mCherry-BirA* (Addgene Plasmid, catalog number: 131136)</li><li>pCEP4-tat (Addgene Plasmid, catalog number: 22502)</li><li>pRSV-Rev (Addgene Plasmid, catalog number: 12253)</li><li>pMDLg/pRRE (Addgene Plasmid, catalog number: 12251)</li><li>pMD2.G (Addgene Plasmid, catalog number: 12259)</li><li>Hek293 FT cells (Thermo Scientific, catalog number: R70007)</li><li>Wildtype mouse embryonic fibroblasts (MEFs, <em>e.g.</em>, Park <em>et al.</em>, 2014)</li><li>Magnetic sepharose streptavidin Beads (GE, catalog number: 28985799, store at 4 Â°C)</li><li>MBS MEFs (MEFs with 24 MS2 binding sites (MBS) inserted in the 3â-UTR of Î²-actin gene loci; see Park <em>et al.</em>, 2014)</li><li>Fugene 6 (Promega, catalog number: E2691, store at 4 Â°C)</li><li>Trypsin (Sigma-Aldrich, catalog number: T4049, store at -20 Â°C)</li><li>Fetal calf serum (FBS; Sigma-Aldrich, catalog number: F4135, store at -20 Â°C)</li><li>Alkaline phosphatase-conjugated streptavidin</li><li>Trypsin, sequencing grade, modified, 0.5 Î¼g/Î¼l (Promega, catalog number: V5113)</li><li>LentiX concentrator (Takara, catalog number: 631231, store at 4 Â°C)</li><li>Lysyl endopeptidase Lys-C (Waco, catalog number: 129-02541), stocks: 0.5 Î¼g/Î¼l in H<sub>2</sub>O</li><li>OptiMEM (Thermo scientific, catalog number: 31985062, store at 4 Â°C)</li><li>DMEM high glucose media (Sigma-Aldrich, catalog number: D6429, store at 4 Â°C)</li><li>1x PBS (Sigma-Aldrich, catalog number: D6429, store at 4 Â°C)</li><li>Penicillin-Streptomycin solution (Sigma-Aldrich, catalog number: P4333, store at -20 Â°C)</li><li>Bleach powder (SchÃ¼lke perform<sup>Â®</sup> disinfectant, catalog number: 107912, store at room temperature)</li><li>Hand disinfectant (STERILLIUM Virugard solution, catalog number: 126064, store at room temperature)</li><li>Biotin (Sigma-Aldrich, catalog number: B4501, store at 4 Â°C)</li><li>Complete protease inhibitor, EDTA free (Roche, catalog number: 11697498001, store at 4 Â°C)</li><li>5x Roti Quant Bradford reagent (Roth, catalog number: K015.3, store at 4 Â°C)</li><li>SDS (sodium dodecyl sulfate) (Roth, catalog number: CN30.1, store at room temperature)</li><li>Bromophenol blue (Roth, catalog number: A512.1, store at room temperature)</li><li>Glycerol (Sigma-Aldrich, catalog number: G6279, store at room temperature)</li><li>Nu-PAGE SDS-PAGE gel 4-12% (Invitrogen, catalog number: WG1402BOX, store at 4 Â°C)</li><li>NuPAGE MOPS SDS Running Buffer (20x) (Invitrogen, catalog number: NP0001, store at 4 Â°C)</li><li>Î²-mercaptoethanol (Roth, catalog number: 42273, store at room temperature)</li><li>p-Nitrotetrazoliumblauchlorid (NBT) (Roth, catalog number: 4421.2, store at -20 Â°C)</li><li>5-Bromo-4-chloro-3-indolyl-phosphate Disodium Salt (BCIP) (Roth, catalog number: A155.2, store at -20 Â°C)</li><li>Ammonium Bicarbonate (Sigma-Aldrich, catalog number: 09830, store at room temperature)</li><li>NuPage SDS-PAGE gels (Thermo scientific, catalog number: NP0315BOX, store at 4 Â°C)</li><li>PHOENIX Peptide Cleanup Kit (PreOmics, catalog number: P.O.00023)</li><li>Dithiothreitol (VWR, catalog number: 0281-5G, store at room temperature)</li><li>Urea (Merck, catalog number: 1.08488.1000, store at room temperature)</li><li>Thiourea (Merck, catalog number: 1.07979.0250, store at room temperature)</li><li>Iodoacetamide (Merck, catalog number: 8.04744.0025, store at room temperature)</li><li>Acetonitrile (Merck, catalog number: 1.00030.2500)</li><li>Formic acid (Sigma-Aldrich, catalog number: 33015-500ML)</li><li>Acetic acid (Roth, catalog number: 3738.5)</li><li>Trifluoracetic acid (TFA) (Merck, catalog number: 1.08262.0100)</li><li>NaCl</li><li>MgCl<sub>2</sub> </li><li>Tris-Cl</li><li>TBS</li><li>Tween 20</li><li>Deoxycholate</li><li>HEPES</li><li>EDTA</li><li>LiCl</li><li>Complete media (500 ml) (see Recipes)</li><li>Serum-free media (100 ml) (see Recipes)</li><li>Biotin Media (100 ml) (see Recipes)</li><li>50 mM Biotin solution (see Recipes)</li><li>BCIP (5-bromo-4-chloro-3-indolyl phosphate) (see Recipes)</li><li>NBT (Nitro blue tetrazolium) (see Recipes)</li><li>AP (Alkaline phosphatase) substrate buffer (see Recipes)</li><li>1x TBS (Tris-buffered saline) (see Recipes)</li><li>1x TBST (Tris-buffered saline-Tween) (see Recipes)</li><li>Lysis Buffer (see Recipes)</li><li>Wash Buffer 1 (see Recipes)</li><li>Wash Buffer 2 (see Recipes)</li><li>Wash Buffer 3 (see Recipes)</li><li>Wash Buffer 4 (see Recipes)</li><li>20 mM ABC (Ammonium bicarbonate) buffer (see Recipes)</li><li>2x SDS (Sodium dodecyl sulfate) sample buffer (see Recipes)</li><li>Denaturation buffer (see Recipes)</li><li>Reduction buffer (see Recipes)</li><li>Alkylation buffer (see Recipes)</li><li>Solvent A* (see Recipes)</li><li>Solvent A (see Recipes)</li><li>HPLC solvent A (see Recipes)</li><li>HPLC solvent B (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30871">### Equipment</p><ol class="npwtwok"><li>37 Â°C water bath (Fisher Scientific, Isotemp 205 Water Bath)</li><li>Cell culture incubator, set to 5% CO<sub>2 </sub>(Thermo Scientific, Heracell<sup>TM</sup> 150i CO<sub>2</sub> Incubators)</li><li>Centrifuge (Eppendorf, model: 5430)</li><li>Cell sorter (BD, model: FACS Aria I)</li><li>End-to-end rotator (Heidolph, model: REAX 2)</li><li>Thermomixer (Eppendorf, catalog number: 5350 000.013)</li><li>Magnetic stand (Bio-Rad, catalog number: 1614916)</li><li>Centrifuge (Heraeus Fresco 17, Thermo Fisher Scientific, catalog number: 75002420)</li><li>Western blot SDS-PAGE running apparatus (Bio-Rad, Mini Trans-Blot<sup>Â®</sup> Cell)</li><li>Western blot SDS-PAGE wet transfer apparatus (Bio-Rad, Mini Trans-Blot Module, catalog number: 1703935)</li><li>Concentrator plus vacuum centrifuge (Eppendorf, catalog number: 5305 000.304)</li><li>EASY-nLC 1200 (Thermo Fisher Scientific) or any (U) HPLC system compatible with the MS system of choice</li><li>LTQ Orbitrap or Q Exactive mass spectrometer (Thermo Fisher Scientific) or any state-of-the-art high-resolution mass spectrometer.</li><li>Fluorescence microscope (Zeiss CellObserver, 40x, 1.4 NA oil immersion objective, GFP filterset 38HE)<br/></li></ol><p class="pptt" id="biaoti30872">### Software</p><ol class="npwtwok"><li>MaxQuant software suite (<a href="http://www.maxquant.org/" target="_blank">www.maxquant.org</a>, Max-Planck-Institute of Biochemistry, Martinsried), or other programs that can be used for downstream analysis</li><li>Perseus (<a href="http://www.coxdocs.org/doku.php?id=perseus:common:download_and_installation" target="_blank">http://www.coxdocs.org/doku.php?id=perseus:common:download_and_installation</a>, Max-Planck-Institute of Biochemistry, Martinsried)<br/></li></ol><p class="pptt" id="biaoti30873">### Procedure</p><p class="ppzz">A flowchart demonstrating the general outline of the experimental procedure is shown in Figure 1.<br/><br/><img alt="" class="layerphoto" height="204" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228232906_4499.jpg" width="300"/><br/><strong>Figure 1. Flowchart of the RNA-BioID procedure<br/><br/></strong> </p><ol class="npwone"><li>Generating stable cell lines expressing an MCP-GFP-BirA* fusion protein<ol class="npwtwo"><li>The MCP-GFP-BirA* gene fusion under control of the UBC promoter was integrated into a third-generation lentiviral vector and is available in three different versions via Addgene (Addgene 131132/131133/131136). The three different versions can be used in similar ways.</li><li>For transduction, use the MCP-GFP-BirA* plasmid Gag/Pol in combination with Env and VSV-G plasmids to assemble and generate the complete viral particles in Hek293FT cells.</li><li>On the day before transfection, seed 4 x 10<sup>6</sup> Hek293FT cells in complete growth media in a 10 cm dish.</li><li>On the day of transfection, set up two 1.5 ml centrifuge tubes, each containing 500 Î¼l of OptiMEM media.</li><li>To one tube, add the corresponding amounts of the following plasmids:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>MCP-GFP-BirA* backbone plasmid:<br/></td><td>20 Âµg<br/></td></tr><tr><td>pCEP4-tat<br/></td><td>1 Î¼g<br/></td></tr><tr><td>pRSV-Rev <br/></td><td>1 Î¼g<br/></td></tr><tr><td>pMDLg/pRRE<br/></td><td>1 Î¼g<br/></td></tr><tr><td>pMD2.G (Insert name VSV-G)<br/></td><td>2 Î¼g<br/></td></tr><tr><td>In total<br/></td><td>25 Î¼g of plasmid DNA<br/></td></tr></tbody></table></li><li>To the other OptiMEM-containing centrifuge tube, add 75 Âµl Fugene HD transfection reagent (3 Î¼l per Î¼g plasmid).</li><li>Incubate both tubes separately for 5 min at room temperature.</li><li>Add the OptiMEM media containing plasmid DNA to the tube containing the OptiMEM-Fugene HD mix, mix briefly by pipetting 2-3 times, and incubate for at least 25 min (up to 45 min) at room temperature. This results in the formation of the DNA-Fugene complex.</li><li>Exchange the media of the 10 cm dish containing Hek293FT cells with fresh 10 ml of complete growth media.</li><li>Add the DNA-Fugene complex on top of the cells in a dropwise manner. Swirl the dish gently to mix the Fugene-DNA mix with the medium and incubate the dishes at 37 Â°C for at least 16 h.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>For biosafety reasons, keep the dishes on a tray inside the incubator to avoid spilling of the media in the incubator due to handling errors.</em> </li><li><em>At this stage, the lentivirus is being produced, so strictly follow the biosafety class II regulations. After the 16 h incubation period, all dishes and pipettes that have and will come in contact with virus-containing solutions have to be treated with bleach prior to disposal. Always wear a lab coat and wipe the entire hood with ethanol. When you leave the working space treat your gloves with bleach as well.</em> </li></ol></li><li>Before the end of the 16 h incubation, prepare 500 ml of 10% bleach and keep it in a large beaker in the tissue culture hood for treating surfaces (prepare fresh 19% bleach every day, it remains effective only for 24 h).</li><li>Once the bleach solution is ready under the hood, carefully remove 8 ml of the media from the 10 cm dish and transfer it to a 50 ml conical tube. Keep it at 4 Â°C overnight.</li><li>Add 8 ml of fresh complete growth media to the dish and return it to the incubator for another 24 h.</li><li>At this time, seed wildtype (WT) MEFs and MBS MEFs (3 x 10<sup>6</sup>/10 cm dish) and incubate at 37 Â°C in a cell culture incubator (5% CO<sub>2</sub>).</li><li>Repeat Steps A11-A13 twice more during the consecutive days. Collect the media from each day and pool it in the same 50 ml conical tube from Step A12, keep at 4 Â°C. After collecting all supernatants, spin the conical tube for 10 min at 500 <em>x g</em> and 4 Â°C to pellet large debris.</li><li>Filter the supernatant from this centrifugation through a 0.45 Î¼m filter.</li><li>Concentrate the lentiviral particles in the supernatant with a LentiX concentrator according to manufacturerâs instructions.</li><li>After concentrating the viral particles, add 1 ml of serum-free media (Recipe 2) and carefully resuspend the pellet.</li><li>Remove the dishes with WT MEFs and MBS-MEFs from the incubator for transduction, replace the old media with fresh complete medium (Recipe 1) and add 100 Î¼l of the viral particle solution per 10 cm dish with 10 ml of media. Store the rest of the virus particles at -80 Â°C. Place the MEFs back into the cell culture incubator and leave for 24 h.</li><li>Check the viral transduction efficiency under a fluorescence microscope (filter setting for GFP). If no GFP fluorescence is visible, repeat Step A19 with another aliquot of the stored viral particles.</li><li>Fluorescent cells should stably express the MCP-GFP-BirA* fusion protein.</li><li>On the last day of collection, bleach all the dishes that came in contact with the viral particles.<br/><br/></li></ol></li><li>Isolating cells expressing a functional MCP-GFP-BirA* fusion protein<ol class="npwtwo"><li>After generation of stable cell lines (WT-BirA*: WT MEFs expressing MCP-GFP-BirA*; MBS-BirA*: MEFs with 24x MBS in the 3'-UTR of Î²-actin and expressing MCP-GFP-BirA*), sort positive, GFP-expressing cells by fluorescence-activated cell sorting against GFP, in FACS (fluorescence-activated cell sorting) tube containing 1 ml of complete growth media. This can be done by a core facility at your institution.</li><li>After sorting, spin down the cells at 500 <em>x g</em> for 5 min at room temperature. Resuspend and, depending on the number of positive cells identified in cell sorting, culture them either on 10 cm dish or bigger dishes (10,000 cells/each 10 cm dish).</li><li>When at least 10,000 cells have been isolated, test for the GFP expression levels in both transduced cell types (WT-BirA*/MBS-BirA*). For checking GFP expression, prepare cell lysates in the lysis buffer (Recipe 10). Add 3 packed cell volume (PCV) of lysis buffer and and pass 10-12 times through a 21 G needle. Centrifuge the lysate at 12,000 <em>x g</em> for 10 min at 4 Â°C to remove cell debris. Take out the soluble fraction and keep in a new 1.5 ml centrifuge tube at 4 Â°C. Use at least 10 Âµg of total protein for a Western blot against GFP and a housekeeping gene (<em>e.g.</em>, GAPDH). Use untransfected WT MEFs as negative control and GAPDH as loading control. This step is important to check whether the full NLS-HA-2XMCP-eGFP-BirA* construct is expressing giving a band at 91.4 kDa (Figure 2A).</li><li>To check for the activity of BirA* and biotinylation efficiency, use a 6-well dish. Seed duplicates of 3 x 10<sup>4</sup> cells of WT untransfected, WT-BirA* and MBS-BirA* and culture them in complete media for 24 h. On the next day, exchange the media. Use complete media containing 50 Î¼M of biotin with only on one well of each duplicate. To the second set of duplicates just exchange the media with new complete media without biotin.</li><li>Culture the cells with the new media (with or without biotin) for at least 6 h and up to 24 h. Biotinylation efficiency might not increase after 6 h, depending on the cell line.</li><li>At the end of the incubation, wash the wells of the 6-well dishes two times with cold PBS and finally isolate the cells by adding trypsin.</li><li>Isolate the cells by adding 500 Î¼l/well of warm trypsin solution and incubate the dish at 37 Â°C for 2-3 min. After 2 min, check under a cell culture microscope (at 10x magnification) if all cells are completely detached. If not, leave the cells in the incubator but do not incubate longer than 5 min.</li><li>Remove the solution with the cells and transfer them to a 15 ml conical tube. Add 6 ml of complete media containing FBS to deactivate the trypsin.</li><li>Spin down the cells for 5 min at 500 <em>x g</em> at room temperature.</li><li>Remove the media and add 1 ml of cold 1x PBS, resuspend the pellet with a 1 ml tip and pellet the cells again by centrifugation for 4 min at 500 <em>x g</em> at 4 Â°C.</li><li>Repeat Step B10 to completely remove the media.</li><li>Add 3 PCV lysis buffer containing 1x complete protease inhibitors to each pellet, resuspend cells and keep them on ice for 10 min. Lyse the cells by passing the pellet 10-15 times through a 21 G needle. After the lysis, spin the lysates for 10 min at 12,000 <em>x g</em> at 4 Â°C to remove the unbroken cells.</li><li>Determine the protein concentration (<em>e.g.</em>, by a Bradford assay following the manufacturerâs protocol).</li><li>Mix 10 Âµg of total protein with SDS-PAGE loading buffer.</li><li>Boil the mixture for 5 min at 95 Â°C and afterward spin down debris for 3 min at 12,000 <em>x g</em> at room temperature.</li><li>Load the supernatant from step 15 on a Nu-PAGE SDS-PAGE gel in 1x MOPS running buffer and run at 200 V until the front dye exits the gel.</li><li>Transfer the protein from the gel onto an activated PVDF membrane by semi-dry Western blotting (2.5 h at 250 mA).</li><li>After the transfer is complete, wash the PVDF membrane in a small box with TBST (Recipe 9) twice for 5 min each at room temperature with rotation or tilting.</li><li>In order to probe for biotinylated proteins, incubate with alkaline phosphatase-conjugated streptavidin.</li><li>Incubate the membrane with AP substrate buffer (Recipe 7) for 5 min at room temperature.</li><li>Remove AP substrate buffer. In a 15 ml conical tube, mix 5 ml of freshly made AP substrate buffer together with NBT (Recipe 6) (final concentration 50 Î¼g/ml) and BCIP (Recipe 5) (final concentration 50 Î¼g/ml). Pour the mixture on top of the membrane and protect it from light.</li><li>Incubate the membrane on a shaker protected from light and check at 1 min intervals until you can see purple bands from the biotinylated proteins (Figure 2B).</li><li>Once you tested the biotinylation is working properly, continue with large scale biotinylation experiment for purification of biotinylated proteins and mass spectrometry.<br/><br/></li></ol></li><li>RNA-BioID<ol class="npwtwo"><li>To capture and analyze biotinylated proteins in RNA-BioID, you need a confluent T175 flask of cells (2-3 x 10<sup>7</sup>) for each round of mass spectrometry analysis.</li><li>On the day before the streptavidin pulldown, confluency should have reached 80%. Exchange media of one T175 flask, containing WT-BirA* or MBS-BirA* cells biotin-containing medium and culture for 16-24 h.</li><li>Wash the cells twice with PBS at room temperature to dispose of dead cells and serum-containing medium. Trypsinize for 2-3 min in the incubator. After checking cell detachment under a microscope, transfer cells into a conical tube and add 5 ml of serum-containing media to inactivate the trypsin.</li><li>Collect cells by centrifugation for 5 min at 500 <em>x g</em> for at 4 Â°C.</li><li>Wash cells twice with ice-cold PBS to completely remove the media.</li><li>Add 3 PCV of lysis buffer (including Complete<sup>TM</sup> protease inhibitor mix) to each pellet and resuspend.</li><li>Lyse cells, collect lysate and determine protein concentration as mentioned above (Steps B12-B13).</li><li>For the isolation of biotinylated proteins, use at least 1 mg of total protein per lysate and 300 Î¼l of magnetic streptavidin beads per 1 mg of protein.</li><li>To prepare the beads, mix the bead solution in the original vendorâs tube by quick vortexing, remove 200 Î¼l of bead slurry to a fresh 1.5 ml tube and separate the beads from its storage buffer by capturing the beads on a magnetic stand for 5 min. Remove storage solution and wash the beads three times with 1 ml of lysis buffer. Rotate tubes with the bead slurry on an end-to-end rotator for 5 min.</li><li>Collect the beads on a magnetic stand and discard the wash buffer. Add 1 mg of total cell lysate onto the beads, remove from magnetic stand and leave mixture overnight with end to end rotation at 4 Â°C.</li><li>On the next day, collect the beads on a magnetic stand and wash twice with 1 ml of wash buffer 1 (Recipe 11), once with 1 ml of wash buffer 2 (Recipe 12), once with 1 ml of wash buffer 3 (Recipe 13) and once with 1 ml of wash buffer 4 (Recipe 14), each time by end-to-end rotation for 5 min at room temperature.</li><li>During the final washing with 1 ml of wash buffer 4, take 100 Î¼l into a new 1.5 ml centrifuge tube and collect the beads via a magnetic stand and store the beads on ice until Step C14 to check enrichment of proteins with silver staining. Collect the beads from the remaining 900 Î¼l on a magnetic stand.</li><li>Wash the beads 3 times with 50 mM of ammonium bicarbonate buffer each for 5 min at room temperature with rotation. Collect the beads and proceed immediately to Procedure D or store beads at -80 Â°C (safe stopping point).</li><li>From the remaining beads from step C12, add 50 Î¼l of 1x SDS sample buffer (Recipe 16), boil the mixture for 5 min at 95 Â°C and afterward spin for 1 min at 12,000 <em>x g</em> at room temperature.</li><li>Keep the centrifuge tube on a magnetic stand for 2 min at room temperature and take out the supernatant to a new 1.5 ml centrifuge tube. Load 20 Î¼l of the supernatant on a Nu-PAGE SDS-PAGE gel in 1x MOPS running buffer and run at 200 V until the front dye exits the gel.</li><li>Perform silver staining (Aboulaich, 2011) on this gel to compare proteins from WT-BirA* (shows proteins which are background biotinylated like mitochondria carboxylases and proteins which are unspecifically bound to the beads) and MBS-BirA* cells after treatment with biotin (Figure 2C).<br/><br/></li></ol></li><li>On-bead tryptic digestion<ol class="npwtwo"><li>Add an appropriate volume of denaturation buffer (Recipe 17) to the beads (usually 30-80 Î¼l), such that they are well covered and can float in the suspension during incubation.</li><li>Add reduction buffer (Recipe 18) to a final concentration of 1 mM, incubate for 1 h at room temperature in a Thermomixer at 1,000 rpm.</li><li>Add alkylation buffer (Recipe 19) to a final concentration of 5.5 mM, incubate for 1 h at room temperature in a Thermomixer at 1,000 rpm.</li><li>Add 1 Î¼g LysC for predigestion and check pH on a pH 6-10 pH strip. The pH needs to be around pH 8, otherwise, adjust with 1 M Tris-base pH 8. Incubate for 3 h at room temperature in the Thermomixer at 1,000 rpm.</li><li>Add 4 volumes of 20 mM ammonium bicarbonate buffer to dilute the urea in the buffer.</li><li>Add 1 Î¼g trypsin for digestion and check pH on a pH 6-10 pH strip (pH should be 8), incubate overnight at room temperature in a Thermomixer at 1,000 rpm.</li><li>Separate beads and supernatant by using a magnetic stand and transfer the supernatant to a new tube.</li><li>Acidify samples with TFA to pH â¤ 2.<br/><br/></li></ol></li><li>Peptide clean-up<ol class="npwtwo"><li>Load sample onto a PHOENIX Peptide Clean-up Kit device.</li><li>Perform clean-up procedure by following the kit protocol (washing and elution steps).</li><li>After elution, reduce the sample volume to â¤ 9 Î¼l in a vacuum concentrator (30-40 min at 30 Â°C).</li><li>Add 1 Î¼l solvent A* (Recipe 20) and adjust the sample volume to 10 Î¼l with solvent A (Recipe 21).<br/><br/></li></ol></li><li>Mass spectrometry analysisF.Mass spectrometry analysis<ol class="npwtwo"><li>Peptide samples are analyzed using online nanoflow liquid chromatography tandem mass spectrometry (LC-MS/MS). Note that this is a routine analysis in proteomics that can be done on any high-performance liquid chromatography system coupled to a state-of-the-art high-resolution mass spectrometer.</li><li>Here, nanoLC-MS/MS analyses are done on an EASY-nLC 1200 system connected to an LTQ-Orbitrap Elite through an electrospray Ionization source.</li><li>Pack a 25 cm long 75 Î¼m-inner diameter analytical column with reversed-phase C18-AQ Reprosil-Pur 1.9 Î¼m particles at a constant pressure of 600 bar, and cut the column to a final length of 20 cm.</li><li>Load peptides directly onto the column at a flow rate of 500 nl/min.</li><li>Reduce flow rate to 200 nl/min after loading, and separate peptides with a segmented linear gradient of HPLC solvent B (Recipe 23) (5-33-50-90%) for 87 min.</li><li>Operate the mass spectrometer in the positive ion mode, with the following acquisition cycle: a full scan is measured using the Orbitrap cell at a resolution R 120,000 followed by MS/MS of the top 15 most intense peptide ions in the linear ion trap using collision-induced dissociation (CID). Acquire precursor ions in the mass range from m/z 300 to 2,000 and use the target values 1E6 charges for the full scans (Orbitrap analyzer), and 5E3 charges for the collision-induced dissociation (CID) in the Linear Ion Trap. Sequenced precursor masses need to be excluded from further selection for 60 s.<br/><br/><img alt="" class="layerphoto" height="160" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191228/20191228232929_9699.jpg" width="300"/><br/><strong>Figure 2. Functional validation of the integrated BirA* fusion construct.</strong> A. Expression check of full-length BirA* construct via Western blot. The full-length fusion protein from the nls-HA-2x mcp-egfp-BirA* transgene, in total 91.4 KDa, is detectable by Western blot against the GFP part only in MBS MEF cells. GAPDH can be used as a negative control protein since it can be detected in lysates from WT and MBS-BirA* MEFs. B. Testing for BirA* activity in MEFs using AP coupled streptavidin. After incubation of MEFs with 50 Î¼M biotin for 24 h, cells were lysed and 10 Î¼g of total cell lysates from WT and MBS-BirA* MEF cells were loaded per lane. Biotinylated proteins were detected in a colorimetric assay by Western blot using AP-streptavidin. C. A pattern of biotinylated proteins in RNA-BioID obtained after purification via streptavidin-coupled magnetic beads. 1/10<sup>th</sup> of the amount of magnetic beads that are normally used for mass spectrometry analysis were boiled in 100 Î¼l of SDS sample buffer. Twenty microliter of these were loaded on a 10% SDS-PAGE, separated and detected by silver staining. The number and intensity of protein bands from the lysate of MBS-BirA* MEFs are much larger than that from WT-MEFs, indicating successful RNA-dependent biotinylation and capturing of associated proteins.<br/></li></ol></li></ol><p class="pptt" id="biaoti30874">### Recipes</p><ol class="npwtwok"><li>Complete media (500 ml)<br/>445 ml DMEM media<br/>50 ml FBS<br/>5 ml Penicillin-Streptomycin solution</li><li>Serum-free media (100 ml)<br/>99 ml DMEM media<br/>1 ml Penicillin-Streptomycin solution</li><li>Biotin Media (100 ml)<br/>89 ml DMEM media<br/>10 ml FBS<br/>1 ml Penicillin-Streptomycin solution<br/>50 ÂµM biotin</li><li>50 mM Biotin solution<br/>2.44 mg biotin<br/>10 ml of serum-free DMEM<br/>Pipetting is required to dissolve biotin completely. Sterilize by passing through a 0.22-Î¼m syringe-driven filter unit<br/>Active up to 8 weeks at 4 Â°C</li><li>BCIP<br/>50 mg/ml BCIP<br/>High-purity H<sub>2</sub>O</li><li>NBT<br/>10 mg/ml NBT<br/>High-purity H<sub>2</sub>O</li><li>AP substrate buffer<br/>100 mM NaCl<br/>5 mM MgCl<sub>2</sub><br/>100 mM Tris-Cl pH 9.5</li><li>1x TBS<br/>50 mM Tris-Cl pH 7.5<br/>150 mM NaCl</li><li>1x TBST<br/>1x TBS<br/>0.1% Tween 20</li><li>Lysis Buffer<br/>50 mM Tris-Cl pH 7.5<br/>150 mM NaCl<br/>2.5 mM MgCl<sub>2</sub><br/>1 mM DTT<br/>1% Triton X-100<br/>1x proteinase inhibitor (Roche)</li><li>Wash Buffer 1<br/>2% SDS<br/>High-purity H<sub>2</sub>O</li><li>Wash Buffer 2<br/>0.1% deoxycholate<br/>1% Tween 20<br/>500 mM NaCl<br/>1 mM EDTA<br/>50 mM HEPES pH 7.5</li><li>Wash Buffer 3<br/>250 mM LiCl<br/>0.5% Tween 20<br/>0.5% deoxycholate<br/>1 mM EDTA<br/>10 mM Tris pH 8.1</li><li>Wash Buffer 4<br/>50 mM Tris pH 7.4<br/>50 mM NaCl</li><li>20 mM ABC buffer<br/>79.06 mg ammonium bicarbonate<br/>50 ml of RNase- and DNase-free water</li><li>2x SDS sample buffer<br/>100 mM Tris-Cl pH 6.8<br/>4% (w/v) SDS<br/>0.2% (w/v) bromophenol blue<br/>20% (v/v) glycerol<br/>200 mM Î²-mercaptoethanol<br/>Make 100 Î¼l aliquots and store at -20 Â°C</li><li>Denaturation buffer<br/>6 M urea<br/>2 M thiourea<br/>10 mM Tris buffer (pH 8.0)</li><li>Reduction buffer<br/>1 M dithiothreitol<br/>50 mM ammonium bicarbonate</li><li>Alkylation buffer<br/>550 mM iodoacetamide<br/>50 mM ammonium bicarbonate</li><li>Solvent A*<br/>2% acetonitrile<br/>1% formic acid</li><li>Solvent A<br/>0.5% acetic acid</li><li>HPLC solvent A<br/>0.1% formic acid<br/>Milli-Q water</li><li>HPLC solvent B<br/>0.1% formic acid<br/>80% ACN in Milli-Q water<br/></li></ol><p class="pptt" id="biaoti30875">### Acknowledgments</p><p class="ppzz">This protocol has been adapted from Mukherjee <em>et al.</em> (2019). We thank Jeff Chao (FMI, Basel) and JuliÃ©n Bethune (BZH, Heidelberg), for plasmids and cell lines. Silke Wahle (PCT TÃ¼bingen) is acknowledged for technical support. The project was funded as a project of the DFG Research Unit FOR2333 by a grant of the Deutsche Forschungsgemeinschaft (DFG JA696/11-1).</p><p class="pptt" id="biaoti30876">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti30877">### References</p><ol class="npwtwok"><li>Aboulaich, N. (2011). <a href="https://bio-protocol.org/bio101/e26" target="_blank">Silver staining.</a> <em>Bio-101</em>: e26.</li><li>Bovaird, S., Patel, D., Padilla, J. A. and LÃ©cuyer, E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30132838" target="_blank">Biological functions, regulatory mechanisms, and disease relevance of RNA localization pathways.</a> <em>FEBS Lett</em> 592(17): 2948-2972.</li><li>Buxbaum, A. R., Haimovich, G. and Singer, R. H. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25549890" target="_blank">In the right place at the right time: visualizing and understanding mRNA localization.</a> <em>Nat Rev Mol Cell Biol</em> 16(2): 95-109.</li><li>Eom, T., Antar, L. N., Singer, R. H. and Bassell, G. J. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14614102" target="_blank">Localization of a beta-actin messenger ribonucleoprotein complex with zipcode-binding protein modulates the density of dendritic filopodia and filopodial synapses.</a> <em>J Neurosci</em> 23(32): 10433-10444.</li><li>Fei, J. and Sharma, C. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30191804" target="_blank">RNA localization in bacteria.</a> <em>Microbiol Spectr</em> 6(5).</li><li>Holt, C. E., Martin, K. C. and Schuman, E. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31270476" target="_blank">Local translation in neurons: visualization and function.</a> <em>Nat Struct Mol Biol</em> 26(7): 557-566.</li><li>HÃ¼ttelmaier, S., Zenklusen, D., Lederer, M., Dictenberg, J., Lorenz, M., Meng, X., Bassell, G. J., Condeelis, J. and Singer, R. H. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16306994" target="_blank">Spatial regulation of Î²-actin translation by Src-dependent phosphorylation of ZBP1.</a> <em>Nature</em> 438(7067): 512-515.</li><li>Jansen, R. P. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11283722" target="_blank">mRNA localization: message on the move.</a> <em>Nat Rev Mol Cell Biol</em> 2(4): 247-256.</li><li>Marchand, V., Gaspar, I. and Ephrussi, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22278045" target="_blank">An intracellular transmission control protocol: assembly and transport of ribonucleoprotein complexes.</a> <em>Curr Opin Cell Biol</em> 24(2): 202-210.</li><li>Mukherjee, J., Hermesh, O., Eliscovich, C., Nalpas, N., Franz-Wachtel, M., MaÄek, B. and Jansen, R. P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31189591" target="_blank">Î²-Actin mRNA interactome mapping by proximity biotinylation.</a> <em>Proc Natl Acad Sci U S A</em> 116(26): 12863-12872.</li><li>Oleynikov, Y. and Singer, R. H. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12573215" target="_blank">Real-time visualization of ZBP1 association with beta-actin mRNA during transcription and localization.</a> <em>Curr Biol</em> 13(3): 199-207.</li><li>Park, H. Y., Lim, H., Yoon, Y. J., Follenzi, A., Nwokafor, C., Lopez-Jones, M., Meng, X. and Singer, R. H. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24458643" target="_blank">Visualization of dynamics of single endogenous mRNA labeled in live mouse.</a> <em>Science</em> 343(6169): 422-424.</li><li>Ramanathan, M., Porter, D. F. and Khavari, P. A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30804549" target="_blank">Methods to study RNA-protein interactions.</a> <em>Nat Methods</em> 16(3): 225-234.</li><li>Ross, A. F., Oleynikov, Y., Kislauskis, E. H., Taneja, K. L. and Singer, R. H. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9121465" target="_blank">Characterization of a Î²-actin mRNA zipcode-binding protein.</a> <em>Mol Cell Biol</em> 17(4): 2158-2165.</li><li>Tolino, M., KÃ¶hrmann, M. and Kiebler, M. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22708593" target="_blank">RNA-binding proteins involved in RNA localization and their implications in neuronal diseases.</a> <em>Eur J Neurosci</em> 35(12): 1818-1836.<br/></li></ol>