<body>
            <h1>Measurement of Chloroplastic NAD Kinase Activity and Whole Tissue NAD Kinase Assay</h1>
            <p><strong>Authors:</strong> Yuuma Ishikawa, Maki Kawai-Yamada, Shin-nosuke Hashida</p>
            <h2>Abstract</h2>
            <p>Nicotinamide adenine dinucleotide phosphate (NADP) synthesis requires nicotinamide adenine dinucleotide (NAD) kinase activity, substrate NAD and ATP. The NAD kinase responds to various environmental stimuli and its activity is regulated via various regulatory pathways, such as Ca<sup>2+</sup>-dependent and redox-dependent signals. Conventional <em>in vitro</em> NAD kinase assay has been useful to evaluate enzyme activity; however, recent reports revealed a dynamics of NADP pool (the sum of NADP<sup>+</sup> and NADPH) under fluctuating light condition, indicating that the rate of NADP synthesis is not always determined by NAD kinase activity. Here, we developed a novel method for the estimation of chloroplastic NAD kinase activity by quantifying the changes in the NADP amounts in response to illumination. As our approach does not involve protein extraction, it saves time (compared to the <em>in vitro</em> assay), thereby allowing for a sequence of assays, and provides several clues in the investigation of regulatory mechanisms behind NADP synthesis under various environmental conditions.</p>
            <p><strong>Keywords:</strong> NAD kinase, NADP, Chloroplasts, Enzyme, <em>In vitro</em></p>
            <h2>Background</h2><p>NADP (including the oxidized form NADP<sup>+</sup> and the reduced form NADPH) is known as one of the most fundamental molecules for cellular metabolic reactions. In nature, NADP<sup>+</sup> captures light-driven energy as electrons, via the photosynthetic electron transfer chain (PETC). Therefore, the level of available NADP<sup>+</sup> in photosynthesis is quite important for autotrophic growth (Hashida and Kawai-Yamada, 2019).<br/><br/>Plant NAD kinase activity has been evaluated in many reports. According to <em>in vitro</em> assay on <em>Arabidopsis thaliana</em> tissue extracts, whole NAD kinase activity in 2-week-old seedlings was estimated as 560 nmol/h/mg protein (Turner <em>et al.</em>, 2004). On the other hand, in 3-week-old rosette leaves of Col-0 it was calculated as 5.3 nmol/h/mg protein and 64 nmol/h/g FW (corresponding to approximately 12.1 nmol/h/mg protein) (Takahashi <em>et al.</em>, 2006 and 2009). This large difference might be from variations in the amount of RubisCO (the most abundant leaf protein) between juvenile and mature leaves. Furthermore, it has been known that NAD kinase activity is under the control of various environmental stimuli, such as light, pathogen infection, oxidative stress <em>etc.</em> (Berrin <em>et al.</em>, 2005). Beside transcriptional responses, several plant NAD kinases are known to be regulated via Ca<sup>2+</sup> and calmodulin (CaM) signaling, cellular redox states and phytochromes (Tezuka and Yamamoto,1975; Delumeau <em>et al.</em>, 2000; Hashida <em>et al.</em>, 2018; Dellâ Aglio <em>et al.</em>, 2019; Tai <em>et al.</em>, 2019). Therefore, robust, reproducible, and standardized methods are indispensable for the evaluation of NAD kinase activity.<br/><br/>In <em>Arabidopsis</em>, the <em>nadk2</em>, a loss-of-function mutant in the chloroplastic NAD kinase (At1g21640) (Chai <em>et al.</em>, 2005), lacked the light-responsive increase in NAD kinase activity (Hashida <em>et al.</em>, 2018), suggesting that the activity of the NADK2 isoform is highly dependent on the light condition. Here we developed a novel NAD kinase assay, especially focusing on the chloroplastic NAD kinase activity. Compared to conventional <em>in vitro</em> NAD kinase assay, our newly developed method can drastically save time and costs, and it enables the evaluation of chloroplastic NAD kinase activity, while being comparable to the conventional method.</p>
        </body>

<p class="pptt" id="biaoti30924">## Materials and Reagents</p><ol class="npwtwok"><li>Finntip Pipette Tips (Finntip Flex 10, 200, and 1,000)</li><li>1.5 ml microcentrifuge tubes (VIOLAMO, F 1.5, catalog number: 1-1600-01)</li><li>12 Well Multidish (Thermo Fisher Scientific, catalog number: 130185)</li><li>Half area 96-Well microplate (Greiner Bio-One, catalog number: 675075)</li><li>Zeba<sup>TM</sup> Spin Desalting Columns, 40K MWCO, 0.5 ml (Pieces, catalog number: 87766)</li><li><em>Arabidopsis thaliana</em> wild-type plants (Ecotype: Col-0)</li><li>Liquid nitrogen</li><li>Tris (hydroxymethyl) aminomethane (HEPES) (Nacalai Tesque, catalog number: GR35406-91)</li><li>MgCl<sub>2</sub> (Nacalai Tesque, catalog number: SP20937)</li><li>CaCl<sub>2</sub> (Nacalai Tesque, catalog number: SP08894)</li><li>PMSF (Nacalai Tesque, catalog number: SP96297)</li><li>cOmplete<sup>TM</sup>, Mini, Protease Inhibitor Cocktail (Roche, catalog number: 4693159001)</li><li>Bio-Rad Protein Assay Kit (Bio-Rad, catalog number: 5000001ja)</li><li>NADP/NADPH-Glo<sup>TM</sup> Assay Kit (Promega, catalog number: G9081 or G9082)</li><li>NaH<sub>2</sub>PO<sub>4</sub> (Nacalai Tesque, catalog number: SP31737)</li><li>ATP (Nacalai Tesque, catalog number: SP08886)</li><li>NAD<sup>+</sup> (Nacalai Tesque, catalog number: GR24338)</li><li>NADP<sup>+</sup> (Nacalai Tesque, catalog number: EP24336)</li><li>NADPH (Nacalai Tesque, catalog number: CP24340)</li><li>HCl (Nacalai Tesque, catalog number: GR18322)</li><li>NaOH (Nacalai Tesque, catalog number: SP06338)</li><li>Protein extraction buffer (see Recipes)</li><li>Reaction buffer (see Recipes)</li><li>Luciferin detection reagent (see Recipes)</li><li>NADP/NADPH Glo detection reagent (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30925">### Equipment</p><ol class="npwtwok"><li>Pipettes (Thermo Fisher Scientific, Finnpipette<sup>TM</sup> F2)</li><li>Conventional cork borer (4.0 mm diameter)</li><li>Homogenizer pestle (VIOLAMO, F-1.5, catalog number: 1-2955-02)</li><li>Refrigerated microcentrifuge (TOMY, model: MX305)</li><li>Ultrasonic disruptor (TOMY, model: UR-21P)</li><li>Absorption spectrometer (Thermo Fisher Scientific, Multiskan<sup>TM</sup> GO)</li><li>Luminescence detector (Promega, model: GloMax<sup>TM</sup> Navigator System GM2000)<br/></li></ol><p class="pptt" id="biaoti30926">### Procedure</p><ol class="npwone"><li>Setting up a conventional <em>in vitro</em> NAD kinase assay<ol class="npwtwo"><li>Protein extraction<ol class="npwthree"><li>Grind freshly detached leaves (200-400 mg) in liquid nitrogen with a mortar and pestle, and add 300 Âµl of the ice-cold protein extraction buffer (prepared immediately before use, Recipe 1).<br/><em>Note: Our laboratory normally uses three to four week-old rosette leaves, sampled midday. As NAD(P) metabolism could dynamically change during the phase transition from vegetative to reproductive growth (Hashida </em><em>et al.</em><em>, 2016), be sure to prepare samples before bolting. We recommend at least three biological replicates and two technical replicates to obtain reliable results.</em> </li><li>Further disrupt leaf sample by ultrasonic processing for 10 s with output control 6 on ice. Repeat this step two times to thoroughly rupture chloroplasts.<br/><em>Note: According to manufacturerâs specifications of the equipment (UR-21P), output control 6 responds to 6 Watts. Ultrasonic frequency is 28 kHz and exponential horn has 2.5 mm diameter.</em> </li><li>After centrifugation at 14,000 <em>x g</em>, 4 Â°C for 10 min, transfer supernatant (<em>i.e.</em>, crude extract) to a 1.5 ml microcentrifuge tube to remove sample debris.</li><li>Remove NAD<sup>+</sup>, NADH, NADP<sup>+</sup> and NADPH using Zeba<sup>TM</sup> Spin Desalting Columns by a 4 Â°C centrifugation at 14,000 <em>x g</em> for 2 min and collect the flow-through containing the extracted proteins in a new 1.5 ml microcentrifuge tube.</li></ol></li><li>Protein measurement<ol class="npwthree"><li>Prepare the dye for the Bio-Rad Protein Assay by diluting 1 volume of the dye concentrate with 4 volumes of distilled, deionized (DDI) water.</li><li>Add 2 and 5 Âµl of protein extracts into 198 and 195 Âµl diluted dye reagent, respectively.</li><li>After 5 min, measure absorbance at 595 nm and estimate the protein concentration with the protein standard II (BSA) from the Bio-Rad Protein Assay Kit (0.2 to 2.0 mg/ml).</li></ol></li><li>NAD<sup>+</sup> phosphorylation reaction<ol class="npwthree"><li>Mix the ingredients in a new 1.5 ml tube bellow in the following amounts:<br/>Extracted proteins, 10 Âµg<br/>Reaction buffer (Recipe 2), 10 Âµl<br/>50 mM NAD<sup>+</sup>, 10 Âµl<br/>50 mM ATP, 10 Âµl</li><li>Incubate the mixture at 30 Â°C for 30 min in a water bath.</li><li>Terminate reaction by heating the sample at 95 Â°C for 2 min.</li><li>After centrifugation at 14,000 <em>x g</em>, 4 Â°C for 5 min, use the supernatant for NADP measurement (designated here as âMeasuring solutionâ).</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Setting up the chloroplastic NAD kinase assay<ol class="npwtwo"><li>NAD<sup>+</sup> phosphorylation reaction<ol class="npwthree"><li>Float freshly prepared leaf discs in a 12-well plate on DDI water, and place the plate under dark for 1 h (dark acclimation).<br/><em>Note: Our laboratory uses only 12 leaf discs (with 4 mm diameter) per well to avoid cross-shading during light exposure (Figure 1).</em> </li><li>Transfer the plate under light (80 Âµmol photons/(sÂ·m<sup>2</sup>)) and start a time-course for the termination of the reaction, by collecting two discs into a 1.5 ml microcentrifuge tube containing 80 Âµl of 0.2 N HCl (for the quantification of the oxidized form, NADP<sup>+</sup>), and another two discs into 80 Âµl of 0.2 N NaOH (for the quantification of the reduced form, NADPH).<br/><em>Note: Be sure to prepare blank controls (no leaf discs in the tube).</em> </li><li>Heat the sample for 2 min at 95 Â°C in a hot water bath, and thoroughly ram the discs by pestle. After adding 8 Âµl of 200 mM sodium-phosphate solution (pH 5.6), adjust pH between 5 and 6 with the appropriate volumes of 0.2 N NaOH or 0.2 N HCl.<br/><em>Note: Check the pH with a test paper. This pH adjustment usually requires 50 to 60 Âµl NaOH and HCl solution.</em> </li></ol></li><li>After centrifugation at 14,000 <em>x g</em>, 4 Â°C for 5 min, use the supernatant for NADP measurement, designated here as the âMeasuring solutionâ.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191226/20191226181814_4232.jpg" width="100"/><br/><strong>Figure 1. Leaf discs for the chloroplastic NAD kinase assay</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>NADP measurement<ol class="npwtwo"><li>Preparing detection reagent for the NADP/NADPH-Glo<sup>TM</sup> assay<ol class="npwthree"><li>For luminescence-based NADP detection, prepare the required amount of luciferin detection reagent (Recipe 3) before all the assays, and stored it at room temperature (approximately 23 to 25 Â°C). When you obtain the âMeasuring solutionâ (see above), prepare the NADP/NADPH-Glo detection reagent just before use (Recipe 4).</li><li>For the calibration curve, prepare the standard using 1 ÂµM NADP<sup>+</sup> and 1 ÂµM NADPH. Add 0, 1, 2, 3, 4 and 5 Âµl of NADP<sup>+</sup> or NADPH standard into a series of wells in a half area 96-well microplate and adjust the volume to 50 Âµl per well with each blank control.</li><li>Transfer 50 Âµl of the measuring solution, which was obtained in the Steps A3d or B1d, into the other empty wells.</li></ol></li><li>Luminescence-based detection with NADP/NADPH-Glo<sup>TM</sup> assay<ol class="npwthree"><li>Add 50 Âµl of the NADP/NADPH-Glo detection reagent to each well, and briefly shake plates on a microplate shaker (gently). Incubate for 20 min at room temperature (approximately 23 to 25 Â°C) in the dark.</li><li>Detect and record luminescence using a GloMax Navigator System. Set the integration at 0.3 s per well.<br/><em>Note: Repeat the detection after 5 to 10 min and check whether the luminescence count was saturated or not. If the count was saturated, dilute the measuring solution and repeat the detection procedure to avoid underestimation.</em> </li></ol></li></ol></li></ol><p class="pptt" id="biaoti30927">### Data analysis</p><ol class="npwtwok"><li>Calculation of the NADP<sup>+</sup> and NADPH quantity from the luminescence units (U.I.) needs to be done by using the standard curve. Values acquired from this protocol will represent the quantity of NADP<sup>+</sup> and NADPH molecules in the measuring solution. In the <em>in vitro</em> assay, NADPH could be sometimes detected even if NAD<sup>+</sup> was used for the unique substrate, due to contaminating activity from the NADP<sup>+</sup> reduction. In order to avoid underestimation, for the calculation of activity we recommend summing up the NADP<sup>+</sup> and NADPH quantities as nmol NADP/h/mg protein or as nmol NADP/h/g fresh weight (FW) as follows:<br/><br/>[sum of NADP<sup>+</sup> and NADPH (nmol)] Ã [volume of the measuring solution (Âµl)]/[used volume for detection (50 Âµl)]/[reaction time (h)]/[protein amount (mg) or sample weight (g fresh weight)]<br/><br/></li><li>Unlike for the <em>in vitro</em> assay, NADPH quantity is required to estimate NAD<sup>+</sup> kinase activity by the approach with leaf disc, simply because PETC generates NADPH from NADP<sup>+</sup> during illumination. A representative result is shown in Figure 2. Increase in the NADP, which is shown as a sum of NADP<sup>+</sup> and NADPH, is never observed under complete dark condition (Figure 2A, Dark). Here, calculation of the NADP<sup>+</sup> and NADPH quantity for each time points was done as follows:<br/><br/>[sum of NADP<sup>+</sup> and NADPH (nmol)] Ã [volume of measuring solution (Âµl)]/[used volume for detection (50 Âµl)]/[leaf area (cm<sup>2</sup>)]<br/><br/></li><li>NADP content can also be expressed as nmol NADP/mg Chlorophyll (Chl) or nmol NADP/h/g FW, for which the appropriate measures (quantification of the total Chl content, or measuring the weight of the leaf discs) would need to be taken, before the assay.</li><li>Light-driven NADP increase can usually be detected within 30 s. In the first approx. 10 min, NADP increase under illumination was proportional to time, as shown in Figure 2A. Figure 2B illustrates the increase in NADP at the beginning of illumination, as nmol NADP/g FW. Hence, the chloroplastic NAD kinase activity could be calculated as follows:<br/><br/>[sum of NADP<sup>+</sup> and NADPH (nmol)] Ã [volume of measuring solution (Âµl)] Ã [60 (min)]/[used volume for detection (50 Âµl)]/[time after light exposure (min)]/[sample weight (g)]<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191226/20191226181846_3528.jpg" width="203"/><br/><strong>Figure 2. Light-driven NADP increase.</strong> A. Increase in the NADP content only occurs under illumination. B. Early NADP increase (for at least 5 min) was proportional to time. Result of straight fitting (dashed line) was quite comparable to the curve fitting (solid line) in the first 5 min of the illumination.<br/><br/></li><li>Curve fitting revealed some plausible chloroplastic NAD kinase activities at various time points (Figure 3, solid line). As we can see the initial activity was estimated to be over 50 nmol/h/g FW, but this activity declined rapidly in the first couple of minutes, and then slowly after about 5 min. On the other hand, simple fitting by straight line shows a representative estimation of the chloroplastic NAD kinase activity (Figure 3, dashed line). We expressed NAD kinase activities both as nmol/h/g FW and nmol NADP/h/mg protein, based on the protein amounts in the leaf discs. In previous reports, NADK2 activity has been estimated to be around 20 to 60 nmol/h/g FW (5 to 15 nmol/h/mg protein) (Takahashi <em>et al.</em>, 2006 and 2009), which is consistent with the estimation from our newly developed assay.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191226/20191226181906_9133.jpg" width="163"/><br/><strong>Figure 3. Estimation of chloroplastic NAD kinase activity.</strong> According to the quantification data, NAD kinase activity was estimated in nmol/h/g FW, and nmol/h/mg Protein. Values on the solid line were calculated from curve fitting, while the value under the dashed line was calculated from straight fitting.<br/><br/></li><li>Besides the NAD kinase activity, NADP synthesis is also determined by other factors (for example the NAD substrate and ATP) and therefore we believe that not only the <em>in vitro</em>, but also the newly developed assay will help to deduce which factors are conditionally involved in the regulation of NADP synthesis under various environmental signals.<br/></li></ol><p class="pptt" id="biaoti30928">### Recipes</p><ol class="npwtwok"><li>Protein extraction buffer<br/>50 mM HEPES/KOH, pH 8.0<br/>10 mM MgCl<sub>2</sub><br/>1 mM CaCl<sub>2</sub><br/>1 mM PMSF<br/>One tablet of cOmplete<sup>TM</sup>, Mini Protease Inhibitor Cocktail per 10 ml</li><li>Reaction buffer<br/>0.5 M HEPES/KOH, pH 8.0<br/>100 mM MgCl<sub>2</sub> </li><li>Luciferin detection reagent<br/>Prepared fresh in the morning of the experiment, and stored at approximately 23 to 25 Â°C<br/>Reconstruction Buffer, included in NADP/NADPH-Glo<sup>TM</sup> Assay Kit<br/>Lyophilized Luciferin, included in NADP/NADPH-Glo<sup>TM</sup> Assay Kit</li><li>NADP/NADPH Glo detection reagent<br/>Luciferin detection reagent, as described above<br/>Reductase, included in NADP/NADPH-Glo<sup>TM</sup> Assay Kit<br/>Reductase Substrate, included in NADP/NADPH-Glo<sup>TM</sup> Assay Kit<br/>NADP Cycling Enzyme, included in NADP/NADPH-Glo<sup>TM</sup> Assay Kit<br/>NADP Cycling Substrate, included in NADP/NADPH-Glo<sup>TM</sup> Assay Kit<br/></li></ol><p class="pptt" id="biaoti30929">### Acknowledgments</p><p class="ppzz">This protocol was adapted and modified from the previous works by Takahashi <em>et al.</em> (2009) and Hashida <em>et al.</em> (2018). This work was supported by the JSPS KAKENHI Grant Number-17H05714 and 19H04715 to M.K.Y.</p><p class="pptt" id="biaoti30930">### Competing interests</p><p class="ppzz">The authors declare that here are no conflicts of interest.</p><p class="pptt" id="biaoti30931">### References</p><ol class="npwtwok"><li>Berrin, J. G., Pierrugues, O., Brutesco, C., Alonso, B., Montillet, J. L., Roby, D. and Kazmaier, M. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15711971" target="_blank">Stress induces the expression of AtNADK-1, a gene encoding a NAD(H) kinase in <em>Arabidopsis thaliana</em>.</a> <em>Mol Genet Genomics</em> 273(1): 10-19.</li><li>Chai, M. F., Chen, Q. J., An, R., Chen, Y. M., Chen, J. and Wang, X. C. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16244906" target="_blank">NADK2, an <em>Arabidopsis</em> chloroplastic NAD kinase, plays a vital role in both chlorophyll synthesis and chloroplast protection.</a> <em>Plant Mol Biol</em> 59(4): 553-564.</li><li>Dellâ Aglio, E., Giustini, C., Kraut, A., CoutÃ©, Y., Mazars, C., Matringe, M., Finazzi, G. and Curien, G. (2019). <a href="http://biorxiv.org/content/early/2019/01/16/521658.abstract" target="_blank">Identification of the Calmodulin-dependent NAD<sup>+</sup> kinase sustaining the elicitor-induced oxidative burst in plants.</a> <em>bioRxiv</em>: 521658.</li><li>Delumeau, O., Renard, M. and Montrichard, F. (2000). <a href="https://onlinelibrary.wiley.com/doi/full/10.1046/j.1365-3040.2000.00636.x" target="_blank">Characterization and possible redox regulation of the purified calmodulin-dependent NAD<sup>+</sup> kinase from <em>Lycopersicon pimpinellifolium</em>.</a> <em>Plant Cell Environ </em>23(11): 1267-1273.</li><li>Hashida, S. N., Itami, T., Takahara, K., Hirabayashi, T., Uchimiya, H. and Kawai-Yamada, M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27590711" target="_blank">Increased rate of NAD metabolism shortens plant longevity by accelerating developmental senescence in <em>Arabidopsis</em>.</a> <em>Plant Cell Physiol</em> 57(11): 2427-2439.</li><li>Hashida, S. N. and Kawai-Yamada, M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31404160" target="_blank">Inter-organelle nad metabolism underpinning light responsive NADP dynamics in plants. </a><em>Front Plant Sci</em> 10: 960.</li><li>Hashida, S. N., Miyagi, A., Nishiyama, M., Yoshida, K., Hisabori, T. and Kawai-Yamada, M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29920827" target="_blank">Ferredoxin/thioredoxin system plays an important role in the chloroplastic NADP status of <em>Arabidopsis</em>.</a> <em>Plant J</em> 95(6): 947-960.</li><li>Tai, L., Li, B. B., Nie, X. M., Zhang, P. P., Hu, C. H., Zhang, L., Liu, W. T., Li, W. Q. and Chen, K. M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31275331" target="_blank">Calmodulin is the fundamental regulator of NADK-mediated NAD signaling in plants. </a><em>Front Plant Sci</em> 10: 681.</li><li>Takahashi, H., Takahara, K., Hashida, S. N., Hirabayashi, T., Fujimori, T., Kawai-Yamada, M., Yamaya, T., Yanagisawa, S. and Uchimiya, H. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19587098" target="_blank">Pleiotropic modulation of carbon and nitrogen metabolism in <em>Arabidopsis</em> plants overexpressing the NAD kinase2 Gene. </a><em>Plant Physiol</em> 151(1): 100-113.</li><li>Takahashi, H., Watanabe, A., Tanaka, A., Hashida, S. N., Kawai-Yamada, M., Sonoike, K. and Uchimiya, H. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17082216" target="_blank">Chloroplast NAD kinase is essential for energy transduction through the Xanthophyll cycle in photosynthesis.</a> <em>Plant Cell Physiol</em> 47(12): 1678-1682.</li><li>Tezuka, T. and Yamamoto, Y. (1975). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16659381" target="_blank">Photoactivation of NAD kinase through phytochrome: Phosphate donors and cofactors.</a> <em>Plant Physiol</em> 56(6): 728-730.</li><li>Turner, W. L., Waller, J. C., Vanderbeld, B. and Snedden, W. A. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15247403" target="_blank">Cloning and characterization of two NAD kinases from <em>Arabidopsis</em>. identification of a calmodulin binding isoform.</a> <em>Plant Physiol</em> 135(3): 1243-1255.<br/></li></ol>