<body>
            <h1>High-throughput Site-directed Scanning Mutagenesis Using a Two-fragment PCR Approach</h1>
            <p><strong>Authors:</strong> Franziska M. Heydenreich, Tamara  MiljuÅ¡, Dalibor  MiliÄ, Dmitry B. Veprintsev</p>
            <h2>Abstract</h2>
            <p> Site-directed scanning mutagenesis is a useful tool applied in studying protein function and designing proteins with new properties, such as increased stability or enzymatic activity. Creating a systematic library of hundreds of site-directed mutants is still a demanding and expensive task. The established protocols for making such libraries include PCR amplification of the recombinant DNA using a pair of primers carrying a target mutation in the same PCR. Unfortunately, this approach is very often coupled with PCR artifacts which compromise overall efficiency of site-directed mutagenesis. To reduce the failure rate due to the PCR artifacts, we have set up a high-throughput mutagenesis protocol based on a two-fragment PCR approach. To this end, each mutation is introduced in two separate PCRs resulting in two linear fragments of the mutated plasmid. In the next steps, the PCR template is digested and the two matching plasmid fragments are joined together using Gibson assembly. Separating the corresponding mutagenic primers into two different PCRs decreases a number of artifacts and thus increases overall cloning efficiency. Furthermore, free software that we developed facilitates both high-throughput primer design and analysis of sequencing results. Overall, this protocol enabled us to efficiently produce several alanine-scanning libraries of 400 single-point mutations with complete coverage of the protein sequence.</p>
            <p><strong>Keywords:</strong> Scanning mutagenesis, Site-directed, High throughput mutagenesis, Two-fragment PCR, Gibson assembly</p>
            <h2>Background</h2><p> High-throughput cloning techniques are widely used for research involving expression and purification of proteins. Site-directed scanning mutagenesis techniques in particular are often used to generate mutants for functional studies or to generate stabilized proteins for biophysics and structural biology. Libraries necessary for these purposes may contain several hundred mutants and their generation can be expensive and time consuming. Many single-point mutant libraries have been created using single vector PCRs with overlapping primers, <em>e.g.</em> QuikChange (Agilent). Although this and similar mutagenesis techniques are well established and widely used (for a recent review see Ortega <em>et al.</em>, 2019), they may lead to a number of different errors in the final vector (Liu and Naismith, 2008; Edelheit <em>et al.</em>, 2009; Sun <em>et al.</em>, 2013; Heydenreich <em>et al.</em>, 2017) caused by inherent sequence overlap of primers. Such errors include additional mutations and primer repeats, which necessitates sequencing of multiple colonies or additional rounds of mutagenesis. The protocol presented here is based on a two-fragment PCR mutagenesis, where the primers used in each PCR reaction have no sequence complementarity. The two halves of a plasmid to be cloned are amplified in two separate PCRs, followed by <em>in vitro</em> circularisation using Gibson assembly reaction (Figure 1). As a result, the number of successful mutagenesis reactions after sequencing only one clone per mutant is increased, allowing us to generate multiple alanine-scanning libraries of complete proteins of <em>ca.</em> 400 amino acids in length within approx. 6 weeks each (Heydenreich <em>et al.</em>, 2017). In addition, our protocol can also be used to insert multiple point mutations in one round of cloning and for sequence modification (<em>e.g.</em>, addition of purification tags, small insertions and deletions) with increased efficiency.<br/><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191230/20191230000200_0715.jpg" width="350"/><br/><strong>Figure 1. Overview of the mutagenesis technique.</strong> AAscan software (Sun <em>et al.</em>, 2013) is used to design mutagenic primers for the high-throughput site-directed scanning mutagenesis. Two PCR reactions are done per mutant, in each of them approximately half of the vector is amplified. Two fragments containing one mutation are combined, followed by DpnI digestion at 37 Â°C overnight. Reaction clean-up is performed to purify DNA fragments, which are then assembled by Gibson assembly reaction. Bacteria are transformed with the resulting circular plasmid and plated on selective LB agar plates. One clone per mutant is sent for sequencing either on a selective LB agar 96-well plate or as purified DNA. All steps excluding the plating of the bacteria are done in 96-well plates. Sequencing results are analyzed in a high-throughput manner using MutantChecker program (Sun <em>et al.</em>, 2013).</p></p>
        </body>

<p class="pptt" id="biaoti30957">## Materials and Reagents</p><p class="ppzz"><em>Note: All reagents must be of analytical grade or higher and stored according to recommendations from the manufacturer. All plasticware must be DNase-free.</em> </p><ol class="npwtwok"><li>LB-agar 96-well plates with the appropriate antibiotic(s) (can be ordered from a sequencing company; if not, see Recipes)</li><li>Pipette tips, 10 Î¼l, 200 Î¼l and 1,250 Î¼l (VWR, European catalog numbers: 613-1093, 613-1092 and 613-1087)</li><li>(Optional) Sterile toothpicks</li><li>Polypropylene conical tubes 15 ml and 50 ml (Falcon, catalog numbers: 352096 and 352070)</li><li>Microcentrifuge tubes 1.5 ml (Simport, catalog number: T330-7G)</li><li>Pipetting reservoir (Diversified Biotech, catalog number: REBP-3000)</li><li>96-well PCR plates, 0.2 ml (VWR, European catalog number: 211-0297)</li><li>Microseal âBâ PCR plate seals (Bio-Rad, catalog number: MSB1001)</li><li>Deep-well plates 96/2,000 Î¼l (Eppendorf, catalog number: 651033405)</li><li>Adhesive gas-permeable film for culture plates (VWR, European catalog number: 391-1261)</li><li>Sterile Petri dishes (<em>e.g.</em>, ThermoScientific Nuclon<sup>TM</sup> Delta Surface, catalog number 150350)</li><li>Disposible sterile spreaders (<em>e.g.</em>, VWR, catalog number: 612-5497) or autoclaved glass beads 2 mm (<em>e.g.</em>, Supelco, catalog number: 1.04014)</li><li>ZR-96 DNA Clean &amp; Concentrator (Zymo Research, catalog number: D4024)</li><li>Chemically competent <em>Escherichia coli</em> XL-1 Blue cells (Agilent, catalog number: 200249) or some other cell strain which is used for cloning (Mach1, DH5Î±, TOP10 <em>etc.</em>)</li><li>Cloning primers<br/>ColE1A: 5â²-GGAGCGAACGACCTACACCGAACTGAGATACCTACAGCG-3â²<br/>ColE1B: 5â²-CGCTGTAGGTATCTCAGTTCGGTGTAGGTCGTTCGCTCC-3â²<br/><em>Note: This pair of primers can be used for all plasmids containing the ColE1 origin of replication.</em> </li><li>Plasmid to be used as a template in PCR</li><li>Nicotinamide adenine dinucleotide, NAD<sup>+</sup>, 50 mM (New England Biolabs, catalog number: B9007L)</li><li>Deoxynucleotide (dNTPs) solution mix, 10 mM each dNTP (New England Biolabs, catalog number: N0447S)</li><li>T5 exonuclease, 10 U/Î¼l (New England Biolabs, catalog number: M0363S)</li><li>Phusion HF DNA polymerase, 2 U/Î¼l (New England Biolabs, catalog number: M0530L)</li><li>Taq DNA ligase, 40 U/Î¼l (New England Biolabs, catalog number: M0208L)</li><li>DpnI, 20 U/Î¼l (New England Biolabs, catalog number: R0176L)</li><li>Nuclease-free water (Cell Signaling Technology, catalog number: 12931S)</li><li>Phusion High-Fidelity PCR Master Mix with GC Buffer (New England Biolabs, catalog number: M0532L) or some other appropriate DNA polymerase master mix (see Procedure B. PCR)</li><li>PEG-8000 (Sigma-Aldrich, catalog number: P5413)</li><li>Tris-HCl buffer, 1 M, pH 7.5 (Thermo Fischer Scientific, catalog number: 15567027)</li><li>MgCl<sub>2</sub>, 1 M (Sigma-Aldrich, catalog number: M1028)</li><li>DL-dithiothreitol (DTT), 1 M (Sigma-Aldrich, catalog number: 43816)</li><li>SOC outgrowth medium (New England Biolabs, catalog number: B9020S)</li><li>Antibiotic stock solution(s) (depending on an antibiotic resistance gene(s) carried by the plasmid of interest)</li><li>Lysogeny broth (LB) broth with agar (Sigma-Aldrich, catalog number: L2897)</li><li>Ultrapure deionized water with electrical resistivity of 18 MÎ© cm at 25 Â°C</li><li>5x isothermal reaction (IT) buffer (see Recipes)</li><li>1.33x Gibson assembly mixture (see Recipes)</li><li>LB-agar plates with the appropriate antibiotic(s)-Petri dishes (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30958">### Equipment</p><ol class="npwtwok"><li>Diverse pipettes including multi-channel pipettes and repeat pipettors</li><li>Thermocycler that can accommodate 96-well PCR plates (<em>e.g.</em>, Eppendorf<sup>TM</sup> Mastercycler<sup>TM</sup> Nexus Thermal Cycler, catalog number: E6332000029)</li><li>Centrifuge for deepwell plates (<em>e.g.</em>, Eppendorf 5810 R with swing-bucket rotor A-4-62, catalog number: 5811000320)</li><li>Laboratory incubator set at 37 Â°C</li><li>Laboratory incubator/shaker set at 37 Â°C<br/></li></ol><p class="pptt" id="biaoti30959">### Software</p><ol class="npwtwok"><li>AAscan (Sun <em>et al.</em>, 2013; <a href="https://github.com/dmitryveprintsev/AAScan/" target="_blank">https://github.com/dmitryveprintsev/AAScan/</a>)</li><li>MutantChecker (Sun <em>et al.</em>, 2013; <a href="https://github.com/dmitryveprintsev/AAScan/" target="_blank">https://github.com/dmitryveprintsev/AAScan/</a>)<br/></li></ol><p class="pptt" id="biaoti30960">### Procedure</p><ol class="npwone"><li>Primer design and ordering<ol class="npwtwo"><li>Download and open AAscan software (Sun <em>et al.</em>, 2013) to design mutagenic primers for the site-directed scanning mutagenesis. A screenshot of the AAscan software interface is shown in Figure 2.</li><li>Replace the default DNA sequence given in AAscan with a DNA sequence which is to be mutated, including the flanking regions (40-50 nucleotides) at the 5â² and 3â² ends (<em>DNA seq. incl.40-50 nt flanking regions</em>).</li><li>Define the position of the first nucleotide in the first triplet codon (<em>Nt position of 1<sup>st</sup> triplet</em>). This relates the given DNA sequence with the amino-acid numbering. In the example given in Figure 2, the 50 nucleotides from the vector upstream of the protein coding sequence are included as a flanking region. Therefore, the nucleotide A from the ATG codon, which corresponds to amino-acid residue 1, is at the position 51. Whenever possible, we recommend using a standard numbering scheme for the protein of interest. This does not have to match the amino-acid numbering in the expressed protein construct (which might include a signal peptide, purification tags, protease cleavage sites, linkers or fusion protein partners).</li><li>Define up to two codons coding for an amino acid (<em>Xxx</em>) by which you want to replace other amino acids in the scanning mutagenesis, <em>e.g.</em>, âGCGâ and âGCAâ for Ala (<em>mut codon 1 and 2)</em>. If only one particular codon is to be used, include it into both <em>mut codon 1 and 2</em> text fields. We recommend choosing the codons based on their frequency in the expression host and the GC content of the gene of interest. Select those codons which are most frequent in the expression host and, if possible, avoid those with high or low GC content if your gene of interest has a very high (&gt; 60%) or very low (&lt; 40%) GC content, respectively. For several constructs with GC content in a range 50-55 % which were to be expressed in human cell culture (HEK 293S), we used the most frequent codons for this host: GCC/GCT for Ala, GGC/GGA for Gly, and GTG/GTC for Val. On the other hand, if the gene of interest has a high GC content, it might be better to use codons with lower GC content, <em>i.e.</em>, GCT/GCA for Ala, GGA/GGT for Gly and GTT/GTA for Val. Such codon choice minimizes the number of designed mutagenic primers with relatively high annealing temperatures.</li><li>If a particular amino acid in the scanning mutagenesis region is already Ala, replace it with another amino acid <em>Yyy</em> by defining up to two corresponding codons, <em>e.g.</em>, in alanine scanning mutagenesis replacing an existing alanine with a glycine (<em>if Ala, use Gly</em>). In choosing the codons, use the recommendations given in the previous step.</li><li>Enter an amino acid mutagenesis range (text fields <em>From</em> and <em>To</em>) by using the amino-acid numbering scheme defined in Step A3.</li><li>Define the primer design parameters. Based on our experience (Sun <em>et al.</em>, 2013; Heydenreich <em>et al.</em>, 2017), we recommend using the following parameters: minimal primer length (<em>min length</em>)â18 nucleotides (nt); maximal primer length (<em>max length</em>)â60 nt; minimal GC clamp (<em>MinGCclamp</em>)â2; optimized GC clamp (<em>OptimisedGCclamp</em>); minimal primer melting temperature (<em>MinTm</em>)â60 Â°C; maximal primer melting temperature (<em>MaxTm</em>)â70 Â°C; maximal difference between melting temperatures of forward and reverse primer (<em>MaxDeltaTm</em>)â5 Â°C; minimal distance from the mutation codon to the 3â² end of the primer (<em>MinAnnealLen</em>)â15 nt; length of the overlap sequence between the ends of the resulting PCR fragment (<em>MinOverlap </em>and<em> MaxOverlap</em>)âbetween 15 and 27 nt; the number of primer pairs designed for the mutation of every specified amino acid position (<em>MaxSuggestions</em>)â1.</li><li>Define the output format. Use separate result columns for forward and reverse primers (<em>Separate F and R</em>). User can choose among the more (<em>Long1</em> and <em>Long2</em>) or less (<em>Short</em> and <em>mFASTA</em>) detailed outputs.</li><li>Click or touch <em>Batch</em> button. This gives two separate lists of the designed forward (â<em>n</em>_Fâ) and reverse (â<em>n</em>_Râ) primers including their names and sequences (and other primer properties if output format <em>Long1</em> or <em>Long2</em> was chosen). â<em>n</em>â in a primer name corresponds to an amino acid number which is to be mutated by using that primer.</li><li>Order the designed primers from a company that provides synthesis of DNA oligonucleotides. Such company usually makes available an order sheet template file into which the AAscan output (obtained in Step A10) can easily be copied. When ordering, use a 96-well plate format with forward and reverse primers separated into different 96-well plates, but in such a way that a position of each forward primer corresponds to exactly the same position of the related reverse primer in the other 96-well plate (<em>e.g.</em>, primers 103_F and 103_R at the position A6 in each 96-well plate). Order desalted DNA oligonucleotides already dissolved at concentration 100 Î¼M either in nuclease-free water or in some standard buffer provided by the company (usually 10 mM Tris-HCl, 0.1 mM EDTA, pH 7.5); see Note 1.</li><li>Purchase a pair of the desalted reverse-complementary primers which anneal approximately on the opposite side of the plasmid. It is advisable to choose such primers that bind either to the origin of replication or the antibiotic resistance gene. In this case, only the properly assembled plasmid DNA will allow growing the transformed bacteria on a plate with selection antibiotic. We use origin-of-replication ColE1 primers (see Materials and Reagents).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191230/20191230005316_6070.jpg" width="450"/><br/><strong>Figure 2. AAscan software interface.</strong> In the example shown, the coding sequence is written in capital letters and the flanking regions are given in small letters (<em>DNA seq incl. 40-50 nt flanking regions</em>). The first nucleotide in the first triplet codon (ATG) is at the position 51 of the specified DNA sequence (<em>Nt position of 1<sup>st</sup> triplet</em>). The mutagenic primers designed for amino-acid residues 2-371 are separated into two output windows: the forward primers are listed in the left one and the reverse primers in the right one.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>PCR<br/>Each mutation is generated in two separate PCRs (denoted A and B) resulting in two PCR fragments that are combined in subsequent Gibson assembly. Before performing PCR for all of the target mutations, we recommend testing PCR conditions on a small number (20-30) of randomly chosen mutations and modifying the protocol accordingly, as detailed in the original article by Heydenreich <em>et al.</em> (2017). We recommend testing different polymerases and buffer condition as well as DMSO addition. To identify efficient PCR conditions, we analyze each PCR product from the test mutation set in agarose gel electrophoresis. Nevertheless, we do not include this or any other control in the large-scale mutagenesis.<br/><ol class="npwtwo"><li>Before doing the experiment, check the orientation of the gene of interest in its plasmid. This is important to determine how to combine in PCR mutagenic primers with the origin-of-replication ColE1 primers. A mutagenic primer and the appropriate ColE1 primer should anneal to complementary DNA strands; <em>e.g.</em>, if the forward mutagenic primers are combined with ColE1B, the reverse mutagenic primers should be combined with ColE1A (and vice versa).</li><li>Dilute the plasmid DNA template with nuclease-free water to concentration 1 ng/Î¼l.</li><li>Dilute the primers ColE1A and ColE1B with nuclease-free water to concentration 6 Î¼M.</li><li>Dilute the mutagenic primers in a 96-well plate with nuclease-free water to 6 Î¼M.</li><li>Two sets of PCRs are necessary. For each set of <em>x</em> mutagenesis PCR reactions, prepare a PCR master mix as defined in Table 1.<br/><br/><strong>Table 1. PCR master mixes.</strong> Each mixture is enough for <em>x</em> mutagenesis PCRs in 20 Î¼l reaction volume.<br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191230/20191230005340_3590.jpg" width="550"/><br/><br/></li><li>To prepare PCR mixtures A, pipette using a multichannel pipette 19 Î¼l PCR Master Mix A into each used well of a 96-well 200-Î¼l PCR plate on ice and add to it 1 Î¼l mutagenic primer (6 Î¼M) from the appropriate primer set (forward or reverse, as discussed in Step B1). Close the 96-well PCR plate with a PCR seal.</li><li>Prepare a PCR program as in Table 2 (see Note 2).<br/><br/><strong>Table 2. PCR program for DNA multiplication using Phusion polymerase</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191230/20191230005355_3460.jpg" width="550"/><br/>*Size of the longest DNA fragment in kilobase pairs (kbp) to be amplified in PCR; the extension time is 30 s per 1 kbp.<br/><br/></li><li>Preheat a thermocycler: a block to 98 Â°C (denaturation temperature) and a lid to 103 Â°C (see Note 3).</li><li>Transfer the sealed 96-well PCR plate with the reaction mixtures directly from ice to the preheated thermocycler, close the lid and start the PCR program defined in Step B7.</li><li>Keep the 96-well PCR plate with completed PCR on ice until further processing.</li><li>To prepare PCR mixtures B, repeat the Steps B6-B9 by using PCR Master Mix B and the corresponding mutagenic primers (the second mutagenic primer set, as discussed in Step B1).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>DNA template digestion and reaction clean-up<ol class="npwtwo"><li>After completion of both PCRs A and B, transfer 20 Î¼l PCR mixture A to the corresponding 20 Î¼l PCR mixture B in a 96-well PCR plate using a multichannel pipette. In this step, we combine two fragment PCR products possessing the same mutation.</li><li>Add 0.5 Î¼l DpnI (20 UÂ·Î¼l<sup>-1</sup>) to each mixture of fragments, seal the 96-well plates and incubate them at 37 Â°C to digest methylated DNA template. DpnI digestion for 1 h is already enough, but the mixtures can be incubated for up to 18 h if this is more convenient in terms of time management.</li><li>In order to purify the PCR fragments from the reaction mixture after the DpnI digestion, use ZR-96 DNA Clean &amp; Concentrator-5 Kit<sup>TM</sup> (Zymo Research) according to the manufacturerâs protocol, but use 10 Î¼l elution buffer in the last step. This small volume should result in higher DNA concentrations for the subsequent Gibson assembly.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Gibson assembly<ol class="npwtwo"><li>In a 96-well PCR plate on ice, mix 1 Î¼l each mixture of cleaned PCR fragments with 3 Î¼l 1.33x Gibson assembly mix (Recipe 2) by pipetting gently up and down ten times using a multichannel pipette set at volume 3 Âµl to avoid foaming (see Note 4).</li><li>Transfer the sealed 96-well plate with Gibson assembly reaction mixtures into a preheated thermocycler (a block at 50 Â°C and a lid at 53 Â°C) and incubate it for 10 min at 50 Â°C followed by 1 h at 37 Â°C and a hold at 10 Â°C. Alternatively and especially for longer nucleotide overlaps within a pair of mutagenic primers (<em>e.g.</em>, 21 or more nt; see Step A8), the Gibson assembly reaction can be conducted for 1 h at 50 Â°C followed by a hold at 10 Â°C. Which Gibson assembly conditions are better suited can be checked first on a small number (20-30) of randomly chosen mutations, as detailed by Heydenreich <em>et al.</em>, 2017 (see Note 5).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell transformation<ol class="npwtwo"><li>Thaw chemically competent <em>Escherichia coli</em> XL-1 Blue cell suspension on ice (see Note 6).</li><li>Onto a 96-well PCR plate on ice, distribute 20 Î¼l cell suspension per each well and mutation using a repeat pipette.</li><li>Add 2 Î¼l Gibson assembly mixture to each 20-Î¼l cell aliquot, seal the plate with a PCR plate seal and incubate on ice for 10-20 min.</li><li>Transfer the 96-well plate with cells-DNA mixtures into a preheated thermocycler (a block at 42 Â°C and a lid at 42 Â°C) and incubate it for 60 s at 42 Â°C.</li><li>Immediately put the cells-DNA mixtures on ice and incubate for 2-5 min. If the plasmid carries the ampicillin resistance gene, you can proceed immediately to Step E8 (although Steps E6 and E7 increase cloning efficiency in this case as well).</li><li>Using a multichannel pipette, transfer the transformed cells to a 96-deep-well plate filled with 300 Î¼l SOC media (without any antibiotic), close the plate with a gas-permeable seal and incubate for 1 h at 37 Â°C while shaking at 160 rpm.</li><li>Spin down the cells (3,000 <em>x g</em>, 5 min) and carefully remove 220 Î¼l supernatant using a multichannel pipette.</li><li>Resuspend a cell pellet in the remaining supernatant (<em>ca.</em> 100 Î¼l) by pipetting up and down using a multichannel pipette set at 80 Âµl (to avoid foaming) until all cell pellets are resuspended without visible clumps during pipetting. Plate the cell suspension onto an LB-agar plate with the appropriate antibiotic. To save some storage space and material, you can plate four different mutants per LB-agar plate (one mutant onto each plate quadrant) or 24 mutants on a 24-well LB-agar plate (one mutant per well) by using an expanding multi-channel pipette.</li><li>Allow the plates to dry for <em>ca.</em> 10 min and then incubate them upside-down at 37 Â°C overnight. Next day, you should find bacterial colonies on the LB-agar plates.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Plasmid DNA preparation and sequencing<br/>For high-throughput projects, sequencing companies offer plasmid preparation and sequencing directly from a bacterial colony on 96-well plates without extra costs in comparison to sending already extracted plasmid samples. We strongly recommend this option.<ol class="npwtwo"><li>Label a position of a selected colony on an LB-agar plate, pick it by a sterile pipette tip or a toothpick and transfer it to LB-agar with the appropriate antibiotic in a 96-well plate. Such plates are usually offered by a sequencing company; if not, you can easily prepare them by your own (Recipe 3).</li><li>Order the sequencing with the appropriate sequencing primer (forward or reverse). For reliable results, the sequencing primer should anneal at least 30-40 nucleotides upstream from the expected mutation. Depending on a sequencing service, one can usually expect reliable sequence read lengths of at least <em>ca.</em> 600 nucleotides (which corresponds to <em>ca.</em> 200 amino acids). For longer sequences, one should use additional sequencing primer(s) to cover the whole region of interest. Send the 96-well LB-agar plates with one colony per mutant to the sequencing company and wait for the sequencing results. Sending one mutant per sequencing round reduces sequencing costs.</li><li>Analyze the sequencing results in MutantChecker software (Sun <em>et al.</em>, 2013), as detailed in the Data analysis section.</li><li>If the mutation is not confirmed in a sequencing round, send a new colony in the next sequencing round (see Note 8).</li><li>You can obtain plasmid solutions in a 96-well plate from the sequencing company. Alternatively or in addition, you can prepare plasmids by yourself from the colonies labeled on LB-agar plates which were positive in sequencing.</li></ol></li></ol><p class="pptt" id="biaoti30961">### Data analysis</p><p class="ppzz">Analysis of sequencing data</p><ol class="npwtwo"><li>Save the sequencing data as separate .fasta, .seq or .txt files in a folder on the computer.</li><li>Open MutantChecker (Figure 3) and paste the reference non-mutated DNA sequence into the indicated field (<em>Reference sequence</em>). The sequence should start with the first codon as defined in Procedure, Step A3. Analyze mutant DNAs sequenced with forward primers separately from those sequenced with reverse primers. For more detailed information see Sun <em>et al.</em> (2013).</li><li>For DNA sequenced with forward primers, change the default values provided in <em>Report mut from</em> # and <em>To</em> in the respective fields, using amino acid numbers. Generally, a forward sequence should cover about 200 to 250 amino acids.</li><li>Once you have checked the program settings, click on âFileâ â âLoad File Listâ. Select one or multiple files to be checked. We recommend checking files one by one initially. If several files are analyzed at once, an alignment will only be displayed for the last file selected.</li><li>The largest window will now display the alignment of the reference sequence and the sequencing result (Figure 3). The graph on the right indicates how many possible alignments have been found. Ideally, you should see a single sharp peak. The window on the lower right displays sequence names followed by any mutation found within the limits entered in step 3 (see Note 7).</li><li>For DNA sequenced with reverse primers, indicate the range of amino acids where a mutation should be reported using the fields <em>RC From</em> and <em>RC To</em>. RC stands for reverse complement. Tick the <em>ReverseComplement</em> box when checking DNA sequenced with reverse primers.</li><li>Repeat steps 4-5 to analyze the remaining sequencing data.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20191230/20191230005428_9545.jpg" width="550"/><br/><strong>Figure 3. MutantChecker software interface.</strong> Sequencing data analysis for mutation R104A is shown as an example. Graph on the right side depicts the alignment score function <em>vs.</em> offset between the sequencing result and the reference sequence (Sun <em>et al.</em>, 2013). In this case, there is a single peak which indicates that one possible alignment was found. Multiple peaks would have implied several alternative alignments due to repeated sequences (usually an indication of a cloning artifact). Mutations identified in the specified amino acid range (1-200) are listed in the lower right output window. âE198@â denotes that sequencing trace starts to be unclear (ânâ in the sequencing result) from amino acid E198.<br/></li></ol><p class="pptt" id="biaoti30962">### Notes</p><ol class="npwtwok"><li>No primer modification (like phosphorylation) is necessary for Gibson assembly and the minimum synthesis scale of DNA oligonucleotides is usually more than sufficient. Store the frozen primers at -20 Â°C for up to 2 years. Several freeze-thaw cycles have no considerable influence on the stability of DNA oligonucleotides.</li><li>Different PCR protocols might be needed for DNA polymerases other than Phusion.</li><li>This step minimizes mis-priming during initial denaturation. It may not be necessary if using hot-start polymerases.</li><li>Instead of the homemade Gibson assembly mix, one can use a commercial Gibson assembly master mix or a similar DNA assembly product.</li><li>More recently, we successfully used incubation for 1 h at 45 Â°C for which it was shown to increase the efficiency of Gibson assembly (Gibson <em>et al.</em>, 2010). This may be especially beneficial if the primers were designed with relatively short overlap to improve their specificity (<em>e.g.</em>, 15 bp).</li><li>The cells should be prepared by the Inoue method (Sambrook and Russell, 2006) and have a transformation efficiency of at least 10<sup>7</sup> colonies per 1 Î¼g pBR322 (or a similar plasmid). Alternatively, one can use some other <em>E. coli</em> strain that is customary for routine cloning, such as Mach1, DH5Î± or TOP10.</li><li>Depending on the company which sequenced the DNA, the annotation of the traces may vary. Some companies decide not to display an âNâ when the trace is unclear, but instead opt for the highest peak. In both cases, a mismatch will be reported, but it may be perceived as a real mutation when the trace does not contain Ns. We recommend that you check the trace in cases like that even if the reported mutation occurs at the very beginning or end of a sequencing result. Especially towards the end of a sequence, a repeat of four or more bases of the same type (all Gs, all Cs, all As or all Ts) may lead to difficult-to-interpret traces, which can lead to <em>n</em> bases being read as <em>n</em>-1 or <em>n</em>+1, resulting in a frameshift. This is usually a sequencing problem rather than a true frameshift. These cases, or sequences trailed by many Ns may lead to additional peaks close to the main peak.</li><li>For troubleshooting, please refer to the original publication (Heydenreich <em>et al.</em>, 2017).</li></ol><p class="pptt" id="biaoti30963">### Recipes</p><ol class="npwtwok"><li>5x Isothermal reaction (IT) buffer (6 ml)<br/>1.5 g PEG 8000<br/>3 ml Tris-HCl (1 M, pH 7.5)<br/>300 Î¼l MgCl<sub>2</sub> (1 M)<br/>300 Î¼l DTT (1 M)<br/>600 Î¼l NAD<sup>+</sup> (50 mM)<br/>600 Î¼l dNTPs solution mix (10 mM each dNTP)<br/>Prepare 500-Î¼l aliquots of 5x IT buffer and store them at -80 Â°C</li><li>1.33x Gibson assembly mixture (1.8 ml)<br/>480 Î¼l 5x IT buffer<br/>0.96 Î¼l T5 exonuclease (10 U/Î¼l)<br/>30 Âµl Phusion HF DNA polymerase (2 U/Î¼l)<br/>240 Âµl Taq DNA ligase (40 U/Î¼l)<br/>1,049 Î¼l nuclease-free water<br/>Make 50-Î¼l aliquots and store at -20 Â°C (or -80 Â°C for long-term storage)<br/><em>Note: Gibson assembly mixture can be stored for more than a year at -20 Â°C and the enzymes are still active after more than 10 freeze-thaw cycles (Gibson et al., 2009).</em> </li><li>LB-agar plates with antibiotic<br/><ol class="npwthree" style="white-space:normal;"><li>Add 35 g LB broth with agar in 1 L ultrapure deionized water</li><li>Autoclave at 121 Â°C for 15 min to dissolve the powder and sterilize LB-agar</li><li>Let it cool down to hand-hot temperature (<em>ca.</em> 45 Â°C), add stock solution of the appropriate antibiotic to achieve the target antibiotic concentration and swirl vigorously to mix</li><li>Pour <em>ca.</em> 15 ml melted LB-agar with antibiotic into each 10-cm-diameter Petri dish and let it solidify</li><li>For 96-well LB-agar plates, use a multi-channel pipette to transfer 150 Î¼l LB-agar with antibiotic per well into a 96-well plate with flat bottom and let the agar solidify</li><li>The plates should be stored upside-down in sealed plastic bags at 4 Â°C until use</li></ol></li></ol><p class="pptt" id="biaoti30964">### Acknowledgments</p><p class="ppzz">This protocol was derived from our previous publication (Heydenreich <em>et al.</em>, 2017). This work was supported by Swiss National Science Foundation Doc.Mobility fellowship 165219 to FMH, Erasmus, EMBO Short Term Fellowship ASTF 420â2016 and Swiss National Science Foundation Doctoral Mobility Grant 31003A_159748/2 travel fellowships, and NanoMem ITN network funding to TM, NRW Strategy Project BioSC BOOST FUND âRIPEâ to the Institute of Biochemical Plant Physiology at the Heinrich Heine University, Marie Curie (274497) and UBS Promedica Foundation post-doctoral fellowships to DM, and Swiss National Science Foundation grants 141898, 159748 to DBV and COST Action CM1207 GLISTEN.</p><p class="pptt" id="biaoti30965">### Competing interests</p><p class="ppzz">The authors declare no competing financial or non-financial interests.</p><p class="pptt" id="biaoti30966">### References</p><ol class="npwtwok"><li>Edelheit, O., Hanukoglu, A. and Hanukoglu, I. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19566935" target="_blank">Simple and efficient site-directed mutagenesis using two single-primer reactions in parallel to generate mutants for protein structure-function studies.</a> <em>BMC Biotechnol</em> 9: 61.</li><li>Gibson, D. G., Smith, H. O., Hutchison, C. A., 3rd, Venter, J. C. and Merryman, C. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20935651" target="_blank">Chemical synthesis of the mouse mitochondrial genome.</a> <em>Nat Methods</em> 7(11): 901-903.</li><li>Gibson, D. G., Young, L., Chuang, R. Y., Venter, J. C., Hutchison, C. A., 3rd and Smith, H. O. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495" target="_blank">Enzymatic assembly of DNA molecules up to several hundred kilobases.</a> <em>Nat Methods</em> 6(5): 343-345.</li><li>Heydenreich, F. M., MiljuÅ¡, T., Jaussi, R., Benoit, R., MiliÄ, D. and Veprintsev, D. B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28754896" target="_blank">High-throughput mutagenesis using a two-fragment PCR approach.</a> <em>Sci Rep</em> 7(1): 6787.</li><li>Liu, H. and Naismith, J. H. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19055817" target="_blank">An efficient one-step site-directed deletion, insertion, single and multiple-site plasmid mutagenesis protocol.</a> <em>BMC Biotechnol</em> 8: 91.</li><li>Ortega, C., Abreu, C., Oppezzo, P. and Correa, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31267446" target="_blank">Overview of high-throughput cloning methods for the post-genomic era.</a> <em>Methods Mol Biol</em> 2025: 3-32.</li><li>Sambrook, J. and Russell, D. W. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22485385" target="_blank">The inoue method for preparation and transformation of competent <em>E. coli</em>: "ultra-competent" cells.</a> <em>CSH Protoc</em> 2006(1).</li><li>Sun, D., Ostermaier, M. K., Heydenreich, F. M., Mayer, D., Jaussi, R., Standfuss, J. and Veprintsev, D. B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24205336" target="_blank">AAscan, PCRdesign and MutantChecker: a suite of programs for primer design and sequence analysis for high-throughput scanning mutagenesis.</a> <em>PLoS One</em> 8(10): e78878.<br/></li></ol>