<body>
            <h1>Quantifying HIV-1-Mediated Gut CD4<sup>+</sup> T Cell Death in the Lamina Propria Aggregate Culture (LPAC) Model</h1>
            <p><strong>Authors:</strong> Stephanie M. Dillon, Kejun Guo, Moriah J. Castleman, Mario L. Santiago, Cara C. Wilson</p>
            <h2>Abstract</h2>
            <p>Gut CD4 T cells are major targets of HIV-1 and are massively depleted early during infection. To better understand the mechanisms governing HIV-1-mediated CD4 T cell death, we developed the physiologically-relevant Lamina Propria Aggregate Culture (LPAC) model. The LPAC model is ideal for studying CD4 T cell death induced by clinically-relevant Transmitted/Founder (TF) HIV-1 strains and is also suitable for studying how enteric microbes and soluble factors (<em>e.g.</em>, Type I Interferons) impact LP CD4 T cell death and function. Here, we detail the protocol to establish LP CD4 T cell infection using a process of spinoculation, the subsequent evaluation of infection levels using multicolor flow cytometry and the determination of overall LP CD4 T cell death using absolute LP CD4 T cell counts. We also describe the preparation of virus stocks of Transmitted/Founder (TF) HIV-1 infectious molecular clones that were successfully used in the LPAC model.</p>
            <p><strong>Keywords:</strong> Lamina Propria Aggregate Culture, Lamina Propria Mononuclear cells, HIV-1, Gut, T cells, Infection, Death</p>
            <h2>Background</h2><p>CD4 T cell depletion is the hallmark pathogenic outcome of HIV-1 infection. However, the magnitude and kinetics of CD4 T cell depletion depends on the tissue compartment evaluated. The gastrointestinal tract is a major site of early HIV-1 replication and CD4 T cell death. At these early stages, the circulating HIV-1 strains primarily utilize CCR5 instead of CXCR4 as HIV co-receptors along with CD4. In fact, nearly all transmitted/founder (TF) HIV-1 strains utilize CCR5 (Keele <em>et al.</em>, 2008; Salazar-Gonzalez <em>et al.</em>, 2009). Unfortunately, CD4 T cells from peripheral blood mononuclear cells (PBMCs) and tonsil histocultures (the Human Lymphoid Aggregate Culture or HLAC model) are generally resistant to cell death following infection by CCR5-tropic HIV-1, likely due to the lack of CCR5 expression. Moreover, blood and tonsil CD4 T cells are mostly of the naÃ¯ve and central memory subsets, which are less prevalent in the gastrointestinal tract. By contrast, gut CD4<sup>+</sup> T cells are mainly of the effector memory subset and express high levels of CCR5. Gut CD4<sup>+</sup> T cells also have higher representation of Th17 and Th22 subsets, which protect the integrity of the intestinal barrier, than that of blood or tonsil CD4<sup>+</sup> T cells. Thus, peripheral blood and the HLAC model have limited utility in understanding TF HIV-1-mediated CD4<sup>+</sup> T cell depletion in the gut. This prompted our group to adapt principles from the HLAC model (Doitsh <em>et al.</em>, 2010; Doitsh <em>et al.</em>, 2014) towards lamina propria mononuclear cells (LPMCs) (Steele <em>et al.</em>, 2014).<br/><br/>The protocol for the isolation of LP mononuclear cells (LPMC) from human small intestinal tissue in the LPAC model is detailed in previous publications, and described in more detail in Procedure A (Howe <em>et al.</em>, 2009; Dillon <em>et al.</em>, 2010, 2012 and 2016; Steele <em>et al.</em>, 2014; Harper <em>et al.</em>, 2015; Yoder <em>et al.</em>, 2017). In brief, LPMC are isolated from surgically resected small intestinal samples by collagenase-digestion after the removal of muscle, fat, mucus and epithelial layers. LPMC are cryopreserved at 5 x 10<sup>6</sup> to 10 x 10<sup>6</sup> LPMC per cryovial and have excellent overall viability on thawing (mean Â± SEM: 87 Â± 0.8% of total leucocytes (CD45<sup>+</sup>); N = 69, see Procedure B). Jejunum LPMCs consist mostly of CD3 T cells. On average, CD4 T cells constitute 65 Â± 1.5% of viable LP leucocytes with smaller percentages of CD8 T cells, B cells and innate immune cells present (Figure 1). On average, 74 Â± 3% (N = 30) of LP CD4 T cells express CCR5 (data not shown).</p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200108/20200108223723_6156.jpg" width="160"/><br/><strong>Figure 1. Distribution of immune cell populations in LPMC isolated from human jejunum tissue.</strong> Multi-color flow cytometry was used to determine frequencies of total CD3<sup>+</sup> cells (n = 77), CD4<sup>+</sup> cells (n = 75), CD8<sup>+</sup> cells (n = 77), CD19<sup>+</sup> B cells (n = 34) (left axis) and various innate immune cells including Î³Î´T cells (n = 45), CD11c<sup>+</sup> myeloid cells (n = 50) and subsets of innate lymphoid cells (ILCs, n = 11) (right axis) within viable, leucocytes (CD45<sup>+</sup>).</p><p class="pptt" id="biaoti31008">## Materials and Reagents</p><ol class="npwtwok"><li>1.5 ml Eppendorf tube graduated (Fisher, catalog number: 05-408<sup>-</sup>137)</li><li>15 ml conical tubes (Thermo Fisher, catalog number: 339651)</li><li>50 ml conical tubes (Thermo Fisher Scientific, catalog number: 339652)</li><li>60 ml disposable syringe</li><li>Vinyl Specimen Molds (Tissue Tek Cryomold, catalog number: 4566)</li><li>Round-bottom Polystyrene tubes (âFACS tubesâ) (Falcon, catalog number: 352052)</li><li>Syringe-driven filter (0.45 Î¼m) (Millipore, catalog number: SLHP033RS)</li><li>Petri dish 100 mm x 20 mm (Fisher, catalog number: FB0875711Z)</li><li>Polycarbonate thick wall ultracentrifugation tube (Beckman Coulter, catalog number: 355631)</li><li>Cell strainer (70 Î¼m) (Fisher, catalog number: 22363548)</li><li>Mega Cassette System Tissue Tek (Sakura, catalog number: 4173)</li><li>96 V-bottom culture plates (Corning, catalog number: 3894)</li><li>96-well white tissue culture-treated sterile microplate (PerkinElmer, catalog number: 6005680)</li><li>T175 culture flasks (Celltreat, catalog number: 229351)</li><li>TZM-bl cells (AIDS Reagent Program, catalog number: 8129)</li><li>TF infectious molecular clones:<ol class="npwthree"><li>CH040 (AIDS Reagent Program, catalog number: 11740)</li><li>CH058 (AIDS Reagent Program, catalog number: 11856)</li><li>CH077 (AIDS Reagent Program, catalog number: 11742)</li><li>CH470 (AIDS Reagent Program, catalog number: 12422)</li></ol></li><li>APC anti-human CD8 (Clone RPA-T8) (Tonbo Biosciences, catalog number: 20-0088)</li><li>Bovine Serum Albumin (BSA) (Fisher Scientific, catalog number: BP1600-100)</li><li>293T cells (ATCC, catalog number: CRL-3216)</li><li>RD-1 HIV-1 core antigen (Clone KC57) (Beckman Coulter, catalog number: 6604667)</li><li>0.25% Trypsin (Corning, catalog number: MT25053Cl)</li><li>Collagenase D (Roche Diagnostics, catalog number 11088882001)</li><li>DNase I (Sigma-Aldrich, catalog number: DN25-1G)</li><li>Fetal Bovine Serum (FBS) (Life Technologies, catalog number: 26140079)</li><li>HIV p24 antigen ELISA (Perkin Elmer, catalog number: NEK05001KT)</li><li>Human AB Serum (Gemini Bioproducts, catalog number: 100-512)</li><li>One Shot<sup>TM</sup> Stbl3 competent cells (ThermoFisher, catalog number: C737303)</li><li>PE/Dazzle 594 anti-human CD3 (Clone UCHT1) (Biolegend, catalog number: 300450)</li><li>2 M CaCl<sub>2</sub> (Sigma-Aldrich, catalog number: C1016)</li><li>10% Buffered Formalin Phosphate (Fisher Scientific, catalog number: SF100-4)</li><li>20% Sucrose solution in PBS</li><li>70% EtOH composed of Absolute EtOH200 proof (Fisher Scientific, catalog number: BP2818<sup>-</sup>4) diluted in Molecular Biology Grade Water (Hyclone, catalog number: SH30538.03)</li><li>Ampicillin Sodium Salt (Fisher Scientific, catalog number: BP1760-5)</li><li>Britelite<sup>TM</sup> plus reagent (PerkinElmer, catalog number: 6066761)</li><li>Cryotube Vial (Thermo Scientific, catalog number: 363401)</li><li>Dextran Sulfate (Fisher Scientific, catalog number: BP1585-100)</li><li>Dimethyl sulfoxide (DMSO) (Sigma, catalog number: D2650-5X10mL)</li><li>DL-Dithiothreitol (DTT) (Sigma, catalog number: D9779-1G)</li><li>Dulbeccoâs Modified Eagle Medium (DMEM) (Corning, catalog number: MT10013CV.</li><li>Dulbeccoâs Phosphate Buffered Saline (DPBS) (GIBCO, catalog number: 14190-144)</li><li>EDTA (Sigma-Aldrich, catalog number: E6511-100G)</li><li>Fungizone (Amphotericin B) (Corning, catalog number: 30-003-CF)</li><li>Hanksâ Balanced Salt Solution 1x (HBSS) (Corning, catalog number: 21-021-CM)</li><li>HEPES (Sigma-Aldrich, catalog number: H0887-100mL)</li><li>Kanamycin Sulfate (Fisher Scientific, catalog number: BP906-5)</li><li>LB broth (Fisher scientific, catalog number: BP9723)</li><li>Medium A (Fixation Medium) (Beckman Coulter, catalog number: GAS001S100)</li><li>Medium B (Permeabilization Buffer) (Beckman Coulter, catalog number: GAS002S100)</li><li>Paraformaldehyde (Fisher Scientific, catalog number: T353-500)</li><li>Penicillin and streptomycin and L-glutamine (Gibco, catalog number: 10378<sup>-</sup>016)</li><li>Penicillin-Streptomycin (Corning, catalog number: 30-002-Cl)</li><li>Qiagen HiSpeed Plasmid Maxi Kit (Qiagen, catalog number: 12662)</li><li>OTC Compound (Tissue Tek, catalog number: 4583)</li><li>RNA Later Solution (Invitrogen, catalog number: AM7021)</li><li>RPMI 1640 (RPMI) (Gibco, catalog number: 11875-093)</li><li>SOC medium (Corning, catalog number: 46-003-CR)</li><li>Trypan Blue (Corning, catalog number: 25-900-CI)</li><li>Zombie Aqua Fixable Viability Dye (Biolegend, catalog number: 423102)</li><li>Zosyn (piperacillin/tazobactium)(Novaplus, catalog number: 44567-801-10)</li><li>KCl</li><li>NaCl</li><li>Na<sub>2</sub>PO<sub>4</sub>Â·7H<sub>2</sub>O</li><li>Dextrose</li><li>1% Paraformaldehyde (see Recipes)</li><li>Thaw Media (see Recipes)</li><li>2x HBS buffer (see recipes)</li><li>Complete DMEM Media (see Recipes)</li><li>Culture RPMI Media (see Recipes)</li><li>Flow-staining buffer (see Recipes)</li><li>Overnight holding media for jejunum tissue (see Recipes)</li><li>Mucous removal media (see Recipes)</li><li>Epithelium removal media (see Recipes)</li><li>Collagenase Media for LPMC isolation (see Recipes)</li><li>Preservative for isolated LPMCs (see Recipes)</li><li>Complete RPMI medium for TF virus concentration (see Recipes)</li><li>Modified epithelium removal media lacking BSA (see Recipes)</li></ol><p class="pptt" id="biaoti31009">### Equipment</p><ol class="npwtwok"><li>37 Â°C water bath</li><li>37 Â°C incubator</li><li>Centrifuge (Beckman Coulter, model: Allegra6R)</li><li>Cool Cell LX Freezing Container (Biocision, catalog number: BCS-405)</li><li>Germinator 500 (Surgical Equipment sterilizer) (Cell Point Scientific Inc., catalog number: 11931)</li><li>Hemocytometer (Hausser Scientific, catalog number: 1492)</li><li>Luminometer plate reader (PerkinElmer 2030 Multlabel Reader, model: Victor<sup>TM</sup> X5)</li><li>Multi-channel pipette (Ranin 20-200 Î¼l LTS, model: LM12-200)</li><li>Multi-color Flow Cytometer (BD Biosciences, LSRII)</li><li>Nutating Mixer Fixed Speed 120V (rotator) (Fisher Scientific, catalog number: 88<sup>-</sup>861-041)</li><li>Orbital shaker (Forma Scientific, model: 4518)</li><li>TC20 Automated Cell Counter (Bio-Rad, catalog number: 1450102)</li><li>Stratacooler (Stratagene Agilent, catalog number: 401349)</li><li>Surgical forceps (VWR, catalog number: 82027-388)</li><li>Surgical scissors (VWR, catalog number: 82027-590)<br/></li></ol><p class="pptt" id="biaoti31010">### Software</p><ol class="npwtwok"><li>FlowJo Software (Tree Star Inc.)</li></ol><p class="pptt" id="biaoti31011">### Procedure</p><ol class="npwone"><li>Isolation of LPMCs from human jejunum<br/>Jejunum tissue was obtained from the University of Colorado Hospital from patients undergoing elective abdominal surgery. The types of surgeries that provide jejunum tissue include Whipple procedures and bowel obstructions. Jejunum tissue received represents the surgical margins (tissue that would otherwise be discarded) and are macroscopically normal (verified by pathological assessment via histology). The time period between removal of jejunum tissue by surgeons and collection of tissue by laboratory personnel varies between 30 min and 3 h. The following protocol details how LPMCs are isolated from whole jejunum tissue, but in brief, tissue samples are obtained from the hospital, small pieces are removed for histology or RNA preservation, the remaining tissue is stored overnight in holding media, the following day the tissue is dissected to isolate the mucosa layer, followed by mechanical and enzymatic digestion to isolate LPMC for cryopreservation.<ol class="npwtwo"><li>Acquire tissue from hospital. Generally, it is provided in a specimen container (depending on the tissue size).</li><li>Place whole tissue in a Petri dish and weigh; record the weight.</li><li>Using surgical forceps and scissors, cut off 3 small pieces of tissue (0.5 cm<sup>2</sup> each), making sure to include the mucosa layer and any underlying tissue.<br/><em>Note: Generally, the tissue is provided as a long tube with surgical sutures closing both ends. You may need to remove surgical sutures on one end in order to access the mucosa inside.</em> <ol class="npwthree"><li>Place 1 tissue piece into a 1.5 ml Eppendorf tube containing 1 ml RNA later. Store the tube at -20 Â°C until use for gene analysis of whole tissue (<em>e.g.</em>, expression of <em>ccr5</em> or mucosa-associated <em>16S</em> for bacteria phyla).</li><li>Place 1 tissue piece into a 1.5 ml Eppendorf tube containing 1 ml of 10% buffered formalin. Store the tube at RT for 24 h. After 24 h, remove the tissue and transfer to a new Eppendorf tube containing 1 ml 70% EtOH. Store at RT until paraffin embedding for use in histological analysis of whole tissue.</li><li>Place 1 tissue piece into a vinyl specimen mold containing OTC. Cover the surface of the tissue with more OTC and place the specimen mold into a mega cassette. Put the entire cassette into liquid nitrogen for 5 min to flash freeze the tissue. Transfer the cassette to -80 Â°C for storage until use in histological analysis of whole tissue.</li></ol></li><li>Rinse the remaining tissue with 5 ml RT HBSS. Repeat rinsing 2-3 times until tissue is clean (<em>i.e.</em>, blood is no longer visible and any feces that were made visible by removing sutures to open one end of the tissue have been washed away).</li><li>Return the tissue to the specimen container and add enough overnight media (Recipe 1) to cover the entire tissue (approximately 10-30 ml).</li><li>Store at 4 Â°C overnight.</li><li>The following morning, transfer the whole tissue to a Petri dish for dissection.<ol class="npwthree"><li>Using surgical forceps and scissors, cut off all surgical sutures in order open both ends of the tissue âtubeâ.</li><li>Make an incision along the long length of the tissue âtubeâ.</li><li>Lay the fully open tissue piece on the Petri dish with the mucosa side down and the muscle/fat side upwards (Figure 2A).<br/><em>Note</em><em>: Fat is usually yellowish/white in color and attached to the muscle. The muscle layer is hard and pink/red in color. Large bands of muscle can often be seen. The mucosa layer is soft and light brown in color.</em> </li><li>Starting from one end, use the forceps to lift the muscle layer away from the mucosa and cut downwards to remove muscle and fat from the mucosa separating the layers as you proceed. The tissue can be inverted for easier access to the mucosa (Figure 2B). Removal of all muscle and fat reveals a smooth and distinct mucosa layer and a spate muscle/fat layer (Figure 2C).</li></ol></li><li>Transfer the mucosa layer to a Petri dish and weigh; record the âmucosa onlyâ weight.</li><li>Cut the mucosa into pieces weighing approximately 1 g. Place each mucosa piece into its own 50 ml conical tube (<em>e.g.</em>, for 5 g mucosa therefore require five 50 ml conical tubes).</li><li>Add 30 ml mucous removal media (Recipe 2) to each tube.</li><li>Place tubes onto rotator positioned inside a 37 Â°C incubator and rotate for 45 min with caps tightly closed.</li><li>Remove liquid from each tube using a graduated pipette and discard directly into liquid waste.</li><li>Add 20 ml RT HBSS to each tube and vortex for 10 s to wash away any remaining mucous on the tissue. Remove wash liquid from each tube using a graduated pipette and discard directly into liquid waste.</li><li>Repeat wash 2-3 times until liquid is clear and viscous mucous has been removed.</li><li>Transfer tissue piece to a new 50 ml conical tube.</li><li>Add 30 ml epithelium removal media (Recipe 3) to each tube.</li><li>Place tubes onto rotator positioned inside a 37 Â°C incubator and rotate for 45 min with caps tightly closed.</li><li>Remove liquid from each tube using a graduated pipette and discard directly into liquid waste. Tissue will now appear light pink in color (Figure 2D).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109011224_5989.jpg" width="117"/><br/><strong>Figure 2. Images of tissue dissection and LPMC isolation.</strong> A. Tissue is placed in a Petri dish with mucosa side down. The muscle layer and fat layer are facing upwards. B. Tissue orientation is inverted with muscle/fat layer facing downwards and mucosa layer facing upwards. Forceps are holding the mucosa layer (light brown in color) and pulling it away from the muscle/fat layer as scissors are used to separate the layers. C. Depiction of the two separated layers after dissection is complete. D. Tissue after epithelium has been removed during incubation with EDTA. Mucosa is now light pink in color. E. Minced tissue to prep for treatment with collagenase to enzymatically remove LPMCs. F. Tissue fragments and cells after incubation in collagenase. (Images by Kaylee Mickens.)<br/><br/></li><li>Add 20 ml RT HBSS to each tube and firmly vortex for 10 s to further dislodge any loose epithelial cells. Remove liquid from each tube using a graduated pipette and discard directly into liquid waste. Repeat 2-3 times until liquid is clear and no more epithelial cells are detached.</li><li>Add 30 ml epithelium removal media (Recipe 3) to the tubes containing the 1 g piece of mucosa.</li><li>Place tubes onto rotator positioned inside a 37 Â°C incubator and rotate for a secondary 45 min incubation with caps tightly closed.</li><li>Repeat Steps 18-19 to remove epithelial cell layer.<br/><em>Note</em><em>: If downstream examination of LPMCs suggests that epithelial cells were not sufficiently removed (</em><em>e.g.</em><em>, presence of epithelial cells detected via microscope or majority of cells are CD45 negative after staining via flow cytometry), then an additional step is suggested.</em><br/>Additional Step: Continuously vortex tube containing 1 g piece of mucosa for 5 min in the presence of 30 ml of modified epithelium removal media lacking BSA (Recipe 13). Remove liquid as described above and repeat vortexing as necessary 1-3x with fresh media.</li><li>Remove mucosa tissue from 1 tube and transfer to Petri dish. Using surgical forceps and scissors, mince tissue into small pieces (Figure 2E). Transfer minced tissue to a new 50 ml conical tube. Repeat for all tubes with mucosa tissue to prep tissue for enzymatic digestion.</li><li>Add 20 ml collagenase media for LPMC isolation (Recipe 4) to each tube.</li><li>Place tubes onto rotator positioned inside a 37 Â°C incubator and rotate for 60 min with caps tightly closed. Tissue fragments with now be white in color (Figure 2F).</li><li>Remove liquid from each tube using a graduated pipette and filter liquid through a cell strainer (70 Î¼m) placed inside a new 50 ml conical collection tube (this liquid will contain LPMCs).</li><li>Add 20 ml RT HBSS to each tube and gently vortex for 5 s to further dislodge any loose LPMCs.<ol class="npwthree"><li>Remove liquid from each tube using a graduated pipette and filter through a cell strainer (70 Î¼m) placed inside 50 ml conical collection tubes.</li><li>Repeat 2-3 times until liquid is clear and no more LPMCs are detached.</li><li>For the 50 ml conical tubes that have been used to collect LPMCs, pellet cells by centrifugation (585 <em>x g</em>, 8 min, at 25 Â°C; break set to âhighâ).</li><li>Add 1 ml culture media to resuspend each LPMC pellet (Recipe 7) and combine cell pellets into 1 tube. Store at 4 Â°C until time to freeze down cells.</li></ol></li><li>Add 20 ml collagenase media for LPMC isolation (Recipe 4) to each tube.</li><li>Place tubes onto rotator positioned inside a 37 Â°C incubator and rotate for a secondary 60 min with caps tightly closed.</li><li>Repeat Steps A26-A27 to collect additional LPMCs.</li><li>Discard any remaining tissue using appropriate biowaste disposal. These tissue fragments will be white in color and fibrous, indicating that full digestion of jejunum tissue is complete.</li><li>In the tube containing collected LPMCs, bring the volume up to 25 ml using HBSS. Make sure the cells are fully resuspended and homogenous within the liquid, otherwise cell counts will be affected.</li><li>Remove 15 Î¼l of cell suspension and add to 15 Î¼l 0.4% Trypan blue.<ol class="npwthree"><li>Load 10 Î¼l of cell mixture and count LPMCs using a hemocytometer.</li><li>Calculate the total number of LPMCs recovered from tissue processing.</li><li>LPMCs should be frozen at 5 to 10 million cells per cryovial, therefore label the appropriate number of cryovials needed to divide total LPMCs among cryovials.</li><li>Pre-cool labeled cryovials in a cold StrataCooler.</li></ol></li><li>Pellet cells by centrifugation (585 <em>x g</em>, 8 min, at 25 Â°C; break set to âhighâ).</li><li>Remove liquid from tube using a graduated pipette and discard directly into liquid waste.</li><li>To the LPMC cell pellet, add 500 Î¼l per cryovial of Culture RPMI media (Recipe 7). Make sure the cells are fully resuspended and homogenous within the liquid, otherwise cell distribution to cryovials will be uneven.<br/><em>Note: If total LPMCs will be split into 4 cryovials, then add 500 Î¼l x 4= 2,000 Î¼l culture RPMI media to cell pellet.</em> </li><li>Slowly add 500 Î¼l per cryovial (total amount added equal in volume to previous step) of preservative (Recipe 5). Gently swirl to mix.<br/><em>Note: If total LPMCs will be split into 4 cryovials, then add 500 Î¼l x 4 = 2,000 Î¼l preservative to cell pÃ«llet</em><em>.</em> </li><li>Transfer 1 ml LPMCs to each cryovial (Example used in previous steps: tube now contains LPMCs resuspended in 4,000 Î¼l composed of 2,000 Î¼l Culture RPMI media <sup>+</sup> 2,000 Î¼l preservative for a total of 4 ml to be divided amongst 4 cryovials).</li><li>Transfer cryovials containing LPMCs to Cool Cell freezing container.</li><li>Place freezing container in -80 Â°C for 24 h.</li><li>Transfer cryopreserved LPMCs to liquid nitrogen for storage until use.<br/><br/></li></ol></li><li>Thawing cryopreserved LPMC<ol class="npwtwo"><li>Place Thaw Media (Recipe 6) in a 37 Â°C water bath for at least 30 min.</li><li>Add 100 Î¼l Thaw Media to a 15 ml conical tube. 1 x 15 ml conical per 1 cryovial of LPMC (cryopreserved 5-10 x 10<sup>6</sup> cells/vial).</li><li>Place cryovial of frozen LPMC in the 37 Â°C water bath and thaw quickly.</li><li>Transfer thawed LPMC into a 15 ml conical containing 100 Î¼l thaw media (Step A2).</li><li>Wait for 1 min.</li><li>Add 200 Î¼l Thaw Media to the 15 ml conical tube, wait for 1 min.</li><li>Add 400 Î¼l Thaw Media to the 15 ml conical tube, wait for 1 min.</li><li>Add 800 Î¼l Thaw Media to the 15 ml conical tube, wait for 1 min.</li><li>Add 1.6 ml Thaw Media to the 15 ml conical tube, wait for 1 min.</li><li>Add 3.2 ml Thaw Media to the 15 ml conical tube, wait for 1 min.</li><li>Add additional Thaw Media for a final volume of 10 ml.</li><li>Pellet cells by centrifugation (585 <em>x g</em>, 8 min) at 25 Â°C (RT; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in 1 ml Thaw Media and gently pipette up and down, add 9 ml Thaw Media.</li><li>Pellet cells by centrifugation (585 <em>x g</em>, 8 min, at 25 Â°C; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in 1 ml RT DPBS and filter through Cell strainer (70 Î¼m).<br/><em>Tip for filtering LPMC:</em> <ol class="npwthree"><li><em>Pre-wet filter with DPBS.</em> </li><li><em>Add LPMC/DPBS.</em> </li><li><em>Rinse 15 ml conical with DPBS and filter.</em> </li><li><em>Add 1 ml DPBS directly to filter to rinse.</em> </li></ol></li><li>Count the LPMC in Trypan Blue using a hemocytometer.</li><li>Pellet cells by centrifugation (585 <em>x g</em>, 8 min) at 25 Â°C (RT; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in Culture Media (Recipe 7) at 2.5 x 10<sup>6</sup> LPMC/ml.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Infection protocol<br/>This protocol has utilized both the laboratory-adapted R5-tropic HIV-1<sub>Ba-L</sub> (Steele <em>et al.</em>, 2014; Dillon <em>et al.</em>, 2016) and the Transmitted/Founder (TF) HIV-1 infectious molecular clones (IMC) CH058, CH470 and CH040 (Harper <em>et al.</em>, 2015; Yoder <em>et al.</em>, 2017) (see Procedures E and F).<ol class="npwtwo"><li>Divide LPMC into 15 ml conical tube/s for âmockâ and âHIVâ conditions with the total number of tubes based on the following âinfection by spinoculationâ criteria (HIV p24 per 2.5 x 10<sup>6</sup> LPMC in 1 ml final volume of culture media):<ol class="npwthree"><li>500 ng HIV p24 antigen (p24) of TF strains or 25 ng p24 HIV-1<sub>Ba-L</sub> per 2.5 x 10<sup>6</sup> LPMC per ml with no more than 2.5 x 10<sup>6</sup> LPMC (<em>i.e.</em>, 1 ml) per 15 ml conical tube. HIV p24 concentrations of virus stocks need to be calculated for each virus preparation using an HIV p24 antigen ELISA following manufacturers protocol.</li><li>If multiple 15 ml conical tubes are needed, evenly distribute the LPMC over tubes: <em>e.g.</em>, if 6 x 10<sup>6</sup> total LPMC are needed for infection, distribute 2 x 10<sup>6</sup> cells per 15 ml tube.</li><li>Mock-infect with matched LPMC/ml per 15 ml conical.</li></ol><em>Note: If using GFP-expressing virus strains, add 1.5 x 10<sup>6</sup> LPMC into 15 ml conical to be used as GFP compensation control when evaluating GFP-expressing LP CD4 T cells using multi-color flow.</em> </li><li>Pellet cells by centrifugation (585 <em>x g</em>, 8 min) at 25 Â°C (RT; break set to âhighâ).</li><li>Remove all supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in Culture Media and then add HIV or mock as calculated (Step B1a). An example calculation is detailed below (Table 1).<ol class="npwthree"><li>Table 1 illustrates the calculations used to infect 2.5 x 10<sup>6</sup> LPMC with 500 ng p24 in a total of 1 ml of culture media. The specific column headings (denoted A-E) are detailed to assist with the calculations.<br/>For example:<br/>The total amount of p24 needed (Column B) is calculated based on the number of cells per 15 ml tube (Column A) when LPMC are infected with 500 ng p24 per 2.5 x 10<sup>6</sup> LPMC. The total final volume for spinoculation is based on infecting no more than 2.5 x 10<sup>6</sup> LPMC/ml per 15 ml tube and is calculated in Column C. The total volume of virus stock added is based on a final amount of 500 ng HIV p24 per 2.5 x 10<sup>6</sup> LPMC and is therefore dependent on the p24 concentration of the stock HIV. This example details the calculation when the stock concentration is 20,000 ng p24/ml and is based on Column B value (<em>i.e.</em>, the total p24 needed) and Column C (<em>i.e.</em>, the total final volume for spinoculation). The volume of culture media to add is based on the difference between the total volume (Column C) and the volume of p24 added (Column D). The volume of mock is matched to the volume of p24 added to the HIV tube.<br/></li></ol><br/><strong>Table 1. Example calculations using an HIV virus strain with a concentration of 20,000 ng p24 per ml<br/><img alt="" class="layerphoto" height="196" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109011311_4276.jpg" width="750"/><br/></strong> </li><br/><li>Centrifuge at 1,200 <em>x g</em>, 2 h, room temp (RT; break set to âoffâ).</li><li>Discard supernatant containing free virus and resuspend in 1 ml of Culture Media. Mock tubes are similarly treated with removal of supernatant and addition of Culture Media.</li><li>Centrifuge at 585 <em>x g</em>, 8 min at 25 Â°C (RT; break set to âhighâ).</li><li>Resuspend in Culture Media for a final concentration of 1 x 10<sup>6</sup> cells/ml. If additional stimulation conditions are required, divide cells into 15 ml conical tubes and add stimuli as appropriate.</li><li>Plate LPMC into 96 V-bottom culture plate (200 Î¼l per well).</li><li>Add 200 Î¼l DPBS to surrounding wells to minimize evaporation of Culture Media.</li><li>Incubate for 4 days at 37 Â°C, 5% CO<sub>2</sub> and 95% humidity.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Multi-color flow cytometry protocol to determine percentage of HIV-infected LP CD4 T cells<ol class="npwtwo"><li>Collect cultured LPMC and pellet cells by centrifugation (585 <em>x g</em>, 8 min, RT; break set to âhighâ).</li><li>Resuspend LPMC in DPBS and count in trypan blue using an automated cell counter. Record the number of viable, LPMC for all conditions.</li><li>Prepare an antibody cocktail containing a viability dye (1 Î¼l per 1 x 10<sup>6</sup> LPMC) and fluorochrome-labeled antibodies directed against CD3 and CD8 in Flow-staining buffer (Recipe 8) (4 Â°C). Concentrations of fluorochrome-labeled antibodies should be titrated for optimal performance by available flow cytometer.<ol class="npwthree"><li>HIV down-regulates CD4 expression therefore LP CD4 T cells are identified as CD3<sup>+</sup> CD8<sup>-</sup> T cells following <em>in vitro</em> infection with HIV-1 (Figure 2).</li><li>Specific viability dyes and fluorochrome-labelled CD3 and CD8 antibodies appropriate for multi-color flow cytometry can be user-specific and optimized for available flow cytometers. The example shown (Figure 3) uses Zombie Aqua Fixable Viability Dye (1 Î¼l per 1 x 10<sup>6</sup> LPMC), PE/Dazzle 594 anti-human CD3 (0.125 Î¼g per 1 x 10<sup>6</sup> LPMC) and APC anti-human CD8 (0.125 Î¼g per 1 x 10<sup>6</sup> LPMC) and were acquired on an LSRII Flow Cytometer (BD Biosciences).</li></ol></li><li>Transfer LPMC to FACS tubes and pellet cells by centrifugation (585 <em>x g</em>, 5 min, 4 Â°C; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in antibody cocktail.</li><li>Incubate in the dark, 30 min, 4 Â°C.</li><li>Add 4 ml DPBS (4 Â°C) and pellet cells by centrifugation (585 <em>x g</em>, 5 min, 4 Â°C; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in 100 Î¼l Medium A (fixation medium).</li><li>Incubate in the dark, 20 min, 4 Â°C.</li><li>Add 4 ml Flow cytometry buffer (4 Â°C) and pellet cells by centrifugation (585 <em>x g</em>, 5 min, 4 Â°C; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in 100 Î¼l Medium B (permeabilization buffer) containing 5 Î¼l RD-1 HIV-1 core antigen.</li><li>Incubate in the dark, 30 min, 4 Â°C.</li><li>Add 4 ml Flow cytometry buffer (4 Â°C) and pellet cells by centrifugation (585 <em>x g</em>, 5 min, 4 Â°C; break set to âhighâ).</li><li>Remove supernatant and flick tube to dislodge pellet.</li><li>Resuspend LPMC in 400 Î¼l 1% paraformaldehyde (Recipe 9).</li><li>Acquire on flow cytometer within 4 h.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109011353_1704.jpg" width="173"/><br/><strong>Figure 3. Identifying productively-infected LP CD4 T cells using multi-color flow cytometry. </strong>A. Identification of CD3<sup>+</sup> CD8<sup>-</sup> LP CD4 T cells. Viable cells are identified as Viability Dye- and then lymphocytes identified within this population based on Forward Side Scatter-Area versus Side Scatter-Area properties as shown. Doublets are excluded based on Forward Scatter-Width versus Forward-Scatter Height properties and CD3<sup>+</sup> CD8<sup>-</sup> T cells identified within these single cells. B. Expression of p24 in CD3<sup>+</sup> CD8<sup>-</sup> LP CD4 T cells using mock conditions as controls.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of Transmitted/Founder (TF) HIV-1 infectious molecular clone stocks<br/>The protocol below is used to specifically prepare TF HIV-1 virus stocks from infectious molecular clones (IMCs) CH040, CH058, CH077, and CH470.<ol class="npwtwo"><li>Preparation of TF HIV-1 plasmids<ol class="npwthree"><li>Transformation of TF plasmids<ol class="npwfour"><li>Set the water bath to 42 Â°C.</li><li>Thaw a vial of OneShot<sup>TM</sup> Stbl3 competent cells on ice for 5 min.</li><li>Add 100-200 ng of stock TF plasmid to the thawed Stbl3 cells, mix by gently flicking the tube a few times.</li><li>Incubate the cells on ice for 30 min. Put the Stbl3 cell tube on a floater and heat shock the cells at 42 Â°C for 45 s.</li><li>Chill the cells on ice for 5 min immediately after the heat shock.</li><li>Add 800 Î¼l of SOC medium to the cells.</li><li>Shake the cells at 30 Â°C in an orbital shaker for 1 h at 200 rpm.</li><li>Add 300-400 Î¼l of cell mixture onto a LB agar plate with compatible antibiotics (Table 2) and spread evenly using standard method.</li><li>Incubate the LB plate at 30 Â°C overnight (16-18 h).<br/></li></ol><br/><strong>Table 2. TE molecular clone vector antibiotic resistance</strong><strong><br/></strong><img alt="" class="layerphoto" height="198" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109011439_4377.jpg" width="750"/><br/></li><br/><li><span id="__kindeditor_bookmark_start_122__" style="display:none;"></span>Maxi-prep<ol class="npwfour"><li>Pick a single colony and inoculate in 6 ml of LB broth containing compatible antibiotics (Table 2).</li><li>Shake the LB broth at 30 Â°C for 16-18 h at 200 rpm.</li><li>Transfer 1 ml of culture and inoculate into 500 ml of the LB broth with compatible antibiotics (Table 2).</li><li>Again, shake the 500 ml of LB broth at 30 Â°C for 16-18 h at 200 rpm.</li><li>Perform Maxi-prep according to maxi-prep kitâs instruction.</li></ol><span id="__kindeditor_bookmark_end_123__" style="display:none;"></span> </li><li>Sequence verification of newly generated TF HIV-1 plasmids<br/>Because the TF HIV-1 molecular clones contain identical repeats on both ends, it is not rare that the plasmids would lose most of the virus genome insert during the transformation and plasmid amplification process. Therefore, it is critical to fully sequence the maxi-prep products before generating the live virus stocks. Included here is a set of 16 sequencing primers to ensure coverage of the entire range of the virus genome insert in the plasmid (Table 3).<br/><br/><strong>Table 3. TF HIV-1 sequencing primers<br/><img alt="" class="layerphoto" height="507" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109011504_2619.jpg" width="750"/><br/></strong> </li></ol></li><br/><li>Transfection of TF HIV-1 infectious molecular clones<ol class="npwthree"><li>Start the 293T cell culture a week before transfection so that the cells can be passaged at least twice. This will help the cells acclimate to the growth conditions and improve transfection efficiency.</li><li>293T culture medium: complete Dulbeccoâs Modified Eagle Medium (DMEM) with 10% FBS and 1% Penicillin-Streptomycin.</li><li>Transfer the 293T cells into T175 culture flasks during passages. The cells are ready for transfection at about 60-70% confluence.</li><li>Calcium Phosphate Transfection.<br/><strong>Reagents:</strong><br/>2 M CaCl<sub>2</sub><br/>2x HBS buffer (Recipe 11)<br/>Sterilize both reagents by filtering through 0.22 Âµm filter, and store at 4 Â°C until use.<br/><strong>Transfection:<ol class="npwfour"><li>Composition of the transfection mixture (per T175 flask): TF plasmid (60 Î¼g), H<sub>2</sub>O, 2x HBS buffer (2 ml), and 2 M CaCl<sub>2</sub> (256 Î¼l). The total volume is 4 ml.</li><li>Mix TF plasmid and H<sub>2</sub>O (total volume: 1,744 Î¼l) in a 15 ml conical tube.</li><li>Add 2 ml 2x HBS buffer and mix well.</li><li>Gently add CaCl<sub>2</sub> (dropwise) into the mix.</li><li>Incubate at room temperature for 30 min without mixing.</li><li>Add the transfection mix slowly to the medium in the 293T flask, mix by gently swirling the flask.</li><li>Transfer the flask into cell culture incubate (37 Â°C) in a BSL3 facility.<br/><em>Note: Due to the high risk working with high concentrations of TF HIV-1 virus, it is recommended to carry out subsequent steps in BSL3 facility if possible.</em> </li><li>After 16 h of incubation, replace half of medium (~20 ml) in the transfected flask with fresh complete DMEM medium (Recipe 10) to minimize the toxic effect of Calcium Phosphate to the cells.</li><li>Incubate for an additional 32 h before harvest.</li></ol></strong> </li></ol></li><li>Harvest the virus supernatant<ol class="npwthree"><li>Transfer the 293T flask into biosafety hood.</li><li>Decant the medium into a 50 ml conical, and centrifuge at 500 <em>x g</em> for 5 min to remove dead cells and cellular debris.</li><li>Attach a 0.45 Î¼m syringe-driven filter to a 60 ml disposable syringe with the plunger removed.</li><li>Decant the centrifuged medium into the syringe, then reinsert the plunger and push the liquid into a new 50 ml conical tube.</li><li>The virus supernatant can be aliquoted and stored at -80 Â°C until use. However, to ensure higher virus titer, it is recommended to concentrate the supernatant via ultracentrifugation.</li></ol></li><li>Concentrating the TF virus<ol class="npwthree"><li>Add 26 ml of virus supernatant into a polycarbonate thick wall ultracentrifugation tube.</li><li>Slowly add 4 ml of 20% sucrose solution (in PBS) under the virus supernatant at the bottom of the tube (underlaying).<br/><em>Note: for details on the underlay technique, please check the Youtube video (<a href="https://youtu.be/-SqlIMLUfcI" target="_blank">https://youtu.be/-SqlIMLUfcI</a>) at 0:43. Although the video shows Ficoll underlayed in whole blood, the same principle applies to underlaying 20% sucrose in the virus supernatant.</em> </li><li>Centrifuge the tubes at 100,000 <em>x g</em> for 2 h (break set to high).</li><li>Gently decant the supernatant, and put the tubes upside down on layers of absorbent papers to drain the excess fluid for 5 min.</li><li>Add 300-400 Î¼l of complete RPMI medium (Recipe 12) with 10% FBS and 1% Penicillin-Streptomycin.</li><li>Seal the tubes with Parafilm and store at 4 Â°C overnight to loosen the virus pellet.</li><li>Pipette up and down gently. Avoid generating too many bubbles to homogenize the mixture, then aliquot to small tubes and store at -80 Â°C until use.<br/><br/></li></ol></li></ol></li><li>Titration of the infectivity of the virus stocks (TZM-bl assay)<br/><ol class="npwtwo"><li>Start TZM-bl cell culture in complete DMEM medium a week before the assay.</li><li>Thaw virus quickly at 37 Â°C in a water bath, then keep cool on ice.</li><li>In a biosafety hood, make serial dilutions of the viruses in a round-bottom 96-well plate. For concentrated virus, a 1:100 dilution (5 Âµl virus into 495 Âµl media) to start is recommended, then a subsequent 1:1000 dilution (10 Âµl of 1:100 dilution into 990 Âµl media) then five 5-fold dilutions (100 Âµl of the 1:100 dilution into 400 Âµl media, then the same ratio subsequently). Keep the virus solutions on ice while preparing the TZM-bl cells. Make sure to include a pre-titrated HIV-1 stock as positive control.</li><li>Trypsin treat the cell culture flask to lift the TZM-bl cells.</li><li>Count the cell concentration in the supernatant using a cell counter.</li><li>Using 10,000/well density to calculate the volume required based on the number wells needed for the assay.</li><li>Add complete DMEM medium to adjust the total volume to equivalent to 160 Î¼l/well.</li><li>Add 18.75 Î¼g/ml of Dextran Sulfate into the cell mixture, mix well then transfer 160 Î¼l to each well on a 96-well white tissue culture-treated sterile microplate using a multi-channel pipette.</li><li>Transfer 40 Î¼l of the virus dilutions to corresponding wells and mix well by pipetting up and down. The final concentration for Dextran Sulfate should be 15 Âµg/ml after the addition of the virus dilution.</li><li>Fill each unused well with 200 Î¼l of sterile PBS to reduce the volume lost due to evaporation during incubation.</li><li>Cover lid, and incubate for 48 h.</li><li>Prepare the BriteLite plus reagent by dissolving the lyophilized powder with 10 ml of BriteLite buffer and let warm up at room temperature.</li><li>In a biosafety hood, remove 95 Î¼l of spent medium from each well using a multi-channel pipette.</li><li>Dispense 100 Î¼l of BriteLite plus reagent to each well and mix well by pipetting up and down.</li><li>Let the plate incubate for 2-3 min then read with a luminometer plate reader (at default luminescence setting for 1 s).</li><li>A luminescence readout below 2 million is considered a valid value. The viral infectivity is reported as relative light unit (RLU), which is calculated by multiplying the luminescence readout with the corresponding dilution factor.</li></ol></li></ol><p class="pptt" id="biaoti31012">### Data analysis</p><ol class="npwone"><li>Determination of percentages of infected LP CD4 T cells based on p24 expression<br/>To determine percentages of productively infected LP CD4 T cells (Figure 2), viable cells are identified based on lack of expression of a viability dye and then total lymphocytes within this population identified based on forward and side scatter properties. Doublets are excluded based on forward-scatter-width versus forward-scatter-height properties. Total T cells are identified by expression of CD3. HIV down-regulates CD4 expression therefore CD4 T cells are identified as CD8<sup>-</sup> T cells with total CD3<sup>+</sup> T cells. To establish specific expression of HIV-1 core antigen (p24), mock conditions are used as controls. Profiles evaluating co-expression of p24 and GFP are detailed in Yoder <em>et al.</em> (2017).<br/><br/></li><li>Quantification of LP CD4 T cell depletion<br/>To quantify HIV-specific LP CD4 T cell depletion after 4 days in culture, the difference in the number of viable LP CD4 T cells in HIV-infected cultures versus matched mock-infected cultures is determined (Figure 4). To enumerate the number of viable LP CD4 T cells, the total viable LPMC collected at the completion of the culture period (determined by trypan blue exclusion) is multiplied by the percentage of CD3<sup>+</sup> CD8<sup>-</sup> T cells in viable LPMC acquired using multi-color flow cytometry. The percent depletion relative to mock is then calculated.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109012621_7144.jpg" width="325"/><br/><strong>Figure 4. Example calculations to determine LP CD4 T cell depletion using the LPAC model.</strong> Line A reflects the total number of viable LPMC determined at the completion of the culture period based on trypan blue exclusion counts for both Mock condition and HIV-1 conditions. Line B reflects the percentage of CD3<sup>+</sup>CD8<sup>-</sup> LP T cells as a fraction of viable lymphocytes enumerated using multi-color flow cytometry. The total absolute number of LP CD4 T cells in each condition is then calculated by multiplying A with B. The graph reflects the number of LP CD4 T cells in mock and in HIV conditions with the percent depletion in HIV relative to mock shown, calculated as number in HIV-1 condition/number in mock condition (%).<br/><br/></li><li>Assay design, reproducibility and statistical analysis<br/>A minimum of 6 x 10<sup>5</sup> cells per condition are plated in triplicate wells at the initiation of the culture. At the completion of the culture period, LPMC from triplicate wells are pooled together, counted and evaluated for infection and depletion. Assays are performed on at least 6 different donor samples. Parametric tests are performed.</li></ol><p class="pptt" id="biaoti31013">### Notes</p><p class="ppzz">This assay is highly reproducible within samples from the same donor; however, variability exists between samples from different donors. This can be minimized by restricting the use of samples to those with &gt; 60% viability and &gt; 50% LP CD4 T cells (as a fraction of total LPMC) at baseline (freshly thawed). Furthermore, we recommend assessment of CCR5 genotype/phenotype for each sample to limit low infection rates due to CCR5 expression.</p><p class="pptt" id="biaoti31014">### Recipes</p><ol class="npwtwok"><li>Overnight holding media for jejunum tissue<br/>RPMI supplemented with:<br/>10% Human AB Serum<br/>1% Penicillin/streptomycin/L-glutamine<br/>500 Î¼g/ml Zosyn<br/>1.25 Î¼g/ml Fungizone<br/>1 M HEPES<br/>Mucous removal media (incubate in 37 Â°C water bath before use)<br/>HBSS supplemented with:<br/>1.67 mM DTT<br/>500 Î¼g/ml Zosyn<br/>1.25 Î¼g/ml Fungizone</li><li>Mucous removal media (incubate in 37 Â°C water bath before use)<br/>HBSS supplemented with:<br/>1.67 mM DTT<br/>500 Î¼g/ml Zosyn<br/>1.25 Î¼g/ml Fungizone</li><li>Epithelium removal media (incubate in 37 Â°C water bath before use)<br/>HBSS supplemented with:<br/>1 mM EDTA<br/>500 Î¼g/ml Zosyn<br/>1.25 Î¼g/ml Fungizone<br/>0.1% BSA</li><li>Collagenase Media for LPMC isolation (incubate in 37 Â°C water bath before use)<br/>RPMI supplemented with:<br/>10% BSA<br/>1% Penicillin/streptomycin/L-glutamine<br/>500 Î¼g/ml Zosyn<br/>1.25 Î¼g/ml Fungizone<br/>10 Î¼g/ml DNase I<br/>0.5 mg/ml Collagenase D<br/><em>Note: Adding more collagenase D could cause unwanted cleavage of cell surface proteins, which may affect identification of specific cells types during downstream techniques such as flow cytometry. Incubating mucosa tissue for longer than 2 periods of 60 min recommended by this protocol could also cause unwanted cleavage of cell surface proteins. New lots of collagenase D should be compared side-by-side for enzymatic activity and their effect on cell surface protein expression. Avoid multiple freeze/thaw cycles of Collagenase D as this will greatly impact enzymatic activity.</em> </li><li><em>Preservative for isolated LPMCs</em><br/>20% DMSO<br/>80% Human AB Serum</li><li>Thaw Media<br/>RPMI supplemented with:<br/>10% FBS<br/>1% Penicillin/streptomycin/L-glutamine<br/>10 Î¼g/ml DNase I</li><li>Culture RPMI Media<br/>RPMI supplemented with:<br/>10% Human AB Serum<br/>1% Penicillin/streptomycin/L-glutamine<br/>500 Î¼g/ml Zosyn</li><li>Flow-staining buffer<br/>DPBS supplemented with:<br/>1% Bovine Serum Albumin<br/>2 mM EDTA</li><li>1% Paraformaldehyde<br/>DPBS supplemented with:<br/>1% paraformaldehyde</li><li>Complete DMEM Media<br/>DMEM supplemented with:<br/>10% FBS<br/>1% Penicillin-Streptomycin</li><li>2x HBS buffer<br/>50 mM HEPES, pH 7.05<br/>10 mM KCl<br/>12 mM dextrose<br/>280 mM NaCl<br/>1.5 mM Na<sub>2</sub>PO<sub>4</sub><br/><strong>For 1 L:<ol class="npwthree"><li>Dissolve 11.92 g HEPES, 745.60 mg KCl, 2.16 g dextrose, 16.36 g NaCl, 40.2 g Na<sub>2</sub>PO<sub>4</sub>Â·7H<sub>2</sub>O in 950 ml H<sub>2</sub>O</li><li>Adjust pH to 7.05 with 1 N HCl, bring up to 1 L with H<sub>2</sub>O</li><li>Sterilize the solution by filtering through a 0.22 Âµm filter, and store at 4 Â°C until use</li></ol></strong> </li><li>Complete RPMI medium for TF virus concentration<br/>RPMI supplemented with:<br/>10% FBS<br/>1% Penicillin-Streptomycin</li><li>Modified epithelium removal media lacking BSA<br/>HBSS supplemented with:<br/>1 mM EDTA<br/>500 Î¼g/ml Zosyn<br/>1.25 Î¼g/ml Fungizone</li></ol><p class="pptt" id="biaoti31015">### Acknowledgments</p><p class="ppzz">This work was supported by National Institute of Health Grant R01 AI108404 (to C.C.W) and R01 AI134220 and AI145428 (to M.L.S. and C.C.W). We would like to thank Dr. Amanda Steele and Eric Lee for assistance with the development of the LPAC model, Kaylee Mickens for the pictures of gut processing in Figure 1, and Christine Purba for critical review of this protocol. We would also like to express our gratitude to the many laboratory members who processed the intestinal tissue samples used in the LPAC model and to the patients for agreeing to the use of their discarded surgical tissue for research purposes.</p><p class="pptt" id="biaoti31016">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt" id="biaoti31017">### Ethics</p><p class="ppzz">LPMC are obtained from discarded surgical tissue samples with patients providing informed consent to have these samples used for research purposes. All samples are de-identified to laboratory personnel and provided an ID code when received in the laboratory. This research was reviewed by the Colorado Multiple Institutional Review Board (IRB) (COMIRB) and met criteria, as defined by their policies in accordance with OHRP and FDA regulations, to be deemed âNot Human Researchâ.</p><p class="pptt" id="biaoti31018">### References</p><ol class="npwtwok"><li>Dillon, S. M., Rogers, L. M., Howe, R., Hostetler, L. A., Buhrman, J., McCarter, M. D. and Wilson, C. C. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20483758" target="_blank">Human intestinal lamina propria CD1c<sup>+</sup> dendritic cells display an activated phenotype at steady state and produce IL-23 in response to TLR7/8 stimulation.</a> <em>J Immunol</em> 184(12): 6612-6621.</li><li>Dillon, S. M., Manuzak, J. A., Leone, A. K., Lee, E. J., Rogers, L. M., McCarter, M. D. and Wilson, C. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22689879" target="_blank">HIV-1 infection of human intestinal lamina propria CD4<sup>+</sup> T cells <em>in vitro</em> is enhanced by exposure to commensal <em>Escherichia coli</em>.</a> <em>J Immunol</em> 189(2): 885-896.</li><li>Dillon, S. M., Lee, E. J., Donovan, A. M., Guo, K., Harper, M. S., Frank, D. N., McCarter, M. D., Santiago, M. L. and Wilson, C. C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26762145" target="_blank">Enhancement of HIV-1 infection and intestinal CD4<sup>+</sup> T cell depletion <em>ex vivo</em> by gut microbes altered during chronic HIV-1 infection.</a> <em>Retrovirology</em> 13: 5.</li><li>Doitsh, G., Cavrois, M., Lassen, K. G., Zepeda, O., Yang, Z., Santiago, M. L., Hebbeler, A. M. and Greene, W. C. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21111238" target="_blank">Abortive HIV infection mediates CD4 T cell depletion and inflammation in human lymphoid tissue.</a> <em>Cell</em> 143(5): 789-801.</li><li>Doitsh, G., Galloway, N. L., Geng, X., Yang, Z., Monroe, K. M., Zepeda, O., Hunt, P. W., Hatano, H., Sowinski, S., MuÃ±oz-Arias, I. and Greene, W. C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24356306" target="_blank">Cell death by pyroptosis drives CD4 T-cell depletion in HIV-1 infection.</a> <em>Nature</em> 505(7484): 509-514.</li><li>Harper, M. S., Guo, K., Gibbert, K., Lee, E. J., Dillon, S. M., Barrett, B. S., McCarter, M. D., Hasenkrug, K. J., Dittmer, U., Wilson, C. C. and Santiago, M. L. (2015).<a href="http://www.ncbi.nlm.nih.gov/pubmed/26529416" target="_blank"> Interferon-alpha subtypes in an <em>ex vivo</em> model of acute HIV-1 infection: expression, potency and effector mechanisms.</a> <em>PLoS Pathog</em> 11(11): e1005254.</li><li>Howe, R., Dillon, S., Rogers, L., McCarter, M., Kelly, C., Gonzalez, R., Madinger, N. and Wilson, C. C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19174326" target="_blank">Evidence for dendritic cell-dependent CD4<sup>+</sup> T helper-1 type responses to commensal bacteria in normal human intestinal lamina propria.</a> <em>Clin Immunol</em> 131(2): 317-332.</li><li>Keele, B. F., Giorgi, E. E., Salazar-Gonzalez, J. F., Decker, J. M., Pham, K. T., Salazar, M. G., Sun, C., Grayson, T., Wang, S., Li, H., Wei, X., Jiang, C., Kirchherr, J. L., Gao, F., Anderson, J. A., Ping, L. H., Swanstrom, R., Tomaras, G. D., Blattner, W. A., Goepfert, P. A., Kilby, J. M., Saag, M. S., Delwart, E. L., Busch, M. P., Cohen, M. S., Montefiori, D. C., Haynes, B. F., Gaschen, B., Athreya, G. S., Lee, H. Y., Wood, N., Seoighe, C., Perelson, A. S., Bhattacharya, T., Korber, B. T., Hahn, B. H. and Shaw, G. M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18490657" target="_blank">Identification and characterization of transmitted and early founder virus envelopes in primary HIV-1 infection.</a> <em>Pro</em><em>c Natl Acad Sci </em><em>U S A</em> 105(21): 7552-7557.</li><li>Salazar-Gonzalez, J. F., Salazar, M. G., Keele, B. F., Learn, G. H., Giorgi, E. E., Li, H., Decker, J. M., Wang, S., Baalwa, J., Kraus, M. H., Parrish, N. F., Shaw, K. S., Guffey, M. B., Bar, K. J., Davis, K. L., Ochsenbauer-Jambor, C., Kappes, J. C., Saag, M. S., Cohen, M. S., Mulenga, J., Derdeyn, C. A., Allen, S., Hunter, E., Markowitz, M., Hraber, P., Perelson, A. S., Bhattacharya, T., Haynes, B. F., Korber, B. T., Hahn, B. H. and Shaw, G. M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19487424" target="_blank">Genetic identity, biological phenotype, and evolutionary pathways of transmitted/founder viruses in acute and early HIV-1 infection.</a> <em>J Exp Med</em> 206(6): 1273-1289.</li><li>Steele, A. K., Lee, E. J., Manuzak, J. A., Dillon, S. M., Beckham, J. D., McCarter, M. D., Santiago, M. L. and Wilson, C. C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24495380" target="_blank">Microbial exposure alters HIV-1-induced mucosal CD4<sup>+</sup> T cell death pathways <em>ex vivo</em>.</a> <em>Retrovirology</em> 11: 14.</li><li>Yoder, A. C., Guo, K., Dillon, S. M., Phang, T., Lee, E. J., Harper, M. S., Helm, K., Kappes, J. C., Ochsenbauer, C., McCarter, M. D., Wilson, C. C. and Santiago, M. L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28241075" target="_blank">The transcriptome of HIV-1 infected intestinal CD4<sup>+</sup> T cells exposed to enteric bacteria.</a> <em>PLoS Pathog</em> 13(2): e1006226.</li></ol>