<body>
            <h1>Ribonucleoprotein Immunoprecipitation (RIP) Analysis</h1>
            <p><strong>Authors:</strong> Jennifer L. Martindale, Myriam Gorospe, Maria L. Idda</p>
            <h2>Abstract</h2>
            <p>RNAs and RNA-binding proteins (RBPs) can interact dynamically in ribonucleoprotein (RNP) complexes that play important roles in controlling gene expression programs. One of the powerful ways to investigate changes in the association of RNAs with an RBP of interest is by immunoprecipitation (IP) analysis of native RNPs. RIP (RNP immunoprecipitation) analysis enables the rapid identification of endogenous RNAs bound to an RBP and to monitor time-dependent changes in this association, as well as changes in response to different metabolic and stress conditions. The protocol is based on the use of an antibody, typically an anti-RBP antibody, to immunoprecipitate the RNP complex. The RNA within the immunoprecipitated complex can then be isolated and further studied using different approaches such as PCR, microarray, Northern blot, and sequencing analyses. Among other advantages, RIP analysis (i) measures RNP associations in many samples relatively quickly, (ii) can be adapted easily to different endogenous RBPs, and (iii) provides extensive information at low cost. Among its limitations, RIP analysis does not inform on the specific sites of interaction of an RBP with a given target RNAs, although recent adaptations of RIP have been developed to overcome this problem. Here we provide an optimized protocol for RIP analysis that can be used to study RNA-protein interactions relevant to many areas of biology.</p>
            <p><strong>Keywords:</strong> RNA-binding proteins, Ribonucleoprotein complex, Post-transcriptional regulation, Gene expression, RNA target</p>
            <h2>Background</h2><p>The post-transcriptional fate of RNA is strongly regulated by its dynamic association with RNA-binding proteins (RBPs) forming ribonucleoprotein (RNP) complexes that govern all aspects of RNA metabolism, including precursor RNA splicing, and RNA modification, folding, translation, stability, transport, and storage (Glisovic <em>et al.</em>, 2008). Additionally, alterations in RBP functions have been implicated in many human pathologies, including neurodegeneration (Kang <em>et al.</em>, 2014; Ravanidis <em>et al.</em>, 2018), immune diseases (Idda <em>et al.</em>, 2018; Yoshinaga and Takeuchi, 2019) and cancers (Pereira <em>et al.</em>, 2017). Thus, there is immense interest in developing methods to investigate RNPs and identify target RNAs that can illuminate the function of RBPs in physiology and disease. Excellent reviews of the most common methods for RNP analysis are available (McHugh <em>et al.</em>, 2014; Cipriano and Ballarino, 2018; Licatalosi <em>et al.</em>, 2019).<br/><br/>Originally developed in the Keene laboratory (Tenenbaum <em>et al.</em>, 2000; Keene <em>et al.</em>, 2006), RNP immunoprecipitation (RIP) analysis is most often used to measure the association of a specific RBP with RNAs in intact cells. The method described here is suitable for most RBPs, but the efficiency is strongly dependent on the quality of the antibodies, the abundance of the RBP analyzed, and the methods to assess bound RNA. In a nutshell, RIP analysis entails the immunoprecipitation (IP) of a specific RBP under mild conditions that preserve the RNP complexes in the IP, whereupon the RNAs present in the RNP complex can be isolated and further analyzed by a range of RNA-detection methods, such as reverse transcription (RT) followed by quantitative PCR (RT-qPCR) analysis (Figure 1A) or other techniques such as Northern, RNA-seq, or microarray analyses (Tenenbaum <em>et al.</em>, 2000; Keene <em>et al.</em>, 2006; Zhao <em>et al.</em>, 2010). RIP analysis is well suited to measure many RNAs at once and to detect changes in binding to RNAs as a function of time or stimulus in a fast and inexpensive way. While this RIP procedure does not reveal the actual site of RBP binding to a target RNA, adaptations of this method including a cross-linking step (crosslinking IP, CLIP) or not (Digestion Optimized-RIP, DO-RIP) do permit the identification of RNA sequences bound by the RBP, although these analyses are often more time-consuming and technically challenging (Nicholson <em>et al.</em>, 2017; Lee and Ule, 2018; Wheeler <em>et al.</em>, 2018; Lin and Miles, 2019).<br/><br/>In summary, RIP identifies RNAs associated with a given RBP and informs on changes in the intracellular composition of RNPs in response to different stimuli. RIP has been used in many laboratories to identify endogenous RBPs associated with endogenous RNAs in a wide range of cell types. In this protocol, we describe the use of RIP in the human monocytic leukemia line THP-1, although the same protocol can be used for other cell lines.<br/></p>
        </body>

<p class="pptt" id="biaoti31033">## Materials and Reagents</p><p class="ppzz"><em>Note: Please ensure that all reagents and materials are confirmed to be RNase-free. We normally use company-certified RNase-free solutions and materials for RIP assay.</em> </p><ol class="npwtwok"><li>MicroAmp<sup>Â®</sup> optical 384-well reaction plate (Thermo Fisher Scientific, Applied Biosystems<sup>TM</sup>, catalog number: 4309849)</li><li>Safe-Lock 1.5-ml Eppendorf Tubes (Eppendorf, catalog number: 0030120086)</li><li>ThermoGrid<sup>TM</sup> rigid strip 0.2-ml PCR tubes (Denville Scientific, catalog number: C18064)</li><li>Disposable cuvettes, 1.5-ml (Stockwell Scientific, catalog number: 2410)</li><li>Protein A-Sepharose<sup>Â®</sup> (PAS) preswollen beads (GE Healthcare, catalog number: 17-1279-02)</li><li>Appropriate primary antibody recognising a specific RBP of interest. Here we use antibody anti-DRBP76/ILF3 (NF90) as an example (Millipore, catalog number: ABF1070)</li><li>A species-appropriate isotype control. For NF90 RIP, we use normal mouse IgG control (Santa Cruz Biotechnology, catalog number: sc-2025)</li><li>DNase I (RNase-free) (Thermo Fisher Scientific, catalog number: AM2222)</li><li>RiboLock RNase inhibitor (40 U/Î¼l) (Thermo Fisher Scientific, catalog number: EO0381)</li><li>Random primers (100 Î¼M) (Thermo Fisher Scientific, catalog number: SO142)</li><li>dNTP mix (10 mM each) (Thermo Fisher Scientific, catalog number: R0192)</li><li>Maxima<sup>TM</sup> reverse transcriptase (Thermo Fisher Scientific, catalog number: EP0741) and 5x RT buffer (provided with Maxima Reverse Transcriptase)</li><li>Proteinase K Solution (20 mg/ml) (Thermo Fisher Scientific, catalog number: AM2546)</li><li>Nuclease-free water (Thermo Fisher Scientific, catalog number: AM9930)</li><li>Dulbeccoâs phosphate-buffered saline (DPBS) (Thermo Fisher Scientific, catalog number: 100100-15)</li><li>Halt Protease &amp; Phosphatase inhibitor cocktail (100x) (Thermo Fisher Scientific, catalog number: 78442)</li><li>TRIzol<sup>TM</sup> (Thermo Fisher Scientific, catalog number: AM9738)</li><li>DTT (Sigma-Aldrich, catalog number: 43815)</li><li>UltraPure 1 M Tris-HCl (pH 7.5) (Thermo Fisher Scientific, catalog number: 15567-027)</li><li>1 M KCl solution (Sigma-Aldrich, catalog number: 60142)</li><li>1 M MgCl<sub>2</sub> solution (Sigma-Aldrich, catalog number: M1028)</li><li>0.5 M EDTA (Thermo Fisher Scientific, catalog number AM9261)</li><li>5 M NaCl solution (Sigma-Aldrich, catalog number: 71386)</li><li>20% SDS solution (Sigma-Aldrich, catalog number: 05030)</li><li>GlycoBlue<sup>TM</sup> (15 mg/ml) (Thermo Fisher Scientific, catalog number: AM9515)</li><li>KAPA SYBR<sup>Â®</sup> FAST ABI prism 2x qPCR master mix (Kapa Biosystems, catalog number: KK4605), or SYBR Green from other vendors</li><li>Nonidet<sup>TM</sup> P-40 (IGEPAL<sup>Â®</sup> CA-630, Sigma, catalog number: I8896)</li><li>Chloroform (Sigma-Aldrich, catalog number: C2432)</li><li>Isopropanol (Sigma-Aldrich, catalog number: I9516)</li><li>Ethanol (Sigma-Aldrich, catalog number: 51976)</li><li>Bio-Rad Protein Assay Dye Reagent Concentrate (for Bradford assay) (Bio-Rad, catalog number: 5000006)</li><li>Polysome extraction buffer (PEB) (see Recipes)</li><li>NT2 buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti31034">### Equipment</p><p class="ppzz"><em>Note: Other analogous equipment can be used for this protocol.</em> </p><ol class="npwtwo"><li>PCR strip tube rotor, mini centrifuge C1201 (Denville Scientific, catalog number: C1201-S [1000806])</li><li>NanoDrop<sup>TM</sup> One spectrophotometer (Thermo Fisher Scientific, catalog number: ND-ONE-W)</li><li>Eppendorf Thermomixer<sup>Â®</sup> R (Eppendorf, catalog number: 022670581)</li><li>Refrigerated centrifuge (Eppendorf, model: 5430R)</li><li>SmartSpec<sup>TM</sup> Plus (Bio-Rad Laboratories, catalog number: 1702525) or another spectrophotometer with 595 nm wavelength</li><li>Tube Revolver/Rotator (Thermo Fisher Scientific, catalog number: 88881001)</li><li>Veriti<sup>TM</sup> 96-well thermal cycler (Thermo Fisher Scientific, catalog number 4375786)</li><li>QuantStudio<sup>TM</sup> 5 Real-Time PCR System, 384-well (Thermo Fisher Scientific, catalog number: A28140)</li><li>Cell culture hoods and CO<sub>2</sub> Incubator for cell culture<br/></li></ol><p class="pptt" id="biaoti31035">### Procedure</p><p class="ppzz"><em>Note: Please be sure to optimize your conditions. Empirical tests are required to optimize RIP conditions and to obtain the optimal signal-to-noise ratio results, meaning that target RNAs are enriched in the specific RBP IP compared to the IgG IP control while nonspecific, non-target RNAs are equally present in both the IgG and RBP IP samples.</em><br/><br/><em style="font-weight:bold;">Day 1<br/></em> </p><ol class="npwone"><li>Antibody coating of Protein A Sepharose (PAS) beads<ol class="npwtwo"><li>Use 60 Î¼l of PAS beads for each reaction.</li><li>Wash the beads 1 time with 1 ml of ice-cold of NT2 by inverting the tube a few times, centrifuge for 2 min at 2,000 <em>x g</em>, 4 Â°C and resuspend the beads in 60 Î¼l NT2 buffer.</li><li>Add 5 Î¼g of the antibody of interest (anti-NF90 in this case) or control IgG (normal mouse IgG in this case) to 60 Î¼l of PAS beads.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Be sure to match the antibody of interest with a species-compatible control IgG. The isotype control is used to determine the specific versus the non-specific signals (RNAs) which are immunoprecipitated due to the immunoglobulin isotype rather than the specific RBP.</em> </li><li><em>The amount of antibody required for IP will depend on the antibody quality and the RBP analyzed. The optimal amount of antibody should be tested beforehand by doing IP with this protocol using 2, 5, 10 and 20 Î¼g Ab.</em> </li></ol></li><li>Add 200 Î¼l NT2 buffer.</li><li>Bind overnight on rotator at 4 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of cultured cells<br/>Prepare cells as needed. For THP-1, the day before RIP, the cells should be plated at a concentration of 0.5 x 10<sup>6</sup>/ml. For two IP reactions (IgG control IP and RBP IP), a total of 1 x 10<sup>7</sup> cells is needed.<br/><em>Note: For adherent cells, one 10-cm culture dish is typically enough for two IP.</em><br/><br/><p class="ppzz" style="margin-left:-20px;"><em style="font-weight:bold;">Day 2<br/></em> </p></li><li>Preparation of the Antibody coated PAS beads<ol class="npwtwo"><li>Add 1 ml of ice-cold NT2 and wash the beads by inverting the tube a few times, centrifuge for 2 min at 2,000 <em>x g</em>, 4 Â°C, and repeat once.</li><li>After the last spin, remove excess NT2 buffer and resuspend the beads in 50 Î¼l NT2 buffer. The PAS/Ab beads are now ready to be used, keep on ice.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of Ribonucleoprotein (RNP) lysate from cultured cells<ol class="npwtwo"><li>Collect the volume of THP-1 cells necessary to have 1 x 10<sup>7</sup> cells for the two IP reactions and centrifuge for 5 min at 1,000 <em>x g</em>, 25 Â°C.</li><li>Wash with PBS and centrifuge again for 5 min at 1,000 <em>x g</em>, 25 Â°C.<br/><em>Note: It is best to use fresh lysate immediately.</em> </li><li>Loosen the final cell pellet by gently flicking the tube and add 1 ml of ice-cold PEB buffer supplemented with RNase inhibitors and protease inhibitors.</li><li>Mix by pipetting a few times (do not vortex), place on ice for 10 min.</li><li>Spin for 15 min at 10,000 <em>x g</em>, 4 Â°C. Transfer supernatant (RNP lysate) to fresh microfuge tubes.</li><li>(Optional step) Preclearing (not necessary for RIP followed by RT-PCR).<ol class="npwthree"><li>Preclear the supernatant with 10 Î¼g of IgG1 antibody control for 30 min at 4 Â°C with rotation.</li><li>Add 50 Î¼l PAS, incubate for 30 min at 4 Â°C with rotation.</li><li>Spin down at 2,000 <em>x g</em> for 2 min at 4 Â°C. Save supernatant in a fresh tube. This is your pre-cleared lysate.</li></ol></li><li>Measure protein concentration of the lysate by the Bradford assay. A typical concentration when using these cells is ~5 Î¼g/Î¼l. You will need 500-1,000 Î¼g of lysate for each RIP.<br/><br/></li></ol></li><li>Immunoprecipitation of RNPs<ol class="npwtwo"><li>Aliquot the precoated PAS/Ab (around 50 Î¼l) into microfuge tubes.</li><li>Prepare a master mix using the following additives for each reaction:<br/>300 Î¼l NT2 buffer<br/>10 Î¼l 0.1 M DTT (do not add the DTT to the pellet directly as this will reduce the antibody and the IP will not work)<br/>10 Î¼l RiboLock<br/>33 Î¼l 0.5 M EDTA</li><li>Add 500 Î¼g RNP lysates.<br/><em>Note: The amount of lysate required for IP will depend on the RBP, the cell type, and the antibody used. Optimal amounts of lysates should be determined beforehand by testing IP with 250, 500, and 1,000 Î¼g of material.</em> </li><li>Add NT2 buffer to 1 ml.</li><li>Incubate for 1-2 h at 4 Â°C with rotation.</li><li>Spin down at 2,000 <em>x g</em>, 2 min, 4 Â°C. Discard supernatant.</li><li>Wash pellet 5 times* each with 1 ml of ice-cold NT2 buffer. Spin between washes: 2,000 <em>x g</em>, 2 min, 4 Â°C.<br/><em>Note: (Optional) Take 100 Î¼l of beads during the last wash to verify the IP by Western blot analysis (more details below in Notes).</em> </li><li>After the last wash, gently add 100 Î¼l of NT2 buffer containing 5 Î¼l DNase I (2 U/Î¼l). Keep at 37 Â°C for 10 min.<br/><em>Note: It is critical not to resuspend the beads harshlyâsimply eject the contents of the pipettor and place at 37 Â°C swirling delicately. It is important not to shake at this step!</em> </li><li>Add 1 ml of NT2 buffer and spin at 2,000 <em>x g</em>, 2 min, discard supernatant.</li><li>Make a master mix using the following reagents for each reaction: 100 Î¼l NT2, 2.5 Î¼l of Proteinase K (20 mg/ml), 1 Î¼l 10% SDS. Add 100 Î¼l of this master mix to each PAS pellet.</li><li>Incubate at 55 Â°C for 20 min, with gentle mixing.</li><li>Spin at 2,000 <em>x g</em>, 2 min, collect the supernatant (~100 Î¼l) to a fresh tube.</li><li>To the beads, add 200 Î¼l NT2 buffer, spin at 2,000 <em>x g</em>, 2 min, collect the supernatant (~200 Î¼l) and combine with the supernatant above. Discard beads.</li><li>To the combined supernatants (100 Î¼l plus 200 Î¼l), add 700 Î¼l of Trizol reagent.</li><li>Vortex for 1 min at 25 Â°C.</li><li>Extract the RNA according to Trizol manufacturerâs instruction.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA extraction and RT-qPCR analysis<br/><ol class="npwtwo"><li>Add 200 Î¼l chloroform to each sample.</li><li>Shake the tubes vigorously by hand for 10 s then centrifuge at 13,000 <em>x g</em> for 15 min at 4 Â°C.</li><li>Transfer the upper aqueous layer to a fresh microfuge tube containing 1 ml isopropanol and 1 Î¼l of Glycoblue.<br/><em>Note: The upper aqueous layer is generally 70% of the initial volume of Trizol.</em> </li><li>Precipitate the RNA by incubating for 10 min at 25 Â°C followed by centrifugation at 13,000 <em>x g</em> for 15 min at 4 Â°C.</li><li>Discard the supernatant and wash the pellet once with 1 ml ice-cold 70% ethanol.</li><li>Centrifuge at 13,000 <em>x g</em> for 5 min at 4 Â°C and discard the supernatant.</li><li>Air dry RNA pellet for 5-10 min at 25 Â°C.</li><li>Dissolve RNA in 20 Î¼l nuclease-free water.<br/><em>Note: Do not measure OD<sub>260</sub>, as the concentration is very low and this will probably waste most of your sample. Just use 5-10 Î¼l directly to prepare your cDNA.</em> </li><li>Use 5 Î¼l of the RNA solution for reverse transcription by preparing a 20-Î¼l reaction in a 0.2 ml PCR tube containing 5 Î¼l of prepared RNA, 1 Î¼l random primers, 4 Î¼l 5x RT buffer, 1 Î¼l RiboLock, 1 Î¼l dNTP mix and 1 Î¼l Maxima reverse transcriptase. Bring to 20 Î¼l with Nuclease-free water.</li><li>Mix and centrifuge for a few seconds.</li><li>For the cDNA synthesis, use these thermal cycler parameters: 25 Â°C for 10 min, 50 Â°C for 30 min and 5 min at 85 Â°C (the manufacturerâs instructions for Maxima reverse transcriptase).</li><li>Dilute the cDNAs with nuclease-free water to 100 Î¼l final volume (1:5 dilution).</li><li>Prepare 10 Î¼l qPCR reactions in a 384-well plate, using the following mix: 5 Î¼l 2x SYBR Green PCR mix, 1 Î¼l cDNA, and 4 Î¼l primer mix (forward and reverse primers). Use specific primers for mRNAs of interest as well as for 1 or 2 mRNAs encoding housekeeping proteins (<em>e.g.</em>, HSP90, <em>ACTB</em>, <em>UBC</em> or <em>GAPDH</em> mRNAs).</li><li>Cover the plate with optical adhesive film and centrifuge the plate for 10 s using Plate Spinner to settle the reactions at the bottom of the wells.</li><li>Use a QuantStudio 5 Real-Time PCR System for qPCR with a cycle set up of 3 min at 95 Â°C and 40 cycles of 5 s at 95 Â°C plus 20 s at 60 Â°C. When using a primer set for the first time, analyze the dissociation curves to verify that the primer set is optimal in these conditions.<br/></li></ol></li></ol><p class="pptt" id="biaoti31036">### Data analysis</p><p class="ppzz">As mentioned above, RNAs isolated with this RIP protocol can be analyzed by several molecular methods including quantitative RT-PCR (if the RNA binding targets of the RBP are known), microarray, or RNA-seq analyses (for an in-depth unbiased approach). Here we have presented a RIP protocol followed by RT-qPCR analysis (Figure 1A).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200109/20200109032647_6428.jpg" width="154"/><br/><strong>Figure 1. Ribonucleoprotein Immunoprecipitation (RIP) Analysis.</strong> A. Schematic of the ribonucleoprotein immunoprecipitation (RIP) assay protocol. B. The association of the RBP NF90 (used here as example) with <em>ACTB</em> and <em>BAFF</em> mRNAs was tested by RIP analysis using anti-NF90 antibody. Following RNA extraction, the abundance of <em>BAFF</em> and <em>ACTB</em> mRNAs in NF90 IP and IgG IP control samples was assessed by RT-qPCR analysis using mRNA-specific primers (<em>ACTB</em>: CATGTACGTTGCTATCCAGGC and CTCCTTAATGTCACGCACGAT; <em>BAFF</em>: CACAATTCAAAGGGGCAGTAA and ACTGAAAAGGAGGGAGTGCAT; <em>UBC</em>: ATTTGGGTCGCGGTTCTTG and TGCCTTGACATTCTCGATGGT). These results were normalized to the levels of <em>UBC</em> mRNA in each sample, and then plotted as the enrichment of mRNAs in the NF90 IP relative to the <em>BAFF</em> and <em>ACTB</em> mRNA levels observed in the IgG IP samples. C. After IP using anti-NF90 or IgG antibodies, the presence of NF90 in the IP material was confirmed by Western blot analysis.<br/><br/>  Analysis of specific enriched mRNAs should be normalized with a proper âbackgroundâ control RNA, a transcript that does not associate specifically with the RBP of interest and allows the user to monitor the evenness of sample input. It is customary to use mRNAs encoding abundant housekeeping proteins, such <em>UBC</em>, <em>GAPDH</em>, or <em>TUB</em> mRNAs, as the nonspecific transcripts. The identification of a good background control RNA, which should yield similar Ct values in both the IgG control and RBP IP when sample input is even, is fundamental for good-quality results. In the example shown in Figure 1B we used <em>UBC</em> mRNA for background control.<br/>  To calculate the fold enrichment of each RIP reaction from qPCR data, first normalize the Ct value of the target RNA to the Ct of the control (âhousekeepingâ) mRNA. The conventional representation of the data is by displaying âfold enrichmentâ of RNAs in RBP IP relative to IgG IP (the latter, usually displayed as â1â, represents the background mRNA level in the IgG isotype control). To calculate fold enrichments, the normalized RIP fraction value (ÎCt of target RNA normalized to the âhousekeepingâ mRNA) is normalized to nonspecific background (the similar ÎCt calculation of the IgG fraction), to yield the ÎÎCt value. The linear conversion of this ÎÎCt renders the fold enrichment, which is plotted for <em>BAFF</em> mRNA binding to NF90 in our example (Figure 1B). Fold enrichment was also calculated for a negative control, a transcript (<em>ACTB</em> mRNA in this case) that does not bind NF90, and this control mRNA was found not to be significantly enriched (Figure 1B).</p><p class="pptt" id="biaoti31037">### Notes</p><p class="ppzz">(*) Take 100 Î¼l of beads during last wash (Step E7) to verify the IP by Western blot analysis, centrifuge briefly, remove the supernatant, and add 30 Î¼l SDS loading buffer to the beads. Boil at 100 Â°C for 10 min and load an aliquot on SDS-PAGE gel for Western blot analysis. If the IP is successful, a stronger signal will be visible in the lane in which the RBP of interest was immunoprecipitated, as compared to the IgG control (Figure 1C).</p><p class="pptt" id="biaoti31038">### Recipes</p><p class="ppzz"><em>Note: It is critical to prepare all the solutions in RNase-free conditions.</em> </p><ol class="npwtwo"><li>Polysome extraction buffer (PEB)<br/>20 mM Tris-HCl, pH 7.5<br/>100 mM KCl<br/>5 mM MgCl<sub>2</sub><br/>0.5% Nonidet P-40<br/>Add fresh protease and RNase inhibitors each time</li><li>NT2 buffer<br/>50 mM Tris HCl, pH 7.5 <br/>150 mM NaCl<br/>1 mM MgCl<sub>2</sub><br/>0.05% Nonidet P-40 <br/><em>Note: Please keep the NT2 uncontaminated and RNase-free. If the RIP does not work, it is a good bet that NT2 needs to be made fresh.</em><br/></li></ol><p class="pptt" id="biaoti31039">### Acknowledgments</p><p class="ppzz">JM, MG, and MLI were supported by the NIA IRP, NIH. MLI was additionally supported by the National Research CouncilâInstitute for Genetic and Biomedical Research (CNR-IRGB), Italy. This protocol was adapted from articles previously published by the laboratory of JD Keene. The protocol was tested and optimized by different researchers in the Gorospe laboratory, NIA IRP, NIH. We thank D Tsitsipatis and G Delogu for careful review of the manuscript.</p><p class="pptt" id="biaoti31040">### Competing interests</p><p class="ppzz">No conflicts of interest or competing interests to declare.</p><p class="pptt" id="biaoti31041">### References</p><ol class="npwtwok"><li>Cipriano, A. and Ballarino, M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29560353" target="_blank">The everâevolving concept of the gene: The use of RNA/protein experimental techniques to understand genome functions.</a> <em>Front Mol Biosci</em> 5: 20.</li><li>Glisovic, T., Bachorik, J. L., Yong, J. and Dreyfuss, G. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18342629" target="_blank">RNA-binding proteins and post-transcriptional gene regulation.</a> <em>FEBS Lett</em> 582(14): 1977-1986.</li><li>Idda, M. L. Lodde, V. McClusky, W. G., Martindale, J. L., Yang, X., Munk, R., Steri, M., OrrÃ¹, V. Mulas, A. Cucca, F., Abdelmohsen, K. and Gorospe M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30272251" target="_blank">Cooperative translational control of polymorphic BAFF by NF90 and miR-15a.</a> <em>Nucleic Acids Res</em> 46(22): 12040-12051.</li><li>Kang, M. J., Abdelmohsen, K., Hutchison, E. R., Mitchell, S. J., Grammatikakis, I., Guo, R., Noh, J. H., Martindale, J. L., Yang, X., Lee, E. K., Faghihi, M. A., Wahlestedt, C., Troncoso, J. C., Pletnikova, O., Perrone-Bizzozero, N., Resnick, S. M., de Cabo, R., Mattson, M. P. and Gorospe M. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24857657" target="_blank">HuD regulates coding and noncoding RNA to induce APPâAÎ² processing.</a> <em>Cell Rep</em> 7(5):1401-1409.</li><li>Keene, J. D., Komisarow, J. M. and Friedersdorf, M. B. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17406249" target="_blank">RIP-Chip: the isolation and identification of mRNAs, microRNAs and protein components of ribonucleoprotein complexes from cell extracts.</a> <em>Nat Protoc</em> 1(1): 302-307.</li><li>Lee, F. C. Y. and Ule, J. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29395060" target="_blank">Advances in CLIP technologies for studies of proteinâRNA interactions.</a> <em>Mol Cell </em>69: 354-369.</li><li>Lin, C. and Miles, W. O. (2019). <a href="https://academic.oup.com/nar/article/47/11/5490/5488019" target="_blank">Beyond CLIP: advances and opportunities to measure RBP-RNA and RNA-RNA interactions.</a> <em>Nucleic Acids Res</em> 47(11):5490-5501.</li><li>Licatalosi, D. D., Ye, X. and Jankowsky, E. (2019). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/wrna.1565" target="_blank">Approaches for measuring the dynamics of RNA-protein interactions.</a> <em>Wiley Interdiscip Rev RNA</em> 11(1): e1565.</li><li>McHugh, C. A., Russell, P. and Guttman M. (2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24467948" target="_blank">Methods for comprehensive experimental identification of RNA-protein interactions.</a> <em>Genome Biology</em> 15(1): 203.</li><li>Nicholson, C. O., Friedersdorf, M. and Keene J. D. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27742911" target="_blank">Quantifying RNA binding sites transcriptome-wide using DO-RIP-seq.</a> <em>RNA</em> 23(1):32-46.</li><li>Pereira, B., Billaud, M. and Almeida, R. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28718405" target="_blank">RNA-binding proteins in cancer: old players and new actors.</a> <em>Trends Cancer</em> 3(7): 506-528.</li><li>Ravanidis, S., Kattan, F. G. and Doxakis, E. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30081499" target="_blank">Unraveling the pathways to neuronal homeostasis and disease: mechanistic insights into the role of RNA-binding proteins and associated factors.</a> <em>Int J Mol Sci</em> 19(8): pii: E2280.</li><li>Tenenbaum, S. A., Carson, C. C., Lager, P. J. and Keene J. D. (2000). <a href="https://www.pnas.org/content/97/26/14085" target="_blank">Identifying mRNA subsets in messenger ribonucleoprotein complexes by using cDNA arrays.</a> <em>PNAS</em> 97(26): 14085-14090.</li><li>Wheeler, E. C., Van Nostrand E. L. and Yeo G. W. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28853213" target="_blank">Advances and challenges in the detection of transcriptome-wide protein-RNA interactions.</a> <em>Wiley Interdiscip Rev RNA</em> 9(1): e1436.</li><li>Yoshinaga, M. and Takeuchi, O. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31449478" target="_blank">RNA binding proteins in the control of autoimmune diseases.</a> <em>Immunol Med</em> 42(2): 53-64.</li><li>Zhao, J., Ohsumi, T. K., Kung, J. T., Ogawa, Y., Grau, D. J., Sarma K., Song J. J., Kingston R. E., Borowsky M. and Lee J. T. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21172659" target="_blank">Genome-wide identification of polycomb-associated RNAs by RIP-seq.</a> <em>Mol Cell</em> 40(6): 939-953.<br/></li></ol>