<body>
            <h1>Sulfatase Assay to Determine Influence of Plants on Microbial Activity in Soil</h1>
            <p><strong>Authors:</strong> Anna  Koprivova, Achim  Schmalenberger, Stanislav Kopriva</p>
            <h2>Abstract</h2>
            <p>Sulfatase activity is often used as a measure of the activity of soil microorganisms. It is thus a suitable tool to investigate the response of microbes to plants. Here we present a method to determine the influence of various <em>Arabidopsis</em> genotypes on the function of soil microbiota using the sulfatase as a quantitative measure. We grew the plants in soil/sand mix under control conditions and measured the sulfatase activity in soil using a spectrophotometric determination of the product. This protocol can be used to test the contribution of individual genes to control of microbiome assembly through analysis of mutants as well as the influence of environment on plant-microbe interactions.</p>
            <p><strong>Keywords:</strong> <em>Arabidopsis</em>, Plant-microbe interactions, Soil, Microbiome, Organic sulfur, Sulfatase, Bacteria</p>
            <h2>Background</h2><p>Plants in their natural environment interact with plethora of microorganisms, pathogenic as well as beneficial. Many microorganisms are beneficial to plants, <em>e.g.</em>, by improving their immunity or their nutrition (Kertesz and Mirleau, 2004; Jacoby <em>et al.</em>, 2017; Stringlis <em>et al.</em>, 2018). Indeed, the role of bacteria in plant sulfur nutrition has long been recognized (Kertesz and Mirleau, 2004). Sulfur is present in soil mainly bound to organic compounds and thus not available to plants. However, bacteria and fungi can metabolize such organosulfur compounds and release the sulfate group to the rhizosphere, where it can be utilized by plants, and improve so plant sulfur nutrition (Gahan and Schmalenberger, 2014). One of the enzymes catalyzing such reactions is sulfatase. This enzyme, catalyzing the reaction X-O-SO<sub>3</sub> + H<sub>2</sub>O â X-OH + HSO<sub>4</sub><sup>-</sup>, is found in many organisms including bacteria, fungi, and humans, but is not present in plants (Gunal <em>et al.</em>, 2019). Sulfatase is induced by sulfate limitation and is the basis of plant growth promoting effects of some bacteria (Kertesz and Mirleau, 2004). Sulfatase activity in soil reflects the activity of microbial communities and can be used for estimation of soil health after various treatments (Tejada <em>et al.</em>, 2006; Zaborowska <em>et al.</em>, 2018) alongside activities of, <em>e.g.</em>, Î²-glucosidase, cellobiohydrolase, chitinase, leucine aminopeptidase phosphatase, and tyrosine aminopeptidase (Maharjan <em>et al.</em>, 2017).<br/> <br/>However, the sulfatase can be used also as a tool to study plant-microbe interactions. The role of the plant microbiome in improving plant performance and fitness has been increasingly recognized and great progress in the understanding of the assembly of plant microbiome has been achieved (Bulgarelli <em>et al.</em>, 2013; Bai <em>et al.</em>, 2015). Clearly, plants shape their microbiome composition, even though the mechanisms are largely unknown. However, most microbiome studies are based on DNA sequencing and therefore taxonomic description of the microbiome composition (Jacoby <em>et al.</em>, 2017). We have used sulfatase activity to identify mechanisms by which plants shape their microbiome (Koprivova <em>et al.</em>, 2019). Through analyzing the effects of <em>Arabidopsis</em> accessions on sulfatase in soil and using the activity for genome-wide association mapping we revealed an important role of the phytoalexin camalexin in the interactions between plant roots and rhizosphere bacteria (Koprivova <em>et al.</em>, 2019). The sulfatase assay, which was adapted from (Margesin <em>et al.</em>, 2014), proved to be an excellent tool to assess the microbiome activity and the effect of plant genotype on such activity. Therefore, here we present a protocol not only for the core enzymatic activity but also for a full assessment of the effects of <em>Arabidopsis</em> genotypes (accessions or mutants) on microbiome function.</p>
        </body>

<p class="pptt" id="biaoti31056">## Materials and Reagents</p><ol class="npwtwok"><li>Tape</li><li>0.5 ml Eppendorf tubes<br/></li><li>Toothpicks</li><li>1.5 ml cuvettes</li><li>Plastic trays, Plant Pots Direct, Heavyweight full seed tray (no holes), catalog number: 2012137PT (Figure 1)</li><li>Plastic inserts (Plant Pots Direct, Seed Tray Inserts 40, catalog number: 2012111PT) (Figure 1)</li><li>Plastic Petri dishes (Sarstedt, catalog number: 821.473)</li><li>Pipette tips (Sarstedt)</li><li>2 ml plastic tubes (Sarstedt, catalog number: 72.695.500)</li><li>Plastic rack for 2 ml tubes, Eppendorf</li><li>Plastic cuvettes (Brand, catalog number: 759115)</li><li>Seeds of <em>Arabidopsis thaliana</em> (can be obtained, <em>e.g.</em>, from NASC <em>Arabidopsis</em> Stock Centre, <a href="http://arabidopsis.info/BasicForm" target="_blank">http://arabidopsis.info/BasicForm</a>)</li><li>Sand, Quarzwerke (Frechen, Germany)</li><li>Soil (<em>e.g.</em>, CAS11 soil, Bulgarelli <em>et al.</em>, 2012)</li><li>Murashige Skoog (MS) medium (Duchefa Biochemie, catalog number: MO222.0025), stored at RT</li><li>Sucrose (Sigma, catalog number: S7903-1KG), stored at RT</li><li>Potassium-4-nitrophenyl sulfate (Sigma, catalog number: N3877-1G), stored at -20 Â°C</li><li>P-nitrophenol (Merck, catalog number: 48549), stored at -20 Â°C</li><li>Calcium chloride dihydrate (Sigma, catalog number C3881-1KG), stored at RT</li><li>Sodium hydroxide (NaOH) (Sigma, catalog number: 71687-500G), stored at RT</li><li>Sodium acetate trihydrate (Sigma, catalog number: S8625-500G), stored at RT</li><li>Calcium nitrate tetrahydrate (Ca(NO<sub>3</sub>)<sub>2</sub>Â·4H<sub>2</sub>O) (Merck, catalog number: C1396), stored at RT</li><li>Potassium nitrate (KNO<sub>3</sub>) (Merck catalog number: P8394), stored at RT</li><li>Potassium dihydrogenphosphate (KH<sub>2</sub>PO<sub>4</sub>) (Merck, catalog number: 1.04873), stored at RT</li><li>Ferric EDTA (Fe-EDTA) (Merck, catalogue number: E6760), stored at RT</li><li>Magnesium chloride hexahydrate (MgCl<sub>2</sub>Â·6H<sub>2</sub>O) (Merck, catalog number: M2670), stored at RT</li><li>Liquid nitrogen</li><li>Glacial acetic acid (Merck, catalog number: A6283), stored at RT</li><li>Sodium hypochlorite solution (NaClO), 12%, Cl (Roth, catalog number: 9062.4)</li><li>Hydrochloric acid (HCl) 37% (Merck KGaA, catalog number: 1.00317.1000), stored at RT</li><li>Agarose (Sigma, catalog number: A9539-500G), stored at RT</li><li>Toluene (Sigma, catalog number: 34866-100 ml), stored under fume hood at RT</li><li>Modified Long Ashton solution (see Recipes)</li><li>0.5 M Acetate buffer (see Recipes)</li><li>0.005 M p-nitrophenyl solution (see Recipes)</li><li>0.5 M calcium chloride solution (see Recipes)</li><li>0.5 M sodium hydroxide solution (see Recipes) </li><li>Standard p-nitrophenol solution 10 mM (100 ml) (see Recipes)</li><li>Half strength MS medium with sucrose (1 L) (see Recipes)<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200108/20200108012751_1971.jpg" width="277"/><br/><strong>Figure 1. Photos of (A) plastic trays and (B) inserts used for plant growth</strong><br/></li></ol><p class="pptt" id="biaoti31057">### Equipment</p><ol class="npwtwok"><li>Glass beaker 250 ml</li><li>Forceps</li><li>Pipettes (Eppendorf)</li><li>Balances (Sartorius)</li><li>Glass beaker</li><li>Sanyo growth chamber, 10 h light/14 h darkness, 22 Â°C</li><li>Rotating shaker, LTF (Labortechnik, Intelli-Mixer, RM-2L)</li><li>Vortex (LMS, model: VTX-3000L)</li><li>Incubator for 37 Â°C (Thermo Scientific, Heratherm Incubator)</li><li>Centrifuge (Eppendorf, 5424)</li><li>Spectrophotometer (Eppendorf)</li><li>Fume hood</li><li>Computer</li><li>Desiccator<br/></li></ol><p class="pptt" id="biaoti31058">### Procedure</p><ol class="npwone"><li><em>Arabidopsis thaliana</em> seed sterilization<ol class="npwtwo"><li>Place small portion of seeds (<em>ca.</em> 10 Âµl) into 0.5 ml Eppendorf tubes.</li><li>Place open tubes into tube rack inside of desiccator (Figure 2).</li><li>Add 125 ml of sodium hypochlorite solution into 250 ml glass beaker and place near the seeds.</li><li>Add 2.5 ml of concentrated HCl into the liquid, which forms chlorine gas, and quickly close the desiccator lid.</li><li>Sterilize seeds for 3 h.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200108/20200108012823_8143.jpg" width="161"/><br/><strong>Figure 2. Photos of (A) open or (B) closed desiccator for seed sterilization</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Initial plates preparation<ol class="npwtwo"><li>Autoclave half-strength MS medium with 0.8% agarose and 0.5% sucrose and pour it into round Petri dishes, let it set.</li><li>Carefully place sterile <em>Arabidopsis</em> seeds onto agar, using sterile toothpicks, approx. 1 seed per cm<sup>2</sup>.</li><li>Seal the plates with tape.</li><li>Place the plates into fridge for 2-3 days for stratification.</li><li>Place plates into a plant growth cabinet for 9 days (10 h light/14 h dark; 22 Â°C, 100 ÂµE m<sup>-2</sup> s<sup>-1</sup>).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of trays<ol class="npwtwo"><li>Mix soil with sterile (autoclaved) sand 1:9 (V/V).</li><li>Place plastic insert into the tray.</li><li>Fill inserts with soil-sand mix.</li><li>Water slightly.</li><li>Using forceps carefully transfer one seedling per insert, covering roots lightly.</li><li>Water a bit more, to keep soil moist.</li><li>Cover tray with the lid.</li><li>Place trays into plant growth cabinet (10 h light/14 h dark; 22 Â°C, 100 ÂµE m<sup>-2</sup> s<sup>-1</sup>) for 2 weeks, uncover it after 3 days, and water daily with modified Long Ashton solution.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200108/20200108012856_7728.jpg" width="248"/><br/><strong>Figure 3. Photos of (A) Petri dish with seedlings before transfer and (B) trays with <em>Arabidopsis</em> ecotypes ready for material collection</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Collection of samples for sulfatase activity<ol class="npwtwo"><li>Carefully remove plants from soil.</li><li>Into a 2 ml microcentrifuge tube collect about 1 g of the soil-sand mix, which was closest to the roots of the plant, <em>i.e.</em>, rhizosphere, record fresh weight.</li><li>Collect at least 2 samples per plant.</li><li>Freeze in liquid nitrogen.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sulfatase activity measurement<ol class="npwtwo"><li>Defrost the samples in a rack.</li><li>Add 400 Âµl of 0.5 M acetate buffer.</li><li>Vortex each sample for at least 5 s.</li><li>Under fume hood add 25 Âµl of toluene.</li><li>Close tubes and vortex for at least 5 s.</li><li>Place tubes into rotating rack for 6 min at 100 rpm.</li><li>Add 100 Âµl of p-nitrophenyl sulfate solution under fume hood.</li><li>Vortex each sample for 10 s.</li><li>Place tubes for additional vigorous shaking for 5 min in an Eppendorf shaker at 1,000 rpm.</li><li>Place rack with the tubes into 37 Â°C incubator for 1 h, mixing the whole rack every 10 min by reversing several times.</li><li>To stop the reaction under fume hood add 100 Âµl of 0.5 M CaCl<sub>2</sub> solution and 400 Âµl of 0.5 M sodium hydroxide solution.</li><li>Vortex each sample for 10 s.</li><li>Centrifuge tubes at RT at maximum speed for 20 min.</li><li>Under fume hood carefully transfer the supernatant into plastic 1.5 ml cuvettes.</li><li>Measure absorption at 400 nm. Use water as blank.</li><li>Prepare standards by diluting 0, 20, 40, 80, 120, 160, and 200 Âµl of 10 mM p-nitrophenol standard to 1 ml H<sub>2</sub>O and measure in the same way.</li><li>Calculate the p-nitrophenol content of the samples from a calibration curve (standards: 0, 200, 400, 800, 1,200, 1,600, and 2,000 nmol ml<sup>-1</sup> p-nitrophenol).</li><li>Using recorded FW data and incubation time of 1 h calculate sulfatase activity in nmol g FW<sup>-1</sup> h<sup>-1</sup>.</li></ol></li></ol><p class="pptt" id="biaoti31059">### Data analysis</p><p class="ppzz">The assay determines the end concentration of the sulfatase product, p-nitrophenol, using a calibration curve (Figure 1) and the activity can be calculated from the weight of soil and time of the assay. All calculations can be easily performed in standard office software, <em>e.g.</em>, Excel (Table 1). Five biological replicates with two technical replicates each should be used for each plant genotype.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200108/20200108012934_9772.jpg" width="170"/><br/><strong>Figure 4. Calibration curve for measurement of sulfatase activity.</strong> Standard p-nitrophenol solution was diluted by water to contain 0-2,000 nmol per ml and OD<sub>400</sub> was measured.<br/><br/><strong>Table 1. Example of calculation of sulfatase activity</strong><br/><img alt="" class="layerphoto" height="342" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200108/20200108020149_7433.jpg" width="600"/><br/><sup>1</sup>The soil FW is adjusted from the FW of soil/sand sample and the ratio of soil/sand mix, here 10% soil.</p><p class="pptt" id="biaoti31060">### Notes</p><p class="ppzz">The assay can easily be adapted for different plant species, different sizes of pots, or different soils.</p><p class="pptt" id="biaoti31061">### Recipes</p><ol class="npwtwok"><li>Modified Long Ashton solution<br/>1.5 mM Ca(NO<sub>3</sub>)<sub>2</sub>Â·4H<sub>2</sub>O<br/>1 mM KNO<sub>3</sub><br/>0.75 mM KH<sub>2</sub>PO<sub>4</sub><br/>0.1 mM Fe-EDTA<br/>0.75 mM MgCl<sub>2</sub>Â·6H<sub>2</sub>O<br/>pH 5.7</li><li>0.5 M acetate buffer, pH 5.8 (1 L)<br/>64 g sodium acetate trihydrate<br/>1.7 ml glacial acetic acid<br/>Stored at 4 Â°C</li><li>0.005 M p-nitrophenyl sulfate solution (100 ml)<br/>0.1287 g of p-nitrophenyl dissolved in 100 ml of 0.5 M acetate buffer<br/>Stored at 4 Â°C</li><li>0.5 M calcium chloride solution (1 L)<br/>73.5 g CaCl<sub>2</sub>Â·2H<sub>2</sub>O dissolved in 1 L H<sub>2</sub>O<br/>Stored at RT</li><li>0.5 M sodium hydroxide solution (1 L)<br/>20 g NaOH dissolved in 1 L H<sub>2</sub>O<br/>Stored at RT</li><li>Standard p-nitrophenol solution 10 mM (100 ml)<br/>139 mg p-nitrophenol dissolved in 100 ml H<sub>2</sub>O<br/>Stored at 4 Â°C</li><li>Half strength MS medium with sucrose (1 L)<br/>2.2 g MS medium<br/>5 g sucrose<br/>8 g agarose<br/>pH 5.7 with 1 M KOH<br/></li></ol><p class="pptt" id="biaoti31062">### Acknowledgments</p><p class="ppzz">The sulfatase assay was adapted from Margesin <em>et al.</em>, 2014. SKâs research is funded by the Deutsche Forschungsgemeinschaft (DFG) under Germanyâs Excellence Strategy, EXC-Nummer 2048/1, project 390686111. AS was funded by FP7-Peoople (293429) and the Irish Research Council New Foundation âS-cycleâ.</p><p class="pptt" id="biaoti31063">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti31064">### References</p><ol class="npwtwok"><li>Bai, Y., Muller, D. B., Srinivas, G., Garrido-Oter, R., Potthoff, E., Rott, M., Dombrowski, N., Munch, P. C., Spaepen, S., Remus-Emsermann, M., Huttel, B., McHardy, A. C., Vorholt, J. A. and Schulze-Lefert, P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26633631" target="_blank">Functional overlap of the <em>Arabidopsis</em> leaf and root microbiota.</a> <em>Nature</em> 528(7582): 364-369.</li><li>Bulgarelli, D., Rott, M., Schlaeppi, K., Ver Loren van Themaat, E., Ahmadinejad, N., Assenza, F., Rauf, P., Huettel, B., Reinhardt, R., Schmelzer, E., Peplies, J., Gloeckner, F. O., Amann, R., Eickhorst, T. andSchulze-Lefert, P. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22859207" target="_blank">Revealing structure and assembly cues for <em>Arabidopsis</em> root-inhabiting bacterial microbiota.</a> <em>Nature</em> 488: 91-95.</li><li>Bulgarelli, D., Schlaeppi, K., Spaepen, S., Ver Loren van Themaat, E. and Schulze-Lefert, P. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23373698" target="_blank">Structure and functions of the bacterial microbiota of plants.</a> <em>Annu Rev Plant Biol</em> 64: 807-838.</li><li>Gahan, J. and Schmalenberger, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25566295" target="_blank">The role of bacteria and mycorrhiza in plant sulfur supply.</a> <em>Front Plant Sci</em> 5: 723.</li><li>Gunal, S., Hardman, R., Kopriva, S.and Mueller, J. W. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31270211" target="_blank">Sulfation pathways from red to green.</a> <em>J Biol Chem</em> 294: 12293-12312.</li><li>Jacoby, R., Peukert, M., Succurro, A., Koprivova, A. and Kopriva, S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28974956" target="_blank">The role of soil microorganisms in plant mineral nutrition-current knowledge and future directions.</a> <em>Front Plant Sci</em> 8: 1617.</li><li>Kertesz, M. A. and Mirleau, P. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15181108" target="_blank">The role of soil microbes in plant sulphur nutrition.</a> <em>J Exp Bot</em> 55(404): 1939-1945.</li><li>Koprivova, A., Schuck, S., Jacoby, R. P., Klinkhammer, I., Welter, B., Leson, L., Martyn, A., Nauen, J., Grabenhorst, N., Mandelkow, J. F., Zuccaro, A., Zeier, J. and Kopriva, S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31311863" target="_blank">Root-specific camalexin biosynthesis controls the plant growth-promoting effects of multiple bacterial strains.</a> <em>Proc Natl Acad Sci U S A</em> 116(31): 15735-15744.</li><li>Margesin, R., Minerbi, S. and Schinner, F. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25008018" target="_blank">Long-term monitoring of soil microbiological activities in two forest sites in South tyrol in the italian alps.</a> <em>Microbes Environ</em> 29: 277-285.</li><li>Maharjan, M., Sanaullah, M., Razavi, B. S. and Kuzyakov, Y. (2017). <a href="http://www.sciencedirect.com/science/article/pii/S092913931730063X" target="_blank">Effect of land use and management practices on microbial biomass and enzyme activities in subtropical top-and sub-soils.</a> <em>Appl Soil Ecol</em> 113: 22-28.<br/></li><li>Stringlis, I. A., Yu, K., Feussner, K., de Jonge, R., Van Bentum, S., Van Verk, M. C., Berendsen, R. L., Bakker, P., Feussner, I. and Pieterse, C. M. J. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29686086" target="_blank">MYB72-dependent coumarin exudation shapes root microbiome assembly to promote plant health.</a> <em>Proc Natl Acad Sci U S A</em> 115: E5213-E5222.</li><li>Tejada, M., Hernandez, M. T. and Garcia, C. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16738385" target="_blank">Application of two organic amendments on soil restoration: effects on the soil biological properties.</a> <em>J Environ Qual </em>35: 1010-1017.</li><li>Zaborowska, M., Kucharski, J. and Wyszkowska, J. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30242485" target="_blank">Biochemical and microbiological activity of soil contaminated with o-cresol and biostimulated with Perna canaliculus mussel meal.</a> <em>Environ Monit Assess</em> 190: 602.</li></ol>