<body>
            <h1>Low-cost and Multiplexable Whole mRNA-Seq Library Preparation Method with Oligo-dT Magnetic Beads for Illumina Sequencing Platforms</h1>
            <p><strong>Authors:</strong> Makoto Kashima, Ayumi Deguchi, Ayumi Tezuka, Atsushi J. Nagano</p>
            <h2>Abstract</h2>
            <p>RNA-Seq is a powerful method for transcriptome analysis used in varied field of biology. Although several commercial products and hand-made protocols enable us to prepare RNA-Seq library from total RNA, their cost are still expensive. Here, we established a low-cost and multiplexable whole mRNA-Seq library preparation method for illumine sequencers. In order to reduce cost, we used cost-effective and robust commercial regents with small reaction volumes. This method is a whole mRNA-Seq, which can be applied even to non-model organisms lacking the transcriptome references. In addition, we designed large number of 3â² PCR primer including 8 nucleotides barcode sequences for multiplexing up to three hundreds samples. To summarize, it is possible with this protocol to prepare 96 directional RNA-Seq libraries from purified total RNA in three days and can be pooled for up to three hundred libraries. This is beneficial for large scale transcriptome analysis in many fields of animals and plant biology.</p>
            <p><strong>Keywords:</strong> RNA-Seq, mRNA, Multiplexing, Transcriptome, Illumina</p>
            <h2>Background</h2><p>In the last decade, sequencing cost has been reduced drastically thanks to the advance in massive parallel sequencing technologies (Muir <em>et al.</em>, 2016). On the other hand, the cost at library preparation step stands out. For example, commercial kits for RNA-Seq library preparation such as TruSeq RNA Library Prep Kit (illumine, USA) and NEBNext Ultra RNA Library Prep Kit (NEB, USA) cost around 40-60 dollars per sample. Recently, several studies have developed cost-effective RNA-Seq library preparation methods (Kumar <em>et al.</em>, 2012; Nagano <em>et al.</em>, 2015; Townsley <em>et al.</em>, 2015; Alpern <em>et al.</em>, 2019; Kamitani <em>et al.</em>, 2019). Particularly, 3â² RNA-Seq protocols (Lasy-Seq and BRB-Seq), enable early-pooling of samples resulting in reducing the cost into about two dollar per sample (Alpern <em>et al.</em>, 2019; Kamitani <em>et al.</em>, 2019). While 3â² RNA-Seq is economically superior to whole mRNA-Seq method, whole mRNA-Seq can sequence full-length of RNA. Thus, whole mRNA-Seq is good for detection of splicing variants and novel transcripts. Although our previously-developed protocol of whole mRNA-Seq using rRNA depletion is cheap (Nagano <em>et al.</em>, 2015), it requires many kinds of antisense oligo against rRNA and enzymes for reverse transcription, 2<sup>nd</sup> strand synthesis, end-repair, A-tailing, adapter ligation and library amplification. In this protocol, (1) we replaced rRNA depletion against mRNA-purification using oligo-dT beads in order to reduce initial cost. (2) Enzymes required for end-repair to library amplification were replaced with KAPA Hyper Prep Kit (Roche, Switzerland); this simplification increases reaction efficiency as well as reduce labor. (3) We saved reaction volumes throughout all steps (â¤ 20 Âµl), which saves cost and enables handling even with 384-well plates. The required cost is approximately one-third of the golden standard kits. (4) We designed 300 kinds of 3â² PCR primers with barcode sequences for multiplexing, which enabled us to sequence 300 samples at a time with a cost-effective and ultra-high-throughput sequencer such as NovaSeq.</p>
        </body>

<p class="pptt" id="biaoti34461">## Materials and Reagents</p><ol class="npwtwok"><li>0.2 ml PCR tubes<br/></li><li>Pipette tips</li><li>5Ã Super Script IV buffer (Invitrogen)</li><li>DTT (Invitrogen)</li><li>Dynabeads<sup>Â®</sup> Oligo (dT)25 (Thermo Fisher scientific)</li><li>Total RNA</li><li>Random primer (N)<sub>6</sub> (TaKaRa)</li><li>dNTP (25 mM each) (Promega)</li><li>SuperScript IV (Invitrogen)</li><li>Actinomycin D (1,000 ng/Î¼l) (Nacalai Tesque)</li><li>AMPure XP (Beckman Coulter)</li><li>EtOH</li><li>Nuclease-free water</li><li>10Ã Blue Buffer (Enzymatics)</li><li>RNase H (Enzymatics)</li><li>DNA polymerase I (Enzymatics)</li><li>KAPA Hyper prep kit (KAPA Biosystems)</li><li>Tris-HCl</li><li>LiCl</li><li>EDTA</li><li>DNA glycosylase (UDG) (Enzymatics)</li><li>KAPA HiFi HotStart ReadyMix (2Ã) (KAPA Biosystems)</li><li>KAPA Library Quantification Kit Illumina (KAPA Biosystems)</li><li>Agilent High Sensitivity DNA kit (Agilent Technologies)</li><li>2Ã binding buffer (see Recipes)</li><li>Washing buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34462">### Equipment</p><ol class="npwtwok"><li>MagnaStand YS-model (FastGene)<br/></li><li>Agilent 2100 Bioanalyzer (Agilent Technologies)</li><li>Quantitative PCR instrument</li><li>Pipette</li><li>Thermal cycler<br/></li></ol><p class="pptt" id="biaoti34463">### Procedure</p><p class="ppzz">In this protocol, we used MagnaStand YS-model (FastGene) in nucleotide acid purification with magnetic beads. All experiments were conducted in 0.2 ml PCR tubes. Reaction solutions should be mixed by tapping or inversion. All operations should be conducted under a DNA/RNA-free environment. The overview of this protocol is shown in Figure 1.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617225212_9221.jpg" width="449"/><br/><strong>Figure 1. The protocol flow chart of this protocol. </strong>The approximate time required is shown for each step.<strong><br/><br/></strong> </p><ol class="npwone"><li>mRNA purification<ol class="npwtwo"><li>Wash 15 Î¼l of Dynabeads<sup>Â®</sup> Oligo (dT)25 (Thermo Fisher Scientific) twice each with 50 Î¼l of 2Ã binding buffer (Recipe 1) using Magna Stand for 0.2 ml PCR Tube (FastGene) and then resuspend the beads in 30 Î¼l of 2Ã binding buffer for the later use.<br/>âWashâ indicates:<br/><ol class="npwthree"><li>Put on a magnet, 5 min, remove supernatant.</li><li>Add 2Ã binding buffer and mix by pipetting.</li><li>Repeat.</li></ol></li><li>Denature total RNA (more than 1 Î¼g and RIN &gt; 7.0 is recommended) in 30 Î¼l of distilled water or 10 mM TrisÂ·HCl (pH 8.5) at 65 Â°C for 2 min and then immediately transfer onto ice. All thermal control procedures in this protocol can be processed in a thermal cycler. Then add 30 Î¼l of the washed Dynabeads<sup>Â®</sup> Oligo (dT)25. Mix the mixture well by pipetting and incubate at room temperature for 10 min. And next wash the mixture with 70 Î¼l of washing buffer (Recipe 2) twice using the Magna Stand.<br/>âWashâ indicates:<br/><ol class="npwthree"><li>Put on a magnet, 5 min, remove supernatant.</li><li>Add washing buffer and mix by pipetting.</li><li>Repeat.</li></ol></li><li>Elute the RNA in 30 Î¼l of pre-warmed distilled water at 80 Â°C for 5 min and then immediately chill on ice. For a second round of polyA-selection, add 30 Î¼l of 2Ã binding buffer to the solution, and incubate at room temperature for 10 min. Again, wash the mixture with 70 Î¼l of washing buffer twice using the Magna Stand. Elute the RNA in 14 Î¼l of pre-warmed distilled water at 80 Â°C for 2 min, then immediately place on the magnet and collect into a new tube. The expected volume of the product is 13 Î¼l, with which the following experiments may only be performed twice at most.<br/><br/></li></ol></li><li>RNA-seq library preparation and sequencing<ol class="npwtwo"><li>Mix 5 Î¼l of purified mRNA obtained in the above section with 4 Î¼l of 5Ã Super Script IV buffer (Invitrogen) and 1 Î¼l of frozen stock of 100 mM DTT (Invitrogen).</li><li>mRNA is fragmented by incubating at 94 Â°C for 4.5 min and then immediately cooled down on ice.</li><li>Then add 0.6 Î¼l of 100 Î¼M random primer (N)<sub>6</sub> (TaKaRa) and 0.9 Î¼l of distilled water to the fragmented mRNA. The volume of the mix is 12 Î¼l.</li><li>Incubate the mixture at 50 Â°C for 5 min and immediately chill on ice to relax the secondary structures of the mRNA.</li><li>Add the reverse transcription master mix (1 Î¼l of frozen stock of 100 mM DTT, 0.4 Î¼l of dNTP (25 mM each) (Promega); 0.1 Î¼l of SuperScript IV (Invitrogen); 0.2 Î¼l of Actinomycin D (1,000 ng/Î¼l) (Nacalai Tesque); and 5.9 Î¼l of distilled water). Now the volume of the mix is 20 Î¼l.</li><li>For the reverse transcription step, incubate the mixture at 25 Â°C for 10 min, followed by 10 min at 50 Â°C. Inactivate SuperScript IV by heating the mixture at 80 Â°C for 15 min. Then add 24 Î¼l of AMPure XP (Beckman Coulter) and 12 Î¼l of 99.5% EtOH, and perform the purification step according to the manufacturerâs manual.<br/>The detail of purification with AMPure XP beads in this protocol is as below:<br/><ol class="npwthree"><li>Mix by pipetting.</li><li>Room temperature, 5 min.</li><li>Put on magnet, 5 min, remove supernatant.</li><li>Add 70 Âµl of 70% EtOH on the magnet stand and remove it. Repeat this step again.</li><li>Dry up for 1 min.</li><li>Add 10.0 Âµl or other Nuclease free water, mix by pipetting.</li><li>R.T. for 1 min.</li><li>Put on magnet, collect 10.0 Âµl or other of supernatant to 384-well PCR plate.</li></ol></li><li>Elute the transcription product with 10 Î¼l of distilled water. Mix the purified DNA/RNA hybrid solution without beads with the second strand synthesis master mix [2 Î¼l of 10Ã Blue Buffer (Enzymatics), 1 Î¼l of dUTP/NTP mix (Fermentas), 0.5 Î¼l of frozen stock of 100 mM DTT, 0.5 Î¼l of RNase H (Enzymatics), 1 Î¼l of DNA polymerase I (Enzymatics), and 5 Î¼l of distilled water] The final volume of the mixture is 20 Î¼l.</li><li>Incubate the mixture at 16 Â°C for 4 h. Purify dsDNA with 24 Î¼l of AMPure XP according to the manufacturerâs manual. Elute the purified dsDNA with 10 Î¼l of distilled water. Use 5 Î¼l of the dsDNA solution for the following step.</li><li>Perform end-repair, A-tailing and adapter ligation using a KAPA Hyper Prep kit (KAPA Biosystems) with 1/10Ã volume of the solutions according to the manufacturerâs manual. Use 1 Î¼l of 0.1 Î¼M Y-shape adapter (Recipe 3, Nagano <em>et al.</em>, 2015) in the adapter ligation step and incubate for 15 min at 20 Â°C. The final volume of this reaction product is 11 Î¼l.</li><li>Perform size selection of the ligation product with 5.5 Î¼l of AMPure XP, resulting in 0.32Ã AMPure XP. The first size-selection is conducted at the smaller ratio than the second due to strong viscosity of ligation reaction mix. Elute the purified dsDNA using 10 Î¼l of distilled water. Carry out the second round of size selection with 10 Î¼l of AMPure XP, resulting in 0.5Ã AMPure XP. Elute the size-selected ligation product with 15 Î¼l of 10 mM Tris-HCl, pH 8.0. Add 1 Î¼l of uracil DNA glycosylase (UDG) (Enzymatics) to the size-selected ligation product. The volume of the mix is 16 Î¼l.</li><li>Incubate the mixture at 37 Â°C for 30 min to exclude the second-strand DNA.</li><li>For library amplification, mix 2 Î¼l of the UDG-digested DNA with 1 Î¼l of 2.5 Î¼M index primer (CAAGCAGAAGACGGCATACGAGATXXXXXXXXGTGACTGGAGTTCAGACGTGT, XXXXXXXX indicates index sequence in the <a href="https://os.bio-protocol.org/attached/file/20200507/3496 Supplemetal file.docx" target="_blank">supplemental file</a>) (Nagano <em>et al.</em>, 2015), 1 Î¼l of 10 Î¼M universal primer (AATGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGATCT) (Nagano <em>et al.</em>, 2015), 0.5 Î¼l of distilled water and 5 Î¼l of KAPA HiFi HotStart ReadyMix (2Ã) (KAPA Biosystems). The volume of the mix is 10 Î¼l.</li><li>Amplify DNA fragments with the adapters and an index sequence using a thermal cycler with the following program: denature at 94 Â°C for 2 min, 18 cycles at 98 Â°C for 10 s, 65 Â°C for 30 s, 72 Â°C for 30 s as an amplification step, and 72 Â°C for 5 min for the final extension. Then perform two rounds of size selection to remove adapter dimer with an equal volume of AMPure XP to the library solution with 10 Âµl of D.W. for elution. Next elute the purified library with 10 Î¼l of distilled water. Now, you can pool the libraries, if needed. Also use library quantification kit (<em>e.g.</em>, KAPA Library Quantification Kit Illumina (KAPA Biosystems)) to determine the concentration followed by pooling.</li><li>Take out 1 Î¼l of the purified library for electrophoresis using an Agilent High Sensitivity DNA kit (Agilent Technologies) to evaluate quality. The typical concentration of the library is more than 1 ng/Âµl, and the size distribution is shown below (Figure 2).<br/><br/><img alt="" class="layerphoto" height="130" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617225305_0501.jpg" width="100"/><br/><strong>Figure 2. The typical size distribution of the final product of this protocol.</strong> A result from electrophoresis produced with Agilent High Sensitivity DNA kit.<br/></li></ol></li></ol><p class="pptt" id="biaoti34464">### Recipes</p><ol class="npwtwok"><li>2Ã binding buffer<br/>40 mM Tris-HCl, pH 7.6<br/>2 M LiCl<br/>4 mM EDTA</li><li>Washing buffer<br/>10 mM Tris-HCl, pH 7.6<br/>0.15 M LiCl<br/>1 mM EDTA</li><li>Y-shape adapter<br/>A mixture of 100 mM adapters (5â²-A*\A*TGATACGGCGACCACCGAGATCTACACTCTTTCCCTACACGACGCTCTTCCGAT*C*T-3â² 5â²-/5Phos/-G*A*TCGGAAGAGCACACGTCTGAACTCCAGTC*A*C-3â².<br/>*signifies a phosphonothioate bond. /5Phos/signifies a phosphorylation) is annealed using a thermal cycler with the following program:<br/>95 Â°C for 2 min, slow-cooled to 25 Â°C (0.1 Â°C/s), followed by 30 min at 25 Â°C. The annealed adapter (50 Î¼M) is diluted into 0.1 Î¼M with D.W and stored at -20 Â°C.<br/></li></ol><p class="pptt" id="biaoti34465">### Acknowledgments</p><p class="ppzz">This work was supported by the JST CREST JPMJCR15O2 to A.J.N. This protocol was modified from the method described in Nagano <em>et al.</em>, 2015.</p><p class="pptt" id="biaoti34466">### Competing interests</p><p class="ppzz">We have no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti34467">### References</p><ol class="npwtwok"><li>Alpern, D., Gardeux, V., Russeil, J., Mangeat, B., Meireles-Filho, A. C. A., Breysse, R., Hacker, D. and Deplancke, B. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30999927" target="_blank">BRB-seq: ultra-affordable high-throughput transcriptomics enabled by bulk RNA barcoding and sequencing.</a> <em>Genome Biol</em> 20(1): 71.</li><li>Kamitani, M., Kashima, M., Tezuka, A. and Nagano, A. J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31068632" target="_blank">Lasy-Seq: a high-throughput library preparation method for RNA-Seq and its application in the analysis of plant responses to fluctuating temperatures.</a> <em>Sci Rep</em> 9(1): 7091.</li><li>Kumar, R., Ichihashi, Y., Kimura, S., Chitwood, D. H., Headland, L. R., Peng, J., Maloof, J. N. and Sinha, N. R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22973283" target="_blank">A high-throughput method for Illumina RNA-Seq library preparation.</a> <em>Front Plant Sci</em> 3: 202.</li><li>Muir, P., Li, S., Lou, S., Wang, D., Spakowicz, D. J., Salichos, L., Zhang, J., Weinstock, G. M., Isaacs, F., Rozowsky, J. and Gerstein, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27009100" target="_blank">The real cost of sequencing: scaling computation to keep pace with data generation.</a> <em>Genome Biol</em> 17: 53.</li><li>Nagano, A. J., Honjo, M. N., Mihara, M., Sato, M. and Kudoh, H. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25287498" target="_blank">Detection of plant viruses in natural environments by using RNA-Seq.</a> <em>Methods Mol Biol</em> 1236: 89-98. </li><li>Townsley, B. T., Covington, M. F., Ichihashi, Y., Zumstein, K. and Sinha, N. R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26052336" target="_blank">BrAD-seq: Breath Adapter Directional sequencing: a streamlined, ultra-simple and fast library preparation protocol for strand specific mRNA library construction.</a> <em>Front Plant Sci</em> 6: 366. <br/></li></ol>