<body>
            <h1>Measurement of the Promoter Activity in <em>Escherichia coli</em> by Using a Luciferase Reporter</h1>
            <p><strong>Authors:</strong> Yuki Yamanaka, Hiroki  Watanabe, Erika  Yamauchi, Yukari  Miyake, Kaneyoshi   Yamamoto</p>
            <h2>Abstract</h2>
            <p>The reporter system is widely used technique for measuring promoter activity in bacterial cells. Until now, a number of reporter system have been developed, but the bioluminescent reporter constructed from the bacterial luciferase genes is one of the useful systems for measuring <em>in vivo</em> dynamics of gene expression. The introduced bioluciferase <em>lux</em> reporter enables easy, fast, and sensitive measurement of the promoter activity without cell lysis because the substrates of bioluminescent reaction are synthesized inside the bacterial cell, thereby allowing low-cost experiments. This protocol describes a high throughput technique to measure the promoter activity in <em>Escherichia coli</em> K-12 using the <em>lux</em> reporter system.</p>
            <p><strong>Keywords:</strong> <em>lux</em> operon, Luciferase, <em>Escherichia coli</em>, Promoter activity, High-throughput assay, H-NS silencer</p>
            <h2>Background</h2><p>The promoter activity <em>in vivo</em> was measured using a reporter system such as the <em>lacZ</em> (encoding Î²-galactosidase), <em>gus</em> (encoding Î²-gulucuronidase), and <em>cat</em> (encoding chloramphenicol acetyltransferase) genes. In the case of the <em>lacZ</em> reporter system, for instance, the test promoter sequence is fused to a promoter-less <em>lacZ</em> gene, creating a test promoter-<em>lacZ</em> fusion gene, which is then transferred into a recipient cell. For the measurement of the activity of the test promoter, however, the whole cell lysate must be prepared to detect <em>in vitro</em> Î²-galactosidase activity by adding a substrate such as ONPG (<em>O</em>-Nitrophenyl-Î²-D-galactopyranoside). To avoid such biochemical procedures, the fluorescent <em>gfp</em> gene, coding green fluorescent protein (GFP), was employed as a reporter which can be detected without cell lysis. Thus, the fluorescent reporter system is more convenient than the systems which requires measurement of enzymatic activity. However, the fluorescent proteins have a technical limitation especially in genes that are expressed at low levels because of high background noise that arises from intrinsic autofluorescence of cells. To overcome this problem, the luminescent reporter has been developed, which catalyzes bioluminescence reactions using the substrate as luciferins (Meighen, 1991). The <em>Photorhabdus luminescens</em> bioluminescence <em>luxCDABE</em> genes, coding two luciferase subunits (LuxAB) and three proteins (LuxCDE), which are important for substrate biosynthesis (Bjarnason <em>et al.</em>, 2003). Once the test promoter is fused to promoter-less <em>luxCDABE</em>, both luciferase and its substrate are expressed under the control of the test promoter, and the promoter activity can be easily determined by measuring luminescence without the cell lysis (Bjarnason <em>et al.</em>, 2003). This bioluminescent reporter system is recognized as a powerful high-throughput assay for studying continuous kinetics of promoter activity (Yamanaka <em>et al.</em>, 2018; Burton <em>et al.</em>, 2010). In this protocol, we describe how to construct the bioluminescent reporter system and how to measure the promoter activity in <em>E. coli</em>.</p>
        </body>

<p class="pptt" id="biaoti30977">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (Thermo Fisher Scientific, catalog number: QSP Liquid Handling Products 110-Q and 111-Q) (Gilson, catalog number: DIAMOND Tips DL10 and D5000)</li><li>1.5 ml plastic tube (Rikaken, catalog number: STAR MicroTestTube 1.5 ml RSV-MTT1.5)<br/></li><li>0.22 Î¼m filter (Advantec, catalog number: 13CP020AS)</li><li>Glass tubes (Iwaki, catalog number: TEST18NP)</li><li>Sterile 50 ml plastic tube (Iwaki, catalog number: 2345-050)<br/></li><li>BD Falcon 96-well plates, Black/Clear BD Optilux (Becton Dickinson, catalog number: 353948)</li><li>96-well white plate (Becton Dickinson, catalog number: 353377)</li><li>Petri dishes (Rikaken, catalog number: STAR SDish9015 ver.2 RSU-SD9015-2)</li><li>pLUX vector (Burton <em>et al.</em>, 2010)</li><li>Specific primers (Thermo Fisher Scientific, Custom DNA Oligos) (Table 1)</li><li>TaKaRa Ex Taq (Takara Bio, catalog number: RR001A)</li><li>NucleoSpin Gel and PCR Clean-up (Macherey-Nagel, catalog number: 740609.10)</li><li>Restriction enzyme <em>Xho</em> I (Takara Bio, catalog number: 1094)</li><li>Restriction enzyme <em>Bam</em> HI (Takara Bio, catalog number: 1010)</li><li>In-Fusion HD Cloning Plus (Takara Bio, catalog number: 638920)</li><li>Kanamycin Monosulfate (Nacalai Tesque, catalog number: 19839-44)</li><li>Plasmid DNA Extraction Mini Kit (Favorgen, catalog number: FAPDE 001)</li><li>BigDye<sup>Â®</sup> Terminator v3.1 Ready Reaction Mix (Applied Biosystems, catalog number:4337455)</li><li>IPTG (Nacalai Tesque, catalog number: 06289-67)</li><li>3 M sodium acetate (Nacalai Tesque, catalog number: 06893-24)</li><li>Ethanol (Nacalai Tesque, catalog number: 14710-25)</li><li>Hi-Di<sup>TM</sup> Formamide (Applied Biosystems, catalog number: 4311320)</li><li>Bacto<sup>TM</sup> tryptone (BD Biosciences, catalog number: 211705) </li><li>Bacto<sup>TM</sup> yeast extract (BD Biosciences, catalog number: 212750), </li><li>NaCl (Nacalai Tesque, catalog number: 31320-05)</li><li>NaOH (Nacalai Tesque, catalog number: 31511-05)</li><li>Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O (Nacalai Tesque, catalog number: 31722-45)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Wako, catalog number: 498748161612) </li><li>MgCl<sub>2</sub>Â·6H<sub>2</sub>O (Nacalai Tesque, catalog number: 20908-65)</li><li>K<sub>2</sub>SO<sub>4</sub>Â·12H<sub>2</sub>O (Nacalai Tesque, catalog number: 01727-25)</li><li>NH<sub>4</sub>Cl<sub>2</sub> (Nacalai Tesque, catalog number: 02424-55)</li><li>CaCl<sub>2</sub> (Nacalai Tesque, catalog number: 08894-25)</li><li>D-(+)-Glucose (Nacalai Tesque, catalog number: 16805-35)</li><li>0.5 M EDTA (Nacalai Tesque, catalog number: 06894-14)</li><li>Competent <em>E. coli</em> DH5Î±, provided from National Institute of Genetics in Japan (preparation at time of use) (see Recipes)</li><li>50 mg/ml kanamycin (see Recipes)</li><li>LB broth (see Recipes)<br/></li><li>LB agar with 50 Î¼g/ml kanamycin (see Recipes)</li><li>125 mM EDTA (see Recipes)</li><li>70% ethanol (see Recipes)</li><li>M9-Glucose medium (see Recipes)<br/></li></ol><p class="pptt" id="biaoti30978">### Equipment</p><ol class="npwtwok"><li>Pipettes (Gilson, models: PIPETMAN P2, P10, P20, P100, P200, P1000, P5000)</li><li>Centrifuge (Tomy, model: MX-301)</li><li>Thermal Cycler (Applied Biosystems, model: 2720Thermal Cycler)</li><li>Temperature chamber (Taitec, model: Thermo minder SM-10R)</li><li>Water bath shaker (Taitec, model: Personal-11)</li><li>DNA sequencer (Applied Biosystems, model: 3500Genetyc Analyzer)</li><li>Plate reader (Corona, model: MTPâ880Lab)</li><li>Autoclave (Tomy Seiko, model: LSX-500)<br/></li></ol><p class="pptt" id="biaoti30979">### Software</p><ol class="npwtwok"><li>SF6 for Windows (Corona, in only Japanese)</li><li>Microsoft Excel (Microsoft)<br/></li></ol><p class="pptt" id="biaoti30980">### Procedure</p><ol class="npwone"><li>Construction of pLUX reporter plasmids<ol class="npwtwo"><li>Determine promoter region around a target gene. Typically, we take a region of DNA from 500 base-pairs (bp) upstream to 150 bp downstream of distal transcription start site (see for instance, the determined promoters of 18 target genes Figure 1).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200110/20200110011618_0767.jpg" width="135"/><br/><strong>Figure 1. Eighteen <em>E. coli</em> promoter regions into pLux</strong><br/><br/></li><li>Design primers which including 15 overlapping nucleotides (nt) with the pLUX vector for amplification of the determined promoter region (see Figure 2 and Table 1).</li><li>Amplify target promoter regions by PCR using Ex Taq polymerase (Takara Bio). Standard reaction condition is used as described in the manual for Ex Taq polymerase (Takara Bio). In most of the cases, thermal cycle conditions are as follows: after heating at 98 Â°C for 5 min, process 30 cycles of 98 Â°C for 10 s (denature), 50 Â°C for 30 s (annealing), and 72 Â°C for 1 min (elongation) (Figure 2).</li><li>Purify the PCR product by NucleoSpin Gel and PCR Clean-up (Macherey-Nagel).<br/><em>Note: When PCR product includes non-specific bands, the target product is cut from agarose gel after electrophoresis with total volume of PCR product and then purified by NucleoSpin Gel and PCR Clean-up (Macherey-Nagel).</em> </li><li>Digest pLUX vector using restriction enzymes <em>Xho</em> I and <em>Bam</em> HI, and then purify the liner pLUX by NucleoSpin Gel and PCR Clean-up (Macherey-Nagel).<br/><em>Note: For cloning into pLUX, we recommend the double-digestion of pLUX by both Xho I and Bam HI (Figure 2).</em> </li><li>Clone a target promoter as the purified PCR product into the liner pLUX using In-Fusion HD Cloning Plus (Takara Bio).<br/><em>Note: We recommend </em><em>in vivo</em> <em>E. coli</em><em> cloning (iVEC) as an alternative DNA cloning method (Nozaki </em><em>et al.</em><em>, 2019), which is provided from National BioResource Project (NBRP) of Japan.</em> </li><li>Add a part (~10 Î¼l) of the In-Fusion reaction mixture into 0.1 ml of the suspension of competent <em>E. coli</em> DH5Î± (see Recipes), and then incubate the mixture on ice for 30 min.</li><li>Heat the mixture at 42 Â°C for 45 s, and immediately add 0.9 ml of LB broth to the mixture.</li><li>Incubate 1 ml of the suspension of <em>E. coli</em> DH5Î± transformants at 37 Â°C for 60 min.</li><li>Spread 0.1-0.2 ml of the suspension of <em>E. coli</em> DH5Î± transformants onto the LB agar with 50 Î¼g/ml kanamycin, and then incubate the agar plate at 37 Â°C overnight.</li><li>Isolate a transformant of <em>E. coli</em> DH5Î± harboring the cloned candidates as a single colony on the LB agar with 50 Î¼g/ml kanamycin.</li><li>Inoculate a single colony into 5 ml of LB broth containing 0.05 ml of 50 mg/ml kanamycin in a glass tube (Iwaki), and incubate the culture in water bath (Taitec) at 37 Â°C with shaking (120 rpm) overnight.</li><li>Isolate the plasmid from the transformant cells of 5 ml overnight culture by Plasmid DNA Extraction Mini Kit (Favorgen).</li><li>Sanger reaction is performed with the isolated plasmid as a template, LUX-R primer (Table 1), and BigDye<sup>Â®</sup> Terminator v3.1 Ready Reaction Mix (Applied Biosystems) according to the recommended procedure by Supplier.</li><li>Transfer 20 Î¼l of sanger product into a sterile 1.5 ml plastic tube and then add 2 Î¼l of 125 mM EDTA, 2 Î¼l of 3 M sodium acetate, and 50 Î¼l of ethanol (Nacalai Tesque).</li><li>Mix the solution by vortex mixer for 15 s.</li><li>Collect DNA pellet by centrifugation (17,800 <em>x g</em>, 4 Â°C, 15 min).</li><li>Add 70 Î¼l of 70% ethanol and collect DNA pellet by centrifugation (17,800 <em>x g</em>, 4 Â°C, 15 min).</li><li>Dissolve DNA pellet by 15 Î¼l of Hi-Di<sup>TM</sup> Formamide (Applied Biosystems).</li><li>Determine DNA sequence of the promoter region cloned into the isolated plasmid with DNA sequencer (Applied Biosystems).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200110/20200110011655_3040.jpg" width="146"/><br/><strong>Figure 2. Strategy for construction of pLUX reporter plasmid.</strong> The promoter region was amplified by PCR using the <em>E. coli</em> K-12 genome as the template and a pair of specific primers (Table 1). The amplified DNA fragment was inserted into a pLUX vector using the In-Fusion HD cloning kit (Takara Bio). After transformation of DH5Î±, the plasmid was purified from a culture of transformant cells. The DNA sequence of insertion on the resulting plasmids was confirmed by DNA sequencing.<br/><br/><strong>Table 1. The used primers and pLUX derivatives</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200110/20200110013653_0704.jpg" width="1000"/><br/>The plasmids constructed in this study could be provided from National BioResource Project (NBRP) <em>E. coli</em> of Japan.<br/><br/></li></ol></li><li>Measurement of luciferase activity in <em>E. coli</em> <ol class="npwtwo"><li>Transform <em>E. coli</em> strains by the cloned luciferase reporter plasmids.<br/><em>Note: The âhnsâhhaâydgT strain, isolated from </em><em>E. coli</em><em> K-12 W3110 strain, deleted three genes, hns, hha, and ydgT, in its genome (Ueda </em><em>et al.</em><em>, 2013). H-NS plays a role in transcriptional silencing of genes, which is modulated by Hha and YdgT proteins in </em><em>E. coli</em><em>. The âhnsâhhaâydgT strain was transformed with pQE80L, pQE80Lhns which carries hns gene in pQE80L, pQE80Lhns-I70A which carries hns gene with substitutions of Ile70Ala, or pQE80Lhns-L75A which carries hns gene with substitutions of Leu75Ala. These transformants were used as hosts for luciferase measurements.</em> </li><li>Inoculate at least three single colonies of <em>E. coli</em> transformant in glass tubes separately under M9-glucose medium including 50 Î¼g/ml kanamycin.<br/><em>Note: In addition of kanamycin, ampicillin should be added in medium at the final concentration of 100 Î¼g/ml for the âhnsâhhaâydgT strain harboring a pQE80L derivative and a pLux derivative.</em> </li><li>Incubate the pre-cultures at 37 Â°C with shaking for overnight.</li><li>Inoculate 100 Î¼l of overnight pre-culture in 10 ml of fresh M9-glucose medium including 50 Î¼g/ ml kanamycin.</li><li>Incubate the 10 ml cultures at 37 Â°C in water bath (Taitec) with shaking (120 rpm) until luciferase activity is measured.</li><li>Transfer 100 Î¼l of culture to a well of a Black/Clear BD Falcon 96-well plate (Becton Dickinson) in triplicate for each culture.<br/><em>Note: We tested 96-well plates for measuring luminescence in </em><em>E. coli</em><em> with the plate reader (Corona), indicating that the 96-well white plate showed the leakage of 1.5% to an adjacent empty well whereas the 96-well black plate reduced leakage by 0.95% (see the row data in both 96 plates in Figure 3). Although a white plate is usually used for luminescent measurements, a black plate was used to prevent leakage of high luminescence from one culture to other well in this study. Additionally, during measurement we arranged cultures with a single well gap between them to minimize leakage to adjacent wells. However, we recommend that a white plate be used first as standard procedure.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200110/20200110013810_9859.jpg" width="194"/><br/><strong>Figure 3. The measured intensity of luciferase activity in <em>E. coli</em> in 96-well plates.</strong> <em>E. coli</em> K-12 W3110 (parent strain) harbouring pLUXappYp, containing <em>appY-lux</em> operon (data not shown) was grown in M9-glucose medium at 37 Â°C, and culture was applied into three wells (A1, A2, and A3) of both the 96-well black plate (Becton Dickinson, catalog number: 353948) (upper) and the 96-well white plate (Becton Dickinson, catalog number: 353377) (lower). M9-glucose medium (H5, H6, and H7) and distilled water (H9, H10, and H11) were also applied into three wells of both the 96-well plates. Each intensity is shown as a raw data measured with the plate reader (Corona). The fluorescent intensity was detected in the empty wells adjacent to wells filled by culture (A4 and B1 to B4).<br/><br/></li><li>Transfer 100 Î¼l of fresh M9-glucose medium to another well of the Black/Clear BD Falcon 96-well plate (Becton Dickinson) used in Step B6 in triplicate for background.</li><li>Set the Black/Clear BD Falcon 96-well plate (Becton Dickinson) containing the samples and the background with plate reader (Corona).</li><li>Measure OD<sub>600</sub> according to the procedure for the plate reader (Corona).</li><li>Measure luminescence as a total intensity. Therefore, no filter is set according to the procedure for the plate reader (Corona).</li></ol></li></ol><p class="pptt" id="biaoti30981">### Data analysis</p><ol class="npwtwok"><li>Extract raw numeric data of both the values of OD<sub>600</sub> and the intensities of luminescence as a text file from SF6 for Windows (Corona).</li><li>Open a text file containing the raw numeric data in Microsoft Excel (Microsoft).</li><li>Normalize the net values of OD<sub>600</sub> and the net intensities of luminescence with background.</li><li>Calculate the ratio of luminescence to OD<sub>600</sub> as specific activity of the promoter of each culture by the following formula: the net intensity of luminescence/the net values of OD<sub>600</sub>.</li><li>Average the ratios of triplicate with standard deviation.<br/><em>Note: The activities of the 18 promoters used are represented in Figure 4, indicating that all 18 promoters silenced by H-NS were de-silenced by two H-NS mutants in agreement with our previous work (Yamanaka </em><em>et al.</em><em>, 2018). To confirm a significant difference, calculated p-values by t-test of statistical analysis should be evaluated at less than 0.01.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200110/20200110013907_5962.jpg" width="136"/><br/><strong>Figure 4. Luciferase reporter reveals silence of 18 promoters by H-NS in <em>E. coli</em>. </strong>The constructed 18 pLux reporter plasmids, pLUXadiAp, pLUXadiCp, pLUXascFp, pLUXascGp, pLUXbglGp, pLUXcadBp, pLUXcadCp, pLUXcbpAp, pLUXddlAp, pLUXgadAp, pLUXgadBp, pLUXgadCp, pLUXgadWp, pLUXglcVp, pLUXiraPp, pLUXpagPp, pLUXproVp, and pLUXslpp (Table 1), were introduced into <em>âhnsâhhaâydgT</em> strains harbouring the <em>hns</em> plasmids (pQE80Lhns, pQE80Lhns-I70A, and pQE80Lhns-L75A) or an empty plasmid (a vector pQE80L). Transformants were grown in M9-glucose medium with 10 Î¼M IPTG at 37 Â°C, and then the promoter activity was calculated as described above.<br/></li></ol><p class="pptt" id="biaoti30982">### Recipes</p><ol class="npwtwok"><li>Competent <em>E. coli</em> DH5Î±<ol class="npwthree"><li>Inoculate a single colony of <em>E. coli</em> DH5Î± in 5 ml of LB broth</li><li>Incubate the pre-cultures at 37 Â°C with shaking for overnight</li><li>Dilute overnight pre-culture 100-fold in 10 ml of fresh LB broth</li><li>Incubate the cultures at 37 Â°C with shaking until mid-logarithmic phase</li><li>Transfer 10 ml of culture to a sterile 50 ml plastic tube (Iwaki)</li><li>Collect the cells by centrifugation (2,300 <em>x g</em>, 4 Â°C, 10 min)</li><li>Suspend the cells in 10 ml of cold 0.1 M CaCl<sub>2</sub> </li><li>Incubate the cells on ice for 30 min.</li><li>Collect the cells by centrifugation (2,300 <em>x g</em>, 4 Â°C, 10 min)</li><li>Suspend the cells in 1 ml of cold 0.1 M CaCl<sub>2</sub> </li><li>Use a part (~100 Î¼l) of the resuspended <em>E. coli</em> DH5Î± for transformation</li></ol></li><li>50 mg/ml kanamycin<ol class="npwthree"><li>Dissolve 0.5 g of kanamycin monosulfate (Nacalai Tesque) in 10 ml of distilled water</li><li>Sterilize the solution by filtration with 0.22 Î¼m filter (Advantec)</li><li>Store the sterilized solution at 4 Â°C</li></ol></li><li>LB broth<ol class="npwthree"><li>Dissolve 10 g of Bacto<sup>TM</sup> tryptone, 5 g of Bacto<sup>TM</sup> yeast extract, and 10 g of NaCl in 800 ml of distilled water</li><li>Adjust pH to 7.5 with NaOH</li><li>Adjust volume to 1 L with distilled water</li><li>Autoclave the solution (set 121 Â°C and 20 min in LSX-500)</li><li>Store the autoclaved LB broth at room temperature</li></ol></li><li>LB agar with 50 Î¼g/ml kanamycin<ol class="npwthree"><li>Dissolve 7.5 g of agar in 1 L of LB broth</li><li>Autoclave (set 121Â°C and 20 min in LSX-500)</li><li>Add kanamycin after cooling in the final concentration of 50 Î¼g/ml</li><li>Pour the media into Petri dishes, and then harden LB agar by cooling at room temperature</li><li>Store the LB agar at 4 Â°C</li></ol></li><li>125 mM EDTA<ol class="npwthree"><li>Dilute 0.5 M EDTA (Nacalai Tesque) to 125 mM with a sterile water</li><li>Store at room temperature</li></ol></li><li>70% ethanol<ol class="npwthree"><li>Dilute ethanol (Nacalai Tesque) to 70% with a sterile water</li><li>Store at room temperature</li></ol></li><li>M9-Glucose medium (to autoclave, set 121 Â°C and 20 min in LSX-500)<ol class="npwthree"><li>For 5x M9 salt (-NH<sub>4</sub>Cl<sub>2</sub>), dissolve 75 g of Na<sub>2</sub>HPO<sub>4</sub>â12H<sub>2</sub>O, 15 g of KH<sub>2</sub>PO<sub>4</sub> and 0.25 g of NaCl in 1 L distilled water, and then the dissolved solution was autoclaved</li><li>Prepare the autoclaved following solutions: 2 M NH<sub>4</sub>Cl<sub>2</sub>, 1 M MgCl<sub>2</sub> , 0.25 M K<sub>2</sub>SO<sub>4</sub>, 10 mM CaCl<sub>2</sub> </li><li>Sterilize the 1 M glucose by filtration (0.22 Î¼m filter)</li><li>Add 200 ml of 5x M9 salt (-NH<sub>4</sub>Cl<sub>2</sub>), 10 ml of 2 M NH<sub>4</sub>Cl<sub>2</sub>, 1 ml of 1 M MgCl<sub>2</sub> , 1 ml of 0.25 M K<sub>2</sub>SO<sub>4</sub>, 10 ml of 10 mM CaCl<sub>2</sub> , 10 ml of 1 M Glucose into 768 ml of autoclaved distilled water</li><li>Store the sterilized M9-glucouse medium at room temperature</li></ol></li></ol><p class="pptt" id="biaoti30983">### Acknowledgments</p><p class="ppzz">We are extremely grateful to Prof. Akira Ishihama, Hosei Univeristy, for his comments and editing of the manuscript. We gratefully acknowledge Dr. Parisa Zangoui Nejad Chahkootahi (National University of Singapore) and Dr. Kyle David Buchan (National University of Singapore) for proofreading of the manuscript. The pLUX vector was a gift from Dr. Peter A. Lund, University of Birmingham. The <em>E. coli</em> â<em>hns</em>â<em>hha</em>â<em>ydgT</em> strain was a gift from Dr. Taku Oshima, Toyama Prefectural University. We thank Eri Arita, Kanako Hasegawa, Sho Watarai, Hideko Nakagawa, and Sato Fukunaga from Hosei university for technical assistance.<br/>  This protocol was adapted from Burton <em>et al.</em> (2010) for the method to luciferase reporter assay and modified from Yamanaka <em>et al.</em> (2018) for the method to high-throughput assay.</p><p class="pptt" id="biaoti30984">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt" id="biaoti30985">### References</p><ol class="npwtwok"><li>Bjarnason, J., Southward, C. M. and Surette, M. G. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12897017" target="_blank">Genomic profiling of iron-responsive genes in <em>Salmonella</em> <em>enterica </em>serovar typhimurium by high-throughput screening of a random promoter library.</a> <em>J Bacteriol</em> 185(16): 4973-4982.</li><li>Burton, N. A., Johnson, M. D., Antczak, P., Robinson, A. and Lund, P. A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20603130" target="_blank">Novel aspects of the acid response network of <em>E. coli</em> K-12 are revealed by a study of transcriptional dynamics.</a> <em>J Mol Biol</em> 401(5): 726-742. </li><li>Meighen, E. A. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2030669" target="_blank">Molecular biology of bacterial bioluminescence.</a> <em>Microbiol Rev</em> 55(1): 123-142.</li><li>Nozaki, S. and Niki, H. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30530516" target="_blank">Exonuclease III (XthA) enforces <em>in vivo</em> DNA cloning of <em>Escherichia coli</em> to create cohesive ends.</a> <em>J Bacteriol</em> 201(5): pii: e00660-18.</li><li>Ueda, T., Takahashi, H., Uyar, E., Ishikawa, S., Ogasawara, N. and Oshima, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23543115" target="_blank">Functions of the Hha and YdgT proteins in transcriptional silencing by the nucleoid proteins, H-NS and StpA, in <em>Escherichia coli</em>.</a> <em>DNA Res</em> 20(3): 263-271.</li><li>Yamanaka, Y., Oshima, T., Ishihama, A., and Yamamoto, K. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25375160" target="_blank">Characterization of the YdeO regulon in <em>Escherichia coli</em>.</a> <em>PLoS ONE</em> 9: e111962.</li><li>Yamanaka, Y., Winardhi, R. S., Yamauchi, E., Nishiyama, S. I., Sowa, Y., Yan, J., Kawagishi, I., Ishihama, A. and Yamamoto, K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29695505" target="_blank">Dimerization site 2 of the bacterial DNA-binding protein H-NS is required for gene silencing and stiffened nucleoprotein filament formation.</a> <em>J Biol Chem</em> 293(24): 9496-9505.<br/></li></ol>