<body>
            <h1>Detection of Individual RNA in Fixed Cells and Tissues by Chromogenic ISH</h1>
            <p><strong>Authors:</strong> Meng Jiang, Chen Lin, Rongqin  Ke</p>
            <h2>Abstract</h2>
            <p>Visualization of RNA molecules <em>in situ</em> helps to better understand the functions of expressed genes. Currently, most conventional <em>in situ</em> hybridization methods for visualization of individual RNAs are based on fluorescence detection. Herein we present a chromogenic <em>in situ</em> hybridization protocol for visualization of single RNA molecules in fixed cells and tissues. The protocol is based on padlock probing and rolling circle amplification to generate detectable chromogenic signal from single RNA molecules. Chromogenic signal can avoid background autofluorescence and can be preserved for a longer period than fluorescence signal.</p>
            <p><strong>Keywords:</strong> Chromogenic <em>in situ</em> hybridization, Gene expression, Padlock probe, Rolling circle amplification</p>
            <h2>Background</h2><p>The abundance and spatial location of expressed RNA molecules indicates the physiological and pathological status of cells and tissues. Therefore, there is an increasing interest for <em>in situ</em> RNA detection. Compared to conventional <em>in situ</em> hybridization methods, novel methods that can detect RNA at individual molecule level are more sensitive and specific (Crosetto <em>et al.</em>, 2015). These methods include single molecule fluorescence <em>in situ</em> hybridization (smFISH) (Femino <em>et al.</em>, 1998), methods based on rolling circle amplification (Larsson <em>et al.</em>, 2010), hybridization chain reaction (HCR) (Shah <em>et al.</em>, 2016), and branched DNA (bDNA) technology (Wang <em>et al.</em>, 2012; Battich <em>et al.</em>, 2013). Current methods for individual RNA <em>in situ</em> detection assays are normally readout as fluorescence signal, which offers good sensitivity and is easy to multiplex. Chromogenic readout is often used in conventional RNA ISH assays, providing stable signal that can be stored for long period of time, and is not affected by autofluorescence or photobleaching. Therefore, we introduce the single molecule chromogenic <em>in situ</em> hybridization (smCISH) assay that enables sensitive and specific detection of individual RNA in fixed cells and tissues (Jiang <em>et al.</em>, 2019). Our protocol is based on padlock probing and rolling circle amplification (Nilsson <em>et al.</em>, 1994; BanÃ©r <em>et al.</em>, 1998). First, padlock probe is designed to directly bind to target RNA specifically. After padlock probe hybridization, probe circularization is performed using SplintR DNA ligase. The closed circle is RNA-templated ligation of its two ends. Next, an RCA primer is hybridized to the circularized padlock probe to initiate rolling circle amplification reaction. The padlock probe is then amplified into its complementary concatenated form to generate RCP. HRP labeled detection probes are then hybridized to the RCP. Finally, chromogenic signal is developed using H<sub>2</sub>O<sub>2</sub> and 3,3'-Diaminobenzidine (DAB) to generate brown insoluble dots from individual RCPs, which corresponds to single RNA molecules. Our results show that single molecule chromogenic <em>in situ</em> hybridization assay can count and localize individual RNA molecules in fixed cells and tissue samples.</p>
        </body>

<p class="pptt" id="biaoti31178">## Materials and Reagents</p><ol class="npwtwok"><li>Secure-Seal hybridization chamber, 9 mm diameter, 0.8 mm deep (Containing seal tabs, Thermo Scientific, catalog number: S24732)</li><li>EasYFlask 25 cm<sup>2</sup> (Thermo Scientific Nunc, catalog number: 156340-ROS)</li><li>Serological Pipette (Thermo Scientific Nunc, 2 ml, 5 ml, 10 ml, 50 ml)</li><li>Cell culture Dish, 150 mm x 20 mm (Thermo Scientific Nunc, catalog number: 168381-ROS)</li><li>Adhesion Microscope Slides (Citotest, catalog number: 188105)</li><li>15 ml Conical Centrifuge Tubes (Thermo Scientific Nunc, catalog number: 339650-SPL)</li><li>50 ml Conical Centrifuge Tubes (Thermo Scientific Nunc, catalog number: 339652-SPL)</li><li>Filtered Pipette Tip (QSP, 0.1-10 Âµl, 2-20 Âµl, 10-100 Âµl, 20-200 Âµl, 100-1,000 Âµl)</li><li>0.2 ml PCR Tubes (AXYGEN, catalog number: PCR-02-C)</li><li>1.5 ml Microtubes (AXYGEN, catalog number: MCT-150-C)</li><li>0.22 Î¼m membrane filter (Millipore, catalog number: SLGP033RS)</li><li>HER2 padlock probe (sequence: 5â²-<u>ATTACTTGCAGGTTCT</u>TTCCTTTTAC<em><strong>GA<span style="border-bottom:5px double black;">CCTCAATGCTGCTGCTGT</span></strong></em><span style="border-bottom:5px double black;">ACTAC</span>TCTT<u>CAGAATTCGTCCCCGG</u>, underline: <u>target hybridization sequences</u> andd <span style="border-bottom:5px double black;">detection probe complementary part</span>, <strong><em>bold italics:</em></strong> <strong><em>RCA primer binding part</em></strong>) (Synbio Tech, China)</li><li>ACTB padlock probe (sequence: 5â²-<u>CTGTGCTCGCGGGGCG</u>TTCCTTTTAC<strong><em>GA<span style="border-bottom:5px double black;">CCTCAATGCTGCTGCTGT</span></em></strong><span style="border-bottom:5px double black;">ACTAC</span>TCTT<u>GGCAAAGGCGAGGCT</u>, underline: <u>target hybridization sequences</u> and <span style="border-bottom:5px double black;">detection probe complementary part</span>, <strong><em>bold italics: RCA primer binding part</em></strong>) (Synbio Tech, China)</li><li>RCA primer (sequence: 5â²-ACAGCAGCAGCATTGAGGTC) (Synbio Tech, China)</li><li>HRP labeled detection probe (sequence: 5â² HRP-CCTCAATGCTGCTGCTGTACTAC) (TaKaRa)</li><li>Pepsin (Sigma-Aldrich, catalog number: P7012-1G), store at -20 Â°C</li><li>Fetal Bovine Serum (Hyclone, catalog number: 10270-106), store at -20 Â°C</li><li>Trypsin (Hyclone, catalog number: SH30042.01), store at -20 Â°C</li><li>ATP, 100 mM Solution (Thermo Scientific, catalog number: R0441), store at -20 Â°C</li><li>T4 Polynucleotide Kinase, 10 U/Âµl (Thermo Scientific, catalog number: EK0032, containing PNK buffer A), store at -20 Â°C</li><li>dNTP Set, 100 mM Solutions (Thermo Scientific, catalog number: R0182), store at -20 Â°C</li><li>SplintR Ligase, 25 U/Âµl (NEB, catalog number: M0375L), store at -20 Â°C</li><li>phi29 DNA Polymerase, 10 U/Âµl (Thermo Scientific, catalog number: EP0094), store at -20 Â°C</li><li>BSA, 20 mg/ml (NEB, catalog number: B9000S), store at -20 Â°C</li><li>RiboLock RNase Inhibitor, 40 U/Âµl (Thermo Scientific, catalog number: EO0384), store at -20 Â°C</li><li>Tween-20 (Sigma-Aldrich, catalog number: P9416-100ML), store at room temperature (RT)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: T8787-100ML), store at RT</li><li>NaOH (Sigma-Aldrich, catalog number: 795429-500G), store at RT</li><li>KCl (Sigma-Aldrich, catalog number: P9541-500G), store at RT</li><li>NaCl (Sigma-Aldrich, catalog number: S3014-1kG), store at RT</li><li>Na<sub>2</sub>HPO<sub>4</sub> (BBI LIFE SCIENCES, catalog number: A610404-0500), store at RT</li><li>KH<sub>2</sub>PO<sub>4</sub> (Sangon Biotech, catalog number: A100781-0500), store at RT</li><li>Sodium iodate (BBI LIFE SCIENCES, catalog number: A600859), store at RT</li><li>Aluminum potassium sulfate dodecahydrate (Sangon Biotech, catalog number: A500755), store at RT</li><li>Glycerol (Sigma-Aldrich, catalog number: G9012-100ML), store at RT</li><li>Hematoxylin (Sigma-Aldrich, catalog number: H3136-25G), store at RT</li><li>Neutral balsam mounting medium (BBI LIFE SCIENCES, catalog number: E675007), store at RT</li><li>Xylene (XILONG SCIENTIFIC, catalog number: 33535-500ML), store at RT</li><li>Ethanol (Macklin, catalog number: E809061-500 ML), store at RT</li><li>Acetate (Macklin, catalog number: A801295-500 ML), store at RT</li><li>Hydrochloric acid (SCR, catalog number: 10011018), store at RT</li><li>DEPC (Sigma-Aldrich, catalog number: D5758-25ML), store at 4 Â°C</li><li>Paraformaldehyde (Sigma-Aldrich, catalog number: 16005-1KG-R), store at 4 Â°C</li><li>Formamide (Sigma-Aldrich, catalog number: F9037-100ML), store at 4 Â°C</li><li>20x SSC buffer (Sigma-Aldrich, catalog number: S6639), store at 4 Â°C</li><li>1x Phosphate Buffered Saline (Hyclone, catalog number: SH30256.01), store at 4 Â°C</li><li>DNase/RNase-Free Water (Solarbio, catalog number: R1600), store at 4 Â°C</li><li>Hydrogen peroxide solution, 30 wt.% in H<sub>2</sub>O (Macklin, catalog number: H811240-500 ml), store at 4 Â°C</li><li>DAB Immunohistochemistry Color Development Kit (BBI LIFE SCIENCES, catalog number: EP670033), store at -20 Â°C</li><li>EDTA (Sigma-Aldrich, catalog number: RDD017-500G), store at RT</li><li>Serum-free medium, DMEM/HIGH GLUCOSE (Hyclone, catalog number: SH30022.01), store at 4 Â°C</li><li>Penicillin-Streptomycin (Hyclone, catalog number: SV30010), store at -20 Â°C</li><li>Culture media (50 ml) (see Recipes)</li><li>10x PBS (500 ml), pH 6.8 (see Recipes)</li><li>1x DEPC-PBS (1 L), pH 7.4 (see Recipes)</li><li>1x DEPC-PBST (1 L) (see Recipes)</li><li>4% (w/v) PFA (500 ml) (see Recipes)</li><li>1x TE buffer (100 ml), pH 8.0 (see Recipes)</li><li>0.5%(v/v) Triton X-100 (5 ml) (see Recipes)</li><li>3% (w/v) H<sub>2</sub>O<sub>2</sub> solution (50 Î¼l) (see Recipes)</li><li>2x hybridization buffer (5 ml) (see Recipes)</li><li>Washing buffer (15 ml) (see Recipes)</li><li>2x detection buffer (5 ml) (see Recipes)</li><li>Staining buffer (50 Î¼l) (see Recipes)</li><li>0.1% hydrochloric acid-ethanol (5 ml) (see Recipes)</li><li>Hematoxylin Staining Solution (50 ml) (see Recipes)</li><li>25 mM dNTPs (100 Î¼l) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti31179">### Equipment</p><ol class="npwtwok"><li>Vortex Mixers (MIULAB, model: MIX-25P)</li><li>Microcentrifuge (MIULAB, model: Mini-6K)</li><li>Magnetic stirrer (IKA, RCT basic)</li><li>pH meter (OHAUS, model: starter3100)</li><li>PCR machine (BIOER, model: TC-XP)</li><li>Analytical balance (OHAUS, model: AS220.R2)</li><li>Pipette (Eppendorf AG, Research plus)</li><li>Cell culture incubator (Esco, model: CCL-170B-8)</li><li>Biosafety cabinet (Esco, model: AC2-4S1)</li><li>Ultra-low temperature freezer (Thermo, model: DW-86L338)</li><li>Freezer (Siemens, model: KG32EV2S0C)</li><li>Electro-heating standing-temperature cultivator (ENXIN, model: DRP-9052)</li><li>Microscope (Leica, model: DM6B, equipped with a DFC7000T camera)<br/></li></ol><p class="pptt" id="biaoti31180">### Software</p><ol class="npwtwok"><li>CellProfiler</li><li>GraphPad Prism<br/></li></ol><p class="pptt" id="biaoti31181">### Procedure</p><ol class="npwone"><li>Cell culture and sample preparation<ol class="npwtwo"><li>Pre-warm 2 ml 1x DEPC-PBS, 23 ml culture media and 0.8 ml 0.25% (w/v) trypsin-EDTA to 37 Â°C.</li><li>Remove the used culture media in the cell culture flask (take 25 cm<sup>2</sup> flask for example).</li><li>Wash briefly with 2 ml 1x DEPC-PBS. Aspirate PBS.</li><li>Add 0.8 ml 0.25% (w/v) trypsin-EDTA into flask and cover the cells.</li><li>Check the cells with inverted microscope to be sure that most of the cells are rounded up, but still attached to the flask. Carefully remove trypsin.</li><li>Add 3 ml culture media to the flask, suspend the cells from attached cell layer and mix them well by pipetting.</li><li>Put 5 slides into a Petri dish and add 20 ml culture media into it to cover the slides.</li><li>Seed the resuspended cells on slides.</li><li>Rest for at least 10 min until resuspended cells subside on the slides.</li><li>Transfer the Petri dish into the cell culture incubator (37 Â°C, 5% CO<sub>2</sub>).</li><li>Culture the cells until they reach 80% confluence (12-48 h).</li><li>After 12-48 h incubation, remove and discard the culture media.</li><li>Wash 3 times with 25 ml 1x DEPC-PBS for 3 min each.</li><li>Add 25 ml 4% (w/v) PFA into the Petri dish to cover the slides for 30 min at RT.</li><li>Wash twice with 25 ml 1x DEPC-PBS for 3 min each.</li><li>Perform dehydration using an ethanol series of 70%, 85%, and ethanol absolute for 5 min each. Air dry.</li></ol><em>Note: The sample slides can be stored at -80 Â°C until used.</em><br/><br/></li><li>Pretreatments<ol class="npwtwo"><li>Phosphorylate padlock probes before use. Prepare 50 Î¼l mix containing 38 Î¼l DEPC-H<sub>2</sub>O, 1 Î¼l padlock probes (100 Î¼M), 5 Î¼l PNK buffer A (10x), 5 Î¼l ATP (10 mM), 1 Î¼l T4 polynucleotide kinase (10 U/Î¼l) in the 0.2 ml PCR Tubes.</li><li>Incubate the above mixture at 37 Â°C for 30 min and 65 Â°C for 10 min in the PCR machine.</li></ol><em>Note: For formalin-fixed, paraffin-embedded pathological tissue sections, perform pretreatments procedures 3 to 20, expect 17 and 18. For cell samples, perform pretreatments procedures 14 to 20.</em> <ol class="npwtwo" start="3"><li>Bake the slide at 60 Â°C for 30 min.</li><li>Put the slide into the xylene for 15 min and another fresh xylene for 10 min, respectively.</li><li>Put the slide into ethanol absolute twice, 95% ethanol twice, and 70% ethanol twice for 2 min each time.</li><li>Treat the slide with DEPC-H<sub>2</sub>O for 5 min to complete rehydration.</li><li>Wash with DEPC-PBS for 2 min.</li><li>Perform the fixation procedure with 4% (w/v) PFA in DEPC-PBS for 10 min.</li><li>Wash with DEPC-PBS for 2 min.</li><li>Put the slide into the solution of 0.1 mg/ml pepsin in 0.1 M HCl for 30 min at 37 Â°C.</li><li>Wash with DEPC-H<sub>2</sub>O for 5 min.</li><li>Wash with DEPC-PBS for 2 min.</li><li>Dehydrate the slide by an an ethanol series of 70%, 85%, and ethanol absolute for 1 min each.</li><li>Air dry.</li><li>Assemble the 9 mm diameter and 0.8 mm depth Secure-Seal hybridization chamber to the surface of the slide on the side where the samples are (Figure 1). All the following incubation step take place in the chamber with a volume of 50 Î¼l.<br/><br/><img alt="" class="layerphoto" height="155" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115182305_5173.jpg" width="100"/><br/><strong>Figure 1. Assemble hybridization chamber to the surface of slide where the samples are.</strong> Place a seal-tab over each filling port to prevent evaporation of the solutions and the drying out of the specimen.<br/><br/></li><li>Rinse the cell sample with DEPC-PBS with 0.05% Tween-20 (DEPC-PBST).</li><li>Add 50 Î¼l 0.5%(v/v) Triton X-100 (in DEPC-PBS) into the chamber to permeabilize the cells for 10 min at RT.</li><li>Wash 3 times with 1x DEPC-PBST.</li><li>Apply 50 Î¼l 3% (w/v) H<sub>2</sub>O<sub>2</sub> solution to the sample for 15 min at RT to block endogenous peroxidases.</li><li>Wash 3 times with DEPC-PBST.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Padlock probe hybridization<ol class="npwtwo"><li>Prepare 50 Î¼l padlock probe hybridization mix containing 22.5 Î¼l DEPC-H<sub>2</sub>O, 25 Î¼l 2x hybridization buffer and 2.5 Î¼l phosphorylated padlock probe (2 Î¼M, from B2).</li><li>Add the mix into the chamber and incubate for 2 h at 37 Â°C.</li><li>Wash 3 times with washing buffer for 5 min each.</li><li>Wash 3 times with DEPC-PBST.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Padlock probe ligation<ol class="npwtwo"><li>Prepare 50 Î¼l padlock probe ligation mix containing 6.5 Î¼l DEPC-H<sub>2</sub>O, 31.25 Î¼l 80% (v/v) glycerol, 5 Î¼l BSA (2 mg/ml), 1.25 Î¼l RiboLock RNase Inhibitor (40 U/Î¼l), 5 Î¼l SplintR ligase reaction buffer (10x) and 1 Î¼l SplintR ligase (25 U/Î¼l).</li><li>Add the mix into the chamber and incubate for 1 h at 37 Â°C.</li><li>Wash 3 times with washing buffer for 5 min each.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RCA primer hybridization<ol class="npwtwo"><li>Prepare 50 Î¼l RCA primer hybridization mix containing 20 Î¼l DEPC-H<sub>2</sub>O, 25 Î¼l 2x hybridization buffer and 5 Î¼l RCA primer (10 Î¼M).</li><li>Add the mix into the chamber and incubate for 30 min at 37 Â°C</li><li>Wash 3 times with washing buffer for 5 min each.</li><li>Wash 3 times with DEPC-PBST.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RCA<ol class="npwtwo"><li>Prepare 50 Î¼l RCA mix containing 26.75 Î¼l DEPC-H<sub>2</sub>O, 5 Î¼l 50% (v/v) glycerol, 2 Î¼l dNTP (25 mM), 5 Î¼l BSA (2 mg/ml), 1.25 Î¼l RiboLock RNase Inhibitor (40 U/Î¼l), 5 Î¼l phi29 DNA polymerase buffer (10x) and 5 Î¼l phi29 polymerase (10 U/Î¼l).</li><li>Add the mix into the chamber and incubate for 16-18 h at RT.</li><li>Wash 3 times with washing buffer for 5 min each.<br/></li><li>Wash 3 times with DEPC-PBST.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RCA products detection<ol class="npwtwo"><li>Prepare 50 Î¼l RCA products detection mix containing 10 Î¼l DEPC-H<sub>2</sub>O, 25 Î¼l 2x detection buffer, 12.5 Î¼l 80% (v/v) glycerol and 2.5 Î¼l HRP-labeled detection probe (2 Î¼M).</li><li>Add the mix into the chamber and incubate for 30 min at RT.</li><li>Wash 3 times with DEPC-PBST.</li></ol><br/></li><li>Staining and mounting<ol class="npwtwo"><li>Incubate the sample with staining solution for 3 to 8 min depending on the color development.</li><li>Wash 3 times with DEPC-H<sub>2</sub>O.</li><li>Stain the nuclei in Hematoxylin staining solution for 1 min.</li><li>Wash 3 times with DEPC-H<sub>2</sub>O.</li><li>Differentiate with 0.1% hydrochloric acid-ethanol for 1 min.</li><li>Wash 3 times with DEPC-H<sub>2</sub>O.</li><li>Place slide in 1x DEPC-PBS for 1 min.</li><li>Wash 3 times with DEPC-H<sub>2</sub>O.</li><li>Remove the chamber and rinse in running tap water for 5 min.</li><li>Dehydrate the slide by an ethanol series of 70%, 85%, and ethanol absolute for 1 min each.</li><li>Mount the slide with 15-20 Î¼l neutral balsam mounting medium.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Image acquisition<ol class="npwtwo"><li>Acquire images with bright-filed microscope using the 40x objective.</li><li>In images, brown dots are detected individual mRNA molecules and nuclei are shown in blue (Figures 2 and 3).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115182406_1598.jpg" width="133"/><br/><strong>Figure 2. Detection of ACTB mRNA molecules in MCF-7 cells.</strong> Scale bar = 40 Î¼m.<br/><br/><img alt="" class="layerphoto" height="150" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115182428_5328.jpg" width="100"/><br/><strong>Figure 3. Detection of HER2 mRNA molecules in HER2-positive and HER2-negative breast cancer FFPE tissue sections.</strong> Scale bars = 40 Î¼m.</li></ol></li></ol><p class="pptt" id="biaoti31182">### Data analysis</p><p class="ppzz">Quantitative analysis of detected RNA molecules is achieved by using CellProfiler image analysis software. The images are processed with âUnmixcolorsâ, âEnhanceOrSuppressFeaturesâ, âIndentifyPrimaryObjectsâ, âIndentifySecondaryObjectsâ, âIndentifyPrimaryObjectsâ and âRelateObjectsâ modules in turn. To remove the background, the âUnmixcolorsâ module is used to split into hematoxylin and DAB two channels. The âEnhanceOrSuppressFeaturesâ module is used to highlight DAB staining spots. Finally, the relation of signals with their host cells is identified with the âRelateObjectsâ module. Then the number of RCPs per cell was listed. Statistical plotting was performed using GraphPad Prism.</p><p class="pptt" id="biaoti31183">### Notes</p><ol class="npwtwo"><li>For FFPE tissue sections and fresh frozen tissue sections, the pretreatment is different due to different kinds of tissue, including permeabilization and fixation.</li><li>Oligonucleotides were dissolved and diluted with 1x TE buffer. The stock concentration of the padlock probe, RCA primer and detection probe were 100 Î¼M, 10 Î¼M and 2 Î¼M respectively.</li><li>According to the cell type, adjust the experimental conditions of nucleic counterstaining, such as staining time, concentration of stain fluid.</li><li>DEPC is heat inactivated before using the solution.</li><li>Aspirate the solutions from the previous steps before adding a new solution into the chamber.<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong>Troubleshooting:</strong> A list of possible reasons for experiment failures and solutions are listed in Table 1.<br/><br/><strong>Table 1. Troubleshooting table</strong><br/><img alt="" class="layerphoto" height="260" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115182736_8083.jpg" width="600"/><br/><p class="pptt" id="biaoti31184">### Recipes</p><ol class="npwtwok"><li>Culture media (50 ml)<br/>45 ml serum-free medium<br/>5 ml fetal bovine serum<br/>0.5 ml Penicillin-Streptomycin Solution<br/>Store at 4 Â°C</li><li>10x PBS (500 ml), pH 6.8<br/>40 g NaCl<br/>1 g KCl<br/>7.2 g Na<sub>2</sub>HPO<sub>4</sub><br/>1.2 g KH<sub>2</sub>PO<sub>4</sub><br/>500 ml ultrapure water<br/>Filter the solution using 0.22 Î¼m membrane filter<br/>Prepare fresh</li><li>1x DEPC-PBS (1 L), pH 7.4<br/>100 ml 10x PBS<br/>900 ml ultrapure water<br/>1 ml DEPC<br/>Store at 4 Â°C</li><li>1x DEPC-PBST (1 L)<br/>1 L DEPC-PBS<br/>0.5 ml Tween-20<br/>Store at 4 Â°C</li><li>4% (w/v) PFA (500 ml)<br/>For a 4% paraformaldehyde solution:<ol class="npwthree"><li>Add 20 g of EM grade paraformaldehyde to 480 ml of 1x DEPC-PBS</li><li>Add 5 ml of 5 M NaOH and stir gently on a heating block at ~30 Â°C until the paraformaldehyde is dissolved</li><li>Allow the mixture to cool to room temperature and adjust the pH to 7.4 with HCl</li><li>Then adjust the final volume to 500 ml with 1x DEPC-PBS</li><li>Filter the solution through a 0.22 Î¼m membrane filter to remove any particulate matter</li><li>Make the paraformaldehyde solution fresh prior to use, or store in aliquots at -20 Â°C for several months. Avoid repeated freeze/thawing</li></ol></li><li>1x TE buffer (100 ml), pH 8.0<br/>1 ml 1 M Tris-HCl (pH 8.0)<br/>0.2 ml 0.5 M EDTA (pH 8.0) <br/>98.8 ml DEPC-H<sub>2</sub>O<br/>Filter the solution using 0.22 Î¼m membrane filter<br/>Store at 4 Â°C</li><li>0.5%(v/v) Triton X-100 (5 ml)<br/>25 Î¼l Triton X-100<br/>4.975 ml DEPC-H<sub>2</sub>O<br/>Store at RT</li><li>3% (w/v) H<sub>2</sub>O<sub>2</sub> solution (50 Î¼l)<br/>5 Î¼l 30% (w/v) H<sub>2</sub>O<sub>2</sub><br/>45 Î¼l DEPC-H<sub>2</sub>O<br/>Prepare fresh</li><li>2x hybridization buffer (5 ml)<br/>3 ml 20x SSC<br/>1 ml Formamide<br/>1 ml DEPC-H<sub>2</sub>O<br/>Store at RT</li><li>Washing buffer (15 ml)<br/>1.5 ml 20x SSC<br/>3 ml Formamide<br/>10.5 ml DEPC-H<sub>2</sub>O<br/>Store at RT</li><li>2x detection buffer (5 ml)<br/>1 ml 20x SSC<br/>5 Î¼l Tween-20<br/>0.2 ml 20 mg/ml BSA<br/>3.975 ml DEPC-H<sub>2</sub>O<br/>Store at 4 Â°C</li><li>Staining buffer (50 Î¼l)<br/>45 Î¼l DAB buffer<br/>2.5 Î¼l 20x DAB substrate solution<br/>2.5 Î¼l 20x DAB chromogen solution<br/>Prepare fresh</li><li>0.1% hydrochloric acid-ethanol (5 ml)<br/>50 Î¼l hydrochloric acid<br/>3.713 ml ethanol<br/>1.237 ml DEPC-H<sub>2</sub>O<br/>Store at RT</li><li>Hematoxylin Staining Solution (50 ml)<ol class="npwthree"><li>Dissolve 0.1 g hematoxylin with 12.5 ml ethanol</li><li>Dissolve 0.85 g aluminum potassium sulfate dodecahydrate with 37.5 ml DEPC-H<sub>2</sub>O</li><li>Then mix the two solutions</li><li>After hematoxylin is oxidized to a purplish red by adding 10 mg sodium iodate, add 0.5 ml acetate</li><li>Filter the solution using 0.22 Î¼m membrane filter. Store at RT</li><li>Concentration of hematoxylin staining solution can be diluted with DEPC-H<sub>2</sub>O</li></ol></li><li>25 mM dNTP (100 Î¼l)<br/>25 Î¼l 100 mM ATP<br/>25 Î¼l 100 mM GTP<br/>25 Î¼l 100 mM CTP<br/>25 Î¼l 100 mM TTP<br/>Store at -20 Â°C<br/></li></ol><p class="pptt" id="biaoti31185">### Competing interests</p><p class="ppzz">Authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti31186">### Acknowledgments</p><p class="ppzz">This work was supported by the funds from the National Natural Science Foundation of China (31770927), the National Key Research and Development Program of China (2017YFA0106800), the Natural Science Foundation of Fujian Province, and the Scientific Research Funds of Huaqiao University.</p><p class="pptt" id="biaoti31187">### References</p><ol class="npwtwok"><li>BanÃ©r, J., Nilsson, M., Mendel-Hartvig, M. and Landegren, U. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9801302" target="_blank">Signal amplification of padlock probes by rolling circle replication.</a> <em>Nucleic Acids Res</em> 26(22): 5073-5078.</li><li>Battich, N., Stoeger, T. and Pelkmans, L. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24097269" target="_blank">Image-based transcriptomics in thousands of single human cells at single-molecule resolution.</a> <em>Nat Methods</em> 10(11): 1127-1133.</li><li>Crosetto, N., Bienko, M. and van Oudenaarden, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25446315" target="_blank">Spatially resolved transcriptomics and beyond.</a> <em>Nat Rev Genet</em> 16(1): 57-66.</li><li>Femino, A. M., Fay, F. S., Fogarty, K. and Singer, R. H. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9554849" target="_blank">Visualization of single RNA transcripts <em>in situ.</em></a> <em>Science</em> 280(5363): 585-590.</li><li>Jiang, M., Liu, L., Hong, C., Chen, D., Yao, X., Chen, X., Lin, C. and Ke, R. (2019). <a href="https://www.researchgate.net/publication/332928161_Single_molecule_chromogenic_in_situ_hybridization_assay_for_RNA_visualization_in_fixed_cells_and_tissues" target="_blank">Single molecule chromogenic <em>in situ</em> hybridization assay for RNA visualization in fixed cells and tissues.</a> <em>RNA</em> 25(8): 1038-1046.</li><li>Larsson, C., Grundberg, I., Soderberg, O. and Nilsson, M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20383134" target="_blank"><em>In situ</em> detection and genotyping of individual mRNA molecules.</a> <em>Nat Methods</em> 7(5): 395-397.</li><li>Nilsson, M., Malmgren, H., Samiotaki, M., Kwiatkowski, M., Chowdhary, B. P. and Landegren, U. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7522346" target="_blank">Padlock probes: circularizing oligonucleotides for localized DNA detection.</a> <em>Science</em> 265(5181): 2085-2088.</li><li>Shah, S., Lubeck, E., Schwarzkopf, M., He, T. F., Greenbaum, A., Sohn, C. H., Lignell, A., Choi, H. M., Gradinaru, V., Pierce, N. A. and Cai, L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27342713" target="_blank">Single-molecule RNA detection at depth by hybridization chain reaction and tissue hydrogel embedding and clearing.</a> <em>Development</em> 143(15): 2862-2867.</li><li>Wang, F., Flanagan, J., Su, N., Wang, L. C., Bui, S., Nielson, A., Wu, X., Vo, H. T., Ma, X. J. and Luo, Y. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22166544" target="_blank">RNAscope: a novel <em>in situ</em> RNA analysis platform for formalin-fixed, paraffin-embedded tissues.</a> <em>J Mol Diagn</em> 14(1): 22-29.<br/></li></ol>