<body>
            <h1>Structural Alignment and Covariation Analysis of RNA Sequences</h1>
            <p><strong>Authors:</strong> Nicolas J. Tourasse, Fabien Darfeuille</p>
            <h2>Abstract</h2>
            <p>RNA molecules adopt defined structural conformations that are essential to exert their function. During the course of evolution, the structure of a given RNA can be maintained via compensatory base-pair changes that occur among covarying nucleotides in paired regions. Therefore, for comparative, structural, and evolutionary studies of RNA molecules, numerous computational tools have been developed to incorporate structural information into sequence alignments and a number of tools have been developed to study covariation. The bioinformatic protocol presented here explains how to use some of these tools to generate a secondary-structure-aware multiple alignment of RNA sequences and to annotate the alignment to examine the conservation and covariation of structural elements among the sequences.</p>
            <p><strong>Keywords:</strong> RNA, Sequence, Structure, Alignment, Covariation, Comparative analysis</p>
            <h2>Background</h2><p>Biological RNA molecules fold into specific secondary (2D) and tertiary (3D) structures that are critical for their function. Therefore, for comparative analysis, which usually requires sequence alignment, it is desirable to take structure information into account in order to obtain a more reliable and meaningful alignment. Numerous computational algorithms and tools have been developed to generate alignments based on secondary structure, such as MAFFT (Katoh and Toh, 2008), TurboFold (Tan <em>et al.</em>, 2017), R-Coffee (Wilm <em>et al.</em>, 2008), locARNA (Will <em>et al.</em>, 2007), ProbCons (Do <em>et al.</em>, 2005), MXSCARNA (Tabei <em>et al.</em>, 2008), and LaRA (Bauer <em>et al.</em>, 2007). In a benchmark comparison of these leading tools (Tan <em>et al.</em>, 2017) TurboFold and MAFFT were shown to have comparable and highest accuracies. The running time of TurboFold is considerably longer than that of MAFFT (Katoh and Toh, 2008; Tan <em>et al.</em>, 2017), thus in this protocol we use MAFFT because its speed, which can be further augmented by parallel processing (Katoh and Standley, 2013), allows alignment of a large number (&gt;100) of sequences in a limited amount time. Like several of the other tools MAFFT employs an iterative strategy where pairwise structural alignments are first computed and are then progressively combined into a multiple alignment through several rounds of refinement.<br/> <br/>Because of the tight structure-function relationship, functional RNAs undergo a selection pressure to maintain their structures (Nowick <em>et al.</em>, 2019). This is reflected by the occurrence of covarying consistent or compensatory mutations in paired nucleotides that can be observed in sequence alignments. Covariation data are therefore very valuable and have been used to validate or predict the secondary, and even tertiary, structure of RNAs and to understand their evolution (Michel and Westhof, 1990; Cannone <em>et al.</em>, 2002). A number of software tools are available for examining covariation within alignments, such as the structural alignment editors RALEE (Griffiths-Jones, 2005), 4SALE (Seibel <em>et al.</em>, 2006), S2S (Jossinet and Westhof, 2005), ConStruct (Wilm <em>et al.</em>, 2008b), or SARSE (Andersen <em>et al.</em>, 2007), R-chie (Lai <em>et al.</em>, 2012), a tool that scores and annotates covariation, and complex programs that include methods for performing statistical analysis of covariation with or without a phylogenetic framework such as R-scape (Rivas <em>et al.</em>, 2017) and CoMap (Dutheil, 2012). R-chie highlights basepairs and employs arc diagrams to represent the secondary structure alongside the alignment, and can generate highly customizable figures.<br/><br/>In the protocol below, we explain how to use MAFFT to compute a structural alignment of multiple RNA sequences, and how to use R-chie to annotate the alignment with conservation and covariation information.</p>
        </body>

<p class="pptt" id="biaoti31267">### Equipment</p><ol class="npwtwok"><li>Personal computer, preferably with multiple processors (CPUs) to speed up computations<br/>A Unix/Linux operating system is preferred. All software mentioned here, except the optional LaRA program, can also be run under Mac and Windows systems. For Windows, a terminal or Linux emulator such as Cygwin (<a href="http://www.cygwin.com/" target="_blank">http://www.cygwin.com/</a>) or Ubuntu (<a href="https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6" target="_blank">https://www.microsoft.com/store/p/ubuntu/9nblggh4msv6</a>) is needed. In any case, familiarity with the use of command-line-driven applications is required.<br/></li></ol><p class="pptt" id="biaoti31268">### Software</p><ol class="npwtwo"><li>MAFFT (Katoh and Toh, 2008, <a href="https://mafft.cbrc.jp/alignment/software/source.html" target="_blank">https://mafft.cbrc.jp/alignment/software/source.html</a>)</li><li>R (R Development Core Team, 2018, <a href="http://www.r-project.org/" target="_blank">http://www.R-project.org/</a>)</li><li>R-chie (Lai <em>et al.</em>, 2012, <a href="https://www.e-rna.org/r-chie/" target="_blank">https://www.e-rna.org/r-chie/</a>)</li><li>(Optional) MXSCARNA (Tabei <em>et al.</em>, 2008, <a href="https://www.ncrna.org/softwares/mxscarna/" target="_blank">https://www.ncrna.org/softwares/mxscarna/</a>)</li><li>(Optional) LaRA (Bauer <em>et al.</em>, 2007, <a href="http://www.mi.fu-berlin.de/w/LiSA/Lara" target="_blank">http://www.mi.fu-berlin.de/w/LiSA/Lara</a>)</li><li>(Optional) FOLDALIGN (Sundfeld <em>et al.</em>, 2016, <a href="http://rth.dk/resources/foldalign" target="_blank">http://rth.dk/resources/foldalign</a>)</li></ol><em>Notes:</em> <ol class="npwthree"><li><em>The MAFFT package is provided in different forms. Be sure to download the bundle that provides support for RNA structural alignment, such as the package with extensions for Unix/Linux, the Standard package for Mac, and the Ubuntu or Cygwin version for Windows.</em> </li><li><em>It is not necessary to install MXSCARNA separately as it is included within the MAFFT package.</em> </li><li><em>Install LaRA and/or FOLDALIGN only if you want to use them as alternatives to MXSCARNA. LaRA runs on Linux only. LaRA version 1.3 may need to be used as the later versions 1.31 and 1.32 frequently abort.</em> </li></ol><p class="pptt" id="biaoti31269">### Procedure</p><ol class="npwtwok"><li>Prepare a set of related (homologous) RNA sequences to analyze, either using local sequences or by downloading sequences from a database. Sequences can be retrieved via keyword searches from general-purpose databases such as NCBI GenBank/RefSeq (<a href="https://www.ncbi.nlm.nih.gov/" target="_blank">https://www.ncbi.nlm.nih.gov/</a>), ENA (<a href="https://www.ebi.ac.uk/ena" target="_blank">https://www.ebi.ac.uk/ena</a>), or Ensembl (<a href="http://www.ensembl.org/" target="_blank">http://www.ensembl.org/</a>), or from specialized RNA databases such as SILVA (<a href="https://www.arb-silva.de/" target="_blank">https://www.arb-silva.de/</a>), ncRNA databases (<a href="https://ncrnadatabases.org/" target="_blank">https://ncrnadatabases.org/</a>), RNAcentral (<a href="https://rnacentral.org/" target="_blank">https://rnacentral.org/</a>), the Comparative RNA Website (<a href="http://www.rna.icmb.utexas.edu/" target="_blank">http://www.rna.icmb.utexas.edu/</a>), or Rfam (<a href="https://rfam.org/" target="_blank">https://rfam.org/</a>; <em>e.g.</em>, see the Nucleic Acids Research website for a non-exhaustive listing of RNA sequence databases; <a href="http://www.oxfordjournals.org/nar/database/cat/2" target="_blank">http://www.oxfordjournals.org/nar/database/cat/2</a>). Some of the RNA databases (<em>e.g.</em>, SILVA, Rfam, Comparative RNA Website) provide sequences that are already aligned in the form of structure-aware multiple alignments. Homologs of RNA sequences of interest can also be identified by sequence similarity search, <em>e.g.</em>, using the well-known BLASTN tool. All general databases and some of the RNA databases provide a BLAST service. One can also search for related RNA sequences at the structural level with the help of covariance models using INFERNAL (Nawrocki and Eddy, 2013) or CMfinder (Yao <em>et al.</em>, 2006).<br/>  All collected sequences must be put in a single file in the commonly used FASTA format (<a href="https://en.wikipedia.org/wiki/FASTA_format" target="_blank">https://en.wikipedia.org/wiki/FASTA_format</a>; Figure 1).<br/><em>Note: The composition of the dataset may influence the analysis of covariation, depending on the amount of similarity or dissimilarity between the sequences and their phylogenetic relationships.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115221314_8538.jpg" width="196"/><br/><strong>Figure 1. Example of unaligned RNA sequences in FASTA format (visualized in Emacs)</strong><br/><br/></li><li>Compute a structural multiple alignment by running MAFFT in the âX-INS-Iâ mode:<br/><span style="font-family:Courier New;">mafft-xinsi --scarnapair --nuc --reorder --maxiterate <em>max_number_of_iterations</em> --thread <em>number_of_CPUs_to_use sequence_file.fasta</em> 1&gt; <em>mafft_alignment.fasta</em> 2&gt; <em>mafft_alignment_details.log</em></span><br/><br/>The alignment generated will be in FASTA format (Figure 2).<br/><em>Notes:</em> <ol class="npwthree"><li><em>The option â--scarnapairâ instructs MAFFT to use MXSCARNA to perform pairwise structural alignments, which is the default option. To use the LaRA aligner instead, invoke the pair of options â--larapair --laraparams parameter_fileâ, where âparameter_fileâ is a file with LaRA configuration parameters. A template file âlara.paramsâ is provided with the LaRA software. To use the FOLDALIGN aligner, invoke the option â--foldalignlocalpairâ or â--foldalignglobalpairâ to perform local or global pairwise alignment, respectively. In benchmarking comparisons (Katoh and Toh, 2008) the accuracy of MXSCARNA was generally higher than that of LaRA, except when the identity among the sequences was low (&lt;40%), in which case LaRA may be preferred. FOLDALIGN is another structural alignment program that is highly accurate and that can carry out structural alignments of sequences with low similarity ( Havgaard </em><em>et al.</em><em>, 2005; Sundfeld </em><em>et al.</em><em>, 2016).</em> </li><li><em>To obtain high accuracy alignments, use a large number of iterative refinements, </em><em>e.g.</em><em>, set âmax_number_of_iterationsâ to 1,000 for the â--maxiterateâ optionâ.</em> </li><li><em>For the â--threadâ option, increase ânumber_of_CPUs_to_useâ to speed up alignment computation. Runtime increases with the number and length of the sequences. For example, for a set of 100 sequences of length 50-200 nt calculations can take 3-10 min on a single CPU, and under a minute when using at least 8 CPUs.</em> </li><li><em>In some environments, MAFFT may abort with the following error: âmafft-xinsi: line 2369: /dev/stderr: Not a directoryâ. The error can be solved by replacing â/dev/stderrâ by â/dev/nullâ in the mafft-xinsi bash script.</em> </li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115221352_3488.jpg" width="196"/><br/><strong>Figure 2. Structural alignment produced by MAFFT visualized in 4SALE</strong><br/><br/></li><li>(Optional) Predict a reference secondary structure:<br/>In order to reveal covariation, a reference secondary structure is needed. It can be the structure of one of the RNA sequences included in the analysis, a consensus structure inferred from the alignment, or an external structural model. In lack of a known or experimentally-determined model, the structure needs to be predicted. Prediction of the 2D structure of a single RNA sequence can be done with widely used tools such as MFOLD (Zuker, 2003), <a href="http://unafold.rna.albany.edu/?q=mfold" target="_blank">http://unafold.rna.albany.edu/?q=mfold</a>), RNAfold from the ViennaRNA package (Lorenz <em>et al.</em>, 2011), <a href="https://www.tbi.univie.ac.at/RNA/" target="_blank">https://www.tbi.univie.ac.at/RNA/</a>), Fold or MaxExpect from the RNAStructure package (Reuter and Mathews, 2010), <a href="https://rna.urmc.rochester.edu/RNAstructure.html" target="_blank">https://rna.urmc.rochester.edu/RNAstructure.html</a>), or RNAshapes from the RNA shapes studio (Janssen and Giegerich, 2015; <a href="https://bibiserv.cebitec.uni-bielefeld.de/rnashapesstudio" target="_blank">https://bibiserv.cebitec.uni-bielefeld.de/rnashapesstudio</a>). All these tools can be run on-line at dedicated webservers or installed locally as command-line programs. For example, standard commands to run MFOLD or RNAfold with default parameters on an RNA sequence in FASTA format would be:<br/><br/><span style="font-family:Courier New;">mfold SEQ=</span><em><span style="font-family:Courier New;">sequence_file.fasta</span></em><span style="font-family:Courier New;"> (structures will be output in files named â</span><em><span style="font-family:Courier New;">sequence_file*.ct</span></em><span style="font-family:Courier New;">â)</span><br/><span style="font-family:Courier New;"> RNAfold &lt; </span><em><span style="font-family:Courier New;">sequence_file.fasta</span></em><span style="font-family:Courier New;"> &gt; </span><em><span style="font-family:Courier New;">structure_file.b</span></em><br/><br/>The above-mentioned packages have numerous options to tune the folding computation (<em>e.g.</em>, by changing the algorithm, temperature, ionic conditions), and most of them provide the possibility to impose constraints on the structure.<br/>  There exists also several pieces of software for predicting consensus structures from alignments, such as RNAalifold (Bernhart <em>et al.</em>, 2008) from the ViennaRNA package, the graphical tool ConStruct (Wilm <em>et al.</em>, 2008b); <a href="http://www.biophys.uni-duesseldorf.de/construct3/" target="_blank">http://www.biophys.uni-duesseldorf.de/construct3/</a>), and RNAalishapes from the RNA shapes studio (VoÃ, 2006); <a href="https://bibiserv.cebitec.uni-bielefeld.de/rnaalishapes" target="_blank">https://bibiserv.cebitec.uni-bielefeld.de/rnaalishapes</a>). Notably, RNAalifold does not weight the sequences and is highly sensitive to the particular sample of sequences under study and the prediction can be affected by the varying amount of similarity between the sequences as well as by sequences containing insertions. ConStruct incorporates custom sequence weighting for optimal consensus prediction while RNAalishapes is based on the concept of abstract structural shapes and also includes gap-aware energy evaluation which makes it outperform RNAalifold when sequences contain insertions and/or alignment is of low quality (VoÃ, 2006). In our experience RNAalishapes performed quite well. RNAalishapes relies on older Linux libraries and may not install on current architectures, thus the on-line version may have to be used.<br/><br/></li><li>Analyze the covariation in the alignment and generate an image of the covariation-annotated alignment using R-chie:<br/>Covariation is revealed by mapping the reference secondary structure onto the multiple alignments. The structure can be provided to R-chie in one of the common formats used by structure prediction and analysis software, including dot-bracket (or Vienna; â.bâ), connect-table (â.ctâ), as well as bpseq (from The Comparative RNA Website [Cannone <em>et al.</em>, 2002], <a href="http://www.rna.icmb.utexas.edu/" target="_blank">http://www.rna.icmb.utexas.edu/</a>). The alignment must be in FASTA format.<br/><br/><span style="font-family:Courier New;">rchie.R --msafile=</span><em><span style="font-family:Courier New;">mafft_alignment.fasta</span></em><span style="font-family:Courier New;"> --pdf --output=</span><em><span style="font-family:Courier New;">alignment_covariation_figure.pdf</span></em><span style="font-family:Courier New;"> --format1=vienna --rule1=7 --group1=4 --legend1 --legend3 --msaspecies --msagrid --msatext </span><em><span style="font-family:Courier New;">reference_structure.b</span></em><span style="font-family:Courier New;"> &amp;&gt; </span><em><span style="font-family:Courier New;">rchie.log</span></em><br/><br/>This will create a figure of the alignment in PDF format, in which the paired regions in the reference secondary structure are represented by arcs colored according to the covariation score and nucleotides in the sequences are colored according to their base-pairing status (Figure 3).<br/><em>Notes:</em> <ol class="npwthree"><li><em>The above command runs with a reference structure in dot-bracket (â.bâ) format, which is indicated by the option â--format1=viennaâ. For a structure in connect (â.ctâ) or bpseq format â--format1=connectâ or â--format1=bpseqâ would be used, respectively.</em> </li><li><em>Option â--rule1=7â is used to group base-pairs on covariation scores and option â--group1=4â is used to set the number of groups to 4. Various other criteria can be chosen for grouping; run ârchie.R --helpâ for details.</em> </li><li><em>R-chie is highly customizable. Options â--legend1â, â--legend3â, â--msaspeciesâ, â--msagridâ, and â--msatextâ can be invoked or omitted at will in order to turn on or off legends, sequence names, and other graphical settings. Colors for every element in the image can be specified by additional options such as â--colour1â, â--palette1â, or â--msacolâ; run ârchie.R --helpâ for details.</em> </li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115221720_2329.jpg" width="285"/><br/><strong>Figure 3. Structure- and covariation-annotated alignment drawn by R-chie</strong><br/><br/></li><li>(Optional) If necessary, manually edit the alignment produced by MAFFT to correct errors or adjust the alignment using a structure-aware editor such as RALEE (Griffiths-Jones, 2005), <a href="http://sgjlab.org/ralee/" target="_blank">http://sgjlab.org/ralee/</a>), 4SALE (Seibel <em>et al.</em>, 2006); <a href="http://4sale.bioapps.biozentrum.uni-wuerzburg.de/" target="_blank">http://4sale.bioapps.biozentrum.uni-wuerzburg.de/</a>), S2S (Jossinet and Westhof, 2005); <a href="http://bioinformatics.org/assemble/" target="_blank">http://bioinformatics.org/assemble/</a>), ConStruct (Wilm <em>et al.</em>, 2008b); <a href="http://www.biophys.uni-duesseldorf.de/construct3/" target="_blank">http://www.biophys.uni-duesseldorf.de/construct3/</a>), or SARSE (Andersen <em>et al.</em>, 2007). Then, regenerate the covariation-annotated figure using R-chie.<br/></li></ol><p class="pptt" id="biaoti31270">### Data analysis</p><p class="ppzz">The protocol described here has been used to examine the covariation within metastable regions in a structural alignment of 107 mRNAs of type I toxin-antitoxin systems from bacteria of the genus <em>Helicobacter</em> (Masachis <em>et al.</em>, 2019, Figure 8, figure supplement 1 and 2 therein). All files including the unaligned sequences (in FASTA format), the unedited alignment produced by MAFFT (run with the â--scarnapairâ option; in FASTA format), the manually-edited alignment (in FASTA format), the reference secondary structure (in dot-bracket/Vienna format), and the covariation-annotated alignment drawn by R-chie (in PDF format) for both datasets (metastable 1 and 2) analyzed in Masachis <em>et al.</em> (2019) are included as <a href="https://os.bio-protocol.org/attached/file/20191218/Bio protocol Tourasse Darfeuille datafiles.zip" target="_blank">supplementary material</a>, to allow the user to learn and reproduce the analyses.</p><p class="pptt" id="biaoti31271">### Notes</p><p class="ppzz">In this protocol, we employ MAFFT to compute structural alignments, but the procedure may be performed similarly using any other tools that produce structure-aware sequence alignments.</p><p class="pptt" id="biaoti31272">### Acknowledgments</p><p class="ppzz">This protocol was derived from our original study of metastable structures in mRNAs of type I toxin-antitoxin systems in the bacterium <em>Helicobacter pylori</em> (Masachis <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti31273">### Competing interests</p><p class="ppzz">The authors declare that there are no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti31274">### References</p><ol class="npwtwok"><li>Andersen, E. S., Lind-Thomsen, A., Knudsen, B., Kristensen, S. E., Havgaard, J. H., Torarinsson, E., Larsen, N., Zwieb, C., Sestoft, P., Kjems, J. and Gorodkin, J. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17804647" target="_blank">Semiautomated improvement of RNA alignments.</a> <em>RNA</em> 13(11): 1850-1859.</li><li>Bauer, M., Klau, G. W. and Reinert, K. (2007). <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-8-271" target="_blank">Accurate multiple sequence-structure alignment of RNA sequences using combinatorial optimization.</a> <em>BMC Bioinformatics</em> 8: 271.</li><li>Bernhart, S. H., Hofacker, I. L., Will, S., Gruber, A. R. and Stadler, P. F. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19014431" target="_blank">RNAalifold: Improved consensus structure prediction for RNA alignments.</a> <em>BMC Bioinformatics</em> 9: 474.</li><li>Cannone, J. J., Subramanian, S., Schnare, M. N., Collett, J. R., DâSouza, L. M., Du, Y., Feng, B., Lin, N., Madabusi, L. V., MÃ¼ller, K. M., Pande, N., Shang, Z., Yu, N. and Gutell, R. R. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11869452" target="_blank">The Comparative RNA Web (CRW) Site: An online database of comparative sequence and structure information for ribosomal, intron, and other RNAs.</a> <em>BMC Bioinformatics</em> 3: 2.</li><li>Do, C. B., Mahabhashyam, M. S. P., Brudno, M. and Batzoglou, S. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15687296" target="_blank">ProbCons: Probabilistic consistency-based multiple sequence alignment.</a> <em>Genome Res</em> 15(2): 330-340.</li><li>Dutheil, J. Y. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21949241" target="_blank">Detecting coevolving positions in a molecule: Why and how to account for phylogeny.</a> <em>Brief Bioinform</em> 13(2): 228-243.</li><li>Griffiths-Jones, S. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15377506" target="_blank">RALEE - RNA alignment editor in Emacs.</a> <em>Bioinformatics</em> 21(2): 257-259.</li><li>Havgaard, J. H., LyngsÃ¸, R. B., Stormo, G. D. and Gorodkin, J. (2005). <a href="https://academic.oup.com/bioinformatics/article/21/9/1815/409074" target="_blank">Pairwise local structural alignment of RNA sequences with sequence similarity less than 40%.</a> <em>Bioinformatics</em> 21(9): 1815-1824.</li><li>Janssen, S. and Giegerich, R. (2015). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4308662/" target="_blank">The RNA shapes studio.</a> <em>Bioinformatics</em> 31(3): 423-425.</li><li>Jossinet, F. and Westhof, E. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15905274" target="_blank">Sequence to Structure (S2S): Display, manipulate and interconnect RNA data from sequence to structure.</a> <em>Bioinformatics</em> 21(15): 3320-3321.</li><li>Katoh, K. and Standley, D. M. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23329690" target="_blank">MAFFT multiple sequence alignment software version 7: Improvements in performance and usability.</a> <em>Mol Biol Evol</em> 30(4): 772-780.</li><li>Katoh, K. and Toh, H. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18439255" target="_blank">Improved accuracy of multiple ncRNA alignment by incorporating structural information into a MAFFT-based framework.</a> <em>BMC Bioinformatics</em> 9: 212.</li><li>Lai, D., Proctor, J. R., Zhu, J. Y. A. and Meyer, I. M. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22434875" target="_blank">R-CHIE: A web server and R package for visualizing RNA secondary structures.</a> <em>Nucleic Acids Res</em> 40(12): e95.</li><li>Lorenz, R., Bernhart, S. H., HÃ¶ner zu Siederdissen, C., Tafer, H., Flamm, C., Stadler, P. F. and Hofacker, I. L. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22115189" target="_blank">ViennaRNA Package 2.0.</a> <em>Algorithms Mol Biol</em> 6: 26.</li><li>Masachis, S., Tourasse, N. J., Lays, C., Faucher, M., Chabas, S., Iost, I. and Darfeuille, F. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31411564" target="_blank">A genetic selection reveals functional metastable structures embedded in a toxin-encoding mRNA.</a> <em>Elife</em> 8: e47549.</li><li>Michel, F. and Westhof, E. (1990). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2258934" target="_blank">Modelling of the three-dimensional architecture of group I catalytic introns based on comparative sequence analysis.</a> <em>J Mol Biol</em> 216(3): 585-610.</li><li>Nawrocki, E. P. and Eddy, S. R. (2013). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3810854/" target="_blank">Infernal 1.1: 100-fold faster RNA homology searches.</a> <em>Bioinformatics</em> 29(22): 2933-2935.</li><li>Nowick, K., Walter Costa, M. B., HÃ¶ner zu Siederdissen, C. and Stadler, P. F. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31496634" target="_blank">Selection pressures on RNA sequences and structures.</a> <em>Evol Bioinforma</em> 15: 117693431987191.</li><li>R Development Core Team. (2018). R: A language and environment for statistical computing. Vienna, Austria.</li><li>Reuter, J. S. and Mathews, D. H. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20230624" target="_blank">RNAstructure: Software for RNA secondary structure prediction and analysis.</a> <em>BMC Bioinformatics</em> 11: 129.</li><li>Rivas, E., Clements, J. and Eddy, S. R. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27819659" target="_blank">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs.</a> <em>Nat Methods</em> 14(1): 45-48.</li><li>Seibel, P. N., MÃ¼ller, T., Dandekar, T., Schultz, J. and Wolf, M. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17101042" target="_blank">4SALE - A tool for synchronous RNA sequence and secondary structure alignment and editing.</a> <em>BMC Bioinformatics</em> 7: 498.</li><li>Sundfeld, D., Havgaard, J. H., De Melo, A. C. M. A. and Gorodkin, J. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26704597" target="_blank">Foldalign 2.5: Multithreaded implementation for pairwise structural RNA alignment.</a> <em>Bioinformatics</em> 32(8): 1238-1240.</li><li>Tabei, Y., Kiryu, H., Kin, T. and Asai, K. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18215258" target="_blank">A fast structural multiple alignment method for long RNA sequences.</a> <em>BMC Bioinformatics</em> 9: 33.</li><li>Tan, Z., Fu, Y., Sharma, G. and Mathews, D. H. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29036420" target="_blank">TurboFold II: RNA structural alignment and secondary structure prediction informed by multiple homologs.</a> <em>Nucleic Acids Res</em> 45(20): 11570-11581.</li><li>VoÃ, B. (2006). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1636479/" target="_blank">Structural analysis of aligned RNAs.</a> <em>Nucleic Acids Res</em> 34(19): 5471-5481.</li><li>Will, S., Reiche, K., Hofacker, I. L., Stadler, P. F. and Backofen, R. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17432929" target="_blank">Inferring noncoding RNA families and classes by means of genome-scale structure-based clustering.</a> <em>PLoS Comput Biol</em> 3(4): 680-691.</li><li>Wilm, A., Higgins, D. G. and Notredame, C. (2008a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18420654" target="_blank">R-Coffee: A method for multiple alignment of non-coding RNA.</a> <em>Nucleic Acids Res</em> 36(9): e52.</li><li>Wilm, A., Linnenbrink, K. and Steger, G. (2008b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18442401" target="_blank">ConStruct: Improved construction of RNA consensus structures.</a> <em>BMC Bioinformatics</em> 9: 219.</li><li>Yao, Z., Weinberg, Z. and Ruzzo, W. L. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16357030" target="_blank">CMfinder - A covariance model based RNA motif finding algorithm.</a> <em>Bioinformatics</em> 22(4): 445-452.</li><li>Zuker, M. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12824337" target="_blank">Mfold web server for nucleic acid folding and hybridization prediction.</a> <em>Nucleic Acids Res</em> 31(13): 3406-3415.<br/></li></ol>