<body>
            <h1>Expression and Purification of Adeno-associated Virus Virus-like Particles in a Baculovirus System and AAVR Ectodomain Constructs in <em>E. coli</em></h1>
            <p><strong>Authors:</strong> Nancy  Meyer, Omar  Davulcu, Qing  Xie, Mark  Silveria, Grant M. Zane, Edward  Large, Michael S. Chapman</p>
            <h2>Abstract</h2>
            <p>Adeno-associated virus (AAV) is a promising gene therapy vector and the biophysical characterization of its interactions with host proteins is a critical foundation for engineering tissue targeting and immune escape. Presented here are protocols for the production of: (a) the outer protein shells (virus-like particles or VLPs) for serotype 2 (AAV-2) and (b) two fragments from the binding ectodomain of AAVâs cellular receptor, AAVR. His<sub>6</sub>PKD1-2 comprises the first two polycystic kidney disease (PKD) domains, the minimal required for efficient binding of AAV, expressed with an N-terminal histidine tag. MBP-PKD1-5 is a fusion of the maltose binding protein with all five of the PKD domains of the AAVR receptor. Presented are the expression and purification of milligram quantities, ample for <em>in vitro</em> analyses. For AAV-2, the protocol offers an alternative to the use of (infectious) wild-type virus or transducing vectors. One of the methods for producing transducing vector is in Sf9 cells, and the production of VLPs is based on this. For AAVR, the protocols enable biochemical and biophysical characterization of virus-binding. The minimal two-domain construct allows more saturated binding to symmetry-equivalent sites on the virus, while the larger construct might be better expected to reflect the native receptor.</p>
            <p><strong>Keywords:</strong> Virus, Adeno-associated virus, AAV, AAVR, Viral entry, Co-receptor, Gene therapy, Virus receptor</p>
            <h2>Background</h2><p>The human parvovirus Adeno-associated virus (AAV) has a 60-subunit protein capsid shell containing a single-stranded DNA genome (Xie <em>et al.</em>, 2002; Chapman and Agbandje-McKenna 2006). Several recombinant AAV serotypes are employed as vectors in emerging clinical gene therapy applications (Gaudet <em>et al.</em>, 2013; Russell <em>et al.</em>, 2017). AAV infection starts with capsid attachment to serotype-specific extracellular glycans, followed by receptor-mediated endocytotic entry. The cellular receptor, AAVR, otherwise known as KIAA0319L, was implicated as key for entry of a panel of AAV serotypes into representative cell types and for transduction in mouse models (Pillay <em>et al.</em>, 2016); although it is now known that the AAV-4 clade, comprising three related AAVs, enter cells independently of AAVR (Dudek <em>et al.</em>, 2018). For the majority of AAVs, AAVR functions not only in cell entry, but in trafficking endosomally localized AAV towards the <em>peri</em>-nuclear <em>trans</em> Golgi network. This process likely mimics an uncharacterized host-cell function of the AAVR protein as a recycling receptor (Pillay <em>et al.</em>, 2016). AAVR is an integral transmembrane glycoprotein but AAVR's function in binding AAV is independent of its glycosylation state, thereby opening several possibilities of heterologous AAVR expression for <em>in vitro</em> study of its interactions with AAV (Pillay <em>et al.</em>, 2016 and 2017).<br/> <br/>Potential relevance for improving gene therapy delivery is driving intense interest in AAV-receptor interactions. It is presumed that interactions with AAVR, or possibly other receptors, might be part-mediators of host-cell specificity that can differ among AAV serotypes. A detailed understanding of AAV-receptor interactions may, in turn, open a path towards engineering different tropism or higher efficiency.<br/> <br/>Immune neutralization can be another limitation of <em>in vivo</em> viral vector delivery (Wang <em>et al.</em>, 2011; Fitzpatrick <em>et al.</em>, 2018). Strategies for developing immune-escape variants are currently being explored (Li <em>et al.</em>, 2012; McCraw <em>et al.</em>, 2012; Tseng and Agbandje-McKenna, 2014; Buning <em>et al.</em>, 2015; Majowicz <em>et al.</em>, 2017; Paulk <em>et al.</em>, 2018). Therefore, a mapping of key AAV-receptor interactions is needed to understand the limits of capsid modification if viable transduction is to be retained (Meyer <em>et al.</em>, 2019).<br/> <br/>When structural studies of AAV were first attempted in the 1990s, methods were not available to produce the milligram levels of AAV needed for crystallization. Structural studies became possible with the optimization of methods to produce wild-type (<em>i.e.</em>, infectious) AAV-2 by transfection of cultured human cells with virus-encoding bacterial plasmid (Samulski <em>et al.</em>, 1982; Laughlin <em>et al.</em>, 1983) in the presence of the adenovirus helper needed for AAV replication (Xie <em>et al.</em>, 2004). The elucidation of the type-species structure, AAV-2 (Xie <em>et al.</em>, 2002), paved the way for studies of immunologically distinct serotypes. Structural studies of several serotypes (<em>e.g.</em>, AAV-3, AAV-4, AAV-5, AAV-6) used sample prepared in analogous ways (Kaludov <em>et al.</em>, 2003; Walters <em>et al.</em>, 2004; Xie <em>et al.</em>, 2008; Lerch <em>et al.</em>, 2010). More recently, biophysical studies have taken advantage of other preparative methods.<br/> <br/>At first, it was not practical to use recombinant AAV (rAAV) vectors produced in human cells by double or triple transfection (Grimm <em>et al.</em>, 1998; Matsushita <em>et al.</em>, 1998), because the quantities, while sufficient for transduction experiments, were insufficient for biophysics. The higher-yield production of rAAV-2 vector in insect cells using baculoviral vectors (Urabe <em>et al.</em>, 2002) inspired a different approach, that was used for the crystal structures of AAV8 and AAV9 (Nam <em>et al.</em>, 2007, DiMattia <em>et al.</em>, 2012). The same bacmids encoding capsid proteins are used, but bacmids encoding the AAV ITR vector genome and the replication protein are omitted. This generates empty capsid protein shells, or virus-like-particles (VLPs), instead of vectors (Lane <em>et al.</em>, 2005). The same approach has been used for <em>cryo</em>-electron microscopy (EM), as exemplified by the engineered vector, AAV-DJ which has been solved at near atomic resolution (Lerch <em>et al.</em>, 2012, Xie <em>et al.</em>, 2017).<br/> <br/>This protocol provides details for purifying milligram quantities of type species AAV-2 VLPs based on the AAV-DJ strategy summarized earlier (Lerch <em>et al.</em>, 2012). We combine AAV8-like insect cell expression of VLPs with high-capacity purification steps used for the biophysics of wild-type viruses (Xie <em>et al.</em>, 2004; Lerch <em>et al.</em>, 2012) that can be more stringent than in rAAV productions designed for <em>in vivo</em> use. Repeated CsCl density gradient ultracentrifugation and desalting via dialysis is employed in contrast to the more common use of iodixanol for rAAV vectors (Hermens <em>et al.</em>, 1999) and VLPs (Drouin <em>et al.</em>, 2016). Furthermore, we take advantage of the heparin binding of AAV2-like viruses with an affinity purification step in lieu of ion exchange chromatography (Drouin <em>et al.</em>, 2016).<br/> <br/>The shift from crystallography to <em>cryo</em>-EM means that AAV structure is now possible with the smaller quantities available by rAAV triple transfection production methods (Zhang <em>et al.</em>, 2019). (In our hands, preparation of a single EM grid uses ~1 Î¼g, but for the optimization of conditions one would like &gt; 25 Î¼g.) The AAV-2 structure has now been determined by crystallography for infectious wild-type (wt) particles (Xie <em>et al.</em>, 2002), by <em>cryo</em>-EM for VLPs (Drouin <em>et al.</em>, 2016) and <em>cryo</em>-EM for rAAV vectors (Zhang <em>et al.</em>, 2019). For most of the capsid protein, the structures superimpose within the experimental error at 2.8 to 3.8 Ã resolution but some differences have been noted near the surface spikes (Drouin <em>et al.</em>, 2016) where crystal packing contacts could influence structure. The three sources of material (wt, rAAV &amp; VLP) can all support studies of atomic structure, but the highest resolution structures are coming from VLPs. There could be other explanations, but one possible reason is the purity achievable with the higher yield VLP protocol presented here.<br/> <br/>The capsids of natural AAV are comprised mostly of viral protein (VP) 3, but about 10% of VP3s in the 60-subunit capsid are replaced by variant VP1, and another ~10% by VP2 (Berns, 1996). The three proteins are mostly the same, but VP1 and 2 have N-terminal extensions derived from alternate start codons. VLP constructs are designed to express variants in the right proportions (Urabe <em>et al.</em>, 2002). However, it is only the 60-fold symmetric parts of AAV that have been resolved in structures. So, even though the properties of wt, rAAV and VLP appear similar, one cannot verify that the structures of the N-terminal extensions are configured exactly the same. In spite of this <em>caveat</em>, VLPs are considered to be a good proxy for wild-type and rAAV vectors, and are often chosen, because of the ease of producing large quantities of pure sample without biosafety level II containment.<br/> <br/>The receptor protein, AAVR, is an integral membrane glycoprotein, that consists of the following domains, starting from the extracellular N-terminus: a signal peptide (SP), a MANEC domain (motif at the N-terminus with eight cysteines), five immunoglobulin-like polycystic kidney disease (PKD) domains (Ibraghimov-Beskrovnaya <em>et al.</em>, 2000), a transmembrane region and a small cytoplasmic domain. A soluble fusion construct consisting of maltose binding protein (MBP), N-terminally fused to PKD1-5 (Pillay <em>et al.</em>, 2016), and lacking SP, MANEC, TM and the C-terminal tail was pivotal in implicating the PKD domains in AAV-binding via inhibition of rAAV cell transduction. This soluble construct, herein denoted MBP-PKD1-5, was used to measure competitive inhibition of AAV cell-transduction by solubilized receptor ectodomain, to measure AAV-binding strength using ELISA and surface plasmon resonance, and also for low resolution <em>cryo</em>-Electron Tomographic imaging of an AAV-2/AAVR complex (Pillay <em>et al.</em>, 2016, Meyer <em>et al.</em>, 2019). The expression and purification of the MBP-PKD1-5 AAVR construct is described in this protocol.<br/> <br/>Over-expression of âmini-AAVRâ on the cell surface supports AAV transduction in an AAVR knock-out (KO) (Pillay <em>et al.</em>, 2016). The mini-AAVR construct consists of the SP domain, omits the MANEC domain, and contains PKD domains 1-3 linked directly to the TM domain and the cytoplasmic tail. Additional AAVR domain deletion mutants, expressed in an AAVR<sup>KO</sup>, indicated that PKD domains 4 and 5 had little negative impact upon rAAV-2 transduction, and that deletion of PKD3-4 had much less of an effect than PKD1-2 (Pillay <em>et al.</em>, 2016). These observations prompted the expression of a number of shorter constructs (Pillay <em>et al.</em>, 2017), the first of which was an N-terminally histidine tagged PKD1-2, denoted His<sub>6</sub>PKD1-2. The expression and purification protocol for His<sub>6</sub>PKD1-2, which was used for the high resolution <em>cryo</em>-EM structure of the AAVR/AAV-2 complex (Meyer <em>et al.</em>, 2019), is also presented here.<br/> <br/>A number of single-PKD-domain constructs, created for binding and inhibition studies using analogous methods of expression and purification (Pillay <em>et al.</em>, 2017), are not further detailed here. Also not covered is another PKD1-5 construct used for structural studies (Zhang <em>et al.</em>, 2019). It is similar to our MBP-PKD1-5 (Pillay <em>et al.</em>, 2016), except that the MBP fusion domain is replaced by an N-terminal histidine tag (Zhang <em>et al.</em>, 2019) similar to the His<sub>6</sub>PKD1-2 construct detailed here (Pillay <em>et al.</em>, 2017). All three types of construct can bind AAV, but higher saturation of symmetry-equivalent sites on the virus is achieved with the smaller constructs, likely because there is less steric occlusion (Meyer <em>et al.</em>, 2019).</p>
        </body>

<p class="pptt">## Materials and Reagents</p><ol class="npwone"><li>AAV-2 VLP expression in Sf9 cells <ol class="npwtwo"><li>Disposable standard transfer pipets, 3 ml (Fisher Scientific, catalog number: 13-680-50)</li><li>Disposable fine-tipped transfer pipets, 3 ml (Fisher Scientific, catalog number: 02-708-018)</li><li>Disposable serological pipets, 1, 10, 25, 50 ml (Thermo Fisher, catalog numbers: 13-678-11B, -11E, -11, -11F)</li><li>Disposable polycarbonate Erlenmeyer flasks, 250 ml (Fisher Scientific, catalog number: 07-200-668)</li><li>Sterile Petri dishes (Thermo Fisher, catalog number: FB0875712)</li><li>500 ml centrifuge bottles (Nalgene, catalog number: 05-564-3)</li><li>6-well tissue culture plate (Sarstedt, catalog number: 83.1839)</li><li>Corning<sup>TM</sup> cell culture flasks, 25 cm<sup>2</sup> surface area (Sigma-Aldrich, catalog number: CLS430639)</li><li>Corning<sup>TM</sup> cell culture flasks, 75 cm<sup>2</sup> surface area (Sigma-Aldrich, catalog number: CLS430641)</li><li>pFBDVPm11 donor plasmid (donation of Rob Kotin) used to generate bacmid shuttle vector (Urabe, Ding <em>et al.</em> 2002)</li><li>Chemically competent DH10Bac cells (Thermo Fisher, catalog number: 10361-012)</li><li>Bluo-Gal (Gold Biotechnology, catalog number: B-673-1)</li><li>IPTG (Gold Biotechnology, catalog number: I2481C25)</li><li>Tetracycline (Gold Biotechnology, catalog number: T-101-25)</li><li>Gentamicin (Gold Biotechnology, catalog number: G-400-5)</li><li>Luria Broth (LB) Base (Thermo Fisher, catalog number: 12795027)</li><li>Agar powder (Thermo Fisher, catalog number: BP1423-500)</li><li>M13F and M13R primers (CCCAGTCACGACGTTGTAAAACG and AGCGGATAACAATTTCACACAGG, respectively)</li><li>Qiagen Large-Construct kit (Qiagen, catalog number: 12462)</li><li>Isopropanol (Thermo Fisher, catalog number: A520-4)</li><li>Gibco<sup>TM</sup> Sf9 cells in Sf-900<sup>TM</sup> II SFM (Thermo Fisher, catalog number: 11496015, store at -80 Â°C)</li><li>Graceâs insect medium, supplemented (Thermo Fisher, catalog number: 11605094, store at 4 Â°C)</li><li>Graceâs insect medium, unsupplemented (Thermo Fisher, catalog number: 11595030, store at 4 Â°C)</li><li>Cellfectin<sup>TM</sup> II reagent (Thermo Fisher, catalog number: 10362100, store at 4 Â°C)</li><li>HyClone<sup>TM</sup> Fetal Bovine Serum (Fisher Scientific, catalog number: SH3007102, store at -20 Â°C)</li><li>Sf-900<sup>TM</sup> II SFM cell medium (Thermo Fisher, catalog number: 10902187, store at 4 Â°C)</li><li>Trypan blue, 0.4% (Gibco, catalog number: 15250-0601)</li><li>BaculoQUANT All-in-One Virus extraction &amp; titration Kit (Genway Biotech, Inc., catalog number GWB-100602)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>AAV-2 VLP purification<ol class="npwtwo"><li>Ultracentrifuge tubes, 13 x 51 mm (Beckman Coulter, catalog number: 344057)</li><li>Pyrex storage bottles, 250, 500, 1,000 ml (Fisher Scientific, catalog numbers: 06-414-1B, -1C, -1D)</li><li>Microcentrifuge tubes, 1.5 ml (Thermo Fisher, catalog number: 02-681-331)</li><li>0.45 Î¼m syringe filter units (Millipore Sigma, catalog number: SLHV033RS)</li><li>Dialysis tubing (50kDa MWCO), <em>e.g.</em>, Repligen Spectra/Por<sup>Â®</sup> 6 Standard Regenerated Cellulose (RC) Dialysis Tubing, Pre-Wetted, catalog number: 888-11389)</li><li>Slide-A-Lyzer dialysis cassette, 10 kDa MWCO (Thermo Fisher, catalog number: 66380)</li><li>HiTrap Heparin HP column, 1 ml (GE, catalog number: 17-0406-01)</li><li>Cesium chloride (Sigma-Aldrich, catalog number: C4036)</li><li>HEPES (Sigma-Aldrich, catalog number: H3375)</li><li>Sodium chloride (NaCl) (Sigma-Aldrich, catalog number: S9888)</li><li>Magnesium chloride hexahydrate (MgCl<sub>2</sub>Â·6H<sub>2</sub>O) (Millipore Sigma, catalog number: 1374248)</li><li>Ethanol (Thermo Fisher, catalog number: S25307B)</li><li>Benzonase (Millipore Sigma, catalog number: E1014-25kU)</li><li>Bleach (Staples, catalog number: KIK8635042)</li><li>CsCl gradient solution (see Recipes)</li><li>Heparin column running buffer (Buffer A) (see Recipes)</li><li>Heparin column elution buffer (Buffer B) (see Recipes)</li><li>HM buffer (see Recipes)</li><li>HN buffer (see Recipes)</li><li>Phosphate buffered saline (PBS) (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>AAVR construct expression in <em>E. coli</em> <ol class="npwtwo"><li>Cell culture tubes (Thermo Fisher, catalog number: 14-956-6D)</li><li>BL21(DE3) <em>E. coli</em> (NEB, catalog number: C2527I)</li><li>pET11a vector (Novagen, catalog number: 69436-3)</li><li>pMAL<sup>TM</sup> Protein Fusion and Purification System (New England Biolabs, catalog number: E8200S), composed of pMAL-c5X, pMAL-p5X, amylose resin, factor Xa, anti-MBP monoclonal antibody, MBP5 protein, MBP5-paramyosin âSal and <em>E. coli</em> ER2523</li><li>NEB Express competent <em>E. coli</em> (NEB, catalog number: C2523I)</li><li>IPTG (Gold Biotechnology, catalog number: I2481C25)</li><li>Ampicillin (Gold Biotechnology, catalog number: A-301-5)</li><li>Kanamycin (Gold Biotechnology, catalog number: K-120-5)</li><li>Tris Base ((CH<sub>2</sub>OH)<sub>3</sub>CNH<sub>2</sub>) (Fisher Scientific, catalog number: BP152-1)</li><li>LB-ampicillin medium (see Recipes)</li><li>LB-ampicillin agar plates (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>AAVR construct purification<ol class="npwtwo"><li>MBPTrap HP column, 5 ml (GE, catalog number: 28-9187-79)</li><li>HiLoad 16/600 Superdex 200 pg SE column (GE Healthcare, catalog number: 28989335)</li><li>HiLoad 16/600 Superdex 75 pg SE column (GE Healthcare, catalog number: 28989333)</li><li>Amicon Ultra-0.5 Centrifugal Filter Units, 3 kDa MWCO (Millipore Sigma, catalog number: UFC500324)</li><li>Amicon Ultra-0.5 Centrifugal Filter Units, 30 kDa MWCO (Millipore Sigma, catalog number: UFC503024)</li><li>10 ml Luer lock syringes (<em>e.g.</em>, Fisher Scientific, catalog number: 14-955-459)</li><li>0.45 Î¼m syringe filter units (Millipore Sigma, catalog number: SLHV033RS)</li><li>HALT Protease Inhibitor Cocktail (100x) (Thermo Fisher, catalog number: 78429)</li><li>HiTrap Chelating HP (IMAC) column (GE, catalog number: 17-0409-01) charged with Co<sup>2+</sup>)</li><li>Gel filtration standard (Bio-Rad, catalog number: 151-1901)</li><li>Imidazole (Acros, catalog number: 12202-5000)</li><li>Sodium phosphate dibasic (Na<sub>2</sub>HPO<sub>4</sub>) (Thermo Fisher, catalog number: S374-500)</li><li>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) (Thermo Fisher, catalog number: P285-500)</li><li>MBPTrap column running buffer (Buffer A) (see Recipes)</li><li>MBPTrap column elution buffer (Buffer B) (see Recipes)</li><li>IMAC column running buffer (Buffer A) (see Recipes)</li><li>IMAC column elution buffer (Buffer B) (see Recipes)</li></ol></li></ol><p class="pptt">### Equipment</p><ol class="npwtwok"><li>Water bath capable of 42 Â°C (<em>e.g.</em>, Thermo Fisher, model: Isotemp)</li><li>Shaking incubator, 27 Â°C (<em>e.g.</em>, Thermo Fisher, model: MaxQ 6000)</li><li>Shaking incubator, 37 Â°C (<em>e.g.</em>, New Brunswick Scientific, model: Excella E25)</li><li>qPCR machine (<em>e.g.</em>, Applied Biosystems, model: StepOne)</li><li>French press for cell lysis (<em>e.g.</em>, SLM Aminco, model: FA-078)</li><li>Table-top microcentrifuge capable 16,000 <em>x g</em> (<em>e.g.</em>, Thermo Fisher, model: accuSpin Micro)</li><li>FPLC (<em>e.g.</em>, GE Healthcare, model: <a class="blackNoUnder" target="_blank"> AKTA chromatography system </a> )</li><li>FPLC system sample loops: 2 ml (GE Healthcare, catalog number: 18111402); 10 ml (superloop) (18-1113-81) (or 50 ml, 18-1113-82)</li><li>SDS-PAGE equipment (<em>e.g.</em>, Bio-Rad, model: Mini Protean 3 Cell)</li><li>Thermocycler (<em>e.g.</em>, MJ Research, model: PTC-200)</li><li>Preparative centrifuge (<em>e.g.</em>, Sorvall, model: RC5C)</li><li>Centrifuge rotor, fixed angle, fitting 50 ml tubes (<em>e.g.</em>, Beckman Coulter JA-20 or FiberLite F13-14x50cy)</li><li>Centrifuge rotor, fixed angle, fitting 500 ml bottles (<em>e.g.</em>, Beckman Coulter J-LITE<sup>Â®</sup> JLA-10.500)</li><li>Ultracentrifuge (<em>e.g.</em>, Beckman model: L7-55)</li><li>Swinging-bucket rotor for CsCl, capable of 135,000 <em>x g</em> (<em>e.g.</em>, Beckman SW50.1)</li><li>Retort stand and clamp used for holding ultracentrifuge tubes</li><li>pH meter (<em>e.g.</em>,Thermo Fisher, model: Acumet AB15)</li><li>Pipets (<em>e.g.</em>, Gilson Pipetman<sup>Â®</sup>, models: P1000, P200, P20)</li><li>Stir plate (<em>e.g.</em>, Fisher Scientific, catalog number: SP88854200)</li><li>Magnetic stir bars (Nalgene, catalog number: 93-5521-5)</li><li>Balance accurate to 0.01 g (<em>e.g.</em>, Ohaus, model: Scout Pro SP402)</li><li>UV-Spectrophotometer (<em>e.g.</em>, Shimadzu, model: UV-1800)</li><li>Custom-made dialysis buttons (parts purchased from McMaster-Carr, catalog number: 8546K15, and machined at the University of Missouri, Columbia; Figure 1 and Table 1 for assembly and measurements). An alternative might be the Slide-A-Lyzer MINI dialysis device, 20K MWCO (Thermo Fisher, catalog number: 69590 or 69555), see Procedure D below: âDesalting by dialysisâ</li><li>Refractometer (<em>e.g.</em>, Abbe, model: C10)</li><li>Hemocytometer (<em>e.g.</em>, Thermo Fisher, catalog number: 02-671-6)</li><li>Inverted microscope (for cell counting) (<em>e.g.</em>, Nikon, model: Eclipse Ts2)<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118223811_1330.jpg" width="198"/><br/><strong>Figure 1. Custom dialysis button.</strong> Parts were machined from PTFE (McMaster-Carr) with threaded caps and chamber. Milled separately, the inner chamber is cemented into the outer chamber. The bore has a capacity of 50 Î¼l (left) or 60 Î¼l (right). Between the bottom screwcap and the cylinder is sandwiched a dialysis membrane, and at the top end is a microscope cover slip, providing a window. After assembling the bottom part, sample is loaded from the top, before the cover slip is added and the top sealed. The button is hung membrane-side down from the side of a beaker in exchange buffer. Air that is trapped in the pocket under the bottom cap must be removed to ensure contact between membrane and buffer.<br/><br/><strong>Table 1. Dimensions of dialysis buttons</strong><br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><strong>Dialysis button part </strong><br/></td><td><strong>Outer diam (mm)</strong><br/></td><td><strong>Inner hole diam (mm)</strong><br/></td><td><strong>Height (mm)</strong><br/></td></tr><tr><td>Top cap<br/></td><td>19<br/></td><td>3<br/></td><td>8.5<br/></td></tr><tr><td>Outer chamber<br/></td><td>19<br/></td><td>13<br/></td><td>18<br/></td></tr><tr><td>Inner chamber<br/></td><td>13<br/></td><td>3<br/></td><td>6.5<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Bottom cap<br/></td><td>19<br/></td><td>7.5<br/></td><td>9.5<br/></td></tr></tbody></table></li></ol><p class="pptt">### Procedure</p><ol class="npwone"><li>Baculoviral expression of AAV-2 VLPs in Sf9 cells<br/>This consists of five steps: (1) starting and maintaining a culture of Sf9 cells, (2) construction and purification of the bacmid DNA containing the AAV-2 capsid gene, (3) transfection of Sf9 cells with the prepared bacmid DNA, (4) collection and amplification of baculovirus stock, and (5) baculoviral infection of Sf9 cells for production of VLPs.<ol class="npwtwo"><li>Growth of Sf9 cells<ol class="npwthree"><li>Warm frozen Sf9 stock at 37 Â°C until nearly thawed.</li><li>Aseptically transfer the cells into a 250 ml Erlenmeyer flask containing 25 ml of pre-warmed (25-30 Â°C) Sf-900<sup>TM</sup> II SFM.</li><li>Incubate at 28 Â°C while shaking at 135 rpm, protected from light (as suggested by the manufacturer).<br/><em>Note: A culture from frozen stock can take a couple of weeks to grow to a suitable density.</em> </li><li>Count cells daily when culture shows signs of growth. To count, mix 1 ml of culture with 100 Âµl of trypan blue and mix gently by flicking the tube. Wait 2 min, then remix and count with a hemocytometer. (It can also usually be assumed that cells will have propagated sufficiently, and can be counted at 48 h, and split immediately, as described in the next paragraph.)</li><li>When cells reach a density of 4 x 10<sup>6</sup>-6 x 10<sup>6</sup> cells/ml, passage to 0.5 x 10<sup>6</sup>-1.0 x 10<sup>6</sup> cells/ml in a final volume of 50 ml. Healthy cultures double in ~24 h and should be passaged every 2-4 days. Continue passaging as needed to maintain an appropriate cell density. Cultures may show decreasing viability after passage 30 and a new culture may need to be started.<br/><em>Note: Starting a culture with a density &lt; 0.5 x 10</em><sup><em>6</em></sup><em> cells/ml or allowing a culture to exceed 8 x 10</em><sup><em>6</em></sup><em> cells/ml is highly discouraged as it leads to longer growth lags and decreased viability.</em> </li></ol></li><li>Generation and purification of bacmid DNA<ol class="npwthree"><li>From the pFBDVPm11 donor plasmid, generate bacmid shuttle vector in DH10Bac cells following the Bac-to-Bac<sup>TM</sup> protocol (Thermo Fisher, publ. MAN0000414).</li><li>Purify bacmid using Qiagenâs endotoxin-free large construct kit.</li></ol></li><li>Transfection of bacmid DNA into Sf9 cells according to the Bac-to-Bac<sup>TM</sup> protocol (see above).</li><li>Isolation of P1 baculovirus stock according to the Bac-to-Bac<sup>TM</sup> protocol. Modifications to the Bac-to-Bac<sup>TM</sup> procedure include:<ol class="npwthree"><li>Quantifying the virus titer by qPCR with the BaculoQUANT All-In-One Extraction &amp; Titration Kit to estimate readiness for virus harvest.</li><li>Optionally, instead of harvesting when the Sf9 cells are 20-30% viable, harvest when the baculovirus titer is at least 10<sup>6</sup> viral genomes (vg)/ml.<br/><em>Note: This tends to take about 4-5 days post transfection.</em> </li></ol></li><li>Amplification of baculovirus stock (generation of P2 stock)<br/>This continues to follow the Bac-to-Bac<sup>TM</sup>, modified as follows:<br/>Quantifying the P2 virus titer by qPCR.<br/><em>Note: This tends to take about 3-4 days post infection.</em> </li><li>Sf9 infection for VLP production<ol class="npwthree"><li>Scale up a Sf9 culture to 600-800 ml and a density of 2 x 10<sup>6</sup>-4 x 10<sup>6</sup> cells/ml, as described in Step A1, above.</li><li>Infect the culture with the P2 baculovirus stock. Add P2 to a MOI of between 5 and 10.</li><li>Incubate the culture as before and check cell density and viability daily. When viability starts to decrease (at 40-60 h), check culture more frequently and harvest when the viability is between 80-60% (typically between 55-80 h).</li><li>Transfer the culture into two 500 ml centrifuge bottles, balance and centrifuge at 5,500 <em>x g</em> for 30 min at 20 Â°C.</li><li>Resuspend each pellet in 10 ml of HM buffer and pool into a 50 ml centrifuge tube. Collect any additional cells with a single aliquot of 5 ml of additional HM buffer.</li><li>Store the ~25 ml of resuspended cell pellet at -80 Â°C until ready for VLP purification.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>AAV-2 VLP purification<br/>AAV-2 VLP purification consists of four steps: (1) lysis of the cell pellet by freeze-thaw cycles, (2) clarification of the VLP product from the remaining soluble components, (3) affinity column purification and (4) desalting by dialysis. First is an initial lysis that extracts VLPs from the cells. The second step relies upon ultracentrifugation of the sample in a cesium chloride gradient to separate the VLP from the other soluble components using differential density centrifugation. Following three rounds of CsCl gradient centrifugation, the sample is 50-70% pure. For greater purity, the sample is run through a heparin affinity column followed by one additional CsCl centrifugation to concentrate the sample. [For serotypes that do not have high affinity for heparin, one can substitute ion exchange for affinity chromatography (Kaludov <em>et al.</em>, 2002), or Cs-gradient ultracentrifugation can be performed with higher stringency, further repeating, or accepting only a narrower fraction near the peak of the band (Xie <em>et al.</em>, 2004)]. The yield for a VLP preparation is typically 2-4 mg/ml in 100 Âµl. Lastly, since the sample contains a high concentration of CsCl which would interfere with electron microscopy visualization, the sample is dialyzed using a custom dialysis device (dialysis button) designed for 40-60 Âµl volumes. Due to the relative stability of the VLP sample in 3.3 M CsCl and the instability in low-salt solution, the last dialysis step is performed on small volumes of purified sample that will be used within 1-2 days. (AAV-2 is more prone to aggregation at low salt than other serotypes, many of which are stable at 4 Â°C for up to 4 weeks.)<ol class="npwtwo"><li>Lysis of the Sf9 cells and collection of soluble proteins<ol class="npwthree"><li>Thaw the VLP-containing cell suspension in 37 Â°C water bath for 30 min.</li><li>Refreeze the cell suspension at -80 Â°C for at least 2 h.</li><li>Repeat Steps B1a and B1b 4 more times (5 cycles of thaw and freeze).</li><li>Thaw the cell suspension in 37 Â°C water bath for 20 min, add 0.25 Âµl benzonase per ml of cell suspension, mix and continue incubating in 37 Â°C water bath for 20 min.<br/><em>Note: Digestion of the nucleic acids reduces the viscosity.</em> </li><li>Add 5 M sodium chloride solution to increase sodium chloride in cell suspension by 100 mM (<em>e.g.</em>, add 0.6 ml to a 30 ml suspension) and freeze at -80 Â°C for at least 2 h.<br/><em>Note: Addition of sodium chloride deactivates benzonase.</em> </li><li>Thaw the suspension a final time in a 37 Â°C bath and dispense 2-ml aliquots into fifteen 2-ml tubes and spin at 16,000 <em>x g</em> for 10 min.</li><li>Pool the supernatant from all tubes and filter through a 0.45 Âµm filter.</li></ol></li><li>CsCl gradient purification of AAV-2 VLP<ol class="npwthree"><li>Add CsCl to the pooled supernatant (above) until the refractive index (RI) is 1.3690 (Â± 0.0005).<br/><em>Note: This will be approximately 0.5 g per ml of sample and brings it to ~3.3 M CsCl.</em> </li><li>Make an extra 50 ml of HM buffer plus CsCl, to the same RI, for balancing (Recipe 11).</li><li>Transfer ~5 ml of sample into each of 6 ultracentrifuge tubes and balance each pair to within 0.05 g, using the solution made above (Step C2b) to top off tubes as needed. Fill each tube to within 1 mm of the top to avoid collapse during centrifugation.</li><li>Centrifuge the samples at 135,000 <em>x g</em> with the SW50.1 rotor (or similar) in an ultracentrifuge at 20 Â°C for 20-24 h.</li><li>Gently transport the tubes to your working area to preserve gradient separation. Wipe the outside of each tube with ethanol and visualize the AAV band using a bright halogen lamp (see Figure 2). As a guide, mark the side of the tube about 2 mm above and below the AAV-2 band (<em>e.g.</em>, with a Sharpie pen).<br/><br/><img alt="" class="layerphoto" height="133" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224512_3802.jpg" width="100"/><br/><strong>Figure 2. Side-illumination of the centrifuge tube after separation of AAV-2 by CsCl gradient centrifugation.</strong> The cloudy white band in the middle is AAV-2 VLP.<br/><br/></li><li>Use a standard disposable transfer pipette to remove the liquid down to the mark 2 mm above the AAV-2 band and discard this into a 10% bleach solution.</li><li>Use a fine-tipped disposable transfer pipette to gently collect the sample containing AAV-2 (take 2 mm above to 2 mm below the band), and transfer it to a new ultracentrifuge tube. Collect the AAV-2 band from all six tubes similarly and pool into a single ultracentrifuge tube. Check and record the refractive index (RI).</li><li>Balance and centrifuge the tube containing the pooled AAV-2 samples as before (Steps C2c-C2e, above) (second spin).</li><li>Repeat the AAV-2 band collection and ultracentrifugation step one more time (third spin).</li></ol></li><li>Collect the sample after the third ultracentrifugation and inject into a Slide-A-Lyzer dialysis cassette with the 1 ml syringe and 18 gauge needle provided, as per product instructions.</li><li>Float the dialysis cassette in 2 L of phosphate buffered saline (PBS) and stir for 2 h at 4 Â°C. Replace with 2 L of fresh PBS and allow to dialyze overnight at 4 Â°C.</li><li>Transfer the dialyzed sample from the Slide-A-Lyzer cassette to a low protein-binding microcentrifuge tube; store at 4 Â°C until loading onto heparin affinity column.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Heparin affinity column purification of AAV-2 VLP<ol class="npwtwo"><li>Prepare AKTA system and heparin affinity column<ol class="npwthree"><li>Verify the 2 ml sample loop is connected to the FPLC system.</li><li>Purge the FPLC system lines A and B with ultra-pure deionized water.</li><li>Purge the FPLC system lines with heparin affinity buffers: line A with the running buffer A and line B with the elution buffer B (Recipe 6).</li><li>With buffer A flowing at 0.5 ml/min connect the Heparin HiTrap column.</li><li>Equilibrate the column with 20 ml of 100% buffer A at 1 ml/min.</li><li>Wash the column with 10 ml of 100% buffer B at 1 ml/min.</li><li>Wash the column with at least 10 ml of 100% buffer A.</li><li>Once UV and conductivity stabilize, end the flow of buffer A.</li></ol></li><li>Run sample on heparin affinity column<ol class="npwthree"><li>Spin the dialyzed sample at 16,000 <em>x g</em> for 10 min to remove aggregates.</li><li>Set the system to âLoadâ and flush the loop with 10 ml of PBS using a Luer lock syringe.</li><li>Set the system to âInjectâ and remove the syringe.</li><li>Load a 1 ml syringe with your VLP sample and connect it to the loop.</li><li>Set the system back to âLoadâ and inject your sample into the loop, being careful not to inject air.</li><li>Set the flow to 1 ml/min and introduce your sample to the column by setting the system to âInjectâ and flushing the loop with 3 ml of 100% buffer A.</li><li>Set back to âLoadâ and continue to run 100% buffer A at 1 ml/min for 5 min.</li><li>Run a linear gradient from 0-85% buffer B over 25 min, collecting 0.5 ml fractions.</li><li>Examine the UV chromatogram and set aside the eluate fractions containing the AAV-2 peak for evaluation by SDS-PAGE.</li><li>Run a sample from each AAV-2 peak-containing fraction on an SDS-PAGE gel, following standard protocols, to verify the presence of VLP.</li><li>Pool the AAV-2-containing fractions and run a final CsCl gradient ultracentrifugation as before (see Steps C2c-C2e above). This gradient serves to concentrate the VLPs.</li><li>Store the collected sample in a low protein-binding microcentrifuge tube.</li><li>Clean the column by running 10 column volumes of elution buffer B, ultra-pure DI water, and 20% ethanol, sequentially. Store the column in 20% ethanol.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Desalting by dialysis<br/>Our protocol continues to use a custom mini-dialysis apparatus that we designed for low volume and high recovery before somewhat similar devices became available commercially. It is likely that the Slide-A-Lyzer MINI dialysis device (see Materials) could be substituted, following the manufacturerâs directions, rejoining the protocol at Step D3i below, âMeasure the absorbanceâ¦â.We continue the protocol based on the custom device:<ol class="npwtwo"><li>Prepare the dialysis membrane, according to the manufacturerâs instructions, then:<br/>Using a 12 mm circular glass cover side as a template, cut small circles out of dialysis membrane; store at 4 Â°C in HM buffer plus 0.02% sodium azide until use.</li><li>Prepare the dialysis button<br/>Assemble the custom dialysis button according to Figure 1, ensuring a smooth fit of the membrane between the bottom inner chamber and the plastic washer inside the bottom cap.</li><li>If fully assembled, remove the top cap, cover slip and rubber washer<ol class="npwthree"><li>Aliquot a sample (30-60 Âµl) of the purified VLP (from Procedure D) into the inner hole of the inner chamber, taking care to avoid introducing air bubbles onto the surface of the dialysis membrane.</li><li>Place the glass coverslip, rubber washer, and top cap in the outer chamber and tighten carefully to seal.</li><li>Use a twist-tie to suspend the dialysis button partially in a 50 ml beaker with the glass coverslip facing up.</li><li>Add HN buffer so that the lower portion of the dialysis button is submerged while the top is above the surface.</li><li>Use a Pasteur pipette that has been melted (to form a 180Â°-hairpin) to remove air bubbles present on the bottom surface of the dialysis membrane in the bottom cap of the dialysis button.<br/><em>Note: The presence of air bubbles will occlude the liquid portion from allowing dialysis of the salts from the sample and may harm protein structure.</em> </li><li>Dialyze overnight with stirring at 4 Â°C.</li><li>Remove the dialysis button from the beaker and gently dry the area near the dialysis membrane by wicking away with strips of filter paper.</li><li>Use a 10 Âµl tip to carefully puncture the dialysis membrane and pipette the sample into a sterile, low protein-binding microcentrifuge tube.</li><li>Measure the absorbance of the sample at 280 nm and determined the protein concentration using an extinction coefficient of 17.3 (g/100 ml)<sup>-1</sup>/cm, otherwise known as <img alt="" class="layerphoto" height="25" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200119/20200119005501_4244.jpg" style="margin-top:2px;" width="57"/>, or E1%).</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Expression of MBP-PKD1-5 and His<sub>6</sub>PKD1-2 AAVR constructs in <em>E. coli</em> <ol class="npwtwo"><li>Subclone desired PKD1-5 fragments from the KIAA0319L cDNA clone (clone ID 3843301; GE Dharmacon; Pillay <em>et al.</em>, 2016). PKD1-5 is subcloned into the pMAL-c5X vector as a fusion with MBP whereas PKD1-2 is subcloned into the pET-11a vector with an N-terminal His-tag (Figure 3).</li><li>Transform pMAL-c5x plasmids (1 pg) into NEB Express cells and pET-11a plasmids (1 pg) into BL21(DE3) cells using standard techniques and plate on LB-ampicillin agar plates (100 Âµg/ml).<br/><em>Note: Both constructs can be expressed using the procedure delineated below (Steps E3-E5). Transformation efficiencies did not vary significantly from the advertised values of 8 x 10<sup>8</sup> CFU/Âµg for NEB Express and 3 x 10<sup>7</sup> CFU/Âµg for BL21(DE3).</em> </li><li>Inoculate a single transformed BL21(DE3) colony from a freshly streaked LB-ampicillin agar plate into a volume of LB-ampicillin (100 Âµg/ml) media that is 5% of the final total desired culture volume (<em>e.g.</em>, 50 ml overnight culture into 1,000 ml final volume). Grow the starter culture overnight at 37 Â°C with shaking at 210 rpm.</li><li>The next day, transfer the starter culture into the final media volume and incubate at 37 Â°C with shaking at 210 rpm until the OD<sub>600</sub> is ~1.0. Remove a 1 ml aliquot (pre-induction sample) and place into a 1.5 ml tube. Spin this aliquot for 1 min at 6,000 <em>x g</em> and discard the supernatant into a 10% bleach solution. Store the bacterial pellet at -20 Â°C for post-expression analysis by SDS-PAGE, if desired.</li><li>Add IPTG to a concentration of 1 mM and incubate the culture at 37 Â°C with shaking at 210 rpm for an additional 4-5 h. After incubation, remove a 1 ml aliquot (post-induction sample) and pellet these cells as described in Step E4 for the pre-induction sample. Store this pellet also at -20 Â°C for post-expression analysis. Centrifuge the remaining culture at 6,000 <em>x g</em>, discard the supernatant into a 10% bleach solution, and store the pellets at -20 Â°C until lysis and purification steps.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224609_3569.jpg" width="142"/><br/><strong>Figure 3. Subcloning of MBP-PKD1-5 and His<sub>6</sub>PKD1-2.</strong> A. Use the KIAA0319L cDNA clone as a template for PCR amplification using primers 1 and 2. Cut the PCR product with SbfI then phosphorylate using T4 Polynucleotide Kinase. Cut pMAL-c5x with XmnI and SbfI. Ligate the cut/phosphorylated PCR product and the cut pMAL-c5x vector together to make MBP-PKD1-5. B. Use the same DNA as a template for PCR using primers 3 and 4. Cut the PCR fragment with NheI and BamHI. Cut pET-11a with NheI and BamHI. Ligate the cut PCR product and the cut pET-11a vector together to make His<sub>6</sub>PKD1-2.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Purification of AAVR constructs<ol class="npwtwo"><li>MBP-PKD1-5 purification<ol class="npwthree"><li>Cell lysis (do on ice)<ol class="npwfour"><li>Resuspend frozen pellets in 25 ml of MBPTrap running buffer containing 100 Âµl HALT Protease Inhibitor Cocktail 100x.</li><li>Load cell slurry into a French press cell and pass through two complete rounds of lysis at 1,000 psi using a slow flow rate to collect the lysate into a 50 ml centrifuge tube.<br/></li><li>Centrifuge lysate at 4 Â°C, 16,000 <em>x g</em>, for 45 min in a F13-14x50cy rotor (or similar). While spinning, prepare the AKTA system.</li></ol></li><li>MBP affinity chromatography on the AKTA system<ol class="npwfour"><li>Washing and equilibrating the MBPTrap column<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Prior to attaching the column, place line A of the AKTA into MBPTrap running buffer and line B into MBPTrap elution buffer.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Inject 10 ml of MBPTrap running buffer into the superloop (valve setting = Load).</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Start flow at 5 ml/min and 50%B, bypassing superloop (valve setting = Load). Run for about 5 min.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-1px;"><div class="float-left">4)</div><div class="media-body">Switch to buffer A (0%B), and run for 5 more min.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-1px;"><div class="float-left">5)</div><div class="media-body">Reduce flow to 2 ml/min and flow through superloop (valve setting = Inject) for 10 min or until the superloop is purged completely.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-1px;"><div class="float-left">6)</div><div class="media-body">Reduce flow to 1 ml/min, switch valve back to âLoadâ, and attach the MBPTrap HP column, avoiding introduction of air.</div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-1px;"><div class="float-left">7)</div><div class="media-body">Equilibrate the column with 5 column volumes of elution buffer B at 1 ml/min and until UV and conductivity baselines stabilize.</div></div><div class="media media-x cnautokh cnautokh8" style="margin-top:-1px;"><div class="float-left">8)</div><div class="media-body">Switch to running buffer A and wash until the baseline stabilizes, at least 10 column volumes.</div></div></li><li>Loading sample onto MBPTrap column<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Remove the tube containing cell lysate from the centrifuge and transfer the supernatant to a new 50 ml centrifuge tube.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Filter through a 0.45 Âµm filter using a 10 ml Luer lock syringe. It may take multiple filters if membranes become fouled.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">With the valve setting set to âLoadâ, inject 10 ml of lysate into the superloop.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-1px;"><div class="float-left">4)</div><div class="media-body">Set flow rate to 0.5 ml/min, fraction size to 10 ml, and the valve to âInjectâ.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-1px;"><div class="float-left">5)</div><div class="media-body">Flow through the superloop into the column until the superloop is almost empty.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-1px;"><div class="float-left">6)</div><div class="media-body">At this point, pause the FPLC, switch the valve to âLoadâ, inject another 10 ml into superloop, switch back to âInjectâ, and resume flow. Repeat this process until entire lysate volume is loaded onto the column (with a high capacity superloop, the entire sample could be loaded in a single step).</div></div></li><li>Elution of MBP-PKD1-5<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">After loading, wash the column by switching the valve to âLoadâ and increasing the flow of running buffer to 1 ml/min.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Once baseline is stabilizedâat least 10 column volumesâreduce the flow to 0.5 ml/min and the fraction size to 2 ml.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Flow 100% elution buffer B for 45 min to elute MBP-PKD-1-5. <br/></div></div></li><li>Processing MBPTrap column fractions<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Run peak fractions (Figure 4) on SDS-PAGE to verify the presence of MBP-PKD-1-5 and pool the appropriate fractions.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Concentrate the pooled sample to ~1 ml using a 30 kDa Amicon Ultra centrifugal filter unit. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224644_6701.jpg" width="143"/><br/><strong>Figure 4. Elution of MBP-PKD1-5 from MBPTrap HP column.</strong> The flow was switched from 100% A to 100% elution buffer B at elution volume 0. <br/><br/></div></div></li></ol></li><li>Superdex 200 SEC on the AKTA system<br/><em>Note: MBP-PKD1-5 oligomerizes slowly (over a ~30 day period at 4 Â°C) (Meyer </em><em>et al.</em><em>, 2019). This SEC step allows dimers to be selected over higher order oligomers shortly before use, and must be repeated, if this separation is desired, after the sample has been stored at 4 Â°C for an extended time.</em> <ol class="npwfour"><li>Washing and equilibrating the Superdex 200<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Prior to attaching the column, place line A of the AKTA into HN buffer and flush line A by running 0%B at 5 ml/min for 5 min.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Inject 10 ml of HN buffer into superloop (ensure valve setting = Load).</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Set flow to 2 ml/min and flow through superloop (valve setting = Inject) for 10 min or until the superloop is purged completely.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-1px;"><div class="float-left">4)</div><div class="media-body">Reduce the flow rate to 1 ml/min and attach a pre-calibrated Superdex 200 size exclusion column, avoiding introduction of air. To equilibrate, flow HN buffer through the column at 1-1.5 ml/min until the baseline stabilizes.</div></div></li><li>Loading sample onto the Superdex 200<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Pause the AKTA and inject into the superloop a sample volume that is no more than 1% of the column volume (valve setting = Load).</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Set flow rate to 1 ml/min and resume flow (valve setting = Inject).</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Collect the fractions (Figure 6) that correspond to the molecular weight of the MBP-PKD1-5 dimer (190 kDa) based on your gel filtration d calibration results and Meyer <em>et al.</em>, 2019. Figure 5 provides an example of what a standard calibration may look like. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224714_5489.jpg" width="135"/><br/><strong>Figure 5. Calibration of Superdex 200 using the BioRad Gel Filtration Standards.</strong> The blue line tracks elution of standards (left to right): bovine thyroglobulin (670 kDa), bovine Î³-globulin (158 kDa), chicken ovalbumin (44 kDa), horse myoglobin (17 kDa) and vitamin B12 (1.35 kDa). Chromatography of the standards should be performed under the same conditions (flow rates, <em>etc.</em>) as will be used for sample (see Step F1c and Figure 7). <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224743_9418.jpg" width="144"/><br/><strong>Figure 6. MBP-PKD1-5 elution from Superdex 200 SEC.</strong> Comparison of the elution times with Figure 5 indicates that the larger left peak corresponds to higher order oligomers of &gt; 380 kDa. The smaller right peak, corresponding to dimers of MBP-PKD1-5 (190 kDa), is collected. <br/><br/></div></div></li></ol></li></ol></li><li>Purification of His<sub>6</sub>PKD1-2<ol class="npwthree"><li>Cell lysis (do on ice)<ol class="npwfour"><li>Resuspend frozen pellets in 25 ml of IMAC running buffer containing 100 Âµl HALT Protease Inhibitor Cocktail 100x.</li><li>Load the cell slurry into a French press cell and pass through 2 complete rounds of lysis at 1,000 psi using a flow rate slow enough to avoid foaming, collecting the lysate into a 50 ml centrifuge tube.</li><li>Centrifuge lysate at 4 Â°C, 16,000 <em>x g</em>, for 45 min in a F13-14x50cy rotor (or similar). While spinning, prepare the AKTA system.</li></ol></li><li>IMAC affinity chromatography on the AKTA system<ol class="npwfour"><li>Washing and equilibrating the IMAC column<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Place line A of the AKTA in the IMAC running buffer and line B in the IMAC elution buffer. The column should not be attached to the AKTA at this point unless the valve is set in the bypass mode.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Inject 10 ml of IMAC running buffer into the superloop (ensure valve setting = Load).</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Start the flow at 5 ml/min and 50%B, bypassing the superloop (valve setting = Load). Run for about 5 min.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-1px;"><div class="float-left">4)</div><div class="media-body">Switch the buffer to A (<em>i.e.</em>, 0%B), and run for an additional 5 min.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-1px;"><div class="float-left">5)</div><div class="media-body">Reduce the flow to 2 ml/min and flow through the superloop (valve setting = Inject) for 10 min or until the superloop is purged completely.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-1px;"><div class="float-left">6)</div><div class="media-body">Reduce the flow to 1 ml/min, switch valve back to âLoadâ, and attach the HiTrap HP (IMAC) column, avoiding introduction of air.</div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-1px;"><div class="float-left">7)</div><div class="media-body">Equilibrate the column by flowing elution buffer for 5 column volumes at 1 ml/min and until UV and conductivity baselines stabilize.</div></div><div class="media media-x cnautokh cnautokh8" style="margin-top:-1px;"><div class="float-left">8)</div><div class="media-body">Switch to running buffer and flow until the baseline stabilizes, at least 10 column volumes.</div></div></li><li>Loading sample onto the IMAC column<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Remove the tube containing the cell lysate from the centrifuge and transfer the supernatant to a new 50 ml centrifuge tube.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Filter through 0.45 Âµm filter. It may take multiple filters if membranes become fouled.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">With the FPLC valve setting set to load, inject 10 ml of lysate into the superloop.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-1px;"><div class="float-left">4)</div><div class="media-body">Set the flow rate to 0.5 ml/min, fraction size to 10 ml, and valve to inject.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-1px;"><div class="float-left">5)</div><div class="media-body">Flow through the superloop into the column until the superloop is almost empty.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-1px;"><div class="float-left">6)</div><div class="media-body">At this point, pause the FPLC, switch the valve to load, inject another 10 ml into the superloop, switch the valve to inject, and continue running the FPLC. Repeat this process until the entire lysate is loaded onto column.</div></div></li><li>Elution of His<sub>6</sub>PKD1-2<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Wash the column by switching the injection valve to âLoadâ and increasing the flow to 1 ml/min.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Once the baseline stabilizesâat least 10 column volumesâreduce the flow rate to 0.5 ml/min and the fraction size to 2 ml.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Run a linear gradient from 0%-100% elution buffer over 45 min.</div></div></li><li>Processing IMAC column fractions<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">Run the peak fractions (Figure 7) on SDS-PAGE and pool the appropriate fractions.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Concentrate pooled sample to ~1 ml using a 3 kDa MWCO Amicon Ultra centrifugal filter unit. <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224819_8482.jpg" width="148"/><br/><strong>Figure 7. Elution of His<sub>6</sub>PKD1-2 from the IMAC column.</strong> The blue line shows the elution of the His-tagged protein peaking in the first 3 ml over a large background. Background, coming from absorption of the imidazole in buffer B (orange line), is approximately removed in the gray line through subtraction of a value proportional to %B. <br/><br/></div></div></li></ol></li><li>Superdex 75 SEC on the AKTA system<br/><em>Note: As with MBP-PKD1-5, His<sub>6</sub>PKD-1-2 oligomerizes slowly at 4 Â°C (Meyer </em><em>et al.</em><em>, 2019). This SEC step allows dimers to be selected over higher order oligomers shortly before use, and must be repeated, if this separation is desired, after the sample has been stored at 4 Â°C for an extended time.</em> <ol class="npwfour"><li>Washing and equilibrating the Superdex 75<br/>Wash and equilibrate the Superdex 75 column with HN buffer exactly as described for the Superdex 200 (Step F1c, above).</li><li>Loading sample onto the Superdex 75<div class="media media-x cnautokh cnautokh1"><div class="float-left">1)</div><div class="media-body">After equilibration, pause the AKTA and, as with the Superdex 200, inject a sample volume that is no more than 1% of the column volume (set valve to âLoadâ).</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-1px;"><div class="float-left">2)</div><div class="media-body">Set flow rate to 1 ml/min and resume flow (valve setting = Inject).</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-1px;"><div class="float-left">3)</div><div class="media-body">Collect the fractions that correspond to the molecular weight of the His<sub>6</sub>PKD1-2 dimer (44 kDa) based on your gel filtration standard calibration results and Meyer <em>et al.</em>, 2019.</div></div><em>Note: Loading and elution procedures, including flow rates, for the Superdex 75 SEC are the same as for the Superdex 200 SEC (Step F1c), but samples have different retention times. The expected retention time of the His6PKD-1-2 dimer can be estimated </em><em>by gel filtration standard calibration results (see Figure 8 for an example) and Meyer </em><em>et al.</em><em>, 2019.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224857_5632.jpg" width="127"/><br/><strong>Figure 8. Calibration of Superdex 75 using the BioRad Gel Filtration Standards.</strong> The blue line tracks elution of standards (left to right): (a) the first major peak has a shoulder indicative that it is unresolved bovine thyroglobulin (670 kDa) and bovine Î³-globulin (158 kDa); (b) chicken ovalbumin (44 kDa), (c) horse myoglobin (17 kDa); and (d) vitamin B12 (1.35 kDa). Chromatography of the standards should be performed under the same conditions (flow rates, etc.) as will be used for sample (see Step F2ci and Figure 9). Monomers of His<sub>6</sub>PKD1-2 (22 kDa) would be expected at about 70 ml and dimers at about 60 ml, the same elution time as ovalbumin.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200118/20200118224924_5429.jpg" width="140"/><br/><strong>Figure 9. His<sub>6</sub>PKD1-2 elution from Superdex 75 SEC. </strong>Comparison of the elution times with Figure 8 indicates that the larger left composite peak corresponds to higher order oligomers exceeding &gt; 75 kDa and approximately 670 kDa. The largest peak, corresponding to dimers of His<sub>6</sub>PKD1-2, is used, and there is only the faintest hint of something approximating a monomer eluting later.</li></ol></li></ol></li></ol></li></ol><p class="pptt">### Recipes</p><ol class="npwtwok"><li>LB-ampicillin medium<br/>20 g/L LB Broth Base<br/>Autoclave at 121 Â°C, 2.6 bar for 30 min to sterilize<br/>Cool to ~60 Â°C in a water bath, then add ampicillin to 100 Âµg/ml</li><li>LB-ampicillin agar plates<br/>20 g/L LB Broth Base<br/>15 g/L agar powder<br/>Autoclave at 121 Â°C, 2.6 bar for 30 min to sterilize<br/>Cool to ~60 Â°C in a water bath, then add ampicillin to 100 Î¼g/ml and pour into sterile Petri dishes<br/>Can be stored at 2-8 Â°C for 2-3 weeks</li><li>CsCl gradient solution (for balancing of ultracentrifuge tubes)<br/>HM buffer (see Recipe 11)<br/>Approx. 0.5 g/ml CsCl<br/><em>Note: Add CsCl to 50 ml HM buffer until the RI is 1.3690 (Â± 0.0005); gives a solution of ~3.3 M CsCl.</em> </li><li>Heparin affinity column running buffer (Buffer A)<br/>Phosphate buffered saline (PBS) (see Recipe 12)<br/>Filter sterilize</li><li>Heparin affinity column elution buffer (Buffer B)<br/>PBS (see Recipe 12)<br/>1 M NaCl<br/>Filter sterilize</li><li>MBPTrap column running buffer (Buffer A)<br/>20 mM Tris<br/>200 mM NaCl<br/>pH to 7.4</li><li>MBPTrap column elution buffer (Buffer B)<br/>20 mM Tris<br/>200 mM NaCl<br/>10 mM maltose<br/>pH to 7.4</li><li>IMAC column running buffer (Buffer A)<br/>20 mM Na<sub>2</sub>HPO<sub>4</sub><br/>500 mM NaCl<br/>40 mM imidazole<br/>pH to 7.4</li><li>IMAC column elution buffer (Buffer B)<br/>20mM Na<sub>2</sub>HPO<sub>4</sub><br/>500 mM NaCl<br/>500 mM imidazole<br/>pH to 7.4</li><li>HM buffer<br/>25 mM HEPES<br/>150 mM NaCl<br/>50 mM MgCl<sub>2</sub><br/>pH to 7.4</li><li>HN buffer<br/>25 mM HEPES<br/>150 mM NaCl<br/>pH to 7.4</li><li>Phosphate buffered saline (PBS)<br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub> (dibasic)<br/>2 mM KH<sub>2</sub>PO<sub>4</sub> (monobasic)<br/>pH to 7.4<br/></li></ol><p class="pptt">### Acknowledgments</p><p class="ppzz">The research was funded the National Institutes of Health: R01-GM066875 (MSC) and R35-GM122564 (MSC). The protocols for the AAVR constructs were summarized in Meyer <em>et al.</em> (2019) which adapts protocols for AAV VLP preparation in Lerch <em>et al.</em> (2012) and for AAV purification in Xie <em>et al.</em> (2004), all listed below.</p><p class="pptt">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt">### References</p><ol class="npwtwok"><li>Berns, K. I. (1996). <a href="https://www.scholars.northwestern.edu/en/publications/paramyxoviridae-the-viruses-and-their-replication-3" target="_blank"><em>Parvoviridae:</em> The viruses and their replication.</a> <em>Virology.</em> Fields, B. N. D., Knipe, M. and Howley, P. M.. Philadelphia, Raven: 1017-1041.</li><li>Buning, H., Huber, A., Zhang, L., Meumann, N. and Hacker, U. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26302254" target="_blank">Engineering the AAV capsid to optimize vector-host-interactions.</a> <em>Curr Opin Pharmacol</em> 24: 94-104.</li><li>Chapman, M. S. and M. Agbandje-McKenna (2006). <a href="https://www.researchgate.net/publication/308007316_Atomic_structure_of_viral_particles" target="_blank">Atomic structure of viral particles.</a> <em>Parvoviruses</em>. Kerr, J. R., Cotmore, S. F., Bloom, M. E., Linden, R. M. and Parrish, C. R. London, Hodder Arnold, Ltd. 107-123.</li><li>DiMattia, M. A., Nam, H. J., Van Vliet, K., Mitchell, M., Bennett, A., Gurda, B. L., McKenna, R., Olson, N. H., Sinkovits, R. S., Potter, M., Byrne, B. J., Aslanidi, G., Zolotukhin, S., Muzyczka, N., Baker, T. S. and Agbandje-McKenna, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22496238" target="_blank">Structural insight into the unique properties of adeno-associated virus serotype 9.</a> <em>J Virol</em> 86(12): 6947-6958.</li><li>Drouin, L. M., Lins, B., Janssen, M., Bennett, A., Chipman, P., McKenna, R., Chen, W., Muzyczka, N., Cardone, G., Baker, T. S. and Agbandje-McKenna, M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27440903" target="_blank"><em>Cryo</em>-electron microscopy reconstruction and stability studies of the wild type and the R432A variant of adeno-associated virus Type 2 reveal that capsid structural stability is a major factor in genome packaging.</a> <em>J Virol</em> 90(19): 8542-8551. </li><li>Dudek, A. M., Pillay, S., Puschnik, A. S., Nagamine, C. M., Cheng, F., Qiu, J., Carette, J. E. and Vandenberghe, L. H. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29343568" target="_blank">An Alternate Route for Adeno-associated Virus (AAV) Entry Independent of AAV Receptor.</a><em> J Virol</em> 92(7). </li><li>Fitzpatrick, Z., Leborgne, C., Barbon, E., Masat, E., Ronzitti, G., van Wittenberghe, L., Vignaud, A., Collaud, F., Charles, S., Simon Sola, M., Jouen, F., Boyer, O. and Mingozzi, F. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29766022" target="_blank">Influence of pre-existing anti-capsid neutralizing and binding antibodies on AAV vector transduction.</a> <em>Mol Ther Methods Clin Dev</em> 9: 119-129.</li><li>Gaudet, D., Methot, J., Dery, S., Brisson, D., Essiembre, C., Tremblay, G., Tremblay, K., de Wal, J., Twisk, J., van den Bulk, N., Sier-Ferreira, V. and van Deventer, S. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22717743" target="_blank">Efficacy and long-term safety of alipogene tiparvovec (AAV1-LPLS447X) gene therapy for lipoprotein lipase deficiency: an open-label trial.</a> <em>Gene Ther</em> 20(4): 361-369.</li><li>Grimm, D., Kern, A., Rittner, K. and Kleinschmidt, J. A. (1998). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9874273" target="_blank">Novel tools for production and purification of recombinant adenoassociated virus vectors.</a> <em>Hum Gene Ther</em> 9(18): 2745-2760. </li><li>Hermens, W. T., ter Brake, O., Dijkhuizen, P. A., Sonnemans, M. A., Grimm, D., Kleinschmidt, J. A. and Verhaagen, J. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10446928" target="_blank">Purification of recombinant adeno-associated virus by iodixanol gradient ultracentrifugation allows rapid and reproducible preparation of vector stocks for gene transfer in the nervous system.</a> <em>Hum Gene Ther</em> 10(11): 1885-1891. </li><li>Ibraghimov-Beskrovnaya, O., Bukanov, N. O., Donohue, L. C., Dackowski, W. R., Klinger K. W. and Landes, G. M. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10861291" target="_blank">Strong homophilic interactions of the Ig-like domains of polycystin-1, the protein product of an autosomal dominant polycystic kidney disease gene, PKD1.</a> <em>Hum Mol Genet</em> 9: 1641-1649.</li><li>Kaludov, N., Handelman, B. and Chiorini, J. A. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12133276" target="_blank">Scalable purification of adeno-associated virus type 2, 4, or 5 using ion-exchange chromatography.</a> <em>Hum Gene Ther</em> 13(10): 1235-1243.</li><li>Kaludov, N., Padron, E., Govindasamy, L., McKenna, R., Chiorini, J. A. and Agbandje-McKenna, M. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12620791" target="_blank">Production, purification and preliminary X-ray crystallographic studies of adeno-associated virus serotype 4.</a> <em>Virology</em> 306(1): 1-6. </li><li>Lane, M. D., Nam, H. J., Padron, E., Gurda-Whitaker, B., Kohlbrenner, E., Aslanidi, G., Byrne, B., McKenna, R., Muzyczka, N., Zolotukhin, S. and Agbandje-McKenna, M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16511095" target="_blank">Production, purification, crystallization and preliminary X-ray analysis of adeno-associated virus serotype 8.</a> <em>Acta Crystallogr Sect F Struct Biol Cryst Commun</em> 61(Pt 6): 558-561.</li><li>Laughlin, C. A., Tratschin, J. D., Coon, H. and Carter, B. J. (1983). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6352411" target="_blank">Cloning of infectious adeno-associated virus genomes in bacterial plasmids.</a> <em>Gene</em> 23(1): 65-73.</li><li>Lerch, Thomas F., O'Donnell, Jason K., Meyer, Nancy L., Xie, Q., Taylor, Kenneth A., Stagg, Scott M. and Chapman, Michael S. (2012). <a href="http://www.sciencedirect.com/science/article/pii/S0969212612001852" target="_blank">Structure of AAV-DJ, a retargeted gene therapy vector: <em>Cryo</em>-electron microscopy at 4.5 Ã resolution.</a> <em>Structure</em> 20(8): 1310-1320.</li><li>Lerch, T. F., Xie, Q. and Chapman, M. S. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20444480" target="_blank">The structure of adeno-associated virus serotype 3B (AAV-3B): insights into receptor binding and immune evasion.</a> <em>Virology</em> 403(1): 26-36.</li><li>Li, C., Diprimio, N., Bowles, D. E., Hirsch, M. L., Monahan, P. E., Asokan, A., Rabinowitz, J., Agbandje-McKenna, M. and Samulski, R. J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22593151" target="_blank">Single amino acid modification of adeno-associated virus capsid changes transduction and humoral immune profiles.</a> <em>J Virol</em> 86(15): 7752-7759. </li><li>Majowicz, A., Salas, D., Zabaleta, N., Rodriguez-Garcia, E., Gonzalez-Aseguinolaza, G., Petry, H. and Ferreira, V. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28596114" target="_blank">Successful repeated hepatic gene delivery in Mice and Non-human primates achieved by sequential administration of AAV5ch and AAV1.</a> <em>Mol Ther</em> 25(8): 1831-1842.</li><li>Matsushita, T., Elliger, S., Elliger, C., Podsakoff, G., Villarreal, L., Kurtzman, G. J., Iwaki, Y. and Colosi, P. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9813665" target="_blank">Adeno-associated virus vectors can be efficiently produced without helper virus.</a> <em>Gene Ther</em> 5(7): 938-945.</li><li>McCraw, D. M., OâDonnell, J. K., Taylor, K. A., Stagg, S. M. and Chapman, M. S. (2012). <a href="http://www.sciencedirect.com/science/article/pii/S0042682212002346" target="_blank">Structure of adeno-associated virus-2 in complex with neutralizing monoclonal antibody A20.</a> <em>Virology</em> 431(1â2): 40-49. <br/></li><li>Meyer, N. L., Hu, G., Davulcu, O., Xie, Q., Noble, A. J., Yoshioka, C., Gingerich, D. S., Trzynka, A., David, L., Stagg, S. M. and Chapman, M. S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31115336" target="_blank">Structure of the gene therapy vector, adeno-associated virus with its cell receptor, AAVR.</a> <em>eLife</em> 8: e44707. </li><li>Nam, H. J., Lane, M. D., Padron, E., Gurda, B., McKenna, R., Kohlbrenner, E., Aslanidi, G., Byrne, B., Muzyczka, N., Zolotukhin, S. and Agbandje-McKenna, M. (2007). <a href="https://jvi.asm.org/content/81/22/12260" target="_blank">Structure of adeno-associated virus serotype 8, a gene therapy vector.</a> <em>J Virol</em> 81(22): 12260-12271.</li><li>Paulk, N. K., Pekrun, K., Zhu, E., Nygaard, S., Li, B., Xu, J., Chu, K., Leborgne, C., Dane, A. P., Haft, A., Zhang, Y., Zhang, F., Morton, C., Valentine, M. B., Davidoff, A. M., Nathwani, A. C., Mingozzi, F., Grompe, M., Alexander, I. E., Lisowski, L. and Kay, M. A. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29055620" target="_blank">Bioengineered AAV capsids with combined high human liver transduction <em>in vivo</em> and unique humoral seroreactivity.</a> <em>Mol Ther</em> 26(1): 289-303.</li><li>Pillay, S., Meyer, N. L., Puschnik, A. S., Davulcu, O., Diep, J., Ishikawa, Y., Jae, L. T., Wosen, J. E., Nagamine, C. M., Chapman, M. S. and Carette, J. E. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26814968" target="_blank">An essential receptor for adeno-associated virus infection.</a> <em>Nature</em> 530: 108-112. </li><li>Pillay, S., Zou, W., Cheng, F., Puschnik, A. S., Meyer, N. L., Ganaie, S. S., Deng, X., Wosen, J. E., Davulcu, O., Yan, Z., Engelhardt, J. F., Brown, K. E., Chapman, M. S., Qiu, J. and Carette, J. E. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28679762" target="_blank">AAV serotypes have distinctive interactions with domains of the cellular receptor AAVR.</a> <em>J Virol</em> 91(18): e00391-00317.</li><li>Russell, S., Bennett, J., Wellman, J. A., Chung, D. C., Yu, Z. F., Tillman, A., Wittes, J., Pappas, J., Elci, O., McCague, S., Cross, D., Marshall, K. A., Walshire, J., Kehoe, T. L., Reichert, H., Davis, M., Raffini, L., George, L. A., Hudson, F. P., Dingfield, L., Zhu, X., Haller, J. A., Sohn, E. H., Mahajan, V. B., Pfeifer, W., Weckmann, M., Johnson, C., Gewaily, D., Drack, A., Stone, E., Wachtel, K., Simonelli, F., Leroy, B. P., Wright, J. F., High, K. A. and Maguire, A. M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28712537" target="_blank">Efficacy and safety of voretigene neparvovec (AAV2-hRPE65v2) in patients with RPE65-mediated inherited retinal dystrophy: a randomised, controlled, open-label, phase 3 trial.</a> <em>Lancet</em> 390(10097): 849-860.</li><li>Samulski, R. J., Berns, K. I., Tan, M. and Muzyczka, N. (1982). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6281795" target="_blank">Cloning of adeno-associated virus into pBR322: rescue of intact virus from the recombinant plasmid in human cells.</a> <em>Proc. Natl. Acad. Sci. </em>79: 2077-2081. </li><li>Tseng, Y. S. and Agbandje-McKenna, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24523720" target="_blank">Mapping the AAV Capsid host antibody response toward the Development of Second Generation Gene Delivery Vectors.</a> <em>Front Immunol</em> 5: 9.</li><li>Urabe, M., Ding, C. and Kotin, R. M. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12427305" target="_blank">Insect cells as a factory to produce adeno-associated virus type 2 vectors.</a> <em>Hum Gene Ther</em> 13(16): 1935-1943.</li><li>Walters, R. W., Agbandje-McKenna, M., Bowman, V. D., Moninger, T. O., Olson, N. H., Seiler, M., Chiorini, J. A., Baker, T. S. and Zabner, J. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15016858" target="_blank">Structure of Adeno-associated virus serotype 5.</a> <em>J Virol</em> 78(7): 3361-3371.</li><li>Wang, L., Calcedo, R., Bell, P., Lin, J., Grant, R. L., Siegel, D. L. and Wilson, J. M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21476868" target="_blank">Impact of pre-existing immunity on gene transfer to nonhuman primate liver with adeno-associated virus 8 vectors.</a> <em>Hum Gene Ther</em> 22(11): 1389-1401.</li><li>Xie, Q., Bu, W., Bhatia, S., Hare, J., Somasundaram, T., Azzi, A. and Chapman, M. S. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12136130" target="_blank">The atomic structure of adeno-associated virus (AAV-2), a vector for human gene therapy.</a> <em>Proc Natl Acad Sci U S A</em> 99(16): 10405-10410.</li><li>Xie, Q., Hare, J., Turnigan, J. and Chapman, M. S. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15488616" target="_blank">Large-scale production, purification and crystallization of wild-type adeno-associated virus-2.</a> <em>J Virol Methods</em> 122(1): 17-27. </li><li>Xie, Q., Ongley, H. M., Hare, J. and Chapman, M. S. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18997346" target="_blank">Crystallization and preliminary X-ray structural studies of adeno-associated virus serotype 6.</a> <em>Acta Crystallogr Sect F Struct Biol Cryst Commun</em> 64(Pt 11): 1074-1078. </li><li>Xie, Q., Spear, J. M., Noble, A. J., Sousa, D. R., Meyer, N. L., Davulcu, O., Zhang, F., Linhardt, R. J., Stagg, S. M. and Chapman, M. S. (2017). <a href="http://dx.doi.org/10.1016/j.omtm.2017.02.004" target="_blank">The 2.8 Ã Electron Microscopy Structure of Adeno-Associated Virus-DJ Bound by a Heparinoid Pentasaccharide.</a> <em>Mol Ther Methods Clin Dev</em> 5: 1-12.</li><li>Zhang, R., Cao, L., Cui, M., Sun, Z., Hu, M., Zhang, R., Stuart, W., Zhao, X., Yang, Z., Li, X., Sun, Y., Li, S., Ding, W., Lou, Z. and Rao, Z. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30742069" target="_blank">Adeno-associated virus 2 bound to its cellular receptor AAVR.</a> <em>Nat Microbiol</em> 4(4): 675-682. <br/></li></ol>