<body>
            <h1>Identification of Target Protein for Bio-active Small Molecule Using Photo-cross Linked Beads and MALDI-TOF Mass Spectrometry</h1>
            <p><strong>Authors:</strong> Kruthi Suvarna, Kaori Honda, Makoto Muroi, Yasumitsu Kondoh, Nobumoto Watanabe, Hiroyuki Osada</p>
            <h2>Abstract</h2>
            <p>Development of methods for protein identification is one of the important aspects of proteomics. Here, we report a protocol for the preparation of compound conjugated beads by photo-crosslinking, affinity purification, gel electrophoresis, and highly sensitive mass spectrometric assay for drug-target identification. Although there are several other methods used for drug-target identification, such as biochemical fractionation or radioactive ligand binding assay, affinity purification is widely used for its straight-forward and easy approach. To identify the target protein of an inhibitor of cancer cell-accelerated fibroblast migration, we prepared the inhibitor-conjugated beads by photo-crosslinking. Proteins were pulled down from cell lysates by the compound beads and separated by SDS-PAGE, and a specifically pulled down protein was cut out, trypsin-digested, analyzed using matrix-assisted laser desorption ionization/time of flight mass spectrometry (MALDI-TOF-MS) and identified by peptide mass fingerprinting (PMF) method. Since the photo-crosslinking enables the immobilization of ligands on an affinity matrix in a functional group-independent manner, we do not have to determine the functional group of the compound to conjugate the matrix. In addition, as compared to other MS techniques such as electrospray ionization, MALDI offers a less complex sample preparation procedure and higher sensitivity, and thus is better suited for the rapid identification of proteins isolated by gel electrophoresis.</p>
            <p><strong>Keywords:</strong> Protein, MALDI-TOF, Affinity beads, Mass spectrometry (MS), NPD8733, Valosin containing protein (VCP)</p>
            <h2>Background</h2><p>The field of chemical proteomics is of great importance to drug discovery research enabling the identification and validation of a drug target and understanding the side effects as well as designing of more specific drugs. In our research, we used photo-crosslinked small molecule affinity matrix as a tool for identification of target protein of compound NPD8733, an inhibitor of cancer cell-accelerated fibroblast migration. The sepharose beads conjugated with NPD8733 and its control compound without activity were prepared by the photo-crosslinking technique (Kanoh <em>et al.</em>, 2005; Suvarna <em>et al.</em>, 2019). The proteins pulled down by the beads were separated using standard electrophoresis method. The protein pulled-down specifically by NPD8733-conjugated beads was cut out, subjected to in-gel trypsin digestion, and identified to be valosin-containing protein (VCP) using MALDI-TOF-MS and PMF analysis (Suvarna <em>et al</em>., 2019).<br/><br/>Pull-down assay using affinity beads is a commonly used method for identification of direct targets of small bioactive molecules. The preparation of affinity beads usually requires analysis of structure-activity relationship (SAR) so that the sites that do not affect the activity of the biomolecule can be identified and used for the attachment of an affinity tag such as biotin. Photo-affinity beads developed by Kanoh <em>et al.</em> overcome these drawbacks (Kanoh <em>et al.</em>, 2005). The affinity beads prepared by photo-crosslinking technique, do not require SAR studies, since the photo-crosslinking technique utilizes a photo-generated carbene species to immobilize the small molecule on an affinity bead in a chemo- and site-nonselective manner. This unbiased nature of the photo-crosslinking technique proves beneficial for identifying the binding partners of unreactive small molecules; however it cannot be used for the small molecules that degrade upon UV irradiation (Kanoh <em>et al.</em>, 2005).<br/><br/>The pulled-down proteins can be separated from the beads using electrophoretic protein separation under denaturing conditions or can also be directly digested on-beads using proteolytic enzymes. The electrophoretic method is inexpensive, easy to use, and has a wide separation range, making relative quantification easy. The separated proteins can be stained using Coomassie blue staining or silver staining methods. The resulting mixture of peptides can be analyzed using MS techniques such as MALDI-TOF or ESI-MS. When compared to ESI, MALDI-TOF is sensitive, requiring low picomole-level of gel-separated proteins. The sample preparation in MALDI is simple and thus suitable for rapid protein identification (Lubec and Afjehi-Sadat, 2007).<br/><br/>The photo-crosslinking method can also be applied for chemical arrays that can be used for ligand screening of a protein of interest. Since small molecules can be fixed in a functional group-independent manner, glass slides with thousands of compounds can be prepared without the need for SAR analysis, also making it possible to study SAR of compounds in a high-throughput manner as well (Kanoh <em>et al.</em>, 2006).</p>
        </body>

<p class="pptt" id="biaoti31311">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml Falcon tubes</li><li>Cloth</li><li>Eppendorf tubes</li><li>Micropipette loading tips for pull-down assay (Sorenson, catalog number: 13791)</li><li>15 cm Petri-dish for cell culture (Iwaki, catalog number: 3030-150)</li><li>Glass vials (Maruemu Corp., No.2, catalog number: 5-098-04), 6 ml</li><li>NHS beads (GE Healthcare, NHS-activated Sepharose 4 Fast Flow 25 ml, catalog number: 17090601)</li><li>Photoaffinity PEG linker (Kondoh <em>et al.</em>, 2015)</li><li>Test compound (any vendor). In addition to the test compound, structurally similar and inactive compound was used as the negative control. We have frequently experienced that negative compound conjugated beads act as better ânegative control beadsâ when compared to the naked beads</li><li>Spin column (GeneDesign, Inc., EconoSpin Empty Column, catalog number: EP-31301)</li><li>Mouse embryonic fibroblast cell line NIH3T3 (obtained from RIKEN Cell Bank, catalog number: RCB1862)</li><li>Human breast cancer cell line MCF7 (obtained from RIKEN Cell Bank, catalog number: RCB1904)</li><li>Dulbecco's Modified Eagle Medium (D-MEM) media (Gibco, catalog number: C11995500BT)</li><li>Fetal calf serum (FCS) (Nichirei Bioscience Inc. Tokyo, Japan) used at 10% in DMEM</li><li>Penicillin and Streptomycin (Pen Strep, Gibco, catalog number: 15140-122) used at 100x dilution in DMEM</li><li>HEPES (Sigma, catalog number: H3375)</li><li>HCl (FUJIFILM Wako Pure Chemical, catalog number: 080-01066)</li><li>NaCl (FUJIFILM Wako Pure Chemical, catalog number: 191-01665)</li><li>KCl (FUJIFILM Wako Pure Chemical, catalog number: 163-03545)</li><li>Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O (FUJIFILM Wako Pure Chemical, catalog number: 196-02835)</li><li>KH<sub>2</sub>PO<sub>4</sub> (FUJIFILM Wako Pure Chemical, catalog number: 169-04245)</li><li>Dioxane (FUJIFILM Wako Pure Chemical, catalog number: 042-03766)</li><li>MQ-water (Purified water produced by MilliQ Q-POD, Millipore)</li><li>NaN<sub>3</sub> (FUJIFILM Wako Pure Chemical, catalog number: 199-11095)</li><li>EDTA (FUJIFILM Wako Pure Chemical, catalog number: 345-01865)</li><li>Sequencing Grade Modified Trypsin (Promega, catalog number: V5111)</li><li>Acetonitrile (Nacalai tesque, catalog number: 00420-41)</li><li>TFA (FUJIFILM Wako Pure Chemical, catalog number: 201-06393)</li><li>Methanol (FUJIFILM Wako Pure Chemical, catalog number: 131-01826)<br/></li><li>Î±-Cyano-4-hydroxycinnamic acid (FUJIFILM Wako Pure Chemical, catalog number: 037-19261)</li><li>NaHCO<sub>3</sub> (FUJIFILM Wako Pure Chemical, catalog number: 191-01305)</li><li>NH<sub>4</sub>HCO<sub>3</sub> (FUJIFILM Wako Pure Chemical, catalog number: 017-02875)</li><li>Ethanolamine (FUJIFILM Wako Pure Chemical, catalog number: 012-12455)</li><li>Tris (FUJIFILM Wako Pure Chemical, catalog number: 201-06273)</li><li>DTT (FUJIFILM Wako Pure Chemical, catalog number: 042-29222)</li><li>Iodoacetamide (FUJIFILM Wako Pure Chemical, catalog number: 093-02892)</li><li>Coomassie Brilliant blue R250 (CBB) (FUJIFILM Wako Pure Chemical, catalog number: 031-17922)</li><li>Acetic acid (FUJIFILM Wako Pure Chemical, catalog number: 017-00256)</li><li>Isopropanol (FUJIFILM Wako Pure Chemical, catalog number: 325-00045)</li><li>Sodium Dodecyl Sulfate (SDS) (FUJIFILM Wako Pure Chemical, catalog number: 192-08672)</li><li>Bromophenol blue (FUJIFILM Wako Pure Chemical, catalog number: 021-02911)</li><li>Glycerol (FUJIFILM Wako Pure Chemical, catalog number: 075-00611)</li><li>2-Mercaptoethanol (FUJIFILM Wako Pure Chemical, catalog number: 28625.01)</li><li>Ammonium Persulfate (APS) (FUJIFILM Wako Pure Chemical, catalog number: 802811)</li><li>N,N,Nâ,Nâ-tetramethylethylenediamine (TEMED) (FUJIFILM Wako Pure Chemical, catalog number: 202-04003)</li><li>Razor (FEATHER, catalog number: 99077)</li><li>cOmplete EDTA-free Protease inhibitor cocktail tablet (Roche diagnostic, Mannheim, Germany, catalog number: 11836170001)</li><li>Pierce BCA protein assay kit (Thermo Fisher Scientific, catalog number: 23225)</li><li>Wash/binding buffer containing protease inhibitors (see Recipes)</li><li>1x SDS buffer (see Recipes)</li><li>Binding/wash buffer (see Recipes)</li><li>aq. HCl (see Recipes)</li><li>PBS (see Recipes)</li><li>Coupling buffer (see Recipes)</li><li>Blocking solution (see Recipes)</li><li>Decolorant (see Recipes)</li><li>Reduction solution (see Recipes)</li><li>Alkylating solution (see Recipes)</li><li>Digestion solution (see Recipes)</li><li>Extraction solution 1 (see Recipes)</li><li>Extraction solution 2 (see Recipes)</li><li>Matrix solution (see Recipes)</li><li>Staining solution (see Recipes)</li><li>Destaining solution (see Recipes)</li><li>10x Tris-glycine running buffer (see Recipes)</li><li>5% Stacking gel (see Recipes)</li><li>10% Resolving gel recipe (see Recipes)<br/></li></ol><p class="pptt" id="biaoti31312">### Equipment</p><ol class="npwtwok"><li>UV crosslinker (UVP, Ultraviolet Crosslinker, CL-1000)</li><li>LCMS (LC: Waters, ACUITY UPLC H-class, MS: SCIEX, API 3200 LC-MS/MS)</li><li>Agilent Oven (Agilent Technologies, Hybridization Oven, G2545A)</li><li>High-performance rotary evaporator (Scrum, Genevac, EZ-2 plus)</li><li>Rotator (TAITEC, model: RT-50)</li><li>Desk-top centrifuge machine (Corning, model: 6765)</li><li>Standard refrigerated centrifuge for 15/50 ml and Eppendorf tubes (Tomy, model: MX-301)</li><li>Savant SpeedVac Concentrator (ThermoFisher Scientific, model: SVC100H)</li><li>Heat block (EYELA, model: MG-1200)</li><li>Ultrasonic bath (AS ONE, US CLEANER US-1R)</li><li>Sonicator (Tomy, model: UR-20P)</li><li>Slab type SDS-electrophoresis equipment (NIHON-EIDO, model: NA-1010)</li><li>MALDI-TOF mass spectrometer (Ultraflex, Bruker Daltonik GmbH)<br/></li></ol><p class="pptt" id="biaoti31313">### Software</p><ol class="npwtwok"><li>Biotools (Biotools version 3.2, Bruker Daltonik GmbH, <a href="https://www.bruker.com/" target="_blank">https://www.bruker.com/</a>)</li><li>Mascot search program (Mascot Server, Matrix Science Inc, <a href="http://www.matrixscience.com/" target="_blank">http://www.matrixscience.com</a>)<br/></li></ol><p class="pptt" id="biaoti31314">### Procedure</p><ol class="npwone"><li>UV stability check of compound<ol class="npwtwo"><li>Put small amount (ca. 0.1-0.3 mg) of compound in glass vial.</li><li>Dissolve compound with 20 Î¼l of methanol (or soluble solvent with relatively low boiling point) and divide it into two vials</li><li>Concentrate and dry in vacuo with a high-performance rotary evaporator (Genevac) at 35 Â°C by âMed + Low BP mixtureâ solvent mode for 1 h.</li><li>Irradiate one of the two vials with UV (365 nm; 4 J/cm<sup>2</sup>) with UV crosslinker.</li><li>Dissolve both UV(+) and UV(-) compounds with methanol and analyze with LCMS.</li><li>Compare LCMS chart of UV(+) and UV(-) compounds and check the stability of the compound against UV irradiation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Compound beads preparation<ol class="npwtwo"><li>Wash 400 Î¼l of NHS beads with 400 Î¼l of 1 mM aq. HCl 3 times and subsequently with 400 Î¼l of coupling buffer 3 times at room temperature. Briefly, add the washing solution, mix by pipetting using a pipet tip to suspend the beads totally, centrifuge in a desk-top centrifuge machine at 2,000 <em>x g</em> for one minute, remove and discard the supernatant.</li><li>Dissolve 8.9 mg (18 Î¼mol) of photoaffinity PEG linker in 400 Î¼l of coupling buffer and add to the beads.</li><li>Incubate the beads suspension under rotation (10 rpm) in Agilent Oven at 37 Â°C for 2 h.</li><li>Wash beads with 400 Î¼l of coupling buffer 3 times as with Step B1.</li><li>Add 1 ml of blocking solution.</li><li>Incubate beads suspension under rotation in Agilent Oven at 37 Â°C for 1 h.</li><li>Wash beads with 400 Î¼l of methanol 3 times and subsequently with 400 Î¼l of MQ-water 3 times in a spin column. Briefly, add washing solution to the beads on the spin column, place the column in an Eppendorf tube, and centrifuge in a desk-top centrifuge machine at 2,000 <em>x g</em> for 1 min, following which the filtrate collected into the Eppendorf tube.</li><li>Transfer the beads to a glass vial and add a methanol-based solution containing the compound (6 Î¼mol/400 Î¼l of methanol).</li><li>Concentrate and dry in vacuo with high-performance rotary evaporator (Genevac) for 1 to 1.5 h (depends on solvent).</li><li>Irradiate the beads with UV using UV crosslinker 4 J/cm<sup>2</sup>.</li><li>Wash the free compounds with 400 Î¼l of methanol (or soluble solvent) 15 times and subsequently with 400 Î¼l of MQ-water 3 times in a spin column as in Step B7.</li><li>Suspend the compound beads in 400 Î¼l of PBS-1 mM NaN<sub>3</sub> and store at 4 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Pull-down assay<ol class="npwtwo"><li>Preparation of cell lysates<ol class="npwthree"><li>Co-culture NIH3T3 cells and MCF7 cells (5:1; 2 x 10<sup>6</sup> and 0.4 x 10<sup>6</sup>/15 cm dish, respectively) in D-MEM with 10% FCS containing penicillin and streptomycin for 48 h and harvest, wash with PBS, resuspend in wash/binding buffer containing protease inhibitors. (We usually use approximately 2 x 10<sup>8</sup> cells from 20 of 15 cm dishes and resuspend the cells in 10 ml of wash/binding buffer in 50 ml Falcon tubes. Sonicate the sample for 3 min (30 s on and 30 s off). Place the Falcon tubes on ice during sonication.)</li><li>Discard the insoluble material by centrifugation at 4,400 <em>x g</em> for 20 min at 4 Â°C and transfer the supernatant into fresh falcon tubes. Typically, the concentration of the lysate used was about 16 mg/ml and we used lysates containing about 8 mg of protein for pull-down with 30 Âµl compound beads. The protein estimation was done using Pierce BCA protein assay kit according to their instruction.</li></ol></li><li>Aliquot 30 Âµl beads (conjugated with test compounds, negative compounds and naked) using wide bore-tip into fresh Eppendorf tubes.</li><li>Wash the beads by adding 300 Âµl wash/binding buffer, mix gently and spin down at maximum speed for 30 s. Discard the supernatant carefully. Repeat this step twice.</li><li>Add lysate containing 8 mg of protein in 500 Âµl of wash/binding buffer to each of the Eppendorf tube containing the beads. (Perform all the steps at 4 Â°C)</li><li>Incubate the mixture containing beads and cell lysate at 4 Â°C under gentle rotating condition using a rotating mixer for at least 3 h.</li><li>Spin down and discard the supernatant carefully. Wash the beads by adding 500 Âµl wash/binding buffer, mix gently and spin down. Discard the supernatant carefully. Repeat this step at least 3 times.</li><li>After the last wash, carefully discard all the supernatant. Try not to leave any supernatant and also be careful not to displace the beads. Use loading tips for removing the supernatant. You may move the beads into fresh Eppendorf tubes if required.</li><li>Add 30 Âµl 1x SDS buffer to the beads, keep the Eppendorf tubes at 100 Â°C for 5 min.</li><li>Allow the sample to cool down and then load the supernatant onto a 10% SDS-PAGE gel using SDS-electrophoresis equipment (NIHON-EIDO NA-1010) and keep it for overnight staining using CBB stain. The gel was then washed in destaining solution for at least two hours. An example of stained gel is shown in Figure 1. The analyzed region is shown.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200115/20200115220900_7862.jpg" width="350"/><br/><strong>Figure 1. NIH3T3 and MCF7 cells were co-cultured at a 5:1 ratio and their cell lysates were incubated with control beads, NPD8126 beads, or NPD8733 beads for 3 h.</strong> The reacted beads were washed, and the eluted proteins were separated using SDS-PAGE and stained by CBB. The protein specifically co-precipitated with NPD8733 beads was identified by MALDI-TOF-MS to be VCP (arrow). The red rectangular box indicates the approximate region of the gel portion cut out for protein identification.<br/><br/></li></ol></li><li>Identification of binding proteins by peptide mass fingerprinting using MALDI-TOF mass spectrometry<ol class="npwtwo"><li>Cut the region of interest [usually 5 mm x 1 mm x 1 mm (thickness)] using a razor and transfer half of it (2.5 mm x 1 mm x 1 mm) into a clean Eppendorf tube. The other half is stored for backup.</li><li>Wash the gel by adding 400 Âµl of water, incubate for 10 min and discard the supernatant. Repeat this step 5 times.</li><li>Incubate with 100 Âµl of decolorant for 10 min at 37 Â°C.</li><li>Wash the gel by adding 200 Âµl of acetonitrile, incubate for 10 min and discard the supernatant. Repeat this step twice.</li><li>Dry in vacuo using SpeedVac concentrator at a machine-fixed rotation rate for about 45 min at room temperature (without heating). As dehydration proceeds, the gel usually becomes opaque.</li><li>Add 50 Âµl of reduction solution to the dried gel and incubate at 56 Â°C for 30 min on a heat block.</li><li>Add 100 Âµl of alkylating solution to the gel and incubate at 37 Â°C for 30 min covered with a cloth to protect from light.</li><li>Wash the gel with 200 Âµl of 100 mM NH<sub>4</sub>HCO<sub>3</sub> solution and discard the supernatant.</li><li>Wash with 200 Âµl of acetonitrile twice for 10 min.</li><li>Dry in vacuo using SpeedVac concentrator.</li><li>Add 3 Âµl of digestion solution to the dried gel on ice for 30 min.</li><li>Add 10 Âµl of 10 mM Tris-HCl (pH 8.8), and incubate at 37 Â°C overnight.</li><li>Remove the supernatant to a new microtube. Add 10 Âµl of extraction solution 1 to the remaining gel bits and incubate for 10 min in an ultrasonic bath with ice-cold water at low power setting.</li><li>Remove the supernatant and add it to the supernatant containing microtube from Step D13.</li><li>Add 10 Âµl of extraction solution 2 to the remaining gel bits and incubate for 10 min in an ultrasonic bath with ice-cold water.</li><li>Remove the supernatant into the supernatant containing microtube from Step D13.</li><li>Repeat the Steps D15 and D16.</li><li>Add 10 Âµl of acetonitrile to the remaining gel bits and incubate for 10 min in an ultrasonic bath with ice-cold water at low power setting.</li><li>Remove the supernatant into the supernatant containing microtube from Step D13.</li><li>Dry in vacuo using SpeedVac concentrator.</li><li>Dissolve with 3 Âµl of 0.1% TFA including 5% acetonitrile.</li><li>Mix 0.25 Âµl of the sample and equal volume of matrix solution on the target plate of a MALDI-TOF mass spectrometer.</li><li>Analyze by operating a MALDI-TOF mass spectrometer in the positive ion reflector mode.</li></ol></li></ol><p class="pptt" id="biaoti31315">### Data analysis</p><p class="ppzz">Perform protein identification by peptide mass finger printing using the Mascot search program (standard setting for trypsinized sample) based on the peak list generated by the software (Biotools) provided by the manufacturer (Bruker Daltonik GmbH).</p><p class="pptt" id="biaoti31316">### Recipes</p><ol class="npwtwok"><li>1x SDS buffer<br/>0.0005% Bromophenol blue<br/>10% Glycerol<br/>2% SDS<br/>63 mM Tris HCl pH 6.8<br/>0.1% 2-Mercaptoethanol (Add before use)</li><li>Binding/wash buffer containing protease inhibitors<br/>50 mM HEPES<br/>150 mM NaCl<br/>1 mM EDTA<br/>Adjust pH 7.5 and volume up with MQ water<br/>Store at 4 Â°C<br/>Add protease inhibitor (complete EDTA free-protease cocktail tablets) immediately before use</li><li>aq. HCl<br/>1 mM HCl</li><li>PBS<br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O<br/>1.8 mM KH<sub>2</sub>PO<sub>4</sub> </li><li>Coupling buffer<br/>0.1 M aqueous NaHCO<sub>3</sub>:dioxane = 1:1</li><li>Blocking solution<br/>1 M ethanolamine in 0.1 M Tris-HCl pH 8.0 buffer</li><li>Decolorant<br/>50% acetonitrile and 50 mM NH<sub>4</sub>HCO<sub>3</sub> </li><li>Reduction solution<br/>100 mM DTT and 100 mM NH<sub>4</sub>HCO<sub>3</sub><br/>Prepare fresh immediately prior to use</li><li>Alkylating solution<br/>100 mM iodoacetamide and 100 mM NH<sub>4</sub>HCO<sub>3</sub><br/>Prepare fresh immediately prior to use</li><li>Digestion solution<br/>25 Âµg/ml trypsin in 10 mM Tris-HCl pH 8.8 buffer</li><li>Extraction solution 1<br/>0.05% TFA and 50% acetonitrile</li><li>Extraction solution 2<br/>0.03% TFA and 66% acetonitrile</li><li>Matrix solution <br/>Î±-Cyano-4-hydroxycinnamic acid (10 mg/ml) dissolved in 50% acetonitrile/0.1% TFA</li><li>Staining solution<br/>0.3 g Coomassie Brilliant blue R250<br/>250 ml isopropanol<br/>300 ml acetic acid<br/>650 ml MQ water</li><li>Destaining solution<br/>300 ml methanol (30%)<br/>100 ml acetic acid (10%)<br/>600 ml Milli Q water</li><li>10x Tris-glycine running buffer<br/>25 mM Tris<br/>192 mM glycine<br/>0.1% SDS</li><li>5% Stacking gel<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td>Milli Q<br/></td><td>1.4 ml<br/></td></tr><tr><td>30% acrylamide mix<br/></td><td>0.33 ml<br/></td></tr><tr><td>1.0 M Tris (pH 6.8)<br/></td><td>0.25 ml<br/></td></tr><tr><td>10% SDS<br/></td><td>0.02 ml<br/></td></tr><tr><td>10% APS<br/></td><td>0.02 ml<br/></td></tr><tr><td>TEMED<br/></td><td>0.002 ml</td></tr></tbody></table></li><li>10% Resolving gel recipe<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td>Milli Q<br/></td><td>1.9 ml<br/></td></tr><tr><td>30% acrylamide mix<br/></td><td>1.7 ml<br/></td></tr><tr><td>1.5 M Tris (pH 8.8)<br/></td><td>1.3 ml<br/></td></tr><tr><td>10% SDS<br/></td><td>0.05 ml<br/></td></tr><tr><td>10% APS<br/></td><td>0.05 ml<br/></td></tr><tr><td>TEMED<br/></td><td>0.002 ml</td></tr></tbody></table></li></ol><p class="pptt" id="biaoti31317">### Acknowledgments</p><p class="ppzz">We thank the members of RIKEN NPDepo for providing the chemical libraries; Miho Tanaka (RIKEN), Dr. Takehiro Suzuki (RIKEN), and Dr. Naoshi Dohmae (RIKEN) for protein identification and Harumi Aono (RIKEN) and Emiko Sanada (RIKEN) for continuous technical help during the study. This research was partially supported by the research grant from JSPS KAKENHI (Grant Number JP16H06276, JP17H06412, JP17K07783, JP18H05503, JP18K05366, JP18H02555, JP19H05302) and AMED (Grant Number JP19cm0106112). The protocol for compound beads preparation was adapted from a previous work (Kondoh <em>et al.</em>, 2015).</p><p class="pptt" id="biaoti31318">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt" id="biaoti31319">### References</p><ol class="npwtwok"><li>Kanoh, N., Honda, K., Simizu, S., Muroi, M. and Osada, H. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15880755" target="_blank">Photo-cross-linked small-molecule affinity matrix for facilitating forward and reverse chemical genetics.</a> <em>Angew Chem Int Ed Engl</em> 44(23): 3559-3562. </li><li>Kanoh, N., Asami, A., Kawatani, M., Honda, K., Kumashiro, S., Takayama, H., Simizu, S., Amemiya, T., Kondoh, Y., Hatakeyama, S., Tsuganezawa, K., Utata, R., Tanaka, A., Yokoyama, S., Tashiro, H. and Osada, H. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17441122" target="_blank">Photo-cross-linked small-molecule microarrays as chemical genomic tools for dissecting protein-ligand interactions.</a> <em>Chem Asian J</em> 1(6): 789-797.</li><li>Kondoh, Y., Honda, K. and Osada, H. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25618334" target="_blank">Construction and application of a photo-cross-linked chemical array.</a> <em>Methods Mol Biol</em> 1263: 29-41.</li><li>Lubec, G. and Afjehi-Sadat, L. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17645314" target="_blank">Limitations and pitfalls in protein identification by mass spectrometry.</a> <em>Chem Rev</em> 107(8): 3568-3584.</li><li>Suvarna, K., Honda, K., Muroi, M., Kondoh, Y., Osada, H. and Watanabe, N. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30610116" target="_blank">A small-molecule ligand of valosin-containing protein/p97 inhibits cancer cell-accelerated fibroblast migration.</a> <em>J Biol Chem</em> 294(9): 2988-2996.<br/></li></ol>