<body>
            <h1>ChIP-Seq from Limited Starting Material of K562 Cells and <em>Drosophila </em>Neuroblasts Using Tagmentation Assisted Fragmentation Approach</h1>
            <p><strong>Authors:</strong> Junaid  Akhtar, Piyush More, Steffen Albrecht</p>
            <h2>Abstract</h2>
            <p>Chromatin immunoprecipitation is extensively used to investigate the epigenetic profile and transcription factor binding sites in the genome. However, when the starting material is limited, the conventional ChIP-Seq approach cannot be implemented. This protocol describes a method that can be used to generate the chromatin profiles from as low as 100 human or 1,000 <em>Drosophila </em>cells. The method employs tagmentation to fragment the chromatin with concomitant addition of sequencing adaptors. The method generates datasets with high signal to noise ratio and can be subjected to standard tools for ChIP-Seq analysis<br/></p>
            <p><strong>Keywords:</strong> Low Input ChIP-Seq, Easy workflow, High Signal-to-Noise ratio, Tagmentation, High reproducibility</p>
            <h2>Background</h2><p>Epigenetic state and transcription factors occupancy are critical determinants of gene expression. To understand this regulation, the precise mapping of chromatin features are very important. Chromatin immunoprecipitation coupled with next generation sequencing is a powerful technique that gives valuable insight into genome wide distributions of chromatin features (Ghavi-Helm <em>et al.</em>, 2016). Although powerful, this technique is limited in its application owing to the need of significant starting material which in some conditions are hard to meet, if not completely impossible. This limitation has driven many recent efforts to adapt ChIP-Seq approach for low amount starting material or for rare cell types (Adli <em>et al.</em>, 2010; Zheng <em>et al.</em>, 2015). One of the key limitations of these efforts is the use of sonication or MNAse for fragmenting the chromatin. The former approach is detrimental for the epitope when present in limited amount as it can lead to destruction of the epitope used for subsequent immunoprecipitation (Stathopulos <em>et al.</em>, 2004). The MNAse approach on the other hand is hard to control for its efficacy and saturation. The recent application of CUT&amp;RUN approach was successful in generating profiles from samples of 100 cells using antibody-targeted micrococcal nuclease (Skene <em>et al.</em>, 2018). However, all these approaches still resort to ligation-based library preparation approach involving multiple steps, potentially leading to increased variability as well as of loss of complexity (Seguin-Orlando <em>et al.</em>, 2013). The method described here overcomes these limitations by using commercially available Tn5 enzyme for fragmenting the chromatin with simultaneous preparation of libraries with fewer intermediate steps. The direct comparison of CUT&amp;RUN approach and TAF-ChIP reveals superior signal to noise ratio in the later, making use of standard bioinformatics pipeline amenable to this approach.</p>
        </body>

<p class="pptt" id="biaoti31355">## Materials and Reagents</p><ol class="npwtwok"><li>DNA Low binding 1.5 ml tubes (Eppendorf, catalog number: 0030108051)</li><li>Phase Lock tubes (5Prime, catalog number: 2302830)</li><li>Agencourt AMPure XP Beads (Beckmann Coultier, catalog number: A63381)</li><li>Protein G Dynabeads (Thermo Fisher Scientific, catalog number: 10003D)</li><li>BSA (Sigma-Aldrich, catalog number: A4737)</li><li>Proteinase K (Thermo Fisher Scientific, catalog number: E00491)</li><li>Yeast tRNA (Sigma-Aldrich, catalog number: R8759)</li><li>Anti H3K27Me3 antibody (Active Motif, catalog number: 39155)</li><li>Anti H3 antibody (Abcam, catalog number: ab1791)</li><li>Anti H3K4Me3 antibody (Abcam, catalog number: ab8580)</li><li>Anti H3K9Me3 antibody (Active Motif, catalog number: 39161)</li><li>Collagenase I (Sigma-Aldrich, catalog number: 1148089)</li><li>Papain (Sigma-Aldrich, catalog number: 1495005)</li><li>Primers (Table 1)<br/><br/><strong>Table 1. List of primers with unique barcodes</strong> <table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:900;"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><br/></td><td>Primers<br/></td></tr><tr><td>fw ATAC-seq primer, general, no index<br/></td><td>AATGATACGGCGACCACCGAGATCTACACTCGTCGGCAGCGTCAGATGT*G<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 34 CATGGC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGCCATGGTCTCGTGGGCTCGGAGATG*T</td></tr><tr><td>rev ATAC-seq primer, Truseq index, 48 TCGGCA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTGCCGAGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 22 CGTACG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCGTACGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 15 ATGTCA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTGACATGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 46 TCCCGA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTCGGGAGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 45 TCATTC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGAATGAGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 40 CTCAGA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTCTGAGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 39 CTATAC</td><td>CAAGCAGAAGACGGCATACGAGATGTATAGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 38 CTAGCT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAGCTAGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 37 CGGAAT<br/></td><td>CAAGCAGAAGACGGCATACGAGATATTCCGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 36 CCAACA<br/></td><td>CAAGCAGAAGACGGCATACGAGATTGTTGGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 35 CATTTT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAAAATGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 25 ACTGAT<br/></td><td>CAAGCAGAAGACGGCATACGAGATATCAGTGTCTCGTGGGCTCGGAGATG*T</td></tr><tr><td>rev ATAC-seq primer, Truseq index, 26 ATGAG<br/></td><td>CAAGCAGAAGACGGCATACGAGATGCTCATGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 27 ATTCCT<br/></td><td>CAAGCAGAAGACGGCATACGAGATAGGAATGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 28 CAAAAG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCTTTTGGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 23 GAGTGG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCCACTCGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 24 GGTAGC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGCTACCGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr><td>rev ATAC-seq primer, Truseq index, 42 TAATCG<br/></td><td>CAAGCAGAAGACGGCATACGAGATCGATTAGTCTCGTGGGCTCGGAGATG*T<br/></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>rev ATAC-seq primer, Truseq index, 41 GACGAC<br/></td><td>CAAGCAGAAGACGGCATACGAGATGTCGTCGTCTCGTGGGCTCGGAGATG*T<br/></td></tr></tbody></table><br/></li><li>NEXTERA XT DNA Library prep Kit (Illumina, catalog number: FC-131-1024)</li><li>NEBNext High-Fidelity 2x PCR Master Mix (New England Biolabs, catalog number: M0541)</li><li>Glycogen (Roche, catalog number: 10901393001)</li><li>37% Formaldehyde (Sigma-Aldrich, catalog number: F8775)</li><li>Glycine (Sigma-Aldrich, catalog number: 50046)</li><li>Protease Inhibitor Cocktail (Bimake, catalog number: B14001)</li><li>Tris Buffer pH 8.0 (Applichem, catalog number: A4577)</li><li>EDTA (Sigma-Aldrich, EDS catalog number: EDS-500G)</li><li>Sodium Chloride (Sigma-Aldrich, catalog number: S3014)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: T8787)</li><li>20% sodium dodecyl sulfate (Sigma-Aldrich, catalog number: 05030)</li><li>Nuclease-free water (Qiagen, catalog number: 129115)</li><li>Tris (hydroxymethyl) aminomethane (Sigma-Aldrich, catalog number: 252859)</li><li>N,N-Dimethylformamide (Sigma-Aldrich, catalog number: D4551)</li><li>Qubit dsDNA High Sensitivity Kit (Thermo Fisher Scientific, catalog number: Q33230)</li><li>DNA High Sensitivity Kit for Bioanalyzer (Agilent, catalog number: 5067-4627)</li><li>Schneiderâs medium (Thermo Fisher Scientific, catalog number: 21720-001)</li><li>Phenol:Chloroform (Sigma-Aldrich, catalog number: P3803)</li><li>3 M sodium acetate (Thermo Fisher Scientific, catalog number: AM9740)</li><li>Ethanol (Sigma-Aldrich, catalog number: 51976)</li><li>Hoechst stain (Thermo Fisher Scientific, catalog number: 62249)</li><li>MgCl<sub>2</sub> (Sigma-Aldrich, catalog number: 63068)</li><li>Dissociation Solution (see Recipes)</li><li>Blocking Buffer (see Recipes)</li><li>RIPA (140 mM) (see Recipes)</li><li>RIPA (250 mM) (see Recipes)</li><li>TE Buffer (see Recipes)</li><li>Tagmentation Buffer (Home-made) (see Recipes)</li></ol><p class="pptt" id="biaoti31356">### Equipment</p><ol class="npwtwok"><li>DiaMag 0.2 ml-magnetic rack (Diagenode, catalog number: B04000001)</li><li>DynaMag-2 magnetic rack (Thermo Fisher Scientific, catalog number: 12321D)</li><li>Bioruptor plus (Diagenode, catalog number: B01020001)</li><li>Thermomix compact (Eppendorf, catalog number: T1317)</li><li>Bio-Rad PCR Cycler (Bio-Rad, catalog number: C1000)</li><li>Mini tube rotator (Biotools, AG, R2001)</li><li>Tabletop minicentrifuge (Thermo Fisher Scientific, catalog number: 75002478)</li><li>Agilent Bioanalyzer 2100 (Agilent, catalog number: G2939BA)</li><li>Qubit Fluorometer 3 (Thermo Fisher Scientific, catalog number: Q33216)</li><li>Flow cytometer (BD FACS Aria)<br/></li></ol><p class="pptt" id="biaoti31357">### Software</p><ol class="npwtwok"><li>MACS2 (version 2.1.1.20160309, <a href="https://github.com/taoliu/MACS" target="_blank">https://github.com/taoliu/MACS</a>)</li><li>Deeptools (version 3.1.3, <a href="https://github.com/deeptools/deepTools" target="_blank">https://github.com/deeptools/deepTools</a>)</li><li>Samtools (version 1.09, <a href="https://github.com/samtools/samtools" target="_blank">https://github.com/samtools/samtools</a>)</li><li>bowtie2 (version 2.3.5, <a href="https://github.com/BenLangmead/bowtie2" target="_blank">https://github.com/BenLangmead/bowtie2</a>)</li><li>FastQC (version 0.10.1, <a href="https://github.com/s-andrews/FastQC" target="_blank">https://github.com/s-andrews/FastQC</a>)<br/></li></ol><p class="pptt" id="biaoti31358">### Procedure</p><ol class="npwone"><li>Cell fixation and sorting (human cells)<ol class="npwtwo"><li>Harvest the cells in a 1.5 ml Eppendorf tube, and resuspend them in PBS.</li><li>Fix the cells for 10 min at room temperature with 1% formaldehyde (in PBS).</li><li>Quench the crosslink with 125 mM glycine and incubate for 5 min at room temperature.</li><li>Stain the cells with Hoechst stain and use the flow cytometer to sort 100 cells into 200 Î¼l of 140 mM RIPA buffer (Recipe 3).<br/><em>Note: Sorting small number of cells will have no effect on buffer composition owing to small volume.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell isolation, fixation and sorting (<em>Drosophila</em> neuroblasts)<ol class="npwtwo"><li>Dissect the required number of larval brains in PBS (for example: transgenic flies expressing GFP-tagged deadpan (Dpn) protein under the control of its endogenous enhancer).</li><li>Wash once with PBS and fix the brains with 1% formaldehyde (in PBT) for 10 min at room temperature.</li><li>Quench the crosslink with 125 mM glycine and incubate for 5 min at room temperature.</li><li>Remove the quenching solution and wash twice with PBS.</li><li>Add the dissociation solution (Recipe 1) to the brain tissue and leave it for 1 h at 30 Â°C with constant shaking at 650 rpm. Stir the brains at every 15 min interval by pipetting them up and down.</li><li>Pellet the cells by centrifuging them at 100 <em>x g</em> and resuspend in PBS.</li><li>Sort cells on flow cytometer under GFP channel, and directly sort 1,000 cells into 200 Î¼l of 140 mM RIPA buffer.<br/><em>Note: Sorting small number of cells will have no effect on buffer composition owing to small volume.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Immunoprecipitation<ol class="npwtwo"><li>Bind 1 Î¼g of antibody to 15 Î¼l of protein G dynabeads in 200 Î¼l blocking buffer (Recipe 2).</li><li>Incubate at 4 Â°C for 2-3 h.</li><li>Break the nuclei of the cells collected in RIPA for 3 cycles at low power setting in a Bioruptor sonicator, 30 sec âONâ/âOFFâ.</li><li>Remove the blocking buffer by putting the tubes on DynaMag-2 magnetic rack, and add the lysate from Step C1.</li><li>Incubate at 4 Â°C for overnight with head over tail rotations.</li><li>Separate the beads with magnetic rack and wash twice with 300 Î¼l of home-made tagmentation buffer (Recipe 6) by resuspending the beads with 1,000 Î¼l pipette. Pulse-spin the tubes on micro centrifuge and separate again with DynaMag-2 magnetic rack, remove the buffer as much as possible without disturbing the beads.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Tagmentation, reverse cross linking, and Phenol:Chloroform extraction<ol class="npwtwo"><li>Resuspend the beads in 20 Î¼l of 1x tagmentation buffer containing 1 Î¼l of Tn5 transposase (from NEXTERA XT DNA Library prep Kit).</li><li>Incubate the resuspended beads at 37 Â°C for 40 min in a thermoblock with constant shaking at 500 rpm.</li><li>Remove tagmentation buffer by separating the beads on DynaMag-2 magnetic rack and wash the beads as following:<ol class="npwthree"><li>Once with 140 mM RIPA;</li><li>Four times with 250 mM RIPA (Recipe 4);</li><li>Twice with TE buffer (Recipe 5).</li></ol><em>Note: Each wash should be for 3 min and with 200 Î¼l of wash buffer. All the washes should be performed in cold room operating at 4 Â°C.</em> </li><li>Resuspend the washed beads in 100 Î¼l of TE buffer, and add 5 Î¼l of 20 mg/ml proteinase K. Incubate the beads for at least 6 h at 60 Â°C in thermoblock with shaking at 500 rpm.</li><li>Add 100 Î¼l of TE buffer, followed by 300 Î¼l of Phenol:Chloroform. Vortex the tubes briefly and transfer the content to the phase lock tube (pre-spun at 14,000 rpm for 30 s). Centrifuge the tubes at 20,000 <em>x g</em> for 5 min at room temperature.</li><li>Transfer the upper aqueous phase to fresh DNA low bind tube, add 5 Î¼l of 20 mg/ml of glycogen and 20 Î¼l of 3 M sodium acetate (pH 5.2) in the exact order, vortex briefly. Add 700 Î¼l of absolute ethanol and vortex briefly. Incubate overnight at -80 Â°C to precipitate.</li><li>Centrifuge the tubes at full speed (20,000 <em>x g</em> ) at 4 Â°C for 30-45 min. Wash the pellet once with 75% ethanol by centrifuging at 20,000 <em>x g</em> at 4 Â°C for 10 min and resuspend the pellet in 30 Î¼l of TE buffer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>PCR amplification and purification<ol class="npwtwo"><li>Set up the PCR reaction as per the following reaction mix.<br/>Forward primer (Universal) 20 Î¼M: 2 Î¼l<br/>Reverse primer (with index) 20 Î¼M: 2 Î¼l<br/>Reverse cross-linked material: 30 Î¼l<br/>Nuclease free water: 16 Î¼l<br/>2x NEBNext High-Fidelity PCR Mix: 50 Î¼l</li><li>Run the PCR reaction in a thermal cycler with heated lid set at 105 Â°C, according to following cycling parameter:<br/>72 Â°C for 3 min<br/>{98 Â°C for 10 s<br/>63 Â°C for 30 s<br/>72 Â°C for 30 s}, 12 cycles<br/>72 Â°C for 5 min<br/>Hold at 4 Â°C.</li><li>In the meantime, when PCR reaction is running, remove the Ampure XP beads from 4 Â°C to room temperature. The Ampure beads should be left at room temperature for at least 30 min to equilibrate, prior to use.</li><li>Add 0.2x volume (relative to PCR reaction volume) of Ampure XP beads to the PCR reaction and incubate at room temperature for 5 min. Separate the beads on DiaMag 0.2 ml magnetic rack and carefully transfer the solution without carrying over the beads to a new PCR tube. Discard the Ampure XP beads.</li><li>Add 0.8x volume of Ampure XP (relative to PCR reaction volume) beads to the solution and incubate at room temperature for 5 min. Separate the beads on DiaMag 0.2 ml magnetic rack and carefully discard the solution without disturbing the beads.</li><li>Wash the beads twice with 200 Î¼l freshly prepared of 80% ethanol, without disturbing the beads.</li><li>Dry the beads until the appearance of the bead surface turns from glossy to matte.</li><li>Elute the finished library in 10-15 Î¼l of nuclease free water.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Quality control of libraries<br/><ol class="npwtwo"><li>Measure the concentration of the library with Qubit dsDNA High Sensitivity Kit, following the manufactureâs protocol.</li><li>Run the library on a DNA High Sensitivity Chip with Agilent Bioanalyzer, to check for size distribution of the DNA fragments. Follow the protocol provided with the DNA High Sensitivity Kit for loading and running the sample.</li><li>Determine the molar concentration of the library and dilute it to 10 nM concentration or per the requirement of the sequencing facility.<br/><em>Note: The concentration of the library and average size fragment of the library (as revealed by Bioanalyzer) is used to calculate the molar concentration. The formula employed to calculate the concentration is, concentration in nM = (concentration in ng/Î¼l from Qubit measurement)/ (660 g/mol x average size of the library) x 106.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti31359">### Data analysis</p><ol class="npwtwok"><li>Sequence quality check<br/>Use FastQC to assess the sequencing quality, using fastqc files. Alternatively, one can also use other equivalent QC tools such as MAPQC.</li><li>Generate index for reference genomes using bowtie2 (download the fasta files from Ensemble database)<br/><span style="font-family:Courier New;">bowtie2-build -f $FASTA_PATH/genome.fa $INDEX_PATH/bowtie2_index</span> </li><li>Mapping the reads either to dm6 or hg38 assembly using bowtie2 (Langmead, 2010)<br/><span style="font-family:Courier New;">bowtie2-x $INDEX_PATH/genome -p 24 -1$SEQ_PATH/*file_1*.gz -2 $SEQ_PATH/*file_2*.gz | samtools view -bS &gt; file.bam</span> </li><li>Sorting mapped reads<br/><span style="font-family:Courier New;">Samtools sort file.bam &gt; file.sorted.bam</span> </li><li>Indexing bam files<br/><span style="font-family:Courier New;">Samtools index file.bam file.bam.bai</span> </li><li>Generating coverage files using DeepTools (Ramirez <em>et al.</em> , 2016)<br/><span style="font-family:Courier New;">bamCoverageâb file.bamânormalizeUsing RPKM-bs 50âsmoothLength 175-o file.bw</span><br/>A representative track example after generating coverage files is shown in Figure 1.</li><li>Peak calling using MACS2, and H3 as control (Zhang <em>et al.</em> , 2008)<br/><span style="font-family:Courier New;">Macs2 callpeak-t $File-c $File -f BAMPE-g dâoutdirâname</span><br/><span style="font-family:Courier New;">Macs2 callpeak-t $File-c $File -f BAMPE-g hsâoutdirâname</span> </li><li>Heatmaps using DeepTools<br/><span style="font-family:Courier New;">ComputeMatrix scale-regionsâS file.bw-R genes.bedâb 600âa 600-o matrix.mat.gz<br/>plotHeatmap-m matrix.mat.gz-out Heatmap.pdf</span><br/><br/>A flow-chart displaying the major steps in the protocol is shown in Figure 1. A representative track example after generating coverage files is shown in Figure 2. The comparison of TAF-ChIP approach performed with 100 cells and recently published CUT&amp;RUN datasets, together with the ENCODE datasets, is shown in Figure 3.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218163508_6586.jpg" width="115"/><br/><strong>Figure 1. Steps involved in the Tagmentation Assisted Fragmentation (TAF)-ChIP protocol and subsequent data processing. </strong>Steps involved in the TAF-ChIP workflow in chronological order represented as a flow-chart.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218163655_0720.jpg" width="195"/><br/><strong>Figure 2. Genome browser track example of H3K4Me3 generated by TAF-ChIP approach and conventional ChIP-Seq approach.</strong> Genome browser tracks of ChIP performed in 1,000 FACS sorted <em>Drosophila</em> neural stem cells (NSCs) with TAF-ChIP approach (TAF_K4Me3) and corresponding conventional ChIP-Seq datasets from 1.2 million FACS sorted NSCs (Conv_K4Me3) with independent biological duplicates. The label below the tracks shows the gene model and the y-axis represents normalized read density in reads per million.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218163743_8751.jpg" width="208"/><br/><strong>Figure 3. Comparison of TAF-ChIP with conventional ChIP-Seq and with another contemporary low amount method.</strong> A. Comparison of TAF-ChIP and CUT&amp;RUN for H3K27Me3 in K562 cells using ROC curves. The plots were generated using the conventional ChIP-Seq ENCODE dataset as reference with 5% FDR cutoff. The peaks for TAF-ChIP replicates and the CUT&amp;RUN datasets were generated with MACS2 without FDR thresholding. Peaks were mapped to 5 kb non-overlapping genomic windows to calculate true-positive rate, false-positive rate and precision for a changing P-Value threshold. Area under the curve (AUC) is indicated in the legend and the* indicates the failure to calculate the AUC. B. Precision-recall curve for TAF-ChIP and CUT&amp;RUN datasets for H3K27Me3 in K562 cells. The figure is adapted from Akhtar <em>et al.</em> (2019).<br/></li></ol><p class="pptt" id="biaoti31360">### Recipes</p><ol class="npwtwok"><li>Dissociation Solution<br/>Schneiderâs medium<br/>1 mg/ml collagenase I<br/>1 mg/ml papain</li><li>Blocking Buffer<br/>10 mM Tris-Cl pH 8.0 <br/>140 mM NaCl<br/>0.5 mM EDTA pH 8.0<br/>1% Triton X-100<br/>0.1% SDS<br/>0.2 mg/ml BSA<br/>0.05 mg/ml Glycogen<br/>0.2 mg/ml Yeast tRNA</li><li>RIPA (140 mM)<br/>10 mM Tris-Cl pH 8.0<br/>140 mM NaCl<br/>0.5 mM EDTA pH 8.0<br/>1% Triton X-100<br/>0.1% SDS</li><li>RIPA (250 mM) <br/>10 mM Tris-Cl pH 8.0 <br/>140 mM NaCl<br/>0.5 mM EDTA pH 8.0<br/>1% Triton X-100<br/>0.1% SDS</li><li>TE Buffer<br/>10 mM Tris-Cl pH 8.0<br/>0.5 mM EDTA pH 8.0</li><li>Tagmentation Buffer (Home-made)<br/>20 mM Tris(hydroxymethyl)aminomethane pH 7.6<br/>10 mM MgCl<sub>2</sub> <br/>20% (vol/vol) dimethyl formamide<br/></li></ol><p class="pptt" id="biaoti31361">### Acknowledgments</p><p class="ppzz">This work was supported by the Deutsche Forschungsgemeinschaft (DFG) grant DFG BE 4728 1-1 and 3-1. The International PhD Programme (IPP) of the Institute of Molecular Biology, Mainz for supporting the PhD work of S.A. and P.M. We would also like to thank IMB FACS core facility for helping us in sorting.<br/>  This work is a detailed and adapted version of our previously published work (Akhtar <em>et al.</em> , 2019). The FACS sorting workflow was adapted and modified from an earlier work (Berger <em>et al.</em> , 2012).</p><p class="pptt" id="biaoti31362">### Competing interests</p><p class="ppzz">The authors declare no financial and non-financial competing interest.</p><p class="pptt" id="biaoti31363">### Ethics</p><p class="ppzz">The described experiments in <em>Drosophila</em> were performed according to the guidelines (Invertebrates are not under animal welfare/ethics laws in Germany). None of the experiments were performed on human subjects.</p><p class="pptt" id="biaoti31364">### References</p><ol class="npwtwok"><li>Adli, M., Zhu, J. and Bernstein, B. E. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20622861" target="_blank">Genome-wide chromatin maps derived from limited numbers of hematopoietic progenitors.</a> <em>Nat Methods</em> 7(8): 615-618.</li><li>Akhtar, J., More, P., Albrecht, S., Marini, F., Kaiser, W., Kulkarni, A., Wojnowski, L., Fontaine, J. F., Andrade-Navarro, M. A., Silies, M. and Berger, C. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31331983" target="_blank">TAF-ChIP: an ultra-low input approach for genome-wide chromatin immunoprecipitation assay.</a> <em>Life Sci Alliance</em> 2(4).</li><li>Berger, C., Harzer, H., Burkard, T. R., Steinmann, J., van der Horst, S., Laurenson, A. S., Novatchkova, M., Reichert, H. and Knoblich, J. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22884370" target="_blank">FACS purification and transcriptome analysis of <em>Drosophila</em> neural stem cells reveals a role for Klumpfuss in self-renewal.</a> <em>Cell Rep</em> 2(2): 407-418.</li><li>Ghavi-Helm, Y., Zhao, B. and Furlong, E. E. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27730588" target="_blank">Chromatin immunoprecipitation for analyzing transcription factor binding and histone modifications in <em>Drosophila</em> .</a> <em>Methods Mol Biol</em> 1478: 263-277.</li><li>Langmead, B. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21154709" target="_blank">Aligning short sequencing reads with Bowtie.</a><em> Curr Protoc Bioinformatics</em> Chapter 11: Unit 11.7.</li><li>Ramirez, F., Ryan, D. P., Gruning, B., Bhardwaj, V., Kilpert, F., Richter, A. S., Heyne, S., Dundar, F. and Manke, T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27079975" target="_blank">deepTools2: a next generation web server for deep-sequencing data analysis.</a> <em>Nucleic Acids Res</em> 44(W1): W160-165.</li><li>Seguin-Orlando, A., Schubert, M., Clary, J., Stagegaard, J., Alberdi, M. T., Prado, J. L., Prieto, A., Willerslev, E. and Orlando, L. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24205269" target="_blank">Ligation bias in illumina next-generation DNA libraries: implications for sequencing ancient genomes.</a> <em>PLoS One</em> 8(10): e78575.</li><li>Skene, P. J., Henikoff, J. G. and Henikoff, S. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29651053" target="_blank">Targeted in situ genome-wide profiling with high efficiency for low cell numbers. </a><em>Nat Protoc</em> 13(5): 1006-1019.</li><li>Stathopulos, P. B., Scholz, G. A., Hwang, Y. M., Rumfeldt, J. A., Lepock, J. R. and Meiering, E. M. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15459333/" target="_blank">Sonication of proteins causes formation of aggregates that resemble amyloid.</a> <em>Protein Sci</em> 13(11): 3017-3027.</li><li>Zhang, Y., Liu, T., Meyer, C. A., Eeckhoute, J., Johnson, D. S., Bernstein, B. E., Nusbaum, C., Myers, R. M., Brown, M., Li, W. and Liu, X. S. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18798982" target="_blank">Model-based analysis of ChIP-Seq (MACS).</a> <em>Genome Biol</em> 9(9): R137.</li><li>Zheng, X., Yue, S., Chen, H., Weber, B., Jia, J. and Zheng, Y. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26549448" target="_blank">Low-Cell-Number epigenome profiling aids the study of lens aging and hematopoiesis.</a> <em>Cell Rep</em> 13(7): 1505-1518.<br/></li></ol>