<body>
            <h1>CRISPR-Cas9 Genome Editing of <em>Plasmodium knowlesi</em></h1>
            <p><strong>Authors:</strong> Franziska Mohring, Melissa N. Hart, Avnish Patel, David A. Baker, Robert W. Moon</p>
            <h2>Abstract</h2>
            <p><em>Plasmodium knowlesi</em> is a zoonotic malaria parasite in Southeast Asia that can cause severe and fatal malaria in humans. The main hosts are Macaques, but modern diagnostic tools reveal increasing numbers of human infections. After <em>P. falciparum</em>, <em>P. knowlesi</em> is the only other malaria parasite capable of being maintained in long term <em>in vitro</em> culture with human red blood cells (RBCs). Its closer ancestry to other non-falciparum human malaria parasites, more balanced AT-content, larger merozoites and higher transfection efficiencies, gives <em>P. knowlesi</em> some key advantages over <em>P. falciparum</em> for the study of malaria parasite cell/molecular biology. Here, we describe the generation of marker-free CRISPR gene-edited <em>P. knowlesi</em> parasites, the fast and scalable production of transfection constructs and analysis of transfection efficiencies. Our protocol allows rapid, reliable and unlimited rounds of genome editing in <em>P. knowlesi</em> requiring only a single recyclable selection marker.</p>
            <p><strong>Keywords:</strong> <em>Plasmodium knowlesi</em>, <em>Plasmodium vivax</em>, CRISPR-Cas9, Genome editing, Orthologue replacement, Transfection, Malaria</p>
            <h2>Background</h2><p>Malaria is an infectious disease caused by <em>Plasmodium</em> parasites worldwide with over 200 million cases yearly (WHO, 2018). Six different <em>Plasmodium</em> species are known to cause significant numbers of human infections, but of these only <em>Plasmodium knowlesi</em> is a zoonotic malaria parasite. Its main hosts are macaque monkeys that are distributed in Southeast Asia, but it can also cause severe and fatal malaria in humans (Rajahram <em>et al.</em>, 2019). Due to its morphological similarity to <em>P. malariae</em>, it was often misdiagnosed by microscopy. New PCR-based diagnostic tools reveal increasing numbers of human infections in Malaysia and Indonesia, revealing that up to 80% of overall malaria cases in some regions are caused by <em>P. knowlesi</em> (Lubis <em>et al.</em>, 2017; Cooper <em>et al.</em>, 2020). After <em>P. falciparum</em>, <em>P. knowlesi</em> is the only other malaria parasite adapted to grow in a continuous human red blood cell <em>in vitro</em> culture system (Moon <em>et al.</em>, 2013; Lim <em>et al.</em>, 2013; Gruring <em>et al.</em>, 2014). <em>P. knowlesi</em> is a close relative of <em>P. vivax</em>, the most widespread human malaria parasite and both rely on Duffy positive (Fy+) red blood cells for invasion.<br/><br/>Compared to <em>P. falciparum</em>, <em>P. knowlesi</em> has a shorter life cycle (27 h in cell culture vs approximately 46 h for <em>P. falciparum</em> 3D7), more balanced AT-content (60% vs 82%), larger merozoites and higher transfection efficiencies (up to 30% with episomal plasmids) giving <em>P. knowlesi</em> some advantages over <em>P. falciparum</em> as an experimental system. The more balanced AT-content alone means that it is easier to design oligonucleotide primers, generate transfection constructs, and find well-positioned PAM sites required for Cas9 binding and cleaving of the target DNA. Successful use of CRISPR-Cas9 genome editing in <em>P. falciparum</em> (Ghorbal <em>et al.</em>, 2014) gave promise of a very efficient method for <em>P. knowlesi</em>.<br/><br/>Here we present a marker-free CRISPR gene editing method for <em>P. knowlesi</em> based on a two plasmid system allowing the use and recycling of a single selection marker. The first plasmid contains a Cas9 expression cassette, as well as positive and negative selection markers. The second plasmid contains the repair template which provides the template for repair. This normally contains the modification to be introduced flanked either side by 800 bp regions of homology to the target locus. <em>P. knowlesi</em> readily accepts linearized plasmids for homologous recombination, therefore repair template generation can be carried out with conventional cloning or with a two- or three-step PCR method. This protocol can be used to generate parasite lines with C- and N-terminal tagging, knock-out and orthologue replacement (Mohring <em>et al.</em>, 2019). The PCR method for donor generation is much faster and ideally suited for generation of knockout or tagging constructs. For more complex or larger constructs like gene replacement lines we recommend using a conventional cloned and sequenced plasmid construct.<br/><br/>These approaches are well suited to undertaking functional analysis of parasite genes through knockout or by addition of a tag. As the desired modification can be introduced at any position within the gene, including internally, the introduction of N-terminal or internal tags is far easier than when using conventional approaches. The resultant parasite lines are markerless and thus, using iterative modifications, gene knockouts and tagged genes can be combined together to aid phenotype or examine large redundant gene families. The method can also be combined with conditional knockout approaches such as the DiCre recombinase that has recently been implemented to study essential genes in <em>P. knowlesi</em> (Knuepfer <em>et al.</em>, 2019). Another major usage for this system is for orthologue replacement (OR) approaches, where a <em>P. knowlesi</em> gene is directly replaced with its orthologue from a different malaria parasite species. It is currently only possible to grow <em>P. falciparum</em> and <em>P. knowlesi</em> in culture with human RBCs, so this provides a mechanism to study genes from the important human malaria pathogen that cannot be maintained in culture, <em>P. vivax</em>âbut also neglected parasites such as <em>P. ovale</em> and <em>P. malariae</em>. The gene is placed directly under the control of the endogenous <em>P. knowlesi</em> promoter ensuring appropriate stage specific and stable expression levelsâproviding significant advantages over the highly variable expression levels of episomal constructs. This has already been used to develop a <em>P. knowlesi</em> PvDBP<sup>OR</sup> line, which enables scalable screening for inhibitory antibodies targeting <em>P. vivax</em> duffy binding protein (PvDBP), the current lead <em>P. vivax</em> blood stage vaccine candidate (Mohring <em>et al.</em>, 2019; Rawlinson <em>et al.</em>, 2019). The same approach can readily be applied to studying other vaccine or drug resistance markers from <em>P. vivax</em>, or indeed used to generate a range of tools to study both the basic biology or support translational research into non-falciparum malaria parasites.</p>
        </body>

<p class="pptt" id="biaoti31423">## Materials and Reagents</p><ol class="npwtwok"><li>1.5 ml reaction tube (Eppendorf, catalog number: 0030120086)</li><li>15 ml centrifuge tube (Falcon, Corning, catalog number: 352196)</li><li>96 flat-bottom plates (CytoOne, Starlab, catalog number: CC7672-7596)</li><li>6-well plate (CytoOne, Starlab, catalog number: CC7672-7506)</li><li><em>Plasmodium knowlesi</em> A1-H.1 wild type (Mike Blackman, Francis Crick Institute London) (Moon <em>et al.</em>, 2013)</li><li>XL10-Gold Ultracompetent Cells (Agilent, catalog number: 200315), -70 Â°C</li><li>Stellar<sup>TM</sup> competent cells (Clontech, catalog number: 636766)</li><li>pCas9/sg plasmid (Robert W. Moon, London School of Hygiene and Tropical Medicine, London, UK)</li><li>pKconGFP plasmid (Robert W. Moon, London School of Hygiene and Tropical Medicine, London, UK)</li><li>RPMI-1640 Medium (Sigma, catalog number: R5886)</li><li>L-glutamine (Sigma, catalog number: G7513-100ML), -20 Â°C</li><li>Horse serum (PAN BIOTECH, catalog number: P30-0711), -20 Â°C</li><li>Nycodenz (Progen, catalog number: 1002424), RT</li><li>4-[7-[(dimethylamino)methyl]-2-(4-fluorphenyl)imidazo[1,2-a]pyridin-3-yl]pyrimidin-2-amine; compound 2 (Michael Blackman, Francis Crick Institute, Lonon, UK)</li><li>P3 Primary cell 4D Nucleofector X Kit L (Lonza, calalog number: V4XP-3024)</li><li>CloneAmp HiFi PCR Premix (TaKaRa, catalog number: 639298)</li><li>BtgZI restriction enzyme (New England BioLabs, catalog number: R0703S), -20 Â°C</li><li>In-Fusion HD Cloning Kit Plus (TaKaRa, catalog number: 638910)</li><li>NEBuffer 2 (New England BioLabs, catalog number: B7002S), -20 Â°C</li><li>Ampicillin sodium salt (Sigma-Aldrich, catalog number: A0166-5G), -20 Â°C</li><li>Hoechst 33342 (New England BioLabs, catalog number: 4082), -20 Â°C</li><li>Agarose, Molecular Grade (Bioline Reagents Ltd, catalog number: BIO-41026)</li><li>TAE Buffer 50x (VWR International, catalog number: A1691.1000)</li><li>HyperLadder 1 kb (Bioline Reagents Ltd, catalog number: BIO-33026)</li><li>QIAquick PCR Purification kit (QIAGEN, calalog number: 28104)</li><li>Sequencing Kit, Mix2Seq (eurofins)</li><li>QIAGEN Plasmid Mini Kit (QIAGEN, catalog number: 12123)</li><li>QIAGEN Plasmid Midi Kit (QIAGEN, catalog number: 12143)</li><li>QIAquick Gel Extraction Kit (QIAGEN, catalog number: 28704)</li><li>LB Broth, Miller (Sigma-Aldrich, catalog number: L3522-250G)</li><li>CH<sub>3</sub>COONa, Sodium acetate, NaAc (Sigma-Aldrich, catalog number: S2889-250G)</li><li>CaNa<sub>2</sub>Â·EDTA (Sigma-Aldrich, catalog number: ED2SC-100G)</li><li>Ethanol (Fisher Chemicals, E/0650DF/17)</li><li>DNeasy blood and tissue kit (QIAGEN, catalog number: 69504)</li><li>GoTaq Green Master Mix (Promega, catalog number: M7122), -20 Â°C</li><li>Ibidi Î¼-slide VI 0.4, Poly-L-Lysine (Ibidi, Thistle Scientific Ltd, catalog number: 80604)</li><li>Oligos (IDT, Integrated DNA Technologies)</li><li>Ancotil/ 5-Fluorocytosine 2.5 g/250 ml Solution for Infusion (PubChem CID: 3366), RT</li><li>DMSO (Sigma-Aldrich, catalog number: 276855-250ML)</li><li>Tris HCl, Trizma<sup>Â®</sup> hydrocloride solution 1 M (Sigma, catalog number: T-3038-1L)</li><li>KCl (Sigma, catalog number: P9541-500g)</li><li>RPMI-1640 (HEPES Modification, With 25 mM HEPES, without L-glutamine, Merck, catalog number: R558)</li><li>Sodium bicarbonate (Sigma, catalog number: S5761)</li><li>Dextrose (Sigma, catalog number: G7021)</li><li>Hypoxanthine (Sigma, catalog number: H9636)</li><li>Albumax II (Gibco, catalog number: 11560376)</li><li>L-glutamine (Merck, catalog number: 59202C)</li><li>Custom Modified RPMI media w/o glutamine (Life Technology Brand) (see Recipes)</li><li>Nycodenz stock solution (100%) (see Recipes) (Alere Technologies AS, catalog number: 1002424)</li><li>Nycodenz working solution (55%) (see Recipes)</li><li>Pyrimethamine (Sigma, catalog number: P4200000) (see Recipes)</li><li>Ancotil 2.5 g/250 ml Solution for Infusion (see Recipes)<br/></li></ol><p class="pptt" id="biaoti31424">### Equipment</p><ol class="npwtwok"><li>Amaxa 4D-Nucleofector<sup>TM</sup> Core Unit (Lonza, catalog number: AAF-1002B)</li><li>Amaxa 4D-Nucleofector<sup>TM</sup> X Unit (Lonza, catalog number: AAF-1002X)</li><li>Centrifuge with cooling function (Eppendorf, catalog number: 5424 R)</li><li>Heat block (USA Scientific, Thermal-Lok 2-Position Dry Heat Bath, catalog number: 2510-1102)</li><li>Gel electrophoresis chamber (Analytik Jena, catalog number: 846-025-100)</li><li>ChemiDoc Imaging System (BioRad, catatog number: 17001401)</li><li>ThermoMixer C (Eppendorf, catalog number: 2231000574)</li><li>Microscope (Invitrogen EVOS Digital Color Fluorescence Microscope, AMEFC4300, catalog number: 15319601)</li><li>Spectrophotometer DS-11 Fx+ (DeNovix)</li><li>Thermal Cycler PCRmax Alpha Cycler (Cole-Parmer, catalog number: SI-93945-12)<br/></li></ol><p class="pptt" id="biaoti31425">### Software</p><ol class="npwtwok"><li>SnapGene (GSL Biotech LLC, <a href="https://www.snapgene.com/" target="_blank">www.snapgene.com</a>)</li><li>Protospacer Software (BIHP, Institute Pasteur, Paris, France, <a href="http://www.protospacer.com/" target="_blank">www.protospacer.com</a>)</li><li>Image Lab Software (BioRad)</li><li>CRISPR software (<em>e.g.</em>, Benchling, Protospacer)<br/></li></ol><p class="pptt" id="biaoti31426">### Procedure</p><ol class="npwone"><li>Cloning of pCas9/sg plasmid<ol class="npwtwo"><li>Choose guide sequence in the gene of interest<ol class="npwthree"><li>Find PAM site (NGG) in the gene region that is going to be deleted/replaced. The positioning of the PAM site will vary depending on the goal. For knock-out or replacement studies it can be placed anywhere within the deleted/replaced region. For tagging studies the guide RNA sequence should cross the position of tag insertion.</li><li>Determine off- and on-target scores of the guide sequence (N20 upstream of the NGG site) with CRISPR software (<em>e.g.</em>, Benchling, Protospacer). Choose one or multiple guide sequences with off-target scores &lt; 0.03. The positioning and off-target scores should be your overriding factors when deciding on a guide sequence. We recommend avoiding extremely AT-rich sequences. The on-target scores are normally determined in the context of other species, and in our experience do not correlate well with success, so whilst they should be considered, the other criteria should be prioritized.</li></ol></li><li>Design In-Fusion oligos for integration of the target sequence into the pCas9/sg plasmid<br/>Forward oligo: 5â²-TTACAGTATATTATT(N20)GTTTTAGAGCTAGAA-3â²<br/>Reverse oligo: 5â²-TTCTAGCTCTAAAAC(N20)AATAATATACTGTAA-3â²<br/>The schematic in Figure 1 depicts the insertion of the guide sequence into the pCas9/sg plasmid.</li><li>Linearize 2 Î¼g pCas9/sg plasmid with 1 Î¼l BtgZI enzyme and 5 Î¼l cut smart buffer in 50 Î¼l reaction volume for at least 2 h at 60 Â°C and clean up with QIAquick PCR Purification kit. Confirm linearization by running on 0.8% agarose gel. Dilute linearized plasmid to 50 ng/Î¼l.</li><li>Anneal guide oligos<ol class="npwthree"><li>Prepare oligo suspensions of 100 Î¼M and mix 10 Î¼l of each (forward and reverse oligo) with 2.2 Î¼l NEB2 buffer in a 1.5 ml reaction tube.</li><li>Incubate in heat block for 10 min at 95 Â°C. Move heat block on ice and let cool down. (It takes approximately 1 min for the block to cool down.) From now on, always keep the reaction tube on ice.</li><li>Dilute annealed oligos 1:200 to 0.5 Î¼M with cold EB buffer from the QIAGEN kit (1 Î¼l of annealed oligos to 199 Î¼l EB buffer).</li></ol></li><li>Prepare In-Fusion reaction<br/><ol class="npwthree"><li>Mix 1 Î¼l of linearized plasmid (50 ng), 1 Î¼l of diluted annealed oligos, 2 Î¼l H<sub>2</sub>O and 1 Î¼l In-Fusion buffer on ice.</li><li>Incubate for 15 min at 50 Â°C.</li></ol></li><li>Transformation of pCas9/sg_guide<br/><ol class="npwthree"><li>Add 5 Î¼l of In-Fusion reaction to 50 Î¼l of XL10 gold ultracompetent cells or alternative cells, for example Stellar<sup>TM</sup> competent cells (Clontech).</li><li>Incubate on ice for 30 min.</li><li>Apply heat shock according to the manufacturerâs instructions. For XL10 gold cells this is 30 s in a heat bath at 42 Â°C.</li><li>Incubate reaction on ice for 2 min.</li><li>Add 200 Î¼l of LB-media and incubate for 20 min at 37 Â°C with 350 rpm shaking.</li><li>Plate reaction out on agar plates with 0.1 Î¼g/ml ampicillin and incubate overnight at 37 Â°C.<br/></li></ol></li><li>Confirm insertion of N20 guide sequence into pCas9/sg plasmid<br/><ol class="npwthree"><li>Inoculate a colony in 5 ml LB-media with 0.1 Î¼g/ml ampicillin and incubate overnight at 37 Â°C.</li><li>Purify plasmid with Mini-prep kit.</li><li>Sequence at least two plasmids (efficiency typically between 50 and 100%) with oligo 5â²-CATTGTTCCCCCCTTTGTTTTGCAAG-3â² to confirm insertion.</li></ol></li><li>Purify plasmid for parasite transfection<br/><ol class="npwthree"><li>Inoculate 250 ml LB-media with 0.1 Î¼g/ml ampicillin and incubate overnight at 37 Â°C.</li><li>Purify plasmid with Midi-prep kit. The yield should be at least 30 Î¼g in order to have enough plasmid for one parasite transfection.</li><li>Precipitate plasmid by mixing with 0.1 volume of 3 M NaAc and 2 volumes of ice-cold 100% ethanol and incubate for at least 30 min at -20 Â°C.</li><li>Spin down plasmid for 30 min at 4 Â°C and 15,871 rcf. Wash pellet twice with 700 Î¼l 70% ethanol, each time spin down pellet for 15 min at 4 Â°C and 15,871 rcf.</li><li>Dry plasmid pellet in sterile laboratory cabinet and resuspend with 10 Î¼l sterile TE buffer.</li><li>Measure DNA concentration.</li><li>Keep plasmid at 4 Â°C until transfection.</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172420_3157.jpg" width="232"/><br/><strong>Figure 1. In-Fusion cloning of guide sequence into pCas9/sg plasmid.</strong> BtgZI cleaves outside of its recognition site. Two BtgZI sites (green) are used to linearize the plasmid and remove the recognition sites. Oligos for In-Fusion cloning are generated by using 15 bp homology regions of either side of the integration site, flanking the 20 bp upstream of the NGG site within the target gene. Gray lines indicate sequence homology.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Generating PCR template DNA<ol class="npwtwo"><li>Generating oligos<br/><ol class="npwthree"><li>Choose oligos to amplify homology regions (HRs) up- (product 1, oligo5 + oligo2) and downstream (product 2, oligo3 + oligo6) of the guide sequence (double-strand break) of at least 500 bp length each, but ideally 800 bp. If possible, avoid long poly-A or T tracts.</li><li>Add an adaptor sequence of the DNA to be integrated (this could be a tag or a whole expression cassette) to oligo 2 and oligo 3 of ~20 bp (Tm of at least 56 Â°C).</li><li>Choose oligos at least 50 bp upstream of the 5â²HR and 50 bp downstream of the 3â²HR (oligo1 and oligo4), these will allow you to carry out the fusion reaction as a semi-nested PCR, this is critical to ensure minimal background products and sufficient yield.</li><li>Choose oligos to amplify the DNA sequence to be integrated (product 3, oligo7 + oligo8).</li></ol></li><li>Amplify 5â²HR (product 1, oligo1 + oligo2), 3âHR (product 2, oligo3 + oligo4) and DNA sequence to be integrated (product 3, oligo7 + oligo8). Run each PCR in duplicate with a total volume of 25 Î¼l according to the user manual. For CloneAmp Hifi PCR Premix this is 32 cycles of:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td>98 Â°C<br/></td><td>10 s<br/></td><td>Denaturation<br/></td></tr><tr><td>55 Â°C<br/></td><td>15 s<br/></td><td>Annealing<br/></td></tr><tr><td>72 Â°C<br/></td><td>5 s/kb<br/></td><td>Extension<br/></td></tr></tbody></table>Run on agarose gel and gel purify.</li><li>Combine product 1 and product 3 by PCR (oligo5 + oligo8). Allow 15 s/kb during the extension step. Run on agarose gel and gel purify.<br/>Example: for 25 Î¼l PCR use: 12.5 Î¼l CloneAmp Premix, 0.5 Î¼l fwd oligo, 0.5 Î¼l rev oligo, 1 Î¼l product 1, 1 Î¼l product 2 and 10 Î¼l H<sub>2</sub>O.</li><li>Combine product 1/3 and product 2 by PCR (oligo5 + oligo6). Again allow 15 s/kb during the extension step. Run at least 6x 50 Î¼l reactions of this final PCR to yield 40 Î¼g of DNA. Note that whilst it sometimes is possible to simultaneously fuse all three products in a single reaction, splitting this into two steps increases robustness and yield so we recommend this as default.</li><li>Without purification (to avoid loss of DNA) precipitate DNA, wash and dry as described for pCas9/sg plasmid. <strong>A schematic of the 3-step PCR for generating template DNA is shown in Figure 2.</strong><br/><em>Notes:</em><br/><ol class="npwthree"><li><em>If the inserted DNA sequence is a small tag (like hemagglutinin tag) the whole tag sequence can be added as adaptors in oligos 2 and 3 and only two PCR steps are needed to fuse 5â²HR and 3â²HR.</em><br/></li><li><em> If the aim is to knock-out a gene, homology regions targeting the 5â² and 3â² UTRs can simply be fused together (or with a short intervening âbarcodeâ or linker) to create a PCR knockout construct.</em><br/></li><li><em> If there is no PAM site located close to the C- or N-term of the gene of interest, a PAM site further away can be chosen but part of the sequence needs to be recodonized to avoid repair of the locus without modification (see example in Figure 3).</em><br/></li><li><em> If larger volumes of the template DNA is needed, or if the sequence needs to be confirmed by sequencing, the final PCR can also be integrated into a plasmid (<em>e.g.</em>, by TA cloning into pGEM).</em><br/></li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172456_3175.jpg" width="154"/><br/><strong>Figure 2. Schematic of 3-step PCR for generating template DNA.</strong> The genomic sequence of the gene of interest is used to design oligos for amplification of homology regions flanking the sequence to be deleted/replaced (red). Separately, oligos are designed to amplify the sequence to be introduced (green). In three PCR steps the template DNA for homogous repair is produced. In the first PCR step, three products are generated. In the second PCR step, products 1 and 3 are fused and in the third PCR, products 1/3 and 2 are fused. ol = oligo.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172532_7242.jpg" width="127"/><br/><strong>Figure 3. Schematic showing generation of template DNA with recodonization.</strong> In this example the aim is to tag the C-terminal end of a gene of interest (purple), while the chosen PAM site is within the 3â² end of the gene and the 20 bp guide sequence would not be disrupted by introduction of a tag sequence. The PAM site must therefore be deleted by introducing a recodonized sequence (blue) that runs from the PAM site up until the tag sequence (to ensure there is no chance of integration between mutated PAM site and tag). In this case the tag is a short sequence like a hemagglutinin-tag or spot-tag (green). The PCR oligos are designed as described before with the difference that at least two reverse oligos for HR1 are needed. In the first reverse oligo (oligo 2.1), the 5â² end contains the recodonized sequence (blue), the second oligo (oligo 2.2) binds within the recodonized sequence and contains the tag sequence in its 5â² end. Template DNA for the first PCR is genomic DNA, the template for second PCR is product 1.1 and the template for the final PCR are product 1.2 and product 2.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Synchronize <em>P. knowlesi</em> parasites via Nycodenz purification<ol class="npwtwo"><li>Transfer 5 ml of 55% Nycodenz working solution to a 15 ml conical tube and warm up to RT.<br/><em>Note: 1 Nycodenz tube is needed for 1 ml of red blood cells (50 ml culture with 2% hematocrit).</em> </li><li>Centrifuge down high parasitemia (4-10%) <em>P. knowlesi</em> culture at 900 <em>x g</em> for 4 min at RT.</li><li>Resuspend parasite pellet at 50% hematocrit in RPMI.</li><li>Carefully layer 2 ml of this culture above the 5 ml Nycodenz.</li><li>Centrifuge at 900 <em>x g</em> for 12 min with low brake/acceleration. Uninfected red cells and ring-stage parasites will sink to the bottom and schizonts form a layer on top of the Nycodenz.</li><li>Transfer top layer schizonts (brownish color) to a new conical tube and wash with RPMI to remove Nycodenz (see Figure 4).</li><li>Incubate schizonts with 1 Î¼M Compound 2 for 2-3 h. Compound 2 is a PKG inhibitor that reversibly blocks merozoite egress. This step is optional but will help to maximize yield of late schizonts and also provides the user with some flexibility in timing for subsequent steps. Viability of parasites will dramatically decline for incubations longer than 3 h.</li><li>Wash off Compound 2 and transfer schizonts back to culture (with 2% hematocrit red blood cells).<br/><em>Note: To get parasites even more synchronous undertake the steps above and then let them invade red blood cells for 30 to 60 min and Nycodenz purify again, only this time keep ring stage parasite pellet and remove schizonts. You can slow down the maturation of the parasites by leaving them at RT for several h in order to get them to the schizont stage at a convenient time for the next purification/ transfection.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172605_1286.jpg" width="139"/><br/><strong>Figure 4. Schizont enrichment with Nycodenz</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transfect <em>P. knowlesi</em> parasites with CRISPR constructs<ol class="npwtwo"><li>For transfection of the parasites, DNA and blood need to be prepared. This can be done simultaneously.<br/><ol class="npwthree"><li>Blood: Spin down fresh Duffy positive red blood cells (stored for less than 1 week) and transfer 200 Î¼l of packed cells to a new 1.5 ml reaction tube. Add 500 Î¼l of complete media and incubate at 37 Â°C with shaking at 550 rpm (in, for example, an Eppendorf Thermomixer).</li><li>DNA:<ol class="npwfour"><li>Combine ~20 Î¼g of the pCas9/sg_guide plasmid with ~40 to 60 Î¼g of template DNA (~1:20 molar ratio) and adjust to a final volume of 10 Î¼l with sterile TE buffer.</li><li>Mix 10 Î¼l DNA with 100 Î¼l of supplement P3 transfection buffer (from the Lonza kit).</li></ol></li><li>Parasites:<ol class="npwfour"><li>Purify <em>P. knowlesi</em> parasites with Nycodenz from a synchronous late-stage schizont (24-26 h old) culture and incubate for 2 h with 1 Î¼M Compound 2. The best indicator of schizont age in <em>P. knowlesi</em> is the presence of a single compact pigment granule and full segregation of merozoites. The compact pigment granule forms approximately 30-45 min prior to egress. Maximizing the proportion of these stages will drastically improve transfection efficiency.</li><li>After Compound 2 incubation of schizonts, wash and transfer 10 to 20 Î¼l schizonts (â¼1-2 x 10<sup>8</sup> cells) to a new 1.5 ml reaction tube. Leave them with at least 100 Î¼l of complete media at 37 Â°C for 15 to 20 min.</li></ol></li></ol></li><li>Add program to Amaxa 4D-Nucleofector X or choose existing experiment with the following setting: Pulse code: FP 158, Solution: Primary Cell P3, Volume: 100 Î¼l.</li><li>Immediately after the incubation from Step D1c.ii spin down schizonts for 1 min at 845 rcf and remove culture media.</li><li>Transfer 100 Î¼l of the DNA/transfection buffer to the schizont pellet and gently mix.</li><li>Transfer 100 Î¼l of the schizont/buffer mix to a nucleofection cuvette and move to the Amaxa 4D-Nucleofector X. Press start and wait until the screen shows green signal to indicate successful transfection. Loading more than 100 Î¼l of mix to the cuvettes can increase probability of error messages indicating partial (âyellowâ) or complete (âredâ) pulse failure.</li><li>Immediately after transfection, transfer schizonts with pastette (included in Lonza kit) to the prepared warm red blood cells. Rinse out cuvette with blood/media to avoid loss of schizont material.</li><li>Incubate schizonts for 20-30 min at 37 Â°C and shaking at 550 rpm to allow invasion, before moving the culture to a 6-well plate and add 4.5 ml of complete media.<br/><em>Note: In order to determine the transfection efficiency an episomal eGFP plasmid (<em>e.g.</em>, PkconGFP) should be transfected as a control.</em> </li><li>The day after transfection change media daily for 5 days with the addition of positive selection drug Pyrimethamine (100 nM final concentration).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Confirm transfection efficie<ol class="npwtwo"><li>Microscopy (if fluorescent episomal plasmid was transfected)<br/><ol class="npwthree"><li>Around 24 h post-transfection stain 30 Î¼l of culture with 30 Î¼l of 4 Î¼g/ml Hoechst in PBS for 15 min at 37 Â°C.</li><li>Dilute with 100 Î¼l PBS and apply 100 Î¼l of the dilution to a microscopy channel slide (ibidi).</li><li>Count 20 to 50 parasites with defined Hoechst signal and check how many of these also show fluorescence of the episomal plasmid. Transfection efficiencies are usually between 5 and 40%.</li><li>Example fluorescent microscopy images are shown in Figure 5.</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172635_6566.jpg" width="347"/><br/><strong>Figure 5. Microscopy image of eGFP positive parasites<br/></strong><br/></li><li>Diagnostic PCR<br/><ol class="npwthree"><li>Choose the forward oligos at least 100 bp outside the repair template HRs in the genome. Choose one reverse oligos inside the wild-type locus and one reverse oligo inside the modified locus.<br/><em>Note: The product must be unique to integrated parasites and should not also amplify out the transfected template DNA. As large quantities are used in transfections the construct can remain detectable for some time after transfection resulting in a potential false-positive PCR result. Additional control PCRs should be designed which provide a product only in wild type parasites and also in an unrelated control locus (to demonstrate there are no issues with the gDNA).</em> </li><li>Spin down 1 ml of parasite culture, wash blood pellet with RPMI or PBS and store pellet at -20 Â°C or immediately use for gDNA extraction (Dneasy blood and tissue kit).</li><li>Set up six PCR reactions to determine wild-type locus, modified locus and an independent locus of wild-type parasites and transfected parasites. Run on agarose gel as shown in Figure 6.</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172701_4531.jpg" width="141"/><br/><strong>Figure 6. Diagnostic PCRs to confirm integration.</strong> The agarose gel is an example of a diagnostic PCR in order to confirm wild-type locus, integration locus and of an unrelated/independent locus. The wild-type forward oligo should be outside of the repair template (~100 bp upstream of the 5â² homology region or downstream of the 3â² homology region). The reverse oligos should be specific for wild-type or integration. The genomic DNA is from wild-type parasites (WT) before transfection or when parasites reappeared after transfection (TF).<br/><em>Note: Integration can be detectable as early as two days after transfection, but PCR works more reliable with higher DNA concentrations, therefore samples should be taken when drug resistant parasites are easily detectable by Giemsa smear in the cultures (&gt; 0.2% parasitemia).<br/><br/></em> </li><li>Sequencing<br/>Amplify the modified gene locus by PCR and confirm by sequencing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Remove Cas9/sg_guide plasmid containing parasites with negative selection<ol class="npwtwo"><li>Add 1 Î¼M Ancotil (5-Fluorocytosine) for 7 days to parasite culture of ~0.5% (parasite should be confirmed integration positive and are stably growing in culture).<br/><em>Note: Without positive selection, parasites lose plasmid very quickly, therefore no big drop in parasitemia is expected.</em> </li><li>Clone out parasites by limiting dilution (Moon <em>et al.</em>, 2013).</li><li>Confirm integration in several clonal lines by PCR or microscopy as described above.</li><li>Confirm removal of Cas9/sg_guide plasmid in integration positive clonal lines by treating parasites for 5 days with Pyrimethamine as described before. Keep culture for another week to test if parasites reappear.<br/><em>Note: Prepare cryo stocks of transfected parasites (bulk culture) and clonal lines.</em> </li></ol></li></ol><p class="pptt" id="biaoti31427">### Notes</p><p class="ppzz">All parasite work should be carried out in a class II microbiological safety hood for both sterility and user protection. As <em>P. knowlesi</em> is a human infective species all work must be carried out under appropriate approved safety and containment conditions. <em>P. knowlesi</em> cultures can be grown in the presence of gentamycin (25 mg/L) antibiotics, however with good sterile technique this should not be necessary, nor is it recommended.<br/>  The transfection and integration efficiencies can be variable if not fully optimized. High efficiencies can be achieved by:</p><ol class="npwtwo"><li>Using fresh blood (7 days old or younger).</li><li>Making sure you have at least 10 Î¼l of schizonts, lower amounts will disproportionately affect efficiency.</li><li>Ensuring that you are using very late and highly synchronous schizonts (with a large proportion fully segmented with a condensed pigment granule).</li><li>Ensuring that your <em>P. knowlesi</em> parasites are growing reliably 3-4 fold per day (for human RBCs) prior to transfection.</li><li>Ensuring you have sufficient DNA.</li><li>Using long homology regions (500-1,000 bp).</li><li>Comparing several guide sequences.</li></ol><p class="pptt" id="biaoti31428">### Recipes</p><ol class="npwtwok"><li>Nycodenz stock solution (100%)<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:990px;"><tbody><tr><td><strong>Compound </strong><br/></td><td><strong>Final concentration </strong><br/></td><td><strong>For 100 ml </strong><br/></td><td><strong>Notes</strong><br/></td></tr><tr><td>Nycodenz<br/></td><td>27.6% (w/v)<br/></td><td>27.6 g<br/></td><td>Add to 50 ml warm H<sub style="bottom:-0.25em;box-sizing:border-box;color:#333333;font-family:&amp;font-size:75%;font-style:normal;font-variant:normal;font-weight:400;letter-spacing:normal;line-height:0;orphans:2;position:relative;text-align:left;text-decoration:none;text-indent:0px;text-transform:none;vertical-align:baseline;-webkit-text-stroke-width:0px;white-space:normal;word-spacing:0px;">2</sub>O and dissolve by heating (60 Â°C) and stirring<br/></td></tr><tr><td>Tris HCl<br/></td><td>5 mM<br/></td><td>2.5 ml of 1 M stock<br/></td><td><br/></td></tr><tr><td>KCl<br/></td><td>3 mM<br/></td><td>22.37 mg<br/></td><td><br/></td></tr><tr><td>CaNa<sub style="bottom:-0.25em;box-sizing:border-box;color:#333333;font-family:&amp;font-size:75%;font-style:normal;font-variant:normal;font-weight:400;letter-spacing:normal;line-height:0;orphans:2;position:relative;text-align:left;text-decoration:none;text-indent:0px;text-transform:none;vertical-align:baseline;-webkit-text-stroke-width:0px;white-space:normal;word-spacing:0px;">2</sub>Â·EDTA<br/></td><td>0.3 mM<br/></td><td>11.23 mg</td></tr></tbody></table>Adjust to 100 ml and pH 7.5, sterile filer and store at 4 Â°C</li><li>Nycodenz working solution (55%)<br/>Mix 55 ml Nycodenz stock solution with 45 ml RPMI. Store at 4 Â°C</li><li>Pyrimethamine<ol class="npwthree"><li>Prepare a 10 mM stock solution in DMSO, prepare 60 Î¼l aliquots and store at -20 Â°C</li><li>Prepare a 100 Î¼M working solution by adding 50 Î¼l of the stock solution to 50 ml of RPMI</li><li>Sterile filter and store at 4 Â°C. Donât keep the working solution for more than 2 weeks</li><li>Add 5 Î¼l of working solution to 5 ml of parasite culture (100 nM final concentration in culture)</li></ol></li><li>Ancotil<ol class="npwthree"><li>Prepare a 1 mM working solution by diluting the sterile stock solution (2.5 g/250 ml = 77 mM = 1%) 1/77 in RPMI</li><li>Keep at room temperature</li><li>Add 5 Î¼l to 5 ml parasite culture (1 Î¼M)</li></ol></li><li>Complete media<br/>RPMI-1640 (HEPES Modification, With 25 mM HEPES, without L-glutamine) with the following additions:<br/>2.3 g/L sodium bicarbonate<br/>2 g/L dextrose<br/>0.05 g/L Hypoxanthine<br/>5 g /L Albumax II<br/>0.3 g/L L-glutamine (10 ml of 200 mM solution pro 1 L media)<br/>10% (vol/vol) horse serum<br/>Sterile filter and store at 4 Â°C<br/></li></ol><p class="pptt" id="biaoti31429">### Acknowledgments</p><p class="ppzz">This work was supported by an MRC Career Development Award (MR/M021157/1) jointly funded by the UK Medical Research Council and Department for International Development (R.W.M, F.M), by a Bloomsbury Colleges research studentship and a Wellcome Trust Senior Investigator Award (106240/Z/14/Z). The protocol is adapted from Ecker <em>et al.</em> (2006), Ghorbal <em>et al.</em> (2014) and Mohring <em>et al.</em> (2019).</p><p class="pptt" id="biaoti31430">### Competing interests</p><p class="ppzz">The authors declare no competing financial interests.</p><p class="pptt" id="biaoti31431">### Ethics</p><p class="ppzz">The project, consent and protocol were approved by the LSH<sup>TM</sup> Observational Research Ethics Committee under project reference 5520-1.</p><p class="pptt" id="biaoti31432">### References</p><ol class="npwtwok"><li>Cooper, D. J., Rajahram, G. S., William, T., Jelip, J., Mohammad, R., Benedict, J., Alaza, D. A., Malacova, E., Yeo, T. W., Grigg, M. J., Anstey, N. M. and Barber, B. E. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30889244" target="_blank"><em>Plasmodium knowlesi</em> malaria in Sabah, Malaysia, 2015-2017: ongoing increase in incidence despite near-elimination of the human-only <em>Plasmodium</em> species.</a> <em>Clin Infect Dis</em> 70(3):361-367.</li><li>Ecker, A., Moon, R., Sinden, R. E. and Billker, O. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16290088" target="_blank">Generation of gene targeting constructs for <em>Plasmodium berghei</em> by a PCR-based method amenable to high throughput applications.</a> <em>Mol Biochem Parasitol</em> 145(2): 265-268.</li><li>Ghorbal, M., Gorman, M., Macpherson, C. R., Martins, R. M., Scherf, A. and Lopez-Rubio, J. J. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24880488" target="_blank">Genome editing in the human malaria parasite <em>Plasmodium falciparum</em> using the CRISPR-Cas9 system.</a> <em>Nat Biotechnol</em> 32(8): 819-821.</li><li>Gruring, C., Moon, R. W., Lim, C., Holder, A. A., Blackman, M. J. and Duraisingh, M. T. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24506567" target="_blank">Human red blood cell-adapted <em>Plasmodium knowlesi</em> parasites: a new model system for malaria research.</a> <em>Cell Microbiol</em> 16(5): 612-620.</li><li>Knuepfer, E., Wright, K. E., Prajapati, S. K., Rawlinson, T. A., Mohring, F., Koch, M., Lyth, O. R., Howell, S. A., Villasis, E., Snijders, A. P., Moon, R. W., Draper, S. J., Rosanas-Urgell, A., Higgins, M. K., Baum, J. and Holder, A. A. (2019).<a href="https://www.ncbi.nlm.nih.gov/pubmed/31185066" target="_blank"> Divergent roles for the RH5 complex components, CyRPA and RIPR in human-infective malaria parasites.</a> <em>Plos Pathog</em> 15(6).</li><li>Lim, C., Hansen, E., DeSimone, T. M., Moreno, Y., Junker, K., Bei, A., Brugnara, C., Buckee, C. O. and Duraisingh, M. T. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23535659" target="_blank">Expansion of host cellular niche can drive adaptation of a zoonotic malaria parasite to humans.</a> <em>Nat Commun</em> 4: 1638.</li><li>Lubis, I. N. D., Wijaya, H., Lubis, M., Lubis, C. P., Divis, P. C. S., Beshir, K. B. and Sutherland, C. J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28201638" target="_blank">Contribution of <em>Plasmodium knowlesi</em> to multispecies human malaria infections in north Sumatera, Indonesia.</a> <em>J Infe</em><em>ct Dis</em> 215(7): 1148-1155.</li><li>Mohring, F., Hart, M. N., Rawlinson, T. A., Henrici, R., Charleston, J. A., Benavente, E. D., Patel, A., Hall, J., Almond, N., Campino, S., Clark, T. G., Sutherland, C. J., Baker, D. A., Draper, S. J. and Moon, R. W. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31205002" target="_blank">Rapid and iterative genome editing in the malaria parasite <em>Plasmodium knowlesi</em> provides new tools for <em>P. vivax</em> research.</a> <em>Elife</em> 8: e45829.</li><li>Moon, R. W., Hall, J., Rangkuti, F., Ho, Y. S., Almond, N., Mitchell, G. H., Pain, A., Holder, A. A. and Blackman, M. J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23267069" target="_blank">Adaptation of the genetically tractable malaria pathogen <em>Plasmodium knowlesi</em> to continuous culture in human erythrocytes.</a><em> Proc Natl Acad Sci U S A</em> 110(2): 531-536.</li><li>Rajahram, G. S., Cooper, D. J., William, T., Grigg, M. J., Anstey, N. M. and Barber, B. E. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30624597" target="_blank">Deaths from <em>Plasmodium knowlesi</em> malaria: case series and systematic review.</a> <em>Clin Infect Dis</em> 69(10):1703-1711.</li><li>Rawlinson, T. A., Barber, N. M., Mohring, F., Cho, J. S., Kosaisavee, V., Gerard, S. F., Alanine, D. G. W., Labbe, G. M., Elias, S. C., Silk, S. E., Quinkert, D., Jin, J., Marshall, J. M., Payne, R. O., Minassian, A. M., Russell, B., Renia, L., Nosten, F. H., Moon, R. W., Higgins, M. K. and Draper, S. J. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31133755" target="_blank">Structural basis for inhibition of <em>Plasmodium vivax</em> invasion by a broadly neutralizing vaccine-induced human antibody.</a> <em>Nat microbial</em> 4(9):1497-1507.</li><li>WHO. (2018). <em><a href="https://www.who.int/malaria/publications/world-malaria-report-2018/en/" target="_blank">World Malaria Report.</a></em> World Health Organization.<br/></li></ol>