<body>
            <h1>Photoactivable Cholesterol as a Tool to Study Interaction of Influenza Virus Hemagglutinin with Cholesterol</h1>
            <p><strong>Authors:</strong> Bodan   Hu, Mohamed Rasheed Gadalla, Christoph  Thiele, Michael  Veit</p>
            <h2>Abstract</h2>
            <p>Non-covalent binding of cholesterol to the transmembrane region of proteins affect their functionalities, but methods to prove such an interaction are rare. We describe our protocol to label the hemagglutinin (HA) of Influenza virus with a cholesterol derivative in living cells or with immunoprecipitated protein. We synthesized a âclickableâ photocholesterol compound, which closely mimics authentic cholesterol. It contains a reactive diazirine group that can be activated by UV-illumination to form a covalent bond with amino acids in its vicinity. Incorporation of photocholesterol into HA is then visualized by âclickingâ it to a fluorophore, which can be detected in an SDS-gel by fluorescence scanning. This method provides a convenient and practical way to demonstrate cholesterol-binding to other proteins and probably to identify the binding site.<br/></p>
            <p><strong>Keywords:</strong> Influenza virus, Hemagglutinin, Cholesterol, Lipid-interaction, Membrane, Photocholesterol, Click-chemistry</p>
            <h2>Background</h2><p>Non-covalent interactions of proteins with cholesterol are supposed to regulate trafficking and functionalities of many proteins (de Vries <em>et al.</em>, 2015). However, due to the transient and rather weak nature of this interaction, cholesterol-binding proteins and the respective binding sites are notoriously difficult to identify. Highly sophisticated methods, such as NMR or crystallography require large amounts of purified proteins which need to be integrated into artificial lipid membranes and are thus accessible only to specialized labs. A simple procedure involves measuring the cholesterol content in purified proteins using commercial âkitsâ (<em>e.g.</em>, Amplex red cholesterol assay kit, Molecular Probes). However, this method is insensitive and requires solubilization of proteins from membranes with detergent which often removes non-covalently bound lipids.<br/><br/>One improvement is the synthesis of clickableâphotocholesterol compounds which can be covalently linked to a protein. They can be added to cells where they are rapidly integrated into membranes and thus can interact with proteins in their native environment. They contain a diazirine group at position 6 of the sterol ring, which disintegrates upon uv-illumination into molecular nitrogen plus a highly reactive carbene-group that forms a covalent bond with amino acid side chains in close vicinity. To detect cross-linked proteins, probes have a latent affinity handle, an alkyne group for chemical conjugation under physiological conditions to azide-reporters by copper-catalyzed azide-alkyne cycloaddition (âclick chemistryâ). Reporters are either present on âbeadsâ to enrich and identify probe-interacting proteins by mass spectrometry or are fluorophores which allow their in-gel detection.<br/><br/>We synthesized a compound, 6,6'-Azi-25-ethinylcholesterol [termed improved photoclick-cholesterol, complete synthesis is described in Hu <em>et al.</em> (2019)], which is more similar to genuine cholesterol than a commercially available photoclick-cholesterol (Hex-5'-ynyl 3Ã-hydroxy-6-diazirinyl-5Î±-cholan- 24-oate, Avanti Polar lipids, number 700147) since it contains (besides the alkyne group) no further alterations in cholesterolâs alkyl side chain (Figure 1). A previous study showed that 6-photocholesterol is a faithful mimetic of authentic cholesterol (Mintzer <em>et al.</em>, 2002). Note, however, that some of the diazirine groups might be photoactivated to other reactive species that have a longer half time than the carbene-group which might unspecifically label proteins. Thus, photocrosslinking is a qualitative rather than a quantitative measure of the cholesterol affinity of a protein. Nevertheless, both cholesterol probes label only a few specific proteins out of all cellular membrane proteins (Thiele <em>et al.</em>, 2000; Hulce <em>et al.</em>, 2013), indicating that they are suitable tools for analyzing possible interactions between cholesterol and target proteins.</p>
        </body>

<br/><p class="ppzz"><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172738_5288.jpg" width="226"/><br/><strong>Figure 1. The structure of cholesterol and of two photoactivatable derivatives.</strong> A. Cholesterol. B. Improved Photoclick-cholesterol (6,6'-Azi-25-ethinyl-cholesterol) used in this protocol. C. Photoclick-cholesterol (Hex-5'-ynyl 3Ã-hydroxy-6-diazirinyl-5Î±-cholan-24-oate) commercially available from Avanti lipids. The photolabile azide-group is highlighted blue and the alkyne-group in red in B.</p><p class="pptt" id="biaoti31445">## Materials and Reagents</p><ol class="npwone"><li>Cell culture consumables<ol class="npwtwo"><li>6-well/24-well tissue culture plates, flasks, pipettes (Sarstedt, Germany)</li><li>Pipette tips (Sarstedt, Germany)</li><li>Aluminum foil</li><li>CHO-K1 (Chinese hamster ovary cells) (ATCC, catalog number: CCL-61)</li><li>Plasmid (pCAGGS, Niwa <em>et al.</em>, 1991)</li><li>DMEM (PAN Biotech, catalog number: P04-04500)</li><li>EDTA-Trypsin (PAN Biotech, catalog number: P10-023100)</li><li>DPBS w/o calcium/magnesium (PAN Biotech, catalog number: P04-36500)</li><li>Penicillin/Streptomycin (10,000 U/ml) (PAN Biotech, catalog number: P06-07100)</li><li>FBS (fetal bovine serum) (PAN Biotech, catalog number: P30-3306)</li><li>Opti-MEM<sup>TM</sup> I Reduced Serum Medium (Thermo Fisher, Gibco<sup>TM</sup>, catalog number: 31985070)</li><li>TurboFect Transfection Reagent (Thermo Fisher, Thermo Scientific<sup>TM</sup>, catalog number: R0531)</li><li>Growth medium (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Immunoprecipitation<ol class="npwtwo"><li>Ethanol (Sigma, CAS number: 64-17-5, catalog number: 34852-1L-M)</li><li>Tris-HCl (TRIS hydrochloride) (Carl Roth, CAS number: 1185-53-1, catalog number: 9090.3)</li><li>EDTA (AppliChem, CAS number: 6381-92-6, catalog number: A1104)</li><li>Sodium Pyrophosphate (Fisher Scientific, Alfa Aesar, CAS number: 7722-88-5, catalog number: A17546.30)</li><li>Sodium Fluoride (Fisher Scientific, Alfa Aesar, CAS number: 7681-49-4, catalog number: A13019.30)</li><li>Sodium Orthovanadate (Sigma, CAS number: 13721-39-6, catalog number: S6508-10G)</li><li>Benzamidine Hydrochloride (AppliChem, CAS number: 1670-14-0, catalog number: A1380)</li><li>PMSF (AppliChem, CAS number: 329-98-6, catalog number: A0999)</li><li>NEM (N-Ethylmaleimide) (Sigma, CAS number: 128-53-0, catalog number: E3876-5G)</li><li>cOmplete<sup>TM</sup>, EDTA-free Protease Inhibitor Cocktail (Sigma, Roche, catalog number: 11873580001)</li><li>NP-40 (Thermo Fisher, Thermo Scientific<sup>TM</sup>, catalog number: 85124)</li><li>Protein-G-Sepharose 4 Fast Flow (Sigma, GE Healthcare, GE17-0618-01)</li><li>Anti-HA<sub>2</sub> antiserum (Hu <em>et al.</em>, 2019)</li><li>IP buffer (see Recipes)</li><li>Cell lysis buffer (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>SDS-PAGE and Western blot<ol class="npwtwo"><li>PVDF blotting-membrane, 0.2 Âµm (VWR, Peqlab, catalog number: 732-3200)</li><li>1.5 ml microcentrifuge tubes (Sarstedt, Germany)</li><li>30% acrylamide/ bisacrylamide (37.5:1) (Carl Roth, catalog number: 3029.1)</li><li>Tetramethylethylenediamine (TEMED) (Carl Roth, CAS number: 110-18-9, catalog number: 2367.3)</li><li>Ammonium persulfate (APS) (Carl Roth, CAS number: 7727-54-0, catalog number: 9592.2)</li><li>Sodium dodecyl sulfate (SDS) (Carl Roth, CAS number: 151-21-3, catalog number: 2326.4)</li><li>Bromophenol bule (Sigma, CAS number: 115-39-9, catalog number: 1081220005)</li><li>1,4-Dithiothreitol (DTT) (Carl Roth, CAS number: 3483-12-3, catalog number: 6908.2)</li><li>Glycerol (Carl Roth, CAS number: 56-81-5, catalog number: 3783.1)</li><li>Skimmed milk powder (Carl Roth, CAS number: 68514-61-4, catalog number: T145.3)</li><li>Tween 20 (Carl Roth, CAS number: 9005-64-5, catalog number: 9127.1)</li><li>Pierce<sup>TM</sup> ECL Plus Western Blotting Substrate (Thermo Fisher, catalog number: 32132)</li><li>Stacking-gel solution (see Recipes)</li><li>Separating-gel solution (see Recipes)</li><li>5x Non-reducing loading buffer (see Recipes)</li><li>4x Reducing loading buffer (see Recipes)</li><li>Blocking buffer (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Crosslinking and click chemistry<br/><ol class="npwtwo"><li>Blacklight Blue (UV) Lamp (Sankyo Denki, power 8W, 3.5 A, 60V, wavelength 320-365 nm, Figure 3A)</li><li>Improved Photoclick-cholesterol (Figure 1B, dissolved in ethanol, the tube is sealed and stored at -20 Â°C). The long synthesis is described in all details in Hu <em>et al.</em> (2019). But an intermediate alkyne-cholesterol can be purchased from Click Chemistry Tools (<a href="https://clickchemistrytools.com/product/alkyne-cholesterol/" target="_blank">https://clickchemistrytools.com/product/alkyne-cholesterol/</a>). With this compound it might be easier to synthesize the final product, the improved Photoclick-cholesterol.</li><li>Picolyl-Azide-Sulfo-Cy3 (Jena Bioscience, catalog number: CLK-1178-1)</li><li>CuAAC Biomolecule Reaction Buffer Kit (THPTA-based) (Jena Bioscience, catalog number: CLK-1178-1)<br/></li></ol></li></ol><p class="pptt" id="biaoti31446">### Equipment</p><ol class="npwtwok"><li>-20 Â°C freezer</li><li>Pipette controller (Hirschmann LaborgerÃ¤te)</li><li>Heracell 240i CO<sub>2</sub> incubator (Heraeus)</li><li>Tabletop centrifuge (Eppendorf, model: 5424R)</li><li>Power pack p25 (Analytik Jena, Biometra P25T)</li><li>Chemiluminescence Imaging, Fusion SL (PeqLab)</li><li>PerfectBlue<sup>TM</sup> Semi-Dry Electro Blotting Systems (VWR, PeqLab, model: Sedec<sup>TM</sup> M)</li><li>Tube-rotator (Carl Roth, ThermoFisher, model: ATX1.1)</li><li><a class="blackNoUnder" target="_blank"> Typhoon FLA 9400 scanner </a> (GE Healthcare) (Figure 2)<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172803_0787.jpg" width="133"/><br/><strong>Figure 2. Typhoon FLA 9400 scanner.</strong> The gel from Part I Section D, which contains the cross-linked samples and washed with ddH<sub>2</sub>O, is inserted between two glass plates for scanning with a laser. For results, see Figures 4A and 4B.<br/></li></ol><p class="pptt" id="biaoti31447">### Software</p><ol class="npwtwok"><li>Typhoon Scanner Control v5.0 Software (GE Healthcare)</li><li>ImageQuant (GE Healthcare)</li><li>ImageJ (Fiji)</li><li>Microsoft Excel</li><li>GraphPad Prism<br/></li></ol><p class="pptt" id="biaoti31448">### Procedure</p><p class="ppzzs"><strong><span style="font-size:14px;">Part I: Photocholesterol labeling of HA in living cells</span><br/><br/></strong> </p><ol class="npwone"><li>HA expression in CHO cells by transfection<ol class="npwtwo"><li>Seed 0.9 x 10<sup>6</sup>-1.1 x 10<sup>6</sup> CHO cells into one well of a 6-well plate one day before transfection such that they reach 70-90% confluency the next day.</li><li>Dilute 3 Âµg plasmid (pCAGGS) encoding HA in 300 Âµl serum-free Opti-MEM and mix by pipetting.</li><li>Briefly vortex TurboFect reagent and add 6 Âµl to the diluted DNA. Mix immediately by vortexing.</li><li>Incubate 15-20 min at room temperature.</li><li>During the incubation time, remove the old cell medium and replace by 1 ml serum-free DMEM.</li><li>Add the DNA/TurboFect reagent mixture dropwise to the cell medium.</li><li>Gently rock the plate to achieve an even distribution of the DNA/TurboFect complexes.</li><li>Incubate at 37 Â°C in a CO<sub>2</sub> incubator.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Photocholesterol labeling of HA inside cells and crosslinking    <ol class="npwtwo"><li>At 6 h post transfection, replace the cell medium with 1 ml fresh serum-free DMEM to remove the transfection reagent.</li><li>Add 5 Î¼l photocholesterol (from a 5 mg/ml stock in ethanol, which is stored at -20 Â°C, ï¬nal concentration: 50 Î¼M) to each well and gently rock the plate to achieve an even distribution.</li><li>Incubate at 37 Â°C in a CO<sub>2</sub> incubator until the next step.</li><li>At 24 h post transfection, replace the cell medium with 1 ml fresh serum-free DMEM to remove dead cells floating in the culture supernatant.</li><li>Put the plate in the cold room (~8 Â°C, Figure 3B) or on ice as long as the âUV lamp/6-well plate/ice boxâ can be protected from light. Cold temperature slows down the mobility of lipids within the membrane and prevents conformational changes of proteins, and is thus better suited for subsequent crosslinking.</li><li>Take off the lid from the plate and from the UV lamp since plastic will absorb UV-light (Figure 3C).</li><li>Put the UV lamp directly on top of the 6-well plate, turn on the UV lamp (wavelength 320-365 nm, power 8 W, 3.5 A, 60 V) to expose cells to UV light for 10-20 min in the dark to activate the diazirine group to trigger crosslinking (Figure 3D). The cells are still attached to the 6-well culture plate and are covered by 1 ml serum-free DMEM at this stage. Wear protective googles during illumination.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172838_5247.jpg" width="100"/><br/><strong>Figure 3. Crosslinking with photocholesterol.</strong> A. UV lamp (wavelength 320-365 nm, power 8 W, 3.5 A, 60 V). B. Cold room (Tm: ~8 Â°C). C. Preparation before UV illumination by taking off the lid from the tissue culture plate and from the UV lamp. D. UV illumination of the culture plates in the dark.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Immunoprecipitation of HA (buffers should be cold and procedures are performed at 4 Â°C)<ol class="npwtwo"><li>After exposure to UV light cells are still attached to the 6-well culture plate and are covered in DMEM. Put the 6-well plate on ice and detach the cells by scraping without changing medium.</li><li>Transfer cells to 1.5 ml microcentrifuge tubes and centrifuge at 1,200 <em>x g</em> for 5 min at 4 Â°C.</li><li>Wash the cell pellet once with 1 ml PBS and resuspend the cells in 500 Î¼l cell lysis buffer.</li><li>Lyse the cells for 30 min at 4 Â°C and centrifuge at maximal speed (13,500 rpm/17,000 <em>x g</em> ) for 15 min at 4 Â°C to remove insoluble material.</li><li>Transfer the supernatant to a new 1.5 ml microcentrifuge tube without disturbing the pellet.</li><li>Remove 50 Î¼l from the lysate for analysis of protein expression by Western blot. Add 12 Âµl 5x non-reducing loading buffer to this 50 Î¼l lysate, incubate at 95 Â°C for 5 min and store at -20 Â°C.</li><li>Add anti-HA<sub>2</sub> antiserum (1:1,000) to the remaining 450 Î¼l lysate and incubate at 4 Â°C with agitation overnight (12-16 h) in the cold room.</li><li>Prepare working solution of protein-G-sepharose beads according to the manufacturerâs instruction.<br/>a.The stock of protein-G-sepharose is stored in ethanol. Thus, centrifuge the stock solution at 12,000 <em>x g</em> for 30 s to remove the ethanol.<br/>b.Wash the protein-G-sepharose beads once with PBS by adding the same volume of PBS to the beads and centrifuge at 12,000 <em>x g</em> for 30 s.<br/>c.Resuspend the protein-G-sepharose beads in PBS by adding the same volume of PBS (1:1 dilution).</li><li>Add 50 Î¼l of the protein-G-sepharose beads to each sample and incubate at 4 Â°C for another 4 h in the cold room with agitation.</li><li>Centrifuge at 12,000 <em>x g</em> for 30 s to pellet the HA/antibody/protein-G-sepharose complex.</li><li>Remove the supernatant and wash the complex twice with 1 ml IP-buffer and then twice with 1 ml PBS to exclude the possibility that chemicals from the IP-buffer affect the click chemistry reaction in the next step (described in Part I Section D). For the same reason, remove the liquid as completely as possible after the last washing with PBS.</li><li>Proceed to the click chemistry reaction (described in Part I Section D) as soon as possible. Storing the HA/antibody/protein-G-sepharose complex on ice for 1 h is fine, but storing for longer time (<em>e.g.</em>, overnight at -20 Â°C) is not recommended.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>(Cu(I))-catalyzed Azide-Alkyne Click chemistry reactions (CuAAC) according to the manufacturerâs instruction<br/><ol class="npwtwo"><li>Prepare stock solutions in ddH<sub>2</sub>O: 250 mM THPTA, 100 mM CuSO<sub>4</sub>, 1 M Na-Ascorbate. The efficiency of the reaction is higher when all the reagents are freshly prepared.</li><li>Prepare stock solution of Picolyl-Azide-Sulfo-Cy3 (2 mM, dissolved in H<sub>2</sub>O, red color).</li><li>Prepare CuSO<sub>4</sub>:THPTA-premix <strong>freshly</strong> (molar ratio of 1:5) for each experiment. 6 Âµl (2 Âµl CuSO<sub>4</sub> + 4 Âµl THPTA) mixture is used here for each reaction. (The ratio of CuSO<sub>4</sub>:THPTA is recommended as 1:5 in the manufacturerâs instruction, but could be changed.)</li><li>Resuspend HA/antibody/protein-G-sepharose complex in 164 Âµl of H<sub>2</sub>O.</li><li>Add 10 Âµl of 2 mM Picolyl-Azide-Sulfo-Cy3 to the mixture and mix by pipetting.<br/><em>Note: The amount of Azide-Cy3 added here is adjusted based on the amount used in labeling of immunoprecipitated HA. The same final concentration of photocholesterol (50 ÂµM) is used for HA labeling. The molar ratio of alkyne:azide in click chemistry reaction is 1:4, which could be adjusted to get a better result. A higher ratio would give a stronger fluorescence signal. A ratio of 1:10 is recommended in the manufacturerâs instruction.</em> </li><li>Add 6 Âµl CuSO<sub>4</sub>:THPTA-premix to the mixture and mix by pipetting.</li><li>Add 20 Âµl Na-Ascorbate to the mixture, such that the final volume of each reaction is 200 Âµl. Mix by pipetting.</li><li>Incubate samples at room temperature for 1 h with gentle mixing (avoid vortexing) every 10-15 min. Protect samples from light by wrapping the tubes in aluminum foil.</li><li>Wash samples with 1 ml PBS (at least 4 times) until no red color from Azide-Cy3 is seen in the supernatant. However, extensive washing of the beads is not recommended since the amount of beads becomes slightly reduced after each washing step.</li><li>Resuspend samples in 40 Âµl 1x reducing loading buffer (diluted from 4x reducing loading buffer in ddH<sub>2</sub>O) and incubate at 95 Â°C for 5 min.</li><li>Run SDS-PAGE.</li><li>Wash the gel with ddH<sub>2</sub>O for at least 1 h with periodic changes (<em>e.g.</em>, 10 min each). Sometimes overnight (&gt; 12 h) washing is required to reduce the background signal from uncoupled Picolyl-Azide-Sulfo-Cy3.</li><li>Scan the gel with a biomolecular imager (Typhoon FLA 9400 scanner, Excitation = 550 nm, Emission = 560 nm for Cy3).<br/></li></ol><p class="ppzz"><em>Note: The order of Steps D4-D7 should not be changed.</em> </p></li></ol><br/><p class="ppzzs"><strong><span style="font-size:14px;">Part II: Photocholesterol labeling of immunoprecipitated HA</span></strong> </p><br/><p class="ppzz">Transfect CHO cells to express HA as described in Section A of Part I. 24 h after transfection, immunoprecipitate HA with anti-HA<sub>2</sub> antiserum as described in Section C of Part I, and then proceed to crosslinking.<br/>Crosslinking between immunoprecipitated HA and photocholesterol:</p><ol class="npwtwo"><li>Resuspend the washed HA/antibody/protein-A-sepharose complex in 100 Âµl PBS.</li><li>Add 0.5 Âµl photocholesterol (from a 5 mg/ml stock dissolved in ethanol, ï¬nal concentration: 50 ÂµM), and mix by pipetting. Transfer the solution into one well of a 24-well plate such that a thin liquid layer covering the bottom of the well is formed.</li><li>Expose samples to UV light (see Figure 3).</li><li>Transfer the labeled protein to a microcentrifuge tube and wash the mixture twice with 1 ml IP buffer and then twice with 1 ml PBS.</li><li>Proceed to click chemistry reaction as described in Section D of Part I.<br/><br/></li></ol><p class="ppzzs"><strong><span style="font-size:14px;">Part III: Western blot to determine the expression level of HA</span><br/><br/></strong> </p><ol class="npwtwo"><li>Prepare gel for SDS-PAGE. The concentration of acrylamide in the stacking gel is 4%, while that in the separation gel is 12%.</li><li>Run SDS-PAGE at constant voltage. Use low voltage (80-100 V) while proteins are in the stacking gel and high voltage (160 V) for the separation gel.</li><li>Transfer the proteins from the SDS-gel onto a polyvinylidene difluoride (PVDF) membrane at 200-220 mA for 1 h using a semi-dry transfer system.<br/><em>Note: The time could be adjusted based on the molecular weight of the protein under study. Larger proteins require longer transfer time.</em> </li><li>Incubate the membrane with blocking solution for 30 min at room temperature.</li><li>Incubate the membrane with primary antibody (anti-HA<sub>2</sub> antiserum, diluted 1:1,000 in blocking solution) for 2 h at room temperature or overnight (12-16 h) at 4 Â°C.</li><li>Wash the membrane 4 times (5 min each) with PBST (see Recipes) on an orbital shaker.</li><li>Incubate the membrane with secondary antibody coupled to horseradish peroxidase (1:5,000 dilution in blocking solution) for 1 h at room temperature.</li><li>Repeat Step 6 in Part III.</li><li>Apply the ECL plus reagent to the membrane according to manufacturerâs instruction, which reacts with horseradish peroxidase to give the chemiluminescent signals.</li><li>Detect the protein with a Fusion SL camera system.</li></ol><p class="pptt" id="biaoti31449">### Data analysis</p><ol class="npwtwok"><li>For quantification of band intensities, Western blot and the ï¬uorogram were analyzed with ImageJ (Fiji) software. The ratios of photo-cross-linking (ï¬uorogram) to protein expression (Western blot) were calculated for a HA-mutant lacking the cholesterol binding site, and results were normalized to values of HA wild type (HA-wt, set to 100%).</li><li>For detailed instruction of using ImageJ please go to <a href="https://imagej.net/ImageJ" target="_blank">https://imagej.net/ImageJ</a>. Here we briefly describe the process of quantification (see Figure 4C). 1) Make sure that the bands are horizontally aligned. 2) Subtract the background signal from cells transfected with empty vector. 3) Select the bands [the left panel in Figure 4C(b) and second panel in Figure 4C(c)]. 4) Plot the band, which gives histograms indicating the intensity of each bands [the middle panel in Figure 4C(b) and the third panel in Figure 4C(c)]. 5) Draw lines to seal each histogram. 6) Select the âmagic wandâ button and click inside each histogram, which will give a value indicating the intensity of the analyzed band in a new âResultâ window [the last panel in Figure 4C(b) and Figure 4C(c)]. Values are exported as csv. file and analyzed by Microsoft Excel (see Table 1). Cholesterol incorporation into the HA-mutant lacking the cholesterol-binding site is reduced to 34% relative to HA-wt.<br/><br/><img alt="" class="layerphoto" height="123" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218172924_2046.jpg" width="100"/><br/><strong>Figure 4. Image processing. </strong>A-B. Increasing the voltage from 600 (A) to 800 (B) of the scanner increases the contrast. The molecular weight is shown on the left. The next three lanes (L1, L2 and L3) are samples from cells transfected with empty vector (mock), plasmid expressing a HA mutant lacking the cholesterol binding site (HA-mutant), and plasmid expressing the wild-type HA (HA-wt), respectively. The scan can be exported as a tif-file and evaluated with ImageJ. C. Images processed by ImageJ (Fiji). (a) The working panel of ImageJ (Fiji). (b-c) Quantification of the band intensity of HA-mutant and HA-wt from fluorescence scanner (b) and Western blot (c). The density of bands in the yellow boxes is determined and the area under the curves is calculated for each peak. The results obtained from cells transfected with empty vector (mock) is subtracted and the relative incorporation of photo-cholesterol into the HA-mutant relative to HA-wt is calculated (Table 1).<br/>â<br/><strong>Table 1. Data processing to compare photocholesterol incorporation into HA-mutant with HA-wt</strong> (an example)<br/><img alt="" class="layerphoto" height="214" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218173249_0031.jpg" width="500"/><br/></li></ol><p class="pptt" id="biaoti31450">### Notes</p><ol class="npwtwok"><li>Since the cholesterol concentration in membranes is very high (up to 50% in the plasma membrane) and since both lipids and proteins are mobile within the membrane, every transmembrane region is likely to have transient contact with cholesterol. Thus, in principle every transmembrane protein can be crosslinked to photocholesterol, especially if it is strongly expressed. In order to exclude such an âunspecificâ interaction, it is advisable to include a mutant in which the supposed cholesterol-binding site is mutated. In the case of HA labeling, a mutant with exchange of the site was reduced to 58%, relative to wt HA and normalized for the expression level of both proteins.</li><li>Since HA with mutations in the cholesterol-binding site are transported to the plasma membrane, but are no longer present in cholesterol-enriched domains (de Vries <em>et al.</em>, 2015), one can argue that its reduced labeling inside cells is due to the fact that less photocholesterol is present in their immediate neighborhood. Therefore, we also labeled immunoprecipitated HA, which, however, also revealed a reduced incorporation of photocholesterol (38% relative to wt HA).<br/><br/></li></ol><p class="ppzz"><strong>Trouble shooting</strong>:</p><ol class="npwtwo"><li>Efficient cross-linking requires large amount of the target protein. Therefore, the corresponding gene should be cloned into a vector with a strong promoter to achieve a high expression level.</li><li>The amount of photocholesterol used for labeling can be varied. We tested 20 ÂµM, 50 ÂµM and 100 ÂµM (final concentration) for labeling with immunoprecipitated HA. Higher concentration gives stronger fluorescence signal in the gel. In the final protocol, we used the same <strong>final concentration</strong> of 50 ÂµM for labeling HA both in living cells (5 Âµl photocholesterol in 1 ml DMEM) and with immunoprecipitated protein (0.5 Âµl photocholesterol in 0.1 ml PBS).</li><li>If no signal is seen in the target protein after labeling of living cells, one could still try labeling of the immunoprecipitated protein since higher concentration of photcholesterol can be achieved and the compound does not compete with authentic cholesterol present in the membrane.</li><li>If the analyzed protein is unstable, the step of washing with PBS (Step D9 in Part I) can be omitted. 4x reducing loading buffer can be directly added to the samples, which are boiled and subjected to SDS-PAGE. In this case, overnight washing of the gel with ddH<sub>2</sub>O is necessary to wash away background signal from Picolyl-Azide-Sulfo-Cy3.</li><li>If a positive result cannot be repeated, consider to use a new CuAAC kit.</li></ol><p class="pptt" id="biaoti31451">### Recipes</p><ol class="npwtwok"><li>Growth medium (stored at 4 Â°C)<br/>DMEM<br/>10% FCS<br/>100 U/L Penicillin/Streptomycin</li><li>IP buffer (filtered and stored at 4 Â°C for up to 1 year)<br/>500 mM Tris-HCl (pH 7.4)<br/>20 mM EDTA<br/>30 mM sodium pyrophosphate decahydrate<br/>10 mM sodium fluoride<br/>1 mM sodium orthovanadate<br/>2 mM benzamidine<br/>1 mM PMSF (phenylmethylsulfonyl fluoride)<br/>1 mM NEM<br/>1x protease inhibitor cocktail (freshly added for each experiment)<br/><em>Note: Some protease inhibitor could be omitted depending on the studied protein.</em> </li><li>Lysis buffer (freshly prepared)<br/>1% NP40 in IP buffer</li><li>Stacking-gel solution (freshly prepared)<br/>4% (w/v) acrylamide/bisacrylamide<br/>0.1% SDS<br/>125 mM Tris-HCl (pH 6.8)<br/>0.075% (w/v) APS<br/>0.15% (v/v) TEMED</li><li>Separating-gel solution (freshly prepared)<br/>12% (w/v) acrylamide/bisacrylamide stock solution<br/>0.1% (w/v) SDS<br/>375 mM Tris-HCl (pH 8.8)<br/>0.05% (w/v) APS<br/>0.1% (v/v) TEMED</li><li>5x non-reducing loading buffer (stored at -20 Â°C)<br/>1 M Tris-HCl<br/>10% (w/v) SDS<br/>50% (v/v) glycerin<br/>0.1 % (w/v) bromophenol blue<br/>pH 6.8</li><li>4x reducing loading buffer (stored at -20 Â°C)<br/>1 ml 5x non-reducing buffer<br/>250 Âµl 2 M (20x) DTT</li><li>PBST (stored at room temperature)<br/>PBS with 0.1% Tween-20</li><li>Blocking solution (freshly prepared)<br/>3% skimmed milk in PBST<br/></li></ol><p class="pptt" id="biaoti31452">### Acknowledgments</p><p class="ppzz">This work was supported by the German Research Foundation (SFB 740, TP C3) and by the Human Frontiers Science Program (awarded to M.V.). B.H. is the recipient of a Ph.D. fellowship from the China Scholarship Council (CSC). C.T. was funded by the German Research Foundation under Germanyâs Excellence Strategy (EXC2151-390873048). M.R.G is recipient of a Ph.D. fellowship from the DAAD. This protocol was adapted and modified from Hu <em>et al.</em> (2019).</p><p class="pptt" id="biaoti31453">### Competing interests</p><p class="ppzz">B.H. and co-authors declare no conflicts of interest.</p><p class="pptt" id="biaoti31454">### References</p><ol class="npwtwok"><li>de Vries, M., Herrmann, A. and Veit, M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25330796" target="_blank">A cholesterol consensus motif is required for efficient intracellular transport and raft association of a group 2 HA from influenza virus.</a> <em>Biochem J</em> 465(2): 305-314.</li><li>Hu, B., HÃ¶fer, C. T., Thiele, C. and Veit, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31118253" target="_blank">Cholesterol Binding to the Transmembrane Region of a Group 2 Hemagglutinin (HA) of Influenza Virus Is Essential for Virus Replication, Affecting both Virus Assembly and HA Fusion Activity.</a> <em>J Virol</em> 93(15): e00555-00519.</li><li>Hulce, J. J., Cognetta, A. B., Niphakis, M. J., Tully, S. E. and Cravatt, B. F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23396283" target="_blank">Proteome-wide mapping of cholesterol-interacting proteins in mammalian cells.</a> <em>Nat Methods</em> 10(3): 259-264.</li><li>Mintzer, E. A., Waarts, B. L., Wilschut, J. and Bittman, R. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11801250" target="_blank">Behavior of a photoactivatable analog of cholesterol, 6-photocholesterol, in model membranes.</a> <em>FEBS Lett</em> 510(3): 181-184.</li><li>Niwa, H., Yamamura, K. and Miyazaki, J. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1660837" target="_blank">Efficient selection for high-expression transfectants with a novel eukaryotic vector.</a> <em>Gene</em> 108(2): 193-199.</li><li>Thiele, C., Hannah, M. J., Fahrenholz, F. and Huttner, W. B. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10620806" target="_blank">Cholesterol binds to synaptophysin and is required for biogenesis of synaptic vesicles. </a><em>Nat Cell Biol</em> 2(1): 42-49.<br/></li></ol>