<body>
            <h1>Super-resolution Microscopy-based Bimolecular Fluorescence Complementation to Study Protein Complex Assembly and Co-localization</h1>
            <p><strong>Authors:</strong> Jingjing  Chen, Zulin  Yu, Jay R. Unruh, Brian D. Slaughter, Sue L. Jaspersen</p>
            <h2>Abstract</h2>
            <p>Numerous experimental approaches exist to study interactions between two subunits of a large macromolecular complex. However, most methods do not provide spatial and temporal information about binding, which are critical for dissecting the mechanism of assembly of nanosized complexes <em>in vivo</em>. While recent advances in super-resolution microscopy techniques have provided insights into biological structures beyond the diffraction limit, most require extensive expertise and/or special sample preparation, and it is a challenge to extend beyond binary, two color experiments. Using HyVolution, a super-resolution technique that combines confocal microscopy at sub-airy unit pinhole sizes with computational deconvolution, we achieved 140 nm resolution in both live and fixed samples with three colors, including two fluorescent proteins (mTurquoise2 and GFP) with significant spectral overlap that were distinguished by means of shifting the excitation wavelength away from common wavelengths. By combining HyVolution super-resolution fluorescence microscopy with bimolecular fluorescence complementation (SRM-BiFC), we describe a new assay capable of visualizing protein-protein interactions <em>in vivo</em> at sub-diffraction resolution. This method was used to improve our understanding of the ordered assembly of the <em>Saccharomyces cerevisiae</em> spindle pole body (SPB), a ~1 giga-Dalton heteromeric protein complex formed from 18 structural components present in multiple copies. We propose that SRM-BiFC is a powerful tool for examination of direct interactions between protein complex subunits at sub-diffraction resolution in live cells.</p>
            <p><strong>Keywords:</strong> Protein complex assembly, Super-resolution microscopy, BiFC, SIM, Hyvolution, Yeast</p>
            <h2>Background</h2><p>Protein complexes execute fundamental functions involved in various biological processes. To have these molecular machines assembled in a timely manner in a precise cellular location is critical for cellular activities, including protein synthesis, chromosome segregation, nuclear transport, signaling, motility and other diverse functions. Typically, the formation of many multimeric complexes is thought to occur in a stepwise manner via recruitment and integration of subunits or subcomplexes, which are bridged to existing molecules within the larger assembly by a physical protein-protein contact. Understanding the assembly of large macromolecular complexes is historically the job of biochemists and crystallographers who employ an arsenal of techniques: purification of complexes to determine the identity of subunits, co-purification of proteins, yeast two-hybrid screening, <em>in vitro</em> binding assays, chemical cross-linking studies and structural analysis to understand protein-protein interactions, and finally, reconstitution of the complex to determine the hierarchy of assembly.<br/> <br/>While this traditional approach has been successful in many cases, a major challenge is incorporating knowledge from studies on a handful of proteins to the larger and more relevant problem of macromolecular complex assembly within the cell. In cells, the timing of protein production, protein localization and post-translational modifications affect distribution, local concentrations and interactors that in turn influence complex assembly. A number of imaging-based techniques have been developed to study protein-protein interactions within cells, including fluorescent resonance energy transfer (FRET), which examines energy transfer between pairwise donor-acceptor fluorophores linked to proteins of interest; fluorescence cross-correlation spectroscopy (FCCS), which measures the diffusion pattern of two fluorescent molecules through the focal volume; and bimolecular fluorescence complementational (BiFC), which is based on ability of proteins in close proximity to self-complement and create a functional fluorescent protein from two non-fluorescent protein fragments (Bacia <em><em>et al.</em></em>, 2006; Slaughter <em>et al.</em>, 2007; Kerppola, 2008; Miller <em>et al.</em>, 2015; Unruh <em>et al.</em>, 2018). These techniques can be used in live cells to study endogenously expressed protein-protein interactions in a pairwise manner provided the molecules of interest can be fused to fluorescent proteins (FP). In yeast, genes can be easily tagged at the N- or C-termini using PCR-based methods (Gardner and Jaspersen, 2014). With the development of CRISPR/Cas9 methodologies, genome editing is becoming increasingly feasible such that these approaches can be applied in a wide array of systems. The use of endogenous proteins, rather than transgenes, greatly minimizes false positives caused by high levels of protein expression. In comparison to <em>in vitro</em> assays, FRET, FCCS and BiFC have the advantage of indicating where within a cell the protein-protein interaction occurs (for example, see Muller <em>et al.</em>, 2005; Cabantous and Waldo, 2006; Kerppola, 2006; Slaughter and Li, 2010; Sung <em>et al.</em>, 2013; Chen <em>et al.</em>, 2014; Smoyer <em>et al.</em>, 2016; Hennen <em>et al.</em>, 2018). However, these methods are limited in their spatial resolution due to the diffraction limit (~200 nm for most FPs), and combining FRET or FCCS with super-resolution methods in cells is extremely challenging.<br/> <br/>Structured illumination microscopy (SIM), stimulated emission depletion (STED), stochastic optical reconstruction microscopy (STORM) and photoactivated localization microscopy (PALM) are examples of super-resolution microscopy (SRM) approaches that are pushing cell biology in new directions once thought impossible by beating the diffraction limit (Huang <em>et al.</em>, 2009; Schermelleh <em>et al.</em>, 2010). The ~10-100 nm resolution achievable by these methods has enhanced our overall understanding of cell biology, particularly at the level of organelles, membranes and other cellular structures. The application of SRM to protein-protein interactions within nanosized protein complexes has been slower to develop, in part because it is difficult to study endogenous proteins with many of these methods, particularly if multiple fluorophores/FPs are needed to monitor interactions and to provide orientation of where within the complex or the assembly cycle the interaction is occurring. While a plethora of FPs have been developed, linear unmixing to image multiple fluorophores has not been integrated into SRM yet, limiting the use of many combinations of FPs. To overcome this obstacle, we combined a version of BiFC based on split-GFP to assay protein-protein interactions (Figure 1).To provide spatial and temporal information as to where and when binding within our complex of interest occurs at sub-diffraction resolution, we developed a three-color imaging strategy using HyVolution with 140 nm lateral resolution (Chen <em>et al.</em>, 2019). Importantly, our protocol allows for imaging of endogenously expressed proteins in live cells fused to mTurquoise2 and GFP, along with mCherry, without spectral cross-talk. While the method was developed in <em>Saccharomyces cerevisiae</em>, it is suitable for a wide variety of systems, and it greatly expands our ability to inspect the step-wise assembly pathway of heteromeric macromolecular machines by elucidating protein-protein interactions <em>in vivo</em> at the nanoscale level.</p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200212/20200212183656_8023.jpg" width="371"/><br/><strong>Figure 1. Structured illumination microscopy (SIM)-based split-GFP system to assay protein-protein interaction within nanosized protein complexes.</strong> A. Schematic of the bimolecular fluorescence complementation split-GFP system. Two parts of split-GFP can reconstruct and lead to GFP fluorescence when they associate with each other by interaction between their fusion proteins. B. An illustration of the yeast centrosome viewed from the side to show how it is embedded and anchored in the nuclear envelope (NE) by a series of integral membrane proteins, visualized by reconstituted split-GFP and mCherry tagging. The soluble inner-core plaque linker protein was fused to mTurquoise2. C. We showed that our spindle pole body (SPB) protein (fused to GFP<sub>1-10</sub>) is a component of the membrane complex. In a top down view of the cylinder-shaped SPB, we observed a 170 ring of reconstituted GFP when combined with a known membrane component (fused to mCherry-GFP<sub>11</sub>) using SIM-BiFC (Chen <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti31457">## Materials and Reagents</p><ol class="npwtwok"><li>0.2 Âµm filtration unit-150 ml (Thermo Fisher Scientific, catalog number: 150-0020)</li><li>13 x 100 mm glass culture tube (Fisher, catalog number:14-961-27)</li><li>15 cm plain wood applicators (Fisher, catalog number: 23-400-102)</li><li>1.7 ml microcentrifuge tube (VWR, catalog number: 87003-294)</li><li>25 x 75 mm glass microscope slides (Fisher, catalog number: 12-544-4)</li><li>22 x 30 mm Number 1.5 coverslips (VWR, catalog number: 48393-150)</li><li>25-slide humidified chamber (Daigger Scientific, catalog number: EF15989A)</li><li>Yeast strains (see Note 1)<ol class="npwthree"><li>SLJ10800 (MATa/Î± <em>SPC42-mTurquiose2-NATMX/Spc42-YFP-HIS3MX bar1/bar1 ADE2/ADE2 trp1-1/TRP1 lys2Î/LYS2 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 ura3-1/ura3-1</em>)</li><li>SLJ6209 (MATa <em>SPC42-mCherry-URA3MX bar1 ADE2 trp1-1 lys2Î leu2-3,112 his3-11,15 ura3-1</em>)</li><li>SLJ7904 (MATa/Î± <em>SPC110-GFP-KANMX/Spc110 bar1/bar1 ADE2/ade2 trp1-1/TRP1 lys2Î/LYS2 leu2-3,112/leu2-3,112 his3-11,15/his3-11,15 ura3-1/ura3-1</em>)</li><li>SLJ12889 (MATa <em>Spc110-mTurquiose2-HIS3MX bar1 ADE2 TRP1 lys2Î leu2-3,112 his3-11,15 ura3-1</em>)</li></ol>We have a large library of yeast ORFs fused at their N- or C-terminal to GFP<sub>1-10</sub> via PCR tagging methods in BY4741 (<em>MATa YFG::YFP-GFP<sub>1-10</sub>-URA3MX his3â1 leu2â0 ura3â0 met15â0 LYS2 ADE2</em>). To examine distribution to the nucleus, lumen, ER/outer nuclear membrane or cytoplasm, we created reporters for each compartment with mCherry- GFP<sub>11</sub> expressed from pRS305/pRS315 plasmids (Gardner and Jaspersen, 2014; Smoyer <em>et al.</em>, 2016) (see Notes 2-4)</li><li>Bacto-yeast extract</li><li>Bacto-peptone</li><li>Glucose</li><li>Bacto-agar</li><li>Yeast-nitrogen base with ammonium sulfate without amino acids</li><li>CSM powder (Sunrise Science, catalog number: 1001-100)</li><li>16% paraformaldehyde (Ted Pella, catalog number: 18505)</li><li>Sucrose</li><li>Sodium chloride (NaCl)</li><li>Potassium chloride (KCl)<br/></li><li>Disodium phosphate</li><li>Monopotassium phosphate</li><li>ProLong Diamond mounting medium (Thermo Fisher, catalog number: P36961)</li><li>DeltaVision Immersion Oil Kit; contains bottles of 18 oils with refractive indices from 1.500 to 1.534 (GE Healthcare, catalog number: 29163068)</li><li>Ethanol (Amresco, catalog number: 64-17-5)</li><li>YPD plate (see Recipes)</li><li>SC complete broth (see Recipes)<br/><em>Note: We typically do not culture cells in YPD liquid media since it produces high background levels of fluorescence.</em> </li><li>4% paraformaldehyde solution (see Recipes)</li><li>PBS (see Recipes)<br/></li></ol><p class="pptt" id="biaoti31458">### Equipment</p><ol class="npwtwok"><li>2 L flask</li><li>Rotating tube mixer (Fisher Scientific, catalog number: 346)</li><li>23 Â°C incubator (VWR, catalog number: 7774)</li><li>Culture roller (New Brunswick TC-7, catalog number: M1053-4004)</li><li>Desktop centrifuge (Eppendorf 5424, catalog number: 022620401)</li><li>Pipettes (Gilson P10, catalog number: F144802)</li><li>BioPhotometer (Eppendorf, model: 6131, catalog number: 62111-460)</li><li>GE Healthcare OMX structured illumination microscope (GE Healthcare DeltaVision OMX Blaze V4) with three scientific complementary metal-oxide semiconductor (sCMOS) cameras and 6 laser lines, or a similar microscope (<em>e.g.</em>, Nikon N-SIM, Zeiss Elyria)</li><li>Olympus PLANAPO N 60x, 1.42 NA oil immersion objective</li><li>DeltaVision OMX channel/image registration slide (GE Healthcare, catalog number: 53-852891-000)</li><li>Leica Inverted Confocal SP8 equipped with:<br/><ol class="npwthree"><li>White Light Laser (470-670 nm range)</li><li>An Argon Laser</li><li>3 Leica HyD Detectors </li><li>Leica Application Suite software with Hyvolution Module </li></ol>Our microscope was customized but a similar scope is available at <a href="https://www.leica-microsystems.com/products/confocal-microscopes/p/leica-tcs-sp8-x/" target="_blank">https://www.leica-microsystems.com/products/confocal-microscopes/p/leica-tcs-sp8-x/</a> </li><li>Leica 100x, 1.4-NA HC PL APO OIL CS2 Objective<br/></li></ol><p class="pptt" id="biaoti31459">### Software</p><ol class="npwtwok"><li>OMX acquisition computer and software v3.70 (GE Healthcare, <a href="http://incelldownload.gehealthcare.com/bin/download_data/OMX/V3.70/DeltaVisionOMX.htm" target="_blank">http://incelldownload.gehealthcare.com/bin/download_data/OMX/V3.70/DeltaVisionOMX.htm</a>)</li><li>OMX image-processing station with softWoRx v6.52 (GE Healthcare, <a href="http://incelldownload.gehealthcare.com/bin/download_data/SoftWoRx/6.5.2/SoftWoRx.htm" target="_blank">http://incelldownload.gehealthcare.com/bin/download_data/SoftWoRx/6.5.2/SoftWoRx.htm</a>) software installed</li><li>ImageJ (Java software for image-processing analysis; freely available at <a href="http://rsbweb.nih.gov/ij/" target="_blank">http://rsbweb.nih.gov/ij/</a> or <a href="http://fiji.sc/" target="_blank">http://Fiji.sc</a>) (Schindelin <em>et al.</em>, 2012)</li><li>ImageJ plugins (created in the microscopy center of The Stowers Institute for Medical Research; freely available at <a href="http://research.stowers.org/imagejplugins/index.html" target="_blank">http://research.stowers.org/imagejplugins/index.html</a>)</li><li>Leica Application Suite X (LAS X) software for image acquisition (version 3.1.5.16308) (Leica Microsystems)</li><li>Huygens Professional version 17.10.0p5.64b, Scientific Volume Imaging BV, Hilversum, The Netherlands<br/></li></ol><p class="pptt" id="biaoti31460">### Procedure</p><ol class="npwone"><li>Sample preparation<ol class="npwtwo"><li>Yeast growth and culture<ol class="npwthree"><li>Pick up a single colony from the YPD plate and inoculate into 2 ml SC complete broth. Culture at 23 Â°C overnight in a rotating cell culture tube mixer at 50 rpm.<br/><em>Note: Keep SC complete broth in closed cabinet or wrapped in foil as light exposure over long periods will result in the formation of a precipitate that increases autofluorescence.</em> </li><li>The next day, measure the overnight culture OD<sub>600</sub> with BioPhotometer, so that the culture can be diluted back an OD<sub>600</sub> 0.2-0.3 in 2 ml SC complete broth. Continue growing at 23 Â°C for 3-5 h until the cells reach an OD<sub>600</sub> of ~0.8. If studying temperature-sensitive alleles, after 2 h of growth at 23 Â°C, shift to 34-36 Â°C for another 4 h.</li></ol></li><li>Yeast sample preparation<ol class="npwthree"><li>Harvest cultures when the OD<sub>600</sub> reaches 0.8 or after the temperature shift. Pellet 1-2 ml of cells in a microcentrifuge tube by spinning for 1 min at 13,400 <em>x g</em> in a microcentrifuge at room temperature.</li><li>Aspirate media and resuspend cells in 1 ml 4% paraformaldehyde solution. Place onto rotating tube mixer at 23 Â°C for 15 min.</li><li>Spin for 1 min at 13,400 <em>x g</em> in an microcentrifuge at room temperature.</li><li>Aspirate media and resuspend cells in 1 ml PBS.</li><li>Spin for 1 min at 13,400 <em>x g</em> in an microcentrifuge at room temperature and remove PBS. Repeat two more times. After the last wash carefully remove all the remaining liquid.</li><li>Add appropriate volume of mounting medium ProLong Diamond to attain desired cell intensity in the specimen, resuspend cell pellet by vortexing briefly at low speed to avoid bubble generation.</li><li>Pipet 5 Âµl onto a cleaned 22 mm x 30 mm coverslip. Use the pipet tip to spread cells over surface.<br/><em>Note: Both coverslips and glass slides should be cleaned using 70% ethanol and allowed to dry on a lint-free surface such as lens paper.</em> </li><li>Overlay with a clean glass slide, pressing extremely hard to ensure that the cells form a single monolayer between the two glass surfaces.<br/><em>Note: When preparing the specimen, it is important that cells are less than 10 microns from the coverslip surface for SIM imaging.</em> </li><li>Place in a sealed humidified chamber with the coverslip facing down.</li><li>Incubate in the dark for at least 18-24 h at room temperature. The slides can be stored at 4 Â°C for up to 3 days, after which time the signal decreases and the background increases. If you anticipate your specimen will result in a dim signal, we recommend that it be imaged immediately for optimal results.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Acquire multiple color SIM images with GE Healthcare OMX<br/>We acquire multiple color 3D-SIM images using a GE Healthcare DeltaVision OMX Blaze V3 fitted with an Olympus PlanApo N 100x 1.42 NA oil objective. We first begin by aligning the cameras, then we proceed to our samples. This ensures day to day reproducibility. This protocol is specifically for the DeltaVision OMX Blaze; if using another instrument, follow your manufacturerâs recommended instructions.<ol class="npwtwo"><li>Slide alignment<ol class="npwthree"><li>Place the GE registration slide in the slide holder. Move the stage to locate the center of slider and find an area with the 20 x 20 grid of 100 nm holes with 5 Âµm spacing.</li><li>Choose âconventionalâ as the light path, set up the illumination intensity (DIC in the software) to 50% or 100%. Enable multiple cameras (for example, mCherry, GFP, DAP) and adjust the exposure time of individual channel.</li><li>Monitor the intensity histogram and adjust exposure time for individual channels so the mean intensity across all channels are similar.</li><li>Set up the image size to 1024 x 1024 (the maxiumum size on OMX system) and acquire a 3 Âµm z-stack image with z-spacing of 125 nm.</li><li>We use the central part of acquired image to perfom alignment due to uneven illumination on the corners. We crop the image (set up âOutput Optionsâ: X/Y/Width/Height as 112/112/800/800 or similar values) using the software softWoRx, the intergated software on OMX system for imaging processing. Load the cropped image to softWoRx and process it under âProcessâ and then âCreate OMX Image Alignmentâ.</li><li>Ensure the âShow aligned target imageâ check box is selected (Demmerle <em>et al.</em>, 2017). Display all channels and closely examine the image, especially at the corners. Make sure the holes in each channel co-localize. The manufacturerâs instruction shows a good example of aligned image on page 4 (<a href="http://incelldownload.gehealthcare.com/bin/download_data/SoftWoRx/7.0.0/DVOMXSR_ImageAlignment_04-720165-000CC.pdf" target="_blank">http://incelldownload.gehealthcare.com/bin/download_data/SoftWoRx/7.0.0/DVOMXSR_ImageAlignment_04-720165-000CC.pdf</a>).</li></ol></li><li>mTurquiose2 and YFP alignment evaluation<br/><em>Notes:</em> <ol class="npwfour"><li><em>To display single images, we typically scale the images four times in ImageJ (Image&gt;Scale) with bilinear interpolation. For fitting and analysis, we leave the images at their original scale.</em> </li><li><em>We typically prefer to image mTurquoise2 and YFP together as they have similar folding kinetics in yeast. A similar experiment could be easily extended to three color with mCherry or other common red fluorescent proteins, </em><em>e.g.</em><em>, mKate2 and TagRFP657. Both mKate2 and TagRFP657 have high excitation and emission wavelengths and therefore have no cross-talk with GFP and mTurquoise2 with our current settings. In the current study, the slow folding kinetics of mCherry relative to GFP makes experimental data more difficult to interpret, a problem that does not exist for mTurquoise2 and YFP.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwthree"><li>Switch the system to âSIâ Module. Next, acquire a 2 color image 1.5 Âµm z-stack (spacing = 0.125 Âµm) with diploid yeast strain SLJ10800. In the strain, Spc42 is tagged at the endogenous loci with mTurquiose2 and YFP separately. Even if one does not work on spindle pole bodies, because they form small, bright puncta of a defined size and distribution, this yeast strain useful to evaluate color alignment with SIM.</li><li>Set up 2 channel SIM acquisition with the correct excitation laser (514 nm and 445 nm) and emission filter, and image YFP first and followed by mTurquiose2.</li><li>We use 50-100 ms exposure time for both channels if possible. Longer exposure times may cause artifacts due to sample and/or mechanical drift; shorter exposure times require higher laser power and may cause photobleaching.</li><li>We use 10% or 1% laser power (maximum laser output 100 mW) to excite both channels. Monitor the camera histogram and adjust the exposure time and laser power accordingly to achieve a signal to noise ratio of 5 or more.</li><li>We use immersion oil with a refractive index (RI) of 1.514 or 1.516 for mTurquiose2 and YFP; we use immersion oil with RI of 1.516-1.518 for mCherry and GFP in other experiments. The optimal immersion oil plays critical role for successful reconstruction and it largely depends on the distance between the signal and coverslip. Due to small size, the distance between region of interest (SPB) in yeast cells and coverslip is very constant and less than a few microns. For other samples, the distance may vary significantly. Please refer to two recent protocol papers to adjust the optimal immersion oil (Demmerle <em>et al.</em>, 2017; Wang <em>et al.</em>, 2018).</li><li>Align the successfully reconstructed images by running the softWoRx alignment protocol, utilizing the alignment files created in Step B2.</li><li>Open the reconstructed and aligned (SIR_ALX) images in ImageJ. Do a Z-projection (Image&gt;Stacks&gt;Z Project) with Max Intensity to cover the entire SPB in the z-dimension. Use the line tool in ImageJ to draw a line 4 pixels wide across the 2 SPBs in the image (Figure 2A). Measure the average intensity along the line using âpolyline kymograph jru v1â.</li><li>Verify that SPB intensity measured along the line from the two channels has a similar pattern. The intensity peak for both SPBs should overlap well with an offset of less than 1 pixel (40 nm) (Figure 2B).<br/><em>Note: Due to mechanical shift or sample drift during acquisition, slight misalignment always occurs with SIM. Single-particle averaging will further increase resolution and alignment precision (Burns </em><em>et al.</em><em>, 2015). To minimize misalignment, make sure to closely follow the alignment calibration and be certain that yeast are properly immobilized on the coverslip.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200212/20200212184250_5986.jpg" width="276"/><br/><strong>Figure 2. mTurquiose2 and YFP alignment evaluation at yeast SPBs.</strong> A. Visualization of Spc42-YFP and Spc42-mTurquiose2 signals at two SPB sub-structures ~150 nm apart in the strain SLJ10800, shown here in a maximum intensity projection along the z-axis. B. Line profiles to measure the intensity between the two spots in both YFP and mTurquiose2 channels shows that the color alignment error is less than 1 pixel (40 nm).<br/><br/></li><li>If the channels do not align, repeat the alignment protocol starting with the color alignment slide.</li><li>To align mCherry with mTuruqoise2 and YFP, repeat the alignment procedure starting with the color alignment slide. Use a multi-color sample for alignment using Steps B2b-B2f.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Acquire three-color HyVolution images with Leica SP8<br/>We routinely image three-color samples by SIM and conventional confocal systems. GFP (or split-GFP) has a single absorption peak centered at about 488 nm, which is one of the most common laser lines in the lab and has been routinely used to excite GFP. However, the combination of GFP (or split-GFP) and mTurquiose2 is problematic due to significant cross-talk both in terms of emission and excitation profiles. Traditional imaging approaches on conventional systems that use 488 nm and 455 nm excitation for GFP and mTurquiose2, respectively, will not result in cross-talk free images. Corrective measures must be taken to subtract off cross-talk. Another alternative is to use spectral imaging (Zimmermann <em>et al.</em>, 2003). However, dim samples particularly in the mTurquiose2 channel, combined with significant cross-talk from GFP, can lead to misinterpretation of positive signal. Below we describe another solution to this problem.<br/><br/><strong><em>Spectrum setting for three colors</em></strong> <ol class="npwtwo"><li>Run the Software LAS X in the TCS SP8 mode. Set up a sequential mode with 3 color images in the order of mCherry, reconstituted split-GFP/GFP and mTurquiose2. Choose âbetween frameâ as the default setting for channel switching.</li><li>We found that shifting the excitation of GFP from 488 nm to 496 nm dramatically reduced excitation of mTurquiose2. Excitation efficacy of reconstituted split-GFP/GFP is 89% at 496 nm, which is only 10% less than the value at 488 nm, while the shift to 496 nm excitation leads to absorption efficacy of mTurquiose2 that is less than 0.02% (Lambert, 2019). Thus, using 496 nm laser line to excite reconstituted split-GFP, along with careful selection of emission filters, effectively eliminates bleed through of mTurquiose2 to reconstituted split-GFP/GFP (Figure 3A). </li><li>We set up the excitation of the 3 channels as follows: mCherry (561 nm, white light laser or argon laser), reconstituted split-GFP (496 nm, argon laser), mTurquiose2 (445 nm, white laser or argon laser). Due to weak signal of reconstituted split-GFP and low output of white laser, we use the higher power laser line (argon laser) to excite split-GFP.</li><li>There is no Notch filter for 496 nm laser line in our SP8 system. Therefore, we used the internal polarizer to remove reflected light in the reconstituted split-GFP/GFP channel.</li><li>In the âFluorifier Disc Settingsâ panel, set up a value of â90â for the âPolarizerâ to remove the reflected light from microscope.</li><li>For collection of emission photons with the 3 HyD detectors, the emission filters are set up as follows: mCherry: 570-635 nm or a similar range, reconstituted split-GFP/GFP: 508-550 nm; mTurquiose2: 463-495 nm. The emission filter setting at mTurquoise2 channel will block ~94% emission photons from the reconstituted split-GFP/GFP channel (Figure 3B).</li><li>Switch the image mode to HyVolution in the acquisition software LAS X and focus the sample using mCherry. We begin by using the default settings within HyVolution. In LAS X, increase microscope resolution by shrinking the size of pinhole. A pinhole close to 0.6 AU (airy unit) compensates between resolution and signal intensity (Borlinghaus and Kappel, 2016).</li><li>The SPB contains ~1,000 copies of each component so it is a common source of cross-talk and bleed-through in multicolor imaging experiments (for example, Muller <em>et al.</em>, 2005). Using the HyVolution mode, we verified 3 color imaging settings with the yeast strains tagged with mCherry (SLJ6209), GFP (SLJ7904) and mTurquoise2 (SLJ12889) only. For the yeast strain SLJ7904 containing GFP only, we did not observe emission in the mTurquoise2 channel (463-495 nm) when we excited at 445 nm (mTurquoise2 excitation) (Figure 3C, top panel, left). Also, no mTurquoise2 emission was observed when we excited mTurquoise2 only in the strain SLJ12889 at 496 nm in the GFP channel (508-550 nm) (Figure 3C, center panel, right). For visualization purposes, we apply a filter (gaussian blur, 1 pixel) and set the contrast from 0-30 for all images except Spc110-GFP excited with 488 nm, setting contrast from 1-67 due to the strong signal (Figure 3C). The GFP channel has higher background due to high reflection, there is no mCherry signal for both SLJ7904 and SLJ12889 (data not shown).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200212/20200212184428_6129.jpg" width="129"/><br/><strong>Figure 3. Three color HyVolution image to study multiple SPB components.</strong> A. absorption spectrum and laser illumination for 3 channels (mCherry, split-GFP/GFP and mTurquiose2). B. emission spectrum and emission filter setting for these 3 channels. C. In HyVolution mode, SLJ12889 (Spc110-mTurquiose2) and SLJ7904 (Spc110-GFP) were imaged with the indicated spectrum setting. mTurquiose2 shows signal crosstalk (arrows pointed) by using 488 nm excitation (bottom panel, right) with 508-550 nm emission filter, however, no bleed-through was observed at 496 nm excitation with the same emission filter (center panels, right).<br/></li></ol></li></ol><p class="pptt" id="biaoti31461">### Notes</p><ol class="npwtwok"><li>This protocol was developed for <em>S. cerevisiae</em> but could be applied to other systems. We use yeast that do not contain mutations in the adenine biosynthesis pathway to limit the amount of autofluorescence (Weisman <em>et al.</em>, 1987). This dramatically improves signal-to-noise, which is critical for dim samples (Unruh <em>et al.</em>, 2018). Genes are tagged using PCR-based approaches as previously described (Gardner and Jaspersen, 2014). We generally grow yeast at 23 Â°C rather than 30 Â°C to slow down cell growth, which results in more fluorescent protein due to improved fluorophore maturation.</li><li>When using BiFC to study protein-protein interactions, it is important to consider the topology of target proteins, especially integral membrane proteins. The two halves of split-GFP must be in the same compartment, such as the nucleus or cytoplasm, to reconstitute a functional GFP. For this reason, tagging at the N- or C-terminus may be preferred. It is also imperative to perform appropriate controls, such as a vector only and a mutant/different subunit to confirm specificity in the interaction (see Chen <em>et al.</em>, 2019).</li><li>One part of split-GFP, GFP<sub>11</sub>, is only 25 amino acids long. We have been unable to find a commercially available antibody that recognizes GFP<sub>11</sub> so we typically use mCherry as part of our GFP<sub>11</sub> fusion. mCherry can be added to the opposite terminus, or in between the target and GFP<sub>11</sub>. It is important that both GFP<sub>1-10</sub> and GFP<sub>11</sub> are at the extreme N- or C-terminus of the final fusion protein so they are able to interact.</li><li>In our study, the N-terminally tagged mCherry-GFP<sub>11</sub> fusion protein is expressed under the <em>CDC42</em> promoter. Expression from this promoter is slightly higher than the endogenous expression, but it is sufficiently low that two control proteins do not interact (Smoyer <em>et al.</em>, 2016; Chen <em>et al.</em>, 2019). To facilitate optimal expression and stability in yeast, we use yeast codon optimized versions of all fluorescent proteins, including mCherry-GFP<sub>11</sub> and GFP<sub>1-10</sub>. These PCR tagging plasmids and reporters have been deposited at AddGene (<a href="https://www.addgene.org/search/advanced/?q=jaspersen" target="_blank">https://www.addgene.org/search/advanced/?q=jaspersen</a>).<br/></li></ol><p class="pptt" id="biaoti31462">### Recipes</p><ol class="npwtwok"><li>YPD plate (1 L)<br/>10 g Bacto-yeast extract<br/>20 g Bacto-peptone<br/>20 g Glucose<br/>20 g Bacto-agar<br/>950 ml ddH<sub>2</sub>O<br/>Dissolve the first three ingredients in water in a 2 L flask, then add the Bacto-agar along with a large stir bar. Sterilize for 30 min. After autoclaving, place on a stir plate until the liquid has cooled to ~50-60 Â°C. Pour roughly 25 ml of liquid into 15 cm Petri plates using sterile technique. Let the media fully cool and the plates dry for 1-2 days.</li><li>SC complete broth (1 L)<br/>6.7 g Yeast-nitrogen base with ammonium sulfate without amino acids<br/>20 g Glucose<br/>2.0 g CSM powder<br/>Mix together then autoclave to sterilize</li><li>4% Paraformaldehyde solution (80 ml)<br/>20 ml 16% paraformaldehyde (EM grade, methanol free)<br/>11.7 g sucrose and 60 ml ddH<sub>2</sub>O<br/>Pass through a 0.2 Âµm filter<br/>Store at 4 Â°C in a container shielded from light. Use this solution within 1-2 months<br/><em>Note: Both coverslips and glass slides should be cleaned using 70% ethanol and allowed to dry on a lint-free surface such as lens paper.</em> </li><li>PBS (1 L)<br/>8 g sodium chloride<br/>0.2 g potassium chloride<br/>1.42 g disodium phosphate<br/>0.24 g monopotassium phosphate<br/>1,000 ml ddH<sub>2</sub>O<br/>Adjust the pH to 7.4<br/></li></ol><p class="pptt" id="biaoti31463">### Acknowledgments</p><p class="ppzz">We are grateful to members of the Microscopy Center and the Jaspersen lab for advice during the course of this project. Research reported in this publication was supported by the Stowers Institute for Medical Research and the National Institutes of Health, National Institute of General Medical Sciences under award number R01GM121443 (to SLJ). The authors declare no competing financial interests.</p><p class="pptt" id="biaoti31464">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti31465">### References</p><ol class="npwtwok"><li>Bacia, K., Kim, S. A. and Schwille, P. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16432516" target="_blank">Fluorescence cross-correlation spectroscopy in living cells.</a> <em>Nat Methods</em> 3(2): 83-89. </li><li>Borlinghaus, R. T. and Kappel, C. (2016). <a href="https://doi.org/10.1038/nmeth.f.392" target="_blank">HyVolutionâthe smart path to confocal super-resolution.</a> <em>Nature Methods</em> 13: 276. </li><li>Burns, S., Avena, J. S., Unruh, J. R., Yu, Z., Smith, S. E., Slaughter, B. D., Winey, M. and Jaspersen, S. L. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26371506" target="_blank">Structured illumination with particle averaging reveals novel roles for yeast centrosome components during duplication.</a> <em>Elife</em> 4: e08586. </li><li>Cabantous, S. and Waldo, G. S. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16990817" target="_blank"><em>In vivo</em> and <em>in vitro</em> protein solubility assays using split GFP.</a> <em>Nat Methods</em> 3(10): 845-854. </li><li>Chen, J., Gardner, J. M., Yu, Z., Smith, S. E., McKinney, S., Slaughter, B. D., Unruh, J. R. and Jaspersen, S. L. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30862629" target="_blank">Yeast centrosome components form a noncanonical LINC complex at the nuclear envelope insertion site.</a><em> J Cell Biol</em> 218(5): 1478-1490.</li><li>Chen, J., Smoyer, C. J., Slaughter, B. D., Unruh, J. R. and Jaspersen, S. L. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24515347" target="_blank">The SUN protein Mps3 controls Ndc1 distribution and function on the nuclear membrane.</a> <em>J Cell Biol </em>204(4): 523-539. </li><li>Demmerle, J., Innocent, C., North, A. J., Ball, G., Muller, M., Miron, E., Matsuda, A., Dobbie, I. M., Markaki, Y. and Schermelleh, L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28406496" target="_blank">Strategic and practical guidelines for successful structured illumination microscopy.</a> <em>Nat Protoc</em> 12(5): 988-1010.</li><li>Gardner, J. M. and Jaspersen, S. L. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25213239" target="_blank">Manipulating the yeast genome: deletion, mutation, and tagging by PCR.</a> <em>Methods Mol Biol </em>1205: 45-78.</li><li>Hennen, J., Saunders, C. A., Mueller, J. D. and Luxton, G. W. G. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29514929" target="_blank">Fluorescence fluctuation spectroscopy reveals differential SUN protein oligomerization in living cells.</a> <em>Mol Biol Cell </em>29(9): 1003-1011. </li><li>Huang, B., Bates, M. and Zhuang, X. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19489737" target="_blank">Super-resolution fluorescence microscopy.</a> <em>Annu Rev Biochem </em>78: 993-1016.</li><li>Kerppola, T. K. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17406412" target="_blank">Design and implementation of bimolecular fluorescence complementation (BiFC) assays for the visualization of protein interactions in living cells.</a> <em>Nat Protoc </em>1(3): 1278-1286.</li><li>Kerppola, T. K. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18573091" target="_blank">Bimolecular fluorescence complementation (BiFC) analysis as a probe of protein interactions in living cells.</a> <em>Annu Rev Biophys</em> 37: 465-487.</li><li>Lambert, T. J. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30886412" target="_blank">FPbase: a community-editable fluorescent protein database.</a> <em>Nat Methods</em> 16(4): 277-278. </li><li>Miller, K. E., Kim, Y., Huh, W. K. and Park, H. O. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25772494" target="_blank">Bimolecular fluorescence complementation (BiFC) analysis: advances and recent applications for genome-wide interaction studies.</a> <em>J Mol Biol</em> 427(11): 2039-2055.</li><li>Muller, E. G., Snydsman, B. E., Novik, I., Hailey, D. W., Gestaut, D. R., Niemann, C. A., O'Toole, E. T., Giddings, T. H., Jr., Sundin, B. A. and Davis, T. N. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15872084" target="_blank">The organization of the core proteins of the yeast spindle pole body.</a> <em>Mol Biol Cell</em> 16(7): 3341-3352. </li><li>Schermelleh, L., Heintzmann, R. and Leonhardt, H. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20643879" target="_blank">A guide to super-resolution fluorescence microscopy.</a> <em>J Cell Biol</em> 190(2): 165-175.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682. </li><li>Slaughter, B. D. and Li, R. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21160072" target="_blank">Toward quantitative "<em>in vivo</em> biochemistry" with fluorescence fluctuation spectroscopy.</a> <em>Mol Biol Cell</em> 21(24): 4306-4311.</li><li>Slaughter, B. D., Schwartz, J. W. and Li, R. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18077328" target="_blank">Mapping dynamic protein interactions in MAP kinase signaling using live-cell fluorescence fluctuation spectroscopy and imaging.</a> <em>Proc Natl Acad Sci U S A</em> 104(51): 20320-20325. </li><li>Smoyer, C. J., Katta, S. S., Gardner, J. M., Stoltz, L., McCroskey, S., Bradford, W. D., McClain, M., Smith, S. E., Slaughter, B. D., Unruh, J. R. and Jaspersen, S. L. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27831485" target="_blank">Analysis of membrane proteins localizing to the inner nuclear envelope in living cells.</a> <em>J Cell Biol</em> 215(4): 575-590.</li><li>Sung, M. K., Lim, G., Yi, D. G., Chang, Y. J., Yang, E. B., Lee, K. and Huh, W. K. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23403034" target="_blank">Genome-wide bimolecular fluorescence complementation analysis of SUMO interactome in yeast.</a> <em>Genome Res</em> 23(4): 736-746.</li><li>Unruh, J. R., Slaughter, B. D. and Jaspersen, S. L. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30141044" target="_blank">Functional analysis of the Yeast LINC complex using fluctuation spectroscopy and super-resolution imaging.</a> <em>Methods Mol Biol</em> 1840: 137-161. </li><li>Wang, Y., Yu, Z., Cahoon, C. K., Parmely, T., Thomas, N., Unruh, J. R., Slaughter, B. D. and Hawley, R. S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30072723" target="_blank">Combined expansion microscopy with structured illumination microscopy for analyzing protein complexes.</a> <em>Nat Protoc</em> 13(8): 1869-1895.</li><li>Weisman, L. S., Bacallao, R. and Wickner, W. (1987). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2444598" target="_blank">Multiple methods of visualizing the yeast vacuole permit evaluation of its morphology and inheritance during the cell cycle.</a> <em>J Cell Biol </em>105(4): 1539-1547. </li><li>Zimmermann, T., Rietdorf, J. and Pepperkok, R. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12829241" target="_blank">Spectral imaging and its applications in live cell microscopy.</a> <em>FEBS Lett</em> 546(1): 87-92. <br/><br/></li></ol>