<body>
            <h1>RNA Sequencing of Single Myofibers from Mus musculus</h1>
            <p><strong>Authors:</strong> Darren M. Blackburn, Felicia  Lazure, Vahab D. Soleimani</p>
            <h2>Abstract</h2>
            <p>Whole transcriptome analysis is a key method in biology that allows researchers to determine the effect a condition has on gene regulation. One difficulty in RNA sequencing of muscle is that traditional methods are performed on the whole muscle, but this captures non-myogenic cells that are part of the muscle. In order to analyze only the transcriptome of myofibers we combine single myofiber isolation with SMART-Seq to provide high resolution genome wide expression of a single myofiber.</p>
            <p><strong>Keywords:</strong> Single myofiber, Skeletal muscle, SMART-Seq, Gene expression analysis, Transcriptomics</p>
            <h2>Background</h2><p>Differences in gene expression are key indicators used in studies to see the effect of disease and ageing on cell and tissue types. Switching Mechanism at 5â² end of RNA Templates Sequencing (SMART-Seq) is a powerful technique that enables high resolution sequencing of mRNA from a limited number of cells (Picelli <em>et al.</em>, 2014).<br/> <br/>Traditionally, RNA sequencing of muscle has been performed on the whole muscle (Pette <em>et al.</em>, 1999). However, muscle is a very heterogeneous tissue that is composed of a variety of cell types, such as adipocytes, endothelial cells, mesenchymal cells, and fibroblasts (Arnold <em>et al.</em>, 2007; Christov <em>et al.</em>, 2007; Joe <em>et al.</em>, 2010; Giordani <em>et al.</em>, 2019). Adding to this problem is the variety of different types of fibers, with two main groups known as the fast and slow twitch fibers (Brooke and Kaiser, 1970; Pette and Staron, 1997). In order to clearly determine the changes in transcriptome that arise solely from the muscle fiber without the confounding presence of the other cell types, the fiber must be isolated from these cell types.<br/> <br/>Here, we combine single myofiber isolation with SMART-Seq to perform a high-resolution sequencing of mRNA at a quality comparable to traditional whole muscle sequencing, while removing the presence of the other non-myogenic cell types. This method will allow researchers to analyze changes in the transcriptome in myofibers under different conditions, such as disease and ageing, and to distinguish the transcriptome of slow and fast fiber types. We successfully implemented this method to analyze the differences in the whole transcriptome of single myofibers isolated from young and old mice.</p>
        </body>

<p class="pptt" id="biaoti31344">## Materials and Reagents</p><ol class="npwone"><li>Materials<ol class="npwtwo"><li>Pipette tips</li><li>6-well plate (TC-tested) (Sarstedt, catalog number: 83.3920)</li><li>Glass Pasteur pipettes (VWR, catalog number: 14672-200)</li><li>1.5 ml Microtubes (Sarstedt, catalog number: 72.690.300)</li><li>0.2 ml Microtubes (Sarstedt, catalog number: 72.737)</li><li>Magnetic rack (Thermo Fisher, DynaMag-2 Magnet, catalog number: 12321D)</li><li>0.22 micron filter (UltiDent, catalog number: 229747)</li><li>Large bore glass pipette</li><li>Small bore glass pipette<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Animal<br/>C57BL/6 mouse (Procedure has been successfully performed on mice aged from 1 month to 24 months of age)<br/><br/></li><li>Reagents<br/><strong><em>For digestion of the extensor digitorum longus</em></strong> <ol class="npwtwo"><li>PBS (Wisent, catalog number: 311-425-CL)</li><li>Collagenase from Clostridium histolyticum (Sigma, catalog number: C0130-1G)</li><li>DMEM (Invitrogen, catalog number: 11995073)</li><li>Horse Serum (Wisent, catalog number: 065-250)</li><li>Trypsin (Gibco, catalog number: 15090-046)</li><li>Isoflurane (Fresenius Kabi, catalog number: CP0406V2)</li><li>CO<sub>2</sub> (Praxair, catalog number: GP-700500)</li><li>Digestion buffer (see Recipes)</li><li>Coating media (see Recipes)<br/><br/></li></ol><strong><em>For extraction of RNA from a single myofiber</em></strong> <ol class="npwtwo"><li>RNase free water</li><li>RNase inhibitors (from SMART-Seq HT kit, Takara BIO USA, catalog number: 634456)</li><li>SMART-Seq Reaction Buffer (see Recipes)</li></ol><br/><strong><em>For cDNA library preparation</em></strong> <ol class="npwtwo"><li>SMART-Seq HT kit (Takara BIO USA, catalog number: 634456)</li></ol><br/><strong><em>For library purification</em></strong> <ol class="npwtwo"><li>Ampure XP beads (Beckman Coulter Life Sciences, catalog number: A63882)</li><li>Ethanol (Commercial Alcohols, catalog number: P016EAAN)</li><li>RNase free water<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong><em>For preparing sequencing ready libraries</em></strong> <ol class="npwtwo"><li>Nextera XT DNA Library Preparation kit (Illumina, catalog number: FC-131-1024)</li><li>Nextera XT Index kit (Illumina, catalog number: FC-131-1001)</li><li>Quant-IT Picogreen dsDNA Assay kit (Invitrogen, catalog number: P7589)</li></ol><br/><strong><em>For purification and size selection</em></strong> <ol class="npwtwo"><li>Ampure XP beads (Beckman Coulter Life Sciences, catalog number: A63882)</li><li>Ethanol (Commercial Alcohols, catalog number: P016EAAN)</li><li>RNase free water</li><li>Quant-IT Picogreen dsDNA Assay kit (Invitrogen, catalog number: P7589)</li></ol><br/><strong><em>For quality control</em></strong> <ol class="npwtwo"><li>Agarose (Bioshop, catalog number: AGA001.500)</li><li>Tris (Hydroxymethyl) Amino Methane (Multicell, catalog number: 600-125-LG)</li><li>Glacial acetic acid (Fisherbrand, catalog number: 351272-212)</li><li>Na<sub>2</sub>EDTAÂ·2H<sub>2</sub>O (Bioshop, catalog number: EDT 001)</li><li>GelGreen Nucleic Acid Gel Stain (Biotium, catalog number: 41005)</li><li>SYBR Green (Applied Biosystems, catalog number: A25742)</li><li>Primers (Integrated DNA Technologies, standard desalting, 25nmole) (Table 1)</li><li>100 bp DNA ladder (GenedireX, catalog number: DM001-R500)</li><li>TAE Buffer (see Recipes)<br/><br/><strong>Table 1. Primer sequences used for quality control of libraries</strong><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200213/20200213172625_3587.jpg" width="400"/><br/></div></li></ol></li></ol><p class="pptt" id="biaoti31345">### Equipment</p><ol class="npwtwok"><li>Dissection tools<br/><ol class="npwthree"><li>Fine point high precision forceps (Fisher Scientific, catalog number: 22-327379)</li><li>Sharp-Pointed dissecting scissors (Fisher Scientific, catalog number: 08-935)</li></ol></li><li>Pipettes (P20, P200)</li><li>Diamond pen (VWR, catalog number: 201-0392)</li><li>Microscope (Fisher Scientific, inverted microscope, equipped with transmitted light and a 4x objective)</li><li>Centrifuge (Thermo Scientific, Sorval Legend Micro 21R Microcentrifuge, catalog number: 75002447)</li><li>Thermocycler (Bio-Rad, C1000 Touch Thermal Cycler, catalog number: 1851148)</li><li>Mini-centrifuge (Mandel, Benchmark MyFuge Minicentrifuge, catalog number: BEN-C1008-R)</li><li>Gel imager (LI-COR, Odyssey FC Imaging System)</li><li>qPCR machine (Applied Biosystems, QuantStudio7 Flex)</li><li>Bioanalyzer (Agilent, BioAnalyzer 2100)</li><li>Cell incubator (Thermo Scientific, Forma Series II Water-Jacketed CO<sub>2</sub> Incubator, catalog number:3110)</li><li>Vortex mixer (Mandel)</li><li>Power supply (Bio-Rad, PowerPac, catalog number: 1645050)</li><li>Microplate reader (BioTek, synergy4)<br/></li></ol><p class="pptt" id="biaoti31346">### Procedure</p><p class="ppzz"><strong>Procedure Overview:</strong> </p><ol class="npwone"><li>Dissection and Digestion of Extensor Digitorum Longus (EDL) from <em>Mus musculus</em> (1.5-2 h)</li><li>Myofiber RNA Extraction (30 min)</li><li>cDNA Library Preparation (3 h)</li><li>Purification of cDNA (30 min)</li><li>Preparing Sequencing ready libraries (2.5 h)</li><li>Purification and Size Selection (30 min)</li><li> Quality Control (2-3 h)<br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwone"><li>Dissection and digestion of Extensor Digitorum Longus (EDL) from <em>Mus musculus</em> <ol class="npwtwo"><li>Prepare a digestion buffer of 1,000 U/ml of collagenase in unsupplemented DMEM and filter the buffer with a 0.22 micron filter. 800 Âµl of digestion buffer is required per EDL muscle.</li><li>Aliquot 800 Âµl of the digestion buffer to a 1.5 ml tube and place in an incubator, with the caps closed, at 37 Â°C and 5% CO<sub>2</sub> for 30 min before dissection to warm up the buffer and infuse it with CO<sub>2</sub>.<br/><em>Notes:</em> <ol class="npwthree"><li><em>The Myofibers are very sensitive to oxygen, all buffers and media used on them should be first placed in the incubator for at least 30 min. When working outside the incubator do not do so for longer than 5 min at a time if possible. If more time is needed to work on the fibers, allow them to recover in the incubator for 10 min before reusing them.</em> </li><li><em>Sacrifice the mouse by anesthetizing it with 5% isoflurane, in an induction chamber, and chemically inducing death with CO</em><sub><em>2</em></sub><em>, followed by cervical dislocation.</em> </li></ol></li><li>Remove the skin on the hindlimb of the mouse with a pair of scissors by cutting the skin around the ankle and making an incision along the ventral side of the leg.</li><li>Using a pair of sharp forceps, remove the epimysium covering the Tibialis Anterior (TA) muscle.</li><li>Cut the TA tendon at the ankle, being careful to only cut the top tendon as the bottom tendon belongs to the Extensor Digitorum Longus (EDL) (Figure 1A).</li><li>Using forceps, gently peel off the TA up the leg until you hit the knee, exposing the EDL. Cut out the TA as close to the knee as possible (Figure 1B).</li><li>The biceps femoris is covering the proximal EDL tendon at the patella and must be removed to expose the tendon (Figure 1A).</li><li>Slide a pair of sharp forceps underneath the EDL and gently run it along its length to ensure that both tendons are exposed and accessible (Figure 1B).</li><li>Cut the distal EDL tendon at the ankle. Lift the EDL upright by the tendon with a pair of forceps and, without pulling too hard, cut the proximal tendon as close to the knee as possible.<br/><em>Notes:</em> <ol class="npwthree"><li><em>It is very important to cut the EDL at the tendons, cutting into the muscle will result in most of the fibers being lost.</em> </li><li><em>It should take about 5 minutes to dissect an EDL.</em> </li><li><em>A video demonstrating the EDL isolation procedure has been made available by Pasut </em><em>et al.</em><em> (2013).</em> </li></ol></li><li>Place the dissected EDL into the digestion buffer and incubate for an hour in the incubator, with the cap closed, while periodically inverting the tube to mix the solution. Trypsin can be added to the digestion buffer at a final concentration of 0.25% at this moment if the removal of fiber-associated satellite cells is needed.</li><li>During the digestion process, coat a 6-well plate with enough coating media (10% horse serum (HS) in unsupplemented DMEM) to cover the bottom of the wells (approximately 3-4 ml), for at least 30 min.<br/><em>Note: Myofibers are very sticky and will become stuck to any surface that is not coated with horse serum.</em> </li><li>Replace the coating media with 2 ml of unsupplemented DMEM and place the plate in the incubator for at least 30 min.</li><li>Coat a large bore glass pipette with HS and rinse with coating media. Use the large bore pipette to transfer the EDL from the digestion buffer to the 6-well plate, being careful to transfer as little of the digestion buffer as possible with it.<br/><em>Notes:</em> <ol class="npwthree"><li><em>At this point, the EDL muscle is still intact and will be at the bottom of the tube. It helps to invert the tube to dislodge the EDL from the bottom and then use the large bore pipette to pick up the EDL. The EDL will sink to the tip of the pipette and can then be transferred to the coated 6-well plate in a single drop of digestion buffer.</em> </li><li><em>A large bore glass pipette can be made by cutting a Pasteur pipette with a diamond pen where the area begins to widen, and then flame polishing the edges with a Bunsen burner (Figure 1C).</em> </li></ol></li><li>Disassociate the fibers by gently pipetting the EDL up and down with the large bore pipette. Transfer the EDL to subsequent wells as fibers are isolated to avoid damaging them (Figures 1D-1E).<br/><em>Note: Typically over 300 intact fibers will be isolated from a single EDL.</em><br/><br/><img alt="" class="layerphoto" height="101" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200213/20200213172834_1163.jpg" width="100"/><br/><strong>Figure 1. Representative Results of Fiber Isolation.</strong> A. Photo of the hindlimb of a 5-week old mouse, highlighting the location of the TA muscle and the tendons of the biceps femoris, TA and EDL muscles. B. Photo of the hindlimb of a 5-week old mouse, with the TA removed exposing the EDL muscle. C. Photo of glass pipettes used for the disassociation of the EDL myofibers. 1. Uncut glass pipette, the black band indicates where to cut with a diamond pen. 2. Large bore glass pipette. 3. Small bore glass pipette. D. Microscope image of an unsuccessful myofiber isolation. E. Microscope image of a successful myofiber isolation. F. Microscope image of a live fiber in a microtube. G. Microscope image of a single myofiber in a microtube after RNA extraction. Scale bars = 1 mm (in D and E). Scale bars = 200 Âµm (F and G)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>RNA Extraction from a Single Myofiber<ol class="npwtwo"><li>Coat a 6-well plate with coating media and then replace it with 2 ml of PBS. Place the plate in the incubator for at least 30 min.</li><li>Coat a small-bore glass pipette with HS and rinse with coating media. Use the small-bore pipette to transfer fibers to the PBS filled 6-well plate (Figure 1C).<br/><em>Note: A small bore glass pipette can be made by using a Bunsen burner to bend the thin portion of a Pasteur pipette into a curve. Flame polish the edges of the opening (Figure 1C).</em> </li><li>Using a small-bore pipette under a microscope, transfer a single fiber to a 200 Âµl PCR tube and remove the PBS with a P200 pipette(Figure 1F). Visually confirm that only a single fiber is in the PCR tube by observing the tube under a microscope.</li><li>Prepare a 10x SMART reaction buffer (Recipe 3).</li><li>Lyse the fiber with 10 Âµl of 1x reaction buffer. Pipette the fiber up and down for 3 min with a P20 pipette. Put the tube on ice for 5 min and vortex as needed (Figure 1G).<br/><em>Note: The fiber will never fully breakdown, but it does so enough for the RNA to be extracted in sufficient quantity.</em> </li><li>To remove the residual fiber pieces, spin down the sample and, while visualizing the fiber pieces under a microscope, transfer the supernatant to a new 200 Âµl PCR tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparing cDNA library<ol class="npwtwo"><li>Bring the volume of the sample up to 11.5 Âµl with RNase free water.</li><li>Using the DNA SMART-Seq HT kit perform the following steps:<br/>a.Add 1 Âµl of 3â² SMART-Seq CDS primer II A to the sample.<br/>b.Mix by pipetting or gently vortexing and spin down.</li><li>Incubate for 3 min at 72 Â°C, then place immediately on ice.</li><li>Prepare template switching master mix. For one reaction use the following reagents: 0.7 Âµl nuclease free water, 8 Âµl one-step buffer, 1 Âµl SMART-Seq HT oligonucleotide, 0.5 Âµl RNase inhibitor, 0.3 Âµl SeqAmp DNA polymerase, 2 Âµl SMARTscribe reverse transcriptase, total volume 12.5 Âµl per reaction.<br/><em>Note: Add the enzymes last to the master mix and use immediately.</em> </li><li>Add 12.5 Âµl per reaction. Mix and spin down.</li><li>Place samples in a thermal cycler using the following settings:<br/>42 Â°C 90 min<br/>95 Â°C 1 min<br/>98 Â°C 10 s<br/><p class="ppzz" style="margin-top:-15px;">65 Â°C 30 s      <img alt="" class="layerphoto" height="50" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200213/20200213173133_0472.jpg" width="18"/> 10-12 cycles     </p><p class="ppzz" style="margin-top:-15px;">68 Â°C 3 min</p>72 Â°C 10 min<br/>4 Â°C forever<br/><em>Note: This is a stopping point. cDNA can be left at 4 Â°C overnight or at -20 Â°C until purification.</em> <div style="white-space:nowrap;"><br/></div></li></ol></li><li>cDNA Purification<ol class="npwtwo"><li>Add AMPure XP beads directly into the PCR tube at a 1:1 (v/v) ratio. Mix thoroughly and allow the DNA to bind to the beads at room temperature for 8 min (Figure 2A).<br/><em>Note: Allow AMPure XP beads to warm to room temperature before pipetting. Vortex AMPure XP beads vigorously before use.</em> </li><li>Place the tubes on a magnetic strip and wait for the solution to clear, approximately 5 min (Figure 2B).</li><li>Using a P200 pipette, discard the supernatant while being careful to not remove the magnetic beads. Wash the beads twice with 200 Âµl of 80% ethanol for 20 s.<br/><em>Note: On the last wash, make sure to remove all the ethanol and use an aspirator to remove any residual droplets on the side of the tube, if necessary.</em> </li><li>Elute the DNA using 17 Âµl of elution buffer provided with the SMART-Seq HT kit. Add the buffer to the tube, mix thoroughly until the beads go into solution, and incubate at room temperature for 5 min.<br/><em>Note: Be careful not to allow your beads to over dry. When dried the DNA becomes difficult to elute, resulting in low yield. If the beads are dry place them in elution buffer at 45 Â°C and vortex periodically until no clumps are visible and the beads do not settle at the bottom of the tube.</em> </li><li>Place the tubes on a magnetic strip until the solution clears and keep the supernatant (Figure 2B).<br/><em>Note: This is a stopping point. cDNA can be kept at 4 Â°C or -20 Â°C.</em><br/><br/><img alt="" class="layerphoto" height="132" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200213/20200213174239_7989.jpg" width="100"/><br/><strong>Figure 2. Representative Results in Preparing Sequencing Ready Libraries.</strong> A. Photo of a 0.2 ml microtube with AMPure beads in solution. B. Photo of a 0.2 ml microtube with AMPure beads separated out of solution with a magnetic rack. C. Agarose gel of appropriately-sized sequencing ready libraries. A 100 bp DNA ladder is shown in lane 1. D. Typical bioanalyzer profiles of RNA-Seq libraries made from single fibers.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparing Sequencing ready libraries<ol class="npwtwo"><li>Quantify the samples using Quant-IT PicoGreen dsDNA assay kit.<br/><em>Note: Typically we obtain a total cDNA yield between 20 and 100 ng.</em> </li><li>Prepare 250 pg of single fiber cDNA in 1.25 Âµl of water in a microtube.<br/><em>Note: It is important to accurately quantify the cDNA concentration. The amount of starting material can alter the library fragment size. Too much DNA results in larger fragments while too little gives small fragments that will be lost during size selection.</em> </li><li>Using the Nextera XT DNA Library Preparation kit, add to the mix 2.5 Âµl of TD buffer, and 1.25 Âµl of ATM, mix and spin down.</li><li>Incubate at 55 Â°C for 5 min.</li><li>Remove from heat and add 1.25 Âµl of NT, mix and spin down.<br/><em>Note: Add NT buffer directly after the tagmentation reaction (Step E4) to deactivate the Tn5 transposase, since the enzyme is still active at this point.</em> </li><li>Incubate the sample for 5 min at RT.</li><li>To amplify your libraries, add 1.25 Âµl of an i7 adaptor and 1.25 Âµl of an i5 adaptor and 3.75 Âµl of NPM. Vortex and spin down.<br/><em>Note: When incorporating sequencing adapters, care must be taken to make sure the choice of i5 and i7 adaptors is color-balanced for the type of sequencer used (two vs four channel). Briefly, the goal is to have signal in both color channels (red and green) at each nucleotide position for the pool of adapters. For example, on a NextSeq 500, which is a two-channel system, T appears as red, C appears as green, A appears in both color channels and G is colorless. In this case, at a single nucleotide position, a G must be pooled with at least an A, or a mix of T and C. In the case of a four-channel system, the same rules apply, but A and C both appear as red, and T and G both appear as green. For further information of sequencing pooling on both two- and four-channel systems, visit:</em><br/><a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/illumina-adapter-sequences-1000000002694-11.pdf" target="_blank"><em>https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/illumina-adapter-sequences-1000000002694-11.pdf</em></a><br/><a href="https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/index-adapters-pooling-guide-1000000041074-05.pdf" target="_blank"><em>https://support.illumina.com/content/dam/illumina-support/documents/documentation/chemistry_documentation/experiment-design/index-adapters-pooling-guide-1000000041074-05.pdf</em></a> </li><li>Run the PCR program using the following steps:<br/>72 Â°C for 3 min<br/>95 Â°C for 30 s<br/>95 Â°C for 10 s<br/><p class="ppzz" style="margin-top:-10px;">55 Â°C for 30 s      <img alt="" class="layerphoto" height="50" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200213/20200213173133_0472.jpg" width="18"/> 12 cycles     </p><p class="ppzz" style="margin-top:-15px;">72 Â°C for 30 s</p>72 Â°C for 5 min<br/>10 Â°C forever<br/><em>Note: This is a stopping point.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Purification and size selection<ol class="npwtwo"><li>Add AMPure XP beads directly into the PCR tube at a 0.85x (v/v) ratio to the volume of sample. Mix thoroughly and incubate at room temperature for 8 min.<br/><em>Note: This ratio will remove fragments smaller than 200 bp.</em> </li><li>Place the tubes on a magnetic strip and wait for the solution to clear, approximately 5 min.</li><li>Discard the supernatant. Wash the beads twice with 200 Âµl of 80% ethanol for 20 s.<br/><em>Note: On the last wash, make sure to remove all the ethanol and use an aspirator to remove droplets on the side of the tube, if necessary.</em> </li><li>Elute the DNA fragments by adding 20 Âµl of resuspension buffer that is provided with the Nextera XT kit and mixing thoroughly until the beads go into solution. Incubate at room temperature for 5 min.</li><li>Place the tubes on the magnetic strip until the solution clears and keep the supernatant.</li><li>Quantify your samples using Quant-IT PicoGreen dsDNA assay kit.<br/><em>Note: Typically we have a final concentration between 4-7 ng/Âµl.</em> </li><li>Store at -20 Â°C until ready for sequencing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Quality control<ol class="npwtwo"><li>Run a qPCR of marker genes after the preparation of the cDNA libraries to assess the quality of the RNA.<br/><em>Note: Here we used Myf6 as a positive control and Pax7 as a negative control. However, any genes that are expressed in myofibers can be used as a positive control and any non-expressed genes can be used for a negative control.</em> </li><li>To verify tagmentation and library size, run 20 ng of sample on a 1.25% agarose gel with dsGreen (Figure 2C).<br/><em>Note: Run only 2 Âµl of DNA ladder that has been diluted 1:2 before use to prevent washing out the signal from the library.</em> </li><li>Run the sequencing ready libraries on a bioanalyzer to assess the size of the libraries and the concentration (Figure 2D).</li></ol></li></ol><p class="pptt" id="biaoti31347">### Data analysis</p><p class="ppzz">Follow standard RNA-Seq analysis pipeline.</p><p class="pptt" id="biaoti31348">### Recipes</p><ol class="npwtwok"><li>Digestion buffer<br/><ol class="npwthree"><li>Dissolve collagenase in DMEM at a concentration of 1,000 U/ml</li><li>Filter the solution with a syringe and 0.22 Âµm filter</li><li>Add trypsin to a final concentration of 0.25%</li><li>Use fresh and do not store</li></ol></li><li>Coating media<br/><ol class="npwthree"><li>Add horse serum (HS) to DMEM at a final concentration of 10%</li><li>Coating media can be stored at 4 Â°C and reused multiple times</li></ol></li><li>SMART-Seq reaction buffer<ol class="npwthree"><li>Prepare 10x reaction buffer by adding 1 Âµl of RNase inhibitor to 19 Âµl 10x lysis buffer which is provided with the SMART-Seq kit</li><li>Prepare 1x buffer by adding 1 Âµl of 10x reaction buffer to 9 Âµl of RNase free water</li></ol></li><li>TAE buffer<ol class="npwthree"><li>Prepare 50x stock solution by dissolving 242 g of tris base, 57.1 ml of glacial acetic acid, and 41.2 g of Na<sub>2</sub>EDTAÂ·2H<sub>2</sub>O in 1 L of dH<sub>2</sub>O</li><li>Dilute 50x stock solution to 1x working solution</li></ol></li></ol><p class="pptt" id="biaoti31349">### Acknowledgments</p><p class="ppzz">Funding for this project was provided by grants from the Canadian Institute of Health Research (CIHR) [PJT-15087] and a Natural Sciences and Engineering Council (NSERC) discovery grant to VDS.<br/>  This protocol benefited from Pasut <em>et al.</em>, 2013 for isolation of single myofibers and Picelli <em>et al.</em> (2014) for SMART-Seq technology.</p><p class="pptt" id="biaoti31350">### Competing interests</p><p class="ppzz">The authors have no conflict of interest to declare.</p><p class="pptt" id="biaoti31351">### Ethics</p><p class="ppzz">Animal experimentation: all procedures used on animals were approved by the McGill University Animal Care Committee (UACC) (protocol #2014-7512, valid 07/01/2017 to 07/01/2020).</p><p class="pptt" id="biaoti31352">### References</p><ol class="npwtwok"><li>Arnold, L., Henry, A., Poron, F., Baba-Amer, Y., van Rooijen, N., Plonquet, A., Gherardi, R. K. and Chazaud, B. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17485518" target="_blank">Inflammatory monocytes recruited after skeletal muscle injury switch into antiinflammatory macrophages to support myogenesis.</a><em> J Exp Med</em> 204(5): 1057-1069.</li><li>Brooke, M. H. and Kaiser, K. K. (1970). <a href="https://www.ncbi.nlm.nih.gov/pubmed/4249441" target="_blank">Three "myosin adenosine triphosphatase" systems: the nature of their pH lability and sulfhydryl dependence.</a> <em>J Histochem Cytochem</em> 18(9): 670-672.</li><li>Christov, C., Chretien, F., Abou-Khalil, R., Bassez, G., Vallet, G., Authier, F. J., Bassaglia, Y., Shinin, V., Tajbakhsh, S., Chazaud, B. and Gherardi, R. K. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17287398" target="_blank">Muscle satellite cells and endothelial cells: close neighbors and privileged partners.</a> <em>Mol Biol Cell</em> 18(4): 1397-1409.</li><li>Giordani, L., He, G. J., Negroni, E., Sakai, H., Law, J. Y. C., Siu, M. M., Wan, R., Corneau, A., Tajbakhsh, S., Cheung, T. H. and Le Grand, F. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30922843" target="_blank">High-dimensional single-cell cartography reveals novel skeletal muscle-resident cell populations.</a> <em>Mol Cell</em> 74(3): 609-621 e606.</li><li>Joe, A. W., Yi, L., Natarajan, A., Le Grand, F., So, L., Wang, J., Rudnicki, M. A. and Rossi, F. M. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20081841" target="_blank">Muscle injury activates resident fibro/adipogenic progenitors that facilitate myogenesis.</a> <em>Nat Cell Biol</em> 12(2): 153-163. </li><li>Pasut, A., Jones, A. E. and Rudnicki, M. A. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23542587" target="_blank">Isolation and culture of individual myofibers and their satellite cells from adult skeletal muscle.</a> <em>J Vis Exp</em> (73): e50074.</li><li>Pette, D., Peuker, H. and Staron, R. S. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10468663" target="_blank">The impact of biochemical methods for single muscle fibre analysis.</a> <em>Acta Physiol Scand</em> 166(4): 261-277.</li><li>Pette, D. and Staron, R. S. (1997). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9002237" target="_blank">Mammalian skeletal muscle fiber type transitions.</a> <em>Int Rev Cytol</em> 170: 143-223.</li><li>Picelli, S., Faridani, O. R., Bjorklund, A. K., Winberg, G., Sagasser, S. and Sandberg, R. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24385147" target="_blank">Full-length RNA-seq from single cells using Smart-seq2.</a> <em>Nat Protoc</em> 9(1): 171-181.<br/></li></ol>