<body>
            <h1>Assessing Self-interaction of Mammalian Nuclear Proteins by Co-immunoprecipitation</h1>
            <p><strong>Authors:</strong> Claudia  Cattoglio, Iryna  Pustova, Xavier  Darzacq, Robert  Tjian, Anders S. Hansen</p>
            <h2>Abstract</h2>
            <p>Protein-protein interactions constitute the molecular foundations of virtually all biological processes. Co-immunoprecipitation (CoIP) experiments are probably the most widely used method to probe both heterotypic and homotypic protein-protein interactions. Recent advances in super-resolution microscopy have revealed that several nuclear proteins such as transcription factors are spatially distributed into local high-concentration clusters in mammalian cells, suggesting that many nuclear proteins self-interact. These observations have further underscored the need for orthogonal biochemical approaches for testing if self-association occurs, and if so, what the mechanisms are. Here, we describe a CoIP protocol specifically optimized to test self-association of endogenously tagged nuclear proteins (self-CoIP), and to evaluate the role of nucleic acids in such self-interaction. This protocol has proven reliable and robust in our hands, and it can be used to test both homotypic and heterotypic (CoIP) protein-protein interactions.</p>
            <p><strong>Keywords:</strong> Immunoprecipitation, IP, Co-immunoprecipitation, CoIP, Self-interaction, Cas9-mediated endogenous tagging</p>
            <h2>Background</h2><p>Proteins are the building blocks of all living organisms, where they serve disparate functions that include structural, enzymatic, signaling and regulatory roles. Proteins perform most of these tasks by interacting with other proteins, small molecules (<em>e.g.</em>, hormones) and macromolecules (<em>e.g.</em>, nucleic acids, carbohydrates and lipids), with a wide range of affinities. In addition to these heterotypic interactions, several proteins are capable of homo-oligomerization (Paris <em>et al.</em>, 2003; Akoev <em>et al.</em>, 2004; Baisamy <em>et al.</em>, 2005; Hwang <em>et al.</em>, 2011; Chen <em>et al.</em>, 2014; Kang <em>et al.</em>, 2017), and perturbation of self-assembly capabilities can lead to pathological outcomes (Bourdenx <em>et al.</em>, 2017; Yuan <em>et al.</em>, 2018; Castle <em>et al.</em>, 2019; Loughlin and Wilce, 2019). Our research is mainly focused on eukaryotic transcription factors (TFs), proteins that initiate and regulate gene transcription. Thanks to recent advances in super-resolution and single-molecule imaging, we are now appreciating that several TFs are not homogenously distributed in the nucleus, but rather locally concentrated in clusters (also hubs or condensates), and in rapid exchange between them (Hansen <em>et al.</em>, 2017; Boehning <em>et al.</em>, 2018; Cho <em>et al.</em>, 2018; Chong <em>et al.</em>, 2018; Dufourt <em>et al.</em>, 2018; Mir <em>et al.</em>, 2018). These clusters can emerge and be largely maintained by direct protein-protein self-association (<em>e.g.</em>, cohesin self-interaction [Cattoglio <em>et al.</em>, 2019]) or by a combination of protein and nucleic acid interactions (<em>e.g.</em>, RNA-mediated CTCF self-association [SaldaÃ±a-Meyer <em>et al.</em>, 2014; Hansen <em>et al.</em>, 2019]).<br/> <br/>While a few computational tools exist that predict protein self-interaction potential (Liu <em>et al.</em>, 2013; Li <em>et al.</em>, 2017; Zhai <em>et al.</em>, 2017; Wang <em>et al.</em>, 2018), these are most likely blind to nucleic-acid mediated homo-oligomerization, and of course demand biochemical validation. Available assays to probe protein self-interactions <em>in-vitro</em> include size-exclusion chromatography, microchip self-interaction chromatography and biochemical fractionation after density gradient centrifugation, all of which require purified recombinant proteins (GarcÃ­a <em>et al.</em>, 2003; Yusufzai <em>et al.</em>, 2004; SaldaÃ±a-Meyer <em>et al.</em>, 2014). Yeast or bacterial two-hybrid systems are an alternative <em>in vivo</em> genetic approach (Yusufzai <em>et al.</em>, 2004; Kang <em>et al.</em>, 2017), but again they do not probe self-association in the native protein environment. Co-immunoprecipitation assays (CoIPs) are by far the most common way to test heterotypic protein-protein interactions occurring in the relevant cell type. A few genetic engineering options are available to also detect homotypic protein-protein interactions by CoIP (Figure 1). One possibility is to overexpress a tagged version of the protein of interest (POI) in a given cell line and distinguish it from the endogenous one by virtue of its bigger size (Figure 1A). After immunoprecipitation with an antibody against the tag epitope, a Western blot is performed using an antibody recognizing both the tagged and the wild type protein. The appearance of a wild type band in the immunoprecipitated sample is considered evidence of self-interaction (Pant <em>et al.</em>, 2004; He <em>et al.</em>, 2008; Kang <em>et al.</em>, 2017). This approach has several drawbacks: it uses an exogenously expressed protein, which can alter its physiological behavior, and it cannot exclude that the observed wild type band is simply a degradation product of the tagged protein. To address the latter, a more robust approach is perhaps to transfect two constructs, each expressing the POI tagged with a distinct epitope (âTAG1â and âTAG2â in Figure 1B). In this case, if the TAG1-tagged protein also pulls down the TAG2-protein, it is safe to conclude that the POI does self-interact. Alternatively, in a mixed <em>in-vivo</em> and <em>in-vitro</em> approach, a cell lysate overexpressing the POI tagged with one epitope is incubated with a recombinant POI tagged with an alternative epitope (SaldaÃ±a-Meyer <em>et al.</em>, 2014). This method has the advantage of allowing easy screening of truncated versions of the POI to identify the regions responsible for self-association, but again relies on overexpression and requires purified proteins. An imaging-based approach, alternative to CoIP, to assess self-interaction of exogenously expressed proteins is the bimolecular fluorescence complementation assay, where the POI is fused to one or the other half of a split fluorescent protein (<em>e.g.</em>, GFP or YFP) (Miller <em>et al.</em>, 2015). Only when the protein self-associates is the fluorescent protein reconstituted and emits fluorescence (Kang <em>et al.</em>, 2017). This assay does not overcome the overexpression concerns, and it is additionally limited by the irreversibility of the fluorescence reconstitution reaction, which can introduce biases and makes the method blind to the dynamics of the self-interaction.<br/><br/><img alt="" class="layerphoto" height="125" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218173113_2995.jpg" width="100"/><br/><strong>Figure 1. Testing protein self-interaction by CoIP.</strong> Different strategies to genetically engineer cells and test the self-association of the POI by co-immunoprecipitation. A. A single plasmid encoding the tagged POI is transfected into cells in addition to the wild type protein expressed from the endogenous promoter (1). If the POI self-interacts, the wild type POI will associate both with itself (not shown) and with the exogenously expressed tagged version (2). A CoIP experiment that uses an antibody directed against the tag will also pull-down the wild type POI (3). If the tag is large enough, the tagged POI can be distinguished from the wild type one by size in a Western blot experiment using an antibody against the POI (4). B. Same as in (A) but this time two separate plasmids are transfected, each encoding the POI tagged with different epitopes (TAG1 and TAG2) (1). The immunoprecipitated material is subject to SDS-PAGE followed by Western blotting and self-interaction is assessed with antibodies against each tag (4). C. Our strategy to tag the endogenous POI with 2 different epitopes. A CRISPR/Cas9 targeting construct (for the expression of both Cas9 and the gRNA of choice) is transfected along with two donor plasmids, each containing the tag of choice (TAG1 or TAG2) flanked by genomic sequences homologous to the endogenous POI (LHR, left homology region, and RHR, right homology region) (1). Clones are selected and characterized that contain each endogenous allele tagged with one of the two epitopes (1â). If the POI self-interacts (2), CoIP experiments with antibodies against the TAG1 will also pulldown the TAG2-POI (3). The immunoprecipitated material is subject to SDS-PAGE followed by Western blotting and self-interaction is assessed using antibodies against each tag (4). Nucleases specific for either RNA (RNase), DNA (DNase) or both (benzonase) can be included in the experiment to test whether the self-association relies on protein-protein interactions and/or is mediated by nucleic acids. IN, input lysate; TAG IP, immunoprecipitated material.<br/><br/> Our approach to test self-association of TFs or other nuclear proteins expressed at physiological concentrations is to edit the genome of the cell line of interest via CRISPR/Cas9 (Reference 1; Roberts <em>et al.</em>, 2017; Aird <em>et al.</em>, 2018; Haupt <em>et al.</em>, 2018; Sharma <em>et al.</em>, 2018; Tasan <em>et al.</em>, 2018; Zhang <em>et al.</em>, 2018; Liu <em>et al.</em>, 2019) and introduce two distinct tags in each allele encoding that TF (typically FLAG-Halo and V5-SNAP<sub>f</sub>) (Cattoglio <em>et al.</em>, 2019; Hansen <em>et al.</em>, 2019) (Figure 1C). The dual tagging can be attempted by providing 2 separate homology repair constructs together in parallel with CRISPR/Cas9, but we generally have higher success rate tagging each allele sequentially in a separate experiment, by using heterozygous clones from the first tagging and re-targeting the âwild typeâ allele (in fact, this is often a pseudo-wt allele, where Cas9 introduced indels at the cut site; this can be exploited to design another set of guide RNAs that will specifically target the untagged allele). Using the dual-tagged alleles we can then probe whether the TF self-interacts by co-immunoprecipitation (self-CoIP), and further investigate whether such self-association is mediated by protein-protein interactions and/or nucleic acids performing the experiment with or without specific nucleases. We provide here detailed instructions to perform the self-CoIP, while you can refer to our previous publications for the endogenous dual tagging strategy (Cattoglio <em>et al.</em>, 2019; Hansen <em>et al.</em>, 2019). This method cannot distinguish between a direct and an indirect self-interaction, and because it requires cell lysis and long incubation times, it cannot accurately estimate the amount of self-association happening in live cells, but rather give lower bounds. Nevertheless, endogenous tagging avoids artifacts originating from overexpression (Hansen <em>et al.</em>, 2017; Shao <em>et al.</em>, 2018), and, provided that all the deployed antibodies are specific (see Note 1), the self-CoIP method detailed here only requires standard lab equipment and reagents, and it has been working robustly and reproducibly in our hands with several TFs. While this protocol reliably detects self-association of nuclear proteins, we routinely use it to also test heterotypic protein-protein interaction in a standard CoIP setting, both in the nucleus and in the cytoplasm (see Note 2).</p>
        </body>

<p class="pptt" id="biaoti31367">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips<br/></li><li>150-mm TC-treated culture dishes (Corning<sup>TM</sup> 430599, Thermo Fisher Scientific, catalog number: 08-772-24)</li><li>Cell scraper (Thermo Fisher Scientific, Fisherbrand<sup>TM</sup>, catalog number: 08-100-241)</li><li>Cuvettes (Thermo Fisher Scientific, Fisherbrand<sup>TM</sup>, catalog number: 14-955-127)</li><li>MagneSphere<sup>Â®</sup> technology twelve position, 1.5-ml magnetic separation stand (Promega, catalog number: Z5342)</li><li>Empty gel cassettes, mini, 1.0 mm (Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: NC2010)</li><li>Empty gel cassette combs, mini, 1.0 mm, 10 well (Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: NC3010)</li><li>Amersham<sup>TM</sup> Protran<sup>Â®</sup> Western blotting membrane (Millipore Sigma, catalog number: GE10600041), Store at room temperature. Shelf-life: specified by manufacturer)</li><li>3 MM Blotting paper (Whatman, catalog number 3030-917)</li><li>Falcon<sup>TM</sup> Round-bottom polypropylene 5-ml tubes (Corning<sup>TM</sup> 352063, Thermo Fisher Scientific, catalog number: 14-959-11A)</li><li>50-ml conical tube (Corning<sup>TM</sup> 430291, Thermo Fisher Scientific, catalog number: 05-538-55A)</li><li>Razor blades (VWR<sup>Â®</sup>, catalog number: 55411-050)</li><li>Steriflip-GP sterile centrifuge tube top filter (Millipore Sigma, catalog number: SCGP00525)</li><li>10-ml syringe, Luer-Lock<sup>TM</sup> (Becton Dickinson, catalog number: 302995)</li><li>22G PrecisionGlide<sup>TM</sup> needle (Becton Dickinson, catalog number: 305156)</li><li>1.5-ml low retention microcentrifuge tube (Phenix, catalog number: MH-815S)</li><li>12 x 24-mm round-bottom glass tube (Covaris<sup>Â®</sup>, catalog number: 520056)</li><li>Disposable sterile bottle-top filters with 0.22 Î¼m membrane (Corning<sup>TM</sup> 430513â500 ml or 430626â150 ml, Thermo Fisher Scientific, catalog number: 09-761-52 or 09-761-56)</li><li>CL-XPosure film (Thermo Fisher Scientific, catalog number: 34091)</li><li>Antibodies for the illustrative experiment detailed below:<br/>a.Mouse normal IgGs (Abcam, catalog number: ab37355)<br/>b.Mouse Î±FLAG (Millipore Sigma, catalog number: F3165)</li><li>HEPES (Thermo Fisher Scientific, Fisher BioReagents, catalog number: BP310500), store at room temperature. Shelf-life: specified by manufacturer</li><li>Hydrochloric acid (HCl, Thermo Fisher Scientific, catalog number: A144-500)</li><li>Potassium chloride (KCl, Thermo Fisher Scientific, Fisher Chemical, catalog number: P217-10), store at room temperature. Shelf-life: specified by manufacturer</li><li>Potassium hydroxide (KOH, Thermo Fisher Scientific, Fisher Chemical, catalog number: P250-500), store at room temperature. Shelf-life: specified by manufacturer</li><li>Sodium hydroxide pellets (NaOH, Thermo Fisher Scientific, Fisher Chemical, catalog number: S318-100), store at room temperature. Shelf-life: specified by manufacturer</li><li>Magnesium chloride hexahydrate (MgCl<sub>2</sub>Â·6H<sub>2</sub>O, Thermo Fisher Scientific, Fisher Chemical, catalog number: M33-500), store at room temperature. Shelf-life: specified by manufacturer</li><li>Sodium chloride (NaCl, Thermo Fisher Scientific, catalog number: S271-10), store at room temperature. Shelf-life: specified by manufacturer</li><li>2-Propanol (Thermo Fisher Scientific, catalog number: A451-4), store at room temperature. Shelf-life: specified by manufacturer</li><li>D-Sucrose (C<sub>12</sub>H<sub>22</sub>O<sub>11</sub>, Thermo Fisher Scientific, Fisher BioReagents, catalog number: BP220-212), store at room temperature. Shelf-life: specified by manufacturer</li><li>Glycerol (C<sub>3</sub>H<sub>8</sub>O<sub>3</sub>, Thermo Fisher Scientific, Fisher Chemical, catalog number: G33-20), store at room temperature. Shelf-life: specified by manufacturer</li><li>DTT (C<sub>4</sub>H<sub>10</sub>O<sub>2</sub>S<sub>2</sub>, GOLDBIO, catalog number: DTT10), storage temperature: -20 Â°C. Shelf-life: specified by manufacturer</li><li>cOmplete<sup>TM</sup>, EDTA-free protease inhibitor cocktail (Millipore Sigma, Roche, catalog number: 11873580001), storage temperature: 4 Â°C (tablets), -20 Â°C (dissolved, see Recipes)</li><li>Triton<sup>TM</sup> X-100 (Millipore Sigma, catalog number: T9284), store at room temperature. Shelf-life: specified by manufacturer</li><li>Aprotinin (Millipore Sigma, catalog number: A6279), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer</li><li>20 mg/ml Bovine serum albumin (BSA) (New England Biolabs, catalog number: B9000S), storage temperature: -20 Â°C. Shelf-life: specified by manufacturer. Use to create a BSA standard for protein quantification</li><li>BSA (Millipore Sigma, catalog number: A7906), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer. Use for preclearing magnetic beads.</li><li>Benzamidine (Millipore Sigma, catalog number: B6506), storage temperature: 4 Â°C (powder), -20 Â°C (solution, see Recipes). Shelf-life: specified by manufacturer</li><li>PMSF (Millipore Sigma, Roche, catalog number: 11359061001), storage temperature: room temperature. Shelf-life: specified by manufacturer</li><li>EDTA (C<sub>10</sub>H<sub>18</sub>N<sub>2</sub>Na<sub>2</sub>O<sub>10</sub>, Thermo Fisher Scientific, Fisher Chemical, catalog number: S311-500), storage temperature: room temperature. Shelf-life: specified by manufacturer</li><li>NP-40 alternative, protein grade detergent, 10% solution (Millipore Sigma, catalog number: 492018), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer</li><li>Benzonase<sup>Â®</sup> nuclease HC (Millipore Sigma, Novagen, catalog number: 71205-3), storage temperature: -20 Â°C. Shelf-life: specified by manufacturer</li><li>Bio-Rad protein assay dye reagent (Bio-Rad, catalog number: 5000006), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer</li><li>Dynabeads<sup>TM</sup> Protein A or G (Thermo Fisher Scientific, Invitrogen<sup>TM</sup>, catalog number: 10008D or 10009D), storage temperature: 4 Â°C. Shelf-life: 24 months from date of manufacture</li><li>Precision plus protein<sup>TM</sup> all blue prestained protein standards (Bio-Rad, catalog number: 1610393)</li><li>MOPS (C<sub>7</sub>H<sub>15</sub>NO<sub>4</sub>S, Thermo Fisher Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP308-500), store at room temperature. Shelf-life: specified by manufacturer</li><li>Tris base (C<sub>4</sub>H<sub>11</sub>NO<sub>3</sub>, Thermo Fisher Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP152-5), store at room temperature. Shelf-life: specified by manufacturer</li><li>SDS (Bio-Rad, catalog number: 1610301), store at room temperature. Shelf-life: specified by manufacturer</li><li>Ethanol (Decon Labs, Koptec pure ethanolâ190 Proof, catalog number V1101)</li><li>Bis-Tris (C<sub>8</sub>H<sub>19</sub>NO<sub>5</sub>, Millipore Sigma, catalog number: B9757), store at room temperature. Shelf-life: specified by manufacturer</li><li>Sodium metabisulfite (Na<sub>2</sub>S<sub>2</sub>O<sub>5</sub>, Millipore Sigma, catalog number: S9000), Store at room temperature. Shelf-life: specified by manufacturer</li><li>TEMED (Bio-Rad, catalog number: 1610800), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer</li><li>Ammonium persulfate (APS) (Bio-Rad, catalog number: 1610700), storage temperature: room temperature (powder), -20 Â°C (solution, see Recipes). Shelf-life: specified by manufacturer (powder), 6 months or more (solution)</li><li>ProtoGel (30%) 37.5:1 Acrylamide to Bisacrylamide solution (national diagnostics, catalog number: EC-890), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer</li><li>Ultrapure<sup>TM</sup> water (Thermo Fisher Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: 10977023), store at room temperature. Shelf-life: specified by manufacturer</li><li>Glycine (C<sub>2</sub>H<sub>5</sub>NO<sub>2</sub>, Thermo Fisher Scientific, Fisher BioReagents<sup>TM</sup>, catalog number: BP381-5), store at room temperature. Shelf-life: specified by manufacturer</li><li>Methanol (CH<sub>4</sub>O, Thermo Fisher Scientific, Fisher Chemical, catalog number: A452-4), store at room temperature. Shelf-life: specified by manufacturer</li><li>Tween<sup>Â®</sup>-20 (Millipore Sigma, Calbiochem, catalog number: 655205), store at room temperature. Shelf-life: specified by manufacturer</li><li>Instant Nonfat dry milk, pasteurized extra grade (Signature Kitchens), store at room temperature. Shelf-life: specified by manufacturer</li><li>2-mercaptoethanol (Millipore Sigma, catalog number: M3148), store at room temperature. Shelf-life: specified by manufacturer</li><li>Bromophenol blue sodium salt (C<sub>19</sub>H<sub>9</sub>Br<sub>4</sub>NaO<sub>5</sub>S, Millipore Sigma, catalog number: B5525), Store at room temperature. Shelf-life: specified by manufacturer</li><li>Potassium phosphate monobasic (Na<sub>2</sub>HPO<sub>4</sub>, Thermo Fisher Scientific, Fisher Chemical, catalog number: P285-500)</li><li>Sodium phosphate dibasic anhydrous (KH<sub>2</sub>PO<sub>4</sub>, Thermo Fisher Scientific, Fisher Chemical, catalog number: S375-212)</li><li>Western lightning Plus-ECL (PerkinElmer, catalog number: NEL103E001EA), storage temperature: 4 Â°C. Shelf-life: specified by manufacturer</li><li>70% ethanol</li><li>6 N HCl (see Recipes)</li><li>10x PBS (see Recipes)</li><li>1x PBS, pH 7.4 (see Recipes)</li><li>1 M HEPES (see Recipes)</li><li>1 M Tris-base pH 6.8/7.5 (see Recipes)</li><li>5 M NaCl (see Recipes)</li><li>2 M KCl (see Recipes)</li><li>1 M MgCl<sub>2</sub> (see Recipes)</li><li>1 M DTT (see Recipes)</li><li>10% Triton X-100 (see Recipes)</li><li>0.5 M EDTA pH 8.0 (see Recipes)</li><li>10% APS (see Recipes)</li><li>50x cOmplete protease inhibitors stock (see Recipes)</li><li>1 M benzamidine (see Recipes)</li><li>0.2 M PMSF (see Recipes)</li><li>1.23 M Bis-Tris pH 6.4/6.7 (see Recipes)</li><li>Cell lysis buffer (see Recipes)</li><li>0.1 M CoIP buffer (see Recipes)</li><li>0.2 M CoIP buffer (see Recipes)</li><li>0.5% BSA in 0.2 M CoIP buffer (see Recipes)</li><li>20x MOPS running buffer (see Recipes)</li><li>Transfer buffer (see Recipes)</li><li>TBS-T (see Recipes)</li><li>4x sample buffer (see Recipes)</li><li>Blocking solution (see Recipes)</li><li>Blotting solution (see Recipes)</li></ol><p class="pptt" id="biaoti31368">### Equipment</p><ol class="npwtwok"><li>-80 Â°C freezer<br/></li><li>Pipettes (P20, P200, P1000)</li><li>DynaMag<sup>TM</sup>-15 magnet (Thermo Fisher Scientific, catalog number: 12301D)</li><li>Focused-ultrasonicator (Covaris<sup>Â®</sup>, model: S220) with 12 x 24-tube adaptor (Covaris<sup>Â®</sup>, catalog number: 500199)<br/><em>Note: You can replace this instrument with any type of sonicator (<em>e.g.</em>, ultrasonic processors). We recommend sonication to solubilize chromatin in the absence of nuclease digestion. You can skip the sonication steps when working with cytosolic proteins (note on Step C5) or if digesting with nuclease the entire sample.</em> </li><li>Refrigerated table-top centrifuge (Eppendorf, model: 5810R)</li><li>Refrigerated microfuge (Eppendorf, model: 5418R)</li><li>UV-VIS Spectrophotometer (Shimadzu, model: UV-1900)</li><li>Fisherbrand<sup>TM</sup> Isotemp<sup>TM</sup> Digital Block Heater (Thermo Fisher Scientific, catalog number: 88-860-021)</li><li>XCell SureLock<sup>TM</sup> Mini-Cell (Thermo Fisher Scientific, catalog number: EI0001)</li><li>Electrophoresis power supply (Thermo Fisher Scientific, model: EC-105); use for SDS-PAGE</li><li>Mini Trans-Blot<sup>Â®</sup> electrophoretic transfer cell (Bio-Rad, catalog number: 1703930)</li><li>PowePac<sup>TM</sup> Basic Power supply (Bio-Rad, catalog number: 1645050); use for transfer</li><li>Clay Adams nutator mixer (Becton Dickinson); use to rock lysates</li><li>Rocker platform (Bellco Biotechnology, model: 7740-10010): use to blot Western blot membranes with antibodies overnight</li><li>Benchtop orbital shaker (Thermo Fisher Scientific, model: MaxQ<sup>TM</sup> 2000)<br/></li></ol><p class="pptt" id="biaoti31369">### Procedure</p><ol class="npwone"><li>Collect cells<ol class="npwtwo"><li>Grow your cell line of interest to 80% confluency.<br/><em>Note: For mouse embryonic stem cells, we typically start with two 150-mm confluent tissue culture plates (~30 x 10<sup>6</sup>-40 x 10<sup>6</sup> total cells). The amount of plates to use will depend on how many cells you can obtain from each plate (e.g., mouse embryonic stem cells grow in 3D colonies and thus give a very good yield) and on the cellular abundance of the protein to be immunoprecipitated.</em> </li><li>For adherent cells:<ol class="npwthree"><li>Aspirate media.</li><li>Wash twice with ice-cold 1x PBS:<br/><ol class="npwfour"><li>Pour enough PBS onto the plate to cover it and swirl it around to remove media residues.</li><li>Aspirate the PBS or simply invert the plate to discard it.</li></ol></li><li>Transfer the plate(s) on ice.<br/><em>Note: From now on, keep samples and buffers in ice all the time, unless otherwise specified.</em> </li><li>Scrape cells with ice-cold 1x PBS added of PMSF, benzamidine and aprotinin (use protease inhibitorsâ stock solutions as 1,000x). We typically use 3-4 ml of PBS per 150-mm plate.</li><li>Transfer scraped cells to 15 ml conical tubes (or 50 ml if the final volume exceeds 12 ml).</li><li>Collect all the remaining cells by washing the scraped plate(s) with another 3-4 ml of 1x PBS with protease inhibitors and transfer to the same 15 ml conical tube(s).</li><li>Pellet cells in a refrigerated table-top centrifuge at 4 Â°C and 300 <em>x g</em> for 10 min.</li><li>Aspirate the supernatant and flash-freeze cell pellets in liquid nitrogen and store at -80 Â°C until ready to proceed.</li></ol></li><li>For cells growing in suspension:<ol class="npwthree"><li>Transfer cells from the culture vessel to a 50 ml conical tube; wash the culture vessel once with 1x PBS to collect all the remaining cells.</li><li>Pellet cells in a table-top centrifuge at 25 Â°C and 300 <em>x g</em> for 5 min.</li><li>Transfer the cell pellet on ice.</li><li>Wash once with 40 ml of ice-cold 1x PBS:<br/><ol class="npwfour"><li>Resuspend the pellet in PBS.</li><li>Pellet in a refrigerated table-top centrifuge at 4 Â°C and 300 <em>x g</em> for 10 min.</li></ol></li><li>Resuspend the pellet in 20 ml of ice-cold 1x PBS added of PMSF, benzamidine and aprotinin.</li><li>Pellet cells in a refrigerated table-top centrifuge at 4 Â°C and 300 <em>x g</em> for 10 min.</li><li>Aspirate the supernatant and flash-freeze cell pellets in liquid nitrogen and store at -80 Â°C until ready to proceed.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Turn on the Covaris sonicator and set at 4 Â°C<br/><em>Note: It takes ~30-45 min for the sonicator to complete degassing and cool down.</em><br/><br/></li><li>Lyse nuclei (all volumes refer to 2x 150-mm plates, or 30-40 million cells)<ol class="npwtwo"><li>Prepare the cell lysis buffer (see Recipe B1; 700 Î¼l for 2x 150-mm plates, or 30-40 million cells), adding protease inhibitors (PMSF, aprotinin, benzamidine, and the Roche complete inhibitorsâ50x stock) and Triton X-100 to 0.1% final concentration (use the 10% stock solution as 100x).</li><li>Resuspend each pellet in 700 Î¼l of the freshly prepared cell lysis buffer using a P1000 pipet.</li><li>Transfer the lysate to low-retention 1.5-ml tubes and rock at 4 Â°C for 8 min to allow lysis (<em>e.g.</em>, using a rotator in the cold room).</li><li>Spin in a refrigerated microfuge at 4 Â°C and 3,000 <em>x g</em> for 3 min.</li><li>Optional: keep the supernatant as the âcytoplasmic fractionâ.<br/><em>Note: This protocol is specifically optimized for nuclear proteins, but you can also perform the self-CoIP on the cytoplasmic fraction.</em> </li><li>Remove the supernatant with a P1000 first, and then with a P200, without disturbing the nuclear pellet.</li><li>Prepare the 0.1 M CoIP buffer (see Recipe B2; 700 Î¼l for 2x 150-mm plates, or 30-40 million cells), adding protease inhibitors (PMSF, aprotinin, benzamidine, and the Roche complete inhibitorsâ50x stock).</li><li>Resuspend each nuclear pellet in 700 Î¼l of 0.1 M CoIP buffer.<br/><em>Note: If you are not interested in performing a nuclease digestion, resuspend the nuclear pellet directly in 0.2 M CoIP buffer, sonicate the lysate and jump from Steps C10 to C17.</em> </li><li>Transfer the resuspended nuclei to 12 x 24-mm Covaris tubes.</li><li>Sonicate each sample (Peak power: 100 W, Cycles/burst: 200, Duty Factor: 20, 8 repeats of 20 seconds of treatment followed by a 40-s delay).</li><li>Measure the samplesâ volume while transferring them back to a 1.5-ml low retention tube.</li><li>Bring the volume to 2 ml with 0.1 M CoIP buffer plus protease inhibitors.</li><li>Equally divide the sample to two tubes, 1 ml to be left untreated and 1 ml to treat with Benzonase (1 Î¼l per ml of lysate).<br/><em>Note: Benzonase digests both DNA and RNA, and thus will tell you whether the self-interaction is mostly mediated by protein-protein interactions or it requires nucleic acids. If the self-association is dependent on nucleic acids, you can test whether RNA or DNA are involved replacing benzonase with either RNaseI or DNaseI (you will need to adjust the digestion buffer for DNaseI).</em> </li><li>Rock at 4 Â°C for 3 to 4 h to achieve complete Benzonase digestion.</li><li>Spin briefly in a refrigerated microfuge at 4 Â°C to pull liquid down.</li><li>Add 5 M NaCl to bring the final salt concentration to 0.2 M (<em>i.e.</em>, 20.4 Î¼l per 1 ml of lysate) and immediately invert the tubes.</li><li>Rock at 4 Â°C for 30 min to allow complete nuclear lysis.</li><li>Spin in a refrigerated microfuge at 4 Â°C and at maximum speed for 20 min.</li><li>Transfer supernatants to new low-retention 1.5-ml tubes and quantify by Bradford assay.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Quantify nuclear lysates<ol class="npwtwo"><li>Turn on the UV/VIS spectrophotometer.</li><li>Prepare enough Bradford reagent to quantify all your lysates plus 6 additional samples to prepare a BSA standard curve (blank, 2.5 Î¼g, 5 Î¼g, 7.5 Î¼g, 12 Î¼g). You will need 1 ml of Bradford per sample (dilute the Bio-Rad protein assay dye reagent 1:5 in ddH<sub>2</sub>O).</li><li>Prepare your BSA standard, adding 47.5 Î¼l of 0.2 M CoIP buffer to 2.5 Î¼l of 20 mg/ml BSA (NEB). This will give you a 1 Î¼g/Î¼l BSA solution.</li><li>Aliquot the Bradford reagent to 1.5-ml tubes as follows:<br/>1 ml (blank)<br/>997.5 Î¼l Bradford + 2.5 Î¼l BSA (2.5-Î¼g standard)<br/>995 Î¼l Bradford + 5 Î¼l BSA (5-Î¼g standard)<br/>992.5 Î¼l Bradford + 7.5 Î¼l BSA (7.5-Î¼g standard)<br/>988 Î¼l Bradford + 12 Î¼l BSA (12-Î¼g standard)<br/>998 Î¼l Bradford + 2 Î¼l sample #1/untreated<br/>998 Î¼l Bradford + 2 Î¼l sample #1/Benzonase<br/>For as many samples as you need to quantify.</li><li>Mix by inverting each tube right after adding BSA/lysate.</li><li>Pour the content of each tube into a plastic cuvette and read the absorbance at 595 nm (<em>Auto Zero</em> the instrument with the blank solution).</li><li>Use the template provided in Supplemental <a href="https://os.bio-protocol.org/attached/file/20191125/Supplemental.xlsx" target="_blank">Table S1</a> to build a standard curve with the BSA readings and to interpolate the concentration of your samples.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Setup your self-CoIP experiment and pre-clear lysates and beads<br/>We are here illustrating a typical self-CoIP experiment to probe the self-association of the cohesin subunit Rad21, and its dependency on nucleic acids. Specifically, we are using nuclear lysates from mouse embryonic stem cells with an endogenously dually tagged Rad21 protein (the B4 clone originally described in Cattoglio <em>et al.</em> (2019). This cell line contains a Rad21-SNAP<sub>f</sub>-3xFLAG allele and a Rad21-Halo-V5 allele (Figure 2A). We will pulldown the protein expressed from the first allele with a FLAG antibody (Figure 2B) and check whether the V5-tagged protein produced from the second allele is also immunoprecipitated using a V5 antibody during the Western blot (Figure 2C), both in untreated lysates and in lysates treated with benzonase to digest nucleic acids. We will also check the total IP efficiency of the experiment blotting against the immunoprecipitated FLAG protein (Figure 2D).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218170457_3059.jpg" width="145"/><br/><strong>Figure 2. Self-CoIP of Rad21.</strong> Experimental workflow and results of the Rad21 self-CoIP assay used here to illustrate the protocol. The data in C and D is modified from the original Figure 2F in Cattoglio <em>et al.</em> (2019). See Procedure E for a detailed description. IN, input nuclear lysates; UT, untreated lysates; Benz: benzonase-treated lysates.<br/><br/><ol class="npwtwo"><li>Create a scheme for your experiment like the following:<br/><u>Sample # (mg needed)</u><br/>Sample 1: mESC/untreated_input (0.2 mg)<br/>Sample 2: mESC/untreated_IgG (1 mg)<br/>Sample 3: mESC/untreated_Î±FLAG (1 mg)<br/>Sample 4: mESC/benzonase_input (0.2 mg)<br/>Sample 5: mESC/benzonase_IgG (1 mg)<br/>Sample 6: mESC/benzonase_Î±FLAG (1 mg)<br/>In this case, you will need 2.2 mg of nuclear lysates per condition (untreated and benzonase-treated).<br/><em>Note: The normal IgG sample (immunoglobulins purified from pre-immune serum) controls for non-specific pulldown and it must be of the same species of your Î±FLAG antibody (e.g., mouse anti-FLAG antibody and normal mouse IgG).</em> </li><li>Dilute the amount of lysates needed for your experiment to 1 mg/ml with 0.2 M CoIP buffer added of protease inhibitors (PMSF, aprotinin, benzamidine and Roche cOmplete inhibitors). If your volume exceeds 1.5 ml, move the diluted lysates to round-bottom tubes (Figure 3a).<br/>In our illustrative experiment:<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td>Condition<br/></td><td>Concentration (<a href="https://os.bio-protocol.org/attached/file/20191125/Supplemental.xlsx" target="_blank">Table S1</a>)<br/></td><td>For 2.2 mg<br/></td><td>0.2 M CoIP buffer to 1 mg/ml<br/></td></tr><tr><td>mESC/untreated<br/></td><td>2.5 mg/ml<br/></td><td>880 Î¼l<br/></td><td>1,320 Î¼l<br/></td></tr><tr><td>mESC/benzonase<br/></td><td>2.8 mg/ml<br/></td><td>786 Î¼l<br/></td><td>1,414 Î¼l<br/></td></tr></tbody></table><br/><img alt="" class="layerphoto" height="121" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218170823_3241.jpg" width="100"/><br/><strong>Figure 3. Step-by-step self-CoIP protocol.</strong> The figure illustrates all the steps required to immunoprecipitate the protein of interest, described in detail in the text.<br/><br/></li><li>Preclear lysates with beads to reduce non-specific binding (note that these beads will be discarded at the end of the preclearing) (Figure 3b).<ol class="npwthree"><li>Wash protein A or G Dynabeads.<br/><em>Note: Choose either protein A or G depending on the species and IgG subtype of your primary antibody (e.g., use protein G for rabbit and mouse IgG2a antibodies and protein A for guinea pig). Check the specification sheet of the magnetic beads manufacturer to confirm your choice.</em> <ol class="npwfour"><li>Resuspend vigorously the Dynabeads until homogenously dispersed.</li><li>For each condition, pipet 20 Î¼l of Dynabeads per mg of lysate (<em>e.g.</em>, 44 Î¼l for 2.2 mg) to a low-retention 1.5 ml tube.</li><li>Resuspend in 500 Î¼l of 0.2 M CoIP buffer.</li><li>Insert the tube in the magnet separation rack and wait for 5 min for the solution to clear.</li><li>Aspirate the CoIP buffer with a pipet, remove the tube from the magnet and repeat the wash a second time (Steps E3a-iii to E3a-v).</li></ol></li><li>Remove the tube from the magnet and use the diluted lysate to resuspend the beads and transfer them to the round-bottom tubes with the rest of your nuclear proteins.</li><li>Rock the lysates with the beads for at least 2 h at 4 Â°C.</li></ol></li><li>Preclear beads with BSA (note that you will use these beads to perform the actual pulldown experiment) (Figures 3e-3f)<ol class="npwthree"><li>Wash protein A or G Dynabeads; you will prepare a single pool of Dynabeads to be used with all your samples.<ol class="npwfour"><li>Resuspend vigorously the Dynabeads until homogenously dispersed.</li><li>In 1.5-ml tubes, pipet 20 Î¼l of Dynabeads per sample (<em>e.g.</em>, 20 Î¼l for 6 samples: 120 Î¼l).<br/><em>Note: You will not pipet Dynabeads in your input samples, but we include inputs when preclearing beads to accommodate pipetting errors later on.</em> </li><li>Resuspend in 1 ml of 0.2 M CoIP buffer.</li><li>Insert the tube in the 1.5-ml magnetic separation stand and wait for 5 min for the solution to clear.</li><li>Aspirate the CoIP buffer with a pipet, remove the tube from the magnet and repeat the wash a second time (Steps E4a-iii to E4a-v).</li></ol></li><li>Remove the tube from the magnet and resuspend the beads in 0.2 M CoIP buffer with 0.5% BSA (see Recipe B4). Use an amount of buffer equivalent to 10 times the volume of your beads (<em>e.g.</em>, 1,200 Î¼l BSA buffer for 120 Î¼l of beads).</li><li>Rock the beads with BSA at 4 Â°C overnight.<br/><em>Note: For convenience, we preclear beads with BSA overnight while incubating the lysates with antibodies, but a couple of hours are also sufficient.</em><br/><br/></li></ol></li></ol></li><li>Add antibodies to the precleared lysates (Figures 3c-3d).<br/><ol class="npwtwo"><li>Briefly spin down the precleared lysates at 4 Â°C to remove liquid drops from the tubesâ lids.</li><li>Remove the snap cap and insert the tubes in the DynaMag<sup>TM</sup> magnetic stand.</li><li>Wait for 5 min for the solution to clear.<br/><em>Note: During this and all subsequent magnetic separations, we like to transfer the magnetic stand in the fridge while waiting for the solution to clear, to maintain samples cold at all times.</em> </li><li>Transfer the supernatant to a new tube without disrupting the beads.</li><li>Prepare as many 1.5-ml low-retention tubes as needed for the IP (<em>e.g.</em>, for our 6 samples).</li><li>Distribute the precleared lysates to the new tubes. In our illustrative experiments this will correspond to:<br/>Samples 2-3 and 5-6: pipet 1 ml (corresponding to 1 mg)<br/>Samples 1 and 4 (inputs): pipet 100-150 Î¼l (corresponding to 0.1-1.15 mg)</li><li>Add the specific antibody (4 Î¼g per mg of lysate) and immediately invert the tube by mixing. In our illustrative experiment:<br/>Sample 1 (mESC/untreated_input): no antibody<br/>Sample 2 (mESC/untreated_IgG): 4 Î¼g of mouse normal IgGs<br/>Sample 3 (mESC/untreated_Î±FLAG): 4 Î¼g of mouse Î±FLAG<br/>Sample 4 (mESC/benzonase_input): no antibody<br/>Sample 5 (mESC/benzonase_IgG): 4 Î¼g of mouse normal IgGs<br/>Sample 6 (mESC/benzonase_Î±FLAG): 4 Î¼g of mouse Î±FLAG</li><li>Rotate/rock the lysates with antibodies at 4 Â°C overnight.<br/><em>Note: We obtain best results when incubating the antibody overnight, but you can test your antibody and try to shorten the time.</em> </li></ol><br/></li><li>Bind antibodies to the precleared magnetic beads (Figures 3g-3i).<ol class="npwtwo"><li>Retrieve the samples and the BSA-precleared beads and briefly spin them down at 4 Â°C to remove liquid drops from the tubesâ lids.</li><li>Place the tube with precleared beads in the magnetic separator rack.</li><li>Wait for 5 min for the solution to clear.</li><li>Remove the tube from the magnet and resuspend beads in 0.2 M CoIP buffer with inhibitors (50 Î¼l per sample; in our illustrative experiment, 50 Î¼l x 6 samples = 300 Î¼l total).</li><li>Distibute 50 Î¼l of resuspended beads to each sample to be immunoprecipitated (inputs excluded) and immediately invert to mix.</li><li>Rotate/rock the lysates with antibodies and beads at 4 Â°C for 2 h minimum.</li><li>Leave input samples on ice.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Wash immunoprecipitated samples (IP samples) to remove non-specific binding (Figure 3j).<ol class="npwtwo"><li>Prepare enough ice-cold 0.2 M CoIP buffer plus protease inhibitors to wash each IP sample 8 times with 400 Î¼l of buffer (<em>e.g.</em>, 4 IP samples x 400 Î¼l x 8 = 12.8 ml of 0.2 M CoIP buffer; inputs are excluded).</li><li>Briefly spin down IP samples at 4 Â°C to remove liquid drops from the tubesâ lids.</li><li>Insert the IP samplesâ tubes in the 1.5-ml magnetic stand.</li><li>Wait for 5 min for the solution to clear. At this point your immunoprecipitated protein is attached to the beads via the antibody.</li><li>Remove the CoIP buffer with a P1000 pipet without disturbing the beads.</li><li>Remove tubes from the magnet and put them back on ice.</li><li>Add 400 Î¼l of 0.2 M CoIP buffer.<br/><em>Note: Make sure that the beads are completely resuspended. We typically vortex at low speed.</em> </li><li>Repeat Steps H4-H7 for 8 washes total.</li><li>After the last wash, also remove any residual buffer with a P20.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Elute immunoprecipitated material and prepare inputs and samples for SDS-PAGE (Figures 3k-3l).<br/><ol class="npwtwo"><li>Remove tubes from the magnet and put them back on ice.</li><li>Elute immunoprecipitated proteins from the beads adding 20 Î¼l of 1x SDS loading buffer (Recipe B8).<br/><em>Note: The elution volume depends on how many samples you have. We are here illustrating an experiment that uses a 10-well acrylamide gel, which can easily accommodate 17 Î¼l. We also use 12- or 15-well gels, and adjust the elution volumes accordingly or load less material.</em> </li><li>Vortex to resuspend.</li><li>Boil the beads in a thermal block set at 98 Â°C for 5 min.</li><li>Spin briefly.</li><li>Place tubes in the 1.5-ml magnetic stand.</li><li>Wait for 5 min. Your eluted proteins are now in the supernatant.</li><li>Transfer the supernatant to a new low-retention 1.5 ml tube on ice (the volume should be slightly more than 20 Î¼l). You will load 17 Î¼l of this undiluted sample on one gel to estimate the self-CoIP efficiency (Figure 2C).</li><li>Prepare a new set of tubes with 18 Î¼l of 1x SDS-loading buffer and add 2 Î¼l of the eluted material. You will load 17 Î¼l of this 1:10 dilution on a second gel to estimate the IP efficiency (Figure 2D).</li><li>Prepare 0.75% inputs to load along with your IP samples.<ol class="npwthree"><li>Mix 22 Î¼l of input lysate with 25 Î¼l of 2x SDS-loading buffer and 3 Î¼l of 0.2 M CoIP buffer.<br/><em>Note: This dilutes your inpute lysate to 0.44 Î¼g/Î¼l. Loading 17 Î¼l of this dilution will give you ~7.5 Î¼g, which correspond to 0.75% of the starting material (1 mg).</em> </li><li>Boil 5 min in a thermal block set at 98 Â°C.<br/></li><li>Spin briefly and place tubes on ice.</li></ol></li></ol><br/></li><li>Evaluate self-CoIP and CoIP efficiency by SDS-PAGE and Western blotting<ol class="npwtwo"><li>Prepare SDS-PAGE acrylamide gels<br/><em>Note: If more convenient, you can purchase pre-cast NuPAGE Bis-Tris gels from Invitrogen.</em> <ol class="npwthree"><li>Choose a gel percentage depending on the size of your POI. Gel migration charts are available on the ThermoFisher Scientific website (our gel recipe behaves like the NuPAGE Bis-Tris pre-casted gels form Invitrogen<sup>TM</sup>) (Reference 33). For Rad21 we use 9% gels.</li><li>Check Table 1 for Recipes.<br/><br/><strong>Table 1. Use this table to prepare your SDS-PAGE gel (all specified volumes are for 1-mm cassettes, 1 gel)</strong><br/><table border="0" cellpadding="0" cellspacing="0" class="MsoNormalTable ke-zeroborder" width="550"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><b>Separating gel</b> </td><td></td><td></td><td></td><td></td><td></td></tr><tr><td style="border:none;"><u>Reagent</u> </td><td style="border:none;" valign="bottom"><u>12%</u> </td><td><u>10%</u> </td><td><u>9.5%</u> </td><td><u>9%</u> </td><td><u>8%</u> </td></tr><tr><td>1.23 M Bis-Tris pH 6.7</td><td>1.8 ml</td><td>1.8 ml</td><td>1.8 ml</td><td>1.8 ml</td><td>1.8 ml</td></tr><tr><td>30% Acrylamide</td><td>2.52 ml</td><td>2.1 ml</td><td>2.0 ml</td><td>1.89 ml</td><td>1.68 ml</td></tr><tr><td>double-distilled H<sub>2</sub>O</td><td>1.93 ml</td><td>2.35 ml</td><td>2.45 ml</td><td>2.56 ml</td><td width="85">2.77 ml</td></tr><tr><td>10% APS</td><td>42 Âµl</td><td>42 Âµl</td><td>42 Âµl</td><td>42 Âµl</td><td width="85">42 Âµl</td></tr><tr><td>TEMED</td><td>12 Âµl</td><td>12 Âµl</td><td>12 Âµl</td><td>12 Âµl</td><td width="85">12 Âµl</td></tr><tr><td>Total volume:</td><td>6.3 ml</td><td>6.3 ml</td><td>6.3 ml</td><td>6.3 ml</td><td>6.3 ml</td></tr><tr style="border-top:1px;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td><b>Stacking gel</b> </td><td></td><td></td><td></td><td></td><td></td></tr><tr><td style="border:none;"><u>Reagent</u> </td><td style="border:none;" valign="bottom"><u>4%</u> </td><td style="border:none;" valign="bottom"></td><td style="border:none;" valign="bottom"></td><td style="border:none;" valign="bottom"></td><td style="border:none;" valign="bottom" width="85"></td></tr><tr><td>1.23 M Bis-Tris pH 6.4</td><td>710 Âµl</td><td></td><td></td><td></td><td width="85"></td></tr><tr><td>30% Acrylamide</td><td>330 Âµl</td><td></td><td></td><td></td><td width="85"></td></tr><tr><td>double-distilled H<sub>2</sub>O</td><td>1.43 ml</td><td></td><td></td><td></td><td width="85"></td></tr><tr><td>10% APS</td><td>20 Âµl</td><td></td><td></td><td></td><td width="85"></td></tr><tr><td>TEMED</td><td>7 Âµl</td><td></td><td></td><td></td><td width="85"></td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>Total volume:</td><td>2.5 ml</td><td></td><td></td><td></td><td></td></tr></tbody></table><br/></li><li>Place two 1-mm gel cassettes in a stable upright position.</li><li>In a 15 ml conical tube, start preparing the separating gel, adding reagents in the order specified in Table 1 (Bis-Tris, acrylamide and water).</li><li>Mix well pipetting up and down and finally add APS and TEMED.</li><li>Prompty mix and pipet into the gel cassettes, leaving enough space for the stacking gel.</li><li>Immediately layer 70% ethanol on top of the gel, using a squirt bottle. This will make your gel front neat and straight.</li><li>Wait for polymerization to complete.<br/><em>Note: When solidified, the gel front will appear as a neat line easily distinguishable from the ethanol layered above.</em> </li><li>Aspirate the ethanol and wash the gel extensively with ddH<sub>2</sub>O using a squirt bottle.</li><li>Aspirate the excess ddH<sub>2</sub>O.</li><li>In a new 15 ml conical tube, start preparing the stacking gel, adding reagents in the order specified in Table 1 (Bis-Tris, acrylamide and water).</li><li>Mix well pipetting up and down and finally add APS and TEMED.</li><li>Prompty mix and pipet into the gel cassette on top of the separating gel, all the way to the top of the cassette.</li><li>Slowly place the selected combs (in our case 10-well combs), making sure no bubbles form below the wells.<br/><em>Note: We normally aspirate the excess stacking gel with a P1000 while inserting the comb, transferring it back to the 15 ml tube where we prepared it, so that no acrylamide gets spilled on the bench.</em> </li><li>Wait for polymerization to complete.</li></ol></li><li>Run the SDS-PAGE<ol class="npwthree"><li>Transfer the gels into a running cassette (each cassette can accommodate 2 gels) and lock them in place.<br/><em>Note: Remember to peel off the white adhesive at the bottom of the gel cassette.</em> </li><li>Dilute the 20x MOPS running buffer according to Recipe B5.<br/><em>Note: Remember to add sodium metabisulfite.</em> </li><li>Remove the combs from the gel cassettes.</li><li>Fill the inside chamber between the two cassettes with running buffer all the way to the top.<br/><em>Note: This will allow you to check whether the cassettes are properly locked or rather the buffer is leaking.</em> </li><li>Gently wash the wells using a syringe with a 22G needle filled with running buffer. Do not disturbe the bottom of the wells.</li><li>Add the rest of the running buffer.<br/><em>Note: 500 ml will fill half of the outer tank or less.</em> </li><li>Decide the loading order of your samples. In our illustrative experiment:<br/><br/><strong>Gel#1 (Î±V5 blot, to evaluate self-CoIP)</strong><br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">empty</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Prestained protein standard</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">mESC/untreated_0.75%_input</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">mESC/untreated_IgG_undiluted</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-0.5px;"><div class="float-left">5)</div><div class="media-body">mESC/untreated_ Î±FLAG_undiluted</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-0.5px;"><div class="float-left">6)</div><div class="media-body">empty</div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-0.5px;"><div class="float-left">7)</div><div class="media-body">mESC/benzonase_0.75%_input</div></div><div class="media media-x cnautokh cnautokh8" style="margin-top:-0.5px;"><div class="float-left">8)</div><div class="media-body">mESC/benzonase_IgG_undiluted</div></div><div class="media media-x cnautokh cnautokh9" style="margin-top:-0.5px;"><div class="float-left">9)</div><div class="media-body">mESC/benzonase_ Î±FLAG_undiluted</div></div><div class="media media-x cnautokh cnautokh10" style="margin-top:-0.5px;"><div class="float-left">10)</div><div class="media-body">empty<br/><br/></div></div><strong>Gel#2 (Î±FLAG blot, to evaluate CoIP)</strong> <div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Prestained protein standard</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">empty</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">mESC/untreated_0.75%_input</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">mESC/untreated_IgG_diluted</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-0.5px;"><div class="float-left">5)</div><div class="media-body">mESC/untreated_ Î±FLAG_diluted</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-0.5px;"><div class="float-left">6)</div><div class="media-body">empty</div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-0.5px;"><div class="float-left">7)</div><div class="media-body">mESC/benzonase_0.75%_input</div></div><div class="media media-x cnautokh cnautokh8" style="margin-top:-0.5px;"><div class="float-left">8)</div><div class="media-body">mESC/benzonase_IgG_diluted</div></div><div class="media media-x cnautokh cnautokh9" style="margin-top:-0.5px;"><div class="float-left">9)</div><div class="media-body">mESC/benzonase_ Î±FLAG_diluted</div></div><div class="media media-x cnautokh cnautokh10" style="margin-top:-0.5px;"><div class="float-left">10)</div><div class="media-body">empty</div></div><em>Note: The protein standard is either loaded in well 1 (gel#1) or well 2 (gel#2) so that the two can be easily distinguished.</em> </li><li>Prepare the prestained protein standard (7 Î¼l plus 27 Î¼l of 1x SDS-loading buffer, 17 Î¼l per well).</li><li>Carefully load 17 Î¼l of each sample with a P20. Fill the empty wells with 17 Î¼l of 1x SDS loading buffer.</li><li>Connect the lid of the running cassette to a power supply and place it on top of the cassette.</li><li>Run at 75 V until the samples enter the separating gel.<br/><em>Note: Once the marker hits the separating gel, you will see clear bands forming.</em> </li><li>Increase the voltage to 130 V.<br/><em>Note: To help heat dissipation, we often put the cassette on top of a large metal block.</em> </li><li>Run until the bromophenol blue hits the bottom of the cassette, but before it runs out of the gel.</li><li>Turn off the power supply.</li><li>Remove the lid, unlock the gel cassettes and take them out.</li><li>Wash extensively the cassettes with ddH<sub>2</sub>O, including the wells, to remove any traces of running buffer.</li></ol></li><li>Transfer proteins to a nitrocellulose membrane<ol class="npwthree"><li>Assemble the transfer apparatus (Figure 4).<ol class="npwfour"><li>Pre-wet with ddH<sub>2</sub>O two pieces of blotting membrane, cut only slightly larger than the gel to transfer.<br/><em>Note: Handle the membrane with gloves and only touching the membraneâs sides.</em> </li><li>Take a tray that will fit the gel holder cassettes.<br/></li><li>Add a gel holder cassette, black side facing down, and pour enough transfer buffer to cover it.</li><li>Add one sponge, and make sure it gets fully covered in transfer buffer.</li><li>Pre-wet in transfer buffer 2 pieces of blotting paper, cut only slightly larger than the gel to transfer.</li><li>Place the first gel cassette on a paper towel (wells down) and crack it open with the provided metal spatula.</li><li>Remove the top part, making sure that the gels adheres to the bottom one.</li><li>Use a razor blade to remove the bottom part of the gel that sticks out, and the stacking gel.<br/><em>Note: The stacking gel is gluey. Scrape it with the razor blade from the gel cassette to the paper towel to get rid of all residues.</em> </li><li>Submerge the cassette containing the gel in the tray with transfer buffer.</li><li>Lift the gel from the cassette and transfer it on top of the wet blotting paper; keep the empty cassette on the side.</li><li>Pre-wet the blotting membrane in transfer buffer and layer it on top of the gel.</li><li>Pre-wet a piece of blotting paper and layer it on top of the membrane.</li><li>Use the empty gel cassette to roll out any air bubbles that might have formed between the gel and the membrane, pressing firmly but gently.</li><li>Repeat Steps J3a-xii and J3a-xiii.</li><li>Pre-wet another sponge and add it on top.<br/><em>Note: Always pre-wet blotting paper and sponges to avoid any capillarity effects.</em> </li><li>Carefully close the gel holder cassette without disturbing the sandwich and insert it in the transfer tank.</li><li>Repeat Steps J3a-iii through J3a-xvi for the second gel.<br/><br/><img alt="" class="layerphoto" height="65" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218173359_3459.jpg" width="100"/><br/><strong>Figure 4. Assembly of the transfer âsandwichâ</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Insert the cooling unit.</li><li>Completely fill the transfer tank.</li><li>Close the lid and connect the cables to the transfer power supply.</li><li>Transfer at 100 V at 4 Â°C for 2 h.</li></ol></li><li>Block the membrane<ol class="npwthree"><li>Retrieve the blotted membranes from the transfer apparatus.<br/><em>Note: You will know that the transfer was successful if you see the stained molecular weight transferred to your membrane.</em> </li><li>Quickly rinse each membrane in TBS-T using a container big enough to accommodate the membrane without bending it.</li><li>Add 10 ml of blocking solution (10% milk in TBS-T).<br/><em>Note: Pipet liquid next to the membrane, never on top of it.</em> </li><li>Agitate for at least 1 h at room temperature.</li></ol></li><li>Blot the membrane<ol class="npwthree"><li>Prepare your primary antibodies in blotting solution (5% milk in TBS-T) in 15 ml conical tubes. In our illustrative experiment:<br/>Gel#1: rabbit Î±V5, 1:2,000 (4 Î¼l in 8 ml of 5% milk)<br/>Gel#2: rabbit Î±FLAG, 1:1,000 (8 Î¼l in 8 ml of 5% milk)<br/><em>Note: Check the datasheet of your antibody of choice to find the recommended dilution for Western blot. This can vary a lot from one antibody to another.</em> </li><li>Discard the blocking solution and add the blotting solution with antibodies.</li><li>Agitate at 4 Â°C overnight.<br/><em>Note: Overnight incubation in the cold room increases the specific signal while reducing background noise for all the antibodies we tested so far.</em> </li></ol></li><li>Wash the membrane<ol class="npwthree"><li>Remove the blotting solutions from the membrane.</li><li>Remove any primary antibody residue with a quick rinse with 10 ml of TBS-T.</li><li>Replace with fresh 10 ml of TBS-T and agitate for 15 min.</li><li>Replace with fresh TBS-T and agitate for 5 min.</li><li>Repeat Step J6d.</li></ol></li><li>Blot with secondary antibodies conjugated to HRP<ol class="npwthree"><li>Prepare a 1:5,000 dilution of the HRP-cojugated secondary antibody in 5% milk in TBS-T. In our illustrative experiment, goat Î±-mouse HRP (2 Î¼l in 10 ml of 5% milk).</li><li>Agitate at room temperature for a minimum of 30 min.</li></ol></li><li>Wash the membrane as in Step J6.</li><li>Perform the chemiluminescent reaction<ol class="npwthree"><li>For each membrane, prepare 2 ml of ECL reagent (1 ml of the oxidiziong reagent plus 1 ml of the enhanced luminol reagent).</li><li>Transfer the membrane to a new container and aspirate any TBS-T residues.</li><li>Pipet the ECL reagent to fully cover the membrane.</li><li>Incubate for 2 min gently swirling the container, still making sure that the membrane is fully covered.</li><li>Discard the ECL reagent and image your Western blot.<br/><em>Note: You can either use X-ray films (our preferred choice) or chemiluminescent imaging instruments (</em><em>e.g.</em><em>, Chemidoc).</em> </li></ol></li></ol></li></ol><p class="pptt" id="biaoti31370">### Notes</p><ol class="npwtwok"><li>It is of primary importance that you test the antibodies used to immunoprecipitate and/or detect each tag to exclude cross-reactivity with the other tag and also with the POI (using untagged, wild type cells). For example, while probing self-interaction between a V5-tagged and a FLAG-tagged Rad21 protein, we found that one of the FLAG antibodies did detect a band of the same size of Rad21 in untagged cells, suggesting a possible cross-reactivity with the wild type Rad21 protein (see Figure 2âfigure supplement 1 in Cattoglio <em>et al.</em> [2019]).</li><li>For a standard CoIP of endogenous proteins, you will need to identify good antibodies to immunoprecipitate and detect your POIs. If none are available, you could perform CRISPR/Cas9 endogenous tagging similar to what we do for self-CoIP, but this time tagging each of the proteins to be tested with a different tag (<em>e.g.</em>, FLAG-Halo-tagged CTCF and V5-SNAP-tagged Rad21 in clone C59 in Hansen <em>et al.</em> [2019]).<br/></li></ol><p class="pptt" id="biaoti31371">### Recipes</p><ol class="npwone"><li>Stock solutions<ol class="npwtwo"><li>6 N HCl (500 ml)<br/>Wear eye protection and mask. Work under a fume hood. Dilute 250 ml of the concentrated acid (36.5 to 38.0%) with 250 ml of double-distilled water (ddH<sub>2</sub>O).</li><li>10x PBS, pH 7.4 (1 L)<br/><ol class="npwthree"><li>To 800 ml of ddH<sub>2</sub>O add NaCl (80 g), KCl (2 g), Na<sub>2</sub>HPO<sub>4</sub> (14.4 g) and KH<sub>2</sub>PO<sub>4</sub> (2.4 g). Stir to dissolve</li><li>Wear eye protection and adjust the pH to 7.4 with 6 N HCl (Thermo Fisher Scientific to prepare a 6 N HCl solution dilute the concentrated acid 1:1 with ddH<sub>2</sub>O)</li><li>Bring the volume up to 1 L with ddH<sub>2</sub>O and dispense 100 ml into autoclavable glass bottles</li><li>Autoclave 30 min on liquid cycle</li></ol></li><li>1x PBS, pH 7.4<br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub><br/>2 mM KH<sub>2</sub>PO<sub>4</sub><br/>Dilute from a 10x autoclaved solution (Recipe A2) and re-autoclave<br/><em>Note: You can also purchase ready-to-use, sterile 1x PBS. Storage temperature: 15-30 Â°C. Shelf-life: 12 months from date of preparation.</em> </li><li>1 M HEPES, pH 7.6 (1 L)<br/><ol class="npwthree"><li>Start with 700 ml of ddH<sub>2</sub>O. Add 238.3 g of HEPES and let dissolve</li><li>Add 31.9 g of KOH (wear eye protection and mask)</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Filter through a bottle-top 0.2 Î¼m filter and aliquot 40 ml into 50 ml conical tubes</li><li>Store at -20 Â°C for up to 12 months</li></ol><em>Note: The 1 M solution prepared in this way has a pH of ~7.8, but a 20-fold dilution of the stock solution yields a pH of 7.6.</em> </li><li>1 M Tris-base pH 6.8/7.5 (1 L)<br/><ol class="npwthree"><li>Start with 500 ml of ddH<sub>2</sub>O. Add 121.1 g of Tris-base and let dissolve</li><li>Adjust the pH to 6.8/7.5 with 6 N HCl (wear eye protection and mask)</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Autoclave 30 min on liquid cycle</li></ol></li><li>5 M NaCl (1 L)<br/><ol class="npwthree"><li>Start with 700 ml of ddH<sub>2</sub>O. Add 292.2 g of NaCl and stir to dissolve</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Autoclave 30 min on liquid cycle</li></ol></li><li>2 M KCl (500 ml)<br/><ol class="npwthree"><li>Start with 300 ml of ddH<sub>2</sub>O. Add 74.6 g of KCl and stir to dissolve</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Filter through a bottle-top 0.2 Î¼m filter</li></ol></li><li>1 M MgCl<sub>2</sub> (1 L)<br/><ol class="npwthree"><li>Start with 800 ml of ddH<sub>2</sub>O. Add 203.3 g of MgCl<sub>2</sub> and stir to dissolve</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Autoclave 30 min on liquid cycle</li></ol></li><li>1 M DTT (10 ml)<br/><ol class="npwthree"><li>Start with 8 ml of ddH<sub>2</sub>O. Add 1.54 g of DTT and invert the tube to dissolve</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Prepare in 1-ml aliquots and store at -20 Â°C for up to 12 months</li></ol></li><li>10% Triton X-100 (500 ml)<br/>To 450 ml of ddH<sub>2</sub>O add 50 ml of Triton X-100.</li><li>0.5 M EDTA pH 8.0 (1 L)<br/><ol class="npwthree"><li>Start with 700 ml of ddH<sub>2</sub>O. Add 186.12 g of EDTA and stir (EDTA will not dissolve until the pH is near to 8.0)</li><li>Adjust the pH to 8.0 with NaOH pellets (~20 g; wear eye protection and mask)</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Autoclave 30 min on liquid cycle</li></ol></li><li>10% APS (10 ml)<br/><ol class="npwthree"><li>To 8 ml of ddH<sub>2</sub>O add 1 g of ammonium persulfate and invert the tube to dissolve</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Prepare in 1-ml aliquots and store at -20 Â°C for up to 12 months</li></ol></li><li>50x cOmplete protease inhibitors stock (~ 10 ml)<br/><ol class="npwthree"><li>Dissolve 10 tablets in 10 ml of ddH<sub>2</sub>O in a 15 ml conical tube</li><li>Invert the tube until completely dissolved</li><li>Prepare 500-ml aliquots and store at -20 Â°C for up to 12 months</li></ol></li><li>1 M benzamidine (10 ml; 1,000x)<br/><ol class="npwthree"><li>To 8 ml of ddH<sub>2</sub>O add 1.2 g of benzamidine and invert the tube to dissolve</li><li>Bring the solution up to volume with ddH<sub>2</sub>O</li><li>Prepare in 1-ml aliquots and store at -20 Â°C for up to 12 months</li></ol></li><li>0.2 M PMSF (10 ml; 1,000x)<br/><ol class="npwthree"><li>To 9 ml of isopropanol add 348 mg of PMSF and invert the tube to dissolve</li><li>Bring the solution up to volume with isopropanol</li><li>Prepare in 1-ml aliquots and store at -20 Â°C for up to 12 months</li></ol></li><li>1.23 M Bis-Tris pH 6.4/6.7 (250 ml each)<br/><ol class="npwthree"><li>To 300 ml of ddH<sub>2</sub>O add 128.7 g of Bis-Tris and stir until dissolved</li><li>Bring the volume up to 400 ml and equally divide to two beakers (200 ml each)</li><li>Bring the pH to either 6.4 or 6.7 with concentrated HCl (wear eye protection and mask)</li><li>Bring each solution up to 250 ml with ddH<sub>2</sub>O</li><li>Filter sterilize and store at room temperature for up to 12 months<br/></li></ol><br/></li></ol></li><li>Buffers and other solutions<ol class="npwtwo"><li>Cell lysis buffer (10 ml) [final concentration]:<br/>100 Î¼l of 1 M HEPES, pH 7.9 [10 mM]<br/>50 Î¼l of 2 M KCl [10 mM]<br/>30 Î¼l of 1 M MgCl<sub>2</sub> [3 mM]<br/>1.16 gr of sucrose [~340 mM]<br/>1 ml of glycerol [10%]<br/>10 Î¼l of 1 M DTT [1 mM]<br/>ddH<sub>2</sub>O to 10 ml<br/><ol class="npwthree"><li>Weight sucrose and add ~5 ml of ddH<sub>2</sub>O</li><li>Add all the remaining solutions and bring volume to 10 ml with more ddH<sub>2</sub>O</li><li>Filter with a syringe through a 0.2 Î¼m filter</li><li>Right prior to use, add Triton X-100 to 0.1% final (use the 10% Triton X-100 stock as a 100x concentrated solution) and preotease inhibitors (aprotinin, benzamidine, PMSF and the Roche protease inhibitor cocktail)</li></ol></li><li>0.1 M CoIP buffer (500 ml) [final concentration]<br/>10 ml of 5 M NaCl [100 mM]<br/>12.5 ml of 1 M Hepes pH 7.9 [25 mM]<br/>500 Î¼l of 1 M MgCl<sub>2</sub> [1 mM]<br/>200 Î¼l of 0.5 M EDTA pH 8.0 [0.2 mM]<br/>25 ml of 10% NP-40 alternative [0.5%]<br/>ddH<sub>2</sub>O to 500 ml<br/>To 100 ml of ddH<sub>2</sub>O add all the other solutions. Bring the volume to 500 ml and filter through a bottle-top 0.2 Î¼m filter</li><li>0.2 M CoIP buffer (500 ml) [final concentration]<br/>20 ml of 5 M NaCl [200 mM]<br/>12.5 ml of 1 M HEPES pH 7.9 [25 mM]<br/>500 Î¼l of 1 M MgCl<sub>2</sub> [1 mM]<br/>200 Î¼l of 0.5 M EDTA pH 8.0 [0.2 mM]<br/>25 ml of 10% NP-40 alternative [0.5%]<br/>ddH<sub>2</sub>O to 500 ml<br/>To 100 ml of ddH<sub>2</sub>O add all the other solutions. Bring the volume to 500 ml and filter through a bottle-top 0.2 Î¼m filter</li><li>0.5% BSA in 0.2 M CoIP buffer (50 ml)<br/>To 50 ml of 0.2 M CoIP buffer add 250 mg of BSA. Invert the tube to mix until BSA is dissolved and filter through a 50-ml centrifuge tube top filter</li><li>20x MOPS running buffer (1 L)<br/>209.39 gr of MOPS<br/>121.1 g of Tris-Base<br/>20 g of SDS (wear eye protection and mask)<br/>7.44 g of EDTA<br/><ol class="npwthree"><li>Bring the volume to 1 liter with ddH<sub>2</sub>O and filter through a bottle-top 0.2 Î¼m filter</li><li>Store at room temperature protected from light for up to 12 months</li><li>To run an SDS-PAGE, dilute to 1x with ddH<sub>2</sub>O and add sodium metabisulfute (0.475 g per 500 ml running buffer)</li><li>Prepare the 1x running buffer fresh each time; do not re-use</li></ol></li><li>Transfer buffer (2 L)<br/>3.63 g of Tris-Base<br/>28.83 g of Glycine<br/>400 ml of Methanol<br/><ol class="npwthree"><li>Dissolve Tris-Base and glycine in 1 liter of ddH<sub>2</sub>O</li><li>Bring the volume to 1.6 liters and add methanol</li><li>Store at 4 Â°C for up to 12 months. We typically re-use the same transfer buffer twice before discarding it</li></ol></li><li>TBS-T (2 L)<br/>200 ml of 5 M NaCl<br/>10 ml of 2 M Tris pH 7.5<br/>2 ml of Tween-20<ol class="npwthree"><li>To 1.5 liters of ddH<sub>2</sub>O add NaCl and Tris</li><li>Bring the volume to 2 liters, transfer to a clean bottle, add a magnetic stirrer and Tween-20 (we typically use a P1000, dropping the whole tip filled with Tween-20 inside the bottle)</li><li>Stir until all the Tween-20 is homogenously distributed and filter through a bottle-top 0.2 Î¼m filter </li></ol></li><li>4x sample buffer (10 ml) [final concentration]<br/>1.6 ml of 2-mercaptoethanol [16%]<br/>2 ml of Tris pH 6.8 [200 mM]<br/>0.8 g of SDS (wear eye protection and mask) [8%]<br/>4 ml of Glycerol [40%]<br/>0.62 g of DTT [400 mM]<br/>0.04 gr of Bromophenol Blue [0.4%]<br/><ol class="npwthree"><li>Weight all the ingredients in a 50 ml conical tube</li><li>Filter through a 0.2 Î¼m filter 50 ml filter tube</li></ol>For a 2x sample buffer, dilute the 4x solution 1:1 with ddH<sub>2</sub>O<br/>For a 1x sample buffer, dilute the 2x solution 1:1 with ddH<sub>2</sub>O. Prepare 1 ml aliquots and store at -20 Â°C for up to 12 months</li><li>Blocking solution, 10% milk (w/v) in TBS-T (100 ml)<br/>Dissolve 10 g of instant milk in 100 ml of TBS-T. Store at 4 Â°C for up to 2 weeks</li><li>Blotting solution, 5% milk (w/v) in TBS-T (100 ml)<br/>Dissolve 5 g of instant milk in 100 ml of TBS-T. Store at 4 Â°C for up to 2 weeks</li></ol></li></ol><p class="pptt" id="biaoti31372">### Acknowledgments</p><p class="ppzz">ASH acknowledges support from a Siebel Stem Cell Institute post-doctoral fellowship and NIH NIGMS K99 Pathway to Independence Award K99GM130896. The Tjian-Darzacq lab is supported by NIH Common Fund 4D Nucleome Program U01-EB021236 and U54-DK107980 (XD), the California Institute of Regenerative Medicine grant LA1-08013 (XD), and by the Howard Hughes Medical Institute (003061, RT). We first developed the protocol detailed here to assess the self-interaction of CTCF (Hansen <em>et al.</em>, 2019) and of the cohesin subunit Rad21 (Cattoglio <em>et al.</em>, 2019). We thank Jack Li for providing comments to this manuscript.</p><p class="pptt" id="biaoti31373">### Competing interests</p><p class="ppzz">The authors declare no competing financial/non-financial interests.</p><p class="pptt" id="biaoti31374">### References</p><ol class="npwtwok"><li>Addgene. <a href="https://www.addgene.org/crispr/guide/" target="_blank"><em>CRISPR Guide.</em></a> Accessed 24 October 2019.</li><li>Aird, E. J., Lovendahl, K. N., St Martin, A., Harris, R. S. and Gordon, W. R. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30271937" target="_blank">Increasing Cas9-mediated homology-directed repair efficiency through covalent tethering of DNA repair template.</a> <em>Commun Biol</em> 1: 54. </li><li>Akoev, V., Gogol, E. P., Barnett, M. E. and Zolkiewski, M. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/14978298" target="_blank">Nucleotide-induced switch in oligomerization of the AAA+ ATPase ClpB.</a> <em>Protein Sci</em> 13(3): 567-574.</li><li>Baisamy, L., Jurisch, N. and Diviani, D. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15691829" target="_blank">Leucine zipper-mediated homo-oligomerization regulates the Rho-GEF activity of AKAP-Lbc.</a> <em>J Biol Chem</em> 280(15): 15405-15412.</li><li>Boehning, M., Dugast-Darzacq, C., Rankovic, M., Hansen, A. S., Yu, T., Marie-Nelly, H., McSwiggen, D. T., Kokic, G., Dailey, G. M., Cramer, P., Darzacq, X. and Zweckstetter, M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30127355" target="_blank">RNA polymerase II clustering through carboxy-terminal domain phase separation.</a> <em>Nat Struct Mol Biol</em> 25(9): 833-840.</li><li>Bourdenx, M., Koulakiotis, N. S., Sanoudou, D., Bezard, E., Dehay, B. and Tsarbopoulos, A. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26209472" target="_blank">Protein aggregation and neurodegeneration in prototypical neurodegenerative diseases: Examples of amyloidopathies, tauopathies and synucleinopathies.</a> <em>Prog Neurobiol</em> 155: 171-193.</li><li>Castle, B. T., Odde, D. J. and Wood, D. K. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30891490" target="_blank">Rapid and inefficient kinetics of sickle hemoglobin fiber growth.</a> <em>Sci Adv</em> 5(3): eaau1086. </li><li>Cattoglio, C., Pustova, I., Walther, N., Ho, J. J., Hantsche-Grininger, M., Inouye, C. J., Hossain, M. J., Dailey, G. M., Ellenberg, J., Darzacq, X., Tjian, R. and Hansen, A. S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31205001" target="_blank">Determining cellular CTCF and cohesin abundances to constrain 3D genome models.</a> <em>Elife</em> 8: e40164.</li><li>Chen, Q., Zhuo, Y., Kim, M., Hanson, S. M., Francis, D. J., Vishnivetskiy, S. A., Altenbach, C., Klug, C. S., Hubbell, W. L. and Gurevich, V. V. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24292832" target="_blank">Self-association of arrestin family members.</a> <em>Handb Exp Pharmacol</em> 219: 205-223.</li><li>Cho, W. K., Spille, J. H., Hecht, M., Lee, C., Li, C., Grube, V. and Cisse, I. I. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29930094" target="_blank">Mediator and RNA polymerase II clusters associate in transcription-dependent condensates.</a><em> Science</em> 361(6400): 412-415.</li><li>Chong, S., Dugast-Darzacq, C., Liu, Z., Dong, P., Dailey, G. M., Cattoglio, C., Heckert, A., Banala, S., Lavis, L., Darzacq, X. and Tjian, R. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29930090" target="_blank">Imaging dynamic and selective low-complexity domain interactions that control gene transcription.</a> <em>Science</em> 361(6400).</li><li>Dufourt, J., Trullo, A., Hunter, J., Fernandez, C., Lazaro, J., Dejean, M., Morales, L., Nait-Amer, S., Schulz, K. N., Harrison, M. M., Favard, C., Radulescu, O. and Lagha, M. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30518940" target="_blank">Temporal control of gene expression by the pioneer factor Zelda through transient interactions in hubs.</a> <em>Nat Commun</em> 9(1): 5194. </li><li>GarcÃ­a, C. D., Hadley, D. J., Wilson, W. W. and Henry, C. S. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12790668" target="_blank">Measuring protein interactions by microchip self-interaction chromatography.</a> <em>Biotechnol Prog</em> 19(3): 1006-1010.</li><li>Hansen, A. S., Hsieh, T. S., Cattoglio, C., Pustova, I., SaldaÃ±a-Meyer, R., Reinberg, D., Darzacq, X. and Tjian, R. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31522987" target="_blank">Distinct classes of chromatin loops revealed by deletion of an RNA-binding region in CTCF.</a> <em>Mol Cell</em>. </li><li>Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. and Darzacq, X. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28467304" target="_blank">CTCF and cohesin regulate chromatin loop stability with distinct dynamics.</a> <em>Elife</em> 6: e25776. </li><li>Haupt, A., Grancharova, T., Arakaki, J., Fuqua, M. A., Roberts, B. and Gunawardane, R. N. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30199041" target="_blank">Endogenous protein tagging in human induced pluripotent stem cells using CRISPR/Cas9.</a> <em>J Vis Exp</em>(138). </li><li>He, C., Baba, M., Cao, Y. and Klionsky, D. J. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18829864" target="_blank">Self-interaction is critical for Atg9 transport and function at the phagophore assembly site during autophagy.</a> <em>Mol Biol Cell</em> 19(12): 5506-5516.</li><li>Hwang, M., Ko, J. K., Weisleder, N., Takeshima, H. and Ma, J. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21525429" target="_blank">Redox-dependent oligomerization through a leucine zipper motif is essential for MG53-mediated cell membrane repair.</a> <em>Am J Physiol Cell Physiol</em> 301(1): C106-114. </li><li>Kang, S. H., Dao, T. N. M., Kim, O. K. and Folimonova, S. Y. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28279804" target="_blank">Self-interaction of Citrus tristeza virus p33 protein via N-terminal helix.</a> <em>Virus Res</em> 233: 29-34.</li><li>Li, J. Q., You, Z. H., Li, X., Ming, Z. and Chen, X. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28092572" target="_blank">PSPEL:<em> in silico</em> prediction of self-interacting proteins from amino acids sequences using ensemble learning.</a> <em>IEEE/ACM Trans Comput Biol Bioinform</em> 14(5): 1165-1172.</li><li>Liu, P., Li, Y., Lei, J. and Dong, L. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30353511" target="_blank">Creating knockin alleles in mouse embryonic stem cells by CRISPR/Cas9-mediated homologous recombination without drug selection.</a> <em>Methods Mol Biol</em> 1874:115-137.</li><li>Liu, Z., Guo, F., Zhang, J., Wang, J., Lu, L., Li, D. and He, F. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23422585" target="_blank">Proteome-wide prediction of self-interacting proteins based on multiple properties.</a> <em>Mol Cell Proteomics</em> 12(6): 1689-1700.</li><li>Loughlin, F. E. and Wilce, J. A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31479821" target="_blank">TDP-43 and FUS-structural insights into RNA recognition and self-association.</a> <em>Curr Opin Struct Bio</em>l 59: 134-142. </li><li>Miller, K. E., Kim, Y., Huh, W. K. and Park, H. O. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25772494" target="_blank">Bimolecular fluorescence complementation (BiFC) analysis: advances and recent applications for genome-wide interaction studies.</a> <em>J Mol Biol</em> 427(11): 2039-2055. </li><li>Mir, M., Stadler, M. R., Ortiz, S. A., Hannon, C. E., Harrison, M. M., Darzacq, X. and Eisen, M. B. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30589412" target="_blank">Dynamic multifactor hubs interact transiently with sites of active transcription in.</a> <em>Elife</em> 7: e40497.</li><li>Pant, V., Kurukuti, S., Pugacheva, E., Shamsuddin, S., Mariano, P., Renkawitz, R., Klenova, E., Lobanenkov, V. and Ohlsson, R. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/15060168" target="_blank">Mutation of a single CTCF target site within the H19 imprinting control region leads to loss of Igf2 imprinting and complex patterns of de novo methylation upon maternal inheritance.</a> <em>Mol Cell Biol</em> 24(8): 3497-3504. </li><li>Paris, S., Longhi, R., Santambrogio, P. and de Curtis, I. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12611588" target="_blank">Leucine-zipper-mediated homo- and hetero-dimerization of GIT family p95-ARF GTPase-activating protein, PIX-, paxillin-interacting proteins 1 and 2.</a> <em>Biochem J</em> 372(Pt 2): 391-398. </li><li>Roberts, B., Haupt, A., Tucker, A., Grancharova, T., Arakaki, J., Fuqua, M. A., Nelson, A., Hookway, C., Ludmann, S. A., Mueller, I. A., Yang, R., Horwitz, R., Rafelski, S. M. and Gunawardane, R. N. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28814507" target="_blank">Systematic gene tagging using CRISPR/Cas9 in human stem cells to illuminate cell organization.</a> <em>Mol Biol Cell</em> 28(21): 2854-2874. </li><li>SaldaÃ±a-Meyer, R., GonzÃ¡lez-BuendÃ­a, E., Guerrero, G., Narendra, V., Bonasio, R., Recillas-Targa, F. and Reinberg, D. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696455" target="_blank">CTCF regulates the human p53 gene through direct interaction with its natural antisense transcript, Wrap53.</a> <em>Genes Dev</em> 28(7): 723-734.<br/></li><li>Shao, S., Xue, B. and Sun, Y. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29861035" target="_blank">Intranucleus single-molecule imaging in living cells.</a> <em>Biophys J</em> 115(2): 181-189.</li><li>Sharma, A., Toepfer, C. N., Ward, T., Wasson, L., Agarwal, R., Conner, D. A., Hu, J. H. and Seidman, C. E. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29364519" target="_blank">CRISPR/Cas9-mediated fluorescent tagging of endogenous proteins in human pluripotent stem cells.</a> <em>Curr Protoc Hum Genet</em> 96: 21.11.21-21.11.20.</li><li>Tasan, I., Sustackova, G., Zhang, L., Kim, J., Sivaguru, M., HamediRad, M., Wang, Y., Genova, J., Ma, J., Belmont, A. S. and Zhao, H. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29912475" target="_blank">CRISPR/Cas9-mediated knock-in of an optimized TetO repeat for live cell imaging of endogenous loci.</a> <em>Nucleic Acids Res</em> 46(17): e100.</li><li><em>Thermo Fisher. (2019). <a href="https://www.thermofisher.com/us/en/home/life-science/protein-biology/protein-gel-electrophoresis/protein-gels/precast-midi-gels.html" target="_blank">Gel migration charts.</a> Accessed 18 November 2019.</em> </li><li>Wang, Y. B., You, Z. H., Li, X., Jiang, T. H., Cheng, L. and Chen, Z. H. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30577794" target="_blank">Prediction of protein self-interactions using stacked long short-term memory from protein sequences information.</a> <em>BMC Syst Biol</em> 12(Suppl 8): 129. </li><li>Yuan, J., Ng, W. H., Tian, Z., Yap, J., Baccarini, M., Chen, Z. and Hu, J. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30377225" target="_blank">Activating mutations in MEK1 enhance homodimerization and promote tumorigenesis.</a> <em>Sci Signal</em> 11(554).</li><li>Yusufzai, T. M., Tagami, H., Nakatani, Y. and Felsenfeld, G. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/14759373" target="_blank">CTCF tethers an insulator to subnuclear sites, suggesting shared insulator mechanisms across species.</a> <em>Mol Cell</em> 13(2): 291-298.</li><li>Zhai, J. X., Cao, T. J., An, J. Y. and Bian, Y. T. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28802824" target="_blank">Highly accurate prediction of protein self-interactions by incorporating the average block and PSSM information into the general PseAAC.</a> <em>J Theor Biol</em> 432: 80-86. </li><li>Zhang, X. R., He, J. B., Wang, Y. Z. and Du, L. L. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29703785" target="_blank">A cloning-free method for CRISPR/cas9-mediated genome editing in fission yeast.</a> <em>G3 (Bethesda)</em> 8(6): 2067-2077.<br/></li></ol>