<body>
            <h1>Estimating Cellular Abundances of Halo-tagged Proteins in Live Mammalian Cells by Flow Cytometry</h1>
            <p><strong>Authors:</strong> Claudia  Cattoglio, Xavier  Darzacq, Robert  Tjian, Anders Sejr Hansen</p>
            <h2>Abstract</h2>
            <p>Accurate abundance measurements of cellular proteins are required to achieve a quantitative and predictive understanding of any biological process inside the cell. Existing methods to determine absolute protein abundances are labor-intensive and/or require sophisticated experimental and computational infrastructure (<em>e.g.</em>, fluorescence correlation spectroscopy (FCS)-calibrated imaging and quantitative mass spectrometry). Here we detail a straightforward flow cytometry-based method to measure the absolute abundance of any Halo-tagged protein in live cells that uses a standard mammalian cell line with a known number of Halo-CTCF proteins recently characterized in our lab. The protocol only comprises a few steps. First, a cell line expressing the Halo-tagged protein of interest is grown and labeled side-by-side with our standard line. Then, average fluorescence intensities are measured by conventional flow cytometry analysis and finally a simple calculation is applied to estimate the absolute number of the Halo-tagged protein of interest per cell. Once the protein of interest has been endogenously tagged with HaloTag, which we routinely achieve by Cas9-mediated genome editing, the presented protocol is fast, convenient, reproducible, cost-effective and readily accessible.</p>
            <p><strong>Keywords:</strong> HaloTag, Halo labeling, Absolute quantification, Number of proteins per cell, Flow cytometry, Standard cell line, Protein abundance, Flow cytometry</p>
            <h2>Background</h2><p>Assigning numbers to biological processes and to the molecules participating in them is of foremost importance to constrain models and attain a mechanistic and quantitative knowledge of biological phenomena (Ghaemmaghami <em>et al.</em>, 2003). Proteins are macromolecules essential to cell, tissue, and organism structure, function and regulation. Although highly desirable, measuring absolute abundances of cellular proteins is a non-trivial undertaking. The few methods currently available are time-consuming, costly and technically challenging, and for the most part rely on complicated equipment and analytical tools not readily accessible to most laboratories. This is certainly the case for quantitative mass spectrometry, a powerful yet complex technology that allows high-throughput measurements of un-modified proteins (Bantscheff <em>et al.</em>, 2007; Bantscheff <em>et al.</em>, 2012; Holzmann <em>et al.</em>, 2019). Quantitative FCS-calibrated imaging is a live cell method that can be streamlined to achieve high-throughput abundance measurement during dynamic cellular processes, while retaining single-cell resolution, and can be selective for different sub-cellular compartments (Cai <em>et al.</em>, 2018; Politi <em>et al.</em>, 2018; Walther <em>et al.</em>, 2018; Holzmann <em>et al.</em>, 2019). However, it again requires advanced microscopy equipment and computational infrastructure. Electrophoresis-based methods (<em>e.g.</em>, traditional SDS-PAGE or more advanced capillary systems followed by staining/blotting (Chen <em>et al.</em>, 2013; Bennett <em>et al.</em>, 2017) are less complex, but still rely on accurate cell counting and/or depend on the availability of a pure recombinant protein to use as a calibrator.<br/> <br/>We have recently developed a new flow cytometry-based absolute quantification method that can be easily applied to any Halo-tagged protein in the cell. This was part of a larger effort to quantify nuclear transcription factors and architectural 3D-genome regulators like CTCF and cohesin (Cattoglio <em>et al.</em>, 2019), with the ultimate goal of deciphering the spatiotemporal regulation of transcription and its interplay with 3D genome organization. The method has several advantages over existing ones: it employs a human U2OS-derived cell line standard that is easy to culture; it only requires a flow cytometer, an instrument available in most institutions; it has single-cell resolution; it can be reproducibly repeated multiple times with little effort and cost; and, it involves very basic data analysis.<br/> <br/>We focused on the HaloTag protein-fusion platform (Los <em>et al.</em>, 2008) because of its popularity and versatility, with applications in a broad range of experimental systems (England <em>et al.</em>, 2015), and since it is currently the preferred choice for live-cell single-molecule imaging (Presman <em>et al.</em>, 2017). In fact, we first generated what is here used as a standard cell line with the intent of studying CTCF nuclear dynamics by single-molecule super-resolution microscopy (Hansen <em>et al.</em>, 2017). This is a U2OS human clonal cell line (clone C32) with the endogenous <em>CTCF</em> locus homozygously engineered via Cas9-mediated genome editing to express a Halo-CTCF protein. As such, its expression level is stable and reflects endogenous CTCF. We realized that clone C32 could become a quantification standard for any Halo-tagged protein if we could estimate and cross-validate the absolute protein copy number per cell of Halo-CTCF in this U2OS cell line. We achieved this through a combination of in-gel fluorescence and FCS-calibrated imaging, obtaining consistent results across the two orthogonal methods performed in two different labs (~104,900 Â± 14,600 and 114,600 Â± 10,200 Halo-CTCF proteins per U2OS interphase cell, respectively) (Cattoglio <em>et al.</em>, 2019). We took the mean of the two methods, 109,800 CTCF proteins per cell, as our best and final cross-validated estimate. This number can be now easily used to infer absolute abundances of other Halo-tagged proteins by simply growing, labeling and measuring fluorescence intensities of the cell line of interest side-by-side with our C32 standard, as we have previously demonstrated using Sox2 and TBP in mouse embryonic stem cells (Teves <em>et al.</em>, 2016 and 2018; Cattoglio <em>et al.</em>, 2019) (see also the data re-analysis in Table 1). After subtracting the background fluorescence of unstained cells, the mean fluorescence intensity of the cell line of interest is divided by the C32 standard fluorescence and multiplied by 109,800 to obtain the absolute abundance of the Halo-tagged protein under study (Figure 1). Although we focus exclusively on endogenously tagged cell lines here, the same pipeline can also be deployed to calculate abundances of exogenously expressed Halo-tagged proteins.<br/><br/><img alt="" class="layerphoto" height="158" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218173608_9518.jpg" width="100"/><br/><strong>Figure 1. A streamlined method for absolute quantification of Halo-tagged proteins.</strong> (1) Grow your cell line of interest (COI), expressing the Halo-tagged protein to be quantified, side-by-side with the U2OS C32 Halo-CTCF cell standard. We usually perform the experiment in 6-well plates, but you can scale it up or down as required. (2) Label cells with the HaloTag ligand conjugated to a fluorophore (<em>e.g.</em>, TMR) and (3) measure absolute fluorescence intensities by flow cytometry. (4) Subtract the background fluorescence of unlabeled controls and use the adjusted fluorescence values to calculate the absolute abundance (<em>n</em>) of the protein of interest (POI). 109,800 is the average number of Halo-CTCF molecules we estimated in the U2OS C32 cell standard.<br/><br/> The quantitative approach described here comes with a few limitations. It does not allow direct measurement of the concentration of the protein of interest, nor its quantification in different cellular compartments, it may not be robust for very lowly or very highly expressed proteins (below ~10,000 and above 10-20 million proteins/cell, respectively), and it is obviously restricted to Halo-tagged proteins. However, several streamlined protocols are now available to perform efficient genome editing with CRISPR-Cas9 in a variety of model organisms and cell types, and these are described elsewhere (Reference 2; Roberts <em>et al.</em>, 2017; Aird <em>et al.</em>, 2018; Haupt <em>et al.</em>, 2018; Sharma <em>et al.</em>, 2018; Tasan <em>et al.</em>, 2018; Zhang <em>et al.</em>, 2018; Liu <em>et al.</em>, 2019).<br/> <br/>Apart from these few drawbacks, our quantification protocol is simple and yet powerful. For instance, although we have only estimated average cellular protein numbers so far, the method can be used in principle to measure protein abundances in single cells, since the flow cytometer outputs fluorescence values for each individual cell. Another advantage of using flow cytometry as a readout is that the Halo-labeling can be multiplexed with other staining strategies (<em>e.g.</em>, cell cycle analysis, surface and/or intracellular staining for the markers of interest) (Kovalenko <em>et al.</em>, 2011) to determine protein abundances in different phases of the cell cycle and/or within different subpopulations of complex biological samples, like those obtained in differentiation and reprogramming experiments.<br/> <br/>In summary, we here detail a rapid, straightforward and convenient method that will allow other researchers to estimate absolute cellular abundances of their Halo-tagged protein of interest with only a few hours of hands-on work.</p>
        </body>

<p class="pptt" id="biaoti31468">## Materials and Reagents</p><p class="ppzz"><em>Note: With the exception of the U2OS C32 cell standard, you can replace all the listed reagents, equipment and software with equivalent alternatives.</em> </p><ol class="npwtwo"><li>Falcon<sup>TM</sup> standard 100-mm tissue culture dishes (Corning<sup>TM</sup> 353000, Fisher Scientific, catalog number: 08-772-E)</li><li>Falcon<sup>TM</sup> polystyrene 6-well microplates (Corning<sup>TM</sup> 353046, Fisher Scientific, catalog number: 08-772-1B)</li><li>Cryovials (Corning<sup>TM</sup> 430487, Fisher Scientific, catalog number: 03-374-20)</li><li>Eppendorf<sup>TM</sup> snap-cap microcentrifuge safe-lock<sup>TM</sup> tubes, 1.5 ml (Fisher Scientific, catalog number: 05-402)</li><li>Test tube with cell strainer snap cap (Corning<sup>TM</sup> 352235, Fisher Scientific, catalog number: 08-771-23)</li><li>Eye protection</li><li>Mask</li><li>U2OS C32 Halo-CTCF standard cell line [first described in Hansen <em>et al.</em> (2017), available upon request]. Thaw the contents of one cryovial and plate onto one 100-mm cell culture plate</li><li>User-provided cell line(s) expressing the Halo-tagged protein(s) of interest</li><li>DMEM, low glucose, GlutaMAX<sup>TM</sup> Supplement, pyruvate (Fisher Scientific, catalog number: 10567014). Storage temperature: 4 Â°C. Shelf-life: 12 months from date of manufacture</li><li>Fetal bovine serum (FBS) (HyClone, catalog number: SH30910.03). Not fully-defined FBS will also work. Store in 50 ml aliquots at -20 Â°C. Shelf-life: specified by the manufacturer</li><li>50x Penicillin-streptomycin mixture, 5,000 units potassium penicillin and 5,000 Î¼g streptomycin sulfate per ml in 0.85% saline (Lonza<sup>TM</sup> 17-603E, Fisher Scientific, catalog number: BW17-603E). Store in 5 ml aliquots at -20 Â°C. Shelf-life: 12 months from date of manufacture<br/><em>Note: The working concentration is 100 units of potassium penicillin and 100 Î¼g of streptomycin sulfate per 1 ml of culture media.</em> </li><li>TMR-HaloTag<sup>Â®</sup> ligand (Promega, catalog number: G8251). Storage temperature: -20 Â°C. Shelf-life: specified by the manufacturer<br/><em>Note: If you need spectral properties different from those of TMR (555<sub>Ex</sub>/585<sub>Em</sub>), Promega offers a few alternatives, including HaloTag</em><sup><em>Â®</em></sup><em> ligands conjugated to Janelia Fluor</em><sup><em>Â®</em></sup><em> dyes.</em> </li><li>Dimethyl sulfoxide, DMSO (Millipore Sigma, catalog number: D2650-5X5ML). Store at room temperature. Shelf-life: specified by the manufacturer. We recommend using a freshly opened ampule to dilute fluorescent HaloTag<sup>Â®</sup> ligands</li><li>HCl (Fisher Scientific, catalog number: A144-50)</li><li>NaCl (Fisher Scientific, catalog number: S271-10)</li><li>KCl (Fisher Scientific, catalog number: P217-10)</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Fisher Scientific, catalog number: P285-500)</li><li>KH<sub>2</sub>PO<sub>4</sub> (Fisher Scientific, catalog number: S375-212)</li><li>NaOH pellets (Fisher Scientific, catalog number: S318-100), store at room temperature. Shelf-life: specified by the manufacturer</li><li>2.5% Trypsin (Fisher Scientific, Gibco, catalog number: 15090046). Storage temperature: -20 Â°C. Shelf-life: 24 months from date of manufacture for the 2.5% stock; 12 months from date of dilution for the diluted aliquots</li><li>EDTA (C<sub>10</sub>H<sub>18</sub>N<sub>2</sub>Na<sub>2</sub>O<sub>10</sub>, Fisher Scientific, catalog number: S311-500). Storage temperature: room temperature. Shelf-life: specified by the manufacturer</li><li>10x PBS, pH 7.4 (1 L) (see Recipes)</li><li>1x PBS, pH 7.4 (see Recipes)</li><li>0.1 M EDTA pH 8.0 (see Recipes)</li><li>0.05% Trypsin (see Recipes)</li><li>U2OS media (see Recipes)</li><li>U2OS freezing media (see Recipes)</li></ol><p class="pptt" id="biaoti31469">### Equipment</p><ol class="npwtwo"><li>Cell culture incubator (Sanyo copper alloy IncuSafe humidified incubator, model number: MCO-18AIC(UV)). Set at 37 Â°C and 5.5% CO<sub>2</sub> </li><li>Biosafety cabinet (Baker Sterligard III, model: SG403A)</li><li>Water bath (Thermo Fisher Scientific Precision, model: 185 (2843))</li><li>Microscope (Nikon Diaphot inverted phase-contrast microscope)</li><li>Tabletop centrifuge (Eppendorf, model: 5702)</li><li>Microfuge (Eppendorf, model: 5418)</li><li>Vortex mixer (Scientific Industries, Inc., model: Vortex-Genie 2)</li><li>BD LSRFortessa<sup>TM</sup> flow cytometer</li></ol><em>Note: For TMR staining you will need a 561-nm laser as an excitation source and a 610/20 band pass filter or similar (TMR emission max is ~585 nm).</em> <p class="pptt" id="biaoti31470">### Software</p><ol class="npwtwok"><li>BD FACSDiva<sup>TM</sup> 6.2 (BD, <a href="https://www.bdbiosciences.com/en-us/instruments/research-instruments/research-software/flow-cytometry-acquisition/facsdiva-software" target="_blank">https://www.bdbiosciences.com/en-us/instruments/research-instruments/research-software/flow-cytometry-acquisition/facsdiva-software</a>)</li><li>FlowJo 10.6.1 (BD, <a href="https://www.flowjo.com/" target="_blank">https://www.flowjo.com</a>)</li><li>Excel 16.30 or equivalent (Microsoft Corporation, <a href="https://www.microsoft.com/Microsoft/Excel" target="_blank">https://www.microsoft.com/Microsoft/Excel</a>)<br/></li></ol><p class="pptt" id="biaoti31471">### Procedure</p><ol class="npwone"><li>Cell culture<br/><em>Note: Perform all cell culture procedures in a biosafety cabinet to maintain sterility. General guidelines to perform cell and tissue culture are available from different organizations/companies [Abcam (Reference 1); ATCC (Reference 4); ECACC (Reference 11); LifeTechnologies (Reference 18)].</em> <ol class="npwtwo"><li>Culture your cell line(s) of interest side-by-side with the U2OS C32 cell standard you received from our lab. To thaw U2OS cells:<ol class="npwthree"><li>Warm up U2OS media (Recipe 4) in a water bath set to 37 Â°C.</li><li>Add 9 ml of U2OS media to a 15 ml conical tube.</li><li>Put the cryovial in the 37 Â°C water bath and gently swirl until almost completely thawed.</li><li>With a P1000 pipet, transfer the content of the cryovial (1 ml or less) to the 15 ml conical tube and invert the tube a few times.</li><li>Spin cells down in a tabletop centrifuge (200 <em>x g</em>, 5 min, at room temperature).</li><li>Aspirate the supernatant, flick the tube to resuspend the cells and add 10 ml of U2OS media.</li><li>Plate to a 100-mm tissue culture dish and put cells overnight in the incubator set at 37 Â°C and 5.5% CO<sub>2</sub>.</li></ol></li><li>When cells are ready (60-80% confluent) split them to 6-well plates<br/><em>Note: We normally perform the labeling experiment in 6-well plates, but you can scale it up or down as required.</em> <ol class="npwthree"><li>Warm up U2OS media and 1x PBS in a water bath set to 37 Â°C, and trypsin at room temperature.</li><li>Take the 100-mm plate out of the incubator.</li><li>Aspirate media and wash with 10 ml of 1x PBS.</li><li>Aspirate PBS and add 1.5 ml of trypsin.</li><li>Move plate to the incubator for 1-2 min or until the cells look round and partially detached under a microscope (hitting the side of the plate with your hand helps detaching most of the cells).</li><li>Add 10.5 ml of U2OS media and pipet extensively up and down to achieve a single-cell suspension.</li><li>Distribute the cell suspension to 6-well microplates. Seed at least 2 wells per cell type: one to be left unlabeled to measure background fluorescence and the other to be stained with TMR. One ml of cells per well will give you enough cells to perform staining and flow cytometry the day after seeding (1 ml will contain ~250,000-500,000 U2OS cells when starting from a 60-80% confluent 100-mm plate). Use less if you need to delay the experiment (<em>e.g.</em>, 0.5 ml to wait one additional day or 0.25 ml to wait a couple of days).<br/><em>Note: We suggest you create your own stocks of the U2OS C32 cell standard at this point. To freeze U2OS cells, spin down all the cells you spare in a 15 ml conical tube. Resuspend the pellet in freezing media (Recipe 5) and aliquot to cryovials (we typically prepare 2 vials starting from a confluent 100-mm plate, resuspending the pellet in 500 Î¼l to 1 ml of freezing media per vial).</em> </li><li>Add media to 2 ml final volume and put plates back in the incubator.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>TMR labeling<br/><em>Note: When working with fluorophores, try and minimize light exposure (e.g., keep the hoodâs lights off, cover plates and tubes with foil).</em> <ol class="npwtwo"><li>Pre-warm 1x PBS, U2OS media and any other media required for your specific cell line(s) at 37 Â°C, and trypsin at room temperature. You will need ~5 ml of media per each well of the 6-well plate.<br/><em>Note: If your cell line of interest grows in suspension, spin cells down to replace media or to perform PBS washes. Because the cells will not need trypsinization, after the 5 min incubation to wash off unreacted ligand you can perform a PBS wash and directly resuspend cells in fresh media for flow cytometry.</em> </li><li>Thaw the 5 mM TMR-HaloTag<sup>Â®</sup> ligand provided by Promega.</li><li>Dilute 1:10 in DMSO to create a 1,000x stock solution (500 Î¼M) and make smaller aliquots to be stored at -20 Â°C (<em>e.g.</em>, dilute 10 Î¼l to 90 Î¼l DMSO and prepare 10x 10 Î¼l aliquots). Avoid freeze-thawing the 1,000x stock if you can, although a few freeze-thawing cycles should not be an issue.</li><li>Prepare 1.5 ml of labeling media per each well to be stained, diluting the 1,000x TMR-HaloTag<sup>Â®</sup> ligand stock in cell culture media (1:1,000 dilution: 1.5 Î¼l in each 1.5 ml of media). Remember to leave 1 well unlabeled for each cell line to be measured.</li><li>Take out the 6-well plates you prepared the previous day(s) from the incubator.</li><li>Aspirate the regular cell culture media and add 1.5 ml of either labeling media (labeled wells) or regular media (unlabeled wells for background measurement).</li><li>Transfer the plates back to the 37 Â°C incubator for 30 min.</li><li>Aspirate the supernatant and wash with 4 ml of 1x PBS.</li><li>Aspirate the PBS and add 2 ml of regular media.</li><li>Transfer the plates back to the 37 Â°C incubator for 5 min to remove unreacted ligand.<br/><br/></li></ol></li><li>Collect cells for flow cytometry<ol class="npwtwo"><li>Trypsinize cells. For each well:<ol class="npwthree"><li>Wash with 2 ml 1x PBS.</li><li>Aspirate PBS and add 300 Î¼l of trypsin.</li><li>Move the plates back to the 37 Â°C incubator as long as needed to detach the cells.</li><li>Inactivate trypsin by adding 900 Î¼l of cell culture media.</li><li>Resuspend each well up and down a few times with a P1000 and transfer the contents to a 1.5 ml Eppendorf tube.</li><li>Spin cells down in a microfuge (200 <em>x g</em>, 5 min, at room temperature).</li><li>Resuspend cell pellets in 500 Î¼l of fresh media.</li><li>Transfer cells to flow cytometry test tubes passing them through the tubesâ cell strainer snap cap.</li><li>Store tubes on ice, protected from light, and proceed immediately to flow cytometry.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol></li><li>Measure fluorescence intensities by flow cytometry<br/>Most flow cytometers and software will work, but here we illustrate the protocol using a BD LSRFortessa<sup>TM</sup> instrument and the associated BD FACSDiva<sup>TM</sup> acquisition software.<br/><em>Note: If you are using a different flow cytometer and/or a different acquisition software, refer to their operation manuals. If you are not familiar with flow cytometry, we advise running at least the first flow analysis with an experienced user. Also, we suggest to set up the flow cytometry experiment and adjust all the required settings in advance, so that the stained cells only sit on ice for a limited time (~20 min).</em> <ol class="npwtwo"><li>Turn on the BD LSRFortessa<sup>TM</sup> flow cytometer and the computer connected to it.</li><li>Open the BD FACSDiva<sup>TM</sup> acquisition software and wait for the cytometer to connect. If you are not familiar with the software, you can find the complete reference manual for the 6.0 version at <a href="https://www.bu.edu/flow-cytometry/files/2010/10/BDFACSDivaSoftwareReferenceManual.pdf" target="_blank">https://www.bu.edu/flow-cytometry/files/2010/10/BDFACSDivaSoftwareReferenceManual.pdf</a>.<br/></li><li>If a âCST Mismatchâ window appears, click on âUse CST Settingsâ.</li><li>In the upper menu bar, click âViewâ and make sure the following items are checked, and thus the corresponding window open: <em>Toolbar</em>, <em>Status Bar</em>, <em>Browser</em>, <em>Cytometer</em>, <em>Inspector</em>, <em>Worksheet </em>and <em>Acquisition Dashboard</em>.</li><li>In the upper menu bar, click âExperimentâ and choose âNew Experimentâ from the dropdown menu.</li><li>Select a blank template. Your experiment will now appear as an opened book in the <em>Browser</em> window. Right-click on the experiment name and click âRenameâ to assign a different name.</li><li>Below the experiment icon in the <em>Browser</em> window, click on âCytometer Settingsâ.</li><li>Move to the <em>Inspector</em> window to setup the <em>Cytometer Settings</em>. In the âParametersâ tab, select and delete everything but <em>FSC</em> (forward scatter), <em>SSC</em> (side scatter) and <em>PE-Tx-Red-YG</em> (for TMR staining; the instrument will use a 561-nm laser as an excitation source and read the emission through a 610/20 band pass filter or similar (TMR emission max is ~585 nm). If you are using a different fluorophore, modify accordingly. Check âAâ and âHâ for the <em>FSC</em> and <em>SSC</em> parameters to record the signal area and height (this will allow you to distinguish single cells from doublets during analysis). For the <em>PE-Tx-Red-YG</em> parameter, check âAâ and âLogâ, for the data to be in logarithmic scale.</li><li>Move back to the upper menu bar, click âExperimentâ and choose âNew Specimenâ from the dropdown menu. A <em>Panel Template</em> window will appear. Click âOKâ. A new specimen will be created (syringe icon) as a dependent of your open experiment in the <em>Browser</em> window. Right-click on the specimen name and change it to todayâs date (YYYYMMDD).</li><li>In the <em>Browser</em> window, select the <em>Global Worksheets</em> and then move to the <em>Global Worksheet</em> window.</li><li>Draw a dot plot (5<sup>th</sup> button from the left in the window toolbar, click and drag anywhere in the sheet; the dot plot will appear as you release the mouse).</li><li>Create a polygon gate (14<sup>th</sup> button from the left) by positioning the cursor inside the dot plot and clicking several times to draw your shape of choice. You will adjust the gate to select live cells later, when acquiring your sample.</li><li>Create a histogram plot (7<sup>th</sup> button from the left) and, if needed, click on the <em>x</em>-axis to change the displayed parameter to <em>PE-Tx-Red-YG</em>.<br/></li><li>With the histogram plot selected, move to the <em>Inspector</em> window and check âP1â. This will only display the TMR signal from the live cells that you gated in 12.</li><li>Move back to the <em>Browser</em> window.</li><li>Expand the specimen icon by clicking on the â+â sign to expose the âTubeâ icon. Right-click on it and rename it with your sample name (<em>U2OS_C32_unlabeled</em>).</li><li>Click on the left grey pointer to select the sample you have just created.</li><li>Take your first sample (U2OS_C32_unlabeled), vortex briefly, remove the cap and load it on the flow cytometer.</li><li>Set the flow rate to low using the âLOâ button in the instrument control panel.</li><li>Hit the âRUNâ button in the instrument control panel.</li><li>Go back to the Diva software, move to the <em>Acquisition Dashboard</em> and hit âAcquireâ to visualize your cells in the <em>Worksheet</em> window.</li><li>In the <em>Cytometer</em> window, adjust <em>FSC</em> and <em>SSC</em> voltages to distinguish dead from live cells in the <em>Worksheet</em> dot plot (for U2OS we use ~220 for the <em>FSC</em> and ~280 for the <em>SSC</em>). In the <em>Acquisition Dashboard</em> window, hit âRestartâ to only visualize the cells acquired with the adjusted voltages. Now you can adjust your P1 gate to only include live cells.<br/><em>Note: Forward and side scatter values vary from cell to cell depending on cellular size (FSC) and internal complexity (SSC). Live cells have larger FSC and SSC values than dead cells or cell debris, and normally constitute the majority of the population. Cell doublets/aggregates will have higher FSC and SSC values than single cells but should not represent the majority of cells, if you have thoroughly resuspended the samples (you will exclude doublets at the analysis stage. See Step E5 below).</em> </li><li>Back to the <em>Inspector</em> window, adjust the PE-Tx-Red-YG voltage to move the <em>Worksheet</em> histogram to the leftmost part of the <em>x</em>-axis while still displaying all the events (for U2OS we use ~520).</li><li>In the <em>Acquisition Dashboard</em> window, hit âStop Acquiringâ, and in the instrument control panel hit âSTNDBYâ to pause acquisition.</li><li>Quickly check other unlabeled samples to make sure that the voltages you chose for U2OS cells are reasonable for the other cell types as well. If not, adjust them as necessary.<br/><em>Notes:</em> <ol class="npwthree"><li><em>We usually run water between one sample and the other, to avoid cross-sample contamination.</em> </li><li><em>When working with different cell types you might need to move the P1 gate between one cell type and the other. However, you must find voltage values that are acceptable for all cell types and keep them constant during acquisition of all samples.</em> </li></ol></li><li>Go back to your original sample. Repeat Steps D18-D21. If no further voltage adjustments are required, in the <em>Acquisition Dashboard</em> window modify the <em>Acquisition Setup</em> to decide which <em>Stopping Gate</em> to use (choose P1) and how many Events to Record (choose 50,000). In case you need to re-adjust voltages, go back to Step D25.</li><li>Click the âRecord Dataâ button to save data to an FCS file. Increase the flow rate to MED or HIGH to speed up the acquisition if samples are too diluted. Once the specified number of events are recorded, on the instrument control panel hit âSTNDBYâ to pause acquisition.</li><li>In the <em>Acquisition Dashboard</em> window click on âNext Tubeâ to create the next sample. You can rename it either right-clicking on it in the <em>Browser</em> window or from the âTubeâ tab in the <em>Inspector</em> window.</li><li>Take the next sample and repeat Steps D18-D21 and D27-D28, until you have acquired and recorded all your samples.</li><li>Perform any recommended cleaning procedure for your flow cytometer.</li><li>In the <em>Browser</em> window, export your data by right-clicking on the specimen name. Select âExportâ, âFCS filesâ and leave the default FCS3.0 format. Click âOKâ, when prompted choose a directory where to save your data and click âSaveâ.</li><li>Quit the BD FACSDIVA<sup>TM</sup> software (from the top menu click first âFileâ and then âQuitâ).</li></ol><br/></li><li>Analyze flow cytometry data and obtain mean fluorescence intensities using the FlowJo software (see docs.flowjo.com/d2/ for full software documentation)<br/><em>Note: You can use the BD FACSDiva</em><sup><em>TM</em></sup><em> software for both data acquisition and analysis, instead of FlowJo. A license-free alternative is the Matlab code we used to analyze the data in our original manuscript (Cattoglio </em><em>et al.</em><em>, 2019), available at <a href="https://gitlab.com/tjian-darzacq-lab/cattoglio_et_al_absoluteabundance_2019" target="_blank">https://gitlab.com/tjian-darzacq-lab/cattoglio_et_al_absoluteabundance_2019</a>.</em> <ol class="npwtwo"><li>Open the FlowJo program.</li><li>Drag and drop your FCS 3.0 files into the lower workspace window.</li><li>Select the U2OS C32 unlabeled control.</li><li>Create a gate on live cells.<ol class="npwthree"><li>Double-click on the file name; this opens a dot plot of forward (FSC-A) and side scatter (SSC-A) values.</li><li>Create a free-polygon gate (5<sup>th</sup> button from the top left) to enclose live cells (name it âLive Cellsâ).</li><li>In the workspace window select the âLive Cellsâ gate and apply it to all samples (simply drag and drop it on the above window on the âAll Samplesâ group).</li><li>Go back to the forward and side-scatter plot; move from one sample to the next using the arrow at the top-right of the window and adjust the âLive Cellsâ gate as needed, dragging it around. If necessary, you can also modify the gate shape (Figure 2).<br/><br/><img alt="" class="layerphoto" height="136" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218174419_0832.jpg" width="100"/><br/><strong>Figure 2. Live-cell gating in FlowJo.</strong> Forward- and side-scatter plots generated by FlowJo that illustrate the âLive Cellsâ gating applied to the data re-analyzed here. The gating choice is likely to affect your absolute quantifications, since cell size correlates with protein abundances.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Exclude cell doublets<br/><em>Note: In the original flow cytometry experiments published in Cattoglio </em><em>et al.</em><em> (2019) we did not record FSC-H and SSC-H values. Thus, in the re-analysis of Table 1 below we could not exclude cell doublets as recommended in Step E5.</em> <ol class="npwthree"><li>Go back to the U2OS C32 unlabeled controlâs scatter plot and double-click anywhere inside the âLive Cellsâ gate. This will open a new scatterplot that only displays gated cells.</li><li>Set the <em>y</em>-axis to FSC-H and the <em>x</em>-axis to FSC-A to exclude doublets. Single cells localize along the same diagonal, while doublets lie underneath (Figure 3).</li><li>Create another free-polygon gate to only include cells along the top diagonal. Click âOKâ to name it âSingle Cellsâ.</li><li>In the workspace window select the âSingle Cellsâ gate and apply it to all samples (simply drag and drop it on the above window on the âLive Cellsâ subgroup).</li><li>Adjust the âSingle Cellsâ gate to each sample individually as you did in Step E4d for the âLive Cellsâ gate.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218174100_2495.jpg" width="108"/><br/><strong>Figure 3. Doublets exclusion in FlowJo.</strong> FSC-H and FSC-A scatter plot generated by FlowJo that illustrate single cells lying on the main diagonal (black polygon gate) and cell doublets just underneath it (red dashed gate).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Obtain a mean fluorescence intensity (MFI) table<ol class="npwthree"><li>Back to the workspace, select the <em>Table Editor</em> and click the âEditâ tab. Click âAdd Columnâ and select âMeanâ on the left, and the âSingle Cellsâ Population and the âPE-Tx-Red-YG-Aâ Parameter from the dropdown menus on the right.</li><li>Click OK and go back to the âTable Editorâ tab. Click on âCreate Tableâ. This generates a table with the mean fluorescence intensities for all the samples and the unlabeled controls. To download the table as an Excel file, from the dropdown menus to the right of the âCreate Table buttonâ choose âTo Fileâ and âExcelâ. Choose a âDestinationâ folder and hit âCreate Tableâ again.</li><li>Quit FlowJo saving the workspace.</li></ol></li></ol><br/></li><li>Obtain the absolute amount of your protein of interest<ol class="npwtwo"><li>Open the table generated by FlowJo in Microsoft Excel.</li><li>For each TMR-labeled cell line, subtract the MFI of the unlabeled sample (background fluorescence, <em>bkg</em>) from the MFI of each TMR-labeled sample to obtain the adjusted MFI as follows:<br/><br/><div style="text-align:center;"><span style="font-size:10pt;"><em>MFI</em><sub><em>adj</em></sub><em> = MFI<sub>TMR</sub> -  MFI<sub>bkg</sub></em><br/><br/></span> </div></li><li><span style="font-size:10pt;">Use the MFI<sub>adj</sub> values to calculate the absolute abundance of the protein of interest (n<sub><em>POI</em></sub>) as follows:</span> </li><br/><div style="text-align:center;"><span style="font-size:10pt;">n<em><sub>POI </sub></em>= (<em>MFI<sub>adj,COI</sub></em>) Ã· (<em>MFI<sub>adj,C32</sub></em>) Ã 109,800</span> </div><br/>where MFI<sub>adj,COI</sub> is the adjusted mean fluorescence intensity of the cell line of interest (COI) that expresses the Halo-tagged protein under investigation, MFI<sub>adj,C32</sub> is the background-subtracted mean fluorescence intensity of the C32 cell standard and 109,800 is the absolute amount of Halo-CTCF we estimated in our standard U2OS C32 cell line.<br/><br/><li>Average <em>n</em><sub>POI</sub> obtained over several biological replicates to get mean and standard deviation values.</li><li>Table 1 details a FlowJo re-analysis of the original data published in Cattoglio <em>et al.</em> (2019). You can download the raw FCS 3.0 files from github at <a href="https://gitlab.com/tjian-darzacq-lab/cattoglio_et_al_absoluteabundance_2019/tree/master/Data" target="_blank">https://gitlab.com/tjian-darzacq-lab/cattoglio_et_al_absoluteabundance_2019/tree/master/Data</a>. Note that FSC/SSC gating can affect final absolute estimates, because cells of different size will also have different protein amounts (the gating applied in FlowJo for these samples is in Figure 2). The numbers we report here for mouse embryonic stem cells thus vary from those we initially published. Specifically, we now estimate ~539,000 Â± 36,500 Sox2 molecules (vs. ~460,500 Â± 25,600 in Cattoglio <em>et al.</em>, 2019) and ~111,000 Â± 32,000 TBP molecules (vs. ~99,100 Â± 29,100 in Cattoglio <em>et al.</em>, 2019)<br/><br/><strong>Table 1. Absolute quantification of Sox2 and TBP in mouse embryonic stem cells.</strong> For each cell line, where a different protein of interest (POI) is endogenously Halo-tagged, the table reports flow cytometry fluorescence values (MFI, mean fluorescence intensity) obtained in 4 biologically independent replicates, separated by commas. We then subtract the background fluorescence of the unlabeled sample to obtain adjusted MFI values (MFI<sub>adj</sub>). We finally use these values to calculate absolute abundances of Sox2 and TBP in the mouse embryonic stem cell (mESC) lines C3 and C41, as detailed in Steps F2 to F4. The <em>n</em><sub>POI</sub> we report in the last column is the absolute quantification averaged across the 4 replicates Â± standard deviation.<br/><table border="0" cellpadding="0" cellspacing="0" class="MsoNormalTable ke-zeroborder" width="850"><tbody><tr style="border-top:2pt;border-left:0;border-bottom:1px;border-right:0;border-style:solid;"><td>Cell line</td><td>POI</td><td>MFI (A.U.)</td><td>MFI<sub>adj</sub> </td><td><i>n</i><sub>POI</sub> </td></tr><tr><td>U2OS_unlabeled</td><td></td><td>574, 1071, 1216, 513</td><td>Subtracted from C32</td><td></td></tr><tr><td>U2OS_C32_TMR</td><td>CTCF</td><td>7761, 10883, 10978, 5285</td><td>7187, 9812, 9762, 4772</td><td>109,800; standard</td></tr><tr><td>mESC_unlabeled</td><td></td><td>451, 616, 640, 309</td><td>Subtracted from C3 and C41</td><td></td></tr><tr><td>mESC_C3_TMR</td><td>Sox2</td><td>32260, 49995, 49373, 25052</td><td>31809, 49379, 48733, 24743</td><td>538,996 Â± 36,512</td></tr><tr style="border-top:0pt;border-left:0;border-bottom:2pt;border-right:0;border-style:solid;"><td>mESC_C41_TMR</td><td>TBP</td><td>6434, 9036, 9562, 7175</td><td>5983, 8420, 8922, 6866</td><td>110,991 Â± 31,549</td></tr></tbody></table>A.U.: fluorescence arbitrary units</li></ol></li></ol><p class="pptt" id="biaoti31472">### Data analysis</p><p class="ppzz">Data analysis only entails the calculations detailed above (protocol Steps F2 to F4). Table 1 shows intermediary and final data after re-analysis of a subset of the TMR-labeling experiments first described in Cattoglio <em>et al.</em> (2019). The raw files are available for download at <a href="https://gitlab.com/tjian-darzacq-lab/cattoglio_et_al_absoluteabundance_2019/tree/master/Data" target="_blank">https://gitlab.com/tjian-darzacq-lab/cattoglio_et_al_absoluteabundance_2019/tree/master/Data</a>.</p><p class="pptt" id="biaoti31473">### Recipes</p><ol class="npwtwok"><li>10x PBS, pH 7.4 (1 liter)<ol class="npwthree"><li>To 800 ml of double-distilled water (ddH<sub>2</sub>O) add NaCl (80 g), KCl (2 g), Na<sub>2</sub>HPO<sub>4</sub> (14.4 g) and KH<sub>2</sub>PO<sub>4</sub> (2.4 g). Stir to dissolve</li><li>Wear eye protection and adjust the pH to 7.4 with 6 N HCl (Thermo Fisher Scientific To prepare a 6 N HCl solution dilute the concentrated acid 1:1 with ddH<sub>2</sub>O)</li><li>Bring the volume up to 1 liter with ddH<sub>2</sub>O and dispense 100 ml into autoclavable glass bottles</li><li>Autoclave 30 min on liquid cycle</li></ol></li><li>1x PBS, pH 7.4<br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub><br/>2 mM KH<sub>2</sub>PO<sub>4</sub><br/>Dilute from a 10x autoclaved solution (Recipe 1) and re-autoclave<br/><em>Note: You can also purchase ready-to-use, sterile 1x PBS. Storage temperature: 15-30 Â°C. Shelf-life: 12 months from date of preparation.</em> </li><li>0.1 M EDTA, pH 8.0 (1 L)<ol class="npwthree"><li>To 700 ml of ddH<sub>2</sub>O add 37.22 g of EDTA and stir (EDTA will not dissolve until the pH is near to 8.0)</li><li>Adjust the pH to 8.0 with NaOH pellets (wear eye protection and mask)</li><li>Bring the solution up to volume with ddH<sub>2</sub>O<br/></li><li>Autoclave 30 minutes on liquid cycle</li></ol></li><li>0.05% Trypsin<ol class="npwthree"><li>To 1 liter of 1x PBS add 10 ml 0.1 M EDTA, pH 8.0</li><li>Autoclave for 30 min, prepare 50 ml aliquots and add 1 ml of Gibco â 2.5% trypsin stock solution to each aliquot</li><li>Store aliquots at -20 Â°C</li></ol></li><li>U2OS media<br/>To a 500-ml low-glucose DMEM bottle add 50 ml of FBS (~10% final) and 5 ml of pen/strep mixture</li><li>U2OS freezing media<ol class="npwthree"><li>To 1 ml of DMSO add 9 ml U2OS media (10% DMSO final). Open the DMSO in the biosafety cabinet to ensure sterility and aliquot the remaining DMSO to 15 ml conical tubes (1 ml aliquots)</li><li>Store the freezing media in the fridge for up to 1 month and the DMSO aliquots at -20 Â°C for 12 months</li></ol></li></ol><p class="pptt" id="biaoti31474">### Acknowledgments</p><p class="ppzz">ASH acknowledges support from a Siebel Stem Cell Institute post-doctoral fellowship and NIH NIGMS K99 Pathway to Independence Award K99GM130896. The Tjian-Darzacq lab is supported by NIH Common Fund 4D Nucleome Program U01-EB021236 and U54-DK107980 (XD), the California Institute of Regenerative Medicine grant LA1-08013 (XD), and by the Howard Hughes Medical Institute (003061, RT). The protocol detailed here was first described in Cattoglio <em>et al.</em> (2019) and it used Halo-tagged Sox2 and TBP mESC lines previously generated by Sheila Teves (Teves <em>et al.</em>, 2016 and 2018). We thank Elena Slobodyanyuk for testing this protocol and Nike Walther for providing comments to this manuscript.</p><p class="pptt" id="biaoti31475">### Competing interests</p><p class="ppzz">The authors declare no competing financial/non-financial interests.</p><p class="pptt" id="biaoti31476">### References</p><ol class="npwtwok"><li>Abcam (2019). <a href="https://www.abcam.com/ps/pdf/protocols/cell_culture.pdf" target="_blank"><em>Cell culture guidelines</em>.</a> Accessed 24 October 2019.</li><li>Addgene. <a href="https://www.addgene.org/crispr/guide/" target="_blank"><em>CRISPR Guide</em>.</a> Accessed 24 October 2019.</li><li>Aird, E. J., Lovendahl, K. N., St Martin, A., Harris, R. S. and Gordon, W. R. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30271937" target="_blank">Increasing Cas9-mediated homology-directed repair efficiency through covalent tethering of DNA repair template.</a> <em>Commun Biol</em> 1: 54.<br/></li><li>ATCC. <a href="https://www.atcc.org/~/media/PDFs/Culture Guides/AnimCellCulture_Guide.ashx" target="_blank">Animal cell culture guide.</a> Accessed 24 October 2019.</li><li>Bantscheff, M., Lemeer, S., Savitski, M. M. and Kuster, B. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22772140" target="_blank">Quantitative mass spectrometry in proteomics: critical review update from 2007 to the present.</a> <em>Anal Bioanal Chem</em> 404(4): 939-965. </li><li>Bantscheff, M., Schirle, M., Sweetman, G., Rick, J. and Kuster, B. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17668192" target="_blank">Quantitative mass spectrometry in proteomics: a critical review.</a> <em>Anal Bioanal Chem</em> 389(4): 1017-1031.</li><li>Bennett, R. J., Simpson, D. M., Holman, S. W., Ryan, S., Brownridge, P., Eyers, C. E., Colyer, J. and Beynon, R. J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28368040" target="_blank">DOSCATs: Double standards for protein quantification.</a> <em>Sci Rep</em> 7: 45570.<br/></li><li>Cai, Y., Hossain, M. J., HÃ©richÃ©, J. K., Politi, A. Z., Walther, N., Koch, B., Wachsmuth, M., Nijmeijer, B., Kueblbeck, M., Martinic-Kavur, M., Ladurner, R., Alexander, S., Peters, J. M. and Ellenberg, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30202089" target="_blank">Experimental and computational framework for a dynamic protein atlas of human cell division.</a> <em>Nature</em> 561(7723): 411-415.</li><li>Cattoglio, C., Pustova, I., Walther, N., Ho, J. J., Hantsche-Grininger, M., Inouye, C. J., Hossain, M. J., Dailey, G. M., Ellenberg, J., Darzacq, X., Tjian, R. and Hansen, A. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31205001" target="_blank">Determining cellular CTCF and cohesin abundances to constrain 3D genome models.</a> <em>Elife</em> 8: e40164.</li><li>Chen, J. Q., Heldman, M. R., Herrmann, M. A., Kedei, N., Woo, W., Blumberg, P. M. and Goldsmith, P. K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23896461" target="_blank">Absolute quantitation of endogenous proteins with precision and accuracy using a capillary Western system.</a> <em>Anal Biochem</em> 442(1): 97-103.</li><li>ECACC. (2019). <a href="https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma-Aldrich/General_Information/1/fundamental-techniques-in-cell-culture.pdf" target="_blank">Fundamental techniques in cell culture Fundamental techniques in cell culture.</a> Accessed 24 October 2019.</li><li>England, C. G., Luo, H. and Cai, W. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25974629" target="_blank">HaloTag technology: a versatile platform for biomedical applications.</a> <em>Bioconjug Chem</em> 26(6): 975-986.</li><li>Ghaemmaghami, S., Huh, W. K., Bower, K., Howson, R. W., Belle, A., Dephoure, N., O'Shea, E. K. and Weissman, J. S. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14562106" target="_blank">Global analysis of protein expression in yeast.</a> <em>Nature</em> 425(6959): 737-741.</li><li>Hansen, A. S., Pustova, I., Cattoglio, C., Tjian, R. and Darzacq, X. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28467304" target="_blank">CTCF and cohesin regulate chromatin loop stability with distinct dynamics.</a> <em>Elife</em> 6: e25776.</li><li>Haupt, A., Grancharova, T., Arakaki, J., Fuqua, M. A., Roberts, B. and Gunawardane, R. N. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30199041" target="_blank">Endogenous protein tagging in human induced pluripotent stem cells using CRISPR/Cas9.</a> <em>J Vis Exp</em>(138). doi: 10.3791/58130.</li><li>Holzmann, J., Politi, A. Z., Nagasaka, K., Hantsche-Grininger, M., Walther, N., Koch, B., Fuchs, J., DÃ¼rnberger, G., Tang, W., Ladurner, R., Stocsits, R. R., Busslinger, G. A., Novak, B., Mechtler, K., Davidson, I. F., Ellenberg, J. and Peters, J. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31204999" target="_blank">Absolute quantification of cohesin, CTCF and their regulators in human cells.</a> <em>Elife</em> 8: e46269.</li><li>Kovalenko, E. I., Ranjbar, S., Jasenosky, L. D., Goldfeld, A. E., Vorobjev, I. A. and Barteneva, N. S. (2011). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3189897/" target="_blank">The use of HaloTag-based technology in flow and laser scanning cytometry analysis of live and fixed cells.</a> <em>BMC Res Notes</em> 4: 340.</li><li>LifeTechnologies. (2019). <a href="https://www.vanderbilt.edu/viibre/CellCultureBasicsEU.pdf" target="_blank">Cell culture basics.</a> Accessed on 24 October 2019.</li><li>Liu, P., Li, Y., Lei, J. and Dong, L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30353511" target="_blank">Creating knockin alleles in mouse embryonic stem cells by CRISPR/Cas9-mediated homologous recombination without drug selection.</a> <em>Methods Mol Biol</em> 1874: 115-137.</li><li>Los, G. V., Encell, L. P., McDougall, M. G., Hartzell, D. D., Karassina, N., Zimprich, C., Wood, M. G., Learish, R., Ohana, R. F., Urh, M., Simpson, D., Mendez, J., Zimmerman, K., Otto, P., Vidugiris, G., Zhu, J., Darzins, A., Klaubert, D. H., Bulleit, R. F. and Wood, K. V. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18533659" target="_blank">HaloTag: a novel protein labeling technology for cell imaging and protein analysis.</a> <em>ACS Chem Biol</em> 3(6): 373-382.</li><li>Politi, A. Z., Cai, Y., Walther, N., Hossain, M. J., Koch, B., Wachsmuth, M. and Ellenberg, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29844523" target="_blank">Quantitative mapping of fluorescently tagged cellular proteins using FCS-calibrated four-dimensional imaging.</a> <em>Nat Protoc</em> 13(6): 1445-1464.</li><li>Presman, D. M., Ball, D. A., Paakinaho, V., Grimm, J. B., Lavis, L. D., Karpova, T. S. and Hager, G. L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28315485" target="_blank">Quantifying transcription factor binding dynamics at the single-molecule level in live cells.</a> <em>Methods</em> 123: 76-88.</li><li>Roberts, B., Haupt, A., Tucker, A., Grancharova, T., Arakaki, J., Fuqua, M. A., Nelson, A., Hookway, C., Ludmann, S. A., Mueller, I. A., Yang, R., Horwitz, R., Rafelski, S. M. and Gunawardane, R. N. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28814507" target="_blank">Systematic gene tagging using CRISPR/Cas9 in human stem cells to illuminate cell organization.</a> <em>Mol Biol Cell</em> 28(21): 2854-2874.</li><li>Sharma, A., Toepfer, C. N., Ward, T., Wasson, L., Agarwal, R., Conner, D. A., Hu, J. H. and Seidman, C. E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29364519" target="_blank">CRISPR/Cas9-mediated fluorescent tagging of endogenous proteins in human pluripotent stem cells.</a> <em>Curr Protoc Hum Genet</em> 96: 21 11 21-21 11 20.</li><li>Tasan, I., Sustackova, G., Zhang, L., Kim, J., Sivaguru, M., HamediRad, M., Wang, Y., Genova, J., Ma, J., Belmont, A. S. and Zhao, H. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29912475" target="_blank">CRISPR/Cas9-mediated knock-in of an optimized TetO repeat for live cell imaging of endogenous loci.</a> <em>Nucleic Acids Res</em> 46(17): e100.</li><li>Teves, S. S., An, L., Bhargava-Shah, A., Xie, L., Darzacq, X. and Tjian, R. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29939130" target="_blank">A stable mode of bookmarking by TBP recruits RNA polymerase II to mitotic chromosomes.</a> <em>Elife</em> 7: e35621.</li><li>Teves, S. S., An, L., Hansen, A. S., Xie, L., Darzacq, X. and Tjian, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27855781" target="_blank">A dynamic mode of mitotic bookmarking by transcription factors.</a> <em>Elife</em> 5: e22280.</li><li>Walther, N., Hossain, M. J., Politi, A. Z., Koch, B., Kueblbeck, M., Odegard-Fougner, O., Lampe, M. and Ellenberg, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29632028" target="_blank">A quantitative map of human Condensins provides new insights into mitotic chromosome architecture.</a> <em>J Cell Biol</em> 217(7): 2309-2328.</li><li>Zhang, X. R., He, J. B., Wang, Y. Z. and Du, L. L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29703785" target="_blank">A cloning-free method for CRISPR/Cas9-mediated genome editing in fission yeast.</a> <em>G3 (Bethesda)</em> 8(6): 2067-2077.<br/></li></ol>