<body>
            <h1>Construction of Antisense RNA-mediated Gene Knock-down Strains in the Cyanobacterium <em>Anabaena</em> sp. PCC 7120</h1>
            <p><strong>Authors:</strong> Amit   Srivastava, Anand  Ballal, Karl  Forchhammer, Anil Kumar Tripathi</p>
            <h2>Abstract</h2>
            <p><em>Anabaena</em> sp. PCC 7120 (hereafter <em>Anabaena</em>) is a model cyanobacterium to study nitrogen fixation, cellular differentiation and several other key biological functions that are analogous in plants. As with any other organism, many genes in <em>Anabaena</em> encode an essential life function and hence cannot be deleted, causing a bottleneck in the elucidation of its genomic function. Antisense RNA (asRNA) mediated approach renders the study of essential genes possible by suppressing (but not completely eliminating) expression of the target gene, thus allowing them to function to some extent. Recently, we have successfully implemented this approach using the strong endogenous promoter of the <em>psbA1</em> gene (D1 subunit of Photosystem II) introduced into a high-copy replicative plasmid (pAM1956) to suppress the transcript level of the target gene <em>alr0277</em> (encoding a sigma factor, SigJ/Alr0277) in <em>Anabaena</em>. This protocol represents an efficient and easy procedure to further explore the functional genomics, expanding the scope of basic and applied research in these ecologically important cyanobacteria.</p>
            <p><strong>Keywords:</strong> AntisenseRNA (asRNA), Knock-down, <em>Anabaena</em> PCC 7120, Cyanobacteria, Green fluorescent protein, <em>psbA1</em> promoter</p>
            <h2>Background</h2><p>Cyanobacteria, a diverse phylum of aerobic bacteria capable of photosynthesis, require light (solar energy), carbon dioxide and trace elements for growth. They are genetically amenable due to the availability of facile molecular biology techniques and efficient conjugation systems for the transfer of foreign genes into them (Wolk <em>et al.</em>, 1984; Elhai <em>et al.</em>, 1997). <em>Anabaena</em> is a filamentous cyanobacterium, which is capable of cellular differentiation, wherein specialised cells (termed heterocysts) carry out nitrogen fixation. Classical strategies such as gene knockout to disrupt the function of a âgene of interestâ have been widely employed in cyanobacteria in order to understand their function. Following transformation, segregation of the polyploid genome is required to obtain homozygous mutants. In the case of many essential genes (for example, GroEL, LexA), however, the mutants are not viable or cannot segregate completely as the target protein is essential for its survival (Rajaram and Apte, 2008; Kumar <em>et al.</em>, 2018). In such cases, an option is to knockdown the genes of interest by targeting them using asRNA (Blanco <em>et al.</em>, 2011; Lin <em>et al.</em>, 2013) or dCas9-based CRISPR approaches (Tian <em>et al.</em>, 2017), and then study the down-regulated/knockdown phenotype.<br/> <br/>The dCas9-based approach has its own limitations, for example, in some organisms, expression of the dCas9 protein may be toxic to cells wherein it is expressed (Lee <em>et al.</em>, 2016; Zhang and Voigt, 2018). The reason for the toxicity of Cas9 in cyanobacteria remains unclear, nonetheless recently the employment of a novel RNA directed dsDNA nuclease Cpf1 from <em>Francisella novicida</em> showed far less toxicity in cyanobacteria (Ungerer and Pakrasi, 2016; Niu <em>et al.</em>, 2018). Although CRISPR-based gene editing approaches are being rapidly optimized, they need further development to be efficiently used in future cyanobacterial gene knockdown procedures. In such cases, it would be more fruitful to use another methodology such as asRNA to create knockdown strains. The asRNA, which is the complement of its respective mRNA, binds specifically to the mRNA (<em>i.e.</em>, forms a dsRNA), thereby reducing its ability to be translated by the ribosomal machinery. Due to decreased translation, a reduction in expression of the desired protein (knockdown) is achieved. Recently, we have used asRNA-mediated approach to downregulate a sigma factor SigJ (Alr0277) by using native <em>psbA1</em> promoter for expression of asRNA and achieved a 3-fold reduction in the <em>sigJ</em> transcript (Srivastava <em>et al.</em>, 2017). Similarly, this approach was also employed to downregulate <em>in vivo</em> expression of the Alr3183 protein (an atypical 2 cysteine-containing thiol peroxidase) in <em>Anabaena</em> (Tailor and Ballal, 2017). Here we present the detailed protocol to knockdown genes in <em>Anabaena</em> for applications in basic as well as applied research.</p>
        </body>

<p class="pptt" id="biaoti31332">## Materials and Reagents</p><ol class="npwone"><li>Vessel and consumable materials<ol class="npwtwo"><li>Pipette tips</li><li>Petri dishes, 100 mm (Thermo Fisher Scientific, catalog number: FB0875713)</li><li>1.5 ml centrifuge tubes (Thermo Fisher Scientific, catalog number: 02-682-550)</li><li>15 ml conical centrifuge tubes (Thermo Fisher Scientific, catalog number: 05-527-90)</li><li>50 ml conical centrifuge tubes (Thermo Fisher Scientific, catalog number: 12-565-270)</li><li>Supor<sup>Â®</sup> PES membrane disc filters (Pall corporation, catalog number: 60110)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Biological materials<ol class="npwtwo"><li>pAM1956, a replicative vector in <em>Anabaena</em> (Yoon and Golden, 1998) (Addgene, plasmid number: 40251)<br/><em>Note: The map of pAM1956 is available on Addgene; </em><a href="https://www.addgene.org/40251/)" target="_blank"><em>https://www.addgene.org/40251/</em></a><em>.</em> </li><li>pAM1956-P<sub><em>psbA1</em></sub> (control plasmid, available upon request)</li><li>pRL443 (Elhai <em>et al.</em>, 1997) (Addgene, plasmid number: 70261)</li><li>pRL623 (Elhai <em>et al.</em>, 1997) (Addgene, plasmid number: 58494)</li><li><em>Anabaena</em> PCC 7120 (can be obtained from Pasteur Culture Collection (PCC), Institut Pasteur, Paris, France)</li><li><em>E. coli</em> DH5Î±: F- <em>recA</em>41 <em>endA</em>1 <em>gyrA</em>96 <em>thi</em>-1 <em>hsd</em>R17 (rk<sup>-</sup> mk<sup>-</sup>) <em>supE</em>44 <em>relA</em>Î» Î<em>lacU</em>169 (Gibco-BRL)</li><li><em>E. coli</em> HB101: F- <em>mcrB mrr hsdS20</em> (<em>rB<sup>-</sup> mB<sup>-</sup></em>) <em>recA</em><em>13 </em><em>leuB</em><em>6 ara-14 proA2 lacY1 galK2 xyl-5 mtl-1 rpsL20</em> (<em>SmR</em>) <em>glnV44</em> Î»- (Promega, catalog number: L2015)</li><li><em>E. coli</em> HB101-R2: Donor strain carrying pRL623 (encoding methylases) and pRL443 (conjugal plasmid) (Elhai <em>et al.</em>, 1997; Banerjee <em>et al.</em>, 2012)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Chemicals for preparing media<ol class="npwtwo"><li>MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: 63138)</li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (Sigma-Aldrich, catalog number: C1016)</li><li>Citric Acid (Sigma-Aldrich, catalog number: 251275)</li><li>Ferric ammonium citrate (Sigma-Aldrich, catalog number: RES20400-A7)</li><li>Na<sub>2</sub>-EDTA (Sigma-Aldrich, catalog number: E5134)</li><li>Na<sub>2</sub>CO<sub>3</sub> (Sigma- Aldrich, catalog number: 1613757)</li><li>H<sub>3</sub>BO<sub>3</sub> (Sigma-Aldrich, catalog number: B6768)</li><li>Mn<sub>2</sub>Cl<sub>2</sub>Â·4H<sub>2</sub>O (Sigma-Aldrich, catalog number: 1375127)</li><li>ZnSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: 1.08881)</li><li>Na<sub>2</sub>MoO<sub>4</sub>Â·2H<sub>2</sub>O (Sigma-Aldrich, catalog number: M1003)</li><li>CuSO<sub>4</sub>Â·5H<sub>2</sub>O (Sigma-Aldrich, catalog number: C8027)</li><li>CoCl<sub>2</sub>Â·6H<sub>2</sub>O (Sigma-Aldrich, catalog number: 1.02539)</li><li>NaNO<sub>3</sub> (Sigma-Aldrich, catalog number: S5506) </li><li>NaHCO<sub>3</sub> (Sigma-Aldrich, catalog number: S5761)</li><li>K<sub>2</sub>HPO<sub>4</sub> (Sigma-Aldrich, catalog number: 1551128)</li><li>NaCl (Sigma-Aldrich, catalog number: 63138)</li><li>Tryptone (Casein Hydrolysate) (Sigma-Aldrich, catalog number: 22090)</li><li>Yeast extract (Sigma-Aldrich, catalog number: Y1625)</li><li>Agar (for <em>E. coli</em>) (Sigma-Aldrich, catalog number: A1296)</li><li>Bacto Agar (for <em>Anabaena</em>) (BD Diagonistics, catalog number: 90000-767)</li><li>Co(NO<sub>3</sub>)<sub>2</sub>Â·6H<sub>2</sub>O (Sigma-Aldrich, catalog number: 239267)</li><li>Sodium thiosulfate pentahydrate (Sigma-Aldrich, catalog number: 217247)</li><li>BG11 medium (see Recipes)</li><li>BG11 agar plate (see Recipes)</li><li>1 M TES (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Molecular biology reagents<ol class="npwtwo"><li>Ampicillin sodium salt (Sigma-Aldrich, catalog number: A9393)</li><li>Chloramphenicol (Sigma-Aldrich, catalog number: C0378)</li><li>Kanamycin sulfate (Sigma-Aldrich, catalog number: 60615)</li><li>Neomycin trisulfate salt (Sigma-Aldrich, catalog number: N1876)</li><li>Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System (Promega, catalog number: A9281)</li><li>QIAGEN Plasmid Mini Kit (100) (Qiagen, catalog number: A9281)</li><li>Agarose (Sigma-Aldrich, catalog number: A9539)</li><li>Cfr9I (XmaI) (10 U/Î¼l) (Thermo Fisher Scientific, catalog number: ER0171)</li><li>Reverse Transcriptase (Roche Diagnostics GmbH, catalog number: 38823100)</li><li>Ambion<sup>TM</sup> DNase I (RNase-free) (Thermo Fisher Scientific, catalog number: AM2222)</li><li>Gibson assembly master mix (NEB, catalog number: E2611L)</li><li>Phenol (Sigma-Aldrich, catalog number: P1037)</li><li>Glycerol (Sigma-Aldrich, catalog number: G5516)</li><li>8-hydroxyquinoline (Sigma-Aldrich, catalog number: 252565)</li><li>Na<sub>2</sub>-EDTA (Sigma-Aldrich, catalog number: E5134)</li><li>Sodium acetate (Sigma-Aldrich, catalog number: S2889)</li><li>Guanidine thiocyanate (Sigma-Aldrich, catalog number: G9277)</li><li>Guanidine hydrochloride (Sigma-Aldrich, catalog number: G3272)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: X100)</li><li>PGTX solution (see Recipes; compositions listed points 12-19)</li></ol></li></ol><p class="pptt" id="biaoti31333">### Equipment</p><ol class="npwtwok"><li>Pipetts</li><li>Tweezers, 120 mm (Ideal-Tek, catalog number: 36A-SA)</li><li>Erlenmeyer flask (Thermo Fisher Scientific, catalog number: FB-500-150)</li><li>NanoDrop (Thermo Fisher Scientfic, catalog number: ND-1000)</li><li>Gel documentation machine (Bio-Rad, Gel Doc 2000 Imaging System)</li><li>Fluorescence microscope, excitation filter BP470/40 nm, emission filter BP 525/50, Leica DFC360FX black and white camera (Leica Camera Inc., catalog number: DM5500B)</li><li>Light microscope, 100/1.3 oil objective, Leica DFC420C camera (Leica Camera Inc., catalog number: DM2500)</li><li>PCR thermocycler (Thermo Fisher Scientific, Veriti<sup>Â®</sup> Thermal Cycler)</li><li>UV-Vis spectrophotometer (Analytik Jena AG, Germany, SPECORD<sup>Â®</sup> 205)</li><li>Water bath sonicator (Bandelin Souvrex NK51)</li><li>Centrifuge (Eppendorf 5415R Refrigerated Centrifuge)</li><li>Incubator (New Brunswick<sup>TM</sup> Innova<sup>Â®</sup> 43)</li><li>Vacuum filter system (Thermo Fisher Scientific, Nalgene<sup>TM</sup> 300-4100)</li></ol><p class="pptt" id="biaoti31334">### Software</p><ol class="npwtwok"><li>ImageJ (NIH, USA, <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>)</li><li>Geneious (Biomatters Ltd., <a href="https://www.geneious.com/" target="_blank">https://www.geneious.com</a>)<br/></li></ol><p class="pptt" id="biaoti31335">### Procedure</p><p class="ppzz"><img alt="" class="layerphoto" height="107" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218035751_5088.jpg" width="100"/><br/><strong>Figure 1. Schematic representation of steps for construction of asRNA mediated gene knockdown mutant in </strong><em><strong>Anabaena</strong></em><strong>.</strong> Abbreviations: Km, Kanamycin; Neo12.5, Neomycin (12.5 Âµg/ml); Neo25, Neomycin (25 Âµg/ml); <em>npt</em>, neomycin phosphotransferase; <em>bla</em>, beta-lactamase; <em>cat</em>, chloramphenicol acetyltransferase.<br/><br/><em>Note: The handling of cyanobacterial cultures should be done exclusively under sterile conditions.</em> </p><br/><ol class="npwone"><li>DNA manipulation<ol class="npwtwo"><li>Design primers for (a) promoter <em>e.g.</em>, P<em><sub>psbA1</sub></em> (<em>alr4866</em>; encoding Photosystem II protein D1) and (b) target gene in reverse orientation (hereafter asGENE) and amplify the DNA fragments by PCR. All the steps in this section are illustrated in Figure 1A. The primers can be designed either manually or using programs such as Geneious (<a href="https://www.geneious.com/" target="_blank">https://www.geneious.com</a>). Primers used for construction and confirmation of knock-down mutant of <em>alr0277</em> gene in <em>Anabaena</em> (Srivastava <em>et al.</em>, 2017) are listed in Table 1.<br/><em>Note: In this protocol, use of the Gibson assembly, as described by the manufacturer (Gibson </em><em>et al.</em><em>, 2009), was employed. Primer designing in the Gibson assembly method does not require the addition of restriction sites in primers.</em><br/><br/><strong>Table 1. Primers used for construction and confirmation of knock-down mutant of </strong><em><strong>alr0277</strong></em><strong> gene in </strong><em><strong>Anabaena</strong></em><strong> (Srivastava </strong><em><strong>et al.</strong></em><strong>, 2017)</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218040408_1166.jpg" width="600"/><br/><br/></li><li>Linearize the promoterless-<em>gfpmut2</em> containing replicative shuttle vector pAM1956 (Yoon and Golden, 1998) using the XmaI restriction enzyme.</li><li>Check all three fragments on 1.5% (w/v) agarose gel for the desired size and recover them by gel-purification. Measure the concentration and purity of the purified fragments on a NanoDrop instrument.</li><li>Combine promoter, asGENE and linearized plasmid (pAM1956-XmaI), in the Gibson assembly reaction (contains the enzymes: T5 Exonuclease, Phusion DNA Polymerase and Taq DNA Ligase) and follow the manufacturer's instructions.</li><li>Transform the ligation mix into competent <em>E. coli</em> HB101, spread on a LB agar plate (1.5% agar) supplemented with kanamycin (50 Î¼g/ml) and wait for the colonies to grow overnight at 37 Â°C. The competent <em>E. coli</em> HB101 cells can be prepared by treatment with CaCl<sub>2</sub> (Sambrook <em>et al.</em>, 2006).</li><li>Once the colonies have appeared, they can be confirmed by colony-PCR by using the forward primer of P<em><sub>psbA1</sub></em> and the reverse primer of asGENE.<br/><br/></li></ol></li><li>Construction of recombinant <em>Anabaena</em> strain<ol class="npwtwo"><li>Grow <em>Anabaena</em> in 25 ml BG11 medium (Rippka <em>et al.</em>, 1979) (Recipe 1) under continuous light illumination (40 Î¼mol photons/m<sup>2</sup>/s) at 30 Â°C and shaking at 150 rpm for 4-5 days until it is in the exponential phase (OD<sub>730</sub> ~0.5). All the steps in this section are illustrated in Figure 1B.</li><li>Collect the cells by centrifugation at 4,000 <em>x g</em> for 10 min at room temperature and resuspend the cell pellet in 25 ml BG11 media in a 150 ml Erlenmeyer flask.</li><li>Break filaments by sonicating the cultures for 60-120 s, using a water bath sonicator to an average 3-5 cell length and confirm it by bright field microscopy.</li><li>Grow the broken filaments for one day under continuous light illumination (40 Î¼mol photons/m<sup>2</sup>/s) at 30 Â°C and shaking at 150 rpm.</li><li>On the same day, inoculate the colonies of <em>E. coli</em> HB101 bearing the cargo plasmid (Km<sup>R</sup>), which has been constructed in the step 5 (section A), and <em>E. coli</em> HB101-R2 harboring pRL443 (Ap<sup>R</sup>) and pRL623 (Cm<sup>R</sup>) in 2 ml LB broth with 50 Î¼g/ml kanamycin and 100 Î¼g/ml ampicillin + 25 Î¼g/ml chloramphenicol respectively for 12-16 h at 37 Â°C at 200 rpm.</li><li>Next day, sub-inoculate 100 Î¼l of the overnight-grown cultures of the above-mentioned <em>E. coli</em> strains into 5 ml of fresh LB liquid media supplemented with appropriate antibiotics and allow them to grow until they reach an exponential phase (OD<sub>600</sub> ~0.5).</li><li>Harvest the 5 ml culture by centrifugation at 4,000 <em>x g</em> in a table centrifuge for 10 min at 25 Â°C (room temperature).</li><li>Wash the cell pellets three times with 1 ml LB to remove antibiotics completely, and then add 200 Î¼l LB to re-suspend the pellets in both. Do not vortex the cells and if required, use a sterile pipette to gently re-suspend the <em>E. coli</em> cells.</li><li>For the mating experiment, mix 100 Î¼l HB101 bearing the cargo plasmid with 100 Î¼l HB101-R2 (harboring pRL443 and pRL623) and incubate at room temperature for 30 min (experimental group). For negative control, mix 100 Î¼l LB with 100 Î¼l HB101-R2 and incubate at room temperature for 30 min (control group).<br/><em>Note: We have used an </em><em>E. coli</em><em> strain (HB101) that contains both plasmids pRL443 and pRL623, which differs from the original method (Elhai and Wolk, 1988) where one strain contains conjugal plasmid (pRL443), and other strain contains the helper plasmid (pRL623) and the cargo plasmid.</em> </li><li>The same day, harvest 15 ml <em>Anabaena</em> culture by centrifugation at 4,000 <em>x g</em> for 10 min at room temperature. Re-suspend the cell pellet in 1 ml BG11, transfer the cells into a 1.5 ml Eppendorf tube, and then centrifuge at 6,000 <em>x g</em> for 1 min at room temperature. Re-suspend the pellet in 400 Î¼l of 2x BG11; then divide cells equally into two tubes.</li><li>Mix 200 Î¼l of <em>Anabaena</em> resuspension with 200 Î¼l of the mated <em>E. coli</em> mixture (experimental group) or the control mixture (control group), respectively, and incubate at room temperature and low light (20 Î¼mol photons/m<sup>2</sup>/s) overnight.</li><li>Spread the conjugal mixtures onto the autoclaved Supor<sup>Â®</sup> PES Membrane Disc Filters (Supor-800, 0.8 Î¼m, 47 mm) placed on BG11 agar plate (1.5% agar, Recipe 2) containing 12.5 Î¼g/ml neomycin and incubate at 30Â°C, with continuous low light illumination (20 Î¼mol photons/m<sup>2</sup>/s).</li><li>Transfer the membrane to a fresh BG11 agar plate (25 Î¼g/ml neomycin) every fourth day until neomycin-resistant colonies gradually appear after 15-20 days.</li><li>Observe the <em>Anabaena</em> colonies under a fluorescent microscope and select the colonies emitting GFP fluorescence (100x/1.3 oil objective lens, an excitation filter BP 470/40 nm and emission filter BP 525/50 nm) for further experiments.</li><li>Pick up the desired colony with a sterile inoculation loop or a toothpick and inoculate into a sterile glass tube (3-4 ml total volume) containing 250 Î¼l of BG11 medium with 25 Î¼g/ml neomycin. Keep this tube under continuous low light illumination (20 Î¼mol photons m<sup>2</sup>/s). After 3-4 days, once the visual increase in growth is observed, add another 250-500 Î¼l of BG11 medium (25 Î¼g/ml neomycin). Once the OD<sub>730</sub> of this culture reaches 0.3-0.4, transfer cells into 5 ml of BG11 medium (in a 25 ml sterile glass tube) containing neomycin as mentioned above. Now the tube can be placed under continuous light illumination of 40 Î¼mol photons/m<sup>2</sup>/s. After sufficient growth is observed (OD<sub>730</sub> ~0.6-1.0), cells can be transferred to a 150 ml sterile Erlenmeyer flask containing 25 ml BG11 medium (25 Î¼g/ml neomycin). Now cells can be subcultured as required in the above-mentioned medium for routine maintenance or for performing experiments. All these steps can be performed at 24-26 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Confirmation of expression of asRNA and downregulation of target gene<ol class="npwtwo"><li>To confirm the expression of asRNA by a semi-quantitative PCR approach, design the forward primer internal to asGENE and reverse primer internal to <em>gfpmut2</em> gene. To confirm the downregulation of the target gene, design the forward primer from the upstream sequence of the target gene that is present in the genome but absent in the antisense construct so that it can be selectively amplified from cDNA derived from GENE mRNA and not from its asRNA. All the steps in this section are illustrated in Figure 1C.<br/><em>Note: A regular PCR was used for amplification after the reverse transcriptase reaction. However, qRT-PCR can also be employed to evaluate differences in the expression of mRNA between the wild-type and the knockdown strain, which is a more sensitive technique. Alternatively, if antibodies against the proteins that are knocked down are available, then Western blotting may also be employed to monitor the reduction in the protein content.</em> </li><li>Filter 20 ml of exponential phase (OD<sub>730</sub> ~0.5) <em>Anabaena</em> culture on the membrane filter (Supor<sup>Â®</sup> PES membrane disc filters-supor-800, 0.8 Î¼m, 47 mm) using a vacuum pump assembly for harvesting the cells. Alternatively, cells may be also harvested by centrifugation (4,000 <em>x g</em>, 10 min) and reagents such as TRI Reagent (Sigma)/Trizol (Invitrogen) may be used to isolate total RNA using the manufacturerâs protocol.</li><li>Insert the filter containing cells in the vial containing PGTX solution (see Recipes) using tweezers and vortex until the filter is broken into small flakes and mixed with phenol properly. Keep samples on ice.</li><li>Freeze the samples in liquid nitrogen immediately, store at -80 Â°C and perform the RNA extraction exactly as described by Pinto <em>et al.</em> (2009).</li><li>Detect the RNA on a denaturing formaldehyde-agarose gel (1.5-2.0% w/v agarose). After staining with ethidium bromide, intact total RNA on migration will show sharp, clear 23S and 16S rRNA bands. Measure the RNA concentration and purity using the NanoDrop instrument as described by the manufacturer.</li><li>Treat the RNA (usually 1:10 dilution of original stock) with the DNase I enzyme and subsequently inactivate the DNase I by DNase inactivation reagent according to the manufacturerâs instruction.</li><li>DNA contamination of RNA preparations is not necessarily detected by gel electrophoresis or similar methods. To test if a detectable amount of genomic DNA is absent, use the diluted RNA sample as a template for PCR using primer pairs for a constitutive gene <em>rnpB</em>. Measure the RNA concentration and purity using the NanoDrop instrument.</li><li>Prepare cDNA using Reverse transcriptase with 1 Î¼g of RNA as template and 0.5 Î¼M of random primers, following the manufacturer's instructions. </li><li>PCR amplify a small fragment of the housekeeping gene <em>rnpB</em> as a control and target gene using specific primers in different PCR reactions, increasing the number of cycles from 25 to 36.</li><li>Detect the PCR products on an agarose gel and document the image.</li><li>Measure the intensity of PCR bands using ImageJ software for comparison of the transcript level.</li></ol></li></ol><p class="pptt" id="biaoti31336">### Data analysis</p><p class="ppzz">The gene expression data can be analyzed using ImageJ software (NIH, USA, <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>) as following:</p><ol class="npwtwo"><li>Record the picture of gel with a gel documentation machine and "invert" image color so that the cDNA bands appear black. Open your final gel image file (tiff) in ImageJ software for analysis.</li><li>All the bands in the image should be selected individually. Choose the ârectangularâ selection tool to select the widest band as the first band. The height of the selected area should be almost double of this band.</li><li>Mark the selected box as first by pressing CTRL + 1. As a result, number 1 will be displayed inside the selected box.</li><li>Move the pointer (cursor) inside the first selected box, click and drag the box area to the next band in the image.</li><li>Mark the selected box as second by pressing CTRL + 2. Number 2 will be displayed inside the selected box.</li><li>Repeat steps 4 and 5 to select rest of the bands in the image.</li><li>Once all the bands are selected press CTRL + 3 to display the histogram of each band.</li><li>To measure the area below each peak, first separate the histogram peaks by choosing the âdraw lineâ tool to draw a line across the lowest part of each of the histogram. Then choose the âMagic wandâ tool and click inside the histogram. The selected area will be outlined in yellow color and a âResultâ window will appear.</li><li>The Result window will show the quantitative intensity (value) of the each band.</li></ol><br/><p class="ppzz">The size of the peak will be registered as a percentage of the total size of the entire highlighted peaks. The peak percentage of test gene should be divided by the peak percentage of the housekeeping gene (control gene) to demonstrate the relative percentage of expression of each experimental gene. The relative expression should be measured in triplicates and statistical analyses should be performed considering the mean, standard deviation, minimum and maximum of the values of each sample. For details of the ImageJ procedure please refer to the user manual, section 30.13 (Ferreira and Rasb, 2012).</p><p class="pptt" id="biaoti31337">### Notes</p><p class="ppzz">For appropriate comparison with the knockdown strain that has been constructed as described above, it is recommended to construct a control strain by conjugating into the wild-type <em>Anabaena</em>, a plasmid that contains only the P<em><sub>psbA1</sub></em> promoter fused with promoter-less <em>gfpmut2</em> gene in pAM1956 (<em>i.e.</em>, vector control).</p><p class="pptt" id="biaoti31338">### Recipes</p><ol class="npwtwok"><li>BG11 media (modified from Rippka <em>et al.</em>, 1979)<br/><em>Note: This medium is used for growing </em><em>Anabaena</em><em>.</em> <ol class="npwthree"><li>Prepare 200 ml of each stock solution, 100 ml of trace element stock solution and 50 ml of 1 M NaHCO<sub>3</sub> (Table 2, Table 3 and Table 4 respectively) and sterilize them by autoclaving.<br/><em>Note: Store citric acid and ferric citrate at room temperature, protected from light; whereas other solutions should be stored at 4 Â°C. Filter sterilize the trace elements and NaHCO</em><sub><em>3</em></sub><em> stock.</em> </li><li>To prepare 1 L BG11 media (1x), add 5 ml of each 200x st.ock solution and 1 ml of trace element stock solution (1,000x) in 964 ml Milli-Q water<br/><br/><strong>Table 2. Stock solutions (200x) for BG11 media</strong><br/><img alt="" class="layerphoto" height="266" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218040723_5218.jpg" width="700"/><br/><br/><em>Note: The modified BG11 media includes adjustment in the level of following reagents:</em><br/><em> Na</em><sub><em>2</em></sub><em>CO</em><sub><em>3</em></sub><em>: Modified: 0.04 g/L; Original: 0.02 g/L</em><br/><em> Co(NO</em><sub><em>3</em></sub><em>)</em><sub><em>2</em></sub><em>Â·6H</em><sub><em>2</em></sub><em>O: Modified: 0.078 g/L; Original: 0.0494 g/L</em><br/><br/><strong>Table 3. Trace elements stock solution (1,000x)</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218040849_3018.jpg" width="700"/><br/><br/><strong>Table 4. 1 M NaHCO</strong><sub><strong>3</strong></sub><strong> stock solution</strong><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200218/20200218040945_0119.jpg" width="700"/><br/><br/></div></li></ol></li><li>BG11 1.5% (w/v) Agar plates<br/><ol class="npwthree"><li>To prepare 500 ml of <strong>2x concentrated Bacto</strong><sup><strong>TM</strong></sup><strong> agar</strong> in 1,000 ml bottle, add 30 g Bacto<sup>TM</sup> Agar in 500 ml of Milli-Q water</li><li>Then prepare <strong>2x BG11 medium</strong> in 500 ml bottle by adding 464 ml Milli-Q water and 5 ml of each 200x stock, 1 ml of trace element stock solution (1,000x)</li><li><strong>In addition</strong>, add 3 g/L Sodium thiosulfate pentahydrate and add 10 ml 1 M (1:100) TES, pH 7.6-8.2</li><li>Autoclave both 2x BG11 media and 2x agar</li><li>Combine both solutions when cooled to approx. 60 Â°C under the sterile hood</li><li>Add NaHCO<sub>3</sub> to a final concentration of 5 mM before pouring the plates</li><li>Add antibiotics if desired</li></ol><em>Note: Plates should be thick (use 30-40 ml of BG11 agar for 100 mm diameter plates) since cyanobacteria have long generation time and water evaporates throughout incubation. Additionally, the plates may also be sealed with parafilm.</em><br/></li><li>TES buffer<ol class="npwthree"><li>To prepare 1 M TES buffer, add 229.25 g of TES [N-{Tris(hydroxymethyl)methyl}-2-aminoethanesulfonic acid] in 750 ml of Milli-Q water</li><li>Adjust to pH 8.2 using 10 N NaOH</li><li>Fill to final volume of 1 L with dH<sub>2</sub>O</li><li>Filter sterilize (recommended) or autoclave</li><li>Store at 4 Â°C</li></ol></li><li>PGTX solution (100 ml)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td>Phenol<br/></td><td>39.6 g<br/></td></tr><tr><td>Glycerol<br/></td><td>6.9 ml<br/></td></tr><tr><td>8-hydroxyquinoline<br/></td><td>0.1 g<br/></td></tr><tr><td>Na<sub>2</sub>-EDTA<br/></td><td>0.58 g<br/></td></tr><tr><td>Sodium acetate<br/></td><td>0.8 g<br/></td></tr><tr><td>Guanidine thiocyanate<br/></td><td>9.5 g<br/></td></tr><tr><td>Guanidine hydrochloride<br/></td><td>4.6 g<br/></td></tr><tr><td>Triton X-100<br/></td><td>2 ml<br/></td></tr></tbody></table><ol class="npwthree"><li> Weigh all components of the PGTX solution and add dH<sub>2</sub>O to 100 ml</li><li>The solution should be stirred at 50-60 Â°C to create a homogenous solution</li><li>This mixture forms a monophasic solution at room temperature</li><li>The final pH of the solution should reach ~4.2 without any manual adjustment</li></ol></li></ol><p class="pptt" id="biaoti31339">### Acknowledgments</p><p class="ppzz">The protocol is based on the publications âConjugal transfer of DNA to cyanobacteriaâ (Elhai and Wolk, 1988) and âDown-regulation of the alternative sigma factor SigJ confers a photoprotective phenotype to <em>Anabaena</em> PCC 7120â (Srivastava <em>et al.</em>, 2017). This work is supported by the Department of Science and Technology (DST), New Delhi (grant sanctioned to A.K.T., A.B. and A.S.). A.S. is also supported by German Academic Exchange Service (DAAD) and Ministry of Education, Youth and Sports of the Czech Republic (project LO1416).</p><p class="pptt" id="biaoti31340">### Competing interests</p><p class="ppzz">The authors have no conflicts of interest to declare.</p><p class="pptt" id="biaoti31341">### References</p><ol class="npwtwok"><li>Blanco, N. E., Ceccoli, R. D., Segretin, M. E., Poli, H. O., Voss, I., Melzer, M., Bravo-Almonacid, F. F., Scheibe, R., Hajirezaei, M. R. and Carrillo, N. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21205028" target="_blank">Cyanobacterial flavodoxin complements ferredoxin deficiency in knocked-down transgenic tobacco plants.</a> <em>Plant J</em> 65(6): 922-935.</li><li>Elhai, J. and Wolk, C. P. (1988). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3148842" target="_blank">Conjugal transfer of DNA to cyanobacteria.</a> <em>Methods Enzymol</em> 167: 747-754.</li><li>Elhai, J., Vepritskiy, A., Muro-Pastor, A. M., Flores, E. and Wolk, C. P. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9068647" target="_blank">Reduction of conjugal transfer efficiency by three restriction activities of <em>Anabaena</em> sp. strain PCC 7120.</a><em> J Bacteriol </em>179(6): 1998-2005.</li><li>Ferreira, T. and Rasb, W. (2012). <a href="https://imagej.nih.gov/ij/docs/guide/146.html" target="_blank">ImageJ user guide: IJ 1.46 r.</a> (Most recent accessed date: 9/15/2019)</li><li>Gibson, D. G., Young, L., Chuang, R. Y., Venter, J. C., Hutchison, C. A., 3rd and Smith, H. O. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495" target="_blank">Enzymatic assembly of DNA molecules up to several hundred kilobases.</a> <em>Nat Methods </em>6(5): 343-345.</li><li>Kumar, A., Kirti, A. and Rajaram, H. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30055321" target="_blank">Regulation of multiple abiotic stress tolerance by LexA in the cyanobacterium <em>Anabaena</em> sp. strain PCC7120.</a> <em>Biochim Biophys Acta Gene Regul Mech</em> 1861(9): 864-877.</li><li>Lee, Y. J., Hoynes-O'Connor, A., Leong, M. C. and Moon, T. S. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26837577" target="_blank">Programmable control of bacterial gene expression with the combined CRISPR and antisense RNA system.</a> <em>Nucleic Acids Res</em> 44(5): 2462-2473.</li><li>Lin, C. H., Tsai, Z. T. and Wang, D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24200499" target="_blank">Role of antisense RNAs in evolution of yeast regulatory complexity.</a> <em>Genomics</em> 102(5-6): 484-490.</li><li>Niu, T. C., Lin, G. M., Xie, L. R., Wang, Z. Q., Xing, W. Y., Zhang, J. Y. and Zhang, C. C. (2018). <a href="https://pubs.acs.org/doi/10.1021/acssynbio.8b00437" target="_blank">Expanding the potential of CRISPR-Cpf1-based genome editing technology in the cyanobacterium <em>Anabaena</em> PCC 7120.</a> <em>ACS Synth Biol</em> 8(1): 170-180.</li><li>Pinto, F. L., Thapper, A., Sontheim, W. and Lindblad, P. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19660145" target="_blank">Analysis of current and alternative phenol based RNA extraction methodologies for cyanobacteria.</a> <em>BMC Mol Biol</em> 10: 79.</li><li>Rajaram, H. and Apte, S. K. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18174150" target="_blank">Nitrogen status and heat-stress-dependent differential expression of the <em>cpn60</em> chaperonin gene influences thermotolerance in the cyanobacterium <em>Anabaena</em>.</a> <em>Microbiology</em> 154(Pt 1): 317-325.</li><li>Rippka, R., Deruelles, J., Waterbury, J. B., Herdman, M. and Stainer, R. Y. (1979). <a href="https://www.microbiologyresearch.org/content/journal/micro/10.1099/00221287-111-1-1" target="_blank">Generic assignments, strain histories and properties of pure cultures of cyanobacteria.</a> <em>J Gen Microbiol</em> 111(1): 1-61.</li><li>Sambrook, J. and Russell, D. W. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22485377" target="_blank">Preparation and transformation of competent <em>E. coli</em> using calcium chloride.</a> <em>Cold Spring Harb Protoc</em> (1): pdb-prot3932.</li><li>Srivastava, A., Brilisauer, K., Rai, A. K., Ballal, A., Forchhammer, K. and Tripathi, A. K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27837096" target="_blank">Down-regulation of the alternative sigma factor SigJ confers a photoprotective phenotype to <em>Anabaena</em> PCC 7120.</a> <em>Plant Cell Physiol</em> 58(2): 287-297.</li><li>Tailor, V. and Ballal, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28159708" target="_blank">Novel molecular insights into the function and the antioxidative stress response of a Peroxiredoxin Q protein from cyanobacteria.</a> <em>Free Radic Biol Med</em> 106: 278-287.</li><li>Tian, P., Wang, J., Shen, X., Rey, J. F., Yuan, Q. and Yan, Y. (2017). <a href="https://www.sciencedirect.com/science/article/pii/S2405805X17300108" target="_blank">Fundamental CRISPR-Cas9 tools and current applications in microbial systems.</a> <em>Synth Syst Biotechnol</em> 2(3): 219-225.</li><li>Ungerer, J. and Pakrasi, H. B. (2016). <a href="https://www.nature.com/articles/srep39681" target="_blank">Cpf1 is a versatile tool for CRISPR genome editing across diverse species of cyanobacteria.</a> <em>Sci Rep</em> 6: 39681.</li><li>Wolk, C. P., Vonshak, A., Kehoe, P. and Elhai, J. (1984). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6324204" target="_blank">Construction of shuttle vectors capable of conjugative transfer from <em>Escherichia coli</em> to nitrogen-fixing filamentous cyanobacteria.</a> <em>Proc Natl Acad Sci U S A</em> 81(5): 1561-1565.</li><li>Yoon, H. S. and Golden, J. W. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9794762" target="_blank">Heterocyst pattern formation controlled by a diffusible peptide.</a> <em>Science</em> 282(5390): 935-938.</li><li>Zhang, S. and Voigt, C. A. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30289463" target="_blank">Engineered dCas9 with reduced toxicity in bacteria: implications for genetic circuit design.</a> <em>Nucleic Acids Res</em> 46(20): 11115-11125.<br/></li></ol>