<body>
            <h1>mRNA Extraction from Gill Tissue for RNA-sequencing</h1>
            <p><strong>Authors:</strong> Jukka-Pekka   Verta, Felicity Jones</p>
            <h2>Abstract</h2>
            <p>Adaptation is thought to proceed in part through spatial and temporal changes in gene expression. Fish species such as the threespine stickleback are powerful vertebrate models to study the genetic architecture of adaptive changes in gene expression since divergent adaptation to different environments is common, they are abundant and easy to study in the wild and lab, and have well-established genetic and genomic resources. Fish gills, due to their respiratory and osmoregulatory roles, show many physiological adaptations to local water chemistry, including differences in gene expression. However, obtaining high-quality RNA using popular column-based extraction methods can be challenging from small tissue samples high in cartilage and bone such as fish gills. Here, we describe a bead-based mRNA extraction and transcriptome RNA-seq protocol that does not use purification columns. The protocol can be readily scaled according to sample size for the purposes of diverse gene expression experiments using animal or plant tissue.<br/></p>
            <p><strong>Keywords:</strong> Transcriptome sequencing, Gene regulation, Gene expression evolution, Direct mRNA extraction, Gills</p>
            <h2>Background</h2><p>Transcriptome sequencing (RNA-seq) is used to quantify the expression levels of genes, to identify differences in gene expression levels between groups of samples and to infer gene co-expression. In evolutionary genetics research, RNA-seq can be used as an approach to study the molecular basis of adaptive divergence (<em>e.g.</em>, Rougeux <em>et al.</em>, 2019; Verta and Jones, 2019), identify candidate genes underlying adaptive phenotypes (<em>e.g.</em>, Ferreira <em>et al.</em>, 2017), and infer function for unknown genes (<em>e.g.</em>, Rawat <em>et al.</em>, 2015) among other applications. A key strength of RNA sequencing is that it can be readily applied to almost any species or tissue, and transcriptome sequencing has accelerated evolutionary research by opening non-model species for genomic studies. Obtaining high-quality mRNA is the most important prerequisite for a successful RNA-seq study (Gallego Romero <em>et al.</em>, 2014). This can be challenging because RNA is inherently more unstable (and fragile) than more commonly studied DNA. Standard RNA extraction methods typically yield total RNA, comprising mRNA destined to be translated into proteins, as well as non-translated RNA such as ribosomal and regulatory RNAs. Typically, only 1-4% of total RNA is useful for mRNA sequencing and so mRNA yield from small amounts of tissue can be low. In addition, heterogenous tissue samples such as fish gills that consists of both soft epithelia, cartilage and bone, as well as DNA-rich tissue samples, can perform sub-optimally in column-based RNA extraction methods because of physical clogging of column pores and/or DNA precipitation. In such cases direct extraction of mRNA from tissue lysates using oligo-dT coated magnetic beads offer a straightforward and scalable solution for extraction.<br/><br/>Here, we describe such bead-based protocol for rapid (~1-2 h) mRNA extraction from fish gills and RNA-sequencing. The oligo-dT based technique relies on A-T hybridization of mRNA poly-A tails with T-oligonucleotide fragments covalently attached to magnetic beads, allowing for full-length mRNA purification from crude cell lysates. This protocol has been adapted from the original protocol for oligo-dT Dynabeads from Invitrogen and was briefly described in Verta and Jones (2019). Slightly different versions of this protocol were previously used to study gene expression in small plant tissues that comprise of ~1,000-2,000 cells (Verta <em>et al.</em>, 2016; Ojeda <em>et al.</em>, 2019). Briefly, tissue samples are homogenized and lysed, after which mRNA is captured from tissue lysate using magnetic oligo-dT beads. The bead-mRNA complex is washed and subsequently eluted with water. The protocol yields high-quality mRNA that can be used for applications such as RNA-seq and qPCR. We further provide an example of the steps involving RNA quality-control and quantification, RNA-seq library preparation and data analysis. For general guidelines in RNA-seq study design and best-practices the reader is referred to Conesa <em>et al.</em> (2016).<br/></p>
        </body>

<p class="pptt" id="biaoti31984">## Materials and Reagents</p><ol class="npwtwok"><li>1.5 ml RNase-free Safe-Lock Eppendorf tubes</li><li>21 G needles</li><li>Micro-Tube polypropylene pestles for 1.5 ml Eppendorf tubes (Bel-Art, catalog number: <a class="blackNoUnder" target="_blank"> BAF199230001 </a> )</li><li>Qubit Assay tubes (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> Q32856 </a> )</li><li>RNase-free PCR strips and caps</li><li>Fresh tissue or liquid nitrogen snap-frozen tissue</li><li>Tricaine methanesulfonate (MS-222, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E10521 </a> )</li><li>Sodium bicarbonate NaHCO<sub>3</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S5761 </a> )</li><li>Dynabeads mRNA direct purification kit (Thermo Fisher Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 61011 </a> )</li><li>Turbo DNase-free kit (Thermo Fisher Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> AM1907 </a> )</li><li>Bioanalyzer RNA 6000 Nano kit (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5067-11511 </a> )</li><li>Bioanalyzer High Sensitivity DNA kit (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5067-4626 </a> )</li><li>Qubit RNA BR reagents (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> Q10211 </a> )</li><li>PrimeScript RT Master Mix (Takara, catalog number: <a class="blackNoUnder" target="_blank"> RR036A </a> )</li><li>Sybr Select Master Mix for CFX (Thermo Fisher Applied Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 4472942 </a> )</li><li>TruSeq stranded mRNA kit (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> 20020594 </a> )</li><li>SuperScript II reverse transcriptase (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 18064014 </a> )<br/></li></ol><p class="pptt" id="biaoti31985">### Equipment</p><ol class="npwtwok"><li>Scissors (or a blade) and forceps</li><li>DynaMag2 magnet (Thermo Fisher Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 12321D </a> )</li><li>PCR cycler (<em>e.g.</em>, Bio-Rad C1000 Touch, catalog number: <a class="blackNoUnder" target="_blank"> 1851196 </a> )</li><li>Qubit fluorometer (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> Q33238 </a> )</li><li>CFX96 Touch Real-Time PCR cycler (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1855195 </a> )</li><li>2100 Bioanalyzer (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> G2939BA </a> )<br/></li></ol><p class="pptt" id="biaoti31986">### Software</p><ol class="npwtwok"><li>FastQC (<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>)</li><li>Cutadapt (<a href="https://cutadapt.readthedocs.io/en/stable/#" target="_blank">https://cutadapt.readthedocs.io/en/stable/#</a>)</li><li>STAR (Dobin <em>et al.</em>, 2013)</li><li>Cufflinks (Trapnell <em>et al.</em>, 2012)<br/></li></ol><p class="pptt" id="biaoti31987">### Procedure</p><p class="ppzzs"><em>Notes:</em> </p><ol class="npwtwo"><li><i>For mRNA extraction frozen from stickleback gill tissue (~50 mg) use:</i><br/><i> 250 Î¼l of lysis buffer</i><br/><i> 100 Î¼l of Dynabeads</i><br/><i> 600 Î¼l of wash-A solution (two washes)</i><br/><i> 300 Î¼l of wash-B solution</i><br/><i> Elute to 20 Î¼l RNase free water</i><br/><i> Volumes can be scaled in proportion for different size tissue samples.</i> </li><li><i>Procedures B-G are adapted from the original Invitrogen Dynabeads protocol with modifications to solution volumes, sample homogenization and lysis steps. Additional steps H-J not part of original Invitrogen protocol are required to assure the yield of DNA-free and high-quality mRNA.</i> </li><li><i>Procedures B-H should be followed through immediately. Safe stopping points where the experiment can be put to hold are after Procedures A and H.</i><br/><span style="font-size:13.3333px;"><br/></span> </li></ol><ol class="npwone"><li>Dissect whole stickleback gills<ol class="npwtwo"><li>Euthanize fish following animal protocol guidelines (<em>e.g.</em>, <a href="https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=OJ:L:2010:276:TOC" target="_blank">EU Directive 2010-63-eu</a>, and <a href="https://www.avma.org/sites/default/files/resources/euthanasia.pdf" target="_blank">AVMA Guidelines</a>) For example, by anesthetic overdose with NaHCO<sub>3</sub>-buffered Tricaine methanesulfonate (MS-222, pH 7.5). Note that, appropriate concentrations vary depending on the size of the fish. For sticklebacks we use a 10x solution (1.5 g/L) prepared by dissolving 1.5 g of MS-222 and 3.0 g of NaHCO<sub>3</sub> in 1 L of aquarium water.</li><li>Remove the head using scalpel and forceps (Figure 1). Working from the ventral side, open the operculum to reveal gill arches. Cut gill arches free from the ventral and dorsal attachment points as shown in Figure 1B. Remove any connective tissue that is not part of the gills.</li><li>Collect gill tissue (~50 mg) in 1.5 ml Eppendorf Safe-Lock tubes and immediately snap-freeze in liquid nitrogen.</li><li>Store samples in -80 Â°C.<br/><u><em>SAFE STOPPING POINT</em></u><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200229/20200229044152_2048.png" width="106"/><br/><strong>Figure 1. Dissection of stickleback gills.</strong> Whole gill arches are isolated from euthanized fish by removal of head via dissection along the transverse plane (red line, A). Then with a ventral view (B) the gill arches (blue lines) are excised by dissection of the ventral and dorsal attachment points (red lines, B).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Wash and resuspend Dynabeads<ol class="npwtwo"><li>Allow Dynabeads and solutions to warm to room temperature.</li><li>Prepare one 1.5 ml Eppendorf per tissue sample. Resuspend Dynabeads by gently pipetting up and down, and aliquot 100 Î¼l of Dynabeads into each 1.5 ml Eppendorf tube.</li><li>Place tubes on a DynaMag2 magnet and wait approximately 30 s for beads to form a pellet (~30 s).</li><li>Remove the supernatant without disturbing the beads.</li><li>Remove tubes from magnet and resuspend the Dynabeads in original volume of lysis buffer (100 Î¼l).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Homogenise tissue into a powder using a pestle<br/><em>Notes:</em> <ol class="npwtwo"><li><em>Modify Micro-Tube homogenizing pestles for better sample disruption by cutting the round conical surface of the pestles into angular surface with sterile scissors or a blade (Figure 2).</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200229/20200229044212_4354.png" width="81"/><br/><strong>Figure 2. Modification of homogenizing pestle.</strong> For more efficient sample disruption, cut the surface of the pestle (red lines, A) into an angular surface (B) with sterile scissors or scalpel blade.<br/><br/></li><li><em>Cool down the pestle before sample disruption by dipping it to liquid nitrogen.</em> </li><li><em>Remove samples from -80 Â°C and keep on liquid nitrogen.</em> </li><li><em>Disrupt tissue to powder on liquid nitrogen using a pestle stick in 1.5 ml Eppendorf tubes.</em> </li><li><em>Avoid melting of the sample at all stages before lysis step below. Freezing breaks the cell membranes and exposes RNA to endogenous RNases, which degrade RNA after thawing. Always keep your frozen sample on liquid nitrogen. Once you introduce an RNase inhibiting agent (for example LiCl in the Dynabeads lysis buffer, next step in protocol), RNA degradation is efficiently inhibited and sample can be thawed.<br/></em> <div style="white-space:nowrap;"><br/></div></li></ol></li><li>Lyse the cells<ol class="npwtwo"><li>Take sample tubes one by one out of the liquid nitrogen, remove pestle but keep it in hand, immediately pipet in 250 Î¼l of lysis buffer and then drop the pestle back into the tube.</li><li>Keep the frozen tube in your hand and grind the frozen lysis buffer with the pestle until it melts to ensure the sample is suspended in lysis buffer immediately after thawing.</li><li>Repeat for each of the samples.</li><li>Discard pestle and shear DNA by passing the lysis mix five times through a 21 gauge needle before proceeding with the protocol.</li></ol><em>Note: Introduce lysis buffer while your tube is still frozen.</em><br/><br/></li><li>Bind mRNA to Dynabeads<br/>Add 100 Î¼l of washed and resuspended Dynabeads per sample, incubate 5 min at RT with continuing rotation (~500 rpm).<br/><em>Note: Continuous mixing during this incubation step is important to reduce the binding of Dynabeads to genomic DNA.<br/><br/></em> </li><li>Wash Dynabead-mRNA complex<ol class="npwtwo"><li>Place samples on Dynamag magnet and wait for beads to form a pellet (~30 s).</li><li>Remove all supernatant from tubes without touching the bead pellet.</li><li>Remove tubes from magnet and resuspend beads in 600 Î¼l wash buffer A.</li><li>Repeat Steps F1-F3 for a second buffer A-wash.</li><li>Remove the buffer and perform final wash using 300 Î¼l of wash buffer B.</li></ol><em>Notes:<ol class="npwthree"><li>Wash buffers should be room temperature before use.</li><li>Always resuspend the beads completely in the washing buffers by pipetting solution slowly up and down. Insufficient resuspension leads to increased DNA and ribosomal-RNA contamination. There should be no visible clump of beads after resuspension. Avoid excessive shearing/fragmentation of RNA by using the minimal amount of pipetting necessary to resuspend beads.</li><li>You can perform wash A three times for better sample purity.<br/><div style="white-space:nowrap;"><br/></div></li></ol></em> </li><li>Elute mRNA from beads<ol class="npwtwo"><li>Place samples on magnet and wait for beads to form a pellet (~30 s).</li><li>Remove all supernatant from tubes taking care not to leave any residual buffer.</li><li>Remove tubes from magnet and resuspend beads in 20 Î¼l of RNase free water.</li><li>Incubate samples for 2 min in 65 Â°C in a thermocycler or thermomixer.</li><li>Immediately place samples on the magnet and pipet out the supernatant to a fresh RNase-free 1.5 ml Eppendorf tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Treat mRNA extraction with DNase <ol class="npwtwo"><li>To 20 Î¼l of eluted mRNA, add 2 Î¼l of Turbo DNase buffer + 1 Î¼l Turbo DNase.</li><li>Incubate at 37 Â°C for 20 min.</li><li>Add 5 Î¼l of inactivation solution, flick tubes and incubate 5 min at RT.</li><li>Spin for 1 min 30 s at 10,000 <em>x g</em> and transfer the supernatant to a fresh RNase-free tube.</li><li>Snap-freeze final mRNA extraction as smaller aliquots (<em>e.g.</em>, 10 Î¼l) on liquid nitrogen and store in -80 Â°C.<br/><u><em>SAFE STOPPING POINT</em></u> </li></ol><em>Notes:</em> <ol class="npwthree"><li><em>Treat the entire 20 Î¼l aliquot of mRNA with DNase right after elution from beads, do not freeze-thaw.</em> </li><li><em>Either quantify and quality-control (<em>e.g.</em>, Qubit &amp; BioAnalyzer) your mRNA immediately after DNase treatment or keep an aliquot for these separately. Avoid freeze-thaw cycles of your mRNA sample at all times.</em> </li><li><em>20-30% ribosomal-RNA contamination of mRNA samples is common. rRNA contamination can be reduced by optimizing the A-wash volume. You can also try heating the lysis buffer to 65 Â°C or an elution-rebinding cycle in 65 Â°C while the beads are in the lysis buffer.</em></li><li><em>The Illumina TruSeq RNA-seq protocols start with mRNA selection, which eliminates the majority of ribosomal-RNA and DNA contamination (Figure 3).<br/></em> <br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200229/20200229044235_1731.png" width="144"/><br/><strong>Figure 3. Testing for genomic DNA contamination in extracted mRNA using qPCR of housekeeping gene.</strong> Positive controls (cDNA) and mRNA extractions were amplified using qPCR of housekeeping gene primers (<em>e.g.</em>, claudin). Lower Ct values (PCR cycle when fluorescence can be detected above background level, green line) indicate higher abundance of template, which in the case of non reverse-transcribed mRNA extractions represents contamination from genomic DNA. Non DNase-treated Dynabeads mRNA extractions show DNA contamination, which is almost completely eliminated after second mRNA selection step performed at the beginning of Illumina TruSeq RNA-seq protocol. DNase treated mRNA extractions show no amplification and are thus free of DNA.</li></ol><br/></li><li>Verify the absence of genomic-DNA contamination<ol class="npwtwo"><li>Design a pair of PCR primers that amplify a housekeeping gene (<em>e.g.</em>, claudin), making sure that primers amplify a fragment within a single exon (the PCR fragment should not span an exon-intron boundary). Aim for primers that are specific to the housekeeping gene (verify sequence similarity other genes <em>e.g.</em> using BLAST search), ending in G/C nucleotides and have a melting temperature of 60 Â°C Â± 5 Â°C. For stickleback gill tissue, we used claudin gene as control (Fwd: 5' ACTTGGTGCCCTATCAAATGAGGTA 3', Rev: 5' AGTTATACACGACGGGAGGATTGAG 3').</li><li>Select positive control samples and reverse-transcribe mRNA to cDNA using Takara PrimeScript RT Master Mix.<ol class="npwthree"><li>To 5 Î¼l of mRNA extraction, add 3 Î¼l of water and 2 Î¼l of PrimerScript master-mix.</li><li>Incubate for 15 min at 37 Â°C.</li><li>Store in -20 Â°C.</li></ol></li><li>Prepare qPCR reactions for positive control (cDNA) and selected mRNA extractions for verification.<br/>10 Î¼l of 2X Sybr select for CFX Master Mix<br/>1 Î¼l of 10 mM Forward primer<br/>1 Î¼l of 10 mM Reverse primer<br/>2 Î¼l water<br/>1 Î¼l cDNA template</li><li>Run samples on qPCR using the following program:<ol class="npwthree"><li>50 Â°C 2 min</li><li>95 Â°C 2 min</li><li>95 Â°C 30 s</li><li>55-65 Â°C (primer-specific melting temperature) 30 s</li><li>72 Â°C 30 s</li><li>Return to c. for 39 additional cycles</li><li>Hold at 4 Â°C</li></ol></li><li>Analyze qPCR results (Figure 3). The DNase treated mRNA extractions should not amplify, while the reverse-transcribed cDNA preps should give signal. Amplification in mRNA extractions without reverse-transcription indicates contamination from genomic DNA. In this case re-treat samples with DNase and verify mRNA quality using Bioanalyser.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Verify the quality of extracted mRNA using Bioanalyzer RNA 6000 Nano reagents (Figure 4). For concentrations less than ~5 ng/Î¼l use Bioanalyzer RNA 6000 Pico reagents.<br/><em>Note: RNA Integrity Number (RIN) typically calculated for total RNA extractions does not apply for mRNA extractions because the majority of ribosomal RNA used to calculate RNA quality and detect degradation have been removed. Instead, use the distribution of mRNA fragment sizes to determine whether the sample is of good quality (a range of sizes are of high molecular weight) or poor quality (highly degraded RNA of low molecular weight) (Figure 4).</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200229/20200229044256_5713.png" width="330"/><br/><strong>Figure 4. Verification of mRNA quality using Bioanalyser.</strong> Example of good (A) and bad (B) quality mRNA sample analysed with Bioanalyser RNA 6000 Nano reagents. Good-quality mRNA extraction can show some (10-30%) rRNA contamination. First peak at 25 nt corresponds to marker, two prominent peaks in (A) correspond to 18S and 28S rRNAs and the broad hump corresponds to mRNA. Note the left-shift (smaller size fragments) of the mRNA size distribution in the bad-quality sample, indicating that mRNA fragments are degraded.<br/><br/></li><li>Construct RNA-seq libraries using Illumina TruSeq kit and kit instructions, with the following exceptions:<ol class="npwtwo"><li>Use 150 ng of mRNA as input (quantify using BioAnalyzer RNA Nano or Qubit RNA BR reagents). Dilute with RNase-free water to a final volume of 50 Î¼l. Using adequate amount of input mRNA helps to avoid excessive PCR duplicates in final sequencing library.</li><li>Optimize fragmentation time for the sequencing strategy you are going to use. Test the effect of fragmentation time using a time-series and analyze fragment sizes using Bionalyzer High Sensitivity DNA reagents, or refer to page 109 of Illumina TruSeq RNA Sample Preparation v2 Guide. For stickleback gill mRNA sequencing with HiSeq 3000 150 base-pair paired-end reads, we used two minutes (or 2 min) fragmentation to have a final average insert size of ~290 bp.</li><li>Use provided indexes to identify samples after sequencing and pool in equimolar amounts for sequencing. Recommended strategy to avoid possible batch effects is to use as many indexes as there are samples and sequence all samples in the same lane. Where experimental design allows, use replicate lanes and combine reads from replicate lanes to reach the sequencing depth required.</li></ol></li></ol><p class="pptt" id="biaoti31988">### Data analysis</p><p class="ppzz">The following general example is given for 150 base-pair paired-end RNA-seq reads from gill tissue of threespine stickleback fish analyzed in a Unix/Linux/MacOSX environment with command-line tools. The user is encouraged to modify the paths outlined in the below procedure to match their data, and to execute the code in a terminal window by using shell scripts or copy-paste. Detailed information for the groups of samples tested can be found in Verta and Jones (2019).</p><ol class="npwtwo"><li>Verify read and library quality with <em>FastQC</em> software following the guidelines outlined in <em>FastQC</em> website and manual. (<a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>)<br/><em>Note: FastQC may report RNA-seq libraries to contain high levels of PCR-duplicates. This does not necessarily mean libraries have undergone excessive PCR because many reads are expected to map to highly abundant transcripts, which FastQC can interpret as PCR duplication.</em> </li><li>Trim any Illumina sequencing adapters from the reads using <em>TrimGalore</em>. (<a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>)<br/><em>Note: Alternatively, quality-control and adapter trimming can be performed in one step using the software fastp (Chen et al., 2018).</em><br/><br/><em> <span style='color:#009900;font-family:"'>trim_galore \</span></em><br/><em><span style='color:#009900;font-family:"'> --path_to_cutadapt cutadapt \</span></em><br/><em><span style='color:#009900;font-family:"'> --illumina \</span></em><br/><em><span style='color:#009900;font-family:"'> --stringency 5 \</span></em><br/><em><span style='color:#009900;font-family:"'> --quality 20 \</span></em><br/><em><span style='color:#009900;font-family:"'> --output_dir out_dir \</span></em><br/><em><span style='color:#009900;font-family:"'> --paired \</span></em><br/><em><span style='color:#009900;font-family:"'> reads_R1_001.fastq.gz \</span></em><br/><em><span style='color:#009900;font-family:"'> reads_R2_001.fastq.gz</span></em><br/><br/></li><li>Generate <em>STAR</em> genome file (necessary only the first time a given reference genome used).<br/><br/><span style='color:#009900;font-family:"'>STAR \</span><br/><span style='color:#009900;font-family:"'> --runMode genomeGenerate \</span><br/><span style='color:#009900;font-family:"'> --runThreadN N \</span><br/><span style='color:#009900;font-family:"'> --genomeDir genome \</span><br/><span style='color:#009900;font-family:"'> --genomeFastaFiles gasAcu1.fa \</span><br/><span style='color:#009900;font-family:"'> --sjdbGTFfile transcript.gtf \</span><br/><span style='color:#009900;font-family:"'> --sjdbOverhang 149 \</span><br/><span style='color:#009900;font-family:"'> --genomeChrBinNbits 18</span><br/><br/></li><li>Align RNA-seq reads to the reference genome (<em>e.g.</em>, Broad S1, gasAcu1 [Jones <em>et al.</em>, 2012]) with <em>STAR</em> alignerâa high performing aligner that addresses many of the challenges associated with aligning spliced RNAseq reads across intron/exon boundaries of the reference genome (Dobin <em>et al.</em>, 2013). Use manual two-pass mode for most sensitive detection of novel intron-exon boundaries.<ol class="npwthree"><li>For each sample, perform <em>STAR</em> 1<sup>st</sup> pass to identify intron/exon boundaries in current sample.<br/><br/><span style='color:#009900;font-family:"'>STAR \</span><br/><span style='color:#009900;font-family:"'> --readFilesCommand gunzip -c \</span><br/><span style='color:#009900;font-family:"'> --runThreadN N \</span><br/><span style='color:#009900;font-family:"'> --outFilterIntronMotifs RemoveNoncanonicalUnannotated \</span><br/><span style='color:#009900;font-family:"'> --chimSegmentMin 50 \</span><br/><span style='color:#009900;font-family:"'> --outFilterType BySJout \</span><br/><span style='color:#009900;font-family:"'> --alignSJDBoverhangMin 1 \</span><br/><span style='color:#009900;font-family:"'> --alignIntronMin 20 \</span><br/><span style='color:#009900;font-family:"'> --alignIntronMax 200000 \</span><br/><span style='color:#009900;font-family:"'> --alignMatesGapMax 200000 \</span><br/><span style='color:#009900;font-family:"'> --quantMode GeneCounts \</span><br/><span style='color:#009900;font-family:"'> --outWigType wiggle \</span><br/><span style='color:#009900;font-family:"'> --outSAMtype BAM SortedByCoordinate \</span><br/><span style='color:#009900;font-family:"'> --twopassMode None \</span><br/><span style='color:#009900;font-family:"'> --genomeDir genome \</span><br/><span style='color:#009900;font-family:"'> --readFilesIn reads_val_1.fq.gz reads_val_2.fq.gz</span><br/><br/></li><li>For each sample, run <em>STAR</em> 2<sup>nd</sup> pass using intron/exon boundaries identified across samples above to inform read alignment.<br/><br/><span style='color:#009900;font-family:"'>STAR \</span><br/><span style='color:#009900;font-family:"'> --readFilesCommand gunzip -c \</span><br/><span style='color:#009900;font-family:"'> --runThreadN N \</span><br/><span style='color:#009900;font-family:"'> --outFilterIntronMotifs RemoveNoncanonicalUnannotated \</span><br/><span style='color:#009900;font-family:"'> --chimSegmentMin 50 \</span><br/><span style='color:#009900;font-family:"'> --outFilterType BySJout \</span><br/><span style='color:#009900;font-family:"'> --alignSJDBoverhangMin 1 \</span><br/><span style='color:#009900;font-family:"'> --alignIntronMin 20 \</span><br/><span style='color:#009900;font-family:"'> --alignIntronMax 200000 \</span><br/><span style='color:#009900;font-family:"'> --alignMatesGapMax 200000 \</span><br/><span style='color:#009900;font-family:"'> --quantMode GeneCounts \</span><br/><span style='color:#009900;font-family:"'> --outWigType wiggle \</span><br/><span style='color:#009900;font-family:"'> --outSAMtype BAM SortedByCoordinate \</span><br/><span style='color:#009900;font-family:"'> --twopassMode None \</span><br/><span style='color:#009900;font-family:"'> --genomeDir genome \</span><br/><span style='color:#009900;font-family:"'> --limitSjdbInsertNsj 2000000 \</span><br/><span style='color:#009900;font-family:"'> --readFilesIn reads_val_1.fq.gz reads_val_2.fq.gz \</span><br/><span style='color:#009900;font-family:"'> --sjdbFileChrStartEnd SJ_1.out.tab \</span><br/><span style='color:#009900;font-family:"'> --sjdbFileChrStartEnd SJ_2.out.tab \</span><br/><span style='color:#009900;font-family:"'> â¦</span><br/><div><br/></div></li></ol></li><li>Use <em>Cufflinks2.2</em> pipeline (Trapnell <em>et al.</em>, 2012) for reference-guided transcriptome assembly and transcript and isoform expression level testing. In order to run in âreference guidedâ mode, a transcript gtf file describing known gene models is needed. <em>Cufflinks</em> also allows âunguidedâ de novo transcriptome assemblies if this information is not available.<br/><br/><span style='color:#009900;font-family:"'>cufflinks \</span><br/><span style='color:#009900;font-family:"'> -p N \</span><br/><span style='color:#009900;font-family:"'> --min-intron-length 20 \</span><br/><span style='color:#009900;font-family:"'> --library-type fr-firststrand \</span><br/><span style='color:#009900;font-family:"'> -o ./cloutGuided \</span><br/><span style='color:#009900;font-family:"'> --frag-bias-correct gasAcu1.fa \</span><br/><span style='color:#009900;font-family:"'> --multi-read-correct \</span><br/><span style='color:#009900;font-family:"'> --min-isoform-fraction 0.15 \</span><br/><span style='color:#009900;font-family:"'> --min-frags-per-transfrag 20 \</span><br/><span style='color:#009900;font-family:"'> --max-multiread-fraction 0.5 \</span><br/><span style='color:#009900;font-family:"'> -g transcript.gtf \</span><br/><span style='color:#009900;font-family:"'> reads.out.bam</span><br/><br/></li><li>Merge sample-level annotation files together (specified in assembly_list.txt).<br/><br/><span style='color:#009900;font-family:"'>cuffmerge \</span><br/><span style='color:#009900;font-family:"'> -g transcript.gtf \</span><br/><span style='color:#009900;font-family:"'> -o merged_assembly \</span><br/><span style='color:#009900;font-family:"'> -s gasAcu1.fa \</span><br/><span style='color:#009900;font-family:"'> -p N \</span><br/><span style='color:#009900;font-family:"'> assembly_list.txt</span><br/><br/></li><li>Quantify RNA-seq read coverage over gene models.<br/><br/><span style='color:#009900;font-family:"'>cuffquant \</span><br/><span style='color:#009900;font-family:"'> -o out_dir \</span><br/><span style='color:#009900;font-family:"'> -p N \</span><br/><span style='color:#009900;font-family:"'> -v \</span><br/><span style='color:#009900;font-family:"'> -u \</span><br/><span style='color:#009900;font-family:"'> --library-type fr-firststrand \</span><br/><span style='color:#009900;font-family:"'> -b gasAcu1.fa \</span><br/><span style='color:#009900;font-family:"'> merged.gtf \</span><br/><span style='color:#009900;font-family:"'> reads.out.bam</span><br/><br/></li><li>Normalize read coverages across samples.<br/><br/><span style='color:#009900;font-family:"'>cuffnorm \</span><br/><span style='color:#009900;font-family:"'> -p N \</span><br/><span style='color:#009900;font-family:"'> -o out_dir \</span><br/><span style='color:#009900;font-family:"'> -library-type fr-firststrand \</span><br/><span style='color:#009900;font-family:"'> --labels sample \</span><br/><span style='color:#009900;font-family:"'> merged.gtf \</span><br/><span style='color:#009900;font-family:"'> abundances1a.cxb \</span><br/><span style='color:#009900;font-family:"'> abundances1b.cxb \</span><br/><span style='color:#009900;font-family:"'> abundances2a.cxb \</span><br/><span style='color:#009900;font-family:"'> abundances2b.cxb \</span><br/><span style='color:#009900;font-family:"'> â¦</span><br/><br/></li><li>Test for differential expression between groups of samples using the <em>Cufflinks</em> function <em>cuffdiff</em>.<br/><br/><span style='color:#009900;font-family:"'>cuffdiff \</span><br/><span style='color:#009900;font-family:"'> -p N \</span><br/><span style='color:#009900;font-family:"'> -o outDir \</span><br/><span style='color:#009900;font-family:"'> -b gasAcu1.fa \</span><br/><span style='color:#009900;font-family:"'> -u \</span><br/><span style='color:#009900;font-family:"'> -dispersion-method per-condition \</span><br/><span style='color:#009900;font-family:"'> -library-type fr-firststrand \</span><br/><span style='color:#009900;font-family:"'> --labels Group1,Group2 \</span><br/><span style='color:#009900;font-family:"'> merged.gtf \</span><br/><span style='color:#009900;font-family:"'> abundances1a.cxb,abundances1b.cxb \</span><br/><span style='color:#009900;font-family:"'> abundances2a.cxb,abundances2b.cxb \</span> </li></ol><p class="pptt" id="biaoti31989">### Acknowledgments</p><p class="ppzz">FJ is funded by the European Research Council (FP7) and the Max Planck Society. This protocol is adapted from Verta and Jones (2019).</p><p class="pptt" id="biaoti31990">### Ethics</p><p class="ppzz">All animal experiments were done in accordance to EU and state legislation and avoiding unneccessary harm to animals (EU Directive 2010_63_EU).</p><p class="pptt" id="biaoti31991">### References</p><ol class="npwtwok"><li>Chen, S., Zhou, Y., Chen, Y. and Gu, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30423086" target="_blank">Fastp: an ultra-fast all-in-one FASTQ preprocessor.</a> <em>Bioinformatics</em> 34(17): i884-i890.</li><li>Conesa, A., Madrigal, P., Tarazona, S., Gomez-Cabrero, D., Cervera, A., McPherson, A., Szczesniak, M. W., Gaffney, D. J., Elo, L. L., Zhang, X. and Mortazavi, A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26813401" target="_blank">A survey of best practices for RNA-seq data analysis.</a> <em>Genome Biol</em> 17: 13.</li><li>Dobin, A., Davis, C. A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M. and Gingeras, T. R. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886" target="_blank">STAR: ultrafast universal RNA-seq aligner.</a> <em>Bioinformatics</em> 29(1): 15-21.</li><li>Ferreira, M. S., Alves, P. C., Callahan, C. M., Marques, J. P., Mills, L. S., Good, J. M. and Melo-Ferreira, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28500774" target="_blank">The transcriptional landscape of seasonal coat colour moult in the snowshoe hare.</a> <em>Mol Ecol</em> 26(16): 4173-4185.</li><li>Gallego Romero, I., Pai, A. A., Tung, J. and Gilad, Y. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24885439" target="_blank">RNA-seq: impact of RNA degradation on transcript quantification.</a> <em>BMC Biol</em> 12: 42.</li><li>Jones, F. C., Grabherr, M. G., Chan, Y. F., Russell, P., Mauceli, E., Johnson, J., Swofford, R., Pirun, M., Zody, M. C., White, S., Birney, E., Searle, S., Schmutz, J., Grimwood, J., Dickson, M. C., Myers, R. M., Miller, C. T., Summers, B. R., Knecht, A. K., Brady, S. D., Zhang, H., Pollen, A. A., Howes, T., Amemiya, C., Broad Institute Genome Sequencing, P., Whole Genome Assembly, T., Baldwin, J., Bloom, T., Jaffe, D. B., Nicol, R., Wilkinson, J., Lander, E. S., Di Palma, F., Lindblad-Toh, K. and Kingsley, D. M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22481358" target="_blank">The genomic basis of adaptive evolution in threespine sticklebacks.</a> <em>Nature</em> 484(7392): 55-61.</li><li>Ojeda, D. I., Mattila, T. M., Ruttink, T., Kujala, S. T., Karkkainen, K., Verta, J. P. and PyhÃ¤jÃ¤rvi, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31427456" target="_blank">Utilization of tissue ploidy level variation in de novo transcriptome assembly of pinus sylvestris.</a> <em>G3 (Bethesda) </em>9(10): 3409-3421.</li><li>Rawat, V., Abdelsamad, A., Pietzenuk, B., Seymour, D. K., Koenig, D., Weigel, D., Pecinka, A. and Schneeberger, K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26382944" target="_blank">Improving the annotation of <em>Arabidopsis</em> lyrata using RNA-seq data.</a> PLoS One 10(9): e0137391.</li><li>Rougeux, C., Gagnaire, P. A., Praebel, K., Seehausen, O. and Bernatchez, L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31482603" target="_blank">Polygenic selection drives the evolution of convergent transcriptomic landscapes across continents within a Nearctic sister species complex. </a>Mol Ecol 41(16): 291.</li><li>Trapnell, C., Roberts, A., Goff, L., Pertea, G., Kim, D., Kelley, D. R., Pimentel, H., Salzberg, S. L., Rinn, J. L. and Pachter, L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" target="_blank">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</a> <em>Nat Protoc</em> 7(3): 562-578.</li><li>Verta, J. P. and Jones, F. C. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31090544" target="_blank">Predominance of cis-regulatory changes in parallel expression divergence of sticklebacks.</a> <em>Elife</em> 8: 43785.</li><li>Verta, J. P., Landry, C. R. and Mackay, J. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26891783" target="_blank">Dissection of expression quantitative trait locus and allele specificity using a haploid/diploid plant system - insights into compensatory evolution of transcriptional regulation within populations.</a> <em>The New Phytologist</em> 211(1): 159-171.<br/></li></ol>