<body>
            <h1>Growth Recovery Assay and FACS-based Population Sorting Following Territorial Exclusion in <em>Proteus mirabilis</em></h1>
            <p><strong>Authors:</strong> Murray J. Tipping, Karine  A. Gibbs</p>
            <h2>Abstract</h2>
            <p>Many bacteria take part in self recognition and kin discrimination behavior using contact-dependent effectors. Understanding the effects these effectors cause is important to explain bacterial community formation and population dynamics. Typically, kin discrimination effectors are toxins that kill target cells; their effect is therefore obvious and easily measurable. However, many self-recognition effectors, such as the <em>Proteus mirabilis</em> Ids system, are non-lethal and do not cause obvious physiological changes in target cells. Previously, experimental techniques to probe cells experiencing non-lethal kin recognition have been limited. Here we describe a technique to reliably isolate cells deemed self and non-self through Ids self-recognition for downstream phenotypic analysis. Liquid cultures of fluorescently labeled self-recognition mutants are mixed together and inoculated on swarm-permissive agar. Mixed swarms are harvested, and each strain is isolated through fluorescence-activated cell sorting (FACS). The growth rate of each strain is measured on a plate reader. This protocol is adaptable for other bacterial species. We describe briefly how sorted particles can be used for other analyses such as RNA-Seq library preparation.</p>
            <p><strong>Keywords:</strong> Contact-dependent effectors, Cell-cell communication, Swarm motility, Bacterial growth, Self versus non-self recognition, FACS, Plate reader, <em>Proteus mirabilis</em></p>
            <h2>Background</h2><p>Many organisms have evolved adaptations to community living. One common mechanism present throughout nature is kin discrimination: preferential treatment of close kin and hindrance of non-kin (Smith, 1964). Well studied examples of bacterial kin discrimination include contact-dependent inhibition (CDI) (Aoki <em>et al.</em>, 2009; Garcia <em>et al.</em>, 2016) and toxin transfer through type IV and type VI secretion systems (Brunet <em>et al.</em>, 2013; Souza <em>et al.</em>, 2015). Typically, kin discrimination in bacteria is mediated by exchange of toxic effector proteins, such as nucleases or peptidoglycan degrading enzymes (Alteri and Mobley, 2016). Each strain expresses characteristic effector-immunity pairs, and cells must either express the cognate immunity protein or die. <em>Proteus mirabilis</em>, however, makes use of a non-lethal self recognition system, the Ids system, to perform kin discrimination during swarming (Gibbs <em>et al.</em>, 2008). <em>P. mirabilis</em> displays a sophisticated swarming phenotype on solid surfaces. After surface contact, a subset of cells differentiate into elongated âswarmerâ cells that collectively travel outwards. Periods of swarming are regularly followed by non-expansion âconsolidationâ periods where swarmer cells differentiate into shorter stationary cells. Successive rounds of swarming and consolidation lead to a characteristic bullseye pattern on surfaces (Rauprich <em>et al.</em>, 1996). While swarming, different <em>P. mirabilis</em> isolates express different variants of identity factors IdsD and IdsE. IdsD is transferred between cells in a contact-dependent fashion. If cells do not express a cognate IdsE they do not die but are instead forced out of cooperative swarm behavior in a phenomenon termed territorial exclusion. This behavior is due to forced induction into a persister-like state mediated by the alarmone (p)ppGpp (Tipping and Gibbs, 2019).<br/> <br/>In lethal kin discrimination systems, progression can be measured through several methods: dilution assays, microscopy to visualize killing (<em>e.g.</em>, Basler <em>et al.</em>, 2013), or live-dead staining (Virta <em>et al.</em>, 1998). Quantifying the efficacy of a non-lethal system is more difficult. The relative swarm extent <em>in situ</em> can be measured through microscopy or replica plating (Wenren <em>et al.</em>, 2013), but these methods provide limited information about the swarm state, and none at all about the states of individual cells that have taken part in non-lethal kin discrimination. Isolation of cells from the greater swarm mass is therefore critical to enable their physiological state to be examined in more detail.<br/> <br/>We have developed the first method for separating two populations of cells from a mixed <em>P. mirabilis</em> swarm using fluorescence-activated cell sorting (FACS), so that the state of cells that have taken part in self recognition behavior can be measured. This efficient and reliable protocol allows downstream examination of cells using several different methods. Here, we describe how to measure growth rate of cells that have experienced territorial exclusion. We have also used this method to generate samples for RNA-Seq library preparation, and describe our methods for doing so briefly. Furthermore, sorted cells can potentially be used to generate samples for other metabolic or biochemical assays. The protocol as described is optimized for separating <em>P. mirabilis</em> cells and could be adapted to isolate cells from swarms of other species such as <em>Escherichia coli</em>, <em>Vibrio cholerae</em>, <em>Pseudomonas aeruginosa</em>, and <em>Providencia stuartii</em>.</p>
        </body>

<p class="pptt" id="biaoti32026">## Materials and Reagents</p><p class="ppzz"><em>Note: All materials and reagents are stable at room temperature. <em>P. mirabilis</em> strains are typically transported as stab cultures that are viable at room temperature.</em> </p><ol class="npwtwo"><li>Disposable inoculation loop (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 22-363-602 </a> )</li><li>Petri dishes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 25384-168 </a> )</li><li>50 ml conical tubes (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 14-432-22 </a> )</li><li>1.5 ml microcentrifuge tubes (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 05-408-129 </a> )</li><li>1-10 Âµl pipette tips (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 07-000-854 </a> )</li><li>20-100 Âµl pipette tips (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 07-000-867 </a> )</li><li>200-1,000 Âµl pipette tips (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 07-000-350 </a> )</li><li>Disposable spreaders (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 14-665-230 </a> )</li><li>Spectrophotometry cuvettes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 97000-586 </a> )</li><li>0.22 Âµm vacuum filter kit (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 09-761-1 </a> )</li><li>FACS collection tubes (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 50-187-1139 </a> )</li><li>96-well flat-bottomed plates (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M2936-100EA </a> )</li><li><em>Proteus mirabilis</em> strain KAG107 (BB2000 constitutively expressing DsRed) (Gibbs <em>et al.</em>, 2008) (available on request)</li><li><em>Proteus mirabilis</em> strain KAG966 (BB2000 â<em>ids</em> constitutively expressing GFPmut2) (Gibbs <em>et al.</em>, 2008) (available on request)</li><li>Tryptone (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> DF0123-17-3 </a> )</li><li>Yeast extract (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> DF0127-17-9 </a> )</li><li>Sodium chloride (NaCl) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S9888-500G </a> )</li><li>Oxoid CM55 agar (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> OXCM0055B </a> )</li><li>Glycerol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> G7893-500ML </a> )</li><li>Bacto-agar (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> DF0140-01-0 </a> )</li><li>Magnesium sulfate (MgSO<sub>4</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M7506-500G </a> )</li><li>Nicotinic acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> N4126-100G </a> )</li><li>Sodium hydrogen phosphate dihydrate (Na<sub>2</sub>HPO<sub>4</sub>Â·2H<sub>2</sub>O) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 71645-1KG </a> )</li><li>Potassium hydrogen phosphate (K<sub>2</sub>HPO<sub>4</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1051041000 </a> )</li><li>Sodium hydroxide (NaOH) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 795429-500G </a> )</li><li>Hydrochloric acid (HCl) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 320331-500ML </a> )</li><li>RNA-Protect solution (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 76526 </a> )</li><li>LB Broth (1 L) (see Recipes)</li><li>CM55 swarm agar (1 L) (see Recipes)</li><li>LSW- Agar (1 L) (see Recipes)<br/><ol class="npwthree"><li>Media (1 L)</li><li>LSW supplemental mixture (50 ml)</li></ol></li><li>Phosphate-buffered saline (PBS) (see Recipes)</li></ol><p class="pptt" id="biaoti32027">### Equipment</p><ol class="npwtwok"><li>Vortexer (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 14-955-163 </a> )</li><li>Hot plate (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 97042-690 </a> )</li><li>Stir rods (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 58947-114 </a> )</li><li>Glass culture tubes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 47729-580 </a> )</li><li>250 ml Duran bottles (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89051-796 </a> )</li><li>2 L Erlenmeyer flasks (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89090-858 </a> )</li><li>Milli-Q water purification system (Millipore)</li><li>pH meter (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89231-662 </a> )</li><li>Genesys 10S UV-Vis spectrophotometer (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 840-208100 </a> )</li><li>BD FACSAria II FACS system (BD Biosciences)</li><li>Tecan Infinite 200 Pro plate reader (Tecan Group Ltd.)<br/></li></ol><p class="pptt" id="biaoti32028">### Software</p><ol class="npwtwok"><li>BD FACSDiva 7.0 (BD Biosciences)</li><li>Microsoft Excel 16.0 (Microsoft, Inc.)</li><li>Tecan iControl 3.4.2 (Tecan Group Ltd.)<br/></li></ol><p class="pptt" id="biaoti32029">### Procedure</p><p class="ppzz">This protocol describes how to set up mixed co-swarming assays between two different strains of <em>Proteus mirabilis</em> and how to recover cells of each strain from the mixed swarm using fluorescence-activated cell sorting (FACS). The cells were recovered to measure changes in growth rate caused by phenotypic changes induced by the Ids signaling system during swarm consolidation. We use two <em>P. mirabilis</em> strains, KAG107 and KAG966, which have red and green fluorescent markers respectively. The protocol can be adapted to recover cells at other swarm phases or for other downstream applications. An example of how this protocol can be used to generate cells for RNA-Seq library preparation is described at the end of this procedure.<br/>  Setup of apparatus, particularly the FACSAria cell sorter and Tecan plate reader, should be done in advance prior to preparation of experimental samples.</p><br/><ol class="npwone"><li>Inoculation and growth of bacterial cultures<br/><strong>Day 1</strong> <ol class="npwtwo"><li>Streak out stab cultures of strains of interest onto LSW- non-swarm agar plates using an inoculation loop and sterile technique. Incubate at 37 Â°C overnight.</li><li>Pick a single colony with a sterile pipette tip and use to inoculate 5 ml LB media in a glass culture tube using sterile technique. Incubate at 37 Â°C overnight with shaking.</li><li>Set up CM55 swarm agar plates for the next day by melting premade agar in a microwave and pouring 25 ml aliquots into sterile Petri dishes. Leave to cure under a Bunsen burner or in a laminar-flow cabinet with lids partially covering the plates for 2 h. Plates can be kept at room temperature after curing for 24 h. Consistency in pouring and curing is important for the reproducibility of swarm behavior. Changing the amount of agar per plate, or curing time, can result in unpredictable swarm behavior after inoculation with <em>P. mirabilis</em>.<br/><div style="white-space:nowrap;"><br/></div></li></ol><strong>Day 2</strong> <ol class="npwtwo" start="4"><li>Measure OD<sub>600</sub> of overnight cultures using a spectrophotometer (typically, overnight cultures will reach OD<sub>600</sub> 3.0-3.5), then make 1 ml of OD<sub>600</sub> 1.0 normalized culture by diluting the relevant volume of overnight culture with fresh LB media in a sterile 1.5 ml Eppendorf tube. Cultures can optionally be normalized to OD<sub>600</sub> 0.1 rather than OD<sub>600</sub> 1.0; this serves to extend the swarming time in Step A5 by around six hours without affecting the experimental outcome. Altering the OD in this way can be useful when organizing experimental timings.</li><li>For experiments using more than one strain (<em>e.g.</em>, co-swarming assays), add portions of the normalized cultures (to a total volume of ~200 Âµl) at the desired proportions to a fresh Eppendorf tube. Vortex to mix.</li><li>Inoculate CM55 swarm agar plates by pipetting 1 Âµl of mixed normalized cell culture onto the center of the plate. Incubate at 37 Â°C until completion of the first swarm ring, including consolidation, and emergence of the second swarm ring (around 8 h). It is important to begin the next step in the protocol at this time period to remove experimental errors caused by lack of swarm culture synchronization. To this end, plates should be checked by eye every 30 min after the emergence of the first swarm ring. Plates are most likely ready for harvesting as soon as the second swarm ring has begun to emerge from the consolidation area corresponding to the completed first swarm ring. Example images can be seen in Figure 1. Plates are ready once they have reached a state similar to Figure 1B.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200302/20200302170425_4005.jpg" width="271"/><br/><strong>Figure 1. Images of progressing swarm rings of a single <em>P. mirabilis</em> swarm culture.</strong> A. A swarm culture on a CM55 agar plate that has completed a single swarm ring and has begun consolidating. B. The same swarm culture after emergence of the second swarm ring, visible as a faint band outside the first swarm ring. C. The same swarm culture following completion of the second swarm ring. Arrows show position of outer edge of swarm ring.<br/><br/></li><li>Confirm emergence of the second swarm ring using light microscopy. Elongated swarmer cells should form visible rafts emerging from a densely-packed consolidation area, around 1 cm from the center of the plate. 10x or 40x magnification of a phase contrast, epifluorescence, brightfield or DIC image is sufficient to distinguish rafts of motile, elongated swarming cells from shorter, stationary consolidating cells by direct imaging of a plate. Figure 2 shows example phase contrast and Green Fluorescent Protein (GFP) epifluorescence images of swarming cells and consolidating cells.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200302/20200302170546_5274.jpg" width="135"/><br/><strong>Figure 2. Phase contrast (A, B) and GFP epifluorescence (C, D) microscopy images of consolidating cells (C) and swarming cells (D).</strong> Images taken at 40x magnification. Fluorescent cells were added to a non-fluorescent population at a ratio of 1:10 to allow individual cells to be seen. Scale bar = 40 Âµm.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparing cells for FACS<ol class="npwtwo"><li>Remove and dispose of the inoculum from the rest of the experimental sample by punching through the agar with a disposable pipette tip. It is necessary to discard this subset of cells as it does not take part in self recognition behavior. A demonstration can be seen in Video 1.<br/><br/><!--video3543-1å¼å§--><div class="videodiv" id="vd3543-1" style="height:480px;"></div><div class="videophoto" id="vp3543-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3543/3543-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Removing inoculum from swarm plates using a pipette tip</strong> </div><!--video3543-1ç»æ--><br/></li><li>Wash the remaining cells off the agar plate by pipetting 1-1.5 ml sterile PBS onto the plate and gently lifting cells off the surface with an inoculation plate spreader. Once all cells have been harvested, tilt the plate slightly to pool the PBS/cell mix at one side, then pipette into a 1.5 ml microcentrifuge tube chilled on ice.</li><li>Dilute the cell suspension using ice-cold PBS in a 50 ml conical tube until mixture is barely turbid (approximately OD<sub>600</sub> 0.05) and decant 2 ml into a chilled FACS collection tube.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>FACSAria setup<br/>Here we describe the settings for sorting fluorescently labeled bacteria on a BD FACSAria. We assume familiarity with operation of a FACSAria and precalibration of the C&amp;ST, droplet delay, side stream angle and breakoff. Users should receive training in operating the FACSAria prior to performing this assay. We recommend setting up a practice run to determine sorting settings prior to generating experimental samples for downstream analysis.<ol class="npwtwo"><li>Turn on FACSAria and controlling PC 30 min prior to experimental start. Load FACSDiva software.</li><li>Install the high pressure 70 PSI nozzle.</li><li>Set Sample Temperature in the FACSDiva Cytometer menu to 4 Â°C. If the sort collection chamber can be temperature controlled, set that to 4 Â°C also.</li><li>Insert labeled FACS collection tubes into the sort collection chamber.</li><li>Install the sample into the FACSAria sample injection chamber and load.</li><li>Arrange the FACSDiva workspace so that forward scatter (FSC) and side scatter (SSC) can be charted against one another.</li><li>Set up sorting gates to retain particles with forward scatter (FSC) and side scatter (SSC) of approximately 1,000-50,000 (Figure 3). <br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200302/20200302170648_7219.jpg" width="101"/><br/><strong>Figure 3. Chart of side-scatter (SSC) vs forward-scatter (FSC) of <em>P. mirabilis</em> particles run through a FACSAria.</strong> The area labeled P2 represents a sorting gate that has been set up to isolate single particles from doublets.<br/><br/></li><li>Set up a second chart measuring red fluorescence intensity against green intensity: each color corresponds to a strain of interest. Set up sorting gates to isolate particles with fluorescence intensity of approximately 10<sup>3</sup> or higher (Figure 4).<br/><br/><img alt="" class="layerphoto" height="102" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200302/20200302170725_8658.jpg" width="100"/><br/><strong>Figure 4. Chart of Red Fluorescent Protein (RFP) intensity (PE-Texas Red-A, Y-axis) versus GFP intensity (FITC-A, X-axis) for single </strong><em><strong>P. mirabilis</strong></em><strong> sorted particles (the P2 subset from Figure 2).</strong> RFP+ and GFP+ gates are set up to isolate red and green particles respectively.<br/><br/></li><li>In the Sort Layout window, select the relevant gates for the collection tubes, set Precision to âPurityâ and Particle Count to 1,000,000.</li><li>Press âSortâ in the Sort Layout window to begin sorting into the collection tubes. Check periodically that the tubes do not overfill. If they do, pause the sort from the Sort Layout window, turn off the deflection plates from the Acquisition Dashboard, swap out the collection tubes, and restart the sort from the Sort Layout window.</li><li>Once the sorted particle count has been reached, unload the sample. To clean the FACSAria, load and run a tube containing 20% bleach solution through the cell sorter for 10 min, then load and run a tube containing Milli-Q water for 10 min.</li></ol><em>Note: Samples for RNA-Seq library preparations can be isolated through FACS. If this is done, cells should be harvested from swarm plates with 1x PBS and sorted into a microcentrifuge tube containing 500 Âµl RNA-Protect solution. We recommend limiting sorting time for individual samples to 10 min to avoid transcriptomic shifts caused by harvesting and sorting. Multiple FACS runs can be combined into a single sample. RNA can then be isolated for downstream applications via standard kits such as the Qiagen RNEasy Mini.</em><br/><br/></li><li>Measuring growth recovery following swarming<ol class="npwtwo"><li>Record the number of sorted particles for each condition. Typically, 100,000 to 1,000,000 particles will be sorted. One strain will have a higher sorted particle count than the other. </li><li>Normalize cell mixtures to a standardized concentration (typically 100,000 cells/ml) by adding defined volumes of sorted cell mixture to a defined volume of sterile PBS. Keep normalized cell mixtures on ice.</li><li>Immediately after sorting and normalization, set up growth recovery assays on flat-bottomed 96-well plates. For the Tecan plate reader described here, plates need not be covered or sealed. For each sorted cell mixture, add 50 Âµl sorted cell mixture to a well containing 250 Âµl LB broth. If sufficient volume of cell mixture is available, set up technical duplicates in adjacent wells. Run negative control wells containing 300 Âµl LB broth. Carefully note which mixtures are in which well.</li><li>Turn on Tecan plate reader and the controlling PC. Insert the 96-well plate with samples into the plate reader. Ensure that the 595 nm absorbance filterset is in the filter slot.</li><li>Measure OD<sub>595</sub> of wells using the Tecan plate reader. Open the Tecan iControl software and run the following program:<br/>Temperature 37 Â°C<br/>Plate: Costar 96 Flat Transparent<br/>Part of Plate: Select the wells containing experimental samples<br/>Kinetic cycle: duration 23 h:50 min:0 s<br/><ol class="npwthree"><li>Absorbance measurement 595 nm, 25 flashes</li><li>Shaking 900 s, amplitude 1 mm, linear mode</li><li>Shaking 898 s, amplitude 1 mm, linear mode</li><li>Wait (Timer) 2 s</li></ol>This will take an OD<sub>595</sub> measurement every 30 min for 24 h.</li><li>The Tecan iControl software will automatically export to an open Excel spreadsheet. Save the resulting files for further analysis.</li></ol></li></ol><p class="pptt" id="biaoti32030">### Data analysis</p><p class="ppzz">The Tecan iControl software outputs timecourse files as .xlsx spreadsheet files, which can be analyzed and visualized using most statistics and graphing packages. Typically, growth recovery can be easily visualized by graphing the change in optical density over time for each strain of interest. An example graph, from Tipping and Gibbs (2019), can be seen in Figure 5; here, no difference was observed between strains.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200302/20200302170807_5188.jpg" width="153"/><br/><strong>Figure 5. Territorial exclusion does not result in long-term growth defects.</strong> Optical density at 600 nm was measured over time for liquid cultures of KAG107 and the coswarmed strain KAG966. Liquid cultures were inoculated using cells isolated by FACS from co-swarm colonies where strain KAG966 was actively excluded from the swarm front. Reprinted with permission from Tipping and Gibbs (2019).<br/><br/>Statistical significance of differences between strains can be measured by calculating Kruskal-Wallis tests for each timepoint. We recommend running at least three biological repeats for each experiment.</p><p class="pptt" id="biaoti32031">### Notes</p><p class="ppzz">It is important that FACS is begun immediately after harvesting into PBS to avoid variability between samples. Likewise, measurement of growth recovery should be done immediately after sufficient cells have been separated through FACS. Consider growing bacterial cultures and swarm plates in the vicinity of the FACS apparatus to avoid variability caused by transport time. Other FACS apparatus and plate readers can be used for this assay, but the protocol will vary; seek training in equipment prior to use.</p><p class="pptt" id="biaoti32032">### Recipes</p><ol class="npwtwok"><li>LB Broth (1 L)<br/>Tryptone 10 g<br/>Yeast extract 5 g<br/>Sodium chloride 5 g<br/>Milli-Q H<sub>2</sub>O to 1 L<br/>Mix ingredients in a 2.0 L Erlenmeyer flask, heat with stirring on a hot plate until all ingredients are dissolved, make 250 ml aliquots and autoclave. Recipe adapted from Belas <em>et al.</em> (1991).</li><li>CM55 swarm agar (1 L)<br/>Oxoid CM0055 40 g<br/>Milli-Q H<sub>2</sub>O to 1 L<br/>Mix ingredients in a 2.0 L Erlenmeyer flask, heat with stirring on a hot plate until all ingredients are dissolved, make 250 ml aliquots and autoclave. Recipe adapted from Belas <em>et al.</em> (1991). <br/><em>Note: CM55 agar is sensitive to repeated reheating: once melted, aliquots should be used to pour plates and the remainder thrown away.</em> </li><li>LSW- Agar (1 L)<br/>This media has two components: the main media, and a supplemental mixture that is added immediately before pouring plates. Add 12 ml of supplemental mixture per liter of media.<br/><strong>Media (1 L)</strong><br/>Tryptone 10 g<br/>Yeast extract 5 g<br/>Glycerol (ultrapure) 5 ml<br/>Sodium chloride 0.4 g<br/>Bacto-agar 20 g<br/>Milli-Q H<sub>2</sub>O to 1 L<br/>Mix ingredients in a 2.0 L Erlenmeyer flask, heat with stirring on a hot plate until all ingredients are dissolved, make 250 ml aliquots and autoclave. Recipe adapted from Belas <em>et al.</em> (1991).<br/><br/><strong>LSW supplemental mixture (50 ml)</strong><br/>1 M MgSO<sub>4</sub> solution 2 ml<br/>Glycerol 4 ml<br/>1 % (w/v) nicotinic acid solution 2 ml<br/>Milli-Q water to 50 ml<br/>Add ingredients to a 50 ml conical tube, mix through shaking or vortexing, then sterilize by passing through a 0.22 Âµm filter into a fresh, sterile conical tube.</li><li>Phosphate-buffered saline (PBS)<br/>It is convenient to make this as a 10x concentrated stock for storage. Dilute to the working concentration with sterile Milli-Q water.<br/><strong>10x PBS stock</strong><br/>Sodium chloride 80 g<br/>Potassium chloride 2 g<br/>Sodium hydrogen phosphate dihydrate (Na<sub>2</sub>HPO<sub>4</sub>Â·2H<sub>2</sub>O) 14.4 g<br/>Potassium dihydrogen phosphate (KH<sub>2</sub>PO<sub>4</sub>) 2.4 g<br/>Milli-Q H<sub>2</sub>O to 1 L<br/><ol class="npwthree"><li>Mix ingredients in a 2.0 L Erlenmeyer flask, make 250 ml aliquots in Duran bottles and autoclave.</li><li>Dilute to working concentration immediately before use. The 10x stock should be pH 6.8, rising to pH 7.4 in the working 1x stock.</li><li>Measure with a pH meter or litmus paper and adjust with concentrated hydrochloric acid (HCl) or 10 M sodium hydroxide solution (NaOH) if necessary. Recipe adapted from Sambrook <em>et al.</em> (1989).</li></ol></li></ol><p class="pptt" id="biaoti32033">### Acknowledgments</p><p class="ppzz">We thank the Harvard University Bauer Core Facility, particularly Claire Reardon and Zachary Nizioleck, for assistance with FACS, and Jacob Austerman for swarm plate images. This work was funded by the David and Lucile Packard Foundation, the George W. Merck Fund and Harvard University.</p><p class="pptt" id="biaoti32034">### Competing interests</p><p class="ppzz">The authors have no competing interests.</p><p class="pptt" id="biaoti32035">### References</p><ol class="npwtwok"><li>Alteri, C. J. and Mobley, H. L. T. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27227310" target="_blank">The Versatile Type VI Secretion System.</a> <em>Microbiology spectrum</em> 4(2): 10.1128/microbiolspec.VMBF-0026-2015.</li><li>Aoki, S. K., Webb, J. S., Braaten, B. A. and Low, D. A. (2009). <a href="https://jb.asm.org/content/191/6/1777" target="_blank">Contact-dependent growth inhibition causes reversible metabolic downregulation in <em>Escherichia coli</em>.</a> <em>Journal of Bacteriology</em> 191(6): 1777-1786.</li><li>Basler, M., Ho, Brian T. and Mekalanos, John J. (2013). <a href="https://doi.org/10.1016/j.cell.2013.01.042" target="_blank">Tit-for-Tat: Type VI secretion system counterattack during bacterial cell-cell interactions.</a> <em>Cell</em> 152(4): 884-894.</li><li>Belas, R., Erskine, D. and Flaherty, D. (1991). <a href="https://jb.asm.org/content/173/19/6289" target="_blank">Transposon mutagenesis in <em>Proteus mirabilis</em>.</a> <em>Journal of Bacteriology</em> 173(19): 6289-6293.</li><li>Brunet, Yannick R., Espinosa, L., Harchouni, S., Mignot, T. and Cascales, E. (2013). <a href="https://doi.org/10.1016/j.celrep.2012.11.027" target="_blank">Imaging Type VI secretion-mediated bacterial killing.</a> <em>Cell Reports</em> 3(1): 36-41.</li><li>Garcia, E. C., Perault, A. I., Marlatt, S. A. and Cotter, P. A. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27335458" target="_blank">Interbacterial signaling via <em>Burkholderia</em> contact-dependent growth inhibition system proteins.</a> <em>Proc Natl Acad Sci U S A</em> 113(29): 8296-8301. </li><li>Gibbs, K. A., Urbanowski, M. L. and Greenberg, E. P. (2008). <a href="https://science.sciencemag.org/content/sci/321/5886/256" target="_blank">Genetic determinants of self identity and social recognition in bacteria.</a> <em>Science</em> 321(5886): 256-259.</li><li>Sambrook, J., Fritsch, E. F. and Maniatis, T. (1989). Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY.</li><li>Smith, J. M. (1964). <a href="https://doi.org/10.1038/2011145a0" target="_blank">Group selection and kin selection.</a> <em>Nature</em> 201(4924): 1145-1147.</li><li>Rauprich, O., Matsushita, M., Weijer, C. J., Siegert, F., Esipov, S. E. and Shapiro, J. A. (1996). <a href="https://jb.asm.org/content/178/22/6525" target="_blank">Periodic phenomena in <em>Proteus mirabilis</em> swarm colony development.</a> <em>J Bacteriol</em> 178(22): 6525-6538.</li><li>Souza, D. P., Oka, G. U., Alvarez-Martinez, C. E., Bisson-Filho, A. W., Dunger, G., Hobeika, L., Cavalcante, N. S., Alegria, M. C., Barbosa, L. R. S., Salinas, R. K., Guzzo, C. R. and Farah, C. S. (2015). <a href="https://doi.org/10.1038/ncomms7453" target="_blank">Bacterial killing via a type IV secretion system.</a> <em>Nature Communications</em> 6(1): 6453.</li><li>Tipping, M. J. and Gibbs, K. A. (2019). <a href="https://doi.org/10.1371/journal.ppat.1007885" target="_blank">Peer pressure from a <em>Proteus mirabilis</em> self-recognition system controls participation in cooperative swarm motility.</a> <em>PLOS Pathogens</em> 15(7): e1007885</li><li>Virta, M., Lineri, S., KankaanpÃ¤Ã¤, P., Karp, M., Peltonen, K., Nuutila, J. and Lilius, E. M. (1998). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9464386" target="_blank">Determination of Complement-Mediated Killing of Bacteria by Viability Staining and Bioluminescence.</a> <em>Appl Environ Microbiol</em> 64(2): 515-519.</li><li>Wenren, L. M., Sullivan, N. L., Cardarelli, L., Septer, A. N. and Gibbs, K. A. (2013). <a href="https://mbio.asm.org/content/mbio/4/4/e00374-13" target="_blank">Two independent pathways for self-recognition in <em>Proteus mirabilis</em> are linked by type VI-dependent export.</a> <em>MBio</em> 4(4). DOI: 10.1128/mBio.00374-13.<br/></li></ol>