<body>
            <h1>Assembly of Genetic Circuits with the Mammalian ToolKit</h1>
            <p><strong>Authors:</strong> JoÃ£o P. Fonseca, Alain R Bonny, Jason  Town, Hana  El-Samad</p>
            <h2>Abstract</h2>
            <p>The ability to rapidly assemble and prototype cellular circuits is vital for biological research and its applications in biotechnology and medicine. The Mammalian ToolKit (MTK) is a Golden Gate-based cloning toolkit for fast, reproducible and versatile assembly of large DNA vectors and their implementation in mammalian models. The MTK consists of a curated library of characterized, modular parts that can be assembled into transcriptional units and further weaved into complex circuits. These circuits are easily repurposed and introduced in mammalian cells by different methods.</p>
            <p><strong>Keywords:</strong> Cloning, Golden-gate, High-throughput, Toolkit, Library, Mammalian</p>
            <h2>Background</h2><p>Molecular cloning is a hallmark of modern biotechnology with the ability to repurpose recombinant DNA into a variety of genetic circuits that can represent a spectrum of purposes. However, a major limitation in exploring the permutations possible in construction of genetic circuits is the ability to rapidly prototype, test and implement improvements on circuit designs. For this to come to fruition, the time from designing genetic circuits to delivery into cells needs to be expedited from conventional cloning methods, such as Gibson cloning (Akama-Garren <em>et al.</em>, 2016) or restriction digests. We designed a framework in which a conventional gene circuit is deconstructed into its constituent components such that one can easily swap these components to quickly assemble vast combinations, assessing how each iteration affects function. Inspired by earlier iterations of cloning toolkits (Weber <em>et al.</em>, 2011a and 2011b; Duportet <em>et al.</em>, 2014; Lee <em>et al.</em>, 2015; Martella <em>et al.</em>, 2017; PÃ©rez-GonzÃ¡lez <em>et al.</em>, 2017; Halleran <em>et al.</em>, 2018; Pollak <em>et al.</em>, 2019), we adopted a Golden-gate cloning system where assembly of constituent parts into functional units can be accomplished in a one-pot fashion (Weber <em>et al.</em>, 2011a and 2011b). To concretely realize the promise of this framework, we built and characterized over 400 DNA parts that encode a myriad of reagents. We named the curated library and cloning framework the Mammalian ToolKit (MTK) (Fonseca <em>et al.</em>, 2019). With the MTK, swapping promoters, switching fluorophores, or rapidly prototyping protein tags constitutes changing one plasmid in a reaction. This is markedly different from having to redesign oligos and PCR verifying correct assembly for every use case, thus discouraging continued optimization of a construct of interest. The hierarchical nature of this system enables a library of parts to expand to a library of transcriptional units (TUs) that can be further combined to create multi-transcriptional unit vectors that are then delivered to cells. These TUs are delivery-agnostic, promoting their multiple uses. For example, the same TUs used in a circuit delivered to a specific locus via CRISPR-Cas9 homologous recombination can be recycled in a PiggyBac transposase transfection to be randomly integrated. While the initial time from part verification to delivery into cells is comparable to conventional cloning (approximately 4 days), the repurposing of parts and TUs allows vast combinations of circuits to be assembled in only 2 days.<br/> <br/>Despite the tremendous advantages the MTK presents, the barrier to entry may still dissuade those without a cloning background. To ameliorate the activation energy for adoption across all disciplines, here we present instructions on how to utilize the current library to assemble novel circuits, and a step-by-step protocol to âdomesticateâ new parts (that is to make them compatible with Golden Gate-based assembly of the MTK), assemble Transcriptional Units (TU), and create varied multi-transcriptional units (multi-TU). It is our goal to make this toolkit widely accessible and available to enhance the development of mammalian expression systems towards new and exciting discoveries in biology.</p>
        </body>

<p class="pptt" id="biaoti32069">## Materials and Reagents</p><ol class="npwtwok"><li>Sterile tips for micropipettes (Green-Pak<sup>TM</sup> SpaceSaver<sup>TM</sup> stacked refills for Rainin pipettes)</li><li>PCR microtubes (BioExpress, catalog number: <a class="blackNoUnder" target="_blank"> T-3135-2 </a> )</li><li>Sequencing primer MTK_P072 (gagcctatggaaaaacgc)</li><li>Sequencing primer MTK_P073 (gggcgtaatttgatatcg)</li><li>LB Broth (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L3022 </a> )</li><li>LB Broth with agar (Lennox) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L2897 </a> )</li><li>Chloramphenicol (Spectrum Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 40310016-2 </a> ) (working concentration: 100 mg/ml)</li><li>Carbenicillin solution (Spectrum Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 40310030-2 </a> ) (working concentration: 25 mg/ml)</li><li>Kanamycin sulfate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> k4000 </a> ) (working concentration: 100 mg/ml)</li><li>T4 DNA Ligase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0202L </a> )</li><li>Poly(ethylene glycol) (PEG) (Millipore-Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 81268 </a> )</li><li>T4 Polynucleotide Kinase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> M0201L </a> )</li><li>T4 DNA Ligase Reaction Buffer (10x) (NEB, catalog number: <a class="blackNoUnder" target="_blank"> B0202S </a> )</li><li>BsaI-HFv2 (NEB, catalog number: <a class="blackNoUnder" target="_blank"> R3733S </a> )</li><li>FastDigest Esp3I (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FD0454 </a> )</li><li>Miniprep kit (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FERK0503PR </a> )</li><li>T4 DNA ligase + 2.5% PEG (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32070">### Equipment</p><ol class="npwtwok"><li>Micropipettes (Rainin Pipet-Lite XLS)</li><li>Incubator shaker (New Brunswick<sup>TM</sup> Innova<sup>Â®</sup> 44, catalog number: <a class="blackNoUnder" target="_blank"> M1282-0000 </a> )</li><li>Gel Documentation System (Protein Simple AlphaImager HP)</li><li>Thermocycler (Bio-Rad C1000, catalog number: <a class="blackNoUnder" target="_blank"> 1851148 </a> )</li><li>NanoDrop (Thermo Scientific<sup>TM</sup> NanoDrop<sup>TM</sup> 2000c, catalog number: <a class="blackNoUnder" target="_blank"> ND2000 </a> )<br/></li></ol><p class="pptt" id="biaoti32071">### Procedure</p><ol class="npwone"><li>Overview<br/>The process that generates genetic circuits using the MTK is shown in Figure 1. Parts 1-8 of the MTK library can be ordered from Addgene (<a href="https://www.addgene.org/browse/article/28197510/" target="_blank">https://www.addgene.org/browse/article/28197510/</a>) or generated from oligos and PCR fragments. Domestication of new parts is done by a BsmBI Golden Gate assembly into a chloramphenicol resistant destination vector and by selection of white colonies. From the library of parts, transcriptional units (TUs) can be built by assembling Parts 1-8 with a BsaI Golden Gate reaction and selection of white, carbenicillin resistant clones. TUs, can be delivered as is or further combined, alone or together with additional TUs, into a Kanamycin resistant destination vector. This assembly is done with a BsmBI Golden Gate reaction which generates plasmids ready for introduction into mammalian cells.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303002020_9064.jpg" width="132"/><br/><strong>Figure 1. Overview of MTK cloning procedure.</strong> A. Parts are cloned from Oligos or PCR products into a Chloramphenicol-resistant domestication vector using a BsmBI Golden Gate assembly. Parts, either newly cloned or from the library available at Addgene are combined to produce Carbenicillin-resistant Transcriptional Units (TU). For delivery into mammalian cells, single or multiple TUs can be further combined to produce Kanamycin-resistant Multiple Transcription Units (Multi-TU). B. Standard TUs are composed of 8 parts. Part 1 contains the 5â connector, Part 2 is the 5âUTR (typically a promotor), Part 3 includes the coding sequence, Part 4 is the 3âUTR (typically a terminator), and Part 5 includes the 3â connector. Parts 6 through 8 are not shown and usually encode the bacterial resistance marker.<br/><br/></li><li>Part domestication<ol class="npwtwo"><li>Existing Parts<ol class="npwthree"><li>Order part from Addgene (<a href="https://www.addgene.org/browse/article/28197510/" target="_blank">https://www.addgene.org/browse/article/28197510/</a>).</li><li>Streak bacteria in LB agar plate with corresponding antibiotic.</li><li>Pick a single white colony with a sterile microtip and inoculate 5 ml LB media with corresponding antibiotic.</li><li>Grow bacteria for 16 h at 37 Â°C at 250 rpm in a shaker.</li><li>Prepare plasmid DNA using Thermo Fisher Miniprep kit, following the manufacturerâs instructions.</li><li>Measure the concentration of DNA on a NanoDrop.</li><li>Sequence plasmid with primers MTK_P072 and MTK_P073 to assure presence of the correct part.</li></ol></li><li>PCR<ol class="npwthree"><li>Forward primers are designed as (N<sub>4</sub>)CGTCTCNTCGGTCTCN(X<sub>4-6</sub>)(Y<sub>16-22</sub>).<br/>N is any base.<br/>X are part type-specific BsaI overhang bases. See Table 1.<br/>Y are bases homologous to the specific part and that allow PCR.</li><li>Reverse primers are designed as (N<sub>4</sub>)CGTCTCNGGTCTCN(X<sub>4-6</sub>)(Y<sub>16-22</sub>).<br/>N is any base.<br/>X are part-specific BsaI overhang bases. See Table 1.<br/>Y are reverse complement bases homologous to the specific part and that allow PCR.</li><li>Primers are ordered as custom 25 nmole DNA Oligos from idtDNA (<a href="https://www.idtdna.com/" target="_blank">https://www.idtdna.com/</a>)</li><li>For Part 1 (Connectors), and Part 5â (Reverse connectors), an additional overhang needs to be added to the forward primer (see Table 2).</li><li>For Part 5 (Connectors), and Part 1â (Reverse connectors), an additional overhang needs to be added to the reverse primer (see Table 2).</li><li>For Parts 2, 3, 4 that encode sgRNAs for spCAS9, the following oligos are ordered as custom 25 nmole DNA Oligos from idtDNA:<br/>TGTTTG(N<sub>20</sub>)G<br/>TAAAC(Nâ<sub>20</sub>)CA<br/>N is the guide-specific sequence and Nâ its reverse complement<br/><br/><strong>Table 1. Primers for Part domestication</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003248_9128.jpg" width="600"/><br/><br/><strong>Table 2. Connector-specific overhangs</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003334_7465.jpg" width="320"/><br/><br/></li></ol></li><li>Removal of internal BsmBI and BsaI sites<ol class="npwthree"><li>If internal BsaI or BsmBI sites exist, additional primers that flank these sites are required. These will create additional PCR products that are pooled together in a Golden Gate reaction to generate an MTK-compatible part. The following requirements need to be followed:<ol class="npwfour"><li>Primers should use BsmBI sites in forward direction to ensure they are not incorporated in the final product.</li><li>Overhangs generated by PCRs should be as different as possible from the overhangs generated for part building (TCGG and GGCT). For example CATA and not TAGG.</li><li>Point mutations to remove BsaI/BsmBI sites should be silent in coding DNA sequences.</li><li>Point mutations to remove BsaI/BsmBI sites should maintain CG content in non-coding DNA sequences.</li></ol></li><li>Example of primers used for removal of internal BsaI site in domestication of ERKKTR (ERK Kinase Translocation Reporter, MTK3_021, Figure 2). Overhang in the example is AGGA.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003443_1457.jpg" width="241"/><br/><strong>Figure 2. Part of ERKKTR sequence before domestication.</strong> Bsal site is highlighted in yellow. Both forward and reverse primers encode BsmBI sites for posterior Golden Gate assembly. Additionally, reverse primer has A to T substitution that maintains G48, but removes Bsal from the final product.<br/><br/></li><li>To facilitate part domestication for coding sequences, we have created a python script (available here: <a href="https://github.com/weinerlab/mtk_primer_generator" target="_blank">https://github.com/weinerlab/mtk_primer_generator</a>) that generates candidate primers given an input sequence and a part number. The script finds BsmBI and BsaI restriction sites that need to be removed and then generates a list of potential silent mutations that are capable of removing each restriction site. Finally, it checks combinations of these potential mutations for general compatibility with Golden Gate assembly and for mutual compatibility with one another, enabling a streamlined, one-pot Golden Gate assembly of parts.</li></ol></li><li>Golden-gate reaction for PCR-generated parts<ol class="npwthree"><li>PCR generated (He, 2011) or gene block encoded parts are domesticated in a PCR microtube, using the reaction mixture shown in Table 3.</li><li>Run the protocol described in Table 4 in a thermocycler.<br/><br/><strong>Table 3. Golden Gate reaction setup for Part domestication</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003615_6500.jpg" width="500"/><br/><br/><strong>Table 4. Thermocycling conditions for Golden Gate reaction</strong><br/><img alt="" class="layerphoto" height="148" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003725_1481.jpg" width="450"/><br/><br/></li><li>If internal BsmbI sites must be retained in the domesticated part, the third step of the thermocycling reaction (Table 4, 37 Â°C, 10 min) is removed from the protocol.</li><li>Transform bacteria as described in Sharma <em>et al.</em> (2017).</li><li>Proceed from Step B3 in domestication of existing parts.<br/>Usually, pick 2 white colonies (see Notes) in case the PCR has added a mutation in the newly cloned part.</li></ol></li><li>Golden-gate reaction for oligo-generated parts<ol class="npwthree"><li>Oligos for oligo-generated parts are first phosphorylated by incubating the following mixture at 37 Â°C for 1 h in a PCR microtube (Table 5).<br/><br/><strong>Table 5. Oligo Annealing reaction setup</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003817_8817.jpg" width="500"/><br/><br/></li><li>Dilute the phosphorylated oligos with 190 Î¼l of distilled water.</li><li>To anneal the oligos, a thermocycler protocol was prepared to hold the temperature of the reaction at 96 Â°C for 6 min, and ramp down 0.1 Â°C per second to 23 Â°C. The reaction is then held at 23 Â°C indefinitely.</li><li>Domestication is performed using the reaction mixture described in Table 3, replacing the PCR fragments with the same volume of annealed oligos. For spCAS9 sgRNA domestication, MTK0_027 is replaced by MTK0_001.</li><li>Proceed from Steps 4c to 4e as described above.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transcriptional Unit (TU) assembly<ol class="npwtwo"><li>Single TU<ol class="npwthree"><li>Transcriptional units (TUs) are defined by Parts 1 through 8 from the MTK library. If the desired circuit requires only one TU, we recommend assembly of the TU with MTK678_001, and the appropriate Part 1 LS connector and Part 5 RE connector. This will allow the recycling of the TU for different methods of delivery into mammalian cells.</li><li>Single TUs are built using the reaction mixture described in Table 6.<br/>If additional parts are required (<em>e.g.</em>, instead of Part 3, using Part 3a and Part 3b), change volume of distilled water so reaction volume is in total equal to 10 Î¼l.<br/><br/><strong>Table 6. Golden Gate reaction setup for TU assembly</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003858_6759.jpg" width="500"/><br/><br/></li><li>Proceed with Golden Gate reaction as described in Table 4.</li><li>Transform bacteria as described in Sharma <em>et al.</em> (2017).</li><li>Pick two single white colonies (see Notes) with a sterile microtip and inoculate 5 ml LB media with Carbenicillin.</li><li>Grow bacteria for 16 h at 37 Â°C at 250 rpm in a shaker.</li><li>Prepare plasmid DNA using Thermo Fisher Miniprep kit, following the manufacturerâs instructions.</li><li>Measure the concentration of DNA on a NanoDrop.</li><li>Digest plasmids with NotI or other appropriate restriction enzymes, following the manufacturerâs instructions, and verify correct size of fragments.</li><li>Example of TU encoding the CAG-driven expression of ERKKTR fused to the fluorescent protein mAzamiGreen, with the bgh polyadenylation signal (Figure 3).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303003948_1741.jpg" width="275"/><br/><strong>Figure 3. Assembly and verification of a single TU encoding ERKKTR.</strong> A. Plasmids required for the assembly of TU for the expression of mAzamiGreen-fused ERKKTR. Plasmids enconding connectors LS and RE (highlighted in red) are essential for the generation of single TU circuits. B. Assembled ERKKTR encoding TU. Different parts used in assembly are shown in different colors and Notl sites. C. Expected fragment size of TU encoding ERKKTR after Notl digestion.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>TU for MultiTU Plasmids<br/>The procedure for the generation of TUs for multi-TU plasmids is identical to the one described above for single TUs. However, the connectors used must allow the ordered assembly of the TUs in the final multi-TU plasmid. Below are examples of two separate TUs built from available parts (Figure 4). These two plasmids are assembled in a final plasmid that encodes both transcriptional units (Figure 5B).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303004034_3471.jpg" width="275"/><br/><strong>Figure 4. Assembly of two TUs for posterior multi-TU assembly.</strong> Plasmids required for first TU (A, ERKKTR) and second TU (B, H2B) assembly. Highlighted in red are LS and RE connectors required for assembly into destination vector. Highlighted in green are connectors that allow the order assembly of first and second TU in the destination vector. Note that L1 connects with R1. Final multi-TU plasmid is shown in Figure 5B.<br/><br/></li><li>TU for Multicistronic MTU Plasmids<br/>Multicistronic plasmids are used when the coexpression of two or more genes of interest is desired. In the MTK, this is achieved by using viral P2A elements. These elements lead the ribosome to skip the synthesis of a peptide bond at the C-terminus of a 2A element, therefore creating two peptides from the same mRNA.<ol class="npwthree"><li>TUs for multicistronic multi-TU plasmids need to be assembled using Part 1 connector plasmids that encode P2A-elements and with Part 2 and Part 4 plasmids that encode spacer sequences that maintain coding-DNA sequences in frame.</li><li>The first TU of a multicistronic plasmid must use a Part 1 without P2A elements, a Part 2 that encodes a promoter and a Part 4 without a stop codon and a PA signal (MTK4_006).</li><li>The final TU of a multicistronic multi-Plasmid must use a Part 1 encoding a P2A element, a Part 2 that encodes a spacer (MTK2_013) and a Part 4 with a stop codon and a polyadenylation signal.</li><li>If more than two TUs are used in the multicistronic construct, the middle TUs, must use Part 1 encoding a P2A element, MTK2_013 and MTK4_006.</li><li>The protocol for golden-gate assembly and verification of plasmid is identical to the one detailed above for the single TUs.</li></ol></li><li>TUs for sgRNAs<br/>TUs encoding sgRNAs are assembled in the same manner as single TUs, but Parts 2, 3 and 4 are replaced by the sgRNA-containing Part 234.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Multi transcriptional unit building<ol class="npwtwo"><li>Multi TU units are built from verified TUs that use connectors that will allow them to be assembled, in order, into a destination vector.</li><li>Multi TUs are built using the reaction mixture described in Table 7.<br/><br/><strong>Table 7. Golden Gate reaction setup for MTU assembly</strong><br/><img alt="" class="layerphoto" height="209" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303004126_8006.jpg" width="600"/><br/><br/></li><li>Proceed with Golden Gate reaction as described in Table 4.</li><li>Transform bacteria as described in Sharma <em>et al. </em>(2017).</li><li>Pick two single white colonies (see Notes) with a sterile microtip and inoculate 5 ml LB media with Kan.</li><li>Grow bacteria for 16 h at 37 Â°C at 250 rpm in a shaker.</li><li>Prepare plasmid DNA using Thermo Fisher Miniprep kit, following the manufacturerâs instructions.</li><li>Measure the concentration of DNA on a NanoDrop.</li><li>Digest plasmids with NotI or other appropriate restriction enzymes, following the manufacturerâs instructions and verify the correct size of fragments.</li><li>An example of multi-TU encoding TUs is shown in Figure 4, in the piggybac destination vector (Figure 5).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303004220_2502.jpg" width="129"/><br/><strong>Figure 5. Assembly and verification of a Multi-TU encoding ERKKTR and H2B in piggybac destination vector.</strong> A. Plasmids required for the assembly of multi-TU. Highlighted in red is destination vector enconding required sequences for piggybac-directed insertion into genome. B. Assembled multi-TU encoding ERKKTR and H2B. Different plasmids used in assembly are shown in different colors and Notl sites. C. Expected fragment size of multi-TU encoding ERKKTR and H2B after Notl digestion.</li></ol></li></ol><p class="pptt" id="biaoti32072">### Notes</p><p class="ppzz"><strong><em>Picking white colonies</em></strong><br/>Correct assemblies into MTK0_027, MTK678_001 or any final destination vector will produce white <em>E. coli</em> colonies after transformation. Most wrong assemblies will generate green colonies. This happens because the destination vectors contain a bacterial GFP-expression cassette that is lost upon correct assembly. While green colonies are visible under white light, they are easily distinguishable under blue light. For that, we use a Gel Documentation System as shown in Figure 6.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200303/20200303004345_8494.jpg" width="199"/><br/><strong>Figure 6. Transformants of part domestication.</strong> Plate with <em>E. coli</em> transformed with product of Golden Gate Assembly of MTK0_027 and 3 PCR products under (A) white light and (B) blue light. Incorrect assemblies are easily detectable under blue light (green colonies, arrows).</p><p class="pptt" id="biaoti32073">### Recipes</p><ol class="npwtwok"><li>T4 DNA ligase + 2.5% PEG<br/><ol class="npwthree"><li>Thaw 10x T4 Ligase buffer</li><li>Dilute 50% PEG 1:20 in Ligase buffer</li><li>Aliquot 6 Î¼l in PCR microtubes</li><li>Store at -20 Â°C</li></ol></li></ol><p class="pptt" id="biaoti32074">### Acknowledgments</p><p class="ppzz">The protocol presented herein was adapted from Fonseca <em>et al.</em> (2019). We would like to thank Seesha Takagashi and Marisa Rosa for critical feedback on the protocol. This work was supported by the National Science Foundation Award # 1715108 to H.E-.S and the Defense Advanced Research Projects Agency (grant number HR0011-16-2-0045 to H.E-.S). H.E-.S is an investigator in the Chan Zuckerberg Biohub. This work was also supported by the National Defense Science &amp; Engineering Graduate (NDSEG) Fellowship awarded to A.R.B..</p><p class="pptt" id="biaoti32075">### Competing interests</p><p class="ppzz">No competing interests.</p><p class="pptt" id="biaoti32076">### References</p><ol class="npwtwok"><li>Akama-Garren, E. H., Joshi, N. S., Tammela, T., Chang, G. P., Wagner, B. L., Lee, D. Y., Rideout, W. M., 3rd, Papagiannakopoulos, T., Xue, W. and Jacks, T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26887506" target="_blank">A modular assembly platform for rapid generation of DNA constructs.</a> <em>Sci Rep</em> 6: 16836.</li><li>Duportet, X., Wroblewska, L., Guye, P., Li, Y., Eyquem, J., Rieders, J., Rimchala, T., Batt, G. and Weiss, R. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25378321" target="_blank">A platform for rapid prototyping of synthetic gene networks in mammalian cells.</a> <em>Nucleic Acids Res</em> 42(21): 13440-13451.</li><li>Fonseca, J. P., Bonny, A. R., Kumar, G. R., Ng, A. H., Town, J., Wu, Q. C., Aslankoohi, E., Chen, S. Y., Dods, G., Harrigan, P., Osimiri, L. C., Kistler, A. L. and El-Samad, H. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31686495" target="_blank">A toolkit for rapid modular construction of biological circuits in mammalian cells.</a> <em>ACS Synth Biol</em>.</li><li>Halleran, A. D., Swaminathan, A. and Murray, R. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29715010" target="_blank">Single day construction of multigene circuits with 3G assembly.</a> <em>ACS Synth Biol</em> 7(5): 1477-1480.</li><li>He, F. (2011). <a href="https://doi.org/10.21769/BioProtoc.53" target="_blank">Standard PCR Protocol.</a> <em>Bio-101</em>: e53.</li><li>Lee, M. E., DeLoache, W. C., Cervantes, B. and Dueber, J. E. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25871405" target="_blank">A highly characterized yeast toolkit for modular, multipart assembly.</a> <em>ACS Synth Biol</em> 4(9): 975-986.</li><li>Martella, A., Matjusaitis, M., Auxillos, J., Pollard, S. M. and Cai, Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28418644" target="_blank">EMMA: an extensible mammalian modular assembly toolkit for the rapid design and production of diverse expression vectors.</a> <em>ACS Synth Biol</em> 6(7): 1380-1392.</li><li>PÃ©rez-GonzÃ¡lez, A., Kniewel, R., Veldhuizen, M., Verma, H. K., Navarro-RodrÃ­guez, M., Rubio, L. M. and Caro, E. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29132331" target="_blank">Adaptation of the GoldenBraid modular cloning system and creation of a toolkit for the expression of heterologous proteins in yeast mitochondria.</a> <em>BMC Biotechnol</em> 17(1): 80.</li><li>Pollak, B., Cerda, A., Delmans, M., Ãlamos, S., Moyano, T., West, A., GutiÃ©rrez, R. A., Patron, N. J., Federici, F. and Haseloff, J. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30521109" target="_blank">Loop assembly: a simple and open system for recursive fabrication of DNA circuits.</a> <em>New Phytol</em> 222(1): 628-640.</li><li>Sharma, N., Anleu Gil, M. X. and Wengier, D. (2017). <a href="https://doi.org/10.21769/BioProtoc.2590" target="_blank">A Quick and Easy Method for Making Competent <em>Escherichia coli</em> Cells for Transformation Using Rubidium Chloride.</a> <em>Bio-101</em>: e2590.</li><li>Weber, E., Engler, C., Gruetzner, R., Werner, S. and Marillonnet, S. (2011a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21364738" target="_blank">A modular cloning system for standardized assembly of multigene constructs.</a> <em>PLoS One</em> 6(2): e16765.</li><li>Weber, E., Gruetzner, R., Werner, S., Engler, C. and Marillonnet, S. (2011b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21625552" target="_blank">Assembly of designer TAL effectors by Golden Gate cloning.</a> <em>PLoS One</em> 6(5): e19722.<br/></li></ol>