<body>
            <h1><em>In vitro</em> Measurement of CMP-Sialic Acid Transporter Activity in Reconstituted Proteoliposomes</h1>
            <p><strong>Authors:</strong> James Cahill, Shivani Ahuja, Matthew R. Whorton</p>
            <h2>Abstract</h2>
            <p>Nucleotide-sugar transporters (NSTs) facilitate eukaryotic cellular glycosylation by transporting nucleotide-sugar conjugates into the Golgi lumen and endoplasmic reticulum for use by glycosyltransferases, while also transferring nucleotide monophosphate byproducts to the cytoplasm. Mutations in this family of proteins can cause a number of significant cellular pathologies, and wild type members can act as virulence factors for many parasites and fungi. Here, we describe an <em>in vitro</em> assay to measure the transport activity of the CMP-sialic acid transporter (CST), one of seven NSTs found in mammals. While <em>in vitro</em> transport assays have been previously described for CST, these studies failed to account for the fact that 1) commercially available stocks of CMP-sialic acid (CMP-Sia) are composed of ~10% of the higher-affinity CMP and 2) CMP-Sia is hydrolyzed into CMP and sialic acid in aqueous solutions. Herein we describe a method for treating CMP-Sia with a nonselective phosphatase, Antarctic phosphatase, to convert all free CMP to cytidine. This allows us to accurately measure substrate affinities and transport kinetics for purified CST reconstituted into proteoliposomes.</p>
            <p><strong>Keywords:</strong> Protein purification, Transport assay, Nucleotide-sugar transporters, SLC35, CMP, CMP-Sia, Sialic acid, Lipid vesicle reconstitution</p>
            <h2>Background</h2><p>Once synthesized in the cytoplasm or nucleus, nucleotide-coupled sugars are transported into the lumen of the endoplasmic reticulum (ER) or Golgi apparatus by nucleotide-sugar transporters (NSTs) (Aoki <em>et al.</em>, 2003). Within these subcellular compartments, glycosyltransferases utilize the sugar moieties to glycosylate lipids and proteins, producing nucleotide monophosphates (NMPs) as a byproduct (Capasso and Hirschberg, 1984; Milla and Hirschberg, 1989; Waldman and Rudnick, 1990; Tiralongo <em>et al.</em>, 2006). Many glycosyltransferases are inhibited by NMPs, and the lumenal concentration of the latter must be kept low in order to allow for proper glycan synthesis (Hirschberg <em>et al.</em>, 1998). To facilitate this, NSTs act as antiporters by transporting their corresponding NMP back to the cytoplasm.<br/><br/>By regulating the concentration of nucleotide sugars in the Golgi and ER lumens, NSTs have a direct impact on glycosylationâthe most common form of protein and lipid modification. Mutations that impair NST function can therefore impair proper protein folding, stability, and functionality, with many adverse physiological effects (Dwek <em>et al.</em>, 2002; Moremen <em>et al.</em>, 2012; Ohtsubo and Marth, 2006; Stanley, 2011). There are a number of debilitating genetic diseases arising from mutations in the solute carrier 35 (SLC35) gene family, from which NSTs are derived (Jaeken and Matthijs, 2007; Song, 2013). Additionally, because NSTs are virulence factors for pathogens, they are potential targets for antiparasitic and antifungal drugs (Descoteaux <em>et al.</em>, 1995; Ma <em>et al.</em>, 1997; Hong <em>et al.</em>, 2000; Engel <em>et al.</em>, 2009; Caffaro <em>et al.</em>, 2013; Liu <em>et al.</em>, 2013). Studies have also shown that blocking NSTs can inhibit tumor metastasis, as altered cell surface protein glycosylation profiles are often a feature of cancerous cells (Caffaro and Hirschberg, 2006; Ohtsubo and Marth, 2006; Esko and Bertozzi, 2009; Song, 2013; Hadley <em>et al.</em>, 2014; Stowell <em>et al.</em>, 2015; Wang <em>et al.</em>, 2016).<br/><br/>Given the importance of NSTs, it is necessary to not only understand how they transport their physiological substrates, but also how mutations and potential inhibitors affect their transport activity. Functional characterization of these transporters via transport assays and other means is essential in understanding genetic pathologies and is a key component in drug development. Herein we describe an <em>in vitro</em> method for measuring the uptake of CMP-sialic acid (CMP-Sia) into proteoliposomes reconstituted with the CMP-Sia transporter (CST), one of seven known NSTs found in humans. While developing this method, we realized that commercial stocks of CMP-Sia contain approximately 10% CMP (Ahuja and Whorton, 2019). This observation, coupled with the known fact that CMP-Sia is hydrolyzed in aqueous solution to CMP and sialic acid (Beau <em>et al.</em>, 1984; Horenstein and Bruner, 1996), presented a problem for structural and functional characterization of CST because the affinity of CMP towards CST is approximately 100 times higher than that of CMP-Sia (Ahuja and Whorton, 2019). For transport assays, this abundance of CMP would lead to errors in determining the affinity and transport kinetics of CMP-Siaâan issue that, to our knowledge, had not been addressed in the literature.<br/><br/>While methods have been described to purify CMP from CMP-Sia (Beau <em>et al.</em>, 1984), we ultimately decided to not pursue these since significant amounts of CMP would still be generated through CMP-Sia hydrolysis during long-duration experiments (<em>e.g.</em>, multi-day crystallization trials). In addition, although several CMP-Sia derivatives have been described which are resistant to hydrolysis (Burkart <em>et al.</em>, 2000; Kajihara <em>et al.</em>, 2011; Watts and Withers, 2004), we decided not to employ these since they may have different affinities and transport kinetics than unmodified CMP-Sia, and because they are not commercially available and would thus require custom synthesis. We therefore developed a method of using a nonselective nucleotide phosphatase, Antarctic phosphatase (AnP), to convert all CMP in CMP-Sia solutions to cytidine, which does not have a measurable affinity for CST and would not affect the outcome of functional characterization studies. This approach has allowed us to determine reliable affinity and transport rate constants for CMP-Sia transport by CST. We anticipate that this approach would also be useful for studies of other aspects of glycosylation machinery that require CMP-free solutions of CMP-Sia. Although CMP-Sia is the only nucleotide sugar known to rapidly hydrolyze in aqueous solutions, since AnP is nonselective, it may also be useful for the study of other NSTs if commercial stocks of their nucleotide-sugar substrates also contain high levels of a higher-affinity NMP.</p>
        </body>

<p class="pptt" id="biaoti32297">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml polypropylene conical tubes (Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 352098 </a> )</li><li>High-g-rated microfuge tubes (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> 357448 </a> )</li><li>15 ml 50k MWCO Centrifugal Filter (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> UFC905024 </a> )</li><li>4 ml 50k MWCO Centrifugal Filter (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> UFC805024 </a> )</li><li>0.4 Î¼m extruder filter (Whatman, catalog number: <a class="blackNoUnder" target="_blank"> 800282 </a> )</li><li>0.22 Î¼m mixed cellulose ester membrane (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> GSTF02500 </a> )</li><li>7 ml glass scintillation vials (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 03-340-4A </a> )</li><li>SEC Column (GE Life Sciences Superdex 200 Increase 10/30 GL; catalog number: <a class="blackNoUnder" target="_blank"> 28-9909-44 </a> )</li><li>1 cm diameter glass column (Kimble Flex-Column, catalog number: <a class="blackNoUnder" target="_blank"> 420401 </a> )</li><li>Bio-Beads SM-2 adsorbent media (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 152-8920 </a> )</li><li>cDNA for the <em>Slc35a1</em> gene; mouse CMP-sialic acid transporter; mCST (Biobasic, Uniprot: Q61420)</li><li>EasySelect Pichia Expression Kit (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> K174001 </a> )</li><li><em>Pichia pastoris</em> strain SMD1168H (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> C18400 </a> )</li><li>PreScission protease<br/><em>Note: We express and purify this ourselves, but it can also be obtained commercially, e.g., GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 27084301 </a> .</em> </li><li>Cytidine 5â-monophospho-N-acetylneuraminic acid; CMP-sialic acid; CMP-Sia (Carbosynth, catalog number: <a class="blackNoUnder" target="_blank"> MC04391 </a> )</li><li>[<sup>3</sup>H]CMP-sialic acid (American Radiolabeled Chemicals, catalog number: <a class="blackNoUnder" target="_blank"> ART 0147-50 Î¼Ci </a> )</li><li>4-(2-hydroxyethyl)-1-piperazineethanesulfonic acid; HEPES (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP310-1 </a> )</li><li>Sodium chloride; NaCl (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S271-3 </a> )</li><li>Potassium chloride; KCl (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> P217-3 </a> )</li><li>Magnesium chloride; MgCl<sub>2</sub> (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> M33-500 </a> )</li><li>Zinc chloride; ZnCl<sub>2</sub> (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 380130050 </a> )</li><li>Benzamidine hydrochloride hydrate (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 105240250 </a> )</li><li>Imidazole (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> O3196-500 </a> )</li><li>Deoxyribonuclease I; DNase I grade II, from bovine pancreas (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 10104159001 </a> )</li><li>Pepstatin A (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP2671-25 </a> )</li><li>Leupeptin hemisulfate (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP2662-100 </a> )</li><li>Aprotinin from bovine lung (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP2503-10 </a> )</li><li>Phenylmethanesulfonyl fluoride (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 215740010 </a> )</li><li>N-dodecyl-Î²-D-maltopyranoside solgrade; DDM (Anatrace, catalog number: <a class="blackNoUnder" target="_blank"> D310S </a> )</li><li>N-dodecyl-Î²-D-maltopyranoside anagrade; DDM (Anatrace, catalog number: <a class="blackNoUnder" target="_blank"> D310 </a> )</li><li>20% DDM (solgrade) solution made in water; stored at -20 Â°C</li><li>Methanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A452-SK4 </a> )</li><li>1,3-Bis[tris(hydroxymethyl)amino]propane; Bis-tris-propane-HCl (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 202640250 </a> )</li><li>Sodium hydroxide; NaOH (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S318-1 </a> )</li><li>TALON metal affinity resin (Takara, catalog number: <a class="blackNoUnder" target="_blank"> 635504 </a> )</li><li>DL-1,4-Dithiothreitol; DTT (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 16568-0250 </a> )</li><li>Ethylenediamine tetraacetic acid, disodium salt; EDTA (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S311-500 </a> )</li><li>Yeast polar lipid extract; YPL (Avanti, catalog number: <a class="blackNoUnder" target="_blank"> 190001C </a> )</li><li>Pentane (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> P399-1 </a> )</li><li>Antarctic phosphatase (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0289S </a> )</li><li>Ultima Gold scintillation cocktail (PerkinElmer, catalog number: <a class="blackNoUnder" target="_blank"> 6013329 </a> )</li><li>Argon gas, at least grade 4.8</li><li>Ultrapure water</li><li>Liquid nitrogen</li><li>Lysis Buffer (see Recipes)</li><li>Buffer A (see Recipes)</li><li>Buffer B (see Recipes)</li><li>Buffer C (see Recipes) </li><li>AnP Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32298">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>-80 Â°C freezer</li><li>Miller (Retsch, model: <a class="blackNoUnder" target="_blank"> MM400 </a> )</li><li>Centrifuge</li><li>50 ml Grinding Jars with 25 mm Grinding Balls (Retsch, catalog numbers: <a class="blackNoUnder" target="_blank"> 01.462.0216 </a> and <a class="blackNoUnder" target="_blank"> 05.368.0105 </a> , respectively)</li><li>Glass Hamilton syringe (Hamilton, catalog number: <a class="blackNoUnder" target="_blank"> 81365 </a> )</li><li>12-well filter vacuum manifold (Millipore, model: <a class="blackNoUnder" target="_blank"> 1225 </a> , catalog number: XX2702550)</li><li>Sonicator (Avanti, model: <a class="blackNoUnder" target="_blank"> G112SP1T_B </a> )</li><li>Mini extruder (Avanti, catalog number: <a class="blackNoUnder" target="_blank"> 610000 </a> )</li><li>Liquid Scintillation Counter, <em>e.g.</em>, Beckman LS 6000IC<br/></li></ol><p class="pptt" id="biaoti32299">### Software</p><ol class="npwtwok"><li>Prism 6 (GraphPad)</li></ol><p class="pptt" id="biaoti32300">### Procedure</p><ol class="npwone"><li>CST Protein Expression and Purification<ol class="npwtwo"><li>Generate Pichia pastoris stable cell lines expressing CST according to manufacturer instructions (EasySelect Pichia Expression Kit, Thermo Fisher).<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Briefly, this entails first sub-cloning CST cDNA into the pPICZ vector. This vector is linearized and then introduced into Pichia pastoris (strain SMD1168H) cells by electroporation. Successful transformants are selected on zeocin-containing growth media.</em> </li><li><em>While the bulk of our structural and biochemical characterization of CST has been with the mouse protein, we have observed that the human protein has near-identical properties. Therefore, the methods described here should be applicable to the study of either type of CST.</em> </li></ol></li><li>Express CST in <em>P. pastoris</em> according to manufacturer instructions. We typically induce expression in methanol-containing media for 20-24 h at 25 Â°C. We routinely grow 6 x 1 L cultures, yielding approximately 10 g cell paste per liter of culture.</li><li>Harvest cells by centrifugation at 4,000 <em>x g</em> for 15 min. Discard supernatant and freeze cell pellets with liquid nitrogen.</li><li>Lyse frozen cells in a mixer mill. We typically lyse 10 g of frozen cell paste per 50 ml milling chamber for 5 cycles of 3 min at 25 Hz. Chambers are cooled in liquid nitrogen between cycles.</li><li>Resuspend milled cells in 2.5 ml of lysis buffer (Recipe 1) per gram of cells and incubate for 1.5 h at 4 Â°C.</li><li>Centrifuge lysate at 35,000 <em>x g</em> for 35 min at 4 Â°C.</li><li>Collect supernatants, adjust pH to 7.2 using 5 M NaOH, and then add imidazole to 5 mM.</li><li>Equilibrate TALON resin in Buffer A (Recipe 2). Use 0.175 ml of packed resin per gram of cells.</li><li>Add pH-adjusted supernatant to TALON resin and incubate at 4 Â°C for 2 h under gentle rotation.</li><li>Wash resin in batch with 5 column volumes of Buffer A containing 5 mM imidazole. Pellet at 1,250 <em>x g</em> for 5 min and resuspend in 1 column volume of Buffer A.</li><li>Load resin onto a 1 cm diameter glass column, and wash using a peristaltic pump at approximately 1 ml/min with 5 column volumes of Buffer A containing 20 mM imidazole, followed by 2 column volumes of Buffer A containing 40 mM imidazole.</li><li>Elute protein with at least 5 column volumes of Buffer A containing 300 mM imidazole.</li><li>Pool eluted protein and add 1 mM DTT and 1 mM EDTA.</li><li>Add PreScission protease at a mass ratio of 1:20 protease:total protein, and dialyze overnight at 4 Â°C against Buffer A. Dialyze against a sufficient volume to reduce the imidazole concentration to 5 mM or lower. For example, dialyze against a volume that is at least 60 times the sample volume.</li><li>Run cleaved protein over a 1 ml TALON column equilibrated in Buffer A. Collect the initial flow-through. Wash the column twice with one column volume buffer A. Pool the flow-through and washes.<br/><em>Note: This step is necessary to remove contaminants that are not resolved from CST during size exclusion chromatography (next step).</em> </li><li>Concentrate the pooled sample to at least 500 Î¼l in a 50k MWCO concentrator and run on a 10/300 Superdex 200 Increase gel filtration column equilibrated in Buffer B (Recipe 3).</li><li>Combine peak fractions for incorporating into proteoliposomes.<br/><br/></li></ol></li><li>Proteoliposome Reconstitution<ol class="npwtwo"><li>To incorporate purified CST into lipid vesicles, the lipids must first be transformed into small unilamellar vesicles (SUVs). Since the lipid stocks are supplied dissolved in chloroform, the first step in this process involves evaporating away the chloroform before the lipids can be resuspended in an aqueous buffer.</li><li>A typical reconstitution uses 40 mg of lipids (yeast polar lipid extract). To aliquot this amount, use a glass Hamilton syringe with a Teflon plunger to measure 1.6 ml of a 25 mg/ml solution of lipids dissolved in chloroform. Place the lipid solution into a glass test tube and evaporate the chloroform using a gentle stream of argon gas. Then, re-dissolve the dried lipids with ~2 ml pentane and then evaporate the pentane using a gentle stream of argon. Repeat this pentane wash once more and then place the dried lipids in a vacuum desiccator overnight to remove any residual solvent.</li><li>Re-suspend the dried lipids in Buffer C (Recipe 4) at 11.1 mg/ml and sonicate for 2-3 h in a bath sonicator. Use a ring stand and clamp to suspend the glass test tube in the bath such that the lipid suspension is completely submerged. Run the sonicator in 5-10 min intervals to prevent overheating (above ~40 Â°C). During this time, the solution should change from opaque to translucent indicating the formation of SUVs.</li><li>Add DDM to a final concentration of 5 mM, using a 20% stock solution, to partially solubilize the SUVs. Vortex gently and then incubate for 10-15 min at room temperature.</li><li>Sonicate again for up to 1 h, in 5-10 min intervals, until the solution becomes semitransparent.</li><li>Add 200 Î¼g of purified CST from the previous section. The protein should be sufficiently concentrated using a 50k MWCO concentrator such that the addition of protein to the lipids does not dilute the lipids below 10 mg/ml. For protein-free vesicle controls, substitute CST for an equivalent volume of Buffer B. For filling the vesicles with a substrate (<em>e.g.</em>, 300 Î¼M CMP), add the desired substrate at this step.<br/><em>Note: This protein:lipid mass ratio of 1:200 was empirically determined to give the best signal to noise in our assay. Other proteins or lipid compositions may require different protein:lipid ratios.</em> </li><li>Bring the final lipid concentration to 10 mg/ml by diluting with Buffer C, if necessary, then incubate for 1 h at 4 Â°C.</li><li>To form proteoliposomes, remove DDM by adding Bio-Beads to 100 mg/ml. Incubate for 2 h at 4 Â°C under gentle rotation, then briefly pellet Bio-Beads and transfer supernatant to fresh Bio-Beads at 100 mg/ml. Incubate again for 2 h at 4 Â°C under gentle rotation, then transfer to a third fresh batch of Bio-Beads at 100 mg/ml. Incubate this overnight at 4 Â°C under gentle rotation. The finished proteoliposomes should be noticeably less translucent at the end of this incubation.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Prepare Bio-Beads by adding 30 ml methanol to ~2 g of Bio-Beads in a 50 ml Falcon tube. Rotate beads for 15 min, then remove methanol by applying a vacuum to a small hole created in the bottom of the Falcon tube using a needle. Wash the beads four times in 50 ml water, removing water by applying a vacuum. Wash the beads once in 10 ml of Buffer C, and then store under Buffer C until needed.</em> </li><li><em>To obtain the desired amount of beads, briefly drain the washed beads by vacuum and weigh out the desired amount using a balance.</em> </li></ol></li><li>Aliquot the liposomes and flash freeze in liquid nitrogen. Store at -80 Â°C.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>The liposomes can be stored at -80 Â°C for at least several months.</em> </li><li><em>We can typically incorporate about 60% of the starting amount of protein into the vesicles.</em> </li><li><em>To determine the efficiency of protein incorporation, solubilize a small amount of vesicles using DDM, and run on a gel filtration column. Then compare the peak height to an equivalent protein sample that was not incorporated into vesicles.</em> </li></ol><br/></li></ol></li><li>CMP-Sia Ligand Purification<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Commercial stocks of CMP-Sia contain approximately 10% CMP and will further hydrolyze to form free CMP and Sia over time. For transport assays, all free CMP must first be converted to cytidineâwhich has a very low affinity for CSTâusing Antarctic phosphatase (AnP).</em> </li><li><em>We screened a number of commercially-available phosphatases. Most were able to effectively convert CMP to cytidine, but we ultimately selected AnP because it retains high enzymatic activity in acidic pHs, which is the pH range where most of our crystallization hits were obtained.</em> </li><li><em>The hydrolysis of CMP-Sia is temperature and pH dependent (faster at higher temperature and/or lower pH), so keep AnP-treated CMP-Sia solutions in neutral pH buffers and on ice during use and store at -80 Â°C.</em> </li><li><em>We determined that the rate of hydrolysis of CMP-Sia in the transport assay buffer (Buffer C) is 0.005%/min at room temperature. This should be a factor to consider when designing and interpreting data from transport assay experiments. However, given that most of our transport experiments used relatively low concentrations of CMP-Sia and short incubation times, we did not consider the small amount of CMP generated to be significant.</em> </li></ol><ol class="npwtwo"><li>The Antarctic phosphatase (AnP) stock from NEB is 5,000 U/ml. This is not concentrated enough to adequately treat a concentrated (100 mM) stock of CMP-Sia; therefore, the AnP must be concentrated. To do this, first dilute the AnP to 100 U/ml in AnP Buffer (Recipe 5) (to reduce glycerol concentration), then concentrate to 10,000 U/ml with a 10k MWCO concentrator.</li><li>Dissolve CMP-Sia powder into the 10,000 U/ml AnP solution to a final concentration of 100 mM. Add ZnCl<sub>2</sub> to a final concentration of 0.2 mM and MgCl<sub>2</sub> to a final concentration of 2 mM.</li><li>Incubate this mixture for 8 h at room temperature.</li><li>Remove AnP by filtering the mixture through a 3k MWCO concentrator, which retains the 70 kDa AnP.<br/><em>Note: We confirmed that we do not observe any AnP activity after this step.</em> </li><li>If desired, confirm the elimination of CMP form the CMP-Sia stock by HPLC analysis.</li><li>Aliquot and freeze at -80 Â°C. Aliquots can be stored for at least several months.<br/><br/></li></ol></li><li>Transport Assay<ol class="npwtwo"><li>Thaw the lipid vesicles from Section B and extrude 10 times through a 0.4 Î¼m Whatman Nuclepore filter using an Avanti mini extruder.</li><li>Remove external CMP by pelleting the vesicles via ultracentrifugation at 194,800 <em>x g</em> for 60 min at 4 Â°C, using high-g rated 1.5 ml Beckman tubes. 45 Î¼l vesicles are needed per assay point.</li><li>Discard supernatant and rinse pellet twice with 500 Î¼l of Buffer C. Then resuspend the pellet in Buffer C. The final volume should be 0.444 times the starting volume; however, it is helpful to initially only add a small volume (~50 Î¼l) of Buffer C to make it easier to completely homogenize the vesicles before bringing the sample up to the final volume. Store on ice until needed.</li><li>For each assay point, initiate transport by taking 20 Î¼l of the resuspended vesicles and adding 30 Î¼l of Buffer C with the desired concentration of AnP-treated CMP-Sia containing 30-50 nM [<sup>3</sup>H]CMP-Sia (20 Ci/mmol).<br/><em>Note: We have not evaluated the purity of the [<sup>3</sup>H]CMP-Sia that we use for transport assays, but we do not anticipate the presence of any CMP impurity to be a significant problem for the following reasons: 1) the tritium is on the Sia moiety so we donât have to worry about detecting [<sup>3</sup>H]CMP transport, and free Sia has a non-detectable affinity for CST and is most likely not transported; 2) the [<sup>3</sup>H]CMP-Sia is only included as a tracer and is typically diluted at least 100x with cold CMP-Sia for the assay; therefore, the relative concentration of any CMP will be very low.</em> </li><li>Incubate the mixture for the desired time and temperature depending on the experiment. We typically incubate at 23 Â°C for convenience and to minimize CMP-Sia hydrolysis. At this temperature, the vesicles appear to reach equilibrium by 10 min, with the rate of uptake being linear up until 30 s.</li><li>Stop transport by adding 0.6 ml of ice-cold Buffer C and store on ice.</li><li>To separate the vesicles from the external solution, filter the mixture through a 0.22 Î¼m mixed cellulose ester membrane. To reduce background counts (non-specific adherence of [<sup>3</sup>H]CMP-Sia to the filters), it helps to first pre-wash the filters with 2 ml ice-cold Buffer C before applying the vesicles. After the vesicles have filtered through, wash the filters three times with 2 ml of ice-cold Buffer C. We typically use a 12-well filter vacuum manifold (MilliporeSigma) that accommodates 25 mm diameter filters.</li><li>Place filters in 7 ml scintillation vials, add 5 ml scintillation cocktail, and count in a liquid scintillation counter.<br/></li></ol></li></ol><p class="pptt" id="biaoti32301">### Data analysis</p><ol class="npwtwok"><li>To determine background counts, use values from protein-containing vesicles mixed with the hot substrate and then immediately quenched by adding 0.6 ml ice-cold Buffer C. Alternatively, protein-free vesicles that undergo the same experimental conditions as protein-containing vesicles can be used. We have found that these approaches yield similar results.</li><li>Subtract the background counts from the total counts for each sample to determine specific counts and convert to mol/min for substrate transport.</li><li>Fit data to a Michaelis-Menten model to determine K<sub>m</sub> and V<sub>max</sub> (see Figure 1 for example data).<br/><br/><img alt="" class="layerphoto" height="148" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316053132_4328.jpg" width="300"/><br/><strong>Figure 1. Functional properties of mouse CST.</strong> A. Time course of 30 Î¼M CMP-Sia uptake for purified mouse CST (mCST) reconstituted into lipid vesicles with (black) or without (red) 300 Î¼M CMP inside the vesicles. (Inset), the first minute of CMP-Sia uptake for mCST reconstituted into CMP-filled vesicles is shown to illustrate the linear relationship between uptake and time for up to 30 s, which we used to determine the initial velocity of transport for a given concentration of CMP-Sia. B. The initial velocity of CMP-Sia uptake for mCST reconstituted into CMP-filled vesicles is plotted as a function of substrate concentration to determine the K<sub>m</sub> and V<sub>max</sub> for transport. In all panels, the symbols show the mean Â± standard error of the mean (SEM) for <em>n</em> = 2 (A) or <em>n</em> = 4 (B). Adapted from Ahuja and Whorton, 2019.<br/></li></ol><p class="pptt" id="biaoti32302">### Recipes</p><ol class="npwtwok"><li>Lysis Buffer<br/>50 mM HEPES, pH 7.5<br/>150 mM NaCl<br/>0.01 mg/ml deoxyribonuclease I<br/>0.7 Î¼g/ml pepstatin<br/>1 Î¼g/ml leupeptin<br/>1 Î¼g/ml aprotinin<br/>1 mM benzamidine<br/>0.5 mM phenylmethylsulfonyl fluoride<br/>2% (w/v) DDM (solgrade)</li><li>Buffer A<br/>50 mM HEPES, pH 7.5<br/>150 mM NaCl<br/>0.1% (w/v) DDM (solgrade)</li><li>Buffer B (Gel Filtration Buffer)<br/>25 mM HEPES, pH 7.5<br/>150 mM NaCl<br/>0.1% (w/v) DDM (anagrade)<br/>5 mM DTT<br/>1 mM EDTA</li><li>Buffer C (Reconstitution Buffer)<br/>20 mM HEPES, pH 7.5<br/>0.1 M KCl</li><li>AnP Buffer<br/>50 mM Bis-Tris-Propane-HCl, pH 6.0<br/>0.1 mM ZnCl<sub>2</sub><br/>1 mM MgCl<sub>2</sub><br/></li></ol><p class="pptt" id="biaoti32303">### Acknowledgments</p><p class="ppzz">This work was supported by NIH grant R01GM130909, and by Oregon Health and Science University. This protocol has been expanded from previous work (Ahuja and Whorton, 2019).</p><p class="pptt" id="biaoti32304">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti32305">### References</p><ol class="npwtwok"><li>Ahuja, S. and Whorton, M. R. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30985278" target="_blank">Structural basis for mammalian nucleotide sugar transport.</a> <em>Elife</em> 8: e45221.</li><li>Aoki, K., Ishida, N. and Kawakita, M. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12682060" target="_blank">Substrate recognition by nucleotide sugar transporters: further characterization of substrate recognition regions by analyses of UDP-galactose/CMP-sialic acid transporter chimeras and biochemical analysis of the substrate specificity of parental and chimeric transporters.</a> <em>J Biol Chem</em> 278(25): 22887-22893.</li><li>Beau, J. M., Schauer, R., Haverkamp, J., Kamerling, J. P., Dorland, L. and Vliegenthart, J. F. (1984). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6705794" target="_blank">Chemical behaviour of cytidine 5'-monophospho-N-acetyl-beta-D-neuraminic acid under neutral and alkaline conditions.</a> <em>Eur J Biochem</em> 140(1): 203-208.</li><li>Burkart, M. D., Vincent, S. P., Duffels, A., Murray, B. W., Ley, S. V. and Wong, C. H. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11003139" target="_blank">Chemo-enzymatic synthesis of fluorinated sugar nucleotide: useful mechanistic probes for glycosyltransferases.</a> <em>Bioorg Med Chem</em> 8(8): 1937-1946.</li><li>Caffaro, C. E. and Hirschberg, C. B. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17115720" target="_blank">Nucleotide sugar transporters of the Golgi apparatus: from basic science to diseases.</a> <em>Acc Chem Res</em> 39(11): 805-812.</li><li>Caffaro, C. E., Koshy, A. A., Liu, L., Zeiner, G. M., Hirschberg, C. B. and Boothroyd, J. C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23658519" target="_blank">A nucleotide sugar transporter involved in glycosylation of the toxoplasma tissue cyst wall is required for efficient persistence of bradyzoites.</a> <em>PLoS Pathog</em> 9(5): e1003331.</li><li>Capasso, J. M. and Hirschberg, C. B. (1984). <a href="http://www.ncbi.nlm.nih.gov/pubmed/6095266" target="_blank">Mechanisms of glycosylation and sulfation in the Golgi apparatus: evidence for nucleotide sugar/nucleoside monophosphate and nucleotide sulfate/nucleoside monophosphate antiports in the Golgi apparatus membrane.</a> <em>Proc Natl Acad Sci U S A</em> 81(22): 7051-7055.</li><li>Descoteaux, A., Luo, Y., Turco, S. J. and Beverley, S. M. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7569927" target="_blank">A specialized pathway affecting virulence glycoconjugates of <em>Leishmania</em>.</a> <em>Science</em> 269(5232): 1869-1872.</li><li>Dwek, R. A., Butters, T. D., Platt, F. M. and Zitzmann, N. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12119611" target="_blank">Targeting glycosylation as a therapeutic approach.</a> <em>Nat Rev Drug Discov</em> 1(1): 65-75.</li><li>Engel, J., Schmalhorst, P. S., Dork-Bousset, T., FerriÃ¨res, V. and Routier, F. H. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19840949" target="_blank">A single UDP-galactofuranose transporter is required for galactofuranosylation in <em>Aspergillus fumigatus</em>.</a> <em>J Biol Chem</em> 284(49): 33859-33868.</li><li>Esko, J. D. and Bertozzi, C. R. (2009). Chap. 50: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20301258" target="_blank">Chemical tools for inhibiting glycosylation.</a> In: Varki A., Cummings R. D., Esko J. D., Freeze H. H., Stanley P., Bertozzi C. R., Hart G. W. and Etzler M. E. (Eds.). In: <em>Essentials of Glycobiology</em>. 2nd edition. Cold Spring Harbor (NY): Cold Spring Harbor Laboratory Press.</li><li>Hadley, B., Maggioni, A., Ashikov, A., Day, C. J., Haselhorst, T. and Tiralongo, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25210595" target="_blank">Structure and function of nucleotide sugar transporters: Current progress.</a> <em>Comput Struct Biotechnol J</em> 10(16): 23-32.</li><li>Hirschberg, C. B., Robbins, P. W. and Abeijon, C. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9759482" target="_blank">Transporters of nucleotide sugars, ATP, and nucleotide sulfate in the endoplasmic reticulum and Golgi apparatus.</a> <em>Annu Rev Biochem</em> 67: 49-69.</li><li>Horenstein, B. A., Bruner, M. (1996). <a href="https://pubs.acs.org/doi/10.1021/ja961811z" target="_blank">Acid-Catalyzed Solvolysis of CMP-N-Acetyl Neuraminate: Evidence for a Sialyl Cation with a Finite Lifetime.</a> <em>J Am Chem Soc</em> 118(43): 10371-10379.</li><li>Hong, K., Ma, D., Beverley, S. M. and Turco, S. J. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10684651" target="_blank">The Leishmania GDP-mannose transporter is an autonomous, multi-specific, hexameric complex of LPG2 subunits.</a> <em>Biochemistry</em> 39(8): 2013-2022.</li><li>Jaeken, J. and Matthijs, G. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17506657" target="_blank">Congenital disorders of glycosylation: a rapidly expanding disease family.</a> <em>Annu Rev Genomics Hum Genet</em> 8: 261-278.</li><li>Kajihara, Y., Nishigaki, S., Hanzawa, D., Nakanishi, G., Okamoto, R. and Yamamoto, N. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21598321" target="_blank">Unique self-anhydride formation in the degradation of cytidine-5'-monophosphosialic acid (CMP-Neu5Ac) and cytidine-5'-diphosphosialic acid (CDP-Neu5Ac) and its application in CMP-sialic acid analogue synthesis.</a> <em>Chemistry</em> 17(27): 7645-7655.</li><li>Liu, L., Xu, Y. X., Caradonna, K. L., Kruzel, E. K., Burleigh, B. A., Bangs, J. D. and Hirschberg, C. B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23443657" target="_blank">Inhibition of nucleotide sugar transport in <em>Trypanosoma brucei</em> alters surface glycosylation.</a> <em>J Biol Chem</em> 288(15): 10599-10615.</li><li>Ma, D., Russell, D. G., Beverley, S. M. and Turco, S. J. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9013638" target="_blank">Golgi GDP-mannose uptake requires Leishmania LPG2. A member of a eukaryotic family of putative nucleotide-sugar transporters.</a> <em>J Biol Chem</em> 272(6): 3799-3805.</li><li>Milla, M. E. and Hirschberg, C. B. (1989). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2928302" target="_blank">Reconstitution of Golgi vesicle CMP-sialic acid and adenosine 3'-phosphate 5'-phosphosulfate transport into proteoliposomes.</a> <em>Proc Natl Acad Sci U S A</em> 86(6): 1786-1790.</li><li>Moremen, K. W., Tiemeyer, M. and Nairn, A. V. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22722607" target="_blank">Vertebrate protein glycosylation: diversity, synthesis and function.</a> <em>Nat Rev Mol Cell Biol</em> 13(7): 448-462.</li><li>Ohtsubo, K. and Marth, J. D. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16959566" target="_blank">Glycosylation in cellular mechanisms of health and disease.</a> <em>Cell</em> 126(5): 855-867.</li><li>Song, Z. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23506892" target="_blank">Roles of the nucleotide sugar transporters (SLC35 family) in health and disease.</a> <em>Mol Aspects Med</em> 34(2-3): 590-600.</li><li>Stanley, P. (2011). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3062213/" target="_blank">Golgi glycosylation.</a> <em>Cold Spring Harb Perspect Biol</em> 3(4): a005199.</li><li>Stowell, S. R., Ju, T. and Cummings, R. D. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25621663" target="_blank">Protein glycosylation in cancer.</a> <em>Annu Rev Pathol</em> 10: 473-510.</li><li>Tiralongo, J., Ashikov, A., Routier, F., Eckhardt, M., Bakker, H., Gerardy-Schahn, R. and von Itzstein, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16118285" target="_blank">Functional expression of the CMP-sialic acid transporter in <em>Escherichia coli</em> and its identification as a simple mobile carrier.</a> <em>Glycobiology</em> 16(1): 73-81.</li><li>Waldman, B. C. and Rudnick, G. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2322548" target="_blank">UDP-GlcNAc transport across the Golgi membrane: electroneutral exchange for dianionic UMP.</a> <em>Biochemistry</em> 29(1): 44-52.</li><li>Wang, L., Liu, Y., Wu, L. and Sun, X. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26192491" target="_blank">Sialyltransferase inhibition and recent advances.</a> <em>Biochim Biophys Acta</em> 1864(1): 143-153.</li><li>Watts, A.G. and Withers, S.G. (2004). <a href="https://www.chem.ubc.ca/synthesis-some-mechanistic-probes-sialic-acid-processing-enzymes-and-labeling-sialidase-trypanosoma" target="_blank">The synthesis of some mechanistic probes for sialic acid processing enzymes and the labeling of a sialidase from <em>Trypanosoma rangeli</em>.</a> <em>Can J Chem</em> 82: 1581-1588.<br/></li></ol>