<body>
            <h1>Single-cell qPCR Assay with Massively Parallel Microfluidic System</h1>
            <p><strong>Authors:</strong> Marta Prieto-Vila, Takahiro Ochiya, Yusuke  Yamamoto</p>
            <h2>Abstract</h2>
            <p>The single-cell transcriptome is the set of messenger RNA molecules expressed in one cell. It is extremely variable and changes according to external, physical and biochemical conditions. Due to sensitivity shortages, most of genetic studies use bulk samples, providing only the average gene expression. Single-cell technologies have provided a powerful approach to a more detailed understanding of the heterogenic populations and minority cells. However, since it is still a quite novel technique, standardized protocol has to be established. Single-cell qPCR, although partly limited by the number of genes, is relatively simple to analyze. Therefore, its use is accessible without the necessity to recourse to complex bioinformatics analyses. The main steps for single-cell qPCR, as illustrated in this protocol, are composed by single-cell isolation, cell lysate, cDNA reverse-transcription synthesis, amplification for cDNA library generation, and finally, quantitative polymerase chain reaction.</p>
            <p><strong>Keywords:</strong> Single-cell, Transcriptomics, Single-cell qPCR, mRNA, Microfluidic system, Cell heterogeneity</p>
            <h2>Background</h2><p>The single-cell transcriptome is the complete set of messenger RNA (mRNA) molecules expressed in one cell. It is extremely variable and changes according to external, physical and biochemical conditions. Hence, it is the source of the biological heterogeneity, in which the cells from the same environment are similar -but not identical- to the other ones.<br/> <br/>Due to sensitivity shortages, most of genetic studies use bulk samples, with hundreds to thousands of cells, providing only the average gene expression. This vastly limits the study of minor populations of cells, which may poses specific properties, such as drug resistance or metastatic ability in the case of cancer. Single-cell technologies enable the analysis of the transcriptome at single cell level, unraveling the complexity of heterogenic populations. These technologies have been applied not only in cancer but in many cell biology studies, including adult tissues, stem cells, immune cells; as well as other fields such as microbiology and virology (Wen and Tang, 2016; Karaiskos <em>et al.</em>, 2017; Rato <em>et al.</em>, 2017; Woyke <em>et al.</em>, 2017; Zheng <em>et al.</em>, 2017).<br/> <br/>There are two main single-cell transcriptome approaches: mRNA sequencing and quantitative polymerase chain reaction. Although mRNA sequencing enables the study of the whole transcriptome without prior knowledge, manipulating an extremely large amount of genes can be overwhelming for novel researchers in the field; moreover high levels of bioinformatics are required (Prieto-vila <em>et al.</em>, 2018). To counterbalance, single-cell qPCR analysis is limited to a certain number of genes, but can be analyzed as normal qPCR, being more approachable.<br/> <br/>In our previous research, we aimed to study a drug resistant subpopulation of cells by using single-cell qPCR in breast cancer cell lines. With that purpose, we used massively parallel single-cell amplification, that allowed us the study of 96 genes at single cell level to 96 cells per run (Prieto-Vila <em>et al.</em>, 2019). This is one of the most commonly used techniques for single-cell qPCR in which reactions are carried in specific 96-well plates, where individual samples are reverse transcribed at the same time. The C1 nanofluidics along with the Fluidigm systems, are the most commonly used systems for sc-qPCR. C1 machine is a hydrodynamic cell trap chip, formed by a net of channels where the cells are eluted and single cells are fiscally separated their size. Thanks to this system, rarely doublets are found. The 96 single cells are then eluded into 96 individual wells. On these wells lysis, reverse transcription, and pre-amplification for qPCR are carried in parallel using a very small reaction volume (Ziegenhain <em>et al.</em>, 2017). Following, massive parallel qPCR is done by analyzing the expression of 96 genes in the 96 individual cells in a matrix-manner chip by the Fluidigm system.<br/> <br/>In the present protocol, we provide a single-cell transcriptome analysis protocol, including experimental details to perform single-cell qPCR with the Fluidigm systems.</p>
        </body>

<p class="pptt" id="biaoti32418">## Materials and Reagents</p><ol class="npwtwo"><li>Pipette tips 1,000 Î¼l/200 Î¼l/20 Î¼l (Sorenson Bioscience, catalog numbers: <a class="blackNoUnder" target="_blank"> 34000 </a> /<a class="blackNoUnder" target="_blank"> 14220 </a> /<a class="blackNoUnder" target="_blank"> 15020 </a> )</li><li>Low binding centrifuge tubes 1.5 ml (Watson BioLab, catalog number: <a class="blackNoUnder" target="_blank"> pk-15 c-500 </a> )</li><li>Rainin LTS tips 10 Î¼l (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 30389228 </a> )</li><li>qPCR 96-well plates (Applied Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> 4346907 </a> )</li><li>Film for qPCR plates (Applied Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> 4311971 </a> )</li><li>Nuclease-Free water (Ambion, catalog number: <a class="blackNoUnder" target="_blank"> AM9932 </a> )</li></ol><br/><strong><em>For cell culture</em></strong> <ol class="npwtwo"><li>Cell culture 100 mm dishes (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 172931 </a> )</li><li>15 ml conical centrifuge tubes (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 339650 </a> )</li><li>Desired cell line, in this case we used the breast cancer cell lines MDA-MB-231 and MCF7</li><li>RPMI 1640x basic medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> C11875500BT </a> )</li><li>Fetal bovine serum (FBS) (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 10270-106 </a> )<br/><em>Note: FBS should be heat inactivated at 56 Â°C for 30 min before use.</em> </li><li>Antibiotic/antimitotic solution (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15240062 </a> )</li><li>Phosphate buffer Saline (PBS) (DS Pharma Biomedical, catalog number: <a class="blackNoUnder" target="_blank"> DSBN200 </a> )</li><li>TrypLE Express (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12604013 </a> )</li><li>Trypan Blue Solution 0.4% (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 15250-061 </a> )</li><li>Neubauer Improved C-Chip (NanoEnTek, catalog number: <a class="blackNoUnder" target="_blank"> DHC-N01 </a> )</li><li>0.5 mol/L EDTA (Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 06894-14 </a> )</li><li>Initial cell suspension buffer (see Recipes)</li></ol><br/><strong><em>For single cell isolation and pre-amplification</em></strong><br/><ol class="npwtwo"><li>C1 Single-Cell Auto Prep IFC for Preamp (10-17 Î¼m) (Fluidigm, catalog number: <a class="blackNoUnder" target="_blank"> 100-5749 </a> )<br/><em>Note: There are several channel size plates to adjust to the cell size. See notes for more information.</em> </li><li>20x gene specific Assays (Taqman)</li><li>C1 Single-Cell Auto Prep Reagent Kit (Fluidigm, catalog number: <a class="blackNoUnder" target="_blank"> 100-5319 </a> )</li><li>Ambion single cell-to-Ct qRT-PCR Kit (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 4458237 </a> )</li><li>Live/Dead Kit for mammalian cells (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> L-3224 </a> ) (see Recipes)</li><li>Lysis mix (see Recipes)</li><li>Reverse transcription final mix (see Recipes)</li><li>0.2x Pooled Taqman primers (see Recipes)</li><li>Pre-amplification mix (see Recipes)</li></ol><br/><strong><em>For single cell qPCR</em></strong> <ol class="npwtwo"><li>Fluidigm 96.96 quantitative PCR Dynamic Array microfluidic chips (Fluidigm, catalog number: <a class="blackNoUnder" target="_blank"> BMK-M-96.96GT </a> )</li><li>Taqman Universal PCR Master Mix 2x (Applied Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> PN 4304437 </a> ) </li><li>20x GE Sample Loading Reagent (Fluidigm, catalog number: <a class="blackNoUnder" target="_blank"> PN 100-7610 </a> )<br/></li><li>2x Assay Loading Reagent (Fluidigm, catalog number: <a class="blackNoUnder" target="_blank"> PN 100-7611 </a> )</li><li>10x Assays for single-cell qPCR (see Recipes)</li><li>Sample pre-mix (see Recipes)</li></ol><p class="pptt" id="biaoti32419">### Equipment</p><ol class="npwtwok"><li>Neubauer chamber</li><li>Micropipettes (P2, P10, P20, P200, P1000)</li><li>Multichannel pipette P10 with high precision (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17013802 </a> )</li><li>Cell culture incubator: 37 Â°C and 5% CO<sub>2</sub> (Panasonic Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> MCO-170AICUVH-PJ </a> )</li><li>Clean bench (Panasonic, catalog number: <a class="blackNoUnder" target="_blank"> MCV-131BNF-PJ </a> )</li><li>Centrifuge for 1.5 ml tubes (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5418R </a> )</li><li>Centrifuge for 15 ml tubes (Tomy, model: <a class="blackNoUnder" target="_blank"> Ax-511 </a> )</li><li>Centrifuge for 96-well plate (Sigma, model: <a class="blackNoUnder" target="_blank"> 4-16KS </a> )</li><li>Vortex (Scientific Industries, model: <a class="blackNoUnder" target="_blank"> Vortex-Genie 2 </a> )</li><li>Fluorescent confocal microscope (Keyence, model: <a class="blackNoUnder" target="_blank"> BZ-X700 </a> )</li><li>Refrigerator and Freezer (any companyâs product should be fine)</li><li>C1 Auto Prep System (Fluidigm)</li><li>IFC Controler HX (Fluidigm)</li><li>BioMarkHD system for real-time qPCR (Fluidigm)<br/></li></ol><p class="pptt" id="biaoti32420">### Software</p><ol class="npwtwok"><li>Fluidigm Real-Time PCR Analysis Software v.2.1.3</li><li>Fluidigm Data Collection Software v.2.1.3</li><li>Excel</li><li>R (version 3.3.2)<br/></li></ol><p class="pptt" id="biaoti32421">### Procedure</p><p class="ppzz">Single-cell qPCR protocol can be mainly divided into 4 fractions: The first one comprises cell culture and cell suspension preparation. The second and third are mainly done by the C1 integrated fluidic circuits, which allows single cell separation, cell lysis and pre-amplification. The final fraction is the single real time qPCR, which is done by the BioMarkHD system. Those steps are represented in Figure 1.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316200722_7286.jpg" width="371"/><br/><strong>Figure 1. Scheme of single-cell qPCR protocol<br/><br/></strong> </p><ol class="npwone"><li>Sample preparation<br/>This protocol commences with cell culture already ongoing.<ol class="npwtwo"><li>When the cells reach 70% of confluence are ready to collect.</li><li>Remove the medium and wash the dish with PBS once.</li><li>Add 1 ml of TrypLE Express.</li><li>Place the dishes in the 37 Â°C incubator, and incubate for 3-5 min until the cells detach.</li><li>Add 4 ml of pre-warmed complete medium at 37 Â°C to stop the reaction.</li><li>Pipette delicately to ensure the complete detach of all the cells but without damaging the cells.</li><li>Transfer the cells into 15 ml tube.</li><li>Centrifuge at 300 <em>x g</em> 4 Â°C for 5 min.</li><li>Aspirate the medium and add 5 ml of PBS.</li><li>Take apart 100 Î¼l to count the cells.</li><li>Centrifuge the 15 ml tube at 300 <em>x g</em> 4 Â°C for 5 min.</li><li>While centrifuging, mix the separated cells at a proportion of 1:1 with Trypan blue solution.</li><li>Insert 10 Î¼l of the mixt into Neubauer chamber.</li><li>Count the total cell number with the microscope (including dead cells).</li><li>Remove PBS.</li><li>Re-suspend the cells at 300 cells/Î¼l (3 x 10<sup>5</sup> cells/ml) in Initial cell suspension buffer (Recipe 1) and keep on ice.<br/><em>Note: An amount of at least 10,000 cells are recommended to obtain a complete capture, we observed that less than that amount leaves multiple channels with cells unattached in the IFC plate.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Single-cell separation<ol class="npwtwo"><li>Take all the necessary reagents from -20 Â°C, where are stocked, and let them thaw on ice for 30 min before starting the experiment. Just before starting, take the reagents kept at 4 Â°C and put them on ice during its use.<br/><em>Caution: The plate must not be placed on ice.</em> </li><li>Remove the black plastic situated at the button of the integrated fluidic circuits (IFC) plate.</li><li>Before starting, turn on the C1 machine since it takes some time to start up (about 3 min).</li><li>Pipette several reagents following this scheme:<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316200756_8735.jpg" width="138"/><br/><strong>Figure 2. Representation of IFC plate indicating the wells to add the different reagents and solutions.</strong> Same number indicates same solution and volume. The wells covered in grey, represent the section of the chip that is covered by a film of paper, which at the end of the experiment will contain, each single-cell sample pre-amplified ready to use in sc-qPCR.<br/>â .Add 200 Î¼l of Harvest reagent. This well is like a button with a plastic lid on an iron spindle. To apply the reagent, press slightly with the tip, and once slips down, put the tip on the space. Then add slowly the reagent. As can be seen in Figure 3.<br/>â¡.Add 20 Î¼l of Harvest reagent.<br/>â¢.Pipette 15 Î¼l of Blocking reagent.<br/>â£.Add 20 Î¼l of Preloading reagent.<br/>â¤.Add 20 Î¼l of Wash Buffer.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316200833_7492.jpg" width="207"/><br/><strong>Figure 3. Scheme of Harvest reagent (200 Î¼l) application in the IFC plate</strong><br/><br/></li><li>Insert the plate in the IFC machine with the barcode facing outside.<br/><em>Note: If the bottom of the plate is dirty, the machine will not detect it and will give an error. If so, wipe the bottom with ethanol 70% and place it again.</em> </li><li>Run the mode STA:Prime (1782x/1783x/1784x) â according to the selected size of channels of the chip. This procedure takes 10-12 min.<br/>While priming, prepare the cell mixture.</li><li>Prepare the cell mixture: 30 Î¼l of cell suspension and 20 Î¼l of suspension reagent.<br/><em>Note: Less amount of cell mix can be prepared as long as the ratio is maintained. For instance, 6 Î¼l and 4 Î¼l.</em> </li><li>After priming is finished, click eject and take out the plate.</li><li>Remove the leftovers of C1 blocking reagent with a P200 pipette (wells are indicated in Figure 2-â¢).</li><li>Insert the cells and Live/Dead staining solutions, as follows:<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316200909_7941.jpg" width="169"/><br/><strong>Figure 4. Representation of IFC plate indicating the wells to add the different reagents and solutions</strong><br/>â .Add 6 Î¼l of cell mixture.<br/><em>Note: Do not vortex the cells. Pipette them before the addition.</em><br/>â¡.Add 20 Î¼l of Live/Dead staining solution (Recipe 2).<br/><br/></li><li>Place the IFC plate into the C1 system.</li><li>Select STA: Cell Load&amp;Stain (1782x/1783x/1784x)âaccording to the selected size of channels of the chip. It takes 30-65 min, depending on the cell size of the plate.</li><li>Meanwhile, prepare the Lysis Mix, RT Mix and PreAmp Mix (Recipes 3, 4 and 5).</li><li>When Loading is complete, withdraw the plate from the C1 and take pictures of the cells with Keyence microscope. Using the Phase contrast at 10x, channels and fluorescence-stained cells can be easily observed on the central part of the plate, colored in grey color in Figure 4. Also each well number can also be observed.</li><li>Write down the channels with dead cells, empty channels and channels with more than one cell to be discarded <em>a posteriori</em> (Figure 5).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316200943_2598.jpg" width="177"/><br/><strong>Figure 5. Example of Live/Dead staining.</strong> Green cells are living cells while read are dead, or highly damaged cells. Moreover, right chamber shows a doublet, and left one shows a clear single cell.<br/><br/></li></ol></li><li>Cell lysis, reverse transcription and pre-amplification.<ol class="npwtwo"><li>Add the reagents following the scheme in Figure 6:<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316201115_8747.jpg" width="136"/><br/><strong>Figure 6. Representation of IFC plate indicating the wells to add the different reagents and solutions</strong><br/>â .Add 180 Î¼l of Harvest reagent. Carefully move the plate from side-to-side to ensure that the reagent covers all the surface.<br/>â¡.Add 7 Î¼l of Lysis Final Fix.<br/>â¢.Add 7 Î¼l of RT Final mix.<br/>â£.Add 24 Î¼l of Pre-amplification Mix (Recipe 6).<br/><br/></li><li>Place the plate into the C1 system.</li><li>Select STA: PreAmp (1782x/1783x/1784x)âaccording to the selected size of channels of the chip- and choose the finish hour. We usually leave it overnight and collect it next morning to proceed with the experiments. The process itself takes 5-6 h approximately.</li><li>When STA: Preamp process has finished, select EJECT and remove the IFC plate.</li><li>Using a new PCR 96-well plate, add 25 Î¼l of C1 dilution reagent in each well.</li><li>Remove the papers than protect the output wells on IFC plate (those covers are represented in grey color in Figures 2, 4 and 6).</li><li>Using an eight-channel micropipette (Rainin), transfer the samples onto the 96-well plate.<br/><em>Note: The amount of sample should be close to 3 Î¼l, but to ensure, please set the micropipette at 4 Î¼l.</em> </li><li>Transfer the samples using this specific order, described in Figure 7.<br/><em>Note: Using 8-channel micropipette, and taking in consideration that IFC plate has 16 well per lane, it has the specific size to pic up every-other well.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316201150_4750.jpg" width="330"/><br/><strong>Figure 7. Scheme to illustrate the order of well transfer from IFC plate to PCR 96-well plate</strong><br/><br/></li><li>Vortex the 96-well plate and spin down. At this point, the samples can be frozen at -20 Â°C and be kept for few weeks before use, in that case, seal the plate with a PCR-plate film.<br/><br/></li></ol></li><li>Single-cell qPCR<ol class="npwtwo"><li>Take all the necessary reagents from -20 Â°C, where are stocked, and let them thaw on ice for 30 min before starting the experiment. Just before starting, take the reagents kept at 4 Â°C and put them on ice during its use.<br/><em>Caution: The plate must not be placed on ice.</em> </li><li>Turn on the Standard 96-well Thermal Cycler, because it takes some time to start up.</li><li>Remove the blue protective film from the bottom of the 96.96 chip.</li><li>Using the syringe from the kit, inject the control line fluid in the two big central wells, pressing between the cross (The cross can be seen, marked with an arrow in Figure 8).<br/><em>Note: The oil leaks easily, so be careful when managing the syringe.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316201718_5466.jpg" width="158"/><br/><strong>Figure 8. Schematic representation of the 96.96 quantitative PCR Dynamic Array microfluidic chip.</strong> The arrow shows the space in where the line fluid should be applied.<br/><br/></li><li>Insert the plate (with the barcode facing the outside).</li><li>Select the option Prime(136x). This process takes approximately 20 min.</li><li>Meanwhile, prepare the Pre-mix sample (Recipe 8).</li><li>In a new PCR 96-well plate, add 3.3 Î¼l of sample pre-mix in each well.</li><li>Add 2.7 Î¼l of cDNA sample with a multichannel micropipette (Sample obtained at Step C9) making a total volume of 6 Î¼l in each inlet.</li><li>Seal with PCR-plate film.</li><li>Prepare the Taqman assays Mix (Recipe 7) and seal the plate with PCR-plate film.</li><li>Vortex both plates for 10 sec and spin down.</li><li>When priming is complete, collect the chip.</li><li>Transfer the samples and the Taqman assays. In the following order (Figures 9 and 10):<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316202038_5921.jpg" width="144"/><br/><strong>Figure 9. Scheme of 96.96 chip showing in yellow color the inlets to insert Taqman assays and, in green, the inlets to insert sample premix</strong><br/>â .Add 5 Î¼l of Taqman assays in the yellow inlets, using the following order (Figure 10A) to transfer from qPCR 96-well plate to IFC chip.<br/>â¡.Add 5 Î¼l of Sample Premix, using the same order (Figure 10A) in the green inlets.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>The inlet wall is not horizontal, but has a diagonal angle. To ensure better application; the best option is insert the tip inclined, as is illustrated in Figure 10B.</em> </li><li><em>Avoid bubbles when pipetting. However, if there are bubbles but they move, means that they are on the surface, and then they will not interfere in the analysis. If bubbles do not move, with a micropipette tip remove the bubble.</em> </li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200316/20200316202121_3134.jpg" width="240"/><br/><strong>Figure 10. Scheme of order and methodology to transfer samples into 96.96 chip.</strong> A. Scheme to illustrate the order of well transfer from PCR plate to 96.96 chip. B. Scheme of sample application in 96.96 inlets.<br/><br/></li><li>Insert the chip into the IFC Controller HX and run the script Load Mix (136x).</li><li>While running the Load script, turn on the computer connected to the BioMarkHD system for real-time qPCR.</li><li>Run the software Biomark data collection.<br/>Double click to turn on the option âlampâ to start up. This process takes approximately 20-30 min.</li><li>When the Loading finish, bring the chip to BioMarkHD system and follow these steps:<br/><ol class="npwthree"><li>Click âStart a new runâ.</li><li>Insert the chip with the bar code facing outside.<br/>Select: Load<br/><strong>     â</strong>Next</li><li>Select the settings of the 96.96 chip.<br/><ol class="npwfour"><li>Choose the name and the folder where you want to save the document.<br/><strong>â</strong>Next</li><li>Application type: Single probe.</li><li>Probes: FAM_MGB<br/><strong>â</strong>Next</li><li>Protocol: GE 96x96 Standard v1.<br/><strong>â</strong>Next</li></ol></li><li>Start the run.</li></ol></li><li>Once it finishes remove the plate, close the software and shut down the computer.</li></ol></li></ol><p class="pptt" id="biaoti32422">### Data analysis</p><p class="ppzz">The data analysis procedure refers to our previous report (Prieto-Vila <em>et al.</em>, 2019).</p><ol class="npwone"><li>Remove dead/damaged cells<ol class="npwtwo"><li>After running the qPCR, collect the raw data to be analyzed with the software Biomark.</li><li>Use the settings Linear (derivative) and User (detectors) settings to generate Ct values for each gene.</li><li>Remove all values with confidence lower than 0.2.</li><li>Export the data to excel file.</li><li>Using the data obtained in Step B14 from the protocol (pictures of Live/Dead staining), remove all the cells that were stained red (dead), chambers with more than one cell, or none.</li><li>Remove cells whose housekeeping genes have Ct values lower than 16. In our case, we used both BACTIN and GAPDH.<br/><br/></li></ol></li><li>Data normalization<br/>Normalization can be conducted by two systems:<ol class="npwtwo"><li>With the objective to remove the variation between assays, normalization can be done using a housekeeping gene; in our case, we utilized GAPDH. The normalization was calculated by the formula: 2<sup>-ÎÎCT</sup>.</li><li>Another broadly used methodology in single-cell analysis is the normalization by log<sub>2</sub>Ex, which expresses the transcript levels above the background in log base 2 (Livak <em>et al.</em>, 2013). In our case, we set the LOD (limit of detection) to 26, as it was the average of the lowest detectable Ct value for each gene. This can calculated by the formula: Log<sub>2</sub>Ex = LOD (Limit Of Detection) C<sub>q</sub> â C<sub>q</sub> [Gene].</li></ol><em>Notes:</em> <ol class="npwthree"><li><em>Both normalizations can be done with Excel.</em> </li><li><em>For a more detailed protocol containing examples, please refer to this manual: âSingular: Analysis Toolsetâ (P/N 100-5066) </em><a href="https://www.fluidigm.com/binaries/content/assets/fluidigm/singular-analysis-toolset-3.5/singular-analysis-toolset-3.5-userguide.pdf" target="_blank"><em>https://www.fluidigm.com/binaries/content/assets/fluidigm/singular-analysis-toolset-3.5/singular-analysis-toolset-3.5-userguide.pdf</em></a><em>.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Several statistical analysis can be done for single-cell qPCR<br/>Probably the most common statistical analysis for single-cell are t-SNE Plot, Heatmap and ViolinPlots.<br/>For Heatmap we used the software Partek Genomics Suite Software; while both t-SNE and Violin Plot were done by the free software R using the package âggplot2â.</li></ol><p class="pptt" id="biaoti32423">### Notes</p><ol class="npwtwok"><li>For single-cell analyses, it is considered that rather than do replicates, which is also physically impossible, is more important to increase the sample number. Thus, we ran each sample twice. Analyzing a total of 192 individual cells per sample.</li><li>There are three sizes of C1 IFC plates to adjust the channel to the different sizes of cells. Prior to choose the size of the plate, we calculated the cell size by taking pictures of the cells after being detached, and measuring the size by Keyence software. The available sizes are: S (5-10 Î¼m), M (10-17 Î¼m), L (17-25 Î¼m). As a reference, we used M size for MDA-MM-231 and MCF7 cells; and we have run hepatocytes in L size plate (Katsuda <em>et al.</em>, 2019).</li><li>One of the most important steps is single-cell isolation. During this process, cells are isolated from their local environment, and thus, keeping those cells alive and in good condition for proper further analyses is indispensable (Van Den Brink <em>et al.</em>, 2017). Please manage them carefully.</li><li>This protocol has been done following the Fluidigm system manufactureâs protocols (PN-100-6177 J1 and PN 68000130 E1), which can be found at: <a href="https://www.fluidigm.com/binaries/content/documents/fluidigm/resources/c1-taqman-pr-100%E2%80%906117/c1-taqman-pr-100%E2%80%906117/fluidigm%3Afile" target="_blank">https://www.fluidigm.com/binaries/content/documents/fluidigm/resources/c1-taqman-pr-100%E2%80%906117/c1-taqman-pr-100%E2%80%906117/fluidigm%3Afile</a> and <a href="https://www.fluidigm.com/binaries/content/documents/fluidigm/resources/96.96-ge-taqman%E2%80%90std-qr-68000130/96.96-ge-taqman%E2%80%90std-qr-68000130/fluidigm%3Afile" target="_blank">https://www.fluidigm.com/binaries/content/documents/fluidigm/resources/96.96-ge-taqman%E2%80%90std-qr-68000130/96.96-ge-taqman%E2%80%90std-qr-68000130/fluidigm%3Afile</a> respectively.<br/></li></ol><p class="pptt" id="biaoti32424">### Recipes</p><p class="ppzz"><em>Note: All the recipes, unless otherwise stated, are the reagent necessary for one run.</em> </p><ol class="npwtwo"><li>Initial cell suspension buffer<br/>ComponentsVolume<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:550px;"><tbody><tr><td><strong>Components</strong> </td><td><strong>Volume</strong> </td></tr><tr><td>EDTA (5M) <br/></td><td>0.1 ml<br/></td></tr><tr><td>FBS <br/></td><td>50 Î¼l<br/></td></tr><tr><td>PBS<br/></td><td>10 ml<br/></td></tr></tbody></table>*This solution can be used multiple times and should be stored at 4 Â°C.</li><li>LIVE/DEAD Cell Staining solution (Total 1253.13 Î¼l)<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:550px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume</strong><br/></td></tr><tr><td>Cell Wash Buffer (C1 Single-Cell Auto Prep Reagent Kit)<br/></td><td>1,250 Î¼l<br/></td></tr><tr><td>Ethidium homodimer-1 (Live/Dead kit)<br/></td><td>2.5 Î¼l<br/></td></tr><tr><td>Calcein AM (Live/Dead kit)<br/></td><td>0.63 Î¼l<br/></td></tr></tbody></table></li><li>Lysis Mix (Total 18.1 Î¼l)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume</strong><br/></td></tr><tr><td>Single Cell Lysis Solution (Ambion single cell-to-Ct qRT-PCR Kit)<br/></td><td>12.75 Î¼l<br/></td></tr><tr><td>C1 Lysis Plus Reagent (C1 Single-Cell Auto Prep Reagent Kit)<br/></td><td>4.35 Î¼l<br/></td></tr><tr><td>C1 DNA Dilution Reagent (C1 Single-Cell Auto Prep Reagent Kit)<br/></td><td>1 Î¼l<br/></td></tr></tbody></table>*Vortex thoroughly</li><li>Reverse transcription Final Mix (Total 12.4 Î¼l)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume</strong><br/></td></tr><tr><td>Stop Solution (Ambion single cell-to-Ct qRT-PCR Kit)<br/></td><td>1.94 Î¼l<br/></td></tr><tr><td>Single cell VILO RT Mix (Ambion single cell-to-Ct qRT-PCR Kit)<br/></td><td>5.84 Î¼l<br/></td></tr><tr><td>Single Cell SuperScript RT (Ambion single cell-to-Ct qRT-PCR Kit)<br/></td><td>3.62 Î¼l<br/></td></tr><tr><td>Loading reagent (C1 Single-Cell Auto Prep Reagent Kit)<br/></td><td>1 Î¼l<br/></td></tr></tbody></table>*Vortex thoroughly</li><li>0.2x Pooled Taqman Primers (Total 100 Î¼l)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume</strong><br/></td></tr><tr><td>1 Î¼l of each Taqman Probe (x96 probes)<br/></td><td>1 Î¼l each<br/></td></tr><tr><td>C1 DNA Dilution Reagent (C1 Single-Cell Auto Prep Reagent Kit)<br/></td><td>4 Î¼l<br/></td></tr></tbody></table>*Vortex thoroughly<br/>**This reagent can be made in advance and aliquots stocked up to 6 months at -20 Â°C.</li><li>Pre-amplification Mix (Total 60 Î¼M)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume</strong><br/></td></tr><tr><td>Single-Cell PreAm Mix (Ambion single cell-to-Ct qRT-PCR Kit)<br/></td><td>12 Î¼l<br/></td></tr><tr><td>Loading Reagent (C1 Single-Cell Auto Prep Reagent Kit)<br/></td><td>3 Î¼l<br/></td></tr><tr><td>Pooled primers (The mix previously prepared)<br/></td><td>15 Î¼l<br/></td></tr><tr><td>DNA-free water<br/></td><td>30 Î¼l<br/></td></tr></tbody></table>*Vortex thoroughly</li><li>10x Assays for single-cell qPCR<br/>In a qPCR 96-well plate, in each inlet add the following mixture for each of the Taqman assays.<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:800px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume per inlet</strong><br/></td><td><strong>Volume 10 reactions stock</strong><br/></td></tr><tr><td>20x Taqman gene expression Assay<br/></td><td>2.5 Î¼l<br/></td><td>25 Î¼l<br/></td></tr><tr><td>2x Assay Loading Reagent<br/></td><td>2.5 Î¼l<br/></td><td>25 Î¼l<br/></td></tr></tbody></table>**This reagent can be made in advance and stocked up to 6 months at -20 Â°C</li><li>Sample pre-mix<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td><strong>Components</strong><br/></td><td><strong>Volume per inlet</strong><br/></td><td><strong>Sample pre-mix for all wells</strong><br/></td></tr><tr><td>Taqman Universal PCR Master Mix<br/></td><td>2.5 Î¼l<br/></td><td>360 Î¼l<br/></td></tr><tr><td>20X GE Sample Loading Reagent <br/></td><td>0.25 Î¼l</td><td>36 Î¼l<br/></td></tr></tbody></table></li></ol><p class="pptt" id="biaoti32425">### Acknowledgments</p><p class="ppzz">The present work was supported in part by a Grant-in-Aid for Scientific Research (C) JSPS KAKENHI Grant Number: 19K16761, Grant-in-Aid for Young Scientists (A) JSPS KAKENHI Grant Number: 17H04991 and the âDevelopment of Diagnostic Technology for Detection of miRNA in Body Fluidsâ grant from the Japan Agency for Medical Research and Development (AMED).<br/>  This protocol was adapted from Fluidigm system manufactureâs protocols (PN-100-6177 J1 and PN 68000130 E1).</p><p class="pptt" id="biaoti32426">### Competing interests</p><p class="ppzz">We have no conflict of interest to declare.</p><p class="pptt" id="biaoti32427">### References</p><ol class="npwtwok"><li>Karaiskos, N., Wahle, P., Alles, J., Boltengagen, A., Ayoub, S., Kipar, C., Kocks, C., Rajewsky, N. and Zinzen, R. P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28860209" target="_blank">The <em>Drosophila</em> embryo at single-cell transcriptome resolution.</a> <em>Science</em> 358(6360): 194-199.</li><li>Katsuda, T., Hosaka, K., Matsuzaki, J., Usuba, W., Prieto-Vila, M., Yamaguchi, T., Tsuchiya, A., Terai, S. and Ochiya, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31493546" target="_blank">Transcriptomic dissection of hepatocyte heterogeneity: linking ploidy, zonation, and stem/progenitor cell characteristics.</a> <em>Cell Mol Gastroenterol Hepatol</em>.</li><li>Livak, K. J., Wills, Q. F., Tipping, A. J., Datta, K., Mittal, R., Goldson, A. J., Sexton, D. W. and Holmes, C. C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23079396" target="_blank">Methods for qPCR gene expression profiling applied to 1440 lymphoblastoid single cells.</a> <em>Methods</em> 59(1): 71-79.</li><li>Prieto-Vila, M., Usuba, W., Takahashi, R. U., Shimomura, I., Sasaki, H., Ochiya, T. and Yamamoto, Y. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31289135" target="_blank">Single-cell analysis reveals a preexisting drug-resistant subpopulation in the luminal breast cancer subtype.</a> <em>Cancer Res</em> 79(17): 4412-4425.</li><li>Prieto-vila, M., Yamamoto, Y., Takahashi, R. Ochiya, T. (2018). <a href="https://www.springer.com/gp/book/9789811089527" target="_blank">Single-cell transcriptomics.</a> In: <em>Handbook of Single Cell Technologies</em>. Springer, Singapore.</li><li>Rato, S., Golumbeanu, M., Telenti, A. and Ciuffi, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27816430" target="_blank">Exploring viral infection using single-cell sequencing.</a> <em>Virus Res</em> 239: 55-68.</li><li>Van den Brink, S. C., Sage, F., VÃ©rtesy, Ã., Spanjaard, B., Peterson-Maduro, J., Baron, C. S., Robin, C. and van Oudenaarden, A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28960196" target="_blank">Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations.</a> <em>Nat Methods</em> 14(10): 935-936.</li><li>Wen, L. and Tang, F. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27083874" target="_blank">Single-cell sequencing in stem cell biology.</a> <em>Genome Biol</em> 17: 71.</li><li>Woyke, T., Doud, D. F. R. and Schulz, F. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29088131" target="_blank">The trajectory of microbial single-cell sequencing.</a> <em>Nat Methods</em> 14(11): 1045-1054.</li><li>Zheng, C., Zheng, L., Yoo, J. K., Guo, H., Zhang, Y., Guo, X., Kang, B., Hu, R., Huang, J. Y., Zhang, Q., Liu, Z., Dong, M., Hu, X., Ouyang, W., Peng, J. and Zhang, Z. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28622514" target="_blank">Landscape of infiltrating T cells in liver cancer revealed by single-cell sequencing.</a> <em>Cell</em> 169(7): 1342-1356 e1316.</li><li>Ziegenhain, C., Vieth, B., Parekh, S., Reinius, B., Guillaumet-Adkins, A., Smets, M., Leonhardt, H., Heyn, H., Hellmann, I. and Enard, W. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28212749" target="_blank">Comparative Analysis of Single-Cell RNA Sequencing Methods.</a> <em>Mol Cell</em> 65(4): 631-643 e634.<br/></li></ol>