<body>
            <h1>High-resolution and Deep Phylogenetic Reconstruction of Ancestral  States from Large Transcriptomic Data Sets</h1>
            <p><strong>Authors:</strong> Sumanth Kumar Mutte, Dolf Weijers</p>
            <h2>Abstract</h2>
            <p>Phylogenetics is an important area of evolutionary biology that helps to understand the origin and divergence of genes, genomes and species. Building meaningful phylogenetic trees is needed for the accurate reconstruction of the past. To achieve a correct phylogenetic understanding of genes or proteins, reliable and robust methods are needed to construct meaningful trees. With the rapidly increasing availability of genome and transcriptome sequencing data, there is a need for efficient and accurate methodologies for ancestral state reconstruction. Currently available methods are mostly specific for certain gene families, and require substantial adaptation for their application to other gene families. Hence, a generalized framework is essential to utilize large transcriptome resources such as OneKP and MMETSP. Here, we have developed a flexible yet efficient method, based on core strengths such as emphasis on being inclusive in homolog selection, and defining orthologs based on multi-layered inferences. We illustrate how specific steps can be modified to fit the needs of any protein family under consideration. We also demonstrate the success of this protocol by studying and testing the orthologs in various gene families. Taken together, we present a protocol for reconstructing the ancestral states of various domains and proteins across multiple kingdoms of eukaryotes, using thousands of transcriptomes.</p>
            <p><strong>Keywords:</strong> Phylogenomics, OneKP, MMETSP, Plants, Phylogenetics, Evolution, Transcriptome</p>
            <h2>Background</h2><p>Phylogenetic trees are fundamental to understanding the evolution of genes, gene families, species, phyla and even kingdoms. They help to depict the diversity and also resolve the differences at various levels. For example, at protein level, they help us to identify orthologous groups based on amino acid differences across various species. Earlier, phylogenetic trees were constructed based on few gene/protein sequences from limited numbers of species. With the ever-growing sequencing data, as more and more genomes and transcriptomes are becoming accessible, there is tremendous potential for <em>e.g.</em>, discovery of new lineages, âgap-fillingâ in phylogenies and hence, an improved understanding of biology (Levy and Myers, 2016; Burki <em>et al.</em>, 2019).<br/><br/>In the last decade, many efforts have been made towards defining transcriptomes of hundreds (or even thousands) of species due to the popularity of RNA-Seq (Stark <em>et al.</em>, 2019). Transcriptomes provide a quick insight into the (expressed) gene content of a genome. Even though the individual transcriptomes do not cover the entire gene content of an organism, combining them from multiple cells, tissues and conditions, may comprise the majority of the transcribed genes of that species. Hence, it is a relatively straightforward approach to sequence and assemble a transcriptome without <em>a priori</em> knowledge of the genome. The current-day long-read and single-cell RNA-sequencing technologies make it even easier to build a complete transcriptome (Wang <em>et al.</em>, 2016). Utilizing these technological advances, two large transcriptome sequencing projects, 1000 plant transcriptomes (OneKP; Carpenter <em>et al.</em>, 2019; One Thousand Plant Transcriptomes Initiative, 2019) and Marine Micro Eukaryote Transcriptome Sequencing Project (MMETSP; Keeling <em>et al.</em>, 2014), were developed. OneKP represents the majority of the land plants and algal groups, whereas MMETSP covers majority of the SAR group and other (unidentified) phyla in Chromista.<br/><br/>From their inception, diverse approaches have been developed and applied to these transcriptomes and estimate the ancestral states of various genes across multiple classes, families and even phyla (Li <em>et al.</em>, 2014; Wickett <em>et al.</em>, 2014; Yerramsetty <em>et al.</em>, 2016). The majority of these methods focus on one gene family, and need substantial modifications in methodology to apply them to other gene families. Moreover, the methods used are neither inclusive nor robust in terms of multi-layered inferences. The orthologous inferences are based on only one evidence, Best Bi-directional Hit or protein domains or simple phylogenies based on few genomes. To overcome these disadvantages, we developed a unified framework to build high-resolution phylogenies that utilize the rich OneKP and MMETSP transcriptome resources. This new method is not only inclusive, but also utilizes multi-layered orthology to interpret phylogenies with high confidence, leading to the identification of new (sub-)classes of orthologs.<br/><br/><strong>Overview of the protocol</strong><br/>The current protocol is developed to reconstruct ancestral states and high-resolution phylogenetic trees of various gene families using transcriptomes and/or proteomes. Ancestral state represents the minimal gene complement at each evolutionary node, where species-specific gene duplications and (or) losses would have modified the gene complement in individual species. Hence, selecting the correct, orthologous as well as diverse, sequences is a crucial step in such a deep phylogenetic tree construction. This protocol is built on three core strengths: (1) Inclusive: Include more sequences at the start with liberal parameters, and remove sequences as one goes through various steps in the pipeline, resulting in a high-quality logical sequence set for phylogenetic tree construction. (2) Multi-layered: Multiple levels of orthology confirmation, <em>i.e.</em>, based on the domain architecture, reciprocal BLAST and the phylogenetic tree. (3) Robust: No limitations on length of the protein or the number of sequences used in the phylogeny, with suggestions on alternate analysis packages tested in various steps. Overall, the protocol comprises 14 steps that are divided into three sections: Homolog identification (Steps 1-5), Ortholog detection (Steps 6-8) and Phylogeny construction (Steps 9-14). All the general parameters and recommendations for the respective steps are indicated below.</p>
        </body>

<p class="pptt" id="biaoti32453">### Equipment</p><ol class="npwtwok"><li>Linux machine<br/>Computer set-up: Majority of the mentioned programs in Software section run only on Linux environment; hence it is recommended to perform the analysis on a Linux machine with access to the BASH shell (terminal). The average time needed to perform the analysis for a gene family is 1-1.5 days on a generic Linux workstation with 64 GB RAM and 8-core processor setup. The disk space needed for this analysis is less than 1 GB.<br/></li></ol><p class="pptt" id="biaoti32454">### Software</p><ol class="npwtwok"><li>tblastn and blastp from BLAST+ module v2.9.0 (Camacho <em>et al.</em>, 2009) (<a href="ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/" target="_blank">ftp://ftp.ncbi.nlm.nih.gov/blast/executables/blast+/LATEST/</a>)</li><li>faSomeRecords: Linux binary from UCSC (<a href="http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/" target="_blank">http://hgdownload.cse.ucsc.edu/admin/exe/linux.x86_64/</a>)</li><li>TransDecoder v5.5.0 (Haas <em>et al.</em>, 2013) (transdecoder.github.io)</li><li>MEME motif discovery v5.1.0 (Bailey <em>et al.</em>, 2009) (<a href="http://meme-suite.org/" target="_blank">http://meme-suite.org/</a>)</li><li>ScanProsite web-tool (<a href="https://prosite.expasy.org/scanprosite" target="_blank">https://prosite.expasy.org/scanprosite</a>)</li><li>InterProScan v5.38-76.0 (Jones <em>et al.</em>, 2014) (<a href="https://github.com/ebi-pf-team/interproscan" target="_blank">https://github.com/ebi-pf-team/interproscan</a>)</li><li>MAFFT v7 (Katoh and Standley, 2013) (<a href="https://mafft.cbrc.jp/alignment/software/" target="_blank">https://mafft.cbrc.jp/alignment/software/</a>)</li><li>JalView (Waterhouse <em>et al.</em>, 2009) (<a href="https://www.jalview.org/" target="_blank">https://www.jalview.org/</a>)</li><li>ModelFinder (Kalyaanamoorthy <em>et al.</em>, 2017) (accessed as in-built module from IQ-TREE)</li><li>ModelTest-NG (Darriba <em>et al.</em>, 2020) (<a href="https://github.com/ddarriba/modeltest" target="_blank">https://github.com/ddarriba/modeltest</a>)</li><li>PartitionFinder v2 (Lanfear <em>et al.</em>, 2012) (<a href="http://www.robertlanfear.com/partitionfinder/" target="_blank">http://www.robertlanfear.com/partitionfinder/</a>)</li><li>IQ-TREE v1.6.12 (Nguyen <em>et al.</em>, 2015) (<a href="http://www.iqtree.org/" target="_blank">http://www.iqtree.org</a>)</li><li>RAxML v8 (Stamatakis, 2014) (<a href="https://cme.h-its.org/exelixis/web/software/raxml/index.html" target="_blank">https://cme.h-its.org/exelixis/web/software/raxml/index.html</a>)</li><li>PhyML v3.3 (Guindon <em>et al.</em>, 2010) (<a href="https://github.com/stephaneguindon/phyml" target="_blank">https://github.com/stephaneguindon/phyml</a>)</li><li>MrBayes v3.2.7 (Ronquist <em>et al.</em>, 2012) (<a href="https://github.com/NBISweden/MrBayes" target="_blank">https://github.com/NBISweden/MrBayes</a>)</li><li>iTOL v4 (Letunic and Bork, 2019) (<a href="https://itol.embl.de/" target="_blank">https://itol.embl.de</a>)</li><li>Linux BASH shell (terminal) âcut, sort and uniqâ functions (<a href="https://tiswww.case.edu/php/chet/bash/bashref.html" target="_blank">https://tiswww.case.edu/php/chet/bash/bashref.html</a>)</li><li>Scripts used for automating certain steps in the protocol are available through GitHub (<a href="https://github.com/sumanthmutte/Phylogenomics" target="_blank">https://github.com/sumanthmutte/Phylogenomics</a>)</li></ol><br/><strong>Data</strong><ol class="npwtwok"><li>OneKP dataset (1000 plant transcriptomes project): Contains 1341 transcriptomes from 1179 species covering all the major classes of land plants, green algae, red algae and glaucophytes (Carpenter <em>et al.</em>, 2019; One Thousand Plant Transcriptomes Initiative, 2019); <a href="http://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/oneKP_capstone_2019" target="_blank">http://datacommons.cyverse.org/browse/iplant/home/shared/commons_repo/curated/oneKP_capstone_2019</a> </li><li>MMETSP dataset (Marine Microbial Eukaryote Transcriptome Sequencing Project): Contains 678 transcriptomes from 410 species covering all the major classes of Stramenopila and Alveolata (SAR group) and many unclassified (unicellular) marine eukaryotes (Keeling <em>et al.</em>, 2014); <a href="https://gold.jgi.doe.gov/study?id=Gs0128947" target="_blank">https://gold.jgi.doe.gov/study?id=Gs0128947</a><br/></li></ol><p class="pptt" id="biaoti32455">### Procedure</p><p class="ppzz" style="margin-left:30px;">Commands used, along with the parameters used in each step of the protocol, with step numbers corresponding to Figure 1 are given below. Before starting the protocol, we first created a BLAST database for each transcriptome or proteome. This was carried out only once for each transcriptome or proteome using the <em>makeblastdb</em> function, where â-inâ takes a FASTA file of the transcriptome, or the proteome and â-dbtypeâ is the database type with nucl and prot for transcriptomes and proteomes, respectively.<br/><br/><span style="font-family:Courier New;">$ makeblastdb -dbtype nucl -in transcriptome.fasta</span><br/><br/><span style="font-family:Courier New;"> $ makeblastdb -dbtype prot -in proteome.fasta</span><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200315/20200315204733_1277.png" width="300"/><br/><strong>Figure 1. Methodology schematic showing various steps of the protocol used for ortholog identification and phylogenetic tree construction.</strong> Circled numbers correspond to the various steps of the protocol as indicated in the procedure. Programs/software/algorithms used are indicated next to the arrows in grey. File formats for text and FASTA are depicted as shown in legend.</p><p class="ppzz"><br/></p><ol class="npwone"><li>Homolog identification<ol class="npwtwo"><li>To perform a BLAST search to the respective database(s), we created a query protein sequence file (in FASTA format), with sequences from (relatively) well-annotated genomes and from a diverse range of species, if present, across multiple kingdoms. A list of various species used along with a link to the sequence data resource is available in <a href="https://os.bio-protocol.org/attached/file/20200312/3566 Appendix 1.docx" target="_blank">Appendix-1</a>.</li><li>Using the query sequence file (-query) perform the BLAST search with <em>tblastn</em> and <em>blastp</em> modules, against transcriptome and proteome databases (-db), respectively. When the E-value cut-off (-evalue) is less than 0.01, save the output (-out) in a tab-delimited text file indicated with -outfmt 6 . The remainder of the parameters are kept at default settings.<br/><br/><span style="font-family:Courier New;">$ tblastn -query filename.fa -db transcriptome.fasta -out output.blast -evalue 0.01 -outfmt '6 qseqid sseqid slen qstart qend sstart send evalue bitscore score length pident nident positive ppos mismatch gaps frames qcovs qcovhsp sseq'</span><br/><span style="font-family:Courier New;"> $ blastp -query filename.fa -db proteome.fasta -out output.blast -evalue 0.01 -outfmt '6 qseqid sseqid slen qstart qend sstart send evalue bitscore score length pident nident positive ppos mismatch gaps frames qcovs qcovhsp sseq'</span><br/><br/></li><li>The BLAST output contains all the scoring information about the subject (transcript/protein) sequence that has a similarity to the corresponding query sequence. To retrieve the subject sequence identifiers from the BLAST output, we have used the <em>âcutâ</em>, <em>âsortâ</em> and <em>âuniqâ</em> functions of a Linux BASH shell (terminal). <em>âcutâ</em> takes the BLAST output (output.blast) from the previous step, and takes the second column (-f2), <em>i.e.</em>, subject sequence identifiers and sends/pipes them (|) to the <em>âsortâ</em> function. After sorting, they are passed on to the <em>âuniqâ</em> function to remove the duplicates and the output is written to the file (SubjectIdentifiers.txt).<br/><br/><span style="font-family:Courier New;">$ cut -f2 output.blast | sort | uniq &gt; SubjectIdentifiers.txt</span><br/><br/></li><li>Using these identifiers (SubjectIdentifiers.txt) to extract the corresponding transcript (SelectedTranscripts.fasta) or protein sequences (SelectedProteins.fasta) from the respective transcriptome or proteome by running the â<em>faSomeRecords</em>â program.<br/><br/><span style="font-family:Courier New;">$ faSomeRecords transcriptome.fasta SubjectIdentifiers.txt SelectedTranscripts.fasta</span><br/><span style="font-family:Courier New;"> $ faSomeRecords proteome.fasta SubjectIdentifiers.txt SelectedProteins.fasta</span><br/><br/></li><li>The protein sequences are more informative due to the higher number of site patterns and can be directly used for phylogeny construction. Whereas, the transcript sequences should be translated to protein sequences using the program <em>TransDecoder</em> with default settings. First, determine the longest Open Reading Frames (ORFs of at least 100 amino acids in length) of the transcript by TransDecoder.LongOrfs. And then the CDS and the corresponding amino acid sequences of these ORFs thorugh TransDecoder.Predict. If the tree based on protein sequences result in poor bootstraps, we would suggest generating the tree with CDS (DNA) sequences.<br/><br/><span style="font-family:Courier New;">$ perl TransDecoder.LongOrfs -t SelectedTranscripts.fasta</span><br/><span style="font-family:Courier New;"> $ perl TransDecoder.Predict -t SelectedTranscripts.fasta</span><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Ortholog detection<ol class="npwtwo" start="6"><li>Not all the sequences that have an E-value &lt; 0.01 are true orthologs of a query protein. Hence, additional filters are needed to remove non-orthologs. One such filter is the presence of the same domains in orthologous proteins. For some well-annotated proteins (<em>e.g.</em>, Auxin Response Factors, Kinases, <em>etc.</em>), domain information is readily available in the InterPro domain database. Scan the protein sequences from the previous step (-i SelectedProteins.fasta) for the presence of known domains using <em>InterProScan</em> tool (interproscan.sh), which produces a tab-delimited (TSV) file as well as HTML/XML files (-f TSV,HTML,XML), with all the domains identified along with the corresponding InterPro identifiers (-iprlookup) in each protein sequence. A Python script was developed (InterproscanSummary.py) to process this TSV file, in order to extract the final set of protein sequences that have the domains of interest (See GITHUB page for more details). <em>InterProScan</em> is a time-consuming step, hence we used pre-annotated data where available, or reduced the number of databases to scan (using -appl Pfam,CDD setting), in order to save time. In some cases, we split the data in smaller batches and ran on multiple processors.<br/><br/><span style="font-family:Courier New;">$ interproscan.sh -f TSV,HTML,XML -iprlookup -i SelectedProteins.fasta</span><br/><span style="font-family:Courier New;"> $ python InterproscanSummary.py</span><br/><br/></li><li>Certain proteins (<em>e.g.</em>, SOSEKI in <em>Arabidopsis</em>; Yoshida <em>et al.</em>, 2019) lack annotated (functional) domain information. Use the <em>MEME</em> program to predict the conserved motifs/domains in those proteins with Zero or One Occurrence Per Sequence criteria (-mod zoops) and a minimum width of 10 (-minw), with a maximum of 10 motifs predicted per set (-nmotifs). The <em>MEME</em> outputs the motifs along with their patterns in HTML/TEXT format. Then use these motif patterns in <em>ScanProsite</em> web-tool to identify the domains in the protein sequences that do not have annotated domains. We have applied this approach successfully to annotate the SOSEKI protein family and identify its orthologs (van Dop <em>et al.</em>, 2020).<br/><br/><span style="font-family:Courier New;">$ meme ProteinSequences.fa -o OutputName -protein -mod zoops -nmotifs 10 -minw 10 </span><br/><br/></li><li>After selecting the protein sequences that have the domains of interest, they are queried back to the proteomes of the species used in Step A1 to confirm the orthologous relationships using the best Bi-directional BLAST Hits (BBH) strategy. Here we have used the option of maximum target sequences or the number of best hits in the output (-max_target_seqs) set to 1, or sometimes 2 when domains are abundant in the genome (for <em>e.g.</em>, bHLH), with E-value &lt; 0.01 (-evalue). This final set of proteins that have hits with the protein under consideration are regarded as the âtrueâ orthologous proteins for further analysis. Output is recorded in a TSV files, same as in Step A2 (-outfmt 6).<br/><br/><span style="font-family:Courier New;">$ blastp -query filename.fa -db ArabidopsisProteome.fasta -out BBhits.blastp -max_target_seqs 1 -evalue 0.01 -outfmt '6 qseqid sseqid slen qstart qend sstart send evalue bitscore score length pident nident positive ppos mismatch gaps frames qcovs qcovhsp sseq'</span><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Phylogeny construction<ol class="npwtwo" start="9"><li>These âtrueâ sets of orthologs are used for alignment followed by the phylogenetic tree construction. <em>MAFFT</em> is used to align protein sequences. The <em>E-INS-i</em> <em>(</em>--genafpair<em>)</em> algorithm is used while aligning proteins with multiple domains separated by poorly conserved sequences (<em>e.g.</em>, ARF or Aux/IAA proteins), whereas <em>G-INS-i (</em>--globalpair<em>)</em> is used while aligning only domain-specific sequences (<em>e.g.</em>, PB1 domain). An iterative refinement method is used in both cases, with a maximum of 1000 iterations (--maxiterate 1000), after which the final alignment is written to a FASTA file (output_file).<br/><br/><span style="font-family:Courier New;">$ mafft --genafpair --maxiterate 1000 input_file &gt; output_file</span><br/><span style="font-family:Courier New;"> $ mafft --globalpair --maxiterate 1000 input_file &gt; output_file</span><br/><br/></li><li>Once the alignments are generated, use the <em>trimAl</em> to remove the sequence positions (columns) with more than 50%-80% gaps, as they are considered to lack phylogenetic signal. Hence, for phylogenetic tree construction, only use the sequences without spurious gaps. A gap-threshold of 0.2 (-gt 0.2), is set to remove all positions in the alignment with gaps in 80% (or more) of the sequences. For the gene families that have moderately conserved domains (<em>e.g.</em>, ARF, Aux/IAA), use a threshold of 0.3 or 0.4, whereas for poorly conserved domains (<em>e.g.</em>, PB1) it is set at 0.2, and for highly conserved proteins (<em>e.g.</em>, ROP, ROPGEF) it is set between 0.6 and 0.8. An additional (optional) check is kept in place, where the sequences that are shorter than 1/4<sup>th</sup> of the average sequence length are further removed in <em>JalView</em>.<br/><em>Note: There are various tools specialized for the clean-up of the alignment, such as GBlocks, Guidance, AliScore, ZORRO etc. However, a simple gap-based trimming in trimAl resulted in (almost) the same quality of alignment and tree topology when compared to these specialized tools. Hence, we used trimAl for alignment clean-up throughout this study.</em><br/><br/><span style="font-family:Courier New;">$ trimal -in inputfile.fa -out outputfile.fa -fasta -gt 0.2</span><br/><br/></li><li>Then use this âcleanâ alignment to identify the most appropriate model of evolution for each protein family. <em>ModelFinder</em> and <em>ModelTest-NG</em> are used to predict the best model based on the Akaike- and Bayesian- Information Criterion (AIC and BIC). For the majority of the protein families, both programs provide the same models as the best models. The situations where there is a mis-match between the two programs, use a third program (either <em>PartitionFinder</em> or a Perl script from <em>RAxML</em> distribution) to decide on the best model based on the majority rule. As expected, various proteins evolve differently, leading to different models of evolution. <em>ModelFinder</em> is run as a part of IQ-TREE, hence it does not require any additional steps. <em>ModelTest-NG</em> requires the type (either amino acid or nucleotide -d) of input dataset (-i INFILE) and writes the statistics and the best model to the output file (-o OUTFILE). <em>PartitionFinder</em> requires the alignment, in the <em>PHYLIP</em> format (instead of <em>FASTA</em> format as in others) placed in the folder âpartition_finder_modelsâ, where the output statistics and best model are also recorded. <em>FASTA</em> to <em>PHYLIP</em> format conversion can be made through the Perl script (fasta2relaxedPhylip.pl), which takes input <em>FASTA</em> (-f input.fa) and writes the output in <em>PHYLIP</em> format (-o output.phylip).<br/><br/><span style="font-family:Courier New;">$ modeltest-ng -d aa -i INFILE -o OUTFILE</span><br/><span style="font-family:Courier New;"> $ perl RAxML_ProteinModelSelection.pl alignment.fasta</span><br/><span style="font-family:Courier New;"> $ perl fasta2relaxedPhylip.pl -f input.fa -o output.phylip</span><br/><span style="font-family:Courier New;"> $ python PartitionFinderProtein.py partition_finder_models</span><br/><br/></li><li>Phylogenetic trees are built mainly using <em>IQ-TREE</em> and <em>RAxML</em> based on the âcleanâ alignment produced in Step C10 and the evolutionary model predicted in Step C11. For the phylogenetic trees made through <em>IQ-TREE</em>, we have used 1,000 rapid bootstraps (-bb 1,000) and SH-like approximate Likelihood Ratio Test (-aLRT 1,000), combined with automatic model finding through <em>ModelFinder</em> (-m MFP+MERGE). For the trees made with RAxML, we have also used rapid bootstrapping and Maximum Likelihood search in the same run (-f a) but with an extended majority rule (-# autoMRE) based bootstopping criteria. In addition, we gave a random seed number (-x and -p) to turn-on rapid bootstrapping and parsimony inference, whereas -m takes in the model from the previous Step C11. For trees with very poor bootstrap support for majority of the branches, we used another phylogenetic tree construction program, <em>PhyML</em>, with 100 bootstrap replicates (-b 100), empirical amino-acid frequencies (-f e), gamma shape parameter estimated from maximum likelihood (-a e) and the topology was searched based on the sub-tree pruning and re-grafting approach (-s SPR). After running these multiple programs, the trees obtained were compared to understand the overall topology based on the congruent branches (see next step). We have also tried and tested various Bayesian approaches (using <em>MrBayes</em>), but the trees never converged even after months of computation, and provided various incongruent topologies. Hence, all the analyses were performed with Maximum Likelihood approaches.<br/><br/><span style="font-family:Courier New;">$ iqtree -s CleanAlignment.fa -pre OutputName -alrt 1000 -bb 1000 -m MFP+MERGE</span><br/><span style="font-family:Courier New;"> $ raxmlHPC-PTHREADS-AVX2 -f a -x 12345 -p 12345 -j -# autoMRE -m PROTGAMMAJTT -s CleanAlignment.fa -n OutputName</span><br/><span style="font-family:Courier New;"> $ PhyML-3.1_linux64 -i CleanAlignment.fa -d aa -b 100 -m JTT -f e -s SPR -a e</span><br/><br/></li><li>Visualize all the final phylogenetic trees using the iTOL webserver and then various datasets on the phylogenetic trees. Generate protein domain information from the <em>InterProScan</em> or <em>MEME</em>, sequence length from <em>TransDecoder</em> and clade/taxonomy information from OneKP and MMETSP databases following the instructions provided in the iTOL documentation.</li><li>Once the trees are obtained, they are manually checked for errors. Manually remove the branches with long branch attraction, or partial sequences or any misplaced taxa. If the proportion of these misplaced branches is too high, re-analyze the phylogeny with more sequences from other species, as well as by removing the spurious sequences. These steps are repeated until obtain better trees that are not only supported by good bootstraps but also obeys the taxonomy of those phyla.</li></ol><br/><strong>Limitations and Conclusions</strong><br/>Due to the generalized nature of the method, it was difficult to automate the complete protocol. Hence, wherever possible, the method was simplified with scripts/commands dedicated for fast and parallel processing. On the other hand, it gave control over the decision-making process based on the protein under consideration. When dealing with highly redundant protein families, we removed highly similar proteins (&gt; 90% similarity), prior to phylogeny, which reduced the (computational) time without losing accuracy. In many cases we observed that the best-hit in <em>reciprocal-BLAST</em> is not really a BBH, as sometimes a second hit was still the best one due to one or few amino acid difference(s) (especially in proteins with common domains <em>e.g.</em> bHLH or PB1). Hence, in those cases we considered two best hits and used both for phylogeny construction. The false positive orthologs were eventually placed in the outgroup (or at least separate from the ingroup) in the phylogenetic tree. As we were dealing with transcriptomes, we could not predict the actual gene copy number in each species, but only the ancestral copy number for that class or phylum, by comparing the ancestral copies across the majority of the species in that phylum. Another issue of dealing with (low-depth) transcriptomes was that we found many partial transcripts leading to the truncated proteins/domains, or we might fail to identify the transcripts that were not expressed in that particular tissue or condition. In that regard, combining ortholog sequence information from multiple transcriptomes or species of various families is mandatory to confirm the ancestral state for each class or phylum.<br/><br/>Based on this protocol and the guidelines mentioned above, we have reconstructed the ancestral states of various protein families along with their orthologs in a âdeepâ phylogenetic space, across multiple kingdoms. We demonstrated how this method was implemented for proteins that are well-defined with known domains, novel proteins with unknown domains, poorly conserved domains and phylum/kingdom-specific proteins that (dis)appeared at various stages in evolution. This approach was successfully applied for the core proteins of the auxin signalling (Nuclear Auxin Pathway (NAP)) and biosynthesis pathways. NAP includes Auxin Response Factor (ARF), Auxin/Indole-3-Acetic-Acid (Aux/IAA) and Transport Inhibitor Response 1/Auxin-signalling F-Box (TIR1/AFB; Mutte <em>et al.</em>, 2018). Biosynthesis pathway proteins include TAA family of amino transferase (TAA) and YUCCA family of monooxygenases (YUC). It was also applied to the individual domains, Phox and Bem1 (PB1; Mutte and Weijers <em>et al.</em>, 2020), along with various downstream targets of the auxin pathway, such as SOSEKI (SOK; van Dop <em>et al.</em>, 2020), Target of MOnopteros 5 (TMO5) and its interaction partner Lonesome HighWay (LHW; Lu <em>et al.</em>, 2020). Taken together, by following this protocol in combination with ever-growing high-quality sequence data, and leaping developments in the methods and algorithms in phylogenetics, reveal new evolutionary insights into our understanding of proteins and the crucial pathways.</li></ol><p class="pptt" id="biaoti32456">### Acknowledgments</p><p class="ppzz">The authors would like to thank the 1,000 plant transcriptomes (OneKP) and Marine Micro Eukaryotic Transcriptome Sequencing Project (MMETSP) consortiums for providing such a massive data resources for the scientific community. Efforts of all the authors are highly appreciated, who developed many extremely useful and efficient programs and algorithms for phylogenetics, and making them freely accessible to the scientific community.</p><p class="pptt" id="biaoti32457">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt" id="biaoti32458">### References</p><ol class="npwtwok"><li>Bailey, T. L., Boden, M., Buske, F. A., Frith, M., Grant, C. E., Clementi, L., Ren, J., Li, W. W. and Noble, W. S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19458158" target="_blank">MEME SUITE: tools for motif discovery and searching.</a> <em>Nucleic Acids Res</em> 37(Web Server issue): W202-208.</li><li>Burki, F., Roger, A. J., Brown, M. W. and Simpson, A. G. B. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31606140" target="_blank">The new tree of Eukaryotes.</a> <em>Trends Ecol Evol</em> 35(1):43-55.</li><li>Camacho, C., Coulouris, G., Avagyan, V., Ma, N., Papadopoulos, J., Bealer, K. and Madden, T. L. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500" target="_blank">BLAST+: architecture and applications.</a> <em>BMC Bioinformatics</em> 10: 421.</li><li>Carpenter, E. J., Matasci, N., Ayyampalayam, S., Wu, S., Sun, J., Yu, J., Jimenez Vieira, F. R., Bowler, C., Dorrell, R. G., Gitzendanner, M. A., Li, L., Du, W., K, K. U., Wickett, N. J., Barkmann, T. J., Barker, M. S., Leebens-Mack, J. H. and Wong, G. K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31644802" target="_blank">Access to RNA-sequencing data from 1,173 plant species: The 1,000 Plant transcriptomes initiative (1KP).</a> <em>Gigascience</em> 8(10).</li><li>Darriba, D., Posada, D., Kozlov, A. M., Stamatakis, A., Morel, B. and Flouri, T. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31432070" target="_blank">ModelTest-NG: a new and scalable tool for the selection of DNA and protein evolutionary models.</a> <em>Mol Biol Evol</em> 37(1):291-294.</li><li>Guindon, S., Dufayard, J. F., Lefort, V., Anisimova, M., Hordijk, W. and Gascuel, O. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" target="_blank">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0.</a> <em>Syst Biol</em> 59(3): 307-321.</li><li>Haas, B. J., Papanicolaou, A., Yassour, M., Grabherr, M., Blood, P. D., Bowden, J., Couger, M. B., Eccles, D., Li, B., Lieber, M., MacManes, M. D., Ott, M., Orvis, J., Pochet, N., Strozzi, F., Weeks, N., Westerman, R., William, T., Dewey, C. N., Henschel, R., LeDuc, R. D., Friedman, N. and Regev, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23845962" target="_blank">De novo transcript sequence reconstruction from RNA-seq using the Trinity platform for reference generation and analysis.</a> <em>Nat Protoc</em> 8(8): 1494-1512.</li><li>Jones, P., Binns, D., Chang, H. Y., Fraser, M., Li, W., McAnulla, C., McWilliam, H., Maslen, J., Mitchell, A., Nuka, G., Pesseat, S., Quinn, A. F., Sangrador-Vegas, A., Scheremetjew, M., Yong, S. Y., Lopez, R. and Hunter, S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" target="_blank">InterProScan 5: genome-scale protein function classification.</a> <em>Bioinformatics</em> 30(9): 1236-1240.</li><li>Kalyaanamoorthy, S., Minh, B. Q., Wong, T. K. F., von Haeseler, A. and Jermiin, L. S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28481363" target="_blank">ModelFinder: fast model selection for accurate phylogenetic estimates.</a> <em>Nat Methods</em> 14(6): 587-589.</li><li>Katoh, K. and Standley, D. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23329690" target="_blank">MAFFT multiple sequence alignment software version 7: improvements in performance and usability.</a> <em>Mol Biol Evol</em> 30(4): 772-780.</li><li>Keeling, P. J., Burki, F., Wilcox, H. M., Allam, B., Allen, E. E., Amaral-Zettler, L. A., Armbrust, E. V., Archibald, J. M., Bharti, A. K., Bell, C. J., Beszteri, B., Bidle, K. D., Cameron, C. T., Campbell, L., Caron, D. A., Cattolico, R. A., Collier, J. L., Coyne, K., Davy, S. K., Deschamps, P., Dyhrman, S. T., Edvardsen, B., Gates, R. D., Gobler, C. J., Greenwood, S. J., Guida, S. M., Jacobi, J. L., Jakobsen, K. S., James, E. R., Jenkins, B., John, U., Johnson, M. D., Juhl, A. R., Kamp, A., Katz, L. A., Kiene, R., Kudryavtsev, A., Leander, B. S., Lin, S., Lovejoy, C., Lynn, D., Marchetti, A., McManus, G., Nedelcu, A. M., Menden-Deuer, S., Miceli, C., Mock, T., Montresor, M., Moran, M. A., Murray, S., Nadathur, G., Nagai, S., Ngam, P. B., Palenik, B., Pawlowski, J., Petroni, G., Piganeau, G., Posewitz, M. C., Rengefors, K., Romano, G., Rumpho, M. E., Rynearson, T., Schilling, K. B., Schroeder, D. C., Simpson, A. G., Slamovits, C. H., Smith, D. R., Smith, G. J., Smith, S. R., Sosik, H. M., Stief, P., Theriot, E., Twary, S. N., Umale, P. E., Vaulot, D., Wawrik, B., Wheeler, G. L., Wilson, W. H., Xu, Y., Zingone, A. and Worden, A. Z. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24959919" target="_blank">The Marine Microbial Eukaryote Transcriptome Sequencing Project (MMETSP): illuminating the functional diversity of eukaryotic life in the oceans through transcriptome sequencing.</a> <em>PLoS Biol</em> 12(6): e1001889.</li><li>Lanfear, R., Calcott, B., Ho, S. Y. and Guindon, S. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22319168" target="_blank">Partitionfinder: combined selection of partitioning schemes and substitution models for phylogenetic analyses.</a> <em>Mol Biol Evol</em> 29(6): 1695-1701.</li><li>Letunic, I. and Bork, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30931475" target="_blank">Interactive Tree Of Life (iTOL) v4: recent updates and new developments.</a> <em>Nucleic Acids Res</em> 47(W1): W256-W259.</li><li>Levy, S. E. and Myers, R. M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27362342" target="_blank">Advancements in next-Generation sequencing.</a> <em>Annu Rev Genomics Hum Genet</em> 17: 95-115.</li><li>Li, F. W., Villarreal, J. C., Kelly, S., Rothfels, C. J., Melkonian, M., Frangedakis, E., Ruhsam, M., Sigel, E. M., Der, J. P., Pittermann, J., Burge, D. O., Pokorny, L., Larsson, A., Chen, T., Weststrand, S., Thomas, P., Carpenter, E., Zhang, Y., Tian, Z., Chen, L., Yan, Z., Zhu, Y., Sun, X., Wang, J., Stevenson, D. W., Crandall-Stotler, B. J., Shaw, A. J., Deyholos, M. K., Soltis, D. E., Graham, S. W., Windham, M. D., Langdale, J. A., Wong, G. K., Mathews, S. and Pryer, K. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24733898" target="_blank">Horizontal transfer of an adaptive chimeric photoreceptor from bryophytes to ferns.</a> <em>Proc Natl Acad Sci U S A</em> 111(18): 6672-6677.</li><li>Lu, K. J., van 't Wout Hofland, N., Mor, E., Mutte, S., Abrahams, P., Kato, H., Vandepoele, K., Weijers, D. and De Rybel, B. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31874927" target="_blank">Evolution of vascular plants through redeployment of ancient developmental regulators.</a> <em>Proc Natl Acad Sci U S A</em> 117(1): 733-740. </li><li>Mutte, S. K., Kato, H., Rothfels, C., Melkonian, M., Wong, G. K. and Weijers, D. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29580381" target="_blank">Origin and evolution of the nuclear auxin response system.</a> <em>Elife</em> 7: e33399.</li><li>Mutte, S.K., Weijers, D. (2020). <a href="https://doi.org/10.1038/s41598-020-60733-9" target="_blank">Deep Evolutionary History of the Phox and Bem1 (PB1) Domain Across Eukaryotes.</a> <em>Sci Rep</em> 10: 3797.</li><li>Nguyen, L. T., Schmidt, H. A., von Haeseler, A. and Minh, B. Q. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25371430" target="_blank">IQ-TREE: a fast and effective stochastic algorithm for estimating maximum-likelihood phylogenies.</a> <em>Mol Biol Evol</em> 32(1): 268-274.</li><li>One Thousand Plant Transcriptomes, I. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31645766" target="_blank">One thousand plant transcriptomes and the phylogenomics of green plants.</a> <em>Nature</em> 574(7780): 679-685. </li><li>Ronquist, F., Teslenko, M., van der Mark, P., Ayres, D. L., Darling, A., HÃ¶hna, S., Larget, B., Liu, L., Suchard, M. A. and Huelsenbeck, J. P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22357727" target="_blank">MrBayes 3.2: efficient Bayesian phylogenetic inference and model choice across a large model space.</a> <em>Syst Biol</em> 61(3): 539-542.</li><li>Stamatakis, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623" target="_blank">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</a>. <em>Bioinformatics</em> 30(9): 1312-1313. </li><li>Stark, R., Grzelak, M. and Hadfield, J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31341269" target="_blank">RNA sequencing: the teenage years.</a> <em>Nat Rev Genet</em> 20(11): 631-656.</li><li>van Dop, M., Fiedler, M., Mutte, S., de Keijzer, J., Olijslager, L., Albrecht, C., Liao, C-Y., Janson, M., Bienz, M., and Weijers, D. (2020). A conserved biochemical paradigm underlies cell polarity across multicellular kingdoms. <em>Cell</em> (in press).</li><li>Wang, B., Tseng, E., Regulski, M., Clark, T. A., Hon, T., Jiao, Y., Lu, Z., Olson, A., Stein, J. C. and Ware, D. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27339440" target="_blank">Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing.</a> <em>Nat Commun</em> 7: 11708.</li><li>Waterhouse, A. M., Procter, J. B., Martin, D. M., Clamp, M. and Barton, G. J. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19151095" target="_blank">Jalview Version 2--a multiple sequence alignment editor and analysis workbench.</a> <em>Bioinformatics</em> 25(9): 1189-1191.</li><li>Wickett, N. J., Mirarab, S., Nguyen, N., Warnow, T., Carpenter, E., Matasci, N., Ayyampalayam, S., Barker, M. S., Burleigh, J. G., Gitzendanner, M. A., Ruhfel, B. R., Wafula, E., Der, J. P., Graham, S. W., Mathews, S., Melkonian, M., Soltis, D. E., Soltis, P. S., Miles, N. W., Rothfels, C. J., Pokorny, L., Shaw, A. J., DeGironimo, L., Stevenson, D. W., Surek, B., Villarreal, J. C., Roure, B., Philippe, H., dePamphilis, C. W., Chen, T., Deyholos, M. K., Baucom, R. S., Kutchan, T. M., Augustin, M. M., Wang, J., Zhang, Y., Tian, Z., Yan, Z., Wu, X., Sun, X., Wong, G. K. and Leebens-Mack, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25355905" target="_blank">Phylotranscriptomic analysis of the origin and early diversification of land plants.</a> <em>Proc Natl Acad Sci U S A</em> 111(45): E4859-4868.</li><li>Yerramsetty, P., Stata, M., Siford, R., Sage, T. L., Sage, R. F., Wong, G. K., Albert, V. A. and Berry, J. O. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27356975" target="_blank">Evolution of RLSB, a nuclear-encoded S1 domain RNA binding protein associated with post-transcriptional regulation of plastid-encoded rbcL mRNA in vascular plants.</a> <em>BMC Evol Biol</em> 16(1): 141.</li><li>Yoshida, S., van der Schuren, A., van Dop, M., van Galen, L., Saiga, S., Adibi, M., Moller, B., Ten Hove, C. A., Marhavy, P., Smith, R., Friml, J. and Weijers, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30737509" target="_blank">A SOSEKI-based coordinate system interprets global polarity cues in <em>Arabidopsis</em>.</a> <em>Nat Plants</em> 5(2): 160-166.<br/></li></ol>