<body>
            <h1>Bimolecular Fluorescence Complementation (BiFC) for Studying Sarcomeric Protein Interactions in <em>Drosophila</em></h1>
            <p><strong>Authors:</strong> OcÃ©ane  Marescal, Frieder  SchÃ¶ck, Nicanor GonzÃ¡lez-Morales</p>
            <h2>Abstract</h2>
            <p>Protein-protein interactions in <em>Drosophila</em> myofibrils are essential for their function and formation. Bimolecular Fluorescence Complementation (BiFC) is an effective method for studying protein interactions and localization. BiFC relies on the reconstitution of a monomeric fluorescent protein from two half-fragments when in proximity. Two proteins tagged with the different half-fragments emit a fluorescent signal when they are in physical contact, thus revealing a protein interaction and its spatial distribution. Because myofibrils are large networks of interconnected proteins, BIFC is an ideal method to study protein-protein interactions in myofibrils. Here we present a protocol for generating transgenic flies compatible with BiFC and a method for analyzing protein-protein interactions based on the fluorescent BiFC signal in myofibrils. Our protocol is applicable to the majority of <em>Drosophila</em> proteins and with few modifications may be used to study any tissue.</p>
            <p><strong>Keywords:</strong> Muscles, BiFC, Sarcomeres, <em>Drosophila</em>, Indirect flight muscle, Z-disc</p>
            <h2>Background</h2><p>Sarcomeres are the smallest contractile units in striated muscle and extend in a repeating pattern along the length of a myofibril (Reedy and Beall, 1993). The ability of sarcomeres to produce muscle contractions relies on two myofibril components: thin filaments and thick filaments. Myosin thick filaments are anchored at the M-line, located at the center of the sarcomere, whereas actin thin filaments are anchored at the Z-discs, which border either side of the sarcomere. Z-discs are therefore essential to the maintenance of myofibril structure and contractility and their failure to form can result in drastic defective muscle phenotypes underlying various human myopathies (Lemke and Schnorrer, 2017).<br/><br/>As multi-protein complexes, Z-discs are a hub for various protein-protein interactions, which are key to the assembly and stabilization of Z-disc structure. The indirect flight muscles of <em>Drosophila melanogaster</em> share many features with human striated muscle and are thus an ideal model to study Z-disc protein interactions (Weitkunat and Schnorrer, 2014). Previous studies on protein interactions in <em>Drosophila</em> indirect flight muscles took advantage of biochemical methods, such as co-immunoprecipitation, or used yeast-two hybrid assays (Katzemich <em>et al.</em>, 2013; Liao <em>et al.</em>, 2016; GonzÃ¡lez-Morales <em>et al.</em>, 2017 and 2019b). However, although these methods can indicate the presence of an interaction, they do not show the location of these interactions inside cells. Furthermore, such isolated <em>ex vivo</em> systems do not consider potential disruptions and complexities that can occur within an organism and additional difficulties often arise with co-immunoprecipitation due to the differences in solubility of Z-disc proteins.<br/><br/>On the other hand, BiFC is an <em>in vivo</em> approach used not only to prove the presence of a protein-protein interaction, but also to show its location (Kerppola, 2008). BiFC relies on the reconstitution of a fluorescent molecule to indicate an interaction between two proteins (Figure 1). In this approach, the N-terminal half of a fluorescent molecule is fused to one protein of interest, known as a bait, while the C-terminal half of the fluorescent molecule is expressed with a second potentially interacting protein, known as the prey. An interaction between the bait and prey reconstitutes a detectable fluorescing molecule (Kerppola, 2008). In flies, BiFC has previously been used to detect interactions between cytoskeletal proteins (Gohl <em>et al.</em>, 2010), transcription factors (Hudry <em>et al.</em>, 2012; Altamirano-Torres <em>et al.</em>, 2018; Bischof <em>et al.</em>, 2018), and myofibril proteins (GonzÃ¡lez-Morales <em>et al.</em>, 2019b).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200329/20200329055011_7693.jpg" width="100"/><br/><strong>Figure 1. Principle of BiFC.</strong> N-terminal and C-terminal halves of a fluorescent molecule (YFP) are fused separately to two proteins of interest. An interaction between proteins results in reconstitution of a functional YFP and fluorescent signal.<br/><br/>The <em>Drosophila</em> Genomics Resource Center (DGRC) offers a big collection of cDNA clones from <em>D. melanogaster</em>. The FlyBi ORFeome collection at the DGRC contains open reading frames cloned into the pDONR223 Gateway vector, facilitating the cloning of BiFC-tagged proteins. This collection represents two thirds of the fly proteome. <br/><br/>Here, we present a detailed BiFC protocol to study protein-protein interactions in <em>Drosophila</em> myofibrils. We describe in detail the cloning process using the FlyBi ORFeome collection together with destination vectors that incorporate the BiFC-required tags (Gohl <em>et al.</em>, 2010). We will describe in detail generation of BiFC flies, as well as necessary controls. We also suggest various strategies for imaging and analysis of BiFC-marked Z-discs and propose ways in which our methodology can be extended to other myofibril structures.</p>
        </body>

<p class="pptt" id="biaoti32530">## Materials and Reagents</p><ol class="npwtwok"><li>Medium-sized pointed brush</li><li>1.5 ml microcentrifuge tube</li><li>15 ml glass test tube</li><li>Pipette tips</li><li>Fisherbrand<sup>TM</sup> Petri dishes with clear lid (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 0875713 </a> )</li><li>Subcloning efficiency<sup>TM</sup> DH5Î±<sup>TM</sup> competent cells (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 18265017 </a> )</li><li>M13 primers, M13 primers can be acquired from Alpha DNA (<a href="http://alphadna.com/" target="_blank">http://alphadna.com/</a>)</li><li>Potassium chloride (KCl) (BioShop<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> POC308 </a> )</li><li>Calcium chloride (CaCl<sub>2</sub>) (BioShop<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> CCL302 </a> )</li><li>Magnesium chloride, hexahydrate (MgCl<sub>2</sub>) (BioShop<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> MAG510 </a> )</li><li>Bacto<sup>TM</sup> tryptone (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 211699 </a> )</li><li>Bacto<sup>TM</sup> yeast extract (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 212720 </a> )</li><li>Sodium chloride (NaCl) (BioShop<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> SOD001 </a> )</li><li>Agar (BioShop<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> AGR003 </a> )</li><li>Spectinomycin dihydrochloride pentahydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S9007 </a> )</li><li>Ampicillin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A9393 </a> )</li><li>Ethyl alcohol anhydrous (Commercial Alcohols, catalog number: <a class="blackNoUnder" target="_blank"> P000EAAN </a> )</li><li>D-Glucose, anhydrous (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 200003-818 </a> )</li><li>UltraPure<sup>TM</sup> 1 M Tris-HCI, pH 8.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15568025 </a> )</li><li>Ethylenediaminetetraacetic acid (EDTA), pH 8.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9260G </a> )</li><li>Ribonuclease A from bovine pancreas (RNase) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> R4642-10MG </a> )</li><li>Sodium hydroxide (NaOH) (VWR International, catalog number: <a class="blackNoUnder" target="_blank"> BDH9292 </a> )</li><li>Sodium dodecyl sulfate (SDS) electrophoresis grade (BioShop<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> SDS001 </a> )</li><li>Potassium acetate (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP364-500 </a> )</li><li>Glacial acetic acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> ARK2183 </a> )</li><li>TE, pH 8 (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> AM9849 </a> )</li><li>Gateway<sup>TM</sup> LR Clonase<sup>TM</sup> (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 11791100 </a> )</li><li>Proteinase K solution (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> AM2548 </a> )</li><li>Invitrogen<sup>TM</sup> PureLink<sup>TM</sup> HiPure plasmid midiprep kit (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> K210004 </a> )</li><li>Phalloidin-tetramethylrhodamine B isothiocyanate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P1951 </a> )</li><li>5x KCM buffer (see Recipes)</li><li>LB broth (see Recipes)</li><li>Spectinomycin- or ampicillin-selective LB agar plates (see Recipes)</li><li>Solution 1 for small-scale preparation of plasmid DNA (100 ml) (see Recipes)</li><li>Solution 2 for small-scale preparation of plasmid DNA (4 ml) (see Recipes)</li><li>Solution 3 for small-scale preparation of plasmid DNA (100 ml) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32531">### Equipment</p><ol class="npwtwok"><li>Water bath</li><li>250 ml Erlenmeyer flask</li><li>IEC Micromax RF refrigerated microcentrifuge</li><li>VWR<sup>Â®</sup> analog vortex mixer (VWR International, catalog number: <a class="blackNoUnder" target="_blank"> 10153-838 </a> )</li><li>VWR<sup>Â®</sup> digital dry block heaters</li><li>Shaker</li><li>CO<sub>2</sub> flypad, standard size (8.1 x 11.6 cm) (Genesee Scientific, Flystuff, catalog number: <a class="blackNoUnder" target="_blank"> 59-114 </a> )</li><li>P2, P20, P100, and P1000 micro pipettes (<em>e.g.</em>, Gilson, catalog numbers: <a class="blackNoUnder" target="_blank"> F144801 </a> , <a class="blackNoUnder" target="_blank"> F123600 </a> , <a class="blackNoUnder" target="_blank"> F123615 </a> and <a class="blackNoUnder" target="_blank"> F123602 </a> )</li><li>Stereomicroscope (Leica Microsystems, model: <a class="blackNoUnder" target="_blank"> Leica MS5 </a> )</li><li>Point-scanning confocal system (Leica Microsystems, model: <a class="blackNoUnder" target="_blank"> Leica SP8 </a> )<br/></li></ol><p class="pptt" id="biaoti32532">### Software</p><ol class="npwtwok"><li>Fiji (<a href="https://fiji.sc/" target="_blank">https://fiji.sc/</a>)</li><li>R Statistics package (<a href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a>)<br/></li></ol><p class="pptt" id="biaoti32533">### Procedure</p><ol class="npwone"><li>General cloning<ol class="npwtwo"><li>Obtain open reading frame (ORF) clones of the proteins whose interactions will be tested in pDONR223 entry gateway vectors from the <em>Drosophila</em> Genomics Research Center (<a href="https://dgrc.bio.indiana.edu/" target="_blank">https://dgrc.bio.indiana.edu</a>).</li><li>Transform entry vector into DH5Î± competent bacterial cells<br/><ol class="npwthree"><li>Centrifuge entry vector at 2,100 <em>x g</em> for 1 min at 4 Â°C.</li><li>Add 20 Âµl of autoclaved water.</li><li>Vortex for 1 min.<br/></li><li>Heat at 50 Â°C on a heat block for 10 min.<br/></li><li>Keep prepared DNA on ice.<br/></li><li>Add 80 Âµl of autoclaved water, 20 Âµl of 5x KCM buffer, and 1 Âµl of prepared DNA (above) with 100 Âµl of competent DH5Î± bacterial cells in a 1.5 ml microtube.<br/></li><li>Place tube on ice for 30 min.</li><li>Heat shock cells at 42 Â°C by placing the tube in a water bath for 90 s. If a water bath is not available, any standard heating block will do as well.<br/></li><li>Add 500 Âµl of LB broth into the tube and let them recover for 1 h at 37 Â°C while shaking.<br/></li><li>Plate on spectinomycin selective LB agar plates.<br/></li><li>Incubate bacteria overnight at 37 Â°C.</li></ol></li><li>Make alkaline lysis minipreparations of the pDONR223 entry gateway vectors<br/><ol class="npwthree"><li>Make a stock of spectinomycin and LB medium in a 1:1,000 ratio.</li><li>Select one successfully transformed colony from the spectinomycin selective LB agar plates using a 200 Âµl pipet tip.</li><li>Place colony in 3 ml of the spectinomycin/LB stock in a sterile 15 ml tube and pipet up and down five times to mix.</li><li>Loosely cap the tube and incubate the culture overnight in a 37 Â°C shaker.<br/></li><li>The next day, pour 1.5 ml of the culture into a 1.5 ml microtube.<br/></li><li>Centrifuge at ~13,000 <em>x g</em> for 30 s at 4 Â°C.<br/></li><li>Discard supernatant.<br/></li><li>Add the remaining 1.5 ml of the culture into the same tube.<br/></li><li>Centrifuge at ~13,000 <em>x g</em> for 30 s at 4 Â°C.<br/></li><li>Discard supernatant. Remove all liquid from the bacterial pellet, using a pipet to aspirate any remaining liquid if necessary.<br/></li><li>Add 100 Âµl of ice-cold <em>Solution 1</em> to the bacterial pellet in the 1.5 Âµl microtube.</li><li>Vortex for 30 s.<br/></li><li>Add 200 Âµl of <em>Solution 2</em>.<br/></li><li>Mix the contents by inverting the tube rapidly five times. Do not vortex. Store tube on ice.<br/></li><li>Add 150 Âµl of ice-cold <em>Solution 3</em> and invert the tube rapidly five times.<br/></li><li>Store the tube on ice for 3-5 min.<br/></li><li>Centrifuge at ~13,000 <em>x g</em> for 5 min at 4 Â°C.<br/></li><li>Transfer supernatant to a fresh 1.5 Âµl microtube. Discard the pellet.<br/></li><li>Add 2 ml of 100% room-temperature ethanol and vortex for 30 s.<br/></li><li>Allow mixture to stand for 2 min at room temperature to precipitate the double-stranded DNA.<br/></li><li>Centrifuge at ~13,000 <em>x g</em> for 5 min at 4 Â°C.<br/></li><li>Discard the supernatant and allow the pellet to air-dry for 10 min.<br/></li><li>Rinse the pellet with 1 ml of 70% ice-cold ethanol.<br/></li><li>Remove the supernatant and allow pellet to air-dry for 10 min.<br/></li><li>Dissolve the DNA pellet in 50 Âµl of ultra-pure water.</li></ol><em>Note: Commercial column-based kits also work for Steps A3a-A3y. Use them if available.</em><br/></li><li>Confirm ORF identities by Sanger sequencing using m13 primers.</li><li>Perform Gateway cloning from pDONR223 entry vector to pUAS-BiFC destination vectors (Figure 2; Table 1).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330080927_2257.jpg" width="100"/><br/><strong>Figure 2. Gateway cloning of pDONR223 vector sequences into pUAS-BiFC vectors.</strong> pDONR223 vector confers spectinomycin resistance and contains the gene of interest between attL recombination sites. pUAS-BiFC vectors confer ampicillin resistance and contain the CcdB toxin between attB recombination sites. Recombination between the attB and attL sites mediates the transfer of the gene of interest into pUAS-BiFC.<br/><br/><strong>Table 1. Cloning vectors required to make BiFC transgenes</strong><br/><img alt="" class="layerphoto" height="192" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330081036_6203.png" width="700"/><br/><br/><ol class="npwthree"><li>Add 75 ng of pDONR223 entry vector from minipreparations (see Step A3) and 75 ng of pUAS-BiFC destination vector to a 1.5 ml microtube.</li><li>To the same tube, add TE buffer, pH 8 to 4 Âµl.</li><li>Thaw the LR Clonase<sup>TM</sup> II enzyme on ice for 2 min. Vortex the enzyme mix for 2 s.</li><li>Add 1 Âµl of LR Clonase<sup>TM</sup> II enzyme to the microtube containing the DNA mix and TE buffer. Vortex for 2 s.<br/></li><li>Incubate at 25 Â°C for 1 h.<br/></li><li>Add 0.5 Âµl of Proteinase K to the mix and vortex for 2 s.<br/></li><li>Incubate at 37 Â°C for 10 min.</li></ol></li><li>Transform pUAS-BiFC destination vectors into DH5Î± competent bacterial cells.<br/><em>Note: Procedure is identical to what has been described above with one exception: bacteria must be plated on ampicillin selective LB agar plates.</em> </li><li>Perform a midiprep of the pUAS-BiFC destination vectors from successfully transformed bacterial cells using the PureLink<sup>TM</sup> HiPure plasmid midiprep kit.<br/><ol class="npwthree"><li>In a 250 ml Erlenmeyer flask, add 100 ml of LB medium and 100 Âµl of ampicillin.</li><li>Select one successfully transformed colony containing pUAS-BiFC destination vector from the ampicillin-selective LB agar plates using a 200 Âµl pipet tip.</li><li>Add colony to the LB/ampicillin mix in the 250 ml Erlenmeyer flask and pipette up and down five times to mix.</li><li>Incubate overnight in a 37 Â°C shaker.<br/></li><li>Follow instructions as specified in the PureLink<sup>TM</sup> HiPure plasmid midiprep kit.<br/></li><li>Resuspend the purified plasmid DNA in 100 Âµl of water and place in a 1.5 ml microtube. Store at -20 Â°C.</li></ol></li><li>Confirm ORF identities by Sanger sequencing using the primers in Table 2 to confirm the insert and orientation.<br/><br/><strong>Table 2. Primers for pUAS-BiFC destination vector sequencing</strong><br/><img alt="" class="layerphoto" height="283" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330081126_1320.png" width="760"/><br/><br/></li></ol></li><li>Making transgenic lines and genetic crosses<ol class="npwtwo"><li>Transgenic flies are generated by microinjections using the Î¦C31 Integrase method. We suggest outsourcing this part to a company that handles microinjections like BestGene or Genome ProLab.</li><li>We suggest using the following landing sites to create transgenic fly lines:<br/><ol class="npwthree"><li>ZH-58A BDSC#244844: M[3xP3-RFP.attP]ZH-58A on the second chromosome.</li><li>ZH-86Fb BDSC#24749: M[3xP3-RFP.attP]ZH-86Fb on the third chromosome.</li></ol><em>Note: It is convenient to have CYFP- and NYFP-containing transgenes on different chromosomes.</em> </li><li>To activate expression of pUAS BiFC-tagged proteins, it is necessary to incorporate a Gal4 driver.<br/>Prepare a stock of each pUAS BiFC construct crossed with a Gal4-containing line (Table 3).<br/><br/><strong>Table 3. Example of some suggested experimental and control experiments.</strong> We also suggest similar experiments using N-terminally-tagged proteins of interest (<em>i.e.</em>, YFP-Protein1).<br/><img alt="" class="layerphoto" height="185" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330081220_8221.png" width="660"/><br/><br/></li><li>Set the appropriate crosses to obtain both YFP halves and the Gal4 driver in a single fly. Set crosses for the necessary controls at this time.</li><li>Example crossing scheme and some possibilities of combinations with controls are provided in Figure 3. We also suggest testing two proteins whose interactions are known as a positive control and for a baseline fluorescence measurement.<br/><br/><img alt="" class="layerphoto" height="103" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330081259_5870.jpg"/><br/><strong>Figure 3. Example crossing scheme.</strong> A. Establish stocks carrying versions of the proteins of interest with NYFP and CYFP tags. B. Incorporation of an indirect flight muscle specific Gal4 driver on the X chromosome. C. Use the established stock in B to test for interactions and negative controls.<br/><br/></li></ol></li><li>Dissection and imaging of indirect flight muscles<ol class="npwtwo"><li>Collect flies of the correct genotype.</li><li>Dissect the indirect flight muscle for imaging as described previously (Xiao <em>et al.</em>, 2017).<br/>We suggest using tetramethylrhodamine (TRITC)-conjugated phalloidin for actin staining.</li><li>Image indirect flight muscles by confocal microscopy.<br/><ol class="npwthree"><li>Use the <a class="blackNoUnder" target="_blank"> Leica SP8 </a> point-scanning confocal system. Other confocal microscopes should work as well.</li><li>Image muscle fibers using the 63x/1.4 oil objective and a pinhole of 1 airy unit (AU).</li><li>Initially locate the muscle fibers using brightfield to not bleach the sample.</li><li>Set appropriate filters for YFP and TRITC detection.<br/></li><li>Take images at 9x magnification 1,024 x 1,024 pixels (~20 Âµm x ~20 Âµm).</li></ol></li><li>Sample images are provided in Figure 4.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330081321_1037.jpg"/><br/><strong>Figure 4. Example BIFC images and plots.</strong> A. Myofibrils with positive fluorescence signal at the Z-disc (red asterisks below selected Z-discs), indicating YFP reconstitution and interaction between proteins of interest. Dotted red line shows the Plot Profile tool in ImageJ software. B. Myofibrils from a negative control sample without detectable YFP at the Z-discs (red asterisks). C. Plot profile of a single sarcomere with positive BiFC signal. D. Plot profile of several sarcomeres with positive BiFC signal. In both C and D, the intensity peak corresponds to the Z-disc. E. Plot profile of a single sarcomere with negative BiFC signal. D. Plot profile of several sarcomeres with negative BiFC signal. In C-F, the fluorescence intensity values correspond to the y-axis and Z-disc position on the x-axis. A peak of fluorescence at the Z-disc is an indication of a positive BiFC signal.<br/></li></ol></li></ol><p class="pptt" id="biaoti32534">### Data analysis</p><ol class="npwone"><li>Measure YFP intensities in Fiji/ImageJ<ol class="npwtwo"><li>Open myofibril images in Fiji/ImageJ program.</li><li>Select the straight-line tool and set line width to 30 pixels.</li><li>Draw a straight line on each Z-disc from M-line to M-line, making sure to encompass the Z-disc in the center.</li><li>Create a plot of fluorescence against distance using the Plot Profile tool, which can be accessed in the Analyze menu.</li><li>Repeat for at least thirty different Z-discs from different myofibril images per genotype.<br/><br/></li></ol></li><li>Collapse plots from several Z-discs into one using R Statistics package (R version 3.5.1).<br/><ol class="npwtwo"><li>Save all the Plot profiles from the previous step into a single folder.</li><li>Open R Statistics package</li><li>Set the folder containing all the Plot profiles as the working directory</li><li>Run the R script (Code 1).<br/></li><li>Example of plots for positive and negative BiFC signals are provided in Figure 4.</li></ol><em>Note: Code 1 aligns the intensity signals according to the highest intensity point which corresponds to the Z-disc. The alignment part of the script may result in strange plots if the BiFC signal comes from more than one place.</em><br/><br/><strong>Code 1. General R script to read Plot Profiles and collapse them into a single plot</strong><br/><span style="font-family:Courier New;">library(data.table)</span><br/><span style="font-family:Courier New;"> #Get the data</span><br/><span style="font-family:Courier New;"> setwd("Plot Profile containing folder")</span><br/><span style="font-family:Courier New;"> temp = list.files(pattern="*.csv")</span><br/><span style="font-family:Courier New;"> myfiles = lapply(temp, read.csv)</span><br/><span style="font-family:Courier New;"> a = lapply(myfiles,"[",2,drop=FALSE)</span><br/><span style="font-family:Courier New;"> a = lapply(a, scale, center=F)</span><br/><span style="font-family:Courier New;"> b=matrix(nrow = 80, ncol = length(a))</span><br/><span style="font-family:Courier New;"> #Align the profiles</span><br/><span style="font-family:Courier New;"> for (i in 1:length(a)) {</span><br/><span style="font-family:Courier New;"> x=unlist(a[i])[1:80]</span><br/><span style="font-family:Courier New;"> n=length(x)/2 - which( x == max(x))</span><br/><span style="font-family:Courier New;"> n=sqrt(n*n)</span><br/><span style="font-family:Courier New;"> s=shift(x,n=n, fill = NA,type = "lead")</span><br/><span style="font-family:Courier New;"> s[is.na(s)]=x[1:n]</span><br/><span style="font-family:Courier New;"> b[,i]=s</span><br/><span style="font-family:Courier New;"> }</span><br/><span style="font-family:Courier New;"> #Make a x/y plot</span><br/><span style="font-family:Courier New;"> plot(-1,-1,xlim = c(1,80), ylim = c(0,3), ann=FALSE)</span><br/><span style="font-family:Courier New;"> for (i in 1:length(b[1,])) {points(b[,i],pch=16,col= adjustcolor("orange", .3))}</span><br/><span style="font-family:Courier New;"> lines(rowMeans(b), type = "l", lwd=3)</span><br/><span style="font-family:Courier New;"> title(main="BiFC")</span> </li></ol><p class="pptt" id="biaoti32535">### Notes</p><ol class="npwtwok"><li>The importance of negative controls<br/>False positive signal can occur when tagged proteins bind either the CYFP or the NYFP tag. To rule this out, it is crucial that the selected proteins are tagged with CYFP and NYFP and then tested against the complementary tag alone. It is possible that the tagged protein binds only one of the YFP halves. In this scenario, the interacting protein and the interacting YFP half should be fused together. As this information is only available after the experiment is complete, to avoid wasting time, all possible combinations between YFP halves and proteins should be done in the cloning step.</li><li>BiFC during indirect flight muscle development<br/>We have only analyzed adult indirect flight muscles, but our protocol is compatible with pupa dissections. Protocols that employ GFP to mark the indirect flight muscle during development are useful to dissect the IFM during pupa stages (Kao <em>et al.</em>, 2019). Using GFP as a counter stain is not compatible with BiFC protocol, but other fluorescent proteins like Zasp52-mCherry can be used in combination with BiFC (Gonzalez-Morales <em>et al.</em>, 2019b). Importantly, protein abundance and isoform usage are highly dynamic during indirect flight muscle development (Spletter <em>et al.</em>, 2018). This should be considered when doing pupal BiFC experiments.</li><li>BiFC in other muscle types<br/>Our protocol is optimized for the analysis of Z-disc proteins in the indirect flight muscle but can be applied to other muscles as well. Protein and isoform distribution vary among muscle types. This is particularly true for the Zasp proteins in flies (Gonzalez-Morales <em>et al.</em>, 2019a). A certain interaction is therefore likely to happen in only a subset of muscles. To adjust our protocol to other muscles a different Gal4 line must be used, Mef2-Gal4 for example drives strong expression in all muscle types, Act79B-Gal4 only in jump muscle (Bryantsev <em>et al.</em>, 2012; Fernandes and SchÃ¶ck, 2014; Spletter and Schnorrer, 2014).</li><li>Use other techniques or databases to validate the results<br/>It is important to note that our BiFC protocol is based on the overexpression of tagged proteins. Muscles are particularly sensitive to relative protein concentrations (Beall <em>et al.</em>, 1989) and thus the overexpression of proteins can have a potential impact on the muscle ultrastructure and protein binding partners. Another independent method for inferring protein-protein interactions should always be used to validate the BiFC results. For the indirect flight muscle, a super-resolved atlas of protein localization using dSTORM is available and serves to estimate if the proteins of interest are in proximity to support direct binding (Szikora <em>et al.</em>, 2020). In addition, yeast two hybrid or GST pulldown experiments are good validation tools.</li><li>Compatibility of other DNA collections at the DGRC<br/>There are other large collections of plasmids publicly available that allow cloning of the BIFC tags into specific genes and isoforms (MiMIC, Gold collection, Fosmid collection, <em>etc.</em>), but the advantage here is that the ORFeome is in the pDONR223 vector that facilitate Gateway cloning. It would be possible to use other cloning approaches and plasmid collections to create BiFC-tagged proteins for isoform-specific protein tags or ORFs not found in the ORFeome collection.</li><li>Discriminate real BiFC signal from sample noise<br/>The BiFC signal comes from the reconstituted YFP that forms once both YFP halves are in proximity. While this is a powerful system that allows visualization of protein interactions in single Z-discs, the YFP signal is sometimes low enough that distinguishing real binding events from the sample noise can be challenging. However, because sarcomeres are well organized structures with well defined regions (Z-disc, M-line, A-band, <em>etc.</em>) we can compare the localization of the endogenous proteins of interest to the localization of the BiFC signal. If the endogenous proteins and the BiFC signal localize to the same structure, it likely represents a real binding event.</li><li>Mapping the domains responsible for protein interactions<br/>Identifying the residues or the protein domains that mediate the physical interaction between proteins is an important piece of information. Our BiFC setup can be used for this purpose by introducing mutations directly into the entry vector. It is important to verify the correct localization of mutated protein versions, as a decrease in BiFC signal may come from a mislocalization problem. This can easily be tested using Myc or HA tags in the destination vectors.</li><li>The problems or negative results when interpreting BiFC results<br/>Even in the complete absence of detectable BiFC signal it is nearly impossible to completely rule out binding between a pair of proteins based only on BiFC. The addition of the BiFC tags may disrupt protein-protein interaction by steric hindrance or by causing the misfolding of proteins. In addition, even if the proteins are bound together, the two YFP halves must be in proximity for reconstituting a functional YFP. If one of the proteins is rather big, the distance between tags might be large enough to prevent YFP reconstitution. Testing all combinations between N-terminal and C-terminal tags might be required.<br/></li></ol><p class="pptt" id="biaoti32536">### Recipes</p><ol class="npwtwok"><li>5x KCM buffer<br/>5.1 ml 1 M KCl<br/>1.5 ml 1 M CaCl<sub>2</sub><br/>2.5 ml 1 M MgCl<sub>2</sub><br/>0.9 ml of ddH<sub>2</sub>O</li><li>LB broth<br/><ol class="npwthree"><li>Dissolve in 1 L of ddH<sub>2</sub>O:<br/>50 g of Bacto<sup>TM</sup> tryptone<br/>25 g of Bacto<sup>TM</sup> yeast extract<br/>50 g of NaCl<br/></li><li>Autoclave</li></ol></li><li>Spectinomycin- or ampicillin-selective LB agar plates<br/><ol class="npwthree"><li>Prepare spectinomycin 50 mg/ml and ampicillin 100 mg/ml stock solutions</li><li>Dissolve in 1 L of ddH<sub>2</sub>O:<br/>50 g of Bacto<sup>TM</sup> tryptone<br/>25 g of Bacto<sup>TM</sup> yeast extract<br/>50 g of NaCl<br/>75 g of agar<br/></li><li>Autoclave</li><li>Add 1 ml of spectinomycin or ampicillin stock solution (1:1,000 concentration)<br/></li><li>Pour medium into Petri dishes and allow to solidify<br/></li><li>Store, covered at 4 Â°C</li></ol></li><li>Solution 1 for small-scale preparation of plasmid DNA (makes 100 ml)<br/>5 ml of 1 M glucose (final concentration is 50 mM of glucose)<br/>2.5 ml of 1 M Tris-HCl pH 8.0 (final concentration is 25 mM of Tris-HCl)<br/>2 ml of 500 mM EDTA pH 8.0 (final concentration is 10 mM of EDTA)<br/>90.5 ml of ddH<sub>2</sub>O<br/>2 g of RNase A (20 mg/ml)</li><li>Solution 2 for small-scale preparation of plasmid DNA (makes 4 ml)<br/>160 Âµl of 5 M NaOH (final concentration is 0.2 M of NaOH)<br/>400 Âµl of 10% SDS (final concentration is 1% SDS)<br/>3.44 ml of ddH<sub>2</sub>O</li><li>Solution 3 for small-scale preparation of plasmid DNA (makes 100 ml)<br/>60 ml of 5 M potassium acetate<br/>11.5 ml of glacial acetic acid<br/>28.5 ml of H<sub>2</sub>O<br/></li></ol><p class="pptt" id="biaoti32537">### Acknowledgments</p><p class="ppzz">We thank Beili Hu for help with <em>Drosophila</em> injections, the CIAN imaging facility for help with confocal microscopy, the <em>Drosophila</em> Genomics Resource Center, and Sven Bogdan. This work was supported by operating grant MOP-142475 from the Canadian Institutes of Health Research. We have no conflicts of interest to declare.</p><p class="pptt" id="biaoti32538">### Competing interests</p><p class="ppzz">We declare no competing interests.</p><p class="pptt" id="biaoti32539">### References</p><ol class="npwtwok"><li>Altamirano-Torres, C., Salinas-Hernandez, J. E., Cardenas-Chavez, D. L., Rodriguez-Padilla, C. and Resendez-Perez, D. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30321227" target="_blank">Transcription factor TFIIEbeta interacts with two exposed positions in helix 2 of the Antennapedia homeodomain to control homeotic function in <em>Drosophila</em>.</a> <em>PLoS One</em> 13(10): e0205905.</li><li>Beall, C. J., Sepanski, M. A. and Fyrberg, E. A. (1989). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2714648" target="_blank">Genetic dissection of <em>Drosophila</em> myofibril formation: effects of actin and myosin heavy chain null alleles.</a> <em>Genes Dev</em> 3(2): 131-140.</li><li>Bischof, J., Duffraisse, M., Furger, E., Ajuria, L., Giraud, G., Vanderperre, S., Paul, R., Bjorklund, M., Ahr, D., Ahmed, A. W., Spinelli, L., Brun, C., Basler, K. and Merabet, S. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30247122" target="_blank">Generation of a versatile BiFC ORFeome library for analyzing protein-protein interactions in live <em>Drosophila</em>.</a> <em>Elife</em> 7: 38853.</li><li>Bryantsev, A. L., Baker, P. W., Lovato, T. L., Jaramillo, M. S. and Cripps, R. M. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22008792" target="_blank">Differential requirements for myocyte enhancer factor-2 during adult myogenesis in <em>Drosophila</em>.</a> <em>Dev Biol</em> 361(2): 191-207.</li><li>Fernandes, I. and SchÃ¶ck, F. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25113030" target="_blank">The nebulin repeat protein Lasp regulates I-band architecture and filament spacing in myofibrils.</a> <em>J Cell Biol</em> 206(4): 559-572.</li><li>Gohl, C., Banovic, D., Grevelhorster, A. and Bogdan, S. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20937809" target="_blank">WAVE forms hetero- and homo-oligomeric complexes at integrin junctions in <em>Drosophila</em> visualized by bimolecular fluorescence complementation.</a> <em>J Biol Chem</em> 285(51): 40171-40179.</li><li>GonzÃ¡lez-Morales, N., Holenka, T. K. and SchÃ¶ck, F. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28732005" target="_blank">Filamin actin-binding and titin-binding fulfill distinct functions in Z-disc cohesion.</a> <em>PLoS Genet</em> 13(7): e1006880.</li><li>GonzÃ¡lez-Morales, N., Marsh, T. W., Katzemich, A., Marescal, O., Xiao, Y. S. and SchÃ¶ck, F. (2019a) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31123042" target="_blank">Different evolutionary trajectories of two insect-specific paralogous proteins involved in stabilizing muscle myofibrils.</a> <em>Genetics</em> 212(3): 743-755.</li><li>GonzÃ¡lez-Morales, N., Xiao, Y. S., Schilling, M. A., Marescal, O., Liao, K. A. and SchÃ¶ck, F. (2019b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31746737" target="_blank">Myofibril diameter is set by a finely tuned mechanism of protein oligomerization in <em>Drosophila</em>.</a> <em>Elife</em> 8: 50496.</li><li>Hudry, B., Remacle, S., Delfini, M. C., Rezsohazy, R., Graba, Y. and Merabet, S. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22745600" target="_blank">Hox proteins display a common and ancestral ability to diversify their interaction mode with the PBC class cofactors.</a> <em>PLoS Biol</em> 10(6): e1001351.</li><li>Kao, S. Y., Nikonova, E., Ravichandran, K. and Spletter, M. L. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31680668" target="_blank">Dissection of <em>Drosophila</em> melanogaster flight muscles for omics approaches.</a> <em>J Vis Exp</em> (152).</li><li>Katzemich, A., Liao, K. A., Czerniecki, S. and SchÃ¶ck, F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23505387" target="_blank">Alp/Enigma family proteins cooperate in Z-disc formation and myofibril assembly.</a> <em>PLoS Genet.</em> 9(3): e1003342.</li><li>Kerppola, T. K. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18573091" target="_blank">Bimolecular fluorescence complementation (BiFC) analysis as a probe of protein interactions in living cells.</a> <em>Annu Rev Biophys</em> 37: 465-487.</li><li>Lemke, S. B. and Schnorrer, F. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27913119" target="_blank">Mechanical forces during muscle development.</a> <em>Mech Dev</em> 144(Pt A): 92-101.</li><li>Liao, K. A., GonzÃ¡lez-Morales, N. and SchÃ¶ck, F. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27783625" target="_blank">Zasp52, a Core Z-disc protein in <em>Drosophila</em> indirect flight muscles, interacts with Î±-actinin via an extended PDZ domain.</a> <em>PLoS Genet</em> 12(10): e1006400.</li><li>Reedy, M. C. and Beall, C. (1993). <a href="https://www.ncbi.nlm.nih.gov/pubmed/8253277" target="_blank">Ultrastructure of developing flight muscle in <em>Drosophila</em>. I. assembly of myofibrils.</a> <em>Dev Biol</em> 160(2): 443-465.</li><li>Spletter, M. L. and Schnorrer, F. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24145055" target="_blank">Transcriptional regulation and alternative splicing cooperate in muscle fiber-type specification in flies and mammals.</a> <em>Exp Cell Res</em> 321(1): 90-98.</li><li>Spletter, M. L., Barz, C., Yeroslaviz, A., Zhang, X., Lemke, S. B., Bonnard, A., Brunner, E., Cardone, G., Basler, K., Habermann, B. H. and Schnorrer, F. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29846170" target="_blank">A transcriptomics resource reveals a transcriptional transition during ordered sarcomere morphogenesis in flight muscle.</a> <em>Elife</em> 7: 34058.</li><li>Szikora, S., Gajdos, T., NovÃ¡k, T., Farkas, D., FÃ¶ldi, I., Lenart, P., ErdÃ©lyi, M. and MihÃ¡ly, J. (2020). <a href="https://doi.org/10.1083/jcb.201907026" target="_blank">Nanoscopy reveals the layered organization of the sarcomeric H-zone and I-band complexes.</a> <em>J Cell Biol</em> 219(1): e201907026.</li><li>Weitkunat, M. and Schnorrer, F. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24625467" target="_blank">A guide to study <em>Drosophila</em> muscle biology.</a> <em>Methods</em> 68(1): 2-14.</li><li>Xiao, Y. S., SchÃ¶ck, F. and GonzÃ¡lez-Morales, N. (2017). <a href="https://bio-protocol.org/e2606" target="_blank">Rapid IFM dissection for visualizing fluorescently tagged sarcomeric proteins.</a> <em>Bio-protocol</em> 7(22): e2606.<br/></li></ol>