<body>
            <h1>Preparation, FPLC Purification and LC-FT-ICR-MS of Proteins</h1>
            <p><strong>Authors:</strong> Tyler B. Johnson, Jiri Adamec, Paul Blum</p>
            <h2>Abstract</h2>
            <p>High magnetic field Fourier transform ion cyclotron resonance (FT-ICR) mass spectrometers provide extremely high mass resolution (resolving power of ~200,000 at 400 m/z) protein detection across a broad mass range, enabling analysis of fine structure of isotopic peak clusters that is missed in other types of mass spectrometers. The protocol detailed here describes preparation of cellular extracts for purification of DNA-binding proteins using multiple chromatographic chemistries via fast protein liquid chromatography (FPLC), and identification and quantitation of the protein isoforms and their post-translational modifications by liquid chromatography coupled to Fourier transform ion cyclotron resonance mass spectrometry (LC-FT-ICR-MS). This protocol benefits from selectively purifying proteins for identification and quantitation by high resolution FT-ICR, which has the resolution to definitively distinguish between acetylation and trimethylation post-translational modification (PTM) additions.</p>
            <p><strong>Keywords:</strong> FT-ICR, Magnetic resonance mass spectrometry, Intact mass, Post-translational modifications, Chromatin protein</p>
            <h2>Background</h2><p>Post-translational modifications (PTMs) of proteins play very important roles in regulation of cellular processes and function. The characterization of PTMs is challenging because of the large number of potential modifications and number of amino acid residues that can be modified. This can be particularly true for proteins with a limited number of proteolytic sites that cannot be analyzed in proteolytically digested forms and require an intact protein MS analysis. To achieve this goal, this protocol describes a combination of protein separation by HPLC and high accuracy/resolution mass spectrometry. The HPLC separation helps to; 1) eliminate various salts and other interfering molecules from the protein of the interest, 2) separate various PTM isoforms of the protein, and 3) improve ionization of the protein. A high accuracy/resolution MS instrument is, on the other hand, necessary to identify specific charge envelopes and isotope pattern distribution that are the important features essential for an accurate PTM determination. Although results of the intact protein MS analysis depend on many factors including a molecular mass, amino acid composition and tertiary structure of proteins, usually it can only be achieved by two types of MS instruments: Fourier transform ion cyclotron resonance mass spectrometer (FT-ICR-MS) or recently developed generations of the Orbitrap-based mass spectrometer. Of those, FT-ICR-MS is preferred as data can be acquired at higher resolution and accuracy. On the other hand, this instrument requires liquid helium to keep the magnet temperature low which increases maintenance cost. Electrospary Ionization (ESI) used in mass spectrometry of intact proteins generates many different charge states of the protein. This represents a potential problem in detection of PTMs of the protein as peaks corresponding to individual proteoforms must be resolved. Therefore, only mass spectrometers with high resolving power such as FT-ICR-MS can be used. Depending on a magnetic field, they can have the resolving power at 1,000,000 and still accuracy between 0.05 and 1 ppm.</p>
        </body>

<p class="pptt" id="biaoti32542">## Materials and Reagents</p><ol class="npwtwok"><li>Nitrile gloves</li><li>15-ml Amicon centrifugal filter units 3 kDa MWCO (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> UFC900308 </a> )</li><li>Pipette tips 1,000 Âµl, 200 Âµl, 20 Âµl, 2 Âµl</li><li>Serological pipets</li><li>1.5 ml Microcentrifuge tubes</li><li>1.7 ml low retention microtubules (PHENIX Research Products, catalog number: <a class="blackNoUnder" target="_blank"> MAX-715L </a> )</li><li>15 ml conical tubes</li><li>Disposable borosilicate glass tubes (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14-961-29 </a> )</li><li>30 ml syringe</li><li>10 ml syringe</li><li>60 ml syringe</li><li>0.45 Âµm filtration apparatus</li><li>0.45 Âµm mixed cellulose ester gridded filters (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> HABG04700 </a> )</li><li>1-ml HiTrap SP HP prepacked columns (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 17115101 </a> )</li><li>1-ml HiTrap Heparin HP prepacked columns (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 17040601 </a> )</li><li>Spectra/Por 3 RC Dialysis Membrane Tubing 3500 Dalton MWCO (Spectrum, catalog number: <a class="blackNoUnder" target="_blank"> 132724 </a> )</li><li>Reverse phase HPLC column (Thermo Scientific, ProSwift RP-10R 1 mm x 50 mm, catalog number: <a class="blackNoUnder" target="_blank"> 164586TS </a> )</li><li>20% ethanol</li><li>Sodium phosphate, monobasic (NaH<sub>2</sub>PO<sub>4</sub>)</li><li>0.5 M EDTA</li><li>DL-dithiothreitol, anhydrous (DTT) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D9779 </a> )</li><li>Potassium chloride (KCl) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 31248 </a> )</li><li>HPLC-grade water (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 270733 </a> )</li><li>0.1% formic acid in water (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 576913 </a> )</li><li>0.1% formic acid in acetonitrile (CH<sub>3</sub>CN) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 576956 </a> )</li><li>Ammonium sulfate ((NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>)</li><li>Sodium hydroxide (NaOH) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 1.06462 </a> )</li><li>Hydrochloric acid (HCl) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> H1758 </a> )</li><li>4-20% Mini-PROTEAN TGX Precast Protein Gels (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 4561096 </a> )</li><li>50 mM Ammonium bicarbonate (NH<sub>4</sub>HCO<sub>3</sub>) pH 8.0</li><li>Buffer A (see Recipes)</li><li>Buffer B (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32543">### Equipment</p><ol class="npwtwok"><li>Pipettes (P1000, P200, P20, P2)</li><li>Sonicator (Qsonica, Q500 sonicator, catalog number: <a class="blackNoUnder" target="_blank"> Q500-110 </a> )</li><li>Centrifuge (Sorvall RC-6+)</li><li><a class="blackNoUnder" target="_blank"> ÃKTA pure FPLC system </a> (or similar liquid chromatography system) <a href="https://www.gelifesciences.com/en/gb/shop/chromatography/chromatography-systems/akta-pure-p-05844" target="_blank">https://www.gelifesciences.com/en/gb/shop/chromatography/chromatography-systems/akta-pure-p-05844</a> </li><li>High pressure liquid chromatography capillary pump (Agilent, 1100 Series HPLC, model: <a class="blackNoUnder" target="_blank"> CapPump/G1376A </a> )</li><li>Autosampler (Agilent, 1200 Series, model: <a class="blackNoUnder" target="_blank"> MicroWPS/G1377A </a> )</li><li>Degasser (Agilent, 1200 Series, model: <a class="blackNoUnder" target="_blank"> Degasser/G1379B </a> )</li><li>Fourier-transform ion cyclotron resonance mass spectrometer (Bruker, 7T solariX 70 Hybrid FTMS System, model: <a class="blackNoUnder" target="_blank"> 277070 </a> )<br/></li></ol><p class="pptt" id="biaoti32544">### Software</p><ol class="npwtwok"><li>UNICORN 7 control software for ÃKTA pure FPLC system</li><li>HyStar 3.4 software (Bruker)</li><li>Compass SolariXcontrol (Bruker) software</li><li>Compass Data Analysis software version 4.0 (Bruker, <a href="https://www.bruker.com/service/support-upgrades/software-downloads/mass-spectrometry.html" target="_blank">https://www.bruker.com/service/support-upgrades/software-downloads/mass-spectrometry.html</a>)</li><li>IDCalc software (MacCoss Lab, <a href="https://proteome.gs.washington.edu/software/IDCalc/" target="_blank">https://proteome.gs.washington.edu/software/IDCalc/</a>)<br/></li></ol><p class="pptt" id="biaoti32545">### Procedure</p><p class="ppzz">A workflow of the protocol from preparation of cellular extracts to identification and quantitation of the protein isoforms and their post-translational modifications by FT-ICR-MS is shown in Figure 1.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330094222_6553.jpg" width="100"/><br/><strong>Figure 1. Schematic workflow of the steps of this protocol</strong> </p><ol class="npwone"><br/><li>Protein sample preparation<ol class="npwtwo"><li>Following culturing and harvesting of desired cell line, resuspend cells in 10 ml of Buffer A in conical tube.</li><li>Place cells on ice and lyse cellular suspension by sonication, alternating on/off pulses at 30 s (30 s on/30 s off) until cells are fully lysed as indicated by their appearance using a light microscope.</li><li>Clarify lysates by centrifugation at 10,000 <em>x g </em>for 15 min at 4 Â°C.</li><li>Transfer supernatant to a separate conical tube and add ammonium sulfate to final concentration of 70%.<br/><em>Note: A useful calculator for determining volume can be found at </em><a href="https://www.encorbio.com/protocols/AM-SO4.htm" target="_blank"><em> https://www.encorbio.com/protocols/AM-SO4.htm</em></a><em>.</em> </li><li>Incubate solution with constant agitation at 4 Â°C for at least one hour.</li><li>Pellet precipitated protein via centrifugation at 10,000 <em>x g </em>at 4 Â°C.</li><li>Resuspend pellet in 10 ml Buffer A and filter through 0.45 Âµm filter.</li><li>Store at 4 Â°C until ready to inject into sample loop for purification or until mass spectrometry if purification is not being pursued.<br/><em>Note: If going straight to mass spectrometry, resuspend the suspension in 50 mM ammonium bicarbonate pH 8.0 (or MS compatible buffer of your choosing) instead of Buffer A.</em><br/><br/></li></ol></li><li>FPLC method creation and setup<ol class="npwtwo"><li>Machine Pre-startup<br/><ol class="npwthree"><li>Check all the volumes of all buffers being used during FPLC run to ensure no air enters lines/pumps during run.<br/><em>Note: All buffers must be a minimum of 300 ml.</em><br/>50-ml Falcon tube with 20% ethanol for back-rinsing solution (lower right corner of ÃKTA machine)<br/>Filtered nano-pure water<br/>20% ethanol<br/>Buffer A/B<br/></li><li>Make sure the correct lines are in the correct buffers.<br/>A1 = Purification Buffer A<br/>A2 = Nano-pure water (filtered)<br/>B1 = Purification Buffer B<br/>B2 = 20% ethanol (filtered)<br/></li><li>Empty waste container prior to each run to prevent over filling and back pressure on the system.</li></ol></li><li>Priming inlets and purging pump heads<br/><ol class="npwthree"><li>Turn on ÃKTA Pure FPLC and Allow a short time for pumps and lamp to initialize for ~30 s.</li><li>Make sure that all inlet tubing that is to be used during the method run is placed in the correct buffer.</li><li>In the Process Picture of the Open System Control module:<br/><ol class="npwfour"><li>Click on the buffer inlets.</li><li>Select the position of the inlet to be filled. Select the positions in reverse alphabetical order and start with the highest number. For example, if all the four inlets in the Inlet valve AB are to be filled, fill them in the following order B2, B1, A2, A1.</li></ol></li><li>Connect a 60 ml syringe to the purge valve of one of the pump heads of the pump that is being prepared (see page 91 in ÃKTA Pure Operating Instructions guide). Make sure that the syringe fits tightly into the purge connector.<br/><ol class="npwfour"><li>Open the purge valve by turning it counter-clockwise about three quarters of a turn. Draw liquid slowly into the syringe until the liquid reaches the pump.</li><li>Close the purge valve by turning it clockwise. Disconnect the syringe and discard its contents.</li></ol></li><li>Repeat Steps B2c-B2d for each piece of inlet tubing that is to be used during the run.</li></ol></li><li>Creating a custom method in Method Editor<br/><ol class="npwthree"><li>Open the Method Editor window.</li><li>Click [Create New Method] â Select [Machine Number] â Check [Predefined Application] and choose application for affinity, ion exchange, <em>etc.</em> chromatography or [Empty Method] if you prefer to define your own parameters from scratch â Click [OK]. Predefined Application is the suggested choice for method creation.</li><li>Method Settings:<br/><ol class="npwfour"><li>Choose column from list or leave as âanyâ.<br/>If set at âanyâ make sure to determine the column pressure limits and set in this window accordingly.<br/></li><li>Choose the correct column position that you will be using in the column valve.<br/>This is usually set as â1â unless using multiple chromatographies.<br/></li><li>Choose correct buffer inlets<br/>A1 and B1 are default and typically used for buffers A and B.</li></ol></li><li>Equilibration:<br/><ol class="npwfour"><li>Choose total volume (Column volume = CV) or condition for equilibration.<br/>Set equilibration to 10 CV buffer A or stable UV for 15 s, either method is fine.<br/></li><li>Choose the % buffer B for equilibration.<br/>Set at 0%</li></ol></li><li>Sample Application:<br/><ol class="npwfour"><li>Choose loop type (<em>i.e.</em>, Superloop or Capillary loop).</li><li>Choose loop position on loop valve (1-5).</li><li>Set total sample volume that will be injected (1-99 ml).</li><li>Choose fractionation desired:<br/>Fraction collector for collection of elution fractions.<br/></li><li>If fraction collector is chosen set the volume of fractionation desired.</li></ol></li><li>Column Wash:<br/>Wash column either by volume or condition stable UV.<br/>Stable UV is preferred to ensure that all unbound proteins are washed from the column.</li><li>Elution<br/><ol class="npwfour"><li>Set Elution method to Gradient elution.<br/>Linear = set target %B and length of gradient in CV<br/>Target %B should be set to 100 and CV set to 25<br/></li><li>Set fractionation to fraction collector.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Set type of fractionation to fixed volume.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Set fractionation volume to 1 ml.</div></div></li><li>Optional: Up-flow elution can be selected if you wish the sample to be eluted from the bottom of the column to the top.</li></ol></li><li>Equilibration:<br/>Standard set to equilibrate the system and column back to buffer A.</li></ol></li><li>FPLC Run Initialization<br/><ol class="npwthree"><li>Connecting a column (see page 109 in ÃKTA Pure Operating Instructions guide):<br/><ol class="npwfour"><li>Connect a suitable tubing to a Column valve port (Example: 1A).</li><li>Open the System Control module.</li><li>In the Process Picture:<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Click on the [Column] â Select [Column down flow].</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Click on the [Pumps] â Enter a low System flow (<em>e.g.</em>, 0.25 ml/min.) â Click [Set flow rate].</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">When buffer leaves the tubing in a continuous mode and the top part of the column is filled with buffer, connect the tubing to the top of the column.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">When buffer leaves the tubing at the bottom of the column in a continuous mode, connect the column to the Column valve. Use the port opposite to the one already connected to the column, in this example port 1B if 1A is used.</div></div><em>Note: Make sure that the column location matches the location set in the method that was created for the run.</em> </li></ol></li><li>Sample injection:<br/><ol class="npwfour"><li>Sample injection using a Superloop (see page 122 in ÃKTA Pure Operating Instructions guide):<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Attach the Superloop to the instrument using a Column holder.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">In the Process Picture, click the [Injection valve] â Select [Manual Load].</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">In the Process Picture, click the [Loop valve] â Select the position the loop is connected to, for example [Position 2].</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">Fill a syringe with your filtered protein sample from Step A7.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-0.5px;"><div class="float-left">5)</div><div class="media-body">Approximately 0.5 Î¼g of protein sample was used.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-0.5px;"><div class="float-left">6)</div><div class="media-body">Connect the syringe to Injection valve port (Syr on V9-Inj of ÃKTA machine).</div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-0.5px;"><div class="float-left">7)</div><div class="media-body">Load sample into the Superloop by emptying the syringe into the injection valve.<br/><em>Note: Leave the syringe connected to the Syr port until the end of the run.</em> </div></div></li><li>Open the Method Editor module and open the method you will be using for the elution run.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">In Method settings, make sure to set column position that you have your column connected to, <em>e.g.</em>, 1.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">In Sample Application, set your total sample volume for injection and the loop type and position.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">Select the method to run and click the [Run] icon.</div></div></li><li>Collect fractions utilizing fraction collector filled with disposable glass tubes<br/><br/></li></ol></li></ol></li></ol></li><li>Sample preparation for subsequent chromatographic runs<ol class="npwtwo"><li>Combine peak fractions together and transfer to 15-ml Amicon centrifugal filtration units with molecular weight cut-off of 3.5 kDa.<br/><em>Note: Spin samples based on manufacturerâs guidelines until sample is concentrated.</em> </li><li>Fill filtration column with Buffer A and repeat centrifuge step as above.<br/><em>Note: Repeat this step 5-10 times.</em> </li><li>Pipet out protein sample from on top of the filter, transfer to a new collection tube and add enough volume of Buffer A to fill your chosen sample loop for additional chromatography runs.<br/>10 ml if using the 10 ml Superloop.</li><li>Optional: Perform second round of cation exchange chromatography this time using a shallow gradient <br/><em>Note: Create identical method except elute over 45 ml as opposed to 25 as completed for first round of Cation exchange chromatography for better separation of elution peaks.</em> </li><li>Repeat Steps C1-C3 for following elution of protein for final round of Heparin chromatography.</li><li>Run the same Elution method that was used for round 1 cation exchange chromatography (<em>e.g.</em>, 25 CV elution gradient).<br/>See Figure 2 for representative chromatograph from heparin affinity run.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330094323_1390.jpg"/><br/><strong>Figure 2. FPLC separation of chromatin proteins by heparin affinity chromatography.</strong> Proteins are fractionated by heparin affinity chromatography and elution is monitored by absorbance at 280 nm.<br/><br/></li><li>Save individual peak fractions for MS analysis of intact protein via FT-ICR-MS.<br/>You may choose to concentrate fractions to higher protein concentrations to accommodate mass spectrometry methods.<br/><ol class="npwthree"><li>Repeat Step C1-C3 for following elution of protein from heparin to exchange buffer into a MS-compatible buffer (<em>i.e.</em>, 50 mM ammonium bicarbonate).</li><li>Optional: You may choose to vacuum dry protein samples for reconstitution into buffer of choice as an alternative to diafiltration. It may, however, be necessary to remove salts prior to vacuum drying to prevent salt crystal formation.</li></ol></li><li>Analyze the protein composition of the via SDS-PAGE to determine respective masses of purified proteins in individual FPLC peaks.<br/>See Figure 3 for representative image of protein content in peaks in Figure 2.<br/><em>Note: In-gel analysis is for qualitative evaluation of protein content in respective peaks. All peaks are analyzed as intact, undigested proteins in-solution for subsequent FT-ICR-MS analysis.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330094612_8924.jpg" width="100"/><br/><strong>Figure 3. SDS-PAGE analysis of purified native chromatin extracts following heparin affinity chromatography.</strong> Eluted individual peak fractions from heparin resins were analyzed for protein identity prior to MS identification.<br/><br/></li></ol></li><li>LC-FT-ICR Mass Spectrometry<ol class="npwtwo"><li>Creating a Mass Spectrometry custom method in Method Editor.<br/><ol class="npwthree"><li>Turn on computer and open Compass SolariXcontrol software.<br/><ol class="npwfour"><li>Software will upload the last method used by the instrument.</li><li>Open [Method Editor] window.</li></ol></li><li>In the Method Editor window click [Ion Transfer].<br/><ol class="npwfour"><li>In [Source Optics] set parameters to:<br/>Capillary Exit to 180 V<br/>Deflector Plate to 280 V<br/>Funnel 1 to 110 V<br/>Skimmer 1 to 30 V<br/>Funnel RF Amplitude to 210 Vpp<br/></li><li>In [Octopole] set parameters to:<br/>Frequency to 5 VMHz<br/>RF Amplitude to 150 Vpp<br/></li><li>In [Quadrupole] parameters set:<br/>Q1 mass to 250 m/z<br/></li><li>In [Collision Cell] set parameters to:<br/>Collision Voltage to -1.5 V<br/>DC Extract Bias to 0.6 V<br/>RF Frequency 2 MHz<br/>Collision RF Amplitude to 1,500 Vpp</li><li>In [Transfer Optics] set parameters to:<br/>Time of Flight to 1 ms<br/>Frequency 4 MHz<br/>RF Amplitude to 440 Vpp</li><li>In [Gas Control] set parameters to:<br/>Flow 15%<br/>Check [Enable] box<br/></li></ol></li><li>In the Method Editor window click [API Source].<br/><ol class="npwfour"><li>In [API Source] set parameters to:<br/>Source type to ESI<br/>Capillary is set to 4,500 V<br/>End Plate Offset to -500 V<br/></li><li>In [Source Gas Tune] set parameters to:<br/>Check [Dry Gas Flow] box<br/>Check [Dry Gas Heater] box<br/>Check [Nebulizer Gas Flow] box<br/></li><li>In [Divert Valve] parameters set:<br/>Check [Position] to Waste</li></ol></li><li>In the Method Editor window click [Mode].<br/><ol class="npwfour"><li>In [Acquisition Mass Control] set parameters to:<br/>Detection Mode to Broadband<br/>TD (Acquisition) to 2 M<br/>Low Mass to 150.59 m/z<br/>High Mass to 3,000.00 m/z<br/></li><li>In [Accumulation] set parameters to:<br/>Average Spectra to 1<br/>Source Accumulation to 0.010 s<br/>Ion Accumulation Time to 0.800 s<br/>Ion Cooling Time to 0.010 s<br/>Time of Flight to 1 ms</li></ol></li><li>In the Method Editor window click [Analyzer].<br/>Set parameters to:<br/>Transfer Exit Lens to -16 V<br/>Analyzer Entrance to -10 V<br/>Side Kick to 7 V<br/>Side Kick Offset to -1.5 V<br/>Front Trap Plate to 1 V<br/>Back Trap Plate to 0.9 V<br/>Sweep Excitation Power to 15%</li><li>Save method as [nameMS].<br/></li></ol></li><li>Creating a Chromatography custom method in Method Editor<br/><ol class="npwthree"><li>Turn on computer and open HyStar 3.4 software.</li><li>Click [Method] â Select [New Method] if you prefer to define your own parameters from scratch â Click [OK] or open any Method and Click [Edit].</li><li>In the Method Editor window click [General].<br/><ol class="npwfour"><li>In [Author] write your name</li><li>In [Description] describe your method</li></ol></li><li>In the Method Editor window click [LC Parameters].<br/><ol class="npwfour"><li>In [Total Runtime] set parameters to:<br/>Total Runtime to 90 min<br/></li><li>In [Data Acquisition] set parameters to:<br/>Start delay to 0 min<br/>Runtime to 90 min<br/>Check [Dry Gas Heater] box<br/></li><li>In [LC Pump] set parameters to:<br/>Flow Rate 0.02 ml/min<br/>Pressure limits to Min 0 and Max 400 bar<br/>Solvent to 7<br/>Solvent A1 to 100%<br/>Solvent B1 to 0%</li></ol></li><li>In the Method Editor window click [LC Timetable].<br/><ol class="npwfour"><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 0 min<br/></li><li>In [Function] drop down menu select [Flow Rate].<br/>Set Flow Rate to 0.02 ml/min<br/></li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 0 min<br/></li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 100% A1 and 0% B1<br/></li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 1 min<br/></li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 100% A1 and 0% B1<br/></li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 75 min<br/></li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 20% A1 and 80% B1</li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 78 min</li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 0% A1 and 100% B1</li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 79 min</li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 0% A1 and 100% B1</li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 80 min</li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 100% A1 and 0% B1</li><li>In [Device] drop down menu select [Pump-Agilent G1376A Micro].<br/>Set [Time] to 90 min</li><li>In [Function] drop down menu select [Solvent Mix].<br/>Set Solvent Mix to 100% A1 and 0% B1<br/></li></ol></li><li>In the Method Editor window click [Signals 1-8]<br/><ol class="npwfour"><li>From [Signal No. 1] select MS (apex series) MS and Interval BPC, All</li><li>From [Signal No. 2] select MS (apex series) MS and Interval TIC, All</li></ol></li><li>Save method as [nameLC]<br/>See Figure 4A for representative HPLC spectrum.</li></ol></li><li>LC-FT-ICR-MS data acquisition<br/><ol class="npwthree"><li>Turn on computer and open Compass SolariX Control and HyStar 3.4 software.</li><li>Open [nameMS] Method in Compass SolariX Control software.</li><li>Open [nameLC] Method in HyStar 3.4 software.</li><li>Check all the volumes of all buffers being used during HPLC run to ensure no air enters lines/pumps during run.<br/><em>Note: All solvents must be a minimum of 100 ml.</em> </li><li>Make sure the correct lines are in the correct solvents<br/>A1 = Solvent A (0.1% formic acid in water)<br/>B1 = Solvent B (0.1% formic acid in acetonitrile)</li><li>Empty waste container prior to each run to prevent over filling and back pressure on the system.</li><li>Initializing Agilent 1100 HPLC system.<br/><ol class="npwfour"><li>Turn on the Agilent 1100 HPLC system (right hand side).</li><li>Allow a short time for pumps and lamp to initialize (~30 s).</li><li>Ensure that the machine and computer are communicating.</li></ol></li><li>Priming inlets and purging pump heads.<br/><ol class="npwfour"><li>Open the System Control module.</li><li>In the Process Picture at the bottom of the window:<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Click on [Purge].</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Select the position of the inlet to be filled. Select the positions in reverse alphabetical order and start with the highest number. For example, if all the four inlets in the Inlet valve AB are to be filled, fill them in the following orderL B2, B1, A2, A1.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">Click on [Start Purge].</div></div></li></ol></li><li>Connect a ProSwift column (1 x 50 mm) between Agilent LC system outlet and MS instrument inlet.</li><li>Place sample into a correct position in Agilent 1200 serie Autosampler (<em>e.g.</em>, 1-B,1).</li><li>Add sample to Sample Table.<br/><ol class="npwfour"><li>Click [Sample Table] in HyStar 3.4 software.</li><li>Click on [Add new sample].<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">In [Vial] box define position of your sample in the Autosampler (<em>e.g.</em>, 1-B,1).</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">In [Sample ID] define name of your sample.</div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">Set [Injection] to 1.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">Set [Volume] to 5 Âµl.</div></div><div class="media media-x cnautokh cnautokh5" style="margin-top:-0.5px;"><div class="float-left">5)</div><div class="media-body">In [Data path] define where results will be stored.</div></div><div class="media media-x cnautokh cnautokh6" style="margin-top:-0.5px;"><div class="float-left">6)</div><div class="media-body">For [LC Method Part] box select [nameLC].</div></div><div class="media media-x cnautokh cnautokh7" style="margin-top:-0.5px;"><div class="float-left">7)</div><div class="media-body">For [Autosampler Method Part] box select [Standard].</div></div><div class="media media-x cnautokh cnautokh8" style="margin-top:-0.5px;"><div class="float-left">8)</div><div class="media-body">For [MS Method Part] box select [nameMS].</div></div></li></ol></li><li>In the sample table check the line with your sample and click [Start].</li><li>Following the analytical run, close all software.</li><li>For representative m/z spectrum at different retention times see Figures 4B and 4C.</li></ol><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200330/20200330094645_1651.jpg"/><br/><strong>Figure 4. FT-ICR-MS chromatograpm and mass spectra of intact proteins.</strong> A. HPLC chromatograph following reverse phase purification. B-C. Intact mass of 26.4-28.1 min retention time corresponding to Cren7 protein isoforms B and Sso7D isoforms C. All peak m/z shifts correspond to +14 Da indicating variable degrees of methylation.<br/></li></ol></li></ol><p class="pptt" id="biaoti32546">### Data analysis</p><p class="ppzz">FT-ICR-MS data analysis can be found in Johnson <em>et al.</em>, 2019, materials and methods: FT-ICR-MS section. Data analysis must be determined on a protein-by protein basis as many parameters may play role for specific results depending on protein(s) and PTM(s) of interest.</p><p class="pptt" id="biaoti32547">### Notes</p><ol class="npwtwok"><li>All buffers must be a minimum of 300 ml.</li><li>If the instrument is turned off or disconnected from the network it will appear dimmed and cannot be connected.</li><li>During the system preparation run watch the system pressure tracing on the chromatogram pane in the system control window. If you see a very erratic fluctuation in system pressure there is air in the system buffer lines. All of the buffer pumps will need to be purged to fix this problem before your elution run (see Step B2: Priming inlets and purging pump heads).</li><li>Watch for any buffer leaks in the system during the system preparation run and tighten any loose connections to rectify the problem.</li><li>Some general parameters, however, should be set as follows:<br/>Signal to noise ratio (S/N) to 3. Minimum and maximum mass of the expected protein (including any known PTMs including truncation and processing). Maximum Charge State also depends on the mass of expected protein and should be set between 20 (small proteins) and 150 (large proteins). Precursor/fragment errors to 10 ppm.<br/></li></ol><p class="pptt" id="biaoti32548">### Recipes</p><ol class="npwtwok"><li>Buffer A<br/>30 mM sodium phosphate<br/>0.1 mM EDTA<br/>1 mM DTT<br/>pH 6.6</li><li>Buffer B<br/>30 mM sodium phosphate<br/>0.1 mM EDTA<br/>1 mM DTT.<br/>0.75 M KCl<br/>pH 6.6<br/></li></ol><p class="pptt" id="biaoti32549">### Acknowledgments</p><p class="ppzz">The protocol described above was adapted from work described in a recent publication (Johnson <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti32550">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti32551">### References</p><ol class="npwtwok"><li>Johnson, T., Payne, S., Grove, R., McCarthy, S., Oeltjen, E., Mach, C., Adamec, J., Wilson, M. A., Van Cott, K. and Blum, P. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30918025" target="_blank">Methylation deficiency of chromatin proteins is a non-mutational and epigenetic-like trait in evolved lines of the archaeon <em>Sulfolobus solfataricus</em>.</a> <em>J Biol Chem</em> 294(19): 7821-7832.</li></ol>