<body>
            <h1>Conjugation Protocol Optimised for <em>Roseburia inulinivorans</em> and <em>Eubacterium rectale</em></h1>
            <p><strong>Authors:</strong> Paul O. Sheridan, Jennifer C. Martin, Karen P. Scott</p>
            <h2>Abstract</h2>
            <p><em>Roseburia</em> and <em>Eubacterium</em> species of the human gut microbiota play an important role in the maintaince of human health, partly by producing butyrate, the main energy source of our colonic epithelial cells. However, our knowledge of the biochemistry and physiology of these bacteria has been limited by a lack of genetic manipulation techniques. Conjugative transposons previously introduced into <em>Roseburia</em> species could not be easily modified, greatly limiting their applicability as genetic modification platforms. Modular plasmid shuttle vectors have previously been developed for <em>Clostridium</em> species, which share a taxonomic order with <em>Roseburia</em> and <em>Eubacterium</em>, raising the possibility that these vectors could be used in these organisms. Here, we describe an optimized conjugation protocol enabling the transfer of autonomously replicating plasmids from an <em>E. coli</em> donor strain into <em>Roseburia inulinivorans</em> and <em>Eubacterium rectale</em>. The modular nature of the plasmids and their ability to be maintained in the recipient bacterium by autonomous replication makes them ideal for investigating heterologous gene expression, and as a platform for other genetic tools including antisense RNA silencing or mobile group II interon gene disruption strategies.</p>
            <p><strong>Keywords:</strong> Conjugation, Gut microbiota, Lachnospiraceae, Gene transfer, Shuttle vector</p>
            <h2>Background</h2><p><em>Roseburia</em> and <em>Eubacterium</em> species are among the most abundant bacteria in the human gut microbiota (Zhernakova <em>et al.</em>, 2016), impacting human health by utilising dietary and host derived polysaccharides (Scott <em>et al.</em>, 2006 and 2011; Cockburn <em>et al.</em>, 2015; Sheridan <em>et al.</em>, 2016) and producing the health promoting metabolite butyrate as a fermentation end product (Duncan <em>et al.</em>, 2002 and 2006). Additionally, these species are capable of modulating host immunity via flagella (Neville <em>et al.</em>, 2013). The lack of genetic modification techniques for these organisms has prevented a more complete understanding of the complex interactions between these bacteria and their human host.<br/> <br/>Previously, conjugative transposons were successfully transferred into <em>Roseburia inulinivorans</em> from <em>Eubacterium cellulosolvens</em> and <em>Clostridium cf. saccharolyticum</em> (Scott <em>et al.</em>, 2008). These large, novel mobile genetic elements could not be easily modified and thus were a suboptimal platform for detailed genetic modification. This work did however illustrate that conjugative mating was possible between Lachnospiraceae bacteria including <em>Roseburia</em> species. The development of easily modified conjugative plasmids for clostridial species (Purdy <em>et al.</em>, 2002; Heap <em>et al.</em>, 2009) raised the possibility that these techniques could be adapted for <em>Roseburia</em> and <em>Eubacterium</em> species.<br/> <br/>The detailed protocol presented here is based on procedures established in Sheridan <em>et al.</em> (2019). In this work, the different conjugative plasmids developed for use in <em>Clostridium</em> species (Heap <em>et al.</em>, 2009) were tested for transferability into the <em>Roseburia</em> and <em>Eubacterium rectale</em> species. Plasmid pMTL83151 was successfully transferred into two strains of <em>E. rectale</em>, while pMTL83151 and pMTL82151 were transferred into <em>Roseburia inulinivorans</em> A2-194. Transfer frequencies ofâ¯10<sup>-6</sup>-10<sup>-8</sup> per potential recipient were obtained. These frequencies are similar to those observed when suicide vectors were introduced into other Gram-positive bacteria (Williams <em>et al.</em>, 1990; Aquino de Muro and Priest, 2000). The ability to add exogenous DNA to bacterial species opens up opportunities for genetically manipulation, including knockout mutagenesis. Alternatively, these plasmids could be modified as expression vectors for mobile group II interon gene disruption strategies, as has been demonstrated in several clostridial species (Heap <em>et al.</em>, 2007). Additionally, Plasmid pMTL83151 was shown to be a suitable vector for heterologous gene expression (Sheridan <em>et al.</em>, 2019), producing an enzymatically active <em>Streptococcus</em> glycoside hydrolase in both species and thus proving the utility of this technique in studying researcher-selected functional gains in these important bacteria. The protocol below is a stepwise guide to introducing foreign DNA to these bacteria.</p>
        </body>

<p class="pptt" id="biaoti32600">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips</li><li>Cuvettes (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1652083 </a> )</li><li>Petri dishes (Greiner Bio-One, catalog number: <a class="blackNoUnder" target="_blank"> 633180 </a> )</li><li>Glass Pasteur pipettes (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FB50261 </a> )</li><li>Nylon membranes (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11417240001 </a> )</li><li>X-ray film (Fujifilm) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12735325 </a> )</li><li>50 ml conical centrifuge tubes (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 10038980 </a> )</li><li>Strains (Table 1)</li><li>Plasmids (Table 1)<br/><em>Note: Modular plasmids can be obtained from CHAINbiotools (</em><a href="http://clostron.com/pMTL80000.php" target="_blank"><em>http://clostron.com/pMTL80000.php</em></a><em>).</em> </li><li>Primers (Table 2)</li><li>PBS tablets (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P4417 </a> )</li><li>Chloramphenicol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C0378 </a> ) stock solution 10 Âµg/ml, stored at -20 Â°C</li><li>HindIII restriction endonuclease (NEB, catalog number: <a class="blackNoUnder" target="_blank"> R0104S </a> )</li><li>Wizard genomic DNA Purification kit (Promega, catalog number: <a class="blackNoUnder" target="_blank"> A1120 </a> )</li><li>DIG High Prime DNA Labelling and Detection Starter Kit II (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11585614910 </a> )</li><li>PCR reagents (Taq Polymerase kit Bioline, catalog number: <a class="blackNoUnder" target="_blank"> BIO-21040 </a> and dNTPâs Promega, catalog number: <a class="blackNoUnder" target="_blank"> U1240 </a> )</li><li>Dipotassium phosphate, K<sub>2</sub>HPO<sub>4</sub> (Fisher Scientific, catalog number: P/5240/53)</li><li>Potassium dihydrogen phosphate, KH<sub>2</sub>PO<sub>4</sub> (Fisher Scientific, catalog number: P/4800/60)</li><li>Ammonium sulfate, (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> (Fisher Scientific, catalog number: A/6480/53)</li><li>Sodium chloride, NaCl (Fisher Scientific, catalog number: S/3160/53)</li><li>Magnesium sulfate, MgSO<sub>4</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M7506 </a> )</li><li>Calcium chloride, CaCl<sub>2</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C1016 </a> )</li><li>Acetic acid (Fisher Scientific, catalog number: A/10400/PB17)</li><li>Propionic acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P1386 </a> )</li><li>n-Valeric acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> V9759 </a> )</li><li>Iso-Valeric acid (Sigma-Aldrich, catalog number: I7128)</li><li>Iso-Butyric acid (Sigma-Aldrich, catalog number: I1754)</li><li>Biotin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> B4501 </a> )</li><li>Cobalamin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> V2876 </a> )</li><li>p-Aminobenzoic acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A9878 </a> )</li><li>Folic acid (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> F7876 </a> )</li><li>Pyridoxamine (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P9755 </a> )</li><li>Potassium hydroxide, KOH (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P5958 </a> )</li><li>Ethanol 95% (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> E/0650DF/17 </a> )</li><li>Haemin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> H5533 </a> )</li><li>Bacto tryptone (BD Diagnostics Systems, catalog number: <a class="blackNoUnder" target="_blank"> 211705 </a> )</li><li>Yeast Extract (BD Diagnostics Systems, catalog number: <a class="blackNoUnder" target="_blank"> 212750 </a> )</li><li>Potassium chloride, KCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P3911 </a> )</li><li>Magnesium chloride hexahydrate, MgCl<sub>2</sub>Â·6H<sub>2</sub>O (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M9272 </a> )<br/></li><li>Magnesium sulfate heptahydrate, MgSO<sub>4</sub>Â·7H<sub>2</sub>O (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 230391 </a> )</li><li>Bacto casitone (BD Diagnostics Systems, catalog number: <a class="blackNoUnder" target="_blank"> 225930 </a> )</li><li>Sodium bicarbonate, NaHCO<sub>3</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S5761 </a> )</li><li>Glucose (Fisher Scientific, catalog number: G/0500/53)</li><li>Soluble starch (Sigma-Aldrich, catalog number: S2004)</li><li>Cellobiose (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C7252 </a> )</li><li>Resazurine (Sigma-Aldrich, catalog number: R2127)<br/></li><li>L-cysteine (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C1276 </a> )</li><li>Thiamin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T1270 </a> )</li><li>Riboflavin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> R9504 </a> )</li><li>Gas mix 10% Carbon Dioxide, 10% Hydrogen balance Nitrogen (Anaerobic) Cylinder (BOC, catalog number: 290564-L)</li><li>Agar (Oxoid, catalog number: <a class="blackNoUnder" target="_blank"> LP0011 </a> )</li><li>Resazurin solution (see Recipes)</li><li>Anaerobic phosphate buffered saline (PBS) (see Recipes)</li><li>Mineral solution 1 (see Recipes)</li><li>Mineral solution 2 (see Recipes)</li><li>Short chain fatty acid solution (see Recipes)</li><li>Vitamin solution 1 (see Recipes)</li><li>Vitamin solution 2 (see Recipes)</li><li>Haemin solution (see Recipes)</li><li>SOC (see Recipes)</li><li>LB (see Recipes)</li><li>LA (see Recipes)</li><li>YCFAGSC and AMM (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32601">### Equipment</p><ol class="npwtwok"><li>Pipettes Gilson P1000 (Gilson, catalog number: <a class="blackNoUnder" target="_blank"> F123602 </a> )</li><li>Pipettes Gilson P200 (Gilson, catalog number: <a class="blackNoUnder" target="_blank"> F123601 </a> )</li><li>Pipettes Gilson P20 (Gilson, catalog number: <a class="blackNoUnder" target="_blank"> F123600 </a> )</li><li>Pipettes Gilson P2 (Gilson, catalog number: <a class="blackNoUnder" target="_blank"> F144801 </a> )</li><li>Concept Plus Anaerobic Workstation, Ruskinn Technology</li><li>Hungate tubes (Sciquip, catalog number: 2047-00125)</li><li>Hungate lids butyl rubber septa (Sciquip, catalog numbers: 2047-11600 and 2047-16000)</li><li>Wheaton bottles (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 33110-U </a> )</li><li>CO<sub>2</sub> hooks (made in-house) and CO<sub>2</sub> piped gas supply</li><li>Shaking Incubator for <em>E. coli</em> growth (Sanyo Orbital incubator)</li><li>Static Incubator for anaerobic bacteria (Sciquip incu-160S)</li><li>Gene pulser (Bio-Rad, model: 1652078)</li><li>Platform rocker (Stuart, model: STR6)</li><li>UV-linker, Bio-Rad GS Gene linker UV chamber (UVP CL-1000 Ultraviolet Crosslinker)</li><li>Hybridizer (UVP laboratory products HB 1000 hybridizer)</li><li>Blot transfer pump (Hybaid Vacu-aid blot processing pump, DA7C.VAC/T)</li><li>Centrifuge (Jouan MR1822)</li><li>Transilluminator (UVtec BXT-20.L)<br/></li></ol><p class="pptt" id="biaoti32602">### Procedure</p><ol class="npwone"><li>Electroporation of plasmids into <em>E. coli</em> CA434<ol class="npwtwo"><li>Pre-chill 1 mm electroporation cuvette in refrigerator overnight.</li><li>Dilute 1 Âµl of plasmid solution (10 Âµg/ml) in 4 Âµl of dH<sub>2</sub>O (final concentration 2 Âµg/ml).</li><li>Thaw 55 Âµl electrocompetent <em>E. coli</em> CA434 (prepared in house following standard protocols, <a href="https://www.protocols.io/view/Making-your-own-electrocompetent-cells-imsv6m" target="_blank">https://www.protocols.io/view/Making-your-own-electrocompetent-cells-imsv6m</a>) on ice and add the plasmid solution, mixing gently.</li><li>Transfer the mixture to the cuvette and place in Gene Pulser.</li><li>Electroporate at 1.8 kV, 200 Ohms, 25 ÂµF.</li><li>Transfer into 1 ml SOC medium, pre-warmed to 37 Â°C.</li><li>Incubate for 1 h at 37 Â°C, 200 RPM.</li><li>Dilute in sterile dH<sub>2</sub>O in ten-fold dilutions (10<sup>1</sup>, 10<sup>2</sup> and 10<sup>3</sup>).</li><li>Spread 50 Âµl of these dilutions onto LA plates (see Recipes) supplemented with chloramphenicol 10 Âµg/ml (Cm10) and incubate at 37 Â°C for 24 h or until colonies appear.</li><li>Pick individual colonies into LB Cm10 and incubate at 37 Â°C, 200 RPM for 24 h.<br/><br/></li></ol></li><li>Anaerobic culturing of recipient bacterium<ol class="npwtwo"><li>Prepare anaerobic media YCFAGSC and AMM in broth as 7.5 ml aliquots in Hungate tubes, sealed with butyl rubber septa and 2% agar solutions divided into 100 ml aliquots in Wheaton bottles, with all dispensing and inoculating carried out under CO<sub>2</sub> using the Hungate technique as described by Bryant, 1972 (Figure 1).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200324/20200324195232_4385.jpg" width="163"/><br/><strong>Figure 1. Culturing of strictly anaerobic bacteria using the Hungate methodology (growth in tubes) and anaerobic workstation (growth on Petri plates)</strong><br/><br/></li><li>Incubate anaerobic YCFAGSC liquid cultures at 37 Â°C, without agitation in Hungate tubes.</li><li>Pour anaerobic agar media into Petri plate (20 ml per plate) within anaerobic workstation (gas mix CO<sub>2</sub>:N<sub>2</sub>:H<sub>2</sub>, ratio 7:11:2) approximately 24 h before use, enabling agar to adjust to anaerobic atmosphere.</li><li>Incubate anaerobic agar cultures at 37 Â°C in an anaerobic workstation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Mating of donor <em>E. coli</em> CA434 and recipient bacterium<ol class="npwtwo"><li>Prepare 7.5 ml overnight culture of recipient. Donor, recipients and plasmids are described in Table 1.<br/><br/><strong>Table 1. Strains and plasmids</strong><br/><img alt="" class="layerphoto" height="434" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200324/20200324195307_5178.jpg" width="600"/><br/>*pBP1, pCB102, pCD6 and pIM13 (replicons of these plasmids). ColE1 (Gram-negative replicon), catP (chloramphenicol resistance gene), traJ (origin of transfer) and MCS (multiple cloning site).<br/><br/></li><li>Inoculate donor <em>E. coli</em> CA434 in 40 ml of LB Cm10 in 50 ml conical centrifuge tubes and incubate at 37 Â°C, 200 RPM overnight.</li><li>Centrifuge donor overnight culture at 1,200 <em>Ã g</em> for 10 min.</li><li>Decant supernatant and resuspend pellet in 20 ml of anaerobic PBS.</li><li>Centrifuge donor again at 1,200 <em>Ã g</em> for 10min.</li><li>Decant supernatant in anaerobic cabinet.</li><li>Transfer overnight culture of recipient bacterium (grown as described in Steps B1 and B2) to anaerobic cabinet.</li><li>Resuspend donor pellet in 1 ml of recipient overnight culture in anaerobic cabinet.</li><li>Spot 100 Âµl of this solution on the centre of AMM agar plate and incubate in the anaerobic cabinet at 37 Â°C for 48 h.</li><li>Scrape mating mix off centre of AMM plate with sterile loop and suspend in 500 Âµl of anaerobic PBS.</li><li>Spread 50 Âµl of this solution on to the YCFAGSC plates supplemented with either 5 or 7.5 Âµg/ml chloramphenicol and incubate anaerobically at 37 Â°C until colonies appear (usually 48 to 96 h).</li><li>Restreak single colonies on fresh YCFAGSC plates supplemented with either 5 or 7.5 Âµg/ml chloramphenicol and incubate anaerobically at 37 Â°C until single colonies appear. This may take 3 days incubation.</li><li>Diagrammatic representation of the optimised conjugation protocol shown in Figure 2.<br/><br/><img alt="" class="layerphoto" height="136" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200324/20200324195410_4335.jpg" width="100"/><br/><strong>Figure 2. Detailed diagrammatic representation of the optimised conjugation protocol for </strong><em><strong>Roseburia inulinivorans</strong></em><strong> and </strong><em><strong>Eubacterium rectale</strong></em><strong>.</strong> Supplemented with 5 or 7.5 Âµg/ml chloramphenicol (Cm5 or Cm7.5).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Verification of putative transconjugants<br/>It is good practice to perform various tests to confirm the validity of transconjugants. Some simple tests can readily eliminate bacteria that are not transconjugants, and can save time and expense.<ol class="npwtwo"><li>Incubate putative transconjugants aerobically on AMM agar at 37 Â°C. The recipient bacterium is incapable of aerobic growth while the donor <em>E. coli</em> bacterium will grow.</li><li>Gram-stain using standard procedures. Coccus species are common contaminants in gut microbiology and are easily differentiated from the rod-like <em>Roseburia</em> and <em>Eubacterium</em> by Gram-staining.</li><li>PCR amplify the 16S rRNA gene directly from colonies or liquid culture using the universal bacterial 16S rRNA gene primers FD1 and RP2 (Table 2) (annealing temp. 52 Â°C), generating an amplicon of 1,495 bp. PCR amplification conditions are provided in Table 3.<br/><br/><strong>Table 2. Primers</strong><br/><img alt="" class="layerphoto" height="255" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200324/20200324195447_2295.jpg" width="600"/><br/>Nucleotide code: Guanine (G), adenine (A), thymine (T), cytosine (C), adenine or thymine (W) and guanine or thymine (K)<br/><br/><strong>Table 3. PCR protocol</strong><br/><img alt="" class="layerphoto" height="242" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200324/20200324195542_0099.jpg" width="450"/><br/><br/></li><li>Sanger sequence the resulting amplicon with the primers 519R and 926F (Table 2).</li><li>BLASTn query each sequence against the NCBI 16S rRNA gene database to confirm identity.</li><li>Confirm the presence of the plasmid in putative transconjugants by amplifying a nucleotide sequence common to all of the modular plasmids but absent in the recipient's chromosome. The primers PS#MTL-for and PS#MTL-rev (Table 2) (annealing temp 60 Â°C) amplify a 514 bp region incorporating sections of <em>cat</em>P gene and ColE1 Gram-negative replicon.</li><li>Expected results and comments are described in Table 4.<br/><br/><strong>Table 4. Verification procedures for putative transconjugants</strong><br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" height="345" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200324/20200324195640_8214.jpg" width="600"/><br/><br/></div></li></ol></li><li>Verification of autonomous plasmid replication by Southern blotting<ol class="npwtwo"><li>Extract DNA from transconjugants using the Wizard Genomic DNA Purification kit, following the manufacturer's instructions.</li><li>Digest 1 Âµg of extracted DNA with HindIII for 3 h at 37 Â°C to produce restriction fragments of various sizes.<br/><em>Note: HindIII only cuts the plasmid once, resulting in a single linear fragment, whereas restriction of the genomic DNA results in fragments of various sizes.</em> </li><li>Separate restriction fragments by size by gel electrophoresis (0.8% agarose, TBE).</li><li>Photograph gel image with transilluminator to facilitate size inference in the final blot.</li><li>Gently rock gel in depurination solution (0.25 M HCl) for 7 min and rinse in ddH<sub>2</sub>O (double distilled water).</li><li>Gently rock the gel in denaturation solution (0.5 M NaOH, 1.5 M NaCl) for 30 min three times and rinse in ddH<sub>2</sub>O.</li><li>Gently rock the gel in neutralisation solution (0.5 M Tris-HCl [pH 7.4], 3 M NaCl) for 30 min three times and rinse in ddH<sub>2</sub>O.</li><li>Transfer DNA from gel to nylon membrane using the Hybaid blotter (1.5 h) and UV-crosslinked.</li><li>Create probe by digesting pMTL83151 with HindIII and ApaLI. This produces two fragments (~3,000 bp and ~1,500 bp), the smaller of which is specific to a region common to all of the plasmids, but not present in the recipient chromosome. Gel purify the smaller fragment and use as probe template DNA.</li><li>Perform Southern blotting with DIG High Prime DNA Labelling and Detection Starter Kit II, following the manufacturer's instructions.</li><li>Visualise hybridizing bands by exposing membrane to x-ray film.</li></ol>This protocol enables the establishment and verification of autonomously replicating plasmids transferred into <em>Roseburia inulinivorans</em> and <em>Eubacterium rectale</em> and will provide an essential tool in investigating the biochemistry and physiology of these important organisms by genetic manipulation.<br/></li></ol><p class="pptt" id="biaoti32603">### Recipes</p><ol class="npwtwok"><li>Resazurin solution<br/>Add 100 mg of powdered resazurin to 100 ml ddH<sub>2</sub>O</li><li>Anaerobic phosphate buffered saline (PBS)<ol class="npwthree"><li>Dissolve PBS tablets in 1 L of ddH<sub>2</sub>O and add 1 ml resazurin solution</li><li>Place solution in boiling waterbath for 15 min</li><li>Bubble solution with 100% CO<sub>2</sub> until liquid turns from purple to clear</li><li>Dispense in 100 ml aliquots in Wheaton bottles</li></ol></li><li>Mineral solution 1<br/>K<sub>2</sub>HPO<sub>4</sub> 3.0 g<br/>ddH<sub>2</sub>O to 1 L<br/>Store at 4 Â°C</li><li>Mineral solution 2<br/>KH<sub>2</sub>PO<sub>4</sub> 3.0 g<br/>(NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub> 6.0 g<br/>NaCl 6.0 g<br/>MgSO<sub>4</sub> 0.6 g<br/>CaCl<sub>2</sub> 20.6 g <br/>ddH<sub>2</sub>O to 1 L <br/>Store at 4 Â°C</li><li>Short chain fatty acid solution<br/>Acetic acid 17 ml<br/>Propionic acid 6 ml <br/>n-Valeric acid 1 ml<br/>Iso-Valeric acid 1 ml<br/>Iso-Butyric acid 1 ml <br/>Store at 4 Â°C</li><li>Vitamin solution 1<br/>Biotin 1 mg<br/>Cobalamin 1 mg<br/>p-Aminobenzoic acid 3 mg<br/>Folic acid 5 mg<br/>Pyridoxamine 15 mg<br/>ddH<sub>2</sub>O to 100 ml<br/>Store at -20 Â°C</li><li>Vitamin solution 2<br/>Thiamin 5.0 mg <br/>Riboflavin 5.0 mg <br/>ddH<sub>2</sub>O to 100 ml</li><li>Haemin solution<br/>KOH 0.28 g<br/>Ethanol 95% 25 ml<br/>Haemin 100 mg<br/>ddH<sub>2</sub>O to 100 ml<br/>Store at 4 Â°C</li><li>SOC<br/>Bacto tryptone 2 g<br/>Yeast Extract 0.5 g<br/>NaCl 200 Î¼l of 5 M<br/>KCl 250 Î¼l of 1 M<br/>ddH<sub>2</sub>O to 100 ml<br/><ol class="npwthree"><li>Stir to dissolve, autoclave, then cool to room temperature</li><li>Add 1 ml of filter sterile 2 M Mg stock solution (1 M MgCl<sub>2</sub>Â·6H<sub>2</sub>O and 1 M MgSO<sub>4</sub>Â·7H<sub>2</sub>O) to give a final conc of 20 mM</li><li>Before use add 20 Âµl of 1 M sterile glucose per 1 ml of SOC</li></ol></li><li>LB<br/>Bacto tryptone 1 g<br/>Yeast extract 0.5 g<br/>NaCl 1 g<br/>Deionized H<sub>2</sub>O to 100 ml</li><li>LA<br/>Bacto tryptone 1 g<br/>Yeast extract 0.5 g<br/>NaCl 1 g <br/>Agar 1.5 g<br/>Deionized H<sub>2</sub>O to 100 ml</li><li>YCFAGSC and AMM<br/>Bacto casitone 10.0 g<br/>Yeast extract 2.5 g<br/>NaHCO<sub>3</sub> 4.0g<br/>Glucose 2.0 g<br/>Soluble starch 2.0 g<br/>Cellobiose 2.0 g<br/>Mineral solution 1 150.0 ml<br/>Mineral solution 2 150.0 ml<br/>Haemin solution 10.0 ml<br/>Vitamin solution 1 (before autoclaving) 1.0 ml<br/>Vitamin solution 2 (after autoclaving) 1.0 ml<br/>Resazurine 1.0 ml<br/>L-cysteine 1.0 g<br/>Acetic acid (0.7 ml) to make AMM or short chain fatty acid solution (3.1 ml) to make YCFAGSC<br/>Add 20 g of agar to these recipes to make AMM or YCFAGSC agar<br/>Distilled water up to 1 L<br/></li></ol><p class="pptt" id="biaoti32604">### Acknowledgments</p><p class="ppzz">The Rowett Institute (University of Aberdeen) receives financial support from the Scottish Government Rural and Environmental Sciences and Analytical Services (RESAS). The protocol is derived from work published in âHeterologous gene expression in the human gut bacteria <em>Eubacterium rectale</em> and <em>Roseburia inulinivorans</em> by means of conjugative plasmidsâ Anaerobe 59: 131-140 (2019) (Sheridan <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti32605">### Competing interests</p><p class="ppzz">The authors state that there are no competing interests</p><p class="pptt" id="biaoti32606">### References</p><ol class="npwtwok"><li>Aquino de Muro, M. and Priest, F. G. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11037132" target="_blank">Construction of chromosomal integrants of <em>Bacillus sphaericus</em> 2362 by conjugation with <em>Escherichia coli</em>.</a> <em>Res Microbiol</em> 151(7): 547-555.</li><li>Barcenilla, A., Pryde, S. E., Martin, J. C., Duncan, S. H., Stewart, C. S., Henderson, C. and Flint, H. J. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10742256" target="_blank">Phylogenetic relationships of butyrate-producing bacteria from the human gut.</a> <em>Appl Environ Microbiol</em> 66(4): 1654-1661.</li><li>Bryant, M. P. (1972). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4565349" target="_blank">Commentary on the Hungate technique for culture of anaerobic bacteria.</a> <em>Am J Clin Nutr </em>25(12): 1324-1328.</li><li>Cockburn, D. W., Orlovsky, N. I., Foley, M. H., Kwiatkowski, K. J., Bahr, C. M., Maynard, M., Demeler, B. and Koropatkin, N. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25388295" target="_blank">Molecular details of a starch utilization pathway in the human gut symbiont <em>Eubacterium rectale</em>.</a> <em>Mol Microbiol</em> 95(2): 209-230.</li><li>Duncan, S. H., Aminov, R. I., Scott, K. P., Louis, P., Stanton, T. B. and Flint, H. J. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17012576" target="_blank">Proposal of <em>Roseburia</em> <em>faecis</em> sp. nov., <em>Roseburia</em> <em>hominis</em> sp. nov. and <em>Roseburia inulinivorans</em> sp. nov., based on isolates from human faeces.</a> <em>Int J Syst Evol Microbiol</em> 56(Pt 10): 2437-2441.</li><li>Duncan, S. H., Hold, G. L., Barcenilla, A., Stewart, C. S. and Flint, H. J. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12361264" target="_blank"><em>Roseburia intestinalis</em> sp. nov., a novel saccharolytic, butyrate-producing bacterium from human faeces.</a> <em>Int J Syst Evol Microbiol </em>52(Pt 5): 1615-1620.</li><li>Heap, J. T., Pennington, O. J., Cartman, S. T., Carter, G. P. and Minton, N. P. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17658189" target="_blank">The ClosTron: a universal gene knock-out system for the genus <em>Clostridium</em>.</a> <em>J Microbiol Methods</em> 70(3): 452-464.</li><li>Heap, J. T., Pennington, O. J., Cartman, S. T. and Minton, N. P. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19445976" target="_blank">A modular system for <em>Clostridium</em> shuttle plasmids.</a> <em>J Microbiol Methods</em> 78(1): 79-85.</li><li>Muyzer, G., Teske, A., Wirsen, C. O. and Jannasch, H. W. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7545384" target="_blank">Phylogenetic relationships of <em>Thiomicrospira</em> species and their identification in deep-sea hydrothermal vent samples by denaturing gradient gel electrophoresis of 16S rDNA fragments.</a> <em>Arch Microbiol</em> 164(3): 165-172.</li><li>Neville, B. A., Sheridan, P. O., Harris, H. M., Coughlan, S., Flint, H. J., Duncan, S. H., Jeffery, I. B., Claesson, M. J., Ross, R. P., Scott, K. P. and O'Toole, P. W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23935906" target="_blank">Pro-inflammatory flagellin proteins of prevalent motile commensal bacteria are variably abundant in the intestinal microbiome of elderly humans.</a> <em>PLoS One</em> 8(7): e68919. </li><li>Purdy, D., O'Keeffe, T. A., Elmore, M., Herbert, M., McLeod, A., Bokori-Brown, M., Ostrowski, A. and Minton, N. P. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12406220" target="_blank">Conjugative transfer of clostridial shuttle vectors from <em>Escherichia coli</em> to <em>Clostridium difficile</em> through circumvention of the restriction barrier.</a> <em>Mol Microbiol</em> 46(2): 439-452.</li><li>Scott, K. P., Martin, J. C., Campbell, G., Mayer, C. D. and Flint, H. J. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16740940" target="_blank">Whole-genome transcription profiling reveals genes up-regulated by growth on fucose in the human gut bacterium "<em>Roseburia inulinivorans</em>".</a> <em>J Bacteriol</em> 188(12): 4340-4349.</li><li>Scott, K. P., Martin, J. C., Chassard, C., Clerget, M., Potrykus, J., Campbell, G., Mayer, C. D., Young, P., Rucklidge, G., Ramsay, A. G. and Flint, H. J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20679207" target="_blank">Substrate-driven gene expression in <em>Roseburia inulinivorans</em>: importance of inducible enzymes in the utilization of inulin and starch.</a> <em>Proc Natl Acad Sci U S A</em> 108 Suppl 1: 4672-4679.</li><li>Scott, K. P., Martin, J. C., Mrazek, J. and Flint, H. J. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18456856" target="_blank">Transfer of conjugative elements from rumen and human Firmicutes bacteria to <em>Roseburia inulinivorans</em>.</a> <em>Appl Environ Microbiol </em>74(12): 3915-3917.</li><li>Sheridan, P. O., Martin, J. C., Minton, N. P., Flint, H. J., O'Toole, P. W. and Scott, K. P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31228669" target="_blank">Heterologous gene expression in the human gut bacteria <em>Eubacterium rectale</em> and <em>Roseburia inulinivorans</em> by means of conjugative plasmids.</a> <em>Anaerobe</em> 59: 131-140.</li><li>Sheridan, P. O., Martin, J. C., Lawley, T. D., Browne, H. P., Harris, H. M. B., Bernalier-Donadille, A., Duncan, S. H., O'Toole, P. W., K, P. S. and H, J. F. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28348841" target="_blank">Polysaccharide utilization loci and nutritional specialization in a dominant group of butyrate-producing human colonic Firmicutes.</a> <em>Microb Genom</em> 2(2): e000043.</li><li>Turner, S., Pryer, K. M., Miao, V. P. and Palmer, J. D. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10461381" target="_blank">Investigating deep phylogenetic relationships among cyanobacteria and plastids by small subunit rRNA sequence analysis.</a> <em>J Eukaryot Microbiol</em> 46(4): 327-338.</li><li>Williams, D. R., Young, D. I. and Young, M. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2199603" target="_blank">Conjugative plasmid transfer from <em>Escherichia coli</em> to <em>Clostridium</em> <em>acetobutylicum</em><em>.</em></a> <em>J Gen Microbiol</em> 136(5): 819-826.</li><li>Wood, J., Scott, K. P., Avgustin, G., Newbold, C. J. and Flint, H. J. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9758785" target="_blank">Estimation of the relative abundance of different <em>Bacteroides</em> and <em>Prevotella</em> ribotypes in gut samples by restriction enzyme profiling of PCR-amplified 16S rRNA gene sequences.</a> <em>Appl Environ Microbiol</em> 64(10): 3683-3689.</li><li>Zhernakova, A., Kurilshikov, A., Bonder, M. J., Tigchelaar, E. F., Schirmer, M., Vatanen, T., Mujagic, Z., Vila, A. V., Falony, G., Vieira-Silva, S., Wang, J., Imhann, F., Brandsma, E., Jankipersadsing, S. A., Joossens, M., Cenit, M. C., Deelen, P., Swertz, M. A., LifeLines Cohort, S., Weersma, R. K., Feskens, E. J., Netea, M. G., Gevers, D., Jonkers, D., Franke, L., Aulchenko, Y. S., Huttenhower, C., Raes, J., Hofker, M. H., Xavier, R. J., Wijmenga, C. and Fu, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27126040" target="_blank">Population-based metagenomics analysis reveals markers for gut microbiome composition and diversity.</a> <em>Science</em> 352(6285): 565-569.<br/></li></ol>