<body>
            <h1>High-throughput Flow Cytometry Assay to Investigate TDP43 Splicing Function</h1>
            <p><strong>Authors:</strong> H. Broder Schmidt, Rajat  Rohatgi</p>
            <h2>Abstract</h2>
            <p>Mutations in RNA-binding proteins (RBPs) such as TDP43 are associated with transcriptome-wide splicing defects and cause severe neurodegenerative diseases, including amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). The impact of RBP mutations on splicing function is routinely studied using PCR-based bulk measurements. However, the qualitative and low-throughput nature of this assay make quantitative and systematic analyses, as well as screening approaches, difficult to implement. To overcome this hurdle, we have developed a quantitative, high-throughput flow cytometry assay to investigate TDP43 splicing function on a single-cell level<br/></p>
            <p><strong>Keywords:</strong> RNA splicing, Minigene, Flow cytometry, TDP43, Amyotrophic lateral sclerosis</p>
            <h2>Background</h2><p>RNA-binding proteins (RBPs) such as TDP43 regulate post-transcriptional gene regulation by orchestrating RNA stability, transport and processing, including mRNA splicing (Gerstberger <em>et al.</em>, 2014). Impairment of RBP function as a result of mutations and/or aggregation has been implicated in the etiology of many neurodegenerative diseases such as amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) (Harrison and Shorter, 2017). Notably, ALS-associated mutations in RBPs like TDP43 can cause transcriptome-wide splicing defects, suggesting that misregulation of RBP splicing function may be a key driver of disease (Arnold <em>et al.</em>, 2013; Sun <em>et al.</em>, 2015). Thus, an important goal in ALS research is to uncover the molecular mechanisms of RBP function, requiring experimental approaches for the systematic interrogation of RBP splicing function.<br/><br/>Widely-used methods for studying splicing are reverse transcription-PCR (RT-PCR) and RNA sequencing (RNA-seq). Both methods can be used to study the splicing of endogenous transcripts; however, typically only as a bulk measurement of an entire population of cells. RT-PCR is simple and affordable, yet efficient only to study splicing of a select few endogenous transcripts. Moreover, RT-PCR read-outs are often only semi-quantitative. In contrast, RNA-seq provides quantitative insights into global splicing, but is time- and cost-intensive and requires sophisticated data analysis pipelines (Wang <em>et al.</em>, 2009). Thus, both RT-PCR and RNA-seq are poorly suited for high-throughput applications, for example to systematically compare the splicing function of multiple RBP variants.<br/><br/>A powerful tool to study splicing are minigenes, which typically are plasmid-based, simplified genes containing introns and exons that recapitulate defined splicing events outside of the native gene context (Baralle and Baralle, 2005). In particular, minigenes have been instrumental in discovering <em>cis</em>- and <em>trans</em>-acting splicing elements (Cooper, 2005). Instead of endogenous targets, minigenes are also routinely used to streamline investigation of RBP splicing function (D'Ambrogio <em>et al.</em>, 2009; Kino <em>et al.</em>, 2011). Conventionally, the splicing of minigenes is evaluated with analytical or quantitative RT-PCR. However, the low-throughput and bulk read-out of this method precludes leveraging the full potential of minigene-based splicing reporters.<br/><br/>To overcome these hurdles, we developed a minigene that encodes for a fluorescent protein (FP minigene), thus allowing us to convert a defined splicing event into a sensitive, inherently quantitative and readily detectable signal on a single-cell level by flow cytometry (Schmidt <em>et al.</em>, 2019). To normalize splicing of the FP minigene to the overall abundance of the parent transcript, we fused it to another, intron-free fluorescent protein (constitutive FP). While this approach is broadly applicable (Gurskaya <em>et al.</em>, 2012; Sorenson and Stevens, 2014; Gonatopoulos-Pournatzis <em>et al.</em>, 2018), we adapted it specifically to study TDP43-dependent splicing by interrupting the FP minigene with exon 9 of the CFTR gene, which is skipped only in the presence of functional TDP43 (Buratti <em>et al.</em>, 2001). To rapidly compare the splicing efficiency of multiple TDP43 variants in TARDBP (<em>i.e.</em>, the gene encoding TDP43) knock-out cells, we used a bidirectional promoter to drive the coordinated expression of both our splicing reporter and a given BFP-tagged TDP43 variant from the same plasmid. A graphical overview of our splicing reporter design in shown in Figure 1.<br/></p>
        </body>

<p class="ppzz"><strong><br/><img class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200411/20200411104838_5651.jpg" width="165"/><br/>Figure 1. Splicing reporter design.</strong> A. The fluorescent protein (FP)-based splicing module and BFP-tagged TDP43 variants are expressed from the same plasmid via a bi-directional promoter (pBI). B. Production of a dually-fluorescent GFP-mCherry fusion protein depends on the presence of functional TDP43, which is required to mediate the skipping of CFTR gene-derived exon (labeled E2).<br/><br/>Here, we describe in detail how to perform and analyze our flow cytometry-based splicing assay. This protocol is optimized for use with HEK-293T wild-type and HEK-293T TARDBP knock-out cells (Schmidt <em>et al.</em>, 2019), yet in principle suitable for other recently published constitutive or inducible TARDBP knock-out cell lines as well (Chiang <em>et al.</em>, 2010; Gasset-Rosa <em>et al.</em>, 2019; Roczniak-Ferguson and Ferguson, 2019).</p><p class="pptt" id="biaoti32849">## Materials and Reagents</p><ol class="npwone"><li>Cell lines<ol class="npwtwo"><li>HEK-293T wild-type cells (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> CRL-3216 </a> )</li><li>HEK-293T TARDBP knock-out cells (available upon request)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell culture<ol class="npwtwo"><li>1.5 ml microtubes (Eppendorf, catalog number: 0<a class="blackNoUnder" target="_blank"> 22363204 </a> )</li><li>60 Âµm scepter tips for cell counting (Milipore-Sigma, catalog number: <a class="blackNoUnder" target="_blank"> PHCC60050 </a> )</li><li>10 cm tissue culture plates for propagating cell lines (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-556-002 </a> )</li><li>6-well tissue culture plates for splicing assays (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14-832-11 </a> )</li><li>70 Âµm cell strainer (Corning, Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 08-771-2 </a> )</li><li>5 ml round-bottom polystyrene tubes (Corning, Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 14-959-1A </a> )</li><li>SH800S flow cytometer setup beads (Sony Biotechnology, catalog number: <a class="blackNoUnder" target="_blank"> LE-B3001 </a> )</li><li>HyClone DMEM high glucose (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> SH30081FS </a> ) supplemented with:<ol class="npwthree"><li>10% FBS (Atlanta Biologicals, catalog number: <a class="blackNoUnder" target="_blank"> S11150H </a> )</li><li>1 mM sodium pyruvate (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 11-360-070 </a> )</li><li>2 mM L-glutamine (Gemini Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 400106 </a> )</li><li>1x MEM non-essential amino acids (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 11-140-076 </a> )</li><li>40 U/ml penicillin and 40 Âµg/ml streptomycin (Gemini Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 400109 </a> )</li></ol></li><li>Sterile 1x PBS at pH 7.4 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P3813-5x10PAK </a> )</li><li>Trypsin-EDTA (Gemini Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 400150 </a> )</li><li>X-tremeGENE9 transfection reagent (Roche, catalog number: 0<a class="blackNoUnder" target="_blank"> 6365787001 </a> )</li><li>OptiMEM (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 31985-070 </a> )<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Plasmids (available at <a href="https://www.addgene.org/Rajat_Rohatgi/" target="_blank">https://www.addgene.org/Rajat_Rohatgi/</a>, unless otherwise noted)<ol class="npwtwo"><li>Splicing reporter only: pHBS1389 IBB-GFP-mCherry3E (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 118803 </a> )</li><li>Positive control: pHBS1503 [IBB-GFP-mCherry3E]-[BFP-TDP43 WT] (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 133327 </a> )</li><li>Negative control: pHBS1501 [IBB-GFP-mCherry3E]-[BFP-TDP43 âRRM] (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 133325 </a> )</li><li>Extensive collection of TDP43 CTD mutants in the splicing reporter (<em>e.g.</em>, Addgene, catalog numbers: <a class="blackNoUnder" target="_blank"> 133328-133333 </a> )</li><li>Backbone for subcloning of CTD mutants into splicing reporter: pHBS1224 [IBB-GFP-mCherry3E]-[BFP-TDP43 âCTD EcoRV] (available upon request)</li><li>Compensation probes: mammalian expression vectors with BFP, GFP and mCherry inserts (<em>e.g.</em>, Addgene, catalog numbers: <a class="blackNoUnder" target="_blank"> 54665 </a> , <a class="blackNoUnder" target="_blank"> 54759 </a> and <a class="blackNoUnder" target="_blank"> 54563 </a> )</li></ol></li></ol><p class="pptt" id="biaoti32850">### Equipment</p><ol class="npwtwok"><li>Single-channel pipettes (Rainin or equivalent)</li><li>Cell culture incubator (Thermo Scientific or equivalent)</li><li>Scepter cell counter (Milipore-Sigma) or equivalent manual/automatic cell counter</li><li>Sony SH800S cell sorter (Sony Biotechnology) or equivalent cell sorter equipped with 405 nm, 488 nm and 561 nm lasers, as well as appropriate filters</li><li>Computer for data analysis (<em>e.g.</em>, 3.1 GHz Intel i7 processor with 16 GB RAM)<br/></li></ol><p class="pptt" id="biaoti32851">### Software</p><ol class="npwtwok"><li>SH800S operating software for data collection (Sony Biotechnology) or operating software for equivalent flow cytometers</li><li>Mathematica Version 11.3 or higher (Wolfram Research) + custom data analysis pipelines:<ol class="npwthree"><li>Script âCSA_Gating.nbâ (available at <a href="https://github.com/RohatgiLab/TDP43-analysis" target="_blank">https://github.com/RohatgiLab/TDP43-analysis</a>)</li><li>Script âCSA_Plotting.nbâ (available at <a href="https://github.com/RohatgiLab/TDP43-analysis" target="_blank">https://github.com/RohatgiLab/TDP43-analysis</a>) </li></ol></li></ol><p class="pptt" id="biaoti32852">### Procedure</p><ol class="npwone"><li>Seeding of cells<br/><em>Note: Work in a tissue culture hood to prevent contaminations. Culture all HEK-293T wild-type and TARDBP knock-out cells in DMEM high-glucose plus supplements (see Materials).</em> <ol class="npwtwo"><li>Harvest HEK-293T wild-type and TARDBP knock-out cells by trypsinization and determine cell numbers using the Scepter (or equivalent manual/automatic cell counters).</li><li>For setup and calibration of the flow cytometer, seed four wells of a standard six-well tissue culture plate with 400,000 HEK-293T wild-type cells each (<strong>compensation samples</strong>).</li><li>For positive and negative controls, seed another four wells of a standard six-well tissue culture plate with 400,000 HEK-293T TARDBP knock-out cells each (<strong>control samples</strong>).</li><li>For each TDP43 variant to be tested, seed two wells of a standard six-well tissue culture plate with 400,000 HEK-293T TARDBP knock-out cells each (<strong>experimental samples</strong>).</li><li>Let the cells adhere overnight in a cell culture incubator at 37 Â°C and 5% CO<sub>2</sub>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transfection of cells<br/><em>General workflow: Work in a tissue culture hood to prevent contaminations. Per transfection, carefully add 3 Âµl of X-tremeGENE9 transfection to 250 Âµl OptiMEM without touching the walls of the microtube and incubate for 20 minutes at room temperature. Per transfection, mix 1 Âµg of reporter plasmid with 250 Âµl OptiMEM in a separate microtube. After incubation, combine the reactions and carefully spread the mix dropwise across the cells in a given well.</em> <ol class="npwtwo"><li>To prepare the <strong>compensation samples</strong>, transfect one well of HEK-293T cells with 1 Âµg of either BFP, GFP or mCherry mammalian expression constructs. Note that one well will purposely remain untransfected.</li><li>To prepare <strong>control samples</strong>, transfect two wells of HEK-293T TARDBP knock-out cells with 1 Âµg each of either pHBS1501 or pHBS1503.</li><li>To prepare <strong>experimental samples</strong>, transfect two wells of HEK-293T TARDBP knock-out cells with 1 Âµg each of the desired construct. A reference collection of TDP43 CTD mutants in the splicing reporter background is available at <a href="https://www.addgene.org/Rajat_Rohatgi/" target="_blank">https://www.addgene.org/Rajat_Rohatgi/</a>.</li><li>Incubate all samples for 24 h in a cell culture incubator at 37 Â°C and 5% CO<sub>2</sub>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Flow cytometer setup and calibration<br/><em>Note: Only a basic outline of the SH800S setup and calibration procedure is given below. Please consult the manual or contact the technical service for further questions and troubleshooting.</em> <ol class="npwtwo"><li>Before starting the flow cytometer, check the sheath fluid and ddH<sub>2</sub>O levels (refill if necessary) and empty the waste.</li><li>Make sure that the sheath fluid container and lines are sealed, then turn on the compressor.</li><li>Turn on the SH800S flow cytometer and start the operating software.</li><li>Follow the prompts during the chip loading and fluidics check steps. We use 100 Âµm chips for analyzing HEK-293T cells. Select the 405 nm, 488 nm and 561 nm lasers when prompted.</li><li>Follow the prompts during the chip alignment step. Sort calibration is not required, as analyzer mode is sufficient.</li><li>During setup, harvest the four <strong>compensation samples</strong> by trypsinization. Make sure to keep the sample volumes low to prevent unnecessary dilution of the cell suspensions (we typically use 250 Âµl trypsin per well and quench by adding 250 Âµl complete DMEM). Filter each sample through a cell strainer into a 5 ml round-bottom polystyrene tube. Keep tubes on ice until further use.</li><li>Once setup is finished, create a new experiment in the operating software. In the measurement settings pane, select only the FSC (forward scatter), BSC (side scatter), FL1 (BFP), FL2 (GFP) and FL3 (mCherry) channels. When prompted, start the compensation wizard and follow the steps. The <strong>compensation samples</strong> will be needed at this point.</li><li>Upon completion of the compensation step, setup is finished and the remaining samples can be prepared.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Prepare and analyze samples by flow cytometry<ol class="npwtwo"><li>Harvest the <strong>control</strong> and <strong>experimental samples</strong> by trypsinization as described in Step C6. Filter each sample through a cell strainer into a 5 ml round-bottom polystyrene tube. Keep samples on ice until further use.</li><li>Start by analyzing the <strong>control samples</strong> to set the gates and get an idea of the expected effect sizes.</li><li>In the BSC-A vs. FSC-A density plot, ensure that the gate for the cell population set during compensation still applies (<em>Gate 1</em>; Figure 2A).</li><li>In <em>Gate 1</em>, create a FSC-H vs. FSC-A density plot to further gate for single cells (<em>Gate 2</em>; Figure 2B). This filter makes use of disproportions between the area and height of cell clumps compared to single cells (Hazen <em>et al.</em>, 2018).</li><li>In <em>Gate 2</em>, create FL1-A (BFP), FL2-A (GFP) and FL3-A (mCherry) histogram plots. The BFP signal is proportional to the total TDP43 amount. The GFP signal scales with total splicing reporter transcript levels, whereas the mCherry signal reflects TDP43 splicing activity.</li><li>To get a preview of TDP43 splicing efficiency, first gate for BFP-positive cells in the FL1-A histogram (<em>Gate 3</em>; Figure 2C). This step is to select for successfully transfected cells only. The transfection efficiency (<em>i.e.</em>, fraction of BFP-positive cells) should be 50% or greater. Then in <em>Gate 3</em>, create a FL2-A (GFP) vs. FL3-A (mCherry) density plot. There should be a significant difference in the position of the positive and negative control populations in this plot (Figure 2D). <br/><em>Note: These gates are only for data preview, but not final data analysis.</em> </li><li>Collect at least 50,000 events in <em>Gate 3</em> for each <strong>control</strong> and <strong>experimental sample</strong>. Make sure to hit the âRecordâ button to ensure that data is saved during data collection. As a reference, we recommend to take a screen shot of the above specified plots for each sample after data collection. Unfortunately, this has to be done manually for each sample. Right-click on the worksheet area to bring up the quick access menu and select âCopy Worksheet Pictureâ. Paste the picture into a simple image processor such as Microsoft Paint and save.</li><li>After all samples have been analyzed, export the data as an fcs-file by right-clicking on the experiment in the âActive Experimentsâ panel.</li><li>Shut-down the flow cytometer and software following the instructions of the shutdown wizard, which can be found in the âCytometerâ tab.</li><li>After the instrument shut-down, turn-off the compressor and release the pressure from the sheath tank.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200411/20200411104908_2772.jpg" width="149"/><br/><strong>Figure 2. Example data and gating strategy during data acquisition.</strong> A. BSC-A vs. FSC-A density plot showing all detected events and <em>Gate 1</em> that selects for cells. B. FSC-H vs. FSC-A density plot and <em>Gate 2</em> to select for single cells. C. FL1-A (BFP) histogram and <em>Gate 3</em> to select for transfected cells. D. FL3-A (mCherry) vs. FL2-A (GFP) density plots for full-length, wild-type (WT) TDP43 and TDP43 lacking its RNA-binding domains (âRRM).</li></ol></li></ol><p class="pptt" id="biaoti32853">### Data analysis</p><p class="ppzz"><em>Note: The data analysis pipeline outlined below uses Mathematica scripts that have been written specifically to analyze fcs-files generated by the Sony SH800 software and may not be immediately compatible with fcs-files from other machines without tweaking the code.</em><br/><br/>For analysis of the splicing data, the mCherry signal of each cell is normalized to the GFP signal of the same cell to quantify splicing efficiency (Figure 3A). To further normalize to the TDP43 level of that cell, the splicing efficiency is plotted against the BFP signal. We have developed Mathematica scripts (available at <a href="https://github.com/RohatgiLab/TDP43-analysis" target="_blank">https://github.com/RohatgiLab/TDP43-analysis</a>) that extract the raw measurements for each observation from standard fcs-files, gate for cells and calculate the splicing ratio (script âCSA_Gating.nb), as well as plot the data (âCSA_Plotting.nbâ).<br/>  First open the âCSA_Gating.nbâ script in Mathematica. Under âStep 0â, the gates for the analysis are specified (Figure 3B). Try the pre-defined values or refer to the reference screen shots to set appropriate boundary conditions. Confirm by evaluating the cell. In âStep 1â, the location of the data files on the hard drive is specified. To the parameter âRandomFileâ, attribute the file path to a random file in the folder containing the fcs-files from the experiment (Figure 3C). Evaluate the cell to confirm; a list of all fcs-files found will be returned. Finally, evaluate the cell titled âStep 2â. In this analysis loop, the data will be loaded, gated and analyzed (Figure 3D). For each sample, a summary of the analysis as well as csv-files with the raw numerical values will be exported.<br/>  For further analysis and plotting of the data, open the âCSA_Plotting.nbâ script in Mathematica. Start by attributing the file path to the csv-files from the above analysis to the parameter âRandomFileâ (Figure 3E) and evaluate the cell to confirm. Then evaluate the next cell to re-format the data for further analysis. To plot TDP43 splicing efficiency, expressed as the ratio of the mCherry-to-GFP signals, versus the total cellular TDP43 concentration (BFP signal), evaluate the cell labeled âPlot the data!â. To change the axis of the plots, update the âyScaleâ, âxStartâ and âxEndâ parameters (Figure 3F), then re-evaluate the cell. This step generates both individual plots for each sample that is automatically rendered and saved (see example in Figure 3G). The last two cells in the script allow the comparison of TDP43 splicing efficiency in a range of TDP43 levels defined by the parameters âstartâ and âendâ, generating a violin plot (Figure 3G). To manually re-arrange the order of the samples in the plot, specify the desired sample order in the list termed âOrderedDataâ and evaluate the cell (Figure 3H).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200411/20200411104935_0978.jpg" width="90"/><br/><strong>Figure 3. Splicing assay data analysis and plotting.</strong> A. Outline of the analysis pipeline. A flow cytometer is used to measure and deconvolute the mCherry, GFP and BFP fluorescence signals of individual cells in a population. Using custom scripts, the splicing efficiency is calculated by taking the ratio of the mCherry to GFP signal of a given cell (Figure 1) and compared to the TDP43 level of the same cell (BFP signal). B. Screenshot highlighting where to change the gate boundaries and cut-offs in the âCSA_Gating.nbâ script. C. Screenshot indicating where to provide the file path to the fcs raw data in the âCSA_Gating.nbâ script. D. Example density plots generated during evaluation of the âCSA_Gating.nbâ script to visualize the gates defined for analysis. E. Screenshot illustrating where in the âCSA_Plotting.nbâ script to provide the file path to the csv files generated during evaluation of the âCSA_Gating.nbâ script. F. Screenshot highlighting where to adjust the plot scales in the âCSA_Plotting.nbâ script. G. Example density plot generated by the âCSA_Plotting.nbâ script that depicts splicing efficiency vs. TDP43 levels for wild-type (WT) and âRRM TDP43 variants (top) and example violin plot highlighting the splicing efficiencies in the defined BFP signal range (bottom). H. Screenshot of the âCSA_Plotting.nbâ script indicating where to define the BFP signal range for the violin plots comparing splicing efficiency.</p><p class="pptt" id="biaoti32854">### Acknowledgments</p><p class="ppzz">This work has been supported by the Deutsche Forschungsgemeinschaft (SCHM 3082/2-1 to H.B.S.) and the National Institutes of Health (DP2 GM105448 and R35 GM118082 to R. R.). The original protocol was published in Schmidt <em>et al.</em> (2019).</p><p class="pptt" id="biaoti32855">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti32856">### References</p><ol class="npwtwok"><li>Arnold, E. S., Ling, S. C., Huelga, S. C., Lagier-Tourenne, C., Polymenidou, M., Ditsworth, D., Kordasiewicz, H. B., McAlonis-Downes, M., Platoshyn, O., Parone, P. A., Da Cruz, S., Clutario, K. M., Swing, D., Tessarollo, L., Marsala, M., Shaw, C. E., Yeo, G. W. and Cleveland, D. W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23382207" target="_blank">ALS-linked TDP-43 mutations produce aberrant RNA splicing and adult-onset motor neuron disease without aggregation or loss of nuclear TDP-43.</a> <em>Proc Natl Acad Sci U S A</em> 110(8): E736-745.</li><li>Baralle, D. and Baralle, M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16199547" target="_blank">Splicing in action: assessing disease causing sequence changes.</a> <em>J Med Genet</em> 42(10): 737-748.</li><li>Buratti, E., DÃ¶rk, T., Zuccato, E., Pagani, F., Romano, M. and Baralle, F. E. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11285240" target="_blank">Nuclear factor TDP-43 and SR proteins promote <em>in vitro</em> and <em>in vivo</em> CFTR exon 9 skipping.</a> <em>EMBO J</em> 20(7): 1774-1784.</li><li>Chiang, P. M., Ling, J., Jeong, Y. H., Price, D. L., Aja, S. M. and Wong, P. C. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20660762" target="_blank">Deletion of TDP-43 down-regulates Tbc1d1, a gene linked to obesity, and alters body fat metabolism.</a> <em>Proc Natl Acad Sci U S A</em> 107(37): 16320-16324.</li><li>Cooper, T. A. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16314262" target="_blank">Use of minigene systems to dissect alternative splicing elements.</a> <em>Methods</em> 37(4): 331-340.</li><li>D'Ambrogio, A., Buratti, E., Stuani, C., Guarnaccia, C., Romano, M., Ayala, Y. M. and Baralle, F. E. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19429692" target="_blank">Functional mapping of the interaction between TDP-43 and hnRNP A2 <em>in vivo</em>.</a> <em>Nucleic Acids Res</em> 37(12): 4116-4126.</li><li>Gasset-Rosa, F., Lu, S., Yu, H., Chen, C., Melamed, Z., Guo, L., Shorter, J., Da Cruz, S. and Cleveland, D. W. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30853299" target="_blank">Cytoplasmic TDP-43 de-mixing independent of stress granules drives inhibition of nuclear import, loss of nuclear TDP-43, and cell death. </a><em>Neuron</em> 102(2): 339-357 e337.</li><li>Gerstberger, S., Hafner, M. and Tuschl, T. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25365966" target="_blank">A census of human RNA-binding proteins. </a><em>Nat Rev Genet </em>15(12): 829-845.</li><li>Gonatopoulos-Pournatzis, T., Wu, M., Braunschweig, U., Roth, J., Han, H., Best, A. J., Raj, B., Aregger, M., O'Hanlon, D., Ellis, J. D., Calarco, J. A., Moffat, J., Gingras, A. C. and Blencowe, B. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30388412" target="_blank">Genome-wide CRISPR-Cas9 interrogation of splicing networks reveals a mechanism for recognition of autism-misregulated neuronal microexons.</a> <em>Mol Cell</em> 72(3): 510-524 e512.</li><li>Gurskaya, N. G., Staroverov, D. B., Zhang, L., Fradkov, A. F., Markina, N. M., Pereverzev, A. P. and Lukyanov, K. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22259036" target="_blank">Analysis of alternative splicing of cassette exons at single-cell level using two fluorescent proteins.</a> <em>Nucleic Acids Res</em> 40(8): e57.</li><li>Harrison, A. F. and Shorter, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28389532" target="_blank">RNA-binding proteins with prion-like domains in health and disease. </a><em>Biochem J</em> 474(8): 1417-1438.</li><li>Hazen, A. L., Bushnell, T. and Haviland, D. L. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29371108" target="_blank">The importance of area scaling with FACS DIVA software.</a> <em>Methods</em> 134-135: 130-135.</li><li>Kino, Y., Washizu, C., Aquilanti, E., Okuno, M., Kurosawa, M., Yamada, M., Doi, H. and Nukina, N. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21109527" target="_blank">Intracellular localization and splicing regulation of FUS/TLS are variably affected by amyotrophic lateral sclerosis-linked mutations. </a><em>Nucleic Acids Res</em> 39(7): 2781-2798.</li><li>Roczniak-Ferguson, A. and Ferguson, S. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31527135" target="_blank">Pleiotropic requirements for human TDP-43 in the regulation of cell and organelle homeostasis.</a> <em>Life Sci Alliance</em> 2(5).</li><li>Schmidt, H. B., Barreau, A. and Rohatgi, R. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31653829" target="_blank">Phase separation-deficient TDP43 remains functional in splicing.</a> <em>Nat Commun</em> 10(1): 4890.</li><li>Sorenson, M. R. and Stevens, S. W. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24671766" target="_blank">Rapid identification of mRNA processing defects with a novel single-cell yeast reporter.</a> <em>RNA</em> 20(5): 732-745.</li><li>Sun, S., Ling, S. C., Qiu, J., Albuquerque, C. P., Zhou, Y., Tokunaga, S., Li, H., Qiu, H., Bui, A., Yeo, G. W., Huang, E. J., Eggan, K., Zhou, H., Fu, X. D., Lagier-Tourenne, C. and Cleveland, D. W. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25625564" target="_blank">ALS-causative mutations in FUS/TLS confer gain and loss of function by altered association with SMN and U1-snRNP.</a> <em>Nat Commun</em> 6: 6171.</li><li>Wang, Z., Gerstein, M. and Snyder, M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19015660" target="_blank">RNA-Seq: a revolutionary tool for transcriptomics.</a> <em>Nat Rev Genet</em> 10(1): 57-63.<br/></li></ol>