<body>
            <h1>Enzymatic Construction of Protein Polymer/Polyprotein Using OaAEP1 and TEV Protease</h1>
            <p><strong>Authors:</strong> Yibing Deng, Shengchao Shi, Bin Zheng, Tao Wu, Peng Zheng</p>
            <h2>Abstract</h2>
            <p>The development of chemical and biological coupling technologies in recent years has made possible of protein polymers engineering. We have developed an enzymatic method for building polyproteins using a protein ligase OaAEP1 (asparagine endopeptidase 1) and protease TEV (tobacco etching virus). Using a mobile TEV protease site compatible with the OaAEP1 ligation, we achieved a stepwise polymerization of the protein on the surface. The produced polyprotein can be verified by protein unfolding scenario using atomic force microscopy-based single-molecule force spectroscopy (AFM-SMFS). Thus, this study provides an alternative method for polyprotein engineering and immobilization.</p>
            <p><strong>Keywords:</strong> Polymerization, Enzyme, Single-molecule force spectroscopy, Polyprotein engineering, OaAEP1</p>
            <h2>Background</h2><p>Several methods based on biochemical reactions for protein polymerization have been developed. For example, one approach is to design protein monomer with additional cysteines as the basic unit for ligation (Dietz <em>et al.</em>, 2006; Durner <em>et al.</em>, 2017). Another approach is to build the complete gene for the artificial protein oligomer (Carrion-Vazquez <em>et al.</em>, 1999; Giganti <em>et al.</em>, 2018). However, the engineering of large-sized protein polymer is often challenging. To address this challenge, we have developed an enzyme-mediated method that builds protein polymers/polyproteins in a stepwise fashion, using protein ligase and protease (Yang <em>et al.</em>, 2017; Deng <em>et al.</em>, 2019). When a TEV site (ENLYFQ/G) plus a leucine (L) is designed as ENLYFQ/GL-POI (POI: Protein of Interest) at the N-terminus of the protein unit, TEV cleavage produces an N-terminal GL residue of GL-POI, which is compatible with further OaAEP1 ligation. Our enzymatic method provides a new method for the preparation of the polyprotein sample with a controlled sequence and suitable for single-molecule studies, especially for complex protein systems.<br/><br/>The engineered polyprotein sequence is confirmed by individual protein unfolding event using AFM-SMFS. Besides a powerful imaging tool (Mannix <em>et al.</em>, 2018), AFM can manipulate single molecule mechanically and directly measure its unfolding, unbinding, and rupture force (He <em>et al.</em>, 2012; Scholl and Marszalek, 2018; Zhang <em>et al.</em>, 2019). Thus, it is widely used to study protein (un)folding and mechanics (Infante <em>et al.</em>, 2019; Krieg <em>et al.</em>, 2019), protein-protein/ligand-receptor interaction (Ott <em>et al.</em>, 2017; Zhang <em>et al.</em>, 2019) and even chemical bond (Pill <em>et al.</em>, 2019; Song <em>et al.</em>, 2019; Yuan <em>et al.</em>, 2019). Together with other established polyprotein engineering and immobilization methods (Dietz <em>et al.</em>, 2006; Popa <em>et al.</em>, 2013; Hoffmann <em>et al.</em>, 2015; Walder <em>et al.</em>, 2017), our enzymatic methodology can improve the quality and efficiency of SMFS study.<br/></p>
        </body>

<p class="pptt" id="biaoti32871">## Materials and Reagents</p><ol class="npwtwok"><li>Glass coverslip (Sail Brand, China)</li><li><em>E. coli</em> BL21(DE3) (-80 Â°C)</li><li>pQE80L-POI or pET28a-POI plasmid (Vector information in the <a href="https://os.bio-protocol.org/attached/file/20200323/3596 Supplemental file.docx" target="_blank">Supplemetal file</a>)</li><li>Milli-Q water (18.2 MÎ©/cm)</li><li>Luria-Bertani (LB) medium (BD Difco)</li><li>Iron (III) chloride hexahydrate (99%, Energy chemical)</li><li>Calcium chloride hydrate crystalline aggregate (99.9965%, Alfa Aesar)</li><li>(3-aminopropyl) triethoxysilane (APTES, 99%, Sigma-Aldrich)</li><li>Sulfosuccinimidyl 4-(N-maleimidomethyl) cyclohexane-1-carboxylate (sulfo-SMCC, Thermo Scientific)</li><li>Sodium chloride (NaCl) (99%, BBL Life Science)</li><li>Tris(hydroxymethyl) aminomethane (Tris, &gt; 99.9%, Sangon Biotech)</li><li>EDTA (99.5%, Macklin)</li><li>5,5'-Dithiobis (2-nitrobenzoic Acid) (99%, Alfa Aesar)</li><li>Ampicillin sodium salt (99% Zhu YanBIO)</li><li>Kanamycin (â¥ 750 Î¼g/mg, Diamond)</li><li>Magnesium sulfate (MgSO<sub>4</sub>) (AR, Shanghai Lingfeng Chemical Reagent Co., Ltd)</li><li>Calcium chloride (CaCl<sub>2</sub>) (AR, Aladdin)</li><li>D(+)-Glucose (AR, Aladdin)</li><li>IPTG (Isopropyl-Î²-d-thiogalactopyranoside) (99% Zhu YanBIO)</li><li>DNase (from Bovine Pancreas, Activity â¥ 500 Kunitz U/mg, Sangon Biotech)</li><li>RNase (from Bovine Pancreas, Activity â¥ 60 U/mg, Sangon Biotech)</li><li>PMSF (Zhu YanBIO)</li><li>Potassium chromate (99.5%, Macklin)</li><li>Concentrated sulfuric acid (AR, Sinopharm Chemical Reagent Co.,Ltd)</li><li>Ethyl alcohol (AR, Sinopharm Chemical Reagent Co.,Ltd)</li><li>Imidazole (99% Aladdin)</li><li>Urea (AR, Sinopharm Chemical Reagent Co.,Ltd)</li><li>TEV protease (produced by ourselves)</li><li>OaAEP1 (produced by ourselves)</li><li><em>BamH</em>I (TaKaRa)</li><li><em>Bgl</em>II (TaKaRa)</li><li><em>Kpn</em>I (TaKaRa)</li><li>T4 Ligase (TaKaRa)</li><li>Glycerol (99%, Macklin)</li><li>M9 medium (see Recipes)</li><li>Chromic acid (see Recipes)</li><li>Lysis buffer (see Recipes)</li><li>Wash buffer (see Recipes)</li><li>Elution buffer (see Recipes)</li><li>DTNB solution (see Recipes)</li><li>TEV protease solution (see Recipes)</li><li>AFM measurement buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32872">### Equipment</p><ol class="npwtwok"><li>Avanti JXN-30 Centrifuge (Beckman Coulter)</li><li><a class="blackNoUnder" target="_blank"> AKTA FPLC system </a> (GE Healthcare)</li><li>Mono Q 5/50 GL (GE Healthcare)</li><li>NanoDrop 2000 spectrophotometer (Thermo Scientific)</li><li>Nanowizard 4 AFM (JPK Instruments AG)</li><li>Silicon nitride cantilever (MLCT, Bruker Corp)</li><li>Sonictor (Biosafer 650-92)<br/></li></ol><p class="pptt" id="biaoti32873">### Software</p><ol class="npwtwok"><li>JPK data processing (developed by JPK Instruments AG)</li><li>Igor Pro 6.12 (Wavemetrics)<br/></li></ol><p class="pptt" id="biaoti32874">### Procedure</p><ol class="npwone"><li>Gene cloning<ol class="npwtwo"><li>Use three-restriction digestion enzyme system <em>BamH</em>I-POI-<em>Bgl</em>II-stop codon-<em>Kpn</em>I for connecting the gene of different protein fragments. In this enzyme system use <em>Bgl</em>II and <em>Kpn</em>I digest to produce vector and use <em>BamH</em>I and <em>Kpn</em>I digest to produce the insert. As <em>BamH</em>I digestion and <em>Bgl</em>II digestion leads to the same cohesive end (GATC), it is possible to use T4 ligase connect the vector and the insert.</li><li>Confirm all genes by sequencing.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Expression and production of interest proteins<br/><em>Note: In the experimental operation, always pay attention to the aseptic operation. Different proteins have different E. coli harvest shelf life. It is necessary to test the shelf life for the specific protein.</em> <ol class="npwtwo"><li>Transform the pQE80L-POI or pET28a-POI plasmid into <em>E. coli</em> BL21(DE3) cell.</li><li>Apply the bacterial solution to the LB plate containing ampicillin sodium salt (100 Î¼g/ml) for pQE80L plasmid or kanamycin (50 Î¼g/ml) for pET28a plasmid and incubate at 37 Â°C for 14-16 h.</li><li>Pick single colonies into 15 ml LB medium containing ampicillin sodium salt (100 Î¼g/ml) or kanamycin (50 Î¼g/ml). Keep shaking at 200 rpm at 37 Â°C for 16-20 h.</li><li>After grown saturation, dilute the overnight cultures into 800 ml fresh LB media (add to the LB medium in a ratio of 1:50) containing respective antibiotics (the concentration of antibiotics is the same as above). As for the metalloprotein, centrifuged the culture at 1,800 <em>x g</em> and resuspended the precipitation by 16 ml M9 medium, and then add to 800 ml respective antibiotics containing M9 medium as well.</li><li>Shake culture at 200 rpm at 37 Â°C until the optical density at 600 nm (OD<sub>600</sub>) reaches 0.6-0.8 (this takes about 2.5-3 h, while M9 medium always takes about 5-6 h). Reserve a 100-Î¼l sample of the culture as the pre-induction control for testing protein expression.</li><li>Add IPTG (Isopropyl-Î²-d-thiogalactopyranoside) to a final concentration of 1 mM to induce protein expression, while metalloprotein always needs some additional metal ion, and shake the culture at 200 rpm at 37 Â°C for 4-6 h.</li><li>Harvest the culture at 13,000 <em>x g</em> for 25 min at 4 Â°C and store at -80 Â°C before purification.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Purification of interest proteins<ol class="npwtwo"><li>Resuspend the cells in 25 ml lysis buffer (contained 0.06 mg DNase, 0.06 mg RNase, 0.08 mg PMSF) and produce the lysis using a Biosafer sonicator (15% amplitude) on ice for 30 min. Clarify the cell lysate at 19,600 <em>x g</em> for 40 min at 4 Â°C and collect the supernatant fraction.</li><li>Pack a Poly-Prep column with 1-1.5 ml (bed volume) of Co-NTA or Ni-NTA affinity column and wash it with ten column volumes (CV) of Milli-Q water and then 10 CVs wash buffer (by gravity flow).</li><li>Load the supernatant onto the column by gravity flow and collect the flow-through. Load the flow-through onto the column by gravity and collect the secondary flow-through. Load the secondary flow-through onto the column by gravity and finish the sample loading.</li><li>Wash the column with 50 CVs of wash buffer.</li><li>Elute the bound protein with 3 CVs of elution buffer at 4 Â°C for 15 min. Moreover, carry out anion exchange purification for pure metal form rubredoxin protein using AKTA with Mono Q column at pH 8.5 at 4 Â°C.</li><li>Analyze the samples of the eluate and control group by SDS-PAGE.</li><li>Use the Ellman method to confirm the concentration of ELPs, in which 10 Î¼l protein incubated with 20 Î¼l DTNB solution for 30 min at room temperature and the solution composited with 10 Î¼l the elution buffer solution and 20 Î¼l DTNB solution served as blank to detect the absorbance of protein and DTNB reaction solution at 412 nm (Îµ = 13,700 M<sup>â1</sup> cm<sup>â1</sup>) by NanoDrop 2000 spectrophotometer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Functionalized coverslip surface preparation<br/><em>Note: The functionalized coverslip should be immediately used or stored at 4âÂ°C for a month after preparation.</em> <ol class="npwtwo"><li>Clean and activate glass coverslips by immersing in chromic acid at 80 Â°C for 30 min. Wash away chromic acid traces on the coverslips by water and then ethyl alcohol. Dry the coverslips by a stream of nitrogen.</li><li>Aminosilylate the coverslips by immersing them in 1% (v/v) APTES toluene solution for 1âh at room temperature avoiding light.</li><li>Wash the coverslips with toluene and absolute ethyl alcohol and dry the coverslips with nitrogen.</li><li>Bake the coverslips at 80âÂ°C for 15âmin.</li><li>Add 200 Î¼l sulfo-SMCC (1âmg/ml) dimethyl sulfoxide (DMSO) solution between two immobilized coverslips after the coverslips cooling down to room temperature and incubate for 1 h protected from light at room temperature.</li><li>Wash every piece of the coverslips with 15 ml DMSO first and then with absolute ethyl alcohol to remove residual sulfo-SMCC. Use a stream of nitrogen to dry the coverslips.</li><li>Stick cleaned quartz ring on the functioned side of the coverslip to build a chamber.</li><li>Pipet 200âÎ¼l of 200âÎ¼M GL-ELP<sub>50nm</sub>-C protein solution onto the functionalized coverslip in the chamber and incubate for approximately 3 h.</li><li>Wash the chamber with Milli-Q water to remove the unreacted GL-ELP<sub>50nm</sub>-C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Functionalized cantilevers surface preparation<br/><em>Note: Silicon nitride cantilever was used as a force probe. The surface chemistry of the cantilevers was similar to that of the coverslip.</em> <ol class="npwtwo"><li>Immerse the cantilevers at 80 Â°C for 10 min by chromic acid treatment. Remove the traces of chromic acid by water first and then ethyl alcohol. Use a piece of filter paper blot ethyl alcohol traces up.</li><li>Functionalize the cleaned cantilevers by amino-silanization with APTES and then conjugate the cantilevers to sulfo-SMCC as the method mentioned in Section D, Functionalized coverslip surface preparation.</li><li>Immerse the functionalized cantilever with Cys-ELP<sub>50nm</sub>-NGL at the concentration of 200âÎ¼M to the surface with maleimide group of sulfo-SMCC for 1.5âh.</li><li>Wash the coverslip with Milli-Q water to remove the unreacted Cys-ELP<sub>50nm</sub>-NGL and store the cantilever at -20 Â°C.</li><li>Before the AFM experiment, immerse a ELP-functionalized cantilever in 50âÎ¼M GL-CBM-XDoc protein solution with 200ânM OaAEP1 for 20-30 min at 25 Â°C, and then wash away the unreacted proteins by 15-20 ml AMF buffer.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Stepwise polyprotein preparation with controlled sequences<br/><em>Note: Use 15-20 ml AFM buffer to wash away residual proteins after each reaction step. Figure 1 presents the schematic diagram of the controlled sequence polyprotein preparation.</em> <ol class="npwtwo"><li>Mix unit Coh-tev-L-POI-NGL(50 Î¼M) and OaAEP1 (200 nM) and add 60 Î¼l to the functionaliazed coverslip. link the ligation unit Coh-tev-L-POI-NGL to the functionalized coverslip.</li><li>Add 100âÎ¼l TEV protease solution to cleave the TEV site in the protein unit for about 1âh at 25â Â°C. And GL-Ub-NGL-glass will produce with exposed N-terminal GL residues.</li><li>Repeat Steps F1-F2 (N-1) times to get the protein-polymer GL-(Ub)<sub>N</sub>-NGL-Glass. After OaAEP ligation of the final ligation unit, do not take the TEV cleavage reaction to obtain Coh-tev-L-(Ub)<sub>n</sub>-NGL-Glass.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414025729_6411.jpg" width="300"/><br/><strong>Figure 1. The schematic displays the stepwise ligation and cleavage procedure to produce sequence-controlled polyprotein on the glass surface and probed with an XDoc-protein maker-modified cantilever</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>AFM measurements<br/><em>Note: The AFM experiment parameters (Z length, Z speed, setpoint, sample rate and duration time) are adjustable in different experiments.</em> <ol class="npwtwo"><li>Add 1 ml AFM buffer containing10 mM CaCl<sub>2</sub> to the chamber (Figure 2).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414025747_3732.jpg" width="300"/><br/><strong>Figure 2. A photo image about AFM chamber with 1 ml AFM buffer solution.</strong> The black ring shows the position of the polyprotein.<br/><br/></li><li>Move the laser to the D tip of the functionalized probe (Figure 3). Use the equipartition theorem to calibrate the spring constant (Îº) of each cantilever in solution with an accurate value before the experiment.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414025805_1566.jpg" width="300"/><br/><strong>Figure 3. CCD picture of the cantilever tip in the AFM experiment.</strong> The triangle which the red ring located is D tip. The red ring points out the lase radiation position on D tip.<br/><br/></li><li>The cantilever tip attached to the sample surface and hold at this position for a while to formed the cohesin/dockerin pair as shown in Figure 4, segment 0 and segment 1.</li><li>Retracted the cantilever from the surface at a constant velocity of 400 nm s<sup>â1</sup> while recording the distance and cantilever deflection at a sampling rate of 4,096 Hz. As shown in Figure 4, segment 2.</li><li>Hold the cantilever for a while to relax the cohesin/dockerin pair as shown in Figure 4, segment 3.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414025821_4369.jpg" width="300"/><br/><strong>Figure 4. The dialog about the AFM experiment parameter setting.</strong> One AFM measurement contain four step. Segment 0 makes cantilever move towards the sample surface. Segment 1 keep the cantilever staying at the height reached by segment 0 for duration time. Segment 2 retract the cantilever from the surface and the protein unfold during this period. Segment 3 keep the cantilever at the height reached by segment 2 for duration time.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol><p class="pptt" id="biaoti32875">### Data analysis</p><ol class="npwtwok"><li>Use JPK data processing select force-extension traces.</li><li>Use Igor Pro 6.12 (Wavemetrics) analysis the data traces. Only the data traces containing all the specific contour length increments belong to fingerprint protein domains [GB1(H18 nm) and CBM (H58 nm)] and a high rupture force from the cohesin-dockerin dissociation meet the requirement (more than 200-300 pN) (Figure 5). Fit The curves by the worm-like-chain (WLC) model of polymer elasticity with the persistence length of ~0.4 nm.</li><li>Use Gauss model fit the histograms of unfolding forces to determine values of the most probable unfolding force (<f>) and contour length increment (&lt;âLc&gt;).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414025836_7948.jpg" width="300"/><br/><strong>Figure 5. Typical force versus distance trace for the sequence-controlled polyprotein, Coh-Ub</strong><sub><strong>4</strong></sub><strong>, containing signals of CBM unfolding, dockerin-cohesin pair rupture and four copies of Ub</strong><br/></f></li></ol><p class="pptt" id="biaoti32876">### Recipes</p><ol class="npwtwok"><li>Lysis buffer<br/>50âmM Tris<br/>150 mM NaCl<br/>pH 7.4</li><li>Wash buffer<br/>20âmM Tris<br/>400âmM NaCl<br/>2âmM imidazole<br/>pH 7.4</li><li>Elution buffer<br/>20âmM Tris<br/>400âmM NaCl<br/>250âmM imidazole<br/>pH 7.4</li><li>DTNB solution<br/>0.5 mM 5,5'-Dithiobis (2-nitrobenzoic Acid)<br/>1 mM EDTA<br/>7.2 M Urea</li><li>TEV protease solution<br/>0.4âmg/ml TEV protease<br/>75âmM NaCl<br/>0.5âmM EDTA<br/>25âmM Tris-HCl, pH 8.0<br/>10% [v/v] glycerol</li><li>AFM measurement buffer<br/>100âmM Tris<br/>100âmM NaCl<br/>pH 7.4</li><li>Chromic acid<br/>20 g potassium chromate<br/>40 ml ultrapure water<br/>360 ml concentrated sulfuric acid</li><li>M9 medium<br/>0.4% glucose<br/>0.1 mM CaCl<sub>2</sub><br/>2 mM MgSO<sub>4</sub><br/></li></ol><p class="pptt" id="biaoti32877">### Acknowledgments</p><p class="ppzz">National Natural Science Foundation of China (Grant Nos. 21771103, 21977047), Natural Science Foundation of Jiangsu Province (Grant No. BK20160639). This protocol was modified from previous work âEnzymatic biosynthesis and immobilization of polyprotein verified at the single-molecule level (Deng <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti32878">### Competing interests</p><p class="ppzz">The authors declare no conï¬ict of interest.</p><p class="pptt" id="biaoti32879">### References</p><ol class="npwtwok"><li>Carrion-Vazquez, M., Oberhauser, A. F., Fowler, S. B., Marszalek, P. E., Broedel, S. E., Clarke, J. and Fernandez, J. M. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10097099" target="_blank">Mechanical and chemical unfolding of a single protein: A comparison.</a> <em>Proc Natl Acad Sci U S A</em> 96(7): 3694-3699. </li><li>Deng, Y., Wu, T., Wang, M., Shi, S., Yuan, G., Li, X., Chong, H., Wu, B. and Zheng, P. (2019). <a href="https://www.nature.com/articles/s41467-019-10696-x" target="_blank">Enzymatic biosynthesis and immobilization of polyprotein verified at the single-molecule level.</a> <em>Nat Commun</em> 10(1): 2775. </li><li>Dietz, H., Bertz, M., Schlierf, M., Berkemeier, F., Bornschlogl, T., Junker, J. P. and Rief, M. (2006). <a href="https://www.nature.com/articles/nprot.2006.12" target="_blank">Cysteine engineering of polyproteins for single-molecule force spectroscopy.</a> <em>Nat Protoc</em> 1(1): 80-84.</li><li>Durner, E., Ott, W., Nash, M. A. and Gaub, H. E. (2017). <a href="https://pubs.acs.org/doi/full/10.1021/acsomega.7b00478" target="_blank">Post-translational sortase-mediated attachment of high-strength force spectroscopy handles.</a> <em>Acs Omega</em> 2(6): 3064-3069. </li><li>Giganti, D., Yan, K., Badilla, C. L., Fernandez, J. M. and Alegre-Cebollada, J. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29330363" target="_blank">Disulfide isomerization reactions in titin immunoglobulin domains enable a mode of protein elasticity.</a> <em>Nat Commun</em> 9(1): 185. </li><li>He, C., Genchev, G. Z., Lu, H. and Li, H. (2012). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3663486/" target="_blank">Mechanically untying a protein slipknot: multiple pathways revealed by force spectroscopy and steered molecular dynamics simulations.</a> <em>J Am Chem Soc</em> 134(25): 10428-10435. </li><li>Hoffmann, T., Tych, K. M., Crosskey, T., Schiffrin, B., Brockwell, D. J. and Dougan, L. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26284289" target="_blank">Rapid and robust polyprotein production facilitates single-molecule mechanical characterization of Î²-Barrel assembly machinery polypeptide transport associated domains.</a> <em>ACS Nano</em> 9(9): 8811-8821.</li><li>Infante, E., Stannard, A., Board, S. J., Rico-Lastres, P., Rostkova, E., Beedle, A. E. M., Lezamiz, A., Wang, Y. J., Gulaidi Breen, S., Panagaki, F., Sundar Rajan, V., Shanahan, C., Roca-Cusachs, P. and Garcia-Manyes, S. (2019). <a href="https://www.nature.com/articles/s41567-019-0551-3" target="_blank">The mechanical stability of proteins regulates their translocation rate into the cell nucleus.</a> <em>Nat Phys</em> 15: 973-981.</li><li>Krieg, M., FlÃ¤schner, G., Alsteens, D., Gaub, B. M., Roos, W. H., Wuite, G. J. L., Gaub, H. E., Gerber, C., DufrÃªne, Y. F. and MÃ¼ller, D. J. (2019). <a href="https://www.nature.com/articles/s42254-018-0001-7" target="_blank">Atomic force microscopy-based mechanobiology.</a> <em>Nat Rev Phys</em> 1(1): 41-57. </li><li>Mannix, A. J., Zhang, Z. H., Guisinger, N. P., Yakobson, B. I. and Hersam, M. C. (2018). <a href="https://www.nature.com/articles/s41565-018-0157-4" target="_blank">Borophene as a prototype for synthetic 2D materials development.</a> <em>Nat Nanotech</em> 13(6): 444-450. </li><li>Ott, W., Jobst, M. A., Schoeler, C., Gaub, H. E. and Nash, M. A. (2017). <a href="http://www.sciencedirect.com/science/article/pii/S1047847716300260" target="_blank">Single-molecule force spectroscopy on polyproteins and receptorâligand complexes: The current toolbox.</a> <em>J Stru Bio</em> 197(1): 3-12. </li><li>Pill, M. F., East, A. L. L., Marx, D., Beyer, M. K. and Clausen-Schaumann, H. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31112349" target="_blank">Mechanical activation drastically accelerates amide bond hydrolysis, matching enzyme activity.</a> <em>Angew Chem Int Ed</em> 58(29): 9787-9790. </li><li>Popa, I., Berkovich, R., Alegre-Cebollada, J., Badilla, C. L., Rivas-Pardo, J. A., Taniguchi, Y., Kawakami, M. and Fernandez, J. M. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23909704" target="_blank">Nanomechanics of HaloTag Tethers.</a> <em>J Am Chem Soc</em> 135(34): 12762-12771. </li><li>Scholl, Z. N. and Marszalek, P. E. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29956225" target="_blank">AFM-based single-molecule force spectroscopy of proteins.</a> <em>Methods Mol Biol</em> 1814: 35-47. </li><li>Song, Y., Ma, Z., Yang, P., Zhang, X., Lyu, X., Jiang, K. and Zhang, W. (2019). <a href="https://pubs.acs.org/doi/10.1021/acs.macromol.8b02702" target="_blank">Single-molecule force spectroscopy study on force-induced melting in polymer single crystals: the chain conformation matters.</a> <em>Macromolecules</em> 52(3): 1327-1333. </li><li>Walder, R., LeBlanc, M. A., Van Patten, W. J., Edwards, D. T., Greenberg, J. A., Adhikari, A., Okoniewski, S. R., Sullan, R. M. A., Rabuka, D., Sousa, M. C. and Perkins, T. T. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28677396" target="_blank">Rapid characterization of a mechanically labile Î±-helical protein enabled by efficient site-specific bioconjugation.</a> <em>J Am Chem Soc</em> 139(29): 9867-9875. </li><li>Yang, R., Wong, Y. H., Nguyen, G. K. T., Tam, J. P., Lescar, J. and Wu, B. (2017). <a href="https://pubs.acs.org/doi/10.1021/jacs.6b12637" target="_blank">Engineering a catalytically efficient recombinant protein ligase.</a> <em>J Am Chem Soc</em> 139(15): 5351-5358.</li><li>Yuan, G., Liu, H., Ma, Q., Li, X., Nie, J., Zuo, J. and Zheng, P. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31433648" target="_blank">Single-molecule force spectroscopy reveals that iron-ligand bonds modulate proteins in different modes.</a> <em>J Phys Chem Lett</em> 10(18): 5428-5433. </li><li>Zhang, S., Qian, H.-j., Liu, Z., Ju, H., Lu, Z.-y., Zhang, H., Chi, L. and Cui, S. (2019). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/anie.201811152" target="_blank">Towards unveiling the exact molecular structure of amorphous red phosphorus by single-molecule studies.</a> <em>Angew Chem Int Ed</em> 58(6): 1659-1663. <br/><br/></li></ol>