<body>
            <h1>Dual Fluorescence Cytometry Assay to Assess Cellular Protein Levels</h1>
            <p><strong>Authors:</strong> Ashley M Ngo, Ruofan  Wang, Andreas S Puschnik</p>
            <h2>Abstract</h2>
            <p>Expression levels of cellular proteins can be affected by various perturbations, such as genetic knockout of interactors, drug treatments or cell stress. To specifically measure the effects on protein levels post-synthesis under different experimental conditions, it is important to compensate for transcriptional and other upstream changes. Here, we provide a protocol for a dual-fluorescence flowcytometry-based assay to determine protein levels. The protein of interest is genetically linked to enhanced GFP (eGFP) followed by a viral 2A self-cleaving peptide sequence and mCherry. As a result, translation of the reporter construct leads to two fluorescent protein products from the same mRNA template, which enables unambiguous protein expression analysis with normalization across samples.</p>
            <p><strong>Keywords:</strong> Protein expression, Protein stability, Flow cytometry, Fluorescent proteins, Transmembrane proteins, ER membrane protein complex</p>
            <h2>Background</h2><p>Synthesis and maintenance of cellular proteins depend on multiple processes, from transcriptional regulation, processing and degradation of mRNA to translation, folding, localization, post-translational modification and protein degradation (Vogel and Marcotte, 2012). To specifically study the effects of cellular perturbations on protein levels post-synthesis, it is important to compensate for variability in upstream steps of protein expression. Here, we provide a protocol for a dual-fluorescence flow cytometry-based assay to determine protein levels at steady state as previously described (Itakura <em>et al.</em>, 2016; Chitwood <em>et al.</em>, 2018; Ngo <em>et al.</em>, 2019). The protein of interest is genetically fused to enhanced GFP (eGFP) followed by a viral 2A self-cleaving peptide sequence and a second fluorescent protein, mCherry (Figure 1). Translation of the fusion construct generates two protein products at a 1:1 ratio due to peptide bond skipping by the ribosome at the 2A site: the protein of interest fused to eGFP and mCherry. As both proteins are produced from the same transcribed mRNA molecule and under control by the same promoter, this system allows assessment of the abundance of the protein with normalization of transcriptional differences or variability in transfection efficiency. Furthermore, the flow cytometry-based single-cell readout may enable the detection of more subtle differences in protein levels between experimental conditions, which could otherwise be masked in bulk readouts, such as immunoblot.<br/><br/>We demonstrated the use of this method to study the expression levels of two multi-pass transmembrane proteins, beta1-adrenergic receptor (ADRB1) and the dengue virus non-structural proteins (NS) 4A and 4B, in relation to functional ER membrane protein complex (EMC) expression as previously published (Chitwood <em>et al.</em>, 2018; Ngo <em>et al.</em>, 2019). The obtained data shows that EMC is important in facilitating stable expression of these transmembrane proteins (Figure 2).<br/><br/>This method can thus be used to study the expression levels of specific proteins of interest under different cellular conditions, such as genetic perturbations, drug treatments or other cellular stress.<br/></p>
        </body>

<p class="pptt" id="biaoti32882">## Materials and Reagents</p><ol class="npwtwok"><li>12-well cell culture plates (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12556005 </a> )</li><li>Falcon 5 ml Round-Bottom Tubes with 35 Î¼m strainer lids (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 352235 </a> )</li><li>293FT cells (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> R70007 </a> )</li><li>EMC4-KO 293FT cells (generated as described in PMID: 31516121)</li><li><a class="blackNoUnder" target="_blank"> Stbl3 </a> competent <em>E. coli</em> (MacroLab, catalog number: <a class="blackNoUnder" target="_blank"> Stbl3 </a> )</li><li>eGFP coding sequence, amplified from pX458 (gift from Feng Zhang; Addgene plasmid, catalog number: <a class="blackNoUnder" target="_blank"> 48138 </a> )</li><li>mCherry coding sequence, amplified from pCAGGS-mCherry (gift from Phil Sharp; Addgene plasmid, catalog number: <a class="blackNoUnder" target="_blank"> 41583 </a> )</li><li>ADRB1 coding sequence, amplified from pcDNA3 Flag beta-1-adrenergic-receptor (gift from Robert Lefkowitz; Addgene plasmid, catalog number: <a class="blackNoUnder" target="_blank"> 14698 </a> )</li><li>Dengue virus serotype 2 16681 strain NS4A-NS4B coding sequence, amplified from infectious clone (gift from Karla Kirkegaard, NCBI Reference Sequence: NC_001474)</li><li>Kapa HiFi HotStart ReadyMix (Kapa Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> KK2602 </a> )</li><li>DNA oligos (Integrated DNA Technologies, see Table 1 for sequences)<br/><br/><strong>Table 1. DNA oligo sequences</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414033543_1111.jpg" width="600"/><br/><br/></li><li>QIAquick Gel Extraction Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 28706 </a> )</li><li>Molecular Biology Agarose (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1613102 </a> )</li><li>NEBuilder<sup>Â®</sup> HiFi DNA Assembly Master Mix (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> E2621L </a> )</li><li>pLenti CMV Puro DEST (w118-1) (gift from Eric Campeau and Paul Kaufman; Addgene plasmid, catalog number: <a class="blackNoUnder" target="_blank"> 17452 </a> )</li><li>EcoRV-HF (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R3195S </a> )</li><li>Quick CIP (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0525S </a> )</li><li>S.O.C. Medium (Teknova, catalog number: <a class="blackNoUnder" target="_blank"> S1657 </a> )</li><li>LB agar plates with carbenicillin (Teknova, catalog number: <a class="blackNoUnder" target="_blank"> L1010 </a> )</li><li>LB Broth (Teknova, catalog number: <a class="blackNoUnder" target="_blank"> L8050 </a> )</li><li>Carbenicillin 100 mg/ml (Teknova, catalog number: <a class="blackNoUnder" target="_blank"> C2135 </a> )</li><li>QIAfilter Plasmid Midi Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 12243 </a> )</li><li>Dulbecco's modified Eagle medium (DMEM) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11965118 </a> )</li><li>Fetal bovine serum (FBS) (Omega Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FB-11 </a> )</li><li>Opti-MEM reduced serum medium (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 31985062 </a> )</li><li>Penicillin-Streptomycin 100x (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 15140122 </a> )</li><li>L-glutamine 100x (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 25030081 </a> )</li><li>Dulbecco's phosphate-buffered saline (DPBS) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14190-250 </a> )</li><li>Trypsin-EDTA 0.05% (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 25300120 </a> )</li><li>TransIT<sup>Â®</sup>-LT1 Transfection Reagent (Mirus Bio, catalog number: <a class="blackNoUnder" target="_blank"> MIR2300 </a> )</li><li>Lipofectamine 3000 Transfection Reagent (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> L3000008 </a> )</li><li>Culture medium for 293FT (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32883">### Equipment</p><ol class="npwtwok"><li>Nanodrop One Microvolume UV-Vis Spectrophotometers (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> ND-ONE-W </a> )</li><li>CytoFLEX S V4-B2-Y4-R3 Flow Cytometer with a 488 and 561nm lasers (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> B96621 </a> )</li><li>Shaker</li><li>Waterbath<br/></li></ol><p class="pptt" id="biaoti32884">### Software</p><ol class="npwtwok"><li>CytExpert Version 2.3 (Beckman Coulter Inc., <a href="https://www.beckmancoulter.com/" target="_blank">https://www.beckmancoulter.com/</a>)</li><li>FlowJo v10.6.1 (FlowJo, LLC, <a href="https://www.flowjo.com/" target="_blank">https://www.flowjo.com/</a>)</li><li>Microsoft Excel (Microsoft Corp., <a href="https://www.microsoft.com/en-us/" target="_blank">https://www.microsoft.com/en-us/</a>)</li><li>Prism 8 (GraphPad Software, LLC, <a href="https://www.graphpad.com/" target="_blank">https://www.graphpad.com/</a>)<br/></li></ol><p class="pptt" id="biaoti32885">### Procedure</p><ol class="npwone"><li>Cloning of dual fluorescence reporter constructs<ol class="npwtwo"><li>Design primers for the amplification of the gene of interest (here: ADRB1 or dengue NS4A-4B), eGFP and mCherry with the addition of 20-30 bp overlaps between the neighboring fragments for assembly of multiple DNA fragments. The forward primer for the gene of interest contains an overlap with left arm of EcoRV-linearized pLenti CMV Puro DEST (w118-1), a Kozak sequence and an ATG start codon. The reverse primer overlaps with the 5â² end of the eGFP coding sequence. The reverse primer for eGFP has no stop codon and contains an overhang encoding the first ~40 bp of T2A thus creating an overlap with the T2A-mCherry fragment. mCherry is amplified with a forward primer containing parts of the T2A sequence and a reverse primer with stop codon and overlap to the right arm of EcoRV-linearized pLenti CMV Puro DEST (w118-1). A control construct, which only encodes eGFP-T2A-mCherry, can be generated, where the forward primer for eGFP contains an overlap with left arm of EcoRV-linearized pLenti CMV Puro DEST (w118-1), a Kozak sequence and an ATG start codon.</li><li>Amplify DNA fragments from plasmid templates using the KAPA HiFi HotStart ReadyMix (or any other high-fidelity polymerase). Use PCR cycling conditions as recommended by the manufacturer. After PCR, run amplified fragments on a 1% agarose gel followed by extraction using a gel purification kit.</li><li>Assemble purified PCR fragments together with EcoRV-linearized and CIP-treated plenti-CMV-Puro plasmid using NEBuilder<sup>Â®</sup> HiFi DNA Assembly Master Mix according to the manufacturerâs instructions. Note that other mammalian expression vectors can be used as recipient plasmid but you need to modify the overhang sequences for assembly of multiple DNA fragments in Step A1 accordingly.</li><li>Transform <a class="blackNoUnder" target="_blank"> Stbl3 </a> <em>E. coli</em> with assembled plasmids by heat shocking at 42 Â°C waterbath for 45 s. Recover heat shocked bacteria in S.O.C. medium and 30 Â°C shaker for 1 h at 225 RPM. Spread transformed bacteria on 10 mm agar plates that contain 100 Î¼g/ml Carbenicillin and incubate at 30 Â°C or 37 Â°C overnight.</li><li>The next day, pick a single colony from the plate and inoculate 50 ml of LB broth that contains 100 Î¼g/ml Carbenicillin. Shake inoculated LB broth in 30 Â°C (if plasmid sequence is prone to recombination, such as lentiviral plasmids) or 37 Â°C shaker at 225 RPM overnight.</li><li>Extract plasmid DNA from <em>E. coli</em> using a plasmid DNA Midiprep kit according to the manufacturerâs instruction. Determine concentration and purity (260/280 ratio) of nucleic acid using Nanodrop. Sequence-verify plasmid insert by Sanger sequencing using a forward primer upstream of eGFP and a reverse primer downstream of mCherry.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transfection of cell lines<ol class="npwtwo"><li>Plate ~3 x 10<sup>5</sup> 293FT cells (WT and EMC4 KO genotypes) in 12-well plates the day before transfection using 293FT culture medium (see Recipes). Cells should be 70%-80% confluent 24âh after plating.</li><li>The next day, transfect dual fluorescence constructs into 293FT cells using Mirus TransIT<sup>Â®</sup>-LT1 Transfection Reagent or Lipofectamine 3000. To do so, dilute required amount of plasmid DNA in serum-free media or OptiMEM and add the transfection reagent according to the manufacturerâs instructions. Then, add DNA-lipid complexes dropwise to cells.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Flow cytometry analysis<ol class="npwtwo"><li>After 24-48 h, detach cells by adding 100 Î¼l 0.05% Trypsin-EDTA per well followed by incubation at 37 Â°C for 3 min. Inactivate trypsin by adding an equal volume of culture medium containing FBS. Pass cells through a 35 Î¼m cell strainer to obtain a single-cell suspension.</li><li>Analyze cells on a flow cytometer. First, use untransfected cells to draw negative gates. Gate for forward (FSC) and side scatter (SSC), then height and area of FSC for a singlet gate, then mCherry fluorescence (mCherry positive cells are the successfully transfected cells) and finally eGFP fluorescence. Then run your samples (eGFP-2A-mCherry control and protein of interest constructs). Record 5 x 10<sup>3</sup>-5 x 10<sup>4</sup> mCherry-positive events per transfected sample.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414032834_0118.jpg" width="300"/><br/><strong>Figure 1. Experimental design for dual fluorescence cytometry assay.</strong> A. Schematic representation of mammalian expression cassette showing gene of interest followed by eGFP, a viral T2A sequence and mCherry. The primers to generate DNA fragments containing overhangs for plasmid construction are indicated by black arrows (1F = PLENTI-CMV-ADRB1-F or PLENTI-CMV-DENV_NS4A-F; 1R = ADRB1(NOSTOP)_GFP-R or DENV_NS4B_GFP-R; 2F = EGFP-F; 2R=EGFP-T2A_GIBSON-R; 3F = T2A-MCHERRY-F; 3R = MCHERRY-PLENTI-R). Transcribed mRNA molecules encode both fluorescent proteins, which are then translated into two protein products by peptide bond skipping at the T2A site. This results in the production of the GFP fusion protein and mCherry at an initial 1:1 stoichiometry. Thus, the eGFP:mCherry ratio allows measuring effects on protein levels post-synthesis. B. Experimental workflow. eGFP-T2A-mCherry reporter constructs are transfected into cells of different experimental conditions (<em>e.g.</em>, wild-type vs knockout genotypes, drug treatments, cell stress <em>etc.</em>). Cells are then analyzed for GFP and mCherry fluorescence levels using flow cytometry. Lastly, eGFP:mCherry ratios are calculated for the protein of interest under the different experimental conditions and graphed as frequency distributions.</li></ol></li></ol><p class="pptt" id="biaoti32886">### Data analysis</p><ol class="npwtwok"><li>FCS files are exported from CytExpert cytometer software and imported in FlowJo 10 for FACS analysis. The gating order is as follows: FSC-H/SSC-H (cells vs debris), FSC-H/FSC-A (singlets), ECD-H (mCherry channel)/FSC-H and the FITC-H (eGFP channel)/FSC-H. First, use untransfected cells to set the negative gates for mCherry and eGFP fluorescence. Then apply all gates onto the remaining samples. The mCherry-positive cells represent the successfully transfected cells. As GFP and mCherry are expressed from the same construct/promoter, all mCherry positive cells also have GFP fluorescence. Export their relative intensity values of the mCherry positive gated populations as comma-separated values (CSV) files by right-click on the âmCherry positiveâ group and then âExport/Concatenate Groupâ.</li><li>Calculate the eGFP-to-mCherry ratios for each row in all datasheets (each row represents an individual recorded cell) using Microsoft Excel. As eGFP and mCherry have high structural similarity and are expressed under the same promoter, it can be assumed that their protein levels are identical. Thus, use the sample with the eGFP-2A-mCherry only construct and calculate the mean of all eGFP-to-mCherry ratios. The mean of the ratios will be applied as scaling factor to the eGFP-to-mCherry ratios of all other samples to normalize for differences of laser intensities and detector settings between the eGFP and mCherry channels.</li><li>Import the normalized eGFP-to-mCherry values for each sample to a column table in Prism 8. Select âAnalyzeâ and âFrequency distributionâ in order to generate a histogram of all calculated fluorescence ratios. Choose âFrequency distributionâ under Create and select âXY graphâ as output. Alternatively, histograms can be generated using other software packages, such as R or Python.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200414/20200414032848_5847.jpg" width="300"/><br/><strong>Figure 2. Example data for dual fluorescence cytometry assay.</strong> A. Gating strategy for transfected cells in FlowJo: forward and side scatter, singlets, mCherry fluorescence and eGFP fluorescence. The fluorescence intensity values for the individual mCherry-positive cells are exported for downstream analysis. B. Example of CSV table containing individual fluorescence intensity values for ADRB1-eGFP-T2A-mCherry construct transfected into wild-type 293FT cells. eGFP:mCherry ratios are calculated from intensity values and normalized by the mean of the eGFP:mCherry ratio of the eGFP-T2A-mCherry only construct. Similar tables are generated for other samples. eGFP:mCherry ratios for all samples are normalized to the mean of the ratios in the eGFP-T2A-mCherry sample. Finally, normalized ratios are imported to Prism 8 and histograms are generated. While eGFP-only levels are not affected by knockout of EMC4, ADRB1 and dengue NS4A-4B display reduced expression in cells lacking EMC4.<br/></li></ol><p class="pptt" id="biaoti32887">### Recipes</p><ol class="npwtwok"><li>Culture medium for 293FT<br/>500 ml Dulbecco's modified Eagle medium (DMEM)<br/>50 ml heat-inactivated FBS<br/>5 ml 100x Penicillin and streptomycin (final concentration 100 U/ml)<br/>5 ml 100x L-glutamine (final concentration 2 mM)<br/></li></ol><p class="pptt" id="biaoti32888">### Acknowledgments</p><p class="ppzz">This protocol is based on previous work and example data published in eLife (Ngo <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti32889">### Competing interests</p><p class="ppzz">The authors do not have any competing interests.</p><p class="pptt" id="biaoti32890">### References</p><ol class="npwtwok"><li>Chitwood, P. J., Juszkiewicz, S., Guna, A., Shao, S. and Hegde, R. S. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30415835" target="_blank">EMC is required to initiate accurate membrane protein topogenesis.</a> <em>Cell</em> 175(6): 1507-1519 e1516.</li><li>Itakura, E., Zavodszky, E., Shao, S., Wohlever, M. L., Keenan, R. J. and Hegde, R. S. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27345149" target="_blank">ubiquilins chaperone and triage mitochondrial membrane proteins for degradation.</a> <em>Mol Cell</em> 63(1): 21-33.</li><li>Ngo, A. M., Shurtleff, M. J., Popova, K. D., Kulsuptrakul, J., Weissman, J. S. and Puschnik, A. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31516121" target="_blank">The ER membrane protein complex is required to ensure correct topology and stable expression of flavivirus polyproteins.</a> <em>Elife</em> 8: e48469.</li><li>Vogel, C. and Marcotte, E. M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22411467" target="_blank">Insights into the regulation of protein abundance from proteomic and transcriptomic analyses.</a> <em>Nat Rev Genet</em> 13(4): 227-232.<br/><br/></li></ol>