<body>
            <h1>Sequence Alignment Using Machine Learning for Accurate Template-based Protein Structure Prediction</h1>
            <p><strong>Authors:</strong> Shuichiro Makigaki, Takashi Ishida</p>
            <h2>Abstract</h2>
            <p>Template-based modeling, the process of predicting the tertiary structure of a protein by using homologous protein structures, is useful when good templates can be available. Indeed, modern homology detection methods can find remote homologs with high sensitivity. However, the accuracy of template-based models generated from the homology-detection-based alignments is often lower than that from ideal alignments. In this study, we propose a new method that generates pairwise sequence alignments for more accurate template-based modeling. Our method trains a machine learning model using the structural alignment of known homologs. When calculating sequence alignments, instead of a fixed substitution matrix, this method dynamically predicts a substitution score from the trained model.<br/></p>
            <p><strong>Keywords:</strong> Template-based modeling, Homology modeling, Sequence alignment, Machine learning, <em>k</em>-Nearest Neighbor</p>
            <h2>Background</h2><p>Proteins are key molecules in biology, biochemistry and pharmaceutical sciences. To reveal the functions of proteins, it is essential to understand the relationships between proteins' structure and function. Protein structures can be determined by experimental; the protein structures are often registered to and accessible in the Protein Databank (PDB) (wwPDB consortium, 2018). However, despite improvements in experimental methods for determining protein structures, the speed at which amino acid sequences can be revealed has overtaken our ability to ascertain the corresponding proteins' structures (Muhammed <em>et al.</em> 2019). Therefore, protein structure prediction remains essential.<br/><br/>As one of various methods for protein structure prediction, template-based or homology modeling predicts structures based on templates and their sequence alignment to a target protein. Template structures are the structures of homologous proteins, often found by homology detection methods. Currently, template-based modeling methods are the most practical because the predicted models are often accurate if we can find good templates and protein sequence alignments. These accurate models by template-based modeling can be used for computer-aided drug design (CADD).<br/><br/>Indeed, recent homology search methods have been able to detect remote homologs (Boratyn <em>et al.</em>, 2012; Zimmermann <em>et al.</em>, 2018). Although, sometimes sufficiently accurate structure models cannot be obtained because the quality of the sequence alignment generated by homology detection program is poor. If a more accurate model is required, researchers must manually edit alignments to improve their quality before modeling. In structural alignment, the structural difference between a target protein structure and a template protein structure is minimized; thus, sequence alignments generated by structural alignment are almost ideal for template-based modeling. Often, the sequence alignments generated by the homology detection methods are dissimilar to those generated by structural alignment, especially for remote homologs. Thus far, a methodâs ability to detect remote homologs has been prioritized because models cannot be generated without a template. However, to achieve higher-accuracy template-based modeling, the improvement of sequence alignment generation is a critical open problem. This problem has been mentioned in several studies (Kopp <em>et al.</em>, 2007) in which researchers have tried to improve alignments manually based on their knowledge of biology; fully automated methods are still required.<br/><br/>Recently, machine learning methods have demonstrated power in various fields (Lyons <em>et al.</em>, 2014; Cao <em>et al.</em>, 2016; Wang, Peng, <em>et al.</em>, 2016; Wei and Zou, 2016; Manavalan and Lee, 2017; Wang, Sun, <em>et al.</em>, 2017). Machine learning also seems effective in tackling the problem of alignment generation for homology modeling. However, this topic has not been studied because it is challenging to treat alignment generation as a classification or regression problem.<br/><br/>For the problem, we proposed a new sequence alignment generation protocol based on a machine learning that learns the structural alignments of known homologs (Makigaki and Ishida, 2019). We use a dynamic programming algorithm during aligning sequences to dynamically predict a substitution score from the <em>k</em>-Nearest Neighbor (<em>k</em>-NN) model instead of a fixed substitution matrix or profile comparison. Machine learning is used in this substitution score prediction process.<br/><br/>The proposed method is valuable for researchers who use template-based modeling with remote homologs whose sequence identity is not high. In this paper, we show the overview of our method as a procedure, and more detailed usage of our tool and some examples are available in the source code repository <a href="https://github.com/shuichiro-makigaki/exmachina" target="_blank">(https://github.com/shuichiro-makigaki/exmachina</a>).<br/></p>
        </body>

<p class="pptt" id="biaoti32982">### Equipment</p><ol class="npwtwok"><li>Computer<br/>&gt; 128 GiB RAM and &gt; 150 GiB free storage are recommended</li><li>Linux (&gt; 3.10) or SUSE Linux Enterprise Server 12<br/></li></ol><p class="pptt" id="biaoti32983">### Software</p><ol class="npwtwok"><li>PSI-BLAST (&gt; v2.9)<br/>To generate PSSM of an amino acid sequence<br/>Download URL: <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download" target="_blank">https://blast.ncbi.nlm.nih.gov/Blast.cgi?CMD=Web&amp;PAGE_TYPE=BlastDocs&amp;DOC_TYPE=Download</a> (Last access date: 2020-02-22)<br/>Installation document URL: <a href="https://www.ncbi.nlm.nih.gov/books/NBK279690/" target="_blank">https://www.ncbi.nlm.nih.gov/books/NBK279690/</a> </li><li>TM-align (&gt; v20190822) (Zhang and Skolnick, 2005)<br/>To generate structural alignment of homologs<br/>Download and installation document URL: <a href="https://zhanglab.ccmb.med.umich.edu/TM-align/" target="_blank">https://zhanglab.ccmb.med.umich.edu/TM-align/</a> (Last access date: 2020-02-22)</li><li>Implementation: Source code and installation document are available in the source code repository.<br/>Download URL: <a href="https://github.com/shuichiro-makigaki/exmachina/archive/master.zip" target="_blank">https://github.com/shuichiro-makigaki/exmachina/archive/master.zip</a><br/>Installation Procedure: <a href="https://github.com/shuichiro-makigaki/exmachina#how-to-use" target="_blank">https://github.com/shuichiro-makigaki/exmachina#how-to-use</a><br/>(Last access date: 2020-02-22)<br/>Python 3.6: Required python packages are listed in the repository.</li><li>FLANN (Muja and Lowe, 2009): <em>k</em>-Nearest Neighbor implementation. The installation procedure also contains the FLANN installation document.</li><li>Structural Classification of Proteins (SCOP) database<br/>The SCOP database classifies proteins by class, folds, superfamily (SF), family and domain based on manually curated function/structure classifications and contains redundant sequences. Thus, we used the SCOP40 database instead, which contains only domains whose sequence identity is &lt; 40% to avoid overfitting and reduce execution time.<br/>Download URL: <a href="https://scop.berkeley.edu/astral/pdbstyle/ver=1.75" target="_blank">https://scop.berkeley.edu/astral/pdbstyle/ver=1.75</a> (Last access date: 2020-02-22)</li><li>UniRef (The UniProt Consortium, 2016) database<br/>For Position Specific Scoring Matrix (PSSM) generation, we used three-iteration PSI-BLAST (Altschul <em>et al.</em>, 1997) with the UniRef90 database.<br/>Download URL: <a href="https://www.uniprot.org/downloads#unireflink" target="_blank">https://www.uniprot.org/downloads#unireflink</a> (Last access date: 2020-02-22)<br/></li></ol><p class="pptt" id="biaoti32984">### Procedure</p><p class="ppzz">The primary purpose of the training phase is to generate <em>k</em>-NN model that will be used for substitution score prediction in the prediction and alignment generation phase. The prediction phase consists of score prediction and alignment generation. Figure 1 shows the overview of the method. More detailed step-by-step commands and the example are available at source code repository (<a href="https://github.com/shuichiro-makigaki/exmachina" target="_blank">https://github.com/shuichiro-makigaki/exmachina</a>).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200424/20200424073204_2607.png" width="141"/><br/><strong>Figure 1. Overview of the proposed method<br/><br/></strong> </p><ol class="npwone"><li>Model training<ol class="npwtwo"><li>Download SCOP40 database.</li><li>Generate structural alignments of every domain pair in the same SF by TM-align.</li><li>Select only pairs that the TM-score is â¥ 0.5.</li><li>Generate a PSSM of the domain by three-iteration PSI-BLAST with the UniRef90 database.</li><li>Generate training data and labels.<br/>As a hyper-parameter, window size is 5.</li><li>Reduce training dataset to 1/10 by random sampling.<br/>Because the original training dataset became too large to process within a reasonable computation time.</li><li>Save the training dataset and the labels as FLANN-acceptable data format.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Score prediction and sequence alignment generation<ol class="npwtwo"><li>Prepare two homologous amino acid sequences<br/>As a current limitation, our implementation expects that the inputs are sub-domains. When the protein consists of multiple domains, it should be split into domains. Usually, the domain regions can be predicted by homology detection.</li><li>Generate PSSMs of each sequence by three-iteration PSI-BLAST with the UniRef90 database.</li><li>Predict all substitution scores of each residue pairs.<ol class="npwthree"><li>Query vector format is the same as the training phase, and the <em>k</em>-NN's classification scores are used for the substitution score directly.</li><li>As hyper-parameters, the window size is 5, and the number of the neighbor is 1,000.</li></ol></li><li>Save predicted substitution score matrix.</li><li>Generate local sequence alignment by Smith-Waterman algorithm implemented in Biopython (<a href="https://biopython.org/" target="_blank">https://biopython.org/</a>).<br/>During the dynamic-programming, the predicted substitution scores are used for score calculation.</li></ol></li></ol><p class="pptt" id="biaoti32985">### Acknowledgments</p><p class="ppzz">This work was supported by JSPS KAKENHI [18K11524] and (Makigaki and Ishida, 2019).</p><p class="pptt" id="biaoti32986">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti32987">### References</p><ol class="npwtwok"><li>Altschul, S. F., Madden, T. L., SchÃ¤ffer, A. A., Zhang, J., Zhang, Z., Miller, W. and Lipman, D. J. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" target="_blank">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs.</a> <em>Nucleic Acids Res</em> 25(17): 3389-3402.</li><li>Boratyn, G. M., Schaffer, A. A., Agarwala, R., Altschul, S. F., Lipman, D. J. and Madden, T. L. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22510480" target="_blank">Domain enhanced lookup time accelerated BLAST.</a> <em>Biol Direct</em> 7: 12.</li><li>Cao, R., Bhattacharya, D., Hou, J. and Cheng, J. (2016). <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1405-y" target="_blank">DeepQA: improving the estimation of single protein model quality with deep belief networks.</a> <em>BMC Bioinformatics</em> 17(1): 495.</li><li>Kopp, J., Bordoli, L., Battey, J. N., Kiefer, F. and Schwede, T. (2007). Assessment of CASP7 predictions for template-based modeling targets. <em>Proteins</em> 69 Suppl 8: 38-56.</li><li>Lyons, J., Dehzangi, A., Heffernan, R., Sharma, A., Paliwal, K., Sattar, A., Zhou, Y. and Yang, Y. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25212657" target="_blank">Predicting backbone Calpha angles and dihedrals from protein sequences by stacked sparse auto-encoder deep neural network.</a> <em>J Comput Chem</em> 35(28): 2040-2046.</li><li>Makigaki, S. and Ishida, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31197318" target="_blank">Sequence alignment using machine learning for accurate template-based protein structure prediction.</a> Bioinformatics.</li><li>Manavalan, B. and Lee, J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28419290" target="_blank">SVMQA: support-vector-machine-based protein single-model quality assessment.</a> <em>Bioinformatics</em> 33(16): 2496-2503.</li><li>Muhammed, M. T. and Aki-Yalcin, E. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30187647" target="_blank">Homology modeling in drug discovery: Overview, current applications, and future perspectives.</a> <em>Chem Biol Drug Des</em> 93(1): 12-20.</li><li>Muja, M. and Lowe, D. (2009). <a href="http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.160.1721" target="_blank">Fast approximate nearest neighbors with automatic algorithm configuration.</a> In Muja M. and Lowe D. (Eds). <em>VISAPP International Conference on Computer Vision Theory and Applications.</em> Lisboa, Portugal, February 5-8, 2009 - Volume 1.</li><li>The UniProt Consortium (2016). <a href="https://academic.oup.com/nar/article/45/D1/D158/2605721" target="_blank">UniProt: the universal protein knowledgebase. </a><em>Nucleic Acids Research</em> 45(D1): D158-D169.</li><li>Wang, S., Peng, J., Ma, J. and Xu, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26752681" target="_blank">Protein secondary structure prediction using deep convolutional neural fields.</a> <em>Sci Rep</em> 6: 18962.</li><li>Wang, S., Sun, S., Li, Z., Zhang, R. and Xu, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28056090" target="_blank">Accurate de novo prediction of protein contact map by ultra-deep learning model.</a> <em>PLoS Comput Biol</em> 13(1): e1005324.</li><li>Wei, L. and Zou, Q. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27999256" target="_blank">Recent progress in machine learning-based methods for protein fold recognition. </a><em>Int J Mol Sci</em> 17(12).</li><li>wwPDB consortium. (2018). <a href="https://academic.oup.com/nar/article/47/D1/D520/5144142" target="_blank">Protein Data Bank: the single global archive for 3D macromolecular structure data.</a> <em>Nucleic Acids Res</em> 47(D1): D520-D528.</li><li>Zhang, Y. and Skolnick, J. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15849316" target="_blank">TM-align: a protein structure alignment algorithm based on the TM-score.</a> <em>Nucleic Acids Res</em> 33(7): 2302-2309.</li><li>Zimmermann, L., Stephens, A., Nam, S. Z., Rau, D., Kubler, J., Lozajic, M., Gabler, F., Soding, J., Lupas, A. N. and Alva, V. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29258817" target="_blank">A completely reimplemented MPI bioinformatics toolkit with a new HHpred server at its core.</a> <em>J Mol Biol</em> 430(15): 2237-2243.<br/></li></ol>