<body>
            <h1>Confocal and Super-resolution Imaging of RNA in Live Bacteria Using a Fluorogenic Silicon Rhodamine-binding Aptamer</h1>
            <p><strong>Authors:</strong> Regina Wirth, Peng Gao, G. Ulrich Nienhaus, Murat Sunbul, Andres JÃ¤schke</p>
            <h2>Abstract</h2>
            <p>Genetically encoded light-up RNA aptamers have been shown to be promising tools for the visualization of RNAs in living cells, helping us to advance our understanding of the broad and complex life of RNA. Although a handful of light-up aptamers spanning the visible wavelength region have been developed, none of them have yet been reported to be compatible with advanced super-resolution techniques, mainly due to poor photophysical properties of their small-molecule fluorogens. Here, we describe a detailed protocol for fluorescence microscopy of mRNA in live bacteria using the recently reported fluorogenic silicon rhodamine binding aptamer (SiRA) featuring excellent photophysical properties. Notably, with SiRA, we demonstrated the first aptamer-based RNA visualization using super-resolution (STED) microscopy. This imaging method can be especially valuable for visualization of RNA in prokaryotes since the size of a bacterium is only a few times greater than the optical resolution of a conventional microscope.<br/></p>
            <p><strong>Keywords:</strong> Light-up aptamer, Silicon rhodamine, RNA imaging, STED microscopy, Confocal fluorescence microscopy, Live cell imaging, SiRA aptamer</p>
            <h2>Background</h2><p>The visualization of specific RNA molecules by fluorescence microscopy has become invaluable during the past two decades to extend our knowledge of RNA function within cells in a spatiotemporal manner (Tyagi, 2009; Xia <em>et al.</em>, 2017). Due to the lack of inherently fluorescent RNAs, development of fluorogenic RNA labeling tools for live-cell imaging and their adaptation to state-of-the-art microscopesâespecially to super-resolution microscopesâis imperative. Super-resolution microscopy (SRM) is particularly attractive for imaging RNAs in prokaryotic systems since a bacterium is very small (~2.5 Î¼m long, ~0.5-1 Î¼m wide) and the resolution of a standard fluorescence microscope is restricted to ~200-300 nm due to the diffraction limit of light (Reshes <em>et al.</em>, 2008; Nienhaus and Nienhaus, 2016). Although a plethora of tools have been developed to fluorescently label an RNA of interest (ROI), their live-cell visualization using SRM (resolution ~20-50 nm) is still challenging (Alexander and Devaraj, 2017), mainly due to the stringent requirements on the photophysical properties of the fluorophores needed for live-cell SRM. These properties include high photostability and brightness, cellular membrane permeability, water solubility, absorption and emission wavelengths preferentially in the far-red and near-infrared (NIR) region, on-off switching capability and a sufficient signal-to-background ratio during imaging (Wang <em>et al.</em>, 2019). Thus, there is a strong need for novel RNA reporter systems that are compatible with SRM.<br/><br/>An especially attractive approach to visualize RNAs in live cells is the use of genetically encoded fluorescence light-up aptamers, which are short oligonucleotides that bind small, conditionally fluorescent probes with high affinity and are developed using systematic evolution of ligands by exponential enrichment (SELEX) (Holeman <em>et al.</em>, 1998; Ouellet, 2016). Tagged to the ROI, the aptamers can be visualized due to a fluorescence increase upon small molecule binding (Figure 1). Several fluorescence enhancing RNA aptamers spanning the visible region of the spectrum have recently been developed and proven to be powerful tools for RNA labeling (Bouhedda <em>et al.</em>, 2017; Neubacher and Hennig, 2019).<br/><br/>The famous Spinach, Broccoli, Mango, Corn aptamers and their improved variants bind to fluorophores with low fluorescence quantum yield in solution and become fluorescent due to the restriction of intramolecular movements upon aptamer binding (Figure 1A) (Paige <em>et al.</em>, 2011; Filonov <em>et al.</em>, 2014; Dolgosheina <em>et al.</em>, 2014; Song <em>et al.</em>, 2017). Other prominent examples are SRB-2, DNB, BHQ and Riboglow aptamers that disrupt a quencher-dye conjugate upon binding to either the dye or the quencher moiety, thereby generating fluorescence enhancement (Figure 1B) (Murata <em>et al.</em>, 2011; Sunbul and JÃ¤schke, 2013 and 2018; Arora <em>et al.</em>, 2015; Braselmann <em>et al.</em>, 2018). Most of these aptamer-ligand pairs, however, do not absorb and emit in the far-red or NIR region. Although MG, Riboglow, Mango TO-3 and the recently reported DIR-pro (which has not been reported for RNA labeling) are excited in the far-red, their fluorophores show either low photostability and brightness, phototoxicity or are hardly cell permeable (Tan <em>et al.</em>, 2017; Autour <em>et al.</em>, 2018; Yerramilli and Kim, 2018).<br/><br/>Prompted by these shortcomings, we recently developed a novel light-up RNA aptamer that binds to 3-carboxy silicon rhodamine (SiR) (Wirth <em>et al.</em>, 2019). SiRs are an exciting new fluorophore class in the field of protein labeling for SRM because they are photostable, NIR-emitting fluorophores that change their equilibrium between the non-colored spirolactone and the fluorescent zwitterion in response to their environment (Figure 1C) (LukinaviÄius <em>et al.</em>, 2013). This property is responsible for their high cell permeability and fluorogenic behavior and contributes to the popularity of SiR dyes and their derivatives in the last decade (Wang <em>et al.</em>, 2019). Applying SELEX on a combinatorial RNA library with a diversity of ~2 x 10<sup>15</sup>, we discovered a 50 nt-minimal aptamer, SiRA, which binds the target SiR <strong>1</strong> with a K<sub>D</sub> of 430 nM and shows a significant fluorescence turn-on upon dye binding (Figure 2). This finding introduces a new concept of fluorogenicity to the field of aptamer-based RNA imaging, namely environment-dependent intramolecular spirocyclization (Figure 1C). SiRA is remarkably resistant to photobleaching and constitutes the brightest far-red light up aptamer system known to date (quantum yield 0.98; extinction coefficient 86,000 M<sup>-1</sup>Â·cm<sup>-1</sup>). Using the SiRA system, we visualized the expression of RNAs in bacteria in no-wash live-cell imaging experiments (Wirth <em>et al.</em>, 2019). The innovation in this protocol is that we can use SiRAâs excellent photophysical properties for stimulated emission depletion (STED) SRM of mRNA in live bacteria. To date, SiRA is the only light-up aptamer system proven to work in SRM applications.<br/></p>
        </body>

<br/><p class="ppzz"><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115454_7699.jpg" width="248"/><br/><strong>Figure 1. Fluorogenicity concepts of light-up RNA aptamers used for RNA imaging. </strong>A. A fluorophore with low fluorescence quantum yield due to molecular motions in solution (OFF) becomes highly fluorescent upon aptamer binding due to the restriction of intramolecular movements (ON). B. The contact-quenched fluorophore-quencher conjugates (OFF) light up upon binding to either the fluorophore or the quencher moiety (ON). Here, an example of a light-up system using a fluorophore binding aptamer was shown. C. In solution, SiRs can favor the non-colored spirolactone (OFF). The change in environment upon aptamer binding causes a shift of the equilibrium to the fluorescent zwitterion (ON).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115522_8632.jpg" width="333"/><br/><strong>Figure 2. The SiRA aptamer and its properties.</strong> A. The predicted secondary structure of the 50-nt SiRA aptamer evolved during the selection process (important features are marked in color). B. Structure of the fluorogenic dye <strong>1</strong> used in this protocol, which binds to SiRA with nanomolar affinity. C. Normalized absorption and emission spectra of free SiR <strong>1</strong> and the [SiRA*<strong>1</strong>] complex.<br/><br/>In the following protocol, we describe the preparation of <em>E. coli</em> bacteria for live-cell confocal and STED imaging of GFP-mRNA in detail. We herein use a plasmid-based expression system and tag the GFP-mRNA with tandem repeats of the novel SiRA aptamer (SiRA<sub>5</sub>) at the 3â² untranslated region (UTR). This protocol can be adapted to visualize any plasmid expressed target RNA; however, the expression levels and the half-life of RNAs need to be kept in mind. Further optimization of the protocol might be required for specific applications.</p><p class="pptt" id="biaoti33439">## Materials and Reagents</p><ol class="npwtwok"><li>Pipette tips (Sarstedt, catalog numbers: <a class="blackNoUnder" target="_blank"> 70.113 </a> 0, <a class="blackNoUnder" target="_blank"> 70.760.002 </a> , <a class="blackNoUnder" target="_blank"> 70.762 </a> )</li><li>Microcentrifuge tubes 1.5 ml (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.001 </a> )</li><li>15 Î¼-Slide 8-well uncoated, glass bottom #1.5 polymer coverslip (Ibidi GmbH, catalog number: <a class="blackNoUnder" target="_blank"> 80827 </a> )</li><li>High clarity PP conical centrifuge tube 15 ml (Falcon<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 352096 </a> )</li><li>High clarity PP conical centrifuge tube 50 ml (Falcon<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 352070 </a> )</li><li>Medical Millex-GS syringe filter unit, 0.22 Î¼m (EDM Merck, catalog number: <a class="blackNoUnder" target="_blank"> SLGSM33SS </a> )</li><li>BD Discardit disposable syringes 10 ml (Becton Dickinson, catalog number: <a class="blackNoUnder" target="_blank"> 300296 </a> )</li><li>PYREX<sup>Â®</sup> Media bottles, graduated, Corning<sup>Â®</sup>, 1,000 ml (VWR, PYREX<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 1395-10L </a> )</li><li>FluoSpheres<sup>TM</sup> Carboxylate-Modified Microspheres, 0.04 Î¼m, dark red fluorescent (660/680), 5% solids, azide free (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> F8789 </a> )</li><li>Gold beads (GC80, BB International, catalog number: <a class="blackNoUnder" target="_blank"> EM. GC80 </a> )</li><li><em>E. coli</em> BL21 Star<sup>TM</sup> (DE3) Invitrogen competent <em>E. coli</em> strains (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> C601003 </a> ), store at -80 Â°C</li><li>pET-GFP expression plasmid (pET His6 GFP TEV LIC cloning vector, 1 GFP) (Addgene, Plasmid number: 29663), store at -20 Â°C</li><li>pET-GFP-SiRA<sub>5</sub> expression plasmid, store at -20 Â°C<br/>The pET-GFP-SiRA<sub>5</sub> plasmid used in this protocol is not commercially available. Generally tandem repeats of an aptamer of choice (depending on the target RNA and expression level), can directly be fused to the ROI (in this case GFP mRNA) using proper restriction sites in the 3â² UTR (here: SalI and XhoI fast digest restriction enzymes, Thermo Scientific<sup>TM</sup>, catalog numbers: <a class="blackNoUnder" target="_blank"> FD0644 </a> and <a class="blackNoUnder" target="_blank"> FD0695 </a> , respectively) as previously described in the literature (Sunbul <em>et al.</em>, 2018). In this study, we inserted five repeats of the SiRA aptamer with interspersed linker nucleotides according to Figure 3.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115546_9751.jpg" width="1011"/><br/><strong>Figure 3. Excerpt of the pET-GFP-SiRA<sub>5</sub> bacterial expression plasmid used for the visualization of GFP mRNA in this protocol.</strong> lacO: lac operator, rbs: ribosomal binding site, ROI: RNA of interest, P: promoter, T: terminator, SalI and XhoI: Restriction sites, stop: Stop codon for RNA translation.<br/><br/>The following sequences show an excerpt of the cloned expression vector (pET-GFP-SiRA<sub>5</sub>) as well as the pET-GFP used in this study.<br/><br/><div style="word-break:break-all;"><strong>pET-GFP-SiRA<sub>5</sub>:</strong><br/>...CGAGGAAACCTGTACTTCCAATCCAATATTGGAAGTGGATAACGGATCCGAATTCGAGCGCC<strong><strong><span style="color:#337FE5;">GTCGAC</span></strong></strong><strong><span style="color:#CCCCCC;">AGGC</span></strong><strong>CCACCGGGTTTGAAAACCTGGCTGCTTCGGCAGTTGTATCCTTTGG</strong><strong><span style="color:#CCCCCC;">GCCTAAGAGAGA</span></strong><strong>CCACCGGGTTTGAAAACCTGGCTGCTTCGGCAGTTGTATCCTTTGG</strong><span style="color:#CCCCCC;"><strong>TCTCAATAACCAG</strong></span><strong>CCACCGGGTTTGAAAACCTGGCTGCTTCGGCAGTTGTATCCTTTGG</strong><span style="color:#CCCCCC;"><strong>GCCTAAGAGAGA</strong></span><strong>CCACCGGGTTTGAAAACCTGGCTGCTTCGGCAGTTGTATCCTTTGG</strong><span style="color:#CCCCCC;"><strong>TCTCAATAACCAG</strong></span><strong>CCACCGGGTTTGAAAACCTGGCTGCTTCGGCAGTTGTATCCTTTGG</strong><span style="color:#CCCCCC;"><strong>CTGG</strong></span><strong><span style="color:#EE33EE;">CTCGAG</span></strong>CACCACCACCACCACCACTGAGATCCGGCTGCTAACAAAGCCCGAAAGGAAGCTGAGTTGGCTGCTGCCACCGCTGAGCAATAACTAGCATAACCCCTTGGGGCCTCTAAACGGGTCTTGAGGGGTTTTTTGCTGAAAGGAGGAACTATATCCGGATTGGCGAATGGGACGCGCCCTGTAGCGGCGCATTAAGCGCGGCGGGTGâ¦<br/><strong>pET-GFP:</strong><br/>...CGAGGAAACCTGTACTTCCAATCCAATATTGGAAGTGGATAACGGATCCGAATTCGAGCGCC<strong><span style="color:#337FE5;">GTCGAC</span></strong>AAGCTTGCGGCCGCA<strong><span style="color:#EE33EE;">CTCGAG</span></strong>CACCACCACCACCACCACTGAGATCCGGCTGCTAACAAAGCCCGAAAGGAAGCTGAGTTGGCTGCTGCCACCGCTGAGCAATAACTAGCATAACCCCTTGGGGCCTCTAAACGGGTCTTGAGGGGTTTTTTGCTGAAAGGAGGAACTATATCCGGATTGGCGAATGGGACGCGCCCTGTAGCGGCGCATTAAGCGCGGCGGGTGâ¦<br/></div><strong>black</strong>     SiRA (3-nt stem loop)<br/><strong><span style="color:#CCCCCC;">grey</span></strong>     Variable Stem Loop or Spacer Region<br/><strong><span style="color:#337FE5;">blue</span></strong>     SalI Restriction Site<br/><strong><span style="color:#EE33EE;">pink</span></strong>     XhoI Restriction Site<br/><br/></li><li>Autoclaved LB medium, Lennox (Carl Roth<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> X964.4 </a> )</li><li>LB Broth with agar, Lennox (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L2897 </a> )</li><li>Kanamycin sulfate from <em>Streptomyces kanamyceticus</em> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> K1377 </a> ), prepare 1 M aqueous solution, filter-sterilize and store at -20 Â°C</li><li>Isopropyl-Î²-<em>D</em>-thio-galactopyranoside (IPTG) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> I5502 </a> ), prepare 1 M aqueous solution, filter-sterilize and store at -20 Â°C</li><li>Potassium hydroxide pellets EMPLURA<sup>Â®</sup> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1050121000 </a> ), prepare 10 M aqueous solution</li><li>Poly-<em>D</em>-lysine hydrobromide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P7886 </a> ), prepare 2.5 mg/ml in DPBS, store at 4 Â°C</li><li>Dulbecco's Phosphate Buffered Saline (DPBS) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D8537 </a> )</li><li>MilliQ water</li><li>Silicon rhodamine based fluorogenic dye, 100 Î¼M solution in DMSO<br/>The fluorogenic dye used in this protocol is not commercially available. Therefore, it has to be synthesized and purified by HPLC as described in the literature (Wirth <em>et al.</em>, 2019).<br/><em>Note: The HPLC purification has to be carried out with great care, since the fluorescence of remaining impurities can decrease the signal-to-background ratio during the imaging experiment.</em> </li><li>M9 Broth (Fluka Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 63011 </a> )</li><li>Magnesium sulfate monohydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 434183 </a> ), prepare 1 M aqueous solution and autoclave</li><li>D-(+)-Glucose (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> G8270 </a> ), prepare 40% (w/v) aqueous solution and filter-sterilize</li><li>Magnesium chloride hexahydrate (Sigma-Aldrich, catalog number <a class="blackNoUnder" target="_blank"> M2670 </a> ), prepare 1 M aqueous solution and autoclave</li><li>Calcium chloride dihydrate (Sigma-Aldrich: catalog number: <a class="blackNoUnder" target="_blank"> 1725801000 </a> ), prepare 1 M aqueous solution and autoclave</li><li>Live-cell imaging solution (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A14291DJ </a> )</li><li>M9 imaging solution (see Recipes)<br/></li></ol><p class="pptt" id="biaoti33440">### Equipment</p><ol class="npwtwok"><li>Research<sup>Â®</sup> plus pipettes, variable volume 200 and 1,000 Î¼l (Sigma-Aldrich, Eppendorf<sup>Â®</sup>, catalog numbers: <a class="blackNoUnder" target="_blank"> Z683817 </a> and <a class="blackNoUnder" target="_blank"> Z683825 </a> )</li><li>Steam Sterilizer (<em>e.g.</em>, Varioklav<sup>Â®</sup>, model: <a class="blackNoUnder" target="_blank"> 155S </a> )</li><li>Thermomixer comfort 24 x 1.5 ml (Eppendorf<sup>Â®</sup>)</li><li>Incubator and shaker (Thermo Fisher Scientific, type 496)</li><li>BioPhotometer (Eppendorf<sup>Â®</sup>, type 6131)</li><li>High speed Micro Centrifuge (Neuation Technology Pvt. Ltd., catalog number: <a class="blackNoUnder" target="_blank"> iFuge M12 </a> )</li><li>Freezer</li><li>Refrigerator</li><li>Confocal microscope (Nikon, model: <a class="blackNoUnder" target="_blank"> A1R </a> )<br/>For this protocol, a point scanning confocal microscope with hybrid scanner (galvano/resonant) was used, equipped with a Nikon N Apo 60x NA 1.4 Î»s OI (WD 0.14 mm, FOV 0.21 x 0.21 mm) objective. For fluorescence excitation, 488 nm (green channel) and 640 nm (red channel) lasers were used. For detection, a high-sensitive GaAsP detector combined with 525/50 nm and 700/75 nm (center wavelength/bandwidth) emission filters for green and red channels, respectively, was used.</li><li>STED microscope<br/>The STED microscope used in this protocol is home-built and has been described in detail (Gao <em>et al.</em>, 2017). In addition, a 650-nm long-pass dichroic mirror (ET700SP; Chroma, Bellows Falls, VT) and a time-correlated single photon counting (TCSPC) card (SPC-150, Becker &amp; Hickl GmbH, Berlin, Germany) are needed.<br/><em>Note: The experiment in the protocol was carried out with 640 nm excitation and 779 nm depletion; only the vortex phase plate for plain 2D STED imaging was used.</em><br/></li></ol><p class="pptt" id="biaoti33441">### Software</p><ol class="npwtwok"><li>ImageJ Fiji, version 2.0.0. with Java 1.6.0_24</li><li>Software confocal microscope: NIS-Elements image acquisition and analysis software (Nikon)</li><li>Software STED microscope: Imspector (Max-Planck-Innovation GmhH, Muenchen, Germany)</li><li>Matlab (MathWorks, Natick, MA)</li><li>OriginPro 2015 32 bit (OriginLab, Northampton, MA)<br/></li></ol><p class="pptt" id="biaoti33442">### Procedure</p><ol class="npwone"><li><em>E. coli</em> transformation (Day 1: ~2.5 h followed by over-night incubation)<br/><em>Note: We recommend a fresh transformation prior to each imaging experiment to ensure proper mRNA-SiRA expression.</em> <ol class="npwtwo"><li>Add 50 ng of the plasmids (pET-GFP and pET-GFP-SiRA<sub>5</sub>) into each autoclaved microcentrifuge tube and put them on ice.</li><li>Thaw chemically competent <em>E. coli</em> BL21 Star<sup>TM</sup> (DE3) cells for 15 min on ice.</li><li>Add 40 Î¼l of <em>E. coli</em> BL21 Star<sup>TM</sup> (DE3) cells to each tube and incubate for an additional 30 min on ice.</li><li>Heat the tubes for 45 s at 42 Â°C followed by incubation on ice for 2 min.</li><li>Add 250 Î¼l of pre-warmed (37 Â°C) LB medium to each tube and shake them at 650 rpm for 60 min.</li><li>For the selection of bacteria carrying the plasmid, streak 20 Î¼l of the bacteria on LB agar plates containing 30 Î¼g/ml of kanamycin.</li><li>Incubate the LB agar plates overnight at 37 Â°C.<br/><em>Note: In the morning, place the agar plates at 4 Â°C and keep them there throughout the day.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Induction of mRNA expression in bacteria (Day 2: 15 min followed by over-night incubation; Day 3: ~ 5 h)<ol class="npwtwo"><li>Add 5 ml of LB medium supplemented with 30 Î¼g/ml kanamycin into a 15 ml Falcon and start an overnight culture using a single colony picked from each LB agar plate using a sterile pipette tip.</li><li>Incubate the Falcon at 37 Â°C with vigorous shaking at 170 rpm overnight.</li><li>The next morning, measure the optical density (OD<sub>600</sub>) for a 1:10 dilution of each overnight culture.</li><li>Generate a starter culture from the overnight culture. For this, add 10 ml of LB medium supplemented with 30 Î¼g/ml kanamycin in a 50 ml Falcon and add the overnight culture to give an OD<sub>600</sub> of 0.05.</li><li>Shake the culture at 37 Â°C and 170 rpm till the OD<sub>600</sub> reaches 0.35.<br/><em>Note: This usually takes 1.5-2 h.</em> </li><li>Induce the expression of GFP mRNA by adding 10 Î¼l of IPTG (1 mM final concentration) and shake the culture for an additional 3 h at 37 Â°C.<br/><em>Note: Currently the SiRA system has only been used for imaging highly overexpressed GFP mRNAs. In order to visualize lower copy numbers, a larger number of tandem repeats can be designed and further optimization of the protocol (</em><em>e.g.</em><em>, dye concentration) might be necessary.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of poly-<em>D</em>-lysine coated glass chamber slides (Day 3: ~3 h, in parallel to Steps B5-B6)<br/><em>Note: The poly-</em><em>D</em><em>-lysine solution for coating needs to be prepared freshly every time just before adding it to the well. The poly-</em><em>D</em><em>-lysine coated glass chamber slides, however, can be prepared up to 24 h in advance and stored in the refrigerator at 6 Â°C. Warming up to RT before use is recommended.</em> <ol class="npwtwo"><li>Remove the 8-well ibidi glass slides from the packaging and incubate each well with 200 Î¼l of a 10 M aqueous KOH solution for 5 min.</li><li>Remove the KOH solution from the wells very carefully, wash them thoroughly multiple times with 400 Î¼l sterile MilliQ water and air-dry them for 60 min.</li><li>Mix 1,000 Î¼l of water with 20 Î¼l of the poly-<em>D</em>-lysine stock (~50 Î¼g/Î¼l final concentration) and add 250 Î¼l of the solution quickly into each well (recipe for 4 wells).</li><li>Incubate the wells for 45 min at room temperature.</li><li>Wash each well twice with 300 Î¼l sterile MilliQ water and air-dry for at least 30 min prior to the immobilization of the bacteria.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of <em>E. coli</em> samples for imaging (Day 3: ~1 h)<br/><em>Note: For live-cell imaging experiments, we recommend to image up to 2 wells at a time, due to the short mRNA half-life and bacterial metabolism.</em> <ol class="npwtwo"><li>Transfer a 200 Î¼l aliquot of each culture (Step B6) to an autoclaved microcentrifuge tube and pellet the cells at 5,600 <em>x g</em> for 2 min at room temperature.</li><li>Remove the LB medium and suspend the bacteria in 1 ml of M9 imaging solution (Recipe 1).</li><li>Add 200 Î¼l of the cell suspension into each well of the coated 8-well ibidi slide.</li><li>Incubate the bacteria for 35 min at room temperature to allow for adhesion to the surface.</li><li>Gently wash each well with 400 Î¼l of M9 imaging solution.</li><li>Add 300 Î¼l of M9 imaging solution containing 500 nM of the silicon rhodamine fluorogenic dye and directly prepare for imaging.<br/><em>Note: Permeation of the dye into the bacterium is quite fast (5-10 min) at 37 Â°C; therefore, the time for microscope adjustment is sufficient for incubation. If permeation issues occur, prewarming of the dye/M9 imaging solution to 37 Â°C is recommended.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Imaging GFP-SiRA<sub>5</sub> mRNA using a confocal microscope (Day 3: ~1-2 h)<ol class="npwtwo"><li>Turn on the 488 nm (green channel) and 640 nm (red channel) lasers for fluorescence illumination of the point scanning confocal microscope and use the 525/50 nm and 700/75 nm emission filters for green and red channel, respectively.</li><li>Attach the temperature incubation chamber to the microscope, prewarm to 37 Â°C and place the 8-well slide with adherent <em>E. coli</em> into the chamber.</li><li>Use the bright-field illumination to focus on the immobilized <em>E. coli</em> with the Nikon N Apo 60x NA 1.4 Î»s OI (WD 0.14 mm, FOV 0.21 x 0.21 mm) objective, starting with the pET-GFP-SiRA<sub>5</sub> transformed cells.</li><li>Determine a suitable exposure time for the GFP and red channel as well as the correct z-axis. Choose the settings such that you do not saturate the pixels but still obtain the highest fluorescent signal.</li><li>Acquire the brightfield and fluorescence images of all channels (Figure 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115643_7652.jpg" width="84"/><br/><strong>Figure 4. Example of GFP mRNA imaging in live <em>E. coli</em> using a confocal microscope.</strong> A. Schematic of the plasmid used in the imaging experiment. Binding of the fluorophore to the SiRA aptamers generates a fluorescence signal in the red detection channel of the microscope. B. No-wash live-cell imaging of GFP mRNA in live <em>E. coli</em> cells. The cells were transformed with either the GFP plasmid or the GFP-SiRA<sub>5</sub> plasmid, and the images taken with the bright field, green (protein expression) and red channel (mRNA detection). Scale bars = 5 Âµm. C. Zoom-in of the images depicted in Figure 4B. Scale bars = 5 Âµm.<br/><br/></li><li>Repeat Steps E3-E5 with the <em>E. coli</em> cells carrying the control plasmid pET-GFP using the same microscope settings.</li><li>For background correction, we use Fiji/Image J image analysis software. Manually pick a surface area without attached <em>E. coli</em> cells and subtract its mean fluorescence signal from the whole image.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>(Optional) Adjustment of the STED microscope<ol class="npwtwo"><li>Ensure spatial and temporal overlap of the 640-nm excitation and 779-nm depletion beam foci by scanning a gold bead (GC80) with detection in both color channels.</li><li>At first, separate the reflected/scattered excitation and depletion light beams into two beams by a 650-nm long-pass dichroic mirror. Then, filter the two beams by band-path filters 780/10 nm (center/width) and 640/10 nm until they are detected by the two avalanche photodiodes (APDs).<br/>Use the Imspector software to synchronize the scan signal and photon detection intervals, thus generating two-channel images without spectral crosstalk.<br/><em>Note: If necessary, the depletion beam needs to be slightly adjusted to ensure that the center of the STED image (with a doughnut-like profile) is precisely overlaid with the excitation image (with a Gaussian profile).</em> </li><li>Check the temporal overlap between the excitation and depletion pulses. Therefore, collect the reflected 640-nm and 779-nm light by the gold beads by using a time-correlated single photon counting (TCSPC) card and sort them with respect to the common trigger signal. The delay between the 640 nm excitation and 779 nm pulses can be monitored in the oscilloscope display-module of the TCSPC software.<br/><em>Note: For optimal depletion, the delay between the excitation and depletion pulses can be adjusted (in our example ~200 ps) by using a delay box, which routes the trigger output of the depletion laser to the trigger input of the excitation laser.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Imaging dark-red beads (Day 3: ~1 h)<br/><em>Note: Images of dark-red beads are taken to determine the resolution of the STED microscope and should be carried out before or in parallel to the sample preparation.</em> <ol class="npwtwo"><li>Dilute Fluosphere<sup>TM</sup> 40-nm dark-red beads (660/680) from the stock solution into PBS at a volume ratio of 1:10<sup>4</sup>.</li><li>Add 200 Î¼l of the solution to one of the poly-<em>D</em>-lysine coated imaging chambers and incubate for 5 min at room temperature to immobilize the beads on the surface.</li><li>Wash the coverslip twice with 250 Î¼l MilliQ water and finally add 250 Î¼l MilliQ water for imaging.</li><li>Place the coverslip onto the microscope slide holder.</li><li>Adjust the excitation power to 5 Î¼W at the sample and image with the depletion laser at different powers between 0 mW to 150 mW. Scan a rectangular area of 10 x 10 Î¼m<sup>2</sup> with 1,024 x 1,024 pixels, covering dozens of individual dark-red beads.<br/><em>Note: To minimize photo-bleaching upon multiple scans of the same region, it is recommended to use a fresh region for STED imaging at the different STED laser intensities.</em> </li><li>Generate a comparable image between confocal and STED modes (Figure 5A). Therefore, image the same area sequentially by confocal and STED microscopy.<br/><em>Note: By changing the depletion power, the effective resolution can be varied, as shown by images of a bead taken at different powers of the depletion beam (Figure 5B).</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115706_5091.jpg" width="271"/><br/><strong>Figure 5. Dark-red bead images.</strong> A. Confocal (upper left) and STED (lower right) image of fluorescent beads on a glass coverslip, showing the enormous resolution enhancement due to STED imaging. Scale bar = 3 Âµm. B. Image of an individual bead taken at different powers of the depletion beam.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Evaluation of the STED resolution using dark-red bead images (Day 3: ~1 h)<ol class="npwtwo"><li>For data evaluation, select ten beads for each STED intensity. For each bead, extract a line (with single-pixel width) crossing the bead through its center and fit with a Gaussian function.<br/><em>Note: The fit yields a raw full width at half maximum (FWHM<sub>r</sub>), which includes the size of the dark-red beads (d<sub>0</sub> = 40 nm). The real STED resolution (FWHM) can then be estimated as (FWHM</em><sub><em>r</em></sub><sup><em>2</em></sup><em> - d</em><sub><em>0</em></sub><sup><em>2</em></sup><em>)</em><sup><em>1/2</em></sup><em>. Based on the statistics of ten beads, the symbols and error bars in Figure 6A show, respectively, the means and standard deviations of the FWHMs at each STED intensity. The theoretical curve, which is fitted with the model FWHM = FWHM<sub>0</sub>/(1 + Î³Â·I</em><sub><em>STED</em></sub><sup><em>2</em></sup><em>)</em><sup><em>1/2</em></sup><em>, describes the image resolution as a function of STED power. Here, FWHM<sub>0</sub> is the resolution of the confocal microscope (with depletion beam off), I<sub>STED</sub> is the intensity of the depletion beam, and Î³ is a coefficient that depends on the slope of the STED doughnut and the saturation intensity of the depletion beam (Harke </em><em>et al.</em><em>, 2008).</em> </li><li>Compare cross-sectional intensity distributions obtained by confocal (zero depletion power) and STED imaging (150 mW depletion power) of a 40 nm bead (Figure 6B). [Example Figure 6A: FWHMs of Gaussian fits (solid lines in Figure 6A) yield <em>FWHM</em><sub>r</sub> = 339 Â± 11 nm for confocal mode and 50 Â± 3 nm for STED mode. Removing the size of the dark-red beads, we obtain an estimated resolution of our STED microscope as 32 Â± 5 nm.]<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115730_0550.jpg" width="291"/><br/><strong>Figure 6. Determination of the STED resolution.</strong> A. Solid squares: Quantification of the widths of cross-sections through individual beads as a function of the power of the depletion laser (see Figure 5B). Data were corrected for the intrinsic size of the beads and are shown as means Â± standard deviations from ten bead images. Red line: fit with a square-root function according to STED theory. B. Cross sections through a single bead imaged at 0 and 150 mW depletion laser power (see Figure 5) showing the enormous resolution enhancement offered by STED microscopy. Gaussian fits are shown as solid lines.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Imaging GFP-SiRA<sub>5</sub> mRNA in live <em>E. coli</em> with STED microscopy (Day 3: ~1-2 h)<ol class="npwtwo"><li>Attach the temperature-controlled sample chamber to the microscope, prewarm to 37 Â°C and add the 8-well imaging slide used in Step E2.</li><li>Set the power of the excitation and depletion beams (Example Figure 7: Excitation 5 Î¼W, depletion 150 mW). Scan repeatedly a square region of 10 x 10 Î¼m<sup>2</sup>, covering a group of <em>E. coli</em> cells, for five frames of 1,024 x 1,024 pixels (Pixel dwell time in Figure 7A: 30 Î¼s, the main image of Figure 7A corresponds to the inset with t = 126 s).</li><li>After taking the STED images, switch off the depletion laser and take a confocal image (Figure 7B).<br/><em>Note: No deconvolution was carried out on either the STED or the confocal images. Different color maps can be used for the STED and confocal images to optimally display cell structures.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425115755_4271.jpg" width="210"/><br/><strong>Figure 7. STED imaging of live <em>E. coli</em> bacteria expressing GFP-SiRA <sub>5</sub> mRNA.</strong> A. Time series of STED images taken in 31.5 s intervals for a total of 126 s; top: magnified image at 126 s; B. Regular confocal image taken directly after the STED time series.</li></ol></li></ol><p class="pptt" id="biaoti33443">### Data analysis</p><ol class="npwtwok"><li>To determinate the STED resolution, statistics with multiple beads are necessary. In our experiment, ten individual beads for each STED intensity were selected to calculate the raw full width at half maximum (FWHMr) with the cross-section intensity distribution of the bead. The data points and error bars in Figure 6A present the means and standard deviations of the ten beads. <br/><em>Note: To avoid selection of a cluster of beads (rather than single DR beads), those were excluded (and replaced with new selections) when their cross-section intensity distribution has the fitted Gaussian amplitude 1.5 times higher than the averaged amplitude.</em> </li><li>To reduce the influence of sample defocusing on STED resolution determination, for each STED intensity two images were taken (from a new sample area), and the in-focus position of the sample was checked each time, which provides the maximal brightness for the beads in the confocal mode. Generally, five beads from each image were selected for the statistics of the STED resolution. However, for the case that one image has systematically-larger FWHMs than the other image (due to defocusing), ten beads from the in-focus image solely will be selected for the FWHM analysis.</li></ol><p class="pptt" id="biaoti33444">### Recipes</p><ol class="npwtwok"><li>M9 imaging solution (50 ml)<ol class="npwthree"><li>Preparation of the 5x M9 salt broth:<br/><ol class="npwfour"><li>Add 26.25 g of M9 broth powder [Na<sub>2</sub>HPO<sub>4</sub> (15 g), KH<sub>2</sub>PO<sub>4</sub> (7.5 g), NaCl (1.25 g), NH<sub>4</sub>Cl (2.5 g)] into a 1,000 ml PYREX<sup>Â®</sup> glass bottle</li><li> Add 500 ml of MilliQ water, shake thoroughly and check the pH (7.4 at 37 Â°C)</li><li>Autoclave with a standard liquid program</li></ol><em>Note: This 5x M9 mix can be stored at RT.</em> </li><li>Fill 10 ml of the 5x M9 into a 50 ml Falcon and supplement the solution with 250 Î¼l of MgSO<sub>4</sub> (1 M stock, 5 mM final concentration), 5 Î¼l of CaCl<sub>2</sub> (1 M stock, 100 Î¼M final concentration) and 500 Î¼l of Glucose (40% stock, final concentration 0.4%)</li><li>Fill the Falcon to 50 ml with MilliQ water and filter-sterilize the solution before using it for the imaging experiment</li></ol><em>Note: Alternatively, the imaging experiment can also be carried out in live-cell imaging solution purchased from Thermo Fisher Scientific, supplemented with additional magnesium and glucose as described above.</em><br/></li></ol><p class="pptt" id="biaoti33445">### Acknowledgments</p><p class="ppzz">M.S. and A.J. were supported by the Deutsche Forschungsgemeinschaft (Grant # Ja794/11-1), and G.U.N. by the Helmholtz Association (Program Science and Technology of Nanosystems) and the Karlsruhe School of Optics and Photonics (KSOP). This protocol was adopted from our previously published work (Wirth <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti33446">### Competing interests</p><p class="ppzz">The authors declare no competing financial interest.</p><p class="pptt" id="biaoti33447">### References</p><ol class="npwtwok"><li>Alexander, S. C. and Devaraj, N. K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28671838" target="_blank">Developing a fluorescent toolbox to shed light on the mysteries of RNA.</a> <em>Biochemistry</em> 56(39): 5185-5193.</li><li>Arora, A., Sunbul, M. and JÃ¤schke, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26175046" target="_blank">Dual-colour imaging of RNAs using quencher- and fluorophore-binding aptamers.</a> <em>Nucleic Acids Res</em> 43(21): e144.</li><li>Autour, A., S, C. Y. J., A, D. C., Abdolahzadeh, A., Galli, A., Panchapakesan, S. S. S., Rueda, D., Ryckelynck, M. and Unrau, P. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29440634" target="_blank">Fluorogenic RNA Mango aptamers for imaging small non-coding RNAs in mammalian cells.</a> <em>Nat Commun</em> 9(1): 656.</li><li>Bouhedda, F., Autour, A. and Ryckelynck, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29295531" target="_blank">Light-up RNA aptamers and their cognate fluorogens: from their development to their applications.</a> <em>Int J Mol Sci</em> 19(1).</li><li>Braselmann, E., Wierzba, A. J., Polaski, J. T., ChromiÅski, M., Holmes, Z. E., Hung, S. T., Batan, D., Wheeler, J. R., Parker, R., Jimenez, R., Gryko, D., Batey, R. T. and Palmer, A. E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30061719" target="_blank">A multicolor riboswitch-based platform for imaging of RNA in live mammalian cells.</a> <em>Nat Chem Biol</em> 14(10): 964-971.</li><li>Dolgosheina, E. V., Jeng, S. C., Panchapakesan, S. S., Cojocaru, R., Chen, P. S., Wilson, P. D., Hawkins, N., Wiggins, P. A. and Unrau, P. J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25101481" target="_blank">RNA mango aptamer-fluorophore: a bright, high-affinity complex for RNA labeling and tracking.</a> <em>ACS Chem Biol</em> 9(10): 2412-2420.</li><li>Filonov, G. S., Moon, J. D., Svensen, N. and Jaffrey, S. R. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25337688" target="_blank">Broccoli: rapid selection of an RNA mimic of green fluorescent protein by fluorescence-based selection and directed evolution.</a> <em>J Am Chem Soc</em> 136(46): 16299-16308.</li><li>Gao, P., Prunsche, B., Zhou, L., Nienhaus, K. and Nienhaus, G. U. (2017). <a href="https://www.nature.com/articles/nphoton.2016.279" target="_blank">Background suppression in fluorescence nanoscopy with stimulated emission double depletion.</a> <em>Nature Photonics</em> 11: 163-169.</li><li>Harke, B., Keller, J., Ullal, C. K., Westphal, V., Schonle, A. and Hell, S. W. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18542512" target="_blank">Resolution scaling in STED microscopy.</a> <em>Opt Express</em> 16(6): 4154-4162.</li><li>Holeman, L. A., Robinson, S. L., Szostak, J. W. and Wilson, C. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9889155" target="_blank">Isolation and characterization of fluorophore-binding RNA aptamers.</a> <em>Fold Des</em> 3(6): 423-431.</li><li>LukinaviÄius, G., Umezawa, K., Olivier, N., Honigmann, A., Yang, G., Plass, T., Mueller, V., Reymond, L., CorrÃªa, I. R., Jr., Luo, Z. G., Schultz, C., Lemke, E. A., Heppenstall, P., Eggeling, C., Manley, S. and Johnsson, K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23344448" target="_blank">A near-infrared fluorophore for live-cell super-resolution microscopy of cellular proteins.</a> <em>Nat Chem</em> 5(2): 132-139.</li><li>Murata, A., Sato, S., Kawazoe, Y. and Uesugi, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21412566" target="_blank">Small-molecule fluorescent probes for specific RNA targets.</a> <em>Chem Commun</em> (Camb) 47(16): 4712-4714.</li><li>Neubacher, S. and Hennig, S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30102012" target="_blank">RNA structure and cellular applications of fluorescent light-up aptamers.</a> <em>Angew Chem Int Ed Engl</em> 58(5): 1266-1279.</li><li>Nienhaus, K. and Nienhaus, G. U. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26743847" target="_blank">Where do we stand with super-resolution optical microscopy?</a> <em>J Mol Biol</em> 428(2 Pt A): 308-322.</li><li>Ouellet, J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27446908" target="_blank">RNA fluorescence with light-up aptamers.</a> <em>Front Chem</em> 4: 29.</li><li>Paige, J. S., Wu, K. Y. and Jaffrey, S. R. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21798953" target="_blank">RNA mimics of green fluorescent protein.</a> <em>Science</em> 333(6042): 642-646.</li><li>Reshes, G., Vanounou, S., Fishov, I. and Feingold, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17766333" target="_blank">Cell shape dynamics in <em>Escherichia coli</em>.</a> <em>Biophys J</em> 94(1): 251-264.</li><li>Song, W., Filonov, G. S., Kim, H., Hirsch, M., Li, X., Moon, J. D. and Jaffrey, S. R. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28945233" target="_blank">Imaging RNA polymerase III transcription using a photostable RNA-fluorophore complex.</a> <em>Nat Chem Biol</em> 13(11): 1187-1194.</li><li>Sunbul, M. and JÃ¤schke, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24133044" target="_blank">Contact-mediated quenching for RNA imaging in bacteria with a fluorophore-binding aptamer.</a> <em>Angew Chem Int Ed Engl</em> 52(50): 13401-13404.</li><li>Sunbul, M. and JÃ¤schke, A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29931157" target="_blank">SRB-2: a promiscuous rainbow aptamer for live-cell RNA imaging.</a> <em>Nucleic Acids Res</em> 46(18): e110.</li><li>Sunbul, M., Arora, A. and JÃ¤schke, A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29130205" target="_blank">Visualizing RNA in live bacterial cells using fluorophore- and quencher-binding aptamers.</a> <em>Methods Mol Biol</em> 1649: 289-304.</li><li>Tan, X., Constantin, T. P., Sloane, K. L., Waggoner, A. S., Bruchez, M. P. and Armitage, B. A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28644615" target="_blank">Fluoromodules consisting of a promiscuous RNA aptamer and red or blue fluorogenic cyanine dyes: selection, characterization, and bioimaging.</a> <em>J Am Chem Soc</em> 139(26): 9001-9009.</li><li>Tyagi, S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19404252" target="_blank">Imaging intracellular RNA distribution and dynamics in living cells.</a> <em>Nat Methods</em> 6(5): 331-338.</li><li>Wang, L., Frei, M. S., Salim, A. and Johnsson, K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30550714" target="_blank">Small-molecule fluorescent probes for live-cell super-resolution microscopy.</a> <em>J Am Chem Soc</em> 141(7): 2770-2781.</li><li>Wirth, R., Gao, P., Nienhaus, G. U., Sunbul, M. and JÃ¤schke, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30986047" target="_blank">SiRA: a silicon rhodamine-binding aptamer for live-cell super-resolution RNA imaging.</a> <em>J Am Chem Soc</em> 141(18): 7562-7571.</li><li>Xia, Y., Zhang, R., Wang, Z., Tian, J. and Chen, X. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28345687" target="_blank">Recent advances in high-performance fluorescent and bioluminescent RNA imaging probes.</a> <em>Chem Soc Rev</em> 46(10): 2824-2843.</li><li>Yerramilli, V. S. and Kim, K. H. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29513000" target="_blank">Labeling RNAs in live cells using malachite green aptamer scaffolds as fluorescent probes.</a> <em>ACS Synth Biol</em> 7(3): 758-766.<br/></li></ol>