<body>
            <h1>Split Nano Luciferase-based Assay to Measure Assembly of Japanese Encephalitis Virus</h1>
            <p><strong>Authors:</strong> Simon Goto, Kotaro Ishida, Ryosuke Suzuki, Eiji Morita</p>
            <h2>Abstract</h2>
            <p> Cells infected with flavivirus release various forms of infectious and non-infectious particles as products and by-products. Comprehensive profiling of the released particles by density gradient centrifugation is informative for understanding viral particle assembly. However, it is difficult to detect low-abundance minor particles in such analyses. We developed a method for viral particle analysis that integrates a high-sensitivity split luciferase system and density gradient centrifugation. This protocol enables high-resolution profiling of particles produced by cells expressing Japanese encephalitis virus factors.</p>
            <p><strong>Keywords:</strong> Flavivirus, Japanese encephalitis virus, Virion assembly, HiBiT tag, Split nano luciferase</p>
            <h2>Background</h2><p>The flavivirus, a group of arboviruses including dengue virus, West Nile virus, tick-borne encephalitis virus, Zika virus, yellow fever virus, and Japanese encephalitis virus (JEV), is associated with substantial morbidity and mortality in human populations (Chambers <em>et al.</em>, 1990). Flaviviruses have a single ORF genome encoding a long polypeptide, which is post-translationally cleaved into three structural (C, prM and E) and seven non-structural (NS1, NS2A, NS2B, NS3, NS4A, NS4B and NS5) proteins by the host signal peptidase and NS2B-NS3 viral protease (Chambers <em>et al.</em>, 1990).<br/><br/>Viral genomic RNA synthesized by the NS5 RNA-dependent RNA polymerase associates with C protein to form the nucleocapsid, which in turn is packaged in the envelope composed of host cell-derived lipid membrane, and viral prM and E transmembrane proteins to form the virion. It has been found that a reduced form of flaviviral genomic RNA, designated as subgenomic replicon, which lacks all structural genes, can replicate itself in host cells (Suzuki <em>et al.</em>, 2014), and that the subgenomic RNA is incorporated into the virus-like particle to form single-round infectious particles (SRIPs) when C, prM and E proteins are supplied in <em>trans</em> (Suzuki <em>et al.</em>, 2014; Matsuda <em>et al.</em>, 2018).<br/><br/>Besides major infectious particles, virus-infected cells often produce minor particles, some of which are hard to detect with immunochemical methods, due to their small amounts. To study typical and atypical particles released from flavivirus-infected cells, we modified a SRIP system (Matsuda <em>et al.</em>, 2018) to develop a JEV HiBiT-SRIP assay system. In this system, the C protein is labeled with a highly sensitive HiBiT peptide tag, to enable high-resolution sedimentation profiling of particles produced by the cells expressing viral structural and nonstructural factors (Ishida <em>et al.</em>, 2019). Cells transfected with a set of plasmids (referred to as SRIPs-producer cells) produce SRIPs and non-infectious subviral particles containing C-HiBiT, which can be separated by sucrose density gradient centrifugation. SRIPs in sedimentation fractions can be detected by measurement of cellular NanoLuc levels after inoculation of other cells (referred to as target cells) with all the fractions, which allows SRIP-borne NanoLuc expression (Figure 1). It is possible to apply the principle of JEV HiBiT-SRIP assay to other viruses.<br/><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425205900_6509.jpg" width="300"/><br/><strong>Figure 1. Overview of HiBiT-SRIP system.</strong> A. HiBiT-SRIP system plasmids. HDV-Rz: self-cleaving HDV Ribozyme; pA: SV40 polyadenylation signal; CMVp: CMV promoter; CAGp: CAG promoter. B. Schematic presentation of HiBiT-SRIP experiments. JErep-nluc: subgenomic replicon with a gene for NanoLuc as the reporter.<br/></p></p>
        </body>

<p class="pptt" id="biaoti33043">## Materials and Reagents</p><ol class="npwtwok"><li>1.5 ml tube (Watson, catalog number: <a class="blackNoUnder" target="_blank"> 131-815C </a> )</li><li>2 ml tube (Watson, catalog number: <a class="blackNoUnder" target="_blank"> 132-620C </a> )</li><li>6-well cell culture plate (Violamo, catalog number: <a class="blackNoUnder" target="_blank"> 2-8588-01 </a> )</li><li>96-well cell culture plate (Violamo, catalog number: <a class="blackNoUnder" target="_blank"> 2-8588-05 </a> )</li><li>14 x 95 mm open-top polyclear centrifuge tubes (Seton Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 7031 </a> )</li><li>White 384-well immuno plates (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 460372 </a> )</li><li>0.22 Î¼m filter (Merck Millipore, catalog number: <a class="blackNoUnder" target="_blank"> UFC30GV00 </a> )</li><li>293T cells (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> CRL-3216 </a> )</li><li>Huh7 cells (JCRB, catalog number: <a class="blackNoUnder" target="_blank"> JCRB0403 </a> )</li><li>Plasmid pCAG-JEC-FLAG-HiBiT which encodes HiBiT-tagged wild-type JEV C protein (Ishida <em>et al.</em>, 2019; available from authors upon request) or its variant encoding mutant forms of C protein</li><li>Plasmid pCAG-prME which encodes JEV prM and E proteins (Suzuki <em>et al.</em>, 2014; available from authors upon request)</li><li>Plasmid pCMV-JErep-nluc which expresses a subgenomic replicon RNA of JEV, which contains a NanoLuc reporter gene (Matsuda <em>et al.</em>, 2018, Ishida <em>et al.</em>, 2019; available from authors upon request)</li><li>Nano Glo HiBiT lytic detection system (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N3040 </a> )</li><li>Nano-Glo luciferase assay system (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N1120 </a> )</li><li>Dulbeccoâs modified Eagleâs media (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 08458-16 </a> )</li><li>FBS (fetal bovine serum) (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 10270-106 </a> )</li><li>PBS (phosphate-buffered saline without calcium and magnesium) (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 14249-24 </a> )</li><li>Benzylpenicillin potassium (Fujifilm, catalog number: <a class="blackNoUnder" target="_blank"> 021-07732 </a> )</li><li>Streptomycin sulfate (Tokyo Chemical Industry, catalog number: <a class="blackNoUnder" target="_blank"> S0585 </a> )</li><li>PEI MAX (Polyscience, catalog number: <a class="blackNoUnder" target="_blank"> 24765-1 </a> ), a transfection reagent</li><li>Opti-MEM (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 31985062 </a> ), reduced serum medium which allows to keep high transfection efficiency with less cytotoxicity</li><li>Sucrose, centrifugal density-gradient grade (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 30406-25 </a> )</li><li>Tris (Tris (hydroxymethyl) aminomethane) (Nacalai, catalog number:<a class="blackNoUnder" target="_blank"> 35406-91 </a> )</li><li>NaCl (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 31320-05 </a> )</li><li>Triton X-100 (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 35501-15 </a> )</li><li>KCl (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 28513-85 </a> )</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 31726-05 </a> )</li><li>KH<sub>2</sub>PO<sub>4</sub> (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 28720-65 </a> )</li><li>PEI solution (1 mg/ml) (see Recipes)</li><li>100x penicillin G + streptomycin stock solution (see Recipes)</li><li>Culture media (see Recipes)</li><li>10x PBS (pH 7.4) (see Recipes)</li><li>10% sucrose/PBS (see Recipes)</li><li>45% sucrose/PBS (see Recipes)</li><li>10-45% sucrose linear gradient/PBS (see Recipes)</li><li>Lysis buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti33044">### Equipment</p><ol class="npwtwok"><li>Humidified incubator (PHCBI, model: <a class="blackNoUnder" target="_blank"> MCO-170AICUVD-PJ </a> , 37 Â°C, 5% CO<sub>2</sub>)</li><li>Ultracentrifuge (Hitachi, model: <a class="blackNoUnder" target="_blank"> CP80NX </a> )</li><li>Swing bucket rotor (Hitachi, model: <a class="blackNoUnder" target="_blank"> P40ST </a> )</li><li>Gradient mixer (BIOCOMP, Gradient Mate)</li><li>Piston gradient fractionator (BIOCOMP, catalog number: <a class="blackNoUnder" target="_blank"> 153 </a> )</li><li>Microplate luminometer (Thermo, Varioskan LUX Multimode Microplate Reader)</li><li>-30 Â°C freezer (PHCBI, catalog number: <a class="blackNoUnder" target="_blank"> MDF-MU539D </a> )<br/></li></ol><p class="pptt" id="biaoti33045">### Procedure</p><p class="ppzz">Flow of following procedures is presented in Figure 2.</p><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425205640_3384.jpg" width="300"/><br/><strong>Figure 2. Flow of procedures</strong><br/> </p><br/><ol class="npwone"><li>Production of HiBiT-containing particles<ol class="npwtwo"><li>Seed 293T cells in 6-well plates, at density of ~50,000 cells/well in 2 ml growth medium.</li><li>Place the plates in a CO<sub>2</sub> incubator for 24 h.</li><li>Mix 0.2 Âµg pCAG-JEC-HiBiT, 0.4 Âµg pCMV-JErep-nluc and 0.2 Âµg pCAG-JEprME in 200 Âµl Opti-MEM, mix well with a vortex mixer.<br/>(Optional) Set control experiments using an empty vector in place of each plasmid.<br/>(Optional) Set experiments using plasmids encoding mutant forms of viral elements of interest.</li><li>Add 2.4 Âµl PEI solution (1 mg/ml) to the mixture and mix well with a vortex mixer.</li><li>Incubate the DNA-PEI mixture for 15 min at room temperature.</li><li>Apply the DNA-PEI mixture to cell-containing wells (~200 Î¼l/well).</li><li>Place the plates in a CO<sub>2</sub> incubator.</li><li>Replenish growth media 6 h post-transfection.<br/><em>Note: Add media slowly to the side of the well to avoid cell detachment.</em> </li><li>Replenish growth media 48 h post-transfection.<br/><em>Note: Add media slowly to the side of the well to avoid cell detachment.</em> </li><li>Transfer the culture supernatant 72 h post-transfection to 2 ml tubes.</li><li>Centrifuge the tube at 20,000 <em>x g</em>, 4 Â°C for 10 min.</li><li>Transfer supernatant to fresh tubes and proceed to Procedure B or Procedure C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Evaluation of replication efficiency of the subgenomic replicon RNA in SRIPs-producer cells (a typical result is presented in Figure 2A)<ol class="npwtwo"><li>Suspend cells in each well in 1 ml PBS and transfer to a 1.5 ml tube.</li><li>Centrifuge the tubes at 2,400 <em>x g</em>, 4 Â°C for 5 min.</li><li>Aspirate the supernatant, add 50 Î¼l lysis buffer and mix by pipetting.</li><li>Centrifuge the tubes at 20,000 <em>x g</em>, 4 Â°C for 10 min.</li><li>Transfer 5 Î¼l of the clear lysate in each tube to 384-well plate.</li><li>Add 5 Î¼l of NanoLuc substrate diluted 1:600 with lysis buffer.</li><li>Incubate for 10 min at room temperature.</li><li>Measure the luminescence using a microplate reader.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Titration of SRIPs in culture supernatant (a typical result is presented in Figure 2C)<ol class="npwtwo"><li>Seed Huh7 cells in 96-well plates, at density of ~10,000 cells/well and incubate in a CO<sub>2</sub> incubator overnight.</li><li>Inoculate 100 Î¼l of culture supernatant (prepared with Procedure B) into each well.</li><li>Incubate for 72 h.</li><li>Aspirate growth media and add 50 Î¼l lysis buffer to each well and mix by pipetting.</li><li>Transfer 5 Î¼l of the lysate in each well to a 384-well plate.</li><li>Add 5 Âµl of NanoLuc substrate diluted 1:600 with lysis buffer.</li><li>Incubate for 10 min at room temperature.</li><li>Measure the luminescence using a microplate reader.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sucrose density gradient centrifugation of the culture supernatant<ol class="npwtwo"><li>Layer 300 Î¼l of the culture supernatant containing HiBiT-labeled particles on continuous 10-45% sucrose gradient/PBS in Open-Top Polyclear centrifuge tubes (SETON, 14 x 95 mm).</li><li>Centrifuge the gradients at 100,000 <em>x g</em> for 3 h at 4 Â°C with a pre-chilled swing bucket rotor.</li><li>Fractionate the gradients from top to bottom (total &lt; 40 fraction), with a piston fractionator (2.3 mm/fraction, 0.3 mm/s) to a 96-well plate.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Titration of C-HiBiT-containing particles in sedimentation fractions (a typical result is presented in Figure 2B)<ol class="npwtwo"><li>Transfer 5 Î¼l of the fraction in each well to 384-well plate.</li><li>Add 5 Î¼l of HiBiT reagent to each well.</li><li>Incubate for 10 min at room temperature.</li><li>Measure the luminescence using a microplate reader. Typical results out of 3 &gt; experiments are indicated in the figures.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Titration of SRIPs in sedimentation fractions (a typical result is presented in Figure 2D)<ol class="npwtwo"><li>Seed Huh7 cells in 96-well plates, at density of ~10,000 cells/well and incubate in a CO<sub>2</sub> incubator overnight.</li><li>Inoculate 5 Î¼l of sedimentation fractions to each well.</li><li>Incubate for 72 h.</li><li>Aspirate growth media and add 50 Î¼l lysis buffer to each well and mix by pipetting.</li><li>Transfer 5 Î¼l of the lysate in each well to 384-well plate.</li><li>Add 5 Âµl of NanoLuc substrate diluted 600 fold with lysis buffer.</li><li>Incubate for 10 min at room temperature.</li><li>Measure the luminescence using a microplate reader (Figure 3).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200425/20200425205715_6511.jpg" width="300"/><br/><strong>Figure 3. Typical results.</strong> (A) NanoLuc activity of producer cell lysate to evaluate autonomous replication of the subgenomic RNA (B) HiBiT activities of fractions of producer cell culture supernatant, fractionated via sucrose density gradient centrifugation. (C) NanoLuc activity of target cells inoculated with producer cell supernatant, which reflects the infectious titer of SRIPs in the supernatant. (D) NanoLuc activities of target cells inoculated with sucrose density gradient centrifugation fractions of producer cell supernatant, which reflect the infectious titer of SRIPs in the fractions.</li></ol></li></ol><p class="pptt" id="biaoti33046">### Notes</p><ol class="npwtwok"><li>Strong luminescence emitted by NanoLuc/HiBiT system sometimes leaks to adjacent wells in 384-well plates. Therefore, the recommendation is to load samples into every other well. In addition, sometimes it is better to dilute lysates or fractions which produce strong signals to minimize leakage of luminescence to neighboring wells (Procedures B, D, E and F).</li><li>Be aware that it is difficult to estimate relative amounts of C-HiBiT in top fractions when titrating HiBiT-SRIPs in sedimentation fractions (Procedure B), because free NanoLuc proteins that leak from cells in the fractions also contribute to signal production.<br/></li></ol><p class="pptt" id="biaoti33047">### Recipes</p><ol class="npwtwok"><li>PEI solution (1 mg/ml)<br/>PEI MAX 100 mg<br/>ddH<sub>2</sub>O up to 100 ml<br/>Sterilize via a 0.22 Î¼m filter, aliquot to 15 ml tubes and keep a tube as a working stock in a refrigerator and the rest of the tubes in a -30 Â°C freezer</li><li>100x penicillin G + streptomycin stock solution<br/>PBS 100 ml<br/>Benzylpenicillin potassium 0.626 g<br/>Streptomycin sulfate 1 g<br/>Sterilize via a 0.22 Î¼m filter and store in a refrigerator</li><li>Culture media<br/>Dulbeccoâs modified Eagleâs medium 500 ml<br/>FBS, heat-inactivated by incubation at 56 Â°C for 45 min 50 ml<br/>100x penicillin G + streptomycin stock solution 5 ml</li><li>10x PBS (pH 7.4)<br/>NaCl 80 g<br/>KCl 2.0 g<br/>Na<sub>2</sub>HPO<sub>4</sub> 14.4 g<br/>KH<sub>2</sub>PO<sub>4</sub> 2.4 g<br/>ddH<sub>2</sub>O up to 1 L</li><li>10% sucrose/PBS<br/>Sucrose 100 g<br/>10x PBS 100 ml<br/>ddH<sub>2</sub>O up to 1 L<br/>Sterilize via a 0.22 Î¼m filter and store at room temperature</li><li>45% sucrose/PBS<br/>Sucrose 450 g<br/>10x PBS 100 ml<br/>ddH<sub>2</sub>O up to 1 L<br/>Sterilize via a 0.22 Î¼m filter and store at room temperature<br/></li><li>10-45% sucrose linear gradient/PBS<br/>Pour appropriate volume of 10% sucrose/PBS to 14 x 95 mm open-top polyclear centrifuge tubes, underlay 45% sucrose/PBS from bottom of the tubes using a needle attached syringe. Form linear gradients using gradient former with following parameters:<br/>Time: 2 min 5 s<br/>Angle: 82Â°<br/>Speed: 16 rpm<br/>After forming the gradients, chill them by standing several h at 4 Â°C</li><li>Lysis buffer<br/>1 M Tris pH 7.5 20 ml<br/>5 M NaCl 30 ml<br/>Triton X-100 10.7 g<br/>ddH<sub>2</sub>O up to 1 L<br/></li></ol><p class="pptt" id="biaoti33048">### Acknowledgments</p><p class="ppzz">We thank Jiahui Ong for proofreading the manuscript. We thank Mami Matsuda for the helpful advice and suggestions to develop HiBiT-SRIP system.</p><p class="pptt" id="biaoti33049">### Competing interests</p><p class="ppzz">The authors have no competing interests directly relevant to the content of this article.</p><p class="pptt" id="biaoti33050">### References</p><ol class="npwtwok"><li>Chambers, T. J., Hahn, C. S., Galler, R. and Rice, C. M. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2174669" target="_blank">Flavivirus genome organization, expression, and replication.</a> <em>Annu Rev Microbiol</em> 44: 649-688.</li><li>Ishida, K., Goto, S., Ishimura, M., Amanuma, M., Hara, Y., Suzuki, R., Katoh, K. and Morita, E. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31315991" target="_blank">Functional correlation between subcellular localizations of japanese encephalitis virus capsid protein and virus production.</a> <em>J Virol</em> 93(19).</li><li>Matsuda, M., Yamanaka, A., Yato, K., Yoshii, K., Watashi, K., Aizaki, H., Konishi, E., Takasaki, T., Kato, T., Muramatsu, M., Wakita, T. and Suzuki, R. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30413742" target="_blank">High-throughput neutralization assay for multiple flaviviruses based on single-round infectious particles using dengue virus type 1 reporter replicon.</a> <em>Sci Rep</em> 8(1): 16624.</li><li>Suzuki, R., Ishikawa, T., Konishi, E., Matsuda, M., Watashi, K., Aizaki, H., Takasaki, T. and Wakita, T. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24096319" target="_blank">Production of single-round infectious chimeric flaviviruses with DNA-based Japanese encephalitis virus replicon.</a> <em>J Gen Virol 95</em>(Pt 1): 60-65.<br/></li></ol>