<body>
            <h1>Live Cell Imaging of Male Meiosis in <em>Arabidopsis</em> by a Landmark-based System</h1>
            <p><strong>Authors:</strong> Maria Ada Prusicki, Emma Mathilde Keizer, Rik Peter van Rosmalen, Christian Fleck, Arp Schnittger</p>
            <h2>Abstract</h2>
            <p>Live cell imaging has tremendously promoted our understanding of cellular and subcellular processes such as cell division. Here, we present a step-by-step protocol for a robust and easy-to-use live cell imaging approach to study male meiosis in the plant <em>Arabidopsis thaliana</em> as recently established. Our method relies on the concomitant analysis of two reporter genes that highlight chromosome configurations and microtubule dynamics. In combination, these reporter genes allowed the discrimination of five cellular parameters: cell shape, microtubule array, nucleus position, nucleolus position, and chromatin condensation. These parameters can adopt different states, <em>e.g.</em>, the nucleus position can be central or lateral. Analyzing how tightly these states are associated gives rise to landmark stages that in turn allow a quantitative and qualitative dissection of meiotic progression. We envision that such an approach can also provide valuable criteria for the analysis of cell differentiation processes outside of meiosis.</p>
            <p><strong>Keywords:</strong> Cell division, Cell differentiation, Meiosis, Chromosome, Microtubules, Cytoskeleton, <em>Arabidopsis</em></p>
            <h2>Background</h2><p>Meiosis is a special cell division cycle that serves two major purposes. First, the DNA content of the meiotic mother cell is reduced by half that leads, in the case of a diploid organism, to haploid meiotic products. This reduction is necessary for sexually reproducing organisms so that, after the fusion of two gametes during fertilization, the original genome size is restored. Second, meiosis promotes genetic diversity through the exchange of DNA segments between the parental chromosomes (homologous chromosomes or shortly homologs), named meiotic recombination, and by the generation of new, yet complete chromosome sets in which randomly either the homolog of the mother or the father is present for each chromosome (in case of a diploid organism). Thus, understanding meiosis is interesting for different fields of research ranging from cell biology and reproductive biology via genetics to evolutionary biology (Wijnker and Schnittger, 2013; Mercier <em>et al.</em>, 2015; Melamed-Bessudo <em>et al.</em>, 2016; Lambing and Heckmann, 2018; PelÃ© <em>et al.</em>, 2018; Wang and Copenhaver, 2018).<br/> <br/>Plants are a powerful model system to study the various aspects of meiosis. In addition, research in plant meiosis is also promoted by an applied interest since a low level of recombination often restricts breeding programs.<br/> <br/>Research on plant meiosis has largely relied on the analysis of fixed cells, <em>e.g.</em>, for chromosome spreads and immunolocalization studies, both for <em>Arabidopsis</em> (Armstrong <em>et al.</em>, 2009; Martinez-Garcia and Pradillo, 2017; Parra-Nunez <em>et al.</em>, 2020; Sims <em>et al.</em>, 2020) as well as for other plant species (Chelysheva <em>et al.</em>, 2013; Sepsi <em>et al.</em>, 2018; Darrier <em>et al.</em>, 2020; Stack <em>et al.</em>, 2020). These techniques have been and continue to be important tools as they offer a great spatial resolution to address the structure of meiotic chromosomes and the localization of meiotic regulators for instance demonstrated by an immuno-cytological analysis of the 3D configuration of meiotic chromosomes (Hurel <em>et al.</em>, 2018). However, these studies provide little to no information about the underlying dynamics of meiosis. In addition, temporal aspects of meiosis have to be indirectly deduced by the frequency of observed stages, a procedure that is inherently error-prone and can easily misguide the researcher <em>e.g.</em>, when two or more populations of meiocytes exist in the same sample undergoing an altered course of meiosis that can be mistaken as one population with cells at different stages (Prusicki <em>et al.</em>, 2019; Sofroni <em>et al.</em>, unpublished). Moreover, short-lived phases, for instance nuclear envelope breakdown, are difficult to catch, and there has been for instance a long discussion in the field whether the nuclear envelop is reformed in <em>Arabidopsis</em> after the first meiotic division.<br/> <br/>Live cell imaging of meiosis can complement the analyses of fixed material and build together with these techniques a powerful approach to reach molecular mechanistic insights into this important cell division program. Live cell-imaging of plant meiosis has been previously initiated in maize (Yu <em>et al.</em>, 1997; Sheehan and Pawlowski, 2009; Nannas <em>et al.</em>, 2016). However, only short phases of meiosis in maize could be recorded with a genetic reporter for microtubules and a chemical stain that highlights DNA. In addition, a recent protocol for live cell imaging of meiosis by light-sheet microscopy has been published (Valuchova <em>et al.</em>, 2020). While this set-up is very powerful to follow meiosis in entire flower buds, it still does not reach the subcellular resolution as obtained by confocal laser scanning microscopy (CLSM).<br/> <br/>Here, we describe in detail a method to follow meiosis in <em>Arabidopsis</em> anthers based on a recently established procedure by CLSM (Prusicki <em>et al.</em>, 2019). Importantly, this method allows keeping the samples alive up to several days allowing the analysis of meiosis in its entirety. Furthermore, the use of <em>Arabidopsis</em> allowed the generation of plants containing fluorescent reporters for different meiotic regulators, foremost the meiosis-specific kleisin subunit of the cohesin complex called RECOMBINATION 8 (REC8), ASYNAPTIC 1 (ASY1) and ASY3, two components of the chromosome axis, and ZYP1, a component of the central region of the synaptonemal complex (Prusicki <em>et al.</em>, 2019; Yang <em>et al.</em>, 2019 and 2020). This set up has recently been used to study the control of cohesin in mutants in which REC8 is prematurely cleaved (Cromer <em>et al.</em>, 2019).<br/> <br/>A major challenge is the quantitative analysis of the obtained movies. As typical for biological processes, meiosis is a continuous and gradual succession of events. To dissect these movies, we focused on five cellular parameters as visualized by the KINGBIRD reporter system (REC8 labeled with GFP in combination with microtubules highlighted in red; Prusicki <em>et al.</em>, 2019). These five cellular parameters are: cell shape, microtubule array, nucleus position, nucleolus position and chromatin state. Each of these parameters can adapt different states, see Table 1 (<a href="https://os.bio-protocol.org/attached/file/20200322/table1 bioprotocol.xlsx" target="_blank">Table 1 Excel file</a>), this publication and Figure 3 in Prusicki <em>et al.</em>, 2019.<br/> <br/>Looking then at the association of these different parameter states revealed that they are not randomly associated but often tightly linked. This gives rise to a biological landmark system where one landmark is a prominent cellular configuration with distinct parameter states. In turn, this landmark system can be used as a map to qualitatively (appearance of the same or new landmarks) and quantitatively (duration of these landmarks) dissect meiotic progression in mutants or different environmental conditions. The use of other meiotic reporters can then be used to refine and/or complement this landmark system. The principle of this analysis can be easily translated to other cellular differentiation processes, of course including other cellular parameters, which need to be identified.</p>
        </body>

<p class="pptt" id="biaoti32949">## Materials and Reagents</p><ol class="npwtwok"><li>Black marker</li><li>Squared Petri dishes 100 x 100 x 20 mm (Sardstedt, catalog number: <a class="blackNoUnder" target="_blank"> 82.9923.422 </a> )</li><li>Round Petri dishes 35 x 10 mm (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 82.1135.500 </a> )</li><li>Round Petri dishes 60 x 15 mm (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 82.1194.500 </a> )</li><li>Safeseal tube 1.5 ml (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.706 </a> )</li><li>Parafilm M (Neolab, Bemis, 3-1011, PM-996)</li><li>Sterilizing filter Millex-GV 0.22 Î¼m (Merck, catalog number: <a class="blackNoUnder" target="_blank"> SLGV033RS </a> )</li><li>Needle 30 G Â½â 0.3 x 13 mm (BD Microlance, model: <a class="blackNoUnder" target="_blank"> 304000 </a> )</li><li>Seeds of <em>Arabidopsis thaliana</em> plants containing meiotic fluorescent reporter constructs as a hallmark of meiosis such as the <em>KINGBIRD</em> line presented in Prusicki <em>et al.</em>, 2019 as well as reporters for the meiotic chromosome axis and the synaptonemal complex (Yang <em>et al.</em>, 2019 and 2020)</li><li>NaClO in H<sub>2</sub>O, solution 13% (Applichem GmbH, ITW Reagents, catalog number: <a class="blackNoUnder" target="_blank"> 213322.0715 </a> )</li><li>HCl 37% (VWR Chemicals, catalog number: <a class="blackNoUnder" target="_blank"> 20255.29 </a> 0)</li><li>Agarose (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A9539 </a> )</li><li>Agar, powdered food grade (Applichem GmbH, ITW Reagents, catalog number: <a class="blackNoUnder" target="_blank"> A00917,5000 </a> )</li><li>MS Basal Salt Mixture (Duchefa Biochemie, catalog number: <a class="blackNoUnder" target="_blank"> M0221.0050 </a> )</li><li>Myo-Inositol (Duchefa Biochemie, catalog number: <a class="blackNoUnder" target="_blank"> I0609.0100 </a> )</li><li>Nicotinic acid (Duchefa, catalog number: <a class="blackNoUnder" target="_blank"> N0611 </a> )</li><li>Pyridoxin hydrochloride (Duchefa, catalog number: <a class="blackNoUnder" target="_blank"> P0612 </a> )</li><li>Thiamine hydrochloride (Duchefa, catalog number: <a class="blackNoUnder" target="_blank"> T0614 </a> )</li><li>Glycine (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> G-7126 </a> )</li><li>Sterile dH<sub>2</sub>O</li><li>Acetocarmine</li><li>Isopropanol</li><li>Sucrose</li><li>KOH</li><li>Murashike and Skoog medium (MS medium) for plant germination (see Recipes)</li><li><em>Arabidopsis</em> Apex Culture Medium (ACM) for imaging (see Recipes)<br/></li></ol><p class="pptt" id="biaoti32950">### Equipment</p><ol class="npwtwok"><li>Pipettes 10/100/1000 (<em>e.g.</em>, Eppendorf Research plus pipette)</li><li>Growth chambers MobyLux GroBanks (CLF Plant Climatics, model: <a class="blackNoUnder" target="_blank"> BrightBoy </a> )</li><li>Vacuum chamber</li><li>Glass bicker</li><li>Clean bench</li><li>Fine tweezers, thickness: 0.02/0.01 mm (Dumont, Dumoxel, catalog numbers: <a class="blackNoUnder" target="_blank"> 0203-4-PO and 0203-5-PO </a> )</li><li>Dissection microscopes (<em>e.g.</em>, Zeiss, models: Stemi2000 and Stemi508)</li><li>Laser Scanning Microscope (<em>e.g.</em>, Zeiss, model: <a class="blackNoUnder" target="_blank"> LSM880 </a> ), equipped with a water dipping objective (<em>e.g.</em>, Zeiss, model: <a class="blackNoUnder" target="_blank"> W Plan-Apochromat 40x/1.0 DIC objective </a> )<br/></li></ol><p class="pptt" id="biaoti32951">### Software</p><ol class="npwtwok"><li>ZEN 2.3 SP1 FP1 black (Zeiss) (or the related imaging software of your microscope system)<br/></li><li>MetaMorph Version 7.8.0.0 (Molecular devices)</li><li>Fiji version 1.52b (ImageJ, <a href="https://imagej.net/Fiji" target="_blank">https://imagej.net/Fiji</a>) (Schindelin <em>et al.</em>, 2012)</li><li>Excel (Microsoft Office)</li><li>Python programming language (Version 3.6, Python Software Foundation)<br/></li></ol><p class="pptt" id="biaoti32952">### Procedure</p><ol class="npwone"><li>Preparation of plant material<ol class="npwtwo"><li>Surface-sterilize seeds of <em>Arabidopsis thaliana</em>, carrying a fluorescent reporter expressed at meiotic stages (an example is available in Prusicki <em>et al.</em>, 2019). One efficient method to surface-sterilize seeds is by chloride gas: pour the seeds in 1.5 ml tubes, and mark the tubes with a black marker (other colors will faint due to the sterilization process). Leave the lid of the tubes open, and place the tubes in a racket inside a vacuum chamber. In the same chamber place a glass bicker containing 25 ml of a 7.5% bleach solution (eluted with tap water). Add to the bicker 1 ml of 37% HCl, quickly close the chamber and apply vacuum. The whole procedure should be carried out under a chemical hood. Be careful not to breathe the chloride gas! Leave the seeds in the chamber for 3-4 h. After the treatment, the seeds are ready to be sawn. If instead you are planning to store the sterilized seeds in the tube for some time, let the tubes open under a clean bench for a minimum of 30 min to ensure the gas has completely evaporated.</li><li>Saw the seeds on Murashike and Skoog (Â½ MS, Recipe 1) squared Petri dishes enriched with selecting agent according with the resistance of the inserted construct. <em>e.g.</em>, add 25 mg/L Hygromycin B to select for <em>PRO<sub>REC8</sub>:REC8:GFP</em> as done in Prusicki <em>et al.</em> (2019). Alternatively, the selection can be performed by genotyping or based on fluorescence detection, if the protein of interest is expressed in seedlings.</li><li>Store the plates for stratification in the dark for 3 days at 4 Â°C and then transfer them to long-day conditions (16 h day/8 h night regime at 22 Â°C/18 Â°C) for germination.</li><li>10 days after germination transfer the seedlings on soil and grow them at long-day conditions.</li><li>Plants will be ready to image in 3 to 4 weeks, depending on growth conditions and genotypes.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Preparation of imaging samples<ol class="npwtwo"><li>Prepare Apex Culture Medium (ACM) with agarose to 0.8%, 1% and 2% concentrations and 1,000x vitamin stock as indicated in the session (Recipe 2).</li><li>Sterilize the ACM by autoclaving.</li><li>When the autoclaved (or reheated) ACM with 0.8% agarose is hand warm, add the vitamin mix to a 1x working concentration (<em>e.g.</em>, to 50 ml of ACM add 50 Î¼l of a 1,000x vitamin stock solution), and pour it into small Petri dishes (35 x 15 mm), seal with parafilm and store at 4 Â°C.</li><li>Pour the ACM with 1% agarose in medium-sized Petri dishes (60 x 15 mm), seal with parafilm and store at 4 Â°C.</li><li>Aliquot the ACM with 2% agarose in sterile 1.5 ml Eppendorf tubes and store them at 4 Â°C.<br/><br/></li></ol></li><li>Sample mounting<ol class="npwtwo"><li>Cut an inflorescence and anchor it on a Petri dish with ACM with 1% agarose (Video 1).<br/><br/><!--video3611-1å¼å§--><div class="videodiv" id="vd3611-1" style="height:480px;"></div><div class="videophoto" id="vp3611-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3611/3611-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Removal of large flower buds and selection of the flower bud for image acquisition, Steps C1 and C2</strong> </div><!--video3611-1ç»æ--><br/></li><li>Remove all the open and the elongated flowers at the base of the pedicel but the flower bud with a roundish shape and the size between 0.4-0.6 mm containing early meiocytes (Video 1). The characteristics of flower buds with meiocytes in an early stage can vary depending on the growth conditions and the genotype, <em>e.g.</em>, mutants in which meiosis is delayed usually takes places in flower buds of larger sizes. It is therefore recommended to analyze several flower buds with acetocarmine or similar staining to estimate in which flower buds meiotic stages can be found. In general, flower buds at developmental stage 9 (Smyth <em>et al.</em>, 1990) contain male meiocytes undergoing meiosis.</li><li>Remove the upper sepal of the selected flower bud, being careful to not touch the inner organs of the flower, in particular the anthers (Videos 2 and 3).<br/><br/><!--video3611-2å¼å§--><div class="videodiv" id="vd3611-2" style="height:480px;"></div><div class="videophoto" id="vp3611-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3611/3611-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. Removal of sepal to give access to two anthers in the selected flower bud, Steps C3 and C4</strong> </div><!--video3611-2ç»æ--><br/><!--video3611-3å¼å§--><div class="videodiv" id="vd3611-3" style="height:480px;"></div><div class="videophoto" id="vp3611-3"><a class="newapic" href="javascript:;" onclick="openvideo(3)" style="background:url('https://en-cdn.bio-protocol.org/video/3611/3611-3s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 3. Close up of a flower bud ready to image, after Steps C3 and C4</strong> </div><!--video3611-3ç»æ--><br/></li><li>Remove the remaining flower buds (Videos 2 and 3). The next smallest/youngest flower buds can be left attached as a backup flower in the case the main flower bud gets damaged during the removal of the sepal.</li><li>Lift the inflorescence from the preparation medium and cleanly cut the stem to a length of circa (<em>ca.</em>) 0.5 cm (with forceps or better using a needle) (Video 4). This will facilitate the uptake of nutrients from the medium and will keep the sample in good condition for a long time.<br/><br/><!--video3611-4å¼å§--><div class="videodiv" id="vd3611-4" style="height:480px;"></div><div class="videophoto" id="vp3611-4"><a class="newapic" href="javascript:;" onclick="openvideo(4)" style="background:url('https://en-cdn.bio-protocol.org/video/3611/3611-4s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 4. Positioning of the selected and processed flower bud for imaging, steps C5 and C6</strong> </div><!--video3611-4ç»æ--><br/></li><li>Transfer the sample onto the small Petri dish with ACM with 0.8% agarose, anchor it in the middle of the plate and cover the flower bud with a drop of ACM with 2% agarose (Video 4 and Figures 1A and 1B).</li><li>It is possible to follow more than one flower bud in one microscopy session. In this case, prepare a new sample as previously explained (starting from Procedure C1 of the <em>Bio-protocol</em>), and locate all flower buds as close as possible in the center of the Petri dish (Video 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200427/20200427014215_5422.jpg" width="102"/><br/><strong>Figure 1. Sample mounting and positioning at the microscope.</strong> A. The sample is placed in the middle of a Petri dish and covered by a drop of 2% agarose. B. Close up of the sample from panel A. C Overview and D zoom onto the microscope stage with the sample positioned under the water immersion objective.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Image acquisition<ol class="npwtwo"><li>It is important to have stable acquisition conditions, <em>e.g.</em>, room temperature, humidity, <em>etc.</em> Therefore, it is advisable to turn on the confocal microscope in advance and position the sample in the room with a few ml of water covering the mounted samples.</li><li>After approximately one hour you can start the image acquisition.</li><li>Position the sample on the microscope stage, submerge the water-dipping objective and fill up the Petri dish to the top with autoclaved water (Figures 1C and 1D). It is advisable to thoroughly clean the objective with isopropanol before the acquisition to reduce the risk of bacteria growth during image acquisition.</li><li>Using the visual function (or related software tools), identify the position of each flower bud in the sample and save it using the multi-position function on ZEN black software.</li><li>The set-up for image acquisition needs to be adapted to the purpose of the experiment and to the characteristics of the used fluorophores. To follow a complete meiotic division in a time-lapse experiment, and to identify the meiotic landmarks using the <em>KINGBIRD</em> line as presented in Prusicki <em>et al.</em> (2019), the following set-up can be used:<br/><ol class="npwthree"><li>Use the Argon laser at Î» 488 nm to excite GFP and the DPSS 561-10 laser (Î» 561) to excite TagRFP.</li><li>Use two tracks in sequential line mode for signal detection, and the Beam splitter MBS 488/561. In the first track, use the GaAsP detector to record the GFP signal filtered for Î» between 498 and 550 nm. In the second track, use the GaAsP detector to observe the TagRFP signal filtered for Î» between 578 and 649 nm. A third channel to detect the autofluorescence of chloroplast can be added to the second track and filtered for Î» between 680 and 750 nm. Additionally, the Transmitted Light detection can be added to one of the two tracks.</li><li>Set the Pinhole at 1 Airy Unity for the TagRFP detection, use bidirectional scan function and set the pixel dwell to a value around 2 Î¼s.</li><li>Parameters for image acquisition such as laser intensity, gain and offset have to be adapted to the individual imaging conditions. Among others, they depend on the laser status, as well as on the reporter line used, and thus the level of protein expression. In general, a compromise between sample viability and high-resolution imaging has to be found to obtain the best image quality while maintaining the sample in good conditions. In our case, we set the intensity of the Argon laser between 1% and 4.5%, while the DPSS 561-10 laser was set between 0.3% and 1%. The detector gain to collect GFP and TaqRFP signals was set between 700 and 850, and was set between 650 and 750 for the detection of chloroplast autofluorescence. In all the cases, the offset parameter was 0.</li><li>Perform averaging on 2 lines in case of the KINGBIRD line; for other reporters, different setting may be needed.</li><li>Acquire time lapse of multiple positions with 10 min interval time. At each position, perform a Z-stack of 6 to 7 planes with a step size to a maximum of 50 Î¼m per time point. To follow the entire meiotic division, images should be acquired for <em>ca.</em> 30 h (Prusicki <em>et al.</em>, 2019).</li></ol></li><li>The use of the Autofocus function at each position and each time point is suggested, especially for long acquisition times.</li><li>In case the sample is still going through meiosis but the data acquisition needs to be shortly interrupted (<em>e.g.</em>, in case of readjustment of the focal plane or in case of reaching the last acquisition cycle) it is advisable to fit the time of interruption to match the interval time, so to start the new time-lapse acquisition at the end of the 10 min interval.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Image processing<ol class="npwtwo"><li>Save each position of the acquired time-lapse as a separate file, for instance in the case of the ZEN black software: processing â copyâ subset.</li><li>Open the image in Fiji and save it in a separate folder as image sequence (save as â image sequence â format TIFF â digits must be enough to cover all time points <em>e.g.</em>, 110 time points need 3 digits).</li><li>Open the image sequence using MetaMorph (App â review multiple image sequence file â open sequence â follow the instructions), and select for each time point the z-stack layer, which shows the cells of interest. Finally, load the selected z-stacks for each wavelength and save them as tiff. files. This step helps to correct for the drift of the sample in the z-direction, and together with the autofocus function activated during image acquisition (see Procedure D6) allows following the same cells for the complete duration of the time lapse.</li><li>Open the new files in Fiji, combine the different channels, adjust their brightness and contrast and generate an RGB version of the file.</li><li>If an x-y drift is detectable, it can be corrected by using the plug-in Stackreg from Fiji (Plugins â Stackreg â Rigid body) (ThÃ©venaz <em>et al.</em>, 1998).</li><li>An example of a time-lapse series after image processing is given in Video 5.<br/><br/><!--video3611-5å¼å§--><div class="videodiv" id="vd3611-5" style="height:480px;"></div><div class="videophoto" id="vp3611-5"><a class="newapic" href="javascript:;" onclick="openvideo(5)" style="background:url('https://en-cdn.bio-protocol.org/video/3611/3611-5s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 5. Example of the outcome of a time lapse experiment after image processing as explained in Steps E1-E6</strong> </div></li></ol></li></ol><p class="pptt" id="biaoti32953">### Data analysis</p><p class="ppzz"><strong>Landmark identification using the KINGBIRD line</strong> </p><ol class="npwone"><li>Data set annotation<ol class="npwtwo"><li>Give an ID number to each visible cell within one anther. The ID number could be impressed on each frame of the RGB.tiff file using the Text tool of Fiji (Figure 2).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200427/20200427014254_1517.jpg" width="277"/><br/><strong>Figure 2. Data analysis.</strong> A. One frame of an acquired time-lapse movie depicting an anther of an <em>Arabidopsis</em> plant carrying the KINGBIRD reporter (microtubules are in magenta, REC8-chromatin is in green and chloroplasts are blue). Each meiocyte is given a number, which will be its ID on the spreadsheet in B. B. Example of a spreadsheet used for analysis as described in the paragraph Data-analysis A2. The numbering of the stage refers to the numbers in Table 1. Each stage is color-coded to facilitate the data organization.<br/><br/></li><li>Prepare a table using Microsoft Excel that includes 7 columns: frame number, time in min, cell shape, microtubule array, nucleus position, nucleolus position and chromatin state (Figure 2). The number of rows will be equal to the number of frames acquired for a single cell. The first imaged frame will be counted as Time 0 (Figure 2).</li><li>Fill in each column with a number that corresponds to the state of the analyzed cellular parameter as annotated in Table 1 and Figure 2 (for a further reference see Prusicki <em>et al.</em>, 2019). For instance, if at time 0 the cell presents a round shape, the number 4 must be filled in in the third column. Proceed with the annotation of each cellular parameter at each time point. If the cell is not visible at one time point, or the parameter state cannot be recognized, fill in the table an n for non-visible.</li><li>Annotate all the visible cells of one another on different Excel sheets (one sheet per cell). This file constitutes the data set that will be analyzed as explained in the following session.<br/><br/><strong>Table 1. Cell features of meiocytes as obtained with the KINGBIRD reporter system.</strong> The table provides microscopy images that illustrate each described cell feature (adapted from Figures 3 and 4 in Prusicki <em>et al.,</em> 2019. <a href="http://creativecommons.org/licenses/by/4.0/" target="_blank">Creative Commons Attribution License</a>). The original table file could be accessed here (<a href="https://os.bio-protocol.org/attached/file/20200322/table1 bioprotocol.xlsx" target="_blank">Table 1 Excel file</a>).<br/><img class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200427/20200427014332_0109.jpg" width="1000"/><br/><br/></li></ol></li><li>Data set analysis<br/>A Python script to perform analysis of the annotated data set created in part A. is available at <a href="https://gitlab.com/wurssb/arabidopsis-thaliana---landmark-analysis" target="_blank">https://gitlab.com/wurssb/arabidopsis-thaliana---landmark-analysis</a>. This script contains the procedure to extract landmarks from the data and takes the following steps:<ol class="npwtwo"><li>Prepare the data for analysis by filtering out invalid or duplicate entries. Each entry, consisting of the observed state for each cellular parameter should have an identifier for the sample and the specific cell within the sample.</li><li>If the data is not sampled at regular intervals, resample to a common interval. However, take care when resampling, as going from short to large intervals might remove short-lived states from the data set while the reverse could introduce (large) errors from interpolating the states.</li><li>A cellular state is defined by the ensemble of all parameter states. For example, cell 11 from Figure 2 at time point 10 has the cellular state: 1-1-2-2-2. Cellular states diverging for up to two different parameter states were defined as neighboring states (or neighbors). Hence, the cellular states 1-1-2-2-1, 1-1-2-3-2 and 1-1-2-3-3 but not 1-1-3-3-3 are neighboring states to 1-1-2-2-2.</li><li>Partially observed states can be either discarded, or interpolated from neighboring states. Again, take care when interpolating, for example, by setting a maximum time interval that can be safely interpolated depending on the observed processes.</li><li>Calculate how much time is spent in each cellular state.</li><li>For each observed cellular state, list the neighboring states.</li><li>For each cellular state, calculate the neighboring score using the time spent in the specific state (<em>t<sub>state</sub></em>) and the time spent its neighbors (<em>t<sub>neighbors</sub></em>):<br/><br/><img alt="" class="layerphoto" height="60" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200427/20200427014739_3924.jpg"/><br/><br/></li><li>Sort the cellular states by their score. Landmarks can now be assigned with a suitable cut-off value (an example in Prusicki <em>et al.</em>, 2019). An example of the final result of the neighboring analysis is given in Table 2 (<a href="https://os.bio-protocol.org/attached/file/20200322/table2 example of analysis.xlsx" target="_blank">Table 2 Excel file</a>), where all the cellular states observed are listed together with their counts (number of cases they have been observed), their percentage of appearance in relation to the other cellular states and finally their neighboring score. The source of the data listed in Table 2 are the results published in Prusicki <em>et al.</em>, 2019.</li><li>To assess the robustness of the analysis, bootstrap the analysis by resampling the data with replacement and re-running the whole analysis multiple times.<br/>Bootstrapping can be done on the level of single observations, or by resampling groups of observations of specific cells or anthers to uncover different sources of possible bias in the analysis.<br/><br/><strong>Table 2. Example of an outcome of the data analysis.</strong> The original table file could be accessed here (<a href="https://os.bio-protocol.org/attached/file/20200322/table2 example of analysis.xlsx" target="_blank">Table 2 Excel file</a>).<br/><img alt="" class="layerphoto" height="1396" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200427/20200427015732_0644.jpg" width="600"/><br/></li></ol></li></ol><p class="pptt" id="biaoti32954">### Recipes</p><ol class="npwtwok"><li>Murashike and Skoog medium (MS medium) for plant germination<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:450px;"><tbody><tr><td><strong>Ingredient</strong><br/></td><td><strong>Concentration</strong><br/></td><td><strong>For 1 L</strong><br/></td></tr><tr><td>MS basal medium<br/></td><td>0.5x<br/></td><td>2.2 g/L<br/></td></tr><tr><td>Sucrose<br/></td><td>1%<br/></td><td>10 g/L<br/></td></tr><tr><td>Agar<br/></td><td>1%<br/></td><td>10 g<br/></td></tr><tr><td>dH<sub>2</sub>O<br/></td><td>---<br/></td><td>up to 1 L of volume<br/></td></tr><tr><td>KOH<br/></td><td>---<br/></td><td>---<br/></td></tr></tbody></table><ol class="npwthree"><li>Dissolve MS basal and sucrose in 800 ml of dH<sub>2</sub>O</li><li>Adjust the pH to 5.8 using KOH and bring to the final volume of 1 L</li><li>Add the agar to the solution</li><li>Sterilize by autoclaving and store at room temperature until usage</li></ol></li><li><em>Arabidopsis</em> Apex Culture Medium (ACM) for imaging<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td><br/></td><td><strong>Ingredient</strong><br/></td><td><strong>Concentration</strong><br/></td><td><strong>Quantity</strong><br/></td></tr><tr><td>MS-Medium base<br/></td><td>MS basal medium<br/></td><td>0.5x<br/></td><td>2.2 g/L<br/></td></tr><tr><td><br/></td><td>Sucrose<br/></td><td>1%<br/></td><td>10 g/L<br/></td></tr><tr><td><br/></td><td>Agarose<br/></td><td>0.8%-1%-2%<br/></td><td>8 g-10 g-20 g<br/></td></tr><tr><td><br/></td><td>dH<sub>2</sub>O<br/></td><td>---<br/></td><td>up to 1 L of volume<br/></td></tr><tr><td><br/></td><td>KOH<br/></td><td>---<br/></td><td>---<br/></td></tr><tr><td>Vitamins (1,000x)<br/></td><td>Myo-Inositol<br/></td><td>10%<br/></td><td>5 g/50 ml<br/></td></tr><tr><td><br/></td><td>Nicotinic acid<br/></td><td>0.1%<br/></td><td>0.05 g/50 ml<br/></td></tr><tr><td><br/></td><td>Pyridoxin hydrochloride<br/></td><td>0.1%<br/></td><td>0.05 g/50 ml<br/></td></tr><tr><td><br/></td><td>Thiamine hydrochloride<br/></td><td>1%<br/></td><td>0.5 g/50 ml<br/></td></tr><tr><td><br/></td><td>Glycine<br/></td><td>0.2%<br/></td><td>0.1 g/50 ml<br/></td></tr><tr><td><br/></td><td>dH<sub>2</sub>O<br/></td><td>---<br/></td><td>50 ml<br/></td></tr></tbody></table><ol class="npwthree"><li>Prepare the MS base as described in the previous recipe, being careful to add agarose to the three different concentrations (0.8%, 1% and 2%) instead of agar and store it at room temperature</li><li>Dissolve the vitamins in dH<sub>2</sub>O to obtain the 1,000x stock solution</li><li>Sterilize the vitamin stock using a 0.22 Î¼m filter, aliquot them and store at -20 Â°C</li></ol></li></ol><p class="pptt" id="biaoti32955">### Acknowledgments</p><p class="ppzz">This work was supported by the European Union Marie-Curie âCOMRECâ network FP7 ITN-606956 to M.A.P. and A.S. In addition, core funding of the University of Hamburg to A.S. is gratefully acknowledged. The here-presented protocol was presented in Prusicki <em>et al.</em> (2019) and further described in Prusicki <em>et al.</em> (2020).</p><p class="pptt" id="biaoti32956">### Competing interests</p><p class="ppzz">Arp Schnittger on behalf of all authors confirms that none of the authors has competing financial interests in this work.</p><p class="pptt" id="biaoti32957">### References</p><ol class="npwtwok"><li>Armstrong, S. J., Sanchez-Moran, E. and Franklin, F. C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19685323" target="_blank">Cytological analysis of <em>Arabidopsis thaliana</em> meiotic chromosomes.</a> <em>Methods Mol Biol</em> 558: 131-145.</li><li>Chelysheva, L. A., Grandont, L. and Grelon, M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23559205" target="_blank">Immunolocalization of meiotic proteins in Brassicaceae: method 1.</a> <em>Methods Mol Biol</em> 990: 93-101.</li><li>Cromer, L., Jolivet, S., Singh, D. K., Berthier, F., De Winne, N., De Jaeger, G., Komaki, S., Prusicki, M. A., Schnittger, A., Guerois, R. and Mercier, R. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31324745" target="_blank">Patronus is the elusive plant securin, preventing chromosome separation by antagonizing separase.</a> <em>Proc Natl Acad Sci U S A</em> 116(32): 16018-16027.</li><li>Darrier, B., Arrieta, M., Mittmann, S. U., Sourdille, P., Ramsay, L., Waugh, R. and Colas, I. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31583662" target="_blank">Following the formation of synaptonemal complex formation in wheat and barley by high-resolution microscopy.</a> <em>Methods Mol Biol</em> 2061: 207-215.</li><li>Hurel, A., Phillips, D., Vrielynck, N., MÃ©zard, C., Grelon, M. and Christophorou, N. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29681056" target="_blank">A cytological approach to studying meiotic recombination and chromosome dynamics in <em>Arabidopsis thaliana</em> male meiocytes in three dimensions.</a> <em>Plant J</em> 95(2): 385-396.</li><li>Lambing, C. and Heckmann, S. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29971082" target="_blank">Tackling plant meiosis: from model research to crop improvement.</a> <em>Front Plant Sci</em> 9: 829.</li><li>Martinez-Garcia, M. and Pradillo, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28608340" target="_blank">Functional Analysis of <em>Arabidopsis</em> ARGONAUTEs in Meiosis and DNA Repair.</a> <em>Methods Mol Biol</em> 1640: 145-158.</li><li>Melamed-Bessudo, C., Shilo, S. and Levy, A. A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26939088" target="_blank">Meiotic recombination and genome evolution in plants.</a> <em>Curr Opin Plant Biol</em> 30: 82-87.</li><li>Mercier, R., Mezard, C., Jenczewski, E., Macaisne, N. and Grelon, M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25494464" target="_blank">The molecular biology of meiosis in plants.</a> <em>Annu Rev Plant Biol</em> 66: 297-327.</li><li>Nannas, N. J., Higgins, D. M. and Dawe, R. K. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27609836" target="_blank">Anaphase asymmetry and dynamic repositioning of the division plane during maize meiosis.</a> <em>J Cell Sci</em> 129(21): 4014-4024.</li><li>Parra-Nunez, P., Pradillo, M. and Santos, J. L. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31583650" target="_blank">How to perform an accurate analysis of metaphase I chromosome configurations in autopolyploids of <em>Arabidopsis thaliana</em>.</a> <em>Methods Mol Biol</em> 2061: 25-36.</li><li>PelÃ©, A., Rousseau-Gueutin, M. and Chevre, A. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30002669" target="_blank">Speciation success of polyploid plants closely relates to the regulation of meiotic recombination.</a> <em>Front Plant Sci</em> 9: 907.</li><li>Prusicki, M. A., Keizer, E. M., van Rosmalen, R. P., Komaki, S., Seifert, F., Muller, K., Wijnker, E., Fleck, C. and Schnittger, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31107238" target="_blank">Live cell imaging of meiosis in <em>Arabidopsis thaliana</em>.</a> <em>Elife</em> 8: 42834 .</li><li>Prusicki, M. A., Hamamura, Y. and Schnittger, A. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31583648" target="_blank">A practical guide to live-cell imaging of meiosis in <em>Arabidopsis</em>.</a> <em>Methods Mol Biol</em> 2061: 3-12. </li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682.</li><li>Sepsi, A., Fabian, A., Jager, K., Heslop-Harrison, J. S. and Schwarzacher, T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30154816" target="_blank">Immunofish: simultaneous visualisation of proteins and DNA sequences gives insight into meiotic processes in nuclei of grasses.</a> <em>Front Plant Sci</em> 9: 1193.</li><li>Sheehan, M. J. and Pawlowski, W. P. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19926853" target="_blank">Live imaging of rapid chromosome movements in meiotic prophase I in maize.</a> <em>Proc Natl Acad Sci U S A</em> 106(49): 20989-20994.</li><li>Sims, J., Chouaref, J. and Schlogelhofer, P. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31583653" target="_blank">Whole-mount immuno-fish on <em>Arabidopsis</em> meiocytes (whomi-fish).</a> <em>Methods Mol Biol</em> 2061: 59-66.</li><li>Smyth, D. R., Bowman, J. L. and Meyerowitz, E. M. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2152125" target="_blank">Early flower development in <em>Arabidopsis</em>.</a> <em>Plant Cell</em> 2(8): 755-767.</li><li>Sofroni K., Takatsuka H., Yang, C., Dissmeyer N., Komaki, S., Hamamura, Y., BÃ¶ttger, L., Umeda M. and Schnittger A. CDKD-dependent activation of CKDA;1 controls microtubule dynamics and cytokinesis during meiosis. (Unpublished).</li><li>Stack, S. M., Shearer, L. A., Lohmiller, L. D. and Anderson, L. K. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31583655" target="_blank">Preparing maize synaptonemal complex spreads and sequential immunofluorescence and fluorescence <em>in situ</em> hybridization.</a> <em>Methods Mol Biol</em> 2061: 79-115.</li><li>ThÃ©venaz, P., Ruttimann, U. E. and Unser, M. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18267377" target="_blank">A pyramid approach to subpixel registration based on intensity.</a> <em>IEEE Trans Image Process</em> 7(1): 27-41.</li><li>Valuchova, S., Mikulkova, P., Pecinkova, J., Klimova, J., Krumnikl, M., Bainar, P., Heckmann, S., Tomancak, P. and Riha, K. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32041682" target="_blank">Imaging plant germline differentiation within <em>Arabidopsis</em> flowers by light sheet microscopy.</a> <em>Elife</em> 9: e52546.</li><li>Wang, Y. and Copenhaver, G. P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29489392" target="_blank">Meiotic recombination: mixing it up in plants.</a> <em>Annu Rev Plant Biol</em> 69: 577-609.</li><li>Wijnker, E. and Schnittger, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23852379" target="_blank">Control of the meiotic cell division program in plants.</a> <em>Plant Reprod</em> 26(3): 143-158.</li><li>Yang, C., Hamamura, Y., Sofroni, K., BÃ¶wer, F., Stolze, S. C., Nakagami, H. and Schnittger, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30988453" target="_blank">SWITCH 1/DYAD is a WINGS APART-LIKE antagonist that maintains sister chromatid cohesion in meiosis.</a> <em>Nat Commun</em> 10(1): 1755.</li><li>Yang, C., Sofroni, K., Wijnker, E., Hamamura, Y., Carstens, L., Harashima, H., Stolze, S. C., Vezon, D., Chelysheva, L., Orban-Nemeth, Z., Pochon, G., Nakagami, H., Schlogelhofer, P., Grelon, M. and Schnittger, A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31556459" target="_blank">The <em>Arabidopsis</em> Cdk1/Cdk2 homolog CDKA;1 controls chromosome axis assembly during plant meiosis.</a> <em>EMBO J</em>: e101625.</li><li>Yu, H. G., Hiatt, E. N., Chan, A., Sweeney, M. and Dawe, R. K. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9362502" target="_blank">Neocentromere-mediated chromosome movement in maize.</a> <em>J Cell Biol</em> 139(4): 831-840.<br/></li></ol>