<body>
            <h1>Rapid Generation of Human Neuronal Cell Models Enabling Inducible Expression of Proteins-of-interest for Functional Studies</h1>
            <p><strong>Authors:</strong> Xinzhu Wang, Erik Friesen, Iris MÃ¼ller, Mackenzie Lemieux, Ramona Dukart, Isabella BL Maia, Suneil Kalia, Gerold   Schmitt-Ulms</p>
            <h2>Abstract</h2>
            <p>CRISPR-Cas9 technology has transformed the ability to edit genomic sequences and control gene expression with unprecedented ease and scale. However, precise genomic insertions of coding sequences using this technology remain time-consuming and inefficient because they require introducing adjacent single-strand cuts through Cas9 nickase action and invoking the host-encoded homology-directed repair program through the concomitant introduction of large repair templates. Here, we present a system for the rapid study of any protein-of-interest in two neuronal cell models following its inducible expression from the human <em>AAVS1</em> safe harbor locus. With lox-flanked foundation cassettes in the <em>AAVS1</em> site and a tailor-made plasmid for accepting coding sequences-of-interest in place, the system allows investigators to produce their own neuronal cell models for the inducible expression of any coding sequence in less than a month. Due to the availability of preinserted enhanced green fluorescent protein (EGFP) coding sequences that can be fused to the protein-of-interest, the system facilitates functional investigations that track a protein-of-interest by live-cell microscopy as well as interactome analyses that capitalize on the availability of exquisitely efficient EGFP capture matrices.</p>
            <p><strong>Keywords:</strong> Human neuronal cells, CRISPR-Cas9, <em>AAVS1</em>, Inducible expression, Protein-of-interest, EGFP</p>
            <h2>Background</h2><p>The ability to engineer the genomes of cell models and organisms holds tremendous potential for research and targeted therapy. With the advent of CRISPR-Cas9 technology, the efficiency of precision genome engineering has greatly improved, and useful reagents are coming online at a rapid pace. Despite these advances, there still is a scarcity of cell-based <em>in vitro</em> paradigms for studying neurological diseases that is holding back the pace of progress in this area. Whereas CRISPR-Cas9 mediated homology-directed repair (HDR) can now easily be applied to generate knockout models or to knock in small transgenic segments, the goal to integrate larger segments continues to present a considerable challenge (Devkota, 2018).<br/> <br/>In our own work, we often would like to learn more about the function of a protein by studying its subcellular localization and protein-protein interactions. To this end, it would help if neuronal cell models could be equipped with inducible expression systems that provide temporal control of the expression of a protein-of-interest and facilitate its visualization and capture. Not long ago, this need would call for the stable transfection or lentiviral transduction of an expression cassette, an approach that can result in unpredictable confounding effects due to the random integration of transgenes. CRISPR-Cas9 provides alternative strategies to accomplish this task in a precise way but these approaches remain time-consuming and require considerable investments in resources and expertise.<br/> <br/>To address this unmet need, we sought to develop resources that enable the rapid and flexible integration of large inducible expression cassettes into the <em>AAVS1</em> human safe harbor locus (DeKelver <em>et al.</em>, 2010) of several neuronal cell models. Stable insertion into the <em>AAVS1</em> locus, as opposed to viral integration, was chosen to preclude transgene silencing and insertional mutagenesis, notorious confounders associated with non-directed transgene insertions. To maximize flexibility and speed, the system was built around two genome-editing steps. The first step, which makes use of CRISPR-Cas9-mediated HDR followed by selection of positive clones, is time-consuming and uncertain to succeed but we have already accomplished this in 3 cell lines that are available to be shared. The second step, which relies on a Cre recombinase-mediated exchange of an expression cassette, is relatively easy to accomplish and fast because it can be followed by puromycin-based selection of positive clones.<br/> <br/>More specifically, a paired Cas9 nickase CRISPR strategy, used to improve the specificity of targeting and to minimize off-target effects (Ran <em>et al.</em>, 2013), was employed to insert a foundation cassette (FC), comprising a pair of lox sites flanking a G418 resistance marker into Intron 1 of the <em>AAVS1</em> locus, a human genetic safe harbour. In parallel, a ~7 kb inducible expression plasmid (IEX) was built. The expression cassette within the IEX is flanked by compatible lox sites and comprises elements designed to accept the insertion and control the expression of any coding sequence-of-interest (Figure 1). More specifically, the IEX comprises a promoter-less puromycin resistance marker, the reverse transactivator (rtTA<sub>3</sub>), and an EGFP tag fused to a protein-of-interest insertion site. With this arrangement, the rtTA3 is designed to drive the inducible expression of the EGFP-fused protein-of-interest that is under the control of the TREtight inducible promoter.<br/> <br/>With these tools in hand, novel cell models for the inducible expression of a protein-of-interest fused to EGFP can be generated in three straightforward steps:<br/>1.Insert coding sequence for protein-of-interest into the IEX plasmid.<br/>2.Transiently co-transfect the IEX plasmid and a plasmid coding for Cre recombinase into the parental cell equipped with one or two FCs.<br/>3.Select stable integrant clones by puromycin selection or fluorescence activated cell sorting (FACS).<br/> <br/>In addition to its ease of handling and speed, the system restricts the copy number of insertions to two, corresponding to the two <em>AAVS1</em> alleles present in the human genome. The system also lends itself to comparative analyses of several proteins, because its design ensures that cells coding for different proteins-of-interest can be generated side-by-side, requiring no specific resources, except for a human cDNA library and the corresponding primers to amplify and insert a given coding sequence into the IEX cassette. Note that fusing the coding sequence for a protein-of-interest to EGFP does not only provide the ability to undertake live cell imaging but the EGFP tag also allows the high affinity purification of fusion proteins by GFP nanotrap technology for the study of protein-protein interactions (Rothbauer <em>et al.</em>, 2008).<br/> <br/>We recently applied the system to study the TAU protein and BAG5 in several human neuronal cell models, including ReN VM human neural progenitor cells that can be differentiated into a co-culture of neural and glial cells (Donato <em>et al.</em>, 2007). These studies have led us to detect subtle differences in protein-protein interactions of wild-type versus mutant forms of TAU and BAG5 (Figure 1). This article was written with the intent to share these resources and facilitate their adaptation to the study of other proteins-of-interest.</p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200428/20200428003612_9160.jpg" width="100"/><br/><strong>Figure 1. Design of <em>AAVS1</em> embedded system for inducible expression of proteins-of-interest.</strong> A. CRISPR-Cas9 nickase-mediated insertion of foundation cassette (FC) into <em>AAVS1</em> safe harbour. B. Insertion of coding sequence-of-interest into inducible expression plasmid (IEX). C. Reverse-tetracycline-controlled transactivator (rtTA<sub>3</sub>)-mediated expression of protein-of-interest following addition of doxycycline to cell culture medium. D. Example 1: Parallel induction of wild-type and P301L mutant TAU fused to C-terminal EGFP in ReN cells. Antibodies directed against beta-actin served as the loading control. E. Example 2: Side-by-side induction of N-terminally EGFP-tagged wild-type or mutant (DARA) BAG5. Cells expressing EGFP only served as additional controls in this analysis.</p><p class="pptt" id="biaoti33115">## Materials and Reagents</p><ol class="npwone"><li>Biological materials<ol class="npwtwo"><li>Plasmids (Table 1)</li><li>Cell lines (Table 2)<br/><br/><strong>Table 1. List of available plasmids for generating inducible cells and their Addgene ID</strong><br/><img alt="" class="layerphoto" height="201" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200428/20200428004112_7875.jpg" width="520"/><br/><br/><strong>Table 2. List of available cell lines</strong><br/><img alt="" class="layerphoto" height="241" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200428/20200428004140_7252.jpg" width="560"/><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cloning<br/><em>Note: All cloning enzymes and PCR reagents are stored at -20 Â°C, stable for 1 year.</em> <ol class="npwtwo"><li>Q5<sup>Â®</sup> Hot Start high-fidelity 2x master mix (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0494S </a> )</li><li>Primers: ordered as Value Oligos or Custom Oligos, 25 nmole, desalted (Invitrogen). Reconstitute to 100 Î¼M stocks in water, store at -20 Â°C</li><li>Q5<sup>Â®</sup> Site-directed mutagenesis kit (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> E0554S </a> )</li><li>EZ-10 spin column PCR products purification kit (BioBasic, catalog number: <a class="blackNoUnder" target="_blank"> BS664-250preps </a> )</li><li>Restriction enzyme BbvCI with CutSmart buffer (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0601S </a> )</li><li>Restriction enzyme NheI-HF with CutSmart buffer (New England Biolabs, CutSmart, catalog number: <a class="blackNoUnder" target="_blank"> R3131S </a> )</li><li>Fast AP thermosensitive alkaline phosphatase (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> EF0651 </a> )</li><li>PureLink Quick Gel Extraction Kit (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> K210012 </a> )</li><li>T4 DNA ligase (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0202S </a> )</li><li>Endura<sup>TM</sup> Chemically Competent <em>E. coli </em>Cells (Lucigen, catalog number: <a class="blackNoUnder" target="_blank"> 60241-2 </a> ). Store at <br/>-80 Â°C</li><li>EZ-10 spin column plasmid DNA minipreps kit (BioBasic, catalog number: <a class="blackNoUnder" target="_blank"> BS614-250preps </a> )</li><li>PureLink<sup>Â®</sup> HiPure plasmid filter maxi kit (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> K210026 </a> )</li><li>Ampicillin sodium salt powder, 91.0-100.5%, cell culture tested (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A1066-5G </a> ). Store at 4 Â°C, stable for 1 year<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Inducible cell generation<ol class="npwtwo"><li>paavCAG-iCre plasmid (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 51904 </a> , a gift from Jinhyn Kim)</li><li>jetPRIME (PolyPlus, catalog number: <a class="blackNoUnder" target="_blank"> 114-07 </a> ). Store at 4 Â°C, stable for 1 year</li><li>TransfeX transfection reagent (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> ACS-4005 </a> ). Store at 4 Â°C, stable for 1 year</li><li>Millex-HA syringe filters, 0.45 Î¼m pore size (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 5010-SLHA033SS </a> )</li><li>Millex-GP syringe filters, 0.22 Î¼m pore size (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> SLGP033RS </a> )</li><li>6-well tissue-culture plates (Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 353046 </a> )</li><li>24-well tissue-culture plates (Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 353047 </a> )</li><li>Puromycin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P7255-25MG </a> ). Reconstitute to 10 mg/ml stock, filter sterilize, aliquot, store at -20 Â°C</li><li>Doxycycline (BioBasic, catalog number: <a class="blackNoUnder" target="_blank"> DB0889-25G </a> ). Reconstitute to 10 mg/ml stock, aliquot, store at -20 Â°C<br/><br/></li></ol></li><li>Affinity capture with GFP nanotrap<ol class="npwtwo"><li>15-cm tissue-culture plates (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 83.3903 </a> )</li><li>Agarose-conjugated GFP nanotrap (Chromotek, catalog number: <a class="blackNoUnder" target="_blank"> gta-10 </a> ). Store at 4 Â°C, stable for 1 year<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell culture<ol class="npwtwo"><li>6-cm tissue-culture plates (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 83.3901 </a> )</li><li>1.5 ml centrifuge tubes (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.690.300 </a> )</li><li>12-well tissue-culture plates (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 83.3921 </a> )</li><li>Dulbecco's Phosphate-Buffered Saline (D-PBS), without calcium chloride and magnesium chloride, liquidPhosphate buffered saline (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D8537-500ML </a> ). Store at 4 Â°C, stable for 1 year5.</li><li>0.25% Trypsin with EDTA 4Na (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 25200072 </a> ). Aliquot, store at -20 Â°C for long-term storage. Store at 4 Â°C for short-term use, stable for 6 months</li><li>StemPro<sup>Â®</sup> Accutase<sup>Â®</sup> Cell Dissociation Reagent (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> A1110501 </a> ). Aliquot, store at -20 Â°C for long-term storage. Store at 4 Â°C for short-term use, stable for 6 months</li><li>Matrigel, growth-factor reduced (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 354230 </a> ). Aliquot, store at -20 Â°C, stable for 1 year</li><li>Recovery<sup>TM</sup> Cell Culture Freezing Medium (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 12648010 </a> ). Aliquot, store at -20 Â°C, stable for 1 year</li><li>DMEM with high glucose, L-glutamine, and sodium pyruvate (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 11995073 </a> ). Store at 4 Â°C, stable for 1 year</li><li>10% Fetal Bovine Serum, qualified (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 12483020 </a> ). Aliquot, store at -20 Â°C for long-term storage</li><li>1% GlutaMAX<sup>TM</sup>-I supplement (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 35050061 </a> ). Store at 4 Â°C, for stable for 1 year</li><li>DMEM/F12, 1:1, contains L-glutamine, but no HEPES (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 21041025 </a> ). Store at 4 Â°C, stable for 1 year</li><li>2% N21-MAX supplement (R&amp;D Systems, catalog number <a class="blackNoUnder" target="_blank"> AR008 </a> ) or 2% B-27 serum-free supplement (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 17504044 </a> ). Store at -20 Â°C, stable for 1 year</li><li>Basic fibroblast growth factor (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> PHG0261 </a> ). Store at -80 Â°C, stable for 1 year</li><li>Epidermal growth factor (Reprokine, catalog number: <a class="blackNoUnder" target="_blank"> RKP01133 </a> ). Store at -80 Â°C, stable for 1 year</li><li>Heparin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> H3149-10KU </a> ). Store at -80 Â°C, stable for 1 year</li><li>2 ng/ml glial-derived neurotrophic factor (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> PHC7045 </a> )</li><li>500 Î¼M dibutyryl-cyclic-adenosine monophosphate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D0627-250MG </a> )</li><li>Proliferation media for IMR-32 and HEK-293 cells (see Recipes)</li><li>Proliferation media for ReN VM cells (see Recipes)</li><li>Neuronal differentiation media for ReN VM cells (see Recipes)<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell lysis<br/><em>Note: Original reagents are stored at room temperature. Reconstituted stocks are stored at 4 Â°C for 3 months. All buffers (see Recipes) are prepared fresh from reagent stocks immediately before use.</em> <ol class="npwtwo"><li>Tris-HCl, &gt; 99% (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> TRS002.5 </a> ). Reconstitute to 1 M stock in water</li><li>NP-40 (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> NON505.500 </a> ). Reconstitute to 10% stock in water</li><li>Sodium deoxycholate (DOC), &gt; 99% (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> DCA333.50 </a> ). Reconstitute to 10% w/v stock in water, pH 8.0</li><li>Sodium chloride (NaCl), &gt; 99% (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> SOD002.205 </a> ). Reconstitute to 5 M stock in water</li><li>Ethylenediaminetetraacetate (EDTA), &gt;99% (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> EDT001.500 </a> ). Reconstitute to 0.5 M stock in water</li><li>Sodium orthovanadate (Na<sub>3</sub>VO<sub>4</sub>) (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> SOV664.25 </a> ). Reconstitute to 0.1 M stock in water</li><li>Sodium fluoride (NaF) (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> SFL001.100 </a> ). Reconstitute to 1 M stock in water</li><li>Phenylmethylsulfonyl fluoride (PMSF) (BioShop, catalog number: <a class="blackNoUnder" target="_blank"> PMS123.5 </a> ). Reconstitute to 0.5 M stock in ethanol</li><li>cOmplete protease inhibitor (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11836170001 </a> )</li><li>PhosStop phosphatase inhibitor (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 4906837001 </a> )</li><li>HEPES (Bioshop, catalog number: <a class="blackNoUnder" target="_blank"> HEP001.500 </a> ). Reconstitute to 0.1 M stock in water</li><li>Trifluoroacetic acid (TFA), &gt; 99%, HPLC grade (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 302031-100ML </a> )</li><li>Acetonitrile (ACN), HPLC grade (Caledon, catalog number: <a class="blackNoUnder" target="_blank"> 1401-7-40 </a> )</li><li>Lysis buffer (see Recipes)</li><li>Wash buffer (see Recipes)</li><li>Elution buffer (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti33116">### Equipment</p><ol class="npwtwok"><li>-80 Â°C freezer (Thermo Fisher, model: <a class="blackNoUnder" target="_blank"> Standard Performance Ultra-Low Freezers </a> )</li><li>Refrigerated micro centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5424R </a> , catalog number: <a class="blackNoUnder" target="_blank"> 5404000138 </a> )</li><li>Avanti J-26S series high-speed centrifuge (Beckman-Coulter, catalog number: <a class="blackNoUnder" target="_blank"> B22984 </a> )</li><li>Orbital shaker (Jeio Tech, model: <a class="blackNoUnder" target="_blank"> ISF-7000 series </a> )</li><li>Fluorescence microscope (Leica Microsystems, model: <a class="blackNoUnder" target="_blank"> Leica DMI 6000 B </a> )</li><li>NanoDrop spectrophotometer (Thermo Fischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ND-1000 </a> )</li><li>ESBE Scientific Save cell UV cell culture incubator (Panasonic, catalog number: <a class="blackNoUnder" target="_blank"> MCO-19AIC </a> )</li><li>SterilGARD biosafety cabinet (The Baker Company, catalog number: <a class="blackNoUnder" target="_blank"> SG603A-HE </a> )<br/></li></ol><p class="pptt" id="biaoti33117">### Software</p><ol class="npwtwok"><li>Snapgene (GSL Biotech LLC, <a href="https://www.snapgene.com/" target="_blank">https://www.snapgene.com/</a>)</li><li>Snapgene Viewer (GSL Biotech LLC, <a href="https://www.snapgene.com/snapgene-viewer/" target="_blank">https://www.snapgene.com/snapgene-viewer/</a>)</li><li>Tm Calculator (New England Biolabs, <a href="https://tmcalculator.neb.com/#!/main" target="_blank">https://tmcalculator.neb.com/#!/main</a>)</li><li>NEBioCalculator (New England Biolabs, https://nebiocalculator.neb.com/#!/ligation)</li><li>NanoDrop 1000 (Thermo Fischer Scientific, <a href="https://www.thermofisher.com/ca/en/home/industrial/spectroscopy-elemental-isotope-analysis/molecular-spectroscopy/ultraviolet-visible-visible-spectrophotometry-uv-vis-vis/uv-vis-vis-instruments/nanodrop-microvolume-spectrophotometers/nanodrop-software-download.html" target="_blank">https://www.thermofisher.com/ca/en/home/industrial/spectroscopy-elemental-isotope-analysis/molecular-spectroscopy/ultraviolet-visible-visible-spectrophotometry-uv-vis-vis/uv-vis-vis-instruments/nanodrop-microvolume-spectrophotometers/nanodrop-software-download.html</a>)<br/></li></ol><p class="pptt" id="biaoti33118">### Procedure</p><p class="ppzz">The protocol is designed to enable the generation of neural cell models for functional analyses of proteins-of-interest in less than a month (Figure 2). The system is scalable by parallelization of procedures.<br/><br/><img alt="" class="layerphoto" height="110" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200428/20200428004431_4592.jpg" width="100"/><br/><strong>Figure 2. Time-line for the execution of procedures described in this protocol.</strong> All steps described in the protocol take approximately one month to implement. The generation of vectors coding for the inducible expression of a protein-of-interest can be completed in approximately one week. Once positive clones have been selected and validated, cell-based and biochemical analyses can be undertaken in parallel. Naturally, the one-month timeline will be exceeded, if complex follow-on downstream analyses are planned.</p><br/><ol class="npwone"><li>Week 1. Cloning a gene of interest into the IEX<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Cloning strategies can be simulated, verified, and visualized using Snapgene and Snapgene Viewer.</em> </li><li><em>The coding sequence (CDS) of a new protein-of-interest can be cloned into the IEX using the unique restriction enzyme sites BbvCI and KpnI:</em><br/><em> BbvCI: located 5â of the start codon of new CDS.</em><br/><em> KpnI: present in the linker region connecting EGFP to the C-terminus of the new protein.</em> </li><li><em>Multiple other unique restriction enzyme sites (e.g., NheI, BstBI, KflI, etc.) may also be used.</em> </li></ol><br/><ol class="npwtwo"><li>Order primers to append BbvCI and KpnI restriction enzyme sites to the 5â and 3â ends of the CDS of the protein-of-interest, respectively.<br/>Example:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>Primer name<br/></td><td>Sequence<br/></td></tr><tr><td>Insert BbvCI site 5â²â¦CC<sup>V</sup> TCAGCâ¦3â²<br/></td><td>5â² N<sub>8</sub>CCTCAGC<span style="color:#337FE5;">GCCACC</span><u style="color:#337FE5;"><span style="color:#337FE5;">ATG</span></u>X<sub>20-25</sub> 3â²<br/></td></tr><tr><td>Insert KpnI site 5â²â¦GGTA<sup>V</sup> CCâ¦3â²<br/></td><td>5â² N<sub>8</sub>GGTACCX<sub>20-25</sub> 3â²<br/></td></tr></tbody></table><ol class="npwthree"><li>N<sub>8</sub> represent 8 random nucleotides to serve as overhangs required by restriction enzymes for efficient digestion.</li><li>X<sub>20-25</sub> represent sequences of the CDS of the protein-of-interest.</li><li>Blue: Kozak sequence, including the <u>ATG</u> start codon, required for translation.</li><li>Do not include a stop codon in the X<sub>20-25</sub> sequence used for designing the âInsert KpnIâ primer if the new protein is to be expressed as an EGFP fusion protein.</li></ol></li><li>Determine the annealing temperature of primers using the New England Biolabs T<sub>m</sub> Calculator tool, which returns this information when primer sequences are entered into the online form field at: <a href="https://tmcalculator.neb.com/#!/main" target="_blank">https://tmcalculator.neb.com/#!/main</a>.<br/><br/></li></ol><strong><em>Day 1. Cloning</em></strong> <ol class="npwtwo" start="3"><li>Perform PCR to append BbvCI and KpnI sites to CDS of protein-of-interest using the following PCR reaction:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:650px;"><tbody><tr><td>Reagents<br/></td><td>50 Î¼l reaction (Âµl)<br/></td><td>Final concentration<br/></td></tr><tr><td>Q5 HiFi Hot start master mix (2x)<br/></td><td>25<br/></td><td>1x<br/></td></tr><tr><td>Template DNA (20 ng)<br/></td><td>Calculate<br/></td><td><br/></td></tr><tr><td>âInsert BbvCIâ primer (10 Î¼M)<br/></td><td>2.5<br/></td><td>0.5 ÂµM<br/></td></tr><tr><td>âInsert KpnIâ primer (10 Î¼M)<br/></td><td>2.5<br/></td><td>0.5 ÂµM<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O<br/></td><td>Add to 50 <br/></td><td><br/></td></tr><tr><td>Total  <br/></td><td>50</td><td><br/></td></tr></tbody></table></li><li>Perform the following PCR cycles:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td><br/></td><td>Step<br/></td><td>Temperature (Â°C)<br/></td><td>Time<br/></td></tr><tr><td><br/></td><td>Initial denaturation<br/></td><td>98<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>Denaturation<br/></td><td>98<br/></td><td>10 s<br/></td></tr><tr><td>30 cycles<br/></td><td>Annealing<br/></td><td>To be determined<br/></td><td>30 s<br/></td></tr><tr><td><br/></td><td>Extension<br/></td><td>72<br/></td><td>10-20 s/kb<br/></td></tr><tr><td><br/></td><td>Final extension<br/></td><td>72<br/></td><td>2 min<br/></td></tr><tr><td><br/></td><td>Hold<br/></td><td>4<br/></td><td>Until sample storage<br/></td></tr></tbody></table></li><li>Proceed to PCR purification per manufacturerâs protocol. Measure DNA concentration on NanoDrop 1000 as per the manufacturerâs protocol. </li><li>Set up restriction digest of IEX vector and new CDS insert using the following reactions:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>Reagents<br/></td><td>IEX vector<br/></td><td>New CDS insert<br/></td></tr><tr><td>BbvCI<br/></td><td>4 Âµl<br/></td><td>4 Âµl<br/></td></tr><tr><td>KpnI<br/></td><td>4 Âµl<br/></td><td>4 Âµl<br/></td></tr><tr><td>CutSmart buffer (10x)<br/></td><td>5 Âµl<br/></td><td>5 Âµl<br/></td></tr><tr><td>Alkaline phosphatase<br/></td><td>1 Âµl<br/></td><td>--<br/></td></tr><tr><td>DNA<br/></td><td>Up to 5 Âµg<br/></td><td>Up to 5 Âµg<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O<br/></td><td>Add to 50 Âµl<br/></td><td>Add to 50 Âµl<br/></td></tr><tr><td>Total<br/></td><td>50 Âµl<br/></td><td>50 Âµl<br/></td></tr></tbody></table></li><li>Incubate at 37 Â°C for 1 h.</li><li>During incubation, cast a 1% agarose gel to be used for isolating the digested products.</li><li>Undertake gel electrophoresis with the entire volume of the restriction digest mixture.<br/>Troubleshoot: it is advised to include a sample of the uncut and singly-cut IEX vector as controls.</li><li>Excise the doubly digested IEX vector (7.0 kb) and new CDS insert.</li><li>Proceed to gel extraction purification per manufacturerâs protocol. Measure DNA concentration.</li></ol><br/><strong><em>Day 1. Ligation of new CDS insert into doubly digested IEX vector</em></strong> <ol class="npwtwo" start="12"><li>Determine the appropriate vector: insert mass ratios to achieve an approximate 1:1 molar ratio. This task can be simplified by using online algorithms, including the NEBioCalculator tool (version 1.10.0; <a href="https://nebiocalculator.neb.com/#!/ligation" target="_blank">https://nebiocalculator.neb.com/#!/ligation</a>).</li><li>Set up the following ligation reactions:<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>Reagents<br/></td><td>Vector-only control (Âµl)<br/></td><td>Vector + Insert (Âµl)<br/></td></tr><tr><td>T4 DNA ligation buffer (10x)<br/></td><td>2<br/></td><td>2<br/></td></tr><tr><td>T4 DNA ligase<br/></td><td>2<br/></td><td>2<br/></td></tr><tr><td>Vector (100 ng)<br/></td><td>Calculate<br/></td><td>Calculate<br/></td></tr><tr><td>Insert (x ng, depends on ratio)<br/></td><td>--<br/></td><td>Calculate<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O<br/></td><td>Add to 20<br/></td><td>Add to 20<br/></td></tr><tr><td>Total<br/></td><td>20<br/></td><td>20<br/></td></tr></tbody></table></li><li>Incubate at room temperature for 15 min.</li><li>Proceed to bacterial transformation per manufacturerâs protocol.</li><li>Plate bacteria on agar plates with 100 Âµg/ml ampicillin (the same concentration is used for all applications involving ampicillin), incubate plates upside down at 30 Â°C overnight.<br/>Endura chemically competent cells are used to minimize truncation of complex plasmids at 30 Â°C growth temperature on agar. Agar plates are incubated upside down to reduce evaporation of moisture from the agar during overnight incubations.</li></ol><br/><strong><em>Day 2. Pick transformant colonies</em></strong> <ol class="npwtwo" start="17"><li>Compare the number of colonies on the âvector-onlyâ control plate, which should have none to very few colonies, and the âvector plus insertâ plate, which should have many colonies.</li><li>Pick 6-10 colonies from the âvector plus insertâ plate to grow in 3 ml of LB medium with ampicillin with shaking at 225 rpm overnight at 37 Â°C.</li></ol><br/><strong><em>Day 3. Extract DNA for sequencing</em></strong> <ol class="npwtwo" start="19"><li>Save 500 Âµl of the overnight bacteria culture as a glycerol stock.</li><li>Extract the DNA from the remaining bacteria culture by the miniprep method as per manufacturerâs protocol.</li><li>Measure the DNA concentration and send the purified DNA plasmid for sequencing.</li><li>If additional site-directed mutagenesis is required, perform the reaction using the Q5<sup>Â®</sup> Site-directed mutagenesis kit according to manufacturerâs protocol.<br/></li></ol><br/><strong><em>Days 4-5. Maxiprep new IEX and iCre vectors for transfection</em></strong> <ol class="npwtwo" start="23"><li>Use the bacteria stock with the correct sequencing result to grow in a 125-250 ml culture with ampicillin, then undertake a DNA maxiprep following manufacturer instructions.<br/><em>Note: It is important to use a maxiprep kit that removes endotoxins from the bacteria to minimize undesired cellular toxicity during future transfections.</em> </li><li>At the same time, maxiprep the iCre vector for future transfections.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Week 2. Generating inducible cells with new IEX vector<br/>Next steps are designed to create inducible cells by co-transfection of the IEX vector and the iCre vector, followed by puromycin selection and confirmation of inducibility by doxycycline (Dox) treatment.<br/><br/><strong><em>Ahead of Day 8</em></strong> <ol class="npwtwo" start="25"><li>Culture target cells in cell culture incubators at 37 Â°C, 95% moisture, and 5% CO<sub>2</sub>.</li><li>Perform all protocols involving mammalian cells in certified biosafety cabinets to prevent contamination and adhere to biosafety rules and regulations.</li><li>Passage at least once before transfection to check that cells are growing normally as they can be quite fragile after thawing. See âAuxiliary informationâCell cultureâ section for instructions on regular cell culture, passaging, and saving stocks.</li><li>Determine the appropriate puromycin concentration to use for selection for each cell line by treating cells at various concentrations (<em>e.g.</em>, 0.0, 0.2, 0.5, 1.0, 2.0, 5.0, and 10 Âµg/ml) of puromycin and selecting the lowest concentration that kills 100% of cells in 3 days.</li></ol><br/><strong><em>Day 8. Plate cells for transfection</em></strong> <ol class="npwtwo" start="29"><li>Plate all cells in media without antibiotics or antimycotics.</li><li>For ReN VM cells, plate ~135,000 cells/well on Matrigel-coated 12-well plates in proliferation media without heparin. Cells should be ~50% confluent the next day.</li><li>For cells that do not need to be plated on coated cell culture dishes, plate ~350,000 cells/well on 6-well plates with the expectation that they are ~80% confluent the next day.</li><li>Set 1-2 wells aside as controls for effective puromycin selection; they will not be transfected.</li></ol><br/><strong><em>Day 9. Transfection</em></strong> <ol class="npwtwo" start="33"><li>Transfect cells using a ratio of 1 iCre vector: 4 IEX vectors with desired insert.<br/>A ratio of 1 iCre vector: 1-3 IEX vectors have also worked.</li><li>For ReN VM cells, use a total of 1.0 Î¼g of DNA with 1 Âµl of TransfeX transfection reagent per well following manufacturerâs instructions.<br/>Use proliferation media without heparin during the transfection because heparin may compromise effectiveness.</li><li>For most other cells, use a total of 3 Âµg of DNA with jetPrime transfection reagent per well following manufacturerâs instructions.<br/>For IMR-32 cells, use a ratio of 1 Âµg DNA:3 Âµl jetPrime.<br/>For HEK-293 cells, use a ratio of 1 Âµg DNA:2 Âµl jetPrime.</li><li>To minimize toxicity change media 4 h after transfection to regular proliferation media.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Week 3. Puromycin selection and Dox treatment<ol class="npwtwo" start="37"><li>Passage transfected cells onto a few larger plates and add the appropriate amount of puromycin and 2.0 Âµg/ml of Dox.<br/>One may check for induction levels 24 h post Dox addition on a fluorescence microscope.</li><li>Continue puromycin selection until all cells in the control wells have died.<br/><em>Note: It is recommended to refresh puromycin and Dox every other day to keep the inducible transgene transcriptionally active.</em> </li><li>Allow surviving cells to proliferate to sizable colonies of &gt; 100 cells/colony.</li><li>Induce cells with 2.0 Âµg/ml Dox; check on the fluorescence microscope for successfully induced colonies.<br/><em>Caution: Induction in ReN VM cells may not be homogenous even within the same clone and may lose induction during the course of differentiation. This is because endogenous epigenetic silencing may cause variable and mosaic expression of inducible constructs (Chanda et al., 2017). Also, if differentiation is desired, induction of ReN VM cells and other neural progenitor cell lines may be improved by beginning induction 2-4 days during the proliferation stage before the start of differentiation.</em> </li><li>Mark and transfer selected colonies with desired level of induction to new wells. Expand them to save as stocks and to validate the inducibility and identity of the protein-of-interest by Western blot.</li><li>If removal of the Puro<sup>R</sup> gene is desired, transfect cells with the âFlp<sub>o</sub>â optimized Flippase vector.</li><li>Select cells that have lost puromycin resistance by plating duplicates of the same cell clone, one of them puromycin treated the other left untreated.<br/>In ReN VM cells, retaining Puro<sup>R</sup> may help with keeping the inducible transgene region transcriptionally active.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Week 4. Preparing inducible cells for mass spectrometry-based interactome studies<br/><em>Note: The following steps are designed to capture EGFP-tagged inducible protein and its interactors using GFP nanotrap for downstream analysis in mass spectrometer.</em><br/><ol class="npwtwo" start="44"><li>Culture inducible protein-of-interest-EGFP fusion cells in biological triplicates on 15-cm plates along with cells inducibly expressing EGFP only as a control against unspecific binding to the GFP nanotrap matrix.<br/><em>Note: If induced expression is very low, scale up accordingly.</em> </li><li>Induce cells for the desired length of time.</li><li>Follow manufacturerâs protocol on using GFP nanotrap for affinity capture experiments.<br/>All procedures should be performed at 4 Â°C to prevent protein degradation.</li><li>Cells may be lysed in RIPA buffer as per manufacturerâs protocol or buffer composed of 0.5% NP-40 and 0.25% DOC (Recipe 4).<br/>Remove cellular debris by centrifugation at 3,000 <em>x g</em> for 5 min at 4 Â°C.</li><li>Due to its high binding capacity, 20 Î¼l of GFP nanotrap bead slurry/biological replicate should be sufficient for mass spectrometry-based interactome studies.</li><li>EGFP fusion proteins or EGFP alone may be captured on GFP nanotrap beads during 2 h incubations at 4 Â°C with continuous mixing with the lysate.<br/>Shorter nanotrap bead incubations of 5 min followed by light centrifugation may be sufficient to visualize the capture of EGFP fusion proteins.</li><li>Centrifuge at 100 <em>x g</em> at 4 Â°C to collect agarose-conjugated GFP nanotrap beads and bait proteins.</li><li>Wash beads 5 times in Wash buffer. Washing procedures should be done within 2 h to minimize loss of interactors.<br/><ol class="npwthree"><li>First 3 times in Washing buffer (Recipe 5). NaCl salt concentration of 150-500 mM can be used during these washing steps, depending on desired stringency.</li><li>The last two washes should be 25 mM HEPES (pH 7.0) and 10 mM HEPES (pH 7.0). Tris-HCl must not be used.</li></ol></li><li>Elute in 0.2% trifluoroacetic acid with 20% acetonitrile in water (pH 1.9). Immediately save and flash freeze or pH neutralize a portion of the eluates (<em>e.g.</em>, 25% of total) for downstream validation of bait protein complexes by Western blot.<br/>A delay in the collection of the eluate material dedicated to Western blot analysis can lead to precipitation of eluate proteins in the pH 1.9 environment, thereby compromising the Western blot analysis.</li><li>Other special considerations:<br/><ol class="npwthree"><li>One may use BCA to adjust for equal protein content before affinity capture.</li><li>Remember to save a fraction of the cellular lysates as âinputâ and âunboundâ for Western blot analyses.</li></ol></li></ol><br/><strong><u>Auxiliary information</u></strong><br/><strong>Cell culture</strong> <ol class="npwtwo"><li>After thawing cells at 37 Â°C, remove DMSO content from the cell stock by diluting the cell stock in 4 ml of proliferation media and centrifuging cells at 100 <em>x g </em>for 3 minutes at 25 Â°C. Remove the supernatant to retain only the cell pellet. DMSO may be toxic to cells in regular culture.</li><li>Culture cells in the appropriate proliferation media for about one week to allow recovery from thawing.</li><li>For ReN VM cells, do not include antibiotics or antimycotics in the proliferation media, as they will cause cell death.</li><li>For ReN VM cells, please coat the tissue-culture plates with Matrigel according to the manufacturerâs protocol at least 2 h before seeding cells onto the plates. 2 h is needed for Matrigel to solidify at 37 Â°C in a humidified incubator.</li><li>Passage cells regularly before they reach 90% confluency to prevent the adverse effects of overgrowing with the appropriate dissociation reagent. For ReN VM cells, use Accutase. For most other cell types, use trypsin.</li><li>Remember to save a portion of the cells, especially valuable clones, as stocks and while using the rest for experiments or regular culture.</li><li>For preserving cell stocks, resuspend cells in Cell Freezing Media in labelled cryovials. Transfer cells to -80 Â°C immediately and store at -80 Â°C for 1 day before transferring into liquid nitrogen for long term storage.</li></ol></li></ol><p class="pptt" id="biaoti33119">### Data analysis</p><p class="ppzz">Information on statistical analyses as well as were to find publicly available mass spectrometry data from the TAU and BAG5 analyses was included with the article describing the first use of this system (Wang <em>et al.</em>, 2019), with Figure 2 of the original article providing details on the design of a quantitative interactome analysis, Figure 3 describing mass spectrometry sequence coverage of the TAU protein, Figure 6 comparing the TAU interactome in IMR32 and ReN VM cells, and Figure 9 documenting results from a post-translational modification analysis of TAU produced with this system.</p><p class="pptt" id="biaoti33120">### Recipes</p><ol class="npwtwok"><li>Proliferation media for IMR-32 and HEK-293 cells<br/><em>Note: Make media fresh, store at 4 Â°C, stable for 2 months.</em><br/>DMEM media with high glucose, L-glutamine, and sodium pyruvate supplemented with 10% qualified fetal bovine serum and 1% GlutaMAX<sup>TM</sup>-I supplement</li><li>Proliferation media for ReN VM cells<br/><em>Note: Make media fresh, store at 4 Â°C, stable for 2 weeks.</em><br/>DMEM/F12, 1:1 with L-glutamine but no HEPES supplemented with 2% N21-MAX supplement or 2% B-27 serum-free supplement<br/>20 ng/ml basic fibroblast growth factor<br/>20 ng/ml epidermal growth factor<br/>2 ng/ml heparin</li><li>Neuronal differentiation media for ReN VM cells<br/><em>Note: Make media fresh, store at 4 Â°C, stable for 1 month.</em><br/>DMEM/F12 1:1 with L-glutamine but no HEPES supplemented with 2% N21-MAX supplement or 2% B-27 serum-free supplement<br/>2 ng/ml glial-derived neurotrophic factor<br/>500 Î¼M dibutyryl-cyclic-adenosine monophosphate</li><li>Lysis buffer<br/><em>Note: Make fresh before use, stable at 4 Â°C for 1 day.</em><br/><ol class="npwthree"><li>Mix 0.5% (v/v) NP-40, 0.5% (w/v) DOC, Tris/HCl, pH 8.3, 5 mM EDTA, 1 mM Na<sub>3</sub>VO<sub>4</sub>, 10 mM NaF, 1 mM PMSF, 1x cOmplete protease inhibitor, and 1x PhosStop phosphatase inhibitor</li><li>Prepare NP-40 and DOC as 50x stock solutions in water and warm gently to dissolve. DOC will precipitate at pH value below 8.0. Therefore, ensure that the pH exceeds this value and is buffered before adding this detergent</li></ol>If the intention is to use lysates for immunoaffinity capture of the protein-of-interest, it is advisable to prepare a larger amount of this buffer and divert some of it for the preparation of the Wash (see below) before addition of protease and phosphatase inhibitor cocktails. Also, we recommend chilling the buffer to 4 Â°C before adding temperature-labile protease and phosphatase inhibitor cocktails.</li><li>Wash buffer<br/><em>Note: Make fresh before use, stable at 4 Â°C for 2 days.</em><br/>The composition of this buffer is similar to the lysis buffer with the following considerations:<br/>Protease and phosphatase inhibitors can be omitted.<br/>If samples are used for mass spectrometry downstream, the last two washes must use only 25 mM HEPES (pH 7.0) and 10 mM HEPES (pH 7.0) in deionized water. Tris-HCl must not be used.</li><li>Elution buffer<br/><em>Note: Make fresh before use, stable at room temperature for 1 day.</em><br/><ol class="npwthree"><li>Mix 0.2% TFA and 20% acetonitrile in deionized water</li><li>The pH of this solution will be 1.9 without adjustment</li></ol></li></ol><p class="pptt" id="biaoti33121">### Acknowledgments</p><p class="ppzz"><br/>XW held an Ontario Graduate Scholarship. GS received funds from the Krembil Foundation, Canadian Institutes for Health Research (grant number 137651) and the Alberta Prion Research Institute (grant number 201600028). Support by the Arnold Irwin family for the purchase of a mass spectrometer was instrumental for getting started on this project. The authors gratefully acknowledge generous philanthropic support by the Borden Rosiak family.</p><p class="pptt" id="biaoti33122">### Competing interests</p><p class="ppzz">The authors declare that they have no financial or non-financial competing interests.</p><p class="pptt" id="biaoti33123">### Ethics</p><p class="ppzz">The study did not involve human subjects or animal work. All handling of human cell lines and chemicals were undertaken in accordance with an active biosafety protocol (number 208-S06-2) approved by the environmental health and safety (EHS) office at the University of Toronto, Toronto, Canada.</p><p class="pptt" id="biaoti33124">### References</p><ol class="npwtwok"><li>Chanda, D., Hensel, J. A., Higgs, J. T., Grover, R., Kaza, N. and Ponnazhagan, S. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28926997" target="_blank">Effects of cellular methylation on transgene expression and site-specific integration of adeno-associated virus.</a> <em>Genes (Basel)</em> 8(9): 232.</li><li>DeKelver, R. C., Choi, V. M., Moehle, E. A., Paschon, D. E., Hockemeyer, D., Meijsing, S. H., Sancak, Y., Cui, X., Steine, E. J., Miller, J. C., Tam, P., Bartsevich, V. V., Meng, X., Rupniewski, I., Gopalan, S. M., Sun, H. C., Pitz, K. J., Rock, J. M., Zhang, L., Davis, G. D., Rebar, E. J., Cheeseman, I. M., Yamamoto, K. R., Sabatini, D. M., Jaenisch, R., Gregory, P. D. and Urnov, F. D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20508142" target="_blank">Functional genomics, proteomics, and regulatory DNA analysis in isogenic settings using zinc finger nuclease-driven transgenesis into a safe harbor locus in the human genome.</a> <em>Genome Res</em> 20(8): 1133-1142.</li><li>Devkota, S. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30103848" target="_blank">The road less traveled: strategies to enhance the frequency of homology-directed repair (HDR) for increased efficiency of CRISPR/Cas-mediated transgenesis.</a> <em>BMB Rep </em>51(9): 437-443.</li><li>Donato, R., Miljan, E. A., Hines, S. J., Aouabdi, S., Pollock, K., Patel, S., Edwards, F. A. and Sinden, J. D. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17531091" target="_blank">Differential development of neuronal physiological responsiveness in two human neural stem cell lines.</a> <em>BMC Neurosci</em> 8: 36.</li><li>Ran, F. A., Hsu, P. D., Lin, C. Y., Gootenberg, J. S., Konermann, S., Trevino, A. E., Scott, D. A., Inoue, A., Matoba, S., Zhang, Y. and Zhang, F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23992846" target="_blank">Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity.</a> <em>Cell</em> 154(6): 1380-1389.</li><li>Rothbauer, U., Zolghadr, K., Muyldermans, S., Schepers, A., Cardoso, M. C. and Leonhardt, H. (2008).<a href="http://www.ncbi.nlm.nih.gov/pubmed/17951627" target="_blank"> A versatile nanotrap for biochemical and functional studies with fluorescent fusion proteins.</a> <em>Mol Cell Proteomics </em>7(2): 282-289.</li><li>Wang, X., Williams, D., MÃ¼ller, I., Lemieux, M., Dukart, R., Maia, I. B. L., Wang, H., Woerman, A. L. and Schmitt-Ulms, G. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31700063" target="_blank">Tau interactome analyses in CRISPR-Cas9 engineered neuronal cells reveal ATPase-dependent binding of wild-type but not P301L Tau to non-muscle myosins.</a> <em>Sci Rep</em> 9(1): 16238.<br/></li></ol>