<body>
            <h1>Negative Ion Mode nanoLC-ESI-MS/MS Analyses of Permethylated Sulfated Glycans</h1>
            <p><strong>Authors:</strong> Shin-Yi Yu, Chu-Wen Cheng, Kay-Hooi Khoo</p>
            <h2>Abstract</h2>
            <p>We have developed enabling techniques for sulfoglycomics based on MALDI-MS mapping and MS/MS sequencing of permethylated sulfated glycans. We then extended further the analytical workflow to C18 reverse phase (RP)-nanoLC-nanoESI-MS/MS analyses of permethylated sulfated glycans in the negative ion mode. The advantages are that extra sulfates on permethylated di- and multiply sulfated glycans will survive in nanoESI conditions to allow detection of multiply charged intact molecular ions, and more comprehensive MS/MS can be performed in an automated fashion at higher sensitivity, compared with MALDI-MS/MS. Parallel higher energy collision dissociation (HCD) and ion trap collision induced dissociation (CID)-based MS<sup>2</sup>, coupled with product-dependent MS<sup>3</sup> in data dependent acquisition mode proved to be highly productive when applied to resolve and identify the isomeric sulfated glycan structures. In-house glycomic data mining software, GlyPick, was developed and used to automate the downstream process of identification and relative quantification of target sulfated glycotopes based on summed intensity of their diagnostic MS<sup>2</sup> ions extracted from thousands of HCD-MS<sup>2</sup> and/or CID-MS<sup>2</sup> data.</p>
            <p><strong>Keywords:</strong> Permethylated sulfated glycans, Sulfoglycomics, Mass spectrometry, Nanospray, LC-MS/MS, GlyPick</p>
            <h2>Background</h2><p>Current mass spectrometry (MS)-based glycomic mapping remains insufficient to delineate the full complexity of the glycome. Although MALDI-MS mapping and MS/MS sequencing would efficiently afford a very useful first impression glycomic profile, it is nearly impossible to acquire MALDI-MS/MS on every putative glycan signal detected, especially those of low intensities and/or occurring at higher masses. In that respect, LC-ESI-MS/MS analysis in an automated and data-dependent acquisition (DDA) mode provides a far more comprehensive MS/MS data coverage (Cheng <em>et al.</em>, 2013; Cheng <em>et al.</em>, 2015; Hsiao <em>et al.</em>, 2017; Yu <em>et al.</em>, 2018). We have demonstrated that permethylation in conjunction with 2 steps C18/amine SPE or a single step mixed mode MAX SPE fractionation can yield fully methylated non-sulfated, mono-sulfated, and di-sulfated glycans in separate pools (Yu <em>et al.</em>, 2020) for MALDI-MS screening. We found that loss of sodium sulfite from permethylated di-sulfated glycans occurred readily during MALDI-ionization (Figure 1A), while the two sulfates on permethylated glycans were fully retained when analyzed by nanoESI-MS in negative ion mode, allowing them to be detected as [M-2H]<sup>2-</sup> (Figure 1B), and further selected for MS/MS analyses at high sensitivity. Moreover, we showed that the negative mode nanoLC-MS/MS analysis of permethylated sulfated glycans on a C18 reverse phase capillary column could be efficiently carried out using the common acetonitrile/formic acid/water solvent system, and hence be fully compatible with the normal set-up of an analytical laboratory devoted to proteomics.<br/><br/>By virtue of a panel of synthetic sulfated glycans, we have shown that the beam-type higher energy collision dissociation (HCD) MS/MS as implemented on the hybrid Orbitrap series would afford a range of low mass fragment ions. These diagnostic ions would define the location of sulfate on which glycosyl residue at which position (Cheng <em>et al.</em>, 2015). Ion trap-based CID MS<sup>2</sup>, on the other hand, would not retain these very useful ions due to its one third low mass cut-off (Patnode <em>et al.</em>, 2013). It does, however, offer a higher sensitivity and acquisition rate, allowing definitive assignment of target sulfated glycotopes via MS<sup>2</sup> product ion-dependent MS<sup>3</sup>. Occasionally, when a di-sulfated <em>O</em>-glycan may carry a different permutation of sulfated glycotopes on either the 6-arm or the extended 3-arm, additional targeted MS<sup>n</sup> analyses can help identify the existence of isomers (Yu <em>et al.</em>, 2018). Moving from the original linear ion trap-Orbitrap hybrid systems to more recent tribrid Orbitrap Fusion systems, the current MS systems afford a higher degree of flexibility for different combination of single to multiple stages of HCD versus ion trap-based CID MS/MS to be acquired either in Orbitrap for greater resolution and mass accuracy, or the ion trap for better sensitivity and speed. These aspects will not be further dealt with here. Suffice to point out that the basic principle remains the same, namely to acquire as many HCD/CID MS<sup>2</sup> within an elution time window-compatible DDA duty cycle and to couple each, if possible and desirable, to a pre-determined list of product-dependent MS<sup>3</sup> for greater depth of structural details. With or without MS<sup>3</sup>, it is important that the low mass ions produced in negative mode MS<sup>2</sup> be retained and detected at reasonably high resolution and mass accuracy (&lt; 5 ppm, if possible).<br/><br/>Online LC and data-dependent MS/MS acquisition will produce a huge dataset which is near impossible to manually analyze systematically. Unlike proteomics, neither the glycan database nor the MS/MS sequencing algorithm is well developed to allow direct MS<sup>2</sup> ions or spectral matching search for unambiguous identification. An in-house computational tool, GlyPick, was developed to filter out <em>bona fide</em> glycan MS<sup>2</sup> spectra by user-defined criteria, usually by presence of at least 2 to 3 diagnostic glycan fragment ions. It can also extract out and compute the occurrence and summed intensity of a list of user-input diagnostic MS<sup>2</sup> ions that will define the presence and relative abundance of those important glycotopes such as 6-sulfated GlcNAc, 3â-sulfated Gal, 6â-sulfated Gal, sulfated LacNAc, sulfated Le, <em>etc</em>, from these MS<sup>2</sup> spectra (Hsiao <em>et al.</em>, 2017; Yu <em>et al.</em>, 2018). Results are output in CSV format, which can be conveniently interrogated using Excel for further data mining and graph plotting.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200512/20200512001140_0859.jpg" width="300"/><br/><strong>Figure 1. MS spectra of di-sulfated permethylated <em>N</em>-glycans acquired by different ionizations, which are</strong> (A) MALDI-MS and (B) nanospray ESI-MS, both in the negative ion mode.</p>
        </body>

<p class="pptt" id="biaoti33171">## Materials and Reagents</p><ol class="npwtwok"><li>Plastic pipette tips</li><li>Microcentrifuge tube</li><li>ZipTipC<sub>18</sub> (Merck Millipore, catalog number: <a class="blackNoUnder" target="_blank"> ZTC18S096 </a> )</li><li>Acetonitrile with 0.1% (v/v) Formic Acid for LC-MS, JT Baker<sup>Â®</sup> (VWR, catalog number: <a class="blackNoUnder" target="_blank"> JT9832-2 </a> )</li><li>Water with 0.1% (v/v) Formic acid, BAKER ANALYZED<sup>TM</sup>, JT Baker<sup>Â®</sup> (VWR, catalog number: <a class="blackNoUnder" target="_blank"> JT9826-3 </a> )<br/></li></ol><p class="pptt" id="biaoti33172">### Equipment</p><ol class="npwtwok"><li>SpeedVac</li><li>nanoACQUITY UPLC system (Waters Corporation)</li><li>nanoACQUITY M-Class BEH130 C18 column, 1.7 Î¼m, 75 Î¼m x 250 mm (Waters Corporation, catalog number: <a class="blackNoUnder" target="_blank"> 186003545 </a> )</li><li>Picoview nanospray source 550 (New Objective, catalog number: <a class="blackNoUnder" target="_blank"> PV-550 </a> )</li><li>A hybrid LTQ-Orbitrap Elite<sup>TM</sup> Mass Spectrometer (Thermo Scientific), or any equivalent high resolution/mass accuracy MS system including the tribrid Orbitrap Fusion (Thermo Fisher Scientific)<br/></li></ol><p class="pptt" id="biaoti33173">### Software</p><ol class="npwtwok"><li>Masslynx<sup>TM</sup> software v. 4.1 (Waters)</li><li>Xcalibur<sup>TM</sup> software v2.2 (Thermo Fisher Scientific)</li><li>GlyPick (in-house, available upon request)<br/></li></ol><p class="pptt" id="biaoti33174">### Procedure</p><ol class="npwone"><li>Sample preparation<ol class="npwtwo"><li>Before subjecting to UPLC system, permethylated glycan sample should be cleaned up further by ZipTipC<sub>18</sub> (described in Yu <em>et al.</em>, 2020), and dried by SpeedVac.</li><li>Re-dissolve the permethylated glycan sample in 10 Î¼l of 5% acetonitrile, 0.1% formic acid.<br/><br/></li></ol></li><li>Set up the UPLC system parameters<ol class="npwtwo"><li>Connect the nanoACQUITY UPLC system to an LTQ-Orbitrap Elite<sup>TM</sup> hybrid mass spectrometer via PicoView nanospray source for nanoLC separation at 35 Â°C, using a 75 Î¼m ID, 25 cm length C18 BEH column packed with 1.7 Î¼m particles with a 300 Ã pore size.</li><li>Set up the constant flow rate of 300 nl/min</li><li>Set the solvent system containing 100% water with 0.1% formic acid (FA) for mobile phase A, and 100% acetonitrile with 0.1% FA for mobile phase B. Use a linear gradient of 30-60% of B over the course of 30 min and then increase to 80% acetonitrile over the course of 5 min and hold isocratically for another 10 min (shown in Figure 2).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200512/20200512005925_4636.jpg" width="100"/><br/><strong>Figure 2. Steps in setting up HPLC gradient method</strong><br/><br/></li></ol></li><li>Set up parameters for MS acquisition (shown in Figure 3)<br/><ol class="npwtwo"><li>Double-click the Xcalibur software and then, click the Instrument Setup.</li><li>Click âNth order double playâ.</li><li>Choose âInitialize method with Orbitrap Elite supportâ, and type â5â in Analyze Top N peaks, then click âOKâ. The page named âThermo Xcalibur Instrument Setupâ will be shown.</li><li>Set up parameters which are<br/><ol class="npwthree"><li>Acquire time is set for 60 min and type 3 in the âScan eventsâ in Segment 1 settings (indicated by blue arrow).<br/><em>Note: The acquisition time can be adjusted when LC gradient is changed or different UPLC system/reverse phase capillary column is used.</em><br/></li><li>Click Scan Event 1 (active window shown as yellow background).<br/><ol class="npwfour"><li>Scan Description: Analyzer: FTMS; Mass Range: Normal; Resolution: 120,000; Scan Type: Full; Polarity: Negative; Data type: Profile.</li><li>Scan Ranges: First Mass (<em>m/z</em>) 650; Last Mass (<em>m/z</em>) 2000.</li></ol></li><li>Click Scan Event 2 (active window shown as yellow background). <br/><ol class="npwfour"><li>Scan Description: Analyzer: Ion Trap; Mass Range: Normal; Scan type: Normal; Data type: Centroid</li><li>Click Settings and the window âData Dependent Settingsâ will be shown.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Choose âDynamic Exclusionâ; Click âEnabledâ; Type Repeat count: 2; Repeat duration (s): 15; Exclusion list size: 500; Exclusion duration (s): 60; Exclusion mass width, choose âBy massâ; Low: 1.5; High: 1.5, Click âOKâ.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Choose âCharge Stateâ; Click âEnable charge state screeningâ and âEnable monoisotopic precursor selectionâ; Click âEnabledâ and choose âReject charge stateâ: â2â, â3â, â4 and upâ. Click âOKâ.<br/><em>Note: This setting is for mono-sulfated permethylated glycans due to singly charge of mono-sulfated glycans in the negative ion mode.</em> </div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">Choose âCurrent Scan Eventâ; Type Minimum signal threshold (counts): 5000. Click âOKâ.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">Choose âActivationâ; Choose Activation type: CID; Default charge state: 2; Isolation width (<em>m/z</em>): 2; Normalized collision energy: 35; Activation Q: 0.25; Activation time (ms): 10. Click âOKâ.</div></div></li></ol></li><li>Click Scan Event 3 (active window shown as yellow background)<br/><ol class="npwfour"><li>Scan Description: Analyzer: FTMS; Mass Range: Normal; Resolution: 15,000; Data type: Centroid.</li><li>Click Settings and the window âData Dependent Settingsâ will be shown.<br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Choose âDynamic Exclusionâ; Click âEnabledâ; Repeat count: 2; Repeat duration (s): 15; Exclusion list size: 500; Exclusion duration (s): 60; Exclusion mass width, choose âBy massâ; Low: 1.5; High: 1.5. Click âOKâ.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Choose âCharge Stateâ; Click âEnable charge state screeningâ and âEnable monoisotopic precursor selectionâ; Click âEnabledâ and choose âReject charge stateâ: â2â, â3â, â4 and upâ. Click âOKâ.<br/><em>Note: This setting is for mono-sulfated permethylated glycans due to singly charge of mono-sulfated glycans in the negative ion mode.</em> </div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left">3)</div><div class="media-body">Choose âCurrent Scan Eventâ; Type Minimum signal threshold (counts): 5000. Click âOKâ.</div></div><div class="media media-x cnautokh cnautokh4" style="margin-top:-0.5px;"><div class="float-left">4)</div><div class="media-body">Choose âActivationâ; Choose Activation type: HCD; Default charge state: 2; Isolation width (<em>m/z</em>): 2; Normalized collision energy: 110; Activation time (ms): 0.1. Click âOKâ.</div></div></li></ol></li></ol></li><li>Go to Tune plus window, and choose SetupâFT Injection Control or click <img alt="" class="layerphoto" height="20" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200512/20200512011752_6314.jpg" width="30"/> in the Instrument Control toolbar. The window named âInjection Controlâ will be shown. Choose FT, AGC Target Settings, Type âFull MSâ: 1.00e<sup>6</sup>; SIM: 5.00e<sup>4</sup>; MS<sup>n</sup>: 5.00e<sup>4</sup>; Click âEnable Full Scan Injection Waveformsâ. Choose Ion Trap, Full MS: 3.00e<sup>4</sup>; SIM: 1.00e<sup>4</sup>; MS<sup>n</sup>: 1.00e<sup>4</sup>; Zoom: 3000, Click âOKâ.<br/><br/><span style="font-size:10pt;"></span><span style="font-size:10pt;"></span> </li></ol><em>Note: These settings are based on the ones used in published work (Yu et al., 2018). Other settings should be adjusted according to sample type. For example,</em><br/><ol class="npwtwo"><li><em>Scan ranges in the scan event 1 will be changed to 1,000-4,000 for permethylated mono-sulfated N-glycans due to singly charged property.</em> </li><li><em>Check the section âData dependent settingsâ â âSegmentâ â âCharge Stateâ, The rejected charge state should be adjusted depends on charge states of glycans. For di-sulfated N- or O-glycans detected in the negative ion mode are normally doubly charged. The rejected charge states should be 1, 3, and 4-and up.</em> </li><li><em>One should always optimize and calibrate the LC-MS/MS system accordingly.</em><br/><br/><img alt="" class="layerphoto" height="184" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200512/20200512010055_3701.jpg" width="300"/><br/><strong>Figure 3. Setting up the MS acquisition</strong><br/></li></ol></li></ol><p class="pptt" id="biaoti33175">### Data analysis</p><ol class="npwone"><li>ESI-MS analysis in the negative ion mode<br/>All data were processed by Xcalibur software v2.2 manually. We observed that permethylated mono-sulfated glycans were eluted earlier than di-sulfated glycans by using the nanoACQUITY UPLC BEH130 column and acetonitrile/formic acid/ water solvent system. The ion signals of mono-sulfated <em>N</em>- and <em>O</em>-glycans were detected as [M-H]<sup>-</sup>, whereas di-sulfated <em>N</em>- and <em>O</em>-glycans were detected as [M-2H]<sup>2-</sup> in the ESI-MS spectra.<br/><br/></li><li>ESI-MS/MS analyses in the negative ion mode<br/>The acquired HCD-MS<sup>2</sup> and CID-MS<sup>2</sup> spectra are typically averaged over a period of retention time according to the elution profile of its precursor, and then interpreted manually. In general, fragmentation pattern of permethylated sulfated glycans in the negative ion mode is similar to fragmentation pattern in the positive ion mode (Yu <em>et al.</em>, 2006; Hsiao <em>et al.</em>, 2017), except that only fragment ions retaining the sulfate and hence the negative charge will be detected in the negative ion mode. Considerable expertise and experience are required to correctly and fully assign the various cleavage ions, while identification of the few well-established diagnostic ions is more straightforward (summarized in Table 1). An example is shown below to illustrate the characteristic fragmentation patterns that can be expected (Figure 4).<br/><br/><img alt="" class="layerphoto" height="225" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200512/20200512010318_2016.jpg" width="300"/><br/><strong>Figure 4. HCD-MS/MS and CID-MS/MS spectra of di-sulfated <em>O</em>-glycan corresponding to 2SO3Hex3HexNAc3-itol at m/z 769.81, doubly charged [M-2H]<sup>2-</sup></strong><br/><br/><strong>Table 1. A selection of diagnostic fragment ions afforded by HCD-MS/MS analyses that can be used to assess the presence and relative abundance of some of the most commonly found sulfated glycotopes.</strong> The MS<sup>2</sup> ions at <em>m/z</em> 528, 702, 889, 1063 can be further subjected to MS<sup>3</sup> to produce those diagnostic ions at &lt; <em>m/z</em> 400 that are informative of the precise location of sulfate on either the Gal or GlcNAc residue. The signal at <em>m/z</em> 97 is highly abundant, ubiquitous and useful to confirm presence of sulfate but itself is not informative of the particular sulfated glycotopes. It can be excluded from the mass range of LC-MS<sup>2</sup> data acquisition to better detect the other much lower abundant ions.<br/><img alt="" class="layerphoto" height="345" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200512/20200512010410_5227.png" width="600"/><br/><br/><ol class="npwtwo"><li>HCD-MS/MS analysis<br/><ol class="npwthree"><li>A strong B ion at <em>m/z</em> 528 and related E ion at <em>m/z</em> 457 indicate the existence of sulfated LacNAc unit.</li><li><sup>3,5</sup>A ion at <em>m/z</em> 371 and D ion at <em>m/z</em> 514 indicate the existence of sulfated type 2 LacNAc.</li><li>Low mass ions at <em>m/z</em> 139, 195, 234, 264 indicate that both sulfate groups are located on the 6-carbon position of HexNAc.<br/></li><li>The ions at <em>m/z</em> 588/616/676 are cleavage ions at HexNAc-itol, indicating that the sulfated LacNAc unit is carried on the 6-arm of HexNAc-itol, which are informative. This glycotope is further confirmed by Y ion at <em>m/z</em> 807 and Z ion at <em>m/z</em> 789.</li></ol></li><li>CID-MS/MS analysis<br/><ol class="npwthree"><li>Those fragment ions shown in CID MS<sup>2</sup> were also observed in the HCD MS<sup>2</sup> data except i) ion at <em>m/z</em> 1442, corresponding to elimination of sulfate from precursor ion; ii) Z ion at <em>m/z</em> 652, which is doubly charged and corresponds to the elimination of terminal hexose residue; iii) ions at <em>m/z</em> 1039 and 1108 are X ions at HexNAc residue; iv) ions at <em>m/z</em> 1224 and 1208, derived from loss of hexose from the ion at <em>m/z</em> 1442 and concerted elimination of sulfate and hexose from precursor ion at <em>m/z</em> 1540, respectively (Yu <em>et al.</em>, 2006).</li><li>Informative low mass fragment ions, which would indicate the location of sulfate, are lost in the CID-MS/MS data due to cut-off.<br/><br/></li></ol></li></ol><em>Notes:</em><br/><ol class="npwthree"><li><em>Readers are referred to Figure 4 in the published work (Cheng </em><em>et al.</em><em>, 2015) for the characteristic MS</em><sup><em>2</em></sup><em> fragment ions afforded by HCD-MS/MS analyses that are useful to identify the location of sulfate. These characteristic fragment ions were shown not only in HCD-MS/MS analyses of permethylated sulfated </em><em>O</em><em>-glycans, but also </em><em>N</em><em>-glycans.</em> </li><li><em>CID-MS<sup>3</sup> spectra of permethylated sulfated glycans can afford information about sulfated Gal and sulfated GlcNAc, but not the exact location of sulfate (Figures 3C and 3D in Cheng et al., 2015). This can be further resolved and defined by HCD-MS<sup>3</sup> analysis enabled on the Orbitrap Fusion (Figure S3 in Hsiao et al., 2017).</em> </li><li><em>Currently, singly charged precursor ions for permethylated mono-sulfated N-glycans and O-glycans at over </em><em>m/z</em><em> 2000 in the negative ion mode cannot be efficiently isolated in the quadrupole or trap of Orbitrap MS systems for MS/MS analysis.</em> </li></ol><br/></li><li>In-house GlyPick computation tool<ol class="npwtwo"><li>Additional mapping of the sulfated glycotopes based on their respective MS<sup>2</sup> diagnostic ions (Figure 5) can be performed using the in-house developed software, GlyPick. The representative figure describing the user interface for GlyPick has been shown as Figure S4 in Hsiao <em>et al.</em> (2017).</li><li>Parameters in GlyPick for input data of permethylated sulfated glycans<br/><ol class="npwthree"><li>Set up the mass tolerance at Â± 5 ppm and choose Orbitrap as instrument type for the target HCD-MS<sup>2</sup> to be extracted from the acquired data sets, with their absolute and relative intensity threshold set at above 100 and 1%.</li><li>Choose negative ion mode.</li><li>Select the known diagnostic fragment ions from a built-in list as target ions. Additional ions not in the list can be user-input.</li><li>Choose âReducedâ as reducing end when analyzing permethylated reduced glycans.<br/></li><li>Modify other parameters depending on different samples such as mono-sulfated or di-sulfated; <em>N</em>-glycans or <em>O</em>-glycans.<br/></li><li>MS<sup>3</sup> setting will be used when pd-MS<sup>3</sup> acquisition is additionally performed.<br/></li><li>The summed ion intensity for each of the extracted target ions will be computed and the results output in CSV format. These can be further calculated as percentage total of all selected MS<sup>2</sup> ions, as a convenient way of normalization.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti33176">### Acknowledgments</p><p class="ppzz">This LC-MS/MS experimental workflow for sulfoglycomics was adapted from two published works (Cheng <em>et al.</em>, 2015; Yu <em>et al.</em>, 2018) while additional glycotope-centric data mining with GlyPick was developed by Hsiao <em>et al.</em> as reported in Hsiao <em>et al.</em>, 2017. MS data was acquired by LTQ-Orbitrap at the Academia Sinica Common Mass Spectrometry Facilities for Proteomics and Protein Modification Analysis located at the Institute of Biological Chemistry, Academia Sinica, supported by Academia Sinica Core Facility and Innovative Instrument Project (AS-CFII-108-107). These works were supported by Academia Sinica and Taiwan Ministry of Science and Technology (MoST) grants to KKH.</p><p class="pptt" id="biaoti33177">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti33178">### References</p><ol class="npwtwok"><li>Cheng, C. W., Chou, C. C., Hsieh, H. W., Tu, Z., Lin, C. H., Nycholat, C., Fukuda, M. and Khoo, K. H. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26016788" target="_blank">Efficient mapping of sulfated glycotopes by negative ion mode nanoLC-MS/MS-Based sulfoglycomic analysis of permethylated glycans.</a> <em>Anal Chem</em> 87(12): 6380-6388.</li><li>Cheng, P. F., Snovida, S., Ho, M. Y., Cheng, C. W., Wu, A. M. and Khoo, K. H. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23797909" target="_blank">Increasing the depth of mass spectrometry-based glycomic coverage by additional dimensions of sulfoglycomics and target analysis of permethylated glycans.</a> <em>Anal Bioanal Chem</em> 405(21): 6683-6695.</li><li>Hsiao, C. T., Wang, P. W., Chang, H. C., Chen, Y. Y., Wang, S. H., Chern, Y. and Khoo, K. H. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29066631" target="_blank">Advancing a High Throughput Glycotope-centric Glycomics Workflow Based on nanoLC-MS(2)-product Dependent-MS(3) Analysis of Permethylated Glycans.</a> <em>Mol Cell Proteomics</em> 16(12): 2268-2280.</li><li>Patnode, M. L., Cheng, C. W., Chou, C. C., Singer, M. S., Elin, M. S., Uchimura, K., Crocker, P. R., Khoo, K. H. and Rosen, S. D. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23880769" target="_blank">Galactose 6-O-sulfotransferases are not required for the generation of Siglec-F ligands in leukocytes or lung tissue.</a> <em>J Biol Chem</em> 288(37): 26533-26545.</li><li>Yu, S. Y., Hsiao, C. T., Izawa, M., Yusa, A., Ishida, H., Nakamura, S., Yagi, H., Kannagi, R. and Khoo, K. H. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30093410" target="_blank">Distinct substrate specificities of human GlcNAc-6-sulfotransferases revealed by mass spectrometry-based sulfoglycomic analysis.</a> <em>J Biol Chem</em> 293(39): 15163-15177. </li><li>Yu, S. Y., Snovida, S. and Khoo, K. H. (2006). <a href="https://bio-protocol.org/e3617" target="_blank">Permethylation and microfractionation of sulfated glycans for MS analysis.</a> <em>Bio-protocol</em> 10(10): e3617.</li><li>Yu, S. Y., Wu, S. W. and Khoo, K. H. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/168971786):%20355-369." target="_blank">Distinctive characteristics of MALDI-Q/TOF and TOF/TOF tandem mass spectrometry for sequencing of permethylated complex type N-glycans.</a> <em>Glycoconj J</em> 23(5-6): 355-369. <br/></li></ol>