<body>
            <h1>Flow Cytometry Analysis and Fluorescence-activated Cell Sorting of Myeloid Cells from Lung and Bronchoalveolar Lavage Samples from <em>Mycobacterium tuberculosis</em>-infected Mice</h1>
            <p><strong>Authors:</strong> Alissa C. Rothchild, Dat  Mai, Alan  Aderem, Alan H. Diercks</p>
            <h2>Abstract</h2>
            <p><em>Mycobacterium tuberculosis </em> (Mtb) is transmitted by aerosol and can cause serious bacterial infection in the lung that can be fatal if left untreated. Mtb is now the leading cause of death worldwide by an infectious agent. Characterizing the early events of <em>in vivo</em> infection following aerosol challenge is critical for understanding how innate immune cells respond to infection but is technically challenging due to the small number of bacteria that initially infect the lung. Previous studies either evaluated Mtb-infected cells at later stages of infection when the number of bacteria in the lung is much higher or used <em>in vitro</em> model systems to assess the response of myeloid cells to Mtb. Here, we describe a method that uses fluorescent bacteria, a high-dose aerosol infection model, and flow cytometry to track Mtb-infected cells in the lung immediately following aerosol infection and fluorescence-activated cell sorting (FACS) to isolate naÃ¯ve, bystander, and Mtb-infected cells for downstream applications, including RNA-sequencing. This protocol provides the ability to monitor Mtb-infection and cell-specific responses within the context of the lung environment, which is known to modulate the function of both resident and recruited populations. Using this protocol, we discovered that alveolar macrophages respond to Mtb infection <em>in vivo</em> by up-regulating a cell protective transcriptional response that is regulated by the transcription factor Nrf2 and is detrimental to early control of the bacteria.</p>
            <p><strong>Keywords:</strong> <em>Mycobacterium tuberculosis</em>, Pulmonary infection, Alveolar macrophages, Fluorescent bacteria, Lung, Bronchoalveolar lavage, Flow cytometry, Cell sorting</p>
            <h2>Background</h2><p> Aerosol transmission is a critical component of the natural cycle of <em>Mycobacterium tuberculosis</em> (Mtb) infection, contributing to the virulence of the bacteria and leading to a unique pattern of infection in the lung (North, 1995; Riley <em>et al.</em>, 1995; Pai <em>et al.</em>, 2016). In order to understand the early events of infection, it is critical to be able to track the cell types that become infected and isolate those populations for downstream analysis. Studies using a standard low-dose mouse model (~100 CFU deposition) have failed to detect bacteria earlier than 14 days following infection, a time point at which Mtb can be found distributed within alveolar macrophages, neutrophils, interstitial macrophages and dendritic cells (Wolf <em>et al.</em>, 2007; Huang <em>et al.</em>, 2018). By pairing a high-dose infection model with fluorescent bacteria, flow cytometry, and cell sorting techniques, this protocol enables the detection of Mtb-infected cells immediately following aerosol challenge and the ability to isolate Mtb-infected populations at different time-points as disease develops. While the high-dose infection model allows for very early detection of the first cells that are infected with Mtb and for evaluation of the early dynamics of Mtb-infected cells and bacterial dissemination, it may accelerate the adaptive stages of the immune response and disease progression compared to more commonly used low-dose models. Therefore, its utility is in studying the early stages of infection (within the first two weeks) and may not be as useful in studying later stages of disease. By combining this protocol with RNA-sequencing, we identified a cell protective transcriptional response generated by Mtb-infected alveolar macrophages that is regulated by the transcription factor Nrf2 and is detrimental to early control of the bacteria (Rothchild <em>et al.</em>, 2019). Other downstream applications include qPCR and <em>ex vivo</em> functional assays to characterize both infected and bystander cells. This approach is also applicable to other pulmonary infection models.</p>
        </body>

<p class="pptt" id="biaoti33301">## Materials and Reagents</p><ol class="npwtwok"><li>5 ml syringe (BD, catalog number: <a class="blackNoUnder" target="_blank"> 309646 </a> )</li><li>18 G x 1Â½ needle (BD, catalog number: <a class="blackNoUnder" target="_blank"> 305196 </a> )</li><li>15 ml conicals (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 525-1069 </a> )</li><li>50 ml conicals (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 505-1074 </a> )</li><li>70 Î¼m cell strainers (Fisherbrand, catalog number: <a class="blackNoUnder" target="_blank"> 22363548 </a> )</li><li>96-well plate, U-bottom (Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 353077 </a> )</li><li>1 ml Luer-Lok Tip syringes (BD, catalog number: <a class="blackNoUnder" target="_blank"> 309628 </a> )</li><li>20 gauge x 1-inch Introcan Safety IV catheter (Braun, catalog number: <a class="blackNoUnder" target="_blank"> 4252543-02 </a> )</li><li>Counting Beads: Polybead Polystyrene 15.0 Î¼m Microspheres (Polysciences, catalog number: <a class="blackNoUnder" target="_blank"> 18328 </a> )</li><li>GentleMACS M Tubes (Miltenyi Biotec, catalog number: <a class="blackNoUnder" target="_blank"> 130-093-236 </a> )</li><li>Middlebrook and Cohn 7H10 Agar Plates (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> B21174X </a> )</li><li>Mesh Nylon 23 x 23 cm (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 47746-106 </a> )</li><li>5 ml Polystyrene Round-Bottom Tubes, âFACS tubesâ (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 352054 </a> )</li><li>5 ml Polystyrene Round-Bottom Tubes with filter caps (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 253235 </a> )</li><li>GentleMACS C Tubes (Miltenyi Biotec, catalog number: <a class="blackNoUnder" target="_blank"> 130-093-237 </a> )</li><li>Mice: C57BL/6 (Jackson Laboratories, catalog number: 000<a class="blackNoUnder" target="_blank"> 664 </a> )</li><li>Fluorescent Mtb strain: mEmerald-H37Rv (non-commercially available, H37Rv background strain available from BEI, catalog number: <a class="blackNoUnder" target="_blank"> NR-123 </a> , see Procedure notes below for more information)</li><li>Hygromycin B (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 80501-072 </a> )</li><li>Liberase Blendzyme 3 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 5401020001 </a> )</li><li>DNase I (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 10104159001 </a> </li><li>RPMI (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 11875-093 </a> )</li><li>FBS (Peak Serum, catalog number: <a class="blackNoUnder" target="_blank"> PS-FB1 </a> )</li><li>70% ethanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 04-255-92 </a> )</li><li>PREempt RTU Disinfectant Solution (Contec, catalog number: <a class="blackNoUnder" target="_blank"> 21105 </a> )</li><li>PBS without calcium and magnesium (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 21-040-CV </a> )</li><li>ACK Lysing Buffer (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> A10492-01 </a> )</li><li>FcBlock, anti-CD16/32 (clone 2.4G2) (BD Pharmingen, catalog number: <a class="blackNoUnder" target="_blank"> 553141 </a> )</li><li>Zombie Violet Fixable Viability Kit (Biolegend, catalog number: <a class="blackNoUnder" target="_blank"> 423113 </a> )</li><li>Flow cytometry antibodies (also listed in Procedure section)<ol class="npwthree"><li>Anti-mouse Siglec F (clone E50-2440, BD Pharmingen, PE, catalog number: <a class="blackNoUnder" target="_blank"> 552126 </a> )</li><li>Anti-mouse CD45.2 (clone 104, Biolegend, PerCp-Cy5.5, catalog number: <a class="blackNoUnder" target="_blank"> 109828 </a> )</li><li>Anti-mouse CD64 (clone X54-5/7.1, Biolegend, APC, catalog number: <a class="blackNoUnder" target="_blank"> 139306 </a> )</li><li>Anti-mouse CD3 (clone 17A2, eBioscience, APC-eFluor780, catalog number: <a class="blackNoUnder" target="_blank"> 47-0032-82 </a> )</li><li>Anti-mouse CD19 (clone 1D3, eBioscience, APC-eFluor780, catalog number: <a class="blackNoUnder" target="_blank"> 47-0193-82 </a> )</li><li>Anti-mouse/human CD11b (clone M1/70, Biolegend, Brilliant Violet 570, catalog number: <a class="blackNoUnder" target="_blank"> 101233 </a> )</li><li>Anti-mouse I-A/I-E (MHC II) (clone M5/114.15.2, Biolegend, APC, catalog number: <a class="blackNoUnder" target="_blank"> 107614 </a> )</li><li>Anti-mouse CD11c (clone N418, Biolegend, Brilliant Violet 605, catalog number: <a class="blackNoUnder" target="_blank"> 117333 </a> )</li><li>Anti-mouse Ly6G (clone 1A8, Biolegend, Brilliant Violet 711, catalog number: <a class="blackNoUnder" target="_blank"> 127643 </a> )</li></ol></li><li>Paraformaldehyde, 20% Solution (Electron Microscopy Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 15713-S </a> )</li><li>Trizol (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 15596018 </a> )</li><li>Tween-80 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P1754 </a> )</li><li>HEPES (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> H4034 </a> )</li><li>MgCl<sub>2</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M2393 </a> )</li><li>KCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P3911 </a> )</li><li>NaCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S5886 </a> )</li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C3306 </a> )</li><li>NaOH (10 M) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 72068 </a> )</li><li>NaN<sub>3</sub> (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S2002 </a> )</li><li>Infection Buffer (see Recipes) </li><li>HEPES Buffer (see Recipes)</li><li>Lung Digestion Buffer (see Recipes)</li><li>CFU Dilution Buffer (sees Recipes)</li><li>FACS Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti33302">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>Centrifuge: capacity for 15 and 50 ml conicals and 96-well plates</li><li>Full Size Inhalation Exposure System (Model 099C A4212 with 7-1/2â x 6â stainless steel 5 basket system) (Glas-Col, catalog number: <a class="blackNoUnder" target="_blank"> 099C 0616700025 </a> )</li><li>Dissection scissors and forceps</li><li>Vannas Micro Scissors (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 82030-284 </a> )</li><li>gentleMACS Tissue Dissociator (Miltenyi Biotec)</li><li>BD LSR II (BD Bioscience), uses BD FACSDiva Software<br/>4-laser set-up: Violet (405 nm), Blue (488 nm), Green (532 nm), and Red (640 nm)</li><li>BD Aria (BD Bioscience), uses BD FACSDiva Software<br/>4-laser set-up: Violet (405 nm), Blue (488 nm), Green (532 nm), and Red (640 nm)</li><li>37 Â°C incubator</li><li>-80 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti33303">### Software</p><ol class="npwtwok"><li>BD FACSDiva (BD, <a href="https://www.bdbiosciences.com/en-eu" target="_blank">https://www.bdbiosciences.com/en-eu</a>)</li><li>FlowJo (Tree Star Inc., <a href="https://www.flowjo.com/" target="_blank">https://www.flowjo.com/</a>)<br/></li></ol><p class="pptt" id="biaoti33304">### Procedure</p><p class="ppzz"><em>Note: Purchase, maintenance, and experiments of mice requires prior approval by the Institutional Animal Care and Use Committee. In this case, protocols were approved by the Institutional Animal Care and Use Committee at Seattle Childrenâs Research Institute.<br/><br/></em> </p><ol class="npwone"><li>Preparation of <em>Mycobacterium tuberculosis</em> (Mtb)<br/><em>Notes: <ol class="npwthree"><li>All work with Mtb should be performed in BSL-3/ABSL-3 suites within biosafety cabinets as needed with appropriate PPE as designated by the facility manager and EHS department.</li><li>The mEmerald-H37Rv strain used here is not commercially available (generously provided by Chris Sassetti at UMass Medical School). The H37Rv background strain is available (BEI, catalog number: <a class="blackNoUnder" target="_blank"> NR-123 </a> ) and a fluorescent reporter strain can be generated by bacterial transformation by electroporation. The generation of a fluorescent strain of H37Rv is beyond the scope of this protocol, but similar reagents are reviewed by MacGilvary et al. (2018) and described in detail in a protocol by Goude et al. (2015).</li></ol></em> <ol class="npwtwo"><li>To achieve an initial dose of ~2,000-4,000 bacteria, thaw 10 x 1 ml vials of fluorescent marked Mtb (mEmerald-H37Rv). Bacteria were initially grown in 7H9 supplemented with 50 Î¼g/ml hygromycin B, frozen down in log phase growth at O.D. = 1.0 (without glycerol supplement), and stored at -80 Â°C.</li><li>Combine 10 ml Mtb in 15 ml conical and spin down in centrifuge at 1,850 <em>x g</em> for 10 min. Discard the supernatant and resuspend the bacterial pellet initially in 1 ml Infection Buffer (Recipe 1) with p1000 pipet tip to break up any clumps. Then, add an additional 4 ml Infection Buffer and vortex sample (for at least 10 s) until there are no visible clumps.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Mtb aerosol infection in Glas-Col inhalation Exposure System<br/><em>Note: For a detailed protocol of mycobacterial aerosol infection using the Glas-Col system, please refer to the following protocol: âAnimal Models of Mycobacteria Infectionâ (Ordway et al., 2011).</em><br/><ol class="npwtwo"><li>Load mice into compartments of Glas-Col stainless steel basket. Include at least 3 additional mice of the same age and strain for measuring the initial infectious dose (deposition).<br/><em>Note: See Section I: âBacterial burden deposition by colony forming unit assayâ for details on determining the infectious dose.</em> </li><li>Position the nebulizer on the outlets of the Glas-Col machine, checking for proper alignment.</li><li>Secure the nebulizer with the three metal clamps and tighten nut on each clamp to ensure seal. Place Teflon septum into screw cap and screw onto nebulizer firmly, making sure not to over-tighten.</li><li>Take up 5 ml Mtb culture with an 18 G needle attached to a 5 ml syringe. Bring syringe over to nebulizer and insert the needle into the septum. Slowly push the plunger and inject Mtb into the bottom of the nebulizer reservoir. Carefully remove syringe and place it in a sharps container within the biosafety cabinet.</li><li>Close the lid to the aerosol chamber, pressing firmly on each of the four latches until there is an audible click.</li><li>Turn main power switch on and set nebulization program as follows:<ol class="npwthree"><li>15 min Preheat.</li><li>40 min Nebulization.</li><li>40 min Cloud Decay.</li><li>15 min Decontamination Cycle.</li></ol></li><li>Press F1 to start run.</li><li>After nebulization cycle is completed, confirm less than 1 ml of liquid remains in the nebulizer. Detach nebulizer and immediately place nebulizer, screw top, and clamps in 100% PREempt bath to decontaminate for 30 min. Following decontamination, rinse all pieces with tap water three times and spray thoroughly with 70% ethanol to clean. Secure in a storage box once dry. </li><li>Open lid and remove mice from chamber. </li><li>Dissemble basket and decontaminate all removable parts, chamber, and lid with PREempt, followed by 70% ethanol. Reassemble basket.</li></ol><br/></li><li>Lung Tissue Harvest and Processing<ol class="npwtwo"><li>Lung tissue can be harvested anywhere from 12 h to 14 days following infection for analysis.</li><li>Euthanize mice by either CO<sub>2</sub> exposure or cervical dislocation. Because CO<sub>2</sub> exposure is known to perturb both Mtb and immune cell function (Ellner <em>et al.</em>, 1967; Howard <em>et al.</em>, 1990), cervical dislocation is preferred for transcriptional analysis or experiments characterizing immune cell function. Dissection serves as the required secondary physical method of euthanasia.</li><li>Place mouse in a supine position and dampen fur with 70% ethanol. Make initial incision just below the rib cage. Make a midline incision and retract skin above the head and below the thighs. Make an incision in the center of the peritoneal wall and cut to reveal diaphragm.</li><li>Open pulmonary cavity by cutting through diaphragm and up the sides of the rib cage without disturbing the lung tissue.</li><li>Collect all lung tissue in a gentleMACS C-Tube containing 5 ml freshly prepared Lung Digestion Buffer (Recipe 3).</li><li>Connect tubes to gentleMACS dissociator and run program âm_lung_1â.</li><li>Incubate tubes for 30 min at 37 Â°C for liberase digestion.</li><li>Reconnect tubes to gentleMACS dissociator and run program âm_lung_2â.</li><li>Pass digested lung through 70 Î¼m cell strainer into 50 ml conical. Use 3 ml syringe plunger to push any remaining tissue through the strainer. Rinse C-tube with RPMI and pass through strainer. Rinse strainer with additional RPMI for a total of around 40 ml of media added to original digestion buffer.</li><li>Centrifuge digested lung sample at 500 <em>x g</em> for 10 min at 4 Â°C. Discard supernatant and resuspend cell pellet.</li><li>(Optional) Add 1 ml ACK Lysing Buffer and incubate for 2 min at room temperature. Add RPMI with 10% FBS to halt lysis and centrifuge sample at 500 <em>x g</em> for 10 min at 4 Â°C. Discard supernatant.<br/><em>Note: Removing red blood cells from the sample can decrease the overall time of sample collection on the flow cytometer and increase the percentages of the cell populations of interest. However, if the lysis is not performed carefully, it can impact the overall viability of the cells.</em> </li><li>Resuspend pellet in 1 ml RPMI of FACS buffer or RPMI and pass through a second 70 Î¼m cell strainer. Transfer cells to 96-well U-bottom plate for flow cytometry antibody staining.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Bronchoalveolar Lavage (BAL) Cell Harvest<br/><em>Note: To isolate Mtb-infected alveolar macrophages, pre-enrich for airway-accessible cells, and avoid liberase digestion steps, perform bronchoalveolar lavage (BAL) instead of harvesting total lung tissue.</em> <ol class="npwtwo"><li>BAL can be harvested anywhere from 12 h to 14 days following infection for analysis.</li><li>Euthanize mice by cervical dislocation.</li><li>Place mouse in a supine position on top of a Styrofoam panel (single use) and dampen fur with 70% ethanol. Secure the mouse by pinning down limbs. In order to gain access to the trachea, it can be helpful to position the mouse so that the head extends just beyond the Styrofoam pad (Figure 1A).</li><li>Make an initial incision just below the rib cage. Carefully make a midline incision in the skin from the abdomen up to the mouth. Tear skin with forceps to reveal the thoracic and peritoneal areas. Make an incision in the center of the peritoneal wall and cut to reveal diaphragm (Figures 1B-1C).</li><li>Expose thoracic cavity by cutting through diaphragm and up the sides of rib cage without disturbing the lung tissue. Exposing the lung tissue allows for observation of the airways as they expand during the lavage and relieves pressure on the lobes to allow them to expand more freely during each lavage.</li><li>Cut up through the center of the submandibular salivary glands. Be careful not to nick the carotid arteries running parallel on either side. Then, carefully cut through sternothyroid muscle to expose the trachea (Figure 1D).</li><li>Make one tiny incision in the center of the trachea using microscissors to make a hole just large enough to thread the catheter (Figure 1E).</li><li>Attach a 1 ml syringe filled with cold PBS to the catheter and insert it into the trachea just far enough to be able to hold it securely. Slowly inject 1 ml PBS while watching lung inflate and then slowly and evenly pull syringe out (Figures 1F-1H). Do not continue to inject volume if resistance is felt prior to reaching the full 1 ml. Often, only 0.7-0.8 ml can be injected on the first wash (especially for younger or smaller mice). Repeat injection and retraction of liquid 3 times in total without removing catheter. Then, on the third retraction, collect liquid in 15 ml conical (often, the full 1 ml will not be recovered).<br/><em>Note: For reference, video examples of similar lavage procedures can be found in Van Hoecke et al. (2017) and Gereke et al. (2012). Note that in these protocols the catheter is tied to the trachea with suture thread, while we have found that inserting and removing the catheter without tying it down (Figures 1F-H), for repeated washes, is more efficient and leads to equivalent isolation of cells.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200514/20200514193620_4423.jpg" width="300"/><br/><strong>Figure 1. Bronchoalveolar lavage harvest using 20 gauge x 1-inch Introcan Safety IV catheter.</strong> A. Pin mouse in a supine position on top of Styrofoam. B. Cut skin to reveal the thoracic and peritoneal areas. C. Cut through the peritoneal wall and then cut diaphragm to expose lung cavity. D. Cut through submandibular salivary glands and sternothyroid muscle to expose trachea (Rib cage removed for photo clarity). E. Make a small incision in the center of the trachea using microscissors (white arrow). F. Thread catheter into trachea. G-H. Observe as lung expands upon injection of 1 ml PBS.<br/><br/></li><li>Repeat the 1 ml wash 3 additional times for a total of ~4 ml of BAL fluid recovered. Store sample on ice.</li><li>A single mouse yields ~0.5 x 10<sup>5</sup>-1 x 10<sup>5</sup> alveolar macrophages. BAL samples from multiple mice can be pooled at this point.</li><li>Pass BAL fluid through a 70 Î¼m cell strainer. This is a critical step despite the sample appearing very dilute, because the lavage process will collect bits of tissue that can clog a flow cytometer or cell sorter.</li><li>Centrifuge sample in a 15 ml conical at 500 <em>x g</em> for 10 min at 4 Â°C. Discard supernatant and resuspend cell pellet in 200 Î¼l RPMI or FACS buffer. Transfer cells to a 96-well U-bottom plate for flow cytometry antibody staining.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Flow cytometry antibody staining of cell suspensions<ol class="npwtwo"><li>For optimal antibody staining, include 1/4 of a lung sample or less per well from a naÃ¯ve mouse or from a mouse that has been infected for less than 10 days. Even less lung sample should be used per well from mice infected for more than 10 days, so as not to exceed 1 x 10<sup>6</sup>-2 x 10<sup>6</sup> cells/well. For staining of BAL samples, pool no more than 5-7 mice per well.</li><li>Centrifuge U-bottom plate at 500 <em>x g</em> for 4 min at 4 Â°C.</li><li>Resuspend samples in 50 Î¼l FACS Buffer with FcBlock at 1:100 dilution. Incubate for 10 min at 4 Â°C. Add 150 Î¼l PBS to wash and centrifuge plate at 500 <em>x g</em> for 4 min at 4 Â°C.</li><li>Resuspend samples in 50 Î¼l PBS with 1:200 Zombie Violet viability dye. Incubate in the dark for 10 min at 4 Â°C. Add 150 Î¼l FACS Buffer to wash and centrifuge plate at 500 <em>x g</em> for 4 min at 4 Â°C.</li><li>Resuspend samples in 50 Î¼l FACS Buffer with antibody staining cocktail. See below. <br/><em>Note: Make single color controls for compensation using cells from a digested naÃ¯ve lung. Include a naÃ¯ve lung sample to set all bacterial fluorescence gates and include fluorescence minus one (FMO) controls for all antibodies marked with * for which the correct location of the flow gating is not always obvious.</em> <ol class="npwthree"><li>Antibody staining cocktail for alveolar macrophage cell sorting:<br/><em>Note: All antibodies are from Biolegend unless otherwise noted. Antibodies are generally used at 1:100, but titration may be required.</em> <ol class="npwfour"><li>Siglec F (clone E50-2440)âPE (BD Pharmingen)</li><li>CD45 (clone 104)âPerCp-Cy5.5</li><li>CD64* (clone X54-5/7.1)âAPC</li><li>Dump channel: CD3 (clone 17A2), CD19 (clone 1D3)âAPC-eFluor 780 (eBioscience)</li><li>CD11b (clone M1/70)âBrilliant Violet 570</li><li>Leave FITC channel open for mEmerald, Brilliant Violet 421 open for Zombie Violet viability dye</li></ol></li><li>Antibody staining cocktail for flow cytometry analysis of lung myeloid populations:<br/><em>Note: All antibodies are from Biolegend unless otherwise noted.</em> <ol class="npwfour"><li>Siglec F (clone E50-2440)âPE (BD Pharmingen)</li><li>CD45 (clone 104)âPerCp-Cy5.5</li><li>CD64* (clone X54-5/7.1)âPE-Cy7</li><li>I-A/I-E (MHC II)* (clone M5/114.15.2)âAPC</li><li>Dump channel: CD3 (clone 17A2), CD19 (clone 1D3)âAPC-eFluor 780 (eBioscience)</li><li>CD11b (clone M1/70)âBrilliant Violet 570</li><li>CD11c* (clone N418)âBrilliant Violet 605</li><li>Ly6G (clone 1A8)âBrilliant Violet 711</li><li>Leave FITC channel open for mEmerald, Brilliant Violet 421 open for Zombie Violet viability dye</li></ol></li></ol></li><li>Incubate in the dark for 20 min at 4 Â°C. Add 150 Î¼l FACS Buffer to wash and centrifuge plate at 500 <em>x g</em> for 4 min at 4 Â°C.</li><li>After antibody staining:<ol class="npwthree"><li>For samples generated for flow cytometry analysis, resuspend samples in 100 Î¼l 2% paraformaldehyde. Incubate for 1 hour at room temperature and transfer sample to a new plate before removing from ABSL-3 suite.</li><li>For samples generated for cell sorting, resuspend samples in 150 Î¼l FACS buffer and transfer into FACS tubes. Keep tubes on ice while transferring to cell sorter.</li></ol><br/></li></ol></li><li>Counting bead assay<ol class="npwtwo"><li>To prepare samples for cell number analysis, combine 10 Î¼l sample with 100 Î¼l diluted counting beads (diluted to ~1.0 x 10<sup>5</sup>/ml, count final concentration using a hemocytometer) and 100 Î¼l 4% paraformaldehyde in a 96-well U-bottom.</li><li>Incubate for 1 h at room temperature and transfer sample to a new plate before removing from ABSL-3 suite.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell Sorting of Lung or BAL samples<ol class="npwtwo"><li>Immediately before sorting, filter samples through 35 Î¼m mesh nylon into new FACS tubes. Alternatively, samples can be filtered using FACS tubes with built-in filters, but in our experience for samples with low cell numbers, there is more sample loss using these.</li><li>Run single color controls to set voltages and compensation matrix.</li><li>Because the BAL samples are so precious with so few cells, it is critical to have the voltages, compensation, and gating established on control samples. It is useful to run both naÃ¯ve and Mtb-infected digested lung samples as controls prior to running the samples for sorting. Alveolar macrophages tend to be larger and more autofluorescent than many other myeloid cell types, so it is helpful to have a sample containing alveolar macrophages to set proper gates and voltages.</li><li>Set up gates as described in Figure 2.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200514/20200514193644_6553.jpg" width="300"/><br/><strong>Figure 2. Flow cytometry gating strategy for sorting alveolar macrophages from bronchoalveolar lavage samples.</strong> NaÃ¯ve, bystander and Mtb-infected AM populations were identified and sorted following using the following stains and antibodies: Zombie Violet, CD45, CD3/CD19, CD11b, Siglec F, and CD64.<br/><br/></li><li>Collect sorted cells in RPMI with 10% FBS. We have found that yield increases in 15 ml conicals compared to 5 ml FACS tube-style collection tubes. We collect as many cells as possible when sorting for RNA isolation. Pooling BAL from 10-12 mice typically yields around 1 x 10<sup>3</sup>-2 x 10<sup>3</sup> Mtb-infected AM and 1 x 10<sup>5</sup>-2 x 10<sup>5</sup> bystander AM.</li><li>Spin down sorted samples at 500 <em>x g</em> for 10 min at 4 Â°C.</li><li>Discard supernatant and resuspend samples in 1 ml TRIzol. Freeze sample at -80 Â°C overnight prior to RNA isolation.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Flow cytometry analysis of lung samples<ol class="npwtwo"><li>Run and collect single color control samples to set compensation matrix.</li><li>Collect as many events as possible in order to capture enough Mtb-infected cells for analysis, a minimum of 200,000 events, but ideally &gt; 1,000,000 events.</li><li>Collect at least 10,000-20,000 events for counting bead samples.<br/><em>Note: Forward scatter and side scatter voltages will need to be adjusted from those set for the cell sample acquisitions to accommodate the beads that show up with higher forward and side scatter values to ensure they are visible within the plots.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Bacterial burden deposition by colony forming unit assay<br/><em>Note: Because the infectious dose is relatively low, it is important to use the entire lung (all lobes) to estimate the bacterial deposition instead just of a single lobe.</em> <ol class="npwtwo"><li>Prepare GentleMACS M-tubes with 2 ml CFU Dilution Buffer.</li><li>Euthanize mice, isolate lung (as described above), and place lung in M-tube.</li><li>Connect tubes to gentleMACS dissociator and run program âProtein_01â.</li><li>From each sample, plate 200 ul of the following dilutions onto 7H10 plate:<ol class="npwthree"><li>neat sample (1:10 of total lung)</li><li>1:10 dilution (1:100 of total lung)</li><li>1:100 dilution (1:1,000 of total lung)</li></ol></li><li>Allow plates to dry for 5-10 min. Place plates upside down in 37 Â°C incubator for 21 days to allow colonies to grow, prior to enumeration.</li></ol></li></ol><p class="pptt" id="biaoti33305">### Data analysis</p><ol class="npwone"><li>Flow cytometry analysis<ol class="npwtwo"><li>Compensate samples in FlowJo using single color controls to generate compensation matrix.</li><li>Analyze data in FlowJo following lung myeloid gating schemes in Figure 3 (adapted from Misharin <em>et al.</em>, 2013 and Kopf <em>et al.</em>, 2015). Events are initially gated on FSC-A and SSC-A to exclude low FSC/SSC events consisting of cellular debris. Cells are then gated on FSC-A and FSC-H to exclude doublets.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200514/20200514193733_5254.jpg" width="300"/><br/><strong>Figure 3. Flow cytometry gating strategy for identifying Mtb-infected myeloid cell populations.</strong> A. Initial gating of Mtb-infected lung. B. (<em>blue outline</em>) Cell population gating strategy for all cells following doublet exclusion. (<em>green outline</em>) Cell population gating for mEmerald<sup>+</sup> Mtb-infected cells. C. Fluorescence minus one (FMO) and naÃ¯ve controls for key gates.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Counting bead analysis<ol class="npwtwo"><li>Set bead gating in FlowJo using gating scheme in Figure 4.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200514/20200514193751_0073.jpg" width="300"/><br/><strong>Figure 4. Flow cytometry gating for counting bead analysis</strong><br/><br/></li><li>Calculate cell number as follows:<ol class="npwthree"><li>Calculate <u>total # of beads</u> added per sample:<br/>For example: 1.1 x 10<sup>5</sup>/ml x 100 Î¼l = 1.1 x 10<sup>4</sup> beads</li><li>Calculate <u>multiplicative factor</u> of sample measured compared to total lung:<br/>For example: 10 Î¼l used from 1 ml total sample = multiplicative factor of 100</li><li>Calculate <u>% of beads</u> and <u>% of cells</u> in FlowJo gating.<br/>For example: 10% beads and 90% cell events</li><li>Calculate <u>total events</u> in the lung.<br/><em>Note: This number will be larger than the total number of cells, because it will include cellular debris recognized and counted as an event by the flow cytometer.</em><br/>(% cells/% beads) x (# of beads) x (multiplicative factor of sample)<br/>(90/10) x (1.1 x 10<sup>4</sup>) x (100) = 9.9 x 10<sup>6</sup> total events in the lung</li><li>Calculate <u>total number of cell population of choice</u> in the lung:<ol class="npwfour"><li>(Total events, see 2d) x (fraction of total for cell population) = (Total # in the lung)</li><li>For example, alveolar macrophages measured as 2.0% of total events in the lung: (9.9 x 10<sup>6</sup>) x (0.02) = 1.98 x 10<sup>5</sup> alveolar macrophages total in the lung</li></ol></li></ol></li></ol></li></ol><p class="pptt" id="biaoti33306">### Notes</p><ol class="npwtwok"><li>It is critical to keep the samples on ice for as much of the protocol as possible including during cell isolation, antibody staining, cell sorting, and post sorting if the cells will be used for transcriptional analysis or <em>ex vivo</em> functional assays.</li><li>Because the lung samples can be sticky, it is important to filter them immediately before sorting in addition to the double filtering that occurs during the tissue isolation to prevent clogging the cell sorter.<br/></li></ol><p class="pptt" id="biaoti33307">### Recipes</p><ol class="npwtwok"><li>Infection buffer (sterile filter)<br/>0.01% Tween-80<br/>Sterile H<sub>2</sub>O</li><li>HEPES buffer for lung digestion<br/><em>Note: All components from Sigma-Aldrich. Buffer can also be purchased directly.</em><br/>2.38 g HEPES (12.5 mM)<br/>0.203 g MgCl<sub>2</sub> (0.8 mM)<br/>0.373 g KCl (6.25 mM)<br/>8.77 g NaCl (0.187 M)<br/>0.265 g CaCl<sub>2</sub>Â·2H<sub>2</sub>O (2.25 mM)<br/>50 Î¼l NaOH (10 M)<ol class="npwthree"><li>Add all components to 800 ml dH<sub>2</sub>O in 2 L bucket. Combine with stir bar for 5-10 min until dissolved.</li><li>Adjust pH to 7.40 by adding NaOH in increments of 50 Î¼l.</li><li>Sterile filter and store at room temperature (storage up to 1 year) or 4 Â°C (storage up to 2 years). Check pH periodically.</li></ol><em>Note: The Lung Digestion Buffer should be pink in color. If the HEPES Buffer is at the wrong pH, it will produce an orange colored digestion buffer.</em> </li><li>Lung digestion buffer<br/><em>Note: Make fresh daily.</em><br/>70 Î¼g/ml Liberase Blendzyme 3<br/>30 Î¼g/ml DNase I<br/>Dissloved in HEPES Buffer (see above)</li><li>CFU Dilution Buffer (sterile filter)<br/>0.05% Tween-80<br/>PBS</li><li>FACS Buffer (store at 4 Â°C)<br/>1% FBS<br/>0.01% NaN<sub>3</sub><br/>PBS<br/></li></ol><p class="pptt" id="biaoti33308">### Acknowledgments</p><p class="ppzz">This work was supported by the National Institute of Allergy and Infectious Disease of the NIH under awards U19AI106761, U19AI135976, U19AI100627, and R01AI032972 (to A.A). The mEmerald bacterial strain was generously provided by C. Sassetti and C. Baer at the University of Massachusetts Medical School. This protocol was developed and reported in a previous publication (Rothchild <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti33309">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti33310">### Ethics</p><p class="ppzz">All experiments were performed in compliance with the Institutional Animal Care and Use Committee at Seattle Childrenâs Research Institute. under IACUC00510 (Activity ID TR00053) valid 6/11/2016-6/9/2022.</p><p class="pptt" id="biaoti33311">### References</p><ol class="npwtwok"><li>Ellner, P. D., Elbogen, S. and Frankel, H. (1967). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4959427" target="_blank">A simplified method for providing increased carbon dioxide tension in cultures for acid-fast bacilli.</a> <em>Am Rev Respir Dis</em> 95(1): 107-108.</li><li>Gereke, M., Autengruber, A., GrÃ¶be, L., Jeron, A., Bruder, D. and Stegemann-Koniszewski, S. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/23287741/" target="_blank">Flow Cytometric Isolation of Primary Murine Type II Alveolar Epithelial Cells for Functional and Molecular Studies.</a> <em>J Vis Exp</em> (70): e4322.</li><li>Goude, R., Roberts, D. M. and Parish T. (2015). <a href="https://pubmed.ncbi.nlm.nih.gov/25779313/" target="_blank">Electroporation of mycobacteria.</a> <em>Methods Mol Biol</em> 1285:117-130.</li><li>Howard, H. L., McLaughlin-Taylor, E. and Hill, R. L. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2170752" target="_blank">The effect of mouse euthanasia technique on subsequent lymphocyte proliferation and cell mediated lympholysis assays.</a> <em>Lab Anim Sci</em> 40(5): 510-514.</li><li>Huang, L., Nazarova, E. V., Tan, S., Liu, Y. and Russell, D. G. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29500179" target="_blank">Growth of <em>Mycobacterium tuberculosis in vivo</em> segregates with host macrophage metabolism and ontogeny.</a> <em>J Exp Med</em> 215(4): 1135-1152.</li><li>Kopf, M., Schneider, C. and Nobs, S. P. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25521683" target="_blank">The development and function of lung-resident macrophages and dendritic cells.</a> <em>Nat Immunol</em> 16(1): 36-44.<br/></li><li>MacGilvary, N. J. and Tan, S. (2018). <a href="https://pubmed.ncbi.nlm.nih.gov/29718182/" target="_blank">Fluorescent <em>Mycobacterium tuberculosis</em> reporters: illuminating host-pathogen interactions.</a> <em>Pathog Dis</em> 76(3): fty017. doi: 10.1093/femspd/fty017.</li><li>Misharin, A. V., Morales-Nebreda, L., Mutlu, G. M., Budinger, G. R. and Perlman, H. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23672262" target="_blank">Flow cytometric analysis of macrophages and dendritic cell subsets in the mouse lung.</a> <em>Am J Respir Cell Mol Biol</em> 49(4): 503-510.</li><li>North, R. J. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7594715" target="_blank"><em>Mycobacterium tuberculosis</em> is strikingly more virulent for mice when given via the respiratory than via the intravenous route.</a> <em>J Infect Dis</em> 172(6): 1550-1553.</li><li>Ordway, D. J. and Orme I. M. (2011). <a href="https://pubmed.ncbi.nlm.nih.gov/21809318/" target="_blank">Animal models of mycobacteria infection.</a> <em>Curr Protoc Immunol</em> Chapter 19: Unit19.5. doi: 10.1002/0471142735.im1905s94.</li><li>Pai, M., Behr, M. A., Dowdy, D., Dheda, K., Divangahi, M., Boehme, C. C., Ginsberg, A., Swaminathan, S., Spigelman, M., Getahun, H., Menzies, D. and Raviglione, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27784885" target="_blank">Tuberculosis.</a> <em>Nat Rev Dis Primers</em> 2: 16076.</li><li>Riley, R. L., Mills, C. C., Nyka, W., Weinstock, N., Storey, P. B., Sultan, L. U., Riley, M. C. and Wells, W. F. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7785671" target="_blank">Aerial dissemination of pulmonary tuberculosis. A two-year study of contagion in a tuberculosis ward.</a> 1959. <em>Am J Epidemiol</em> 142(1): 3-14.</li><li>Rothchild, A., Olson, G., Nemeth, J., Amon, L., Mai, D., Gold, E., Diercks, A. and Aderem, A. (2019). <a href="https://immunology.sciencemag.org/content/4/37/eaaw6693" target="_blank">Alveolar macrophages generate a noncanonical NRF2-driven transcriptional response to <em>Mycobacterium tuberculosis in vivo</em>.</a> <em>Sci Immun</em> 4: eaaw6693. </li><li>Van Hoecke, L., Job, E. R., Saelens, X. and Roose, K. (2017). <a href="https://pubmed.ncbi.nlm.nih.gov/28518083/" target="_blank">Bronchoalveolar Lavage of Murine Lungs to Analyze Inflammatory Cell Infiltration.</a> <em>J Vis Exp</em> (123): e55398.</li><li>Wolf, A. J., Linas, B., Trevejo-Nunez, G. J., Kincaid, E., Tamura, T., Takatsu, K. and Ernst, J. D. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17675513" target="_blank"><em>Mycobacterium tuberculosis</em> infects dendritic cells with high frequency and impairs their function <em>in vivo</em>.</a> <em>J Immunol</em> 179(4): 2509-2519.<br/></li></ol>