<body>
            <h1>RNA Extraction from Ears and Draining Lymph Nodes of Mice Infected with <em>Leishmania amazonensis</em></h1>
            <p><strong>Authors:</strong> Emilie Giraud, Evie Melanitou</p>
            <h2>Abstract</h2>
            <p>Parasites of the genus <em>Leishmania</em> infect the mammalian hosts, including mice and humans and cause cutaneous or visceral leishmaniasis depending upon the parasite species transmitted by the vector sandfly. <em>Leishmania amazonensis</em> is one of the <em>Leishmania</em> species responsible for the cutaneous form of the disease. We have inoculated with these parasites the ear dermis of mice. RNA preparations were performed from fragmented tissues using a buffer containing guanidin isothiocynate (RLT buffer, RNeasy Mini Kit, Qiagen, SAS, France) and Î²-mercaptoethanol. Both reagents facilitate the isolation of intact RNA from tissues and the use of the RNeasy Kits present with several advantages that facilitate the isolation of pure non-degraded total RNA: i) This method allows to avoid the presence of phenol in the RNA extraction buffer, commonly used in alternative protocols; ii) Moreover Diethylpyrocarbonate (DEPC) treatment of glassware, to avoid RNAses contamination of the samples, is not required with this protocol; iii) Finally, it is a fast procedure and the isolated total RNA may be concentrated in a small volume thus facilitating its use for downstream experimental procedures.</p>
            <p><strong>Keywords:</strong> RNA isolation, Ear pinna dermis, Lymph nodes, <em>Leishmania amazonensis</em>, Mouse</p>
            <h2>Background</h2><p>Pathogens of the genus <em>Leishmania</em> cause vector born zoonotic diseases with a variety of clinical manifestations and a variability in disease severity (Mansueto <em>et al.</em>, 2007). The cutaneous form of leishmaniasis is caused by several <em>Leishmania</em> species including <em>Leishmania amazonensis</em> (McMahon-Pratt and Alexander, 2004; Christensen <em>et al.</em>, 2019).<br/><br/>The existence of mouse models, with high, intermediate or no susceptibility to the cutaneous form of leishmaniasis have contributed to the development of experimental protocols for parasite dissemination (de Oliveira Cardoso <em>et al.</em>, 2010). The parasites are naturally transmitted to the mammalian hosts by the bite of infected phlebotomine sandflies of the genus <em>Lutzomyia</em> (Bates, 2007). Thus transmission protocols use the ear dermis, the dorsal skin or the hind footpad (Schuster <em>et al.</em>, 2014). In our protocol the <em>Leishmania amazonensis</em> parasites were inoculated through the ear dermis of the mouse, a method that closely imitates the natural transmission of the parasites by the sandfly (Giraud <em>et al.</em>, 2019b). In addition, the ear pinna is appropriate for phenotypic evaluation of the lesion as well as for the isolation of nucleic acids from the inoculation sites, for two reasons: 1) parasites, in their infectious path, induce two distinct phases: the first, is a clinically silent phase characterized by the absence of lesions and the increase of parasite load and the second phase, during which visible lesions develop, and are associated with immune cell infiltration at the site of inoculation (Liu and Uzonna 2012; Giraud <em>et al.</em>, 2019a). 2) The immune cells expand in the draining lymph nodes, which are also used for nucleic acids preparation (Cortes <em>et al.</em>, 2010). Therefore the immune response of the host to the parasites can be studied on the site of inoculation and on the secondary immune adjacent tissue, the ear draining lymph nodes.<br/><br/><em>Leishmania amazonensis</em> infection triggers an inflammatory response characterized by the presence of macrophages on the lesion sites. The parasites develop in the resident dermal macrophages (MF) and in Dendritic cells (DC) (Liu and Uzonna, 2012). The mechanisms of the parasite adaptation in the mammalian phagocytes and tissues remain elusive. Macrophages do not prevent the proliferation of the infection but on the contrary are the resident cells for these parasites.<br/><br/>We have studied the immune response of the host to <em>Leishmania amazonensis</em> infection in mice (Giraud <em>et al.</em>, 2019b). Parasites transmitted through the ear dermis induce the immune response of the adjacent lymph nodes. RNAs prepared from the ear lesions and the lymph nodes were used to address the immune-related gene expression prior and after infection with the parasites. Several methods exist for preparation of ribonucleic acids. The classical protocols use guanidium isothiocyanate, a chaotropic agent that disintegrates the cellular structures and dissociates rapidly the nucleoproteins from the nucleic acids (Reid, 1991). RNases are also inactive in the presence of 4 M guanidium thiocyanate. Beta-mercaptoethanol (Î²-ME) is added to reduce RNA degradation by RNases. These protocols require disposable sterile glassware and plastic ware for the preparation of non-degraded RNA. Often pretreatment with diethylpyrocarbonate (DEPC) (0.1%) is used to treat glassware for 12 h at 37 Â°C and then heating at 100 Â°C for 15 min to remove any traces of diethylpyrocarbonate. The cellular proteins are removed from the nucleic acids preparations by extractions with phenol/chloroform. Ethanol precipitation of the isolated RNAs is then required and the nucleic acids are obtained by precipitation in the presence of 0.3 M sodium acetate. Our method benefits from the existence of a simplified protocol provided by the Qiagen kits. The methodological advantages of this protocol are as follows: 1) the method is simplified and quick, 2) there is no need to use DEPC, 3) phenol extraction of the nucleic acids from the proteins is not required, 4) the silica columns for separation of the nucleic acids from the cells or tissues constituents are very efficient, 5) RNAs are at a correct concentration and ethanol precipitation is not necessary and finally this method yields a high quality of RNAs. Whilst accompanied protocols are provided by the Qiagen kits, the additional details in the description of our method, applied to the mouse ear and lymph nodes, completes the methodology of the corresponding published article (Giraud <em>et al.</em>, 2019b), presents with an interest for the isolation of pure non-degraded RNAs and may be useful to other investigators.</p>
        </body>

<p class="pptt" id="biaoti34269">## Materials and Reagents</p><ol class="npwtwok"><li>60 x 15 mm, Cell culture-treated Permanox Petri Dish sterile (Thermo Fisher Scientific, Nunc<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 150340 </a> )</li><li>Pipette Barrier (Filter) tips (Certified Free for RNases, DNases, sterile) (Thermo Fisher Scientific, Invitrogen, catalog numbers: <a class="blackNoUnder" target="_blank"> AM126-45 </a> (10 Î¼l), <a class="blackNoUnder" target="_blank"> AM126-35 </a> (20 Î¼l), <a class="blackNoUnder" target="_blank"> AM126-50 </a> (200 Î¼l), <a class="blackNoUnder" target="_blank"> AM126-60 </a> (1,000 Î¼l)</li><li>2 ml and 1.5 ml tubes (Eppendorf, Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 3810X </a> )</li><li>2 ml hard tissue homogenizing tubes, containing 2.8 mm ceramic (zirconium oxide) beads CK28 (Bertin Instruments, catalog number: <a class="blackNoUnder" target="_blank"> P000911-LYSK0-A </a> , for Precellys tissue homogenizer, see Equipment 1)</li><li>Glass Pasteur Pipettes (Merck, catalog number: <a class="blackNoUnder" target="_blank"> CLS7095D5X </a> )</li><li>Adult C57BL/6J and B6(Cg)-Spp1t<sup>m1Blh</sup>/J mice (8-12 weeks of age)</li><li>Phosphate buffered saline (PBS BioPerformance Certified, pH 7.4, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P5368-10PAK </a> , storage at room temperature)</li><li>Î²-Mercaptoethanol (Î²-ME), Molecular Biology Grade (Millipore, Merck, Calbiochem, catalog number: <a class="blackNoUnder" target="_blank"> 444203-250ML </a> or alternatively dithiothreitol (DTT) may replace Î²-ME [Mommaerts <em>et al.</em>, 2015])</li><li>RLT (RNA lysis buffer, containing guanidine thiocyanate) (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 79216 </a> )</li><li>RW1 (RNA wash buffer containing ethanol) (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 74134 </a> )</li><li>RPE (RNA wash buffer) (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 74134 </a> )</li><li>RNeasy<sup>R</sup> Plus Mini Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 74134 </a> )</li><li>Alternatively the Qiagen RNeasy Kit not including the gDNA separation columns may be used (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 75154 </a> )</li><li>RNase-free DNase Set (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 79254 </a> )</li><li>Buffers and separation columns (RNeasy Plus Mini Kit) (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 74134 </a> )</li><li>Ethanol (CH<sub>3</sub>CH<sub>2</sub>OH) (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 02851-1L </a> )</li><li>Ethanol 70% in H<sub>2</sub>O (see Recipes)</li><li>RLT + Î²-ME or instead RLT-DTT buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34270">### Equipment</p><ol class="npwtwok"><li>Adjustable Pipettes (Sartorius Biohit Proline France), or Micropipettes PIPETMAN L (GILSON, catalog number: <a class="blackNoUnder" target="_blank"> 070053-070058 </a> )</li><li>Reagent Vessel (Sartorius, catalog number: <a class="blackNoUnder" target="_blank"> 783500 </a> )</li><li>Surgical instruments (including a pair of scissors, two pair of fine scissors, and two tweezers) (Figure 1i-iv)</li><li>Mice dissecting table and dissecting glass plate (Figure 1v).</li><li>Precellys 24 System tissue homogenizer (Bertin Technologies, catalog number: <a class="blackNoUnder" target="_blank"> P000669-PR240-A.0 </a> )</li><li>Precellys<sup>Â®</sup> 2 ml Hard Tissue homogenizing ceramic beads kit (CK28) (Bertin Instruments, catalog number: <a class="blackNoUnder" target="_blank"> 10011151 </a> , 230 Rockville, MD 20850 USA)</li><li>4 Â°C fridge and -20 Â°C freezer</li><li>Centrifuge Eppendorf 5415R or 5424R (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5404000410 </a> , Radius 8.3 cm)</li><li>NanoDrop 1000 Spectrophotometer (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ND-1000 </a> ) (Schroeder <em>et al.</em>, 2006)<br/><br/><img alt="" class="layerphoto" height="231" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602131342_8904.jpg" width="300"/><br/><strong>Figure 1. Instruments used for tissues dissection and lymph node isolation.</strong> i. Scissors; ii. Curved scissors; iii. Tweezers; iv. Fine scissors; v. Tissue dissection glass plate.<br/></li></ol><p class="pptt" id="biaoti34271">### Procedure</p><p class="ppzz">Special safety conditions should be applied when working with <em>Leishmania</em> spp. parasites in the laboratory. The main route of exposure is through the use of needles for infecting the ears of the animals. Therefore protective measures should be taken in consideration, <em>i.e.</em>, wearing gloves, needle precautions and working under a hood in a P2 plus laboratory. All instruments used for dissection of the tissues should be placed in alcohol 70% recipients and then washed in the presence of chlorine and sterilized.</p><br/><ol class="npwone"><li>Preparation<ol class="npwtwo"><li>Prepare 2 ml hard tissue homogenizing tubes containing the ceramic beads and label with the samples identification.</li><li>Prepare RLT-Î²-ME buffer (see Recipes).</li><li>Complement RPE wash buffer with 4 volumes of ethanol (100%).</li><li>Prepare a dilution of 70% Ethanol using sterile nuclei acids-free water.</li><li>Label the tubes to be used for the next steps of RNA isolation from the fragmented tissues.</li><li>Prepare cell culture Petri dish and add 3 ml PBS.<br/><br/></li></ol></li><li>Dissection of tissues<ol class="npwtwo"><li>Adult C57BL/6J and B6(Cg)-Spp1t<sup>m1Blh</sup>/J mice at various ages selected for the experiments are anesthetized with CO<sub>2</sub>.</li><li>After washing with Ethanol and subsequently with PBS the areas from which the tissues will be isolated, cut the infected with the parasites ear (left ear) and discard the surrounding tissue of the infected area. Isolate a similar area from the right ear (non-infected) to be used for control (Figure 2).<br/><br/><img alt="" class="layerphoto" height="128" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602131359_8908.jpg" width="300"/><br/><strong>Figure 2. Sites of inoculation with <em>Leishmania amazonensis</em> parasites of the ear dermis of C57BL/6J mice.</strong> A. Ear lesions are shown on the infected (INF) ear pinna, (<strong>i</strong> and <strong>ii</strong>). Control non-infected ear pinna, (<strong>iii</strong>). Circles show the ear areas isolated for total RNA preparation. B-C. Hematoxylin and Eosin (H&amp;E) staining of histology sections of ear pinna from C57BL/6J mice. B. Infected with <em>Leishmania amazonensis</em> (INF): a: epidermis with acanthosis; b: inflamed dermis; c: cartilage. C. Non infected (Non INF) ears: a: epidermis; b: dermis; c: cartilage. D. Bioluminescence imaging of luciferase expressing <em>Leishmania amazonensis</em> parasites in infected ear dermis of mice. The color bar indicates the density of the luciferase emission delineating the number of parasites present in the infected area of the ear pinna.<br/><br/></li><li>Wash rapidly each tissue fragment with PBS and place it in separate homogenizing tubes.</li><li>After shaving the hair, cut through the skin at the area on the side of each ear and isolate the ear draining lymph nodes from both sides (left side infected and right side to be used as control). Clean the lymph nodes from the surrounding tissue using the glass dissection dish (Figure 1vi).</li><li>Place the lymph node in a new homogenizing tube.</li><li>The dissection procedure of the tissues is described on Figure 3.<br/><br/><img alt="" class="layerphoto" height="209" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602131415_0702.jpg" width="300"/><br/><strong>Figure 3. Step by step procedure for isolation of tissues from mice.</strong> Mice, after various times of infection with <em>Leishmania amazonensis</em> parasites at the ear pinna, were sacrificed (1). The infected area of the ear tissue is excised and after a brief wash with PBS in a Petri dish plate, the ear fragment is placed in a Precellys 24 tube containing the beads and the RLT buffer (2). The auricular, ear draining lymph node is surgically removed and placed in a new tube containing the beads and the RLT buffer (3). The lymph node is dense and sinks into the tube liquid. The tissues are homogenized in the Precellys 24 homogenizer (4) and then the homogenate is placed into two new Eppendorf 2 ml tubes for total RNA preparation (5). All procedures involving animals have been approved by the Institutional Committees on animal Welfare under strict accordance with the European guidelines.<br/><br/></li></ol></li><li>RNA isolation from tissues<br/><em>Note: The description of our procedure for Total RNA preparation follows the protocol of RNA isolation with the RNeasy Plus Mini Kit as supplied by Qiagen with minor modifications. Our starting material was always one ear fragment or one lymph node per preparation. These tissues yielded approximately 10-50 Î¼g of total RNA, depending on the tissue. 1 Î¼g of total RNA yields 1 Î¼g of cDNA. From these cDNA preparations we used 10 ng per Real time PCR reaction.</em><br/><ol class="npwtwo"><br/><li>Prepare the tubes that will contain the tissues by adding 1 ml of RLT + Î²-ME buffer for the ear fragment and 400 Î¼l of the same buffer for the lymph nodes.</li><li>Add the tissues into the tubes and proceed in tissue fragmentation in the Precellys 24 apparatus. Tissues were homogenized at 600 <em>x g</em> for 60 s, in accordance with <a href="https://www.youtube.com/watch?v=uMT60IHYDuQ" target="_blank">the manufacturerâs instructions</a>.</li><li>Take out the tubes from the Precellys 24 and centrifuge at 15,000 <em>x g</em> (13,000 rpm), at room temperature for 2 min.</li><li>Place 250 Î¼l from the ear homogenate and the entire lymph node homogenate (400 Î¼l) on separate for each tissue DNA retention columns (gDNA Eliminator Spin Column, Qiagen). Centrifuge at 9,000 <em>x g</em> (10,000 rpm) for 90 s.</li><li>Keep the effluent and discard the DNA separation tubes.</li><li>Add 175 Î¼l of ethanol 70% into the tubes containing the ear homogenate and 280 Î¼l into the tubes containing the lymph nodes. Use a P1000 Pipetman to well homogenize the samples.</li><li>Transfer the samples on the RNA retention column (RNeasy spin column, Qiagen) and centrifuge at 9000 <em>x g</em> (10,000 rpm) for 90 s.</li><li>Eliminate the tube containing the effluent and place the column on a new collection tube (1.5 ml). If DNase treatment is not used continue as in Step C10.</li><li>If DNase digestion is performed (with Qiagen kit for RNA preparation), samples (Step C4) are placed directly on the RNA retention columns and additional Steps D1 to 4 are followed.</li><li>Add on the column 700 Î¼l of RW1 RNA binding buffer (Qiagen) and centrifuge at 9,000 <em>x g</em> (10,000 rpm) for 60 s. Eliminate the effluent but use for the next step the same collector tube.</li><li>Add on the column 500 Î¼l of RPE + ethanol buffer (ensure that it is at room temperature).</li><li>Centrifuge the samples at 9,000 <em>x g</em> (10,000 rpm) for 60 s. Eliminate the effluent and reuse the collector tube.</li><li>Repeat Steps C10 and C11 once more and eliminate the collector tube.</li><li>Place the column onto a new collector tube and centrifuge at 9,000 <em>x g</em> (10,000 rpm) for 2 min to dry out completely the column.</li><li>Repeat the centrifugation step once more at 15,000 <em>x g</em> (13,000 rpm) for 2 min.</li><li>Replace collector tube with a 1.5 ml Eppendorf new tube and proceed to the RNA elution step by adding 30 Î¼l of RNase/DNase-free sterile water on the center of the column for each sample.</li><li>Centrifuge at 15,000 <em>x g</em> (10,000 rpm) for 60 s.</li><li>Dilute 1.5 Î¼l of each RNA sample in 20 Î¼l of sterile water in a new tube.</li><li>1 Î¼l of each diluted sample is used for RNA quantification and quality evaluation, by measurements of the Optical Density (OD) in a NanoDrop <a class="blackNoUnder" target="_blank"> ND-1000 </a> micro spectrophotometer (Thermo Fisher Scientific) (Figure 4).</li><li>Keep all RNAs at -80 Â°C until use.<br/><br/><img alt="" class="layerphoto" height="318" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602131433_7790.jpg" width="300"/><br/><strong>Figure 4. Total RNA concentration and quality evaluation.</strong> Representative images of Optical Densities (OD) of RNAs measured by the NanoDrop <a class="blackNoUnder" target="_blank"> ND-1000 </a> micro spectrophotometer. Non-degraded RNA preparation (Sample 1, upper panel) and poor quality (degraded) RNA preparation (Sample 2, lower panel) are shown. The wavelengths (Î») of 260 nm for nucleic acid (solid blue lines) and 280 nm for protein (dashed blue lines) are indicated. The ratios of absorbance A<sub>260/280</sub> (~2.0) and 260/230 (â¥ 2.0) indicate the purity of the RNA (ex. absence of organic compounds and protein contaminants respectively). RNAs concentrations are given in ng/Î¼l, taking in consideration 1 OD A<sub>260</sub> unit of ssRNA = 40 Î¼g/ml. Low A<sub>260/280</sub> concentrations (lower panel) indicate residual contamination of reagents in the preparation and absence of a clear pick of absorbance at A<sub>260</sub> indicates a poor quality of RNA (see Note 2).<br/><br/></li></ol></li><li>DNase treatment of the samples (Qiagen kit cat. NÂ° <a class="blackNoUnder" target="_blank"> 79254 </a> )<ol class="npwtwo"><li>Add to the RNeasy spin column 350 Î¼l Buffer RW1, close the lid gently and centrifuge at 8,000 <em>x g</em>, for 15 s to wash the spin column membrane. Discard the flow-through.</li><li>Prepare 70 Î¼l buffer RDD (supplied with the RNase-free DNase Set) in a tube and add 10 Î¼l DNase I stock solution. Mix gently as DNase is sensitive to physical denaturation. Do not vortex.</li><li>Add 80 Î¼l of the DNase I incubation mix directly into the membrane of the RNeasy spin column and leave on the benchtop for 15 min (20-30 Â°C).</li><li>Add 350 Î¼l buffer RW1 to the RNeasy spin column, close the lid gently and centrifuge at 8,000 <em>x g</em> for 15 s. Discard the flow-through and continue with the first RPE buffer wash (Step C11).<br/></li></ol></li></ol><p class="pptt" id="biaoti34272">### Data analysis</p><p class="ppzz">According to our protocol we have isolated RNAs from animals infected and non-infected with the <em>Leishmania amazonensis</em> parasites. Our method allowed for the preparation of good quantity, non-degraded total RNA sufficient for RT-PCR experiments of several genes (Giraud <em>et al.</em>, 2019b).<br/>  The data obtained from these preparations are described on Figure 6d and Figure S8 in the original published article (Giraud <em>et al.</em>, 2019b).<br/>  As mentioned above on section C (RNA isolation from tissues), we have taken in consideration that 1 Î¼g of RNA yields 1 Î¼g of cDNA. For the cDNA preparation we used random hexamers (Roche Diagnostics) and MMLTV-RT reverse transcriptase (Invitrogen, Life technologies). 10 ng of cDNA was used per RT-PCR reaction (with QuantiTect SYBR Green Kit, Qiagen) as described in the Methods section of the original article (Giraud <em>et al.</em>, 2019b). Raw Crossing Point (<em>Cp</em>) values were used as input for the qBase software program (<a href="https://www.qbaseplus.com/" target="_blank">https://www.qbaseplus.com/</a>). Calculation of relative quantification values are corrected by appropriate normalization using the qPCR quantification values of the house keeping genes. This normalizations were performed for each experimental tissue. qBase software can calculate the normalized and rescaled quantities in a logarithmic scale. This is an automated process supplied by the integrated algorithm of the qBase program as described by Hellemans <em>et al.</em> (2007).<br/>  In our experiments described on Figure 6 and Figure S8 of the manuscript (Giraud <em>et al.</em>, 2019b), the relative expressions (quantities) of the <em>opn</em> gene and the genes encoding for the inflammasome multicomplex were calculated on log-transformed ratios by comparison of the fold differences in expression of the <em>Leishmania</em> infected tissues in the osteopontin mutant mice (C57BL/6<sup>-/-</sup>) <em>versus</em> the wild type osteopontin positive (C57BL/6<sup>+/+</sup>) mice.</p><p class="pptt" id="biaoti34273">### Notes</p><ol class="npwtwok"><li>The following formula:<br/><strong>g Force (RCF) = (rpm)<sup>2</sup> x 1.118 x 10<sup>-5</sup> x r</strong> is used for the conversion of rpm (revolutions per minute) to g Force or Relative Centrifugal Force (RCF). RCF or g is the acceleration applied to a sample during centrifugation (see: <a href="https://www.promegaconnections.com/converting-rpm-to-g-force-rcf-and-vice-versa-2/" target="_blank">details</a>).<br/>  This conversion is dependent on the radius r of the rotor.</li><li>RNA concentration and quality are important for the optimum performance of reactions using this nucleic acid. The average concentration of RNA and its purity is measured by spectrophotometric quantification. In the NanoDrop spectrophotometer is included a preset program for RNA concentrations measurements, that automatically reads the OD at 230, 260 and 280 nm and calculates the above discussed ratios (Figure 4). In addition only 1 to 2 Î¼l of sample are required. A Blank is used with the same solution in which the RNA is diluted prior to RNA measurements with the spectrophotometer. RNA has a maximum absorbance (Optical Density) at 260 nm and the absorbance unit used as conversion factor for RNA concentration is A<sub>260</sub> ssRNA = 40 Î¼g/ml. For the evaluation of RNA purity OD<sub>260</sub> and OD<sub>230</sub> measurements also should be taken in consideration. A<sub>260/280</sub> ratio of ~2 is accepted for pure RNA. Low 260/280 ratios usually indicates the presence of protein contaminants. A high pick at A<sub>230</sub> absorbance indicates the presence of other than proteins contaminants such as residual phenol, guanidine or other reagents. A<sub>260/230</sub> ratio values for pure RNA are often higher that the respective A<sub>260/280</sub> values. Commonly 260/230 ratios are in the range of 2.0-2.2. Lower than 1.5 ratios indicate the presence of contaminants. RNA integrity can be checked by a run of the RNA on a 1% agarose gel and assess for the integrity of the ribosomal RNA bands. For non-degraded RNA the intensity of the 28S band (upper rRNA band) in eukaryotic cells should be twice the intensity of the lower rRNA band (18S). Both bands should be clear and tight, otherwise the presence of a smear indicates RNA degradation. The presence of high molecular weight bands shows contamination by the DNA. Another possible method for checking RNA integrity is by using the Agilent Bioanalyzer (See <a href="https://www.agilent.com/cs/library/applications/5989-1165EN.pdf" target="_blank">Application Notes</a>). This analyzer measures the sizes of the rRNA bands and determines by comparison a RNA integrity number (RIN).<br/></li></ol><p class="pptt" id="biaoti34274">### Recipes</p><ol class="npwtwok"><li>Ethanol 70%<br/>Add 30 ml of sterile (RNase/DNase-free) water in 70 ml of Ethanol (CH<sub>3</sub>CH<sub>2</sub>OH) (Merck) in a 200 ml glass bottle. Mix</li><li>RLT + Î²-ME buffer<br/><em>Note: RNeasy Lysis Buffer (Qiagen RLT or supplied with the Qiagen RNeasy Kits) contains guanidine isothiocyanate (GITC). In order to reduce RNases activity, Î²-mercaptoethanol (Î²-ME) is added to the RLT buffer before use.</em><br/><ol class="npwthree"><li>Add 20 Î¼l of Î²-ME per 1 ml of RLT buffer. This Î²-ME containing RLT buffer is stable for up to one month<br/><em>Warning: Î²-ME is a toxic chemical and should be dispensed in a fume hood and appropriate protective clothing should be worn.</em><br/></li><li>Alternatively, Î²-ME may be replaced by 20 Î¼l of Dithiothreitol 2M (DTT) per ml of RLT buffer<br/></li><li>RLT complemented buffer with Î²-ME or DTT, as described, is added in the homogenization tubes that will contain the ear pinna (1 ml/tube) and the lymph node (400 Î¼l/tube) (Mommaerts <em>et al.</em>, 2015)</li></ol></li></ol><p class="pptt" id="biaoti34275">### Acknowledgments</p><p class="ppzz">This work was supported by institutional funding from the Institut Pasteur. We are grateful to the members of the Immunophysiology and Parasitism Unit for their recommendations for these experimental procedures and their participation to the original work described in the published paper from where this protocol is derived (Giraud <em>et al.</em>, 2019b).</p><p class="pptt" id="biaoti34276">### Competing interests</p><p class="ppzz">The authors declare that there is none financial or non-financial competing interests associated with this work.</p><p class="pptt" id="biaoti34277">### Ethics</p><p class="ppzz">All the animals used in our experiments were kept under SPF conditions at the animal facility of the Institut Pasteur, raised under a 12/12 hour dark-light cycle with ad libitum access to food and water. All experimental protocols were approved by the Institutional Committees on animal Welfare under strict accordance with the European guidelines (Directive 2010/63/EU) for animal care.</p><p class="pptt" id="biaoti34278">### References</p><ol class="npwtwok"><li>Bates, P. A. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17517415" target="_blank">Transmission of <em>Leishmania</em> metacyclic promastigotes by phlebotomine sand flies.</a> <em>Int J Parasitol</em> 37(10): 1097-1106.</li><li>Christensen, S. M., Belew, A. T., El-Sayed, N. M., Tafuri, W. L., Silveira, F. T. and Mosser, D. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30845223" target="_blank">Host and parasite responses in human diffuse cutaneous leishmaniasis caused by L. amazonensis.</a> <em>PLoS Negl Trop Dis</em> 13(3): e0007152. </li><li>Cortes, D. F., Carneiro, M. B., Santos, L. M., Souza, T. C., Maioli, T. U., Duz, A. L., Ramos-Jorge, M. L., Afonso, L. C., Carneiro, C. and Vieira, L. Q. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20944986" target="_blank">Low and high-dose intradermal infection with <em>Leishmania</em> major and <em>Leishmania amazonensis</em> in C57BL/6 mice.</a> <em>Mem Inst Oswaldo Cruz</em> 105(6): 736-745.</li><li>de Oliveira Cardoso, F., de Souza Cda, S., Mendes, V. G., Abreu-Silva, A. L., Goncalves da Costa, S. C. and Calabrese, K. S. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20462353" target="_blank">Immunopathological studies of <em>Leishmania amazonensis</em> infection in resistant and in susceptible mice.</a> <em>J Infect Dis</em> 201(12): 1933-1940. </li><li>Giraud, E., Martin, O., Yakob, L. and Rogers, M. (2019a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30854476" target="_blank">Quantifying <em>Leishmania</em> metacyclic promastigotes from individual sandfly bites reveals the efficiency of vector transmission.</a> <em>Commun Biol</em> 2: 84. </li><li>Giraud, E., Rouault, E., Fiette, L., Colle, J. H., Smirlis, D. and Melanitou, E. (2019b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30736736" target="_blank">Osteopontin in the host response to <em>Leishmania amazonensis</em>.</a> BMC Microbiol 19(1): 32.</li><li>Hellemans, J., Mortier, G., De Paepe, A., Speleman, F. and Vandesompele, J. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17291332" target="_blank">qBase relative quantification framework and software for management and automated analysis of real-time quantitative PCR data.</a> <em>Genome Biol</em> 8(2): R19.</li><li>Liu, D. and Uzonna, J. E. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22919674" target="_blank">The early interaction of <em>Leishmania</em> with macrophages and dendritic cells and its influence on the host immune response.</a> <em>Front Cell Infect Microbiol</em> 2: 83.</li><li>Mansueto, P., Vitale, G., Di Lorenzo, G., Rini, G. B., Mansueto, S. and Cillari, E. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17880757" target="_blank">Immunopathology of leishmaniasis: an update.</a> <em>Int J Immunopathol Pharmacol</em> 20(3): 435-445.</li><li>McMahon-Pratt, D. and Alexander, J. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15361243" target="_blank">Does the <em>Leishmania</em> major paradigm of pathogenesis and protection hold for New World cutaneous leishmaniases or the visceral disease?</a> <em>Immunol Rev</em> 201: 206-224.</li><li>Mommaerts, K., Sanchez, I., Betsou, F. and Mathieson, W. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25841674" target="_blank">Replacing beta-mercaptoethanol in RNA extractions.</a> <em>Anal Biochem</em> 479: 51-53. </li><li>Reid, G. A. (1991). <a href="https://www.sciencedirect.com/science/article/abs/pii/016777999190068S?via%3Dihub" target="_blank">A vade-mecum for molecular biologists.</a> <em>Trends in Biotechnology</em> 9(1): 213-214. </li><li>Schroeder, A., Mueller, O., Stocker, S., Salowsky, R., Leiber, M., Gassmann, M., Lightfoot, S., Menzel, W., Granzow, M. and Ragg, T. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16448564" target="_blank">The RIN: an RNA integrity number for assigning integrity values to RNA measurements.</a> <em>BMC Mol Biol</em> 7: 3. </li><li>Schuster, S., Hartley, M. A., Tacchini-Cottier, F. and Ronet, C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24904841" target="_blank">A scoring method to standardize lesion monitoring following intra-dermal infection of <em>Leishmania</em> parasites in the murine ear.</a> <em>Front Cell Infect Microbiol</em> 4: 67. <br/></li></ol>