<body>
            <h1>Safe DNA-extraction Protocol Suitable for Studying Tree-fungus Interactions</h1>
            <p><strong>Authors:</strong> Susanna KeriÃ¶, Eeva  Terhonen, Jared M. LeBoldus</p>
            <h2>Abstract</h2>
            <p>We present a safe and low-cost method suitable for DNA extraction from mycelium and tree tissue samples. After sample preparation, the extraction takes about 60 min. Method performance was tested by extracting DNA from various tree tissue samples and from mycelium grown on solid and liquid media. DNA was extracted from juvenile and mature host material (<em>Picea abies</em>, <em>Populus trichocarpa</em>, <em>Pseudotsuga menziesii</em>) infected with different pathogens (<em>Heterobasidion annosum</em>, <em>Heterobasidion parviporum</em>, <em>Leptographium wagenerii</em>, <em>Sphaerulina musiva</em>). Additionally, DNA was extracted from pure cultures of the pathogens and several endophytic fungi. PCR success rate was 100% for young poplar material and fungal samples, and 48-72% for conifer and mature broadleaved plant samples. We recommend using 10-50 mg of fresh sample for the best results. The method offers a safe and low-cost DNA extraction alternative to study tree-fungus interactions, and is a potential resource for teaching purposes.</p>
            <p><strong>Keywords:</strong> DNA extraction, Plant DNA, Fungal DNA, Forest pathology, Plant-microbe interactions, Low-cost, Non-toxic</p>
            <h2>Background</h2><p>DNA extraction is a central technique in plant-microbe interaction research and for plant disease diagnostic purposes. The abundance of plant-fungal interactions is reflected by the high numbers of cultivable strains isolated from plant samples (Arnold <em>et al.</em>, 2001; Higgins <em>et al.</em>, 2007; Terhonen <em>et al.</em>, 2011). Processing of large numbers of samples for routine PCR reactions with commercial kits can be costly. Additionally, the requirement for hazardous chemicals, such as 2-mercaptoethanol, chloroform, or phenol can restrict the suitability of the protocols for teaching and training purposes.<br/><br/>Due to high secondary metabolite content, many plant samples, and in particular tree tissues, can pose challenges for nucleic acid extraction. Current protocols for DNA extraction from recalcitrant plant tissue utilize organic solvents (chloroform, 2-mercaptoethanol) or surfactants (<em>e.g.</em>, cetyl trimethylammonium bromide) (Porebski <em>et al.</em>, 1997; Chiong <em>et al.</em>, 2017; Yi <em>et al.</em>, 2018). Despite their benefits for DNA extraction, these chemicals pose hazards to user health and the environment. Several versions of low-cost, fast, and low health risk protocols for DNA extraction exist for mycelium (Chi <em>et al.</em>, 2009), juvenile plant tissue (Edwards <em>et al.</em>, 1991; Lu, 2011), grains (Saini <em>et al.</em>, 1999), and dried plant tissues (Chabi Sika <em>et al.</em>, 2015). However, these methods have not been applied to study tree-fungus interactions, and many times they have been tested only on limited number of sample types. Our goal was to develop and test a safe and low-cost DNA-extraction protocol that is suitable for extracting DNA from various tree tissues and tree-associated fungal samples. After sample preparation, the extraction takes about 60 min. The extracted DNA is suitable for PCR-based downstream applications, such as DNA-based pathogen detection with species-specific primers. Due to its safety and affordability, the method is also a potential resource for teaching and training purposes.</p>
        </body>

<p class="pptt" id="biaoti34120">## Materials and Reagents</p><ol class="npwone"><li>Standard materials and reagents<ol class="npwtwo"><li>Sterile microcentrifuge tubes, 1.5 or 2.0 ml</li><li>Micropipette tips: 10, 200, 1,000 Âµl</li><li>Miracloth (Calbiochem,<em> e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 475855-1 </a> )</li><li>Plant or fungal samples</li><li>Purified (RO, DI, MilliQ, Nanopure) and sterilized water</li><li>100 bp DNA ladder (Jena Bioscience, catalog number: <a class="blackNoUnder" target="_blank"> M-214S </a> )</li><li>1 kb DNA ladder (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> N0552 </a> )</li><li>NaCl</li><li>KCl</li><li>EDTA</li><li>SDS</li><li>Tris (pH 7.5)</li><li>Polyvinypyrrolidone (PVP, CAS 900-39-8, FW 40,000, <em>e.g.</em>, Caisson Labs, catalog number: <a class="blackNoUnder" target="_blank"> P071-100GM </a> )</li><li>Isopropanol</li><li>Ethanol (EtOH)</li><li>Extraction buffer (see Recipes)</li><li>Wash buffer (see Recipes)<br/><br/></li></ol></li><li>Special materials and reagents for different tissue homogenization optionsB.Special materials and reagents for different tissue homogenization options<ol class="npwtwo"><li>Option 1 for soft leaf tissue and mycelium: No special materials or reagents</li><li>Option 2 for various plant tissue and mycelium, larger than 100 mg: Liquid nitrogen</li><li>Option 3 for various plant tissue and mycelium, smaller than 100 mg: Bead beater tubes (<em>e.g.</em>, Lysing Matrix I, MP Biomedicals, catalog number: <a class="blackNoUnder" target="_blank"> 116918050-CF </a> )<br/><em>Note: To reduce plastic waste and save on costs, the bead beater tubes can be washed and re-used (see Notes).</em><br/></li></ol></li></ol><p class="pptt" id="biaoti34121">### Equipment</p><ol class="npwone"><li>Standard equipment<ol class="npwtwo"><li>Scalpels</li><li>Tweezers</li><li>Spatulas</li><li>Scale (<em>e.g.</em>, Metler-Toledo, model: <a class="blackNoUnder" target="_blank"> ML54T </a> )</li><li>Micropipettes: 1, 10, 100, 1,000 Âµl</li><li>Heat block (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 12621-096 </a> )</li><li>Vortex (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 10153-838 </a> )</li><li>Microcentrifuge (<em>e.g.</em>, Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5424 </a> )</li><li>One of the following to heat up water: Microwave, waterbath (<em>e.g.</em>, VWR, model: <a class="blackNoUnder" target="_blank"> WB05 </a> ), or hot plate (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> NO97042-642 </a> )</li><li>Freezer, -20 Â°C or -80 Â°C<br/><br/></li></ol></li><li>Special equipment for different tissue homogenization options<ol class="npwtwo"><li>Option 1 for soft leaf tissue and mycelium: No special equipment</li><li>Option 2 for various plant tissue and mycelium, more than 100 mg:<br/>Dewar for liquid nitrogen<br/>Ceramic mortars and pestles</li><li>Option 3 for various plant tissue and mycelium, less than 100 mg:<br/>Bead beater (<em>e.g.</em>, Biospec, model: <a class="blackNoUnder" target="_blank"> Mini-Beadbeater 16 </a> , catalog number: <a class="blackNoUnder" target="_blank"> 607/607EUR </a> )</li><li>For sampling xylem tissue from mature trees: chisel (<em>e.g.</em>, Grainger, catalog number: <a class="blackNoUnder" target="_blank"> 2AJA6 </a> ) and mallet (<em>e.g.</em>, Grainger, catalog number: <a class="blackNoUnder" target="_blank"> 4YR61 </a> ), cutting board<br/></li></ol></li></ol><p class="pptt" id="biaoti34122">### Procedure</p><ol class="npwone"><li>Sample preparation: Weigh 10-50 mg of sample (see Figure 1).<br/><ol class="npwtwo"><li>Plant tissue: Use tweezers and scalpel to cut the sample to approximately 5 Ã 5 Ã 1 mm pieces. Smaller and thinner pieces will result in better sample quality.</li><li>Mycelium: Use scalpel/spatula to scrape mycelium from the surface of Petri plates, or use spatula to collect mycelium from liquid culture.</li><li>Xylem tissue from mature trees: Place wood sample on cutting board. Use chisel and mallet to harvest pieces of xylem tissue. Cut to 5 Ã 5 Ã 1 mm pieces with scalpel and spatula.</li></ol><em>Note: For mycelium, minimize the amount of agar for higher DNA quality.</em><br/><br/><img alt="" class="layerphoto" height="75" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602120037_5744.jpg" width="300"/><br/><strong>Figure 1. Examples of sample sizes for DNA extraction.</strong> A. Fresh phloem from 8-week-old <em>Populus trichocarpa</em> trees. Weight 20 mg. B. Fresh phloem from mature <em>P. trichocarpa</em> trees. Weight 15 mg. C. Fresh <em>Leptographium wagnerii</em> mycelium harvested from liquid malt extract cultures. Weight 20 mg. Interval between vertical lines = 1 mm.<br/><br/></li><li>Homogenize sample and add extraction buffer<br/>Three options are available depending on sample type. Complete homogenization is not necessary.<br/><ol class="npwtwo"><li>Option 1: Mycelium and soft leaf tissue<br/><ol class="npwthree"><li>Place sample in a 1.5 or 2.0 ml microcentrifuge tube.<br/></li><li>Add 1 ml extraction buffer.<br/></li><li>Vortex rigorously for 20 s.<br/></li><li>Proceed to Procedure C.</li></ol></li><li>Option 2: Various plant tissue and mycelium â¥ 50 mg:<br/><ol class="npwthree"><li>Grind sample in mortar with pestle and liquid nitrogen.<br/></li><li>Transfer 10-50 mg of homogenized sample with a spatula or by decanting to a 1.5/2.0 ml microcentrifuge tube.<br/></li><li>Add 1 ml extraction buffer.<br/></li><li>Vortex rigorously for 20 s.</li><li>Proceed to Procedure C.</li></ol></li><li>Option 3: Various plant tissue and mycelium â¤ 50 mg:<br/><ol class="npwthree"><li>Transfer the sample into a 2.0 ml beat beater tube.<br/></li><li>Add 1 ml extraction buffer.<br/></li><li>Process for 20 s in a bead beater.</li><li>Proceed to Procedure C.</li></ol><em>Note: Use 1 ml of extraction buffer per 50 mg or less tissue. To allow sufficient vortexing, use only 1 ml of buffer per tube.</em><br/><br/></li></ol></li><li>DNA extraction<ol class="npwtwo"><li>Heat the wash buffer in a water bath or in a beaker with warm water (approximately 65 Â°C) to dissolve any precipitants. Temperature is not critical, as long as no precipitants remain. Mix by inversion.</li><li>Lysis and debris elimination: Incubate the samples in extraction buffer at 65 Â°C for 15 min. Vortex once during incubation. Centrifuge at 6,000 <em>x g</em> for 10 min.</li><li>Eliminate debris: After centrifugation, transfer <em>ca.</em> 0.5 volume of the supernatant into a new tube. Add 1 volume of pre-heated wash buffer, and vortex samples for 20 s. Centrifuge at 21,000 <em>x g</em> for 10 min.<br/><em>Note: Complete debris elimination is not critical when pipetting the supernatant.</em> </li><li>Precipitate: Transfer <em>ca.</em> 0.7 volume of supernatant into a new tube, add 0.85 volume of isopropanol (room temperature), and mix by inversion for 20 s. Centrifuge at 21,000 <em>x g</em> for 10 min.<br/><em>Note: Minimize transferring any debris while pipetting the supernatant.</em> </li><li>Wash the pellet: Pour out the supernatant, and remove remaining supernatant by tapping the tubes upside down on a paper towel. Add 200 Âµl of 70% ethanol, and centrifuge at 21,000 <em>x g</em> for 5 min.<br/><em>Note: Consider local regulations for correct handling of isopropanol waste.</em> </li><li>Dry the pellet: Pipet out the ethanol. Leave the caps open, and dry pellets in a heat block at 65 Â°C for 5 min.<br/><em>Note: For faster drying, remove as much of the ethanol as possible.</em> </li><li>Resuspension: Dissolve the pellet in 20-50 Âµl of TE buffer or nuclease free water. Vortex to dissolve if needed, and centrifuge briefly to collect any droplets to the bottom of the tube. Store DNA samples at -20 Â°C or -80 Â°C until used.<br/><em>Note: If the DNA pellet is not colorless or white post 70% ethanol wash, add 20-50 Âµl of TE buffer or nuclease-free water to resuspend the DNA without disturbing the pellet. Gently pipet the liquid a few times in the tube and collect the supernatant as DNA for downstream processes. Keep the pellet until DNA is quantified. Use the DNA for PCR-based detection, or store in freezer until used.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti34123">### Data analysis</p><p class="ppzzs"><strong>Analysis of protocol performance:</strong> </p><ol class="npwone"><li>Plant and fungal material used for DNA extractions<br/>To test the suitability of the protocol, we extracted DNA from artificially inoculated trees, naturally infected trees, and mycelium (Table 1). The artificially inoculated samples included 8-week-old <em>Populus trichocarpa</em> plants spray-inoculated with <em>Sphaerulina musiva</em> (LeBoldus <em>et al.</em>, 2010; Abraham <em>et al.</em>, 2018), and 3-year-old <em>Picea abies</em> plants plug-inoculated with <em>Heterobasidion</em> sp. (Terhonen <em>et al.</em>, 2019). The naturally infected plant samples included stem cankers on mature <em>P. trichocarpa</em> trees caused by <em>S. musiva</em> infection, and mature <em>Pseudotsuga menziesii</em> roots infected with <em>Leptographium wageneri</em>. Necrotic or discolored phloem or xylem samples with 10-380 mg of tissue (average 86 mg) were used for DNA extraction. For every sample, 1 ml of extraction buffer was used regardless of sample weight. Fungal DNA was extracted from mycelium harvested from pure cultures (Table 1). For solid medium, either malt extract agar (2% malt extract, 2% agar) or KV8 agar (18% V8 juice, 0.2% CaCO<sub>3</sub>, 2% agar) were used. Cultures on solid medium were grown in ambient room temperature. For liquid medium, either malt extract medium (2% malt extract) or KV8 medium (18% V8 juice, 0.2% CaCO<sub>3</sub>) was used (Table 1). Cultures in liquid medium were grown in ambient room temperature on a rotary shaker (100-150 rpm). The mycelium was harvested from liquid medium by filtering through Miracloth (Calbiochem) and rinsed with DI water.<br/><br/></li><li>Assessment of DNA sample quality<br/>DNA concentrations were measured with Nanodrop, Nanophotometer, or Qubit. PCR, quantitative real-time PCR (qPCR), agarose gel electrophoresis (Figure 2), and fungal ITS sequencing were used to evaluate sample quality. For PCR and qPCR, no-template negative controls and positive template controls were included into each run to evaluate detection reliability.<br/>  For the <em>P. trichocarpa</em> samples that were inoculated or naturally infected with <em>S. musiva</em>, we used a host-pathogen specific assay (Abraham <em>et al.</em>, 2018). For detection of <em>Heterobasidion</em> species from inoculated wood samples, species-specific primers for <em>H. annosum</em> and <em>H. parviporum</em> (Hantula and Vainio, 2003) were used (Terhonen <em>et al.</em>, 2019). For detection of <em>L. wageneri</em> from <em>P. pseudotsuga</em> roots and fungal cultures, we used <em>Leptographium</em>-specific primers (Schweigkofler <em>et al.</em>, 2005). Primers for <em>P. menziesii</em> (Winton <em>et al.</em>, 2002) were used to distinguish PCR-inhibition from negative samples, and to amplify DNA extracted from Douglas-fir needles.<br/>  DNA samples from <em>Diplodia sapinea</em> cultures were amplified with primers targeting the nuclear large subunit, elongation factor and calmodulin regions (Vilgalys and Hester, 1990; Carbone and Kohn, 1999; GrÃ¼nig <em>et al.</em>, 2007; Nelsen <em>et al.</em>, 2011). Additionally, DNA from <em>D. sapinea</em> and fungal endophytes was amplified with primers ITS1-F and ITS4 for the fungal ribosomal internal transcribed spacer region (White <em>et al.</em>, 1990; Gardes and Bruns, 1993). The PCR conditions are specified in Table 2 and primer sequences in Table 3. All PCR amplicons were visualized under UV light on 1.5% agarose gels with StainIN<sup>TM</sup> RED or GelRed<sup>TM</sup> nucleic acid stains. For fungal species used for sequencing, the PCR products were purified and sequenced using the respective primers (Table 1) at Microsynth SEQLAB (GÃ¶ttingen, Germany).<br/>  The effect of potential PCR inhibitors in the DNA samples on target detection was evaluated with a multiplex Taqman qPCR protocol (Abraham <em>et al.</em>, 2018). DNA extracted with a commercial kit (DNeasy Plant Mini, Qiagen) from comparable tissue samples was used as a reference for low-inhibitor samples. Seven-point dilution series were prepared for <em>P. trichocarpa</em> (10-fold dilution series, 60-6 Ã 10<sup>-4</sup> ng/Âµl) and <em>S. musiva</em> DNA samples (5-fold dilution series, 50-3.2 Ã 10<sup>-3</sup> ng/Âµl) (Abraham <em>et al.</em>, 2018) extracted with the developed method and with the commercial kit. The quantification cycle (Cq) values for the samples from the two extraction methods were compared, to evaluate the impact of potential PCR inhibitors on target detection.<br/><br/><img alt="" class="layerphoto" height="117" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602120056_8641.jpg" width="300"/><br/><strong>Figure 2. Examples of DNA samples extracted with the developed protocol.</strong> A. Three <em>Diplodia sapinea</em> DNA samples extracted from mycelium grown on malt extract agar (MEA). Lanes 1, 3, and 5: 50-500 ng of DNA. Lanes 2, 4, and 6: 10-fold dilutions of samples in lanes 1, 3, and 5. Gel: 2.0% agarose in 1Ã TAE, 100 V, 35 min. Ladder: 100 bp DNA ladder. B. DNA extracted from <em>Leptographium</em> wagnerii mycelium from liquid malt extract (lane 1), <em>Sphaerulina musiva</em> mycelium grown on KV8 agar (lane 2), <em>S. musiva</em> grown in liquid KV8 medium (3), fresh poplar phloem (lane 4), fresh poplar leaves (lane 5), mature Douglas-fir xylem (lane 6), and fresh Douglas-fir needles (lane 7). Lanes 1-4 and 6-7: 20-100 ng DNA. Lane 5: 500 ng DNA. Gel: 1% agarose in 1Ã TAE, 120 V, 70 min. Ladder: 1 kb DNA ladder.<br/><br/><strong>Table 1. Sample types extracted with the protocol, PCR success rates (%), and number of sequenced samples<img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603225759_3819.jpg" width="900"/></strong><br/><sup>a</sup>Gardes and Bruns 1993, White <em>et al.</em>, 1990<br/><sup>b</sup>GrÃ¼nig <em>et al.</em>, 2007<br/><sup>c</sup>Carbone and Kohn, 1999<br/><sup>d</sup>Vilgalys and Hester, 1990, Nelsen <em>et al.</em>, 2011<br/><br/><strong>Table 2. PCR conditions and primers used to test sample quality</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603230105_4741.jpg" width="900"/><br/><sup>a</sup>Carbone and Kohn 1999, GrÃ¼nig <em>et al.</em>, 2007<br/><sup>b</sup>Carbone and Kohn, 1999 <br/><sup>c</sup>Vilgalys and Hester, 1990, Nelsen <em>et al.</em>, 2011<br/><sup>d</sup>Gardes and Brunns, 1993; White <em>et al.</em>, 1990 <br/><sup>e</sup>Hantula and Vainio, 2003<br/><sup>f</sup>Schweigkofler <em>et al.</em>, 2005<br/><sup>g</sup>Abraham <em>et al.</em>, 2018<br/><sup>h</sup>Winton <em>et al.</em>, 2002<br/><div style="white-space:nowrap;"><br/></div><strong>Table 3. Primer sequences used in the PCR reactions.</strong><br/><img alt="" class="layerphoto" height="261" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603230145_6758.jpg" width="900"/><br/><sup>a</sup>Carbone and Kohn, 1999, GrÃ¼nig <em>et al.</em>, 2007<br/><sup>b</sup>Carbone and Kohn, 1999 <br/><sup>c</sup>Vilgalys and Hester, 1990, Nelsen <em>et al.</em>, 2011<br/><sup>d</sup>Gardes and Brunns, 1993, White <em>et al.</em>, 1990 <br/><sup>e</sup>Hantula and Vainio, 2003<br/><sup>f</sup>Schweigkofler <em>et al.</em>, 2005<br/><sup>g</sup>Abraham <em>et al.</em>, 2018<br/><sup>h</sup>Winton <em>et al.</em>, 2002<br/><br/><div style="white-space:nowrap;"><span style="font-size:10pt;"></span> </div></li><li>Data analysis<br/>Data analysis was conducted in R version 3.6.1. The effect of PCR inhibitors on quantification cycle (C<sub>q</sub>) values was estimated using ANOVA followed by Tukeyâs HSD tests (Figure 3A). We visualized the contribution of sample weight, DNA concentration, and sample purity (A<sub>260/280</sub> and A<sub>260/230</sub>) on PCR success by plotting the results from principal component analysis (PCA) for the plant and fungal samples (Figures 3B-3D). The PCA results were visualized with the R package factoextra (Kassambara and Mundt 2017). For <em>P. abies</em> samples, necrosis length was also included in the model. Separate PCAâs were computed for <em>P. trichocarpa</em> samples (n = 85), <em>P. abies</em> samples (n = 48), and fungal samples (n = 129). Differences in template properties between failed and successful PCR reactions within the same sample type were compared by two-sample t-tests. If necessary, data were normalized with log-transformations.<br/><br/></li><li>Protocol performance<br/>All the DNA samples extracted from 8-week-old <em>P. trichocarpa</em> phloem inoculated with <em>S. musiva</em> were PCR-positive for the pathogen (Table 1). PCR amplification worked for all fungal DNA samples, and the PCR products were suitable for fungal ITS sequencing. In comparison, PCR success rate was lower for DNA samples extracted from naturally infected mature <em>P. trichocarpa</em> samples (72%), inoculated 3-year-old <em>P. abies</em> samples (48%), and naturally infected mature <em>P. menziesii</em> roots (60%) (Table 1). The extraction protocol yields total DNA with the majority of the fragments larger than 10 kb (Figure 2). The samples are stable at least for 2 years in -20 Â°C. Partial DNA fragmentation (Figure 2B) did not affect PCR performance.<br/>  Based on the comparison of C<sub>q</sub> values for DNA samples extracted with a commercial kit, the extracted DNA samples may contain inhibitors that can affect the accuracy of target quantification by qPCR (Figure 3A). The Cq values were lower for the three highest dilutions (<em>P</em> &lt; 0.040). After 1,000-fold dilution, the extraction method had no effect on quantification. The standard curves prepared from the DNA samples extracted with the protocol had lower amplification efficiencies compared to the commercial kit (Figure 1A). It is possible that the qPCR protocol in Abraham <em>et al.</em> (2018) is not optimal for the samples extracted with the developed DNA extraction protocol. However, the extraction method did not affect target detection.<br/><br/><img alt="" class="layerphoto" height="225" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200602/20200602120525_5148.jpg" width="300"/><br/><strong>Figure 3. Comparative qPCR analysis to estimate the presence of sample inhibitors (A), and principal component analysis to visualize sample properties in </strong><em><strong>Populus trichocarpa</strong></em><strong> samples (B), young </strong><em><strong>Picea abies</strong></em><strong> samples (C), and fungal samples (D)</strong><br/><br/>  We explored the association of DNA sample properties with PCR success by visualizing the results from principal component analysis (PCA). The two first principal components (PC) explained 68-87% of the total variation. The A<sub>260/280</sub> and A<sub>260/230</sub> values explained majority of variation along PC1 for <em>P. trichocarpa</em>, <em>P. abies</em>, and fungal DNA samples (55%, 85% and 75% contribution to PC1 variation, respectively) (Figures 1B-1D). For <em>P. trichocarpa</em>, the samples split into two groups on both sides of the vertical PC2 axis (Figure 3B). For the samples on the left side of the PC2 axis, PCR success rate was 100% and sample purity was relatively high (Figure 3B).<br/>  Sample weight explained 55% and 40% of the variation along the PC2 in <em>P. trichocarpa</em> and <em>P. abies</em>, respectively. Larger initial sample weights were associated with lower PCR success rate for mature poplar and young Norway spruce DNA samples (Table 4). Among the 34 mature poplar samples that clustered on the right side of PC2 axis (Figure 3B), initial sample weights and DNA concentrations were higher for the failed PCR reactions (t = 2.882, <em>P</em> = 0.007 and t = 2.195, <em>P</em> = 0.035, respectively). Similarly, initial sample weights and DNA concentrations were higher for <em>P. abies</em> samples that failed PCR amplification (t = 1.9511, <em>P</em> = 0.057 and t = 2.060, <em>P</em> = 0.045, respectively). For the mature poplar samples, PCR failure both for host and pathogen amplification despite higher DNA concentration is probably associated with low template quality combined with high polyphenolic content in the mature bark tissue samples. For Norway spruce, PCR failure for pathogen amplification despite high DNA concentration and sufficient template quality is probably explained by low amount of pathogen DNA in less colonized samples. Alternatively, high phenolic content in heavily colonized samples may have inhibited pathogen detection, as increasing necrosis length was associated with lower PCR success in Norway spruce samples. This indicates that DNA samples extracted from tree tissue samples with high amounts of lignin or other polyphenolic compounds may have lower PCR success rates.<br/>  The fungal DNA samples were highly variable based on A<sub>260/230</sub> and A<sub>260/280</sub> values (75% of variation on PC1, Figure 3D). The DNA samples extracted from mycelium grown in liquid cultures typically had higher measures of purity compared to fungal samples with agar (Table 4). Based on this, we recommend minimizing the amount of agar in the fungal samples used for DNA extractions. Despite high variation in sample purity, PCR amplification was successful for all the fungal DNA samples.<br/><br/><strong>Table 4. Sample weight, DNA concentrations, and DNA absorbance values for different DNA sample types extracted with the protocol. Median, minimum and maximum values are indicated.</strong><br/><img alt="" class="layerphoto" height="212" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603230223_7920.jpg" width="900"/><br/>*Both host and pathogen PCR failed<br/>**Only pathogen PCR tested</li></ol><p class="pptt" id="biaoti34124">### Notes</p><ol class="npwtwok"><li>Amount of extraction buffer and sample weight<br/>We recommended to use 1 ml of extraction buffer per 50 mg or less of plant or fungal tissue. Low buffer volume relative to extracted tissue can have a negative effect on PCR success.</li><li>Dissolving the pellet<br/>If the DNA pellet is not colorless or white post 70% ethanol wash, add 20-50 Î¼l of TE buffer or nuclease-free water to resuspend the DNA without disturbing the pellet. Gently pipet the liquid a few times in the tube and collect the supernatant as DNA for downstream processes. Keep the pellet until DNA is quantified. Use the DNA for PCR-based detection, or store in freezer until used.</li><li>Re-using the bead beater tubes<br/>To reduce costs and plastic waste, the bead beater tubes can be washed, treated with bleach to degrade DNA, autoclaved, and re-used. Separate the beads, caps and tubes in separate containers. Fill the containers with warm soap water, agitate for 5 min, and pour out the soap water. Rinse with tap water until runoff is clear. Shake tubes and caps to remove remaining tap water. Rinse twice with DI-water. To remove any remaining DNA, soak the components 1 h in 3% w/v NaOCl solution (1:1 solution with commercial bleach and DI-water) (Kemp and Smith 2005). Rinse twice with tap water, followed by two DI-water rinses. Let the tube components dry overnight, or dry in an oven. Once the components are dry, compile the tubes and autoclave at 121 Â°C for 30 min.<br/></li></ol><p class="pptt" id="biaoti34125">### Recipes</p><ol class="npwtwok"><li>Extraction buffer<br/>1 M NaCl<br/>100 mM Tris HCl<br/>10 mM EDTA<br/>2% PVP<br/><ol class="npwthree"><li>Mix all ingredients in a beaker on a stirring hot plate<br/></li><li>Fill to desired volume with sterile purified water<br/></li><li>Heat the solution until PVP is dissolved</li></ol></li><li>Wash buffer<br/>1% SDS<br/>0.5 M KCl or NaCl<br/><ol class="npwthree"><li>Mix all ingredients in a beaker on a stirring hot plate<br/></li><li>Fill to desired volume with sterile purified water<br/></li><li>Heat the solution until no precipitation is visible<br/></li><li>Heat the wash buffer before use to dissolve any precipitants</li></ol></li></ol><p class="pptt" id="biaoti34126">### Acknowledgments</p><p class="ppzz">This research was supported by the DOE Office of Science, Office of Biological and Environmental Research (BER), grant no. DE-SC0018196, and by funding from the Faculty of Forest Sciences and Forest Ecology, University of GÃ¶ttingen. Jumoke Aduke Babalola and David Robert RÓscuÅ£oi, University of GÃ¶ttingen, are highly acknowledged for their contribution to the DNA extraction and PCR experimental set up. Dr. Patrick Bennett is thanked for providing the <em>L. wagenerii</em> strains and naturally infected Douglas-fir root samples, and Dr. Kelsey SÃ¸ndreli for providing the mature <em>P. trichocarpa</em> samples naturally infected with <em>S. musiva</em>.</p><p class="pptt" id="biaoti34127">### Competing interests</p><p class="ppzz">No competing interests declared.</p><p class="pptt" id="biaoti34128">### References</p><ol class="npwtwok"><li>Abraham, N. D., Chitrampal, P., KeriÃ¶, S. and LeBoldus, J. M. (2018). <a href="https://bsppjournals.onlinelibrary.wiley.com/doi/10.1111/ppa.12913" target="_blank">Multiplex qPCR for detection and quantification of <em>Sphaerulina musiva</em> in <em>Populus</em> stems.</a> <em>Plant Pathol</em> 67(9): 1874-1882.</li><li>Arnold, A. E., Maynard, Z. and Gilbert, G. S. (2001). <a href="https://www.sciencedirect.com/science/article/abs/pii/S0953756208620365" target="_blank">Fungal endophytes in dicotyledonous neotropical trees: patterns of abundance and diversity.</a> <em>Mycol Res</em> 105(12): 1502-1507.</li><li>Carbone, I. and Kohn, L. M. (1999). <a href="https://www.tandfonline.com/doi/abs/10.1080/00275514.1999.12061051" target="_blank">A method for designing primer sets for speciation studies in filamentous ascomycetes.</a> <em>Mycologia</em> 91(3): 553-556.</li><li>Chabi Sika, K., Kefela, T., Adoukonou-Sagbadja, H., Ahoton, L., Saidou, A., Baba-Moussa, L., Jno Baptiste, L., Kotconi, S. O. and Gachomo, E. W. (2015). <a href="http://www.sciencedirect.com/science/article/pii/S2352407315000086" target="_blank">A simple and efficient genomic DNA extraction protocol for large scale genetic analyses of plant biological systems.</a> <em>Plant Gene</em> 1: 43-45.</li><li>Chi, M. H., Park, S. Y. and Lee, Y. H. (2009). <a href="https://www.researchgate.net/publication/255587119_A_Quick_and_Safe_Method_for_Fungal_DNA_Extraction" target="_blank">A quick and safe method for fungal DNA extraction.</a> <em>Plant Pathol J</em> 25: 108-111. </li><li>Chiong, K. T., Damaj, M. B., Padilla, C. S., Avila, C. A., Pant, S. R., Mandadi, K. K., Ramos, N. R., Carvalho, D. V. and Mirkov, T. E. (2017). <a href="https://plantmethods.biomedcentral.com/articles/10.1186/s13007-017-0255-6" target="_blank">Reproducible genomic DNA preparation from diverse crop species for molecular genetic applications.</a> <em>Plant Methods</em> 13(1): 106.</li><li>Edwards, K., Johnstone, C. and Thompson, C. (1991). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2030957" target="_blank">A simple and rapid method for the preparation of plant genomic DNA for PCR analysis.</a> <em>Nucleic Acids Res</em> 19(6): 1349. </li><li>Gardes, M. and Bruns, T. D. (1993). <a href="https://www.ncbi.nlm.nih.gov/pubmed/8180733" target="_blank">ITS primers with enhanced specificity for basidiomycetes--application to the identification of mycorrhizae and rusts.</a> <em>Mol Ecol</em> 2(2): 113-118. </li><li>GrÃ¼nig, C. R., Brunner, P. C., Duo, A. and Sieber, T. N. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17289408" target="_blank">Suitability of methods for species recognition in the <em>Phialocephala fortinii-Acephala applanata</em> species complex using DNA analysis.</a> <em>Fungal Genet Biol</em> 44(8): 773-788. </li><li>Hantula, J. and Vainio, E. (2003). <a href="https://silvafennica.fi/article/500" target="_blank">Specific primers for the differentiation of <em>Heterobasidion</em> annosum (s.str.) and <em>H. parviporum</em> infected stumps in northern Europe.</a> <em>Silva Fennica</em> 37. </li><li>Higgins, K. L., Arnold, A. E., Miadlikowska, J., Sarvate, S. D. and Lutzoni, F. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17005421" target="_blank">Phylogenetic relationships, host affinity, and geographic structure of boreal and arctic endophytes from three major plant lineages.</a> <em>Mol Phylogenet Evol</em> 42(2): 543-555. </li><li>Kassambara, A., and Mundt, F. (2017). <a href="https://cran.r-project.org/web/packages/factoextra/index.html" target="_blank"><em>factoextra: Extract and visualize the results of multivariate data analyses</em>.</a> </li><li>Kemp, B. M. and Smith, D. G. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16182949" target="_blank">Use of bleach to eliminate contaminating DNA from the surface of bones and teeth.</a> <em>Forensic Sci Int</em> 154(1): 53-61. </li><li>LeBoldus, J. M., Blenis, P. V. and Thomas, B. R. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30743608" target="_blank">A method to induce stem cankers by inoculating nonwounded <em>Populus</em> clones with <em>Septoria musiva</em> spore suspensions.</a> <em>Plant Dis</em> 94(10): 1238-1242. </li><li>Lu, Y. (2011). <a href="https://bio-protocol.org/e90" target="_blank">Extract genomic DNA from <em>Arabidopsis</em> leaves (can be used for other tissues as well).</a> <em>Bio-protocol</em> 1(13): e90.</li><li>Nelsen, M. P., LÃ¼cking, R., Mbatchou, J. S., Andrew, C. J., Spielmann, A. A. and Lumbsch, H. T. (2011). <a href="https://link.springer.com/article/10.1007/s13225-011-0144-7" target="_blank">New insights into relationships of lichen-forming <em>Dothideomycetes</em>.</a> <em>Fungal Diversity</em> 51(1): 155-162.</li><li>Porebski, S., Bailey, L. G. and Baum, B. R. (1997). <a href="https://link.springer.com/article/10.1007/BF02772108" target="_blank">Modification of a CTAB DNA extraction protocol for plants containing high polysaccharide and polyphenol components.</a> <em>Plant Mol Biol Rep</em> 15(1): 8-15.</li><li>Saini, H. S., Shepherd, M. and Henry, R. J. (1999). <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/j.2050-0416.1999.tb00018.x" target="_blank">Microwave extraction of total genomic DNA from barley grains for use in PCR.</a> <em>J Inst Brew</em> 105(3): 185-190.</li><li>Schweigkofler, W., W.J, O., S.L, S., D.R, C., Maeda, K., Peay, K. and Garbelotto, M. (2005). <a href="https://www.fs.usda.gov/treesearch/pubs/25271" target="_blank">Detection and quantification of <em>Leptographium wageneri</em>, the cause of black-stain root disease, from bark beetles (<em>Coleoptera: Scolytidae</em>) in Northern California using regular and real-time PCR.</a> <em>Canadian Journal of Forest Research</em> 35: 1798-1808. </li><li>Terhonen, E., Langer, G. J., BuÃkamp, J., RÓscuÅ£oi, D. R. and Blumenstein, K. (2019). <a href="https://www.mdpi.com/1999-4907/10/1/55" target="_blank">Low water availability increases necrosis in <em>Picea abies</em> after artificial inoculation with fungal root rot pathogens <em>Heterobasidion</em> <em>parviporum</em> and <em>Heterobasidion</em> <em>a</em><em>nnosum</em><em>.</em></a> <em>Forests</em><em> 10(1): 55. </em> </li><li>Terhonen, E., Marco, T., Sun, H., Jalkanen, R., Kasanen, R., Vuorinen, M. and Asiegbu, F. (2011). <a href="https://silvafennica.fi/article/104/selected/104" target="_blank">The effect of latitude, season and needle-age on the mycota of Scots pine (<em>Pinus sylvestris</em>) in Finland.</a> <em>Silva Fenn</em> 45</li><li>Vilgalys, R. and Hester, M. (1990). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2376561" target="_blank">Rapid genetic identification and mapping of enzymatically amplified ribosomal DNA from several <em>Cryptococcus</em> species.</a> <em>J Bacteriol</em> 172(8): 4238-4246.</li><li>White, T., Bruns, T., Lee, S., Taylor, J., Innis, M., Gelfand, D. and Sninsky, J. (1990). <a href="https://www.researchgate.net/publication/262687766_Amplification_and_Direct_Sequencing_of_Fungal_Ribosomal_RNA_Genes_for_Phylogenetics" target="_blank">Amplification and direct sequencing of fungal ribosomal RNA genes for phylogenetics.</a> In: <em>PCR Protocols</em>, Academic Press, Inc., pp. 315-322.</li><li>Winton, L. M., Stone, J. K., Watrud, L. S. and Hansen, E. M. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18944147" target="_blank">Simultaneous one-tube quantification of host and pathogen DNA with real-time polymerase chain reaction.</a> <em>Phytopathology</em> 92(1): 112-116.</li><li>Yi, S., Jin, W., Yuan, Y. and Fang, Y. (2018). <a href="https://en.bio-protocol.org/e2906" target="_blank">An optimized CTAB method for genomic DNA extraction from freshly-picked pinnae of fern, <em>Adiantum capillus-veneris</em> L.</a> <em>Bio-protocol</em> 8(13): e2906.<br/></li></ol>