<body>
            <h1><em>Ex vivo</em> Culture Assay Using Human Hair Follicles to Study Circadian Characteristics</h1>
            <p><strong>Authors:</strong> Atsuhiro Nishida, Yoshiki Miyawaki, Koichi Node, Makoto Akashi</p>
            <h2>Abstract</h2>
            <p><em>Ex vivo</em> culture assays of biopsy specimens are advantageous for the experimental evaluation of human circadian characteristics. We developed a simple and non-invasive experimental evaluation method for monitoring the expression of circadian clock genes in an <em>ex vivo</em> culture assay using human hair follicles. This method imposes little burden on subjects. This assay is useful for validating correlations between circadian characteristics in hair follicles and intrinsic characteristics observed in physiological and behavioral studies. While they should be further validated, this <em>ex vivo</em> method constitutes a useful tool for estimating <em>in vivo</em> circadian characteristics.</p>
            <p><strong>Keywords:</strong> Circadian rhythm, Clock gene, Human, Hair follicle, <em>Ex vivo</em>, Non-invasive</p>
            <h2>Background</h2><p>Living organisms exhibit circadian rhythms in physiology and behavior that are driven by the circadian clock (Young and Kay, 2001). The circadian clockwork consists of cell-autonomous and clock gene-driven negative feedback loops of transcription (Dunlap, 1999). In mammals, the transcription factors BMAL1 and CLOCK activate the transcription of clock and clock-related genes such as <em>Period</em> (<em>Per</em>) and <em>Cryptochrome</em> (<em>Cry</em>) via E-box elements. PER together with CRY, a potent transcriptional inhibitor, subsequently functions to negatively regulate this complex (Reppert and Weaver, 2002). The <em>in vivo</em> evaluation of individual intrinsic circadian characteristics in humans, either with a constant routine or forced desynchrony protocol, is expensive and labor-intensive. Therefore, evaluation using <em>ex vivo</em> culture assays to estimate <em>in vivo</em> circadian characteristics could present important advantages. For example, several studies have concluded that circadian characteristics in peripheral cells reflect individual circadian preferences, known as chronotype (Brown <em>et al.</em>, 2005; Hida <em>et al.</em>, 2013). For a simple and non-invasive evaluation of cell-autonomous circadian performance in humans, including intrinsic period length, we developed a method to monitor clock gene expression in real time using an <em>ex vivo</em> culture of hair follicles (Yamaguchi <em>et al.</em>, 2017).</p>
        </body>

<p class="pptt" id="biaoti34164">## Materials and Reagents</p><ol class="npwtwok"><li>(Optional) Keep-warm bag</li><li>Sterile sampling tubes (the volume size should be within 0.2 from 1.5 ml) (<em>e.g.</em>, 0.2 ml tube, Bio-Bik, catalog number: <a class="blackNoUnder" target="_blank"> 133003 </a> ; 1.5 ml tube, VIOLAMO, catalog number: <a class="blackNoUnder" target="_blank"> 1-1600-1 </a> )</li><li>35 mm dishes (<em>e.g.</em>, IWAKI, catalog number: <a class="blackNoUnder" target="_blank"> 1000-035 </a> )</li><li>DMEM without Phenol Red (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D2902 </a> )</li><li>DMEM (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 08456-94 </a> )<br/>Although DMEM from other suppliers is usable as a substitute for these DMEMs, the possibility cannot be excluded that the difference in concentrations of minor ingredients affects success rates and results slightly.</li><li>Sodium bicarbonate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S8761 </a> )</li><li>HEPES (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 17557-94 </a> )</li><li>D-Glucose (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> G8769 </a> )</li><li>Penicillin/streptomycin (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15070-063 </a> )</li><li>L-Glutamin (Nacalai, catalog number: <a class="blackNoUnder" target="_blank"> 16948-04 </a> )</li><li>Sodium pyruvate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S8636 </a> )</li><li>Luciferin (WAKO, catalog number: <a class="blackNoUnder" target="_blank"> 126-05116 </a> )</li><li>Silicone (Shin-Etsu, catalog number: <a class="blackNoUnder" target="_blank"> KS-64 </a> )<br/><em>Note: Reagents #6-13: Similar items from other suppliers serve as a substitute.</em> </li><li>Adenovirus vectors carrying the luciferase (luc) gene driven by circadian promoter/enhancer elements (<em>e.g.</em>, <em>Bmal1-luc</em>, <em>Per2-luc</em> and <em>Per3-luc</em>). Some entry vectors we constructed and deposited are available from the RIKEN bank as follows:<br/>RDB15083 mouse <em>Period2</em> (-2.8 ~ +0.1 kb) - <em>Luciferase</em>/pENTR-1A<br/>RDB15084 human <em>Period3</em> (-3.1 ~ +0.2 kb) - <em>Luciferase</em>/pENTR-1A<br/>RDB15085 human <em>Bmal1</em> (-1.7 ~ +0.1 kb) - <em>Luciferase</em>/pENTR-1A<br/>The ViraPower Adenoviral Gateway Expression Kit (K4940-00, Thermo Fisher Scientific) is required to generate adenoviruses using these entry vectors. Procedures using adenoviruses must be performed inside P2 areas. Autoclave and discard any infectious wastes</li><li>Infection medium (see Recipes)</li><li>Pre-incubation medium (see Recipes)</li><li>Rinse medium (see Recipes)</li><li>Luciferin-containing medium (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34165">### Equipment</p><ol class="npwtwok"><li>Non-slip, cosmetic-use tweezers (<em>e.g.</em>, Shiseido, eyebrow nippers 211)</li><li>Tweezers (<em>e.g.</em>, FST, catalog number: <a class="blackNoUnder" target="_blank"> 18132-12 </a> )</li><li>Block incubator set to 36.5 Â°C (<em>e.g.</em>, ASTEC, model: <a class="blackNoUnder" target="_blank"> BI-516C </a> )<br/></li><li>Laminar flow cabinet (<em>e.g.</em>, SANYO, model: <a class="blackNoUnder" target="_blank"> MCV-B131F </a> )<br/><em>Note: Equipment #1-4: Similar items from other suppliers serve as a substitute.</em> </li><li>Incubator at 35-36.5 Â°C with 5% CO<sub>2</sub> (<em>e.g.</em>, ASTEC, model: <a class="blackNoUnder" target="_blank"> SCA-165DRS </a> )<br/>Use a CO<sub>2</sub> incubator that utilizes an infrared sensor that is not affected by humidity inside the chamber (see details in Procedure Step 18).</li><li>Photomultiplier tube (Hamamatsu, model: <a class="blackNoUnder" target="_blank"> LM2400 </a> )</li><li>Luminescence microscope (Olympus, model: <a class="blackNoUnder" target="_blank"> LV200 </a> )<br/></li></ol><p class="pptt" id="biaoti34166">### Software</p><ol class="npwtwok"><li>Cosinor software (freely downloadable from a website: <a href="https://www.circadian.org/softwar.html" target="_blank">https://www.circadian.org/softwar.html</a>)</li></ol><p class="pptt" id="biaoti34167">### Procedure</p><p class="ppzz"><em>Ex vivo</em> culture of human hair follicles to investigate circadian gene expression:</p><ol class="npwtwo"><li>Human studies must be conducted in accordance with the Declaration of Helsinki and approved by institutional review boards.</li><li>Recruit subjects and obtain informed consent from each subject or the subject's family.</li><li>(Optional) Although dependent on research aim, volunteers may be asked to follow a set schedule for one or several weeks. For example, wake-up time and meal times may be set based on the lifestyle habits of each volunteer. They may also be asked to refrain from consuming excess alcohol and snacks and taking long naps. Individual chronotypes can be assessed using the morningness-eveningness questionnaire (MEQ) or the Munich chronotype questionnaire (MCTQ) (Zavada <em>et al.</em>, 2005).</li><li>Prepare infection medium (see Recipes) by adding an adenovirus carrying the luciferase gene driven by circadian promoter/enhancer elements to fresh DMEM at a 1:20 dilution. The titer of our adenovirus stocks is within a range of 1.24 x 10<sup>9</sup>-1.47 x 10<sup>9</sup> infectious units/ml. The use of adenovirus vectors carrying <em>Per3-luc</em> may be recommended (see the following note for this reason). For vector construction, we used the ViraPower Adenoviral Gateway Expression Kit (Yamaguchi <em>et al.</em>, 2017). Briefly, together with the luciferase gene, the transcription-regulatory region of a clock gene was subcloned into the pENTR-1A entry vector and subsequently introduced into the pAd/PL-DEST vector using LR recombination.<br/><em>Note: We previously compared period length, robustness and standard deviation (SD) among three adenovirus vectors carrying Bmal1-luc, </em><em>Per2-luc</em><em> and </em><em>Per3-luc</em><em> (Yamaguchi </em><em>et al.</em><em>, 2017). Although the average period length was almost identical among the three vectors, the SD of the period length of Per3 was consistently smaller than that of the others. Additionally, the period length of Per3 was the most robust at the smallest SD values. We therefore preferably used the adenovirus vector carrying </em><em>Per3-luc</em><em> in our experimental conditions.</em> </li><li>Before sampling hair follicles, prepare sterile sampling tubes containing 80 Î¼l of pre-incubation medium, rinse medium or infection medium (see Recipes for the composition of each medium). One tube containing the respective medium is required per hair follicle. Pre-warm and maintain medium at 36.5 Â°C using a block incubator.</li><li>(Optional) On cold winter days, warming the carotid arteries by attaching pocket warmers to both sides of the neck may improve the success rate of hair follicle sampling.</li><li>As shown in Figure 1, collect several strands of scalp (or facial) hair by firmly holding and pulling individual strands of hair by the root using non-slip tweezers (for cosmetic use). To avoid hair follicle cell death by drying, quickly immerse the hair follicle cells into pre-warmed pre-incubation medium without removing the hair shaft. Keep samples at 36.5 Â°C until the next step (infection: Step 11). Although hair collection does not necessarily require a sterile environment, procedures after Step 11 are recommended to be performed under sterile conditions.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603021824_3128.jpg" width="265"/><br/><strong>Figure 1. Collection of strands of scalp hair. </strong>A. Firmly hold and pull individual strands of hair by the root using non-slip tweezers. B. Use plucked hairs whose root surface is mostly or fully covered with hair follicle cells. C. Quickly immerse hair follicle cells in pre-warmed pre-incubation medium without removing the hair shaft.<br/><br/></li><li>Cut the hair shaft, leaving about 10 to 20 mm.</li><li>To obtain a strong bioluminescence signal, use plucked scalp (or facial) hairs whose root surface is mostly or fully covered with hair follicle cells. A representative image showing successful infection of an adenovirus carrying <em>Per2-luc</em> to a hair root mostly covered with hair follicle cells is shown in our original paper (Figure 1B, Yamaguchi <em>et al.</em>, 2017). We previously investigated the effect of differences in hair stage on circadian period length and found no significant differences in period length among stages (Yamaguchi <em>et al.</em>, 2017).</li><li>(Optional) In the case of out-of-lab sampling, maintain samples in a keep-warm bag without CO<sub>2</sub> during transport to the laboratory. We confirmed that shipping duration up to approximately 10 h had no obvious effect on the circadian period length of clock gene expression.</li><li>Inside a laminar flow cabinet, use sterilized tweezers to pick up the hair shaft, immerse the hair follicles into pre-warmed rinse medium, and transfer and completely immerse the follicles into pre-warmed infection medium on a block incubator (36.5 Â°C). Procedures using adenoviruses must be performed inside P2 areas. Autoclave and discard any infectious wastes.</li><li>Close the lid of tubes and incubate them for 24 h at 36.5 Â°C with 5% CO<sub>2</sub>.</li><li>The next day, prepare sterile sampling tubes containing pre-warmed rinse medium (at 36.5 Â°C) for the removal of viruses. Two tubes are required for washing twice.</li><li>Syringe and push out silicone into a mound on the bottom of 35 mm culture dishes (Figure 2 and Video 1). Place the silicone somewhat off-center to facilitate locating the hair follicle at the center of the culture dish because reducing the distance between a PMT surface and infected cells enhances bioluminescence counts. Use one dish per hair follicle. Sterilization of silicone is not essential, but we recommend using silicone from an unopened packet designated for culture use. In addition, prepare and pre-warm luciferin-containing medium (see Recipes for the composition of each medium) at a concentration of 0.1 mM.<br/><br/><!--video3638-1å¼å§--><div class="videodiv" id="vd3638-1" style="height:480px;"></div><div class="videophoto" id="vp3638-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3638/3638-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Placing silicone in a mound on the bottom of the culture dishes</strong> </div><!--video3638-1ç»æ--><br/></li><li>Use sterile tweezers to pick up the hair by the shaft and transfer the infected hair follicles into 80 Î¼l of pre-warmed rinse medium. After one-minute incubation, again transfer infected hair follicles into the same volume of fresh rinse medium and incubate them for the same duration to remove adenoviruses as completely as possible.</li><li>Transfer the hair follicle to the silicone mound in a 35 mm culture dish for bioluminescence monitoring (Video 2). To avoid floating during measurement, push the hair shaft into the silicone so that the shaft is stuck to the silicone and is fixed on the bottom of the dish (Figure 2). Position the hair follicle near to the center of the dish for efficient bioluminescence detection.<br/><br/><!--video3638-2å¼å§--><div class="videodiv" id="vd3638-2" style="height:480px;"></div><div class="videophoto" id="vp3638-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3638/3638-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. Transferring a hair follicle to a silicone mound</strong> </div><!--video3638-2ç»æ--><br/></li><li>Completely cover the immobilized hair follicle with 2-3 ml pre-warmed luciferin-containing medium.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603021854_8137.jpg" width="262"/><br/><strong>Figure 2. Fixing a hair follicle to the bottom of a culture dish.</strong> A. Transfer a hair follicle to a silicone mound in a 35 mm culture dish. Push the hair shaft into the silicone so that the shaft is stuck to the silicone and is fixed to the bottom of the dish. B. Position the hair follicle near to the center of the dish.<br/><br/></li><li>As shown in Figure 3, measure bioluminescence in real-time using a photomultiplier tube inside a dark box specifically designed to reduce background noise to detect ultra-weak photon emissions (<a class="blackNoUnder" target="_blank"> LM2400 </a> , Hamamatsu, Japan) or a luminescence microscope optimized for single-cell imaging (Olympus, Japan) at 35 Â°C with 5% CO<sub>2</sub>. To avoid rust formation, the <a class="blackNoUnder" target="_blank"> LM2400 </a> is located inside a CO<sub>2</sub> culture incubator under low humidity conditions. To prevent the culture medium from drying out, instead of using the water tray from the CO<sub>2</sub> incubator, use the one inside the <a class="blackNoUnder" target="_blank"> LM2400 </a> . Use a CO<sub>2</sub> incubator that utilizes an infrared sensor that is not affected by humidity inside the chamber. For measurement, open the top of the <a class="blackNoUnder" target="_blank"> LM2400 </a> and simply place culture dishes on the metal tray. Culture dishes without hair follicles should provide reads of 5,000-10,000 counts per minute (background control). In successful cases, bioluminescence from the hair follicles should be greater than the background counts and show circadian oscillation over more than four to five days.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603021921_9938.jpg" width="184"/><br/><strong>Figure 3. Monitoring bioluminescence in real time.</strong> Bioluminescence can be measured in real time using a photomultiplier tube (top three panels, <a class="blackNoUnder" target="_blank"> LM2400 </a> ) or a luminescence microscope optimized for single-cell imaging (bottom three panels, <a class="blackNoUnder" target="_blank"> LV200 </a> ) at 35 to 36.5 Â°C with 5% CO<sub>2</sub>. Left panels, wide views of the instruments; middle panels, views of sample holders; right panels, views of culture dishes on the sample holders.</li></ol><p class="pptt" id="biaoti34168">### Data analysis</p><p class="ppzz">Data collection is performed using the associated software âSoftware for Photon Detection Unit C10749 (<a class="blackNoUnder" target="_blank"> LM2400 </a> v21)-JPâ. To analyze circadian parameters, baseline changes need to be removed. As shown in Figure 4, data sets are therefore detrended using Microsoft Excel by subtracting the 24 h running average from the raw data (A, raw data; B, detrended data). Circadian robustness, circadian phase (angle) and circadian period length are calculated using Cosinor software, provided as a gift by Dr. Refinetti. Oscillation data are considered reliable and useful not only when clear circadian oscillation persists for â¥ 4 days but also when circadian robustness is 70% or greater.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603021945_2039.jpg" width="221"/><br/><strong>Figure 4. Example data.</strong> Data sets are detrended by subtracting the 24 h running average from the raw data. A. Raw data (software, Photon Detection Unit C10749 (<a class="blackNoUnder" target="_blank"> LM2400 </a> v21)-JP); B. detrended data (software, Microsoft Excel).</p><p class="pptt" id="biaoti34169">### Notes</p><ol class="npwtwok"><li>Although the absolute value of circadian period length in cultured hair follicles differs from those obtained from physiological and behavioral studies, we confirmed that the relative difference in period length among mouse genotypes is similar between clock gene expression in hair follicles and locomotor activity (Yamaguchi <em>et al.</em>, 2017). Our <em>ex vivo</em> method may therefore be a useful tool for estimating <em>in vivo</em> circadian characteristics.</li><li>Culture medium composition reportedly affects period length (Lee <em>et al.</em>, 2011; Noguchi <em>et al.</em>, 2012). For example, we have found that the absence of phenol red results in damping and relatively longer periods (Yamaguchi <em>et al.</em>, 2017). It is therefore important to use identical medium composition and product lot numbers throughout all sets of experiments for comparison of relative differences between subjects.</li><li>Dexamethasone (DEX) is a well-known synchronizer for peripheral clocks. Peripheral clock gene expression is often monitored <em>in vitro</em> and <em>ex vivo</em> after DEX treatment. However, adenoviral infection can also synchronize clock gene expression. To simplify the experimental protocol and reduce cellular stress, DEX treatment is therefore excluded when using adenovirus-infected hair follicles.<br/></li></ol><p class="pptt" id="biaoti34170">### Recipes</p><ol class="npwtwok"><li>Infection medium (80 Î¼l or more per hair follicle)<br/>DMEM (Nacalai, Japan)<br/>1% penicillin/streptomycin<br/>5% Adenovirus</li><li>Pre-incubation medium (80 Î¼l or more per hair follicle)<br/>DMEM without Phenol Red (Sigma-Aldrich)<br/>0.035% sodium bicarbonate<br/>10 mM HEPES<br/>4.5 g/L D-Glucose<br/>1% penicillin/streptomycin<br/>1 mM L-Glutamin<br/>1 mM sodium pyruvate</li><li>Rinse medium (80 Î¼l or more for each washing)<br/>DMEM (Nacalai, Japan)<br/>1% penicillin/streptomycin</li><li>Luciferin-containing medium (2-3 ml per hair follicle)<br/>DMEM (Nacalai, Japan)<br/>1% penicillin/streptomycin<br/>0.1 mM luciferin<br/></li></ol><p class="pptt" id="biaoti34171">### Acknowledgments</p><p class="ppzz">We thank Ai Yamaguchi, Akihiko Okamoto, Ritsuko Matsumura, Miho Sato, Rie Okamitsu and Junko Sumino for their expert technical assistance. We also acknowledge the support of fellowships from the Yamaguchi Gerontology Research Institute, the Akaeda Medical Research Foundation, the SENSHIN Medical Research Foundation, and the Japan Society for the Promotion of Science. The authors declare no competing financial interests. This protocol was originally developed in Yamaguchi <em>et al.</em>, 2017.</p><p class="pptt" id="biaoti34172">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti34173">### Ethics</p><p class="ppzz">This study was conducted in accordance with the Declaration of Helsinki and was approved by the institutional review boards of Yamaguchi University (approval number: H25-81-4). Informed consent was obtained from all subjects.</p><p class="pptt" id="biaoti34174">### References</p><ol class="npwtwok"><li>Brown, S. A., Fleury-Olela, F., Nagoshi, E., Hauser, C., Juge, C., Meier, C. A., Chicheportiche, R., Dayer, J. M., Albrecht, U. and Schibler, U. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16167846" target="_blank">The period length of fibroblast circadian gene expression varies widely among human individuals.</a> <em>PLoS Biol</em> 3(10): e338.</li><li>Dunlap, J. C. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9988221" target="_blank">Molecular bases for circadian clocks.</a> <em>Cell</em> 96(2): 271-290.</li><li>Hida, A., Kitamura, S., Ohsawa, Y., Enomoto, M., Katayose, Y., Motomura, Y., Moriguchi, Y., Nozaki, K., Watanabe, M., Aritake, S., Higuchi, S., Kato, M., Kamei, Y., Yamazaki, S., Goto, Y., Ikeda, M. and Mishima, K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23797865" target="_blank"><em>In vitro</em> circadian period is associated with circadian/sleep preference.</a> <em>Sci Rep</em> 3: 2074.</li><li>Lee, S. K., Achieng, E., Maddox, C., Chen, S. C., Iuvone, P. M. and Fukuhara, C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22115778" target="_blank">Extracellular low pH affects circadian rhythm expression in human primary fibroblasts.</a> <em>Biochem Biophys Res Commun</em> 416(3-4): 337-342.</li><li>Noguchi, T., Wang, C. W., Pan, H. and Welsh, D. K. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22734566" target="_blank">Fibroblast circadian rhythms of PER2 expression depend on membrane potential and intracellular calcium.</a> <em>Chronobiol Int</em> 29(6): 653-664.</li><li>Reppert, S. M. and Weaver, D. R. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12198538" target="_blank">Coordination of circadian timing in mammals.</a> <em>Nature</em> 418(6901): 935-941.</li><li>Yamaguchi, A., Matsumura, R., Matsuzaki, T., Nakamura, W., Node, K. and Akashi, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28755004" target="_blank">A simple method using <em>ex vivo</em> culture of hair follicle tissue to investigate intrinsic circadian characteristics in humans.</a> <em>Sci Rep</em> 7(1): 6824.</li><li>Young, M. W. and Kay, S. A. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11533719" target="_blank">Time zones: a comparative genetics of circadian clocks.</a> <em>Nat Rev Genet</em> 2(9): 702-715.</li><li>Zavada, A., Gordijn, M. C., Beersma, D. G., Daan, S. and Roenneberg, T. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16021843" target="_blank">Comparison of the munich chronotype questionnaire with the horne-ostberg's morningness-eveningness score.</a> <em>Chronobiol Int</em> 22(2): 267-278.<br/></li></ol>