<body>
            <h1>Biochemical Pulldown of mRNAs and Long Noncoding RNAs from Cellular Lysates Coupled with Mass Spectrometry to Identify Protein Binding Partners</h1>
            <p><strong>Authors:</strong> Anca F. Savulescu, Stoyan  Stoychev, Sipho Mamputha, Musa M. Mhlanga</p>
            <h2>Abstract</h2>
            <p>RNA binding proteins (RBPs) interact with cellular mRNAs, controlling various steps throughout the lifetime of these transcripts, including transcription, cellular transport, subcellular localization, translation and degradation. In addition to binding mRNA transcripts, a growing number of RBPs are shown to bind long noncoding RNAs (lncRNAs), controlling key cellular processes, including gene expression and translation of proteins. Current methodologies aimed at identifying and characterizing protein binding partners of specific RNAs of interest typically rely on tagging of the RNA with affinity aptamers, using <em>in vitro</em> transcribed RNA or immobilized oligonucleotides to capture RNA-protein complexes under native conditions. These assays are coupled with mass spectrometry or Western Blot analysis to identify or/and confirm interacting proteins. Here, we describe an alternative approach to identify protein binding partners of mRNAs and large long noncoding RNAs. This approach relies on biochemical pulldown of specific target RNAs and interacting protein partners from cellular lysates coupled with mass spectrometry to identify novel interacting proteins. By using 24-48 ~20 mer biotinylated DNA probes that hybridize to the target RNA, the method ensures high specificity and minimal off target binding. This approach is reproducible and fast and serves as a base for discovery studies to identify proteins that bind to RNAs of interest.</p>
            <p><strong>Keywords:</strong> mRNA, Long noncoding RNA, RNA Binding Protein, RNA-protein interaction, Mass spectrometry, Biochemical pulldown</p>
            <h2>Background</h2><p>RNA-binding proteins (RBPs) interact with mRNAs in cells to form ribonucleoprotein (RNP) complexes that regulate the post-transcriptional activity of these transcripts by affecting their processing, subcellular localization, translation and stability (reviewed in Lunde <em>et al.</em>, 2007; Glisovic <em>et al.</em>, 2008). Recent studies have expanded the number of mRNA binding proteins in mammalian cells (for example Baltz <em>et al.</em>, 2012; Castello <em>et al.</em>, 2012; Kwon <em>et al.</em>, 2013); additionally, a growing number of long noncoding RNAs (lncRNAs) have been shown to bind RBPs (for example Gumireddy <em>et al.</em>, 2013; Ng <em>et al.</em>, 2013; McHugh <em>et al.</em>, 2015; Yoon and Gorospe, 2016), emphasizing the importance of characterizing the roles of RNA-protein interactions. There are several methodologies available to date to identify and characterize protein binding partners of specific RNAs (reviewed in Ramanathan <em>et al.</em>, 2019). Shortly, these include tagging of RNAs of interest with affinity aptamers (Slobodin and Gerst, 2010), using <em>in vitro</em> transcribed RNAs (Abdelmohsen <em>et al.</em>, 2011; Tominaga <em>et al.</em>, 2011; Grammatikakis <em>et al.</em>, 2016), RNA affinity pulldown typically using a limited number of RNA or DNA oligonucleotides (Kar <em>et al.</em>, 2011; Castello <em>et al.</em>, 2012; Yoon <em>et al.</em>, 2012; Abdelmohsen <em>et al.</em>, 2014; Zhang <em>et al.</em>, 2016) coupled with Western blot or mass spectrometry analyses (Simon <em>et al.</em>, 2011, McHugh <em>et al.</em>, 2015, Di Tomasso <em>et al.</em>, 2016, Kar <em>et al.</em>, 2017, McHugh and Guttman, 2018), or immobilized oligonucleotides to capture RNA-protein complexes under native conditions (Zeng <em>et al.</em>, 2006).<br/> <br/>Here, we built upon the Chromatin Isolation by RNA Purification (ChIRP) method (Chu <em>et al.</em>, 2011 and 2012), adapted it to maximize the yield of proteins bound to specific target RNAs and coupled it with mass spectrometry analysis for thorough identification and characterization of protein binding partners of these RNA transcripts (Figure 1). Similarly to the ChIRP approach (Chu <em>et al.</em>, 2011), to obtain high specificity in isolation of a specific target RNA we used sets of pre-biotinylated single molecule FISH probes. A typical set is comprised of 24-48 ~20 mer DNA probes, complementary to and tiled along the target RNA. As in the ChIRP protocol (Chu <em>et al.</em>, 2011), we used glutaraldehyde crosslinking as compared to other crosslinking methods, it yielded the highest signal-to-noise ratio. However, this protocol can be adjusted to use various cross linkers, including ultraviolet light and formaldehyde, to focus on specific interactions of interest.<br/> <br/>Typically, we grow cells in desired conditions, cross link them and freeze cell pellets for future use or use them immediately. Cell pellets are lysed and pre-cleared cell lysates are incubated with a set of 24-48 biotinylated DNA probes, followed by additional incubation with streptavidin beads (Figure 1). Eluted proteins are separated using SDS-PAGE with each lane cut into 8 equal sections and in-gel digested as per Shevchenko <em>et al.</em>, 2006 (Figure 1). Extracted peptides are LC-MSMS analyzed (Figure 1). This method allows for the identification and characterization of protein binding partners of specific RNAs of interest.</p>
        </body>

<p class="pptt" id="biaoti34177">## Materials and Reagents</p><ol class="npwtwok"><li>150 mm tissue culture plate (TPP, catalogue number: <a class="blackNoUnder" target="_blank"> 93150 </a> )</li><li>Cell scrapers (TPP, catalog number: <a class="blackNoUnder" target="_blank"> 99003 </a> )</li><li>Falcon tubes (TPP catalog number: <a class="blackNoUnder" target="_blank"> 91015 </a> )<br/></li><li>Surgical Blade (Swann-Morton, catalog number: 0<a class="blackNoUnder" target="_blank"> 103 </a> )</li><li>96 Deepwell Plate (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 95040450 </a> )</li><li>2 ml Protein LoBind Tubes (Eppendorf, catalog number: 00<a class="blackNoUnder" target="_blank"> 30108132 </a> )</li><li>1.5 ml Protein LoBind Tubes (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> Z666505 </a> )</li><li>0.5 ml Protein LoBind Tubes (Eppendorf, catalog number: 00<a class="blackNoUnder" target="_blank"> 30108094 </a> )</li><li>12 Channel Pipette, 30-300 Î¼l (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 3122000060 </a> )</li><li>10% NP-40 (Thermo Fischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 85124 </a> )</li><li>PBS (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 10010-015 </a> )</li><li>Glutaraldehyde (Merk, catalog number: <a class="blackNoUnder" target="_blank"> 111-30-8 </a> )</li><li>Glycine (Fluka, catalog number: <a class="blackNoUnder" target="_blank"> 15527H </a> )</li><li>Protease inhibitors cOmplete, Mini, EDTA-free (Roche, catalog number: 0<a class="blackNoUnder" target="_blank"> 4693159001 </a> )</li><li>DTT (Merck, catalog number: 3483-12-3)</li><li>Superase-In (Ambion, catalog number: <a class="blackNoUnder" target="_blank"> AM2694 </a> )</li><li>Stellaris biotinylated probes (custom made by Biosearch as per gene of interest)</li><li>MagReSyn<sup>Â®</sup> Streptavidin Affinity binding/capture of biotinylated biomolecules (AEC-AMERSHAM SOC Ltd, catalog number: <a class="blackNoUnder" target="_blank"> MR-STV002 </a> )</li><li>Precision Plus Protein Dual Color Standards (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 161-0374 </a> )</li><li>4x Laemmli sample buffer (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1610747 </a> )<br/></li><li>Coomassie Brilliant Blue R-250 (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1610436 </a> )</li><li>Bovine Serum Albumin Fraction V (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 10735078001 </a> )</li><li>LC-MS Water (Honeywell, catalog number: <a class="blackNoUnder" target="_blank"> LC365 </a> )</li><li>Ammonium Bicarbonate (Fluka, catalog number: 0<a class="blackNoUnder" target="_blank"> 9830 </a> )</li><li>Methanol (Honeywell, catalog number: <a class="blackNoUnder" target="_blank"> LC230 </a> )</li><li>Acetonitrile (Honeywell, catalog number: <a class="blackNoUnder" target="_blank"> LC015 </a> )</li><li>Iodoacetamide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> I1149 </a> )</li><li>Trypsin (Promega, catalog number: <a class="blackNoUnder" target="_blank"> V511A </a> )</li><li>Formic Acid (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 28905 </a> )</li><li>Tris-HCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T1503-250G </a> )</li><li>KCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P9333-1KG </a> )</li><li>Triton X100 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> X-100-100ML </a> )</li><li>NaCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> s7653-1KG </a> )</li><li>SDS (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L3771-100G </a> )</li><li>Formamide (Thermo Fischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 17899 </a> )</li><li>20x SSC (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S6639-1L </a> )</li><li>RNase A (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 10109142001 </a> )</li><li>RNase H (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 10786357001 </a> )</li><li>DNase 1 (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> EN0521 </a> )<br/></li><li>Lysis Buffer (see Recipes)</li><li>Hybridization Buffer (see Recipes)</li><li>Washing Buffer (see Recipes)</li><li>DNase Buffer (see Recipes)</li><li>Elution Buffer (see Recipes)</li><li>Destaining Buffer (see Recipes)</li><li>Dehydration Buffer 1 (see Recipes)</li><li>Reduction Buffer (see Recipes)</li><li>Alkylation Buffer (see Recipes)</li><li>Dehydration Buffer 2 (see Recipes)</li><li>Digestion Buffer (see Recipes)</li><li>Trypsin Solution (see Recipes)</li><li>Extraction Buffer (see Recipes)</li><li>Loading Solvent (see Recipes)</li><li>Gradient solvent A (see Recipes)</li><li>Gradient solvent B (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34178">### Equipment</p><ol class="npwtwok"><li>Refrigerated microcentrifuge (Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> 22R </a> )</li><li>Shaker (TCS Scientific Corp Nutator)</li><li>Rotator (Intelli-mixer RM-2L)</li><li>Magnetic separator (Resyn Biosciences)</li><li>Deepwell Plate Thermal Mixer (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5355 </a> )</li><li>Deepwell Plate Vacuum Drying System (Gel plugs dried in vacuum desiccator jar trough Agilentâs DS 102 dual stage rotary vane pump, catalog number: <a class="blackNoUnder" target="_blank"> 9499315 </a> )</li><li>Vacuum Concentrator (Labconco, catalog number: <a class="blackNoUnder" target="_blank"> 7310035 </a> )</li><li>Water Bath Sonicator (Bandelin, catalog number: <a class="blackNoUnder" target="_blank"> 301 </a> )</li><li>Dionex Ultimate 3000 RSLC System (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> IQLAAAGABHFAPBMBEZ </a> )</li><li>Desalting Column (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 164567 </a> )</li><li>Separation Column (Thermo, catalog number: <a class="blackNoUnder" target="_blank"> 164534 </a> )<br/></li></ol><p class="pptt" id="biaoti34179">### Software</p><ol class="npwtwok"><li>Protein Pilot (Sciex, <a href="https://sciex.com/products/software/proteinpilot-software" target="_blank">https://sciex.com/products/software/proteinpilot-software</a>)</li></ol><p class="pptt" id="biaoti34180">### Procedure</p><ol class="npwone"><li>Crosslinking of cells<ol class="npwtwo"><li>Grow ~1 x 10<sup>7</sup> cells in desired conditions in 150 mm plates (in a volume of 30 ml medium).<br/><em>Note: Higher number of cells can be used, we found 1 x 10</em><sup><em>7</em></sup><em> cells to be the minimal amount required to obtain data at high resolution.</em> </li><li>Wash in 20 ml 1x PBS.</li><li>Crosslink cells in the plate with 20 ml 1% Glutaraldehyde in 1x PBS for 10 min at RT on a shaker.<br/><em>Note: Alternative cross linkers can be used instead of Glutaraldehyde, including Ultraviolet light and Formaldehyde. The protocol has to be adjusted accordingly.</em> </li><li>Quench with 20 ml 0.125 M Glycine for 5 min at RT, on a shaker.</li><li>Wash with 20 ml ice-cold 1x PBS.</li><li>Scrape cells and transfer to 15 ml falcon tubes.</li><li>Centrifuge cells at 2,500 <em>x g</em>, 5 min, at 4 Â°C and discard supernatant.</li><li>Snap freeze pellet and store at -80 Â°C or use immediately.<br/><br/></li></ol></li><li>Lysis of cells<ol class="npwtwo"><li>Resuspend pellet in 250-300 Î¼l Lysis buffer + 1x protease Inhibitors.</li><li>Transfer the re-suspended cells to an Eppendorf, mix well and incubate at RT for 25 min on a shaker.</li><li>Centrifuge cells at 4 Â°C for 10 min at 10,000 <em>x g</em>.</li><li>Transfer the supernatant to a new Eppendorf and discard pellet.</li><li>Centrifuge supernatant to pre-clear, at 4 Â°C for 10 min at 12,000-14,000 <em>x g</em>.</li><li>Collect the supernatant (the pellet is not always visible at this stage).<br/><br/></li></ol></li><li>Probe hybridization<ol class="npwtwo"><li>Prepare the hybridization reaction:<br/>200 Î¼l lysate<br/>100 Î¼l Hybridization buffer<br/>0.3 Î¼l 1 M DTT (to a final of 0.1 mM)<br/>1x Protease inhibitors<br/>1.5 Î¼l 20 U/Î¼l Superase (to a final of 0.1 U/Î¼l)<br/>Add biotinylated probes to a final concentration of 100 pmol per reaction.<br/><em>Note: We recommend the use of a set of scrambled biotinylated single molecule probes as a negative control. Alternatively, a set of probes that target the antisense strand can be used. Additionally, we recommend a parallel reaction, were the set of probes are omitted. Data sets of the experiment should be compared with these control datasets to eliminate proteins that tend to aggregate or bind in a non-specific manner.</em> </li><li>Incubate hybridization reaction at 37 Â°C for 4 h, rotating.<br/><br/></li></ol></li><li>Streptavidin beads preparation (Video 1)<br/><em>Note: In the video the tube in which the reaction will occur is labeled âReactionâ. Subsequently, there are three tubes, labeled â1x PBSâ, â1 mg/ml BSAâ and âHybâ (hybridization buffer).</em> <ol class="npwtwo"><li>Wash 40 Î¼l slurry Streptavidin beads (equals to ~10-15 Î¼l solid Streptavidin beads) in ~250 Î¼l 1x PBS x 3.</li><li>Block beads with 250-500 Î¼l 1 mg/ml BSA in 1x PBS for 1 h at RT, shaking.</li><li>Discard the blocking solution and wash beads with Hybridization buffer. Repeat wash three times.<br/><br/><!--video3639-1å¼å§--><div class="videodiv" id="vd3639-1" style="height:480px;"></div><div class="videophoto" id="vp3639-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3639/3639-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Streptavidin beads preparation</strong> </div><br/></li></ol></li><li>Biotin-Streptavidin reaction<ol class="npwtwo"><li>Add the hybridization reaction to beads and incubate at 37 Â°C for 30-60 min, rotating.</li><li>Discard the unbound reaction.</li><li>Wash beads with 40x solid bead volume (~400-600 Î¼l) of washing buffer.</li><li>Repeat wash three times.<br/><br/></li></ol><strong>Elution Option I:</strong> <ol class="npwtwo"><li>Resuspend beads in 3x the original volume of solid beads (~30-45 Î¼l) of DNase buffer.</li><li>Add 100 Î¼l Elution buffer and incubate at 37 Â°C for 30 min, rotating.</li><li>Add Sample Loading Buffer and boil at 95 Â°C for 5 min.<br/><br/><img alt="" class="layerphoto" height="195" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603032215_6375.jpg" width="100"/><br/><strong>Figure 1. Schematic of biochemical pulldown of RNAs and associated proteins and characterization by LS-MSMS</strong><br/></li></ol><br/><strong>Elution Option II:</strong> <ol class="npwtwo"><li>Add ~50 Î¼l 2x Sample Loading Buffer and boil at 95 Â°C for 5 min.</li><li>Run samples on a SDS-PAGE page, stain with Coomassie and cut bands for mass-spectrometry. A representative SDS-PAGE gel is shown in Figure 2.<br/><br/><img alt="" class="layerphoto" height="135" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200603/20200603032236_5932.jpg" width="100"/><br/><strong>Figure 2. A representative image of an SDS-PAGE stained with Coomassie representing a pulldown experiment in 3T3 mouse fibroblasts: a set of probes against an mRNA of interest were used. The negative control consisted of a set of scrambled, non-specific probes</strong><br/><br/></li></ol></li><li>In-gel digestion (based on Reference 19)<ol class="npwtwo"><li>Cut relevant gel lane with surgical blade into multiple slices (about 8 horizontal slices along length of lane in this case, large high intensity protein bands were cut out separately to minimize potential masking of low abundance proteins).</li><li>Dice each gel slice into small pieces (1 x 1 mm) and place into deepwell plate.</li><li>Add ~200 Î¼l (or enough to cover) of Destaining buffer and vortex (800 rpm max.) in a thermomixer for 20 min and discard the supernatant.</li><li>Repeat Step F3.</li><li>Add 100 Î¼l (or enough to cover) of Dehydration buffer 1, vortex in a thermomixer for 20 min (800 rpm max).</li><li>Remove and discard the supernatant.</li><li>Vacuum the gel pieces to complete dryness (10-20 min).</li><li>Add 25 Î¼l (or enough to cover gel pieces) Reduction buffer to dried gels.</li><li>Vortex and spin briefly (800 rpm max).</li><li>Allow reaction to proceed at 60 Â°C for 1 h.</li><li>Chill tubes to room temp (22-23 Â°C).</li><li>Add 500 Î¼l Dehydration buffer 1.</li><li>Incubate for 10 min (22-23 Â°C).</li><li>Remove supernatant and discard.</li><li>Add 25 Î¼l (or enough to cover gel pieces) Alkylation buffer to the gel pieces.</li><li>Vortex and spin briefly (800 rpm max).</li><li>Allow reaction to proceed in the dark for 20 min at room temperature (22-23 Â°C).</li><li>Remove supernatant and discard.</li><li>Wash gels with 100 Î¼l Digestion buffer, vortex 10 min, spin.</li><li>Remove supernatant and discard.</li><li>Dehydrate gels with 100 Î¼l (or enough to cover gel pieces) of Dehydration buffer 2, vortex 5 min, spin, remove supernatant and discard (800 rpm max).</li><li>Repeat Step F21.</li><li>Vacuum the gel pieces to complete dryness (~20 min, keep time to minimum).</li><li>Place the deepwell plate on ice.</li><li>Add freshly prepared trypsin solution to just barely cover the gel pieces (~5-50 Î¼l).</li><li>After 30 min check if all trypsin solution has absorbed and add more if necessary. (Gel pieces should be covered completely with trypsin).</li><li>Rehydrate the gel pieces on ice or at 4 Â°C for further 1 h.</li><li>Add Digestion buffer as needed to cover the gel pieces.</li><li>Incubate at 37 Â°C for 18 h.</li><li>Transfer the digest solution (aqueous extraction) into 2 ml Protein Lo-Bind tubes.</li><li>To the gel pieces, add 30 Î¼l (enough to cover) of Extraction buffer.</li><li>Vortex 20-30 min.</li><li>Sonicate in ultrasonic bath for 5 min.</li><li>Add aqueous extraction to solution obtained at Step F30.</li><li>Repeat Steps F31-F33.</li><li>Speed Vac the digest solution to dryness.<br/><br/></li></ol></li><li>LC-MSMS<ol class="npwtwo"><li>Add 20 Î¼l loading solvent to dried peptide digests and vortex for 1 min.</li><li>Transfer 18 Î¼l to glass HPLC vial or 0.5 ml Eppendorf Protein LoBind tube and place in autosampler.</li><li>Inject 2-5 Î¼l on the trap column (PepMap 75 Î¼m x 2 cm) at 5 Î¼l/min using Loading solvent and after 5 min switch inline with analytical column (PepMap 75 Î¼m x 15 cm).</li><li>Elute peptides using a 15 min linear gradient 6-35% Gradient solvent B at 300 nl/min.</li><li>Apply an electrospray voltage of 2.0-2.5 kV to nanospary emitter.</li><li>Acquire data in Data Dependant Acquisition (DDA) mode with precursor ions acquired using a single 250 ms scan in the range 400-1,500 m/z and 50 product ion scans at 50 ms scan time in the range 100-1,800 m/z that are automatically trigger.</li></ol></li></ol><p class="pptt" id="biaoti34181">### Data analysis</p><p class="ppzz">Data was processed using Protein pilot v5.0.1 using Paragon search engine (Sciex) using a Uniprot Swissprot Mouse database supplemented with sequences of common contaminating proteins. A 1% FDR cut-off was applied on PSM, peptide and proteins levels. Only proteins that were not present in the control sample lists were selected for further analysis.</p><p class="pptt" id="biaoti34182">### Recipes</p><ol class="npwtwok"><li>Lysis Buffer<br/>50 mM Tris-HCl pH 7.5<br/>150 mM KCl<br/>1 mM EDTA<br/>0.5 % Triton X100</li><li>Hybridization Buffer<br/>500 mM NaCl<br/>1% SDS<br/>100 mM Tris-HCl, pH 7.0<br/>10 mM EDTA<br/>15 % Formamide<br/>Add fresh before use: 1 mM DTT, Protease Inhibitors, U/Î¼l Superase</li><li>Washing Buffer<br/>2x SSC <br/>0.5% SDS<br/>Add fresh 1 mM DTT</li><li>DNase Buffer<br/>10 mM NaCl<br/>0.1% NP-40</li><li>Elution Buffer<br/>100 Î¼g/ml RNase A<br/>0.1 U/Î¼l RNase H<br/>100 U/ml DNase 1<br/>All In-Gel Digestion Buffers below are prepared with LC-MS Water</li><li>Destaining Buffer <br/>50% Methanol<br/>50 mM Ammonium Bicarbonate</li><li>Dehydration Buffer 1<br/>100% Acetonitrile</li><li>Reduction Buffer<br/>10 mM Dithiothreitol<br/>25 mM Ammonium Bicarbonate</li><li>Alkylation Buffer<br/>55 mM Iodoacetamide<br/>25 mM Ammonium Bicarbonate</li><li>Dehydration Buffer 2<br/>50% Acetonitrile<br/>25 mM Ammonium Bicarbonate</li><li>Digestion Buffer<br/>25 mM Ammonium Bicarbonate</li><li>Trypsin Solution<br/>10 ng/Î¼l Trypsin<br/>25 mM Ammonium Bicarbonate</li><li>Extraction Buffer<br/>50% Acetonitrile<br/>5% Formic Acid</li><li>Loading Solvent<br/>2% Acetonitrile<br/>0.2% Formic Acid</li><li>Gradient solvent A<br/>0.1% Formic acid</li><li>Gradient solvent B<br/>80% Acetonitrile<br/>0.1% Formic acid<br/></li></ol><p class="pptt" id="biaoti34183">### Acknowledgments</p><p class="ppzz">We thank Dr. Sam Barichievy and Dr. Stephanie Fanucchi for stimulating and fruitful discussions. We also thank Dr. Nicolas Beaume for his assistance with the making of the video. This research has been supported by the following grants: V2YGE81 to A.F.S., PG-V2KYPO7, TA 2011 011 from the Council for Industrial and Scientific Research (CSIR, South Africa) to M.M.M and by a grant from the Emerging Research Area Program of The Department of Science and Technology (DST, South Africa) Department of Science &amp; Technology Centre of Competence Grant, SA Medical Research Council SHIP grant, CSIR Parliamentary Grant.</p><p class="pptt" id="biaoti34184">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti34185">### References</p><ol class="npwtwok"><li>Abdelmohsen, K., Panda, A. C., Kang, M. J., Guo, R., Kim, J., Grammatikakis, I., Yoon, J. H., Dudekula, D. B., Noh, J. H., Yang, X., Martindale, J. L. and Gorospe, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25123665" target="_blank">7SL RNA represses p53 translation by competing with HuR.</a> <em>Nucleic Acids Res</em> 42(15): 10099-10111.</li><li>Abdelmohsen, K., Tominaga, K., Lee, E. K., Srikantan, S., Kang, M. J., Kim, M. M., Selimyan, R., Martindale, J. L., Yang, X., Carrier, F., Zhan, M., Becker, K. G. and Gorospe, M. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21737422" target="_blank">Enhanced translation by Nucleolin via G-rich elements in coding and non-coding regions of target mRNAs.</a> <em>Nucleic Acids Res</em> 39(19): 8513-8530.</li><li>Baltz, A. G., Munschauer, M., SchwÃ¤nhausser, B., Vasile, A., Murakawa, Y., Schueler, M., Youngs, N., Penfold-Brown, D., Drew, K., Milek, M., Wyler, E., Bonneau, R., Selbach, M., Dieterich, C. and Landthaler, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22681889" target="_blank">The mRNA-bound proteome and its global occupancy profile on protein-coding transcripts.</a> <em>Mol Cell</em> 46(5): 674-690.</li><li>Castello, A., Fischer, B., Eichelbaum, K., Horos, R., Beckmann, B. M., Strein, C., Davey, N. E., Humphreys, D. T., Preiss, T., Steinmetz, L. M., Krijgsveld, J. and Hentze, M. W. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22658674" target="_blank">Insights into RNA biology from an atlas of mammalian mRNA-binding proteins.</a> <em>Cell</em> 149(6): 1393-1406.</li><li>Chu, C., Quinn, J. and Chang, H. Y. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22472705" target="_blank">Chromatin isolation by RNA purification (ChIRP).</a> <em>J Vis Exp</em>(61).</li><li>Chu, C., Qu, K., Zhong, F. L., Artandi, S. E. and Chang, H. Y. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21963238" target="_blank">Genomic maps of long noncoding RNA occupancy reveal principles of RNA-chromatin interactions.</a> <em>Mol Cell</em> 44(4): 667-678.</li><li>Di Tomasso, G., Miller Jenkins, L. M. and Legault, P. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27659051" target="_blank">ARiBo pull-down for riboproteomic studies based on label-free quantitative mass spectrometry.</a> <em>RNA</em> 22(11): 1760-1770.</li><li>Glisovic, T., Bachorik, J. L., Yong, J. and Dreyfuss, G. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18342629" target="_blank">RNA-binding proteins and post-transcriptional gene regulation.</a> <em>FEBS Lett</em> 582(14): 1977-1986.</li><li>Grammatikakis, I., Zhang, P., Panda, A. C., Kim, J., Maudsley, S., Abdelmohsen, K., Yang, X., Martindale, J. L., Motino, O., Hutchison, E. R., Mattson, M. P. and Gorospe, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27117401" target="_blank">Alternative splicing of neuronal differentiation factor TRF2 regulated by HNRNPH1/H2.</a> <em>Cell Rep</em> 15(5): 926-934.</li><li>Gumireddy, K., Li, A., Yan, J., Setoyama, T., Johannes, G. J., Orom, U. A., Tchou, J., Liu, Q., Zhang, L., Speicher, D. W., Calin, G. A. and Huang, Q. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23974796" target="_blank">Identification of a long non-coding RNA-associated RNP complex regulating metastasis at the translational step.</a> <em>EMBO J</em> 32(20): 2672-2684.</li><li>Kar, A., Fushimi, K., Zhou, X., Ray, P., Shi, C., Chen, X., Liu, Z., Chen, S. and Wu, J. Y. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21343338" target="_blank">RNA helicase p68 (DDX5) regulates tau exon 10 splicing by modulating a stem-loop structure at the 5' splice site.</a> <em>Mol Cell Biol</em> 31(9): 1812-1821.</li><li>Kar, A. N., Vargas, J. N. S., Chen, C. Y., Kowalak, J. A., Gioio, A. E. and Kaplan, B. B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28161363" target="_blank">Molecular determinants of cytochrome C oxidase IV mRNA axonal trafficking.</a> <em>Mol Cell Neurosci</em> 80: 32-43.</li><li>Kwon, S. C., Yi, H., Eichelbaum, K., FÃ¶hr, S., Fischer, B., You, K. T., Castello, A., Krijgsveld, J., Hentze, M. W. and Kim, V. N. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23912277" target="_blank">The RNA-binding protein repertoire of embryonic stem cells.</a> <em>Nat Struct Mol Biol</em> 20(9): 1122-1130.</li><li>Lunde, B. M., Moore, C. and Varani, G. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17473849" target="_blank">RNA-binding proteins: modular design for efficient function.</a> <em>Nat Rev Mol Cell Biol</em> 8(6): 479-490.</li><li>McHugh, C. A., Chen, C. K., Chow, A., Surka, C. F., Tran, C., McDonel, P., Pandya-Jones, A., Blanco, M., Burghard, C., Moradian, A., Sweredoski, M. J., Shishkin, A. A., Su, J., Lander, E. S., Hess, S., Plath, K. and Guttman, M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25915022" target="_blank">The Xist lncRNA interacts directly with SHARP to silence transcription through HDAC3.</a> <em>Nature</em> 521(7551): 232-236.</li><li>McHugh, C. A. and Guttman, M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29130217" target="_blank">RAP-MS: a method to identify proteins that interact directly with a specific RNA molecule in cells.</a> <em>Methods Mol Biol</em> 1649: 473-488.</li><li>Ng, S. Y., Bogu, G. K., Soh, B. S. and Stanton, L. W. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23932716" target="_blank">The long noncoding RNA RMST interacts with SOX2 to regulate neurogenesis.</a> <em>Mol Cell</em> 51(3): 349-359.</li><li>Ramanathan, M., Porter, D. F. and Khavari, P. A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30804549" target="_blank">Methods to study RNA-protein interactions.</a> <em>Nat Methods</em> 16(3): 225-234.</li><li>Shevchenko, A., Tomas, H., HavliÅ¡, J., Olsen, J. V. and Mann, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406544" target="_blank">In-gel digestion for mass spectrometric characterization of proteins and proteomes.</a> <em>Nat Protoc</em> 1(6): 2856-2860.</li><li>Simon, M. D., Wang, C. I., Kharchenko, P. V., West, J. A., Chapman, B. A., Alekseyenko, A. A., Borowsky, M. L., Kuroda, M. I. and Kingston, R. E. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22143764" target="_blank">The genomic binding sites of a noncoding RNA.</a> <em>Proc Natl Acad Sci U S A</em> 108(51): 20497-20502.</li><li>Slobodin, B. and Gerst, J. E. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20876833" target="_blank">A novel mRNA affinity purification technique for the identification of interacting proteins and transcripts in ribonucleoprotein complexes.</a> <em>RNA</em> 16(11): 2277-2290.</li><li>Tominaga, K., Srikantan, S., Lee, E. K., Subaran, S. S., Martindale, J. L., Abdelmohsen, K. and Gorospe, M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21859890" target="_blank">Competitive regulation of nucleolin expression by HuR and miR-494.</a> <em>Mol Cell Biol</em> 31(20): 4219-4231.</li><li>Yoon, J. H., Abdelmohsen, K., Srikantan, S., Yang, X., Martindale, J. L., De, S., Huarte, M., Zhan, M., Becker, K. G. and Gorospe, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22841487" target="_blank">LincRNA-p21 suppresses target mRNA translation.</a><em> Mol Cell</em> 47(4): 648-655.</li><li>Yoon, J. H. and Gorospe, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26721479" target="_blank">Cross-linking immunoprecipitation and qPCR (CLIP-qPCR) analysis to map interactions between long noncoding RNAs and RNA-binding proteins.</a> <em>Methods Mol Biol </em>1402: 11-17.</li><li>Zeng, F., Peritz, T., Kannanayakal, T. J., Kilk, K., Eiriksdottir, E., Langel, U. and Eberwine, J. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406325" target="_blank">A protocol for PAIR: PNA-assisted identification of RNA binding proteins in living cells.</a> <em>Nat Protoc</em> 1(2): 920-927.</li><li>Zhang, W., Xie, M., Shu, M. D., Steitz, J. A. and DiMaio, D. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27659050" target="_blank">A proximity-dependent assay for specific RNA-protein interactions in intact cells.</a> <em>RNA</em> 22(11): 1785-1792.<br/></li></ol>