<body>
            <h1>Preparation of Yeast tRNA Sample for NMR Spectroscopy</h1>
            <p><strong>Authors:</strong> Marjorie  Catala, Alexandre  Gato, Carine  TisnÃ©, Pierre Barraud</p>
            <h2>Abstract</h2>
            <p>Transfer RNAs (tRNAs) are heavily decorated with post-transcriptional modifications during their biosynthesis. To fulfil their functions within cells, tRNAs undergo a tightly controlled biogenesis process leading to the formation of mature tRNAs. In addition, functions of tRNAs are often modulated by their modifications. Although the biological importance of post-transcriptional RNA modifications is widely appreciated, methods to directly detect their introduction during RNA biosynthesis are rare and do not easily provide information on the temporal nature of events. To obtain information on the tRNA maturation process, we have developed a methodology, using NMR as a tool to monitor tRNA maturation in a non-disruptive and continuous fashion in cellular extracts. By following the maturation of a model yeast tRNA with time-resolved NMR, we showed that modifications are introduced in a defined sequential order, and that the chronology is controlled by cross-talk between modification events. The implementation of this method requires the production for NMR spectroscopy of tRNA samples with different modification status, in order to identify the NMR signature of individual modifications. The production of tRNA samples for the analysis of modification pathways with NMR spectroscopy will be presented here and examplified on the yeast tRNA<sup>Phe</sup>, but can be extended to any other tRNA by changing the sequence of the construct. The protocol describes the production of unmodified tRNA samples by <em>in vitro</em> transcription, and the production of modified tRNA samples by recombinant expression of tRNAs in <em>E. coli.</em></p>
            <p><strong>Keywords:</strong> Transfer RNA, tRNA, Post-transcriptional modifications, <em>In vitro</em> transcription, <em>In vivo</em> RNA production, RNA purification, NMR, NMR spectroscopy</p>
            <h2>Background</h2><p>In all domains of life, the synthesis and maturation of RNAs involve post-transcriptional chemical modifications of their nucleotides at specific sites. Among the different RNA families, tRNAs display not only the highest variety of chemical modifications, but also the highest density of modification per transcript (~8-25% of modified nucleotides in tRNAs of various organisms) (Boccaletto <em>et al.</em>, 2018). The biogenesis of tRNAs is tightly regulated, and in particular, the introduction of post-transcriptional modifications in tRNAs is controlled and influenced by multiple factors (El Yacoubi <em>et al.</em>, 2012; Jackman and Alfonzo, 2013; Barraud and TisnÃ©, 2019). Although all aspects of tRNA biology are controlled and modulated by modifications, methods to directly detect their introduction during tRNA biosynthesis are rare and do not easily provide information on the temporality of modification events. We have developed a methodology, using NMR as a tool to monitor tRNA maturation in a non-disruptive and continuous fashion (Barraud <em>et al.</em>, 2019). Briefly, introducing isotope-labeled tRNAs into unlabeled cell extracts containing the cellular enzymatic activities combined with the use of isotope-filters in NMR experiments enables the detection of the tRNA of interest within the complex cell extract environment. This method requires the production of tRNA samples with different modification status, in order to identify the NMR signature of individual modifications (Barraud <em>et al.</em>, 2019).<br/><br/>Here we describe the production and purification of two forms of the yeast tRNA<sup>Phe</sup> differing in their modification content. The first sample corresponds to a yeast tRNA<sup>Phe</sup> produced by <em>in vitro</em> transcription, which presents no post-transcriptional modifications. The second sample corresponds to a recombinant yeast tRNA<sup>Phe</sup> produced in an <em>E. coli</em> strain overproducing this tRNA. Since <em>E. coli</em> has a less prolific but related tRNA modification machinery than <em>S. cerevisiae</em>, the modification pattern of this sample resembles that of the fully modified yeast tRNA<sup>Phe</sup>, but overall exhibits fewer modifications (Barraud <em>et al.</em>, 2019). The production and purification of tRNA samples with different modification content is essential to reveal the NMR signature of individual post-transcriptional modifications and correspond to the first steps towards the time-resolved NMR monitoring of tRNA maturation (Barraud <em>et al.</em>, 2019). The reported protocol describes the production of unmodified tRNA samples by <em>in vitro</em> transcription, and the production of modified tRNA samples by recombinant expression of tRNAs in <em>E. coli</em>. These productions and purifications will be examplified on the yeast tRNA<sup>Phe</sup>. Part of the protocol has been published elsewhere (Barraud <em>et al.</em>, 2019), but some parts are described here in more detail. Protocol for the expression and purification of active T7 RNA polymerase in <em>E. coli</em> are not detailed here, but have been described comprehensively elsewhere (Rio, 2013; DÃ©gut <em>et al.</em>, 2016).</p>
        </body>

<p class="pptt" id="biaoti34451">## Materials and Reagents</p><ol class="npwone"><li>Production of unmodified yeast tRNA<sup>Phe</sup> by <em>in vitro</em> transcription<br/><strong><em>For tRNA </em></strong><em><strong>in vitro</strong></em><strong><em> transcription</em></strong><br/><ol class="npwtwo"><li>Commercial or <em>in-house</em> produced T7 RNA polymerase (stock stored at -20 Â°C at 1 mg/ml in 20 mM Na-phosphate pH 7.7, 100 mM NaCl, 10 mM DTT, 1 mM EDTA, 50% (v/v) glycerol).<br/></li><li>RNase-free water (double-distilled or MilliQ water 18.2 MÎ©)<br/></li><li>Ready to use ATP, UTP, GTP, CTP at 100 mM in water at pH 8.0 (Jena Bioscience, catalog number: <a class="blackNoUnder" target="_blank"> NU-1014L </a> )<br/></li><li>Ready to use <sup>15</sup>N-labeled UTP and GTP at 100 mM in water at pH 7.8 (Eurisotop, catalog numbers: <a class="blackNoUnder" target="_blank"> NLM-4270-CA </a> , NLM-4268-CA)<br/></li><li>DNA template<br/>5â² TGGTGCGAATTCTGTGGATCGAACACAGGACCTCCAGATCTTCAGTCTGGCGCTCTCCCAACTGAGCTAAATCCGCTATAGTGAGTCGTATTA 3â² (Eurogentec, 1 Î¼mol of synthesis scale, PAGE purification)<br/></li><li>T7 promotor primer: 5â² TAATACGACTCACTATAG 3â² (Eurogentec, 0.2 Î¼mol of synthesis scale, desalted)<br/></li><li>280 mM MgCl<sub>2</sub> (prepared in water) (CAS Number 7791-18-6)<br/></li><li>80 mM GMP (prepared in water at pH 7.5, pH adjusted with concentrated HCl) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> G8377-56 </a> )<br/></li><li>0.5 M Ethylenediaminetetraacetic acid (EDTA) (prepared in water at pH 8.0, pH adjusted with concentrated NaOH) (CAS Number 60-00-4)<br/></li><li>0.1 M Spermidine (Sigma-Aldrich, catalog number: 0<a class="blackNoUnder" target="_blank"> 5292 </a> )<br/></li><li>1 M 1,4-Dithiothreitol (DTT) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 43816 </a> )<br/></li><li>Triton<sup>TM</sup> X-100 solution (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T9284 </a> )<br/></li><li>20x T7 transcription buffer (TB) (see Recipes)<br/><br/></li></ol><strong><em>For denaturing Urea-PAGE</em></strong><br/><ol class="npwtwo"><li>Urea (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> EU0014-B </a> )<br/></li><li>40% acrylamide/bis-acrylamide (19:1) solution (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> EU0076-B </a> )<br/></li><li>Ammonium persulfate (APS) (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> EU0009-B </a> ) prepared as 10% (w/v) solution in water<br/></li><li>N,N,Nâ,Nâ-Tetramethylethylenediamine (TEMED) (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> 50406 </a> )<br/></li><li>Bromophenol blue (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> B0126 </a> )<br/></li><li>Xylene cyanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> X4126 </a> )<br/></li><li>Toluidine blue (CAS Number 92-31-9, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 198161 </a> )<br/></li><li>Ethanol absolute (CAS Number 64-17-5, Millipore/Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1070172511 </a> )<br/></li><li>10x TBE buffer (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T4415-1L </a> )<br/></li><li>2x RNA loading dye (see Recipes)<br/></li><li>Toluidine-blue staining solution (see Recipes)<br/></li></ol><br/><strong><em>For tRNA transcript purification and NMR sample preparation</em></strong><br/><ol class="npwtwo"><li>Sterile syringe filters 0.22 Î¼m (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 514-0073 </a> )<br/></li><li>Dialysis sacks Spectra/Por 3 Dialysis Tubing MWCO &lt; 3.5 kDa (Spectra/Por, catalog number: 132724)<br/></li><li>Standard Closures for Dialysis Tubing (Spectrum<sup>Â®</sup>, catalog number: 132735)<br/></li><li>Concentrators Amicon<sup>Â®</sup> Ultra-15 3,000 MWCO (Millipore/Merck, catalog number: <a class="blackNoUnder" target="_blank"> UFC900308 </a> )<br/></li><li>NaH<sub>2</sub>PO<sub>4</sub> and Na<sub>2</sub>HPO<sub>4</sub>Â·12H<sub>2</sub>O (CAS Numbers 7558-80-7 and 10039-32-4, Sigma-Aldrich, catalog numbers: <a class="blackNoUnder" target="_blank"> S3139 </a> and <a class="blackNoUnder" target="_blank"> 71649 </a> )<br/></li><li>UltraPure<sup>TM</sup> 5 M NaCl (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 24740011 </a> )<br/></li><li>1 M MgSO<sub>4</sub> (prepared in water) (CAS Number 7487-88-9)<br/></li><li>1 M MgCl<sub>2</sub> (prepared in water) (CAS Number 7791-18-6)<br/></li><li>Purification buffer A (see Recipes)<br/></li><li>Purification buffer B (see Recipes)<br/></li><li>NMR buffer (see Recipes)<br/><div><span style="font-size:10pt;"></span> </div></li></ol><br/></li><li>Production of modified yeast tRNA<sup>Phe</sup> by <em>in vivo</em> recombinant expression in <em>E. coli</em><br/><strong><em>For modified yeast tRNA</em></strong><sup><strong><em>Phe</em></strong></sup> <strong><em>in vivo</em></strong><strong><em> expression and extraction</em></strong><br/><ol class="npwtwo"><li>Gene Pulser<sup>Â®</sup>/MicroPulser<sup>TM</sup> Electroporation Cuvettes 0.2 cm gap (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1652082 </a> )<br/></li><li>50 ml conical tube (<em>e.g.</em>, Falcon<sup>TM</sup>, product number: 352070)<br/></li><li>Electrocompetent bacteria Escherichia coli JM101Tr strain (Mechulam <em>et al.</em>, 1987)<br/><em>Note: Alternatively, Escherichia coli JM101 or JM109 strains may also be used.</em><br/></li><li>pBSTNAV plasmid (Addgene, catalog number: <a class="blackNoUnder" target="_blank"> 45801 </a> ) previously cloned with the yeast tRNA<sup>Phe</sup> DNA gene sequence (see Note 1 and Figure 3)<br/></li><li>Lysogeny Broth (LB) medium Gibco<sup>Â®</sup> (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10855021 </a> )<br/></li><li>SPECTRA 9 medium (<sup>15</sup>N, 98%) (Eurisotop, catalog number: <a class="blackNoUnder" target="_blank"> CGM-3030-N-1 </a> )<br/></li><li>Celtone<sup>Â®</sup> Base Powder (<sup>15</sup>N, 98%+) (Eurisotop, catalog number: <a class="blackNoUnder" target="_blank"> CGM-1030P-N-0.5 </a> )<br/></li><li>Ampicillin, Ready Made Solution, 100 mg/ml, 0.2 Î¼m filtered (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A5354-10ML </a> )<br/></li><li>Water-Saturated Phenol, pH 6.6 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9710 </a> )<br/></li><li>Ethanol absolute (<em>e.g.</em>, Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1070172511 </a> )<br/></li><li>UltraPure<sup>TM</sup> 5 M NaCl (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 24740011 </a> )<br/></li><li>1 M NaCl solution (<em>e.g.</em>, FisherScientific, catalog number: <a class="blackNoUnder" target="_blank"> S25877 </a> )<br/></li><li>2 M Tris-HCl pH 8.0<br/></li><li>1 M Magnesium acetate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 63052 </a> )<br/></li><li>RNase-free water (double-distilled or MilliQ water 18.2 MÎ©)<br/></li><li>Extraction buffer (see Recipes)<br/></li></ol><br/><strong><em>For modified yeast tRNA</em></strong><sup><strong><em>Phe</em></strong></sup><strong><em> purification and NMR sample preparation</em></strong><br/><ol class="npwtwo"><li>Microtubes 0.2 ml</li><li>Dialysis sacks Spectra/Por 3 Dialysis Tubing MWCO &lt; 3.5 kDa (Spectra/Por, product number: 132724)</li><li>Standard Closures for Dialysis Tubing (Spectrum<sup>Â®</sup>, product number: 132735)</li><li>Concentrators Amicon<sup>Â®</sup> Ultra-15 3,000 MWCO (Millipore/Merck, catalog number: <a class="blackNoUnder" target="_blank"> UFC900308 </a> )</li><li>HiLoad 26/600 Superdex 75 pg column (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 28989334 </a> )</li><li>Mono Q 10/100 GL column (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 17516701 </a> )</li><li>Phenyl superose HR 10/10 column (GE Healthcare)<br/><em>Note: Alternatively, a HiPrep 16/10 Phenyl HP column (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 29018184 </a> ).</em> </li><li>HiPrep 26/10 Desalting column (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 17508701 </a> )</li><li>0.5 M NaOH solution (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15614900 </a> )</li><li>RNase-free water (double-distilled or MilliQ water 18.2 MÎ©)</li><li>Ammonium sulfate powder (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10325410 </a> )</li><li>40% acrylamide/bis-acrylamide (19:1) solution (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> EU0076-B </a> )</li><li>1 M Tris-HCl pH 6.8</li><li>1 M Tris-HCl pH 8.8</li><li>20% (w/v) Sodium dodecyl sulfate solution (SDS) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 05030-1L-F </a> )</li><li>Ammonium persulfate (APS) (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> EU0009-B </a> ) prepared at 10% (w/v) solution in water</li><li>N,N,Nâ,Nâ-Tetramethylethylenediamine (TEMED) (Euromedex, catalog number: <a class="blackNoUnder" target="_blank"> 50406 </a> )</li><li>Tris-Glycine-SDS Buffer 10x Concentrate (TGS) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T7777 </a> )</li><li>2-mercaptoethanol (CAS Number 60-24-2, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M3148 </a> )</li><li>Glycerol (CAS Number 56-81-5, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> G5516 </a> )</li><li>Complementary DNA oligonucleotide at 100 ÂµM in water 5â² TCGAACACAGGACCTCCAGATCTTCAGTCTGGCGCTCT 3â² (Eurogentec, 0.2 Î¼mol of synthesis scale, desalted)</li><li>2x Laemmli loading buffer (see Recipes)</li><li>Toluidine-blue staining solution (see Recipes)</li><li>Purification buffer C (see Recipes)</li><li>Purification buffer D (see Recipes)</li><li>Purification buffer E (see Recipes)</li><li>NMR buffer (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti34452">### Equipment</p><ol class="npwone"><li>Production of unmodified yeast tRNA<sup>Phe</sup> by <em>in vitro</em> transcription<br/><strong><em>For tRNA </em></strong><strong><em>in vitro</em></strong><strong><em> transcription</em></strong><br/><ol class="npwtwo"><li>Water Bath</li></ol><br/><strong><em>For transcription analysis by denaturing Urea-PAGE</em></strong><br/><ol class="npwtwo"><li>Electrophoresis chamber, glass slides, spacers, clamps and combs for vertical PAGE (<em>e.g.</em>, Bio-Rad Mini-PROTEAN), power supply (<em>e.g.</em>, Bio-Rad PowerPac)</li></ol><br/><strong><em>For tRNA transcript purification and NMR sample preparation</em></strong><br/><ol class="npwtwo"><li><a class="blackNoUnder" target="_blank"> ÃKTA Purifier </a> (GE Healthcare) or similar chromatographic system</li><li>MonoQ 10/100 GL column (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 17516701 </a> )</li><li>Centrifuge (<em>e.g.</em>, Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> Allegra X-30R </a> )</li><li>NanoDrop<sup>TM</sup> spectrophotometer (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ND-2000 </a> )<br/><br/></li></ol></li><li>Production of modified yeast tRNA<sup>Phe</sup> by <em>in vivo</em> recombinant expression in <em>E. coli</em><br/><strong><em>For modified yeast tRNA</em></strong><sup><strong><em>Phe</em></strong></sup> <strong><em>in vivo</em></strong><strong><em> expression and extraction</em></strong><br/><ol class="npwtwo"><li>2 L Erlenmeyer flasks (<em>e.g.</em>, Fisher Brand<sup>TM</sup>, Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15459103 </a> )<br/></li><li>Incubation shaker (<em>e.g.</em>, INFORS-HT Minitron)<br/></li><li>MicroPulser<sup>TM</sup> Electroporator (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1652100 </a> )<br/></li><li>Centrifuge for culture liters (<em>e.g.</em>, Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> AvantiÂ® J-E </a> )<br/></li><li>Tube rotator (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 88881002 </a> )<br/></li><li>Vortex mixer (<em>e.g.</em>, Scientific Industries SI<sup>TM</sup>, model: Vortex-Genie<sup>TM</sup> 2, catalog number: <a class="blackNoUnder" target="_blank"> SI-0236 </a> )<br/></li></ol><br/><strong><em>For modified yeast tRNA</em></strong><sup><strong><em>Phe</em></strong></sup><strong><em> purification and NMR sample preparation</em></strong><br/><ol class="npwtwo"><li>ÃKTA Purifier (GE Healthcare) or similar chromatographic system</li><li>Sample loop 5 ml (GE Healthcare, catalog number: <a class="blackNoUnder" target="_blank"> 18114053 </a> )</li><li>Superloop 10 ml and 50 ml (ÃKTA design) (GE Healthcare, catalog numbers: <a class="blackNoUnder" target="_blank"> 18111381 </a> and <a class="blackNoUnder" target="_blank"> 18111382 </a> )</li><li>Electrophoresis chamber, glass slides, spacers, clamps and combs for vertical PAGE (<em>e.g.</em>, Bio-Rad Mini-PROTEAN), power supply (<em>e.g.</em>, Bio-Rad PowerPac)</li><li>Dry bath incubator (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 88870004 </a> )</li><li>Centrifuge (<em>e.g.</em>, Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> Allegra X-30R </a> )</li><li>NanoDrop<sup>TM</sup> spectrophotometer (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ND-2000 </a> )</li></ol></li></ol><p class="pptt" id="biaoti34453">### Procedure</p><ol class="npwone"><li>Production of unmodified yeast tRNA<sup>Phe</sup> by <em>in vitro</em> transcription<ol class="npwtwo"><li>Small scale tRNA <em>in vitro</em> transcription<br/>The optimal <em>in vitro</em> transcription conditions depend greatly on the MgCl<sub>2</sub> and NTPs concentrations. The [MgCl<sub>2</sub>]/[NTP] ratio should be adjusted to optimize the yield of the RNA of interest (Figure 1). It is recommended to test small-scale reactions (<em>e.g.</em>, 40 Î¼l) before large-scale production of RNA (5-20 ml). In practice, the concentration of MgCl<sub>2</sub> is varied in small-scale reactions as described in Table 1. The yield of tRNA transcript is then analyzed by Urea-PAGE (Step A2). The conditions of Table 1 are first starting points to determine best MgCl<sub>2</sub> concentration. Concentrations of T7 RNA polymerase, DNA template and the addition of GMP at 2-6 mM can also be further optimized in a second step.<br/><ol class="npwthree"><li>Prepare small scale reaction tests of 40 Î¼l as described in Table 1 in PCR microtubes.<br/></li><li>Incubate the reactions in a water bath for 3 h at 37 Â°C. After ~1 h, the appearance of a white precipitate of magnesium pyrophosphate is the sign of a working transcription reaction.<br/></li><li>Stop the reaction by adding 5 Î¼l of 0.5 M EDTA pH 8.0 to each small scale reaction.<br/></li><li>Store at -20 Â°C or analyze the reactions by Urea-PAGE (Step A2).<br/><br/><strong>Table 1. Small scale reaction tests of 40 Î¼l to determine best MgCl</strong><sub><strong>2</strong></sub><strong> concentration</strong><br/><img alt="" class="layerphoto" height="390" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200614/20200614074334_6131.png" width="900"/><br/><br/></li></ol></li><li>Transcription analysis by denaturing Urea-PAGE<br/><ol class="npwthree"><li>To analyze the products of transcription, prepare a Urea-PAGE gel using the same materials as those used for protein SDS-PAGE analysis. For tRNA of ~75 nts, prepare a gel mix with 7M Urea and 14% (w/v) acrylamide:bisacrylamide 19:1 by mixing 16.8 g of urea, 14 ml of 40% (w/v) acrylamide:bisacrylamide 19:1, 4 ml of 10x TBE and adjusting the volume to 40 ml final with H<sub>2</sub>O. (Note that this mix can be stored at room temperature for several months). Take 6 ml of this mix and add 60 Î¼l of 10% APS and 6 Î¼l of TEMED just before placing the gel between the glass plates and put the comb immediately.<br/></li><li>Take an aliquot of each small-scale reaction (typically 2.5 Î¼l) and add 2.5 Î¼l of 2x RNA loading dye (Recipe 2). Load your samples in the wells and run the gel for 1 h at 200 V in 1x TBE buffer.<br/></li><li>The visualization is then done by toluidine blue staining. Carefully remove the gel from the glass plates and let it stain in the toluidine-blue staining solution (Recipe 3) for 5-10 min. Pour the staining solution back into the stock bottle, and destain with water. Destain several times with fresh water until the background color of the gel is clear.<br/></li><li>Take a picture with a regular camera. The gel corresponding to the assays described in Table 1 and revealed by toluidine-blue staining is shown in Figure 1. The best yield for the yeast tRNA<sup>Phe</sup> is obtained for conditions #5 and #6 with [MgCl<sub>2</sub>]/[NTPs] = 1.7-2.0, <em>i.e.</em>, [MgCl<sub>2</sub>] = 34-40 mM and [NTPs] = 20 mM. These concentrations are thus used for large-scale preparation of this tRNA.<br/>The yield of RNA produced by <em>in vitro</em> transcription is sometimes very sensitive to the concentration of MgCl<sub>2</sub>. We recommend to perform these small-scale assays each time that a new DNA template is used and each time new NTPs are used, indeed, best MgCl<sub>2</sub> concentration might be slightly different with different sources of NTP. Especially, best MgCl<sub>2</sub> concentration might differ slightly for <sup>15</sup>N-labeled nucleotides and must therefore be determined with the actual NTPs that will be used in large scale transcription reaction.<br/><br/><img alt="" class="layerphoto" height="336" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200614/20200614074429_1455.jpg" width="300"/><br/><strong>Figure 1. Sensitivity of T7 polymerase <em>in vitro</em> RNA production to the magnesium concentration.</strong> Urea-PAGE revealed by toluidine-blue staining showing the analysis of small-scale transcription assays of Table 1 corresponding to the production of unmodified yeast tRNA<sup>Phe</sup>. The MgCl<sub>2</sub> concentration is varied from 6 mM to 60 mM. Numbers above the gel correspond to the conditions #1 to #8 in Table <br/></li></ol><br/></li><li>Large scale <em>in vitro</em> transcription of unmodified yeast tRNA<sup>Phe</sup><br/>For large scale <em>in vitro</em> transcription, up-scale the best conditions as determined in Step A1 with small scale reactions. In our hands, a concentrated NMR sample of unmodified yeast tRNA<sup>Phe</sup> <sup>15</sup>N-labeled on Us and Gs can be obtained from a 10 ml transcription reaction prepared as described below.<br/><ol class="npwthree"><li>Prepare the large scale transcription reaction of 10 ml in a 50 ml tube as follow:<br/>40 Î¼l of DNA template pre-mixed with the T7 promotor primer at a 1:1 ratio (stock at 50 Î¼M)<br/>500 Î¼l of 20x T7 transcription buffer (TB).<br/>500 Î¼l of non-labeled ATP (stock at 100 mM)<br/>500 Î¼l of non-labeled CTP (stock at 100 mM)<br/>500 Î¼l of <sup>15</sup>N-labeled UTP (stock at 100 mM)<br/>500 Î¼l of <sup>15</sup>N-labeled GTP (stock at 100 mM)<br/>1,300 Î¼l of MgCl<sub>2</sub> (stock at 280 mM)<br/>500 Î¼l of GMP (stock at 80 mM)<br/>250 Î¼l of T7 RNA polymerase (stock at 1 mg/ml)<br/>RNase-free water to 10 ml<br/></li><li>Incubate the reactions in a water bath for 3-4 h at 37 Â°C.<br/></li><li>Stop the reaction by adding 1-1.5 ml of 0.5 M EDTA pH 8.0.<br/></li><li>Store at -20Â°C until purification or purify readily as described in Step A4.<br/></li></ol><br/></li><li>Purification and NMR sample preparation of unmodified yeast tRNA<sup>Phe</sup><br/>The purification is performed on a ÃKTA Purifier (or other FPLC purification system) at 4 Â°C. Transcribed tRNAs are sensitive to RNases, one should therefore be careful to operate the purification in RNase free conditions. For that, wash the purification system and the columns with 0.5 M NaOH and rinse them extensively with water before equilibrating the MonoQ HR 10/100 column in buffer A.<br/><ol class="npwthree"><li>Filter the large scale transcription sample of Step A3 with 0.22 Î¼m filters.<br/></li><li>Load 5-6 ml of the transcription sample on the MonoQ column previously equilibrated with buffer A.<br/><em>Remark: It is not recommended to load larger quantities of tRNAs onto the column. The purification indeed becomes difficult due to a broadening of the elution peaks. Perform Steps A4b to A4d as many times as needed for large scale transcriptions of more than 5 ml.</em><br/></li><li>The purification of the tRNA transcript is achieved using a gradient from 20% to 80% of buffer B with a length of 10 column volumes (CV) and a flow rate of 3 ml/min. Collect 1 ml fractions. Transcribed yeast tRNA<sup>Phe</sup> is usually eluted between 50% and 55% of buffer B.<br/></li><li>Analyse the content of the fractions with denaturing Urea-PAGE as described in Step A2, and pool the fractions containing the tRNA of interest (usually 5 fractions of 1 ml). Do not add fractions of tRNA<sup>Phe</sup> eluted after 55% of buffer B, since they usually contain longer RNA transcripts originating from non-templated nucleotide addition at the 3â²-end of the transcript.</li><li>Dialyze extensively the pooled fractions of yeast tRNA<sup>Phe</sup> in a dialysis sack against 1 mM Na-phosphate pH 6.5. For that one usually dialyzes the sample against a 2 L dialysis solution for at least 2 h and changes it with a fresh solution 3 times.<br/></li><li>Recover the dialyzed tRNA<sup>Phe</sup> in a 50 ml plastic tube and refold it by heating the sample at 95 Â°C for 5 min and letting it cool down slowly at room temperature.<br/></li><li>Add concentrated buffer stock solutions (<em>e.g.</em>, 0.5 M Na-phosphate pH 6.5 and 1 M MgCl<sub>2</sub>) to place the refolded sample in the final NMR buffer (<em>i.e.</em>, 10 mM Na-phosphate pH 6.5, 10 mM MgCl<sub>2</sub>).<br/></li><li>Concentrate the unmodified yeast tRNA<sup>Phe</sup> sample in the NMR buffer to ~1.5-2.0 mM using Amicon 3,000 MWCO. In our hands, with a 10 ml transcription with <sup>15</sup>N-labeled on Us and Gs, one usually obtains a final NMR sample of 300 Î¼l at ~1.8 mM (<em>i.e.</em>, 0.5-0.6 Âµmol).<br/></li><li>Store the sample at -20 Â°C or use it directly for NMR spectra measurements (see Note 2).<br/><br/></li></ol></li></ol></li><li>Production of modified yeast tRNA<sup>Phe</sup> by <em>in vivo</em> recombinant expression in <em>E. coli</em> <ol class="npwtwo"><li>Modified yeast tRNA<sup>Phe</sup> recombinant expression and extraction<br/><ol class="npwthree"><li>Transformation of <em>E. coli</em> with the tRNA expression plasmid and <sup>15</sup>N-labeled expression<br/>The volumes and quantities mentioned below refer to a 1 L culture. We recommend to grow several liters of culture (<em>i.e.</em>, 2-4 L) to increase the quantity of modified yeast tRNA<sup>Phe</sup> purified following this protocol. We also recommend to grow 500 ml of culture in 2 L flasks to get enough oxygenation or ideally to use a fermenter to optimize certain parameters (<em>e.g.</em>, pH of the medium, oxygenationâ¦).<br/><ol class="npwfour"><li>Use a 100 Î¼l aliquot of electrocompetent <em>E. coli</em> JM101Tr strain. Add 2 Î¼l of the plasmid (at 50-200 ng/Î¼l).<br/></li><li>Put the mix in an electroporation cuvette. Electroporate the cells, with the machine set according to the electroporation cuvette and organism that is used (namely set the electroporator to âEc2â meaning â<em>E. coli</em> 0.2 cm cuvetteâ). Then add 1 ml of LB medium and incubate for 30 min at 37 Â°C.<br/></li><li>Divide the 1.1 ml of transformation product in two and pour each half directly into a 2 L Erlenmeyer flasks containing 0.5 L of <sup>15</sup>N-labeled rich growth media (<em>i.e.</em> Spectra 9) supplemented with 0.5 g of <sup>15</sup>N-labeled celtone base powder and 50 Âµg/ml of ampicillin (total culture volume of 1 L).<br/></li><li>Grow the bacteria for ~20 h at 37 Â°C.<br/></li><li>Recover the bacteria by centrifugation at 5,000 <em>x g</em> for 20 min.</li></ol></li><li>Total small RNAs extraction<br/><ol class="npwfour"><li>Resuspend the pellets with 10 ml of extraction buffer in a 50 ml conical tube.<br/></li><li>Add 10 ml of water-saturated phenol. Mix gently for 1 h at room temperature on a tube rotator.<br/></li><li>Centrifuge at 8,000 <em>x g</em> for 30 min at 20 Â°C. Carefully recover the aqueous phase, which corresponds to the upper phase lying above the phenol phase.<br/></li><li>Do a counter-extraction with the addition of 1 ml of extraction buffer to the phenol phase. Mix gently for 15 min, centrifuge at 8,000 <em>x g</em> for 30 min at 20 Â°C, and carefully recover the aqueous phase.<br/></li><li>Add 0.1 volume of 5 M NaCl and 2 volumes of ethanol absolute to the pooled aqueous phases of Steps B1biii and B1biv.<br/></li><li>Vortex vigourously and then centrifuge at 8,000 <em>x g</em> for 20 min at 4 Â°C. Discard the supernatant.<br/></li><li>Resuspend the pellet in 5 ml of 1 M NaCl. Centrifuge at 8,000 <em>x g</em> for 30 min at 4 Â°C.<br/></li><li>Recover the supernatant, and add 2.5 volume of ethanol absolute.<br/></li><li>Centrifuge again 30 min at 8,000 <em>x g</em> at 4 Â°C. All the small RNAs are now precipitated. Discard the supernatant.<br/></li><li>Resuspend the pellet in 2 ml of 2 M Tris-HCl pH 8.0 and incubate 3 h at 37 Â°C to completely deacylate the 3â² extremity of the tRNAs.<br/></li><li>Precipitate the deacylated RNAs by adding 2 volumes of ethanol absolute, vortex vigourously and centrifuge at 8,000 <em>x g</em> for 30 min at 4 Â°C.<br/></li><li>Dissolve the pellet in 5 ml of purification buffer C. The sample is ready for the purification step (Step B2).</li></ol></li></ol></li><li>Modified yeast tRNA<sup>Phe</sup> purification and NMR sample preparation<br/><ol class="npwthree"><li>Before starting the purification, wash the purification system, the loops and the columns with 0.5 M NaOH and rinse them extensively with water in order to work in RNase free conditions.<br/></li><li>Gel-filtration chromatography<br/><ol class="npwfour"><li>Load the 5 ml sample (obtained at the end of Step B1) at 0.5 ml/min on a HiLoad 26/600 Superdex 75 pg column previously equilibrated with purification buffer C.<br/></li><li>Elute at 0.5 ml/min, collect 3.5 ml per fraction.</li></ol></li><li>Gel-shift assay to identify the fractions containing the yeast tRNA<sup>Phe</sup><br/><ol class="npwfour"><li>Design a DNA oligonucleotide sequence that is specifically complementary to the overexpressed yeast tRNA<sup>Phe</sup> and not conserved in the other <em>E. coli</em> tRNAs. In our hands, it works well when the complementary DNA oligonucleotide includes the anticodon arm and parts of the D- and TÎ¨C-arms (see Figure 2A).<br/></li><li>In 0.2 ml-microtubes, mix 5 Î¼l of each fraction with 1 Î¼l of the complementary DNA oligonucleotide (stock at 100 ÂµM), or with 1 Î¼l of water (negative control).<br/></li><li>Heat the samples at 95 Â°C for 2 min in a dry bath incubator, and cool them down on ice.<br/></li><li>Add 6 Î¼l of the 2x Laemmli loading buffer to the samples.<br/></li><li>Prepare a 14% SDS polyacrylamide gel: For the separating gel, mix 1.75 ml of 40% acrylamide/bis-acrylamide (19:1) solution, 1.875 ml of 1 M Tris-HCl pH 8.8, 25 Î¼l of 20% (w/v) SDS and adjust to 5 ml with water. Add 40 Î¼l of 10% APS and 5 Î¼l of TEMED just before pouring the gel between the glass plates at 2/3 of the total height. For the stacking gel, mix 500 Î¼l of 40% acrylamide/bis-acrylamide (19:1) solution, 630 Î¼l of 1 M Tris-HCl pH 6.8, 50 Î¼l of 20% (w/v) SDS and adjust to 5 ml with water. Add 40 Î¼l of 10% APS and 5 Î¼l of TEMED just before pouring the gel on top of the polymerized separating gel. Put immediately the 10-well comb.<br/></li><li>Load the samples on the 14% SDS polyacrylamide gel. Migrate in 1x TGS buffer at 200 V for ~1 h 30 min.<br/></li><li>Visualize the RNAs in the fractions with the toluidine-blue staining (as described in Step A2c). Fractions containing the tRNA of interest can be easily identified with the appearance of an additionnal band at higher molecular weight in the presence of the complementary DNA oligonucleotide (Figure 2B).</li></ol></li><li>Ion exchange chromatography<br/><ol class="npwfour"><li>Pool the fractions from Step B2b. that contain the yeast tRNA<sup>Phe</sup> as identified in Step B2c.<br/></li><li>Load the pool on a Mono Q 10/100 GL column previously equilibrated with purification buffer C.<br/></li><li>Apply a gradient from 45% to 60% of buffer D over 30 CV at 2 ml/min. Collect fractions of 1.5 ml.<br/></li><li>Identify the fractions containing the yeast tRNA<sup>Phe</sup> with a gel-shift assay as described in Step B2c and pool the fractions of interest.<br/><br/><img alt="" class="layerphoto" height="127" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200614/20200614074501_8640.jpg" width="300"/><br/><strong>Figure 2. Gel-shift assays with a complementary DNA oligonucleotide.</strong> A. Schematic representation of the complementary DNA oligonucleotide used to identify the yeast tRNA<sup>Phe</sup> in purification fractions. The exact sequence is given in the list of Material and Reagents. B. Gel-shift assay with a 14% SDS-PAGE done with fractions (#1-#4) of the Mono Q column elution step revealed by toluidine-blue staining. Fractions containing the tRNA of interest (<em>i.e.</em>, yeast tRNA<sup>Phe</sup>) can be easily identified with the appearance of an additionnal band at higher molecular weight in the presence of the complementary DNA oligonucleotide (labeled with a star), as compared with the control samples without the complementary DNA oligonucleotide.<br/><br/></li></ol></li><li>Hydrophobic interaction chromatography<br/><ol class="npwfour"><li>Add ammonium sulfate to the pooled fractions of Step B2d to reach a final concentration of 1.7 M. Dissolve the salt by vortexing.<br/></li><li>Load this sample at 0.5 ml/min on a Phenyl superose HR 10/10 column previously equilibrated with purification buffer E.<br/></li><li>Apply a gradient from 20% to 45% of buffer C over 15 CV at 0.5 ml/min. Collect fractions of 2.5 ml.<br/></li><li>Identify the fractions containing the yeast tRNA<sup>Phe</sup> with a gel-shift assay as described in Step B2c and pool the fractions of interest.</li></ol></li><li>Desalting the sample<br/>Desalt the sample with a HiPrep 26/10 Desalting column previously equilibrated with the NMR buffer or dialyze the sample extensively against the NMR buffer. For that, one usually dialyzes the sample against a 2 L dialysis solution for at least 2 h and changes it with a fresh solution 3 times.</li><li>Sample concentration<br/>Concentrate the modified yeast tRNA<sup>Phe</sup> sample in the NMR buffer to ~0.2-0.5 mM using Amicon 3,000 MWCO. In our hands, with a 2 L culture in <sup>15</sup>N-labeled rich media, one usually obtains a final NMR sample of 250 Î¼l at ~0.4 mM (<em>i.e.,</em> 0.08-0.12 Âµmol).</li><li>Store the sample at -20 Â°C or use it directly for NMR spectra measurements (see Note 2).</li></ol></li></ol></li></ol><p class="pptt" id="biaoti34454">### Notes</p><ol class="npwtwok"><li>The cloning step is not described in the present protocol since the approach and the protocol have been previously detailed in DÃ©gut <em>et al.</em> (2016). Briefly, five DNA oligonucleotides are designed to create after a phosphorylation and a hybridization step a DNA insert which corresponds to the yeast tRNA<sup>Phe</sup> gene flanked by overhangs corresponding to EcoRI and PstI restriction sequences at the 5â²- and 3â²-ends, respectively (see Figure 3). The pBSTNAV vector is linearized with EcoRI and PstI and classical ligation protocol is performed. A counter-selection step with SmaI can be performed to increase the number of positive clones bearing the yeast tRNA<sup>Phe</sup> gene as described in DÃ©gut <em>et al.</em> (2016). Positive clones are checked by sequencing.<br/><br/>Oligonucleotides for yeast tRNA<sup>Phe</sup> gene cloning in pBSTNAV:<br/>oligo #1: 5â² AATTCGCGGATTTAGCTCAGTTGGGAGAG 3â²<br/>oligo #2: 5â² CGCCAGACTGAAGATCTGGAGGTCCTG 3â²<br/>oligo #3: 5â² TGTTCGATCCACAGAATTCGCACCACTGCA 3â²<br/>oligo #4: 5â² GTGGTGCGAATTCTGTGGATCGAACACAGGACCTCCAGA 3â²<br/>oligo #5: 5â² TCTTCAGTCTGGCGCTCTCCCAACTGAGCTAAATCCGCG 3â²<br/><br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200614/20200614074521_6316.jpg" width="300"/><br/><strong>Figure 3. Schematic representation of the cloning region of pBSTNAV vector and sequence coding for the yeast tRNA<sup>Phe</sup> inserted between the EcoRI and PstI restriction sites.</strong> In pBSTNAV, the strong constitutive lpp promoter upstream of the tRNA gene leads to high levels of recombinant transcripts production. The five oligos used to generate the yeast tRNA<sup>Phe</sup> coding sequence (oligos #1-#5) are labeled on the figure and displayed with different colors for clarity.<br/><br/></li><li>For the yeast tRNA<sup>Phe</sup> samples produced as described in the current protocol, we measured NMR spectra on Bruker AVIII-HD 600 MHz and AVIII-HD 700 MHz (equipped with TCI 5-mm cryoprobes) with 5-mm Shigemi tubes. Measurements were conducted at 38 Â°C in the NMR buffer (10 mM Na-phosphate pH 6.5, 10 mM MgCl<sub>2</sub>) supplemented with 5% (v/v) D<sub>2</sub>O. Other types of NMR spectrometers and/or NMR tubes may obviously also be used.<br/></li></ol><p class="pptt" id="biaoti34455">### Recipes</p><ol class="npwone"><li>For the production of unmodified yeast tRNA<sup>Phe</sup> by <em>in vitro</em> transcription<ol class="npwtwo"><li>20x T7 transcription buffer (TB)<br/>1 M Tris-HCl pH 8.04 ml<br/>100 mM spermidine1 ml<br/>1 M DTT0.5 ml<br/>Triton<sup>TM</sup> X-10010 Î¼l<br/>H<sub>2</sub>O to10 ml<br/>Store at -20 Â°C in 1 ml aliquots for up to 1 year</li><li>2x RNA loading dye<br/>8 M Urea<br/>0.01% (w/v) bromophenol blue<br/>0.01% (w/v) xylene cyanol<br/>store at -20 Â°C in 1 ml aliquots</li><li>Toluidine-blue staining solution<br/>Toluidine blue0.5 g<br/>Ethanol200 ml<br/>Concentrated acetic acid5 ml<br/>H<sub>2</sub>O to500 ml<br/>store at room temperature</li><li>Purification and NMR buffers<br/><strong>Buffer A</strong><br/>25 mM Na-phosphate pH 6.5<br/>50 mM NaCl<br/>5 mM MgSO<sub>4</sub><br/>0.22 Î¼m filtered<br/><br/><strong>Buffer B</strong><br/>25 mM Na-phosphate pH 6.5<br/>1 M NaCl<br/>0.22 Î¼m filtered<br/><br/><strong>NMR buffer</strong><br/>10 mM Na-phosphate pH 6.5<br/>10 mM MgCl<sub>2</sub><br/><br/></li></ol></li><li>For the production of modified yeast tRNA<sup>Phe</sup> by <em>in vivo</em> recombinant expression in <em>E. coli</em> <ol class="npwtwo"><li>Extraction buffer<br/>1 mM Tris-HCl pH 7.4<br/>10 mM Mg-acetate</li><li>2x Laemmli loading buffer<br/>125 mM Tris-HCl pH 6.8<br/>20% (v/v) glycerol<br/>4% (w/v) SDS<br/>10% (v/v) 2-mercaptoethanol<br/>0.004% (w/v) bromophenol blue</li><li>Purification and NMR buffers<br/><strong>Buffer C</strong><br/>25 mM K-phosphate pH 6.5<br/>0.22 Î¼m filtered<br/><br/><strong>Buffer D</strong><br/>25 mM K-phosphate pH 6.5<br/>1 M NaCl<br/>0.22 Î¼m filtered<br/><br/><strong>Buffer E</strong><br/>25 mM K-phosphate pH 6.5<br/>1.7 M (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub><br/>0.22 Î¼m filtered<br/><br/><strong>NMR buffer</strong><br/>10 mM Na-phosphate pH 6.5<br/>10 mM MgCl<sub>2</sub><br/></li></ol></li></ol><p class="pptt" id="biaoti34456">### Acknowledgments</p><p class="ppzz">This work was supported by the ANR NMR-VitAmin (ANR-14-CE09-0012), the CNRS, the UniversitÃ© de Paris, and the Labex DYNAMO (ANR-11-LABX-0011).</p><p class="pptt" id="biaoti34457">### Competing interests</p><p class="ppzz">The authors declare no competing interest.</p><p class="pptt" id="biaoti34458">### References</p><ol class="npwtwok"><li>Boccaletto, P., Machnicka, M. A., Purta, E., Piatkowski, P., Baginski, B., Wirecki, T. K., de Crecy-Lagard, V., Ross, R., Limbach, P. A., Kotter, A., Helm, M. and Bujnicki, J. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29106616" target="_blank">MODOMICS: a database of RNA modification pathways. 2017 update.</a> <em>Nucleic Acids Res</em> 46(D1): D303-D307.</li><li>El Yacoubi, B., Bailly, M. and de CrÃ©cy-Lagard, V. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22905870" target="_blank">Biosynthesis and function of posttranscriptional modifications of transfer RNAs.</a> <em>Annu Rev Genet</em> 46: 69-95.</li><li>Jackman, J. E. and Alfonzo, J. D. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23139145" target="_blank">Transfer RNA modifications: nature's combinatorial chemistry playground.</a> <em>Wiley Interdiscip Rev RNA</em> 4(1): 35-48. </li><li>Barraud, P. and TisnÃ©, C. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30932315" target="_blank">To be or not to be modified: Miscellaneous aspects influencing nucleotide modifications in tRNAs.</a> <em>IUBMB Life</em> 71(8): 1126-1140.</li><li>Barraud, P., Gato, A., Heiss, M., Catala, M., Kellner, S. and TisnÃ©, C. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31358763" target="_blank">Time-resolved NMR monitoring of tRNA maturation.</a> <em>Nat Commun</em> 10(1): 3373.</li><li>Rio, D. C. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24184761" target="_blank">Expression and purification of active recombinant T7 RNA polymerase from <em>E. coli.</em></a> <em>Cold Spring Harb Protoc</em> 2013(11).</li><li>DÃ©gut, C., Monod, A., Brachet, F., CrÃ©pin, T. and TisnÃ©, C. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26227036" target="_blank"><em>In vitro</em>/<em>in vivo</em> production of tRNA for X-Ray studies.</a> <em>Methods Mol Biol</em> 1320: 37-57.</li><li>Mechulam, Y., Blanquet, S. and Fayat, G. (1987). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3326947" target="_blank">Dual level control of the <em>Escherichia coli</em> pheST-himA operon expression. tRNA(Phe)-dependent attenuation and transcriptional operator-repressor control by himA and the SOS network.</a> <em>J Mol Biol</em> 197(3): 453-470.<br/></li></ol>