<body>
            <h1>Electrophoretic Mobility Shift Assay of <em>in vitro</em> Phosphorylated RNA Polymerase II Carboxyl-terminal Domain Substrates</h1>
            <p><strong>Authors:</strong> Joshua E. Mayfield, Seema Irani, Yan Zhang</p>
            <h2>Abstract</h2>
            <p>Eukaryotic RNA polymerase II transcribes all protein-coding mRNAs and is highly regulated. A key mechanism directing RNA polymerase II and facilitating the co-transcriptional processing of mRNAs is the phosphorylation of its highly repetitive carboxyl-terminal domain (CTD) of its largest subunit, RPB1, at specific residues. A variety of techniques exist to identify and quantify the degree of CTD phosphorylation, including phosphorylation-specific antibodies and mass spectrometry. Electrophoretic mobility shift assays (EMSAs) have been utilized since the discovery of CTD phosphorylation and continue to represent a simple, direct, and widely applicable approach for qualitatively monitoring CTD phosphorylation. We present a standardized method for EMSA analysis of recombinant GST-CTD substrates phosphorylated by a variety of CTD kinases. Strategies to analyze samples under both denatured/reduced and semi-native conditions are provided. This method represents a simple, direct, and reproducible means to monitor CTD phosphorylation in recombinant substrates utilizing equipment common to molecular biology labs and readily applicable to downstream analyses including immunoblotting and mass spectrometry.</p>
            <p><strong>Keywords:</strong> RNA polymerase II, Transcription, Phosphorylation, Kinase, mRNA, CTD Code</p>
            <h2>Background</h2><p>Eukaryotic RNA polymerase II (RNAPII) generates all protein-coding mRNAs, small nuclear, small nucleolar, and many micro RNAs (Jeronimo <em>et al.</em>, 2013; Mayfield <em>et al.</em>, 2016). A variety of mechanisms regulate RNAPII activity to confer specificity to gene expression and facilitate biological processes. Among these is the direct post-translational modification of RNAPII itself in the form of phosphorylation (Mayfield <em>et al.</em>, 2016), prolyl isomerization (Mayfield <em>et al.</em>, 2015), methylation (Dias <em>et al.</em>, 2015), and acetylation (Schroder <em>et al.</em>, 2013). Some of the best-studied modifications are phosphorylations of the C-terminal domain of RNAPIIâs largest subunit RPB1 (CTD). This domain is evolutionarily conserved from yeast to mammals and composed of a species-specific number of repeats of the consensus amino acid heptad YSPTSPS (conventionally numbered as Tyr1, Ser2, Pro3, Thr4, Ser5, Pro6, and Ser7). Phosphorylations of specific heptad residues act to recruit transcription factors and coincide with distinct stages of the transcription cycle. For instance, Ser5 phosphorylation hallmarks transcription initiation while Ser2 phosphorylation is apparent during the transition from promoter escape to productive elongation. These dynamic post-translational modifications and precisely recruited protein factors constitute the âCTD Codeâ for eukaryotic transcription and ensure the production of mature and functional transcripts (Jeronimo <em>et al.</em>, 2013; Mayfield <em>et al.</em>, 2016).<br/> <br/>RNAPII is recruited to the pre-initiation complex when the CTD is in the unphosphorylated form. Upon transcription initiation, the CTD becomes hyperphosphorylated, indicative of a transcriptionally engaged RNAPII. At transcription termination, the CTD undergoes dephosphorylation and is recycled to initiate another round of transcription. This is achieved through the action of multiple CTD kinases; including CDK7, of the TFIIH complex, and CDK9, of the Positive transcription elongation factor b (P-TEFb) complex; and their counterparts the CTD phosphatases, including SSU72 and CTDP1 (Jeronimo <em>et al.</em>, 2013; Mayfield <em>et al.</em>, 2016). Distinct pools of unphosphorylated and hyperphosphorylated RPB1 are detected in cell lysates using polyacrylamide gel electrophoresis (PAGE) due to a dramatic shift in the isoelectric point of RPB1 resulting from hyperphosphorylation. This physical characteristic of hyperphosphorylated RPB1 allowed for the initial discovery and characterization of the CTD and continues to be a useful tool in the study of CTD phosphorylation (Corden <em>et al.</em>, 1985; Mayfield <em>et al.</em>, 2019).<br/> <br/>There are many ways to interrogate both the identity and abundance of modifications in CTD substrates, including antibodies (Jeronimo <em>et al.</em>, 2013), mass spectrometry (Mayfield <em>et al.</em>, 2017) and biophysical approaches like small-angle x-ray scattering (Portz <em>et al.</em>, 2017). However, EMSA has the advantage of direct visualization, easy setup, and a rapid completion time without the requirement for specialized equipment. Here we treat GST-CTD fusion proteins with various CTD kinases, resolve reaction products in electrophoresis, and visualize product bands to describe reaction outcomes qualitatively. This approach is useful for verifying kinase and phosphatase activity against CTD substrates, processive/stochastic addition or removal of phosphates, and qualitative estimation of the number of phosphates added to CTD substrates. This approach is informative under both denatured/reduced and semi-native conditions allowing for tunable resolution and increasing downstream applicability in techniques like immunoblotting and band-excision coupled to mass spectrometry analysis.</p>
        </body>

<p class="pptt" id="biaoti34314">## Materials and Reagents</p><ol class="npwtwok"><li>SnakeSkin Dialysis Tubing, 10K MWCO, 22 mm (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 68100 </a> )</li><li>Amicon Ultra-15 Centrifugal Filter Unit (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> UFC900308 </a> )</li><li>GST-yeast CTD Bacterial Expression Vector (subcloned as in Mayfield <em>et al.</em>, 2019, or similar)</li><li>BL21 (DE3) Competent Cells (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> EC0114 </a> )</li><li>Yeast extract (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> Y1625 </a> )</li><li>Sodium chloride (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S9888 </a> )</li><li>Bacto tryptone (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 211699 </a> )</li><li>Bacto agar (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> DF0140010 </a> )</li><li>Kanamycin monosulfate (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> K1205 </a> )</li><li>IPTG (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> 367931 </a> )</li><li>Tris Base (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP1525 </a> )</li><li>Hydrochloric acid (Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> A144SI212 </a> )</li><li>Sodium Hydroxide (Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> S3201 </a> )</li><li>Triton X-100 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> X100 </a> )</li><li>Glycerol (Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> G334 </a> )</li><li>Imidazole (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 56750 </a> )</li><li>Î²-mercaptoethanol (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 125472500 </a> )</li><li>Pierce Coomassie Plus (Bradford) Assay Reagent (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 23238 </a> )</li><li>Ni-NTA His Bind Resin (EMD Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 70666 </a> )</li><li>Magnesium chloride hexahydrate (Sigma-Aldrich, catalog number M9272)</li><li>Adenosine 5â triphosphate disodium salt hydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A26209 </a> )</li><li>Cdk9/Cyclin T1 Protein, active (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 14-685 </a> )</li><li>Cdk7/Cyclin H/MAT1 (CAK complex) Protein, active (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 14476 </a> )</li><li>c-Abl kinase (ProQinase, catalog number: <a class="blackNoUnder" target="_blank"> 0992-0000-1 </a> )</li><li>pET-28 a vector (Novagen, catalog number: <a class="blackNoUnder" target="_blank"> 698643 </a> )</li><li>Sodium Dodecyl Sulfate (SDS) (OmniPur, catalog number: <a class="blackNoUnder" target="_blank"> 7910 </a> )</li><li>Bromophenol blue (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> B55255G </a> )</li><li>30% Acrylamide/Bis Solution 37.5:1 (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1610158 </a> )</li><li>Ammonium persulfate (APS) (Fisher Scientific, Research Products International Corp, catalog number: <a class="blackNoUnder" target="_blank"> A2050010.0 </a> )</li><li>N,N,Nâ,Nâ-Tetramethyl ethylenediamine TEMED (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP15020 </a> )</li><li>Glycine (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP381-5 </a> )</li><li>PageRuler Plus Prestained Protein Ladder, 10 to 250 kDa (Thermo Scientific, catalog number 26619), or equivalent pre-stained protein ladder</li><li>Brilliant Blue R250 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> B0149 </a> )</li><li>Ethanol, Ethyl alcohol 200 Proof (Pharmaco, catalog number: <a class="blackNoUnder" target="_blank"> 111000200 </a> )</li><li>Glacial Acetic acid (Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> A38C212 </a> )</li><li>LB Medium (1 L) (see Recipes)</li><li>1,000x Kanamycin stock (50 mg/ml) (see Recipes)</li><li>1,000x IPTG stock (400 mM) (see Recipes)</li><li>Lysis buffer (see Recipes)</li><li>Wash buffer (see Recipes)</li><li>Elution buffer (see Recipes)</li><li>Dialysis buffer (see Recipes)</li><li>4x Kinase Buffer (see Recipes)</li><li>4x ATP (5 ml) (see Recipes)</li><li>4x GST-yeast CTD substrate (see Recipes)</li><li>4x P-TEFb kinase solution (see Recipes)</li><li>4x TFIIH kinase solution (see Recipes)</li><li>4x c-Abl kinase solution (see Recipes)</li><li>1 M Tris-HCl pH 6.8 (500 ml) (see Recipes)</li><li>1.5 M Tris-HCl pH 8.8 (500 ml) (see Recipes)</li><li>10% SDS (100 ml) (see Recipes)</li><li>2x Laemmliâs sample buffer (see Recipes)</li><li>Resolving Semi-native PAGE/SDS-PAGE Gel (10% Acrylamide; 5 mlâsufficient for 1 gel, scale as necessary) (see Recipes)</li><li>Stacking Semi-native PAGE/SDS-PAGE Gel (5% Acrylamide, 2 mlâsufficient for 1 gel, scale as necessary) (see Recipes)</li><li>1x Laemmli running buffer (Tris-glycine) (1 L) (see Recipes)</li><li>1x Native Laemmli running buffer (Tris-glycine) (1 L) (see Recipes)</li><li>Coomassie brilliant blue stain (100 ml) (see Recipes)</li><li>Destain (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34315">### Equipment</p><ol class="npwtwok"><li>1 L flasks</li><li>Pipettes</li><li>-80 Â°C freezer</li><li>Bacterial Culture Incubator (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> SHKE435HP </a> ) sufficient to hold 1 L culture volumes</li><li>UV-Vis Spectrophotometer (SmartSpec Plus Spectrophotometer (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 170-2525 </a> , or equivalent)</li><li>High-Speed Floor Centrifuge with appropriate rotors for 1 L and 50 ml centrifuge containers (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Sorvall RC 6+ </a> , or equivalent)</li><li>Sonicator for Bacterial Lysis (Q500 Sonicator) (Qsonica, catalog number: <a class="blackNoUnder" target="_blank"> Q500-110 </a> , or equivalent)</li><li>Econo-Column Chromatography Column 2.5 x 10 cm (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 7374251 </a> )</li><li>FPLC System with automated fraction collector and UV280 monitoring capabilities (Bio-Rad, NGC system, or equivalent)</li><li>HiLoad 16/600 Superdex 200 pg (GE Lifesciences, catalog number: <a class="blackNoUnder" target="_blank"> 28989335 </a> , or equivalent)</li><li>NanoDrop One Microvolume UV-Vis Spectrophotometer (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ND-ONE-W </a> , or equivalent)</li><li>Mini-PROTEAN Tetra Vertical Electrophoresis cell (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1658004 </a> )</li><li>Mini-PROTEAN Tetra cell casting stand and clamps (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1658050 </a> )</li><li>Mini-PROTEAN Spacer Plates with 1.0 mm Integrated spacers (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1653311 </a> )</li><li>Mini-PROTEAN Short Plates (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1653308 </a> )</li><li>Mini-PROTEAN Comb, 15-well, 1 mm, 26 Âµl (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1653360 </a> )</li><li>PowerPac Basic Power Supply (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1645050 </a> )</li><li>G: BOX imaging systems (Syngene), or equivalent<br/></li></ol><p class="pptt" id="biaoti34316">### Software</p><ol class="npwtwok"><li>ImageJ (Open source image processing software, imagej.net)</li></ol><p class="pptt" id="biaoti34317">### Procedure</p><ol class="npwone"><li>Expression and Purification of GST-yeast CTD Substrate<br/><em>Note: GST fusions of the CTDs from various organisms, including Homo sapiens, Drosophila melanogaster, and Saccharomyces cerevisiae, are utilized extensively. The easy availability of synthetic constructs containing defined numbers of heptad repeats greatly simplified cloning. The EMSA method presented here applies to all of these various substrates. However, a universal purification method does not apply to all constructs. We provide a purification protocol for a 6X-HIS tagged GST and Saccharomyces cerevisiae CTD fusions due to their historical precedence, the content of primarily consensus heptad repeats, and extensive use as a model substrate regardless of CTD modifying enzyme origin. Purification schemes for alternative CTD constructs are available throughout published literature and/or should be determined empirically.</em><br/><br/><ol class="npwtwo"><li>Subclone GST-yeast CTD sequence (as described in Mayfield <em>et al.</em>, 2019) into the pET-28 a (+) vector, or equivalent, using any standard cloning method.</li><li>Transform sequence-verified plasmid into BL21 (DE3) Competent E. coli cells using conventional methods and select for transformants by growing overnight at 37 Â°C on an agar plate with 50 Î¼g/ml kanamycin.</li><li>Inoculate a single colony into 10ml of LB media containing 50 Î¼g/ml kanamycin (10 ml LB + 10 Î¼l 1,000x kanamycin stock) and grow overnight at 37 Â°C to generate a saturated culture.</li><li>Inoculate two 1 L flasks of LB containing 50 Î¼g/ml kanamycin (1 L LB + 1,000 Î¼l 1,000x kanamycin stock) with 4 ml of the saturated overnight culture each flask. Incubate at 37 Â°C with shaking at 180 rpm. Monitor OD<sub>600</sub> using UV-Vis spectrophotometry until the cultures reach a value of approximately 0.6-0.8.</li><li>Induce expression by adding 1 ml of 1,000x IPTG stock and allow the culture to grow at 37 Â°C with shaking at 180 rpm for an additional four hours.</li><li>Harvest cell pellet by transferring culture to appropriate 1 L centrifuge flasks, counterbalancing the flasks if necessary, and centrifuging at 5,000 <em>x g</em> for 20 min at room temperature. Cell pellets can be processed immediately or frozen at -20 Â°C for up to 1 year.</li><li>Combine cell pellets from 2 L of growth media and resuspend in 100 ml of Lysis Buffer. Allow pellet to resuspend fully by stirring solution vigorously on ice for 30 min.</li><li>Lyse cells via sonication using a properly tuned instrument. Sonicate suspended cells on ice using 10 cycles of 30 s continuous sonication followed by 1 min of recovery with constant stirring. In the end, the solution will appear yellowish for an effective sonication.</li><li>Clear lysate of cell debris by splitting lysate equally among 50 ml centrifuge tubes, verifying they are of equal mass, and centrifuging samples at 10,000 <em>x g</em> for 40 min at 4 Â°C. Pool supernatant fractions and discard cell debris containing pellets.</li><li>Apply 5 ml of Ni-NTA His Bind Resin slurry to an empty chromatography column and allow supernatant to flow through. Equilibrate beads with 5 ml of lysis buffer and combine bead/lysis buffer slurry with pooled supernatant fractions. Stir the supernatant with the beads on a stir plate in the cold room for 30 min to ensure protein binding.</li><li>Apply supernatant/bead mixture to the chromatography column and allow the supernatant to flow through. Collect flow-through and pour it back over the beads one time. Your protein is now bound to the beads.</li><li>Wash the beads with 100 ml of wash buffer, allowing it to flow through.</li><li>Elute GST-yeast CTD from the column by adding 10 ml of Elution buffer and allowing the solution to slowly drip from the chromatography column and collect it in a clean 50 ml conical tube.</li><li>Verify protein content by mixing 200 Î¼l Pierce Coomassie Plus (Bradford) Assay Reagent with 2 Î¼l of the elution. If protein is present, the solution turns a vibrant blue. Verify protein content and identity using classical SDS-PAGE analysis of the elution. GST-yeast CTD (as described in Mayfield <em>et al.</em>, 2019) has a theoretical molecular weight of 48.7 kDa.</li><li>Transfer eluted protein to SnakeSkin Dialysis tubing and dialyzed against 1 L of Dialysis Buffer overnight at 4 Â°C.</li><li>Equilibrate HiLoad 16/600 Superdex 200 pg column with Dialysis Buffer using an FPLC by washing the column with 1.5 column volumes at a flow rate of 1 ml/min.</li><li>Prepare an Amicon Ultra-15 Centrifugal Filter Unit by adding 5-10 ml deionized water to the unit and spinning the unit at 3,500 <em>x g</em> for 5 min. Discard remaining water on both sides of the molecular weight cut off filter.</li><li>Apply dialyzed protein to the prepared centrifugal filter unit and centrifuge the unit at 3,500 <em>x g</em> for 10-min intervals at 4 Â°C until the volume above the molecular weight cut-off filter reaches 1 ml. Transfer the volume above the filter to a 1.5 ml tube and centrifuge at 13,000 <em>x g</em> at 4 Â°C for 10 min to remove any protein aggregates. Transfer cleared supernatant to a fresh 1.5 ml tube.</li><li>Inject concentrated sample onto the FPLC and prepared column. Monitor elution absorbance at 280 nm and collect 1 ml fractions beginning at the void volume and ending at 1.2 column volumes. Using the absorbance trace, determine fractions containing GST-yeast CTD and prepare SDS-PAGE samples by mixing 20 Î¼l of the fraction with 20 Î¼l of 2x Laemmliâs Sample Buffer and boiling the samples at 95 Â°C for 5 min. Verify protein identity and purity using established SDS-PAGE and Coomassie Brilliant Blue staining protocols, similar to those presented below.</li><li>Pool fractions containing GST-yeast CTD and concentrate in Amicon centrifugal unit until approximately 10mg/ml as determined by absorbance at 280 nm on NanoDrop. Aliquot to 50 Î¼l fractions to 1.5 ml tubes and flash freeze in liquid nitrogen. Store at -80 Â°C.<br/><br/></li></ol></li><li><em>In vitro</em> phosphorylation of GST-yeast CTD Substrate using CTD Kinases<ol class="npwtwo"><li>Kinase reactions<br/>Combine stock solutions as follows:<br/>5 Î¼l 4x Kinase Buffer (Recipe 8)<br/>5 Î¼l 4x GST-yeast CTD Substrate (Recipe 10)<br/>5 Î¼l 4x Kinase Solution (Recipes 11, 12, 13 as desired)<br/>5 Î¼l 4x ATP<br/>Initiate reactions by adding 4x ATP. Incubate reactions in a thermocycler at 30 Â°C for the desired amount of time. Reaction time to completion is kinase-dependent and should be determined empirically. In our hands, TFIIH and P-TEFb reactions approach completion after approximately 1 h, while c-Abl reactions require approximately 4 h to reach completion. Exact reaction times depend on the kinase, experimental goal, and production lot and should be determined empirically. Completed reactions can be analyzed immediately or stored at -80 Â°C for up to 1 year.<br/><em>Note: To date, various CTD kinases are commercially available or described in the literature. Here, we provide a reaction condition for studying CTD phosphorylation by three commercially available human CTD kinases/kinase complexes: TFIIH, P-TEFb, and c-Abl. Alternative kinases can also be used, but reaction conditions should be optimized for the kinase of interest. Special attention should be paid to the lot number, and commercial sources of kinases as their individual activity varies widely and require optimization.</em> </li><li>No kinase control reactions<br/>Combine stock solutions as follows:<br/>5 Î¼l 4x Kinase Buffer (Recipe 8)<br/>5 Î¼l 4x GST-yeast CTD Substrate (Recipe 10)<br/>5 Î¼l Deionized water<br/>5 Î¼l 4x ATP<br/>Initiate reactions by adding 4x ATP. Incubate reactions in thermocycler at 30 Â°C for desired amount of time. No kinase control reactions should be run in parallel with the kinase-containing reactions. Completed reactions can be analyzed immediately or stored at -80 Â°C for up to 1 year.<br/><br/></li></ol></li><li>Preparation of Electrophoresis Samples<br/><em>Note: Samples for both SDS-PAGE and Semi-Native PAGE are prepared identically. The denaturation/reduction of Semi-Native PAGE samples in Laemmli sample buffer increases the sharpness and resolution of final bands relative to purely native samples. It tends to increase the degree of electrophoretic mobility shift between unphosphorylated and phosphorylated substrates over that observed in SDS-PAGE.</em><br/><br/>Quench reactions by adding 20 Î¼l of 2x Laemmliâs Sample Buffer and boiling at 95 Â°C for 5 min. This yields approximately 40 Î¼l of the final sample with a GST-yeast CTD substrate concentration of around 0.5 Î¼g/Î¼l.<br/><br/></li><li>SDS-PAGE (Denaturing/Reducing)<br/><em>Note: 10% acrylamide SDS-PAGE gels are suggested here as they resolve proteins well in the mass range of </em><em>GST-yeast CTD</em> <em>substrates. Consider alternative acrylamide concentrations when using alternative substrates or a different resolution is desired.</em><br/><ol class="npwtwo"><li>Set up the Bio-Rad Mini-PROTEAN Tetra Cell according to factory directions with SDS-PAGE gels (Recipes 18-19) and 1x Laemmliâs Running Buffer (Tris-glycine) (Recipe 20). Load 2 Î¼l (1 Î¼g) of prepared electrophoresis samples to each well, making sure to include both kinase containing and no kinase control reactions. Include a molecular weight ladder.</li><li>Run the gel at 150 V (constant) for approximately 1 h, or until the dye front reaches the bottom of the gel.</li><li>Remove the gel from the electrophoresis chamber and glass casting and proceed immediately to Coomassie Brilliant Blue Staining or alternative downstream application.<br/><br/></li></ol></li><li>Semi-Native PAGE<br/><em>Note: 10% acrylamide Semi-native PAGE gels are suggested here as they resolve proteins well in the mass range of </em><em>GST-yeast CTD</em> <em>substrates. Consider alternative acrylamide concentrations when using alternative substrates or a different resolution is desired.</em><br/><ol class="npwtwo"><li>Set up the Bio-Rad Mini-PROTEAN Tetra Cell according to factory directions with Semi-native PAGE gels (Recipes 18-19) and 1x Native Laemmliâs Running Buffer (Tris-glycine) (Recipe 21). Load 2 Î¼l (1 Î¼g) of prepared electrophoresis samples to each well, making sure to include both kinase containing and no kinase control reactions. Include a pre-stained molecular weight ladder. The ladder masses are not directly interpretable in Semi-native PAGE but aid in reproducibility of Semi-native PAGE experiments and provide gel progress information if the proteins of interest are not well separated before the dye front leaves the gel.</li><li>Run the gel at 150 V (constant) for approximately 1-4 h with an ice pack. Long runs may be necessary to obtain well-separated bands, and this time should be optimized for your particular samples.</li><li>Remove the gel from the electrophoresis chamber and glass casting and proceed immediately to Coomassie Brilliant Blue Staining or alternative downstream application.<br/><br/></li></ol></li><li>Coomassie brilliant blue staining<ol class="npwtwo"><li>Briefly rinse gel with deionized water. Decant deionized water and add sufficient Coomassie Brilliant Blue stain to cover the gel. Incubate gel at room temperature with gentle agitation for a minimum of 1 h or overnight. Decant Coomassie Brilliant Blue stain into an appropriate container. The stain can be reused multiple times.</li><li>Rinse gel thoroughly in deionized water to remove residual Coomassie Brilliant Blue stain and cover gel completely with Destain. Incubate gel at room temperature with gentle agitation until stained protein bands become visible, and background is minimal, changing the destain as necessary.<br/></li></ol></li></ol><p class="pptt" id="biaoti34318">### Data analysis</p><p class="ppzz">The destained gel can be immediately visually interpreted. In SDS-PAGE applications, phosphorylation decreases the mobility of the GST-yeast CTD substrate. This is evidenced by a higher apparent molecular weight of the kinase treated sample relative to the no kinase control sample. In the semi-native PAGE, phosphorylation increases the mobility of the GST-yeast CTD substrate. This will appear as bands of lower apparent molecular weight in the kinase treated sample relative to the no kinase control. Gels can be imaged in any conventional gel imagine system, including Chemi-Doc (Bio-Rad) and G: Box (Syngene) systems. Cropping of images can be performed in ImageJ. ImageJ may also be used to adjust brightness and contrast equally across the image to increase the interpretability of the gels. Special care should be taken to ensure image modifications are applied judiciously and equivalently across all samples considered.<br/>  EMSA analyses have been employed throughout the literature. Examples of the method presented here can be found in Mayfield <em>et al.</em> (2019). An application of the SDS-PAGE EMSA to two types of GST-CTD substrate can be found in Figures 1C and 2A. Semi-native PAGE is presented as a useful alternative when increase resolution is desired and requires minimal alteration of the more established SDS-PAGE protocol as in Supplementary Figure 1D.</p><p class="pptt" id="biaoti34319">### Recipes</p><ol class="npwtwok"><li>LB Medium (1 L)<br/>Combine the following in 900 ml deionized water:<br/>10 g Bacto tryptone<br/>5 g yeast extract<br/>10 g NaCl<br/>12 g Bacto agar (for agar plates only)<br/><ol class="npwthree"><li>Shake or stir the solution to dissolve solutes<br/></li><li>Adjust final volume to 1 L using deionized water<br/></li><li>Sterilize by autoclaving for 20 min at 15 psi on a liquid cycle<br/></li><li>If desired, add antibiotic once media has cooled completely (for liquid media) or is molten but comfortable to the touch (for agar plates). Plates should then be poured under aseptic conditions</li></ol></li><li>1,000x Kanamycin stock (50 mg/ml)<br/><ol class="npwthree"><li>Dissolve 500 mg of kanamycin monosulfate (M.W. = 582.60) in 9 ml deionized water<br/></li><li>Adjust volume to 10 ml with deionized water<br/></li><li>Filter sterilize solution through a 0.22 Î¼m filter and aliquot into sterile tubes<br/></li><li>Store at -20 Â°C</li></ol></li><li>1,000x IPTG stock (400 mM)<br/><ol class="npwthree"><li>Dissolve 950 mg of Isopropyl-beta-D-thiogalactoside (IPTG, M.W. = 238.30) in 9 ml deionized water<br/></li><li>Adjust volume to 10 ml with deionized water<br/></li><li>Filter sterilize solution through a 0.22 Î¼m filter and aliquot into sterile tubes</li><li>Store at -20 Â°C<br/></li></ol></li><li>Lysis buffer (1 L)<br/>50 mM Tris-HCl pH 8.0 (50 ml diluted from 1 M Tris-His pH 8.0 stock [Recipe 14])<br/>500 mM NaCl (29.25 g)<br/>15 mM Imidazole (15 ml from 1 M stock)<br/>10% Glycerol (10 ml from 100% stock)<br/>0.1% Triton X-100 (1 ml from 100% stock)<br/>10 mM Î²-mercaptoethanol (0.7 ml from stock of 14.3 M)</li><li>Wash buffer (1 L)<br/>50 mM Tris-HCl pH 8.0 (50 ml diluted from 1 M Tris-His pH 8.0 stock [Recipe 14])<br/>500 mM NaCl (29.25 g)<br/>15 mM Imidazole (15 ml from 1 M stock)<br/>10 mM Î²-mercaptoethanol (0.7 ml from stock of 14.3 M)</li><li>Elution buffer (1 L)<br/>50 mM Tris-HCl pH 8.0 (50 ml diluted from 1 M Tris-His pH 8.0 stock [Recipe 14])<br/>500 mM NaCl (29.25 g)<br/>400 mM Imidazole (400 ml from 1 M stock)<br/>10 mM Î²-mercaptoethanol (0.7 ml from stock of 14.3 M)</li><li>Dialysis buffer (1 L)<br/>50 mM Tris-HCl pH 8.0 (50 ml diluted from 1 M Tris-His pH 8.0 stock [Recipe 14])<br/>50 mM NaCl (2.9 g)<br/>10 mM Î²-mercaptoethanol (0.7 ml from the stock of 14.3 M)<br/>For FPLC applications, the buffer should be filtered through a 0.4 Î¼m filter to remove any particulates</li><li>4x Kinase Buffer (100 ml)<br/>200 mM Tris-HCl pH 7.5 (20 ml diluted from 1 M Tris-His pH 7.5 stock [Recipe 14])<br/>200 mM MgCl<sub>2</sub> (10 ml from 2 M stock)</li><li>4x ATP (5 ml)<br/>22 mg Adenosine 5â triphosphate disodium salt hydrate<br/>1 M Tris-HCl pH 7.5 (variable)<br/><ol class="npwthree"><li>Dissolve adenosine 5â triphosphate disodium salt hydrate in 4 ml of deionized water<br/></li><li>Verify pH by spotting a small amount (1-2 Î¼l) of solution onto pH paper. The initial solution should be very acidic<br/></li><li>Adjust pH with 1 M Tris-HCl pH 7.5 by adding 20-100 Î¼l at a time and checking the pH after each addition. Once the pH registers 7.5, adjust the final volume of the solution to 5 ml using deionized water. Adjusting the pH of the ATP solution is essential</li></ol></li><li>10% SDS (100 ml)<br/>10 g Sodium Dodecyl Sulfate (SDS)<br/>80 ml Deionized water</li><li>2x Laemmliâs Sample Buffer<br/>0.125 M Tris-HCl pH 6.8<br/>4% SDS (w/v)<br/>10% Î²-mercaptoethanol (v/v)<br/>20% Glycerol (v/v)<br/>0.02% Bromophenol blue (w/v)</li><li>Resolving Semi-native PAGE/SDS-PAGE Gel (10% Acrylamide; 5 mlâsufficient for 1 gel, scale as necessary)<br/>1.7 ml 30% Acrylamide<br/>1.3 ml 1.5M Tris-HCl pH 8.8<br/>1.9 ml Deionized water<br/>50 Î¼l 10% SDS (w/v) (for SDS-PAGE gels only)<br/>50 Î¼l Deionized water (for Semi-native PAGE gels only)<br/>50 Î¼l 10% Ammonium persulfate (w/v)<br/>4 Î¼l TEMED</li><li>Stacking Semi-native PAGE/SDS-PAGE Gel (5% Acrylamide, 2 mlâsufficient for 1 gel, scale as necessary)<br/>330 Î¼l 30% Acrylamide<br/>250 Î¼l 1M Tris-HCl pH 6.8<br/>1.4 ml Deionized water<br/>20 Î¼l 10% SDS (w/v) (for SDS-PAGE gels only)<br/>20 Î¼l Deionized water (for Semi-Native PAGE gels only)<br/>20 Î¼l 10% Ammonium persulfate (w/v)<br/>2 Î¼l TEMED</li><li>1x Laemmli running buffer (Tris-glycine) (1 L)<br/>Combine the following in 900 ml deionized water:<br/>3.03 g Tris base<br/>14.2 g Glycine<br/>1 ml 10% SDS (w/v)<br/>Allow all reagents to dissolve and adjust pH to 8.3 if necessary. Adjust volume to 1 L with additional deionized water.<br/><ol class="npwthree"><li>Adjust pH to 6.8 (or 7.5 and 8.0) using concentrated hydrochloric acid and bring the final volume to 500 ml with additional deionized water<br/></li><li>Store at room temperature</li></ol></li><li>1x Native Laemmli running buffer (Tris-glycine) (1 L)<br/>Combine the following in 900 ml deionized water:<br/>3.03 g Tris base<br/>14.2 g Glycine<br/>Allow all reagents to dissolve and adjust pH to 8.3 if necessary. Adjust volume to 1 L with additional deionized water.<br/><ol class="npwthree"><li>Combine Tris base and deionized water<br/></li><li>Adjust pH to 8.8 with concentrated hydrochloric acid and bring the final volume to 500 ml with additional deionized water<br/></li><li>Store at room temperature</li></ol></li><li>Coomassie Brilliant Blue Stain (100 ml)<br/>Combine the following:<br/>0.25 g Coomassie Brilliant Blue R250<br/>45 ml Ethanol<br/>45 ml Deionized water<br/>10 ml Glacial acetic acid<br/>Stir constantly until completely dissolved<br/><ol class="npwthree"><li>Combine SDS and deionized water and stir until dissolved<br/></li><li>Adjust final volume to 100 ml with additional deionized water<br/></li><li>Store at room temperature</li></ol></li><li>Destain<br/>Combine the following:<br/>50 ml Ethanol<br/>75 ml Glacial acetic acid<br/>875 ml Deionized water<br/>Stir or invert container until thoroughly mixed<br/></li><li>Resolving Semi-native PAGE/SDS-PAGE Gel (10% Acrylamide; 5 mlâsufficient for 1 gel, scale as necessary)<br/>1.7 ml 30% Acrylamide<br/>1.3 ml 1.5M Tris-HCl pH 8.8<br/>1.9 ml Deionized water<br/>50 Î¼l 10% SDS (w/v) (for SDS-PAGE gels only)<br/>50 Î¼l Deionized water (for Semi-native PAGE gels only)<br/>50 Î¼l 10% Ammonium persulfate (w/v)<br/>4 Î¼l TEMED<br/><ol class="npwthree"><li>Combine 30% Acrylamide, 1.5 M Tris-HCl pH 8.8, deionized water, SDS (for SDS-PAGE), or additional deionized water (for Semi-native PAGE) in a screw-top vial and invert to combine<br/></li><li>Add ammonium persulfate and TEMED to initiate polymerization, invert gently, and transfer solution to prepared Bio-Rad Mini-PROTEAN gel casting<br/></li><li>Fill casting, leaving sufficient space at the top of gel for stacking layer and comb</li><li>Gently apply a layer of 100% ethanol to the top of resolving gel and allow it to polymerize completely and pour off ethanol<br/></li></ol></li><li>Stacking Semi-native PAGE/SDS-PAGE Gel (5% Acrylamide, 2 mlâsufficient for 1 gel, scale as necessary)<br/>330 Î¼l 30% Acrylamide<br/>250 Î¼l 1M Tris-HCl pH 6.8<br/>1.4 ml Deionized water<br/>20 Î¼l 10% SDS (w/v) (for SDS-PAGE gels only)<br/>20 Î¼l Deionized water (for Semi-Native PAGE gels only)<br/>20 Î¼l 10% Ammonium persulfate (w/v)<br/>2 Î¼l TEMED<br/><ol class="npwthree"><li>Combine 30% Acrylamide, 1 M Tris-HCl pH 6.8, deionized water, and SDS (for SDS-PAGE) or additional deionized water (for Semi-native PAGE) in a screw-top vial and invert to combine<br/></li><li>Add ammonium persulfate and TEMED to initiate polymerization, invert gently, and transfer solution to prepared Bio-Rad Mini-PROTEAN gel casting containing polymerized resolving gel<br/></li><li>Fill the remainder of the casting with stacking gel solution and insert a comb<br/></li><li>Allow the gel to polymerize completely</li><li>Store final gels wrapped in moist paper towels and plastic wrap at 4 Â°C for up to 2 weeks<br/></li></ol></li><li>1x Laemmli running buffer (Tris-glycine) (1 L)<br/>Combine the following in 900 ml deionized water:<br/>3.03 g Tris base<br/>14.2 g Glycine<br/>1 ml 10% SDS (w/v)<br/>Allow all reagents to dissolve and adjust pH to 8.3 if necessary. Adjust volume to 1 L with additional deionized water.</li><li>1x Native Laemmli running buffer (Tris-glycine) (1 L)<br/>Combine the following in 900 ml deionized water:<br/>3.03 g Tris base<br/>14.2 g Glycine<br/>Allow all reagents to dissolve and adjust pH to 8.3 if necessary. Adjust volume to 1 L with additional deionized water.</li><li>Coomassie Brilliant Blue Stain (100 ml)<br/>Combine the following:<br/>0.25 g Coomassie Brilliant Blue R250<br/>45 ml Ethanol<br/>45 ml Deionized water<br/>10 ml Glacial acetic acid<br/>Stir constantly until completely dissolved</li><li>Destain<br/>Combine the following:<br/>50 ml Ethanol<br/>75 ml Glacial acetic acid<br/>875 ml Deionized water<br/>Stir or invert container until thoroughly mixed</li></ol><p class="pptt" id="biaoti34320">### Acknowledgments</p><p class="ppzz">This work is supported by grants from the National Institutes of Health (R01 GM104896 and 125882 to YJZ) and Welch Foundation (F-1778 to YJZ). Methods presented here are adapted from previous work (Mayfield <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti34321">### Competing interests</p><p class="ppzz">The authors declare no competing financial interests.</p><p class="pptt" id="biaoti34322">### References</p><ol class="npwtwok"><li>Corden, J. L., Cadena, D. L., Ahearn, J. M., Jr. and Dahmus, M. E. (1985). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2999785" target="_blank">A unique structure at the carboxyl terminus of the largest subunit of eukaryotic RNA polymerase II.</a> <em>Proc Natl Acad Sci U S A</em> 82(23): 7934-7938. </li><li>Dias, J. D., Rito, T., Torlai Triglia, E., Kukalev, A., Ferrai, C., Chotalia, M., Brookes, E., Kimura, H. and Pombo, A. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26687004" target="_blank">Methylation of RNA polymerase II non-consensus Lysine residues marks early transcription in mammalian cells.</a> <em>Elife</em> 4: 11215. </li><li>Jeronimo, C., Bataille, A. R. and Robert, F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23837720" target="_blank">The writers, readers, and functions of the RNA polymerase II C-terminal domain code.</a> <em>Chem Rev</em> 113(11): 8491-8522.</li><li>Mayfield, J. E., Burkholder, N. T. and Zhang, Y. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26779935" target="_blank">Dephosphorylating eukaryotic RNA polymerase II.</a> <em>Biochim Biophys Acta</em> 1864(4): 372-387. </li><li>Mayfield, J. E., Fan, S., Wei, S., Zhang, M., Li, B., Ellington, A. D., Etzkorn, F. A. and Zhang, Y. J. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26332362" target="_blank">Chemical tools to decipher regulation of phosphatases by proline isomerization on eukaryotic RNA Polymerase II.</a> <em>ACS Chem Biol</em> 10(10): 2405-2414. </li><li>Mayfield, J. E., Irani, S., Escobar, E. E., Zhang, Z., Burkholder, N. T., Robinson, M. R., Mehaffey, M. R., Sipe, S. N., Yang, W., Prescott, N. A., Kathuria, K. R., Liu, Z., Brodbelt, J. S. and Zhang, Y. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31385803" target="_blank">Tyr1 phosphorylation promotes phosphorylation of Ser2 on the C-terminal domain of eukaryotic RNA polymerase II by P-TEFb.</a> <em>Elife</em> 8: 48725. </li><li>Mayfield, J. E., Robinson, M. R., Cotham, V. C., Irani, S., Matthews, W. L., Ram, A., Gilmour, D. S., Cannon, J. R., Zhang, Y. J. and Brodbelt, J. S. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28103682" target="_blank">Mapping the phosphorylation pattern of <em>Drosophila melanogaster</em> RNA polymerase II carboxyl-terminal domain using ultraviolet photodissociation mass spectrometry.</a> <em>ACS Chem Biol</em> 12(1): 153-162.</li><li>Portz, B., Lu, F., Gibbs, E. B., Mayfield, J. E., Rachel Mehaffey, M., Zhang, Y. J., Brodbelt, J. S., Showalter, S. A. and Gilmour, D. S. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28497792" target="_blank">Structural heterogeneity in the intrinsically disordered RNA polymerase II C-terminal domain.</a> <em>Nat Commun</em> 8: 15231.</li><li>Schroder, S., Herker, E., Itzen, F., He, D., Thomas, S., Gilchrist, D. A., Kaehlcke, K., Cho, S., Pollard, K. S., Capra, J. A., Schnolzer, M., Cole, P. A., Geyer, M., Bruneau, B. G., Adelman, K. and Ott, M. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24207025" target="_blank">Acetylation of RNA polymerase II regulates growth-factor-induced gene transcription in mammalian cells.</a> <em>Mol Cell</em> 52(3): 314-324. <br/></li></ol>