<body>
            <h1>Site-specific DNA Mapping of Protein Binding Orientation Using Azidophenacyl Bromide (APB)</h1>
            <p><strong>Authors:</strong> Himasha M. Perera, Michael  A. Trakselis</p>
            <h2>Abstract</h2>
            <p>The orientation of a DNA-binding protein bound on DNA is determinative in directing the assembly of other associated proteins in the complex for enzymatic action. As an example, in a replisome, the orientation of the DNA helicase at the replication fork directs the assembly of the other associated replisome proteins. We have recently determined the orientation of <em>Saccharalobus solfataricus (Sso)</em> Minichromosome maintenance (MCM) helicase at a DNA fork utilizing a site-specific DNA cleavage and mapping assay. Here, we describe a detailed protocol for site-specific DNA footprinting using 4-azidophenacyl bromide (APB). This method provides a straightforward, biochemical method to reveal the DNA binding orientation of <em>Sso</em>MCM helicase and can be applied to other DNA binding proteins.</p>
            <p><strong>Keywords:</strong> MCM helicase, DNA replication, Site-specific footprinting, Orientation, DNA translocation, DNA mapping</p>
            <h2>Background</h2><p>DNA replication is the process in which the duplex genomic strands separate into two template strands, the leading and lagging strands. This function is executed by a ring-shaped hexameric helicase in all Domains of life. Like other ring-shaped hexameric helicases, MCM consists of two domains; an N-terminal domain (NTD) and a C-terminal domain (CTD). In theory, either of these domains could be oriented towards the replication fork during translocation and be consistent with the known 3â²-5â² translocation directionality. The MCM helicase loads onto DNA origins as a double hexamer with NTDs facing each other. The orientation of the helicase during translocation determines whether the two hexamers dissociate away from each other or bypass one another during active unwinding. Our recent paper shows that the <em>Saccharolobus solfataricus</em> (<em>Sso</em>MCM) unwinds DNA with NTD leading the way (Perera and Trakselis, 2019).<br/><br/>In order to directly determine the translocation orientation, we utilized a combination of site-specific DNA footprinting, single turnover unwinding, and translocation assays. Here, we provide detailed protocols of site-specific DNA footprinting assays with 4-azidophenacyl bromide (APB) to analyze the translocation orientation of <em>Sso</em>MCM (Pendergrast <em>et al.</em>, 1992, Kassabov and Bartholomew, 2004, Nodelman <em>et al.</em>, 2017).<br/><br/>APB is a heterobifunctional photoactivatable crosslinking agent. Its bromide functional group reacts by S-alkylation with reduced thiols (<em>i.e.</em>, cysteines) to form stable thioether products (Figure 1). After binding of the functionalized protein to DNA and then exposure to UV light, a reactive singlet nitrene forms that can crosslink to either protein or DNA (in close proximity) through multiple insertion or addition mechanisms (Figure 1). The resulting crosslinked protein-DNA complex can be cleaved at the crosslinked nucleotide(s) under induced alkali/heat treatment. The lengths of the resulting DNA fragments can be used to determine the orientation distribution of the <em>Sso</em>MCM helicase on DNA. This is a straightforward biochemical method that reveals unique positions of <em>Sso</em>MCM helicases on DNA. It has an important advantage over traditional footprinting in that it can determine protein binding orientation on DNA instead of just binding site size.<br/><br/>Useful applications of this method can be employed at instances where the 3D structure of the protein is known or can be predicted, but the 3D structure of the protein-DNA complex is unknown (Pendergrast <em>et al.</em>, 1992). It is especially useful in determining the orientation of proteins that translocate along DNA or for those that bind to specific DNA sequences. Alternatively, the orientation of DNA-binding proteins can also be determined by other biochemical methods that employ localized hydroxyl radical Fenton footprinting reactions utilizing 1-(p-Bromoacetamidobenzyl) ethylenediamine N,N,N (Fe-BABE), similarly (Owens <em>et al.</em>, 1998).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200615/20200615061733_8906.jpg" width="300"/><br/><strong>Figure 1. Conjugation of APB to free Cys on <em>Sso</em>MCM and UV induced crosslinking reaction mechanism to DNA</strong></p>
        </body>

<p class="pptt" id="biaoti34325">## Materials and Reagents</p><ol class="npwtwok"><li>Cover slips (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-546 </a> )</li><li>Glass Petri dish (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 3160100 </a> )</li><li>Kimwipe (Kimberly-Clark, catalog number: <a class="blackNoUnder" target="_blank"> 06-666 </a> )</li><li>p-Azidophenacyl bromide (APB) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 57018-46-9 </a> , storage: 4 Â°C or (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 50-520-767 </a> )</li><li>Dimethylformamide (DMF) (EMD Millipore Corporation, catalog number: <a class="blackNoUnder" target="_blank"> DX1730-6 </a> )</li><li>Tris Base (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 77-86-1 </a> )</li><li>Glacial acetic acid (Mallinckrodt Baker, Inc., catalog number: <a class="blackNoUnder" target="_blank"> UN 2789 </a> )</li><li>Sodium chloride (NaCl) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S271-3 </a> )</li><li>Sodium hydroxide (NaOH) (Mallinckrodt Baker, Inc., catalog number: 7708-10)</li><li>Hydrochloric acid (HCl) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A144-212 </a> )</li><li>Magnesium chloride (MgCl<sub>2</sub>) (Spectrum Chemical, catalog number: <a class="blackNoUnder" target="_blank"> M1035 </a> )</li><li>Glycerol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP-229-4 </a> )</li><li>Sodium acetate (NaOAc) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S210-500 </a> )</li><li>Potassium acetate (KOAc) (EM Science Industries, catalog number: <a class="blackNoUnder" target="_blank"> PX1330-1 </a> )</li><li>Magnesium acetate (MgOAc) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M-0631 </a> )</li><li>Ammonium acetate (NH<sub>4</sub>OAc) (Mallinckrodt Baker, Inc., catalog number: <a class="blackNoUnder" target="_blank"> 0596-01 </a> )</li><li>Bovine serum albumin (BSA) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP1600-100 </a> )</li><li>Dithiotheritol (DTT) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP172-5 </a> )</li><li>Ethylenediaminetetraacetic acid (EDTA) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E5134 </a> )</li><li>Sodium dodecyl sulfate (SDS) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 151-21-3 </a> )</li><li>Salmon sperm DNA (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 15632-011 </a> , storage temperature: -20 Â°C)</li><li>Ethanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A962-4 </a> )</li><li>Poly-L-lysine (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P4832 </a> )</li><li>Orange G dye (EMD Millipore Corporation, catalog number: <a class="blackNoUnder" target="_blank"> 312-12 </a> )</li><li><em>Sso</em>MCM protein<br/><em>Sso</em>MCM protein was purified as previously described (McGeoch <em>et al.</em>, 2005; Graham <em>et al.</em>, 2011). Any DNA binding protein with a suitable single cysteine residue can be utilized with this approach. It will help if that cysteine residue is solvent accessible from structural data. If a single cysteine is not available, then site-directed mutagenesis can be used to add a cysteine at the appropriate position to test.</li><li>Fluorescent (5â² or 3â²) labelled (Cy3 or Cy5) DNA (purchased from IDT (Coralville, IA) or Sigma-Aldrich (St. Louis, MO)) (Synthesis scaleâ0.025 Î¼mole, Purification: Desalt, Format: Dry).<br/><em>Note: PAGE purification of DNA can be performed depending on the purity of the DNA.</em> </li><li>DNA Sequences for the 3â-long arm fork substrates<br/><ol class="npwthree"><li>DNA 165-3 (5â² Cy3)<br/>5â² 3TCCCACCCAACCCGACCGGCATCTAGTCTGGTAGCGTGAGCGAACGGACC<br/></li><li>DNA 171<br/>5â² CTAACTGCGCCGGTCGGGTTGGGTGGGA</li></ol></li><li>1x complex buffer (CB) (see Recipes)</li><li>Annealing buffer (see Recipes)</li><li>Post irradiation buffer (see Recipes)</li><li>Glycerol loading buffer (see Recipes)</li><li>20% Native PAGE (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34326">### Equipment</p><ol class="npwtwok"><li>Pipettes</li><li>-80 Â°C freezer</li><li>Benchtop 3UV transilluminator (UVP, model: <a class="blackNoUnder" target="_blank"> LMS-20E </a> , catalog number: P/N 95-0220-01)</li><li><a class="blackNoUnder" target="_blank"> Typhoon FLA 9000 imager </a> (GE Healthsciences)</li><li>PCR Instrument (BioRad laboratories (Hercules, CA)</li><li>Benchtop microcentrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5424R </a> )</li><li>Mini-PROTEAN Tetra Vertical Electrophoresis Cell for Mini Precast Gels (Bio-Rad Laboratories)</li><li>LightSafe micro centrifuge tubes (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> Z688312 </a> )</li><li>Dark room (Recommended)<br/></li></ol><p class="pptt" id="biaoti34327">### Software</p><ol class="npwtwok"><li>Image Quant (v.5.0) (GE Healthsciences, (Pittsburgh, PA))</li></ol><p class="pptt" id="biaoti34328">### Procedure</p><ol class="npwone"><li><em>Sso</em>MCM Protein labelling with APB<ol class="npwtwo"><li>In the dark, dissolve APB in 100% DMF at a concentration of 40 mM.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Dark conditions can be provided by using LightSafe micro centrifuge tubes, covering Eppendorf tubes in foil or turning off the lights.</em><br/><ol class="npwfour"><li><em>Use LightSafe micro centrifuge tubes when preparing APB solutions.</em><br/></li><li><em>Use clear micro centrifuge tubes covered in foil in subsequent reactions for easy visualization of a pellet and/or to achieve complete discarding of a supernatant.</em> </li></ol></li><li><em>When measuring APB, quantities as small as ~2 mg can be measured under dark conditions.</em><br/></li><li><em>Use freshly resuspended APB for each new experiment. Discard the unused APB.</em> </li></ol></li><li>Dilute the APB to 4 mM in 20 mM Tris pH 7.5, 75 mM NaCl, 10% glycerol and 10% DMF.</li><li>Add the diluted APB to a sample of the protein (~10 ÂµM monomer) containing a single reduced cysteine (in 20 mM Tris [pH 7.5], 75 mM NaCl, 10% glycerol), to achieve a final concentration of 0.4 mM APB and 1% DMF.<br/><em>Note: Cysteines must be reduced before conjugation with APB, followed by gel filtration or dialysis to remove the reducing agent. Alternatively, tris(2-carboxyethyl)phosphine (TCEP) can be used without subsequent removal.</em> </li><li>Proceed labelling for 2-3 h at room temperature.<br/><br/></li></ol></li><li>Preparing silanized coversli<ol class="npwtwo"><li>Dilute poly-L-lysine solution with distilled water for 1:10.</li><li>Place the coverslips (~15) in a Petri dish.</li><li>Pour the solution (~20 ml) on the coverslips (make sure the coverslips sink in).</li><li>Leave it in the solution for 15-30 min.</li><li>Rinse the coverslips with water three times at Room Temperature (RT).</li><li>Air dry the coverslips placing a kimwipe on top of the Petri dish (avoid touching the coverslips).</li><li>Coverslips can be prepared ahead of time and can be stored at RT for several weeks.<br/><br/></li></ol></li><li>Preparing annealed fork DNA<ol class="npwtwo"><li>Prepare stock solutions (100 Î¼M) of each complementary DNA strand in water (DNA 165-3 and DNA 171).</li><li>Mix the DNA in equal 1:1 (5 Î¼M) concentration in water or annealing buffer.<br/><em>Note: EDTA (chelates divalent metal ions) and salt in the annealing buffer facilitate duplex stability. But annealing DNA in water has also yielded in a complete duplex.</em> </li><li>Heat at 95 Â°C for 5 min and cool at a rate of 1 Â°C/min to room temperature in a PCR instrument.<br/><br/></li></ol></li><li>Preparing the crosslinked protein-DNA complex<ol class="npwtwo"><li>Incubate APB labelled <em>Sso</em>MCM with fluorescent (Cy3) fork DNA (150 nM) for 10-20 min in 1x CB buffer in 50 Âµl volumes (maintaining stochiometric ~1:1 MCM<sub>6</sub>:DNA ratio).<br/><em>Note: Fluorescently labelled DNA is used here, but <sup>32</sup>P labelled DNA can also be utilized.</em> </li><li>Transfer the sample into silanized coverslips. Pipette 50 Âµl of the sample into the middle of the coverslip.</li><li>UV irradiate for 15 s (302 nm UV-B).</li><li>Transfer the sample into an Eppendorf tube.</li><li>Add 150 Âµl of post irradiation buffer.</li><li>Vortex at RT and place at 70 Â°C for 20 min.<br/><br/></li></ol></li><li>Separate the crosslinked protein-DNA complex<ol class="npwtwo"><li>Next, add 1 Âµl of 10 mg/ ml Salmon sperm DNA, 30 Âµl of 3.0 M NaOAc, 750 Âµl of ice cold 100% ethanol, vortex, and leave on ice for 1-2 h at -80 Â°C.</li><li>Spin down the samples in microfuge at 4 Â°C, 16,000 <em>x g</em> for 30 min.</li><li>Discard the supernatant and wash the pellet twice with ice cold 70% ethanol.<br/><em>Note: In most of the steps, the DNA pellets are invisible. Discard the supernatant completely and resuspend in the subsequent buffer.</em> </li><li>Spin down the samples in microfuge at 4 Â°C, 16,000 <em>x g</em> for 30 min.</li><li>Remove ethanol and air dry the pellets by inverting on bench for 1 h.</li><li>Resuspend the pellet in 100 Âµl: 20 mM NH<sub>4</sub>OAc, 2% SDS, 0.1 mM EDTA pH 8.0 by vortexing.</li><li>Spin down the samples in microfuge at room temperature, 16,000 <em>x g</em> for 10 min.</li><li>Transfer the supernatant into fresh tubes and place in heat block at 90 Â°C for 2 min.</li><li>Then, add 1 Âµl of 2 M NaOH, vortex briefly, and incubate at 90 Â°C for 20 min.</li><li>After incubation, pulse spin the samples, add 101 Âµl 20 mM Tris-HCl pH 8.0, 1 Âµl of 2 M HCl, 1 Âµl of 2 M MgCl<sub>2</sub>, 480 Âµl 100% ethanol, vortex, and place at -80 Â°C for 1-2 h.</li><li>Pellet the samples in microfuge at 4 Â°C at 16,000 <em>x g</em> for 30 min, wash two times with ice cold 70% ethanol, and air dry on bench for 1 h.</li><li>Resuspend the DNA pellet with 5 Âµl of 40% glycerol loading buffer containing Orange G dye for gel loading.</li><li>Electrophorese the samples on a 20% TBE-PAGE (native PAGE) in 1x TBE (Tris base, Boric acid, EDTA) at constant 30 mA for 45 min, and visualize on a Typhoon FLA 9000 imager (GE Healthsciences).<br/></li></ol></li></ol><p class="pptt" id="biaoti34329">### Data analysis</p><ol class="npwtwok"><li>Quantifying the density of DNA bands on the gel (Figure 2A) is performed by ImageQuant software.</li><li>Calculation of the footprinting percentages (Figure 2B) is performed by quantifying the relative density (minus background) for the labelled strand, divided at the midpoint on the ssDNA arm according to the following equation:<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="69" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200615/20200615064712_3120.png" width="650"/> </div><br/><br/><img alt="" class="layerphoto" height="195" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200615/20200615064753_1276.jpg" width="300"/><br/><strong>Figure 2. <em>Sso</em>MCM orientation mapping onto 3â²-(DNA171/165-3) long arm fork DNA substrate.</strong> A. APB orientation mapping of the 3â²-encircled strand labelled at the 5â² duplex end with Cy3 on a 3â²-long arm fork DNA substrate with a 20 base duplex. <em>Sso</em>MCM was labelled at C682 with APB specifically. B. Quantification of the relative amount of DNA cleavage for bases 20-35 or 36-50 from the 5â²-end indicate the relative orientation for placing the N @duplex (orange) or C@duplex (blue), respectively closer to the duplex junction. DNA markers (M) indicate 18, 50 bases and fork DNA. Error bars represent standard error from 3-5 independent experiments. <em>P</em>-value is defined as ** &lt; 0.01.<br/><br/></li><li>A standard two-tailed equal variance Studentâs <em>t</em>-test is used to determine significant differences of C@duplex versus N@duplex. <em>P</em>-values are reported for each experimental condition. Detailed explanation on data analysis can be found in the original research paper Figures 1C and 1F, 2B and 2D, 3C and 2E (Perera and Trakselis, 2019).<br/></li></ol><p class="pptt" id="biaoti34330">### Notes</p><ol class="npwtwok"><li>Control experiments are performed with APB-MCM-DNA in the absence of UV light and/or with DNA alone in the absence of UV and APB. After activation by UV light, the crosslinking of APB to DNA bases is generally non-specific, yet we detected significant crosslinking and subsequent ssDNA cleavage even in the absence of direct UV light.</li><li>Similarly, 1-(p-Bromoacetamidobenzyl) ethylenediamine N,N,N (Fe-BABE) (Dojindo, Rockville, Maryland) can also be used in orientation mapping studies (Figure 1B and 1E in Perera and Trakselis, 2019). FeBABE utilizes a localized hydroxyl radical Fenton footprinting reaction. Rapid reactions generating high yields, mild protein conjugation and cleavage reaction conditions, non-sequence specific DNA cleavage are some advantages of utilizing FeBABE (Ishihama, 2000).<br/></li></ol><p class="pptt" id="biaoti34331">### Recipes</p><ol class="npwtwok"><li>1x CB (complex buffer)<br/>20 mM TrisOAc<br/>25 mM KOAc<br/>10 mM MgOAc<br/>0.1 mg/ml BSA<br/>1 mM DTT</li><li>Annealing buffer<br/>10 mM Tris pH 7.5<br/>50 mM NaCl<br/>1 mM EDTA</li><li>Post irradiation buffer<br/>20 mM Tris-HCl pH 8.0<br/>0.1% SDS<br/>50 mM NaCl</li><li>Glycerol loading buffer<br/>40% glycerol<br/>10 mM EDTA (pH 8.0)<br/>7.5% Orange G dye</li><li>20% Native PAGE<br/>1x TBE<br/>20% acrylamide<br/>0.05% APS<br/>4 Î¼l TEMED<br/></li></ol><p class="pptt" id="biaoti34332">### Acknowledgments</p><p class="ppzz">This work was supported by the National Science Foundation Division of Molecular and Cellular Biosciences [NSF1613534 to M.A.T.] and supported by Baylor University. We thank Gregory Bowman for introducing us to the feasibility of this technique for orientation mapping (Nodelman <em>et al.</em>, 2017).</p><p class="pptt" id="biaoti34333">### Competing interests</p><p class="ppzz">The authors declare that they have no conflicts of interest with the contents of this article.</p><p class="pptt" id="biaoti34334">### References</p><ol class="npwtwok"><li>Graham, B. W., Schauer, G. D., Leuba, S. H. and Trakselis, M. A. (2011). <a href="https://academic.oup.com/nar/article/39/15/6585/1023709" target="_blank">Steric exclusion and wrapping of the excluded DNA strand occurs along discrete external binding paths during MCM helicase unwinding.</a> <em>Nucleic Acids Res</em> 39: 6585-6595.</li><li>Ishihama, A. (2000). <a href="https://pubs.rsc.org/en/content/articlelanding/2000/cc/a908392h/unauth#!divAbstract" target="_blank">Molecular anatomy of RNA polymerase using protein-conjugated metal probes with nuclease and protease activities.</a> <em>Chem Commun</em> 13: 1091-1094. </li><li>Kassabov, S. R. and Bartholomew, B. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14870668" target="_blank">Site-directed histone-DNA contact mapping for analysis of nucleosome dynamics.</a> <em>Methods Enzymol</em> 375: 193-210.</li><li>McGeoch, A. T., Trakselis, M. A., Laskey, R. A. and Bell, S. D. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16116441" target="_blank">Organization of the archaeal MCM complex on DNA and implications for the helicase mechanism.</a> <em>Nat Struct Mol Biol</em> 12:756-762.</li><li>Nodelman, I. M., Bleichert, F., Patel, A., Ren, R., Horvath, K. C., Berger, J. M. and Bowman, G. D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28111016" target="_blank">Interdomain communication of the Chd1 chromatin remodeler across the DNA Gyres of the nucleosome.</a> <em>Mol Cell</em> 65(3): 447-459 e446.</li><li>Owens, J. T., Miyake, R., Murakami, K., Chmura, A. J., Fujita, N., Ishihama, A. and Meares, C. F. (1998). <a href="https://www.pnas.org/content/95/11/6021.long" target="_blank">Mapping the Ï<sup>70</sup> subunit contact sites on <em>Escherichia coli</em> RNA polymerase with a Ï<sup>70</sup>-conjugated chemical protease.</a> <em>PNAS</em> 95(11): 6021-6026.</li><li>Pendergrast, P. S., Chen, Y., Ebright, Y. W. and Ebright, R. H. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1332042" target="_blank">Determination of the orientation of a DNA binding motif in a protein-DNA complex by photocrosslinking.</a> <em>Proc Natl Acad Sci U S A</em> 89(21): 10287-10291.</li><li>Perera, H. M. and Trakselis, M. A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31661075" target="_blank">Amidst multiple binding orientations on fork DNA, Saccharolobus MCM helicase proceeds N-first for unwinding.</a> <em>Elife</em> 8: e46096.<br/></li></ol>