<body>
            <h1>Live Cell Measurement of the Intracellular pH of Yeast by Flow Cytometry Using a Genetically-Encoded Fluorescent Reporter</h1>
            <p><strong>Authors:</strong> Catherine G. Triandafillou, D. Allan Drummond</p>
            <h2>Abstract</h2>
            <p>The intracellular pH of yeast is a tightly regulated physiological cue that changes in response to growth state and environmental conditions. Fluorescent reporters, which have altered fluorescence in response to local pH changes, can be used to measure intracellular pH. While microscopy is often used to make such measurements, it is relatively low-throughput such that collecting enough data to fully characterize populations of cells is challenging. Flow cytometry avoids this drawback, and is a powerful tool that allows for rapid, high-throughput measurement of fluorescent readouts in individual cells. When combined with pH-sensitive fluorescent reporters, it can be used to characterize the intracellular pH of large populations of cells at the single-cell level. We adapted microscopy and flow-cytometry based methods to measure the intracellular pH of yeast. Cells can be grown under near-native conditions up until the point of measurement, and the protocol can be adapted to single-point or dynamic (time-resolved) measurements during changing environmental conditions.</p>
            <p><strong>Keywords:</strong> Flow cytometry, Yeast, Intracellular pH, pHluorin, Biosensors</p>
            <h2>Background</h2><p>The intracellular pH of yeast is correlated with characteristics like viability and growth rate, and the regulation of intracellular pH consumes a large proportion of cellular energetic resources (Orij <em>et al.</em>, 2011). However, intracellular pH can change rapidly and is highly environmentally sensitive, so it is crucial to have a fast, minimally perturbative method of measurement for this important aspect of cell physiology.<br/><br/>Genetically-encoded biosensors that convert local concentrations of a compound of interest into fluorescent readouts have revolutionized our ability to characterize the intracellular environment. For some sensors, the absolute fluorescence intensity is correlated with the readout. This can be a problem when performing in-cell measurements, since the fluorescence depends both on the sensor expression level, which varies cell to cell, and the characteristic of interest. Ratiometric sensors, which depend on the ratio of fluorescence in two different parts of the spectrum in the same fluorophore, do not suffer from this drawback. One such sensor, pHluorin (MiesenbÃ¶ck <em>et al.</em>, 1998), is a pH-sensitive fluorescent biosensor based on GFP that can be used to measure intracellular pH; the emission intensity (measured around 520 nm) after excitation in the near-UV and blue (405 and 488 nm typically) varies with pH such that the ratio of emission intensities can be related to pH.<br/><br/>In this protocol, we outline how to use pHluorin to measure intracellular pH in living budding yeast cells using flow cytometry. Flow cytometry is a method that combines microfluidic focusing and optical interrogation to measure the fluorescence of single cells in liquid culture with little to no special sample preparation required. Others have also used this method to measure intracellular pH in yeast cells (Weigert <em>et al.</em>, 2009; Valkonen <em>et al.</em>, 2013). The advantage of flow cytometry is that it is much higher throughput than microscopy-based methods (Bagar <em>et al.</em>, 2009; Orij <em>et al.</em>, 2009), while still being able to characterize individual cells and measure the fluorescence of multiple fluorophores. This access to both single-cell measurements and enough data to generate population-level statistics with a great deal of confidence is highly valuable. One potential downside to using flow cytometry to analyze the fluorescence of biosensors is that the spatial distribution of the fluorophore within the cell is not accessible (at least with traditional flow cytometry), as only a single, average fluorescence value in each channel is reported for each event (cell). However, for intracellular pH measurements in particular, the variation in any compartment (here, the cytosol, although pHluorin can be targeted to other organelles) (Orij <em>et al.</em>, 2009) is expected to be minimal due to the unique properties of proton exchange in buffered aqueous solutions such as the cellular interior (Boron, 2004).<br/><br/>The outline of a typical experiment is illustrated in Figure 1. Cells expressing pHluorin are suspended in buffer of known pH and an ionophore, in this case nigericin, is added. This addition makes the cell membrane permeable to protons and thus equilibrates the intracellular and extracellular pH. The ratiometric fluorescence of pHluorin is measured for these cells with known intracellular pH, and then these data are used to construct a calibration curve that can be used to convert measured fluorescence ratios in other cells to real intracellular pH values.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616095437_3587.jpg" width="300"/><br/><strong>Figure 1. Flowchart of Protocol.</strong> Experimental steps are colored blue, analysis steps are in green. Optional steps are in gray text.</p>
        </body>

<p class="pptt" id="biaoti34346">## Materials and Reagents</p><ol class="npwtwok"><li>50 ml conical flasks (Olympus Plastics, catalog number: <a class="blackNoUnder" target="_blank"> 28-108 </a> )</li><li>0.22 Î¼m filter (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> CLS 431118 </a> )</li><li>5 ml polystyrene round-bottom tubes, 12 x 75 mm style (Corning Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 358058 </a> )</li><li>Strain of <em>Saccharomyces cerevisiae</em> from the S288C background, such as BY4741/2/3<br/>Alternatively, another strain of yeast may be used, as long as both non-fluorescent and pHluorin-expressing strains are available or can be made by the investigator (see optional reagents below). More information on yeast strains and genotypes can be found at <a href="https://wiki.yeastgenome.org/index.php/Commonly_used_strains" target="_blank">https://wiki.yeastgenome.org/index.php/Commonly_used_strains</a>.</li><li>2-Deoxy-D-glucose (Sigma, Sigma Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> D8375-5G </a> )</li><li>Nigericin (Adipogen, Adipogen Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> AG-CN2-0020 </a> ), prepared as a 10 mM stock in 100% ethanol and stored at -20 Â°C</li><li>Yeast Nitrogen Base (YNB) + Nitrogen (Sunrise Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1501-250 </a> )</li><li>Synthetic Complete (SC) dropout mix (Sunrise Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1300-030 </a> )</li><li>D-(+)-Glucose (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> G32040-5000.0 </a> )</li><li>MES (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP300-100 </a> )</li><li>HEPES (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> H-400-1 </a> )</li><li>KCl (potassium chloride) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> P217-3 </a> )</li><li>NaCl (sodium chloride) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP358-212 </a> )</li><li>2 M HCl (hydrochloric acid) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> A144-212 </a> )</li><li>2 M KOH (potassium hydroxide) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 484016-1KG </a> )</li><li>Ammonium acetate (Sigma, catalog number A1542-250G)</li><li>2X Calibration Curve Buffer (see Recipes)</li><li>Yeast growth media (see Recipes)<br/><br/></li></ol><p class="ppzzs">Optional reagents</p><ol class="npwtwok"><li>pCGT05 or pHluorin expression vector (see Notes and attached plasmid map for more details; if the strain expressing pHluorin does not already exist and must be made by the investigator, then this reagent is not optional)</li><li>PmeI (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0560S </a> ; comes with CutSmart<sup>Â®</sup> Buffer)</li><li>Salmon testes DNA (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D1626-250MG </a> ) prepared as a 2 mg/ml solution in TE Buffer (see Recipes)</li><li>Molecular Biology grade agarose (Apex Bioresearch Products, catalog number: <a class="blackNoUnder" target="_blank"> 20-102GP </a> )</li><li>10x TBE Buffer (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 161-0733 </a> ), diluted to 1x in ultrapure water</li><li>SCD (2%) plates without leucine, for selection</li><li>Tris-HCl (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP153-1 </a> )</li><li>Tetrasodium EDTA (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> S311-100 </a> ) prepared as a 1 M pH 8.0 stock</li><li>Lithium Acetate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L6883-1KG </a> )</li><li>Polyethylene glycol (PEG) 3350 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P4338-500G </a> )<br/></li></ol><p class="pptt" id="biaoti34347">### Equipment</p><ol class="npwtwok"><li>Microcentrifuge capable of spinning 1.5 ml tubes at 3,000 <em>x g</em> </li><li>BD Biosciences LSR Fortessa flow cytometer (see note for instrument settings), or any flow cytometer capable of exciting between 380 and 410 nm and between 470-490 nm, and measuring emission (for both excitations) between 500 and 550 nm (see Notes). Other laser configurations could also be used; see MiesenbÃ¶ck <em>et al.</em> 1998 for full characterization of pHluorin</li><li>Electronic pH meter such as the Mettler Toledo SevenCompact pH meter S220 (Mettler Toledo, model: <a class="blackNoUnder" target="_blank"> 30019032 </a> )</li><li>(Optional) For yeast transformation<br/><ol class="npwthree"><li>Heat block/dry bath/water bath<br/></li><li>Mold for casting agarose gels<br/></li><li>Standard electrophoresis power supply and setup for running agarose gels</li></ol></li></ol><p class="pptt" id="biaoti34348">### Software</p><ol class="npwtwok"><li>Software that can read FCS files. In this protocol we use R (CRAN, <a href="https://www.r-project.org/" target="_blank">https://www.r-project.org/</a>) with the flowCore package (Bioconductor, <a href="https://bioconductor.org/packages/release/bioc/html/flowCore.html" target="_blank">https://bioconductor.org/packages/release/bioc/html/flowCore.html</a>); see example script for data analysis pipeline</li></ol><p class="pptt" id="biaoti34349">### Procedure</p><ol class="npwone"><li>(Optional) Create pHluorin-expressing yeast strain<ol class="npwtwo"><li>To integrate at the <em>leu2</em> locus (S288C backgrounds), cut vector pCGT05 with PmeI by mixing the following reagents and incubating at 37 Â°C for 60 min.<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:400px;"><tbody><tr><td><strong>Reagent (Concentration)</strong><br/></td><td><strong>Amount (Î¼l)</strong><br/></td></tr><tr><td>pCGT05 (~200 ng/Î¼l)<br/></td><td>5</td></tr><tr><td>Cutsmart Buffer (5x)<br/></td><td>5</td></tr><tr><td>PmeI (10,000 units/ml)<br/></td><td>1</td></tr><tr><td>Ultrapure water<br/></td><td>39</td></tr></tbody></table><ol class="npwthree"><li>Alternatively, a plasmid with a selectable marker may be used rather than integration; see Notes. Use this as a template in the transformation in Step A3 and plate on appropriate selection media.<br/></li><li>If the <em>leu2</em> locus is not available in the desired strain, integration at another locus is also possible.</li></ol></li><li>(Optional) Check digestion efficacy<br/>To check digestion, run 2-5 Î¼l of product on a 1% agarose gel, along with uncut vector and a ladder. The cut product should run at two bands, one at 5.2 kb and the other at 3.5 kb; see Figure 2.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103323_4370.jpg" width="100"/><br/><strong>Figure 2. Vector pCGT05 before and after cutting with PmeI.</strong> The first lane is a ladder for size reference, the second lane is the vector without incubation with the restriction enzyme PmeI, the third lane is the vector after incubation with PmeI (there are two cut sites such that enzymatic digestion yields two fragments).<br/><br/></li><li>Transform into desired strain of yeast<br/><ol class="npwthree"><li>To integrate into the <em>leu2</em> locus, follow a standard lithium acetate yeast transformation protocol (Gietz and Schiestl, 1994) using the product of the above digestion reaction as the DNA template.<br/></li><li>Screen colonies on SC plates without leucine (30 Â°C, allow 2-3 days for growth), and further colony purify on selection plates followed by non-selective plates to ensure that no wild-type cells remain.<br/></li><li>Cells in media may be mixed with glycerol to form a 20% final solution and stored at -80 Â°C for years.<br/><br/></li></ol></li></ol></li><li>Grow cells (night before experiment)<ol class="npwtwo"><li>Inoculate starter cultures (at least 5 ml) of each strain of yeast youâd like to measure; include wild-type yeast not expressing pHluorin (for background subtraction) and a sufficient volume of pHluorin-expressing cells to both generate experimental samples and make a calibration curve to convert fluorescence to pH (see below for volume estimates).<br/><ol class="npwthree"><li>Aim for cells to be at around OD<sub>600</sub> 0.05-0.1 by the following morning (about 16 h later) so that they can either be analyzed right away, or mildly diluted in order to be analyzed later the same day.<br/></li><li>The calibration curve will require approximately 5 ml of pHluorin-expressing yeast at OD<sub>600</sub> 0.05-0.2.<br/></li><li>The target OD<sub>600</sub> for experimental samples is 0.05 to 0.1, lower than what is typically used for microscopy or biochemistry-based experiments. Calibration curve samples can be slightly higher, up to OD<sub>600</sub> 0.2.</li></ol></li><li>Prepare 2x Calibration Curve Buffer. It can be prepared ahead of time and stored at 4 Â°C.<br/><br/></li></ol></li><li>Prepare calibration curve (morning of experiment)<ol class="npwtwo"><li>Add 5 ml of 2x Calibration curve buffer to eight 50 ml conical tubes. Add 3 ml of ultrapure water (or sufficient volume such that the electrode of the pH meter can be fully submerged in the liquid, keeping in mind that variable amounts of acid and base will need to be added to adjust the pH).</li><li>Using concentrated HCl and KOH, adjust the pH of each tube to half pH units ranging from 5.0 to 8.5 (8 total; this spans the range of pH values to which the fluorophore is sensitive; measurement outside this range is not accurate). The initial pH should be in the range of 5.5-6.5.<br/><em>Note: Make sure that the buffer is at room temperature before adjusting pH, as the pH can be temperature-sensitive.</em> </li><li>Once the pH is in the correct range, add ultrapure water until the final volume is 10 ml (calibration curve buffer is now at 1x). Note the final pH and record the exact value on the outside of the tube.<br/>pH-adjusted 1x calibration curve buffer (without ionophore) can be stored at 4 Â°C for around a month. When re-using, bring to room temperature, re-measure the pH, and note any changes on the outside of the tube. Small changes (~0.05 pH units) are tolerable, but large changes indicate that the buffer should be discarded and re-made from the 2x stock.<br/><br/></li></ol></li><li>Prepare the calibration curve (~30 min before experiment)<ol class="npwtwo"><li>Aliquot 1 ml of calibration curve buffer at each pH into labeled 1.5 ml tubes and to each tube add 1 Î¼l of 10 mM nigericin in 100% ethanol to a final concentration of 10 Î¼M. This step should be done as close to the time of measurement as possible, as the ionophore degrades more rapidly at room temperature.</li><li>Label additional 1.5 ml tubes for each pH that will be measured in the calibration curve (for cells), plus one additional tube for unlabeled cells.</li><li>Once cells are between OD<sub>600</sub> 0.05 and 0.2, aliquot ~500 Î¼l of pHluorin-expressing cells into each labeled tube. Aliquot one additional tube of unlabeled cells, which will be used for background subtraction.</li><li>Spin cells at 3,000 <em>x g</em> for 3 min. Decant supernatant (media) with a pipette, being careful to remove as much of the liquid as possible without disturbing the cell pellet.</li><li>Briefly wash cells by resuspending in 100 Î¼l of deionized water and spinning at 3,000 <em>x g</em> for 3 min. Decant supernatant.</li><li>Resuspend cells in 400 Î¼l of the appropriate pH prepared calibration curve buffer with nigericin.<br/><ol class="npwthree"><li>If many cells were lost during decanting (for example if the pellet was disturbed by pipetting) or if the concentration is low to begin with, a smaller volume of buffer may be used, but at least 200 Î¼l of volume is needed to make the measurement (this may vary with the model of cytometer used to make the measurements).<br/></li><li>The unlabeled cells may be resuspended in any pH, as the fluorescence background does not vary with the pH of the buffer.</li></ol></li><li>Allow cells to incubate in buffer for 15 min to equilibrate pH. After this, proceed to Procedure E as rapidly as possible. It may be possible to hold the cells in buffer for longer without loss of integrity or a change in the fluorescence values, but this should be tested empirically by running full curves multiple times at different time intervals.<br/><br/></li></ol></li><li>Analyze samples on the flow cytometer<ol class="npwtwo"><li>Briefly vortex tube or mix by inversion before loading each sample.</li><li>Add 200 Î¼l of wild type (non-pHluorin-expressing) cells, in media, into a 5 ml polystyrene flow tube. Place the tube on the flow cytometer and begin acquiring data. While running the sample on the lowest possible flow rate, isolate the events associated with cells by drawing a gate in the forward scatter vs. side scatter plot to collect the vast majority of cell events, see Figure 3 for typical scattering profile.<br/><ol class="npwthree"><li>Collect at least 10,000, but preferably 20,000+ events for each sample.<br/></li><li>Voltage settings for a BD LSR Fortessa instrument are given in Table 1. Other instruments and settings can be used; see Notes for more details.<br/></li><li><strong>It is essential that the same voltages are used within the same experiment</strong>, and preferably should not be changed even between experiments. Data taken with different voltage settings cannot be directly compared.<br/><br/><img alt="" class="layerphoto" height="247" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103349_2811.jpg" width="300"/><br/><strong>Figure 3. Typical scattering profile of <em>Saccharomyces cerevisiae</em> cells.</strong> Data from a BD Biosciences LSR Fortessa with voltages as listed in Table 1.<br/><br/><strong>Table 1. Instrument Settings </strong>(BD LSR Fortessa)<br/><div style="white-space:nowrap;"><img alt="" class="layerphoto" height="206" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103409_4695.png" width="600"/><br/></div><br/></li></ol></li><li>Using the same settings and volumes, run and collect data for pHluorin-expressing cells in media, and wild type cells in calibration curve buffer. All of these samples are essential for background subtraction.</li><li>Using the same settings and volumes, run and collect data for all the samples in calibration curve buffer prepared in Procedure C.</li><li>Run any additional experimental samples using the same settings.<br/><em>Note: In the case of experimental samples in different medium, a separate sample of unlabeled cells in the corresponding medium must also be analyzed in order to do proper background subtraction; see Data analysis section below.</em><br/><br/></li></ol></li><li>Export Data<br/>Export all data. The standard format for flow cytometry data is the .fcs format; data analysis below assumes data in this format.<br/></li></ol><p class="pptt" id="biaoti34350">### Data analysis</p><p class="ppzz">The goal of the analysis is to first use data from cells in buffer to construct a calibration curve, which can then be used to convert the fluorescence readings of cells in media to intracellular pH. In the example script, the two green channels used are called âFITCâ, corresponding to excitation at 488 nm and emission at 525/50 nm, and âBV510â, corresponding to excitation at 405 nm and emission at 525/50 nm. Below, they will be referred to below as F<sub>405</sub> for emission intensity at 525 after excitation at 405, and F<sub>488</sub> for emission intensity at 525 after excitation at 488. Superscripts will refer to the strain and background.<br/>  The following is a general description of the data analysis pipeline. An example script executing this analysis using the R programming language is attached.</p><ol class="npwone"><li>Load the data, contained in an FCS file, and convert it to human-readable data in table format<br/>In the attached example script, this is accomplished by reading the file using the flowCore package for the R programming language, and then converting the resulting object into an R data frame.<br/><br/></li><li>Calculate background readings<br/>The median fluorescence of unlabeled cells (in both the FITC and the BV510 channels) will be subtracted as background. Unlabeled cells in buffer will be used to subtract background from labeled cells in buffer, and likewise for samples consisting of cells in media.<br/><br/></li><li>Make calibration curve<br/><ol class="npwtwo"><li>For each cell (event), calculate the background-subtracted ratio of fluorescence that varies with pH.<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="69" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103453_1880.png" style="font-size:10pt;" width="300"/> </div><br/></li><li>Bin/group events from each calibration curve sample, corresponding to each pH buffer. Calculate the median fluorescence ratio for each pH. It may be helpful to establish a fluorescence cutoff in order to account for any low, non-expressing, or damaged cells. However, this protocol should retain the majority of the events, and may not be necessary for strains with pHluorin genomically integrated.</li><li>To make the calibration curve, first plot the known pH of each buffer against the median R<sub>405/488</sub> value, see Figure 4 for an example calibration curve.<br/><br/><img alt="" class="layerphoto" height="226" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103528_7360.jpg" width="300"/><br/><strong>Figure 4. pHluorin calibration curve.</strong> Points are medians of a population of at least 10,000 cells, error bars are the 25th and 75th percentile.<br/><br/></li><li>Fit the points to a sigmoid of the form:<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="53" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103552_7988.png" style="font-size:10pt;" width="300"/> </div><br/>where a, b, c, and d are fitting parameters (<em>a</em> corresponds to the max of the curve, <em>b</em> to the steepness of the rise, <em>c</em> to the midpoint, and <em>d</em> to the baseline). In the example script this is accomplished using the nonlinear least squares method implemented by the nls() function in the R programming language; see documentation at <a href="https://stat.ethz.ch/R-manual/R-devel/library/stats/html/nls.html" target="_blank">https://stat.ethz.ch/R-manual/R-devel/library/stats/html/nls.html</a> for more information on this method, and Figure 4 for an example calibration curve. Once these parameters have been fit, this function can be used to map between the fluorescence ratio for any individual cell and intracellular pH by calculating the background subtracted R<sub>405/488</sub> value, and rearranging the above equation as follows:<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="78" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103701_8119.png" style="font-size:10pt;" width="270"/> </div><br/><em>Note: This function can only be used in the range of sensitivity of the fluorophore, pH 5.0 to 8.5 (MiesenbÃ¶ck </em><em>et al.</em><em>, 1998). In general, a calibration curve must extend to these limits in order to set the baselines correctly, and only ratios that fall within the minimum and maximum pHs used in the calibration curve can be confidently assigned to a pH. Ratios that fall outside this range are ambiguous and should not be analyzed.</em><br/><br/></li></ol></li><li>Analyze experimental samples<ol class="npwtwo"><li>Subtract background from each experimental sample by first calculating the median fluorescence of unlabeled cells in the same medium as the sample, then calculating the R<sub>405/488</sub> as in step C1 above (substituting F<sub>405</sub> and F<sub>488</sub> values for unlabeled cells in medium rather than buffer).</li><li>Using the relationship established in step C4, convert R<sub>405/488</sub> to pH.<br/></li></ol></li></ol><p class="pptt" id="biaoti34351">### Notes</p><ol class="npwtwok"><li><em>On yeast strains and strain availability:</em> In the attached example script, data collected from strain yCGT028, a diploid (BY4743 background) yeast strain expressing pHluorin from the <em>leu2</em> locus, which is empty (<em>leu2Î0</em>) in the parent strain, and <em>SSA4::mCherry</em>. However, any strain expressing pHluorin, either from the genome or from a plasmid maintained with a selectable marker could be used in this protocol. Because the probe is ratiometric, the expression level variation expected for plasmid expression should not significantly affect the data analysis, however care should be taken that any cells that fail to express the fluorophore are not included; because expressing and non-expressing cells are easily distinguished (see Figure 5), non-expressing cells can be manually gated out on the instrument, or excluded during data analysis.<br/><br/><img alt="" class="layerphoto" height="190" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103821_0316.jpg" width="300"/><br/><strong>Figure 5. Separation of pHluorin-expressing and wild type populations by fluorescence.</strong> Data from a BD Biosciences LSR Fortessa with voltages as listed in Table 1.<br/><br/></li><li><em>On model of flow cytometer:</em> In this protocol we give specific instructions for running samples on a BD Biosciences LSR Fortessa; but any cytometer capable of measuring excitation and emission in the correct range: excitation between 380 and 410 nm and between 470 and 490 nm and emission (for both excitations) between 500 and 550 nm. To set the voltages for these channels, it is helpful to run both cells in pH 5 and pH 8.5 calibration curve buffer, as these represent the maximum and minimum fluorescence intensities expected (MiesenbÃ¶ck <em>et al.</em>, 1998). Both samples should be well-resolved in both channels (<em>i.e.</em>, not saturating the detector at zero or the max value).</li><li><em>On fluorescence data:</em> The data exported from the flow cytometer may contain pulse height and width information as well as the total fluorescence (area). Area measurements are used in all aspects of this protocol.</li><li><em>On compensation:</em> Although compensation (a procedure used to calculate and correct for the overlap between fluorescence channels) is common in multicolor flow cytometry, in this protocol only one fluorophore is being analyzed and it is not necessary to perform compensation. pHluorin may be co-expressed with other fluorophores, such as mCherry, that are not excited in the same range, and analyzed in the same fashion.</li><li><em>On frequency of measuring calibration curve:</em> Because instrument settings and mechanics can vary day-to-day, it is best practice to measure a new calibration curve every time an experiment is performed. If ratio values seem to be very consistent day-to-day, then only the minimum and maximum pH buffers can be analyzed; if the previously generated calibration curve correctly predicts these samples, then that calibration curve can be used to analyze data taken that day.</li><li><em>On reproducibility of calibration curves:</em> Although the absolute value of the ratios may change on different days, the midpoint of the curve, which is the apparent pKa of pHluorin, should be consistent. To check whether a calibration curve accurately reproduces the known characteristics of the fluorophore, plot the pH of the buffer for each sample against the following quantity (Bagar <em>et al.</em>, 2009):<br/><br/><div style="text-align:center;"><img alt="" class="layerphoto" height="78" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200616/20200616103855_8459.png" width="230"/> </div><br/>This plot can be fit by linear least squares, and where the resulting line crosses the y = 0 line is the apparent pKa of the fluorophore. The published <em>in vitro</em> pKa is 6.9 (Bagar <em>et al.</em>, 2009). See the attached script for an example of this analysis.</li><li><em>On filtering events:</em> In some samples, depending on handling, there is a subpopulation of cells that both lose fluorescence in the pHluorin-associated channels, but also are much more (auto) fluorescent in another channel (ex 405, em 450/50). We interpret these cells as being dead or damaged in some way, and exclude them from analysis. They generally appear in experimental samples and not in calibration curve samples. See analysis script for an example of isolating and filtering out this subpopulation.<br/></li></ol><p class="pptt" id="biaoti34352">### Recipes</p><ol class="npwtwok"><li>2x Calibration Curve Buffer<br/>100 mM MES<br/>100 mM HEPES<br/>100 mM KCl<br/>100 mM NaCl<br/>400 mM ammonium acetate<br/>20 mM 2-Deoxy-D-glucose</li><li>TE Buffer<br/>10 mM Tris-HCl<br/>1 mM Na<sub>2</sub>EDTA, pH 8.0</li><li>Yeast growth media (1 L)<br/>6.7 g YNB<br/>2.0 g SC Dropout mix<br/>20 g Glucose<br/>Bring to 1 L with ultrapure water and filter through a 0.22 Î¼m filter to sterilize<br/></li></ol><p class="pptt" id="biaoti34353">### Acknowledgments</p><p class="ppzz">Research reported in this publication was supported by the National Institute of Biomedical Imaging And Bioengineering of the National Institutes of Health (NIH) under Award Number T32EB009412, and the National Science Foundation Graduate Research Fellowship under Grant No. DGE-1144082. DAD acknowledges support from the NIH, award numbers R01 GM126547 and R01GM127406, and from the US Army Research Office, award number W911NF-14-1-0411. The authors would also like to thank the University of Chicago Flow Cytometry Core.<br/>  Methodology reported in this paper was used in the preprint âTransient intracellular acidification regulates the core transcriptional heat shock responseâ, <a href="https://doi.org/10.1101/414706" target="_blank">https://doi.org/10.1101/414706</a>.</p><p class="pptt" id="biaoti34354">### Competing interests</p><p class="ppzz">The authors declare they do not have any competing interests.</p><p class="pptt" id="biaoti34355">### References</p><ol class="npwtwok"><li>Bagar, T., Altenbach, K., Read, N. D. and BenÄina, M. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19286983" target="_blank">Live-Cell imaging and measurement of intracellular pH in filamentous fungi using a genetically encoded ratiometric probe.</a> <em>Eukaryot Cell</em> 8(5): 703-712.</li><li>Boron, W. F. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15545345" target="_blank">Regulation of intracellular pH.</a> <em>Adv Physiol Educ</em> 28(1-4): 160-179.</li><li>Gietz R. D. and Schiestl R. (1994). <a href="https://www.researchgate.net/publication/284689164_Transforming_yeast_with_DNA" target="_blank">Transforming yeast with DNA.</a> <em>Methods Mol Cell Biol</em> 5(5): 255-269.</li><li>MiesenbÃ¶ck, G., De Angelis, D. A. and Rothman, J. E. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9671304" target="_blank">Visualizing secretion and synaptic transmission with pH-sensitive green fluorescent proteins.</a> <em>Nature</em> 394(6689): 192-195.</li><li>Orij, R., Brul, S. and Smits, G. J. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21421024" target="_blank">Intracellular pH is a tightly controlled signal in yeast.</a> <em>Biochim Biophys Acta</em> 1810(10): 933-944.</li><li>Orij, R., Postmus, J., Ter Beek, A., Brul, S. and Smits, G. J. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19118367" target="_blank"><em>In vivo</em> measurement of cytosolic and mitochondrial pH using a pH-sensitive GFP derivative in <em>Saccharomyces cerevisiae</em> reveals a relation between intracellular pH and growth.</a> <em>Microbiology</em> 155(Pt 1): 268-278.</li><li>Valkonen, M., Mojzita, D., PenttilÃ¤, M. and Bencina, M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24038689" target="_blank">Noninvasive high-throughput single-cell analysis of the intracellular pH of <em>Saccharomyces cerevisiae</em> by ratiometric flow cytometry.</a> <em>Appl Environ Microbiol</em> 79(23): 7179-7187.</li><li>Weigert, C., Steffler, F., Kurz, T., Shellhammer, T. H. and Methner, F. J. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19581482" target="_blank">Application of a short intracellular pH method to flow cytometry for determining <em>Saccharomyces cerevisiae</em> vitality.</a> <em>Appl Environ Microbiol</em> 75(17): 5615-5620.<br/></li></ol>