<body>
            <h1>Quantitative Nucleocytoplasmic Transport Assays in Cellular Models of Neurodegeneration</h1>
            <p><strong>Authors:</strong> Joni Vanneste, Thomas Vercruysse, Philip Van Damme, Ludo Van Den Bosch, Dirk Daelemans</p>
            <h2>Abstract</h2>
            <p>Nucleocytoplasmic transport deficits are suggested to play a role in neurodegenerative disorders, including amyotrophic lateral sclerosis (ALS). Given the importance and complexity of this process, understanding when these aberrations occur and which pathways are involved is of great importance. Here, we make use of CRISPR-Cas9 technology to design cell lines stably expressing fluorophore proteins shuttling between the nucleus and cytoplasm by karyopherins of choice. To validate this protocol, we measured an ALS-associated nucleocytoplasmic transport pathway in the presence of the disease-associated peptide poly-PR. This technique allows measuring a particular active nucleocytoplasmic transport pathway in intact cells in a neurodegenerative disease-associated context. Moreover, these experiments can be performed without the need for expensive equipment and have the potential to be upscaled for high-throughput screening purposes.</p>
            <p><strong>Keywords:</strong> Neurodegenerative disease, Nucleocytoplasmic transport, Amyotrophic lateral sclerosis, C9orf72, Poly-PR, CRISPR-Cas9</p>
            <h2>Background</h2><p>Nucleocytoplasmic transport is crucial for cellular homeostasis and occurs across the nuclear membrane via large multi-protein complexes that form aqueous channels, called nucleopore complexes (NPCs) (Stoffler <em>et al.</em>, 1999; Ryan and Wente, 2000). Although these channels allow small proteins to passively equilibrate across the nuclear membrane (passive transport), most proteins appear to be actively transported by karyopherins, namely importins or exportins (active transport). Cargo proteins that contain a nuclear localization signal (NLS) are targeted by importins. Various receptor-mediated import pathways have been identified, but the best characterized pathway involves importin-Î²1/importin-Î± (KPNB1/KPNAx) of which the cargos contain a classical NLS (cNLS), exemplified by the SV40 large T antigen NLS (Lange <em>et al.</em>, 2007). Protein export from the nucleus into the cytoplasm is mediated by exportins. Several pathways of nuclear export have been identified. The most common type of nuclear export signal (NES) is a leucine-rich sequence motif recognized by exportin1 (XPO1), of which the NES from protein kinase inhibitor (PKI) is the most often used prototype (Ossareh-Nazari <em>et al.</em>, 2001).<br/> <br/>Altered subcellular distribution of proteins is a common characteristic among neurodegenerative disorders, such as Alzheimerâs disease (AD), Parkinsonâs disease (PD), Huntingtonâs disease (HD) and amyotrophic lateral sclerosis (ALS) (Fahrenkrog and Harel, 2018). Increasing evidence indicates impaired nucleocytoplasmic transport as a possible mechanism explaining this aberrant protein localization in the degenerating neurons. For example, in the majority of ALS patients the RNA binding protein TDP-43 (Arai <em>et al.</em>, 2006; Neumann <em>et al.</em>, 2006) or, to a lesser extent, FUS (Kwiatkowski <em>et al.</em>, 2009; Vance <em>et al.</em>, 2009; Tyzack <em>et al.</em>, 2019) becomes partially depleted from the nucleus and aggregates in the cytoplasm. Interestingly, the most common genetic cause associated with ALS, a G<sub>4</sub>C<sub>2</sub>-repeat expansion in the <em>C9orf72</em> gene (Majounie <em>et al.</em>, 2012), has been suggested to affect nucleocytoplasmic transport in various ways (Boeynaems <em>et al.</em>, 2016; Yuva-Aydemir <em>et al.</em>, 2018). One possible toxic pathway is the bidirectional translation of the repeat expansion by repeat-associated non-ATG (RAN) translation into dipeptide repeat proteins (DPRs), namely poly-GP, poly-GA, poly-GR, poly-PA and poly-PR (Ash <em>et al.</em>, 2013; Zu <em>et al.</em>, 2010; Mori <em>et al.</em>, 2013a and 2013b). Here, we expressed the DPR poly-PR using a lentiviral vector transduced into our reporter cells, to subsequently measure nucleocytoplasmic transport.<br/> <br/>Although there is growing evidence indicating that impaired nucleocytoplasmic transport is a component of neurodegenerative diseases, it is still under large debate whether nucleocytoplasmic transport deficits are a consequence of neurodegeneration or an instigator (Hutten and Dormann, 2019). As nucleocytoplasmic transport is a complex and stress-sensitive process, reliable assays with a minimal interference are essential to answer this question, as we did before (Vanneste <em>et al.</em>, 2019). We developed several reporter cell lines stably expressing a shuttling-fluorophore allowing us to consistently measure nucleocytoplasmic import as a function of time and in intact cells without the need for transfection. Moreover, this method is cheap and can easily be up scaled for high throughput screening purposes. By fine-tuning the NLS and NES fused to the reporter, the subcellular localization of the reporter can be modified and different transport pathways of interest can be analyzed. While the experiments here are performed with Hela Kyoto cells, the protocol can easily be adapted for other cell types.</p>
        </body>

<p class="pptt" id="biaoti34410">## Materials and Reagents</p><ol class="npwtwok"><li>6-well plate (Corning Life science, 3516)</li><li>96-well plate (TTP, catalog number: <a class="blackNoUnder" target="_blank"> 92696 </a> )</li><li>T-25 culture flask (TTP, catalog number: <a class="blackNoUnder" target="_blank"> 90026 </a> )</li><li>Cover Glass (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 631-0150 </a> )</li><li>Nunc CryoTube (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> V7884 </a> )</li><li>Hela Kyoto cells (Cellosaurus ID: CVCL_1922)</li><li>HEK-293T cells (ATCC, catalog number: CRL-3216<sup>TM</sup>)</li><li>pcDNA<sup>TM</sup> 3.1 (+) plasmid (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> V790-20 </a> )</li><li>pGL4 plasmid (Addgene)</li><li>Plasmid containing Cas9 was a kind gift from Jonathan L. Schmid-Burgk (Schmid-Burgk <em>et al.</em>, 2016).</li><li>Plasmid containing the gRNA was a kind gift from Jonathan L. Schmid-Burgk (Schmid-Burgk <em>et al.</em>, 2016)</li><li>Gibco<sup>TM</sup> Dulbeccoâs Modified Eagleâs Medium (DMEM), high glucose (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11965092 </a> )</li><li>Fetal Bovine Serum (GE Healthcare, Hyclone, catalog number: <a class="blackNoUnder" target="_blank"> SV30160.03 </a> )</li><li>Gibco<sup>TM</sup> Gentamicin (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15750045 </a> )</li><li>Gibco<sup>TM</sup> Puromycin Dihydrochloride (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A1113802 </a> )</li><li>Gibco<sup>TM</sup> Trypsin-EDTA (0.05%), phenol red (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 253000054 </a> )</li><li>Lipofectamine<sup>TM</sup> 3000 Transfection Reagent (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> L3000015 </a> )</li><li>Dimethyl sulfoxide (DMSO) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D2650-100ml </a> )</li><li>Importazole (Selleckchem, catalog number: <a class="blackNoUnder" target="_blank"> S8446 </a> )</li><li>Leptomocyin B (Invivogen, catalog number: <a class="blackNoUnder" target="_blank"> tlrl-lep </a> )</li><li>NucBlue Live ReadyProbes<sup>TM</sup> Reagent Hoechst 33342 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R37605 </a> )</li><li>Gibco<sup>TM</sup> Dulbeccoâs Phosphate-Buffered Saline (DPBS), no calcium, no magnesium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14190250 </a> )</li><li>Formaldehyde 16 % methanol free, Ultra Pure (Polysciences Inc., catalog number: <a class="blackNoUnder" target="_blank"> 18814-20 </a> )</li><li>Hexadimethrine bromide (polybrene) (MedChem Express, catalog number: <a class="blackNoUnder" target="_blank"> 2878-55-4 </a> )</li><li>X-tremeGENE9 DNA transfection reagent (Sigma-Aldrich/Roche, catalog number: <a class="blackNoUnder" target="_blank"> 06 365 809 001 </a> )</li><li>ON-TARGETplus KPNB1 siRNA human (Horizon Discovery, catalog number: <a class="blackNoUnder" target="_blank"> L-01523-00-0005 </a> )</li><li>ON-TARGETplus KPNA2 siRNA human (Horizon Discovery, catalog number: <a class="blackNoUnder" target="_blank"> L-004702-00-0005 </a> )</li><li>Gene Pulser Electroporation Butter (Bio-Rad, catalog number:165-2677)</li><li>siRNA buffer (Horizon Discovery, catalog number: <a class="blackNoUnder" target="_blank"> B-00200-UB-100 </a> )<br/></li></ol><p class="pptt" id="biaoti34411">### Equipment</p><ol class="npwtwok"><li>Incubator</li><li>Forceps</li><li>Mr. Frosty freezing container (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 5100-0001 </a> )</li><li>Scanning confocal microscope, <em>e.g.</em>, SP8 confocal microscope (Leica, model: <a class="blackNoUnder" target="_blank"> SP8 MDi8 </a> )</li><li>CellInsight CX5 high content screening platform (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> CX51110 </a> )<br/></li></ol><p class="pptt" id="biaoti34412">### Software</p><ol class="npwtwok"><li>ImageJ (Wayne Rasband (retired from), <a href="https://imagej.nih.gov/ij/download.html" target="_blank">https://imagej.nih.gov/ij/download.html</a>)</li><li>CellProfiler (Broad Institute, <a href="https://cellprofiler.org/" target="_blank">https://cellprofiler.org/</a>)</li><li>Snapgene (GSL Biotech LLC, <a href="https://www.snapgene.com/" target="_blank">https://www.snapgene.com/</a>)<br/></li></ol><p class="pptt" id="biaoti34413">### Procedure</p><p class="ppzz">A summary of the protocol is shown in Figure 1. The first part of this protocol describes the generation of three different reporter cell lines (Figure 2), which allows us to investigate the import pathways involved in ALS pathology. The protocol can be easily adapted to investigate other transport pathways by simply changing the nuclear localization or nuclear export signal in the constructs. The second part describes how to measure and analyze nucleocytoplasmic transport.<br/><br/><img alt="" class="layerphoto" height="119" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204559_4487.jpg" width="100"/><br/><strong>Figure 1. Workflow summary</strong><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204621_8791.jpg" width="323"/><br/><strong>Figure 2. Generated reporter cell lines.</strong> Using CRISPR-Cas9 technology three different reporter cell lines were generated that stably express the indicated constructs. This allows us to investigate the import pathways that are involved in ALS: (1 + 3) KPNB1/KPNAx-mediated import pathway (pathway used by TDP43) and (2) the TPNO1-mediated pathway (pathway used by FUS). In cell line 1, the reporter mainly localizes into the cytoplasm. This allows, in combination with an XPO1-inhibitor, to induce a time-dependent transport from the cytoplasm into the nucleus and as such to measure KPNB1/KPNAx-mediated import over time. Cell lines 2 and 3 allow measuring a shift in the balance between import and export in a simple way.</p><br/><ol class="npwone"><li>Generation of cell lines to study nucleocytoplasmic transport<br/>The reporter constructs, driven by a CMV promotor, were integrated in the AAVS1 safe harbor locus of Hela Kyoto cells using a CRISPR-Cas9 knock-in protocol. Three different CRISPR-Cas9 knock-in protocols were used.<br/><br/><ol class="npwtwo"><li>CRISPR-Cas9 knock-in protocols for creating nucleocytoplasmic transport cell lines<br/><u>Protocol 1</u> <u></u><span></span><u></u><span></span>(Sakuma <em>et al.</em>, 2016):<br/>Use classical PCR to insert the nucleocytoplasmic transport reporter cassette of interest into a pcDNA3.1 plasmid. When designing the insert, make sure the 5' and 3' ends contain two identical targeting sites and micro-homology sequences. The CRISPR-Cas9 targeting sites are needed to cut the insert out of the plasmid in the following step. The microhomology sequences are needed to specifically insert the cassette into the AAVS1 safe harbor site. Here, we cloned a cassette expressing the NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> and fused it with P2A-PuromycinR (Figure 3A).<br/><u>Protocol 2</u> (Geisinger <em>et al.</em>, 2016):<br/>A PuromycinR-P2A-NESp<sub>ki2</sub>-mNeongreen2x-NLS<sub>FUS</sub> reporter was cloned into a pcDNA3.1 plasmid (Figure 3B) that serves as a template for a PCR reaction with 5â² phosphorothioate modified primers (fw primer: T*C*C*CCTCCACCCCACAGTGGGGCC ACTAACGGGCCAGATATACGCG; rv primer: T*C*C*CCTCCACCCCACAGTGGGGCCACTA CTGGTTCTTTCCGCCTCAGAA, with * denoting a phosphorothioate modified nucleotide). This PCR product is used as the donor for the CRISPR knock-in protocol. The PCR primers contain the 5â² end microhomology sequences to enable specific integration of the insert into the AAVS1 safe harbor site (similar as for protocol 1).<br/><u>Protocol 3</u> (Schmid-Burgk <em>et al.</em>, 2016):<br/>NLS<sub>cMYC</sub>-AcGFP2x-NES<sub>ikb2</sub> was cloned into a pGL4-plasmid that contains at the 5â² end the CRISPR-Cas9 targeting side, to linearize it for insertion into the AAVS1 safe harbor site using the CRISPaint knock-in protocol (Figure 3C).<br/><em>Notes:</em> <ol class="npwthree"><li><em>Pay attention to the final size of your reporter protein. If your goal is to study active transport, use a fusion protein of two or three times your fluorescent protein reporter (approximately 27 kDa). We used two copies of mNeonGreen to exceed the limit for passive transport (Â± 50 kDa).</em> </li><li><em>NES<sub>pki2</sub> contains a K to P mutation (MSLNELALPLAGLDI) which weakens the original NES<sub>pki</sub> export signal. As such, the equilibrium between import and export is shifted towards import and the reporter is localized mainly inside the nucleus.</em> </li><li><em>Because the Schmid-Burgk protocol is not based on microhomology (Schmid-Burgk </em><em>et al.</em><em>, 2016), in contrast to 1 and 2, this is the least efficient knock-in protocol. Moreover, the full donor plasmid is integrated in the Hela Kyoto AAVS1 locus in this protocol. The Sakuma protocol (Sakuma </em><em>et al.</em><em>, 2016) has the highest knock-in efficiency, but is the most difficult one to clone.</em><br/><br/></li></ol></li><li>Stable cell line generation (for used plasmids see Figure 3)<br/>Cell culture medium used in all steps consists of Gibco DMEM medium supplemented with 10% fetal bovine serum (FBS) and 50 Âµg/ml gentamicin.<ol class="npwthree"><li>Day 1: Seed 100,000 Hela Kyoto cells in a 6-well plate.</li><li>Day 2:<br/><u>Protocol 1</u>: Co-transfect these cells using Lipofectamine 3000 (Thermo Fisher Scientific). Add 500 ng of the respective pcDNA3.1-donor plasmid (Figure 3A), 1,000 ng of a plasmid expressing Cas9 plus a gRNA targeting the AAVS1 locus (GTCACCAATCCTGTCCCTAG) (Figure 3D) and 500 ng of a plasmid expressing a gRNA (GCCAGTACCCAAAAAGCGGG) that cleaves the reporter plasmid at two sites to excise the cassette (Figure 3E).<br/><u>Protocol 2</u>: Co-transfect these cells using Lipofectamine 3000 (Thermo Fisher Scientific). Add 500 ng of the cassette PCR donor product (Figure 3B) and 1,000 ng of a plasmid expressing Cas9 plus a gRNA targeting the AAVS1 locus (GTCACCAATCCTGTCCCTAG) (Figure 3D).<br/><u>Protocol 3</u>: Co-transfect these cells using Lipofectamine 3000 (Thermo Fisher Scientific). Add 500 ng of the respective pGL4-donor plasmid (Figure 3C), 1,000 ng of a plasmid expressing Cas9 plus a gRNA targeting the AAVS1 locus (GTCACCAATCCTGTCCCTAG) (Figure 3D) and 500 ng of a plasmid expressing a gRNA (GCCAGTACCCAAAAAGCGGG) that cleaves the reporter plasmid at one site to linearize the donor plasmid (Figure 3E).<br/><em>Tip: Include a negative control condition by omitting the donor plasmid expressing Cas9/gRNA in one well.</em> </li><li>Day 3: Change medium to limit toxicity caused by the transfection reagents.</li><li>Day 4: Trypsinize transfected cells and select with puromycin (1 Âµg/ml).</li><li>Analyze plates in the next days for discrete colonies that start to appear.<br/><em>Tip: To exclude falls positives: make sure to only continue when all cells are dead in the negative control (without donor plasmid). In addition, keep cells under selection with antibiotics in the following days.</em> </li><li>When the 6-well is completely confluent with this polyclonal cell mixture, monoclonal cells can be grown. Therefore, seed cells in a 96-well plate at a density of 0.5 cell/well. This can be achieved as follows:<ol class="npwfour"><li>Take off medium and wash cells with 1 ml of 1x PBS.</li><li>Remove PBS and add 500 Âµl of 0.05% trypsin per well. Return to the incubator for 5 min.</li><li>Confirm that cells are detached. If not, place the cells again in the incubator for several minutes until they do.</li><li>Add 1.5 ml medium per well. It is important to pipette up and down to mix and obtain single cells. Measure the obtained cell number.</li><li>Make a 1/100 dilution (10 Âµl to 990 Âµl of medium) to facilitate the next step. Mix well.</li><li>Add a volume that contains 50 cells to 10 ml of medium (10 ml is sufficient for one 96-well plate).</li><li>Add 100 Âµl per well to obtain a concentration of 0.5 cells/well. As such, the majority of wells will have zero or one cells, with a small number containing two or more cells.</li></ol><em>Tip: Seed a higher density (e.g., 1,000 cells/well) in the left upper corner. As such, it will be easier to find the right focus on the microscope when you search for colonies.</em> </li><li>Scan the plate five days later for the formation of monoclonal colonies. Do not include wells that contain two (or more) separate colonies, as they most probably arose from more than one cell and will therefore not result in a monoclonal culture.<br/><em>Tip: Pay attention to the sides of the wells, as cells tend to attach there.</em> </li><li>When the 96-well is completely full, expand the monoclonal culture.<br/><em>Tip: Expand gradually by starting with a T-25 culture flask.</em> </li><li>Freeze cells at -80 Â°C before long-term storage in liquid nitrogen. This can be achieved as follows:<ol class="npwfour"><li>Prepare freezing medium (for a total of 10 ml: 8.2 ml culturing medium + 800 Âµl DMSO + 1 ml FBS). Place the medium on ice, as DMSO is toxic to your cells at room temperature.</li><li>Label Cryotubes and pre-chill them on ice.</li><li>Take off medium and wash cells with 1x PBS.</li><li>Remove PBS and add 0.05% trypsin. Return to the incubator for 5 min.</li><li>Confirm that cells are detached. If not, return cells to the incubator for several minutes until they do.</li><li>Add medium to inhibit the trypsin and bring everything into a 50 ml tube.<br/></li><li>Count cells.</li><li>Spin off for 5 min at 500 rcf.</li><li>Remove supernatant and resuspend pellet in pre-made freezing medium. Add 1 ml per 1.5 x 10<sup>6</sup>-2 x 10<sup>6</sup> cells.</li><li>Pipette 1 ml of the freezing medium in each pre-chilled cryotube.</li><li>Place tubes at -80 Â°C with the help of a Mr. Frosty<sup>TM</sup> freezing container.</li><li>For long-term storage, move your samples the next day to a liquid nitrogen container.<br/><br/><img alt="" class="layerphoto" height="118" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204701_4097.jpg" width="100"/><br/><strong>Figure 3. Plasmids used to generate reporter cell lines.</strong> A. Donor plasmid containing the reporter NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> used in protocol 1. B. Plasmid containing the reporter NES<sub>pki2</sub>-mNeongreen2x-NLS<sub>FUS</sub>, which served as a template for the PCR reaction performed in protocol 2. NES<sub>pki2</sub>* contains a K to P mutation. C. Donor plasmid containing the reporter NLS<sub>cMYC</sub>-AcGFP2x-NES<sub>ikb2</sub> used in protocol 3. D. Plasmid expressing Cas9 plus a gRNA targeting the AAVS1 locus (Schmid-Burgk <em>et al.</em>, 2016) used in protocols 1, 2 and 3. E. Plasmid expressing a gRNA to cleave the donor plasmid containing the reporter (Schmid-Burgk <em>et al.</em>, 2016) used in protocols 1 and 3.<br/></li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Validation<br/><em>Note: It is important to validate your reporter. This can be done by:</em><br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left"><em>1)</em> </div><div class="media-body"><em>Treating your cells with available compounds e.g., leptomycin B to inhibit XPO1-mediated export (Kudo </em><em>et al.</em><em>, 1999) or importazole to inhibit KPNB1-mediated import (Soderholm </em><em>et al.</em><em>, 2011).</em> </div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left"><em>2)</em> </div><div class="media-body"><em>Transfecting your cells with appropriate constructs e.g., M9M expression to inhibit TPNO1/TPNO2 (Cansizoglu </em><em>et al.</em><em>, 2007).</em> </div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left"><em>3)</em> </div><div class="media-body"><em>Knock down of targeted transport receptors e.g., siRNA against KPNB1 or KPNA2.</em> </div></div><br/><em>Note: Cell culture medium used in all steps consists of Gibco DMEM medium supplemented with 10% FBS and 50 Âµg/ml gentamicin.</em> <ol class="npwthree"><li>Validate XPO1-mediated export with leptomycin B<ol class="npwfour"><li>Day 1: Seed NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter cells at a density to obtain 60-70% confluency the next day.</li><li>Day 2: Change half of the medium in the wells to obtain a final concentration of 50 nM of the XPO1-inhibitor leptomycin B (Kudo <em>et al.</em>, 1999).</li><li>Return cells to the incubator for 30 min.</li><li>Fix cells with 4% PFA for 15 min.</li><li>Wash cells three times with 1x PBS.</li><li>Stain nuclei using NucBlue Live Cell Stain reagent (Invitrogen).</li><li>Image cells to confirm increased nuclear localization of the reporter in the leptomycin B treated cells (Figure 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204733_6427.jpg" width="290"/><br/><strong>Figure 4. Validation of XPO1-mediated export.</strong> Representative images of cells expressing the reporter NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub>. A. In steady state, the reporter localizes in the cytoplasm of cells. B. Addition of the XPO1-inhibitor leptomcyin B (LMB) results in nuclear localization of the reporter. Scale bars = 20 Âµm.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Validate KPNB1/KPNAx-mediated import with importazole<ol class="npwfour"><li>Day 1: Seed NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter cells at a density to obtain 60-70% confluency the next day.</li><li>Day 2: Incubate the cells for 2 h with 50 ÂµM of the KPNB1-inhibitor importazole (Soderholm <em>et al.</em>, 2011).</li><li>Change half of the medium in the wells to obtain a final concentration of 50 nM leptomycin B.<br/><em>Note: The reporter localizes steady state in the cytoplasm of the cells (due to XPO1-mediated export) and serves therefore, in combination with an XPO1-inhibitor (leptomycin B), as an import-reporter for KPNB1/KPNAx-mediated import.</em> </li><li>Return cells to the incubator and fix cells with 4% PFA after 30 min.</li><li>Wash cells three times with 1x PBS.</li><li>Stain nuclei using NucBlue Live Cell Stain reagent (Invitrogen).</li><li>Image cells to confirm a decreased nuclear localization of the reporter in the importazole-treated cells (Figure 5).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204757_7096.jpg" width="187"/><br/><strong>Figure 5. Validation of KPNB1/KPNAx-mediated import.</strong> Representative images of cells expressing the reporter NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub>. A. Addition of leptomycin B results in nuclear localization of the reporter, due to KPNB1/KPNAx-mediated import. B. Pre-incubation with importazole partially abolishes KPNB1/KPNAx-mediated import. Scale bars = 20 Âµm.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Validate TPNO1/TPNO2-mediated import through M9M expression<ol class="npwfour"><li>Day 1: Seed NES<sub>pki2</sub>-mNeongreen2x-NLS<sub>FUS</sub> reporter cells at a density to obtain 60-70% confluency the next day.</li><li>Day 2: Transfect cells with a plasmid expressing the peptide M9M (Cansizoglu <em>et al.</em>, 2007) using lipofectamine 3000 (Thermo Fisher Scientific).<br/><em>Note: Express M9M in fusion with a fluorescent protein (</em><em>e.g.</em><em>, Red fluorescent protein) to identify transfected cells.</em> </li><li>Change medium 4 h after transfection to reduce toxicity.</li><li>Incubate cells overnight.</li><li>Day 3: Fix cells with 4% PFA for 15 min.</li><li>Wash three times with 1x PBS.</li><li>Stain nuclei using NucBlue Live Cell Stain reagent (Invitrogen).</li><li>Image cells to confirm decreased nuclear localization of the reporter in the M9M-expressing cells (Figure 6).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204823_5246.jpg" width="463"/><br/><strong>Figure 6. Validation of TPNO1/TPNO2-mediated import.</strong> Representative images of cells expressing the reporter NES<sub>pki2</sub>-mNeongreen2x-NLS<sub>FUS</sub>. A. In steady state, the reporter localizes inside the nucleus, caused by TPNO1/TPNO2-mediated import. Reporter cells were transfected with an mCherry-plasmid as negative control. B. Reporter cells transfected with an RFP-M9M-plasmid, which inhibits TPNO1/TPNO2-mediated import and results in cytoplasmic redistribution of the reporter. Scale bars = 20 Âµm.<br/><br/></li></ol></li><li>Validate KPNB1/KPNAx-mediated import through KPNB1/KPNA2-knock down<ol class="npwfour"><li>Day 1: Seed NLS<sub>cMYC</sub>-AcGFP2x-NES<sub>ikb2</sub> reporter cells at a density to obtain 60-70% confluency the next day.</li><li>Day 2: Electroporate cells with siRNA against KPNB1 or KPNA2 (Horizon Discovery siRNA resuspension buffer, Bio-Rad: 230V, 950 ÂµF, âÎ©, 10,000 ms, #1 pulse).</li><li>Incubate cells for 48 h.</li><li>Fix cells with 4% PFA for 15 min.</li><li>Wash three times with 1x PBS.</li><li>Stain nuclei using NucBlue Live Cell Stain reagent (Invitrogen).</li><li>Image cells to confirm decreased nuclear localization of the reporter (Figure 7).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204850_5809.jpg" width="136"/><br/><strong>Figure 7. Validation of KPNB1/KPNAx-mediated import.</strong> Representative images of cells expressing the reporter NLS<sub>cMYC</sub>-AcGFP2x-NES<sub>ikb2</sub>. A. The reporter localizes steady state in the nucleus of the cells, due to KPNB1/KPNAx-mediated import. B. Expression of siRNA targeting KPNB1 or KPNA2 results in cytoplasmic localization of the reporter. Scale bars = 25 Âµm.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Optimization of the NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter<br/>In steady state, the NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter localizes in the cytoplasm of the cells (caused by XPO1-mediated nuclear export) and serves therefore, in combination with an XPO1-inhibitor (leptomycin B; LMB), as an import-reporter for KPNB1/KPNAx-mediated import.<br/><em>Note: Because the nuclear intensity results from the balance between export and import, it is important to optimize LMB concentrations to maximize the sensitivity of the assay to measure import deficits. In addition, to minimalize variability it is recommended to work with a LMB concentration that induces a gradual import of the reporter and allows import measurements to occur over longer periods (30 min). Moreover, it is recommended to measure at least two different time points to assure that your measurement falls within the dynamic range of the nuclear import process and saturation is not yet reached.</em><br/><br/><em>Note: Cell culture medium used in all steps consists of Gibco DMEM medium supplemented with 10% FBS and 50 Âµg/ml gentamicin.</em> <ol class="npwthree"><li>Day 1: Seed NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter cells in a 96-well plate at a density to obtain 70% confluency the next day.<br/><em>Note: Keep in mind that you will need three different time points and this in duplicate or triplicate per condition.</em> </li><li>Day 2: Replace old medium with 100 Âµl of medium containing 25 ÂµM of the KPNB1-inhibitor importazole.</li><li>Return cells to the incubator for 1.5 h.<br/><em>Note: The importazole concentration was chosen with the aim to induce average import inhibition, to not miss potential small import deficits induced by poly-PR.</em> </li><li>Meanwhile, prepare different dilutions of LMB (dissolved in ethanol) in pre-warmed medium at a final concentration of 0 nM, 15 nM, 30 nM, 45 nM, 60 nM, 75 nM and 90 nM. Vortex.<br/><em>Note: Keep in mind the dilution in the next steps. Therefore, prepare twice the final concentration, e.g., 120 nM for a final concentration of 60 nM.</em> </li><li>Treat the cells intended for the 30 min time point with the different concentrations of LMB, by adding 100 Âµl of medium containing LMB to the 100 Âµl of medium already in the wells.</li><li>Return cells to the incubator for 15 min.</li><li>Treat the cells intended for the 15 min time point with the different concentrations of LMB by adding 100 Âµl of the medium containing LMB.</li><li>Return cells to the incubator for 15 min.</li><li>Fix with 4% PFA for 15 min.</li><li>Wash cells three times with 1x PBS.</li><li>Stain nuclei using NucBlue Live Cell Stain reagent (Invitrogen).</li><li>Cells can be analyzed automatically by the CellInsight CX5 high content screening platform (Thermo Fisher Scientific) to determine the LMB concentration that induces the largest difference between control cells and importazole-treated cells (Figure 8).<br/><em>Note: Use a multichannel pipette to reduce variability.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204914_8097.jpg" width="138"/><br/><strong>Figure 8. Optimization of KPNB1/KPNAx-mediated import of the NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter measured over time.</strong> A-B. Import of importazole-treated cells relative to import of DMSO-treated cells for different concentrations of leptomycin B (LMB). A-15 min; B-30 min. A. 45 nM of LMB results in the strongest difference between control cells and cells-treated with importazole. B. Lower concentrations allow measuring import-inhibition, as import becomes saturated at this later time point for higher concentrations of LMB. C. Left graph: lower concentrations (<em>e.g.</em>, 30 nM) of LMB result in less overall import and make it therefore harder to observe disturbed import after 15 min. Import defects can be observed at later time points (30 min). Middle graph: 45 nM of LMB revealed import deficits at both time points. Right graph: higher concentrations of LMB (<em>e.g.</em>, 90 nM) can push import too hard to observe a minimal impediment of import. A-B. Dot represents the average of one experiment with 2,500-7,000 cells per experiment. Mean + S.D. C. Dot represent the average of one experiment with 2,500-7,000 cells per experiment.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Measuring nucleocytoplasmic transport in the presence of mutant C9orf72-related peptides<ol class="npwtwo"><li>Production of lentiviral vectors expressing mutant C9orf72-related peptides<br/>DPR<sub>100</sub>-plasmids were a kind gift of Dr. Daisuke Ito (Department of Neurology, Keio University, Tokyo, Japan). Classical PCR was used to clone mCherry-DPR<sub>100</sub> constructs into lentiviral vector plasmids (LentiCrisprV2_Puro from Addgene) (Figure 9). Production of lentiviral vectors was done using HEK-293T cells as previously described (Hart <em>et al.</em>, 2017).<br/><em>Note: We strongly advise to use transduction instead of transfection to express desired disease-related constructs. Transfection is more stressful which can influence nucleocytoplasmic transport measurements.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617204942_2024.jpg" width="239"/><br/><strong>Figure 9. Lentiviral vector plasmids used to express mCherry or mCherry-PR<sub>100</sub>.</strong> A. Lentiviral vector containing mCherry. B. Lentiviral vector containing mCherry-PR<sub>100</sub>.<br/><br/></li><li>Assay<br/><em>Note: Cell culture medium used in all steps consists of Gibco DMEM medium supplemented with 10% FBS and 50 Âµg/ml gentamicin.</em> <ol class="npwthree"><li>Passage cells and seed onto cover glasses<ol class="npwfour"><li>Day 0: Place a single cover glass into each well of a sterile 24-well polystyrene tissue culture plate with the use of sterile forceps.<br/><em>Notes:</em><br/><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left"><em>1)</em> </div><div class="media-body"><em>Pre-coating coverslips with poly-L-lysine and poly-L-ornithine or other cell-surface treatments may be necessary to adhere other cell types to glass. Hela Kyoto cells adhere to glass without additional coating.</em> </div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left"><em>2)</em> </div><div class="media-body"><em>For high throughput screens, cells can be seeded into 96-well plates or other formats.</em> </div></div><div class="media media-x cnautokh cnautokh3" style="margin-top:-0.5px;"><div class="float-left"><em>3)</em> </div><div class="media-body"><em>Keep in mind that you will need three different time points and this in duplicate or triplicate per condition.</em> </div></div></li><li>Seed at a density that will result in 70% confluency at the day you plan to perform the nucleocytoplasmic transport assay.</li><li>Incubate cells overnight in a tissue culture incubator (37 Â°C with 5% CO<sub>2</sub>).</li></ol></li><li>Transduction of cells to express C9orf72-related peptides<ol class="npwfour"><li>Day 1: Dilute polybrene in medium at a final concentration of 10 Âµg/ml. Add lentiviral particles to each well. Leave the viral particles overnight on the cells (about 18 h).</li><li>Day 2: Remove the lentivirus-containing medium and replace with fresh DMEM medium.</li><li>Incubate cell for an additional two days in a tissue culture incubator (37 Â°C with 5% CO<sub>2</sub>).<br/><em>Note: Addition of polybrene can significantly improve transduction efficiency. However, first confirm that polybrene is not toxic to your cells as it can induce cell death in certain cell types (e.g., iPSC-derived neurons).</em> </li></ol></li><li>Measure nucleocytoplasmic transport<ol class="npwfour"><li>Day 5: Prepare a desired volume of pre-warmed medium containing 45 nM LMB.</li><li>Treat the cells intended for the 30 min time point with the medium containing LMB.</li><li>Return cells to the incubator for 15 min.</li><li>Treat the cells intended for the 15 min time point with the medium containing LMB.</li><li>Return cells to the incubator for 15 min.</li><li>Fix cells by adding 4% PFA (final concentration) to the medium for 15 min.</li><li>Stain nuclei using NucBlue Live Cell Stain reagent (Invitrogen).</li><li>Use a 20 Âµl pipette to place a drop of mounting medium on a slide, before placing the cover glass upside down on the slide using forceps. Remove excess mounting medium with a paper towel and seal the slides using nail polish.</li><li>Image cells using a confocal microscope (Figure 10).<br/><em>Notes:</em> <div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left"><em>1)</em> </div><div class="media-body"><em>Keep in mind to maintain the same setting between different conditions and to focus on the middle of the nuclei (nucleoli are clearly visible).</em> </div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left"><em>2)</em> </div><div class="media-body"><em>For high-throughput screens, intensity can be automatically analyzed by using a high content analyzer (e.g., CellInsight CX5 high content screening platform from Thermo Fisher Scientific).</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617205010_0493.jpg" width="146"/><br/><strong>Figure 10. Measuring KPNB1/KPNAx-mediated import over time in a neurodegenerative-associated context. <span style="font-weight:normal;">Representative images of cells expressing the reporter NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub>. The reporter localizes steady state in the cytoplasm of the cells, as a consequence of XPO1-mediated nuclear export. Addition of an XPO1-inhibitor (leptomycin B; LMB) allows to measure KPNB1/KPNAx-mediated import over time. A. Reporter cells were transduced with a mCherry-lentiviral vector as control. B. Reporter cells transduced with a poly-PR<sub>100</sub>-mCherry-lentiviral vector. Poly-PR dipeptides have been linked to the neurodegenerative disease amyotrophic lateral sclerosis. Scale bar = 20 Âµm.</span></strong> </div></div></li></ol></li></ol></li></ol></li></ol><p class="pptt" id="biaoti34414">### Data analysis</p><ol class="npwone"><li>Manual<ol class="npwtwo"><li>ImageJ<br/>Images taken on the confocal microscope can be manually analyzed with the free-online available software ImageJ.<br/><ol class="npwthree"><li>Import images in ImageJ.<br/><em>Note: It is recommended to analyze the images in 64-bit format.</em> </li><li>Select region of interest + Ctrl M (Figure 11).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617205034_9531.jpg" width="295"/><br/><strong>Figure 11. Manual analysis in ImageJ.</strong> Images are representative for NLS<sub>SV40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter cells before addition of LMB and transfected with mCherry. Graph contains analyzed data of images from Figure 10. Mean Â± S.D. are shown.</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Automated<ol class="npwtwo"><li>Cell profiler<br/>Cells imaged on the confocal microscope can be automatically analyzed with the free-online available software Cell profiler (Figure 12).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617205055_6608.jpg" width="173"/><br/><strong>Figure 12. Workflow for automatic analysis in Cell profiler.</strong> Images are representative for NLS<sub>sv40</sub>-mNeongreen2x-NES<sub>pki</sub> reporter cells before addition of LMB and transfected with mCherry. Graph contains analyzed data of images from Figure 10. Mean Â± S.D. are shown.<br/></li></ol><br/><strong>Statistical analysis</strong> <ol class="npwtwo"><li>We recommend to analyze nuclear-to-total (= cytoplasm + nucleus) ratios as nucleus-to-cytoplasm ratios may be susceptible to large variations. The difference between nuclear and cytoplasmic values can become very large, especially before addition of LMB (very low nuclear intensity) or at the end of the assay (very low cytoplasmic intensity). In addition, when the data have to be normalized to the initial intensity, a small difference in the nuclear and cytoplasmic intensity might create a large difference in the ratio.</li><li>Ratios are intrinsically asymmetric, meaning that all decreases are expressed as ratios between zero and one, and all increases are expressed as ratios greater than 1.0. Therefore, we recommend analyzing the logarithm of the ratios. As such, no change will be zero (the logarithm of 1.0), increases will be positive and decreases will be negative.</li></ol></li></ol><p class="pptt" id="biaoti34415">### Acknowledgments</p><p class="ppzz">We thank Liesbeth Mercelis and Bob Massant for technical assistance. This work was supported by the KU Leuven (C1 and Opening the Future Fund); Fund for Scientific Research Flanders (FWO-Vlaanderen GOB3318N â G098314N), the ALS Liga (Belgium). J.V. is doctoral fellow of FWO. P.V.D. holds a senior clinical investigator ship of FWO-Vlaanderen and is supported by Fund âEen hart voor ALSâ and Laevers Fund for ALS Research.</p><p class="pptt" id="biaoti34416">### Competing interests</p><p class="ppzz">The authors declare they have no financial and non-financial competing interests.</p><p class="pptt" id="biaoti34417">### References</p><ol class="npwtwok"><li>Arai, T., Hasegawa, M., Akiyama, H., Ikeda, K., Nonaka, T., Mori, H., Mann, D., Tsuchiya, K., Yoshida, M., Hashizume, Y. and Oda, T. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17084815" target="_blank">TDP-43 is a component of ubiquitin-positive tau-negative inclusions in frontotemporal lobar degeneration and amyotrophic lateral sclerosis.</a> <em>Biochem Biophys Res Commun</em> 351(3): 602-611.</li><li>Ash, P. E., Bieniek, K. F., Gendron, T. F., Caulfield, T., Lin, W. L., Dejesus-Hernandez, M., van Blitterswijk, M. M., Jansen-West, K., Paul, J. W., 3rd, Rademakers, R., Boylan, K. B., Dickson, D. W. and Petrucelli, L. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23415312" target="_blank">Unconventional translation of C9ORF72 GGGGCC expansion generates insoluble polypeptides specific to c9FTD/ALS.</a> <em>Neuron</em> 77(4): 639-646.</li><li>Boeynaems, S., Bogaert, E., Van Damme, P. and Van Den Bosch, L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27271576" target="_blank">Inside out: the role of nucleocytoplasmic transport in ALS and FTLD.</a> <em>Acta Neuropathol</em> 132(2): 159-173. </li><li>Cansizoglu, A. E., Lee, B. J., Zhang, Z. C., Fontoura, B. M. and Chook, Y. M. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17435768" target="_blank">Structure-based design of a pathway-specific nuclear import inhibitor.</a> <em>Nat Struct Mol Biol</em> 14(5): 452-454.</li><li>Fahrenkrog, B. and Harel, A. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30486313" target="_blank">Perturbations in traffic: aberrant nucleocytoplasmic transport at the heart of neurodegeneration.</a> <em>Cells</em> 7(12). DOI: 10.3390/cells7120232.</li><li>Geisinger, J. M., Turan, S., Hernandez, S., Spector, L. P. and Calos, M. P. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26762978" target="_blank"><em>In vivo</em> blunt-end cloning through CRISPR/Cas9-facilitated non-homologous end-joining.</a> <em>Nucleic Acids Res</em> 44(8): e76.</li><li>Hart, T., Tong, A. H. Y., Chan, K., Van Leeuwen, J., Seetharaman, A., Aregger, M., Chandrashekhar, M., Hustedt, N., Seth, S., Noonan, A., Habsid, A., Sizova, O., Nedyalkova, L., Climie, R., Tworzyanski, L., Lawson, K., Sartori, M. A., Alibeh, S., Tieu, D., Masud, S., Mero, P., Weiss, A., Brown, K. R., Usaj, M., Billmann, M., Rahman, M., Constanzo, M., Myers, C. L., Andrews, B. J., Boone, C., Durocher, D. and Moffat, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28655737" target="_blank">Evaluation and design of genome-wide CRISPR/SpCas9 knockout screens.</a> <em>G3 (Bethesda)</em> 7(8): 2719-2727. </li><li>Hutten, S. and Dormann, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31152789" target="_blank">Nucleocytoplasmic transport defects in neurodegeneration - Cause or consequence?</a> <em>Semin Cell Dev Biol</em> S1084-9521 (18). 301090-301093.</li><li>Kudo, N., Matsumori, N., Taoka, H., Fujiwara, D., Schreiner, E. P., Wolff, B., Yoshida, M. and Horinouchi, S. (1999). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10430904" target="_blank">Leptomycin B inactivates CRM1/exportin 1 by covalent modification at a cysteine residue in the central conserved region.</a> <em>Proc Natl Acad Sci U S A</em> 96(16): 9112-9117.</li><li>Kwiatkowski, T. J., Jr., Bosco, D. A., Leclerc, A. L., Tamrazian, E., Vanderburg, C. R., Russ, C., Davis, A., Gilchrist, J., Kasarskis, E. J., Munsat, T., Valdmanis, P., Rouleau, G. A., Hosler, B. A., Cortelli, P., de Jong, P. J., Yoshinaga, Y., Haines, J. L., Pericak-Vance, M. A., Yan, J., Ticozzi, N., Siddique, T., McKenna-Yasek, D., Sapp, P. C., Horvitz, H. R., Landers, J. E. and Brown, R. H., Jr. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19251627" target="_blank">Mutations in the FUS/TLS gene on chromosome 16 cause familial amyotrophic lateral sclerosis.</a> <em>Science</em> 323(5918): 1205-1208.</li><li>Lange, A., Mills, R. E., Lange, C. J., Stewart, M., Devine, S. E. and Corbett, A. H. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17170104" target="_blank">Classical nuclear localization signals: definition, function, and interaction with importin Î±.</a> <em>J Biol Chem </em>282(8): 5101-5105. </li><li>Majounie, E., Renton, A. E., Mok, K., Dopper, E. G., Waite, A., Rollinson, S., Chio, A., Restagno, G., Nicolaou, N., Simon-Sanchez, J., van Swieten, J. C., Abramzon, Y., Johnson, J. O., Sendtner, M., Pamphlett, R., Orrell, R. W., Mead, S., Sidle, K. C., Houlden, H., Rohrer, J. D., Morrison, K. E., Pall, H., Talbot, K., Ansorge, O., Chromosome, A. L. S. F. T. D. C., French research network on, F. F. A., Consortium, I., Hernandez, D. G., Arepalli, S., Sabatelli, M., Mora, G., Corbo, M., Giannini, F., Calvo, A., Englund, E., Borghero, G., Floris, G. L., Remes, A. M., Laaksovirta, H., McCluskey, L., Trojanowski, J. Q., Van Deerlin, V. M., Schellenberg, G. D., Nalls, M. A., Drory, V. E., Lu, C. S., Yeh, T. H., Ishiura, H., Takahashi, Y., Tsuji, S., Le Ber, I., Brice, A., Drepper, C., Williams, N., Kirby, J., Shaw, P., Hardy, J., Tienari, P. J., Heutink, P., Morris, H. R., Pickering-Brown, S. and Traynor, B. J. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22406228" target="_blank">Frequency of the C9orf72 hexanucleotide repeat expansion in patients with amyotrophic lateral sclerosis and frontotemporal dementia: a cross-sectional study.</a> <em>Lancet Neurol</em> 11(4): 323-330. </li><li>Mori, K., Arzberger, T., Grasser, F. A., Gijselinck, I., May, S., Rentzsch, K., Weng, S. M., Schludi, M. H., van der Zee, J., Cruts, M., Van Broeckhoven, C., Kremmer, E., Kretzschmar, H. A., Haass, C. and Edbauer, D. (2013a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24132570" target="_blank">Bidirectional transcripts of the expanded C9orf72 hexanucleotide repeat are translated into aggregating dipeptide repeat proteins.</a> <em>Acta Neuropathol</em> 126(6): 881-893.</li><li>Mori, K., Weng, S. M., Arzberger, T., May, S., Rentzsch, K., Kremmer, E., Schmid, B., Kretzschmar, H. A., Cruts, M., Van Broeckhoven, C., Haass, C. and Edbauer, D. (2013b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23393093" target="_blank">The C9orf72 GGGGCC repeat is translated into aggregating dipeptide-repeat proteins in FTLD/ALS.</a> <em>Science </em>339(6125): 1335-1338.</li><li>Neumann, M., Sampathu, D. M., Kwong, L. K., Truax, A. C., Micsenyi, M. C., Chou, T. T., Bruce, J., Schuck, T., Grossman, M., Clark, C. M., McCluskey, L. F., Miller, B. L., Masliah, E., Mackenzie, I. R., Feldman, H., Feiden, W., Kretzschmar, H. A., Trojanowski, J. Q. and Lee, V. M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17023659" target="_blank">Ubiquitinated TDP-43 in frontotemporal lobar degeneration and amyotrophic lateral sclerosis.</a> <em>Science</em> 314(5796): 130-133.</li><li>Ossareh-Nazari, B., Gwizdek, C. and Dargemont, C. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11576444" target="_blank">Protein export from the nucleus.</a> <em>Traffic</em> 2(10): 684-689. </li><li>Ryan, K. J. and Wente, S. R. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10801463" target="_blank">The nuclear pore complex: a protein machine bridging the nucleus and cytoplasm.</a> <em>Curr Opin Cell Biol</em> 12(3): 361-371.</li><li>Sakuma, T., Nakade, S., Sakane, Y., Suzuki, K. T. and Yamamoto, T. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26678082" target="_blank">MMEJ-assisted gene knock-in using TALENs and CRISPR-Cas9 with the PITCh systems.</a> <em>Nat Protoc</em> 11(1): 118-133.</li><li>Schmid-Burgk, J. L., HÃ¶ning, K., Ebert, T. S. and Hornung, V. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27465542" target="_blank">CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism.</a> <em>Nat Commun</em> 7: 12338.</li><li>Soderholm, J. F., Bird, S. L., Kalab, P., Sampathkumar, Y., Hasegawa, K., Uehara-Bingen, M., Weis, K. and Heald, R. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21469738" target="_blank">Importazole, a small molecule inhibitor of the transport receptor importin-Î².</a> <em>ACS Chem Biol</em> 6(7): 700-708. </li><li>Stoffler, D., Fahrenkrog, B. and Aebi, U. (1999). <a href="http://www.sciencedirect.com/science/article/pii/S0955067499800556" target="_blank">The nuclear pore complex: from molecular architecture to functional dynamics.</a> <em>Current Opinion Cell Biology</em> 11(3): 391-401.</li><li>Tyzack, G. E., Luisier, R., Taha, D. M., Neeves, J., Modic, M., Mitchell, J. S., Meyer, I., Greensmith, L., Newcombe, J., Ule, J., Luscombe, N. M. and Patani, R. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31368485" target="_blank">Widespread FUS mislocalization is a molecular hallmark of amyotrophic lateral sclerosis.</a> <em>Brain</em> 142(9): 2572-2580. </li><li>Vance, C., Rogelj, B., Hortobagyi, T., De Vos, K. J., Nishimura, A. L., Sreedharan, J., Hu, X., Smith, B., Ruddy, D., Wright, P., Ganesalingam, J., Williams, K. L., Tripathi, V., Al-Saraj, S., Al-Chalabi, A., Leigh, P. N., Blair, I. P., Nicholson, G., de Belleroche, J., Gallo, J. M., Miller, C. C. and Shaw, C. E. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19251628" target="_blank">Mutations in FUS, an RNA processing protein, cause familial amyotrophic lateral sclerosis type 6.</a> <em>Science</em> 323(5918): 1208-1211.</li><li>Vanneste, J., Vercruysse, T., Boeynaems, S., Sicart, A., Van Damme, P., Daelemans, D. and Van Den Bosch, L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31673013" target="_blank">C9orf72-generated poly-GR and poly-PR do not directly interfere with nucleocytoplasmic transport.</a> <em>Sci Rep</em> 9(1): 15728. </li><li>Yuva-Aydemir, Y., Almeida, S. and Gao, F. B. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29729808" target="_blank">Insights into C9ORF72-Related ALS/FTD from Drosophila and iPSC Models.</a> <em>Trends Neurosci</em> 41(7): 457-469. </li><li>Zu, T., Gibbens, B., Doty, N. S., Gomes-Pereira, M., Huguet, A., Stone, M. D., Margolis, J., Peterson, M., Markowski, T. W., Ingram, M. A., Nan, Z., Forster, C., Low, W. C., Schoser, B., Somia, N. V., Clark, H. B., Schmechel, S., Bitterman, P. B., Gourdon, G., Swanson, M. S., Moseley, M. and Ranum, L. P. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21173221" target="_blank">Non-ATG-initiated translation directed by microsatellite expansions.</a> <em>Proc Natl Acad Sci U S A</em> 108(1): 260-265.<br/></li></ol>