<body>
            <h1>Genomic Edition of <em>Ashbya gossypii</em> Using One-vector CRISPR/Cas9</h1>
            <p><strong>Authors:</strong> Gloria MuÃ±oz-FernÃ¡ndez, Alberto JimÃ©nez, JosÃ© Luis Revuelta</p>
            <h2>Abstract</h2>
            <p>The CRISPR/Cas9 system is a novel genetic tool which allows the precise manipulation of virtually any genomic sequence. In this protocol, we use a specific CRISPR/Cas9 system for the manipulation of <em>Ashbya gossypii</em>. The filamentous fungus <em>A. gossypii</em> is currently used for the industrial production of riboflavin (vitamina B2). In addition, <em>A. gossypii</em> produces other high-value compounds such as folic acid, nucleosides and biolipids. A large molecular toolbox is available for the genomic manipulation of this fungus including gene targeting methods, rapid assembly of heterologous expression modules and, recently, a one-vector CRISPR/Cas9 editing system adapted for <em>A. gossypii</em> that allows marker-free engineering strategies to be implemented. The CRISPR/Cas9 system comprises an RNA guided DNA endonuclease (Cas9) and a guide RNA (gRNA), which is complementary to the genomic target region. The Cas9 nuclease requires a 5â²-NGG-3â² trinucleotide, called protospacer adjacent motif (PAM), to generate a double-strand break (DSB) in the genomic target, which can be repaired with a synthetic mutagenic donor DNA (dDNA) by homologous recombination (HR), thus introducing a specific designed mutation. The CRISPR/Cas9 system adapted for <em>A. gossypii</em> largely facilitates the genomic edition of this industrial fungus.</p>
            <p><strong>Keywords:</strong> CRISPR/Cas9, One-vector, <em>Ashbya gossypii</em>, Genome engineering, Gene editing, Biotechnology</p>
            <h2>Background</h2><p>The CRISPR/Cas9 system has emerged as a powerful tool for genome engineering (Horvath <em>et al.</em>, 2010). A CRISPR/Cas9 system comprises an RNA guided Cas9 endonuclease that produces DSBs at specific genomic loci. The Cas9 associated RNA is formed by a CRISPR targeting RNA (crRNA) and a transactivating crRNA (tracrRNA), which are required to form the catalytic active Cas9 (Jinek <em>et al.</em>, 2012). Both the crRNA and tracrRNA can be combined in a synthetic guide RNA (sgRNA) (Jinek <em>et al.</em>, 2012). The gRNA determines the target loci and the Cas9 nuclease requires the presence of a PAM sequence in the genomic target to generate a DSB (Pattanayak <em>et al.</em>, 2013). There are two mechanisms for DSB repair: homologous recombination (HR) and non-homologous end joining (NHEJ). The presence of a synthetic dDNA provides the ability to repair a targeted DSB by HR (Ran <em>et al.</em>, 2013).<br/> <br/>The CRISPR/Cas9 systems need to be adapted for each organism in order to achieve an efficient expression of both the gRNA and Cas9. For the <em>A. gossypii</em> system, a one-vector strategy was followed to contain all the required modules for CRISPR/Cas9 functionality (Figure 1): Cas9 expression, sgRNA expression and dDNA modules. The <em>CAS9</em> expression module that was previously reported in <em>Saccharomyces cerevisiae</em> was used, where the human codon-optimized <em>Streptococcus pyogenes CAS9</em> gene is under the control of the yeast <em>TEF1</em> promoter and <em>CYC1</em> terminator sequences (Dicarlo <em>et al.</em>, 2013). In addition, an sgRNA expression module was designed under the control of the promoter and terminator sequences from the <em>A. gossypii</em> SNR52 gene, which is transcribed by RNA polymerase III. The sgRNA contained two sequences: a 20 bp sequence that targets a selected genomic locus and a 79 bp sequence for Cas9 binding (Figure 1). Also, the <em>A. gossypii</em> system contained the dDNA for DSB repair by HR (Figure 1). The assembly of the CRISPR/Cas9 vector with a specific synthetic gRNA-dDNA is carried out using a directional cloning strategy (JimÃ©nez <em>et al.</em>, 2019). Therefore, for each specific genomic edition, it is only necessary to design a specific sgRNA-dDNA both for gene targeting and DNA repair by HR. The CRISPR/Cas9 vector also contained a <em>loxP-KanMX-loxP</em> marker (G418<sup>R</sup>) for plasmid selection in <em>A. gossypii</em> (JimÃ©nez <em>et al.</em>, 2019).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617183939_6855.jpg" width="101"/><br/><strong>Figure 1. Map of the CRISPR/Cas9 plasmid.</strong> The sgRNA-dDNA module is depicted: the sgRNA comprises both sequences for gene targeting (blue) and Cas9 binding (red). The expression of the sgRNA is driven by the promoter and terminator sequences of <em>SNR52</em>. A short spacer sequence (grey) links the sgRNA module with the dDNA module (light green).<br/> <br/>Most of the CRISPR/Cas9 systems require two transformation events: one for the introduction of the Cas9 and gRNA modules and a second one for the introduction of dDNA (DiCarlo <em>et al.</em>, 2013). This strategy hinders the efficiency of the system in a multinucleated syncytium such as the <em>A. gossypii</em> mycelia. Hence, the use of a one-vector strategy in the <em>A. gossypii</em> CRISPR/Cas9 system largely increases its efficiency (JimÃ©nez <em>et al.</em>, 2019). Indeed, an average editing efficiency of 60% is achieved with the <em>A. gossypii</em> CRISPR/Cas9 system (JimÃ©nez <em>et al.</em>, 2019).<br/> <br/>The CRISPR/Cas9 system for <em>A. gossypii</em> represents a novel methodology for marker-less gene deletions, insertions and nucleotide substitutions. Also, multiplexing CRISPR/Cas9 engineering can be achieved for the simultaneous edition of different targets and metabolic pathways, as previously demonstrated for <em>S. cerevisiae</em> (Bao <em>et al.</em>, 2015; Jakociunas <em>et al.</em>, 2015). In summary, the CRISPR/Cas9 system for <em>A. gossypii</em> will help with the rapid expansion of precise and efficient genomic editing strategies for this microorganism with industrial interest.</p>
        </body>

<p class="pptt" id="biaoti34420">## Materials and Reagents</p><p class="ppzz"><em>Note: Materials and reagents can be purchased from different suppliers. Here, we show one list of all reagents and materials that we used to develop the protocol.</em><br/><br/><strong><em>Materials</em></strong> </p><ol class="npwtwo"><li>Pipette tips (<em>e.g.</em>, VWR, catalog numbers: <a class="blackNoUnder" target="_blank"> 613-0340 </a> , <a class="blackNoUnder" target="_blank"> 613-0360 </a> , <a class="blackNoUnder" target="_blank"> 613-1068 </a> )</li><li>PCR microtubes (<em>e.g.</em>, Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 72.737 </a> )</li><li>1.5 ml microtube (Deltalab, catalog number: <a class="blackNoUnder" target="_blank"> 200400P </a> )</li><li>50 ml centrifuge tube (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 352070 </a> )</li><li>Graduated pipettes 10 ml (Brand, catalog number: <a class="blackNoUnder" target="_blank"> 27011 </a> )</li><li>Petri dishes (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11309283 </a> )</li><li>Microscope slides (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 630-2099 </a> )</li><li>Cover slides (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 631-0125 </a> )</li><li>0.4 cm electroporation cuvettes (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1652088 </a> )</li><li>Syringe 50 ml (Omnifix, catalog number: <a class="blackNoUnder" target="_blank"> 4616502F </a> )</li><li>Sterile syringe filter w/0.45 Âµm cellulose acetate membrane (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 28145-481 </a> )</li><li>Filter paper</li></ol><br/><strong><em>Reagents</em></strong> <ol class="npwtwo"><li>PCR primers (Invitrogen), desalted, standard 25-50 nmol scale</li><li><em>Escherichia coli</em> DH5Î± (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 18265017 </a> )</li><li>D(+)-Glucose monohydrate (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 450740050 </a> )</li><li>D(+)-Sucrose (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131621.1211 </a> )</li><li>Peptone (Condalab, catalog number: <a class="blackNoUnder" target="_blank"> 1616.05 </a> )</li><li>Tryptone (Condalab, catalog number: <a class="blackNoUnder" target="_blank"> 1612.05 </a> )</li><li>Yeast extract (Condalab, catalog number: <a class="blackNoUnder" target="_blank"> 1702.05 </a> )</li><li>Malt extract (VWR, catalog number: <a class="blackNoUnder" target="_blank"> J873-500G </a> )</li><li>Myo-inositol 99% (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 87-89-8 </a> )</li><li>Soybean oil (Santa Cruz Biotechnology, catalog number: <a class="blackNoUnder" target="_blank"> 8001-22-7 </a> )<br/></li><li>Corn steep liquor (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 66071-94-1 </a> )</li><li>European bacteriological agar (Condalab, catalog number: <a class="blackNoUnder" target="_blank"> 1800.05 </a> )</li><li>Tris Base (Fisher BioReagents, catalog number: <a class="blackNoUnder" target="_blank"> BP152-500 </a> )</li><li>Boric acid (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131015 </a> )</li><li>EDTA (Ethylenediaminetetraacetic acid disodium salt 2-hydrate) (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131669 </a> )</li><li>Magnesium chloride hexahydrate (MgCl<sub>2</sub>Â·6H<sub>2</sub>O) (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 7791-18-6 </a> )</li><li>Sodium chloride (NaCl) (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131659.1214 </a> )</li><li>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) (Fisher BioReagents, CAS number: <a class="blackNoUnder" target="_blank"> 7778-77-0 </a> )</li><li>Potassium phosphate dibasic (K<sub>2</sub>HPO<sub>4</sub>) (Fisher BioReagents, CAS number: 7758-11-4)</li><li>Sodium carbonate anhydrous (Na<sub>2</sub>CO<sub>3</sub>) (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 131648.1211 </a> )</li><li>Sodium hydrogen carbonate (NaHCO<sub>3</sub>) (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 141638.121 </a> 0)</li><li>Triton X-100 (Acros Organics, catalog number: <a class="blackNoUnder" target="_blank"> 215680010 </a> )</li><li>DTT (dithiothreitol) (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0861 </a> )</li><li>PVP 40 (Polyvinylpyrrolidone) (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 9003-39-8 </a> )</li><li>Tween 20 (PanReac AppliChem, catalog number: <a class="blackNoUnder" target="_blank"> 142312.1611 </a> )</li><li>Bovine Serum Albumin (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 9048-46-8 </a> )</li><li>G-418 Disulphate (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 108321-42-2 </a> )</li><li>Kanamycin sulfate (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 25389-94-0 </a> )</li><li>Ampicillin sodium salt (Sigma-Aldrich, CAS number: <a class="blackNoUnder" target="_blank"> 69-52-3 </a> )</li><li>Zymolyase<sup>Â®</sup>, 20T (Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 07663-91 </a> )</li><li>FavorPrep Plasmid Extraction Mini Kit (Favorgen, catalog number: <a class="blackNoUnder" target="_blank"> FADPE 300 </a> )</li><li>DreamTaq Green PCR Master Mix (2x) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> K1082 </a> )</li><li>RedSafe Nucleic Acid Staining Solution (iNtRON Biotechnology, catalog number: <a class="blackNoUnder" target="_blank"> 21141 </a> )</li><li>Agarose D1 Low EEO (Condalab, catalog number: <a class="blackNoUnder" target="_blank"> 8010.01 </a> )</li><li>LB medium (see Recipes)</li><li>MA2 medium (see Recipes)</li><li>SPA medium (see Recipes)</li><li>STM buffer (see Recipes)</li><li>TBE 20x (see Recipes)</li><li>DNA extraction buffer (see Recipes)</li><li>Transformation buffer (see Recipes)</li><li>Potassium phosphate buffer (see Recipes)</li></ol><p class="pptt" id="biaoti34421">### Equipment</p><ol class="npwtwok"><li>Pipettes P2, P20, P200 and P1000 (<em>e.g.</em>, Gilson, PIPETMAN Classic<sup>TM</sup>, catalog numbers: <a class="blackNoUnder" target="_blank"> F144801 </a> , <a class="blackNoUnder" target="_blank"> F144056M </a> , <a class="blackNoUnder" target="_blank"> F144058M </a> , <a class="blackNoUnder" target="_blank"> F144059M </a> )</li><li>Motorized pipette filler (<em>e.g.</em>, Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 4430000018 </a> )</li><li>Sentino Magnetic Filter Funnels (Pall, catalog number: <a class="blackNoUnder" target="_blank"> 4273 </a> )</li><li>Sterile glass handle or glass beads</li><li>Erlenmeyer flasks 500 ml (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 10536-926 </a> )</li><li>Filtering flask 250 ml (Simax, catalog number: <a class="blackNoUnder" target="_blank"> PJH006 </a> )</li><li>Gene-Pulser (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> MicroPulser </a> )</li><li>Heat block (<em>e.g.</em>, Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5355 000.011 </a> )</li><li>Thermal Cycler (<em>e.g.</em>, Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 170-6700 </a> )</li><li>Microcentrifuge (<em>e.g.</em>, Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10236249 </a> )</li><li>Centrifuge (<em>e.g.</em>, Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5703000010 </a> , model: <a class="blackNoUnder" target="_blank"> 5702R </a> )</li><li>Power Source (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 700-0115 </a> , model: <a class="blackNoUnder" target="_blank"> 250 V </a> )</li><li>Gel Doc EZ System (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1708270 </a> )</li><li>Optical microscope</li><li>Vacuum pump</li><li>Microflow vertical laminar flow workstation</li><li>28 Â°C incubator</li><li>Shaker incubator 28 Â°C</li><li>Autoclave<br/></li></ol><p class="pptt" id="biaoti34422">### Software</p><ol class="npwtwok"><li>Image lab<sup>TM</sup> software 6.0.1. (Bio-Rad, catalog number: 1709690)</li></ol><p class="pptt" id="biaoti34423">### Procedure</p><p class="ppzz">The following protocol can be divided into three different sections (see Figure 2 for a schematic workflow):<br/>â¢Isolation of the CRISPR/Cas9 plasmid. The design and assembly of CRISPR/Cas9 plasmids has been previously described in JimÃ©nez <em>et al.</em>, 2019. See Figure 3A for a schematic design of the synthetic gRNA-dDNA.<br/>â¢ <em>A. gossypii</em> transformation. In this section, a detailed method for <em>A. gossypii</em> transformation is provided.<br/>â¢Isolation and screening of engineered homokaryotic strains. The methodology for the screening of positive clones depends on the phenotype produced by the corresponding genomic edition. Phenotypes such as colored mycelia can be followed directly by visual observation (JimÃ©nez <em>et al.</em>, 2019). Alternatively, the genomic editions can be confirmed by analytical PCR (see Figure 3B for a typical three primer PCR strategy) using the corresponding primer pair for each modification (JimÃ©nez <em>et al.</em>, 2019).</p><div style="white-space:nowrap;"><br/></div><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617184106_1180.jpg" width="166"/><br/><strong>Figure 2. Workflow of the CRISPR/Cas9 genomic edition in <em>A. gossypii</em></strong><br/><br/><img alt="" class="layerphoto" height="103" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617184143_7956.jpg" width="100"/><br/><strong>Figure 3. CRISPR/Cas9 design and analysis.</strong> A. Schematic design of the synthetic gRNA-dDNA. B. Analytical three primer PCR strategy.<br/><br/><em>Note: Perform all steps involving microbial culture under sterile environment in laminar airflow.</em><br/><ol class="npwone"><li>Isolate the CRISPR/Cas9 plasmid from the corresponding <em>E. coli</em> strain<ol class="npwtwo"><li>Inoculate the corresponding <em>E. coli</em> strain harboring the CRISPR/Cas9 plasmid in 4 ml of LB plus 50 Âµg/ml kanamycin and 100 Âµg/ml ampicillin.</li><li>Incubate at 37 Â°C in a shaker incubator overnight.</li><li>Collect the cells and extract the plasmid DNA using the FavorPrep Plasmid Extraction Mini Kit.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li><em>A. gossypii</em> transformation<br/><em>Note: Perform Steps B8 to B20 strictly on ice.</em> <ol class="npwtwo"><li>Inoculate 750 Âµl of spores (approx. 10<sup>8</sup> spores) of the target strain of <em>A. gossypii</em> in a flask with 100 ml of liquid MA2 medium.<br/><em>Note: A counting chamber such as Neubauer/Thoma can be used for calculation of spore number.</em> </li><li>Incubate at 28 Â°C for 10-12 h at 180 rpm.</li><li>After this time, check the status of the spores under a microscope. The spores must be in a germination state called â<em>germlings</em>â for the optimal transformation efficiency (Figure 4A) (JimÃ©nez <em>et al.</em>, 2005).</li><li>Collect the spores by vacuum filtration using a sterile paper filter.<br/><em>Note: Use an autoclave sterilized sentino magnetic filter funnel.</em> </li><li>Prepare the transformation buffer (see Recipes).</li><li>Introduce the paper filter with the spores in a tube with transformation buffer and resuspend the spores by inversion.<br/><em>Note: Remove the paper filter after resuspending the spores.</em> </li><li>Incubate the tube with the spore suspension at 28 Â°C for 30 min with agitation at 180 rpm.<br/><em>Note: Pre-chill liquid MA2 medium, STM buffer, electroporation cuvettes and Eppendorf tubes on ice.</em> </li><li>Centrifuge for 10 min at 3,000 <em>x g</em> at 4 Â°C.</li><li>Carefully discard the supernatant without resuspending the spores.</li><li>Gently wash the spores with 20 ml of ice-cold STM buffer.</li><li>Repeat Steps B8-B10.</li><li>Centrifuge for 10 min at 3,000 <em>x g</em> at 4 Â°C.</li><li>Resuspend the spores in 2 ml of ice-cold STM buffer and aliquot the spores in Eppendorf tubes.</li><li>Centrifuge for 1 min at 17,000 <em>x g</em> at 4 Â°C.</li><li>Discard the supernatant and resuspend the spores to a final volume of 400 Âµl with ice-cold STM buffer.</li><li>Mix 190 Âµl of the spore suspension and 5-10 Âµg of DNA (15-20 Âµl approx.) in an electroporation cuvette.<br/><em>Notes:</em><br/><em> a.Use cut pipette tips to avoid clogging of the spore suspension.</em><br/><em> b.For the negative control use the same volume of water instead of DNA.</em> </li><li>Use the following settings in the Gene-Pulser: 1,500 V, 200 W and 25 mF.</li><li>Pulse the mixture and immediately add 1 ml of ice-cold MA2 medium to resuspend the spores. Transfer the spores to a new Eppendorf.</li><li>Centrifuge for 1 min at 17,000 <em>x g</em> at 4 Â°C.</li><li>Discard 1 ml of the supernatant and plate 200 Âµl of the spores onto an MA2 agar plate.<br/><em>Note: Use a sterile glass handle to plate the spores.</em> </li><li>Incubate at 28 Â°C for 6 h for the regeneration of spores.</li><li>After this time, the transformation plates must be covered with an upper agarose layer containing G418. Add 6 ml of 0.4% agarose, previously melted at 95 Â°C and cooled to 42 Â°C, with 200 Âµl of G418 at 50 Âµg/ml (stock concentration).</li><li>Incubate the plates at 28 Â°C for 2 days.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Isolation and screening of primary heterokaryotic transformants<ol class="npwtwo"><li>Isolate G418-resistant heterokaryotic clones from the transformation plates using fresh MA2-G418 plates (Figures 4B and 4C). Incubate the heterokaryotic transformants at 28 Â°C for 2 days.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200617/20200617184216_6089.jpg" width="312"/><br/><strong>Figure 4. <em>A. gossypii</em> germlings and transformation isolates.</strong> A. Micrograph of <em>A. gossypii</em> germling. B. Transformation plate. C. Isolated heterokaryotic transformants.<br/><br/></li><li>Extraction of genomic DNA for analytical PCR of the heterokaryotic clones:<ol class="npwthree"><li>Prepare DNA extraction buffer (Recipe 6).</li><li>Pick approximately 2 mm<sup>2</sup> of mycelia from the isolated G418-resistant heterokaryotic clones and put into 20 Âµl of DNA extraction buffer.</li><li>Incubate at 100 Â°C for 10 min.</li><li>Centrifuge for 2 min at 17,000 <em>x g</em>.</li><li>Keep the supernatant at 4 Â°C for analytical PCR. The pellet can be discarded.<br/></li></ol><em>Notes:</em><br/><ol class="npwthree"><li><em>Perform a reaction for each primary heterokaryotic transformant.</em> </li><li><em>If possible, a three primer PCR (TP-PCR) strategy should be followed (Figure 3B) (JimÃ©nez </em><em>et al.</em><em>, 2019): design a pair of primers for the amplification of the same region in both edited and non-edited nuclei/genomes/DNAs. Additionally, design a third primer that exclusively aligns in edited genome templates for the amplification of the edited amplicon.</em> </li></ol></li><li>Analytical PCR.<br/><ol class="npwthree"><li>Set up the following reaction using PCR microtubes:<br/>15 Âµl of DreamTaq Green PCR Master Mix (2x)<br/>1 Âµl of each PCR primer (10 ÂµM)<br/>1 Âµl of the supernatant from DNA extraction (50-100 ng/Âµl)<br/>dH<sub>2</sub>O up to 30 Âµl</li><li>Set up the following conditions in the thermal cycler:<br/>A denaturation step at 95 Â°C for 3 min<br/>35 cycles of:<br/>  Denaturation: 95 Â°C for 30 s.<br/>  Annealing: 50-65 Â°C (adjust to the primers Tm) for 30 s.<br/>  Elongation: 72 Â°C for 1 minute (adjust to the amplicon length by 1 kb/min).<br/>Final elongation step at 72 Â°C for 10 min.<br/>Store at 4 Â°C.</li></ol></li><li>Run a 0.8% agarose gel electrophoresis with RedSafe (0.5 Âµg/ml).<br/><ol class="npwthree"><li>Use 5 Âµl of the PCR product.</li><li>Prepare TBE 1x Buffer from TBE 20x buffer (Recipe 5).</li><li>Expose the agarose gel under a UV transilluminator (<em>e.g.</em>, Gel Doc EZ System, Bio-Rad).</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Sporulation of the primary heterokaryotic positives transformants<br/><em>Note: The mycelium of </em><em>A. gossypii</em><em> is organized as multinucleated cells that are separated by septa along the hyphae. Only a limited number of nuclei within each syncytium are recipients of the transforming DNA. Sporulation of the primary heterokaryotic transformants is a required step for the isolation of homokaryotic clones, which are derived from uninucleated spores.</em> <ol class="npwtwo"><li>Spread mycelia from the positive G418-resistant clones onto SPA plates lacking G418 and incubate at 28 Â°C for 2 days. This step promotes the loss of the CRISPR/Cas9 plasmid, since episomic plasmids are not fully stable in <em>A. gossypii</em>.</li><li>After two days on SPA medium, confirm the presence of spores under a microscope.</li><li>Prepare a 1 mg/ml zymolyase-20T buffer to isolate the spores. Sterilize by filtration through a 0.45 Âµm filter.</li><li>Collect 1 cm<sup>2</sup> of mycelia and transfer into an Eppendorf tube with 1 ml of 1mg/ml of zymolyase buffer.</li><li>Incubate at 37 Â°C for 50 min with gentle agitation at 850 rpm.</li><li>After this time, check the mycelial lysis under a microscope.</li><li>Centrifuge for 2 min at 17,000 <em>x g</em>.</li><li>Discard the supernatant carefully by pipetting and wash the spores with 1 ml of 0.03% Triton X-100.</li><li>Repeat Steps D7 and D8.</li><li>Centrifuge for 2 min at 17,000 <em>x g</em>.</li><li>Discard the supernatant and resuspend the spores with 1 ml of MA2.</li><li>Finally, make appropriate dilutions of the spore suspension and plate approx. 10<sup>2</sup> spores onto MA2 solid medium lacking G418.<br/><em>Note: Use a sterile glass handle to plate.</em> </li><li>Incubate at 28 Â°C for 2 days.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Isolation and screening of homokaryotic transformants:<ol class="npwtwo"><li>Isolate homokaryotic clones using fresh MA2 plates lacking G418. Incubate the homokaryotic transformants 28 Â°C for 2 days. Use MA2 plates plus G418 to confirm the loss of the CRISPR/Cas9 plasmid.</li><li>Repeat Steps C2, C3 and C4 to verify the presence of the designed mutation in the genome.</li><li>Sequence the amplicons with appropriate primers to further confirm the presence of the desired genomic edition.</li></ol></li></ol><p class="pptt" id="biaoti34424">### Data analysis</p><p class="ppzz">The editing efficiency is estimated as the frequency of primary heterokaryotic transformants containing edited nuclei. For calculation of the editing efficiency, 10-30 heterokaryotic transformants are usually analyzed. On average, an efficiency of 60% is obtained using different sgRNA-dDNA combinations.</p><p class="pptt" id="biaoti34425">### Recipes</p><ol class="npwtwok"><li>LB medium<br/>1% Tryptone<br/>0.5% Yeast extract<br/>1% NaCl<br/>1.5% Agar (for solid media)<br/>Dissolve completely in distilled water and sterilize by autoclaving.<br/><em>Notes: </em><br/><ol class="npwthree"><li><em>Adjust pH to 7.2 before autoclaving.</em> </li><li><em>Add 50 Âµg/ml of kanamycin and 100 Âµg/ml ampicillin for selective medium after autoclaving.</em> </li></ol></li><li>MA2 medium<br/>2% Glucose<br/>2% Peptone<br/>0.2% Yeast extract<br/>0.06% Myo-inositol<br/>2% Agar (for solid media)<br/>Dissolve completely in distilled water and sterilize by autoclaving<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Adjust pH to 6.8 for solid media.</em> </li><li><em>Add 200 Âµg/ml of G418 for selective media after autoclaving.</em> </li></ol></li><li>SPA medium<br/>2% Corn steep liquor<br/>1% Glucose<br/>0.3% Soybean oil<br/>0.3% Yeast extract<br/>0.3% Malt extract<br/>4% Agar (for solid media)<br/>Dissolve completely in distilled water and sterilize by autoclaving<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Adjust pH to 6.8 for solid media.</em> </li><li><em>Add 200 Âµg/ml G418 for selective media after autoclaving.</em> </li></ol></li><li>STM buffer<br/>270 mM sucrose<br/>10 mM Tris base pH 7.5<br/>1 mM MgCl<sub>2</sub><br/>Dissolve completely in distilled water and sterilize by autoclaving</li><li>TBE buffer (20x final concentration)<br/>90 mM Tris base<br/>90 mM Boric acid<br/>2 mM EDTA<br/>Dissolve completely and sterilize by autoclaving</li><li>DNA extraction buffer<br/>2% PVP 40<br/>0.2% BSA<br/>0.05 M Tween 20<br/>0.05 M sodium carbonate buffer pH 9.6<br/><em>Note: For the sodium carbonate buffer pH 9.6, add 0.45 ml solution A and 0.182 ml solution B in 9.36 ml of distilled water.</em><br/><em> Solution A = Sodium hydrogen carbonate (NaHCO</em><sub><em>3</em></sub><em>) 0.1 M (Add 0.84 g in 10 ml of dH</em><sub><em>2</em></sub><em>O).</em><br/><em> Solution B = Sodium carbonate anhydrous (Na</em><sub><em>2</em></sub><em>CO</em><sub><em>3</em></sub><em>) 0.1 M (Add 1.06 g in 10 ml of dH</em><sub><em>2</em></sub><em>O).</em><br/><em>Store the DNA extraction buffer at -20 Â°C.</em> </li><li>Transformation buffer<br/>0.5 ml of DTT 1M<br/>1 ml of potassium phosphate buffer (see below)<br/>18.5 ml of distilled water<br/>Mix and sterilize by filtration through a 0.45 Âµm filter<br/><em>Note: Prepare 20 ml of transformation buffer for each strain of transformation.</em> </li><li>Potassium phosphate buffer<br/><ol class="npwthree"><li>Mix potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) 1 M with potassium phosphate dibasic (K<sub>2</sub>HPO<sub>4</sub>) 1 M in equal volumes</li><li>Adjust the pH to 7.0</li><li>Sterilize by autoclaving</li></ol></li></ol><p class="pptt" id="biaoti34426">### Acknowledgments</p><p class="ppzz">This work was financed by grants from the Spanish Ministerio de EconomÃ­a y Competitividad (BIO2017-88435-R) and Junta de Castilla y LeÃ³n (SA016P17) to JLR and AJ. GM-F was recipient of an FPI predoctoral contract from the Spanish Ministerio de EconomÃ­a y Competitividad. We thank MarÃ­a Dolores SÃ¡nchez and Silvia DomÃ­nguez for excellent technical help.<br/>  This protocol was derived from the following original research paper (JimÃ©nez <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti34427">### Competing interests</p><p class="ppzz">The authors declare no financial or commercial conflict of interest.</p><p class="pptt" id="biaoti34428">### References</p><ol class="npwtwok"><li>Bao, Z., Xiao, H., Liang, J., Zhang, L., Xiong, X., Sun, N., Si, T. and Zhao, H. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25207793" target="_blank">Homology-integrated CRISPR-Cas (HI-CRISPR) system for one-step multigene disruption in <em>Saccharomyces cerevisiae</em>.</a> <em>ACS Synth Biol</em> 4(5): 585-594. </li><li>DiCarlo, J. E., Norville, J. E., Mali, P., Rios, X., Aach, J. and Church, G. M. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23460208" target="_blank">Genome engineering in <em>Saccharomyces cerevisiae</em> using CRISPR-Cas systems.</a> <em>Nucleic Acids Res</em> 41(7): 4336-4343.</li><li>Horvath, P. and Barrangou, R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20056882" target="_blank">CRISPR/Cas, the immune system of bacteria and archaea.</a> <em>Science</em> 327(5962): 167-170. </li><li>Jinek, M., Chylinski, K., Fonfara, I., Hauer, M., Doudna, J. A. and Charpentier, E. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22745249" target="_blank">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity.</a> <em>Science</em> 337(6096): 816-821.</li><li>Jakociunas, T., Bonde, I., Herrgard, M., Harrison, S. J., Kristensen, M., Pedersen, L. E., Jensen, M. K. and Keasling, J. D. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25638686" target="_blank">Multiplex metabolic pathway engineering using CRISPR/Cas9 in <em>Saccharomyces cerevisiae</em>.</a> <em>Metab Eng</em> 28: 213-222. </li><li>JimÃ©nez, A., MuÃ±oz-FernÃ¡ndez, G., Ledesma-Amaro, R., Buey, R. M. and Revuelta, J. L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31055883" target="_blank">One-vector CRISPR/Cas9 genome engineering of the industrial fungus <em>Ashbya gossypii</em>.</a> <em>Microb Biotechnol</em> 12(6): 1293-1301. </li><li>JimÃ©nez, A., Santos, M. A., Pompejus, M. and Revuelta, J. L. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16204483" target="_blank">Metabolic engineering of the purine pathway for riboflavin production in <em>Ashbya gossypii</em>.</a> <em>Appl Environ Microbiol</em> 71(10): 5743-5751.</li><li>Pattanayak, V., Lin, S., Guilinger, J. P., Ma, E., Doudna, J. A. and Liu, D. R. (2013). <a href="https://www.nature.com/articles/nbt.2673" target="_blank">High-throughput profiling of off-target DNA cleavage reveals RNA-programmed Cas9 nuclease specificity.</a> <em>Nat Biotech</em> 31(9): 839-843.</li><li>Ran, F. A., Hsu, P. D., Wright, J., Agarwala, V., Scott, D. A. and Zhang, F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24157548" target="_blank">Genome engineering using the CRISPR-Cas9 system.</a> <em>Nat Protoc</em> 8(11): 2281-2308. <br/></li></ol>