<body>
            <h1>Maintenance and Quantitative Phenotyping of the Oomycete-plant Model Pathosystem <em>Hyaloperonospora arabidopsidis</em>â<em>Arabidopsis</em></h1>
            <p><strong>Authors:</strong> Fang-Yu  Hwu, Martin Parniske</p>
            <h2>Abstract</h2>
            <p>The interaction between the host plant <em>Arabidopsis thaliana (Arabidopsis)</em> and the oomycete <em>Hyaloperonospora arabidopsidis</em> (<em>Hpa</em>) is an established model system for the study of an obligate biotrophic downy mildew interaction. The evaluation of the developmental success of <em>Hpa</em> is often based on the quantification of reproductive structures that are formed on the surface of leaves, such as the sporangiophores or the conidiospores they carry. However, the structural basis of this interaction lies within the plant tissue and, in particular, the haustoria that form inside plant cells. Therefore, valuable additional information about the performance and compatibility of the downy mildew interaction can be gained by light microscopical inspection of the hyphal and haustorial shape inside the plant tissue and within plant cells respectively. Here we describe a protocol for the visualization and quantification of morphological phenotypes inside the plant. While we focus specifically on the quantification of haustorial shape variants, the protocol can easily be adapted for the quantification of other morphological features such as hyphal deformations, or oogonia frequency. By including and refining already existing protocols from a variety of sources, we assembled the entire experimental pipeline for the <em>Arabidopsis Hpa</em> bioassay to provide a practical guide for the initial setup of this system in the laboratory. This pipeline includes the following steps: A) growing <em>Arabidopsis</em>, B) <em>Hpa</em> propagation and strain maintainance C) <em>Hpa</em> inoculation and incubation D) staining of plant tissues for visualization of the pathogen and E) an introduction of the Keyence VHX microscope and Fiji plugin of ImageJ for the quantification of structures of interest. While described here for <em>Arabidopsis</em> and <em>Hpa</em>, the protocol steps B-E should be easily adjustable for the study of other plant-oomycete pathosystems.</p>
            <p><strong>Keywords:</strong> <em>Arabidopsis</em>, <em>Hyaloperonospora arabidopsidis</em>, Trypan blue staining, Haustoria quantification, Multilobed haustoria, Plant-microbe interaction</p>
            <h2>Background</h2><p>Downy mildews are a group of phytopathogenic oomycetes characterized by branched sporangiophores that protrude through leaf stomata, giving the appearance of a white to grey âdownâ on the affected tissues (Agrios, 2005). Hundreds of Downy mildew species have been described on a wide variety of host plants including monocotyledon and dicotyledon crops (Callan and Carris, 2004). They are typically obligate biotrophs, meaning that they have not yet been successfully cultured outside the plant host with completion of their life cycle. A model system to study the molecular and genetic determinants underlying the downy mildew disease is the interaction between the host plant <em>Arabidopsis thaliana</em> and its oomycetal pathogen <em>Hyaloperonospora arabidopsidis</em> (<em>Hpa</em>). Downy mildew oomycetes typically form a structure called haustorium inside the plant host cell. The haustorium is surrounded by a plant cell membrane called extrahaustorial membrane (EHM) that prevents direct contact of the oomycete with the plant cytoplasm. While the hypothetical role of the haustorium in nutrient uptake from the plant host remains to be confirmed, it probably represents a platform for delivering oomycetal effectors to plants (Judelson and Ah-Fong, 2019) which enable the establishment and maintenance of a biotrophic interaction with the host. It is therefore not surprising that several proteins required for the resistance against <em>Hpa</em> are localized to the EHM (Wang <em>et al.</em>, 2009; Caillaud <em>et al.</em>, 2014). Despite the obvious relevance of the haustorium for the downy mildew interaction, surprisingly few studies evaluated the interaction by phenotyping the haustoria. A method for the inoculation of <em>Arabidopsis</em> by <em>Hpa</em> was described by Asai <em>et al.</em> (2015). Here we expanded this protocol resulting in a step-by-step guide for the study of this pathosystem in the lab. This includes details on how to sow and grow <em>Arabidopsis</em>, how to propagate <em>Hpa</em>, and specific details on how to systematically record structural features of this interaction with the Keyence VHX digital microscope, and quantify them with the image analysis software Fiji, here exemplified by the quantification of haustoria-shape variation. Our step-by-step guide for the microscopic analysis of the <em>Hpa-Arabidopsis</em> interaction should be easily adaptable for the use not only in other downy mildew interactions in host species other than <em>Arabidopsis</em> but also other hyphal pathogens other than oomycetes. The steps that most likely need attention when moving to other species combinations are highlighted in the protocol below.</p>
        </body>

<p class="pptt" id="biaoti34431">## Materials and Reagents</p><ol class="npwtwok"><li>Falcon<sup>Â®</sup> tube (50 ml) * 2 (Every similar-sized version will do)</li><li>2 ml Eppendorf<sup>Â®</sup> tubes (Every similar version will do)</li><li>Miracloth<sup>Â®</sup> (8 cm x 8 cm) (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> 475855-1R </a> )</li><li><em>Arabidopsis thaliana</em> Col-0, wildtype and <em>shrk1</em> x <em>shrk2</em> [<em>shrk1</em> x <em>shrk2</em> produces multilobed haustoria at higher frequency than the wildtype (Ried <em>et al.</em>, 2019)]</li><li><em>Hpa</em> Noco2 (Col-0 is susceptible to the Hpa strain Noco2)</li><li>Toothpick with pointed ends (see Figure 1)</li><li>A white sheet of paper (A5 size, ISO 216)</li><li>Several 6.5 cm x 7 cm x 7 cm pots with soil</li><li>Plastic tray with a transparent lid [38 cm x 24 cm x 18 cm (height includes lid)]<br/><em>Note: The one we used is not produced anymore. Connex Indoor greenhouse, FLOR79045, is a similar version. Every similar sized box from a gardening supplier with a transparent lid should be suitable.</em> </li><li>Soil (Stender GmbH, A210)</li><li>Fertilizer (WUXAL<sup>Â®</sup> Super 8-8-6)</li><li>Counting chamber (Hemocytometer) (Marienfeld, catalog number: 0<a class="blackNoUnder" target="_blank"> 640110 </a> )</li><li>Spray bottle<br/><em>Note: A small (20-30 ml volume) spray bottle from the cosmetic store which can produce a fine mist/aerosol is suitable for inoculation. <strong>Important! </strong>Use the same type for all your experiments and use the same distance and spray time to apply similar spore numbers from experiment to experiment. This may be critical for applying a similar infection pressure in each independent biological repeat.</em> </li><li>Ethanol 70% (for sterilization, ethanol of technical grade is sufficient)</li><li>Sterile water</li><li>Phenol (Roth, catalog number: 00<a class="blackNoUnder" target="_blank"> 40.1 </a> , any brand will do)</li><li>Lactic acid (Roth, catalog number: <a class="blackNoUnder" target="_blank"> 8460.1 </a> , any brand will do)</li><li>Trypan blue (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> T6146 </a> , any brand will do)</li><li>35% (v/v) glycerol/water (Roth, catalog number: <a class="blackNoUnder" target="_blank"> 7530.4 </a> , any brand will do)</li><li>0.01% trypan blue solution (see Recipes)</li><li>Saturated chloral hydrate solution (2.5 g/ml, see Recipes)<br/></li></ol><p class="pptt" id="biaoti34432">### Equipment</p><ol class="npwtwok"><li>Scissors with a small and pointy front (Hammacher, HSB 544-09, any version similar to this should be suitable for cutting off infected leaves).</li><li>Vortex (Vortex-Genie<sup>Â®</sup> 2, one with similar function will do)</li><li>Growth chamber (Binder, model: <a class="blackNoUnder" target="_blank"> KBWF 720 </a> )</li><li>Eppendorf ThermoMixer C (or other brand. Any temperature controlled Eppendorf<sup>Â®</sup> tube shaker with heating function will do)</li><li>VHX-6000 digital microscope (Keyence, Osaka, Japan)<br/><em>Note: The advantage of using the VHX digital microscope is that it can record a large sample area by stitching together individual pictures in x, y and z directions. Any other bright field microscope with a 10x objective will be also suitable but perhaps less comfortable to use for this particular task.</em><br/></li></ol><p class="pptt" id="biaoti34433">### Software</p><ol class="npwtwok"><li>Fiji (Schindelin <em>et al.</em>, 2012, <a href="https://imagej.net/Fiji" target="_blank">https://imagej.net/Fiji</a>)</li><li>Cell counter (A plugin for the Fiji software, Kurt De Vos, <a href="https://imagej.nih.gov/ij/plugins/cell-counter.html" target="_blank">https://imagej.nih.gov/ij/plugins/cell-counter.html</a>)<br/></li></ol><p class="pptt" id="biaoti34434">### Procedure</p><ol class="npwone"><li>Growing of <em>Arabidopsis</em> <ol class="npwtwo"><li>Prepare <em>Arabidopsis</em> seedlings for the multilobed haustoria observation<ol class="npwthree"><li>Stratify all the testing seeds at 4 Â°C overnight once after you harvest the seeds.<br/><em>Note: Stratification can synchronize the growing of the plants.</em> </li><li>Spread the amount of the seeds that you need onto a white paper (5-9 seeds for one pot) (Figure 1A).</li><li>Moisture the tip of the toothpick by dipping it into water (Figure 1B).</li><li>Distribute 5-9 seeds evenly in one 6.5 cm x 7 cm x 7 cm pot with the now sticky toothpick tip to pick up one seed each time (Figures 1C-1D).<br/><em>Notes:</em> <ol class="npwfour"><li><em>Donât grow more than 12 plants in this pot size. A crowded pot will lead to an uneven infection because overlapping leaves will cover each other.</em> </li><li><em>Do not cover the seeds with the soil! Put the seeds directly on the surface of the soil.</em> </li></ol></li><li>Grow plants at 22 Â°C, 16 h light (100 Î¼mol/m<sup>2</sup>/s)/ 8 h dark, 60% relative humidity for 2 weeks. The plants were watered twice a week with the 1/1000 diluted fertilizer. Figure 1F shows 2-week-old plants grown under the conditions and pot size indicated.</li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611024904_6762.jpg" width="300"/><br/><strong>Figure 1. Procedure for growing and infecting the <em>Arabidopsis</em>.</strong> A. Spreading seeds evenly on a white sheet. B. Moisture the tip of the toothpick. C. Pick up one seed at once with a moisture tip. D. Put the seed on the surface of the soil. E. Pick up seeds by your index finger. F. 2-week-old <em>Arabidopsis</em>. G. The 2-week-old <em>Arabidopsis</em>âs leaves are evenly filled with the droplets. Scale bar = 0.8 cm. H. The 7 dpi <em>Hpa</em> infected <em>Arabidopsis</em>. The arrows show the first pair of leaves to collect. Scale bar = 0.8 cm.<br/><br/></li><li>Prepare <em>Arabidopsis</em> seedlings for <em>Hpa</em> propagation<ol class="npwthree"><li>Stratify all the testing seeds at 4 Â°C overnight once after you harvest the seeds.</li><li>Spread the amount of the seeds that you need onto a white paper (about 50-100 seeds in one pot).</li><li>Collect the seeds with your index finger and spread them onto the soil evenly like spreading salt on food (Figure 1E).<br/><em>Note: Do not cover the seeds with the soil! Put the seeds directly on the surface of the soil.</em> </li><li>Grow plants at 22 Â°C, 16 h light (100 Î¼mol/m<sup>2</sup>/s)/8 h dark, 60% relative humidity for 2 weeks.<br/><br/></li></ol></li></ol></li><li><em>Hpa</em> propagation and preservation<ol class="npwtwo"><li><em>Hpa</em> propagation<ol class="npwthree"><li>Sterilize scissors with 70% ethanol.</li><li>Clean and decontaminate the plastic tray and lid first with soap (detergent) then with 70% ethanol. Let them dry before further use.</li><li>Place the pots which are prepared in Step A2 in the plastic tray and add 250 ml sterilized water in each tray to keep the humidity for the later step.</li><li>Carefully open the tray of the 7 dpi <em>Hpa</em> infected plants and lift the cover slowly in a defined area for the <em>Hpa</em> infection. Reduce air movement around the plants to a minimum.<br/><em>Note: If you start the Hpa infection for the very first time in your lab, the Hpa source can be the pots of 7 dpi Arabidopsis or the freezed material (see Step B2) from the provider.</em> </li><li>Harvest the aerial parts of the infected plants covered with spores (Figure 2A) into a 50 ml Falcon<sup>Â®</sup> tube.<br/><em>Note: The infected Arabidopsis seedlings should be loosely placed in the tube to let the spores easily release to the sterilized water (see Figure 1B in Asai et al., 2015,). Usually, 2-3 full pots of the infected Arabidopsis are used.</em> </li><li>Add 15 ml of sterile water in the Falcon<sup>Â®</sup> tube and vortex it at the full speed (3,200 rpm) for 10 s.</li><li>Filter the suspension using one layer of Miracloth.</li><li>Measure the concentration of the conidiospores in the suspension by using the counting chamber (Hemocytometer). Adjust the concentration to 10<sup>5</sup> spores/ml water. If the spores are not enough, repeat Steps B1d-B1f for the rest of the materials.</li><li>Spray the 2-week-old <em>Arabidopsis</em> with the filtered suspension until the leaf surface is evenly filled with the droplets (Figure 1G).</li><li>Cover the plastic tray with the transparent lid and seal the openings with transparent tape to maintain high humidity (90%-100%).</li><li>Incubate the infected plants in a growth chamber at 18 Â°C, 16 h light (80 Î¼mol/m<sup>2</sup>/s*, 70 Î¼mol/m<sup>2</sup>/s under the transparent lid)/8 h dark for 6 or 7 days.<br/><em>Notes: <ol class="npwthree"><li>*This is the actual measurement from the Binder chamber. The plants grow in the walk in chamber with the light intensity 100 Î¼mol/m<sup>2</sup>/s, but the incubation has been done in the Binder chamber.</li><li>For the propagation and the collection of the fresh conidiospores, Hpa should be maintained on Arabidopsis in a 6- or 7-day-regime. In other words, infect the plants every 6 or 7 days. Usually, Hpa will start to sporulate on 5 dpi. 6 dpi or 7 dpi are the better days for the conidiospores collection.</li></ol></em> </li></ol></li><li><em>Hpa</em> preservation and regeneration<ol class="npwthree"><li>Follow Steps B1a-B1e to collect the infected plants into a 50 ml Falcon<sup>Â®</sup> tube.</li><li>Preserve the infected plants at -80 Â°C.<br/><em>Note: The Hpa conidiospores can be stored in this condition for at least 1 year or longer.</em> </li><li>For the regeneration, take out the preserved infected leaves from the -80 Â°C and put the tube on ice for 10 min.</li><li>After 10 min, add 15 ml of sterile water in the Falcon<sup>Â®</sup> tube and vortex for 10 s at full speed.</li><li>Filter the suspension using one layer of Miracloth.</li><li>Spray the <em>Arabidopsis</em> (prepared from Step A2) with the filtered suspension until the leaf surface is evenly filled with the droplet (Figure 1G).<br/><em>Note: Since the viability of the frozen conidiospores will be lower, we donât adjust (dilute) the concentration for the regeneration.</em><br/><br/></li></ol></li></ol></li><li><em>Hpa</em> inoculation and incubation<ol class="npwtwo"><li>Inoculation<ol class="npwthree"><li>Randomly place different genotypes of <em>Arabidopsis</em> pots prepared from Step A1 in the tray.</li><li>Infect the plants as in Step B1.</li></ol></li><li>Incubation <br/>Incubate the infected plants in a growth chamber at 18 Â°C, 16 h light (80 Î¼mol/m<sup>2</sup>/s, 70 Î¼mol/m<sup>2</sup>/s under the transparent lid)/ 8 h dark until the collection day.<br/><em>Note: Under these conditions, in the Arabidopsis thaliana Col-0 x Hpa Noco2 interaction, the multilobed haustoria start to accumulate on 5 dpi (Figure 3) and reach to a higher proportion at later infection stages (7 dpi-8 dpi) in both wild type and the shrk1 x shrk2 mutant.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Staining of plant tissues for visualization of the pathogen<ol class="npwtwo"><li>Harvest one of the infected first pair of true leaves from each seedling (5-8 biological repeats for each genotype= 5-8 plants) (Figure 1H).<br/><em>Note: For other oomycete-plant pathosystems, the choice of leaf may vary. If nothing else matters, pick the youngest infected leaves as they are typically the smallest and easiest to stain.</em> </li><li>Incubate them in the trypan blue solution which cover the leaves in 2 ml Eppendorf<sup>Â®</sup> tubes at 95 Â°C for 3-10 min or until the tissues completely soaked up the solution and incubate them at room temperature overnight.<br/><em>Notes: <ol class="npwthree"><li>To avoid the splitting of the trypan blue while cooking the samples, donât put trypan blue solution excess than 1 ml in the 2 ml Eppendorf<sup>Â®</sup>.</li><li>Do not heat them longer than 30 min, the tissues will become too soft for further steps. However, the room temperature overnight process can be extended if the tissues are not stained enough. Roll the Eppendorf tube a bit to see whether the tissue is well soaked by the trypan blue. The ready tissues should be dark blue (Figure 2B).</li><li>For the other oomycete pathosystems, heat and incubate the samples until the tissues completely soaked up the solution. The required incubation time may increase with leaf and cell wall thickness.</li></ol></em> </li><li>Replace the solution with the saturated chloral hydrate solution and leave overnight until the tissues look clear (Figure 2C).<br/><em>Note: If the tissues are not yet clear, replace the chloral hydrate supernatant with a fresh aliquot and incubate until the tissues become clear.</em> </li><li>Replace the chloral hydrate with 35% glycerol.<br/><em>Note: The samples can be stored in 35% glycerol for long time (at least 3 months).</em> </li><li>Place the tissues on microscopy slides with 35% glycerol for further observation under the Keyence VHX digital microscope. Use a cover slip to prevent the sample from drying out.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611024944_8886.jpg" width="300"/><br/><strong>Figure 2. <em>Hpa</em> infected <em>Arabidopsis</em> and the trypan blue stained leaf.</strong> A. The non-infected mock leaf (left) and the 7 dpi <em>Hpa</em> infected leaf (right). Scale bar = 2 mm. B. The trypan blue-stained leaf before the de-staining process. Scale bar = 1 mm. C. The de-stained trypan blue-treated leaf. The hyphae can be clearly seen under the microscope or by the eyes. Scale bar = 1 mm.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611024959_1857.jpg" width="300"/><br/><strong>Figure 3. The <em>Hpa</em> haustoria in the Col-0 and the <em>shrk1</em> x <em>shrk2</em> leaf.</strong> The pictures were taken from the 5 dpi leaves by the 1,000x objective lens with the VHX digital microscope. The asterisks indicate the multilobed haustoria. Scale bars = 25 Âµm.<br/></li></ol> </li><li>An introduction of the Keyence VHX microscope and Fiji plugin of ImageJ for the quantification of structures of interest<ol class="npwtwo"><li>The using of the Keyence VHX digital microscope<ol class="npwthree"><li>Turn on the microscope and follow the instruction of the software to position the sample in focus.<br/><em>Note: For the example showing below, we used 500x objective lens to visualize the samples.</em> </li><li>Use the âStitchingâ function in the software to acquire pictures that contain larger areas.</li><li>Use the âXY lengthâ function in the âStitchingâ window to set the area you want to include them in one picture (Figure 4, X direction 1,000 Âµm, Y direction 1,000 Âµm). The yellow square will also show you the area microscope will record this time.<br/><em>Note: X direction 1,000 Âµm and Y direction 1,000 Âµm is a suggested area size that works well for Arabidopsis leaves at the indicated dpi. You can change the area according to the biological system requirements, but always maintain one consistent size within the same experiment.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611025018_2454.jpg" width="300"/><br/><strong>Figure 4. The Stitching function in the VHX digital microscope software.</strong> The yellow square shows you the area microscope will record this time. The picture in the yellow square shows the position of the area which is displayed by the live imaging window now.<br/><br/></li><li>Take pictures from 5 different and not overlapping areas from one leaf.<br/><em>Note: Take pictures continually by using the stitching function. It will show you the area youâve just imaged. This guide should help to avoid overlaps between samples (Figure 5).</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611025031_3477.jpg" width="300"/><br/><strong>Figure 5. The Stitching function helps to avoid overlaps between samples.</strong> The yellow square shows us the area just taken. This area should be excluded from additional stitchings from the same leaf to avoid overlapping samples.<br/><br/></li></ol></li><li>Quantification of multilobed and round-shaped haustoria by the Fiji plugin: cell counter<br/><em>Note: For a guide how to distinguish round-shaped (ânormalâ) and multi-lobed haustoria, see: Ried et al., 2019, Figure 2 or the figures taken by the VHX digital microscope in this protocol: Figure 3.</em> <ol class="npwthree"><li>Use the âcell counterâ plugin in the Fiji package to count and record the haustoria shape.</li><li>Open the picture in Fiji, click on the window of the picture and then start the âcell counterâ plugin. Press âInitializeâ to activate the picture you want to quantify (Figure 6).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611025051_0840.jpg" width="300"/><br/><strong>Figure 6. The plugin âcell counterâ in the Fiji package.</strong> Initialization of the picture. Scale bar =100 Î¼m.<br/><br/></li><li>Define the name at the âCountersâ (Figure 6 above, round-shaped (normal) haustoria; multilobed haustoria).</li><li>Label them with a simple click beside the haustoria you want to label. Different types will then be labeled in different colors (Figure 7, normal haustoria in blue, multilobed haustoria in red).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611025104_3028.jpg" width="300"/><br/><strong>Figure 7. Use the plugin âcell counterâ to label the haustoria.</strong> Label the haustoria by a simple click beside the haustoria you want to label. Scale bar = 100 Î¼m.<br/><em>Note: Label only the haustoria which are in focus. Ignore the ones for which the shape is difficult to judge.<br/><br/></em> </li><li>You can change the labeling colors by clicking on the âoptionsâ (Figure 8).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611025117_4739.jpg" width="300"/><br/><strong>Figure 8. Change the labeling colors.</strong> Big ranges of the labeling colors can be chosen from the option window by right click on the colors. Scale bar = 100 Î¼m.<br/><br/></li><li>Pick up 2 or more branches from one picture to count the proportion of the multilobed haustoria (Figure 9). At least 10 branches in total should be counted in one biological repeat (one leaf).<br/><br/></li><li>You can save the labels with the âSave Markersâ function.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611025130_3178.jpg" width="300"/><br/><strong>Figure 9. The labeling of the Haustoria.</strong> A. Two branches from one picture. Scale bar = 100 Âµm. B and C. The magnification of the two branches. Scale bar = 50 Âµm. D. Haustoria labeled with different colors, green: normal haustoria, magenta: multilobed haustoria. Scale bar = 50 Âµm.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti34435">### Recipes</p><ol class="npwtwok"><li>0.01% trypan blue solution<br/>10 ml lactic acid*<br/>10 ml glycerol<br/>10 g phenol*<br/>10 ml sterile water<br/>10 mg trypan blue<br/><em>Note: Handle the materials labeled with asterisk cautiously in the chemical hood. Trypan blue is light sensitive. The solution should be stored in a brown bottle, it can be reused at least two times.</em> </li><li>Chloral hydrate solution (2.5 g/ml)<br/>Add 40 ml water to 100 g chloral hydrate in a 100 ml bottle<br/><em>Note: 100 g chloral hydrate will already occupy almost the entire space of a 100 ml bottle, therefore; don't put more than this amount in a 100 ml bottle.</em><br/></li></ol><p class="pptt" id="biaoti34436">### Acknowledgments</p><p class="ppzz">The initial establishment of the <em>Hpa x Arabidopsis</em> pathosystem in the authorâs laboratory was largely based on <em>Hpa</em> genotypes and advise given by the team of Jane Parker, while still working at The Sainsbury Laboratory in Norwich. The Steps B1 and D described in this protocol are similar to the one described by Asai <em>et al.</em> (2015). Procedures A-D were used as described in Ried <em>et al.</em> (2019). MP acknowledges funding by the Deutsche Forschungsgemeinschaft (DFG) and the European Research Council (ERC).</p><p class="pptt" id="biaoti34437">### Competing interests</p><p class="ppzz">The authors have no competing interests to declare.</p><p class="pptt" id="biaoti34438">### References</p><ol class="npwtwok"><li>Asai, S., Shirasu, K. and Jones, J. D. G. (2015). <a href="https://doi.org/10.21769/BioProtoc.1627" target="_blank"><em>Hyaloperonospora arabidopsidis</em> (Downy Mildew) Infection Assay in <em>Arabidopsis</em>.</a> <em>Bio-protocol</em> 5(20): e1627.</li><li>Caillaud, M. C., Wirthmueller, L., Sklenar, J., Findlay, K., Piquerez, S. J., Jones, A. M., Robatzek, S., Jones, J. D. and Faulkner, C. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25393742" target="_blank">The plasmodesmal protein PDLP1 localises to haustoria-associated membranes during downy mildew infection and regulates callose deposition.</a> <em>PLoS Pathog</em> 10(10): e1004496.</li><li>Callan, B. and Carris, L. (2004). Biodiversity of Fungi: Inventory and Monitoring Methods. Elsevier Inc. pp: 113-114</li><li>Judelson, H. S. and Ah-Fong, A. M. V. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30538168" target="_blank">Exchanges at the Plant-Oomycete Interface That Influence Disease.</a> <em>Plant Physiol</em> 179(4): 1198-1211. </li><li>Agrios, N. G. (2005). Plant Pathology. 5th edition. Elsevier Inc. pp. 427-433.</li><li>Ried, M. K., Banhara, A. and Hwu, F. Y., Binder, A., Gust, A. A., HÃ¶fle, C., HÃ¼ckelhoven, R., NÃ¼rnberger, T., Parniske, M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31299058" target="_blank">A set of <em>Arabidopsis</em> genes involved in the accommodation of the downy mildew pathogen <em>Hyaloperonospora arabidopsidis</em>.</a> <em>PLoS Pathog</em> 15(7): e1007747.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682. </li><li>Wang, W., Wen, Y., Berkey, R. and Xiao, S. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19749153" target="_blank">Specific targeting of the <em>Arabidopsis</em> resistance protein RPW8.2 to the interfacial membrane encasing the fungal Haustorium renders broad-spectrum resistance to powdery mildew.</a> <em>Plant Cell</em> 21(9): 2898-2913. <br/></li></ol>