<body>
            <h1>Identification of Buffer Conditions for Optimal Thermostability and Solubility of Herpesviral Protein UL37 Using the Thermofluor Assay</h1>
            <p><strong>Authors:</strong> Andrea L. Koenigsberg, Jared D. Pitts, Ekaterina E. Heldwein</p>
            <h2>Abstract</h2>
            <p>Structural and biochemical studies of proteins require high amounts of stable, purified proteins. Protein stability often depends on the buffer composition, which includes pH and concentration of salts or other solutes such as glycerol, hence an efficient method for identifying optimal buffer conditions for stability would minimize time and resources used for protein purification and further studies. This protocol describes the use of the Thermofluor assay, in combination with a custom 24-condition screen, to identify buffer conditions that increase protein thermostability, using the conserved herpesviral protein UL37 as an example. Detailed instructions on screen conditions, running the Thermofluor MATLAB script, and analyzing the data are provided. In comparison to circular dichroism (CD), the buffer screen in combination with Thermofluor assay provides a faster and more informative method to analyze protein thermostability.</p>
            <p><strong>Keywords:</strong> Thermofluor, Protein thermostability, Protein solubility, Differential scanning fluorescence, Buffer optimization, Viral protein</p>
            <h2>Background</h2><p>Conserved across herpesviruses, the protein UL37 has multiple functions in the viral lifecycle and is required for efficient viral replication. As a component of viral tegumentâa layer sandwiched between the genome-containing capsid and the lipid envelopeâUL37 is not only necessary for viral assembly (Desai <em>et al.</em>, 2001; Jambunathan <em>et al.</em>, 2014), but is also required for efficient capsid trafficking (Leege <em>et al.</em>, 2009), neuroinvasion (Richards <em>et al.</em>, 2017), and counteraction of the host innate immune response (Liu <em>et al.</em>, 2008; Zhao <em>et al.</em>, 2016). A better understanding of the nature of its multifunctionality requires detailed knowledge of its structure and biochemical properties, which, in turn requires well-behaved purified protein.<br/><br/>Initial preparations of different UL37 constructs varied in solubility, monodispersity, and yield of purified protein, making protein preparations inconsistent and further characterizations irreproducible. Since buffer optimization is a straightforward approach to addressing these concerns, we developed a 24-condition screen that varies buffering agents, pH, and concentrations of NaCl and glycerol and used it in combination with a Thermofluor assay (Phillips <em>et al.</em>, 2011) to identify buffer conditions that maximize thermostability. The N-terminal and C-terminal halves of UL37 had variable average thermal stabilities and optimal buffer conditions (Figure 1, Table 1). Yet in all cases, conditions that increased protein stability also improved protein solubility, ultimately increasing the yields of purified protein, which facilitated downstream biochemical characterization.<br/><br/>The Thermofluor assay has many advantages over circular dichroism (CD), which is also used to monitor protein stability (Greenfield, 2006). First, the CD can only test one condition at a time whereas the Thermofluor assay is high throughout, screening up to 384 conditions simultaneously. Second, CD only estimates the secondary structure, whereas the Thermofluor assay provides an assessment of the tertiary structure. Third, CD signal is sensitive to changes in buffer components whereas essentially any buffer can be used in the Thermofluor assay. The buffer screen described here is a good starting point for characterizing a protein of unknown biochemical properties (stability, solubility) but can be customized or expanded for individual proteins based on available biochemical information. Finally, the Thermofluor assay can also be used to assess the domain organization of larger proteins by determining whether multiple domains of a large protein represent independently folded units, thereby increasing understanding of protein structure and function.</p>
        </body>

<p class="pptt" id="biaoti34441">## Materials and Reagents</p><ol class="npwtwok"><li>1.7 ml microcentrifuge tubes (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1159M35 </a> )</li><li>96-well PCR microplate (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AB0800 </a> )</li><li>TBS (Tris-buffered saline) (150 mM NaCl, 20 mM Tris pH 7.5)</li><li>SYPRO Orange Protein Gel Stain (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S6650 </a> )</li><li>Transparent microplate sealing film (Hampton Research, ClearSeal, catalog number: <a class="blackNoUnder" target="_blank"> HR4-521 </a> )</li><li>Sodium Acetate (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP334 </a> )</li><li>Sodium Citrate (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S279 </a> )</li><li>MES (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M2933 </a> )</li><li>Sodium Cacodylate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C4945 </a> )</li><li>Bis-Tris (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP301 </a> )</li><li>Imidazole (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AC301872500 </a> )</li><li>MOPS (American Bioanalytical, catalog number: <a class="blackNoUnder" target="_blank"> AB01270 </a> )</li><li>HEPES (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP310500 </a> )</li><li>Tris (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP152 </a> )</li><li>Bicine (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> B3876 </a> )</li><li>Glycine (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP381 </a> )</li><li>CHES (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C2885 </a> )</li><li>Ethanolamine (ACROS organics, catalog number: <a class="blackNoUnder" target="_blank"> AC149582500 </a> )</li><li>CAPS (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C2632 </a> )</li><li>Glycerol (Chem-Impex, catalog number: <a class="blackNoUnder" target="_blank"> 30144 </a> )</li><li>Sodium Chloride (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S271 </a> )</li><li>45 Âµg protein of interest (for 24 conditions).<br/><em>Notes: <ol class="npwthree"><li>Protein should be homogenous (single band on an SDS-PAGE gel) and monodisperse (a single peak on size-exclusion chromatography).</li><li>Proteins used for this protocol included the N and C terminal domains of UL37 homolog from pseudorabies virus (PRV UL37N, PRV UL37C) and the full-length protein (PRV UL37FL). All proteins used here were purified using two rounds of affinity chromatography and one round of size-exclusion chromatography. Detailed instructions on purification of PRV UL37 can be found in Pitts et al. (2014) and Koenigsberg et al. (2018). Multiple protein batches resulted in comparable thermostability profiles for each protein.</li></ol></em> </li></ol><p class="pptt" id="biaoti34442">### Equipment</p><ol class="npwtwok"><li>Benchtop centrifuge (Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> Allegra X-15R </a> )</li><li>qPCR machine (Roche, LightCycler 480, catalog number: 0<a class="blackNoUnder" target="_blank"> 5015278001 </a> )<br/></li></ol><p class="pptt" id="biaoti34443">### Software</p><ol class="npwtwok"><li>ThermoQ (MATLAB, MathWorks, <a href="http://www.hotproteins.com/" target="_blank">http://www.hotproteins.com</a>)</li></ol><p class="pptt" id="biaoti34444">### Procedure</p><ol class="npwone"><li>Buffer and Protein Preparation<ol class="npwtwo"><li>Make 1 ml of the following buffers:<br/>100 mM Sodium Acetate pH 5.0<br/>100 mM Sodium Citrate pH 5.5<br/>100 mM MES pH 6.0<br/>100 mM Sodium Cacodylate pH 6.5<br/>100 mM Bis-Tris pH 6.5<br/>100 mM Imidazole pH 7.0<br/>100 mM PIPES pH 7.0<br/>100 mM Sodium Malonate pH 7.0<br/>100 mM MOPS pH 7.5<br/>100 mM HEPES pH 7.5<br/>100 mM Tris pH 8.0<br/>100 mM Bicine pH 9.0<br/>100 mM Glycine pH 9.0<br/>100 mM CHES pH 9.5<br/>100 mM CAPS pH 9.5<br/>200 mM HEPES pH 7.5<br/>200 mM HEPES pH 7.5, 50 mM NaCl<br/>200 mM HEPES pH 7.5, 150 mM NaCl<br/>200 mM HEPES pH 7.5, 300 mM NaCl<br/>200 mM HEPES pH 7.5, 500 mM NaCl<br/>200 mM HEPES pH 7.5, 150 mM NaCl, 5% glycerol<br/>200 mM HEPES pH 7.5, 150 mM NaCl, 10% glycerol<br/>200 mM HEPES pH 7.5, 500 mM NaCl, 5% glycerol<br/><em>Note: Buffers should be made 2x the concentration intended to be screened since addition of the protein solution in a 1:1 ratio will dilute the buffer by half. Any buffers (i.e., pH, salt, detergents, etc.) can be used. Here a screen of a pH range from 5.0-9.5 is used as well as targeted optimization of the buffer used for protein purification. Optimization includes increasing salt and glycerol concentrations, both of which are known to improve protein stability (Yamasaki et al., 1990 and Vagenende et al., 2009).</em> </li><li>Make a 300 Âµl solution of protein to a final concentration of 0.15 mg/ml in TBS.</li><li>Add SYPRO orange dye to protein solution at a 1:1,000 dilution.</li><li>Pipet 10 Âµl of screen buffers (from Step A1) per well of 96-well microplate.</li><li>Add 10 Âµl protein-dye solution to each well containing buffer (1:1).<br/><em>Note: Final protein concentration is 0.075 mg/ml and final buffer concentrations are 0.5x what is listed in Step A1.</em> </li><li>Set-up control well with buffer and dye (no protein).</li><li>Seal plate with transparent film.</li><li>Spin 96-well microplate at 500 <em>x g</em> for 1 min at room temperature.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Create Thermofluor method on Roche LightCycler 480<ol class="npwtwo"><li>Set filters to 465 nm for excitation and 610 nm for emission</li><li>Click the New Method icon, select Melting Curves as the Analysis Mode in the Programs window</li><li>Create a two-step method:<ol class="npwthree"><li>Step 1: Set a target of 25 Â°C held for 10 s (without data acquisition)</li><li>Step 2: Set a target of 95 Â°C with continuous acquisition at a rate of 3 fluorescent measurements per Â°C</li></ol></li><li>Save method as a template</li><li>Insert 96-well microplate containing protein-dye-buffer solution</li><li>New Method Template &gt; Thermofluor &gt; Apply Template</li><li>Click Start Run, enter name for data file</li><li>When run is finished, export data as a .txt file, save to drive</li></ol></li></ol><p class="pptt" id="biaoti34445">### Data analysis</p><ol class="npwtwok"><li>Import .txt file onto computer.</li><li>Open the MATLAB ThermoQ.m script (downloaded from www.hotproteins.com).</li><li>Click Run &gt; Roche &gt; 96 wells &gt; Select File &gt; choose .txt file &gt; Next &gt; Analyze.</li><li>Script will generate four figures:<ol class="npwthree"><li>ThermoQ figure 1: raw fluorescence data from all wells.</li><li>ThermoQ figure 2: smoothed, polynomial fit of the curves in ThermoQ figure 1.</li><li>ThermoQ figure 3: first derivative of melting curves in ThermoQ figure 1, where peak corresponds to the unfolding transition.</li><li>ThermoQ figure 4: overview of melting temperatures for all wells.</li></ol></li><li>To get exact melting temperature from ThermoQ figure 4 select Data Cursor from top menu, then click the lower left corner of a square, the Z coordinate represents the melting temperature (Figure 2).</li><li>ThermoQ figure 3 can be used to determine presence of multiple melting events (<em>i.e.</em>, multiple peaks).<br/>Thermofluor was performed with UL37N, two versions of UL37C, and the full-length UL37 (Figure 1). The UL37N protein had the highest thermal stability in PIPES (pH 7.0) or Na-Malonate (pH 7.0) buffers, very low concentrations of NaCl, and glycerol (Figure 1C, Table 1). Thus, all subsequent purification batches and biochemical and structural assays, including crystallization, for UL37N protein was conducted in its optimal buffer of 50 mM PIPES pH 7.0, 50 mM NaCl, and 0.5 mM TCEP. Glycerol was added to cell lysis buffer but was left out during the rest of purification and storage because it can hinder crystallization.<br/>The longer UL37C construct was most stable in HEPES buffer (pH 7.5), with the highest melting temperature (T<sub>M</sub>) in HEPES pH 7.5, 150 mM NaCl and 10% glycerol (B11) (Figure 1A, Table 1), which was used in subsequent purification, characterization and storage of all UL37C constructs. The melting curve of the full-length UL37 resulted in two distinct melting events, referred to as T<sub>M</sub> low and T<sub>M</sub> high, which are nearly identical to the respective T<sub>M</sub> of UL37C and UL37N (Figure 1D, Table 1). This suggests that UL37N and UL37C are two separately folded domains of UL37. In the case of UL37N and UL37C, increased thermostability correlated with increased solubility. No specific assay was used to determine protein solubility. A protein construct was considered to have an increased solubility if it could be concentrated to a higher concentration and remained in solution without precipitating at that concentration.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611032207_9294.jpg" width="300"/><br/><strong>Figure 1. Results of Thermofluor assay of PRV UL37.</strong> ThermoQ script generated images (ThermoQ figure 3) that represent the first derivative of the melting curves (ThermoQ figure 1). Maxiumum values represent the unfolding transition or melting temperature (T<sub>M</sub>). Each colored line represents a different buffer condition from the screen. Black lines represent the buffer condition resulting in the highest T<sub>M</sub>. Results are shown for PRV UL37 constructs containing (A) the C terminal half (UL37C), residues 478-919; (B) UL37C, residues 478-884; (C) the N-terminal half (UL37N), residues 1-496; and (D) the full-length UL37, residues 1-919. This research was originally published in the Journal of Biological Chemistry (Koenigsberg and Heldwein, 2018).<br/><br/><strong>Table 1. UL37 melting temperatures determined by Thermofluor</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611032225_0100.jpg" width="500"/><br/><br/>Exact buffer conditions screened for all UL37 constructs with the corresponding melting temperature, as determined by using the method described in Data analysis step 5.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200611/20200611032250_6538.jpg" width="300"/><br/><strong>Figure 2. Determine exact melting temperature from ThermoQ figure 4.</strong> The ThermoQ script displays melting temperatures as a heatmap (ThermoQ figure 4) to visualize relative melting temperatures between different buffer conditions. To determine the exact melting temperature, click the âData Cursorâ option circled in pink in Figure 2A. Once âData Cursorâ is selected the mouse is displayed as a crossbar. Identify the square on the heatmap that corresponds to the well of interest on the plate and click the bottom left corner of the square indicated by the pink arrow in Figure 2B. A small yellow box with XYZ coordinates will appear. The Z coordinate represents the melting temperature for that square in Â°C. Blue squares on gray background did not contain buffer and were not part of the screen. Note: this is example data that does not correspond to data shown in Figure 1 or Table 1.</li></ol><p class="pptt" id="biaoti34446">### Acknowledgments</p><p class="ppzz">This work was supported by National Institutes of Health Grant R01 AI056346 (to E. E. H.), the Investigators in the Pathogenesis of Infectious Disease Award from the Burroughs Wellcome Fund (to E. E. H.), and a Faculty Scholar grant from the Howard Hughes Medical Institute (to E. E. H.). A.L.K. was supported by NIH training grant T32 AI007422. This protocol was alluded to in the original research papers which also contain some of the data reported here (Pitts <em>et al.</em>, 2014 and Koenigsberg <em>et al.</em>, 2018).</p><p class="pptt" id="biaoti34447">### Competing interests</p><p class="ppzz">E.E.H. is the scientific co-founder of Thyreos, LLC. J.D.P. is currently an employee of Gilead Sciences. The authors declare that they have no conflicts of interest with the contents of this article.</p><p class="pptt" id="biaoti34448">### References</p><ol class="npwtwok"><li>Desai, P., Sexton, G. L., McCaffery, J. M. and Person, S. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11581394" target="_blank">A null mutation in the gene encoding the herpes simplex virus type 1 UL37 polypeptide abrogates virus maturation.</a> <em>J Virol</em> 75(21): 10259-10271.</li><li>Greenfield, N. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17406506" target="_blank">Using circular dichroism as a function of temperature to determine the thermodynamics of protein unfolding and binding interactions.</a> <em>Nat Protoc</em> 1(6): 2527-2535.</li><li>Jambunathan, N., Chouljenko, D., Desai, P., Charles, A. S., Subramanian, R., Chouljenko, V. N. and Kousoulas, K. G. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24600000" target="_blank">Herpes simplex virus 1 protein UL37 interacts with viral glycoprotein gK and membrane protein UL20 and functions in cytoplasmic virion envelopment.</a> <em>J Virol</em> 88(11): 5927-5935.</li><li>Koenigsberg, A. L. and Heldwein, E. E. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30166339" target="_blank">The dynamic nature of the conserved tegument protein UL37 of herpesviruses.</a> <em>J Biol Chem</em> 293(41): 15827-15839.</li><li>Leege, T., Granzow, H., Fuchs, W., Klupp, B. G. and Mettenleiter, T. C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19297610" target="_blank">Phenotypic similarities and differences between UL37-deleted pseudorabies virus and herpes simplex virus type 1.</a> <em>J Gen Virol</em> 90(Pt 7): 1560-1568. </li><li>Liu, X., Fitzgerald, K., Kurt-Jones, E., Finberg, R. and Knipe, D. M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18682563" target="_blank">Herpesvirus tegument protein activates NF-kappaB signaling through the TRAF6 adaptor protein.</a> <em>Proc Natl Acad Sci U S A</em> 105(32): 11335-11339. </li><li>Pitts, J. D., Klabis, J., Richards, A. L., Smith, G. A. and Heldwein, E. E. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24599989" target="_blank">Crystal structure of the herpesvirus inner tegument protein UL37 supports its essential role in control of viral trafficking.</a> <em>J Virol</em> 88(10): 5462-5473.</li><li>Phillips, K. and de la Pena, A. H. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21472694" target="_blank">The combined use of the Thermofluor assay and ThermoQ analytical software for the determination of protein stability and buffer optimization as an aid in protein crystallization.</a> <em>Curr Protoc Mol Biol</em> Chapter 10: Unit10 28.</li><li>Richards, A. L., Sollars, P. J., Pitts, J. D., Stults, A. M., Heldwein, E. E., Pickard, G. E. and Smith, G. A. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29216315" target="_blank">The pUL37 tegument protein guides alpha-herpesvirus retrograde axonal transport to promote neuroinvasion.</a> <em>PLoS Pathog</em> 13(12): e1006741. </li><li>Vagenende, V., Yap, M. G. and Trout, B. L. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19817484" target="_blank">Mechanisms of protein stabilization and prevention of protein aggregation by glycerol.</a> <em>Biochemistry</em> 48(46): 11084-11096.</li><li>Yamasaki, M., Yano, H. and Aoki, K. (1990). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2096908" target="_blank">Differential scanning calorimetric studies on bovine serum albumin: I. Effects of pH and ionic strength.</a> <em>Int J Biol Macromol</em> 12(4): 263-268.</li><li>Zhao, J., Zeng, Y., Xu, S., Chen, J., Shen, G., Yu, C., Knipe, D., Yuan, W., Peng, J., Xu, W., Zhang, C., Xia, Z. and Feng, P. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27866900" target="_blank">A viral deamidase targets the helicase domain of RIG-I to block RNA-induced activation.</a> <em>Cell Host Microbe</em> 20(6): 770-784. <br/></li></ol>