<body>
            <h1>Observing Nutrient Gradients, Gene Expression and Growth Variation Using the "Yeast Machine" Microfluidic Device</h1>
            <p><strong>Authors:</strong> Zoran S. Marinkovic, ClÃ©ment Vulin, Mislav Acman, Xiaohu Song, Jean Marc Di Meglio, Ariel B. Lindner, Pascal Hersen</p>
            <h2>Abstract</h2>
            <p>The natural environment of microbial cells like bacteria and yeast is often a complex community in which growth and internal organization reflect morphogenetic processes and interactions that are dependent on spatial position and time. While most of research is performed in simple homogeneous environments (<em>e.g.</em>, bulk liquid cultures), which cannot capture full spatiotemporal community dynamics, studying biofilms or colonies is complex and usually does not give access to the spatiotemporal dynamics at single cell level. Here, we detail a protocol for generation of a microfluidic device, the âyeast machineâ, with arrays of long monolayers of yeast colonies to advance the global understanding of how intercellular metabolic interactions affect the internal structure of colonies within defined and customizable spatial dimensions. With <em>Saccharomyces cerevisiae</em> as a model yeast system we used the âyeast machineâ to demonstrate the emergence of glucose gradients by following expression of fluorescently labelled hexose transporters. We further quantified the expression spatial patterns with intra-colony growth rates and expression of other genes regulated by glucose availability. In addition to this, we showed that gradients of amino acids also form within a colony, potentially opening similar approaches to study spatiotemporal formation of gradients of many other nutrients and metabolic waste products. This approach could be used in the future to decipher the interplay between long-range metabolic interactions, cellular development, and morphogenesis in other same species or more complex multi-species systems at single-cell resolution and timescales relevant to ecology and evolution.<br/></p>
            <p><strong>Keywords:</strong> Yeast colony, Microfluidics, Gene expression, Spatial organization, Metabolism, Microbial ecology, Emerging properties</p>
            <h2>Background</h2><p>The majority of fundamental research in microbiology to date relies on reduction of complexity in order to increase experimental control and to facilitate precise quantification of observed phenomena. This underlies the strong focus on studies of single cells in homogeneous environments or populations of cells grown in batch or continuous liquid cultures, despite the fact that in many natural niches cells form structured heterogeneous environments which behave as complex temporal and spatial dynamic systems. Cells of same or different species within multicellular assemblies absorb and metabolize nutrients while producing and secreting metabolites hence creating spatial gradients of nutrients and metabolites. This in turn has an effect on cellular physiology, linking it tightly to position and moment in space and time, resulting in areas that are well perfused with nutrients and areas that are scarce. Many multicellular systems exhibit variations in cellular physiologyâfrom bacterial and yeast colonies (Cap <em>et al.</em>, 2012; Vulin <em>et al.</em>, 2014) to biofilms (Nadell <em>et al.</em>, 2016) and tumors (Carmona-Fontaine <em>et al.</em>, 2013 and 2017; Delarue <em>et al.</em>, 2014). These variations are often a product of emergence of long-range metabolic interactions between cells. Namely, cellular microenvironment at one position depends on the nutrient uptake rate and metabolite production at other positions. Such multicellular communities (Shapiro, 1998; Shou <em>et al.</em>, 2007; Xavier and Foster, 2007) exhibit many adaptive benefits, including higher cell proliferation, improved access to resources and niches (Koschwanez <em>et al.</em>, 2011), and collective defence (Nadell <em>et al.</em>, 2016). This results in optimization of population survival when confronted with adverse physical, chemical, nutritional or biological challenges (PalkovÃ¡ and VÃ¡chovÃ¡, 2006). Hence the importance to understand the emergence and maintenance of complex spatial multicellular structures from ecological (Gonzalez <em>et al.</em>, 2012; Widder <em>et al.</em>, 2016; Antwis <em>et al.</em>, 2017), medical (Bryers, 2008; Estrela and Brown, 2018; Gilbert <em>et al.</em>, 2018) and evolutionary (Kim <em>et al.</em>, 2008; Nadell <em>et al.</em>, 2010; Ratcliff <em>et al.</em>, 2012) perspectives.<br/><br/>Numerous calls in recent reviews to study its nutrient sensing, signaling, and related growth and development control within the natural colony context (Broach, 2012; HorÃ¡k, 2013; Conrad <em>et al.</em>, 2014). While there have recently been some advances in observing complex bacterial communities (Liu <em>et al.</em>, 2015; Cooper <em>et al.</em>, 2017; Dal Co <em>et al.</em>, 2019; Hornung <em>et al.</em>, 2018), technological advances involving the widely used eukaryotic model organism <em>Saccharomyces cerevisiae</em> are lagging behind. The reason lies in the complexity of designing an experimental system that would allow monitoring, over long time periods and at single cell resolution, the development of a spatially defined extended multicellular assembly. Current direct observations of three-dimensional colonies are cumbersome and often constrained by existing technologies (Nadell <em>et al.</em>, 2016). In yeast, two-photon microscopy of sliced agarose-encapsulated yeast colonies was required to show that yeast cells may adopt different physiologiesâand possibly different cell typesâdepending on their position within a colony (Cap <em>et al.</em>, 2012). All mentioned approaches have some of the following limitations: small device dimensions (&lt; 100 Âµm), necessary use of low nutrient concentrations (&lt; 1 mM), limited scope of nutrient types (carbon-rich media with glucose rarely used), inability to access single cell level measurements, and inability to observe cells at high temporal resolution. For these reasons, it is challenging to apply such approaches to the general study of a large monolayer of cells in standard range and scope of nutrients, often used in biological research in liquid cultures. We designed the âyeast machineâ in a way that overcomes all these limitations (Figure 1). It is of large enough dimensions (800 Âµm long) to observe formation of gradients and depletion of nutrients up to glucose concentrations of 444 mM, it can be used to observe gradients of many carbon sources and other nutrients like amino acids, and it is easy to segment and track cells at single cell level in real-time over many days with time-lapse fluorescence microscopy. With this device we demonstrated the emergence of heterogeneous microenvironments and quantified spatial variation in cellular growth rate and the formation of gene expression landscapes for key metabolic genes involved in glucose transport and utilization. The application of this approach could be extended to study how the microenvironment dynamically changes when external conditions are altered in frequency-dependent manner. This is an uncharted territory at the scale of a multicellular assembly that is central to the understanding of microbial ecosystem resistance to stress, environmental fluctuations and adaptation. In addition, similar approaches could be used to study aging, cooperation and competition, cell memory, bet hedging, evolutionary dynamics, as well as quantitative characterization of (synthetic) ecological systems and mixtures of cells relevant to ecology and chemical biology.<br/></p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701084102_3556.jpg" width="298"/><br/><strong>Figure 1. Microfluidic device design.</strong> A. Each single âyeast machineâ has two sets of cell chambers of various widths (5 Âµm, 10 Âµm, 25 Âµm and 50 Âµm). The cell chambers are connected perpendicularly to a large flow channel (1 mm wide, 25 Âµm high). This design facilitates adaptation for different model systems (<em>e.g.</em>, bacteria, yeast, mammalian cells) and high-throughput depending on the length, width and height dimensions. B. A close-up sketch of a set of cell chambers used in our experiments. They are 800 Âµm long, 50 Âµm wide, and 4.5 Âµm high. A single cell chamber fits a monolayer of up to 2,500 yeast cells. This figure is modified from the original article (Marinkovic <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti34557">## Materials and Reagents</p><ol class="npwtwok"><li>Fluorinated ethylene-propylene (FEP) tubing 1/16" OD x .020" ID (IDEX Health &amp; Science, catalog number: <a class="blackNoUnder" target="_blank"> 1548L </a> )</li><li>Polyetheretherketone (PEEK) tubing blue 1/32" OD x .010" ID (IDEX Health &amp; Science, catalog number: <a class="blackNoUnder" target="_blank"> 1581 </a> )</li><li>Steel tubing 23 G (Darwin Microfluidics, view <a href="https://darwin-microfluidics.com/products/23g-steel-tubing-microfluidic-adapters-kit" target="_blank">details</a>)</li><li>Plastic weighing dish (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 08-732-113 </a> )</li><li>Glass coverslips, 24 x 50 mm #1.5 (VWR, Menzel-GlÃ¤ser, catalog number: <a class="blackNoUnder" target="_blank"> 631-9430 </a> )</li><li>Petri dishes 100 mm (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FB0875713 </a> )</li><li>Microcentrifuge tubes 1.5 ml (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 05-408-129 </a> )</li><li>Falcon round bottom tubes 14 ml (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14-959-1B </a> )</li><li>Falcon conical centrifuge tubes 50 ml (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14-432-22 </a> )</li><li>Glass media bottles (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FB800500 </a> )</li><li>Pipet tips, standard lab autoclaved, various sizes (Genesee Scientific)</li><li>Chrome photomask 5ââ x 5ââ .09 (Front Range PhotoMask, <a href="https://www.frontrangephotomask.com/" target="_blank">https://www.frontrangephotomask.com/</a>)</li><li>Plastic transfer pipets (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 13-680-50 </a> )</li><li>Scalpel (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 08-920B </a> )</li><li>Razor blade (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-640 </a> )</li><li>Cutting mat (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> NC0671274 </a> )</li><li>Wash bottles (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 02-897-10 </a> )</li><li>Scotch magic tape (Scotch<sup>TM</sup> Brand)</li><li>Yeast strains</li><li>Compressed air (in-house compressed air line)</li><li>Polydimethylsiloxane (PDMS, Dow Corning Sylgard 184 silicone elastomer kit) (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> NC9285739 </a> )</li><li>70% ethanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> NC9663244 </a> )</li><li>Isopropanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A416-500 </a> )</li><li>Acetone (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A18-500 </a> )</li><li>Distilled water</li><li>Yeast nitrogen base without amino acids (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> DF0919-15-3 </a> )</li><li>Complete supplement mixture (CSM) of amino acids (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> MP114500012 </a> )</li><li>D-(+)-Glucose (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> G7021 </a> )</li><li>Aluminum foil (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 01-213-100 </a> )</li><li>SU-8 2005 photoresist (Microchem)</li><li>SU-8 2025 photoresist (Microchem)</li><li>SU-8 developer (Microchem)</li><li>95% (3-Mercaptopropyl) trimethoxysilane (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 175617 </a> )</li><li>Pluronic F-127 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P2443 </a> )</li><li>Compressed N<sub>2</sub> (Airgas, catalog number: <a class="blackNoUnder" target="_blank"> NI 300 </a> )</li><li>Synthetic complete (SC) medium + 2% w/vol glucose (1 L) (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34558">### Equipment</p><ol class="npwtwok"><li>Glass Erlenmyer flasks 250 ml (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FB500250 </a> )</li><li>Timer (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 06-664-252 </a> )</li><li>Silicon wafers (University Wafers, catalog number: <a class="blackNoUnder" target="_blank"> 783 </a> )</li><li>Biopsy punch 1 mm (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 95039-092 </a> )</li><li>Motorized inverted epifluorescence microscope (Olympus, model: <a class="blackNoUnder" target="_blank"> IX81 </a> )</li><li>EMCCD camera (Photometrics, model: <a class="blackNoUnder" target="_blank"> Evolve 512 </a> )</li><li>X-Cite exacte fluorescence light source (Lumen Dynamics)</li><li>Optical filter ET-EGFP (Chroma Technology Corporation, catalog number: <a class="blackNoUnder" target="_blank"> U-N49002 </a> ; Ex 470/40 nm Di495 Em 525/50 nm)</li><li>Optical filter ET-DsRed (Chroma Technology Corporation, catalog number: <a class="blackNoUnder" target="_blank"> U-N49005 </a> ; Ex 545/30nm Di570 Em620/60 nm)</li><li>Objective 10Ã (Olympus, Plan 10x/0.25 NA)</li><li>Objective 60Ã (Olympus, PlanApo N 60x/1.42 NA Oil)</li><li>Objective 100Ã (Olympus, UPlanFL N 100x/1.3 NA Oil)</li><li>Microscope temperature control system (Life Imaging Services, The Cube &amp; The Box)</li><li>Mid to high end computer for the control of microscope and pressure-based microfluidic flow control system, as well as image analysis</li><li>Pressure-based microfluidic flow control system (Fluigent, MFCS)</li><li>Flow rate platform (Fluigent, Flowboard)</li><li>Flow rate control module (Fluigent, Flow unit M)</li><li>Vacuum desiccator (SP Scienceware, catalog number: <a class="blackNoUnder" target="_blank"> F42020-0000 </a> )</li><li>O<sub>2</sub> plasma cleaner (Harrick plasma, model: <a class="blackNoUnder" target="_blank"> PDC-002 </a> )</li><li>Vacuum pump (Harrick plasma, model: <a class="blackNoUnder" target="_blank"> PDC-VPE-2 </a> )</li><li>Laboratory oven (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 51-028-113H </a> )</li><li>Balance (Mettler Toledo, catalog number: <a class="blackNoUnder" target="_blank"> ML4002T </a> )</li><li>Benchtop centrifuge (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5405000042 </a> , model: <a class="blackNoUnder" target="_blank"> 5425 </a> )</li><li>3D printed holder (printing files available at <a href="https://github.com/Lab513/contrib" target="_blank">https://github.com/Lab513/contrib</a>)</li><li>Spin coater (Laurell, model: <a class="blackNoUnder" target="_blank"> WS-650 </a> ), used for loading cells with special 3D printed holder</li><li>Shaking incubator (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> SHKE4000-1CE </a> )</li><li>Spectrophotometer (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 6133000010 </a> )</li><li>Hot plate (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> S50462CH </a> )</li><li>Tweezers (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 16-100-103 </a> )</li><li>Bunsen burner (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 03-902Q </a> )</li><li>Pipetman, various sizes (Gilson)</li><li>Chemical hood</li><li>Photomask aligner (SÃ¼ss Microtec, model: <a class="blackNoUnder" target="_blank"> MJB4 </a> )</li><li>Mecanic surface profilometer (Veeco, Dektak 150)</li><li>Optical microscope (Nikon, model: <a class="blackNoUnder" target="_blank"> LV150 </a> )</li><li>Photoresist spin coater (Brewer Science, Cee Model 200x-F)</li><li>2 Hot plates (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 90-691-11 </a> )<br/></li></ol><p class="pptt" id="biaoti34559">### Software</p><ol class="npwtwok"><li>ImageJ (Schneider <em>et al.</em>, 2012, <a href="https://imagej.nih.gov/ij/" target="_blank">https://imagej.nih.gov/ij/</a>)</li><li>ÂµManager (Edelstein <em>et al.</em>, 2014, <a href="https://micro-manager.org/" target="_blank">https://micro-manager.org/</a>)</li><li>Matlab R2016b (MathWorks, <a href="https://www.mathworks.com/products/matlab.html" target="_blank">https://www.mathworks.com/products/matlab.html</a>)</li><li>Adobe Illustrator 2015 (Adobe, <a href="https://www.adobe.com/products/illustrator.html" target="_blank">https://www.adobe.com/products/illustrator.html</a>)</li><li>L-Edit software (Tanner, <a href="https://www.mentor.com/tannereda/l-edit" target="_blank">https://www.mentor.com/tannereda/l-edit</a>)</li><li>All-in-One (A-i-O, Fluigent, <a href="https://www.fluigent.com/" target="_blank">https://www.fluigent.com/</a>)<br/></li></ol><p class="pptt" id="biaoti34560">### Procedure</p><ol class="npwone"><li>Master wafer microfabrication<br/>Microfabrication can be tedious and complex process for which training is often required by the clean room staff. Each clean room is likely equipped with different instruments that have unique settings and operating procedures that may differ from the following microfabrication protocol. Use caution and adjust microfabrication protocol according to your local conditions. Once the process is mastered, the master wafer of âyeast machineâ complexity can be produced in 4 h (Figure 2).<ol class="npwtwo"><li>Turn on the mask aligner, gas and vacuum lines, and the UV lamp.</li><li>Set hot plates to 65 Â°C and 95 Â°C.</li><li>Cover the surface of spin coater with aluminum foil to minimize SU-8 contamination.</li><li>Rinse the polished side of the silicon wafer with acetone, isopropanol, and water sequentially to remove any dirt. Blow both sides of the wafer with compressed N<sub>2</sub> after the rinse with each solution until the wafer is completely dry to the eye.</li><li>Clean and activate the wafer surface with O<sub>2</sub> plasma cleaner at 6 mbar for 6 min.</li><li>Center the wafer on the chuck of a spin coater with polished side up.</li><li>Open the vacuum line to hold the wafer in place and make sure that it is perfectly centered, otherwise SU-8 will not spread evenly.</li><li>Program the spin coater according to the SU-8 manual. We used 500 rpm for 10 s and 6,000 rpm for 30 s. These settings are not necessarily consistent in time as they depend on SU-8 age and environmental conditions.</li><li>Spread SU-8 2000 epoxy-based resin of known thickness on the silicon wafer. There are twelve variants of the SU-8 2000 resin that differ in their viscosities. With them resin layer of 0.5 Âµm to 200 Âµm thickness through a single coating process can be made. Since the âyeast machineâ is composed of two layers that overlap, one 4.5 Âµm thick that serves for cell chambers and the other 25 Âµm thick that serves for nutrient channel, start with the thinner one. For 4.5 Âµm thickness use SU-8 2005 resin.</li><li>Pour 1 ml/in of wafer diameter (4 ml for 4-inch wafer) in the center and try to avoid bubbles by pouring SU-8 resin close to the wafer surface.</li><li>Wait 3-5 min for gas and bubbles to rise.</li><li>Run the pre-prepared program of the coat spinner.</li><li>Once finished, release vacuum and remove the wafer from the chuck. If SU-8 layer is not evenly spread, clean the wafer with isopropanol and repeat the procedure from Step A4.</li><li>Transfer the wafer with tweezers directly to 95 Â°C hot plate and âsoft-bakeâ it for 2 min.</li><li>Transfer the wafer with tweezers to the mask aligner. Epoxy-based resin is photosensitive and it solidifies through cross-linking once it is exposed to high-energy photons from the UV light. The chrome mask is placed above the silicon wafer covered with resin and the silicon wafer is exposed to UV light only through the pattern on the mask. Use the exposure duration of 4.5 s with the vacuum contact between the mask and the wafer. The exposure energy of the UV lamp we use is close to 20 mJ cm<sup>-2</sup> s<sup>-1</sup>.</li><li>Remove the wafer with tweezers from the aligner and place it on a hot plate for 3 min at 95 Â°C for a post exposure bake.</li><li>Take the wafer with tweezers off the hot plate and let it cool down to room temperature in the chemical hood. Transferring the wafer directly into the developer can cause cracking.</li><li>Pour SU-8 developer in a glass dish making sure there is enough of it to cover the wafer completely.</li><li>Submerge the wafer into SU-8 developer. This will remove the parts of the resin that did not solidify during the exposure to UV light.</li><li>Agitate the wafer for 2 min by hand.</li><li>Rinse the wafer gently with isopropanol wash bottle and dry it carefully with compressed N<sub>2</sub>.</li><li>Check the patterns under optical microscope. If some microcracks (submicron thin dark lines) are visible, put the wafer on 95 Â°C hot plate for 5-10 min. This will completely cure microcracks.</li><li>Repeat the same procedure (Steps A6-A22) with adjusted parameters for the 25 Âµm thick nutrient channel layer as follows in Steps A24-A32.</li><li>Spin SU-8 2025 in two steps, 500 rpm for 10 s and 4,000 rpm for 30 s.</li><li>âSoft bakeâ the wafer for 3 min at 65 Â°C and 6 min at 95 Â°C.</li><li>Transfer the wafer with tweezers to the mask aligner and be careful to align the patterns of the nutrient channel on the mask properly with the existing cell chamber pattern on the wafer. To do this, use the alignment crosses on the mask.</li><li>Expose the wafer to UV light for 8 s with the vacuum contact between the mask and the wafer.</li><li>Remove the wafer with tweezers from the aligner and place it on a hot plate for 1 min at 65 Â°C and 6 min at 95 Â°C for post exposure bake.</li><li>Take the wafer with tweezers off the hot plate and let it cool down to room temperature in the chemical hood.</li><li>Submerge the wafer into SU-8 developer and agitate for 5 min by hand.</li><li>Rinse the wafer gently with isopropanol wash bottle and dry it carefully with compressed N<sub>2</sub>.</li><li>Check the patterns under optical microscope. If some microcracks (submicron thin dark lines) are visible, put the wafer on 95 Â°C hot plate for 5-10 min. This will completely cure microcracks.</li><li>Check the dimensions of the patterns on the master wafer using Dektak 150 surface profiler. If the dimensions are not correct (up to 10% errors are fine), make new master wafer.</li><li>Finally, put the master wafer in a Petri dish and treat it with (3-Mercaptopropyl)trimethoxysilane 95% for 1 h in vapour phase by placing the wafer close to a plastic cup with a few drops of silane in a vacuumed dessicator. Be careful to do this in a chemical hood, as silane is a toxic compound.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701084127_6748.jpg" width="87"/><br/><strong>Figure 2. Microfluidic device fabrication.</strong> Epoxy-based resin is spread on a silicon wafer and illuminated with the UV light through a custom designed chrome mask. Soluble part of the resin is washed off and a new cycle of resin deposition can be made depending on the number of layers of the microfluidic device. The master wafer is silanized and used as a re-usable negative mold to produce microfluidic chips. Polydimethylsiloxane (PDMS) is then poured on the master wafer and cured until it is polymerized. The PDMS chip is bonded with a glass coverslip by plasma bonding. The final microfluidic chip is used to grow cells and deliver nutrients. The thinner parts are the cell chambers, and the thicker part is the main nutrient channel. This figure is modified from the original article (Marinkovic <em>et al.</em>, 2019).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Casting the âyeast machineâ in polydimethylsiloxane (PDMS)<br/>Now that the desired pattern on the master wafer is made, it can be used as a negative for replica molding of the âyeast machineâ chip with the help of PDMS. PDMS is a transparent biocompatible silicon polymer permeable to gases. It is a viscous fluid when it is monomeric and it is elastic once it homopolymerizes in a reaction catalyzed by a curing agent.<ol class="npwtwo"><li>Mix PDMS base and curing agent in a plastic weighing dish at a mass ratio of 10:1. When casting the whole PDMS mold for the first time use 30 g of PDMS base and 3 g of curing agent. Once only the âyeast machineâ part is cut out, it is sufficient to re-cast only 3 g of PDMS base and 0.3 g of curing agent.</li><li>Mix well with a pipet tip or a similar device.</li><li>Degas the PDMS in a vacuum desiccator until all bubbles disappear, approximately 30 min.</li><li>Pour degassed PDMS over the master wafer in a Petri dish. If some bubbles appear again, degas the whole wafer until the bubbles are gone.</li><li>Cure PDMS at 65 Â°C for 2 h minimum or overnight in a laboratory oven. <br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Assembling the âyeast machineâ<br/>For each experiment always use a newly assembled microfluidic chip. âYeast machineâ cannot be recycled for additional experiments.<ol class="npwtwo"><li>Gently cut the PDMS in rectangular shape around the patterns. Be careful not to crack the master wafer by putting too much pressure on the scalpel. It is also advisable to let the cured PDMS cool down to room temperature to avoid damage to the master wafer.</li><li>Carefully peel off the PDMS from the master wafer.</li><li>Inspect the PDMS and its patterns under the microscope in order to detect possible defects. If defects come from a damaged master wafer, make new master wafer.</li><li>Place the cut PDMS on a cutting mat with pattern side up.</li><li>Punch out carefully the holes at each end of the nutrient channel with the biopsy punch. Always use a sharp biopsy punch and check under a microscope that there are no cracks in the hole, otherwise the media might leak.</li><li>Clean the surface of the PDMS and glass cover slips using tape.</li><li>Place the PDMS pattern side up and glass cover slip in plasma cleaner.</li><li>Turn on the vacuum pump and close the plasma cleaner.</li><li>Set the power to high to ignite the plasma.</li><li>After pink glowing plasma appears, wait for 1 min and turn off the power.</li><li>Open the valve of the plasma cleaner gently to allow air to flow in. It is safe to turn off the vacuum pump now.</li><li>Immediately and carefully place the PDMS pattern side down onto glass cover slip side that was exposed to plasma.</li><li>Use tweezers to touch gently the top of the PDMS to facilitate bonding. Be careful, pressing down the patterns could collapse them.</li><li>Put a piece of tape on top of the chip to prevent dust from bonding to the chip.</li><li>Bake the bonded device at 65 Â°C for 1 h or overnight to cure the bonding.</li><li>Inject 10 Âµl of filter sterilized 1% Pluronic F-127 dissolved in distilled water and wait for 30 min at room temperature to passivate interior surfaces. Be careful, before injecting the liquid into the channel with a pipette, pour a drop of liquid onto the inlet to prevent bubbles entering the channel.</li><li>Flush the channel with sterile distilled water or SC media to remove Pluronic F-127. Be careful again not to inject bubbles into the channel. Pour a drop of liquid onto the inlet first.</li><li>The microfluidic chip is now ready to be loaded with yeast.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell culturing and loading<ol class="npwtwo"><li>Preculture yeast cells overnight the day before the experiment in 5 ml of synthetic complete (SC) medium containing 2% w/vol glucose in a shaking incubator at 30 Â°C.</li><li>On the day of the experiment dilute cells 50-fold into 50 ml of SC + 2% w/vol glucose.</li><li>Culture them for 5-6 h in a shaking incubator at 30 Â°C until they reach and OD<sub>600</sub> of 0.2-0.4.</li><li>Put 1 ml of cells in 1.5 ml Eppendorf tube and centrifuge them for 1 min at RCF of 300 <em>x g</em>.</li><li>Resuspend the cells in 20 Âµl SC media + 2% w/vol glucose.</li><li>Load 10 Âµl of resuspended cells into the channel with a pipette. To make sure that no bubbles get in, pour a drop of liquid onto the inlet first and then load the cells.</li><li>Tape the chip on a custom made 3D printed holder (Figure 3) with cell chambers facing outwards, so that the centrifugal force can push the cells from media channel into the cell chamber. The chip must be firmly pressed against the holder and taped on one side as shown in Figure 3A. After taping the chip should be completely immobilized.</li><li>Attach the 3D printed holder onto the Laurell <a class="blackNoUnder" target="_blank"> WS-650 </a> spin coater and centrifuge for 2 min at 110 <em>x g</em>.</li><li>Alternatively, tape the chip into a centrifuge making sure that the forces push cells into the cell chambers.</li><li>Check under a microscope that cells have been loaded into the cell chambers. Usually, a couple of layers of cells are pushed inside towards the dead end (point furthest away from media channel) of the cell chamber.</li><li>Flush the remaining cells out of the media channel with SC media + 2% w/vol glucose, making sure that no bubbles enter the system. To prevent bubbles from entering, pour a drop of liquid onto the inlet first and then flush the cells.<br/>â<br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701084202_3710.jpg" width="190"/><br/><strong>Figure 3. Cell-loading equipment.</strong> A. 3D printed chip holder for centrifugation with the âyeast machineâ mounted on it (channels are colored with different dyes for better visualization). B. Spin coater that we use for centrifugation of cells into the cell chamber. This figure is modified from the original article (Marinkovic <em>et al.</em>, 2019).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Media perfusion and imaging<br/>There are several options to perfuse media in microfluidic chips using hydrostatic pressure, peristaltic pumps, syringe pumps, or pressure based pumps. We opted for pressure-based pumps as they offer seamless control over flow of media through the system. The flow rate control module, assures constant media flow rate, no matter if there are clogs or bubbles in the system. These are often eliminated as the control system attempts to maintain the same flow rate in time.<ol class="npwtwo"><li>Turn on the microscope and heat it up to 30 Â°C.</li><li>Prepare 50 ml of SC media with appropriate glucose concentration in 50 ml Falcon tube.</li><li>Assemble the pressure-based microfluidic flow control system (MFCS, Fluigent) coupled with a flow rate platform (Fluigent) and a flow rate control module (Fluigent) according to the instructions provided with the system. See Figure 4 for details.</li><li>Connect input and output tubing to the chip. Make sure that the media is flowing very slowly out of the tubing prior to and while connecting it to the chip to prevent bubbles entering the system.</li><li>Set the flow in All-in-One (A-i-O, Fluigent) software to maximum to flush out all the remaining cells from the nutrient channel and then set the flow to constant 5 Âµl/min.</li><li>Keep the output at a constant pressure of 100 mbar above atmospheric pressure to minimize formation of air bubbles inside the nutrient channel.</li><li>Inspect the chip for potential leaks before mounting it to the microscope.</li><li>Place the chip onto the microscope stage and fix it in place with tape.</li><li>Set up the image acquisition parameters relevant to your experiments. Ideally, image as many positions as possible at the lowest intensity and exposure as possible and at shortest time intervals as possible. If you want to observe a set of several complete cell chambers in one frame, use 10x objective. If you want to observe details at the single cell level, use 60x or 100x oil-immersed objective. Duration of a typical experiment is 24-72 h. However, we did manage to run experiments for up to one week without major disruptions.</li><li>Check the first few intervals of imaging to make sure that the focus is stable and that the field is not drifting.</li><li>Check occasionally that no clogs or leaks are impeding the ongoing experiment. Clogs mainly occur if bubbles are formed in the system and leaks mainly occur if inlet or outlet is cracked.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701084222_4969.jpg" width="113"/><br/><strong>Figure 4. Microfluidic device setup.</strong> A. Media reservoirs are pressurized with Fluigent MFCS pressure pump resulting in flow through the flow sensors, into the chip and then to waste. Flow sensors and pressure pump are connected to the flow-rate control module, which maintains a constant flow through the system. Nutrient supply and media conditions can be changed in real time. B. Microfluidic chip mounted on a microscope. This figure is modified from the original article (Marinkovic <em>et al.</em>, 2019).</li></ol></li></ol><p class="pptt" id="biaoti34561">### Data analysis</p><p class="ppzz">Information about data processing and analyses, Matlab code, statistics, criteria for data inclusion/exclusion, details of number of replicates, exact data sets for each figure, and notes about reproducibility and variability can be found here (Marinkovic <em>et al.</em>, 2019).<br/>  In order to optimize acquisition time and image analysis, microscopic images were most often taken with a 10x objective. This allows an entire set of cell chambers to fit within a single image without the need to stich multiple images. Open-source ÂµManager microscopy software (Edelstein <em>et al.</em>, 2014) was used to control the microscopeâs components and to set up image acquisition. A stack of time-lapse images acquired through a particular optical filter was saved into a single .tif (Tagged Image File Format) file and analyzed using an open-source ImageJ (1.51p) software (Schneider <em>et al.</em>, 2012).</p><br/><ol class="npwone"><li>Front and Local velocity<br/>Image analysis for calculating the front and local velocity is explained in the methods part of the original article under âimage analysisâ (Marinkovic <em>et al.</em>, 2019).<br/><br/></li><li>Gene expression<br/>During the image analysis, each cell chamber was tracked using Region of Interest (ROI) manager (ImageJ package) that covered 50 Âµm width and 800 Âµm length of a single cell chamber. All the values of obtained pixel intensities were normalized with the values of âyeast machineâ background (<em>i.e.</em>, the background noise) for each position along the 800 Âµm long chamber. By using Multi Plot function within ROI manager, the mean pixel intensity of the stacked images at a timepoint of interest was computed for each longitudinal position. To export position and intensity data from ImageJ to Excel, the Save function in the Profiles window was used. Data in Excel sheet was then imported into Matlab for further processing. This way we obtained information about average cell fluorescence for each region of the chamber and processed it with custom made Matlab scripts available in the original article (Marinkovic <em>et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti34562">### Notes</p><ol class="npwtwok"><li>Representative results can be seen in figures and a movie in the original article (Marinkovic <em>et al.</em>, 2019).</li><li>A comprehensive video of typical results obtained with the âyeast machineâ can be seen at the following link <a href="https://doi.org/10.7554/eLife.47951.005" target="_blank">https://doi.org/10.7554/eLife.47951.005</a>.</li><li>Production of master wafer from part A of âProcedureâ can be outsourced to commercial microfabrication companies (<em>e.g.</em>, <a href="https://www.micruxfluidic.com" target="_blank">https://www.micruxfluidic.com</a>).<br/></li></ol><p class="pptt" id="biaoti34563">### Recipes</p><ol class="npwtwok"><li>Synthetic complete (SC) medium + 2% w/vol glucose (1 L)<br/>6.7 g Yeast nitrogen base without amino acids<br/>0.79 g Complete supplement mixture (CSM) of amino acids<br/>20 g Glucose<br/>Distilled autoclaved water to 1 L<br/>Keep each component as filter sterilized 10x stock solution in distilled water<br/></li></ol><p class="pptt" id="biaoti34564">### Acknowledgments</p><p class="ppzz">This work was supported by the Agence Nationale de la Recherche (ICEBERG-ANR-10-BINF-06-01; ANR-16-CE12-0025-01), the interdisciplinary program of the University Sorbonne Paris CitÃ©, the Who am I? Laboratory of Excellence (ANR-11-LABX-0071 and ANR-11-IDEX-0005-01) and the European Research Council (ERC) under the European Unionâs Horizon 2020 research and innovation programme (grant agreement No 724813). This protocol was adapted from Marinkovic <em>et al.</em> (2019).</p><p class="pptt" id="biaoti34565">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti34566">### References</p><ol class="npwtwok"><li>Antwis, R. E., Griffiths, S. M., Harrison, X. A., Aranega-Bou, P., Arce, A., Bettridge, A. S., Brailsford, F. L., de Menezes, A., Devaynes, A., Forbes, K. M., Fry, E. L., Goodhead, I., Haskell, E., Heys, C., James, C., Johnston, S. R., Lewis, G. R., Lewis, Z., Macey, M. C., McCarthy, A., McDonald, J. E., Mejia-Florez, N. L., O'Brien, D., Orland, C., Pautasso, M., Reid, W. D. K., Robinson, H. A., Wilson, K. and Sutherland, W. J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28379446" target="_blank">Fifty important research questions in microbial ecology.</a> <em>FEMS Microbiol Ecol</em> 93(5).</li><li>Broach, J. R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22964838" target="_blank">Nutritional control of growth and development in yeast.</a> <em>Genetics</em> 192(1): 73-105.</li><li>Bryers, J. D. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18366134" target="_blank">Medical biofilms.</a> <em>Biotechnol Bioeng</em> 100(1): 1-18.</li><li>Cap, M., Stepanek, L., Harant, K., Vachova, L. and Palkova, Z. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22560924" target="_blank">Cell differentiation within a yeast colony: metabolic and regulatory parallels with a tumor-affected organism.</a> <em>Mol Cell</em> 46(4): 436-448.</li><li>Carmona-Fontaine, C., Bucci, V., Akkari, L., Deforet, M., Joyce, J. A. and Xavier, J. B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24218566" target="_blank">Emergence of spatial structure in the tumor microenvironment due to the Warburg effect.</a> <em>Proc Natl Acad Sci U S A</em> 110(48): 19402-19407.</li><li>Carmona-Fontaine, C., Deforet, M., Akkari, L., Thompson, C. B., Joyce, J. A. and Xavier, J. B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28246332" target="_blank">Metabolic origins of spatial organization in the tumor microenvironment.</a> <em>Proc Natl Acad Sci U S A</em> 114(11): 2934-2939.</li><li>Conrad, M., Schothorst, J., Kankipati, H. N., Van Zeebroeck, G., Rubio-Texeira, M. and Thevelein, J. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24483210" target="_blank">Nutrient sensing and signaling in the yeast <em>Saccharomyces cerevisiae</em>.</a> <em>FEMS Microbiol Rev</em> 38(2): 254-299.</li><li>Cooper, R. M., Tsimring, L. and Hasty, J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29091031" target="_blank">Inter-species population dynamics enhance microbial horizontal gene transfer and spread of antibiotic resistance.</a> <em>Elife</em> 6: 25950.</li><li>Dal Co, A., van Vliet, S. and Ackermann, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31587651" target="_blank">Emergent microscale gradients give rise to metabolic cross-feeding and antibiotic tolerance in clonal bacterial populations.</a> <em>Philos Trans R Soc Lond B Biol Sci</em> 374(1786): 20190080.</li><li>Delarue, M., Montel, F., Vignjevic, D., Prost, J., Joanny, J. F. and Cappello, G. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25418163" target="_blank">Compressive stress inhibits proliferation in tumor spheroids through a volume limitation.</a> <em>Biophys J</em> 107(8): 1821-1828.</li><li>Edelstein, A. D., Tsuchida, M. A., Amodaj, N., Pinkard, H., Vale, R. D. and Stuurman, N. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25606571" target="_blank">Advanced methods of microscope control using muManager software.</a> <em>J Biol Methods</em> 1(2).</li><li>Estrela, S. and Brown, S. P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29927925" target="_blank">Community interactions and spatial structure shape selection on antibiotic resistant lineages.</a> <em>PLoS Comput Biol</em> 14(6): e1006179.</li><li>Gilbert, J. A., Blaser, M. J., Caporaso, J. G., Jansson, J. K., Lynch, S. V. and Knight, R. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29634682" target="_blank">Current understanding of the human microbiome.</a> <em>Nat Med</em> 24(4): 392-400.</li><li>Gonzalez, A., King, A., Robeson, M. S., 2nd, Song, S., Shade, A., Metcalf, J. L. and Knight, R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22154467" target="_blank">Characterizing microbial communities through space and time.</a> <em>Curr Opin Biotechnol</em> 23(3): 431-436.</li><li>Koschwanez, J. H., Foster, K. R. and Murray, A. W. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21857801" target="_blank">Sucrose utilization in budding yeast as a model for the origin of undifferentiated multicellularity.</a> <em>PLoS Biol</em> 9(8): e1001122.</li><li>HorÃ¡k, J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23455612" target="_blank">Regulations of sugar transporters: insights from yeast.</a> <em>Curr Genet</em> 59(1-2): 1-31.</li><li>Hornung, R., Grunberger, A., Westerwalbesloh, C., Kohlheyer, D., Gompper, G. and Elgeti, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29445038" target="_blank">Quantitative modelling of nutrient-limited growth of bacterial colonies in microfluidic cultivation.</a> <em>J R Soc Interface</em> 15(139).</li><li>Kim, H. J., Boedicker, J. Q., Choi, J. W. and Ismagilov, R. F. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19011107" target="_blank">Defined spatial structure stabilizes a synthetic multispecies bacterial community.</a> <em>Proc Natl Acad Sci U S A</em> 105(47): 18188-18193.</li><li>Liu, J., Prindle, A., Humphries, J., Gabalda-Sagarra, M., Asally, M., Lee, D. Y., Ly, S., Garcia-Ojalvo, J. and SÃ¼el, G. M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26200335" target="_blank">Metabolic co-dependence gives rise to collective oscillations within biofilms.</a> <em>Nature</em> 523(7562): 550-554.</li><li>Marinkovic, Z. S., Vulin, C., Acman, M., Song, X., Di Meglio, J. M., Lindner, A. B. and Hersen, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31259688" target="_blank">A microfluidic device for inferring metabolic landscapes in yeast monolayer colonies.</a> <em>eLife</em> 8: 47951.</li><li>Nadell, C. D., Drescher, K. and Foster, K. R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27452230" target="_blank">Spatial structure, cooperation and competition in biofilms.</a> <em>Nat Rev Microbiol</em> 14(9): 589-600.</li><li>Nadell, C. D., Foster, K. R. and Xavier, J. B. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20333237" target="_blank">Emergence of spatial structure in cell groups and the evolution of cooperation.</a> <em>PLoS Comput Biol</em> 6(3): e1000716.</li><li>PalkovÃ¡, Z. and VÃ¡chovÃ¡, L. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16911045" target="_blank">Life within a community: benefit to yeast long-term survival.</a> <em>FEMS Microbiol Rev</em> 30(5): 806-824.</li><li>Ratcliff, W. C., Denison, R. F., Borrello, M. and Travisano, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22307617" target="_blank">Experimental evolution of multicellularity.</a> <em>Proc Natl Acad Sci U S A</em> 109(5): 1595-1600.</li><li>Schneider, C. A., Rasband, W. S. and Eliceiri, K. W. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22930834" target="_blank">NIH Image to ImageJ: 25 years of image analysis.</a> <em>Nature Methods</em> 9(7): 671-675.</li><li>Shapiro, J. A. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9891794" target="_blank">Thinking about bacterial populations as multicellular organisms.</a> <em>Annu Rev Microbiol</em> 52: 81-104.</li><li>Shou, W., Ram, S. and Vilar, J. M. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17267602" target="_blank">Synthetic cooperation in engineered yeast populations.</a> <em>Proc Natl Acad Sci U S A</em> 104(6): 1877-1882.</li><li>Vulin, C., Di Meglio, J. M., Lindner, A. B., Daerr, A., Murray, A. and Hersen, P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24853750" target="_blank">Growing yeast into cylindrical colonies.</a> <em>Biophys J</em> 106(10): 2214-2221.</li><li>Widder, S., Allen, R. J., Pfeiffer, T., Curtis, T. P., Wiuf, C., Sloan, W. T., Cordero, O. X., Brown, S. P., Momeni, B., Shou, W., Kettle, H., Flint, H. J., Haas, A. F., Laroche, B., Kreft, J. U., Rainey, P. B., Freilich, S., Schuster, S., Milferstedt, K., van der Meer, J. R., Grobetakopf, T., Huisman, J., Free, A., Picioreanu, C., Quince, C., Klapper, I., Labarthe, S., Smets, B. F., Wang, H., Isaac Newton Institute, F. and Soyer, O. S. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27022995" target="_blank">Challenges in microbial ecology: building predictive understanding of community function and dynamics.</a> <em>ISME J</em> 10(11): 2557-2568.</li><li>Xavier, J. B. and Foster, K. R. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17210916" target="_blank">Cooperation and conflict in microbial biofilms.</a> <em>Proc Natl Acad Sci U S A</em> 104(3): 876-881.<br/></li></ol>