<body>
            <h1>Methylation-sensitive Amplified Polymorphism as a Tool to Analyze Wild Potato Hybrids</h1>
            <p><strong>Authors:</strong> NicolÃ¡s Cara, Carlos F. Marfil, MarÃ­a V. Bertoldi, Ricardo W. Masuelli</p>
            <h2>Abstract</h2>
            <p>Methylation-Sensitive Amplification Polymorphism (MSAP) is a versatile marker for analyzing DNA methylation patterns in non-model species. The implementation of this technique does not require a reference genome and makes it possible to determine the methylation status of hundreds of anonymous loci distributed throughout the genome. In addition, the inheritance of specific methylation patterns can be studied. Here, we present a protocol for analyzing DNA methylation patterns through MSAP markers in potato interspecific hybrids and their parental genotypes.</p>
            <p><strong>Keywords:</strong> MSAP, DNA Methylation Pattern Inheritance, Epigenetic changes, <em>Solanum</em>, Interspecific hybridization, Epialleles, R Script</p>
            <h2>Background</h2><p>Nucleotide sequences are not the only form of genomic information: DNA methylation, histone proteins, enzymes that modify histones and nucleotide residues on DNA and even RNAs, influence gene activity and provide another layer of instructions to the cell. Epigenetic changes, also called <em>epimutations</em>, can be inherited and have important phenotypic consequences. In plants, methylation reactions modify cytosine residues into 5-methylcytosines. This epigenetic mechanism is essential to maintain the genomic integrity and contributes to regulate gene expression in developmental processes and in response to biotic and abiotic stresses. In addition, changes in DNA methylation are triggered by genomic shocks like hybridization and polyploidization, two vital phenomena in plant evolution (Cara <em>et al.</em>, 2019).<br/> <br/>There are different alternatives to study changes in DNA methylation. Global cytosine methylation can be assessed by using high-performance liquid chromatography (HPLC), analytical methods that allow to quantify cytosines and 5-methylcytosines and to calculate the percentage of methylated residues in the genome. For studying the DNA methylation at specific positions on the genome two alternatives can be mentioned. One is to use isoschizomers with different sensitivities to methylation at cytosines of the restriction site. For example, Methylation-Sensitive Amplification Polymorphism (MSAP) markers characterize the methylation pattern at anonymous 5â²-CCGG sequences from random genomic DNA. This is an adaptation of the original AFLP protocol (Vos <em>et al.</em>, 1995) substituting the frequent cutter enzyme <em>Mse</em>I by <em>Hpa</em>II and <em>Msp</em>I. These enzymes recognize the same tetranucleotide restriction site (5â²-CCGG), but <em>Hpa</em>II is sensitive to full methylation (both strands methylated) and cleaves the hemi-methylated external cytosine, whereas <em>Msp</em>I is sensitive only to methylation of the external cytosines of the restriction site. Another possibility to study site-specific methylation status is by using a bisulphite sequencing methodology. Bisulphite treatments on DNA convert cytosines into uracils while 5-methylcytosines remain unchanged. Then, by sequencing amplicons (<em>i.e.</em>, a target gene or promoter) of bisulphited and control DNA, it is possible to distinguish between methylated and unmethylated cytosines. With the development of next generation sequencing technologies, whole-genome bisulphite sequencing, or WGBS, can be implemented to infer the position for all 5-methylcytosines in a genome. However, this approach requires a high-quality reference genome to perform epigenetic analyses. Although the increasing number of draft genomes and the reduction in sequencing costs offer possibilities to implement massive methylation analyses in non-model species, the use of MSAP markers continues to be a valuable tool in many laboratories.<br/><br/>Wild potatoes (<em>Solanum</em>, section <em>Petota</em>) are a group of species related to the cultivated potato <em>Solanum tuberosum</em> L. Internal breeding barriers can be incomplete, thus, interspecific hybridization occurs in areas of sympatry (Camadro <em>et al.</em>, 2012). Epigenetic changes in response to interspecific hybridization have been documented in synthetic and natural hybrids of wild potato species (Cara <em>et al.</em>, 2019). <em>Solanum</em> x <em>rechei</em> H. &amp; H. is a hybrid species that grows in sympatry with its wild progenitors, <em>Solanum kurtzianum</em> B. &amp; W. and <em>Solanum microdontum</em> B. Here, we present a protocol for analyzing DNA methylation patterns through MSAP markers in hybrids and their parental genotypes. Using an R script, fragments present in the synthetic hybrids are categorized as <em>S. microdontum</em> or <em>S. kurtzianum</em> species-specific if they are present on the parental genotypes, <em>S.</em> x <em>rechei</em> species-specific, if they are present in at least one of the <em>S.</em> x <em>rechei</em> evaluated genotypes or as novel if they are only observed in the synthetic hybrids.</p>
        </body>

<p class="pptt" id="biaoti34593">## Materials and Reagents</p><ol class="npwone"><li>Consumables<ol class="npwtwo"><li>Microfuge tubes (DELTALAB, catalog numbers: <a class="blackNoUnder" target="_blank"> 4095.5N </a> , <a class="blackNoUnder" target="_blank"> 4095.9N </a> )</li><li>Pipette tips (DELTALAB, catalog numbers: <a class="blackNoUnder" target="_blank"> 200072 </a> , <a class="blackNoUnder" target="_blank"> 200016 </a> )</li><li>PCR microplate 96-well (Axygen, catalog number: <a class="blackNoUnder" target="_blank"> 32165051 </a> )</li><li>Plastic pestle (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> Z359947 </a> )<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Chemicals<ol class="npwtwo"><li>Tris base (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 200016 </a> 6800)</li><li>Ethylenediaminetetraacetic acid (EDTA) (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 2000964500 </a> )</li><li>Sodium Chloride (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 200016 </a> 4606)</li><li>Cetyltrimethyl ammonium bromide (CTAB) (Bio Basic INC, catalog number: <a class="blackNoUnder" target="_blank"> DB0108 </a> )</li><li>Î²-mercaptoethanol (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 2000954500 </a> )</li><li>Chloroform (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 200016 </a> 5100)</li><li>Isoamyl alcohol (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 2000972500 </a> )</li><li>Ethyl alcohol (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 200016 </a> 5400)</li><li>Sodium acetate (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 200016 </a> 8000)</li><li>RNase A (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> EN0531 </a> )</li><li>Lambda DNA/<em>EcoR</em>I+HindIII Marker (Promega, catalog number: <a class="blackNoUnder" target="_blank"> G1731 </a> )</li><li>Glycerol (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 200016 </a> 2000)</li><li>Bromophenol blue (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 2000962200 </a> )</li><li>Xylene cyanol FF (SIGMA, catalog number: <a class="blackNoUnder" target="_blank"> X4126 </a> )</li><li>Boric acid (BIOPACK, catalog number: <a class="blackNoUnder" target="_blank"> 2000935900 </a> )</li><li>Agarose (TransGen, catalog number: <a class="blackNoUnder" target="_blank"> GS201 </a> )</li><li>UltraPure<sup>TM</sup> Ethidium Bromide (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15585011 </a> )</li><li><em>EcoR</em>I (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0101S </a> )</li><li>Bovine Serum Albumin (BSA) (Promega, catalog number: <a class="blackNoUnder" target="_blank"> R3961 </a> )</li><li><em>Hpa</em>II (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0171S </a> )</li><li><em>Msp</em>I (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0106 </a> )</li><li>T4 DNA Ligase (Promega, catalog number: <a class="blackNoUnder" target="_blank"> M1801 </a> )</li><li>Oligonucleotides (Table 1) (IDT, Integrated DNA Technologies, Inc., Iowa, USA)<br/><br/><strong>Table 1. Sequences of adaptors and primers used</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701184340_3365.jpg" width="500"/><br/><br/></li><li><em>Taq</em> DNA Polymerase (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11615044 </a> )</li><li>MgCl<sub>2</sub> (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11615044 </a> )</li><li>dNTP Set (100 mM) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10297018 </a> )</li><li>Hi-Di formamide (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 4311320 </a> )</li><li>GeneScan 500HD Rox size standard (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 401734 </a> )</li><li>ddH<sub>2</sub>O (sterile)</li><li>Extraction buffer (see Recipes)</li><li>TBE (Tris-Borate-EDTA) buffer (see Recipes)</li><li>6x DNA loading buffer (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti34594">### Equipment</p><ol class="npwtwok"><li>Pipettes (BOECO, Wheaton SOCOREX and Finnpipette)</li><li>Vortex (IKA, model: <a class="blackNoUnder" target="_blank"> MS1 </a> )</li><li>UltraCentrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5804R </a> )</li><li>Thermostatic bath (Vicking, model: <a class="blackNoUnder" target="_blank"> Masson </a> )</li><li>Agarose gel electrophoresis system (Bio-Rad Laboratories, model: <a class="blackNoUnder" target="_blank"> 1645056 </a> )</li><li>Microwave oven (HITPLUS, model: <a class="blackNoUnder" target="_blank"> CM203M </a> )</li><li>Spectrophotometer (AmpliQuant, model: <a class="blackNoUnder" target="_blank"> AQ-07 </a> ) </li><li>Gel Imager (Bio-Rad Laboratories, model: <a class="blackNoUnder" target="_blank"> Gel Doc 1000 </a> )</li><li>Incubator (SANYO, model: <a class="blackNoUnder" target="_blank"> MIR 262 </a> )</li><li>Thermocycler Veriti 96 well (Applied Biosystems, model: <a class="blackNoUnder" target="_blank"> 9902 </a> )</li><li>Genetic Analyzer (Invitrogen, model: <a class="blackNoUnder" target="_blank"> ABI PRISM 3130 </a> )</li><li>Ultra-low Freezer -80 Â°C (Forma Scientific, model: <a class="blackNoUnder" target="_blank"> 8270 </a> )<br/></li></ol><p class="pptt" id="biaoti34595">### Software</p><ol class="npwtwok"><li>GeneMapper v3.7. (Applied Biosystems, Foster City, CA, USA)</li><li>R v2.15.1. (R Foundation for Statistical Computing, Vienna, Austria)</li><li>RStudio v1.1.442 (RStudio Inc., Boston, MA, USA)<br/></li></ol><p class="pptt" id="biaoti34596">### Procedure</p><ol class="npwone"><li>DNA extraction procedure<ol class="npwtwo"><li>Collect 50-100 mg of leaf tissue in a 1.5 ml centrifuge microtube. Flash-freeze the tissue in liquid nitrogen and grind it with a plastic pestle.<br/></li><li>Add 300 Âµl of extraction buffer and mix it vigorously by vortexing (5-10 s) to homogenize the tissue. Incubate the homogenate at 65 Â°C for 30 min in a thermostatic bath.</li><li>Add 100 Âµl of chloroform:isoamyl alcohol (24:1) and mix the sample gently by inversion.</li><li>Centrifuge the homogenate at 20,800 <em>x g</em> at 4 Â°C for 5 min.</li><li>By using a 200 Âµl micropipette, carefully pipette the aqueous phase and transfer it to a new microtube.</li><li>Add 500 Âµl of ice-cold ethanol:acetate (24:1) and mix the sample by inverting the microtube 2-3 times to precipitate the DNA, and let it settle at -20 Â°C for 30 min.</li><li>Centrifuge the sample at 20,800 <em>x g</em> at 4 Â°C for 10 min to form a DNA pellet at the bottom of the microtube.</li><li>Discard the supernatant and clean the pellet with 100 Âµl of 70% ethanol. Vortex the tube for 1 min to wash the pellet.</li><li>Centrifuge the sample at 15,300 <em>x g</em> at room temperature for 5 min and let the pellet dry by inverting the microtube onto a paper towel for 30 min until the ethanol evaporates completely or by placing it in a drying chamber for 15 min.</li><li>Resuspend the DNA in 50 Âµl of ultrapure water with RNase A (50 Î¼gÂ·ml<sup>-1</sup>). Incubate at room temperature for 30 min.</li><li>Store the DNA sample at 4 Â°C until use. If the DNA will not be used for a long time, store it at -20 Â°C.</li><li>Check the integrity and purity of the extraction by loading 2 Î¼l of the DNA solution on a 0.8% agarose gel and running it at 90 V for 30 min. It should show a thick band of high molecular weight, above 21 Kbp, with no smear (Figure 1).</li><li>Measure the concentration of DNA samples in a spectrophotometer and prepare working solutions of ~100 ngÂ·Î¼l<sup>-1</sup>.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701184500_6160.jpg" width="145"/><br/><strong>Figure 1. Representative image of DNA samples visualized in a 0.8% agarose gel</strong><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Digestion with <em>EcoR</em>I<ol class="npwtwo"><li>Calculate the volume of the sample that corresponds to 700 ng of DNA, according to its concentration. However, consider that DNA concentration should be higher than 40.5 ngÂ·Î¼l<sup>-1</sup>, otherwise it will exceed the reactionâs final volume.</li><li>Calculate the amount of each reagent needed for the total number of samples based on Table 2. To compensate pipetting errors, it is convenient to consider an additional sample to the total number of analyzed samples.<br/><br/><strong>Table 2. Master mix for the enzymatic digestion with </strong><em><strong>EcoR</strong></em><strong>I</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701184538_9621.jpg" width="400"/><br/><br/></li><li>Prepare the Master mix, homogenize by vortexing (2-3 s) and centrifuge for 10 s.</li><li>Distribute the Master mix in individual tubes and add the DNA. Homogenize gently by inversion or by flicking the tubes (do not vortex) and centrifuge for 10 s.</li><li>Incubate at 37 Â°C overnight in an incubator.</li><li>Place the tubes on ice if you continue with the next step. Otherwise, store them in freezer at -20 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Digestion with <em>Hpa</em>II and <em>Msp</em>I<ol class="npwtwo"><li>For each tube from the previous reaction (B), distribute the content into two tubes (10 Î¼l in each tube) and label the new tubes with the sample code, adding the letter "H" to one and the letter "M" to the other. From now on you will work with two tubes for each analyzed sample.</li><li>Two Master mixes will be prepared, only differing on the restriction enzyme, one with <em>Hpa</em>II and the other with <em>Msp</em>I. For the two reactions, calculate the required amount of each reagent for the total number of samples plus one, based on Table 3.<br/><br/><strong>Table 3. Master mixes for enzymatic digestions with </strong><em><strong>Hpa</strong></em><strong>II y </strong><em><strong>Msp</strong></em><strong>I</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701184627_4161.jpg" width="360"/><br/><br/></li><li>Prepare the Master mixes, homogenize by vortexing (2-3 s) and centrifuge for 10 s.</li><li>Distribute the Master mix H in the tubes labeled with the letter "H" and the Master mix M in the tubes labeled with the letter "M". From now on you will have two tubes per sample. Homogenize gently by inversion or by flicking the tubes (do not vortex) and centrifuge for 10 s.</li><li>Incubate at 37 Â°C overnight in an incubator.</li><li>(Optional) Examine the efficiency of both digestions by running 6 Î¼l of the restriction products on a 1.2% agarose gel, at 90 V for 30 min. It should show a subtle smear from 5,100 to 100 bp (Figure 2).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701184707_6800.jpg" width="286"/><br/><strong>Figure 2. Representative image of restriction products visualized in a 1.2% agarose gel</strong><br/><br/></li><li>Keep the tubes on ice if you will continue with the next step. Otherwise, store them in the freezer at -20 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Adaptors ligation<ol class="npwtwo"><li>Prepare the double stranded <em>EcoR</em>I and <em>Msp</em>I/<em>Hpa</em>II adaptors from the single strand oligonucleotides by adding equal amounts of each oligo in a tube (Table 1). Place the tubes in the thermocycler and run the ADAPTORS program (Table 4) with a hot lid.<br/><br/><strong>Table 4. PCR programs</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701184802_7262.jpg" width="800"/><br/><br/></li><li>Based on Table 5, calculate the amount of each reagent for the total number of samples plus one.<br/><br/><strong>Table 5. Master mix for the ligation reaction</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185114_6785.jpg" width="450"/><br/><br/></li><li>Prepare the Master mix, homogenize by vortexing (2-3 s) and centrifuge for 10 s.</li><li>Distribute the Master mix in individual tubes and add 10 Î¼l of digestions. Homogenize gently by inversion or by flicking the tubes (do not vortex) and centrifuge for 10 s.</li><li>Incubate at room temperature for 3 h.</li><li>Keep the tubes on ice if you will continue with the next step. Otherwise, store them in a freezer at -20 Â°C until use.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>First amplification (Preamplification)<br/>For this PCR reaction, oligonucleotides with the same sequences as the adaptors plus 1 selective nucleotide are used. These oligonucleotides are identified as Primer <em>EcoR</em>I +1 and Primer H/M +1 (Table 1).<ol class="npwtwo"><li>Calculate the necessary amount of each reagent needed for the total samples based on Table 6, considering an additional sample to compensate for pipetting errors.<br/><br/><strong>Table 6. Master mix for the pre-amplification</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185153_6795.jpg" width="400"/><br/><br/></li><li>Prepare the Master mix, homogenize by vortexing (2-3 s) and centrifuge for 10 s.</li><li>Distribute 18 Î¼l of the Master mix in each tube and add 2 Î¼l of the ligations. Add 2 Î¼l of water to the remaining mix to use it as a negative reaction control. Homogenize gently by inversion or flicking (do not vortex) and centrifuge for 10 s.</li><li>Place the samples plus the negative control in the thermocycler and run the PREAMPLIFICATION program (Table 4) with a hot lid.</li><li>Check the efficiency of the reaction by loading 5 Î¼l of the pre-amplifications on a 1.2% agarose gel and running at 90 V for 30 min. It should show an intense smear with few or some diffuse bands from 1,500 to 100 bp (Figure 3).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185230_5877.jpg" width="235"/><br/><strong>Figure 3. Representative image of pre-amplification products visualized in a 1.2% agarose gel</strong><br/><br/></li><li>Dilute the pre-amplifications 1:3. Keep on ice if you continue with the next step or store in a freezer at -20 Â°C. Preamplifications can be safely stored in a freezer for several months.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Second amplification (Selective amplification)<br/>For this PCR reaction, oligonucleotides with the same sequences as the adaptors plus 3 selective nucleotides are used. These oligonucleotides are identified as Primer <em>EcoR</em>I +3 and Primer H/M +3 (Table 1).<ol class="npwtwo"><li>Calculate the necessary amount of each reagent needed for the total number of samples plus one, based on Table 7.<br/><br/><strong>Table 7. Master mix for the amplification reaction</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185306_1212.jpg" width="450"/><br/><br/></li><li>Prepare the Master mix, homogenize by vortexing (2-3 s) and centrifuge for 10 s.</li><li>Distribute 19 Î¼l of the Master mix in each tube and add 1 Î¼l of 1:3 diluted preamplifications. To the remaining mix add 1 Î¼l of water to use it as a negative reaction control. Homogenize gently by inversion or flicking (do not vortex) and centrifuge for 10 s.</li><li>Place in the thermocycler and run the AMPLIFICATION program (Table 4) with a hot lid.</li><li>Check the efficiency of the reaction by loading 5 Î¼l of the amplification products on a 1.2% agarose gel and running at 90 V for 30 min. Depending of the primer combination, it should show a smear with diffuse bands from 700 to 100 bp (Figure 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185344_4293.jpg" width="263"/><br/><strong>Figure 4. Representative image of amplification products visualized in a 1.2% agarose gel</strong><br/><br/></li><li>Keep the tubes on ice if you continue with the next step or store them in a freezer at -20 Â°C. Fluorescence-labeled amplifications can be safely stored at least for 6 months at -20 Â°C maintaining their light emission.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Electrophoresis<ol class="npwtwo"><li>Calculate the necessary amount of each reagent needed for the total samples based on Table 8, considering an extra sample to compensate for pipetting errors.<br/><br/><strong>Table 8. Master mix for the amplification reaction</strong><br/><img alt="" class="layerphoto" height="134" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185416_3671.jpg" width="600"/><br/><br/></li><li>Prepare the Master mix, homogenize by vortexing (2-3 s) and centrifuge for 10 s.</li><li>Distribute 9 Î¼l of the Master mix in each well of the 96-well plate and add 1 Î¼l of amplification products.</li><li>Place the plate in the thermocycler and run the program DENATURATION (Table 4) with a hot lid.</li><li>Take the plate out of the thermocycler and put it on ice.</li><li>Place the plate in the ABI Prism 3,130 DNA sequencer (Applied Biosystems) and run.</li></ol></li></ol><p class="pptt" id="biaoti34597">### Data analysis</p><ol class="npwone"><li>Allele calling and matrix generation<ol class="npwtwo"><li>Semi-automated scoring was performed on the resulting electronic profiles using GeneMapper v3.7 (Applied Biosystems). The length range of analyzed fragments was from 100 to 500 bp. Full specifications of the parameters used in GeneMapper are summarized in Cara <em>et al.</em> (2014). GeneMapperâs Report Manager tool was used to retrieve only sample names and allele columns into a Comma Separated Values (.csv) file.</li><li>Patterns of presence/absence between the <em>EcoR</em>I/<em>Hpa</em>II and the <em>EcoR</em>I/<em>Msp</em>I digests were codified from 0 to 3; then, this codification was converted into a binary matrix for either presence (1) or absence (0) of patterns 3, 2 or 1 (Figure 5). Pattern 0 was not codified in the binary matrix because the absence of fragments in <em>EcoR</em>I/<em>Hpa</em>II and <em>EcoR</em>I/<em>Msp</em>I digests might be due to either methylation of external cytosines or variations in the nucleotide sequence. The following R script was developed to perform this analysis.<br/><br/><strong>R Script for generating an MSAP binary matrix:</strong><br/><span style="font-family:Courier New;">##### R Commands used to generate an MSAP binary matrix from a Genemapper csv file</span><br/><br/><span style="font-family:Courier New;"> ## The following command makes sure that the working memory is clear</span><br/><span style="font-family:Courier New;"> rm(list = ls(all = TRUE))</span><br/><br/><span style="font-family:Courier New;"> ## Set working directory (PATH will be specific to where data are)</span><br/><span style="font-family:Courier New;"> setwd("C:/...")</span><br/><br/><span style="font-family:Courier New;"> ## Read in dataset. Input file should contain Samples by rows and Loci by columns, including header (loci names) and sample names in the first column</span><br/><span style="font-family:Courier New;"> ## Each sample digested by <em>Hpa</em>II and <em>Msp</em>I is identified as 'SampleName_H' and 'SampleName_M', one below the other. </span><br/><span style="font-family:Courier New;"> Data = read.csv("MSAP_Genemapper.csv", header=T, row.names = 1, check.names=F)</span><br/><br/><span style="font-family:Courier New;"> ## Create objects</span><br/><span style="font-family:Courier New;"> out &lt;- c()</span><br/><span style="font-family:Courier New;"> MSAP_patterns &lt;- matrix(nrow = dim(Data)[2], ncol = 0)</span><br/><br/><span style="font-family:Courier New;"> ## This is a double loop for j hybrids and i loci. It generates a matrix of MSAP patterns (0, 1, 2 or 3).</span><br/><span style="font-family:Courier New;"> for (j in 1:dim(Data)[1]) if (j %% 2 ==1)</span><br/><span style="font-family:Courier New;"> { </span><br/><span style="font-family:Courier New;"> for (i in 1:dim(Data)[2])</span><br/><span style="font-family:Courier New;"> {</span><br/><span style="font-family:Courier New;"> out &lt;- append(out, ifelse(Data[j,i]+Data[j+1,i]==0, '0',</span><br/><span style="font-family:Courier New;"> ifelse(Data[j,i]+Data[j+1,i]==2, '1',</span><br/><span style="font-family:Courier New;"> ifelse(Data[j,i]==1, '2', '3'))))</span><br/><span style="font-family:Courier New;"> }</span><br/><span style="font-family:Courier New;"> MSAP_patterns &lt;- cbind(MSAP_patterns, matrix(out, dimnames = list(colnames(Data), substr(row.names(Data)[j], 1, nchar(as.character(row.names(Data)[j]))-2))))</span><br/><span style="font-family:Courier New;"> out &lt;- c()</span><br/><span style="font-family:Courier New;"> }</span><br/><br/><span style="font-family:Courier New;"> ##This is a loop that creates a binary matrix of presence/absence of each pattern (1, 2 or 3) for i loci.</span><br/><span style="font-family:Courier New;"> dummies &lt;- matrix(nrow = dim(MSAP_patterns)[2], ncol = 0)</span><br/><span style="font-family:Courier New;"> for (i in 1:dim(MSAP_patterns)[1])</span><br/><span style="font-family:Courier New;"> {</span><br/><span style="font-family:Courier New;"> dummies &lt;- cbind(dummies, matrix(as.numeric(MSAP_patterns[i,] == 1), dimnames = list(colnames(MSAP_patterns), paste(row.names(MSAP_patterns)[i], "1", sep = "_"))), matrix(as.numeric(MSAP_patterns[i,] == 2), dimnames = list(colnames(MSAP_patterns), paste(row.names(MSAP_patterns)[i], "2", sep = "_"))), matrix(as.numeric(MSAP_patterns[i,] == 3), dimnames = list(colnames(MSAP_patterns), paste(row.names(MSAP_patterns)[i], "3", sep = "_"))))</span><br/><span style="font-family:Courier New;"> }</span><br/><span style="font-family:Courier New;"> binary_matrix &lt;- dummies[,colSums(dummies != 0) != 0]</span><br/><br/><span style="font-family:Courier New;"> ## Save output to working directory</span><br/><span style="font-family:Courier New;"> write.csv(binary_matrix, file = "MSAP_binary_matrix.csv") </span><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701185520_5855.jpg" width="150"/><br/><strong>Figure 5. Methylation-sensitive Amplified Polymorphism (MSAP) pattern analysis.</strong> A. Inference of methylation status of CCGG restriction sites from the fragment amplification profiles and DNA methylation pattern codification. Black C represents methylated cytosines. B. Binary matrix obtained by codifying the presence (1) or absence (0) of the particular methylation pattern.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Determination of species-specific and novel fragments<ol class="npwtwo"><li>The amplification patterns observed in the evaluated natural hybrids were compared with the patterns obtained in the synthetic hybrids and their parental genotypes. Fragments present in the synthetic hybrids were categorized as <em>S. microdontum</em> or <em>S. kurtzianum</em> species-specifics. In addition, the presence of novel amplification fragments in the synthetic hybrids (that is present in the hybrids but absent in the genotypes of the parental species <em>S. microdontum</em> and <em>S. kurtzianum</em>) was assessed in the natural hybrids. Novel fragments were classified as <em>S.</em> x <em>rechei</em> species-specific if they were also present in at least one of the <em>S.</em> x <em>rechei</em> evaluated genotypes or as novel if they were only observed in the synthetic hybrids. The following R script was developed to perform this analysis.<br/><br/><strong>R Script for analyzing MSAP epiloci inheritance in hybrids:</strong><br/><span style="font-family:Courier New;">#### R Commands used to analyze MSAP epiloci inheritance in Hybrids</span><br/><br/><span style="font-family:Courier New;"> ## The following command makes sure that the working memory is clear</span><br/><span style="font-family:Courier New;"> rm(list = ls(all = TRUE))</span><br/><br/><span style="font-family:Courier New;"> ## Set working directory (PATH will be specific to where data are)</span><br/><span style="font-family:Courier New;"> setwd("C:/...")</span><br/><br/><span style="font-family:Courier New;"> ## Read in dataset. Data is organized as Samples by rows and Loci by columns</span><br/><span style="font-family:Courier New;"> Data = read.csv("MSAP_binary_matrix.csv", header=T, row.names = 1, check.names=F)</span><br/><br/><span style="font-family:Courier New;"> ## Create objects</span><br/><span style="font-family:Courier New;"> out &lt;- c()</span><br/><span style="font-family:Courier New;"> results &lt;- matrix(nrow = 4, ncol = 0)</span><br/><br/><span style="font-family:Courier New;"> ## This is a double loop for j hybrids and i loci. Rows are as follows: 1-5 Hybrids, 6 <em>S. kurtzianum</em> (Parental1), 7 <em>S. microdontum</em> (Parental2), 8-15 S. x rechei (Natural hybrid). Any modifications in these numbers should be applied to the following commands.</span><br/><span style="font-family:Courier New;"> for (j in 1:5)</span><br/><span style="font-family:Courier New;"> {</span><br/><span style="font-family:Courier New;"> for (i in 1:dim(Data)[2])</span><br/><span style="font-family:Courier New;"> {</span><br/><span style="font-family:Courier New;"> out &lt;- append(out, ifelse(Data[j,i]&gt;0 &amp; Data[6,i]&gt;0 &amp; Data[7,i]==0 &amp; sum(Data[8:15,i])==0, 'A',  ## Fragments shared only by hybrids and <em>S. kurtzianum</em></span><br/><span style="font-family:Courier New;"> ifelse(Data[j,i]&gt;0 &amp; Data[6,i]==0 &amp; Data[7,i]&gt;0 &amp; sum(Data[8:15,i])==0, 'B',  ## Fragments shared only by hybrids and <em>S. microdontum</em></span><br/><span style="font-family:Courier New;"> ifelse(Data[j,i]&gt;0 &amp; Data[6,i]==0 &amp; Data[7,i]==0 &amp; sum(Data[8:15,i])&gt;0, 'C',  ## Fragments shared only by hybrids and S. x rechei</span><br/><span style="font-family:Courier New;"> ifelse(Data[j,i]&gt;0 &amp; Data[6,i]==0 &amp; Data[7,i]==0 &amp; sum(Data[8:15,i])==0, 'D', 'E')))))  ## Fragments specific to hybrids</span><br/><span style="font-family:Courier New;"> }</span><br/><br/><span style="font-family:Courier New;"> results &lt;- cbind(results, matrix(table(out, exclude = "E"),dimnames = list(c("Kurtzianum", "Microdontum", "Rechei", "Hybrid"), row.names(Data)[j])))</span><br/><span style="font-family:Courier New;"> out &lt;- c()</span><br/><span style="font-family:Courier New;"> } </span><br/><br/><span style="font-family:Courier New;"> ## Transform counts by hybrid into percentages </span><br/><span style="font-family:Courier New;"> percentages &lt;- t(results)/colSums(results)*100</span><br/><br/><span style="font-family:Courier New;"> ## Save output to working directory</span><br/><span style="font-family:Courier New;"> write.csv(percentages, file = "MSAP_Hybrids.csv")</span><br/><br/><span style="font-family:Courier New;"> ## A basic graphical representation</span><br/><span style="font-family:Courier New;"> barplot(t(percentages), horiz = TRUE, las=1, col = c("red", "blue", "green", "yellow"))</span> </li></ol></li></ol><p class="pptt" id="biaoti34598">### Recipes</p><ol class="npwtwok"><li>Extraction buffer<br/>100 mM Tris-HCl (pH 8.0)<br/>20 mM EDTA (pH 8.0)<br/>1.4 M NaCl<br/>0.4% (v/v) Î²-mercaptoethanol<br/>2% (w/v) CTAB</li><li>1x TBE (Tris-Borate-EDTA) gel running buffer<br/>89 mM Tris base<br/>89 mM Boric acid<br/>2 mM EDTA (pH 8.0)</li><li>6x DNA loading buffer<br/>30% (v/v) Glycerol<br/>0.25% (w/v) Bromophenol blue<br/>0.25% (w/v) Xylene cyanol FF<br/></li></ol><p class="pptt" id="biaoti34599">### Notes</p><p class="ppzz">Prepare all Master mixes by first adding the larger volumes and then the smaller ones, leaving the enzymes to the end.</p><p class="pptt" id="biaoti34600">### Acknowledgments</p><p class="ppzz">We thank Dr MarÃ­a Virginia Sanchez Puerta for critical reading of the draft manuscript. This work was supported by Agencia Nacional de PromociÃ³n CientÃ­fica y TecnolÃ³gica and Universidad Nacional de Cuyo, Argentina PICT 1243. This protocol was adapted from our previous work (Cara <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti34601">### Competing interests</p><p class="ppzz">The authors have not competing interest.</p><p class="pptt" id="biaoti34602">### References</p><ol class="npwtwok"><li>Camadro, E. L., Erazzu, L. E., Maune, J. F. and Bedogni, M. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22372767" target="_blank">A genetic approach to the species problem in wild potato.</a> <em>Plant Biol (Stuttg)</em> 14(4): 543-554.</li><li>Cara, N., Ferrer, M. S., Masuelli, R. W., Camadro, E. L. and Marfil, C. F. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30681145" target="_blank">Epigenetic consequences of interploidal hybridisation in synthetic and natural interspecific potato hybrids.</a> <em>New Phytol</em> 222(4): 1981-1993.</li><li>Cara, N., Marfil, C. F., GarcÃ­a Lampasona, S. C., Masuelli, R. W. (2014). <a href="https://www.nrcresearchpress.com/doi/abs/10.1139/cjb-2014-0047#.XoQ9_agzaUk" target="_blank">Comparison of two detection systems to reveal AFLP markers in plants.</a> <em>Botany</em> 92: 607-610.</li><li>Vos, P., Hogers, R., Bleeker, M., Reijans, M., van de Lee, T., Hornes, M., Frijters, A., Pot, J., Peleman, J., Kuiper, M. and <em>et al.</em> (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7501463" target="_blank">AFLP: a new technique for DNA fingerprinting.</a> <em>Nucleic Acids Res</em> 23(21): 4407-4414.<br/></li></ol>