<body>
            <h1>Immunofluorescent Staining of Claudin-2 in Cultured Kidney Tubular Cells</h1>
            <p><strong>Authors:</strong> Shaista Anwer, Katalin Szaszi</p>
            <h2>Abstract</h2>
            <p>Members of the claudin family of tight junction proteins regulate paracellular permeability and modulate cell signaling. During junction remodeling, these proteins are selectively inserted into or retrieved from the tight junctions, but the control and coordination of these processes remain incompletely understood. Visualization of claudins allows the assessment of changes in their localization and abundance. We use the described protocol to stain claudin-2, but it can also be adapted to stain any tight junction protein. We found that using methanol for fixing allows the best preservation of claudin-2 both at the membrane and in cytoplasmic vesicles. Staining is done using a claudin-2 specific primary and a fluorescently labelled secondary antibody, along with DAPI to label nuclei. The samples are then imaged using confocal microscopy, and a z-stack is obtained allowing visualization of both junctional and intracellular claudin-2. Total claudin-2 signal can be quantified after 3D reconstruction of the images using the Imaris software.</p>
            <p><strong>Keywords:</strong> Claudin, Epithelial cells, Confocal microscopy, Indirect immunofluorescent staining, Fixed cells, Maximum intensity projection, 3D reconstitution, Analysis</p>
            <h2>Background</h2><p>Tight junctions (TJ) are multiprotein complexes localized at the apical-most portion of the intercellular junctional complexes that connect epithelial cells (Van Itallie and Anderson, 2014). These structures generate a permeability barrier and ion-specific paracellular pathways, maintain apico-basal polarity, provide input for various vital functions and modulate signaling pathways. The proteins located at the TJs can be divided into transmembrane and associated cytoplasmic proteins (reviewed in (GonzÃ¡lez-Mariscal <em>et al.</em>, 2003). The transmembrane proteins can be further subdivided into three major categories: claudins, tight junction-associated Marvel proteins (<em>e.g.</em>, occludin) and single span proteins (<em>e.g.</em>, Junctional Adhesion Molecules). TJ-associated cytosolic proteins include a large array of adaptors connected to signaling and cytoskeletal proteins. Collectively, these proteins are referred to as the cytoplasmic plaque. Claudins are small molecular weight (20-34 kDa) tetraspan membrane proteins that are essential components of the TJs (Tsukita <em>et al.</em>, 2019). They incorporate into the TJ strands that typically contain a mosaic of various claudin isoforms that determine permselectivity (Van Itallie and Anderson, 2004). Interestingly, claudins have also recently emerged as key modulators of signaling, an effect that is likely due to the interactions with cytosolic adapters.<br/> <br/>In this protocol we focus on claudin-2 (Cldn-2), a 230 amino acid member of the family, with a calculated molecular mass of 24.5 kDa. Cldn-2 was first described by Dr Shoichiro Tsukita and coworkers (Furuse <em>et al.</em>, 1998a and 1998b) (for review see Venugopal <em>et al.</em>, 2019). It is highly enriched in the kidney proximal tubules (Enck <em>et al.</em>, 2001) and in intestinal and liver cells (<em>e.g.</em>, Sakaguchi <em>et al.</em>, 2002; Escaffit <em>et al.</em>, 2005). It generates a cation selective paracellular channel, and therefore, its presence corresponds to elevated paracellular permeability (<em>e.g.</em>, Amasheh <em>et al.</em>, 2002). Multiple studies have revealed that in various cells Cldn-2 can be found not only at the membrane, <em>i.e.</em>, in the TJs, but also in intracellular vesicles (Dukes <em>et al.</em>, 2012; Lu <em>et al.</em>, 2014; Amoozadeh <em>et al.</em>, 2015), and in the nucleus (Ikari <em>et al.</em>, 2014; Amoozadeh <em>et al.</em>, 2018).<br/><br/>Many studies have used immunofluorescent staining in fixed cells followed by confocal microscopy to visualize TJ proteins and analyze their localization and abundance. These approaches allow investigation of the mechanisms that regulate TJ protein trafficking. The protocols differ in the fixation and blocking method and use primary antibodies targeted against specific proteins. Here we describe the method used in our lab to stain Cldn-2 in cultured kidney tubular cells based on (Amoozadeh <em>et al.</em>, 2015 and 2018; Dan <em>et al.</em>, 2019). We also detail our protocol for 3D reconstitution and analysis of changes in staining intensity following treatment, <em>e.g.</em> with cytokines. While we focus specifically on claudin-2, the same approach can be used for other TJ proteins in any epithelial or endothelial cells.</p>
        </body>

<p class="pptt" id="biaoti34697">## Materials and Reagents</p><ol class="npwtwok"><li>Micro cover glasses, round, No. 1, 18 mm (VWR<sup>Â®</sup> International, catalog number: <a class="blackNoUnder" target="_blank"> CA-48380-046 </a> ), sterilize coverglasses using autoclave</li><li>Microscope slides (Fisherbrand, Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-550-15 </a> )</li><li>Clear tissue culture-treated 12-well microplates, (Corning Costar<sup>Â®</sup>, Millipore-Sigma, catalog number: <a class="blackNoUnder" target="_blank"> CLS3513 </a> )</li><li>Parafilm</li><li>Paper towel</li><li>Cell line: LLC-PK<sub>1</sub> kidney tubule epithelial cell line (European Collection of Authenticated Cell Cultures (Salisbury, UK), catalog number: <a class="blackNoUnder" target="_blank"> 86121112 </a> )</li><li>For cell culture:<br/><ol class="npwthree"><li>Dulbecco's Modified Eagle Medium (D-MEM), low glucose, with L-glutamine<br/></li><li>110 mg/L sodium pyruvate (Thermo Fisher, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 11885084 </a> )<br/></li><li>10% Fetal Bovine Serum (FBS) (Thermo Fisher, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 12483-020 </a> )</li><li>1% Penicillin-Streptomycin (Penicillin-Streptomycin, 100x, sterile-filtered, cell culture tested) (Millipore-Sigma Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P-4333 </a> )<br/></li><li>Trypsin-EDTA (0.05% Trypsin with EDTA 4Na) (Thermo Fisher, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 25300062 </a> )<br/></li><li>Phosphate buffered saline (PBS) (Thermo Fisher, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 10010023 </a> )</li></ol></li><li>Methanol (BioShop Canada, catalog number: <a class="blackNoUnder" target="_blank"> Met302 </a> ), cool to -20 Â°C before using</li><li>10x PBS Buffer (pH 7.4 Sterile, w/o Ca, Mg) (BioShop Canada, catalog number: <a class="blackNoUnder" target="_blank"> PBS415 </a> ), open in hood to maintain sterility for storage</li><li>Bovine serum albumin (BSA) (fraction V, heat shock isolation, &gt; 98%) (BioShop Canada, catalog number: <a class="blackNoUnder" target="_blank"> ALB001 </a> )</li><li>Polyclonal rabbit anti-Claudin-2 antibody (Thermo Fisher Scientific, Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 28530 </a> ), aliquot to avoid repeated thawing, store at -20 Â°C</li><li>Fluorophore-labeled secondary antibody:<br/>Donkey anti-Rabbit IgG labelled with Alexa Fluor 488 (green) (Thermo Fischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A-21206 </a> )<br/>or <br/>Donkey anti-Rabbit IgG labelled with Alexa Fluor 555 (red) (Thermo Fischer Scientific, catalog number: A- A31572 )</li><li>2,4-diamidino-2-phenylindole (DAPI) (Thermo Fischer Scientific, catalog number: <a class="blackNoUnder" target="_blank"> D1306 </a> )<br/>Make a 5 mg/ml DAPI stock solution by dissolving the contents of one vial (10 mg) in 2 ml of deionized water. Aliquot the stock, protect from light. DAPI stock solution can be stored at 4 Â°C for up to 6 months, or at -20 Â°C for longer periods.</li><li>Fluorescent mounting medium (Dako, Agilent catalog number: <a class="blackNoUnder" target="_blank"> S3023 </a> ), store at 4 Â°C</li><li>Cell culture medium (see Recipes)</li><li>1x PBS (see Recipes)</li><li>Blocking buffer (see Recipes)</li><li>Antibody buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34698">### Equipment</p><ol class="npwtwok"><li>Tweezers</li><li>Nikon Eclipse TS100 microscope</li><li>WaveFX spinning-disk confocal microscope (Quorum Technologies, Guelph, Canada) with an ORCA-flash4.0 digital camera with Gen II sCMOS image sensor<br/></li></ol><p class="pptt" id="biaoti34699">### Software</p><ol class="npwtwok"><li>Metamorph Image Analysis Software, Molecular Devices</li><li>Imaris software 8.0.2 (Bitplane)<br/></li></ol><p class="pptt" id="biaoti34700">### Procedure</p><ol class="npwone"><li>Seeding cells on cover glasses<ol class="npwtwo"><li>Grow LLC-PK<sub>1</sub> cells in a T75 flask with culture medium (low glucose DMEM supplemented with 10 % FBS and 1% Pen-Strep, see Recipes) in an incubator with 5% CO<sub>2</sub>. Use flask of LLC-PK<sub>1</sub> cells grown to about 90% confluence. Rinse with 5-8 ml PBS and add 1-2 ml trypsin according to standard cell culture procedures. Place back into incubator for about 10 min, until cells are floating. Note that details of culturing have to be adjusted for specific cell lines.</li><li>While cells are in trypsin, place autoclaved coverglass in multi-well plates using sterile tweezers.</li><li>When cells are floating, resuspend them in 8 ml fresh medium, mix well and seed an equal number of cells onto coverglasses. Use 2 x 10<sup>5</sup>-5 x 10<sup>5</sup>/ml cells in 1 ml medium/well (12-well plate).</li><li>Gently shake the culture dish to evenly distribute the cells and place into culture incubator.</li><li>Grow cells to 100% confluency (about 48 h). Of note, we found that expression of Cldn-2 is low in subconfluent cell cultures and increases as the cells become confluent (Amoozadeh <em>et al.</em>, 2018). Therefore, it is important to allow the cells to become fully confluent and generate mature TJs, before performing the immunostaining (see Figure 1).<br/><br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200712/20200712085445_9169.jpg" width="300"/><br/><strong>Figure 1. Subconfluent (A) and confluent (B) LLC-PK<sub>1</sub> cultures.</strong> Phase contrast images of live cells grown on cover glasses were obtained using a Nikon eclipse TS100 microscope (10x objective).<br/><br/></li></ol></li><li>Fixation<ol class="npwtwo"><li>Treat cells according to experimental needs, <em>e.g.</em>, with cytokines. At the end of the treatment, aspirate the medium and quickly rinse with 1x PBS. The steps after rinsing do not need to be performed under sterile conditions. The multi-well dish used for culturing cells can be used for the staining protocol under non-sterile conditions.</li><li>Aspirate PBS and immediately fix with 0.5-1 ml methanol (stored at -20 Â°C) for 5 min. Make sure the methanol fully covers the cover glasses. These 2 steps should be done as fast as possible.</li><li>Wash with 1x PBS 5 times. After this step the samples can be stored overnight at 4 Â°C if needed. However, we prefer to proceed to the next steps immediately. Longer storage is not recommended.<br/><br/></li></ol></li><li>Blocking, staining and mounting<ol class="npwtwo"><li>Block with 0.5-1 ml blocking buffer in PBS for 1 h at room temperature. Make sure the buffer fully covers the cover glasses.</li><li>Primary antibody incubation:<br/><ol class="npwthree"><li>Dilute antibody 1:100 in antibody buffer.<br/></li><li>To reduce antibody use, place 50 Î¼l of diluted antibody on parafilm and place the cover glass with cells facing the drop. Avoid bubbles and donât let cover glass dry out. Incubate for 1h at room temperature.<br/></li><li>Non-specific secondary antibody binding control: use one of the cover glasses to demonstrate specificity of the secondary antibody. Omit primary antibody but do all other steps similarly.<br/></li><li>Control sample for primary antibody specificity: use samples where Cldn-2 was silenced or knocked down (Dan <em>et al.</em>, 2019).</li></ol></li><li>At the end of the incubation use tweezers to place cover glasses back into the same culture dish turning the cells upside. Wash with 1x PBS 5 times.</li><li>Secondary antibody incubation:<br/><ol class="npwthree"><li>Dilute secondary antibody (1:1,000) and DAPI (1:5,000) in antibody buffer.<br/></li><li>Add 500 Î¼l antibody solution to the cover glasses.<br/></li><li>Incubate for 1h at room temperature in the dark.</li></ol></li><li>Wash with 1x PBS 5 times over 15 min.</li><li>Label microscope slides to identify samples. Place a drop of fluorescent mounting medium on a microscope slide. Lift up cover glasses using a tweezer and touch the edge to a paper towel to remove excess fluid. Place stained cover glass onto the drop of fluorescent mounting medium with cells facing down. Avoid air bubbles. Aspirate access fluid.</li><li>Let the fluorescent mounting medium dry for at least 30 min before imaging or storing. Store dried slides in the dark at 4 Â°C.<br/><br/></li></ol></li><li>Visualization using confocal microscopy<ol class="npwtwo"><li>Set up the confocal microscope<br/><ol class="npwthree"><li>Settings (<em>e.g.</em>, laser, objective etc) will vary depending on your experiment. We use a 63x objective, which was chosen to give the desired resolution and size of the imaged field.<br/></li><li>Chose the excitation/emission settings for the fluorophore of your secondary antibody (Alexa488 or 555 in our experiments) and DAPI.<br/></li><li>Select exposure time to avoid image saturation. Obtain z-stack sections at both wavelength through the whole sample by defining the top and bottom focal plane positions and a step size of around 2 Î¼m (12-16 focal planes).</li></ol></li><li>A single plane at the level of the TJs of the stack can be used for display. However, LLC-PK<sub>1</sub> cells in a monolayer often have slight height differences and can also form domes. This causes the TJs in different areas of a visual field to be at slightly different focal planes, making some areas to be out of focus. Further, TJ-localized and intracellular Cldn-2 are also in different focal planes. To visualize all Cldn-2 within the cells, use Metamorph to generate a Maximum Intensity Projection from the z-stack (see Figure 2 for an example). This process uses the brightest pixel from each layer and displays its intensity value in the final image, thus allowing visualization of a 3D image collapsed into a 2D picture. The processing must be disclosed in the manuscript.<br/><br/><img alt="" class="layerphoto" height="324" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200712/20200712085533_5887.jpg" width="300"/><br/><strong>Figure 2. Typical Claudin-2 staining in LLC-PK<sub>1</sub> cells.</strong> LLC-PK<sub>1</sub> cells were fixed with methanol, and Cldn-2 stained as described in the protocol. The nuclei were counterstained with DAPI. Z-stacks were obtained using confocal microscopy for each staining, maximal intensity projection pictures were generated using the Metamorph software. The images obtained for the two wavelengths were combined. The bar represents 10 Î¼m.<br/></li></ol></li></ol><p class="pptt" id="biaoti34701">### Data analysis</p><p class="ppzz"><strong>Image processing and data analysis:</strong> </p><ol class="npwone"><li>3D reconstruction In Imaris<br/>To quantify intensity of total Cldn-2 staining, we use the Imaris software for a 3D reconstruction that can then be used for quantification. Of note, the above generated Maximum Intensity Projection images cannot be used for quantification. For basic familiarity with the Imaris software, users are referred to free online tutorials at <a href="http://www.bitplane.com/learning" target="_blank">http://www.bitplane.com/learning</a>.<br/><ol class="npwtwo"><li>Upload z-stack images to the Imaris 8.0.2 software.</li><li>Click âEdit/Add Channelsâ and select the appropriate image.</li><li>In the main menu, Select âSurpass/Add new surfacesâ and click âCreate Surfaceâ.</li><li>Select the appropriate âSurface Area Detail Levelâ. Select âAbsolute Intensityâ for thresholding.<br/><em>Note: Both settings are automatically calculated, however, users can alter these parameters. Keep these settings constant throughout image analysis.</em> </li><li>Click âFinishâ to complete 3D reconstruction of the stack.</li><li>Create a 3D spot object for quantifying the number of nuclei by adding a new spot object.</li><li>Select âDifferent Spot Sizeâ.</li><li>For âSource Channelâ, select the DAPI channel.</li><li>For âSpot Detectionâ, enter âEstimated XY Diameter and Z Diameterâ of the nuclei.</li><li>Manually adjust the âSpot Qualityâ to include any undetected nuclei.</li><li>Select âIntensity Max = masked channel number of the cell outlineâ to restrict extra nuclei spots. Note that the values for all the described parameters depend on the properties of the images acquired and will differ in specific images.</li><li>Adjust âSpot Region Thresholdâ and ensure all nuclei spots are detected.</li><li>Click âFinishâ to complete the detection of nuclei spots.</li><li>Select the required statistic parameters (volume and total number of spots) and export the .csv files. These can be opened using a graph program.<br/><br/></li></ol></li><li>Data analysis<ol class="npwtwo"><li>Use a graph program (Excel or GraphPad) to calculate the sum of volumes of each surface (Î¼m<sup>3</sup>).</li><li>Express the data as total surface volume per cell (nuclei spots).</li></ol></li></ol><p class="pptt" id="biaoti34702">### Notes</p><ol class="npwtwok"><li><u>Cell line choice</u>: The LLC-PK<sub>1</sub> cell line we use originated from the European Collection of Authenticated Cell Cultures. This LLC-PK<sub>1</sub> clone was originally also available at ATCC (ATCC<sup>Â®</sup> CRL-1392<sup>TM</sup>) but has since been discontinued. ATCC distributes another LLC-PK<sub>1</sub> cell line, however, we found that LLC-PK<sub>1</sub> cells from various sources differ in their claudin-2 expression levels. Thus, prior to choosing a cell line for the project, good expression of the claudin of interest should be verified.</li><li><u>Cell counting and confluency</u>: TJs develop as the layer becomes confluent and therefore it is recommended that fully confluent layers are used. Claudin-2 total expression is low in subconfluent cells and increases as confluency develops. Confluency should be kept constant when comparing different conditions. We found that when the same conditions are consistently applied during culture and seeding, cell numbers can be kept constant. Thus, after establishing the protocol, routine cell counting is not necessary. However, experimental manipulations might alter cell growth and thus confluency should be routinely checked and considered as an important variable.</li><li>Throughout the staining process, drying of the cover glasses must be avoided.</li><li>The protocol described here is used to stain claudin-2. The same protocol can be used to visualize other junction proteins in any epithelial cell lines. We found that the mode of fixation is key for good preservation and visualization of junctional proteins. We had good results using methanol when staining claudins. For occludin we used acetone fixation. For these fixation methods additional permeabilization is not needed. For a detailed analysis of the various fixation methods used for junctional proteins, please refer to Buckley <em>et al.</em>, 2018.</li><li>The quantification described is not specific for TJ-localized claudin-2 but reflects total claudin-2 abundance, including cytosolic protein. In our experiments the quantification was used for a comparison between control and TNF-Î±-treated samples. The data were expressed as staining/cell, relative to the control (untreated) to demonstrate fold changes. In this case, the control, untreated samples serve as reference. For other type of comparisons (<em>e.g.</em>, comparison of untreated samples), a reference protein, <em>e.g.</em>, another TJ protein should be used (<em>e.g.</em>, costaining for occludin or ZO-1 and expressing ratio of claudin-2 to occludin staining). The choice of the reference however can pose difficulties for cytokine-treated samples, as we found that expression of all TJ proteins we tested (<em>e.g.</em>, other claudins, occludin), as well as the adherens junction protein E-cadherin were all altered by TNFÎ± (Amoozadeh <em>et al.</em>, 2015). Thus, before choosing a reference, it is important to show that it is unaltered by the treatment. Finally, quantification should be verified by other quantitative methods, such as Western blotting.</li></ol><p class="pptt" id="biaoti34703">### Recipes</p><ol class="npwtwok"><li>Cell culture media<br/>Dulbecco's Modified Eagle Medium (D-MEM), low glucose, with L-glutamine and 110 mg/L sodium pyruvate was supplemented with 10% Fetal Bovine Serum (FBS) and sterile-filtered 1% Penicillin-Streptomycin</li><li>1x PBS<br/>Dilute from 10x PBS: add 50 ml 10x PBS to 450 ml distilled H<sub>2</sub>O</li><li>Blocking buffer<br/>3% BSA in PBS<br/><ol class="npwthree"><li>Mix 1.5 g BSA in 50 ml PBS<br/></li><li>Store short term (1 day) at 4 Â°C</li></ol>Long-term storage not recommended</li><li>Antibody buffer<br/>0.3% BSA in PBS<br/><ol class="npwthree"><li>Mix 0.15 g BSA in 50 ml PBS<br/></li><li>Store short term (1 day) at 4 Â°C</li></ol>Long-term storage not recommended</li></ol><p class="pptt" id="biaoti34704">### Acknowledgments</p><p class="ppzz">Funding: Kidney Foundation of Canada, Canadian Institutes of Health Research (CIHR) grants PJT-149058 and MOP-142409.<br/>   This protocol was derived from: Amoozadeh <em>et al.</em>, 2015 and 2018; Dan <em>et al.</em>, 2019.</p><p class="pptt" id="biaoti34705">### Competing interests</p><p class="ppzz">The authors declare that they have no financial or non-financial conflicts of interest.</p><p class="pptt" id="biaoti34706">### References</p><ol class="npwtwok"><li>Amasheh, S., Meiri, N., Gitter, A. H., Schoneberg, T., Mankertz, J., Schulzke, J. D. and Fromm, M. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12432083" target="_blank">Claudin-2 expression induces cation-selective channels in tight junctions of epithelial cells.</a> <em>J Cell Sci</em> 115(Pt 24): 4969-4976.</li><li>Amoozadeh, Y., Anwer, S., Dan, Q., Venugopal, S., Shi, Y., Branchard, E., Liedtke, E., Ailenberg, M., Rotstein, O. D., Kapus, A. and Szaszi, K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29187366" target="_blank">Cell confluence regulates claudin-2 expression: possible role for ZO-1 and Rac.</a> <em>Am J Physiol Cell Physiol</em> 314(3): C366-C378.</li><li>Amoozadeh, Y., Dan, Q., Xiao, J., Waheed, F. and Szaszi, K. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25948735" target="_blank">Tumor necrosis factor-Î± induces a biphasic change in claudin-2 expression in tubular epithelial cells: role in barrier functions.</a> <em>Am J Physiol Cell Physiol</em> 309(1): C38-50.</li><li>Buckley, A. G., Looi, K., Iosifidis, T., Ling, K. M., Sutanto, E. N., Martinovich, K. M., Kicic-Starcevich, E., Garratt, L. W., Shaw, N. C., Lannigan, F. J., Larcombe, A. N., Zosky, G., Knight, D. A., Rigby, P. J., Kicic, A. and Stick, S. M. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29434527" target="_blank">Visualisation of multiple tight junctional complexes in human airway epithelial cells.</a> <em>Biol Proced Online</em> 20: 3. </li><li>Dan, Q., Shi, Y., Rabani, R., Venugopal, S., Xiao, J., Anwer, S., Ding, M., Speight, P., Pan, W., Alexander, R. T., Kapus, A. and SzÃ¡szi, K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31481470" target="_blank">Claudin-2 suppresses GEF-H1, RHOA, and MRTF, thereby impacting proliferation and profibrotic phenotype of tubular cells.</a> <em>J Biol Chem</em> 294(42): 15446-15465.</li><li>Dukes, J. D., Whitley, P. and Chalmers, A. D. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22396724" target="_blank">The PIKfyve inhibitor YM201636 blocks the continuous recycling of the tight junction proteins claudin-1 and claudin-2 in MDCK cells.</a> <em>PLoS One</em> 7(3): e28659.</li><li>Enck, A. H., Berger, U. V. and Yu, A. S. L. (2001). <a href="https://pubmed.ncbi.nlm.nih.gov/11592954/" target="_blank">Claudin-2 is selectively expressed in proximal nephron in mouse kidney.</a> <em>Am J Physiol Renal Physiol</em> 281(5): F966-F974.</li><li>Escaffit, F., Boudreau, F. and Beaulieu, J. F. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15389642" target="_blank">Differential expression of claudin-2 along the human intestine: Implication of GATA-4 in the maintenance of claudin-2 in differentiating cells.</a> <em>J Cell Physiol</em> 203(1): 15-26.</li><li>Furuse, M., Fujita, K., Hiiragi, T., Fujimoto, K. and Tsukita, S. (1998a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9647647" target="_blank">Claudin-1 and -2: novel integral membrane proteins localizing at tight junctions with no sequence similarity to occludin.</a> <em>J Cell Biol</em> 141(7): 1539-1550.</li><li>Furuse, M., Sasaki, H., Fujimoto, K. and Tsukita, S. (1998b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9786950" target="_blank">A single gene product, claudin-1 or -2, reconstitutes tight junction strands and recruits occludin in fibroblasts.</a> <em>J Cell Biol</em> 143(2): 391-401.</li><li>GonzÃ¡lez-Mariscal, L., Betanzos, A., Nava, P. and Jaramillo, B. E. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12475568" target="_blank">Tight junction proteins.</a> <em>Prog Biophys Mol Biol</em> 81(1): 1-44.</li><li>Ikari, A., Watanabe, R., Sato, T., Taga, S., Shimobaba, S., Yamaguchi, M., Yamazaki, Y., Endo, S., Matsunaga, T. and Sugatani, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24907662" target="_blank">Nuclear distribution of claudin-2 increases cell proliferation in human lung adenocarcinoma cells.</a> <em>Biochim Biophys Acta</em> 1843(9): 2079-2088.</li><li>Lu, R., Johnson, D. L., Stewart, L., Waite, K., Elliott, D. and Wilson, J. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24694596" target="_blank">Rab14 regulation of claudin-2 trafficking modulates epithelial permeability and lumen morphogenesis.</a> <em>Mol Biol Cell</em> 25(11): 1744-1754.</li><li>Sakaguchi, T., Gu, X., Golden, H. M., Suh, E., Rhoads, D. B. and Reinecker, H. C. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11934881" target="_blank">Cloning of the human claudin-2 5'-flanking region revealed a TATA-less promoter with conserved binding sites in mouse and human for caudal-related homeodomain proteins and hepatocyte nuclear factor-1Î±.</a> <em>J Biol Chem</em> 277(24): 21361-21370.</li><li>Tsukita, S., Tanaka, H. and Tamura, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30665499" target="_blank">The claudins: from tight junctions to biological systems.</a> <em>Trends Biochem Sci</em> 44(2): 141-152.</li><li>Van Itallie, C. M. and Anderson, J. M. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15546850" target="_blank">The molecular physiology of tight junction pores.</a> <em>Physiology (Bethesda)</em> 19: 331-338.</li><li>Van Itallie, C. M. and Anderson, J. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25171873" target="_blank">Architecture of tight junctions and principles of molecular composition.</a> <em>Semin Cell Dev Biol</em> 36: 157-165.</li><li>Venugopal, S., Anwer, S. and Szaszi, K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31726679" target="_blank">Claudin-2: roles beyond permeability functions.</a> <em>Int J Mol Sci</em> 20(22). Doi.org/10.3390/ijms20225655.<br/></li></ol><p><br/></p>