<body>
            <h1>RNA Stability Measurements Using RT-qPCR in <em>Arabidopsis</em> Seedlings</h1>
            <p><strong>Authors:</strong> Tianran Jia, Brandon H. Le</p>
            <h2>Abstract</h2>
            <p>Steady-state mRNA levels are determined by both the rates of transcription and degradation. Regulation of mRNA stability and/or degradation are key factors that can significantly affect mRNA levels and its biological functions. mRNA stability can be measured indirectly after transcription inhibition. This protocol described a rapid and sensitive method of mRNA stability measurement through quantitative reverse transcription PCR (RT-qPCR) after inhibition of RNA transcription by cordycepin in <em>Arabidopsis</em> seedlings.</p>
            <p><strong>Keywords:</strong> RNA Stability, RNA Degradation, <em>Arabidopsis</em>, RT-qPCR, Cordycepin, Primary MicroRNA Transcripts</p>
            <h2>Background</h2><p>The regulation of mRNA stability is a key control point in gene expression regulation. mRNA stability has a profound impact on gene expression, molecular and cellular phenotype, and ultimately, plant development, defense and other biological processes. Multiple methods, <em>e.g.</em>, Northern blot analysis, <em>in situ</em> hybridization, can be used to measure mRNA stability after transcription inhibition. In this protocol, we describe a rapid and sensitive method to measure mRNA stability through RT-qPCR after inhibition of transcription by cordycepin. Cordycepin or 3'-deoxyadenosine is an adenosine analogue (see Reference 2). 3'-deoxyadenosine can be incorporated into RNA and inhibits transcription elongation and RNA synthesis due to the absence of a hydroxyl moiety at the 3' position (see Reference 6). We have successfully used this convenient and sensitive method to measure the stability of several low-abundant mRNAs, including primary microRNA transcripts, in <em>Arabidopsis</em> seedlings (Jia <em>et al.</em>, 2017). Here we present this protocol with detailed experimental procedures and data analysis methods.</p>
        </body>

<p class="pptt" id="biaoti34719">## Materials and Reagents</p><ol class="npwone"><li>Supplies<ol class="npwtwo"><li>Petri dishes, polystyrene, size 150 mm Ã 15 mm, sterile</li><li>6-well clear multiwell plates, sterile</li><li>Kimwipes<sup>TM</sup> </li><li>Low retention tips<br/><br/></li></ol></li><li>Materials<ol class="npwtwo"><li><em>Arabidopsis</em> seeds, including control and treatment or mutant groups<br/>It is strongly recommended to include a mutant with defects in mRNA stability as a positive control, <em>e.g.</em>, <em>ect2-1</em>(SALK_002225).<br/><br/></li></ol></li><li>Reagents<br/><ol class="npwtwo"><li>Liquid nitrogen<br/></li><li>RNaseZap<sup>TM</sup> (ThermoFisher, catalog number: <a class="blackNoUnder" target="_blank"> AM9780 </a> ) or ELIMINase<sup>Â®</sup> (DeconLabs, catalog number: <a class="blackNoUnder" target="_blank"> 1101 </a> )<br/></li><li>Murashige and Skoog Basal Medium (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M5519 </a> )<br/></li><li>Agar (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A1296 </a> )<br/></li><li>Household bleach (5.25% w/v sodium hypochlorite)<br/></li><li>Tween<sup>Â®</sup> 20 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P9416 </a> )<br/></li><li>Nuclease-free water<br/></li><li>Cordycepin (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C3394 </a> )<br/></li><li>RNA extraction kit (RNeasy Plant Mini Kit) (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 74904 </a> )<br/></li><li>cDNA synthesis kit (iScript cDNA Synthesis Kit) (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1708890 </a> )<br/></li><li>qPCR kit (SYBR<sup>TM</sup> Green PCR Master Mix) (ThermoFisher, catalog number: <a class="blackNoUnder" target="_blank"> 4309155 </a> )<br/></li><li>qPCR primers (Design qPCR primers on Primer3Plus (Untergasser <em>et al.</em>, 2012), and order qPCR primers from Integrated DNA Technology)</li></ol><em>Note: Design qPCR primers closer to the 3â² end of the mRNAs to be detected. QIAGEN RNeasy Plant Mini Kit, Bio-Rad iScript cDNA Synthesis Kit and ThermoFisher SYBR</em><sup><em>TM</em></sup><em> Green PCR Master Mix were tested and worked well, and therefore listed here only as recommended kits. However, other suitable RNA extraction and RT-qPCR kits could also be used for this protocol.</em><br/></li></ol><p class="pptt" id="biaoti34720">### Equipment</p><ol class="npwtwok"><li>Multichannel pipettes</li><li>Mortar and pestle</li><li>Autoclave</li><li>-20 Â°C freezer</li><li><em>Arabidopsis</em> growth room or chamber</li><li>VWR<sup>Â®</sup> Standard Orbital Shaker, Model 3500 (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89032-092 </a> )</li><li>Eppendorf Benchtop Centrifuge 5418, non-refrigerated, with Rotor FA-45-18-11 and rotor lid, 230 V/50-60 Hz (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5418000017 </a> )</li><li>Applied Biosystems<sup>TM</sup> QuantStudio<sup>TM</sup> 5 Real-Time PCR System (ThermoFisher, catalog number: <a class="blackNoUnder" target="_blank"> A34322 </a> ), or Bio-Rad CFX96 Touch Real-Time PCR Detection System (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1855196 </a> )<br/></li></ol><p class="pptt" id="biaoti34721">### Software</p><ol class="npwtwok"><li>Primer3Plus (A. Untergasser, <a href="https://primer3plus.com/cgi-bin/dev/primer3plus.cgi" target="_blank">https://primer3plus.com/cgi-bin/dev/primer3plus.cgi</a>)</li><li>Excel (Microsoft)</li><li>Applied Biosystems<sup>TM</sup> Analysis Software or Thermo Fisher Cloud (Thermo Fisher Scientific, available <a href="https://www.thermofisher.com/account-center/cloud-login.html?samlContext=ad31bbdb-537c-4589-857f-44c6fb776bd0" target="_blank">here</a>)</li></ol><p class="pptt" id="biaoti34722">### Procedure</p><ol class="npwone"><li>Prepare reagents and buffers<br/><em>Note: Calculate the required amount of MS medium and cordycepin before starting reagent preparation based on experimental settings. Approximately 50 ml MS medium is required to prepare one 150 mm plate. Each well of the six-well plate requires around 3 ml MS medium. At least two replicates should be included for each sample at each time point. Depending on the size of the seedlings, approximately 10 to 20 seedlings could fit into one well of the six-well plate.</em><br/><br/><ol class="npwtwo"><li>Prepare 1Ã MS agar medium plates using 150 mm Petri dishes as described in the ABRC <em>Arabidopsis</em> protocols (Rivero <em>et al.</em>, 2014). Prepared plates can be wrapped up and stored at 4 Â°C for at least a month.</li><li>Prepare 0.5Ã MS liquid growth medium by making a two-fold dilution of 1Ã MS medium. Do not add agar. Autoclaved medium can be stored at 4 Â°C for at least a month.</li><li>Prepare seeds surface-sterilization solution: Use 50% household bleach with distilled water to make solutions with final concentration of 2.6 % w/v sodium hypochlorite, add Tween<sup>Â®</sup> 20 to a final concentration of 0.05%.</li><li>Dissolve cordycepin in nuclease-free water to make 0.6 M stock concentration. Store 0.6 M cordycepin stock solution at -20 Â°C.</li><li>Dilute qPCR primers to 100 Î¼M in nuclease-free water. Store at -20 Â°C.<br/><br/></li></ol></li><li>Prepare plant materials<br/><ol class="npwtwo"><li>Sterilize seeds in microcentrifuge tubes by soaking them in the surface-sterilization solution for 5 to 10 min. Remove all bleach by rinsing five to seven times with sterile distilled water (Rivero <em>et al.</em>, 2014).<br/></li><li>The <em>Arabidopsis</em> seeds (control and mutant groups, as shown in Figure 1) are sown onto the 1Ã MS agar plates at a density of about one seed per 1 cm<sup>2</sup>, facilitating manipulation during subsequent seedling transfer. After cold stratification, the plates are placed vertically in the growth chamber, thus preventing growth of the root into the solid media. Grow <em>Arabidopsis</em> in sterile condition for approximately 12 days following the ABRC <em>Arabidopsis</em> protocols (Rivero <em>et al.</em>, 2014).<br/><br/><img alt="" class="layerphoto" height="258" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713071501_8216.jpg" width="300"/><br/><strong>Figure 1. Work flow illustration of RNA stability measurements using RT-qPCR in <em>Arabidopsis</em>.</strong> Twelve-day-old <em>Arabidopsis</em> wild type and mutant whole seedlings were transferred from MS medium agar plates to six-well plates with 0.5Ã MS liquid medium and incubated overnight (Note: Only roots will be submerged in liquid MS). The next day, cordycepin was added to a final concentration of 0.6 mM and the seedlings were collected for RNA extraction at indicated time points after cordycepin treatment. RT-qPCR was then performed to determine RNA levels, and statistical analysis was used to determine the significant difference between wild type and mutant RNA stability.<br/><br/></li></ol><em>Note: Prolonged sterilization might kill some </em><em>Arabidopsis</em><em> mutants. Seedlings of </em><em>Arabidopsis</em><em> can also be grown in liquid medium following ABRC protocols (Rivero </em><em>et al.</em><em>, 2014). If grown in six-well plate in liquid medium, skip Day 1 procedures and add cordycepin to six-well plate directly as described in Day 2 Step 1.</em><br/><br/></li><li>Experimental procedures<br/><strong>Day 1</strong><br/><ol class="npwtwo"><li>Use sterilized forceps to transfer <em>Arabidopsis</em> whole seedlings to six-well plates with 3 ml 0.5Ã MS medium per well in tissue culture hood. Be careful not to damage the seedling roots during the transfer.<br/></li><li>Incubate the plates on a shaker at 120 rpm in an <em>Arabidopsis</em> growth room or chamber to equilibrate the seedlings overnight.<br/><br/></li></ol><strong>Day 2</strong><br/><em>Note: Read the RNA extraction kit, cDNA synthesis kit and qPCR kit complete protocols before starting the experiment if using those kits for the first time. Use RNaseZap</em><sup><em>TM</em></sup><em> (Invitrogen) or ELIMINase</em><sup><em>Â®</em></sup><em> (DeconLabs) to wipe and decontaminate all pipettes and workstations.</em> <ol class="npwtwo"><li>Add cordycepin (Sigma-Aldrich) to a final concentration of 0.6 mM (1 to 1,000 dilution of 0.6 M stock), and treat for required lengths of time (<em>e.g.</em>, 0 min, 30 min, 90 min, 150 min, 210 min, 270 min, as shown in Figure 1).<br/><em>Note: The longest treatment (</em><em>e.g.</em><em>, 270 min in this experiment) can be performed first, and then add cordycepin based on required lengths of treatment time, and finally collect all samples at the same time (Figures 1 and 2).</em><br/><br/><img alt="" class="layerphoto" height="114" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713071522_3073.jpg" width="300"/><br/><strong>Figure 2. Schematic representation of cordycepin treatment time course analysis</strong><br/><br/></li><li>Collect seedlings and blot on Kimwipes<sup>TM</sup> to remove excessive liquid.</li><li>Immediately, freeze the seedlings in liquid nitrogen and grind to a fine powder. Transfer to a cooled, appropriately sized, tube and allow liquid nitrogen to evaporate without allowing the sample to thaw, and then add lysis buffer. Repeat for the remaining seedling samples. Proceed to the next step of RNA extraction as soon as possible following the RNA extraction kit protocol (see Reference 8).</li><li>Perform RNA quality control: Use a Nanodrop or Qubit to determine the RNA sample purity and concentration.</li><li>Perform cDNA synthesis following Bio-Rad iScript<sup>TM</sup> cDNA Synthesis Kit protocol (see Reference 3), and qPCR following SYBR<sup>Â®</sup> Green PCR Master Mix protocol (see Reference 10).<br/><em>Note: Negative controls for reverse transcription are required to make sure all DNA are eliminated during RNA extraction. To set up negative controls, replace reverse transcriptase with nuclease-free water during the cDNA synthesis reactions. Include the negative control samples in the following qPCR analysis. qPCR reaction conditions will need to be optimized for different primers, targets or cDNA samples. The amount of the cDNA reaction that is recommended for downstream qPCR is one-tenth of the reaction volume, typically 1 Î¼l for 10 Î¼l reaction. Try to keep Ct values under 35, which indicates strong positive signals. Include at least two housekeeping controls for qPCR analysis. Because qPCR assay is highly sensitive to pipetting errors and variabilities, use multichannel pipettes and low-retention tips to setup qPCR reactions.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti34723">### Data analysis</p><ol class="npwtwok"><li>Quality control: Make sure the amplification curve is typical, melting curve shows a single peak indicating a single amplification product from each pair of primers (if melting curve analysis was not performed, run an agarose gel to check the amplification product for single amplification), coefficient of variation in replicate group is equal to or less than 10%, and no signal was detected in the negative control groups (Figure 3). If using QuantStudio<sup>TM</sup>, review the quality flags using Applied Biosystems<sup>TM</sup> Analysis Software or Thermo Fisher Cloud (see Reference 1). Remove poor quality or flagged values.<br/><br/><img alt="" class="layerphoto" height="436" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713071541_6282.jpg" width="300"/><br/><strong>Figure 3. Typical amplification curves and melting curves for an endogenous control and a pri-miRNA gene.</strong> Normal amplification curves are smooth during the exponential phase of the PCR. The threshold cycle (Ct) value is the cycle at which the fluorescent signal of the reaction crosses the threshold, which is usually automatically set by the qPCR software at where signal significantly increases over the background. A good melting curve shows a single peak indicating a single amplification product from each pair of primers. The clustering of the curves should be tight for replicates.<br/><br/></li><li>Use comparative Ct (â³â³C<sub>T</sub>) method to determine relative quantity of each targets in test samples via either software analysis or manual calculation in excel (see Reference 1, Schmittgen and Livak, 2008). The targets are normalized to the endogenous control (<em>e.g.</em>, <em>GAPDH</em>, <em>ACTIN</em>, <em>UBQ</em> <em>etc.</em>), and the control or reference sample. The relative quantification or fold change of each target is plotted against time points (Figure 4, Schmittgen and Livak, 2008).</li><li>Statistical analysis: Studentâs <em>t</em>-test was performed to check the difference between control and mutant groups at each time point. The cutoff is set as <em>P</em> value &lt; 0.05 and fold change more than 2.<br/><br/><img alt="" class="layerphoto" height="142" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713071600_3014.jpg" width="300"/><br/><strong>Figure 4. Representative results of RNA stability measurements for pri-miR172a, and <em>EIF4A</em> mRNA.</strong> <em>UBQ5</em> served as an internal control. Samples collected at time point 0 served as the sample control or reference here. Values at time 0 were set to 1. Error bars indicate standard deviation from two sample replicates. For each sample replicate, technical triplicates were performed for individual gene target.<br/></li></ol><p class="pptt" id="biaoti34724">### Acknowledgments</p><p class="ppzz">We thank Dr. Xuemei Chen and UC Riverside for supporting this work. This work was funded by the National Institutes of Health (GM061146) and Gordon and Betty Moore Foundation (GBMF3046) to Dr. Xuemei Chen. This protocol was adapted from previous works by Lidder <em>et al.</em> (2005), Zhang <em>et al.</em> (2014), and Jia <em>et al.</em> (2017).</p><p class="pptt" id="biaoti34725">### Competing interests</p><p class="ppzz">The authors declare that there is no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti34726">### References</p><ol class="npwtwok"><li>Applied Biosystems<sup>TM</sup> Analysis Software: Applied Biosystems<sup>TM</sup> Relative Quantitation Analysis Module USER GUIDE. Thermo Fisher Scientific. Retrieved from: <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0014820_RelativeQuant_UG.pdf" target="_blank">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/MAN0014820_RelativeQuant_UG.pdf</a>.</li><li>Cordycepin: Product Information. Sigma-Aldrich. Retrieved from: <a href="https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Product_Information_Sheet/c3394pis.pdf." target="_blank">https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Product_Information_Sheet/c3394pis.pdf</a>.</li><li>iScript<sup>TM</sup> cDNA Synthesis Kit: iScript<sup>TM</sup> cDNA Synthesis Kit Product Insert, Ver D. Bio-Rad. Retrieved from: <a href="https://www.bio-rad.com/webroot/web/pdf/lsr/literature/4106228.pdf" target="_blank">https://www.bio-rad.com/webroot/web/pdf/lsr/literature/4106228.pdf</a>.</li><li>Jia, T., Zhang, B., You, C., Zhang, Y., Zeng, L., Li, S., Johnson, K. C. M., Yu, B., Li, X. and Chen, X. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28947490" target="_blank">The <em>Arabidopsis</em> MOS4-associated complex promotes microRNA biogenesis and precursor messenger RNA splicing.</a> <em>Plant Cell</em> 29: 2626-2643.</li><li>Lidder, P., GutiÃ©rrez, R. A., SalomÃ©, P. A., McClung, C. R. and Green, P. J. (2005). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16055688" target="_blank">Circadian control of messenger RNA stability. Association with a sequence-specific messenger rna decay pathway.</a> <em>Plant Physiol</em> 138 (4): 2374-2385.</li><li>NCI Drug Dictionary. Retrieved from: <a href="https://www.cancer.gov/publications/dictionaries/cancer-drug/def/cordycepin." target="_blank">https://www.cancer.gov/publications/dictionaries/cancer-drug/def/cordycepin</a>.</li><li>Rivero, L., Scholl, R., Holomuzki, N., Crist, D., Grotewold, E. and Brkljacic, J. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24057358" target="_blank">Handling <em>Arabidopsis</em> plants: growth, preservation of seeds, transformation, and genetic crosses.</a> <em>Methods Mol Biol</em> 1062: 3-25.</li><li>RNeasy Plant Mini Kit: RNeasy Mini Handbook - (EN). QIAGEN. Retrieved from: <a href="http://www.bea.ki.se/documents/EN-RNeasy handbook.pdf" target="_blank">http://www.bea.ki.se/documents/EN-RNeasy%20handbook.pdf</a>.</li><li>Schmittgen, T. D. and Livak, K. J. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18546601" target="_blank">Analyzing real-time PCR data by the comparative C(T) method.</a> <em>Nat Protoc</em> 3(6): <a class="blackNoUnder" target="_blank"> 1101 </a> -1108.</li><li>SYBR<sup>Â®</sup> Green PCR Master Mix: User Guide SYBR<sup>Â®</sup> Green PCR Master Mix and SYBR Green RT-PCR<sup>Â®</sup> Reagents Kit. Life Biotechnologies. Retrieved from: <a href="https://assets.thermofisher.com/TFS-Assets/LSG/manuals/cms_041053.pdf" target="_blank">https://assets.thermofisher.com/TFS-Assets/LSG/manuals/cms_041053.pdf</a>.</li><li>Untergasser, A., Cutcutache, I., Koressaar, T., Ye, J., Faircloth, B. C., Remm, M. and Rozen, S. G. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22730293" target="_blank">Primer3--new capabilities and interfaces.</a> <em>Nucleic Acids Res</em> 40(15): e115.</li><li>Zhang, S., Liu, Y. and Yu, B. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25474114" target="_blank">PRL1, an RNA-Binding Protein, Positively Regulates the Accumulation of MiRNAs and SiRNAs in <em>Arabidopsis</em>.</a> <em>PLoS Genet</em> 10 (12): e1004841.<br/></li></ol>