<body>
            <h1>Gene Expression and Genome Editing Systems by Direct Delivery of Macromolecules Into Rice Egg Cells and Zygotes</h1>
            <p><strong>Authors:</strong> Erika Toda, Takashi  Okamoto</p>
            <h2>Abstract</h2>
            <p>Polyethylene glycol calcium (PEG-Ca<sup>2+</sup>)-mediated transfection allows rapid and efficient examination to analyze diverse cellular functions of genes of interest. In plant cells, macromolecules, such as DNA, RNA and protein, are delivered into protoplasts derived from somatic tissues or calli via PEG-Ca<sup>2+</sup> transfection. To broaden and develop the scope of investigations using plant gametes and zygotes, a procedure for direct delivery of macromolecules into these cells has recently been established using PEG-Ca<sup>2+</sup> transfection. This PEG-Ca<sup>2+</sup>-mediated delivery into rice egg cells/zygotes consists of four microtechniques, (i) isolation of gametes, (ii) production of zygotes by electrofusion of gametes, (iii) PEG-Ca<sup>2+</sup>-mediated delivery of macromolecules into isolated egg cells or produced zygotes, and (iv) culture and subsequent analyses of the transfected egg cells/zygotes. Because the full protocol for microtechniques (i) and (ii) have already been reported in Toda <em>et al.</em>, 2016, microtechniques (iii) and (iv) are mainly described in this protocol.</p>
            <p><strong>Keywords:</strong> CRISPR/Cas9, Electro-fusion, Gamete, Genome editing, <em>In vitro</em> fertilization, PEG-Ca<sup>2+</sup> transfection, Rice, Zygote</p>
            <h2>Background</h2><p>In angiosperms, fertilization and subsequent developmental events, such as embryogenesis and endosperm development, occur in the embryo sac deeply embedded in ovular tissue (Nawaschin, 1898; Guignard, 1899; Russell, 1992; Raghavan, 2003). Therefore, studies on post-fertilization events in the embryo sac have been conducted predominantly through analyses of <em>Arabidopsis</em> mutants or transformants coupled with live-imaging. Alternatively, direct analyses using isolated gametes or zygotes are possible, and an <em>in vitro</em> fertilization (IVF) system using the isolated gametes can be used to observe and analyze post-fertilization processes directly (reviewed in Wang <em>et al.</em>, 2006). Isolated gametes can be fused electrically, following which the produced zygotes then divide and develop into embryo-like structures and regenerate into plantlets in maize, rice and wheat (Kranz and LÃ¶rz, 1993; Uchiumi <em>et al.</em>, 2007; Maryenti <em>et al.</em>, 2019). These cell manipuration techniques, such as isolation of gametes or zygotes and production of <em>in vitro</em> zygotes, make it possible to identify genes expressing in gametes, zygotes and embryos, and fertilization-induced/suppressed genes in zygotes and early embryos have been investigated (Sprunck <em>et al.</em>, 2005; Ning <em>et al.</em>, 2006; Yang <em>et al.</em>, 2006; Steffen <em>et al.</em>, 2007; Wang <em>et al.</em>, 2010; Wuest <em>et al.</em>, 2010; Ohnishi <em>et al.</em>, 2011; Abiko <em>et al.</em>, 2013; Anderson <em>et al.</em>, 2013; Khanday <em>et al.</em>, 2019; Rahman <em>et al.</em>, 2019; Zhao <em>et al.</em>, 2019). However, the functions of most of the genes expressing in a gamete-specific or fertilization-induced/suppressed manner have not been analyzed. Therefore, to broaden and develop the scope of investigations using gametes and zygotes, a procedure for direct delivery of plasmid DNAs into these cells has been established by PEG-Ca<sup>2+</sup>-mediated transfection. By using this procedure, approximately 30% and 70% respectively of PEG-Ca<sup>2+</sup>-transfected rice egg cells and zygotes showed exogenous and transient expressions of fluorescent proteins from plasmid DNA delivered into the cells (Koiso <em>et al.</em>, 2017; Toda <em>et al.</em>, 2019). Recently, it was shown that PEG-Ca<sup>2+</sup>-mediated ectopic expression of <em>OsASGR-BBML1</em>, an AP-type transcription factor, in rice egg cells resulted in induction of nuclear/cell division of the cells (Rahman <em>et al.</em>, 2019). In addition, a genome editing system by direct delivery of CRISPR-Cas9 components into rice zygotes has been developed through PEG-Ca<sup>2+</sup> transfection. Plasmid DNA harboring a Cas9-gRNA expression cassette or Cas9-gRNA ribonucleoproteins (RNPs) was transfected into zygotes, resulting in the regeneration of plants with targeted mutations in the range of 4%-64% (Toda <em>et al.</em>, 2019). In addition to rice, complete IVF systems have also been established in maize and wheat (Kranz and LÃ¶rz, 1993; Uchiumi <em>et al.</em>, 2007; Maryenti <em>et al.</em>, 2019), with all three being the major crop species that provide the majority of human energy intake globally. Taken together, a combination of an IVF system and the present gene expression system described here has enormous potential for efficiently investigating post-fertilization events as well as helping to advance molecular breeding to improve these important crop species.</p>
        </body>

<p class="pptt" id="biaoti34729">## Materials and Reagents</p><ol class="npwtwok"><li>Coverslips (24 x 40 mm) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 125485J </a> ) (siliconized at the edges with 3% dichlorodimethylsilane in diethyl ether, Figure 1C)<br/><em>Note: Non-coated coverslips should be used, as using coated coverslips will lead to the cells becoming attached to the surface of the coverslip.</em> </li><li>Non-treated plastic dishes with a diameter of 3.5 cm (Iwaki, catalog number: <a class="blackNoUnder" target="_blank"> 1000-035 </a> )</li><li>Glass capillary</li><li>Rice egg cells, sperm cells and zygotes (Figures 1D, 1E and 1G)<br/><em>Note: Rice plants (Oryza sativa L. cv. Nipponbare or cv. Yukihikari) are grown in an environmental chamber at 26 Â°C in a 13/11 h light/dark cycle with a photosynthetic photon flux density of 150-300 Î¼mol photons/m<sup>2</sup>/s. Under these growth conditions, panicles can be obtained throughout all seasons. Rice egg cells and sperm cells were isolated according to the full protocol described by Toda et al. (2016). Rice zygotes are also prepared using electrofusion as described in the same protocol.</em><br/><br/><img alt="" class="layerphoto" height="162" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713082057_8190.jpg" width="300"/><br/><strong>Figure 1. Equipment and materials for cell manipulation (A-C) and preparation of isolated gametes and an <em>in vitro</em> zygote (D-G).</strong> A. Sliding stage for insertion of a coverslip and a plastic dish. B. Glass capillary-connected handling injector. White arrowhead indicates a glass capillary. C. Image of droplets inside the mineral oil on the siliconized coverslip. D. Isolated rice egg cell. E. Isolated rice sperm cells (black arrowheads). F. Image of electrofusion of an egg cell and a sperm cell using electrodes connected to an electrofusion apparatus. G. Rice zygote produced by electrofusion. Scale bars = 5 cm in A, 20 Âµm in D and G, and 15 Âµm in E.<br/><br/></li><li>Feeder cells: rice suspension cell culture (Line Oc, provided by RIKEN Bio-Resource Center, Tsukuba, Japan)<br/><em>Note: Rice suspension cells, Line Oc, were subcultured once a week according to instructions from RIKEN Bio-Resource Center. No difference in feeder effects between freshly subcultured cells and 1-week-cultured cells has been observed.</em> </li><li>Mannitol (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 133-00845 </a> )</li><li>370 mosmol/kg H<sub>2</sub>O (330 mM) mannitol solution (autoclaved)</li><li>450 mosmol/kg H<sub>2</sub>O (385 mM) mannitol solution (autoclaved)</li><li>Mineral oil (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M5310-100ML </a> )</li><li>Absolute ethanol (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 057-00451 </a> )</li><li>Dichlorodimethylsilane (Tokyo Chemical Industry, catalog number: <a class="blackNoUnder" target="_blank"> D0358 </a> )</li><li>Diethyl ether (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 055-01155 </a> )</li><li>MgCl<sub>2</sub>Â·6H<sub>2</sub>O (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 135-00165 </a> )</li><li>CaCl<sub>2</sub>Â·2H<sub>2</sub>O (FUJIFILM Wako Pure Chemical, catalog number: <a class="blackNoUnder" target="_blank"> 038-19735 </a> )</li><li>2-Morpholinoethanesulfonic acid, monohydrate (MES, C<sub>6</sub>H<sub>13</sub>NO<sub>4</sub>SÂ·H<sub>2</sub>O, Dojindo, catalog number: <a class="blackNoUnder" target="_blank"> 349-01623 </a> )</li><li>Polyethylene glycol 4000 (Sigma-Ardrich, catalog number: <a class="blackNoUnder" target="_blank"> 81240-1KG </a> )</li><li>Single guide RNA (synthesized using the GeneArt Precision gRNA Synthesis Kit) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A29377 </a> ), storage -80 Â°C</li><li>Cas9 protein (Thermo Fisher Scientific, GeneArt Platinum Cas9 Nuclease; Thermo Fisher Scientific, TrueCut Cas9 Protein v2, catalog number: <a class="blackNoUnder" target="_blank"> A36496 </a> ), storage -20 Â°C</li><li>The tRNA-based multiplex CRISPR-Cas9 vector (pMgPoef4_129-2A-GFP; accession number LC460477)</li><li>Zygote culture medium, storage 4 Â°C</li><li>Regeneration media, storage 4 Â°C</li><li>Rooting media, storage 4 Â°C<br/><em>Note: Recipes for zygote culture medium, regeneration media and rooting media are presented in Toda et al. (2016).</em> </li><li>PrepMan Ultra (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 4318930 </a> )</li><li>PrimeSTAR GXL DNA Polymerase (Takara, catalog number: <a class="blackNoUnder" target="_blank"> R050A </a> )</li><li>ExoSAP-IT Express PCR Cleanup Reagents (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 75001.200.UL </a> )</li><li>Zero Blunt PCR Cloning Kit (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> K270020 </a> )</li><li>MMG solution (see Recipes)</li><li>PEG-Ca<sup>2+</sup> solution (see Recipes)<br/><em>Note: MMG solution and PEG-Ca<sup>2+</sup> solution should be freshly prepared, as using old solutions results in decrease of PEG-Ca<sup>2+</sup> transfection efficiency.</em><br/></li></ol><p class="pptt" id="biaoti34730">### Equipment</p><ol class="npwtwok"><li>Forcep</li><li>Inverted microscope (OLYMPUS, model: <a class="blackNoUnder" target="_blank"> IX-71 or IX-73 </a> )</li><li>Sliding stage for the insertion of a coverslip and a plastic dish (Figure 1A)</li><li>Manual handling injector (PrimeTech, HDJ-M3, Figure 1B)</li><li>Glass capillaries made from 50 Î¼l aspirator tubes (Drummond Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 2-000-050 </a> ) (Figure 1B)<br/><em>Note: The procedure for making glass capillaries is presented in Toda et al. (2016).</em> </li><li>Electrofusion apparatus (Nepa Gene, model: <a class="blackNoUnder" target="_blank"> ECFG21 </a> )</li><li>Manipulator (NARISHIGE, model: <a class="blackNoUnder" target="_blank"> M-152 </a> ) with a double pipette holder (Narishige, model: <a class="blackNoUnder" target="_blank"> HD-21 </a> )</li><li>Electrodes (Nepa Gene, model: <a class="blackNoUnder" target="_blank"> CUY5100Ti100 </a> ) fixed to the pipette holder (Figure 1F)</li><li>Millicell-CM inserts, diameter 12 mm (EMD Millipore, catalog number: <a class="blackNoUnder" target="_blank"> PICM01250 </a> )</li><li>Environmental chambers</li><li>Shaker for zygote culture (EYELA, model: <a class="blackNoUnder" target="_blank"> Multi Shaker MMS-3020 </a> )</li><li>Osmometer (Gonotec, model: <a class="blackNoUnder" target="_blank"> Osmomat 3000 basic </a> )<br/></li></ol><p class="pptt" id="biaoti34731">### Software</p><ol class="npwtwok"><li>Focas webtool (Focas UI, <a href="http://focas.ayanel.co/" target="_blank">http://focas.ayanel.co/</a>)</li><li>RNA Folding Form software (mfold web server, <a href="http://unafold.rna.albany.edu/?q=mfold/RNA-Folding-Form2.3" target="_blank">http://unafold.rna.albany.edu/?q=mfold/RNA-Folding-Form2.3</a>)</li><li>Parallel Editor in GENETYX software (GENETYX)</li><li>SnapGene Viewer (SnapGene)<br/></li></ol><p class="pptt" id="biaoti34732">### Procedure</p><ol class="npwone"><li>PEG-Ca<sup>2+</sup> transfection of rice egg cells with plasmid DNA<ol class="npwtwo"><li>An outline of procedures for PEG-Ca<sup>2+</sup> transfection of rice egg cells with plasmid DNA is presented in Figure 2A.</li><li>As shown in Figures 2C and 2D, prepare 0.5-1 Âµl droplets of mannitol solution, MMG solution, MMG solution containing plasmid DNA (MMG-DNA, 80-320 ng/Âµl) and PEG-Ca<sup>2+</sup> solution (Figures 2C and 2D).<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Our study suggested that a MMG-DNA solution containing a higher plasmid concentration (ca. 270-320 ng/Âµl) generally results in efficient transient expression (Koiso et al., 2017).</em><br/></li><li><em>For preparation of the MMG solution and PEG-Ca<sup>2+</sup> solution, use mannitol solution adjusted to 370 mosmol/kg H<sub>2</sub>O (330 mM) and 450 mosmol/kg H<sub>2</sub>O (385 mM) by osmometer for rice egg cells and zygotes, respectively.</em><br/></li><li><em>If the concentration of prepared plasmid DNA sample is low, and the fluid volume of plasmid DNA exceeds that of the MMG solution when you prepare the MMG-DNA solution (80-320 ng/Âµl), using 2x the MMG solution (double concentration) is recommended.</em><br/><br/><img alt="" class="layerphoto" height="277" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713082147_9652.jpg" width="300"/><br/><strong>Figure 2. Procedures for PEG-Ca<sup>2+</sup> transfection of rice egg cells (A) and zygotes (B), preparation of droplets of chemical solutions for PEG-Ca<sup>2+</sup> transfection (C and D), and rice egg cells upon or after PEG-Ca<sup>2+</sup> treatment (E).</strong> A. Illustration of the procedure for PEG-Ca<sup>2+</sup> transfection of rice egg cells with plasmid DNA. The transfected egg cells are cultured in a droplet or millicell insert filled with zygote culture medium to observe and analyze the effects of transient gene expression on the cells. B. Illustration of the procedure for PEG-Ca<sup>2+</sup> transfection of rice zygotes with plasmid DNA. The transfected zygotes are cultured in a millicell insert filled with zygote culture medium to observe and analyze the effects of transient gene expression on zygotic development. C and D. Photo (C) and image (D) of each droplet inside the mineral oil on the siliconized coverslip for PEG-Ca<sup>2+</sup> transfection. Mannitol, Mannitol solution; MMG, MMG solution; MMG + Macromolecules, MMG solution containing plasmid DNA or ribonucleoprotein; PEG, PEG-Ca<sup>2+</sup> solution. E. Rice egg cell in the merged droplet of MMG-DNA and PEG-Ca<sup>2+</sup> solution (left panel) or in mannitol solution (right panel). Egg cell shrank upon PEG-Ca<sup>2+</sup> treatment, and the size of the egg cell recovered in mannitol solution after the treatment. In A and B, the pink, green and orange circles indicate the egg, sperm and zygotic nuclei, respectively. The yellow lightning bolt and gray rings in A and B indicate the point of electrofusion and plasmid DNAs, respectively. In D, the arrowheads indicate the procedures for transferring cells into each droplet during PEG-Ca<sup>2+</sup> transfection. The curved arrows above the on MMG + Macromolecules and PEG droplets indicate the process of merging and mixing the two droplets in Step A6. Scale bars = 1 cm in C and 20 Âµm in E.</li></ol><br/></li><li>Transfer approximately three to six egg cells isolated from rice flowers into a 0.5-1 Âµl droplet of mannitol solution (370 mosmol/kg H<sub>2</sub>O) overlaid with mineral oil on a coverslip using a glass capillary (Figure 2D).</li><li>Wash the cells twice by transferring them into a MMG solution droplet on a coverslip.<br/><em>Note: Because the cells tend to attach to the surface of the coverslip in the MMG solution droplet, the manipulation of sucking and releasing cells in the MMG solution droplet should be performed with the tip of the glass capillary positioned slightly above the bottom of the coverslip.</em> </li><li>Transfer the cells into a droplet of MMG-DNA (Video 1).<br/><br/><!--video3681-1å¼å§--><div class="videodiv" id="vd3681-1" style="height:480px;"></div><div class="videophoto" id="vp3681-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3681/3681-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Rinse rice egg cells with MMG solution by transferring the cells into a MMG droplet, and subsequently transferring the cells into a droplet of MMG-DNA</strong> </div><!--video3681-1ç»æ--><br/></li><li>Merge the MMG-DNA droplet, where the cells are transferred, with a droplet of PEG-Ca<sup>2+</sup> solution using a glass capillary (Figure 2D; Video 2). Then immediately and gently mix the merged droplet using a glass capillary for approximately 1 min.<br/><em>Note: Although the cells often float in the mixed solution when the MMG-DNA droplet is merged with PEG-Ca<sup>2+</sup> solution, these cells gradually sink in the solution during incubation.</em><br/><br/><!--video3681-2å¼å§--><div class="videodiv" id="vd3681-2" style="height:480px;"></div><div class="videophoto" id="vp3681-2"><a class="newapic" href="javascript:;" onclick="openvideo(2)" style="background:url('https://en-cdn.bio-protocol.org/video/3681/3681-2s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 2. Mixing of the MMG-DNA droplet, where rice egg cells are held, with a droplet of PEG-Ca2+ solution using a glass capillary</strong> </div><!--video3681-2ç»æ--><br/></li><li>Incubate for 5 min at room temperature. Herein, cell shrinkage can be observed (Figure 2E).</li><li>After incubation, replace the mixed solution containing the MMG-DNA and PEG-Ca<sup>2+</sup> solution with the mannitol solution by adding the mannitol solution to the merged droplet and gently mix the droplet using a glass capillary.</li><li>Suck out approximately half the volume of the mixed solution using a glass capillary (Video 3).<br/><br/><!--video3681-3å¼å§--><div class="videodiv" id="vd3681-3" style="height:480px;"></div><div class="videophoto" id="vp3681-3"><a class="newapic" href="javascript:;" onclick="openvideo(3)" style="background:url('https://en-cdn.bio-protocol.org/video/3681/3681-3s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 3. Replacement of the mixed MMG-DNA and PEG-Ca<sup>2+</sup> solution with mannitol solution.</strong> Mannitol solution is added to a droplet of the MMG-DNA and PEG-Ca<sup>2+</sup>-mixed solution, and then the droplet is gently mixed using a glass capillary. Thereafter, approximately half of the volume of the solution is removed from the droplet.</div><!--video3681-3ç»æ--><br/></li><li>Repeat Steps A9 and A10 five to six times to reduce the concentration of MMG and PEG-Ca<sup>2+</sup> in the droplet where PEG-Ca<sup>2+</sup> treated cells exist. During this process, the shrunken cell recovers its initial round shape (Figure 2E).</li><li>Transfer the treated cells to fresh mannitol solution (Figure 2D).</li><li>Wash the cells three to four times by transferring them into droplets of fresh mannitol solution (450 mosmol/kg H<sub>2</sub>O) using a glass capillary.</li><li>Transfer the cells into a droplet of zygote culture medium with several aggregates of the feeder cells (Figure 3A, Nakajima <em>et al.</em>, 2010) or into zygote culture medium in a Millicell insert with a drop (approximately 50 Âµl) of several aggregates of feeder cells (Figure 3B).<br/><em>Note: Culture of the treated egg cells in the droplet is well suited for short-term observation and analysis (approximately 1-2 days). For long-term culture (more than 2-3 days), a millicell culture is recommended (see Procedure B).</em> </li><li>Culture the egg cells at 26 Â°C in the dark without shaking for an appropriate period and conduct analyses of the cells.<br/><em>Note: In our study, the transfected egg cells cultured in droplets expressed GFP and DsRed2 fluorescent signals, which are derived from plasmid DNAs transfected into the cells, at approximately 1 day after transfection. Further, when we transfected the rice egg cells with plasmid DNA harboring OsASGR-BBML1, the treated cells cultured in the millicell initiated nuclear/cell division around 2 days after transfection (Koiso et al., 2017, Rahman et al., 2019).</em><br/><br/></li></ol></li><li>PEG-Ca<sup>2+</sup> transfection of rice zygotes with plasmid DNA<ol class="npwtwo"><li>An outline of procedures for PEG-Ca<sup>2+</sup> transfection of rice zygotes with plasmid DNA is presented in Figure 2B.</li><li>For PEG-Ca<sup>2+</sup> transfection, use IVF-produced rice zygotes within 1 h after electrofusion.<br/><em>Note: As cell wall formation around the plasma membrane of zygotes generally starts after gamete fusion, the early zygotes in which cell walls are thought to be immature should be used for PEG-Ca<sup>2+</sup> transfection.</em> </li><li>The PEG-Ca<sup>2+</sup> transfection of rice zygotes is performed as Steps A2-A11, except that the mannitol solution is adjusted to 450 mosmol/kg H<sub>2</sub>O instead of 370 mosmol/kg H<sub>2</sub>O.</li><li>For culture of PEG-Ca<sup>2+</sup> transfected zygotes in a Millicell insert, place 0.2 ml zygote culture medium in a Millicell-CM insert and put it into a 3.5 cm plastic dish containing 2 ml of the medium. Add 40-60 Âµl of a rice suspension cell culture into the dish as feeder cells (Figure 3B).</li><li>Wash the transfected zygotes three to four times by transferring them into droplets of fresh mannitol solution (450 mosmol/kg H<sub>2</sub>O) using a glass capillary.</li><li>Transfer the transfected zygotes onto the membranes of a Millicell insert preprared in Step B4 (Figure 3B).<br/><br/></li></ol></li><li>Culture of PEG-Ca<sup>2+</sup> transfected zygotes<ol class="npwtwo"><li>After overnight culture of zygotes at 26 Â°C in the dark without shaking, continue culture with gentle shaking (40 rpm) (Figures 3C-a and 3C-b).</li><li>At 5 to 7 days after fusion, remove feeder cells by transferring the Millicell dishes containing the embryos into new 35-mm-diameter dishes filled with 2 ml of fresh zygote culture medium and continue culturing as above (Figure 3C-c).</li><li>After approximately 18-20 days in culture, subculture cell colonies developed from the transfected zygotes in Millicell inserts onto a regeneration medium in plastic dishes by use of a sterilized forcep. Incubate under a 13/11 h light/dark cycle at 28 Â°C for approximately 15-30 days (Figure 3C-d).</li><li>Transfer the calli with regenerated shoots into a rooting medium in plastic dishes and culture them under a 13/11 h light/dark cycle at 28 Â°C for around 15 days (Figure 3C-e).</li><li>Transfer the resulting plantlets into soil and grow in environmental chambers. If needed, harvest seeds from the regenerated plants (Figure 3C-f).<br/><em>Note: In our study, approximately 60%-70% of the PEG-Ca<sup>2+</sup> transfected zygotes grew and regenerated into plantlets.</em><br/><br/><img alt="" class="layerphoto" height="329" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713082217_1389.jpg" width="300"/><br/><strong>Figure 3. Culture of rice egg cells and zygotes in a droplet (A) and a millicell insert (B), and developmental profiles of rice zygotes (C).</strong> A. Illustration of the culture method for rice egg cells in a droplet of zygote culture medium with feeder cells. B. An illustration of the culture method for rice egg cells or zygotes in a millicell insert. C. Developmental profiles of rice zygotes. a. A zygote at 2 h after fusion. b. A globular embryo-like structure at 3 days after fusion. c. A cell mass at 10 days after fusion. d. A white callus at 25 days after fusion. e. A callus with regenerated shoots after 23 days of subculturing in regeneration media (47 days after fusion). f. A plantlet after 17 days of subculturing a callus with regenerated shoots in rooting medium (63 days after fusion). Scale bars = 20 Î¼m in a and b, 50 Âµm in c, 1 mm in d, 2 mm in e and 1 cm in f.<br/><br/></li></ol></li><li>Genome editing in rice by direct delivery of CRISPR-Cas9 vector into zygotes<ol class="npwtwo"><li>Use the Focas webtool for selection of target sequences on genes of interest.</li><li>In the âon_target_score_calculatorâ of the Focas webtool, upload the genome file (IRGSP-1.0_genome.fasta) and add target genome sequence information (up to 1 kbp) to obtain candidates of gRNA target sequences (20 mer).</li><li>Select gRNA target sequences (on_target_score: more than 0.4, without poly T sequence) (Figure 4).<br/><br/><img alt="" class="layerphoto" height="88" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713082234_4895.jpg" width="300"/><br/><strong>Figure 4. The candidates of gRNA target sequences found by the Focas webtool</strong><br/><br/></li><li>Put the selected gRNA sequences with a PAM sequence in âCasOTâ of the Focas webtool for off-target searches. Select gRNA sequences that show low off-target scores.</li><li>Predict the secondary structure of single guide RNAs (sgRNAs) containing the selected gRNA sequences using the RNA Folding Form software, available from the mfold web server.<br/><em>Note: If prediction of the secondary structure of the sgRNAs indicates that a stem-loop structure will be formed through complementary binding in the target-sequence region, avoiding the selected sgRNAs is recommended, as the difference in the targeting efficiency is most likely due to the secondary structure of the guide sequence (Jensen et al., 2017).</em> </li><li>After selection of the target sequence, prepare a vector harboring a CRISPR-Cas9 expression cassette as below.<br/><em>Note: In our study, the tRNA-based multiplex CRISPR-Cas9 vector (pMgPoef4_129-2A-GFP; accession number LC460477) used for both single and multiplex gRNA expression was employed for genome editing. In brief, the vector harbors an O. sativa codon-optimized Streptococcus pyogenes Cas9 (OsCas9) sequence, fused to GFP via a self-cleaving 2A peptide under the control of the ubiquitin promoter, and the tRNA-gRNA units under a OsU3 gene promoter. The details of the vector construction are described in Toda et al. (2019).</em> </li><li>To construct the tRNA-gRNA units for single genome editing, amplify the target sequence by PCR using specific primers to obtain an artificial gene including tRNA and gRNA scaffold sequences.<br/><em>Note: Information on specific primers is described in Toda et al. (2019).</em> </li><li>To construct the tRNA-gRNA units for multiplex genome editing, amplify the tRNA-gRNA1-tRNA-gRNA2 sequence by PCR using specific primers to obtain an artificial gene including tRNA and gRNA scaffold sequences.<br/><em>Note: Information on specific primers is described in Toda et al. (2019).</em> </li><li>Insert amplified target sequences or tRNA-gRNA fragments into the <em>BsaI</em> site of the multiplex CRISPR-Cas9 vector using Golden Gate Cloning methods (New England Biolabs).</li><li>Deliver the produced CRISPR-Cas9 vector into rice zygotes by PEG-Ca<sup>2+</sup> transfection according to Procedure B.</li><li>Culture the transfected zygotes as described in Procedure C.<br/><br/></li></ol></li><li>Genome editing in rice by direct delivery of Cas9-gRNA ribonucleoproteins into zygotes<ol class="npwtwo"><li>Select target sequences on each target gene according to Steps D1-D5.</li><li>Synthesize and purify sgRNA containing the target sequence using the GeneArt Precision gRNA Synthesis Kit according to the manufacturerâs instructions (Thermo Fisher Scientific).</li><li>Premix Cas9 protein (1 Âµl) and <em>in vitro</em> transcribed sgRNA (2-3 Âµg) in a total volume of 1.5-2.1 Âµl and incubate the premix solution at room temperature for 10 min.</li><li>After incubation, gently mix the solution containing the CRISPR RNPs with 10 Âµl of MMG solution (MMG-RNP).</li><li>Perform PEG-Ca<sup>2+</sup> transfection of rice zygotes as Procedure B except for the use of MMG-RNP instead of MMG-DNA.</li><li>Culture the transfected zygotes as described in Procedure C.<br/><br/></li></ol></li><li>Validation of targeted mutagenesis in the plants regenerated from PEG-Ca<sup>2+</sup>-transfected zygotes by genomic DNA PCR and sequencing of the target sites<ol class="npwtwo"><li>Cut small pieces of the leaves (2-3 mm<sup>2</sup>) of rice plants regenerated from zygotes previously PEG-Ca<sup>2+</sup>-transfected with a CRISPR-Cas9 plasmid or RNP.</li><li>Prepare genomic DNAs from the cut leaves using PrepMan Ultra according to the manufacturerâs instructions.<br/><em>Note: In our study, a cut plant leaf is put into a 40 Âµl solution of PrepMan Ultra and heated at 95 Â°C for 20 min. Thereafter, the heated sample is diluted with 100 Âµl sterilized water for subsequent genomic DNA PCR.</em> </li><li>Amplify PCR fragments containing the target site using PrimeSTAR GXL DNA polymerase with genomic DNAs and a primer set for each target.</li><li>After genomic DNA PCR, purify the amplified PCR fragments using ExoSAP-IT Express PCR Cleanup Reagents, and determine the nucleotide sequence of the PCR fragment.</li><li>Confirm targeted mutagenesis of the genome-edited plants by Parallel Editor program in GENETYX software. Sanger sequencing chromatograms at target sites of genes on genomic DNA should also be confirmed using SnapGene Viewer.</li><li>When the sequencing results show heterologous target mutations, the amplified PCR fragments in Step F3 are subcloned into the pCR-Blunt vector using the Zero Blunt PCR Cloning Kit, and sequence of plasmid inserts from four or more independent clones of transformed <em>E. coli</em> are determined.</li><li>Confirm targeted mutagenesis of genome-edited plants as described in Step F5.<br/><em>Note: A flow chart of the present genome editing procedures are presented in Figure 5.</em><br/><br/><img alt="" class="layerphoto" height="126" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200713/20200713082255_5709.jpg" width="300"/><br/><strong>Figure 5. A Flow chart of the procedures for genome editing in rice by direct delivery of the CRISPR-Cas9 vector or RNP in zygotes.</strong> The upper section shows the duration in days of each procedure. Pink, green and orange circles indicate the egg, sperm and zygotic nuclei, respectively. The yellow lightning bolt and green leaf shapes indicate the point of electro-fusion and plant leaves, respectively.<br/></li></ol></li></ol><p class="pptt" id="biaoti34733">### Recipes</p><ol class="npwtwok"><li>MMG solution<br/>4 mM MES-KOH (pH 5.7)<br/>15 mM MgCl<sub>2</sub><br/>In mannitol solution (370 mosmol/kg H<sub>2</sub>O for rice egg cells; 450 mosmol/kg H<sub>2</sub>O for rice zygotes)</li><li>PEG-Ca<sup>2+</sup> solution<br/>30% PEG4000<br/>100 mM CaCl<sub>2</sub><br/>In mannitol solution (370 mosmol/kg H<sub>2</sub>O for rice egg cells; 450 mosmol/kg H<sub>2</sub>O for rice zygotes)<br/></li></ol><p class="pptt" id="biaoti34734">### Acknowledgments</p><p class="ppzz">This protocol was adapted from Uchiumi <em>et al.</em> (2007), Okamoto (2011), Toda <em>et al.</em> (2016 and 2019), Koiso <em>et al.</em> (2017), Toda and Okamoto (2020). This work was supported in part by a research fellowship from the JSPS (Grant-in-Aid for JSPS Research Fellow; grant no. 19J12912 to ET), MEXT KAKENHI (Grant-in-Aid for Scientific Research on Innovation Areas; grant nos. 17H05845 and 19H04868 to TO).</p><p class="pptt" id="biaoti34735">### Competing interests</p><p class="ppzz">ET and TO are co-inventors on a patent application covering the protocols described in this paper.</p><p class="pptt" id="biaoti34736">### References</p><ol class="npwtwok"><li>Abiko, M., Maeda, H., Tamura, K., Hara-Nishimura, I. and Okamoto, T. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23570690" target="_blank">Gene expression profiles in rice gametes and zygotes: identification of gamete-enriched genes and up- or down-regulated genes in zygotes after fertilization.</a> <em>J Exp Bot</em> 64(7): 1927-1940.</li><li>Anderson, S. N., Johnson, C. S., Jones, D. S., Conrad, L. J., Gou, X., Russell, S. D. and Sundaresan, V. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24215296" target="_blank">Transcriptomes of isolated <em>Oryza sativa</em> gametes characterized by deep sequencing: evidence for distinct sex-dependent chromatin and epigenetic states before fertilization.</a> <em>Plant J</em> 76(5): 729-741.</li><li>Guignard., M. L. (1899). <a href="https://www.researchgate.net/publication/247252023_Botanique_-_Sur_les_antherozoides_et_la_double_copulation_sexuelle_chez_les_vegetaux_angio-spermes" target="_blank">Sur les anthÃ©rozoÃ¯des et la double copulation sexuelle chez les vegetaux angiosperms.</a> <em>Rev GÃ©n Bot</em> 11: 129-135.</li><li>Jensen, K. T., FlÃ¸e, L., Petersen, T. S., Huang, J., Xu, F., Bolund, L., Luo, Y. and Lin, L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28580607" target="_blank">Chromatin accessibility and guide sequence secondary structure affect CRISPR-Cas9 gene editing efficiency.</a> <em>FEBS Lett</em> 591(13): 1892-1901.</li><li>Khanday, I., Skinner, D., Yang, B., Mercier, R. and Sundaresan, V. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30542157" target="_blank">A male-expressed rice embryogenic trigger redirected for asexual propagation through seeds.</a> <em>Nature</em> 565(7737): 91-95.</li><li>Koiso, N., Toda, E., Ichikawa, M., Kato, N. and Okamoto, T. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31245659" target="_blank">Development of gene expression system in egg cells and zygotes isolated from rice and maize.</a> <em>Plant Direct</em> 1(3): e00010.</li><li>Kranz, E. and LÃ¶rz, H. (1993). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12271084" target="_blank"><em>In vitro</em> fertilization with isolated, single gametes results in zygotic embryogenesis and fertile maize plants.</a> <em>Plant Cell</em> 5(7): 739-746.</li><li>Maryenti, T., Kato, N., Ichikawa, M. and Okamoto, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30605551" target="_blank">Establishment of an <em>in vitro</em> fertilization system in wheat (<em>Triticum aestivum</em> L.).</a> <em>Plant Cell Physiol</em> 60(4): 835-843.</li><li>Nakajima, K., Uchiumi, T. and Okamoto, T. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20462944" target="_blank">Positional relationship between the gamete fusion site and the first division plane in the rice zygote.</a> <em>J Exp Bot</em> 61(11): 3101-3105.</li><li>Nawaschin, S. (1989). <a href="http://www.mathnet.ru/php/archive.phtml?wshow=paper&amp;jrnid=im&amp;paperid=4615&amp;option_lang=eng" target="_blank">Resultate einer Revision der BefruchtungsvorgaÌnge bei Lilium Martagon und Fritillaria tenella.</a> <em>Bulletin de l'AcadeÌmie ImpeÌriale des Sciences</em> 9(4):377-382.</li><li>Ning, J., Peng, X. B., Qu, L. H., Xin, H. P., Yan, T. T. and Sun, M. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16510144" target="_blank">Differential gene expression in egg cells and zygotes suggests that the transcriptome is restructed before the first zygotic division in tobacco.</a> <em>FEBS Lett</em> 580(7): 1747-1752.</li><li>Ohnishi, T., Takanashi, H., Mogi, M., Takahashi, H., Kikuchi, S., Yano, K., Okamoto, T., Fujita, M., Kurata, N. and Tsutsumi, N. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21106719" target="_blank">Distinct gene expression profiles in egg and synergid cells of rice as revealed by cell type-specific microarrays.</a> <em>Plant Physiol</em> 155(2): 881-891.</li><li>Okamoto, T. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21207258" target="_blank"><em>In vitro</em> fertilization with rice gametes: production of zygotes and zygote and embryo culture.</a> <em>Methods Mol Biol</em> 710: 17-27.</li><li>Raghavan, V. (2003). <a href="http://onlinelibrary.wiley.com/doi/10.1046/j.1469-8137.2003.00846.x/full" target="_blank">Some reflections on double fertilization, from its discovery to the present.</a> <em>New Phytol</em> 159(3): 565-583.</li><li>Rahman, M. H., Toda, E., Kobayashi, M., Kudo, T., Koshimizu, S., Takahara, M., Iwami, M., Watanabe, Y., Sekimoto, H., Yano, K. and Okamoto, T. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30801122" target="_blank">Expression of genes from paternal alleles in rice zygotes and involvement of <em>OsASGR-BBML1</em> in initiation of zygotic development.</a> <em>Plant Cell Physiol</em> 60(4): 725-737.</li><li>Russell, S. D. (1992). <a href="http://www.sciencedirect.com/science/article/pii/S007476960861102X" target="_blank">Double fertilization.</a> <em>International Review of Cytology</em> 140: 357-388.</li><li>Sprunck, S., Baumann, U., Edwards, K., Langridge, P. and Dresselhaus, T. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15703054" target="_blank">The transcript composition of egg cells changes significantly following fertilization in wheat (<em>Triticum aestivum</em> L.).</a> <em>Plant J</em> 41(5): 660-672.</li><li>Steffen, J. G., Kang, I. H., Macfarlane, J. and Drews, G. N. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17559508" target="_blank">Identification of genes expressed in the <em>Arabidopsis</em> female gametophyte.</a> <em>Plant J</em> 51(2): 281-292.</li><li>Toda, E., Ohnishi, Y. and Okamoto, T. (2016). <a href="https://bio-protocol.org/e2074" target="_blank">Electro-fusion of gametes and subsequent culture of zygotes in rice.</a> <em>Bio-protocol</em> 6(24): e2074.</li><li>Toda, E., Koiso, N., Takebayashi, A., Ichikawa, M., Kiba, T., Osakabe, K., Osakabe, Y., Sakakibara, H., Kato, N. and Okamoto, T. (2019). <a href="https://www.nature.com/articles/s41477-019-0386-z" target="_blank">An efficient DNA- and selectable-marker-free genome-editing system using zygotes in rice.</a> <em>Nat Plants</em> 5(4): 363-368.</li><li>Toda, E. and Okamoto, T. (2020). <a href="https://currentprotocols.onlinelibrary.wiley.com/doi/abs/10.1002/cppb.20111" target="_blank">CRISPR/Cas9-based genome editing using rice zygotes.</a> <em>Curr Protoc Plant Biol</em> 5(2): e20111.</li><li>Uchiumi, T., Uemura, I. and Okamoto, T. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17361458" target="_blank">Establishment of an <em>in vitro</em> fertilization system in rice (<em>Oryza sativa</em> L.).</a> <em>Planta</em> 226(3): 581-589.</li><li>Wang, D., Zhang, C., Hearn, D. J., Kang, I. H., Punwani, J. A., Skaggs, M. I., Drews, G. N., Schumaker, K. S. and Yadegari, R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20550711" target="_blank">Identification of transcription-factor genes expressed in the <em>Arabidopsis</em> female gametophyte.</a> <em>BMC Plant Biol</em> 10: 110.</li><li>Wang, Y. Y., Kuang, A., Russell, S. D. and Tian, H. Q. (2006). <a href="http://link.springer.com/article/10.1007/s00497-006-0029-1" target="_blank"><em>In vitro</em> fertilization as a tool for investigating sexual reproduction of angiosperms.</a> <em>Sex Plant Reprod</em> 19(3): 103-115.</li><li>Wuest, S. E., Vijverberg, K., Schmidt, A., Weiss, M., Gheyselinck, J., Lohr, M., Wellmer, F., RahnenfÃ¼hrer, J., von Mering, C. and Grossniklaus, U. (2010). <a href="https://www.cell.com/current-biology/fulltext/S0960-9822(10)00140-5?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS0960982210001405%3Fshowall%3Dtrue" target="_blank"><em>Arabidopsis</em> female gametophyte gene expression map reveals similarities between plant and animal gametes.</a> <em>Curr Biol</em> 20(6): 506-512.</li><li>Yang, H., Kaur, N., Kiriakopolos, S. and McCormick, S. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16718485" target="_blank">EST generation and analyses towards identifying female gametophyte-specific genes in <em>Zea mays</em> L.</a> <em>Planta</em> 224(5): 1004-1014.</li><li>Zhao, P., Zhou, X., Shen, K., Liu, Z., Cheng, T., Liu, D., Cheng, Y., Peng, X. and Sun, M. X. (2019) <a href="https://www.sciencedirect.com/science/article/abs/pii/S1534580719302862" target="_blank">Two-step maternal-to-zygotic transition with two-phase parental genome contributions.</a> <em>Dev Cell</em> 49(6): 1-12.<br/></li></ol>