<body>
            <h1>A SsrA/NIa-based Strategy for Post-Translational Regulation of Protein Levels in Gram-negative Bacteria</h1>
            <p><strong>Authors:</strong> Gonzalo Durante-RodrÃ­guez, BelÃ©n Calles, VÃ­ctor  de Lorenzo, Pablo I. Nikel</p>
            <h2>Abstract</h2>
            <p>Strategies to control the levels of key enzymes of bacterial metabolism are commonly based on the manipulation of gene of interest within the target pathway. The development of new protocols towards the manipulation of biochemical processes is still a major challenge in the field of metabolic engineering. On this background, the FENIX (<em>functional engineering of SsrA/NIa-based flux control</em>) system allows for the post-translational regulation of protein levels, providing both independent control of the steady-state protein amounts and inducible accumulation of target proteins. This strategy enables an extra layer of control over metabolic fluxes in bacterial cell factories (see Graphical abstract below). The protocol detailed here describes the steps needed to design FENIX-tagged proteins and to adapt the system to virtually any pathway for fine-tuning of metabolic fluxes.<br/><br/><strong>Graphical abstract</strong><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715120524_5449.jpg" width="300"/><br/></p>
            <p><strong>Keywords:</strong> Synthetic biology, Metabolic engineering, Proteolysis, <em>Escherichia coli</em>, Pathway engineering</p>
            <h2>Background</h2><p>Controlling protein production has become a challenging problem that has been mostly tackled by manipulating the regulation of its production at different levels, such as the DNA level or the amount of protein(s) produced (Wu <em>et al.</em>, 2016; Avcilar-Kucukgoze <em>et al.</em>, 2017). At the DNA level, both transcription and translation have been widely studied in several microorganisms, enabling the design and development of a large number of synthetic circuits to improve bioproduction processes (GuzmÃ¡n <em>et al.</em>, 1995; Lutz and Bujard, 1997). In contrast, the protein level has been much less studied, with protocols mainly based on RNA interference, riboregulators and specific transcriptional regulators (Isaacs <em>et al.</em>, 2004; Lou <em>et al.</em>, 2012).<br/> <br/>In order to implement a new approach towards the regulation of protein levels, we recently proposed a synthetic biology tool called FENIX (<em>functional engineering of SsrA/NIa-based flux control</em>) (Durante-RodrÃ­guez <em>et al.</em>, 2018). FENIX is based on the activity of the protease NIa, capable of recognizing and cleaving a specific sequence of 13 amino acids (the <em>nia</em> target, GESNVVVHQADER) (Verchot <em>et al.</em>, 1991; Stevens, 2000; Calles <em>et al.</em>, 2019), and the action of the proteasome, which recognizes a sequence of 11 amino acids located in the C-terminal region of a protein (the <em>ssrA</em> target, AANDENYALAA) (Doma and Parker, 2007; Shoemaker <em>et al.</em>, 2010). We have designed a synthetic NIa/SsrA tag (GESNVVVHQADERâ¢AANDENYALAA) that can be easily fused to the C-terminal module of any protein of interest via a single cloning step in a standardized vector. The FENIX system relies on the constitutive degradation of the target protein by the proteasome, followed by conditionally restoring protein accumulation by the cleavage of the proteasome tag (<em>ssrA</em>) through the action of the protease NIa. The whole circuit is triggered by the addition of 3-methylbenzoate, which induces the XylS/<em>Pm</em> expression system, thus controlling the individual components of the FENIX system. This novel tool can be also used in <em>Pseudomonas putida</em> for tight control of protein accumulation (Volke <em>et al.</em>, 2020). The protocol described below discusses the overall strategy along with specific details on its implementation.</p>
        </body>

<p class="pptt" id="biaoti34769">## Materials and Reagents</p><ol class="npwone"><li>Kits<ol class="npwtwo"><li>High-Pure plasmid isolation kit (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11754777001 </a> )</li><li>Gene-Clean Turbo kit (Q-BIOgene, catalog number: <a class="blackNoUnder" target="_blank"> 1102-400 </a> )<br/><br/></li></ol></li><li>Enzymes<ol class="npwtwo"><li><em>Pfu</em> DNA polymerase (Promega, catalog number: <a class="blackNoUnder" target="_blank"> M7741 </a> )</li><li>T4 DNA ligase (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0202L </a> )</li><li>Restriction enzyme <em>Nhe</em>I (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R3131S </a> )</li><li>Restriction enzyme <em>Bsr</em>GI (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R3575S </a> )</li><li>Restriction enzyme <em>Spe</em>I (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R3133L </a> )<br/><br/></li></ol></li><li>Chemicals and other consumables<ol class="npwtwo"><li>0.22 Î¼m filter membranes (Merck, catalog number: <a class="blackNoUnder" target="_blank"> GSWP02500 </a> )</li><li>Multiwell microtiter plates, 96-well, Flat Bottom (Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 353072 </a> )</li><li>50 ml plastic tubes (Falcon)</li><li>2-ml microcentrifuge tubes</li><li>1.5-ml microcentrifuge tubes</li><li>Aluminum foil</li><li>Petri dishes</li><li>Chloramphenicol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C0378-25G </a> )</li><li>Kanamycin sulfate (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 10106801001 </a> )</li><li>Absolute ethanol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1009832500 </a> )</li><li>85% (v/v) glycerol (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1040942500 </a> )</li><li>NaCl (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.06404.1000 </a> )</li><li>NaOH (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.06498.0500 </a> )</li><li>CaCl<sub>2</sub> (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.02382.1000 </a> )</li><li>RbCl (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> R2252-100G </a> )</li><li>MnCl<sub>2</sub> (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.05927.0100 </a> )</li><li>Potassium acetate (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.04820.1000 </a> )</li><li>Acetic acid (glacial) (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 1.00063.1000 </a> )</li><li>MOPS (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M1254-250G </a> )</li><li>3-methylbenzoate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 89890 </a> )</li><li>Agar (Conda, catalog number: <a class="blackNoUnder" target="_blank"> 1806 </a> )</li><li>Agarose D1 (Conda, catalog number: <a class="blackNoUnder" target="_blank"> 8019.22 </a> )</li><li>Bacto<sup>TM</sup> tryptone (BD, catalog number: <a class="blackNoUnder" target="_blank"> 211705 </a> )</li><li>Bacto<sup>TM</sup> yeast (BD, catalog number: <a class="blackNoUnder" target="_blank"> 212750 </a> )</li><li>Competent <em>E. coli</em> DH10B cells (see Recipes)</li><li>Solution for DH10B competent cells TBFI (see Recipes)</li><li>Solution for DH10B competent cells TBFII (see Recipes)</li><li>30 mg/ml chloramphenicol and 50 mg/ml kanamycin (see Recipes)</li><li>LB broth (see Recipes)</li><li>LB agar with 30 Î¼g/ml chloramphenicol and/or 50 Î¼g/ml kanamycin (see Recipes)</li><li>0.5 M 3-methylbenzoate (see Recipes)</li><li>50% (v/v) and 70% (v/v) ethanol (see Recipes)<br/></li></ol></li></ol><p class="pptt" id="biaoti34770">### Equipment</p><ol class="npwtwok"><li>ABI Prism 377 automated DNA sequencer (Applied Biosystems Inc.)</li><li>Electroporator (Gene Pulser, Bio-Rad)</li><li>Orbital shaker</li><li>Flasks (50 ml, 1 L)</li><li>Pipettes (Gilson, models: PIPETMAN P2, P10, P20, P100, P200, P1000, P5000)</li><li>Centrifuge (Tomy, model: <a class="blackNoUnder" target="_blank"> MX-301 </a> )</li><li>Thermal Cycler (Applied Biosystems, model: <a class="blackNoUnder" target="_blank"> 2720Thermal Cycler </a> )</li><li>Temperature chamber (Taitec, model: <a class="blackNoUnder" target="_blank"> Thermo minder SM-10R </a> )</li><li>Water bath shaker (Taitec, model: <a class="blackNoUnder" target="_blank"> Personal-11 </a> )</li><li>Plate reader (Corona, model: <a class="blackNoUnder" target="_blank"> MTP-880Lab </a> )</li><li>Autoclave (Tomy Seiko, model: <a class="blackNoUnder" target="_blank"> LSX-500 </a> )<br/></li></ol><p class="pptt" id="biaoti34771">### Software</p><ol class="npwtwok"><li>Microsoft Excel (Microsoft)</li><li>BioEdit Sequence Alignment Editor</li><li>ApE-A plasmid Editor (<a href="https://jorgensen.biology.utah.edu/wayned/ape" target="_blank">https://jorgensen.biology.utah.edu/wayned/ape</a>)<br/></li></ol><p class="pptt" id="biaoti34772">### Procedure</p><p class="ppzz">The design and construction of a target gene tagged according to the FENIX system could be divided into the following 4 steps (A to D), summarized in Figure 1.<br/><br/><img alt="" class="layerphoto" height="274" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715123156_7783.jpg" width="300"/><br/><strong>Figure 1. Rationale and construction of the FENIX system for one-step cloning and tagging of individual target proteins.</strong> The gene encoding the target polypeptide (gene of interest, <em>goi</em>) is amplified by PCR with specific oligonucleotides that include <em>Nhe</em>I and <em>Bsr</em>GI restriction sites. The resulting amplicon can be directly cloned into plasmid pFENIXÂ·<em>gfp*</em> (which contains a <em>nia/ssrA</em> tagged version of the green fluorescent protein, see Table 1 and <a href="https://os.bio-protocol.org/attached/file/20200706/3688 Supplementary pFENIX gfp sequence.docx" target="_blank">Supplementary file</a>) upon digestion with these two restriction enzymes. The <em>Nhe</em>I restriction site is located between the RBS and the ATG of <em>gfp</em> gene so the N-terminal region of <em>goi</em> will keep without any modification, and the C-terminal region will contain eight extra bp (in red) between <em>goi</em> and <em>nia</em> sequences, translated as XYK amino acids (X could be L, M or V amino acid depending on the last bp of the <em>goi</em>). After ligation, the new plasmid pFENIXÂ·<em>goi</em> is transformed into competent <em>E</em><em>. coli</em> DH10B cells. In all pFENIX plasmids, the expression of the <em>nia/ssrA</em>-tagged variant of the <em>goi</em> depends on the constitutive <em>P<sub>tetA</sub></em> promoter.<br/><br/><strong>Table 1. Bacterial strains and </strong><strong>plasm</strong><strong>ids</strong><br/><img alt="" class="layerphoto" height="508" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715123230_8164.png" width="800"/> </p><br/><ol class="npwone"><li>Construction of pFENIX plasmids for one-step cloning and tagging of individual target proteins<ol class="npwtwo"><li>Select the gene encoding the target polypeptide (gene of interest, <em>goi</em>).</li><li>Design specific oligonucleotides to amplify the <em>goi</em>, including <em>Nhe</em>I (5'-GCTAGC-3') and <em>Bsr</em>GI (5'-TGTACA-3') restriction sites in the 5' and 3' termini, respectively.</li><li>Set up a 100-Î¼l PCR reaction using proof-reading <em>Pfu</em> DNA polymerase (Promega). Standard reaction conditions can be used as described in the provider manualâs for <em>Pfu</em> DNA polymerase. In most cases, thermal cycling conditions are as follows: after heating at 98 Â°C for 5 min, set 30 cycles of 98 Â°C for 30 s (denaturation), 57 Â°C for 30 s (annealing, temperature varies depending on the actual Tm of the oligonucleotides used), and 72 Â°C for 1 min (elongation, time varies depending on the size of the <em>goi</em> to be amplified, typically 2 min/kb).</li><li>Purify the PCR product by using the Gene-Clean Turbo kit (Q-BIOgene).<br/><em>Note</em><em>: When the PCR amplification yields non-specific bands, the target product is cut from an agarose gel after electrophoresis by loading the total volume of PCR and the amplicon is gel-purified by means of the Gene-Clean Turbo kit.</em> </li><li>Digest plasmid pFENIX and the PCR product of the <em>goi</em> with the restriction enzymes <em>Nhe</em>I and <em>Bsr</em>GI, and then purify both DNA fragments using the Gene-Clean Turbo kit.</li><li>Clone the <em>goi</em> in the pFENIX backbone by the T4 DNA ligase (New England Biolabs), adding a <em>goi</em> concentration 5-10 times higher than the plasmid pFENIX in a final volume of 20 Î¼l.<br/><br/></li></ol></li><li>Cell transformation with the pFENIX-<em>goi</em> plasmid<ol class="npwtwo"><li>Keep competent <em>E. coli</em> DH10B cells on ice. This protocol has been carried out with strain DH10B taking into account its high competence skills, but any <em>E. coli</em> strain routinely used for cloning (<em>e.g.</em>, DH5Î±) could be likewise used.</li><li>Take the 20-Î¼l ligation mixture from the step before and add it into 0.1 ml of the cell suspension of competent <em>E. coli</em> DH10B (see Recipes), and then incubate the mixture on ice for 30 min.</li><li>Heat-shock the cells at 42 Â°C for 45 s, and immediately add 0.9 ml of LB broth to the mixture.</li><li>Incubate 1 ml of the suspension of <em>E. coli</em> DH10B transformants at 37 Â°C for 60 min.</li><li>Spread 0.1 ml and/or the rest of the suspension of <em>E</em><em>. coli</em> DH10B transformants onto LB agar plates with 30 Î¼g/ml chloramphenicol, and then incubate the plates at 37 Â°C overnight.</li><li>Isolate a transformant of <em>E. coli</em> DH10B harboring the cloned candidates on the LB agar with 30 Î¼g/ml chloramphenicol.</li><li>Inoculate a single colony into 10 ml of LB broth containing 30 Î¼g/ml chloramphenicol in a 50-ml conical tube, and incubate the culture in an orbital shaker at 37 Â°C with shaking (180 rpm) overnight.</li><li>Isolate the plasmid from the transformant cells of 10 ml overnight culture by means of the High-Pure plasmid isolation kit (Roche).</li><li>Check the DNA sequence of the tagged <em>goi</em> cloned into the pFENIX plasmid with an ABI Prism 377 automated DNA sequencer (Applied Biosystems Inc.), using the universal primers R24 (5'-AGCGGATAACAATTTCACACAGGA-3') and F24 (5'-CGCCAGGGTTTTCCCAGTCACGAC-3').</li><li>Compare the sequence of the clones with that of the reference sequence to ensure the absence of mutations. An alignment software such as Bioedit or ApE-A could be used.</li><li>Long-term storage of bacterial strains transformed with these plasmids is routinely done as 15% glycerol stocks according to well-established procedures (Sambrook and Russell, 2001).<br/><br/></li></ol></li><li>Electroporation of pFENIX recombinants with the plasmid encoding the NIa protease<ol class="npwtwo"><li>Incubate a 20-ml culture of <em>E. coli</em> DH10B harboring plasmid pFENIXÂ·<em>goi</em> in LB medium overnight at 37 Â°C with shaking (180 rpm) in an orbital shaker.</li><li>Distribute the whole culture in a 50-ml sterile conical tube and centrifuge at 2,300 <em>x g</em> and 4Â°C for 10 min.</li><li>Remove supernatant, resuspend the pellet gently in 10-ml of cold sterile water (4 Â°C) and centrifuge at 2,300 <em>x g</em> and 4 Â°C for 10 min.</li><li>Discard supernatant, add 1 ml of sterile cold water, resuspend the pellet and transfer the suspension to a 2-ml microcentrifuge tube.</li><li>Centrifuge at 3,800 <em>x g</em> and 4 Â°C for 5 min. Repeat this step 2 times.<br/><em>Note: The cells </em><em>a</em><em>re</em><em> ready to be electroporated when they form a very soft and loose pellet after the centrifugation with cold water.</em> </li><li>Dispose of the supernatant and add 300 Î¼l of cold sterile water, resuspend and distribute 100 Î¼l of the resulting suspension into 2-mm gap width electroporation cuvettes.</li><li>Add 200 ng of the plasmid pS238Â·NIa to the electroporation cuvette containing 100 Î¼l of the suspension of <em>E. coli</em> DH10B cells harboring the plasmid pFENIXÂ·<em>goi</em>, mix gently and proceed to electroporate in an electroporator (Gene Pulser, Bio-Rad).<br/><em>Note: The </em><em>par</em><em>ameters</em><em> of electroporation are resistance, 200 Î©; capacitance, 25 ÂµF; and voltage, 2.5 kV.</em> </li><li>Add 0.9 ml of LB medium to the cell suspension and incubate for 1 h at 37 Â°C with shaking (180 rpm) in an orbital shaker.</li><li>Plate the whole culture onto LB medium plates added with 30 Âµg/ml chloramphenicol and 50 Î¼g/ml kanamycin.</li><li>Streak a few colonies and check for the presence of both plasmid pFENIXÂ·<em>goi</em> and pS238Â·NIa in the transformant bacteria by plasmid extraction (High-Pure plasmid isolation kit, Roche).</li><li>Digest the mixture of both plasmids extracted with <em>S</em><em>p</em><em>e</em>I (5'-ACTAGT-3') and run an agarose gel electrophoresis to check the relevant sizes. Expect a band of approximately 3.6 kb plus the size of the corresponding <em>goi</em> with pFENIXÂ·<em>goi</em>, and a band of approximately 5.8 kb of the empty pS238Â·NIa vector.<br/><br/></li></ol></li><li>Protocol for activation of the protease NIa in the FENIX native system with GFP* (Figure 2)<ol class="npwtwo"><li>Select a single colony of the <em>E. coli</em> transformants harbouring the plasmids listed below (Table 1):<br/>pS238Â·NIa (negative control)<br/>pS341TÂ·<em>gfp</em> (positive control)<br/>pSEVA238 and pFENIXÂ·<em>gfp*</em> (control strain without the protease NIa)<br/>pS238Â·NIa and pFENIXÂ·<em>gfp*</em> </li><li>Individually inoculate the four recombinant strains in 50-ml conical tubes with 10 ml of LB medium containing 30 Î¼g/ml chloramphenicol (for the positive control), 50 Î¼g/ml kanamycin (for the negative control), or both antibiotics (for strains containing both plasmids).</li><li>Incubate the pre-culture overnight at 37 Â°C with shaking (180 rpm).</li><li>Inoculate 4 Î¼l of each overnight pre-culture in a 96-well microtiter plate containing 200 Î¼l of fresh LB medium per well, including 30 Î¼g/ml chloramphenicol and/or 50 Î¼g/ml kanamycin, and in presence or absence of 1 mM 3-methylbenzoate as the inducer of <em>ni</em><em>a</em> expression (in the case of strains harbouring the pFENIXÂ·<em>gfp*</em> plasmid). Each experiment should be carried out in three independent wells as technical replicates.</li><li>Incubate the 96-well microtiter plate at 37 Â°C with shaking in a plate reader during 24 h, till the stationary phase.</li><li>Measure the OD<sub>600</sub> and the GFP fluorescence in the plate reader at the endpoint of the growth curve (Figure 2). GFP is excited at 488 nm, the fluorescence signal is recovered at 511 nm, and it was used the bottom reading mode.<br/><br/><img alt="" class="layerphoto" height="199" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715123317_9303.jpg" width="300"/><br/><strong>Figure 2. Performance of the FENIX system with GFP fluorescence as the readout.</strong> <strong>A. Scheme of NIa and SsrA-dependent post-translational control of GFP with the FENIX system.</strong> The gene encoding the green fluorescence protein (GFP) is added with a synthetic, hybrid <em>nia/ssrA</em> tag, resulting in a tagged GFP variant in which the C-terminus displays the GESNVVVHQADERÂ·AANDENYALAA amino acid sequence. The SsrA tag is directly recognized by the ClpXP and ClpAP proteases of the bacterial proteasome <em>i</em><em>n </em><em>viv</em><em>o</em>, thus degrading the protein. Upon action of the specific potyvirus NIa protease (the recognition site in the synthetic <em>ni</em><em>a/ssrA</em> tag is indicated with an inverted red triangle in the diagram), the SsrA tag is released and GFP can be accumulated, resulting in the appearance of GFP<sup>+</sup> cells. <strong>B. Evaluation of the FENIX system in recombinant </strong><em><strong>E. coli</strong></em><strong> using GFP protein.</strong> Plasmid pFENIXÂ·<em>gfp*</em>, which contain the <em>nia/ssrA</em>-tagged version of GFP (indicated with blue and orange strips in the first row of the table), were transformed into <em>E. coli</em> DH10B carrying either plasmid pS238Â·NIa or the empty pSEVA238 vector (indicated as + and â, respectively, in the second row of the table). The first two columns represent a negative and positive control, respectively, carried out with <em>E. coli</em> DH10B transformed either with the empty pFENIX vector (<em>i.e.</em>, no GFP protein) or with a plasmid constitutively expressing the gene encoding <em>gfp*</em> (pS341TÂ·<em>gfp*</em>, Table 1). Strains indicated in the next four columns contain the <em>nia/ssrA</em>-tagged GFP. These experiments were done in 96-well microtiter plates containing LB medium with the necessary antibiotics and additives (1 mM 3-methylbenzoate as the inducer of <em>nia</em> expression, as indicated in the third row of the table), and were inoculated with a culture of the corresponding strain previously grown in LB medium with the necessary antibiotics overnight. Cells were incubated at 37 Â°C with rotary agitation, and fluorescence and bacterial growth (expressed as the optical density measured at 600 nm, OD<sub>600</sub>) were recorded after 24 h. The specific (Sp) activity of the fluorescent proteins under study was calculated as the arbitrary fluorescence units (a.f.u.) normalized to the OD<sub>600</sub>. Each bar represents the mean value of the Sp activity Â± standard deviation calculated from at least three independent experiments. The lower panel shows bacterial pellets harvested from shake-flask cultures after 24 h of incubation under the same growth conditions indicated for the microtiter-plate cultures as observed under blue light.<br/></li></ol></li></ol><p class="pptt" id="biaoti34773">### Data analysis</p><ol class="npwtwok"><li>Extract raw numeric data of both OD<sub>600</sub> values and the measurement of GFP fluorescence in absence or presence of the inducer as a text file.</li><li>Open a text file containing the raw numeric data in a software for data management such as Microsoft Excel, R or Sigma Plot.</li><li>Normalize the values of OD<sub>600</sub> and the net intensities of the GFP fluorescence.</li><li>Calculate the ratio of fluorescence to OD<sub>600</sub> as specific activity of the promoter of each culture by the following formula:<br/><br/><div style="text-align:center;"><strong>Sp = IF/OD</strong><sub><strong>600</strong></sub> </div><br/><strong>Sp</strong> is the specific activity<br/><strong>IF</strong> is the intensity of fluorescence in arbitrary units<br/><strong>OD</strong><sub><strong>600</strong></sub> is the cell concentration in units of optical density</li><li>Average the ratios from three independent experiments with the technical replicates that are used to calculate the mean specific activity along with the deviations typical corresponding.</li><li>To confirm a significant difference, calculated <em>P</em>-values by Studentâs <em>t</em>-test (one-tailed and unpaired) of statistical analysis should be evaluated at less than 0.01 (Durante-RodrÃ­guez <em>e</em><em>t al.</em>, 2018).<br/></li></ol><p class="pptt" id="biaoti34774">### Recipes</p><ol class="npwtwok"><li>Competent <em>E. coli</em> DH10B cells<br/><ol class="npwthree"><li>Inoculate a single colony of <em>E. coli</em> DH10B in 10 ml of LB broth in a 50-ml Erlenmeyer flask<br/></li><li>Incubate the pre-cultures overnight at 37 Â°C with shaking (180 rpm)<br/></li><li>Dilute the pre-culture 100-fold in 250 ml of fresh LB broth in a 1-L Erlenmeyer flask<br/></li><li>Incubate the cultures at 37 Â°C with shaking (180 rpm) until mid-exponential phase (optical density measured at 600 nm ~0.5-0.6)<br/></li><li>Keep the cell suspension in an ice batch for 30 min<br/></li><li>Distribute the 250 ml of culture to sterile 50-ml conical tubes<br/></li><li>Collect the cells by centrifugation (2,300 <em>x g</em>, 4 Â°C, 5 min) and combine the pellets in a single tube<br/></li><li>Suspend the cells in 30 ml of TBFI (see Recipe 2) per 100 ml of original cell suspension<br/></li><li>Incubate the cells in an ice bath for 2 h<br/></li><li>Collect the cells by centrifugation (2,300 <em>x g</em>, 4 Â°C, 5 min)<br/></li><li>Suspend the cells in 4 ml of TBFII (see Recipe 3) per 100 ml of cells<br/></li><li>Transfer the suspension to pre-chilled 1.5-ml microcentrifuge tubes in aliquots of 100 Î¼l<br/></li><li>Use an aliquot (~100 Î¼l) of the resuspended <em>E. coli</em> DH10B for transformation</li></ol></li><li>Solution for DH10B competent cells TBFI<br/><ol class="npwthree"><li>To prepare 250 ml of final volume of TBFI, dissolve 0.74 g of potassium acetate, 0.37 g of CaCl<sub>2</sub> and 43.1 ml of 85% (v/v) glycerol in distilled water<br/></li><li>Adjust the pH to 6.0 by dropwise addition of 0.2 M acetic acid<br/></li><li>Add 2.47 g of MnCl<sub>2</sub> and 3.02 g of RbCl, checking that the pH value stay at 5.8 (if not, re-adjust by dropwise addition of 0.2 M acetic acid)<br/></li><li>Sterilize the solution by filtration with a 0.22-Î¼m filter membrane (Merck)<br/></li><li>Store the sterilized solution at 4 Â°C wrapped in aluminum foil</li></ol></li><li>Solution for DH10B competent cells TBFII<br/><ol class="npwthree"><li>To prepare 250 ml of final volume of TBFII, dissolve 2.75 g of CaCl<sub>2</sub>, 0.3 g of RbCl, 0.52 g of MOPS and 43.1 ml of 85% (v/v) glycerol in distilled water<br/></li><li>Adjust the pH to 6.8 by dropwise addition of 10 N NaOH<br/></li><li>Sterilize the solution by filtration with 0.22-Î¼m filter membrane (Merck)<br/></li><li>Store the sterilized solution at 4 Â°C wrapped in aluminum foil</li></ol></li><li>30 mg/ml chloramphenicol and 50 mg/ml kanamycin<br/><ol class="npwthree"><li>Dissolve 0.3 g of chloramphenicol (Sigma) and 0.5 g of kanamycin sulfate (Sigma) in 10 ml of 70% (v/v) ethanol or distilled water, respectively<br/></li><li>Sterilize the solutions by filtration with 0.22-Î¼m filter membrane (Merck)<br/></li><li>Store the sterilized solutions at -20 Â°C</li></ol></li><li>LB broth<br/><ol class="npwthree"><li>Dissolve 10 g of Bacto<sup>TM</sup> tryptone, 5 g of Bacto<sup>TM</sup> yeast extract, and 10 g of NaCl in 800 ml of distilled water<br/></li><li>Adjust pH to 7.5 with 10 N NaOH<br/></li><li>Adjust volume to 1 L with distilled water<br/></li><li>Autoclave the solution (set at 121 Â°C and 20 min in an <a class="blackNoUnder" target="_blank"> LSX-500 </a> autoclave)</li><li>Store the sterile LB broth at room temperature<br/></li></ol></li><li>LB agar with 30 Î¼g/ml chloramphenicol and/or 50 Î¼g/ml kanamycin<br/><ol class="npwthree"><li>Dissolve 7.5 g of agar in 1 L of LB broth<br/></li><li>Autoclave (set 121 Â°C and 20 min in <a class="blackNoUnder" target="_blank"> LSX-500 </a> )<br/></li><li>Add chloramphenicol and/or kanamycin after cooling in the final concentration of 30 Î¼g/ml and 50 Î¼g/ml, respectively<br/></li><li>Pour the media into Petri dishes, and then let the LB agar solidify by cooling at room temperature</li><li>Store the LB agar plates upside down at 4 Â°C<br/></li></ol></li><li>0.5 M 3-methylbenzoate<br/><ol class="npwthree"><li>Dissolve 6.75 g 3-methylbenzoate (Sigma) in 100 ml of 50% (v/v) ethanol<br/></li><li>Sterilize the solution by filtration with 0.22-Î¼m filter membrane (Merck)<br/></li><li>Store the solution at 4 Â°C</li></ol></li><li>50% (v/v) and 70% (v/v) ethanol<br/><ol class="npwthree"><li>Dilute ethanol to 50% (v/v) or 70% (v/v) with sterile water<br/></li><li>Store at room temperature</li></ol></li></ol><p class="pptt" id="biaoti34775">### Acknowledgments</p><p class="ppzz">Financial support from The Novo Nordisk Foundation (NNF10CC1016517 and NNF 18CC0033664), the Danish Council for Independent Research (<em>SWEET</em>, DFF-Research Project 8021-00039B), and the European Unionâs Horizon2020 Research and Innovation Program under grant agreement No. 814418 (<em>SinFonia</em>) to P.I.N. is gratefully recognized. This work was funded also by the MADONNA (H2020-FET-OPEN-RIA-2017-1-766975), BioRoboost (H2020-NMBP-BIO-CSA-2018), SYNBIO4FLAV (H2020-NMBP/0500) and MIX-UP (H2020-Grant 870294) Contracts of the European Union and the S2017/BMD-3691 InGEMICS-CM Project of the Comunidad de Madrid (European Structural and Investment Funds) to VdL. This protocol was adapted from Durante-RodrÃ­guez <em>et al.</em> (2018) for the method to design and construct a post-translational metabolic switch driven to decoupling the bacterial growth from biopolymer production in <em>E. coli</em>.</p><p class="pptt" id="biaoti34776">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest.</p><p class="pptt" id="biaoti34777">### References</p><ol class="npwtwok"><li>Avcilar-Kucukgoze, I., and Ignatova, Z. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27704283" target="_blank">Rewiring host activities for synthetic circuit production: a translation view.</a> <em>Biotechnol Lett</em> 39(1): 25-31.</li><li>Calles, B., GoÃ±i-Moreno, A. and de Lorenzo, V. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31885200" target="_blank">Digitalizing heterologous gene expression in Gram-negative bacteria with a portable ON/OFF module.</a> <em>Mol Syst Biol</em> 15(12): e8777.</li><li>Doma, M. K. and Parker, R. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18022361" target="_blank">RNA quality control in eukaryotes.</a> <em>C</em><em>ell</em> 131(4): 660-668.</li><li>Durante-RodrÃ­guez, G., de Lorenzo, V. and Nikel, P. I. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30346720" target="_blank">A post-translational metabolic switch enables complete decoupling of bacterial growth from biopolymer production in engineered <em>Escherichia coli</em>.</a> <em>ACS Synth Biol</em> 7(11): 2686-2697.</li><li>Durfee, T., Nelson, R., Baldwin, S., Plunkett, G., 3rd, Burland, V., Mau, B., Petrosino, J. F., Qin, X., Muzny, D. M., Ayele, M., Gibbs, R. A., CsoÌrgoÌ, B., PoÌsfai, G., Weinstock, G. M. and Blattner, F. R. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18245285" target="_blank">The complete genome sequence of <em>Escherichia coli</em> DH10B: insights into the biology of a laboratory workhorse.</a> <em>J Bacteriol</em> 190(7): 2597-2606. </li><li>GuzmÃ¡n, L. M., Belin, D., Carson, M. J. and Beckwith, J. (1995). <a href="https://www.ncbi.nlm.nih.gov/pubmed/7608087" target="_blank">Tight regulation, modulation, and high-level expression by vectors containing the arabinose PBAD promoter.</a> <em>J Bacteriol</em> 177(14): 4121-4130.</li><li>Isaacs, F. J., Dwyer, D. J., Ding, C., Pervouchine, D. D., Cantor, C. R. and Collins, J. J. (2004). <a href="https://www.nature.com/articles/nbt986" target="_blank">Engineered riboregulators enable post-transcriptional control of gene expression.</a> <em>Nat Biotechnol</em> 22(7): 841-847.</li><li>Lou, C., Stanton, B., Chen, Y. J., Munsky, B. and Voigt, C. A. (2012). <a href="https://www.nature.com/articles/nbt.2401" target="_blank">Ribozyme-based insulator parts buffer synthetic circuits from genetic context.</a> <em>Nat Biotechnol</em> 30(11): 1137-1142.</li><li>Lutz, R. and Bujard, H. (1997). <a href="https://www.ncbi.nlm.nih.gov/pubmed/9092630" target="_blank">Independent and tight regulation of transcriptional units in <em>Escherichia coli</em> via the LacR/O, the TetR/O and AraC/I<sub>1</sub>-I<sub>2</sub> regulatory elements.</a> <em>Nucleic Acids Res</em> 25(6): 1203-1210.</li><li>Sambrook, J. and Russell, D. W. (2001). Molecular cloning: a laboratory manual. Cold Spring Harbor Laboratory, Cold Spring Harbor, NY, USA.</li><li>Shoemaker, C. J., Eyler, D. E. and Green, R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20947765" target="_blank">Dom34:Hbs1 promotes subunit dissociation and peptidyl-tRNA drop-off to initiate no-go decay.</a> <em>Scie</em><em>nce</em> 330(6002): 369-372.</li><li>Silva-Rocha, R., Martinez-Garcia, E., Calles, B., ChavarrÃ­a, M., Arce-RodrÃ­guez, A., de Las Heras, A., PaÌez-Espino, A. D., Durante-RodrÃ­guez, G., Kim, J., Nikel, P. I., Platero, R. and de Lorenzo, V. (2013). <a href="https://academic.oup.com/nar/issue/41/D1" target="_blank"><em>Nucleic Acids Research.</em></a> 41(Database issue): D666-675.</li><li>Stevens, R. C. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10986469" target="_blank">Design of high-throughput methods of protein production for structural biology.</a> <em>Struc</em><em>ture</em> 8(9): R177-185. </li><li>Verchot, J., Koonin, E. V. and Carrington, J. C. (1991). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1962435" target="_blank">The 35-kDa protein from the N-terminus of the potyviral polyprotein functions as a third virus-encoded proteinase.</a> <em>Vir</em><em>ology</em> 185(2): 527-535.</li><li>Volke, D. C., Turlin, J., Mol, V. and Nikel, P. I. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30864281" target="_blank">Physical decoupling of XylS/Pm regulatory elements and conditional proteolysis enable precise control of gene expression in <span id="__kindeditor_bookmark_start_156__" style="display:none;"></span><em>Pseudomonas putida</em>.</a> <em>Microb Biotechnol</em> 13(1): 222-232. </li><li>Wu, G., Yan, Q., Jones, J. A., Tang, Y. J., Fong, S. S., and Koffas, M. A. G. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26996613" target="_blank">Metabolic burden: cornerstones in synthetic biology and metabolic engineering applications.</a> <em>Trends Biotechnol</em> 34(8): 652-664.<br/></li></ol>