<body>
            <h1>Generation and <em>in Planta</em> Functional Analysis of Potato Virus Y mutants</h1>
            <p><strong>Authors:</strong> Katja Stare, Anna Coll, Ion GutiÃ©rrez-Aguirre, Magda TuÅ¡ek Å½nidariÄ, Maja Ravnikar, Andreja KeÅ¾ar, Luka  KavÄiÄ, Marjetka Podobnik, Kristina Gruden</p>
            <h2>Abstract</h2>
            <p>Potato virus Y (PVY), the type member of the genus <em>Potyvirus</em> (family <em>Potyviridae</em>), is the most widespread virus affecting potato and is included in the top five most economically detrimental plant viruses. Recently, the structure of the PVY virion has been determined by cryo-electron microscopy, which has opened the doors to functional studies that explore the involvement of selected amino acids in different stages of the viral cycle. The only way to functionally challenge <em>in planta</em> the role of particular amino acids in the coat protein of PVY, or in other viral proteins, is by using cDNA clones. The use and manipulation of PVY cDNA clones, unlike those of other potyviruses, has been traditionally impaired by the toxicity that certain sequences within the PVY genome pose to <em>Escherichia coli</em>. Here, we describe the use of a published PVY cDNA clone, which harbours introns that overcome the aforementioned toxicity, to explore the effects of different coat protein modifications on viral infection. The protocol includes manipulation of the cDNA clone in <em>E. coli</em>, biolistic inoculation of plants with the constructed clones, observation of the biological effects on plants, quantification of cDNA clones by reverse transcription quantitative PCR, and confirmation of virion formation by transmission electron microscopy. Future possibilities involve the use of PVY cDNA clones tagged with fluorescent protein reporters to allow further insights into the effects of coat protein mutations on the cell-to-cell movement of PVY virions.</p>
            <p><strong>Keywords:</strong> Potato virus Y, Deletion mutants, Functional analysis, Infective cDNA clones, Biolistic inoculation</p>
            <h2>Background</h2><p>Potato virus Y (PVY) is the most economically important virus that infects cultivated potato and causes major crop losses worldwide (Valkonen, 2007). Besides potato, PVY can infect a broad range of other plant species, including mainly other cultivated <em>Solanacea</em> species such as tomato, pepper, or tobacco, as well as weeds and ornamentals (Kerlan, 2006). PVY is naturally transmitted by aphids in a non-persistent manner. Up to 65 aphid species have been reported to be potential vectors for PVY, which makes it difficult to implement efficient viral control and management measures (Lacomme and Jacquot, 2017). Typical for the <em>Potyviridae</em> family, the PVY virion is structured as a flexuous filamentous particle that encapsidates a 9.7 kb positive-sense single-stranded RNA genome. It encodes 10 proteins synthesised from a single open reading frame and one protein, P3N-PIPO, synthesised from an overlapping reading frame (Chung <em>et al.</em>, 2008).<br/><br/>Understanding the complex interaction network established between PVY and its host plant is indispensable for designing more efficient and long-lasting virus control measures, such as targeted crop-breeding strategies (Lacomme and Jacquot, 2017). During the last decade, technological advancements have improved our knowledge of potato-PVY interactions (KriÅ¾nik <em>et al.</em>, 2017; Lukan <em>et al.</em>, 2018; Stare <em>et al.</em>, 2019) and viral mechanisms of the infection cycle (MÃ¤kinen and HafrÃ©n, 2014; MÃ¤kinen <em>et al.</em>, 2017). The potential of such technical developments has been demonstrated in our recent study (KeÅ¾ar <em>et al.</em>, 2019). By combining knowledge of the PVY virion structure (based on high-resolution cryo-electron microscopy) with <em>in planta</em> coat protein (CP) functional analysis (using a PVY cDNA clone), we provided insights into the crucial role of CP and its N- and C- terminal regions in PVY infectivity.<br/><br/>The study reveals the value of an easy-to-manipulate PVY cDNA infectious clone for <em>in planta</em> functional analysis of the virus. The production and use of PVY infectious clones has been more challenging than that for other potyviruses due to the expression of cryptic prokaryotic promoters in the PVY genome, which results in products that are cytotoxic to <em>E. coli</em> (Fakhfakh <em>et al.</em>, 1996; Jakab <em>et al.</em>, 1997). Thus, only a few PVY cDNA clones have been used to date to investigate the viral determinants involved in the vein necrosis symptom (Tribodet <em>et al.</em>, 2005), the rate of PVY cell-to-cell movement via inclusion of a GFP reporter gene into the polyprotein (Rupar <em>et al.</em>, 2015), and the role of temperature in Ry<sub>sto</sub>-mediated extreme resistance (Grech-Baran <em>et al.</em>, 2020).<br/><br/>However, the use of PVY infectious clones as a reverse genetics tool for <em>in planta</em> functional analysis of PVY is still scarce because of the low inoculation yields and complex manipulation of some PVY clones. Therefore, we here describe a protocol to functionally characterise the role of the viral CP by using the PVY-N605(123) cDNA clone (Bukovinszki <em>et al.</em>, 2007), a full-length infectious clone that was previously developed to avoid the cytotoxic effects to <em>E. coli</em> through intron insertion. We present a detailed description of the cloning strategy to introduce desired CP deletions into the PVY-N605(123) cDNA clone followed by gene gun bombardment of <em>Nicotiana clevelandii</em> plants. The protocol includes characterisation of the resulting virus amplification and spread using reverse transcription quantitative PCR to quantify viral genome and transmission electron microscopy to confirm virion formation. We recently used this protocol to study the role of viral CP N- and C- terminal regions in virus infectivity (KeÅ¾ar <em>et al.</em>, 2019). The same procedure, with an appropriate modification of the mutagenesis strategy, can also be used to introduce any mutation into PVY cDNA or alternatively to insert fluorescent protein into cDNA and tag specific PVY proteins for purification or imaging purposes. Therefore, the protocol described here offers an indispensable tool for <em>in planta</em> analysis of the viral cycle and functional studies of viral proteins.</p>
        </body>

<p class="pptt" id="biaoti34811">## Materials and Reagents</p><ol class="npwtwok"><li>384-well plates (Thermo Fisher Scientific, Applied Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> 4309849 </a> )</li><li>15 ml polypropylene centrifuge tubes (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 430791 </a> )</li><li>Filter paper for laboratory use (Sartorius)</li><li>1.5 ml microfuge tube (Brand, catalog number: <a class="blackNoUnder" target="_blank"> 780502 </a> )</li><li>10 ml syringe (BD, catalog number: <a class="blackNoUnder" target="_blank"> 309110 </a> )</li><li>0.22 Î¼m disposable filter (Sartorius, catalog number: <a class="blackNoUnder" target="_blank"> 16532-Q </a> )</li><li><em>N. clevelandii</em> plants grown from seeds at 24 Â°C and 60%-70% humidity<br/><em>Note: Seeds were from an in-house collection.</em> </li><li>PVY cDNA clone PVY-N605(123) (Bukovinszki <em>et al.</em>, 2007)<br/><em>Note: The plasmid can be provided by the National Institute of Biology, Slovenia, upon request.</em> </li><li>QuikChange II XL Site-Directed Mutagenesis Kit (Agilent Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 200522 </a> )</li><li>Bacto tryptone (BD, catalog number: <a class="blackNoUnder" target="_blank"> 211705 </a> )</li><li>Bacto yeast extract (BD, catalog number: <a class="blackNoUnder" target="_blank"> 212750 </a> )</li><li>Bacto agar (BD, catalog number: <a class="blackNoUnder" target="_blank"> 214010 </a> ) </li><li>NaCl (Merck, Supelco, catalog number: <a class="blackNoUnder" target="_blank"> 1.06404.1000 </a> )</li><li>KCl (Merck, Supelco, catalog number: <a class="blackNoUnder" target="_blank"> 1.04936.0500 </a> )</li><li>MgCl<sub>2</sub> (Merck, Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M8266-100G </a> )</li><li>Glucose, anhydrous (Kemika, catalog number: 0<a class="blackNoUnder" target="_blank"> 705007 </a> )</li><li>Ampicillin (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A9393-5G </a> )</li><li>KAPA2G Robust HotStart Polymerase with dNTP (Kapa Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> KK5516 </a> )</li><li>GenElute<sup>TM</sup> Plasmid Miniprep Kit (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> PLN350-1KT </a> )</li><li>Monarch Plasmid Miniprep Kit (NEB, catalog number: <a class="blackNoUnder" target="_blank"> T1010 </a> ) </li><li>Agarose (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A9539 </a> )</li><li>Ethidium bromide solution 10 mg/ml (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E1510-10ML </a> )</li><li>Trizma base (Tris base) (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T1503 </a> )</li><li>Acetic acid (glacial) 100% (Merck, Supelco, catalog number: <a class="blackNoUnder" target="_blank"> 1.00063 </a> )</li><li>EDTA (disodium salt, dihydrate) (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> E5134 </a> )</li><li>Fresh 100% ethanol (Ethanol for spectroscopy Uvasol<sup>Â®</sup>, Merck, Sigma-Aldrich, Supelco, catalog number: <a class="blackNoUnder" target="_blank"> 1009800500 </a> )</li><li>0.05 M spermidine (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S0266-1G </a> )</li><li>1 M CaCl<sub>2</sub> (Merck, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C1016-100G </a> )</li><li>MagMAX<sup>TM</sup>-96 Total RNA Isolation Kit (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> AM1830 </a> )</li><li>Plant RNA Isolation Aid (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> AM9690 </a> )</li><li>Lysing Matrix A: tubes containing garnet and one 1/4" ceramic sphere (MP Biomedicals, catalog number: <a class="blackNoUnder" target="_blank"> 116910050-CF </a> )</li><li>OneStep RT-PCR Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 210210 </a> )</li><li>AgPath-ID One-Step RT-PCR Reagents (Thermo Fisher Scientific, Applied Biosystems<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> AM1005 </a> )</li><li>NaH<sub>2</sub>PO<sub>4</sub>Â·2H<sub>2</sub>O (Merck, SAFC, catalog number: <a class="blackNoUnder" target="_blank"> 1.06345.1000 </a> )</li><li>Na<sub>2</sub>HPO<sub>4</sub> (Merck, Supelco, catalog number: <a class="blackNoUnder" target="_blank"> 1.06586.0500 </a> )</li><li>1% (w/v) uranyl acetate in ddH<sub>2</sub>O (SPI Supplies, catalog number: <a class="blackNoUnder" target="_blank"> 6159-44-0 </a> )</li><li>S.O.C. media (see Recipes)</li><li>LB medium liquid (low salt) (see Recipes)</li><li>Antibiotic (see Recipes)</li><li>0.1 M phosphate buffer (see Recipes)</li><li>50x TAE buffer (see Recipes)</li><li>0.5 M EDTA (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34812">### Equipment</p><ol class="npwtwok"><li>Tweezers</li><li>Pipettes</li><li>T100<sup>TM</sup> Thermal Cycler (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1861096 </a> )</li><li>Water bath WB-13 (KambiÄ, catalog number: <a class="blackNoUnder" target="_blank"> 1201 </a> )</li><li>Incubator, orbital shaker SI500 (Stuart)</li><li>Agarose gel electrophoresis (Bio-Rad)</li><li>UVP ChemStudio PLUS (Analytik Jena, catalog number: <a class="blackNoUnder" target="_blank"> 849-97-0847-04 </a> )</li><li>NanoDrop<sup>TM</sup> 1000 Spectrophotometer (Thermo Fisher Scientific, out of production)</li><li>Sanger sequencing (Eurofins Scientific)</li><li>Helios Gene Gun System (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 165-2432 </a> ), includes:<ol class="npwthree"><li>Helium hose assembly with regulator and a tubing prep station (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 165-2420 </a> )</li><li>Syringe kit, Tefzel tubing, tubing cutter, optimisation kit (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 165-2424 </a> , includes 0.6 Î¼m, 1.0 Î¼m, and 1.6 Î¼m gold microcarriers, PVP (Polyvinylpyrrolidone), desiccant, five cartridge storage vials, 50 feet of Tefzel gold-coat tubing, for 1,000 samples)</li></ol></li><li>Helium tank (Messer, grade 6.0)</li><li>Nitrogen tank (Messer, grade 5.0)</li><li>Nitrogen regulator (<em>e.g.</em>, Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 165-2425 </a> )</li><li>Ultrasonic cleaning bath Sonis 3 GT (Iskra Pio)</li><li>FastPrep-24<sup>TM</sup> Classic Instrument (MP Biomedicals, catalog number: <a class="blackNoUnder" target="_blank"> 116004500 </a> )</li><li>KingFisher Instrument (Thermo Scientific, out of production)</li><li>QuantStudio 7 Flex Real-Time PCR System (Thermo Fisher Scientific, Applied Biosystems<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 4485701 </a> )</li><li>Automated liquid handling system (Microlab STARlet, Hamilton)</li><li>Transmission electron microscope (Philips, CM 100) equipped with couple-charged device camera ORIUS SC 200 (Gatan Inc., Pleasanton, USA)<br/></li></ol><p class="pptt" id="biaoti34813">### Software</p><ol class="npwtwok"><li>SnapGene (<a href="https://www.snapgene.com/snapgene-viewer/" target="_blank">https://www.snapgene.com/snapgene-viewer/</a>)</li><li>QuantGenius software (Baebler <em>et al.</em>, 2017, <a href="http://quantgenius.nib.si/user/login" target="_blank">http://quantgenius.nib.si/user/login</a>)</li><li>Digital Micrograph, version 2.11.1404.0 (Gatan Inc., Pleasanton, USA)<br/></li></ol><p class="pptt" id="biaoti34814">### Procedure</p><ol class="npwone"><li>Gene cloning to prepare the mutated PVY cDNA clone<br/>In Procedure A, the mutation is introduced into the desired location of the PVY plasmid, followed by transformation of the mutant plasmid into <em>E. coli</em>, and confirmation of the introduced mutation by colony PCR. In the end, mutated and WT plasmids are multiplied and purified before they are introduced into the plants.<ol class="npwtwo"><li>Introduce mutation into the plasmid (PVY cDNA clone PVY-N605(123), Bukovinszki <em>et al.</em>, 2007) by site-directed mutagenesis using the QuikChange II XL Site-Directed Mutagenesis Kit (Agilent technologies).<br/><ol class="npwthree"><li>Design primers for mutation (<em>e.g.</em>, deletion).<br/><ol class="npwfour"><li>The desired mutation should be in the middle of the primer.</li><li>Minimum GC content should be 40%.</li><li>Terminate in one or more C or G bases.</li><li>The primer should be 40-50 bases long.</li><li>The melting temperature (Tm) of the primer is â¥ 78 Â°C (Tm calculation: 81.5 + 0.41(%GC) â (675/<em>N</em>); <em>N</em> is the primer length in bases and does not include the bases that are being inserted or deleted.</li><li>Purify with HPLC.</li><li>An example of designing a forward primer to introduce a deletion of 60 amino acid residues at the C terminus of PVY CP is provided (Figure 1):<br/>The forward primer (excluding the nucleotides to be deleted) is annealed to the parental plasmid, forming a hairpin in the parental plasmidâs region to be deleted. This results in an amplified fragment that includes the desired deletion after PCR.</li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715001535_2469.jpg" width="300"/><br/><strong>Figure 1. An example of designing a forward primer using SnapGene viewer</strong><br/><br/></li><li>Set up the PCR reaction with the reagents specified in Table 1 and the cycling program shown below.<br/><br/><strong> Table 1. Reaction components for mutagenic PCR</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715001843_9382.jpg" width="450"/><br/><br/>Thermal cycler conditions<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:60%;"><tbody><tr><td>92 Â°C 2 min <br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">92 Â°C 50 s<br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">65 Â°C to 55 Â°C 50 s, touch down <br/></td><td>  10 cycles<br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">68 Â°C 2 min/kb<br/></td><td><br/></td></tr><tr><td><br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">92 Â°C 50 s<br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">55 Â°C 50 s <br/></td><td>  8 cycles<br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">68 Â°C 2 min/kb <br/></td><td><br/></td></tr><tr><td>68 Â°C 60 min<br/></td><td><br/></td></tr></tbody></table><br/></li></ol></li><li>Digest unmodified plasmid with <em>Dpn</em>I enzyme from the QuikChange II XL Site-Directed Mutagenesis Kit. Use 4 Âµl of enzyme per 25 Âµl of reaction, incubate 2 h at 37 Â°C.</li><li>Transform <em>E. coli</em> XL10-Gold ultracompetent cells (included in the QuikChange II XL Site-Directed Mutagenesis Kit):<ol class="npwthree"><li>Gently thaw the XL10-Gold ultracompetent cells on ice. Aliquot 45 Î¼l of the ultracompetent cells to a 2 ml tube for each sample reaction.</li><li>Add 2 Î¼l of the Î²-ME mix (provided with the kit) to the 45 Î¼l of cells.</li><li>Mix gently. Incubate the cells on ice for 10 min, mixing gently every 2 min.</li><li>Transfer 2 Î¼l of <em>Dpn</em>I-treated DNA to each aliquot of the ultracompetent cells.</li><li>Gently mix the transformation reactions and incubate on ice for 30 min.</li><li>Heat shock tubes in a 42 Â°C water bath for 30 s. The duration of the heat shock is critical for obtaining the highest efficiencies.</li><li>Incubate the tubes on ice for 2 min.</li><li>Add 0.5 ml of preheated (42 Â°C) S.O.C. media to each tube.</li><li>Incubate at 37 Â°C for 1 h with shaking at 225-250 rpm.</li><li>Plate the entire volume of each transformation reaction (250 Âµl x 2 plates) on LB agar plates containing the appropriate antibiotic for the plasmid (<em>e.g.</em>, ampicillin).</li><li>Incubate plates overnight at 37 Â°C.</li></ol></li><li>Set up Colony PCR using the KAPA2G Robust HotStart PCR Kit (Kapa Biosystems).<ol class="npwthree"><li>Resuspend five transformed colonies in 50 Âµl sterile ddH<sub>2</sub>O.</li><li>Use 1 Âµl of colony suspension to prepare a colony PCR.<br/><em>Note: Store the remaining colony suspension at 4 Â°C; it will be used to prepare overnight cultures of positive colonies.</em> </li><li>Set up the PCR reaction with the reagents specified in Table 2 and the cycling program shown below.<br/><em>Note: Amplify also non-mutated plasmid (10 ng/Âµl) as a control.</em><br/><br/><strong>Table 2. Reaction components for colony PCR</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715002352_5624.jpg" width="550"/><br/><br/>Thermal cycler conditions<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:60%;"><tbody><tr><td>95 Â°C 5 min <br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">95 Â°C 30 s <br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">Tm Â°C 15 s <br/></td><td>  30 cycles<br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">72 Â°C 1 min/Kb <br/></td><td><br/></td></tr><tr><td>72 Â°C 5 min <br/></td><td><br/></td></tr></tbody></table><br/></li><li>Load 1 Âµl of each PCR product (also non-mutated plasmid) onto 1% agarose gel, made in 1x TAE buffer with ethidium bromide (final concentration 0.5 Âµg/ml).</li><li>For colonies with mutations (observed as a shift on the gel, see Figure 2), add 5 Âµl of colony suspension to 5 ml of LB liquid media with antibiotic (<em>e.g.</em>, ampicillin).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715002548_0212.jpg" width="300"/><br/><strong>Figure 2. Agarose gel showing colonies with deletions.</strong> 1: non-mutated plasmid; 2-8: colonies with deletions; Mw: molecular weight; bp: base pairs.<br/><br/></li><li>Incubate for 16-18 h at 37 Â°C with shaking at 200â250 rpm.<br/></li></ol></li><li>Isolate plasmids using the GenElute<sup>TM</sup> Plasmid Miniprep Kit (Sigma) following <a href="https://www.sigmaaldrich.com/content/dam/sigma-aldrich/docs/Sigma/Bulletin/pln70bul.pdf" target="_blank">manufacturerâs instructions</a> with a few modifications:<ol class="npwthree"><li>Centrifuge 5 ml of overnight culture in a 15 ml Falcon tube for 5 min at 4,500 rpm at room temperature.</li><li>Elution step:<br/>Add 30 Âµl of nuclease-free water to the spin column. Incubate for 10 min at room temperature. For plasmids â¥ 10 kb or â¥ 20 kb, use water preheated to 70 Â°C or 80 Â°C, respectively. Centrifuge at top speed for 2 min at room temperature.</li></ol></li><li>Measure plasmid concentration with the NanoDrop<sup>TM</sup> 1000 Spectrophotometer. The expected yield range is 1.5-4.5 Âµg, and the expected purity ranges for the indicator ratios A<sub>260</sub>/A<sub>230</sub> and A<sub>260</sub>/A<sub>280</sub> are 1.8-2.4 and 1.8-2.1, respectively.</li><li>Send samples for Sanger sequencing to confirm the right sequence (company: Eurofins Scientific). Use a 5' PCR primer upstream of the mutation site and a 3' PCR primer downstream of the mutation site. The obtained sequence should be 100% identical at the amino acid level to the expected sequence including the planned modification. Silent mutations are acceptable.</li><li>After confirming the sequence, prepare 50 Âµg of each plasmid by using the same protocol and combining 5-10 mini-, midi-, or maxi-preps.<br/><em>Note: For plasmid isolation, the Monarch Plasmid Miniprep Kit (NEB, see Quick Protocol Card) can also be used.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Biolistic bombardment using the Helios Gene Gun system (Bio-Rad) to infect plants with the PVY cDNA clones (WT and mutated)<br/>Here, the gold nanoparticles that help transfer the plasmids into the plants are prepared. First, the plasmids are bound to the gold nanoparticles, and then the Gene Gun cartridges are loaded with the plasmid-bound nanogold particles. The cartridges are inserted into the Gene Gun, and the plasmid-bound gold nanoparticles are shot into the plant leaf tissue.<ol class="npwtwo"><li>Precipitation of plasmid DNA onto microcarriers.<ol class="npwthree"><li>Prepare a stock solution of 20 mg/ml PVP in 100% ethanol. Dilute this solution with 100% ethanol to prepare 3 ml of PVP solution with a concentration of 0.05 mg/ml. Keep these solutions tightly capped when not in use. Only use freshly prepared dilutions.</li><li>Weigh gold microcarriers in a 1.5 ml microfuge tube (size of gold particles: 0.6 Âµm, use 6.25 mg/plasmid).</li><li>Add 0.05 M spermidine to the 6.25 mg of gold in the tube. The volume of 0.05 M spermidine should be equal to the volume of 50 Âµg of plasmid.</li><li>Vortex the gold and spermidine mixture for a few seconds, and then sonicate with an ultrasonic cleaning bath for 3-5 s to break up gold clumps.</li><li>Add 50 Âµg of plasmid to the gold and spermidine mixture.</li><li>Mix DNA, spermidine, and gold by vortexing for ~5 s.</li><li>While vortexing the mixture at a moderate rate, add 1 M CaCl<sub>2</sub> dropwise to the mixture. The volume added should be equal to the volume of plasmid. The maximal volume of the entire mixture (gold, spermidine, plasmid, and CaCl<sub>2</sub>) should not exceed 1,200 Âµl.</li><li>Allow the mixture to precipitate at room temperature for 10 min.</li><li>Most of the gold will now be in the pellet, but some may remain on the sides of the tube. Spin (10,000 rpm) the microcarrier solution in a microfuge for ~30 s to pellet the gold. Discard the supernatant.</li><li>Resuspend the pellet in the remaining supernatant by vortexing briefly. Wash the pellet three times with 1 ml of fresh 100% ethanol each time, sonicate if you see aggregates of gold particles (only for a short time to avoid DNA damage); spin for ~30 s in a microfuge between each wash. Discard the supernatant.</li><li>After the final ethanol wash, resuspend the pellet in 200 Âµl of ethanol solution containing 0.05 mg/ml of PVP (prepared in Step B1a). Transfer the suspension to a 15 ml disposable polypropylene centrifuge tube with a screw cap. Rinse the microfuge tube with 200 Âµl of the same ethanol/PVP solution and add to the 15 ml centrifuge tube. Rinse again until all gold is washed from the tube. Add up to 3 ml of ethanol/PVP solution to the 15 ml centrifuge tube.</li></ol></li><li>Preparation of cartridges<ol class="npwthree"><li>Load the DNA/microcarrier suspension into the Gold-Coat tubing using the Tubing Prep Station.<ol class="npwfour"><li>Prior to cartridge preparation, ensure that the Gold-Coat tubing is completely dry by purging with nitrogen. Insert an uncut piece of tubing into the Tubing Prep Station. Connect the Tubing Prep Station to a nitrogen gas tank. Using the knob on the flowmeter of the Tubing Prep Station, turn on the nitrogen and adjust the flow to 0.3-0.4 LPM (liters per minute). Allow nitrogen to flow through the Gold-Coat tubing for at least 15 min immediately prior to using it in the following steps.</li><li>Remove the Gold-Coat tubing from the Tubing Prep Station.</li><li>Turn off the nitrogen flow to the Tubing Prep Station using the knob on the flowmeter.</li><li>Cut a piece of dried Gold-Coat tubing with a length of the Tubing Station plus 10 cm. Insert one end of the Gold-Coat tubing into the end of the adaptor tubing fitted to the 10 ml syringe.</li><li>Vortex the microcarrier suspension and, if necessary, briefly sonicate to achieve an even suspension of gold. Invert the tube several times to resuspend the gold. Immediately remove the cap and quickly draw the gold suspension into the Gold-Coat tubing to cover the tube. Avoid drawing bubbles into the Gold-Coat tubing. Remove the tubing from the suspension and continue drawing the suspension into the tubing for another ~3 cm to leave some space at each end. Cut the part of the tube that was soaked in the liquid to avoid contamination of Tubing Prep Station.</li><li>Immediately bring the Gold-Coat tubing to a horizontal position and slide the loaded tube, with syringe still attached, into the tubing support cylinder in the Tubing Prep Station until the tubing passes through the O-ring.</li><li>Allow the microcarriers to settle for 5 min.</li><li>Remove the ethanol by slowly drawing the syringe plunger outwards at a rate of 1-2 cm/s.</li><li>Detach the syringe from the Gold-Coat tubing and attach a disposable filter (0.2 Âµm) at the end of the tube. Immediately turn the Gold-Coat tubing by 180Â° while in the groove and allow the gold to begin coating the inside surface of the tubing for 3-4 s.</li><li>Start rotating the Tubing Prep Station. Allow the gold to spread within the tube for 20-30 s and then open the valve on the flowmeter to allow 0.35-0.4 LPM of nitrogen to dry the Gold-Coat tubing, while it continues to rotate.</li><li>Continue drying the Gold-Coat tubing while rotating for 5 min.</li><li>Turn off the motor of the Tubing Prep Station. Turn off the nitrogen by closing the valve on the flowmeter.</li><li>Remove the tubing from the tubing support cylinder.</li></ol></li><li>Prepare cartridges using the tubing cutter. Store the cartridges in a cartridge storage vial containing a desiccant pellet.</li></ol></li><li>Bombardment with the Gene Gun.<ol class="npwthree"><li>Experimental set-up:<ol class="npwfour"><li>Use <em>N. clevelandii</em> plants at the 6 to 10 leaf stage.</li><li>Bombard seven plants (five leaves per plant) for one construct.</li><li>Use a yellow fluorescent protein-expressing plasmid or some other plasmid that lacks any PVY-related sequence as a negative bombardment control (to check for any background signal due to PVY carry over during the bombardment procedure).</li><li>The WT virus should always be used last to bombard the leaves (to avoid contamination).</li></ol><span id="__kindeditor_bookmark_start_327__" style="display:none;"></span> </li><li>Bombardment:<br/><em>Notes:</em><br/><em>1)Use a disposable coat and glasses and work in a chamber with UV light.</em><br/><em>2)After bombardment and before proceeding with the next plasmids, carefully clean with DNA remover and turn on the UV light in the chamber.</em><br/><ol class="npwfour"><li>Prepare the Gene Gun: insert the battery, tighten the Barrel Liner on the gun, and insert the Barrel O-ring in the Barrel Liner.</li><li>Connect the gun to the Helium tank (open the first valve fully and set the second valve to a pressure of 200 psi).<br/><em>Note: Check after the first bombardment that the pressure is still 200 psi.</em> </li><li>Fill cartridge holder with cartridges (use tweezers).</li><li>Insert the cartridge holder into the gun (open all locks), set number 12 on the top, and press the Cylinder Advance lever (see Figure 3 for clarification).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715002906_0003.jpg" width="300"/><br/><strong>Figure 3. Components and controls on the Helios Gene Gun (manual for Helios Gene Gun System)</strong><br/><br/></li><li>Touch the leaf with the gun (with the Barrel Liner), hold the Safety Interlock Button, and press the Trigger button to bombard the leaf (see Figure 4). Press the Cylinder Advance lever to switch to a new cartridge.</li><li>Repeat the procedure on five leaves per plant.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715002925_8137.jpg" width="300"/><br/><strong>Figure 4. Bombardment of the plant leaf</strong> </li></ol><br/></li><li>Collect the material in Fast-Prep tubes.<ol class="npwfour"><li>At 7 days post-bombardment, separately collect three bombarded leaves per each plant and immediately freeze them in liquid N<sub>2</sub>.</li><li>Using the same approach, sample systemic infected leaves at 14 days post-bombardment.</li></ol></li></ol><br/></li></ol></li><li>PCR analysis (following amplification of the virus in plants)<br/>Here, the total RNA is purified from plant leaves bombarded with mutant PVY clones, and the presence of introduced mutations in the viral progeny is confirmed by classic RT-PCR and agarose gel electrophoresis. Then, the viral load in plants bombarded with WT and mutated virus clones is relatively quantified using a PVY universal RT-qPCR assay.<br/><ol class="npwtwo"><li>RNA isolation<ol class="npwthree"><li>Use the MagMAX-96 Total RNA Isolation kit (Thermo Fisher Scientific). The RNA can be isolated manually using a magnetic stand and orbital shaker for 96-well plates or automatically using the KingFisher instrument (Thermo Fisher Scientific).</li><li>Follow <a href="https://www.thermofisher.com/document-connect/document-connect.html?url=https%3A%2F%2Fassets.thermofisher.com%2FTFS-Assets%2FLSG%2Fmanuals%2Fcms_078409.pdf&amp;title=VXNlciBHdWlkZTogTWFnTUFYLTk2IFRvdGFsIFJOQSBJc29sYXRpb24gS2l0IEluc3RydWN0aW9uIE1hbnVhbCAoRW5nbGlzaCAp" target="_blank">manufacturerâs instructions</a> with modifications regarding homogenate preparation: Add 450 Âµl of Lysis Binding solution concentrate and 50 Âµl of Plant RNA Isolation Aid to each frozen leaf and immediately start homogenisation with the FastPrep instrument (6 m/s, 60 s). After complete homogenisation, centrifuge samples at 14,000 <em>x g</em> for 10 min. Discard the pellet. Transfer 50 Î¼l of homogenised sample to the wells of the processing plate and follow the manufacturerâs protocol.</li><li>Include a negative control for the extraction procedure.</li><li>The optimal RNA yield range is 1-9 Âµg, and the optimal purity ranges for the indicator ratios A<sub>260/280</sub> and A<sub>260/230</sub> are 1.7-2.2 and 1.0-2.1, respectively. In our experience, values that are not within these ranges also yield optimal qPCR results.</li></ol></li><li>Set up classic one-step RT-PCR to confirm that the introduced mutation is maintained in the viral progeny <em>in planta</em> using the Qiagen OneStep RT-PCR kit. Use reagents specified in Table 3 and the cycling program shown below. Load the PCR product on 1% agarose gel made in 1x TAE buffer with ethidium bromide (final concentration 0.5 Âµg/ml). Inspect the shift on the agarose gel to confirm the mutation.<br/><br/><strong>Table 3. Reaction components for One-Step RT-PCR</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715003451_4226.jpg" width="550"/><br/><br/>Thermal cycler conditions<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:60%;"><tbody><tr><td>50 Â°C 30 min<br/></td><td><br/></td></tr><tr><td>95 Â°C 15 min <br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">94 Â°C 30 s<br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">Tm Â°C 30 s<br/></td><td>  35 cycles<br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">72 Â°C 1 min/Kb<br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">72 Â°C 10 min<br/></td><td><span style="color:#002000;font-family:Arial,Helvetica,sans-serif;"></span><br/></td></tr></tbody></table><br/></li><li>Set up the qPCR reaction (single-step quantitative RT-PCR) using the AgPath-IDTM One-Step RT-PCR kit (Thermo Fisher Scientific) for virus detection.<ol class="npwthree"><li>Quantify the PVY-N605 virus.<ol class="npwfour"><li>Use the amplicon targeting the CP to detect the PVY clone (PVY) and cytochrome oxidase transcript (COX) as the endogenous control (see primers and probes in Table 4).<br/><em>Note: Ensure that the amplicon for the PVY clone is not at the position of the introduced mutation (e.g., deletion). This amplicon quantifies WT virus and mutants and enables their comparison.</em><br/><br/><strong>Table 4. Primers and probes for the COX and PVY amplicons</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715003636_8644.jpg" width="600"/><br/><br/></li><li>Set up a One-Step quantitative RT-PCR with the reagents shown in Table 5 and cycling conditions below.<br/><br/><strong>Table 5. Reaction components for One-Step quantitative RT-PCR</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715003709_6270.jpg" width="550"/><br/><br/>Thermal cycler conditions<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:60%;"><tbody><tr><td>48 Â°C 10 min<br/></td><td><br/></td></tr><tr><td>95 Â°C 10 min <br/></td><td><br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">95 Â°C 15 s<br/></td><td>  40 cycles<br/></td></tr><tr><td style="border-top:0pt;border-left:0;border-bottom:0px;border-right:1px;border-style:solid;">60 Â°C 1 min <br/></td><td><span style="color:#002000;font-family:Arial,Helvetica,sans-serif;"></span><br/></td></tr></tbody></table><br/></li></ol></li><li>Prepare a standard curve: combine 2 Âµl of each sample. Dilute 5x, 25x, 125x, and 625x.</li><li>Include a non-template control.</li><li>If an automated liquid handling station will be used to pipet onto 384-well plates, two technical replicates are sufficient, otherwise, three are recommended.</li><li>RT-qPCR can detect both viral RNA and residual plasmid DNA. To quantify only viral RNA, the contribution of the plasmid-derived signal to the concentration of PVY RNA must be eliminated. To detect any plasmid-derived signals, a qPCR reaction without RT is run for each sample. For this, denature the RT enzyme from the AgPath-ID One-Step RT-PCR kit by heating the 25x RT-PCR Enzyme Mix at 95 Â°C for 10 min before the qPCR reaction. To calculate the quantity of viral RNA, the plasmid-derived signal is subtracted from the overall signal (viral RNA + plasmid DNA) (see Data analysis, step 2).</li><li>Calculate relative copy numbers of PVY RNA using the quantGenius software (<a href="http://quantgenius.nib.si/user/login" target="_blank">http://quantgenius.nib.si/user/login</a>, Baebler <em>et al.</em>, 2017), see Data analysis.</li></ol></li></ol><br/></li><li>Sample observation with transmission electron microscopy and image acquisition<br/>Here, extracts from plant leaves bombarded with WT and mutated virus clones are observed by electron microscopy to visualise virion formation.<ol class="npwtwo"><li>Cut 3-4 pieces (of 1-2 mm<sup>2</sup>) of the lower leaves of <em>N. clevelandii</em> 7 days post-bombardment, place them into Eppendorf tubes, and macerate them with 300 Âµl of 0.1 M phosphate buffer. </li><li>Place a formvar- and carbon-coated grid on a drop of sample for 5 min.</li><li>Soak away the sample by carefully touching the edge of the grid with filter paper and wash the grid with 5-10 drops of deionised water.</li><li>Stain the sample with one drop of 1% (w/v) uranyl acetate.</li><li>Immediately remove excess uranyl acetate by touching the edge of the grid with filter paper.</li><li>Dry the sample at room temperature.</li><li>Observe the grids with a transmission electron microscope and acquire the images with a couple-charged device camera (Figure 5).</li><li>Use Digital Micrograph software to process the images.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200715/20200715003914_4837.jpg" width="300"/><br/><strong>Figure 5. Negative-stain transmission electron microscopy of clone PVY-N605(123) in bombarded leaves.</strong> PVY-N605(123) flexuous virions are indicated by arrows.<br/></li></ol></li></ol><p class="pptt" id="biaoti34815">### Data analysis</p><ol class="npwtwok"><li>Check for any background signal due to PVY carry over during the bombardment procedure (viral RNA from plants with plasmid expressing yellow fluorescent protein). All samples with Cq (cycle quantification) values above the background (<em>e.g.</em>, Cq = 32) should be considered negative.</li><li>Calculate relative copy numbers of PVY RNA using quantGenius software (<a href="http://quantgenius.nib.si/user/login" target="_blank">http://quantgenius.nib.si/user/login</a>, Baebler <em>et al.</em>, 2017). With this software, relative gene quantification is calculated using the standard curve. Enter Cq-s for the reference gene (COX) and set extraction quality control to 28. A Cq value of &lt; 28 indicates efficient RNA extraction. Then enter Cq-s for the target gene (PVY-N605 CP). The final outputs from the software are quality-controlled relative copy numbers for each sample. Subtract the copy numbers calculated from qPCR without RT (presence of plasmid DNA in the sample) from the copy numbers calculated from RT-qPCR. The final results are relative copy numbers of viral RNA.</li><li>Perform one-way analysis of variance with Bonferroni post-hoc analysis to statistically evaluate the differences in virus accumulation between the PVY clones. P-values of less than the selected value indicate significant differences among the groups. Post-hoc analysis reveals which PVY mutants accumulate differently than the WT PVY. For each group, include at least 20 samples (<em>i.e.</em>, seven plants and three leaves per plant) and perform the complete experiment at least twice.<br/></li></ol><p class="pptt" id="biaoti34816">### Recipes</p><ol class="npwtwok"><li>S.O.C. Medium<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td><strong>Component</strong><br/></td><td><strong>amount/L</strong><br/></td></tr><tr><td>Bacto Tryptone<br/></td><td>20 g<br/></td></tr><tr><td>Yeast extract<br/></td><td>5 g<br/></td></tr><tr><td>NaCl<br/></td><td>580 mg<br/></td></tr><tr><td>KCl<br/></td><td>186 mg<br/></td></tr><tr><td>MgCl<sub>2</sub><br/></td><td>952 mg<br/></td></tr><tr><td>Glucose<br/></td><td>3.603 g<br/></td></tr></tbody></table>Set pH to 7.0<br/>Autoclave 15 min at 121 Â°C</li><li>LB medium liquid (low salt)<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td><strong>Component</strong><strong></strong><br/></td><td><strong>amount/L</strong><br/></td></tr><tr><td>Bacto Tryptone<br/></td><td>10 g<br/></td></tr><tr><td>Yeast extract<br/></td><td>5 g<br/></td></tr><tr><td>NaCl <br/></td><td>5 g<br/></td></tr></tbody></table>Set pH to 7.5<br/>LB agar (1 L): Add 15 g of Bacto agar to LB medium liquid<br/>Autoclave 15 min at 121 Â°C</li><li>Antibiotic<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:90%;"><tbody><tr><td><strong>Antibiotic</strong><strong></strong><br/></td><td><strong>Final concentration (Âµg/ml)</strong><br/></td></tr><tr><td>Ampicillin<br/></td><td>100<br/></td></tr></tbody></table>Filter sterilise using a 0.22 Î¼m disposable filter</li><li>0.1 M phosphate buffer<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:55%;"><tbody><tr><td><strong>Component </strong><br/></td><td><strong>amount/100 ml </strong><br/></td></tr><tr><td>NaH<sub>2</sub>PO<sub>4</sub>Â·2H<sub>2</sub>O<br/></td><td>6.24 g<br/></td></tr><tr><td>Na<sub>2</sub>HPO<sub>4</sub><br/></td><td>7.50 g<br/></td></tr></tbody></table>Set pH to 6.8</li><li>50x TAE buffer<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td><strong>Component</strong><strong></strong><br/></td><td><strong>amount/L</strong><br/></td></tr><tr><td>Trizma base<br/></td><td>242 g <br/></td></tr><tr><td>Acetic acid<br/></td><td>57.1 ml<br/></td></tr><tr><td>0.5 M EDTA<br/></td><td>100.0 ml<br/></td></tr></tbody></table>Add 100 ml of 50x TAE buffer to 4900 ml of dH<sub>2</sub>O to obtain 1x TAE buffer</li><li>0.5 M EDTA<br/><table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:60%;"><tbody><tr><td><strong>Component</strong><br/></td><td><strong>amount/L</strong><br/></td></tr><tr><td>EDTA disodium salt, dihydrate<br/></td><td>186.1 g<br/></td></tr></tbody></table>Set pH to 8.0<br/></li></ol><p class="pptt" id="biaoti34817">### Acknowledgments</p><p class="ppzz">This work was supported by the Slovenian Research Agency (P1-0391, J7-7248, P4-0165, P2-0145, and P1-0010, PhD grants for A.K.).<br/>  We thank all the authors of the research paper KeÅ¾ar <em>et al.</em> (2019) and J. LenarÄiÄ, M. Srnko, L. Mourrain, M. Kisovec, A. Blejec, B. DuÅ¡ak, and R. Vollmeier for support, and F. Cillo for the PVY clone. The protocol for biolistic bombardment was adapted and optimised from Dobnik <em>et al.</em> (2013).</p><p class="pptt" id="biaoti34818">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest within the work.</p><p class="pptt" id="biaoti34819">### References</p><ol class="npwtwok"><li>Baebler, Å ., Svalina, M., Petek, M., Stare, K., Rotter, A., Pompe-Novak, M. and Gruden, K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28545393" target="_blank">quantGenius: implementation of a decision support system for qPCR-based gene quantification.</a> <em>BMC Bioinformatics</em> 18(1): 276.</li><li>Bukovinszki, Ã., GÃ¶tz, R., Johansen, E., Maiss, E. and BalÃ¡zs, E. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17482305" target="_blank">The role of the coat protein region in symptom formation on <em>Physalis floridana</em> varies between PVY strains.</a> <em>Virus Res </em>127(1): 122-125.</li><li>Chung, B. Y., Miller, W. A., Atkins, J. F. and Firth, A. E. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18408156" target="_blank">An overlapping essential gene in the Potyviridae.</a> <em>Proc Natl Acad Sci U S A</em> 105(15): 5897-5902.</li><li>Dobnik, D., Baebler, Å ., KogovÅ¡ek, P., Pompe-Novak, M., Å tebih, D., Panter, G., JaneÅ¾, N., Morisset, D., Å¾el, J. and Gruden, K. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24273610" target="_blank">Î²-1,3-glucanase class III promotes spread of PVY(NTN) and improves <em>in planta</em> protein production.</a> <em>Plant Biotechnol Rep</em> 7: 547-555.</li><li>Fakhfakh, H., Vilaine, F., Makni, M. and Robaglia, C. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8601790" target="_blank">Cell-free cloning and biolistic inoculation of an infectious cDNA of Potato virus Y.</a> <em>J Gen Virol</em> 77 (Pt 3): 519-523.</li><li>Grech-Baran, M., Witek, K., Szajko, K., Witek, A. I., Morgiewicz, K., Wasilewicz-Flis, I., Jakuczun, H., Marczewski, W., Jones, J. D. G. and Hennig, J. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31397954" target="_blank">Extreme resistance to Potato virus Y in potato carrying the Ry<sub>sto</sub> gene is mediated by a TIR-NLR immune receptor.</a> <em>Plant Biotechnol J</em>.</li><li>Jakab, G., Droz, E., Brigneti, G., Baulcombe, D. and MalnoÃ«, P. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9400962" target="_blank">Infectious <em>in vivo</em> and <em>in vitro</em> transcripts from a full-length cDNA clone of PVY-N605, a Swiss necrotic isolate of Potato virus Y.</a> <em>J Gen Virol</em> 78 (Pt 12): 3141-3145.</li><li>Kerlan, C. (2006). <a href="http://www.dpvweb.net/dpv/showdpv.php?dpvno=414" target="_blank"><em>Potato virus Y</em>.</a> Descriptions of plant viruses. <em>Am Appl Biol</em>.</li><li>KeÅ¾ar, A., KavÄiÄ, L., PolÃ¡k, M., NovÃ¡Äek, J., GutiÃ©rrez-Aguirre, I., Å½nidariÄ, M. T., Coll, A., Stare, K., Gruden, K., Ravnikar, M., Pahovnik, D., Å½agar, E., Merzel, F., Anderluh, G. and Podobnik, M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31328164" target="_blank">Structural basis for the multitasking nature of the Potato virus Y coat protein.</a> <em>Sci Adv</em> 5(7): eaaw3808.</li><li>KogovÅ¡ek, P., Gow, L., Pompe-Novak, M., Gruden, K., Foster, G. D., Boonham, N. and Ravnikar, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18328576" target="_blank">Single-step RT real-time PCR for sensitive detection and discrimination of Potato virus Y isolates.</a> <em>J Virol Methods</em> 149(1): 1-11.</li><li>KriÅ¾nik, M., Petek, M., Dobnik, D., RamÅ¡ak, Å½., Baebler, Å ., Pollmann, S., Kreuze, J. F., Å½el, J. and Gruden, K. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29312421" target="_blank">Salicylic acid perturbs sRNA-gibberellin regulatory network in immune response of potato to Potato virus Y infection.</a> <em>Front Plant Sci</em> 8: 2192. </li><li>Lacomme, C. and Jacquot, E. (2017). <a href="https://link.springer.com/chapter/10.1007/978-3-319-58860-5_1" target="_blank">General characteristics of Potato virus Y (PVY) and its impact on potato production: An Overview.</a> In: Lacomme, C., Glais, L., Bellstedt, D., Dupuis, B., Karasev, A., Jacquot, E. (Eds.). <em>Potato virus Y: biodiversity, pathogen</em><em>icity, epidemiology and management.</em> Springer, Cham, pp 1-19.</li><li>Lukan, T., Baebler, Å ., Pompe-Novak, M., GuÄek, K., ZagorÅ¡Äak, M., Coll, A. and Gruden, K. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29497431" target="_blank">Cell death is not sufficient for the restriction of Potato Virus Y spread in hypersensitive response-conferred resistance in potato.</a> <em>Front Plant Sci</em> 9: 168.</li><li>MÃ¤kinen, K. and HafrÃ©n, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24723931" target="_blank">Intracellular coordination of potyviral RNA functions in infection.</a> <em>Front Plant Sci</em> 5: 110.</li><li>MÃ¤kinen, K., LÃµhmus, A. and Pollari, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29312371" target="_blank">Plant RNA regulatory network and RNA granules in virus infection.</a> <em>Front Plant Sci</em> 8: 2093.</li><li>Rupar, M., Florence, F., Michel, T., GutiÃ©rrez-Aguirre, I., AgnÃ¨s, D., Laurent, G., Maja, K., David, D., Kristina, G., Emmanuel, J., Ravnikar M. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25761209" target="_blank">Fluorescently tagged Potato virus Y: a versatile tool for functional analysis of plant-virus interactions.</a> <em>Mol Plant Microbe Interact</em> 28(7): 739-750.</li><li>Stare, T., RamÅ¡ak, Å½., KriÅ¾nik, M. and Gruden, K. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31673114" target="_blank">Multiomics analysis of tolerant interaction of potato with Potato virus Y.</a> <em>Sci Data</em> 6(1): 250.</li><li>Tribodet, M., Glais, L., Kerlan, C. and Jacquot, E. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15958688" target="_blank">Characterization of Potato virus Y (PVY) molecular determinants involved in the vein necrosis symptom induced by PVYN isolates in infected <em>Nicotiana tabacum</em> <em>cv. Xanthi</em><em>.</em></a> <em>J Gen Virol</em> 86(Pt 7): 2101-2105.</li><li>Valkonen, J. (2007). <a href="https://researchportal.helsinki.fi/en/publications/potato-viruses-economical-losses-and-biotechnological-potential" target="_blank">Potato viruses: economical losses and biotechnological potential.</a> Vreugdenhil, D., Bradshaw, J., Gebhardt, C., Govers, F., MacKerron, D. K. L., Taylor, M. A. and Ross H. A. (Eds.). In: <em>Potato Biology and Biotechnology: Advances and Perspectives</em> (pp. 619-641). Elsevier Scientific Publ. Co.</li><li>Weller, S. A., Elphinstone, J. G., Smith, N. C., Boonham, N. and Stead, D. E. (2000). <a href="http://www.ncbi.nlm.nih.gov/pubmed/10877778" target="_blank">Detection of Ralstonia solanacearum strains with a quantitative, multiplex, real-time, fluorogenic PCR (TaqMan) assay.</a> <em>Appl Environ Microbiol</em> 66(7): 2853-2858.<br/></li></ol>