<body>
            <h1> Metabolomic and Lipidomic Analysis of Bone Marrow Derived Macrophages</h1>
            <p><strong>Authors:</strong> Gretchen L. Seim, Steven V. John, Jing Fan</p>
            <h2>Abstract</h2>
            <p>Macrophages are highly plastic immune cells that are capable of adopting a wide array of functional phenotypes in response to environmental stimuli. The changes in macrophage function are often supported and regulated by changes in cellular metabolism. Capturing a comprehensive picture of metabolism is vital for understanding the role of metabolic rewiring in the immune response. Here we present a method for systematically quantifying the abundance of metabolites and lipids in primary murine bone marrow derived macrophages (BMDMs). This method simultaneously extracts polar metabolites and lipids from BMDMs using a rapid two-phase extraction procedure. The polar metabolite fraction and lipid fraction are subsequently analyzed by separate liquid chromatography-mass spectrometry (LC-MS) methods for optimized coverage and quantification. This allows for a comprehensive characterization of cellular metabolism that can be used to understand the impact of a variety of environmental stimuli on macrophage metabolism and function.</p>
            <p><strong>Keywords:</strong> LC-MS, Metabolomics, Lipidomics, Macrophage, Metabolism</p>
            <h2>Background</h2><p>Macrophages, cells of the innate immune system, can adopt a multitude of functional phenotypes in response to cues within the local microenvironment. The activation of macrophages is coupled to, and highly reliant on, specific reprograming of cellular metabolism (Tannahill <em>et al.</em>, 2013; GalvÃ¡n-PeÃ±a and OâNeill, 2014; Jha <em>et al.</em>, 2015; Kelly and O'Neill, 2015; Cordes <em>et al.</em>, 2016; Mills and O'Neill 2016; <em>et al.</em>; Mills <em>et al.</em>, 2016; Liu <em>et al.</em>, 2013; Lampropoulou <em>et al.</em>, 2016; Van den Bossche <em>et al.</em>, 2017; Williams <em>et al.</em>, 2018; Martin <em>et al.</em>, 2017). The metabolic reprogramming is wide-spread, involving changes in central metabolism, amino acid metabolism, and lipid remodeling (GalvÃ¡n-PeÃ±a and OâNeill, 2014; OâNeill and Pearce, 2016; Van den Bossche <em>et al.</em>, 2017). These changes in different pathways are interdependent, allowing cells to produce energy, signaling molecules (<em>e.g.</em>, eiconsanoids), and effector molecules (<em>e.g.</em>, nitric oxide and reactive oxygen species) for immune functions. Systematic profiling of metabolites and lipids is an important tool to further understand the role these biomolecules play in macrophages.<br/><br/>Development of metabolomic and lipidomic methods has enabled reliable quantification of hundreds of metabolites and lipids simultaneously (Fiehn, 2002; Wenk, 2005). This protocol presents a method for isolation of murine bone marrow derived macrophages (BMDMs), extraction of their lipids and small molecule metabolites and subsequent quantification of their abundance using liquid chromatographyâmass spectrometry (LC-MS). BMDMs are a frequently used model for understanding macrophage function and metabolism. Adaptation of the âBUME (butanol and methanol)â extraction method (LÃ¶fgren <em>et al.</em>, 2012 and 2016), allows for simple and quick extraction of a wide range of both polar metabolites and lipids from a single sample of BMDMs. This reduces variability and gives good recovery. The cell extracts are analyzed by LC-MS, which provides the sensitivity and selectivity required for metabolomic and lipidomic analyses (Theodoridis <em>et al.</em>, 2012; Gika <em>et al.</em>, 2014). Utilization of parallel LC-MS methods for separate analysis of metabolites and lipids, as outlined in this protocol, provides the broad coverage of the metabolome and lipidome. The LC-MS data are then analyzed using previously developed and publically available analysis tools, MAVEN and LipiDex, giving confident identification and quantification of a wide range metabolite and lipid species (Melamud <em>et al.</em>, 2010; Clasquin <em>et al.</em>, 2012; Hutchins <em>et al.</em>, 2018).<br/><br/>This method can be used to characterize how metabolism in macrophages is altered in response to different stimuli, and how it may be regulated by other microenviromental factors (<em>e.g.</em>, nutrient availability). This LC-MS based metabolomics and lipidomics method can be further coupled with isotopic tracing approaches to determine the changes in metabolic flux during immune response, elucidate the mechanisms controlling metabolic rewiring, and facilitate the investigation of the mechanisms connecting altered metabolism to broader macrophage functions.</p>
        </body>

<p class="pptt" id="biaoti34822">## Materials and Reagents</p><ol class="npwone"><li>Isolation and culture of BMDM<br/><ol class="npwtwo"><li>27Â½ G needle (BD Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> 305109 </a> )</li><li>10 ml plastic disposable syringe (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S7515-10 </a> )</li><li>70 Î¼m cell strainer (Thermo Fisher Scientific, catalog number: 0<a class="blackNoUnder" target="_blank"> 87712 </a> )</li><li>0.45 Î¼m PES filter unit (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 162-0045 </a> )</li><li>50 ml conical tubes (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 339652 </a> )</li><li>Countess cell counting chambers (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> C10283 </a> )</li><li>Petri dishes (Fisherbrand, catalog number: <a class="blackNoUnder" target="_blank"> FB0875712 </a> )</li><li>6-well culture plates (Eppendorf, catalog number: 00<a class="blackNoUnder" target="_blank"> 30720113 </a> )</li><li>175 cm<sup>2</sup> flasks (Eppendorf, catalog number: 00<a class="blackNoUnder" target="_blank"> 30710029 </a> )</li><li>Sterile top filters (Nalgene, catalog number: <a class="blackNoUnder" target="_blank"> 596-3320 </a> )</li><li>Cell scraper (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 3008 </a> )</li><li>6+ weeks old mouse (C57BL/6J, Jackson Laboratory)*</li><li> L929 cells (ATCC<sup>Â®</sup>, catalog number: CCL-1<sup>TM</sup>)</li><li>0.4% Trypan blue stain (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 15250-061 </a> )</li><li>70% Ethanol (Pharmco, catalog number: <a class="blackNoUnder" target="_blank"> 1110002000 </a> )</li><li>DMEM High Glucose Medium (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D1152 </a> )</li><li>Penicillin Streptomycin (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15-140-122 </a> )</li><li>Fetal Bovine Serum (FBS, Hyclone, catalog number: <a class="blackNoUnder" target="_blank"> 89133-098 </a> )</li><li>Dialyzed Fetal Bovine Serum (dFBS, Hyclone, catalog number: <a class="blackNoUnder" target="_blank"> 16777-212 </a> )</li><li>Recombinant mouse macrophage colony stimulating factor (mCSF, R&amp;D Systems, catalog number: <a class="blackNoUnder" target="_blank"> 416ML050 </a> )</li><li>RPMI 1640 Medium without Glutamine (Hyclone, catalog number: <a class="blackNoUnder" target="_blank"> SH30096.01 </a> )</li><li>Glutamine (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP379-100 </a> )</li><li>HEPES (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 16777-032 </a> )</li><li>Lipopoylsaccharide from <em>E. coli</em> O111:B4 (LPS, Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> L3024 </a> )**</li><li>Interferon-Î³ (IFNÎ³, R&amp;D Systems, catalog number: <a class="blackNoUnder" target="_blank"> 485-MI-100 </a> )**</li><li>Accutase (STEMCell Technologies, catalog number: 0<a class="blackNoUnder" target="_blank"> 7922 </a> )</li><li>Trypsin-EDTA (0.05%) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 25300062 </a> )</li><li>Phosphate buffered saline (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14190144 </a> )</li><li>L929 conditioned media (see Recipes)</li><li>BMDM maintenance media (see Recipes)</li><li>BMDM differentiation media (see Recipes)</li><li>L929 culture media (see Recipes)</li></ol><em>Notes:</em><em></em> <ol class="npwthree"><li><em>*Any mouse model with sufficiently healthy bone marrow can be used.</em> </li><li><em>**Optional based on desired stimulation protocol. LPS and IFN-Î³ stimulation will induce classical activation.</em> </li></ol><br/></li><li>Metabolite and Lipid Extraction and Sample Preparation<ol class="npwtwo"><li>Pipette tips 1,000 Î¼l, 200 Î¼l (VWR, catalog numbers: <a class="blackNoUnder" target="_blank"> 89079-470 </a> , 89079-478)</li><li>1.5 ml plastic tube (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89000-028 </a> )</li><li>Glass bottles (used for LC-MS grade solvents only, see Notes)</li><li>LC-MS polypropylene vials (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 03-377-299 </a> )</li><li>LC-MS glass vials (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 03-FIRVA </a> )</li><li>Snap caps (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> C4011-53 </a> )</li><li>Polypropylene microcentrifuge tubes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89000-028 </a> )</li><li>Glass vials (Wheaton, catalog number: <a class="blackNoUnder" target="_blank"> 224740 </a> )</li><li>Nitrogen gas</li><li>1-Butanol LC-MS grade (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 34867 </a> )</li><li>Methanol LC-MS grade (Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> A456-4 </a> )</li><li>n-Heptane LC-MS grade (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 1036541000 </a> )</li><li>Ethyl acetate LC-MS grade (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> E195-4 </a> )</li><li>Acetic acid LC-MS grade (Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> A11350 </a> )</li><li>Water LC-MS grade (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> W64 </a> )</li><li>Tributylamine (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 90780 </a> )</li><li>Ammonium Acetate LC-MS grade (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A114-50 </a> )</li><li>Acetonitrile LC-MS grade (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A955-4 </a> )</li><li>Isopropanol (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A461-4 </a> )</li><li>SPLASH LipidoMIX Internal Standard (Avanti Polar Lipids, catalog number: <a class="blackNoUnder" target="_blank"> 330707 </a> )</li><li>Ice<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>LC-MS Analysis<ol class="npwtwo"><li>Acquity UPLC BEH C18 Column, 130Ã, 1.7 Âµm, 2.1 mm x 100 mm (Waters, catalog number: <a class="blackNoUnder" target="_blank"> 186002352 </a> )</li><li>Acquity UPLC BEH C18 VanGuard Pre-column, 130Ã, 1.7 Âµm, 2.1 mm x 5 mm (Waters, catalog number: <a class="blackNoUnder" target="_blank"> 186003975 </a> )</li><li>Acquity UPLC CSH C18 Column, 130Ã, 1.7 Âµm, 2.1 mm x 100 mm (Waters, catalog number: <a class="blackNoUnder" target="_blank"> 186005297 </a> )</li><li>Acquity UPLC CSH C18 VanGuard Pre-column, 130Ã, 1.7 Âµm, 2.1 mm x 5 mm (Waters, catalog number: <a class="blackNoUnder" target="_blank"> 186005303 </a> )</li><li>Solvent B (100% LC-MS grade methanol)<br/></li><li>Solvent A (see Recipes)</li><li>Solvent C (see Recipes)</li><li>Solvent D (see Recipes)</li></ol></li></ol><p class="pptt" id="biaoti34823">### Equipment</p><ol class="npwtwok"><li>Surgery Scissors (Fisherbrand, catalog number: <a class="blackNoUnder" target="_blank"> 08-935 </a> )</li><li>Surgery Forceps (BrainTree Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FC03-41 </a> )</li><li>Thermo Q-Exactive Quadrupole Orbitrap Mass Spectrometer (Thermo Scientific) coupled to a Vanquish Horizon UHPLC (Thermo Scientific)</li><li>Nitrogen Stream Sample Concentrator (Techne, catalog number: <a class="blackNoUnder" target="_blank"> FSC400D </a> ), equipped with 127 mm needles (Techne, catalog number: <a class="blackNoUnder" target="_blank"> F7210 </a> )</li><li>Vortexer (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 10153-838 </a> )</li><li>Micro Centrifuge (Beckman Coulter, model: <a class="blackNoUnder" target="_blank"> Microfuge 20 </a> , catalog number: <a class="blackNoUnder" target="_blank"> B31599 </a> )</li><li>Benchtop Centrifuge (Thermo, model: <a class="blackNoUnder" target="_blank"> IEC Centra CL2 </a> )</li><li>-20 Â°C Freezer</li><li>4 Â°C Fridge</li><li>Standard Fume Hood</li><li>Pipettes (P1000, P200) </li><li>Sterile Cell Culture Hood</li><li>Cell Incubator (37 Â°C, 5% CO<sub>2</sub>)</li><li>Countess II Cell Counter (AMQAX1000)<br/></li></ol><p class="pptt" id="biaoti34824">### Software</p><ol class="npwtwok"><li>Thermo Scientific XCalibur 4.1</li><li>MAVEN (v. 682) (<a href="http://genomics-pubs.princeton.edu/mzroll/index.php)" target="_blank">http://genomics-pubs.princeton.edu/mzroll/index.php</a>) (Melamud <em>et al.</em>, 2010; Clasquin <em>et al.</em>, 2012)</li><li>MZMine2 (<a href="http://mzmine.github.io/" target="_blank">http://mzmine.github.io/</a>) (Pluskal <em>et al.</em>, 2010)</li><li>MSConvertGUI (ProteoWizard, <a href="http://proteowizard.sourceforge.net/download.html" target="_blank">http://proteowizard.sourceforge.net/download.html</a>) (Kessner <em>et al.</em>, 2008)</li><li>LipiDex (<a href="https://github.com/coongroup/LipiDex" target="_blank">https://github.com/coongroup/LipiDex</a>) (Hutchins <em>et al.</em>, 2018)<br/></li></ol><p class="pptt" id="biaoti34825">### Procedure</p><ol class="npwone"><li>Isolation of Bone Marrow Cells and Differentiation into Macrophages (Figure 1)<br/>See additional resource for helpful demonstration of bone marrow isolation (Ying <em>et al.</em>, 2013).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200714/20200714233625_5734.jpg" width="300"/><br/><strong>Figure 1. Timing of BMDM Differentiation Protocol.</strong> TC treated: tissue culture treated; BMDM: bone marrow derived macrophages.<br/><br/><ol class="npwtwo"><li>Euthanize mice by rapid cervical dislocation.</li><li>Thoroughly spray mouse with 70% ethanol and bring into sterile hood.</li><li>Using aseptic technique, isolate femur and tibia from each leg of mouse.</li><li>Using a sharp scissors soaked in ethanol, sever bone proximal to each joint.</li><li>Using a 27Â½ G needle attached to a 10 ml syringe, flush bones with cold, sterile BMDM Differentiation Media into a 10 cm dish, until bone cavity appears white. Use as much media is required to fully flush the bone.</li><li>Pipette cell suspension thoroughly to fully resuspend bone marrow cells.</li><li>For removal of any muscle or bone fragments, filter cells through 70 Âµm filter, into a sterile 50 ml conical centrifuge tube.</li><li>Centrifuge cells for 3 min at 500 <em>x g</em>, 4 Â°C.</li><li>Discard supernatant and resuspend cell pellet in 100 ml of BMDM Differentiation Media.</li><li>Plate 10 ml of cell suspension onto each Petri-dish (non tissue culture treated plates are important here as macrophages will attach too strongly to treated plates)</li><li>Place in cell culture incubator for 3 days.</li><li>Four days post plating aspirate media from cells and replace with 10 ml fresh BMDM Differentiation Media. Repeat on day 5 and day 6.</li><li>Seven days post plating aspirate media from cells and add 4 ml of Accutase per plate. Place back in incubator for approximately 5 min.</li><li>Using pipette, gently dislodge cells from plate and combine all cells in a 50 ml conical tube.</li><li>Spin down (1,000 <em>x g</em>, 5 min) and resuspend into 20 ml BMDM Maintenance media.</li><li>Count cells by staining 1:1 with 0.4% Trypan Blue.</li><li>Resuspend cells to 2.5 x 10<sup>5</sup> cells per ml in BMDM Maintenance Media and plate on TC treated cell culture plates (2 ml per well of 6-well plate). Three replicates (<em>e.g.</em>, 3 wells of a 6-well plate) are recommended for each condition. Additional wells should be designated for protein or cell count normalization.</li><li>Cell will adhere to plate with in a matter of a couple hours. After adherence, cells are ready for stimulation and/or extraction. To stimulate macrophages, treat with 50 ng/ml of LPS and 10 ng/ml IFN-Î³, or other stimuli of interest, for desired duration.</li></ol><br/></li><li>Isolation of Polar Metabolites and Lipids (Figure 2)<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200714/20200714233716_1872.jpg" width="300"/><br/><strong>Figure 2. Work flow for extraction of lipids and small polar metabolites.</strong> ACN: acetonitrile, IPA: isopropanol.<br/><br/><ol class="npwtwo"><li>Before beginning, make the following solutions:<br/>3:1 1-butanol:methanol (LC-MS grade), with 1% SPLASH LipidoMIX Internal Standard (SPLASH)<br/>3:1 heptane:ethyl acetate (LC-MS grade)<br/>1% acetic acid (LC-MS grade)<br/>65:30:5 Acetonitrile:Isopropanol:Water (LC-MS grade, ACN:IPA:H<sub>2</sub>O)</li><li>Chill 3:1 butanol:methanol (+SPLASH) and 3:1 heptane:ethyl acetate in the -20 Â°C and 1% acetic acid in the 4 Â°C for at least an hour prior to beginning.</li><li>Aspirate media from cells.</li><li>Wash cells twice with room temperature PBS.</li><li>Add 300 Âµl of -20 Â°C 3:1 butanol:methanol (+SPLASH) to each well, place plates immediately on ice.</li><li>While keeping cell plate on ice, scrape cells using cell scraper, and transfer solution from each well to a separate 1.5 ml plastic microcentrifuge tube. Place samples on ice.<br/><em>Note: Also perform procedural blanks (N &gt; 2) by performing the same Steps B1-B5 with an empty tube/plate.</em> </li><li>Vortex each tube for 1 min, place back on ice.</li><li>In the fume hood, add 300 Âµl -20 Â°C 3:1 heptane:ethyl acetate to each sample.</li><li>Vortex all tubes for 3 min, place back on ice.</li><li>In the fume hood, add 300 Âµl 1% acetic acid to each sample.</li><li>Vortex all tubes for 3 min, place back on ice.</li><li>Centrifuge all tubes (20,000 <em>x g</em>, 10 min, 4 Â°C)âtwo separate layers should form.</li><li>Using a pipette, transfer a set volume (~400 Âµl) of the top organic layer to a glass vial (this is the lipid and fatty acid fraction) (perform in fume hood)</li><li>Using a pipette, transfer a set volume (~300 Âµl) of the bottom aqueous layer to a 1.5 ml plastic tube (this is the polar metabolite fraction) (perform in fume hood)</li><li>Completely dry down all samples by nitrogen stream using a N<sub>2</sub> sample concentrator in the fume hood. Samples typically dry within 2h at room temperature. Retrieve samples immediately after drying.</li><li>Resuspend the metabolite fraction in LC-MS grade water (50 Âµl) and the lipid fraction in 65:30:5 ACN:IPA:H<sub>2</sub>O (50 Âµl).</li><li>Transfer metabolite fraction to polypropelyne LC-MS vials and the lipid fraction to glass LC-MS vial, cap and place in LC-MS autosampler for analysis.</li></ol><br/></li><li>Liquid-chromatography Mass spectrometry<ol class="npwtwo"><li>LC-MS vials should be placed in sample trays in randomized order. Sample runs should start with injection of an equal volume of at least three appropriate blanks: LC-MS H<sub>2</sub>O for polar metabolites, 65:30:5 ACN:IPA:H<sub>2</sub>O for lipids. Blanks should additionally be run throughout, between at least every 10 samples.</li><li>Analyses of both the polar metabolite and lipid fractions were performed on a Thermo Q-Exactive mass spectrometer coupled to a Vanquish Horizon UHPLC.</li><li>Polar metabolite samples were separated on a 100 x 2.1 mm 1.7uM Acquity UPLC BEH C18 Column (Waters) coupled to a 130Ã, 1.7 Âµm, 2.1 mm x 5 mm Acquity UPLC BEH C18 VanGuard Pre-column (Waters). A gradient of solvent A (97:3 H<sub>2</sub>O:methanol, 10 mM TBA, 9 mM acetate, pH 8.2) and solvent B (100% methanol) was used at a 0.2ml/min flow rate. The gradient is: 0 min, 5% B; 2.5 min, 5% B; 17 min, 95% B; 21 min, 95% B; 21.5 min, 5% B (Figure 3A). Five Âµl of each metabolite sample was injected for analysis. Data was collected on a full scan negative mode. Detailed MS parameters are laid out in in Table 1.</li><li>Lipid samples were separated on a 100 x 2.1 mm 1.7uM Acquity UPLC CSH C18 Column (Waters) coupled to a 130Ã, 1.7 Âµm, 2.1 mm X 5 mm Acquity UPLC CSH C18 VanGuard Pre-column. A gradient of solvent C (70:30 ACN:H<sub>2</sub>O, 10 mM NH<sub>4</sub>Ac) and solvent D (90:10 IPA:ACN, 10 mM NH<sub>4</sub>Ac) was used at a 0.4 ml/min flow rate. The gradient is: 0 min, 2% D; 2 min, 2% D; 5 min, 30% D; 19 min, 85% D; 20 min 99% D; 27 min, 99% D; 28 min, 2% D; 32 min, 2%D (Figure 3B). Five Âµl of each lipid sample was injected for analysis. Samples were run twice, once in negative mode and once in positive mode both using data dependent MS2 (ddMS2) analysis. Detailed MS parameters are laid out in in Table 1.<br/><br/><strong>Table 1. Mass Spectrometry Parameters</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200714/20200714233806_2340.jpg" width="550"/><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200714/20200714233848_9964.jpg" width="300"/><br/><strong>Figure 3. Solvent gradients and elution profiles.</strong> A. Solvent gradient and example elution profile for LC-MS polar metabolite method. B. Solvent gradient and example elution profile for LC-MS lipid analysis, profiles from both negative and positive ionization runs are shown.</li></ol></li></ol><p class="pptt" id="biaoti34826">### Data analysis</p><p class="ppzz">MS1 data is used for both polar metabolite and free fatty acid quantification using MAVEN software (Melamud <em>et al.</em>, 2010; Clasquin <em>et al.</em>, 2012). Lipid data is identified and quantified based on full scanâddMS2 using LipiDex Software (Hutchins <em>et al.</em>, 2018). Both methods are described below. Representative data obtained from analysis of metabolite and lipids profiles of unstimulated (M0) and LPS+IFNÎ³ stimulated BMDM (M1) is shown in Figure 4.</p><p class="ppzz"><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200714/20200714233903_4964.jpg"/><br/><strong>Figure 4. Representative metabolite and lipid data.</strong> A volcano plot showing fold changes in identified metabolites and lipids in LPS+IFNÎ³ stimulated (M1) BMDM as compared to unstimulated (M0) BMDM. Highlighted are itaconate and succinate, two immunoregulatory metabolites known to be dynamically upregulated in M1 macrophages (Jha <em>et al.</em>, 2015; Cordes <em>et al.</em>, 2016; Lampropoulou <em>et al.</em>, 2016; Mills <em>et al.</em>, 2016). Also highlighted is triglyceride 50:2. Triglyceride accumulation has also been shown to occur in M1 macrophages (KÃ¶berlin <em>et al.</em>, 2014; Lee <em>et al.</em>, 2017; Feingold <em>et al.</em>, 2012).<br/><br/><strong><em>Polar Metabolites and Fatty Acids</em></strong><br/>Polar metabolites and fatty acids are quantified in a targeted fashion by matching the expected exact mass and retention time. The list of retention times and M/z is determined from analysis of metabolite standards using the same LC-MS method for samples as outlined above, and is in the supplementary information (supplemental file â<a href="https://os.bio-protocol.org/attached/file/20200706/Polar Metabolites RT List.xlsx" target="_blank">Polar Metabolites RT List .xlsx</a>â). We suggest that particularly with metabolites of interest, users run standards of their own. To do so, dilute standards to a range of concentrations (<em>e.g.</em>, 1 Î¼M, 10 Î¼M, 100 Î¼M, 1,000 Î¼M, for inject 1 Î¼l injection volume) in the appropriate sample solvents, and run on LC-MS using the corresponding method. For untargeted metabolomics analysis, see additional resources (Smith <em>et al.</em>, 2005 and 2006; Kind <em>et al.</em>, 2018; Wang <em>et al.</em>, 2019). Additionally, this method will only describe what is necessary for relative quantification of metabolites. For measurement of absolute concentration of metabolites see additional resources (Bennett <em>et al.</em>, 2008). We use MAVEN software for integration of peaks (Melamud <em>et al.</em>, 2010; Clasquin <em>et al.</em>, 2012). This software and more detailed guides on its use are available <a href="http://genomics-pubs.princeton.edu/mzroll/index.php?show=download" target="_blank">here</a>.</p><br/><ol class="npwtwo"><li>Convert Thermo generated .raw files to .mzXML files using MSConvertGUI (Holman <em>et al.</em>, 2014; Adusumilli <em>et al.</em>, 2017).</li><li>Open .mzXML files in MAVEN. When properly loaded samples IDs will appear in the samples window to the left.</li><li>Make sure âCompounds Widgetâ is turned on. Load compound list by clicking on compound tab on left and selecting âLoad Custom Compound Listâ. Load the .csv file with known m/z and retention times generated from metabolite standards or as provided in supplemental material. Adjust the ppm window on top right of Maven window to match that of the sensitivity of your instrument.</li><li>With compound list loaded, click on name of compound of interest and the peaks corresponding to that metabolite in each sample will appear in the main window. The red line corresponds to the observed retention time according to the imported compound list (Figure 5). To save the data from selected peak double click on the peakâthe name of the compound and corresponding parameters will be displayed in a list within the software. Save data only from those peaks whose retention time matches that on the compound list, has a relatively smooth profile, and whose signal is above desired cut off (<em>e.g.</em>, 5E4 and at least 2x times the blank). See Figure 5 for example.<br/><em>Note: More detailed information about software functions and parameter setting can be found in MAVEN using manual.</em> </li><li>When all desired peaks have been saved, export this data from Maven by selecting âExport Peaks to SpreadSheet (.csv) above the list of saved peaks.</li><li>Blank the data by subtracting the average signal from the blanks from the signal from each sample. Subsequently normalize data to cell count or protein.<br/><em>Note: Only compare relative abundance from samples run within the same sequence, as changes in instrument conditions over time may cause difference in signal response. If absolute quantitation of metabolite concentration is desired, use of isotopic internal standards is recommended.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200714/20200714234007_2769.jpg" width="300"/><br/><strong>Figure 5. Example of a good LC-MS peak.</strong> Succinate peaks from unstimulated (M0) and LPS+IFNÎ³ stimulated (M1) BMDM, as displayed by Maven software.</li></ol><p><br/></p><p class="ppzz"><strong><em>Lipids</em></strong><br/>Lipid identification and quantification is performed using the open access LipiDex software (Hutchins <em>et al.</em>, 2018). Based on full scan-ddMS2 data, LipiDex identifies lipids. The software is coupled to MZmine2 to quantify lipid species. Below is our workflow for this analysis (Pluskal <em>et al.</em>, 2010).</p><ol class="npwtwo"><li>Convert Thermo generated .raw files to .mgf files (we use MSConvertGUI software [Holman <em>et al.</em>, 2014; Adusumilli <em>et al.</em>, 2017]).</li><li>Perform Spectrum Search in LipiDex<ol class="npwthree"><li>Open LipiDex and go to Spectrum Searcher.</li><li>Upload .mgf files.</li><li>Enable the appropriate libraries.</li><li>Specify desired parameters for search (<em>e.g.</em>, MS1 search tolerance: 0.01, MS2 search tolerance: 0.01).</li><li>Click âSpectra Searchââthis will produce result .csv files with identifications.</li></ol></li><li>Generate chromatographic feature tables using MZmine2. Do separately for positive and negative ionization runs. Follow step-by-step instructions as provided <a href="https://github.com/coongroup/LipiDex/wiki/Peak-Finding-Using-mzMine-2" target="_blank">here</a>.</li><li>Perform Peak Finding in LipiDex<ol class="npwthree"><li>Open LipiDex and go to Peak Finder.</li><li>Click the MZMine2 option.</li><li>Load the positive and negative polarity peak tables generated by MZMine2.</li><li>Upload the MS/MS .csv result files, making sure to identify the files with the appropriate polarity (positive or negative).</li><li>Set desired filtering parameters, for instance, using the following:<br/>MS2 filtering parameters: Min. Lipid Spectral Purity (%): 75, Min. MS2 Search Dot Product: 500, Min. MS2 Search Rev. Dot Product: 700.<br/>Feature Association Parameters: FWHM Window Multiplier: 2.0, Max. Mass Difference (ppm): 15.<br/>Result Filtering Parameters: Check Adduct/Dimer Filtering and In-source Fragment Filtering. Check Max. RT M.A.D Factor with 3.5. Feature found in n Files: 2.</li><li>Click Identify Chromatographic Peaksâthis will produce .csv files containing identified lipid species and their quantification.</li></ol></li><li>Blank data using the procedural blanks that were run parallel to samples, and normalize data to cell count or protein content.</li><li><em>Quality Control:</em> The SPLASH LipidoMIX Internal Standard is a mixture of isotopic labeled lipids, representing major lipid classes. SPLASH (1%) was added to the initial extraction buffer (3:1 1-butanol:methanol). The presence of this internal standard in all analyzed lipid samples allows for determination of retention time drift across samples, and potential difference in sample recovery. The retention time drift within a sequence is typically &lt; 0.1min, and the variation of internal standards is &lt; 20%. If the variation is significant, data can be normalized to internal standard of the same lipid class or nearest retention time (if there is no standard of the same class). See supplemental file â<a href="https://os.bio-protocol.org/attached/file/20200706/SPLASH LipidoMix Stds Compound list.xlsx" target="_blank">SPLASH LipidoMix Stds Compound list .xlsx</a>â for list of compounds in SPLASH LipidoMIX and their respective retention times using the outlined method.</li></ol><p class="pptt" id="biaoti34827">### Notes</p><ol class="npwtwok"><li>Keep and measure out all LC-MS grade solvents in clean designated glassware. Do not use this glassware for other purposes and do not wash with tap water or soap. Residual detergent and other material will cause background. When necessary, rinse with HPLC grade solvents only.</li><li>Plastic used during making of extraction solvents and the extraction of lipids can cause background, particularly for fatty acids. Always perform a procedural blank to appropriately account for this background signal. Alternative use of glass vials and pipettes in place of plastic microcentrifuge tubes and pipette tips can help reduce background if a considerable amount is observed.<br/></li></ol><p class="pptt" id="biaoti34828">### Recipes</p><ol class="npwtwok"><li>BMDM Differentiation Media<br/>RPMI 1640 without Glutamine<br/>10% FBS<br/>20% L929 Conditioned Media<br/>10 U/ml penicillin/streptomycin<br/>25 mM HEPES<br/>2 mM glutamine</li><li>BMDM Maintenance Media<br/>RPMI 1640 without Glutamine<br/>10% dFBS<br/>10 U/ml penicillin/streptomycin<br/>25 mM HEPES<br/>2 mM glutamine<br/>20 ng/ml m-CSF</li><li>L929 Culture Media<br/>DMEM High Glucose <br/>10 U/ml penicillin/streptomycin<br/>10% FBS</li><li>L929 Conditioned Media<ol class="npwthree"><li>Culture L929 cells in L929 culture media until fully confluent</li><li>Trypsinize cells, spin down at 1,000 <em>x g</em> for 5 min</li><li>Aspirate trypsin, resuspend cell pellet in L929 culture media</li><li>Count cells and resuspend to 8 x 10<sup>3</sup> cells/ml</li><li>Add 130 ml of this suspension to a 175 cm<sup>2</sup> flask and place in incubator</li><li>7 days later collect all the spent media</li><li>Centrifuge at 1,000 <em>x g</em>, 4 Â°C, for 5 min. Sterile filter using a 0.45 Î¼m filter into a sterile glass bottle</li><li>Aliquot to 50 ml plastic centrifuge tubes, and freeze at -20 Â°C until desired use (use within 6 months)</li></ol></li><li>Solvent A<br/>97:3 water:methanol (LC-MS grade)<br/>10 mM tributylamine <br/>Acetic acid (LC-MS grade), pH to 8.2<br/><em>Note: This takes a great deal of vigorous mixing to fully dissolve the tributylamine and reach a stable pH.</em> </li><li>Solvent C<br/>70:30 acetonitrile:water (LC-MS grade)<br/>10 mM Ammonium Acetate (LC-MS grade)<br/>250 Âµl/L Acetic Acid (LC-MS grade) </li><li>Solvent D<br/>90:10 isopropanol:acetonitrile (LC-MS grade)<br/>10 mM Ammonium Acetate (LC-MS grade)<br/>250 Âµl/L Acetic Acid (LC-MS grade)<br/></li></ol><p class="pptt" id="biaoti34829">### Acknowledgments</p><p class="ppzz">The methods for bone marrow derived macrophages isolation, metabolites and lipids extraction, LC-MS analysis and data analysis are adapted and optimized based on previous works, each referenced in the respective section.</p><p class="pptt" id="biaoti34830">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti34831">### Ethics</p><p class="ppzz">Animal care and experimental procedures were carried out in accordance with a protocol approved by University of Wisconsin-Madison Institutional Animal Care and Use Committee.</p><p class="pptt" id="biaoti34832">### References</p><ol class="npwtwok"><li>Adusumilli, R. and Mallick, P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28188540" target="_blank">Data conversion with proteowizard msConvert.</a> <em>Methods Mol Biol</em> 1550: 339-368.</li><li>Bennett, B. D., Yuan, J., Kimball, E. H. and Rabinowitz, J. D. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18714298" target="_blank">Absolute quantitation of intracellular metabolite concentrations by an isotope ratio-based approach.</a> <em>Nat Protoc</em> 3(8): 1299-1311.</li><li>Clasquin, M. F., Melamud, E. and Rabinowitz, J. D. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22389014" target="_blank">LC-MS data processing with MAVEN: a metabolomic analysis and visualization engine.</a> <em>Curr Protoc Bioinformatics</em> Chapter 14: Unit14 11.</li><li>Cordes, T., Wallace, M., Michelucci, A., Divakaruni, A. S., Sapcariu, S. C., Sousa, C., Koseki, H., Cabrales, P., Murphy, A. N., Hiller, K. and Metallo, C. M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27189937" target="_blank">Immunoresponsive gene 1 and itaconate inhibit succinate dehydrogenase to modulate intracellular succinate levels.</a> <em>J Biol Chem</em> 291(27): 14274-14284.</li><li>Feingold, K. R., Shigenaga, J. K., Kazemi, M. R., McDonald, C. M., Patzek, S. M., Cross, A. S., Moser, A. and Grunfeld, C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22753953" target="_blank">Mechanisms of triglyceride accumulation in activated macrophages.</a> <em>J Leukoc Biol</em> 92(4): 829-839.</li><li>Fiehn, O. (2002). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11860207" target="_blank">Metabolomics--the link between genotypes and phenotypes.</a> <em>Plant Mol Biol</em> 48(1-2): 155-171.</li><li>GalvÃ¡n-PeÃ±a, S. and OâNeill, L. A. J. (2014). <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4151090/" target="_blank">Metabolic reprogramming in macrophage polarization.</a> <em>Front Immunol</em> 5: 420.</li><li>Gika, H. G., Theodoridis, G. A., Plumb, R. S. and Wilson, I. D. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23916607" target="_blank">Current practice of liquid chromatography-mass spectrometry in metabolomics and metabonomics.</a> <em>J Pharm Biomed Anal</em> 87: 12-25.</li><li>Holman, J. D., Tabb, D. L. and Mallick, P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24939128" target="_blank">Employing proteowizard to convert raw mass spectrometry data.</a> <em>Curr Protoc Bioinformatics</em> 46: 13 24 11-19.</li><li>Hutchins, P. D., Russell, J. D. and Coon, J. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29705063" target="_blank">Lipidex: an integrated software package for high-confidence lipid identification.</a> <em>Cell Syst</em> 6(5): 621-625 e625.</li><li>Jha, A. K., Huang, S. C., Sergushichev, A., Lampropoulou, V., Ivanova, Y., Loginicheva, E., Chmielewski, K., Stewart, K. M., Ashall, J., Everts, B., Pearce, E. J., Driggers, E. M. and Artyomov, M. N. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25786174" target="_blank">Network integration of parallel metabolic and transcriptional data reveals metabolic modules that regulate macrophage polarization.</a> <em>Immunity</em> 42(3): 419-430.</li><li>Kelly, B. and O'Neill, L. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26045163" target="_blank">Metabolic reprogramming in macrophages and dendritic cells in innate immunity.</a> <em>Cell Res</em> 25(7): 771-784.</li><li>Kessner, D., Chambers, M., Burke, R., Agus, D. and Mallick, P. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18606607" target="_blank">ProteoWizard: open source software for rapid proteomics tools development.</a> <em>Bioinformatics</em> 24(21): 2534-2536.</li><li>Kind, T., Tsugawa, H., Cajka, T., Ma, Y., Lai, Z., Mehta, S. S., Wohlgemuth, G., Barupal, D. K., Showalter, M. R., Arita, M. and Fiehn, O. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28436590" target="_blank">Identification of small molecules using accurate mass MS/MS search.</a> <em>Mass Spectrom Rev</em> 37(4): 513-532.</li><li>KÃ¶berlin, M. S., Heinz, L. X. and Superti-Furga, G. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26895312" target="_blank">Functional crosstalk between membrane lipids and TLR biology.</a> <em>Curr Opin Cell Biol</em> 39: 28-36.</li><li>Lampropoulou, V., Sergushichev, A., Bambouskova, M., Nair, S., Vincent, E. E., Loginicheva, E., Cervantes-Barragan, L., Ma, X., Huang, S. C., Griss, T., Weinheimer, C. J., Khader, S., Randolph, G. J., Pearce, E. J., Jones, R. G., Diwan, A., Diamond, M. S. and Artyomov, M. N. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27374498" target="_blank">Itaconate links inhibition of succinate dehydrogenase with macrophage metabolic remodeling and regulation of inflammation.</a> <em>Cell Metab</em> 24(1): 158-166.</li><li>Lee, J. W., Mok, H. J., Lee, D. Y., Park, S. C., Kim, G. S., Lee, S. E., Lee, Y. S., Kim, K. P. and Kim, H. D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28251853" target="_blank">UPLC-QqQ/MS-Based Lipidomics Approach To Characterize Lipid Alterations in Inflammatory Macrophages.</a> <em>J Proteome Res</em> 16(4): 1460-1469.</li><li>Liu, P. S., Wang, H., Li, X., Chao, T., Teav, T., Christen, S., Di Conza, G., Cheng, W. C., Chou, C. H., Vavakova, M., Muret, C., Debackere, K., Mazzone, M., Huang, H. D., Fendt, S. M., Ivanisevic, J. and Ho, P. C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28714978" target="_blank">Î±-ketoglutarate orchestrates macrophage activation through metabolic and epigenetic reprogramming.</a> <em>Nat Immunol</em> 18(9): 985-994.</li><li>LÃ¶fgren, L., Forsberg, G. B. and StÃ¥hlman, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27282822" target="_blank">The BUME method: a new rapid and simple chloroform-free method for total lipid extraction of animal tissue.</a> <em>Sci Rep</em> 6: 27688.</li><li>LÃ¶fgren, L., Stahlman, M., Forsberg, G. B., Saarinen, S., Nilsson, R. and Hansson, G. I. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22645248" target="_blank">The BUME method: a novel automated chloroform-free 96-well total lipid extraction method for blood plasma.</a> <em>J Lipid Res</em> 53(8): 1690-1700.</li><li>Martin, S. L., Seim, G. L., Wawire, S., Chapleau, G. M., Young, S. L. and Dickin, K. L. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26898417" target="_blank">Translating formative research findings into a behaviour change strategy to promote antenatal calcium and iron and folic acid supplementation in western Kenya.</a> <em>Matern Child Nutr</em> 13(1).</li><li>Melamud, E., Vastag, L. and Rabinowitz, J. D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21049934" target="_blank">Metabolomic analysis and visualization engine for LC-MS data.</a> <em>Anal Chem</em> 82(23): 9818-9826.</li><li>Mills, E. L. and O'Neill, L. A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26643360" target="_blank">Reprogramming mitochondrial metabolism in macrophages as an anti-inflammatory signal.</a> <em>Eur J Immunol</em> 46(1): 13-21.</li><li>Mills, E. L., Kelly, B., Logan, A., Costa, A. S. H., Varma, M., Bryant, C. E., Tourlomousis, P., Dabritz, J. H. M., Gottlieb, E., Latorre, I., Corr, S. C., McManus, G., Ryan, D., Jacobs, H. T., Szibor, M., Xavier, R. J., Braun, T., Frezza, C., Murphy, M. P. and O'Neill, L. A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27667687" target="_blank">Succinate dehydrogenase supports metabolic repurposing of mitochondria to drive inflammatory macrophages.</a> <em>Cell</em> 167(2): 457-470 e413.</li><li>O'Neill, L. A. and Pearce, E. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26694970" target="_blank">Immunometabolism governs dendritic cell and macrophage function.</a> <em>J Exp Med</em> 213(1): 15-23.</li><li>Pluskal, T., Castillo, S., Villar-Briones, A. and OreÅ¡iÄ, M. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20650010" target="_blank">MZmine 2: modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data.</a> <em>BMC Bioinformatics</em> 11: 395.</li><li>Smith, C. A., O'Maille, G., Want, E. J., Qin, C., Trauger, S. A., Brandon, T. R., Custodio, D. E., Abagyan, R. and Siuzdak, G. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16404815" target="_blank">METLIN: a metabolite mass spectral database.</a> <em>Ther Drug Monit</em> 27(6): 747-751.</li><li>Smith, C. A., Want, E. J., O'Maille, G., Abagyan, R. and Siuzdak, G. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16448051" target="_blank">XCMS: processing mass spectrometry data for metabolite profiling using nonlinear peak alignment, matching, and identification.</a> <em>Anal Chem</em> 78(3): 779-787.</li><li>Tannahill, G. M., Curtis, A. M., Adamik, J., Palsson-McDermott, E. M., McGettrick, A. F., Goel, G., Frezza, C., Bernard, N. J., Kelly, B., Foley, N. H., Zheng, L., Gardet, A., Tong, Z., Jany, S. S., Corr, S. C., Haneklaus, M., Caffrey, B. E., Pierce, K., Walmsley, S., Beasley, F. C., Cummins, E., Nizet, V., Whyte, M., Taylor, C. T., Lin, H., Masters, S. L., Gottlieb, E., Kelly, V. P., Clish, C., Auron, P. E., Xavier, R. J. and O'Neill, L. A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23535595" target="_blank">Succinate is an inflammatory signal that induces IL-1Î² through HIF-1Î±.</a> <em>Nature</em> 496(7444): 238-242.</li><li>Theodoridis, G. A., Gika, H. G., Want, E. J. and Wilson, I. D. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22152789" target="_blank">Liquid chromatography-mass spectrometry based global metabolite profiling: a review.</a> <em>Anal Chim Acta</em> 711: 7-16.</li><li>Van den Bossche, J., O'Neill, L. A. and Menon, D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28396078" target="_blank">Macrophage immunometabolism: where are we (going)?</a> <em>Trends Immunol</em> 38(6): 395-406.</li><li>Wang, L., Xing, X., Chen, L., Yang, L., Su, X., Rabitz, H., Lu, W. and Rabinowitz, J. D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30586294" target="_blank">Peak annotation and verification engine for untargeted LC-MS metabolomics.</a> <em>Anal Chem</em> 91(3): 1838-1846.</li><li>Wenk, M. R. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16052242" target="_blank">The emerging field of lipidomics.</a> <em>Nat Rev Drug Discov</em> 4(7): 594-610.</li><li>Williams, N. C. and O'Neill, L. A. J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29459863" target="_blank">A role for the krebs cycle intermediate citrate in metabolic reprogramming in innate immunity and inflammation.</a> <em>Front Immunol</em> 9: 141.</li><li>Ying, W., Cheruku, P. S., Bazer, F. W., Safe, S. H. and Zhou, B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23851980" target="_blank">Investigation of macrophage polarization using bone marrow derived macrophages.</a> <em>J Vis Exp</em>(76): 50323.<br/></li></ol>