<body>
            <h1>Generation and Testing of Fluorescent Adaptable Simple Theranostic (FAST) Proteins</h1>
            <p><strong>Authors:</strong> Andrew S. Flies, Jocelyn M. Darby, Peter R. Murphy, Terry L. Pinfold, Amanda L. Patchett, Patrick R. Lennard</p>
            <h2>Abstract</h2>
            <p>This protocol provides a step-by-step method to create recombinant fluorescent fusion proteins that can be secreted from mammalian cell lines. This builds on many other recombinant protein and fluorescent protein techniques, but is among the first to harness fluorescent fusion proteins secreted directly into cell culture supernatant. This opens new possibilities that are not achievable with proteins produced in bacteria or yeast, such as direct use of the fluorescent protein-secreting cells in live co-culture assays. The Fluorescent Adaptable Simple Theranostic (FAST) protein system includes a histidine purification tag and a tobacco etch virus (TEV) cleavage site, allowing the purification tag and fluorescent protein to be removed for therapeutic use. This protocol is split into five parts: (A) <em>In silico</em> characterization of the gene-of-interest (GOI) and protein-of-interest (POI); (B) design of the expression vector; (C) assembly of the expression vector; (D) transfection of a eukaryotic cell line with the expression vector; (E) testing of the recombinant protein. This extensive protocol can be completed with only polymerase chain reaction (PCR) and cell culture training. Additionally, each part of the protocol can be used independently.</p>
            <p><strong>Keywords:</strong> Fluorescent, Fusion protein, Recombinant, Comparative immunology, Non-model organism, Soluble, Trans-endocytosis</p>
            <h2>Background</h2><p>Recombinant proteins are key tools for many basic research and biomedical fields. Production of recombinant proteins generally entails design and assembly of expression vectors followed by production of the recombinant protein in prokaryotic or eukaryotic cells. Expression of proteins in eukaryotic cells is often preferred when post-translational modifications, such protein glycosylation, is important for downstream functional testing. Many excellent protocols are available for aspects of the full recombinant protein production cycle (Benson <em>et al.</em>, 2013; Flies <em>et al.</em>, 2020), but few are available the provide step-by-step details for the entire production and functional testing process. This protocol can be adapted to produce species-specific recombinant proteins with and without a fluorescent reporter protein fused to a protein of interest (POI). It can also be used for creating vectors for non-secreted proteins (<em>e.g.</em>, cell surface proteins). The methods can be used for most eukaryotic species, but we have focused on a single gene from the Tasmanian devil (<em>Sarcophilus harrisii</em>) for illustrative purposes. This protocol will result in a recombinant protein that includes the extracellular domain (ECD) of the CD200 (aka OX-2) protein that is fused directly to a fluorescent reporter protein. CD200 is an immune checkpoint protein that is highly expressed on several types of cancer. This protein will be secreted from mammalian cells after transfection and can be used directly from supernatant or purified for downstream use. An overview of the complete protocol can be seen in <strong>Figure 1</strong> (Flies <em>et al.</em>, 2020).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701014430_5967.jpg" width="450"/><br/><strong>Figure 1. FAST protein schematic and initial testing.</strong> A. Schematic diagram of FAST protein therapeutic and diagnostic (<em>i.e.</em>, theranostic) features. B. Graphic overview of FAST protein system including key steps: (1) characterize gene-of-interest (GOI) <em>in silico</em>; (2) design expression vectors; (3) digest FAST base vectors and insert alternative GOIs or colors; (4) transfect expression vectors into mammalian cells and monitor using fluorescent microscopy or flow cytometry; (5) purify the protein using 6xHistidine tag, visualize fluorescent color to show protein is in-frame and correctly folded. Image of microfuge tubes shows 100 Î¼l of mCitrine, mOrange, and mCherry FAST proteins (1 mg/ml) excited with blue light with amber filter. Full protocols for vector construction and protein testing are available in the Supplementary Materials. C. Results of flow cytometry binding assay using Tasmanian devil 41BB (aka TNFRSF9, CD137) FAST proteins and cell lines expressing 41BB ligand (aka TNFSF9, CD137L). The colored lines in the histograms show binding of devil 41BB fused to mTagBFP, mCerulean3, mAzurite, mCitrine, mOrange, mCherry, or mNeptune2 to Chinese hamster ovary (CHO) cells transfected with devil 41BBL, and the black lines show binding to untransfected CHO cells. Figure reprinted from Flies <em>et al.</em> (2020) under CC BY-NC license.<br/><br/>The recombinant protein construct includes a linker protein with three additional features (<strong>Figure 1A</strong>). First is a TEV cleavage site, which allows the protein to be cleaved to separate the POI and the fluorescent reporter protein. Second is a rigid linker protein that provides additional separation of the POI and the reporter (<em>i.e.</em>, so the two proteins do not interact with each other). Third is a 6x histidine (6xHis) tag that allows for easy purification from cell culture supernatant.<br/><br/>The protocol here describes how to make Tasmanian devil CD200-mTagBFP and CD200-mOrange Fluorescent Adaptable Simple Theranostic (FAST) proteins, but can be adapted to make most other type I transmembrane proteins and secreted proteins (<em>e.g.</em>, cytokines). Simply replace the CD200 coding sequence with a different gene of interest (GOI) and repeat the step-by-step protocol. We have also used the expression vectors to produce type II transmembrane proteins (see definitions below for type I and II proteins).<br/><br/>We have made a spreadsheet available that has templates for performing each of the major experiments necessary to complete this protocol. The spreadsheet contains a tab with the list of reagents, a tab for recipes, and a tab for each experiment. Every experiment in our lab is given a unique ID based on the person performing the experiment. For example, the first experiment done by Andrew S. Flies would be exp_ASF_1. Each new lab member has their own three-letter code. We have titled the experiments in the accompanying spreadsheet as exp_ID_1.<br/><br/><a href="https://wildimmunity.com/wp-content/uploads/2019/11/FAST_protein_experiment_templates.xlsx" target="_blank">FAST protein experiment templates</a> from Flies <em>et al.</em> (2020)âScience Advances DOI: 10.1126/sciadv.aba5031.<br/></p>
        </body>

<p class="pptt" id="biaoti34673">## Materials and Reagents</p><ol class="npwtwok"><li>0.22 Î¼m PVDF syringe filter (Merck/Millipore, catalog number: <a class="blackNoUnder" target="_blank"> SLGV033RS </a> )</li><li>20 ml syringe (any supplier)</li><li>6-well plates, tissue culture treated (any supplier)</li><li>96-well plate, flat-bottom, tissue culture treated (any supplier)</li><li>96-well plate, U-bottom, tissue culture treated (any supplier)</li><li>Amicon Ultra centrifugal filter units, Ultra-15, MWCO 10 kDa (Merck/Millipore, catalog number: <a class="blackNoUnder" target="_blank"> UFC901008 </a> )</li><li>Bacterial spreader (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> Z723193-500EA </a> )</li><li>Flask, T175, tissue culture treated (T25, T75, T175, any supplier)</li><li>PCR tubes (any supplier)</li><li>Petri dishes, 100 mm (any supplier)</li><li>pH meter or pH strips (any supplier)</li><li>Pipet tips (unfiltered): 10, 200 and 1,000 Î¼l (any supplier)</li><li>Pipet tips (filtered): 10, 200 and 1,000 Î¼l (any supplier)</li><li>HisTrap excel columns (5 x 1 ml) (GE Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 17371205 </a> )</li><li>CHO-K1 cells (ATCC, catalog number: <a class="blackNoUnder" target="_blank"> CCL-61 </a> )</li><li><em>E. coli</em> DH5Î± competent cells (New England Biolabs, comes with NEBuilder kit)</li><li>Agarose (any supplier)</li><li>Agencourt CleanSEQâDye Terminator Removal (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> A29151 </a> )</li><li>Antibiotic Antimycotic Solution (100x) (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A5955 </a> )</li><li>Big Dye Terminator kit (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 4337455 </a> )</li><li>EX-CELL CHO protein free media (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 14361C-1000ML </a> )</li><li>ExpiCHO expression system (optional for high yield protein production) (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> A29133 </a> )</li><li>Formalin solution, neutral buffered, 10% (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> HT501128-4L </a> )</li><li>Hyrdochloric acid (HCl) 12 M or higher (any supplier)</li><li>Hygromycin (Merck, catalog number: <a class="blackNoUnder" target="_blank"> H0654-1G </a> )</li><li>NEBuilder HiFi DNA assembly cloning kit (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> E5520S </a> )</li><li>NotI-HF (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R3189S </a> )</li><li>Nucleic acid gel stain (any supplier)</li><li>OneTaq<sup>Â®</sup> Hot Start Quick-Load<sup>Â®</sup> 2x Master Mix with Standard Buffer (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0488L </a> )</li><li>Plasmid miniprep kit (Machery-Nagel, catalog number: <a class="blackNoUnder" target="_blank"> 740499.5 </a> 0)</li><li>Polyethylenimine (PEI) (linear, MW 25,000) (Polysciences, catalog number: <a class="blackNoUnder" target="_blank"> 23966-2 </a> )</li><li>Pur-A-Lyzer<sup>TM</sup> Mega Dialysis Kit (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> PURG12020-1KT </a> )</li><li>Q5<sup>Â®</sup> Hot Start High-Fidelity 2x Master Mix (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0494L </a> )</li><li>RPMI-1640 medium (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> R7509 </a> )</li><li>SmaI (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0141S </a> )</li><li>SOC media (New England Biolabs, comes with NEBuilder kit)</li><li>Sodium hydroxide (NaOH) 10 M or higher (any supplier)TrypLE Express enzyme (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 12604039 </a> )</li><li>Water (<em>e.g.</em>, Milli-Q deionized-distilled) (any supplier)</li><li>NaCl</li><li>KCl</li><li>Na<sub>2</sub>HPO<sub>4</sub> </li><li>KH<sub>2</sub>PO<sub>4</sub> </li><li>Tris base</li><li>FBS (heat inactivated)</li><li>Glutamine</li><li>2-ME</li><li>HEPES</li><li>DMEM</li><li>FCS</li><li>Sodium-pyruvate</li><li>DMSO</li><li>Tryptone</li><li>Yeast extract</li><li>KHCO<sub>3</sub> </li><li>Glycerol</li><li>Glycine</li><li>Bromophenol blue</li><li>1x Phosphate-buffered saline pH 7.4 (see Recipes)</li><li>10x Phosphate-buffered saline pH 7.4 (see Recipes)</li><li>Tris-buffered saline (TBS) pH 7.5 (see Recipes)</li><li>20% ethanol (see Recipes)</li><li>70% ethanol (see Recipes)</li><li>1 M HCl (see Recipes)</li><li>1 M NaOH (see Recipes)</li><li>Complete RPMI with 10% fetal bovine serum (cRF10) (see Recipes)</li><li>Complete RPMI with 5% fetal bovine serum (cRF5) (see Recipes)</li><li>Complete RPMI without fetal bovine serum (cRF0) (see Recipes)</li><li>Complete DMEM with 10% FBS (cDF10) (see Recipes)</li><li>Complete IMDM with 20% FBS (cIF20) for hybridomas (see Recipes)</li><li>2-mercaptoethanol (2-ME) (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M3148 </a> ) (see Recipes)</li><li>2x cell freezing media (see Recipes)</li><li>Ampicillin stock (see Recipes)</li><li>Luria broth (LB medium) (see Recipes) or (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L3522 </a> )</li><li>Luria broth (LB) agar (see Recipes) or (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L3147 </a> )</li><li>LB medium (low salt) (see Recipes)</li><li>LB agar (low salt) (see Recipes)</li><li>Bacterial freezing media (2x) (see Recipes)</li><li>1.5 M Tris pH 8.8 (see Recipes)</li><li>1 M Tris pH 6.8 (see Recipes)</li><li>1 M Tris pH 9.0 (see Recipes)</li><li>Ammonium chloride (block lysosomal degradation) (see Recipes)</li><li>Chloroquine diphosphate (Merck/Sigma, catalog number: <a class="blackNoUnder" target="_blank"> C6628 </a> ) (see Recipes)</li><li>RBC lysis buffer (see Recipes)</li><li>10% sodium azide (see Recipes)</li><li>4% paraformaldehyde (see Recipes)</li><li>Flow cytometry wash buffer with sodium azide, without EDTA (aka FACS buffer) (see Recipes)</li><li>Flow cytometry wash buffer with sodium azide and EDTA (aka FACS buffer) (see Recipes)</li><li>Flow cytometry fixation buffer (see Recipes)</li><li>2x Laemmli buffer (see Recipes)</li><li>6x SDS reducing buffer (see Recipes)</li><li>0.01 M Tris-HCl (see Recipes)</li><li>Tris-Glycine SDS running buffer (see Recipes)</li><li>0.15 M NaCl (see Recipes)</li><li>0.5 M EDTA (pH 8.0) (see Recipes)</li><li>50x TAE (see Recipes)</li><li>Glycerol &amp; bromophenol blue gel loading buffer (6x) (see Recipes)</li><li>ELISA coating buffer (see Recipes)</li><li>ELISA Wash buffer (see Recipes)</li><li>D-luciferin potassium salt (see Recipes)</li><li>Lysis buffer for RNA extraction (see Recipes)</li><li>BDT sequencing buffer (see Recipes)</li><li>Immunofluorescence blocking buffer (see Recipes)</li><li>HisTrap Excel Reagents (see Recipes)<br/>5 M imidazole<br/>10x equilibration buffer<br/>Equilibration buffer<br/>Wash buffer<br/>Elution buffer<br/></li></ol><p class="pptt" id="biaoti34674">### Equipment</p><ol class="npwtwok"><li>1 L beaker (any supplier)</li><li>Magnetic stir bar (any supplier)</li><li>ÃKTA start protein purification system (GE Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 29022094-ECOMINSSW </a> )<br/><em>Note: A simple peristaltic pump or gravity flow columns can be used instead to save cost.</em> </li><li>Access to a DNA sequencing machine (any supplier)</li><li>Automated cell counter or haemocytometer (any supplier)</li><li>Bacteria shaker at 37 Â°C (any supplier)</li><li>Biosafety cabinet for sterile cell culture (BSC) (any supplier)</li><li>Cell culture incubators with 5% CO<sub>2</sub> and 37 Â°C (any supplier)</li><li>Centrifuge, refrigerated is preferred but not necessary (any supplier)</li><li>Computer (any supplier)</li><li>Flow cytometer (any supplier)</li><li>Gel electrophoresis equipment (any supplier)</li><li>Gel Documentation System (any supplier) </li><li>Magnetic stirrer (any supplier)</li><li>Pipet controller (aka Pipet-Aid, pipettor) (any supplier)</li><li>Pipets (10 Î¼l, 100 Î¼l, 200 Î¼l, 1,000 Î¼l) (any supplier)</li><li>Thermocycler (any supplier)</li><li>Spectrophotometer (any supplier)<br/></li></ol><p class="pptt" id="biaoti34675">### Software</p><ol class="npwtwo"><li>NCBI BLASTN for sequence comparison (free use online; free download available: <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">https://blast.ncbi.nlm.nih.gov/Blast.cgi</a>)</li><li>SnapGene (free sequence viewer use online; purchase recommended for plasmid construction, SnapGene: <a href="https://www.snapgene.com/" target="_blank">https://www.snapgene.com/</a>)<br/><em>Note: Software for designing plasmids and aligning sequencing results. Other options are available, but this is the best in our opinion.</em> </li><li>CLC Sequence Viewer (CLC sequence viewer is free; CLC Main Workbench, Qiagen available <a href="https://digitalinsights.qiagen.com/products-overview/discovery-insights-portfolio/analysis-and-visualization/qiagen-clc-genomics-workbench/" target="_blank">here</a>)</li><li>Phobius (free use online: <a href="http://phobius.sbc.su.se/" target="_blank">http://phobius.sbc.su.se/</a>)<br/>Online algorithm for determine the location of SigP (Signal peptide), ECD, TMD (Transmembrane domain), and Intracellular domain, (ICD) for the protein-of-interest.</li><li>TMHMM server (free use online: <a href="https://services.healthtech.dtu.dk/service.php?TMHMM-2.0" target="_blank">https://services.healthtech.dtu.dk/service.php?TMHMM-2.0</a>)<br/>Prediction of transmembrane helices in proteins. Should produce results that nearly match Phobius.</li><li>SignalP (free use online: <a href="http://www.cbs.dtu.dk/services/SignalP/" target="_blank">http://www.cbs.dtu.dk/services/SignalP/</a>)<br/>The SignalP 5.0 server predicts the presence of signal peptides and the location of their cleavage sites in proteins from Archaea, Gram-positive Bacteria, Gram-negative Bacteria and Eukarya.</li><li>Simple Modular Architecture Research Analysis (SMART) (free use online: <a href="http://smart.embl-heidelberg.de/" target="_blank">http://smart.embl-heidelberg.de/</a>)<br/>Identifies key structural domains in proteins. Complements Phobius and TMHMM analysis.</li><li>Eukaryotic Linear Motif (ELM) resource (free use online: <a href="http://elm.eu.org/index.html" target="_blank">http://elm.eu.org//index.html</a>)<br/>Annotation and detection of eukaryotic linear motifs (<em>i.e.</em>, regions of the protein sequence that have known functions).</li><li>Webcutter (free use online: <a href="http://heimanlab.com/cut2.html" target="_blank">http://heimanlab.com/cut2.html</a>)<br/>Tool for finding potential restriction sites in a DNA sequence by introducing silent mutations that do not change the protein sequence. This is used to introduce restriction sites in the plasmid during construction that can then be used for downstream modifications of the plasmid.</li><li>Russel Lab (free use online: <a href="http://www.russelllab.org/aas/" target="_blank">http://www.russelllab.org/aas/</a>)<br/><em>Note: If you need to change an amino acid to introduce a restriction site, then check the amino acid proper ties and consequences of substitutions to see which amino acid is the best substitution.</em> <br/><br/></li></ol><p class="ppzz"><strong>Databases</strong> (free to use online)</p><ol class="npwtwo"><li>Genbank (<a href="https://www.ncbi.nlm.nih.gov/genbank/" target="_blank">https://www.ncbi.nlm.nih.gov/genbank/</a>) (Benson <em>et al.</em>, 2013)<br/>Sequence database maintained by the National Center for Biotechnology Information (NCBI), National Library of Medicine (NLM), and National Institutes of Health (NIH)</li><li>Ensembl (<a href="http://ensembl.org/" target="_blank">http://ensembl.org/</a>) (Zerbino <em>et al.</em>, 2018)<br/>Sequence database maintain by the European Molecular Biology Laboratory (EMBL)</li><li>UniProt (<a href="https://www.uniprot.org/" target="_blank">https://www.uniprot.org/</a>) (Consortium, T. U. 2018)<br/>This should be a first stop for basic understanding of any protein of interest. Annotations for human and mouse genes are very detailed and in most cases the proteins for other species have similar features (<em>e.g.</em>, locating the extracellular and transmembrane domains within the protein). Commercial suppliers are also useful for cross-referencing UniProt and your own protein analysis results (<em>e.g.</em>, RnD Systems).</li></ol><br/><p class="ppzz"><em><strong>Note: We have found that for species with limited information (e.g., Tasmanian devils), a de novo transcriptome assembly is extremely useful.</strong> The initial genome assemblies for many species are incomplete and/or inaccurate, so Genbank and Ensembl may not contain the correct sequence for your GOI. A de novo assembly of RNA sequencing data can provide accurate full gene transcripts that can be used to cross-check with Genbank and Ensembl sequences. Furthermore, the use of a de novo transcriptome assembly allows this protocol to be applied in species where reference genome assemblies are not available. RNA sequencing data can obtain with single-read or paired-end protocols, although paired-end data is recommended for greater sequence confidence. A transcriptome from peripheral blood cells, spleen, or lymph node should yield most of the immune system related genes. A de novo transcriptome assembly requires deeper bioinformatics skills than the rest of this protocol, so for teams without this expertise we recommend finding an experienced collaborator in the first case and then developing your own skills if needed. Please contact Dr Andrew Flies <a href="https://twitter.com/WildImmunity" target="_blank">@WildImmunity</a> for help finding a collaborator with the necessary skills.</em> </p><p class="pptt" id="biaoti34676">### Procedure</p><p class="ppzz">We have separated the protocol into five parts.<br/><strong>Procedure A: Characterize the GOI and POI</strong><br/><strong> Procedure B: Design the expression vector</strong><br/><strong> Procedure C: Assemble the expression vector</strong><br/><strong> Procedure D: Transfect a eukaryotic cell line with the expression vector</strong><br/><strong> Procedure E: Test the recombinant protein<br/></strong> </p><ol class="npwone"><li>Characterize the GOI and POI<ol class="npwtwo"><li>Identify GOI (<em>e.g.</em>, CD200, CD200R1, 41BB, 41BBL, PD1, or PDL1).</li><li>Find approved gene name from the HUGO Gene Nomenclature Committee for your GOI (<a href="http://www.genenames.org/" target="_blank">http://www.genenames.org/</a>).<br/><em>e.g.</em>, <a href="https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/7203" target="_blank">CD200</a> (OX2, OX-2, MOX1, MOX2, MRC)</li><li>Use Ensembl or NCBI Genbank to search for the gene by name.<br/><a href="http://asia.ensembl.org/index.html" target="_blank">http://asia.ensembl.org/index.html</a><br/><a href="https://www.ncbi.nlm.nih.gov/gene" target="_blank">https://www.ncbi.nlm.nih.gov/gene</a> </li><li>Locate the transcript sequence(s) for the gene.<br/>Multiple versions may exist for each gene. <em>e.g.</em>, CD200_X1, X2, X3, X4 (Genbank) or CD200-201, 202 (Ensembl).<br/><em>Note: Transcript sequences in Genbank and Ensembl should match, but we find that they are often different, so it is best to check both databases for your GOI/POI.</em> </li><li>Save the GenBank (.gb) file cDNA transcripts to your computer using the appropriate naming convention (<strong>Figure 2</strong>).<br/><ol class="npwthree"><li>Species code _ approved gene name _ reference code.<br/><em>e.g.</em>, saha_CD200_ENSSHAT00000000104-201 (Ensembl).<br/><em>e.g.</em>, saha_CD200_X1_XM_023500980.1 (GenBank).</li><li>See the species table below for appropriate species code.<br/>First two letters for the Latin genus and species.<ol class="npwfour"><li><em>e.g.</em>, Tasmanian devil: <em>Sarcophilus harrisii</em> (saha).</li><li>Ensembl uses three letter species codes (sha), but we recommend the four letter code (saha).</li></ol></li><li>Ensembl codes start with ENS, then the species code, then either G for gene or T for transcript.<br/><em>e.g.</em>, <em>ENS</em><u>SHA</u><strong>G</strong>00000000091<br/><em>e.g.</em>, <em>ENS</em><u>SHA</u><strong>T</strong>00000000104</li><li>An NCBI account can be created to save your transcripts online.</li><li>The FASTA (.fa or .fasta) sequences are also ok, but the GenBank (.gb) have additional useful information.</li></ol><br/><img alt="" class="layerphoto" height="231" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701190556_8977.jpg" width="600"/><br/><strong>Figure 2. Transcript file export from NCBI GenBank</strong><br/><br/></li><li>Open the files using SnapGene. Note that other software programs can be used.</li><li>If the coding sequence (<em>i.e.</em>, open reading frame (ORF) for the gene) is not annotated, then check Ensembl. If additional information is needed, perform a <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">BLAST</a> search to compare your GOI sequence to other species. This should allow you to determine where the potential start codons (ATG) and stop codons (TAA, TAG, or TGA) are for your GOI.</li><li>Create a new feature in the SnapGene file that indicates where the coding sequence is located.<ol class="npwthree"><li>Highlight the DNA sequence.</li><li>Select Add Feature and set the feature type to âCDSâ for coding sequence.</li></ol></li><li>Copy the amino acid translation (<em>i.e.</em>, protein sequence) of the coding sequence and use this to characterize the protein using freely available analysis software.<ol class="npwthree"><li>Phobius: predicts transmembrane topology and signal peptides (<strong>Figure 3</strong>) (<a href="http://phobius.sbc.su.se/" target="_blank">http://phobius.sbc.su.se/</a>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701190653_4775.jpg" width="450"/><br/><strong>Figure 3. CD200 Phobius protein analysis of devil CD200.</strong> Analysis of Tasmanian devil CD200 using Phobius algorithm to identify the signal peptide, and extracellular domain, transmembrane and intracellular domains, and ICD (aka cytoplasmic). CD200 is predicted to have a weak signal peptide (aka SigP). The TMHMM server can be used for additional analyses, but usually yields results similar to Phobius.<br/><br/></li><li>ProtParam tool on the ExPASy bioinformatics server: predicts molecule weight and extinction coefficients (<a href="http://web.expasy.org/protparam/" target="_blank">http://web.expasy.org/protparam/</a>).<br/>Remove the SigP from the protein sequence for the molecular weight and extinction coefficient analysis, because the SigP should not be present in the final protein product.</li><li><a href="http://www.cbs.dtu.dk/services/SignalP/" target="_blank">SignalP</a>: predicts the signal peptide location and the probability that a peptide will be targeted the secretory pathway (<strong>Figure 4</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701190739_3987.jpg" width="450"/><br/><strong>Figure 4. CD200 SignalP 5.0 protein analysis of devil CD200.</strong> Analysis of Tasmanian devil CD200 using the SignalP 5.0 algorithm to determine the probability of having a signal peptide. The output suggests that the CD200 SigP has a 40% chance of leading to a secreted protein. We used this native CD200 SigP for a CD200-mTagBFP and CD200-mOrange FAST proteins and we recovered an adequate yield of these proteins in supernatant, suggesting the CD200 SigP is sufficient for FAST protein secretion.<br/><br/></li><li>Eukaryotic Linear Motif (ELM) resource: annotation and detection of eukaryotic linear motifs (<a href="http://elm.eu.org/index.html" target="_blank">http://elm.eu.org//index.html</a>).</li><li>Simple Modular Architecture Resource Tool (SMART): identification and annotation of signaling domain sequences (<a href="http://smart.embl-heidelberg.de/" target="_blank">http://smart.embl-heidelberg.de/</a>).<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Save the results from each of the above programs.<ol class="npwthree"><li>Save as either a webpage (html) or PDF.</li><li>Save with the correct file names that matches your SnapGene file name.<ol class="npwfour"><li>saha_CD200_X1_ XM_023500980.1_Phobius</li><li>saha_CD200_X1_ XM_023500980.1_ProtParam</li><li>saha_CD200_X1_ XM_023500980.1_SignalP</li><li>saha_CD200_X1_ XM_023500980.1_ELM</li><li>saha_CD200_X1_ XM_023500980.1_SMART</li></ol></li></ol></li><li>If you donât have a gene table yet, create a spreadsheet and record the key information for the protein so that you have a single reference to lookup key information about your protein. You can also save most of the information in the SnapGene files.</li><li>Update the annotation of the SnapGene file with key information from each prediction tool.<br/><em>Note: It is useful to refer to UniProt during the steps below to see if the POI in your species is similar to the ortholog in humans (<a href="https://www.uniprot.org/uniprot/P41217" target="_blank">https://www.uniprot.org/uniprot/P41217</a>).</em> <ol class="npwthree"><li>Phobius: signal peptide (SigP; <strong>Figure 5</strong>), extracellular domain (ECD; <strong>Figure 6</strong>), transmembrane domain (TMD), intracellular domain (ICD, aka cytoplasmic domain) (<strong>Figure 7</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191136_5381.jpg" width="450"/><br/><strong>Figure 5. Assigning the CD200 signal peptide.</strong> Highlight the SigP sequence identified by SignalP and then choose Feature â Add Feature. When the dialog box opens paste in the name with the sequence reference number and the suffix SigP (<em>e.g.</em>, saha_CD200_X1_ XM_023500980.1_SigP). Change the "type" to sig_peptide.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191156_0025.jpg" width="450"/><br/><strong>Figure 6. Assigning CD200 functional domains.</strong> From the "Sequence" view of the CD200 annotation in SnapGene. To annotate the CD200 sequence, highlight the ECD sequence identified by Phobius and then choose Feature â Add Feature. When the dialog box opens paste in the name with the sequence reference number and the suffix ECD (<em>e.g.</em>, saha_CD200_X1_ XM_023500980.1_ECD). Tick the box for "Translate this feature in sequence view". Repeat this process for the TMD and ICD. You can also add features for the 5' untranslated region (5' UTR) and 3' untranslated region (3' UTR).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191219_8031.jpg" width="450"/><br/><strong>Figure 7. Final devil CD200 annotation domains.</strong> Linear "map" view of CD200 annotation in SnapGene. Map view of the Tasmanian devil CD200 transcript, including the 5â and 3â UTR, and the predicted SigP, ECD, TMD, and ICD.<br/><br/></li><li>SignalP: Prediction of signal peptides and secreted proteins.</li><li>ProtParam: molecular weight in kilodaltons (kDA).</li><li>ELM: key motifs (<em>e.g.</em>, Immunoreceptor tyrosine-based inhibition motif (ITIM), Immunoreceptor tyrosine-based activation motif (ITAM).</li><li>SMART: Immunoglobulin (Ig) or Tumour necrosis factor (TNF) domains.<br/></li></ol></li><li>Make sure to save your SnapGene file. You can also export it as a GenBank (.gb) file as a backup.</li><li>In SnapGene, copy the amino acid sequence and create a new protein sequence in CLC Sequence Viewer.<ol class="npwthree"><li>Include the full sequence name to match your SnapGene file.<br/><em>e.g.</em>, saha_CD200_X1_ XM_023500980.1</li><li>Include the common name.<br/><em>e.g.</em>, devil</li><li>Include the Latin species name.<br/><em>e.g.</em>, Sarcophilus harrisii</li></ol></li><li>Create a new folder for each gene.<br/><em>e.g.</em>, CD200</li><li>Save the file into the correct gene folder.<br/><br/></li></ol><p class="ppzz"><strong>Compare the POI across species</strong> </p><ol class="npwtwo" start="17"><li>Use the NCBI âGeneâ database to find the name of your GOI (<em>e.g.</em>, CD200). The gene page will have an âorthologsâ link that will show all published orthologs (<em>i.e.</em>, the same gene in different species) for the gene.</li><li>Use NCBI blast to find homologs for the species listed in the gene table below.</li><li>Download multiple sequences simultaneously (tick boxes for matched genes that are relevant).</li><li>Save the files to your computer or NCBI account.</li><li>Copy the protein sequences into CLC Sequence Viewer and create a protein alignment (<strong>Figure 8</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191302_3296.jpg" width="450"/><br/><strong>Figure 8. Multispecies protein alignment of CD200.</strong> The black bar graphs below the alignment represent the percent conservation of amino acids across eight species. The RasMol coloring scheme is used to highlight traditional amino acid properties (<em>e.g.</em>, green = aliphatic amino acids leucine, isoleucine valine), which aids in interpretation of amino differences across species.<br/><br/></li><li>Determine if key structural features of the protein are conserved among species.<ol class="npwthree"><li><em>e.g.</em>, cysteines that form disulfide bonds</li><li><em>e.g.</em>, conserved signal peptide and transmembrane domain</li></ol><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Design expression vector</li><ol class="npwtwo"><li>Choose an expression vector (aka plasmid) for the gene.<br/>Here we will demonstrate the process by inserting the Tasmanian devil CD200 analyzed above into our pAF164 vector. The FAST protein vectors were constructed using Sleeping Beauty plasmids available from <a href="https://www.addgene.org/browse/article/9749/" target="_blank">Addgene and the Erik Kowarz lab</a> (Kowarz <em>et al.</em>, 2015). We will cut the saha_TNFRSF9_ XM_003765221.2-ECD DNA sequence out of the pAF164 vector using restriction enzymes, and then use Gibson Assembly (<em>e.g.</em>, NEBuilder) to insert the saha_CD200_X1_XM_023500980.1-SigP-ECD.<br/><em>***Critical step***<ol class="npwthree"><li>If you are making a soluble protein, then you need to include only the coding region for the SigP and ECD of your protein (e.g., SigP-ECD) into your expression vector.</li><li>If you are making a full-length cell surface protein, then you want to include the whole ORF for your gene (e.g., SigP-ECD-TMD-ICD) in your expression vector but make sure to remove the stop codon at the end of the ORF so that the ORF continues to the 3'-end of the fluorescent protein coding sequence.</li></ol></em> </li><li>In SnapGene, set the origin of the plasmid to the first base at the 5'-end of the ORI. This creates a standard reference point for all the plasmids in our library.<ol class="npwthree"><li>Open the SnapGene file and place the cursor between the first base of the 5â ORI and the base directly preceding the 5â base.</li><li>View â Set origin.</li></ol></li><li>Find the location of the coding sequence to be replaced (TNFRSF9-SigP-ECD) (<strong>Figure 9</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191339_3973.jpg" width="450"/><br/><strong>Figure 9. Selection of the GOI feature to be replaced by a new GOI.</strong> Map view of the plasmid file and GOI region to be replaced by a new GOI.<br/><br/></li><li>Switch from "Map" view to "Sequence" view.</li><li>Make sure the entire coding sequence for the GOI to be replaced is highlighted (<strong>Figure 10</strong>).<br/><em>***Critical step: Make sure to only highlight the GOI; do not highlight the SmaI/XmaI restriction site following the GOI, TEV site, linker, 6xHis, SalI restriction site, or fluorescent protein (e.g., mOrange).***</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191355_0548.jpg" width="450"/><br/><strong>Figure 10. Sequence view of the GOI region to be replaced by a new GOI</strong><br/><br/></li><li>Switch to the SnapGene file containing your GOI. In this case we will switch to the saha_CD200_X1_XM_023500980.1 file created in <strong>Procedure A</strong>.</li><li>Highlight the saha_CD200_X1_XM_023500980.1 SigP and ECD (<strong>Figure 11</strong>).<br/><em>***Critical step: To make a secreted protein, you need to make sure that only the SigP and ECD of the GOI/POI is included in the new plasmid construct.***</em><br/>If the SignalP algorithm predicts that you POI has a weak SigP, then you might want to replace your SigP with a stronger SigP. We often use the hamster interleukin 2 (mau_IL2_NM_001281629) SigP because we produce our proteins using Chinese hamster ovary (CHO) cells, so the hamster SigP works well. The SigP of the protein is usually cleaved off in the endoplasmic reticulum by a signal peptidase, and thus is generally not included in the mature secreted protein and should not alter downstream function. Basically, a strong SigP will yield a higher protein concentration at the end of this process.<ol class="npwthree"><li>mau_IL2_NM_001281629_SigP DNA: <br/>atgtacagcatgcagctcgcatcctgtcttgcactgacgctcgcactccttgtcagcagt</li><li>mau_IL2_NM_001281629_SigP protein: MYSMQLASCLALTLALLVSS</li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191415_8208.jpg" width="450"/><br/><strong>Figure 11. SigP-ECD of the POI.</strong> Map view of the DNA file for the GOI to be inserted into the expression vector. Make sure to select the SigP + ECD. If you GOI does not have a SigP, then you might consider using the SigP from a different gene. We usually use the hamster IL-2 SigP because it works well in the CHO cells that we use for producing soluble proteins.<br/><br/></li><li>Switch back to the pAF164 plasmid file.</li><li>Rename the file so you do not destroy the previous file.<br/>pAF176_OFP is the devil CD200-mOrange FAST protein</li><li>Paste the copied saha_CD200_X1_XM_023500980.1_SigP-ECD where the previous GOI is in the pAF176 file (<strong>Figure 12</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191432_2732.jpg" width="450"/><br/><strong>Figure 12. New CD200 expression vector (pAF176).</strong> The POI is highlighted in blue. Note that the stop codon at the 3'-end of the fluorescent protein should be the only in-frame stop codon in the entire ORF (5'-end SigP to the 3'-end of mOrange).<br/><br/></li><li>Update the pAF176_ORF feature to run from the ATG start codon at the start of the SigP to the TAG stop codon at the end of mOrange (<strong>Figure 13</strong>).<br/>For pAF176 with saha_CD200_X1_XM_023500980.1 fused to mOrange, the exact DNA locations should be 4246 to 5745. This should yield a 1497 bp ORF and a 690 amino acid protein. Note that the number of base pairs in an ORF should always be divisible by three because each amino acid requires a three base pair codon. If your ORF is not divisible by three, then the protein might be out-of-frame and will not produce the correct protein.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191451_3075.jpg" width="450"/><br/><strong>Figure 13. Update ORF to match the ATG start codon for the newly inserted gene</strong><br/><br/></li><li>Examine the updated pAF176 sequence to make sure the complete ORF is in frame. For example, the ORF for the saha_CD200_X1_XM_023500980.1 mOrange FAST protein N-terminus (<em>i.e.</em>, CD200 start) should have MELQV (<strong>Figure 14</strong>) and the C-terminus (<em>i.e.</em>, mOrange end) should have DELYK*. The '*' indicates the stop codon.<br/><em>***Critical step: Make sure there are no early stop codons in your sequence. This can arise by including the ICD of your GOI in the construct, which will result in the linker and fluorescent reporter protein not being translated.***</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191510_8048.jpg" width="450"/><br/><strong>Figure 14. N-terminus of the new ORF.</strong> If you are using Tasmanian devil CD200, then make sure that the N-terminus matches the MELQV amino acid sequence shown here. If you have inserted a different gene, the sequence will be different, but the start location should be the same (position 4246 if you have âSet originâ correctly in Step B2b).<br/><br/></li><li>At this point you have a full <em>in silico</em> plasmid construct. Next, we need to develop a method to get the actual DNA for our GOI into this construct. There are two options. The first and easiest method is to order a synthetic double stranded DNA block (<em>e.g.</em>, gBlock) from a commercial supplier (<em>e.g.</em>, IDT DNA). If you are confident that you know the exact sequence for your GOI, then we recommend this option. If you are not 100% confident about your DNA sequence, then it might be better to use PCR to amplify your GOI. Both methods require that the DNA sequence to be inserted into the expression vector has at least 20 bp overlaps with the insertion site in the expression vector. In the supplied pAF164 vector, we have annotated the overlap regions as "Gibson_5'_overlap" and "Gibson_3'_overlap". The pAF176 vector that contains the devil CD200 sequence is available from the authors of this protocol if you want to use it to practice replicating a working construct before embarking on your own sequences.<ol class="npwthree"><li>gBlock method: Highlight the Gibson_5'_overlap + SigP + ECD + Gibson_3'_overlap and create a new feature called pAF176_gBlock.<br/>The gBlock should be 778 bp for this example.</li><li>PCR method: Create a forward and reverse primer to amplify the insert from cDNA from your species (<strong>Figure 15</strong>).<ol class="npwfour"><li>The primers will contain 18-30 bases that match the 5'-end of your GOI. The annealing (<em>i.e.</em>, melting) temperature for the match to your GOI should be at least 55 Â°C. For the pAF176 construct, the match to saha_CD200_X1_XM_023500980.1 can be 21 bp (~57 Â°C): atggagctgcaagtacatgag.</li><li>The primers also need to contain extensions to add the vector overlap regions during each cycle of PCR amplification.</li><li>Use the cursor to highlight the 5'-end of the Gibson_5'_overlap through the end of your intended match for your GOI primer (59 bp total): gtgaaaactaccccaagctggcctctgcggccgccaccatggagctgcaagtacatgag.</li><li>Note that the general principles of primer design should be adhered to, but we find that most primers will work on the first try. We try to have our primers end in a C or G and avoid having the primer end in duplicates (<em>e.g.</em>, GG or CC) or triplicates, but these are acceptable if the GOI has long repeats and there is no other option.</li><li>Repeat steps i-iv for constructing the reverse primer for the 3â-end of your GOI.</li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191533_8217.jpg" width="450"/><br/><strong>Figure 15. Gibson assembly primers.</strong> Forward primer for amplifying GOI, flanking the region and providing sequence overlap with the vector backbone.</li></ol><br/></li><li>Order the gBlock or primers from a commercial supplier and proceed to Procedure C.<br/><br/></li></ol><li>Assemble the expression vector<br/><em>Notes: <ol class="npwthree"><li>We will essentially use the Gibson Assembly process (Gibson et al., 2009), but we generally use the NEBuilder kit.</li><li>The full set of primers, gBlocks, and plasmids are available in the supplementary material from the publication associated with this protocol (Flies et al., 2020).</li></ol></em><br/><u>Prepare the DNA insert for assembly into the expression vector using Gibson assembly</u><br/><strong>exp_ID_1: Insert_PCR tab in the</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<br/><ol class="npwtwo"><li>Before opening the vials containing gBlocks or primers, spin the vials at 10,000 <em>x g</em> for one minute to ensure all lyophilized DNA is at the bottom of the vial.</li><li>Dilute the gBlock and/or primers to 100 Î¼M in sterile deionized distilled water (ddH<sub>2</sub>O). TE buffer can also be used, but ddH<sub>2</sub>O has worked well for us.</li><li>If you are inserting a gBlock into the expression vector, then skip to Step C14.</li><li>Perform touchdown PCR to amplify and extend your target GOI.<br/>The extension primers will have a low annealing temperature (~60 Â°C) during the initial amplification because only about ~50% of the primers bind to the target DNA (<strong>Figure 16</strong>). After the target DNA has been amplified the primer annealing temperature will be higher (<em>e.g.</em>, 68 Â°C). To account for the potential trade-off between target specificity and the difficulty of amplifying rare target transcripts we use a touchdown PCR. We start at a high annealing temperature and decrease the temperature by 1 Â°C for each of the first ten cycles. Then we run an additional 25-35 cycles at a higher annealing temperature to increase specificity.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191608_4767.jpg" width="450"/><br/><strong>Figure 16. Overlap extension PCR to add the 5â and 3â overlaps onto the GOI for assembly into the expression vector</strong><br/><br/></li><li>Set up PCR reaction using high-fidelity polymerase. We use New England Biolabs Q5<sup>Â®</sup> Hot Start High-Fidelity 2x Master Mix.</li><li>While the PCR is running pour a 1% TAE agarose gel with nucleic acid stain.</li><li>When the PCR is finished, dilute 20 Î¼l of reaction mix with gel loading buffer.</li><li>Add ~20 Î¼l of reaction + bromophenol blue gel loading buffer to the appropriate lanes on the gel.<br/><em>Note: You will need a deep well to add a 20 Î¼l volume, but it is good to use as much DNA as possible because DNA recovery from gels can be low.</em> </li><li>Run for ~30 min at 100 V in 1x TAE buffer.</li><li>Visualize the gel on a blue light transilluminator.<br/>If a blue light transilluminator is not available, a UV light source can be used. Care should be taken to limit DNA exposure to UV light as this can damage DNA and be a considerable source of downstream cloning problems. Use a sterile razor blade or plastic cutting instrument to cut around the target DNA band. Make sure not to cut directly on the glass of your UV/blue light as this can damage the glass. Use a new blade for each band to prevent carry over of DNA between samples.</li><li>Purify the DNA using a commercially available DNA gel purification kit.</li><li>Quantify the DNA on a NanoDrop spectrophotometer using the A<sub>280</sub> DNA setting.</li><li>Dilute the DNA to 10 ng/Î¼l using TE and store the DNA at -20 Â°C until needed.</li></ol><br/><u>Prepare the plasmid DNA for assembly into the expression vector</u><br/><strong>exp_ID_2: Plasmid_Digest</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<br/><ol class="npwtwo" start="14"><li>Thaw plasmid pAF137 (or other vector as needed) and store on ice.</li><li>Thaw restriction enzymes on ice (<em>e.g.</em>, NotI-HF, SmaI).<br/><em>Note: XmaI and SmaI both have the same recognition site, but SmaI yields a blunt cut and is thus less likely to re-anneal during the Gibson assembly process.</em> </li><li>Mix the DNA and restriction enzymes according to the manufacturerâs instructions.<br/><em>***Critical step: Incomplete digests are a major source of downstream issues so ensuring a clean digest is critical.***</em><br/>After we have confirmed a clean cut by using the digest in a successful assembly, we keep the digest and reuse for future assemblies.</li><li>Option 1: Prepare a 1% agarose gel with nucleic acid stain capable of holding 60 Î¼l of reaction mix + bromophenol blue loading buffer. We generally accomplish this by placing a piece of tape between two of the teeth in the comb when setting the gel.<br/>Option 2: Use 2-4 wells to load the entire 60 Î¼l volume. You will have to cut several bands from the gel instead of one band if you use option 2.</li><li>Add the digested DNA to gel. Always include undigested plasmid as a control and a DNA ladder to estimate the size of bands.</li><li>Run for ~30 min at 100 V in 1x TAE buffer.</li><li>Visualize the gel on a blue light transilluminator (<strong>Figure 17</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191725_8202.jpg" width="450"/><br/><strong>Figure 17. Image of digested and undigested plasmid DNA migration in agarose gel electrophoresis.</strong> Lane 1 shows the DNA ladder and lane 2 shows the plasmid in its native state (<em>i.e.</em>, no restriction digest). Lane 3 shows a plasmid that has been digested with two different restriction enzymes. When each enzyme cuts only one site in the plasmid, this results in two fragments. The upper band needs to be excised and purified and then used to assemble your new plasmid. The lower band is the fragment that needed to be removed from your plasmid prior to assembly. Lanes 4 and 5 show plasmids that have been digested with a single restriction enzyme. Note that circular DNA (lane 2) travels through the agarose gel faster than linear DNA (lanes 4 and 5) even when they are equivalent size. Image adapted from Addgene (<a href="https://blog.addgene.org/plasmids-101-how-to-verify-your-plasmid" target="_blank">https://blog.addgene.org/plasmids-101-how-to-verify-your-plasmid</a>).<br/><br/></li><li>Purify the DNA using a commercially available DNA gel purification kit.</li><li>Quantify the DNA on a NanoDrop spectrophotometer using the A<sub>280</sub> DNA setting.</li><li>Store the digested DNA at -20 Â°C until needed.</li></ol><br/><u>Assemble the expression vector</u><br/><strong>exp_ID_3: Plasmid_Assembly</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<br/><ol class="npwtwo" start="24"><li>Calculate the amount of plasmid DNA, insert DNA, and Gibson assembly master mix (<em>e.g.</em>, <a href="https://www.nebiolabs.com.au/protocols/2014/11/26/nebuilder-hifi-dna-assembly-reaction-protocol" target="_blank">NEBuilder</a>) reagent needed.<ol class="npwthree"><li>We generally use a 1:3 ratio of plasmid:insert for single inserts less than 1,000 bp.</li><li>We also use 50 ng of digested plasmid DNA in 10 Î¼l reactions instead of the 20 Î¼l assemblies that are recommended by the manufacturer.</li></ol></li><li>Thaw NEBuilder HiFi DNA assembly master mix on ice.</li><li>Add the DNA and reagents to the mix.</li><li>Use NEBuilder positive control (provided by the manufacturer) for your positive control.</li><li>Incubate at 50 Â°C for 60 min for single inserts. 15 min is sufficient, but 60 min seems to work better.</li><li>Store samples on ice or freeze until needed. It is best to perform assembly within one day, but assemblies stored for up to 14 days can still work.</li></ol><br/><u>Transform assembled expression vector in bacteria for plasmid amplification</u><br/><em>Note: Use a separate set of pipets for bacterial work and for sterile cell culture work.</em><br/><ol class="npwtwo" start="30"><li>Transfer bacterial growth plates with LB agar and 100 Î¼g/ml of ampicillin to a 37 Â°C incubator to pre-warm them.</li><li>Use DH5Î± competent bacteria in 1.5 ml microfuge tubes for transformation.</li><li>Thaw the bacteria on ice and be prepared to add the assembled plasmid to the bacteria as soon as they are thawed (2-5 min).</li><li>Add 2 Î¼l (10 ng) of the DNA assembly directly to the bacteria and pipet cells up-and-down 5x to mix.<br/>Gently tap the vial to ensure the DNA is thoroughly mixed with the bacteria.</li><li>Immediately place the vial on ice and incubate for 30-45 min.<br/>The next step requires a rapid transition from ice to 42 Â°C and back to ice. Either set a microfuge heat block for 42 Â°C or prepare a water bath at 42 Â°C. We have achieved more consistent results using a water bath. If a water bath is not available, you can use any liquid container, fill it with warm water and adjust the temperature to 42 Â°C immediately prior to the heat shock step.</li><li>Transfer the bacteria vials from the ice to 42 Â°C for 30 s.</li><li>Transfer back to ice for at least two min.</li><li>Add 950 Î¼l of SOC media to each tube.</li><li>Close the tubes tightly.</li><li>Transfer tubes to an orbital shaker, set to at least 180 rpm. Storing the tubes horizontally or at a 45-degree angle helps to ensure proper mixing.</li><li>Incubate on an orbital shaker at 37 Â°C for 60 min.</li><li>Spin transformed bacteria at 8,000 <em>x g</em> for 30 s.</li><li>Resuspend the bacteria in 100 Î¼l of LB media with 100 Î¼g/ml of ampicillin and transfer to the LB agar plates with ampicillin.</li><li>Spread the bacteria evenly on half of the plate using a bacteria spreader.</li><li>Use an inoculating loop to streak bacteria to the other half of the plate. This approach should yield individually recognizable colonies even with high and low transfection efficiencies.</li><li>Store the residual bacteria in the vial at 4 Â°C to use in case of downstream problems.</li><li>Incubate the plates overnight at 37 Â°C.</li></ol><br/><u>Perform colony PCR to determine if insert is present in bacteria</u><br/><strong>exp_ID_4: Colony_PCR</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<ol class="npwtwo" start="47"><li>Examine the bacteria plates within 18 h of plating the bacteria. Count the number of colonies on each plate.<br/><em>***Critical step: A few bacterial colonies on the negative control plates is ok if the number of colonies on your new assemblies is at least 5x higher. The positive control plate should have too many colonies to count.***</em> </li><li>Determine the number of colonies to test.</li><li>Create a colony PCR protocol (see âcolony_PCRâ tab in protocol spreadsheet).</li><li>Use pSB_EF1a_seq.FOR and pSB_bGH_seq.REV to amplify the entire ORF of the FAST protein coding sequences. The forward primer binds to the EF1Î± promotor upstream of the GOI and the reverse primer binds to the bGH UTR region downstream of the fluorescent protein coding sequence.</li><li>Use SnapGene to calculate the expected size of the amplicon based on the length of the amplified DNA and primers.</li><li>Label PCR tubes to match the protocol.</li><li>Create an extra set of PCR tubes to match the bacterial colonies to be tested. Label the tubes as âbacteria tubesâ.</li><li>Add 10 Î¼l of sterile PBS to each of the bacteria tubes.</li><li>Thaw the primers and OneTaq<sup>Â®</sup> Hot Start Quick-Load<sup>Â®</sup> 2x Master Mix with Standard Buffer (NEB # <a class="blackNoUnder" target="_blank"> M0488L </a> ).<br/>OneTaq Master Mix contains gel loading buffer, so do not add additional loading buffer prior to running the gel.</li><li>Mix the PCR reagents according to the protocol in the âcolony PCRâ spreadsheet.</li><li>Distribute the PCR mix to the colony PCR tubes.</li><li>Use a filtered pipet tip to gently touch one of the selected bacterial colonies.<br/><em>***Critical step***<ol class="npwthree"><li>Make sure to touch only a single colony and do not take the entire colony. Mixing two colonies will lead to failure. Also, the colony might have to be used to start a new culture if other problems arise, so leave enough bacteria in the colony to inoculate a new culture.</li><li>Try not to get agar on the tip, as too much can overload the PCR. Even a single bacterium should have enough DNA for the PCR.</li></ol></em> </li><li>Insert the pipet tip with the bacteria into the appropriate âbacteria tubeâ that contains 10 Î¼l of sterile PBS. Pipet up-and-down several times and drag the tip around the tube to ensure that some of the bacteria remain in the PBS and some of the bacteria remain on the tip. These tubes are the bacterial stock for seeding an overnight culture if the colony PCR yields positive results.</li><li>Directly transfer the tip with the bacteria to appropriate colony PCR tube. Pipet up-and-down several times and drag the tip around the tube to ensure that some bacteria are transferred into the colony PCR tube.</li><li>Discard the tip into bleach and repeat for each additional colony.</li><li>Run the PCR reaction for 30-35 cycles with an annealing temperature of 55 Â°C. See experiment template for additional details.</li><li>Pour a 1% TAE agarose gel with nucleic acid stain while the PCR is running.</li><li>Transfer 10 Î¼l from colony PCR tubes into the appropriate wells on the gel.</li><li>Run for ~30 min at 100 V in 1x TAE buffer.</li><li>Record an image of the gel.</li><li>Determine which lanes/tubes had the correct size amplicon (<strong>Figure 18</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701191957_4459.jpg" width="450"/><br/><strong>Figure 18. Example of a colony PCR agarose gel electrophoresis.</strong> The three strong bands indicated match the expected size of a plasmid that contains the desired insert. Select 5 of the bacterial colonies that match these results and culture them overnight (not more than 16 h) in 5 ml of LB media with ampicillin at 37 Â°C and shaking at 200 rpm.<br/><br/></li><li>Set up overnight cultures for colonies that yielded amplicons with the correct size by transferring 5 ml of LB media with 100 Î¼g/ml of freshly thawed ampicillin into a sterile 50 ml tube.</li><li>Transfer 5 Î¼l from the âbacteria tubesâ into the appropriate 50 mL tubes to inoculate the overnight culture.<ol class="npwthree"><li>Inoculated untransformed bacteria can be used as a negative control (<em>i.e.</em>, untransformed bacteria should not have ampicillin resistance). The NEBuilder positive control cells or any bacteria with ampicillin resistance can be used as a positive control.</li><li>Incubate at 37 Â°C overnight on an orbital shaker with at least 180 rpm.</li></ol></li><li>After overnight incubation (less than 16 h) check to see if the bacteria have proliferated. A turbid (<em>i.e.</em>, cloudy) tube indicates that the bacteria have proliferated.<br/><div style="white-space:nowrap;"><br/></div></li></ol><u>Miniprep to isolate plasmid DNA from bacteria</u> <ol class="npwtwo" start="71"><li>Label 2 ml microfuge tubes for the tubes that show bacterial proliferation.</li><li>Transfer 1.5 ml of bacterial broth to the appropriate microfuge tube.</li><li>Spin at 8,000 <em>x g</em> for 30 s.</li><li>Remove the supernatant with a pipette.</li><li>Transfer another 1.5 ml of the same bacterial broth(s) to their respective tubes and spin and remove supernatant again. If the bacteria produce a high copy number of the plasmid, then this second step is not needed.</li><li>Follow the manufacturerâs instructions for the miniprep procedure.<br/>Ensuring complete lysis of the bacteria to release the plasmid DNA is critical to getting a high yield of plasmid DNA.</li><li>Quantify the DNA using the A<sub>280</sub> DNA setting on a NanoDrop spectrophotometer.<br/>Blank the spectrophotometer with the elution buffer provided with the plasmid DNA kit.</li><li>Store the plasmid DNA in the freezer until you are ready to sequence the DNA.</li></ol><br/><u>Sequencing plasmid DNA</u><br/>We strongly recommend that you use a commercial DNA sequencing service provider to sequence your DNA. For example, in Australia there are many Australian Genome Research Facility (AGRF) collection points where DNA with sequencing primers can be dropped off. Sequencing results are usually returned within 1-2 days. If your institution has a DNA sequencing machine, contact the manager of the machine to get the correct protocol and advice. The protocol below is specific for using an Applied Biosystems 3400 Genetic Analyzer with Big Dye Terminator (BDT). Other reagents and sequencing machines will have different protocols.<br/><br/><strong>exp_ID_5: Plasmid_Sequencing</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<br/><ol class="npwtwo" start="79"><li>Label PCR tubes for the sequencing PCR.<br/>PCR for sequencing requires only a single primer, as opposed to normal PCR that requires a forward and reverse primer. However, you will want to sequence the plasmid insert from both the forward and reverse orientation, so two reactions are needed for each plasmid.<ol class="npwthree"><li>pSB_EF1a_seq.FOR: Binds upstream of the DNA insert.</li><li>pSB_bGH_seq.REV: Binds downstream of the DNA insert.</li></ol></li><li>Retrieve primers, plasmids, sequencing buffer, and the BDT on ice.<ol class="npwthree"><li>Primers and plasmids are stored at -20 Â°C.</li><li>Sequencing buffer is stored at 4 Â°C.</li><li>BDT is stored in 10 Î¼L aliquots at -80 Â°C.<br/>Purchasing a BDT kit is a relatively large investment, so hopefully you can just use a sequencing service instead of doing the whole process yourself.</li></ol></li><li>Prepare the sequencing mix.<br/>Make a little extra to make sure you donât run out on the last tube.<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td><br/></td><td><u>1x vol (Î¼l)</u><br/></td></tr><tr><td>Sequencing Buffer<br/></td><td>1.75<br/></td></tr><tr><td>Big Dye Terminator<br/></td><td>0.25<br/></td></tr><tr><td>DNA Î¼l (100-500 ng for plasmid)<br/></td><td>X<br/></td></tr><tr><td>Primer<br/></td><td>1<br/></td></tr><tr><td>Add H<sub>2</sub>O to 10 Î¼l<br/></td><td>10 - 1.75 - 0.25 - X - 1<br/></td></tr></tbody></table></li><li>Aliquot ddH<sub>2</sub>O into PCR tubes.</li><li>Aliquot DNA into PCR tubes.</li><li>Aliquot primer into PCR tubes.</li><li>Mix total BDT + sequencing buffer and aliquot into PCR tubes (2 Î¼l/tube).</li><li>Run sequencing reaction on thermal cycler. <ol class="npwthree" start="4"><li>Denature at 96 Â°C for 10 s.</li><li>Anneal at 55 Â°C for 5 s.<br/>50 Â°C is recommended but the annealing temperatures are generally higher, so we usually use 55 Â°C.</li><li>Extend at 60 Â°C for 4 min.</li><li>Run for 35 cycles <br/>25 cycles should work, but usually results in a residual 'dye front' in the results around 70 bp from the 5âstart region.</li></ol></li></ol><br/><u>Cleaning your sequencing reaction with Agencourt CleanSEQ</u> <ol class="npwtwo" start="87"><li>Vortex the CleanSeq tube to resuspend any magnetic particles that may have settled.</li><li>Add 10 Î¼l of CleanSeq to each sample.</li><li>Add 42 Î¼l of 85% ethanol to the sample(s) according to the table below.</li><li>Mix reaction thoroughly by pipet mixing or by replacing the lids and inverting several times. The mixture should be homogenous after mixing.</li><li>Place samples onto a magnetic plate for 3 min. Wait for the solution to clear before proceeding to the next step.</li><li>Aspirate the cleared solution and discard. This must be performed while on the magnetic plate.</li><li>Do not disturb the ring of beads stuck to edge of the tube.<br/><em>***Critical step: Pipet with great care to avoid carrying over beads into your sequencing assay.***</em> </li><li>Add 100 Î¼l 85% ethanol.</li><li>Incubate for 30 s.</li><li>Aspirate and discard the ethanol.</li><li>Repeat Steps 95-97 once.</li><li>Dry samples at room temperature for at least 15 min.<br/><em>***Critical step: Ensure the samples can fully dry; residual ethanol will affect results.***</em> </li><li>Add 30 Î¼l of water and mix using a pipet or again by adding the caps and inverting several times. Incubate the plate for 2 min at room temperature.</li><li>Remove 15 Î¼l of the clear liquid, avoiding any beads, and add it to the 96-well sequencing plate. Be careful not to disturb the ring of beads.</li><li>Denature the clean sequencing products by placing the plate in the thermal cycler (with the grey septa lid in place) and heating at 95 Â°C for 5 min. Make sure the heated lid function is switched off.</li><li>The reaction plate is now ready to be loaded into the sequencer.</li><li>Kindly ask the manager of the sequencing machine to setup your first protocol. <br/><div style="white-space:nowrap;"><br/></div></li></ol><u>Analyze sequencing results</u><br/>This protocol shows how to analyze your results using SnapGene, but many other programs, including the Genomics Workbench Sequence Viewer, can also be used.<ol class="npwtwo" start="104"><li>Export the .abi files from the sequencing machine and save the files in your experiment data folder.</li><li>Other information is available about the sequencing run, but the .abi files provide the chromatograms for assessing quality and aligning to your plasmid files.</li><li>Open the SnapGene file for your plasmid.</li><li>Click the alignment button.</li><li>Click âAlign imported sequencesâ.</li><li>Locate your .abi chromatogram files.</li><li>First examine the âMap viewâ alignment to get an idea of where your sequencing results align relative to the whole plasmid.</li><li>Next click the âSequence viewâ tab to see the DNA alignments.<br/><em>***Critical step: Your sequence needs to be perfect. If your sequencing results show a substitution, insertion, or deletion, you need to pick a new bacterial colony from your agar plates and do another colony PCR, overnight bacterial culture, plasmid purification, and sequencing reaction.***</em><br/><em> Note: The start and end of the chromatogram can be messy, so seek advice from more experienced colleagues to interpret the results from these regions if needed.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transfect a eukaryotic cell line with the expression vector<br/>Now that you have assembled and sequenced your plasmid expression vector, the plasmid DNA can be used to generate your POI. The Sleeping Beauty transposon system (learn more at <a href="https://blog.addgene.org/sleeping-beauty-awakens-for-genome-engineering" target="_blank">https://blog.addgene.org/sleeping-beauty-awakens-for-genome-engineering</a>) within the FAST protein vector should be effective in many different cell lines, but considerations should be made when selecting a cell line to make your protein. The all-in-one vector contains the transposon cassette (<em>i.e.</em>, the POI to be expressed in the cell line) and the transposase (<em>i.e.</em>, the enzyme that inserts the cargo into the genome of the cell), so you only need to transfect one plasmid rather than separate plasmids for the transposon and transposase. Once inside the cell, the cytomegalovirus promoter (CMV) will drive high expression of the transposase enzyme. The transposase enzyme then binds the inverted terminal repeats of the transposon cassette and cuts the cassette out of the plasmid. The remaining plasmid DNA is then degraded. The cassette is inserted between 'TA' dinucleotides in the host cell genome, so the insertion site could be almost anywhere, and multiple copies can be integrated into a single cell. Excess transposase should be degraded over time as part of the normal cell regulatory processes.<br/>  Cassettes that successfully integrate into the genome are permanently integrated. As the insertion site can be anywhere in the genome that contains a 'TA' dinucleotide, the insertion site could affect cell function. For example, if the cassette inserts into the coding region of a host cell gene that is critical for cell division, the host cell will no longer divide and should eventually die. Alternatively, the cassette could insert into an inactive region of the genome (<em>e.g.</em>, highly methylated), which could lead to reduced expression of the POI. However, we have performed RNAseq on a few of our Sleeping Beauty cell lines, and the transcript for the GOI is generally among the most highly expressed transcripts in the cell (<em>i.e.</em>, the cells are making a lot of this RNA).<br/>  We use CHO cells for making most of our secreted proteins. We also use CHO cells for initial testing of cell surface proteins because these cells are reported to not express Fc receptors, which could bind to FAST proteins with a Fc region and to secondary antibodies in downstream experiments. CHO cells are used to make most proteins for clinical use and yield protein glycosylation patterns similar to other mammalian cell lines (Zerbino <em>et al.</em>, 2018). A basic adherent CHO-K1 cell line can be used for all aspects of this protein production. We have switched to a suspension ExpiCHO cell line that can be grown at high density (5 x 10<sup>6</sup> cells/ml) in protein-free media. We suggest starting with a low passage CHO-K1 cell line to keep things simple and cheap.<br/>  When you move into functional testing of your proteins you will likely want to use a cell line from your study species. We have made dozens of devil facial tumor cell lines using this system. We have also transfected human (<em>e.g.</em>, HEK-293, K562) and dog (A-72, DH82) cell lines. A similar Sleeping Beauty system has been used to make chimeric antigen receptor T cells (CAR-T) for human clinical trials, so the system should work for immune cells, however we have not attempted this yet.<br/>  After a quick batch of protein, you can collect the culture supernatant for 3-7 days after transfection and use it to test your protein. A good yield of protein will require high transfection efficiency (<em>i.e.</em>, at least half of the cells in the flask have the transposon cassette integrated into the genome). Your GOI and subsequent POI can be expressed directly from the plasmid if it fails to integrate into the genome. However, the non-integrated plasmid DNA will be degraded within a few days. We generally create stable cell lines instead of collecting protein from the initial transfection. Unless your transfection is 100% efficient, which is very unlikely, you will need to either sort your cells or use drug selection to make a stable cell line. Sorting cells based on the fluorescent protein expression is fast because the sort can be done the next day, but we have observed high cell death when we sort cells soon after the transfection. Thus, we add a high concentration of a selection drug (<em>e.g.</em>, hygromycin) as soon as the fluorescent reporter is visible by microscopy or by taking an aliquot of your cells and running them through a flow cytometer. We have used hygromycin, geneticin (G418), blastomycin, and puromycin for selection. All of these selection drugs work, but we have found that the hygromycin selection is fast and complete. As many of these drugs kill cells by damaging DNA, expression of your drug resistance gene needs to be high to avoid unnecessary DNA damage. This is critical if you are making a cell line for functional testing.<br/>  We use polyethyleneimine (PEI) at a 3:1 ratio with DNA (3 Î¼g PEI for each 1 Î¼g DNA) for most of our transfections. It is among the cheapest of transfection reagents, can be freeze/thawed many times, and can be stored for long periods of time at -20 Â°C. Our transfection efficiency is generally 5-30%, which is considered low by some standards, but the rapid proliferation of the CHO cells allows us to develop a fully selected cell line within 5-10 days in most cases.<br/>â<br/><u>Prepare the PEI transfection reagent using the heat method (see manufacturer for alternatives)</u><br/>Protocol from Polysciences Inc. (Protocols for product # 23966âlinear polyethyleneimine 25,000)<ol class="npwtwo"><li>Weigh 500 mg to 2 g of PEI powder.</li><li>Suspend PEI in beaker with 500 ml to 2 L of water (to final concentration of 1 mg/ml).</li><li>Cover top of container and heat to 80 Â°C and stir for three h, or until solution is mostly clear.</li><li>Cool the solution to room temperature.</li><li>Adjust the pH of the solution to 7.0.</li><li>Sterile filter through 0.22 Âµm membrane</li><li>Aliquot to desired volumes.</li><li>Store aliquots at -20 Â°C.</li></ol><br/><u>Transfection of mammalian cells</u><br/><strong>exp_ID_6: Transfection</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<br/><em>Note: If your plasmids do not contain a fluorescent reporter protein, then we suggest obtaining at least one verified plasmid that does contain a fluorescent reporter protein. This will allow for easy monitoring of the transfection efficiency and drug selection by microscopy or flow cytometry.</em><br/><em> ***Critical step: We strongly suggest making additional cell lines that express only the base vectors (pAF112, pAF123) that can be used as negative controls in downstream testing.***</em><br/><br/><u>Day 0: Thaw cells for transfection</u><br/><ol class="npwtwo" start="9"><li>Thaw at least 1 x 10<sup>6</sup> CHO-K1 cells.</li><li>Add to T75 flask in cRF10.</li><li>Culture overnight and replace media with fresh cRF10.<br/><em>Note: If the cells are approaching 80% confluence, then passage the cells to preserve log phase growth and cell viability.</em> </li></ol><br/><u>Day 2: Aliquot cells into 6-well plates</u> <ol class="npwtwo" start="12"><li>Remove media from the cells.</li><li>Rinse with 5 ml of sterile room temperature PBS.</li><li>Remove PBS from the flask.</li><li>Add 4 ml of TrypLE (<em>i.e.</em>, recombinant trypsin) to the flask to detach adherent cells.</li><li>Incubate at 37 Â°C for 3-5 min.</li><li>Examine the flask under a microscope. The cells should be floating or appear round at this point.</li><li>Gently tap the side of the flask to detach the remaining weakly adherent cells.</li><li>Add 8 ml of cRF10 to the flask.</li><li>Pipet the cells up-and-down 5x. Each time, expel the media towards the base of the flask (<em>i.e.</em>, where the adherent cells were) to gently force the weakly adherent cells from the flask.</li><li>Transfer the cells to a 50 ml tube.</li><li>Spin at 200 <em>x g</em> for 3 min at room temperature.</li><li>Count the cells using a hemocytometer or automated counter.<br/>Automated counters are acceptable for cell lines, but often not for primary tissues (<em>e.g.</em>, blood cells).</li><li>Dilute the cells to 1 x 10<sup>5</sup> cells/ml in cRF10.</li><li>Transfer 2.5 ml of cells (2.5 x 10<sup>5</sup> cells) to each well in a 6-well plate.</li><li>Incubate overnight.</li></ol><br/><u>Day 3: Transfection</u> <ol class="npwtwo" start="27"><li>Examine the cells using a microscope. Cells should be 50-70% confluent prior to the transfection.<ol class="npwthree"><li>If the density is too low, then wait until the next day.</li><li>If the density is too high, then discard this plate and go back to day 2 to plate a new batch of cells.</li></ol></li><li>Thaw plasmids and PEI transfection reagent on ice.<br/><em>Note: Thaw the plasmids and PEI completely (i.e., no solids remain in the tubes).</em> </li><li>Whilst thawing, label a microfuge tube for each plasmid.</li><li>Prepare a master mix of PEI in PBS at 60 Î¼g/ml in a microfuge tube (or 15 ml tube for larger transfections).<br/>Make 1.2x the total amount needed to ensure there is enough master mix. For example, if 10 plasmids are to be transfected, then 100 Î¼l of PEI in PBS is needed for each plasmid to yield a total of 1.0 ml. Make 1.2 ml to ensure there is enough.</li><li>Add PBS to master mix tube.</li><li>Add PEI to master mix tube.</li><li>Mix by gently pipetting. Do not vortex.</li><li>Add 100 Î¼l of PBS to a microfuge tubes for each plasmid.</li><li>Add 2.0 Î¼g of plasmid DNA to the appropriate microfuge tube (<em>e.g.</em>, 2 Î¼g DNA per 100 Î¼l PBS).</li><li>Mix by gently pipetting. Do not vortex.</li><li>Add 100 Î¼l from the master mix (PEI in PBS) into the microfuge tubes that contain plasmid DNA. This should give 6 Î¼g of PEI to achieve a 3:1 PEI:DNA ratio.</li><li>Mix by gently pipetting once. Do not vortex.</li><li>Incubate at room temperature for 15 min.</li><li>Aspirate the media on the CHO cells.</li><li>Gently replace media on CHO cells with 2 ml of fresh cRF10.</li><li>Add 200 Î¼l from the DNA:PEI solutions dropwise to the appropriate wells. </li><li>Gently rock the plate back and forth to spread the PEI:DNA complex throughout the well.<br/><em>Note: Do not swirl the plate.</em> </li><li>Incubate the cells with transfection reagents in cRF10 for 3-24 h.</li></ol><br/><u>Day 4: Drug selection and monitoring</u> <ol class="npwtwo" start="45"><li>Replace the media with 2 ml of warm cRF10 the next morning. If toxicity is a problem, then change the media after 3-6 h.</li><li>Examine the cells using a fluorescent microscope or via flow cytometry to determine transfection efficiency.<ol class="npwthree"><li>If your plasmids do not contain a fluorescent reporter, then include a standard fluorescent protein vector (<em>e.g.</em>, GFP) as a positive control to help monitor transfection success.</li><li>For flow cytometry, we often collect the floating cells and run through the flow cytometer. This will consist mainly of dead cells but should provide a yes/no answer as to whether the transfection was successful.</li></ol></li><li>Add selection reagent (<em>i.e.</em>, hygromycin at 800 Î¼g/ml) as soon as the reporter is visible or after 24 h.</li><li>Monitor daily and record fluorescence and confluence.<ol class="npwthree"><li>Massive cell death should occur within 1-3 days if using a high dose of hygromycin. Remove the floating cells every other day and replace with fresh cRF10 with 800 Î¼g/ml of hygromycin.</li><li>Small colonies of cells should start forming in 2-5 days.</li><li>Allow dividing colonies to reach at least 20 cells before using TrypLE to detach the cells.</li></ol></li><li>When the cells are actively proliferating in high dose hygromycin, then passage the cells using TrypLE as described above in the Day 2 steps.</li><li>When all cells express the reporter protein (<em>i.e.</em>, drug selection is complete), or when all cells appear healthy in the flask, then the hygromycin dose can be dropped to 100-200 Î¼g/ml.</li><li>Expand the cells and freeze.<br/><ol class="npwthree"><li>3-5 vials are enough unless you will be using these cells regularly.</li><li>For cells secreting recombinant proteins, we generally purify one large batch of proteins that lasts for several years.</li></ol><em>Note: If your vector was used to make surface protein or other non-secreted protein, then you can skip the remainder of Part D. Make sure to freeze at least two vials of cells at a minimum of 1 x 10<sup>6</sup> cells at -80 Â°C.</em><em></em><br/></li></ol><br/><u>Days 10-18: Collect supernatant for testing</u> <ol class="npwtwo" start="52"><li>Passage cells to a low density (1 x 10<sup>5</sup> cells/ml) in cRF5 in T75 flasks.</li><li>The selection drug can be omitted at this step as long as you have a frozen stock in the freezer.<br/>We have maintained Sleeping Beauty-transfected cells without ongoing drug selection for 3 months and observed minimal decrease in the expression of target proteins.</li><li>Allow cells to proliferate to &gt; 90% confluence.</li><li>Collect supernatant.</li><li>Centrifuge at &gt; 3,000 <em>x g</em> for 15 min at 4 Â°C.</li><li>Collect the supernatant and use a 20 ml syringe with a 0.22 Î¼m PVDF filter to remove cellular debris.</li><li>Store the supernatant at 4 Â°C and use within 28 days.<br/>We have observed that some FAST proteins are still capable of binding their expected receptor after 2 months in supernatant at 4 Â°C. However, protein stability is variable, so it is best to test the protein immediately after collection of supernatant. <br/><br/></li></ol><u>Adapt cells to protein-free media to produce high quantities of pure protein</u><br/><ol class="npwtwo" start="59"><li>Cells can be used from the steps above or thawed from a frozen vial.</li><li>Ensure the cells are fully-drug selected and cultured in cRF5.<br/>Ongoing drug selection is optional at this point. We generally maintain a low concentration of drug (<em>e.g.</em>, 100 Î¼g/ml of hygromycin).</li><li>When cells are in log-phase proliferation and 30-50% confluence, remove half of the cRF5 from the flask and replace with warm CHO EX-CELL protein free media.<ol class="npwthree"><li>Any protein-free CHO cell specific media should work here, but we have had good results with CHO EX-CELL media. Note that this CHO EX-CELL media requires addition of 4x the amount of L-glutamine than for RPMI.</li><li>Alternatively, proteins can be purified from cRF5 supernatant. The HisTrap columns should purify only the His-tagged proteins, with minimal carryover of the FBS components (<em>e.g.</em>, bovine serum albumin). Ensure the drug used for selection of the cell line (<em>e.g.</em>, hygromycin) does not contain a His-tag.</li></ol></li><li>Repeat Step D61 except replace all the media with CHO EX-CELL protein-free media.<ol class="npwthree"><li>There is often a decrease in cell viability at this stage. Ensure that you have a large batch of cells with greater than 90% viability to continue.</li><li>If you are using a suspension culture system, such as a spinner flask or propeller flask, then you will need to seed the cells at a high density (minimum of 5 x 10<sup>5</sup> cells/ml).</li></ol></li><li>Collect supernatant from this point forward.<ol class="npwthree"><li>Centrifuge supernatant and store at 4 Â°C as described above.</li><li>Filtration can be omitted here; the purified proteins will be filtered later.</li></ol></li><li>Expand the cells into 3 x T175 flasks by seeding 5 x 10<sup>6</sup> cells/flask with 30 ml of CHO EX-CELL media.</li><li>Allow the cells to proliferate to ~80% confluence.<br/><em>***Critical step: Do not let the cells exceed 80% confluence, as they need to be rapidly proliferating at all times following transfer to protein-free media.***</em> </li><li>Continue passaging cells until at least 200 ml of supernatant has been collected.<br/><em>Note: Cell lines do not always adapt well to the protein-free media. If you have repeated problems expanding the cell line, then you might want to purify the proteins from cRF2.5 media instead of protein-free media.</em> </li><li>Centrifuge all supernatant and store at 4 Â°C.</li><li>Purify protein within 24 h to minimize aberrant protein aggregation and degradation. Many proteins will be stable for longer periods of time, but the best practice is to purify the proteins soon after collection.</li></ol><br/><u>Protein purification</u><br/>The six histidine residues (6xHis) included in the linker peptide between the POI and the fluorescent reporter in the FAST protein allows for easy purification of protein secreted from eukaryotic cells. The basic protocol is to centrifuge your supernatant to remove debris, then use a peristaltic pump to apply your supernatant to an affinity column. The 6xHis-tag binds to Ni<sup>2+</sup> ions in the column, and the proteins are then eluted using high concentration of imidazole, which competes with the 6xHis for binding to Ni<sup>2+</sup> ions.<br/>  Most medium and large institutions will likely have a protein purification system available. We suggest asking around to see what you can find. Otherwise, a basic peristaltic pump will suffice. Plastic tubing can be found a home improvement or home brewing shop. The connector parts for the purification should come with the protein columns that you purchase. Determine what speed on the pump is required for a flow rate of 1 ml/min. You can do this by placing the output tubing into a graduated cylinder and measuring how long it takes to run 10 ml of water through the column. Adjust the pump and repeat as many times as needed until you are confident that you have an accurate flow rate. Note that when the column is connected the flow rate can vary slightly. Also determine the volume of liquid that your tubing holds from intake to output. This helps with the elution step, so you don't collect unnecessary flow through into your protein collection tubes.<br/>  After you calibrate the flow rate to 1 mL/minute, then plug the pump into a simple electrical outlet timer, which will allow you to walk away while you run large supernatant volumes through the column.<br/>  The manufacturer's protocol for HisTrap excel columns is currently found <a href="https://www.gelifesciences.com/en/au/shop/chromatography/prepacked-columns/affinity-tagged-protein/histrap-excel-p-00310" target="_blank">here</a>.<br/><br/><ol class="npwtwo" start="69"><li>Prepare all necessary solutions listed in the <a href="https://os.bio-protocol.org/attached/file/20200426/Recipes spreadsheet.xlsx" target="_blank">Recipes spreadsheet</a>.<ol class="npwthree"><li>5 M imidazole.</li><li>HisTrap excel equilibration buffer (20 mM sodium phosphate, 0.5 M NaCl, pH 7.4).</li><li>If several purifications are planned, then prepare a 10x HisTrap excel equilibration buffer. Otherwise prepare a 1x HisTrap excel equilibration buffer.</li><li>HisTrap excel wash buffer (20 mM sodium phosphate, 0.5 M NaCl, 10 mM imidazole, pH 7.4).</li><li>HisTrap elution buffer (20 mM sodium phosphate, 0.5 M NaCl, 0.5 M imidazole, pH 7.4).</li></ol></li><li>Label 10 microfuge tubes to collect your eluted protein.</li><li>Combine all supernatants that contain the same protein (<em>e.g.</em>, pAF176 supernatant).<ol class="npwthree"><li>If some supernatant aliquots are fresh and some are &gt; 7 days old, then it might be better to purify only the fresh supernatant.</li><li>All supernatants should have been centrifuged at &gt; 3,000 x g and 4 Â°C for 15 min before this step.<br/><em>Note: The supernatants can be filtered through a 0.45 Î¼m PVDF membrane to remove debris, but this is not necessary with the HisTrap excel purification columns that our team generally uses for 6xHis-tag purification.</em> </li></ol></li><li>Dilute your supernatant 1:1 (v/v) with equilibration buffer. The pH should now be ~7. If the pH is not between 6 and 8, then adjust with a weak acid or base.</li><li>Place the output tubing into a waste collection vessel.</li><li>If the tubing in the pump or your protein purification machine has not been used for several weeks or was not cleaned properly after its last use, then wash the tubing with 1 M NaOH. Set the pump to draw the NaOH solution into the tubing. After running enough NaOH solution into the tube to fill the tubing, you can stop the pump and let it soak for 1-24 h. Then empty the tube out and rinse with sterile water.<br/><em>Note: Make sure all of the NaOH is removed from the tubing before using it for protein purification.</em> </li><li>Fill the pump tubing with sterile ddH<sub>2</sub>O.<br/><em>Note: Make sure the tubing intake is completely submerged in all solutions throughout this process; air bubbles in the tubing or column can ruin your purification and protein.</em> </li><li>Remove the stopper plug from the top of your column.</li><li>Set the pump to run at a low flow rate (&lt; 1 ml/min) and allow the ddH<sub>2</sub>O to drip onto the intake port on top of the column to force out the air. A small volume of the ddH<sub>2</sub>O should overflow the column as this helps to ensure there is no air in the intake port.</li><li>Stop the pump.</li><li>Twist the tubing connector into the column, making sure there is no air trapped in the column.<br/><em>***Critical step: Ensure that no air bubbles enter the column.***</em> </li><li>Remove the snap-off end of the column outlet.<br/></li><li>Transfer the intake tubing into the equilibration buffer.</li><li>Equilibrate the column with 5 column volumes of equilibration buffer at 1 mL/minute.</li><li>Stop the pump.<br/><em>Note: Be gentle with the tubing so that air does not enter the tubing during the transfer.</em> </li><li>Submerge the intake tubing to the bottom of your sample vessel.<br/><em>Note: If the sample is in a graduated cylinder (make sure it is clean), then you will know precisely your sample volume and can set the timer accordingly.</em> </li><li>Transfer the output tubing into a clean "flow through" collection vessel. If for some reason your purification fails, then you can re-run the flow through to salvage the protein.</li><li>Set the timer to run ~80% of the sample volume through the column.</li><li>Start the pump.<br/></li><li>Start the timer.<ol class="npwthree"><li>Make sure you monitor the tube during the transition from equilibration buffer to the sample. If you observe bubbles in the tubing, then disconnect the tubing from the column and allow the bubble to be forced out. Then re-connect the tubing and column using the "drop-to-drop" method described above.</li><li>Make sure to monitor the pump as the timer approaches its stopping point. If your flow rate was faster than expected, then you could end up sucking air into the column and ruining your protein.</li></ol></li><li>Re-start the pump if you have sample left in your vessel and let it run until your sample is nearly gone. It is better to leave a few mL of your sample in the vessel than to try to get every drop and risk introducing air into the column.</li><li>Stop the pump.</li><li>Submerge the intake tubing into the wash buffer vessel.</li><li>Start the pump.</li><li>Run 20 column volumes of wash buffer through the column.</li><li>Stop the pump.</li><li>Submerge the intake tubing into the 0.5 M imidazole elution buffer.</li><li>Start the pump and run several mL into the flow through container.<br/><em>Note: The volume to run into the flow through is determined by the volume that your tubing holds (i.e., a long tube needs more solution to flow through it to clear the tube).</em> </li><li>Transfer the output tubing into your first protein collection microtube and collect ~1 ml.</li><li>Repeat this until you have collected at least 10 ml and are confident that all of your protein has been eluted.</li><li>Check the protein collection tubes.<br/><em>Note: If you are purifying a FAST protein, then you simply place your tubes on a blue light and see if they fluoresce (<strong>Figure 19</strong>).</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701192553_9282.jpg" width="450"/><br/><strong>Figure 19. Image of FAST protein elution tubes.</strong> The first few tubes will likely contain only a low concentration of protein, the middle tubes should contain the bulk of the protein, and the last tubes should contain little or no protein. This indicates that the fluorescent fusion protein has been eluted from the column and shows that the fluorescent protein is properly folded.<br/><br/></li><li>Check the protein concentration using a spectrophotometer that can be used with small volumes (<em>e.g.</em>, 1 Î¼l on a NanoDrop spectrophotometer).</li><li>Set the extinction coefficients on the machine to match the extinction coefficients calculated in Procedure A, Step 9b and use the absorbance at the 280 nm setting on the NanoDrop spectrophotometer.</li><li>Blank the spectrophotometer using elution buffer.</li><li>Measure and record the protein concentration in each of your protein collection tubes. Also record the absorbance at 280 nm and the ratio of absorbances at 260 and 280 nm. These should give an indication of the purity of your protein.</li><li>Store the tubes on ice while you wash your column and the tubing.</li><li>Submerge the intake tubing in equilibration buffer.</li><li>Start the pump.</li><li>Run 10 column volumes through the column.</li><li>Stop the pump.</li><li>Submerge the intake tubing in 20% ethanol.</li><li>Start the pump.</li><li>Run 10 column volumes through the column.</li><li>Stop the pump.</li><li>Twist the column outlet cover (should come with your column) onto the outlet port of the column.</li><li>Twist the connector out of your column.</li><li>Place a few drops of 20% ethanol into the intake port on the column to make sure there is no air in the column.</li><li>Twist the original stopper plug back into the column.</li><li>Store the column in a 15 ml tube contain 20% ethanol. Columns can be stored for several months (potentially years) until you need to re-use it.<br/>It is best to use a different column for each different protein, but with proper elution and washing of the columns you should be able to use the same column for different proteins if you are on a limited budget. Re-using a column for a different protein should be reported in experimental methods.</li><li>At this point your tubing should be clean. You can connect the intake tubing to the output tubing to create a circular system. If you are confident that your tubing will not leak, then you can store it with ethanol inside for a day or two or rinse with water for long term storage.<br/><br/></li></ol><u>Protein dialysis</u> <ol class="npwtwo" start="119"><li>Prepare 3 L of autoclaved PBS.</li><li>Store the PBS on ice or at 4 Â°C until the PBS is cold.</li><li>Wet your dialysis cassettes with cold, sterile PBS.</li><li>Transfer your PBS to a clean 1 L beaker.</li><li>Drop a sterile magnetic stir bar into your PBS cover with aluminum foil and store the beaker on ice or at 4 Â°C.</li><li>Combine the solutions from your protein collection tubes that contain a reasonable yield of your POI. Basically, if there is no protein in some of the tubes, then you don't need them.</li><li>Transfer your combined protein solution into the dialysis cassette.</li><li>Float the cassette in the cold PBS.</li><li>Place the container on a magnetic stirrer in a refrigerator or cold room.</li><li>Set the magnetic stirrer to a low rpm, just enough so that the PBS is clearly moving in a circular motion around the container.</li><li>Stir for at least one hour; overnight is OK as long as the temperature is 4 Â°C.</li><li>Transfer the cassette to a new beaker with fresh, cold PBS.</li><li>Stir again for at least one hour.</li></ol><br/><u>Protein filtration and aliquoting</u><br/><em>***Critical step: Perform these steps in sterile laminar flow biosafety cabinet (BSC). The centrifugation steps will take place outside the laminar flow BSC, but the tubes should always remain closed when outside of the BSC.***</em><br/><ol class="npwtwo" start="132"><li>Place a sterile 50 ml collection tube in a sturdy rack in the BSC.</li><li>Remove the plunger from a 20 ml syringe in the BSC.</li><li>Connect a 0.22 Î¼m PVDF filter to a 20 ml syringe filter.<br/><em>Note: Be careful not to touch the outlet of the filter, as the bottom half of the filter needs to remain sterile.</em> </li><li>Add ~5 ml of cold, sterile PBS into the syringe to pre-wet the filter.</li><li>Pipet your dialyzed protein into the PBS within the syringe.</li><li>Place the plunger back into the syringe and gently expel the solution through the filter into the 50 ml collection tube.<br/><em>Note: Some residual liquid will be retained in the filter, resulting in reduced protein recovery. Flushing the filter with additional PBS can reduce protein loss.</em> </li><li>Transfer the sterile protein solution into a protein spin filter unit (<em>e.g.</em>, Amicon ultra).<br/>We generally use an Amicon Ultra centrifugal filter units, Ultra-15, MWCO 10 kDa. Most fluorescent proteins are &gt; 20 kDa and our POIs are also generally &gt; 20 kDa, so a large cut-off weight is acceptable without a risk of losing the purified protein. However, our proteins are produced in cell culture using protein free media, and His-tag purified, so using a 10 kDa cut off should result in minimal protein loss and &gt; 90% pure protein.</li><li>Add cold, sterile PBS to fill the spin filter to full capacity (15 ml).</li><li>Centrifuge the unit according to the manufacturer's instructions.<ol class="npwthree"><li>Ensure the solution remains cold at all times.</li><li>30 min at 3,000 <em>x g</em> should result in concentration of the solution from a ~15 ml volume to &lt; 3 ml.</li></ol></li><li>Top up to 15 ml again with cold, sterile PBS for a second wash.</li><li>Centrifuge for 30 min at 3,000 <em>x g</em>.</li><li>Place the unit back into the sterile BSC.</li><li>Highly concentrated FAST proteins should be visible even with only visible light excitation at this step (<strong>Figure 20</strong>).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200701/20200701192621_3203.jpg" width="450"/><br/><strong>Figure 20. Concentrated FAST proteins.</strong> Image of FAST proteins in centrifugal concentration tubes. Highly concentrated FAST proteins can be seen even using visible light. From left to right are mCherry, mCitrine, mOrange, and mCherry.<br/><br/></li><li>Transfer the concentrated protein to a microfuge tube placed on ice.</li><li>Measure the protein concentration as described above in Procedure DâProtein Purification.<ol class="npwthree"><li>Blank with cold PBS this time instead of elution buffer.</li><li>Enter the exact extinction coefficients determined in Section A, step 9b into the spectrophotometer.</li><li>Record absorbance ratios.</li></ol></li><li>If the concentration is too low, centrifuge the filter unit again to achieve a smaller volume. Then take another aliquot and measure.</li><li>Repeat until the protein concentration is at least 0.5 mg/ml.<br/>Concentration should be in the 0.5 mg/ml to 2.0 mg/ml range to ensure the spectrophotometer is accurate.</li><li>Optional step: Dilute your protein to 1 mg/ml or 0.5 mg/ml with cold, sterile PBS. This makes for easy calculations for downstream experiments.</li><li>Aliquot your protein into sterile 1.5 ml snap cap microfuge tubes and store on ice.<ol class="npwthree"><li>10 Î¼l/tube is ideal, as you can thaw a fresh batch each time you need the protein.</li><li>If storage space in -80 Â°C freezer is limited, then you can aliquot 50-100 Î¼l/tube.</li></ol></li><li>Label your tubes with the protein name, date, and concentration.</li><li>Freeze at -80 Â°C until needed.<ol class="npwthree"><li>If you will use the protein within the next week, you can store an aliquot at 4 Â°C.</li><li>Most of our proteins seems to tolerate a few freeze/thaw cycles, but it is best to thaw only once and use the entire aliquot.</li></ol></li></ol><br/></li><li>Test the recombinant protein<br/>There are many ways to test your protein. SDS-PAGE and western blots can be used to confirm protein size and epitopes and detailed protocols are available elsewhere (Consortium, 2018). However, if the ultimate goal is to determine receptor-ligand interactions and the protein has been well-studied in other species, then proceeding directly production of a cell line that expresses the receptor to test binding via flow cytometry can be the most efficient approach. This is the approach we will describe here in <strong>Procedure E</strong>.<br/>  If your protein is likely to have a functional effect on certain cell types, then functional testing is also a good option. For example, if you made a recombinant interferon-gamma cytokine, then you could treat primary cells or cell lines from your species of interest with a range of doses and assess MHC-I and PD-L1 upregulation via qPCR. This should give you an answer within one week on whether or not your protein is functional. Additionally, check the products available from commercial suppliers of human and mouse recombinant proteins. In many cases you can attempt to replicate their validation methods for your species.<br/>  If you made an expression vector for the receptor (<em>e.g.</em>, CD200R1) for your soluble POI (<em>e.g.</em>, CD200), then proceed with <strong>Procedure E</strong>. If you have not made a cell line with you target receptor yet, then <strong>repeat Procedures A-D</strong>, with the exception that when you design your insert you will need to keep the TMD and ICD of your target receptor GOI/POI.<br/><br/><u>Flow cytometry to test receptor-ligand interaction</u><br/><strong>exp_ID_7: Flow_Cytometry</strong> (<a href="https://os.bio-protocol.org/attached/file/20200426/Experiments spreadsheet.xlsx" target="_blank">experiments spreadsheet</a>)<br/><br/>Purpose:<br/>To test the binding of FAST proteins to cell lines transfected with receptors expected to bind to the POI (<em>e.g.</em>, CD200-mOrange or CD200-mTagBFP bind to CHO.CD200 cell lines).<br/><br/>Overview:<br/>A single-step assay that adds FAST proteins to live target cells. The soluble protein can bind the surface receptor and remain on the surface of the cell or enter the cell via receptor-mediated endocytosis. Lysosomal degradation of the captured protein is blocked via the use of chloroquine or ammonium chloride, which allows fluorescent signal to accumulate over time and yield a stronger signal. The cells are washed a single time prior to analysis via flow cytometry.<br/><em>Note: If you only have a single FAST protein (e.g., CD200-mOrange), then you won't have a proper negative control (e.g., mOrange only). However, by testing the single FAST protein against several cell lines you can determine if your protein binds specifically to the expected cell line. For example, test the CD200 FAST protein against wild type (CHO), vector only (CHO.pAF112), CD200R1 (CHO.pNP1), and PDL1 (CHO.pAF146) cell lines. If the CD200 FAST protein binds only to CHO.pNP1, then a strong inference can be made that the expected protein interaction occurs.</em><br/><br/>  Thaw your original CHO cell line that secreted the FAST proteins for each color used. These can be used as single-color positive controls for setting up the flow cytometer. These can be thawed the day before the assay in cRF10 and used the next day.<br/>  In this assay we will prepare a "quick load" plate that contains the FAST proteins at their final staining concentration. This allows the use of a multichannel pipet for quick transfer of the treatment to the target cells and reduces variation in staining time.<br/><br/><strong>See "exp_ID_FlowCytometry" for fillable Plate layout</strong><br/><u>Materials and reagents needed for Procedure E:</u> <ol class="npwtwo"><li>T75 sterile tissue-culture treated cell culture flask with vented capâ1 for each cell line used</li><li>1x U-bottom 96-well plate (sterile with lid)</li><li>1x U-bottom 96-well plate (sterile with lid) for aliquoting treatments for rapid loading of warm treatments (<em>i.e.</em>, this is the "quick load" plate)</li><li>Sterile microfuge tubesâ1 for each FAST protein used</li><li>Sterile 50 ml tubesâ2 for each cell line</li><li>Sterile 5 ml tubesâ1 for each cell line used</li><li>Automatic pipettor and 10 ml sterile pipets</li><li>Pipets and sterile pipet tips (10 Î¼l, 100 or 200 Î¼l, and 1,000 Î¼l)</li><li>Flow cytometer that can detect mTagBFP, mCitrine, and mCherry</li><li>Centrifuge</li><li>Incubator at 37 Â°C with 5% CO<sub>2</sub> </li><li>Laminar flow biosafety cabinet (<em>i.e.</em>, sterile hood to work in)</li><li>70% ethanol</li><li>Bleach</li><li>Warm TrypLE</li><li>Warm cRF5 or cRF10 (&gt; 300 ml); use cRF5 for CHO cells, cRF10 for most other cell lines</li><li>Hygromycin 200 Î¼g/mL (H200) in cRF5. Hygromycin stock generally is at ~50 mg/ml</li><li>0.1 M chloroquine. Ammonium chloride can be used instead of chloroquine and won't require purchasing new reagents for most labs, but we generally use chloroquine</li><li>FAST proteins (___ Î¼gâdepends on number of cell lines used) *Thaw fresh batch if needed*.</li><li>Cold FACS buffer (without EDTAâwe haven't tested if EDTA affects FAST protein binding to cell lines). A healthy CHO cell line does not clump, so no need for EDTA</li><li>Warm cRF5 (~100 ml)</li></ol><br/><u>Thaw and culture cells prior to starting the experiment</u><br/><u>Day 0: Thaw cells</u> <ol class="npwtwo"><li>Thaw target cells in cRF10.<table border="0" bordercolor="#000000" cellpadding="2" cellspacing="0" class="ke-zeroborder" style="width:50%;"><tbody><tr><td>Cell line<br/></td><td>target<br/></td></tr><tr><td>CHO<br/></td><td>wild type<br/></td></tr><tr><td>CHO.pAF123 <br/></td><td>vector control<br/></td></tr><tr><td>CHO.pMK1<br/></td><td>saha_CD200 (full-length surface protein)<br/></td></tr><tr><td>CHO.pNP1<br/></td><td>saha_CD200R1 (full-length surface protein)<br/></td></tr></tbody></table></li></ol><u><br/>Days 1-2: Passage cells</u> <ol class="npwtwo" start="2"><li>Replace media with cRF5. Cell confluence should be &lt;80% on the day the assay begins. Passage cells as needed to ensure cells are in proliferation phase when assay begins.<br/></li></ol><br/><u>Day 3: Binding assay</u> <ol class="npwtwo" start="3"><li>Warm TrypLE and cRF5 in a water bath.</li><li>Thaw the FAST proteins on ice.</li><li>See "exp_ID_Flow_Cytometry" Table 1âFAST protein dilutions for a fillable table.<br/><em>Notes:<ol class="npwthree"><li>Protein binding kinetics are influenced by temperature, so it is important to maintain consistent temperatures across treatments.</li><li>Avoid using the outside wells on a plate for an assay in which temperature and moisture content are important. These wells heat the fastest and have the highest evaporation rate.</li></ol></em> </li><li>Transfer 5 ml of cRF5 to a 5 ml tube.</li><li>Add 5 Î¼l of 100 mM chloroquine to achieve a 100 Î¼M concentration of chloroquine in cRF5. Chloroquine is added to block acidification of the lysosome, which allows internalised fluorescent proteins to accumulate and yield a stronger signal (Qureshi <em>et al.</em>, 2011).</li><li>Transfer 1.4 ml of cRF5 with 100 Î¼M chloroquine into microfuge tubes (1 for each FAST protein).</li><li>Transfer the appropriate volume of FAST protein into the microfuge tube.</li><li>Aliquot 110 Î¼l of diluted FAST proteins into the appropriate wells of a U-bottom 96-well plate.</li><li>Aliquot 110 Î¼l of PBS into all other wells. This is performed to maintain humidity and temperature in wells without cells.<br/><em>Note: You will transfer 100 Î¼l from this plate to the assay plate below.</em> </li><li>Store the quick load plate at 37 Â°C with the lid on until needed (no more than an hour or two).</li></ol><br/><u>Preparation of adherent target cell lines</u> <ol class="npwtwo" start="13"><li>Decant media from target cells in T75 flasks into a waste reservoir containing bleach.<br/><em>Notes: <ol class="npwthree"><li>Final concentration of bleach in the reservoir at the end of the experiment should be &gt; 10%.</li><li>If using a different target cell line that is not adherent (i.e., suspension cells), then TrypLE is not needed and cells can be transferred directly to a 50 ml tube and centrifuged.</li></ol></em> </li><li>Transfer 5 ml of room temperature PBS into the flask.</li><li>Gently rock the flask back-and-forth to rinse the cells.</li><li>Decant the PBS from the flask into the waste reservoir.</li><li>Add 3 ml of warm TrypLE to each flask.</li><li>Incubate for 5 min at 37 Â°C.</li><li>Add 7 ml of warm cRF5 to each T75 flask.</li><li>Use a 10 ml pipet to aspirate and gently expel the media against the base of the flask (<em>i.e.</em>, where some cells might still be adherent).</li><li>Use a 10 ml pipet to transfer the cells to the appropriate 50 ml tube.</li><li>Centrifuge at 200 <em>x g</em> for 4 min.</li><li>Decant the media into the waste reservoir.</li><li>Resuspend the cells in 2 ml of warm cRF5 (Without chloroquine for this step).</li><li>Count the cells using an automated cell counter or a hemocytometer.<br/><em>Notes: <ol class="npwthree"><li>See <strong>"exp_ID_7: Flow_Cytometry"</strong> Table Cell Counts for fillable table.</li><li>See plate layout above.</li><li>25,000-100,000 of each cell line per well is acceptable as long it is consistent across cell lines.</li></ol></em> </li><li>Transfer the appropriate volume of cell stock into a new 50 ml tube (<em>i.e.</em>, volume to reach 1 x 10<sup>6</sup> cells).</li><li>Top up the tube to 10 ml with cRF5.</li><li>Spin at 200 <em>x g</em> for 4 min.</li><li>Whilst spinning, aliquot 1 x 10<sup>5</sup> to 1 x 10<sup>6</sup> cells back into original flask for future experiments if needed.</li><li>Add cRF5 with H200 to reach 5 ml in a T25 flask, or 12 ml in T75 flask.</li></ol><br/><u>Quickly load the assay plates</u> <ol class="npwtwo" start="31"><li>Retrieve the 50 ml tubes from the centrifuge and remove the supernatant (always check for a pellet before removing supernatant).</li><li>Transfer 1 ml of cRF5 + chloroquine into the tube and pipet 5x to resuspend 1 x 10<sup>6</sup> cells/ml.</li><li>Quickly transfer 100 Î¼l of cells from the appropriate tube to the appropriate wells (see plate layout above).</li><li>Quickly examine the cells using a microscope to ensure they are round and healthy.<br/><em>Note: It is difficult to focus the lens on U-bottom plates, but focusing on the very bottom of the well allows visualization of the cells well enough to confirm they are present.</em> </li><li>Remove the quick load plate from 37 Â°C only when you are ready to load.</li><li>Use a multichannel pipet to quickly transfer 100 Î¼l from the quick load plate containing the FAST proteins to the assay plate containing the cells.<br/><em>Note: Use the multichannel pipet to gently mix the cells 5x. Discard the pipet tips and get new tips for each transfer.</em> </li><li>Incubate at 37 Â°C for 1 to 3 h.</li></ol><br/><u>Prepare the assay plates for flow cytometry analysis</u> <ol class="npwtwo" start="38"><li>Spin the plates at 500 <em>x g</em> for 3 min.</li><li>Remove the lid of the plate, turn it upside down and dump out the media.<br/><em>Note: If you do not have experience doing this, fill a test plate with water and practice before using the real cells. Surface tension holds the liquid in plate when the plate is inverted. To dump the media, use a rapid downward movement and then stop the motion smoothly without jerking the plate back up.</em> </li><li>Add 100 Î¼l of warm TrypLE to each well that contains cells.</li><li>Incubate at 37 Â°C for 5 min.</li><li>Examine the cells using a microscope to ensure the cells are no longer adherent to the plate.</li><li>Add 100 Î¼l of cold FACS buffer (with NaN<sub>3</sub>).</li><li>Pipet cells up-and-down 5x to create a single-cell suspension.</li><li>Spin the plates at 500 <em>x g</em> for 3 min at 4 Â°C.</li><li>Remove the lid of the plate, turn it upside down and dump out the media.</li><li>Pipet cells up-and-down 5x in 200 Î¼l FACS fix using a multichannel pipet.</li><li>Incubate at 4 Â°C for at least 15 min.<br/><em>Notes:<ol class="npwthree"><li>Storing the plate on a rocking platform during this step is helpful but not necessary.</li><li>For FAST proteins with predicted low affinity, it might be best to run the cells directly in the FACS buffer.</li></ol></em> </li><li>Spin at 500 <em>x g</em> for 3 min.</li><li>Resuspend in 250 Î¼l FACS buffer and transfer to FACS tubes.<br/><em>Note: If your flow cytometer has a plate loader, you can run the cells directly from the plate. However, if this is the first time trying the assay it might be best to run the cells from tubes instead of plates.</em> </li><li>Store at 4 Â°C until flow cytometry analysis.</li></ol><br/><u>Flow cytometry</u><br/><em>Note: If this is your first time doing flow cytometry, then you will need to find a colleague with experience to help you on this assay. Do not be shy about reaching across disciplines, this could be a good opportunity for collaboration.</em> <ol class="npwtwo" start="52"><li>Decant media from your single-color positive control cell lines (<em>e.g.</em>, cells that secreted your FAST proteinsâCHO.pAF176).</li><li>Transfer 5 ml of room temperature PBS into the flask.</li><li>Gently rock the flask back-and-forth to rinse the cells.</li><li>Decant the PBS from the flask into the waste reservoir.</li><li>Add 3 ml of warm TrypLE to each flask.</li><li>Incubate for 5 min at 37 Â°C.</li><li>Examine the cells using a microscope to ensure they look round and not adherent. If the cells look round but are still adherent, give the flask a gentle tap to dislodge the cells from the flask.</li><li>Add 7 ml of cold FACS buffer to the flask.</li><li>Use a 10 ml pipet to mix up and down and gently expel the media against the base of the flask (<em>i.e.</em>, where some cells might still be adhering to the flask).</li><li>Centrifuge at 500 <em>x g</em> for 3 min at 4 Â°C.</li><li>Thoroughly resuspend the cells in flow cytometry fixation buffer.</li><li>Store at 4 Â°C in the dark (<em>i.e.</em>, wrapped in aluminum foil) until needed.<br/><em>Note: Fixed cells can be stored for months and reused as controls for setting up the flow cytometer.</em> </li><li>Activate the lasers and filters on your flow cytometer to match your fluorescent proteins. If you do not have experience with flow cytometry, then please consult with the flow cytometry facility manager.</li><li>Create plots for analyzing your samples.<ol class="npwthree"><li>Forward Scatter-Height against Side Scatter-Height (use to identify target cell population - <em>i.e.</em>, live cells).</li><li>Forward Scatter-Area against Forward Scatter-Height (use to exclude doublets).</li><li>BFP-Height against mOrange-Height.</li></ol><em>Note: You can also create a histogram for each color.</em> </li><li>Run your first sample using the default settings on the flow cytometer (based on daily quality control testing).</li><li>Identify your target population using the Forward Scatter (size) and Side Scatter (granularity) properties.</li><li>Draw a gate around your target population.</li><li>Identify single cells using the Forward Scatter-Area against Forward Scatter-Height plot.</li><li>Draw a gate around the single cell population.</li><li>Add a quadrant gate to each of the multicolor plots. These gates can be adjusted as needed based on the fluorescent output of your single-color controls.</li><li>Run your single-color controls and adjust voltage and compensation as needed. This requires an experienced flow cytometry user.</li><li>Adjust gates as needed.<br/><em>Note: If analyzing unfixed cells, it is important to use unfixed single-color controls as fixation may alter the background fluorescence of cells.</em> </li><li>After setting voltages and gates, run the samples and record at least 10,000 events.</li><li>Analyze the data using the software available at your institution (<em>e.g.</em>, FCS Express, FlowJo).</li></ol></li><p class="pptt" id="biaoti34677">### Notes</p><p class="ppzz"><strong>Procedure A: Troubleshooting</strong><br/><table border="1" bordercolor="#878787" frame="hsides" rules="center" width="1150px"><tbody><tr><td>Step<br/></td><td>Problem<br/></td><td>Possible solution(s)<br/></td></tr><tr><td>3<br/></td><td>Cannot find your GOI in any databases<br/></td><td>The GOI might not exist in your species. We suggest trying this process with a known gene before trying to discover a completely new gene.<br/>Prepare a <em>de novo</em> transcriptome assembly and BLAST the <em>de novo</em> assembly with the GOI sequence from the most closely related species that you can find.<br/></td></tr><tr><td>4<br/></td><td>Numerous isoforms of GOI<br/></td><td>Compare coding regions and functional domains for the optimal transcript. If needed, compare to orthologs and homologs to identify a master transcript.<br/></td></tr><tr><td>4<br/></td><td>Conflicting transcripts of GOI<br/></td><td>Identify changes to DNA/protein sequence and implications for final protein. Compare to orthologs and homologs.<br/></td></tr><tr><td>7<br/></td><td>No start or end signal in GOI transcript<br/></td><td>Use the partial sequence to identify full-length sequences from an alternate database. Investigate <em>de novo</em> transcriptome if possible.<br/></td></tr><tr><td>9<br/></td><td>Limited information available or retrievable for GOI annotation<br/></td><td>Identify potential functional domains by comparison to orthologs or homologs.<br/></td></tr><tr><td>9<br/></td><td>No SigP for GOI transcript<br/></td><td>Use the partial sequence to identify full-length sequences from an alternate database. Investigate transcriptome if possible. Alternately, use native SigP from other similar, well-expressed protein.<br/></td></tr><tr><td>17-22<br/></td><td>Limited or no conservation of functional domains between GOI and orthologs/homologs<br/></td><td>Investigate other potential transcripts, using ortholog and homolog queries. If none other recovered, the transcript may still be of relevance, if not functionally equivalent.<br/></td></tr></tbody></table><br/><strong>Procedure B: Troubleshooting</strong><br/><table border="1" bordercolor="#878787" frame="hsides" rules="center" width="1150px"><tbody><tr><td>Step<br/></td><td>Problem<br/></td><td>Possible solution(s)<br/></td></tr><tr><td>7<br/></td><td>No SigP in GOI transcript<br/></td><td>Use the hamster IL2 SigP as described in step 7. See vector pAF92 for additional details (<a href="https://www.addgene.org/135929/" target="_blank">https://www.addgene.org/135929/</a>).<br/></td></tr><tr><td>13<br/></td><td>GOI segments not in optimal GC% range<br/></td><td>Order more than one possible primer to find a combination that can amplify the target sequence with the necessary vector overlap segments.<br/></td></tr></tbody></table><br/><strong>Procedure C: Troubleshooting</strong><br/><table border="1" bordercolor="#878787" frame="hsides" rules="center" width="1150px"><tbody><tr><td>Step<br/></td><td>Problem<br/></td><td>Possible solution(s)<br/></td></tr><tr><td>PCR and gel<br/></td><td>No PCR product<br/></td><td>See general PCR troubleshooting guidelines elsewhere.<br/></td></tr><tr><td>DNA extraction from gel (PCR or plasmid digest)<br/></td><td>Insufficient PCR product yield or plasmid DNA used in restriction digest.<br/></td><td>Re-run the PCR or restriction digest. If the yield is low again, then use the low yield DNA in the assembly as long as the band size in the gel was correct.<br/></td></tr><tr><td>Assembly and transformation<br/></td><td>Too many colonies<br/></td><td>Perform new overnight restriction digest<br/>Split bacteria in half or thirds<br/>Plate less bacteria<br/>Check ampicillin concentration (check the date plates were made)<br/>Incubation on orbital shaker following the head shock step was too long. One hour of incubation is sufficient. If the incubation time is extended, then the bacteria can enter exponential growth phase and result in vast expansion of positive clones.<br/></td></tr><tr><td>Assembly and transformation<br/></td><td>No colonies or negative for insert<br/></td><td>Poor restriction digest.<br/>Check enzymes, reaction conditions, methylation of the restriction enzyme site.<br/>Overlaps in the insert may be not long enough or poorly designed.<br/>Homemade competent bacteria may be of poor quality, purchase competent cells from a supplier (<em>e.g.</em>, NEBuilder kit comes with pre-aliquoted DH5Î±).<br/>If a UV light source was used to visualize DNA in gel, then care should be taken to limit DNA exposure to UV light as this can damage DNA and be a considerable source of downstream cloning problems.<br/>Use a sterile razor blade or plastic cutting instrument to cut around the target DNA band. Make sure not to cut directly on the glass of your UV/blue light as this can damage the glass. Use a new blade for each band to prevent carry over of DNA between samples.<br/></td></tr><tr><td>Overnight culture of bacteria<br/></td><td>Bacterial culture fails to expand<br/></td><td>Make sure the correct antibiotic and dosage was used.<br/></td></tr><tr><td>Analyzing your sequencing results<br/></td><td>Sequencing results are not clean<br/></td><td>Residual ethanol on the DNA. Repeat the sequencing protocol and allow more time for your DNA to dry when trying to remove the ethanol.<br/>Single-primer sequencing PCR failed. Try using the paired sequencing primers on the plasmid with a 'normal' polymerase (<em>e.g.</em>, OneTaq) to make sure the reaction works at 55 Â°C.<br/>Ask the manager of the sequencing facility for insight.</td></tr></tbody></table><br/><strong>Procedure D: Troubleshooting</strong><br/><table border="1" bordercolor="#878787" frame="hsides" rules="center" width="1150px"><tbody><tr><td>Step<br/></td><td>Problem<br/></td><td>Possible solution(s)<br/></td></tr><tr><td>Transfection<br/></td><td>No reporter proteins evident or cells did not survive drug selection<br/></td><td>Human error is the most common problem. Specifically, pipetting of the plasmid DNA.<br/>Use fresh media and a new batch of PEI.<br/>Cells density was too high.<br/></td></tr><tr><td>Transfection<br/></td><td>No survival after transfection or selection<br/></td><td>Drug dose was too high.<br/>Human error is the most common problem. Specifically, pipetting of the plasmid DNA.<br/></td></tr><tr><td>Transfection<br/></td><td>Populations not expanding after &gt; 7 days drug selection.<br/></td><td>Drug dose was too high.<br/>Increase percentage of FBS in the media to 15-20%.<br/></td></tr><tr><td>Protein expression<br/></td><td>Selected cell populations not expressing desired protein.<br/></td><td>If neither the reporter or POI are expressed, then check the expression vector construct to ensure the ORF does not have an early stop codon. Also confirm the expression vector sequencing was accurate and perfect.<br/>Some proteins will fail for unknown reasons. For well-characterized proteins such as IFN-Î³, we expect this system to work &gt; 95% of the time. For POIs that have not been produced recombinantly in mammalian cells, the failure rate is likely to be 10-20%.<br/></td></tr><tr><td>Protein purification<br/></td><td>Low protein expressed/collected/purified.<br/></td><td>Cells were not rapidly dividing in culture.<br/>Protein was degraded quickly in the supernatant. <br/></td></tr></tbody></table><br/><strong>Procedure E: Troubleshooting<br/></strong> <table border="1" bordercolor="#878787" frame="hsides" rules="center" width="1150px"><tbody><tr><td>Step<br/></td><td>Problem<br/></td><td>Possible solution(s)<br/></td></tr><tr><td>No signal<br/></td><td>FAST protein degraded<br/></td><td>Chloroquine or ammonium chloride not added.<br/>Receptor-mediated endocytosis can activate proteasomal degradation. Try proteasome blocker if the literature suggests this pathway is active.<br/></td></tr></tbody></table></p><p class="pptt" id="biaoti34678">### Recipes</p><p class="ppzz">See associated <a href="https://os.bio-protocol.org/attached/file/20200426/Recipes spreadsheet.xlsx" target="_blank">Recipes spreadsheet</a>, which includes calculations for adjusting volumes and concentrations.</p><p class="pptt" id="biaoti34679">### Acknowledgments</p><p class="ppzz">We thank the co-authors of the manuscript associated with this protocol: Chrissie E.B. Ong, Alana De Luca, A. Bruce Lyons, Gregory M. Woods. We wish to thank Ginny Ralph for her ongoing care of Tasmanian devils, and the Bonorong Wildlife Sanctuary for providing access to Tasmanian devils, and Ruth Pye for providing care for devils and collecting blood samples. We would like to thank Mahalia Kingsley and Nirdesh Poudel for plasmid construction.<br/>  Funding: ARC DECRA grant # DE180100484, ARC Linkage grant # LP0989727, ARC Discovery grant # DP130100715, Morris Animal Foundation Grant-in-Aid # D14ZO-410, University of Tasmania Foundation Dr Eric Guiler Tasmanian Devil Research Grant through funds raised by the Save the Tasmanian Devil Appeal (2013, 2015, 2017 and 2018), and Entrepreneursâ Programme - Research Connections grant with Nexvet Australia Pty. Ltd. # RC50680, Royal Hobart Hospital Research Foundation Incubator Grant # F0026331.</p><p class="pptt" id="biaoti34680">### Competing interests</p><p class="ppzz">The authors received funding from Nexvet Australia Pty. Ltd for related studies.</p><p class="pptt" id="biaoti34681">### References</p><ol class="npwtwok"><li>Benson, D. A., Cavanaugh, M., Clark, K., Karsch-Mizrachi, I., Lipman, D. J., Ostell, J. and Sayers, E. W. (2013). GenBank. <em>Nucleic Acids Res</em> 41(Database issue): D36-D42.</li><li>Consortium, T. U. (2018). <a href="https://academic.oup.com/nar/article/47/D1/D506/5160987" target="_blank">UniProt: a worldwide hub of protein knowledge.</a> <em>Nucleic Acids Res</em> 47(D1): D506-D515. </li><li>Flies, A. S., Darby, J. M., Lennard, P. R., Murphy, P. R., Ong, C. E. B., Pinfold, T. L., De Luca, A., Lyons, A. B., Woods, G. M. and Patchett, A. L. (2020). <a href="https://advances.sciencemag.org/content/6/27/eaba5031" target="_blank">A novel system to map protein interactions reveals evolutionarily conserved immune evasion pathways on transmissible cancers.</a> <em>Sci Adv</em> 6(27): eaba5031.</li><li>Gibson, D. G., Young, L., Chuang, R.-Y., Venter, J. C., Hutchison, C. A. and Smith, H. O. (2009). <a href="https://www.nature.com/articles/nmeth.1318" target="_blank">Enzymatic assembly of DNA molecules up to several hundred kilobases.</a> <em>Nature Methods</em> 6(5): 343-345.</li><li>Kowarz, E., LÃ¶scher, D. and Marschalek, R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25650551" target="_blank">Optimized Sleeping Beauty transposons rapidly generate stable transgenic cell lines.</a> <em>Biotechnol J</em> 10(4): 647-653.</li><li>Qureshi, O. S., Zheng, Y., Nakamura, K., Attridge, K., Manzotti, C., Schmidt, E. M., Baker, J., Jeffery, L. E., Kaur, S., Briggs, Z., Hou, T. Z., Futter, C. E., Anderson, G., Walker, L. S. and Sansom, D. M. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21474713" target="_blank">Trans-endocytosis of CD80 and CD86: a molecular basis for the cell-extrinsic function of CTLA-4.</a> <em>Science</em> 332(6029): 600-603.</li><li>Zerbino, D. R., Achuthan, P., Akanni, W., Amode, M. R., Barrell, D., Bhai, J., Billis, K., Cummins, C., Gall, A., Giron, C. G., Gil, L., Gordon, L., Haggerty, L., Haskell, E., Hourlier, T., Izuogu, O. G., Janacek, S. H., Juettemann, T., To, J. K., Laird, M. R., Lavidas, I., Liu, Z., Loveland, J. E., Maurel, T., McLaren, W., Moore, B., Mudge, J., Murphy, D. N., Newman, V., Nuhn, M., Ogeh, D., Ong, C. K., Parker, A., Patricio, M., Riat, H. S., Schuilenburg, H., Sheppard, D., Sparrow, H., Taylor, K., Thormann, A., Vullo, A., Walts, B., Zadissa, A., Frankish, A., Hunt, S. E., Kostadima, M., Langridge, N., Martin, F. J., Muffato, M., Perry, E., Ruffier, M., Staines, D. M., Trevanion, S. J., Aken, B. L., Cunningham, F., Yates, A. and Flicek, P. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29155950" target="_blank">Ensembl 2018.</a> <em>Nucleic Acids Res</em> 46(D1): D754-D761. <br/></li></ol></ol><style>    .protocolbody td {      padding: 0 10px;    }  </style>