<body>
            <h1>A Method for User-defined Mutagenesis by Integrating Oligo Pool Synthesis Technology with Nicking Mutagenesis</h1>
            <p><strong>Authors:</strong> Paul J. Steiner, Zachary T Baumer, Timothy A. Whitehead</p>
            <h2>Abstract</h2>
            <p>Saturation mutagenesis is a fundamental enabling technology for protein engineering and epitope mapping. Nicking mutagenesis (NM) allows the user to rapidly construct libraries of all possible single mutations in a target protein sequence from plasmid DNA in a one-pot procedure. Briefly, one strand of the plasmid DNA is degraded using a nicking restriction endonuclease and exonuclease treatment. Mutagenic primers encoding the desired mutations are annealed to the resulting circular single-stranded DNA, extended with high-fidelity polymerase, and ligated into covalently closed circular DNA by <em>Taq</em> DNA ligase. The heteroduplex DNA is resolved by selective degradation of the template strand. The complementary strand is synthesized and ligated, resulting in a library of mutated covalently closed circular plasmids. It was later shown that because very little primer is used in the procedure, resuspended oligo pools, which normally require amplification before use, can be used directly in the mutagenesis procedure. Because oligo pools can contain tens of thousands of unique oligos, this enables the construction of libraries of tens of thousands of user-defined mutations in a single-pot mutagenesis reaction, which significantly improves the utility of NM as described below.<br/> <br/>Use of oligo pools afford an economically advantageous approach to mutagenic experiments. First, oligo pool synthesis is much less expensive per nucleotide synthesized than conventional synthesis. Second, a mixed pool may be generated and used for mutagenesis of multiple different genes. To use the same oligo-pool for mutagenesis of a variety of genes, the user must only quantify the fraction of the oligo-pool specific to her mutagenic experiment and adjust the volume and effective concentration of the oligo-pool for use in nicking mutagenesis.</p>
            <p><strong>Keywords:</strong> Nicking Mutagenesis, Oligo pools, Protein Design, Protein Engineering, Site-saturation mutagenesis, Directed Evolution, Deep Mutational Scanning</p>
            <h2>Background</h2><p>Evaluation of the sequence dependence of protein function is of tremendous importance for applied and fundamental protein science. In recent years, deep mutational scanning (DMS) has risen to the forefront of protein-based research (Fowler and Fields, 2014). DMS experiments allow for the elucidation of genotype-phenotype relationships and the generation of biomolecular fitness landscapes using large numbers of protein variants assessed using deep sequencing. DMS has been employed for protein engineering (Romero <em>et al.</em>, 2015), epitope mapping (Van Blarcom <em>et al.</em>, 2015; Kowalsky <em>et al.</em>, 2015a), and evolutionary biology (Doud <em>et al.</em>, 2017; Faber <em>et al.</em>, 2019). The key to DMS is correlating the abundance of cells expressing a particular protein variant to a particular property of that variant. If this can be done then quantitative deep sequencing can measure the property for tens of thousands of protein variants in an <em>in vivo</em> library. The generation of libraries containing large numbers of programmed variants is thus essential to obtaining high quality data from a DMS experiment. Often, a saturation mutagenesis library containing all possible single mutations is the desired starting point.<br/> <br/>Methods for saturation mutagenesis have continued to improve over time, starting with uracil-dependent Kunkel and Pfunkel Mutagenesis (Kunkel, 1985; Firnberg and Ostermeier, 2012). More recently, the development of nicking mutagenesis by Wrenbeck <em>et al.</em> (2016), which employs complementary nicking restriction endonucleases has improved on Kunkel and Pfunkel in time and convenience. At the same time, oligonucleotide pool technology, which yields tens of thousands of specifically designed oligos in one pot, has rapidly advanced. Oligo-pools afford practical and economical benefits to the original NM protocol, permitting many more mutations to be designed at lower cost (Medina-Cucurella <em>et al.</em>, 2019). The combination of nicking mutagenesis with oligo pool-derived primers may benefit any research where many amino acid substitutions are desired at one or many sites in a target protein, and specifically in directed evolution studies applicable to protein design and evolutionary biology. As an example, this protocol was recently used to program nearly all single point mutants in critical genes in viruses (Faber <em>et al.</em>, 2020).</p>
        </body>

<p class="pptt" id="biaoti34940">## Materials and Reagents</p><ol class="npwtwok"><li>245 mm Square BioAssay Dishes (Corning<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 431111 </a> )</li><li>PCR Tubes (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 52509-304 </a> )</li><li>Glass Pasteur pipette</li><li>Scraper</li><li>50 ml conical tube (<em>e.g.</em>, Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 14-432-22 </a> )</li><li>High-efficiency electrocompetent <em>E. coli</em> cells (essential that these are &gt; 10<sup>9</sup> CFU/Î¼g plasmid DNA, <em>e.g.</em>, Agilent XL-1-Blue Electroporation Competent Cells) (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 200228 </a> )</li><li>Oligo pool containing mutagenic oligos (or manually pooled mutagenic oligos), <em>e.g.</em>, Agilent SurePrint Oligonucleotide Libraries</li><li>Single primer that anneals to the template strand at a non-mutagenized location with an opposite orientation compared with mutagenic oligos (synthesized by <em>e.g.</em>, Integrated DNA Technologies)</li><li>Plasmid to mutagenize containing a single BbvCI restriction site (multiple BbvCI restriction sites are acceptable as long as all are in the same orientation)<br/>We have successfully mutagenized plasmids up to 8 kb in length. The plasmid pEDA5-GFPmut3-Y66H described in Wrenbeck <em>et al.</em> (2016) (Addgene ID 80085) is a useful control (see Note 3).</li><li>Monarch<sup>Â®</sup> PCR &amp; DNA Cleanup Kit (5 Î¼g) (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> T1030S/L </a> )</li><li>Miniprep kit (<em>e.g.</em>, Monarch<sup>Â®</sup> Plasmid Miniprep Kit, New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> T1010S </a> )</li><li>T4 Polynucleotide Kinase (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0201S/L </a> )</li><li>Nt.BbvCI (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> R0632S/L </a> )</li><li>Nb.BbvCI (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> R0631S/L </a> )</li><li>Exonuclease I (<em>E. coli</em>) (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0293S/L </a> )</li><li>Exonuclease III (<em>E. coli</em>) (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0206S/L </a> )</li><li>Taq DNA Ligase (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0208S/L </a> )</li><li>Phusion<sup>Â®</sup> High-Fidelity DNA Polymerase (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0530S/L </a> )</li><li>DpnI (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> R0176S/L </a> )</li><li>Nuclease free water (<em>e.g.</em>, New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> B1500S/L </a> )</li><li>Molecular biology grade Dithiothreitol (DTT) (<em>e.g.</em>, GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> DTT10 </a> )</li><li>10x CutSmart<sup>Â®</sup> Buffer (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> B7204S </a> )</li><li>SYBR Safe DNA gel stain (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> S33102 </a> )</li><li>5x HF Buffer (included with Phusion<sup>Â®</sup> polymerase) (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> B0518S </a> )</li><li>10 mM Adenosine 5â-Triphosphate (ATP) (<em>e.g.</em>, New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> P0756S/L </a> ) prepared in single-use aliquots of 10 Î¼l</li><li>10 mM Deoxynucleotide (dNTP) Solution Mix (<em>e.g.</em>, New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> N0447S </a> )</li><li>Î²-Nicotinamide adenine dinucleotide (NAD<sup>+</sup>) (50 mM) (<em>e.g.</em>, New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> B9007S </a> ) prepared in single-use aliquots of 10 Î¼l</li><li>Single-use aliquots (see Recipes)<br/>10 mM ATP<br/>50 mM NAD<sup>+</sup><br/>10 mM dNTPs</li><li>50 mM DTT (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34941">### Equipment</p><ol class="npwtwok"><li>-20 Â°C freezer</li><li>Pipettes</li><li>Thermal Cycler (<em>e.g.</em>, Eppendorf, model: Mastercycler<sup>TM</sup> pro, catalog number: <a class="blackNoUnder" target="_blank"> 950040025 </a> )</li><li>Microcentrifuge (<em>e.g.</em>, Fisher, model: <a class="blackNoUnder" target="_blank"> accuSpin microcentrifuge 17 </a> , catalog number <a class="blackNoUnder" target="_blank"> 13-100-675 </a> ; does not need to be refrigerated)</li><li>Electroporator (<em>e.g.</em>, Eppendorf Eporator<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 4309000027 </a> )</li></ol><p class="pptt" id="biaoti34942">### Software</p><ol class="npwtwok"><li>Python<br/>For making large numbers of specific mutations at many different amino acid positions, automated primer design software is indispensable. We provide a flexible python script for automated primer design in the supplementary information. This script can design mutagenic primers either using a user-specified degenerate codon (<em>e.g.</em>, NNK), or by generating one or more mutation specific primers using common codons for a specified organism. The outputs from the script can be specified as tab-delimited or comma-separated values (CSV) files with programmable primer naming. Finally, the script facilitates precise specifications of which residues to mutate using the resfile format from the Rosetta macromolecular modeling suite (Leaver-Fay <em>et al.</em>, 2011). Below, we provide some basic examples of inputs and command lines for the script.<br/>    The software is written such that the 5â² and 3â² arm lengths are constant, either at a user defined length or a default length of 30 bases. Constant oligonucleotide lengths result in differential melting temperatures for individual members of the oligo pool and is a compromise with length and cost restrictions for oligo pool synthesis. Within a fixed-length pool, the primers with the highest predicted melting temperatures appear to be incorporated up to tenfold more efficiently than those with the lowest predicted melting temperatures (Medina-Cucurella <em>et al.</em>, 2019), but the bias is tolerable for all end-uses in our laboratory.<br/>    The only required input to the script is a plain text file containing three lines. The first line should contain sequence of the coding sequence of the gene to mutagenize (at least the length of the 5â² homology arm length), the second line should contain the coding sequence (CDS) to mutagenize <strong>without</strong> a stop codon, and the third line should contain the stop codon and sequence downstream of the coding sequence to mutagenize (at least the length of the 3â² homology arm length). The additional sequence is required in order to design primers that mutagenize the first and last few amino acids, the amount of additional sequence is dependent on the desired 5â² and 3â² arm lengths for the mutagenic primers. The full list of command-line flags is shown in <a href="https://os.bio-protocol.org/attached/file/20200720/3697 Supplemental file Table S1.docx" target="_blank">Table S1</a>.<br/><br/>Suppose we want to mutagenize the amino acid sequence MVTAGENSIS, which is encoded (<strong>CDS; bold</strong>) within the following DNA sequence:<br/>ATAGACAGTA<strong>ATGGTGACCGCGGGCGAAAACGAAAGCATTAGCTAG</strong>ACAGTTG<br/><br/>The input file (input.seq) we would use with the script would be:<br/>ATAGACAGTA<br/><strong>ATGGTGACCGCGGGCGAAAACGAAAGCATTAGC</strong><br/><strong> TAG</strong>ACAGTTG<br/>(The CDS is shown in <strong>boldface</strong> for clarity.) Note that the third line includes the TAG stop codon.<br/><br/><em>Example 1:</em><br/>Make every possible amino acid mutation (including wild-type) at every position in the coding sequence. Generate two primers for each mutation, using the two most common codons in yeast for the mutant amino acid. Design each primer with 28 basepairs of homology to the template before and after the mismatched codon. Output a CSV to the file primers.csv.<br/><span style="font-family:Courier New;">python nm_primers.zip --five-prime-arm-length 28 --three-prime-arm-length 28 --organism yeast --separator comma --output primers.csv input.seq</span><br/><br/><em>Example 2:</em><br/>Mutate each residue using the degenerate codon NNK. Output a tab-separated file to primers.tsv. Design each primer with 20 base pairs homology upstream of the mutation and 30 basepairs homology downstream of the mutation.<br/><span style="font-family:Courier New;">python nm_primers.zip --five-prime-arm-length 20 --three-prime-arm-length 30 --custom-codon NNK --separator tab âoutput primers.tsv input.seq</span><br/></li></ol><p class="pptt" id="biaoti34943">### Procedure</p><p class="ppzz">A visual depiction of the protocol steps is pictured in Figure 1 followed by detailed written descriptions of each step. Briefly, steps [A] (preparation of plasmid DNA) and [B]-[D] (design, pooling, and phosphorylation of oligonucleotides) are performed ahead of time and can be done in parallel. Steps [E]-[L] are all performed on a single day. ssDNA is prepared from plasmid dsDNA [E] and the mutagenic strand is synthesized from the ssDNA template [F]. The heteroduplex is column purified [G] and the template strand is specifically degraded [H]. The mutagenic strand is regenerated [I], and the mutagenic DNA is enriched [J], column purified [K], and transformed into <em>E. coli</em> [L]. The next day plasmids containing user-defined mutations are prepped from <em>E. coli</em> transformants [M].<br/><br/><img alt="" class="layerphoto" height="209" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200724/20200724224231_1194.png" width="300"/><br/><strong>Figure 1. Nicking Mutagenesis Procedure.</strong> Steps [A]-[D] shown in green boxes are prepared independently of the rest of the method. [A] Preparation of mutagenesis template plasmid with unique BbvCI restriction site; [B] Automated user-defined design of mutagenic primers; [C] (as needed) Pooling of mutagenic primers; [D] Phosphorylation of oligonucleotides; [E] ssDNA template prep; [F] Mutagenic strand synthesis; [G] Column purification; [H] Bottom template strand degradation; [I] Mutagenic strand regeneration; [J] Removal of input plasmid with DpnI; [K] Column purification; [L] Transformation; [M] Plasmid library preparation.</p><br/><ol class="npwone"><li>Prepare mutagenesis template plasmid<br/>The template plasmid must contain a BbvCI site (CCTCAGC). It is acceptable for the plasmid to contain multiple BbvCI sites <strong>only</strong> if all are in the same orientation. If not already present, the site can easily be added using any site-directed mutagenesis procedure (<em>e.g.</em>, Q5<sup>Â®</sup> Site-Directed Mutagenesis). If using the nicking enzymes in the order presented here (Nt.BbvCI first, Nb.BbvCI second), the orientation of the nicking site will determine if the mutagenic primers should match the sense or antisense strand of the plasmid. BbvCI is needed because nickase versions that nick only the top or only the bottom strand are available. In principle other nickases could be used but the buffers may need to be reformulated to allow compatibility with the nucleases and other DNA-modifying enzymes.<br/>    Nicked plasmid DNA will be degraded during nicking mutagenesis, so plasmid preparations should be freshly prepared, either immediately prior to use, or not more than one month in advance. The plasmid should not be subjected to more than one freeze-thaw cycle to maximize the fraction of covalently closed molecules. Plasmid prepared and frozen for up to one month in a non-defrosting -20 Â°C freezer can be used (see troubleshooting tips in <strong>Table 1</strong>).<br/><br/></li><li>Design mutagenic oligonucleotides<br/>Design primers encoding the desired mutations. Primers can encode multiple mutations as long as sufficient homology to the template is included upstream and downstream of the first and last mutations respectively. We typically have these homologous regions be at least 24 bp. Using software makes it possible to design thousands of primers quickly and minimizes errors in design even for small numbers of primers. A primer design script written in Python is provided in the supplementary information (see Software above).<br/>  Small numbers of primers can be ordered individually in tubes or 96-well plates, but when making thousands of programmed mutations (<em>e.g.</em>, a large number of proximal double mutants), oligo pools are significantly more economical. Because nicking mutagenesis uses femtomole amounts of primers (&lt; 0.05 pmol per reaction), commercial oligo pools can be used directly without PCR amplification. Furthermore, non-homologous primers present in an oligo pool will not anneal to templates or interfere with the procedure, making it possible to synthesize mutagenic primer sets for multiple genes in a single pool.<br/><br/></li><li>Pool mutagenic oligonucleotides<br/>If using primers from an oligo pool, resuspend the pool according to the manufacturerâs instructions and compute the fraction of the oligo pool containing relevant primers for this mutagenesis. If pooling individual primers, resuspend each primer and pool all primers so that the final concentration of <strong>all</strong> primers is 10 Î¼M. For example, if pooling 100 mutagenic primers, the final concentration of each individual primer should be 100 nM, giving a final concentration of 10 Î¼M total primer.<br/><br/></li><li>Phosphorylate oligonucleotides<br/><ol class="npwtwo"><li>Assemble the following two reactions in PCR tubes.<br/>Primer pool phosphorylation:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:600px;"><tbody><tr><td>Component<br/></td><td>Volume (Î¼l)<br/></td><td>Final oncentration or Amount<br/></td></tr><tr><td>Pooled primers (10 Î¼M)<br/></td><td>20</td><td>8.2 Î¼M<br/></td></tr><tr><td>10x T4 Polynucleotide Kinase Buffer<br/></td><td>2.44<br/></td><td>1x<br/></td></tr><tr><td>10 mM ATP<br/></td><td>1</td><td>0.41 mM<br/></td></tr><tr><td>T4 Polynucleotide Kinase (10 U/Î¼l)<br/></td><td>1<br/></td><td>10 U<br/></td></tr></tbody></table>Secondary primer phosphorylation:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="font-family:Arial, Helvetica, sans-serif;width:600px;"><tbody><tr><td>Component<br/></td><td>Volume (Î¼l)<br/></td><td>Final Concentration or Amount<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O<br/></td><td>18<br/></td><td>N/A<br/></td></tr><tr><td>10x T4 Polynucleotide Kinase Buffer<br/></td><td>3</td><td>1x</td></tr><tr><td>Secondary primer (100 Î¼M)<br/></td><td>7</td><td>23.3 Î¼M<br/></td></tr><tr><td>10 mM ATP<br/></td><td>1</td><td>0.33 mM<br/></td></tr><tr><td>T4 Polynucleotide Kinase (10 U/Î¼l)<br/></td><td>1</td><td>10 U<span style="color:#666666;font-family:Arial, Helvetica, sans-serif;font-size:10pt;text-align:justify;"></span> </td></tr></tbody></table></li><li>Incubate the two reactions at 37 Â°C for one hour in a thermocycler. As an optional step, inactivate T4 kinase by incubating at 65 Â°C for twenty min.<br/><br/></li></ol></li><li>Top strand degradation (Template Prep)<ol class="npwtwo"><li>Assemble the following reaction in a PCR tube on ice:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:1000px;"><tbody><tr><td>Component<br/></td><td>Volume (Î¼l)<br/></td><td>Final Concentration or Amount<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O<br/></td><td>To 20 Î¼l final volume<br/></td><td>N/A<br/></td></tr><tr><td>Fresh plasmid DNA containing BbvCI site<br/></td><td>0.76 pmol*<br/></td><td>0.76 pmol<br/></td></tr><tr><td>10x CutSmart<sup>Â®</sup> Buffer<br/></td><td>2</td><td>1x</td></tr><tr><td>Exonuclease III (diluted tenfold to 10 U/Î¼l)<br/></td><td>1</td><td>10 U</td></tr><tr><td>Exonuclease I (20 U/Î¼l)<br/></td><td>1</td><td>20 U<br/></td></tr><tr><td>Nt.BbvCI (10 U/Î¼l)<br/></td><td>1</td><td>20 U<br/></td></tr></tbody></table>*0.76 pmol ofâ¦ 3 kb plasmid = 1.41 Î¼g; 4 kb plasmid = 1.88 Î¼g; 5 kb plasmid = 2.35 Î¼g; <em></em><em>etc.</em> In order to constrain the volume of plasmid to 1-15 Î¼l, the plasmid must be at a concentration of 51-760 nM. (<em>e.g.</em>, a 3 kb plasmid will be between 95 and 1410 ng/Î¼l; a 5 kb plasmid, between 157 and 2350 ng/Î¼l, <em>etc.</em>).</li><li>Run the following program on the thermocycler:<br/>60 min at 37 Â°C<br/>20 min at 65 Â°C<br/>Hold at 4 Â°C<br/><br/></li></ol></li><li>Top strand synthesis<ol class="npwtwo"><li>Transfer the 20 Î¼l strand preparation reaction on ice and add the following:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td>Component<br/></td><td>Volume (Î¼l)<br/></td><td>Final Concentration or Amount<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O <br/></td><td>26.7<br/></td><td>N/A<br/></td></tr><tr><td>5x Phusion HF Buffer<br/></td><td>20</td><td>1x<br/></td></tr><tr><td>1:1,000 diluted phosphorylated oligo pool (final concentration 8.2 nM total primer)<br/></td><td>4.3<br/></td><td>0.035 pmol<br/></td></tr><tr><td>50 mM DTT<br/></td><td>20</td><td>10 mM<br/></td></tr><tr><td>50 mM NAD<sup>+</sup><br/></td><td>1</td><td>0.5 mM<br/></td></tr><tr><td>10 mM dNTPs<br/></td><td>2</td><td>0.2 mM<br/></td></tr><tr><td>Phusion polymerase (2 U/Î¼l)<br/></td><td>1</td><td>2 U<br/></td></tr><tr><td><em>Taq</em> DNA Ligase (40 U/Î¼l)<br/></td><td>5</td><td>200 U<br/></td></tr></tbody></table></li><li>Then run the following protocol on the thermocycler:<br/><ol class="npwthree"><li>2 min at 98 Â°C<br/></li><li>15 cycles of:<br/>30 s at 98 Â°C<br/>30 s at 55 Â°C<br/>30 s/kb at 72 Â°C<br/>At the end of cycles 5 and 10, add an additional 4.3 Î¼l of 1:1,000 diluted phosphorylated oligo pool. To facilitate this, add pauses after the fifth and tenth cycles of step b above. Adding primers in boluses yields a larger number of transformants while keeping the plasmid-to-primer molar ratio low.<br/></li><li>20 min at 45 Â°C<br/></li><li>Hold at 4 Â°C</li></ol><br/></li></ol></li><li>Column purification<ol class="npwtwo"><li>Clean up the reaction using a Monarch<sup>Â®</sup> PCR &amp; DNA Cleanup Kit (New England BioLabs) according to the manufacturerâs instructions, with the exception of using five volumes of binding buffer instead of two volumes. </li><li>Elute in 15 Î¼l nuclease-free H<sub>2</sub>O at room temperature, waiting five min after applying the eluant to the column before centrifugation.<br/><br/></li></ol></li><li>Bottom strand degradation<ol class="npwtwo"><li>Transfer 14 Î¼l of the column purified reaction eluate to a fresh PCR tube and place on ice. Add the following to the tube (final volume 20 Î¼l):<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td>Component<br/></td><td>Volume (Î¼l)<br/></td><td>Final Concentration or Amount<br/></td></tr><tr><td>10x CutSmart<sup>Â®</sup> Buffer<br/></td><td>2<br/></td><td>1x<br/></td></tr><tr><td>2 U/Î¼l diluted Exonuclease III<br/>(a 50-fold dilution of the stock concentration of 100 U/ Î¼l into 1x CutSmart<sup>Â®</sup>)<br/></td><td>2</td><td>4 U<br/></td></tr><tr><td>Exonuclease I (20 U/Î¼l)<br/></td><td>1<br/></td><td>20 U<br/></td></tr><tr><td>1 U/Î¼l BbvCI.Nb <br/>(a 10-fold dilution into 1x CutSmart<sup>Â®</sup> from the stock) <br/></td><td>1</td><td>1 U<br/></td></tr></tbody></table></li><li>Run the following program on the thermocycler:<br/>60 min at 37 Â°C<br/>20 min at 65 Â°C<br/>Hold at 4 Â°C<br/><br/></li></ol></li><li>Bottom strand synthesis<ol class="npwtwo"><li>Place the 20 Î¼l bottom strand degradation reaction on ice and add the following:<br/><table border="0" bordercolor="#000000" cellpadding="0" cellspacing="0" class="ke-zeroborder" style="width:700px;"><tbody><tr><td>Component<br/></td><td>Volume (Î¼l)<br/></td><td>Final Concentration or Amount<br/></td></tr><tr><td>Nuclease-free H<sub>2</sub>O<br/></td><td>27.7<br/></td><td><br/></td></tr><tr><td>5x Phusion HF Buffer<br/></td><td>20</td><td>1x</td></tr><tr><td>1:20 diluted phosphorylated secondary primer<br/></td><td>4.3<br/></td><td>5 pmol<br/></td></tr><tr><td>50 mM DTT<br/></td><td>20<br/></td><td>10 mM<br/></td></tr><tr><td>50 mM NAD<sup>+</sup><br/></td><td>1</td><td>0.5 mM<br/></td></tr><tr><td>10 mM dNTPs<br/></td><td>2</td><td>0.2 mM<br/></td></tr><tr><td>Phusion polymerase (2 U/Î¼l) <br/></td><td>1</td><td>2 U<br/></td></tr><tr><td>Taq DNA Ligase (40 U/Î¼l)<br/></td><td>5</td><td>200 U<br/></td></tr></tbody></table></li><li>Run the following program on the thermocycler:<br/><ol class="npwthree"><li>30 s at 98 Â°C<br/></li><li>30 s at 55 Â°C<br/></li><li>10 min at 72 Â°C<br/></li><li>20 min at 45 Â°C<br/></li><li>Hold at 4 Â°C<br/><br/></li></ol></li></ol></li><li>Removal of input plasmid<br/>Add 2 Î¼l DpnI (20 U/Î¼l) to the 100 Î¼l bottom strand synthesis PCR and incubate at 37 Â°C for one hour in a thermocycler.<br/><br/></li><li>Column purification<ol class="npwtwo"><li>Clean up the reaction using a Monarch<sup>Â®</sup> PCR &amp; DNA Cleanup Kit (New England BioLabs) according to the manufacturerâs instructions.</li><li>Use five volumes of binding buffer (not two).</li><li>Elute in 6 Î¼l nuclease-free H<sub>2</sub>O at room temperature, waiting five min after applying the eluant to the column before centrifugation.<br/><br/></li></ol></li><li>Transformation<ol class="npwtwo"><li>Transform the entire purified product into high efficiency electrocompetent <em>E. coli</em> according to the manufacturerâs instructions.</li><li>Using high efficiency cells (&gt; 10<sup>9</sup> CFU/Î¼g plasmid DNA) is crucial to the success of the protocol. Plate serial dilutions (<em>e.g.</em>, 10<sup>â4</sup>, 10<sup>â5</sup>, and 10<sup>â6</sup> of the full transformation) on appropriate selective medium in order to determine efficiency.</li><li>Plate the remainder of the transformation on a large square bioassay plate of appropriate selective medium.</li><li>Incubate the plates overnight at 37 Â°C.<br/><br/></li></ol></li><li>Plasmid library preparation<br/>A successful transformation should yield a lawn of colonies on the large bioassay plate. A large scraper, as can be prepared by bending a heated glass Pasteur pipette, is useful for resuspending colonies for plasmid preparation.<br/><ol class="npwtwo"><li>Apply 5-7 ml of liquid medium (such as LB) to the bioassay plate and scrape the colonies off of the agar surface using the scraper.</li><li>Slightly incline the plate so that the cell suspension pools in one corner and transfer the suspension to a sterile 50 ml conical tube with either a 1 ml pipette or a 10 ml serological pipette. Typically, 1-2 ml of the first liquid media dispensed to the plate is absorbed by the agar.</li><li>Continue to add medium, scrape colonies, and transfer the suspension until the agar is clear of all bacterial growth.</li><li>Mix the cell suspension by aggressive vortexing and/or trituration with a serological pipette to completely disperse aggregated biomass. If the biomass is suspended in roughly 10 ml, the culture will be completely opaque and it will be significantly denser than a saturated <em>E. coli</em> liquid culture in LB.</li><li>Finally, recover plasmid DNA from a small amount of the suspension using of the cell suspension using a miniprep kit (<em>e.g.</em>, New England BioLabs Monarch<sup>Â®</sup> Plasmid Miniprep Kit <a class="blackNoUnder" target="_blank"> T1010S </a> ). For a lawn on a bioassay plate resuspended in 10 ml of LB, 300 Î¼l is usually sufficient to obtain 15 Î¼g of high-copy number plasmid DNA.<br/></li></ol></li></ol><p class="pptt" id="biaoti34944">### Data analysis</p><p class="ppzz">After mutagenesis, approximate the number of transformants by counting colonies from the plated dilution series. Specifically, identify the dilution factor where the number of colonies is between 10-100 and multiply the number of colonies on each plate by the corresponding dilution according to the equation below:</p><p class="ppzz"><br/></p><div style="text-align:center;"><em># of transformants = number of colonies</em> Ã <em>dilution factor</em><em></em><br/></div><p><br/></p><p class="ppzz">To approach complete coverage of the desired set of mutations, a general guideline is to recover at minimum ten times more transformants than there are library variants. Recovering many more transformants is preferable. If the distribution of variant frequencies in the library were uniform, roughly 4.6-fold coverage would be required for 99% coverage (Bosley <em>et al.</em>, 2005). However, variant frequencies with nicking mutagenesis are typically distributed log-normally, which necessitates higher fold coverage (Wrenbeck <em>et al.</em>, 2016; Medina-Cucurella <em>et al.</em>, 2019). Figure 2 shows simulated coverage results demonstrating that 100-fold coverage is a better target. Actual coverage can be measured by next generation sequencing, which is described elsewhere (Kowalsky <em>et al.</em>, 2015b).</p><br/><p class="ppzz"><img alt="" class="layerphoto" height="192" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200724/20200724224322_7174.png" width="300"/><br/><strong>Figure 2. Simulated distribution of coverage for 1000-member libraries with uniform or log-normal variant frequency distributions.</strong> Each data point shows mean and standard deviation of library coverage from 100,000 simulated experiments. For most points, the standard deviation is too small to be seen on this scale. The log-normally distributed frequencies were generated by normalizing a lognormal (Î¼ = 4.6, Ï = 1.15) distribution seen in experimental datasets (Wrenbeck <em>et al.</em>, 2016). For log-normally distributed libraries, 100 transformants per library member is a good target.</p><p class="pptt" id="biaoti34945">### Notes</p><ol class="npwtwok"><li>Plasmid preparation for mutagenesis<br/>Preparation of high quality circular closed dsDNA is crucial to the success of the protocol. Plasmid DNA nicked on the strand opposite to that targeted by the nicking enzyme will be degraded completely rather than to closed ssDNA. It is therefore important that plasmid DNA used as input is not nicked, which is best accomplished by using freshly prepared plasmid that has not been subjected to freeze-thaw cycles. Plasmid quality can be assessed by gel electrophoresis. Supercoiled plasmid DNA should be the dominant species in a high-quality sample.</li><li>Efficiency<br/>Using the comprehensive mutagenesis procedure described here, we typically find that 20-50% of transformants are wild type plasmid. Anecdotally, smaller plasmid templates lead to much lower fractions of wild-type transformants (and, as is typical, to much larger numbers of transformants in general). If a large number of transformants is required, it can be helpful to mutagenize only the relevant fragment of a gene in a minimal vector and later subclone the resulting library. If the library will eventually be transformed into yeast, it is particularly useful to transform the yeast with linear vector backbone linear mutagenized fragment library and rely on <em>in vivo</em> homologous recombination.</li><li>Troubleshooting<br/>Frequently observed issues with performing the protocol are given in Table 1. Typically, the first step in troubleshooting nicking mutagenesis should be to attempt the procedure using the GFP control plasmid pEDA5-GFPmut3-Y66H described in Wrenbeck <em>et al.</em> (2016) (Addgene ID 80085). The Y66H mutation encoded in the plasmid eliminates the encoded GFP fluorescence and mutagenesis recovers this fluorescence, allowing easy screening for success.</li><li>The next step should be to confirm ssDNA template preparation (Procedure E) and regeneration (Procedure F) using your unique plasmid and primer sets with both Nt.BbvCI and Nb.BbvCI. Perform a scaled-up digestion with the nicking enzyme, Exonuclease I, and Exonuclease III. Use 20 Î¼l of this reaction for the 100 Î¼l polymerization/ligation reaction. Finally, analyze uncut plasmid, nicked/degraded plasmid, and regenerated template by agarose gel electrophoresis. It is helpful to use SYBR Safe DNA gel stain (Thermo Fisher <a class="blackNoUnder" target="_blank"> S33102 </a> ), because it causes ssDNA to appear orange and dsDNA to appear green when illuminated by a blue-light transilluminator.<br/><br/><strong>Table 1. Common list of issues with troubleshooting suggestions</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200805/20200805015020_3788.jpg" width="1000"/> </li></ol><p class="pptt" id="biaoti34946">### Recipes</p><ol class="npwtwok"><li>Single-use aliquots<br/>10 mM ATP<br/>50 mM NAD<sup>+</sup><br/>10 mM dNTPs<br/>Store at -20 Â°C<br/>ATP and NAD<sup>+</sup> should not be freeze-thawed</li><li>50 mM DTT<br/>50 mM DTT in nuclease-free water and store aliquots at -20 Â°C<br/></li></ol><p class="pptt" id="biaoti34947">### Acknowledgments</p><p class="ppzz">Research reported in this publication was supported by the National Institute Of Allergy And Infectious Diseases of the National Institutes of Health under Award Number R01AI141452 to T.A.W. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. The authors wish to acknowledge Emily Wrenbeck, Justin Klesmith, Jim Stapleton, and Angelica Medina-Cucurella for original research upon which this protocol was derived.</p><p class="pptt" id="biaoti34948">### Competing interests</p><p class="ppzz">T.A.W. is on a U.S. patent application 16/115.029 covering the nicking mutagenesis method. P.J.S and Z.T.B. do not have competing interests.</p><p class="pptt" id="biaoti34949">### References</p><ol class="npwtwok"><li>Bosley, A. D. and Ostermeier, M. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15857784" target="_blank">Mathematical expressions useful in the construction, description and evaluation of protein libraries.</a> <em>Biomol Eng</em> 22(1-3): 57-61.</li><li>Doud, M. B., Hensley, S. E. and Bloom, J. D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28288189" target="_blank">Complete mapping of viral escape from neutralizing antibodies.</a> <em>PLoS Pathog</em> 13(3): e1006271.</li><li>Faber, M. S., Van Leuven, J. T., Ederer, M. M., Sapozhnikov, Y., Wilson, Z. L., Wichman, H. A., Whitehead, T. A. and Miller, C. R. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31825605" target="_blank">Saturation mutagenesis genome engineering of infective PhiX174 bacteriophage via unamplified oligo pools and golden gate assembly.</a> <em>ACS Synth Biol</em> 9(1): 125-131. </li><li>Faber, M. S., Wrenbeck, E. E., Azouz, L. R., Steiner, P. J. and Whitehead, T. A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31400199" target="_blank">Impact of <em>in vivo</em> protein folding probability on local fitness landscapes.</a> <em>Mol Biol Evol</em> 36(12): 2764-2777.</li><li>Fowler, D. M. and Fields, S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25075907" target="_blank">Deep mutational scanning: a new style of protein science.</a> <em>Nat Methods</em> 11(8): 801-807. </li><li>Firnberg, E. and Ostermeier, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23284860" target="_blank">PFunkel: efficient, expansive, user-defined mutagenesis.</a> <em>PLoS One</em> 7(12): e52031.</li><li>Kowalsky, C. A., Faber, M. S., Nath, A., Dann, H. E., Kelly, V. W., Liu, L., Shanker, P., Wagner, E. K., Maynard, J. A., Chan, C. and Whitehead, T. A. (2015a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26296891" target="_blank">Rapid fine conformational epitope mapping using comprehensive mutagenesis and deep sequencing.</a> <em>J Biol Chem</em> 290(44): 26457-26470. </li><li>Kowalsky, C. A., Klesmith, J. R., Stapleton, J. A., Kelly, V., Reichkitzer, N. and Whitehead, T. A. (2015b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25790064" target="_blank">High-resolution sequence-function mapping of full-length proteins.</a> <em>PLoS One</em> 10(3): e0118193. </li><li>Kunkel, T. A. (1985). <a href="http://www.ncbi.nlm.nih.gov/pubmed/3881765" target="_blank">Rapid and efficient site-specific mutagenesis without phenotypic selection.</a> <em>Proc Natl Acad Sci U S A</em> 82(2): 488-492. </li><li>Leaver-Fay, A., Tyka, M., Lewis, S. M., Lange, O. F., Thompson, J., Jacak, R., Kaufman, K., Renfrew, P. D., Smith, C. A., Sheffler, W., Davis, I. W., Cooper, S., Treuille, A., Mandell, D. J., Richter, F., Ban, Y. E., Fleishman, S. J., Corn, J. E., Kim, D. E., Lyskov, S., Berrondo, M., Mentzer, S., Popovic, Z., Havranek, J. J., Karanicolas, J., Das, R., Meiler, J., Kortemme, T., Gray, J. J., Kuhlman, B., Baker, D. and Bradley, P. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21187238" target="_blank">ROSETTA3: an object-oriented software suite for the simulation and design of macromolecules.</a> <em>Methods Enzymol</em> 487: 545-574.</li><li>Medina-Cucurella, A. V., Steiner, P. J., Faber, M. S., Beltran, J., Borelli, A. N., Kirby, M. B., Cutler, S. R. and Whitehead, T. A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31297523" target="_blank">User-defined single pot mutagenesis using unamplified oligo pools.</a> <em>Protein Eng Des Sel</em> 32(1): 41-45. </li><li>Romero, P. A., Tran, T. M. and Abate, A. R. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26040002" target="_blank">Dissecting enzyme function with microfluidic-based deep mutational scanning.</a> <em>Proc Natl Acad Sci U S A</em> 112(23): 7159-7164.</li><li>Van Blarcom, T., Rossi, A., Foletti, D., Sundar, P., Pitts, S., Bee, C., Melton Witt, J., Melton, Z., Hasa-Moreno, A., Shaughnessy, L., Telman, D., Zhao, L., Cheung, W. L., Berka, J., Zhai, W., Strop, P., Chaparro-Riggers, J., Shelton, D. L., Pons, J. and Rajpal, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25284753" target="_blank">Precise and efficient antibody epitope determination through library design, yeast display and next-generation sequencing.</a> <em>J Mol Biol</em> 427(6 Pt B): 1513-1534.</li><li>Wrenbeck, E. E., Klesmith, J. R., Stapleton, J. A., Adeniran, A., Tyo, K. E. and Whitehead, T. A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27723752" target="_blank">Plasmid-based one-pot saturation mutagenesis.</a> <em>Nat Methods</em> 13(11): 928-930.<br/></li></ol>