<body>
            <h1>A Rigorous Quantitative Approach to Analyzing Phagocytosis Assays</h1>
            <p><strong>Authors:</strong> Michael D. Caponegro, Kaitlyn Koenig Thompson, Maryam Tayyab, Stella E. Tsirka</p>
            <h2>Abstract</h2>
            <p>Studying monocytic cells in isolated systems <em>in vitro</em> contributes significantly to the understanding of innate immune physiology. Functional assays produce read outs which can be used to measure responses to selected stimuli, such as pathogen exposure, antigen loading, and cytokine stimulation. Integration of these results with high quality <em>in vivo</em> models allows for the development of therapeutics which target these cell populations. Current methodologies to quantify phagocytic function of monocytic cells <em>in vitro</em> either measure phagocytic activity of individual cells (average number of beads or particles/cell), or a population outcome (% cells that contain phagocytosed material). Here we address technical challenges and shortcomings of these methods and present a protocol for collecting and analyzing data derived from a functional assay which measures phagocytic activity of macrophage and macrophage-like cells. We apply this method to two different experimental conditions, and compare to existing work flows. We also provide an online tool for users to upload and analyze data using this method.</p>
            <p><strong>Keywords:</strong> Monocyte, Macrophage, Microglia, Phagocytosis, Quantification, Analysis, Microscopy</p>
            <h2>Background</h2><p>Monocytic lineage cells (monocytes, macrophages, and microglia) survey tissue for pathological threats, and act as hubs of the innate immune system. They exercise innate immune functions in numerous ways: by secreting cytokines and chemokines, acting through complement signaling pathways, and phagocytosing pathogens and debris. The extent of macrophage activation can tip the balance of immune response within the body during health and disease. In diseases such as cancer, the activation of monocytic cells can result in either pro- or anti- tumoral responses, which trigger different disease outcomes. Similarly, monocytic cells modulate the progression and remission of auto-immune and inflammatory diseases. It is therefore critical to characterize these cells and their responses, to identify steps which may be exploited therapeutically.<br/> <br/><em>In vitro</em> systems interrogate the functions of these innate immune cells in response to stimuli via specific assays. The phagocytosis assay is an essential tool in characterizing the magnitude of monocytic cell activation. Several variations, such as pulse-chase assay, particle loading, and/or bead loading, exist. Upon stimulation of the monocytic cell population with drugs, cytokines, or preconditioned cell media, the results of the assay report to what extent the cells have increased or decreased their ability to engulf particles from the environment, thus giving insight into how such stimuli may affect monocytic cells during health and disease <em>in vivo</em>.<br/> <br/>Here we discuss some of the technical challenges associated with deriving high-quality data from standard phagocytosis assays. The variability of phagocytic activity of cells within a single sample makes this a particularly difficult function to assess. Most current studies count phagocytosed particles per cell, however, individual particles can be difficult to discern, and this measure can be misleading in cells that have higher baseline phagocytosis. Other studies simply analyze how many cells are phagocytosing (versus not), which gives a very broad and less quantitative picture of phagocytosis. Here, we have developed and propose the use of a new technical pipeline to collect and more sensitively analyze phagocytosis data. We then apply it to two different experiments for which macrophage phagocytosis is often assayed.</p>
        </body>

<p class="pptt" id="biaoti39621">## Materials and Reagents</p><ol class="npwtwok"><li>Glass coverslips (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-545-100 </a> )</li><li>40 Î¼m cell strainer (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 22-363-547 </a> )</li><li>Adult C57Bl/6 mice</li><li>L929 cells [or 100 U/ml recombinant M-CSF (Prospec, catalog number: <a class="blackNoUnder" target="_blank"> CYT-439 </a> )]</li><li>GL261 murine glioma cells</li><li>2.5% avertin</li><li>Hankâs Buffered Saline Solution (HBSS)</li><li>Fluorescent latex beads (carboxylate-modified polystyrene, fluorescent red; 0.5-Î¼m mean particle size, Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L3280 </a> )</li><li>Fluorescence-conjugated AlexaFluor 488 goat anti-rabbit (Life Technologies, catalog number: <a class="blackNoUnder" target="_blank"> A-11008 </a> )</li><li>Hankâs Buffered Saline Solution (HBSS) (Corning, catalog number: <a class="blackNoUnder" target="_blank"> MT-20-023-CV </a> )</li><li>DMEM (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 10-017-CM </a> )</li><li>FBS (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 10221-866 </a> )</li><li>Antibiotic-Antimycotic (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 10-221-866 </a> )</li><li>Sodium pyruvate (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 13-115E </a> )</li><li>PFA (4% paraformaldehyde) (Acros, catalog number: <a class="blackNoUnder" target="_blank"> AC169650010 </a> )</li><li>Triton X-100 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> X100 </a> )</li><li>Normal goat serum (Southern Biotech, catalog number: <a class="blackNoUnder" target="_blank"> 0060-01 </a> )</li><li>BSA (VWR, catalog number: <a class="blackNoUnder" target="_blank"> RLBSA50 </a> )</li><li>DAPI Fluormount (Southern Biotech, catalog number: <a class="blackNoUnder" target="_blank"> 0100-20 </a> )</li><li>Iba1 antibody (Wako USA, catalog number: <a class="blackNoUnder" target="_blank"> 19-19741 </a> )</li><li>LPS (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L6511 </a> )</li><li>Mouse IFNÎ³ (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11276905001 </a> )<br/></li></ol><p class="pptt" id="biaoti39622">### Equipment</p><ol class="npwtwok"><li>Leica SP8X confocal imaging system<br/>Beads (Î»<sub>ex</sub> ~575 nm; Î»<sub>em</sub> ~610 nm)<br/>AlexaFluor 488 (Î»<sub>ex</sub> 490 nm; Î»<sub>em</sub> ~525 nm)</li><li>-80 Â°C freezer<br/></li></ol><p class="pptt" id="biaoti39623">### Software</p><ol class="npwtwok"><li>Fiji (Schindelin <em>et al.</em>, 2012)</li></ol><p class="pptt" id="biaoti39624">### Procedure</p><ol class="npwone"><li>Cell Culture<br/><em>Note: The cells, unless otherwise noted, are grown in 10% FBS with 1% Antibiotic-Antimycotic at 5% CO<sub>2</sub> and 37 Â°C.</em><br/><ol class="npwtwo"><li>Generate glioma conditioned media (GCM) using GL261 murine glioma cells which are grown to 80-90% confluency and serum starved for 24 h. The media are collected and stored at -80 Â°C until use.</li><li>Generate L cell media by collecting it from L929 cells grown to 90-100% confluency and maintained for 10 days. These media are rich in colony stimulating factor 1 (CSF1), a factor used to drive proliferation and differentiation of monocytic cells into macrophages.</li><li>Monocytes are cultured from adult C57Bl/6 mice as previously described (Zhang <em>et al.</em>, 2008): Briefly, animals are deeply anesthetized with 2.5% avertin and euthanized by cervical dislocation. Femurs and tibiae are dissected, and the internal bone cavity is flushed with 1% serum in Hankâs Buffered Saline Solution (HBSS). The collected bone marrow is filtered through a 40-Î¼m cell strainer and plated in complete media containing DMEM, 10% FBS, 1% Antibiotic-Antimycotic, 1% sodium pyruvate, and 20% L cell media (or 100 U/ml recombinant M-CSF).</li><li>Cells are kept at 5% CO<sub>2</sub> and 37 Â°C. Macrophage differentiation is achieved after 7-9 days of culture.<br/><br/></li></ol></li><li>Phagocytosis assay<br/><u>Experiment 1</u>: Macrophages are plated on glass coverslips and incubated with either complete media, as described above, or complete media plus 50% GCM for 24 h.<br/><u>Experiment 2</u>: Macrophages, as above, are incubated with saline (control) or both 100 ng/ml LPS (Sigma) and 100 U/ml IFNÎ³ (Roche) for 24 h.<br/><ol class="npwtwo"><li>Fluorescent latex beads (carboxylate-modified polystyrene, fluorescent red; 0.5-Î¼m mean particle size, Sigma, <a class="blackNoUnder" target="_blank"> L3280 </a> ) are added to a final concentration of 0.1 Î¼g/ml, and incubated for 2 h. After rinsing with HBSS to remove excess beads, cells are fixed in 4% PFA at room temperature for 20 min.</li><li>Fixed cells are rinsed with PBS and blocked for 1 h in 0.1% Triton X-100 and 5% normal goat serum. They are incubated with anti-Iba1 (1:500, Wako) in 0.01% Triton X-100 and 30 mg/ml BSA overnight at 4 Â°C.</li><li>The cells are then rinsed with PBS, and incubated with fluorescence-conjugated AlexaFluor 488 goat anti-rabbit for 1 h.</li><li>Cells are rinsed again with PBS and coverslipped using DAPI Fluormount (Southern Biotech).<br/><br/></li></ol></li><li>Confocal Imaging<br/>Images are acquired on a Leica SP8X confocal imaging system. 10 Âµm z-stacks are taken at 40x at a resolution of 512 x 512 pixels. Confocal images are processed using Fiji (Schindelin <em>et al.</em>, 2012). For this procedure, five fields are imaged, and at least 100 cells are quantified in total per treatment. For evaluation of phagocytosis, total number of cells, and number of cells with internalized beads are counted. The images are further analyzed by counting the number of beads per cell, and measuring cell area and the integrated density of sum-projected pixels from the fluorescent bead channel. Integrated density values are analyzed for outliers before downstream analysis. The step-by-step image analysis protocol is provided in <a href="https://os.bio-protocol.org/doc/upprotocol/p3124/Abstract10_0_20220222105241799/3698%20Supplementary%20document%201.docx">Supplementary document 1</a>.<br/></li></ol><p class="pptt" id="biaoti39625">### Data analysis</p><p class="ppzz">Data analysis is performed in R for Windows. The method and interactive UI can be accessed via GitHub for users to upload and analyze data (<a href="https://mcap91.shinyapps.io/Monocyte-Phagocytosis-Assay/" target="_blank">RShinyApp</a>; <a href="https://github.com/mcap91/Monocyte-Phagocytosis-Assay" target="_blank">GitHub Repo</a>).</p><p class="ppzz"><br/><strong>Results</strong><br/><em><strong>A high throughput method of data collection</strong></em><br/>In the first experiment, we quantified macrophage phagocytosis in response to GCM. Tumor associated macrophages are known to be highly activated and pro-tumorigenic. Their phagocytic activity may be related to their tumor supporting behavior by mechanisms that promote wound healing and anti-inflammatory phenotypes. Confocal imaging allows complete 3D capture of macrophages engulfing fluorescent beads (Figure 1A). Standard methods attempt to count the number of beads that each cell has engulfed. However, some cells exhibit low, basal level of phagocytic activity, while others become stimulated and intake more beads. This non-standardized phagocytic activity can be problematic for quantification and analysis. The beads overlap within and across z-stack layers, making distinguishing and counting individual beads difficult. Further, counting the number of beads per cell in a single field of view can take more than 15 min, making data acquisition time-consuming for large numbers of technical replicates. This highlights the technical challenges associated with this method of quantification.<br/>    To address this, others have quantified the number of cells positive or negative for bead signal, thereby creating a ratio of cells which are phagocytic in response to treatment (Lian <em>et al.</em>, 2016; Roy <em>et al.</em>, 2018). While more time-efficient, this method does not capture the changes in <em>relative amount</em> of phagocytosis occurring, <em>i.e.</em>, phagocytic activity of each cell. As seen in Figure1A, cells engulf variable amounts of beads. Other methods to access phagocytosis, such as fluorescently-activated cell sorting (FACS), have also been used (Pul <em>et al.</em>, 2013), however they require more preparation time, experimental controls (negative fluorescent controls, compensation techniques <em>etc.</em>), and several costly reagents and conjugated antibodies.<br/>    We developed a technical pipeline which takes advantage of the discrete fluorescent signal produced by the latex beads. By summing the fluorescent pixels across the z-stack of the bead channel, we create a single metric. By using region of interest (ROI) functionality, we can capture this metric for each cell within a field of view. This method of data collection allows the analysis of all cells within a field in under 2 min, appreciably faster than counting individual beads/cell. It allows hundreds of cells to be analyzed in a single experiment.<br/>    To validate this fluorescent metric, we compared the signal produced by this method with the counted number of beads per cell, as previously performed (Figure 1B). Simple linear regression analysis was used. The model yielded high significance (<em>P</em> &lt; 0.001) with a slope coefficient of 0.53 and an adjusted r<sup>2</sup> = 0.363. This relationship was also confirmed with a Spearmanâs rho value of 0.683. Thus, generating a fluorescent signal by summing the pixels throughout the z-stack is an accurate and sensitive measurement, reflecting the number of beads each cell has engulfed. It provides a high throughput alternative to existing quantification methods.<br/><br/><img alt="" class="layerphoto" height="200" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200726/20200726070920_8648.png" width="300"/><br/><strong>Figure 1. Confocal imaging and analysis of macrophages following incubation with fluorescent latex beads.</strong> A. Macrophages stained for Iba1 (green) can be seen engulfing fluorescent latex beads (red) in both treatment conditions. Orthogonal views are also shown, demonstrating complete uptake of the beads by the cells. Note the varying levels of bead signal among cells. DAPI marks nuclei. B. Scatter plot comparing the fluorescence signal derived from beads versus the actual number of beads counted per cell. y = -22.40619 + 0.52782x. <em>P</em> &lt; 2.2e<sup>-16</sup>. Adjusted r<sup>2</sup> = 0.3634. Spearmanâs = 0.683. C. Histogram and density plots of hypothetical datasets. For both populations <em>dist1</em> and <em>dist2</em>, n = 100, mean = 50, median = 50. D. Histogram and density plots of experimentally derived data from panel A, Experiment 1. Complete descriptive statistics are in <a href="https://os.bio-protocol.org/attached/file/20200721/3698 Table S1.docx" target="_blank">Table S1</a>. Scale bar for panel A is 20Î¼.<br/><br/><strong><em>Utilizing empirical cumulative distribution functions as a measure of phagocytic activity</em></strong><br/>The data rendered by this method create a continuous distribution of fluorescent signal across all cells within an experimental group (Figure 1D). Standard statistical methods would use a parametric or non-parametric statistical test to compare the means or medians of each distribution, respectively. However, let us consider two hypothetical distributions, for which the probability distribution functions (PDF) are non-congruent, but the mean and median between them are equal (Figure 1C, <a href="https://os.bio-protocol.org/attached/file/20200721/3698 Table S1.docx" target="_blank">Table S1</a>). Both parametric and non-parametric tests reject the alternative hypothesis and infer no difference between the two distributions, yet it is clear visually that the distribution of events is vastly different between the two samples. This suggests that for non-parametric, multi-modal distributions the standard <em>t</em>-test or Mann-Whitney U test is insufficient, and can break down under certain conditions. Interpreted biologically, these tests may lack the ability to detect the âshiftsâ in phagocytic influx that occur when macrophages are challenged, for example by treatment with GCM.<br/>    If we consider the empirical cumulative distribution function (ECDF) of these distributions, it more aptly represents the changes that exist as the PDF expands (Figure 2A). To compare these distributions, we applied principals from the field of optimal mass transport, otherwise known as Earth moverâs theory. These methods seek to compare data based on their underlying ECDFs. To compute differences between the ECDF derived from our hypothetical and experimental data, we next performed the Kolmogorov-Smirnov test (KS test), where the D statistic given by:</p><p class="ppzz"><br/></p><p class="ppzz" style="text-align:center;"><img alt="" class="layerphoto" height="32" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200726/20200726071038_1198.jpg" width="220"/> </p><p class="ppzz"><br/>Where F<sub>1,n</sub> and F<sub>2,m</sub> are the ECDF from two 1 dimensional datasets, and sup is the supremum function, computes the absolute maximum distance between the ECDFs (Figure 2A). To reject the null hypothesis Ho, which states that both samples come from the same population, at a given significance levelÎ±, D<sub>n,m</sub> &gt; D<sub>n,m,Î±</sub>, where D<sub>n,m,Î±</sub> is the critical value given by</p><p class="ppzz"><br/></p><p class="ppzz" style="text-align:center;"><img alt="" class="layerphoto" height="63" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200726/20200726071111_2027.jpg" width="150"/> </p><p class="ppzz"><br/>For the hypothetical distributions <em>dist1</em> and <em>dist2</em> the two sample KS test yields D = 0.5 and <em>P</em> = 2.778e<sup>-11</sup>. This clearly indicates the advantage of using this method for multimodal distributions over standard <em>t</em>-test or Mann-Whitney U test. When applied to our experimental data, KS results in D = 0.24 and <em>P</em> = 2.2e<sup>-4</sup> (Figure 2B), indicating that the probability that a macrophage has increased phagocytic activity is more accurately reflected in the collected data by the shift in its PDF, which is captured by analyzing the ECDF. These results also demonstrate that exposure to GCM significantly increases macrophage phagocytic activity.<br/><br/><em><strong>Comparing KS test to reported methods</strong></em><br/>To further assess KS test sensitivity, we performed a second experiment exposing macrophages to the canonical pro-inflammatory/M1 activating agents LPS and IFNÎ³. We calculated the percentage of phagocytosing cells by dividing the number of cells positive for engulfed beads by the total number of cells within the field of view, as reported (Lian <em>et al.</em>, 2016; Roy <em>et al.</em>, 2018). The results yielded a trend towards increase in phagocytosis in cells exposed to LPS and IFNÎ³, however the high variability and lack of technical replicates fail to reject the null hypothesis using standard two sample parametric and non-parametric tests (Figure 2C). We then employed the KS test to generate PDF and ECDF of the two treatment groups (Figure 2D). This resulted in D = 0.15 and <em>P</em> = 0.04, thus detecting a small but significant shift in phagocytic activity, which can be seen by the rightward shift in the histogram, density curve, and ECDF. While not as large of an increase relative to that observed after GCM treatment, this result agrees with published literature (Wu <em>et al.</em>, 2009; Scheraga <em>et al.</em>, 2016; Caponegro <em>et al.</em>, 2019) and validates the approach, highlighting its high throughput and sensitivity.</p><p class="ppzz"><br/></p><p class="ppzz"><img alt="" class="layerphoto" height="313" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200726/20200726071257_5773.png" width="300"/> <br/><strong>Figure 2. Comparison of quantification of phagocytosis using traditional metrics vs the proposed new method of analysis.</strong> A. Empirical cumulative distribution functions (ECDFs) of non-parametric, multimodal hypothetical datasets <em>dist1</em> and <em>dist2</em>. The ECDF more accurately portrays the changes that occur in the underlying PDF. B. ECDFs of data experimentally derived from Figure 1A, Experiment 1. The D statistic is plotted, showing the maximal distance between the curves. C. Using traditional metrics to quantify phagocytosis <em>in vitro</em>, cells that were positive for bead engulfment were divided by the total number of cells per field of view. Data are derived from Experiment 2. D. <em>Top panel</em>: Using the new methodology histogram and density curves of data derived from Experiment 2. A right-shift in the PDF is denoted by an arrow. <em>Bottom panel</em>: ECDFs of data derived from Experiment 2. The D statistic is plotted, showing the maximal distance between the curves.<br/><br/><strong>Discussion</strong><br/>There is a lack of standardization in collecting and analyzing phagocytosis assay data. While several techniques offer solutions to quantification of actively phagocytosing cells, the increase of phagocytic activity, <em>i.e.</em>, the relative <em>amount</em> or magnitude of phagocytic increases/decreases, is not investigated. Here we propose using the KS test following transformation of the continuous data into an ECDF. The KS test is supported by the theory of optimal mass transport, which ultimately describes two distributions, and seeks to calculate the distances between them. Many principles of optimal mass transport theory, such as those that utilize the Wasserstein distance metric or Earth Moverâs distance, have been used in various areas of research from engineering, to machine learning, to cell biology (Kolouri <em>et al.</em>, 2018; Schiebinger <em>et al.</em>, 2019). Such analyses of CDFs thus deeply describe the higher order PDF and real-world data from which it is derived. This is a sensitive and accurate technical pipeline to collect and analyze data from monocytic cell phagocytosis assays. The finer detail of data derived reflects the magnitude of changes these innate immune cells undergo. While we show that the signal captured by our method correlates directly with number of beads per cell, investigators should be aware of experimental caveats, such as background from non-engulfed beads, cell-cell engulfment, as well as incomplete capture of the axial plane.<span style="font-size:10pt;font-weight:lighter;white-space:nowrap;"></span> </p><div><p class="pptt" id="biaoti39626">### Acknowledgments</p><p class="ppzz">We would like to thank members of the Tsirka lab and Dr Rome Sandhu for helpful discussions and edits. This work was partially supported by K12GM102778 (KKT), R25CA214272 (MT) and SBU seed funding (SET).</p><p class="pptt" id="biaoti39627">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti39628">### References</p><ol class="npwtwok"><li>Caponegro, M. D., Torres, L. F., Rastegar, C., Rath, N., Anderson, M. E., Robinson, J. K. and Tsirka, S. E. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29962076" target="_blank">Pifithrin-mu modulates microglial activation and promotes histological recovery following spinal cord injury.</a> <em>CNS Neurosci Ther</em> 25(2): 200-214. </li><li>Kolouri, S., Pope, P. E., Martin, C. E. and Rohde, G. K. (2018) <a href="https://arxiv.org/abs/1804.01947" target="_blank">Sliced-wasserstein autoencoder: an embarrassingly simple generative model.</a> <em>arXiv e-prints</em>.</li><li>Lian, H., Roy, E. and Zheng, H. (2016). <a href="https://bio-protocol.org/cn/e1988" target="_blank">Microglial phagocytosis assay.</a> <em>Bio-protocol</em> 6(21): e1988.</li><li>Pul, R., Chittappen, K. P. and Stangel, M. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23813376" target="_blank">Quantification of microglial phagocytosis by a flow cytometer-based assay.</a> <em>Methods Mol Biol </em>1041: 121-127.</li><li>Roy, S., Bag, A. K., Dutta, S., Polavaram, N. S., Islam, R., Schellenburg, S., Banwait, J., Guda, C., Ran, S., Hollingsworth, M. A., Singh, R. K., Talmadge, J. E., Muders, M. H., Batra, S. K. and Datta, K. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30111533" target="_blank">Macrophage-derived neuropilin-2 exhibits novel tumor-promoting functions.</a> <em>Cancer Res</em> 78(19): 5600-5617. </li><li>Scheraga, R. G., Abraham, S., Niese, K. A., Southern, B. D., Grove, L. M., Hite, R. D., McDonald, C., Hamilton, T. A. and Olman, M. A. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26597012" target="_blank">TRPV4 mechanosensitive ion channel regulates lipopolysaccharide-stimulated macrophage phagocytosis.</a> <em>J Immunol</em> 196(1): 428-436.</li><li>Schiebinger, G., Shu, J., Tabaka, M., Cleary, B., Subramanian, V., Solomon, A., Gould, J., Liu, S., Lin, S., Berube, P., Lee, L., Chen, J., Brumbaugh, J., Rigollet, P., Hochedlinger, K., Jaenisch, R., Regev, A. and Lander, E. S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30712874" target="_blank">Optimal-transport analysis of single-cell gene expression identifies developmental trajectories in reprogramming.</a> <em>Cell</em> 176(4): 928-943 e922.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682. </li><li>Thompson, K.K., and Tsirka, S.E. (2020). <a href="https://pubmed.ncbi.nlm.nih.gov/33168944/" target="_blank">Guanabenz modulates microglia and macrophages during demyelination.</a> <em>Scientific reports</em> 10, 19333.</li><li>Wu, T. T., Chen, T. L. and Chen, R. M. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19735705" target="_blank">Lipopolysaccharide triggers macrophage activation of inflammatory cytokine expression, chemotaxis, phagocytosis, and oxidative ability via a toll-like receptor 4-dependent pathway: validated by RNA interference.</a> <em>Toxicol Lett</em> 191(2-3): 195-202.</li><li>Zhang, X., Goncalves, R. and Mosser, D. M. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19016445" target="_blank">The isolation and characterization of murine macrophages.</a> <em>Curr Protoc Immunol Chapter</em> 14: Unit 14 11.<br/></li></ol></div>