<body>
            <h1>SMART (Single Molecule Analysis of Resection Tracks) Technique for Assessing DNA end-Resection in Response to DNA Damage</h1>
            <p><strong>Authors:</strong> Angela Altieri, Milena DellâAquila, Francesca Pentimalli, Antonio Giordano, Alfano  Luigi</p>
            <h2>Abstract</h2>
            <p>DNA double strand breaks (DSBs) are among the most toxic lesions affecting genome integrity. DSBs are mainly repaired through non-homologous end joining (NHEJ) and homologous recombination (HR). A crucial step of the HR process is the generation, through DNA end-resection, of a long 3â² single-strand DNA stretch, necessary to prime DNA synthesis using a homologous region as a template, following DNA strand invasion. DNA end resection inhibits NHEJ and triggers homology-directed DSB repair, ultimately guaranteeing a faithful DNA repair. Established methods to evaluate the DNA end-resection process are the immunofluorescence analysis of the phospho-S4/8 RPA32 protein foci, a marker of DNA end-resection, or of the phospho-S4/8 RPA32 protein levels by Western blot. Recently, the Single Molecule Analysis of Resection Tracks (SMART) has been described as a reliable method to visualize, by immunofluorescence, the long 3â² single-strand DNA tails generated upon cell treatment with a S-phase specific DNA damaging agent (such as camptothecin). Then, DNA tract lengths can be measured through an image analysis software (such as Photoshop), to evaluate the processivity of the DNA end-resection machinery. The preparation of DNA fibres is performed in non-denaturing conditions so that the immunofluorescence detects only the specific long 3â² single-strand DNA tails, generated from DSB processing.</p>
            <p><strong>Keywords:</strong> DNA repair, Homologous Recombination, DNA end-resection, BrdU, Immunofluorescence</p>
            <h2>Background</h2><p>Genomic instability is one of the enabling characteristics leading to tumour development (Hanahan and Weinberg, 2011). Many sources of DNA damage, either endogenous or exogenous, induce various DNA lesions including double strand breaks (DBSs), which activate the DNA damage response (DDR)âthe cell process aimed at preserving DNA integrity. Two main repair pathways are involved in the DSBs repair: non-homologous end joining (NHEJ) and homologous recombination (HR) (Mao <em>et al.</em>, 2008). The HR process prevents the loss of genetic information upon DNA damage through a faithful repair using a complementary DNA sequence. By inhibiting the error prone NHEJ and triggering homology-directed DSB repair, DNA end resection has a crucial function in directing the repair pathway choice towards a faithful repair. So, to promote a correct HR, the DSBs are processed through the DNA end-resection machinery which is necessary to generate the long 3â² single-strand DNA (ssDNA) tails essential for the homologous strand invasion. DNA end-resection is a finely regulated process; the first step consists of the recruitment of the MRN complex (MRE11-RAD50-NBSI) and CtIP (RBBP8) onto the DNA lesions to produce short stretches of ssDNA (Stracker and Petrini, 2011) followed by further end-resection mediated by exonuclease I (EXOI) or DNA replication helicase/nuclease 2 (DNA2) in complex with the Bloom syndrome helicases (BLM) (Nimonkar <em>et al.</em>, 2011). Subsequently, the replication protein A (RPA) complex binds the ssDNA generated by DNA end-resection, preventing the formation of DNA hairpins (Chen <em>et al.</em>, 2013) and to facilitate the loading of RAD51 required for the strand exchange process (Krogh and Symington, 2004).<br/><br/>The evaluation of DNA end-resection is crucial to dissect the molecular mechanisms underlying this finely tuned process and to identify the key players involved. Various methods can be used to assess the DNA end-resection process indirectly, analyzing phosphorylated S4/8 RPA32 as mentioned above, or RAD51 foci, or detection of BrdU in ssDNA following long incubation time (TkÃ¡Ä <em>et al.</em>, 2016). A method to determine the length of resection and the extent of ssDNA at a specific DSB site was developed in 2014 by Tania Paull, although this is based on the expression of an endonuclease (such as AsiSI) to induce DSBs at specific sites within the genome, which is then followed by PCR analysis (Zhou <em>et al.</em>, 2014).<br/> <br/>The SMART assay was first developed by Huertas and co-authors as a reliable method to measure the length of resected DNA following exposure to any damaging agent in any cellular system, at the level of single molecules (Cruz-GarcÃ­a <em>et al.</em>, 2014). The assay was based on the previously developed DNA combing assay, which enables the physical stretching of DNA fibres onto a solid support allowing visualization through immunofluorescence (Alfano <em>et al.</em>, 2016 and 2017). We recently identified the RNA binding protein HNRNPD as a new player in the HR process and used the SMART assay, along with other techniques, to evaluate DNA end-resection (Alfano <em>et al.</em>, 2019). We pulse-labeled HeLa cells with the halogenated IdU (5-Iodo-2â²-deoxyuridine)âalthough any other pyrimidine analogue can be used, such as BrdU (Bromodeoxyuridine)or CldU (5-Chloro-2'-deoxyuridine)âwhich is incorporated during S phase into the newly synthesized DNA, for approximately 24 h followed by treatment with the DNA damaging agent, camptothecin. At the end of drug incubation time, cells were lysed and the DNA fibres were extracted in non-DNA denaturing conditions (avoiding hydrochloric acid treatment); this is an important criterion for the assay, because the anti-BrdU antibody is able to recognize the DNA-incorporated IdU only when DNA is in the single strand conformation, as is the end-resected DNA. Through the SMART assay, it is possible to visualize the long 3â² single-strand DNA tails and measure the DNA tracts evaluating the efficiency of the endogenous resection machinery without genetic manipulations. Finally, this technique can be used in any model cellular system from bacteria to humans.<br/> <br/>The DNA fibres, stretched on a microscope slide, and challenged with the anti-BrdU Ab and the secondary fluorescinated Ab were then visualized through immunofluorescence microscopy. Analysis through the SMART assay allowed us to quantify the amount of resected DNA: by using an image analysis software (Photoshop CS5), we were able to measure the length of the DNA fibre tracts analysing the efficiency of the DNA end-resection machinery.</p>
        </body>

<p class="pptt" id="biaoti34977">## Materials and Reagents</p><ol class="npwtwok"><li>6 cm dish (Corning Inc., catalog number: <a class="blackNoUnder" target="_blank"> 430166 </a> )</li><li>Silane Prep-slide (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S4651 </a> )</li><li>HeLa cells (ATCC, <a class="blackNoUnder" target="_blank"> CCL2 </a> , catalog number: <a class="blackNoUnder" target="_blank"> CCL2 </a> )</li><li>Anti-BrdU clone BU1/75 (ICR1) (AbdSerotec, catalog number: <a class="blackNoUnder" target="_blank"> OBT0030CX </a> )</li><li>Alexa Fluor 488-conjugated chicken anti-rat (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A-21470 </a> )</li><li>Anti phospho-RPA32 S4/8 antibody (Bethyl Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> A300-245 </a> )</li><li>Roswell Park Memorial Institute (RPMI) 1640 Medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11875093 </a> )</li><li>10% fetal bovine serum (FBS), EU Approved (South American) (Thermo Fisher Scientific, Gibco<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 10270-106 </a> )</li><li>Penicillin-streptomycin (Thermo Fisher Scientific, Gibco<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 15070-063 </a> )</li><li>Trypsin-EDTA (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 25200056 </a> )<br/><em>Note: HeLa cells were cultured in RPMI 1640 supplemented with 10% FBS, 1 Î¼g/ml penicillin and 1 Âµg/ml streptomycin. </em> </li><li>5-Iodo-2â²-deoxyuridine (IdU) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> I7125 </a> , storage at 4 Â°C)</li><li>(S)-(+)-Camptothecin (CPT) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C9911 </a> , storage at -20 Â°C)</li><li>Sodium dodecyl sulphate (SDS) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 71725 </a> , storage at RT)</li><li>TRIS (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 17926 </a> , storage at RT)</li><li>EDTA 0.5 M pH 8.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9262 </a> , storage at 4Â°C)</li><li>Sodium phosphate dibasic (Na<sub>2</sub>HPO<sub>4</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 255793 </a> , storage at RT)</li><li>Potassium phosphate monobasic (KH<sub>2</sub>PO<sub>4</sub>) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P0662 </a> , storage at RT)</li><li>Potassium Cloride (KCl) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> P9541 </a> , storage at RT)</li><li>Hydrochloric acid 39.5% (HCl) (Carlo Erba Reagents, catalog number: <a class="blackNoUnder" target="_blank"> 405761 </a> , storage at RT)</li><li>Methanol (Carlo Erba Reagents, catalog number: <a class="blackNoUnder" target="_blank"> 528101 </a> , storage at RT)</li><li>Acetic Acid Glacial (Carlo Erba Reagents, catalog number: <a class="blackNoUnder" target="_blank"> 401424 </a> , storage at RT)</li><li>Ethanol Absolute (Carlo Erba Reagents, catalog number: <a class="blackNoUnder" target="_blank"> 414607 </a> , storage at RT)</li><li>Bovine Serum Albumine (BSA) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A9418 </a> , storage at 4 Â°C)</li><li>Sodium Chloride (NaCl) (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S9888 </a> , storage at RT)</li><li>ProLong Gold Antifade Reagent (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> P36935 </a> , storage at RT)</li><li>Phosphate buffered saline (PBS) (see Recipes)</li><li>Spreading buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti34978">### Equipment</p><ol class="npwtwok"><li>Zeiss Axiovert LSM100M confocal microscope (Carl Zeiss, Germany)</li><li>CO<sub>2</sub> incubator (Panasonic, catalog number: <a class="blackNoUnder" target="_blank"> MCO-18AC </a> )</li><li>Centrifuge<br/></li></ol><p class="pptt" id="biaoti34979">### Software</p><ol class="npwtwok"><li>Adobe Photoshop CS5</li><li>GraphPad Prism 8 software<br/></li></ol><p class="pptt" id="biaoti34980">### Procedure</p><ol class="npwone"><li>Representation of SMART assay<br/>Figure 1 shows a representative scheme of the SMART assay.<br/><br/><img alt="" class="layerphoto" height="243" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200729/20200729004015_9106.png" width="300"/><br/><strong>Figure 1. Schematics of SMART assay.</strong> First of all, label HeLa cells with IdU for 24 h followed by CPT treatment; at the end of incubation trypsinize cells and dilute them at a 2.5 x 10<sup>5</sup> cells/ml concentration. Mix labelled and unlabelled cells 1:8 and spot 2.5 Î¼l of this mix onto the upper part of the slide; after incubation with 7.5 Î¼l of spreading buffer, tilt the slide form the end to allow the stretching of the DNA fibres.<br/><br/>Figure 2 shows representative images acquired through confocal microscopy.<br/><br/><img alt="" class="layerphoto" height="229" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200729/20200729004032_5509.png" width="300"/><br/><strong>Figure 2. Representative images of the SMART technique.</strong> Panel A shows two images from HeLawt or siEXOI cells treated with 1 Î¼M of campthotecin for one hour; following the SMART assay images were analyzed as reported in the protocol through the Fiji software. As shown in panel B the silencing of EXOI, one of the principal exonucleases involved in the generation of single strand DNA upon DNA damage, reduces the length of the processed DNA up to two times. The Figure 3B reports the mean Â± standard deviation of the fibre length measured in Î¼m, for each condition.<br/><br/></li><li>SMART assay protocol<ol class="npwtwo"><li>Culture HeLa cells in RPMI 1640 medium supplemented with 10% FBS, 1 Î¼g/ml penicillin and 1 Âµg/ml streptomycin.</li><li>Treat HeLa cells in a 6 cm dish at 80% confluency with 10 Î¼M of IdU (we made a 28 mM IdU stock solution in 0.2 N NaOH) and incubate at 37 Â°C in a standard 5% CO<sub>2</sub> incubator for 24 h.<br/><em>Note: The length of incubation with BrdU or its analogs such as IdU has to be determined for each different cell type, depending on the cell cycle length. A pulse labelling time needs to allow BrdU incorporation during one complete cell cycle.</em> </li><li>Prepare a 6 cm dish of HeLa cells at 80% confluence as non-IdU-marked.<br/><em>Note: The dilution of IdU marked cell with the non-marked cells is essential to reduce the presence of marked fibres onto the glass slides favouring a correct immunofluorescence analysis.</em> </li><li>Treat HeLa cells with 1 Î¼M of camptothecin in RPMI 1640 medium supplemented with 10% FBS, 1 Î¼g/ml penicillin and 1 Âµg/ml streptomycin and incubate at 37 Â°C in a 5% CO<sub>2</sub> incubator for 2 h.<br/><em>Note: The incubation time and drug concentration for the DNA damage agent can change in different cell types. As control, perform a DNA damage response analysis with different conditions evaluating DNA damage and resection through Western blot using an anti phospho-RPA32 S4/8 antibody (Bethyl Laboratories).</em> </li><li>Detach HeLa cells from the dish with 1 ml of Trypsin-EDTA solution for 5 min at 37 Â°C in a standard 5% CO<sub>2</sub> incubator.</li><li>Inactivate the trypsin-EDTA solution with 3 ml of RPMI 1640 medium supplemented with 10% FBS, 1 Î¼g/ml penicillin and 1 Âµg/ml streptomycin.</li><li>Centrifuge HeLa cells for 5 min at 800 <em>x g</em> (rcf).</li><li>Wash the cell pellet with 5 ml of ice-cold PBS followed by centrifugation for 5 min at 800 <em>x g</em> (rcf). Repeat Step B8 twice.</li><li>Resuspend HeLa labelled cells with the ice-cold PBS at 2.5 x 10<sup>5</sup> cells/ml. Prepare another mix of unlabelled cells at the same concentration (2.5 x 10<sup>5</sup> cells/ml).</li><li>Dilute labelled cells by mixing them in a 1:8 proportion with unlabelled cells.</li><li>Mix 2.5 Î¼l of cells with 7.5 Î¼l of spreading buffer directly on the upper part of the slides.</li><li>Incubate at room temperature (RT) for 10 min.</li><li>Tilt the slide, from the ends, to 15Â°.</li><li>DNA spreads are air-dried, approximately 10 min.</li><li>Fix DNA fibres in 3:1 methanol/acetic acid at -20 Â°C for 15 min.</li><li>Wash slides in PBS twice.<br/><em>Note: The volume of PBS depends on the container used for the immersion of the slides</em> </li><li>Incubate DNA fibres in 70% ethanol and 30% water overnight at 4 Â°C.</li><li>Wash slides with PBS twice.<br/><em>Note: This step is essential to remove all the ethanol residues before blocking DNA fibres. The volume of PBS depends on the container used for the immersion of the slides.</em> </li><li>Block DNA fibres with PBS + 5% BSA for 30 min at RT.<br/><em>Note: The volume of PBS depends on the container used for the immersion of the slides</em> </li><li>Incubate slides with 1:300 Anti-BrdU clone BU1/75 (ICR1) for one hour at 37 Â°C.</li><li>Wash slides with PBS twice.<br/><em>Note: The volume of PBS depends on the container used for the immersion of the slides.</em> </li><li>Incubate slides with 1:400 Alexa Fluor 488-conjugated chicken anti-rat.</li><li>Wash slides with PBS twice.<br/><em>Note: The volume of PBS depends on the container used for the immersion of the slides.</em> </li><li>Mount the slide with ProLong Gold Antifade Reagent.</li><li>Analyse the slide through a confocal microscope (we used the Zeiss LSM100) with a minimum of 63x oil immersion objective.<br/></li></ol></li></ol><p class="pptt" id="biaoti34981">### Data analysis</p><p class="ppzz">Analyse approximately 150 fibres for each condition and measure the length of fibres in pixel through the Fiji image software analysis.<br/>To obtain the fibre length in Î¼m:</p><ol class="npwtwok"><li>First of all download the LSM toolbox form the plugin window.<br/><em>Note: This is an important step to open directly the confocal image in the .lsm format without jpg conversion.</em> </li><li>Open the .lsm file, as reported in the Figure 3 below, there is the resolution of the picture and the conversion in Î¼m.<br/><br/><img alt="" class="layerphoto" height="133" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200729/20200729004053_2520.png" width="300"/><br/><strong>Figure 3. A representative image opened with the Fiji software showing in the upper left part the pixel resolution and Î¼m conversion; in the lower corner of the figure the Î¼m dimension.</strong> A. Micrometer conversion into kilobase depends on the DNA type, the nucleosome and chromatin fibre assembly; for HeLa cells and the combing assay 1 Î¼m of DNA fibre length, during the S-phase, should approximately correspond to 2.59 kb as reported originally by Jackson and Pombo (Jackson and Pombo, 1998) and others (Bianco <em>et al.</em>, 2012; Jamroskovic <em>et al.</em>, 2020; Petermann <em>et al.</em>, 2008). B. A magnification image of DNA fibre showing in red (start) and blue (end) the dimension of the analysed DNA tracts. C. The yellow arrows indicate, in lower part of the figure, the direction and the length of the DNA fibres; these fibres cannot be used for the measurement given the high number of other DNA fibres, which reduce the identification of single DNA traits. The red arrow indicates a fibre at a density and fluorescence, which are optimal for the analysis of the single DNA traits. D. Example of a screenshot of the Straight Line tool from the Fiji software analysis.<br/><br/></li><li>Select the Straight Line from the main window and measure the length of the fibres. The Straight Line tool of the Fiji software provides the length of the selected fibre identified through the x, y and angle coordinates; see an example of the Straight Line tool screen in Figure 3D.</li><li>In this example, the mean fibre length Â± standard deviation, calculated out of three independent experiments was plotted. Statistical significance was evaluated through the Studentâs <em>t</em>-test using the GraphPad Prism 8 software in accordance with previous work (Cruz-GarcÃ­a <em>et al.</em>, 2014). The more informative way to represent data and the appropriate test for statistical analysis, however, will have to be determined according to the experimental plan upon checking data for normality through GraphPad Prism or similar programs.<br/></li></ol><p class="pptt" id="biaoti34982">### Recipes</p><ol class="npwtwok"><li>Phosphate buffered saline (PBS)<br/>137 mM NaCl<br/>2.7 mM KCl<br/>10 mM Na<sub>2</sub>HPO<sub>4</sub><br/>1.8 mM KH<sub>2</sub>PO<sub>4</sub> </li><li>Spreading Buffer<br/>0.5% Sodium dodecyl sulphate (SDS)<br/>200 mM Tris-HCl pH 7.4<br/>50 mM EDTA pH 8<br/></li></ol><p class="pptt" id="biaoti34983">### Acknowledgments</p><p class="ppzz">We are grateful to the Sbarro Health Research Organization (<a href="http://www.shro.org/" target="_blank">http://www.shro.org</a>), for its support.<br/>    We are grateful to Andres Cruz-GarcÃ­a, AnaLÃ³pez-Saavedra and Pablo Huertas who first developed the SMART assay (Cruz-GarcÃ­a <em>et al.</em>, 2014), from which this protocol was adapted.</p><p class="pptt" id="biaoti34984">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti34985">### References</p><ol class="npwtwok"><li>Alfano, L., Costa, C., Caporaso, A., Altieri, A., Indovina, P., Macaluso, M., Giordano, A. and Pentimalli, F. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25893301" target="_blank">NONO regulates the intra-S-phase checkpoint in response to UV radiation.</a> <em>Oncogene</em> 35(5): 567-576.</li><li>Alfano, L., Giordano, A. and Pentimalli, F. (2017). <a href="https://en.bio-protocol.org/e2301?sid=810b3a72-45eb-486f-9355-497e0b150074&amp;sousouid=20200607112431933&amp;ptype=0&amp;px=1&amp;intwzid=2301" target="_blank">DNA Fiber Assay upon Treatment with Ultraviolet Radiations.</a> <em>Bio-protocol </em>7(11): e2301.</li><li>Alfano, L., Caporaso, A., Altieri, A., Dell'Aquila, M., Landi, C., Bini, L., Pentimalli, F. and Giordano, A. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30799487" target="_blank">Depletion of the RNA binding protein HNRNPD impairs homologous recombination by inhibiting DNA-end resection and inducing R-loop accumulation.</a> <em>Nucleic Acids Res</em> 47(8): 4068-4085.</li><li>Bianco, J. N., Poli, J., Saksouk, J., Bacal, J., Silva, M. J., Yoshida, K., Lin, Y. L., Tourriere, H., Lengronne, A. and Pasero, P. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22579803" target="_blank">Analysis of DNA replication profiles in budding yeast and mammalian cells using DNA combing.</a> <em>Methods</em> 57(2): 149-157.</li><li>Chen, H., Lisby, M. and Symington, L. S. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23706822" target="_blank">RPA coordinates DNA end resection and prevents formation of DNA hairpins.</a> <em>Mol Cell</em> 50(4): 589-600.</li><li>Cruz-GarcÃ­a, A., LÃ³pez-Saavedra, A. and Huertas, P. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25310973" target="_blank">BRCA1 accelerates CtIP-mediated DNA-end resection.</a> <em>Cell Rep</em> 9(2): 451-459.</li><li>Hanahan, D. and Weinberg, R. A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21376230" target="_blank">Hallmarks of cancer: the next generation.</a> <em>Cell</em> 144(5): 646-674.</li><li>Jackson, D. A. and Pombo, A. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9508763" target="_blank">Replicon clusters are stable units of chromosome structure: evidence that nuclear organization contributes to the efficient activation and propagation of S phase in human cells.</a> <em>J Cell Biol</em> 140(6): 1285-1295.</li><li>Jamroskovic, J., Doimo, M., Chand, K., Obi, I., Kumar, R., Brannstrom, K., Hedenstrom, M., Nath Das, R., Akhunzianov, A., Deiana, M., Kasho, K., Sulis Sato, S., Pourbozorgi, P. L., Mason, J. E., Medini, P., Ohlund, D., Wanrooij, S., Chorell, E. and Sabouri, N. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31990532" target="_blank">Quinazoline Ligands Induce Cancer Cell Death through Selective STAT3 Inhibition and G-Quadruplex Stabilization.</a> <em>J Am Chem Soc</em> 142(6): 2876-2888.</li><li>Krogh, B. O. and Symington, L. S. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15568977" target="_blank">Recombination proteins in yeast.</a> <em>Annu Rev Genet</em> 38: 233-271.</li><li>Mao, Z., Bozzella, M., Seluanov, A. and Gorbunova, V. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18675941" target="_blank">Comparison of nonhomologous end joining and homologous recombination in human cells.</a> <em>DNA Repair (Amst)</em> 7(10): 1765-1771.</li><li>Nimonkar, A. V., Genschel, J., Kinoshita, E., Polaczek, P., Campbell, J. L., Wyman, C., Modrich, P. and Kowalczykowski, S. C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21325134" target="_blank">BLM-DNA2-RPA-MRN and EXO1-BLM-RPA-MRN constitute two DNA end resection machineries for human DNA break repair.</a> <em>Genes Dev</em> 25(4): 350-362.</li><li>Petermann, E., Helleday, T. and Caldecott, K. W. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18353973" target="_blank">Claspin promotes normal replication fork rates in human cells.</a> <em>Mol Biol Cell</em> 19(6): 2373-2378.</li><li>Stracker, T. H. and Petrini, J. H. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21252998" target="_blank">The MRE11 complex: starting from the ends.</a> <em>Nat Rev Mol Cell Biol</em> 12(2): 90-103.</li><li>TkÃ¡Ä, J., Xu, G., Adhikary, H., Young, J.T.F., Gallo, D., Escribano-DÃ­az, C., Krietsch, J., Orthwein, A., Munro, M., Sol, W., <em>et al.</em> (2016). HELB Is a Feedback Inhibitor of DNA End Resection. <em>Mol Cell</em> 61: 405-418.</li><li>Zhou, Y., Caron, P., Legube, G. and Paull, T. T. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24362840" target="_blank">Quantitation of DNA double-strand break resection intermediates in human cells.</a> <em>Nucleic Acids Res</em> 42(3): e19.<br/></li></ol>