<body>
            <h1>Mapping mRNA-18S rRNA Contacts Within Translation Initation Complex by Means of Reverse Transcriptase Termination Sites and RNAseq</h1>
            <p><strong>Authors:</strong> Irene DÃ­az-LÃ³pez, RenÃ© Toribio, IvÃ¡n Ventoso</p>
            <h2>Abstract</h2>
            <p>The nucleotides involved in RNA-RNA interaction can be tagged by chemical- or UV-induced crosslinking, and further identified by classical or modern high throughput techniques. The contacts of mRNA with 18S rRNA that occur along the mRNA channel of 40S subunit have been mapped by site-specific UV crosslinking followed by reverse transcriptase termination sites (RTTS) using radioactive or fluorescent oligonucleotides. However, the sensitivity of this technique is restricted to the detection of those fragments that resulted from the most frequent crosslinkings. Here, we combined RTTS with RNAseq to map the mRNA-18S rRNA contacts with a much deeper resolution. Although aimed to detect the interaction of mRNA with the ES6S region of 18S rRNA, this technique can also be applied to map the interaction of mRNA with other non-coding RNA molecules (<em>e.g.</em>, snRNAs, microRNAs and lncRNAs) during transcription, splicing or RNA-mediated postranscriptional regulation.<br/></p>
            <p><strong>Keywords:</strong> RNA-RNA interaction, Translation initiation, Ribosome, mRNA, Site-specific crosslinking, Reverse transcription stop, RNAseq</p>
            <h2>Background</h2><p>Interaction of mRNA with non-coding RNAs is involved in every step of mRNA life cycle, from its biogenesis and processing into the nucleus to translation and ultimate degradation in the cytoplasm. These interactions can occur either within large molecular machines as ribosome and spliceosome, and in smaller complexes such as RISC (RNA-induced silencing complex) or during lncRNA-mediated gene expression regulation (Pisarev <em>et al.</em>, 2008; Engreitz <em>et al.</em>, 2014; Sharma <em>et al.</em>, 2016). For translation, the use of <em><em>in vitro</em></em> reconstructed initiation complexes or drugs that freeze ribosomes during the initiation or elongation steps have allowed to snapshot those residues in mRNA and ribosomal RNAs involving in RNA-RNA contact. The incorporation of photoactivatable nucleotides (<em>e.g.</em>, 4-thio-UTP) surrounding the AUG<sub>i</sub> in the mRNA allows site-specific crosslinking upon UV irradiation at 360 nm without affecting non-labeled residues. The resulting crosslinking adducts block primer extension by reverse transcriptase, thus allowing the identification of the crosslinking sites by sequencing the 3â² end of the resulting cDNA fragment (Kielpinski <em>et al.</em>, 2013). Recently, a combination of reverse transcriptase termination sites (RTTS) with RNAseq has been described, allowing the identification of every premature stop during reverse transcriptase (DÃ­az-LÃ³pez <em>et al.</em>, 2019). Apart from UV-induced crosslinking, there are other sources of RT stops, including accidental fragmentation of RNA template during extraction or preexisting chemical modification in mRNA residues (methylation and pseudourylation) that must be controlled during the experiment. Here, we describe in more detail this protocol that could be potentially applied to map any RNA-RNA interaction with nucleotide resolution using a labeled RNA bait. Our data working on mRNA-rRNA interaction during translation initiation indicates that the sensitivity of this protocol could be enough to detect even transient RNA-RNA interactions that regulate gene expression.<br/></p>
        </body>

<p class="pptt" id="biaoti35162">## Materials and Reagents</p><p class="ppzz"><em>Note: Make sure all reagents and materials are RNase-free. Materials are stored at room temperature unless otherwise indicated.</em> </p><ol class="npwtwok"><li>1.5 ml Eppendorf tubes (Eppendorf, catalog number: 00<a class="blackNoUnder" target="_blank"> 30120086 </a> )</li><li>1 ml thickwall polycarbonate ultracentrifuge tubes (Beckman, catalog number: <a class="blackNoUnder" target="_blank"> 343778 </a> )</li><li>Parafilm</li><li>Ultrapure nuclease-free water (Invitrogen, catalog number:<a class="blackNoUnder" target="_blank"> 10977-035 </a> )</li><li>Sucrose (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 84097 </a> )</li><li>Agarose (Pronadisa, catalog number: <a class="blackNoUnder" target="_blank"> 8010 </a> .00)</li><li>Ethanol (Merck, absolute for analysis, catalog number: <a class="blackNoUnder" target="_blank"> 51976 </a> )</li><li>Isopropanol (Merck, absolute for analysis, catalog number: <a class="blackNoUnder" target="_blank"> I9516 </a> )</li><li>Phenol:clorophorm:isoamyl alcohol (25:24:1) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P2069 </a> , store at 4 Â°C)</li><li>Chroma-Spin STE-10 purification columns (Takara, catalog number: <a class="blackNoUnder" target="_blank"> 636055 </a> )</li><li>Magnesium chloride 1 M solution (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M1028 </a> )</li><li>Magnesium acetate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M5661 </a> )</li><li>Potasium acetate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P9541 </a> )</li><li>Sodium acetate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> S2889 </a> )</li><li>Trizma base (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 93362 </a> )</li><li>EDTA (Sigma, catalog number: 0<a class="blackNoUnder" target="_blank"> 3677 </a> )</li><li>LiDS (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L9781 </a> )</li><li>Nuclease-treated rabbit reticulocyte lysates (including amino acid mix 1mM) (Promega, catalog number: <a class="blackNoUnder" target="_blank"> 4960 </a> , store at -70 Â°C)</li><li>GMP-PNP (Calbiochem, catalog number: <a class="blackNoUnder" target="_blank"> sc-215113 </a> , store at -20 Â°C)</li><li>RNase inhibitor (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0307S </a> , store at -20 Â°C)</li><li>HighScribe T7 polymerase kit (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> E2040S </a> , store at -20 Â°C)</li><li>4-thio-UTP (Jena Biosciences, catalog number: <a class="blackNoUnder" target="_blank"> NU-1156S </a> , sotre at -20 Â°C)</li><li>Oligo d(T) 25 magnetic beads (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> S1419S </a> , store at 4 Â°C)</li><li>Glycogen (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> 15073019 </a> , store at -20 Â°C)</li><li>QIAquick Gel extraction kit (QIAGEN, catalog number: <a class="blackNoUnder" target="_blank"> 28704 </a> )</li><li>SuperScript III First-strand synthesis system for RT-PCR (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 18080-051 </a> , store at -20 Â°C)</li><li>RNase H (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 18021-014 </a> , store at -20 Â°C)</li><li>Supreme 2x green master mix (NZYtech, catalog number: <a class="blackNoUnder" target="_blank"> MB05403 </a> , store at -20 Â°C)</li><li>Phusion DNA polymerase (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0530S </a> , store at -20 Â°C)</li><li>CHROMA SPIN-30+TE-30 (Takara, catalog number: <a class="blackNoUnder" target="_blank"> 636069 </a> )</li><li>DNA clean &amp; Concentrator-25 (Epigenetics, catalog number: <a class="blackNoUnder" target="_blank"> D4005 </a> )</li><li>Trizma base (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 93362 </a> )</li><li>Agencourt AMPURE XP (Beckman coulter, catalog number: <a class="blackNoUnder" target="_blank"> A63880 </a> )</li><li>DNA Kb ladder (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> N3232L </a> )</li><li>Circligase (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> ASLPA1212 </a> , store at -20 Â°C)</li><li>Primers (all from Sigma, store at -20 Â°C):<br/>18S_rev: CACCTCTAGCGGCGCAATACG<br/>adapter_ligation: [phos] AGATCGGAAGCGTCGGACTGTAGAACTCTGAACGTGT<br/>18S adapter: AGACGTGTGCTCTTCCGATCTCACCTCTAGCGGCGCAATACG<br/>PR_fwd: ATGATACGGCGACCACCGAGATCTACACGTTCAGAGTTCTACAGTCCGAG<br/>INDEX: CAAGCAGAAGAACGGCATACGAGA<strong>NNNNNNN</strong>GTGACTGGAGTTCAGACGT GTGCTCTTCCGATCT<br/>Flat_fw: GGATCCTAATACGACTCACTATAGGGACCCACCAACACAGCACCA<strong>T</strong>GAACAACGAGCCACCGACAGG<strong>T</strong>GA<strong>T</strong>GAG<strong>T</strong>GA<strong>T</strong>GACGGAGGCACACACGACAGACAACCGAGAGAGCAGAACGAGACCACAC<br/>U_rev polyA: (T)<sub>25</sub> GTGTGGTCTCGTTCTGCTC<br/>T7_U_fw: GGATCCTAATACGACTCACTATAGGG</li><li>Polysome buffer (see Recipes)</li><li>TE buffer (see Recipes)</li><li>1x TAE buffer (see Recipes)</li><li>SuperScript III cDNA synthesis mix (per reaction) (see Recipes)</li><li>Circligase enzyme mix (per reaction) (see Recipes)</li><li>Lysis/binding buffer (see Recipes)</li><li>Wash Buffer 1 (see Recipes)</li><li>Wash Buffer 2 (see Recipes)</li><li>Low Salt Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35163">### Equipment</p><ol class="npwtwok"><li>P20, P200 and P1000 Pipetman automatic pipettes (Gilson)</li><li>TL100 ultracentrifuge (Beckman)</li><li>TLA 100.2 rotor</li><li>Lamp holder with 2 x 6W 360 nm lamps (Philips, model: <a class="blackNoUnder" target="_blank"> TL6W </a> ) (home made)</li><li>Vortex (Biosan, vortex V-1 plus)</li><li>-20 Â°C freezer (Liehberr)</li><li>Fume hood</li><li>Thermomixer (Eppendorf, comfort)</li><li>Thermoblock equiped with metalblocks (LabNet, Accublock)</li><li>NanoDrop spectrophotometer (Thermo Scientific, Model 1000)</li><li>Magnetic separator rack (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> S1506S </a> )</li><li>Benchtop centrifuge (Thermo, model: <a class="blackNoUnder" target="_blank"> Megafuge 16R </a> )</li><li>Minifuge (Hettich, Mikroliter)</li><li>Gel electrophoretic system (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> Mini-Sub Cell GT System </a> )</li><li>Bioanalyzer (Agilent, model: <a class="blackNoUnder" target="_blank"> 2100 </a> )</li><li>NextSeq 550 system (Illumina)<br/></li></ol><p class="pptt" id="biaoti35164">### Software</p><ol class="npwtwok"><li>FASTX toolkit version 0.0.13 (<a href="http://hannonlab.cshl.edu/fastx_toolkit" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit</a>)</li><li>Bowtie 2 v2.3.5.1 (<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a>) (Langmead and Salzberg, 2012)</li><li>Featurecounts. Available in <a href="http://www.bioconductor.org" target="_blank">http://www.bioconductor.org</a> (Liao <em>et al.</em>, 2014)<br/></li></ol><p class="pptt" id="biaoti35165">### Procedure</p><ol class="npwone"><li>Preparation of mRNA containing 4-thio-UTP (4<sup>s</sup>U-mRNA) by <em>in vitro</em> transcription<br/>We synthesized a 128-nt mRNA containing minimal secondary structure bearing a short (19 nt) 5â² UTR an 3' polyadenylated. Photoactivatable 4-thio-UTP can be incorporated at positions +2 (the A of AUG is denoted as +1), +24, +27, +31 or +34 of mRNA. We use 4-thio-UTP at a four-fold higher concentration than UTP to ensure that every mRNA molecule is labeled with a least one 4-thio-UTP (Figure 1A).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200814/20200814073750_4697.jpg" width="59"/><br/><strong>Figure 1. Overview of the protocol for the identification of mRNA:18S rRNA contacts within 48S complex assembled </strong><em><strong>in vitro</strong></em><strong>.</strong> A. Schematic diagram of the mRNA used to assemble translation initiation complex (48S-PIC) in RRL. The positions in the mRNA labeled with 4-thio-U are numbered respect to the AUG<sub>i</sub>. B. Overview of the experimental method to identify mRNA-18S rRNA contacts by specific crosslinking of 4-thio-U, followed by reverse transcriptase termination site (RTTS) assay and next-generation sequencing (NGS).<br style="font-size:13.3333px;"/><span style="font-size:13.3333px;"><br/><ol class="npwtwo"><li>DNA template preparation<ol class="npwthree"><li>Mix primers Flat_fw and U_rev polyA at 1 Î¼M final concentration in 50 Î¼l of 1x Supreme green master mix and incubate at 95 Â°C for 5 min, followed by 1 min at 45 Â°C and 10 min at 72 Â°C.</li><li>Then, use 2 Î¼l of this mixture to program a PCR amplification with T7_U_fw and U_rev polyA primers (both at 0.5 Î¼M final concentration) in 50 Î¼l 1x Supreme green master mix (30 cycles, 95 Â°C 2 min, 45 Â°C 2 min, 72 Â°C 45 s, final extension of 72 Â°C 10 min).</li><li>Use a 2 Î¼l sample to check the size of the PCR product (about 200 nt) by electrophoresis in 1.5% agarose gel in TAE 1x buffer.</li><li>Then, purify the PCR fragment with DNA clean &amp; Concentrator-25 columns, that typically yields 25 Î¼l at 0.3-0.5 Î¼g/Î¼l.</li></ol></li><li>mRNA synthesis and purification<ol class="npwthree"><li>Program an <em>in vitro</em> transcription using the HighScribe T7 polymerase kit in the presence of 4-thio-UTP.</li><li>Mix the following components in a final volume of 20 Î¼l:<br/>2 Î¼l of 10x Reaction buffer<br/>1 Î¼l of ATP/CTP/GTP 20mM each (1 mM final concentration)<br/>1 Î¼l UTP 5 mM (0.25 mM final concentration)<br/>1 Î¼l 4-thio-UTP 20 mM (1 mM final concentration)<br/>0.5 Î¼l RNasin<sup>Â®</sup> Plus RNase Inhibitor<br/>1 Î¼g DNA template<br/>2 Î¼l T7 RNApol</li><li>Bring to 20 Î¼l with H<sub>2</sub>O and incubate in the dark for 2-3 h at 37 Â°C.</li><li>Digest the DNA template with 1 Î¼l of DNase I (1U/ Î¼l) for 15 min at 37 Â°C. Then, bring the final volume of the mixture to 40 Î¼l with H<sub>2</sub>O.</li><li>Then load the mixture in a prepacked chromaspin-30 column and centrifuge at 700 <em>x g</em> for 5 min in a benchtop centrifuge at room temperature.</li><li>Recover the elute and use 1 Î¼l to measure the concentration of the RNA in a NanoDrop spectrophotometer (typically 0.2-0.5 Î¼g/Î¼l). Store the mRNA at -70 Â°C.</li></ol></li></ol><br/></span> </li><li>Assembly of translation initiation complex (48S-PIC) in rabbit reticulocyte lysate (RRL) and UV crosslinking<br/>Program two 100 Î¼l translation mixtures with 4<sup>s</sup>U-mRNA in the presence of non hydrolyzable GMP-PNP to promote the accumulation of 48S preinitiation complex (PIC) (Figure 1). One of the samples will not receive irradiation (control).<ol class="npwtwo"><li>48S-PIC assembly<br/><ol class="npwthree"><li>Mix the following components per duplicate:<br/>70 Î¼l of RRL (70% of final volume)<br/>16 Î¼l of H<sub>2</sub>O<br/>2 Î¼l aminoacid mix (1 mM)<br/>8 Î¼l of magnesium acetate 20 mM (2.5 mM final concentration)<br/>2 Î¼l of GMP-PNP 100 mM (2 mM final concentration)<br/>2 Î¼l of RNase inhibitor</li><li>Pre-incubate 5 min at 30 Â°C and then add 0.5 Î¼g of 4<sup>s</sup>U-mRNA.</li><li>Mix gently and incubate another 25 min at 30 Â°C.</li><li>Place the tube on ice to stop the reaction.</li></ol><br/></li><li>UV crosslinking.<br/><div style="white-space:nowrap;"><ol class="npwthree"><li>Cut a piece of parafilm and place it between two metal blocks with the plastic cover facing up.</li><li>Fix the assembly carefully with elastic rubber bands and precool for at least 2 h at -20 Â°C.</li><li>Carefully remove the upper block and the plastic cover, and place the bottom block with the parafilm on an ice box as indicated below (Figure 2):<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200814/20200814073900_7525.jpg" width="279"/><br/><strong>Figure 2. UV lamp assembly on sample for crosslinking.</strong> Sample (red dot) is placed on parafilm for UV irradiation as indicated. Up to six samples can be irradiated simultaneously.<br/><br/></li><li>Using a P200 pipet, place one of the samples on the parafilm and extend it with the tip to form a 2 cm diameter smear.</li><li>Place the lamp holder on the box so that UV lamps are 3 cm away from the sample and irradiate 20 min.</li><li>Let the other sample on ice (not irradiated control).</li></ol></div></li></ol><br/></li><li>Poly (A) mRNA isolation from whole ribosomal pellet<ol class="npwtwo"><li>Transfer both the irradiated and not irradiated samples (100 Î¼l) to new Eppendorf tubes on ice and slowly add 800 Î¼l of polysome buffer (Recipe 1)</li><li>Mix well and carefully place the mix (850-900 Î¼l) on sucrose cushion made in a 1 ml ultracentrifuge tube (100 Î¼l 20% sucrose solution in polysome buffer).</li><li>Place the tubes in a TLA 100.2 rotor and centrifuge at 250,000 <em>x g</em> for 3 h at 4 Â°C. At this step, the ribosomal pellet resembles a translucent lentil.</li><li>Remove the supernatant with a P1000 pipette and gently wash the ribosomal pellet with 200 Î¼l of TE buffer for 3-5 s avoiding touching the pellet with the tip.</li><li>Repeat washing twice more. Add 500 Î¼l of Lysis-Binding Buffer (Recipe 6) and let pellet dissolve for 15 min at room temperature.</li><li>Save a 30 Î¼l aliquot from the not irradiated sample to isolate total RNA (see below).</li><li>To the rest, add 200 Î¼l of oligo d(T)<sub>25</sub> magnetic beads (0.25 ml of 5 mg/ml suspension) previously equilibrated in 500 Î¼l Lysis/Binding Buffer for 5 min.</li><li>Incubate at room temperature for 15 min in a rotating carousel at 30 rpm.</li><li>Place the tube into magnetic rack and pull magnetic beads to the side of the tube.</li><li>Then remove and discard the supernatant and add 0.5 ml of Wash Buffer 1 (Recipe 7) to the beads.</li><li>Mix with rotation for 1 min (as describe in Step C8) and capture the beads as described above.</li><li>Repeat washing once more with Wash Buffer 1, twice with Wash Buffer 2 (Recipe 8) and once with Low Salt Buffer (Recipe 9).</li><li>Finally, add 100 Î¼l of H<sub>2</sub>O to the beads and heat at 50 Â°C for 2 min in a thermomixer to elute the RNA.</li><li>Measure the concentration in a NanoDrop (typically 50-100 ng/Î¼l) and store at -70 Â°C.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Total RNA extraction<br/><ol class="npwtwo"><li>To extract total RNA, bring the 30 Î¼l aliquot saved before (Step C6) to 200 Î¼l with H<sub>2</sub>O.</li><li>Then, add 200 Î¼l of phenol:chloroform:isoamyl alcohol and vortex vigorously for 1 min.</li><li>Let the tube stand for 5 min and repeat the vortexing three more times.</li><li>Centrifuge at maximum speed for 5 min in a minifuge at room temperature and transfer the upper phase to a new tube avoiding touching the interphase.</li><li>Finally, precipitate the RNA with 2 volumes of 100 % ethanol and 0.2 M of sodium acetate pH 5.2 (final concentration) at -20 Â°C O/N.</li><li>Centrifuge at maximum speed for 15 min in a minifuge at 4 Â°C and wash the pellet with 0.5 ml 70% ethanol.</li><li>Repeat the centrifugation at room temperature, remove the ethanol and let the pellet dry 5 min on the bench.</li><li>Then, dissolve the pellet in 20 Î¼l of H<sub>2</sub>O and store at -70 Â°C.</li></ol><br/></li><li>Retrotranscription of 18S rRNA<br/>Use irradiated, not irradiated and total RNA samples for retrotranscription with primer 18S_rev <br/>(targeting 18S rRNA) following the manufacturer specifications:<br/><ol class="npwtwo"><li>Mix 0.5 Î¼g of RNA samples (typically 5-10 Î¼l) with 0.5 Î¼M of 18S_rev and 1 Î¼l of 10 mM dNTPs in a final volume of 10 Î¼l.</li><li>Denature the RNA by heating at 65 Â°C for 5 min and then place the tubes on ice for 1 min.</li><li>Prepare the SuperScript III cDNA synthesis mix (Recipe 4) and add 10 Î¼l to the RNA samples.</li><li>Incubate 50 min at 50 Â°C and then terminate the reaction by heating at 85 Â°C for 10 min. Spin down the tubes briefly and keep them on ice.</li><li>Then add 1 Î¼l of RNase H to the tubes and incubate 20 min at 37 Â°C to degrade the RNA.</li><li>Purify the cDNA using the AMPURE XP magnetic beads following the manufacturerÂ´s recommendation, but using two-fold more beads to keep smaller fragments <a href="https://www.beckmancoulter.com/wsrportal/techdocs?docname=B37419" target="_blank">https://www.beckmancoulter.com/wsrportal/techdocs?docname=B37419</a>.</li><li>Elute the samples in 15 Î¼l of H<sub>2</sub>O.</li></ol><br/></li><li>Adapter ligation and first PCR amplification<br/><ol class="npwtwo"><li>Adapter ligation to the 3â² OH of cDNA and circularization is carried out with Circligase in a final volume of 20 Î¼l, containing 10 Î¼l of purified cDNA, 2.5 Î¼M of adapter_ligation primer and 10 Î¼l of Circligase enzyme mix (Recipe 5).<br/><ol class="npwthree"><li>Incubate the samples 2 h at 60 Â°C, then 1 h at 68 Â°C and 10 min at 80 Â°C.</li><li>Let the samples on ice for at least 2 min.</li></ol><br/></li><li>Perform the first PCR amplification using Phusion DNA polymerase in a final volume of 10 Î¼l using the PR_fwd and 18S adapter primers (0.5 Î¼M) and 5x Phusion HF buffer.<br/>To optimize the amplification, try three different volumes of the ligated cDNA mix described above (0.5, 1 and 2 Î¼l).<br/><div style="white-space:nowrap;"><ol class="npwthree"><li>Program the PCR using the following parameters:<br/>3 min initial denaturation at 95 Â°C, followed by 30 cycles of 15 s at 95 Â°C, 15 s at 50 Â°C and 20 s at 72 Â°C with a final extension of 10 min at 72 Â°C.</li><li>Run a 2 Î¼l aliquot of each sample in an 1% agarose-TAE gel to check the amplification sizes (bands between 100-950 nt are expected).</li></ol></div></li></ol><br/></li><li>Second PCR amplification with index primers<br/>Use 3 Î¼l of the first PCR amplification to program a second PCR amplification with PR_fwd and INDEX primers (0.5 Î¼M final concentration). Use a different index primer for each sample (three in this case).<ol class="npwtwo"><li>Perform a PCR amplification using the same parameters as described above but reduce the number of cycles to 10.</li><li>Then, bring the samples to 40 Î¼l with H<sub>2</sub>O and purify the PCR fragments with AMPURE XP beads following the manufacturer specifications.</li><li>Elute the samples in 20 Î¼l H<sub>2</sub>O and check them in a Bioanalyzer.</li><li>A distribution of peaks in the range of 100-950 nt is expected (see Figure 3A). In some samples a strong peak of around 150 nt is observed, that presumably corresponds to adapter dimers. In this case, perform a size fractionation of the sample in a 1% agarose-TAE gel by cutting the piece of gel above the 150 nt marker. Extract the DNA from agarose piece by using QIAquick gel extraction kit.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200814/20200814073930_8651.jpg" width="98"/><br/><strong>Figure 3. An example of the results obtained.</strong> A. A typical picture of agarose gel electrophoresis showing final PCR libraries obtained from RNA total and oligo d(T) captured (right panel). Left panel shows the electropherogram corresponding to oligo d(T) captured sample from above. The sample was analyzed in a bioanalyzer and the range of DNA peaks obtained is indicated by a blue bracket. Size markers are also shown (M). B. A typical histogram of 5â² reverse transcription (RT) arrests on 18S rRNA after crosslinking of mRNA assembled into the 48S-PIC. Total or oligo(dT)-purified RNA was subjected to reverse transcriptase termination site (RTTS) assay of 18S rRNA. Some arrests enriched in oligo(dT)-purified RNA are indicated by green arrows. A strong RT arrest at C726 was frequently observed in total RNA, presumably due to fragmentation of 18S rRNA during the purification process.<br/><br/></li></ol></li><li>Sequencing<br/>Mix the resulting indexed libraries (three in this case) by adjusting to the sample of lower concentration to get a similar representation of each library in the mix. Send samples for sequencing in a NextSeq550 platform, 1x75 reads. The typical output was 0.5Mreads/sample.</li></ol><p class="pptt" id="biaoti35166">### Data analysis</p><p class="ppzz">FASTQ files from sequencing are processed under the FASTX software pipeline (<a href="http://hannonlab.cshl.edu/fastx_toolkit/galaxy.html" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit/galaxy.html</a>):</p><ol class="npwtwok"><li>Demultiplex the sequencing reads using FASTQ Barcode splitter and index sequences:<br/>Command line: <span style="font-family:Courier New;">cat sequencing_complete.fastq | fastx_barcode_splitter âbcfile barcodes_sequences.txt âbol âprefix â/sequencing/RTTS_â --suffix â.fastq.</span> </li><li>Remove index sequences using FASTQC Clipper option. Remove the sequences lacking the adapter:<br/>Command line: <span style="font-family:Courier New;">fastx_clipper âa ADAPTER_SEQUENCE âc âi INFILE âo OUTFILE.</span> </li><li>Perform a quality-based filtering of sequences using FASTQ Quality Filter option deleting all the sequences below 20 quality score in at least 80% of the molecule:<br/>Command line: <span style="font-family:Courier New;">fastq_quality_filter âq 20 âp 80 âI INFILE âo OUTFILE</span>).</li><li>Align the sequences to rabbit 18S rRNA (accession X06778.1, <a href="https://www.ncbi.nlm.nih.gov/nuccore/X06778" target="_blank">https://www.ncbi.nlm.nih.gov/nuccore/X06778</a>) using Bowtie 2 with default parameters:<br/>Command line: <span style="font-family:Courier New;">bowtie2 -q âx rabbit_18S_indexes âU sequences.fq âS aligment.output.sam</span>.</li><li>Once aligned, use FeatureCounts software to extract the 5' end position of each read using <em>read2pos5</em> parameter (command line: <span style="font-family:Courier New;">featureCounts input_files.sam âo output_file.txt âread2pos5</span>). Then, we used the command <span style="font-family:Courier New;">cut -f 3 feature_counts_input.txt | sort -g | uniq -c &gt; final_output.txt</span> to generate a table with the frequency representation of every nucleotide of 18S rRNA at the 5' end of reads. The exact crosslinking site is considered the -1 position respect to the mapped 5' end of every read.</li><li>Once mapped the crosslinking sites on 18S rRNA sequence for every sample, identify those arrests which are enriched in oligo d(T)-captured RNA sample. To do this, subtract the data with that obtained using total RNA and not crosslinked samples. Usually, we consider only the crosslinking sites representing &gt; 0.5 % of total reads.<br/></li></ol><p class="pptt" id="biaoti35167">### Recipes</p><ol class="npwtwok"><li>Polysome buffer<br/>Tris-HCl 20 mM pH 7.5<br/>KCl 100 mM<br/>MgCl<sub>2</sub> 2 mM<br/>DTT 2mM</li><li>TE buffer<br/>10 mM Tris pH 8.0<br/>1 mM EDTA</li><li>1x TAE buffer<br/>40 mM Tris-acetate<br/>1 mM EDTA</li><li>SuperScript III cDNA synthesis mix (per reaction)<br/>2 Î¼l 10x RT Buffer<br/>4 Î¼l 25 mM MgCl<sub>2</sub><br/>2 Î¼l 0.1 M DTT<br/>1 Î¼l RNaseOUT<br/>1 Î¼l SuperScript III RT</li><li>Circligase enzyme mix (per reaction)<br/>2 Î¼l CL 10x reaction buffer<br/>1 Î¼l ATP<br/>1 Î¼l MnCl<sub>2</sub><br/>1 Î¼l Circligase<br/>5 Î¼l H<sub>2</sub>O</li><li>Lysis/binding buffer<br/>20 mM Tris-HCl (pH 7.5)<br/>500 mM LiCl<br/>0.5% LiDS<br/>1 mM EDTA<br/>5 mM DTT</li><li>Wash Buffer 1<br/>20 mM Tris-HCl (pH 7.5)<br/>500 mM LiCl<br/>0.1% LiDS<br/>1 mM EDTA<br/>5 mM DTT</li><li>Wash Buffer 2<br/>20 mM Tris-HCl (pH 7.5)<br/>500 mM LiCl<br/>1 mM EDTA</li><li>Low Salt Buffer<br/>20 mM Tris-HCl (pH 7.5)<br/>200 mM LiCl<br/>1 mM EDTA<br/></li></ol><p class="pptt" id="biaoti35168">### Acknowledgments</p><p class="ppzz">The technical support of Laura Barbado is acknowledged. This work was funded by Ministerio de EconomÃ­a y Competitividad (BFU2017-84955-R) grant.</p><p class="pptt" id="biaoti35169">### Competing interests</p><p class="ppzz">The authors declare neither conflicts of interest nor competing interests.</p><p class="pptt" id="biaoti35170">### References</p><ol class="npwtwok"><li>DÃ­az-LÃ³pez, I., Toribio, R., Berlanga, J. J. and Ventoso, I. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31789591" target="_blank">An mRNA-binding channel in the ES6S region of the translation 48S-PIC promotes RNA unwinding and scanning.</a> <em>Elife </em>8: 48246.</li><li>Engreitz, J. M., Sirokman, K., McDonel, P., Shishkin, A. A., Surka, C., Russell, P., Grossman, S. R., Chow, A. Y., Guttman, M. and Lander, E. S. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25259926" target="_blank">RNA-RNA interactions enable specific targeting of noncoding RNAs to nascent Pre-mRNAs and chromatin sites.</a> <em>Cell </em>159(1): 188-199.</li><li>Kielpinski, L. J., Boyd, M., Sandelin, A. and Vinther, J. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23872978" target="_blank">Detection of reverse transcriptase termination sites using cDNA ligation and massive parallel sequencing.</a> <em>Methods Mol Biol</em> 1038: 213-231.</li><li>Langmead, B and Salzberg, S. L. (2012). <a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" target="_blank">Fast gapped-read alignment with Bowtie 2.</a> <em>Nat Methods</em> 9(4): 357-359.</li><li>Liao, Y., Smyth, G. K. and Shi, W. (2014). <a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" target="_blank">featureCounts: and efficient general purpose program for assigning sequence reads to genomic features.</a> <em>Bioinformatics </em>30(7): 923-930.</li><li>Pisarev, A. V., Kolupaeva, V. G., Yusupov, M. M., Hellen, C. U. and Pestova, T. V. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18464793" target="_blank">Ribosomal position and contacts of mRNA in eukaryotic translation initiation complexes.</a> <em>EMBO J</em> 27(11): 1609-1621.</li><li>Sharma, E., Sterne-Weiler, T., O'Hanlon, D. and Blencowe, B. J. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27184080" target="_blank">Global mapping of human RNA-RNA interactions.</a> <em>Mol Cell</em> 62(4): 618-626. <br/></li></ol>