<body>
            <h1><em>Candida albicans</em> Agar Invasion Assays</h1>
            <p><strong>Authors:</strong> Shamoon Naseem, Lois M. Douglas, James B. Konopka</p>
            <h2>Abstract</h2>
            <p>The ability of the human fungal pathogen <em>Candida albicans</em> to disseminate into tissues is promoted by a switch from budding to invasive hyphal growth. This morphological transition is stimulated by multiple environmental factors that can vary at different sites of infection. To identify genes that promote invasive growth, <em>C. albicans</em> mutants can be screened for defects in growing invasively into solid agar medium as a substitute for studying tissue invasion. This <em>in vitro</em> approach has advantages in that it permits the media conditions to be varied to mimic different host environments. In addition, the concentration of agar can be varied to determine the effects of altering the rigidity of the matrix into which the cells invade, as this provides a better indicator of invasive growth than the ability to form hyphae in a liquid culture. Testing under multiple conditions can be used to identify mutant cells with the strongest defects. Therefore, protocols and media for analyzing invasive growth of <em>C. albicans</em> under different conditions will be described that are appropriate for testing a single strain or high-throughput analysis of a collection of mutant <em>C. albicans</em> strains.</p>
            <p><strong>Keywords:</strong> Hyphae, Pseudohyphae, Morphological switch, Invasive growth, <em>Candida albicans</em></p>
            <h2>Background</h2><p><em>Candida albicans</em> is a multimorphic fungal pathogen that can grow by forming buds (small spherical cells), pseudohyphal cells (chains of elongated cells), or hyphal cells (long filamentous chains of cells with parallel walls) (Figure 1) (Noble <em>et al.</em>, 2017). A wide range of environmental stimuli encountered in the host have been shown to promote the switch to invasive growth, including human body temperature (37 Â°C), alkaline pH, CO<sub>2</sub>, factors in serum, and the sugar GlcNAc (N-acetyl glucosamine) (Kornitzer, 2019). Budding cells are thought to facilitate bloodstream dissemination and colonization of the gastrointestinal tract (Pierce and Kumamoto, 2012; Witchley <em>et al.</em>, 2019). In contrast, the filamentous pseudohyphal and hyphal forms of <em>C. albicans</em> are better able to promote invasive growth into tissues (Noble <em>et al.</em>, 2017). For example, our recent studies have shown a key role for invasive growth in establishing oral <em>C. albicans</em> infections (Naseem <em>et al.</em>, 2019).<br/> <br/>Genetic screens have identified a large number of mutants that regulate the ability of <em>C. albicans</em> to switch from budding to hyphal growth (Noble <em>et al.</em>, 2010; O'Meara <em>et al.</em>, 2015; Glazier <em>et al.</em>, 2017; Kornitzer, 2019). However, a weak hyphal stimulus was often used in these studies as a way to sensitively detect a partial defect. Many of the previously reported mutants can still be induced by stronger stimuli, such as those encountered in the host (Riggle <em>et al.</em>, 1999; Noble <em>et al.</em>, 2010; Naseem <em>et al.</em>, 2011; Douglas <em>et al.</em>, 2013). Other mutants are only defective in forming hyphae in liquid medium and can still invade into a solid substrate. This phenotype is likely caused by contact with a solid substrate, which triggers an independent hyphal pathway (Kumamoto, 2005 and 2008; Douglas <em>et al.</em>, 2013). Therefore, to identify genes that are broadly important for invasive hyphal growth, we screened for <em>C. albicans</em> mutants that were defective in invading into solid agar media under a diverse set of strongly inducing conditions (Naseem <em>et al.</em>, 2019). Analysis of the strongest mutants in a mouse model of oral candidiasis revealed an important role for the <em>RVS161</em> and <em>RVS167</em> genes, which are known to promote the scission phase of endocytosis (Naseem <em>et al.</em>, 2019). The <em>RVS</em> genes are also important for systemic infections caused by <em>C. albicans</em>, further implicating endocytosis in virulence (Douglas <em>et al.</em>, 2009).<br/> <br/>Blocking endocytosis may have therapeutic value, as small molecules inhibitors of endocytosis and hyphal growth showed efficacy against oral candidiasis (Pierce <em>et al.</em>, 2015; Bar-Yosef <em>et al.</em>, 2017; Romo <em>et al.</em>, 2017; Vila <em>et al.</em>, 2017). Furthermore, the rvsï and other <em>C. albicans</em> endocytosis mutants were much more susceptible to killing by copper (Douglas and Konopka, 2019), suggesting copper could enhance the therapeutic value of endocytosis inhibitors. Thus, <em>C. albicans</em> agar invasion assays can be useful for screening mutant strains and novel inhibitors for efficacy in blocking the invasive growth of <em>C. albicans</em>.</p>
        </body>

<p class="ppzz"><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200818/20200818191915_2792.jpg" width="413"/><br/><strong>Figure 1. Morphologies of <em>C. albicans</em> cells.</strong> Scale bar, 5 Î¼m.</p><p class="pptt" id="biaoti35337">## Materials and Reagents</p><ol class="npwtwok"><li>Inoculating loop</li><li>Culture tubes (Sarstedt, catalog numbers: 62.554.205 for 15 ml size; 62.547.100 for 50 ml size)</li><li>2 L flasks (Pyrex, catalog number: 4980-2L)</li><li>Glass test tubes (Pyrex, catalog number: 9820)</li><li>Micropipette 20-200 Î¼l tips (VWR, catalog number: 89079-478)</li><li>Micropipette 100-1,000 Î¼l tips (Denville, catalog number: P2103-N)</li><li>Microcentrifuge tubes (Denville, catalog number: C2170)</li><li>Spectrophotometer cuvettes (Grainger, catalog number: 23M350)</li><li>10 cm-sized Petri dishes (Kord-Valmark, catalog number: 41141000)</li><li>96-well flat bottom polystyrene plate (Fisher Scientific, catalog number: 2420811)</li><li>Whatman filter paper (Fisher Scientific, catalog numbers: 1003-055 [55 mm], 1450-080 [80 mm], 1002-090 [90 mm], 1003-125 [125 mm])</li><li>Razor blade (Fisher Scientific, catalog number: 18100970)</li><li><em>C. albicans</em> strains can be obtained from the Fungal Genetics Stock Center (<a href="http://www.fgsc.net/" target="_blank">http://www.fgsc.net</a>)</li><li>Agar (Fisher Scientific, catalog number: BP9744-5)</li><li>BD Bacto Yeast extract (Fisher Scientific, catalog number: 212720)</li><li>Peptone (Fisher Scientific, catalog number: BP1420-2)</li><li>Dextrose (Fisher Scientific, catalog number: D16-10)</li><li>Bovine Calf Serum (Hyclone, catalog number: SH30073.03)</li><li>N-acetylglucosamine (GlcNAc) (Chem-Impex, catalog number: 01427)</li><li>Yeast Nitrogen Base: BD Bacto (Fisher Scientific, catalog number: 291920)</li><li>Adenine (Chem-Impex, catalog number: 00006)</li><li>Arginine (Fisher Scientific, catalog number: BP370-100)</li><li>Asparagine (Fisher Scientific catalog number: BP374-100)</li><li>Alanine (Fisher Scientific catalog number: BP369-100)</li><li>Citrulline (Argos, catalog number: 372-75-8)</li><li>Cysteine (Fisher Scientific, catalog number: BP376-100)</li><li>Glutamic acid (Fisher Scientific, catalog number: 194677)</li><li>Glycine (Fisher Scientific, catalog number: BP381-500)</li><li>Histidine (Fisher Scientific, catalog number: BP382-100)</li><li>Isoleucine (Fisher Scientific, catalog number: BP384-100)</li><li>Leucine (Fisher Scientific, catalog number: BP385-100)</li><li>Lysine (Fisher Scientific, catalog number: BP386-100)</li><li>Methionine (Fisher Scientific, catalog number: BP388-100)</li><li>Ornithine (Fisher Scientific, catalog number: BP389-100)</li><li>Phenylalanine (Fisher Scientific, catalog number: BP391-100)</li><li>Proline (Fisher Scientific, catalog number: BP392-100)</li><li>Serine (Chem-Impex, catalog number: 00279)</li><li>Tryptophan (Fisher Scientific, catalog number: BP395-100)</li><li>Threonine (Chem-Impex, catalog number: 00282)</li><li>Tyrosine (Fisher Scientific catalog number: BP396-100)</li><li>Uracil (Sigma, catalog number: U0750-100G)</li><li>Uridine (Chem-Impex, catalog number: 00339)</li><li>Valine (Fisher Scientific, catalog number: BP397-100)</li><li>p-Aminobenzoic acid (Sigma, catalog number: A-0254)</li><li>Sodium Phosphate (dibasic, Na<sub>2</sub>HPO<sub>4</sub>) (Sigma, catalog number: S9390)</li><li>HEPES (Millipore, catalog number: 391333-100GM)</li><li>Potassium phosphate (dibasic, K<sub>2</sub>HPO<sub>4</sub>) (Fisher Scientific, catalog number: P288-500)</li><li>Mannitol (Sigma, catalog number: M4125-500G)</li><li>GlcNAc Medium (see Recipes)</li><li>Serum Medium (see Recipes)</li><li>Spider Medium (see Recipes)</li><li>Alkaline pH Medium (150 mM HEPES pH 8) (see Recipes)</li><li>Synthetic Dextrose Medium (see Recipes)</li><li>Yeast Extract Peptone Dextrose (YEPD or YPD) Medium (see Recipes)</li><li>YP sucrose Medium (see Recipes)</li><li>10x supplemental solution for SD media (standard) (see Recipes)</li><li>10x supplemental solution for SD media (see Recipes) <br/></li></ol><p class="pptt" id="biaoti35338">### Equipment</p><ol class="npwtwok"><li>Roller</li><li>Centrifuge</li><li>Autoclave</li><li>Spectrophotometer (Thermo Fisher GENESYS 6, catalog number: 335908)</li><li>Pipet Aid (Drummond, catalog number: 4-000-100)</li><li>Micropipettor (Gilson, catalog numbers: FA10003M [2-20 Î¼l], FA10005M [20-200 Î¼l], FA10006M [100-1,000 Î¼l])</li><li>Microplate Replicator, 48-Well Format (also called a pinner) (Boekel, catalog number: 140501)</li><li>A light microscope equipped with a 2x objective and also a 40x long working distance objective<br/><em>Note: The model is not that important, but it will be essential to have a camera attached for recording images.</em> </li><li>Digital camera for taking macroscopic images of colony morphology<br/><em>Note: The type of camera is not that critical; a camera on a mobile phone will work fine.</em><br/></li></ol><p class="pptt" id="biaoti35339">### Procedure</p><ol class="npwone"><li>Invasion of cells spotted on top of agar<br/><em>Notes:</em> <ol class="npwthree"><li><em>If working with strains from a large collection of mutants, use PCR to confirm that the appropriate genes are deleted in the mutant strains.</em> </li><li><em>To help distinguish between mutants that are defective in the initiation or maintenance of invasive growth, cells can be preinduced to form hyphae in liquid by incubation for 90 min at 37 Â°C in YPD plus 10% BCS or 50 mM GlcNAc prior to application onto the surface of an agar plate (Naseem </em><em>et al.</em><em>, 2019). (Mutants that are defective in maintenance will not continue to form hyphae. Mutants with a defect in initiation, but not maintenance, can continue to form hyphae if they are first induced under conditions that will stimulate hyphal growth.)</em> </li></ol><br/><ol class="npwtwo"><li>Wild type and mutant <em>C. albicans</em> cells are streaked on to YPD plates from frozen stocks and incubated for 2 days at 30 Â°C.</li><li>A single colony from freshly streaked overnight culture grown on a YPD plate is transferred with an inoculating loop into 5 ml YPD liquid medium in a 15 ml glass tube. Grow cells overnight in a 30 Â°C incubator on a roller. </li><li>Harvest the cells from the overnight culture by centrifuging for 5 min at 1,000 <em>x g</em> in a 15 ml centrifuge tube. Resuspend the pellet in 10 ml PBS and centrifuge it again at 1,000 <em>x g</em> for 5 min. Repeat the same process twice and then resuspend the cells into 5 ml PBS.</li><li>Adjust the cells to 1 x 10<sup>7</sup> cells/ml. A convenient way to do this is to use a spectrophotometer at 600 nm wavelength. However, it is necessary to first set up a calibration curve to determine the relationship between cell number and optical density (OD). The relationship can vary between different spectrophotometers because a spectrophotometer measures light scattering, not absorbance. The final cell density can then be adjusted as follows: [Cell density desired] x [volume 1] = [Cell density initial] x [volume 2]. (Note that the volumes are indicating a ratiometric relationship. Volume 1 is the amount of cells that the investigator wants to use. This depends on how big of an experiment they are doing. Volume 2 is determined by volume 1.)</li><li>Use a micropipettor to spot a sample of each strain (3 Î¼l of 1 x 10<sup>7</sup> cells/ml in PBS) onto the surface of a 10 cm agar plate (Figure 2B). Place the spots at least 2 cm apart to prevent inhibition of growth due to nutrient depletion. See Figure 2A for a template.<br/><em>Note: On a typical 10 cm Petri plate we can spot at least 12 strains.</em><br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200818/20200818192001_9836.jpg" width="156"/><br/><strong>Figure 2. Spotting cells onto agar.</strong> A. Template for spotting cells on a Petri plate. B. Spotting cells onto the surface of a Petri plate with a micropipettor. C. Transferring cells onto the surface of an agar plate with a pinner.<br/><br/></li><li>Repeat this step to spot the samples onto different types of media to test invasive growth under different conditions, including agar with Bovine Calf Serum (BCS), GlcNAc, alkaline pH, and spider medium. Recipes are described below. The concentration of agar can be varied when preparing the medium to test the effects of differing the rigidity of the surrounding matrix (Douglas <em>et al.</em>, 2013) (see Recipe 2 below). Additional types of media that stimulate invasive hyphal growth have also been described (Liu <em>et al.</em>, 1994; Biswas and Morschhauser, 2005; Vinces <em>et al.</em>, 2006).</li><li>Invert the plates and incubate plates at 37 Â°C. It may be necessary to add a tray of water to the incubator to maintain humidity to prevent the plates from drying out if keeping plates for over 5 days.</li><li>Monitor the edge of the spots for invasive growth daily by eye (Figure 3A) or use a microscope (Figure 3B). It may be possible to detect initiation of invasive growth with the aid of a microscope after 1 day. Visible signs of invasive growth may take 2 days to one week.<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200818/20200818192027_0999.jpg" width="153"/><br/><strong>Figure 3. Invasive growth of <em>C. albicans</em> cells spotted on top of agar.</strong> A. The indicated cell types were spotted onto the surface of an agar plate with 4% BCS, and then incubated at 37 Â°C for 7 days. The plate was then photographed before (left side) or after (right side) washing off the non-adherent cells. B. Higher magnification of the edges of spots showing invasive hyphal growth defect for the <em>rvs161â</em> mutant. Panel A: Scale bar = 1 cm; Panel B: Scale bar = 200 Î¼m.</li></ol><br/></li><li>Invasion of cells applied to surface of agar with a 48 prong pinner<br/>To facilitate the analysis of a large set of strains, a 48 prong pinner can be used to transfer cells to the surface of an agar plate (<em>e.g.</em>, a Boekel Microplate Replicator). A 48 prong pinner is used because it can transfer cells onto agar in a standard 10 cm-sized Petri plate.<ol class="npwtwo"><li>Inoculate strains into wells of a 96-well flat bottom polystyrene plate containing 300 Âµl appropriate liquid medium. Incubate overnight at 30 Â°C until growth is saturated.</li><li>Sterilize 48 pin replicator.<br/><ol class="npwthree"><li>Assemble four containers together in a row (empty Petri dishes or empty micropipette tip box lids may be used).</li><li>Fill containers with 25 ml sterile deionized H<sub>2</sub>O, 10% bleach, or 97% ethanol in the following order: #1 = H<sub>2</sub>O, #2 = 10% bleach, #3 = H<sub>2</sub>O, #4 = 97% ethanol.</li><li>First, dip replicator into H<sub>2</sub>O, agitate to clean pins, then blot replicator on Whatman filter paper.</li><li>Place replicator into 10% bleach for 20-40 s. Blot replicator on filter paper.</li><li>Transfer to second container of H<sub>2</sub>O, agitate to rinse pins, blot replicator on filter paper one last time.</li><li>Dip replicator into 97% ethanol (do not blot).</li><li>Shake off excess ethanol, and pass through the flame of a Bunsen burner.</li><li>Once flames have extinguished, turn replicator ninety degrees, dip into 97% ethanol and flame.</li><li>After cooling, turn replicator ninety degrees once again, dip into 97% ethanol, flame, and cool.</li></ol></li><li>Insert cooled sterile replicator into the 48 wells on the left half of the 96-well strain plate.</li><li>Mix wells of plate with replicator pins, and then press replicator lightly onto Petri plate containing the desired medium (<em>e.g.</em>, 2% agar containing 4% BCS). Repeat this procedure to pin cells onto plates containing additional types of media.</li><li>Repeat sterilization of replicator (Steps B2c to B2i) when switching to new strains.</li><li>Incubate pinned plates at 37 Â°C for 48 h and examine for invasive growth as described above (Step A8).</li></ol><br/></li><li>Embedding method for analysis of invasive hyphal growth<br/>Embedding cells in agar is an excellent way to determine invasive growth since the cells are surrounded by agar in three dimensions. However, it can be more difficult to compare a set of mutant strains since they each have to be analyzed on a different Petri plate. Thus, a good approach is to screen mutants by spotting on the agar surface and then retesting those with mutant phenotypes by embedding into agar.<ol class="npwtwo"><li>Grow overnight cultures of the desired strains to be analyzed. Transfer a single colony from a fresh culture grown on a YPD plate into into 5 ml YPD liquid medium in a 15 ml glass tube. Incubate on a tube roller or in a shaking incubator at 30 Â°C.</li><li>Use a spectrophotometer to measure cell density at 600 nm. Adjust the cells to 5 x 10<sup>5</sup> cells/ml in 15 ml YPD broth (see Step A4).</li><li>Incubate in an incubator-shaker at 30 Â°C, 200-250 RPM, for 4 h to obtain log phase cells.</li><li>During incubation, prepare medium and cool in a water bath adjusted to 50 Â°C (approximately an hour may be required for cooling).</li><li>Types of media that may be used include 4% BCS, GlcNAc, alkaline pH, Spider medium, and YP sucrose as these activate a diverse set of different cell signaling pathways (Kornitzer, 2019). As described above, the concentration of agar can be varied to test the effects of differing the rigidity of the surrounding matrix.<br/><em>Warning: Do not add serum to media before autoclaving, as it will denature.</em> </li><li>After the 4 h incubation, measure density of log phase cells at 600 nm and calculate the volume of culture equivalent to 200 cells. Sample calculation: 2,000 cells/ml x 0.1 ml = 200 cells. See Step A4 for additional information on adjusting cell density.</li><li>Pour 25 ml of agar medium cooled to 50 Â°C into a 50 ml centrifuge tube.</li><li>For serum agar samples, add 1 ml of BCS, previously warmed to 50 Â°C.</li><li>Add volume of <em>C. albicans</em> culture equivalent to 200 cells.</li><li>Mix well (but avoid trapping bubbles) and then pour into a sterile 10 cm-sized Petri dish. Allow to solidify for 1 h.</li><li>Incubate plates at 37Â°C except for YP + Sucrose plates, which are incubated at room temperature or in a 25Â°C incubator.</li><li>Examine daily for evidence of invasive growth. Spots on agar were photographed with a digital camera (Figure 3A), or with a microscope using a 40x objective to view invasion extending from the edge of a spot (Figure 3B). The embedded colonies were photographed with a microscope through a 2x lens (Figure 4).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200818/20200818192059_2875.jpg" width="324"/><br/><strong>Figure 4. Invasive growth of cells embedded into agar.</strong> The indicated strains were embedded into agar containing 4% BCS and then incubated at 37 Â°C for one week. Scale bar = 1 mm.</li></ol></li></ol><p class="pptt" id="biaoti35340">### Data analysis</p><ol class="npwtwok"><li>Visually inspect the plates for signs of invasion into the agar surrounding the original spot of cells. Note the extensive invasive growth emanating from the original spot of cells seen for the wild type sample but is absent for the <em>rvs161â</em> mutant (Figure 3A).</li><li>To enhance detection of invasive growth, the agar plates can be gently washed with a stream of water to remove non-invasive cells (samples on the right in Figure 3A). Wear gloves and gently use a finger to dislodge non-adherent cells without damaging the agar.</li><li>At early times of incubation, or to check for even a very limited extent of invasive growth, the edges of the spots of cells can be viewed through a microscope with a 40x lens and photographed (Figure 3B). If appropriate, the zone of invasive growth can be measured with a ruler to quantify the extent of invasive growth.</li><li>Alternatively, the agar can be cut with a razor blade through the center of the spot and then turned on its side to visualize the depth of invasive growth (Warenda <em>et al.</em>, 2003). However, better results can be obtained by studying embedded samples.</li><li>Embedded samples can be analyzed similar to samples spotted on the surface. A sample of cells embedded into agar containing 4% BCS is shown in Figure 4. Note how the wild type and the complemented <em>rvs161â</em> + <em>RVS161</em> strains showed extensive invasive hyphal growth into the agar that was absent for the <em>rvs161â</em> mutant.</li><li>The extent of invasive growth should be analyzed on a daily basis for at least a week. Some mutant strains may show a strong defect at early time points, but will start to show significant invasive growth after ~7 days (Naseem <em>et al.</em>, 2019).<br/></li></ol><p class="pptt" id="biaoti35341">### Recipes</p><p class="ppzz"><em>Note: In many recipes the agar and other components are autoclaved separately and then combined later.</em> </p><ol class="npwtwo"><li>GlcNAc Medium (per liter) <br/>6.7 g Yeast Nitrogen Base in 400 ml water in 1 L flask<br/>18 g Agar in 400 ml water in 2 L flask<br/>Autoclave these two separately at 121Â°C for 40 min, when cooled to 55 Â°C, add 100 ml sterile 10x Amino Acid Supplement (if required), and 100 ml of filter sterilized 500 mM GlcNAc to the 2 L flask containing agar solution<br/>Mix well and pour into Petri dishes<br/><em>Note: GlcNAc is heat sensitive, add when agar is cooled to 55 Â°C (final concentration 50 mM), then pour into Petri plates.</em> </li><li>Serum Medium (per liter)<br/><ol class="npwthree"><li>Agar mass to desired % w/v required in 2 L flask<br/><em>Note: Higher concentrations of agar (4% and above) are difficult to work with as they become, making it difficult to pour and easier to trap bubbles. Use a flask that is at least four times the volume of the medium inside.</em> </li><li>Bovine Calf Serum volume filter sterilized separately, added to desired % v/v</li><li>1 L dH<sub>2</sub>O (adjust volume as needed)</li><li>Autoclave flask 121Â°C 40 min</li></ol><em>Note: Bovine Calf Serum is heat sensitive. Add appropriate volume at 55 Â°C (to final concentration and volume). Stir to mix, then pour into Petri plates.</em> </li><li>Spider Medium (per liter)<br/><ol class="npwthree"><li>13.5 g Agar 2 L flask</li><li>2 g K<sub>2</sub>HPO<sub>4</sub>, 10 g Yeast Nitrogen Base and 10 g Mannitol filter sterilized separately, adjust to pH 7.2</li><li>1 L dH<sub>2</sub>O (adjust volume as needed)</li><li>Autoclave flask 121Â°C 40 min and pour contents of smaller container into larger flask. Stir to mix, cool to 55 Â°C, then pour into Petri plates</li></ol></li><li>Alkaline pH Medium (150 mM HEPES pH 8) (per liter)<br/>18 g Agar<br/>6.7 g Yeast Nitrogen Base<br/>20 g Dextrose<br/>35.75 g HEPES<br/>100 ml 10x Amino Acid Supplement (if required)<br/>Place the 18 g agar in a 2 L flask with 500 ml deionized H<sub>2</sub>O<br/>In a separate flask, add 6.7 g Yeast Nitrogen Base, 20 g Dextrose, 35.75 g HEPES and 400 ml deionized H<sub>2</sub>O<br/><em>Note: A stock solution of HEPES can be made and sterile filtered ahead of time.</em><br/>Adjust the pH to pH 8<br/>If adding the Amino Acid Supplement Solution, use 100 ml less water<br/>Autoclave flasks 121 Â°C 40 min and combine the contents into one flask. Stir to mix, cool to 55 Â°C, then pour into Petri plates</li><li>Synthetic Dextrose Medium (per liter)<br/><ol class="npwthree"><li>6.7 g Yeast Nitrogen Base in 1 L flask</li><li>18 g Agar in 2 L flask</li><li>100 ml 10x Amino Acid Supplement (if required)</li><li>20 g Dextrose autoclaved separately</li><li>800 ml dH<sub>2</sub>O distributed among flasks (adjust volume as needed)</li><li>Autoclave flasks 121 Â°C 40 min and then pour contents of smaller flasks into the larger flask. Stir to mix, cool to 55 Â°C, then pour into Petri plates</li></ol></li><li>Yeast Extract Peptone Dextrose (YEPD or YPD) Medium (per liter)<br/><ol class="npwthree"><li>20 g Dextrose in 1 L flask</li><li>18 g Agar, 10 g Yeast Extract, 20 g Peptone, 120 mg Adenine in 2 L flask</li><li>1 L dH<sub>2</sub>O distributed among flasks (adjust volume as needed)</li><li>Autoclave flasks 121Â°C 40 min and then pour contents of smaller flasks into the larger flask. Stir to mix, cool to 55 Â°C, then pour into Petri plates</li></ol></li><li>YP sucrose Medium (per liter)<br/><ol class="npwthree"><li>20 g sucrose in 1 L flask</li><li>18 g Agar, 10 g Yeast Extract, 20 g Peptone, 120 mg Adenine in 2 L flask</li><li>1 L dH<sub>2</sub>O distributed among flasks (adjust volume as needed) </li><li>Autoclave flasks 121Â°C for 40 min and pour contents of smaller flasks into the larger flask. Stir to mix, cool to 55 Â°C, then pour into Petri plates</li></ol></li><li>10x supplemental solution for SD media (standard) (1 L)<br/>Mix the following into 1 L dH<sub>2</sub>O in beaker:<br/>Adenine 0.4 g<br/>Uracil 0.4 g <br/>Tryptophan 0.2 g<br/>Histidine 0.2 g <br/>Arginine 0.2 g<br/>Methionine 0.2 g<br/>Tyrosine 0.3 g<br/>Leucine 0.3 g<br/>Isoleucine 0.3 g<br/>Lysine 0.3 g <br/>Phenylalanine 0.5 g<br/>Glycine 1 g<br/>Asparagine 1 g<br/>Valine 1.5 g<br/>Threonine 2 g<br/>Serine 4 g<br/>Add 100 ml amounts into bottles and autoclave 121 Â°C 40 min</li><li>10x supplemental solution for SD media (1 L)<br/>Mix the following into 1 L dH<sub>2</sub>O in beaker:<br/>Alanine 0.5 g<br/>Asparagine 0.5 g<br/>Cysteine 0.5 g<br/>Glutamic acid 0.5 g<br/>Glycine 0.5 g<br/>Proline 0.5 g<br/>p-Aminobenzoic acid 0.05 g<br/>Sodium Phosphate (Dibasic) 4 g<br/>Add 100 ml amounts into each of 10 bottles and autoclave 121 Â°C 40 min</li></ol><p class="pptt" id="biaoti35342">### Acknowledgments</p><p class="ppzz">This work was supported by Public Health Service grants from the National Institutes of Health awarded to J.B.K. from the (R01GM116048, and R01AI047837).<br/>  The methods for studying invasive growth were briefly described in Naseem <em>et al.</em> (2019).</p><p class="pptt" id="biaoti35343">### Competing interests</p><p class="ppzz">The authors declare that they do not have any competing interests.</p><p class="pptt" id="biaoti35344">### References</p><ol class="npwtwok"><li>Bar-Yosef, H., Vivanco Gonzalez, N., Ben-Aroya, S., Kron, S. J. and Kornitzer, D. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28720834" target="_blank">Chemical inhibitors of <em>Candida albicans</em> hyphal morphogenesis target endocytosis.</a> <em>Sci Rep</em> 7(1): 5692. </li><li>Biswas, K. and Morschhauser, J. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15819622" target="_blank">The Mep2p ammonium permease controls nitrogen starvation-induced filamentous growth in <em>Candida albicans</em>.</a> <em>Mol Microbiol</em> 56(3): 649-669.</li><li>Douglas, L. M. and Konopka, J. B. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30633741" target="_blank">Plasma membrane architecture protects <em>Candida albicans</em> from killing by copper.</a> <em>PLoS Genet</em> 15(1): e1007911. </li><li>Douglas, L. M., Martin, S. W. and Konopka, J. B. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19596778" target="_blank">BAR domain proteins Rvs161 and Rvs167 contribute to <em>Candida albicans</em> endocytosis, morphogenesis, and virulence.</a> <em>Infect Immun</em> 77(9): 4150-4160. </li><li>Douglas, L. M., Wang, H. X. and Konopka, J. B. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24281718" target="_blank">The MARVEL domain protein Nce102 regulates actin organization and invasive growth of <em>Candida albicans</em>.</a> <em>mBio</em> 4(6): e00723-00713.</li><li>Glazier, V. E., Murante, T., Murante, D., Koselny, K., Liu, Y., Kim, D., Koo, H. and Krysan, D. J. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28793308" target="_blank">Genetic analysis of the <em>Candida albicans</em> biofilm transcription factor network using simple and complex haploinsufficiency.</a> <em>PLoS Genet</em> 13(8): e1006948. </li><li>Kornitzer, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30823468" target="_blank">Regulation of <em>Candida albicans</em> hyphal morphogenesis by endogenous signals.</a> <em>J Fungi (Basel)</em> 5(1): 10.3390/jof5010021.</li><li>Kumamoto, C. A. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15800048" target="_blank">A contact-activated kinase signals <em>Candida albicans</em> invasive growth and biofilm development.</a> <em>Proc Natl Acad Sci U S A</em> 102(15): 5576-5581.</li><li>Kumamoto, C. A. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18679170" target="_blank">Molecular mechanisms of mechanosensing and their roles in fungal contact sensing.</a> <em>Nat Rev Microbiol</em> 6(9): 667-673. </li><li>Liu, H., Kohler, J. and Fink, G. R. (1994). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7992058" target="_blank">Suppression of hyphal formation in <em>Candida albicans</em> by mutation of a STE12 homolog.</a> <em>Science</em> 266(5191): 1723-1726.</li><li>Naseem, S., Douglas, L. M. and Konopka, J. B. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31719181" target="_blank"><em>Candida albicans</em> <em>rvs161â</em> and <em>rvs161â</em> endocytosis mutants are defective in invasion into the oral cavity.</a> <em>mBio</em> 10(6): 10.1128/mBio.02503-02519. </li><li>Naseem, S., Gunasekera, A., Araya, E. and Konopka, J. B. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21700702" target="_blank">N-acetylglucosamine (GlcNAc) induction of hyphal morphogenesis and transcriptional responses in <em>Candida albicans</em> are not dependent on its metabolism.</a> <em>J Biol Chem</em> 286(33): 28671-28680. </li><li>Noble, S. M., French, S., Kohn, L. A., Chen, V. and Johnson, A. D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20543849" target="_blank">Systematic screens of a <em>Candida albicans</em> homozygous deletion library decouple morphogenetic switching and pathogenicity.</a> <em>Nat Genet</em> 42(7): 590-598. </li><li>Noble, S. M., Gianetti, B. A. and Witchley, J. N. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27867199" target="_blank"><em>Candida albicans</em> cell-type switching and functional plasticity in the mammalian host.</a> <em>Nat Rev Microbiol</em> 15(2): 96-108.</li><li>O'Meara, T. R., Veri, A. O., Ketela, T., Jiang, B., Roemer, T. and Cowen, L. E. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25824284" target="_blank">Global analysis of fungal morphology exposes mechanisms of host cell escape.</a> <em>Nat Commun</em> 6: 6741.</li><li>Pierce, C. G., Chaturvedi, A. K., Lazzell, A. L., Powell, A. T., Saville, S. P., McHardy, S. F. and Lopez-Ribot, J. L. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26691764" target="_blank">A novel small molecule inhibitor of <em>Candida albicans</em> biofilm formation, filamentation and virulence with low potential for the development of resistance.</a> <em>NPJ Biofilms Microbiomes</em> 1. </li><li>Pierce, J. V. and Kumamoto, C. A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22829676" target="_blank">Variation in <em>Candida albicans</em> EFG1 expression enables host-dependent changes in colonizing fungal populations.</a> <em>mBio</em> 3(4): e00117-00112.</li><li>Riggle, P. J., Andrutis, K. A., Chen, X., Tzipori, S. R. and Kumamoto, C. A. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10377153" target="_blank">Invasive lesions containing filamentous forms produced by a <em>Candida albicans</em> mutant that is defective in filamentous growth in culture.</a> <em>Infect Immun</em> 67(7): 3649-3652. </li><li>Romo, J. A., Pierce, C. G., Chaturvedi, A. K., Lazzell, A. L., McHardy, S. F., Saville, S. P. and Lopez-Ribot, J. L. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29208749" target="_blank">Development of anti-virulence approaches for Candidiasis via a novel series of small-molecule inhibitors of <em>Candida albicans</em> filamentation.</a> <em>mBio</em> 8(6): e01991-01917.</li><li>Vila, T., Romo, J. A., Pierce, C. G., McHardy, S. F., Saville, S. P. and Lopez-Ribot, J. L. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27268130" target="_blank">Targeting <em>Candida albicans</em> filamentation for antifungal drug development.</a> <em>Virulence</em> 8(2): 150-158. </li><li>Vinces, M. D., Haas, C. and Kumamoto, C. A. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16682460" target="_blank">Expression of the <em>Candida albicans</em> morphogenesis regulator gene CZF1 and its regulation by Efg1p and Czf1p.</a> <em>Eukaryot Cell</em> 5(5): 825-835. </li><li>Warenda, A. J., Kauffman, S., Sherrill, T. P., Becker, J. M. and Konopka, J. B. (2003). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12819094" target="_blank"><em>Candida albicans</em> septin mutants are defective for invasive growth and virulence.</a> <em>Infect Immun</em> 71(7): 4045-4051. </li><li>Witchley, J. N., Penumetcha, P., Abon, N. V., Woolford, C. A., Mitchell, A. P. and Noble, S. M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30870623" target="_blank"><em>Candida albicans</em> morphogenesis programs control the balance between gut commensalism and invasive infection.</a> <em>Cell Host Microbe</em> 25(3): 432-443 e436.<br/></li></ol>