<body>
            <h1>Live-cell Imaging by Super-resolution Confocal Live Imaging Microscopy (SCLIM): Simultaneous Three-color and Four-dimensional Live Cell Imaging with High Space and Time Resolution</h1>
            <p><strong>Authors:</strong> Kazuo Kurokawa, Akihiko Nakano</p>
            <h2>Abstract</h2>
            <p>Many questions in cell biology can be solved by state-of-the-art technology of live cell imaging. One good example is the mechanism of membrane traffic, in which small membrane carriers are rapidly moving around in the cytoplasm to deliver cargo proteins between organelles. For directly visualizing the events in membrane trafficking system, researchers have long awaited the technology that enables simultaneous multi-color and four-dimensional observation at high space and time resolution. Super-resolution microscopy methods, for example STED, PALM/STORM, and SIM, provide greater spatial resolution, however, these methods are not enough in temporal resolution. The super-resolution confocal live imaging microscopy (SCLIM) that we developed has now achieved the performance required. By using SCLIM, we have conducted high spatiotemporal visualization of secretory cargo together with early and late Golgi resident proteins tagged with three different fluorescence proteins. We have demonstrated that secretory cargo is indeed delivered within the Golgi by cisternal maturation. In addition, we have visualized details of secretory cargo trafficking in the Golgi, including formation of zones within a maturing cisterna, in which Golgi resident proteins are segregated, and movement of cargo between these zones. This protocol can be used for simultaneous three-color and four-dimensional observation of various phenomena in living cells, from yeast to higher plants and animals, at high spatiotemporal resolution.</p>
            <p><strong>Keywords:</strong> Live cell imaging, Simultaneous 3-color and 4-dimensional observation, Super-resolution and high-speed observation, Secretory cargo, Golgi</p>
            <h2>Background</h2><p>Development of live cell imaging by state-of-the-art light microscopy has greatly contributed to address many problems remaining in cell biology. For example, to visualize the events in secretory pathway, in which small membrane carriers are rapidly moving around in the cytoplasm, requires high speed and three-dimensional (3D) image acquisition. Super-resolution confocal live imaging microscopy (SCLIM), which we have developed (Kurokawa <em>et al.</em>, 2013), can achieve high speed visualization of dynamic behavior of 3D structures tagged with green and red fluorescence proteins in living cells with spatial resolution beyond the diffraction limit of light (Matsuura-Tokita <em>et al.</em>, 2006; Ito <em>et al.</em>, 2012; Okamoto <em>et al.</em>, 2012; Suda <em>et al.</em>, 2013; Uemura <em>et al.</em>, 2014; Kurokawa <em>et al.</em>, 2014; Iwai <em>et al.</em>, 2016; Ishii <em>et al.</em>, 2016 and 2019; Kurokawa <em>et al.</em>, 2016). We can now use 3-color version of SCLIM, which enable us to conduct simultaneous 3-color (green, red, and infrared) observation in living yeast, <em>Drosophila</em>, plant, and mammalian cells at high spatial and time resolution (Ito <em>et al.</em>, 2018; Kurokawa <em>et al.</em>, 2019; Maeda <em>et al.</em>, 2019; Tojima <em>et al.</em>, 2019; Fujii <em>et al.</em>, 2020). We have been working on the molecular mechanisms underlying protein transport and sorting in the Golgi apparatus and have utilized the cutting-edge performance of our technology to tackle a big question in this field. The purpose of this article is to introduce to readers our approaches by super-resolution and high-speed live cell imaging and provide basic protocols.<br/> <br/>The Golgi apparatus is the central station of membrane traffic. One third of proteins in eukaryotic cells are newly synthesized in the endoplasmic reticulum (ER) and then delivered to the Golgi. In the Golgi, they are processed and glycosylated before being sorted to their final destinations (Emr <em>et al.</em>, 2009). The Golgi is usually in the form of ordered stacks of several cisternae. Secretory cargo travels across the stack from the <em>cis</em> side to the <em>trans</em> side of cisternae and then to the <em>trans</em>-Golgi network (TGN) (Griffiths and Simons, 1986). The budding yeast <em>Saccharomyces cerevisiae</em> presents a unique example of the Golgi, in which individual cisternae, <em>cis</em>, medial and <em>trans</em>, do not stack but scatter in the cytoplasm (Matsuura-Tokita <em>et al.</em>, 2006; Okamoto <em>et al.</em>, 2012). Three major models of secretory cargo traffic within the Golgi have been proposed: 1) traffic via cisternal maturation, 2) traffic by anterograde vesicular carriers, and 3) traffic via interconnected continuity of cisternae (Glick and Nakano, 2009; Nakano and Luini, 2010; Pfeffer, 2010; Glick and Luini, 2011). Among these models, the cisternal maturation model has been favored to explain the live imaging observation in budding yeast that Golgi cisternae mature from early to late cisternae (Losev <em>et al.</em>, 2006; Matsuura-Tokita <em>et al.</em>, 2006; Rivera-Molina and Novick, 2009) and the movement of large cargo through the Golgi stacks without leaving cisternae in mammals (Bonfanti <em>et al.</em>, 1998; Lanoix <em>et al.</em>, 2001; Martinez-Menarguez <em>et al.</em>, 2001; Mironov <em>et al.</em>, 2001).The lacking proof of this mechanism was to visualize the delivery of cargo remaining in the maturing Golgi cisterna.<br/> <br/>Electron microscopy shows detailed snap information where secretory cargo and Golgi resident proteins localize in the individual Golgi cisternae, but cannot directly visualize how secretory cargo traverse between these cisternae. Live imaging by using conventional epi- and confocal fluorescence microscopies visualize behavior of secretory cargo <em>in vivo</em>, but cannot provide detailed information where secretory cargo and Golgi resident proteins locate within the individual Golgi cisternae because of low spatial resolution. Super-resolution microscopy methods, for example STED, PALM/STORM, and SIM overcoming the diffraction limit of light, provide greater spatial resolution (Schermelleh <em>et al.</em>, 2010). However, these methods are not high in temporal resolution, because time and space resolutions usually trade off each other. By using SCLIM technology, we have shown that secretory cargo is delivered within the Golgi by cisternal maturation. Maturing cisterna involves segregated zones of the earlier and later Golgi resident proteins. The location of cargo changes from the early to the late zone within the cisterna during the progression of maturation (Kurokawa <em>et al.</em>, 2019).</p>
        </body>

<p class="pptt" id="biaoti35390">## Materials and Reagents</p><ol class="npwtwok"><li>90 mm diameter Cell culture dishes, sterile (Iwaki, catalog number: <a class="blackNoUnder" target="_blank"> SH90-15 </a> )</li><li>22 mm x 40 mm Micro cover glasses (Matusnami Glass, thickness No.1, 0.13-0.17mm)</li><li>18 mm x 18 mm Micro cover glasses (Matusnami Glass, thickness No.1, 0.13-0.17mm)</li><li>20 ml Syringes (Terumo, catalog number: <a class="blackNoUnder" target="_blank"> SS-20ESZ </a> )</li><li>Pipette tips, sterile (Sorenson, 1-200 Âµl Graduated tip, catalog number: <a class="blackNoUnder" target="_blank"> 27730 </a> )</li><li>Kimwipes (Nihon Seishi, catalog number: <a class="blackNoUnder" target="_blank"> 62011 </a> )</li><li>Tissue culture test plates, 24-well plate, sterile (TPP, catalog number: <a class="blackNoUnder" target="_blank"> 92424 </a> )</li><li>Toothpicks sterilized before use (any brand)</li><li>Budding yeast cells expressing two Golgi markers and secretory cargo tagged with different fluorescent proteins. (see Procedure B for details)</li><li>Silicon grease (Dow Corning Toray, catalog number: <a class="blackNoUnder" target="_blank"> HVG-50 TUBE </a> )</li><li>Concanavalin A (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> C2010 </a> )</li><li>Yeast nitrogen base without amino acids (Difco Laboratories)</li><li>Casamino acids (Difco Laboratories)</li><li>Glucose (Nacalai Tesque)</li><li>Agar (Wako)</li><li>MCD agar plates (see Recipes)</li><li>MCD medium (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35391">### Equipment</p><ol class="npwtwok"><li>Micropipette (Gilson, Pipetman, model: <a class="blackNoUnder" target="_blank"> P200 </a> )</li><li>Bio Shaker (Taitec, model: <a class="blackNoUnder" target="_blank"> M-BR-022UP </a> )</li><li>SCLIM (Figure 1) is equipped with: <br/><ol class="npwthree"><li>Inverted fluorescence microscope (Olympus, model: <a class="blackNoUnder" target="_blank"> IX-73 </a> ) <br/></li><li>Objective lens (Olympus, UPlanSApo 100x NA 1.4 oil objective lens)<br/></li><li>Spinning-disk confocal scanner (Yokogawa Electric, CSU10 custom made model)<br/>Other type of CSU can be used (<em>e.g.</em>, CSU10, CSUX1, CSUW1). Confocal scanner is equipped with custom made dichroic mirror having high spectroscopic properties for 3-color observation (Kurokawa <em>et al.</em>, 2013).</li><li>Excitation lasers: Solid-state 3 lasers with emission at 473 nm (Cobolt, CW 473nm, DPSS, 50 mW), 561 nm (Cobolt, CW 561 nm, DPSS, 50 mW), and 671 nm (CrystaLaser, CW 671 nm, DPSS, 100 mW)<br/>These 3 lasers can excite GFP, mRFP or mCherry, and iRFP simultaneously. Irradiation of the sample by these 3 lasers is controlled by 3 independent electromagnetic shutters (Sigma Koki). These shutters are operated by the demand of the custom-made software. Irradiation powers of 3 lasers can be adjusted with variable ND filters.</li><li>Spectroscopic unit: A custom made system equipped with custom-made dichroic mirrors, reflection mirrors, band pass filters, and long pass filter to separate 3-color fluorescence (green, red, and infra-red) images to 3 different channels (Kurokawa <em>et al.</em>, 2013)<br/>The angles of dichroic mirrors and reflection mirrors can be adjusted from outside of the spectroscopic unit.</li><li>Magnification lens (NIKON, VM lens C-4x [4x] or C-4x plus C-2.5x [10x])<br/>A magnification lens is putted in the light path between the confocal scanner and spectroscopic unit for raising spatial resolution.</li><li>EM-CCD cameras (Hamamatsu Photonics, C9100-13)<br/>They contain 512 x 512 frame transfer CCD sensor. Pixel size of sensor is 16 x 16 Âµm. 3 EM-CCD cameras are set up for taking green, red, and infra-red fluorescence images.</li><li>Image intensifier (Hamamatsu Photonics)<br/>3 Image intensifiers are equipped with each 3 EMCCD cameras. Image intensifiers are cooled with a custom-made cooling system to achieve low signal-to-noise ratio amplification. The amplification gain of these 3 image intensifiers are controlled independently.</li><li>Piezo actuator (Yokogawa Electric, custom made model)<br/>It is equipped at the neck of objective lens to oscillate Z-axis position of the lens at high frequency (maximum, 30 Hz). Normally, we use it at 4-15 Hz.</li><li>Thermo-control stage (Tokai Hit)<br/>The temperature of the sample during observation can be controlled by thermo-control stage.</li><li>Image acquisition system<br/>Custom made system controls simultaneous collection of 3 EM-CCD cameraâs images at each Z-axis position and each time point (Yokogawa Electric).<br/><br/><img alt="" class="layerphoto" height="441" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902024704_7965.png" width="300"/><br/><strong>Figure 1. Set up of 3-color SCLIM.</strong> A. An inverted microscopy (a) is combined with, an ultra-fast piezo actuator (i), a custom-made spinning-disk confocal scanner (c), 3 excitation lasers (d), a magnifier lens (f), a custom-made spectroscopic unit (containing dichroic mirrors, refraction mirrors, and band-pass and long-pass filters) (e), 3 cooled image intensifiers (h), and 3 EM-CCD cameras (g). B. Another view of the custom-made spectroscopic unit (e), 3 cooled image intensifiers (h), and 3 EM-CCD cameras (g) is shown.</li></ol></li></ol><p class="pptt" id="biaoti35392">### Software</p><ol class="npwtwok"><li>Volocity software (PerkinElmer)</li><li>MetaMorph software (Molecular Devices)<br/></li></ol><p class="pptt" id="biaoti35393">### Procedure</p><ol class="npwone"><li>Preparation of a Concanavalin A-coated glass dish for imaging living yeast cells<br/>Concanavalin A is used to immobilize living yeast cells on the cover glass surface for microscopic observation.<br/><ol class="npwtwo"><li>Make a square bank in 1-2 mm depth of 18 mm x 18 mm or less with silicon grease on a 22 mm x 40 mm cover glass (Figure 2A). Silicon grease is extruded from the syringe through a pipet tip with a cut end.<br/><br/><img alt="" class="layerphoto" height="579" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902024734_8300.png" width="300"/><br/><strong>Figure 2. Sample preparation.</strong> A. A square bank in 1-2 mm depth of 18 mm x 18 mm or less is made with silicon grease on a 22 mm x 40 mm cover glass. The cover glass surface within square bank is coated with Concanavalin A. B. An aliquot of yeast liquid culture is applied into the square bank of Concanavalin A-coated 22 mm x 40 mm cover glass. C. The yeast liquid culture is sandwiched by the Concanavalin A-coated cove glass and a 18 x 18 mm cover glass. The yeast cells are then subjected to microscopic observation.<br/><br/></li><li>Apply about 25 Âµl of 0.2 mg/ml Concanavalin A on the cover glass within square bank.</li><li>Wait for about 2 min to coat the cover glass with Concanavalin A.</li><li>Remove the excess Concanavalin A solution from the glass surface.</li><li>Dry Concanavalin A at room temperature for about 10 min. The Concanavalin A-coated glass dish needs to be freshly prepared for well immobilizing living yeast cells.<br/><br/></li></ol></li><li>Preparation of yeast cells for the fluorescent microscopic pulse-chase assay of cargo transport<br/>Yeast temperature-sensitive mutants such as <em>sec31-1</em> and <em>uso1-1</em> show a defect in ER-to-Golgi cargo transport at the restrictive temperature (37 Â°C). By combining the gene expression of cargo by a heat-shock inducible promoter and the use of yeast temperature-sensitive mutants, the fluorescence microscopic pulse-chase assay has been established to visualize cargo transport from the ER to the Golgi and within the Golgi (Kurokawa <em>et al.</em>, 2014 and 2019).<br/><ol class="npwtwo"><li>Pick up temperature-sensitive mutant yeast cells expressing cargo and two Golgi cisternal markers (<em>e.g.</em>, <em>uso1-1</em> cells expressing heat-shock-inducible Axl2-GFP (transmembrane cargo) and constitutive Mnn9-mCherry (<em>cis</em>-Golgi marker) and Sys1-iRFP (<em>trans</em>-Golgi marker)) from a frozen stock using a sterile toothpick, and streak yeast cells on a MCD agar plate with appropriate amino acid supplements.</li><li>Incubate yeast cells on the agar plate at 24 Â°C for 2-3 days.</li><li>Inoculate yeast cells into 1 ml MCD medium with appropriate amino acid supplements in a 24-well plate by using a sterile toothpick.</li><li>Grow yeast cells to the early-to-mid logarithmic phase at 24 Â°C with shaking at 500 rpm.</li><li>Incubate yeast cells at 37 Â°C for 15-30 min to induce GFP-tagged cargo (Axl2-GFP) synthesis and to inhibit the ER-to-Golgi cargo transport.</li><li>Apply about 100 Âµl of yeast liquid culture into a square bank of Concanavalin A-coated 22 mm x 40 mm cover glass (Figure 2B).</li><li>Wait for 2 min until yeast cells are immobilized on a Concanavalin coated cover glass.</li><li>Cover the square bank with 18 mm x 18 mm cover glass (Figure 2C).<br/><em>Note: Yeast cells can grow well because two cover glasses are spaced with silicone grease.</em> </li><li>Remove overflowing yeast liquid culture from sandwiched space of two cover glasses with Kimwipes.</li><li>Place the cover glasses sandwiching yeast cells on a microscope stage with the Concanavalin coated larger glass down. The temperatures of the microscopic stage and objective lens can be set at 24 Â°C by an automatic thermo-control system.<br/><br/></li></ol></li><li>Image acquisition with SCLIM<ol class="npwtwo"><li>Set up of SCLIM before observation<br/>Due to various factors, such as chromatic aberrations of lenses in the whole microscopic and spectroscopic system, the 3 different fluorescence signals coming from the same position (XYZ) may be projected to different positions (XYZ) of 3 cameras. To conduct simultaneous 3-color 3D and 4D observation, it is necessary to correct the XYZ misalignments of these fluorescence signals. SCLIM has devised a method to correct the misalignments of XYZ positions in 3 cameras by using fluorescence beads (see Procedure F). Setting up SCLIM needs an expert operator who has well known the optics of SCLIM system.</li><li>Image acquisition<br/>Fluorescent molecules have a broad range of emission. Thus, for example, when GFP is expressed at a very high level, the fluorescence signal of GFP may be detected even by a camera with a longer-wavelength emission filer. Such a leakage of fluorescence signals is called as bleed-through or crosstalk. It is critical to avoid such bleed-through signals as much as possible. The powers of 3 excitation lasers and the amplification gains of 3 image intensifiers should be independently controlled to minimize bleed-through. Users need to be trained in how to operate SCLIM correctly for âImage acquisitionâ step.<br/><ol class="npwthree"><li>Observe the yeast cells expressing cargo (heat-shock-inducible Axl2-GFP) and Golgi cisternal markers, <em>cis</em>-Golgi marker (Mnn9-mCherry) and <em>trans</em>-Golgi marker (Sys1-iRFP), by SCLIM. In these cells, we can observe that <em>cis</em>-Golgi cisterna labeled with Mnn9-mCherry matures to <em>trans</em>-Golgi cisterna labeled with Sys1-iRFP while maintaining Axl2-GFP cargo.<br/></li><li>2D confocal fluorescence data are separated in the custom-made spectroscopic unit. GFP, mCherry, and iRFP fluorescence images finally pass through the spectral windows defined by 2 band pass filters (GFP; 490-545 nm, and mCherry; 580-660 nm) and 1 long pass filter (iRFP; 680 nm-) and are simultaneously collected by synchronized 3 EMCCD cameras.<br/></li><li>Collect Z slice of 3-color images at appropriate time intervals during observation time (<em>e.g.</em>, at every about 5 seconds during total 2-3 min observation). An appropriate range of Z-axis (<em>e.g.</em>, 3 Î¼m cover about 80% of yeast cell volume) is scanned into optical slices 100 nm apart by the piezo actuator equipped to the objective lens.<br/></li><li>Raw data sets (Z slice 3-color images at each time points) are once stored in the TIFF format in the hard-disk memory of the image acquisition computer.<br/><br/></li></ol></li></ol></li><li>Reconstruction of 3D images from raw data sets and making of deconvolved 3D images using Volocity software<ol class="npwtwo"><li>Transfer raw data sets to the computer installed with Volocity software.</li><li>Reconstruct 3D images from raw data sets using Volocity software.<br/><ol class="npwthree"><li>Select âCreate newâ from Actions menu in Volocity to make Image sequence.<br/></li><li>Read out raw data sets in 3 channels into Image sequence to reconstruct 3D images at each time point (Figure 3A).<br/></li><li>Select âRemove Noiseâ from the Tools menu and remove noise from all 3D images using a fine filter.<br/></li><li>Select âChange Colorsâ from the Tools menu and change colors of 3 channels (<em>e.g.</em>, GFP as green, mCherry as red, and iRFP as blue).<br/></li><li>Select âCorrect Photobleachingâ from the Tools menu and correct for photobleaching of 3D images to maintain intensities that are reduced by photobleaching during the observation time.<br/></li><li>Select âPropertiesâ from the Edit menu and change information of XYZ pixel sizes in 3D images. Pixels of (X) and (Y) are 0.06 Âµm and pixel of (Z) is 0.1 Âµm when we use 100x objective lens and 4x amplification lens and scan Z-axis with optical slices 100 nm apart.<br/></li><li>Select âFast restorationâ from the Tools menu and make deconvolved 3D images using PSF (point-spread function) parameters of 3 fluorescence proteins optimized for the spinning-disk confocal scanner. These parameters are made with âCalculated PSFâ in âCreate newâ from Actions menu.<br/></li><li>Raw 3D images and deconvolved 3D images can be shown in different modes of viewing, XYZ mode (Figure 3B), 3D opacity mode (Figure 3C), and so on.<br/><br/><img alt="" class="layerphoto" height="388" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902024755_2255.png" width="300"/><br/><strong>Figure 3. Reconstruction of raw and deconvolved 3D images.</strong> A. Image sequence of raw data sets in 3 channels at each time point are shown. B. Reconstructed raw and deconvolved 3D images at one time point are shown in XYZ mode. C. Reconstructed raw and deconvolved 3D images at one time point are shown in 3D opacity mode. <em>cis</em>-Golgi (Mnn9-mcherry, red) and <em>trans</em>-Golgi (Sys1-iRFP, blue) existed dispersed in the cytoplasm. Some cargos (Axl2-GFP, green) were transported to their final destination, the plasma membrane. Other cargos localized within Golgi cisternae. Dotted circles in the images highlight the edges of yeast cells.<br/></li></ol><br/></li></ol></li><li>Presentation and analysis of 3D and 4D images<ol class="npwtwo"><li>Regions of interest in deconvolved 3D images can be manually selected. Present the images as XYZ mode and outline the region of interest using the selection tool (Figure 4A, left panel). Select âCrop to Selectionâ from the Actions menu to create a new deconvolved 3D image (Figure 4A, right 2 panels). 3D images shown in 3D opacity mode can be presented at any desired angles (Figure 4A, right 2 panels).</li><li>To analyze the change of fluorescence signal intensities over time, we make time-lapse movie of deconvolved 3D images using âMake Movieâ from the Movie menu (Video 1). Total fluorescence signal intensities of 3D images in 3 channels (green, red, and infrared) are collected to calculate relative fluorescence intensity values of <em>e.g.</em>, Axl2-GFP (cargo, green), Mnn9-mCherry (<em>cis</em>-Golgi, red), and Sys1-iRFP (<em>trans</em>-Golgi, blue) using Metamorph (Figure 4B, Video 1). Note, by using 3D opacity mode, we can make 3D images viewed at various angles, which helps avoid collecting incorrect fluorescence signals of another Golgi cisterna (Figure 4B, arrowheads).</li><li>To analyze correct localization of cargo and Golgi resident proteins, 3D images shown in 3D opacity mode at any angles can be used for line-scan analysis using MetaMorph (Figure 4C).<br/><br/><img alt="" class="layerphoto" height="439" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902024814_4929.png" width="300"/><br/><strong>Figure 4. Presentation and analysis of 3D and 4D images.</strong> A. Select the region of interest in XYZ mode image (left panel) to make a new cropped 3D image. Cropped 3D images are shown in 3D opacity mode at two different angles (right 2 panels). Cargos (Axl2-GFP, green) localized at the plasma membrane (arrowheads) and within the Golgi cisterna (dotted circles) labeled with both of <em>cis</em>- (Mnn9-mcherry, red) and <em>trans</em>-Golgi (Sys1-iRFP, blue) markers. B. 4D images (time-lapse 3D deconvolved images) are shown (left panels) (see also Video 1). The frame where Mnn9-mCherry signals disappeared was set as 0 s. Relative fluorescence intensities of Axl2-GFP (cargo, green), Mnn9-mCherry (<em>cis</em>, red), and Sys1-iRFP (<em>trans</em>, blue) at each time point are shown on the right. Maturation of Golgi cisterna with cargo was observed (dotted circles). The cisterna was labeled with <em>cis</em>-Golgi marker (Mnn9-mcherry, red) and <em>trans</em>-Golgi marker (Sys1-iRFP, blue) at the beginning (-25 s). It lost Mnn9-mcherry signals and gained Sys1-iRFP signals, and then it was labeled only with Sys1-iRFP at -5.6 s, indicating it matured to <em>trans</em>-Golgi. Finally, this cisterna matured to the TGN because Sys1-iRFP signals were disappeared at 11.2 s. Note, the angle showing 3D images is important to obtain the correct fluorescence intensities in one cisterna. Top view analysis alone of fluorescence intensities over time may have obtained incorrect values of red fluorescence, because arrowheads in left panels indicate another <em>cis</em> cisterna, which was present just below the analyzed cisterna. C. 3D deconvolved images were subjected to line scan analysis (upper panels). 3D deconvolved images (merged image, Axl2-GFP, Mnn9-mCherry, and Sys1-iRFP) at -16.8 s are shown (upper panels). Lower graph shows relative fluorescence intensities of Axl2-GFP (cargo, green), Mnn9-mCherry (<em>cis</em>, red), and Sys1-iRFP (<em>trans</em>, blue) along the white arrows in the maturing cisterna. Mnn9-mCherry and Sys1-iRFP showed segregated localization within the cisterna. Cargo located both regions.<br/><br/><!--video3732-1å¼å§--><div class="videodiv" id="vd3732-1" style="height:480px;"></div><div class="videophoto" id="vp3732-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3732/3732-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Maturation of the Golgi cisterna with cargo viewed from two different angles</strong> </div><br/></li></ol></li><li>Checking and correcting the misalignments of XYZ positions in 3 cameras<br/>Before the observation of yeast cells expressing 3 fluorescence proteins, we always check and correct the misalignments of XYZ positions in 3 cameras by the following ways.<br/><br/><strong>Check and correct the misalignment in XY positions by using pinhole images of 3 cameras</strong><br/><ol class="npwtwo"><li>Stop the rotation of the spinning-disk of the confocal scanner.<br/></li><li>Apply a weak light from a halogen lamp light source to objective lens.<br/></li><li>Obtain the halogen light images passed through the pinhole with 3 cameras.<br/><em>Note: Because the gain of amplification in 3 Image Intensifiers are controlled independently, we can acquire images of almost the same intensities in 3 cameras.</em> </li><li>Adjust the angles of dichroic millers and reflection millers of the custom-made spectroscopic unit to correct the XY positions of 3 camerasâ images.<br/></li></ol><br/><strong>Check and correct the misalignment in XYZ positions in 3 cameras by using bleed-through signals of fluorescence proteins using Volocity</strong><br/><ol class="npwtwo"><li>Prepare yeast cells expressing GFP (<em>e.g.</em>, yeast cells expressing Sec13-GFP which show the dotted signals around the peripheral ER and nuclear membrane).<br/></li><li>Excite GFP with the 473 nm laser at high power and obtain emission signals and bleed-through signals of GFP simultaneously with 2 cameras for green and red fluorescence by adjusting the amplification gain of 2 Image Intensifiers. Check the misalignment in XYZ positions of emission signals and bleed-through signals from GFP dots in 2 cameras dedicated for green and red fluorescence.<br/></li><li>In the same way, obtain emission signals and bleed-through signals of red fluorescence (<em>e.g.</em>, yeast cells expressing Sec13-mCherry) with 2 cameras dedicated for red and infrared fluorescence and then check the misalignment.<br/></li><li>Select âRegistration Correctionâ from Create New in Actions menu to create new registration correction parameters using the misalignment in XYZ positions in 3 channels.<br/></li><li>Select âCorrect Registrationâ from Tools menu and correct the misalignments in XYZ positions in 3 channels using the registration correction parameters.</li></ol></li></ol><p class="pptt" id="biaoti35394">### Recipes</p><ol class="npwtwok"><li>MCD agar plate<br/>0.67% yeast nitrogen base without amino acids<br/>0.5% casamino acids<br/>2% glucose<br/>2% agar<br/><ol class="npwthree"><li>Mix Yeast Nitrogen Base 3.4 g, Vitamin assay casamino acid 2.5 g, and Agar 10 g in 450 ml of distilled water<br/></li><li>Autoclave at 120 Â°C for 20 min<br/></li><li>Add 50 ml of 20% Glucose and 5 ml of appropriate amino acid supplements<br/></li><li>Pour the medium into a cell culture dish (approximately 20 ml/dish)<br/></li><li>Solidify the plates for 2-3 days before use and store at 4 Â°C<br/></li></ol></li><li>MCD media<br/>0.67% yeast nitrogen base without amino acids<br/>0.5% casamino acids<br/>2% glucose<br/><ol class="npwthree"><li>Mix Yeast Nitrogen Base 3.4 g and Vitamin assay casamino acid 2.5 g in 450 ml of distilled water<br/></li><li>Add 50 ml of 20% Glucose and 5 ml of appropriate amino acid supplements<br/></li><li>Store at room temperature</li></ol></li></ol><p class="pptt" id="biaoti35395">### Acknowledgments</p><p class="ppzz">This work has been supported by JSPS grants to the authors (JP25221103, JP17H06420, and JP18H05275 to K.K and A.N). The protocol has been adapted from Kurokawa <em>et al.</em> (2019).</p><p class="pptt" id="biaoti35396">### Competing interests</p><p class="ppzz">The authors declare no competing interests.</p><p class="pptt" id="biaoti35397">### References</p><ol class="npwtwok"><li>Bonfanti, L., Mironov, A. A., Jr., Martinez-Menarguez, J. A., Martella, O., Fusella, A., Baldassarre, M., Buccione, R., Geuze, H. J., Mironov, A. A. and Luini, A. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9875853" target="_blank">Procollagen traverses the Golgi stack without leaving the lumen of cisternae: evidence for cisternal maturation.</a> <em>Cell</em> 95(7): 993-1003. </li><li>Emr, S., Glick, B. S., Linstedt, A. D., Lippincott-Schwartz, J., Luini, A., Malhotra, V., Marsh, B. J., Nakano, A., Pfeffer, S. R., Rabouille, C., Rothman, J. E., Warren, G. and Wieland, F. T. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19948493" target="_blank">Journeys through the Golgi--taking stock in a new era.</a> <em>J Cell Biol</em> 187(4): 449-453.<br/></li><li>Fujii, S., Kurokawa, K., Inaba, R., Hiramatsu, N., Tago, T., Nakamura, Y., Nakano, A., Satoh, T. and Satoh, A. K. (2020). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31974113" target="_blank">Recycling endosomes attach to the <em>trans</em>-side of Golgi stacks in <em>Drosophila</em> and mammalian cells.</a> <em>J Cell Sci</em> 133(4).</li><li>Glick, B. S. and Luini, A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21875986" target="_blank">Models for Golgi traffic: a critical assessment.</a> <em>Cold Spring Harb Perspect Biol</em> 3(11): a005215. </li><li>Glick, B. S. and Nakano, A. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19575639" target="_blank">Membrane traffic within the Golgi apparatus.</a> <em>Annu Rev Cell Dev Biol</em> 25: 113-132. </li><li>Griffiths, G. and Simons, K. (1986). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2945253" target="_blank">The <em>trans</em> Golgi network: sorting at the exit site of the Golgi complex.</a> <em>Science</em> 234(4775): 438-443. </li><li>Ishii, A., Kurokawa, K., Hotta, M., Yoshizaki, S., Kurita, M., Nakano, A., and Kimura,Y. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31616012" target="_blank">Role of Atg8 in the regulation of vacuolar membrane invagination.</a> <em>Sci Rep</em> 9(1): 14828.</li><li>Ishii, M., Suda, Y., Kurokawa, K. and Nakano, A. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27445311" target="_blank">COPI is essential for Golgi cisternal maturation and dynamics.</a> <em>J Cell Sci</em> 129(17): 3251-3261.</li><li>Ito, Y., Uemura, T., Shoda, K., Fujimoto, M., Ueda, T. and Nakano, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22740633" target="_blank">cis-Golgi proteins accumulate near the ER exit sites and act as the scaffold for Golgi regeneration after brefeldin A treatment in tobacco BY-2 cells.</a> <em>Mol Biol Cell</em> 23(16): 3203-3214.</li><li>Ito, Y., Uemura, T., and Nakano, A. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28839076" target="_blank">The Golgi entry core compartment functions as a COPII-independent scaffold for ER-to-Golgi transport in plant cells.</a> <em>J Cell Sci</em> 131(2)</li><li>Iwai, M., Yokono, M., Kurokawa, K., Ichihara, A. and Nakano, A. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27416900" target="_blank">Live-cell visualization of excitation energy dynamics in chloroplast thylakoid structures.</a> <em>Sci Rep</em> 6: 29940.</li><li>Kurokawa, K., Ishii, M., Suda, Y., Ichihara, A. and Nakano, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24295310" target="_blank">Live cell visualization of Golgi membrane dynamics by super-resolution confocal live imaging microscopy.</a> <em>Methods Cell Biol</em> 118: 235-242. </li><li>Kurokawa, K., Okamoto, M. and Nakano, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24728174" target="_blank">Contact of cis-Golgi with ER exit sites executes cargo capture and delivery from the ER.</a> <em>Nat Commun</em> 5: 3653.</li><li>Kurokawa, K., Osakada, H., Kojidani, T., Waga, M., Suda, Y., Asakawa, H., Haraguchi, T. and Nakano, A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30858192" target="_blank">Visualization of secretory cargo transport within the Golgi apparatus.</a> <em>J Cell Biol</em> 218(5): 1602-1618. </li><li>Kurokawa, K., Suda, Y. and Nakano, A. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27432890" target="_blank">Sar1 localizes at the rims of COPII-coated membranes <em>in vivo</em>.</a> <em>J Cell Sci</em> 129(17): 3231-3237.</li><li>Lanoix, J., Ouwendijk, J., Stark, A., Szafer, E., Cassel, D., Dejgaard, K., Weiss, M. and Nilsson, T. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11748249" target="_blank">Sorting of Golgi resident proteins into different subpopulations of COPI vesicles: a role for ArfGAP1.</a> <em>J Cell Biol</em> 155(7): 1199-1212.</li><li>Losev, E., Reinke, C. A., Jellen, J., Strongin, D. E., Bevis, B. J. and Glick, B. S. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16699524" target="_blank">Golgi maturation visualized in living yeast.</a> <em>Nature</em> 441(7096): 1002-1006.</li><li>Maeda, M., Kurokawa, K., Katada, T., Nakano, A. and Saito, K. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31089171" target="_blank">COPII proteins exhibit distinct subdomains within each ER exit site for executing their functions.</a> <em>Sci Rep</em> 9(1): 7346.</li><li>Martinez-Menarguez, J. A., Prekeris, R., Oorschot, V. M., Scheller, R., Slot, J. W., Geuze, H. J. and Klumperman, J. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11748250" target="_blank">Peri-Golgi vesicles contain retrograde but not anterograde proteins consistent with the cisternal progression model of intra-Golgi transport.</a> <em>J Cell Biol</em> 155(7): 1213-1224. </li><li>Matsuura-Tokita, K., Takeuchi, M., Ichihara, A., Mikuriya, K. and Nakano, A. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16699523" target="_blank">Live imaging of yeast Golgi cisternal maturation.</a> <em>Nature</em> 441(7096): 1007-1010.</li><li>Mironov, A. A., Beznoussenko, G. V., Nicoziani, P., Martella, O., Trucco, A., Kweon, H. S., Di Giandomenico, D., Polishchuk, R. S., Fusella, A., Lupetti, P., Berger, E. G., Geerts, W. J., Koster, A. J., Burger, K. N. and Luini, A. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11756473" target="_blank">Small cargo proteins and large aggregates can traverse the Golgi by a common mechanism without leaving the lumen of cisternae.</a> <em>J Cell Biol</em> 155(7): 1225-1238.</li><li>Nakano, A. and Luini, A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20605430" target="_blank">Passage through the Golgi.</a> <em>Curr Opin Cell Biol</em> 22(4): 471-478.</li><li>Okamoto, M., Kurokawa, K., Matsuura-Tokita, K., Saito, C., Hirata, R. and Nakano, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22467862" target="_blank">High-curvature domains of the ER are important for the organization of ER exit sites in Saccharomyces cerevisiae.</a> <em>J Cell Sci</em> 125(Pt 14): 3412-3420.</li><li>Pfeffer, S. R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21045128" target="_blank">How the Golgi works: a cisternal progenitor model.</a> <em>Proc Natl Acad Sci U S A</em> 107(46): 19614-19618. </li><li>Rivera-Molina, F. E. and Novick, P. J. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19666511" target="_blank">A Rab GAP cascade defines the boundary between two Rab GTPases on the secretory pathway.</a> <em>Proc Natl Acad Sci U S A</em> 106(34): 14408-14413. </li><li>Schermelleh, L., Heintzmann, R. and Leonhardt, H. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20643879" target="_blank">A guide to super-resolution fluorescence microscopy.</a> <em>J Cell Biol</em> 190(2): 165-175.</li><li>Suda, Y., Kurokawa, K., Hirata, R. and Nakano, A. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24194547" target="_blank">Rab GAP cascade regulates dynamics of Ypt6 in the Golgi traffic.</a> <em>Proc Natl Acad Sci U S A</em> 110(47): 18976-18981.</li><li>Tojima, T., Suda, Y., Ishii, M., Kurokawa, K. and Nakano, A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31289195" target="_blank">Spatiotemporal dissection of the trans-Golgi network in budding yeast.</a> <em>J Cell Sci</em> 132(15).</li><li>Uemura, T., Suda, Y., Ueda, T. and Nakano, A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24443496" target="_blank">Dynamic behavior of the trans-golgi network in root tissues of Arabidopsis revealed by super-resolution live imaging.</a> <em>Plant Cell Physiol</em> 55(4): 694-703. <br/></li></ol>