<body>
            <h1>Nonenzymatic RNA-templated Synthesis of N3â²âP5â² Phosphoramidate DNA</h1>
            <p><strong>Authors:</strong> Derek K. O'Flaherty, Lijun Zhou, Jack W. Szostak</p>
            <h2>Abstract</h2>
            <p>The RNA world hypothesis describes a scenario where early life forms relied on RNA to govern both inheritance and catalyze useful chemical reactions. Prior to the emergence of enzymes capable of replicating the RNA genome, a nonenzymatic replication process would have been necessary to initiate Darwinian Evolution. However, the one-pot nonenzymatic RNA chemical copying of templates with mixed-sequences is insufficient to generate strand products long enough to encode useful function. The use of alternate (RNA-like) genetic polymers may overcome hurdles associated with RNA copying, and further our understanding of nonenzymatic copying chemistry. This protocol describes the nonenzymatic copying of RNA templates into N3â²âP5â² phosphoramidate DNA (3â²-NP-DNA). We describe, in detail, the synthesis of 3â²-amino-2â²,3â²-dideoxyribonucleotide monomers activated with 2-aminoimidazole (3â²-NH<sub>2</sub>-2AIpddN), and their use in template-directed polymerization.</p>
            <p><strong>Keywords:</strong> Nonenzymatic template-directed synthesis, Origins of life, N3â²âP5â² phosphoramidate DNA, 3â²-amino-2â², 3â²-dideoxyribonucleotide 5â²-phosphoroimidazolide</p>
            <h2>Background</h2><p>Primitive life forms could have consisted of a genome capable of replication and function, encapsulated within a spatially-defined compartment (Szostak <em>et al.</em>, 2001; Szostak, 2012; Blain and Szostak, 2014). Prior to the emergence of enzymes capable of replicating the RNA genome, a nonenzymatic replication process would have been necessary to initiate Darwinian Evolution (Szostak, 2012 and 2017). RNA is a logical candidate for the primordial genetic polymer given its ability to catalyze critical chemical reactions such as protein synthesis, and to act as a genetic information carrier in modern biological contexts.<br/> <br/>Despite major breakthroughs toward demonstrating nonenzymatic RNA replication over the past 5 decades, there remain major hurdles to overcome before achieving this seemingly overarching problem in origins of life research (Szostak, 2012 and 2017). Pioneering work by Leslie Orgel showed that high-energy nucleoside 5'-phosphoroimidazolides are capable of spontaneously polymerizing in a template-directed fashion (Szostak, 2017). Our group and others have used nonenzymatic primer extension by activated mononucleotides as a model for the study of prebiotic genome copying (Figure 1A). Recently, we showed that short activated oligoribonucleotides that can bind to an RNA template segment downstream of an activated ribonucleotide can catalyze the reaction on the monomer with the RNA primer (Prywes <em>et al.</em>, 2016). In addition, by screening a small library of leaving groups, Li <em>et al.</em> discovered that 2-aminoimidazole (2AI) is a superior nucleotide activating group (Figure 1) (Li <em>et al.</em>, 2017). These two improvements allowed copying of mixed-sequence RNA templates in one-pot for the first time. Nevertheless, the template-directed chemical copying into complementary RNA sequences remains limited to only seven nucleotides in solution (Li <em>et al.</em>, 2017). As a result, we hypothesize that other factors can further optimize primer extension reaction, and that it may be possible to discover prebiotically plausible pathways to copy RNAs of sufficient length to encode useful biological function (typically 30-50 nucleotides in length). Toward this end, our group and others have explored alternative genetic materials, which may provide insights into how longer RNA sequences could be propagated nonenzymatically (Rojas StÃ¼tz and Richert, 2001; Zhang <em>et al.</em>, 2013a and 2013b; HÃ¤nle and Richert, 2018). We chose N3â²âP5â² phosphoramidate DNA (3â²-NP-DNA, Figure 1B) as a potential proxy for RNA for a number of reasons. First, the 3â²-NP-DNA biopolymer has a strong structural resemblance to the A-form RNA duplex (double stranded RNA) (Tereshko <em>et al.</em>, 1998). Moreover, it hybridizes with RNA and DNA with higher thermal stability than the homo-duplexes (Gryaznov <em>et al.</em>, 1995; Gryaznov, 1997), and 3â²-amino-2â²,3â²-dideoxyribonucleotides (Figure 1B, the monomeric units that polymerize into 3â²-NP-DNA) adopt a C3â²-endo conformation in solution similar to ribonucleotides (Ding <em>et al.</em>, 1998; Zhang <em>et al.</em>, 2012). The latter feature is known to speed up nonenzymatic primer extension reactions (Zhang <em>et al.</em>, 2012).<br/><br/><img alt="" class="layerphoto" height="100" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200905/20200905020011_9282.jpg" width="138"/><br/><strong>Figure 1. Nucleic acid primer extension model.</strong> A. Schematic representation of primer extension experiments. Activated mononucleotides are indicated above the arrow and participate in nonenzymatic primer extension. B. Chemical structure of ribonucleotides and 3â²-amino-2â²,3â²-dideoxyribonucleotides (3â²-NP) activated with 2-aminoimidazole (2AI) (3â²-NH<sub>2</sub>-2AIpddN, where N signifies the identity of the nucleobase). The 2AI leaving groups are highlighted in blue, and the furanose moieties are highlighted in red to illustrate the chemical differences between ribose and 3â²-amino-2â²,3â²-dideoxyribose. Note that conventionally, thymine is used for the 3â²-NP system (as opposed to uracil, which is found in RNA).<br/><br/>There are few detailed protocols for performing nonenzymatic primer extension, including the preparation of activated 5â²-phosphorimidazolides (<em>e.g.</em>, Figure 1B). Recently, our group contributed a manuscript, in video format, concerning the preparation and use of fatty acid-containing vesicles, with a section detailing the procedures for nonenzymatic RNA copying within vesicles (Jin <em>et al.</em>, 2018). The current protocol describes the procedures for primer extension experiments in much more detail, including the synthesis of the activated mononucleotides, synthesis of oligomers, and primer extension reactions. This protocol may be used in the origins of life field and for research concerning the bottom-up approach to building an artificial cell.</p>
        </body>

<p class="pptt" id="biaoti35454">## Materials and Reagents</p><ol class="npwtwok"><li>Disposable Norm-ject syringe (Thermo Fisher Scientific, 1 ml, 5 ml, 10 ml, 20 ml)</li><li>Disposable needle 18G (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14-840-96 </a> )</li><li>Syringe filter (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> SLGVM33RS </a> )</li><li>pH-indicator strips pH 0-14 Universal indicator (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 1095350001 </a> )</li><li>1.5 ml Microcentrifuge tubes (Bag of 500) (conical bottom, Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 1000-0785 </a> )</li><li>2.0 ml Fisherbrand<sup>TM</sup> Free-Standing Microcentrifuge Tubes with Screw Caps (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 02-682-558 </a> )</li><li>Filter paper (Whatman, to fit Buchner funnel)</li><li>3'-Amino-2',3'-dideoxyguanosine, 99% (Alfa Aesar, catalog number: <a class="blackNoUnder" target="_blank"> J65326 </a> )</li><li>3'-Amino-2',3'-dideoxythymidine, 99% (Alfa Aesar, catalog number: <a class="blackNoUnder" target="_blank"> J64342 </a> )</li><li>3'-Amino-2',3'-dideoxyadenosine, 98% (Alfa Aesar, catalog number: <a class="blackNoUnder" target="_blank"> J65090 </a> )</li><li>3'-Amino-2',3'-dideoxycytidine (Biosynth Carbosynth, catalog number: <a class="blackNoUnder" target="_blank"> NA05183 </a> )</li><li>3'-(TFA)amino-2',3'-dideoxycytidine 5'-CED phosphoramidite (ChemGenes, catalog number: <a class="blackNoUnder" target="_blank"> ANP-1242 </a> )</li><li>Reverse RNA phosphoramidites:<br/>2'-TBDMS-3'-DMT-rA (N-bz) (ChemGenes, catalog number: <a class="blackNoUnder" target="_blank"> AN-3401 </a> )<br/>2'-TBDMS-3'-DMT-rC (N-acetyl) (ChemGenes, catalog number: <a class="blackNoUnder" target="_blank"> AN-3405 </a> )<br/>2'-TBDMS-3'-DMT-rG (N-iPr PAC) (ChemGenes, catalog number: <a class="blackNoUnder" target="_blank"> AN-3406 </a> )<br/>2'-TBDMS-3'-DMT-rU (ChemGenes, catalog number: <a class="blackNoUnder" target="_blank"> AN-3404 </a> )</li><li>3'-(6-FAM) CPG (Glen Research, catalog number: <a class="blackNoUnder" target="_blank"> 20-2961-41E </a> for Expedite<sup>TM</sup> 8900 Nucleic Acid Synthesis System)</li><li>9-Fluorenylmethyl N-succinimidyl carbonate, Fmoc-OSu (Combi-Blocks, catalog number: <a class="blackNoUnder" target="_blank"> ST-5099 </a> )</li><li>Pyridine (Py) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 270970-100ML </a> )</li><li>Dichloromethane (DCM) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 320269-4L </a> )</li><li>Diethylether (Et<sub>2</sub>O) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 296082-2.5L </a> )</li><li>Sodium perchlorate (NaClO<sub>4</sub>) (Millipore Sigma, catalogue number: 410241-500G)</li><li>Acetone (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 179124-4L </a> )</li><li>Molecular Sieves 3A (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AC197255000 </a> )</li><li>N,N-Dimethylformamide (DMF) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 227056-100ML </a> )</li><li>Dimethylsulfoxide, anhydrous (DMSO) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 276855-100ML </a> )</li><li>Dimethyl sulfoxide-d<sub>6</sub> (d<sub>6</sub>-DMSO) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 156914-25G </a> ) </li><li>Trimethylphosphate (TMP) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AC157970500 </a> )</li><li>N,N-Diisopropylethylamine (DIPEA) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 387649-100ML </a> )</li><li>Triethylamine (TEA) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 8083521000 </a> )</li><li>Acetonitrile, anhydrous (MeCN) (Glen Research, catalog number: <a class="blackNoUnder" target="_blank"> 40-4050-50 </a> )</li><li>Acetonitrile, DNA synthesis grade (MeCN) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A21 </a> )</li><li>Phosphorus (V) oxychloride, 99% (POCl<sub>3</sub>) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AA1052530 </a> )</li><li>Dry ice (solid carbon dioxide)</li><li>2-Amino-1H-imidazole, HCl (2AIÂ·HCl) (Combi-Blocks, catalog number: <a class="blackNoUnder" target="_blank"> SS-6610 </a> )</li><li>Triphenylphosphine (TPP) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 8082700250 </a> )</li><li>2,2'-Dipyridyldisulfide (DPDS) (Combi-Blocks, catalog number: <a class="blackNoUnder" target="_blank"> QA-4326 </a> )</li><li>Piperidine (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 411027-100ML </a> )</li><li>Sodium hydroxide (NaOH) (50% solution in water) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 415413-100ML </a> )</li><li>Triethylamine trihydrofluoride (NEt<sub>3</sub>Â·3HF) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 344648-25G </a> )</li><li>Sodium acetate (NaOAc) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> S2889-250G </a> )</li><li>EDTA solution (500 mM, pH 8, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9260G </a> )</li><li>Xylene cyanol FF (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> X4126-10G </a> )</li><li>Bromophenol blue (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> B0126-25G </a> )</li><li>Tris-Borate-EDTA buffer (TBE) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> T4415-1L </a> )</li><li>Urea (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> U5378 </a> )</li><li>Ethanol 200 proof (EtOH) (Decon Labs, catalog number: <a class="blackNoUnder" target="_blank"> 2716 </a> )</li><li>Methanol (MeOH) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A412 </a> )</li><li>Chloroform (CHCl<sub>3</sub>) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 472476-2.5L </a> )</li><li>Sodium chloride (NaCl) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> S7653-250G </a> )</li><li>Magnesium chloride (MgCl<sub>2</sub>) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9530G </a> )</li><li>NH<sub>4</sub>OH solution (28% NH<sub>3</sub> in water, Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 338818-100ML </a> )</li><li>MeNH<sub>2</sub> solution (40% wt. in water, Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 426466-100ML </a> )</li><li>SequaGel (UreaGel Concentrate) (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-830 </a> ) to make 19:1 (w/w) acrylamide:bis-acrylamide</li><li>SequaGel (UreaGel Diluent) (National Diagnostics, catalog number: <a class="blackNoUnder" target="_blank"> EC-840 </a> ) to make 19:1 (w/w) acrylamide:bis-acrylamide</li><li>N,N,Nâ²,Nâ²-Tetramethylethylenediamine (TEMED) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> T9281-50ML </a> )</li><li>1 M Na<sup>+</sup>-HEPES pH 8 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AAJ63578AK </a> )</li><li>Ammonium persulfate (APS) (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A3678-100G </a> )</li><li>Sep-Pak C18 Plus Short Cartridge, 360 mg (Waters, catalog number: <a class="blackNoUnder" target="_blank"> WAT020515 </a> )</li><li>RNA template (Integrated DNA technologies, HPLC purified)</li><li>RNA primer containing 3'-NH<sub>2</sub> terminal (synthesized in house, see Procedure D)<br/></li></ol><p class="pptt" id="biaoti35455">### Equipment</p><ol class="npwtwok"><li>Analytical balance (<em>e.g.</em>, Sartorius<sup>TM</sup> Secura<sup>TM</sup> Analytical Weighing Balances, catalog number: <a class="blackNoUnder" target="_blank"> 14-557-401 </a> )</li><li>Round bottom flask (25 ml, 50 ml, 100 ml, 500 ml, Chemglass Life Sciences)</li><li>Septum stoppers for round bottom flasks (Chemglass Life Sciences)</li><li>Vacuum filtration setup (Buchner funnel, Erlenmeyer filter flask, filter adaptors (neoprene), Chemglass Life Sciences)</li><li>Water bath (<em>e.g.</em>, large Erlenmeyer flask, Chemglass Life Science)</li><li>Single-channel pipettor (1,000 Î¼l, 200 Î¼l, 20 Î¼l, 2 Î¼l, Thermo Fisher Scientific)</li><li>Pipet Controller (Thermo Fisher Scientific, <em>e.g.</em>, BrandTech<sup>TM</sup> accu-jet<sup>TM</sup> pro, catalog number: <a class="blackNoUnder" target="_blank"> 03-840-311 </a> )</li><li>Stir bar (<em>e.g.</em>, 19 mm (3/4â) x 19 mm (3/4â) x 9.5 mm (3/8â), VWR, catalog number: <a class="blackNoUnder" target="_blank"> 58947-822 </a> )</li><li>Rotary evaporator (Heidolph Laborota 4001 efficient)</li><li>Low-resolution mass spectrometry (Bruker Esquire 6000)</li><li>Combi Flash Rf-200 automated flash chromatography system (Teledyne Isco, part number: 68523006)</li><li>RediSep Rf columns (Teledyne Isco, catalog numbers: <a class="blackNoUnder" target="_blank"> 69-2203-340 </a> (40 g) and 69-2203-380 (80 g))</li><li>RediSep Rf Gold C18 Column (Teledyne Isco, catalog number: <a class="blackNoUnder" target="_blank"> 69-2203-336 </a> )</li><li>Benchtop centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5910R </a> )</li><li>Razor blade (Fisherbrand<sup>TM</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 12-640 </a> )</li><li>Speedvac concentrator (Thermo Scientific Savant SPD111V)</li><li>Digital block heater (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 12621-088 </a> )</li><li>NanoDrop 2000c spectrophotometer (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> DN-2000C </a> )</li><li>Nucleic acid automated synthesizer Expedite 8909</li><li>Thermal cycler (Bio-Rad, T100<sup>TM</sup> Thermal Cycler)</li><li>Electrophoresis power supply (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 164-5056 </a> )</li><li>0.75 mm thick, 1.5 mm thick spacers (Millipore Sigma, catalog numbers: <a class="blackNoUnder" target="_blank"> EP1418-2EA </a> , <a class="blackNoUnder" target="_blank"> EP1420-2EA </a> , respectively)</li><li>30-well VS20 comb (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> EP1428 </a> )</li><li>Preparatory comb (Apogee Electrophoresis, catalog number: <a class="blackNoUnder" target="_blank"> 21076021 </a> )</li><li>Vertical electrophoresis system glass plates (Apogee Electrophoresis, catalog number 11074010)</li><li>Electrophoresis glass plates (15 x 17 cm vertical gel plates, Apogee Electrophoresis)</li><li>Amersham Typhoon RGB Biomolecular Imager (GE Healthcare Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 29187193 </a> )</li><li>Nuclear Magnetic Resonance (NMR, 400MHz Variant)<br/></li></ol><p class="pptt" id="biaoti35456">### Software</p><ol class="npwtwok"><li>ChemDraw (PerkinElmer)</li><li>ImageQuant<sup>TM</sup> (GE Healthcare Life Sciences)<br/></li></ol><p class="pptt" id="biaoti35457">### Procedure</p><p class="ppzz">The preparation of activated 3'-amino monomers is accomplished via four chemical transformations from commercially available starting materials. The nucleophilic amine is first protected using a fluorenylmethyloxycarbonyl (Fmoc) group followed by a chemical phosphorylation step to generate the nucleoside 5'-monophosphates. The installation of the 2-aminoimidazoyl group at the 5'-monophosphate, followed by removal of the Fmoc protecting group, furnishes the final activated monomers. This process is repeated for each of the four monomers used (3'-amino-2',3'-dideoxynucleoside 5â²-phosphoro-2-aminoimidazolide (3â²-NH<sub>2</sub>-2AIpddN) where the nucleoside (N) is adenosine (A), guanosine (G), cytidine (C), and thymidine (T)). The step-by-step protocol is outlined for 3'-amino-3'-dideoxythymidine (Figure 2) and differences with respect to the other three monomers are highlighted. It should be noted that all chemical reactions should be carried out in a fumehood designated for chemical reactions.<br/><br/><img alt="" class="layerphoto" height="44" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902093236_7128.png" width="300"/><br/><strong>Figure 2. Synthesis of 3'-amino-3'-dideoxythymidine activated monomer (3â²-NH<sub>2</sub>-2AIpddT)</strong> </p><br/><ol class="npwone"><li>Preparation of Fmoc protected 3'-amino-2',3'-dideoxynucleoside (Figure 2, second chemical structure from the left)<br/><ol class="npwtwo"><li>Weigh out 1.0 g (4.1 mmol) of 3'-amino-3'-deoxythymidine using an analytical balance, and transfer to a 100 ml round bottom flask equipped with a stir bar.<br/></li><li>Add 12 ml of pyridine using a 20 ml disposable syringe equipped with a disposable needle.<br/></li><li>Place the flask on top of a stir plate and initiate stirring for 2 min at 25 Â°C.<br/></li><li>Weigh out 1.7 g (5.0 mmol) of Fmoc-OSu using an analytical balance, and transfer to the flask containing the nucleoside solution.<br/></li><li>Allow the reaction to stir for 1 h at 25 Â°C.</li><li>Remove the solvent using a rotary evaporator.<br/><em>Note: The water bath was slowly ramped up from room temperature until condensation could be observed on the rotary evaporator cold finger.</em><br/></li><li>Co-evaporate remnant pyridine with 20 ml MeCN using a rotary evaporator.<br/><em>Note: The water bath was slowly ramped up from room temperature until condensation could be observed on the rotary evaporator cold finger.</em> </li><li>Repeat Step A7 twice to afford a yellow gum.</li><li>Redissolve the gum using DCM (approximately 10 ml).<br/><em>Note: If the redissolution is difficult, MeOH may be titrated in dropwise under stirring until full dissolution is achieved.</em> </li><li>Purify the product using automated normal-phase flash chromatography, equipped with a RediSep<sup>Â®</sup> Rf Normal Phase Silica column (40 g), using gradient elution between (A) DCM and (B) MeOH. The column gradient was as follows: 0% B for the first 2 column volumes (CVs), 0% to 20% B over 11 CVs, and 20% B for 2 CVs, with a flow rate of 40 ml/min.</li><li>Fractions containing product can be identified using low-resolution mass spectrometry. Alternatively, thin layer chromatography may be used (<em>e.g.</em>, using a 1:9 (v/v) MeOH:DCM solvent system).</li><li>Pool fractions containing purified product.</li><li>Remove solvent using rotary evaporation.<br/><em>Note: The water bath was slowly ramped up from room temperature until condensation could be observed on the rotary evaporator cold finger.</em> </li><li>Allow the nucleoside to further dry on high vacuum (&gt; 8 h at 25 Â°C).<br/></li></ol><br/><strong>Considerations for the other nucleosides:</strong><br/><u>3'-amino-2',3'-dideoxycytidine</u>: A similar procedure can be used for 3'-amino-2',3'-dideoxycytidine (500 mg) with a few exceptions. The reaction solvent used was DMF (22 ml). The solvent was removed using a rotary evaporator equipped with a heat bath. The column gradient (A: DCM, B: MeOH) was as follows: 0% B for the first 2 CVs, 0% to 40% B over 11 CVs, and 20% B for 2 CVs, with a flow rate of 40 ml/min. Further dry the product under high vacuum (&gt; 8 h at 25 Â°C).<br/><u>3'-amino-2',3'-dideoxyadenosine</u>: Combine the nucleoside (1 g) with pyridine (10 ml) and water (5 ml), followed by Fmoc-OSu (1.6 g). Stir vigorously for 1h. Remove most of the solvent using rotary evaporation (until approximately 3 ml). Precipitate the product by adding 100 ml of cold (0 Â°C) 1:1 (v/v) MeCN:CHCl<sub>3</sub> solution. Isolate the product using vacuum filtration (equipment specified in the equipment section). Further dry the product under high vacuum (&gt; 16 h at 25 Â°C).<br/><u>3'-amino-2',3'-dideoxyguanosine</u>: Combine the nucleoside (0.4 g) with pyridine (10 ml) and water (4 ml), followed by Fmoc-OSu (0.6 g). Stir vigorously for 1h. Precipitate the product by adding 500 ml of cold (0 Â°C) 1:1 (v/v) MeCN:CHCl<sub>3</sub> solution. Isolate the product using vacuum filtration (equipment specified in the equipment section). Further dry the product under high vacuum (&gt; 16 h at 25 Â°C).<br/><br/></li><li>Preparation of Fmoc protected 3'-amino-2',3'-dideoxynucleoside-5'-monophosphate (similar procedure for all 4 nucleosides) (Figure 2, third chemical structure from the left)<ol class="npwtwo"><li>Weigh out 230 mg (0.5 mmol) of 3'-NHFmoc-3'-deoxythymidine using an analytical balance, and transfer to a 25 ml round bottom flask equipped with a stir bar.</li><li>Add 5 ml of trimethylphosphate using a 10 ml disposable syringe equipped with a disposable needle.<br/><em>Note: This results in a 0.1 M solution of nucleoside, which is important for this reaction.</em> </li><li>Stopper the flask using a rubber stopper.</li><li>Place the flask containing the solution in an ice-water bath and place on top of a stir plate.</li><li>Set the stir plate to generate vigorous sample stirring in the ice-water bath for 5-10 min.</li><li>Add 187 Âµl (2 mmol) of phosphorus (V) oxychloride (POCl<sub>3</sub>) using a single-channel pipettor (200 Âµl capacity).</li><li>Allow to stir vigorously for 10 min in the ice-water bath.</li><li>Add 44 Âµl (0.25 mmol) of diisopropylethylamine (DIPEA) dropwise using a single-channel pipettor (200 Âµl capacity).<br/><em>Note: Reaction fuming may be observed here.</em> </li><li>Allow to stir vigorously for 20 min in ice-water bath.</li><li>Repeat Steps B8 and B9 three additional times for a total addition of 1 mmol of DIPEA.</li><li>Using an single-channel pipettor, quench ~1 Âµl of the reaction into 500 Âµl of deionized water in a microcentrifuge tube (Note that the precipitation may be observed during quenching). Mix thoroughly and, if precipitation is observed, add MeCN dropwise until the precipitate is solubilized. Analyze the quenched material using low-resolution mass spectrometry in negative ion mode (<em>Note: Quenched solutions should be filtered using a disposable syringe fitted with a syringe filter, or centrifuged (</em><em>e.g.</em><em>, 20,000 x g at room temperature) to avoid injecting particulates into the mass spectrometer</em>). Signal corresponding to starting material should be absent in the mass spectrum (note that a chloride adduct may be observed). Alternatively, thin layer chromatography may be used to monitor the reaction progress if a mass spectrometer is unavailable.</li><li>In the event that starting material is still present, add 47 Âµl (0.5 mmol) of POCl<sub>3</sub> to the reaction mixture, followed by 44 Âµl (0.25 mmol) of DIPEA. Repeat reaction analysis (Step B11).</li><li>Once the reaction is complete, add the reaction mixture to a 250 ml round bottom flash containing 50 ml of a triethylammonium bicarbonate (1 M) solution with vigorous stirring at 0 Â°C. This will produce fuming and perhaps foaming.<br/><em>Note: Product precipitation is to be expected.</em> </li><li>Allow to warm to room temperature under vigorous stirring (approximately 30 min).</li><li>If precipitation persists after Step B14, slowly titrate MeCN into the quenched reaction until solubilization of the solid material is observed.<br/><em>Note: The solution may remain slightly cloudy, especially in the case of ddG.</em> </li><li>Purify the product using automated reverse-phase flash chromatography, using a RediSep Rf Gold C18 column with gradient elution between (A) aqueous 2 mM TEAB (pH 7.5) and (B) acetonitrile. The column gradient should be between 0% and 70% B over 13 CVs with a flow rate of 40 ml/min.</li><li>Identify fractions containing product by low-resolution mass spectrometry. Alternatively, thin layer chromatography may be used to identify fractions containing product if a mass spectrometer is unavailable.</li><li>Pool fractions containing pure product in a 500 ml round bottom flask.</li><li>Remove the majority of the solvent by rotary evaporation until &lt; 10 ml remains in order to remove most of the MeCN.<br/><em>Caution: Subjecting solutions containing TEAB to low pressure can cause vigorous/rapid foaming/boiling. Initiate the rotary evaporation in an ice water bath and gradually increase the temperature. Allow the solution "bubbling" to stabilize prior to increasing the temperature (this can be time consuming).</em> </li><li>Collect the solution in a clean 50 ml round bottom flask (record the mass of the empty flask prior to the transfer).</li><li>Freeze the solution in a freezer (-20 Â°C) or in a dry ice-acetone bath, and freeze-dry the product under high vacuum at room temperature. Alternatively, if freeze drying is unavailable, rotary evaporate to completion. Co-evaporate the residue three times using anhydrous pyridine (3 x 5 ml). Allow the residue to further dry under high vacuum at room temperature.</li><li>Record the mass of the flask containing the product.<br/><br/></li></ol></li><li>Preparation of 3'-amino-2',3'-dideoxynucleoside-5'-phosphoro-(2-aminoimidazolide) (similar procedure for all 4 nucleosides) (Figure 2, final product, fourth chemical structure from the left)<ol class="npwtwo"><li>Weigh 270 mg of 3'-NHFmoc-3'-dideoxythymidine-5'-monophosphate using an analytical balance and transfer to a dry 50 ml round bottom flask equipped with a stir bar.</li><li>Add 16 ml of anhydrous DMSO using a 20 ml disposable syringe equipped with a disposable needle.</li><li>Weigh 297 mg of 2-aminoimidazole hydrochloride using an analytical balance and transfer to the reaction vessel.</li><li>Initiate stirring.</li><li>Add 0.43 ml of DIPEA using a 1 ml disposable syringe equipped with a disposable needle.</li><li>Weigh 1.3 g of triphenylphosphine (TPP) using an analytical balance and transfer to the reaction vessel.</li><li>Weigh 910 mg of 2-2'-dipyridyldisulfide (DPDS) using an analytical balance and transfer to the reaction vessel.</li><li>Allow the reaction to stir for 1 h at 25 Â°C.</li><li>Monitor the reaction progress using <sup>31</sup>P NMR by sampling 450 Âµl of the reaction with 150 Âµl of d<sub>6</sub>-DMSO (add 1 Âµl of PO(OEt)<sub>3</sub> as a reference standard).<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>The starting material signal is near 1 ppm, PO(OEt)<sub>3</sub> near -3 ppm, TPP is near -6 ppm, triphenylphosphine oxide (TPPO) is near 27 ppm, and product should be near -10.5 ppm. </em><em></em><br/></li><li><em>Low-resolution mass spectrometry may also be used to track reaction progress (as in Step B11). </em><em></em><br/></li></ol></li><li>If the reaction is incomplete, add 650 mg triphenylphosphine and 455 mg of DPDS and allow to stir for 30 min. Repeat Step C9.</li><li>Remove the Fmoc protecting group by the addition of DIPEA (300 Âµl) followed by piperidine (960 Âµl). The reaction should be monitored by <sup>31</sup>P NMR (approximately 10-30 min) at 25 Â°C.</li><li>Add the reaction mixture to a pre-chilled (0 Â°C) 500 ml solution of EtO<sub>2</sub>:Acetone 1:1 (v/v) containing 2 ml of a saturated solution of sodium perchlorate (NaClO<sub>4</sub>) in acetone (in a 1 L Bucher flask).</li><li>Allow the precipitate to settle in an ice bath.</li><li>Remove the majority of the supernatant using a pipette controller.</li><li>Transfer the remaining ~100 ml into two 50 ml conical centrifugal tubes.<br/><em>Note: If more than 100 ml remains from the suction Step C14, repeat the centrifugation step twice.</em> </li><li>Centrifuge the tubes at 3,000 rpm (approximately 4,700 <em>x g</em>) for 3 min using a benchtop centrifuge (at 0 Â°C, if possible).</li><li>Decant and discard the supernatant from the tubes.</li><li>Remove residual solvent for the pelleted material by placing the tubes under high vacuum (using a freeze dryer vacuum vessel if possible).</li><li>Resuspend the dried crude material in 5 ml 1 M TEAB (pH 7.5).</li><li>Purify the product by automated reverse-phase flash chromatography, using a RediSep Rf Gold C18 column with gradient elution between (A) aqueous 2 mM TEAB (pH 7.5) and (B) acetonitrile. The column gradient should be between 0% and 15% B over 13 CVs with a flow rate of 40 ml/min.</li><li>Test fractions containing UV-active material for presence of product by LR-MS, NMR or TLC.</li><li>Pool fractions containing pure product and test the pH using a calibrated pH probe.</li><li>Adjust the pH of the purified stock to 9.25-10 using 1 N NaOH (typically 5-20 Âµl for a ~20 ml of product).<br/><em>Note: Although this step is time-sensitive, the pH should not be "overshot". As such, the titration of base should be done meticulously.</em> </li><li>Take a UV measurement and record the reading at 260 nm.</li><li>Calculate the concentration using the Beer-Lambert law (A = Îµcl, where A is absorbance (a.u.), Îµ is the molar extinction coefficient (M<sup>-1</sup>cm<sup>-1</sup>), and l is the path length (cm)).<br/><em>Note: Molar extinction coefficients can be acquired from the corresponding triphosphates (</em><em>e.g.</em><em>, Maravai LifeSciences-TriLinkTechnologies, 9650 M<sup>-1</sup>cm<sup>-1</sup> for 3'-Amino-ddT triphosphate).</em> </li><li>Aliquot the stock solution in 2 Î¼mol samples in 1.5 ml microcentrifuge tubes.</li><li>Freeze product aliquots in liquid nitrogen.<br/><em>Note: Cryoboxes are convenient for this step.Proper personal protective equipment should be utilized when handling liquid nitrogen.</em> </li><li>Freeze dry samples (preferably at -20 Â°C).</li><li>Store samples at -80 Â°C until ready to use.<br/><br/></li></ol></li><li>Preparation of 5'-FAM-labeled primer with 3'-terminal 3'-amino-2',3'-dideoxy-cytidine (Figure 3, top sequence labeled "3'-NH<sub>2</sub>-RNA primer")<ol class="npwtwo"><li>Prepare reverse RNA phosphoramidite solutions according to manufacturer's specifications (typically 66 mM in anhydrous acetonitrile).</li><li>Prepare the 3'-(TFA)amino-2',3'-dideoxycytidine 5'-CED phosphoramidite solution (typically 66 mM in anhydrous acetonitrile). </li><li>Add dried molecular sieves to each of the solutions in order to ensure and maintain anhydrous conditions.</li><li>Prepare the oligonucleotide using automated solid-phase synthesis according to the manufacturer's specifications on the 3'-(6-FAM) CPG solid support.</li><li>Cleave and deprotect the oligonucleotide according to the specifications of the 3'-(6-FAM) CPG (Glen Research). More-specifically, first treat the CPG with 1 ml aqueous NH<sub>4</sub>OH (28%) for 30 min at room temperature, followed by the addition of an equal volume of aqueous MeNH<sub>2</sub> (40%). Incubate for another 30 min, then remove the solution from the CPG column.</li><li>Transfer the solution to a clean 2 ml screw-cap microcentrifuge tube and seal the tube.</li><li>Heat the solution to 65 Â°C for 10 min using a dry block incubator.</li><li>Allow the solution to cool to room temperature.</li><li>Remove the solvent using a speed-vac concentrator without additional heating.</li><li>Add 100 Î¼l of DMSO to the pellet.</li><li>Add 125 Î¼l of NEt<sub>3</sub>Â·3HF in a fumehood and seal the tube carefully.</li><li>Gently mix using vortex.</li><li>Heat the solution to 65 Â°C for 2.5 h using a dry block incubator. Vortex 1 min after 1h of heating.</li><li>Allow the solution to cool to room temperature. Caution: Do not open the cap when the solution is hot.</li><li>Add 1 ml of isopropanol and 100 Âµl 3 M sodium acetate (pH 5.2) to the solution containing the oligomer to initiate precipitation.</li><li>Store the solution at -20 Â°C for 20 min.</li><li>Centrifuge the sample at 20,000 <em>x g</em> at 4 Â°C for 10 min.</li><li>Carefully remove the supernatant using an single-channel pipettor (200 Î¼l capacity). <br/><em>Note: The supernatant should be retained until recovery of the product is confirmed.</em> </li><li>Resuspend the pellet in 1 ml 75% ethanol in water, then repeat Steps D16-D18.</li><li>Resuspend the pellet in 200 Î¼l of a solution of 8 M aqueous urea. Split the sample in half and store one half of the crude at -80 Â°C.</li><li>Wash 20 x 20 cm gel plates with 80% (v/v) ethanol:water.</li><li>Assemble a preparatory gel apparatus according to manufacturer specifications. A thickness of 1.5 mm is typically sufficient for purification.</li><li>Prepare 75 ml of an 20% solution of 19:1 (w/w) acrylamide:bis-acrylamide in 1x TBE buffer. <br/><em>Note: Acrylamide/bis-acrylamide solutions are toxic and should be handled with proper precautions.</em> </li><li>Initiate the polymerization by the addition of TEMED (30 Î¼l) and freshly prepared 20% (w/v) aqueous ammonium persulfate (150 Î¼l) to the solution reported in Step D23.<br/><em>Note: It is important to have the preparatory gel apparatus completely set up so that the gel solution can be poured into the plate assembly prior to "solidification" of the gel matrix.</em> </li><li>Carefully pour the gel solution into the plate assembly.</li><li>Insert preparatory gel comb at the top of the plate assembly.<br/><em>Note: Avoid the air bubbles; this can typically be accomplished by inserting the comb in a diagonal fashion.</em> </li><li>Allow the gel to polymerize for 1 h.</li><li>Remove the comb and wash the wells thoroughly with 1x TBE.</li><li>Pre-run the gel, according to manufacturer specifications, using 1x TBE as the running buffer (5 W power for 10 min).</li><li>Stop the power.</li><li>Clean the well thoroughly with 1x TBE right before the introduction of the sample, using a 60 ml disposable syringe equipped with a disposable needle.</li><li>Evenly deposit the 100 Î¼l sample (resuspended in 8 M urea) at the base of the well.</li><li>Add indicators (xylene cyanol and bromophenol blue) to the reference well. <br/><em>Note: This well is typically placed near the edge of the gel. The indicators are useful as molecular-weight size markers (</em><em>e.g.</em><em>, the bromophenol blue migrates similarly to an octanucleotide species for this concentration of acrylamide gel).</em> </li><li>Load sample into the gel matrix by applying 5W of power for 15 min.</li><li>Increase the power to 20 W.</li><li>Allow the electrophoretic separation to occur until the blue dye (bromophenol blue) reaches the bottom of the gel.<br/><em>Note: Visual tracking of FAM-containing oligonucleotides is typically possible (yellow species). This is useful in resolving other FAM-containing species in real-time.</em> </li><li>Remove the gel from the running chamber.</li><li>Carefully dislodge the plates from one another.</li><li>Excise the gel band corresponding to product using a square razor blade.<br/><em>Note: If it is unclear which band corresponds to the desired product, excise all potential product bands.</em><br/><em>Caution: Razor blades should be handled with care and properly stored/disposed of immediately after use.</em> </li><li>Transfer the gel band to a new 50 ml conical tube and crush into small gel pieces.<br/><em>Note: If multiple gel bands were excised, place each band in a separate conical tube.</em> </li><li>Add 9 ml of 100 mM sodium acetate and 1 ml of a 500 mM EDTA (pH 8).</li><li>Allow the mixture to gently rock overnight.</li><li>Desalt the oligonucleotide-containing supernatant using reverse-phase Sep-Pak cartridges according to the manufacturer's specification.<br/><em>Note: This is typically carried out with 10 ml syringes and a three way valve.</em> </li><li>Elute the oligomer from the stationary phase using 1:1 (v/v) MeCN:water solution into two 2 ml screw-cap vials.<br/><em>Note: Most of the oligomer will be captured in the first vial.</em> </li><li>Use a speedvac concentrator to dry down the material.</li><li>Resuspend the oligomer in 300 Âµl of deionized water.</li><li>Dilute 1 Âµl of sample into 49 Âµl of water. </li><li>Record the UV absorbance at 260 nm.</li><li>Estimate the concentration using a calculated molar extinction coefficient of the oligomer (<em>e.g.</em>, calculated molar extinction coefficient using the nearest neighbor method) (<em>e.g.</em>, accessible at <a href="https://www.idtdna.com/pages/tools" target="_blank">https://www.idtdna.com/pages/tools</a>).</li><li>Prepare a stock solution of primer at a concentration of 100 ÂµM.</li><li>Analyze the oligomer by liquid chromatography-mass spectrometry.<br/><em>Note: Oligomer purity should exceed 95%. Consider repurifying the material by another means (</em><em>e.g.</em><em>, IEX-HPLC or RP-HPLC) if the purity is less than 95%.</em><br/><br/></li></ol></li><li>Primer extension reaction (example shown in Figure 3).<br/><br/><img alt="" class="layerphoto" height="357" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902093339_1200.png" width="300"/><br/><strong>Figure 3. Chemical copying of a mixed-sequence RNA templates into 3â²-NP-DNA using 3â²-NH<sub>2</sub>-2AIpddNs.</strong> Top: the primer sequence is shown in green, the RNA template in blue, and the stepwise extension of the primer in red. Bottom: PAGE analysis of primer extension time course, carried out using 10 mM activated NP-DNA monomers (3â²-NH<sub>2</sub>-2AIpddA, 3â²-NH<sub>2</sub>-2AIpddG, 3â²-NH<sub>2</sub>-2AIpddC, 3â²-NH<sub>2</sub>-2AIpddT), 50 mM MgCl<sub>2</sub>, 200 mM Na<sup>+</sup>-HEPES pH 8.0, 25 Â°C. P, primer. C, control, which is the reaction mixture without activated NP-DNA monomers.<br/><br/><ol class="npwtwo"><li>Combine the FAM-labeled primer, RNA template (synthesized in-house, or purchased), 50 mM Na<sup>+</sup>-HEPES (pH 8) buffer, NaCl, EDTA (pH 8), and water according to Table 1 in a PCR tube.<br/><br/><strong>Table 1. Component amounts for the pre-annealed primer-template duplex master mix</strong><br/><img alt="" class="layerphoto" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902093402_4671.jpeg" width="650"/><br/><br/></li><li>Anneal the primer-template duplex using a thermal cycler (95 Â°C to 25 Â°C over 15 min).</li><li>Combine the annealed duplex, Na<sup>+</sup>-HEPES (pH 8) buffer, MgCl<sub>2</sub>, and water according to Table 2.<br/><em>Note: The experiment should be carried out in triplicate (i.e., The primer extension reaction is set up in three individual tubes, making three replicates).</em><br/><br/><strong>Table 2. Component amounts for each primer extension experiment replicate</strong><br/><img alt="" class="layerphoto" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902093434_1026.jpeg" width="650"/><br/><br/></li><li>Resuspend each of the monomers in 19 Âµl water with 1 Âµl of 1M HEPES (pH 8) at 0 Â°C to generate stock solutions at a concentration of 100 mM.</li><li>Check the pH of each monomer stock using pH paper.<br/><em>Note: This is a rough estimate and is to verify that the stock concentrations are near pH 8.0-8.5.</em> </li><li>Mix 10 Âµl of each monomer in a new microcentrifuge tube.</li><li>Initiate the primer extension reaction by the addition of the monomer mixture (6 Âµl) to each of the replicates. Reactions are typically kept in a Digital block heater or a thermal cycler at 25 Â°C. <br/><em>Note: It is not necessary to protect the reaction from light.</em> </li><li>Quench 1 Âµl of the primer extension reaction mixture in 29 Âµl of quench solution (13 mM EDTA (pH 8), ~8 ÂµM complementary RNA (synthesized in-house or purchased), and 90% (v/v) formamide:water) at desired reactions times.<br/><em>Note: In this specific case, time intervals of 0.5 h, 1 h, 2 h, 3 h, 5 h, and 24 h were chosen based on previous literature/data. If this information is lacking, two independent experiments are typically conducted: A coarse time-course (</em><em>e.g.</em><em>, 5 min, 10 min, 0.5 h, 1 h, 2 h, 4 h, 24 h, 48 h) followed by a refined time-course (</em><em>e.g.</em><em>, as described above).</em> </li><li>Prepare an analytical PAGE gel (similar to preparatory PAGE) using 0.75 mm spacers and a 30-well comb.<br/><em>Note: 30 ml of gel mix is typically sufficient, with polymerization initiated using 15 Âµl TEMED and 75 Âµl of 20% (v/v) aqueous ammonium persulfate.</em> </li><li>Prerun the gel at 5 W for 15min.</li><li>Wash each well using 1x TBE using a disposable syringe equipped with a disposable needle. <br/><em>Note: This should be done right before loading the samples onto the gel.</em> </li><li>Load the quenched samples from the time-course (3 Âµl) in separate wells.</li><li>Load dye solution (bromophenol blue and xylene cyanol) as a molecular weight marker in flanking wells.</li><li>Begin the electrophoresis at 5 W for 10 min.</li><li>Turn up the power to 20 W for 1 h.</li><li>Stop the power and wash the outside of the plates with warm water.</li><li>Dry the plates.</li><li>Analyze the gel using an Amersham typhoon biomolecular imager according to the manufacturer's directions.<br/><em>Note: The FAM fluorophore is best excited by the blue laser (488 nm). All gels should be imaged at a minimum resolution of 100 Î¼m.</em> </li><li>Quantify the fraction of primer remaining, and the fraction of each primer extension product, at each timepoint (in %) using ImageQuant<sup>TM</sup>. This can be automatically accomplished by the software (<em>e.g.</em>, background subtraction, detection of bands etc.). The user then manually adjusts the parameters as needed (<em>e.g.</em>, selected area for each band, add bands, remove automatically detected artifacts). Refer to the user manual for more information concerning quantification.<br/><em>Note: Background subtraction is important to generate reproducible and reliable data.</em><br/></li></ol></li></ol><p class="pptt" id="biaoti35458">### Data analysis</p><p class="ppzz">The main data processing involves relative quantification of the primer and primer extension product bands (Procedure E of the protocol). ImageQuant<sup>TM</sup> was used for all reported analysis. Alternative software, such as ImageJ (free software from the National Institute of Health), can be utilized. Software compatibility should be verified before using softwares other than ImageQuant<sup>TM</sup>.</p><p class="pptt" id="biaoti35459">### Notes</p><p class="ppzz">To obtain reproducible primer extension results, the materials used in independent experiments must be of the same quality, and must be prepared and stored in the same way. Significant variability has been observed when monomer stocks are either contaminated with &gt; 10% of degraded products, or if the stock solutions are not at the correct pH after resuspension in 50 mM Na<sup>+</sup>-HEPES (pH 8).</p><p class="pptt" id="biaoti35460">### Recipes</p><ol class="npwtwok"><li>8 M urea (in water)<br/><ol class="npwthree"><li>8 mol of urea (480.48 g) was transferred to a 1 L bottle<br/></li><li>Deionized water was added to the ~850 ml mark and allowed to stir for 30 min<br/></li><li>The mixture was further diluted to the 1 L mark to make an 8 M urea solution</li></ol></li><li>20% APS<br/><ol class="npwthree"><li>200 mg of ammonium persulfate was transferred to a 1.5 ml microcentrifuge tube<br/></li><li>1 ml of deionized water was added to the microcentrifuge tube and centrifuged until full dissolution at room temperature</li></ol><em>Note: This solution should be used within approximately 1 month and stored at 4 Â°C.</em> </li><li>1 M TEAB<br/><ol class="npwthree"><li>1 mol of triethylamine (101.2 g/139.4 ml) was transferred to a 1 L bottle<br/></li><li>Deionized water was added to the ~900 ml mark<br/></li><li>Place this solution in an ice bath<br/></li><li>Using a gas diffusion apparatus, bubble gaseous CO<sub>2</sub> into the triethylamine/water until the solution pH reaches 7.5 (approximately 3 h depending on the rate of diffusion)<br/></li><li>Then, dilute the solution with deionized water to the 1 L mark<br/></li><li>Store at 4 Â°C<br/></li><li>Check the pH before use, and re-bubble CO<sub>2</sub> to pH 7.5 if need be</li></ol></li></ol><p class="pptt" id="biaoti35461">### Acknowledgments</p><p class="ppzz">The original work was reported in the <em>Journal of the American Chemical Society</em> (J. Am. Chem. Soc. 2019, 141, 10481-10488). The authors are indebted to Mr. Constantin Giurgiu, Dr. Anna Wang, and Dr. Tom Wright for helpful discussions concerning this work. J.W.S. is an Investigator of the Howard Hughes Medical Institute. This work was supported in part by a grant (290363) from the Simons Foundation to J.W.S. and by a grant from the NSF (CHE-1607034) to J.W.S. D.K.O. is a recipient of a Postdoctoral Research Scholarship (B3) from the Fonds de Recherche du QuebecâNature et Technologies (FRQNT), Quebec, Canada, and a Postdoctoral Fellowship from Canadian Institutes of Health Research (CIHR) from Canada.</p><p class="pptt" id="biaoti35462">### Competing interests</p><p class="ppzz">The authors declare no competing financial interest.</p><p class="pptt" id="biaoti35463">### References</p><ol class="npwtwok"><li>Blain, J. C. and Szostak, J. W. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24606140" target="_blank">Progress toward synthetic cells.</a> <em>Annu Rev Biochem</em> 83: 615-640. </li><li>Ding, D., Gryaznov, S. M. and Wilson, W. D. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9724520" target="_blank">NMR solution structure of the N3' â P5' phosphoramidate duplex d(CGCGAATTCGCG)2 by the iterative relaxation matrix approach.</a> <em>Biochemistry</em> 37(35): 12082-12093.</li><li>Gryaznov, S. (1997). <a href="https://www.tandfonline.com/doi/abs/10.1080/07328319708006106" target="_blank">Synthesis and properties of the oligonucleotide N3â² â P5â² phosphoramidates.</a> <em>Nucleos Nucleot Nucl</em> 16(7-9): 899-905.</li><li>Gryaznov, S. M., Schultz, R. G., Chen, Jer-kang. (1995). <a href="https://www.sciencedirect.com/science/article/abs/pii/S0167478199001517" target="_blank">Oligonucleotide N3â² â P5â² phosphoramidates.</a> <em>Proc Natl Acad Sci</em> 92: 5798-5802.</li><li>HÃ¤nle, E. and Richert, C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29779237" target="_blank">Enzyme-free replication with two or four bases.</a> <em>Angew Chem Int Ed Engl</em> 57(29): 8911-8915. </li><li>Jin, L., Engelhart, A. E., Adamala, K. P. and Szostak, J. W. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29553563" target="_blank">Preparation, purification, and use of fatty acid-containing liposomes.</a> <em>J Vis Exp</em>(132): 57324.</li><li>Li, L., Prywes, N., Tam, C. P., O'Flaherty, D. K., Lelyveld, V. S., Izgu, E. C., Pal, A. and Szostak, J. W. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28117989" target="_blank">Enhanced nonenzymatic RNA copying with 2-aminoimidazole activated nucleotides.</a> <em>J</em> <em>Am Chem Soc</em> 139(5): 1810-1813.</li><li>Prywes, N., Blain, J. C., Del Frate, F. and Szostak, J. W. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27351102" target="_blank">Nonenzymatic copying of RNA templates containing all four letters is catalyzed by activated oligonucleotides.</a> <em>Elife</em> 5: e17756.</li><li>Rojas StÃ¼tz, J. A. and Richert, C. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11741453" target="_blank">A steroid cap adjusts the selectivity and accelerates the rates of nonenzymatic single nucleotide extensions of an oligonucleotide.</a> <em>J Am Chem Soc</em> 123(50): 12718-12719. </li><li>Szostak, J. (2012). <a href="https://jsystchem.springeropen.com/articles/10.1186/1759-2208-3-2" target="_blank">The eightfold path to non-enzymatic RNA replication.</a> <em>J Syst Chem</em> 3: 2.</li><li>Szostak, J. W. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28514493" target="_blank">The narrow road to the deep past: in search of the chemistry of the Origin of Life.</a> <em>Angew Chem Int Ed Engl</em> 56(37): 11037-11043.</li><li>Szostak, J. W., Bartel, D. P. and Luisi, P. L. (2001). <a href="http://www.ncbi.nlm.nih.gov/pubmed/11201752" target="_blank">Synthesizing life.</a> <em>Nature</em> 409(6818): 387-390. </li><li>Tereshko, V., Gryaznov, S. and Egli, M. (1998). <a href="https://pubs.acs.org/doi/pdf/10.1021/ja971962h" target="_blank">Consequences of replacing the dna 3â²-oxygen by an amino group: high-resolution crystal structure of a fully modified N3â² â P5â² phosphoramidate dna dodecamer duplex<sup>+</sup>.</a> <em>J Am Chem Soc</em> 120(2): 269-283.</li><li>Zhang, N., Zhang, S. and Szostak, J. W. (2012). <a href="https://pubs.acs.org/doi/10.1021/ja212027q" target="_blank">Activated ribonucleotides undergo a sugar pucker switch upon binding to a single-stranded RNA template.</a> <em>J Am Chem Soc</em> 134(8): 3691-3694.</li><li>Zhang, S., Blain, J. C., Zielinska, D., Gryaznov, S. M. and Szostak, J. W. (2013a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24101473" target="_blank">Fast and accurate nonenzymatic copying of an RNA-like synthetic genetic polymer.</a> <em>Proc Natl Acad Sci U S A</em> 110(44): 17732-17737. </li><li>Zhang, S., Zhang, N., Blain, J. C. and Szostak, J. W. (2013b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23252395" target="_blank">Synthesis of N3'-P5'-linked phosphoramidate DNA by nonenzymatic template-directed primer extension.</a> <em>J Am Chem Soc</em> 135(2): 924-932. <br/></li></ol>