<body>
            <h1>Quantitative Kinetic Analyses of Histone Turnover Using Imaging and Flow Cytometry</h1>
            <p><strong>Authors:</strong> Hanae Sato, Robert H. Singer, John M. Greally</p>
            <h2>Abstract</h2>
            <p>Dynamic histone changes occur as a central part of chromatin regulation. Deposition of histone variants and post-translational modifications of histones are strongly associated with properties of chromatin status. Characterizing the kinetics of histone variants allows important insights into transcription regulation, chromatin maintenance and other chromatin properties. Here we provide a protocol of quantitative and sensitive approaches to test the timing of incorporation and dissociation of histones using a two-color SNAP-labeling system, labelling pre-existing and newly-incorporated histones distinctly. Together with cell cycle synchronization methods and cell cycle markers, this approach enables a pulse-chase analysis to determine the turnover of histone variants during the cell cycle, detected using imaging or flow cytometry methods at single cell resolution. As well as testing global histone turnover, cell cycle-dependent cellular localization of histone variants can be also addressed using imaging approaches.</p>
            <p><strong>Keywords:</strong> Chromatin dynamics, Histone, Cell cycle, Imaging, Flow Cytometry</p>
            <h2>Background</h2><p>Chromatin remodeling is part of the numerous fundamental cellular activities in eukaryotic cells (Geiman and Robertson, 2002; Clapier and Cairns, 2009). Accessibility by transcription factors and RNA polymerases are generally associated with changes in DNA methylation and chromatin states, including accessibility, post-translational histone modifications and deposition of histone variants. Histone variants differentially coordinate the gene expression that regulates development, cell differentiation or other physiological activities (Banaszynski <em>et al.</em>, 2010). They also play diverse roles in DNA repair, telomere maintenance, heterochromatin formation and chromatin segregation (Henikoff and Smith, 2015; Zink and Hake, 2016). Moreover, dysregulation of a histone variantâs incorporation is associated with cancer (Vardabasso <em>et al.</em>, 2014), indicating a significant role in human disease. To understand the kinetics of histone variants, including incorporation into and dissociation from specific chromatin regions during the cell cycle, is crucial since it tightly links regulatory properties with the mitotic maintenance of epigenetic (heritability from parent to daughter cells). DNA replication involves major chromatin remodeling to duplicate the entire chromatin structure following mitosis. Histones that associate with chromatin prior to DNA replication are transiently dissociated from DNA by the access of the DNA polymerase complex. Release of pre-existing histones randomly re-associate at newly synthesizing replication forks together with newly synthesized histones (Balhorn <em>et al.</em>, 1975; Alabert and Groth, 2012; Annunziato, 2012). Pre-existing post-translational modifications on histones and some histone variants also re-associate on the newly synthesized DNA at the replication fork, explaining re-association of pre-existing histone at the replication fork is one part of the maintenance of mitotic inheritance of chromatin states. To determine the timing of post-translational modifications or incorporation of histone variants, a sensitive pulse-chase system that can distinguish the detection of newly incorporated histone from pre-existing histones is required. Newly synthesized canonical histones are unmodified when they are incorporated into chromatin during DNA replication. Unlike <em>de novo</em> DNA methylation, which occurs together with DNA replication (Tillo <em>et al.</em>, 2016), most histone marks are not established on newly incorporated histones on the replicating fork. Proteomic based pulse-chase approaches (see in <strong>Alternative methods section</strong> below) have been used to determine the global kinetics of histone post-translational modifications by detecting the pre-existing and new deposition of histone acetylation and methylation at lysine residues (Pesavento <em>et al.</em>, 2008; Scharf <em>et al.</em>, 2009; Martinez-Garcia <em>et al.</em>, 2011; Xu <em>et al.</em>, 2011; Zee <em>et al.</em>, 2012; Alabert <em>et al.</em>, 2015). This approach identified two distinctive kinetic patterns of histone modifications. One group, such as histone H3 acetyl-lysine at 27 (H3 K27 ac), exhibits rapid turnover to equalize, and notably they are not maintained through the cell cycle (Scharf <em>et al.</em>, 2009). This rapid acetylation kinetics probably represent temporally active transcription dynamics (Stasevich <em>et al.</em>, 2014). While another group including H3 lysine tri-methylation at 9 (K9me3) or H3 K27me3 is acquired more slowly and step-wise from mono and di to tri-methylation, established during G1 phase following the cell cycle instead of before mitosis (Pesavento <em>et al.</em>, 2008; Scharf <em>et al.</em>, 2009; Martinez-Garcia <em>et al.</em>, 2011; Xu <em>et al.</em>, 2011; Zee <em>et al.</em>, 2012; Alabert <em>et al.</em>, 2015). Importantly, this group has the property of mitotic chromosome memory. The potential mechanism of maintenance of histone marks over cell division has been addressed by imaging approaches. <em>In situ</em> proximity ligation assays using specific antibodies against histone lysine methylations and their methyltransferases detected that histone modifiers continuously associate with the replicating DNA component in <em>Drosophila</em> embryos, suggesting the association of modifiers on replicating DNA may provide a âtagâ to be methylated (Petruk <em>et al.</em>, 2012).<br/><br/>Recent advances in chemical protein labeling technologies provide us with a powerful tool for protein tagging applications in living cells (<em>e.g.</em>, SNAP (New England Biolabs), CLIP (New England Biolabs), Halo (Promega) and TMP (Active Motif) tag). These technologies are based on the covalent labeling of genetically encoded tags that bind with specific ligands conjugated to cell permeable substrates such as synthetic fluorescent dyes or biotin, which can mediate affinity purification in biochemical applications. In contrast with common genetically encoded tags, this chemical labeling of protein can be utilized in timing-dependent labeling with many choices of fluorophores, which allows the pulse-chase labeling of specific protein. This labeling technology has revealed the deposition timing of histone variants at specific chromatin architectures using imaging detection (<em>e.g.</em>, CENP-A at centromeres [Dunleavy <em>et al.</em>, 2009] and macroH2A at heterochromatin [Sato <em>et al.</em>, 2019]).<br/><br/>Here, we provide a detailed protocol of a pulse-chase method using the SNAP-tag labeling system which has utilized quantitative histone variant detection with single cell resolution. Using this protocol, distinct histone kinetics, dissociation of pre-existing histones and association of newly synthesized histones, can be detected simultaneously. We describe two detection approaches, fluorescence microscopy and flow cytometry, as well as the detail of imaging analysis using FIJI/ImageJ software which is freely available (<a href="https://fiji.sc/" target="_blank">https://fiji.sc/</a>).<br/><br/><strong>Advantages and Limitations</strong><br/>The pulse-chase method using a chemical protein labeling system is an easy, non-hazardous and sensitive approach compared with a conventional pulse-chase approach using radioactive molecules (see in <strong>Alternative methods section</strong>). Most of the required reagents and fluorophores are commercially available. Global turnover of histones can be addressed using imaging and flow cytometric applications and notably, timing-specific localization at specific chromatin architectures also can be characterized with imaging approaches. Using this protocol, both pre-existing and newly incorporated histone variants can be detected simultaneously in the same cells with single cell resolution. Unlike radioisotope or metabolic labeling approaches, which detect endogenous histones, this labeling approach relies on the genetically encoded tags (<em>e.g.</em>, SNAP-, CLIP-, Halo-tag) that can be linked with specific substrates. Therefore, a plasmid construct that expresses the target histones with SNAP-tag and its use in a stably expressing cell line are required. In addition, the localization and other biological functions of desired tagging histones must be examined to determine whether it remains functioning as an endogenous histone. Optimization of the construct (<em>e.g.</em>, N-terminus or C-terminus tagging, changing the choice of promoter) and levels of expression might be necessary for obtaining accurate observations. Another limitation of this approach is that it is not applicable for detection of post-translational modifications of histones.<br/><br/><strong>Alternative methods</strong><br/><strong><em>Isotope labelling with proteomic detection</em></strong><br/>SILAC (Stable Isotope Labelling with Amino acids in Culture) followed by mass spectrometry is a powerful approach to investigate global turnover of endogenous histone variants and post-translational modifications (Yuan <em>et al.</em>, 2014). In this approach, newly synthesized histones are labeled with radioactive heavy isotope and chase the turnover of labeled his tones compared with pre-existing histones containing light amino acids. Following mass spectrometry analysis determines the pre-existing and deposition of post-translational modifications or variants. This approach is suitable to detect global histone turnover, but is not able to detect histone marks at specific chromatin architecture or genomic loci.<br/><br/><strong><em>Metabolic labeling with genome-wide approaches</em></strong><br/>Non-radioactive metabolic labeling of nascent proteins can be an alternative approach to label global newly synthesized histones. The approach, âCovalent Attachment of Tags to Capture Histones and Identify Turnoverâ, also called âCATCH-ITâ, enables genome-wide investigation to characterize active histone replacement (Deal <em>et al.</em>, 2010). This approach is based on the labeling scheme of nascent peptide by incorporation of methionine homolog, azidohomoalanine (Aha), which is generally used for the detection of active translation in cells. In this approach, the nucleosomes containing Aha-labeled newly synthesized histones are bioconjugated with biotin by a cycloaddition reaction (as known as âclickâ chemistry), and pulled down with streptavidin beads. Isolated DNA from pull-down was applied on a tiling microarray to determine the genomic loci that exhibit active histone replacement in <em>Drosophila</em> S2 cells. The characterized genomic sites that have active histone turnover correspond with the site of incorporation of histone H3 variant, H3.3, which is detected at transcriptionally active loci (Henikoff <em>et al.</em>, 2009). This approach enables the detection of genomic loci with active turnover of histones.<br/><br/>Chemical labeling approaches such as the SNAP-tagging system can also be the alternative option to investigate genome-wide histone variant incorporation (Sato <em>et al.</em>, 2019). In this approach, newly incorporated SNAP-tagged histones are linked with SNAP-biotin after the treatment of SNAP-Cell<sup>Â®</sup> Block (bromothenylpteridine, BTP), a non-fluorescent substrate to mask the reactivity of pre-existing histones. Then, biotin-linked, newly incorporated histones can be pulled down with streptavidin beads. The purified DNA fraction from pull-down samples can be sequenced with massive parallel sequencing. This approach might be useful to detect timing and genomic loci dependent incorporation of histone variants but unable to detect post-translational modifications.</p>
        </body>

<p class="pptt" id="biaoti35502">## Materials and Reagents</p><ol class="npwtwok"><li>50-ml Falcon tubes</li><li>15-ml Falcon tubes</li><li>1.5-ml tubes</li><li>Tissue culture plates (12 or 24-well; Falcon, catalog number: <a class="blackNoUnder" target="_blank"> 353043 </a> or <a class="blackNoUnder" target="_blank"> 353047 </a> )</li><li>Tissue culture dishes (35 x 10 mm or 60 x 15 mm, and 100 x 20 mm, Falcon, catalog numbers: <a class="blackNoUnder" target="_blank"> 353001 </a> or <a class="blackNoUnder" target="_blank"> 353002 </a> and <a class="blackNoUnder" target="_blank"> 353003 </a> )</li><li>Coverslips (18 or 12 mm No.1.5)</li><li>Microscope slides (<em>e.g.</em>, Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12-544-2 </a> )</li><li>5-ml FACS tubes (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 149595 </a> )</li><li>Adherent cells of interest (<em>e.g.</em>, mouse embryonic fibroblasts [MEFs], HEK293 cells)</li><li>SNAP-Cell Oregon Green (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> S9104S </a> )</li><li>SNAP-Cell Block (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> S9106S </a> )</li><li>SNAP-Cell TMR-Star (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> S9105S </a> )</li><li>Thymidine (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> T1895 </a> )</li><li>RO-3306 (Santa Cruz Biotechnology, catalog number: <a class="blackNoUnder" target="_blank"> sc-358700 </a> )</li><li>Nocodazole (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> M1404 </a> )</li><li>Prolong Diamond Antifade Mountant (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> P36970 </a> )</li><li>Prolong Diamond Antifade Mountant with DAPI (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> P36962 </a> )</li><li>Drug for selection of stably expressing cells [<em>e.g.</em>, 600-1,200 Âµg/ml G418 (geneticin), Zeocin (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R25001 </a> )]</li><li>Click-iT<sup>TM</sup> EdU Cell Proliferation Kit for Imaging, Alexa Fluor<sup>TM</sup> 647 dye (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> C10340 </a> ) </li><li>pSNAPf Vector (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> N9183S </a> )</li><li>pCCL-CellCycle (Sato <em>et al.</em>, 2019)</li><li>Suitable culture media and supplements [<em>e.g.</em>, DMEM (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D6429 </a> ) supplemented with 10% FBS (Atlanta Biologicals, Inc., catalog number: <a class="blackNoUnder" target="_blank"> S11150H </a> ) and 100 U/ml Penicillin-Streptomycin (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 15140148 </a> )]</li><li>EDTA, 0.5 M, pH 8.0, Molecular Biology Grade (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 324506 </a> )</li><li>HEPES solution, 1 M, pH 7.0-7.6, sterile-filtered (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> H0887 </a> )</li><li>Coverslips coating solution, <em>e.g.</em>, 0.01% Poly-L-lysine solution (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P4707 </a> ), collagen coating solution (according to the manufacturer's instruction, Sigma, catalog number: <a class="blackNoUnder" target="_blank"> SAFC-125-50 </a> )</li><li>Sterile 1x PBS pH 7.4, no calcium, no magnesium for cell culture (<em>e.g.</em>, Corning, catalog number: <a class="blackNoUnder" target="_blank"> 21-031-CV </a> )</li><li>10Ã DPBS, (<em>e.g.</em>, Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11666789001 </a> )</li><li>32% (wt/vol) paraformaldehyde (PFA) (Electron Microscopy Sciences, catalog number: <a class="blackNoUnder" target="_blank"> 15714 </a> )<br/><em>!CAUTION: Paraformaldehyde is a hazardous solution and a cross-linking agent. Wear protective gloves and handle it under a fume hood.</em> </li><li>Triton X-100, 0.1% (vol/vol) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BD 151500 </a> )</li><li>Nuclease-free water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10977-015 </a> )</li><li>Immersion oil 1.518, for the microscope/objective</li><li>Bovine serum albumin (BSA) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A7030 </a> )</li><li>Cloning of SNAP-histone expression vector and its stably expressing cell line (see Recipes)</li><li>Fixation Buffer (see Recipes)</li><li>Permeabilization buffer (see Recipes)</li><li>Blocking buffer (see Recipes)</li><li>Sorting Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35503">### Equipment</p><ol class="npwtwok"><li>Vortex mixer</li><li>Tabletop centrifuge</li><li>Pointed tip tweezer</li><li>Water bath</li><li>Biological hood/biosafety cabinet (for cell culture work)</li><li>Cell culture incubator suitable for cell culture of your choice (<em>e.g.</em>, 37 Â°C, 5% CO<sub>2</sub>)</li><li>Wide-field fluorescent microscope (<em>e.g.</em>, Olympus, model: <a class="blackNoUnder" target="_blank"> BX-61 </a> , equipped with four filter sets for DAPI (Semrock, model: <a class="blackNoUnder" target="_blank"> DAPI-5060C-Zero </a> ), Cy3 (Chroma, model: <a class="blackNoUnder" target="_blank"> 41007 </a> ), FITC (Semrock, model: FITC-5050A-Zero), and Cy5 (Semrock, model: <a class="blackNoUnder" target="_blank"> Cy5-4040C-Zero </a> ), an EXFO X-Cite Series 120 PC metal halide light source, Photometrics Cool SNAP HQ CCD camera, and molecular Devices Metamorph acquisition software or equivalent microscope set-ups or equivalent microscope system</li><li>Flow cytometer (BD Biosciences, model: <a class="blackNoUnder" target="_blank"> LSR II </a> )<br/></li></ol><p class="pptt" id="biaoti35504">### Software</p><ol class="npwtwok"><li>Metamorph software for image acquisition (<a href="https://www.moleculardevices.com/systems/metamorph-research-imaging/metamorph-microscopy-automation-and-image-analysis-software" target="_blank">https://www.moleculardevices.com/systems/metamorph-research-imaging/metamorph-microscopy-automation-and-image-analysis-software</a>)</li><li>ImageJ/FIJI (Schindelin <em>et al.</em>, 2012) (avalable at <a href="https://fiji.sc/" target="_blank">https://fiji.sc/</a>)</li><li>FlowJo (<a href="https://www.flowjo.com/solutions/flowjo/downloads" target="_blank">https://www.flowjo.com/solutions/flowjo/downloads</a>)<br/></li></ol><p class="pptt" id="biaoti35505">### Procedure</p><p class="ppzz">The protocol contains five main processes: (i) Generation of the SNAP-tagged histone expression vector and stable expressing cell lines, (ii) Optimization of cell cycle synchronization, (iii) Detection of global and local histone incorporation using imaging, (iv) Detection of global histone incorporation using flow cytometry, (v) Analysis. Although the first process (i) explains general procedures for cloning and establishing stably expressing cell lines, we emphasize the tips on how to design the SNAP-tagged histone expression vector and isolation of stably expressing cells. This protocol mainly describes the procedures for the fluorescent labeling approaches to investigate histone turnover using microscope (iii) and flow cytometry (iv) (Figure 1).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902002126_7729.jpg" width="300"/><br/><strong>Figure 1. Workflow of protocol.</strong> Three sections of pipelines, Generation of constructs, Sample preparation for imaging and Sample preparation for flow cytometry, are shown as boxes.<br/><br/><strong>(i) Generation of the SNAP-tagged histone expression vector and stable expressing cell lines </strong><br/>Protein tagging with a small epitope is valuable for detection of the protein of interest in various biochemical approaches. However, tagging a small peptide sometimes interferes with the biological function of the target protein and localization of the protein. In general, it is desirable to test whether the N-terminal or C-terminal tagging of protein alters its functions. The localization of the SNAP-tagged histone variant can be confirmed using immunofluorescence by comparing with the endogenous histone variant whether it is localized at the expected chromatin sites. Additionally, it may be necessary to test if the SNAP-tagged construct retains its specific function in chromatin.<br/>  To determine the precise turnover of histone variants, the generation of stably expressing cell lines is highly recommended, since expression level is expected to be altered after mitosis when a transiently transfected SNAP-tagged histone is used. Inserting the SNAP-tag sequence into the endogenous histone locus using CRISPR/Cas9 technology is ideal but not necessary. The SNAP-expressing vector is commercially available from NEB, which contains a neomycin selection gene. In the process of establishing stably expressing cell lines, titration of the drug concentration in your cells is required for the isolation of positive cells. If your cell line already obtains neomycin resistance, such as HEK 293T cell which is immortalized with the large T antigen with neomycin resistance, the neomycin selection marker needs to be replaced.<br/>  The expression level of the SNAP-tagged histone may also influence the timing of incorporation of histones. Since overexpression of histone variants may cause undesirable non-specific incorporation into chromatin, isolating a cell population with a moderate to lower expression of SNAP-tagging histones by Fluorescence-activated cell sorting (FACS) before the pulse-chase experiment is recommended. We also recommend testing the level of SNAP-histone variants compared with endogenous histone variants in purified chromatin fractions by western blotting after establishment of the cell line.<br/><br/><strong>(ii) Optimization of cell cycle synchronization</strong><br/>Cell cycle synchronization is a common method to arrest the whole cell population into a particular cell cycle phase. Most cell cycle synchronization methods rely on the use of a drug which blocks a specific function required for cell cycle progression. While various cell cycle synchronization methods were established in the past (<strong>Table 1</strong>), the efficiency of synchronization with drug concentration might depend on the cell type used. Any cell synchronization methods can be used in this protocol after the optimization of drug treatment and staining scheme of pre-existing and newly incorporated histones. Here we introduce the protocol in HEK293T cells using a double thymidine block for synchronization at G1/S phase and mitotic shake-off for synchronization at G2/M phase (Figure 2). Both are widely utilized as general cell cycle synchronization methods. Successful cell synchronization and release should be confirmed by flow cytometry, western blotting or immunofluorescence (IF) with cell cycle indicators or markers such as co-expressing the Fucci cell cycle reporter (Sakaue-Sawano <em>et al.</em>, 2008), DNA staining with Hoechst 33342, anti-phosphorylated Histone-3 at Serine-28 (M phase marker) or any other cell cycle markers.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902002151_6118.jpg" width="300"/><br/><strong>Figure 2. Workflow of cell synchronization and labeling of histones.</strong> Each step of cell synchronization methods (<strong>A.</strong> double thymidine block and <strong>B.</strong> mitotic shake-off) and timing of labeling of pre-existing histones are shown.<br/><br/><strong>Table 1. Common drug list for cell synchronization</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902005648_0752.jpg" width="600"/><br/><br/><strong>(iii) Detection of global and local histone incorporation using imaging</strong><br/>A key procedure for imaging detection is to grow cells on coverslips. Specific procedures for coating of coverslips might be required for different cell types. Careful handling is important to keep the adherent cells attached to the coverslips during the entire procedure. Localization of the histone variant with specific chromatin structure can be addressed by colocalization analysis with a marker of desired chromatin structure using IF after the fixation of cells. In this case, carefully consider the choice of fluorophores and the filter setting of your microscope to avoid the incompatible bleed-through detection of fluorophores.</p><p class="ppzz"><br/></p><ol class="npwone"><li>Labeling of histone incorporation during S-G2 phase â <span style="color:#E56600;">Timing</span> 5 days<ol class="npwtwo"><li>Place coverslips in 12-well or 24-well plate and perform coating coverslips (<em>e.g.</em>, 0.01% poly-L lysine, 0.01% collagen) at room temperature for 1 h. After washing 3 times with double distilled water, spread cells on coated coverslips and grow cells at least 24 h.</li><li>Cell synchronization at the G1/S can be performed using double thymidine block methods (Jackman and O'Connor, 2001). Incubation time and drug concentrations must be optimized for each cell type. Synchronize the cells at the G1/S transition with a treatment of 2 mM thymidine for 16 h, release from thymidine for 9 h and treat again with 2 mM thymidine again for 17 h.<br/><span style="color:#009900;">? TROUBLESHOOTING</span> (Table 2)<br/><br/><strong>Table 2. Troubleshooting Table</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902005904_8298.jpg" width="600"/><br/><br/></li><li>Label pre-existing SNAP-tagged histones with SNAP-Cell Oregon Green (1 ÂµM) in culture medium for 30 min at 37 Â°C in 5% CO<sub>2</sub>.</li><li>Block unlabeled pre-existing SNAP-tagged histones with non-fluorescent SNAP-substrates, SNAP-cell Block (10 ÂµM) for 30 min in culture medium at 37 Â°C in 5% CO<sub>2</sub>.<br/><em><span style="color:#9933E5;">CRITICAL STEP</span><span style="color:#9933E5;">: </span>This step is important to avoid insufficient initial labeling of pre-existing histones that will result in inaccurate detection of newly incorporated histones.</em> </li><li>Add culture medium containing 10 Î¼M of EdU and 10 Î¼M RO-3306 and incubate for 12 h at 37 Â°C in 5% CO<sub>2</sub>.<br/><span style="color:#009900;">? TROUBLESHOOTING</span> (Table 2)</li><li>Label newly incorporated SNAP histones using SNAP-Cell TMR (1 ÂµM) in culture medium with RO-3306 for 30 min at 37 Â°C in 5% CO<sub>2</sub>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Labeling of histone incorporation during G1 phase â <span style="color:#E56600;">Timing</span> 5 days <br/>To collect mitotic cells, a mitotic shake off can be performed.<ol class="npwtwo" start="7"><li>Spread cells in 100 mm dish.</li><li>Synchronize the cells at S/G1 border with 2 mM thymidine for 24 h, then treat with 20-500 nM nocodazole for 12 h. Incubation time and drug concentrations must be optimized for each cell type.</li><li>Check the cells forming rounding shape and weak attachment using light microscopy.<br/><em><span style="color:#9933E5;">CRITICAL STEP:</span> If cells are not forming rounded shape, synchronizing at M phase is not successful.</em><br/><span style="color:#009900;">? TROUBLESHOOTING</span> (Table 2)</li><li>Gently shake the dish to collect mitotic cells into 1.5 ml Eppendorf tube or 15 ml Falcon tubes.</li><li>Centrifuge the cells at 500 <em>x g</em> for 4 min and wash cells with a culture medium.</li><li>Wash cells twice with culture medium and spread cells on pre-coated coverslips as described in Step 1.</li><li>One to two hours later, check the cells if they attached to the coverslips.<br/><em><span style="color:#9933E5;">CRITICAL STEP:</span> The cells must be attached to the cover glass. If not, releasing from nocodazole treatment is not successful.</em><br/><span style="color:#009900;">? TROUBLESHOOTING</span> (Table 2)</li><li>Label pre-existing SNAP-tagged histones with SNAP-Cell Oregon Green (1 ÂµM) in culture medium for 30 min at 37 Â°C in 5% CO<sub>2</sub>.</li><li>Block unlabeled pre-existing SNAP-tagged histones with non-fluorescent SNAP-substrates, SNAP-cell Block (10 ÂµM) for 30 min in culture medium at 37 Â°C in 5% CO<sub>2</sub>.<br/><em><span style="color:#9933E5;">CRITICAL STEP:</span> This step is important to avoid insufficient initial labeling of pre-existing histones that will result in inaccurate detection of newly incorporated histones.</em> </li><li>Add 10 ÂµM of EdU and 2 mM thymidine in culture medium for 12-18 h at 37 Â°C in 5% CO<sub>2</sub>.</li><li>Label newly incorporated SNAP histones using SNAP-Cell TMR (1 ÂµM) in culture medium with 2 mM thymidine for 30 min at 37 Â°C in 5% CO<sub>2</sub>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Cell fixation, Permeabilization and Click-it EdU labeling â <span style="color:#E56600;">Timing</span> 2 days<ol class="npwtwo" start="18"><li>Fix cells with fixation buffer for 15 min at room temperature.<br/><em><span style="color:#009900;">PAUSE POINT:</span> The fixed cells can be stored in PBS at 4 Â°C for up to 1 month.</em><br/><span style="color:#E53333;">! CAUTION</span> Paraformaldehyde is a hazardous solution and a cross-linking agent. Wear protective gloves and handle it under a fume hood. </li><li>Permeabilize the cells with permeabilization buffer for 15 min at room temperature.</li><li>Block cells using 1x PBS containing 3% BSA for 1 h and perform click-it EdU labeling as described in the manufacturer's instruction.</li><li>(Optional) Immunofluorescence (IF) can be performed using the desired antibody if co-localization with histones needs to be addressed. However, cautious optimization is required for the microscope filter setup and wavelength range to avoid bleed-through from other fluorophores that co-labeled in the same cells.</li><li>DNA staining with DAPI or Hoechst 33342.</li><li>Mount coverslips using anti-Prolong Diamond Antifade Mountant.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Detection using microscope â <span style="color:#E56600;">Timing</span> 1-2 days<ol class="npwtwo" start="24"><li>Images can be acquired with an Olympus BX61 widefield, epifluorescent microscope using a 60x 1.4 PlanApo objective or equivalent. Filter sets for DAPI (Semrock), Cy3 (Chroma), FITC (Semrock), and Cy5 (Semrock), with an EXFO X-Cite Series 120 PC metal halide light source, Photometrics Cool SNAP HQ CCD camera, Olympus Type-F immersion oil (nd 1.516) and Molecular Devices Metamorph acquisition software or equivalent microscope set-ups are required. Images can be taken with optically sectioned using a 0.5 Î¼m Z step, spanning a 6~10.0 Î¼m Z depth in total depending on the thickness of each cell type. Exposure times of 10 to 200 ms are typically used to acquire each plane in the Cy3, Cy5, FITC and DAPI channels.</li></ol></li></ol><br/><strong>(iv) Detection of global incorporation using flow cytometry</strong><br/><p class="ppzz">We also introduce the protocol addressing the global histone turnover using flow cytometry detection. Although this approach can determine the global histone kinetics, it is unable to detect the histone kinetics at specific genomic loci or chromatin architectures.</p><br/><ol class="npwone" start="5"><li>Labeling of histone incorporation<br/><ol class="npwtwo" start="25"><li>Spread cells in 12 of 35 mm or 60 mm dishes.</li><li>Synchronize the cells at the G1/S transition by double thymidine block. Briefly, treated cells with 2 mM thymidine for 16 h, wash 3 times with 1x PBS, and release from thymidine by replacing culture medium for 9 h and treat with 2 mM thymidine again for 17 h.</li><li>Label pre-existing SNAP-tagged histones with SNAP-Cell Oregon Green (1 ÂµM) in culture medium for 30 min at 37 Â°C in 5% CO<sub>2</sub>.</li><li>Treat cells with SNAP-cell Block (10 ÂµM) for 30 min in culture medium at 37 Â°C in 5% CO<sub>2</sub> to mask insufficient labeling of pre-existing SNAP-tagged histones with non-fluorescent SNAP-substrates.<br/><em><span style="color:#9933E5;">CRITICAL STEP:</span> This step is important to avoid insufficient initial labeling of pre-existing histones that will result in inaccurate detection of newly incorporated histones.</em> </li><li>Release 6 dishes from cell synchronization for detection of 12- to 22-hour time points. Keep synchronizing other 6 dishes for detection of 2- to 12-hour time point.</li><li>Incubate cells for 12 h.</li><li>Release the other 6 dishes from cell synchronization for detection of 2- to 12-hour time point.</li><li>Harvest cells every 2 h and store cells at -80 Â°C.<br/><em><span style="color:#009900;">PAUSE POINT:</span> The frozen medium and protocol are depending on cell types.</em> </li><li>Thaw cells and wash one time with a pre-warmed medium.</li><li>Label newly incorporated SNAP histones by adding culture medium with SNAP-JF646 (1 ÂµM) to cell pellets. Pipet cells and incubate for 30 min at 37 Â°C water bath.</li><li>Centrifuge the cells at 500 <em>x g</em> for 4 min at room temperature and wash twice with pre-warmed culture medium.</li><li>(Optional) Commercially available Dyes detecting cell cycle (<em>e.g.</em>, Hoechst 33342) are also optional instead of the Fucci cell cycle indicator.</li><li>Detection of expression levels of Oregon green (Pre-existing histones), JF646 (newly incorporated histones), mCherry (S-G2 phase cell cycle indicator) and TagBFP (G1 phase cell cycle indicator) using Flow cytometry.</li></ol><br/><strong>BOX 1. Manual imaging Analysis workflow of global pre-existing and newly incorporated histones (see also Figure 3)</strong><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902010258_7886.jpg" width="300"/><br/><strong>Figure 3. Imaging analysis workflow.</strong> Outlines of imaging analysis to detect intensity of pre-existing, newly incorporated histones, nuclear staining (<em>e.g.</em>, DAPI and Hoechst 33342) and EdU (S phase marker) are shown.<br/><br/><ol class="npwtwo"><li>Process all image sets to Maximum Intensity Z-projection using the batch process function as follows:<br/><ol class="npwthree"><li><em>Process/Batch/Macro</em>.</li><li>Choose the Input folder in "Inputâ¦".</li><li>Choose the output folder in "Outputâ¦".</li><li>(Optional) Put specific name in âFile name contains:â if specific files in selected folder need to be processed.</li><li>Put macro code ârun("Z Project...", "projection=[Max Intensity]");â in the script box.</li><li>Click âProcessâ.</li></ol>It can be manually processed as follows:<br/><ol class="npwthree"><li>Open four images by dragging the files into the main interface of FIJI or following:<br/><em>File/Open/</em>, then choose your image.</li><li><em>Image/Stacks/Z Project/</em>, then choose projection type <em>âMax Intensityâ</em>.</li></ol></li><li>Define the region of interest (ROI) to measure the intensities of pre-existing and newly incorporated histones, and EdU incorporation by making a mask using nuclear staining such as DNA staining with DAPI.<br/>Duplicate the image displaying DNA staining such as DAPI.<br/><ol class="npwthree"><li>Select duplicated DAPI channel image.</li><li><em>Image/Duplicateâ¦ </em> </li></ol>Define the nuclear area using the intensity of nuclear staining.<ol class="npwthree" start="3"><li><em>Image/Adjust/Threshold</em>; then choose suitable threshold method (<em>Default</em> can be chosen as a start), check â<em>Dark background</em>â and <em>Apply</em>.</li><li><em>Process/Binary/Convert to Mask</em>.</li><li>(Optional) <em>Process/Binary/Fill holes</em>.</li><li>(Optional) <em>Process/Binary/Watershed</em> (<a href="https://imagej.net/Nuclei_Watershed_Separation" target="_blank">https://imagej.net/Nuclei_Watershed_Separation</a>).</li><li><em>Analyze/Set Measurements</em>, check <em>âAreaâ, âMean gray valueâ</em> and <em>âDisplay labelâ</em>, then OK.</li><li><em>Analyze/Analyze Particles</em>, add proper number to accomplish the selection of proper area into <em>âSize (pixel^2)â</em> and <em>âCircularityâ</em>, check <em>âAdd to Managerâ</em>, then OK.</li></ol></li><li>Measure DAPI signal (DNA) in the masked area using Region of interest (ROI).<ol class="npwthree"><li>Select channel image.</li><li>Click â<em>Measure</em>â in ROI Manager.</li></ol></li><li>Measure Oregon green signal (Pre-existing histones) using ROI.<ol class="npwthree"><li>Select Oregon green channel image.</li><li>Click â<em>Measure</em>â in ROI Manager.</li></ol></li><li>Measure TMR-STAR signal (newly incorporated histones) signal using ROI.<ol class="npwthree"><li>Select TMR-STAR channel image.</li><li>Click â<em>Measure</em>â in ROI Manager.</li></ol></li><li>Measure EdU signal (Nascent DNA) using ROI.<ol class="npwthree"><li>Select EdU channel image.</li><li>Click âMeasureâ in ROI Manager.</li></ol></li><li>Save results and analyze.</li></ol><br/><strong>BOX 2. Analysis workflow of local pre-existing and newly incorporated histones using color profiler or RBG profiler (see also Figure 5A)</strong> <ol class="npwtwo"><li>Install the plugin in Fiji/ImageJ software.</li><li>Open all image sets that will be compared by dragging the files into the main interface of FIJI or following:<br/><em>File/Open/</em>, then choose your image.</li><li>Merge these images into a single image.<br/><em>Image/Color/Merge Channels</em>.</li><li>Change the image type to RGB color.<br/><em>Image/Type/RGB color</em>.</li><li>Make a line to make a profile.<ol class="npwthree"><li>Choose a line tool.</li><li>Make a line.</li><li><em>Plugins/RGB Profiler</em>.</li></ol></li></ol><br/></li></ol><p><strong>Imaging analysis (~1 week)</strong><br/>Steps 1-6, cell synchronization using double thymidine block and labeling pre-existing histones for imaging analysis: 4 days.<br/>Steps 7-17, cell synchronization using mitotic shake off, and labeling pre-existing and newly incorporated histones for imaging analysis: 4 days.<br/>Steps 18- 19, fixation and permeabilization of the cells: 1 h.<br/>Step 20, labeling of EdU using click-it kit: 1 h.<br/>Step 21, (optional) IF using the desired antibody to determine the histone deposition at specific chromatin architecture or factors: 3 h.<br/>Steps 22- 23, DNA staining and mount coverslips: Overnight.<br/>Step 24, microscopy imaging: 1 day.<br/><br/><strong>Flow cytometry analysis (~1 week)</strong><br/>Step 25, Spread cells and cell synchronization using double thymidine block and labeling pre-existing histones: 4 days.<br/>Step 32, Release from cell synchronization and harvest cells: 2 days.<br/>Steps 34-37, labeling of newly incorporated cells and Flow cytometry analysis: 2 h.</p><p class="pptt" id="biaoti35506">### Data analysis</p><p class="ppzz">We introduce a basic analytical pipeline for imaging approach using FIJI/ImageJ which is a free open source image processing package. An automated analysis using ImageJ macro code would be time-saving for large data sets if you are familiar with programming. Additionally, many other imaging processing programs (<em>e.g.</em>, Matlab) can be also a great option to process enormous data sets.<br/>  Data analysis for global histone turnover using flow cytometry detection can be easily accomplished using available flow cytometers such as <a class="blackNoUnder" target="_blank"> LSR II </a> (BD Bioscience). The software such as FlowJo (BD Bioscience) can assess the FCS data sets and permit visualizing complex cytometric data sets. In this protocol, we can use a mean or median of the total intensity of fluorophore in the entire cell population with single-cell resolution to investigate the timing of incorporation and deposition of histones. Displaying histograms of fluorescent intensities from labeling with pre-existing and newly incorporated histones as well as cell cycle markers in this protocol, can help to visualize the timing of global histone kinetics including the dissociation of pre-existing histones and incorporation of newly synthesized histones during the cell cycle.<br/><br/><strong>Anticipated results</strong><br/>In this protocol, we summarized our previously published results for the cell cycle-specific dynamics of histone H3.1 and macroH2A1.2 in HEK293T cells to show that the SNAP-labeling pulse-chase system works reliably and to illustrate how to analyze the imaging data sets (Sato <em>et al.</em>, 2019).<br/>  Although we illustrate three parts in this protocol (<strong>Figure 1</strong>) including generation of constructs, sample preparation for imaging and sample preparation for flow cytometry, we mainly described the detailed protocol for the latter two. We also illustrate how to analyze the imaging data sets using ImageJ/FIJI (<strong>Figure 3</strong> and <strong>Box 1</strong>). Here we present an example of the SNAP pulse-chase detection during the cell cycle using an imaging approach detecting SNAP-histone H3.1 (<strong>Figure 4</strong>). We demonstrated two types of cell synchronization, double thymidine block and mitotic shake off, to investigate histone incorporation and dissociation during the S-G2 or G1 phase. The pre-existing and newly incorporated SNAP-H3.1 stably expressed in HEK 293T were labeled as shown in Figures 4A-4B. EdU click-it detection was also performed to confirm successful cell synchronization. The global histone incorporation rates in individual cells were determined by the mean of pixel intensity of newly incorporated histones (TMR-Star) normalized by the mean of pixel intensity of pre-existing histones (Oregon Green) in the nucleus. Using this approach, we detected H3.1 incorporation during S-G2 phase as it is known as DNA replication-dependent deposition (<strong>Figures 4C-4D</strong>).<br/>  We also illustrate the detection of colocalization in imaging analysis using RGB Profiler (<strong>Figure 5A</strong> and <strong>Box 2</strong>). Here we present the colocalization of newly incorporated macroH2A histone variant at inactivate X chromosomes (Xi) in HEK 293T cells (<strong>Figures 5A-5B</strong>). The pre-existing and newly incorporated SNAP-macroH2A1.2 were labeled as shown in <strong>Figure 4A</strong> and incorporation of SNAP-macroH2A1.2 in Xi during the cell cycle was determined by the colocalization with pre-existing SNAP-macroH2A1.2. As shown in <strong>Figure 5B</strong>, macroH2A1.2 is incorporated into the Xi during the G1 phase.<br/>  Global histone incorporation and dissociation also can be detected by flow cytometry. We demonstrated the SNAP-macroH2A1.2 pulse-chase approach and detected the global SNAP-macroH2A dynamics by flow cytometry as shown in <strong>Figures 5C-5D</strong>. Here we synchronized cells at G1/S border and labeled pre-existing SNAP-macroH2A1.2 with Oregon green, then detected newly incorporated SNAP-macroH2A1.2 every two h after releasing from cell synchronization. Although we determined the cell cycle transition using the Fucci cell cycle sensor (<strong>Figure 5C</strong> [two right panels] and <strong>5D</strong> [top]), other cell cycle indicator such as Hoechst 33342 DNA staining also could be the option (<strong>Figure 6</strong>). A histogram shows the fluorescent intensities of cell populations. The transitions of signal intensity of pre-existing SNAP-macroH2A1.2 (Oregon green, Figure 5C [left] and 5D [middle]) show that the signal intensity decreased when cells entered mitosis but little pre-existing SNAP-macroH2A dissociate from chromatin through the entire cell cycle. The transitions of newly incorporated SNAP-macroH2A1.2 (JF646, <strong>Figure 5C</strong> [second left], and <strong>5D</strong> [bottom]).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902010503_9106.jpg" width="300"/><br/><strong>Figure 4. Detection of global pre-existing and newly incorporated histone H3.1 during specific cell cycle.</strong> <strong>A.</strong> The illustration of labelling histones at S/G2 and G1 phases for the analysis in B-D. (Left panel) To label newly incorporated histones in S/G2 phase, HEK 293T cells stably expressing SNAP-tagged H3 were synchronized at the G1/S phase border by double thymidine block as shown as TH. Pre-existing histones were labeled with SNAP-Oregon Green (green arrow), subsequently blocked using the non-fluorescent SNAP-Block reagent to avoid non-sufficient labeling. The cells were released from synchronization to progress to the G2/M transition until they were synchronized at the G2/M border using RO-3306 (a CDK1/cyclin B1 and CDK1/cyclin A inhibitor, shown as RO), then newly-incorporated SNAP-tagged histones were labeled with SNAP-TMR Star (red arrow). (Right panel) To detect newly incorporated histones in the G1 phase, mitotic cells were collected by shake-off following nocodazole treatment for 12 h and spread onto coverslips (shown as Noc). After 2 h, pre-existing SNAP-H3 were labelled with Oregon Green and treated with the blocking reagent. Cells were released from synchronization and progressed to the G1/S transition using double thymidine block, then newly-incorporated SNAP-tagged histones were labeled with SNAP-TMR Star. After being released from the first synchronization, the cells were also treated with EdU until the second synchronization, allowing cells that had undergone DNA synthesis to be identified. <strong>B.</strong> Representative images of pre-existing and newly-synthesized SNAP-tagged H3.1 in S/G2 or G1 phase. Scale bar = 10 Âµm. Image analysis showing a higher incorporation of histone H3 during the S-G2 phase (<strong>C-D</strong>). <strong>C.</strong> Frequency distribution of histone incorporation rate in S-G2 and G1 cell cycle phase. Histone incorporation rate was calculated by the mean pixel intensity of red (newly incorporated histones) normalized by mean pixel intensity of green (Pre-existing histones) in the nucleus during the S/G2 and in G1 phases. Successful cell synchronization was confirmed by EdU labeling. Hoechst 33342 labeling was performed to define the area of a nucleus in the following analysis. <strong>D.</strong> Quantification of histone incorporation rate during S-G2 and G2 phase. Each dot denotes the incorporation rate as described in <strong>Figure 4C</strong> from single cells. The error bars represent one standard deviation from the number of single cells (n = that indicated on each data set. *<em>P</em> &lt; 0.0001. The P values were determined using two-tailed unpaired <em>t</em>-tests.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902010529_2439.jpg" width="300"/><br/><strong>Figure 5. Detection of local and global pre-existing and newly incorporated histone macroH2A1.2 during specific cell cycle.</strong> <strong>A.</strong> The outline of image analysis using RBG Profiler plugin (<a href="https://imagej.nih.gov/ij/plugins/rgb-profiler.html" target="_blank">https://imagej.nih.gov/ij/plugins/rgb-profiler.html</a>, Color Profiler plugin (<a href="https://imagej.nih.gov/ij/plugins/color-profiler.html" target="_blank">https://imagej.nih.gov/ij/plugins/color-profiler.html</a>) also provides the same functionality) to address the colocalization of newly incorporated histones with pre-existing histones. Cell cycle-specific histone incorporation at specific chromatin architecture can also be addressed with the detection of chromatin structure using IF. Here we show the example detecting colocalization with pre-existing and newly incorporating histones. <strong>B.</strong> the example of image analysis using cell profiler. The left images of pre-existing (Oregon Green: green), newly incorporated (TMR: red) SNAP-tagged macroH2A during S-G2 (upper) and G1 (lower) phase were merged into single images and colocalization on the two inactive X chromosomes (Xi) were determined in HEK 293T cells. In the upper and lower right panels, the signal intensities measured along the white lines in the images of pre-existing and newly incorporated are shown. <strong>C.</strong> Flow cytometric analysis of pre-existing (left: Oregon Green) and newly incorporated macroH2A (second left: JF646), with S-G2 (second right: mCherry-tagged hGeminin) and G1 (right: TagBFP-tagged hCdt1) cell cycle markers derived from Fucci sensor. Histograms indicate the fluorescent intensity at each time point (2-22 hour) after releasing from synchronization at the G1/S border. The cell cycles at each time point were estimated from the Fucci cell cycle sensor. <strong>D.</strong> the median of fluorescent intensity of pre-existing (middle: Oregon Green) and newly incorporated macroH2A (bottom: JF646), with S-G2 and G1 (top: mCherry-tagged hGeminin and TagBFP-tagged hCdt1) cell cycle markers from Flow cytometric analysis are plotted. Error bars indicate 95% confident intervals.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902010643_0246.jpg" width="300"/><br/><strong>Figure 6. Cell cycle determination using Hoechst 33342 staining and Flow Cytometry detection.</strong> <strong>A.</strong> Flow cytometric analysis of cell cycle (left: Hoechst 33342) and pre-existing SNAP-macroH2A (right: Oregon Green) labeled and released from cell synchronization as described in Steps 25-36. Histograms indicate the fluorescent intensity at each time point (2-22 h) after releasing from synchronization at the G1/S border. The cell cycles at each time point could be determined by the intensity transitions of Hoechst 33342 in the cell population. <strong>B.</strong> the median of fluorescent intensity of DNA staining (top: Hoechst 33342) and pre-existing SNAP-macorH2A1.2 (middle: Oregon Green) from flow cytometric analysis are plotted. Error bars indicate 95% confident intervals.</p><p class="pptt" id="biaoti35507">### Recipes</p><ol class="npwtwok"><li>Cloning of SNAP-histone expression vector and its stably expressing cell line<br/>A plasmid that encodes desired histone with SNAP-tag needs to be cloned by general cloning methods. pSNAPf Vector (catalog # <a class="blackNoUnder" target="_blank"> N9183S </a> from NEB) is useful to establish your construct and stably expressing cell line since it contains neomycin resistance gene for efficient drug selection.</li><li>Fixation Buffer<br/>4% (wt/vol) PFA in 1x PBS<br/><em>Note: This solution can be prepared from 32% (wt/vol) PFA. Protected from light for up to 2 years.</em> </li><li>Permeabilization buffer<br/>0.5% Triton in 1x PBS</li><li>Blocking buffer<br/>3% BSA in 1x PBS</li><li>Sorting Buffer<br/>1 mM EDTA<br/>25 mM HEPES pH 7.0<br/>1% FBS (Heat inactivated) in 1x DPBS<br/></li></ol><p class="pptt" id="biaoti35508">### Acknowledgments</p><p class="ppzz">This work was supported by NIH grant R01 DA030317 (JMG). We thank members of the Greally and Singer laboratories for discussions, the Einstein FACS and Genomics cores. We also thank L. Lavis for SNAP-JF646. This protocol was adopted from previous work (Sato <em>et al.</em>, 2019).<br/> <strong> Author contributions statement:</strong> H.S. designed and performed the experiments and analyzed the data. H.S., J.M.G. and R.H.S. wrote the manuscript. J.M.G. and R.H.S. supervised the research.</p><p class="pptt" id="biaoti35509">### Competing interests</p><p class="ppzz">The authors declare no competing financial interests.</p><p class="pptt" id="biaoti35510">### References</p><ol class="npwtwok"><li>Alabert, C., Barth, T. K., Reveron-Gomez, N., Sidoli, S., Schmidt, A., Jensen, O. N., Imhof, A. and Groth, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25792596" target="_blank">Two distinct modes for propagation of histone PTMs across the cell cycle.</a> <em>Genes Dev</em> 29(6): 585-590. </li><li>Alabert, C. and Groth, A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22358331" target="_blank">Chromatin replication and epigenome maintenance.</a> <em>Nat Rev Mol Cell Biol</em> 13(3): 153-167. </li><li>Annunziato, A. T. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24459722" target="_blank">Assembling chromatin: The long and winding road.</a> <em>Biochimic Biophys Acta</em> 1819(3-4): 196-210. </li><li>Balhorn, R., Jackson, V., Granner, D. and Chalkey, R. (1975). <a href="https://www.ncbi.nlm.nih.gov/pubmed/166660" target="_blank">Phosphorylation of the lysine-rich histones throughout the cell cycle.</a> <em>Biochemistry</em> 14(11): 2504-2511.</li><li>Banaszynski, L. A., Allis, C. D. and Lewis, P. W. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21074717" target="_blank">Histone variants in metazoan development.</a> <em>Dev Cell</em> 19(5): 662-674.</li><li>Biegel, J. A., Leslie, D. S., Bigner, D. D. and Bigner, S. H. (1987). <a href="https://www.ncbi.nlm.nih.gov/pubmed/3618122" target="_blank">Hydroxyurea synchronization increases mitotic yield in human glioma cell lines.</a> <em>Acta Neuropathol</em> 73(3): 309-312.</li><li>Boquest, A. C., Day, B. N. and Prather, R. S. (1999). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10084979" target="_blank">Flow cytometric cell cycle analysis of cultured porcine fetal fibroblast cells.</a> <em>Biol Reprod</em> 60(4): 1013-1019.</li><li>Clapier, C. R. and Cairns, B. R. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19355820" target="_blank">The biology of chromatin remodeling complexes.</a> <em>Annu Rev Biochem</em> 78: 273-304. </li><li>Deal, R. B., Henikoff, J. G. and Henikoff, S. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20508129" target="_blank">Genome-wide kinetics of nucleosome turnover determined by metabolic labeling of histones.</a> <em>Science</em> 328(5982): 1161-1164.</li><li>Dunleavy, E. M., Roche, D., Tagami, H., Lacoste, N., Ray-Gallet, D., Nakamura, Y., Daigo, Y., Nakatani, Y. and Almouzni-Pettinotti, G. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19410545" target="_blank">HJURP is a cell-cycle-dependent maintenance and deposition factor of CENP-A at centromeres.</a> <em>Cell</em> 137(3): 485-497.</li><li>Gallo, J. H., Ordonez, J. V., Brown, G. E. and Testa, J. R. (1984). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6585345" target="_blank">Synchronization of human leukemic cells: relevance for high-resolution chromosome banding.</a> <em>Hum Genet</em> 66(2-3): 220-224. </li><li>Geiman, T. M. and Robertson, K. D. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12244565" target="_blank">Chromatin remodeling, histone modifications, and DNA methylation-how does it all fit together?</a> <em>J Cell Biochem</em> 87(2): 117-125. </li><li>Henikoff, S., Henikoff, J. G., Sakai, A., Loeb, G. B. and Ahmad, K. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19088306" target="_blank">Genome-wide profiling of salt fractions maps physical properties of chromatin.</a> <em>Genome Res</em> 19(3): 460-469.</li><li>Henikoff, S. and Smith, M. M. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25561719" target="_blank">Histone variants and epigenetics.</a> <em>Cold Spring Harb Perspect Biol</em> 7(1): a019364. </li><li>Hoebeke, J., Van Nijen, G. and De Brabander, M. (1976). <a href="https://www.ncbi.nlm.nih.gov/pubmed/1267789" target="_blank">Interaction of oncodazole (R 17934), a new antitumoral drug, with rat brain tubulin.</a> <em>Biochem Biophys Res Commun</em> 69(2): 319-324.</li><li> Jackman, J. and O'Connor, P. M. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18228388" target="_blank">Methods for synchronizing cells at specific stages of the cell cycle.</a> <em>Curr Protoc Cell Biol</em> Chapter 8: Unit 8 3.</li><li>Keyomarsi, K., Sandoval, L., Band, V. and Pardee, A. B. (1991). <a href="https://www.ncbi.nlm.nih.gov/pubmed/1711413" target="_blank">Synchronization of tumor and normal cells from G1 to multiple cell cycles by lovastatin.</a> <em>Cancer Res</em> 51(13): 3602-3609.</li><li>Lampkin, B. C., Nagao, T. and Mauer, A. M. (1971). <a href="https://www.ncbi.nlm.nih.gov/pubmed/5286686" target="_blank">Synchronization and recruitment in acute leukemia.</a> <em>J Clin Invest</em> 50(10): 2204-2214. </li><li>Li, C. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21755445" target="_blank">Specific cell cycle synchronization with butyrate and cell cycle analysis.</a> <em>Methods Mol Biol</em> 761: 125-136.</li><li>Martinez-Garcia, E., Popovic, R., Min, D. J., Sweet, S. M., Thomas, P. M., Zamdborg, L., Heffner, A., Will, C., Lamy, L., Staudt, L. M., Levens, D. L., Kelleher, N. L. and Licht, J. D. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20974671" target="_blank">The MMSET histone methyl transferase switches global histone methylation and alters gene expression in t(4;14) multiple myeloma cells.</a> <em>Blood</em> 117(1): 211-220.</li><li>Pesavento, J. J., Yang, H., Kelleher, N. L. and Mizzen, C. A. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17967882" target="_blank">Certain and progressive methylation of histone H4 at lysine 20 during the cell cycle.</a> <em>Mol Cell Biol</em> 28(1): 468-486.</li><li>Petruk, S., Sedkov, Y., Johnston, D. M., Hodgson, J. W., Black, K. L., Kovermann, S. K., Beck, S., Canaani, E., Brock, H. W. and Mazo, A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22921915" target="_blank">TrxG and PcG proteins but not methylated histones remain associated with DNA through replication.</a> <em>Cell</em> 150(5): 922-933.</li><li>Sakaue-Sawano, A., Kurokawa, H., Morimura, T., Hanyu, A., Hama, H., Osawa, H., Kashiwagi, S., Fukami, K., Miyata, T., Miyoshi, H., Imamura, T., Ogawa, M., Masai, H. and Miyawaki, A. (2008). <a href="https://www.ncbi.nlm.nih.gov/pubmed/18267078" target="_blank">Visualizing spatiotemporal dynamics of multicellular cell-cycle progression.</a> <em>Cell</em> 132(3): 487-498. </li><li>Sato, H., Wu, B., Delahaye, F., Singer, R. H. and Greally, J. M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31110057" target="_blank">Retargeting of macroH2A following mitosis to cytogenetic-scale heterochromatic domains.</a> <em>J Cell Biol</em> 218(6): 1810-1823.</li><li>Scharf, A. N., Barth, T. K. and Imhof, A. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19541851" target="_blank">Establishment of histone modifications after chromatin assembly.</a> <em>Nucleic Acids Res</em> 37(15): 5032-5040.</li><li>Schindelin, J., Arganda-Carreras, I., Frise, E., Kaynig, V., Longair, M., Pietzsch, T., Preibisch, S., Rueden, C., Saalfeld, S., Schmid, B., Tinevez, J. Y., White, D. J., Hartenstein, V., Eliceiri, K., Tomancak, P. and Cardona, A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743772" target="_blank">Fiji: an open-source platform for biological-image analysis.</a> <em>Nat Methods</em> 9(7): 676-682. </li><li>Stasevich, T. J., Hayashi-Takanaka, Y., Sato, Y., Maehara, K., Ohkawa, Y., Sakata-Sogawa, K., Tokunaga, M., Nagase, T., Nozaki, N., McNally, J. G. and Kimura, H. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25252976" target="_blank">Regulation of RNA polymerase II activation by histone acetylation in single living cells.</a> <em>Nature</em> 516(7530): 272-275.</li><li>Tillo, D., Mukherjee, S. and Vinson, C. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26910768" target="_blank">Inheritance of cytosine methylation.</a> <em>J Cell Physiol</em> 231(11): 2346-2352. </li><li>Vardabasso, C., Hasson, D., Ratnakumar, K., Chung, C. Y., Duarte, L. F. and Bernstein, E. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23652611" target="_blank">Histone variants: emerging players in cancer biology.</a> <em>Cell Mol Life Sci</em> 71(3): 379-404.</li><li>Vassilev, L. T. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17172841" target="_blank">Cell cycle synchronization at the G2/M phase border by reversible inhibition of CDK1.</a> <em>Cell Cycle</em> 5(22): 2555-2556.</li><li>Wang, T. S. (1991). <a href="https://www.ncbi.nlm.nih.gov/pubmed/1883203" target="_blank">Eukaryotic DNA polymerases.</a> <em>Annu Rev Biochem</em> 60: 513-552.</li><li>Webber, L. M. and Garson, O. M. (1983). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6218868" target="_blank">Fluorodeoxyuridine synchronization of bone marrow cultures.</a> <em>Cancer Genet Cytogenet</em> 8(2): 123-132.</li><li>Whitfield, M. L., Sherlock, G., Saldanha, A. J., Murray, J. I., Ball, C. A., Alexander, K. E., Matese, J. C., Perou, C. M., Hurt, M. M., Brown, P. O. and Botstein, D. (2002). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12058064" target="_blank">Identification of genes periodically expressed in the human cell cycle and their expression in tumors.</a> <em>Mol Biol Cell</em> 13(6): 1977-2000.</li><li>Xu, M., Wang, W., Chen, S. and Zhu, B. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22056817" target="_blank">A model for mitotic inheritance of histone lysine methylation.</a> <em>EMBO Rep</em> 13(1): 60-67. </li><li>Yuan, Z. F., Arnaudo, A. M. and Garcia, B. A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24896311" target="_blank">Mass spectrometric analysis of histone proteoforms.</a> <em>Annu Rev Anal Chem (Palo Alto Calif)</em> 7: 113-128.</li><li>Yunis, J. J., Bloomfield, C. D. and Ensrud, K. (1981). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6941081" target="_blank">All patients with acute nonlymphocytic leukemia may have a chromosomal defect.</a> <em>N Engl J Med</em> 305(3): 135-139.</li><li>Zee, B. M., Britton, L. M., Wolle, D., Haberman, D. M. and Garcia, B. A. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22547680" target="_blank">Origins and formation of histone methylation across the human cell cycle.</a> <em>Mol Cell Biol</em> 32(13): 2503-2514.</li><li>Zink, L. M. and Hake, S. B. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26826795" target="_blank">Histone variants: nuclear function and disease.</a> <em>Curr Opin Genet Dev</em> 37: 82-89. <br/></li></ol>