<body>
            <h1>Fluorescent Polysome Profiling in <em>Caenorhabditis elegans </em></h1>
            <p><strong>Authors:</strong> Dan  Shaffer, Jarod A Rollins</p>
            <h2>Abstract</h2>
            <p>An important but often overlooked aspect of gene regulation occurs at the level of protein translation. Many genes are regulated not only by transcription but by their propensity to be recruited to actively translating ribosomes (polysomes). Polysome profiling allows for the separation of unbound 40S and 60S subunits, 80S monosomes, and actively translating mRNA bound by two or more ribosomes. Thus, this technique allows for actively translated mRNA to be isolated. Transcript abundance can then be compared between actively translated mRNA and all mRNA present in a sample to identify instances of post-transcriptional regulation. Additionally, polysome profiling can be used as a readout of global translation rates by quantifying the proportion of actively translating ribosomes within a sample. Previously established protocols for polysome profiling rely on the absorbance of RNA to visualize the presence of polysomes within the fractions. However, with the advent of flow cells capable of detecting fluorescence, the association of fluorescently tagged proteins with polysomes can be detected and quantified in addition to the absorbance of RNA. This protocol provides detailed instructions on how to perform fluorescent polysome profiling in <em>C. elegans </em>to collect actively translated mRNA, to quantify changes in global translation, and to detect ribosomal binding partners.</p>
            <p><strong>Keywords:</strong> <em>C. elegans</em>, Selective translation, Fluorescence, Polysome profiling, Ribosome</p>
            <h2>Background</h2><p>Multiple actively translating ribosomes bound to the same mRNA transcript are called polysomes. Polysomes can be separated from the other ribosomal forms and unbound mRNA using polysome profiling. Polysome profiling has been a key technique in the field of protein translation. In conjunction with mRNA-seq, polysome profiling allows for transcripts that are regulated by post-transcriptional mechanisms to be detected and quantified (Lan <em>et al.</em>, 2019; Rollins <em>et al.</em>, 2019) by quantitative PCR (Panda<em> et al.</em>, 2017). Polysome profiling has also been used as a readout for global translation rates by quantifying RNA associated with polysomes under different conditions (Merret <em>et al.</em>, 2015; ChassÃ© <em>et al.</em>, 2017). Additionally, polysome profiling has been used to isolate proteins that comprise polysomes and have allowed their contents to be probed using techniques like western blotting (Tiedje<em> et al.</em>, 2012; Jin and Xiao, 2018).<br/><br/>This polysome profiling protocol is different from previously published protocols in two regards; it is developed specifically for<em> C. elegans</em> and it uses fluorescence to detect association of a fluorescently tagged protein with polysomes. Compared to cell culture or harvested tissue, the cuticle of<em> C. elegans </em>requires additional considerations when creating the lysate for polysome profiling.<br/><br/>Existing polysome profiling methods allow for associations between a protein of interest and active ribosomes to be detected. However, this requires the protein to be extracted from the polysome fractions and probed for the protein of interest using western blotting. To resolve differences in associations between a protein of interest and the different forms of ribosomes (40S, 60S, monosomes, polysomes) multiple fractions need to be collected and probed from each sample (Figure 1). Instead, the use of a flow cell capable of detecting fluorescence allows for the presence of a fluorescently tagged protein to be detected and quantified among the entire profile. Thus, associations between the tagged protein and all forms of the ribosome are captured with high resolution and the need for western blotting is eliminated. The following polysome profiling protocol will cover how to prepare, load, centrifuge, and scan sucrose gradients using lysate from <em>C. elegans</em> (Figure 2). Additionally, instructions are given to quantify the proportion of actively translating ribosomes in a polysome profile.<br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902235712_5760.jpg" width="586"/><br/><strong>Figure 1. Polysome profiling overview.</strong> A. Ultra-centrifugation of a cell, tissue, or whole animal lysate allows separation of free ribonucleoprotein complexes (RNPs) from RNA bound to 40S (blue), 60S (red), and 80S monosomes (yellow), and polysomes (green). B. RNA absorbance measurements and fractionation are used to isolate free mRNA and mRNA bound to monosomes and polysomes.<br/><br/><img alt="" class="layerphoto" height="450" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200902/20200902235954_8932.jpg" width="343"/><br/><strong>Figure 2. Protocol Overview.</strong> The protocol begins with preparing sucrose gradients onto which worm lysate will be loaded. Ultracentrifugation of the lysate allows the polysomal RNA to be sedimented away from RNA not bound by polysomes. During the centrifugation, the gradient station can be prepared by flushing the lines and blanking the flow cell. The centrifuged gradients are loaded into the station to read the RNA absorbance and protein fluorescence (optional). The resulting fractions are frozen for subsequent processing. The polysome profiles can be analyzed for changes in the area under the polysome curve.</p>
        </body>

<p class="pptt" id="biaoti35610">## Materials and Reagents</p><ol class="npwtwok"><li>Syringe, 50 ml, Luer (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> S7510-50 </a> , Stability: Room Temperature)</li><li>Ultracentrifuge Tube, 14 x 89 mm, Open-Top Polyclear (Seton Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 7030 </a> , Stability: Room Temperature)</li><li>Tube, Conical Screw Cap, 50 ml (USA Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1500-1811 </a> , Stability: Room Temperature)</li><li>Tube, Microcentrifuge, 1.5 ml (USA Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1615-5599 </a> , Stability: Room Temperature)</li><li>Disposable Sterile Bottle-Top Filter 0.22 Âµm (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 431118 </a> )</li><li><em>Caenorhabditis elegans</em> </li><li>cOmplete Mini, EDTA-free, Protease Inhibitor (Roche, catalog number: <a class="blackNoUnder" target="_blank"> 11836170001 </a> , Stability: &gt; 1 year at 2-8 Â°C)</li><li>Cycloheximide (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> 9880 </a> , Stability: &gt; 2 years at -20 Â°C)</li><li>EGTA (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> E57060-25.0 </a> , Stability: Room Temperature)</li><li>Magnesium Chloride Hexahydrate (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> M35-500 </a> , Stability: Room Temperature)</li><li>Magnesium Sulfate Heptahydrate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> M1880-500G </a> , Stability: Room Temperature)</li><li>Pestle for 1.5 ml Microcentrifuge Tubes (USA Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1615-5599 </a> , Stability: Room Temperature)</li><li>Pipet tip, TipOne Ultra Low Retention Filter Tip, sterile, 200 Âµl (USA Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1180-8810 </a> , Stability: Room Temperature)</li><li>Potassium Phosphate Monobasic (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> P41200-500.0 </a> , Stability: Room Temperature)</li><li>RNase Inhibitor, SUPERaseâ¢In, 2,500 U/20 Âµl (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM2696 </a> , Stability: Unknown)</li><li>RNaseZap (Thermo Fisher Scientific (Invitrogen/Ambion), catalog number: <a class="blackNoUnder" target="_blank"> 9780 </a> , Stability: Unknown)</li><li>Sodium Chloride (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> S23020-5000.0 </a> , Stability: Room Temperature)</li><li>Sodium Deoxycholate (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> 30970-25G </a> , Stability: &gt; 2 years at -20 Â°C)</li><li>Sodium Phosphate Dibasic (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> S3264-500G </a> , Stability: Room Temperature)</li><li>Sucrose (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> S24060-1000.0 </a> , Stability: Room Temperature)</li><li>Tris-HCl (Research Products International, catalog number: <a class="blackNoUnder" target="_blank"> T60050-500.0 </a> , Stability: Room Temperature)</li><li>Triton X-100 (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> T-6878 </a> , Stability: Room Temperature)</li><li>0.5 M EGTA Stock Solution (see Recipes)</li><li>1 M MgSO<sub>4</sub> Stock Solution (see Recipes)</li><li>M9 Buffer (see Recipes)</li><li>Worm Solubilization Buffer (see Recipes)</li><li>10x High-Salt Resolving Buffer (see Recipes)</li><li>Sucrose Solution in High-Salt Resolving Buffer (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35611">### Equipment</p><ol class="npwtwok"><li>Balance, Digital, TS-T Series Advanced Laboratory (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1160T82 </a> )</li><li>Centrifuge, Bench top, Refrigerated, Eppendorf, 5430R (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1227T47 </a> )</li><li>Cordless Homogenizer Unit (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1191H98 </a> )</li><li>Filters, Set of LED Excitation and 2 Emission filters for EGFP fluorescence (BioComp Instruments, catalog number: <a class="blackNoUnder" target="_blank"> FC-2-EGFP </a> )</li><li>Flow cell, Triax, with FC-1 monitor for dual UV and fluorescence scans (BioComp Instruments, catalog number: <a class="blackNoUnder" target="_blank"> FC-2-UV/VIS -260 </a> )</li><li>Fractionator, Piston Gradient/Gradient Station (BioComp Instruments, Model <a class="blackNoUnder" target="_blank"> 152/153 </a> ), alternatively fractionation of samples be achieved using a Density Gradient Fractionation System (Teledyne ISCO, catalog numbers: <a class="blackNoUnder" target="_blank"> 68-1610-010 </a> , <a class="blackNoUnder" target="_blank"> 60-3877-060 </a> , <a class="blackNoUnder" target="_blank"> 60-0084-054 </a> , <a class="blackNoUnder" target="_blank"> 68-1140-006 </a> , <a class="blackNoUnder" target="_blank"> 68-0940-016 </a> )</li><li>Pipettes, Set of 4, ErgoOne, Single Channel (USA Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 7104-2521 </a> )</li><li>14-gauge cannula, 4â (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 8957G54 </a> )</li><li> Platform Rocker, Vari-Mix (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> M79735Q </a> )</li><li>Rotor, TH-641 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> NC1235764 </a> )</li><li>Ultracentrifuge, Sorvall, WX 80 Plus (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> NC1235763 </a> )</li><li>Vortexer V-32 (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1154J82 </a> )</li><li>-80 Â°C freezer (Thermo Scientific Revco Elite Plus, catalog number: <a class="blackNoUnder" target="_blank"> ULT2586-6-D </a> )<br/></li></ol><p class="pptt" id="biaoti35612">### Software</p><ol class="npwtwok"><li>R version 3.6.1, The R Project for Statistical Computing,<a href="https://www.r-project.org/" target="_blank"> https://www.r-project.org/</a> </li><li>The provided R script <em>'</em><em>PolysomeProfileAUC.R</em><em>'</em> </li><li>FlowCell version 1.56, BioComp Instruments, <a href="http://biocompinstruments.com/" target="_blank">http://biocompinstruments.com</a><br/></li></ol><p class="pptt" id="biaoti35613">### Procedure</p><ol class="npwone"><li>Sucrose Gradient Assembly (Figure 3)</li><ol class="npwtwo"><li>To maximize reproducibility, we recommend preparing large batches of the sucrose solutions used to form the gradients. We typically prepare 500 ml of 50% and 5% sucrose solutions in high salt resolving buffer (Recipe 4). Sterile filter the solutions by passing them each through a 0.22 Âµm sterile bottle-top filter using a vacuum. Store aliquots of the sterile solutions in 50 ml conical tubes at 4 Â°C. Sucrose solutions can be stored at 4 Â°C for 3 months without affecting performance.</li><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200903/20200903000129_0986.jpg" width="555"/><br/><strong>Figure 3. Sucrose Gradient Formation.</strong> A. The heavy sucrose solution is layered under the light solution using a cannula. B. After the heavy solution is dispensed, insert the caps at an angle to allow air to escape and to prevent the formation of bubbles.<br/><br/><li>Prior to use, treat centrifuge tubes and cannulas with RNaseZap. Spray the insides of the tubes to ensure the surface is coated. Swirling or vortexing may be necessary to evenly coat the surface. Rinse tubes 3 times with RNase free water and allow to dry before use.</li><li>Allow the 5% and 50% sucrose solutions to come to room temperature before forming the gradient.</li><li>Place each tube in the marker block provided by Biocomp and mark the fill line on the side of the tube.</li><li>Use a 50 ml syringe with a clean cannula to fill the tube just past the fill line with âlightâ 5% sucrose. Take care not to introduce air bubbles into the solution while transferring.</li><li>Use a second syringe and cannula to deposit the âheavyâ 50% sucrose under the 5% solution (Figure 3A).</li><ol class="npwthree"><li>Start with the end of the cannula at the bottom of the tube and slowly raise the tip of cannula up with the level of the 50% solution as it rises.</li><li>Stop dispensing when the interface between the heavy and light layers is at the fill line.</li><li> Remove the cannula quickly and carefully so as not to introduce bubbles or disturb the two layers.</li></ol><li>Cap the centrifuge tubes using the plugs provided by Biocomp. Place the end of the cap with the hole into the tube first and then insert the rest of the cap as shown in Figure 3B.</li><ol class="npwthree"><li>Any excess solution will be pushed through the hole in the cap.</li><li>If the level of the âlightâ solution is below the cap, remove the cap and add more solution to the top of the tube. There should be no air bubble in the tube as it will lead to uneven mixing.</li><li>If bubbles are trapped in the lip of the cap, remove it, wipe it dry, and try inserting it again. </li><li>Place all capped tubes into the magnetic rack and place the rack on the center of the plate.</li></ol><li>Turn on the Biocomp Gradient Station and select GMST to enter the Gradient Master Station mode.</li><li>Ensure the rotating platform is level using a bubble level.</li><li>Press GRAD and then LIST to see pre-programmed gradients.</li><li>Select âShort Sucr 5-50% vwâ gradient (short cap and sucrose) and press USE.</li><li>The platform will tilt, spin, and mix the two fractions to make a perfect linear gradient.</li><li>Inspect the tubes during the process to look for bubbles moving in the tubes, if seen, discard the gradient and start anew.</li><li>The Station will beep when finished. Place the gradients at 4 Â°C to cool down before centrifugation. While the gradients are generally robust, take care to minimize any disturbances to them from bumping or tilting the tubes.<br/><br/></li></ol><li>Worm lysis</li><ol class="npwtwo"><li>Prepare fresh Worm Solubilization Buffer as described in Recipe 4. For six profiles we typically prepare 10 ml.</li><li> For each profile, aliquot 350 Âµl of solubilization buffer into a 1.5 ml micro-centrifuge tube.</li><li>Add 5 Âµl of SUPERaseâ¢In to each aliquot. Place the tubes on ice in preparation for lysis. </li><li>This protocol requires 150 Âµl of densely packed, day-one adult synchronized <em>C. elegans </em>per gradient for good results. This is roughly 15,000 worms for each gradient performed. This number of worms can be obtained from cultivating worms on 4-6 100 mm NMG plates. Worms should not be starved/burrowed for profiling. See the Bio-protocol âSynchronization of Wormâ (He) to obtain age-synchronized adult worms. </li><li> For each condition/strain, wash worms off the plates using 15 ml total of M9 buffer in a 50 ml conical tube.</li><ol class="npwthree"><li>Allow the worms to gravity settle in the 50 ml conical for 2-5 min.</li><li>When most worms have settled to the bottom, pipette 1 ml of worm slurry from the bottom of the conical and into a 1.5 ml microcentrifuge tube.</li><li>Gravity settle again and remove as much M9 as possible and wash with 1 ml fresh M9.</li><li>Repeat wash twice more or until no bacteria is present, typically three washes in total.</li><li>After the final wash, remove as much M9 as possible and then briefly spin the tube using a tabletop centrifuge at no more than 2,000 <em>x g</em>.</li><li>Directly after centrifugation, the worms will temporarily be packed at the bottom of the tube. Quickly pipette away the remaining M9 (around 20 Âµl) to concentrate the worms further.</li><li>Using a low-retention filter tip with 3 mm of the end cut off, remove 150 Âµl of the concentrated worm slurry and transfer it into the 350 Âµl aliquot of worm solubilization buffer.</li><li>Pipette up and down slowly to ensure that no worms remain sticking to the pipette tip.</li></ol><li>Immediately homogenize the worms on ice with a disposable plastic pestle for 1 min. Take care not to create too much foam as the homogenate will spill out of the tube. We typically use 60 strokes of the pestle in the tube during the homogenization. After homogenization, add an additional 200 Âµl of worm solubilization buffer to the tube. Let the homogenate solubilize for a 30 min at 4 Â°C using a tube rotator to mix the samples. Alternatively, keep tubes on ice and invert by hand to mix them every 10 min.</li><li>Spin lysate at 14,000<em> x g</em> at 4 Â°C for 5 min to pellet debris and transfer the supernatant to prechilled tube. </li><li>Remove 100 Âµl of lysate supernatant and place it into a second pre-chilled tube and keep on ice until the fractions have been collected. This will be used as a matched control âtotal RNAâ sample for the lysate. This aliquot will be used for quantification of the RNA present in the lysate. It can also be used for RNA isolation if transcripts in the total and polysomal fractions are to be compared via qPCR or mRNA-sequencing.<br/><br/></li></ol><li>Loading lysate onto the gradients</li><ol class="npwtwo"><li>Load the gradients as close to the ultracentrifuge as possible to minimize mixing during transport. </li><ol class="npwthree"><li>Gradients must be balanced within 0.001 g of each other. </li><li>Take care to remove ice, water, and condensate from tubes and buckets immediately before weighing. </li><li>All rotor buckets must be used in the rotor even if empty.</li><li> Pre-cool the centrifuge to 4 Â°C and to ensure that it pulls a proper vacuum.</li></ol><li>Prepare the gradients for loading by removing 300 Âµl from the top of the gradient with a pipette. This is to ensure enough room for addition of the sample.</li><ol class="npwthree"><li>Tare the weight of the holder, tube, bucket and lid together.</li><li>Next, take the gradient that will be centrifuged opposite of the first sample and remove the gradient from the top using a pipette until it weighs the same as the first tube. Optimally, this would be 300 Âµl but bucket and tube weights can vary.</li><li>Repeat this process for every pair of gradients to be run.</li><li>It is not recommended to load more than 300 Âµl of sample (<em>e.g.</em>, no more than a 2.5 mm thick layer) to keep resolution of the profile as high as possible. However, there are cases where loading more is necessary to obtain sufficient signal. Up to 500 Âµl of sample has been loaded without a drastic loss of resolution in the profile.</li><li>We consistently achieve good results loading a total mass of 200-300 Âµg of RNA as estimated from the lysate prepared in Step B6.</li><li>Place the gradient into a rotor bucket held by a multi-use tube block and place on the scale.</li></ol><li>Return the first gradient to the balance and load 300 Âµl of lysate.</li><ol class="npwthree"><li>This is achieved using a 200 Âµl tip and pipetting 150 Âµl twice.</li><li>For best results, hold pipette at about 45Â° and place the tip under the meniscus and lift it up with surface tension as shown in Figure 4.</li><li> Dispense the lysate slowly. You should see it form a discrete layer on top (akin to an oil slick). When the 300 Âµl is loaded, tare the weight on the balance. <br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200903/20200903000402_5278.jpg" width="348"/><br/><strong>Figure 4. Loading lysate onto the sucrose gradient.</strong> With the gradient in the swinging bucket slowly pipette the lysate on top of the gradient at a 45Â° angle.<br/><br/></li></ol><li>Like before, place the opposing gradient on the balance and load sample until the tared weight is back to 0.000.</li><ol class="npwthree"><li>Again, in a perfect world this should be 300 Âµl, the same volume as loaded in the first tube. </li><li>Repeat for all sets of opposing tubes.</li><li>Once all gradients are loaded, screw on the caps to the rotor bucket, hang the buckets in the rotor, and place the rotor into the ultracentrifuge.</li><li>Be sure to follow all the manufactureâs guidelines for ultracentrifuge use.</li></ol><li>Centrifuge at 247,103.6<em> x g</em> (38,000 rpm using the TH-641 rotor).<br/><div style="white-space:nowrap;"><span style="font-size:10pt;"></span> </div></li><ol class="npwthree"><li>Set acceleration and deaccelerating each set to 9 (the maximum). </li><li>Over two hours a total of 1.14E11 Ï<sup>2</sup>T force will be applied to the gradients.<br/><div style="white-space:nowrap;"><br/></div></li></ol></ol><li>Preparing the Gradient Station</li><ol class="npwtwo"><li>Start preparing the Gradient Station Fractionator (takes &gt; 30 min) while waiting for the centrifuge run to complete.</li><li>Turn on station and select SCAN from the menu.</li><ol class="npwthree"><li>Turn on laptop and open the Triax software.</li><ol class="npwfour"><li>Select the user profile and then fill in the experimental details.</li><li>Select UV OD with Single Fluorescence scan.</li></ol><li>Fill syringe reservoir with filtered water and place a beaker under the pump tubing outlet.<br/></li></ol><li>Make sure fraction collector arm (if used) is above the waste tray and that there is a beaker for the flow through to collect.</li><li>Prime the water pump.</li><ol class="npwthree"><li>Flip the toggle switch down on the right side and then press the RINSE key to engage pump.</li><li>Once a stream of water exits, return the toggle switch to the up position.</li></ol><li>Press RINSE again to clean tubing with at least 10 ml of water. </li><li>Open brass valve on the piston arm and dry tubing with 10 s of air from pump, then close brass valve. </li><li>Turn on the fraction collector if automatic collection is to be used.</li><ol class="npwthree"><li>If the fraction collector is to be used, choose SCAN SETUP to setup the fractions desired. </li><li>Otherwise select GO TO GRAPH in the software.</li></ol><li>You will be prompted to Zero calibrate the flow cell detectors.</li><ol class="npwthree"><li>Both UV and Fluor can be blanked at the same time. </li><li>Use a 50 ml syringe with a luer lock adaptor to manually push water though the piston and flow cell. </li><li>Zero the detector while water is in flow cell.</li></ol><li>After blanking, push water through the system at a rate of a one drop per second.</li><ol class="npwthree"><li>Use the graph to ensure that the readings are constant and that there are no spikes.</li><li>Ideally the absorbance should be around 0.00012 arbitrary units (AU).</li><li>If there is a large flux in the absorbance, rinse with another 10 ml of water to remove debris and blank again.</li></ol><li>Remove adaptor from piston and replace with piston tip, take care not to overtighten.</li><li>Press the AIR key on the Station to clear out the flow cell. The absorbance should rise to about 0.5 AU.<br/></li></ol><br/><li>Scanning and Fractionating Gradients</li><ol class="npwtwo"><li>Insert the top of the gradient tube into the tube holder lid.</li><ol class="npwthree"><li>Then place the tube with the attached lid into the holder.</li><li>Take care to align the groves of the lid with the screws in the holder and twist the cap to lock it into place.</li><li>Lock the holder into place on the Station by sliding it over the bayonet (under the piston) and then rotating it 90Â° to lock into place.</li><li>For further detail on these the steps, the reader is directed to the Biocomp fractionator manual.</li></ol><li>Click START RUN.</li><ol class="npwthree"><li>The scan parameters will be shown as a reminder.</li><li> Click OK.</li></ol><li>You will be reminded to rinse out the remainder of the previous sample from the system and to air dry the lines leading to the flow cell.</li><ol class="npwthree"><li>If this is the first tube of the run, this has been done already.</li><li>Click OK and the piston will start to descend.</li></ol><li>Upon reaching the start of the gradient, the system will detect the meniscus and automatically start recording.</li><ol class="npwthree"><li>The graph will reset and will plot the absorbance in real time as the sample passes through the flow cell.</li><li>To see fluorescence readings, click the corresponding box in the software.</li></ol><li>The graph will show a green line lagging behind the UV signal (black line) which represents when the sample will come out of the tube (based on the tube length entered in the SCAN SETUP). Use this for manual fraction collection.</li><li>The first peak will be the free ribonucleoproteins, followed by the 40S subunit, the 60S subunit, and then the monosomes (see Figure 1). As the monosomal peak declines, be prepared to start collecting the polysome fraction.</li><li>Collect the polysome fraction into a 15 ml conical on ice.</li><ol class="npwthree"><li>Start collecting when the green line approaches the first peak of the disomes.</li><li>This polysome peak starts at about 26.5 mm into the gradient.</li><li>Continue collecting until the end of the gradient.</li><li>The total volume will be ~5 ml.</li></ol><li>Once the piston reaches the bottom of the tube it will automatically ascend to the starting position.</li><li>You will be prompted to press the AIR button to blow out the remaining sample in the flow cell. Click OK when the line is cleared from any liquid.</li><li>You will be asked to save your run.<br/><ol class="npwthree"><li>Click YES and you can change the file name and location if desired.</li><li>If NO is clicked or an error occurs, the file can typically be recovered by selecting File &gt; Rescue Lost Data.</li><li>The next tube can be run with the same settings by clicking NEW RUN? in the software.<br/><div style="white-space:nowrap;"><ol class="npwfour"><li>You will be given the option to re-zero the detector, but this is typically only needed if debris was introduced during the run.</li><li>Debris may show up as spikes in the absorbance.</li><li> If debris was introduced, rinse the system with 10 ml of water and repeat Steps D6-D11 of preparing the gradient station.<br/><div style="white-space:nowrap;"><br/></div></li></ol></div></li></ol></li></ol><li>Sample storage</li><ol class="npwtwo"><li>Once fractions are collected keep them on ice for further processing.</li><li>For long term storage keep fractions at -80 Â°C.</li></ol></ol><p class="pptt" id="biaoti35614">### Data analysis</p><p class="ppzz">The absorbance and fluorescence data are saved as a comma delimited file (.csv) for each profile. This data can be used to plot the profiles and to quantify the proportion of actively translating ribosomes. Use the provided R function polyproAUC() (<a href="https://os.bio-protocol.org/attached/file/20200820/3742%20Supplemental%20File%201%20PolysomeProfileAUC.zip">Supplemental File 1</a>) to determine the area under the curve for the peaks corresponding to the 40S, 60S, 80S monosomes, and polysome peaks. In the example below, we performed polysome profiling using <em>C. elegans</em> strain CZ18550 (Noma <em>et</em> <em>al</em>., 2017) in which the large ribosomal protein RPL-29 has been GFP-tagged. Two example files are provided 'rpl-29_20C.csv' (<a href="https://os.bio-protocol.org/attached/file/20200820/3742%20Supplemental%20File%202%20rpl%2029%2020C.zip">Supplemental File 2</a>) and 'rpl-29_35C.csv' (<a href="https://os.bio-protocol.org/attached/file/20200820/3742%20Supplemental%20File%203%20rpl%2029%2035C.zip">Supplemental File 3</a>). The examples will be used to examine the effect of heat shock on the area under the curve (AUC) of the polysome and monosome fractions. Addionally, this experiment demonstrates that RPL-29 associates with the 60S large ribosomal subunit and polysomes but not the 40S subunit as predicted (Figure 5). The following directions assumes that the csv files are in your working directory. It they are not, make sure to include their path in the following commands.<br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200903/20200903000539_1686.jpg" width="875"/><br/><strong>Figure 5. Fluorescent polysome profiles of <em>C. elegans </em>strain CZ18550 (juSi123 [rpl-29::GFP] II.) under control (left) or heat shock (right) conditions.</strong> Absorbance of RNA is plotted in black, fluorescence of RPL-29::GFP is in red. After 1h heat shock there is a shift in signal from the polysomes to monosomes, indicative of a decrease in global translation rates. The signal from RPL-29::GFP is absent from the 40S peak but present in the 60S peak and polysome. This result confirms association of RPL-29 with the large ribosomal subunit and with active ribosomes.</p><br/><ol class="npwone"><li>Load the function</li><ol class="npwtwo"><li>Type <em>source</em>(<em>'PolysomeProfileAUC.R'</em>) into the R console and press enter.</li><li>The package âMESSâ will be installed the first time the script is loaded.<br/><div style="white-space:nowrap;"><span style="font-size:10pt;"></span> </div></li></ol><br/><li>Run the function <em>polyproAUC()</em> using data from the control example</li><ol class="npwtwo"><li>Type<em> polyproAUC</em>(<em>'rpl-29_20C.csv'</em>) into the R console and press enter.</li><li>The plotted profile will appear as in Figure 5 and the cursor should appear as a crosshair.<br/><div style="white-space:nowrap;"><br/></div></li></ol><li>You will now click on the plot with the mouse cursor to instruct the function where each of the peaks are</li><ol class="npwtwo"><li>First click at the start of the 40S peak.</li><li>Second click is at the bottom of the base line in between the 40S and the 60S peaks.</li><li>Third click is at the bottom of the base line in between the 60S and 80S peaks.</li><li>Forth click is at the bottom of the base line in between the 80S and the start of the polysomes.</li><li>Fifth click is at the bottom of the base line in between the 5<sup>th</sup> and 6<sup>th</sup> polysome peaks.</li><ol class="npwthree"><li>This will divide the peak into mRNA with 2-6 ribosomes bound and those with &gt; 6 bound.</li><li>The former will be reported as the âLowâ polysome fraction in the output as these mRNAs are translated with lower efficiency due to fewer bound ribosomes.</li><li>The latter will be reported as the âHighâ polysome fraction in the output as they are translated with higher efficiency due to more bound ribosomes.</li></ol><li>The sixth and final click is at the end of the polysome peak.</li><li>If you misclick, you will have to repeat the process.<br/><br/></li></ol><li>After the final mouse click, the console will report the results<br/><ol class="npwtwo"><li>The position in the gradient represented by each click is provided for reference.</li><li>For the RNA absorbance channel, the AUC for the 40S, 60S, 80S, Low, High, and total polysomes peaks are given as a percent of the total area under the curve.</li><li>The fluorescence channel for the 40S, 60S, 80S, and polysomes peaks are given as a percent of the total area under the curve.</li><ol class="npwthree"><li>If the function is run with the âExcelâ option,<em> e.g., polyproAUC</em>(<em>'rpl-29_20C.csv',âExcelâ</em>) the results are instead placed in the clipboard to be pasted into a spreadsheet.</li><li>The calculated percent under the curve should be similar to those calculated in Table 1.</li></ol><li>Differences in the percent area under the curve between conditions can be compared using a t-test on three on more biological replicates.<br/><div style="white-space:nowrap;"><br/></div><strong>Table 1. Percent area under the curve for the different ribosomal forms from control (20 Â°C) and heat shocked (35 Â°C) RPL-29::GFP <em>C. elegans</em>.</strong> The percent is given for the RNA absorbance channel (RNA) and the GFP fluorescence channel. Profiles are the same as depicted in Figure 5. The polysomes decrease from 70.2% of the area under the curve to 47.1% due to heat shock. The starting position in the gradient (in mm) for each form is provided except for the Polysome which is the position at the end. LowâmRNA with 2 to 6 ribosomes attached. HighâmRNA with &gt; 6 ribosomes bound.<br/></li></ol><img alt="" class="layerphoto" height="250" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200903/20200903001229_4960.jpg" width="731"/><br/></li></ol><p class="pptt" id="biaoti35615">### Notes</p><br/><ol class="npwtwok"><li>A common error is not culturing enough worms to have 150 Âµl pellet per profiles. It is preferable to have too many worms than not enough.</li><li>Conducting profiles with worms older than day three of adulthood is possible but as translation rates typically decrease with age more starting material may be needed for a strong signal in the profiles. For older worms with tougher cuticles, we also recommend increasing the grinding time with the pestle by 10 s and the time left to solubilize on ice by 10 min.</li><li>If the RNP signal is too great it can overlap with the 40S peak. If quantification of the 40S peak is desired in this case, we recommend centrifuging for an additional 30 mins or reducing the amount of lysate applied to the gradient.</li><li>The end of the polysome peak is where the signal starts to flatline near the end of the gradient.<br/></li></ol><p class="pptt" id="biaoti35616">### Recipes</p><ol class="npwtwok"><li>0.5 M EGTA Stock Solution (100 ml)</li><ol class="npwthree"><li>Add 19.0 g EGTA to 80 ml deionized H<sub>2</sub>O</li><li>While stirring, monitor the pH as NaOH pellets are slowly added to achieve pH 8.0</li><li>Add deionized H<sub>2</sub>O to 100 ml final volume</li><p class="ppzz" style="margin-left:-20px;"><i>Note: EGTA will not go into solution until ~pH 7.5.</i> </p><li>Store at room temperature<br/></li></ol><li>1 M MgSO<sub>4</sub> Stock Solution</li><ol class="npwthree"><li>Weigh 246.48 g of Magnesium sulfate heptahydrate (MgSO<sub>4</sub>â¢7H<sub>2</sub>O)</li><li>Add deionized H<sub>2</sub>O to 1.0 L</li><li>Sterilize by autoclaving</li><li>Store at room temperature</li></ol><li>M9 Buffer</li>22 mM KH<sub>2</sub>PO<sub>4</sub><br/>42 mM Na<sub>2</sub>HPO<sub>4</sub><br/>86 mM NaCl<br/>1 mM MgSO<sub>4</sub><br/><br/><strong>For 1 liter of M9 buffer</strong><br/>3.0 g KH<sub>2</sub>PO<sub>4</sub><br/>6.0 g Na<sub>2</sub>HPO<sub>4</sub><br/>5.0 g NaCl<br/><div style="white-space:nowrap;"><ol class="npwthree"><li>Add deionized H<sub>2</sub>O to 999 ml</li><li>Sterilize by autoclaving</li><li>After solution cools down, aseptically add 1 ml of 1 M MgSO<sub>4</sub> Stock Solution</li><li> Store at room temperature</li></ol></div><li>Worm Solubilization Buffer</li>50 mM Tris-HCl (pH 8.0)<br/>300 mM NaCl<br/>10 mM MgCl<sub>2</sub><br/>1 mM EGTA<br/>0.1 mg/ml cycloheximide<br/>1% Triton X-100<br/>0.1% sodium deoxycholate<br/>1x cOmplete EDTA-free protease inhibitor<br/>125 U/Âµl SUPERaseâ¢In<br/>A stable stock consisting of 50 mM Tris-HCl (pH 8.0), 300 mM NaCl, 10 mM MgCl<sub>2</sub>, 1 mM EGTA should be assembled, sterile filtered and stored at room temperature. The remaining temperature-sensitive reagents are added just before use.<br/><br/><strong>For 250 ml of Worm Solubilization Buffer stable stock</strong><br/>17.4 ml of 4 M NaCl<br/>12.5 ml of 1 M Tris-HCl (pH 8.0)<br/>2.5 ml of 1 M MgCl<sub>2</sub><br/>25 ml of 10 mM EGTA<br/>Deionized H<sub>2</sub>O to 250 ml<br/>Store at room temperature<br/>Add the following temperature-sensitive reagents to 10 ml of stable stock before use<br/>1 mg cycloheximide<br/>100 Âµl of Triton X-100<br/>100 Âµl of 10% sodium-deoxycholate<br/>One cOmplete Mini, EDTA-free, Protease Inhibitor tablet<br/>Use immediately, do not store overnight<br/>Add 5 Âµl SUPERaseâ¢In RNase-Inhibitor to each 350 Âµl aliquot of Worm Solubilization Buffer as directed in the protocol<li>10x High-Salt Resolving Buffer</li>250 mM Tris-HCl (pH 8.0)<br/>1.4 M NaCl<br/>100 mM MgCl<sub>2</sub><br/><br/><strong>For 100 ml of 10x High-Salt Resolving Buffer</strong><br/>33.6 ml of 4 M NaCl<br/>25 ml of 1 M Tris-HCl (pH 8.0)<br/>10 ml of 1 M MgCl<sub>2</sub><br/>31.4 ml of Deionized H<sub>2</sub>O<span style="font-size:16px;white-space:nowrap;"></span> <li>Sucrose Solution in High-Salt Resolving Buffer<br/><b>For 500 ml of 50% Sucrose Solution in High-Salt Resolving Buffer:</b><br/><ol class="npwthree"><li>Add 50 ml 10x High-Salt Resolving Buffer to a 500 ml graduated bottle</li><li>Add 250 g sucrose</li><li>Add enough deionized H<sub>2</sub>O at 35-40 Â°C to achieve approximately 450 ml total volume</li><li>Tightly cap the bottle and mix until the sucrose has fully dissolved. Further heating at 37 Â°C may be necessary to ensure all sucrose is fully dissolved in solution.</li><li>Adjust the volume to 500 ml with deionized H<sub>2</sub>O</li><li>Filter the solution through a 0.22 Î¼m bottle top filter into an autoclaved 500 ml bottle</li><li>Store 50 ml aliquots at 4 Â°C<br/><div style="white-space:nowrap;"><br/><p class="ppzz" style="margin-left:-20px;"><b>For 500 ml of 5% Sucrose Solution in High-Salt Resolving Buffer:<br/></b> </p></div></li></ol><ol class="npwthree"><li>Add 50 ml 10x High-Salt Resolving Buffer to a 500 ml graduated bottle</li><li>Add 25 g Sucrose</li><li>Add enough deionized H<sub>2</sub>O at 35-40 Â°C to achieve approximately 450 ml total volume</li><li>Tightly cap the bottle and mix until the sucrose has fully dissolved</li><li>Adjust the volume to 500 ml with deionized H<sub>2</sub>O</li><li>Filter the solution through a 0.22 Î¼m bottle top filter into an autoclaved 500 ml bottle</li><li>Store 50 ml aliquots at 4 Â°C</li></ol></li></ol><p class="pptt" id="biaoti35617">### Acknowledgments</p><p class="ppzz">Research reported in this publication was supported by an Institutional Development Award (IDeA) from the National Institute of General Medical Sciences of the National Institutes of Health under grant numbers P20GM103423 and P20GM104318. The strain CZ185550 was provided by the CGC, which is funded by NIH Office of Research Infrastructure Programs (P40OD010440). This protocol for<em> C. elegans</em> is based on that from Laberge<em> et al. </em>(2015) for cultured cells.</p><p class="pptt" id="biaoti35618">### Competing interests</p><p class="ppzz">The authors have no competing interests to declare.</p><p class="pptt" id="biaoti35619">### References</p><ol class="npwtwok"><li>ChassÃ©, H., Boulben, S., Costache, V., Cormier, P. and Morales, J. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28180329" target="_blank">Analysis of translation using polysome profiling.</a> <em>Nucleic Acids Res</em> 45(3): e15.</li><li>He, F. (2011). <a href="https://doi.org/10.21769/BioProtoc.56" target="_blank">Synchronization of Worm.</a><em>Bio-101: </em>e56.</li><li>Jin, H. Y. and Xiao, C. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29224064" target="_blank">An integrated polysome profiling and ribosome profiling method to investigate in vivo translatome.</a> <em>Methods Mol Biol</em> 1712: 1-18.</li><li>Laberge, R. M., Sun, Y., Orjalo, A. V., Patil, C. K., Freund, A., Zhou, L., Curran, S. C., Davalos, A. R., Wilson-Edell, K. A., Liu, S., Limbad, C., Demaria, M., Li, P., Hubbard, G. B., Ikeno, Y., Javors, M., Desprez, P. Y., Benz, C. C., Kapahi, P., Nelson, P. S. and Campisi, J. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26147250" target="_blank">MTOR regulates the pro-tumorigenic senescence-associated secretory phenotype by promoting IL1A translation.</a> <em>Nat Cell Biol </em>17(8): 1049-1061. </li><li>Lan, J., Rollins, J. A., Zang, X., Wu, D., Zou, L., Wang, Z., Ye, C., Wu, Z., Kapahi, P., Rogers, A. N. and Chen, D. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31340143" target="_blank">Translational regulation of non-autonomous mitochondrial stress response promotes longevity.</a> <em>Cell Rep</em> 28(4): 1050-1062 e1056.</li><li>Merret, R., Nagarajan, V. K., Carpentier, M. C., Park, S., Favory, J. J., Descombin, J., Picart, C., Charng, Y. Y., Green, P. J., Deragon, J. M. and Bousquet-Antonelli, C. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25845591" target="_blank">Heat-induced ribosome pausing triggers mRNA co-translational decay in <em>Arabidopsis thaliana</em>.</a> <em>Nucleic Acids Res</em> 43(8): 4121-4132. </li><li>Noma, K., Goncharov, A., Ellisman, M. H. and Jin, Y. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28767038" target="_blank">Microtubule-dependent ribosome localization in <em>C. elegans </em>neurons.</a> <em>Elife</em> 6: 26376.</li><li>Panda, A. C., Martindale, J. L. and Gorospe, M. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28516123" target="_blank">Polysome fractionation to analyze mRNA distribution profiles.</a><em> Bio-protocol</em> 7(3): e2126.   </li><li>Rollins, J. A., Shaffer, D., Snow, S. S., Kapahi, P. and Rogers, A. N. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31253655" target="_blank">Dietary restriction induces posttranscriptional regulation of longevity genes.</a> <em>Life Sci Alliance</em> 2(4).</li><li>Tiedje, C., Ronkina, N., Tehrani, M., Dhamija, S., Laass, K., Holtmann, H., Kotlyarov, A. and Gaestel, M. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23028373" target="_blank">The p38/MK2-driven exchange between tristetraprolin and HuR regulates AU-rich element-dependent translation.</a> <em>PLoS Genet</em> 8(9): e1002977.<br/></li></ol>