<body>
            <h1>Ratiometric Measurement of Protein Abundance after Transient Expression of a Transgene in <em>Nicotiana benthamiana</em></h1>
            <p><strong>Authors:</strong> Aashima  Khosla, David C. Nelson</p>
            <h2>Abstract</h2>
            <p>Ratiometric reporters are tools to dynamically measure the relative abundance of a protein of interest. In these systems, a target protein fused to a fluorescent or bioluminescent reporter is expressed with fixed stoichiometry to a reference protein fused to a second reporter. Both fusion proteins are encoded on a single transcript but are separated during translation by a 2A âself-cleavingâ peptide. This approach enables changes in the relative abundance of a target protein to be detected sensitively, reducing variability in expression of the ratiometric reporter transgene that may occur across different tissues or transformation events. We recently developed a set of Gateway-compatible plant transformation vectors termed pRATIO that combine a variety of promoters, fluorescent and bioluminescent reporters, and 2A peptides derived from foot-and-mouth disease virus. Here, we describe in detail how to use the dual-fluorescent ratiometric reporter pRATIO3212 to examine the relative abundance of a target protein after transient expression in <em>Nicotiana benthamiana</em> leaves. For this example, we analyze degradation of the SUPPRESSOR OF MAX2 1 (SMAX1) protein from <em>Arabidopsis thaliana</em> in response to treatments with karrikins and <em>rac</em>-GR24. This protocol provides a simple, rapid, and readily scalable method for <em>in vivo</em> analysis of relative protein abundance in <em>Agrobacterium</em>-infiltrated <em>Nicotiana</em> leaf tissues.</p>
            <p><strong>Keywords:</strong> Ratiometric reporter, <em>Nicotiana benthamiana</em>, Transient expression, 2A peptide, Fluorescence, Protein quantitation</p>
            <h2>Background</h2><p>Karrikins (KARs) are butenolide compounds found in smoke that stimulate seed germination and enhance seedling photomorphogenesis of <em>Arabidopsis thaliana</em> (Flematti, 2004; Nelson <em>et al.</em>, 2009, 2010 and 2012). KAR responses in <em>Arabidopsis</em> require the Î±/Î²-hydrolase KARRIKIN INSENSITIVE2/HYPOSENSITIVE TO LIGHT (KAI2/HTL) (Sun and Ni, 2011; Waters <em>et al.</em>, 2012). KAI2 can bind KAR<sub>1</sub> <em>in vitro</em> and is thought to function as a KAR receptor (Guo <em>et al.</em>, 2013). KAI2 works with the F-box protein MORE AXILLARY GROWTH2 (MAX2) to mediate KAR responses, likely through polyubiquitination and degradation of SUPPRESSOR OF MAX2 1 (SMAX1) (Nelson <em>et al.</em>, 2011; Stanga <em>et al.</em>, 2013; Waters <em>et al.</em>, 2017). This signaling mechanism is highly similar to that of the plant hormone strigolactone (SL). In SL signaling, DWARF14 (D14)/DECREASED APICAL DOMINANCE2 (DAD2), which is an ancient paralog of KAI2, works with MAX2 to target a subset of SMAX1-LIKE proteins for degradation (Hamiaux <em>et al.</em>, 2012; Waters <em>et al.</em>, 2012; Jiang <em>et al.</em>, 2013; Zhou <em>et al.</em>, 2013; Soundappan <em>et al.</em>, 2015; Wang <em>et al.</em>, 2015). Interestingly, KAI2 and D14 are both responsive to <em>rac</em>-GR24, a commonly used racemate of synthetic SL analogs. However, they show different preferences for the two enantiomers in this mixture, only one of which has a stereochemical configuration that mimics natural SLs (Scaffidi <em>et al.</em>, 2014; Waters <em>et al.</em>, 2015; Flematti <em>et al.</em>, 2016).<br/><br/>In addition to mediating KAR responses, there is growing evidence that the primary role of KAI2 is the recognition of an unknown endogenous signal known as KAI2 ligand (KL) (Conn and Nelson, 2015). Identification of KL will require a highly specific assay for the activation of KAI2. Recently, we found evidence that SMAX1 degradation occurs after KAR and <em>rac</em>-GR24 treatment, and this response requires MAX2 and KAI2 (Khosla <em>et al.</em>, 2020b). This observation led us to develop a bioassay for SMAX1 degradation that is appealing as a direct and specific readout of KAI2 activation.<br/><br/>We generated a series of ratiometric reporter vectors (pRATIO) that can be used to assess protein abundance <em>in vivo</em> by monitoring the relative abundance of two co-expressed fluorescent or bioluminescent reporters, one of which is fused to a target protein of interest (Khosla <em>et al.</em>, 2020a). Rather than express a pair of target and reference genes from two promoters on a single vector or two co-transformed vectors, in the pRATIO system the target and reference genes are transcribed on the same mRNA. Importantly, pRATIO vectors can normalize for differences in transformation efficiency or transgene expression across samples by encoding a 2A ribosomal skipping peptide from foot-and-mouth disease virus (FMDV) between the target and reference genes. During translation, the nascent 2A peptide blocks the ribosomal exit channel. This disrupts formation of the bond between the C-terminal Gly and Pro residues of the 2A peptide. Translation can resume on the mRNA, producing two proteins in near stoichiometric ratios (Luke and Ryan, 2018).<br/><br/>Here we present a detailed protocol for using the pRATIO system. As an example, we monitor KAR- and <em>rac</em>-GR24- induced proteolysis of <em>Arabidopsis</em> SMAX1 that has been transiently expressed in <em>Nicotiana benthamiana</em>.</p>
        </body>

<p class="pptt" id="biaoti35568">## Materials and Reagents</p><ol class="npwtwok"><li>Corning<sup>TM</sup> 96-Well Black Polystyrene Microplates (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 07-200-590 </a> )</li><li>5 ml Centrifuge tubes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 10002-731 </a> )</li><li>Polystyrene culture tubes (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 60818-703 </a> )</li><li>1 ml Norm-Ject Syringes (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 1481725 </a> ) </li><li>50 ml Falcon tubes (Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 28-108 </a> )</li><li>24-Well, Flat Bottom plates (Olympus, catalog number: <a class="blackNoUnder" target="_blank"> 25-102 </a> )</li><li>96-Well Microplates, Clear (Greiner Bio-One, catalog number: <a class="blackNoUnder" target="_blank"> 655801 </a> )</li><li>Potting soil (Professional Growing Mix, sungro Horticulture) supplemented with Gnatrol WDG, Marathon (imidacloprid), and Osmocote 14-14-14 fertilizer</li><li>Wild type <em>Nicotiana benthamiana</em> seeds (seeds kindly provided by Dr. Martha Orozco-Cardenas; Plant Transformation Research Center)</li><li><em>Agrobacterium tumefaciens</em> strain GV3101 strain with helper plasmid pMP90 (Koncz and Schell, 1986; kindly provided by Dr. Meng Chen, University of California, Riverside)</li><li>pRATIO3212-SMAX1 plasmid (Khosla <em>et al.</em>, 2020b)</li><li><em>Agrobacterium tumefaciens</em> strain GV3101 with helper plasmid pMP90 (Koncz and Schell, 1986) carrying pBIN61-p19 construct (Habibi <em>et al.</em>, 2018)</li><li>Antibiotics<br/>Gentamicin Sulfate (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> G-400-10 </a> )<br/>Spectinomycin Dihydrochloride Pentahydrate (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> S-140-5 </a> )<br/>Kanamycin Monosulfate (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> K-120-10 </a> )<br/>Rifampicin (GoldBio, catalog number: <a class="blackNoUnder" target="_blank"> R-120-1 </a> )</li><li>Magnesium chloride hexahydrate (MgCl<sub>2</sub>Â·6H<sub>2</sub>O) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 14222322 </a> )</li><li>2-(N-Morpholino) ethanesulfonic acid (MES) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP300100 </a> ) </li><li>Potassium hydroxide (KOH) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> P250-500 </a> )</li><li>Acetosyringone (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> AC115540010 </a> )</li><li>Dimethyl Sulfoxide (DMSO) (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP231-100 </a> )</li><li>Luria-Bertani (LB) broth (Fisher, catalog number: <a class="blackNoUnder" target="_blank"> BP9723-2 </a> )</li><li>Bacteriological agar (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A5306-1KG </a> )</li><li>Solid LB plates with 1.5% (w/v) agar</li><li>KAR<sub>1</sub> </li><li>KAR<sub>2</sub> </li><li><em>rac</em>-GR24</li><li>Acetone</li><li>MgCl<sub>2</sub> (1 M stock) (see Recipes)</li><li>MES (0.5 M stock) (see Recipes)</li><li>Acetosyringone (1 M stock) (see Recipes)</li><li>Infiltration Buffer (see Recipes)</li><li>KAR<sub>1</sub>, KAR<sub>2</sub>, <em>rac</em>-GR24 (50 mM stock) (see Recipes)<br/><em>Notes:</em> <ol class="npwthree"><li><em>KAR<sub>2</sub> was synthesized as previously reported (Goddard-Borger et al., 2007). rac-GR24 was synthesized according to Mangus et al. and recrystallized from diethyl ether/hexanes (Mangnus et al., 1992).</em> </li><li><em>The chemicals can be purchased from several commercial suppliers: KAR<sub>1</sub>: Toronto Research Chemicals (M305480), Chiralix (CX27716); KAR<sub>2</sub>: Toronto Research Chemicals (F864800), Chiralix (CX94877); rac-GR24: Chiralix (CX23880), Strigolab (ST23b rac), PhytoTech Labs (G3324), Toronto Research Chemicals (S687590).</em> </li></ol></li><li>10 ÂµM working chemical stocks (see Recipes)</li><li>0.02% (v/v) acetone control (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35569">### Equipment</p><ol class="npwtwok"><li>Centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5804 R </a> )</li><li>30 Â°C incubator shaker (ThermoFisher, model: <a class="blackNoUnder" target="_blank"> MaxQ 6000 R </a> )</li><li>CLARIOstar microplate reader (BMG Labtech)</li><li>4 mm Round Ticket Hole Punch</li><li>Dumont #5 Fine Forceps (Fine Science Tools, catalog number: <a class="blackNoUnder" target="_blank"> 11252-20 </a> )</li><li>1 mm electroporation cuvette (Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 40-100 </a> )</li><li>MicroPulser Electroporator (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 165-2100 </a> )</li><li>Vacuum-driven filter system (Olympus, catalog number: <a class="blackNoUnder" target="_blank"> 25-227 </a> )</li><li>3.5-inch square plastic pots (Farrand, catalog number: <a class="blackNoUnder" target="_blank"> K0-25SQ-G </a> )<br/></li></ol><p class="pptt" id="biaoti35570">### Software</p><ol class="npwtwok"><li>Prism (v8.2.0, GraphPad Software Inc.)</li></ol><p class="pptt" id="biaoti35571">### Procedure</p><ol class="npwone"><li>Transformation of <em>Agrobacterium tumefaciens</em> <ol class="npwtwo"><li>Transform pRATIO3212-SMAX1 into <em>Agrobacterium tumefaciens</em> strain GV3101 by electroporation method (Mersereau <em>et al.</em>, 1990).</li><li>Select the transformants on LB agar plates [containing 50 Âµg/ml Spectinomycin (expression construct), 25 Âµg/ml Gentamicin (helper plasmid), and 25 Âµg/ml Rifampicin (genomic)] at 30 Â°C for 2-3 days.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Propagation of <em>Nicotiana benthamiana</em> <ol class="npwtwo"><li>Germinate ~50-100 seeds in a 3.5-inch square plastic pot containing potting soil at ~21-24 Â°C and 16 h light/8 h dark photoperiod with ~80-100 Î¼mol/m<sup>2</sup>s light intensity. Cover the pot with a plastic cover to ensure high humidity.</li><li>As soon as the first true leaves emerge (10-12 days), transfer the seedlings to 3.5-inch plastic pots containing potting soil (two seedlings per pot) and cover again for 4 days with a plastic cover. Remove cover and maintain in growth conditions described above.</li><li>Water plants every second day or as required.</li><li>At 3-3.5 weeks after transplanting, the plant has reached the optimal developmental stage for agroinfiltration.<br/><em>Note: It is best to stop watering plants one day before the infiltration.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Transient expression<ol class="npwtwo"><li>Culture <em>A. tumefaciens</em> carrying pRATIO3212-SMAX1 or pBIN61-p19 (hereafter, p19) in 2-3 ml liquid LB medium containing 25 Âµg/ml Gentamicin (helper plasmid), 25 Âµg/ml Rifampicin (genomic), 50 Âµg/ml Spectinomycin (pRATIO3212-SMAX1), 50 Âµg/ml Kanamycin (p19); grow overnight at 30 Â°C incubator, 200-220 rpm.<br/><em>Notes:</em> <ol class="npwthree"><li><em>The presence of p19 partially blocks RNA silencing and thus establishes a higher and more uniform transgene expression.</em> </li><li><em>For each transformed strain, prepare glycerol stocks by mixing 0.5 ml of culture with 0.5 ml of 50% glycerol. Store at -80 Â°C.</em> </li></ol></li><li>Dilute the starter culture 1:1,000 in 15 ml LB + antibiotics in a 50 ml Falcon tube. Grow overnight at 30 Â°C incubator, 200 -220 rpm.<br/><em>Note: Perform this step in the evening. The culture volume can be scaled up depending on the number of transformations intended. However, we recommend keeping the culture volume less than 30 ml in a 50 ml Falcon tube for proper aeration.</em> </li><li>Pellet cells at 3,900 <em>x g</em>, 10 min at room temperature, remove supernatant, add 10 ml of infiltration media (see Recipes) and resuspend.</li><li>Repeat Step C3, resuspend the pelleted cells in ~3-4 ml of infiltration media.<br/><em>Note: These washes remove traces of antibiotics that can kill the leaf tissue after infiltration.</em> </li><li>Measure absorbance of the culture at 600 nm. Add 100 Âµl <em>Agrobacterium</em> culture to a clear 96-well plate and measure the optical density at 600 nm (OD<sub>600</sub>) with a microplate reader.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Blank the plate reader with a well containing infiltration media and use blank subtracted OD<sub>600</sub> readings for further calculations.</em> </li><li><em>Alternatively, OD<sub>600</sub> can be measured in a 1 ml plastic cuvette using a spectrophotometer.</em> </li></ol></li><li>Determine the titer of culture required for infiltration and calculate the dilution factor for a final volume of 4.0-5.0 ml of culture in infiltration media, depending on the number of transformations intended. For example, if you require a final OD<sub>600</sub> of 0.6 and culture reads 1.5, then you will need initial volume = (final OD<sub>600</sub> x final volume)/initial OD<sub>600</sub>. Therefore, the required initial volume = (0.6 x 4 ml)/1.5 = 1.6 ml.<br/><em>Note: In this protocol, SMAX1 and p19 were diluted to OD<sub>600</sub> of 0.6 and 0.4, respectively, in the final infiltration volume of 4.0 ml. However, the optimum ratio might be different for other proteins. The final OD<sub>600</sub> should not exceed 1.2 as this induces tissue necrosis, while OD<sub>600</sub> of 0.3 and lower on the other hand drastically reduces transient expression efficiencies (Kim et al., 2009) (see Note 1).</em> </li><li>Vortex and incubate the cultures for 4 h at room temperature (without shaking).</li><li>Choose the proper leaves for infiltration. Select the healthiest looking plants and avoid leaves that are torn or otherwise damaged. The ideal leaves are young leaves that are ~3-5 cm wide. Use 2-3 leaves per plant and at least 3 leaves (from independent plants) per construct.<br/><em>Notes:</em> <ol class="npwthree"><li><em>The developmental stage of the Nicotiana benthamiana plants is important for the success of this method. Healthy plants should have at least two large leaves, which may be the third or fourth leaf from the apical meristem.</em> </li><li><em>Avoid using small leaves as they are harder to infiltrate and display reduced transformation rates.</em> </li><li><em>Plants should not have flowered at the time of infiltration.</em> </li></ol></li><li>Swirl the solution in the tube before drawing it up into the syringe, in order to evenly suspend the <em>Agrobacterium</em>. Press the nozzle of a 1 ml syringe without a needle against the abaxial (lower) surface of the leaf and hold a gloved finger on the other side and inject slowly. Try to be as gentle as possible when injecting, to avoid damaging the leaf. The infiltrated area appears as a dark, water-soaked region. If the infiltrated region stops expanding without pushing too hard, infiltrate again in a different area of the leaf. After infiltration, pat dry the leaf surface with a Kimwipe and mark the edge of the infiltrated circle with a Sharpie permanent marker. Mark the leaf by attaching a small tag to the petiole.<br/><em>Notes:</em> <ol class="npwthree"><li><em>(Optional) Before infiltration, a small incision can be made at the site of infiltration using a sterile razor. Make sure that the wounds do not perforate the leaf, which could decrease the efficiency of the infiltration.</em> </li><li><em>We usually do not infiltrate more than one construct per leaf.</em> </li><li><em>It is not necessary to infiltrate the total leaf space. We typically infiltrate 2-3 spots in a single leaf.</em> </li><li><em>If you are infiltrating leaves with different constructs on the same plant, you should clean your gloves with 70% ethanol (or wipe them with Kimwipe) in between infiltrations and ensure that the leaves are not touching each other to prevent cross-contamination.</em> </li><li><em>We suggest using leaves transformed with p19 only (untransformed leaves are less preferable) as negative controls.</em><br/></li></ol></li><li>Return infiltrated <em>N. benthamiana</em> plants to the growth room, water them, and incubate for 3 days under normal growth conditions.<br/><br/></li></ol></li><li>Degradation Assay<ol class="npwtwo"><li>Before setting up treatment plates, it is necessary to verify the expression of the reporter proteins.</li><li>Prepare a black 96-well microplate (low autofluorescence) and add 300 Âµl dH<sub>2</sub>O in the wells you are going to use. Excise a single leaf disc from each transformed (and p19 only or untransformed) leaf using a hole punch and place them in a microplate abaxial (<em>i.e.</em>, the leaf underside) side <strong>up</strong>. Read the plate as per section E to identify those leaves that show a good ratio of transformed over p19 (or untransformed) signal, and proceed with these samples (see <strong>Notes 2</strong> and <strong>3</strong>).<br/><em>Note: This step is critical to identify leaves suitable for the degradation assay (Steps D3-D6).</em> </li><li>Prepare a 24-well treatment plate for sample harvesting and treatments (Figure 1). Label wells and add 1 ml of freshly prepared 10 ÂµM KAR<sub>1</sub>, KAR<sub>2</sub>, <em>rac</em>-GR24, or 0.02% (v/v) acetone control (see Recipe 9) to each well.</li><li>Excise 5 or 6 leaf discs per leaf per treatment from within the infiltrated region/circle and float them (abaxial side <strong>down</strong>) on chemical treatments. Place no more than 4 discs in each well.<br/><em>Notes:</em> <ol class="npwthree"><li><em>It is critical to minimize mechanical damage to the leaf during handling (cleanly punch the disc edges and try to avoid the forcep damage). Exclude major veins from the excised discs.</em> </li><li><em>For each construct, we recommend using at least 3 different leaves from independent plants. Also, include multiple negative control leaf discs per treatment.</em> </li><li><em>For each leaf, punch out leaf discs at random from lamina on each side of the midrib. When the desired number of leaf discs are collected, they are immediately and randomly transferred to the treatment plates to prevent them from drying. Alternatively, place the discs in distilled water until ready to be transferred.</em> </li><li><em>For each construct, randomly allocate discs from the same leaf to different treatments rather than allocating discs from one leaf for one treatment and another leaf to a different treatment.</em> </li></ol></li><li>Cover the plate and incubate at room temperature 12-16 h.<br/><em>Note: The plate should be carefully placed in the lab drawer (in the dark) and not allowed to remain on the bench top.</em> </li><li>Set up a black 96-well microplate with 300 Âµl dH<sub>2</sub>O in each well to be used. Transfer one leaf disc per well (abaxial side <strong>up</strong>) for measurement.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Discard leaf discs that appear translucent (water-logged) or had submerged after overnight treatment. </em> </li><li><em>It is necessary to include negative controls (p19 only or untransformed) on each plate, as these need to be measured with the same gain to perform the background subtraction correctly.</em> </li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200901/20200901225005_6506.jpg" width="300"/><br/><strong>Figure 1. Workflow for ratiometric measurement of SMAX1 abundance in <em>Nicotiana benthamiana</em>.</strong> Schematic of the degradation assay using pRATIO3212-SMAX1 ratiometric reporter system in <em>N. benthamiana</em>.<br/><br/></li></ol></li><li>Quantification of fluorescence intensity<br/>We suggest using the following optimized settings on the CLARIOstar plate reader to measure mScarlet-I and Venus fluorescence.<br/><em>Note: It would be possible to use these settings with other dual-fluorescent pRATIO reporters (Khosla et al., 2020a).</em> <ol class="npwtwo"><li>Basic settings<ol class="npwthree"><li>Measurement method: Fluorescence intensity</li><li>Reading mode: Well scanning</li><li>Microplate type: The CLARIOstar software contains a database with dimensions of microplates from most microplate manufacturers. In this case, we select âCOSTAR BLACK 96â from the pull-down menu.<br/><em>Note: For measuring fluorescence, we recommend using black plates as they absorb light and reduce background and crosstalk.</em> </li></ol></li><li>Well scan settings<br/><em>Note: Well scanning measures different points of a well in a matrix style. The output is an average measurement value of all scan points.</em> <ol class="npwthree"><li>Number of flashes per scan point: 8<br/><em>Note: For fluorescence measurements using pRATIO reporters., 8-10 flashes are usually adequate. Increasing the number of flashes also increases the reading time.</em> </li><li>Scan mode: matrix scan</li><li>Scan matrix dimension: 9 x 9<br/><em>Note: You can use this option to define the matrix size for the scanning procedure and, therefore, the number of measurement (scan) points.</em> </li><li>Scan width [mm]: 4<br/><em>Note: This value is optimized to reduce the edge effect (see Note 3).</em> </li></ol></li><li>Optic settings<br/><em>Note: We used spectral scanning to identify excitation and emission wavelengths for each fluorescent protein (FP) that produced the strongest signal above background autofluorescence of tobacco leaves (Table 1) (Khosla et al., 2020a).</em><br/><br/><strong>Table 1. Optimal filter settings for tobacco</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200901/20200901225541_5000.jpg" width="600"/><br/><br/><ol class="npwthree"><li>Gain adjustment: For each fluorophore, we performed an automatic gain adjustment on the entire plate before the measurement. The instrument finds the well with the highest intensity and determines an appropriate gain.<br/><em>Notes:</em> <ol class="npwfour"><li><em>The purpose of a gain adjustment is to optimize the signal amplification so that the results have the maximum sensitivity and dynamic range.</em> </li><li><em>It is preferred to have the highest gain for each signal coming from the same well.</em> </li></ol></li><li>Focal height [mm]: Perform automatic focal height adjustment on the well with the optimal gain (identified above). For a disc floating on 300 Î¼l water, this is typically ~13 mm.<br/></li></ol></li><li>General settings<br/>Optic: Top optic used<br/><em>Note: Top reading relies on measuring reflected light above the wells. For our protocol, we expect top reading to be more sensitive than bottom measurements. Floating leaf discs on the water in 96 well microplates ensure they do not get soaked and sit at the right height for accurate measurement.</em><br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200901/20200901225958_9563.jpg" width="300"/><br/><strong>Figure 2. Ratiometric detection of SMAX1 degradation in tobacco.</strong> A. Diagram of the pRATIO3212-SMAX1 ratiometric reporter system. The <em>35S</em> promoter drives expression of a multicistronic transcript encoding an SMAX1-mScarlet-I fusion that is separately translated from the Venus fluorescent protein due to the *F2A peptide. B. Relative fluorescence of SMAX1 in the pRATIO3212 system after transient expression in tobacco and overnight treatment with 10 Î¼M KAR<sub>1</sub>, KAR<sub>2</sub>, <em>rac</em>-GR24, or 0.02% (v/v) acetone control. n = 5-6 leaf discs. *<em>P</em> &lt; 0.01, Mann-Whitney U test comparisons to control treatment. The data shown is representative of one leaf out of three independent leaves with similar results.</li></ol></li></ol><p class="pptt" id="biaoti35572">### Data analysis</p><ol class="npwtwok"><li>Normalize the data with the following four steps.<ol class="npwthree"><li>Average the fluorescence signals obtained with mScarlet-I detection settings for negative control leaf discs transformed with <em>Agrobacterium</em> carrying p19 (or untransformed). Similarly, average the Venus fluorescence values from the wells containing p19 (or untransformed) leaf discs. These two values represent the background signal contributed by leaf autofluorescence for each reporter.</li><li>Subtract the mScarlet-I background value from the mScarlet-I signals obtained for each sample well on the microplate.</li><li>Subtract the Venus background value from the Venus signals obtained for each sample well on the microplate.</li><li>Calculate the fluorescence intensity using the following formula: ([mScarlet-I signals]-[mScalrlet-I background])/([Venus signals]-[Venus background]). After 12 h treatment with 10 Î¼M KAR<sub>1</sub>, KAR<sub>2</sub>, or <em>rac</em>-GR24, we observed a &gt; 2-fold decline in the mScarlet-I/Venus ratio in response to all three treatments, indicating degradation of SMAX1 (Figure 2, <a href="https://os.bio-protocol.org/attached/file/20200820/3747 Supplemental file 1.xlsx" target="_blank">Supplemental file 1</a>).</li></ol></li><li>Prism (v8.2.0, GraphPad Software Inc.) is used to perform data visualization and statistical analysis. The Centerline of the box-whisker plot indicates the median, and the colored box represents the interquartile range. The whiskers go down to the smallest value and up to the largest, and each individual value depicted as a point superimposed on the graph.<br/></li></ol><p class="pptt" id="biaoti35573">### Notes</p><ol class="npwtwok"><li>p19 GV3101 strain<br/>If you want to co-express multiple proteins, we recommend transforming the pRATIO construct into a strain of GV3101 that is already carrying p19. This minimizes the amount of <em>Agrobacterium</em> used for infiltration. It can also lead to less variation than you may get from mixing several strains of bacteria.</li><li>We do not recommend using leaves that show mScarlet-I/Venus ratio less than 0.3 for degradation analysis. To calculate mScarlet-I/Venus ratio, perform data normalization (see DATA ANALYSIS section) using a negative control leaf with the lowest signal for each mScarlet-I and Venus for background subtraction.</li><li>The gain and focal height settings determined in this step must be the same settings used when reading discs after overnight chemical treatment.</li><li>Compatibility of other pRATIO vectors<br/>Although we use the pRATIO3212 ratiometric system as an example in this study, the protocol works equally well with pRATIO vectors containing bioluminescent reporters (Khosla <em>et al.</em>, 2020a) in both tobacco and Arabidopsis protoplast transient assays. When using bioluminescent reporters, we suggest using white or gray plates as they reflect light, which will maximize the light output signal.</li><li>Comparison of different reading modes<br/>In this protocol, we measured fluorescence with well scan mode. Besides well scanning, fluorescence intensity can be measured by CLARIOstar using endpoint, orbital, and spiral averaging modes. A center endpoint reading excites and measures only in the center of the well, giving a single number. This can lead to inaccuracies as the disc may move between time points. When using spiral and orbital averaging modes, the plate reader takes multiple measurements for each well on a defined orbit, giving one averaged number over the entire area of the well. Well scanning measures different points of a well in a matrix style. Although both spiral averaging and well scanning yielded comparable sensitivity, well scanning is more versatile as it provides raw data value from single scan points. It is also possible to define thresholds to exclude scan points valid for all wells. Furthermore, the leaf disc harvest can sometimes induce edge damage, resulting in higher background signals. Using the scanning mode, you can define the diameter, such as measurement points that are inside the circle with the defined diameter will be measured, thereby excluding potential inaccurate readings. One limitation, however, is that well scanning is slower than orbital or spiral averaging modes.</li><li>Performing a time course assay<br/>Our protocol is also suitable for performing a time course study. To examine the time course of degradation, set up a black 96-well microplate with 300 Âµl chemical treatments and transfer one leaf disc per well with the abaxial side up. Use at least 12 leaf discs (4 discs per leaf x 3 leaves). Measure relative fluorescence at the indicated time points in the plate mode (slow kinetics) with spiral averaging option. Define number of cycles as the number of times the entire plate will be measured and cycle time, which is the amount of time it takes to measure the plate during one cycle. For example, if you want to read a plate every 15 min per hour for 6 h, then cycle time will be 300 s and the number of cycles 24. However, there are likely to be some limitations when performing a longer time course experiment (&gt; 6 h) using this method, foremost of which is the leaf discs sinking to the bottom of the well, and therefore decreasing the efficiency of measurement in top reading microplate readers. Alternatively, perform the assay as described in this study. At the indicated time points, discs are transferred to 96-well plate for fluorescence measurement and then returned to 24-well treatment plates until the next time point. Careful handling is important to minimize damage to the discs.<br/></li></ol><p class="pptt" id="biaoti35574">### Recipes</p><ol class="npwtwok"><li>MgCl<sub>2</sub> (1 M stock)<ol class="npwthree"><li>Dissolve 101.65 g of MgCl<sub>2</sub>Â·6H<sub>2</sub>O in 400 ml of deionized H<sub>2</sub>O</li><li>Adjust the volume to 0.5 L with deionized water</li><li>Filter-sterilized by vacuum filtration</li></ol></li><li>MES (0.5 M stock)<ol class="npwthree"><li>Dissolve 19.52 g MES in 200 ml deionized water</li><li>Adjust to pH 5.7 with 1 N KOH</li><li>Filter-sterilized by vacuum filtration</li></ol></li><li>Acetosyringone (1 M stock)<ol class="npwthree"><li>Dissolve 0.196 g acetosyringone in 1 ml of DMSO</li><li>Divide into single use aliquots are store at -20 Â°C</li></ol></li><li>Infiltration buffer<br/><em>Note: Infiltration medium is made fresh from the stock solutions on the day of infiltration and kept at room temperature until required.</em><br/>10 mM MgCl<sub>2</sub><br/>10 mM MES<br/>150 Î¼M acetosyringone</li><li>KAR<sub>1</sub> (50 mM stock)<br/>Dissolve 3.75 mg of KAR<sub>1</sub> in 500 Âµl acetone and store at -20 Â°C.</li><li>KAR<sub>2</sub> (50 mM stock)<br/>Dissolve 3.4 mg of KAR<sub>2</sub> in 500 Âµl acetone and store at -20 Â°C.</li><li><em>rac</em>-GR24 (50 mM stock)<br/>Dissolve 7.45 mg of <em>rac</em>-GR24 in 500 Âµl acetone and store at -20 Â°C.<br/><em>Note: Acetone stocks (5-7) may evaporate during storage, so we use amber color glass vials with screw thread cap and wrap the vials with Parafilm for long term storage. Allow the tubes to warm up to room temperature before opening to avoid introduction of atmospheric moisture, which can hydrolase GR24.</em> </li><li>10 ÂµM working chemical stocks (freshly prepared)<br/>2 Âµl 50 mM stock<br/>Dilute with 9,998 Âµl dH<sub>2</sub>O</li><li>0.02% (v/v) acetone control (freshly prepared)<br/>2 Âµl of acetone<br/>Dilute with 9,998 Âµl dH<sub>2</sub>O<br/></li></ol><p class="pptt" id="biaoti35575">### Acknowledgments</p><p class="ppzz">This protocol is a detailed expansion of that described in Khosla <em>et al.</em>, 2020b. We gratefully acknowledge funding support from NSF IOS-1737153 and NSF IOS-1740560 to DCN. We thank Dr. Gavin Flematti and Dr. Adrian Scaffidi (University of Western Australia) for chemical synthesis and purification of KARs and <em>rac</em>-GR24. We thank Dr. Mark Waters (University of Western Australia) for valuable feedback on an early draft of this protocol.</p><p class="pptt" id="biaoti35576">### Competing interests</p><p class="ppzz">We declare no competing interests.</p><p class="pptt" id="biaoti35577">### References</p><ol class="npwtwok"><li>Conn, C. E. and Nelson, D. C. (2015). <a href="https://www.frontiersin.org/articles/10.3389/fpls.2015.01219/full" target="_blank">Evidence that KARRIKIN-INSENSITIVE2 (KAI2) Receptors may Perceive an Unknown Signal that is not Karrikin or Strigolactone.</a> <em>Front Plant Sci</em> 6(1219).</li><li>Flematti, G., Ghisalberti, E., Dixon, K. and Trengove, R. (2004). <a href="https://science.sciencemag.org/content/305/5686/977" target="_blank">A Compound from Smoke That Promotes Seed Germination.</a> <em>Science</em> 305(5686): 977. </li><li>Flematti, G. R., Scaffidi, A., Waters, M. T. and Smith, S. M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27105887" target="_blank">Stereospecificity in strigolactone biosynthesis and perception.</a> <em>Planta</em> 243(6): 1361-1373.</li><li>Goddard-Borger, E. D., Ghisalberti, E. L. and Stick, R. V. (2007). <a href="https://chemistry-europe.onlinelibrary.wiley.com/doi/abs/10.1002/ejoc.200700334" target="_blank">Synthesis of the germination stimulant 3-methyl-2h-furo[2,3-c]pyran-2-one and analogous compounds from carbohydrates.</a> <em>Eur J Organic Chemistry</em> 38(49): 3925-3934.</li><li>Guo, Y., Zheng, Z., La Clair, J. J., Chory, J. and Noel, J. P. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23613584" target="_blank">Smoke-derived karrikin perception by the Î±/Î²-hydrolase KAI2 from <em>Arabidopsis</em>.</a> <em>Proc Natl Acad Sci U S A</em> 110(20): 8284-8289.</li><li>Hamiaux, C., Drummond, R., Janssen, B., Ledger, S., Cooney, J., Newcomb, R. and Snowden, K. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22959345" target="_blank">DAD2 is an Î±/Î² hydrolase likely to be involved in the perception of the plant branching hormone, strigolactone.</a> <em>Current Biology</em> 22(21): 2032-2036.</li><li>Habibi, P., Soccol, C. R., OâKeefe, B. R., Krumpe, L. R. H., Wilson, J., de Macedo, L. L. P., Faheem, M., Dos Santos, V. O., Prado, G. S., Botelho, M. A., Lacombe, S. and Grossi-de-Sa, M. F. (2018). <a href="https://www.sciencedirect.com/science/article/pii/S1359511318301740" target="_blank">Gene-silencing suppressors for high-level production of the HIV-1 entry inhibitor griffithsin in <em>Nicotiana benthamiana</em>.</a> <em>Process Biochemistry </em>70: 45-54.</li><li>Jiang, L., Liu, X., Xiong, G., Liu, H., Chen, F., Wang, L., Meng, X., Liu, G., Yu, H., Yuan, Y., Yi, W., Zhao, L., Ma, H., He, Y., Wu, Z., Melcher, K., Qian, Q., Xu, H. E., Wang, Y. and Li, J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24336200" target="_blank">DWARF 53 acts as a repressor of strigolactone signalling in rice.</a> <em>Nature</em> 504(7480): 401-405.</li><li>Khosla, A., Rodriguez-Furlan, C., Kapoor, S., Van Norman, J. M. and Nelson, D. C. (2020a). <a href="https://www.biorxiv.org/content/10.1101/2020.02.10.939363v1" target="_blank">A series of dual-reporter vectors for ratiometric analysis of protein abundance in plants.</a> <em>bioRxiv</em>: 2020.2002.2010.939363.</li><li>Khosla, A., Morffy, N., Li, Q., Faure, L., Chang, S. H., Yao, J., Zheng, J., Cai, M. L., Stanga, J. P., Flematti, G. R., Waters, M. and Nelson, D. C. (2020b). <a href="https://pubmed.ncbi.nlm.nih.gov/32434855/" target="_blank">Structure-Function Analysis of SMAX1 Reveals Domains that Mediate its Karrikin-Induced Proteolysis and Interaction with the Receptor KAI2.</a> <em>The Plant Cell</em>. Doi.org/10.1105/tpc.19.00752.</li><li>Kim, M. J., Baek, K. and Park, C.-M. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19484242" target="_blank">Optimization of conditions for transient <em>Agrobacterium</em>-mediated gene expression assays in <em>Arabidopsis</em>.</a> <em>Plant Cell Reports</em> 28: 1159-1167. </li><li>Koncz, C . and Schell, J. (1986). <a href="https://link.springer.com/article/10.1007%2FBF00331014" target="_blank">The promoter of TL-DNA gene 5 controls the tissue-specific expression of chimaeric genes carried by a novel type of <em>Agrobacterium</em> binary vector.</a> <em>Molecular General Genetics</em> 204(3): 383-396.</li><li>Luke, G.A. and Ryan, M.D. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29671261" target="_blank">Using the 2A Protein Coexpression System: Multicistronic 2A Vectors Expressing Gene(s) of Interest and Reporter Proteins.</a> <em>Methods Mol Biol</em> 1755: 31-48.</li><li>Mersereau, M., Pazour, G. J. and Das, A. (1990). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2165971" target="_blank">Efficient transformation of <em>Agrobacterium tumefaciens</em> by electroporation.</a> <em>Gene</em> 90(1): 149-151.</li><li>Nelson, D. C., Flematti, G. R., Ghisalberti, E. L., Dixon, K. W. and Smith, S. M. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22404467" target="_blank">Regulation of seed germination and seedling growth by chemical signals from burning vegetation.</a> <em>Annu Rev Plant Biol</em> 63(1): 107-130.</li><li>Nelson, D. C., Flematti, G. R., Riseborough, J. A., Ghisalberti, E. L., Dixon, K. W. and Smith, S. M. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20351290" target="_blank">Karrikins enhance light responses during germination and seedling development in <em>Arabidopsis thaliana</em>.</a> <em>Proc Natl Acad Sci U S A</em> 107(15): 7095-7100.</li><li>Nelson, D. C., Riseborough, J.-A., Flematti, G. R., Stevens, J., Ghisalberti, E. L., Dixon, K. W. and Smith, S. M. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19074625" target="_blank">Karrikins discovered in smoke trigger arabidopsis seed germination by a mechanism requiring gibberellic acid synthesis and light.</a> <em>Plant Physiol</em> 149(2): 863-873.</li><li>Nelson, D. C., Scaffidi, A., Dun, E. A., Waters, M. T., Flematti, G. R., Dixon, K. W., Beveridge, C. A., Ghisalberti, E. L. and Smith, S. M. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21555559" target="_blank">F-box protein MAX2 has dual roles in karrikin and strigolactone signaling in <em>Arabidopsis thaliana</em>.</a> <em>Proc Natl Acad Sci U S A</em> 108(21): 8897-8902.</li><li>Scaffidi, A., Waters, M. T., Sun, Y. K., Skelton, B. W., Dixon, K. W., Ghisalberti, E. L., Flematti, G. R. and Smith, S. M. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24808100" target="_blank">Strigolactone hormones and their stereoisomers signal through two related receptor proteins to induce different physiological responses in <em>Arabidopsis</em>.</a> <em>Plant Physiol</em> 165(3): 1221-1232.</li><li>Soundappan, I., Bennett, T., Morffy, N., Liang, Y., Stanga, J. P., Abbas, A., Leyser, O. and Nelson, D. C. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26546447" target="_blank">SMAX1-LIKE/D53 family members enable distinct max2-dependent responses to strigolactones and karrikins in <em>Arabidopsis</em>.</a> <em>Plant Cell</em> 27(11): 3143-3159.</li><li>Stanga, J. P., Smith, S. M., Briggs, W. R. and Nelson, D. C. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23893171" target="_blank"><em>SUPPRESSOR OF MORE AXILLARY GROWTH2 1</em> controls seed germination and seedling development in <em>Arabidopsis</em>.</a> <em>Plant Physiol</em> 163(1): 318-330.</li><li>Sun, X. D. and Ni, M. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20864454" target="_blank">HYPOSENSITIVE to LIGHT, an alpha/beta fold protein, acts downstream of ELONGATED HYPOCOTYL 5 to regulate seedling de-etiolation.</a> <em>Mol Plant</em> 4: 116-126.</li><li>Wang, L., Wang, B., Jiang, L., Liu, X., Li, X., Lu, Z., Meng, X., Wang, Y., Smith, S. M. and Li, J. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26546446" target="_blank">Strigolactone signaling in <em>Arabidopsis</em> regulates shoot development by targeting d53-like smxl repressor proteins for ubiquitination and degradation.</a> <em>Plant Cell </em>27(11): 3128-3142.</li><li>Waters, M. T., Gutjahr, C., Bennett, T. and Nelson, D. C. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28125281" target="_blank">Strigolactone signaling and evolution.</a> <em>Annu Rev Plant Biol</em> 68(1): 291-322.</li><li>Waters, M., Nelson, D., Scaffidi, A., Flematti, G., Sun, Y., Dixon, K. and Smith, S. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22357928" target="_blank">Specialisation within the DWARF14 protein family confers distinct responses to karrikins and strigolactones in <em>Arabidopsis</em>.</a> <em>Development</em> 139: 1285-1295. </li><li>Waters, M. T., Scaffidi, A., Moulin, S. L. Y., Sun, Y. K., Flematti, G. R. and Smith, S. M. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26175507" target="_blank"><em>A Selaginella</em> <em>moellendorffii</em> ortholog of KARRIKIN INSENSITIVE2 functions in <em>Arabidopsis</em> development but cannot mediate responses to karrikins or strigolactones.</a> <em>Plant Cell</em> 27(7): 1925-1944.<br/></li><li>Zhou, F., Lin, Q., Zhu, L., Ren, Y., Zhou, K., Shabek, N., Wu, F., Mao, H., Dong, W., Gan, L., Ma, W., Gao, H., Chen, J., Yang, C., Wang, D., Tan, J., Zhang, X., Guo, X., Wang, J., Jiang, L., Liu, X., Chen, W., Chu, J., Yan, C., Ueno, K., Ito, S., Asami, T., Cheng, Z., Wang, J., Lei, C., Zhai, H., Wu, C., Wang, H., Zheng, N. and Wan, J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24336215" target="_blank">D14âSCFD3-dependent degradation of D53 regulates strigolactone signalling.</a> <em>Nature</em> 504(7480): 406-410. <br/></li></ol>