<body>
            <h1>Screening Method for CRISPR/Cas9 Inhibition of a Human DNA Virus: Herpes Simplex Virus</h1>
            <p><strong>Authors:</strong> Werner M. Neuhausser, Hyung S. Oh, Pierce Eggan, Magdalena Angelova, Rory Kirchner, Kevin C. Eggan, David M. Knipe</p>
            <h2>Abstract</h2>
            <p>The efficiency of cleavage of individual CRISPR/Cas9-sgRNAs remains difficult to predict based on the CRISPR target sequence alone. Different intracellular environments (dependent on cell type or cell cycle state for example) may affect sgRNA efficiency by altering accessibility of genomic DNA through DNA modifications such as epigenetic marks and DNA-binding proteins (<em>e.g.</em>, histones) as well as alteration of the chromatin state of genomic DNA within the nucleus.<br/><br/> We recently reported a multi-step screening method for the identification of efficient sgRNAs targeting the Herpes simplex virus (HSV-1) genome and reported a differential mechanism for viral inhibition by CRISPR-Cas9 in the latent versus lytic phase. The screening platform detailed in this protocol allows step-by-step testing of the efficiency of cleavage in a cell-free system and in the context of viral target cells such as human foreskin fibroblasts followed by functional testing of the effects of CRISPR/sgRNA on viral protein expression, replication, and reactivation. This strategy could be readily applied to other target cells such as pluripotent stem cell-derived human sensory neurons or other human DNA viruses.<br/></p>
            <p><strong>Keywords:</strong> Herpes simplex virus, HSV, <em>Herpesviridae</em>, CRISPR, CRISPR/Cas9, HSV keratitis, Recurrent genital herpes, HSV encephalitis</p>
            
        </body>

<p class="pptt" id="biaoti_no">### Background</p><p class="ppzz">Herpes simplex virus (HSV) is a neurotropic DNA virus of the <em>Herpesviridae </em>family that causes lifelong and incurable infection in the majority of the human population and can lead to significant morbidity and mortality (Liesegang<em> et al.</em>, 1989; Roizman<em> et al.</em>, 2013). After the initial acute (primary) infection, HSV establishes lifelong latent infection within sensory ganglia. Latent HSV can reactivate periodically and cause serious illness such as HSV keratitis, the most common cause of corneal blindness (Liesegang <em>et al.</em>, 1989), and recurrent genital herpes, which carries high morbidity. In the fetus, neonate and the immunosuppressed, herpesviruses such as HSV-1 and -2, human cytomegalovirus (HCMV), and Varicella-Zoster Virus (VZV) can cause disseminated disease with high mortality including meningitis and encephalitis. In the US, HSV infection occurs in 1 in 3,200 neonates (Thompson and Whitley, 2011), and half of these infants have disseminated disease or encephalitis (Kimberlin, 2004), which, despite antiviral treatment, carries 25% mortality and long-term neurologic morbidity in more than two-thirds of survivors (Kimberlin<em> et al.</em>, 2001). In addition, HSV1 infection has been linked with Alzheimer's disease risk in APOE-E4 carriers (Itzhaki, 2018).<br/><br/>Current antiviral treatments (nucleoside analogues) do not prevent establishment of latency or episodes of reactivation and are not effective for clearing latent HSV from infected individuals. Thus, there is a critical need for novel, specific, and efficacious therapeutic and prophylactic avenues that directly cleave and/or edit viral genomes and permanently eliminate latent virus or disable its reactivation and effectively suppress viral replication in disseminated acute HSV infections. <br/><br/>CRISPR/Cas9 systems have evolved in bacteria as a form of adaptive immunity to provide resistance against bacteriophage infection and plasmid transformation (Bhaya et al., 2011; Doudna and Charpentier, 2014; Gaj <em>et al.</em>, 2013; Hsu <em>et al.</em>, 2014). This system has been successfully transported into human cells (Cong <em>et al.</em>, 2013; Mali <em>et al</em>., 2013). However, sgRNAs with comparable cutting efficiencies in a cell-free system can differ significantly when used in living cells (Oh<em> et al.</em>, 2019), indicating that the intracellular environment may alter the accessibility of genomic DNA for cleavage by CRISPR/Cas9. Possible mechanisms are the covalent modification of genomic DNA in the form of epigenetic marks, differential density of DNA binding proteins (such as histones), and alteration of the chromatin state of genomic DNA in different cell types and/or cellular states (Yarrington <em>et al.</em>, 2018; Liu <em>et al.</em>, 2020).<br/><br/>The screening platform described in this protocol allows identification of sgRNAs that efficiently cleave the HSV genome and inhibit lytic viral replications as well as reactivation of quiescent virus in the chosen type of target cells. We used SaCas9 to allow for further testing of the antiviral activity of individual SaCas9/sgRNAs <em>in vivo</em> using recombinant AAV-based delivery systems. This screening strategy can be extended to other HSV target cells such as human sensory neurons which can be differentiated from human pluripotent stem cells (Chambers <em>et al.</em>, 2012; Young <em>et al.</em>, 2014). In addition, this screening strategy can be applied to clinically important members of the <em>Herpesviridae</em> family such as HSV-2, HCMV, VZV and KSHV as well as other human DNA viruses lacking effective therapies.</p><p class="pptt" id="biaoti35589">## Materials and Reagents</p><ol class="npwtwok"><li>Eppendorf Safe-Lock Tubes, 1.5 ml (Thermo Fisher Scientific, Eppendorf, catalog number: 0<a class="blackNoUnder" target="_blank"> 22363212 </a> )</li><li>PCR tubes (any kind)</li><li>Pipette tips 10 Î¼l, 200 Î¼l, 1,000 Î¼l (Thermo Fisher Scientific, Thermo Scientific<sup>TM</sup>, catalog numbers: <a class="blackNoUnder" target="_blank"> 2140-05 </a> , <a class="blackNoUnder" target="_blank"> 2160P </a> , <a class="blackNoUnder" target="_blank"> 2079 </a> )</li><li>Tissue culture flasks T-75 filtered flasks (Corning, Falcon<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 353824 </a> )</li><li>50 ml Conical tube (Corning, Falcon<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 352098 </a> )</li><li>0.45 Âµm syringe filter (Pall, catalog number: <a class="blackNoUnder" target="_blank"> 4508 </a> )</li><li>6-well plates (Corning, catalog number: <a class="blackNoUnder" target="_blank"> CLS3516 </a> )</li><li>AmpliTaq Gold<sup>TM</sup> 360 Master Mix (Thermo Fisher Scientific, Applied Biosystems, 4398901)</li><li> Wizard<sup>Â®</sup> SV Gel and PCR Clean-Up System (Promega, catalog number: <a class="blackNoUnder" target="_blank"> A9281 </a> )</li><li>Platinum<sup>TM</sup> II Taq Hot-Start DNA Polymerase (Thermo Fisher Scientific, Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 14966001 </a> )</li><li>MAXIscript<sup>TM</sup> T7 Transcription Kit (Thermo Fisher Scientific, Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> AM1314 </a> )</li><li>Phusion<sup>Â®</sup> High-Fidelity DNA Polymerase (Thermo Fisher Scientific, New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0530S </a> )</li><li>Gene Snipper<sup>TM</sup> SaCas9 Protein (CRISPR-associated endonuclease Cas9 from Staphylococcus aureus) with SaCas9 Buffer: 200 mM HEPES, 50 mM MgCl<sub>2</sub>, 1 M NaCl,1 mM EDTA, pH 6.5 (Biovision, catalog number: <a class="blackNoUnder" target="_blank"> M1280 </a> ) or SpCas9 Protein (CRISPR-associated endonuclease Cas9 from <em>S. pyogenes</em>) with 1x NEBuffer 3.1: 100 mM NaCl, 50 mM Tris-HCl, 10 mM MgCl<sub>2</sub>, 100 Âµg/ml BSA, pH 7.9 (New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> M0386 </a> )</li><li>E.Z.N.A.<sup>Â®</sup> Micro RNA Kit (Omega Bio-Tek, catalog number: <a class="blackNoUnder" target="_blank"> R7034-01 </a> )</li><li>QIAfilter Plasmid Midi Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 12245 </a> )</li><li>Lenti-X qRT-PCR Titration Kit (Takara, catalog number: <a class="blackNoUnder" target="_blank"> 631235 </a> )</li><li>BsmBI restriction enzyme (Thermo Fisher Scientific, New England Biolabs, catalog number: <a class="blackNoUnder" target="_blank"> R0580 </a> )</li><li>DNeasy Blood &amp; Tissue kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 69504 </a> )</li><li>Agarose (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> A9539 </a> )</li><li>Polyethyleneimine (Polysciences, catalog number: <a class="blackNoUnder" target="_blank"> 23966 </a> , average Mw ~25,000)</li><li>Effectene (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 301425 </a> )</li><li>NuPAGE<sup>TM </sup>LDS Sample Buffer (4x) (Thermo Fisher Scientific, Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> NP0008 </a> )</li><li>NuPAGE 4-12% Bis-Tris Gels (Thermo Fisher Scientific, Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> NP0321PK2 </a> )</li><li>NuPAGE MOPS SDS Running Buffer (Thermo Fisher Scientific, Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> NP0001 </a> )</li><li>Nitrocellulose membrane (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1620112 </a> )</li><li> Odyssey Blocking Buffer (TBS) (LI-COR, catalog number: <a class="blackNoUnder" target="_blank"> 927-50000 </a> )</li><li>IRDye 800CW (P/N: 926-32211) or IRDye 680RD (P/N: 926-68070) secondary antibody (LI-COR)</li><li>Giemsa stain (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> G4507 or GS500 </a> )</li><li>Assay buffer (see Recipes)</li><li>KCl</li><li>MgCl<sub>2</sub> </li><li>Tris</li><li>Glycerol</li><li>EDTA</li><li>SDS</li><li>Bromophenol blue</li><li>Reaction Stop Buffer (see Recipes)<br/></li><br/><p class="ppzz" style="margin-left:-20px;"><b>Media<br/></b> </p></ol><ol class="npwtwok"><li>DMEM (Thermo Fisher Scientific, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 11960051 </a> )</li><li>Opti-MEM<sup>TM</sup> I (Thermo Fisher Scientific, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 31985062 </a> )</li><li>Fetal Bovine Serum (Thermo Fisher Scientific, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 26140079 </a> )</li><li>Bovine calf serum (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 16030074 </a> )</li><li>L-Glutamine (Thermo Fisher Scientific, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 25030164 </a> )</li><li>G418 Geneticin (Thermo Fisher Scientific, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 10131035 </a> )</li><li>Puromycin Dihydrochloride (Thermo Fisher Scientific, Gibco, catalog number: <a class="blackNoUnder" target="_blank"> A1113802 </a> or Santa Cruz, catalog number: <a class="blackNoUnder" target="_blank"> sc-134220 </a> )</li><li>Polybrene Infection reagent (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> TR-1003-G </a> or Santa Cruz: <a class="blackNoUnder" target="_blank"> sc-134220 </a> )</li><li>GAMMAGARD LIQUID Immune Globulin Infusion (Takeda, Shire, catalog number: <a class="blackNoUnder" target="_blank"> LE1500190 </a> , NDC: 0944 2700 02)<br/></li><br/><p class="ppzz" style="margin-left:-20px;"><strong>Plasmids<br/></strong> </p></ol><ol class="npwtwok"><li>lenti-SaCas9-sgRNA-Puro<p class="ppzz" style="word-break:break-all;width:100%;">Can be obtained by cloning SaCas9 and its trans-activating sgRNA into the lentiCRISPRv2 plasmid (addgene #<a class="blackNoUnder" target="_blank"> 52961 </a> ). lentiCRISPRv2 is digested using BsmBI (NEB) and BamHI (NEB) and purified using a DNA purification kit (Zymo). DNA assembly can be performed using NEBuilder<sup>Â®</sup> HiFi DNA Assembly (NEB) according to the manufacturerâs protocol with three DNA fragments, the purified linear lentiCRISPRv2, a double-stranded DNA gBlock (IDT) containing the sgRNA sequence (IDT, 5â CTTTATATATCTTGTGGAAAGGA CGAAACACCGGAGACGtGATATCaCGTCTCAGTTTTAGTACTCTGGAAACAGAATCTACTAAAACAAGGCAAAATGCCGTGTTTATCTCGTCAACTTGTTGGCGAGATTTTTGAATTCGTAGACTCGAGGCGTTGACAT TG 3â) and a PCR fragment containing SaCas9.</p></li><li>psPAX2 (Addgene plasmid #<a class="blackNoUnder" target="_blank"> 12260 </a> )</li><li>pVSV-G (Addgene plasmid # 8454)</li></ol><p class="ppzz"><br/></p><p class="ppzz" style="margin-left:10px;"><b>Cell lines and media</b><br/>Human Foreskin Fibroblasts (HFF, Hs27, ATCC, catalog number: CRL-1634), U2OS (known as U-2 OS, ATCC, catalog number: HTB-96), 293T (ATCC, catalog number: CRL-3216) and Vero (ATCC, catalog number: CCL-81) cells can be obtained from the American Type Culture Collection.</p><ol class="npwtwok"><li>HFF and 293T cells: Dulbeccoâs modified Eagle medium (DMEM) supplemented with 10% (v/v) Fetal Bovine Serum (FBS)</li><li>V27 cells: DMEM supplemented with 5% (v/v) FBS, 5% (v/v) bovine calf serum (BCS), 2 mM L-glutamine, and 500 Î¼g/ml of G418 in 5% CO<sub>2 </sub>(Rice <em>et al.</em>, 1989)</li><li>U2OS and Vero cells: DMEM supplemented with 5% (v/v) FBS, 5% (v/v) bovine calf serum (BCS), and 2 mM L-glutamine in 5% CO<sub>2</sub> </li><li>FO6 cells: are maintained in DMEM supplemented with 5% (v/v) FBS, 5% (v/v) BCS, and 2 mM L-glutamine with 500 Î¼g/ml of G418 (Gibco) and 300 Î¼g/ml of hygromycin B (Invitrogen) in 5% CO<sub>2</sub> (Samaniego<em> et al.</em>, 1997)<br/></li><br/><p class="ppzz" style="margin-left:-20px;"><b>Viruses</b><br/>The KOS strain is a commonly used wild-type HSV-1 strain (Schaffer<em> et al.</em>, 1970). HSV-1 <em>d</em>109 is a KOS strain-derived mutant that is deleted for all five immediate early (IE) genes and contains the green fluorescent protein (GFP) gene under the control of the human cytomegalovirus (HCMV) IE promoter in the HSV-1 <em>UL54</em> gene locus. <em>d</em>109 can be grown and titrated on U2OS ICP4/27 and FO6 cells (Samaniego <em>et al.</em>, 1998).<br/><br/><b>MiSeq adapter primers (Table 1)</b><br/>Forward adapter primer: 5â²-TCTTTCCCTACACGACGCTCTTCCGATCT-CRISPR site forward primer-3â²<br/>Reverse adapter primer: 5â²-TGGAGTTCAGACGTGTGCTCTTCCGATCT-CRISPR site reverse primer-3â²<br/><br/><b>HSV-1 gene specific primers</b> </p></ol>   UL30-5_for: 5â²-CCCAAGGTGTACTGCGGG-3â²<br style="color:#666666;font-family:Arial;font-size:13.3333px;text-align:justify;white-space:normal;"/>   UL30-5_rev: 5â²-CTCCACGTTCTCCAGGATGT-3â²<br style="color:#666666;font-family:Arial;font-size:13.3333px;text-align:justify;white-space:normal;"/>   UL29-3_for: 5â²-ATAGACTCGAGGGCCAGGG-3â²<br style="color:#666666;font-family:Arial;font-size:13.3333px;text-align:justify;white-space:normal;"/>   UL29-3_rev: 5â²-TGACGAAAACCACGAGGGC-3â²<br/><ol class="npwtwok"><div><div><p><br/></p></div></div><b></b> <p><b style="mso-bidi-font-weight:normal;">Table 1. MiSeq barcode primers</b> </p></ol><p class="ppzz"><img alt="" class="layerphoto" height="650" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904001224_6184.jpg" width="677"/> </p><p class="pptt" id="biaoti35590">### Equipment</p><ol class="npwtwok"><li>Pipettes (capable of accurately pipetting from 1 Î¼l-10 ml) (any source)</li><li>Thermocycler (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> C1000 </a> )</li><li>Vortex (any source)</li><li>Micro centrifuge capable of 14 kG (Eppendorf)</li><li>Incubators (37 Â°C)</li><li>DNA gel electrophoresis apparatus</li><li>Odyssey CLx Imaging System (LI-COR)</li><li>UV transilluminator (or handheld UV lamp)</li><li>Spectrophotometer (Thermo Fisher Scientific, model: NanoDrop 2000)<br/></li></ol><p class="pptt" id="biaoti35591">### Software</p><ol class="npwtwok"><li>Outknocker (<a href="http://www.outknocker.org/" target="_blank">http://www.outknocker.org/</a>)</li><li>Cas-OFFinder (<a href="http://www.rgenome.net/cas-offinder/" target="_blank">http://www.rgenome.net/cas-offinder/</a>)</li><li>mutect2 (<a href="https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2" target="_blank">https://gatk.broadinstitute.org/hc/en-us/articles/360037593851-Mutect2</a>)</li><li>Prism 6 (Version 6.01) software<br/></li></ol><p class="pptt" id="biaoti35592">### Procedure</p><ol class="npwone"><li>Cell-free CRISPR/Cas9 cutting assay</li><ol class="npwtwo"><li>Design sgRNAs</li><ol class="npwthree"><li>Identify essential genes (<em>e.g., UL29, UL30, UL54</em>, a4 for HSV-1) or DNA regions within the viral genome.</li><li>Design appropriate sgRNA target sequences within ~1 kbp downstream of the start codon. The requirement for SaCas9 cleavage is the presence of a PAM sequence (5â-NNGRRT-3â) immediately downstream of the ~21 base pair target. Select multiple independent CRISPR sgRNAs for each viral gene target for efficiency testing.</li><li> Assess off-target activity of sgRNAs: The sgRNA sequences should be unique to the viral genome to prevent off-target cleavage of human DNA. Useful web sites for sgRNA identification and ranking of off-target activity are <a href="http://chopchop.cbu.uib.no/" target="_blank">http://chopchop.cbu.uib.no/</a> [CHOP CHOPv2 (Labun <em>et al.</em>, 2016) and crispr.mit.edu (Zhang lab, MIT) (Hsu<em> et al.</em>, 2013)].</li></ol><li><em>In vitro</em> sgRNA synthesis</li><ol class="npwthree"><li>Generate templates for T7<em> In vitro</em> transcription of sgRNAs</li><ol class="npwfour"><li>Design and order oligos for sgRNA synthesis: For each sgRNA design a 60 nt forward oligo (target specific oligo), containing the T7 promoter (5â-TAATACGACTCACTATA) followed by 21 nt of the specific SaCas9 sgRNA DNA binding sequence with the first 2 nucleotides replaced by GG (GGNNNNNNNNNNNNNNNNNNN) for T7 initiation and a constant 23 nt tail of sgRNA scaffold sequence for annealing (GTTTTAGTACTCTGGAAACAGAA-3â). These forward oligos are used in combination with a constant 82 nt reverse oligo (5â- AAAAAATCTCGCCAACAAGTTGACGAGATAAAC ACGGCATTTTGCCTTGTTTTAGTAGATTCTGTTTCCAGAGTACTAAAAC-3â) encoding the reverse-complement of the tracrRNA tail to add the sgRNA invariable 3â end (SaCas9 scaffold oligo, Figure 1).</li><li>Anneal sgRNA primers with the tail primer: For each sgRNA assemble the following reaction in PCR tubes:<br/><strong>Reagent                                                             [Î¼l]</strong><br/>Oligo 1 (variable sgRNA oligo), 100 Î¼M           1<br/>Oligo 2 (constant tail oligo), 100 Î¼M                 1<br/>H<sub>2</sub>O                                                                        8<br/><strong>Total                                                                       10</strong><br/><br/><strong>Anneal protocol:</strong><br/>Hold at 95 Â°C for 5 min.<br/>Ramp from 95 Â°C -&gt; 85 Â°C at 2 Â°C/s<br/>Ramp from 85 Â°C -&gt; 25 Â°C at 0.1 Â°C/s<br/>Hold at 4 Â°C.</li><li>Fill in single-stranded DNA (ssDNA) overhangs to generate double- stranded DNA (dsDNA) templates for T7 transcription by assembling the following reaction for each sgRNA in PCR tubes (Figure 1):<br/><strong>Reagent                                                    [Î¼l]</strong><br/>AmpliTaq Gold<sup>TM</sup> 360 Master Mix        10<br/>Taq polymerase (5 U)                                1<br/>Anneal Rx from ii.                                      9<br/><strong>Total                                                             20</strong><br/>Incubate at 72 Â°C for a total of 1.5 h and add an additional 1 Î¼l Taq after 45 min.</li><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904001528_4273.jpg" width="943"/><br/><strong></strong><strong>Figure 1. T7 <em>in vitro</em> transcription of sgRNAs.</strong> The 60 nt target-specific oligo contains the T7 promoter sequence followed by 21 nt of the specific <em>S. aureus</em> Cas9 sgRNA DNA binding sequence with the first 2 nucleotides replaced by GG and a 23 nucleotide overlap sequence of the SaCas9 scaffold. The overlap sequence is complementary to the first 23 nt of the (constant) SaCas9 scaffold oligo. The two oligos are annealed and both oligos extended from their 3â² ends by DNA polymerase resulting in a double-stranded DNA template for T7 transcription. RNA polymerase recognizes the double-stranded DNA of the T7 promoter and initiates transcription at the âGGâ. The resulting sgRNA contains the target-specific CRISPR sequence as well as the RNA scaffold.<br/><br/><li> Purify the dsDNA using the Promega Wizard Clean up kit and elute in 20 Î¼l H<sub>2</sub>O.</li><li> (Optional) Run dsDNA templates on an agarose gel to check the size.<br/></li></ol><li>sgRNA <em>in vitro</em> transcription (IVT):<br/><ol class="npwfour"><li>For each sgRNA assemble the following reaction using the MAXIscript<sup>TM</sup> T7 Transcription Kit and approximately 150 ng of dsDNA template and incubate overnight at 37 Â°C:<br/><br/><strong>Reagent                                             [Î¼l]</strong><br/>ATP solution                                         2<br/>CTP solution                                         2<br/>GTP solution                                         2<br/>UTP solution                                         2<br/>10X Reaction Buffer                             2<br/>Template DNA (150 ng)                       ~4<br/>T7 Enzyme Mix                                       2<br/>H<sub>2</sub>O                                                          4<br/><strong>Total                                                         20</strong><br/><em>Note: Assemble transcription reaction at room temp (The spermidine in the 10x Reaction Buffer can coprecipitate the template DNA if the reaction is assembled on ice). Add the 10x Reaction Buffer after the water and the ribonucleotides are already in the tube.</em><br/><br/></li><li> Remove template DNA: Add 56 Î¼l H<sub>2</sub>O and 4Î¼l DNAse turbo to each IVT reaction and incubate at 37 Â°C for 15 min.</li><li>sgRNA purification: Purify the sgRNAs using the Omega EZNA PF kit and measure the concentration of the purified sgRNAs using a NanoDrop 2000. Elute in H<sub>2</sub>O or assay buffer). After purification, dilute the sgRNAs to 200 ng/Âµl each and store at -80 Â°C.</li><li>(Optional) Confirm the expected size (~97bp) of the sgRNA on an RNA gel.</li></ol></li></ol><li>Generate dsDNA substrate for the <em>In vitro</em> cutting assay</li><ol class="npwthree"><li>For each sgRNA amplify a 500-2,000 bp segment around the sgRNA sequence to be tested from HSV-1 genomic DNA using PCR. If the amplification is difficult, dsDNA for the cleavage assay can also be obtained as g-blocks (IDT).</li><li>Purify the PCR product using the Promega Wizard Clean up kit.</li></ol><li><em>In vitro</em> cutting assay with CRISPR sgRNA and SaCas9 protein<br/><ol class="npwthree"><li>Reagent setup:</li><ol class="npwfour"><li>Dilute Cas9 protein in assay buffer to 500 nM. </li><li>Dilute purified PCR product to 50-100 ng/Î¼l in H<sub>2</sub>O.</li><li>Confirm each sgRNA is at 200 ng/Î¼l or higher.</li></ol><li>Assembly of cleavage assay: Each guide requires four wells of a PCR strip. Cleavage assays are performed in a reaction volume of 10 Î¼l with final SaCas9 concentrations of 300 nM, 100 nM, 33 nM, and 0 nM, 400ng sgRNA, 200 ng dsDNA substrate in 1x SaCas9 nuclease reaction buffer.<br/><ol class="npwfour"><li>Add 6 Î¼l of assay buffer to wells 2-4 of a PCR strip and 6 Î¼l of 500 nM SaCas9 protein to well 1. Then add 2 Î¼l of 500 nM SaCas9 protein to well 2, mix, transfer 2 Î¼l of this to well 3, mix and remove and discard 2 Î¼l of this mix. Since the final volume of the reaction is 10 Î¼l, the final Cas9 concentrations are 300 nM, 100 nM, 33 nM, and 0 nM.</li><li>Add 2 Î¼l of sgRNA to each of the 4 wells.</li><li> Incubate for 10 min at 37 Â°C to allow SaCas9-sgRNA complexes to form.</li><li>Add 2 Î¼l purified PCR product to each of the 4 wells.</li><li>Incubate for 30 min at 37 Â°C.</li><li>Add 4 Î¼l of reaction stop buffer to each well and incubate at 80 Â°C for 10 min.</li><li>Run all 10 Î¼l of all reactions on a 2% agarose gel and image. </li><li> Analysis: Because the assay provides sgRNA in excess, SaCas9 protein is the limiting factor. Cutting efficiency is constrained by the total amount of template provided, but in general, sgRNAs that work well show robust cutting (larger band at 0-50% intensity of no SaCas9 control) and there will be evidence of graded activity at lower SaCas9 concentrations. Figure 2 shows the results of the <em>in vitro</em> cleavage assay for three sgRNA targeting the essential HSV<em> UL30</em> gene (encoding the viral polymerase). </li><br/></ol></li></ol></li><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904001914_8400.jpg" width="548"/><br/><strong>Figure 2. <em>In vitro</em> cleavage assay for the essential HSV-1 gene <em>UL30</em> </strong>(Oh <em>et al.</em>, 2019). A. Schematic diagram of <em>in vitro</em> cleavage assay; B. Results are shown for three sgRNAs targeting UL30 (UL30-3, -4, and -5). T7 <em>in vitro</em> transcribed sgRNA was combined with SpCas9 protein and a PCR template containing the CRISPR sequence, incubated 1 h at 37 Â°C and run on an agarose gel. Lane (1) SpCas9+sgRNA, lane (2) Cas9 only, lane (3) sgRNA only, lane (4) no Cas9/sgRNA. Efficient cutting is seen for UL30-4 and -5 but not UL30-3.<br/><br/></ol><li>Cell-based CRISPR/SaCas9 cutting assay using next generation sequencing (MiSeq)</li><ol class="npwtwo"><li>Cloning of sgRNAs into SaCas9 lentivirus expression vector</li><ol class="npwthree"><li>Clone the sgRNA CRISPR sequence into lentiCRISPR SaCas9 after <em>Bsm</em>BI digestion using oligos as described previously (Sanjana <em>et al.</em>, 2014; Shalem <em>et al.</em>, 2014) to generate a lenti vector expressing SaCas9 land a sgRNA as well as the selection marker puro.</li><li>Purify plasmid DNAs using the endotoxin-free Midiprep kit (Qiagen) according to the manufacturerâs protocol.</li></ol><li>Lentivirus prep</li><ol class="npwthree"><li>Plate 3 x 10<sup>6</sup> human 293T cells in a 100 mm dish 20-24 h before transfection.</li><li>Transfect the 293T cells with lenti-SaCas9-sgRNA-Puro (5 Âµg), psPAX2 (4 Âµg), and pVSV-G (1 Âµg) using Effectene according to the manufacturerâs protocol or using polyethyleneimine (PEI).</li><li>For PEI transfection, mix a total of 10 Âµg of DNAs in 500 Âµl of Opti-MEMI (Gibco) and 30 Âµg of PEI in 500 Âµl of Opti-MEMI.</li><li>Mix DNA and PEI (total 1 ml) and incubate at ambient temperature for 20 min.</li><li>Add mixture directly to 293T cells containing fresh 5 ml of DMEM (supplemented with 10% (vol/vol) FBS and 2 mM glutamine).</li><li>Incubate the cells at 37 Â°C for 8-12 h, then replace media with 10 ml of fresh DMEM supplemented with 10-30% (vol/vol) FBS and 2 mM glutamine, and continue incubation at 37 Â°C.</li><li>At every 12-24 h for 48-60 h, harvest media and replace with fresh DMEM (supplemented with 10-30% (vol/vol) FBS and 2 mM glutamine).</li><li>Pool the harvested media on ice, filter the pooled media using 0.45 Âµm syringe filter (Pall).</li><li>Optional: To concentrate, centrifuge the virus for 1.5 h at 25,000 rpm (SW28 rotor of Beckman, ~120,000 <em>x g</em>) at 4 Â°C.</li><li>Titration of lentivirus: use the Lenti-X qRT-PCR Titration Kit (Takarabio) or qPCR Lentivirus Complete Titration Kit (abm) according to the manufacturerâs protocol.</li><li>Keep lentivirus at 0-4 Â°C until transduction of target cells (within one day) or freeze at -80 Â°C for long-term storage.</li></ol><li>For quiescent HSV infection: </li><ol class="npwthree"><li>Establish quiescent <em>HSV-1</em> infection in HFF cells:</li><ol class="npwfour"><li>Infect HFF cells with the <em>HSV-1</em> mutant <em>d</em>109 (which is missing all 5 immediate early genes) at an MOI of 10 in PBS containing 0.1% glucose (wt/vol) and 0.1% BCS (v/v) for 1 h with shaking at 37 Â°C (Ferenczy and DeLuca, 2011).</li><li>Replace virus inoculum with DMEM supplemented with 10% (v/v) FBS and incubate at 37 Â°C for 7-10 days.</li></ol><li>Transduction of HFFs containing quiescent HSV-1 viral genomes with SaCas9/sgRNA<br/><div style="white-space:nowrap;"><span style="font-size:10pt;"></span> </div></li><ol class="npwfour"><li>Transduce the <em>d</em>109 infected cells with 2 or 10 ml of lentivirus (or at an MOI of 5) containing 3-4 Âµg/ml of polybrene in 6-well plates or in T150 flasks respectively (optional: repeat the lentivirus transduction next day).</li><li>The next day, replace the transduction medium with fresh medium and incubate cells for 2 days at 37 Â°C followed by puromycin treatment (1 Âµg/ml) for 7-10 days.</li><li>Isolation of HSV genomic DNA: Harvest cells and isolate genomic DNA using the DNeasy Blood &amp; Tissue kit (Qiagen) according to the manufacturerâs protocol.</li></ol></ol><li>For lytic HSV infection:</li><ol class="npwthree"><li>Transduction of HFFs with SaCas9/sgRNA:</li><ol class="npwfour"><li>To transduce HFF cells with lentivirus for the lytic infection assay, plate HFF cells at a density of 2 x 10<sup>5</sup>/well in a T25 flask one day prior to transduction.</li><li>Transduce with 2-5 ml of lentivirus (or at an MOI of 5) containing 3-4 Âµg/ml of polybrene (optional: repeat the lentivirus transduction next day).</li><li>The next day, replace the transduction medium with fresh medium and incubate cells for 2 days at 37 Â°C followed by puromycin treatment (1 Âµg/ml) for 7-10 days.</li></ol><li>Lytic infection with wild-type HSV-1<br/></li><ol class="npwfour"><li>Infect the transduced cells with wild-type HSV-1 at an MOI of 0.1 or 5 in phosphate-buffered saline (PBS) containing 0.1% glucose (wt/vol), 0.1% BCS (v/v) for 1 h with shaking at 37 Â°C.</li><li>Replace virus inoculum with DMEM containing 1% BCS and incubate at 37 Â°C.</li><li>Harvest cells at various time points for viral replication by qPCR and titration.</li><li>Isolation of genomic and viral DNA: isolate genomic DNA using the DNeasy Blood &amp; Tissue kit according to the manufacturerâs protocol.</li></ol></ol><li>Deep sequencing (MiSeq): The process of deep sequencing involves an initial PCR to generate a ~200 bp product around the region of interest followed by a second PCR to add barcodes to the amplicons. Following this, each of the products is run on an agarose gel to determine the relative amount of each amplicon. Based on the brightness of the amplicon band on the gel, the PCR products of a particular loci are pooled such that each amplicon is ~equally represented in the final pool. The pooled product is then run on an agarose gel and gel-purified before being submitted for sequencing. With a set of 24 unique barcoding primer pairs a total of 24 sgRNAs can be analyzed in parallel with one deep sequencing sample.</li><ol class="npwthree"><li>Adapter (first round) PCR with target gene-specific primers</li><ol class="npwfour"><li>Design a primer set generating a ~200 bp product around the CRISPR site and add the adapter sequences to forward and reverse primer (see Figure 3). The CRISPR site should be in the center 1/3 of the amplicon to obtain good sequencing across the targeted site. The adapter sequences are used in the second round (barcoding) PCR for primer annealing.</li><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904002421_1555.jpg" width="710"/><br/><strong>Figure 3. Deep sequencing (MiSeq) process. </strong>A diagram of the two step PCR (adapter PCR, barcode PCR). The first step (adapter PCR) uses gene specific primers (black) with extensions (orange) for the second step (barcode PCR). The following sequences are the adaptors that should be added to the gene-specific primers: Adapter sequence for forward primer: 5â-TCTTTCCCTACACGACGCTCTTCCGATCT-target specific sequence-3â, Adapter sequence for reverse primer: 5â-TGGAGTTCAGACGTGTGCTCTTCCGATCT-target specific sequence-3â. The barcode PCR primers anneal to the adapter sequences and contain a barcode (blue) as well as P5/P7 sequences for binding to the flow cell. The three sequencing reads cover the amplicon/target sequence and read the barcode.<br/><br/><li>Perform a Phusion touch-down PCR, which increases the specificity of the reaction, according to the following protocol. Use the NEB calculator to determine the annealing temperature (<a href="http://tmcalculator.neb.com/#!/" target="_blank">http://tmcalculator.neb.com/#!/</a>).<br/><strong>Reagent                                                            [Î¼l]</strong><br/>5x GC buffer                                                       2<br/>dNTPs (10 mM)                                                  0.2<br/>Forward primer (25 Î¼M)                                    0.05<br/>Reverse primer (25 Î¼M)                                     0.05<br/>Template (~1-10 ng gDNA from B4b-iii)           1<br/>Phusion polymerase                                            0.1<br/>DMSO                                                                     0.3<br/>H<sub>2</sub>O                                                                         6.3<br/><strong>Total                                                                        10</strong><br/><br/><strong>Cycle protocol:</strong><br/>Initial denaturation 98 Â°C 30 s <br/><br/>15x<br/>Denaturation 98 Â°C 10 s<br/>Annealing 72 Â°C (-0.5 Â°C/cycle) 30 s<br/>Elongation 72 Â°C 2 min<br/><br/>15x <br/>Denaturation 98 Â°C 10 s<br/>Annealing Tm 30 s<br/>Elongation 72 Â°C 2 min<br/>Final elongation 72 Â°C 5 min<br/>Hold at 4 Â°C</li><li>Run a small amount (2 Î¼l) on a 1.5% agarose gel to ensure that the PCR worked. If so, move on to the second round (barcoding) PCR.</li></ol><li>Barcode (second round) PCR with barcoding primers: This PCR uses universal primers (not gene-specific, see Table 1) to attach the sequencing adaptors and barcode to the amplicon. The forward primer is the same, but there are 24 reverse primers containing different barcodes (for testing of up to 24 different sgRNAs in parallel).<br/><ol class="npwfour"><li>Set up a standard 10 Î¼l PCR mix using 1 Î¼l of the first round PCR product as the template as follows:<br/><strong>PCR Rx                                          [Î¼l]</strong><br/>5x GC buffer                                   2<br/>dNTPs (10 mM)                              0.2<br/>Forward primer (10 Î¼M)                0.5<br/>Reverse primer (5 Î¼M)                    1<br/><strong>Adapter PCR (template)</strong>                 1.0<br/>Phusion                                             0.1<br/>DMSO                                                0.3<br/>H<sub>2</sub>O                                                     4.9<br/><strong>Total                                                     10</strong><br/><br/><strong>Cycle protocol:</strong><br/>Initial denaturation 98 Â°C 30 s <br/>20x<br/>Denaturation 98 Â°C10 s<br/>Annealing 66 Â°C20 s<br/>Elongation 72 Â°C20 s<br/><br/>Final elongation 72 Â°C 3 min<br/>Hold at 4 Â°C</li><li> Run a small amount (2 Î¼l) on a 1.5% agarose gel. This allows you to ensure that the PCR worked and to normalize the amount of each amplicon when PCR products are pooled together for MiSeq. Of note, in some cases, one may see a large primer dimer amplicon around 175 bp in addition to the correct band.</li><li>Pool amplicons and perform gel extraction:</li><div class="media media-x cnautokh cnautokh1" style="margin-top:-0.5px;"><div class="float-left">1)</div><div class="media-body">Pool (up to 24) amplicons for different sgRNAsâdo not pool across amplicons from different targeted viral genes as this will make the gel extraction more difficult. From the previous gel, estimate equal molarities when pooling (<em>i.e.</em>, compensate for weaker looking amplicons by adding more to the pool). This will even out the read counts across different amplicons during MiSeq.</div></div><div class="media media-x cnautokh cnautokh2" style="margin-top:-0.5px;"><div class="float-left">2)</div><div class="media-body">Run pooled amplicons on a 1.5% agarose gel and gel extract the DNA using the Promega Gel Extraction Kit.</div></div><li>Submit purified/pooled amplicons for IlluminaMiSeq using the 16S Metagenomic Sequencing Library Preparation kit (Illumina). Determine DNA concentrations using the Qubit fluorometer 2.0 (Life Technologies, USA) with the Qubit dsDNA High Specificity assay kit. Sequence DNA libraries by IlluminaMiSeq (150 bp paired-end [Gagnon <em>et al.</em>, 2014]).</li></ol></li></ol><li>Indel analysis: Determine indel frequencies for each sgRNA by quantifying aligned reads containing insertions or deletions 1bp or larger using (<a href="http://www.outknocker.org/" target="_blank">http://www.outknocker.org/</a>). Analyze indel size using the ICE analysis toolbox (<a href="https://www.synthego.com/products/bioinformatics/crispr-analysis" target="_blank">https://www.synthego.com/products/bioinformatics/crispr-analysis</a>). Figure 4 shows the analysis of indel mutations in the HSV-1 genome during the quiescent infection for sgRNAs targeting the essential HSV-1 genes <em>UL29 </em>and<em> UL30</em>.</li><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904002728_2190.jpg" width="511"/><br/><strong>Figure 4. Indel mutations in the HSV-1 genome during the quiescent infection</strong> (Oh <em>et al.</em>, 2019). A. Indel mutation frequencies of quiescent <em>d</em>109 genomes are shown at the indicated sgRNA target sites. B. Histogram representing the frequency (count) of indel lengths induced by SaCas9/UL30-5 in quiescent <em>d</em>109 genomes. (-) deletions (+) insertions. C. Examples of sequences that show mutations induced by SaCas9/UL30-5 in quiescent <em>d</em>109 viral genomes.<br/><br/><li> Analysis of off-target effects in the human an viral genome<br/><ol class="npwthree"><li>Transduce HFFs with lentivirus expressing SaCas9/sgRNA as above and SaCas9 without sgRNA as control.</li><li>Isolate genomic DNA using the DNeasy Blood &amp; Tissue kit according to the manufacturerâs protocol.</li><li>Perform 30x whole genome sequencing (WGS) of SaCas9/sgRNA treated sample and control.</li><li>To detect off-target effects in the human genome: Map WGS reads to the human genome (hg38) and use mutect2's tumor-normal calling treating the CRISPRed sample as a âtumorâ to identify possible variants within the human genome in the sgRNA treated HFFs relative to the control. Use Cas-OFFinder (<a href="http://www.rgenome.net/cas-offinder/" target="_blank">http://www.rgenome.net/cas-offinder/</a>) in offline mode to look for all predicted off-target sites with 6 or fewer mismatches within hg38 for each sgRNA and determine the overlap between predicted variants and off-target sites.</li><li>To detect off-target effects in the viral (HSV) genome: Align the unmapped reads to the HSV-1 KOS genomic sequence (GenBank: KT899744) and call variants using mutect2's tumor-normal calling. One can then determine the overlap between predicted variants and off-target sites within HSV-1 KOS.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li></ol><li>Functional testing of CRISPR/SaCas9 antiviral activity (see Figure 5A)</li><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904003042_0820.jpg" width="223"/><br/><strong>Figure 5. Effect of CRISPR-Cas9 on HSV-1 lytic infection</strong> (Oh<em> et al.</em>, 2019). A. Experimental scheme of SaCas9/sgRNA-mediated inhibition of HSV lytic infection. B. and C. HFFs transduced with lentivirus expressing SaCas9 and sgRNAs were infected with HSV-1 at an MOI of 0.1 (C) or 5 (D) and harvested at 48 hpi or 24 hpi, respectively. Viral yields were determined by plaque assays. The histogram shows the mean values and standard deviations of biological replicates at an MOI of 0.1 (N = 3) or at an MOI of 5 (N = 4). All the sgRNA added conditions showed statistical significance compared to +Cas9/-gRNA (<em>t</em>-test, *<em>P</em> &lt; 0.05). D. HFFs transduced with lentivirus expressing SaCas9 and sgRNA were infected with HSV-1 at a MOI of 5 and harvested at 10 hpi. Proteins were detected using immunoblotting with antibodies specific for the indicated proteins. Three SDS-PAGE gels loaded with the same amount of proteins were used to detect multiple proteins. Immunoblots of GAPDH were shown as a control under the individual immunoblots.<br/><br/><ol class="npwtwo"><li>Inhibition of viral protein expression: SDS-PAGE and immunoblotting</li><ol class="npwthree"><li>Lyse HFF cells in 1x NuPAGE sample buffer (Life Technologies) with protease inhibitors (cOmplete Protease Inhibitor Cocktail, Millipore Sigma).</li><li>Resolve samples in NuPAGE 4-12% Bis-Tris Gels (Life Technologies).</li><li>Transfer proteins to a nitrocellulose membrane (Bio-Rad).</li><li>Block the membrane in Odyssey Blocking Buffer (LI-COR) for 1 h.</li><li>Incubate the membrane with antibodies specific for individual proteins [anti-ICP4 (1:3,000), anti-ICP8 (1:5,000), anti-ICP27 (1:5,000), anti-UL30 (1:2,000), anti-GAPDH (1:10,000), and anti-HA (1:5,000)] for 1 h at room temperature or overnight at 4 Â°C.</li><li>Wash membrane 5 x 5 min with PBS-T (PBS with 0.1% Tween 20).</li><li>Incubate the membranes with secondary antibodies, IRDye 680RD (1:15,000, LI-COR) or IRDye 800CW (1:15,000, LI-COR), for 45 min. </li><li>Wash membrane 5 x 5 min with PBS-T (PBS with 0.1% Tween 20).</li><li> Detect near-infrared fluorescence using Odyssey (LI-COR) and quantify protein expression levels using Image J or ImageStudio V4 (LI-COR) (see Figure 5D).</li></ol><li>Inhibition of lytic replication</li><ol class="npwthree"><li>Transduction of HFFs with SaCas9/sgRNA:</li><ol class="npwfour"><li>Prepare HFF cells at a density of 2 x 10<sup>5</sup>/well in a T25 flask one day prior to transduction</li><li>Transduce with 2-5 ml of lentivirus (or at an MOI of 5) and 3-4 Âµg/ml of polybrene.</li><li>The next day, replace the transduction medium with fresh medium and incubate cells for additional 2 days at 37 Â°C followed by puromycin treatment (1 Âµg/ml) for 7-10 days.</li></ol><li>Lytic infection with wild-type HSV-1</li><ol class="npwfour"><li>Infect the transduced cells with wild-type HSV-1 at an MOI of 0.1 or 5 in phosphate-buffered saline (PBS) containing 0.1% glucose (wt/vol), 0.1% heat-inactivated BCS (v/v) for 1 h with shaking at 37 Â°C.</li><li>Replace virus inoculum with DMEM containing 1% BCS and incubate at 37 Â°C.</li><li>Harvest cells at various time points for viral replication by qPCR and titer.</li><li>To determine viral titer, harvest media and cells and mix with the same volume of autoclaved powder milk and freeze down at -80 Â°C (addition of sterilized milk to regular media can stabilize HSV (Blaho <em>et al.</em>, 2005).</li><li>To isolate whole DNA (cellular and viral DNAs), isolate genomic DNA using the DNeasy Blood &amp; Tissue kit according to the manufacturerâs protocol.</li></ol><li>Titration of viral yield:<br/><ol class="npwfour"><li>Thaw harvested cells in media/milk and repeat freeze-thaw cycle two times (3 cycles total).</li><li>Dilute the thawed samples in PBS containing 0.1% glucose (wt/vol), 0.1% BCS (v/v) (10 fold serially up to 6-8 logs) and incubate the serially diluted virus inoculum with the appropriate cell type monolayer, which is prepared one day prior, for 1 h with shaking at 37 Â°C.</li><li>Replace the virus inoculum with DMEM containing 1% BCS and 0.17% human IgG and incubate at 37 Â°C for 2-3 days until plaques appear.</li><li>Remove media and fix cells with 100% cold methanol for 10 min.</li><li>Stain monolayer with GIEMSA stain and count plaques (see Figures 5B and 5C)</li></ol></li></ol><li> Inhibition of reactivation of quiescent virus (see Figure 6A)<br/></li><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200904/20200904003432_0344.jpg" width="375"/><br/><strong>Figure 6. CRISPR/Cas9-induced mutagenesis of quiescent <em>d</em>109 genomes and effect on reactivation</strong> (Oh<em> et al</em>., 2019). A. Experimental scheme of SaCas9/sgRNA-mediated inhibition of reactivation of quiescent <em>d</em>109 genomes in HFFs. HFFs were infected with HSV-1<em> d</em>109 virus to establish quiescent infection for 7-10 days and transduced with lentivirus expressing SaCas9 and sgRNAs for 7-10 days. B. and C. HFF were infected with <em>d</em>109 to establish quiescent infection for 7-10 days and transduced with lentivirus expressing SaCas9 and sgRNAs for 7-10 days. To reactivate quiescent <em>d</em>109 genomes, HFFs were superinfected with WT HSV-1 at an MOI of 5 and harvested at 24 hpi. GFP-positive viral yields were determined by plaque assays on FO6 and V27 cells. The histogram shows the mean values and standard deviations of biological replicates (B and C: N = 5 and N = 7 respectively). All the sgRNA added conditions showed statistical significance compared to +Cas9/-gRNA (Ratio paired t test, *<em>P</em> &lt; 0.05 and **<em>P</em> &lt; 0.01).<br/><br/><ol class="npwthree"><li>Establish quiescent <em>HSV-1</em> infection in HFF cells (Ferenczy and DeLuca, 2011):</li><ol class="npwfour"><li>Infect HFF cells with the HSV-1 <em>d</em>109 mutant virus at an MOI of 10 in PBS containing 0.1% glucose (wt/vol) and 0.1% BCS (v/v) for 1 h with shaking at 37 Â°C.</li><li>Incubate at 37 Â°C for 7-10 days in DMEM supplemented with 10% (v/v) FBS.</li></ol><li>Transduction of HFFs containing quiescent HSV-1 viral genomes with SaCas9/sgRNA<br/><ol class="npwfour"><li>Transduce the <em>d</em>109 virus-infected cells with 2 or 10 ml of lentivirus (or at a MOI of 5) and 3-4 Âµg/ml of polybrene in 6-well plates or in T150 flasks.</li><li>The next day, replace the transduction medium with fresh medium and incubate cells for 2 days at 37 Â°C followed by puromycin treatment (1 Âµg/ml) for 7-10 days.</li><li>To reactivate quiescent <em>d</em>109 virus, superinfect HFF cells with wild-type HSV-1 at an MOI of 5 in PBS containing 0.1% glucose (wt/vol) and 0.1% BCS (v/v) for 1 h with shaking at 37 Â°C.</li><li>Replace virus inoculum to DMEM containing 1% BCS for 24 h.</li><li>Harvest cells at 24 hpi and follow the protocol for titration of viral yield in Step C2c above using FO6 and V27 cells.</li><li>To count the number of green plaques, fix the cells with 1% formaldehyde in PBS for 10 min.</li><li>To calculate reactivated <em>d</em>109, count green plaques and subtract the number of GFP-positive plaques on V27 cells from the number of GFP-positive plaques on FO6 cells.</li><li>Reactivation analysis: For the reactivation assay, it is important to include a control for recombination between WT HSV-1 and<em> d</em>109 which could transfer the GFP sequence to WT HSV-1 and result in production of false GFP-positive plaques. We quantify <em>d</em>109 viral reactivation using plaque assays by counting GFP-positive plaques on complementing FO6 cells. FO6 is a Vero-derived cell line expressing ICP4, ICP27, and ICP0 upon HSV-1 infection, thereby complementing replication of<em> d</em>109 virus (Samaniego <em>et al.</em>, 1998). To measure these recombinant viruses, we count GFP-positive plaques formed on V27 cells, which express ICP27 (but not ICP4 and ICP0) upon infection with HSV. Because<em> ICP</em>27 is replaced with <em>GFP</em> in HSV-1 <em>d</em>109, any recombinant GFP-positive but ICP27-negative HSV mutants that arise (but not HSV-1 <em>d</em>109), can replicate in V27 cells. To calculate the number of plaques originating from reactivated <em>d</em>109 genomes, we subtract the number of GFP-positive plaques on V27 cells from the number of GFP-positive plaques on FO6 cells (see Figures 6B and 6C).</li></ol></li></ol></ol></ol><p class="pptt" id="biaoti35593">### Data analysis</p><p class="ppzz"><strong>Controls:</strong> We include a control without sgRNA (SaCas9 only) for all experiments.<br/><br/><strong>Statistical analysis:</strong> We use a minimum of three biological replicates (independent experiments that are performed using the same test at different times) to determine statistical significance. We use Prism 6 (Version 6.01) software (GraphPad Software) for statistical analysis. To determine statistical significance, we use Studentâs <em>t</em>-test, ratio paired <em>t</em>-test, or one-way ANOVA with Dunnettâs multiple comparison test with two-sided (95% confidence level) (Oh <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti35594">### Recipes</p><ol class="npwtwok"><li>Assay buffer (concentrations given are final concentration)</li>200 mM KCl<br/>10 mM MgCl<sub>2</sub><br/>20 mM Tris pH 8<br/>Store at RT<br/><li>Reaction Stop Buffer (Enzyme Assay)</li>30% glycerol by weight 15 g<br/>0.5 M EDTA (6 ml)<br/>2% SDS by weight 1 g<br/>Small amount Bromophenol blue<br/>Bring volume to 50 ml with water and vortex<br/>to mix thoroughly<br/>Store at RT<br/></ol><p class="pptt" id="biaoti35595">### Acknowledgments</p><p class="ppzz">This research was supported by NIH grants AI135423 (KE and DMK) and AI106934 (DK) and a Q-FASTR Award from Harvard University (DMK). This protocol and results are derived from the following original research paper (Oh<em> et al</em>., 2019).</p><p class="pptt" id="biaoti35596">### Competing interests</p><p class="ppzz">The authors declare no conflicts of interest or competing interests.</p><p class="pptt" id="biaoti35597">### References</p><ol class="npwtwok"><li>Bhaya, D., Davison, M. and Barrangou, R. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22060043" target="_blank">CRISPR-Cas systems in bacteria and archaea: versatile small RNAs for adaptive defense and regulation.</a> <em>Annu Rev Genet</em> 45: 273-297.</li><li>Blaho, J. A., Morton, E. R. and Yedowitz, J. C. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18770556" target="_blank">Herpes simplex virus: propagation, quantification, and storage.</a> <em>Curr Protoc Microbiol</em> Chapter 14: Unit 14E 11.</li><li>Chambers, S. M., Qi, Y., Mica, Y., Lee, G., Zhang, X. J., Niu, L., Bilsland, J., Cao, L., Stevens, E., Whiting, P., Shi, S. H. and Studer, L. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22750882" target="_blank">Combined small-molecule inhibition accelerates developmental timing and converts human pluripotent stem cells into nociceptors.</a> <em>Nat Biotechnol</em> 30(7): 715-720.</li><li>Cong, L., Ran, F. A., Cox, D., Lin, S., Barretto, R., Habib, N., Hsu, P. D., Wu, X., Jiang, W., Marraffini, L. A. and Zhang, F. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23287718" target="_blank">Multiplex genome engineering using CRISPR/Cas systems.</a> <em>Science</em> 339(6121): 819-823. </li><li>Doudna, J. A. and Charpentier, E. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25430774" target="_blank">Genome editing. The new frontier of genome engineering with CRISPR-Cas9.</a> <em>Science</em> 346(6213): 1258096. </li><li>Ferenczy, M. W. and DeLuca, N. A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21191021" target="_blank">Reversal of heterochromatic silencing of quiescent herpes simplex virus type 1 by ICP0.</a> <em>J Virol</em> 85(7): 3424-3435.</li><li>Gagnon, J. A., Valen, E., Thyme, S. B., Huang, P., Akhmetova, L., Pauli, A., Montague, T. G., Zimmerman, S., Richter, C. and Schier, A. F. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24873830" target="_blank">Efficient mutagenesis by Cas9 protein-mediated oligonucleotide insertion and large-scale assessment of single-guide RNAs.</a> <em>PLoS One</em> 9, e98186.</li><li>Gaj, T., Gersbach, C. A. and Barbas, C. F., 3rd (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23664777" target="_blank">ZFN, TALEN, and CRISPR/Cas-based methods for genome engineering.</a> <em>Trends Biotechnol </em>31(7): 397-405.</li><li>Hsu, P. D., Lander, E. S. and Zhang, F. (2014). <a href="Development and applications of CRISPR-Cas9 for genome engineering." target="_blank">Development and applications of CRISPR-Cas9 for genome engineering.</a> <em>Cell</em> 157(6): 1262-1278. </li><li>Hsu, P. D., Scott, D. A., Weinstein, J. A., Ran, F. A., Konermann, S., Agarwala, V., Li, Y., Fine, E. J., Wu, X., Shalem, O., Cradick, T. J., Marraffini, L. A., Bao, G. and Zhang, F. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23873081" target="_blank">DNA targeting specificity of RNA-guided Cas9 nucleases.</a> <em>Nat Biotechnol</em> 31(9): 827-832.</li><li>Itzhaki, R. F. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30405395" target="_blank">Corroboration of a Major Role for Herpes Simplex Virus Type 1 in Alzheimer's Disease.</a> <em>Front Aging Neurosci</em> 10: 324.</li><li>Kimberlin, D. W. (2004). <a href="https://www.ncbi.nlm.nih.gov/pubmed/14726453" target="_blank">Neonatal herpes simplex infection.</a> <em>Clin Microbiol Rev </em>17(1): 1-13.</li><li>Kimberlin, D. W., Lin, C. Y., Jacobs, R. F., Powell, D. A., L, F., Gruber, W., Rathore, M., Bradley, J., Diaz, P. S., Kumar, M., Arvin, A. M., Gutierrez, K., Shelton, M., Weiner, L. B., Sleasman, J. W., de Sierra, T. M., Soong, S. J., Lakeman, F. D., Whitley, R. J. and the National Institute of Allergy and Infectious Diseases Collaborative Antiviral Study Group (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11483781" target="_blank">Natural history of neonatal herpes simplex virus infections in the acyclovir era.</a> <em>Pediatrics</em> 108: 223-229. </li><li>Labun, K., Montague, T. G., Gagnon, J. A., Thyme, S. B. and Valen, E. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27185894" target="_blank">CHOPCHOP v2: a web tool for the next generation of CRISPR genome engineering.</a> <em>Nucleic Acids Res</em> 44(W1): W272-276.</li><li>Liesegang, T. J., Melton, L. J., 3rd, Daly, P. J. and Ilstrup, D. M. (1989). <a href="https://www.ncbi.nlm.nih.gov/pubmed/2787981" target="_blank">Epidemiology of ocular herpes simplex. Incidence in Rochester, Minn, 1950 through 1982.</a> <em>Arch</em> <em>Ophthalmol</em> 107(8): 1155-1159.</li><li>Liu, B., Chen, S., Rose, A., Chen, D., Cao, F., Zwinderman, M., Kiemel, D., Aissi, M., Dekker, F. J. and Haisma, H. J. (2020). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31799598" target="_blank">Inhibition of histone deacetylase 1 (HDAC1) and HDAC2 enhances CRISPR/Cas9 genome editing.</a> <em>Nucleic Acids Res</em> 48(2): 517-532. </li><li>Mali, P., Yang, L., Esvelt, K. M., Aach, J., Guell, M., DiCarlo, J. E., Norville, J. E. and Church, G. M. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23287722" target="_blank">RNA-guided human genome engineering via Cas9.</a> <em>Science</em> 339(6121): 823-826.</li><li>Oh, H. S., Neuhausser, W. M., Eggan, P., Angelova, M., Kirchner, R., Eggan, K. C. and Knipe, D. M. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31789594" target="_blank">Herpesviral lytic gene functions render the viral genome susceptible to novel editing by CRISPR/Cas9.</a> <em>Elife</em> 8: e51662.</li><li>Rice, S. A., Su, L. S. and Knipe, D. M. (1989). <a href="http://www.ncbi.nlm.nih.gov/pubmed/2545913" target="_blank">Herpes simplex virus alpha protein ICP27 possesses separable positive and negative regulatory activities.</a><em> J Virol </em>63(8): 3399-3407.</li><li>Roizman, B., Knipe, D. M. and Whitley, R. J. (2013). Herpes Simplex Viruses. Fields Virology. D. M. Knipe and P. M. Howley. Philadelphia, Lippincott Williams &amp; Wilkins: 1823-1897.</li><li>Samaniego, L. A., Neiderhiser, L. and DeLuca, N. A. (1998). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9525658" target="_blank">Persistence and expression of the herpes simplex virus genome in the absence of immediate-early proteins.</a><em> J Virol </em>72(4): 3307-3320.</li><li>Samaniego, L. A., Wu, N. and DeLuca, N. A. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9151855" target="_blank">The herpes simplex virus immediate-early protein ICP0 affects transcription from the viral genome and infected-cell survival in the absence of ICP4 and ICP27.</a> <em>J Virol</em> 71(6): 4614-4625.</li><li>Sanjana, N. E., Shalem, O. and Zhang, F. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25075903" target="_blank">Improved vectors and genome-wide libraries for CRISPR screening.</a> <em>Nat Methods </em>11(8): 783-784. </li><li>Schaffer, P., Vonka, V., Lewis, R. and Benyesh-Melnick, M. (1970). <a href="http://www.ncbi.nlm.nih.gov/pubmed/4321307" target="_blank">Temperature-sensitive mutants of herpes simplex virus.</a> <em>Virology</em> 42(4): 1144-1146. </li><li>Shalem, O., Sanjana, N. E., Hartenian, E., Shi, X., Scott, D. A., Mikkelson, T., Heckl, D., Ebert, B. L., Root, D. E., Doench, J. G. and Zhang, F. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24336571" target="_blank">Genome-scale CRISPR-Cas9 knockout screening in human cells.</a> <em>Science</em> 343(6166): 84-87. </li><li>Thompson, C. and Whitley, R. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21120729" target="_blank">Neonatal herpes simplex virus infections: where are we now?</a> <em>Adv Exp Med Biol</em> 697: 221-230. </li><li>Yarrington, R. M., Verma, S., Schwartz, S., Trautman, J. K. and Carroll, D. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30201707" target="_blank">Nucleosomes inhibit target cleavage by CRISPR-Cas9 <em>in vivo</em></a>. <em>Proc Natl Acad</em> <em>Sci U S A </em>115(38): 9351-9358.</li><li>Young, G. T., Gutteridge, A., Fox, H. D., Wilbrey, A. L., Cao, L., Cho, L. T., Brown, A. R., Benn, C. L., Kammonen, L. R., Friedman, J. H., Bictash, M., Whiting, P., Bilsland, J. G. and Stevens, E. B. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24832007" target="_blank">Characterizing human stem cell-derived sensory neurons at the single-cell level reveals their ion channel expression and utility in pain research.</a> <em>Mol Ther</em> 22(8): 1530-1543.<br/><br/></li></ol>