<body>
            <h1>Probe-Seq: Method for RNA Sequencing of Specific Cell Types from Animal Tissue</h1>
            <p><strong>Authors:</strong> Ryoji Amamoto, Mauricio D. Garcia, Emma R. West, Jiho  Choi, Sylvain W. Lapan, Elizabeth A. Lane, Norbert Perrimon, Constance L. Cepko</p>
            <h2>Abstract</h2>
            <p>Most organs and tissues are composed of many types of cells. To characterize cellular state, various transcription profiling approaches are currently available, including whole-tissue bulk RNA sequencing, single cell RNA sequencing (scRNA-Seq), and cell type-specific RNA sequencing. What is missing in this repertoire is a simple, versatile method for bulk transcriptional profiling of cell types for which cell type-specific genetic markers or antibodies are not readily available. We therefore developed Probe-Seq, which uses hybridization of gene-specific probes to RNA markers for isolation of specific types of cells, to enable downstream FACS isolation and bulk RNA sequencing. We show that this method can enable isolation and profiling of specific cell types from mouse retina, frozen human retina, <em>Drosophila</em> midgut, and developing chick retina, suggesting that it is likely useful for most organisms.</p>
            <p><strong>Keywords:</strong> RNA-sequencing, Transcriptional Profiling, Cell Types, FISH, RNA</p>
            <h2>Background</h2><p>Transcriptional profiling using RNA-Seq and microarrays has become ubiquitous in biological research in the past two decades. Profiling is now one of the main tools used to understand cells and cellular states across most organisms. It is applied to studies of normal development, abnormal development and disease, and has greatly expanded our understanding of evolutionary relationships. In particular, scRNA-Seq has led to the identification of novel cell types at an unprecedented rate (Picelli <em>et al.</em>, 2013; Jaitin <em>et al.</em>, 2014; Klein<em> et al.</em>, 2015; Macosko <em>et al.</em>, 2015). To allow a deeper appreciation of such newly described cell types, a method that allows their isolation, without need for a transgenic marker or specific antigen, would be of great benefit. Although scRNA-Seq can be used to gain deeper coverage of all cell types, this approach entails a great deal of sequencing of abundant cell types to gain even modest coverage of less abundant cell types.<br/><br/>To complement single cell modalities, we developed Probe-Seq, which enables transcriptional profiling of potentially any cell type from any organism, leveraging the newly-available RNA markers defined by scRNA-Seq efforts. Probe-Seq uses gene-specific probe sets designed for SABER Fluorescent <em>in situ</em> Hybridization (FISH) (Kishi<em> et al.</em>, 2019) to fluorescently label specific RNA markers in dissociated, fixed cells or nuclei for downstream FACS isolation and RNA sequencing. While FISH has been previously used for FACS and RNA sequencing of cell cultured <em>in vitro</em> (Klemm <em>et al.</em>, 2014), Probe-Seq works with cells taken directly from tissue, as we show for the mouse retina, human frozen retina, Drosophila midgut, and developing chick retina. To date, it has worked with all organisms and tissues tested, suggesting that it may be useful for any organism (Amamoto<em> et al.</em>, 2019). This method is also cost and time effective (Figure 1). Libraries ready for sequencing can be made by Probe-Seq in less than three days with less than 6 hours of hands-on time. Moreoever, the whole procedure does not require any specialized equipment, and most of the reagents can be found in a general molecular biology lab.<br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915021811_5179.jpg" width="182"/><br/><b style="mso-bidi-font-weight:normal;">Figure 1. Flow Chart for Probe-Seq protocol</b> </p>
        </body>

<p class="ppzz"><span style='font-weight:bold;text-decoration-line:underline;color:#333333;font-family:"'> </span> </p><p class="pptt" id="biaoti35729">## Materials and Reagents</p><p class="ppzz"><strong> Reagents for Probe Synthesis</strong> </p><ol class="npwtwok"><li>10x PBS (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9624 </a> ). Store at RT</li><li>100 mM MgSO<sub>4 </sub>(NEB, catalog number: <a class="blackNoUnder" target="_blank"> B1003S </a> ). Store at -20 Â°C</li><li>dNTP (A, C, T 6 mM each) (NEB, catalog number: <a class="blackNoUnder" target="_blank"> N0446S </a> ). Store at -20 Â°C. Do not mix dGTP</li><li>Clean.G (1 Î¼M), store at -20 Â°C<br/>Clean.G is ordered through IDT with the following sequence (standard desalting, diluted with UltraPure water): CCCCGAAAGTGGCCTCGGGCCTTTTGGCCCGAGGCCACTTTCG</li><li>Bst DNA Polymerase (McLab, catalog number:BPL-300). Store at -20 Â°C</li><li>IDTE pH 7.5 (IDT, catalog number: <a class="blackNoUnder" target="_blank"> 11-05-01-05 </a> ). Store at RT</li><li>Sterile reagent reservoir (<em>i.e.</em>, MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> CLS4870 </a> ). Store at RT</li><li>MinElute PCR Purification Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 28004 </a> ). Store columns at 4 Â°C, everything else at RT. Other purification kits have been tested, but this kit has provided the most reliable results<br/></li><br/></ol><p class="ppzz"><strong>   Reagents for Tissue Dissociation (optional)</strong> </p><ol class="npwtwok"><li>HBSS with calcium/magnesium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14025092 </a> ). Store at RT</li><li>FBS (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10437028 </a> ). Store aliquots at -20 Â°C. Thawed aliquots can be stored at 4 Â°C for up to 2 weeks</li><li>PBS pH 7.4 without calcium/magnesium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10010023 </a> ). Store at RT</li><li>DMEM (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11995065 </a> ). Store at 4 Â°C</li><li>BSA (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> A9418 </a> ). Store at 4 Â°C</li><li>1 M HEPES pH 7.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15630080 </a> ). Store at RT</li><li>50 mM L-Cysteine (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> 168149 </a> )<br/>L-Cysteine is stored at RT. The solution is prepared fresh before experiment. Dissolve 15 mg of L-Cysteine in 2.5 ml of UltraPure water.</li><li>0.5 M EDTA pH 8.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9260G </a> ). Store at RT</li><li>Papain, Suspension (Worthington, catalog number: 00<a class="blackNoUnder" target="_blank"> 3124 </a> ). Store at 4 Â°C</li><li>5-ml Polypropylene Tube (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14-959-11A </a> ). Store at RT</li><li>HBSS/FBS (see Recipes)</li><li>DMEM/BSA (see Recipes)</li><li>Papain Mix (see Recipes)<br/></li><br/></ol><p class="ppzz"> <strong>  Reagents for Probe Hybridization</strong> </p><ol class="npwtwok"><li>UltraPure Water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10977015 </a> ). Store at RT</li><li>16% Paraformaldehyde Ampule (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 28908 </a> ). Store at RT<br/>Prepare 4% PFA by diluting in PBS before each experiment. Mix whole ampule (10 ml) with 30 ml of PBS. Dilute fresh 4% PFA before experiment and discard after use.</li><li>Hoechst 33342 (10 mg/ml) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> H3570 </a> ). Store at 4 Â°C</li><li>10% Triton X-100 (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> T8787 </a> ). Store at RT<br/>100% Triton X-100 is diluted in UltraPure water, and dissolved overnight on a shaker at RT.</li><li>20x SSC (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15557044 </a> ). Store at RT</li><li>Deionized formamide (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> S4117 </a> ). Aliquot and store at -20 Â°C. Do not thaw and refreeze</li><li>Dextran sulfate sodium salt (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> D8906 </a> ). Store powder at 4 Â°C<br/>To make a 50% solution, dissolve 5 g of powdered dextran sulfate in 8 ml of UltraPure water overnight, then bring total volume to 10 ml. Store 50% dextran sulfate at -20 Â°C</li><li>RNasin Plus (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N2615 </a> ). Store at -20 Â°C</li><li>5 ml round-bottom polysterene test tube with 35 Âµm cell strainer cap (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 352235 </a> ). Store at RT<br/>Depending on the tissue and cell type of interest, other strainer sizes are available (<em>i.e</em>., 70 Âµm).</li><li>Oligo probes and hairpin (see section on âOligo and hairpin orderingâ)</li><li>Fluorescent oligos (see section on âProbe Hybridization Recipesâ)</li><li>Permeabilization Buffer (see Recipes)</li><li>FACS Collection Solution (see Recipes)</li><li>40% wHyb (see Recipes)</li><li>Hyb1 (see Recipes)</li><li>Probe Mix (see Recipes)</li><li>Fluorescent Oligo Mix (see Recipes)<br/></li><br/></ol><p class="ppzz"><strong>   Reagents for Library Preparation</strong> </p><ol class="npwtwok"><li>RecoverAll Total Nucleic Acid Isolation Kit for FFPE (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM1975 </a> ). Store components as indicated in the kit user manual</li><li>DNA LoBind microcentrifuge tube (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> Z666548 </a> ). Store at RT</li><li>SMART-Seq v.4 Ultra Low Input RNA Kit (Takara Bio, catalog number: <a class="blackNoUnder" target="_blank"> 634889 </a> ). Store components as indicated in the kit user manual</li><li>Nextera XT DNA Library Preparation Kit (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> FC-131-1024 </a> ). Store at -20 Â°C</li><li>Nextera XT Index Kit (Illlumina, catalog number: <a class="blackNoUnder" target="_blank"> FC-131-1001 </a> ). Store at -20 Â°C</li><li>Ampure XP Beads (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> A63880 </a> ). Store at 4 Â°C Necessary for cDNA purification during RNA-Seq library preparation</li><li>High Sensitivity DNA Analysis Kit (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5067-4626 </a> ). Store at 4 Â°C<br/></li><br/></ol><p class="ppzz">   <strong>Reagents for Nuclei Isolation (optional)</strong> </p><ol class="npwtwok"><li>Sucrose (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> S9378 </a> ). Store at RT</li><li>KCl (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> P9541 </a> ). Store at RT</li><li>MgCl<sub>2</sub> (MilliporeSigma, catalog number: <a class="blackNoUnder" target="_blank"> M8266 </a> ). Store at RT</li><li>1 M Tris Buffer pH 8.0 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9855G </a> ). Store at RT</li><li>Protease Inhibitor Cocktail (50x) (Promega, catalog number: <a class="blackNoUnder" target="_blank"> G6521 </a> ) Reconstitute in DMSO. Store at 4 Â°C It will be frozen at 4 Â°C</li><li>DTT (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 10197777001 </a> ). Store at RT</li><li>Isopentane (Millipore Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 32361 </a> ). Store at RT</li><li>Buffer 1 (see Recipes)</li><li>Homogenization Buffer (see Recipes)</li><li>Sucrose Buffer (see Recipes)</li><li>Sucrose Bed (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35730">### Equipment</p><ol class="npwtwok"><li>P200 multichannel pipette</li><li>P1000 pipette</li><li>Hybridization Oven (Bellco, catalog number: <a class="blackNoUnder" target="_blank"> 7930-01230 </a> )<br/>Any other hybridization oven that can be set at 37 Â°C and 43 Â°C will likely work. Make sure to measure the temperature with a thermometer before starting. If a hybridization oven is not available, a water bath should work, although it has not been tested. Ideally, 2 ovens should be used.</li><li>Centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5804 </a> )</li><li>Centrifuge rotor (Eppendorf, model: <a class="blackNoUnder" target="_blank"> A-4-44 </a> )<br/>Any other centrifuge/rotor that can hold 5-ml polypropylene tubes should suffice. Ideally, a swinging bucket rotor so that the pellet settles at the bottom of the tube.</li><li>FACS Aria with 405 nm, 488 nm, 561 nm, and 633 nm lasers (BD Biosciences)<br/>Other FACS machines can be used. We have also tested MoFlo Astrios EQ with success. The important parameter to take note of is the availability of lasers. If using a 633 nm laser, make sure to use fluorescent oligos with 633 nm excitation fluorophore.</li><li>BioAnalyzer 2100 (Agilent)</li><li>Thermal cycler (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> C1000 </a> ). Any other thermal cycler should suffice</li><li>Dounce homogenizer (Thomas Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1234F37 </a> ). Only for nuclei isolation<br/></li></ol><p class="pptt" id="biaoti35731">### Software</p><ol class="npwtwok"><li>OligoMiner (Brian Beliveau, GitHub,<a href="https://github.com/beliveau-lab/OligoMiner" target="_blank"> https://github.com/beliveau-lab/OligoMiner</a>)</li><li>UCSC Genome Browser (UCSC, <a href="https://genome.ucsc.edu/" target="_blank">https://genome.ucsc.edu</a>)</li><li>Brackets (Adobe, <a href="https://brackets.io/" target="_blank">https://brackets.io</a>)<br/></li></ol><p class="pptt" id="biaoti35732">### Procedure</p><p class="ppzz"><strong>Gene-Specific Probe Set Synthesis</strong><br/><em>Note: For optimal isolation of cell types of interest using Probe-Seq, the marker used is the most important parameter. Based on other methods such as single cell RNA sequencing, it should be verified that the marker is expressed highly and specifically in the cell type of interest. Then, check the length of the transcript. Ideally, the transcript is at least 2 kb to ensure that enough oligo sequences are generated in the BED file. It is possible, however, that shorter transcript (i.e., 1 kb) will produce enough oligo sequences (&gt; 20 oligos).</em><br/><br/><strong>Oligo Design</strong><br/>Adapted from the original protocols on Primer Exchange Reaction and SABER (Kishi <em>et al.</em>, 2018 and 2019).<br/><em>Notes:</em> </p><ol class="npwthree"><li><em>Brian Beliveau has previously published Genome-wide probe sets for C. elegans, D. melanogaster, H. sapiens, A. thaliana, M. musculus, and S. danios. If you would like to use other species, please refer to OligoMiner (Beliveau et al., 2018) to generate your genome-wide probe set.</em> </li><li><em>A user-friendly web-version of probe design software called PaintSHOP has been recently created (Hershberg et al., 2020). The website can be accessed here: <a href="https://oligo.shinyapps.io/paintshop/" target="_blank">https://oligo.shinyapps.io/paintshop/</a>.</em><br/><em> The manual probe design protocol described below uses genome-wide probe sets with a length window of 35-41 nucleotides while the new Balanced probe sets for PaintSHOP has a length window of 30-37 nucleotides. Therefore, more oligos are expected using the PaintSHOP software.</em> </li><li><em>Another probe design software is available (Passaro et al., 2020). The website can be accessed here: <a href="http://oligominerapp.org/home" target="_blank">http://oligominerapp.org/home</a>.</em> </li></ol><p><br/></p><p class="ppzz"><em>However, probe design for Probe-Seq did not use these websitse because they were not available at the time. Therefore, we have not tested the probes generated by these software.</em><br/><br/>Before you start:<br/>â¢Get access to your institutionâs computing cluster.<br/>â¢Create a folder on the cluster (<em>i.e.</em>, âSABER_Probe_Designâ).<br/>â¢Download Brian Beliveauâs OligoMiner-master folder from Github: <a href="https://github.com/brianbeliveau/OligoMiner" target="_blank">https://github.com/brianbeliveau/OligoMiner</a>.<br/>â¢Transfer the OligoMiner-master folder in the SABER_Probe_Design folder on the cluster. For example, use the rsync command in the Terminal (Mac) to transfer files between the cluster and the local computer (example shown below).<br/>â¢Download the Genome-wide probe set for your favorite organism from Ting Wuâs OligoPaint website: <a href="https://oligopaints.hms.harvard.edu/genome-files" target="_blank">https://oligopaints.hms.harvard.edu/genome-files</a>.<br/>â¢Download the âComplete Genomeâ with the âBalanceâ setting. Unzip the file and transfer the folder in the SABER_Probe_Design folder on the cluster.<br/><br/><strong>Obtaining the gene-specific BED file (30 min)</strong> </p><ol class="npwtwok"><li>Go to the UCSC Genome Browser: <a href="https://genome.ucsc.edu/" target="_blank">https://genome.ucsc.edu/</a>.</li><li>Click on Genomes and go to the genome browser of your organism (<em>i.e.</em>, GRCm38/mm10 for mouse).</li><li>Enter the gene of your choice in the search box (<em>i.e.</em>, Grik1), click on the correct mRNA transcript in the drop-down menu [<em>i.e.</em>, Grik1 (Mus musculus glutamate receptor, ionotropic, kainate 1 (Grik1), transcript variant 1, mRNA (from RefSeq NM_146072)], and hit Enter.</li><li>With the correct mRNA in the viewer, hover the cursor over Tools, and click on Table Browser in the drop-down menu.</li><li>Fill in as follows:<br/>Group: Genes and Gene Predictions<br/>Track: NCBI RefSeq<br/>Table: UCSC RefSeq (refGene)<br/>Region: position<br/>Output format: BED<br/>File type returned: plain text</li><li>Name the output file (<em>i.e.</em>, Grik1_Mm).</li><li>Click on Get Output.</li><li>Click on âExons plus 0 bases at each endâ.<br/><em>Note: If the target RNA is short and does not produce sufficient number of oligos, including introns may help. This is especially true if using nuclei as most of the RNA will be nascent.</em> </li><li>Click âget BEDâ. This will download your BED file.</li><li>Open the BED file. We use a software called Brackets, which is free to download at: <a href="http://brackets.io/" target="_blank">http://brackets.io/</a>.</li><li>If there are multiple isoforms listed (<em>i.e</em>., multiple NM_# with different #), choose one of the isoforms to keep and manually delete the others. Save the file.</li><li>Note the 2 features in the BED file as you will need them later:<br/>a.The chromosome number.<br/>b.Strand orientation â+â or â-â.</li><li>Transfer the BED file to the SABER_Probe_Design folder on the cluster [<em>i.e</em>., type <span style="font-family:Courier New;">ârsync Grik1_Mm user@o2.hms.harvard.edu:/home/user/SABER_Probe_Designâ</span> in the Terminal (Mac)].</li><li>Log into the cluster.<br/><em>Note: How you log into your cluster will be different institution to institution.</em> </li><li>Run an interactive mode (<em>i.e.</em>, â<span style="font-family:Courier New;">srun --pty -p interactive --mem 500M -t 0-06:00 /bin/bashâ</span>).</li><li>Load these modules:<br/>a.conda2 (<em>i.e.</em>, type <span style="font-family:Courier New;">âmodule load conda2â</span>).<br/>b.bedtools (<em>i.e.</em>, type <span style="font-family:Courier New;">âmodule load bedtools/2.27.1â</span>).<br/>c.Any other dependencies (<em>i.e.</em>, gcc).<br/><em>Note: The versions will be different for every cluster.</em> </li><li>Run intersectBed between the genome-wide probe set (using the correct chromosome BED file) and your BED file with the option ofâf 1. (<em>i.e</em>., Type in <span style="font-family:Courier New;">âintersectBed âa mm10b/mm10_chr21b.bed âb Grik1_Mm âf 1 &gt; Grik1_Mm+_probes.bedâ</span>)<br/><em>Notes:</em><br/><em> a.Make sure youâre running this command in the folder with the genome-wide probe set (i.e., mm10b) and your gene-specific BED file (i.e., Grik1_Mm).</em><br/><em> b.If you have enough computing power, itâs also possible to run this command locally.</em> </li><li>If your gene-specific BED file had a â+â orientation, you need to reverse complement. Use the probeRC.py file from the OligoMiner-master folder (<em>i.e.</em>, Type in <span style="font-family:Courier New;">âpython OligoMiner-master/probeRC.py âf Grik1_Mm_probes.bed âo Grik1_Mm-_probesâ</span>)</li><li>Transfer the finalized gene-specific probe set file to the local computer (<em>i.e.</em>, using the Terminal, be in a folder on the local computer, and type <span style="font-family:Courier New;">ârsync user@o2.hms.harvard.edu:/home/ra108/SABER_Probe_Design/Grik1_Mm-_probes.bed.â</span> </li><li>Open the gene-specific probe set file (<em>i.e.</em>, using Brackets) and see how many oligos you have.<br/><em>Note: Sufficient number of oligos is crucial for successful Probe-Seq. We have used as few as 12 oligos for high expressing transcripts and saw a nice separation by FACS. However, in general, more is better. For low expressing transcripts, or if fewer than &lt; 20 oligos, Probe-Seq may not work well.</em><br/><br/><p class="ppzz" style="margin-left:-30px;"><strong>Oligo and hairpin ordering (10 min + wait time for synthesis and delivery)</strong><br/><em>Note: The oligos without the primer sequence contain the sequences for hybridization to the RNA, but not the hairpin concatemers that are hybridized by fluorescent oligos. Specific hairpin primer sequences are appended to the oligo sequences to attach hairpin concatemers during the probe set synthesis step (see below). Therefore, at this step, determine which hairpin will be used for which gene. If using two genes for one sort, two different hairpin primer sequences should be added because fluorescent oligos will hybridize to the hairpin region. For example, in order to sort two populations using Vsx2 and Grik1, for Vsx2, add the 25.25 hairpin primer sequence (âCCAATAATAâ) to each oligo sequence and extend Vsx2 oligos with the 25 hairpin during the probe set synthesis step, while for Grik1, add the 27.27 hairpin primer sequence (âCATCATCATâ) to each oligo sequence and extend Grik1 oligos with the 27 hairpin during the probe set synthesis step. Then, hybridize 25.488 nm fluorescent oligo and 27.565 nm fluorescent oligo to detect Vsx2 and Grik1, respectively.</em> </p></li><p><br/></p><li>For ordering the oligos, we use IDT. Oligos from other companies have not been tested.</li><li>Go to www.idtdna.com and under âProducts &amp; Servicesâ click on âCustom DNA oligos.â</li><li>Under âDNA oligos,â click on âOrder Nowâ and âAll ordering options.â</li><li>Under âOrdering,â click on âPlatesâ and order 25 nmole DNA Plate oligo in a 96 well format.<br/><em>Note: For most experiments, 25 nmole permits thousands of Probe-Seq experiments, which is likely excessive. If possible, order less DNA (i.e., 10 nmole) in order to decrease cost.</em> </li><li>Click on âUpload Platesâ and download the sample ordering template.</li><li>Open the template with Excel.</li><li>Copy and paste the contents of the gene-specific probe set file into a region of the Excel template that is not being used.</li><li>Copy the oligo sequences into the column under âSequences.â Delete the other pasted information.</li><li>For each oligo sequence, add âTTTâ and a primer sequence. From our experience, the hairpin primer sequences that have worked well are:<br/>25.25 CCAATAATA<br/>27.27 CATCATCAT<br/>28.28 CAACTTAAC<br/>36.36 AACTAATCT<br/><em>Note: Other hairpins have been used by our group and others. However, these are the hairpins that have worked most consistently. For other hairpins, please see the <a href="https://os.bio-protocol.org/attached/file/20200729/Oligonucleotide List.xlsx" target="_blank">Oligonucleotide_List file.</a></em> </li><li>For each line, the sequence should read â(oligo sequence)TTT(primer sequence)â.<br/><em>Note: You can find examples of sequences in the<a href="https://os.bio-protocol.org/attached/file/20200729/Oligonucleotide List.xlsx" target="_blank"> Oligonucleotide_List file</a>.</em> </li><li>Name each line with the name of the gene, hairpin used, and a number (<em>i.e.</em>, Grik1_25_1, Grik1_25_2...).</li><li>Many genes can fit on one plate, but itâs advisable that a new gene starts in a new row every time (<em>i.e</em>., If Grik1 oligos use 30 wells, and thus 3 rows: 2 full rows + 6 wells, then the Grm6 oligos should start on the 4<sup>th</sup> row).</li><li>Upload the Excel file on IDT.</li><li>Under Plate Specification, use the following:<br/>Scale: 10 nmole (or 25 nmole)<br/>Purification: Standard Desalting<br/>Plate Type: V-Bottom<br/>Ship Option: Wet<br/>Buffer: IDTE 7.5 pH<br/>Normalization Type: Normalized Yield<br/>Quantity: 5 nmol<br/>Concentration: 100 ÂµM<br/>Volume: 50 Âµl</li><li>Order plate.</li><li>When the plate arrives, thaw the oligos, and use a P200 multichannel pipette to pipette the oligos into a sterile reservoir (<em>i.e.</em>, MilliporeSigma) to pool the oligos. The pooled oligos can be stored at -20 Â°C.</li><li>Make a working stock of the pooled oligos (10 ÂµM) by diluting the 100 ÂµM stock with IDTE pH 7.5 (IDT).<br/><em>Notes:</em><br/><em> a.Do not use other TE buffers.</em><br/><em> b.The working stock can undergo freeze-thaw multiple times.</em> </li><li>Order hairpins as written in the Oligonucleotide_List or Supplementary Files of the SABER (Kishi <em>et al.</em>, 2019) or Probe-Seq (Amamoto<em> et al.</em>, 2019) paper. They can be ordered dry and resuspended in IDTE pH 7.5 as 100 ÂµM stock solution. Working stock (5 ÂµM) can be made using IDTE pH 7.5 and aliquoted for further use.<br/><em>Note: The 25 hairpin requires a InvdT at the 3â end and HPLC purification.</em><br/><br/><strong></strong> <p class="ppzz" style="margin-left:-30px;"><strong>Probe set synthesis (2-3 h)</strong> </p></li><li>Add the following regents in a PCR tube on ice:<br/>UltraPure Water                            44.5 Âµl<br/>10x PBS                                          10 Âµl<br/>100mM MgSO<sub>4</sub> (NEB)                    10 Âµl<br/>dNTP(A, C, T 6 mM each, NEB)      5 Âµl<br/>Clean.G (1 ÂµM)                                10 Âµl<br/>BST enzyme (McLab)                       0.5 Âµl<br/>Hairpin (5 ÂµM)                                  10 Âµl<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>6 mM A, C, T dNTP is made by mixing dATP, dCTP, and dTTP to a final concentration of 6 mM.</em> </li><li><em>Clean.G is ordered through IDT with the following sequence (standard desalting, diluted with UltraPure water): CCCCGAAAGTGGCCTCGGGCCTTTTGGCCCGAGGCCACTTTCG.</em><em></em><br/></li></ol></li><li>In a thermal cycler, incubate for 15 min at 37 Â°C.</li><li>Add 10 Âµl of the oligo pool (10 ÂµM) and mix.</li><li>Incubate for 100 min at 37 Â°C, incubate for 20 min at 80 Â°C, incubate indefinitely at 4 Â°C. You can store this unpurified reaction at -20 Â°C.</li><li>Check the length of the extended probe set by taking 8 Âµl of the unpurified probe set and mix it with 1.6 Âµl of 6x Loading Dye (or any other Loading Dye should suffice). Run a 1.25% ethidium bromide agarose gel (2-3 Âµl of EtBr for 50 ml volume) at 150 V for 8 min. The band should be between 300-700 bp. See Figure 2 for an example gel.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Longer running time will lead to fainter band. Two bands are expectedâthe top band is your extended probe and the bottom band contains catalytic hairpins and the Clean.G oligo. Even after purification, this bottom band remains and may even help stabilize the long ssDNA probes during storage.</em> </li><li><em>If the probe set is too short or too long, you can change the extension time from 100 min to 80 min (if too long) or 120 min (it too short). The hairpin amount can also be changed. For example, if you used 10 Âµl of hairpin and your probe runs at ~250 bp, you can try doubling the hairpin to 20 Âµl (and decrease the amount of water in the reaction) to theoretically double the length to 500 bp.</em><br/><br/></li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915021912_9384.jpg"/><br/><strong>Figure 2. Example DNA gel of three gene-specific probe sets. </strong>The left lane is the DNA ladder with the 500 bp band indicated in red. For each lane, the bottom band is the unextended hairpin and other small DNA pieces. The top band is the extended probe set. The bands are always fuzzy because the probes are ssDNA.</li><br/><li>Purify the remaining 92 Âµl of probe set using a MinElute PCR Purification Kit (Qiagen). We use 7x PB buffer (instead of 5x) to increase binding efficiency.</li><li>Elute in 25 Âµl of UltraPure water.</li><li>Determine the concentration by NanoDrop using the ssDNA setting. The concentration should be ~200-400 ng/Âµl.</li><li>Store purified probe set at -20 Â°C. Freeze-thaw is tolerated.<br/></li><br/></ol><p class="ppzz"><strong> <em>Probe-Seq: Day 1</em></strong><br/><em> Notes:</em> </p><ol class="npwthree"><li><em>If Probe-Seq is not successful for any reason, please check the Troubleshooting Guide at the end of the document.</em> </li><li><em>If you are interested in using nuclei from frozen tissue, please refer to the Appendix.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><p class="ppzz"><strong> Tissue Dissociation (30 min)</strong><br/><em> Notes:</em> </p><ol class="npwthree"><li><p class="ppzz" style="margin-left:10px;"><em>Dissociation may differ for different tissues. This dissociation protocol is for the adult mouse retina.</em> </p></li><li><em>If starting with already dissociated cells, start with Probe Hybridization (Step 12).</em> </li></ol><ol class="npwtwok"><li>Prepare all solutions.<br/><em>Note: Be sure that RNasin has been added to all solutions. Papain mix should be moved to 37 Â°C to warm for ~15 min. HBSS/FBS and DMEM/BSA should be at room temperature. 4% PFA and Permeablization Buffer should be placed on ice. 40% wHyb (warm 500 Âµl per sample) and Probe Mix should be warmed to 43 Â°C.</em> </li><li>Dissect retinas in HBSS or PBS at RT.</li><li>Place retina in a microcentrifuge tube and remove most of the PBS/HBSS without disturbing the retina tissue.</li><li>Add 400 Âµl of the pre-warmed (37 Â°C) Papain Mix.</li><li>Incubate for 7 min at 37 Â°C with no agitation.</li><li>Spin 600<em> x g </em>(rcf) for 2.5 min at RT.</li><li>Remove supernatant and add 1 ml of HBSS/FBS without agitation.</li><li>Spin 600<em> x g</em> for 2.5 min at RT.</li><li>Remove supernatant and add 600 Âµl of DMEM/BSA.</li><li>Dissociate by triturating the tissue with a P1000 pipette. ~15 times in/out pipetting should be sufficient to dissociate into single cells. The solution should turn homogenous with no visible clumps.<br/><em>Note: Avoid bubbles. If large chunks of debris or clumped cells are present at this stage, carefully remove them using a pipette or forceps.</em> </li><li>Transfer homogenate to a 5 ml polypropylene tube (Thermo Fisher Scientific). Make sure to seal the cap by pushing in.<br/><br/><strong></strong> <p class="ppzz" style="margin-left:-30px;"><strong>Probe Hybridization (~1 h hands on time + overnight incubation)</strong> </p></li><li>Spin 600 <em>x g </em>for 5 min at 4 Â°C.<br/><em>Note: If a temperature-controlled centrifuge is not available, RT spin is also acceptable. However, there may be slightly more RNA degradation.</em> </li><li>Remove supernatant and add 1 ml of 4% PFA. Resuspend. Incubate for 15 min at 4 Â°C with rocking.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>To minimize cell loss, ~50-100 Âµl of liquid can be left at the bottom after each spin step.</em> </li><li><em>For tissues with a high amount of debris (i.e., mouse or human brain), an extra filtering with a FACS cell strainer tube can be done after the 15 min incubation. This may improve the staining index.</em><em></em><br/></li></ol></li><li>Spin 2,000 <em>x g</em> for 5 min at 4 Â°C.<br/><em>Note: The centrifugation speed has changed to 2,000 x g after fixation.</em> </li><li>Remove supernatant and add 1 ml of Permeabilization Buffer. Resuspend. Incubate for 10 min at 4 Â°C with rocking.</li><li>Spin 2,000 <em>x g</em> for 5 min at 4 Â°C.</li><li>Remove supernatant and add 500 Âµl of pre-warmed (43 Â°C) 40% wHyb. Resuspend. Incubate for at least 30 min at 43 Â°C in the oven.<br/><em>Note: If an in situ hybridization oven is not available (i.e., Bellco), a water bath should suffice although it has not been tested in our lab.</em> </li><li>Spin 2,000 <em>x g</em> for 5 min at RT.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>When changing liquids, it is critical to perform every step quickly to minimize change in temperature. Ideally, the oven and the centrifuge are close together.</em> </li><li><em>It is crucial that the Probe Mix and the sample are at 43 Â°C when it is added to the sample. If the Probe Mix is added at a lower temperature, significant off-target binding of probes can occur causing unwanted background. This is also important for subsequent addition of wHyb solution for washes.</em> </li><li><em>From here, resuspension is not necessary until FACS. Resuspension will cause cells to get stuck in the pipette tip.</em><em></em><br/></li></ol></li><li>Remove supernatant and add 100 Âµl of pre-warmed (43 Â°C) Probe Mix. Incubate for at least 16 h at 43 Â°C in the oven.<br/><br/><strong><em></em></strong> <p class="ppzz" style="margin-left:-30px;"><strong><em>Probe-Seq: Day 2</em></strong><br/><br/><strong>Fluorescent oligo hybridization (~2 h)</strong><br/><em>Note: If RNA sequencing is not going to be performed for your samples and the RNA quality is not important after FACS, you can perform a longer incubation. Increasing the incubation time to 2 overnight incubations (~48 h) has increased the staining index. However, the RNA quality after such long incubation has not been tested.<br/><br/></em> </p></li><li>Prepare all solutions (see Recipes).<br/><em>Notes:</em><br/><ol class="npwthree"><li><em> Be sure that RNasin has been added to all solutions.</em> </li><li><em>Approximately 30 min before starting, you should warm the 40% wHyb (1 ml/sample) and 2x SSC (1 ml/sample) to 43 Â°C. It is critical that they reach 43 Â°C.</em> </li><li><em>At this point, another oven should be set to 37 Â°C for the fluorescent oligo hybridization and washes. You should warm the Fluorescent Oligo Mix and PBS (1.5 ml/sample) to 37 Â°C.</em> </li><li><em>If you are working with a single oven, you can pause after Step 24 below. Remove the supernatant from the 2x SSC wash (leaving ~100 Î¼l and tightly cap to prevent desiccation) and switch the oven temperature to 37 Â°C. Put the sample in the oven at 37 Â°C and wait ~10 min for equilibration. Then, resume at Step 25 below by adding pre-warmed 37 Â°C PBS.</em><em></em><br/></li></ol></li><li>Add 500 Âµl of pre-warmed (43 Â°C) 40% wHyb and spin 2,000<em> x g </em>for 5 min at RT.<br/><em>Note: The pellet might become transparent after incubation in the 40% wHyb. Continue to leave ~100 Î¼l in the tube when removing supernatant to avoid disturbing the pellet.</em> </li><li>Remove supernatant and add 500 Âµl of pre-warmed (43 Â°C) 40% wHyb. Incubate for 15 min at 43 Â°C.</li><li>Spin 2,000<em> x g</em> for 5 min at RT.</li><li>Remove supernatant and add 1 ml of pre-warmed (43 Â°C) 2x SSC. Incubate for 5 min at 43 Â°C.</li><li>Spin 2,000<em> x g</em> for 5 min at RT.</li><li>Remove supernatant and add 500 Âµl of pre-warmed (37 Â°C) PBS.</li><li>Spin 2,000 <em>x g</em> for 5 min at RT.</li><li>Remove supernatant and add 100 Âµl of pre-warmed (37 Â°C) Fluorescent Oligo Mix. Incubate for 10 min at 37 Â°C.</li><li>Add 500 Âµl of pre-warmed (37 Â°C) PBS and spin 2,000<em> x g</em> for 5 min at RT.</li><li>Remove supernatant and add 500 Âµl of pre-warmed (37 Â°C) PBS. Incubate for 5 min at 37 Â°C.</li><li>Spin 2,000<em> x g </em>for 5 min at RT.<br/><em>Note: Before resuspending, pipette up and down with FACS Collection Solution (BSA/PBS) to coat the inside of the pipette tip with BSA to help minimize cells sticking inside the pipette tip.</em> </li><li>Resuspend in 500-1,000 Âµl of PBS depending on the number of cells. Keep tubes on ice.</li><li>Filter into FACS cell strainer tube and proceed to FACS.<br/><em>Notes:</em><br/><em>a</em><em>.</em> Make sure to bring collection tubes (microcentrifuge tubes) with 500 Âµl of FACS Collection Solution per population.<br/><em>b.</em><em> To visualize the cells before FACS, pipette 10 Âµl of the suspension onto a glass slide and air dry. Coverslip with mounting media.</em><br/><br/><strong></strong> <p class="ppzz" style="margin-left:-30px;"><strong>FACS (~30 min for 1 million cells)</strong> </p></li><li>Gate first based on Hoechst histogram <strong>(Figure 3)</strong>. This step will ensure that you get 2N cells. Make sure to NOT include doublets. You can also gate based on FSC/SSC for cells vs. debris, but this step is likely unnecessary as debris will be negative for Hoechst.<br/><em>Note: If your tissue contains polyploid cells (4N+ cells), it will be difficult to distinguish them from doublets or other cell clumps. In this case, it may be best to use the nuclei Probe-Seq protocol.</em> </li><li>Gate using a 2D plot with appropriate wavelengths. The negative population will likely run diagonal. The positive population will be left- or right-shifted compared to the diagonal events (<em>i.e.</em>, the Vsx2<sup>+</sup> cells). Often, the separation will not be as obvious as GFP or well characterized cell surface markers. Thus, itâs a good idea to run a negative control (fluorescent oligo only control).<br/><br/><img alt="" class="layerphoto" height="350" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915021953_6239.jpg" width="592"/><br/><strong>Figure 3. FACS gating strategy for Probe-Seq.</strong> A. Using a Hoeschst histogram, single cell peak is gated first. B. The percentage of single cell events will depend on the tissue. For example, the mouse retina will have a higher percentage of single cell events compared to the mouse brain because less debris is generated during the dissociation protocol for the retina. Out of the single cell events, the population of interest is identified. Here, the Vsx2 fluorescence is indicated on the x-axis, and the Vsx2<sup>+</sup> population is right-shifted from the negative population. It is advisable to use an empty autofluorescence channel as the opposing axis as this will help to determine the events that have high fluorescence in all channels. * Do not collect these events even if they have a high fluorescence intensity in the Vsx2 channel. These events are also highly fluorescent for the empty autofluorescence (561 nm) channel. If using a histogram or a plot without an autofluorescence channel, these events will look like they are Vsx2<sup>+</sup> even though they are unlikely to be the population of interest.<br/><br/><em>Note: It is advisable to keep one laser channel unused as this can be used to determine autofluorescence (i.e., the 561 nm channel in Figure 3). If there are events that are highly fluorescent in your channel of interest (i.e., the Vsx2 channel) AND the autofluorescence channel (i.e., 561 nm), these events are likely autofluorescent in all channels (i.e., the cells indicated by <sup>*</sup> in the FACS plot). For cells that are true positive, they generally form a cluster of events as opposed to a stream. Thus, these events should not be collected.</em> </li><br/><li>FACS isolated cells are sorted into FACS Collection Solution in a microcentrifuge tube (start with 500 Âµl per population already in the tube) and kept at 4 Â°C. This is <u>NOT</u> a stopping point. Go to the decrosslinking step ASAP.<br/><em>Note: The BSA in the FACS Collection Solution helps pellet the cells in the subsequent steps.</em><br/><br/><strong></strong> <p class="ppzz" style="margin-left:-30px;"><strong>Decrosslinking and RNA isolation (~4 h)</strong> </p></li><li>Transfer the collected cells to a 5-ml polypropylene tube (with white caps).</li><li>Spin at 3,000 <em>x g</em> for 7 min at 4 Â°C (or at RT if temp-controlled centrifuge is not available)<br/><em>Note: Centrifugation at RT is also acceptable. Note the change in spin speed.</em> </li><li>Remove as much supernatant as possible. Usually, there is ~40-50 Âµl left. Depending on the number of cells, the pellet may not be visible (<em>i.e.</em>, not visible with 10,000 cells).</li><li>From the Recoverall Total Nucleic Acid Isolation Kit for FFPE (Thermo Fisher Scientific), mix 100 Âµl of Digestion Buffer and 4 Âµl of protease for each sample.</li><li>Add the Digestion Buffer/Protease Mix (104 Âµl) to the pellet. Resuspend.</li><li>Transfer the resuspended cells to a DNA LoBind microcentrifuge tube (Eppendorf) if available. Wrap the lid with parafilm.</li><li>Incubate at 50 Â°C for 3 h.<br/><em>Note: This incubation time differs from the manufacturerâs protocol.</em> </li><li>After 3 hours, the samples can be stored at -80 Â°C indefinitely or proceed to next steps according to the kit protocol.<br/><br/><u></u> <p class="ppzz" style="margin-left:-30px;"><u>STOPPING POINT</u> </p></li><br/><li>Prep Isolation Additive/Ethanol mix (for 100 Âµl digest, 120 Âµl isolation + 275 Âµl 100% Ethanol. Usually, there is ~140 Âµl so adjust accordingly).</li><li>Add Isolation/Ethanol Mix and pipette to mix.</li><li>Add onto filter and spin 10,000<em> x g</em> for 30 s at RT.</li><li>Discard flow-through. Wash with 700 Âµl of Wash 1. Spin 10,000<em> x g</em> for 30 s at RT.</li><li>Discard flow-through. Wash with 500 Âµl of Wash 2. Spin 10,000 <em>x g</em> for 30 s at RT.</li><li>Discard flow-through. Spin 10,000 <em>x g</em> for 30 s at RT.</li><li>Prep DNase Mix (6 Âµl 10x DNase Buffer, 4 Âµl DNase, 50 Âµl UltraPure Water).</li><li>Add 60 Âµl of DNase Mix to center of cartridge. Close cap and incubate for 30 min at RT.</li><li>Add 700 Âµl of Wash 1, incubate 1 min at RT. Spin 10,000<em> x g </em>for 30 s at RT.</li><li>Discard flow-through. Wash twice with 500 Âµl of Wash 2.</li><li>Spin 10,000 <em>x g</em> for 1 min to remove residual fluid.</li><li>Elute in ~17 Âµl of UltraPure water (Use more water if lower concentration desired. Eluting with 17 Âµl will leave you with ~12 Âµl of RNA, which is enough for SMART-Seq v.4, Qubit, and/or BioAnalyzer).</li><li>Store RNA at -80 Â°C.<br/><br/><u></u> <p class="ppzz" style="margin-left:-30px;"><u>STOPPING POINT</u><br/><br/><em>Note: Qubit or BioAnalyzer can be used to estimate RNA concentration. If less than 10,000 cells, concentration may not be available by these methods. In this case, proceed directly to SMART-Seq v.4 using 9.5 Âµl of RNA as the starting volume.</em> </p></li><p><br/></p><li>Proceed to the SMART-Seq v.4 protocol per manufacturerâs protocol for cDNA synthesis and amplification. Depending on the number of cells, you will use 1-9.5 Âµl of RNA for cDNA synthesis.<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>The number of cycles will depend on the cell type, RNA concentration, RNA integrity. The goal is to use the least number of cycles to obtain ~500 pg/Âµl of cDNA. This will ensure as little amplification bias as possible while having enough cDNA for the subsequent steps. One strategy would be to perform SMART-Seq with 20-50% of your purified RNA with a high number of cycles (i.e., 18 cycles) and then calculate the number of cycles needed from there.</em> </li><li><em>If you collected the gene-negative and gene-positive populations and you would like to know if you collected the correct cell types, you can run qPCR or ddPCR with cell type specific markers at this stage.</em> </li><li><em>Other Library prepration kits can be used. However, they have not been tested.</em><em></em><br/></li></ol></li><li>Use 150 pg of cDNA for Nextera XT library kit.<br/><em>Note: HS DNA BioAnalyzer chip should be run on the BioAnalyzer 2100 after the SMART-Seq v.4 protocol and after Nextera XT indexing to ensure proper library construction.</em><br/></li><br/></ol><p class="ppzz"><strong>Appendix: Nuclei Isolation (30 min)</strong><br/><em>Notes:</em> </p><ol class="npwthree"><li><em>All of the FISH reagents remain the same as the whole cell protocol. Before beginning, make sure to add RNasin to every solution. Also, before beginning the nuclei isolation, make sure to pre-warm the reagents. 40% wHyb (warm 500 Âµl per sample) and Probe Mix should be warmed to 43 Â°C.</em> </li><li><em>Because you are using only nuclear RNA as the marker, the signal intensity is likely to decrease compared to whole cell. Therefore, the expression level and transcript length of the marker gene will be more critical.</em> </li><li><em>If not enough signal is detected, you can design the oligos with both Exon and Intron sequences, which should be present in nascent nuclear RNA.</em> </li><li><em>The total extracted RNA is less abundant compared to whole cell. Therefore, when performing SMART-Seq v.4, more cycles will be needed to amplify your cDNA.</em> </li><li><em>This nuclei isolation method works for mouse and human retina and brain. It has not been tested for other tissues in other organisms.</em> </li><li><em>This nuclei isolation is meant for frozen tissue. If you have fresh tissue, flash-freeze the tissue in isopentane/dry ice slurry. This nuclei isolation protocol has not been tested with fixed-frozen tissue. Keep the collection tubes on dry ice before starting. The tissue goes directly into the slurry, without too much residual liquid around the tissue. Quickly transfer the frozen tissue into the collection tube. Keep at -80 Â°C.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol><ol class="npwtwok"><li>Prepare all solutions and keep on ice with the Dounce homogenizer (<em>i.e.</em>, Thomas Scientific).</li><li>Fill the glass Dounce homogenizer with 1 ml of cold homogenization buffer. Keep on ice.</li><li>Mince the tissue into little pieces and place in 1% PFA for 5 min on ice.</li><li>Transfer the tissue pieces into the homogenizer with the buffer. With the tight pestle, homogenize with 10-15 strokes on ice. Avoid foaming.</li><li>Transfer the homogenate into a 5-ml polypropylene tube (Thermo Fisher Scientific). Make sure to seal the cap by pushing in.</li><li>Carefully add 2 ml of Sucrose Bed solution to the bottom of the tube so that the homogenate is above the sucrose solution.</li><li>Spin at 500<em> x g</em>, 12 min, 4 Â°C.</li><li>Remove supernatant. Add 1 ml of 4% PFA solution and resuspend pellet. Incubate for 15 min at 4 Â°C with rocking.<br/><em>Note: Remove the top layer supernatant first. Then, remove the rest of the supernatant.</em> </li><li>Centrifuge at 2,000 <em>x g</em> for 5 min at 4 Â°C.</li><li>Go to Step 17 under Probe Hybridization of the Probe-Seq protocol.<br/></li></ol><p class="pptt" id="biaoti35733">### Notes</p><p class="ppzz"><strong>Troubleshooting Guide<br/></strong><br/><strong>Table 1. Troubleshooting Guide</strong><br/><img alt="" class="layerphoto" height="550" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917191323_1249.jpg" width="759"/> </p><p class="pptt" id="biaoti35734">### Recipes</p><ol class="npwone"><li>Retina Dissociation Recipes (optional if using other tissue)<br/><em>Notes:</em><br/><em> a.Especially for the Probe Hybridization/Probe Synthesis Reagents, use only the reagents with the catalog numbers listed below. We have not tested a wide range of reagents made by other companies.</em><br/><em> b.These volumes are per sample except the DMEM/BSA, which is for 5 samples. If performing Probe-Seq for more than one sample, adjust volume accordingly.</em><br/>1.HBSS/FBS<br/>    HBSS                    900 Âµl <br/>    FBS                      100 Âµl <br/>    Total Volume      1,000 Âµl<br/>2.DMEM/BSA<br/>    DMEM                3,000 Âµl <br/>    BSA                     12 mg<br/>    Total Volume      3,000 Âµl<br/>3.Papain Mix<br/>    HBSS                             315 Âµl<br/>    1 M HEPES pH 7.0        35 Âµl<br/>    50 mM L-Cysteine         20 Âµl <br/>    0.5 M EDTA pH 8.0        0.4 Âµl <br/>    UltraPure Water            19.6 Âµl<br/>    Papain                             10 Âµl<br/>    Total Volume                  400 Âµl<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Make the above solutions fresh before the experiment.</em> </li><li><em> Keep HBSS/FBS and DMEM/BSA at RT.</em> </li><li><em>The papain mix should be incubated at 37 Â°C for 10 min before use. It will be cloudy initially and become clear after warming.</em> </li><li><em> Add 1 Âµl of RNasin Plus (Promega) for every 1 ml of every solution used. Incubate for at least 10 min with the RNasin before use.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Probe Hybridization Recipes<br/>The details of the SABER FISH reagents can be found in the original SABER FISH paper (Kishi <em>et al.</em>, 2019). For detailed protocol for gene-specific probe set design, see the saber.fish website. Here are the reagents that you will need for Probe-Seq:<br/><ol class="npwtwo"><li>Permeabilization Buffer<br/>PBS                                            989 Âµl<br/>Hoechst 33342 (10 mg/ml)       1 Âµl<br/>10% Triton X-100                       10 Âµl<br/>Total Volume                              1,000 Âµl <br/>Make fresh before experiment</li><li>FACS Collection Solution (1% BSA/PBS)<br/>BSA                                        10 mg<br/>PBS                                         1,000 Âµl <br/>Total Volume                         1,000 Âµl<br/>Make fresh before experiment</li><li>40% wHyb<br/>20x SSC                                         5 ml<br/>UltraPure Water                           25 ml<br/>Deionized formamide                  20 ml<br/>Total Volume                                 50 ml<br/>Store at -20 Â°C. This solution should not freeze at -20 Â°C<br/>Pre-warm to 4 Â°C before use<br/><em>Note: Compared to the original SABER-FISH paper (Kishi et al., 2019), these solutions do not contain Tween-20. This is because we have found that Tween-20 causes clumping in dissociated cells.</em> </li><li>Hyb1<br/>20x SSC                          1 ml<br/>UltraPure Water             1 ml<br/>Deionized formamide     4 ml<br/>50% Dextran sulfate       2 ml<br/>Total Volume                   8 ml<br/>Store aliquots of 120 Âµl at -20 Â°C. This solution should not freeze at -20 Â°C</li><li>Probe Mix<br/>Hyb1                                      96 Âµl<br/>Purified Probe set                 1 Âµg<br/>UltraPure Water                     up to 120 Âµl<br/>Pre-warm to 43 Â°C before use<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>Increasing the amount of purified probe set to 2 Âµg has shown to increase fluorescence intensity. It is unclear if, with 2 Âµg, a point of saturation has been achieved. Therefore, more probe set may improve fluorescence intensity.</em> </li><li><em>The total volume of purified probe sets cannot exceed 24 Âµl.</em><br/></li></ol></li><li>Fluorescent Oligo Mix<br/>PBS                                                            100 Âµl<br/>10 ÂµM Fluorescent oligo                          2 Âµl (each)<br/>Pre-warm to 37 Â°C before use<br/><em>Notes: </em><br/><ol class="npwthree"><li><em> Fluorescent oligos are ordered from IDT. The sequences and the fluorescent tags (on the 5â end) can be found in the <a href="https://os.bio-protocol.org/attached/file/20200729/Oligonucleotide List.xlsx" target="_blank">Oligonucleotide_List file</a>. Listed are 2 different fluorophores per hairpin, but other fluorophores can be used as well. They are resuspended in UltraPure water as 100 ÂµM stocks, and 10 ÂµM working stocks (diluted in water) can be aliquoted and stored at -20 Â°C.</em> </li><li><em>Add 1 Âµl of RNasin Plus (Promega) for every 1 ml of every solution used. Incubate for at least 10 min with the RNasin before use.</em><br/><div style="white-space:nowrap;"><span style="font-size:10pt;"><em></em></span> </div></li></ol></li></ol></li><br/><li>Nuclei Isolation Recipes<br/>1.Buffer 1<br/>    1.5 M Sucrose                   2,500 Âµl<br/>    1 M KCl                               375 Âµl<br/>    1 M MgCl<sub>2  </sub>                         75 Âµl<br/>    1 M Tris Buffer pH 8.0      150 Âµl<br/>    UltraPure Water                11,900 Âµl<br/>    Total Volume                     15,000 Âµl<br/>    Buffer 1 can be stored at 4 Â°C for up to 6 months<br/>2.Homogenization Buffer<br/>    Buffer 1                                968 Âµl<br/>    10% Triton X-100                 10 Âµl<br/>    Protease Inhibitor (50x)     20 Âµl<br/>    1 mM DTT                             1 Âµl <br/>     Hoechst 33342                    1 Âµl <br/>    Total Volume                        1,000 Âµl<br/>    Homogenization Buffer should be made fresh<br/>  <em>  Note: 1 ml of Homogenization Buffer is necessary for each sample.</em><br/>3.Sucrose Buffer<br/>    1 M KCl                          2,250 Âµl<br/>    1 M MgCl<sub>2</sub>                      450 Âµl<br/>    1 M Tris Buffer pH 8.0   900 Âµl<br/>    UltraPure Water             11,400 Âµl<br/>    Total Volume                   15,000 Âµl<br/>    Sucrose Buffer can be stored at 4 Â°C for up to 6 months<br/>4.Sucrose Bed<br/>    Sucrose Buffer               2,500 Âµl<br/>    24% Sucrose                  12,500 Âµl<br/>    Total Volume                  15,000 Âµl<br/>    Sucrose Bed can be stored at 4 Â°C for up to 6 months<br/>   <em> Note: Add 1 Âµl of RNasin Plus (Promega) for every 1 ml of every solution used. Incubate for at least 10 min with the RNasin before use.</em> </li></ol><p class="pptt" id="biaoti35735">### Acknowledgments</p><p class="ppzz">We would like to thank former and current members of the Cepko and Tabin Labs for the insightful discussion and feedback. We thank C. Araneo, F. Lopez, and the Flow Cytometry Core Facility for their assistance with flow cytometry. This work was supported by the Howard Hughes Medical Institute (J.C., C.L.C. and N.P.), Edward R. and Anne G. Lefler Postdoctoral Fellowship (R.A.), HSCI Internship Program (M.D.G), and NIH NEI K99/R00 Grant 5K99EY028215 (S.W.L.). This protocol was derived from the original Probe-Seq and SABER manuscripts (Amamoto <em>et al.</em>, 2019; Kishi <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti35736">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti35737">### Ethics</p><p class="ppzz">All animals were handled according to protocols approved by the Institutional Animal Care and Use Committee (IACUC) of Harvard University (protocol number: 1695).</p><p class="pptt" id="biaoti35738">### References</p><ol class="npwtwok"><li>Amamoto, R., Garcia, M. D., West, E. R., Choi, J., Lapan, S. W., Lane, E. A., Perrimon, N. and Cepko, C. L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31815670" target="_blank">Probe-Seq enables transcriptional profiling of specific cell types from heterogeneous tissue by RNA-based isolation.</a> <em>Elife</em> 8: e51452.</li><li>Beliveau, B. J., Kishi, J. Y., Nir, G., Sasaki, H. M., Saka, S. K., Nguyen, S. C., Wu, C. T. and Yin, P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29463736" target="_blank">OligoMiner provides a rapid, flexible environment for the design of genome-scale oligonucleotide in situ hybridization probes.</a> <em>Proc Natl Acad Sci U S A</em> 115(10): E2183-E2192.</li><li>Jaitin, D. A., Kenigsberg, E., Keren-Shaul, H., Elefant, N., Paul, F., Zaretsky, I., Mildner, A., Cohen, N., Jung, S., Tanay, A. and Amit, I. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24531970" target="_blank">Massively parallel single-cell RNA-seq for marker-free decomposition of tissues into cell types.</a> <em>Science</em> 343(6172): 776-779.</li><li>Hershberg, E. A., Close, J. L., Camplisson, C. K., Attar, S., Chern, R., Liu, Y., Akilesh, S., Nicovich, P. R. and Beliveau, B. J. (2020). <a href="https://www.biorxiv.org/content/10.1101/2020.07.05.188797v1" target="_blank">PaintSHOP enables the interactive design of transcriptome- and genome-scale oligonucleotide FISH experiments.</a> <em>bioRxiv</em> doi: https://doi.org/10.1101/2020.07.05.188797.</li><li>Kishi, J. Y., Lapan, S. W., Beliveau, B. J., West, E. R., Zhu, A., Sasaki, H. M., Saka, S. K., Wang, Y., Cepko, C. L. and Yin, P. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31110282" target="_blank">SABER amplifies FISH: enhanced multiplexed imaging of RNA and DNA in cells and tissues.</a> <em>Nat Methods</em> 16(6): 533-544.</li><li>Kishi, J. Y., Schaus, T. E., Gopalkrishnan, N., Xuan, F. and Yin, P. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29359755" target="_blank">Programmable autonomous synthesis of single-stranded DNA.</a> <em>Nat Chem</em> 10(2): 155-164.</li><li>Klein, A. M., Mazutis, L., Akartuna, I., Tallapragada, N., Veres, A., Li, V., Peshkin, L., Weitz, D. A. and Kirschner, M. W. (2015).<a href="http://www.ncbi.nlm.nih.gov/pubmed/26000487" target="_blank"> Droplet barcoding for single-cell transcriptomics applied to embryonic stem cells.</a> <em>Cell</em> 161(5): 1187-1201.</li><li>Klemm, S., Semrau, S., Wiebrands, K., Mooijman, D., Faddah, D. A., Jaenisch, R. and van Oudenaarden, A. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24681693" target="_blank">Transcriptional profiling of cells sorted by RNA abundance.</a> <em>Nat Methods</em> 11(5): 549-551.</li><li>Macosko, E. Z., Basu, A., Satija, R., Nemesh, J., Shekhar, K., Goldman, M., Tirosh, I., Bialas, A. R., Kamitaki, N., Martersteck, E. M., Trombetta, J. J., Weitz, D. A., Sanes, J. R., Shalek, A. K., Regev, A. and McCarroll, S. A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26000488" target="_blank">Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets.</a> <em>Cell </em>161(5): 1202-1214.</li><li>Passaro, M., Martinovic, M., Bevilacqua, V., Hershberg, E. A., Rossetti, G., Beliveau, B. J, Bonnal, R. J. P. and Pagani, M. (2020). <a href="https://academic.oup.com/nar/article/48/W1/W332/5822955" target="_blank">OligoMinerApp: a web-server application for the design of genome-scale oligonucleotide <em>in situ</em> hybridization probes through the flexible OligoMiner environment.</a> <em>Nucleic Acid Res</em> 48(W1): w332-w339.</li><li>Picelli, S., Bjorklund, A. K., Faridani, O. R., Sagasser, S., Winberg, G. and Sandberg, R. (2013). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24056875" target="_blank">Smart-seq2 for sensitive full-length transcriptome profiling in single cells.</a> <em>Nat Methods</em> 10(11): 1096-1098.<br/></li></ol>