<body>
            <h1><em>In vitro</em> Differentiation of Human iPSC-derived Cardiovascular Progenitor Cells (iPSC-CVPCs)</h1>
            <p><strong>Authors:</strong> Agnieszka DâAntonio-Chronowska, Matteo DâAntonio, Kelly A. Frazer</p>
            <h2>Abstract</h2>
            <p>Induced pluripotent stem cell derived cardiovascular progenitor cells (iPSC-CVPCs) provide an unprecedented platform for examining the molecular underpinnings of cardiac development and disease etiology, but also have great potential to play pivotal roles in the future of regenerative medicine and pharmacogenomic studies. Biobanks like iPSCORE ( Stacey <em>et al.</em>, 2013; Panopoulos <em>et al.</em>, 2017), which contain iPSCs generated from hundreds of genetically and ethnically diverse individuals, are an invaluable resource for conducting these studies. Here, we present an optimized, cost-effective and highly standardized protocol for large-scale derivation of human iPSC-CVPCs using small molecules and purification using metabolic selection. We have successfully applied this protocol to derive iPSC-CVPCs from 154 different iPSCORE iPSC lines obtaining large quantities of highly pure cardiac cells. An important component of our protocol is <u>C</u>ell <u>c</u>onfluency <u>estimate</u>s (ccEstimate), an automated methodology for estimating the time when an iPSC monolayer will reach 80% confluency, which is optimal for initiating iPSC-CVPC derivation, and enables the protocol to be readily used across iPSC lines with different growth rates. Moreover, we showed that cellular heterogeneity across iPSC-CVPCs is due to varying proportions of two distinct cardiac cell types: cardiomyocytes (CMs) and epicardium-derived cells (EPDCs), both of which have been shown to have a critical function in heart regeneration. This protocol eliminates the need of iPSC line-to-line optimization and can be easily adapted and scaled to high-throughput studies or to generate large quantities of cells suitable for regenerative medicine applications.</p>
            <p><strong>Keywords:</strong> Human induced pluripotent stem (iPSC), Cardiovascular progenitor cells (CVPCs), cardiomyocytes (CMs), Epicardium cells (EPDCs), Human induced pluripotent stem cell-derived cardiovascular progenitor cells (iPSC-CVPCs), Human induced pluripotent stem cell-derived cardiomyocytes cells (iPSC-CMs), Human induced pluripotent stem cell-derived epicardium derived cells (iPSC-EPDCs), Cardiovascular disease, Heart, Differentiation, Genetic studies, Small molecules, Pharmacogenomics, Regenerative medicine</p>
            <h2>Background</h2><p>Cardiovascular diseases (CVDs) remain the leading cause of death worldwide and account for about 30% of all mortality causes globally. Coronary artery disease (CAD) and myocardial infarction (MI) are among the most common CVDs, and in the USA alone, every 40 s someone suffers a heart attack (Association, 2016; Heron, 2017; WHO, 2018; Benjamin <em>et al.</em>, 2019; D'Antonio-Chronowska <em>et al.</em>, 2019a). Heart failure results in the death of cardiac muscle cells and is the consequence of morphological and functional changes (cardiac remodeling: necrosis, scar formation, inflammation, fibrosis, dilation and reshaping) that occur in response to pre-existing cardiac conditions, including CAD, MI, hypertension, cardiomyopathy, myocarditis and abnormal cardiac valve function (Cohn <em>et al.</em>, 2000; Reed <em>et al.</em>, 2017). Heart failure is commonly treated with beta blockers, ACE inhibitors and aldosterone antagonists which partially reverse cardiac remodeling and thereby improve prognosis (Reis Filho <em>et al.</em>, 2015), but does not result in the regeneration of cardiac tissue. There are currently several ongoing clinical trials, including ESCORT and DREAM-HF (Menasche <em>et al.</em>, 2015 and 2018; Borow <em>et al.</em>, 2019), which are aimed at evaluating the effectiveness of transplanting iPSC-derived Cardiovascular Progenitor Cells (iPSC-CVPCs) or embryonic stem cell derived CVPCs (ESC-CVPCs) as a therapeutic treatment for heart failure. The ability to generate iPSC-CVPCs in large quantities, as is required for regenerative medicine, using biological material obtained directly from the patient would enable autologous transplantations and thereby eliminate the need of immunosuppression. Thus, the development of a robust and cost-effective protocol for generating large amounts of high-quality iPSC-CVPCs without requiring individualized optimization for each iPSC line is imperative for the advancement of future therapeutic treatments of heart failure.<br/> <br/>Large collections of iPSC-CVPCs (D'Antonio-Chronowska <em>et al.</em>, 2019b) generated from genetically and ethnically diverse individuals could also be used for cost-effective large-scale testing of drugs for cardiotoxicity or proarrhythmic effects. Previous studies (Burridge <em>et al.</em>, 2016) and initiatives like CiPA Project (Blinova <em>et al.</em>, 2017) have shown the utility of iPSC-CVPCs for testing drugs for cardiotoxicity, which, if scaled to examine large collections of iPSC-CVPCs derived from both healthy or disease bearing individuals, could greatly improve the efficiency of testing new drugs for safety, and in turn decrease the cost of drug development.<br/> <br/>We have previously demonstrated the feasibility of using a highly standardized protocol for successfully deriving high quality iPSC-CVPCs from hundreds of iPSC lines reprogrammed from ethnically diverse individuals (D'Antonio-Chronowska <em>et al.</em>, 2019b). In this study, we performed 193 differentiations to derive iPSC-CVPCs from 154 iPSCORE iPSC lines (Panopoulos <em>et al.</em>, 2017) from 144 individuals. We obtained large numbers of high quality cells, specifically, on average we derived 1.5 x 10<sup>8</sup> (and up to 6 x 10<sup>8</sup>) cells from a 450 cm<sup>2</sup> culture with median cardiac troponin T (cTnT; TNNT2) positive cells of 89.2%. Importantly, while previous differentiation studies acknowledged cellular heterogeneity and the presence of beating cardiomyocytes and non-contractile cell types, the origin and cellular identity of the non-contractile cells had not been addressed. We characterized the 154 iPSC-CVPCs lines using single cell RNA-seq and bulk RNAs-seq and determined that across all the iPSC-CVPC samples there were two distinct fetal-like cardiac cell types: cardiomyocytes (CMs) and epicardium-derived cells (EPDCs), which were present in varying proportions. Of note, both CMs and EPDCs have been show to contribute to the post-infarction heart regeneration (Bargehr <em>et al.</em>, 2019). Moreover, our previous studies have shown how molecular characterization of iPSC-CVPCs can result in the identification of genetic variants that contribute to heart development and cardiac pathologies (Benaglio <em>et al.</em>, 2019).<br/> <br/>Protocols to derive cardiac cells from ESCs or iPSCs have been developed to mimic the processes naturally occurring during cardiogenesis. Initially, cardiac cells were derived as embryoid bodies cultures, first as spontaneous differentiations by culturing ESCs in medium containing 20% fetal calf serum or by stimulation with several reagents known to enhance cardiac differentiation like dimethyl sulfoxide, retinoic acid, or 5-aza-2â-deoxycytidine, after which beating cardiac cells were manually or mechanically purified (Maltsev <em>et al.</em>, 1993; Burridge <em>et al.</em>, 2014). Differentiation efficiency was greatly improved by the development of directed differentiation protocols that incorporated recombinant proteins including fibroblast growth factor 2, transforming growth factor Î², superfamily growth factors activin A and BMP4, vascular endothelial growth factor and the WNT inhibitor DKK-1 proteins (Schneider and Mercola, 2001; Marvin <em>et al.</em>, 2001; Beqqali <em>et al.</em>, 2006; Laflamme <em>et al.</em>, 2007), and by modification of the format of cell differentiation from embryoid bodies to monolayer culture (Paige <em>et al.</em>, 2010; Lian <em>et al.</em>, 2012). Further advancements were made by the introduction of small molecule protocols (Lian <em>et al.</em>, 2012 and 2013) and chemically defined differentiation media (Burridge <em>et al.</em>, 2014). Finally, by taking advantage of the adaptation of the developing heart to metabolize lactate, we and others were able to eliminate all non-cardiac cells (Burridge <em>et al.</em>, 2014; D'Antonio-Chronowska <em>et al.</em>, 2019b; Tohyama <em>et al.</em>, 2013). Importantly, previous studies have optimized differentiation protocols to derive cardiac cells from a limited number of iPSC or ESC lines, and in most cases utilized small format culture vessels. Here, we present an optimized, cost-effective and highly standardized protocol which we applied to derive iPSC-CVPCs from 154 genetically and ethnically diverse human iPSC lines in large-sized culture flasks. We optimized the concentration of IWP-2 used to drive the cardiac cell differentiation, which resulted in improved formation of a thick cardiac syncytium and strong wave-like beating (Video 1) (D'Antonio-Chronowska <em>et al.</em>, 2019b). We also demonstrated that simple mechanical disruption of the cardiac syncytium prior to metabolic purification of iPSC-CVPCs using lactate results in improved selection and virtually pure cardiac cells (CMs and EPDCs). Additionally, we developed <u>Cell</u> <u>c</u>onfluency <u>estimate</u>s (ccEstimate), an automated method for estimation of cell confluency during the monolayer stage. ccEstimate estimates the point in time for each iPSC line when the monolayer will reach 80% of confluency, which is optimal condition at which to initiate iPSC-CVPC differentiation. Thereby ccEstimate overcomes some of the technical issues in standardizing a differentiation protocol across iPSC lines which have widely varying growth rates. The derived iPSC-CVPCs beat synchronously, are positive for multiple cardiac markers and can be used directly for molecular or electrophysiological assays like multielectrode array (MEA), or they can be cryopreserved for future analysis. Our optimized protocol allowed us to derive high quality iPSC-CVPC samples from 154 iPSC lines generated from ethnically diverse individuals under identical culturing conditions without the requirement of any individualized optimization steps.</p>
        </body>

<p class="pptt" id="biaoti35753">## Materials and Reagents</p><ol class="npwone"><li>iPSC Cell Culture as described also in detail in DâAntonio-Chronowska <em>et al.</em> (2019)<ol class="npwtwo"><li>6-well plates (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 3506 </a> )</li><li>Syringe filter 0.2 Î¼m (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 28145-501 </a> )</li><li>Soft-Ject<sup>Â®</sup> 3-Part Disposable Syringe, Air-Tite-3 ml (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89215-234 </a> )</li><li>5 ml Borosilicate serological pipettes (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1367827E </a> )</li><li>5 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0050-S01 </a> )</li><li>10 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0100-S01 </a> )</li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent sterile tips with filter compatible with pipettes</li><li>P1000 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-02 </a> ) or other equivalent P1000 sterile tips with filter compatible with pipettes</li><li>15 ml conical tubes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> CNT-15R </a> )</li><li>iPSC cells</li><li>UltraPure<sup>TM</sup> DNase/RNase-Free Distilled Water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10977023 </a> )</li><li>Corning<sup>Â®</sup> Matrigel<sup>Â® </sup>Growth Factor Reduced (GFR) Basement Membrane Matrix (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 354230 </a> )</li><li>mTeSR<sup>TM</sup> 1 (Stem Cell Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 85850 </a> )</li><li>DMEM/F-12 medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11330057 </a> )</li><li>Dispase II, powder (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 17105041 </a> )</li><li>200 proof Ethanol denatured (<em>i.e.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 71002-398 </a> )âto prepare 70% ethanol</li><li>70% ethanol (see Recipes: Table 20)</li><li>Matrigel solution (Matrigel) (see Recipes: Table 1)</li><li>10 mM ROCK inhibitor, Y-27632 dihydrochloride solution (ROCK Inhibitor) (see Recipes: Table 3)</li><li>10x Dispase (see Recipes: Table 4)</li><li>mTeSR<sup>TM</sup> 1 complete medium (mTeSR) (see Recipes: Table 5)<br/><br/></li></ol></li><li>PSC passaging using VerseneâVersene I passage<ol class="npwtwo"><li>6-well plates (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 3506 </a> )</li><li>Automated cell counter slides (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1450019 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent</li><li>5 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0050-S01 </a> )</li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent P200 sterile tips with filter compatible with pipettes</li><li>P1000 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-02 </a> ) or other equivalent P1000 sterile tips with filter compatible with pipettes</li><li>15 ml conical tubes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> CNT-15R </a> )</li><li>iPSC cells</li><li>UltraPure<sup>TM</sup> DNase/RNase-Free Distilled Water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10977023 </a> )</li><li>Corning<sup>Â®</sup> Matrigel<sup>Â®</sup> Growth Factor Reduced (GFR) Basement Membrane Matrix (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 354230 </a> )</li><li>mTeSR<sup>TM</sup> 1 (Stem Cell Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 85850 </a> )</li><li>DMEM/F-12 medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11330057 </a> )</li><li>Versene<sup>Â®</sup> (EDTA) 0.02% (Lonza, catalog number: <a class="blackNoUnder" target="_blank"> 17-711E </a> )</li><li>200 proof Ethanol denatured (<em>i.e.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 71002-398 </a> )âto prepare 70% ethanol</li><li>70% ethanol (see Recipes: Table 20)</li><li>Matrigel solution (Matrigel) (see Recipes: Table 1)</li><li>10 mM ROCK inhibitor, Y-27632 dihydrochloride solution (ROCK Inhibitor) (see Recipes: Table 3)</li><li>mTeSR<sup>TM</sup> 1 complete medium (mTeSR) (see Recipes: Table 5)<br/><br/></li></ol></li><li>iPSC passaging using Versene - Versene II passage<ol class="npwtwo"><li>100 mm tissue culture dishes (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 430167 </a> )</li><li>Automated cell counter slides (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1450019 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent</li><li>5 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0050-S01 </a> )</li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent sterile tips with filter compatible with pipettes</li><li>P1000 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-02 </a> ) or other equivalent P1000 sterile tips with filter compatible with pipettes</li><li>15 ml conical tubes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> CNT-15R </a> )</li><li>iPSC cells</li><li>UltraPure<sup>TM</sup> DNase/RNase-Free Distilled Water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10977023 </a> )</li><li>Corning<sup>Â®</sup> Matrigel<sup>Â®</sup> Growth Factor Reduced (GFR) Basement Membrane Matrix (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 354230 </a> )</li><li>mTeSR<sup>TM</sup> 1 (Stem Cell Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 85850 </a> )</li><li>DMEM/F-12 medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11330057 </a> )</li><li>Versene<sup>Â®</sup> (EDTA) 0.02% (Lonza, catalog number: <a class="blackNoUnder" target="_blank"> 17-711E </a> )</li><li>200 proof Ethanol denatured (<em>i.e.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 71002-398 </a> )âto prepare 70% ethanol</li><li>70% ethanol (see Recipes: Table 20)</li><li>Matrigel solution (Matrigel) (see Recipes: Table 1)</li><li>10 mM ROCK inhibitor, Y-27632 dihydrochloride solution (ROCK Inhibitor) (see Recipes: Table 3)</li><li>mTeSR<sup>TM</sup> 1 complete medium (mTeSR) (see Recipes: Table 5)<br/><br/></li></ol></li><li>Monolayer plating is also described in detail in DâAntonio-Chronowska <em>et al.</em> (2019a)<ol class="npwtwo"><li>100 mm tissue culture dishes (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 430167 </a> )</li><li>Automated cell counter slides (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1450019 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent</li><li>5 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0050-S01 </a> )</li><li>10 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0100-S01 </a> )</li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent P20 sterile tips with filter compatible with pipettes</li><li>P200 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-01 </a> ) or other equivalent P200 sterile tips with filter compatible with pipettes</li><li>P1000 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-02 </a> ) or other equivalent P1000 sterile tips with filter compatible with pipettes</li><li>15 ml conical tubes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> CNT-15R </a> )</li><li>50 ml conical tubes (Bio Pioneer catalog number: <a class="blackNoUnder" target="_blank"> CNT-50R </a> )</li><li>Corning<sup>Â®</sup> Matrigel<sup>Â®</sup> Growth Factor Reduced (GFR) Basement Membrane Matrix (Matrigel) (Corning, catalog number: <a class="blackNoUnder" target="_blank"> 354230 </a> )</li><li>mTeSR<sup>TM</sup> 1 (Stem Cell Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 85850 </a> )</li><li>DMEM/F-12 medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11330-057 </a> )</li><li>Accutase (Innovative Cell Technologies, Inc., catalog number: <a class="blackNoUnder" target="_blank"> AT 104 </a> )</li><li>Trypan Blue Solution, 0.4% (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15250061 </a> )</li><li>ROCK inhibitor, Y-27632 dihydrochloride (Selleck hem, catalog number: <a class="blackNoUnder" target="_blank"> S1049 </a> )</li><li>iPSC cell culture</li><li>200 proof Ethanol denatured (<em>i.e.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 71002-398 </a> )âto prepare 70% ethanol</li><li>70% ethanol (see Recipes: Table 20)</li><li>Matrigel solution (see Recipes: Table 1)</li><li>10 mM ROCK inhibitor, Y-27632 dihydrochloride solution (see Recipes: Table 3)</li><li>mTeSR<sup>TM</sup> 1 complete medium (see Recipes: Table 5)<br/><br/></li></ol></li><li>Estimation of optimal time for initiation of iPSC-CVPCs differentiation using ccEstimate<ol class="npwtwo"><li>iPSCs monolayer</li><li>Marker pen resistant to 70% ethanol<br/><br/></li></ol></li><li>iPSC-CVPCs differentiation<ol class="npwtwo"><li>T150 tissue culture flasks, vented (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> Z707929 </a> )<br/><em>Note: At the time of preparation of this manuscript <a class="blackNoUnder" target="_blank"> Z707929 </a> was no longer available. The same flasks are available under the catalog number <a class="blackNoUnder" target="_blank"> Z707511-36EA </a> (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> Z707511-36EA </a> ).</em> </li><li>Automated cell counter slides (Bio-Rad Laboratories, catalog number: <a class="blackNoUnder" target="_blank"> 1450019 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent</li><li>10 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0100-S01 </a> )</li><li>25 ml Serological pipettes (Bio Pioneer catalog number: <a class="blackNoUnder" target="_blank"> GEX250-S01 </a> )</li><li>50 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX500-S01 </a> )</li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent P20 sterile tips with filter compatible with pipettes</li><li>P200 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-01 </a> ) or other equivalent P200 sterile tips with filter compatible with pipettes</li><li>P1000 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-02 </a> ) or other equivalent P1000 sterile tips with filter compatible with pipettes</li><li>Cell scraper (VWR International, catalog number: <a class="blackNoUnder" target="_blank"> 179707 </a> )</li><li>15 ml conical tubes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> CNT-15R </a> )</li><li>50 ml conical tubes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> CNT-50R </a> )</li><li>125 ml Nalgene<sup>Â®</sup> PET sterile bottle (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 342040-0125 </a> )</li><li>Nalgene<sup>Â®</sup> Cryogenic vials (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 5000-1020 </a> )</li><li>iPSCs monolayer</li><li>Corning<sup>Â®</sup> Matrigel<sup>Â®</sup> Growth Factor Reduced (GFR) Basement Membrane Matrix (Matrigel) (Corning<sup>Â®</sup>, catalog number: <a class="blackNoUnder" target="_blank"> 354230 </a> )</li><li>RPMI 1640 medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11875119 </a> )</li><li>RPMI 1640 medium, no glucose (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11879020 </a> )</li><li>1x Dulbeccoâs phosphate buffered saline (DPBS; PBS) without calcium and magnesium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14190250 </a> )</li><li>B-27<sup>TM</sup> Supplement, minus insulin (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A1895601 </a> )</li><li>B-27<sup>TM</sup> Supplement (50x), serum free (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 17504044 </a> )</li><li>FBS (Omega Scientific, catalog number: <a class="blackNoUnder" target="_blank"> FB-02 </a> ) or equivalent, or KnockOut<sup>TM</sup> Serum Replacement (KOSR) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10828028 </a> )</li><li>MEM Non-Essential Amino Acids Solution 100x (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11140050 </a> )</li><li>Penicillin-Streptomycin (10,000 U/ml) (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15140122 </a> )</li><li>CHIR-99021 (CT99021) (CHIR99021) HCl (Selleckchem, catalog number: <a class="blackNoUnder" target="_blank"> S2924 </a> )</li><li>IWP-2 (Tocris, catalog number: <a class="blackNoUnder" target="_blank"> 3533 </a> )</li><li>Sodium L-lactate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 71718-10G </a> )</li><li>HEPES sodium salt solution 1M, BioReagent, suitable for cell culture (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> H3662-100ML </a> )</li><li>Accutase (Innovative Cell Technologies, Inc., catalog number: <a class="blackNoUnder" target="_blank"> AT 104 </a> )</li><li>Trypan Blue Solution, 0.4% (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15250061 </a> )</li><li>Dimethyl Sulfoxide (DMSO) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> D2650-100ML </a> )</li><li>Liquid nitrogen</li><li>200 proof Ethanol denatured (<em>i.e.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 71002-398 </a> ) â to prepare 70% ethanol</li><li>70% ethanol (see Recipes: Table 20)</li><li>RPMI Minus (-) medium (see Recipes: Table 6)</li><li>RPMI Plus (+) medium (see Recipes: Table 7)</li><li>RPMI Lactate medium (see Recipes: Table 8)</li><li>iPSC-CVPCs Harvest Medium (see Recipes: Table 9)</li><li>iPSC-CVPCs 2x freezing medium (see Recipes: Table 10)</li><li>10 mM CHIR-992021 solution (see Recipes: Table 11)</li><li>5 mM IWP-2 solution (see Recipes: Table 12)</li><li>1M Sodium L-lactate solution (see Recipes: Table 13)<br/><br/></li></ol></li><li>Flow cytometry<br/><ol class="npwtwo"><li>96-well round bottom assay plates (Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 25-224 </a> )</li><li>Corning<sup>TM</sup> Falcon<sup>TM</sup> Test Tube with Cell Strainer Snap Cap (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 352235 </a> )<br/></li><li>Corning<sup>TM</sup> Costar<sup>TM</sup> Sterile Disposable Reagent Reservoirs (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 4870 </a> )<br/></li><li>5 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0050-S01 </a> )<br/></li><li>10 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0100-S01 </a> )<br/></li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent P20 sterile tips with filter compatible with pipettes<br/></li><li>P200 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-01 </a> ) or other equivalent P200 sterile tips with filter compatible with pipettes<br/></li><li>P1000 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-02 </a> ) or other equivalent P1000 sterile tips with filter compatible with pipettes<br/></li><li>P20 sterile pipette tips without filter (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 83009-694 </a> ) or other equivalent P20 sterile tips without filter compatible with pipettes<br/></li><li>(Optional) P200 sterile pipette tips without filter (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 89495-378 </a> ) or other equivalent P20 sterile tips without filter compatible with pipettes<br/></li><li>1x Dulbeccoâs phosphate buffered saline (PBS) without calcium and magnesium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14190250 </a> )<br/></li><li>Methanol, ACS reagent, â¥ 99.8% (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 179337-4L-PB </a> )<br/></li><li>Bovine Serum Albumin (BSA) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A2153-100G </a> )<br/></li><li>(Optional) NaN<sub>3</sub> (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> S2002-5G </a> )<br/></li><li>37% Formaldehyde (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> F-1635-500ML </a> )<br/></li><li>Triton<sup>TM</sup> X-100 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> X-100-500ML </a> )<br/></li><li>Goat serum, New Z<br/></li><li>Mouse monoclonal anti-Troponin T, Cardiac Isoform (cTNT; TNNT2) Ab-1antibody, clone 13-11 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> MS-295-P0 </a> )<br/></li><li>Mouse IgG1 antibody (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> MG100 </a> )<br/></li><li>Goat-anti-Mouse Alexa Fluor<sup>TM</sup> 488 conjugated antibody (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> A-11001 </a> )<br/></li><li>FACS Buffer (see Recipes: Table 14)<br/></li><li>FACS-FIX Buffer (see Recipes: Table 16)</li></ol><em>Note: For antibody working concentration, see Recipes: Table 21.</em><br/><br/></li><li>Immunofluorescence<br/><ol class="npwtwo"><li>Millicell EZ SLIDE 8-well glass slides (Millipore, catalog number: <a class="blackNoUnder" target="_blank"> PEZGS0816 </a> )<br/></li><li>Coverslip glass slides (Fisherbrand, catalog number: <a class="blackNoUnder" target="_blank"> 12-545-F </a> [coverslip thickness #1]). Different coverslip thickness may be used if required<br/></li><li>5 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0050-S01 </a> )<br/></li><li>10 ml Serological pipettes (Bio Pioneer, catalog number: <a class="blackNoUnder" target="_blank"> GEX0100-S01 </a> )<br/></li><li>P20 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-00 </a> ) or other equivalent P20 sterile tips with filter compatible with pipettes<br/></li><li>P200 pipette tips sterile with filter (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 21-403-01 </a> ) or other equivalent P200 sterile tips with filter compatible with pipettes<br/></li><li>1x Dulbeccoâs phosphate buffered saline (PBS) without calcium and magnesium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 14190250 </a> )<br/></li><li>Bovine Serum Albumin (BSA) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A2153-100G </a> )<br/></li><li>Paraformaldehyde (PFA) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 158127-100G </a> )<br/></li><li>Tween<sup>Â®</sup> 20 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P9416-100ML </a> )<br/></li><li>Triton<sup>TM</sup> X-100 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> X100-500ML </a> )<br/></li><li>Gelatin from porcine skin (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> G1890-100G </a> )<br/></li><li>Mouse monoclonal anti-Î±-Actinin (Sarcomeric) antibody, clone EA-53 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> A7811 </a> )<br/></li><li>Rabbit polyclonal anti-Connexin 43 (CX43/GJA1) antibody (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 71-0700 </a> )<br/></li><li>Rabbit polyclonal anti-Myosin Light Chain 2 (MYL2; MLC2v) antibody (Proteintech, catalog number: <a class="blackNoUnder" target="_blank"> 10906-1-AP </a> )<br/></li><li>Mouse monoclonal anti-Myosin Atrial Light Chain 2 (MYL7; MLC2a) antibody clone 56F5 (Synaptic Systems, catalog number: <a class="blackNoUnder" target="_blank"> 311 011 </a> )<br/></li><li>Donkey anti-Rabbit Alexa Fluor<sup>TM</sup> 488 conjugated antibody (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> A21206 </a> )<br/></li><li>Goat anti-Mouse Alexa Fluor<sup>TM</sup> 568 conjugated antibody (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> A-11004 </a> )<br/></li><li>ProLong<sup>Â®</sup> Gold Antifade Reagent with DAPI (Cell Signaling Technologies, catalog number: <a class="blackNoUnder" target="_blank"> 8961 </a> )<br/></li><li>0.1% Gelatin solution (see Recipes: Table 2)<br/></li><li>4% PFA solution in PBS<br/></li><li>IF Perm Buffer II (see Recipes: Table 17)<br/></li><li>IF Blocking Buffer II (see Recipes: Table 18)<br/></li><li>IF Staining Buffer (see Recipes: Table 19)</li></ol><em>Note: For antibody working concentration, see Recipes: Table 21.</em><br/></li></ol><p class="pptt" id="biaoti35754">### Equipment</p><ol class="npwone"><li>iPSC Cell Culture<ol class="npwtwo"><li>Biosafety cabinet (Labconco, model: <a class="blackNoUnder" target="_blank"> Logic+ </a> )</li><li>Incubator with humidity and gas control set to maintain 37 Â°C and 95% humidity in an atmosphere of 5% CO<sub>2</sub> in air (Panasonic, model: <a class="blackNoUnder" target="_blank"> MCO-170AICUVH-PA </a> )</li><li>Water bath (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Precision </a> )</li><li>Tissue culture centrifuge with rotors for 15 ml conical tubes and 50 ml conical tubes (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Legend RT+ </a> )</li><li>Phase contrast inverted microscope (objectives: x4, x10, x20) (Olympus, model: <a class="blackNoUnder" target="_blank"> CKX41SF </a> )</li><li>(Optional) Phase contrast inverted microscope with camera (objectives: x4, x10, x20) (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> EVOS XL Core </a> )</li><li>Microscope Object marker (Nikon, model: <a class="blackNoUnder" target="_blank"> MBW10020 </a> )</li><li>Pipette aid (Labnet, catalog number: FastPette<sup>TM</sup> V2 P2000) or other equivalent available pipette aid</li><li>P20 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014392 </a> ) or other available P20 pipette</li><li>Non-frost-free freezer -20 Â°C</li><li>Refrigerator 2-8 Â°C<br/><br/></li></ol></li><li>iPSC passaging using Versene (Versene I and Versene II passage)<ol class="npwtwo"><li>Biosafety cabinet (Labconco, model: <a class="blackNoUnder" target="_blank"> Logic+ </a> )</li><li>Incubator with humidity and gas control set to maintain 37 Â°C and 95% humidity in an atmosphere of 5% CO<sub>2</sub> in air (Panasonic, model: <a class="blackNoUnder" target="_blank"> MCO-170AICUVH-PA </a> )</li><li>Tissue culture centrifuge with rotors for 15 ml conical tubes and 50 ml conical tubes (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Legend RT+ </a> )</li><li>Phase contrast inverted microscope (objectives: x4, x10, x20) (Olympus, model: <a class="blackNoUnder" target="_blank"> CKX41SF </a> )</li><li>(Optional) Phase contrast inverted microscope with camera (objectives: x4, x10, x20) (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> EVOS XL Core </a> )</li><li>Pipette aid (Labnet, catalog number: FastPette<sup>TM</sup> V2 P2000) or other equivalent available pipette aid</li><li>P20 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014392 </a> ) or other available P20 pipette</li><li>P200 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014391 </a> ) or other available P200 pipette</li><li>P1000 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014382 </a> ) or other available P1000 pipette</li><li>Automated cell counter (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> TC20 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent</li><li>Non-frost-free freezer -20 Â°C</li><li>Refrigerator 2-8 Â°C<br/><br/></li></ol></li><li>Monolayer plating<ol class="npwtwo"><li>Biosafety cabinet (Labconco, model: <a class="blackNoUnder" target="_blank"> Logic+ </a> )</li><li>Incubator with humidity and gas control set to maintain 37 Â°C and 95% humidity in an atmosphere of 5% CO<sub>2</sub> in air (Panasonic, model: <a class="blackNoUnder" target="_blank"> MCO-170AICUVH-PA </a> )</li><li>Tissue culture centrifuge with rotors for 15 ml conical tubes and 50 ml conical tubes (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Legend RT+ </a> )</li><li>Phase contrast inverted microscope (objectives: x4, x10, x20) (Olympus, model: <a class="blackNoUnder" target="_blank"> CKX41SF </a> )</li><li>(Optional) Phase contrast inverted microscope with camera (objectives: x4, x10, x20) (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> EVOS XL Core </a> )</li><li>Pipette aid (Labnet, catalog number: FastPette<sup>TM</sup> V2 P2000 or other equivalent available pipette aid</li><li>P20 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014392 </a> ) or other available P20 pipette</li><li>P200 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014391 </a> ) or other available P200 pipette</li><li>P1000 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014382 </a> ) or other available P1000 pipette</li><li>Automated cell counter (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> TC20 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent</li><li>Non-frost-free freezer -20 Â°C</li><li>Refrigerator 2-8 Â°C<br/><br/></li></ol></li><li>iPSC-CVPCs differentiation and cryopreservation<ol class="npwtwo"><li>Biosafety cabinet (Labconco, model: <a class="blackNoUnder" target="_blank"> Logic+ </a> )</li><li>Incubator with humidity and gas control set to maintain 37 Â°C and 95% humidity in an atmosphere of 5% CO<sub>2</sub> in air (Panasonic, model: <a class="blackNoUnder" target="_blank"> MCO-170AICUVH-PA </a> )</li><li>Tissue culture centrifuge with rotors for 15 ml conical tubes and 50 ml conical tubes (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Legend RT+ </a> )</li><li>Phase contrast inverted microscope (objectives: x4, x10, x20) (Olympus, model: <a class="blackNoUnder" target="_blank"> CKX41SF </a> )</li><li>(Optional) Phase contrast inverted microscope with camera (objectives: x4, x10, x20) (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> EVOS XL Core </a> )</li><li>Pipette aid (Labnet, catalog number: FastPette<sup>TM</sup> V2 P2000) or other equivalent available pipette aid</li><li>P20 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014392 </a> ) or other available P20 pipette</li><li>P200 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014391 </a> ) or other available P200 pipette</li><li>P1000 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014382 </a> ) or other available P1000 pipette</li><li>Automated cell counter (Bio-Rad, model: <a class="blackNoUnder" target="_blank"> TC20 </a> ) or a hemocytometer (Hausser Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 1483 </a> ) or equivalent.</li><li>Mr. Frosty freezing container (Corning, model: CoolCell<sup>Â®</sup> FTS30)</li><li>Refrigerator 2-8 Â°C</li><li>Non-frost-free freezer -20 Â°C</li><li>Freezer -80 Â°C</li><li>Liquid nitrogen vapor tank<br/><br/></li></ol></li><li>Estimation of optimal time for initiation of iPSC-CVPCs differentiation using ccEstimate<ol class="npwtwo"><li>Phase contrast inverted microscope with camera (objective: x4) (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> EVOS XL Core </a> ) or equivalent or automatic imaging system</li><li>Any computer with R 3.5.1 and the R package EBImage (Pau <em>et al.</em>, 2010) installed and 4GB RAM<br/><br/></li></ol></li><li>Flow cytometry<ol class="npwtwo"><li>Pipette aid (Labnet, catalog number: FastPette<sup>TM</sup> V2 P2000) or other equivalent available pipette aid</li><li>P20 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014392 </a> ) or other available P20 pipette</li><li>P200 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014391 </a> ) or other available P200 pipette</li><li>P1000 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014382 </a> ) or other available P1000 pipette</li><li>P200 Multichannel micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17013805 </a> ) or other available multichannel P200 pipette</li><li>Tissue culture centrifuge with rotors suitable to centrifuge 96-well plates (Thermo Scientific, model: <a class="blackNoUnder" target="_blank"> Legend RT+ </a> )</li><li>Refrigerator 2-8 Â°C</li><li>Non-frost-free freezer -20 Â°C</li><li>Flow cytometer (BD Biosciences, model: <a class="blackNoUnder" target="_blank"> FACSCanto II </a> ) or equivalent.<br/><br/></li></ol></li><li>Immunofluorescence<ol class="npwtwo"><li>Pipette aid (Labnet, catalog number: FastPette<sup>TM</sup> V2 P2000) or other equivalent available pipette aid</li><li>P20 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014392 </a> ) or other available P20 pipette</li><li>P200 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014391 </a> ) or other available P200 pipette</li><li>P1000 Micropipette (Rainin, catalog number: <a class="blackNoUnder" target="_blank"> 17014382 </a> ) or other available P1000 pipette</li><li>Refrigerator 2-8 Â°C</li><li>Non-frost-free freezer -20 Â°C</li><li>Confocal laser scanning fluorescence microscope (Olympus, FluoView1000)<br/></li></ol></li></ol><p class="pptt" id="biaoti35755">### Software</p><ol class="npwtwok"><li>FlowJo (Version 10) (FlowJo, LLC, <a href="https://www.flowjo.com/" target="_blank">https://www.flowjo.com</a>/)</li><li>FlowView ASW V03.01.03.03 or V4.2a (Olympus Life Science, <a href="https://www.olympus-lifescience.com/en/support/downloads/" target="_blank">https://www.olympus-lifescience.com/en/support/downloads/</a>)</li><li>ccEstimate available on request</li><li>R 3.5.1</li><li>R package EBImage (Pau <em>et al.</em>, 2010) <br/></li></ol><p class="pptt" id="biaoti35756">### Procedure</p><ol class="npwone"><li>iPSC cell culture is also described in detail in DâAntonio-Chronowska <em>et al.</em> (2019a)<ol class="npwtwo"><li>Thaw iPSC cells<br/><ol class="npwthree"><li>Prepare 12 ml of mTeSR containing 10 Î¼M ROCK Inhibitor.<br/></li><li>Transfer 9 ml of mTeSR containing 10 Î¼M ROCK Inhibitor into a sterile conical tube labeled with the name of the line.<br/></li><li>Remove vial of cryopreserved cells from liquid nitrogen tank. Keep vial on dry ice.<br/></li><li>Place and shake gently in a 37 Â°C water bath until a pea-sized ice crystal remains (around 2 min).<br/></li><li>Wipe off excess water from the vial, spray with 70% ethanol before placing in the hood.<br/></li><li>Remove thawed cells from the vial and add gently into 9 ml mTeSR containing 10 Î¼M ROCK Inhibitor in a conical tube. Wash the vial with 1-2 ml of mTeSR containing 10 Î¼M ROCK Inhibitor. Collect all cells in the same conical tube.<br/></li><li>Centrifuge cells for 5 min at 53 <em>x g</em> (500 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/></li><li>Aspirate supernatant, and gently resuspend cell pellet in 2 ml of mTeSR containing 10 Î¼M ROCK Inhibitor (1 cryovial is thawed into 1 well of 6-well plate).<br/><em>Note: The iPSC cells are cryopreserved in clumps and cannot be counted. Therefore, we recommend to cryopreserve cells from 1 well of a 6-well plate into 1 cryovial, and then to thaw cells from one cryovial onto 1 well of a 6-well plate.</em> </li><li>Label a Matrigel plate with name of line, clone and passage number. Aspirate DMEM/F-12 from Matrigel-coated plate. Add +1 to the passage number after thawing.<br/><em>Note: Do not add +1 to the passage number if the passage number was increased during cryopreservation of iPSCs.</em> </li><li>Plate cells resuspended in 2 ml into one well of a Matrigel-coated 6-well plate (final volume 2 ml/well).</li><li>24 h after plating, observe cells. Wash cells gently with DMEM/F-12 (2 ml/well) to remove cell debris and feed using fresh mTeSR medium without ROCK Inhibitor (2 ml/well). </li><li>Daily, observe the iPSCs, remove the differentiated cells, and change the medium with fresh mTeSR (2 ml/well).<br/><em>Note: It is critical to maintain iPSC culture differentiation free. For details on how to mark and remove differentiated iPSC cells, please refer below to procedure: 2. iPSC passaging using Dispase (Steps A2c-A2d) and Figure 1.</em> </li><li>Cells should reach 80-90% of confluency and be ready for passage in about 5 days.<br/></li></ol><br/>iPSCs can be passaged using multiple methods, depending on the needs. For routine iPSC culture and expansion, cells should be passaged using Dispase, which cleaves the proteins of extracellular matrix used for iPSC culture (Matrigel). Dispase will dissociate iPSCs into clumps of cells, which is a gentle method of iPSC expansion and maintenance. We recommend passaging iPSC with Dispase at the ratio 1:2 (one well split onto 2 wells) or 1:3 (one well split onto 3 wells), which allows to easily schedule iPSC cell maintenance. Cells passaged with Dispase should reach about 80% confluency in about 4-5 days. Other ratios also can be used (<em>i.e.</em>, 1:1 or 1:4), however we recommend testing it for individual lines.<br/>  When iPSC cells have to be expanded at much higher ratios or using a defined cell number, iPSC should be dissociated as single cells using Versene or Accutase. Please note that single cell passage is very stressful for iPSCs and leads to cell apoptosis. To prevent dissociation-induced apoptosis of iPSC, reagents like ROCK inhibitor, Y-27632 dihydrochloride (or others) have to be used. We do not recommend dissociating iPSCs into single cells for routine expansion or prior to cryopreservation.<br/>  Versene is a non-enzymatic, gentle method of single cell dissociation which acts by chelating metal ions (mainly magnesium and calcium) required by integrins to maintain cell-cell and cell-extracellular matrix contacts. We have routinely expanded over 150 different iPSC lines with Versene prior to plating large-scale monolayers, however we do not recommend dissociating cells using Versene in more than two consecutive passages. Accutase is an enzymatic reagent that allows for very efficient single cell dissociation. Although Accutase is a gentle reagent, commonly used for iPSC, we recommend using Accutase only for monolayer plating.<br/><br/><em>Notes:</em><br/><ol class="npwthree"><li><em>It is critical for a successful iPSC-CVPCs differentiation to use heathy and pluripotent iPSC cells and use iPSC which were culture was maintained free of differentiation.</em><br/></li><li><em>Optimal cell number will vary depending on the scale of differentiation. Differentiation at the scale described in this protocol requires that cells are expanded gradually, by performing two Versene passages (Passages: Versene I and Versene II).</em><br/><br/></li></ol></li><li>iPSC passaging using Dispase<br/><ol class="npwthree"><li>Prepare 1x (2 mg/ml) Dispase solution by adding 9 ml DMEM/F-12 to 1 ml of 10x Dispase (20 mg/ml).<br/></li><li>Allow 1x Dispase solution to come to room temperature.<br/><em>Note: 1x Dispase solution can be stored at 4 Â°C for maximum 2 weeks.</em><br/>Mark any areas of differentiation in the well(s) to be split, using the Microscope Object marker refer to Figure 1 for representative image of healthy pluripotent colonies (A) and differentiated colonies which need to be removed (B).<br/></li><li>Aspirate spent media. Aspirate marked areas of differentiation, if any, by gently tapping a Pasteur pipette within the marked circle (Figure 1C). Wash with 2 ml of DMEM/F-12 per well.<br/><br/><img alt="" class="layerphoto" height="239" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080146_0720.png" width="300"/><br/><strong>Figure 1. Pluripotent and differentiated iPSC colonies.</strong> A. Healthy and pluripotent iPSC colonies fuse into larger colonies: before fusion (left and middle) and after fusion (right). Healthy and pluripotent iPSC colonies have a smooth, flat, colony surface, are very compact with tightly growing cells that have almost invisible cell membranes (borders between cells), and uniform, well defined, colony edges. Cells have high nucleus:cytoplasm ratios with clearly visible nucleoli. Almost any deviation from this morphology may suggest loss of pluripotency and may indicate unwanted differentiation. B. Examples of differentiated iPSC cell which need to be removed from the culture. Black arrows indicate differentiated cells. Cells of various iPSC lines may have diverse morphology upon differentiation and these are only a few representative examples. C. Differentiated colony has been marked in purple dye using Microscope Object marker (left) and aspirated with a Pasteur pipette (right).<br/><br/></li><li>Add 1 ml of 1x Dispase in each well to be split. Incubate at 37 Â°C for 5 min.<br/></li><li>Check morphology of colonies after 5 min.<br/>When edges of the colonies are slightly curled up, cells are ready to be passaged. If edges of colonies are not curled up, incubate cells at 37 Â°C for another 1-2 min. Do not incubate with Dispase for longer than 8 min. Refer to Figure 2A for representative image of colonies with edges curled up.</li><li>Aspirate Dispase from all wells.<br/><em>Note: iPSC colonies passaged with Dispase will remain fully attached to the well. Only the edges of the colonies will be slightly curled up indicating that cells are ready for the next steps.</em> </li><li>Rinse the wells gently 3 times with DMEM/F12 (2 ml/well).</li><li>Add 1 ml of mTeSR media to each well to be passaged.</li><li>Use a glass serological pipette to detach colonies. Hold the pipette at a 90Â° angle to the surface of the plate. Scrape across the surface of the 6-well plate in the motion outlined in Figure 2B (start from top left side of the well and zig-zag tightly down to bottom-right side, then turn plate clockwise or counterclockwise and scrape again). Scrape until at least 90% of the colonies are detached from the well.</li><li>Wash plate with the volume of mTeSR required to bring cells up to the final volume needed to seed a new Matrigel-coated vessel. Calculate the final volume considering 2 ml per each well to be seeded with passaged cells. For example, if cells are to be passaged 1 to 3 the final volume will be 6 ml, therefore the volume of mTeSR used to wash the plate is 5 ml.</li><li>Seed cells on a new Matrigel plate plating 1 ml of cell suspension per well and then add 1 more ml of cell suspension to each well (Figure 2C). Plate cells dropwise across the entire surface of the well to ensure uniform plating.</li><li>Observe seeded cells under microscope to ensure even plating.</li><li>Place in a 37 Â°C incubator. Shake the plate in T-shape to homogenously distribute the colonies pieces in the well.</li><li>Twenty-four hours after plating gently, wash cells with DMEM/F-12 before adding fresh mTeSR medium.<br/><em>Note: For a healthy and efficient iPSC culture, it is critical to plate cells uniformly. Plate cells uniformly across the entire surface of the well and, when plating multiple wells plate cells uniformly across all wells.</em><br/><br/><img alt="" class="layerphoto" height="275" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080213_4489.png" width="300"/><br/><strong>Figure 2. Passaging iPSC with Dispase.</strong> A. Edges of the iPSCORE_2_1 iPSC colonies are curled up after 5 min incubation with Dispase. B. Pattern of movement of a glass serological pipette during the iPSCs passaging with Dispase. After scraping the well in one direction, turn the plate clockwise or counterclockwise by 90Â° and scrape remaining iPSC colonies again. About 90% cells should be detached from the well. C. Example of plating cells when passaging iPSC cells from 2 wells onto 6 wells. Add 1 ml of cell suspension into each well following a pattern: when plating cells onto three wells: 1-2-3-3-2-1 or when plating cells onto six wells: 1-2-3-4-5-6-6-5-4-3-2-1. Please refer to the section on iPSC passaging using Dispase for details. (Described also in detail in DâAntonio-Chronowska <em>et al.</em>, 2019a)<br/></li></ol><br/></li></ol></li><li>Passaging iPSC using Versene (Versene I)<br/><em>Notes:</em><br/><ol class="npwthree"><li><em>After thawing an iPSC line, passage cells with Dispase at least once before passaging cells with Versene.</em><br/></li><li><em>Optimal cell number will vary depending on the scale of iPSC-CVPC differentiation. Differentiation at the scale described in this protocol requires cells from 1-2 wells for Versene I passag</em>e.</li></ol><br/><ol class="npwtwo"><li>Remove 6-well plates from the incubator. When iPSC cells are at around 80% confluency (cells are ready for a passage), iPSC cells are ready for Versene I passage. Mark all differentiated cells, which need to be removed (Figure 1).</li><li>Aspirate the spent medium. Remove all marked differentiated cells and wash cells with DMEM/F-12 (2 ml/well).</li><li>Aspirate DMEM/F-12 and add 1 ml of room temperature Versene<sup>Â®</sup> (EDTA) 0.02% to well of a 6-well plate. Incubate cell for 5 min at 37 Â°C.</li><li>After 5 min of incubation check cells under microscope to ensure the cells are readyâindividual cell borders should be visible. (If the cells are not ready, return cells to incubator and allow cells to act for another 30 s-1 min. Do not allow Versene incubation exceed total of 8 min.</li><li>Gently aspirate the Versene from each well. DO NOT wash cells with DMEM/F12.<br/><em>Note: Individual cells incubated with Versene will be clearly visible. Cells will remain attached to the surface of the well however the plate should be handled gently.</em> </li><li>Add 1 ml per well of mTeSR containing 5 Î¼M ROCK inhibitor and re-suspend cells as single cells without scraping plate surface, by pipetting up and down using a P1000 pipette. Pipette cells 10-12 times, turn the dish by 180Â° (upside down) and pipette 5 more times. Collect cells from all wells in a 15 ml conical tube. You should not see any cell clumps.</li><li>Wash well (wells) twice with 3 ml of mTeSR containing 5 Î¼M ROCK Inhibitor. Collect all cells in the same conical tube.</li><li>Centrifuge the cells at 53 <em>x g</em> (500 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) for 5 min at room temperature. Aspirate the supernatant and resuspend cells in 3-5 ml of mTeSR containing ROCK Inhibitor.</li><li>Mix the pooled cell suspension by inverting 20 times or more if necessary. Perform the live cell count using 0.4% Trypan Blue Solution.<br/><em>Note: iPSC cell viability should not be lower than 80%.</em> </li><li>Prepare required number of cells. For a Versene I passage on three wells of a 6-well plate, prepare 7 ml of cell suspension containing 1.05 x 10<span style="font-size:13.3333px;"><sup>6</sup> </span>cells in a 15 ml conical tube. Mix cell suspension very well by inverting the tube 20 times.<br/><em>Note: Optimal cell numbers will vary depending on the scale of differentiation. iPSC-CVPCs differentiated at the scale described in this protocol require plating 9.0 x 10<sup>5</sup> live cells onto three wells of a 6-well plate during the Versene I passage (3.0 x 10<sup>5</sup> live cells per each well of a 6-well plate).</em> </li><li>Add 2 ml of cell suspension per each of the three well of a 6-well plate. Add cells dropwise using a 5 or 10 ml pipette.<br/><em>Note: It is critical to plate cells uniformly on the entire surface of the plate. To help distribute the cells uniformly plate one dish at the time and shake the newly plated dish in a cross shape (T-shape).</em> </li><li>Place plates in the incubator without stacking the plates. Incubate the cells until next morning, at 37 Â°C, 5% CO<sub>2</sub>.</li><li>Next day change medium for fresh mTeSR without ROCK inhibitor (2 ml/well). iPSC after Versene I passaged requires culturing cells for about 3-4 days until the culture reaches 80% confluency. Change medium daily with fresh mTeSR. Maintain differentiation-free iPSC culture.<br/><br/></li></ol></li><li>Passaging iPSC using Versene (Versene II)<br/><em>Note: Optimal cell number will vary depending on the scale of differentiation. iPSC-CVPCs differentiation at the scale described in this protocol requires cells from 3 wells for Versene II passage.</em><br/><ol class="npwtwo"><li>Remove 6-well plates from the incubator. When iPSC cells are at around 80% confluency (cells are ready for a passage), iPSC cells are ready for Versene II passage. Mark all differentiated cells, which need to be removed (Figure 1).</li><li>Aspirate the spent medium. Remove all marked differentiated cells and wash cells with DMEM/F-12 (2 ml/well).</li><li>Aspirate DMEM/F-12 and add 1 ml of room temperature Versene<sup>Â®</sup> (EDTA) 0.02% to well of a 6-well plate. Incubate cell for 5 min at 37 Â°C.</li><li>After 5 min of incubation check cells under microscope to ensure the cells are readyâindividual cell borders should be visible. (If the cells are not ready, return cells to incubator and allow cells to act for another 30 s-1 min. Do not allow Versene incubation exceed total of 8 min.</li><li>Gently aspirate the Versene from each well. DO NOT wash cells with DMEM/F12</li><li>Add 1 ml per well of mTeSR containing 5 Î¼M ROCK inhibitor and re-suspend cells as single cells without scraping plate surface, by pipetting up and down using a P1000 pipette. Pipette cells 10-12 times, turn the dish by 180Â° (upside down) and pipette 5 more times. Collect cells from all wells in a 15 ml conical tube. You should not see any cell clumps.</li><li>Wash all wells twice with 5 ml of mTeSR containing 5 Î¼M ROCK Inhibitor. Collect all cells in the same conical tube.</li><li>Centrifuge the cells at 53 <em>x g</em> (500 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) for 5 min at room temperature. Aspirate the supernatant and resuspend cells in 5-7 ml of mTeSR containing ROCK Inhibitor.</li><li>Mix the pooled cell suspension by inverting 20 times or more if necessary. Perform the live cell count using 0.4% Trypan Blue Solution.<br/><em>Note: iPSC cell viability should not be lower than 80%.</em> </li><li>Prepare required number of cells. For a Versene II passage on three 100 mm dishes, prepare 33 ml of cell suspension containing 6.6 x 10<sup>6</sup> cells in a 50 ml conical tube. Mix cell suspension very well by gently inverting the tube 20 times.<br/><em>Note: Optimal cell number will vary depending on the scale of differentiation. iPSC-CVPCs differentiated at the scale described in this protocol require plating 6.0 x 10<sup>6</sup> live cells onto three 100 mm dishes during the Versene II passage (6.0 x 10<sup>6</sup> live cells per each 100 mm dish).</em> </li><li>Add 10 ml of cell suspension per each of the three 100 mm dishes. Add cells dropwise using a 10 ml pipette. Before plating each of the 100 mm dishes, mix cells by inverting tubes 5-10 times.<br/><em>Note: It is critical to plate cells uniformly on the entire surface of the plate. To help distribute the cells uniformly plate one dish at the time and shake the newly plated dish in a cross shape (T-shape).</em> </li><li>Place dishes in the incubator without stacking the plates. Incubate the cells until next morning, at 37 Â°C, 5% CO<sub>2</sub>.</li><li>Next day change medium for fresh mTeSR without ROCK inhibitor (10 ml/100 mm dish). iPSC after Versene II passaged requires culturing cells for about 3-4 days until the culture reaches 80% confluency. Change medium daily with fresh mTeSR. Maintain differentiation-free iPSC culture.<br/><br/></li></ol></li><li>Monolayer plating<ol class="npwtwo"><li>The day before plating monolayer mark 10 spots (views) at the bottom of each of the three T150 flasks using a marker resistant to 70% ethanol (Figure 3A). Coat three T150 flasks with Matrigel (20ml per flask). Place flasks in the incubator.</li><li>Prepare 140-150 ml of mTeSR medium containing 5 Î¼M ROCK inhibitor.</li><li>Add 25 ml mTeSR medium containing 5 Î¼M ROCK inhibitor to each of three T150 flasks coated overnight with matrigel. Place flasks in the incubator.</li><li>Remove 6-well plates from the incubator. When iPSC cells are at around 80% confluency (cells are ready for a passage), iPSC cells are ready for Monolayer. Mark all differentiated cells, which need to be removed (Figure 1).</li><li>Aspirate the spent medium. Remove all marked differentiated cells and wash cells with DMEM/F-12 (2 ml/well).</li><li>Aspirate DMEM/F-12 and add 5 ml of room temperature Accutase to each 100 mm dish. Incubate cell for 8 min at 37 Â°C.</li><li>After 8 min of incubation, add 5 ml per well of mTeSR containing 5 Î¼M ROCK inhibitor and re-suspend cells as single cells without scraping plate surface, by pipetting up and down using a P1000 pipette. Pipette cells 10-12 times, turn the dish by 180Â° (upside down) and pipette 5 more times. Collect cells from all wells in a 50 ml conical tube. You should not see any cell clumps.<br/><em>Note: DO NOT aspirate the iPSC cells after incubation with Accutase. iPSC cells will be detached from the surface of the well.</em> </li><li>Wash all wells twice with 5 ml of mTeSR containing 5 Î¼M ROCK Inhibitor. Collect all cells in the same conical tube.</li><li>Centrifuge the cells at 53 <em>x g</em> (500 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) for 7 min at room temperature. Aspirate the supernatant and resuspend cells in 10 ml of mTeSR containing ROCK Inhibitor.</li><li>Mix the pooled cell suspension by inverting 20 times or more if necessary. Perform the live cell count using 0.4% Trypan Blue Solution.<br/><em>Note: iPSC cell viability should be not lower than 80%.</em> </li><li>Prepare required number of cells. Optimal cell number will vary depending on the scale of differentiation. iPSC-CVPCs differentiation protocol requires 3.66 x 10<sup>4</sup> live cells per cm<sup>2</sup> (5.5 x 10<sup>6</sup> per one 150 cm<sup>2</sup> flask). For three T150 flasks, prepare in a 50 ml conical tube 33 ml of cell suspension containing 1.815 x 10<sup>7</sup> cells. Mix cell suspension very well by inverting the tube 20 times.</li><li>Add 10 ml of cell suspension per T150 flask dropwise using a 10 ml pipette to each of the three flasks containing 25 ml of mTeSR with ROCK Inhibitor.<br/><em>Note: It is critical to plate cells uniformly on the entire surface of the flask. To help distribute the cells uniformly plate one flask at the time, adding cells dropwise to the entire surface of the flask. This step may require practice.</em> </li><li>Once 10ml of cells was added shake the newly plated flask in a cross shape (T-shape). Place the flask at a clean, leveled surface (table) and shake the flask again in a cross shape (T-shape). Note: Start with stronger movement and continually decrease the shaking, finishing with a very gentle movement. This step may require practice.</li><li>Place flasks in the incubator without stacking them. Incubate the cells until next morning, at 37 Â°C, 5% CO<sub>2</sub>.</li><li>Next day change medium for fresh mTeSR without ROCK inhibitor (35 ml/T150 flask). Monolayer for iPSC-CVPCs cell differentiation requires culturing cells for about 4-5 days until the monolayer reaches 80% confluency. Change medium with fresh mTeSR daily.<br/><br/></li></ol></li><li>Estimation of monolayer confluency and optimal time for initiation of iPSC-CVPCs differentiation using ccEstimate<br/>Variable growth rates across different iPSC lines results in them reaching optimal confluency at the monolayer stage at different time points (<em>i.e.</em>, faster growing lines will obtain the optimal confluency earlier) and hence impact differentiation outcome. To enable the differentiation of large number of different iPSC lines we developed cell confluency Estimates (ccEstimate), an automatic pipeline, that analyzes images of monolayer-grown cells, determines their confluency at various timepoints and predicts when the cells will reach 80% confluency. We also used ccEstimate after the initiation of differentiation, <em>i.e.</em>, after addition of CHIR99021, to measure the actual cell confluency in an unbiased, <em>i.e.</em> operator independent, way.<br/>    ccEstimates are performed by first dividing each T150 flask into 10 sections (Figure 3) and acquiring images for each section every 24 h after cells are plated as a monolayer. The final images are acquired immediately after treatment with CHIR99021, which occurs when confluency is at 80% (Day 0). The time required for cells to reach 80% confluency is estimated on the basis of the confluency curve derived for each section in each flask. To digitally measure iPSC confluency, ccEstimate performs image analysis using the EBImage package in R (Pau <em>et al.</em>, 2010). Images are read using the readImage function. As lighting may be different between the center and the border of an image, only the central part of the image is retained. To separate cells from the background and calculate confluency (<em>i.e.</em>, the fraction of the surface of the flask that is covered by cells) the following operations are performed. Confluency measurement data is collected for at least the first three days after plating as monolayer to train a generalized linear model (GLM) using the function glm in R to estimate when cells must be treated with CHIR99021. Estimation is performed separately for each flask section and CHIR is added to all three flasks associated to a given line when at least 75% of sections have 80% confluency (Figure 3).<br/>    Using ccEstimate, allows one to start differentiation at the same confluency level for each iPSC line, thereby reducing or neutralizing the effects of different growth rates. Based on our data, on average, each line required 4.23 Â± 1.12 days after plating a monolayer to reach 80% confluency. The correlation between the number of days required to reach 80% confluency and the %CM population was -0.05, suggesting that iPSC growth rate does not affect differentiation outcome (D'Antonio-Chronowska <em>et al.</em>, 2019b).<br/><br/><ol class="npwtwo"><li>Mark the spots (views) at the bottom of the vessel. In case of using T150 flask mark 10 spots as indicated in Figure 3A.<br/></li><li>Take images of the cells starting from 24 h after plating monolayer. Provide the following nomenclature to the file: <br/>UDID_NNN_SUBJECT_CLONE_PASSAGE_MONO_DAY_FLASK_VIEW_DATE_<br/>Example: UDID_001_iPSCORE-2-3_C5_P22_MONO_D1_FL1_VIEW1_20150723_<br/>Where:<br/>UDID â Unique Differentiation Identifier<br/>NNN â UDID number<br/>SUBJECT â Subject ID from whom iPSC was derived (iPSC line name)<br/>CLONE â iPSC clone number<br/>PASSAGE â iPSC passage number<br/>MONO â indicates Monolayer stage<br/>DAY â number of the day of monolayer stage<br/>FLASK â Indicates number of flask when performing differentiation in multiple flasks<br/>VIEW â location in the flask (Please refer to the Figure 3A for details)<br/>DATE â Date of the imaging in format YYYYMMDD<br/><em>Note: Make sure that the markers are not present at the image. Make sure that the files names are unique and according to the abovementioned example. This will allow for an automatic analysis of the images and prediction of the time when monolayer reaches 80% confluency.</em><br/></li><li>Every 24 h take images in the same spots<br/><em>Optional: Use automatic imaging system to take images</em> </li></ol><br/><strong>Images (JPG, PNG or TIFF format) are analyzed according to following procedure:</strong><br/><ol class="npwtwo"><li>The image is transformed to monochromatic by determining the intensity of each pixel as the average of the intensities of the red, green and blue channels.<br/></li><li>Edges are sharpened using high-pass filter. The matrix used for this filter is 15 x 15 with values -1 on the diagonals and +28 in the center.<br/></li><li>Contrasts are enhanced by multiplying the pixel intensities by 2.<br/></li><li>Mean and standard deviation of the pixel intensities are calculated. The image is transformed from monochromatic to binary by setting all pixels with intensity more than two standard deviations higher than the mean to white (intensity = 1) and all other pixels to black (intensity = 0).<br/></li><li>The resulting binary image is dilated using a disc-shaped structuring element with diameter 5 pixels.<br/></li><li>1,000 50 x 50 pixels sub-images are randomly selected. For each sub-image, the number of white pixels is calculated. Each sub-image is considered as âcellâ if at least 50% of its pixels is white, âbackgroundâ if at least 50% of its pixels is black. Confluence is calculated as the number of âcellâ sub-images divided by 1,000.<br/></li></ol>ccEstimate is available on GitHub (<a href="https://github.com/s041629/ccEstimate" target="_blank">https://github.com/s041629/ccEstimate</a>) and requires minimal knowledge of R. After installing the package and its dependencies, it is sufficient for the user to run the function <em>ccEstimate(input_file.jpg)</em>, where <em>input_file.jpg</em> is the image obtained from the monolayer iPSC culture. This function returns a numeric value, included between 0 and 1, representing the confluency. For predicting the optimal moment for initiating differentiation, the user will have to save all images associated with the same sample in a single folder and then run <em>run_confluency_estimation(input_folder)</em>. The ccEstimate package includes an example dataset, which can be investigated using the <em>run_confluency_estimation(run_example = TRUE)</em> function.<br/><br/><img alt="" class="layerphoto" height="446" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080247_1524.png" width="300"/><br/><strong>Figure 3. Schematic representation of the estimation of monolayer confluency and optimal time for initiation of iPSC-CVPCs differentiation using ccEstimate.</strong> A. Schematic representation of where to locate marks for taking images. Lines and numbers indicate 10 locations (views) where images will be taken daily. B. Schematic representation of image processing performed by ccEstimate. Adapted from D'Antonio-Chronowska <em>et al.</em> (2019b).<br/><br/><div><span style="font-size:10pt;"></span> </div></li><li>iPSC-CVPCs differentiation<br/>Refer to Figure 4 for a schematic representation of the differentiation protocol and to Figure 5, Figure 6 and Figure 7 and Video 1 for representative images of iPSC-CVPCs.<br/><br/><img alt="" class="layerphoto" height="52" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080320_1376.png" width="300"/><br/><strong>Figure 4. Schematic representation of the iPSC-CVPCs differentiation protocol.</strong> CHIR99021 (CHIR), Versene 1 (V1), Versene 2 (V2). Adapted from D'Antonio-Chronowska <em>et al.</em> (2019b).<br/><br/><em>Notes:</em><br/><ol class="npwthree"><li><em> It is critical to maintain 24h schedule of media change (media change should be performed at the exact same time, on the days indicated in Steps F1-F26), especially during first 10 days of differentiation. For example: if D0âinitiation of the differentiation (Step F1) is performed at 9:15 AM, following media changes should be done at 9:15AM according the schedule of media changes indicated in Steps F2-26.</em><br/></li><li><em> Mix well all the supplements with media by inverting 20-30 times.</em><br/></li><li><em> All media changes are performed with 35 ml/T150 flask.</em><br/></li></ol><br/><div style="white-space:nowrap;"><span style="font-size:10pt;"></span> </div><ol class="npwtwo"><li>Day 0 (D0): When iPSC monolayer reaches about 80% confluency initiate the iPSC-CVPCs differentiation by replacing mTeSR medium with RPMI Minus medium (see Recipes: Table 6) supplemented with 12 Î¼M CHIR99021 (see Recipes: Table 11).</li><li>D1: 24 h after initiation of the differentiation replace spent medium with fresh RPMI Minus medium.</li><li>D2: No media change.</li><li>D3: 72 h after initiation of the differentiation prepare a combined fresh and spent RPMI Minus medium by mixing 18 ml of fresh RPMI Minus medium and 18 ml of spent RPMI Minus medium. Mix media gently by inverting and add 7.5 Î¼M IWP-2 (see Recipes: Table 12). Mix well by inverting. Replace spent medium with combined RPMI Minus medium supplemented with 7.5 Î¼M IWP-2.<br/><em>Note: Replace spent medium with the combined medium prepared with the medium from the same flask. The effect of replacing combined medium prepared with the medium from different flask has not been extensively tested.</em> </li><li>D4: No media change.</li><li>D5: 48 h after addition of IWP-2 (120 h after initiation of the differentiation) replace spent medium with fresh RPMI Minus medium.</li><li>D6: No media change.</li><li>D7: 168 h after initiation of the differentiation replace spent RPMI Minus medium with fresh RPMI Plus medium (see Recipes: Table 7).</li><li>D8: No media change.</li><li>D9: 216 h after initiation of the differentiation replace spent medium with fresh RPMI Plus medium.</li><li>D10: No media change.</li><li>D11: Replace spent medium with fresh RPMI Plus medium.</li><li>D12: No media change.</li><li>D13: Replace spent medium with fresh RPMI Plus medium.</li><li>D14: No media change. Coat new T150 flasks with Matrigel (20 ml per flask).</li><li>D15: Passage the iPSC-CVPCs using Accutase.<br/>Note: Mechanical separation of iPSC-CVPCs at D15 is critical for efficient metabolic cell purification using Lactate.<br/><ol class="npwthree"><li>(Optional) Take images and/or video recording of the cells. Refer to Figure 5A for a representative image of iPSC-CVPCs at D15.<br/></li><li>Aspirate spent medium and wash cells with PBS (20 ml/T150 flask).<br/></li><li>Aspirate PBS and add 10 ml of room temperature Accutase to each T150 flask. Incubate cells for 10 min at 37 Â°C.<br/></li><li>After 10 min of incubation add 10 ml per flask of CVPCs Harvest medium (see Recipes: Table 9).<br/></li><li>Gently scrape cells from the surface of the flask, using a cell scraper.<br/></li><li>Collect cells in a 125 ml sterile bottle.<br/></li><li>Wash plate three additional times, each with 10 ml of CVPCs Harvest medium. Collect all cells in the same bottle.<br/><em>Note: Do not break the cells into single cells. Cells will re-attach better if they are in small clumps of few cells per clump.</em><br/></li><li>Mix all the cells by inverting the bottle and divide them into appropriate amount of 50ml conical tubes to centrifuge the cells.<br/></li><li>Centrifuge cells for 8 min at 136 <em>x g</em> (800 RPE in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/></li><li>After centrifugation, aspirate the supernatant and resuspend the cells in 20 ml of RPMI Plus medium supplemented with 5 Î¼M ROCK Inhibitor.<br/></li><li>Mix the cell suspension by inverting 10-20 times. Perform the live cell count using 0.4% Trypan Blue Solution.<br/><em>Note: Because the cells were passaged in small clumps (and not as single cells), the goal is to obtain an estimate and not a precise cell count. The estimated cell count will be used to determine the number of new T150 flasks onto which cells will be re-plated.</em> </li><li>Determine the number of flasks necessary to re-plate the cells. Calculate 1.5-2.0 x 10<sup>8</sup> cells per each fresh flask (1-1.3 x 10<sup>6</sup>/cm<sup>2</sup>).</li><li>Resuspend the cells into appropriate volume of RPMI Plus medium supplemented with 5 Î¼M ROCK Inhibitor, necess</li><li>Aspirate the DMEM medium used to coat the fresh flasks with matrigel.</li><li>Re-plate the cells onto fresh flasks coated overnight with matrigel (35 ml/T150 flask) at the density 1-1.3 x 10<sup>6</sup>/cm<sup>2</sup> (please see Step F16k). Distribute cells uniformly across the entire surface of the flask.<br/><em>Note: To help distribute the cells uniformly plate one dish at the time and shake the newly plated dish in a cross shape (T-shape).</em> </li><li>Place plates in the incubator. Incubate the cells until next day, at 37 Â°C, 5% CO<sub>2</sub> without stacking the plates.<br/></li></ol></li><li>D16: Aspirate spent medium and wash cells with PBS (20 ml/T150 flask). Replace RPMI Plus medium with RPMI Lactate medium (see Recipes: Table 8).</li><li>D17: Replace spent medium with fresh RPMI Lactate medium.</li><li>D18: No media change.</li><li>D19: Replace spent medium with fresh RPMI Lactate medium.</li><li>D20: No media change.</li><li>D21: Aspirate spent medium and wash cells with PBS (20 ml/T150 flask). Replace RPMI Lactate medium with RPMI Plus medium</li><li>D22: No media change.</li><li>D23: Replace spent medium with fresh RPMI Plus medium.</li><li>D24: No media change.</li><li>D25: Collect all cells.<br/><ol class="npwthree"><li>(Optional) Take images and/ or video recording of the cells. Refer to Figure 5B for a representative image of iPSC-CVPCs at D25.<br/></li><li>Aspirate spent medium and wash cells with PBS (20 ml/T150 flask).<br/></li><li>Aspirate PBS and add 10 ml of room temperature Accutase to each T150 flask. Incubate cells for 10 min at 37 Â°C.<br/></li><li>After 10 min of incubation add 10 ml per flask of CVPCs Harvest medium.<br/></li><li>Gently scrape cells from the surface of the flask, using a cell scraper.<br/></li><li>Collect cells in a 125 ml sterile bottle.<br/></li><li>Wash plate three additional times, each with 10 ml of CVPCs Harvest medium. Collect all cells in the same bottle.<br/></li><li>Mix all the cells by inverting the bottle and divide them into appropriate amount of 50ml conical tubes to centrifuge the cells.<br/></li><li>Centrifuge cells for 8 min at 136 <em>x g</em> (800 RPE in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/></li><li>After centrifugation, aspirate the supernatant and resuspend the cells by pipetting up and down in 10-20 ml of CVPCs Harvest medium (5 ml for each flask from which the cells were collected).<br/></li><li>Mix the cell suspension by inverting 10-20 times. Perform the live cell count using 0.4% Trypan Blue Solution.</li></ol></li><li>Estimate the number of cells to be cryopreserved or processed for molecular assays, flow cytometry and/or immunofluorescence.<br/><br/><img alt="" class="layerphoto" height="101" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080347_7445.png" width="300"/><br/><strong>Figure 5. Representative images of iPSC-CVPCs.</strong> A. D15: Cells grow in multiple layers and form a thick sheet. B. D25: Most of the cells form a single cell layer, however some cells can form aggregates and clusters.<br/><br/><!--video3755-1å¼å§--><div class="videodiv" id="vd3755-1" style="height:480px;"></div><div class="videophoto" id="vp3755-1"><a class="newapic" href="javascript:;" onclick="openvideo(1)" style="background:url('https://en-cdn.bio-protocol.org/video/3755/3755-1s.jpg') no-repeat center center #000 !important;"> <img class="videopic" src="https://en-cdn.bio-protocol.org/imagesUP/indexCover/videopalybtn.png" style="cursor:pointer !important;"/> </a> </div><div class="videostrong"><strong>Video 1. Representative video of iPSC-CVPCs beating at D15</strong> </div><br/></li></ol></li><li>Cryopreservation of iPSC-CVPCs<br/><ol class="npwtwo"><li>Prepare 2x iPSC- CVPCs freezing medium by preparing 20% DMSO solution in FBS. Prepare 0.25 ml of 2x iPSC-RPE freezing medium per each cryovial intended to be cryopreserved. Freeze cells at a final density of 1.2 x 10<sup>7</sup>/ml (depending on the downstream experiments the volume and the concentration of the cryopreserved iPSC-CVPCs cells in a single cryovial can be modified).<br/><em>Optional: If a serum free conditions are required, prepare the 2x iPSC-CVPCs freezing medium using KOSR instead of FBS.</em><br/></li><li>Prepare and print the labels for cryovials. Prepare n + 2 number of labels (n = number of cryovials to be cryopreserved). Prepare and affix the labels on all cryovials to be frozen, use one label for the Mr. Frosty and one label for record keeping (<em>i.e.</em>, lab book).<br/></li><li>Transfer desired number of cells to be cryopreserved into a new 15 ml or 50 ml conical tube.<br/></li><li>Centrifuge cells for 5-8 min at 136 <em>x g</em> (800 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature (adjust the time of centrifugation depending on the volume of cells).<br/></li><li>After centrifugation, aspirate the supernatant and resuspend the cells in 0.25 ml of FBS (or KOSR) per each cryovial to be frozen at the concentration of 2.4 x 10<sup>7</sup>/ml (<em>i.e.</em>, for 10 cryopreserved vials resuspend 6 x 10<sup>7</sup> cells in 2.5 ml of FBS or KOSR).<br/></li><li>Open all pre-labeled cryovials and add 0.25 ml of cell suspension to each cryovial.<br/></li><li>Add 0.25 ml of 2x iPSC-CVPCs freezing medium to each cryovial containing the iPSC-CVPCs cell suspension.<br/></li><li>Close all cryovials and gently invert them 5-6 times to mix cell suspension and 2x iPSC-CVPCs freezing medium (see Recipes: Table 10). Transfer cryovials to Mr. Frosty freezing container.<br/></li><li>Immediately transfer Mr. Frosty into a -80 Â°C freezer. When freezing large number of cryovials (<em>i.e.</em>, multiple Mr. Frosties) prepare individual batches, with each batch containing only the number of cryovials that will fit into one Mr. Frosty.<br/></li><li>After 24-48 h, transfer the cells into a liquid nitrogen vapor tank. Update accordingly the records (<em>i.e.</em>, box maps).<br/></li></ol><br/>Here, we provide detailed protocols for flow cytometry (FC) and immunofluorescence (IF) which can be applied to perform quantitative (FC) and qualitative (FC and IF) quality control of derived iPSC-CVPC cells.<br/><br/></li><li>Flow cytometry<ol class="npwtwo"><li>After the live cell count (Step F26k), determine how many cells should be fixed for flow cytometry analysis and transfer desired number of cells into a 15 ml conical tube. Use at least 2-5 x 10<sup>6</sup> cells.</li><li>Centrifuge cells for 5 min at 136 <em>x g</em> (800 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.</li><li>After centrifugation, aspirate the supernatant and resuspend the cells in 10 ml of PBS. <br/><em>Optional: If the volume of cell suspension used for flow cytometry is smaller than 0.5 ml then add directly to the cells 14 ml of PBS and centrifuge mix of cells and PBS for 8 min at 136 </em><em>x g</em><em> (800 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.</em> </li><li>Fix the cells using cold 90% methanol:<br/><ol class="npwthree"><li>During centrifugation prepare fresh 1% Formaldehyde in PBS. Mix well. Prepare 1ml of 1% Formaldehyde for each sample.<br/></li><li>After centrifugation decant or aspirate supernatant.<br/></li><li>Resuspend the cells using with P1000 pipette in 1ml of 1% Formaldehyde. Incubate cells for 20 min at room temperature.<br/></li><li>After the incubation add 5 ml of PBS and centrifuge samples for 8 min at 216 <em>x g</em> (1,000 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/></li><li>After centrifugation, dispose the supernatant containing formaldehyde into an appropriate waste container.<br/></li><li>Resuspend cells in 5 ml of PBS and centrifuge samples again for 8 min at 216 <em>x g</em> at room temperature.<br/></li><li>After centrifugation, aspirate the supernatant and using a P1000 pipette resuspend the cells in 1-2 ml of cold 90% Methanol.<br/></li><li>Fix cells for at least 30 min at +4 Â°C or store them in 90% Methanol for up to few weeks.</li></ol></li><li>After cells were fixed proceed to permeabilization and blocking<br/><ol class="npwthree"><li>Remove fixed cells from +4 Â°C fridge and add 5-8 ml of PBS to dilute Methanol.<br/></li><li>Centrifuge samples for 10-12min at 216 <em>x g</em> (1,000 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/></li><li>After centrifugation, dispose the supernatant containing methanol into an appropriate waste container.<br/></li><li>Add 5 ml of PBS.<br/></li><li>Centrifuge samples for 10 min at 216 <em>x g</em> (1,000 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.</li><li>After centrifugation decant or aspirate supernatant.<br/></li><li>Using a P1000 pipette resuspend the cells in 2-3 ml of Permeabilizing/Blocking buffer (see Recipes: Table 15).<br/></li><li>Incubate cells for 30 min at room temperature<br/></li><li>After incubation add 5 ml of PBS<br/></li><li>Centrifuge samples for 10 min at 216 <em>x g</em> (1,000 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/></li><li>After centrifugation decant or aspirate supernatant, add 5 ml of PBS and centrifuge the cells again for 10 min at 216 <em>x g</em> at room temperature.<br/></li><li>After centrifugation decant or aspirate supernatant.</li><li>Using a P1000 resuspend the cells in FACS buffer (see Recipes: Table 14). At the concentration of 1 x 10<sup>7</sup>/ ml.</li></ol></li><li>After cell permeabilization and blocking stain the cells<br/><ol class="npwthree"><li>Following Table 21 prepare the primary antibodies staining mix. Stain cells at the concentration of 1 x 10<sup>7</sup>/ ml (<em>i.e.</em>, 2.5 x 10<sup>5</sup> cells should be stained in 25 Î¼l of staining mix).<br/></li><li>Transfer 25 Î¼l of fixed and permeabilized cells into 2 wells of a 96-well round bottom assay plate. In order to limit usage of the antibodies and cells when staining multiple lines, mix equal number of cells from each line and transfer 25 Î¼l of the cell mix into the control well (Mouse IgG1 antibody).<br/><em>Note: Staining of 2.5 x 10</em><sup><em>5</em></sup><em> cells allows for an efficient cells and reagent usage, however it is also possible to use 5 x 10</em><sup><em>5</em></sup><em> or 1 x 10</em><sup><em>6</em></sup><em> cells per staining maintaining the same antibodies dilution ratios and scaling up the volume.</em><br/></li><li>Centrifuge plate at 863 <em>x g</em> (2,000 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) for 8-10 s counting from when the speed reaches 863 <em>x g</em> (2,000 RPM in a Sorvall 75006445 rotor with 75006441 K buckets) at room temperature.<br/>In detail:<br/><ol class="npwfour"><li>Set the centrifuge for 863 <em>x g</em> (2,000 RPM), 1 min, room temperature); start the centrifuge and wait until the speed reaches 863 <em>x g</em> (2,000 RPM).<br/></li><li>Count to 8-10 s and stop the centrifuge. The pellet after the centrifugation should be clearly visible especially when using 1 x 10<sup>6</sup> cells.</li></ol></li><li>After centrifugation, gently aspirate the supernatant, being very careful not to aspirate any cells. If using vacuum to aspirate cells, use a P20 tip without a filter (or a P200 + P20 tips without filters). Leave about 10-20 Î¼l of liquid in each well to avoid aspirating the cells.<br/></li><li>Add 25 Î¼l (or appropriate amount) of staining mix. Using a multichannel pipette set for 20 Î¼l mix cells and antibodies gently by pipetting up and down 20 times.<br/></li><li>Incubate cells with primary antibodies for 45 min in a +4 Â°C fridge.<br/></li><li>During last 5 min of incubation, prepare the secondary staining mix following Table 21. Keep the secondary staining mix on ice, protected from light until use.<br/></li><li>After 45 min, add 150 Î¼l of FACS buffer (see Recipes: Table 14).<br/></li><li>Centrifuge plate at 863 <em>x g</em> for 8-10 s as described above (Step H6c).<br/></li><li>After centrifugation, gently aspirate the supernatant, being very careful not to aspirate any cells as described above (H6d).<br/></li><li>Using a multichannel pipette add 200 Î¼l of FACS buffer and mix cells gently 5-6 times.<br/></li><li>Centrifuge plate like in Step H6c.<br/></li><li>Repeat Steps H6j-F6l to wash the cells for total of two washes.<br/></li><li>Resuspend cells in 25 Î¼l (or appropriate amount) of secondary staining mix. Using a multichannel pipette set for 20 Î¼l mix cells and antibodies gently by pipetting up and down 20 times.<br/></li><li>Incubate cells with secondary antibodies for 45 min in a +4 Â°C fridge, protected from light.<br/></li><li>After the incubation with secondary antibodies follow the Steps H6h-F6m to wash the cells for total of two washes.<br/></li><li>After the last centrifugation aspirate gently the supernatant and using a multichannel pipette resuspend the cells in 200 Î¼l of FACS-FIX buffer (see Recipes: Table 16). Gently pipette the cells 5-6 times.<br/></li><li>Using a P1000 pipette transfer each sample, one at a time, into a Corning<sup>TM</sup> Falcon<sup>TM</sup> Test Tubes with Cell Strainer Snap Cap passing the cells through the strainer in a cap.<br/><em>Note: Straining cells is critical to ensure single cell acquisition and to prevent clogging flow cytometer.</em><br/></li><li>With an additional 250 Î¼l wash each well and transfer to the appropriate tube passing the cells through the strainer in a cap. Depending on the number of cells used for staining dilute the cells to an appropriate concentration to avoid clogging the flow cytometer.</li><li>Place all the tubes in an appropriate rack and wrap them in an aluminum foil to protect from light.<br/></li></ol></li><li>Proceed with acquisition using a flow cytometer FACS Canto II (or an alternative flow cytometer).</li><li>Perform the flow cytometry analysis using FlowJo software V 10.4. Refer to the Figure 6 for an example of iPSC-CVPCs flow cytometry staining results.<br/>Figure 6. Flow-cytometry analysis of iPSC-CVPCs (iPSCORE_28_2) at Day 25 showing high staining of cardiac troponin T (cTnT, TNNT2) in red and immunoglobulin class control IgG1 in blue. Percentage of live single cardiac troponin T positive cells (cTNT+).<br/><br/><img alt="" class="layerphoto" height="288" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080424_7622.png" width="300"/><br/><strong>Figure 6. Flow-cytometry analysis of iPSC-CVPCs (iPSCORE_28_2) at Day 25 showing high staining of cardiac troponin T (cTnT, TNNT2) in red and immunoglobulin class control IgG1 in blue.</strong> Percentage of live single cardiac troponin T positive cells (cTNT+).<br/><br/></li></ol></li><li>Immunofluorescence<ol class="npwtwo"><li>Coat Millicell EZ SLIDE 8-well glass slides for at least 15 min with 0.1% Gelatin solution (see Recipes: Table 2).</li><li>Plate fresh or cryopreserved iPSC-CVPCs cells on the Matrigel coated Millicell EZ SLIDE 8-well glass slides. Plate at least 6 wells per line at the density of 1.6-2.1 x 10<sup>5</sup>/cm<sup>2</sup>.</li><li>Culture cells for at least 3-5 days until cells recover from passage or cryopreservation.</li><li>Aspirate the medium and wash cells twice with PBS. Aspirate the PBS.</li><li>Fix cells with 4% PFA for 20 min at room temperature.</li><li>Remove the PFA solution disposing it into an appropriate waste container.</li><li>Wash cells twice with PBS.</li><li>Permeabilize the cells using IF Perm Buffer II (see Recipes: Table 17). Incubate the cells for 8 min at room temperature.</li><li>Aspirate the IF Perm Buffer II.</li><li>Block the cells with IF Blocking Buffer II (see Recipes: Table 18) for 30 min at room temperature.</li><li>In the last 5 min of the blocking prepare the primary antibody solutions in IF Staining Buffer (see Recipes: Table 19 and Table 21) for the appropriate concentrations of the antibodies. Store antibodies solutions on ice until use.</li><li>After blocking aspirate all the buffer, wash cells twice with PBS and add antibodies solutions to the appropriate wells.</li><li>Incubate cells with the antibodies solution overnight at 4 Â°C.</li><li>Next day (morning) prepare the secondary antibody solutions in IF staining Buffer. Refer to the Table 21 for the appropriate concentrations of the antibodies. Keep antibodies solutions on ice until use, protected from light.</li><li>Aspirate the primary antibodies solutions and wash cells three times with PBS. After last wash aspirate all PBS.</li><li>Immediately add the secondary antibodies solutions to the appropriate wells. Incubate cells for 45-60 min at room temperature in darkness.</li><li>Aspirate the secondary antibodies solutions and wash cells three times with PBS. After last wash aspirate all PBS.</li><li>Detach the walls of the Millicell EZ SLIDE 8-well glass slides.</li><li>Add ProLong<sup>Â®</sup> Gold Antifade Reagent with DAPI following manufacturerâs recommendations and gently mount the cover glass slide avoiding bubbles. Use a pencil rubber to gently remove any bubbles. Store the slide(s) at room temperature for several hours (best until next day) in darkness to allow proper mounting.</li><li>Acquire images using an appropriate immunofluorescence microscope (best is a confocal laser scanning fluorescence microscope). Refer to Figure 7 for an example of iPSC-CVPCs immunofluorescence staining.<br/><br/><img alt="" class="layerphoto" height="157" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080445_1368.png" width="300"/><br/><strong>Figure 7. Immunofluorescence analysis of gap junction connexin 43 (CX43), anti-Myosin Light Chain 2 (MLC2v) antibody (Mouse monoclonal anti-Myosin Atrial Light Chain 7 (MLC2a), Î±-Actinin of iPSC-CVPCs at D30. </strong>Adapted from D'Antonio-Chronowska <em>et al.</em> (2019b).<br/></li></ol></li></ol><p class="pptt" id="biaoti35757">### Recipes</p><ol class="npwone"><li>Cell culture reagents and media preparation<ol class="npwtwo"><li>Matrigel solution (Table 1)<br/>â<br/><strong>Table 1. Preparation of Matrigel solution</strong><br/><img alt="" class="layerphoto" height="674" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080523_6045.png" width="800"/><br/><br/></li><li>Gelatin solution (Table 2)<br/><br/><strong>Table 2. Preparation of 0.1% Gelatin solution</strong><br/><img alt="" class="layerphoto" height="187" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080544_7453.png" width="750"/><br/><br/></li><li>1.10 mM ROCK inhibitor, Y-27632 dihydrochloride (Table 3)<br/><br/><strong>Table 3. Preparation of 1.10 mM ROCK inhibitor, Y-27632 dihydrochloride</strong><br/><img alt="" class="layerphoto" height="438" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080640_1223.png" width="750"/><br/><br/></li><li>10x Dispase Solution (Table 4)<br/><br/><strong>Table 4. Preparation of 10x Dispase Solution</strong><br/><img alt="" class="layerphoto" height="275" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080716_2298.png" width="750"/><br/><br/></li><li>mTeSR<sup>TM </sup>1 complete medium (Table 5)<br/><br/><strong>Table 5. Preparation of mTeSR<sup>TM</sup> 1 complete medium </strong><br/><img alt="" class="layerphoto" height="214" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080738_5561.png" width="750"/><br/><br/></li><li>RPMI Minus (-) medium (Table 6)<br/><br/><strong>Table 6. Preparation of RPMI Minus (-) medium</strong><br/><img alt="" class="layerphoto" height="246" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080813_9953.png" width="750"/><br/><br/></li><li>RPMI Plus (+) medium (Table 7)<br/><br/><strong>Table 7. Preparation of RPMI Plus (+) medium</strong><br/><img alt="" class="layerphoto" height="262" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080839_5590.png" width="750"/><br/><br/></li><li>RPMI Plus (+) medium (Table 8)<br/><br/><strong>Table 8. Preparation of RPMI Lactate medium (4mM Sodium L-lactate)</strong><br/><img alt="" class="layerphoto" height="304" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915080901_9023.png" width="750"/><br/><br/></li><li>RPMI Lactate medium (4 mM Sodium L-Lactate) (Table 9)<br/><br/><strong>Table 9. Preparation of iPSC-CVPCs Harvest Medium</strong><br/><img alt="" class="layerphoto" height="217" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081327_1846.png" width="750"/><br/><br/></li><li>iPSC-CVPCs 2x freezing medium (Table 10)<br/><br/><strong>Table 10. Preparation of iPSC-CVPCs 2X freezing medium</strong><br/><img alt="" class="layerphoto" height="219" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081355_0731.png" width="750"/><br/><br/></li><li>10 mM CHIR-992021 solution (Table 11)<br/><br/><strong>Table 11. Preparation of 10 mM CHIR-992021 solution</strong><br/><img alt="" class="layerphoto" height="244" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081459_4333.png" width="750"/><br/><br/></li><li>5 mM IWP-2 solution (Table 12)<br/><br/><strong>Table 12. Preparation of 5 mM IWP-2 solution</strong><br/><img alt="" class="layerphoto" height="276" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081534_1687.png" width="750"/><br/><br/></li><li>1 M Sodium L-lactate solution (Table 13)<br/><br/><strong>Table 13. Preparation of 1 M Sodium L-lactate solution</strong><br/><img alt="" class="layerphoto" height="249" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081557_2484.png" width="750"/><br/><br/></li></ol></li><li>Buffer preparation<ol class="npwtwo"><li>FACS Buffer (Table 14)<br/><br/><strong>Table 14. Preparation of FACS Buffer</strong><br/><img alt="" class="layerphoto" height="248" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081623_0592.png" width="750"/><br/><br/></li><li>Permeabilizing/Blocking Buffer (Table 15)<br/><br/><strong>Table 15. Preparation of Permeabilizing/Blocking Buffer</strong><br/><img alt="" class="layerphoto" height="277" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081655_0292.png" width="750"/><br/><br/></li><li>FACS-FIX Buffer (Table 16)<br/><br/><strong>Table 16. Preparation of FACS-FIX Buffer</strong><br/><img alt="" class="layerphoto" height="215" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081729_2322.png" width="750"/><br/><br/></li><li>IF Perm Buffer II (Table 17)<br/><br/><strong>Table 17. Preparation of IF Perm Buffer II</strong><br/><img alt="" class="layerphoto" height="238" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081756_5108.png" width="750"/><br/><br/></li><li>IF Blocking Buffer II (Table 18)<br/><br/><strong>Table 18. Preparation of IF Blocking Buffer II</strong><br/><img alt="" class="layerphoto" height="215" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081815_2365.png" width="750"/><br/><br/></li><li>IF Staining Buffer (Table 19)<br/><br/><strong>Table 19. Preparation of IF Staining Buffer</strong><br/><img alt="" class="layerphoto" height="303" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081841_9343.png" width="750"/><br/><br/></li><li>70% ethanol (Table 20)<br/><br/><strong>Table 20. Preparation of 70% ethanol</strong><br/><img alt="" class="layerphoto" height="365" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081904_0282.png" width="750"/><br/><br/></li></ol></li><li>Antibodies<br/><br/><strong>Table 21. Antibodies concentrations</strong><br/><img alt="" class="layerphoto" height="361" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915081932_9486.png" width="750"/><br/></li></ol><p class="pptt" id="biaoti35758">### Acknowledgments</p><p class="ppzz">This work was supported by a CIRM grant GC1R-06673-B, NSFCMMI division award 1728497, and NIH grants HG008118, HL107442, DK105541, and DK112155. This protocol was adapted from previous work (D'Antonio-Chronowska <em>et al.</em>, 2019b).</p><p class="pptt" id="biaoti35759">### Competing interests</p><p class="ppzz">Authors are co-founders and shareholders in ATGCures, Inc. Otherwise, authors declare no conflicts or competing interests.</p><p class="pptt" id="biaoti35760">### Ethics</p><p class="ppzz">iPSC lines were generated from 181 individuals (108 female and 73 male) recruited as part of the iPSCORE project (Panopoulos <em>et al.</em>, 2017). These individuals included seven monozygotic (MZ) twin pairs, members of 32 families (2-10 members/family) and 71 singletons (<em>i.e.</em>, not related with any other individual in this study) and were of diverse ancestries: European (118), Asian (27), Hispanic (12), African American (4), Indian (3), Middle Eastern (2) and mix ethnicity (15). The age of subject at recruitment ranged from 9 to 88 years old. Informed consent was obtained from all subjects in the study. Recruitment was approved by the Institutional Review Boards of the University of California, San Diego and The Salk Institute (Project no. 110776ZF).</p><p class="pptt" id="biaoti35761">### References</p><ol class="npwtwok"><li>Association, A. H. (2016). <a href="https://healthmetrics.heart.org/projections-of-cardiovascular-disease/" target="_blank">Projections of Cardiovascular Disease Prevalence and Costs: 2015â2035.</a> <em>Technical Report</em>.</li><li>Bargehr, J., Ong, L. P., Colzani, M., Davaapil, H., Hofsteen, P., Bhandari, S., Gambardella, L., Le Novere, N., Iyer, D., Sampaziotis, F., Weinberger, F., Bertero, A., Leonard, A., Bernard, W. G., Martinson, A., Figg, N., Regnier, M., Bennett, M. R., Murry, C. E. and Sinha, S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31375810" target="_blank">Epicardial cells derived from human embryonic stem cells augment cardiomyocyte-driven heart regeneration.</a> <em>Nat Biotechnol</em> 37(8): 895-906. </li><li>Benaglio, P., D'Antonio-Chronowska, A., Ma, W., Yang, F., Young Greenwald, W. W., Donovan, M. K. R., DeBoever, C., Li, H., Drees, F., Singhal, S., Matsui, H., van Setten, J., Sotoodehnia, N., Gaulton, K. J., Smith, E. N., D'Antonio, M., Rosenfeld, M. G. and Frazer, K. A. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31570892" target="_blank">Allele-specific NKX2-5 binding underlies multiple genetic associations with human electrocardiographic traits.</a> <em>Nat Genet</em> 51(10): 1506-1517.</li><li>Benjamin, E. J., Muntner, P., Alonso, A., Bittencourt, M. S., Callaway, C. W., Carson, A. P., Chamberlain, A. M., Chang, A. R., Cheng, S., Das, S. R., Delling, F. N., Djousse, L., Elkind, M. S. V., Ferguson, J. F., Fornage, M., Jordan, L. C., Khan, S. S., Kissela, B. M., Knutson, K. L., Kwan, T. W., Lackland, D. T., Lewis, T. T., Lichtman, J. H., Longenecker, C. T., Loop, M. S., Lutsey, P. L., Martin, S. S., Matsushita, K., Moran, A. E., Mussolino, M. E., O'Flaherty, M., Pandey, A., Perak, A. M., Rosamond, W. D., Roth, G. A., Sampson, U. K. A., Satou, G. M., Schroeder, E. B., Shah, S. H., Spartano, N. L., Stokes, A., Tirschwell, D. L., Tsao, C. W., Turakhia, M. P., VanWagner, L. B., Wilkins, J. T., Wong, S. S., Virani, S. S., American Heart Association Council on, E., Prevention Statistics, C. and Stroke Statistics, S. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/30700139" target="_blank">Heart disease and stroke statistics-2019 update: a report from the American heart association.</a> <em>Circulation</em> 139(10): e56-e528.</li><li>Beqqali, A., Kloots, J., Ward-van Oostwaard, D., Mummery, C. and Passier, R. (2006). <a href="https://www.ncbi.nlm.nih.gov/pubmed/16675594" target="_blank">Genome-wide transcriptional profiling of human embryonic stem cells differentiating to cardiomyocytes.</a> <em>Stem Cells</em> 24(8): 1956-1967.</li><li>Blinova, K., Stohlman, J., Vicente, J., Chan, D., Johannesen, L., Hortigon-Vinagre, M. P., Zamora, V., Smith, G., Crumb, W. J., Pang, L., Lyn-Cook, B., Ross, J., Brock, M., Chvatal, S., Millard, D., Galeotti, L., Stockbridge, N. and Strauss, D. G. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27701120" target="_blank">Comprehensive translational assessment of human-induced pluripotent stem cell derived cardiomyocytes for evaluating drug-induced Arrhythmias.</a> <em>Toxicol Sci</em> 155(1): 234-247.</li><li>Borow, K. M., Yaroshinsky, A., Greenberg, B. and Perin, E. C. (2019). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31318648" target="_blank">Phase 3 DREAM-HF trial of mesenchymal precursor cells in chronic heart failure.</a> <em>Circ Res</em> 125(3): 265-281</li><li> Burridge, P. W., Li, Y. F., Matsa, E., Wu, H., Ong, S. G., Sharma, A., Holmstrom, A., Chang, A. C., Coronado, M. J., Ebert, A. D., Knowles, J. W., Telli, M. L., Witteles, R. M., Blau, H. M., Bernstein, D., Altman, R. B. and Wu, J. C. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27089514" target="_blank">Human induced pluripotent stem cell-derived cardiomyocytes recapitulate the predilection of breast cancer patients to doxorubicin-induced cardiotoxicity.</a> <em>Nat Med</em> 22(5): 547-556. </li><li>Burridge, P. W., Matsa, E., Shukla, P., Lin, Z. C., Churko, J. M., Ebert, A. D., Lan, F., Diecke, S., Huber, B., Mordwinkin, N. M., Plews, J. R., Abilez, O. J., Cui, B., Gold, J. D. and Wu, J. C. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24930130" target="_blank">Chemically defined generation of human cardiomyocytes.</a> <em>Nat Methods</em> 11(8): 855-860.</li><li>Cohn, J. N., Ferrari, R. and Sharpe, N. (2000). <a href="https://www.ncbi.nlm.nih.gov/pubmed/10716457" target="_blank">Cardiac remodeling--concepts and clinical implications: a consensus paper from an international forum on cardiac remodeling. Behalf of an International Forum on Cardiac Remodeling.</a> <em>J Am Coll Cardiol</em> 35(3): 569-582. </li><li>D'Antonio-Chronowska, A., D'Antonio, M. and Frazer, K. A. (2019a). <a href="https://bio-protocol.org/e3469" target="_blank"><em>In vitro</em> differentiation of human iPSC-derived retinal pigment epithelium cells (iPSC-RPE).</a> <em>Bio-protocol</em> 24: e3469. </li><li>D'Antonio-Chronowska, A., Donovan, M. K. R., Young Greenwald, W. W., Nguyen, J. P., Fujita, K., Hashem, S., Matsui, H., Soncin, F., Parast, M., Ward, M. C., Coulet, F., Smith, E. N., Adler, E., D'Antonio, M. and Frazer, K. A. (2019b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/31668852" target="_blank">Association of human iPSC gene signatures and X chromosome dosage with two distinct cardiac differentiation trajectories.</a> <em>Stem Cell Reports</em> 13(5): 924-938. </li><li>Heron, M. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29235984" target="_blank">Deaths: Leading Causes for 2015.</a> <em>Natl Vital Stat Rep</em> 66(5): 1-76. </li><li>Laflamme, M. A., Chen, K. Y., Naumova, A. V., Muskheli, V., Fugate, J. A., Dupras, S. K., Reinecke, H., Xu, C., Hassanipour, M., Police, S., O'Sullivan, C., Collins, L., Chen, Y., Minami, E., Gill, E. A., Ueno, S., Yuan, C., Gold, J. and Murry, C. E. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17721512" target="_blank">Cardiomyocytes derived from human embryonic stem cells in pro-survival factors enhance function of infarcted rat hearts.</a> <em>Nat Biotechnol</em> 25(9): 1015-1024. </li><li>Lian, X., Hsiao, C., Wilson, G., Zhu, K., Hazeltine, L. B., Azarin, S. M., Raval, K. K., Zhang, J., Kamp, T. J. and Palecek, S. P. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22645348" target="_blank">Robust cardiomyocyte differentiation from human pluripotent stem cells via temporal modulation of canonical Wnt signaling.</a> <em>Proc Natl Acad Sci U S A</em> 109(27): E1848-1857. </li><li>Lian, X., Zhang, J., Azarin, S. M., Zhu, K., Hazeltine, L. B., Bao, X., Hsiao, C., Kamp, T. J. and Palecek, S. P. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23257984" target="_blank">Directed cardiomyocyte differentiation from human pluripotent stem cells by modulating Wnt/beta-catenin signaling under fully defined conditions.</a> <em>Nat Protoc</em> 8(1): 162-175.</li><li>Maltsev, V. A., Rohwedel, J., Hescheler, J. and Wobus, A. M. (1993). <a href="https://www.ncbi.nlm.nih.gov/pubmed/8155574" target="_blank">Embryonic stem cells differentiate <em>in</em> <em>vitro</em> into cardiomyocytes representing sinusnodal, atrial and ventricular cell types.</a> <em>Mech Dev</em> 44(1): 41-50. </li><li>Marvin, M. J., Di Rocco, G., Gardiner, A., Bush, S. M. and Lassar, A. B. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11159912" target="_blank">Inhibition of Wnt activity induces heart formation from posterior mesoderm.</a> <em>Genes Dev</em> 15(3): 316-327. </li><li>Menasche, P., Vanneaux, V., Hagege, A., Bel, A., Cholley, B., Cacciapuoti, I., Parouchev, A., Benhamouda, N., Tachdjian, G., Tosca, L., Trouvin, J. H., Fabreguettes, J. R., Bellamy, V., Guillemain, R., Suberbielle Boissel, C., Tartour, E., Desnos, M. and Larghero, J. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25990469" target="_blank">Human embryonic stem cell-derived cardiac progenitors for severe heart failure treatment: first clinical case report.</a> <em>Eur Heart J</em> 36(30): 2011-2017. </li><li>Menasche, P., Vanneaux, V., Hagege, A., Bel, A., Cholley, B., Parouchev, A., Cacciapuoti, I., Al-Daccak, R., Benhamouda, N., Blons, H., Agbulut, O., Tosca, L., Trouvin, J. H., Fabreguettes, J. R., Bellamy, V., Charron, D., Tartour, E., Tachdjian, G., Desnos, M. and Larghero, J. (2018). <a href="https://www.ncbi.nlm.nih.gov/pubmed/29389360" target="_blank">Transplantation of human embryonic stem cell-derived cardiovascular progenitors for severe ischemic left ventricular dysfunction.</a> <em>J Am Coll Cardiol</em> 71(4): 429-438.</li><li>WHO. (2018). <a href="https://www.who.int/gho/publications/world_health_statistics/2018/en/" target="_blank"><em>World health statistics 2018: monitoring health for the SDGs, sustainable development goals</em>.</a> </li><li>Paige, S. L., Osugi, T., Afanasiev, O. K., Pabon, L., Reinecke, H. and Murry, C. E. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20559569" target="_blank">Endogenous Wnt/beta-catenin signaling is required for cardiac differentiation in human embryonic stem cells.</a> <em>PLoS One</em> 5(6): e11134.</li><li>Panopoulos, A. D., D'Antonio, M., Benaglio, P., Williams, R., Hashem, S. I., Schuldt, B. M., DeBoever, C., Arias, A. D., Garcia, M., Nelson, B. C., Harismendy, O., Jakubosky, D. A., Donovan, M. K. R., Greenwald, W. W., Farnam, K., Cook, M., Borja, V., Miller, C. A., Grinstein, J. D., Drees, F., Okubo, J., Diffenderfer, K. E., Hishida, Y., Modesto, V., Dargitz, C. T., Feiring, R., Zhao, C., Aguirre, A., McGarry, T. J., Matsui, H., Li, H., Reyna, J., Rao, F., O'Connor, D. T., Yeo, G. W., Evans, S. M., Chi, N. C., Jepsen, K., Nariai, N., Muller, F. J., Goldstein, L. S. B., Izpisua Belmonte, J. C., Adler, E., Loring, J. F., Berggren, W. T., D'Antonio-Chronowska, A., Smith, E. N. and Frazer, K. A. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/28410642" target="_blank">iPSCORE: a resource of 222 iPSC lines enabling functional characterization of genetic variation across a variety of cell types.</a> <em>Stem Cell Reports</em> 8(4): 1086-1100.</li><li>Pau, G., Fuchs, F., Sklyar, O., Boutros, M. and Huber, W. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20338898" target="_blank">EBImage--an R package for image processing with applications to cellular phenotypes.</a> <em>Bioinformatics</em> 26(7): 979-981.</li><li>Reed, G. W., Rossi, J. E. and Cannon, C. P. (2017). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27502078" target="_blank">Acute myocardial infarction.</a> <em>Lancet</em> 389(10065): 197-210. </li><li>Reis Filho, J. R., Cardoso, J. N., Cardoso, C. M. and Pereira-Barretto, A. C. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26131706" target="_blank">Reverse cardiac remodeling: a marker of better prognosis in heart failure.</a> <em>Arq Bras Cardiol</em> 104(6): 502-506. </li><li>Schneider, V. A. and Mercola, M. (2001). <a href="https://www.ncbi.nlm.nih.gov/pubmed/11159911" target="_blank">Wnt antagonism initiates cardiogenesis in Xenopus laevis.</a> <em>Genes Dev</em> 15(3): 304-315. </li><li>Stacey, G. N., Crook, J. M., Hei, D. and Ludwig, T. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24094320" target="_blank">Banking human induced pluripotent stem cells: lessons learned from embryonic stem cells?</a> <em>Cell Stem Cell</em> 13(4): 385-388.</li><li>Tohyama, S., Hattori, F., Sano, M., Hishiki, T., Nagahata, Y., Matsuura, T., Hashimoto, H., Suzuki, T., Yamashita, H., Satoh, Y., Egashira, T., Seki, T., Muraoka, N., Yamakawa, H., Ohgino, Y., Tanaka, T., Yoichi, M., Yuasa, S., Murata, M., Suematsu, M. and Fukuda, K. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23168164" target="_blank">Distinct metabolic flow enables large-scale purification of mouse and human pluripotent stem cell-derived cardiomyocytes.</a> <em>Cell Stem Cell</em> 12(1): 127-137.<br/></li></ol>