<body>
            <h1>Whole-genome Identification of Transcriptional Start Sites by Differential RNA-seq in Bacteria</h1>
            <p><strong>Authors:</strong> RamÃ³n  Cervantes-Rivera, Andrea  Puhar</p>
            <h2>Abstract</h2>
            <p>Gene transcription in bacteria often starts some nucleotides upstream of the start codon. Identifying the specific Transcriptional Start Site (TSS) is essential for genetic manipulation, as in many cases upstream of the start codon there are sequence elements that are involved in gene expression regulation. Taken into account the classical gene structure, we are able to identify two kinds of transcriptional start site: primary and secondary. A primary transcriptional start site is located some nucleotides upstream of the translational start site, while a secondary transcriptional start site is located within the gene encoding sequence.<br/> <br/>Here, we present a step by step protocol for genome-wide transcriptional start sites determination by differential RNA-sequencing (dRNA-seq) using the enteric pathogen <em>Shigella flexneri</em> serotype 5a strain M90T as model. However, this method can be employed in any other bacterial species of choice. In the first steps, total RNA is purified from bacterial cultures using the hot phenol method. Ribosomal RNA (rRNA) is specifically depleted via hybridization probes using a commercial kit. A 5â²-monophosphate-dependent exonuclease (TEX)-treated RNA library enriched in primary transcripts is then prepared for comparison with a library that has not undergone TEX-treatment, followed by ligation of an RNA linker adaptor of known sequence allowing the determination of TSS with single nucleotide precision. Finally, the RNA is processed for Illumina sequencing library preparation and sequenced as purchased service. TSS are identified by in-house bioinformatic analysis.<br/> <br/>Our protocol is cost-effective as it minimizes the use of commercial kits and employs freely available software.<br/></p>
            <p><strong>Keywords:</strong> dRNA-seq, Transcriptional start site, TSS, RNA purification, Hot phenol RNA extraction, rRNA depletion, 5â²-monophosphate-dependent exonuclease (TEX), Phenol chloroform:isoamyl alcohol RNA extraction, Bacterial gene regulation, RNA precipitation, RNA phosphorylation</p>
            <h2>Background</h2><p>Transcription in bacteria is initiated by the RNA polymerase holoenzyme, which recognizes specific sequence elements on the DNA within the promotor region, to which sigma factors are bound (Feklistov <em>et al.</em>, 2014). This RNA polymerase holoenzyme binding site defines the Transcriptional Start Site and the direction of transcription. For example, the most common house-keeping sigma factor, named ð<sup>70</sup> in <em>Escherichia coli</em>, recognizes two elements centered approximately 10 and 35 bp upstream of the TSS (Feklistov <em>et al.</em>, 2014). The RNA polymerase holoenzyme melts the double stranded DNA between 11 nt upstream (position -11) to 3 nt downstream (+3) of the TSS (+1), and the single-stranded DNA can then be used as template for the addition of tri-phosphorylated ribonucleotides. The initiation starts mainly at a specific position, but sometimes âwobblesâ of one or more bases up- or downstream are encountered (Murakami and Darst, 2003; Robb <em>et al.</em>, 2013; Vvedenskaya <em>et al.</em>, 2015). The DNA sequence around TSS have long been recognized as crucial for gene regulation in bacteria (Jacob and Monod, 1961). Depending on the position within the gene structure, which begins with a start codon (usually ATG) and finishes with one of the three stop codons, we can identify two types of transcriptional start sites: primary and secondary. Primary transcriptional start sites (pTSS) are located some nucleotides upstream of the translational start site, while the secondary transcriptional start sites (sTSS) are located within the gene encoding sequence (Figure 1).<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915202555_5518.png" width="300"/><br/><strong>Figure 1. Schematic representation of the Primary and Secondary Transcriptional Start Site definition</strong><br/><br/>Until the advent of next-generation sequencing, in order to locate the TSS of a specific RNA, it was necessary to examine each transcript individually, using either the S1 protection assay, primer extension or a 5â RACE method (Sharma and Vogel, 2014). Owing to the increasing popularity and a decrease in costs of high-through put sequencing, in 2010 differential RNA-seq (dRNA-seq) was developed to simultaneously map all TSS of a genome using <em>Helicobacter pylori</em> as first model organism (Sharma <em>et al.</em>, 2010). Since then, this method has been widely employed to determine the TSS of several bacterial species (Berghoff <em>et al.</em>, 2009; Jager <em>et al.</em>, 2009; Albrecht <em>et al.</em>, 2010 and 2011; Bohn <em>et al.</em>, 2010; Irnov <em>et al.</em>, 2010; Schluter <em>et al.</em>, 2010; Sharma <em>et al.</em>, 2010; Beckmann <em>et al.</em>, 2011; Deltcheva <em>et al.</em>, 2011; Filiatrault <em>et al.</em>, 2011; Mitschke <em>et al.</em>, 2011a and 2011b; Kroger <em>et al.</em>, 2012 and 2013; Madhugiri <em>et al.</em>, 2012; Ramachandran <em>et al.</em>, 2012 and 2014; Sahr <em>et al.</em>, 2012; Schmidtke <em>et al.</em>, 2012; Wilms <em>et al.</em>, 2012; Cortes <em>et al.</em>, 2013; Dugar <em>et al.</em>, 2013; Mentz <em>et al.</em>, 2013; Nickel <em>et al.</em>, 2013; Pfeifer-Sancar <em>et al.</em>, 2013; Porcelli <em>et al.</em>, 2013; Schluter <em>et al.</em>, 2013; Voss <em>et al.</em>, 2013; Wiegand <em>et al.</em>, 2013; Zhang <em>et al.</em>, 2013; Voigt <em>et al.</em>, 2014; Cervantes-Rivera <em>et al.</em>, 2020).<br/> <br/>Primary transcripts of prokaryotes carry a triphosphate at their 5â-ends. In contrast, processed or degraded RNAs only carry a monophosphate at their 5â-ends. This is also the case of ribosomal RNA (rRNA) (Schoenberg, 2007). The dRNA-seq approach used here exploits the properties of a 5â-monophosphate-dependent exonuclease (TEX) to selectively degrade processed transcripts, thereby enriching for unprocessed RNA species carrying a native 5â-triphosphate (Schoenberg, 2007). TSS can then be identified by comparing TEX-treated and untreated RNA-seq libraries, where TSS appear as localized maxima in coverage enriched upon TEX-treatment (Sharma <em>et al.</em>, 2010).<br/> <br/>Until 2013 TSS annotation was performed manually, but this method is arduous and time-consuming. Nowadays many computational tools are available for automatic TSS annotation using dRNA-seq data. These include TSSPredator (Dugar <em>et al.</em>, 2013), TSSAR (Amman <em>et al.</em>, 2014), TruHMM (Li <em>et al.</em>, 2013), TSSer (Jorjani and Zavolan, 2014) and ReadXplorer2 (Hilker <em>et al.</em>, 2016).<br/> <br/>Here, we present a step by step protocol for TSS determination through comparison of TEX-treated and untreated RNA libraries in <em>Shigella flexneri</em> serotype 5a strain M90T as originally performed in (Cervantes-Rivera <em>et al.</em>, 2020). The overall workflow is illustrated in Figure 2.<br/><br/> <img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200915/20200915202629_2578.png" width="300"/><br/><strong>Figure 2. Workflow of dRNA-seq for whole-genome Transcriptional Start Sites identification</strong></p>
        </body>

<p class="pptt" id="biaoti35862">## Materials and Reagents</p><ol class="npwtwok"><li>Sterile syringe filter with pore size 0.22 Î¼m (Merck, catalog number: <a class="blackNoUnder" target="_blank"> SLGV033RK </a> )</li><li>Nitrile gloves</li><li>Culture tube of 13 ml (TPP, catalog number: <a class="blackNoUnder" target="_blank"> 91016 </a> )</li><li>Microfuge tubes 1.5 ml (Eppendorf, Safe-Lock tubes, catalog number: 00<a class="blackNoUnder" target="_blank"> 30120086 </a> )<br/><em>Note: In principle all prepackaged tubes on the market are RNase free and can be used.</em> </li><li>Centrifuge tubes of 50 ml (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 62.547.254 </a> )</li><li>Pipette tips (VWR, catalog numbers: <a class="blackNoUnder" target="_blank"> 89041-404 </a> , <a class="blackNoUnder" target="_blank"> 89041-412 </a> , <a class="blackNoUnder" target="_blank"> 89041-400 </a> )<br/><em>Note: In principle all prepackaged tips on the market are RNase free and can be used.</em> </li><li>Semi-micro cuvette (Sarstedt, catalog number: <a class="blackNoUnder" target="_blank"> 67.742 </a> )</li><li><em>Shigella flexneri</em> 5a M90T (Sansonetti <em>et al.</em>, 1982), can also be purchased from ATCC and available on request from the authors</li><li>Tryptone soy broth (TSB) ready to use powder (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 105459 </a> )</li><li>Tryptone soy agar (TSA) ready to use powder (Merck, catalog number: <a class="blackNoUnder" target="_blank"> 105458 </a> )</li><li>Congo red (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> C6277 </a> )</li><li>Diethyl pyrocarbonate (DEPC) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 40718 </a> )</li><li>Sodium dodecyl sulfate (SDS) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> L3771 </a> )</li><li>Bromophenol blue sodium salt (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> B8026 </a> )</li><li>Xylene cyanol FF (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> X4126 </a> )</li><li>Glycerol (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> G5516 </a> )</li><li>Formamide (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 47671 </a> )</li><li>Ethanol absolute (VWR Chemicals, catalog number: <a class="blackNoUnder" target="_blank"> 20821.558 </a> )</li><li>Sodium acetate 3 M, pH 5.5 (Ambion, catalog number: <a class="blackNoUnder" target="_blank"> AM9740 </a> ).</li><li>EDTA, disodium salt, dihydrate (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 324503 </a> )</li><li>Sodium hydroxide (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> S8045 </a> )</li><li>Trizma base (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> T1503 </a> )</li><li>Acetic acid (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> 695092 </a> )</li><li>Agarose (VWR Life Science, catalog number: <a class="blackNoUnder" target="_blank"> 35-1020 </a> )</li><li>DNase I 10 U/ml (Roche, catalog number: 0<a class="blackNoUnder" target="_blank"> 4716728001 </a> )</li><li>RNaseOUT 40 U/ml (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> 10777019 </a> )</li><li>Chloroform (Sigma, Catalog number: <a class="blackNoUnder" target="_blank"> C2432 </a> )</li><li>Phenol solution pH 4.3 Â± 0.2 (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> P4682-400ML </a> )</li><li>Isoamyl alcohol (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> W205702 </a> )</li><li>Sodium acetate pH 5.5 (Ambion, catalog number: <a class="blackNoUnder" target="_blank"> AM9740 </a> )</li><li>Glycogen 20 mg/ml (Invitrogen, catalog number: <a class="blackNoUnder" target="_blank"> R0551 </a> )</li><li>Oligos SF-Hfq-F 5â²-ACGATGAAATGGTTTATCGAG-3â² and SF-Hfq-R 5â²-ACTGCTTTACCTTCACCTACA-3â² (Sigma, custom order)</li><li>GeneRuler 1 kb DNA ladder (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> SM0211 </a> )</li><li>Linker-RNA-adaptor 5â²-GACCUUGGCUGUCACUCA-3â² (Sigma, custom order)</li><li>Ribo-Zero rRNA Removal Kit for bacteria (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> MRZB12424 </a> )<br/><em>Note: This product is not available anymore on the market, but can be replaced with Pan-Prokaryote riboPOOL kit (siTOOLS BIOTECH, Pan-Prokaryote v2), RiboCop rRNA Depletion Kit for Bacteria (Lexogen, catalog number: <a class="blackNoUnder" target="_blank"> 125-127 </a> ) or any other rRNA depletion method based on the use of hybridization probes. Replacement products relying on enzymatic-based rRNA depletion such as the Ribo-Zero Plus rRNA Depletion Kit (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> 20037125 </a> ) are not suitable.</em> </li><li>Terminator-5â²-Phosphate-Dependent Exonuclease (TEX) (Lucigene, catalog number: <a class="blackNoUnder" target="_blank"> TER51020 </a> )</li><li>5â-Pyrophosphohydrolase 5,000 units/ml (RppH) (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0356S </a> )</li><li>T4 RNA ligase 10,000 U/ml (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0204L </a> )</li><li>T4 polynucleotide kinase (PNK) 10,000 U/ml (New England BioLabs, catalog number: <a class="blackNoUnder" target="_blank"> M0201L </a> )</li><li>RNaseZap (Ambion, catalog number: <a class="blackNoUnder" target="_blank"> AM9780 </a> )</li><li>Dream Taq DNA polymerase (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> EP0701 </a> )</li><li>dNTPs mix, 2 mM each (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0241 </a> )</li><li>GelRed nucleic acid gel stain (Biotium, catalog number: <a class="blackNoUnder" target="_blank"> 41003 </a> )</li><li>ATP solution (100 mM) (Thermo Scientific, catalog number: <a class="blackNoUnder" target="_blank"> R0441 </a> )</li><li>Tryptone soy broth (TSB) medium (see Recipes)</li><li>Tryptone soy agar (TSA) plates (see Recipes)</li><li>Congo red solution (see Recipes)</li><li>Nuclease-free water (see Recipes)</li><li>Sodium dodecyl sulfate (SDS) 10% solution (see Recipes)</li><li>EDTA 0.5 M, pH 8 (see Recipes)</li><li>6x DNA Loading buffer (see Recipes)</li><li>2x RNA Loading buffer (see Recipes)</li><li>Stop Solution (see Recipes)</li><li>Ethanol 80% (see Recipes)</li><li>Lysis solution (see Recipes)</li><li>10x TAE (see Recipes)</li><li>1% Agarose (see Recipes)</li><li>Phe-CHISAM solution (see Recipes)</li><li>CHISAM (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35863">### Equipment</p><ol class="npwtwok"><li>Microcentrifuge, refrigerated (VWR, model: <a class="blackNoUnder" target="_blank"> Micro Star 17R </a> )</li><li>Spectrophotometer (Amersham, model: <a class="blackNoUnder" target="_blank"> Ultrospec 2100 pro </a> )</li><li>Pipettes (Eppendorf Research plus)</li><li>Microwave oven (Whirlpool, model: <a class="blackNoUnder" target="_blank"> MD101 </a> )</li><li>Centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> 5810R </a> )</li><li>NanoDrop ND-1000 spectrophotometer (Saveen &amp; Werner AB)</li><li>Orbital Shaker (Edmund BÃ¼hler, model: <a class="blackNoUnder" target="_blank"> Swip SM25 </a> )</li><li>Block heater (VWR, model: <a class="blackNoUnder" target="_blank"> 460-0353 </a> )</li><li>DynaMag-2 Magnet (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12321D </a> )</li><li>T100 Thermo cycler (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 186-1096 </a> )</li><li>Chemical hood</li><li>Horizontal gel electrophoresis system (VWR, model: <a class="blackNoUnder" target="_blank"> 700-0015 </a> )</li><li>Electrophoresis power supply (VWR, model: <a class="blackNoUnder" target="_blank"> E0202 </a> )<br/></li></ol><p class="pptt" id="biaoti35864">### Software</p><p class="ppzz"><strong>Programs</strong><br/>All programs used in this protocol are freely available</p><ol class="npwtwok"><li>bcl2fastq (<a href="https://emea.support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html" target="_blank">https://emea.support.illumina.com/downloads/bcl2fastq-conversion-software-v2-20.html</a>)</li><li>FastQC/0.11.8 (<a href="https://www.bioinformatics.babraham.ac.uk/projects/download.html%23fastqc" target="_blank">https://www.bioinformatics.babraham.ac.uk/projects/download.html#fastqc</a>) (Wingett and Andrews, 2018)</li><li>MultiQC/1.8 (<a href="https://multiqc.info/" target="_blank">https://multiqc.info/</a>) (Ewels <em>et al.</em>, 2016)</li><li>trimmomatic/0.36 (<a href="http://www.usadellab.org/cms/?page=trimmomatic" target="_blank">http://www.usadellab.org/cms/?page=trimmomatic</a>) (Bolger <em>et al.</em>, 2014)</li><li>samtools/1.9 (<a href="http://www.htslib.org/doc/samtools.html" target="_blank">http://www.htslib.org/doc/samtools.html</a>) (Li <em>et al.</em>, 2009)</li><li>bowtie2/2.3.5.1 (<a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a>) (Langmead and Salzberg, 2012)</li><li>IGV/2.3.82 (<a href="https://software.broadinstitute.org/software/igv/" target="_blank">https://software.broadinstitute.org/software/igv/</a>) (Thorvaldsdottir <em>et al.</em>, 2013)</li><li>ReadXplorer/2.2.3 (<a href="https://www.uni-giessen.de/fbz/fb08/Inst/bioinformatik/software/ReadXplorer" target="_blank">https://www.uni-giessen.de/fbz/fb08/Inst/bioinformatik/software/ReadXplorer</a>) (Hilker <em>et al.</em>, 2014; Hilker <em>et al.</em>, 2016)<br/><br/></li></ol><p class="ppzzs"><strong>Databases</strong> </p><ol class="npwtwok"><li>GenBank (<a href="https://www.ncbi.nlm.nih.gov/genome/182?genome_assembly_id=493862" target="_blank">https://www.ncbi.nlm.nih.gov/genome/182?genome_assembly_id=493862</a>)</li><li>RefSeq (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP037923.1" target="_blank">https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP037923.1</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP037924.1" target="_blank">https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP037924.1</a>)</li><li>RegulonDB (<a href="http://regulondb.ccg.unam.mx/" target="_blank">http://regulondb.ccg.unam.mx/</a>)</li><li>dRNA-seq raw data are available on the SRA database under the accession numbers: SRR8921222(dRNA-Seq_TEX_Positive), SRR8921223 (dRNA-Seq_TEX_Negative) (<a href="https://dataview.ncbi.nlm.nih.gov/" target="_blank">https://dataview.ncbi.nlm.nih.gov/</a>)<br/></li></ol><p class="pptt" id="biaoti35865">### Procedure</p><p class="ppzz">Before starting the RNA purification be sure that you have cleaned pipets, centrifuges, bench, the chemical hood, equipment for gels and the marker pen with RNaseZap to ensure that everything is RNase free. At every step of the protocol you should wear nitrile/latex gloves to avoid RNase contamination.</p><br/><ol class="npwone"><li>Total RNA purification using the hot phenol method and RNA quality control by electrophoresis<ol class="npwtwo"><li>Prepare a pre-culture by inoculating 5 ml of TSB medium with a single colony of <em>S. flexneri</em> 5a M90T from a Congo red tryptone soy agar plate and grow overnight (~16 h) at 37 Â°C with agitation (150 RPM).<br/><em>Note: Ensure that the colonies used are secretion-competent (forming red colonies on the plate due to Congo red absorption) (Sharma and Puhar, 2019).</em> </li><li>Subculture by diluting 100 Î¼l of the pre-culture in 10 ml of TSB (1:100 dilution).</li><li>Grow to OD<sub>600</sub> = 0.3 with shaking. This will take approximately 2 h if the medium is pre-warmed.</li><li>Mix the culture with 2 ml of cold stop solution.</li><li>Incubate the mixed culture with the stop solution on ice for 30 min. <br/><em>Note: To stabilize the RNA and prevent degradation, incubate it for at least 30 min, but not longer than 2 h on ice.</em> </li><li>Transfer the culture into a 15 ml tube.</li><li>Centrifuge for 5 min at 16,200 <em>x g</em> at 4 Â°C.</li><li>Remove the supernatant by decanting. Keep ~2 ml of supernatant to resuspend the pellet in it.</li><li>Transfer the resuspended cells into two 1.5 ml tubes.</li><li>Centrifuge for 5 min at 16,200 <em>x g</em> at 4 Â°C.</li><li>Discard the supernatant by decanting.<br/><em>Note: At this point the samples could be frozen with liquid nitrogen and stored at -80 Â°C. Samples are stable for months. </em> </li><li>Purify the total RNA using the hot phenol method as follows.</li><li>Resuspend cell pellets in 500 Î¼l of lysis solution at room temperature.</li><li>Under the chemical hood add 500 Î¼l of phenol pH 4 pre-warmed to 65 Â°C. Mix by inversion at least 20 times.<br/><em>Note: Pre-warm in a heat block the phenol before starting the RNA purification.</em> </li><li>Vortex vigorously for 20 s.</li><li>Incubate 5 min at 65 Â°C in the heat block.</li><li>Centrifuge 3 min at 13,800 <em>x g</em> at room temperature.<br/><em>Note: If the centrifugation step is done at 4 Â°C the SDS is going to precipitate and the solution turns white, which does not affect the efficacy of the process but makes it difficult to differentiate the two phases. After this centrifugation time you should see two phases, the upper one should be whitish (when the SDS is cooling down it becomes insoluble). RNA is in the aqueous phase (upper phase), while in the clearly white disk interphase there are protein layers and DNA, whereas lipids and some other cellular components are dissolved in the lower transparent phase that is the organic phase.</em> </li><li>Transfer the upper phase (~500 Î¼l) to a new RNase-free tube under the chemical hood.</li><li>Add 1 ml of cold (4 Â°C) absolute ethanol.</li><li>Mix thoroughly by vortexing for 5 s until you see a homogeneous mixture.</li><li>Centrifuge at 16,200 <em>x g</em> for 5 min at 4 Â°C.</li><li>Discard supernatant that contains the ethanol solution by decanting.<br/><em>Note: At this step you can see a white/transparent pellet on the bottom.</em> </li><li>Add 1 ml of 80% ethanol at room temperature. Shake the tube vigorously for 10 s.<br/><em>Note: Do not use the vortex, it can detach the pellet from the bottom.</em> </li><li>Centrifuge at 16,200 <em>x g</em> for 2 min at 4 Â°C.</li><li>Discard the supernatant by decanting.</li><li>Centrifuge (short spin) the tubes and discard the remaining liquid by pipetting.</li><li>Dry samples at room temperature by leaving the tubes uncapped on the bench.<br/><em>Note: Once that the sample is completely dry it turns transparent. It can take 10 min to obtain a completely dry sample.</em> </li><li>Resuspend the RNA sample in 100 Î¼l of nuclease free water. </li><li>Check the quality of the purified RNA in a 1% agarose gel in TAE 1x as follows.</li><li>Prepare the sample to be analyzed in a 1.5 ml tube. Add 1 Î¼l of sample plus 5 Î¼l of RNA loading buffer 2x and 4 Î¼l of nuclease free water. </li><li>Heat the sample for 5 min at 65 Â°C for secondary and tertiary structures denaturation.</li><li>Cool down the tube in ice for 5 min.</li><li>Spin down for 2 s and load the gel.</li><li>Run the gel in TAE 1x buffer for 40 min at 100 V.</li><li>Stain the gel with 3x-concentrated Gel Red for 10 min.</li><li>Wash the gel with 20 ml of nuclease free water for 10 min shaking at room temperature.</li><li>Visualize the gel under UV light (Figure 3A). If the total RNA purification was successful, the 23S and 16S are visible as clear, heavy bands, while in the lower part of the gel weaker tRNA bands are observed. It is not possible to see mRNAs. In case of RNA degradation, bands appear as smear.</li><li>Quantify the RNA in a NanoDrop spectrophotometer (Figure 3B).<br/><em>Note: At least 20 Î¼g (10 Î¼g for each of two conditions) of total RNA are necessary to continue with the protocol. Typical concentrations should be around 500-900 ng/ml. Very successful purifications yield up to 1,500 ng/ml RNA.</em><br/><br/><img alt="" class="layerphoto" height="293" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916003924_9309.png" width="300"/><br/><strong>Figure 3. Total RNA quality control.</strong> A. Separation of total RNA from three replicates in a 1% agarose gel in TAE of <em>Shigella flexneri</em> 5a M90T as total RNA quality control. B. Examples of typical values obtained from total RNA quantification in a NanoDrop. The 260/280 ratio should be around 1.88-2.00.<br/><br/></li><li>Freeze the RNA samples at -20 Â°C. The RNA samples are stable for months.<br/><br/></li></ol></li><li>DNase treatment to remove contaminating genomic DNA from total RNA<br/>Protocol volumes are for one reaction, but every step is scalable to two or more reactions.<br/><ol class="npwtwo"><li>Dilute 10 Î¼g of total RNA to a final volume of 30 Î¼l using nuclease free water.</li><li>Denature RNA in nuclease free water for 5 min at 65 Â°C in the heat block.</li><li>Cool down in ice for 5 min.</li><li>Prepare a reaction mix containing the components listed in Table 1. The indicated quantities are sufficient for one sample containing 10 Î¼g of RNA.<br/><br/><strong>Table 1. DNase reaction mix for one reaction</strong><br/><em>Note: 1 Unit of DNase I for 1 Î¼g of total RNA.</em><br/><img alt="" class="layerphoto" height="211" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004026_0698.png" width="480"/><br/><br/></li><li>Add the reaction mix to the RNA sample tube on ice.</li><li>Incubate the reaction mix for 60 min at 37 Â°C in the heat block.<br/><em>Note: Be sure that the heat block is at the correct temperature, DNase works efficiently at 37 Â°C.</em> </li><li>Add to the reaction tube 60 Î¼l of nuclease free water to obtain 100 Î¼l final volume.<br/><br/></li></ol></li><li>RNA clean-up by chloroform:isoamyl alcohol extraction and ethanol precipitation<ol class="npwtwo"><li>Add 100 Î¼l of CHISAM to the reaction tube under the chemical hood.</li><li>Mix with the vortex vigorously for 15 s.</li><li>Centrifuge the tube for 2 min at 13,800 <em>x g</em>, 4 Â°C.</li><li>Under the chemical hood transfer ~100 Î¼l from the aqueous phase (upper) that contains the RNA to a new 1.5 ml tube. Discard the organic phase (lower), proteins and salts from the buffer are dissolved in this phase.</li><li>Repeat Steps C1-C4.</li><li>Add 300 Î¼l of cold absolute ethanol (3 sample volumes), 10 Î¼l of sodium acetate 3 M pH 5.5 (1/10 of sample volume) and 1 Î¼l of glycogen (20 mg/ml).</li><li>Vortex the mix vigorously for 5 s.</li><li>Incubate the tube at -20 Â°C for 30 min.<br/><em>Note: This step is optional, but it is good to do it to decrease the loss of RNA in the next steps.</em> </li><li>Centrifuge at 16,200 <em>x g</em> (or maximum speed) for 30 min at 4 Â°C.<br/><em>Note: Centrifugation step can be for longer time. The minimum is 30 min, but it could be for 2 to 4 h.</em> </li><li>Discard the supernatant carefully by decanting.<br/><em>Note: On the bottom of the tube there should be a white pellet. Discard the supernatant without detaching the pellet. </em> </li><li>Add 1 ml of 80% ethanol at room temperature to wash the RNA.</li><li>Mix the tube by inverting 20 times.</li><li>Centrifuge the tube for 5 min at 13,800 <em>x g</em>, 4 Â°C.</li><li>Discard the supernatant by decanting.</li><li>Centrifuge (short spin) and discard the remaining liquid with a 100 Î¼l pipette.</li><li>Keep the tubes open on ice to dry the samples.<br/><em>Note: The time required to dry the samples depends on how well the liquid phase was removed (usually it takes 10-15 min). The pellet is completely transparent once it is completely dry.</em> </li><li>Resuspend the total RNA in 23 Î¼l of nuclease free water.</li><li>Check the integrity of the DNase-free RNA by visual inspection of the 23S and 16S band on 1% agarose gel as performed after the total RNA extraction.</li><li>Prepare the running sample as in Steps A30-A37.</li><li>Visualize the gel under UV light (Figure 4A).</li><li>Quantify the RNA in the NanoDrop spectrophotometer (the final RNA concentration should be ~600-1,000 ng/ml) (Figure 4B). It is normal to detect lower concentrations after the process, as the presence of DNA increase the absorbance of the sample.<br/><br/><img alt="" class="layerphoto" height="344" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004134_6035.png" width="300"/><br/><strong>Figure 4. Total RNA quality control after DNase treatment.</strong> A. Separation of total RNA from three replicates in a 1% agarose gel in TAE of <em>Shigella flexneri</em> 5a M90T as total RNA quality control. B. Examples of typical values obtained from total RNA quantification after DNase treatment in a NanoDrop.<br/><br/></li><li>Freeze the RNA samples at -20 Â°C. The RNA samples are stables for months.<br/><br/></li></ol></li><li>PCR to verify the removal of genomic DNA<br/>To be sure that there is no trace of DNA in the RNA samples it is extremely important to check the absence of genomic DNA by PCR.<br/><ol class="npwtwo"><li>Use 1 Î¼l (~1 Î¼g) of the DNase treated RNA sample as a template to check for the absence of genomic DNA.</li><li>As a positive control use ~10 ng of genomic DNA from <em>S. flexneri</em> 5a M90T. Genomic DNA can be extracted using the phenol-chloroform protocol as described in (He, 2011).</li><li>As a negative control use a reaction without any template.</li><li>Prepare the PCR mix (Table 2). The indicated quantities are sufficient for one sample.<br/><br/><strong>Table 2. Reaction mix for the PCR control</strong><br/><img alt="" class="layerphoto" height="288" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004230_5968.png" width="500"/><br/><em>Note: The oligos used here amplify a ~909 bp long product of the hfq gene (303 nt) from S. flexneri 5a M90T including 300 bp upstream and 300 bp downstream of the hfq gene.</em><br/><br/></li><li>PCR reaction settings (Table 3).<br/><br/><strong>Table 3. Settings for control PCR</strong><br/><img alt="" class="layerphoto" height="185" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004350_0719.png" width="500"/><br/><br/></li><li>Check the product in a 1% agarose gel as follows.</li><li>Prepare the sample to be analyzed. Add 6 Î¼l of sample plus 2 Î¼l of DNA loading buffer 6x.</li><li>Run the 1% agarose gel in TAE 1x buffer for 40 min at 100 V.</li><li>Stain the gel with Gel Red 3x for 10 min.</li><li>Wash the gel with nuclease free water.</li><li>Visualize the gel under UV light (Figure 5).<br/><br/><img alt="" class="layerphoto" height="155" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004439_3698.png" width="300"/><br/><strong>Figure 5. Control PCR to verify the absence of genomic DNA with the </strong><em><strong>hfq</strong></em><strong> gene plus 300 pb upstream and downstream of </strong><em><strong>Shigella flexneri</strong></em><strong> 5a M90T as target using genomic DNA as a positive control. </strong>Agarose gel 1% in TAE 1x. The absence of a PCR product seen as a 909 bp band in lanes 2-4 indicates that the purified RNA is devoid of DNA contaminations.<br/><br/></li></ol></li><li>rRNA depletion<br/>Ribosomal RNA is the most abundant RNA species. If rRNA is not removed, most reads from sequencing will originate from rRNA. As a consequence, samples which are compared need to all either have or be devoid of rRNA, or else the sequencing depth will be very different and statistical analysis cannot be performed. The TEX treatment described in the next section to enrich for primary RNA removes all RNA that is not tri-phosphorylated including rRNA, making this library rRNA-depleted. Therefore, to allow the comparison of TEX-treated with mock-treated samples, rRNA needs to be specifically depleted first from total RNA before splitting the RNA into two for further treatment.<br/>    rRNA depletion is carried out using a commercial kit, which specifically depletes bacterial rRNA, according to manufacturerâs instructions. Specific depletion of bacterial rRNA generally relies on hybridization with tagged pan-bacterial probes that recognize conserved sequences in rRNA, followed by removal of captured rRNAs, for example by precipitation with magnetic beads that bind the tag of the probes.<br/>We have used the Ribo-Zero rRNA Removal Kit for Bacteria from Illumina, but unfortunately it is not available anymore on the market. However, it can be replaced with any hybridization-based kit of your choice. We provided some suggestions in the Material section.<br/><br/></li><li>Terminator exonuclease (TEX) treatment to enrich for primary transcripts <br/><ol class="npwtwo"><li>Prepare two (TEX- and TEX+) 1.5 ml reaction tubes with 10 Î¼g (10 Î¼l) each of DNase I treated RNA (from the same sample) in nuclease free water.<br/></li><li>Denature RNAs for 2 min at 90 Â°C.<br/></li><li>Cool down on ice for 5 min.<br/></li><li>Add to each tube the reaction mix described in Table 4. The indicated quantities are sufficient for one sample containing 10 Î¼g of RNA:<br/><br/><strong>Table 4. Reaction mix for TEX treatment</strong><br/><img alt="" class="layerphoto" height="196" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004542_8839.png" width="500"/><br/></li></ol><br/><em>Note: The final reaction volume is going to be 20 Î¼l after adding the enzyme for the TEX+ sample or water for the TEX- sample.</em><br/><ol class="npwtwo" start="5"><li>Add 1 Î¼l of nuclease free water to the TEX- sample.</li><li>Add 1 Î¼l of TEX (1 U/ml) to the TEX+ sample.</li><li>Incubate for 60 min at 30 Â°C in the thermo cycler.<br/><em>Note: This step could be carried in a heat block, but the temperature variation and liquid evaporation are important factors that hamper good results, and these effects are stronger in a heat block.</em> </li><li>Add 80 Î¼l of nuclease free water to obtain 100 Î¼l final volume.</li><li>Add 100 Î¼l of Phe-CHISAM to the reaction tube under the chemical hood.</li><li>Mix with the vortex vigorously for 15 s.</li><li>Centrifuge the tube for 2 min at 13,800 <em>x g</em>, 4 Â°C.</li><li>Transfer ~100 Î¼l of the upper phase that contains the RNA to a new tube under the chemical hood.</li><li>Repeat Steps F9 to F12.</li><li>Add 300 Î¼l of cold absolute ethanol (3 sample volumes), 10 Î¼l of sodium acetate 3M pH 5.5 (1/10 of sample volume) and 2 Î¼l of glycogen (20 mg/ml).</li><li>Vortex the sample vigorously.</li><li>Incubate the tube at -20 Â°C overnight (around 16 h), but could be longer, for example over the weekend.<br/><em>Note: This step is very important. It helps complete recovery of the RNA.</em> </li><li>Centrifuge at 16,200 <em>x g</em> for 1 h at 4 Â°C.<br/><em>Note: The centrifugation step could be for longer time. Minimum 1 h, but could be for 2-4 h.</em> </li><li>Discard the aqueous phase carefully by decanting.<br/><em>Note: On the bottom of the tube there should be a white pellet. Discard the aqueous phase without detaching the pellet. In the case that you do not see any pellet before discarding the supernatant, just add 10 Î¼l of sodium acetate 3 M pH 5.5 (1/10 of sample volume) and 2 Î¼l of glycogen (20 mg/ml) and repeat Steps F16 to F18).</em> </li><li>Add 1 ml of cold 80% ethanol at room temperature to wash the RNA.</li><li>Centrifuge for 5 min at 13,800 <em>x g</em>, 4 Â°C.</li><li>Discard the aqueous phase by decanting.</li><li>Centrifuge (short spin) and discard the remaining liquid with a 100 Î¼l pipette.</li><li>Keep the tubes open on ice to dry the samples.<br/><em>Note: The time required to dry the samples depends on how well the liquid phase was removed. It can take around 10-15 min. The pellet will be completely transparent once it is completely dry.</em> </li><li>Resuspend the RNA in 13 Î¼l of nuclease free water.</li><li>Check the integrity of the TEX- and TEX+ treated RNA by visual inspection on 1% agarose gel.</li><li>Prepare the running sample as in Steps A29-A36. </li><li>Visualize the gel under UV light (Figure 6).<br/><br/><img alt="" class="layerphoto" height="158" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004640_9482.png" width="300"/><br/><strong>Figure 6. <em>Shigella flexneri</em> 5a M90T RNA treated (TEX+) and untreated (TEX-) with TEX</strong><br/><br/><em>Note: It is normal not to see clear bands in this gel for the mRNA, which is a mix of different transcripts of variable length. To see a smear in the gel means that the treatment was successful. If you do not have a high quantity of RNA maybe you will see only tRNAs.</em><br/><br/></li><li>Quantify the RNA in the NanoDrop spectrophotometer (final RNA concentration should be ~20 ng/ml).</li><li>Freeze the RNA samples at -20 Â°C. The RNA samples are stables for months.<br/><br/></li></ol></li><li>RNA 5â Pyrophosphohydrolase (RppH) treatment of RNA to remove terminal phosphates<br/>In the following steps, the 5â²-ends of the RNA prepared in Procedure F is modified with a linker-RNA-adaptor of known sequence, which allows the identification of TSS with single nucleotide precision. This is necessary because the sequence of adaptors added during library preparation for sequencing can be a trade secret (the full sequence of Illumina adaptors was only revealed recently and other platforms using adaptors of unknown sequence exist). During bioinformatic analysis of sequencing results, the sequence of adaptors is occasionally incompletely removed from reads and the remaining nucleotides could be confused for transcribed bases if they happened to align with the DNA sequence. However, this problem can be avoided by introducing the RNA adaptor of known sequence between the adaptors and the TSS. Moreover, the linker-RNA-adaptor allows to univocally identify 5â² ends present before RNA shearing during library preparation. To allow the ligation of adaptors to the 5â end of the linker-RNA-adaptor during library preparation, the linker-RNA-adaptor must have a triphosphate in the 5â² end.<br/><ol class="npwtwo"><li>Denature remaining 10 Î¼l of TEX+ and TEX- treated RNA for 1 min at 90 Â°C.</li><li>Cool down on ice for 5 min.</li><li>Prepare RppH mix as described in the Table 5. The indicated quantities are sufficient for one sample:<br/><br/><strong>Table 5. Reaction mix for RppH reaction of TEX treated RNA</strong><br/><img alt="" class="layerphoto" height="231" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004732_2081.png" width="680"/><br/><br/></li><li>Incubate at 37 Â°C for 30 min in the thermocycler.</li><li>Add 80 Î¼l of nuclease free water and follow the Steps F8-F23.</li><li>Resuspend the RNA sample in 13 Î¼l of nuclease free water.</li><li>Check the integrity of the RppH treated RNA by visual inspection on 1% agarose gel.<br/><em>Note: This step it not strictly necessary, and with every step you are losing some RNA. The gel should look similar as the one shown in Figure 6.</em> </li><li>Quantify the RNA in the NanoDrop spectrophotometer.<br/><br/></li></ol></li><li>Linker-RNA-adaptor phosphorylation prior to ligation to TEX+ and TEX- libraries<ol class="npwtwo"><li>Denature 100 pmol (10 Î¼l) of the linker-RNA-adaptor per sample dissolved in nuclease free water for 5 min at 90 Â°C.</li><li>Cool down on ice for 5 min.<br/><em>Note: You can keep the oligo on ice for longer time while you prepare the reaction.</em> </li><li>Prepare the T4 PNK mix according the Table 6.<br/><br/><strong>Table 6. Reaction mix for T4 PNK phosphorylation of the linker-RNA-adaptor</strong><br/><img alt="" class="layerphoto" height="254" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004843_0566.png" width="500"/><br/><br/></li><li>Incubate at 37 Â°C for 30 min in the thermo cycler.</li><li>Add 80 Î¼l of nuclease free water, follow the Steps F8-F23.</li><li>Resuspend the RNA sample in 10 Î¼l of nuclease free water.<br/><em>Note: After this step the RNA adaptor can be frozen at -20 Â°C and used later.</em><br/><br/></li></ol></li><li>Linker-RNA-adaptor ligation to TEX+ and TEX- samples<br/>The efficiency of linker-RNA-adaptor ligation is around 80-90%.<br/><ol class="npwtwo"><li>Denature the phosphorylated RNA adaptor (10 Î¼l) and RNAs treated with RppH (10 Î¼l) at 65 Â°C for 5 min.</li><li>Cool down on ice for 5 min.</li><li>Prepare the T4 RNA ligase mix as described in Table 7 (quantities described on the table are for only one sample, if you are processing more samples just scale them up).<br/><br/><strong>Table 7. Reaction mix for linker-RNA-adaptor ligation</strong><br/><img alt="" class="layerphoto" height="341" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916004945_8827.png" width="580"/><br/><br/></li><li>Incubate at 25 Â°C for 16 h in the thermo cycler.</li><li>Add 60 Î¼l of nuclease free water.</li><li>Stop the reaction and precipitate the reaction following Steps F9-F23.</li><li>Resuspend the RNA sample in 23 Î¼l of nuclease free water.<br/><em>Note: Resuspension volume is defined by the sequencing facilities that you have decided to use. The RNA concentration in our case was 150 ng in 23 Î¼l.</em> </li><li>Quantify the RNA adaptor by NanoDrop spectrophotometer.<br/><br/></li></ol></li><li>Library preparation for Illumina sequencing<br/>Library preparation for sequencing and RNA-sequencing is performed at the service platform or company of your choice. Stranded libraries must be prepared to identify the coding DNA strand. Paired-end sequencing with 150 nt read length should be performed. We used a TrueSeq library on a HiSeq 2000 platform. For microbial genomes, 10-20 M reads per sample will provide appropriate sequencing depth, but 5 M it is enough for an acceptable coverage.<br/></li></ol><p class="pptt" id="biaoti35866">### Data analysis</p><p class="ppzz">All the programs used in this protocol are based on Linux and freely available on the web. We perform all analyses remotely on a high-performance computer cluster. However, if no access to high-speed remote analysis is available, it is possible to run all the analysis locally after downloading the programs, but this may take a long time if the computing power is low.<br/>    Download all the programs, raw data and the reference genome into the working directory. In this protocol as example we have called the working directory dRNA_seq.<br/><br/><em>Note: If you use Windows system you should install Ubuntu (</em><a href="https://ubuntu.com/download/desktop" target="_blank"><em>https://ubuntu.com/download/desktop</em></a><em>) and run all the programs in this environment.</em> </p><br/><ol class="npwone"><li>Demultiplexing of Illumina reads<br/><em>Note: This step is only necessary in case you perform the sequencing in your own laboratory. If it is the case you will get a file called *.bcl.gz, which first needs demultiplexing for downstream analysis. This step is not necessary if you use a sequencing service, because you will get the already processed *.fastq.gz file.</em><br/>bcl2fastq<br/><ol class="npwthree"><li>Go to your working directory where you have your *.bcl.gz file.<br/></li><li>Execute bcl2fastq<br/><span style="font-family:Courier New;">$ bcl2fastq [options]</span><br/></li></ol><br/></li><li>Quality control of raw data<ol class="npwtwo"><li>FastQC (Figure 7)<br/><ol class="npwthree"><li>Go to your working directory. In this example the working directory: <br/><span style="font-family:Courier New;">$ cd dRNA_seq</span><span style="font-family:Courier New;"></span><br/></li><li>Create a new directory for the output of FastQC analysis.<br/><span style="font-family:Courier New;">$ mkdir FastQC_1</span><span style="font-family:Courier New;"></span><br/></li><li>Execute FastQC.<br/><span style="font-family:Courier New;">$ fastqc /dRNA_seq/your_file.fq.gz -o /dRNA_seq/ FastQC_1</span><br/><br/><img alt="" class="layerphoto" height="108" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916005102_7269.png" width="300"/><br/><strong>Figure 7. FastQC report of quality control. A. Successful quality control.</strong> The quality scores should be in the green area for the entire length of the sequenced fragment. B. Unsuccessful quality control. The quality scores for part of the reads are low (found in the orange or red areas).<br/></li></ol><br/></li><li>MultiQC<br/>Run MultiQC to merge all the FastQC outputs.<br/><span style="font-family:Courier New;">$ multiqc /FastQC_1<br/></span><br/></li></ol></li><li>Cleaning raw data from low quality sequences and Illumina adaptors<br/>Trimmomatic<br/>Run Trimmomatic in your working directory.<br/><span style="font-family:Courier New;">$ java -jar trimmomatic.jar PE /your_raw_data_1.fq.gz your_raw_data_2.fq.gz your_raw_data_paired_1.fq.gz your_raw_data_unpaired_1.fq.gz your_raw_data_paured_2.fq.gz your_raw_data_unpaired_2.fq.gz ILLUMINACLIP:/TruSeq3-PE-2.fa:2:30:10 LEADING:20 TRAILING:20 SLIDINGWINDOW:4:15 MINLEN:100<br/></span><br/></li><li>Select the fastq files that contains the RNA adaptor sequence<br/>The easiest way to perform this step is using the command grep in your terminal. The linker-RNA-adaptor at this step should be in DNA format.<br/><span style="font-family:Courier New;">$ zgrep -B 1 -A 2 GACCTTGGCTGTCACTCA your_raw_data_paired_1.fq.gz &gt; Picked_Up.fastq<br/><br/></span> </li><li>Genome reference index<ol class="npwtwo"><li>It is quite likely that your reference genome has more than one replicon. We would recommend to merge all of them into one single file to be used as a refence genome. <br/><span style="font-family:Courier New;">$ cat chromosome.fasta plasmid.fasta &gt; reference_genome.fasta</span> </li><li>Create the index with bowtie2.<br/><span style="font-family:Courier New;">$ bowtie2-build reference_genome.fasta index_base_name<br/></span><br/></li></ol></li><li>Genome reference mapping<br/>Map the selected reads with the reference genome.<br/><span style="font-family:Courier New;">$ bowtie2 -q -N 0 âno-unal -x index_base_name -U your_raw_data_paured_1.fq.gz -S mapping_1.sam</span><br/><br/></li><li>Sort by position and build an index of BAM file<br/>The output file from Bowtie2 should be sorted and indexed for use in IGV.<br/><span style="font-family:Courier New;">$ samtools view -Sb -o mapping_1.bam mapping_1.sam</span><br/><span style="font-family:Courier New;"> $ samtools sort -o mapping_Sorted_1.bam mapping_1.bam</span><br/><br/></li><li>Visualize the mapping files in IGV (Figure 8)<br/><ol class="npwtwo"><li>Create a genome reference index.<br/><span style="font-family:Courier New;">$ samtools index reference_genome.fasta reference_genome</span> </li><li>Create your genome file in IGV.<br/><span style="font-family:Courier New;">IGV&gt;Genomes&gt;Create .genome File</span><br/><span style="font-family:Courier New;"> Load your reference_genome.fasta and reference_genome.gbk files</span> </li><li>Load your mapping file.<br/><span style="font-family:Courier New;">IGV&gt;File&gt;Load from File</span><br/><br/><img alt="" class="layerphoto" height="143" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916005143_0233.png" width="300"/><br/><strong>Figure 8. Screenshot of the IGV browser for alignment visualization</strong><strong>.</strong> Alignment of TEX+ and TEX- libraries with the reference genome of <em>S. flexneri</em> 5a M90T. The blue histogram shows the coverage of the TEX- sample, the red histogram shows the TEX+ sample coverage. The red highlight shows the putative transcriptional start site.<br/><br/></li></ol></li><li>Identify the transcriptional start sites<br/><ol class="npwtwo"><li>Merge the three replicates in one file.<br/><span style="font-family:Courier New;">$ samtools merge output.sam mapping_Sorted_1.bam mapping_Sorted_2.bam mapping_Sorted_3.bam</span><span style="font-family:Courier New;"></span><br/></li><li>Load the merged file in ReadXplorer.<br/></li><li>Run the TSS analysis with these parameters (Table 8):<br/><br/><strong>Table 8. Settings for the TSS analysis in ReadXplorer</strong><br/><img alt="" class="layerphoto" height="582" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916005259_0715.png" width="700"/><br/><br/></li><li>The output file of ReadXplorer analysis for TSS is a table in plain text with 39 different columns that you can use for a downstream analysis. As an example you can see an extract of some columns of the output file in Figure 9.<br/><br/><img alt="" class="layerphoto" height="57" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916005340_3234.png" width="300"/><br/><strong>Figure 9. Example of extract from the ReadXplorer output file</strong><br/><br/></li></ol>According to the analysis with ReadXplorer, we identified 6,723 pTSS and 7,328 sTSS in <em>S. flexneri</em> 5a M90T (Cervantes-Rivera <em>et al.</em>, 2020), which is around the same number of TSS identified in <em>E. coli</em> MG1655 (Conway <em>et al.</em>, 2014; Thomason <em>et al.</em>, 2015). These results are also available at RegulonDB at <a href="http://regulondb.ccg.unam.mx/central_panel_menu/integrated_views_and_tools_menu.jsp" target="_blank">http://regulondb.ccg.unam.mx/central_panel_menu/integrated_views_and_tools_menu.jsp</a>.</li></ol><p class="pptt" id="biaoti35867">### Recipes</p><ol class="npwtwok"><li>Tryptone soy broth (TSB)<br/>30 g TSB medium powder<br/>MilliQ water to 1 L<br/>Autoclave the solution</li><li>Tryptone soy agar (TSA) plates<br/>40 g Tryptone soy agar powder<br/>MilliQ water to 1 L<br/>Autoclave the solution<br/>Cool to ~50 Â°C in the water bath, add 10 ml of 1% Congo red solution and pour plates</li><li>Congo red solution 1% (w/v)<br/>1 g of Congo red dye<br/>100 ml of MilliQ-water<br/>Filter sterilize the solution, using a 0.2 Î¼m membrane filter.<br/><em>Note: The Congo red solution is stable for months at room temperature.</em> </li><li>Nuclease free water<br/>1 ml of Diethyl Pyrocarbonate (DEPC), (0.1% final concentration)<br/>1 L of MilliQ water<br/>Mix well over night using a magnetic stirring bar<br/>Autoclave<br/>Let cool down to room temperature prior to use</li><li>Sodium dodecyl sulfate (SDS) 10% (w/v)<br/>10 g of SDS powder<br/>Nuclease free water 100 ml<br/>Autoclave the solution</li><li>EDTA 0.5 M, pH 8<br/>18.61 g disodium ethylenediaminetetraacetate dihydrate (EDTA)<br/>2 g Sodium hydroxide<br/>Nuclease free water 100 ml<br/>Mix well and sterilize<br/><em>Note: EDTA is solved at pH 8.</em> </li><li>DNA loading buffer 6x<br/>0.125 g bromophenol blue (0.025% final concentration)<br/>0.125 g xylene cyanol (0.025% final concentration)<br/>30 ml Glycerol <br/>Nuclease free water to 100 ml<br/>Mix well, it is stable at room temperature for years</li><li>RNA loading buffer 2x<br/>0.125 g bromophenol blue (0.025% final concentration)<br/>0.125 g xylene cyanol (0.025% final concentration)<br/>100 ml of formamide<br/>Mix well, it is stable at room temperature for years</li><li>Stop Solution<br/>95 ml of ethanol<br/>5 ml of phenol pH 4<br/>Final volume 100 ml<br/>Mix well, store it at 4 Â°C</li><li>Ethanol 80%<br/>80 ml of ethanol<br/>20 ml of nuclease free water<br/>Mix well and store it at 4 Â°C</li><li>Lysis solution<br/>5 ml of SDS 10% (0.5% final concentration)<br/>0.6 ml of sodium acetate pH 5.5 (20 mM final concentration)<br/>2 ml of EDTA pH 8 (10 mM final concentration)<br/>Nuclease free water to 100 ml (92.4 ml) <br/>Mix well, store at room temperature. <br/><em>Note: At cold temperature the SDS precipitates. If this happens warm up for 15 s at maximum power in the microwave oven.</em> </li><li>10x TAE<br/>11.42 ml of acetic acid<br/>48.4 g of Trizma-base<br/>20 ml of EDTA 0.5 M, pH 8<br/>Nuclease free water to 1 L</li><li>Agarose 1%<br/>1 g of agarose powder<br/>100 ml of TAE 1x<br/>Melt the agarose in a microwave oven</li><li>Phe-CHISAM solution<br/>25 ml of phenol acid pH 4<br/>24 ml of chloroform<br/>1 ml of isoamyl alcohol<br/>Mix well, store at room temperature in the chemical hood </li><li>CHISAM<br/>96 ml of chloroform<br/>4 ml of isoamyl alcohol<br/>Mix well, store at room temperature in the chemical hood<br/></li></ol><p class="pptt" id="biaoti35868">### Acknowledgments</p><p class="ppzz">A short version of this protocol was published in Cervantes-Rivera <em>et al.</em> (2020). We gratefully acknowledge funding from the Kempe Foundation (JCK-1528), the Knut and Alice Wallenberg Foundation (KAW 2015.0225), UmeÃ¥ Centre for Microbial Research (UCMR), UmeÃ¥ University and The Laboratory for Molecular Infection Medicine Sweden (MIMS). We are grateful for travel support and courses organized by The National Doctoral Program in Infections and Antibiotics (NDPIA). The computations were performed on resources provided by SNIC through Uppsala Multidisciplinary Center for Advanced Computational Science (UPPMAX) under Project SNIC 2017-7-258.</p><p class="pptt" id="biaoti35869">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti35870">### References</p><ol class="npwtwok"><li>Albrecht, M., Sharma, C. M., Dittrich, M. T., Muller, T., Reinhardt, R., Vogel, J. and Rudel, T. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21989159" target="_blank">The transcriptional landscape of <em>Chlamydia pneumoniae</em>.</a> <em>Genome Biol</em> 12(10): R98.</li><li>Albrecht, M., Sharma, C. M., Reinhardt, R., Vogel, J. and Rudel, T. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19923228" target="_blank">Deep sequencing-based discovery of the <em>Chlamydia trachomatis</em> transcriptome.</a> <em>Nucleic Acids Res</em> 38(3): 868-877.</li><li>Amman, F., Wolfinger, M. T., Lorenz, R., Hofacker, I. L., Stadler, P. F. and Findeiss, S. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24674136" target="_blank">TSSAR: TSS annotation regime for dRNA-seq data.</a> <em>BMC Bioinformatics</em> 15: 89.</li><li>Beckmann, B. M., Burenina, O. Y., Hoch, P. G., Kubareva, E. A., Sharma, C. M. and Hartmann, R. K. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21881410" target="_blank"><em>In vivo</em> and <em>in vitro</em> analysis of 6S RNA-templated short transcripts in <em>Bacillus subtilis</em>.</a> <em>RNA Biol</em> 8(5): 839-849. </li><li>Berghoff, B. A., Glaeser, J., Sharma, C. M., Vogel, J. and Klug, G. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19906181" target="_blank">Photooxidative stress-induced and abundant small RNAs in <em>Rhodobacter sphaeroides</em>.</a> <em>Mol Microbiol</em> 74(6): 1497-1512.</li><li>Bohn, C., Rigoulay, C., Chabelskaya, S., Sharma, C. M., Marchais, A., Skorski, P., Borezee-Durant, E., Barbet, R., Jacquet, E., Jacq, A., Gautheret, D., Felden, B., Vogel, J. and Bouloc, P. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20511587" target="_blank">Experimental discovery of small RNAs in <em>Staphylococcus aureus</em> reveals a riboregulator of central metabolism.</a> <em>Nucleic Acids Res</em> 38(19): 6620-6636.</li><li>Bolger, A. M., Lohse, M. and Usadel, B. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24695404" target="_blank">Trimmomatic: a flexible trimmer for Illumina sequence data.</a> <em>Bioinformatics</em> 30(15): 2114-2120.</li><li>Cervantes-Rivera, R., Tronnet, S. and Puhar, A. (2020). <a href="https://www.ncbi.nlm.nih.gov/pubmed/32252626" target="_blank">Complete genome sequence and annotation of the laboratory reference strain <em>Shigella flexneri</em> serotype 5a M90T and genome-wide transcriptional start site determination.</a> <em>BMC Genomics</em> 21(1): 285.</li><li>Conway, T., Creecy, J. P., Maddox, S. M., Grissom, J. E., Conkle, T. L., Shadid, T. M., Teramoto, J., San Miguel, P., Shimada, T., Ishihama, A., Mori, H. and Wanner, B. L. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25006232" target="_blank">Unprecedented high-resolution view of bacterial operon architecture revealed by RNA sequencing.</a> <em>mBio</em> 5(4): e01442-01414.</li><li>Cortes, T., Schubert, O. T., Rose, G., Arnvig, K. B., Comas, I., Aebersold, R. and Young, D. B. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24268774" target="_blank">Genome-wide mapping of transcriptional start sites defines an extensive leaderless transcriptome in <em>Mycobacterium tuberculosis</em>.</a> <em>Cell Rep</em> 5(4): 1121-1131. </li><li>Deltcheva, E., Chylinski, K., Sharma, C. M., Gonzales, K., Chao, Y., Pirzada, Z. A., Eckert, M. R., Vogel, J. and Charpentier, E. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21455174" target="_blank">CRISPR RNA maturation by trans-encoded small RNA and host factor RNase III.</a> <em>Nature</em> 471(7340): 602-607.</li><li>Dugar, G., Herbig, A., Forstner, K. U., Heidrich, N., Reinhardt, R., Nieselt, K. and Sharma, C. M. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23696746" target="_blank">High-resolution transcriptome maps reveal strain-specific regulatory features of multiple <em>Campylobacter jejuni</em> isolates.</a> <em>PLoS Genet</em> 9(5): e1003495.</li><li>Ewels, P., Magnusson, M., Lundin, S. and Kaller, M. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27312411" target="_blank">MultiQC: summarize analysis results for multiple tools and samples in a single report.</a> <em>Bioinformatics</em> 32(19): 3047-3048.</li><li>Feklistov, A., Sharon, B. D., Darst, S. A. and Gross, C. A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25002089" target="_blank">Bacterial sigma factors: a historical, structural, and genomic perspective.</a> <em>Annu Rev Microbiol</em> 68: 357-376.</li><li>Filiatrault, M. J., Stodghill, P. V., Myers, C. R., Bronstein, P. A., Butcher, B. G., Lam, H., Grills, G., Schweitzer, P., Wang, W., Schneider, D. J. and Cartinhour, S. W. (2011). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22216251" target="_blank">Genome-wide identification of transcriptional start sites in the plant pathogen <em>Pseudomonas syringae</em> pv. <em>tomato</em> str. DC3000.</a> <em>PLoS One</em> 6(12): e29335. </li><li>He, F. (2011). <a href="https://bio-protocol.org/bio101/e97" target="_blank"><em>E. coli</em> Genomic DNA Extraction.</a> <em>Bio-protocol</em> 1(14): e97.</li><li>Hilker, R., Stadermann, K. B., Doppmeier, D., Kalinowski, J., Stoye, J., Straube, J., Winnebald, J. and Goesmann, A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24790157" target="_blank">ReadXplorer--visualization and analysis of mapped sequences.</a> <em>Bioinformatics</em> 30(16): 2247-2254.</li><li>Hilker, R., Stadermann, K. B., Schwengers, O., Anisiforov, E., Jaenicke, S., Weisshaar, B., Zimmermann, T. and Goesmann, A. (2016). <a href="https://www.ncbi.nlm.nih.gov/pubmed/27540267" target="_blank">ReadXplorer 2-detailed read mapping analysis and visualization from one single source.</a> <em>Bioinformatics</em> 32(24): 3702-3708.</li><li>Irnov, I., Sharma, C. M., Vogel, J. and Winkler, W. C. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20525796" target="_blank">Identification of regulatory RNAs in <em>Bacillus subtilis</em>.</a> <em>Nucleic Acids Res</em> 38(19): 6637-6651.</li><li>Jacob, F. and Monod, J. (1961). <a href="https://www.ncbi.nlm.nih.gov/pubmed/13718526" target="_blank">Genetic regulatory mechanisms in the synthesis of proteins.</a> <em>J Mol Biol</em> 3: 318-356. </li><li>Jager, D., Sharma, C. M., Thomsen, J., Ehlers, C., Vogel, J. and Schmitz, R. A. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19996181" target="_blank">Deep sequencing analysis of the <em>Methanosarcina mazei</em> Go1 transcriptome in response to nitrogen availability.</a> <em>Proc Natl Acad Sci U S A</em> 106(51): 21878-21882.</li><li>Jorjani, H. and Zavolan, M. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24371151" target="_blank">TSSer: an automated method to identify transcription start sites in prokaryotic genomes from differential RNA sequencing data.</a> <em>Bioinformatics</em> 30(7): 971-974.</li><li>Kroger, C., Colgan, A., Srikumar, S., Handler, K., Sivasankaran, S. K., Hammarlof, D. L., Canals, R., Grissom, J. E., Conway, T., Hokamp, K. and Hinton, J. C. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24331466" target="_blank">An infection-relevant transcriptomic compendium for <em>Salmonella enterica</em> Serovar Typhimurium.</a> <em>Cell Host Microbe</em> 14(6): 683-695.</li><li>Kroger, C., Dillon, S. C., Cameron, A. D., Papenfort, K., Sivasankaran, S. K., Hokamp, K., Chao, Y., Sittka, A., Hebrard, M., Handler, K., Colgan, A., Leekitcharoenphon, P., Langridge, G. C., Lohan, A. J., Loftus, B., Lucchini, S., Ussery, D. W., Dorman, C. J., Thomson, N. R., Vogel, J. and Hinton, J. C. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22538806" target="_blank">The transcriptional landscape and small RNAs of <em>Salmonella enterica</em> serovar Typhimurium.</a> <em>Proc Natl Acad Sci U S A</em> 109(20): E1277-1286. </li><li>Langmead, B. and Salzberg, S. L. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22388286" target="_blank">Fast gapped-read alignment with Bowtie 2.</a> Nat Methods 9(4): 357-359. </li><li>Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G. and Durbin, R. (2009). <a href="https://www.ncbi.nlm.nih.gov/pubmed/19505943" target="_blank">The Sequence Alignment/Map format and SAMtools.</a> <em>Bioinformatics</em> 25(16): 2078-2079.</li><li>Li, S., Dong, X. and Su, Z. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23899370" target="_blank">Directional RNA-seq reveals highly complex condition-dependent transcriptomes in <em>E. coli</em> K12 through accurate full-length transcripts assembling.</a> <em>BMC Genomics</em> 14: 520.</li><li>Madhugiri, R., Pessi, G., Voss, B., Hahn, J., Sharma, C. M., Reinhardt, R., Vogel, J., Hess, W. R., Fischer, H. M. and Evguenieva-Hackenberg, E. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22258152" target="_blank">Small RNAs of the <em>Bradyrhizobium</em>/<em>Rhodopseudomonas</em> lineage and their analysis.</a> <em>RNA Biol</em> 9(1): 47-58.</li><li>Mentz, A., Neshat, A., Pfeifer-Sancar, K., Puhler, A., Ruckert, C. and Kalinowski, J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24138339" target="_blank">Comprehensive discovery and characterization of small RNAs in <em>Corynebacterium glutamicum</em> ATCC 13032.</a> <em>BMC Genomics</em> 14: 714.</li><li>Mitschke, J., Georg, J., Scholz, I., Sharma, C. M., Dienst, D., Bantscheff, J., Voss, B., Steglich, C., Wilde, A., Vogel, J. and Hess, W. R. (2011a). <a href="https://www.ncbi.nlm.nih.gov/pubmed/21245330" target="_blank">An experimentally anchored map of transcriptional start sites in the model cyanobacterium <em>Synechocystis</em> sp. PCC6803.</a> <em>Proc Natl Acad Sci U S A</em> 108(5): 2124-2129.</li><li>Mitschke, J., Vioque, A., Haas, F., Hess, W. R. and Muro-Pastor, A. M. (2011b). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22135468" target="_blank">Dynamics of transcriptional start site selection during nitrogen stress-induced cell differentiation in <em>Anabaena</em> sp. PCC7120.</a> <em>Proc Natl Acad Sci U S A</em> 108(50): 20130-20135.</li><li>Murakami, K. S. and Darst, S. A. (2003). <a href="https://www.ncbi.nlm.nih.gov/pubmed/12581657" target="_blank">Bacterial RNA polymerases: the wholo story.</a> <em>Curr Opin Struct Biol</em> 13(1): 31-39.</li><li>Nickel, L., Weidenbach, K., Jager, D., Backofen, R., Lange, S. J., Heidrich, N. and Schmitz, R. A. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23619576" target="_blank">Two CRISPR-Cas systems in <em>Methanosarcina mazei</em> strain Go1 display common processing features despite belonging to different types I and III.</a> <em>RNA Biol</em> 10(5): 779-791.</li><li>Pfeifer-Sancar, K., Mentz, A., Ruckert, C. and Kalinowski, J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24341750" target="_blank">Comprehensive analysis of the Corynebacterium <em>glutamicum transcriptome</em> using an improved RNAseq technique.</a> <em>BMC Genomics</em> 14: 888.</li><li>Porcelli, I., Reuter, M., Pearson, B. M., Wilhelm, T. and van Vliet, A. H. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24028687" target="_blank">Parallel evolution of genome structure and transcriptional landscape in the Epsilonproteobacteria.</a> <em>BMC Genomics</em> 14: 616.</li><li>Ramachandran, V. K., Shearer, N., Jacob, J. J., Sharma, C. M. and Thompson, A. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22251276" target="_blank">The architecture and ppGpp-dependent expression of the primary transcriptome of <em>Salmonella</em> Typhimurium during invasion gene expression.</a> <em>BMC Genomics</em> 13: 25.</li><li>Ramachandran, V. K., Shearer, N. and Thompson, A. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24664308" target="_blank">The primary transcriptome of Salmonella enterica <em>Serovar Typhimurium</em> and its dependence on ppGpp during late stationary phase.</a> <em>PLoS One</em> 9(3): e92690.</li><li>Robb, N. C., Cordes, T., Hwang, L. C., Gryte, K., Duchi, D., Craggs, T. D., Santoso, Y., Weiss, S., Ebright, R. H. and Kapanidis, A. N. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23274143" target="_blank">The transcription bubble of the RNA polymerase-promoter open complex exhibits conformational heterogeneity and millisecond-scale dynamics: implications for transcription start-site selection.</a> <em>J Mol Biol</em> 425(5): 875-885.</li><li>Sahr, T., Rusniok, C., Dervins-Ravault, D., Sismeiro, O., Coppee, J. Y. and Buchrieser, C. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22546937" target="_blank">Deep sequencing defines the transcriptional map of <em>L. pneumophila</em> and identifies growth phase-dependent regulated ncRNAs implicated in virulence.</a> <em>RNA Biol</em> 9(4): 503-519.</li><li>Sansonetti, P. J., Kopecko, D. J. and Formal, S. B. (1982). <a href="https://www.ncbi.nlm.nih.gov/pubmed/6279518" target="_blank">Involvement of a plasmid in the invasive ability of <em>Shigella flexneri</em>.</a> <em>Infect Immun</em> 35(3): 852-860.</li><li>Schluter, J. P., Reinkensmeier, J., Barnett, M. J., Lang, C., Krol, E., Giegerich, R., Long, S. R. and Becker, A. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23497287" target="_blank">Global mapping of transcription start sites and promoter motifs in the symbiotic alpha-proteobacterium <em>Sinorhizobium meliloti</em> 1021.</a> <em>BMC Genomics</em> 14: 156.</li><li>Schluter, J. P., Reinkensmeier, J., Daschkey, S., Evguenieva-Hackenberg, E., Janssen, S., Janicke, S., Becker, J. D., Giegerich, R. and Becker, A. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20398411" target="_blank">A genome-wide survey of sRNAs in the symbiotic nitrogen-fixing alpha-proteobacterium <em>Sinorhizobium meliloti</em>.</a> <em>BMC Genomics</em> 11: 245.</li><li>Schmidtke, C., Findeiss, S., Sharma, C. M., Kuhfuss, J., Hoffmann, S., Vogel, J., Stadler, P. F. and Bonas, U. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22080557" target="_blank">Genome-wide transcriptome analysis of the plant pathogen <em>Xanthomonas</em> identifies sRNAs with putative virulence functions.</a> <em>Nucleic Acids Res</em> 40(5): 2020-2031.</li><li>Schoenberg, D. R. (2007). <a href="https://www.ncbi.nlm.nih.gov/pubmed/17710097" target="_blank">The end defines the means in bacterial mRNA decay.</a> <em>Nat Chem Biol</em> 3(9): 535-536. </li><li>Sharma, A. and Puhar, A. (2019). <a href="https://bio-protocol.org/e3292" target="_blank">Gentamicin protection assay to determine the number of intracellular bacteria during infection of human tc7 intestinal epithelial cells by <em>Shigella flexneri</em>.</a> <em>Bio-protocol</em> 9(13): e3292. </li><li>Sharma, C. M., Hoffmann, S., Darfeuille, F., Reignier, J., Findeiss, S., Sittka, A., Chabas, S., Reiche, K., Hackermuller, J., Reinhardt, R., Stadler, P. F. and Vogel, J. (2010). <a href="https://www.ncbi.nlm.nih.gov/pubmed/20164839" target="_blank">The primary transcriptome of the major human pathogen <em>Helicobacter pylori</em>.</a> <em>Nature</em> 464(7286): 250-255. </li><li>Sharma, C. M. and Vogel, J. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25024085" target="_blank">Differential RNA-seq: the approach behind and the biological insight gained.</a> <em>Curr Opin Microbiol</em> 19: 97-105. </li><li>Thomason, M. K., Bischler, T., Eisenbart, S. K., Forstner, K. U., Zhang, A., Herbig, A., Nieselt, K., Sharma, C. M. and Storz, G. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25266388" target="_blank">Global transcriptional start site mapping using differential RNA sequencing reveals novel antisense RNAs in <em>Escherichia coli.</em></a> <em>J Bacteriol</em> 197(1): 18-28.</li><li>Thorvaldsdottir, H., Robinson, J. T. and Mesirov, J. P. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22517427" target="_blank">Integrative Genomics Viewer (IGV): high-performance genomics data visualization and exploration.</a> <em>Brief Bioinform</em> 14(2): 178-192.</li><li>Voigt, K., Sharma, C. M., Mitschke, J., Lambrecht, S. J., Voss, B., Hess, W. R. and Steglich, C. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24739626" target="_blank">Comparative transcriptomics of two environmentally relevant cyanobacteria reveals unexpected transcriptome diversity.</a> <em>ISME J</em> 8(10): 2056-2068.</li><li>Voss, B., Bolhuis, H., Fewer, D. P., Kopf, M., Moke, F., Haas, F., El-Shehawy, R., Hayes, P., Bergman, B., Sivonen, K., Dittmann, E., Scanlan, D. J., Hagemann, M., Stal, L. J. and Hess, W. R. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23555932" target="_blank">Insights into the physiology and ecology of the brackish-water-adapted Cyanobacterium <em>Nodularia spumigena</em> CCY9414 based on a genome-transcriptome analysis.</a> <em>PLoS One</em> 8(3): e60224.</li><li>Vvedenskaya, I. O., Zhang, Y., Goldman, S. R., Valenti, A., Visone, V., Taylor, D. M., Ebright, R. H. and Nickels, B. E. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/26626484" target="_blank">Massively systematic transcript end readout, "MASTER": transcription start site selection, transcriptional slippage, and transcript yields.</a> <em>Mol Cell</em> 60(6): 953-965.</li><li>Wiegand, S., Dietrich, S., Hertel, R., Bongaerts, J., Evers, S., Volland, S., Daniel, R. and Liesegang, H. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24079885" target="_blank">RNA-Seq of <em>Bacillus licheniformis</em>: active regulatory RNA features expressed within a productive fermentation.</a> <em>BMC Genomics</em> 14: 667.</li><li>Wilms, I., Overloper, A., Nowrousian, M., Sharma, C. M. and Narberhaus, F. (2012). <a href="https://www.ncbi.nlm.nih.gov/pubmed/22336765" target="_blank">Deep sequencing uncovers numerous small RNAs on all four replicons of the plant pathogen <em>Agrobacterium tumefaciens</em>.</a> <em>RNA Biol</em> 9(4): 446-457. </li><li>Wingett, S. W. and Andrews, S. (2018). <a href="https://pubmed.ncbi.nlm.nih.gov/30254741" target="_blank">FastQ Screen: A tool for multi-genome mapping and quality control.</a> <em>F1000Res</em> 7: 1338.</li><li>Zhang, Y., Heidrich, N., Ampattu, B. J., Gunderson, C. W., Seifert, H. S., Schoen, C., Vogel, J. and Sontheimer, E. J. (2013). <a href="https://www.ncbi.nlm.nih.gov/pubmed/23706818" target="_blank">Processing-independent CRISPR RNAs limit natural transformation in <em>Neisseria meningitidis</em>.</a> <em>Mol Cell</em> 50(4): 488-503.<br/></li></ol>