<body>
            <h1>Novel Protein-oligonucleotide Conjugation Method Involving a High-affinity Capture HaloTag</h1>
            <p><strong>Authors:</strong> Junshi Yazaki</p>
            <h2>Abstract</h2>
            <p>Highly sensitive quantitative protein profiling can play a key role in the early diagnosis of diseases, such as autoimmune diseases and cancer. We developed a modified protein-oligonucleotide conjugation method termed HaloTag-mediated barcoding, for quantifying protein molecules at a higher sensitivity than conventional protein quantification methods. This novel and efficient conjugation method can be used to prepare HaloTag-barcoded proteins using a click chemistry-based labeling technique. Here, we describe the preparation of protein-DNA complexes and detection of protein-protein interactions which can be used in a HaloTag protein barcode assay to detect an antibody. The protocol includes procedures for preparing the ligand-oligonucleotide complex, plasmid DNA preparation for protein expression, and preparation of the protein-oligonucleotide complex. The described click reaction-based protocols simplify the conventional amine-ester reaction methods which require additional steps for chromatography purification.</p>
            <p><strong>Keywords:</strong> Protein barcoding, Protein-protein interaction, Digital quantification, Click chemistry, Antigen-antibody specificity</p>
            <h2>Background</h2><p>Protein molecules can be quantified by conventional experimental approaches such as enzyme-linked immunosorbent assay, western blotting, and mass spectrometry. These conventional quantitative protein profiling techniques involve the use of calibration curves for relative measure, without considering the high sensitivity of DNA amplification, that limits the detection of the absolute amount of proteins itself. Chemical proteomics have made possible multiplex assays in a relative quantification manner, such as tandem mass tag labelling methods coupled with mass spectrometry (Thompson <em>et al.</em>, 2003). Protein barcoding methods coupled with next-generation sequencing have emerged to identify target protein molecules; these methods include CITE-seq, Ab-seq, and LIBRA-seq for detecting antibody-antigen interactions (Shahi <em>et al.</em>, 2017; Stoeckius <em>et al.</em>, 2017, Setliff <em>et al.</em>, 2019). A high-affinity capture tag, known as the HaloTag, mediated conjugation method was developed with double stranded DNA through amide-bond coupled with high performance liquid chromatography for complex purification (Gu <em>et al.</em>, 2014). Here we describe a protocol for a novel proteinâoligonucleotide conjugation method involving the HaloTag. This method is advantageous compared to other conjugation methods because it is easy to accurately determine the number of fused protein molecules, as HaloTag-labeled proteins irreversibly bind to the small chemical ligand chloroalkane, which connects to the DNA oligonucleotide barcode in a 1:1 ratio (Yazaki <em>et al.</em>, 2020). Additionally, click chemistry-based HaloTag barcode conjugation requires less labor and time by eliminating the need for chromatography purification compared to other conjugation methods where the number and the position of tagging DNA on proteins varies (Gu <em>et al.</em>, 2014, Stoeckius <em>et al.</em>, 2017, Setliff <em>et al.</em>, 2019). This method can be adapted for additional applications, including the identification of antibody-antigen interactions for clinical diagnosis. This technology can also be adapted for single B-cell receptor sequencing to capture antigen-specific cells.</p>
        </body>

<p class="pptt" id="biaoti35830">## Materials and Reagents</p><ol class="npwtwok"><li>Strips of 8 low profile tubes (0.2 ml) with flat caps (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AB-0776 </a> ), store at RT</li><li>96-deepwell plate (Mettler Toledo, catalog number: <a class="blackNoUnder" target="_blank"> LR-P2-96P </a> )</li><li>Pure Speed PS Tips 1000/20 Î¼l IMAC 12-pk <a class="blackNoUnder" target="_blank"> PT-10-N20 </a> Ni-IMAC resin, 12 Ã 1,000-Î¼l tips (Mettler Toledo, catalog number: <a class="blackNoUnder" target="_blank"> PT-10-N20 </a> , material number: 17012572), store at 4 Â°C</li><li>5 ml conical tubes (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> Corning 352063 </a> , store at RT)</li><li>Amicon Ultra-2 Centrifugal Filter Unit (10k column; Millipore-Sigma, catalog number: <a class="blackNoUnder" target="_blank"> UFC201024 </a> ), store at RT </li><li>0.6 ml SnapLock Microtube Non-Sterile MaxyClearMaxymum Recovery (Axygen, catalog number: <a class="blackNoUnder" target="_blank"> MCT-060-V </a> )</li><li>DH5a-T1-R competent cells (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12297016 </a> ), store at <br/>-80 Â°C</li><li>LeftRight (LR) clonase enzyme mix (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11791019 </a> ), store at -80 Â°C</li><li>DMSO (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> D8418-50ML </a> ), store at -20 Â°C</li><li>Cycloalkyne-5â-modified 100-bp DNA oligonucleotides (Table 1, Eurofins, Luxembourg, <a class="blackNoUnder" target="_blank"> custom made </a> ), store at -20 Â°C</li><li>1 Î¼g/Î¼l HaloTag-Proein G (Kazusa DNA Research Institute, Chiba, Japan, ProteinG-His, store at -80 Â°C)</li><li>Plasmid template DNA for HaloTag fusion protein (<a class="blackNoUnder" target="_blank"> custom made </a> , see Procedure A and B), store at -20 Â°C</li><li>Plusgrow II for microorganism culture (Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 08202-75 </a> ), store at 4 Â°C</li><li>Super Optimal broth with Catabolite repression (SOC) medium (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15544034 </a> ), store at 4 Â°C</li><li>Glycerol (FujiFilm, catalog number: <a class="blackNoUnder" target="_blank"> 075-00616 </a> )</li><li>QIAprep Spin Miniprep Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 27104 </a> ), store at RT</li><li>Transcription and Translation (TNT) T7 coupled wheat germ extract kit for <em>in vitro</em> expression (Promega, catalog number: <a class="blackNoUnder" target="_blank"> L4140 </a> ), store at -80 Â°C</li><li>RNasin RNase Inhibitor (Promega, catalog number: <a class="blackNoUnder" target="_blank"> N2511 </a> ), store at -20 Â°C</li><li>Magne HaloTag beads (Promega, catalog number: <a class="blackNoUnder" target="_blank"> G7281 </a> ), store at 4 Â°C</li><li>5 mM Ethylenediaminetetraacetic acid (EDTA) (Dojindo, catalog number: <a class="blackNoUnder" target="_blank"> 347-07481 </a> ), store at RT</li><li>Phosphate-buffered saline (PBS) Tablets pH 7.4 (Takara, catalog number: <a class="blackNoUnder" target="_blank"> T9181 </a> ), store at RT</li><li>Tween-20 (Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 23926-35 </a> ), store at RT</li><li>PBST (PBS Tablets pH 7.4, Takara, catalog number: <a class="blackNoUnder" target="_blank"> T9181 </a> and Tween-20, Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 23926-35 </a> ), store at RT</li><li>HaloTag succinimidyl ester (O4) ligand (Promega, catalog number: <a class="blackNoUnder" target="_blank"> P6751 </a> ), store at -80 Â°C</li><li>Precision Plus Protein<sup>TM</sup> Dual Color Standards (Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1610374 </a> ), store at -20 Â°C</li><li>Mighty Amp DNA polymerase PCR (Takara, catalog number: <a class="blackNoUnder" target="_blank"> R071A </a> ), store at -20 Â°C</li><li>SPRI beads (Beckman Coulter, catalog number: <a class="blackNoUnder" target="_blank"> A63880 </a> ), store at 4 Â°C</li><li>Sodium dihydrogen phosphate dihydrate (NaH<sub>2</sub>PO<sub>4</sub>, Wako, catalog number: <a class="blackNoUnder" target="_blank"> 192-02815 </a> , store at RT) for 1 M NaH<sub>2</sub>PO<sub>4</sub> </li><li>Disodium hydrogenphosphate 12-Water (Na<sub>2</sub>HPO<sub>4</sub>, Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 31723-35 </a> , store at RT) for 1 M Na<sub>2</sub>HPO<sub>4</sub> </li><li>Imidazole (Sigma-Aldrich, catalog number: <a class="blackNoUnder" target="_blank"> I2399 </a> )</li><li>Sodium chloride (NaCl; Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9759 </a> )</li><li>Precast 5-12.5% sodium dodecyl sulfate (SDS)-polyacrylamide XV-Pantera gel (DRC, Tokyo, catalog number: <a class="blackNoUnder" target="_blank"> NXV-212 </a> ), store at 4 Â°C</li><li>GelRed nucleic acids stain (Biotium, catalog number: <a class="blackNoUnder" target="_blank"> 41002 </a> ), store at RT</li><li>HaloTag TMR ligand (Promega, catalog number: <a class="blackNoUnder" target="_blank"> G8251 </a> ), store at -20 Â°C</li><li>RNase Free Water (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10977015 </a> ), store at RT</li><li>Polyethylene glycol (PEG) 8000 (Promega, catalog number: <a class="blackNoUnder" target="_blank"> V3011 </a> ), store at RT</li><li>80% ethanol diluted with ddH<sub>2</sub>O from 100% ethanol (Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 09666-85 </a> ), store at RT</li><li>Qiagen elution buffer (buffer EB, Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 19086 </a> ), store at RT</li><li>QIAquick Gel Extraction Kit (Qiagen, catalog number: <a class="blackNoUnder" target="_blank"> 28704 </a> ), store at RT</li><li>Library Quantification Kit (Takara Bio, catalog number: <a class="blackNoUnder" target="_blank"> 638324 </a> ), store master mix at 4 Â°C, store enzyme at -20 Â°C</li><li>High Sensitivity DNA Kit (Agilent, catalog number: <a class="blackNoUnder" target="_blank"> 5067-4626 </a> ), store reagents at 4 Â°C and lab chips at RT</li><li>Ampicillin sodium salt (Nacalai Tesque, catalog number: <a class="blackNoUnder" target="_blank"> 02739-32 </a> ), store at 4 Â°C</li><li>MiSeq reagent kit v3-150 cycle (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> MS-102-3001 </a> ), store at 4 Â°C</li><li>Azido-HaloTag ligand, AzHLT-1 (<a class="blackNoUnder" target="_blank"> custom made </a> , see Recipes), store at -20 Â°C</li><li>10 Î¼M forward index primer, 10 Î¼M reverse index primer for library preparation (see Recipes and Table 2, Eurofins, catalog number: <a class="blackNoUnder" target="_blank"> custom made </a> ), store at -20 Â°C</li><li>Plasmid template DNA of HaloTag fusion protein (see Recipes)</li><li>Plusgrow II (see Recipes)</li><li>Ampicillin (see Recipes)</li><li>Azido-HaloTag ligand, AzHLT-1 (see Recipes)</li><li>In Vitro Expression (IVX) mix for one reaction (see Recipes)</li><li>Buffer A2 (see Recipes)</li><li>Buffer B2 (see Recipes)</li><li>1 M imidazole (see Recipes)</li><li>PBS buffer (see Recipes)</li><li>PBST (see Recipes)</li><li>Dilution buffer (see Recipes)</li><li>HaloTag TMR ligand (see Recipes)</li><li>50% PEG (see Recipes)</li><li>500 ÂµM cycloalkyne-modified 100-bp DNA oligonucleotides (see Recipes)</li><li>Forward and reverse index primers for library preparation (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35831">### Equipment</p><ol class="npwtwok"><li>Incubator for <em>Escherichia coli</em> culture (preset to 37 Â°C, TAITEC, model: <a class="blackNoUnder" target="_blank"> BioShaker BR-11UM </a> )</li><li>Refrigerated centrifuge (TOMY, model: <a class="blackNoUnder" target="_blank"> MX-201 </a> )</li><li>Thermal Cycler (Bio-Rad, MyCycler, catalog number: <a class="blackNoUnder" target="_blank"> 1709701EDU </a> )</li><li>Thermal Cycler for qPCR (Roche, model: <a class="blackNoUnder" target="_blank"> LightCycler 480 </a> , catalog number: 0<a class="blackNoUnder" target="_blank"> 5015278001 </a> )</li><li>NanoDrop instrument (Thermo Fisher Scientific, NanoDrop, catalog number: <a class="blackNoUnder" target="_blank"> ND1000 </a> )</li><li>Incubator for protein expression (preset to 30 Â°C, TAITEC, model: <a class="blackNoUnder" target="_blank"> Hybridization Incubator HB80 </a> )</li><li>Electrophoresis chamber (DRC, XVA Pantera system, model: <a class="blackNoUnder" target="_blank"> NXV-HM2Bset </a> )</li><li>Pure Speed PS/E4 Multi Starter Kit PT-S10-E12 (E12-1200XLS multichannel pipette and accessory kit: base, SD card, 96-deepwell plate, ColorTrak<sup>TM</sup> guide, and tip adapters, Mettler Toledo, Material number: 17013547)</li><li>Eppendorf Thermomixer rocking shaker (Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 22670506 </a> )</li><li>Magna Stand (Nippon Genetics, Fast Gene, catalog number: <a class="blackNoUnder" target="_blank"> FG-SSMAG2 </a> )</li><li>Bioanalyzer (Agilent, model: Bioanalyzer <a class="blackNoUnder" target="_blank"> 2100 </a> , catalog number: <a class="blackNoUnder" target="_blank"> 2100 </a> )</li><li>MiSeq system (Illumina, catalog number: <a class="blackNoUnder" target="_blank"> SY-410-1003 </a> )</li><li>VORTEX-GENIE 2 (Scientific Industries, catalog number: <a class="blackNoUnder" target="_blank"> 68316797 </a> )</li><li>Fluorescent Image Analyzer (FujiFilm, model: <a class="blackNoUnder" target="_blank"> FLA3000 </a> , catalog number: <a class="blackNoUnder" target="_blank"> FLA3000 </a> )<br/></li></ol><p class="pptt" id="biaoti35832">### Procedure</p><p class="ppzz">This procedure provides step-by-step instructions for HaloTag-fused protein expression, molecular barcoding of HaloTag-fused proteins, and detection of protein-protein interactions with the barcode (Figure 1).<br/><br/><img alt="" class="layerphoto" height="205" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111349_1031.png" width="300"/><br/><strong>Figure 1. Flowchart of HaloTag-barcoding assay procedures. </strong>Designations on the left indicate the amount of time expected to complete each step. Prepare HaloTag expression clones (Steps 1-13) and Ligand-DNA complex (Steps 21-22) in advance before use. Because reagents include only DNA and chemical compounds, they require no special storage condition (-20 Â°C). Prepare the HaloTag fused proteins immediately prior to use. We have not tested the shelf-life of HaloTag fused proteins (Steps 14-20) and purified barcode proteins (Steps 26-40). We recommend that the barcoded proteins be used within a few months.</p><br/><ol class="npwone"><li>Preparation of DNA clones with HaloTag fusion<ol class="npwtwo"><li>ORFs cloned into pENTR are available from the <em>Arabidopsis</em> Biological Resource Center (ABRC, <a href="https://www.arabidopsis.org/servlets/TairObject?type=stock&amp;id=3501867839" target="_blank">here</a>) for $13 USD per pENTR clone (Yazaki <em>et al.</em>, 2018). The pENTR clones are then recombined with the custom-designed HaloTag expression vector pIX-His-Halo:ccdB that was derived from pIX-Halo:ccdB (Yazaki <em>et al.</em>, 2016; available <a href="https://www.arabidopsis.org/servlets/TairObject?type=vector&amp;id=1001200298" target="_blank">here</a>). Note that pENTR clones, in the link listed above, are named pENTR-sï¬ and pENTR-topo and contain spectinomycin and kanamycin resistance genes, respectively. Recombination of the clones with the HaloTag fusion vectors is performed to construct the barcoded proteins in this procedure; however, the procedure can be applied to HaloTag-ORF fusion clones from other organisms such as humans and mice.</li><li>Reagents for a single reaction (modified from a standard Gateway cloning reaction)<br/>1 Î¼l 5x Buffer (Invitrogen, Carlsbad, CA, USA)<br/>1 Î¼l pIX vector DNA (100 ng/Î¼l)<br/>1.5 Î¼l TE buffer<br/>0.5 Î¼l LR clonase enzyme (Invitrogen)<br/>Sub total volume: 4 Î¼l per single LR recombination reaction</li><li>Pipet 4 Î¼l from Step 2 into low profile tubes (0.2 ml).</li><li>Add 1 Î¼l pENTR clone DNA (50 ng/Î¼l) to each tube<br/>Final total volume: 5 Î¼l per single LR recombination reaction</li><li>Seal and vortex briefly to mix the contents. Then centrifuge briefly to collect the sample.</li><li>Incubate the sealed tube overnight at RT.<br/><br/></li></ol></li><li>Transformation<ol class="npwtwo" start="7"><li>Thaw DH5Î±-T1-R competent cells on ice and aliquot 10 Î¼l of cells per tube (low profile tubes) using pipette.</li><li>Add 1 Î¼l of the LR recombination reaction to aliquoted competent cells (do not pipet up and down). Mix by tapping the tube several times on the bench. Incubate on ice for 30 min.</li><li>Heat shock cells at 42 Â°C for 45 s in a heat block. Place cells on ice for ~2 min.</li><li>Add 100 Î¼l SOC to each tube. SOC should be prewarmed to 37 Â°C prior to addition. Incubate at 37 Â°C for 1 h with or without shaking.</li><li>Inoculate 50 Î¼l of transformation mix from Step 10 into 3 ml of Plusgrow II with ampicillin in a 5 ml conical inoculate tube and culture overnight at 37 Â°C with shaking. </li><li>Prepare frozen glycerol stocks of the transformants. Pipette 300 Î¼l of 50% glycerol to each of empty tube. Add 700 Î¼l of overnight culture from Step 11 and mix well by pipetting up and down. Glycerol stocks should be stored at -80 Â°C.</li><li>Proceed to plasmid DNA extraction from the remaining culture using the Qiagen Miniprep Kit, following the manufacturer's recommendations.<br/><br/></li></ol></li><li>Preparation for HaloTag fused DNA and proteins<ol class="npwtwo" start="14"><li>Centrifuge the 5-ml conical inoculate tube at 1,500 <em>Ã g</em> at RT (25 Â°C) for 10 min to pellet the bacterial cells.</li><li>Decant the supernatant under a draft chamber and open the caps and dry the tube on a bench.</li><li>Purify plasmid DNA from the bacterial cell pellet using the Qiagen Miniprep Kit, following the manufacturer's recommendations.</li><li>Measure the DNA concentration using the Nanodrop. The expected concentration should be approximately 700 ng/Î¼l.</li><li>Proceed with protein expression using IVX mix. IVX is an <em>in vitro</em> protein expression system that expresses proteins from a DNA template using a coupled transcription and translation reaction without eukaryote or prokaryote cells. The HaloTag-fused query proteins are individually expressed using the TNT system (see Recipe 5 of IVX mix). The IVX mix should be prepared immediately before use.</li><li>Incubate the tube with IVX mix at 30 Â°C for 2 h.</li><li>Freeze and store at -80 Â°C<br/><br/></li></ol></li><li>Preparation of ligand-DNA complex and barcoded HaloTag-fused proteins<ol class="npwtwo" start="21"><li>Prepare 500 Î¼M barcode DNA oligonucleotide with 5â² modification by dibenzocyclooctyne (DBCO) in RNase-free water (Table 1 and Recipe 14). Prepare 500 Î¼M azido-HaloTag ligand, AzHLT-1 (Recipe 4).<br/><br/><strong>Table 1. Cycloalkyne-modified 100-bp DNA oligonucleotides. </strong>This table has been adapted from Yazaki <em>et al.</em> (2020). The Barcode DNA are synthesized by Eurofin as single stranded DNA with the 5â²-modified by dibenzocyclooctyne (DBCO) and the 3â²-modified tetramethylrhodamine (TMR).<br/><img alt="" class="layerphoto" height="950" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111415_4332.png" width="900"/><br/><br/></li><li>Mix 2 Î¼l of 500 Î¼M DNA (Table 1) and 0.5 Î¼l of 500 Î¼M azido-HaloTag ligand AzHLT-1 (4:1) and incubate at RT for 1 h (click reaction). The final concentrations of the mixture ingredients are 400 Î¼M DNA and 100 Î¼M HaloTag ligand.</li><li>Combine 100 Î¼l of HaloTag fused proteins obtained in Step 20 with 1 Î¼l of DNA-ligand mixture from Step 22 at RT. Separately, combine 1 Î¼g of HaloTag protein G with 1 Î¼l of DNA-ligand mix for conjugation control.</li><li>Incubate at RT for 1 h.</li><li>Confirm DNA-protein conjugation by SDS-PAGE using 5-12.5% gradient gels (Figure 2). The size of the conjugated products should be higher than that of non-barcoded proteins. DNA conjugated proteins (10 Î¼l; ~10% of total amount of reaction 101 Î¼l) were loaded. Non-barcoded proteins (2.5 Î¼l: 2.5% of input) were loaded as a negative control (lanes B1-B5, B6-B10) to indicate the original protein size. The results of HaloTag protein staining using TMR-modified DNA barcode are shown in Figure 2 (lanes A1-A5, A6-A10). The size of the protein-DNA conjugation shows a higher molecular weight in Figure 2 (lanes A1-A5, A6-A10) compared to HaloTag proteins with no conjugated DNA barcode according to TMR ligand staining in Figure 2 (lanes B1-B5, B6-B10). Stained proteins were visualized based on the fluorescence of TMR at 532 nm using a Fluorescent Image Analyzer (FujiFilm, <a class="blackNoUnder" target="_blank"> FLA3000 </a> ).<br/><br/><img alt="" class="layerphoto" height="65" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111437_6669.png" width="300"/><br/><strong>Figure 2. Protein-DNA Conjugation results processing crude proteins expressed <em>in vitro</em>.</strong> Lanes A1-A10, HaloTag-protein conjugated with DNA was detected using dibenzocyclooctyne-TMR staining and DNA oligonucleotide barcode conjugated with TMR. Lanes B1-B10, HaloTag-protein was detected using TMR ligand staining. Lane L, molecular ladder. All proteins were mixed with a DNA barcode and the azido-ligand AzHLT-1 at a 1:1:2 ratio (mol/mol/mol; lanes A1-A10) and incubated at RT for 1 h. Proteins: 1, At1g32640 (68 kDa); 2, At2g36270 (47 kDa); 3, At3g08500 (38 kDa); 4, At4g17870 (22 kDa); 5, At1g71930 (37 kDa); 6, At3g62420 (17 kDa); 7, At5g28770 (34 kDa); 8, At5g62380 (40 kDa); 9, At1g12260 (46 kDa); 10, At2G40330 (24 kDa). Protein sizes are specified without the HaloTag (33 kDa). Protein identification numbers correspond to previously assigned <em>Arabidopsis thaliana</em> gene identification numbers (Yazaki <em>et al.</em>, 2016) (<a href="https://www.arabidopsis.org/" target="_blank">https://www.arabidopsis.org/</a>). Right panel (proteins 6-10) was modified from Supplemental Figure S3 of Yazaki <em>et al.</em>, 2020.<br/><br/></li><li>Purify DNA-HaloTag fusion proteins (barcoded proteins) from Step 24 using a histidine tag purification kit (Pure Speed PS/E4 Multi Starter Kit and 1000/20 Î¼l IMAC tips) at RT (Figure 3). Set the ColorTrack Guide under the platform with the deep-well plate aligned above (Figure 3A). Place the deep-well plate on the platform and slide it into position along the rails of the platform of the Pure Speed PS/E4 Multi Starter Kit (Figure 3B).</li><li>For 10 barcoded protein purifications, use the E12-1200XLS multichannel pipette to add 500 Î¼L buffer A2 (Recipe 7) to wells A2-A11 in a 96-deepwell plate for column equilibration (Equilibration rows in Figure 3A, 3B). Add 1,100 Î¼l buffer A2 to wells B2-B11 to capture the sample (Capture rows in Figure 3A, 3B). Add 500 Î¼l buffer A2 to wells C2-C11 to wash the sample (1<sup>st</sup> wash rows in Figure 3A, 3B). Add 500 Î¼l buffer A2 to wells D2-D11 to wash the sample (2<sup>nd</sup> wash rows in Figure 3A, 3B). Add ~100 Î¼l buffer B2 to wells E2-E11 to elute the purified sample (Elution rows in Figure 3A, 3B). After preparing the plate, add all barcoded protein sample (101 Î¼l or rest of all samples) from Step 24 to wells B2-B11 immediately before starting purification (Capture rows in Figure 3A, 3B).</li><li>Load the PureSpeed resin tips (PS Tips 1000/20 Î¼l IMAC) to channel numbers 2-11 on the E12-1200XLS multichannel pipette (Figure 3C).</li><li>Load the two adapter tips (blue adapter for PS Tips 1000/20 Î¼l IMAC) to channels number 1 and 12, which is next to the outer resin tip on the E12-1200XLS multichannel pipette to secure the pipette assembly (Figure 3C).</li><li>Place the E12-1200XLS multichannel pipette with 12 loaded tips in wells A1-12 (equilibration rows) of the 96-deep-well plate for column equilibration (Figure 3D).</li><li>Choose the default program for IMAC 1000/20 Î¼l on the E12-1200XLS multichannel pipette. The program for histidine tag purification contains resin equilibration (Figure 3D), capture (Figure 3E), wash (Figures 3F, 3G), and elution of the sequence (Figure 3H).</li><li>Press the CONT key on the E12-1200XLS multichannel pipette to start the purification process and follow the instructions given on the screen of the multichannel pipette. This step allows for the elimination of excess amounts of barcode DNA, ligand, and barcode-ligand complex which are not conjugated to the proteins.<br/><br/><img alt="" class="layerphoto" height="159" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111523_6511.png" width="300"/><br/><strong>Figure 3. Purification of the DNA-HaloTag fusion proteins.</strong> A. Set the ColorTrack Guide under the platform with the deepwell plate aligned above. B. Place the deepwell plate on the platform and slide it into position along the rails of the platform. C. Load the PureSpeed resin tips to channel numbers 2-11 on the pipette. Load the two adapter tips (blue adapter) to channels number 1 and 12. D. Place the pipette with 12 loaded tips in wells A1-12 (equilibration rows) of the 96-deepwell plate for column equilibration. Choose the default program for IMAC 1000/20 Î¼l on the multichannel pipette. The program for histidine tag purification contains the sequence. E. Capture in wells B1-12. F. First wash in wells C1-12. G. Second wash in wells D1-12. H. Elution of the sample in wells E1-12.<br/><br/></li><li>Procedure for dialysis. Insert the Amicon Ultra-2 10k column device into the filtrate collection tube.</li><li>Pipet all purified barcoded proteins into the Amicon-ultra 10k column and add 1x PBS up to 2 ml in the column. Cover the column with the collection tube.</li><li>Place the filtrate collection tube, covered with the 10k column, into the centrifuge rotor. Orient the membrane panel in the column to the center of the rotor (one panel facing up and the other panel facing down). Confirm that the device composed of the filtrate collection, filter column, and collection tube are seated on the bottom of the rotor. Ensure that the rim of the collection tube is completely inside the rotor walls. Ensure that the rotor is balanced with a second column device placed directly opposite of the first column.</li><li>Centrifuge at 4,000 <em>Ã g</em> at 4 Â°C for 20-30 min.<br/></li><li>Remove the filtrate collection tube. Invert the filter column and collection tube.</li><li>Place the collection tube with filter column in the centrifuge. Ensure that the rotor is balanced by a second column device placed directly opposite of the first column.</li><li>Centrifuge at 1,000 <em>Ã g</em> at 4 Â°C for 2 min to transfer the concentrated and dialyzed proteins from the filter column to the collection tube. The final concentrate volume is to be 30-60 Î¼l. The protein levels produced by TNT and conjugation ratios are constant for most ORFs (Yazaki <em>et al.</em>, 2016 and 2020 and Figure 2). Therefore, it is not necessary to check all purified conjugations using SDS-PAGE.</li><li>Freeze and store the sample at -80 Â°C (not necessary to Snap-freeze).<br/><br/></li></ol></li><li>Detection of protein-protein interaction with barcoded proteins<ol class="npwtwo" start="41"><li>Pipette 5 Î¼l HaloTag Magna beads into a 200-Î¼l PCR tube. Wash the beads three times with 1x PBST on the Magna Stand. Add 10 Î¼l of PBST to the washed beads.</li><li>Add 9 Î¼l of bait protein from Step 20 to tubes containing 10 Î¼l of beads in a total volume of 19 Î¼l. Shake on an Eppendorf Thermomixer rocking shaker at 1,000 rpm at 6 Â°C for 1 h.</li><li>Wash beads three times with 80 Î¼l PBST on a Magna Stand. Add 10 Î¼l of PBST to the washed bait beads.</li><li>Add 5 Î¼l of query barcoded protein from Step 39 to tubes containing 10 Î¼l of bait beads in a total volume of 15 Î¼l. Shake on an Eppendorf Thermomixer rocking shaker at 1,000 rpm at 6 Â°C for 1 h.</li><li>Wash the beads three times with PBST on the Magna Stand. Add 10 Î¼l of PBST to the washed bait beads.</li><li>Freeze and store the samples at -80 Â°C.<br/><br/></li></ol></li><li>Library preparation for detection of protein-protein interactions (PPIs)<ol class="npwtwo" start="47"><li>Perform quantitative PCR (qPCR) with the beads included in the reaction to determine the concentration of the sample. Use the following primer pair: <br/>Primer 1: 5â²-ACACTCTTTCCCTACACGACGCTCTTCCGATCT-3â²<br/>Primer 2: 5â²-GTGACTGGAGTTCAGACGTGTGCTCTTCCGATCT-3â²<br/>Primers for qPCR are designed for the following conditions: 5 min at 95 Â°C, 35 cycles of 30 s at 95 Â°C, and 45 s at 60 Â°C. Hold the sample at 4 Â°C. Calculate the sample concentration based on the standard curve with 200 pM, 20 pM, 2 pM, and 0.2 pM of Oligo01 (Table 1) without TMR modification.</li><li>Vortex the sample from Step 46. Prepare 12,000 molecules per 2 Î¼l of sample with beads based on the qPCR results in Step 47 using the dilution buffer (Recipe 11). <em>i.e.</em>, to obtain 12,000 molecules, follow the dilution condition below;<br/><ol class="npwthree"><li>Prepare 1 pM of barcoded proteins based on the results of Step 47.<br/></li><li>Next, dilute 1 pM sample from Procedure A by 100-fold to prepare 0.01 pM of the proteins.<br/></li><li>Pipette 2 Î¼l of 0.01 pM of the proteins as 12,000 molecules.</li></ol></li><li>Prepare the PCR master mix for all sample reactions. Next, aliquot the reaction mix into each tube.<br/>For one reaction;<br/>12.5 Î¼l 2Ã MightyAmp buffer<br/>0.5 Î¼l MightyAmp DNA polymerase<br/>8.5 Î¼l RNase-free water<br/>Pipet 21.5 Î¼l in to each tube.</li><li>Add 0.75 Î¼l of 10 Î¼M forward index primer and 0.75 Î¼l of 10 Î¼M reverse index primer in Table 2 to each tube.<br/><br/><strong>Table 2. Primer sequences for protein indexing and counting library preparation. Index sequences are underlined.</strong> This table has been adapted from Yazaki <em>et al.</em> (2020).<br/><img alt="" class="layerphoto" height="997" img="" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111613_8691.png" width="820"/><br/><br/></li><li>Add 12,000 molecules/2 Î¼l of sample with beads from Step 49. Vortex and spin down by centrifugation at 100 <em>x g</em> for 5 s.</li><li>Place the tube in the Bio-Rad thermal cycler and run the following parameters:<br/>98 Â°C, 2 min<br/>:1 cycle<br/>98 Â°C, 10 s<br/>60 Â°C, 10 s<br/>68 Â°C, 1 min <br/>:4 cycles<br/>98 Â°C, 10 s<br/>60 Â°C, 2 s<br/>68 Â°C, 1 min<br/>68 Â°C, 5 min <br/>:25 cycles<br/>Hold at 4 Â°C when the cycles are complete</li><li>Prepare AMPure XP beads 30 min before finishing Step 52.</li><li>Prepare PEG/NaCl solution:<br/>44 Î¼l 50% PEG<br/>75 Î¼l 5 M NaCl<br/>1 Î¼l RNase-free water<br/>Final 10% PEG, 1.7 M NaCl</li><li>Pipet 25 Î¼l of RNase-free water into 1.5-ml tubes.</li><li>Add all of the sample from Step 52 to the 1.5-ml tube (total volume 50 Î¼l).</li><li>Add 120 Î¼l of PEG/NaCl solution to the sample from Step 56 (total volume 170 Î¼l).</li><li>Vortex AMPure beads and add 50 Î¼l of AMPure beads to the tube from Step 58 (total volume 220 Î¼l). Vortex, spin down at 100 <em>Ã g</em> for 5 s, and incubate for 5 min at RT.</li><li>Set the tube on the MagnaStand and wait 5 min.</li><li>Discard the supernatant by pipetting and add 250 Î¼l of 80% ethanol.</li><li>Discard the 80% ethanol by pipetting and add 250 Î¼l of 80% ethanol.</li><li>Discard the 80% ethanol by pipetting and let the tube dry.</li><li>Remove the MagnaStand and pipet 23 Î¼l of Qiagen elution buffer (buffer EB) into each tube. Incubate 1 min at RT.</li><li>Set the tube on the MagnaStand and incubate 2 min at RT.</li><li>Pipet 20 Î¼l of the sample from Step 64 and transfer into new tubes.</li><li>Check the sample quality using the Bioanalyzer high sensitivity chip mode (Figure 4A). The size of the amplified products should be 174 base pairs.<br/><em>Note: PCR primers with the forward and reverse index can be used for PCR amplification with Mighty Amp. Non-specific amplification may occur based on the combination of primers and amounts of template DNA used (Figure 4B black arrows). To avoid nonspecific amplification, we recommend changing the primer combination and template DNA, as non-specific amplification products may be obtained rather than the 174-bp product. Non-specific amplification in Figure 4B was observed with a primer combination of Foward6-Reverse3 in Table 2. To avoid this, pairs of primer Foward8-Reverse1 in Table 2 showed specific amplification (Figure 4A black arrows).</em><br/><br/><img alt="" class="layerphoto" height="275" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111720_1817.png" width="300"/><br/><strong>Figure 4. Barcode DNA amplification results from library preparation steps.</strong> A. PCR amplification yields consistent results as 174-bp products (arrow). B. Anticipated results from nonspecific PCR amplification. Improper pairs of forward and reverse combination of primers lead to unexpected results in amplification (arrows). We recommend changing the pairs of indexed primer and template DNA when unexpected amplification results are observed. Please see the notes section for our recommended changes to the primer pairs to avoid unexpected amplification products. L: molecular ladder.<br/><br/></li><li>Prepare &gt; 2 nM sample mixture (if any multiple pairs of PPI) based on the molarity concentration obtained using the Bioanalyzer (pmol/L).<br/></li><li>Purify the mixture of samples from Step 67 using the GelRed nucleic acid gel stain and agarose gel extraction kit. Excise the DNA band of the target size (174 bp) from the agarose gel. Extract the DNA from the gel using the QIAquick Gel Extraction Kit following the manufacturer's recommendations.</li><li>Quantify the sample from Step 68 by qPCR. The expected concentration is approximately 8 nM (1-20 nM) on average.<br/>Prepare the premix for PCR.<br/>For one reaction<br/>1 Î¼l of sample<br/>2 Î¼l RNase-free water<br/>2 Î¼l 5x primer mix<br/>5 Î¼l Terra PCR Direct SYBR Premix</li><li>Place the tube in the <a class="blackNoUnder" target="_blank"> LightCycler 480 </a> (Roche) and run the following parameters:<br/>98 Â°C, 2 min<br/>:1 cycle<br/>98 Â°C, 10 s<br/>60 Â°C, 15 s<br/>68 Â°C, 45 s<br/>:35 cycles<br/>Hold the sample at 40 Â°C</li><li>Prepare 10 Î¼l of 2 nM sample mixture from Step 68.</li><li>Progress to sequencing. Run the MiSeq system in 76 base pair-end mode with the MiSeq reagent Kit v3-150cycle. For sequencing, denature 10 Î¼l of 2 nM sample DNA mixture from Step 71 with 10 Î¼l of 0.2 N NaOH at RT for 5 min.</li><li>Add 980 Î¼l prechilled HT1 to the tube containing 1 nM denatured library pool to prepare a 20 pM denatured library.</li><li>Use prechilled HT1 from to dilute the denatured 20 pM library solution to 7 pM at a final volume of 600 Î¼l.</li><li>Load the prepared library onto the MiSeq cartridge.</li><li>Run MiSeq. The sample library which is amplified 174-bp DNA, including the Illumina adapter and index sequence, were applied to the MiSeq system. The running conditions were dual-indexed sequencing in a paired-end flow cell, to read 76 bp, including the amplified 38 bp of the barcode sequences. Representative results of MiSeq running are shown in Figure 5. This protocol is also suitable for other platform such HiSeq series in Illumina.<br/><br/><img alt="" class="layerphoto" height="146" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200916/20200916111740_5096.png" width="300"/><br/><strong>Figure 5. Overview of a MiSeq run using the Sequencing Analysis Viewer (SAV).</strong> Representative results of a MiSeq run for a library of protein-protein interactions using barcoded proteins. SAV is an application that allows one to view quality metrics of the sequencing run (details available <a href="https://jp.support.illumina.com/content/dam/illumina-support/documents/documentation/software_documentation/sav/sequencing-analysis-viewer-user-guide-15020619-f.pdf" target="_blank">here</a>). The Flow Cell Chart (left) shows a color matrix of the data quality at each swath (section of flow cell). The flow cell for MiSeq is a single lane flow cell. The Data by Cycle (top middle) shows plots of signal intensities, Q30 score, and base percentages, which can indicate the quality of the sequence data by each cycle. The Data By Lane (bottom middle) shows the density of clusters passing the filter as a percentage of raw cluster density. QScore Distribution (top right) shows the percentage of reads with a QScore. A QScore â¥ 30, the percentage of bases with a quality score of 30 or higher, is positively correlated to the base call quality. A higher QScore indicates a smaller probability of error. A quality score of 30 represents an error rate of 1 in 1,000, with a corresponding call accuracy of 99.9%. A detailed explanation for QScore can be found <a href="https://jp.illumina.com/science/technology/next-generation-sequencing/plan-experiments/quality-scores.html" target="_blank">here</a>.<br/></li></ol></li></ol><p class="pptt" id="biaoti35833">### Recipes</p><p class="ppzz">For Reagents and Solutions</p><ol class="npwtwok"><li>Plasmid template DNA of HaloTag fusion protein<br/>ORF entry clones are available from <em>Arabidopsis</em> Biological Resource Center (ABRC, available <a href="https://www.arabidopsis.org/servlets/TairObject?type=stock&amp;id=3501867839" target="_blank">here</a>) at $13 USD per pENTR clone.</li><li>Plusgrow II<br/>a.Add 40 g Plusgrow II per liter of distilled water<br/>b.Autoclave for 15 min<br/>c.Add 1 ml of 100 mg/ml ampicillin after autoclaving when its 50 Â°C</li><li>Ampicillin<br/>Add 1,000 mg ampicillin sodium salt per 10 ml of distilled water (100 mg/ml)</li><li>Azido-HaloTag ligand, AzHLT-1<br/>Prepare 500 Î¼M ligand in a final concentration of 10 mM DMSO<br/>Ligands are available upon request from Dr Takamitsu Hosoya, Tokyo Medical and Dental University, Japan (<a href="mailto:thosoya.cb@tmd.ac.jp" target="_blank">thosoya.cb@tmd.ac.jp</a>)</li><li>IVX mix for one reaction using Transcription and Translation (TNT) T7 Coupled Wheat Germ Extract System<br/>50 Î¼l wheat germ (thaw before use from -80 Â°C)<br/>4 Î¼l TNT buffer<br/>6 Î¼l minus Met amino acid mix<br/>6 Î¼l minus Leu/Cys amino acid mix<br/>2 Î¼l RNase inhibitor<br/>2 Î¼l T7 polymerase<br/>2 Î¼g HaloTag plasmid template DNA<br/>Bring volume up to 100 Î¼l with DEPC water</li><li>Buffer A2<br/><ol class="npwthree"><li>Prepare 1 M of NaHPO<sub>4</sub> (sodium hydrogen phosphate), 1 M of NaH<sub>2</sub>PO<sub>4</sub> (sodium dihydrogen phosphate, 5 M NaCl (sodium chrolide), and 2 M imidazole<br/></li><li>Mix 1.935 ml of 1 M of Na<sub>2</sub>HPO<sub>4</sub>, 0.565 ml of 1 M of NaH<sub>2</sub>PO<sub>4</sub>, 5 ml of 5 M NaCl, and 0.75 ml of 2 M imidazole<br/></li><li>Add dH<sub>2</sub>O to bring the mixture to a total volume of 50 ml (approximately 41.75 ml of dH<sub>2</sub>O will be added)</li></ol></li><li>Buffer B2<br/><ol class="npwthree"><li>Mix 1.935 ml of 1 M of Na<sub>2</sub>HPO<sub>4</sub>, 0.565 ml of 1 M of NaH<sub>2</sub>PO<sub>4</sub>, 5 ml of 5 M NaCl, and 12.5 ml of 2 M imidazole<br/></li><li>Add dH<sub>2</sub>O to bring the mixture to a total volume of 50 ml (approximately 30 ml of dH<sub>2</sub>O will be added)</li></ol></li><li>1 M imidazole<br/>Dissolve 3.4 g of imidazole in distilled water to 50 ml<br/>Store at RT</li><li>PBS buffer<br/>Dissolve 1 tablet from PBS tablets pH 7.4 (Takara) in distilled water to 1,000 ml<br/>Store at RT</li><li>PBST<br/>Dissolve 1 tablet of PBS tablets pH 7.4 (Takara) in distilled water to 1,000 ml with 0.1% (w/v) Tween-20<br/>Store at RT</li><li>Dilution buffer<br/>10 ml of RNase-free water plus 10 Î¼l of 10% Tween 20</li><li>HaloTag TMR ligand<br/>Dilute 1:100 in DMSO (<em>i.e.</em>, for staining HaloTagged protein, add 1 Î¼l TMR ligand after diluting by DMSO to 5-10 Î¼l of HalTag fused proteins from Step 20)</li><li>50% PEG<br/><ol class="npwthree"><li>Add 10 g PEG8000 per 20 ml of distilled water<br/></li><li>Autoclave 15 min</li></ol></li><li>500 ÂµM cycloalkyne-modified 100-bp DNA oligonucleotides<br/>The Barcode DNA is listed in Table 1. These are synthesized by Eurofins as single-stranded DNA 5â²-modified by dibenzocyclooctyne (DBCO) and 3â²-modified tetramethylrhodamine (TMR). Add RNase-free water to prepare 500 ÂµM. Store at -20 Â°C.</li><li>Forward and reverse index primers for library preparation<br/>The primer sequences are listed in Table 2 and synthesized as single-stranded DNA by Eurofins<br/></li></ol><p class="pptt" id="biaoti35834">### Acknowledgments</p><p class="ppzz">This protocol has been derived and adapted from Yazaki <em>et al.</em> (2020). I thank Atsuo Kobayashi and Yusuke Kawashima for technical help. This work was supported by the Sumitomo Foundation Grant for Basic Science Research Project [140384 to J.Y.] and Takeda Science Foundation, Visionary Research Grant to J.Y. This work was also supported partially by the Ministry of Education, Science, Sports and Culture, Grant-in-Aid for Scientific Research (C), 2019-2021 [19K0574 to J.Y.]. I would also like to thank Editage (www.editage.jp) for English language editing.</p><p class="pptt" id="biaoti35835">### Competing interests</p><p class="ppzz">The authors declare no conflict of interest.</p><p class="pptt" id="biaoti35836">### References</p><ol class="npwtwok"><li>Gu, L., Li, C., Aach, J., Hill, D. E., Vidal, M. and Church, G. M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25252978" target="_blank">Multiplex single-molecule interaction profiling of DNA-barcoded proteins.</a> <em>Nature</em> 515(7528): 554-557.</li><li>Setliff, I., Shiakolas, A. R., Pilewski, K. A., Murji, A. A., Mapengo, R. E., Janowska, K., Richardson, S., Oosthuysen, C., Raju, N., Ronsard, L., Kanekiyo, M., Qin, J. S., Kramer, K. J., Greenplate, A. R., McDonnell, W. J., Graham, B. S., Connors, M., Lingwood, D., Acharya, P., Morris, L. and Georgiev, I. S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31787378" target="_blank">High-throughput mapping of B cell receptor sequences to antigen specificity.</a> <em>Cell</em> 179(7): 1636-1646 e1615.</li><li>Shahi, P., Kim, S. C., Haliburton, J. R., Gartner, Z. J. and Abate, A. R. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28290550" target="_blank">Abseq: Ultrahigh-throughput single cell protein profiling with droplet microfluidic barcoding.</a> <em>Sci Rep</em> 7: 44447.</li><li>Stoeckius, M., Hafemeister, C., Stephenson, W., Houck-Loomis, B., Chattopadhyay, P. K., Swerdlow, H., Satija, R. and Smibert, P. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28759029" target="_blank">Simultaneous epitope and transcriptome measurement in single cells.</a> <em>Nat Methods</em> 14(9): 865-868.</li><li>Thompson, A., SchÃ¤fer, J., Kuhn, K., Kienle, S., Schwarz, J., Schmidt, G., Neumann, T., Johnstone, R., Mohammed, A. K., Hamon, C. (2003). <a href="https://pubmed.ncbi.nlm.nih.gov/12713048/" target="_blank">Tandem mass tags: a novel quantification strategy for comparative analysis of complex protein mixtures by MS/MS.</a> <em>Analytical Chemistry</em> 75(18): 4942-4942.</li><li>Yazaki, J., Galli, M., Kim, A. Y., Nito, K., Aleman, F., Chang, K. N., Carvunis, A. R., Quan, R., Nguyen, H., Song, L., Alvarez, J. M., Huang, S. S., Chen, H., Ramachandran, N., Altmann, S., Gutierrez, R. A., Hill, D. E., Schroeder, J. I., Chory, J., LaBaer, J., Vidal, M., Braun, P. and Ecker, J. R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27357687" target="_blank">Mapping transcription factor interactome networks using HaloTag protein arrays.</a> <em>Proc Natl Acad Sci U S A</em> 113(29): E4238-4247.</li><li>Yazaki, J., Galli, M., Kim, A. Y. and Ecker, J. R. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30106517" target="_blank">Profiling interactome networks with the HaloTag-NAPPA <em>in situ</em> protein array.</a> <em>Curr Protoc Plant Biol</em> 3(3): e20071.</li><li>Yazaki, J., Kawashima, Y., Ogawa, T., Kobayashi, A., Okoshi, M., Watanabe, T., Yoshida, S., Kii, I., Egami, S., Amagai, M., Hosoya, T., Shiroguchi, K. and Ohara, O. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31752022" target="_blank">HaloTag-based conjugation of proteins to barcoding-oligonucleotides.</a> <em>Nucleic Acids Res</em> 48(2): e8.<br/></li></ol>