<body>
            <h1><em>In vitro</em> AMPylation/Adenylylation of Alpha-synuclein by HYPE/FICD</h1>
            <p><strong>Authors:</strong> Ali Camara, Anwesha  Sanyal, Sayan Dutta, Jean-Christophe Rochet, Seema Mattoo</p>
            <h2>Abstract</h2>
            <p>One of the major histopathological hallmarks of Parkinsonâs disease are Lewy bodies (LBs) âcytoplasmic inclusions, enriched with fibrillar forms of the presynaptic protein alpha-synuclein (Î±-syn). Progressive deposition of Î±-syn into LBs is enabled by its propensity to fibrillize into insoluble aggregates. We recently described a marked reduction in Î±-syn fibrillation <em>in vitro</em> upon posttranslational modification (PTM) by the Fic (Filamentation induced by cAMP) family adenylyltransferase HYPE/FICD (Huntingtin yeast-interacting protein E/FICD). Specifically, HYPE utilizes ATP to covalently decorate key threonine residues in Î±-synâs N-terminal and NAC (non-amyloid-Î² component) regions with AMP (adenosine monophosphate), in a PTM termed AMPylation or adenylylation. Status quo <em>in vitro</em> AMPylation reactions of HYPE substrates, such as Î±-syn, use a variety of ATP analogs, including radiolabeled Î±-<sup>32</sup>P-ATP or Î±-<sup>33</sup>P-ATP, fluorescent ATP analogs, biotinylated-ATP analogs (N6-[6-hexamethyl]-ATP-Biotin), as well as click-chemistry-based alkyl-ATP methods for gel-based detection of AMPylation. Current literature describing a step-by-step protocol of HYPE-mediated AMPylation relies on an Î±-<sup>33</sup>P-ATP nucleotide instead of the more commonly available Î±-<sup>32</sup>P-ATP. Though effective, this former procedure requires a lengthy and hazardous DMSO-PPO (dimethyl sulfoxide-polyphenyloxazole) precipitation. Thus, we provide a streamlined alternative to the Î±-<sup>33</sup>P-ATP-based method, which obviates the DMSO-PPO precipitation step. Described here is a detailed procedure for HYPE mediated AMPylation of Î±-syn using Î±-<sup>32</sup>P-ATP as a nucleotide source. Moreover, our use of a reusable Phosphor screen for AMPylation detection, in lieu of the standard, single-use autoradiography film, provides a faster, more sensitive and cost-effective alternative.</p>
            <p><strong>Keywords:</strong> AMPylation, Adenylylation, HYPE, FICD, Alpha-synuclein, Post-translational modification</p>
            <h2>Background</h2><p>Deposition of the small (140 amino acids), intrinsically disordered protein alpha-synuclein (Î±-syn) into Lewy bodies (LBs) is a major hallmark of Parkinsonâs disease (PD) (Weinreb <em>et al.</em>, 1996; Spillantini <em>et al.</em>, 1997). On its way to forming the insoluble aggregates found in LBs, monomeric Î±-syn adopts a Î²-sheet-rich, amyloid-like conformation, leading to progressive fibrillation and concomitant insolubility (Iwai <em>et al.</em>, 1995). Multiple lines of research describe a compelling correlation between post-translational modifications (PTMs) of Î±-syn and its oligomeric state. Phosphorylation at Ser 129, for example, is prevalent on aggregated Î±-syn found within LBs (Chen and Feany, 2005). Tyrosine nitrations of Î±-syn have been linked to formation of stable oligomers and reduced fibril formation (Yamin <em>et al.</em>, 2003), while O-GlcNAcylation of threonine residues reduces Î±-syn aggregation (Levine <em>et al.</em>, 2019). Recently, we described a novel Î±-syn modificationâadenylylation or AMPylationâwhich reduces Î±-syn fibrillation <em>in vitro</em> (Sanyal <em>et al.</em>, 2019). Moreover, AMPylated Î±-syn displayed diminished disease-linked phenotypes, such as membrane permeabilization (Ysselstein <em>et al.</em>, 2017; Sanyal <em>et al.</em>, 2019).<br/><br/>We demonstrated that HYPE, the sole Fic protein in humans, carried out AMPylation of Î±-syn (Worby <em>et al.</em>, 2009; Sanyal <em>et al.</em>, 2019). Like other members of the conserved Fic domain protein family, HYPEâs AMPylation activity is intrinsically regulated by an inhibitory helix called Î±<sub>inh</sub> (Engel <em>et al.</em>, 2012). Mutation of HYPEâs conserved Glu234 to Gly within the Î±<sub>inh</sub> renders HYPE constitutively active for its adenylyltransferase activity (Engel <em>et al.</em>, 2012; Sanyal <em>et al.</em>, 2015). HYPE then covalently attaches the AMP portion of an ATP co-substrate onto target hydroxyl side-chain residues (<em>e.g.</em>, serine or threonine) (Ham <em>et al.</em>, 2014; Sanyal <em>et al.</em>, 2015; Truttmann <em>et al.</em>, 2016; Preissler <em>et al.</em>, 2017). Indeed, tandem mass spectrometric analysis of purified, recombinant Î±-syn mapped AMPylation sites to three threoninesâT33, T54, and T75âlocated within regions hypothesized to be crucial for oligomerization (Sanyal <em>et al.</em>, 2019).<br/><br/>Several biochemical assays monitoring Fic-mediated AMPylation have been reported (Worby <em>et al.</em>, 2009; Mattoo <em>et al.</em>, 2011; Yarbrough <em>et al.</em>, 2009; Lewallen <em>et al.</em>, 2012; Truttmann <em>et al.</em>, 2016; Dedic <em>et al.</em>, 2016). Most common amongst these are assays using a fluorescent (Fl-ATP) or radioactive (Î±-<sup>32</sup>P-ATP or Î±-<sup>33</sup>P-ATP) ATP analogs. While Fl-ATP has served as a useful research tool, its large, bulky fluorescent moiety is known to stunt kinetic efficiencies (Lewallen <em>et al.</em>, 2012). Moreover, no step-by-step protocol for Fl-ATP AMPylation has been documented. A detailed procedure for Î±-<sup>33</sup>P-ATP, however, is available (Truttmann and Ploegh, 2017). This method entails incubation of HYPE and its protein target with radiolabelled Î±-<sup>33</sup>P-ATP, followed by separation via SDS-PAGE, DMSO-PPO precipitation, and then imaging of radiolabeled proteins on autoradiography film. Though effective in detecting AMPylation, this procedure suffers from additional time-consuming steps that are expensive and require hazardous chemicals (<em>e.g.</em>, DMSO-PPO and autoradiography exposure and development). Herein, we present an optimized method for monitoring AMPylation <em>in vitro</em>, using Î±-syn as a target and Î±-<sup>32</sup>P-ATP as a nucleotide source. Î±-<sup>32</sup>P-ATP preserves signal sensitivity, while eliminating DMSO-PPO precipitation. Our protocol also replaces the expensive single-use autoradiography film with a reusable Phosphor screen, which allows more sensitive detection of radioactive phosphate-containing proteins under ambient light and at shorter exposure times. Given the broad specificities of Fic proteins and other enzymes for co-substrates including GTP, CTP, phosphocholine, and even ATP used for phosphorylation, this protocol can be adapted to assess other PTMs <em>in vitro</em> (Mattoo <em>et al.</em>, 2011; Veyron <em>et al.</em>, 2018).</p>
        </body>

<p class="pptt" id="biaoti35666">## Materials and Reagents</p><ol class="npwtwok"><li>Kimwipes (Kimberly-Clark Professional, KimTech Science Brand, catalog number: <a class="blackNoUnder" target="_blank"> 34155 </a> )</li><li>Set of micropipettes, with volumes ranging from 0.5 to 1,000 Î¼l (generic)</li><li>Appropriately sized sterile micropipette tips (generic)</li><li>2 ml pipette bulbs (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 82024-554 </a> )</li><li>Pasteur pipette (VWR, catalog number: <a class="blackNoUnder" target="_blank"> 14672-380 </a> )</li><li>Glass thermometer, 20-100 Â°C range (generic)</li><li>1.5 ml tubes (Eppendorf Safe-lock microcentrifuge tubes, catalog number: 0<a class="blackNoUnder" target="_blank"> 22363204 </a> )</li><li>Gel loading pipet tips (Fisher Scientific, Fisher Brand, catalog number: <a class="blackNoUnder" target="_blank"> 02-707-181 </a> )</li><li>MTC Stop-Pop locking tubes clips (MTC Bio, catalog number: <a class="blackNoUnder" target="_blank"> C2086 </a> )</li><li>Bacterially purified HYPE<sub>102-456</sub> (WT, E234G, and E234G/H363A) (Sanyal <em>et al.</em>, 2019)</li><li>Bacterially purified full-length Î±-syn (Sanyal <em>et al.</em>, 2019)</li><li>ddH<sub>2</sub>O (ELGA LabWater, PURELAB Flex 2 Dispenser, model number: <a class="blackNoUnder" target="_blank"> PF2XXXXM1 </a> )</li><li>HEPES (Sigma-Aldrich, Sigma-Millipore, catalog number: <a class="blackNoUnder" target="_blank"> H4034 </a> )</li><li>Manganese (II) chloride tetrahydrate (Sigma-Aldrich, Sigma Millipore, catalog number: <a class="blackNoUnder" target="_blank"> M3634 </a> )</li><li>12.1 N Hydrochloric acid (Fisher Scientific, Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> A144-212 </a> )</li><li>50% (w/w) Sodium hydroxide solution (Fisher Scientific, Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> SS254-4 </a> )</li><li>Tris base (Fisher Scientific, Fisher Bioreagents, catalog number: <a class="blackNoUnder" target="_blank"> L-15863 </a> )</li><li>Sodium chloride (Fisher Scientific, Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> L-11620 </a> )</li><li>Glycerol (Fisher Scientific, Fisher Bioreagents, catalog number: <a class="blackNoUnder" target="_blank"> L-13751 </a> )</li><li>Î±-<sup>32</sup>P-ATP 3000Ci/mmol 10mCi/ml, 250 ÂµCi (Perkin Elmer, BLU003H250UC)</li><li>4-20% Precast SDS-PAGE gel (Bio-Rad, Mini-PROTEAN TGX Gels, catalog number: <a class="blackNoUnder" target="_blank"> 456-1091 </a> )</li><li>Protein ladder (Bio-Rad, Precision Plus Protein Dual Color Standards, catalog number: <a class="blackNoUnder" target="_blank"> 1610374TGX </a> )</li><li>2-Mercaptoethanol (Sigma-Aldrich, Sigma Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> M3148 </a> )</li><li>Sodium dodecyl sulfate (Sigma-Aldrich, Sigma Life Sciences, catalog number: <a class="blackNoUnder" target="_blank"> L4390 </a> )</li><li>Bromophenol Blue sodium salt, ACS (Alfa Aesar, catalog number:32639)</li><li>Methanol (Fisher Scientific, Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> L-21063 </a> )</li><li>Acetic acid, glacial (Fisher Scientific, Fisher Chemical, catalog number: <a class="blackNoUnder" target="_blank"> A38C-212 </a> )</li><li>Glycine (Fisher Scientific, Fisher Bioreagents, catalog number: <a class="blackNoUnder" target="_blank"> L-15696 </a> )</li><li>Coomassie Brilliant Blue R-250 (Amresco, catalog number: <a class="blackNoUnder" target="_blank"> 0472-25G </a> )</li><li>Phosphor screen (General Electric, Kodak Phosphor Storage Screen, catalog number: 00<a class="blackNoUnder" target="_blank"> 146931 </a> )</li><li>Protein Storage Buffer (see Recipes)</li><li>10x AMPylation Buffer (see Recipes)</li><li>4x SDS-PAGE Loading Buffer (see Recipes)</li><li>1x SDS-PAGE Running Buffer (see Recipes)</li><li>Coomassie Stain (see Recipes)</li><li>Destain (see Recipes)<br/></li></ol><p class="pptt" id="biaoti35667">### Equipment</p><ol class="npwtwok"><li>Plexiglass or Lucite shield, 0.5 inches thick (Nalgene Acrylic Benchtop Beta Radiation Shield, Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 6700-1812 </a> )</li><li>Geiger counter (Model 3 Survey Meter, Ludlum Measurements, model: 44-7)</li><li>ddH<sub>2</sub>O dispenser (PURELAB Flex 2 Dispenser, ELGA LabWater, model: <a class="blackNoUnder" target="_blank"> PF2XXXXM1 </a> )</li><li>Heatblock (Analog Heatblock, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 12621-108 </a> )</li><li>pH meter (Seven Easy, Mettler Toledo, catalog number: <a class="blackNoUnder" target="_blank"> 1227396066 </a> )</li><li>Centrifuge (Eppendorf, model: <a class="blackNoUnder" target="_blank"> Centrifuge 5417C </a> , catalog number: 00<a class="blackNoUnder" target="_blank"> 23210 </a> )</li><li>Power supply (<a class="blackNoUnder" target="_blank"> PowerPac 300 </a> /DCode, Bio-Rad, model: <a class="blackNoUnder" target="_blank"> PowerPac 300 </a> )</li><li>PAGE system (Mini-PROTEAN Tetra Cell, 4-Gel System, Bio-Rad, catalog number: <a class="blackNoUnder" target="_blank"> 1658004 </a> )</li><li>Fluorescent Image Analyzer (Typhoon FLA 9500, General Electric, model: <a class="blackNoUnder" target="_blank"> Typhoon FLA 9000 </a> )</li><li>Phosphor screen eraser (<a class="blackNoUnder" target="_blank"> FLA Image Eraser </a> , General Electric, model: <a class="blackNoUnder" target="_blank"> FLA Image Eraser </a> )</li><li>Rocker (VWR, Rocking Platform, model: <a class="blackNoUnder" target="_blank"> 200 </a> )</li><li>Image scanner (Epson, Epson Perfection V700 Photo)<br/></li></ol><p class="pptt" id="biaoti35668">### Software</p><ol class="npwtwok"><li>Typhoon FLA 9500 (General Electric)</li></ol><p class="pptt" id="biaoti35669">### Procedure</p><ol class="npwone"><li>AMPylation reaction<br/><em>Note: This protocol requires use of radioactive Î±-</em><sup><em>32</em></sup><em>P-ATP. As such, all radioactive materials or materials in contact with them must be handled, stored, and disposed of in accordance with your institutionâs radioactive safety guidelines. Radioactivity must be handled behind a radiation shield at all times. A Geiger counter should be used to constantly monitor work environment for radioactive contamination. Appropriate personal protective equipment should be worn at all times.</em> <ol class="npwtwo"><li>Pipette calculated volume of ddH<sub>2</sub>O into designated 1.5 ml tubes.</li><li>Pipette calculated volume of 10x AMPylation Buffer into designated 1.5 ml tubes.<br/><em>Note: Components of 10x AMPylation Buffer may be modified as needed to accommodate differing experimental conditions, such as enzymes, substrates, etc.</em> </li><li>Pipette 2 Î¼g of purified, recombinant HYPE enzyme to designated 1.5 ml tubes.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Reactions were performed at a final volume of 20 Âµl. The actual reaction amounts may be modified so long as a â¥ 10:1 molar substrate:enzyme ratio is maintained, and a detectable amount of each protein (i.e., â¥ 0.1 Î¼g) is loaded onto the gel.</em> </li><li><em>All proteins are purified as previously described (Sanyal et al., 2019) and come dissolved in protein storage buffer.</em> </li><li><em>It is advisable to prepare twice the amount of required sample in case the gel needs to be rerun.</em> </li></ol></li><li>Pipette 20 Î¼g of purified, recombinant Î±-syn (or other substrate) into designated 1.5 ml tubes.<br/><em>Note: When performing kinetic experiments or comparing different substrates side-by-side, molar concentrations should be used in order to ensure that the relative numbers of molecules in the reaction are consistent.</em> </li><li>Pipette 2.5 Î¼Ci Î±-<sup>32</sup>P-ATP into designated 1.5 ml tubes.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Reaction start time begins with Î±-<sup>32</sup>P-ATP addition. As Î±-<sup>32</sup>P-ATP is the only radioactive component, the other sample ingredients may be added at a standard lab bench, and samples can then be transported to a designated radioactive area for Î±-<sup>32</sup>P-ATP addition.</em> </li><li><em>Alternative concentrations of Î±-<sup>32</sup>P-ATP may be used depending on the catalytic efficiencies of the test enzyme and/or the freshness of the nucleotide. These concentrations should be determined empirically. AMPylation reactions should ideally occur within the first half-life (14.3 days) of Î±-<sup>32</sup>P-ATP.</em> </li></ol></li><li>Incubate the samples in a heat block set to 30 Â°C for 30 min.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Be sure to place the heat block behind a radiation shield.</em> </li><li><em>Different AMPylases may require different reaction temperatures or durations for optimal enzymatic activity.</em> </li><li><em>Samples may be snap-centrifuged for 5 s at 2,000 x g in order to consolidate any sample components not at the bottom of the tube.</em> </li></ol></li><li>Quench reaction with an appropriate volume of 4x SDS Loading Buffer.</li><li>Heat samples for 10 min at 95 Â°C on a heat block.<br/><em>Note: Ensure that tube lids are firmly closed to avoid lids popping open during heating. A weighted sample tray may be placed flatly over tube lids to prevent opening. Alternatively, MTC Stop-Pop locking tubes clips (MTC Bio, Inc.) could be used.</em> </li><li>Snap centrifuge samples for 5 s at 2,000 <em>x g</em>.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>SDS-PAGE separation<ol class="npwtwo"><li>Carefully remove well comb and green seal tape from precast gel cassette.</li><li>Assemble gel cassette and tank according to manufacturersâ instructions.<br/><em>Note: Fresh 1x Running Buffer should be used inside of the gel cassette assembly, although used 1x Running Buffer may be used to fill the tank.</em> </li><li>Using gel loading pipette tips, carefully load 2-4 Î¼l of protein ladder into the first lane.</li><li>Using gel loading pipette tips, carefully load 10-30 Î¼l of each sample into lanes adjacent to the first lane.</li><li>Securely close tank lid.</li><li>Run gel at constant <a class="blackNoUnder" target="_blank"> 200 </a> V for 45 min. Alternatively, better band resolution can be obtained by first running the gel at 80 V for ~30 min (until the samples clear the stacking layer), then ramping up to 120 V for ~1 h (until the dye front reaches the reference line on the gel cassette).<br/><em>Note: Ensure proper current flow via production of bubbles in the running buffer and migration of the dye front towards the bottom of the gel.</em><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Phosphor imaging<ol class="npwtwo"><li>Carefully remove gel using a cassette lever.<br/><em>Note: The gel is radioactive at this point and should be handled and disposed of in accordance with your institutionâs regulations/guidelines for radioactive materials.</em> </li><li>Place gel in between a clean, plastic cover sheet. Smooth out any air bubbles in the gel.</li><li>Place cover sheet with gel on top of the imaging side (white) of a clean Phosphor screen.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Phosphor screen should be erased with the Phosphor screen eraser after each use to prevent signal contamination.</em> </li><li><em>Avoid touching the imaging side of the Phosphor screen, as this can damage the screen.</em> </li></ol></li><li>Carefully place Phosphor screen and gel inside of the foam phosphor screen storage container.</li><li>Expose the screen for 3 h at room temperature.<br/><em>Note: Duration of Phosphor screen exposure is dependent on the AMPylation reaction parameters and must therefore be determined empirically.</em> </li><li>Place the Phosphor screen imaging-face down in an imager.<br/><em>Note: Ensure imager screen is clear of debris before imaging, as these may mask sample signal. Kimwipes and ddH<sub>2</sub>O should be used to clean imager screen as needed.</em> </li><li>Open the Typhoon FLA 9500 software.</li><li>Under âMode Settingsâ, select the âPhosphorimagingâ tab.</li><li>Under âReader Settings,â go to the âMethodâ dropdown tab and select â[Phosphor]â option.<br/><em>Note: The â[Phosphor]â option will be colored red and correspond to a wavelength of 673 nm.</em> </li><li>Under âReader Settings,â go to âPMTâ (Photo-multiplier Tube) and select a value between 250-1,000 V.<br/><em>Note: A PMT of 500 V is usually a good starting point to gauge band intensity; the higher the PMT, the stronger the signal.</em> </li><li>Under âReader Settings,â go to âAreaâ and select the grid area corresponding to where the gel is placed on the screen. Alternatively, the entire screen may be imaged, but this takes additional time.</li><li>Under âReader Settingsâ go to âPixel Sizeâ and Select â50 Î¼mâ pixel size.<br/><em>Note: Lower or higher resolutions may be selected based on specific image quality needs.</em> </li><li>Select âScanâ to scan image.</li><li>Select âSave Asâ to save image as a Tiff file in the appropriate folder.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Coomassie staining<ol class="npwtwo"><li>Once exposure of the Phosphor screen to the gel is complete, incubate the gel in Coomassie Stain with a rocker set to low to medium speed for 10 min.<br/><em>Note: The amount of stain used will vary depending on the shape and dimensions of the container used for incubation but should be enough to fully immerse the gel.</em> </li><li>Pour off Coomassie Stain in its original container for reuse.</li><li>Incubate gel in Destain on rocker for 30 min or until sufficient contrast between protein bands and gel background is established.</li><li>Discard Destain in appropriate waste container.</li><li>Scan gel on generic document scanner between cover sheet, or with phone camera or scanning application.<br/><em>Notes:</em> <ol class="npwthree"><li><em>Coomassie staining is done to control for protein loading, and should yield bands that validate experimental amounts of protein (see Figure 1, bottom).</em> </li><li><em>If reactions are set up using molar ratios instead of mass-based concentrations, the apparent band sizes may vary with the differing molecular weights of the proteins.</em> </li></ol><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917015140_7350.jpg" width="300"/><br/><strong>Figure 1. HYPE AMPylates Î±-syn <em>in vitro</em>.</strong> AMPylation competent E234G-HYPE robustly AMPylates Î±-syn and itself (auto-AMPylation) with Î±-<sup>32</sup>P-AMP (top). Intrinsically autoinhibited WT-HYPE and a catalytically dead E234G/H363A mutant show no auto-AMPylation or substrate AMPylation (top). 1 Î¼g of enzyme and 10 Î¼g of substrate from the AMPylation reaction were loaded on the gel (bottom). Phosphor screen was exposed for 3 h (top).</li></ol></li></ol><p class="pptt" id="biaoti35670">### Recipes</p><p class="ppzz"><em>Note: Dissolve the following solutions in ddH<sub>2</sub>O. Always add solvent before addition of other components. No pH adjustments are required unless explicitly stated. Adjust pH to appropriate value with 12.1 N HCl or 50% NaOH with pipette and rubber suction bulb. Diluted acid or base may also be prepared to avoid overshooting target pH.<br/><br/></em> </p><ol class="npwtwok"><li>Protein Storage Buffer<br/>50 mM Tris base<br/>300 mM NaCl<br/>10% (v/v) glycerol<br/>Adjust to pH 7.5</li><li>10x AMPylation Buffer<br/>50 mM HEPES<br/>10 mM MnCl<sub>2</sub><br/>Adjust to pH 7.5</li><li>4x SDS-PAGE Loading Buffer<br/>2.86 M BME<br/>250 mM Tris, pH 6.8<br/>10% (w/v) SDS<br/>0.008% (v/v) bromophenol blue<br/>40% (v/v) glycerol</li><li>1x SDS-PAGE Running Buffer<br/>0.3% Tris base<br/>1.4% glycine<br/>0.1% (w/v) SDS</li><li>Coomassie Stain<br/>50% (v/v) methanol<br/>10% (v/v) acetic acid<br/>0.1% Coomassie Brilliant Blue</li><li>Destain<br/>50% (v/v) methanol<br/>10% (v/v) acetic acid<br/></li></ol><p class="pptt" id="biaoti35671">### Acknowledgments</p><p class="ppzz">This publication was made possible with partial support from the Indiana CTSI TL1 Predoctoral Fellowship (Grant # UL1TR002529 [A. Shekhar, PI], 5/18/2018-4/30/2023, and Grant # TL1TR002531 [T. Hurley, PI], 5/18/2018-4/30/2023, from the National Institutes of Health, National Center for Advancing Translational Sciences, Clinical and Translational Sciences Award) to A.C.; and Eli Lilly-Stark Neuroscience Research Institute-CTSI predoctoral fellowship to S.D.; a grant from the Branfman Family Foundation to J.C.R.; and the National Institute of General Medical Sciences of the National Institute of Health (R01GM10092), an Indiana Clinical and Translational Research Grant (CTSI-106564), and a Purdue Institute for Inflammation, Immunology, and Infectious Disease Core Start Grant (PI4D-209263) to S.M. The authors would also like to thank members of the Rochet and Mattoo laboratories for their helpful discussions.</p><p class="pptt" id="biaoti35672">### Competing interests</p><p class="ppzz">The authors declare no conflicting or competing interests.</p><p class="pptt" id="biaoti35673">### References</p><ol class="npwtwok"><li>Chen, L. and Feany, M. B. (<a class="blackNoUnder" target="_blank"> 200 </a> 5). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15834418" target="_blank">Alpha-synuclein phosphorylation controls neurotoxicity and inclusion formation in a <em>Drosophila</em> model of Parkinson disease.</a> <em>Nat Neurosci</em> 8(5): 657-663.</li><li>Dedic, E., Alsarraf, H., Welner, D. H., Ostergaard, O., Klychnikov, O. I., Hensbergen, P. J., Corver, J., van Leeuwen, H. C. and Jorgensen, R. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27076635" target="_blank">A Novel Fic (Filamentation Induced by cAMP) protein from clostridium difficile reveals an inhibitory motif-independent adenylylation/AMPylation mechanism.</a> <em>J Biol Chem</em> 291(25): 13286-13300.</li><li>Engel, P., Goepfert, A., Stanger, F. V., Harms, A., Schmidt, A., Schirmer, T. and Dehio, C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22266942" target="_blank">Adenylylation control by intra- or intermolecular active-site obstruction in Fic proteins.</a> <em>Nature</em> 482(7383): 107-110.</li><li>Ham, H., Woolery, A. R., Tracy, C., Stenesen, D., Kramer, H. and Orth, K. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25395623" target="_blank">Unfolded protein response-regulated <em>Drosophila</em> Fic (dFic) protein reversibly AMPylates BiP chaperone during endoplasmic reticulum homeostasis.</a> <em>J Biol Chem</em> 289(52): 36059-36069.</li><li>Iwai, A., Masliah, E., Yoshimoto, M., Ge, N., Flanagan, L., de Silva, H. A., Kittel, A. and Saitoh, T. (1995). <a href="http://www.ncbi.nlm.nih.gov/pubmed/7857654" target="_blank">The precursor protein of non-A beta component of Alzheimer's disease amyloid is a presynaptic protein of the central nervous system.</a> <em>Neuron</em> 14(2): 467-475.</li><li>Levine, P. M., Galesic, A., Balana, A. T., Mahul-Mellier, A.-L., Navarro, M. X., De Leon, C. A., Lashuel, H. A. and Pratt, M. R. (2019). <a href="https://www.pnas.org/content/116/5/1511" target="_blank">Î±-Synuclein O-GlcNAcylation alters aggregation and toxicity, revealing certain residues as potential inhibitors of Parkinsonâs disease.</a> <em>Proc Natl Acad Sci U S A</em> 116(5): 1511-1519.</li><li>Lewallen, D. M., Steckler, C. J., Knuckley, B., Chalmers, M. J. and Thompson, P. R. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22456874" target="_blank">Probing adenylation: using a fluorescently labelled ATP probe to directly label and immunoprecipitate VopS substrates.</a> <em>Mol Biosyst</em> 8(6): 1701-1706.</li><li>Mattoo, S., Durrant, E., Chen, M. J., Xiao, J., Lazar, C. S., Manning, G., Dixon, J. E. and Worby, C. A. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21795713" target="_blank">Comparative analysis of Histophilus somni immunoglobulin-binding protein A (IbpA) with other fic domain-containing enzymes reveals differences in substrate and nucleotide specificities.</a> <em>J Biol Chem</em> 286(37): 32834-32842.</li><li>Preissler, S., Rato, C., Perera, L., Saudek, V. and Ron, D. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27918543" target="_blank">FICD acts bifunctionally to AMPylate and de-AMPylate the endoplasmic reticulum chaperone BiP.</a> <em>Nat Struct Mol Biol</em> 24(1): 23-29.</li><li>Sanyal, A., Chen, A. J., Nakayasu, E. S., Lazar, C. S., Zbornik, E. A., Worby, C. A., Koller, A. and Mattoo, S. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25601083" target="_blank">A novel link between Fic (filamentation induced by cAMP)-mediated adenylylation/AMPylation and the unfolded protein response.</a> <em>J Biol Chem</em> 290(13): 8482-8499.</li><li>Sanyal, A., Dutta, S., Camara, A., Chandran, A., Koller, A., Watson, B. G., Sengupta, R., Ysselstein, D., Montenegro, P., Cannon, J., Rochet, J. C. and Mattoo, S. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31034889" target="_blank">Alpha-synuclein is a target of fic-mediated adenylylation/AMPylation: possible implications for parkinson's disease.</a> <em>J Mol Biol</em> 431(12): 2266-2282.</li><li>Spillantini, M. G., Schmidt, M. L., Lee, V. M., Trojanowski, J. Q., Jakes, R. and Goedert, M. (1997). <a href="http://www.ncbi.nlm.nih.gov/pubmed/9278044" target="_blank">Alpha-synuclein in Lewy bodies.</a> <em>Nature</em> 388(6645): 839-840.</li><li>Truttmann, M. C., Cruz, V. E., Guo, X., Engert, C., Schwartz, T. U. and Ploegh, H. L. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27138431" target="_blank">The <em>Caenorhabditis elegans</em> protein FIC-1 is an AMPylase that covalently modifies heat-shock 70 family proteins, translation elongation factors and histones.</a> <em>PLoS Genet</em> 12(5): e1006023. </li><li>Truttmann, M. C. and Ploegh, H. L. (2017). <a href="https://bio-protocol.org/cn/e2416" target="_blank"><em>In vitro</em> AMPylation assays using purified, recombinant proteins.</a> <em>Bio-protocol</em> 7(14): e2416.</li><li>Veyron, S., Peyroche, G. and Cherfils, J. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29617857" target="_blank">FIC proteins: from bacteria to humans and back again.</a> <em>Pathog Dis</em> 76(2). </li><li>Weinreb, P. H., Zhen, W., Poon, A. W., Conway, K. A. and Lansbury, P. T., Jr. (1996). <a href="http://www.ncbi.nlm.nih.gov/pubmed/8901511" target="_blank">NACP, a protein implicated in Alzheimer's disease and learning, is natively unfolded.</a> <em>Biochemistry</em> 35(43): 13709-13715. </li><li>Worby, C. A., Mattoo, S., Kruger, R. P., Corbeil, L. B., Koller, A., Mendez, J. C., Zekarias, B., Lazar, C. and Dixon, J. E. (<a class="blackNoUnder" target="_blank"> 200 </a> 9). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19362538" target="_blank">The fic domain: regulation of cell signaling by adenylylation.</a> <em>Mol Cell</em> 34(1): 93-103.</li><li>Yamin, G., Uversky, V. N. and Fink, A. L. (<a class="blackNoUnder" target="_blank"> 200 </a> 3). <a href="http://www.ncbi.nlm.nih.gov/pubmed/12729915" target="_blank">Nitration inhibits fibrillation of human alpha-synuclein <em>in vitro</em> by formation of soluble oligomers.</a> <em>FEBS Lett</em> 542(1-3): 147-152.</li><li>Yarbrough, M. L., Li, Y., Kinch, L. N., Grishin, N. V., Ball, H. L. and Orth, K. (<a class="blackNoUnder" target="_blank"> 200 </a> 9). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19039103" target="_blank">AMPylation of Rho GTPases by Vibrio VopS disrupts effector binding and downstream signaling.</a> <em>Science</em> 323(5911): 269-272. </li><li>Ysselstein, D., Dehay, B., Costantino, I. M., McCabe, G. P., Frosch, M. P., George, J. M., Bezard, E. and Rochet, J. C. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/28069058" target="_blank">Endosulfine-alpha inhibits membrane-induced alpha-synuclein aggregation and protects against alpha-synuclein neurotoxicity.</a> <em>Acta Neuropathol Commun</em> 5(1): 3. <br/></li></ol>