<body>
            <h1>Site-specific Labeling of B Cell Receptor and Soluble Immunoglobulin</h1>
            <p><strong>Authors:</strong> Yu Wang, Zhixun Shen, Zhengpeng Wan, Wanli Liu</p>
            <h2>Abstract</h2>
            <p>B lymphocyte activation is regulated by its membrane-bound B cell receptors (BCRs) upon recognizing diverse antigens. It is hypothesized that antigen binding would trigger conformational changes within BCRs, followed by a series of downstream signaling activation. To measure the BCR conformational changes in live cells, a fluorescent site-specific labeling technique is preferred. Genetically encoded fluorescent tags visualize the location of the target proteins. However, these fluorescent proteins are large (~30 kDa) and would potentially perturb the conformation of BCRs. Here, we describe the general procedures of utilizing short tag-based site-specific labeling methodologies combining with fluorescence resonance energy transfer (FRET) assay to monitor the conformational changes within BCR extracellular domains upon antigen engagement.</p>
            <p><strong>Keywords:</strong> B lymphocytes, BCR, Conformational changes, Site-specific labeling</p>
            <h2>Background</h2><p>B lymphocytes are responsible for the production of protective antibodies against pathology arising from the recognition of antigens by the cell membrane expressed B cell receptors (BCRs). BCR complex comprises a membrane-bound immunoglobulin (mIg) and a non-covalently linked heterodimer of IgÎ± and IgÎ². The mIg is composed of two surrogate light chains and two immunoglobulin heavy chains. The mIg heavy chain contains the extracellular domain, the transmembrane domain, and the intracellular domain. At the N-terminal domain of extracellular mIg, there are two variable antigen-binding motifs, following which are the constant domains (Reth, 1992). In terms of the heavy chain of IgM-BCR, it includes 4 domains, CÎ¼1, CÎ¼2, CÎ¼3, and CÎ¼4 (Figure 1). Structurally variable domains in the heavy and light chain polypeptides form an antigen-binding site unique to the antibody. For example, VRC01 broadly neutralizing antibodies (bnAbs) target the CD4-binding site (CD4BS) of the human immunodeficiency virus-1 (HIV-1) envelope glycoprotein 120 (gp120), neutralize over 90% of circulating HIV-1 isolates (Wu <em>et al.</em>, 2010). BCRs expressing variable domains of VRC01 would recognize the gp120 and regulate B cell activation. In the BCR complex, disulfide-linked IgÎ±/IgÎ² are the signaling subunits, contain immunoreceptor tyrosine activation motif (ITAM) in their intracellular domains. Antigen binding to BCR induces ITAM phosphorylation by tyrosine-protein kinase Lyn followed by subsequent recruitments and phosphorylation of the second kinase, spleen tyrosine kinase (Syk), resulting in the onset of several downstream signal transduction pathways altering gene expression to potentiate the immune mechanisms (Xu <em>et al.</em>, 2014).<br/><br/>It is assumed that the antigen engagement induces conformational changes within the BCR complex to initiate B cell activation (Harwood and Batista, 2010; Pierce and Liu, 2010). However, it is technically difficult to capture the conformational information of BCR extracellular domains accurately during the transmembrane initiation of BCR activation. Fluorescence resonance energy transfer (FRET) is a powerful tool for analyzing the statics and dynamics of protein structures by introducing light-sensitive donor and acceptor. Usually, to acquire the location of target proteins in live cells, genetically encoded tags such as green fluorescent protein (GFP) are widely used. However, such fluorescent proteins are too large (~30 kDa) to show the conformational changes of BCRs (~150 kDa). Moreover, the presence of these proteins within BCR fragments may perturb the activity, localization, and functions of BCRs. To overcome this, we introduced short tag-based site-specific labeling techniques to label different domains of BCRs fluorescently (Shen <em>et al.</em>, 2019). We constructed various BCR complexes with ybbR tag and tetracysteine tag inserted that permit targeted incorporation of pre-defined fluorophores. In terms of ybbR tag-based labeling, Sfp phosphopantetheinyl transferase covalently transfers 4â²-phosphopantetheinyl (Ppant) groups from CoA to conserved serine residues on peptidyl carrier protein (PCP) and acyl carrier protein (ACP) domains (Lambalot <em>et al.</em>, 1996). Sfp and AcpS have been widely used for site-specific protein labeling in cell lysates or on live-cell surfaces by fusing PCP or ACP at either the N or the C terminus of the target protein, and Sfp can enzymatically attach a small molecular probe, such as biotin, fluorophores, sugars, and peptides, to the PCP or ACP tag (George <em>et al.</em>, 2004; Yin <em>et al.</em>, 2004a and 2004b). Most recently, the ybbR tag, a short peptide (DSLEFIASKLA), was found to be an efficient substrate for Sfp-catalyzed protein labeling, thereby replacing the PCP or ACP domain for the construction of fusions of the target protein (Yin <em>et al.</em>, 2006). And these fusions also can be labeled with probes by Sfp, further expanding the versatility of Sfp mediated site-special labeling system.<br/><br/>Another short tag site-specific labeling strategy used in this protocol is tetracysteine tag technology. This method is based on the binding of a small fluorescein derivative, fluorescein arsenical hairpin binder (FlAsH) to an optimized short peptide sequence (CCPGCC). Subsequently, flanking sequences were optimized and introduced as part of the tag (FLNCCPGCCMEP and HRWCCPGCCKTF) (Martin <em>et al.</em>, 2005). FlAsH emits green-yellow fluorescence, while the color variant, resorufin arsenical hairpin binder (ReAsH) fluorescence peaks in the red (609 nm), would serve as a FRET acceptor for green fluorescent signal, such as Coenzyme A (CoA)-488. FlAsH and ReAsH are used as the nonfluorescent complex with ethanedithiol (EDT) and become fluorescent on binding to tetracysteine tag. Tetracysteine tag technology has been used in FRET studies, allowing the investigation of conformational changes in target proteins, such as AMPA receptors (Morishita <em>et al.</em>, 2004), calmodulin (Chen <em>et al.</em>, 2005), adrenergic receptor (ZÃ¼rn <em>et al.</em>, 2009), and HIV Gag proteins (Turville <em>et al.</em>, 2008). In this chapter, we describe the procedures to label the BCRs and soluble Ig by inserting the ybbR tag and tetracysteine tag in BCR molecule, as well as the usage of necessary components to set up the site-special labeling system. This labeling protocol proceeds with high efficiency and can be easily carried out in living cells.</p>
        </body>

<p class="pptt" id="biaoti35709">## Materials and Reagents</p><ol class="npwtwok"><li>Glass coverslips (VWR International, catalog number: <a class="blackNoUnder" target="_blank"> 16004-094 </a> )</li><li>8-well chamber frame (Nunc Lab-Tek chamber, Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 155411PK </a> )</li><li>Six-well plate (NEST, catalog number: <a class="blackNoUnder" target="_blank"> 703001 </a> )</li><li>Sfp phosphopantetheinyl transferase (NEB, catalog number: <a class="blackNoUnder" target="_blank"> P9302S </a> )</li><li>Coenzyme A (CoA)-488 (NEB, catalog number: <a class="blackNoUnder" target="_blank"> S9348 </a> )</li><li>4,5-bis(1,3,2-dithiarsolan-2-yl)resorufin (ReAsH-EDT2, Santa Cruz, catalog number: <a class="blackNoUnder" target="_blank"> sc-391916 </a> )</li><li>TCEP (tris(carboxyethyl) phosphine) (Sigma, catalog number: <a class="blackNoUnder" target="_blank"> C4706 </a> )</li><li>X-tremeGENE HP DNA Transfection Reagent (Roche, catalog number: 0<a class="blackNoUnder" target="_blank"> 6366236001 </a> )</li><li>MES (2-mercaptoethanesulfonate) (Macklin, catalog number: <a class="blackNoUnder" target="_blank"> S818461 </a> )</li><li>BAL (2,3-dimercaptopropanol) (Macklin, catalog number: <a class="blackNoUnder" target="_blank"> D807026 </a> )</li><li>Alexa Fluor 647 AffiniPure Fab Fragment Goat Anti-Human IgM, Fc5Î¼ fragment specific (Jackson ImmunoResearch, catalog number: <a class="blackNoUnder" target="_blank"> 109-607-043 </a> )</li><li>HBSS (Hank's Balanced Salt Solution) (Gibco, catalog number: <a class="blackNoUnder" target="_blank"> 14025092 </a> )</li><li>Zeba Spin Desalting Columns (Thermo Fisher, catalog number: <a class="blackNoUnder" target="_blank"> 89882 </a> )</li><li>SYLGARD<sup>TM</sup> 164 Silicone Elastomer Kit 210 ML KIT (Dow Corning, catalog number: <a class="blackNoUnder" target="_blank"> 4028273 </a> )<br/></li></ol><p class="pptt" id="biaoti35710">### Equipment</p><ol class="npwtwok"><li>Fluoview FV1000 Laser Scanning Confocal Microscope (Olympus)</li></ol><p class="pptt" id="biaoti35711">### Procedure</p><ol class="npwone"><li>Site-specific labeling of VRC01-IgM-BCR expressed in 293T cell<ol class="npwtwo"><li>The constant region of human IgM heavy chain was fused with VRC01-specific variable region of heavy chain to construct mIg heavy chain of VRC01-mIgM-BCR (pHAGE-VRC01-H), while human IgÎº constant region was fused with VRC01-specific variable region of light chain to construct mIg light chain of VRC01-mIgM-BCR (pHAGE-VRC01-L).</li><li>A similar strategy was used for the construction of Ig heavy chain and Ig light chain of soluble VRC01-IgM. In addition, His<sub>6</sub> tag was fused at C terminus of Ig heavy chain of soluble VRC01-IgM for protein purification (pHAGE-VRC01-H-His6).</li><li>ybbR tag and tetracysteine tag were inserted in plasmid carrying mIg heavy chain of VRC01-IgM-BCR to construct dually tagged VRC01-mIgM-BCR (pHAGE-VRC01-H-Tag), while ybbR tag and tetracysteine tag were inserted in plasmid carrying Ig heavy chain of soluble VRC01-IgM to construct dually tagged soluble VRC01-IgM (pHAGE-VRC01-H-His6-Tag). Insertion sites for the two tags are indicated in bold in the following table (Table 1).<br/>All the plasmids are not commercially available. Please find the construction strategy illustrated in Figure 1. The DNA sequence of VRC01 variable region, ybbR tag, and tetracysteine tag are listed in Table 2. All the plasmid constructions were performed following Gibson assembly protocol (Gibson <em>et al.</em>, 2009). Protein sequences are listed in Table 3.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917022422_9139.jpg" width="300"/><br/><strong>Figure 1. Schematic figures of the construction strategies on mIgM-BCR structure and tagged VRC01-mIgM-BCR/soluble VRC01-IgM construction</strong><br/><br/><strong>Table 1. Insertion sites for the ybbR tag or tetracysteine tag in IgM-BCR</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917022631_0326.jpg" width="600"/><br/><br/><strong>Table 2. DNA sequences for plasmids construction</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917022648_3855.jpg" width="600"/><br/><br/><strong>Table 3. Amino acid sequences of the constructions</strong><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917022709_4870.jpg" width="800"/><br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>An overall scheme of site-specific labeling procedures can be found in Figure 2.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917022738_5323.jpg" width="300"/><br/><strong>Figure 2. Overall scheme of site-specific labeling procedures for VRC01-IgM-BCR expressed in 293T cells</strong><br/><br/><ol class="npwtwo"><li>Plate 293T cell in 6-well plate with DMEM containing 10% FBS and 1% penicillin-streptomycin reaching 50-60% confluency.</li><li>For each 35-mm well in a six-well plate, prepare one vial of transfection medium containing 15 Î¼l of X-tremGENE transfection reagent and 3.5 Î¼g of the pHAGE-VRC01-H/pHAGE-VRC01-H-Tag, 2.5 Î¼g pHAGE-VRC01-L, 2.5 Î¼g pHAGE-CD79A and 2.5 Î¼g pHAGE-CD79B plasmid diluted in 300 Î¼l of opti-MEM (1Ã).</li><li>Mix the transfection medium well and allow it to come to equilibrium for 30 min at room temperature.</li><li>Incubate the cells with transfection medium for 5 h, and then incubate the cells in DMEM medium (1Ã) with penicillin and streptomycin for 24 h to allow time for protein expression.</li><li>For ybbR tag labeling, detach the cells expressing dually tagged or untagged VRC01-IgM-BCR with trypsin and incubate them with 1 Î¼M SFP Synthase, 2 Î¼M CoA 488 and 10 mM MgCl<sub>2</sub> in 500 Î¼l HBSS for 20 min at room temperature, then wash the cells with HBSS.</li><li>Labeling of tetracysteine tag in cells is adopted from published protocols (Hoffmann <em>et al.</em>, 2010). Pre-incubate the cells in HBSS containing 5 mM MES and 0.5 mM TCEP for 20 min at room temperature, then wash with HBSS and stain 1 mM ReAsH-EDT2 and 25 mM 2,3-dimercapto-1-propanol in 500 Î¼l HBSS for 10 min at 4 Â°C. After staining, wash the cells with HBSS containing 100 mM BAL.</li><li>Stain the cells with 100 nM Alexa Fluor 647 AffiniPure Fab Fragment Goat Anti-Human IgM Fc5Î¼ for 5 min at 4 Â°C in 500 Î¼l HBSS, then wash twice with HBSS. Cells are ready for imaging.</li></ol><br/></li><li>Site-specific labeling of soluble VRC01-IgM<ol class="npwtwo"><li>293F cells were co-transfected with plasmid carrying dually tagged (pHAGE-VRC01-H-His6-Tag) or untagged Ig heavy chain of VRC01-IgM (pHAGE-VRC01-H-His6) and mIg light chain of VRC01-IgM (pHAGE-VRC01-L).<br/><em>Note: Please refer to B1 to B4 for transfection protocols.</em> </li><li>Ectopically express and purify the IgM protein in HEK293F cells according to standard protocols (Portolano <em>et al.</em>, 2014).</li><li>For ybbR tag labeling in soluble VRC01-IgM, purified proteins were incubated in HBSS containing 1 mM SFP synthase, 10 mM MgCl<sub>2</sub>, 5 mM CoA 488, 10 mM HEPES for 30 min at 37 Â°C. Then the labeled proteins were desalted using Zeba Spin Desalting Columns.</li><li>For tetracysteine tag labeling, soluble VRC01-IgMs were treated with 1 mM ReAsH-EDT2 in HBSS at room temperature for 30 min and were desalted using Zeba Spin Desalting Columns according to standard protocol. In short, after removing the columnâs bottom closure and loosen cap, centrifuge at 1,500 <em>Ã g</em> for 1 min to remove storage solution. Slowly apply 30-130 Âµl of the protein sample to the center of the compacted resin bed, followed with centrifuge at 1,500 <em>Ã g</em> for 2 min to collect the desalted sample.<br/><div style="white-space:nowrap;"><br/></div></li></ol></li><li>Imaging of labeled VRC01-IgM-BCR expressing 293T cells or labeled soluble VRC01-IgM<ol class="npwtwo"><li>Chamber coverslip was fabricated according to our previous publication (Wang <em>et al.</em>, 2018). In short, the coverslip was pretreated with piranha buffer (H<sub>2</sub>SO<sub>4</sub>:H<sub>2</sub>O<sub>2</sub> = 7:3) for overnight, after extensively washing with ddH<sub>2</sub>O, the coverslip was glued to the 8-well chamber frame with SYLGARD<sup>TM</sup> 164. After curing, the chamber coverslip was ready to use.</li><li>Labeled 293T cells expressing dually tagged or untagged VRC01-IgM-BCR were resuspended in HBSS and then loaded on chamber coverslip.</li><li>Labeled dually tagged or untagged soluble VRC01-IgMs were loaded on coverslip for 30 min at 4 Â°C to allow the adherence on the surface. Then soluble VRC01-IgMs were stained with 100 nM Alexa Fluor 647 AffiniPure Fab Fragment Goat Anti-Human IgM Fc5Î¼ in HBSS for 10 min at 4 Â°C, followed by washing with HBSS.</li><li>Confocal images were acquired by Fluoview FV1000 Laser Scanning Confocal Microscope with 60Ã 1.42 NA oil objective lens. Typical images were shown in Figure 3.</li><li>All the images were analyzed with ImageJ.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200917/20200917022751_3151.jpg" width="300"/><br/><strong>Figure 3. Site-specific labeling in mIg heavy chain of IgM-BCR.</strong> Representative confocal images of tagged (A) and untagged VRC01-IgM-BCR (B) expressed in 293T cells. ybbR tag and tetracysteine tag were labeled by CoA 488 and ReAsH, respectively. Alexa Fluor 647 (AF647) Fab fragment of goat anti-human IgM Fc5Î¼ was used for IgM-BCR staining. Scale bars = 10 Î¼m.</li></ol></li></ol><p class="pptt" id="biaoti35712">### Acknowledgments</p><p class="ppzz">This work has been supported by funds from National Natural Science Foundation of China (81825010, 81730043, 81621002, 81961130394 and 31811540397). It is also supported by Beijing Advanced Innovation Center for Structural Biology, Center for Life Sciences, and Institute for Immunology at Tsinghua University. This protocol was derived from previous paper published in <em>eLife</em> (Shen <em>et al.</em>, 2019).</p><p class="pptt" id="biaoti35713">### Competing interests</p><p class="ppzz">The authors declare no financial or commercial conflict of interest.</p><p class="pptt" id="biaoti35714">### References</p><ol class="npwtwok"><li>Chen, B., Mayer, M. U., Markillie, L. M., Stenoien, D. L. and Squier, T. C. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15654746" target="_blank">Dynamic motion of helix A in the amino-terminal domain of calmodulin is stabilized upon calcium activation.</a> <em>Biochemistry</em> 44(3): 905-914.</li><li>George, N., Pick, H., Vogel, H., Johnsson, N. and Johnsson, K. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15264811" target="_blank">Specific labeling of cell surface proteins with chemically diverse compounds.</a> <em>J Am Chem Soc</em> 126(29): 8896-8897.</li><li>Gibson, D. G., Young, L., Chuang, R. Y., Venter, J. C., Hutchison, C. A., 3rd and Smith, H. O. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19363495" target="_blank">Enzymatic assembly of DNA molecules up to several hundred kilobases.</a> <em>Nat Methods</em> 6(5): 343-345.</li><li>Xu, Y., Xu, L., Zhao, M., Xu, C., Fan, Y., Pierce, S., and Liu, W. (2014). <a href="https://pubmed.ncbi.nlm.nih.gov/24839903/" target="_blank">No receptor stands alone: IgG B-cell receptor intrinsic and extrinsic mechanisms contribute to antibody memory.</a> <em>Cell Research</em> 24(6): 651-664.</li><li>Harwood, N. E. and Batista, F. D. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20192804" target="_blank">Early events in B cell activation.</a> <em>Annu Rev Immunol</em> 28: 185-210.</li><li>Wang, J., Wan, Z. and Liu, W. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29388111" target="_blank">Understanding of B Cell Receptor Signaling Through a Photo-Activatable Antigen Presentation System.</a> <em>Methods Mol Biol</em> 1707: 225-234.</li><li>Lambalot, R. H., Gehring, A. M., Flugel, R. S., Zuber, P., Lacelle, M., Marahiel, M. A., Reid, R., Khosla, C. and Walsh, C. T. (1996). <a href="https://www.sciencedirect.com/science/article/pii/S1074552196901817" target="_blank">A new enzyme superfamily--the phosphopantetheinyl transferases.</a> <em>Chemistry Biology</em> 3: 923-936.</li><li>Martin, B. R., Giepmans, B. N., Adams, S. R. and Tsien, R. Y. (2005). <a href="http://www.ncbi.nlm.nih.gov/pubmed/16155565" target="_blank">Mammalian cell-based optimization of the biarsenical-binding tetracysteine motif for improved fluorescence and affinity.</a> <em>Nat Biotechnol</em> 23(10): 1308-1314.</li><li>Ju, W., Morishita, W., Tsui, J., Gaietta, G., Deerinck, T. J., Adams, S. R., Garner, C. C., Tsien, R. Y., Ellisman, M. H. and Malenka, R. C. (2004). <a href="http://www.ncbi.nlm.nih.gov/pubmed/14770185" target="_blank">Activity-dependent regulation of dendritic synthesis and trafficking of AMPA receptors.</a> <em>Nat Neurosci </em>7(3): 244-253.</li><li>Pierce, S. K. and Liu, W. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20935671" target="_blank">The tipping points in the initiation of B cell signalling: how small changes make big differences.</a> <em>Nat Rev Immunol</em> 10(11): 767-777.</li><li>Portolano, N., Watson, P. J., Fairall, L., Millard, C. J., Milano, C. P., Song, Y., Cowley, S. M. and Schwabe, J. W. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25349981" target="_blank">Recombinant protein expression for structural biology in HEK 293F suspension cells: a novel and accessible approach.</a> <em>J Vis Exp</em>(92): e51897.</li><li>Reth, M. (1992). <a href="http://www.ncbi.nlm.nih.gov/pubmed/1591006" target="_blank">Antigen receptors on B lymphocytes.</a> <em>Annu Rev Immunol</em> 10: 97-121.</li><li>Shen, Z., Liu, S., Li, X., Wan, Z., Mao, Y., Chen, C. and Liu, W. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31290744" target="_blank">Conformational change within the extracellular domain of B cell receptor in B cell activation upon antigen binding.</a> <em>Elife</em> 8: e42271.</li><li>Turville, S. G., Aravantinou, M., Stossel, H., Romani, N. and Robbiani, M. (2008). <a href="http://www.ncbi.nlm.nih.gov/pubmed/18059278" target="_blank">Resolution of de novo HIV production and trafficking in immature dendritic cells.</a> <em>Nat Methods</em> 5(1): 75-85.</li><li>Wu, X., Yang, Z. Y., Li, Y., Hogerkorp, C. M., Schief, W. R., Seaman, M. S., Zhou, T., Schmidt, S. D., Wu, L., Xu, L., Longo, N. S., McKee, K., O'Dell, S., Louder, M. K., Wycuff, D. L., Feng, Y., Nason, M., Doria-Rose, N., Connors, M., Kwong, P. D., Roederer, M., Wyatt, R. T., Nabel, G. J. and Mascola, J. R. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20616233" target="_blank">Rational design of envelope identifies broadly neutralizing human monoclonal antibodies to HIV-1.</a> <em>Science</em> 329(5993): 856-861.</li><li>Yin, J., Lin, A. J., Golan, D. E. and Walsh, C. T. (2006). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17406245" target="_blank">Site-specific protein labeling by Sfp phosphopantetheinyl transferase.</a> <em>Nat Protoc</em> 1(1): 280-285.</li><li>Yin, J., Liu, F., Li, X. and Walsh, C. T. (2004a). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15212504" target="_blank">Labeling proteins with small molecules by site-specific posttranslational modification.</a><em> J Am Chem Soc</em> 126(25): 7754-7755.</li><li>Yin, J., Liu, F., Schinke, M., Daly, C. and Walsh, C. T. (2004b). <a href="http://www.ncbi.nlm.nih.gov/pubmed/15493886" target="_blank">Phagemid encoded small molecules for high throughput screening of chemical libraries.</a> <em>J Am Chem Soc</em> 126(42): 13570-13571.</li><li>ZÃ¼rn, A., Zabel, U., Vilardaga, J. P., Schindelin, H., Lohse, M. J. and Hoffmann, C. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19106230" target="_blank">Fluorescence resonance energy transfer analysis of alpha 2a-adrenergic receptor activation reveals distinct agonist-specific conformational changes.</a> <em>Mol Pharmacol</em> 75(3): 534-541.<br/></li></ol>