<body>
            <h1>Using RNA Sequencing and Spike-in RNAs to Measure Intracellular Abundance of lncRNAs and mRNAs</h1>
            <p><strong>Authors:</strong> Megan D. Schertzer, McKenzie M. Murvin, J. Mauro Calabrese</p>
            <h2>Abstract</h2>
            <p>Long noncoding RNAs (lncRNAs) play essential roles in normal physiology and in disease but their mechanisms of action can be challenging to identify. For mechanistic studies, it is often useful to know a lncRNAâs intracellular abundance, <em>i.e.</em>, approximately how many molecules of the lncRNA are present in a typical cell of a cell-type of interest. At least two approaches have been used to approximate lncRNA intracellular abundance: single-molecule sensitivity RNA fluorescence <em>in situ</em> hybridization (smFISH) and single-gene, calibrated reverse-transcription followed by quantitative PCR (RT-qPCR). However, like all experimental approaches, these methods have their limitations. smFISH, when analyzed using diffraction-limited microscopy, can underestimate intracellular abundance, especially for lncRNAs that accumulate in focused subcellular regions. Calibrated RT-qPCR may return inaccurate estimates of abundance because individual PCR amplicons spaced across the length of a transcript can vary in their efficiency of reverse transcription. Here, we describe a sequencing-based approach that is straightforward, orthogonal to smFISH and RT-qPCR, and can be used to approximate the intracellular abundance for most expressed long RNAs (lncRNAs and mRNAs) in a cell type of interest. Firstly, the average weight of total RNA per cell for the cell type of interest is estimated by replicate rounds of RNA purification from a known number of cells. Secondly, an rRNA-depletion RNA-Seq protocol is performed after adding spike-in control RNAs to a known quantity of total cellular RNA. Lastly, by comparing read counts per transcript to a standard curve derived from the spiked-in RNAs, the intracellular abundance for each transcript is estimated. The sequencing-based approach provides a powerful complement to existing methods, particularly in situations where it is desirable to quantify the abundance of multiple lncRNAs and/or mRNAs simultaneously.</p>
            <p><strong>Keywords:</strong> RNA-Seq, Ribosomal RNA depletion, lncRNA, <em>Xist</em>, ERCC Spike-In RNAs, Transcriptome, RNA FISH, smFISH</p>
            <h2>Background</h2><p>Long noncoding RNAs (lncRNAs) play essential roles in biology but their mechanisms of action can be difficult to determine (Kopp and Mendell, 2018; Gil and Ulitsky, 2020). Relative to protein-coding genes, lncRNAs can evolve rapidly and are not constrained by codon usage (Cabili <em>et al.</em>, 2011; Kutter <em>et al.</em>, 2012; Necsulea <em>et al.</em>, 2014; Schuler <em>et al.</em>, 2014; Washietl <em>et al.</em>, 2014; Hezroni <em>et al.</em>, 2015; Chen <em>et al.</em>, 2016; Ulitsky, 2016). Accordingly, they often lack easily identifiable domains that might otherwise provide insight into their molecular actions. Thus, for a given lncRNA, initial footholds into its molecular mechanism are often gained by examining its sub-cellular localization and its intracellular abundance (<em>i.e.</em>, on average, how many molecules of the lncRNA are present in a single cell of a cell type of interest).<br/> <br/>To these ends, single-molecule sensitivity RNA fluorescence <em>in situ</em> hybridization (smFISH) has been a boon, providing a convenient and cost-effective way to simultaneously investigate lncRNA sub-cellular localization and intracellular abundance (Cabili <em>et al.</em>, 2015; Tsanov <em>et al.</em>, 2016; Raj and Rinn, 2019). Nevertheless, while smFISH offers unparalleled benefits in regards to visualizing lncRNA sub-cellular distribution, it does have limitations in regards to estimating lncRNA intracellular abundance. In the simplest form of smFISH, the number of FISH puncta per cell is used as a proxy for a lncRNAâs intracellular abundance. However, especially when smFISH is performed using diffraction-limited microscopy, separate lncRNA molecules that are located in spatial proximity may not be individually resolved; instead, such lncRNAs may appear as single puncta. Thus, particularly for those lncRNAs that accumulate to high concentrations in specific subcellular regions (Chujo and Hirose, 2017; Ninomiya and Hirose, 2020), smFISH may underestimate intracellular abundance. This potential limitation can be overcome by performing smFISH using super-resolution microscopy and carefully quantifying signal intensity within individual puncta (Cerase <em>et al.</em>, 2014; Smeets <em>et al.</em>, 2014; Sunwoo <em>et al.</em>, 2015). However, this latter approach requires high-end equipment and expertise that may not be readily accessible.<br/> <br/>Here, we describe a sequencing-based approach that is orthogonal to smFISH and can provide estimates for the intracellular abundance of lncRNAs as well as mRNAs in cultured cells (Figure 1; Schertzer <em>et al.</em>, 2019). The approach relies on RNA-Seq and requires minimal expertise beyond the ability to follow standard protocols in molecular biology and bioinformatics. The approach allows for the simultaneous quantitation of the intracellular abundance of all long RNAs (lncRNAs and mRNAs) that are expressed in a cell type of interest.<br/> <br/>The sequencing-based approach is also likely to yield estimates of intracellular abundance that are more robust than those produced by single-gene approaches that rely on calibrated reverse transcription followed by gene-specific quantitative PCR (RT-qPCR [Schwaber <em>et al.</em>, 2019]). A primary reason for this is because different qPCR amplicons spaced across the length of a transcript may vary dramatically in their efficiency of reverse transcription. In contrast, in RNA-seq, local, intra-transcript variations in reverse transcription efficiency are inherently averaged, owing to the chemical fragmentation of RNA that occurs just prior to reverse transcription of the RNA into cDNA (Hrdlickova <em>et al.</em>, 2017). Moreover, in the sequencing-based approach, the use of ERCC Spike-In RNAs, which harbor diverse GC-contents, lengths, and abundances, obviates the need to explicitly estimate reverse transcription efficiency and instead allows transcript abundance to be estimated by comparison to a standard curve (Jiang <em>et al.</em>, 2011).<br/> <br/>Potential sources of errors in the sequencing-based approach include (1) errors associated with read alignment, which are most relevant if the lncRNA of interest contains sequence that is highly repetitive relative to other positions in the genome, (2) errors associated with transcript isoform uncertainty, which may arise if the predominant isoform of the lncRNA of interest is misannotated in the cell type of interest, and (3) errors associated with inaccurate or variable estimates of the total weight of RNA in the cell type of interest. Nevertheless, recently, the sequencing-based approach and smFISH performed using super-resolution microscopy have been shown to arrive at similar estimates of abundance for the lncRNA <em>Xist</em> (Smeets <em>et al.</em>, 2014; Sunwoo <em>et al.</em>, 2015; Schertzer <em>et al.</em>, 2019), lending confidence that both approaches provide reasonable estimates of RNA abundance. The sequencing-based approach is additionally useful because in a single experiment, it can be used to estimate the abundance of all expressed mRNAs and lncRNAs simultaneously.<br/><br/><img alt="" class="layerphoto" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200925/20200925181640_8127.png" width="300"/><br/><strong>Figure 1. Overview of sequencing-based approach to quantify lncRNA and mRNA intracellular abundance</strong></p>
        </body>

<p class="pptt" id="biaoti36837">## Materials and Reagents</p><ol class="npwone"><li>For Cell Counting<ol class="npwtwo"><li>Disposable Borosilicate Glass Pasteur Pipets (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 13-678-20D </a> ), sterilize before use</li><li>5 ml/10 ml sterile serological pipets (<em>e.g.</em>, Genesee Scientific, catalog numbers: <a class="blackNoUnder" target="_blank"> 12-102 </a> , <a class="blackNoUnder" target="_blank"> 12-104 </a> )</li><li>15 ml/50 ml conical bottom centrifuge tubes (<em>e.g.</em>, Corning, catalog numbers: <a class="blackNoUnder" target="_blank"> 05-538-59A </a> , <a class="blackNoUnder" target="_blank"> 05-526B </a> )</li><li>Glass coverslip (<em>e.g.</em>, Fischer Scientific, Hausser Hemacytometer Cover Glass, catalog number: <a class="blackNoUnder" target="_blank"> 02-671-53 </a> )</li><li>Kimwipes (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 06-666 </a> )</li><li>6 cm tissue culture dishes (<em>e.g.</em>, Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 25-260 </a> )</li><li>Mammalian cell type of interest (<em>e.g.</em>, mouse trophoblast stem cells [Calabrese <em>et al.</em>, 2012])</li><li>Cell Culture Media supplemented with 10% serum (<em>e.g.</em>, DMEM supplemented with 10% FBS; DMEM, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 11995065 </a> ; FBS, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 97068-085 </a> )</li><li>Sterile 1x PBS (<em>e.g.</em>, Corning, catalog number: <a class="blackNoUnder" target="_blank"> 21-040-CM </a> )</li><li>0.25% Trypsin-EDTA (<em>e.g.</em>, GIBCO, catalog number: <a class="blackNoUnder" target="_blank"> 25200-072 </a> )</li><li>Trypan Blue Solution, 0.4% (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15-250-06 </a> )</li><li>70% ethanol, stored at room temperature<br/><br/></li></ol></li><li>For RNA Purification<ol class="npwtwo"><li>RNase Zap (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9780 </a> )</li><li>P20/200/1000 Barrier pipette tips (<em>e.g.</em>, Olympus brand tips, Genesee Scientific, catalog numbers: <a class="blackNoUnder" target="_blank"> 23-404 </a> , <a class="blackNoUnder" target="_blank"> 24-412 </a> , <a class="blackNoUnder" target="_blank"> 24-430 </a> )</li><li>1.7 ml microcentrifuge tubes (<em>e.g.</em>, Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 22-282 </a> )</li><li>TRIzol Reagent (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 15596018 </a> )</li><li>Chloroform (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP1145-1 </a> )</li><li>Isopropanol (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP2618-1 </a> )</li><li>Linear Acrylamide (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> AM9520 </a> )</li><li>RNase-free water (<em>e.g.</em>, we use deionized 18.2 MÎ© water produced from a Synergy Water Purification System, Millipore, catalog number: <a class="blackNoUnder" target="_blank"> SYNS0HFWW </a> )</li><li>80% ethanol made with RNase-free water, stored at -20 Â°C</li><li>LE Agarose (<em>e.g.</em>, Genesee Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 20-102QD </a> )</li><li>Ethidium Bromide 1% Solution (Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> BP1302-10 </a> )</li><li>50x TAE Buffer (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> B49 </a> )</li><li>Agarose gel loading buffer and dye (<em>e.g.</em>, NEB, catalog number: <a class="blackNoUnder" target="_blank"> B7024S </a> )</li><li>1 Kb Plus DNA Ladder (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 10787026 </a> )<br/><br/></li></ol></li><li>For RNA-Seq<ol class="npwtwo"><li>KAPA RNA HyperPrep Kits with RiboErase (Roche/KAPA Biosystems, catalog number: <a class="blackNoUnder" target="_blank"> KK8560 </a> )</li><li>SeqCap Adapter Kit (see Note 1; Roche, catalog number: <a class="blackNoUnder" target="_blank"> 714153000 </a> )</li><li>ERCC RNA Spike-In Mix 1 (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 4456740 </a> )</li><li>Qubit dsDNA HS Assay Kit (Thermo Fisher Scientific, catalogue number: <a class="blackNoUnder" target="_blank"> Q32854 </a> )</li><li>ERCC annotation files (ERCC92.fa, ERCC92.gtf, and ERCC_Controls_Analysis.txt found in the ERCC RNA Spike-In product page, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 4456740 </a> )</li><li>Transcriptome annotation gtf file (<em>e.g.</em>, downloaded from Illuminaâs iGenomes site: <a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html" target="_blank">https://support.illumina.com/sequencing/sequencing_software/igenome.html</a>)</li><li>Appropriate genome sequence (<em>e.g.</em>, downloaded from Illuminaâs iGenomes site: <a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html" target="_blank">https://support.illumina.com/sequencing/sequencing_software/igenome.html</a>)<br/></li></ol></li></ol><p class="pptt" id="biaoti36838">### Equipment</p><ol class="npwone"><li>For Cell Counting<ol class="npwtwo"><li>Tissue Culture Incubator (<em>e.g.</em>, Forma Series II 3110 Water-Jacketed CO<sub>2</sub> Incubator, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 3110) </a> </li><li>Pipet Aid (<em>e.g.</em>, Drummond, catalog number: <a class="blackNoUnder" target="_blank"> DP-101) </a> </li><li>P2/P20/200/1000 micropipettes (<em>e.g.</em>, Research Plus 4-pack, Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> EPPR4330) </a>  </li><li>Centrifuge for 15 ml/50 ml conical tubes (<em>e.g.</em>, Eppendorf, model: 5810, catalog number: <a class="blackNoUnder" target="_blank"> 022628157) </a> </li><li>Phase Hemocytometer (<em>e.g.</em>, Fischer Scientific, Hausser Bright-Line, catalog number: <a class="blackNoUnder" target="_blank"> 02-671-6) </a> </li><li>Hand Tally Counter (<em>e.g.</em>, VWR, catalog number: <a class="blackNoUnder" target="_blank"> 23609-102) </a> </li><li>Inverted microscope with 10x objective (<em>e.g.</em>, Zeiss Primovert, catalog number: <a class="blackNoUnder" target="_blank"> 491206-0004-000) </a> <br/><br/></li></ol></li><li>For RNA Purification<ol class="npwtwo"><li>Safety goggles (<em>e.g.,</em> Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 19-053-950) </a> </li><li>Microcentrifuge, stored at 4 Â°C (<em>e.g.</em>, 5424 Microcentrifuge, Eppendorf, catalog number: <a class="blackNoUnder" target="_blank"> 5424) </a> </li><li>Mini Centrifuge with 1.7 ml tube rotor and PCR strip tube rotors (<em>e.g.</em>, Genesee Scientific MyFuge Mini, catalog number: <a class="blackNoUnder" target="_blank"> 31-500) </a> </li><li>Nanodrop spectrophotometer (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> ND-LITE) </a> </li><li>Mini Gel Electrophoresis System (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> B1A) </a> </li><li>Agarose Gel Imaging System (<em>e.g.</em>, Bio-Rad, Chemidoc MP, catalog number: <a class="blackNoUnder" target="_blank"> 12003154) </a> <br/><br/></li></ol></li><li>For RNA-Seq<ol class="npwtwo"><li>Thermal Cycler (<em>e.g.</em>, Bio-Rad, model: C1000 Touch, catalog number: <a class="blackNoUnder" target="_blank"> 1851148) </a> </li><li>Magnetic bead stand (<em>e.g.</em>, DynaMag-2 Magnet, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> 12321D) </a> </li><li>Qubit Fluorometer (Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> Q33238) </a> </li><li>Mini Gel Electrophoresis System (<em>e.g.</em>, Thermo Fisher Scientific, catalog number: <a class="blackNoUnder" target="_blank"> B1A) </a> </li><li>Agarose Gel Imaging System (<em>e.g.</em>, Bio-Rad, Chemidoc MP, catalog number: <a class="blackNoUnder" target="_blank"> 12003154) </a> </li><li>Access to an Illumina sequencing instrument (<em>e.g.</em>, Illumina, model: NextSeq500, catalog number: <a class="blackNoUnder" target="_blank"> SY-415-1001) </a> <br/></li></ol></li></ol><p class="pptt" id="biaoti36839">### Software</p><ol class="npwtwok"><li>(Optional) SRA toolkit (Leinonen <em>et al.</em>, 2011); <a href="https://ncbi.github.io/sra-tools/" target="_blank">https://ncbi.github.io/sra-tools/</a> </li><li>STAR aligner (Dobin <em>et al.</em>, 2013); <a href="https://github.com/alexdobin/STAR" target="_blank">https://github.com/alexdobin/STAR </a> </li><li>featureCounts from Subread package (Liao <em>et al.</em>, 2014); <a href="http://subread.sourceforge.net" target="_blank">http://subread.sourceforge.net</a> </li><li>Samtools (Li <em>et al.</em>, 2009); <a href="https://samtools.github.io" target="_blank">https://samtools.github.io</a> </li><li>Microsoft Excel or equivalent, or Rstudio (RStudio_Team, 2015); <!--[if gte mso 9]><xml><br /><o:documentproperties><br /><o:version>15.00</o:version><br /></o:documentproperties><br /><o:officedocumentsettings><br /><o:relyonvml /><br /><o:allowpng /><br /></o:officedocumentsettings><br /></xml><![endif]--><!--[if gte mso 9]><xml><br /><w:worddocument><br /><w:view>Normal</w:view><br /><w:zoom>0</w:zoom><br /><w:trackmoves /><br /><w:trackformatting /><br /><w:donotshowrevisions /><br /><w:donotprintrevisions /><br /><w:donotshowcomments /><br /><w:donotshowinsertionsanddeletions /><br /><w:donotshowpropertychanges /><br /><w:punctuationkerning /><br /><w:drawinggridverticalspacing>7.8 ç£</w:drawinggridverticalspacing><br /><w:displayhorizontaldrawinggridevery>0</w:displayhorizontaldrawinggridevery><br /><w:displayverticaldrawinggridevery>2</w:displayverticaldrawinggridevery><br /><w:validateagainstschemas /><br /><w:saveifxmlinvalid>false</w:saveifxmlinvalid><br /><w:ignoremixedcontent>false</w:ignoremixedcontent><br /><w:alwaysshowplaceholdertext>false</w:alwaysshowplaceholdertext><br /><w:donotpromoteqf /><br /><w:lidthemeother>EN-US</w:lidthemeother><br /><w:lidthemeasian>ZH-CN</w:lidthemeasian><br /><w:lidthemecomplexscript>X-NONE</w:lidthemecomplexscript><br /><w:compatibility><br /><w:spaceforul /><br /><w:balancesinglebytedoublebytewidth /><br /><w:donotleavebackslashalone /><br /><w:ultrailspace /><br /><w:donotexpandshiftreturn /><br /><w:adjustlineheightintable /><br /><w:breakwrappedtables /><br /><w:snaptogridincell /><br /><w:wraptextwithpunct /><br /><w:useasianbreakrules /><br /><w:dontgrowautofit /><br /><w:splitpgbreakandparamark /><br /><w:enableopentypekerning /><br /><w:dontflipmirrorindents /><br /><w:overridetablestylehps /><br /><w:usefelayout /><br /></w:compatibility><br /><m:mathpr><br /><m:mathfont m:val="Cambria Math" /><br /><m:brkbin m:val="before" /><br /><m:brkbinsub m:val="&#45;-" /><br /><m:smallfrac m:val="off" /><br /><m:dispdef /><br /><m:lmargin m:val="0" /><br /><m:rmargin m:val="0" /><br /><m:defjc m:val="centerGroup" /><br /><m:wrapindent m:val="1440" /><br /><m:intlim m:val="subSup" /><br /><m:narylim m:val="undOvr" /><br /></m:mathpr></w:worddocument><br /></xml><![endif]--><!--[if gte mso 9]><xml><br /><w:LatentStyles DefLockedState="false" DefUnhideWhenUsed="false"<br />DefSemiHidden="false" DefQFormat="false" DefPriority="99"<br />LatentStyleCount="371"><br /><w:lsdexception locked="false" priority="0" qformat="true" name="Normal" /><br /><w:lsdexception locked="false" priority="9" qformat="true" name="heading 1" /><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 2"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 3"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 4"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 5"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 6"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 7"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 8"/><br /><w:LsdException Locked="false" Priority="9" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="heading 9"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 6"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 7"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 8"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index 9"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 1"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 2"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 3"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 4"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 5"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 6"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 7"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 8"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" Name="toc 9"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Normal Indent"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="footnote text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="annotation text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="header"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="footer"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="index heading"/><br /><w:LsdException Locked="false" Priority="35" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="caption"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="table of figures"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="envelope address"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="envelope return"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="footnote reference"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="annotation reference"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="line number"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="page number"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="endnote reference"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="endnote text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="table of authorities"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="macro"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="toa heading"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Bullet 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Number 5"/><br /><w:lsdexception locked="false" priority="10" qformat="true" name="Title" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Closing"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Signature"/><br /><w:LsdException Locked="false" Priority="1" SemiHidden="true"<br />UnhideWhenUsed="true" Name="Default Paragraph Font"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text Indent"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="List Continue 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Message Header"/><br /><w:lsdexception locked="false" priority="11" qformat="true" name="Subtitle" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Salutation"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Date"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text First Indent"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text First Indent 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Note Heading"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text Indent 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Body Text Indent 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Block Text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />QFormat="true" Name="Hyperlink"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="FollowedHyperlink"/><br /><w:lsdexception locked="false" priority="22" qformat="true" name="Strong" /><br /><w:lsdexception locked="false" priority="20" qformat="true" name="Emphasis" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Document Map"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Plain Text"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="E-mail Signature"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Top of Form"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Bottom of Form"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Normal (Web)"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Acronym"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Address"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Cite"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Code"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Definition"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Keyboard"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Preformatted"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Sample"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Typewriter"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="HTML Variable"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Normal Table"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="annotation subject"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="No List"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Outline List 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Outline List 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Outline List 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Simple 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Simple 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Simple 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Classic 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Colorful 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Colorful 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Colorful 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Columns 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 6"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 7"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Grid 8"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 4"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 5"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 6"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 7"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table List 8"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table 3D effects 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table 3D effects 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table 3D effects 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Contemporary"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Elegant"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Professional"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Subtle 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Subtle 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Web 1"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Web 2"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Web 3"/><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Balloon Text"/><br /><w:lsdexception locked="false" priority="39" name="Table Grid" /><br /><w:LsdException Locked="false" SemiHidden="true" UnhideWhenUsed="true"<br />Name="Table Theme"/><br /><w:lsdexception locked="false" semihidden="true" name="Placeholder Text" /><br /><w:lsdexception locked="false" priority="1" qformat="true" name="No Spacing" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading" /><br /><w:lsdexception locked="false" priority="61" name="Light List" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3" /><br /><w:lsdexception locked="false" priority="70" name="Dark List" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 1" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 1" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 1" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 1" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 1" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 1" /><br /><w:lsdexception locked="false" semihidden="true" name="Revision" /><br /><w:LsdException Locked="false" Priority="34" QFormat="true"<br />Name="List Paragraph"/><br /><w:lsdexception locked="false" priority="29" qformat="true" name="Quote" /><br /><w:LsdException Locked="false" Priority="30" QFormat="true"<br />Name="Intense Quote"/><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 1" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 1" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 1" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 1" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 1" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 1" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 1" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 1" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 2" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 2" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 2" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 2" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 2" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 2" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 2" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 2" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 2" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 2" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 2" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 2" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 2" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 2" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 3" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 3" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 3" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 3" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 3" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 3" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 3" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 3" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 3" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 3" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 3" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 3" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 3" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 3" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 4" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 4" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 4" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 4" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 4" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 4" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 4" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 4" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 4" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 4" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 4" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 4" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 4" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 4" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 5" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 5" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 5" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 5" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 5" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 5" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 5" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 5" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 5" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 5" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 5" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 5" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 5" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 5" /><br /><w:lsdexception locked="false" priority="60" name="Light Shading Accent 6" /><br /><w:lsdexception locked="false" priority="61" name="Light List Accent 6" /><br /><w:lsdexception locked="false" priority="62" name="Light Grid Accent 6" /><br /><w:lsdexception locked="false" priority="63" name="Medium Shading 1 Accent 6" /><br /><w:lsdexception locked="false" priority="64" name="Medium Shading 2 Accent 6" /><br /><w:lsdexception locked="false" priority="65" name="Medium List 1 Accent 6" /><br /><w:lsdexception locked="false" priority="66" name="Medium List 2 Accent 6" /><br /><w:lsdexception locked="false" priority="67" name="Medium Grid 1 Accent 6" /><br /><w:lsdexception locked="false" priority="68" name="Medium Grid 2 Accent 6" /><br /><w:lsdexception locked="false" priority="69" name="Medium Grid 3 Accent 6" /><br /><w:lsdexception locked="false" priority="70" name="Dark List Accent 6" /><br /><w:lsdexception locked="false" priority="71" name="Colorful Shading Accent 6" /><br /><w:lsdexception locked="false" priority="72" name="Colorful List Accent 6" /><br /><w:lsdexception locked="false" priority="73" name="Colorful Grid Accent 6" /><br /><w:LsdException Locked="false" Priority="19" QFormat="true"<br />Name="Subtle Emphasis"/><br /><w:LsdException Locked="false" Priority="21" QFormat="true"<br />Name="Intense Emphasis"/><br /><w:LsdException Locked="false" Priority="31" QFormat="true"<br />Name="Subtle Reference"/><br /><w:LsdException Locked="false" Priority="32" QFormat="true"<br />Name="Intense Reference"/><br /><w:lsdexception locked="false" priority="33" qformat="true" name="Book Title" /><br /><w:LsdException Locked="false" Priority="37" SemiHidden="true"<br />UnhideWhenUsed="true" Name="Bibliography"/><br /><w:LsdException Locked="false" Priority="39" SemiHidden="true"<br />UnhideWhenUsed="true" QFormat="true" Name="TOC Heading"/><br /><w:lsdexception locked="false" priority="41" name="Plain Table 1" /><br /><w:lsdexception locked="false" priority="42" name="Plain Table 2" /><br /><w:lsdexception locked="false" priority="43" name="Plain Table 3" /><br /><w:lsdexception locked="false" priority="44" name="Plain Table 4" /><br /><w:lsdexception locked="false" priority="45" name="Plain Table 5" /><br /><w:lsdexception locked="false" priority="40" name="Grid Table Light" /><br /><w:lsdexception locked="false" priority="46" name="Grid Table 1 Light" /><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark" /><br /><w:lsdexception locked="false" priority="51" name="Grid Table 6 Colorful" /><br /><w:lsdexception locked="false" priority="52" name="Grid Table 7 Colorful" /><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 1"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 1" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 1" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 1" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 1" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 2"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 2" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 2" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 2" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 2" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 3"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 3" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 3" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 3" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 3" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 4"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 4" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 4" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 4" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 4" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 5"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 5" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 5" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 5" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 5" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="Grid Table 1 Light Accent 6"/><br /><w:lsdexception locked="false" priority="47" name="Grid Table 2 Accent 6" /><br /><w:lsdexception locked="false" priority="48" name="Grid Table 3 Accent 6" /><br /><w:lsdexception locked="false" priority="49" name="Grid Table 4 Accent 6" /><br /><w:lsdexception locked="false" priority="50" name="Grid Table 5 Dark Accent 6" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="Grid Table 6 Colorful Accent 6"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="Grid Table 7 Colorful Accent 6"/><br /><w:lsdexception locked="false" priority="46" name="List Table 1 Light" /><br /><w:lsdexception locked="false" priority="47" name="List Table 2" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark" /><br /><w:lsdexception locked="false" priority="51" name="List Table 6 Colorful" /><br /><w:lsdexception locked="false" priority="52" name="List Table 7 Colorful" /><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 1"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 1" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 1" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 1" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 1" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 1"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 2"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 2" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 2" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 2" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 2" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 2"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 3"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 3" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 3" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 3" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 3" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 3"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 4"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 4" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 4" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 4" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 4" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 4"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 5"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 5" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 5" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 5" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 5" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 5"/><br /><w:LsdException Locked="false" Priority="46"<br />Name="List Table 1 Light Accent 6"/><br /><w:lsdexception locked="false" priority="47" name="List Table 2 Accent 6" /><br /><w:lsdexception locked="false" priority="48" name="List Table 3 Accent 6" /><br /><w:lsdexception locked="false" priority="49" name="List Table 4 Accent 6" /><br /><w:lsdexception locked="false" priority="50" name="List Table 5 Dark Accent 6" /><br /><w:LsdException Locked="false" Priority="51"<br />Name="List Table 6 Colorful Accent 6"/><br /><w:LsdException Locked="false" Priority="52"<br />Name="List Table 7 Colorful Accent 6"/><br /></w:latentstyles><br /></xml><![endif]--><!--[if gte mso 10]><br /><style><br />/* Style Definitions */<br />table.MsoNormalTable<br />{mso-style-name:æ®éè¡¨æ ¼;<br />mso-tstyle-rowband-size:0;<br />mso-tstyle-colband-size:0;<br />mso-style-noshow:yes;<br />mso-style-priority:99;<br />mso-style-parent:"";<br />mso-padding-alt:0cm 5.4pt 0cm 5.4pt;<br />mso-para-margin:0cm;<br />mso-para-margin-bottom:.0001pt;<br />mso-pagination:widow-orphan;<br />font-size:10.0pt;<br />font-family:"Calibri","sans-serif";<br />mso-bidi-font-family:"Times New Roman";}<br /></style><br /><![endif]--><a href="https://rstudio.com">https://rstudio.com</a><br/></li></ol><p class="pptt" id="biaoti36840">### Procedure</p><ol class="npwone"><li>Calculate the average amount of RNA per cell (see Note 2)<ol class="npwtwo"><li>Prior to initiating this portion of the protocol, ensure that you have bench space, pipettes, and pipette tips that are clean and suitable for working with RNA (see Note 3).</li><li>Culture cells in a 6 cm dish until they are 60% to 80% confluent.</li><li>Preheat cell culture media and 0.25% trypsin-EDTA to 37 Â°C; once warmed, clean the bottles containing media, trypsin-EDTA, and 1x PBS with 70% ethanol and place them in a biological-safety cabinet.</li><li>Create a 0.125% trypsin solution by diluting the 0.25% trypsin-EDTA solution with an equal volume of 1x PBS.</li><li>Remove cultured cells from the incubator and place them in a biological-safety cabinet.</li><li>Aspirate the media from the cells using a disposable glass Pasteur pipet (or equivalent).</li><li>Wash cells on the plate by gently adding 4 ml of 1x PBS and then aspirating it.</li><li>Add 2 ml of 0.125% Trypsin solution to the cells and let the cell plate stand in the biological-safety cabinet at room temperature for 3 min.<br/><em>Note: Dissociation protocols may differ for your cell type.</em> </li><li>Using a clean 5 ml serological pipet attached to a pipet aid, pipet the trypsinized cell solution up and down against the plate to obtain a single-cell suspension.</li><li>Transfer the trypsinized cell solution to a 15 ml conical tube that contains 8 ml of culture media with 10% serum.</li><li>Invert the 15 ml conical tube 10-15 times to obtain a homogenous solution.</li><li>Using a P20 micropipette, remove 12 Î¼l of cell suspension and carefully pipette it into the bottom of a 1.7 ml microcentrifuge tube.</li><li>Repeat Steps A11 and A12 one more time to obtain two replicates for cell counting.</li><li>Add 12 Î¼l of Trypan Blue solution to each of the 12 Î¼l cell suspensions and mix by pipetting, taking care to keep the Trypan Blue/cell solution at the bottom of the tube.</li><li>Separately, spin down the remainder of the trypsinized cell solution (~10 ml; in the 15 ml conical) in a centrifuge for 5 min at 1,000 rpm (~200 <em>x g</em>).</li><li>During the spin, clean the hemocytometer and glass coverslip with a Kimwipe sprayed with 70% ethanol, to remove any particulates.</li><li>Add 12 Î¼l of the Trypan Blue cell suspension to the hemocytometer, ensuring that the solution distributes evenly underneath the coverslip.</li><li>Under a 20x objective, count the non-blue cells within each of the four hemocytometer quandrants, keeping track of the counts in each quadrant with a hand tally counter.</li><li>Calculate the number of cells per ml using the equation below, then average the cell-count-per-ml between replicates:<br/><ol class="npwthree"><li>Replicate 1, # of cells per ml = [(sum of counts in all 4 quadrants)/4]*2*10<sup>4</sup><br/></li><li>Replicate 2, # of cells per ml = [(sum of counts in all 4 quadrants)/4]*2*10<sup>4</sup><br/></li><li>Average # of cells per ml = (Replicate 1 counts + Replicate 2 counts)/2</li></ol></li><li>After the spin from Step A15 has completed, remove the trypsin-containing media taking care not to disturb the cell pellet, add 10 ml of 1x PBS, mix by pipeting, and spin down the PBS/cell solution in a centrifuge for 5 min at 1,250 rpm (~300 <em>x g</em>).</li><li>Remove the PBS, replace it with another 10 ml of 1x PBS, mix by pipetting, and spin down the cell solution in a centrifuge for 5 min at 1,250 rpm (~300 <em>x g</em>).</li><li>Remove all traces of PBS and add 1ml of TRIzol to the cell pellet (see Note 4).</li><li>Using a P1000, pipette up and down ~15x to lyse the cells and maximize the efficiency of RNA extraction.</li><li>Place the cell/TRIzol mixture into a -80 Â°C freezer.</li><li>Repeat Steps A1-A24 at least three times, ideally on separate days, to obtain biological replicates.</li><li>Remove the TRIzol suspensions from the freezer and let them thaw at room temperature.</li><li>Once thawed, let the TRIzol suspensions sit for 5 min at room temperature to help ribonucleoprotein complexes dissociate (see Note 5).</li><li>Add 0.2 ml of chloroform to the TRIzol suspension, vortex vigorously for ~20 s, and let the sample stand at room temperature for another 2 min.</li><li>Centrifuge the sample for 15 min at 12,000 <em>x g</em> at 4 Â°C.</li><li>Transfer the aqueous phase containing the RNA to a new 1.7 ml tube (~0.5 ml).</li><li>Add 10 Î¼l of linear acrylamide to the extracted aqueous phase and vortex vigorously.</li><li>Add 0.5 ml of isopropanol to the aqueous phase (amount roughly equal to the volume of the aqueous phase), and vortex vigorously.</li><li>Incubate for â¥1 h at -20 Â°C (this step differs from the manufacturerâs instructions for purification of RNA from TRIzol).</li><li>Centrifuge the sample at top speed for 30 min in a microcentrifuge at 4 Â°C (&gt; 12,000 <em>x g</em>; see Note 6).</li><li>Using a P1000, remove the water/isopropanol solution, being mindful not to remove the RNA pellet, which should be located below the hinge of the microcentrifuge tube.</li><li>To the precipitated RNA pellet, gently add 1 ml of an ice-cold mixture of 80% ethanol and 20% RNase-free water.</li><li>Using a P1000, remove the 80% ethanol, being mindful not to remove the RNA pellet.</li><li>Pulse-spin the 1.7 ml tube in a mini-centrifuge to bring the residual ethanol from the sides of the tube down to the bottom.</li><li>Using a P200 pipette and tip, remove the remaining 80% ethanol.</li><li>Repeat Steps A38-A39 until no 80% ethanol remains.</li><li>Re-suspend the pellet in 30 Î¼l of RNase-free water.</li><li>Let the RNA-containing solution stand for 1 h at room temperature with intermittent mixing (every 15 min) by flicking or vortexing and then pulse-spinning the tube, or by pipetting the solution up and down (see Note 7).</li><li>After the RNA has dissolved, quantify the concentration of RNA using a Nanodrop spectrophotometer.<br/><ol class="npwthree"><li>An ideal ratio of absorbance at 260 nm and 280 nm for RNA is between ~1.8 and ~2.<br/></li><li>If the RNA is contaminated with residual ethanol, phenol, or guanidine, or if the RNA is not completely dissolved, the 260/280 ratio will be lower, usually &lt; 1.6.<br/></li><li>See Thermo Fisher's technical notes on NanoDrop Spectrophotometers for more information.</li></ol></li><li>Next, determine whether the purified RNA is intact. To do this, set up a gel electrophoresis apparatus and prepare a 1% agarose/0.0001% ethidium bromide gel with 1x TAE buffer. Submerge the agarose gel in 1x TAE buffer.</li><li>In a 1.7 ml tube, mix ~250-500 ng of RNA with an appropriate amount of glycerol-based agarose gel loading buffer.</li><li>In separate lanes of the agarose gel, load the RNA/gel loading mixture as well as 0.5-1 Î¼g of DNA ladder (the latter sample provides a size reference).</li><li>Run the samples in 1x TAE about ~8 cm through the agarose gel, at a voltage of 5 V/cm of distance between electrodes.</li><li>Take a picture of the gel on an appropriate gel imaging system. Intact RNA purified from a typical mammalian cell should yield two distinct bands running at apparent sizes of ~1,500 and ~750 nucleotides relative to the DNA ladder, which correspond to the 28s and 18s rRNA species, respectively (Figure 2; see Note 8).<br/><br/><img alt="" class="layerphoto" height="258" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101551_5346.png" width="300"/><br/><strong>Figure 2. RNA run on a 1% agarose gel pre-stained with ethidium bromide.</strong> Relative to the DNA size ladder, the 28S and 18S rRNA species migrate at approximate sizes of ~1,400 and ~750 nucleotides, respectively.<br/><br/></li><li>If the RNA is intact, proceed to calculate the weight of RNA per cell:<br/><ol class="npwthree"><li>RNA-per-cell = [concentration of RNA in g/L] x [volume in L of water used to resuspend RNA (<em>e.g.</em>, 30 x 10<sup>-6</sup> L)]/[the number of cells lysed in TRIzol]<br/></li><li>Calculate the average of the RNA-per-cell numbers obtained from biological replicate RNA preparations (see Note 9).</li></ol><br/></li></ol></li><li>Prepare cDNA libraries for RNA-Seq (see Notes 10, 11, 12, and 13)<ol class="npwtwo"><li>Ensure that cDNA libraries are prepared from two or more biological replicate RNA preparations (see Note 14).</li><li>For each sample to be sequenced, aliquot 1 Î¼g of total cellular RNA in a total volume of 8 Î¼l of RNase-free water.</li><li>To each sample add 2 Î¼l of a fresh 1:100 dilution of ERCC RNA Spike-In Mix #1 (see Note 15).</li><li>Starting with the mixture of ERCC Spike-in RNA and 1 Î¼g of total cellular RNA, follow the manufacturerâs instructions for cDNA library preparation. We follow the instructions essentially as they are written in the user technical datasheet from KAPA (Note 16). The following are inputs required from the user:<br/><ol class="npwthree"><li>From Section 6 of the technical datasheet (âRNA Elution, Fragmentation and Primingâ), we select the fragmentation conditions of 6 min at 94 Â°C, which will generate 200-300 nucleotide-long RNA fragments.<br/></li><li>From Section 12 of the technical datasheet (âLibrary Amplificationâ), we perform PCR amplification of our final cDNA library using only half of our purified cDNA library (<em>i.e.</em>, 10 Î¼l of cDNA library in a 50 Î¼l PCR reaction), rather than all 20 Î¼l of library in the 50 Î¼l reaction (see Note 17).<br/></li><li>From cDNA libraries prepared using 1 Î¼g of total RNA, using half of the purified cDNA library, we typically perform 11 cycles of PCR for the final amplification step.</li></ol></li><li>Using a Qubit fluorometer, quantify the concentration of DNA in the PCR-amplified, purified cDNA library (see Note 18).</li><li>Prepare a 1% agarose/0.0001% ethidium bromide gel with 1x TAE buffer. Submerge the agarose gel in 1x TAE buffer.</li><li>Run 2 Î¼l (1/10<sup>th</sup>) of the amplified cDNA library ~6 cm into the agarose gel, alongside of 250 ng of 1 Kb Plus DNA Ladder.</li><li>Image the agarose gel and estimate the average size in base pairs of each prepared cDNA library (see Note 19).</li><li>Calculate the molarity of the purified cDNA library:<br/><ol class="npwthree"><li>The average molecular weight of a DNA base-pair is 650 Daltons, or 650 g/mole.<br/></li><li>Using the average length determined in Step B8 above, calculate the molar weight of the cDNA library:<br/>[cDNA library g/mole] = [average_length] x 650 g/mole<br/></li><li>Now, using the DNA concentration determined in Step B5 above, calculate the molarity of the cDNA library (see Note 20):<br/>[cDNA library moles/Liter] = [cDNA library concentration in ng/Î¼l]) x ( [1 x 10<sup>-9</sup> g]/[1 x 10<sup>-6</sup> L])*(1/[cDNA library g/mole])</li></ol></li><li>Pool together the cDNA libraries to be sequenced in a way that will ensure an equimolar amount of each library is present in the pool and that each library will be sequenced to a depth of &gt; 20 million reads (see Notes 21 and 22).</li><li>Sequence the pooled cDNA libraries on an Illumina platform (see Note 23).<br/></li></ol></li></ol><p class="pptt" id="biaoti36841">### Data analysis</p><p class="ppzz"><em>Note: See Note 24.</em> </p><ol class="npwone"><li>Align data and obtain read counts (see Note 25)<ol class="npwtwo"><li>Within a UNIX command terminal, create a master directory to perform the sequence alignment, filtering, and read-counting (<em>e.g.</em>, ./ercc_mpc_analysis).<br/><br/><span style="font-family:Courier New;">mkdir ./ercc_mpc_analysis</span><br/><br/></li><li>Obtain an RNA-Seq fastq file from the Illumina sequencing run (<em>e.g.</em> rnaseq_file.fastq), and move this file to the master directory (see Note 26).<br/><br/><span style="font-family:Courier New;">mv rnaseq_file.fastq ./ercc_mpc_analysis</span><br/><br/></li><li>Download the appropriate genome fasta file (<em>e.g.</em>, genome.fa) and gene-gtf file (<em>e.g.</em>, genes.gtf) for your cell type and place them in the master directory (./ercc_mpc_analysis) (see Note 27).</li><li>Download the ERCC Spike-In RNA annotation files from the ERCC RNA Spike-In product page on <a href="https://www.thermofisher.com/order/catalog/product/4456739?SID=srch-srp-4456739#/4456739?SID=srch-srp-4456739" target="_blank">Thermo Fisherâs website</a> and place them in the master directory -- ./ercc_mpc_analysis (see Note 28). File names are:<br/><ol class="npwthree"><li>ERCC Controls Analysis: ERCC RNA Spike-In Control Mixes (<em>e.g.</em>, ERCC92_conc.txt)<br/></li><li>ERCC92.fa &amp; ERCC92.gtf sequence and annotation files (.zip)</li></ol></li><li>Within the master directory (./ercc_mpc_analysis), create a new directory to store the genome index that will be built by STAR<br/><br/><span style="font-family:Courier New;">mkdir ./GenomeDir/</span><br/><br/></li><li>Build a STAR genome-index that includes both the genome and ERCC reference sequences; the index will be created and stored in ./GenomeDir/.<br/><br/>STAR --runThreadN 8<br/><span style="font-family:Courier New;">--runMode genomeGenerate </span><br/><span style="font-family:Courier New;"> --genomeDir ./GenomeDir</span><br/><span style="font-family:Courier New;"> --genomeFastaFiles genome.fa ERCC92.fa</span><br/><span style="font-family:Courier New;"> --sjdbGTFfile genes.gtf ERCC92.gtf</span><br/><br/></li><li>Use STAR to align an RNA-Seq fastq file (<em>e.g.</em>, rnaseq_file.fastq) to the genome index. The alignments will be saved to a file with the appendix âAligned.out.samâ (<em>e.g.</em>, rnaseq_file_out_Aligned.out.sam).<br/><br/>STAR --runThreadN 12 <br/><span style="font-family:Courier New;">--genomeDir ./GenomeDir</span><br/><span style="font-family:Courier New;"> --readFilesIn rnaseq_file.fastq</span><br/><span style="font-family:Courier New;"> --outFileNamePrefix rnaseq_file_out_</span><br/><br/></li><li>Use samtools to filter the âAligned.out.samâ file for mapping quality of &gt; 30 (this step selects for uniquely mapped reads). In this example, the filtered file is named ârnaseq_file_out _q30.samâ.<br/>samtools view -Shq 30<br/><br/><span style="font-family:Courier New;">rnaseq_file_out_Aligned.out.sam &gt; rnaseq_file_out _q30.sam</span><br/><br/></li><li>Use featureCounts in the Subread package to count the number of reads that align to each ERCC Spike-In transcript (Note 29). In this example, the file containing ERCC counts is named âercc_featureCounts_output.txtâ.<br/>featureCounts<br/><br/><span style="font-family:Courier New;">-s 2</span><br/><span style="font-family:Courier New;"> -a ERCC92.gtf </span><br/><span style="font-family:Courier New;"> -o ercc_featureCounts_output.txt </span><br/><span style="font-family:Courier New;"> rnaseq_file_out_q30.sam</span><br/><br/></li><li>Use featureCounts in the Subread package to count the number of reads that align to each genic transcript in the genes.gtf file. In this example, the file containing ERCC counts is named âmm9_genes_featureCounts_output.txtâ.<br/>featureCounts<br/><br/><span style="font-family:Courier New;">-s 2</span><br/><span style="font-family:Courier New;"> -a genes.gtf</span><br/><span style="font-family:Courier New;"> -o mm9_genes_featureCounts_output.txt</span><br/><span style="font-family:Courier New;"> rnaseq_file_out_q30.sam</span><br/><br/></li></ol></li><li>Create a standard curve that relates ERCC Spike-In RNA-Seq read counts to the absolute amount of each ERCC transcript added to the RNA just prior to preparing the cDNA library for RNA-Seq. We recommend using Excel or Rstudio for these calculations. Templates and examples can be found <a href="https://github.com/mschertzer/ercc_analysis" target="_blank">here</a>. Figure 3 below shows a standard curve derived from a single RNA-Seq replicate (Schertzer <em>et al.</em>, 2019).<br/><br/><p class="ppzz" style="margin-left:30px;"><img alt="" class="layerphoto" height="228" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101527_1593.png" width="300"/><br/><strong>Figure 3. Representative standard curve relating RNA-Seq read counts (y-axis) to molecular abundance of the ERCC Spike-In RNAs (x-axis)</strong> </p><ol class="npwtwo"><li>Copy and paste the contents of the ERCC92_conc.txt file (downloaded in step A4) into a new Excel spreadsheet â call this the âERCC Mix Inâ spreadsheet (See Note N30). A picture of the ERCC92_conc.txt file is below (Figure 4).<br/><br/><img alt="" class="layerphoto" height="77" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101509_6274.png" width="300"/><br/><strong>Figure 4. Screenshot of the ERCC92_conc.txt file</strong><br/><br/></li><li>Delete the final three columns of the table in the Excel spreadsheet ("concentration in Mix 2 (attomoles/ul)", "expected fold-change ratio", "log<sub>2</sub>(Mix 1/Mix 2)").</li><li>Create a new column in the tableâcolumn E, âAttomoles addedââthat uses the values in column D "concentration in Mix 1 (attomoles/ul)" to calculate the number of attomoles of ERCC Spike-In RNAs added to the total RNA prior to RNA-Seq library preparation (see Note 31; Figure 5). <br/><br/><img alt="" class="layerphoto" height="68" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101453_6637.png" width="300"/><br/><strong>Figure 5. Screenshot of an example calculation of the number of attomoles of ERCC Spike-Ins added to the RNA prior to library preparation</strong><br/><br/></li><li>Create a new columnâcolumn F, âMolesââthat divides the number in column E âAttomoles addedâ by 1E18.</li><li>Create a new columnâcolumn G, âMoleculesââthat multiplies the values in column F âMolesâ by 6.022E23 (molecules per mole; Avogadroâs number).</li><li>Create a new columnâcolumn H, âlog<sub>2</sub>(molecules)â â that calculates the log-base-2 of the values in column G âMoleculesâ.</li><li>Sort the table such that the data in column B âERCC_IDâ appear in ascending order; this will be important later (Figure 6).<br/><br/><img alt="" class="layerphoto" height="49" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101438_5875.png" width="300"/><br/><strong>Figure 6. Screenshot of sorted ERCC calculation table</strong><br/><br/></li><li>In a separate Excel spreadsheet, copy and paste the contents of the ercc_featureCounts_output.txt file generated in step A9âcall this the âfCounts dataâ spreadsheet (see Note 32).</li><li>Within this new spreadsheet, calculate the number of aligned reads per kilobase per million aligned reads (RPKM) for each ERCC transcript.<br/><ol class="npwthree"><li>First, in your dataset of interest, find the total number of reads that aligned to the genome with a mapping quality of &gt; 30. This can be done from the UNIX command line, using samtools view:<br/><br/><span style="font-family:Courier New;">samtools view -c rnaseq_file_out _q30.sam &gt; rnaseq_file_counts.txt</span><br/><br/><div style="white-space:nowrap;"><span style="font-size:10pt;"></span> </div></li><li>Next, create a new column in the Excel spreadsheetâcolumn H, âRPMââin which the read counts in column G are divided by aligned read count from ârnaseq_file_counts.txtâ then multiplied by 1 million. This gives reads per million (RPM).<br/></li><li>Then, create a new column in the Excel spreadsheetâcolumn I, âRPKMââin which the RPM value in column H is divided by the value in column F âlengthâ (the length in nucleotides of each ERCC transcript), then multiplied by 1,000. This converts RPM into RPKM, or read counts per kilobase of transcript per million aligned reads.<br/></li><li>Finally, create a new column in the Excel spreadsheetâcolumn J, âlog<sub>2</sub>(RPKM)ââthat calculates the log-base-2 of the values in column I âRPKMâ (Figure 7).<br/><div><br/></div></li></ol><img alt="" class="layerphoto" height="36" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101418_6388.png" width="300"/><br/><strong>Figure 7. Screenshot of ERCC RPKM calculation</strong><br/><br/></li><li>Ensure that the Excel spreadsheets created in step B1 and in step B8 are sorted by ERCC ID in ascending order (see Note 33).</li><li>Paste the log<sub>2</sub>(RPKM) values from the âfCounts dataâ spreadsheet (created in step B8) into a new column in the âERCC Mix Inâ spreadsheet (created in step B1).</li><li>Remove ERCC transcripts that had zero aligned reads.</li><li>Generate a scatter plot where log<sub>2</sub>(molecules) is on the x-axis and log<sub>2</sub>(RPKM) is on the y-axis. See Figure 3 and Figure 8 below.<br/><br/><img alt="" class="layerphoto" height="142" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101358_5991.png" width="300"/><br/><strong>Figure 8. Screenshot of scatter plot generation in Excel</strong><br/><br/></li><li>Fit a straight line to the data<br/><ol class="npwthree"><li>In Excel, select the points on the graph, right click, and âAdd Trendlineâ. In the window, select âLinearâ and âDisplay Equation on chartâ (Figure 9).<br/><br/><img alt="" class="layerphoto" height="173" loading="lazy" src="https://en-cdn.bio-protocol.org/attached/image/20200926/20200926101341_5279.png" width="300"/><br/><strong>Figure 9. Screenshot of trendline-adding in Excel</strong><br/><br/><div><span style="font-size:10pt;"></span> </div></li><li>Expect the R<sup>2</sup> value to be greater than 0.90 (typically, it is above 0.95).<br/></li><li>Use the y = mx + b equation in the next section.<br/></li></ol><br/></li></ol></li><li>Calculate molecules per cell for the lncRNA and mRNA genes of interest.<ol class="npwtwo"><li>Copy the contents of the mm9_genes_featureCounts_output.txt file (file generated in step A10) and paste them into a new Excel spreadsheet â call this the âMPCâ spreadsheet (see Note 34).</li><li>Using same procedure outlined in step B9 above, convert the read counts for each gene (column G of the MPC spreadsheet) into RPM as a new column H, then into RPKM as a new column I, and then into log<sub>2</sub>(RPKM) as a new column J.</li><li>For each gene of interest, calculate log<sub>2</sub>(molecules) using the y = mx + b equation from step B14.<br/><ol class="npwthree"><li>x = log<sub>2</sub>(molecules)<br/></li><li>y = log<sub>2</sub>(RPKM) values calculated in column J<br/></li><li>b = y-intercept from equation in step B14 <br/></li><li>m = slope from equation in step B14<br/></li><li>Create a new column in the MPC Excel spreadsheetâcolumn K âlog<sub>2</sub>(molecules)ââwhich performs the following calculation:<br/>x= (y-b)/m</li></ol></li><li>Create a new column in the MPC Excel spreadsheetâcolumn L âmoleculesââin which the value in column K is converted to molecules using the exponential 2x. In Excel notation, this is performed by setting the equation in column L to â=2^column_Kâ (see Note 35).</li><li>Create a final column in the MPC Excel spreadsheetâcolumn M âMPCââin which the value in column L is converted to molecules-per-cell:<br/><ol class="npwthree"><li>Divide 1 Î¼g, the amount of total RNA used to prepare the RNA-seq library, by the weight of RNA-per-cell calculated in Step A49 of the âProcedureâ section of this protocol. This value represents the approximate total number of cells used to prepare the RNA-seq library (see Note 36).<br/></li><li>For each transcript of interest, divide the number of molecules in column L by the total number of cells used for RNA-Seq to estimate molecules of transcript per cell.</li></ol></li></ol></li></ol><p class="pptt" id="biaoti36842">### Notes</p><ol class="npwtwok"><li>Roche recently purchased KAPA Biosystems and discontinued their small reaction-number sequence adapter kits. The SeqCap Adapter Kit is their replacement product, but note that this kit only comes in a 96-reaction format. For less than the list-price of the SeqCap kit, users can purchase their own Illumina-compatible adapters in bulk from a commercial oligonucleotide provider. These adapters then need to be resuspended and annealed by the user. However, the advantage of purchasing adapters in bulk is that the cost per reaction is dramatically reduced. Thus, users that plan to perform many RNA-seq assays (or, for that matter, any other *âseq assay) will find that purchasing and annealing their own adapters is far more cost effective than purchasing a pre-aliquoted set of adapters. For those interested in purchasing and annealing their own adapters, we have provided instructions <a href="https://github.com/mschertzer/ercc_analysis" target="_blank">here</a>. Users that plan to carry out only a small number of RNA-seq assays may find it more cost-effective to purchase their RNA-seq kits and adapters from a manufacturer that sells reagents in small-sized packages, such as NEB.</li><li>This portion of the protocol describes how to calculate the average amount of RNA per cell in a cell type of interest. The protocol is designed for cultured adherent cells but could easily be adapted to cultured suspension cells. With additional optimization, it could also be adapted to a tissue of interest. In this latter case, the user would need a method to approximate the total number of cells per mass of tissue of interest (for example, how many cells are present in one milligram of tissue?). With an accurate estimate of cell-number-per-mass-of-tissue, the user could then perform replicate rounds of RNA purification from a known mass of tissue. The yield in weight of RNA would then be divided by the number of cells that were used to obtain the RNA to derive an estimate of the amount of RNA per average cell in the tissue of interest.</li><li>To clean a bench, spray the area with a light coat of RNase Zap and wipe the solution clean with paper towels. Lightly spray the pipettes to be used for the RNA prep with RNase Zap and then wipe them clean. Use boxes of pipette tips that have not been exposed to any source of RNase. Common sources of RNase are from plasmid DNA preparation kits and human skin/saliva. On a given workday, minimize the chance of RNase contamination by performing RNA work prior to performing any plasmid DNA preparations (or any other protocol that involves an RNase) and wear gloves at all times. Moreover, although it may sound draconian, once an RNA preparation begins in earnest (Step A26, Procedure section) avoid talking, coughing, chewing gum <em>etc.</em> in the vicinity of open microcentrifuge tubes. We also use barrier tips on our pipettes. By taking these simple precautions, you will help to ensure the success of the protocol.</li><li>TRIzol contains phenol, which is corrosive to the eyes, skin, and respiratory tract. When working with TRIzol, users should wear safety goggles, closed-toed shoes, and a lab coat at all times and take care not to splash TRIzol on any part of their body. If users are sensitive to fumes from TRIzol, work should be performed in a fume hood. TRIzol needs to be disposed of by following safety guidelines that are appropriate to the institution.</li><li>Recently, Chujo and colleagues found that certain nuclear-retained lncRNAs are recovered at a higher efficiency when TRIzol suspensions are incubated for 10 min at 55 Â°C (Chujo and Hirose, 2017). In our prior study of lncRNA intracellular abundance (Schertzer <em>et al.</em>, 2019), we did not perform this 55 Â°C incubation step. However, we see no downside to the 55 Â°C incubation, and may perform it in the future.</li><li>Prior to starting the spin cycle, align the spines of each microcentrifuge tube so that the hinges are all facing outward. Aligning the spines will ensure that the RNA/acrylamide pellet in each tube is located directly below the tube hinge. Knowing where in the tube to expect the pellet helps to take some of the guesswork out of the protocol, especially when you are working with small amounts of RNA.</li><li>If mixing by pipetting, please note that the RNA pellet can sometimes stick to the inside of the pipette tip. If this scenario occurs, continue pipetting the water up and down until the pellet visibly dissolves.</li><li>RNA that has experienced varying levels of degradation will appear as a smear on the gel.</li><li>Using this procedure, our RNA-per-cell estimates for mouse embryonic stem and trophoblast stem cells arrived at 20 picograms and 30 picograms, respectively (Calabrese <em>et al.</em>, 2007; Schertzer <em>et al.</em>, 2019).</li><li>To estimate intracellular abundance, we recommend using RNA-Seq library preparation protocols that purify genic transcripts away from rRNA by rRNA-depletion rather than by polyA-selection. In a pilot study, we estimated intracellular abundance using rRNA-depletion and polyA-selection for three lncRNAs of interest (<em>Xist</em>, <em>Airn</em>, and <em>Kcnq1ot1</em>) and found that the two protocols arrived at dramatically different estimates (not shown). Our interpretation of these pilot data is that during the polyA-selection protocol, a number of factors likely cause the efficiency of capture to vary for different polyadenylated RNAs. Variations in the extent of polyadenylation, the length of the polyA tail, the extent to which RNA base-pairing interferes with polyA capture, and the amount of internal A-rich sequence may cause certain polyadenylated transcripts to be captured with greater efficiency than others. These variations would skew estimates of intracellular abundance in ways that are difficult to predict. In contrast, variations in polyA-capture efficiency are not relevant for rRNA-depletion protocols, which deplete rRNA from total RNA preparations using oligonucleotides that are complementary to the major rRNA species. Thus, intracellular abundance may be measured more accurately by rRNA-depletion RNA-Seq than by polyA-selection RNA-seq. That being said, transcripts harboring internal homology to the rRNAs would be selectively depleted by rRNA-depletion and would require special consideration under this protocol.</li><li>To prepare samples for RNA-Seq, our lab routinely uses the RNA HyperPrep Kit with RiboErase from Kapa Biosciences/Roche. The instructions from the KAPA RNA HyperPrep Kit with RiboErase are clear and walk users in-depth through each step of the protocol, which begins with the degradation of rRNA, followed by DNase treatment, RNA fragmentation and priming, cDNA synthesis, second-strand cDNA synthesis and A-tailing, adapter ligation, and finally, library amplification. Most of these steps are followed by a purification and buffer exchange using polystyreneâmagnetite beads that are provided as part of the kit. In our lab, this kit has been robust to multiple users collecting different datasets over timeframes that span multiple years. However, many other companies sell high-quality kits to prepare ribo-deplete RNA-seq libraries, including Illumina and NEB. Generally speaking, these kits should perform equivalently. For users that plan to perform only a few RNA-seq experiments, a company such as NEB may be preferable, because they sell kits in smaller sizes than KAPA.</li><li>The KAPA RNA HyperPrep protocol is a modified version of the dUTP second-strand protocol described in Parkhomchuk <em>et al.</em> (2009) and Levin <em>et al.</em> (2010), in which stranded-ness of the RNA-seq library is maintained by performing second-strand cDNA synthesis in the presence of dUTP, followed by cDNA library amplification using a DNA polymerase that has been engineered to preferentially amplify DNA that contains deoxythymidine and not deoxyuridine. When using this kit or any other kit that employs a dUTP-based method, researchers should be aware that the stranded-ness of the amplified library is not perfect. With a low frequency, the engineered DNA polymerases will still amplify the deoxyuridine-containing second-strand. The result of this low-frequency second-strand amplification that for highly expressed genes, a âshadowâ of RNA-seq signal is often visible on the strand that is opposite (<em>i.e.</em>, antisense) to the correct strand of the gene. In practice, this shadow signal has never affected our downstream analyses, but users should be aware that it exists.</li><li>It is not uncommon to make mistakes during the first run-through of this protocol. We recommend that first-time users start by going through the entire protocol below using only one or two samples from which biological material is non-limiting. This way, users can work out the logistics of library preparation without the stress of needing the protocol to work the very first time.</li><li>For a robust biological replicate, we recommend using RNA prepared on different days or from different animals <em>etc.</em> </li><li>To minimize pipetting error, we recommend pipetting volumes of 2 Î¼l or more. For example, instead of pipetting 1 Î¼l of ERCC Spike-In solution to 99 Î¼l RNase-free water, we would pipette 2 Î¼l of Spike-In solution to 198 Î¼l of RNase-free water.</li><li>KAPA RNA HyperPrep Kit with RiboErase; catalog number: <a class="blackNoUnder" target="_blank"> KK8560 </a> ; technical datasheet version KR1351âv2.17; this same datasheet is also included in our <a href="https://github.com/mschertzer/ercc_analysis" target="_blank">github</a> page.</li><li>The reason for using only half of the library is that it preserves cDNA material in case the user needs to repeat the final PCR reaction owing to an error in PCR setup, or over- or under-amplification of the cDNA library.</li><li>After PCR amplification, clean-up, and elution in 20 Î¼l of buffer as specified in the technical datasheet, the amount of DNA per sample should be in the range of 7-150 ng/Î¼l. Concentrations of DNA outside of this range may be acceptable, but in the rare instances in which the concentrations of our own cDNA libraries have fallen outside of this range, we have elected to repeat the final PCR amplification using an adjusted number of PCR cycles, rather than submit the originally-amplified library for RNA-seq. The reason for this is that the user is trying to ensure that the final PCR remains within the linear range of amplification. Amplification to a concentration of 150 ng/Î¼l may be close to the top of the linear range of the KAPA kit. Similarly, final library concentrations &lt; 7 ng/Î¼l may also be acceptable, but under this scenario, we have elected to repeat the final PCR using more cycle numbers rather than submit the low concentration libraries for sequencing. Most frequently, the final concentration of our amplified libraries is between 10-50 ng/Î¼l; this is our optimal target range.</li><li>Using the conditions for cDNA library preparation specified above, users should expect the average size of amplified DNA fragment in each library to be between 300-400 nucleotides.</li><li>Here is an example with numbers: The concentration calculated in Step B5 (Prodedure section) is 25 ng/Î¼l. The average library size estimated in Step B8 (Prodedure section) is 300 base pairs. The molarity of the library is 25 x (1 x 10<sup>-9</sup>)/(1 x 10<sup>-6</sup>)/(300 x 650) = 128 nM.</li><li>The high-throughput sequencing facility at UNC asks that users submit their pooled cDNA libraries to them at a final concentration of 15 nM. Thus, as an example, if we were hoping to include 12 separate cDNA libraries in a single pool, that would mean each library would need to be present in the pool at a final concentration of 1.25 nM, or [15 nM/12]. One easy way to create a pool of cDNA libraries at the appropriate concentration is to first create separate aliquots of each library at the final concentration of the poolâin this example, that concentration would be 15 nM. Then, equal volumes of each library can be combined to create the 15 nM pool.</li><li>In order to determine the maximum number of libraries that can be included in a pool such that that each library is still sequenced to an appropriate depth, first determine the number of sequencing reads that you expect to be returned from your run on the Illumina sequencing instrument. For example, at UNC, the average 75-cycle high-output run on a NextSeq500 Instrument will return 500 million reads. In order to obtain at least 20 million reads per cDNA library in a pool, the maximum number of libraries we should include in that pool is 500/20, or 25 libraries. In practice, we often include fewer libraries than this maximum number, which results in read-depths per library of &gt; 20 million reads. High read-depth per sample is never a problem. Moreover, fewer than 20 million reads from a single library may also be tolerable; just note that as the number of reads decreases, so does your ability to confidently quantify the abundance lowly-expressed transcripts.</li><li>We have performed our data analyses after obtaining 75 base, single-end reads from an Illumina NextSeq500 instrument. Shorter, longer, or even paired-end reads would also be suitable.</li><li>Please see the github page associated with this protocol for example files and analysis templates in Excel and in R (<a href="https://github.com/mschertzer/ercc_analysis" target="_blank">https://github.com/mschertzer/ercc_analysis</a>).</li><li>For RNA-Seq alignments we generally use STAR, but note that other aligners that support gapped-alignments should perform equivalently (Baruzzo <em>et al.</em>, 2017; Bushnell, 2010; Dobin <em>et al.</em>, 2013).</li><li>If following along with the example provided on the github page, you may use the SRA toolkit to download the fastq file associated with record SRR7685881 in the NCBI Sequence Read Archive (Leinonen <em>et al.</em>, 2011).</li><li>In the example in this protocol, we use the mm9 genome.fa and genes.gtf files compiled by the UCSC Genome Browser (Haeussler <em>et al.</em>, 2019) and downloaded from Illuminaâs iGenomes site: <a href="https://support.illumina.com/sequencing/sequencing_software/igenome.html" target="_blank">https://support.illumina.com/sequencing/sequencing_software/igenome.html</a>.</li><li>Users can also find these files on the github page associated with this protocol (<a href="https://github.com/mschertzer/ercc_analysis" target="_blank">https://github.com/mschertzer/ercc_analysis</a>).</li><li>The â-s 2â option of featureCounts is specific to libraries that are prepared using methods that generate âreverse-strandedâ data, such as the KAPA RNA HyperPrep Kit with RiboErase described in this protocol.</li><li>Steps B2 through B7 (Data analysis) below have already been performed in the âERCC Mix Inâ spreadsheet that is included in the âERCC_analysis_template.xlsxâ template on the github page associated with this protocol (<a href="https://github.com/mschertzer/ercc_analysis" target="_blank">https://github.com/mschertzer/ercc_analysis</a>).</li><li>In Schertzer <em>et al.</em> (2019), we added 2 Î¼l of a 1:100 dilution of ERCC Mix 1 Spike-In RNA to 1 Î¼g total RNA. Thus, to follow our example, divide the ERCC Spike-In Mix concentration in column D by 100 and then multiply by 2 to calculate the number of attomoles added as column E.</li><li>In the template âERCC_analysis_template.xlsxâ provided on the github page, this second spreadsheet is called âfCounts dataâ.</li><li>After sorting, the order in which each ERCC transcript appears in each spreadsheet should be identical.</li><li>For an example of MPC calculations performed over the entire transcriptome using an RNA-seq dataset from Schertzer <em>et al.</em> (2019), see the MPC spreadsheet in the âERCC vprtta analysis example.xlsxâ file on the github page associated with this protocol (<a href="https://github.com/mschertzer/ercc_analysis" target="_blank">https://github.com/mschertzer/ercc_analysis</a>).</li><li>The result of this calculation is an estimation of the number of RNA molecules that were present in the pool of total RNA that was used to prepare the RNA-Seq cDNA library (which in our case was 1 Î¼g total RNA).</li><li>In Schertzer <em>et al.</em> (2019), we estimated that our trophoblast stem cell line harbors 30 picograms per cell. Thus, 1 Î¼g of RNA corresponds to 33,333 cells.<br/></li></ol><p class="pptt" id="biaoti36843">### Acknowledgments</p><p class="ppzz">We thank Keean Braceros for proofreading this protocol and Jackson Trotman for the gel image used in Figure 2. This work was supported by the National Institutes of Health (NIH) Grant GM121806. Schertzer <em>et al.</em> (2019) is the original paper from which this protocol was derived.</p><p class="pptt" id="biaoti36844">### Competing interests</p><p class="ppzz">The authors have no competing interests to declare.</p><p class="pptt" id="biaoti36845">### References</p><ol class="npwtwok"><li>Baruzzo, G., Hayer, K. E., Kim, E. J., Di Camillo, B., FitzGerald, G. A. and Grant, G. R. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27941783" target="_blank">Simulation-based comprehensive benchmarking of RNA-seq aligners.</a> <em>Nat Methods</em> 14(2): 135-139. </li><li>Bushnell, B. (2010). BBMap (sourceforge.net/projects/bbmap/).</li><li>Cabili, M. N., Dunagin, M. C., McClanahan, P. D., Biaesch, A., Padovan-Merhar, O., Regev, A., Rinn, J. L. and Raj, A. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25630241" target="_blank">Localization and abundance analysis of human lncRNAs at single-cell and single-molecule resolution.</a> <em>Genome Biol</em> 16: 20. </li><li>Cabili, M. N., Trapnell, C., Goff, L., Koziol, M., Tazon-Vega, B., Regev, A. and Rinn, J. L. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21890647" target="_blank">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses.</a> <em>Genes Dev</em> 25(18): 1915-1927. </li><li>Calabrese, J. M., Seila, A. C., Yeo, G. W. and Sharp, P. A. (2007). <a href="http://www.ncbi.nlm.nih.gov/pubmed/17989215" target="_blank">RNA sequence analysis defines Dicer's role in mouse embryonic stem cells.</a> <em>Proc Natl Acad Sci U S A</em> 104(46): 18097-18102. </li><li>Calabrese, J. M., Sun, W., Song, L., Mugford, J. W., Williams, L., Yee, D., Starmer, J., Mieczkowski, P., Crawford, G. E. and Magnuson, T. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/23178118" target="_blank">Site-specific silencing of regulatory elements as a mechanism of X inactivation.</a> <em>Cell</em> 151(5): 951-963.</li><li>Cerase, A., Smeets, D., Tang, Y. A., Gdula, M., Kraus, F., Spivakov, M., Moindrot, B., Leleu, M., Tattermusch, A., Demmerle, J., Nesterova, T. B., Green, C., Otte, A. P., Schermelleh, L. and Brockdorff, N. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24469834" target="_blank">Spatial separation of Xist RNA and polycomb proteins revealed by superresolution microscopy.</a> <em>Proc Natl Acad Sci U S A</em> 111(6): 2235-2240.</li><li>Chen, J., Shishkin, A. A., Zhu, X., Kadri, S., Maza, I., Guttman, M., Hanna, J. H., Regev, A. and Garber, M. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26838501" target="_blank">Evolutionary analysis across mammals reveals distinct classes of long non-coding RNAs.</a> <em>Genome Biol</em> 17: 19. </li><li>Chujo, T. and Hirose, T. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29276943" target="_blank">Nuclear bodies built on architectural long noncoding rnas: unifying principles of their construction and function.</a> <em>Mol Cells</em> 40(12): 889-896.</li><li>Dobin, A., Davis, C. A., Schlesinger, F., Drenkow, J., Zaleski, C., Jha, S., Batut, P., Chaisson, M. and Gingeras, T. R. (2013). STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29(1): 15-21. </li><li>Gil, N. and Ulitsky, I. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31729473" target="_blank">Regulation of gene expression by cis-acting long non-coding RNAs.</a> <em>Nat Rev Genet</em> 21(2): 102-117. </li><li>Haeussler, M., Zweig, A. S., Tyner, C., Speir, M. L., Rosenbloom, K. R., Raney, B. J., Lee, C. M., Lee, B. T., Hinrichs, A. S., Gonzalez, J. N., Gibson, D., Diekhans, M., Clawson, H., Casper, J., Barber, G. P., Haussler, D., Kuhn, R. M. and Kent, W. J. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30407534" target="_blank">The UCSC Genome Browser database: 2019 update.</a> <em>Nucleic Acids Res</em> 47(D1): D853-D858. </li><li>Hezroni, H., Koppstein, D., Schwartz, M. G., Avrutin, A., Bartel, D. P. and Ulitsky, I. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25959816" target="_blank">Principles of long noncoding RNA evolution derived from direct comparison of transcriptomes in 17 species.</a> <em>Cell Rep</em> 11(7): 1110-1122.</li><li>Hrdlickova, R., Toloue, M. and Tian, B. (2017). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27198714" target="_blank">RNA-Seq methods for transcriptome analysis.</a> <em>Wiley Interdiscip Rev RNA</em> 8(1).</li><li>Jiang, L., Schlesinger, F., Davis, C. A., Zhang, Y., Li, R., Salit, M., Gingeras, T. R. and Oliver, B. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21816910" target="_blank">Synthetic spike-in standards for RNA-seq experiments.</a> <em>Genome Res</em> 21(9): 1543-1551.</li><li>Kopp, F. and Mendell, J. T. (2018). <a href="http://www.ncbi.nlm.nih.gov/pubmed/29373828" target="_blank">Functional Classification and Experimental Dissection of Long Noncoding RNAs.</a> <em>Cell</em> 172(3): 393-407. </li><li>Kutter, C., Watt, S., Stefflova, K., Wilson, M. D., Goncalves, A., Ponting, C. P., Odom, D. T. and Marques, A. C. (2012). <a href="http://www.ncbi.nlm.nih.gov/pubmed/22844254" target="_blank">Rapid turnover of long noncoding RNAs and the evolution of gene expression.</a> <em>PLoS Genet</em> 8(7): e1002841. </li><li>Leinonen, R., Sugawara, H., Shumway, M. and International Nucleotide Sequence Database, C. (2011). <a href="http://www.ncbi.nlm.nih.gov/pubmed/21062823" target="_blank">The sequence read archive.</a> <em>Nucleic Acids Res</em> 39(Database issue): D19-21.</li><li>Levin, J. Z., Yassour, M., Adiconis, X., Nusbaum, C., Thompson, D. A., Friedman, N., Gnirke, A. and Regev, A. (2010). <a href="http://www.ncbi.nlm.nih.gov/pubmed/20711195" target="_blank">Comprehensive comparative analysis of strand-specific RNA sequencing methods.</a> <em>Nat Methods</em> 7(9): 709-715.</li><li>Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G., Durbin, R. and Genome Project Data Processing, S. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" target="_blank">The Sequence Alignment/Map format and SAMtools.</a> <em>Bioinformatics</em> 25(16): 2078-2079.</li><li>Liao, Y., Smyth, G. K. and Shi, W. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24227677" target="_blank">featureCounts: an efficient general purpose program for assigning sequence reads to genomic features.</a> <em>Bioinformatics</em> 30(7): 923-930. </li><li>Necsulea, A., Soumillon, M., Warnefors, M., Liechti, A., Daish, T., Zeller, U., Baker, J. C., Grutzner, F. and Kaessmann, H. (2014). <a href="https://www.ncbi.nlm.nih.gov/pubmed/24463510" target="_blank">The evolution of lncRNA repertoires and expression patterns in tetrapods.</a> <em>Nature</em> 505(7485): 635-640.</li><li>Ninomiya, K. and Hirose, T. (2020). <a href="http://www.ncbi.nlm.nih.gov/pubmed/32093161" target="_blank">Short tandem repeat-enriched architectural rnas in nuclear bodies: functions and associated diseases.</a> <em>Noncoding RNA</em> 6(1).</li><li>Parkhomchuk, D., Borodina, T., Amstislavskiy, V., Banaru, M., Hallen, L., Krobitsch, S., Lehrach, H. and Soldatov, A. (2009). <a href="http://www.ncbi.nlm.nih.gov/pubmed/19620212" target="_blank">Transcriptome analysis by strand-specific sequencing of complementary DNA.</a> <em>Nucleic Acids Res</em> 37(18): e123.</li><li>Raj, A. and Rinn, J. L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31043413" target="_blank">Illuminating Genomic Dark Matter with RNA Imaging.</a> <em>Cold Spring Harb Perspect Biol</em> 11(5).</li><li>RStudio_Team. (2015). <a href="http://www.rstudio.com/" target="_blank">RStudio: Integrated Development for R.</a> </li><li>Schertzer, M. D., Braceros, K. C. A., Starmer, J., Cherney, R. E., Lee, D. M., Salazar, G., Justice, M., Bischoff, S. R., Cowley, D. O., Ariel, P., Zylka, M. J., Dowen, J. M., Magnuson, T. and Calabrese, J. M. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/31256989" target="_blank">lncRNA-Induced Spread of Polycomb Controlled by Genome Architecture, RNA Abundance, and CpG Island DNA.</a> <em>Mol Cell</em> 75(3): 523-537 e510. </li><li>Schuler, A., Ghanbarian, A. T. and Hurst, L. D. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25158797" target="_blank">Purifying selection on splice-related motifs, not expression level nor RNA folding, explains nearly all constraint on human lincRNAs.</a> <em>Mol Biol Evol</em> 31(12): 3164-3183. </li><li>Schwaber, J., Andersen, S. and Nielsen, L. (2019). <a href="http://www.ncbi.nlm.nih.gov/pubmed/30805297" target="_blank">Shedding light: The importance of reverse transcription efficiency standards in data interpretation.</a> <em>Biomol Detect Quantif</em> 17: 100077.</li><li>Smeets, D., Markaki, Y., Schmid, V. J., Kraus, F., Tattermusch, A., Cerase, A., Sterr, M., Fiedler, S., Demmerle, J., Popken, J., Leonhardt, H., Brockdorff, N., Cremer, T., Schermelleh, L. and Cremer, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/25057298" target="_blank">Three-dimensional super-resolution microscopy of the inactive X chromosome territory reveals a collapse of its active nuclear compartment harboring distinct Xist RNA foci.</a> <em>Epigenetics Chromatin</em> 7: 8.</li><li>Sunwoo, H., Wu, J. Y. and Lee, J. T. (2015). <a href="http://www.ncbi.nlm.nih.gov/pubmed/26195790" target="_blank">The Xist RNA-PRC2 complex at 20-nm resolution reveals a low Xist stoichiometry and suggests a hit-and-run mechanism in mouse cells.</a> <em>Proc Natl Acad Sci U S A</em> 112(31): E4216-4225. </li><li>Tsanov, N., Samacoits, A., Chouaib, R., Traboulsi, A. M., Gostan, T., Weber, C., Zimmer, C., Zibara, K., Walter, T., Peter, M., Bertrand, E. and Mueller, F. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27599845" target="_blank">smiFISH and FISH-quant - a flexible single RNA detection approach with super-resolution capability.</a> <em>Nucleic Acids Res</em> 44(22): e165.</li><li>Ulitsky, I. (2016). <a href="http://www.ncbi.nlm.nih.gov/pubmed/27573374" target="_blank">Evolution to the rescue: using comparative genomics to understand long non-coding RNAs.</a> <em>Nat Rev Genet</em> 17(10): 601-614. </li><li>Washietl, S., Kellis, M. and Garber, M. (2014). <a href="http://www.ncbi.nlm.nih.gov/pubmed/24429298" target="_blank">Evolutionary dynamics and tissue specificity of human long noncoding RNAs in six mammals.</a> <em>Genome Res</em> 24(4): 616-628.<br/></li></ol>